<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Arizona_State_Notebook_August"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Arizona_State/Notebook/August">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Arizona_State/Notebook/August&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Arizona_State/Notebook/August&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Arizona_State/Notebook/August&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Arizona_State/Notebook/August" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Arizona State/Notebook/August</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV style="font-size:18pt;"><FONT face="Calibri" style="color:#000000"><CENTER>Notebook: August
</CENTER></FONT></DIV><DIV><TABLE cellspacing="0"><TBODY><TR><TD style="background-color: #ffffff" width="75px" valign="top"><DIV class="glossymenu"><A class="menuitem" href="https://2011.igem.org/Team:Arizona_State">Home</A><A class="menuitem" href="https://2011.igem.org/Main_Page">iGEM 2011 Home</A><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Project">Project</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/Introduction">Introduction</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/CRISPR">CRISPR</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/E_coli">E. coli</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/B_halodurans">B. halodurans</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/L_innocua">L. innocua</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/Software">Software</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/Future">Future</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Project/References">References</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Lab">Lab</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Photos">Photos</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Safety">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Lab/Acknowledgements">Acknowledgements</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Results">Results</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Results/Data">Data</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Parts">BioBricks</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Lab">Notebook</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/AprilMay">April/May</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/June">June</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/July">July</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/August">August</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/September">September</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/PCRLog">Cas PCR Log</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/Sequences">Sequence Information</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Notebook/Other">Other Documents</A></LI></UL></DIV><A class="menuitem submenuheader" href="https://2011.igem.org/Team:Arizona_State/Outreach">Human Practices</A><DIV class="submenu"><UL><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Events">Events</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Collaborations">Collaborations</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Exploring_Synthetic_Biology">Exploring Synthetic Biology</A></LI><LI><A href="https://2011.igem.org/Team:Arizona_State/Outreach/Outreach_in_Practice">Outreach in Practice</A></LI></UL></DIV></DIV></TD><TD width="880" valign="top" style="padding: 10px; border: 5px solid #ffffff; color: #000; background-color: white"><DIV style="position: relative; margin-top: -450px; min-height: 450px"><DIV> <H2><SPAN class="mw-headline" id="Monday.2C_August_1"> Monday, August 1 </SPAN></H2><UL><LI> Checked plates
</LI></UL><DL><DD><UL><LI> Nothing
</LI></UL></DD></DL><UL><LI> Another RLT
</LI></UL><DL><DD><UL><LI> This time with a vengeance
</LI><LI> 2x Seq1, RA, DA in 1K3
</LI><LI> TSS cells
</LI></UL></DD></DL><UL><LI> Made more LB Kan broth
</LI></UL><DL><DD><UL><LI> Because previous attempt was bad, ampR grew in it
</LI></UL></DD></DL><UL><LI> Tested TSS cells with puc19
</LI><LI> Submitted BMES abstract
</LI></UL><H2><SPAN class="mw-headline" id="Tuesday.2C_August_2"> Tuesday, August 2 </SPAN></H2><UL><LI> Transformation results from yesterday:
</LI></UL><DL><DD><UL><LI> 4 colonies on seq1
</LI><LI> DA colonies all clumped together
</LI><LI> RA fail, among stacks of failed transformations 3:
</LI></UL></DD></DL><UL><LI> New RLT for 2x
</LI></UL><DL><DD><UL><LI> Varying DNA volumes added (1, 2.5, and 5 uL RLTs)
</LI></UL></DD></DL><UL><LI> PCR:
</LI></UL><DL><DD><UL><LI> Seq1, RA, DA in puc57 using puc57 primers
</LI></UL></DD><DD><UL><LI> PCR settings: from previous
</LI></UL></DD></DL><UL><LI> Gel results: look the same as before
</LI></UL><DL><DD><UL><LI> Clear bands ~100 for Seq1
</LI><LI> ~200 for DA, RA
</LI></UL></DD></DL><UL><LI> Submitted for sequencing:
</LI></UL><DL><DD><UL><LI> BAHHH jason (sequencing delays)
</LI><LI> Resubmitted 1x Seq1, RA, DA(pcr and prep), GFP, RFP, Cas3
</LI></UL></DD></DL><UL><LI> Streak plates
</LI></UL><DL><DD><UL><LI> BL21 DE3 (x2, one clearly better than the other)
</LI><LI> 2xDA in pSB1K3 (clumped colonies initially)
</LI></UL></DD></DL><UL><LI> Liquid culture
</LI></UL><DL><DD><UL><LI> 2xSeq1 in psb1k3
</LI><LI> BL21DE3 from glycerol stock
</LI></UL></DD><DD><UL><LI> Test tube with larger stopper may be contaminated
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Wednesday.2C_August_3"> Wednesday, August 3 </SPAN></H2><UL><LI> Liquid cultures
</LI></UL><DL><DD><UL><LI> 2x Seq1
</LI><LI> 1xDA
</LI></UL></DD></DL><UL><LI> PCR
</LI></UL><DL><DD><UL><LI> Cleanup of PCR amplified Seq1, RA, DA from puc57
</LI><LI> (We gelled this last night, looked good)
</LI><LI> Used RA from this for RLT --&gt;</LI></UL></DD></DL><UL><LI> RLT
</LI></UL><DL><DD><UL><LI> RA from PCR cleanup
</LI></UL></DD></DL><UL><LI> Prepared TSS competent cells
</LI><LI> Notes:
</LI></UL><DL><DD><UL><LI> Decided not to continue with p. furiosus due to the cmr array's length (9kb)
</LI><LI> Instead, we decided to explore our other options, like Type II streptococcus strains (which only have one cas protein involved in their mechanisms of silencing)
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Thursday.2C_August_4"> Thursday, August 4 </SPAN></H2><UL><LI> ABCDE PCR with the newest primers (v3)
</LI></UL><DL><DD><UL><LI> 98deg initial denaturation for 30 seconds
</LI><LI> Cycle {10 sec at 98 degrees
</LI></UL></DD><DD><UL><LI> Anneal at 63 degrees for 30 sec
</LI><LI> Elongate at 72 degrees for 130 sec}
</LI></UL></DD></DL><UL><LI> 35 cycles
</LI><LI> Then extension for 450 seconds
</LI></UL><UL><LI> Results show clearly a band above where we would like it, at around 8kbp, and also some bands below the desired length at around 2kbp
</LI></UL><H2><SPAN class="mw-headline" id="Friday.2C_August_5"> Friday, August 5 </SPAN></H2><UL><LI> Miniprep:
</LI></UL><DL><DD><UL><LI> Using old protocol
</LI><LI> RAGE, SEGE, RFP overnight cultures
</LI></UL></DD></DL><UL><LI> Gel:
</LI></UL><DL><DD><UL><LI> PCR from last night
</LI></UL></DD><DD><UL><LI> CasABCDE (upper):
</LI></UL></DD><DD><UL><LI> See photo
</LI><LI> Band &gt; 10k, genomic?
</LI></UL></DD></DL><UL><LI> CasABCDE (lower):
</LI></UL><DL><DD><UL><LI> Same
</LI></UL></DD></DL><UL><LI> Gel extraction of PCR 8-4 (see photo)
</LI><LI> PCR of gel extraction CasABCDE: 
</LI></UL><DL><DD><UL><LI> Sample 1,2,3
</LI></UL></DD></DL><UL><LI> Primers: CasABcDE_V3 F/R
</LI></UL><DL><DD><UL><LI> Thermocycle: ABCDE802 
</LI><LI> Template: gel extraction sample 1 of PCR 8-4 (above)
</LI></UL></DD></DL><UL><LI> Mg2+ gradient
</LI></UL><DL><DD><UL><LI> Sample1: 0 mM 
</LI><LI> Sample2: 1mM 
</LI><LI> Sample3: 2mM
</LI><LI> Cycle strarted 5:30pm
</LI></UL></DD></DL><UL><LI> Colony PCR:
</LI></UL><DL><DD><UL><LI> Sample 1c, 2c,3c
</LI><LI> Primers: CasABcDE_V3 F/R
</LI></UL></DD></DL><UL><LI> Thermocycle: ABCDE802 
</LI></UL><DL><DD><UL><LI> Template: colonies from Jon's source plate 
</LI></UL></DD><DD><UL><LI> 1c:BL21 DE3 -control
</LI><LI> 2c: BL21 DE3 -control
</LI><LI> 3c: Mg1655 +(hopefully we can amplify cas genes this way)
</LI></UL></DD></DL><UL><LI> Cycle started 5:50pm
</LI></UL><UL><LI> New 10mM dNTP solution made from dNTP stock.
</LI><LI> RLT:
</LI></UL><DL><DD><UL><LI> Todo
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Monday.2C_August_8"> Monday, August 8 </SPAN></H2><P>GROUP MEETING
</P><UL><LI> Kylie's Update
</LI></UL><DL><DD><UL><LI> Tried SLIC, not very successful
</LI></UL></DD><DD><UL><LI> (SLIC is a modification of Gibson)
</LI></UL></DD></DL><UL><LI> New Ligation Method
</LI></UL><DL><DD><UL><LI> Idea is to cut with X, S
</LI><LI> This can then stick together multiple times, theoretically
</LI><LI> Different ligation times can allow for longer arrays (1x, 2x, 3x, 4x, 5x…)
</LI><LI> *Tried this for Seq1
</LI></UL></DD><DD><UL><LI> Result: multiple bands at increasingly large lengths
</LI></UL></DD></DL><DL><DD><UL><LI> w/normal ligation control to compare
</LI></UL></DD></DL><UL><LI> Consideration: could ligate &quot;upside-down&quot; - will this have any negative effect?
</LI></UL><UL><LI> Approach
</LI></UL><DL><DD><UL><LI> RB + RA --&gt; RSR or &quot;Seq2&quot; essentially
</LI></UL></DD><DD><UL><LI> Then, we can ligate using the KSB method
</LI></UL></DD></DL><UL><LI> Question: if there is an &quot;upside-down&quot; section, what even happens?
</LI></UL><DL><DD><UL><LI> Is the complement side just skipped over? Will it hairpin? Will it mess something up?
</LI><LI> Our best guess is that it will be fine, perhaps not every spacer will &quot;work&quot; but enough will
</LI></UL></DD></DL><UL><LI> What about self-targeting?
</LI></UL><DL><DD><UL><LI> Sci tag prevents self targeting
</LI></UL></DD></DL><UL><LI> Organization
</LI></UL><DL><DD><UL><LI> Personal Dropbox Lab Notebooks
</LI></UL></DD></DL><UL><LI> Specialization
</LI></UL><DL><DD><UL><LI> Dan -- RB + RA, b. halodurans focus
</LI><LI> Abhi -- &quot;Seq2&quot; assembly (XS method, etc)
</LI><LI> Kylie -- Listeria innocua
</LI><LI> Madeline -- CasABCDE, Cas3 (and put in Duet)
</LI><LI> Keith -- Ginkgo
</LI><LI> Nisarg -- Ginkgo
</LI><LI> Ryan -- CasABCDE, Cas3
</LI><LI> Ruben -- drylab
</LI><LI> Ethan -- drylab
</LI><LI> Joseph -- XS method (seq1, RA)
</LI></UL></DD></DL><UL><LI> To order
</LI></UL><DL><DD><UL><LI> Gloves
</LI><LI> Plates
</LI><LI> T4 polymerase
</LI><LI> Miniprep
</LI><LI> Listeria innocua items
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Tuesday.2C_August_9"> Tuesday, August 9 </SPAN></H2><UL><LI> Compiled cas gene information to table on wiki
</LI><LI> Confirmed that RA was successfully amplified with a gel
</LI></UL><UL><LI> Proceeded to prepare XS restriction/ligation for RA
</LI><LI> Actual reagent volumes for 50 uL restriction/ligation can be found in lab notebook under 8-9-11
</LI><LI> Incubated restriction/ligation at 37 degrees overnight
</LI><LI> Liquid cultures from 8/8 failed to grow
</LI></UL><DL><DD><UL><LI> Potentially bad LB-Kan broth
</LI></UL></DD></DL><UL><LI> Made new LB-Kan broth
</LI><LI> New liquid cultures of Seq1 and RA (2x, each)
</LI></UL><DL><DD><UL><LI> Old LB-Kan broth
</LI><LI> New LB-Kan broth
</LI></UL></DD></DL><P>TO DO for 8-10-11:  
</P><UL><LI> Gel RL results
</LI><LI> PCR cleanup restriction/ligation results
</LI><LI> Possibly PCR cleanup RA inserts from PCR amplification for re-restriction/re-ligation
</LI></UL><H2><SPAN class="mw-headline" id="Wednesday.2C_August_10"> Wednesday, August 10 </SPAN></H2><UL><LI> Liquid cultures from 8/9 all failed.
</LI></UL><DL><DD><UL><LI> Both old and new LB+Kan broth
</LI></UL></DD></DL><UL><LI> Positive and negative controls started
</LI></UL><DL><DD><UL><LI> Old and new LB-Kan broth
</LI></UL></DD></DL><UL><LI> RA and Seq1 from 8/8 streak plated
</LI><LI> Liquid cultures in LB-Amp broth started
</LI></UL><DL><DD><UL><LI> From pUC57, RA and Seq1
</LI><LI> 2 liquid cultures of each
</LI></UL></DD></DL><UL><LI> Gel results for last night's PCR of Cas3 (see pic)
</LI></UL><DL><DD><UL><LI> ((recall settings: Cas3724 protocol, which is the previously successful run))
</LI><LI> Very bright nonspecific band around 1000
</LI><LI> Lots of small bright bands at low lengths (200-600)
</LI><LI> Band at the top near 10k
</LI><LI> MAYBE a slight hint of a band around where we want it, but not much there
</LI></UL></DD></DL><UL><LI> Discussion:
</LI></UL><DL><DD><UL><LI> I am not sure why we are not seeing the same result that we saw on 7/24.
</LI><LI> Perhaps trying a higher elongation time would help.
</LI><LI> I will compile a document with our PCR history, perhaps this will help us figure things out.
</LI></UL></DD></DL><UL><LI> Really, we need to be doing this over and over in a day. Multiple runs instead of just one. Gotta get to that point.
</LI><LI> Cas3 attempt
</LI></UL><DL><DD><UL><LI> Same settings, used 26ng/uL template instead as this is the template that was used back on 7/24 [Sample ID: PRANCER]
</LI></UL></DD></DL><UL><LI> Gel results (see pic)
</LI></UL><DL><DD><UL><LI> Seeing a nonspecific-ish area around 1000, also lots of smaller bands
</LI></UL></DD></DL><UL><LI> Ran another PCR for ABCDE, same settings as before but one degree higher for annealing temp. Also added DMSO. [sample ID: PRANQSTER]
</LI><LI> Ligated sample 4L (Seq1) to pSB1A3 linearized backbone cut with XbaI and SpeI then deposphorylated. 
</LI><LI> Samples 4L 1A3 1 and 4L 1A3 2 (12ul of 4L and 2ul of 1A3 in 20ul reaction)
</LI><LI> Negative control 1A3 L 
</LI><LI> Ligase heat inactivated by &quot;OVERKILL&quot; protocol for 10 minutes.
</LI><LI> Samples stored at -20 degrees on restriction ligation section of tray.
</LI></UL><H2><SPAN class="mw-headline" id="Thursday.2C_August_11"> Thursday, August 11 </SPAN></H2><UL><LI> Ran a gel of last night's PCR of ABCDE (see pic)
</LI></UL><DL><DD><UL><LI> DMSO worked -- same results as previously but less overall bands
</LI><LI> Seemed to hone in on the band that is above our desired length (~8k or so)
</LI></UL></DD></DL><UL><LI> (For reference to the picture, we ran some kind of restriction product that I can't recall on the right side w/ Hyperladder II)
</LI></UL><DL><DD><UL><LI> Does anyone remember this? Might be Restricted RB
</LI></UL></DD></DL><UL><LI> Liquid cultures of leader sequences
</LI></UL><DL><DD><UL><LI> One colony from each of the 4 plates
</LI></UL></DD><DD><UL><LI> Leader Seq MG1655 Dilute
</LI><LI> Leader Seq MG1655 Concentrated
</LI><LI> Leader Seq B. halo Dilute
</LI><LI> Leader Seq B. halo Concentrated
</LI></UL></DD></DL><UL><LI> Temperature Gradient PCR (Madeline and Ryan)
</LI></UL><DL><DD><UL><LI> CasABCDE
</LI></UL></DD><DD><UL><LI> MgCl2 --&gt; 2uL, 4uL, 5uL
</LI><LI> Temp --&gt; 60, 63, 66, 69
</LI></UL></DD></DL><UL><LI> Cas3
</LI></UL><DL><DD><UL><LI> MgCl2 --&gt; 2uL, 4uL, 5uL
</LI><LI> Temp --&gt; 60, 62, 64, 66
</LI></UL></DD></DL><UL><LI> Gel Result (see pic)
</LI></UL><DL><DD><UL><LI> Ran MgCl210 for each of the 8 temp rows (4ea for ABCDE, Cas3)
</LI><LI> No bands for ABCDE 3:
</LI><LI> Similar results that we have been getting for Cas3
</LI></UL></DD><DD><UL><LI> 60 and 62 have bands less than our desired target, up to perhaps 1500
</LI><LI> 64 had a couple of bands above this, perhaps one faint one near our 2600 target
</LI><LI> 66 had more nonspecific amplification, and no clear bands in our target range
</LI></UL></DD></DL><UL><LI> Moving forward
</LI></UL><DL><DD><UL><LI> Nest for ABCDE, we will see how this goes
</LI><LI> Cas3 we'll have to keep trying…perhaps it may be worth it to try DMSO, longer elongation time, something. Maybe Taq.
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Friday.2C_August_12"> Friday, August 12 </SPAN></H2><UL><LI> Visit to Bioscience HS
</LI></UL><DL><DD><UL><LI> Madeline, Abhi, Nisarg
</LI><LI> Took a little tour of the school, saw their lab space (which is pretty divine)
</LI></UL></DD><DD><UL><LI> Fun tidbit: each room has a plaque outside with the room number and an element name for the type of room (ex: O for office, He for men's restroom, Fe for women's restroom, etc..)
</LI></UL></DD></DL><UL><LI> Spoke w/ Ms. Singh's biotech class + some interested sophomores and seniors
</LI><LI> They will likely start an iGEM team, they seem quite interested
</LI></UL><DL><DD><UL><LI> Juan will be captain
</LI></UL></DD></DL><UL><LI> Singh will contact us soon after they speak with the administration about how it will all fit in and what our role will be
</LI></UL><UL><LI> PCR using Nest primers for ABCDE
</LI></UL><DL><DD><UL><LI> Tm1: 74, Tm2: 77 --&gt; Anneal at 72 deg (higher than this is not recommended)
</LI><LI> Used 2-step Phusion protocol
</LI></UL></DD><DD><UL><LI> 98 deg for 30 sec
</LI><LI> {98deg for 10 sec, 72 deg for 2:30} X 35 cycles
</LI><LI> Extension for 7:30 at 72 deg
</LI><LI> Held at 4deg until run was cancelled
</LI></UL></DD></DL><UL><LI> Gel results (see pic)
</LI></UL><DL><DD><UL><LI> The 8x MgCl2 tube has the best result
</LI><LI> Nice clear bands at around 600, 1k, 2k, 4k, 6k, 8k
</LI><LI> PLAN: use this as template for ABCDE PCR
</LI></UL></DD><DD><UL><LI> Use 1uL per tube (note, we don't know the concentration of the strand we want as template, so this is our best guess as to what would get us between 1pg and 10ng as recommended by the Phusion protocol for non-genomic DNA)
</LI></UL></DD></DL><UL><LI> Also today:
</LI></UL><DL><DD><UL><LI> PCR amplification of Seq1, RA
</LI></UL></DD><DD><UL><LI> 14 tubes total (7 each at null, 0, 1, 2, 4, 8, 10)
</LI></UL></DD></DL><UL><LI> Note we are running out of Phusion already!
</LI></UL><UL><LI> Another PCR (Madeline and Kylie)
</LI></UL><DL><DD><UL><LI> Using template from earlier PCR
</LI><LI> 3 tubes, all had 8x MgCl2 (4 uL)
</LI><LI> Titration of template concentration: 1/5, 1/50, 1/500
</LI></UL></DD><DD><UL><LI> Note, for 1/500 we ran out of Phusion (saddest bear 3=) so we used Taq
</LI></UL></DD></DL><UL><LI> Protocol
</LI></UL><DL><DD><UL><LI> Same as ABCDE808 but 35 cycles total (2:10 elongation time, 63deg annealing temp)
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Saturday.2C_August_13"> Saturday, August 13 </SPAN></H2><UL><LI> Restriction on poly x to remove incorrectly oriented pieces
</LI><LI> Gel results for Taq Polymerase try at Cas3 (see picture)
</LI></UL><DL><DD><UL><LI> Old primers: nonspecific all the way down
</LI><LI> Newer primers (v2): a series of bands all below 1000bp
</LI></UL></DD></DL><UL><LI> Other things in lab:
</LI></UL><DL><DD><UL><LI> XS Restriction after overnight ligation
</LI><LI> 8/8 plate growth for Keith's Ginkgos 
</LI><LI> Liquid cultures of 2x attempts, Dan's RABs
</LI><LI> Miniprep of leader sequences &amp; nanodrops (maybe this was Friday)	
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Sunday.2C_August_14"> Sunday, August 14 </SPAN></H2><UL><LI> Genome prep MG1655
</LI></UL><H2><SPAN class="mw-headline" id="Monday.2C_August_15"> Monday, August 15 </SPAN></H2><P>Team meeting
</P><UL><LI> Updates
</LI></UL><DL><DD><UL><LI> Cas genes - not looking good, ran out of Phusion, will keep trying w/ some adjustments
</LI><LI> Ginkgo woes
</LI></UL></DD></DL><UL><LI> Today
</LI></UL><DL><DD><UL><LI> Liquid culture of MG1655
</LI></UL></DD><DD><UL><LI> to make it competent
</LI></UL></DD></DL><UL><LI> Sequencing
</LI></UL><DL><DD><UL><LI> Leader Sequences
</LI><LI> 2x RA
</LI><LI> Duet
</LI><LI> Whatever else is on the board
</LI></UL></DD></DL><UL><LI> Streak plate pRSF Duet (old mini prep, could be weird)
</LI><LI> New media, new plates
</LI><LI> Streak plate polySeq1
</LI></UL><DL><DD><UL><LI> Liquid culture later, mini prep tonight or tomorrow
</LI></UL></DD></DL><UL><LI> Cas3 PCR w/ uber long elongation
</LI></UL><DL><DD><UL><LI> Look again at annealing temps	
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Tuesday.2C_August_16"> Tuesday, August 16 </SPAN></H2><UL><LI> Liquid cultures
</LI></UL><DL><DD><UL><LI> Kylie's 4L1A3 (orig. XS ligation of Seq1 in pSB1A3
</LI><LI> 1655 from glycerol stock for genome prep and competency
</LI><LI> BL21 DE3 for more TSS cells
</LI><LI> Keith's 2x RA and Seq1 streak plates from 8/15 (grew this time!)
</LI></UL></DD><DD><UL><LI> Used LB-Kan broth from 8/11 (made by Abhi)
</LI></UL></DD></DL><UL><LI> PCR amplification of Seq1 in pUC57
</LI></UL><DL><DD><UL><LI> Gel visualized, ~150bp bands in samples 4,8,10
</LI></UL></DD><DD><UL><LI> Still on transilluminator
</LI></UL></DD></DL><UL><LI> Samples labeled and in -20
</LI></UL><UL><LI> PCR log updated
</LI><LI> New PCR of Cas genes
</LI></UL><DL><DD><UL><LI> Going to try Cas3 with 7/24 settings with OLD PRIMERS
</LI><LI> Also, try an ABCDE Nest with 8x MgCl2 (this was good before) and a small temperature gradient
</LI></UL></DD><DD><UL><LI> We can then use the result as a template (and perhaps sequence it for funzies to see if we're getting it)
</LI></UL></DD></DL><UL><LI> ((more details in Madeline's folder, including results, though pic is here too))
</LI></UL><UL><LI> PCR
</LI></UL><DL><DD><UL><LI> Cas3, 7/24 protocol, OLD PRIMERS
</LI></UL></DD><DD><UL><LI> (we have been using the more recent primers…)
</LI><LI> MgCl2 gradient
</LI></UL></DD></DL><UL><LI> ABCDE Nest, 8/13 protocol but with temperature gradient from 70 to 74
</LI></UL><UL><LI> Results: see picture
</LI></UL><DL><DD><UL><LI> Comparable results for Cas3 to the original run…will gel this by itself tomorrow
</LI><LI> Nest does not look very good, especially compared to 8/13 result
</LI></UL></DD><DD><UL><LI> Might just use 8/13 tube as template, try again
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Wednesday.2C_August_17"> Wednesday, August 17 </SPAN></H2><UL><LI> Made tss cells (bl21, mg1655)
</LI><LI> Genome prep: mg1655
</LI><LI> 2 Minipreps of the K12 Leader and the B. Halo Leader.
</LI></UL><H2><SPAN class="mw-headline" id="Thursday.2C_August_18"> Thursday, August 18 </SPAN></H2><UL><LI> Gel of the Cas3 and Nest ABCDE
</LI><LI> Sent RARB and RA 2X for sequencing
</LI></UL><H2><SPAN class="mw-headline" id="Friday.2C_August_19"> Friday, August 19 </SPAN></H2><UL><LI>Synopsis:
</LI></UL><DL><DD><UL><LI> Seq1 PCR's 5 samples are for both SLIC, Poly-X ligation and 1X samples.
</LI><LI> New LB-Amp made. plus a negative control.
</LI><LI> K12 Leader/Seq1 ligations incubated and will be ready for transformation on Saturday 8-20.
</LI><LI> Microcentrifuge tubes, PCR tubes, and Test tubes autoclaved, since nuclease contamination has been present among samples.
</LI><LI> Sterile p10 tips autoclaved and ready to use.
</LI></UL></DD></DL><UL><LI> K12 Leader Sequence dephosphorylated
</LI><LI> Ligations of K12 Leader and 1x Seq1
</LI><LI>Samples:
</LI></UL><DL><DD><UL><LI> L1: K12+Seq1 (no PCR cleanup for Seq1)
</LI><LI> L2: K12+Seq (W/PCR cleanup for seq1 using Quiagen PCR cleanup kit)
</LI><LI> L3: K12+Seq (W/PCR cleanup for seq1 using Quiagen PCR cleanup kit)
</LI><LI> L4: Negative control, phosphatased vector only
</LI></UL></DD></DL><UL><LI> Incubated at 16 degrees for 12 hours.
</LI><LI> Media Prepared, since negative controls of LB-AMP Dan/Abhi 8-15 grew with widltype K12 MG1655. 
</LI></UL><DL><DD><UL><LI> 400mL of LB broth prepared and boiled in microwave using &quot;Frozen Entree&quot;  plus a couple minutes, sample was returned to room temperature then 400ul of 1000x Ampicillin added.
</LI><LI> Negative control ran with widltype K12 MG1655.
</LI></UL></DD></DL><UL><LI> PCR of Seq1:
</LI></UL><DL><DD><UL><LI> 5 reactions of Seq1 PCR off pUC57 prepared and ran. Samples are for both SLIC, Poly-X ligation and 1X samples.
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Saturday.2C_August_20"> Saturday, August 20 </SPAN></H2><UL><LI> Restriction of GFP in 1A3 and RFP in 1K3
</LI></UL><P>Incubated for 2 hours at 37 degrees.
</P><UL><LI> Diagnostic gel ran of GFP and RFP cut. GFP shows slight degradation, with less linearized plasmid than expected. RFP backbone not visible. Seq1-1 PCR product from 8-19 is present at ~200bp. GFP in 1A3 and RFP in 1K3 on the DAN/Abhi tray do not pass.
</LI></UL><DL><DD><UL><LI> Lane 1: Hyperladder 1
</LI><LI> Lane 2: GFP in pSB1A3
</LI><LI> Lane 3: GFP cut with EcorI and PstI
</LI><LI> Lane 4: RFP in pSB1K3
</LI><LI> Lane 5: RFP cut with EcorI and PstI
</LI><LI> Lane 6: K12 Leader 2 (from 8-18)
</LI><LI> Lane 7: K12 Leader 4 (From 8-18)
</LI><LI> Lane 8: Seq1 PCR 1(From 8-19)
</LI></UL></DD></DL><UL><LI> Restrictions of RA and Seq1
</LI></UL><DL><DD><UL><LI> Samples: (Stored at -20 on top shelf &quot;1-5 KSB&quot;)
</LI></UL></DD><DD><UL><LI> 1: Seq1 1 (From 8-19)
</LI><LI> 2: Seq1 2 (From 8-19)
</LI><LI> 3: RA PCR (Dan provided sample)
</LI><LI> 4: RA 0.1 (Dan provided sample)
</LI><LI> 5: RA 8,10 (Dan provided sample)
</LI></UL></DD></DL><UL><LI> Note: Dan's samples where restricted given his protocol overnight at 37 degrees, then samples where accidentally re-restricted by myself; samples where promptly heat inactivated following addition of restriction enzyme.
</LI><LI> Samples 1 through 5 were then cleaned with the Quiagen PCR Cleanup kit spin columns and eluted with 50uL Nuclease free H2O, instead of Elution Buffer (EB) provided by protocol, since EB is known to interfere with downstream enzyme reactions.
</LI><LI> Coupled Restriction Ligation Attempt
</LI></UL><DL><DD><UL><LI> Sample incubated for 1 hour at 25 degrees then 8uL was ran on diagnostic gel. Sample shows nebulization at ~400 bp, this may be nuclease contamination but may not be. It may be a mixture of products forming from multiple ligations. Or it may be an unknown nuance inhibiting mutual restriction and ligation (See image: 8-20 1_2_3_4_5_RE+Lig). Note: the sample may need to be restricted, then column prepared using the Sigma-Aldrich PCR Cleanup Kit (Which appears to more effectively remove small &lt;100nt fragments than Quiagen kit), then attempt coupled restriction ligation. If nebulazation still appears the reaction concept may be flawed.
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Wednesday.2C_August_24"> Wednesday, August 24 </SPAN></H2><UL><LI> Minipreps:
</LI></UL><DL><DD><UL><LI> RARB A1 Kan (1 total)
</LI><LI> GFP Gen 1, GFP Gen 2, RARB LB Kan (3 total)
</LI><LI> All 4 sent to nanodrop
</LI></UL></DD></DL><UL><LI> Gels
</LI></UL><DL><DD><UL><LI> Gel 1: pIDT + Seq1 + Leader (E) [4 lanes], pIDT + Leader [2 Lanes]
</LI></UL></DD><DD><UL><LI> Both samples in the gel look good.
</LI></UL></DD></DL><UL><LI> Gel 2: CasABCDE [3 lanes], Cas3 [4 lanes]
</LI></UL><H2><SPAN class="mw-headline" id="Thursday.2C_August_25"> Thursday, August 25 </SPAN></H2><UL><LI> Listeria received from ATCC.
</LI></UL><DL><DD><UL><LI> Glass vial was opened and bacterial pellet was eluted with 500uL of Brain-Heart Infusion composition broth. The pellet was given 1 minute to dissolve, the 500uL was then placed in 5mL of  Brain-Heart Infusion composition broth (Sample: KSB BHI 8-25 Listeria, Media: BHI KSB 8-24). 200uL was transfered to a BHI agar plate and both the 5mL tube and BHI plate where incubated at 37degrees overnight.
</LI></UL></DD></DL><UL><LI> One streak plate of BL21(DE3) and K12 MG1655 where prepared and incubated at 37 degrees overnight.
</LI><LI> Two liquid cultures where prepared from Jon's source plate, one of BL21(DE3) and the other of K12 MG1655. Samples were incubated over night.
</LI><LI> Tomorrow:
</LI></UL><DL><DD><UL><LI> TSS cells should be prepared.
</LI><LI> BL21 Should be made into glycerol stocks and TSS cells.
</LI><LI> MG1655 Should be made into glycerol stocks and genomic prepped.
</LI><LI> Listeria should be re-plated, transfered to new media, and made into glycerol stocks.
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Friday.2C_August_26"> Friday, August 26 </SPAN></H2><UL><LI> CULTURES:
</LI></UL><DL><DD><UL><LI> Listeria re-streaked, glycerol stocks prepared in both 20% glycerol and 50% glycerol concentrations (Labelled on vials).
</LI></UL></DD><DD><UL><LI> Samples 1-12 20% Glycerol
</LI><LI> Samples 13-16 50% Glycerol
</LI></UL></DD></DL><UL><LI> GENOMIC DNA:
</LI></UL><DL><DD><UL><LI> Genomes of both K12 MG1655 and Listeria innocua Prepared using Sigma-Aldrich genome preparation kit (protocol for gram-negative).
</LI></UL></DD><DD><UL><LI> Samples MG1655 1 and LI 1 where eluted in nuclease free water.
</LI><LI> Sample MG1655 2 and LI 2 where eluted in elution buffer.
</LI></UL></DD></DL><UL><LI> Sample nanodrop results:
</LI></UL><DL><DD><UL><LI> MG1655 1 (H2O): 38.4 ng/ul
</LI><LI> MG1655 2 (Elution Solution): 26.8 ng/ul
</LI><LI> LI 1 (H2O): 77.5 ng/ul
</LI><LI> LI 2 (Elution Solution): 51.3 ng/ul
</LI></UL></DD></DL><UL><LI> Whole Genomes were visualized on a 1% agarose gel, with the goal of visualizing if there is genome fracturing or degradation.
</LI></UL><DL><DD><UL><LI> Lane 1: Hyperladder1
</LI><LI> Lane 2:  MG1655 1 (H2O)
</LI><LI> Lane 3: MG1655 2 (Elution Solution)
</LI><LI> Lane 4: LI 1 (H2O) 
</LI><LI> Lane 5: LI 2 (Elution Solution)
</LI></UL></DD></DL><UL><LI> PRIMERS:
</LI></UL><DL><DD><UL><LI> 16s rRNA Forwar/Revers primers recieved and diluted to 100uM concentrartion
</LI><LI> M13 Forward (-20) and M13 Reverse (-27) primers recieved and diluted to 100uM concentration.
</LI></UL></DD></DL><UL><LI> PCR:
</LI></UL><DL><DD><UL><LI> The aforementioned genome preparations where ran with a positive control PCR using the 16s rRNA primers.
</LI><LI> Using a magnesium gradient set up by Ethan. 
</LI></UL></DD></DL><UL><LI> Results visualized by 1% agarose gel (8-27)
</LI></UL><H2><SPAN class="mw-headline" id="Saturday.2C_August_27"> Saturday, August 27 </SPAN></H2><UL><LI> More Genomic DNA:
</LI></UL><DL><DD><UL><LI> Using sigma-aldrich genomic DNA preparation kit
</LI></UL></DD></DL><UL><LI> NEW -80 DEGREE BOX: 
</LI></UL><DL><DD><UL><LI> &quot;Kylie Glycerol Stocks&quot;
</LI></UL></DD><DD><UL><LI> Currently contains (Source): listeria stocks 8-25/8-26/8-27 (ATCC), BL21DE3 1-16(Misra), DH10B (NEB)
</LI></UL></DD></DL><UL><LI> Transformation of LI_T7LRSR (KanR) and LI_RSR (AmpR) using NEB competent sells and NEB protocol.
</LI></UL><H2><SPAN class="mw-headline" id="Sunday.2C_August_28"> Sunday, August 28 </SPAN></H2><UL><LI> New Listeria Innocua CLIP11262 genome preparations using Sigma-Aldrich Genomic DNA Preparation kit.
</LI></UL><DL><DD><UL><LI> Samples 1-6, 2-28
</LI></UL></DD></DL><UL><LI> Note: All eluted in kit Elution Buffer.
</LI></UL><H2><SPAN class="mw-headline" id="Wednesday.2C_August_31"> Wednesday, August 31 </SPAN></H2><UL><LI> Samples (Primer): Result Summary
</LI></UL><DL><DD><UL><LI> L2-7 (M13 F -20): + match for Seq1 and E0840 spacer present
</LI><LI> L2-7 (M13 R -27): + match for Seq1 and E0840 spacer present
</LI><LI> GFP in pRSF (T7 promoter): No substantial match
</LI><LI> GFP in pRSF (T7 terminator): Several hundred identities for E0840, + match, e value = 0  
</LI><LI> L2-6 (M13 F -20): No seq1
</LI><LI> L2-8 (M13 F -20): No seq1
</LI></UL></DD></DL><UL><LI> Note: more samples were sequenced, available on DNASU. Samples compare to subject &quot;Backbone and Cas Database&quot; using blastn.</LI></UL></DIV></DIV></TD></TR></TBODY></TABLE><CENTER><DIV style="color: #000000; background-color: #EBEBEB"><DIV id="content-footer-box"><DIV class="footer-left-half"><DIV class="footer-left-col"><SMALL><P><B>Contact Us</B></P><P>Arizona State University</P><P>ECG 334, PO BOX 9709</P><P>Tempe, Arizona 85287</P><P><A href="mailto:asu.igem@gmail.com">Email</A></P></SMALL></DIV></DIV></DIV></DIV></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Arizona_State/Notebook/August">http://2011.igem.org/Team:Arizona_State/Notebook/August</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Arizona_State/Notebook/August" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Arizona_State/Notebook/August" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Arizona_State/Notebook/August&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Arizona_State/Notebook/August&amp;oldid=201166" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>