<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cologne-Duesseldorf_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cologne-Duesseldorf/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="sidebar"><UL><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf">Home</A></LI><UL class="sub"><A href="#!">Team</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Collaborations">Collaborations</A></LI></UL><UL class="sub"><A href="#!">Project</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Attributions">Attributions</A></LI></UL><UL class="sub"><A href="#!">Parts</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Basic_Part">Basic Parts</A></LI></UL><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Safety">Safety</A></LI><UL class="sub"><A href="#!">Human Practices</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Engagement">Public Engagement</A></LI></UL><UL class="sub"><A href="#!">Awards</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Plant">Plant</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Software">Software</A></LI></UL><LI><A href="https://igem.org/2017_Judging_Form?team=Cologne-Duesseldorf">Judging Form</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Wiki-Tutorial">Wiki Tutorial</A></LI></UL></DIV><NAV><UL class="menu"><LI class="menu-item-has-children"><A href="#!">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Collaborations">Collaborations</A></LI></UL><LI class="menu-item-has-children"><A href="#!">Project</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Attributions">Attributions</A></LI><LI class="menu-item-has-children"><A href="#!">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Safety">Safety</A></LI><LI class="menu-item-has-children"><A href="#!">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Engagement">Public Engagement</A></LI><LI class="menu-item-has-children"><A href="#!">Awards</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Software">Software</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Cologne-Duesseldorf">Judging Form</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Wiki-Tutorial">Wiki Tutorial</A></LI></NAV><H1>Project description</H1><DIV class="half-width"><SVG xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" viewbox="0 0 1000 1000"><TITLE>artico</TITLE></SVG></DIV><H2>Introduction</H2><H3>Real world application</H3><P>As a proof of concept for our compartmentation strategy we intend to establish the nootkatone pathway inside the peroxisome. Nootkatone is a natural compound found inside the peel of grapefruit, which gives it its characteristic taste and scent. In addition, nootkatone is a natural repellent for mosquitoes and ticks that is already being commercially used and industrially manufactured. Unfortunately, the production costs are extremely high, because it has to either be extracted from the peel of millions of grapefruit or synthesized inside of yeast. The difficulties lie in the toxicity of the nootkatone pathway towards yeast and the resulting low efficiency. Here our compartmentation comes into play: we plan to translocate the whole pathway into the modified peroxisome to prove that we have transformed the peroxisome into an independent compartment with all the features we require.</P><BUTTON class="accordion"><H2 id="ProteinImport">Protein Import</H2><P>The peroxisome possesses two pathways for importing proteins with the main transport proteins being Pex5 and Pex7. We created an orthogonal Pex5 binding pocket and corresponding recognition peptide (PTS1) by structural modeling. We also created a library of Pex7 recognition sequences for import of proteins that are incompatible with the PTS1 peptide.</P></BUTTON><DIV class="panel"><H3>PTS1 Import</H3><P>
The vast majority of peroxisomal matrix proteins is imported by the Pex5 importer. Pex5 recognizes the C-terminal PTS1 peptide whose evolutionary conserved sequence is (S/A/C)-(K/R/H)-(L/M) (<A href="https://www.ncbi.nlm.nih.gov/pubmed/2654139"><ABBR title="
Gould, S. J.; Keller, G. A.; Hosken, N.; Wilkinson, J.; Subramani, S. (1989): A conserved tripeptide sorts proteins to peroxisomes. In: The Journal of cell biology 108 (5), S. 1657–1664.">Gould <I>et al.</I> 1989</ABBR></A>).  Pex5 is a 612 amino acid protein which contains seven tetratrico peptide repeats (TPR). The TPR is a 34 amino acid motif which forms a structure of alpha-helices separated by one turn. A whole TPR domain consists of three of those structures (<A href="https://www.nature.com/nsmb/journal/v7/n12/full/nsb1200_1091.html"><ABBR title=" Gatto, G. J.; Geisbrecht, B. V.; Gould, S. J.; Berg, J. M. (2000): Peroxisomal targeting signal-1 recognition by the TPR domains of human Pex5. In: Nature structural biology 7 (12), S. 1091–1095">Gatto <I>et al.</I> 2000</ABBR></A>).
TPR domains are often involved in protein−protein interactions. As can be seen in the following figure, the TPR regions mediate the binding of the peroxisomal targeting signal.
</P><FIGURE><FIGCAPTION><STRONG>Figure 1.1: </STRONG>TPR domain of the human PEX5, with a pentapeptide in its binding pocket (<A href="https://www.nature.com/nsmb/journal/v7/n12/full/nsb1200_1091.html"><ABBR title=" Gatto, G. J.; Geisbrecht, B. V.; Gould, S. J.; Berg, J. M. (2000): Peroxisomal targeting signal-1 recognition by the TPR domains of human PEX5. In: Nature structural biology 7 (12), S. 1091–1095">Gatto <I>et al.</I> 2000</ABBR></A>)
</FIGCAPTION></FIGURE><P>
The following figure depicts the import mechanism of PTS1 tagged proteins via Pex5.
</P><FIGURE><FIGCAPTION><STRONG>Figure 1.2: </STRONG>Pex5 import mechanism (<A href="https://www.nature.com/articles/nrm1710"><ABBR title=" Erdmann, R.; Schliebs, W. (2005): Peroxisomal matrix protein import: the transient pore model. In: Nature reviews. Molecular cell biology 6 (9), S. 738–742.">Erdmann and Schliebs 2005</ABBR></A>)

Upon recognition of the PTS1 in the cytosol, Pex5 binds to its cargo (i). It docks to the peroxisomal membrane complex, consisting of Pex13, Pex14 and Pex17 (ii). This docking complex is connected to the RING-finger complex, consisting of Pex2, Pex10 and Pex12, via Pex8. This multi-protein complex is known as the importomer. Pex5 and Pex14 form a pore in the membrane, through which the cargo is translocated (iii). Due to competitive binding of Pex8's PTS1 motif, the receptor–cargo complex dissociates at the matrix site of the membrane (iv). The integral PTS1-receptor is either monoubiquitinated by the E2-enzyme Pex4 or polyubiquitinated by Ubc4 or Ubc5. The AAA peroxins Pex1 and Pex6, which are anchored to the peroxisomal membrane by Pex15, dislocate the ubiquitinated Pex5 from the membrane back to the cytosol (v). The polyubiquitinated PTS1-receptors are degraded by the proteasome, whereas the monoubiquitinated receptors are recycled for further rounds of import.
</FIGCAPTION></FIGURE><P>
In this subproject we mutated the Pex5 receptor in a way that enables it to recognize a new signal peptide which does not occur in nature. As Pex5 is responsible for the import of most proteins , we have complete control over the peroxisomal content once we knock out the wild type receptor and replace it with our newly mutated one.
Corresponding to the new receptor, a peroxisomal targeting signal that provides favorable interactions with the residues of the amino acids within the TPR needs to be designed.
      	
Our first approach for the mutation deals with the introduction of site-directed mutagenesis in the TPR of Pex5 followed by computational simulation of the binding affinity between our new designed Pex5 receptor and several peptide variants via <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4655909/">Molecular Dynamics</A>. In the <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">model section </A>we explain the molecular dynamics approach in more detail.

Our second approach relies on recently published literature. We designed a receptor similar to the one designed by <A href="https://www.nature.com/articles/s41467-017-00487-7"><ABBR title=" Cross, Laura L.; Paudyal, Rupesh; Kamisugi, Yasuko; Berry, Alan; Cuming, Andrew C.; Baker, Alison; Warriner, Stuart L. (2017): Towards designer organelles by subverting the peroxisomal import pathway. In: Nature communications 8 (1), S. 454.”&gt;&lt;i&gt;Cross et al. &lt;/i&gt; 2017"> Cross <I> et al. </I></ABBR></A>

in the moss <EM>Physcomitrella patens</EM> in 2017. To understand how and where we set the mutations in the Pex5 receptor in this approach, please proceed with the <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design">design section</A>.          	
</P><H3>PTS2 Import</H3><P>The peroxisomal import depends on two pathways. A vast majority of the proteins normally found in the peroxisome are imported via the <A href="#PTS1">Pex5 importer</A>. 
In <I>S. cerevisiae</I> there is only one protein, the 3-Oxoacyl-CoA thiolase <A href="http://onlinelibrary.wiley.com/doi/10.1002/yea.320100708/full"><ABBR title="Erdmann, R. (1994). The peroxisomal targeting signal of 3‐oxoacyl‐CoA thiolase from Saccharomyces cerevisiae. Yeast, 10(7), 935-944."> (Erdmann 1994)</ABBR></A>, localized in the peroxisome, is imported by the receptor Pex7 and some coreceptors instead <A href="https://www.ncbi.nlm.nih.gov/pubmed/17445803"><ABBR title="Platta, H. W., &amp; Erdmann, R. (2007). The peroxisomal protein import machinery. FEBS letters, 581(15), 2811-2819.
"> (Erdmann<I>et al. </I> 2007)</ABBR></A>.</P><P>The targeting signal for this pathway is localized near the N-terminus of each protein. Kunze and colleagues described the PTS2 consensus sequence (see figure 2.1)</P><FIGURE><FIGCAPTION>Figure 2.1: The peroxisomal targeting signal type two consists of nine amino acids. Residue one contains Arginine or Lysine, residue two Leucine, Valine or Isoleucine. The amino acids three to seven are highly variable. Residue number eight consists of Histidine or Glutamine and the ninth is either Leucine or Alanine. 
<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3247985/ "><ABBR title="2011, Kunze, M., Neuberger, G., Maurer-Stroh, S., Ma, J., Eck, T., Braverman, N., Schmid, J., Eisenhaber, F. &amp; Berger, J. - Structural requirements for interaction of peroxisomal targeting signal 2 and its receptor Pex7."> (Kunze <I>et al</I> 2011)</ABBR></A></FIGCAPTION></FIGURE><P>The five amino acids in the center are not conserved and highly variable. In yeast, among other organisms, the protein Pex7 works as a soluble chaperone, which recognizes PTS2 and directs the protein to the import pore at the peroxisomal membrane 
<A href="https://www.ncbi.nlm.nih.gov/pubmed/17445803 "><ABBR title="Platta, H. W., &amp; Erdmann, R. (2007). The peroxisomal protein import machinery. FEBS letters, 581(15), 2811-2819.
">(Erdmann <I>et al.</I> 2007)</ABBR></A>.</P><P>Towards the aim of implementing a valuable import device for our toolbox we created a library of different PTS2 versions showing variable import efficiencies. Subsequently, one can ensure customizable concentrations of different pathway parts in the peroxisome. Moreover, proteins which require an unmodified C-terminus can be imported via PTS2 since this sequence is located on the N-terminus of the protein (<A href="#PTS1">PTS1 import</A>).</P><P>Kunze <I>et al.</I> performed a mutational analysis for the PTS2 containing human thiolase, specifically for the five variable residues in the core region. The wild type sequence of those residues was defined as glutamine, valine, valine, leucine and glycine. These amino acids were substituted by specific amino acids to be able to evaluate the effect of distinct types in the above stated positions within the sequence. The selected amino acids represent different groups to investigate the biochemical effects of different side chains or other factors: aspartate as a negatively charged, tryptophane as an aromatic, arginine as a basic, leucine as a bulky and lysine as a positively charged amino acid. The thiolase import was subsequently measured with immunofluorescence microscopy. The recognition and import of the PTS2 harboring protein of interest by Pex7 worked out with aspartate at position X1, but not on X2 or X3. Lysine on residue X3 lead to a strong decrease of import activity. Kunze et al. concluded that the import of a given protein relies highly on the amino acid groups in the core region of the PTS2 <A href="https://www.ncbi.nlm.nih.gov/pubmed/22057399 "><ABBR title="2011, Kunze, M., Neuberger, G., Maurer-Stroh, S., Ma, J., Eck, T., Braverman, N., Schmid, J., Eisenhaber, F. &amp; Berger, J. - Structural requirements for interaction of peroxisomal targeting signal 2 and its receptor Pex7.">(Kunze <I> et al.</I> 2011) </ABBR></A>.</P><P>Besides a biased approach, which relies on substitution of single residues in the amino acid sequence of the PTS2, in a second approach we aim to randomly change the sequence to characterize a huge library of different sequence compositions.</P></DIV><BUTTON class="accordion"><H2 id="MembraneIntegration">Membrane Integration</H2><P>To optimize the luminal conditions of our compartment we focused on integrating new proteins into its membrane. This way we can alter specific properties or supply reactions inside with necessary co-factors. To test the import and integration mechanisms, we fused our designed membrane anchors to fluorescent marker proteins and finally integrated the proton pump bacteriorhodopsin into the membrane to acidify our compartment</P></BUTTON><DIV class="panel"><H3>Introduction</H3><P>Many reactions rely on optimal conditions like pH and co-factors. Thus, this subproject aims at the optimization of those circumstances through the integration of new membrane proteins, which alter specific properties of the peroxisomal lumen. Such an approach promises to be very useful for metabolic engineering projects as it can help to adjust the pH, provide cofactors to enzymes, or increase/decrease the concentrations of metabolites inside to peroxisome. In nature two distinct mechanisms exist, which are used for the integration of membrane proteins into the peroxisomal membrane – a Pex19-<A href="http://www.uniprot.org/uniprot/P28795">Pex3</A> dependent and an ER-dependent one  <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1255988"><ABBR title="I.A. Sparkes, C. Hawes, A. Baker, AtPex2 and AtPex10 are targeted to peroxisomes independently of known endoplasmic reticulum trafficking routes, Plant Physiol. 139 (2005) 690–700"> (Sparkes <I>et al.</I> 2005) </ABBR></A>.</P><P>They rely on a so called mPTS sequence, that is used to mark the proteins for transport to and integration into the peroxisomal membrane  <A href="https://www.ncbi.nlm.nih.gov/pubmed/12839494"><ABBR title="H.F. Tabak, J.L. Murk, I. Braakman, H.J. Geuze, Peroxisomes start their life in the endoplasmic reticulum, Traffic 4 (2003) 512–518"> ( Tabak <I>et al.</I> 2003)</ABBR></A>. We will try to take advantage of both mechanisms to incorporate new proteins into the peroxisomal membrane.    
However, to test whether yeast can integrate and use the foreign proteins in its peroxisomal membrane, we will design three different constructs, which will hopefully give us insights into the mechanisms and its efficiency to incorporate new proteins into the peroxisomal membrane.</P><P>As a proof of concept, we will incorporate three proteins through three different approaches into the peroxisomal membrane: (i) mRuby2-<A href="http://www.uniprot.org/uniprot/Q7Z412" style="color:#DB8321">Pex26</A> as proof for the Pex19-dependent mechanism, (ii)  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A>-mRuby2 itself to showcase the ER-dependent mechanism and (iii) <A href="http://www.uniprot.org/uniprot/P02945">bacteriorhodopsin</A>, a unidirectional proton pump, fused to the N-terminal anchor of  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A>. </P><H4>Pex19-dependent Mechanism</H4><P>The exact mechanisms of mPTS binding,  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A>/Pex19 disassembly, mPTS-PMP binding, and release from the  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A>/Pex19 mediated mPTS-PMP docking to the full integration into the membrane are still unknown <A href="https://www.ncbi.nlm.nih.gov/pubmed/20531392"><ABBR title="Schueller, N.; Holton, S. J.; Fodor, K.; Milewski, M.; Konarev, P.; Stanley, Will A. et al. (2010): The peroxisomal receptor Pex19p forms a helical mPTS recognition domain. In: The EMBO journal 29 (15), S. 2491–2500.
"> (Schueller <I>et al.</I> 2010)</ABBR></A>. However, general principles of the integration of a new peroxisomal membrane protein (PMP) through Pex19 and  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A> are studied. Most PMPs feature a membrane targeting signal (mPTS), multiple binding sites for Pex19p, and at least one transmembrane domain (TMD). The mPTS can appear in two different ways, either located in the middle of the primary amino acid sequence, which is the more complex form, or it can be found at the N-terminal part of the PMP as in Pex25. Pex19p is a cytosolic protein, which recognizes the mPTS of the PMP to be incorporated. In the first step Pex19p attaches to the PMP by binding to the mPTS and acts like a chaperone, guiding it to the peroxisome. Next, Pex19p binds N-terminally to the peroxisomal membrane protein  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A>p, which is attached to the peroxisomal membrane through an N-terminal membrane anchor. This will bring the PMP in close proximity to the peroxisomal membrane. Last, Pex19p initiates the membrane integration of the PMP <A href="https://www.ncbi.nlm.nih.gov/pubmed/26777132"><ABBR title=" Liu, Y.; Yagita, Y.; Fujiki, Y. (2016): Assembly of Peroxisomal Membrane Proteins via the Direct Pex19p-Pex3p Pathway. In: Traffic (Copenhagen, Denmark) 17 (4), S. 433–455.">(Liu <I>et al.</I> 2016)</ABBR></A>
. 



</P></DIV><BUTTON class="accordion"><H2 id="Secretion">Secretion</H2><P>Downstream processing is not only time consuming but also cost and energy intensive. Therefore, we aim to simplify the purification of compounds produced in our artificial compartment. We used a concept based on the peroxicretion described by Sagt and colleagues <A href=" https://www.ncbi.nlm.nih.gov/pubmed/19457257"><ABBR title=" Sagt, C. M., ten Haaft, P. J., Minneboo, I. M., Hartog, M. P., Damveld, R. A., van der Laan, J. M., ... &amp; van der Klei, I. (2009). Peroxicretion: a novel secretion pathway in the eukaryotic cell. BMC biotechnology, 9(1), 48.">
      (Sagt <I> et al</I>, 2009)
  </ABBR></A>. </P></BUTTON><DIV class="panel"><H3>Introduction </H3><P>Downstream processing is a very important part of industrial biological compound production. For most biotechnologically produced compounds, it is the most expensive part of the production <A href=" https://www.ncbi.nlm.nih.gov/pubmed/11602307 "><ABBR title=" Keller, K., Friedmann, T., &amp; Boxman, A. (2001). The bioseparation needs for tomorrow. TRENDS in Biotechnology, 19(11), 438-441.">
      (Keller <I> et al</I>, 2001)
  </ABBR></A>
. One step to decrease the costs is to secrete the products into the supernatant <A href=" https://www.ncbi.nlm.nih.gov/pubmed/23385853"><ABBR title=" Berlec, A., &amp; Štrukelj, B. (2013). Current state and recent advances in biopharmaceutical production in Escherichia coli, yeasts and mammalian cells. Journal of industrial microbiology &amp; biotechnology, 40(3-4), 257-274.
">
      (Berlec <I> et al</I>, 2013)
  </ABBR></A>. After secretion, it is possible to remove most cellular compounds from valuable products with one simple centrifugation step. Due to this, secretion is not only a great tool for a compartment toolbox but is of great economic value as well. 
In regards to the whole project, this is an important part of making the compartment more applicable by enabling immediate extraction of metabolites after production. 
At the end of this sub project it should be possible to secrete every compound produced in the modified compartment to the supernatant. This is not trivial because peroxisomes, which provide the basis for our compartment do not possess a known natural secretion mechanism. 
We overcome this problem by using the &quot;peroxicretion&quot; concept of Sagt and colleagues <A href=" https://www.ncbi.nlm.nih.gov/pubmed/19457257"><ABBR title=" Sagt, C. M., ten Haaft, P. J., Minneboo, I. M., Hartog, M. P., Damveld, R. A., van der Laan, J. M., ... &amp; van der Klei, I. (2009). Peroxicretion: a novel secretion pathway in the eukaryotic cell. BMC biotechnology, 9(1), 48.">
      (Sagt <I> et al</I>, 2009)
  </ABBR></A>. They used a v-SNARE (<B>v</B>esicle- synaptosome-associated-<B>S</B>oluble <B>N</B>-ethylmaleimide-sensitive-factor <B>A</B>ttachment <B>RE</B>ceptorprotein) fused to a peroxisomal membrane-protein to secrete the content of peroxisomes. V-SNAREs interact with the t-SNARE (<B>t</B>arget synaptosome-associated-SNARE) at the cell membrane, which leads to an fusion of the vesicle with the membrane <A href=" https://www.nature.com/articles/35052017"><ABBR title=" Chen, Y. A., &amp; Scheller, R. H. (2001). SNARE-mediated membrane fusion. Nature reviews Molecular cell biology, 2(2), 98-106.">
      (Chen <I> et al</I>, 2001)
  </ABBR></A>.
</P></DIV><BUTTON class="accordion"><H2 id="SizeAndNumber">Size and Number</H2><P>
Peroxisomes are eukaryotic organelles that are involved in a vast variety of metabolic processes, most notably the metabolism of lipids as well as various oxidative reactions <A href=" https://www.nature.com/articles/nrm3700 "><ABBR title=" Smith, Jennifer J.; Aitchison, John D. (2013): Peroxisomes take shape. In: Nature reviews. Molecular cell biology 14 (12), S. 803–817.">(Smith and Aitchison 2013</ABBR></A>,<A href=" https://www.ncbi.nlm.nih.gov/pubmed/21441307 "><ABBR title=" Tower, Robert J.; Fagarasanu, Andrei; Aitchison, John D.; Rachubinski, Richard A. (2011): The peroxin Pex34p functions with the Pex11 family of peroxisomal divisional proteins to regulate the peroxisome population in yeast. In: Molecular biology of the cell 22 (10), S. 1727–1738."> Tower <I>et al.</I> 2011)</ABBR></A>
. They are remarkably versatile when it comes to adapting to both, intra- and extracellular cues and changes, responding to environmental variation by changing their size, shape and content accordingly. These exceptional abilities of peroxisomes do not only provide cells with a microenvironment for metabolic reactions but also enable them to respond to metabolic or environmental stress as well as cope with needs of cell division <A href=" https://www.ncbi.nlm.nih.gov/pubmed/21441307 "><ABBR title=" Tower, Robert J.; Fagarasanu, Andrei; Aitchison, John D.; Rachubinski, Richard A. (2011): The peroxin Pex34p functions with the Pex11 family of peroxisomal divisional proteins to regulate the peroxisome population in yeast. In: Molecular biology of the cell 22 (10), S. 1727–1738."> (Tower <I>et al.</I> 2011)</ABBR></A>. Controlling the size and shape of peroxisomes is therefore not only crucial to understanding peroxisome dynamics and utility but also provides a promising opportunity to influence bioengineering pathways within cells. Many studies that have been conducted in order to investigate the complexity of the shape, size and number of peroxisomes have led to the recognition of several Pex genes involved in peroxisome biogenesis and proliferation <A href=" http://europepmc.org/abstract/med/15978793"><ABBR title=" Yan, Mingda; Rayapuram, Naganand; Subramani, Suresh (2005): The control of peroxisome number and size during division and proliferation. In: Current opinion in cell biology 17 (4), S. 376–383."> (Yan <I>et al.</I> 2005)</ABBR></A>. In order to control the size and number of peroxisomes as part of our toolbox, we further investigated the Pex11, Pex31,32, and Pex34 genes.
</P></BUTTON><DIV class="panel"><H3>Introduction</H3><H4>Peroxisome Biogenesis and Proliferation</H4><P> 	 	 	
Peroxisomes can be generated in different ways and their size and abundance is controlled by a number of pathways <A href=" http://europepmc.org/abstract/med/15978793"><ABBR title=" Yan, Mingda; Rayapuram, Naganand; Subramani, Suresh (2005): The control of peroxisome number and size during division and proliferation. In: Current opinion in cell biology 17 (4), S. 376–383."> (Yan <I>et al.</I> 2005)</ABBR></A>. In yeast, peroxisomes can be generated de novo by budding from the endoplasmatic reticulum (ER) or through division from pre-existing peroxisomes using new proteins and lipids supplied from the ER in the form of vesicles <A href=" https://www.nature.com/articles/nrm3700 "><ABBR title=" Smith, Jennifer J.; Aitchison, John D. (2013): Peroxisomes take shape. In: Nature reviews. Molecular cell biology 14 (12), S. 803–817.">(Smith and Aitchison 2013)</ABBR></A>. Both pathways are still being investigated and to date haven’t been fully understood.
</P><FIGURE><FIGCAPTION>Fig.1: Peroxisomes can form through two pathways Nat Rev Mol Cell Biol. 2013 Dec; 14(12): 803–817.</FIGCAPTION></FIGURE><P>	 	 	 	
Peroxisomes are extremely sensitive to environmental cues and are able to proliferate or be degraded accordingly <A href=" http://europepmc.org/abstract/med/15978793"><ABBR title=" Yan, Mingda; Rayapuram, Naganand; Subramani, Suresh (2005): The control of peroxisome number and size during division and proliferation. In: Current opinion in cell biology 17 (4), S. 376–383."> (Yan <I>et al.</I> 2005)</ABBR></A>. Depending on the growth medium and their extracellular environment, peroxisomes are able to divide and multiply separately from cell division <A href=" http://europepmc.org/abstract/med/15978793"><ABBR title=" Yan, Mingda; Rayapuram, Naganand; Subramani, Suresh (2005): The control of peroxisome number and size during division and proliferation. In: Current opinion in cell biology 17 (4), S. 376–383."> (Yan <I>et al.</I> 2005)</ABBR></A>. Their size and number is directly influenced by the presence of e.g. fatty acids, which lead to an increase in both size and number. Furthermore, peroxisome population is regulated by different peroxisomal integral membrane proteins, so called peroxins <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I> 2016)</ABBR></A>.
.
</P><H4>Peroxins</H4><P>
The formation of peroxisomes, both by de novo generation as well as growth and fission, is a highly controlled mechanism. Multiple studies have shown that the growth and division of peroxisomes are regulated by protein families specific to peroxisomes, so called peroxins <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I> 2016)</ABBR></A>. Since <I>S. cerevisiae</I> naturally contains a very small amount of peroxisomes when growing under glucose-rich conditions, biosynthesis and target yield can be increased by altering peroxisome size and number. A short introduction to the peroxins used in this part of the project is given hereafter.
</P><H5>Pex11</H5><P>
Due to its unique ability to promote peroxisome division and its role in peroxisome biogenesis <A href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2174077/ "><ABBR title=" Li, Xiaoling; Gould, Stephen J. (2002): PEX11 promotes peroxisome division independently of peroxisome metabolism. In: The Journal of cell biology 156 (4), S. 643–651."> (Li and Gould 2002)</ABBR></A>
, the first peroxin we chose for our purpose is Pex11, which is located in the inner surface of the peroxisomal membrane <A href=" https://www.ncbi.nlm.nih.gov/pubmed/11431484 "><ABBR title=" Gurvitz, A.; Hiltunen, J. K.; Erdmann, R.; Hamilton, B.; Hartig, A.; Ruis, H.; Rottensteiner, H. (2001): Saccharomyces cerevisiae Adr1p governs fatty acid beta-oxidation and peroxisome proliferation by regulating POX1 and PEX11. In: The Journal of biological chemistry 276 (34), S. 31825–31830 "> (Gurvitz <I>et al.</I> 2001)</ABBR></A>. Erdmann and Blobel have shown that the deletion of the Pex11 gene in S. cerevisiae results in cells with fewer, larger peroxisomes, whereas overexpression results in cells with a higher quantity of smaller peroxisomes <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I> 2016)</ABBR></A>. Studies conducted by Smith and Aitchison confirmed that Pex11p-deficient cells growing on fatty acids failed to increase the amount of peroxisomes and instead the accumulation of a few giant peroxisomes was observed <A href=" https://www.nature.com/articles/nrm3700 "><ABBR title=" Smith, Jennifer J.; Aitchison, John D. (2013): Peroxisomes take shape. In: Nature reviews. Molecular cell biology 14 (12), S. 803–817.">(Smith and Aitchison 2013)</ABBR></A>. Other media, like oleate-containing ones, cause an induction of peroxisomal proliferation, which is due to an oleate responsive element of the Pex11 promoter <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I> 2016)</ABBR></A>.


In order to find out more about the complexity of peroxisome biogenesis and proliferation and also get constructive feedback on our work, we consulted with Florian David from Biopetrolia in Sweden. Their company specializes in yeast engineering in order to improve production titers, yields and rates for the production of biofuels, pharmaceuticals and other products. He suggested to expand our project to working not only with Pex11, but Pex31,32 and Pex34 as well. 
</P><H5>Pex30-32</H5><P>
According to Zhou <I>et al.</I>, the genes of the Pex30 – 32 family have been shown to influence peroxisome proliferation <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I> 2016)</ABBR></A>. Their deletion resulted in the production of a higher quantity of large peroxisomes. Zhou et al. further investigated the effect of a Pex31,32 knockout, showing both number and size increase, also leading to a higher metabolic yield. However, a Pex31,32 knockout has been proven to attribute to a change in the membrane structure, resulting in higher permeability of the peroxisome membrane for fatty aldehydes and other intermediates and byproducts <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377.">(Zhou <I>et al.</I> 2016)</ABBR></A>. Due to these side effects we decided to discard working with a Pex31,32 knockout for now. 

</P><H5>Pex34</H5><P>
Similar to Pex11, Pex34p is another peroxisomal integral membrane protein that can act both, independently and in combination with Pex11p, Pex25p, and Pex27p to control the peroxisome morphology and population. Pex34p is suggested to directly influence peroxisome proliferation as well as constitutive peroxisome division. Specifically, Pex34p overexpression positively affects peroxisome numbers in wild type and pex34 cells, whereas Pex34 deletion results in cells with fewer peroxisomes <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I> 2016,</ABBR></A><A href=" https://www.ncbi.nlm.nih.gov/pubmed/21441307 "><ABBR title=" Tower, Robert J.; Fagarasanu, Andrei; Aitchison, John D.; Rachubinski, Richard A. (2011): The peroxin Pex34p functions with the Pex11 family of peroxisomal divisional proteins to regulate the peroxisome population in yeast. In: Molecular biology of the cell 22 (10), S. 1727–1738."> Tower <I>et al.</I> 2011)</ABBR></A>
. In their studies Zhou et al. targeted synthetic pathways to peroxisomes in order to increase the production of fatty-acid-derived fatty alcohols, alkanes and olefins. By harnessing peroxisomes to produce fatty-acid-derived chemicals and biofuels they were able to show that peroxisome increases the production of target molecules while decreasing byproduct formation. Additionally, analyzing the effect of peroxin knockouts and overexpression, their research revealed that Pex34 overexpression significantly increased their yield <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I> 2016)</ABBR></A>.  The main advantage of working with Pex34p over Pex31,32 is the effect on the peroxisomal membrane. While Pex31,32 significantly increases membrane permeability, Pex34 has less effects on the membrane structure <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I> 2016)</ABBR></A>. 
</P><FIGURE><FIGCAPTION>Fig. 2: Regulation of peroxisome quantity and morphology by different peroxins
</FIGCAPTION></FIGURE><H4>Our Project</H4><P>
One step towards achieving the creation of a fully controllable artificial compartment is the regulation of and control over its morphology. In our case we are aiming at achieving the exact regulation of the size and number of the peroxisome. As a first approach we have chosen to control the Pex11 concentration in the cell. Furthermore Pex11 is to be designed as a 3b toolbox part so it can be combined and its effects tested with different promoters. For that purpose we are working with two constitutive promoters of varying strength as well as two inducible promoters which increase gene expression when grown in varying concentrations of galactose or copper sulfate. To control the range from a few giant peroxisomes to a high quantity of small ones we are working in a pex11D knockout strain. Secondly, following the advice of Florian David from Biopetrolia, we intend to increase both, the size and quantity of peroxisomes in the cell via a Pex34 overexpression. By working with Pex34 we will not only be able to control the peroxisome morphology, but also positively influence production yields.      
</P><H4>How does it integrate into the overall project?</H4><P>
Controlling the size and number of peroxisomes is one of the multiple functions we plan to integrate into our artificial compartment toolbox so that it can be utilized for various projects. However, even though the exact control of proliferation can help understand the complex matter of peroxisome dynamics, the advantages of these findings exceed mere foundational research.
Integrating synthetic pathways into cells is often impeded by competing pathways and accruing intermediates or undesired byproducts that negatively influence biosynthesis. In order to achieve feasible results from microbial production, respective pathways need to be isolated into a suitable environment. Compartmentation provides microenvironments for metabolic functions of cells shielding them from the interference of simultaneously occurring reactions and therefore favoring biosynthesis. Going one step further, establishing synthetic pathways into a fully controlled compartment has the potential to increase the efficiency and productivity of these pathways resulting in higher yields of target products  <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I> 2016</ABBR></A>,<A href=" https://www.ncbi.nlm.nih.gov/pubmed/21441307 "><ABBR title=" Tower, Robert J.; Fagarasanu, Andrei; Aitchison, John D.; Rachubinski, Richard A. (2011): The peroxin Pex34p functions with the Pex11 family of peroxisomal divisional proteins to regulate the peroxisome population in yeast. In: Molecular biology of the cell 22 (10), S. 1727–1738."> Tower <I>et al.</I> 2011)</ABBR></A>
. In our case, we change the peroxisome’s morphology by knocking out or overexpressing Pex11 and Pex34 to obtain either a large amount of smaller peroxisomes or a high amount of enlarged ones. Especially the overexpression of Pex34 which results in a high quantity of large peroxisomes has been shown to actively regulate metabolic processes <A href=" https://www.nature.com/articles/nrm3700 "><ABBR title=" Smith, Jennifer J.; Aitchison, John D. (2013): Peroxisomes take shape. In: Nature reviews. Molecular cell biology 14 (12), S. 803–817.">(Smith and Aitchison 2013)</ABBR></A> and increase the production of target molecules while decreasing byproduct formation <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title=" Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I> 2016)</ABBR></A>. Furthermore, evidence indicates that changing the morphology of a compartment, including both, its shape and size, influences the amount of chemical reactions embedded in that compartment <A href=" https://www.nature.com/articles/nrm3700 "><ABBR title=" Smith, Jennifer J.; Aitchison, John D. (2013): Peroxisomes take shape. In: Nature reviews. Molecular cell biology 14 (12), S. 803–817.">(Smith and Aitchison 2013)</ABBR></A>, a trait that can be used to increase the yield of otherwise inefficient reactions.  Ultimately, even though we decided to discard our work on a Pex31,32 knockout, the effects this knockout has on membrane permeability and structure could potentially be used for further pathways within the peroxisome. </P><H4>Overall goal of this subproject</H4><P>
In our subproject we want to achieve full control over peroxin concentrations in the yeast cell, in order to establish a simple method to regulate the peroxisome morphology and quantity.</P></DIV><BUTTON class="accordion"><H2 id="Sensors"><I>In Vivo</I> Sensors</H2><P>Designing pathways from scratch or transferring pathways into cellular compartments requires a sound understanding of the prevailing state, like existing cofactors in the peroxisomes. 
We aimed at measuring the peroxisomal pH, cofactors such as NADP<SUP>+</SUP> and ATP in wild type yeast and our designed mutants over different time periods as well as in response to changing physiological conditions. To do so, we used ratiometric fluorescent biosensors which we genetically attached to a peroxisomal targeting signal.
These measurements provide important insights into possible issues which may occur if non-peroxisomal pathways are transferred into the peroxisome and furthermore enable more precise predictions and modelling.</P></BUTTON><DIV class="panel"><H4> pH Sensor </H4><P>The activity of enzymatic proteins is mostly pH-dependent. Therefore, it is of high interest to understand the pH-regulating mechanism of the peroxisome and the effects on the imported pathways. Literature has neither agreed whether there is a common peroxisomal pH nor whether there is a regulating mechanism. For our measurements, we use pH Lourin2, a GFP variant with a bimodal excitation spectrum with peaks at 395 and 475 nm and an emission maximum at 509 nm. Upon acidification, the excitation spectrum shifts from 395 to 475 nm <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3152828/"><ABBR title="2011, Mahon et al.- pHluorin2: an enhanced, ratiometric, pH-sensitive green florescent protein"> Mahon <I>et al.</I> (2011)</ABBR>.</A></P><FIGURE><FIGCAPTION><FONT size="3"><STRONG>Figure5.1</STRONG>
pHLuorin2 emission at 509 nm, excited at wavelengths between 350 nm and 500 nm . Five different pH values, ranging from 5.8 to 7.8 are shown <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3152828/"><ABBR title="2011, Mahon et al.- pHluorin2: an enhanced, ratiometric, pH-sensitive green florescent protein"><FONT size="3"> Mahon <I>et al.</I> (2011)</FONT></ABBR></A>.</FONT></FIGCAPTION></FIGURE><H4> roGFP2 Sensor </H4><P>To maintain thermodynamic driving forces and electron fluxes which are needed at steady state, the intact chemeostasis of the redox machinery is of great importance<A href="https://www.ncbi.nlm.nih.gov/pubmed/25867539"><ABBR title="2016, Schwarzländer, et al. - Dissecting Redox Biology Using Fluorescent Protein Sensors"> (2016, Schwarzländer)</ABBR></A>. Glutathione is considered to be inside the peroxisomal lumen <A href="https://www.ncbi.nlm.nih.gov/pubmed/25867539"><ABBR title="2014, Elbaz-Alon, Y., et al. -The Yeast Oligopeptide Transporter Opt2 Is Localized to Peroxisomes and Affects Glutathione Redox Homeostasis">(Elbaz-Alon, Y., et al. 2014)</ABBR></A>. We therefore wanted to monitor glutathione redox potentials inside the peroxisomal lumen using the GFP variant roGFP2, which is able to precisely detect redox changes of glutathione. Two cysteines in the beta barrel structure can either form two thiols or one disulfide bondage, dependent on whether they are reduced or oxidized. This influences the proton transfer of the chromophore and ultimately leads to a ratiometric shift in excitation. Excitation at 485 nm of the reduced roGFP2 exceeds the excitation of oxidized roGFP2 at 485 whereas excitation at 405 nm  of oxidized roGFP2 exceeds excitation of reduced roGFP2<A href="https://link.springer.com/article/10.1007/s12268-016-0683-2"><ABBR title="Morgan, B. and M. Schwarzländer 2016 et al.- The Yeast Oligopeptide Transporter Opt2 Is Localized to Peroxisomes and Affects Glutathione Redox Homeostasis">(Morgan, B. and M. Schwarzländer 2016)</ABBR></A>. </P></DIV><BUTTON class="accordion"><H2 id="Nootkatone">Nootkatone</H2><P>As a proof of concept for our compartment toolbox we decided to shift the metabolic pathway of nootkatone into the peroxisome. With this transfer we want to overcome the obstacle of intermediate toxicity for the yeast cell. A working metabolism will pave the way for an efficient, safe and feasible solution for producing and providing an effective insect repellent. </P></BUTTON><DIV class="panel"><P>Nootkatone is an oxidized sesquiterpene, which is highly valuable for industrial and pharmaceutical applications. We will focus on its repellent effect towards insects
<A href="https://www.ncbi.nlm.nih.gov/pubmed/11441443"><ABBR title="2001, Zhu et al. - Nootkatone is a repellent for Formosan subterranean termite (Coptotermes formosanus)">
    	Zhu <I>et al.</I> (2001)</ABBR></A>.
 
Also, therapeutic activities of nootkatone have been reported, such as anti-platelet effects in rats
<A href="https://www.ncbi.nlm.nih.gov/pubmed/21354294"><ABBR title="2011, Seo et al. - Antiplatelet effects of Cyperus rotundus and its component (+)-Nootkatone">
    	Seo <I>et al.</I> (2011)</ABBR></A>,
 anti-proliferative activity towards cancer cell lines
<A href="https://www.ncbi.nlm.nih.gov/pubmed/21377882"><ABBR title="2011, Gliszczyńska et al. - Microbial Transformation of (+)-Nootkatone and the Antiproliferative Activity of Its Metabolites">
   	Gliszczyńska <I>et al.</I> (2011)</ABBR></A>
 and enhancement of energy metabolism through AMP-activated protein kinase activation in skeletal muscle and liver
<A href="https://www.ncbi.nlm.nih.gov/pubmed/24624065"><ABBR title="2010, Murase et al. - Habituation of the responsiveness of mesolimbic and mesocortical dopamine transmission to taste stimuli
">
   	Murase <I>et al.</I> (2010)</ABBR></A>.</P><P>Nootkatone can be extracted from grapefruit, but the organic material is limited and the yield is very low. So far, industrial production of nootkatone requires toxic substances such as heavy metals and strong oxidants like tert-butyl hydroperoxide which is known to be carcinogenic
<A href="https://www.ncbi.nlm.nih.gov/pubmed/21115006"><ABBR title="2010, Cankar et al. -  A chicory cytochrome P450 mono-oxygenase CYP71AV8 for the oxidation of (+)-valencene
">
   	Cankar <I>et al.</I> (2010)</ABBR></A>.</P><FIGURE><FIGCAPTION><STRONG>Figure 7.1:</STRONG> Conversion of valencene  to Nootkatol and Nootkatone </FIGCAPTION></FIGURE><P>The synthesis of nootkatone starts from the precursor farnesyl pyrophosphate (FPP) and requires at least two enzymes. The initial step is the formation of valencene from FPP by a valencene synthase (ValS) followed by the production of nootkatol, nootkatone and other by-products by a P450 BM3 monooxygenase (BM3). The co-expression of an alcohol dehydrogenase (ADH) with ValS improves nootkatone production by favoring the conversion from nootkatol into nootkatone
 <A href="http://onlinelibrary.wiley.com/doi/10.1002/cctc.201402952/full"><ABBR title="2015, Schulz et al. - Selective Enzymatic Synthesis of the Grapefruit Flavor (+)-Nootkatone">
   	Schulz <I>et al.</I> (2015)</ABBR></A>.</P><P>Previous approaches of nootkatone synthesis in yeast often failed due to toxic intermediates. A specific problem is the toxicity of beta-nootkatol and nootkatone itself for <I>Saccharomyces cerevisiae</I> at concentrations higher than 100 mg/L
<A href="https://www.ncbi.nlm.nih.gov/pubmed/23518241"><ABBR title="2013, Gavira et al. - Challenges and pitfalls of P450-dependent (þ)-valencene bioconversion by Saccharomyces cerevisiae">Gavira <I>et al.</I> (2013)</ABBR></A>.
For an efficient industrial production, concentrations need to be in the range of g/L, which is lethal for yeast cells. Beta-nootkatol seems to accumulate in membranes because of its hydrophobic characteristics, resulting in changes of the membrane permeability, integrity and the function of membrane proteins
<A href="https://www.ncbi.nlm.nih.gov/pubmed/23518241"><ABBR title="2013, Gavira et al. - Challenges and pitfalls of P450-dependent (þ)-valencene bioconversion by Saccharomyces cerevisiae">
   	Gavira <I>et al.</I> (2013)</ABBR></A>.
 
It is presumed that the toxicity is partly caused by this effect. As one of the original purposes of the peroxisome is to reduce hydrogen peroxide, which is harmful to the cell and also alters the membrane composition
<A href="https:/https://www.ncbi.nlm.nih.gov/books/NBK9930/"><ABBR title="2000, Cooper et al. - The Cell: A Molecular Approach. 2nd edition">
   	Cooper <I>et al.</I> (2000)</ABBR></A>,
<A href="https://www.ncbi.nlm.nih.gov/pubmed/1902481"><ABBR title="1991, Block et al. - Hydrogen peroxide alters the physical state and function of the plasma membrane of pulmonary artery endothelial cells">
   	Block <I>et al.</I> (1991)</ABBR></A>,
 we assume that beta-nootkatol does not affect the peroxisomal membrane either. However, in order to validate this hypothesis, we have to collect and evaluate our own data on how beta-nootkatol affects the peroxisome membrane and thus the yield of nootkatone.</P><DIV class="half-width"><FIGURE><FIGCAPTION><STRONG>Figure 7.2</STRONG> Yeast viability after 24 h in the presence of (+)-valencene, beta-Nootkatol or nootkatone in different concentrations <A href="https://www.ncbi.nlm.nih.gov/pubmed/23518241"><ABBR title="2013, Gavira et al. - Challenges and pitfalls of P450-dependent (þ)-valencene bioconversion by Saccharomyces cerevisiae">
   	Gavira <I>et al.</I> (2013)</ABBR></A></FIGCAPTION></FIGURE></DIV><P>Our goal is the successful integration of the nootkatone pathway into our compartment in order to overcome the problem of high concentration toxicity of beta-nootkatol and nootkatone for the yeast cell. This would not only be a more efficient but also a more environmentally friendly method to satisfy the great industrial demand of this sesquiterpene. It would also facilitate the access to a high performing insect repellent in less developed regions of the world and therefore decrease the spread of diseases like malaria, dengue or the Zika virus.</P></DIV><BUTTON class="accordion"><H2 id="Violacein">Violacein</H2><P> Using the tools of synthetic biology in metabolic engineering unleashes the full potential of biofactories. Natural systems use compartmentation to improve biochemical reactions. Here we present the use of peroxisomal import tags to engineer an artificial compartment in <I>Saccharomyces cerevisiae</I> to be further used in metabolic engineering approaches. As an application and proof of concept we are using the well studied biosynthetic pathway of violacein. By designing an import library for the different enzymes we aim to understand basic design principles that can guide future design of compartmentation for metabolic engineering. We chose violacein, not only because of its wide range of biological benefits but also as a solid foundation to proof our sophisticated import machinery. </P></BUTTON><DIV class="panel"><FIGURE class="floatleft"><FIGCAPTION><STRONG>Figure 8.1</STRONG> Structural formula of violacein.
</FIGCAPTION></FIGURE><P>
Violacein (C<SUB>20</SUB>H<SUB>13</SUB>N<SUB>3</SUB>O<SUB>3</SUB>), a bisindole, is a violet pigment, formed by condensation of two tryptophan molecules. It can naturally be found in numerous bacterial strains, for example in the gram-negative <I> Chromobacterium violaceum</I>. Due to its wide range of biological properties, violacein is useful for various industrial applications in pharmaceuticals and cosmetics.

Violacein is known to have a variety of different biological activities, including antitumor
<A href="https://www.ncbi.nlm.nih.gov/pubmed/20416285"><ABBR title="Growth inhibition and pro-apoptotic activity of violacein in Ehrlich ascites tumor">
(Bromberg N<I> et al</I>, 2010)</ABBR></A>, 
antifungal
<A href="https://www.ncbi.nlm.nih.gov/pubmed/18949519"><ABBR title="Amphibian chemical defense: antifungal metabolites of the microsymbiont Janthinobacterium lividum on the salamander Plethodon cinereus">
(Brucker RM <I>et al.</I>, 2008)</ABBR></A>
 and antiviral
<A href="https://www.ncbi.nlm.nih.gov/pubmed/14595466"><ABBR title="Cytotoxicity and potential antiviral evaluation of violacein produced by Chromobacterium violaceum">
(Andrighetti-Fröhner CR <I> et al.</I>, 2003)</ABBR></A>
 effects. Furthermore, it has been shown that violacein enhances the effect of most commercial antibiotics by working synergistically with them
<A href="https://www.ncbi.nlm.nih.gov/pubmed/24073823"><ABBR title="Synergistic antimicrobial profiling of violacein with commercial antibiotics against pathogenic micro-organisms">
(Subramaniam S <I> et al.</I>, 2014)</ABBR></A>. 
This is especially of high interest in the fight against recent antibiotic-resistant strains of pathogenic bacteria such as MRSA (multi resistant <I>Staphylococcus aureus</I>). Violacein’s antibacterial action against <I>S. aureus</I> has been proven by
<A href="https://www.ncbi.nlm.nih.gov/pubmed/21364597"><ABBR title="Antibacterial activity of violacein against Staphylococcus aureus isolated from bovine mastitis">
Cazoto LL <I>et al.</I> (2011)
</ABBR></A>.
It is of high medical interest that toxic effects of Violacein on cultured cancer cells were shown within <I>in vitro</I> tests. Furthermore, the Ehrlich ascites tumor (EAT) mouse model provides prove as an <I>in vivo</I> test: daily injection of violacein ($0.1\,\mu g/kg$ up to $1\,mg/kg$) led to a significant increased survival rate of the mice
<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4538413/"><ABBR title="Violacein: Properties and Production of a Versatile Bacterial Pigment">
(Seong Yeol Choi <I>et al.</I>, 2015)
</ABBR></A>. The ability to weaken cancer growth draws more attention to violacein as a potential cancer therapeutic.  
<A href="https://www.ncbi.nlm.nih.gov/pubmed/16889929"><ABBR title="Cytotoxic activity of violacein in human colon cancer cells">
de Carvalho DD <I>et al.</I> (2006)
</ABBR></A>showed that violacein is capable to induce apoptosis in various cancer cells by inducing the production of oxygen radicals.
 A main focus also lies in violacein’s antimalarial activity, which was tested <I>in vitro</I>  and <I>in vivo</I> on human and murine blood stage forms of <I>Plasmodium</I> parasites
<A href="http://aac.asm.org/content/53/5/2149.full"><ABBR title="Violacein Extracted from Chromobacterium violaceum Inhibits Plasmodium Growth In Vitro and In Vivo">
(Stefanie C. P. Lopes <I> et al.</I>, 2009)
</ABBR></A>. <I>P. falciparum</I> is known to be the deadliest <I>Plasmodium</I> species that causes malaria in humans
<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2720412/"><ABBR title="The origin of malignant malaria">
(Stephen M. Rich <I>et al.</I>, 2009)
</ABBR></A>. Violacein acted effectively against diseases caused by both, young and mature parasite strains, of <I>P. falciparum</I>, and parasite growth was reduced significantly compared to non-treated animals. Moreover, it has a protective effect as mice infected with a lethal strain (<I>P. chabaudi chabaudi</I>) died within 10 days, whereas the majority (80 %) treated with violacein survived the infection
<A href="http://aac.asm.org/content/53/5/2149.full"><ABBR title="Violacein Extracted from Chromobacterium violaceum Inhibits Plasmodium Growth In Vitro and In Vivo">
(Stefanie C. P. Lopes <I> et al.</I>, 2009)
</ABBR></A>. Not at least because the emerge resistance to plant-based malaria drugs becomes more frequent, it is time to look out for further possibilities in the worldwide battle against malaria <A href="https://www.ncbi.nlm.nih.gov/pubmed/26911755"><ABBR title="Synthetic biology's first malaria drug meets market resistance">
(Peplow M, 2016)
</ABBR></A>.
</P><P> As the commercial production of violacein is rather difficult and limited for low productivity
<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4997675/"><ABBR title="Engineering Corynebacterium glutamicum for violacein hyper production">
(Hongnian Sun <I>et al.</I>, 2016)
</ABBR></A>, researchers are working on improving the fermentative titers by metabolic engineering.
Here we want to make use of the existing potential violacein has and even try to promote it. With the great advantages a peroxisomal import has to offer, we want to develop a solid mechanism to not only prove the concept of our project, but also take advantage of violacein’s biological opportunities. By relocalization of the violacein pathway into yeast peroxisomes we want to create a microenvironment with optimized working conditions for the production of violacein to achieve a high yield of the bisindole.
</P></DIV><BUTTON class="accordion"><H2 id="optogenetics">Optogenetics</H2><P>This subproject was about realizing various optogenetic approaches to our compartment toolbox, for example controlling the peroxisomal protein import machinery or our secretion mechanism. The idea behind this was to harness the high level of both spatial and temporal precision provided by optogenetics. 
Using optogenetics also provided us with an opportunity to increase the variability of our compartment toolbox, which is integral to our project. As such, this subproject can be perceived as an additional feature to our toolbox parts.
</P></BUTTON><DIV class="panel"><H3>LOV-PTS1</H3><FIGURE><FIGCAPTION><STRONG>Figure 9.1:</STRONG> Sketch of our GFP-LOV-PTS1 construct</FIGCAPTION></FIGURE><P>
Our first idea was to optogenetically control peroxisomal protein import via PTS1. For this we used the protein LOV2, fused to sfGFP which is further described in <A href=" https://www.nature.com/articles/nrm3700"><ABBR title="Smith, Jennifer J.; Aitchison, John D. (2013): Peroxisomes take shape. In: Nature reviews. Molecular cell biology 14 (12), S. 803–817.">(Smith and Aitchison 2013)</ABBR></A>.
The idea behind this was to use fluorescence microscopy to check whether GFP was mainly found inside peroxisomes after an illumination experiment. </P><H3>Split-TEV-PTS2</H3><FIGURE><FIGCAPTION><STRONG>Figure 9.2:</STRONG> Sketch of our PTS2-import construct</FIGCAPTION></FIGURE><P>
Peroxisomes possess two protein import pathways: via Pex5 and via Pex7. The targeting sequence for Pex7 is N-terminal, so LOV2 cannot be used for controlling the Pex7-import. Therefore we used a different optogenetic switch which does work, namely phytochrome B and its binding partner PIF6 <A href=" https://www.ncbi.nlm.nih.gov/pubmed/27753483"><ABBR title="Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377."> (Zhou <I>et al.</I>, 2016)</ABBR></A>. 
Our plan involved blocking the protein of interests PTS2 with a fluorescent protein and using Phytochrome B and PIF6 to reassemble a split TEV-protease. This would lead to cleavage of the fluorescent protein blocking the PTS2-sequence of the protein of interest and subsequent import via Pex7.</P><H3>Optogenetic gene expression</H3><FIGURE><FIGCAPTION><STRONG>Figure 9.3:</STRONG> Sketch of our optogenetic gene expression construct</FIGCAPTION></FIGURE><P>
Our optogenetic approach was about controlling the expression of genes of interest. For this we used a system which was already used by <A href="https://2016.igem.org/Team:Duesseldorf">last year’s iGEM team optoptosis </A> and also involves Phytochrome B and PIF6.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>