Team:Cologne-Duesseldorf/Design
Home
Team
Team
Collaborations
Project
Description
Design
Experiments
Notebook
InterLab
Model
Results
Demonstrate
Improve
Attributions
Parts
Parts
Basic Parts
Safety
Human Practices
Silver
Integrated Practices
Public Engagement
Awards
Applied Design
Entrepreneurship
Hardware
Model
Plant
Software
Judging Form
Wiki Tutorial
Team
Team
Collaborations
Project
Description
Design
Experiments
Notebook
InterLab
Model
Results
Demonstrate
Improve
Attributions
Parts
Parts
Basic Parts
Safety
Human Practices
Silver
Integrated Practices
Public Engagement
Awards
Applied Design
Hardware
Model
Software
Judging Form
Wiki Tutorial
Design
We designed a novel toolbox for complete control over all major functions of the peroxisome. The toolbox is our solution for improving the engineering workflow and predictability of synthetic constructs. Interested? Find out how.
Scientific background
The root problem
Synthetic biology is an engineering discipline. And while we are able to plan our constructs with tools like biobricks, a major difference to e.g. electrical engineering is that they are not nicely isolated on a chip, but surrounded by all types of interfering agents. One of the major issues regarding protein expression in a novel chassis is unwanted and unexpected crosstalk between engineered pathways and the native cellular processes of the production host. The other one is toxicity of the products or intermediates of the pathway. Both can strongly affect our system’s behavior, which may in some cases lead to us having to trial-and-error find a solution, making our previously modeled optimization useless.
Our approach
The natural approach of organisms to deal with metabolic interference and toxic byproducts is subcellular compartmentation. This has proven to be a functional solution in naturally occurring pathways in eukaryotes as well as new synthetic pathways for biotechnological application. Thus, the creation of a synthetic organelle presents a suitable strategy to increase the efficiency and yield of non-native pathways. A common approach is to build up artificial compartments from scratch. Many breakthroughs have been achieved in the last decade, however the creation of a fully synthetic compartment is yet a milestone to reach for. We on the contrary want to start by engineering artificial compartments through orthogonalization.
The peroxisome is the ideal starting candidate as it has many advantages over other compartments, including it being able to import fully folded proteins and not being essential in yeast under optimal growth conditions. Our project’s aim is to create a toolbox for manipulating and creating customizable peroxisomes as a first step towards synthetic organelles.
By modifying the import machinery of yeast peroxisomes only selected proteins will be imported into the peroxisomes leaving the researcher in full control over the content of the peroxisomal lumen. Furthermore our toolbox will include a secretion mechanism for the synthesized products, various intra-compartmental sensors, modules for the integration of proteins into the peroxisome membrane, as well as optogenetic control for some of these parts for a more precise spatiotemporal control.
As a proof of concept for the functionality of the toolbox and the customizable compartment two metabolic pathways will be integrated into the altered peroxisome: (i) Violacein biosynthesis and (ii) nootkatone biosynthesis. Violacein, a bisindole formed by condensation of two tryptophan molecules, is a violet pigment and thus easy to quantify in the cell. Nootkatone on the other side is a natural compound found inside the peel of the grapefruit, which gives it its characteristic taste and smell. In addition, nootkatone is a natural mosquito and tick repellent that is already being commercially used and industrially manufactured. Unfortunately, the production costs are extremely high. Furthermore, the production of nootkatone inside yeast is challenging as it is toxic for yeast and thus, the production efficiency is rather low. A successful implementation of the synthetic compartment will show increased yields in the production of these compounds and showcase the potential of this approach for similar future applications.
Cloning strategies and the Yeast Toolbox for Multipart-Assembly
While describing our cloning strategies we mentioned several levels, which stand for different stages of our plasmids. They are further described in the work of J.M. Dueber and colleagues, who designed the well established yeast toolkit we used in this project (Dueber) .  The toolkit offers the possibility to design plasmids with desired antibiotic resistances, promoters as well as terminators from standardized parts. It also provides fluorescence proteins, protein-tags and many more useful components as part plasmids. These part plasmids are distinguished in different part types due to their specific overhangs to ensure their combination in the correct order (e.g. promoter - gene of interest - terminator) all in a versatile one-pot Golden Gate reaction without time-consuming conventional cloning steps.
Figure 1: The yeast toolkit starter set comprises of 96 parts and vectors. The eight primary part types can be further divided into subtypes. (Dueber)
The cloning steps regarding the plasmid levels are implemented in E.coli in order to reduce the required time to generate the final plasmids. The different levels are therefore defined by their part content and their antibiotic resistances.
To generate a level 0 plasmid, the gene of interest is ligated into the provided level 0 backbone via Golden Gate assembly using the enzyme BsmBI. The backbone contains a resistance to Chloramphenicol, as well as an origin of replication, creating a very basic yet functional plasmid.
The level 1 plasmid contains a promoter and terminator suited for S. cerevisiae. There is the possibility of including  a polyhistidine-tag if there is a need for Western blot analysis. The antibiotic resistance contained in the level 1 plasmid changes from chloramphenicol to ampicillin which enables filtering out residual level 0 plasmids contained in the Golden Gate product. Furthermore, the Dueber toolbox includes the possibility of designing GFP-Dropout cassettes. These are custom-built level 1 backbones whose inserts are sfGFP as well as promoter and terminator suited for E. coli.  Upon a successful cloning step the GFP is replaced by the part(s) of interest, and correct colony shows a white colour. In case of a wrong ligation event colonies show a green fluorescence. This provides a very useful tool to detect unsuccessful cloned colonies. The enzyme used for level 1 changes from BsmBI to BsaI to avoid any interference between different steps.
The level 2 plasmid combines two or more genes of interest with their respective promoters, terminators and tags. The resistance changes from ampicillin to kanamycin. The enzyme of this step is BsmBI again. This level is useful, if the construct you are designing requires multiple genes to be transformed into one yeast strain.
Yeast nomenclature
To make it fast and easy to differentiate between endogenous and heterologous genes and gene products we decided to use S. cerevisiae nomenclature according to yeastgenome.org .
Below nomenclature at the example of your favorite gene 1, YFG1 is explained.
Letter code
Meaning
YFG1
Your favorite gene S. cerevisiae wild type allele
yfg1Δ
Gene deletion of your favorite gene
Yfg1
Protein product of YFG1
YFG2 A heterologous gene product from mammalian cells
Design of our sub-projects
Protein Import
Control over the peroxisomal proteome is an essential requirement for our project. Our objective is the obtainment of a fully orthogonal import system by utilizing the two peroxins Pex5 and Pex7.
Engineering of Pex5 and PTS1
Designing our receptors
To achieve the engineering of an orthogonal import pathway, we followed two approaches regarding Pex5. The first is based on targeted mutagenesis based on educated guesses which is first verified by molecular dynamics and later experimentally in the laboratory.    The second approach is based on a recently published paper: We searched for literature dealing with the modification of the peroxisomal import machinery. During our research we came across a paper of Alison Baker et al., published in 2017, in which they present a synthetic construct of the Pex5 protein, partly Arabidopsis thaliana and partly Physcomitrella patens. Compared to the wild type Pex5, this one shows different binding affinities since it interacts with a PTS1* variant that does not interact with the wild type Pex5. Since the protein sequences of yeast's and plants's Pex5 differ quite a lot, we aligned both sequences to understand where the mutations were set.
Figure 1.1: Alignment of the Arabidopsis thaliana's PEX5 and Saccharomyces cerevisiaePex5
Figure 1.2: Alignment of the yeast’s Pex5 with the Pex5 variant R19
The alignment shows three red marked amino acids we changed in our receptor sequence. Interestingly, these mutations are located within the TPR motifs of our Pex5 protein and this persuaded us to try out this receptor, we call it R19. Due to lack of time we tested this Pex5 variant in silico and in vivo simultaneously. We started molecular dynamics simulations with a couple of PTS variants that we already tested with our previous designs − one of them was actually the variant they used in the paper (YQSYY). The details and results of our structural modeling can be found in the modeling section .
Furthermore, we synthesized this variant and together with two receptors we designed based on educated guesses we got three receptors for our experimental work.
Experimental design
Verification of peroxisomal protein import was performed  by tagging the fluorescent protein mTurquoise with our designed PTS variants. Additionally, a peroxisomal membrane protein was used to ensure peroxisomal localization. For that reason, we chose the transmembrane domain of Pex13 tagged with the fluorescent protein mRuby.
Pex13−mRuby
We used the peroxisomal membrane protein Pex13 as a fluorescent marker − by just using the transmembrane domain of Pex13 with a short linker, we make sure that it has no influence on the peroxisomal features. To obtain a higher differentiation from mTurquoise, which we use for another construct, we chose to work with mRuby. Literature research revealed that such constructs have been tested before − Erdmann et al. (2004) described a construct containing only PEX13200-310 with a C-terminal GFP.
Figure 1.3: PEX13 construct with C-terminal mRuby.
Pex5 variant
In order to achieve an orthogonal peroxisomal protein import machinery we used a Pex5 knockout yeast strain in which we transformed our artificial  Pex5 variant containing a modified PTS1 binding pocket. Our variation facilitates the detection of a non native PTS1 variant instead of the wild type PTS1. The construct contains a medium strength promotor, the Pex5 gene and a terminator. The remaining plasmid parts can be seen in the plasmid map below.
Figure 1.4: Pex5 gene variant.
mTurquoise−PTS
Our approach for import verification is based on the fluorescent protein mTurquoise tagged with our modeled PTS variants. After several promotor tests with different strengths, we decided to express this construct only in low amounts, since this was the most suitable possibility to detect potential mTurquoise localization.
Our construct is depicted in the figure below.
Figure 1.5: Fluorescent protein mTurquoise tagged with the PTS variant.
Combination of our constructs
To combine our constructs, we cloned our PEX5 and mTurquoise constructs into a level 2 plasmid portrayed below.
Figure 1.6: Level 2 plasmid containing the Pex5 gene and the fluorescent protein.
Subsequently,  co-transformation of the PEX13−mRuby plasmid and the level 2 plasmid was performed in order to verify peroxisomal colocalization.
PTS screening
Trusting on our targeted approach alone seemed risky − that is why we planned a PTS screening to find the most favorable PTS for our three receptors. Dueber et al. (2016) used the Violacein assay for a similar purpose. They screened for the best PTS for the wild type receptor and were successful. Hence another subproject of our team is the integration of the Violacein pathway into the peroxisome ( Violacein ), we were already supplied with all necessary enzymes − VioA, VioB and VioE.
Figure 1.8: Pathway leading to the production of prodeoxyviolacein − the assay is based on the green color to identify colonies without functional import mechanism.
The figure above shows the principles of the assay. VioA and VioB are localized in the cytosol and lead to the production of the IPA imine dimer while VioE is tagged with a PTS1 variant. Successful import leads to white colonies whereas missing import results in green colonies due to the cytosolic production of prodeoxyviolacein.
Our rests upon the following two plasmids which are co-transformed into yeast.
Figure 1.9: Plasmids used for Violacein assay. The left one carries the coding sequence for the Pex5 variant, VioA and VioB whereas the right one contains the gene for VioE plus the attached PTS1 variant.
As shown above, we created one plasmid containing VioA, VioB and one of our Pex5 variants while the other plasmid only contained VioE. We then designed primers which bind to the VioE plasmid to amplify the whole plasmid except the terminator − random PTS1 variants were attached to VioE with the help of a random primer library. Following up, we did the ligation with the corresponding terminator and obtained a mix of several different VioE-PTS1 plasmids.
After plasmid amplification in Escherichia coli we then co-transformed yeast with the two constructs and waited for the colonies to grow. With the yeast growing, prodeoxyviolacein should be produced in yeast cells with absent import (green color) and the IPA imine dimer (white color) should be produced in those with functional import.
Plasmid preparation of those with white color and subsequent sequencing leads to the identification of functional PTS1 variants. Afterwards, we repeat the cloning steps described before to obtain a mTurquoise−PTS1* construct and co-transform it with the corresponding Pex5 variant. Eventually, the correct localization of mTurquoise tagged with these PTS1 variants provides proof for its function.
Mutagenesis of PTS2
Imagine you need different protein concentrations in your artificial compartment. What to do? Take our modified PTS2 sequences with varying import efficiencies.
To characterize the import efficiency for the site-directed PTS2 firefly luciferase was used. Luciferase is a luminescent protein which can be split in a C- and a N-terminal part. Only when combined, luminescence can be detected. To measure the import efficiency the two parts will be expressed and imported into the peroxisome in a separated way. The smaller part (Split2) of the split luciferase will be brought into the peroxisome first via the PTS1 dependent pathway. The other part is imported via the respective modified PTS2 sequence. The better this sequence is recognized by Pex7, the stronger the luminescence of the assembled luciferase can be detected in the peroxisome. There is a chance of split parts of the luciferase assembling in the cytosol if the import is too slow. To avoid wrong conclusions of the luciferase localisation, we designed a negative control experiment. It includes a split luciferase similar to the one used in the initial experiment, but without the peroxisomal targeting sequence. Consequently there will be no import into the peroxisome. If we subtract the luminescence of the negative control experiment from the luminescence of the main experiment we can define the degree of import.
In addition to a directed approach according to Kunze and colleagues we also want to perform a random  mutagenesis experiment to alter the five variable amino acids of the core region of the PTS2 sequence in an unbiased manner. The aim is to generate a library of different peroxisomal PTS2. The 15 nucleotides are assembled by chance. In the DNA synthesis this sequence will either be described as [NNN]5 or [DNK]5. N stands for all four nucleotides mixed, K for either G or T and D for A,G or T. The “DNK” composition prohibits two out of three termination codons. Additionally with this library the amino acid frequency is improved towards a balanced ratio in between the different kinds (Dueber) .
Each approach could generate up to 415 DNA sequences, which is roughly 1,07 billion. On the level of the amino acid sequence there are 3,2 million possibilities, since each residue can be taken by 20 different amino acids.  For the assay we therefore need a high throughput method.
We adapted work of DeLoache, Russ and Dueber using the violacein pathway to measure the import effectiveness of tripeptides. The pathway consists of Violacein A (VioA), Violacein B (VioB) and Violacein E (VioE). It converts tryptophan into the green product prodeoxyviolacein (PDV). The first two enzymes, VioA and VioB, are expressed in the cytosol, and the third one, VioE, is targeted to the peroxisome with a PTS1 sequence. The degree of import can be measured by the intensity of green colour of the colonies. An efficient import signal leads to a strong import of the VioE into the peroxisome and subsequently to white colonies, because the intermediates cannot diffuse into the peroxisome to its respective enzyme. DeLoache et al. showed that there is a proportional correlation between the concentration of the green product PDV and a red fluorescent substance. The concentration of this product displays the import efficiency of the respective sequence.
This assay has been used for the evaluation of the generated PTS2 sequences. The VioE-PTS2 plasmids are harvested and cotransformed with a VioA-VioB plasmid. Each plasmid contains a specific auxotrophy marker. Consequently every growing colony contains both plasmids. To evaluate the respective sequence the concentration of the red fluorescence is measured. The more fluorescence is detected the more VioE is in the cytosol. Therefore the respective PTS2 is not that efficient. The other way around a low concentration of the fluorescent substance correlates with an efficient import via the respective PTS2.
Membrane integration
In our design we worked with both,  the cytosolic import way and the the ER-dependant one. We first tested our hypothesis and then created membrane anchors that can be fused to you protein of interest C-terminally as well as N-terminal. This allows you to add proteins to the compartment that never been introduced to the peroxisomal membrane before.
Experimental Work/Design
In order to test our hypothesis we fused the last 59 amino acids of the C-terminus of human PEX26 (AA 246-305) to a red fluorescent protein, to further elucidate the Pex3 /Pex19-dependent import. mRuby is generally used as a marker in combination with a fluorescent microscope to visualize the localization of the fusion protein. The C-terminus of PEX26 contains a helical signal-anchor, which serves as both, a mPTS and transmembrane domain. We designed our construct with mRuby2 fused to the N-terminal side of the PEX26 -C-terminus, this way the mRuby should face the cytosolic side of the peroxisomal membrane.    Quite similar to our mRuby- PEX26 approach, we designed a construct for the ER-dependent import. Therefore, we fused the mRuby2 fluorescent protein to the N-terminus of Pex3 (AA 1-39). This construct should be N-terminally anchored in the peroxisomal membrane, with mRuby2 again facing the cellular lumen.
Our main goal is to introduce a rather complex membrane protein to the peroxisome that can alter specific traits. For that we fused the Pex3 N-terminus (AA 1-39) to a Halobacterium salinarum bacteriorhodopsin protein (AA 16-262), replacing the first 16 amino acids ( Pex3 -BacR). The original archaeal bacteriorhodopsin acts as a proton pump by capturing light energy to move protons across the membrane out of the cell. The resulting proton gradient is subsequently converted into chemical energy.  Our assumption is that the first transmembrane segment determines the orientation of the following protein and that therefore due to the N-terminal anchoring signal the bacteriorhodopsin will be inserted in reverse orientation, pumping the protons into the peroxisome. This way the pH of the peroxisomal lumen could actively be controlled and adjusted.
Finally, our project involved combining the work of other subteams to verify the localization of our constructs in the peroxisome and analyze the effects they have on the import. Therefore, we are using the superfolded-GFP protein, another fluorescent marker, which is in our case fused to the peroxisomal import sequence PT1, and a version of Pex11 that is fused to the fluorescent marker Venus. Both markers emit light in the green light spectrum, were as mRuby2 emits light in the red part of the spectrum, giving us a strong contrast and an easy way of differentiating between the two under the fluorescent microscope.
To physically create our constructs, we researched the DNA sequences of bacteriorhodopsin , Pex3 and PEX26 via UniProt and pre-designed our fusion constructs with the software tool „Geneious“. We ordered the synthesis of three separate parts ( Pex3 , PEX26 and Pex3 -BacR) from IDT. To ease out the process of assembling our plasmids, we used the „ Dueber Toolbox ", containing various parts such as promoters and terminators, to tailor the plasmids specific to your needs. Finally, to combine all the selected parts, we used the „Golden Gate” assembly method.
Secretion
For the peroxisome secretion in S. cerevisiae we designed fusion proteins of the v-SNARE Snc1 with different peroxisomal membrane anchors. We tested the constructs using a GUS Assay. The assays were performed using transformants of the strain BY4742.
Experimental Design
We will adapt the system of  Sagt and colleagues  to secrete the content of our modified compartments (Sagt et al, 2009) .
For the application of this system in S. cerevisiae we use a truncated version of the v-SNARE Snc1 to decorate our compartments (Figure 3.1) (Gerst et al, 1997) .
Figure 3.1 A diagram of the general domain structure of Snc1. V is a variable domain which is not important for the binding to the t-SNARE. TM is the transmembrane domain. H1 and H2 are the a-helical segments, forming the SNAREpin with the t-SNARE (Gerst et al, 1997)
To decorate the compartments with the SNARE we use a peroxisomal transmembrane protein . In our case we use the proteins Pex15 or PEX26, which were further investigated in another sub project, and fuse Snc1 to the N-terminus. We expressed these constructs of membrane anchor and Snc1 constitutively under control of the RPL18B promotor. In case of Pex15 we used a truncated version, lacking a large part of the N-terminus, only consisting of the transmembrane domain (315-383) (Figure 3.1). For PEX26 we use the truncated version published in Halbach et al. (Halbach et al, 2006) .
Figure 3.2 Concept of secreting peroxisomal contents to the supernatant. For the secretion, the membrane anchor Pex15 or Pex26 is used. This anchor is used to decorate peroxisomes or our modified compartments with the v-SNARE Snc1. For the secretion Snc1 interacts with the t-SNAREs in the cell membrane. Induced from this interaction the vesicle and cell membrane fuse and the content of the compartment is secreted to the supernatant.
We verified our secretion using beta-glucuronidase (GUS) as a reporter protein. In 2012 Stock and colleagues described the GUS reporter assay for unconventional secretion (Stock et al, 2012) . With it, it is possible to determine whether a protein is secreted conventional and is N-glycosylated or secreted unconventional and not N-glycosylated. GUS is a bacterial protein with an N-glycosylation-site, which is active only if the protein is not N-glycosylated. The GUS-activity can be measured with different reagents in plate or liquid assays. Liquid assays can be applied qualitatively as well as quantitatively to measure differences in activity. If GUS is secreted by the conventional pathway the N-glycosylation leads to inactivation of the enzyme (Stock et al, 2012) (Fig 3.2).
Figure 3.3 The GUS Assay. GUS secreted with an unconventional secreted protein like Cts1 from Ustilago maydis active in the supernatant. GUS secreted with a conventional Signal peptide (Sp) inactive in the supernatant. If GUS is in the cytoplasm there is also no activity (Lysis control) (Feldbrügge et al, 2013) .
GUS will be imported to the peroxisome with the PTS1 sequence and measured quantitatively in the supernatant. We will use a coexpression of GUS-PTS1 and Snc1-Pex15 or Snc1-PEX26 to identify the secretion of the compounds. Furthermore, we will use GUS-PTS1 expressed in S. cerevisiae without Snc1 fused to a membrane anchor for a control. We will measure the active GUS in the supernatant with a liquid assay based on the turnover of  4-methylumbelliferyl-beta-D-glucuronide to 4-methyl umbelliferone (4-MU) (Blázquez et al, 2007) . Here we expect a higher activity of GUS in the supernatant of cultures with Snc1 decorated peroxisomes.
To increase the variability of our constructs we also designed vectors with and without a GS-Linker connecting the Snc1 with the Pex15. Additionally we tested our constructs in strains with a deletion of Pex11 . This deletion leads to formation of larger peroxisomes and may increase the efficiency of our secretion mechanism.
Size and Number
Size matters! Want to know how modifying out compartment’s morphology can benefit your reactions? Check out our design!!!
Size&Number
Pex11
For our first approach we took to regulate size and number of our compartments, we chose to work with Pex11, which is naturally involved in the constriction of peroxisomes. Thereby, an overexpression of Pex11 leads to a large amount of smaller peroxisomes, whereas a knock-out provides a small amount of enlarged peroxisomes.
To achieve full control of peroxisome size and number with Pex11, we first aimed at showing the expression of Pex11 in the cell and its integration into the peroxisomal membrane. Therefore, we used parts of the Dueber Toolbox.
Figure 4.1 Level 1 plasmides for expressing Pex11 in yeast cells. Assembled by the Pex11 coding sequence, the fluorecence protein Venus, the Terminator ScADH1 and different promoters. Top left ScSSW12, Top right ScPAB1, Bottom left GAL1 and Bottom right CUP1.
This way we were able control the strength of the gene expression. To reveal the successful expression we designed Pex11 as a Toolbox part and fused the fluorescence protein mVenus to the n-terminal end of the peroxin. This way, we could both detect the expression and the localization in the peroxisomal membrane. To measure the change in size and number of the peroxisome, the yeast cells had to be observed under the microscope. For the examination of the effect of different Pex11 concentrations in the cell, from total deletion to overexpression, we transformed all constructs into the wild type as well as in the PEX11$\Delta$ strain. For comparison with the normal peroxisome phenotype in both strains we use sfGFP fused to a PTS1 sequence which is transported into the peroxisomal matrix. The deviation in peroxisome morphology indicates the functionality of our constructs.
Pex34
Similar to Pex11 we aimed at examining the impact of different concentrations of Pex34 on the peroxisome morphology. In this case we investigated a concentration range between the wild type phenotype and different levels of over expression, achieved by different constitutive promoters and inducible promoters.
ue to the fact that Pex34 is fully inserted into the membrane, a fusion with a large fluorescence protein could lead to the degradation of this peroxin. Therefore we worked with two different approaches.For the first approach, we fused an mTurquoise protein to the C - terminal end of Pex34, assuming that it doesn't affect the native structure and the import into the membrane.
Figure 4.2 Level 1 plasmids for expressing Pex34 in yeast cells. Assembled by the Pex34 coding sequence, the fluorescence protein mTurquoise, the terminator ScADH1 and different promoters. Top left ScSSW12, top right ScPAB1, bottom left GAL1 and bottom right CUP1.
As a second approach we decided to express Pex34 without any tag in order to conserve its natural function.
Figure 4.3 Level 1 plasmids for expressing Pex34 in yeast cells. Assembled by the Pex34 coding sequence,  the terminator ScADH1 and different promoters. Top left ScSSW12, top right ScPAB1, bottom left GAL1 and bottom right CUP1. This constructs do not include any marker, so the expression in the cell must be evaluated by coexpressing sfGFP.
To show the variability of the peroxisome morphology in the second approach we used sfGFP-PTS1, which is imported into the matrix to visualize the change in their size. To analyze the success of our constructs we used fluorescence microscopy, either searching for a membrane localization of mTurquoise or enhanced and amplified peroxisomes visualized by sfGFP.
In Vivo Sensors
Surveillance of key physiology factors is indispensable for a good understanding and optimization of metabolic processes in compartments. In comparison to in vitro measurements, in vivo sensors have major advantages in terms of measure relevant physiological properties in living systems. Former methods required destructive assays, expensive and time consuming preprocessing of cells or compartments to determining fluxes inside cells Frommeret al. (2010) . This leads to inexact measurements with poor spacial and temporal resolution. The usage of in vivo sensors tackles all of these problems at once. They allow detecting fluxes inside the cell and compartments in real time and supersede difficult and time consuming cell and organelle extraction.
Sensors
To enrich our toolbox we decided to measure four essential physiological factors: ATP, NADPH, Glutathione and the pH.
ATP/ADP conversion is used as an energy currency in many cellular processes like translocation of proteins and metabolites or anabolic and catabolic turnovers. It is assumed that ANT1P an ATP/AMP antiporter is located in the peroxisomal membrane. Further the shuttle mechanism of the peroxisomal protein import machinery is ATP dependent (Palmieri L. et al., 2001) . NADPH plays an important role in anabolic pathways and is also indispensable to our desired pathways of nootkatone and violacein. Glutathione is an antioxidant and redox buffer which is also found in yeast peroxisomes. It is used as cofactor by at least two types of peroxisomal proteins the glutathione peroxidases and glutathione transferases, which reduce lipid- and hydrogen peroxides or transfer glutathione to lipid peroxides for the purpose of detoxification (Horiguchi H. et al., 2001) . Furthermore, it partly represents the redox state of the peroxisome. Knowing the pH of a compartment is important to predict the activities of almost all enzymatic processes inside and to follow up acidification and basification upon conversion of metabolites.
We finally chose two ratiometric sensors to perform measurements with. The pH sensitive and glutathione redox state reporting green fluorescent proteins pHLuorin2 (Mahon M. J. et al., 2011) and roGFP2 (Schwarzländer M. et al., 2016) . We aim to target them either in the peroxisomal lumen or the cytosol. To achieve peroxisomal targeting we attach the peroxisomal targeting signal 1 with Golden Gate cloning .
Figure 5.1 Level 1 plasmids with medium promotors and uracil auxotrophy. Top left roGFP2 cytosolic. Top right roGFP2 peroxisomal. Bottom left pHLourin2 cytosolic. Bottom right pHLourin2 peroxisomal.
Using promoters with different expression strength in order to find optimal measurement conditions is of high interest. There is a trade off between a high signal-to-noise ratio and self induced effects which are both dependent on expression levels (Schwarzländer M. et al., 2016) . This cannot be generalized for each sensor. For example, pHlourin2 has sparse influence on the existing pH because of the buffer effect of proteins.
Validation of the peroxisomal localization can be achieved via fluorescence overlap of the sensor and a peroxisomal marker in our case pex13-mRuby ( import mechanism ).
Figure 5.2 Level 2 plasmids with peroxisomal marker Peroxin13-mRuby and uracil auxotrophy for colocalization. Left roGFP2 with strong promoter. Right pHLourin2 with strong promotor.
It can also be validated by transforming the sensors attached to the PTS1 sequence into Pex5 knockout yeast strain. The sensor is expected to show no specific localisation, because of the missing import sequence. We calibrate the sensors in living yeast cells and physiological ranges so that we can not only perform relative but also quantitative measurements. We aim to confirm our hypothesis of an more oxidized redox state of roGFP2 in peroxisomes with violacein pathway activity and want to measure differences in pH within yeast strains with peroxisomal membrane anchored pex3-bacteriorhodopsin protein (membrane proteins) .
Once expression and localization of the sensor is proven by microscopy, measurements with a plate reader or a fluorometer are acceptable. This allows a high number of replicates to be measured accurately in reasonable time. Microscopy is performed with a filter based Nikon Eclipse TI fluorescence microscope at 100-fold magnification and plate reader measurements are performed with a Tecan infinite plate reader (Mahon M. J. et al., 2011) . For pHLuorin2 emission intensity is measured at 535 nm upon excitation at 405 nm and 485 nm. Same settings were used for roGFP2 (Schwarzländer M. et al., 2016) . Evaluation of the reported signals is done by the excitation ratio of the the corresponding excitation wavelength.
Nootkatone
In order to have a proof of concept for our compartmentation strategy we intend to establish the nootkatone pathway inside the peroxisome. Therefore we aim to target the three enzymes for the nootkatone production to compartment. Before starting the actual lab work for the ambitious plan to establish a completely new pathway in the peroxisome, there are several aspects that have to be considered first.
The initial step is to find a reliable source to prove the abundance of our precursor Farnesyl pyrophosphate (FPP) in yeast peroxisomes. So far there is no proof of existence of FPP inside yeast peroxisomes yet. However it is predicted to be present, as it was detected in mammalian and plant peroxisomes Olivier et al. (2000) .
Figure 7.1: The structure of FPP the precursor for nootkatone.
The precursor FPP is converted into valencene by a valencene synthase (ValS). We chose the one from Callitropsis nootkatensis because of its comparably high efficiency in microorganisms. It achieves greater yields in yeast than the citrus valencene synthase. Furthermore, the product specificity is relatively high, while production of byproducts is low Beekwilder et al. (2014) .  The valencene synthase was also chosen because of its robustness towards pH and temperature changes Beekwilder et al. (2014) .   Our modelling approach revealed that for optimal yields an overexpression of valencene synthase is necessary because of its slow conversion rate (Model) . This is why we chose the strongest promoter of the yeast toolbox (Dueber Toolbox) for this attempt.
Figure 7.2:ValS lvl.1 PTS1 plasmid
Figure 7.3: The structure of valencene, an intermediate of nootkatone
The intermediate valencene is then converted into nootkatol by a P450 monooxygenase as well as into small amounts of our desired product nootkatone.  The P450 monooxygenase we chose for this project was taken from the bacterium Bacillus megaterium. In this  case it is not only a simple P450 monooxygenase, but an entire P450 system, consisting of a soluble P450 fused to a cytochrome P450 reductase (CPR) enzyme, making an additional reductase obsolete De Mot et al. (2002) .     Unlike eukaryotic P450s, which are mostly membrane bound, this prokaryotic BM3-P450 is located in the cytosol facilitating an easier transport into the peroxisome, as membrane integration of proteins is a more difficult task to achieve than import of cytosolic proteins Girvan et al. (2006) .   BM3 normally catalyzes the hydroxylation of long chain fatty acids Narhi et al. (1986) , which in our case could inhibit the conversion of valencene. For that reason we used a mutated version of BM3, which is called AIPLF. This variant is an enhanced version of the BM3 AIP version, which has a ten times better substrate oxidation rate for valencene than the wildtype BM3 and produces less byproduct when valencene concentration is saturated. Additionally to previous named benefits the AIPLF variant with 5 point mutations in the active side has a significantly lower binding affinity towards long chain fatty acids and therefore increase the transposition rate of valencene. Schulz et al. (2015) , Lehmann (2016) .
Figure 7.4: BM3 lvl.1 PTS1 plasmid
Figure 7.5: structure Nootkatol
The alcohol dehydrogenase from Pichia pastoris subsequently converts nootkatol into nootkatone by oxidation. It uses NAD+ as a cofactor which is reduced in the reaction. The regeneration of this cofactor is facilitated by the BM3, which oxidizes NADH Schulz et al. (2015) .
Figure 7.6: ADH lvl.1 PTS1 plasmid
Figure 7.7: structure Nootkatone
The first milestone to achieve our goal is the separate integration of each of our three enzymes ValS, BM3 and ADH into level 1 vectors (Dueber Toolbox) in yeast and to verify their expression by Western Blot analysis. Therefore, a 3xFlag/6xHis-tag was added to the N-terminus of each of the proteins.  It enables us to use an anti-His antibody followed by an anti-mouse-antibody to make protein abundances visible. Subsequently, the two enzymes ADH and ValS were combined in a  level 2 cassette plasmid.  BM3 is designed as a level 1 plasmid and will be co-expressed with the level 2 plasmid to achieve a nootkatone production. The expression of the enzymes in the cytoplasm is again verified by Western Blot analysis.   The further approach aims to provide the enzymes with C-terminal peroxisomal targeting signals type one, which finally converts the nootkatone pathway into our artificial compartment.
Since the production of nootkatone does not lead to a change of colour we need to apply different methods for verifying substrates. Once we succeed with the qualitative validation via Western Blot analysis, we can verify the presence of nootkatone by using high performance liquid chromatography and mass spectrometry, respectively.
Model influence on Nootkatone expression
We modeled the nootkatone biosynthesis pathway using ordinary differential equations in order to optimize nootkatone production. We found two hard and an easy problem, all of which we could find a solution for. The easy problem is optimization of the enzyme concentrations of the biosynthesis pathway. According to our model of the Nootkatone pathway we found that overexpression of Valencene Synthase is necessary to maximize the Nootkatone yield, while both alcohol dehydrogenase and p450-BM3 have only minor effects on the yield.
One of the hard problems, as shown in our penalty model is the toxicity of nootkatone and nootkatol. Since the toxicity most likely stems from both nootkatone and nootkatol clogging up the cell wall we present our peroxisome as a solution for this problem. When comparing the cytosolic model to our peroxisomal model we found that, if our assumption that neither Nootkatone nor Nootkatol are able to pass the peroxisomal membrane holds up, we can greatly increase Nootkatone production.
A further  problem is the influx of the pathway precursor farnesyl pyrophosphate (FPP). We used OptKnock analysis to design yeast strains with optimized FPP production. With this analysis we got hints that growing the yeast cells on a fatty acid medium might be a simple alternative to  knocking out the desired genes.
Violacein
Working on a project is a process of well planned steps. It starts with an idea and theoretical research to create a design. When finally demonstrating the mechanism of the project it is important to point out the benefits with a suitable application.
Synthetic biology offers countless numbers of new opportunities, especially in the field of metabolic engineering. To testify some main parts of our artico project, we decided to relocate a metabolic pathway into yeast peroxisomes. There are several reasons why this approach fits perfectly as a proof for our concept.
Based on described of advantages of violacein this pathway was chosen. Violacein is naturally produced in numerous bacterial strains, most popular in the gram-negative Chromobacterium violaceum . It is related to biofilm production and shows typical activities of a secondary metabolite (Seong Yeol Choi et al., 2015).
Figure 8.1 The synthesis of Violacein requires five enzymes encoded by the VioABCDE operon. VioA, a flavin-dependent L-tryptophan oxidase and VioB, a heme protein, work in combination to oxidize and dimerize L-tryptophan to an IPA imine dimer. Hydrogen peroxide is released as a by-product of the VioA reaction. Next step by VioE is the rearrangement of the IPA imine dimer to protodeoxyviolaceinic acid, which can non-enzymatically oxidize to prodeoxyviolacein or, by VioC via deoxyviolaceinic acid, oxidize to pink deoxyviolacein. The flavin-dependent oxygenases VioC and VioD require interaction with the oxidized form of flavin-adenine dinucleotide (FAD) (uniprot, uniprot) . The two enzymes act sequentially: first, VioD hydroxylates protodeoxyviolaceinic acid, leading to protoviolaceinic acid. Second, VioC creates the oxindole at the 2-position of one indole ring, leading to violet violacein (Balibar CJ et al., 2006) (Janis J. Füller et al., 2016).
Relocating the pathway into the peroxisome enables proximity of the enzymes and substrates. Furthermore the yeast cell is protected from the toxic substance hydrogen peroxide. Yeast peroxisomes have no problem with this as their main function is the beta-oxidation of fatty acids and the detoxification of the thereby produced H2O2 (Erdmann R. et al., 2007) . Because VioC and VioD are FAD-dependent, it is additionally an evidence for FAD availability inside of the peroxisome, if the synthesis of Violacein works. Otherwise the two enzymes would not be able to catalyze the reaction.
The genes for VioA, VioB and VioE were amplificated via PCR with Golden Gate compatible overhangs from the biobrick VioABCE (Part: BBa_K274004).
By Golden Gate cloning the peroxisomal targeting sequence (PTS1) was attached to the C-terminus of every pathway protein. Combined with the other necessary parts of the toolbox they represent the level 1 plasmids.
The PTS-tag marks the proteins for the import into peroxisomes. This should first of all point out the functionality of the yeast’s natural import mechanism and also be the basis for demonstrating our own modeled PTS*, proving our designed orthogonal import mechanism . Furthermore we also aim to optimize the working conditions for the enzymes inside of the reaction room - the peroxisomes. For example to vary the pH with new membrane proteins such as bacteriorhodopsin. To secure this change, we can also check the current conditions by our designed sensors .
There are several methods to verify the pathway’s enzymes. First of all, violacein and several intermediates (prodeoxyviolacein, deoxyviolacein, proviolacein) are colorful and the production in yeast can be visualized easily. Furthermore we added a His-/Flag-tag to the N-terminus of every protein (see geneious plasmid cards) to confirm their expression via SDS page and western blot. After verifying the presence of the enzymes the next step is to test their functionality. Before performing in vivo experiments in yeast an in vitro assay was implemented. For this the three enzyme pathway leading to PDV was reconstructed, testing VioA, VioB and VioE. To enable the best conditions for the enzymes, the pathway was studied intensively and all needed cofactors were calculated and added to the in vitro reaction (see protocol prodeoxyviolacein in vitro assay ). This included FAD, MgCl2, catalase for decomposition of hydrogen peroxide, and the substrate L-tryptophan. The in vitro reaction was followed by qualitative analysis via HPLC and mass spectrometry.
Design Rules For Genome Engineering Regarding Customization of Peroxisome Properties
In order to reach the ultimate goal of creating a fully controllable artificial compartment, genome engineering can be utilized for customizing the compartment's properties,.These include aspects such as membrane permeability, size/number, or decoupling of peroxisomes from the cytoskeleton, the peroxisomal proteome or metabolome. In our project we used the Crispr Cas9 system for knocking out several genes (Pex9, Pex31&Pex32, INP1, POT1) at the same time in order to engineer the previously mentioned properties. Furthermore, we designed a yeast strain  with a completely replaced protein-import machinery for controlling the entire peroxisomal lumen.
For the future one could think of much more radical strategies for peroxisomal engineering with a final goal of a “minimal peroxisome” by redirecting metabolic pathways through changing the protein-localization-signal in the yeast genome. Additionally, endogenous metabolic pathways could be redirected to our novel artificial compartment for establishing a customized metabolism specifically tailored for the user's application.
Introduction
In order to achieve a fully controllable artificial compartment, the first step was to design a completely orthogonal import system. Next was the knockout of endogenous import systems. However, a few proteins are imported neither by the Pex5 nor the Pex7 import machinery. Therefore, specific genome engineering designs, such as knockouts, deleting or redirecting the protein localization could be utilized for the ultimate goal of creating a synthetic organelle.
Additionally, knockouts or genome integrations enable customization of the peroxisomal properties, such as membrane permeability, size/number, decoupling of peroxisomes from cytoskeleton and the peroxisomal metabolism.
All these strategies allow a rational design of an artificial compartment, which is fully engineerable regarding  the proteome, metabolome and the entire peroxisomal environment.
Design of yeast multi -knockout strains
The Crispr Cas9 System
The demands on yeast engineering have significantly increased with the design of more complex systems and extensive metabolic pathways. Genetic techniques that have historically relied on marker recycling are unable to keep up with the ambitions of synthetic biologists. In recent years the Crispr Cas9 system has been used for several strain-engineering purposes, including:
Markerless integration of multiple genetic cassettes into selected genomic loci
Multiplexed and iterative gene knockouts without the need to recycle a marker
Precise genome editing – nucleotide substitutions, etc.
We utilized the Cas9 system as a tool for peroxisomal engineering and have adopted the existing toolbox from (Lee et al. 2015) and the complete cloning system which also provides the possibilities for genome integration and gene editing by Cas9. For this, two oligonucleotides have to be designed for targeting the Cas protein to the gene of interest.
Figure 1: Plasmid construction for the gRNA expression plasmid
Two oligos, containing the targeting sequence of the gRNA, have to be annealed and can then be integrated in the gRNA entry Vector by a Golden Gate reaction. Adapted from ( Lee et al. 2015 )
Several gRNA vectors can subsequently be assembled into one vector with a Cas9 expression cassette and then be transformed into yeast. The expression of Cas9 together with gene specific gRNA´s leads to double strand breakage followed by non-homologous end joining repair or homologous recombination, in case of added repair DNA (figure 3).
Figure 2: Plasmid construction for the expression plasmid containing Cas9 and gRNA´s
Vector for Cas9 and gRNA expression, assembled by a Golden Gate reaction, containing a URA marker, Cen6 yeast origin and a kanamycin resistance. Adapted from ( Lee et al. 2015 )
The combination of the Cas9 system with DNA repair sequences enables not only knockouts of peroxisomal proteins, but also allows redirecting protein localization by changing protein targeting signals or integration of linear DNA into yeast chromosomes. Genome engineering facilitates yeast strain development for customized peroxisomes.
Figure 3: Design of repair DNA sequences for homologous recombination after inducing double strand break by Cas9
Repair DNA sequences can be used to increase the efficiency for Cas9 guided knocking out of specific genes, but would also allow genomic integration of targeting signals or complete genes.  Adapted from ( Lee et al. 2015 )
The peroxisomal proteome of yeast (saccharomyces cerevisiae)
The peroxisomal proteome is studied extensively for saccharomyces cerevisiae and contains exactly 67 proteins ( Kohlwein et al. 2013 ). The function is characterized for the most of those proteins and it is known, that yeast peroxisomes are expendable under optimal growth conditions. Nevertheless, some knockouts are lethal under oleate or stress conditions.
Gene
Required for growth on oleate
Expression induced by oleate
Enzyme/activity
Molecular mass (kDa)
Isoelectric point
Molecules per cell
Localization
Function
ß-Oxidation enzymes
PCS60 (FAT2)
No
Yes
Medium chain fatty acyl-CoA synthetase
60.5
9.98
8.770
Peripheral peroxisomal membrane and matrix
Activates fatty acids with a preference for medium chain lengths, C9-C13
FAT1
No
-
Very long chain fatty acyl-CoA synthetase and long chain fatty acid transporter
77.1
8.47
16,900
Lipid droplet, ER, peroxisome Three predicted TM
Activates fatty acids with a preference for very long chain lengths, C20–C26
POX1
Yes
Yes
Acyl-CoA- oxidase
84.0
8.73
ND
Peroxisomal matrix
Oxidation of acyl-CoA
CTA1
No
Yes
Catalase
58.6
7.46
623
Peroxisomal matrix
Degrades hydrogen peroxide produced by Pox1
FOX2 (POX2)
Yes
Yes
Multifunctional enzyme; 3-hydroxyacyl-CoA dehydrogenase and enoyl-CoA hydratase
98.7
9.75
ND
Peroxisomal matrix
-
POT1 (FOX3, POX3)
Yes
Yes
3-Ketoacyl-CoA thiolase
44.7
7.56
ND
Peroxisomal matrix
Cleaves 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA
DCI1 (ECI2)
-
-
Δ(3,5)-Δ(2,4)-dienoyl-CoA isomerase (putative)
30.1
8.83
ND
Peroxisomal matrix
Auxiliary enzyme of fatty acid β-oxidation; role in β-oxidation debated
SPS19 (SPX1)
Yes
Yes
2,4-Dienoyl-CoA reductase
31.1
9.67
ND
Peroxisomal matrix
Auxiliary enzyme of fatty acid β-oxidation
ECI1
Yes
Yes
Δ3, Δ2-enoyl-CoA isomerase
31.7
8.21
ND
Peroxisomal matrix
Auxiliary enzyme of fatty acid β-oxidation
TES1 (PTE1)
Yes
Yes
Acyl-CoA thioesterase
40.3
9.58
ND
Peroxisomal matrix
Auxiliary enzyme of fatty acid β-oxidation
MDH3
Yes
Yes
Malate dehydrogenase
37.3
10.00
3,300
Peroxisomal matrix
Required for the malate-oxaloacetete shuttle, to exchange peroxisomal NADH for cytosolic NAD+, part of the glyoxylate cycle
IDP3
Yes
Yes
NADP+ dependent isocitrate dehydrogenase
47.91
10.02
ND
Peroxisomal matrix
Required for the 2-ketoglutarate/isocitrate shuttle, exchanging peroxisomal NADP+ for cytosolic NADPH
CAT2
No
No
Carnitine acetyl-CoA transferase
77.2
8.34
470
Peroxisome mitochondria
Transfers activated acetyl groups to carnitine to form acetylcarnitine which can be shuttled across membranes
Glyoxylate cycle
CIT2
No
-
Citrate synthase
51.4
6.34
2,310
Peroxisomal matrix
Condensation of acetyl CoA and oxaloacetate to form citrate
MDH3
Yes
Yes
Malate dehydrogenase
37.3
10.00
3,300
Peroxisomal matrix
Required for the malate–oxaloacetete shuttle, to exchange peroxisomal NADH for cytosolic NAD+
MLS1
Yes
-
Malate synthase
62.8
7.18
ND
Peroxisomal protein
Required for utilization of nonfermentable carbon sources
Other peroxisome-associated enzyme activities
GPD1 (DAR1, HOR1, OSG1, OSR5
-
-
NAD-dependent glycerol-3-phosphate dehydrogenase
42.9
5.26
807
Peroxisome, cytosol, nucleus
Key enzyme of glycerol synthesis, essential for growth under osmotic stress
PNC1
-
-
Nicotinamidase
25.0
6.23
7,720
Peroxisome, cytosol
Converts nicotinamide to nicotinic acid as part of the NAD(+) salvage pathway
NPY1
-
-
NADH diphosphatase
43.5
6.26
846
Peroxisome cytosol
Hydrolyzes the pyrophosphate linkage in NADH and related nucleotides
STR3
-
-
Cystathionine β-lyase
51.8
7.96
ND
Peroxisome
Converts cystathionine into homocysteine
STR3
-
-
Cystathionine ß-lyase
51.8
7.96
ND
Peroxisome
Converts cystathionine into homocysteine
GTO1
-
-
ω-Class glutathione transferase
41.3
9.53
-
Peroxisome
Induced under oxidative stress
AAT2(ASP5)
-
Yes
Aspartate aminotransferase
46.1
8.50
7,700
Cytosol, peroxisome
Involved in nitrogen metabolism
PCD1
-
-
Nudix pyrophosphatase with specificity for coenzyme A and CoA derivatives
39.8
6.59
238
Peroxisome
May function to remove potentially toxic oxidized CoA disulfide from peroxisomes
LPX1
-
Yes
Triacylglycerol lipase
43.7
8.16
2,350
Peroxisomal matrix
-
Peroxisomal transporters
PXA1 (LPI1, PAL1, PAT2, SSH2
-
-
Subunit of a heterodimeric ATP-binding cassette transporter complex
100.0
10.34
ND
Peroxisomal membrane
Import of long-chain fatty acids into peroxisomes
PXA2 (PAT1)
-
-
Subunit of a heterodimeric ATP-binding cassette transporter complex
97.1
9.47
ND
Peroxisomal membrane
Import of long-chain fatty acids into peroxisomes
ANT1(YPR118C)
-
-
Adenine nucleotide transporter
36.4
10.6
2,250
Peroxisomal membrane
Involved in β-oxidation of medium-chain fatty acids
Peroxins
Pex1 (PAS1)
-
-
AAA ATPase
117.3
6.93
2,100
Peroxisomal membrane
Involved in recycling of Pex5, forms heterodimer with Pex6
Pex2 (RT1, PAS5)
-
-
E3 ubiquitin ligase
30.8
9.02
339
Peroxisomal membrane
RING finger protein, forms complex with Pex10 and Pex12. Involved in matrix protein import
Pex3 (PAS3)
-
-
-
50.7
6.29
1,400
Peroxisomal membrane
Required for proper localization of PMPs
Pex4 (PAS2, UBC10)
-
-
Ubiquitin conjugating enzyme
21.1
5.36
ND
Peroxisomal membrane
Involved in matrix protein import
Pex5 (PAS10)
-
-
Soluble PTS1 receptor
69.3
4.79
2,070
Cytosol and peroxisomal matrix
Required for import of PTS1-containing peroxisomal proteins, contains TPR domains
Pex6 (PAS8)
-
-
AAA ATPase
115.6
5.44
1,630
Peroxisomal membrane
Involved in recycling of Pex5, forms heterodimer with Pex1
Pex7 (PAS7, PEB1)
-
-
Soluble PTS2 receptor
42.3
8.34
589
Cytosol and peroxisomal matrix
Requires Pex18 and Pex21 for association to the receptor docking site, contains WD40 repeat
PEX8 (PAS6)
-
-
Intra peroxisomal organizer of the peroxisomal import machinery
68.2
7.62
538
Peroxisomal matrix and luminal membrane face
Pex5-cargo dissociation
Pex9
-
-
PTS-receptor
-
-
-
-
-
Pex10
-
-
E3 ubiquitin ligase
39.1
9.88
ND
Peroxisomal membrane
RING finger protein involved in Ubc4-dependent Pex5 ubiquitination. Forms complex with Pex2 and Pex12
PEX11 (PMP24, PMP 27)
-
-
-
26.9
10.65
1,630
Peroxisomal membrane
Involved in peroxisome fission, required for medium-chain fatty acid oxidation
Pex12 (PAS11)
-
-
E3 ubiquitin ligase
46.0
9.86
907
-
RING finger protein, forms complex with Pex2 and Pex10
Pex13 (PAS20)
-
-
Component of docking complex for Pex5 and Pex7
42.7
9.83
7,900
Peroxisomal membrane
Forms complex with Pex14 and Pex17
Pex14
-
-
Central component of the receptor docking complex
38.4
4.61
2,570
Peroxisomal membrane
Interacts with Pex13
Pex15 (PAS21)
-
-
-
43.7
8.42
1,070
Peroxisomal membrane
Recruitment of Pex6 to the peroxisomal membrane, tail anchored PMP
Pex17 (PAS9)
-
-
Component of docking complex for Pex5 and Pex7
23.2
10.24
656
Peroxisomal membrane
Forms complex with Pex13 and Pex14
Pex18
-
-
-
Required for PTS2 import
32.0
4.78
ND
Interacts with Pex7 partially redundant with Pex21
Pex19 (PAS12)
-
-
Chaperone and import receptor for newly synthesized PMP
38.7
4.08
5,350
Cytosol, peroxisome, farnesylated
Interacts with PMPs, involved in PMP sorting. Also interacts with Myo2 and contributes to peroxisome partitioning
Pex21
-
-
Required for PTS2 protein import
33.0
6.67
ND
Cytosol
Interacts with Pex7, partially redundant with Pex18
Pex22(YAF5)
-
-
Required for import of peroxisomal proteins
19.9
8.33
259
Peroxisomal membrane
Recruits Pex4 to the peroxisomal membrane
Pex25
-
-
Involved in the regulation of peroxisome size and maintenance, required for re-introduction of peroxisomes in peroxisome deficient cells
44.9
9.77
2,420
Peripheral peroxisomal membrane
Recruits GTPase RhoI to peroxisomes, interacts with homologous protein Pex27
Pex27
-
-
Involved in the regulation of peroxisome size and number
44.1
10.49
382
Peripheral peroxisomal membrane
Interacts with homologous protein Pex25
Pex28
-
-
Involved in the regulation of peroxisome size, number and distribution
66.1
7.09
ND
Peroxisomal membrane
May act upstream of Pex30, Pex31 and Pex 32
Pex29
-
-
63.5
6.8
5,040
Peroxisomal membrane
May act upstream of Pex30, Pex31 and Pex32
Pex30
-
-
Involved in the regulation of peroxisome number
59.5
5.59
4,570
Peroxisomal membrane
Negative regulator, partially functionally redundant with Pex31 and Pex32
Pex31
-
-
Involved in the regulation of peroxisome number
52.9
10.15
238
Peroxisomal membrane
Negative regulator, partially functionally redundant with Pex30 and Pex32
Pex32
-
-
Involved in the regulation of peroxisome number
48.6
9.14
ND
Peroxisomal membrane
Negative regulator partially functionally redundant with Pex30 and Pex31
PEX34
-
-
Involved in the regulation of peroxisome number
16.6
10.30
ND
Peroxisomal membrane
-
Peroxisome fission and inheritance
DYN2 (SLC1)
-
-
Light chain dynein
10.4
9.03
1,310
Cytosol
Microtubule motor protein
SEC20
-
-
v-SNARE
43.9
5.92
4,910
Golgi, ER
Involved in retrograde transport from the Golgi to the ER, interacts with the Dsl1 complex through Tip20
SEC39(DSL3)
-
-
Component of the Ds11p-tethering complex
82.4
4.65
1,840
ER, nuclear envelope
Proposed to be involved in protein secretion
DSL1 (RNS1)
-
-
Component of the ER target site that interacts with coatomer
88.1
4.69
8,970
Peripheral ER, Golgi membrane
Forms a complex with Sec39 and Tip20 that interacts with ER SNAREs, Sec20 and Use1
FIS1 (MDV2)
-
-
Required for peroxisome fission
17.7
9.87
2,410
Peroxisomal membrane mitochondria
Tail anchored protein recruits Dnm1 via Mdv1/Caf4; also involved in mitochondrial fission
DNM1
-
-
GTPase, dynamin like protein involved in peroxisome fission
85.0
5.25
9,620
-
Also involved in mitochondrial fission
VPS1 (GRD1, LAM1, SPO15, VPL1, VPT26)
-
-
GTPase, dynamin like protein involved in peroxisome fission
78.7
8.15
5,960
-
Also involved in vacuolar protein sorting
VPS34 (END12, PEP15, VPL7, VPT29, STT8, VPS7)
-
-
Phosphatidylinositol 3-kinase
100.9
7.79
1,080
-
Forms complex with Vps15
INP1
-
-
Involved in retention of peroxisomes in mother cells
47.3
8.34
639
Peroxisomal membrane
Recruited to the peroxisome by binding to Pex3
INP2
-
-
Myo2 receptor, involved in peroxisome inheritance
81.5
9.41
736
Peroxisomal membrane
-
RHO1
-
-
GTP binding protein of the Rho subfamily of Ras like proteins, involved in actin assembly at the peroxisome
23.2
6.07
ND
-
Involved in de novo peroxisome formation recruited to peroxisomes by Pex25
Knockout designs in our project
Pex 9
Pex9 is a recently discovered import receptor for PTS1 proteins, which is induced by oleate and is an import receptor for both malate synthase isoenzymes Mls1p and Mls2p. In order to get a completely empty reaction room, a Pex9 knockout was designed to prevent unintended protein import.
Pex 31 & Pex 32
It has been shown that knockouts of Pex31 and Pex32 leads to an increased Peroxisomal size, but additionally the membrane permeability was affected ( Zhou et al. 2016 ). This effect can be used as a tool for engineering membrane permeability by knocking out or overexpress both genes. A knockout would lead to an increased permeability and one could think of an opposite effect in case of overexpression, but this has not been shown yet.
INP1
The INP1 knockout was designed after the skype call of Prof Dueber, who recommended us, decoupling of peroxisomes from cytoskeleton in order to improve the secretion efficiency. INP1 is responsible for the tethering of peroxisome, which would inhibit the secretion of peroxisomes.
POT1
The only protein, which is imported by the Pex7 import machinery in saccharomyces is the 3-ketoacyl-CoA thiolase (POT1). A knockout of POT1 would enable utilizing the Pex7 import for proteins of interest, which cannot be tagged at the C-terminus with pts1, without having unintended import of other enzymes.
Genomic integration of our novel Pex5 import receptor
After testing our new Pex5 import systems, which is completely orthogonal to the natural import, the next step would be to replace the endogenous system with our artificial import system. Therefore, an integration plasmid was designed with help of the previously described yeast toolbox, containing HO locus homologies and a hygromycin resistance (Figure 4). Afterwards the plasmid was transformed into the yeast strain which was created by our collaboration partner Aachen (double knockout strain Pex5 & Pex7).
Figure 4: Design of integration plasmid for integrating our orthogonal Pex5 import receptor.
Therefore, an integration plasmid was designed with help of the previously described yeast toolbox, containing HO locus homologies and a hygromycin resistance
The resulting yeast strain allows full control over the peroxisomal matrix proteome, by replacing the whole protein import machinery, which is the first step for creating our artificial compartment.
Outlook
Besides the genome engineering approaches, which were performed in our project one could think of more radical strategies for peroxisomal engineering. A final goal could be a “minimal peroxisome”, which contains only the proteins that are required for the biogenesis of the peroxisome and import of proteins and metabolites. On the one hand peroxisomal pathways could be redirected to cytosol or other organelles and one the other hand endogenous metabolic pathways could be redirected to our novel artificial compartment by changing the protein localization signal in the yeast genome with help of the Cas9 system. All these strategies would allow tremendous improvements for metabolic engineering applications by creating an artificial compartment, which can be rational designed and customized for specific metabolic pathways.
Optogenetic enhancements
Optogenetics provide a useful tool for controlling cellular processes with high spatial and temporal precision. These include the expression of certain genes as well as the interaction or separation of two proteins. As many of our toolbox’s aspects benefit from precise control, we wanted to include optogenetics as a way of increasing its variability. Our plans included optogenetically controlled protein import, control of compartment size and number, and control of product secretion.
Introduction
Our optogenetic toolbox enhancements can be divided into three subgroups: controllable protein import via Pex5, controllable protein import via Pex7 and controllable gene expression. Each of these sub-projects are of different design which will be illustrated in the following.
Pex5 import with LOV2
LOV2 is an optogenetic protein derived from Avena Sativa’s Phototrophin 1. In its dark state the J$_{\alpha}$-helix located at the C-terminus is bound to the core of the protein. Upon irradiation with blue light (~460 nm), a covalent bond between a cysteine residue on the LOV2 protein and a flavin mononucleotide chromophore causes the J$_{\alpha}$-helix to unfold, which in turn exposes the C-terminus (Spiltoir et al, 2016) . This property is very useful, as short amino acid sequences can be attached to this end of the LOV-protein, for example a peroxisomal targeting sequence! The idea for this project was to attach PTS1 to the C-terminus and the protein of interest to the N-terminus. Upon irradiation with blue light, the fusion protein would be imported into our compartment. We used a mutated version of LOV2 whose C-terminus has an increased dark-state binding affinity. This is caused by the substitutions and T406A and T407A. These mutations greatly reduce the possibility of the J$_{\alpha}$-helix being exposed in the dark state. Our PTS1 sequence consists of the amino acids LQSKL. As a proof of concept for this construct we fused sfGFP to its N-terminus:
Lv1 -plasmid containing the sequence coding for the GFP-LOV2-PTS1 fusion protein
This was done in order to visualize our experiment’s results during microscopy: upon successful import of the fusion protein, one would observe GFP fluorescence localized to our compartment. Otherwise, the whole cell would be illuminated.
Pex7 import
The idea behind this project is to initially block the protein of interest’s PTS2 with a fluorescent protein which can be removed by an optogenetically activated TEV-protease. For this project we use the protein Phytochrome-B from Arabidopsis thaliana and its interaction partner PIF6. These two proteins, also derived from Arabidopsis thaliana, bind together upon irradiation by red light (660 nm) and separate upon irradiation with far-red light (780 nm). We used this property to activate a split version of a TEV-protease whose split halves were each fused to one of the two optogenetic proteins.
The TEV-protease was obtained from the Biobrick BBa_K1319004. This variant contains the anti self-cleavage mutation S219V. Using overhang-PCR we created a split version of the Biobrick protease based on work done by Wehr et al, 2016 and the iGEM team Munich 2013 . The split was made between amino acid 118 and 119.
Our construct for attaching proteins N-and C-terminally is highly variable: it consists only of a TEV-cleavage site, the PTS2 sequence and a short linker and was designed as a 3b-part for the yeast-toolbox . This means that we can attach any protein to its N- or C-terminus we desire.  We planned on attaching different fluorescent proteins to each sides of the 3b-part in a Lv1-ligation. For our experiment we planned on using the pairs mTurquoise-mVenus and GFP-mRuby.
Lv1 -plasmid containing the sequence coding for the GFP-PTS2-Ruby fusion protein
The two other constructs were planned as follows: Phytochrome B was fused to the C-terminal TEV-half, PIF6 was fused to the N-terminal TEV-half. The PhyB-TEV2 part and the TEV1-PIF6 part were supposed to be inserted into a shared plasmid via a LV2 Golden Gate ligation. Finally, the Lv2-plasmid and the remaining Lv1 construct were to be co transformed into S. Cerevisiae. Our experiment consisted of illuminating one sample with red light (660nm) while keeping another sample in the dark. Fluorescence microscopy would then be used to check whether the import was successful [3]. If cleavage and subsequent protein import was successful, fluorescence of one protein would be localized to the compartment while that of the other would be observed throughout the cell.
Optogenetically controlled gene expression
This project is based on work done by Weber et al, 2016 . Using the interaction between Phytochrome B and PIF6 they designed an optogenetic switch for enabling and disabling transcription of a chosen gene. It is based on the tetracycline operon and the transcription factor VP16. The tetO operator is located upstream of a minimal promoter which in turn is located upstream of the gene of interest. The tetR repressor binds to the tetO sequence. Fused to it is PIF6. Phytochrome B is fused to the transcription factor VP16. Upon illumination with red light, Phytochrome binds to the tetR-PIF6 complex. VP16 is now located in close proximity to the minimal promoter, which enables the RNA-polymerase-2 to start transcription of the gene of interest.  We designed a promoter part for the Dueber toolbox which consists of tetO and the minimal promoter region. This can be used as a promoter in a Lv1-ligation for any desired gene of interest (a GFP-tagged Pex11 in our example).
Lv1 -plasmid containing the sequence coding for the GFP-Pex11 fusion protein with the tetO-pmin promoter
Transformation into S. Cerevisiae is accompanied by co-transformation of a Lv2 plasmid containing both the tetR-PIF6 and PhyB-VP16 constructs:
Lv2 -plasmid containing the sequences coding for the PhyB-VP16 and tetR-PIF6 proteins respectively
Verification methods depend on which gene is expressed. See size and number and secretion for details.
