<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cologne-Duesseldorf_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cologne-Duesseldorf/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="sidebar"><UL><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf">Home</A></LI><UL class="sub"><A href="#!">Team</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Collaborations">Collaborations</A></LI></UL><UL class="sub"><A href="#!">Project</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Attributions">Attributions</A></LI></UL><UL class="sub"><A href="#!">Parts</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Basic_Part">Basic Parts</A></LI></UL><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Safety">Safety</A></LI><UL class="sub"><A href="#!">Human Practices</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Engagement">Public Engagement</A></LI></UL><UL class="sub"><A href="#!">Awards</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Plant">Plant</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Software">Software</A></LI></UL><LI><A href="https://igem.org/2017_Judging_Form?team=Cologne-Duesseldorf">Judging Form</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Wiki-Tutorial">Wiki Tutorial</A></LI></UL></DIV><NAV><UL class="menu"><LI class="menu-item-has-children"><A href="#!">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Collaborations">Collaborations</A></LI></UL><LI class="menu-item-has-children"><A href="#!">Project</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Attributions">Attributions</A></LI><LI class="menu-item-has-children"><A href="#!">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Safety">Safety</A></LI><LI class="menu-item-has-children"><A href="#!">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Engagement">Public Engagement</A></LI><LI class="menu-item-has-children"><A href="#!">Awards</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Software">Software</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Cologne-Duesseldorf">Judging Form</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Wiki-Tutorial">Wiki Tutorial</A></LI></NAV><ARTICLE><H1>Results</H1><H2>Introduction</H2><P>
While there still is much that can be done to improve our toolbox further, we are nonetheless extremely proud of our achievements. The many months of lab work definitely paid off! Below you can find the results of our efforts.
</P><H2>Sub-projects</H2><BUTTON class="accordion"><H2 id="Pex5 Import">Pex5 Import</H2><P>
The orthogonalization of the Pex5 import mechanism was an ambitious and challenging task − interested how we did? See our results!
</P></BUTTON><DIV class="panel"><H3>MD simulations</H3><P>
 The first experiments we performed in the wet lab were the tests of the receptors we modelled via molecular dynamics. As soon as we finished building our constructs, we transformed them into Pex5 knock out yeast cells. The results of this experiments can be seen in the following figure.
</P><DIV class="half-width"><FIGURE><FIGCAPTION><STRONG>Figure 1.1: </STRONG>Pex5 variants R8 and R15 obtained in the course of molecular dynamics simulations. The figures show an unspecific localization of mTurquoise in the whole cytosol, meaning there is no functional import mechanism. Due to the indifferent results with any Pex5*−PTS1* combinations, we show just two exemplary results.
</FIGCAPTION></FIGURE></DIV><P>
 The fluorescent signal of mTurquoise was detected in the whole cell of each modelled receptor−peptide combination. This indicates that our receptors were not able to recognize the PTS-variants tagged to mTurquoise and thus we did not obtain any evidences regarding orthogonal peroxisomal protein import.
</P><H3>Pex5 variant R19</H3><P>
 Our second approach for the modification of the importer Pex5 was our designed receptor R19. Based on published literature we built this receptor by replacing three amino acids within the Pex5 protein sequence of the wild type yeast. The corresponding modified PTS1* is characterized by its -SYY sequence at the very end of the peptide. Figure 1.2 displays a fluorescence microscopy image proving our artificial protein import system in a Pex5 deficient yeast strain.
</P><DIV class="max-width"><FIGURE><FIGCAPTION><STRONG>Figure 1.2: </STRONG>Pex5 variant R19 with the PTS1* and the two negative controls consisting of R19 with the wild-type PTS1 and the PTS1* cloned  in the wild-type strain	

</FIGCAPTION></FIGURE></DIV><P>
Our first results show that coexpression of R19 with mTurquoise tagged to PTS1* leads to import of the fluorescent reporter protein, indicated by localized fluorescence areas . The negative control consists of the wild type yeast strain carrying mTurquoise tagged with our designed PTS1* demonstrate the exact opposite: Fluorescence was detected in the whole cell, indicating that R19 is not capable to recognize and import the modified peroxisomal targeting signal with its cargo. Though, this figure does not prove that the reporter protein is located in the peroxisomes. Therefore we validated this results by coexpressing this import machinery with a peroxisomal marker protein as can be seen in the following. 
</P><DIV class="max-width"><FIGURE><FIGCAPTION><STRONG>Figure 1.3: </STRONG>Pex5 variant R19 with the PTS1* and PTS*, co-transformed with the Pex13-mRuby construct − the localization of mTurquoise tagged with PTS1* is clear and due to that import was successfully validated whereas the control did not show any signs of import.

</FIGCAPTION></FIGURE></DIV><DIV class="half-width"><FIGURE><FIGCAPTION><STRONG>Figure 1.4: </STRONG>Wild type Pex5 with the PTS1* and wild type PTS1.
</FIGCAPTION></FIGURE></DIV><P>
Pex13, as an integral protein of the peroxisomal membrane, provides perfect features to mark the membrane in order to clarify whether the localized areas which were shown in figure 1.2 are indeed the peroxisomes. As described in our <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#h2-3">experimental design</A>, we used Pex13's transmembrane domain and fused mRuby to it. Figure 1.3 above shows clearly the location of the peroxisomes as the fluorescent signal of mTurquoise is definitely located in the peroxisomes, which proves that R19 transports our cargo into the peroxisomes. On the contrary, mTurquoise is located in the whole cytosol in the strain with R19 and the natural Pts1, indicating that there is no functional protein import.
On the other hand we observed similar fluorescent signals in wild type yeasts that possess the modified PTS tagged to mTurquoise, as displayed in figure 1.4: Moreover, the figure shows the wild type yeast expressing mTurquoise tagged to the natural PTS1. Comparing both pictures, the wild type receptor is not capable to recognize our artificial PTS1* peptide and thus no import could be detected. 

Conclusively, all our negative controls were not able to import mTurquoise into the peroxisomes, confirming the orthogonality of our artificial protein import mechanism.

Finally, our results clarify that we established a synthetic protein import machinery, which works fully independent from the natural yeast peroxisomal protein import system. We were able to demonstrate the recognition of an artificial PTS1* sequence by our designed Pex5 receptor R19 and that the same receptor does no longer recognize the wild type PTS1 sequence. That means we accomplished to modify a highly conserved protein import mechanism without destroying its function but changing its affinity for our distinguished peptide sequence. This facilitates the possibility to utilize the primordial peroxisomes as an artificial cell compartment.
</P><H3>Conclusion</H3><P>
As the relocalization of an enzymatic pathway like the nootkatone and violacein pathway depends on a working import machinery that selects specifically for certain cargo proteins, this subproject was and is a crucial part for our whole project.
				Our results show that we designed and established a new and orthogonal peroxisomal import system in <I>Saccharomyces cerevisiae</I>. We modified one of the most conserved import machineries within the domain of eukaryotes - no matter if it is plants, mammals or fungi. This opens up new possibilities for biotechnological applications since this import system can be used to shift toxic compound reactions into the natural stress-resistant peroxisomes and thereby increase the yield and efficiency of rare biomolecule production <EM>in vivo</EM>.
				Furthermore, we managed to make a big step further towards a synthetic cell: While many research groups try to build up a synthetic cell from scratch, we decided to build it up from the inside by subverting its natural functional systems and making it fully customizable and controllable.
				This is why our new import machinery shows the potential for biotechnology and real world applications.
</P><DIV class="callout"><H3>Violacein assay</H3><P>
 The Violacein assay would have been an easy and fast application to identify possible yeast colonies that possess a functional new import machinery. It would have eased  finding a fitting peroxisomal targeting signal for our Pex5 variants due to the huge number of different PTS1 variants we could have screened using this method. Because time ran out and we already found a fitting import machinery in our receptor R19 and our modified PTS1* P*, we decided to discontinue this experiment and focused on the validation of our previously generated results.
</P></DIV></DIV><BUTTON class="accordion"><H2 id="Pex7Import">Pex7 Import</H2><P>Imagine you need different protein concentrations in your artificial compartment. What to do? Take our modified PTS2 sequences with varying import efficiencies.</P></BUTTON><DIV class="panel"><P>
The biased mutagenesis of the PTS2 could be characterized with a split-variant of YFP ( yellow fluorescent protein) or a split-luciferase. YFP tends to self assemble, consequently appropriate internal controls have to be designed <A href="http://www.mdpi.com/1422-0067/15/6/9628 "><ABBR title="Horstman, A., Tonaco I., Boutilier K. and Immink R. A cautionary note on the use of split-YFP/BiFC in plant protein-protein interaction studies. International journal of molecular sciences 15.6 (2014): 9628-9643">(Horstman)</ABBR></A>. Luciferase is highly efficient because almost all energy is converted into light, the protein is thus very sensitive <A href="https://www.ncbi.nlm.nih.gov/pubmed/25002334 "><ABBR title="Azad, T., Tashakor A., and Hosseinkhani S. Split-luciferase complementary assay: applications, recent developments, and future perspectives. Analytical and bioanalytical chemistry 406.23 (2014): 5541-5560.">(Azad)</ABBR></A>. It offers a suitable alternative to YFP as a single readout protein. We expected to detect luminescence as well in the actual samples as in the negative control containing no peroxisomal targeting signals due to split assembly in the cytoplasm. Unfortunately no suitable method to measure luminescence in the peroxisomes was established in this project.
Prerequisite for detecting luminescence is the availability of the substrate luciferin. It does not diffuse into the peroxisome in concentrations high enough for the luminescence reaction and becomes the limiting factor <A href="https://www.ncbi.nlm.nih.gov/pubmed/14558144      "><ABBR title="Leskinen, P., Virta M. and Karp M. One‐step measurement of firefly luciferase activity in yeast. Yeast 20.13 (2003): 1109-1113.">(Leskinen)</ABBR></A>.</P><P>An alternative step to verify the localization of the assembled split-luciferase in the peroxisome is to extract and purify the organelles. Prof. Ralph Erdmann established this method: a cell-free homogenate is created and the organelles are pelleted by centrifugation steps <A href="https://www.ncbi.nlm.nih.gov/pubmed/26330630 "><ABBR title="Cramer, J., Effelsberg, D., Girzalsky, W. and Erdmann, R. Isolation of Peroxisomes from Yeast. Cold Spring Harbor Protocols 2015.9 (2015): pdb-top074500.">(Cramer)</ABBR></A>. This workflow can be used to characterize the content of the purified peroxisomes by Western blot analysis.</P><P>To measure the import efficiency of a vast amount of targeting sequences via split-luciferase one needs to ensure a sufficient luciferin concentration in the peroxisome. Therefore luciferin importer have to be implemented in the peroxisomal membrane. Since this implies a huge cloning effort split-luciferase is not suitable for high throughput screening. ´</P><P>At the random mutagenesis approach one expected green and white colonies indicating varying import efficiencies.  The colonies containing “DNK” or “NNN” substitutions in the variable PTS2 region show a wide range of colours between white and dark green. The wild type PTS2 colonies depict a constant light green colour. The negative control containing VioE without a PTS2 shows a dark green colour in every colony. </P><FIGCAPTION><B>Figure 2.1: Colonies of the PTS2 library show a colour range of white to green.</B> It indicates targeting sequences of different import efficiencies. White colour correlates with a strong import, VioE is targeted to the peroxisome and hence no green product (PDV) is detectable.  </FIGCAPTION><P>Therefore we were able to generate targeting  sequences of different effectivities.  Subsequently the OD<SUB>600</SUB>  and the fluorescence with an  excitation wavelength of 535 nm and emission wavelength of 585 nm were measured. According to
<A href="https://www.nature.com/articles/ncomms11152"><ABBR title="DeLoache, W. C., Russ, Z. N., &amp; Dueber, J. E. (2016). Towards repurposing the yeast peroxisome for compartmentalizing heterologous metabolic pathways. Nature communications, 7.
">(DeLoache)</ABBR></A>

 production of PDV was associated with a yet unknown red fluorescent product, detectable at the described wavelength.  The import efficiency can be defined as the fluorescence per OD<SUB>600</SUB>.  A wide distribution of different values were observed indicating a broad variety of different PTS2 versions.</P><FIGCAPTION><B>Figure 2.2: Fluorescence per OD<SUB>600</SUB> of VioABE transformants are shown.</B> For all tested transformants, fluorescence with an  excitation wavelength of 535 nm (λ<SUB>Ex</SUB>: 535 nm) and emission wavelength of 585 nm (λ<SUB>Em</SUB>: 585 nm) per OD<SUB>600</SUB>, referring on yeast cell count, was calculated. Results show a large distribution of import efficiency and therefore different PTS2 versions.
  </FIGCAPTION><P>A high value correlates with an inefficient targeting sequence since VioE is not imported into the peroxisome with the respective sequence. A low fluorescence per OD<SUB>600</SUB> indicates a strong targeting sequence resulting in a low VioE concentration in the cytoplasm and no conversion of Tryptophan to PDV.</P><P>The next step would be to isolate the plasmids of promising yeast strains and sequence them. Subsequently mutations leading to an increased import can be characterized and organized in a library consisting of different parts with varying import efficencies. </P></DIV><BUTTON class="accordion"><H2 id="MembraneIntegration">Membrane Integration</H2><P>Not only can we completely control the import into our compartment we also managed to integrate two marker proteins, namely Pex3 from yeast and PEX26 from human cells into our membrane. Furthermore did we manage to integrate the proton pump bacteriorhodopsin that containing seven membrane domains and was a much bigger challenge.</P></BUTTON><DIV class="panel"><H4>Figure 1: <SPAN style="color: rgb(0, 0, 0);"><A href="http://www.uniprot.org/uniprot/Q7Z412">PEX26</A> expression and integration</SPAN></H4><FIGCAPTION><B>Fluorescent microscopy of bacteriorhodopsin coexpressed with sf-GFP:</B>.<SPAN style="color: rgb(0, 0, 0);">Microscopy pictures were taken with a Zeiss Elyra PS. Peroxisomes were labeled with GFP-PTS1 (green). The green fluorescent spots (on the right) shows a typical peroxisomal shape. The signal for membrane marker mRuby-PEX26 is shown in red (on the left). Both signals show colocalization when merged (middle), which indicates that the protein gets integrated into the membrane.</SPAN></FIGCAPTION><P>In order to have full control over the amount of expressed protein, we designed our plasmids with the inducible galactose promoter &quot;pGAL1&quot;. Not only were we able to see that our fluorescent marked protein anchors from <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A>  and  <A href="http://www.uniprot.org/uniprot/Q7Z412">PEX26</A> would localize at specific points inside our cells but also to show that it was in deed the peroxisome they were accumulating at. For that we coexpressed each of our fluorescent membrane anchors together with a GFP protein that was fused to a PTS1 sequence and thus imported into the peroxisome.Fluorescent microscopy was used to colocalize both, the green fluorescing GFP and the red fluorescing mRuby and it is clearly visible, that our anchors integrated into the peroxisomal membrane.</P><H4>Figure 1: <SPAN style="color: rgb(0, 0, 0);">Bacteriorhodopsin expression and integration</SPAN></H4><FIGURE><FIGCAPTION><B>Fluorescent microscopy of bacteriorhodopsin coexpressed with sf-GFP:</B>.<SPAN style="color: rgb(0, 0, 0);">Microscopy pictures were taken with a Zeiss Elyra PS. Peroxisomes were labeled with GFP-PTS1 (green). The green fluorescent spots (on the right) shows a typical peroxisomal shape. The signal for the membrane marked bacteriorhodopsin is shown in red (on the left). Both signals show colocalization when merged (middle), which indicates that the protein gets integrated into the membrane.</SPAN></FIGCAPTION></FIGURE><P>
Finally we used the same approach to direct a mRuby-tagged <A href="http://www.uniprot.org/uniprot/P02945">bacteriorhodopsin</A> to our compartment. In coexpressing it with the same GFP as in the previous steps, we could show that the <A href="http://www.uniprot.org/uniprot/P02945">bacteriorhodopsin</A> as well as <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A> and <A href="http://www.uniprot.org/uniprot/Q7Z412">PEX26</A> were successfully integrated into the membrane of our compartment. Since <A href="http://www.uniprot.org/uniprot/P02945">bacteriorhodopsin</A> is a rather complex protein, we're very optimistic about integrating other proteins into the membrane using the same approach. 
</P></DIV><BUTTON class="accordion"><H2 id="SizeAndNumber">Size and Number</H2><P>The big picture! Read through our results and find out how we achieved the full control of our compartment’s size and number via Pex34 overexpression!</P></BUTTON><DIV class="panel"><H3>SizeAndNumber</H3><H4>Pex11</H4><P>First we microscope the control, to examine the wild type phenotype of peroxisome in <I>S.cerevisiae</I></P><FIGCAPTION><B>Figure 4.1:</B> Fluorescence Microscopy of sfGFP-PTS1 in WT <I>S. cerevisiae</I> strain. Visible an accumulation of sfGFP in the peroxisomes.  
</FIGCAPTION><P>In the second control, where sfGFP-PTS1 is produced in a <I>PEX11</I> $\Delta$ knock-out strain only minor changes are visible (data not shown here). In this sample the cells mostly contained a single peroxisome, which was however not noticeably larger than without a <I>PEX11</I>-knockout.
The constructs containing Pex11 are transformed in wild type as well as in <I>PEX11</I> $\Delta$ cells. Visualized by the fluorescence marker Venus the localization of Pex11 in the peroxisomal membrane is detected (data not shown here). However, an impact on the phenotype was not measurable.
</P><H4>Pex34</H4><P>To measure the regular phenotype of peroxisomes we expressed sfGFP-PTS1 in a wild type <I>S. cerevisiae</I> strain and examined the cells under the microscope. Because all our constructs are expressed in wild type cells we are able to compare the sample to this control in order to detect any differences in peroxisome morphology.
</P><FIGCAPTION><B>Figure 4.2:</B> Fluorescence Microscopy of sfGFP-PTS1 in WT <I>S. cerevisiae</I> strain. Visible an accumulation of sfGFP in the peroxisomes.  
</FIGCAPTION><P>Figure 4.3 shows Pex34-mTurquoise with the strong constitutive promoter ScSSW12 expressed in wild type yeast strain. It showed clear differences to the normal phenotype.
The peroxisomes could be identified by accumulation of mTurquoise at the membrane. We can clearly see a large amount of small peroxisomes.
</P><FIGCAPTION><B>Figure 4.3:</B> Fluorescence Microscopy of Pex34-mTurquoise with the strong constitutive promotor ScSSW12 expressed in WT <I>S. cerevisiae</I> strain.  
</FIGCAPTION><P>In the following sample Pex34 is coexpressed with sfGFP-PTS1 to visualize the peroxisomes. First there is Pex34 assembled with the strong constitutive promoter ScSSw12 so Pex34 is overexpressed in the cell. A change of the phenotype was observed.
The sample showed an increase of size and number of the peroxisomes. At some points there are some cells appear to contain elongated peroxisomes.
</P><FIGCAPTION><B>Figure 4.4:</B> Fluorescence Microscopy of Pex34 assembled with the strong constitutive promoter ScSSW12  expressed in WT S. cerevisiae strain. 
</FIGCAPTION><P>In this sample Pex34 is assembled with a copper inducible promoter. Cultivating the cells in a medium with 0,5 $\mu$M copper concentration leads to a medium strong induction of the promoter. We can observe slightly enlarged peroxisomes compared to the wild type strain.
</P><FIGCAPTION><B>Figure 4.5:</B>Fluorescence microscopy of Pex34 assembled with the copper inducible promoter CUP1 induced with 0,5 $\mu$M copper expressed in WT <I>S. cerevisiae</I> strain.   
</FIGCAPTION></DIV><BUTTON class="accordion"><H2 id="Secretion">Secretion</H2><P>We are the first to implement  a completely new secretion mechanism in<I> S. cerevisiae</I>, which is able to secrete the peroxisomes` insides. This mechanism does not occur naturally in yeast. First we were able to show that our membrane anchors Pex15 and Pex26 are integrated into the peroxisomal membrane. By fusing our v-SNARE Snc1 to different membrane anchor we were able to cause secretion of the content of our peroxisomes to the extracellular supernatant. When Snc1 is fused to Pex15 via a linker we measured the highest amount of secretion.</P></BUTTON><DIV class="panel"><P>To check whether our membrane anchors localize in the peroxisomal membrane we used a Zeiss Elyra PS microscope. For Pex15 we observed localization using a construct with mVenus fused to the C-terminus of the Pex15 version we used. The fluorescence in the cells showed the typical shape of a peroxisomal localization (Figure 3.1). Shown in figure 3.2 is the localization of Pex26, which was highlighted using an N-terminal fusion with the fluorescent Protein mRuby. The microscopy pictures also indicate peroxisomal localization and even an co-localization with sfGFP-PTS1</P><FIGURE><FIGCAPTION><B> Figure 3.1 Microscopic validation of the peroxisomal membrane anchor Pex15. </B> Microscopy pictures were taken with a Zeiss Elyra PS. The signal for mVenus-Pex15 is shown in yellow. The picture validates the peroxisomal membrane localization of Pex15</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 3.2 Validation of the membrane anchor Peroxisomal membrane anchor Pex26.</B> Microscopy pictures were taken with a Zeiss Elyra PS. Peroxisomes were labeled with GFP-PTS1 (green). It shows a typical  peroxisomal shape. The signal for the membrane marker mRuby-Pex26 is shown in yellow. Both signals co localizing in the overlay. Which indicates that Pex26 is viable as a peroxisomal membrane marker.</FIGCAPTION></FIGURE><P>Next we measured secretion of compounds that are inside our artificial compartment, using a liquid <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#Secretion"> GUS-assay </A>. Towards this purpose we coexpressed GUS-PTS1 and  Snc1 fused to different membrane anchors. For lysis controls, GUS with PTS1 was expressed in the Strains BY4742 and BY4742 with the gene <EM>Pex11</EM> deleted. 
The fluorescence increase over time of the samples which are <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#Secretion"> decorated with SNAREs </A> is higher in comparison to that of the lysis controls. The highest activity could be measured in the samples using the truncated Pex15 membrane anchor without a linker. The same construct in a background strain with a <EM>Pex11</EM> deletion showed a lower GUS activity in the supernatant. The strains expressing Snc1 linked to Pex26 or Snc1 directly fused to the N-terminus of Pex15 only showed minor increase of RFU over time. (Figure 3.3.) </P><DIV class="half-width"><FIGURE><FIGCAPTION><B> Figure 3.3 Relative fluorescence units per minute (RFU/min) measured for supernatants of different <EM>S. cerevisiae</EM> strains.</B> The fluorescence was measured for 12 hours in intervals of 10 minutes with an excitation of 365 nm and an emission of 465 nm. For the strain BY4742 (wt) which was used as the background strain the fluorescence did not increase over the measured time period. The lysis controls (GUS-PTS1; ∆Pex11 GUS-PTS1) show a lower activity than the samples of strains with Snc1-decorated peroxisomes. The highest activity could be measured in the strain using Pex15 with a linker as a membrane anchor (Pex15 L). The assay was performed in three technical replicates.
</FIGCAPTION></FIGURE></DIV></DIV><BUTTON class="accordion"><H2 id="Sensors">Sensors</H2><P>We successfully proved expression and peroxisomal localization of pHlourin2 and roGFP2 PTS1 constructs with fluorescence microscopy. Furthermore we performed  <I>in vitro</I> calibrations and in  <I>in vivo</I> measurements inside the cytosol and peroxisome. We provide a great outlook on application of sensoric measurements for testing and validating of synthetic metabolic pathways in our artificial compartment. </P></BUTTON><DIV class="panel"><H3>Sensors</H3><H4>Localization</H4><P>pHlourin2 and roGFP2 was detectable by fluorescence microscopy and showed a similar excitation spectrum as native GFP<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3152828/"><ABBR title="2011, Mahon et al. - pHluorin2: an enhanced, ratiometric, pH-sensitive green florescent protein"> Mahon <I>et al.</I> (2011)</ABBR></A>. Sensors were compared to wild type with similar growth conditions. Cytosolic constructs were both visible and evenly distributed in the cell except for vacuole areas. </P><FIGCAPTION><P><FONT size="3"><STRONG>Figure5.1 </STRONG>Left: cytosolic pHLuorin2 expression. Right: wild type, without any fluorescence.
</FONT></P></FIGCAPTION><P>
Carboxyl terminal fusion of the peroxisomal targeting signal 1 resulted in small areas with high intensity inside the cells. Cytosolic fluorescence was nearly silenced. The Pex5 receptor recognizes this signaling tag and ensures the import into the peroxisome, which is shown in the following figure. </P><FIGCAPTION><P><FONT size="3"><STRONG>Figure5.2 </STRONG>Left: pHLuorin2 localized in the peroxisome. Right: roGFP2 localized to the peroxisome
</FONT></P></FIGCAPTION><P>
peroxin13 mRuby construct showed concentrated localization inside cellular areas too. In comparison to our tested sensors with the comparable promoter parts <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design">016 and 017</A> peroxin13 mRuby needed a higher gain for same fluorescence intensities. From our observations and literature we therefore considered peroxin13 mRuby construct as suitable peroxisomal marker. Expression of our level 2 plasmids containing both a sensor and peroxin13 mRuby showed colocalization which leaded to yellow spots in merged channels<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2655559/"><ABBR title="2009, Robert Yung-Liang Wang et al.- A Key Role for Heat Shock Protein 70 in the Localization and Insertion of Tombusvirus Replication Proteins to Intracellular Membranes"> Robert Yung-Liang Wang <I>et al.</I> (2009) </ABBR></A>.</P><FONT size="3"><STRONG>Figure5.3 </STRONG>Left: wild type without any fluorescence. Right: Level 2 Plasmid containing peroxin13-mruby and pHLuorin2-PTS1 using the mcherry channel. 

</FONT></DIV></ARTICLE><FONT size="3"><STRONG>Figure5.4 </STRONG> Left: Level 2 Plasmid containing peroxin13-mruby and cytosolic pHLuorin2 using the GFP channel. Right: Level 2 Plasmid containing peroxin13-mruby and pHLuorin2-PTS1 using the GFP channel. 
</FONT><FIGCAPTION><P><FONT size="3"><STRONG>Figure5.5 </STRONG> Left: Level 2 Plasmid containing peroxin13-mruby and cytosolic pHLuorin2. Images of the GFP and mcherry channel were merged.  Right: Level 2 Plasmid containing peroxin13-mruby and peroxisomal pHLuorin2. Images of the GFP and mcherry channel were merged. 

</FONT></P></FIGCAPTION><H4>pHluorin2</H4><P> Initially we desired an <I>in vivo</I> calibration with pH values ranging from pH 5.8 to 7.8. We therefore tried to equilibrate the pH of the cytosol with the supernatant. The cells were incubated in a potassium rich buffers containing the ionophore nigericin. The ionophore nigericin penetrates the cell membrane and acts as a potassium proton antiporter. Sadly we did not noticed any correlation between pH and the 405 to 485nm excitation ratio response even with 5 fold higher concentration.  For that reason we changed calibration method to an <I>in vitro</I> assay .
The cooled protein extracts of yeast strains containing pHlourin2 constructs and a wild type control were separately titrated to the desired pH values and measured at the plate reader afterwards.</P><FIGURE><FIGCAPTION><FONT size="3"><STRONG>Figure5.4 </STRONG>pHLourin2 protein supernatant with different pH values between pH 5.8 and 7.8. Analysis were performed using the Infinite M2000 pro Tecan plate reader with dual excitation at 405/5 nm and 485/5 nm and emission at 535/25 nm.</FONT></FIGCAPTION></FIGURE><P>As demonstrated above the sensor works perfectly fine. An ascending pH results in a shift of the excitation peaks from 485 nm to 405 nm over the entire considered pH area. </P><FIGURE><FIGCAPTION><FONT size="3"><STRONG>Figure5.6 </STRONG> The ratios between 405 nm and 485 nm at 535 nm emission, were obtained from the Infinite M2000 pro Tecan plate reader for the given pH values. </FONT></FIGCAPTION><P>For the peroxisomal usage of our sensor we had to ensure that the PTS1 signal peptide had no effect on the direction or the fold change of the ratio response .
 PTS1 tagged pHluorin2 shows a slightly smaller 405/485 nm ratio compared to the cytosolic located sensor. Despite the high standard deviation, experiments with a higher number of replicates are required examining significant differences. It seems that the linear correlation is not changed. Finally we performed <I>in vivo</I> measurements comparing peroxisomal and cytosolic pH.

For <I>in vivo</I> measurements yeast were grown on yeast nitrogen dropout medium at a pH of 6.0. A pH of 6.7/0.4 was measured in the peroxisomes, whereas in the cytosol we observed a slightly lower pH of 6,4/0,3. All measurements were obtained from yeast cultures with an OD<SUB>600</SUB> ranging from 0,8 up to 1.3. The large standard deviation might be rooted in the different ODs<SUB>600</SUB>. Literature does not agree about whether the pH inside the peroxisome is acidic or alkaline nor whether there are endogenous regulating mechanisms <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1133865//"><ABBR title="2004, Francesco M. Lasorsa et al. - The yeast peroxisomal adenine nucleotide transporter: characterization of two transport modes and involvement in ΔpH formation across peroxisomal membranes"> (Francesco M. Lasorsa<I>et al.</I>(2004), </ABBR></A><A href="http://jcs.biologists.org/content/117/18/4231"><ABBR title="2004, Carlo W. T. van Roermund et al. - The peroxisomal lumen in Saccharomyces cerevisiae is alkaline"> Carlo W. T. van Roermund<I>et al.</I> (2004))</ABBR></A>. Our result suggest a slight acid pH inside the peroxisomes and agreed with<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1133865//"><ABBR title="2004, Francesco M. Lasorsa et al. - The yeast peroxisomal adenine nucleotide transporter: characterization of two transport modes and involvement in ΔpH formation across peroxisomal membranes"> Francesco M. Lasorsa<I>et al.</I>(2004)</ABBR></A>.</P><P>
In summary, stronger promoters promise to gain a better signal to noise ratio. Still pHlourin2 calibration does not dependent on promoter strength which supports the hypothesis that pHlourin2 has sparse effect on the existing pH level. The sensor characteristics are neither changed by the pts1 signal. The <I>in vivo</I> calibration might have failed due to the disability of penetrating the yeast cell wall. Nevertheless we were able to measure the pH <I>in vivo</I>.
With this sensor we provide a part to iGEM which actually can detects pH changes inside our compartment purposed for pathway analyses or research. Data can be easily generated and examined. </P><H3>roGFP2</H3><P>After expression and correct localization to the peroxisome was validated we examined the function of roGFP2. We conducted an <I>in vitro</I> assay on fully oxidized and fully reduced roGFP2 and performed time measurements by subsequently adding  H<SUB>2</SUB>O<SUB>2</SUB>  and DTT to the protein extract. </P><FIGURE><FIGCAPTION><FONT size="3"><STRONG>Figure5.7 </STRONG>In the beginning roGFP2 was either treated with 1 mM DTT reducing the sensor or with 1 mM  H<SUB>2</SUB>O<SUB>2</SUB> to oxidize the protein supernatant. Later the complete oxidation/reduction was achieved through adding additional
and DTT.</FONT></FIGCAPTION><P>We could observe a functional sensor with a high dynamic range in the cytosolic and the PTS1 fused construct, which indicates high sensitivity. Further the  PTS1 Tag  does not seem to disturb the function of roGFP2(data not shown).
The calibration was performed using the mid point calibration method, which was previously performed by assuming the midpoint potential to be at -280mV<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2818.2008.02030.x/full"><ABBR title="2008, Schwarzländer et al.- Confocal imaging of glutathione redox potential in living plant cells"> Schwarzländer  <I>et al.</I>(2008) </ABBR></A>.</P><FIGURE><FIGCAPTION><FONT size="3"><STRONG>Figure5.8 </STRONG>405/485nm excitation ratio plotted against the oxidized proportion of roGFP2 </FONT></FIGCAPTION><P>
Based on the nernst equation we were now able to calculate the redox potential of roGFP2 regarding the oxidation of roGFP2<A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2818.2008.02030.x/full"><ABBR title="2008, Schwarzländer et al.- Confocal imaging of glutathione redox potential in living plant cells"> Schwarzländer,  <I>et al.</I>(2008)</ABBR></A>.
</P><FIGURE><FIGCAPTION><FONT size="3"><STRONG>Figure5.9 </STRONG>Oxidized roGFP2 proportion plotted against the redox potential of glutathione (mV)</FONT></FIGCAPTION>.

<P>
Using our calibrated sensor we could compare the redox states within strains which differ in metabolic physiology. 
</P><FIGURE><FIGCAPTION><FONT size="3"><STRONG>Figure5.10 </STRONG>Cytosolic and peroxisomal 405/485 nm ratios of roGFP2 were obtained. We used a strong(+) and weaker(-) Promoter. Yeast were grown on yeast nitrogen dropout medium at pH 6.0. Cultures showed an OD<SUB>600</SUB> between 0.9 and 1.1.</FONT></FIGCAPTION></FIGURE><P>We conducted two Mann-Whitney-Wilcoxon test on the different targets and on different promoters strength used(n<SUB>1</SUB>=6, n<SUB>2</SUB>=6) resulting in a p value of 0,69 and 0,82. Neither  comparisons of cytosolic and peroxisomal targeting nor comparison of the different promoters showed significant differences in the 405/485 nm ratio. We were able to calculate cytosolic roGFP2 redox states (-264 mV and -269mV) and peroxisomal redox states(-264 mV and -275mV). This result was surprising since varieties were reported in literature before. <A href="https://www.ncbi.nlm.nih.gov/pubmed/25867539"><ABBR title="2015, Schwarzländer et al. -Dissecting Redox Biology Using Fluorescent
Protein Sensors"> Schwarzländer <I>et al.</I> (2015)</ABBR></A>. 

</P><H4>Outlook</H4><P>We planned to calibrate our sensors <I>in vivo</I>as well and wanted to follow up changes induced by <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#Violacein">violacein</A> and <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#h2-2">nootkatone</A> pathways. Furthermore our objective was testing the expected acidification upon induced expression and integration of <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#Membrane_Integration">bacteriorhodopsin</A>into the peroxisomal membrane with pHLuorin2. roGFP2 can be fused to numerous redox catalytic enzymes making it specific to certain redox pools <A href="https://www.ncbi.nlm.nih.gov/pubmed/25867539"><ABBR title="2015, Schwarzländer et al. -Dissecting Redox Biology Using Fluorescent
Protein Sensors"> Schwarzländer <I>et al.</I> (2015)</ABBR></A>.This property makes it interesting to further usage for the iGEM community.


In the future we want to expand our toolbox with an ATP and NADP<SUP>+</SUP> sensor. Both sensors are FRET (Förster Resonance energy Transfer) based sensors. They consist of two coupled fluorescence protein and a ligand- sensing domain. FRET is a distant depend process by which energy transferred from an excited donor fluorophore to an acceptor molecule which is mostly a fluorophore as well. The NADP<SUP>+</SUP> sensor consists of two fluorophore proteins CFP and YFP and a indicator protein KBR. In the presence of NADP<SUP>+</SUP>, the distance between the two fluorophores is increased because of a conformational change of the sensing protein KBR. Exciting these complex by 440 nm results in a emission spectra with peaks at 478 nm and 526 nm. Thus, the 526/478 ratio between these wavelengths changes due to different NADP<SUP>+</SUP> concentrations<A href="https://www.ncbi.nlm.nih.gov/pubmed/26524720"><ABBR title="2015, Feng-Lan Zhao et al. -A genetically encoded biosensor for&lt;i&gt; in vitro&lt;/i&gt; and &lt;i&gt;in vivo&lt;/i&gt; detection of NADP&lt;sup&gt;+&lt;/sup&gt;"> Feng-Lan Zhao <I>et al.</I> (2015)</ABBR></A>.

ATP will be measureable using a Fret based ATP-sensor which consists of the two Fluorescent Proteins CFP and mVenus, derived from the YFP, which are both linked to the 𝜺 subunit of the F0F1-ATP synthase. Upon binding of ATP to the 𝜺 subunit a conformational change happens, which is detectable through fluorescence ratio measurements<A href="http://www.pnas.org/content/106/37/15651.full"><ABBR title="2009, Hiromo Imamura et al.-Visualization of ATP levels inside single living cells with fluorescence resonance energy transfer-based genetically encoded indicators"> Hiromo Imamura <I>et al.</I> (2009)</ABBR></A>. </P></FIGURE></FIGURE></FIGURE></FIGURE></DIV><BUTTON class="accordion"><H2 id="Nootkatone">Nootkatone</H2><P>After successfully integrating all our plasmids into yeast, we were able to verify the expression of each enzyme of the nootkatone pathway. </P></BUTTON><DIV class="panel"><P>In order to verify the cytosolic expression of ValS, BM3 and ADH we performed a Western Blot analysis for each of the enzymes. We were able to verify the expression of ValS, BM3 and ADH with and without PTS1 in the yeast cytoplasm and peroxisomes, respectively.</P><FIGURE><FIGCAPTION><STRONG>Figure 7.1 </STRONG> Protein abundance in WT and transformed cells in S. cerevisiae: Protein abundance was detected using 6xHis Tag Antibody in whole cell lysates. WT = wild type, ValS = Valencene Synthase, PTS1 = Peroxisome Targeting Signal 1, BM3 = Cytochrome P450 from Bacillus megaterium, ADH = Alcohol Dehydrogenase </FIGCAPTION></FIGURE><P>Since protein abundance of ValS, BM3 and ADH, both with and without a peroxisome targeting signal, was verified in the cytosol, a mass spectrometry analysis (MS analysis) of nootkatone and its precursor valencene was performed.
There are three approaches in MS analysis. The first one is the qualitative approach in which is only determined if the substance is present or not. The second and third kinds are the quantitative or semi-quantitative approach in which the absolute or relative amount of a substance is investigated.
First, we tried to validate nootkatone and valencene with the first approach and screened our samples for the existence of the first intermediate valencene and our final product nootkatone.

We could not show the synthesis of nootkatone nor valencene in our yeast yet, but we could smell its characteristic scent. The lack of proof via MS could result from an inefficient sample extraction or to low concentrations of product in the sample. The latter could also explain a peak in the MS analysis where nootkatone was expected. Unfortunately the peak was below detection limits and therefore can not be assumed to be a definite proof of nootkatone production. </P><H3>Outlook</H3><P>For further investigation we plan to do a semi-quantitative analysis by comparing the yield of samples of cytosolic synthesized nootkatone and peroxisomal nootkatone. To do so we need to perform a peroxisome purification in order to compare the amount of product produced. With this comparison we hope to proof that compartmentation, and thereby bypassing the problem of toxicity of substances for yeast, is the key for better yield of nootkatone. Also we intend to do a quantitative MS analysis to clarify if the yield of our nootkatone pathway is anywhere near the yield pathways with other enzymes/ enzyme-combinations could achieve. There was also the idea of exchanging cytosolic enzymes with membrane bound ones to see if there is any change in yield.</P><P> Already planned but not implemented, due to lack of time, we have also a proof of localization of the pathway enzymes. Therefore we exchange the 3a part <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#yeast-toolbox">(Dueber Toolbox)</A> 3xFlag/6xHis of the plasmid with an other fluorescent 3a part, namely mRuby2. We can then show the localization of the enzymes via microscopy. 
 
</P><P>Another factor to be considered in further studies will be the available amount of FPP in peroxisomes. FPP is the essential precursor of nootkatone synthesis. But we cannot say yet if there is enough FPP in the peroxisome to justify an expression of the pathway in it. If there is not enough FPP available to generate nootkatone over the concentration of  100 mg/L it does not matter that beta-nootkatol and nootkatone is toxic to the cell. </P><P>To tackle this problem there are three methods reported to increase the amount of FPP in the peroxisome for nootkatone production. The first one is to introduce a knockout mutation of squalene synthase and obtaining a mutant that is capable of efficient, aerobic uptake of ergosterol to limit the use of FPP for the sterol biosynthesis. The second approach is to knock out a phosphatase activity to limit the endogenous dephosphorylation of FPP. Third is to upregulate the catalytic activity of HMGR.
<A href="https://www.ncbi.nlm.nih.gov/pubmed/17013941"><ABBR title="2007, Takahashi et al. - Metabolic engineering of sesquiterpene metabolism in yeast.">
   	Takahashi <I>et al.</I> (2007)
</ABBR></A>.</P></DIV><BUTTON class="accordion"><H2 id="Violacein">Violacein</H2><P>
In the following the results of the essential experiments will be presented and discussed. The expression of the enzymes could be proven with SDS PAGE and western blot experiments. The following conducted <I>in vitro</I> assay was analyzed via HPLC-MS.
</P></BUTTON><DIV class="panel"><H3>Violacein</H3><P>
The experiments were implemented following the protocols for 
<A href="https://static.igem.org/mediawiki/2017/8/8a/T--Cologne-Duesseldorf--western-blot-protocol.pdf">western blot</A>
and
<A href="https://static.igem.org/mediawiki/2017/a/aa/T--Cologne-Duesseldorf--prodeoxyviolacein_assay.pdf"><I>in vitro</I> assay prodeoxyviolacein</A>.
</P><FIGURE><FIGCAPTION><STRONG>Figure 8.1</STRONG> Western blot analysis of Vio enzyme expression in yeast lysate with anti-His-antibody. The cell cultures were harvested and cells were lysed at the following OD<SUB>600</SUB>: wild type (WT) control: 1.4; VioA_pts: 1.13; VioA: 1.49; VioB: 1.37; VioB_pts: 1.5; VioE: 1.17. The expressed enzymes have the following predicted molecular weights: VioA_pts: 48.9 kDa; VioA: 47.7 kDa; VioB: 112 kDa; VioB_pts: 113.4 kDa; VioE: 22.7 kDa. The pictures are assembled for better analysis, each panel was merged with the protein ladder to allow exact comparison.
</FIGCAPTION></FIGURE><P>The level 1 constructs show the predicted molecular weight, whereas the different intensities of the protein bands correlate with the different optical densities (ODs) and different expression levels in the yeast culture replicates. The unspecific bands in VioA, VioA_pts and VioB_pts have a lower molecular weight than our predicted bands. One reason for this might be to protein degradation by carboxy exonuclease activity
<A href="https://advansta.com/wikis/multiple-bands-in-western-blots-causes-and-solutions-2/"><ABBR title="Multiple Bands in Western Blots – Causes and Solutions">
(Hurley A, 2017)
</ABBR></A>.
The protein extracts of VioA and VioA_pts were frozen before continuing the SDS PAGE on the next day. This, as well as poor handling of samples can lead to degradation. Also the liquid culture of VioB_pts grew over two days to reach our desired OD<SUB>600</SUB>, however the culture may already have reached stationary phase.
To decrease protein degradation in the future, protease inhibitors should be added to the lysis solution and all samples should be kept on ice.
 For further analysis of the enzyme functionality an assay followed by HPLC-MS analysis was implemented.
</P><FIGURE><FIGCAPTION><STRONG>Figure 8.2</STRONG> Overlaid extracted ion chromatograms (EIC) m/z 312.1131 of cell suspension 1 (CS1) time course experiment analyzed via LC-MS (Dionex Ultimate 3000, Thermofisher, USA; Maxis 4G, Bruker, Germany). From this given sample, $10\,\mu l$ were injected and measured in positive ionization mode. The signal intensity is given in counts per second. The section from 5.2 min to 6.2 min is shown. The peaks are corresponding to samples of the time course experiment taken at t=0 min, t=30, t=60, t=90 and t=120 past reaction start. The coloring of the peaks is increasing over time starting at light green for t=0 min to dark green for t=120 min. The prodeoxyviolacein (PDV) peak is shifting over time between 5.62 min from samples taken at 30 min after reaction start up to 5.57 min from samples taken at 120 min after reaction start. The counts increase continuously from about 0.01x10<SUP>5</SUP> to about 0.9x10<SUP>5</SUP>.
</FIGCAPTION></FIGURE><P>
Figure 8.2 shows the results of the mass spectrometry analysis of the PDV <I>in vitro</I> assay. Over a period of 120 min, samples were taken every 30 minutes. The cell suspension reaction mixture 1 (CS1) shows increase of the PDV production over time. PDV has a molecular weight of 312.1131, confirming the peak as the possible expected molecule. The mass spectrometry analysis of the wild type control did not show any peaks at the retention time of the potential PDV (data not shown).
The shown data is from the cell suspension reaction. The LC-MS signals obtained with extracts from the protein suspension were too low to identify any possible molecule. A possible reason is that in contrast to the cell suspension, the protein extract lacks cofactors we did not consider in our master mix. Although the protein extraction was done precise and well-planned, we cannot guarantee a native protein state, which is needed for the enzymes to catalyze the reaction. Also the second reaction mix, containing a higher VioB concentration did not show a higher PDV production (data not shown). VioB is supposed to be the limiting factor of the reaction
<A href="https://www.ncbi.nlm.nih.gov/pubmed/17176066"><ABBR title="In vitro biosynthesis of violacein from L-tryptophan by the enzymes VioA-E from Chromobacterium violaceum">
(Balibar CJ <I>et al.</I>, 2006),
</ABBR></A>
therefore we did the second reaction mixtures with a higher concentration of VioB.
</P><P>To identify PDV as accumulating compound at a retention time of about 5.6 min in LS-MS analysis, MS/MS experiments with standards were conducted afterwards. For further identification of the accumulating compound, fragmentation experiments are essential to exclude the accumulation of other compounds with the same molecular weight. Comparing the potential compound with measurements of standards enables its identification.
</P><DIV class="max-width"><FIGURE><FIGCAPTION><STRONG>Figure 8.3</STRONG> Fragment spectra of violacein/deoxyviolacein standard mix and potential prodeoxyvioalcein measured by direct infusion MS and LC-MS (Dionex Ultimate 3000, Thermofisher, USA; Maxis 4G, Bruker, Germany). Highlighted sub molecular structures represent the corresponding masses caused by fragmentation of the parent ion. The mass differences between the highlighted peaks are caused by the loss of a carbonyl group or nitrogen resulting of the fragmentation. Similar masses between the fragment spectra are marked with yellow/green dotted lines. <STRONG>A</STRONG>: fragment spectrum of violacein standard measured with direct infusion. <STRONG>B</STRONG>: fragment spectrum of deoxyviolacein standard measured with direct infusion. <STRONG>C</STRONG>: fragment spectrum of potential PDV measured with LC-MS; sample CS1 from <A href="https://static.igem.org/mediawiki/2017/a/aa/T--Cologne-Duesseldorf--prodeoxyviolacein_assay.pdf"> prodeoxyviolacein <I>in vitro</I> assay </A>.
</FIGCAPTION></FIGURE></DIV><P><STRONG>Figure 8.3 </STRONG>shows the fragment spectra for violacein (<STRONG>A</STRONG>), deoxyviolacein (<STRONG>B</STRONG>) and PDV (<STRONG>C</STRONG>). To verify if the produced compound with a mass of 312.1132 is PDV, the fragment spectrum of this compound is compared with its structurally similar precursors violacein and deoxyviolacein. These compounds are commercially available. All compounds have in common that they loose CO (-28 Da), PDV based on its structure only one CO-group. All show the loss of 15 Da, corresponding to Nitrogen. Deoxyviolacein and PDV share the same indole system. Both show peaks at m/z 143 Da and 167 Da. Violacein does not show these signals lacking this indole ring. On the other hand violacein and deoxyviolacein share the same oxo-indole ring, resulting in a signal of 211 Da.

This measurements and analysis of the <I>in vitro</I> prodeoxyvioalcein assay prove the functionality of the enzymes VioA, VioB and VioE which are necessary to produce PDV from L-tryptophan (<A href="https://static.igem.org/mediawiki/2017/0/09/T--Cologne-Duesseldorf--Violacein_Pathway_komplett.png">violacein pathway</A>). The results therefore show, that the integration of the bacterial pathway into <I>Saccharomyces cerevisiae</I> has been successful. 
<FONT size="2">
Many thanks to Felix Büchel and the MS platform, Cologne for extraordinary support.</FONT></P><H4>Outlook </H4><P> Our outlook is characterized by the vision to use our own modeled PTS* import sequence for a real world application. But first we want to further investigate our <I>in vitro</I> strategy including the missing enzymes VioC and VioD and move on towards already developed <I>in vivo</I> tests. It would be great to qualify a statement about the efficiency of different import combinations into the peroxisome. To do so, different level 2 plasmids were planned. On each plasmid the combination of enzymes being peroxisomal or cytosolic is different.
</P><FIGURE><FIGCAPTION><STRONG>Figure 8.4 </STRONG>To each one of the five pathway enzymes a peroxisomal targeting signal (PTS1) can be added, leading to the import of this enzyme. For example in the left panel of this figure, a yeast cell with the import of VioE is pictured, whereas VioA, VioB, VioC and VioE remain cytosolic. Depending on PTS1 being attached to the enzyme or not, 20 different enzyme combinations are possible. Also more than one enzyme can be imported into the peroxisome as you can see in the middle or right panel. It is known that some intermediates can pass the peroxisomal membrane including its precursor L-tryptophan
<A href="http://www.nature.com/articles/ncomms11152"><ABBR title="Towards repurposing the yeast peroxisome for compartmentalizing heterologous metabolic pathways">
(John E. Dueber <I>et al.</I>, 2015)
</ABBR></A>
.
</FIGCAPTION></FIGURE><P>
To assure the import of the enzymes and for further analysis it is indispensable to perform peroxisomal purification for an overall quanti- and qualification. Also measurements of the pathway intermediates and their fluxes across the peroxisomal membrane have to further be analyzed. The final step would be the comparison of the differences in the yield level, depending on the localization of the pathway enzymes (cytosolic or peroxisomal). With this the assumed better production in peroxisomes could be shown.
</P></DIV><BUTTON class="accordion"><H2 id="optogenetics">Optogenetic enhancements</H2><P>
Here you can find our results and discussion regarding the optogenetics subtopic
</P></BUTTON><DIV class="panel"><H3>Pex5 import with LOV2</H3><P>
Our GFP-LOV-PTS1 construct was successfully cloned and transformed into <I>S. cerevisiae</I>. As part of our lightbox experiment, GFP-fluorescence was observed throughout the cells in both, the illuminated sample and the dark control, indicating unsuccessful import (data not shown here). 
</P><H3>Pex7 import</H3><P>
All three constructs of our Split-TEV PTS2 subproject have been successfully cloned to level 1 in regards to the <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#yeast-toolbox">(Dueber Toolbox)</A>. A level 2 plasmid containing the PhyB-TEV2 and TEV1-PIF6 constructs is required in order to transform all constructs into <I>S. cerevisiae</I>. This has not been created so far.
</P><H3>Optogenetically controlled gene expression</H3><P>
The TetO-Pmin promoter construct has been brought to level 1 with <I>mRuby</I> and <I>Pex11</I> as genes of interest. The TetR-PIF6 construct has also been brought to level 1. The PhyB-VP16 construct has not been successfully integrated into the Dueber toolbox.
</P><H3>Outlook</H3><P>
The variability of our compartment toolbox could be greatly increased by using optogenetics. We planned on using constructs suited for optogenetic control of protein import via both pathways as well as constructs designed for optogenetically controlled gene expression. Even though we did not get to finish our work  on this sub project, we still want to underline its importance for future applications and improvements of our toolbox. 

As mentioned above, we were not successful in demonstrating optogenetically controlled protein import via PTS1.
Our theory here is that the LOV2-variant obtained from <I>Avena sativa</I> does not correctly function in <I>S. cerevisiae</I>, possibly due to the different cytosolic conditions, such as pH, ion- or enzyme concentrations. Future tests would involve using the LOV2-variant from <I>Arabidopsis thaliana</I>.
Another aspect we considered was the possibility of steric inhibition of the protein of interests function by the LOV2 attached to it. A future approach for solving this problem would be to add a TEV-protease cleavage site between the protein of interest and the LOV2-protein. The corresponding TEV-protease could be fitted with a PTS, leading to cleavage of the fusion protein upon it being imported into the compartment. 

Due to a lack of time we were unable to test our TEV-protease construct, as we did not finish the cloning process. However, we think that upon further development of the toolbox it is an aspect which should be considered, especially since there is only one more cloning step which needs to be completed in order for it to be eligible for transformation into <I>S. cerevisiae</I>.

The optogenetic control of our <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#Secretion">secretion mechanism</A> via gene expression also still awaits testing due to unfinished cloning. If successful, it would enable secretion of our compartments content within a few hours after illumination. 
Another approach we have not pursued yet is attaching the vSNARE-proteins we are using to PIF6 and insert Phytochrome B into the peroxisomal membrane via our Pex26 anchor (see <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Results#MembraneIntegration">membrane integration</A>). In theory, illumination with red light would then lead to instant secretion. 
</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>