<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cologne-Duesseldorf_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cologne-Duesseldorf/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="sidebar"><UL><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf">Home</A></LI><UL class="sub"><A href="#!">Team</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Collaborations">Collaborations</A></LI></UL><UL class="sub"><A href="#!">Project</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Attributions">Attributions</A></LI></UL><UL class="sub"><A href="#!">Parts</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Basic_Part">Basic Parts</A></LI></UL><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Safety">Safety</A></LI><UL class="sub"><A href="#!">Human Practices</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Engagement">Public Engagement</A></LI></UL><UL class="sub"><A href="#!">Awards</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Plant">Plant</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Software">Software</A></LI></UL><LI><A href="https://igem.org/2017_Judging_Form?team=Cologne-Duesseldorf">Judging Form</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Wiki-Tutorial">Wiki Tutorial</A></LI></UL></DIV><NAV><UL class="menu"><LI class="menu-item-has-children"><A href="#!">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Collaborations">Collaborations</A></LI></UL><LI class="menu-item-has-children"><A href="#!">Project</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Attributions">Attributions</A></LI><LI class="menu-item-has-children"><A href="#!">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Safety">Safety</A></LI><LI class="menu-item-has-children"><A href="#!">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Engagement">Public Engagement</A></LI><LI class="menu-item-has-children"><A href="#!">Awards</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Software">Software</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Cologne-Duesseldorf">Judging Form</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Wiki-Tutorial">Wiki Tutorial</A></LI></NAV><ARTICLE><H1>Demonstrate</H1><H2>Pex5 Import</H2><P>The achievement of an orthogonal import was a huge step towards an artificial compartment that could be utilized for all kinds of synthetic biological applications. The relocalization of pathways into an isolated space benefits biotechnological production, opens new doors and enriches possibilities.
                
                Although, we could not demonstrate our concept by fully relocating complex metabolic pathways, importing fluorescent proteins is already a solid proof of concept. Our results show that the transport of fluorescent markers such as mTurquoise work really well, assuming our new import machinery to not alter the function of the cargo of interest. Thus, we provide that any protein of interest can be shifted fully intact into the peroxisome via our import system. This orthogonality emphasizes the importance of this system for synthetic biology as it contributes to the engineering of a synthetic cell. 

The most important difference is that we conquered this big challenge by converting the cells functional systems from the inside. But what is of far more importance: the possibility of this system to <STRONG>fight world threatening diseases</STRONG>. Our system makes the proteome of the peroxisomes accessible and engineerable. Due to the natural function of peroxisomes as a stress-resistant compartment our import machinery enables the shift of metabolic pathways from the cytosol into the peroxisome and by this increases the productivity, yield and efficiency of the pathway’s end product. We chose nootkatone and violacein as exemplary pathways to demonstrate the potential of our project. Our system is not limited only for producing drugs, it is rather a versatile tool to efficiently maintain the toxicity and limited <I>in vivo</I> production of compounds from many different branches of industry. 

As an outlook, we predict major yields for the two products, nootkatone, violacein and many more compounds of interest once we relocate their pathways into the peroxisome.
            </P><H2>Pex7 import</H2><P>We archived to generate various PTS2 sequences showing different import efficiencies. With this valuable tool one is able to alter protein concentrations in the peroxisome to further customize the artificial compartment. Essential proteins as the ubiquitin binding proteins regulate stability and localisation of ubiquitinated proteins 

<A href="https://www.ncbi.nlm.nih.gov/pubmed/16564012 "><ABBR title="Reyes-Turcu, F. E., Horton, J. R., Mullally, J. E., Heroux, A., Cheng, X., &amp; Wilkinson, K. D. (2006). The ubiquitin binding domain ZnF UBP recognizes the C-terminal diglycine motif of unanchored ubiquitin. Cell, 124(6), 1197-1208.">(Wilkinson)</ABBR></A>.

The C-terminal domain of the protein interacts with the ubiquitinated protein. To establish a pathway containing suchlike proteins in the compartment they must not be modified on their C-terminus. Nevertheless we can import them into the compartment with the modified PTS2 sequences. </P><H2> Secretion</H2><P>Using microscopy we were able to show, that our membrane anchors localize in a typical peroxisomal pattern <A href=" http://jcs.biologists.org/content/119/12/2508"><ABBR title=" Halbach, A., Landgraf, C., Lorenzen, S., Rosenkranz, K., Volkmer-Engert, R., Erdmann, R., &amp; Rottensteiner, H. (2006). Targeting of the tail-anchored peroxisomal membrane proteins Pex26 and Pex15 occurs through C-terminal Pex19-binding sites. Journal of cell science, 119(12), 2508-2517.

">
                (Halbach <I> et al</I>, 2006)
                </ABBR></A>.
                The results of the <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#Secretion"> GUS-assay </A> indicate, that the contents of the peroxisomes were successfully secreted into the supernatant. This is the first time it was shown that this system works in <EM>S. cerevisiae</EM>, since this point it has only been demonstrated in <I>Aspergillus niger</I>. 
                Even though our secretion is not as efficient as the unconventional secretion in other organisms <A href=" https://www.ncbi.nlm.nih.gov/pubmed/22446315"><ABBR title="Stock, J., Sarkari, P., Kreibich, S., Brefort, T., Feldbrügge, M., &amp; Schipper, K. (2012). Applying unconventional secretion of the endochitinase Cts1 to export heterologous proteins in Ustilago maydis. Journal of biotechnology, 161(2), 80-91.">(Stock <I> et al</I>, 2012)</ABBR></A>, the possibility to secrete the content of our artificial compartment is still a substantial success with exciting implications. One could, for example, not only secrete proteins but also compounds from metabolic pathways.
                Here we have only shown our general proof of concept. The next steps would be to develop a more efficient system. An easy way to increase the yield of compounds in the supernatant is to manipulate the <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#Size And Number"> size and number</A> of the artificial compartments in the cell. 
                Due to the fact, that peroxisomes are linked to the endoplasmic reticulum (ER) in <EM>S. cerevisiae</EM> cells, the efficiency of our system could be inhibited. To overcome this problem one possible solution is to delete <EM>INP1</EM> in the background strain. Inp1 works as a molecular link between peroxisomal Pex3 and ER Pex3. Deletion of <EM>INP1</EM> leads to mobile peroxisomes leading to more fusion events.  
                After the generation of an optimized background strain, the next goal would be stable genome integration of the secretion system. One of the problems of this integration is the constitutive expression of the snare constructs. This would lead to a constant secretion and loss of the proteins  in our compartment. For an optimal yield it would be better to use inducible promoters to control the secretion of the produced metabolites. Another approach could be to control the fusion of the compartment and the cell membrane by <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#Optogenetics"> optogenetics</A>. 
            </P><H2>Membrane Integration</H2><P>Membrane Integration
Up until this point the integration of native and foreign membrane proteins into the peroxisome isn't fully understood but over the course of our project we were able to use the knowledge available to create a useful extension for our tool box. Not only can we integrate proteins into our compartment, we are also able to choose whether we want to C or N-terminally anchor our protein into the membrane. 
The ultimate goal of this subproject is, to have a complete set of ready to transform membrane proteins that could be combined with any promoter to create the optimal conditions for each desired situation. So far we managed to introduce two marker proteins and the proton pump bacteriorhodopsin to our membrane. This was a major step for our project, since a lot of proteins require specific environmental conditions and the wide variety of membrane proteins that have been characterized so far are only the beginning. To push the subproject further we also started to work with sugar translocators, since yeast does not posses the ability to import it into or export it from the peroxisome. This would open up a whole new chapter of peroxisomal usage, from example as a temporary storage compartment.</P><H2>Size And Number</H2><H3>Pex11</H3><P>Data showed that deletion and overexpression of Pex11 in the cells did not significantly affect the peroxisome morphology. We could not detect the expected differences in the size and number.

The structure as well as the function of Pex11 is not well characterized and understood so that we cannot predict how the fusion of Venus to the N-terminal end of the protein affects the natural function. It is possible that it leads to inhibition of their ability to promote peroxisomal fission.

Furthermore, Pex11 concentration in the cell is naturally regulated by oleic acid in the medium. We cultivated all yeast cells on glucose containing medium, to modify the peroxisome without harming the cells. This could be another reason why our construct did not work.
</P><H4>Outlook</H4><P>To detect if its fusion with the Venus protein is responsible for these results, further experiments should be conducted to express Pex11 without fluorescence tags in the cell and measure the peroxisome phenotype by co-expressing sfGFP-PTS1.

Expression of Pex11 did not show the expected results. Following the kind advice of Dr. Florian David from Biopetrolia, we further decided to work with a Pex34 overexpression in order to positively influence our compartments’ size and number and conclusively achieve a higher yield by creating a larger microenvironment for planned reactions.
</P><H3>Pex34</H3><P>We were able to show that overexpression of pex34 in wild type yeast cells leads to a change of the peroxisomal morphology.
In the first approach, the fluorescent protein mTurquoise was fused to the c-terminus of Pex34 in order to visualize the protein in the cell.
Microscopy of the sample in which this construct was expressed via the strong constitutive promoter ScCCW12 reveals a high number of small peroxisomes in each cell. This only partly represents the expected phenotype: the expected outcome was a high number of larger peroxisomes.  One explanation for this deviation is the possibility of the peroxins’ natural function being partly inhibited by the fluorescent protein fused to it.

The second approach focused on coexpressing an unmodified Pex34-sequence with sfGFP-Pts1. Through resulting accumulation of GFP in the peroxisomes, the phenotype could be visualized without alterations to the peroxins’ native structure and function.

Images were taken from cultures with the <I>PEX34</I> being expressed using the strong promoter ScCCW12 as well as from cultures with a medium-strength copper inducible promoter.
A change of the phenotype is visible here as well: the cells contain a noticeably higher number of peroxisomes, which are also larger than usual. This demonstrates the effects of Pex34 on the cell. The higher its concentration in the cell, the stronger it affects the peroxisome-morphology.
Some cells showed elongated peroxisomes as well as string-like membrane-fragments. This is a result of Pex34's function of recruiting membrane lipids from the ER. These membrane appendages were visualized.

With this subproject we were able to show that we have created a useful tool for affecting the morphology of peroxisomes and change their size and number. This could be used, for example in metabolic engineering, in order to increase yields. The inducible copper-promoter can be used to dynamically adjust the proliferation.
</P><H4>Outlook</H4><P>
Pex34 provides a useful way of adapting the morphology of peroxisomes to the application at hand. Further, more precise measurements would need to be made in order for us to be able to make quantitative predictions.
</P></ARTICLE></DIV></DIV></DIV></DIV></DIV></BODY></HTML>