<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cologne-Duesseldorf_EricDesign skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cologne-Duesseldorf/EricDesign</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="sidebar"><UL><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf">Home</A></LI><UL class="sub"><A href="#!">Team</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Collaborations">Collaborations</A></LI></UL><UL class="sub"><A href="#!">Project</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Attributions">Attributions</A></LI></UL><UL class="sub"><A href="#!">Parts</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Basic_Part">Basic Parts</A></LI></UL><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Safety">Safety</A></LI><UL class="sub"><A href="#!">Human Practices</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Engagement">Public Engagement</A></LI></UL><UL class="sub"><A href="#!">Awards</A><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Plant">Plant</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Software">Software</A></LI></UL><LI><A href="https://igem.org/2017_Judging_Form?team=Cologne-Duesseldorf">Judging Form</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Wiki-Tutorial">Wiki Tutorial</A></LI></UL></DIV><NAV><UL class="menu"><LI class="menu-item-has-children"><A href="#!">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Collaborations">Collaborations</A></LI></UL><LI class="menu-item-has-children"><A href="#!">Project</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Attributions">Attributions</A></LI><LI class="menu-item-has-children"><A href="#!">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Safety">Safety</A></LI><LI class="menu-item-has-children"><A href="#!">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/HP/Gold_Integrated">Integrated Practices</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Engagement">Public Engagement</A></LI><LI class="menu-item-has-children"><A href="#!">Awards</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Software">Software</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=Cologne-Duesseldorf">Judging Form</A></LI><LI><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Wiki-Tutorial">Wiki Tutorial</A></LI></NAV><ARTICLE><H1>Design</H1><P>We designed a novel toolbox for complete control over all major functions of the peroxisome. The toolbox is our solution for improving the engineering workflow and predictability of synthetic constructs. Interested? Find out how.</P><H2 id="Background">Scientific background</H2><H3 id="RootProblem">The root problem</H3><P>Synthetic biology is an engineering discipline. And while we are able to plan our constructs with tools like biobricks, a major difference to e.g. electrical engineering is that they are not nicely isolated on a chip, but surrounded by all types of interfering agents. One of the major issues regarding protein expression in a novel chassis is unwanted and unexpected crosstalk between engineered pathways and the native cellular processes of the production host. The other one is toxicity of the products or intermediates of the pathway. Both can greatly change our system’s behaviour which in some cases leads to us to having to trial-and-error find a solution, making our previously modeled optimization useless. </P><H3 id="Approach">Our approach</H3><P>The natural approach of organisms to deal with metabolic interference and toxic byproducts is subcellular compartmentalization. This has proven to be a functional solution in naturally occurring pathways in eukaryotes as well as new synthetic pathways for biotechnological application. Thus, the creation of a synthetic organelle presents a suitable strategy to increase the efficiency and yield of non-native pathways. A common approach is to build up artificial compartments from scratch. Many breakthroughs have been achieved in the last decade, however the creation of a fully synthetic compartment is yet a milestone to reach for. We on the contrary want to start by engineering artificial compartments through orthogonalization.</P><P>The peroxisome is the ideal starting candidate as it has many advantages over other compartments, including it being able to import fully folded proteins and not being essential in yeast under optimal growth conditions. Our project’s aim is to create a toolbox for manipulating and creating customizable peroxisomes as a first step towards synthetic organelles.</P><P>By modifying the import machinery of yeast peroxisomes only selected proteins will be imported into the peroxisomes leaving the researcher in full control over the content of the peroxisomal lumen. Furthermore our toolbox will include a secretion mechanism for the synthesized products, various intra-compartmental sensors, modules for the integration of proteins into the peroxisome membrane, as well as optogenetic control for some of these parts for a more precise spatiotemporal control.</P><P>As a proof of concept for the functionality of the toolbox and the customizable compartment two metabolic pathways will be integrated into the altered peroxisome: (i) Violacein biosynthesis and (ii) nootkatone biosynthesis. Violacein, a bisindole formed by condensation of two tryptophan molecules, is a violet pigment and thus easy to quantify in the cell. Nootkatone on the other side is a natural compound found inside the peel of the grapefruit, which gives it its characteristic taste and smell. In addition, nootkatone is a natural mosquito and tick repellent that is already being commercially used and industrially manufactured. Unfortunately, the production costs are extremely high. Furthermore, the production of nootkatone inside yeast is challenging as it is toxic for yeast and thus, the production efficiency is rather low. A successful implementation of the synthetic compartment will show increased yields in the production of these compounds and showcase the potential of this approach for similar future applications.</P><H2 id="CloningStrategies">Cloning strategies and the Yeast Toolbox for Multipart-Assembly</H2><P>While describing our cloning strategies we mentioned several <I>levels</I>, which stand for different stages of our plasmids. They are further described in the work of J.M. Dueber and colleagues, who designed the well established yeast toolkit we used in this project  

  <A href="http://pubs.acs.org/doi/abs/10.1021/sb500366v "><ABBR title="Lee, M. E., DeLoache, W. C., Cervantes, B., &amp; Dueber, J. E. (2015). A highly characterized yeast toolkit for modular, multipart assembly. ACS synthetic biology, 4(9), 975-986.">
        (Dueber)</ABBR></A>. 
The toolkit offers the possibility to design plasmids with desired antibiotic resistances, promoters as well as terminators from standardized parts. It also provides fluorescence proteins, protein-tags and many more useful components as part plasmids. These part plasmids are distinguished in different part types due to their specific overhangs to ensure their combination in the correct order (e.g. promoter - gene of interest - terminator) all in a versatile one-pot Golden Gate reaction without time-consuming conventional cloning steps.</P><DIV class="flex-row-2&gt;
&lt;div&gt;&lt;img src=" https:=""><DIV><FIGURE><FIGCAPTION><B>Figure 1: The yeast toolkit starter set comprises of 96 parts and vectors.</B> The eight primary part types can be further divided into subtypes. 

<A href="http://pubs.acs.org/doi/abs/10.1021/sb500366v "><ABBR title="Lee, Michael E., et al. A highly characterized yeast toolkit for modular, multipart assembly. ACS synthetic biology 4.9 (2015): 975-986.">(Dueber)</ABBR></A></FIGCAPTION></FIGURE></DIV></DIV><P>The cloning steps regarding the plasmid <I>levels</I> are implemented in <I>E.coli</I> in order to reduce the required time to generate the final plasmids. The different <I>levels</I> are therefore defined by their part content and their antibiotic resistances. </P><P>To generate a <I>level 0</I> plasmid, the gene of interest is ligated into the provided <I>level 0</I> backbone via Golden Gate assembly using the enzyme BsmBI. The backbone contains a resistance to Chloramphenicol, as well as an origin of replication, creating a very basic yet functional plasmid.</P><P>The <I>level 1</I> plasmid contains a promoter and terminator suited for <I>S. cerevisiae</I>. There is the possibility of including  a polyhistidine-tag if there is a need for Western blot analysis. The antibiotic resistance contained in the <I>level 1</I> plasmid changes from chloramphenicol to ampicillin which enables filtering out residual <I>level 0</I> plasmids contained in the Golden Gate product. Furthermore, the <I>Dueber toolbox</I> includes the possibility of designing GFP-Dropout cassettes. These are custom-built <I>level 1</I> backbones whose inserts are sfGFP as well as promoter and terminator suited for <I>E. coli</I>.  Upon a successful cloning step the GFP is replaced by the part(s) of interest, and correct colony shows a white colour. In case of a wrong ligation event colonies show a green fluorescence. This provides a very useful tool to detect unsuccessful cloned colonies. The enzyme used for <I>level 1</I> changes from BsmBI to BsaI to avoid any interference between different steps. </P><P>The <I>level 2</I> plasmid combines two or more genes of interest with their respective promoters, terminators and tags. The resistance changes from ampicillin to kanamycin. The enzyme of this step is BsmBI again. This level is useful, if the construct you are designing requires multiple genes to be transformed into one yeast strain.</P><H3> Yeast nomenclature </H3><P>To make it fast and easy to differentiate between endogenous and heterologous genes and gene products we decided to use <I> S. cerevisiae </I> nomenclature according to <A href="http://seq.yeastgenome.org/sgdpub/Saccharomyces_cerevisiae.pdf">yeastgenome.org</A>. </P><P>Below nomenclature at the example of <B><U>y</U>our <U>f</U>avorite <U>g</U>ene 1, <I>YFG1</I></B> is explained. </P><TABLE><TBODY><TR><TH>Letter code</TH><TH>Meaning</TH></TR><TR><TD><I>YFG1</I></TD><TD><B>Y</B>our <B>f</B>avorite <B>g</B>ene <I>S. cerevisiae</I> wild type allele</TD></TR><TR><TD><I>yfg1</I>Δ</TD><TD>Gene deletion of <B>y</B>our <B>f</B>avorite <B>g</B>ene</TD></TR><TR><TD>Yfg1</TD><TD>Protein product of <I>YFG1</I></TD></TR></TBODY></TABLE></ARTICLE>YFG2A heterologous gene product from mammalian cells<H2>Design of our sub-projects</H2><BUTTON class="accordion"><H2 id="ProteinImport">Protein Import</H2><P>Control over the peroxisomal proteome is an essential requirement for our project. Our objective is the obtainment of a fully orthogonal import system by utilizing the two peroxins Pex5 and Pex7.</P></BUTTON><DIV class="panel"><H3 id="Pex5">Engineering of Pex5 and PTS1</H3><H4 id="h5-1">Designing our receptors<P>To achieve the engineering of an orthogonal import pathway, we followed two approaches regarding Pex5. The first is based on targeted mutagenesis based on educated guesses which is first verified by <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model#h2-1">molecular dynamics</A> and later experimentally in the laboratory.
   The second approach is based on a recently published paper: We searched for literature dealing with the modification of the peroxisomal import machinery. During our research we came across a <A href="https://www.nature.com/articles/s41467-017-00487-7">paper</A> of Alison Baker <I>et al.</I>, published in 2017, in which they present a synthetic construct of the Pex5 protein, partly <I>Arabidopsis thaliana</I> and partly <I>Physcomitrella patens</I>. Compared to the wild type Pex5, this one shows different binding affinities since it interacts with a PTS1* variant that does not interact with the wild type Pex5. Since the protein sequences of yeast's and plants's Pex5 differ quite a lot, we aligned both sequences to understand where the mutations were set. </P><FIGURE><DIV class="flex-row-2"><DIV><FIGCAPTION><STRONG>Figure 1.1:</STRONG> Alignment of the <I>Arabidopsis thaliana's</I> PEX5 and <I>Saccharomyces cerevisiae</I>Pex5
     </FIGCAPTION></DIV><DIV><FIGCAPTION><STRONG>Figure 1.2:</STRONG> Alignment of the yeast’s Pex5 with the Pex5 variant R19
     </FIGCAPTION></DIV></DIV></FIGURE><P>The alignment shows three red marked amino acids we changed in our receptor sequence. Interestingly, these mutations are located within the TPR motifs of our Pex5 protein and this persuaded us to try out this receptor, we call it R19. Due to lack of time we tested this Pex5 variant in silico and <I>in vivo</I> simultaneously. We started molecular dynamics simulations with a couple of PTS variants that we already tested with our previous designs − one of them was actually the variant they used in the paper (<I>YQSYY</I>). The details and results of our structural modeling can be found in the <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model#h2-1">modeling section</A>. 

Furthermore, we synthesized this variant and together with two receptors we designed based on educated guesses we got three receptors for our experimental work.</P></H4><H4>Experimental design<P>
   Verification of peroxisomal protein import was performed  by tagging the fluorescent protein mTurquoise with our designed PTS variants. Additionally, a peroxisomal membrane protein was used to ensure peroxisomal localization. For that reason, we chose the transmembrane domain of Pex13 tagged with the fluorescent protein mRuby.

 </P></H4><H5>Pex13−mRuby<P>We used the peroxisomal membrane protein Pex13 as a fluorescent marker − by just using the transmembrane domain of Pex13 with a short linker, we make sure that it has no influence on the peroxisomal features. To obtain a higher differentiation from mTurquoise, which we use for another construct, we chose to work with mRuby. Literature research revealed that such constructs have been tested before − <A href="https://www.ncbi.nlm.nih.gov/pubmed/15133130"><ABBR title="Peroxisomal Membrane Proteins Contain Common Pex19p-binding Sites that Are an Integral Part of Their Targeting Signals"> Erdmann<I> et al.</I> (2004)</ABBR></A> described a construct containing only PEX13<SUB>200-310</SUB> with a C-terminal GFP.</P><FIGURE><FIGCAPTION><STRONG>Figure 1.3:</STRONG> PEX13 construct with C-terminal mRuby.
     </FIGCAPTION></FIGURE></H5><H5>Pex5 variant<P>In order to achieve an orthogonal peroxisomal protein import machinery we used a Pex5 knockout yeast strain in which we transformed our artificial  Pex5 variant containing a modified PTS1 binding pocket. Our variation facilitates the detection of a non native PTS1 variant instead of the wild type PTS1. The construct contains a medium strength promotor, the Pex5 gene and a terminator. The remaining plasmid parts can be seen in the plasmid map below.
     </P><FIGURE><FIGCAPTION><STRONG>Figure 1.4:</STRONG> Pex5 gene variant.</FIGCAPTION></FIGURE></H5><H5>mTurquoise−PTS<P>
     Our approach for import verification is based on the fluorescent protein mTurquoise tagged with our modeled PTS variants. After several promotor tests with different strengths, we decided to express this construct only in low amounts, since this was the most suitable possibility to detect potential mTurquoise localization.
     
     Our construct is depicted in the figure below.
		</P><FIGURE><FIGCAPTION><STRONG>Figure 1.5:</STRONG> Fluorescent protein mTurquoise tagged with the PTS variant.</FIGCAPTION></FIGURE></H5><H5>Combination of our constructs<P>To combine our constructs, we cloned our PEX5 and mTurquoise constructs into a level 2 plasmid portrayed below.</P><FIGURE><FIGCAPTION><STRONG>Figure 1.6:</STRONG> Level 2 plasmid containing the Pex5 gene and the fluorescent protein.</FIGCAPTION></FIGURE><P>Subsequently,  co-transformation of the PEX13−mRuby plasmid and the level 2 plasmid was performed in order to verify peroxisomal colocalization.</P></H5><H5>PTS screening<P>Trusting on our targeted approach alone seemed risky − that is why we planned a PTS screening to find the most favorable PTS for our three receptors. <A href="https://www.nature.com/articles/ncomms11152"><ABBR title="Towards repurposing the yeast peroxisome for compartmentalizing heterologous metabolic pathways.">Dueber <I>et al.</I> (2016)</ABBR></A> used the Violacein assay for a similar purpose. They screened for the best PTS for the wild type receptor and were successful. Hence another subproject of our team is the integration of the Violacein pathway into the peroxisome (<A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#Violacein">Violacein</A>), we were already supplied with all necessary enzymes − VioA, VioB and VioE.</P><FIGURE><FIGCAPTION><STRONG>Figure 1.8:</STRONG> Pathway leading to the production of prodeoxyviolacein − the assay is based on the green color to identify colonies without functional import mechanism.
</FIGCAPTION></FIGURE><P>The figure above shows the principles of the assay. VioA and VioB are localized in the cytosol and lead to the production of the IPA imine dimer while VioE is tagged with a PTS1 variant. Successful import leads to white colonies whereas missing import results in green colonies due to the cytosolic production of prodeoxyviolacein.
     
     Our rests upon the following two plasmids which are co-transformed into yeast.</P><FIGURE><FIGCAPTION><STRONG>Figure 1.9:</STRONG> Plasmids used for Violacein assay. The left one carries the coding sequence for the Pex5 variant, VioA and VioB whereas the right one contains the gene for VioE plus the attached PTS1 variant.
       </FIGCAPTION></FIGURE><P>As shown above, we created one plasmid containing VioA, VioB and one of our Pex5 variants while the other plasmid only contained VioE. We then designed primers which bind to the VioE plasmid to amplify the whole plasmid except the terminator − random PTS1 variants were attached to VioE with the help of a random primer library. Following up, we did the ligation with the corresponding terminator and obtained a mix of several different VioE-PTS1 plasmids.
     
     After plasmid amplification in <I>Escherichia coli</I> we then co-transformed yeast with the two constructs and waited for the colonies to grow. With the yeast growing, prodeoxyviolacein should be produced in yeast cells with absent import (green color) and the IPA imine dimer (white color) should be produced in those with functional import.

Plasmid preparation of those with white color and subsequent sequencing leads to the identification of functional PTS1 variants. Afterwards, we repeat the cloning steps described before to obtain a mTurquoise−PTS1* construct and co-transform it with the corresponding Pex5 variant. Eventually, the correct localization of mTurquoise tagged with these PTS1 variants provides proof for its function.</P></H5><H3>Mutagenesis of PTS2<P>Imagine you need different protein concentrations in your artificial compartment. What to do? Take our modified PTS2 sequences with varying import efficiencies. </P><P>To characterize the import efficiency for the site-directed PTS2 firefly luciferase was used. Luciferase is a luminescent protein which can be split in a C- and a N-terminal part. Only when combined, luminescence can be detected. To measure the import efficiency the two parts will be expressed and imported into the peroxisome in a separated way. The smaller part (Split2) of the split luciferase will be brought into the peroxisome first via the PTS1 dependent pathway. The other part is imported via the respective modified PTS2 sequence. The better this sequence is recognized by Pex7, the stronger the luminescence of the assembled luciferase can be detected in the peroxisome. There is a chance of split parts of the luciferase assembling in the cytosol if the import is too slow. To avoid wrong conclusions of the luciferase localisation, we designed a negative control experiment. It includes a split luciferase similar to the one used in the initial experiment, but without the peroxisomal targeting sequence. Consequently there will be no import into the peroxisome. If we subtract the luminescence of the negative control experiment from the luminescence of the main experiment we can define the degree of import.</P><P>In addition to a directed approach according to Kunze and colleagues we also want to perform a random  mutagenesis experiment to alter the five variable amino acids of the core region of the PTS2 sequence in an unbiased manner. The aim is to generate a library of different peroxisomal PTS2. The 15 nucleotides are assembled by chance. In the DNA synthesis this sequence will either be described as [NNN]<SUB>5</SUB> or [DNK]<SUB>5</SUB>. N stands for all four nucleotides mixed, K for either G or T and D for A,G or T. The “DNK” composition prohibits two out of three termination codons. Additionally with this library the amino acid frequency is improved towards a balanced ratio in between the different kinds <A href="https://www.ncbi.nlm.nih.gov/pubmed/27025684 "><ABBR title="DeLoache, W. C., Russ, Z. N., &amp; Dueber, J. E. (2016). Towards repurposing the yeast peroxisome for compartmentalizing heterologous metabolic pathways. Nature communications, 7.">(Dueber)</ABBR></A>.</P><P>Each approach could generate up to 415 DNA sequences, which is roughly 1,07 billion. On the level of the amino acid sequence there are 3,2 million possibilities, since each residue can be taken by 20 different amino acids.  For the assay we therefore need a high throughput method. </P><P>We adapted work of DeLoache, Russ and Dueber using the violacein pathway to measure the import effectiveness of tripeptides. The pathway consists of Violacein A (VioA), Violacein B (VioB) and Violacein E (VioE). It converts tryptophan into the green product prodeoxyviolacein (PDV). The first two enzymes, VioA and VioB, are expressed in the cytosol, and the third one, VioE, is targeted to the peroxisome with a PTS1 sequence. The degree of import can be measured by the intensity of green colour of the colonies. An efficient import signal leads to a strong import of the VioE into the peroxisome and subsequently to white colonies, because the intermediates cannot diffuse into the peroxisome to its respective enzyme. DeLoache <I>et al.</I> showed that there is a proportional correlation between the concentration of the green product PDV and a red fluorescent substance. The concentration of this product displays the import efficiency of the respective sequence.</P><P>This assay has been used for the evaluation of the generated PTS2 sequences. The VioE-PTS2 plasmids are harvested and cotransformed with a VioA-VioB plasmid. Each plasmid contains a specific auxotrophy marker. Consequently every growing colony contains both plasmids. To evaluate the respective sequence the concentration of the red fluorescence is measured. The more fluorescence is detected the more VioE is in the cytosol. Therefore the respective PTS2 is not that efficient. The other way around a low concentration of the fluorescent substance correlates with an efficient import via the respective PTS2. </P></H3><H2 id="Membrane_Integration">Membrane integration</H2><P>Abstract</P><DIV class="panel"><H3>Experimental Work/Design</H3><P>In order to test our hypothesis we fused the last 59 amino acids of the C-terminus of human <A href="http://www.uniprot.org/uniprot/Q7Z412">PEX26</A> (AA 246-305) to a red fluorescent protein, to further elucidate the  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A>/Pex19-dependent import. mRuby is generally used as a marker in combination with a fluorescent microscope to visualize the localization of the fusion protein. The C-terminus of <A href="http://www.uniprot.org/uniprot/Q7Z412">PEX26</A> contains a helical signal-anchor, which serves as both, a mPTS and transmembrane domain. We designed our construct with mRuby2 fused to the N-terminal side of the <A href="http://www.uniprot.org/uniprot/Q7Z412">PEX26</A>-C-terminus, this way the mRuby should face the cytosolic side of the peroxisomal membrane.   
Quite similar to our mRuby-<A href="http://www.uniprot.org/uniprot/Q7Z412">PEX26</A> approach, we designed a construct for the ER-dependent import. Therefore, we fused the mRuby2 fluorescent protein to the N-terminus of  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A> (AA 1-39). This construct should be N-terminally anchored in the peroxisomal membrane, with mRuby2 again facing the cellular lumen.</P><P>Our main goal is to introduce a rather complex membrane protein to the peroxisome that can alter specific traits. For that we fused the  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A> N-terminus (AA 1-39) to a <I>Halobacterium salinarum</I><A href="http://www.uniprot.org/uniprot/P02945">bacteriorhodopsin</A> protein (AA 16-262), replacing the first 16 amino acids (<A href="http://www.uniprot.org/uniprot/P28795">Pex3</A>-BacR). The original archaeal <A href="http://www.uniprot.org/uniprot/P02945">bacteriorhodopsin</A> acts as a proton pump by capturing light energy to move protons across the membrane out of the cell. The resulting proton gradient is subsequently converted into chemical energy. 
Our assumption is that the first transmembrane segment determines the orientation of the following protein and that therefore due to the N-terminal anchoring signal the <A href="http://www.uniprot.org/uniprot/P02945">bacteriorhodopsin</A> will be inserted in reverse orientation, pumping the protons into the peroxisome. This way the pH of the peroxisomal lumen could actively be controlled and adjusted.</P><P>Finally, our project involved combining the work of other subteams to verify the localization of our constructs in the peroxisome and analyze the effects they have on the import. Therefore, we are using the superfolded-GFP protein, another fluorescent marker, which is in our case fused to the peroxisomal import sequence PT1, and a version of Pex11 that is fused to the fluorescent marker Venus. Both markers emit light in the green light spectrum, were as mRuby2 emits light in the red part of the spectrum, giving us a strong contrast and an easy way of differentiating between the two under the fluorescent microscope.  </P><P>To physically create our constructs, we researched the DNA sequences of <A href="http://www.uniprot.org/uniprot/P02945">bacteriorhodopsin</A>,  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A> and <A href="http://www.uniprot.org/uniprot/Q7Z412">PEX26</A> via UniProt and pre-designed our fusion constructs with the software tool „Geneious“. We ordered the synthesis of three separate parts ( <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A>, <A href="http://www.uniprot.org/uniprot/Q7Z412">PEX26</A> and  <A href="http://www.uniprot.org/uniprot/P28795">Pex3</A>-BacR) from IDT. To ease out the process of assembling our plasmids, we used the „<A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#h2-2">Dueber Toolbox</A>&quot;, containing various parts such as promoters and terminators, to tailor the plasmids specific to your needs. Finally, to combine all the selected parts, we used the „Golden Gate” assembly method.</P></DIV><BUTTON class="accordion"><H2 id="Secretion">Secretion</H2><P>For the peroxisome secretion in <I>S. cerevisiae</I> we designed fusion proteins of the v-SNARE <A href="http://parts.igem.org/Part:BBa_K2271060">  Snc1 </A> with different peroxisomal membrane anchors.
We tested the constructs using a GUS Assay. The assays were performed using transformants of the strain BY4742.</P></BUTTON><DIV class="panel"><H3>Experimental Design </H3><P>We will adapt the system of  Sagt and colleagues  to secrete the content of our modified compartments <A href=" https://www.ncbi.nlm.nih.gov/pubmed/19457257"><ABBR title=" Sagt, C. M., ten Haaft, P. J., Minneboo, I. M., Hartog, M. P., Damveld, R. A., van der Laan, J. M., ... &amp; van der Klei, I. (2009). Peroxicretion: a novel secretion pathway in the eukaryotic cell. BMC biotechnology, 9(1), 48.
">
      (Sagt <I> et al</I>, 2009)
  </ABBR></A>. 
For the application of this system in <I>S. cerevisiae</I> we use a truncated version of the v-SNARE Snc1 to decorate our compartments (Figure 3.1) <A href=" http://www.jbc.org/content/272/26/16591.short"><ABBR title=" Gerst, J. E. (1997). Conserved α-helical segments on yeast homologs of the synaptobrevin/VAMP family of v-SNAREs mediate exocytic function. Journal of Biological Chemistry, 272(26), 16591-16598.

">
      (Gerst <I> et al</I>, 1997)
  </ABBR></A>. <FIGURE><FIGCAPTION><B>Figure 3.1 A diagram of the general domain structure of Snc1. </B> V is a variable domain which is not important for the binding to the t-SNARE. TM is the transmembrane domain. H1 and H2 are the a-helical segments, forming the SNAREpin with the t-SNARE  <A href=" http://www.jbc.org/content/272/26/16591.short"><ABBR title=" Gerst, J. E. (1997). Conserved α-helical segments on yeast homologs of the synaptobrevin/VAMP family of v-SNAREs mediate exocytic function. Journal of Biological Chemistry, 272(26), 16591-16598.
 
">
      (Gerst <I> et al</I>, 1997)
  </ABBR></A></FIGCAPTION></FIGURE></P><P>To decorate the compartments with the SNARE we use a <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#MembraneIntegration"> peroxisomal transmembrane protein </A>  . In our case we use the proteins Pex15 or PEX26, which were further investigated in another sub project, and fuse Snc1 to the N-terminus. We expressed these constructs of membrane anchor and Snc1 constitutively under control of the RPL18B promotor. In case of Pex15 we used a truncated version, lacking a large part of the N-terminus, only consisting of the transmembrane domain (315-383) (Figure 3.1). For PEX26 we use the truncated version published in Halbach et al. <A href=" http://jcs.biologists.org/content/119/12/2508"><ABBR title=" Halbach, A., Landgraf, C., Lorenzen, S., Rosenkranz, K., Volkmer-Engert, R., Erdmann, R., &amp; Rottensteiner, H. (2006). Targeting of the tail-anchored peroxisomal membrane proteins PEX26 and PEX15 occurs through C-terminal PEX19-binding sites. Journal of cell science, 119(12), 2508-2517.
 
">
      (Halbach <I> et al</I>, 2006)
  </ABBR></A>. 
</P><FIGURE><FIGCAPTION><B>Figure 3.2 Concept of secreting peroxisomal contents to the supernatant. </B>For the secretion, the membrane anchor Pex15 or Pex26 is used. This anchor is used to decorate peroxisomes or our modified compartments with the v-SNARE Snc1. For the secretion Snc1 interacts with the t-SNAREs in the cell membrane. Induced from this interaction the vesicle and cell membrane fuse and the content of the compartment is secreted to the supernatant.</FIGCAPTION></FIGURE><P>We verified our secretion using beta-glucuronidase (GUS) as a reporter protein. In 2012 Stock and colleagues described the GUS reporter assay for unconventional secretion <A href=" https://www.ncbi.nlm.nih.gov/pubmed/22446315 "><ABBR title=" Stock, J., Sarkari, P., Kreibich, S., Brefort, T., Feldbrügge, M., &amp; Schipper, K. (2012). Applying unconventional secretion of the endochitinase Cts1 to export heterologous proteins in Ustilago maydis. Journal of biotechnology, 161(2), 80-91.

">
      (Stock <I> et al</I>, 2012)
  </ABBR></A>
. With it, it is possible to determine whether a protein is secreted conventional and is <I>N</I>-glycosylated or secreted unconventional and not <I>N</I>-glycosylated. GUS is a bacterial protein with an <I>N</I>-glycosylation-site, which is active only if the protein is not <I>N</I>-glycosylated. The GUS-activity can be measured with different reagents in plate or liquid assays. Liquid assays can be applied qualitatively as well as quantitatively to measure differences in activity. If GUS is secreted by the conventional pathway the <I>N</I>-glycosylation leads to inactivation of the enzyme <A href=" https://www.ncbi.nlm.nih.gov/pubmed/22446315 "><ABBR title=" Stock, J., Sarkari, P., Kreibich, S., Brefort, T., Feldbrügge, M., &amp; Schipper, K. (2012). Applying unconventional secretion of the endochitinase Cts1 to export heterologous proteins in Ustilago maydis. Journal of biotechnology, 161(2), 80-91.

">
      (Stock <I> et al</I>, 2012)
  </ABBR></A>
 (Fig 3.2).</P><FIGURE><FIGCAPTION><B>Figure 3.3 The GUS Assay. </B> GUS secreted with an unconventional secreted protein like Cts1 from Ustilago maydis active in the supernatant. GUS secreted with a conventional Signal peptide (Sp) inactive in the supernatant. If GUS is in the cytoplasm there is also no activity (Lysis control) <A href=" https://www.ncbi.nlm.nih.gov/pubmed/23455565"><ABBR title=" Feldbrügge, M., Kellner, R., &amp; Schipper, K. (2013). The biotechnological use and potential of plant pathogenic smut fungi. Applied microbiology and biotechnology, 97(8), 3253-3265.

">
      (Feldbrügge <I> et al</I>, 2013)
  </ABBR></A>. </FIGCAPTION></FIGURE><P>GUS will be imported to the peroxisome with the PTS1 sequence and measured quantitatively in the supernatant. We will use a coexpression of GUS-PTS1 and Snc1-Pex15 or Snc1-PEX26 to identify the secretion of the compounds. Furthermore, we will use GUS-PTS1 expressed in <I>S. cerevisiae</I> without Snc1 fused to a membrane anchor for a control. We will measure the active GUS in the supernatant with a liquid assay based on the turnover of  4-methylumbelliferyl-beta-D-glucuronide to 4-methyl umbelliferone (4-MU)<A href=" http://cshprotocols.cshlp.org/content/2007/2/pdb.prot4688.abstract"><ABBR title="Blázquez, M. (2007). Quantitative GUS activity assay of plant extracts. Cold Spring Harbor Protocols, 2007(2), pdb-prot4690.

">
      (Blázquez <I> et al</I>, 2007)
  </ABBR></A>. Here we expect a higher activity of GUS in the supernatant of cultures with Snc1 decorated peroxisomes. 
To increase the variability of our constructs we also designed vectors with and without a GS-Linker connecting the Snc1 with the Pex15. Additionally we tested our constructs in strains with a <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#SizeAndNumber"> deletion of Pex11 </A>. This deletion leads to formation of <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#SizeAndNumber"> larger peroxisomes</A> and may increase the efficiency of our secretion mechanism.</P></DIV><BUTTON class="accordion"><H2 id="SizeAndNumber">Size and Number</H2><P>Size matters! Want to know how modifying out compartment’s morphology can benefit your reactions? Check out our design!!! </P></BUTTON><DIV class="panel"><H3>Heading</H3></DIV><BUTTON class="accordion"><H2 id="Sensors"><I>In Vivo</I> Sensors</H2><P>Surveillance of key physiology factors is indispensable for a good understanding and optimization of metabolic processes in compartments. In comparison to <I>in vitro</I> measurements, <I>in vivo</I> sensors have major advantages in terms of measure relevant physiological properties in living systems. Former methods required destructive assays, expensive and time consuming preprocessing of cells or compartments to determining fluxes inside cells <A href="https://www.ncbi.nlm.nih.gov/pubmed/20854260"><ABBR title="2010, Frommer et al. -Dynamic analysis of cytosolic glucose and ATP levels in yeast using optical sensors."> Frommer<I>et al.</I> (2010)</ABBR></A>. This leads to inexact measurements with poor spacial and temporal resolution. The usage of <I>in vivo</I> sensors tackles all of these problems at once. They allow detecting fluxes inside the cell and compartments in real time and supersede difficult and time consuming cell and organelle extraction. </P></BUTTON><DIV class="panel"><H3>Sensors</H3><P>
To enrich our toolbox we decided to measure four essential physiological factors: ATP, NADPH, Glutathione and the pH.
ATP/ADP conversion is used as an energy currency in many cellular processes like translocation of proteins and metabolites or anabolic and catabolic turnovers. It is assumed that ANT1P an ATP/AMP antiporter is located in the peroxisomal membrane and that the re shuttle mechanism of the peroxisomal protein import machinery is ATP dependent
<A href="https://www.ncbi.nlm.nih.gov/pubmed/11566870"><ABBR title="Identification and functional reconstitution of the yeast peroxisomal adenine nucleotide transporter"> (Palmieri L. <I>et al.</I>, 2001)</ABBR></A>. NADPH plays an important role in anabolic pathways and is also indispensable to our desired pathways of <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#h2-5"> nootkatone </A> 
and <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#Violacein">violacein.
</A>Glutathione is an antioxidant and redox buffer which is also found in yeast peroxisomes. It is used as cofactor by at least two types of peroxisomal proteins the glutathione peroxidases and glutathione transferases, which reduce lipid- and hydrogen peroxides or transfer glutathione to lipid peroxides for the purpose of detoxification
<A href="http://www.jbc.org/content/276/17/14279.full.pdf"><ABBR title="2001, Horiguchi H. et al.- Antioxidant System within Yeast Peroxisome">
(Horiguchi H. <I>et al.</I>, 2001)</ABBR></A>.
Furthermore, it partly represents the redox state of the peroxisome. Knowing the pH of a compartment is important to predict the activities of almost all enzymatic processes inside of it and to follow up acidification and basification upon conversion of metabolites.   

We finally chose two ratiometric sensors to perform measurements with. The pH sensitive and glutathione redox state reporting green fluorescent proteins pHLuorin2
<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3152828"><ABBR title="2011, Mahon M. J. - pHluorin2: an enhanced, ratiometric, pH-sensitive green florescent protein">
(Mahon M. J. <I>et al.</I>, 2011)
</ABBR></A>
and roGFP2
<A href="https://www.ncbi.nlm.nih.gov/pubmed/25867539"><ABBR title="2016, Schwarzländer et al.- Dissecting Redox Biology Using Fluorescent Protein Sensors">
(Schwarzländer M. <I>et al.</I>, 2016)</ABBR></A>.
We aim to target them either in the peroxisomal lumen or the cytosol. To achieve peroxisomal targeting we attach the peroxisomal targeting signal 1 with 
 <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#h2-2"> Golden Gate cloning</A>.
</P><FIGCAPTION><FONT size="2"><STRONG> Figure 5.1 </STRONG>Level1 plasmids with  promoters of medium strength and uracil auxotrophy. Top left roGFP2 cytosolic. Top right roGFP2 peroxisomal. Bottom left pHLourin2 cytosolic. Bottom right pHLourin2 peroxisomal.</FONT></FIGCAPTION><P>
Using promoters with different expression strength in order to find optimal measurement conditions is of high interest. There is a trade off between a high signal-to-noise ratio and self induced effects which are both dependent on expression levels
<A href="https://www.ncbi.nlm.nih.gov/pubmed/25867539"><ABBR title="2016, Schwarzländer et al.- Dissecting Redox Biology Using Fluorescent Protein Sensors">
(Schwarzländer M. <I>et al.</I>, 2016)
</ABBR></A>.
This cannot be generalized for each sensor. For example, pHlourin2 has sparse influence on the existing pH because of the buffer effect of proteins.

Validation of the peroxisomal localization can be achieved via fluorescence overlap of the sensor and a peroxisomal marker in our case pex13-mRuby (<A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#h2-4">import mechanism</A>).
</P><FIGCAPTION><FONT size="2"><STRONG>Figure 5.2 </STRONG>Level2 plasmids with peroxisomal marker Peroxin13-mRuby and uracil auxotrophy for colocalization. Left roGFP2 with strong promoter. Right pHLourin2 with strong promotor.</FONT></FIGCAPTION><P>
It can also be validated by transforming the sensors attached to the PTS1 sequence into Pex5 knockout yeast strain. The sensor is expected to show no specific localisation, because of the missing import sequence. We calibrate the sensors in living yeast cells and physiological ranges so that we can not only perform relative but also quantitative measurements. We aim to confirm our hypothesis of an more oxidized redox state of roGFP2 in peroxisomes with <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#Violacein">violacein pathway
</A> activity and want to measure differences in pH within yeast strains with peroxisomal membrane anchored pex3-bacteriorhodopsin protein
<A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#MembraneIntegration"> (membrane proteins) </A>. 

Once expression and localization of the sensor is proven by microscopy, measurements with a plate reader or a fluorometer are acceptable. This allows a high number of replicates to be measured accurately in reasonable time. Microscopy is performed with a filter based Nikon Eclipse TI fluorescence microscope at 100-fold magnification and plate reader measurements are performed with a Tecan infinite plate reader
<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3152828"><ABBR title="2011, Mahon et al.- pHluorin2: an enhanced, ratiometric, pH-sensitive green florescent protein">
(Mahon M. J. <I>et al.</I>, 2011)</ABBR></A>
. For pHLuorin2 emission intensity is measured at 535 nm upon excitation at 405 nm and 485 nm. Same settings were used for roGFP2
<A href="https://www.ncbi.nlm.nih.gov/pubmed/25867539"><ABBR title="2016, Schwarzländer et al.- Dissecting Redox Biology Using Fluorescent Protein Sensors">(Schwarzländer M. <I>et al.</I>, 2016)</ABBR></A>
. Evaluation of the reported signals is done by the excitation ratio of the the corresponding excitation wavelength.
</P></DIV><BUTTON class="accordion"><H2>Nootkatone</H2><P>In order to have a proof of concept for our compartmentation strategy we intend to establish the nootkatone pathway inside the peroxisome. Therefore we aim to target the three enzymes for the nootkatone production to compartment. Before starting the actual lab work for the ambitious plan to establish a completely new pathway in the peroxisome, there are several aspects that have to be considered first.</P></BUTTON><DIV class="panel"><P>The initial step is to find a reliable source to prove the abundance of our precursor Farnesyl pyrophosphate (FPP) in yeast peroxisomes. So far there is no proof of existence of FPP inside yeast peroxisomes yet. However it is predicted to be present, as it was detected in mammalian and plant peroxisomes
<A href="https://www.ncbi.nlm.nih.gov/pubmed/11108725"><ABBR title="2000, Olivier et al. - Identification of peroxisomal targeting signals in cholesterol biosynthetic enzymes. AA-CoA thiolase, hmg-coa synthase, MPPD, and FPP synthase."> Olivier <I>et al.</I> (2000)</ABBR></A>.</P><FIGURE><FIGCAPTION><STRONG>Figure 7.1:</STRONG> The structure of FPP the precursor for nootkatone. </FIGCAPTION></FIGURE><P>The precursor FPP is converted into valencene by a valencene synthase (ValS). We chose the one from <I>Callitropsis nootkatensis</I> because of its comparably high efficiency in microorganisms. It achieves greater yields in yeast than the citrus valencene synthase. Furthermore, the product specificity is relatively high, while production of byproducts is low
<A href="https://www.ncbi.nlm.nih.gov/pubmed/24112147"><ABBR title="Valencene synthase from the heartwood of Nootka cypress (Callitropsis nootkatensis) for biotechnological production of valencene.">Beekwilder <I>et al.</I> (2014)</ABBR></A>.

The valencene synthase was also chosen because of its robustness towards pH and temperature changes
<A href="https://www.ncbi.nlm.nih.gov/pubmed/24112147"><ABBR title="2014, Beekwilder et al. - Valencene synthase from the heartwood of Nootka cypress (Callitropsis nootkatensis) for biotechnological production of valencene.">Beekwilder <I>et al.</I> (2014)</ABBR></A>.


Our modelling approach revealed that for optimal yields an overexpression of valencene synthase is necessary because of its slow conversion rate <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model#MetabolicModel">(Model)</A>. This is why we chose the strongest promoter of the yeast toolbox
<A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#yeast-toolbox">(Dueber Toolbox)</A> for this attempt.
</P><FIGURE><FIGCAPTION><STRONG>Figure 7.2:</STRONG>ValS lvl.1 PTS1 plasmid</FIGCAPTION></FIGURE><FIGCAPTION><STRONG>Figure 7.3:</STRONG> The structure of valencene, an intermediate of nootkatone</FIGCAPTION><P>The intermediate valencene is then converted into nootkatol by a P450 monooxygenase as well as into small amounts of our desired product nootkatone.  The P450 monooxygenase we chose for this project was taken from the bacterium <I>Bacillus megaterium</I>. In this  case it is not only a simple P450 monooxygenase, but an entire P450 system, consisting of a soluble P450 fused to a cytochrome P450 reductase (CPR) enzyme, making an additional reductase obsolete
<A href="https://www.ncbi.nlm.nih.gov/pubmed/12419614"><ABBR title="2002, De Mot et al. - A novel class of self-sufficient cytochrome P450 monooxygenases in prokaryotes.">
   	De Mot <I>et al.</I>  (2002)</ABBR></A>.   

Unlike eukaryotic P450s, which are mostly membrane bound, this prokaryotic BM3-P450 is located in the cytosol facilitating an easier transport into the peroxisome, as membrane integration of proteins is a more difficult task to achieve than import of cytosolic proteins 
<A href="https://www.ncbi.nlm.nih.gov/pubmed/17073779"><ABBR title="2006, Girvan et al. - Flavocytochrome P450 BM3 and the origin of CYP102 fusion species.">Girvan <I>et al.</I> (2006)</ABBR></A>.


BM3 normally catalyzes the hydroxylation of long chain fatty acids
<A href="https://www.ncbi.nlm.nih.gov/pubmed/3086309"><ABBR title="1986, Narhi et al. - Characterization of a catalytically self-sufficient 119,000-dalton cytochrome P-450 monooxygenase induced by barbiturates in Bacillus megaterium.">Narhi <I>et al.</I> (1986)</ABBR></A>, which in our case could inhibit the conversion of valencene. For that reason we used a mutated version of BM3, which is called AIPLF. This variant is an enhanced version of the BM3 AIP version, which has a ten times better substrate oxidation rate for valencene than the wildtype BM3 and produces less byproduct when valencene concentration is saturated. Additionally to previous named benefits the AIPLF variant with 
<ABBR title="R47L/Y51F/F87A/A328I/I401P">5 point mutations</ABBR> in the active side has a significantly lower binding affinity towards long chain fatty acids and therefore increase the transposition rate of valencene.
<A href="http://onlinelibrary.wiley.com/doi/10.1002/cctc.201402952/full"><ABBR title="2015, Schulz et al. - Selective Enzymatic Synthesis of the Grapefruit Flavor (+)-Nootkatone">Schulz <I>et al.</I> (2015)</ABBR></A>,
<A href="https://docserv.uni-duesseldorf.de/servlets/DerivateServlet/Derivate-43193/Dissertation_SvenCarstenLehmann.pdf"><ABBR title="2016, Lehmann - Entwicklung eines P450-basierten Ganzzellkatalysators
für die selektive Oxyfunktionalisierung von α-Pinen">Lehmann (2016)</ABBR></A></P><FIGURE><FIGCAPTION><STRONG>Figure 7.4:</STRONG> BM3 lvl.1 PTS1 plasmid </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><STRONG>Figure 7.5:</STRONG> structure Nootkatol </FIGCAPTION></FIGURE><P>The alcohol dehydrogenase from <I>Pichia pastoris</I> subsequently converts nootkatol into nootkatone by oxidation. It uses NAD<SUP>+</SUP> as a cofactor which is reduced in the reaction. The regeneration of this cofactor is facilitated by the BM3, which oxidizes NADH
<A href="http://onlinelibrary.wiley.com/doi/10.1002/cctc.201402952/full"><ABBR title="2015, Schulz et al. - Selective Enzymatic Synthesis of the Grapefruit Flavor (+)-Nootkatone">Schulz <I>et al.</I> (2015)</ABBR></A>.
</P><FIGURE><FIGCAPTION><STRONG>Figure 7.6:</STRONG> ADH lvl.1 PTS1 plasmid </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><STRONG>Figure 7.7:</STRONG> structure Nootkatone </FIGCAPTION></FIGURE><P>The first milestone to achieve our goal is the separate integration of each of our three enzymes ValS, BM3 and ADH into level 1 vectors <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#yeast-toolbox">(Dueber Toolbox)</A>  in yeast and to verify their expression by Western Blot analysis. Therefore, a 3xFlag/6xHis-tag was added to the N-terminus of each of the proteins.  It enables us to use an anti-His antibody followed by an anti-mouse-antibody to make protein abundances visible. Subsequently, the two enzymes ADH and ValS were combined in a  level 2 cassette plasmid.  BM3 is designed as a level 1 plasmid and will be co-expressed with the level 2 plasmid to achieve a nootkatone production. The expression of the enzymes in the cytoplasm is again verified by Western Blot analysis. 

The further approach aims to provide the enzymes with C-terminal peroxisomal targeting signals type one, which finally converts the nootkatone pathway into our artificial compartment. </P><P>Since the production of nootkatone does not lead to a change of colour we need to apply different methods for verifying substrates. Once we succeed with the qualitative validation via Western Blot analysis, we can verify the presence of nootkatone by using high performance liquid chromatography and mass spectrometry, respectively.</P><H3>Model influence on Nootkatone expression</H3><P>We modeled the nootkatone biosynthesis pathway using ordinary differential equations in order to optimize nootkatone production. We found two hard and an easy problem, all of which we could find a solution for. The easy problem is optimization of the enzyme concentrations of the biosynthesis pathway. According to our <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model#MetabolicModel">model of the Nootkatone pathway</A> we found that overexpression of Valencene Synthase is necessary to maximize the Nootkatone yield, while both alcohol dehydrogenase and p450-BM3 have only minor effects on the yield. </P><P>One of the hard problems, as shown in our <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model#Penalty-Model">penalty model</A> is the toxicity of nootkatone and nootkatol. Since the toxicity most likely stems from both nootkatone and nootkatol clogging up the cell wall we present our peroxisome as a solution for this problem. When comparing the cytosolic model to our <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model#Peroxisome-model">peroxisomal model</A> we found that, if our assumption that neither Nootkatone nor Nootkatol are able to pass the peroxisomal membrane holds up, we can greatly increase Nootkatone production.</P><P>A further  problem is the influx of the pathway precursor farnesyl pyrophosphate (FPP). We used <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Model#OptKnock">OptKnock analysis</A> to design yeast strains with optimized FPP production. With this analysis we got hints that growing the yeast cells on a fatty acid medium might be a simple alternative to 
knocking out the desired genes.</P></DIV><BUTTON class="accordion"><H2 id="Violacein">Violacein</H2><P>Working on a project is a process of well planned steps. It starts with an idea and theoretical research to create a design. When finally demonstrating the mechanism of the project it is important to point out the benefits with a suitable application.
Synthetic biology offers countless numbers of new opportunities, especially in the field of metabolic engineering. To testify some main parts of our <STRONG>artico</STRONG> project, we decided to relocate a metabolic pathway into yeast peroxisomes. There are several reasons why this approach fits perfectly as a proof for our concept.
</P></BUTTON><DIV class="panel"><P> Based on described of<A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#Violacein"> advantages </A> of violacein this pathway was chosen. Violacein is naturally produced in numerous bacterial strains, most popular in the gram-negative <I> Chromobacterium violaceum </I>. It is related to biofilm production and shows typical activities of a secondary metabolite <A href="https://www.hindawi.com/journals/bmri/2015/465056/"><ABBR title="Violacein: Properties and Production of a Versatile Bacterial Pigment">
(Seong Yeol Choi <I>et al.</I>, 2015)
</ABBR>
.
</A></P><FIGURE><FIGCAPTION><STRONG>Figure 8.1</STRONG> The synthesis of Violacein requires five enzymes encoded by the VioABCDE operon. VioA, a flavin-dependent L-tryptophan oxidase and VioB, a heme protein, work in combination to oxidize and dimerize L-tryptophan to an IPA imine dimer. Hydrogen peroxide is released as a by-product of the VioA reaction. Next step by VioE is the rearrangement of the IPA imine dimer to protodeoxyviolaceinic acid, which can non-enzymatically oxidize to prodeoxyviolacein or, by VioC via deoxyviolaceinic acid, oxidize to pink deoxyviolacein. The flavin-dependent oxygenases VioC and VioD require interaction with the oxidized form of flavin-adenine dinucleotide (FAD)
<A href="http://www.uniprot.org/uniprot/Q9S3U9"><ABBR title="UniProtKB - Q9S3U9 (VIOC_CHRVO)">
(uniprot,
</ABBR></A><A href="http://www.uniprot.org/uniprot/Q9S3U8"><ABBR title="UniProtKB - Q9S3U8 (VIOD_CHRVO)">
uniprot)      
</ABBR></A>
. The two enzymes act sequentially: first, VioD hydroxylates protodeoxyviolaceinic acid, leading to protoviolaceinic acid. Second, VioC creates the oxindole at the 2-position of one indole ring, leading to violet violacein
<A href="https://www.ncbi.nlm.nih.gov/pubmed/17176066"><ABBR title="In vitro biosynthesis of violacein from L-tryptophan by the enzymes VioA-E from Chromobacterium violaceum">
(Balibar CJ <I> et al.</I>, 2006)
</ABBR></A><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5025692/"><ABBR title="Biosynthesis of Violacein, Structure and Function of l-Tryptophan Oxidase VioA from Chromobacterium violaceum">
(Janis J. Füller <I> et al.</I>, 2016)
</ABBR></A>
.
</FIGCAPTION></FIGURE><P> Relocating the pathway into the peroxisome enables proximity of the enzymes and substrates. Furthermore the yeast cell is protected from the toxic substance hydrogen peroxide. Yeast peroxisomes have no problem with this as their main function is the beta-oxidation of fatty acids and the detoxification of the thereby produced H<SUB>2</SUB>O<SUB>2</SUB><A href="https://www.ncbi.nlm.nih.gov/pubmed/17445803"><ABBR title="The peroxisomal protein import machinery">
(Erdmann R. <I>et al.</I>, 2007)
</ABBR></A>. Because VioC and VioD are FAD-dependent, it is additionally an evidence for FAD availability inside of the peroxisome, if the synthesis of Violacein works. Otherwise the two enzymes would not be able to catalyze the reaction.

The genes for VioA, VioB and VioE were amplificated via PCR with Golden Gate compatible overhangs from the biobrick VioABCE
<A href="http://parts.igem.org/Part:BBa_K274004"><ABBR title="Part:BBa_K274004">
(Part: BBa_K274004)
</ABBR></A>
.
</P></DIV><P>
By Golden Gate cloning the peroxisomal targeting sequence (PTS1) was attached to the C-terminus of every pathway protein. Combined with the other necessary parts of the <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#h2-2"> toolbox </A> they represent the level 1 plasmids.
</P><P>
The PTS-tag marks the proteins for the import into peroxisomes. This should first of all point out the functionality of the yeast’s natural import mechanism and also be the basis for demonstrating our own modeled PTS*, proving our designed <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#Pts1Import">orthogonal import mechanism</A>.
Furthermore we also aim to optimize the working conditions for the enzymes inside of the reaction room - the peroxisomes. For example to vary the pH with new <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#MembraneIntegration"> membrane proteins </A> such as bacteriorhodopsin. To secure this change, we can also check the current conditions by our designed <A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Description#Sensors"> sensors</A>.
</P><P>
There are several methods to verify the pathway’s enzymes. First of all, violacein and several intermediates (prodeoxyviolacein, deoxyviolacein, proviolacein) are colorful and the production in yeast can be visualized easily. Furthermore we added a His-/Flag-tag to the N-terminus of every protein (see geneious plasmid cards) to confirm their expression via SDS page and western blot. After verifying the presence of the enzymes the next step is to test their functionality. Before performing <I>in vivo</I> experiments in yeast an <I>in vitro</I> assay was implemented. For this the three enzyme pathway leading to PDV was reconstructed, testing VioA, VioB and VioE. To enable the best conditions for the enzymes, the pathway was studied intensively and all needed cofactors were calculated and added to the <I>in vitro</I> reaction (see protocol <A href="https://static.igem.org/mediawiki/2017/a/aa/T--Cologne-Duesseldorf--prodeoxyviolacein_assay.pdf"> prodeoxyviolacein <I>in vitro </I>assay</A>). This included FAD, MgCl<SUB>2</SUB>, catalase for decomposition of hydrogen peroxide, and the substrate L-tryptophan. The <I>in vitro</I> reaction was followed by qualitative analysis via HPLC and mass spectrometry.
</P></DIV><BUTTON class="accordion"><H2 id="designrules">Design Rules For Genome Engineering Regarding Customization of Peroxisome Properties</H2><P>In order to reach the ultimate goal of creating a fully controllable artificial compartment, genome engineering can be utilized for customizing the compartment's properties,.These include aspects such as membrane permeability, size/number, or decoupling of peroxisomes from the cytoskeleton, the peroxisomal proteome or metabolome. In our project we used the Crispr Cas9 system for knocking out several genes (Pex9, Pex31&amp;Pex32, INP1, POT1) at the same time in order to engineer the previously mentioned properties. Furthermore, we designed a yeast strain  with a completely replaced protein-import machinery for controlling the entire peroxisomal lumen.</P><P>For the future one could think of much more radical strategies for peroxisomal engineering with a final goal of a “minimal peroxisome” by redirecting metabolic pathways through changing the protein-localization-signal in the yeast genome. Additionally, endogenous metabolic pathways could be redirected to our novel artificial compartment for establishing a customized metabolism specifically tailored for the user's application.</P></BUTTON><DIV class="panel"><H3>Introduction</H3><P>In order to achieve a fully controllable artificial compartment, the first step was to design a completely orthogonal import system. Next was the knockout of endogenous import systems. However, a few proteins are imported neither by the Pex5 nor the Pex7 import machinery. Therefore, specific genome engineering designs, such as knockouts, deleting or redirecting the protein localization could be utilized for the ultimate goal of creating a synthetic organelle.</P><P>Additionally, knockouts or genome integrations enable customization of the peroxisomal properties, such as membrane permeability, size/number, decoupling of peroxisomes from cytoskeleton and the peroxisomal metabolism.</P><P>All these strategies allow a rational design of an artificial compartment, which is fully engineerable regarding  the proteome, metabolome and the entire peroxisomal environment.</P><H3>Design of yeast multi -knockout strains</H3><H4>The Crispr Cas9 System</H4><P>The demands on yeast engineering have significantly increased with the design of more complex systems and extensive metabolic pathways. Genetic techniques that have historically relied on marker recycling are unable to keep up with the ambitions of synthetic biologists. In recent years the Crispr Cas9 system has been used for several strain-engineering purposes, including:</P><UL><LI>Markerless integration of multiple genetic cassettes into selected genomic loci</LI><LI>Multiplexed and iterative gene knockouts without the need to recycle a marker</LI><LI>Precise genome editing – nucleotide substitutions, etc.</LI></UL><P>We utilized the Cas9 system as a tool for peroxisomal engineering and have adopted the existing toolbox from <ABBR title="Lee, Michael E.; DeLoache, William C.; Cervantes, Bernardo; Dueber, John E. (2015): A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly. In: ACS synthetic biology 4 (9), S. 975–986. DOI: 10.1021/sb500366v.">(Lee et al. 2015)</ABBR> and the complete cloning system which also provides the possibilities for genome integration and gene editing by Cas9. For this, two oligonucleotides have to be designed for targeting the Cas protein to the gene of interest.</P><FIGURE><FIGCAPTION>Figure 1: Plasmid construction for the gRNA expression plasmidTwo oligos, containing the targeting sequence of the gRNA, have to be annealed and can then be integrated in the gRNA entry Vector by a Golden Gate reaction. Adapted from (<ABBR title="Lee, Michael E.; DeLoache, William C.; Cervantes, Bernardo; Dueber, John E. (2015): A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly. In: ACS synthetic biology 4 (9), S. 975–986. DOI: 10.1021/sb500366v.">Lee et al. 2015</ABBR>)</FIGCAPTION></FIGURE><P>Several gRNA vectors can subsequently be assembled into one vector with a Cas9 expression cassette and then be transformed into yeast. The expression of Cas9 together with gene specific gRNA´s leads to double strand breakage followed by non-homologous end joining repair or homologous recombination, in case of added repair DNA (figure 3).</P><FIGURE><FIGCAPTION>Figure 2: Plasmid construction for the expression plasmid containing Cas9 and gRNA´sVector for Cas9 and gRNA expression, assembled by a Golden Gate reaction, containing a URA marker, Cen6 yeast origin and a kanamycin resistance. Adapted from (<ABBR title="Lee, Michael E.; DeLoache, William C.; Cervantes, Bernardo; Dueber, John E. (2015): A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly. In: ACS synthetic biology 4 (9), S. 975–986. DOI: 10.1021/sb500366v.">Lee et al. 2015</ABBR>)</FIGCAPTION></FIGURE><P>The combination of the Cas9 system with DNA repair sequences enables not only knockouts of peroxisomal proteins, but also allows redirecting protein localization by changing protein targeting signals or integration of linear DNA into yeast chromosomes. Genome engineering facilitates yeast strain development for customized peroxisomes.</P><FIGURE><FIGCAPTION>Figure 3: Design of repair DNA sequences for homologous recombination after inducing double strand break by Cas9
  Repair DNA sequences can be used to increase the efficiency for Cas9 guided knocking out of specific genes, but would also allow genomic integration of targeting signals or complete genes.  Adapted from (<ABBR title="Lee, Michael E.; DeLoache, William C.; Cervantes, Bernardo; Dueber, John E. (2015): A Highly Characterized Yeast Toolkit for Modular, Multipart Assembly. In: ACS synthetic biology 4 (9), S. 975–986. DOI: 10.1021/sb500366v.">Lee et al. 2015</ABBR>)</FIGCAPTION></FIGURE><H3>The peroxisomal proteome of yeast (saccharomyces cerevisiae)</H3><P>The peroxisomal proteome is studied extensively for saccharomyces cerevisiae and contains exactly 67 proteins (<ABBR title="Kohlwein, Sepp D.; Veenhuis, Marten; van der Klei, Ida J. (2013): Lipid droplets and peroxisomes: key players in cellular lipid homeostasis or a matter of fat--store 'em up or burn 'em down. In: Genetics 193 (1), S. 1–50. DOI: 10.1534/genetics.112.143362.">Kohlwein et al. 2013</ABBR>). The function is characterized for the most of those proteins and it is known, that yeast peroxisomes are expendable under optimal growth conditions. Nevertheless, some knockouts are lethal under oleate or stress conditions.</P><TABLE><THEAD><TR><TH>Gene</TH><TH>Required for growth on oleate</TH><TH>Expression induced by oleate</TH><TH>Enzyme/activity</TH><TH>Molecular mass (kDa)</TH><TH>Isoelectric point</TH><TH>Molecules per cell </TH><TH>Localization</TH><TH>Function</TH></TR></THEAD><TBODY><TR><TD colspan="”9”">ß-Oxidation enzymes</TD></TR><TR><TD>PCS60 (FAT2)</TD><TD>No</TD><TD>Yes</TD><TD>Medium chain fatty acyl-CoA synthetase</TD><TD>60.5</TD><TD>9.98</TD><TD>8.770</TD><TD>Peripheral peroxisomal membrane and matrix</TD><TD>Activates fatty acids with a preference for medium chain lengths, C9-C13</TD></TR><TR><TD>FAT1</TD><TD>No</TD><TD>-</TD><TD> Very long chain fatty acyl-CoA synthetase and long chain fatty acid transporter</TD><TD>77.1</TD><TD>8.47</TD><TD>16,900</TD><TD>Lipid droplet, ER, peroxisome Three predicted TM</TD><TD>Activates fatty acids with a preference for very long chain lengths, C20–C26</TD></TR><TR><TD>POX1</TD><TD>Yes</TD><TD>Yes</TD><TD>Acyl-CoA- oxidase</TD><TD>84.0</TD><TD>8.73</TD><TD>ND</TD><TD>Peroxisomal matrix</TD><TD>Oxidation of acyl-CoA</TD></TR><TR><TD>CTA1</TD><TD>No</TD><TD>Yes</TD><TD>Catalase</TD><TD>58.6</TD><TD>7.46</TD><TD>623</TD><TD>Peroxisomal matrix</TD><TD>Degrades hydrogen peroxide produced by Pox1</TD></TR><TR><TD>FOX2 (POX2)</TD><TD>Yes</TD><TD>Yes</TD><TD>Multifunctional enzyme; 3-hydroxyacyl-CoA dehydrogenase and enoyl-CoA hydratase</TD><TD>98.7</TD><TD>9.75</TD><TD>ND</TD><TD>Peroxisomal matrix</TD><TD>-</TD></TR><TR><TD>POT1 (FOX3, POX3)</TD><TD>Yes</TD><TD>Yes</TD><TD>3-Ketoacyl-CoA thiolase</TD><TD>44.7</TD><TD>7.56</TD><TD>ND</TD><TD>Peroxisomal matrix</TD><TD>Cleaves 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA</TD></TR><TR><TD>DCI1 (ECI2)</TD><TD>-</TD><TD>-</TD><TD> Δ(3,5)-Δ(2,4)-dienoyl-CoA isomerase (putative)</TD><TD>30.1</TD><TD>8.83</TD><TD>ND</TD><TD>Peroxisomal matrix</TD><TD>Auxiliary enzyme of fatty acid β-oxidation; role in β-oxidation debated</TD></TR><TR><TD>SPS19 (SPX1)</TD><TD>Yes</TD><TD>Yes</TD><TD>2,4-Dienoyl-CoA reductase</TD><TD>31.1</TD><TD>9.67</TD><TD>ND</TD><TD>Peroxisomal matrix</TD><TD>Auxiliary enzyme of fatty acid β-oxidation</TD></TR><TR><TD>ECI1</TD><TD>Yes</TD><TD>Yes</TD><TD>Δ3, Δ2-enoyl-CoA isomerase</TD><TD>31.7</TD><TD>8.21</TD><TD>ND</TD><TD>Peroxisomal matrix</TD><TD>Auxiliary enzyme of fatty acid β-oxidation</TD></TR><TR><TD>TES1 (PTE1)</TD><TD>Yes</TD><TD>Yes</TD><TD>Acyl-CoA thioesterase</TD><TD>40.3</TD><TD>9.58</TD><TD>ND</TD><TD>Peroxisomal matrix</TD><TD>Auxiliary enzyme of fatty acid β-oxidation</TD></TR><TR><TD>MDH3</TD><TD>Yes</TD><TD>Yes</TD><TD>Malate dehydrogenase</TD><TD>37.3</TD><TD>10.00</TD><TD>3,300</TD><TD>Peroxisomal matrix</TD><TD>Required for the malate-oxaloacetete shuttle, to exchange peroxisomal NADH for cytosolic NAD+, part of the glyoxylate cycle
      </TD></TR><TR><TD>IDP3</TD><TD>Yes</TD><TD>Yes</TD><TD>NADP+ dependent isocitrate dehydrogenase</TD><TD>47.91</TD><TD>10.02</TD><TD>ND</TD><TD>Peroxisomal matrix</TD><TD>Required for the 2-ketoglutarate/isocitrate shuttle, exchanging peroxisomal NADP+ for cytosolic NADPH</TD></TR><TR><TD>CAT2</TD><TD>No</TD><TD>No</TD><TD>Carnitine acetyl-CoA transferase</TD><TD>77.2</TD><TD>8.34</TD><TD>470</TD><TD>Peroxisome mitochondria</TD><TD>Transfers activated acetyl groups to carnitine to form acetylcarnitine which can be shuttled across membranes</TD></TR><TR><TD colspan="”9”">Glyoxylate cycle</TD></TR><TR><TD>CIT2</TD><TD>No</TD><TD>-</TD><TD>Citrate synthase</TD><TD>51.4</TD><TD>6.34</TD><TD>2,310</TD><TD>Peroxisomal matrix</TD><TD>Condensation of acetyl CoA and oxaloacetate to form citrate</TD></TR><TR><TD>MDH3</TD><TD>Yes</TD><TD>Yes</TD><TD>Malate dehydrogenase</TD><TD>37.3</TD><TD>10.00</TD><TD>3,300</TD><TD>Peroxisomal matrix</TD><TD>Required for the malate–oxaloacetete shuttle, to exchange peroxisomal NADH for cytosolic NAD+</TD></TR><TR><TD>MLS1</TD><TD>Yes</TD><TD>-</TD><TD>Malate synthase</TD><TD>62.8</TD><TD>7.18</TD><TD>ND</TD><TD>Peroxisomal protein</TD><TD>Required for utilization of nonfermentable carbon sources</TD></TR><TR><TD colspan="”9”"> Other peroxisome-associated enzyme activities</TD></TR><TR><TD> GPD1 (DAR1, HOR1, OSG1, OSR5</TD><TD>-</TD><TD>-</TD><TD> NAD-dependent glycerol-3-phosphate dehydrogenase</TD><TD>42.9</TD><TD>5.26</TD><TD>807</TD><TD>Peroxisome, cytosol, nucleus</TD><TD> Key enzyme of glycerol synthesis, essential for growth under osmotic stress</TD></TR><TR><TD>PNC1</TD><TD>-</TD><TD>-</TD><TD>Nicotinamidase</TD><TD>25.0</TD><TD>6.23</TD><TD>7,720</TD><TD>Peroxisome, cytosol</TD><TD>Converts nicotinamide to nicotinic acid as part of the NAD(+) salvage pathway</TD></TR><TR><TD>NPY1</TD><TD>-</TD><TD>-</TD><TD>NADH diphosphatase</TD><TD>43.5</TD><TD>6.26</TD><TD>846</TD><TD>Peroxisome cytosol</TD><TD>Hydrolyzes the pyrophosphate linkage in NADH and related nucleotides</TD></TR><TR><TD>STR3</TD><TD>-</TD><TD>-</TD><TD> Cystathionine β-lyase</TD><TD>51.8</TD><TD>7.96</TD><TD>ND</TD><TD>Peroxisome</TD><TD>Converts cystathionine into homocysteine</TD></TR><TR><TD>STR3</TD><TD>-</TD><TD>-</TD><TD>Cystathionine ß-lyase</TD><TD>51.8</TD><TD>7.96</TD><TD>ND</TD><TD>Peroxisome</TD><TD>Converts cystathionine into homocysteine</TD></TR><TR><TD>GTO1</TD><TD>-</TD><TD>-</TD><TD> ω-Class glutathione transferase </TD><TD>41.3</TD><TD>9.53</TD><TD>-</TD><TD>Peroxisome</TD><TD>Induced under oxidative stress</TD></TR><TR><TD>AAT2(ASP5)</TD><TD>-</TD><TD>Yes</TD><TD>Aspartate aminotransferase</TD><TD>46.1</TD><TD>8.50</TD><TD>7,700</TD><TD>Cytosol, peroxisome</TD><TD>Involved in nitrogen metabolism</TD></TR><TR><TD>PCD1</TD><TD>-</TD><TD>-</TD><TD>Nudix pyrophosphatase with specificity for coenzyme A and CoA derivatives</TD><TD>39.8</TD><TD>6.59</TD><TD>238</TD><TD>Peroxisome</TD><TD>May function to remove potentially toxic oxidized CoA disulfide from peroxisomes</TD></TR><TR><TD>LPX1</TD><TD>-</TD><TD>Yes</TD><TD>Triacylglycerol lipase</TD><TD>43.7</TD><TD>8.16</TD><TD>2,350</TD><TD>Peroxisomal matrix</TD><TD>-</TD></TR><TR><TD colspan="”9”">Peroxisomal transporters</TD></TR><TR><TD>PXA1 (LPI1, PAL1, PAT2, SSH2</TD><TD>-</TD><TD>-</TD><TD> Subunit of a heterodimeric ATP-binding cassette transporter complex</TD><TD>100.0</TD><TD>10.34</TD><TD>ND</TD><TD>Peroxisomal membrane</TD><TD>Import of long-chain fatty acids into peroxisomes</TD></TR><TR><TD>PXA2 (PAT1)</TD><TD>-</TD><TD>-</TD><TD>Subunit of a heterodimeric ATP-binding cassette transporter complex</TD><TD>97.1</TD><TD>9.47</TD><TD>ND</TD><TD>Peroxisomal membrane</TD><TD>Import of long-chain fatty acids into peroxisomes</TD></TR><TR><TD>ANT1(YPR118C)</TD><TD>-</TD><TD>-</TD><TD>Adenine nucleotide transporter</TD><TD>36.4</TD><TD>10.6</TD><TD>2,250</TD><TD>Peroxisomal membrane</TD><TD>Involved in β-oxidation of medium-chain fatty acids</TD></TR><TR><TD colspan="”9”">Peroxins</TD></TR><TR><TD>Pex1 (PAS1)</TD><TD>-</TD><TD>-</TD><TD>AAA ATPase</TD><TD>117.3</TD><TD>6.93</TD><TD>2,100</TD><TD>Peroxisomal membrane</TD><TD>Involved in recycling of Pex5, forms heterodimer with Pex6</TD></TR><TR><TD>Pex2 (RT1, PAS5)</TD><TD>-</TD><TD>-</TD><TD>E3 ubiquitin ligase</TD><TD>30.8</TD><TD>9.02</TD><TD>339</TD><TD>Peroxisomal membrane</TD><TD>RING finger protein, forms complex with Pex10 and Pex12. Involved in matrix protein import</TD></TR><TR><TD>Pex3 (PAS3)</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>50.7</TD><TD>6.29</TD><TD>1,400</TD><TD>Peroxisomal membrane</TD><TD>Required for proper localization of PMPs</TD></TR><TR><TD>Pex4 (PAS2, UBC10)</TD><TD>-</TD><TD>-</TD><TD>Ubiquitin conjugating enzyme</TD><TD>21.1</TD><TD>5.36</TD><TD>ND</TD><TD>Peroxisomal membrane</TD><TD>Involved in matrix protein import</TD></TR><TR><TD>Pex5 (PAS10)</TD><TD>-</TD><TD>-</TD><TD>Soluble PTS1 receptor</TD><TD>69.3</TD><TD>4.79</TD><TD>2,070</TD><TD>Cytosol and peroxisomal matrix</TD><TD>Required for import of PTS1-containing peroxisomal proteins, contains TPR domains</TD></TR><TR><TD>Pex6 (PAS8)</TD><TD>-</TD><TD>-</TD><TD>AAA ATPase</TD><TD>115.6</TD><TD>5.44</TD><TD>1,630</TD><TD>Peroxisomal membrane</TD><TD>Involved in recycling of Pex5, forms heterodimer with Pex1</TD></TR><TR><TD>Pex7 (PAS7, PEB1)</TD><TD>-</TD><TD>-</TD><TD>Soluble PTS2 receptor</TD><TD>42.3</TD><TD>8.34</TD><TD>589</TD><TD>Cytosol and peroxisomal matrix</TD><TD>Requires Pex18 and Pex21 for association to the receptor docking site, contains WD40 repeat </TD></TR><TR><TD>PEX8 (PAS6)</TD><TD>-</TD><TD>-</TD><TD> Intra peroxisomal organizer of the peroxisomal import machinery </TD><TD>68.2</TD><TD>7.62</TD><TD>538</TD><TD>Peroxisomal matrix and luminal membrane face</TD><TD>Pex5-cargo dissociation</TD></TR><TR><TD>Pex9</TD><TD>-</TD><TD>-</TD><TD>PTS-receptor</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Pex10</TD><TD>-</TD><TD>-</TD><TD>E3 ubiquitin ligase</TD><TD>39.1</TD><TD>9.88</TD><TD>ND</TD><TD>Peroxisomal membrane</TD><TD>RING finger protein involved in Ubc4-dependent Pex5 ubiquitination. Forms complex with Pex2 and Pex12 </TD></TR><TR><TD>PEX11 (PMP24, PMP 27)</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>26.9</TD><TD>10.65</TD><TD>1,630</TD><TD>Peroxisomal membrane</TD><TD>Involved in peroxisome fission, required for medium-chain fatty acid oxidation </TD></TR><TR><TD>Pex12 (PAS11)</TD><TD>-</TD><TD>-</TD><TD>E3 ubiquitin ligase</TD><TD>46.0</TD><TD>9.86</TD><TD>907</TD><TD>-</TD><TD>RING finger protein, forms complex with Pex2 and Pex10</TD></TR><TR><TD>Pex13 (PAS20)</TD><TD>-</TD><TD>-</TD><TD>Component of docking complex for Pex5 and Pex7</TD><TD>42.7</TD><TD>9.83</TD><TD>7,900</TD><TD>Peroxisomal membrane</TD><TD>Forms complex with Pex14 and Pex17</TD></TR><TR><TD>Pex14</TD><TD>-</TD><TD>-</TD><TD>Central component of the receptor docking complex</TD><TD>38.4</TD><TD>4.61</TD><TD>2,570</TD><TD>Peroxisomal membrane</TD><TD>Interacts with Pex13</TD></TR><TR><TD>Pex15 (PAS21)</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>43.7</TD><TD>8.42</TD><TD>1,070</TD><TD>Peroxisomal membrane</TD><TD>Recruitment of Pex6 to the peroxisomal membrane, tail anchored PMP</TD></TR><TR><TD>Pex17 (PAS9)</TD><TD>-</TD><TD>-</TD><TD>Component of docking complex for Pex5 and Pex7</TD><TD>23.2</TD><TD>10.24</TD><TD>656</TD><TD>Peroxisomal membrane</TD><TD>Forms complex with Pex13 and Pex14</TD></TR><TR><TD>Pex18</TD><TD>-</TD><TD>-</TD><TD>-</TD><TD>Required for PTS2 import</TD><TD>32.0</TD><TD>4.78</TD><TD>ND</TD><TD>Interacts with Pex7 partially redundant with Pex21</TD></TR><TR><TD>Pex19 (PAS12)</TD><TD>-</TD><TD>-</TD><TD>Chaperone and import receptor for newly synthesized PMP</TD><TD>38.7</TD><TD>4.08</TD><TD>5,350</TD><TD>Cytosol, peroxisome, farnesylated</TD><TD>Interacts with PMPs, involved in PMP sorting. Also interacts with Myo2 and contributes to peroxisome partitioning</TD></TR><TR><TD>Pex21</TD><TD>-</TD><TD>-</TD><TD>Required for PTS2 protein import</TD><TD>33.0</TD><TD>6.67</TD><TD>ND</TD><TD>Cytosol</TD><TD>Interacts with Pex7, partially redundant with Pex18</TD></TR><TR><TD>Pex22(YAF5)</TD><TD>-</TD><TD>-</TD><TD>Required for import of peroxisomal proteins</TD><TD>19.9</TD><TD>8.33</TD><TD>259</TD><TD>Peroxisomal membrane</TD><TD>Recruits Pex4 to the peroxisomal membrane</TD></TR><TR><TD>Pex25</TD><TD>-</TD><TD>-</TD><TD>Involved in the regulation of peroxisome size and maintenance, required for re-introduction of peroxisomes in peroxisome deficient cells</TD><TD>44.9</TD><TD>9.77</TD><TD>2,420</TD><TD>Peripheral peroxisomal membrane</TD><TD>Recruits GTPase RhoI to peroxisomes, interacts with homologous protein Pex27</TD></TR><TR><TD>Pex27</TD><TD>-</TD><TD>-</TD><TD>Involved in the regulation of peroxisome size and number</TD><TD>44.1</TD><TD>10.49</TD><TD>382</TD><TD>Peripheral peroxisomal membrane</TD><TD>Interacts with homologous protein Pex25</TD></TR><TR><TD>Pex28</TD><TD>-</TD><TD>-</TD><TD>Involved in the regulation of peroxisome size, number and distribution</TD><TD>66.1</TD><TD>7.09</TD><TD>ND</TD><TD>Peroxisomal membrane</TD><TD>May act upstream of Pex30, Pex31 and Pex 32</TD></TR><TR><TD>Pex29</TD><TD>-</TD><TD>-</TD><TD>63.5</TD><TD>6.8</TD><TD>5,040</TD><TD>Peroxisomal membrane</TD><TD>May act upstream of Pex30, Pex31 and Pex32</TD></TR><TR><TD>Pex30</TD><TD>-</TD><TD>-</TD><TD>Involved in the regulation of peroxisome number</TD><TD>59.5</TD><TD>5.59</TD><TD>4,570</TD><TD>Peroxisomal membrane</TD><TD>Negative regulator, partially functionally redundant with Pex31 and Pex32</TD></TR><TR><TD>Pex31</TD><TD>-</TD><TD>-</TD><TD>Involved in the regulation of peroxisome number</TD><TD>52.9</TD><TD>10.15</TD><TD>238</TD><TD>Peroxisomal membrane</TD><TD>Negative regulator, partially functionally redundant with Pex30 and Pex32 </TD></TR><TR><TD>Pex32</TD><TD>-</TD><TD>-</TD><TD>Involved in the regulation of peroxisome number</TD><TD>48.6</TD><TD>9.14</TD><TD>ND</TD><TD>Peroxisomal membrane</TD><TD>Negative regulator partially functionally redundant with Pex30 and Pex31</TD></TR><TR><TD>PEX34</TD><TD>-</TD><TD>-</TD><TD>Involved in the regulation of peroxisome number</TD><TD>16.6</TD><TD>10.30</TD><TD>ND</TD><TD>Peroxisomal membrane</TD><TD>-</TD></TR><TR><TD colspan="”9”">Peroxisome fission and inheritance</TD></TR><TR><TD>DYN2 (SLC1)</TD><TD>-</TD><TD>-</TD><TD>Light chain dynein</TD><TD>10.4</TD><TD>9.03</TD><TD>1,310</TD><TD>Cytosol</TD><TD>Microtubule motor protein</TD></TR><TR><TD>SEC20</TD><TD>-</TD><TD>-</TD><TD>v-SNARE</TD><TD>43.9</TD><TD>5.92</TD><TD>4,910</TD><TD>Golgi, ER</TD><TD>Involved in retrograde transport from the Golgi to the ER, interacts with the Dsl1 complex through Tip20</TD></TR><TR><TD>SEC39(DSL3)</TD><TD>-</TD><TD>-</TD><TD>Component of the Ds11p-tethering complex</TD><TD>82.4</TD><TD>4.65</TD><TD>1,840</TD><TD>ER, nuclear envelope</TD><TD>Proposed to be involved in protein secretion</TD></TR><TR><TD>DSL1 (RNS1)</TD><TD>-</TD><TD>-</TD><TD>Component of the ER target site that interacts with coatomer</TD><TD>88.1</TD><TD>4.69</TD><TD>8,970</TD><TD>Peripheral ER, Golgi membrane</TD><TD>Forms a complex with Sec39 and Tip20 that interacts with ER SNAREs, Sec20 and Use1</TD></TR><TR><TD>FIS1 (MDV2)</TD><TD>-</TD><TD>-</TD><TD>Required for peroxisome fission</TD><TD>17.7</TD><TD>9.87</TD><TD>2,410</TD><TD>Peroxisomal membrane mitochondria</TD><TD>Tail anchored protein recruits Dnm1 via Mdv1/Caf4; also involved in mitochondrial fission</TD></TR><TR><TD>DNM1</TD><TD>-</TD><TD>-</TD><TD>GTPase, dynamin like protein involved in peroxisome fission</TD><TD>85.0</TD><TD>5.25</TD><TD>9,620</TD><TD>-</TD><TD>Also involved in mitochondrial fission</TD></TR><TR><TD> VPS1 (GRD1, LAM1, SPO15, VPL1, VPT26)</TD><TD>-</TD><TD>-</TD><TD>GTPase, dynamin like protein involved in peroxisome fission</TD><TD>78.7</TD><TD>8.15</TD><TD>5,960</TD><TD>-</TD><TD>Also involved in vacuolar protein sorting</TD></TR><TR><TD> VPS34 (END12, PEP15, VPL7, VPT29, STT8, VPS7)</TD><TD>-</TD><TD>-</TD><TD>Phosphatidylinositol 3-kinase</TD><TD>100.9</TD><TD>7.79</TD><TD>1,080</TD><TD>-</TD><TD>Forms complex with Vps15</TD></TR><TR><TD>INP1</TD><TD>-</TD><TD>-</TD><TD>Involved in retention of peroxisomes in mother cells</TD><TD>47.3</TD><TD>8.34</TD><TD>639</TD><TD>Peroxisomal membrane</TD><TD>Recruited to the peroxisome by binding to Pex3</TD></TR><TR><TD>INP2</TD><TD>-</TD><TD>-</TD><TD>Myo2 receptor, involved in peroxisome inheritance</TD><TD>81.5</TD><TD>9.41</TD><TD>736</TD><TD>Peroxisomal membrane</TD><TD>-</TD></TR><TR><TD>RHO1</TD><TD>-</TD><TD>-</TD><TD>GTP binding protein of the Rho subfamily of Ras like proteins, involved in actin assembly at the peroxisome</TD><TD>23.2</TD><TD>6.07</TD><TD>ND</TD><TD>-</TD><TD>Involved in <EM>de novo</EM> peroxisome formation recruited to peroxisomes by Pex25</TD></TR></TBODY></TABLE><H3>Knockout designs in our project</H3><H4>Pex 9</H4><P>Pex9 is a recently discovered import receptor for PTS1 proteins, which is induced by oleate
and is an import receptor for both malate synthase isoenzymes Mls1p and Mls2p. In order to get a completely empty reaction room, a Pex9 knockout was designed to prevent unintended protein import.</P><H4>Pex 31 &amp; Pex 32</H4><P>It has been shown that knockouts of Pex31 and Pex32 leads to an increased Peroxisomal size, but additionally the membrane permeability was affected (<ABBR title="Zhou, Yongjin J.; Buijs, Nicolaas A.; Zhu, Zhiwei; Gómez, Diego Orol; Boonsombuti, Akarin; Siewers, Verena; Nielsen, Jens (2016): Harnessing Yeast Peroxisomes for Biosynthesis of Fatty-Acid-Derived Biofuels and Chemicals with Relieved Side-Pathway Competition. In: Journal of the American Chemical Society 138 (47), S. 15368–15377. DOI: 10.1021/jacs.6b07394.">Zhou et al. 2016</ABBR>). This effect can be used as a tool for engineering membrane permeability by knocking out or overexpress both genes. A knockout would lead to an increased permeability and one could think of an opposite effect in case of overexpression, but this has not been shown yet.</P><H4>INP1</H4><P>The INP1 knockout was designed after the skype call of Prof Dueber, who recommended us, decoupling of peroxisomes from cytoskeleton in order to improve the secretion efficiency. INP1 is responsible for the tethering of peroxisome, which would inhibit the secretion of peroxisomes.</P><H4>POT1</H4><P>The only protein, which is imported by the Pex7 import machinery in saccharomyces is the 3-ketoacyl-CoA thiolase (POT1). A knockout of POT1 would enable utilizing the Pex7 import for proteins of interest, which cannot be tagged at the C-terminus with pts1, without having unintended import of other enzymes.

</P><H3>Genomic integration of our novel Pex5 import receptor</H3><P>After testing our new Pex5 import systems, which is completely orthogonal to the natural import, the next step would be to replace the endogenous system with our artificial import system. Therefore, an integration plasmid was designed with help of the previously described yeast toolbox, containing HO locus homologies and a hygromycin resistance (Figure 4). Afterwards the plasmid was transformed into the yeast strain which was created by our collaboration partner Aachen (double knockout strain Pex5 &amp; Pex7).</P><FIGURE><FIGCAPTION>Figure 4: Design of integration plasmid for integrating our orthogonal Pex5 import receptor.Therefore, an integration plasmid was designed with help of the previously described yeast toolbox, containing HO locus homologies and a hygromycin resistance</FIGCAPTION></FIGURE><P>The resulting yeast strain allows full control over the peroxisomal matrix proteome, by replacing the whole protein import machinery, which is the first step for creating our artificial compartment.</P><H3>Outlook</H3><P>Besides the genome engineering approaches, which were performed in our project one could think of more radical strategies for peroxisomal engineering. A final goal could be a “minimal peroxisome”, which contains only the proteins that are required for the biogenesis of the peroxisome and import of proteins and metabolites. On the one hand peroxisomal pathways could be redirected to cytosol or other organelles and one the other hand endogenous metabolic pathways could be redirected to our novel artificial compartment by changing the protein localization signal in the yeast genome with help of the Cas9 system. All these strategies would allow tremendous improvements for metabolic engineering applications by creating an artificial compartment, which can be rational designed and customized for specific metabolic pathways.</P></DIV><BUTTON class="accordion"><H3 id="optogenetics">Optogenetic enhancements</H3><P>
Optogenetics provide a useful tool for controlling cellular processes with high spatial and temporal precision. These include the expression of certain genes as well as the interaction or separation of two proteins. As many of our toolbox’s aspects benefit from precise control, we wanted to include optogenetics as a way of increasing its variability. Our plans included optogenetically controlled protein import, control of compartment size and number, and control of product secretion.
</P></BUTTON><DIV class="panel"><H3>Introduction</H3><P>
Our optogenetic toolbox enhancements can be divided into three subgroups: controllable protein import via Pex5, controllable protein import via Pex7 and controllable gene expression. Each of these sub-projects are of different design which will be illustrated in the following.
</P><H3>Pex5 import with LOV2</H3>


LOV2 is an optogenetic protein derived from <I>Avena Sativa’</I>s Phototrophin 1. In its dark state the J$_{\alpha}$-helix located at the C-terminus is bound to the core of the protein. Upon irradiation with blue light (~460 nm), a covalent bond between a cysteine residue on the LOV2 protein and a flavin mononucleotide chromophore causes the J$_{\alpha}$-helix to unfold, which in turn exposes the C-terminus
 <A href="https://www.ncbi.nlm.nih.gov/pubmed/26513473"><ABBR title="Spilltor J., Strickland D, Glotzer M., Tucker CL (2016).Optical Control of Peroxisomal Trafficking.
">
      (Spiltoir <I> et al</I>, 2016)
  </ABBR></A>


. This property is very useful, as short amino acid sequences can be attached to this end of the LOV-protein, for example a peroxisomal targeting sequence!
The idea for this project was to attach PTS1 to the C-terminus and the protein of interest to the N-terminus. Upon irradiation with blue light, the fusion protein would be imported into our compartment.
We used a mutated version of LOV2 whose C-terminus has an increased dark-state binding affinity. This is caused by the substitutions and T406A and T407A. These mutations greatly reduce the possibility of the J$_{\alpha}$-helix being exposed in the dark state.
Our PTS1 sequence consists of the amino acids LQSKL.
As a proof of concept for this construct we fused sfGFP to its N-terminus:<FIGURE><FIGCAPTION><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#yeast-toolbox">Lv1</A>-plasmid containing the sequence coding for the GFP-LOV2-PTS1 fusion protein</FIGCAPTION></FIGURE><P>
 This was done in order to visualize our experiment’s results during microscopy: upon successful import of the fusion protein, one would observe GFP fluorescence localized to our compartment. Otherwise, the whole cell would be illuminated. 

</P><H3>Pex7 import</H3><P>
The idea behind this project is to initially block the protein of interest’s PTS2 with a fluorescent protein which can be removed by an optogenetically activated TEV-protease. For this project we use the protein Phytochrome-B from <I>Arabidopsis thaliana</I> and its interaction partner PIF6. These two proteins, also derived from Arabidopsis thaliana, bind together upon irradiation by red light (660 nm) and separate upon irradiation with far-red light (780 nm). We used this property to activate a split version of a TEV-protease whose split halves were each fused to one of the two optogenetic proteins.
</P><P>
The TEV-protease was obtained from the Biobrick BBa_K1319004. This variant contains the anti self-cleavage mutation S219V. Using overhang-PCR we created a split version of the Biobrick protease based on work done by Wehr et al. [2] and the iGEM team Munich 2013. The split was made between amino acid 118 and 119.
</P>

Our construct for attaching proteins N-and C-terminally is highly variable: it consists only of a TEV-cleavage site, the PTS2 sequence and a short linker and was designed as a 3b-part for the yeast-toolbox(LINK). This means that we can attach any protein to its N- or C-terminus we desire. 
We planned on attaching different fluorescent proteins to each sides of the 3b-part in a Lv1-ligation(LINK). For our experiment we planned on using the pairs mTurquoise-mVenus and GFP-mRuby.
<FIGURE><FIGCAPTION><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#yeast-toolbox">Lv1</A>-plasmid containing the sequence coding for the GFP-PTS2-Ruby fusion protein</FIGCAPTION></FIGURE><P>
The two other constructs were planned as follows: Phytochrome B was fused to the C-terminal TEV-half, PIF6 was fused to the N-terminal TEV-half.
The PhyB-TEV2 part and the TEV1-PIF6 part were supposed to be inserted into a shared plasmid via a LV2 Golden Gate ligation.
Finally, the Lv2-plasmid and the remaining Lv1 construct were to be co transformed into S. Cerevisiae.
Our experiment consisted of illuminating one sample with red light (660nm) while keeping another sample in the dark. Fluorescence microscopy would then be used to check whether the import was successful [3]. If cleavage and subsequent protein import was successful, fluorescence of one protein would be localized to the compartment while that of the other would be observed throughout the cell.


</P><H3>Optogenetically controlled gene expression</H3><P>
This project is based on work done by Weber et Al. [3]. Using the interaction between Phytochrome B and PIF6 they designed an optogenetic switch for enabling and disabling transcription of a chosen gene. It is based on the tetracycline operon and the transcription factor VP16. The tetO operator is located upstream of a minimal promoter which in turn is located upstream of the gene of interest. The tetR repressor binds to the tetO sequence. Fused to it is PIF6. Phytochrome B is fused to the transcription factor VP16. Upon illumination with red light, Phytochrome binds to the tetR-PIF6 complex. VP16 is now located in close proximity to the minimal promoter, which enables the RNA-polymerase-2 to start transcription of the gene of interest.

We designed a promoter part for the Dueber toolbox(LINK) which consists of tetO and the minimal promoter region. This can be used as a promoter in a Lv1-ligation(LINK)for any desired gene of interest (a GFP-taggedPEX 11 in our example).
</P><FIGURE><FIGCAPTION><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#yeast-toolbox">Lv1</A>-plasmid containing the sequence coding for the GFP-Pex11 fusion protein with the tetO-pmin promoter</FIGCAPTION></FIGURE><P> Transformation into <I>S. Cerevisiae</I> is accompanied by co-transformation of a Lv2 plasmid containing both the tetR-PIF6 and PhyB-VP16 constructs:
</P><FIGURE><FIGCAPTION><A href="https://2017.igem.org/Team:Cologne-Duesseldorf/Design#yeast-toolbox">Lv2</A>-plasmid containing the sequences coding for the PhyB-VP16 and tetR-PIF6 proteins respectively</FIGCAPTION></FIGURE><P>
Verification methods depend on which gene is expressed. See size and number(LINK to size-number) and secretion(LINK to secretion) for details. 

</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>