"
Page
Discussion
View source
History
teams
Log in
 
Team:Sheffield/WetLabJournal
From 2014.igem.org
Attributions ▾
Acknowledgements
Sponsors
Notebook ▾
Lab Protocols
Wet Lab Journal
Dry Lab Journal
Policy and Practices ▾
SocioBricks
SocioBrick Registry
Notions of Responsibility
Industry Collaboration
Synthetic Biology @ Sheffield
Lab Notation
Outreach
Collaboration
Results ▾
FadR Characterisation
Lipase Assay
Keratinase Assay
Modelling ▾
Product Design
  ↳  Budget
  ↳  Premium
Project
Team ▾
Meet the Team
Official Team Profile
Gallery
Home
Wet Lab Journal
Wet Lab Journal
Week 1
Monday 23rd June
Make up 100ml of LB Broth and 100ml LB Agar .
Make an overnight starter culture of BBa_K258008 to test sterile technique.
Tuesday 24th June
Generate Chemically competent E.Coli.
Make up 8 flasks of 100ml starter culture.
4 flasks (A-D) were checked each time to take the OD600 readings.
OD600 readings to show when the cells are in exponential growth phase and so ready to be made competent.
A
B
C
D
1.25h
0.077
0.078
0.082
0.096
3.25h
0.567
0.674
0.714
0.809
Aliquot into 24 falcon tubes. The cells will need to be spun down in 2 rounds.
72 eppendorf tubes will be produced, labelled ch. comp, dh5alpha.
Store at -80°C.
Week 2
Monday 30th June
Make 5mL overnight starter cultures of the following:
Tube
BioBrick
Antibiotic
1
BBa_K258008
Chloramphenicol
2
BBa_K215107
Tetracycline
3
BBa_K861061
Chloramphenicol
4
BBa_K215002
Ampicillin
5
BBa_K1149021
Chloramphenicol
6
BBa_K861060
Chloramphenicol
5μl of an appropriate antibiotic was added to each culture to allow selection for cells containing biobrick plasmids.
Tuesday 1st July
Conduct 6 mini-preps , one for each of the biobrick cultures, to produce 6 Eppendorfs containing purified DNA.
Wednesday 2nd July
Run a DNA gel using the purified DNA collected yesterday to check DNA is pure.
Use an 8 well comb to produce the gel.
The table below shows the placement of each DNA sample within the Gel.
Well
Biobrick
Part Function
1
1kb gene ruler
2
BBa_K1149021
Promoter, RBS, Lipase
3
BBa_K258008
ABC transporter
4
BBa_K861060
pFadR promoter
5
BBa_K861061
pFadR Promoter + RFP
6
BBa_K215107
ABC transporter (strong)
7
BBa_K215002
pLAC +RBS +secretion signal + streptavidin binding tag
The gel was run at 100v for 45 minutes (due to time constrains).
Below is the image taken of the Gel on the Gel dock.
4 LB agar plates were made, 2 ampicillin (100 μl) and 2 chloramphenicol (25 μl).
Nanodrop the DNA gained from the Mini-prep to generate concentrations of extracted DNA for all 6 biobricks.
Results are shown below:
Biobrick
Concentration (ng/μl)
BBa_K1149021
115.0
BBa_K258008
075.9
BBa_K861060
100.3
BBa_K861061
130.7
BBa_K215107
043.3
BBa_K215002
125.0
Transform cells to confirm competency
For this transformation We used the BioBricks BBa_K258008 and BBa_K215002 to check if the cells were competent.
Calculations are shown below:
BBa_K258008
75.9ng x X = 10ng x 100μl
X = 13.2μl
BBa_K215002
125.0ng x X = 10ng x 100μl
X = 8.0μl
Friday 4th July
The linearised TliA and KerUS synthesised DNA is cloned into a pJet1.2 plasmid using blunt end ligation .
This ligation mix was then used to transform 3 eppendorfs of chemically competant cells (made on tuesday 25th):
Into eppendorf 1, pipette 5μl of the ligation mix containing TliA DNA.
Into eppendorf 2, pipette 5μl of the ligation mix containing KerUs DNA.
Into eppendorf 3, pipette 5μl of dH2O
Incubated for 45 minutes due to time constraints.
Constructs plated on ampicillin LB agar plates to select for the correct colonies.
Plates were checked on saturday AM and positive colonies were observed
Week 3
Monday 7th July
Make 200ml LB Agar and 200ml LB Broth .
Make overnight cultures of positive colonies of pJet 1.2 with TliA and KerUS .
Make overnight cultures of E. coli MC1000 .
Tuesday 8th July
Miniprep the Tlia and KerUS cultures to produce purified plasmid DNA.
Nanodrop the purified DNA to determine the concentration of DNA.
TliA - 5.2ng/μl
KerUS - 3.4ng/μl
Produce glycerol stocks of the Tlia and KerUS cells for storage (retain only if insert confirmed).
Make chemically competent MC1000.
Measure the OD600 of the MC1000 cells using spectrophotometer
Flask 1 - 0.803
Flask 2 - 0.624
Blank - 0.066
Decant the flasks of MC1000 into 50ml falcon tubes. We have 4 tubes of full MC1000, the 5th one is 21ml MC1000 and 9ml water, the 6th one is all water -to ensure the centrifuge is balanced (weight checked too).
Make 10ml overnight cultures of pJet 1.2 containing TliA or KerUS
Wednesday 9th July
Repeat yesterdays miniprep as it didn't yield enough DNA.
Began with two 10ml starter cultures, one of pJet 1.2 TliA and one of pJet 1.2 KerUS.
Decanted by hand into falcon tubes to give four 5ml cultures.
A Restriction Digest was carried out on the Tlia and KerUS minipreps, as well as the FadR vector that was miniprepped last week.
TliA and KerUS were cut with xbaI and pstI, whilst FadR was cut with speI and pstI.
Thursday 10th July
Run Vector, TliA and KerUS DNA on a gel to ensure it has been cut correctly. Once confirmed the DNA was then extracted from the gel.
Add 12.5μl 5X loading buffer to the DNA samples (50μl each) - total of 62.5μl in each eppendorf tube.
Load each sample, using two wells each if necessary.
Run at 100V for 60 minutes.
Visualise the gel.
The bands for TliA and Vector DNA were the correct size.
The band for KerUS showed that the DNA had been cut incorrectly. The two samples of DNA that have been cut correctly are now extracted from the gel.
The gel extraction was then nanodropped to determine DNA concentration.
Vector: 14.3ng/μL
Tlia: 190.9ng/μL
A ligation reaction was carried out using the following amounts of reactants;
3.5 μl vector
0.2 μl lipase
2 μl buffer
1 μl ligase
13.3 μl dH2O
leave for 10 minutes at room temperature
Add 5μl ligation reaction mix to 200μl DH5α and transform .
NOTE: did not work -no positve colonies following antibiotic resistance selection on an LB agar plate
We also did some work to help us characterise the pFadR promoter.
Due to a mistake over biobrick numbers, our overnight contained pFadR without RFP, and so we used the day to test our experimental procedures.
Day cultures were made.
1 x 5ml of empty DH5α
3 x 5ml of DH5α containing BBa_K861060
Day cultures were placed in the incubator at 37°C, 170rpm for 3 hours
Lipid stocks were made using Triton to solubilise.
12 samples were made to be assayed;
1: 5ml DH5α + 0% lipid
2: 5ml DH5α + 1.0% olive oil
3: 5ml DH5α + 1.0% oleic acid
4: 5ml DH5α + BBa_K861060 + 0% lipid
5: 5ml DH5α + BBa_K861060 + 0.1% olive oil
6: 5ml DH5α + BBa_K861060 + 1.0% olive oil
7: 5ml DH5α + BBa_K861060 + 0.1% oleic acid
8: 5ml DH5α + BBa_K861060 + 1.0% oleic acid
9: 5ml LB + 1% olive oil
10: 5ml LB + 1% olive oil + chloramphenicol
11: 5ml LB + 1% oleic acid
12: 5ml LB + 1% oleic acid + chloramphenicol
Samples were incubated at 37°c
OD600 Readings were taken at hourly time points
100μl of each sample was pipetted into wells in a Transparent Greiner 96 microtitre plate.
As well as the 12 samples, 5 Control Wells (blanks) were created, without cells:
A: LB only
B: 0.1% Oleic Acid only
C: 1.0% Oleic Acid only
D: 0.1% Olive Oil only
E: 1.0% Olive Oil only
How to use the Plate Reader
Turn on computer
Select ...
Select open plate icon
Place microtitre plate into cassette holder
Select close plate icon
Open lid of plate reader
Twist wheels - for OD select white and red wires, for fluorescence select yellow and blue wires
Select spanner icon on program to choose between OD and fluorescence
Select Measure icon
Edit ….
Select Excel icon
Open A600 for OD, RFP Charlotte for Fluoresence
Select update icon
Scroll down to Table 2 to view results
Friday 11th July
FadR assay analysis
Week 4
Monday 14th July
Make overnight cultures.
DH5a + pSB3k3
DH5a + BBa_K861061
DH5a + BBa_K861060 x2
DH5a
DH5a + pJeT 1.2 TliA
DH5a + pJeT 1.2 KerUS
Tuesday 15th July
Overnights of pJet 1.2 KerUS and TliA containing cells didn't grow, and so the time was used to make DNA stocks of BBa_K861060 and BBa_K861061.
Overnights were Miniprepped and then Nanodropped .
dH2O was used instead of the elution buffer in the miniprep kit.
FadR - 80.6ng/μl
FadR - 75.9ng/μl
RFP - 50.3ng/μl
Make Overnights of pJet 1.2 + TliA and KerUS.
Wednesday 16th July
Miniprep Tlia and KerUS cells.
Nanodrop
TliA - 522.0ng/μl
Kerus - 264.9ng/μl
Restriction Digests were made with TliA, KerUS and FadR containing plasmids.
The following amounts of DNA were cut with restriction enzymes and the final reaction mixture bought up to 50μl.
2μl Tlia
4μl KerUS
12.5μl FadR
Run a Gel to seperate out the restriction fragments.
Do a Gel extraction on the bands which are the correct size. The DNA was eluted into dH2O. Nanodrop
TliA - 5.7ng/μl
KerUS - 7.0ng/μl
FadR - 5.7ng/μl
A Ligation reaction was set up containing either cut TliA or KerUS and FadR vector.
After an hour, the reaction mix was added to chemically competent DH5α which were transformed and plated on antibiotic containing LB agar plates.
Plates were incubated overnight at 37°C
Thursday 17th July
No positve colonies from the TliA and KerUS with FadR vector transformation
Day cultures of BBa_861060, BBa_861061 and pSB3k3 were made.
Once grown (leave for around 5 hours), 10μl of these cells were then plated onto either 25μl 1% Oleic Acid, 25μl 1% Olive Oil or 25μl LB agar plates.
Cells were left in the incubator overnight.
Thursday 17th July
No zone of clearing observed on olive oil containing plates
Week 5
Tecan data A second run was done on the Tecan. Three different cells were used;
BBa_k861060
BBa_k861061
pSB3k3
Each of these cells were added to solutions containing 0.5% Olive oil, 1% Olive oil and 2% Olive oil. Solutions without cells added were also used as a control.
200μl of each solution was pipetted into a 96 well plate with a clear bottom and white wells.
All unused wells were filled with LB to prevent evaporation of samples in the central wells. Fluorescence Microscopy
Set up overnights of the following;
PSB3K3
pSB3K3 + 1% Olive Oil
pSB3K3 + 1% Oleic Acid
BBa_K861060
BBa_K861060 + 1% Olive Oil
BBa_K861060 + 1% Oleic Acid
Add 10μl overnight cultures to microscope slide cover slips, allow to dry.
add 10μl 4% paraformaldehyde, dry for 30 minutes.
Wash 3 x with PBS buffer. Apply 10μl Prolong Gold Antifade Mountant with DAPI (Life Technologies). Fix to microscope slide, leave to dry overnight.
Use fluorescence Microscope to visualise cells with DAPI and RFP suitable channels. Maintain the same exposure time between images so they are comparable.
Following fresh ligation and transformation positive colonies were achieved for lipase with fadR responsive promoter in vector. Folllowing restriction digestion with xbaI and pst1. Negative result.
Overnights of BBa_k861060, BBa_k861061 x2, TliA x2, PSB3K3 pellets frozen for minipreps later
Week 6
Monday 28th July
Miniprep of overnights from Friday (BBa_k861060, BBa_k861061 x2, TliA x2, PSB3K3)
Plasmid DNA stored in -20
Produce 6 flasks of LB agar .
Overnights of the following;
KerUS
TliA
BBa_k823017
PSB3k3
Tuesday 29th July
Make up new stocks of soluble Olive oil (in Triton) .
Make up 8 of agar plates with tributyrin in order to test lipase activity:
Differing concentrations of Tributyrin was added to either full strength or dilute LB agar. Any Olive oil was added to the empty plate and swirled to mix.
0.5% tributyrin
0.5% tributyrin + 2% Olive Oil
1% tributyrin
1% tributyrin + 2% Olive Oil
Make M9 Media
Too 800ml distilled H2O add;
44g of Na2.2H2O
15g of KH2.H2PO4
2.5g of NaCl
5g of NH4Cl
Pour 800ml of distilled H2O into 1L measuring cylinder
Disolve with magnetic flea and stirrer
Transfer media into 2 durans and autoclave.
To 10mL water add either;
1.2g of MgSO4
1.1g of CaCl2
Shake/invert until salts dissolve
Transfer into conical flasks, cover with cotton wool and foil, autoclave.
Make Glycerol stocks of pJet 1.2 KerUS, pJet 1.2 TliA, PSB3K3, BBa_K823017.
Made 6 LB Agar stocks.
Nanodrop of pJet 1.2 TliA and pJet 1.2 Lipase, BBa_k861060
Restriction and digest of pJet 1.2 TliA and pJet 1.2 Lipase with BBa_k861060. Transform into DH5a.
Wednesday 30th July
One positive colony for TliA in the FadR vector in DH5a -ovenight culture of this
Thursday 31st July
Miniprep the potential TliA in the FadR vector.
Nanodrop 54ng/μl Restriction digest of potential TliA in the FadR vector with NdeI and EcoR1 to confirm - failed
Make glycerol stocks of overnights potential TliA in the FadR vector
Friday 1st August
Restriction digest undertaken.
TliA and a construct thought to contain the gene for TliA in the FadR vector were restricted using Xba1, Nde1 and both. A gel of the restriction digest was made
The gel showed that the fadR plasmid did not contain the lipase.
Week 7
ABC transporters into cells Overnights of both ABC Transporters (BBa_K258008, BBa_K215107), T1SS signal-strep binding tag (BBa_K215002).
Miniprep the above plasmids and transform into stocks of chemically competant DH5a.
Grow overnights of positive colonies, make competant and store at -80 in preperation for the transformation of a lipase containing plasmid.
Cloning
Miniprep the pJet 1.2 TliA and promoters BBa_J23101, BBa_J23110 and BBa_J23100.
Elute into 50&#956 of dH20.
Nanodrop
TliA - 103ng/μl
BBa_J23101 - 278ng/μl
BBa_J23110 - 408ng/μl
BBa_J23100 - 380ng/μl
Carry out a Restriction digest on the TliA and all three promoter plasmids - already had digested kerUS.
Slight variation to usual protocol: After 2 hours, 3μl CIP was added to the promoter plasmids and then returned to the heat block for another hour.
After 2 hours, the lipase was removed from the heat block. Make a gel
Restriction digestion mixes were all run on a DNA agarose gel and the correct sized bands cut.
DNA was extracted from the gel.
The TliA and kerUS inserts were ligated into each of the three different promoter strength harbouring vectors.
The ligation was left in the fridge overnight.
Ligation mixes were transformed into NEB5a cells
Positive colonies were seen for TliA and kerUS in the BBa_J23110 containing vectors
Colony PCR confirmed that TliA and kerUS had successfully been cloned into the BBa_J23110 vector
These colonies were grown in liquid culture and -80°C stocks were created
Tecan run with minimal media
DH5a plus the following plasmids were used for the tecan runs
FadR-Rfp
FadR
J23110
2 x 50ml falcon tubes of M9 media was made
35ml of dH2O was added from duran
10ml of M9 salts was added from duran
2ml of casamino acids was added from falcon tube
100µl of MgSO4 was added from 50ml conical flask
1ml of 20% glucose solution was added to one falcon tube, 1ml of dH2O was added to the      other
5µl of CaCl2 was added from 50ml conical flask
topped up with dH2O to make 50ml solution
Lipid was added to the M9 Solution
Total volume created = 4ml, in 6ml bijou tubes
Since the Olive oil:Triton X-100 ratio was 1:2, the following concentrations were made:
Solution
1%
2%
3%
4%
M9 Media
4.0ml
3.88ml
3.76ml
3.52ml
Olive oil:Triton
0μl
120μl
240μl
480μl
The 4.0% stock was made from the falcon tube without glucose added as a supplementary carbon source.
the 0.0%, 1.0% and 2.0% were made from the falcon tube with glucose added as a supplementary carbon source
Transformed cells & antibiotics were added to all 4 concentrations of M9/Lipid media.
96-Well plate was made.
200μl of each solution containing the 3 different promoters were pipetted into 4 wells to create 4 technical repeats.
200μl of each solution devoid of cells were pipetted into 3 wells in the region of the plate (B-G, 10-11)
Week 9
Monday 18th August
Make overnights of kerUS and BBa_J23110
Tuesday 19th August
Make 4 day cultures ;
5ml KerUS + Nail
5ml KerUS + Hair
5ml BBa_J23110 + Nail
5ml BBa_J23110 + Hair
These were left in the shaking incubator for the cells to grow and produce and secrete keratinase.
Autoclave hair, nail and feather samples to sterilise. This was done in glass beakers with foil 'lids'.
Restreak KerUS and BBa_J23110.
Make overnights of kerUS and BBa_J23110.
Wednesday 20th August
Overnights of BL21 .
Thursday 21st August
Make 2 5ml day cultures of BL21.
Allow around 3 hours for them to reach the exponential growth phase.
Take an OD reading
If the OD600 is over 0.6, use these innoculations to produce Chemically competant cells.
Miniprep KerUS BBa_J23110
Transform the chemically competant BL21 strain using 250ng of the purified KerUS BBa_J23110 DNA.
Plate the following;
100μl of control onto an antibiotic free plate.
100μl of control onto an ampicillin plate.
100μl of transformant onto an ampicillin plate.
Remaining 900μl spun down and resuspended in 100μl of LB and plated onto an ampicillin plate.
Transform TliA BBa_J23110 into stocks of ABC transporter (BBa_K258008) containing competent cells
Pour plates containing Lb Agar, Lb Agar containing 1% casein, Lb Agar containing 1% gelatin and Agar containing 1% gelatin.
Friday 22nd August
Positive colonies of all transformations
Overnights of positive colonies of cells containing TliA/BBa_J23110/BBa_K258008 , KerUs/BBa_J23110 and BBa_J23110
Saturday 23nd August
Plate out KerUs containing overnights onto Casein, Gelatin and Gelatin LB agar plates. BBa_J23110 was used as a negative control. Staphylococcus aureus was used as a positive control.
Incubate for 8 hours, then flood plates in saturated ammonium sulphate.
Week 10
Monday 25th August
Make overnights of KerUS/BBa_J23110, tliA/BBa_J23110, tliA/BBa_J23110/BBa_K258008 and BBa_J23110.
Autoclave four 1 Litre conical flasks each containing 100ml of LB Broth.
Tuesday 26th August
Innoculate each of the autoclaved flasks with 1ml of each of the overnights. Add the correct antibiotic to each.
Incubate at 30°C to promote protein expression.
Give 5 or 6 hours for the cells to reach exponential growth phase and produce an adequate amount of protein to assay.
After 6 hours, samples were taken for analysis by protein gel.
Into 4 eppendorfs, pipette 1.5 ml of each innoculation.
These are spun down and the supernatant removed into a second tube. Following the addition of 150μl trichloroacetic acid (TCA) and kept in the 4 degree overnight.
Into 4 more eppendorfs, pipette 0.5ml of each innoculation.
These are spun down, the supernatant poured away, the pellet labelled and kept in the -20 degree.
Remaining culture was centrifuged, sterile filtered and loaded into Vivaspin 20 10,000 MWCO columns (GE Life Science) (tliA without exporter was excluded) add 5ml of culture supernatant. These were spun down at 3,500 rpm for 8 minutes.
Supernatant that had passed through the column was discarded, and more supernatant was added into the top of the column. This was repeated until around 15ml of supernatant had passed through the column.
Around 1ml of concentrated supernatant was then removed from the top of the column.
Overnight liquid cultures of cells containing kerUS (no promoter), tliA/BBa_J23110 and pSB1C3
Wednesday 27th August
To prepare the protein sample to be loaded onto a gel
To the cell pellet add 100μl 2X SDS buffer. Resuspend the cells in this solution.
Centrifuge the 1.5ml of supernatant/TCA mix for 15 minutes at 13,300rpm, remove supernatant, resuspend pellet in 900μl acetone, repeat centrifugation step, remove supernatant, allow remaining acetone to evaporate then add 50μl of SDS buffer (a ratio of ~1:20)
To the 1ml concentrated supernatant, add 1ml of SDS buffer (a ratio of 1:1)
Place all of these samples into a heat block at 95°C for 5 minutes. The cell pellet may need to be vortexed and left for an additional 5 minutes.
Make a 12% SDS-PAGE gel . Load 5μl cell lysate samples and 15μl supernatant (concentrated or non-concentrated) onto the gel
Stain the SDS-PAGE gel with InstantBlue (Expedeon) overnight
Miniprep plasmids from overnight cultures, set up restriction digests with EcoRI and PstI (add CIP to the acceptor vector). Run donor mixes on a DNA gel and isolate DNA from bands of correct size
Ligate isolated cut kerUS and tliA/J23110 with cut pSB1C3 overnight
Thursday 28th August
Transform both ligation mixes into NEB5a and plate on chloramphenicol containing LB agar plates
Friday 29th August
Positive colonies on both LB agar plates. Colony PCR to confirm insertion of kerUS or tliA/J23110.
Overnight liquid cultures of a positive colony from each.
Saturday 30th August
Miniprep overnight cultures. Send some for sequencing with sequencing primers. Once confirmed send on to the BioBricks Registry
Lab Protocols
Strains
Escherichia coli DH5α - NEB
Escherichia coli MC1000 - Donated by Dr. Graham Stafford
Antibiotics
Antibiotic
Concentration(μg/ml)
Ampicillin
100
Chloramphenicol
25
Tetracycline
20
Primers
Primer Name
Nucleotide Sequence
pSB1C3 seq f
caactcgagtgccacctg
PSB1C3 seq r
gcctttttacggttcctg
TliA seq f
gcagatctgagcaccagc
TliA seq r
gtccagagcacgaaagac
KERUS seq f
ggtgatggtcaggatagc
KERUS seq r
ctgccacaacaacaacac
GGTGATGGTCAGGATAGC
CTGCCACAACAACAACAC
pSB1C3 seq f: caactcgagtgccacctg – 18-mer, 3412...3429, 56degC
pSB1C3 seq r: gcctttttacggttcctg, 18-mer, 1731...1748, 52degC
TliA seq primer f: gcagatctgagcaccagc 18-mer, 711...728, 56degC
TliA seq r: gtccagagcacgaaagac – 18-mer, 831...848, 54degC
Media and Stock Solutions
Luria-Bertani (LB) Broth
LB Broth was made using a ready mix powder from Sigma, 20g/L in ultrapure water. This consists of: 10 g/L Tryptone; 5 g/L Yeast Extract; 5 g/L NaCl. Autoclaved at 121 oC for 15 min to sterilise.
Luria-Bertani (LB) Agar
LB agar was made using a ready mix powder from Sigma, 35 g/L ultrapure water. This consists of 15 g/L Agar and as in LB Broth. Autoclaved at 121 oC for 15 min to sterilise.
5 x M9
5 x M9 was made using the following up to 1 L of ultrapure water: 64 g Na2HPO4-4H2O; 15 g KH2PO4; 2.5 g NaCl.
Working stock, 1 X M9 was made by adding into up to 1 L of ultrapure water: 200 ml of 5 x M9; 2 ml 1M MgSO4; 20 ml 20% glucose/0.4% Glycerol; 100 μl 1M CaCl2
50 x TAE
50 x TAE was made up with: 121 g Tris Base (Trizma); 28.55 gl acetic acid; 9.3 g EDTA made up in 500 ml ultrapure water.
Working stock, 1 x TAE was made by diluting 10 ml in 500 ml ultrapure water.
10 x TBS
10 x TBS was made up with: 500 ml 1M Tris.HCl, pH 7.4; 300 mM NaCl made up to 1L with ultrapure water.
Working stock, 1 x TBS by diluting 100 ml 10 x TBS up to 1000 ml with ultrapure water.
1 x PBS
1 x PBS was made up using Phosphate buffered saline tablets from Sigma.
One tablet dissolved in 200 mL of deionized water yields 0.01 M phosphate buffer, 0.0027 M potassium chloride and 0.137 M sodium chloride, pH 7.4, at 25 °C.
Upper Tris Buffer
Upper Tris Buffer was made up by adding 6.05 g Tris Base in 100 ml deionised water, pH 6.8. Add 0.4 g SDS.
Lower Tris Buffer
Lower Tris Buffer was made up by adding 36.4 g Tris in 100 ml deionised water, pH8.8. Add 0.8 g SDS.
10 x SDS Running Buffer
10 x SDS running buffer was made up by adding: 30.3 g Tris Base; 144.1 g Glycine; 10 g SDS to 1 L deionised water to pH 8.3.
Working concentration, 1 x SDS Running Buffer is made by adding 100 ml of 10 x SDS Running buffer to 10 ml deionised water.
Solubilising lipids
A 10% stock solution of triton-x100 was made by adding 1ml of Triton to 9ml sterile (autoclaved) water and incubated at 37oC, 150 rpm until solubilised.
Filter sterilise and add 10% Triton to Olive oil/Oleic acid in a 2:1 ration. Incubate at 37oC, 150 rpm to solubilise.
Protocol 1: Make overnight starter cultures
Materials and Equipment
Stripette, bunsen burner, sterile loop, falcon tubes, LB broth.
Time
Prep: 10 minutes
Run: 16 hours
Procedure
Use a stripette to take 5ml of LB broth from a 250ml conical flask.
Dispense into a falcon tube.
Sterilise a metal loop in a flame.
Take a culture from an agar plate using the sterile loop and put into one of the falcon tubes.
Agitate.
Replicate this using scrapings from a clean agar plate and a fresh tube to use as a positive control.
Put the tubes into the incubator overnight (37c, 150rpm) to grow up the cultures.
Protocol 2: Generate chemically competent E. Coli
Materials and Equipment
LB broth, starter culture, P1000, P200, pipette tips, incubator, cuvettes, spectrophotometer, Virkon, falcon tubes, ice, weighing scales, centrifuge, CaCl2, 20% glycerol, stripette, eppendorf tubes and -80°C freezer.
Time
Prep: 40 minutes
Run: 5 hours
Procedure
Grow cells
Take 1ml starter culture and add to 100ml LB broth.
Incubate at 37°C, 150rpm.
Check every hour by testing the optical density at 600nm (OD) using a spectrophotometer to determine whether enough cells are present in the culture.
0.600 OD is ideal, this is the point at which the cells are in the exponential growth phase.
Take 1ml of culture into a cuvette to measure; dispose of this after use in Virkon.
Remove cells
Pour 30ml aliquots from the flasks into falcon tubes; 3 tubes per flask.
Put tubes on ice for approx 10mins; all equipment used from this point on must be cold e.g. pipette tips.
Weigh the falcon tubes and pair together similar weight tubes for balance in the centrifuge; tubes paired together must weigh within 0.5g of each other.
Spin at 4°C, 4000rpm for 5mins.
After the tubes have all been spun, pour off the supernatant to remove the LB broth, leaving cells in a pellet. Put tubes back on the ice.
Make cells competent
Add 1ml of CaCl2 to the cells, use the pipette to pull the liquid and cells up and down to resuspend.
Once resuspended, add another 14ml of CaCl2; 15ml total volume.
Put back on ice for approx. 10mins to allow cells to acclimatise at the temperature with the CaCl2.
Re-weigh and pairs tubes again for balance.
Spin again at 4°C, 4000rpm for 5mins.
After tubes have been spun, leave on ice for 5mins. Pour off the supernatant?
Aliquot
Add 600μl of 20% glycerol to each falcon tube.
Label eppendorf tubes.
Aliquot 200μl from each falcon tube into eppendorf tubes (3 per falcon).
Freeze at -80°C.
Protocol 3: Mini-Prep
Materials and Equipment
Ice, starter cultures, mini-prep kit, centrifuge, P100 pipette, weighing scale, Virkon.
Time
Prep: ?
Run: ?
Procedure
Extract cells
Match up weights of falcon tubes so they are paired within 0.5g.
Spin down starter cultures in a centrifuge; 5mins, 4°C, 4000rpm.
Pour off supernatant into Virkon.
Resuspend cells
Add 250μl of P1 resuspension buffer to each falcon tube using P1000.
Resuspend the cells.
Use pipette to move suspension into separate, labelled eppendorf tubes.
Lyse cells
Add 250μl of P2 buffer to each eppendorf tube.
This will lyse cells - a blue colour will indicate they have been lysed.
Do not leave for more than 5mins.
Neutralise cells
Add 350μl of N3 neutralisation buffer.
Once the reaction is complete, the liquid will turn clear/white.
Purify DNA
Spin down the cells in a centrifuge; 10 mins, 17000g, 4°C.
Pour supernatant into mini-prep columns.
Centrifuge columns for 1 min, 17000g, 4°C.
Discard flow through.
Add 500μl of PB buffer to each column.
Centrifuge columns for 1min, 17000g, 4°C.
Discard flow through.
Add 750μl PE buffer to each column.
Centrifuge columns for 1min, 17000g, 4°C.
Pour off supernatent.
Centrifuge columns for 1min, 17000g, 4°C.
Discard bottom of the mini-prep.
Move column to a new labelled eppendorf.
Add 50μl of elution buffer.
Centrifuge columns for 1min, 17000g, 4°C.
Discard column.
Immediately place on ice; store in the B56 sewer sample freeze.
Protocol 4: Run a gel
Materials and Equipment
Agarose mix, TAE buffer, Flask, Microwave, Ethidium bromide, P10, Tips, Autoclave tape, Gel tray, Comb, Buffer tray, Loading buffer, Dna, Eppendorf, Transilluminator.
Time
Prep: 30 mintues
Run: 1 hour
Procedure
Make up a 1% agarose gel
Weigh out 0.4g agarose.
Measure 40ml TAE buffer.
Add both into a 250ml conical flask.
Swirl.
Microwave on full for 2 minute, swirling at intervals to ensure all the agarose has dissolved.
Run side of flask under tap to cool, until comfortable to hold in a gloved hand.
Prepare the gel
Add 1μl ethidium bromide to the conical flask containing the melted agarose.
Pour this solution smoothly into a gel tray (sealed with autoclave tape) and add a comb which allows for either 8 or 13 DNA samples to be run at once.
Leave the gel to set for approximately 15 - 30 minutes.
Once the gel is set remove the comb and autoclave tape.
Place the gel tray inside the buffer tray and fill the remaining space with TAE buffer.
Load 'checking' samples
Load 5μl of 1kn ladder into the first well of the gel.
Pipette 2μl 5x loading buffer onto a sheet of parafilm.
pipette 8μl DNA onto the loading buffer and pipette up and down to mix.
Resting the pipette tip on the back of the well, load 8μl of sample.
Repeat for each sample.
Run the Gel at 100v for 60 minutes.
Load 'extraction' samples
Load 5μl of 1kn ladder into the first well of the gel.
Add 5x loading buffer into each sample, to give a final ratio of 1:4.
Resting the pipette tip on the back of the well, load as much sample as possible.
Repeat for each sample.
Visualise
Remove the agarose gel from the gel tray and place in the trans-illuminator.
Open UVP (on the computer desktop).
Select the type of gel and the exposure time.
Press capture to take an image.
Save image to the iGEM2014 folder.
Protocol 5: Pouring Plates
Materials and Equipment
Sterile hood, Flask, Agar mix, dH2O, antibiotic, empty plates, P200, tips.
Time
Prep: 15 minutes
Run: 20 minutes
Procedure
Use the sterile hood.
Make up 100ml LB Agar (Protocol 2).
Heat in the microwave as follows:
Power setting 4 for 3 minutes.
Wait for 6 minutes.
Power setting 4 for 2 minutes.
Wait to cool (until the flask is warm but comfortable to hold).
Add 100μl 1/1000 stock of appropriate antibiotic to the agar.
Pour 4 plates and leave inside the hood to cool and dry.
Protocol 6: Measure Concentration of DNA with NanoDrop 2000
Materials and Equipment
Nanodrop, P10, Tips, dH2O, Paper towel.
Time
Prep: 5 minutes
Run: 5 minutes
Procedure
Set up the NanoDrop2000, select “Nucleic Acid”, then “Routine Verification”.
Spray paper towel with distilled water, then clean metal nodules on metal part and lid.
Add 1μl of buffer solution to nodule.
Close lid.
Select 'Blank' on the programme.
Reopen lid.
Add 2μl of DNA Sample.
Close lid.
Select 'measure' on the programme.
Take note of the concentration (in ng/ml).
Clean nodules again with distilled water.
Repeat for different DNA sample.
Protocol 7: Transforming Cells To Ensure Competency
Materials and Equipment
Eppendorfs, P1000, P200, P10, Tips, Competent cells, Ligation mix, Ice, Heat block, LB broth, Incubator, Agar plates, Flame, Spreader, Ethanol
Time
Prep: 5 minutes
Run: 1 hour
Procedure
Using concentration measure by the nanodrop, calculate the volume of DNA sample required: C1V1 = C2V2.
Where C2 required is 10ng and V2 is 100μl.
Use the calculated volume of DNA to transform (Protocol 11) chemically competent cells before plating out.
Calculate the transformation efficiency in colonies per ng of DNA.
Protocol 8: Blunt End Ligation
Materials and Equipment
Eppendorfs, P10, tips, Reaction buffer, dH2O, Restricted DNA, T4 Ligase.
Time
Prep: 10 minutes
Run: 10 minutes
Procedure
Make up eppendorfs containing ligation mix, as shown:
10μl reaction buffer.
8μl water.
2μl Cut DNA.
1μl Cut plasmid.
1μl T4 ligase.
Flick to mix and leave for ten minutes.
Protocol 9: Chemical Tranformation
Materials and Equipment
Eppendorfs, P1000, P200, P10, Tips, Competent cells, Ligation mix, Ice, Heat block, LB broth, Incubator, Agar plates, Flame, Spreader, Ethanol.
Time
Prep: 5 minutes
Run: 1 hour 45 minutes
Procedure
Take an appropriate number of eppendorfs, each containing 100μl of competent E. coli cells.
Into each one, pipette 5μl of each ligation mix.
Leave a last one to act as a negative control.
Put the eppendorfs on ice for 30 minutes.
Heat shock eppendorfs at 42°C for 30 seconds.
Put on ice for 2 minutes.
Add 1ml LB broth to each of the eppendorfs and then incubate at 37°C for 60 minutes.
Remove cells from incubator and spin down at 13000RPM for 60 seconds.
Pour off the supernatant and resuspend the cells.
Near a flame, pipette each of the remaining cells onto individual plates (prepared earlier).
Sterilise a glass spreader using ethanol (flamed) and spread cells evenly until the surface of the agar appears dry.
Incubate at 37°C overnight.
Protocol 10: Sticky End ligation
Materials and Equipment
Eppendorf, p10, tips, reaction buffer, dH2O, restricted DNA, T4 Ligase.
Time
Prep: 5 minutes
Run: 30 minutes
Procedure
Make up eppendorfs containing the ligation mix, as follows:
2μ l reaction buffer.
10ng insert.
10ng plasmid.
1μ l T4 ligase.
Make up to 20μ l with dH2O.
Flick to mix and leave for 30 minutes.
Protocol 11: Restriction Digest
Materials and Equipment
Eppendorf, p10, p100, tips, restriction enzymes, ice, heat block.
Time
Prep: 5 minutes
Run: 1 hour
Procedure
To each eppendorf add:
10 units of restriction enzyme 1 (usually 1μl).
10 units of restriction enzyme 2 (usually 1μl).
1μg DNA.
5μl 10x buffer.
Make up to 20μl with dH2O.
Flick to mix.
Incubate at 37°c for 60 minutes.
To stop the reaction, heat the sample to an appropriate temperature to inactivate the restriction enzyme used.
Put on ice.
Protocol 12: Glycerol Stocks
Allows stocks of cells to be kept in the -80
Materials and Equipment
P1000 pipette, tips, eppendorf tubes, microwave, 100% glycerol, overnight cultures of cells.
Time
Prep: 5 minutes
Run: 5 minutes
Procedure
Take 800μl of overnight cells using a P1000 pipette and add into separate, labelled eppendorf tubes.
Microwave the 100% glycerol for 5 seconds.
Take 200μl of the glycerol using a P1000 pipette and add into each eppendorf tube.
Place on ice immediately.
Transfer to the -80°C freezer to be stored.
Retrieved from " http://2014.igem.org/Team:Sheffield/WetLabJournal "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
