<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Sheffield_WetLabJournal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Sheffield/WetLabJournal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Sheffield/WetLabJournal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Sheffield/WetLabJournal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Sheffield/WetLabJournal&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Sheffield/WetLabJournal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Sheffield/WetLabJournal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="topMenu"><UL id="navigationBar"><LI class="menuText"><A style="cursor:default;">Attributions ▾</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/Sponsors">Sponsors</A></LI></UL><LI class="menuText"><A style="cursor:default;">Notebook ▾</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/LabProtocols">Lab Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/WetLabJournal">Wet Lab Journal</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/DryLabJournal">Dry Lab Journal</A></LI><LI class="menuText"><A style="cursor:default;">Policy and Practices ▾</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/SocioBrick">SocioBricks</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/sociobricks/main">SocioBrick Registry</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/Notions">Notions of Responsibility</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/IndustryCollaboration">Industry Collaboration</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/SynBio">Synthetic Biology @ Sheffield</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/LabNotation">Lab Notation</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/collaboration">Collaboration</A></LI><LI class="menuText"><A style="cursor:default;">Results ▾</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/FadR">FadR Characterisation</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/Lipase">Lipase Assay</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/Keratinase">Keratinase Assay</A></LI><LI class="menuText"><A style="cursor:default;">Modelling ▾</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/modelling">Product Design</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/manufacture">  ↳  Budget</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/manufacture2">  ↳  Premium</A></LI><LI class="menuText"><A href="https://2014.igem.org/Team:Sheffield/Project">Project</A></LI><LI class="menuText"><A style="cursor:default;">Team ▾</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/team1">Meet the Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1498">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:Sheffield/gallery">Gallery</A></LI><LI class="menuText"><A href="https://2014.igem.org/Team:Sheffield">Home</A></LI></DIV><H1 class="pageTitle">Wet Lab Journal</H1><DIV class="mainPage"><DIV class="leftHalf"><H1 class="subPageTitle">Wet Lab Journal</H1><H2> Week 1</H2><H3>Monday 23<SUP>rd</SUP> June</H3><P>Make up 100ml of <A href="#protocol1">LB Broth</A> and 100ml <A href="#protocol2">LB Agar</A>.</P><P>Make an <A href="#protocol3">overnight</A> starter culture of <A href="#biobrick1">BBa_K258008</A> to test sterile technique.</P><H3>Tuesday 24<SUP>th</SUP> June</H3><P>Generate <A href="#protocol4">Chemically competent</A> E.Coli.</P><OL><LI>Make up 8 flasks of 100ml starter culture.</LI><LI>4 flasks (A-D) were checked each time to take the OD600 readings.</LI><LI>
						OD600 readings to show when the cells are in exponential growth phase and so ready to be made competent.
						<TABLE><TBODY><TR><TH>A</TH><TH>B</TH><TH>C</TH><TH>D</TH></TR><TR><TD>1.25h</TD><TD>0.077</TD><TD>0.078</TD><TD>0.082</TD><TD>0.096</TD></TR><TR><TD>3.25h</TD><TD>0.567</TD><TD>0.674</TD><TD>0.714</TD><TD>0.809</TD></TR></TBODY></TABLE></LI><LI>Aliquot into 24 falcon tubes. The cells will need to be spun down in 2 rounds.</LI><LI>72 eppendorf tubes will be produced, labelled ch. comp, dh5alpha.</LI><LI>Store at -80°C.</LI></OL><H2> Week 2 </H2><H3>Monday 30<SUP>th</SUP> June</H3><OL><LI>
						Make 5mL <A href="#protocol3">overnight</A> starter cultures of the following:
						<TABLE><TBODY><TR><TH>Tube</TH><TH>BioBrick</TH><TH>Antibiotic</TH></TR><TR><TD>1</TD><TD><A href="#biobrick1">BBa_K258008</A></TD><TD>Chloramphenicol</TD></TR><TR><TD>2</TD><TD><A href="#biobrick2">BBa_K215107</A></TD><TD>Tetracycline</TD></TR><TR><TD>3</TD><TD><A href="#biobrick3">BBa_K861061</A></TD><TD>Chloramphenicol</TD></TR><TR><TD>4</TD><TD><A href="#biobrick4">BBa_K215002</A></TD><TD>Ampicillin</TD></TR><TR><TD>5</TD><TD><A href="#biobrick5">BBa_K1149021</A></TD><TD>Chloramphenicol</TD></TR><TR><TD>6</TD><TD><A href="#biobrick6">BBa_K861060</A></TD><TD>Chloramphenicol</TD></TR></TBODY></TABLE></LI><LI>5μl of an appropriate antibiotic was added to each culture to allow selection for cells containing biobrick plasmids.</LI></OL><H3>Tuesday 1<SUP>st</SUP> July</H3><P>Conduct 6 <A href="#protocol5">mini-preps</A>, one for each of the biobrick cultures, to produce 6 Eppendorfs containing purified DNA.</P><H3>Wednesday 2<SUP>nd</SUP> July</H3><P>Run a <A href="#protocol6">DNA gel</A> using the purified DNA collected yesterday to check DNA is pure. </P><OL><LI>Use an 8 well comb to produce the gel.</LI><LI>
					The table below shows the placement of each DNA sample within the Gel.

					<TABLE><TBODY><TR><TH>Well</TH><TH>Biobrick</TH><TH>Part Function</TH></TR><TR><TD>1</TD><TD>1kb gene ruler</TD></TR><TR><TD>2</TD><TD><A href="#biobrick5">BBa_K1149021</A></TD><TD>Promoter, RBS, Lipase</TD></TR><TR><TD>3</TD><TD><A href="#biobrick1">BBa_K258008</A></TD><TD>ABC transporter</TD></TR><TR><TD>4</TD><TD><A href="#biobrick6">BBa_K861060</A></TD><TD>pFadR promoter</TD></TR><TR><TD>5</TD><TD><A href="#biobrick3">BBa_K861061</A></TD><TD>pFadR Promoter + RFP</TD></TR><TR><TD>6</TD><TD><A href="#biobrick2">BBa_K215107</A></TD><TD>ABC transporter (strong)</TD></TR><TR><TD>7</TD><TD><A href="#biobrick4">BBa_K215002</A></TD><TD>pLAC +RBS +secretion signal + streptavidin binding tag</TD></TR></TBODY></TABLE></LI><LI>The gel was run at 100v for 45 minutes (due to time constrains).</LI></OL><P>Below is the image taken of the Gel on the Gel dock.</P><P>4 LB agar <A href="#protocol7">plates</A> were made, 2 ampicillin (100 μl) and 2 chloramphenicol (25 μl).</P><P><A href="#protocol8">Nanodrop</A> the DNA gained from the Mini-prep to generate concentrations of extracted DNA for all 6 biobricks.</P><P>Results are shown below:</P><TABLE><TBODY><TR><TH>Biobrick</TH><TH>Concentration (ng/μl)</TH></TR><TR><TD><A href="#biobrick5">BBa_K1149021</A></TD><TD>115.0</TD></TR><TR><TD><A href="#biobrick1">BBa_K258008</A></TD><TD>075.9</TD></TR><TR><TD><A href="#biobrick6">BBa_K861060</A></TD><TD>100.3</TD></TR><TR><TD><A href="#biobrick3">BBa_K861061</A></TD><TD>130.7</TD></TR><TR><TD><A href="#biobrick2">BBa_K215107</A></TD><TD>043.3</TD></TR><TR><TD><A href="#biobrick4">BBa_K215002</A></TD><TD>125.0</TD></TR></TBODY></TABLE><P><A href="#protocol9">Transform</A> cells to confirm competency</P><UL><LI>For this transformation We used the BioBricks BBa_K258008 and BBa_K215002 to check if the cells were competent.</LI><LI>
					Calculations are shown below:

					<P>
						BBa_K258008 
						75.9ng x X = 10ng x 100μl
						X = 13.2μl
					</P><P>
						BBa_K215002
						125.0ng x X = 10ng x 100μl
						X = 8.0μl
					</P></LI></UL><H3>Friday 4<SUP>th</SUP> July</H3><P>The linearised TliA and KerUS synthesised DNA is cloned into a pJet1.2 plasmid using <A href="#protocol10">blunt end ligation</A>.</P><P>This ligation mix was then used to <A href="#protocol9">transform</A> 3 eppendorfs of chemically competant cells (made on tuesday 25th):</P><UL><LI>Into eppendorf 1, pipette 5μl of the ligation mix containing TliA DNA.</LI><LI>Into eppendorf 2, pipette 5μl of the ligation mix containing KerUs DNA.</LI><LI>Into eppendorf 3, pipette 5μl of dH<SUB>2</SUB>O</LI></UL><UL><LI>Incubated for 45 minutes due to time constraints.</LI><LI>Constructs plated on ampicillin LB agar plates to select for the correct colonies. </LI><LI>Plates were checked on saturday AM and positive colonies were observed</LI></UL><H2> Week 3 </H2><H3>Monday 7<SUP>th</SUP> July</H3><P>Make 200ml <A href="#protocol2">LB Agar</A>and 200ml <A href="#protocol1">LB Broth</A>.


</P><P>Make <A href="#protocol3">overnight</A> cultures of positive colonies of pJet 1.2 with <A href="#biobrickL">TliA</A> and <A href="#biobrickK">KerUS</A>.
</P><P>Make <A href="#protocol3">overnight</A> cultures of <A href="#cell1">E. coli MC1000</A>.

</P><H3>Tuesday 8<SUP>th</SUP> July</H3><A href="#protocol5">Miniprep</A> the <A href="#biobrickL">Tlia</A> and <A href="#biobrickK">KerUS</A> cultures to produce purified plasmid DNA.

<P><A href="#protocol8">Nanodrop</A> the purified DNA to determine the concentration of DNA.
<OL><LI> 
TliA - 5.2ng/μl
</LI><LI>KerUS - 3.4ng/μl </LI></OL></P><P>Produce <A href="#protocol14">glycerol stocks</A> of the Tlia and KerUS cells for storage (retain only if insert confirmed).

</P><P>Make <A href="#protocol4">chemically competent</A> MC1000. 
</P><P>Measure the OD600 of the MC1000 cells using spectrophotometer
<OL><LI>Flask 1 - 0.803
</LI><LI>Flask 2 - 0.624
</LI><LI>Blank - 0.066</LI></OL><LI>Decant the flasks of MC1000 into 50ml falcon tubes. We have 4 tubes of full MC1000, the 5th one is 21ml MC1000 and 9ml water, the 6th one is all water -to ensure the centrifuge is balanced (weight checked too).</LI></P><P>Make 10ml <A href="#protocol3">overnight</A> cultures of pJet 1.2 containing <A href="#biobrickL">TliA</A> or <A href="#biobrickK">KerUS</A></P><H3>Wednesday 9<SUP>th</SUP> July</H3><P>Repeat yesterdays <A href="#protocol5">miniprep</A> as it didn't yield enough DNA. 
<OL><LI>Began with two 10ml starter cultures, one of pJet 1.2 TliA and one of pJet 1.2 KerUS. 
</LI><LI>Decanted by hand into falcon tubes to give four 5ml cultures.</LI></OL></P><P>A <A href="#protocol5">Restriction Digest</A> was carried out on the Tlia and KerUS minipreps, as well as the FadR vector that was miniprepped last week. 
</P><P>TliA and KerUS were cut with xbaI and pstI, whilst FadR was cut with speI and pstI. 

</P><H3>Thursday 10<SUP>th</SUP> July</H3><A href="#protocol6">Run</A> Vector, TliA and KerUS DNA on a gel to ensure it has been cut correctly. Once confirmed the DNA was then <A href="#protocol18">extracted</A> from the gel.

<LI>Add 12.5μl 5X loading buffer to the DNA samples (50μl each) - total of 62.5μl in each eppendorf tube.
</LI><LI>Load each sample, using two wells each if necessary.
</LI><LI>Run at 100V for 60 minutes.

Visualise the gel. 
</LI><LI>The bands for TliA and Vector DNA were the correct size.
</LI><LI>The band for KerUS showed that the DNA had been cut incorrectly.

The two samples of DNA that have been cut correctly are now extracted from the gel. 
The gel extraction was then nanodropped to determine DNA concentration.
<H4><P class="tab">


Vector: 14.3ng/μL
Tlia: 190.9ng/μL</P></H4>

A <A href="#protocol12">ligation reaction</A> was carried out using the following amounts of reactants;

<H4><P class="tab">3.5 μl vector
0.2 μl lipase
2 μl buffer
1 μl ligase
13.3 μl dH2O
leave for 10 minutes at room temperature</P></H4>

Add 5μl ligation reaction mix to 200μl DH5α and <A href="#protocol9">transform</A>. 

NOTE: did not work -no positve colonies following antibiotic resistance selection on an LB agar plate
We also did some work to help us characterise the pFadR promoter.
<H4><P class="tab">
Due to a mistake over biobrick numbers, our overnight contained pFadR without RFP, and so we used the day to test our experimental procedures.</P></H4><A href="#protocol15">Day cultures</A> were made.
<H4><P class="tab">1 x 5ml of empty DH5α3 x 5ml of DH5α containing <A href="#biobrick6">BBa_K861060</A>Day cultures were placed in the incubator at 37°C, 170rpm for 3 hours</P></H4><A href="#protocol20">Lipid stocks</A> were made using Triton to solubilise. 

12 samples were made to be assayed;
	<H4><P class="tab">1: 5ml DH5α + 0% lipid
	2: 5ml DH5α + 1.0% olive oil
	3: 5ml DH5α + 1.0% oleic acid
	4: 5ml DH5α + BBa_K861060 + 0% lipid
	5: 5ml DH5α + BBa_K861060 + 0.1% olive oil
	6: 5ml DH5α + BBa_K861060 + 1.0% olive oil
	7: 5ml DH5α + BBa_K861060 + 0.1% oleic acid
	8: 5ml DH5α + BBa_K861060 + 1.0% oleic acid
	9: 5ml LB + 1% olive oil
	10: 5ml LB + 1% olive oil + chloramphenicol
	11: 5ml LB + 1% oleic acid
	12: 5ml LB + 1% oleic acid + chloramphenicol</P></H4>
	
Samples were incubated at 37°c

OD600 Readings were taken at hourly time points
<H4><P class="tab">100μl of each sample was pipetted into wells in a Transparent Greiner 96 microtitre plate.
As well as the 12 samples, 5 Control Wells (blanks) were created, without cells:
A: LB only
B: 0.1% Oleic Acid only 
C: 1.0% Oleic Acid only 
D: 0.1% Olive Oil only 
E: 1.0% Olive Oil only </P></H4>

How to use the Plate Reader
<H4><P class="tab">Turn on computer
Select ...
Select open plate icon
Place microtitre plate into cassette holder
Select close plate icon
Open lid of plate reader
Twist wheels - for OD select white and red wires, for fluorescence select yellow and blue wires
Select spanner icon on program to choose between OD and fluorescence
Select Measure icon 
Edit ….
Select Excel icon
Open A600 for OD, RFP Charlotte for Fluoresence
Select update icon
Scroll down to Table 2 to view results </P></H4><H3>Friday 11<SUP>th</SUP> July</H3><U>FadR assay analysis</U><H2> Week 4 </H2><H3>Monday 14<SUP>th</SUP> July </H3>
  
  Make overnight cultures.
  </LI><LI><A href="#biobrick1">DH5a + pSB3k3</A></LI><LI><A href="#biobrick1">DH5a + BBa_K861061</A></LI><LI><A href="#biobrick1">DH5a + BBa_K861060</A> x2
	</LI><LI><A href="#biobrick1">DH5a</A></LI><LI><A href="#biobrick1">DH5a + pJeT 1.2 TliA</A></LI><LI><A href="#biobrick1">DH5a + pJeT 1.2 KerUS</A><H3>Tuesday 15<SUP>th</SUP> July </H3>
 
Overnights of pJet 1.2 KerUS and TliA containing cells didn't grow, and so the time was used to make DNA stocks of BBa_K861060 and BBa_K861061.

Overnights were <A href="#protocol5">Miniprepped</A> and then <A href="#protocol8">Nanodropped</A>.
dH<SUB>2</SUB>O was used instead of the elution buffer in the miniprep kit.
<H4><P class="tab">FadR - 80.6ng/μl
FadR - 75.9ng/μl
RFP - 50.3ng/μl </P></H4>

Make <A href="#protocol3">Overnights</A> of pJet 1.2 + TliA and KerUS.

<H3>Wednesday 16<SUP>th</SUP> July </H3><A href="#protocol5">Miniprep</A> Tlia and KerUS cells.


<A href="#protocol8">Nanodrop</A><H4><P class="tab">
TliA - 522.0ng/μl
Kerus - 264.9ng/μl</P></H4><A href="#protocol13">Restriction Digests</A> were made with TliA, KerUS and FadR containing plasmids. 
The following amounts of DNA were cut with restriction enzymes and the final reaction mixture bought up to 50μl.
<H4><P class="tab">
2μl Tlia
4μl KerUS
12.5μl FadR
</P></H4>
Run a <A href="#protocol6">Gel</A> to seperate out the restriction fragments. 
Do a <A href="#protocol18">Gel extraction</A> on the bands which are the correct size. The DNA was eluted into dH<SUB>2</SUB>O.
<A href="#protocol13">Nanodrop</A><H4><P class="tab"> TliA - 5.7ng/μl
KerUS - 7.0ng/μl
FadR - 5.7ng/μl</P></H4>

A <A href="#protocol6">Ligation</A> reaction was set up containing either cut TliA or KerUS and FadR vector.
 After an hour, the reaction mix was added to chemically competent DH5α which were <A href="#protocol6">transformed</A> and plated on antibiotic containing LB agar plates.
Plates were incubated overnight at 37°C


  <H3>Thursday 17<SUP>th</SUP> July </H3>
 No positve colonies from the TliA and KerUS with FadR vector transformation
<A href="#protocol13">Day cultures</A> of BBa_861060, BBa_861061 and pSB3k3 were made. 
Once grown (leave for around 5 hours), 10μl of these cells were then plated onto either 25μl 1% Oleic Acid, 25μl 1% Olive Oil or 25μl LB agar plates. 
 Cells were left in the incubator overnight.
  <H3>Thursday 17<SUP>th</SUP> July </H3>
  No zone of clearing observed on olive oil containing plates
<H2>Week 5</H2><B> Tecan data </B>
A second run was done on the Tecan. Three different cells were used;
<H4><P class="tab"> BBa_k861060
BBa_k861061
pSB3k3</P></H4>

Each of these cells were added to solutions containing 0.5% Olive oil, 1% Olive oil and 2% Olive oil. Solutions without cells added were also used as a control.
 200μl of each solution was pipetted into a 96 well plate with a clear bottom and white wells. 
All unused wells were filled with LB to prevent evaporation of samples in the central wells.
<B> Fluorescence Microscopy </B>Set up <A href="#protocol3">overnights</A> of the following;
<H4><P class="tab">
PSB3K3
pSB3K3 + 1% Olive Oil
pSB3K3 + 1% Oleic Acid
BBa_K861060
BBa_K861060 + 1% Olive Oil
BBa_K861060 + 1% Oleic Acid</P></H4> Add 10μl overnight cultures to microscope slide cover slips, allow to dry.
 add 10μl 4% paraformaldehyde, dry for 30 minutes. 
Wash 3 x with PBS buffer. Apply 10μl Prolong Gold Antifade Mountant with DAPI (Life Technologies). Fix to microscope slide, leave to dry overnight.
 Use fluorescence Microscope to visualise cells with DAPI and RFP suitable channels. Maintain the same exposure time between images so they are comparable.


Following fresh ligation and transformation positive colonies were achieved for lipase with fadR responsive promoter in vector. Folllowing <A href="#protocol5">restriction digestion</A> with xbaI and pst1. Negative result.
<A href="#protocol5">Overnights</A> of BBa_k861060, BBa_k861061 x2, TliA x2, PSB3K3 pellets frozen for minipreps later

<H2>Week 6</H2><H3>Monday 28<SUP>th</SUP> July</H3><A href="#protocol3">Miniprep</A> of overnights from Friday (BBa_k861060, BBa_k861061 x2, TliA x2, PSB3K3)
<H4><P class="tab">Plasmid DNA stored in -20</P></H4>
Produce 6 flasks of <A href="#protocol3">LB agar</A>.
<A href="#protocol3">Overnights</A> of the following;
<H4><P class="tab">KerUS
TliA
<A href="#biobrick7">BBa_k823017</A>PSB3k3</P></H4><H3>Tuesday 29<SUP>th</SUP> July</H3>

Make up new stocks of <A href="#protocol20">soluble Olive oil (in Triton)</A>.
Make up 8 of agar plates with tributyrin in order to test lipase activity:
<H4><P class="tab">Differing concentrations of Tributyrin was added to either full strength or dilute LB agar. Any Olive oil was added to the empty plate and swirled to mix.


0.5% tributyrin
0.5% tributyrin + 2% Olive Oil
1% tributyrin
1% tributyrin + 2% Olive Oil</P></H4><B>Make M9 Media</B>Too 800ml distilled H<SUB>2</SUB>O add;
<H4><P class="tab"> 44g of Na2.2H2O
	       					 15g of KH2.H2PO4
	     					2.5g of NaCl
	         				               5g of NH4Cl</P></H4>
Pour 800ml of distilled H<SUB>2</SUB>O into 1L measuring cylinder
Disolve with magnetic flea and stirrer
Transfer media into 2 durans and autoclave.

To 10mL water add either; <H4><P class="tab">1.2g of MgSO4
	       					   1.1g of CaCl2</P></H4>Shake/invert until salts dissolve
Transfer into conical flasks, cover with cotton wool and foil, autoclave.

Make <A href="#protocol7">Glycerol stocks</A> of pJet 1.2 KerUS, pJet 1.2 TliA, PSB3K3, BBa_K823017.
Made 6 <A href="#protocol7">LB Agar</A> stocks.
<A href="#protocol7">Nanodrop</A> of pJet 1.2 TliA and pJet 1.2 Lipase, BBa_k861060

<A href="#protocol5">Restriction</A> and <A href="#protocol5">digest</A> of pJet 1.2 TliA and pJet 1.2 Lipase with BBa_k861060. <A href="#protocol5">Transform</A> into DH5a.


<H3>Wednesday 30<SUP>th</SUP> July </H3> One positive colony for TliA in the FadR vector in DH5a -ovenight culture of this

<H3>Thursday 31<SUP>st</SUP> July </H3><A href="#protocol5">Miniprep</A> the potential TliA in the FadR vector.

<A href="#protocol8">Nanodrop</A> 54ng/μl

<A href="#protocol8">Restriction digest</A> of potential TliA in the FadR vector with NdeI and EcoR1 to confirm - failed

Make <A href="#protocol8">glycerol stocks</A> of overnights potential TliA in the FadR vector

<H3>Friday 1<SUP>st</SUP> August </H3><A href="#protocol8">Restriction digest</A> undertaken.
TliA and a construct thought to contain the gene for TliA in the FadR vector were <A href="#protocol5">restricted</A> using Xba1, Nde1 and both.

<A href="#protocol8">A gel</A> of the restriction digest was made

The gel showed that the fadR plasmid did not contain the lipase.



<H2>Week 7</H2><B>ABC transporters into cells </B><A href="#protocol3">Overnights</A> of both ABC Transporters (BBa_K258008, BBa_K215107), T1SS signal-strep binding tag (BBa_K215002).
<A href="#protocol5">Miniprep</A> the above plasmids and <A href="#protocol5">transform</A> into stocks of <A href="#protocol5">chemically competant</A> DH5a.
 Grow overnights of positive colonies, make competant and store at -80 in preperation for the transformation of a lipase containing plasmid.





<B>Cloning</B><A href="#protocol5">Miniprep</A> the pJet 1.2 TliA and promoters BBa_J23101, BBa_J23110 and BBa_J23100. 
Elute into 50&amp;#956 of dH<SUB>2</SUB>0.

<A href="#protocol5">Nanodrop</A><H4><P class="tab">TliA - 103ng/μl
BBa_J23101 - 278ng/μl
BBa_J23110 - 408ng/μl
BBa_J23100 - 380ng/μl</P></H4>Carry out a <A href="#protocol5">Restriction digest</A> on the TliA and all three promoter plasmids - already had digested kerUS.
 
Slight variation to usual protocol: After 2 hours, 3μl CIP was added to the promoter plasmids and then returned to the heat block for another hour. 
After 2 hours, the lipase was removed from the heat block.

<A href="#protocol5">Make a gel</A>Restriction digestion mixes were all run on a DNA agarose gel and the correct sized bands cut.

DNA was <A href="#protocol5">extracted</A> from the gel. 
The TliA and kerUS inserts were <A href="#protocol5">ligated</A> into each of the three different promoter strength harbouring vectors. 
The ligation was left in the fridge overnight.
Ligation mixes were <A href="#protocol5">transformed</A> into NEB5a cells
Positive colonies were seen for TliA and kerUS in the BBa_J23110 containing vectors
Colony PCR confirmed that TliA and kerUS had successfully been cloned into the BBa_J23110 vector
These colonies were grown in liquid culture and <A href="#protocol5">-80°C stocks</A> were created

<B>Tecan run with minimal media</B>DH5a plus the following plasmids were used for the tecan runs
	<P class="tab">FadR-Rfp
	FadR
	J23110</P>2 x 50ml falcon tubes of M9 media was made
<P class="tab">35ml of dH<SUB>2</SUB>O was added from duran 
10ml of M9 salts was added from duran
 2ml of casamino acids was added from falcon tube 
 100µl of MgSO<SUB>4</SUB> was added from 50ml conical flask
 1ml of 20% glucose solution was added to one falcon tube, 1ml of dH<SUB>2</SUB>O was added to the 
    other
 5µl of CaCl<SUB>2</SUB> was added from 50ml conical flask
 topped up with dH<SUB>2</SUB>O to make 50ml solution</P>

Lipid was added to the M9 Solution
Total volume created = 4ml, in 6ml bijou tubes
Since the Olive oil:Triton X-100 ratio was 1:2, the following concentrations were made:
<TABLE style="width:50%; color:#696969"><TBODY><TR><TH>Solution</TH><TH>1%</TH><TH>2%</TH><TH>3%</TH><TH>4%</TH></TR><TR><TD>M9 Media</TD><TD>4.0ml</TD><TD>3.88ml</TD><TD>3.76ml</TD><TD>3.52ml</TD></TR><TR><TD>Olive oil:Triton</TD><TD>0μl</TD><TD>120μl</TD><TD>240μl</TD><TD>480μl</TD></TR></TBODY></TABLE>

  The 4.0% stock was made from the falcon tube without glucose added as a supplementary carbon source.
   the 0.0%, 1.0% and 2.0% were made from the falcon tube with glucose added as a supplementary carbon source

Transformed cells &amp; antibiotics were added to all 4 concentrations of M9/Lipid media.
96-Well plate was made.
200μl of each solution containing the 3 different promoters were pipetted into 4 wells to create 4 technical repeats.
200μl of each solution devoid of cells were pipetted into 3 wells in the region of the plate (B-G, 10-11)
<H2>Week 9</H2><H3>Monday 18<SUP>th</SUP> August</H3>

Make <A href="#protocol3">overnights</A> of <A href="#biobrick1">kerUS</A> and <A href="#biobrick1">BBa_J23110</A><H3>Tuesday 19<SUP>th</SUP> August </H3>

Make 4 <A href="#protocol15">day cultures</A>;
<H4><P class="tab"> 5ml KerUS + Nail
 5ml KerUS + Hair
 5ml BBa_J23110 + Nail
 5ml BBa_J23110 + Hair</P></H4>
These were left in the shaking incubator for the cells to grow and produce and secrete keratinase. 



Autoclave hair, nail and feather samples to sterilise. This was done in glass beakers with foil 'lids'.

<A href="#protocol23">Restreak</A> KerUS and BBa_J23110.

Make <A href="#protocol3">overnights</A> of kerUS and BBa_J23110.

<H3>Wednesday 20<SUP>th</SUP> August</H3><A href="#protocol3">Overnights</A> of <A href="#biobrick1">BL21</A>.


<H3>Thursday 21<SUP>st</SUP> August</H3>
Make 2 5ml <A href="#protocol15">day cultures</A> of BL21.
 Allow around 3 hours for them to reach the exponential growth phase. 
<A href="#protocol16">Take an OD reading </A>If the OD600 is over 0.6, use these innoculations to produce <A href="#protocol4">Chemically competant</A> cells.

<A href="#protocol5">Miniprep</A> KerUS BBa_J23110
<A href="#protocol11">Transform</A> the chemically competant BL21 strain using 250ng of the purified KerUS BBa_J23110 DNA.
Plate the following;
<H4><P class="tab"> 100μl of control onto an antibiotic free plate.
100μl of control onto an ampicillin plate.
100μl of transformant onto an ampicillin plate.
Remaining 900μl spun down and resuspended in 100μl of LB and plated onto an ampicillin plate.

</P></H4><A href="#protocol5">Transform</A> TliA BBa_J23110 into stocks of ABC transporter (BBa_K258008) containing competent cells
<A href="#protocol11">Pour plates</A> containing Lb Agar, Lb Agar containing 1% casein, Lb Agar containing 1% gelatin and Agar containing 1% gelatin.

<H3>Friday 22<SUP>nd</SUP> August</H3>Positive colonies of all transformations
<A href="#protocol3">Overnights</A> of positive colonies of cells containing <A href="#biobrick1">TliA/BBa_J23110/BBa_K258008</A>, KerUs/BBa_J23110 and BBa_J23110
 
<H3>Saturday 23<SUP>nd</SUP> August</H3>Plate out KerUs containing overnights onto Casein, Gelatin and Gelatin LB agar plates. BBa_J23110 was used as a negative control. <I>Staphylococcus aureus</I> was used as a positive control.
 Incubate for 8 hours, then flood plates in saturated ammonium sulphate. 
<H2>Week 10</H2><H3>Monday 25<SUP>th</SUP> August</H3>
Make <A href="#protocol3">overnights</A> of KerUS/BBa_J23110, tliA/BBa_J23110, tliA/BBa_J23110/BBa_K258008 and BBa_J23110.
Autoclave four 1 Litre conical flasks each containing 100ml of LB Broth.

<H3>Tuesday 26<SUP>th</SUP> August</H3>
Innoculate each of the autoclaved flasks with 1ml of each of the overnights. Add the correct antibiotic to each. 
Incubate at 30°C to promote protein expression.
Give 5 or 6 hours for the cells to reach exponential growth phase and produce an adequate amount of protein to assay.

After 6 hours, samples were taken for analysis by protein gel. 
<H4><P class="tab"> Into 4 eppendorfs, pipette 1.5 ml of each innoculation. 
These are spun down and the supernatant removed into a second tube. Following the addition of 150μl trichloroacetic acid (TCA) and kept in the 4 degree overnight.
 Into 4 more eppendorfs, pipette 0.5ml of each innoculation.
These are spun down, the supernatant poured away, the pellet labelled and kept in the -20 degree.
Remaining culture was centrifuged, sterile filtered and loaded into Vivaspin 20 10,000 MWCO columns (GE Life Science) (tliA without exporter was excluded) add 5ml of culture supernatant. These were spun down at 3,500 rpm for 8 minutes. 
Supernatant that had passed through the column was discarded, and more supernatant was added into the top of the column. This was repeated until around 15ml of supernatant had passed through the column.
Around 1ml of concentrated supernatant was then removed from the top of the column.

<A href="#protocol5">Overnight</A> liquid cultures of cells containing kerUS (no promoter), tliA/BBa_J23110 and pSB1C3  

</P></H4><H3>Wednesday 27<SUP>th</SUP> August</H3>
To prepare the protein sample to be loaded onto a gel
<H4><P class="tab">To the cell pellet add 100μl 2X SDS buffer. Resuspend the cells in this solution. 
Centrifuge the 1.5ml of supernatant/TCA mix for 15 minutes at 13,300rpm, remove supernatant, resuspend pellet in 900μl acetone, repeat centrifugation step, remove supernatant, allow remaining acetone to evaporate then add 50μl of SDS buffer (a ratio of ~1:20)
To the 1ml concentrated supernatant, add 1ml of SDS buffer (a ratio of 1:1)
Place all of these samples into a heat block at 95°C for 5 minutes. The cell pellet may need to be vortexed and left for an additional 5 minutes.</P></H4><A href="#protocol5">Make a 12% SDS-PAGE gel</A>. Load 5μl cell lysate samples and 15μl supernatant (concentrated or non-concentrated) onto the gel
Stain the SDS-PAGE gel with InstantBlue (Expedeon) overnight
<A href="#protocol5">Miniprep</A> plasmids from overnight cultures, set up <A href="#protocol5">restriction digests</A> with EcoRI and PstI (add CIP to the acceptor vector). Run donor mixes on a DNA gel and isolate DNA from bands of correct size
<A href="#protocol5">Ligate</A> isolated cut kerUS and tliA/J23110 with cut pSB1C3 overnight

<H3>Thursday 28<SUP>th</SUP> August</H3><A href="#protocol5">Transform</A> both ligation mixes into NEB5a and <A href="#protocol5">plate</A> on chloramphenicol containing LB agar plates
<H3>Friday 29<SUP>th</SUP> August </H3>Positive colonies on both LB agar plates. <A href="#protocol5">Colony PCR</A> to confirm insertion of kerUS or tliA/J23110.
Overnight liquid cultures of a positive colony from each.


<H3>Saturday 30<SUP>th</SUP> August</H3><A href="#protocol5">Miniprep</A> overnight cultures. Send some for sequencing with sequencing primers. Once confirmed send on to the BioBricks Registry 	</LI></DIV></DIV></DIV></DIV><DIV class="rightHalf"><H1 class="subPageTitle">Lab Protocols</H1><DIV class="mainPage"><DIV id="strains" class="pageSection1"><DIV class="sectionHeading"><H2>Strains</H2></DIV><DIV class="sectionContent"><P><I>Escherichia coli</I> DH5α - <B>NEB</B><I>Escherichia coli MC1000</I> - <B>Donated by Dr. Graham Stafford</B></P></DIV></DIV><DIV id="antibiotics" class="pageSection2"><DIV class="sectionHeading"><H2>Antibiotics</H2></DIV><DIV class="sectionContent"><TABLE><TBODY><TR><TH>Antibiotic</TH><TH>Concentration(μg/ml)</TH></TR><TR><TD>Ampicillin</TD><TD>100</TD></TR><TR><TD>Chloramphenicol</TD><TD>25</TD></TR><TR><TD>Tetracycline</TD><TD>20</TD></TR></TBODY></TABLE></DIV></DIV><DIV id="strains" class="pageSection1"><DIV class="sectionHeading"><H2>Primers</H2></DIV><DIV class="sectionContent"><TABLE><TBODY><TR><TH>Primer Name</TH><TH>Nucleotide Sequence</TH></TR><TR><TD>pSB1C3 seq f</TD><TD>caactcgagtgccacctg</TD></TR><TR><TD>PSB1C3 seq r</TD><TD>gcctttttacggttcctg</TD></TR><TR><TD>TliA seq f</TD><TD>gcagatctgagcaccagc</TD></TR><TR><TD>TliA seq r</TD><TD>gtccagagcacgaaagac</TD></TR><TR><TD>KERUS seq f</TD><TD>ggtgatggtcaggatagc</TD></TR><TR><TD>KERUS seq r</TD><TD>ctgccacaacaacaacac</TD></TR></TBODY></TABLE><P>GGTGATGGTCAGGATAGC</P><P>CTGCCACAACAACAACAC</P><P>pSB1C3 seq f: caactcgagtgccacctg – 18-mer, 3412...3429, 56degC</P><P>pSB1C3 seq r: gcctttttacggttcctg, 18-mer, 1731...1748, 52degC</P><P>TliA seq primer f: gcagatctgagcaccagc 18-mer, 711...728, 56degC</P><P>TliA seq r: gtccagagcacgaaagac – 18-mer, 831...848, 54degC</P></DIV></DIV><DIV id="mediaAndStockSols" class="pageSection2"><DIV class="sectionHeading"><H2>Media and Stock Solutions</H2></DIV><DIV class="sectionContent"><H3>Luria-Bertani (LB) Broth</H3><P>
						LB Broth was made using a ready mix powder from Sigma, 20g/L in ultrapure water. This consists of: 10 g/L Tryptone; 5 g/L Yeast Extract; 5 g/L NaCl. Autoclaved at 121 oC for 15 min to sterilise.
					</P><H3>Luria-Bertani (LB) Agar</H3><P>
						LB agar was made using a ready mix powder from Sigma, 35 g/L ultrapure water. This consists of 15 g/L Agar and as in LB Broth. Autoclaved at 121 oC for 15 min to sterilise.
					</P><H3>5 x M9</H3><P>
						5 x M9 was made using the following up to 1 L of ultrapure water: 64 g Na2HPO4-4H2O; 15 g KH2PO4; 2.5 g NaCl. 
					</P><P>
						Working stock, 1 X M9 was made by adding into up to 1 L of ultrapure water: 200 ml of 5 x M9; 2 ml 1M MgSO4; 20 ml 20% glucose/0.4% Glycerol; 100 μl 1M CaCl2
					</P><H3>50 x TAE</H3><P>
						50 x TAE was made up with: 121 g Tris Base (Trizma); 28.55 gl acetic acid; 9.3 g EDTA made up in 500 ml ultrapure water.
					</P><P>
						Working stock, 1 x TAE was made by diluting 10 ml in 500 ml ultrapure water.
					</P><H3>10 x TBS</H3><P>
						10 x TBS was made up with: 500 ml 1M Tris.HCl, pH 7.4; 300 mM NaCl made up to 1L with ultrapure water.
					</P><P>
						Working stock, 1 x TBS by diluting 100 ml 10 x TBS up to 1000 ml with ultrapure water.
					</P><H3>1 x PBS</H3><P>
						1 x PBS was made up using Phosphate buffered saline tablets from Sigma.
					</P><P>
						One tablet dissolved in 200 mL of deionized water yields 0.01 M phosphate buffer, 0.0027 M potassium chloride and 0.137 M sodium chloride, pH 7.4, at 25 °C.
					</P><H3>Upper Tris Buffer</H3><P>
						Upper Tris Buffer was made up by adding 6.05 g Tris Base in 100 ml deionised water, pH 6.8. Add 0.4 g SDS.
					</P><H3>Lower Tris Buffer</H3><P>
						Lower Tris Buffer was made up by adding 36.4 g Tris in 100 ml deionised water, pH8.8. Add 0.8 g SDS.
					</P><H3>10 x SDS Running Buffer</H3><P>
						10 x SDS running buffer was made up by adding: 30.3 g Tris Base; 144.1 g Glycine; 10 g SDS to 1 L deionised water to pH 8.3. 
					</P><P>
						Working concentration, 1 x SDS Running Buffer is made by adding 100 ml of 10 x SDS Running buffer to 10 ml deionised water.
					</P><H3>Solubilising lipids</H3><P>
						A 10% stock solution of triton-x100 was made by adding 1ml of Triton to 9ml sterile (autoclaved) water and incubated at 37oC, 150 rpm until solubilised.
					</P><P>
						Filter sterilise and add 10% Triton to Olive oil/Oleic acid in a 2:1 ration. Incubate at 37oC, 150 rpm to solubilise.
					</P></DIV></DIV><DIV id="protocol3" class="pageSection1"><DIV class="sectionHeading"><H2>Protocol 1: Make overnight starter cultures</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						Stripette, bunsen burner, sterile loop, falcon tubes, LB broth.
					<H3>Time</H3>
						Prep: 10 minutes
						Run: 16 hours
					<H3>Procedure</H3><OL><LI>Use a stripette to take 5ml of LB broth from a 250ml conical flask.</LI><LI>Dispense into a falcon tube.</LI><LI>Sterilise a metal loop in a flame.</LI><LI>Take a culture from an agar plate using the sterile loop and put into one of the falcon tubes.</LI><LI>Agitate.</LI><LI>Replicate this using scrapings from a clean agar plate and a fresh tube to use as a positive control.</LI><LI>Put the tubes into the incubator overnight (37c, 150rpm) to grow up the cultures.</LI></OL></DIV></DIV><DIV id="protocol4" class="pageSection2"><DIV class="sectionHeading"><H2>Protocol 2: Generate chemically competent E. Coli</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						LB broth, starter culture, P1000, P200, pipette tips, incubator, cuvettes, spectrophotometer, Virkon, falcon tubes, ice, weighing scales, centrifuge, CaCl2, 20% glycerol, stripette, eppendorf tubes and -80°C freezer.
					<H3>Time</H3>
						Prep: 40 minutes
						Run: 5 hours
					<H3>Procedure</H3><OL><LI>Grow cells
							</LI><LI>Take 1ml starter culture and add to 100ml LB broth.</LI><LI>Incubate at 37°C, 150rpm.</LI><LI>Check every hour by testing the optical density at 600nm (OD) using a spectrophotometer to determine whether enough cells are present in the culture.</LI><LI>0.600 OD is ideal, this is the point at which the cells are in the exponential growth phase.</LI><LI>Take 1ml of culture into a cuvette to measure; dispose of this after use in Virkon.</LI></OL><LI>Remove cells
							</LI><LI>Pour 30ml aliquots from the flasks into falcon tubes; 3 tubes per flask.</LI><LI>Put tubes on ice for approx 10mins; all equipment used from this point on must be cold e.g. pipette tips.</LI><LI>Weigh the falcon tubes and pair together similar weight tubes for balance in the centrifuge; tubes paired together must weigh within 0.5g of each other.</LI><LI>Spin at 4°C, 4000rpm for 5mins.</LI><LI>After the tubes have all been spun, pour off the supernatant to remove the LB broth, leaving cells in a pellet. Put tubes back on the ice.</LI><LI>Make cells competent
							</LI><LI>Add 1ml of CaCl2 to the cells, use the pipette to pull the liquid and cells up and down to resuspend.</LI><LI>Once resuspended, add another 14ml of CaCl2; 15ml total volume.</LI><LI>Put back on ice for approx. 10mins to allow cells to acclimatise at the temperature with the CaCl2.</LI><LI>Re-weigh and pairs tubes again for balance.</LI><LI>Spin again at 4°C, 4000rpm for 5mins.</LI><LI>After tubes have been spun, leave on ice for 5mins. Pour off the supernatant?</LI><LI>Aliquot
							</LI><LI>Add 600μl of 20% glycerol to each falcon tube.</LI><LI>Label eppendorf tubes.</LI><LI>Aliquot 200μl from each falcon tube into eppendorf tubes (3 per falcon).</LI><LI>Freeze at -80°C.</LI></DIV></DIV><DIV id="protocol5" class="pageSection1"><DIV class="sectionHeading"><H2>Protocol 3: Mini-Prep</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						Ice, starter cultures, mini-prep kit, centrifuge, P100 pipette, weighing scale, Virkon.
					<H3>Time</H3>
						Prep: ?
						Run: ?
					<H3>Procedure</H3><OL><LI>Extract cells
							</LI><LI>Match up weights of falcon tubes so they are paired within 0.5g.</LI><LI>Spin down starter cultures in a centrifuge; 5mins, 4°C, 4000rpm.</LI><LI>Pour off supernatant into Virkon.</LI></OL><LI>Resuspend cells
							</LI><LI>Add 250μl of P1 resuspension buffer to each falcon tube using P1000.</LI><LI>Resuspend the cells.</LI><LI>Use pipette to move suspension into separate, labelled eppendorf tubes.</LI><LI>Lyse cells
							</LI><LI>Add 250μl of P2 buffer to each eppendorf tube.</LI><LI>This will lyse cells - a blue colour will indicate they have been lysed.</LI><LI>Do not leave for more than 5mins.</LI><LI>Neutralise cells
							</LI><LI>Add 350μl of N3 neutralisation buffer.</LI><LI>Once the reaction is complete, the liquid will turn clear/white.</LI><LI>Purify DNA
							</LI><LI>Spin down the cells in a centrifuge; 10 mins, 17000g, 4°C.</LI><LI>Pour supernatant into mini-prep columns.</LI><LI>Centrifuge columns for 1 min, 17000g, 4°C.</LI><LI>Discard flow through.</LI><LI>Add 500μl of PB buffer to each column.</LI><LI>Centrifuge columns for 1min, 17000g, 4°C.</LI><LI>Discard flow through.</LI><LI>Add 750μl PE buffer to each column.</LI><LI>Centrifuge columns for 1min, 17000g, 4°C.</LI><LI>Pour off supernatent.</LI><LI>Centrifuge columns for 1min, 17000g, 4°C.</LI><LI>Discard bottom of the mini-prep.</LI><LI>Move column to a new labelled eppendorf.</LI><LI>Add 50μl of elution buffer.</LI><LI>Centrifuge columns for 1min, 17000g, 4°C.</LI><LI>Discard column.</LI><LI>Immediately place on ice; store in the B56 sewer sample freeze.</LI></DIV></DIV><DIV id="protocol6" class="pageSection2"><DIV class="sectionHeading"><H2>Protocol 4: Run a gel</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						Agarose mix, TAE buffer, Flask, Microwave, Ethidium bromide, P10, Tips, Autoclave tape, Gel tray, Comb, Buffer tray, Loading buffer, Dna, Eppendorf, Transilluminator.
					<H3>Time</H3>
						Prep: 30 mintues
						Run: 1 hour
					<H3>Procedure</H3><OL><LI>Make up a 1% agarose gel
							</LI><LI>Weigh out 0.4g agarose.</LI><LI>Measure 40ml TAE buffer.</LI><LI>Add both into a 250ml conical flask.</LI><LI>Swirl.</LI><LI>Microwave on full for 2 minute, swirling at intervals to ensure all the agarose has dissolved.</LI><LI>Run side of flask under tap to cool, until comfortable to hold in a gloved hand.</LI></OL><LI>Prepare the gel
							</LI><LI>Add 1μl ethidium bromide to the conical flask containing the melted agarose.</LI><LI>Pour this solution smoothly into a gel tray (sealed with autoclave tape) and add a comb which allows for either 8 or 13 DNA samples to be run at once.</LI><LI>Leave the gel to set for approximately 15 - 30 minutes.</LI><LI>Once the gel is set remove the comb and autoclave tape.</LI><LI>Place the gel tray inside the buffer tray and fill the remaining space with TAE buffer.</LI><LI>Load 'checking' samples
							</LI><LI>Load 5μl of 1kn ladder into the first well of the gel.</LI><LI>Pipette 2μl 5x loading buffer onto a sheet of parafilm.</LI><LI>pipette 8μl DNA onto the loading buffer and pipette up and down to mix.</LI><LI>Resting the pipette tip on the back of the well, load 8μl of sample.</LI><LI>Repeat for each sample.</LI><LI>Run the Gel at 100v for 60 minutes.</LI><LI>Load 'extraction' samples
							</LI><LI>Load 5μl of 1kn ladder into the first well of the gel.</LI><LI>Add 5x loading buffer into each sample, to give a final ratio of 1:4.</LI><LI>Resting the pipette tip on the back of the well, load as much sample as possible.</LI><LI>Repeat for each sample.</LI><LI>Visualise
							</LI><LI>Remove the agarose gel from the gel tray and place in the trans-illuminator.</LI><LI>Open UVP (on the computer desktop).</LI><LI>Select the type of gel and the exposure time.</LI><LI>Press capture to take an image.</LI><LI>Save image to the iGEM2014 folder.</LI></DIV></DIV><DIV id="protocol7" class="pageSection1"><DIV class="sectionHeading"><H2>Protocol 5: Pouring Plates</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						Sterile hood, Flask, Agar mix, dH2O, antibiotic, empty plates, P200, tips.
					<H3>Time</H3>
						Prep: 15 minutes
						Run: 20 minutes
					<H3>Procedure</H3><OL><LI>Use the sterile hood.</LI><LI>Make up 100ml LB Agar (Protocol 2).</LI><LI>Heat in the microwave as follows:
							</LI><LI>Power setting 4 for 3 minutes.</LI><LI>Wait for 6 minutes.</LI><LI>Power setting 4 for 2 minutes.</LI><LI>Wait to cool (until the flask is warm but comfortable to hold).</LI><LI>Add 100μl 1/1000 stock of appropriate antibiotic to the agar.</LI><LI>Pour 4 plates and leave inside the hood to cool and dry.</LI></OL></DIV></DIV><DIV id="protocol8" class="pageSection2"><DIV class="sectionHeading"><H2>Protocol 6: Measure Concentration of DNA with NanoDrop 2000</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						Nanodrop, P10, Tips, dH2O, Paper towel.
					<H3>Time</H3>
						Prep: 5 minutes
						Run: 5 minutes
					<H3>Procedure</H3><OL><LI>Set up the NanoDrop2000, select “Nucleic Acid”, then “Routine Verification”.</LI><LI>Spray paper towel with distilled water, then clean metal nodules on metal part and lid.</LI><LI>Add 1μl of buffer solution to nodule.</LI><LI>Close lid.</LI><LI>Select 'Blank' on the programme.</LI><LI>Reopen lid.</LI><LI>Add 2μl of DNA Sample.</LI><LI>Close lid.</LI><LI>Select 'measure' on the programme.</LI><LI>Take note of the concentration (in ng/ml).</LI><LI>Clean nodules again with distilled water.</LI><LI>Repeat for different DNA sample.</LI></OL></DIV></DIV><DIVID><DIV class="sectionHeading"><H2>Protocol 7: Transforming Cells To Ensure Competency</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						Eppendorfs, P1000, P200, P10, Tips, Competent cells, Ligation mix, Ice, Heat block, LB broth, Incubator, Agar plates, Flame, Spreader, Ethanol
					<H3>Time</H3>
						Prep: 5 minutes
						Run: 1 hour
					<H3>Procedure</H3><OL><LI>Using concentration measure by the nanodrop, calculate the volume of DNA sample required: C1V1 = C2V2.</LI><LI>Where C2 required is 10ng and V2 is 100μl.</LI><LI>Use the calculated volume of DNA to transform (Protocol 11) chemically competent cells before plating out.</LI><LI>Calculate the transformation efficiency in colonies per ng of DNA.</LI></OL></DIV></DIVID></DIV><DIV id="protocol10" class="pageSection2"><DIV class="sectionHeading"><H2>Protocol 8: Blunt End Ligation</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						Eppendorfs, P10, tips, Reaction buffer, dH2O, Restricted DNA, T4 Ligase.
					<H3>Time</H3>
						Prep: 10 minutes
						Run: 10 minutes
					<H3>Procedure</H3><OL><LI>Make up eppendorfs containing ligation mix, as shown:
							</LI><LI>10μl reaction buffer.</LI><LI>8μl water.</LI><LI>2μl Cut DNA.</LI><LI>1μl Cut plasmid.</LI><LI>1μl T4 ligase.</LI><LI>Flick to mix and leave for ten minutes.</LI></OL></DIV></DIV><DIV id="protocol11" class="pageSection1"><DIV class="sectionHeading"><H2>Protocol 9: Chemical Tranformation</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						Eppendorfs, P1000, P200, P10, Tips, Competent cells, Ligation mix, Ice, Heat block, LB broth, Incubator, Agar plates, Flame, Spreader, Ethanol.
					<H3>Time</H3>
						Prep: 5 minutes
						Run: 1 hour 45 minutes
					<H3>Procedure</H3><OL><LI>Take an appropriate number of eppendorfs, each containing 100μl of competent E. coli cells.</LI><LI>Into each one, pipette 5μl of each ligation mix.</LI><LI>Leave a last one to act as a negative control.</LI><LI>Put the eppendorfs on ice for 30 minutes.</LI><LI>Heat shock eppendorfs at 42°C for 30 seconds.</LI><LI>Put on ice for 2 minutes.</LI><LI>Add 1ml LB broth to each of the eppendorfs and then incubate at 37°C for 60 minutes.</LI><LI>Remove cells from incubator and spin down at 13000RPM for 60 seconds.</LI><LI>Pour off the supernatant and resuspend the cells.</LI><LI>Near a flame, pipette each of the remaining cells onto individual plates (prepared earlier).</LI><LI>Sterilise a glass spreader using ethanol (flamed) and spread cells evenly until the surface of the agar appears dry.</LI><LI>Incubate at 37°C overnight.</LI></OL></DIV></DIV><DIV id="protocol12" class="pageSection2"><DIV class="sectionHeading"><H2>Protocol 10: Sticky End ligation</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						Eppendorf, p10, tips, reaction buffer, dH2O, restricted DNA, T4 Ligase.
					<H3>Time</H3>
						Prep: 5 minutes
						Run: 30 minutes
					<H3>Procedure</H3><OL><LI>Make up eppendorfs containing the ligation mix, as follows:
							</LI><LI>2μ l reaction buffer.</LI><LI>10ng insert.</LI><LI>10ng plasmid.</LI><LI>1μ l T4 ligase.</LI><LI>Make up to 20μ l with dH2O.</LI><LI>Flick to mix and leave for 30 minutes.</LI></OL></DIV></DIV><DIV id="protocol13" class="pageSection1"><DIV class="sectionHeading"><H2>Protocol 11: Restriction Digest</H2></DIV><DIV class="sectionContent"><H3>Materials and Equipment</H3>
						Eppendorf, p10, p100, tips, restriction enzymes, ice, heat block.
					<H3>Time</H3>
						Prep: 5 minutes
						Run: 1 hour
					<H3>Procedure</H3><OL><LI>To each eppendorf add:
							</LI><LI>10 units of restriction enzyme 1 (usually 1μl).</LI><LI>10 units of restriction enzyme 2 (usually 1μl).</LI><LI>1μg DNA.</LI><LI>5μl 10x buffer.</LI><LI>Make up to 20μl with dH2O.</LI></OL><LI>Flick to mix.</LI><LI>Incubate at 37°c for 60 minutes.</LI><LI>To stop the reaction, heat the sample to an appropriate temperature to inactivate the restriction enzyme used.</LI><LI>Put on ice.</LI></DIV></DIV><DIV id="protocol14" class="pageSection2"><DIV class="sectionHeading"><H2>Protocol 12: Glycerol Stocks</H2></DIV><DIV class="sectionContent"><I>Allows stocks of cells to be kept in the -80</I><H3>Materials and Equipment</H3>
						P1000 pipette, tips, eppendorf tubes, microwave, 100% glycerol, overnight cultures of cells.
					<H3>Time</H3>
						Prep: 5 minutes
						Run: 5 minutes
					<H3>Procedure</H3><OL><LI>Take 800μl of overnight cells using a P1000 pipette and add into separate, labelled eppendorf tubes.</LI><LI>Microwave the 100% glycerol for 5 seconds.</LI><LI>Take 200μl of the glycerol using a P1000 pipette and add into each eppendorf tube.</LI><LI>Place on ice immediately.</LI><LI>Transfer to the -80°C freezer to be stored.</LI></OL></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Sheffield/WetLabJournal">http://2014.igem.org/Team:Sheffield/WetLabJournal</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Sheffield/WetLabJournal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Sheffield/WetLabJournal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Sheffield/WetLabJournal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Sheffield/WetLabJournal&amp;oldid=390984" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>