Team:Wageningen UR/Description/Biocontainment
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
★ Design
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
AWARDS
★ Entrepreneurship
★ Hardware
★ Software
★  Measurement
★ Model
Wageningen UR iGEM 2016
☰
˟
Overview
Description
Results
Achievements
BioBricks
Specificity
Regulation
Biocontainment
Model
Outreach
Team
Notebook
 
 
 
Biocontainment
Light Kill Switch
Hijacking Mechanisms
Kill Switch Design
Cas9 Kill Switch
Cas9 Reinforce Auxotrophy
Expression of Cas9 Variants
In vitro Cas9 Cleavage
References
Overview
Description
Results
Achievements
BioBricks
Applied Design
WetLab
Specificity
In vitro assay
Toxin Engineering
Varroa Isolates
Regulation
Detecting Mites
Population Dynamics
Toggle Switch
E. coli survival
Biocontainment
Light Kill Switch
Cas9 Kill Switch
Collaborations
Model
Population Dynamics
Light Kill Switch
E. coli survival
Beehave
Toxin Scanner
Outreach
Our Vision
Design
Collaboration
Events
Media
Integrated Practices
Team
Members
Students
Supervisors
Gallery
Attributions
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
Biocontainment
Appropriate biocontainment measures form a significant part of the BeeT project's Safety aspect. BeeT is intended to be used outside the lab, in beehives, where it will be in close contact with nature. As we cannot be sure about the effect on existing ecosystems if BeeT would be released in the environment, it must be engineered to die if it leaves the beehive. Our first measure to achieve this is a light-induced kill switch that relies on the balance of a bacterial toxin and an antitoxin that are expressed simultaneously. In the dark beehive, the system is unaffected. In the presence of blue light, a component of sunlight, the balance is disturbed in favour of the bacterial toxin. This will kill the bacterium.
As an additional safety measure, BeeT relies on the presence of a synthetic amino acid that is to be applied to the beehive. In the presence of the synthetic amino acid, catalytically dead Cas9 (dCas9) is produced that has the synthetic amino acid incorporated, at the expense of catalytically active Cas9. If BeeT escapes from the hive, it can no longer access the synthetic amino acid, and dCas9 can no longer be formed. Instead, catalytically active Cas9 is produced which cuts the BeeT genome as well as any heterologous DNA that is present, thereby killing the bacterium and preventing horizontal gene transfer.
Optogenetic Kill Switch
Hijacking Cellular Control Mechanisms
The optogenetic kill switch is the unification of two different genetic systems: an artificially-created promoter system activated by light (pDawn) and a toxin-antitoxin (TA) system native to Escherichia coli (MazEF).
The optogenetic system pDawn 1 is an altered version of a heme-sensing system native to Bradyrhizobium japonicum 2 . pDawn’s sensor component, the histidine kinase YF1, replaces the original oxygen-sensing domain with one that reacts to blue light (470 nm). In the absence of light, YF1 phosphorylates (and therefore activates) the response regulator protein FixJ, which in turn drives the promoter FixK2. This forms the system pDusk (Figure 1, top). When exposed to blue light, YF1 is inactivated, resulting in reduced expression of the "target" gene under FixK2. Addition of repressor protein λ cI to the system as an invertor creates pDawn (Figure 1, bottom), which is inactive in the dark and strongly increases expression when exposed to light.
As the name implies, the TA system MazEF 3 consists of two components: a toxin (MazF) and its complimentary antitoxin (MazE). The MazF protein functions as an endoribonuclease, cleaving mRNA and thereby inhibiting transcription, which is ultimately lethal for the cell. MazE forms a complex with MazF, preventing it from functioning. However, MazE is a more labile protein than MazF, degrading faster. For this reason, TA systems like MazEF are also known as addiction modules; the organism expressing the system becomes "addicted" to the antitoxin, and cell stasis or death ensues should expression be halted. In its native organisms, MazEF has a regulatory function. Under ordinary conditions, the proteins are co-expressed and the organism stays stable. However, under stressful conditions, such as nutrient starvation, this expression ceases. MazF is then free to cleave essential mRNAs, causing what is inferred to be a state of bacteriostasis 4 . This process is reversible up until approximately six hours after taking effect, the point of no return. Much like programmed cell death (PCD) in multicellular organisms, this prevents excess growth the bacterial population cannot afford, improving viability.
The optogenetic kill switch is designed to exploit this mechanism. The theoretical construct functions as follows: rather than having both proteins expressed until the population encounters stress, the kill switch places MazF under control of pDawn. This means that in the darkness of the beehive, where blue light irradiance is close to zero, no toxin is produced, allowing the cell to remain stable. Any leaky expression from the promoter can be countered by constitutively expressed MazE. However, when the cell is exposed to (sun)light over a longer period of time, large amounts of un-countered MazF are produced, resulting in cell death. Like microbial vampires, any BeeT bacteria that make it outside will perish in the light of the sun. With a response time on the order of several hours for both the toxin-antitoxin system and light sensor, the kill switch's design is well-suited to its function. Incidental exposure to light, such as opening the hive for beekeeping activities - as described by our beekeeper contacts - will not suffice to fully trigger the switch. Aside from biocontainment, the optogenetic system is also used in further regulatory mechanisms for BeeT. This is detailed on the Regulation page.
Figure 1: Basic designs of the optogenetic kill switch. Arrows indicate activation of a part, bars indicate inhibition. Top: Constitutively expressed toxin MazF is countered by MazE expressed under pDusk, which is active in the dark. In light, antitoxin production drops, causing cell death.  Bottom: MazF is kept repressed by pDawn, which is inactive in the dark. Leaky expression is countered by weakly expressed MazE, forming equilibrium. In light, toxin expression greatly increases, causing cell death.
Construction of The Kill Switch Genetic Circuit
During the design and creation process, two different configurations for the kill switch were considered. One such configuration was the aforementioned pDawn-MazF combination. Another had MazF constitutively expressed, countered by MazE expressed under pDusk, the inverse of pDawn, active in the dark and deactivated by light. We tested the degree of expression by both systems using fluorescent protein mCherry as a reporter, both in light and in the dark. We observed that, while the fluorescence of the pDawn reporter is greatly increased when exposed to light, the differences between ligth and dark under pDusk are much smaller (Figure 2). This agrees with the modeling results, proving the suitability of the pDawn system as a promoter for MazF expression.
Figure 2: Response of pDawn- and pDusk-expressing E. coli to intense blue light (equivalent to the component in direct sunlight) and total darkness. Left: cell pellets. Right: fluorescence over OD600. Fluorescent protein mCherry is used as a reporter.
Although we had some troubles creating the complete version of the optogenetic kill switch, the prospect of such a switch remains a promising possibility in the field of biocontainment. It is possible that a different cell suicide mechanism than MazEF should be considered.  For more information on the creation process, see the Optogenetic Kill Switch notebook entry.
Cas9 Kill Switch
To bulletproof our biocontainment strategy, we aimed to include a self-killing mechanism based on Cas9. To begin with, we wanted to use a bacterial strain developed by Mandell et al (2014) 1 . This “biocontainment strain” is auxotrophic for a synthetic amino acid, para-L-biphenylalanine (BipA). Several essential proteins of this bacterial strain were engineered to function only when BipA is incorporated in the active site, leading to death of the bacterium when the synthetic amino acid is not available. In case of BeeT, BipA should be applied to the beehive by adding it to the sugar water used to feed the colony. This makes the beehive the only place BeeT is capable of surviving in.
Even though the biocontainment of this organism is assured, heterologous DNA may remain in the environment. Since DNA is rather stable under certain circumstances 2 , there is a risk it is taken up by other bacteria through horizontal gene transfer 3 , 4 . In our project, we proposed to complement the biocontainment strain by creating a switch to cleave heterologous DNA, depending on the presence of BipA. Additionally, auxotrophy for BipA can be further strengthened by targeting genomic DNA as well.
Using Cas9 to Reinforce Auxotrophy
To create the switch, we aimed to engineer Streptococcus pyogenes Cas9Cas9 is a well-studied nuclease that is involved in bacterial adaptive immunity . Wildtype Cas9 has the ability to make double-strand DNA breaks, guided by a small RNA (gRNA) that has a sequence complementary to the target DNA. 5 to undergo the change from catalytically inactive Cas9 (dCas9) to partially active Cas9 (nCas9) when BipA is not available. nCas9 can be made by changing either one of two residues important for cleaving to alanine: the Asp10 residue or the His840 residue. nCas9 can still make single-strand DNA breaks, or nicks. However, it has been shown that nCas9 is able to cause double-strand DNA breaks when gRNAs are available to nick DNA on opposite strands in close proximity 6 .  If both Ala10 and His840 are replaced by alanine, all cleaving activity is lost, resulting in dCas9 7 .
In this project, we replaced the codon for Asp10 (GAT) of the His840Ala-version of nCas9 to the TAG stop codon. We gave a new function to this codon by providing the translation machinery for it: a tRNA that recognizes the TAG codon (tRNACUA) and an amino-acyl-RNA-synthetase (aaRS) that charges the tRNA either with BipA 8 (resulting in a new version of dCas9: dCas9-Ala10BipA(TAG)) or aspartate 9 (resulting in nCas9, or in our case nCas9-Ala10Asp(TAG)). Normally the TAG stop codon is recognized by Release factor 1, thereby terminating translation 10 . For this reason, we used a genetically recoded strain that lacks any TAG stop codons as well as Release Factor 1 11 , to make the translation of the TAG codon more efficient.
As long as charged tRNACUA(BipA) is available to BeeT, BipA is incorporated and dCas9-Ala10BipA(TAG) is formed (Figure 3)(1). To switch from expression of dCas9-Ala10BipA(TAG) to nCas9-Ala10Asp(TAG), we aimed to use dCas9-Ala10BipA(TAG) to repress transcription 12 , 13 of the aaRS/tRNACUA pair for aspartate (2). This happens for as long as BeeT stays in the hive, where BipA is supplied. When BeeT leaves the hive, no charged tRNACUA(BipA) is available. Consequently, the Cas9 transcript is not translated and dCas9-Ala10BipA(TAG) is not formed. This releases repression of the aaRS/tRNACUA pair for aspartate. As a result, tRNACUA(Asp) becomes available and the Cas9 transcript can be translated again, this time resulting in formation of nCas9-Ala10Asp(TAG) (3). The nCas9-Ala10Asp(TAG) cuts any DNA for which two suitable guide RNAs are offered, including all heterologous DNA (4).
Figure 3. Genetic circuit for switching between catalytically dead Cas9 and partially active (nickase form) Cas9 depending on the presence of a synthetic amino acid. Arrows indicate activation of a part, bars indicate inhibition. aaRS = amino-acyl tRNA synthetase, SAA = synthetic amino acid, NAA = native amino acid, dCas9 = catalytically dead form of Cas9, nCas9 = nickase form of Cas9.
Cloning And Expression of Cas9 Variants
We introduced the Ala(GCT)10→BipA/Asp(TAG) mutation in dCas9 13 . To express wild-type Cas9, dCas9 and Cas9-Ala(GCT)10→BipA/Asp(TAG), we cloned these genes into the Expresso system for rhamnose induced expression, from Lucigen 14 . This introduced a C-terminal His-tag, for purification by Ni-NTA chromatography. A vector (pEVOL-BipA) containing the aaRS/tRNACUA pair for translating the TAG stop codon to incorporate BipA was already available from the biocontainment strain 1 . For translating the TAG stop codon to aspartate, we constructed an alternative vector (pEVOL-Asp) containing the aaRS/tRNACUA pair.
To test translation of the TAG stop codon and incorporation of BipA and aspartate in Cas9, we cultured bacteria transformed with Cas9-expresso constructs and pEVOL-vectors and extracted and purified the Cas9 variants. SDS-PAGE of the purified samples showed clear expression of both Cas9 and dCas9, and lower expression of dCas9-Ala10BipA(TAG). A faint band could be seen for nCas9-Ala10Asp(TAG), but since a control sample without any synthetic amino acid also displayed a faint band, this result is not conclusive (Figure 4).
Figure 4. SDS-PAGE of fractions after FPLC purification of Cas9 variants. The expected size of Cas9 is 156 kDa. Yellow arrows indicate bands of the correct size corresponding to Cas9. a) Cas9. b) dCas9. c) dCas9-Ala10BipA(TAG). d) nCas9-Ala10Asp(TAG). e)negative control, Cas9-Ala(GCT)10(TAG), without added synthetic amino acid in the growth medium. Marker: Precision Plus protein ladder (Bio-Rad). CFE = Cell Free Extract.
In vitro Cas9 Cleavage Assay
To assess the functionality of our Cas9 variants we performed in vitro Cas9 cleavage assays. As a substrate, we used a PCR product of 4140 bp that included the gene encoding RFP. After cleavage, this will generate two fragments: one ~3100 bp fragment and one ~1040 bp fragment. All targets were chosen in close proximity on the DNA. As a result, all cleavage products would be roughly the same length, and targets on opposite DNA strands should result in a double strand break when incubated with nCas9-Ala10Asp(TAG).
From the assays (Figure 5) we concluded that our purified Cas9 is functional using several gRNAs targeting RFP, as the linear substrate was at least partially cleaved in all cases. As expected, dCas9 did not cleave DNA. When two gRNAs that target the RFP gene (in fig. 3, guide 1 and 2) on opposite DNA strands are offered to nCas9-Ala10Asp(TAG), it is expected to see some cleaving caused by the two nicks in close proximity. Indeed, partial cleavage was observed. However, we also observed some cleaving when we incubated with only one gRNA, which is unexpected. It has been shown that the His840Ala version of nCas9 6 , which is the same protein as our nCas9-Ala10Asp(TAG), still has some residual double strand cleaving activity. Further testing is needed to find out whether this caused cleaving by nCas9-Ala10Asp(TAG) in our case, or that it is an artifact of some sort.
Figure 5. in vitro Cas9 activity assays with Cas9, dCas9 and dCas9-Ala10Asp. Substrate for cleaving is a PCR product including the gene encoding RFP, which is targeted at the N-terminal side, both on the template strand (gRNA 2 and 4) and the non-template strand (gRNA 1). Size of the uncleaved PCR product is 4140 bp, cleaving generates a 3100 bp and a 1040 bp fragment. Wt = wildtype Cas9, d = dCas9, n = nCas9-Ala(GCT)10→ Asp(TAG). Marker: 1kb ladder (NEB).
In conclusion: this part of our project shows that an artificial amino acid can be incorporated in Cas9 in response to the TAG stop codon. We also tried to incorporate aspartate in response to the TAG stop codon, and while we have some indications that this restored nickase activity, further testing is needed to verify whether this is the case.
References
References for Optogenetic Kill Switch
1. Ohlendorf, R., Vidavski, R., Eldar, A., Moffat, K., Möglich, A. (2012). From Dusk Till Dawn: One-Plasmid Systems for Light-Regulated Gene Expression. Journal of Molecular Biology vol 416, 534-542. ↩
2. Möglich, A., Ayers, R., Moffat, K. (2009) Design and Signaling Mechanism of Light-Regulated Histidine Kinases. Journal of Molecular Biology, vol. 385, 1433-1444. ↩
3. Engelberg-Kulka, H., Hazan, R., Amitai, S. (2005). mazEF: a chromosomal toxin-antitoxin module that triggers programmed cell death in bacteria. Journal of Cell Science 118, 4327-4332. ↩
4. Amitai, S., Yassin Y., Engelberg-Kulka, H. (2004) MazF-Mediated Cell Death in Escherichia coli: a Point of No Return. Journal of Bacteriology vol. 186 no. 24 8295-8300. ↩
References for Cas9-Based Kill Switch
1. Mandell, D. J., Lajoie, M. J., Mee, M. T., Takeuchi, R., Kuznetsov, G., Norville, J. E., ... & Church, G. M. (2015). Biocontainment of genetically modified organisms by synthetic protein design. Nature, 518(7537), 55-60. ↩
2. Romanowski, G., Lorenz, M. G., Sayler, G., & Wackernagel, W. (1992). Persistence of free plasmid DNA in soil monitored by various methods, including a transformation assay. Applied and Environmental Microbiology,58(9), 3012-3019. ↩
3. Thomas, C. M., & Nielsen, K. M. (2005). Mechanisms of, and barriers to, horizontal gene transfer between bacteria. Nature reviews microbiology, 3(9), 711-721. ↩
4. Smillie, C. S., Smith, M. B., Friedman, J., Cordero, O. X., David, L. A., & Alm, E. J. (2011). Ecology drives a global network of gene exchange connecting the human microbiome. Nature, 480(7376), 241-244. ↩
5. Jinek, M., Chylinski, K., Fonfara, I., Hauer, M., Doudna, J. A., & Charpentier, E. (2012). A programmable dual-RNA–guided DNA endonuclease in adaptive bacterial immunity. Science, 337(6096), 816-821. ↩
6. Shen, B., Zhang, W., Zhang, J., Zhou, J., Wang, J., Chen, L., ... & Skarnes, W. C. (2014). Efficient genome modification by CRISPR-Cas9 nickase with minimal off-target effects. Nature methods, 11(4), 399-402. ↩
7. Mali, P., Aach, J., Stranges, P. B., Esvelt, K. M., Moosburner, M., Kosuri, S., ... & Church, G. M. (2013). CAS9 transcriptional activators for target specificity screening and paired nickases for cooperative genome engineering. Nature biotechnology, 31(9), 833-838. ↩
8. Xie, J., Liu, W., & Schultz, P. G. (2007). A genetically encoded bidentate, Metal‐Binding amino acid. Angewandte Chemie, 119(48), 9399-9402. ↩
9. Pastrnak, M., Magliery, T. J., & Schultz, P. G. (2000). A new orthogonal suppressor tRNA/aminoacyl-tRNA synthetase pair for evolving an organism with an expanded genetic code. Helvetica Chimica Acta, 83(9), 2277-2286. ↩
10. Scolnick, E., Tompkins, R., Caskey, T., & Nirenberg, M. (1968). Release factors differing in specificity for terminator codons. Proceedings of the National Academy of Sciences, 61(2), 768-774. ↩
11. Lajoie, M. J., Rovner, A. J., Goodman, D. B., Aerni, H. R., Haimovich, A. D., Kuznetsov, G., ... & Rohland, N. (2013). Genomically recoded organisms expand biological functions. Science, 342(6156), 357-360. ↩
12. Qi, L. S., Larson, M. H., Gilbert, L. A., Doudna, J. A., Weissman, J. S., Arkin, A. P., & Lim, W. A. (2013). Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression. Cell, 152(5), 1173-1183. ↩
13. Bikard, D., Jiang, W., Samai, P., Hochschild, A., Zhang, F., & Marraffini, L. A. (2013). Programmable repression and activation of bacterial gene expression using an engineered CRISPR-Cas system. Nucleic acids research, 41(15), 7429-7437. ↩
14. Expresso® Rhamnose Cloning & Protein Expression System ↩
iGEM Wageningen UR 2016. Send us an email or follow us on Facebook !
For more information about iGEM, check out their website and Twitter !
