<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Wageningen_UR/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><TITLE>Wageningen UR iGEM 2016 </TITLE><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><BUTTON class="hamburger">☰</BUTTON><BUTTON class="cross">˟</BUTTON><DIV class="mobilemenu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><B>Overview</B></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Team</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Notebook</A></LI></UL></DIV><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="#header">Results</A></H4></DIV><LI class="menu-item"><A href="#section1">Specificity</A></LI><LI class="menu-item"><A href="#section2">Regulation</A></LI><LI class="menu-item"><A href="#section3">Biocontaiment</A></LI><LI class="menu-item"><A href="#section4">Testing BeeT</A></LI><LI class="menu-item"><A href="#references">References</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design">Applied Design</A></LI></UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"><DIV class="ca-content">WetLab</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Assay"><I>In vitro</I> assay</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#ToxinEngineering">Toxin Engineering</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates" class="up_menu_fix"><I>Varroa</I> Isolates</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#DetectingMites" class="up_menu_fix">Detecting Mites</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing" class="up_menu_fix"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#Toggleswitch" class="up_menu_fix">Toggle Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/ecoli_survival"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#Lightswitch" class="up_menu_fix">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#SAA" class="up_menu_fix">Cas9 Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model"><DIV class="ca-content">Model</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#light">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#beehave">Beehave</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">Toxin Scanner</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Vision">Our Vision</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#collaboration">Collaboration</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#events">Events</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"><FONT size="1">Integrated Practices</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/contact">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#environment">Environment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#laboratory">Laboratory</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Journals</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments">Protocols</A></LI></DIV></DIV></DIV><DIV class="center_content"><SECTION id="section1"><H1><B>Results</B></H1><P>In this section, we present the main results of the BeeT-project. The BeeT-system is subdivided in three parts: specificity, regulation, and biocontainment. These parts are based on the main demands BeeT had to meet. To ensure BeeT's specificity, we engineered an existing toxin and identified a potential mite pathogen. Furthermore, we developed a toxicity-assay to facilitate future research on <I>Varroa destructor</I>-specific toxins. Also, software was made to aid in the discovery of <I>Varroa destructor</I>-specific toxins. For the regulation of toxin production, we developed two riboswitches that can detect mites, designed a genetic circuit which regulates the formation of BeeT subpopulations, and included a toggle switch which ensures BeeT is only functional in the beehive. BeeT's biocontainment was ensured through the introduction of an optogenetic kill switch and a Cas9 kill switch. We also modeled various parts of BeeT: the optogenetic kill switch, the subpopulation dynamics circuit, the effects of the method of application on the chassis, and the effect of BeeT on mite and bee population dynamics.</P><P>BeeT should only affect <I>Varroa</I> mites. To accomplish this, we decided to use Cry toxins. These toxins are naturally produced by <I>Bacillus thuringiensis</I> and therefore also known as Bt toxins. A functional Cry toxin is only effective when specific binding occurs to the gut membrane of the target organism. After binding, the Cry toxins will form pores in the cell membrane, resulting in cell death<SUP><A href="#res1" id="refres1">1</A></SUP>. To find a Cry toxin active against <I>V. destructor</I> we engineered our own toxins, and also searched for them in nature.</P><P>Because testing toxins on <I>Varroa</I> mites <I>in vivo </I>is near to impossible, we developed an <I>in vitro</I> test for Cry toxins.  Due to their parasitic nature, mites in laboratory conditions are very fragile and frequently die irrespectively of their treatment. For the new assay, we made Brush Border Membrane Vesicles (BBMV's) out of the membranes of <I>Varroa</I> mite. The vesicles were loaded with 6-carboxyfluorescein. Any functional Cry toxin will create pores into the BBMV's, which results in the leaking of fluorophores out of the BBMV's. Due to the self-quenching behaviour of 6-carboxyfluorescein, the leaking can be measured as an increase in fluorescence. As a proof of principle, BBMV's from the gut of <I>Tenebrio molitor</I> were made and loaded with 6-carboxyfluorescein to test the pore formation ability of Cry3Aa, which is known to be toxic to <I>T. molitor</I> larvae<SUP><A href="#res2" id="refres2">2</A></SUP>.

Figure 1a shows how the fluorescence of BBMV's incorporated with fluorophores increases in the presence and absence of Cry3Aa. The kinetic value was calculated for the process, which are shown in Figure 1b. From this we concluded that the <I>in vitro</I> toxicity assay can be used to test for functionality of Cry toxins.</P><FIGURE><FIGCAPTION>Figure 1. (a) The fluorescence of two solutions with BBMV's obtained from <I>T. molitor</I> incorporated with fluorophores were measured over time, one in the presence and one in the absence of Cry3Aa. (b) The reaction rate constants for six individual measurements were calculated with the equation: fluorescence<SUB>t</SUB>=fluorescence<SUB>(t=∞)</SUB>-fluoresence<SUB>(t=∞)</SUB>∙e<SUP>(-k∙t)</SUP>+fluorescence<SUB>(t=0)</SUB>. </FIGCAPTION></FIGURE><P>For the <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#ToxinEngineering ">engineering of a Cry toxin</A> specific for <I>V. destructor</I>, we used the Cry3Aa toxin as a starting point. This Cry toxin consists of three domains, off which one is responsible for binding. Rajamohan <I>et al</I>. (1996) demonstrated that mutations in the binding sites of Cry toxins can both decrease and enhance the specificity of a toxin towards its target organism<SUP><A href="#res3" id="refres3">3</A></SUP>. Three putative binding sites have been identified after analysing the 3D structure of the binding domain of Cry3Aa. These putative sites were changed with random mutagenesis, and the adapted proteins were cloned into <I>Escherichia coli</I>. 144 Cry proteins were produced and tested for activity on BBMV's from <I>V. destructor</I> as described previously. After initial testing, 24 candidates were selected for further testing. The results are shown in Figure 2.  It can be concluded that the third binding site, consisting of amino acids 410-416, is a good candidate for future engineering and specificity adaptation of this particular Cry toxin. Toxins Cry3.3.3 and Cry3.3.7 should not be taken into account due to their highly deviating reaction speeds.This leaves us with one proper candidate – the toxin mutant Cry3.3.5.

<FIGURE><FIGCAPTION>Figure 2. Heatmap of the reaction constants relative to the blank. A higher value indicates a higher toxicity and specificity of the tested toxin. </FIGCAPTION></FIGURE></P><P><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#engineering2 ">Phage display</A> was performed in order to find specific motifs for binding to <I>Varroa</I> gut-membrane receptors. Phages (filamentous bacteriophage M13) with a binding motif (The Ph.D.™-12 Phage Display Peptide Library) on their exterior were exposed to the gut membrane of <I>V. destructor</I> by feeding the phages to <I>Varroa</I> mites. Subsequently, we isolated and sequenced the recovered phages. The consensus sequence of the binding motif of the 12-mer is shown in Figure 3.</P><FIGURE><FIGCAPTION>Figure 3. Consensus sequence of recovered phages in <I>in vivo</I> Phage Display in <I>V. destructor</I>. The legend shows which aminoacid has which properties. The letters &quot;N&quot; and &quot;C&quot; in the graph indicate the N-terminus and the C-terminus, respectively.</FIGCAPTION></FIGURE><P> Alongside creating a Cry toxin ourselves, we <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates">searched nature</A> for existing Cry toxins. We gathered 800 dead <I>Varroa</I> mites and from them isolated <I>B. thuringiensis</I> or related species, which might have been the cause of death. Figure 4 shows the morphology of <I>B. thuringiensis</I> and two strains we discovered. Five out of 106 isolates were identified as <I>Bacillus</I>-like species. One strain, which turned out to be not <I>B. thuringiensis</I>, showed the presence of a large over-expressed protein and was sent for sequencing. We reduced the dataset to four candidate genes with the toxin scanner. This is a great starting point for future research.</P><FIGURE><FIGCAPTION>Figure 4.  Microscopy images of Coomassie-stained isolates, 1000x magnification with a Zeiss Axio Scope.A1 brightfield microscope. (a) <I>B. thuringiensis</I> HD350. The red arrow points to a Cry toxin, the green arrow to a spore, and the yellow arrow to a vegetative cell. (b) Isolate 62, a coccus. Most isolates had this morphology. (c) Isolate 82, showing <I>Bacillus</I>-like morphology.</FIGCAPTION></FIGURE></SECTION><SECTION id="section2"><P>A constant low level of Cry toxin can facilitate resistances<SUP><A href="#res4" id="refres4">4</A></SUP>. This is why the toxin production should be regulated when BeeT spreads through the hive. We created two main systems that regulate the toxin production. One is a system designed with <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#DetectingMites ">riboswitches</A> that promote toxin production when <I>Varroa</I> mites are present. The other system, that works in parallel with the riboswitches, uses quorum sensing to start toxin production only when the density of BeeT is high.</P><P>Riboswitches are parts of mRNA that can regulate gene expression depending on whether a ligand is bound. The ligands we used are guanine and vitamin B12. Both substances indicate the presence of <I>Varroa</I> mites. 95% of the mite faeces consists of guanine. Vitamin B12 is present in the haemolymph of the honey bees, and can be expected to leak into the hive when <I>Varroa</I> mites damage the honeybees. Both riboswitches were constructed successfully so that when the ligand is present, toxin can be produced. They have been tested with RFP as reporter gene in the presence of different concentrations of their corresponding ligand. The results for the vitamin B12 riboswitch are shown in Figure 5. As can be seen here, RFP production increases with increasing vitamin B12 concentration.</P><FIGURE><FIGCAPTION>Figure 5. <I>E. coli </I> with the vitamin B12 riboswitch coupled to a RFP output was grown overnight in the presence of different concentration vitamin B12. (a) The fluorescence divided by OD over time is shown for different concentrations vitamin B12. (b) The relation between the fluorescence divided by OD after 12 hours incubations and different concentrations of vitamin B12.</FIGCAPTION></FIGURE><P>The second regulatory system uses <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing ">quorum sensing</A>. A quorum sensing mechanism enables the bacteria to regulate their expression based on their density. We adopted the lux system originating from <I>Vibrio fischeri</I> and demonstrated this system’s functionality using a newly constructed GFP reporter (Figure 6). When cell density increases, the cells will sense each other’s autoinducers. This induces production of more autoinducers and GFP.</P><FIGURE><FIGCAPTION>Figure 6. Fluorescence and absorbance data for E. coli quorum sensing strains. The continuous lines represent the fluorescence divided by OD600. The dashed lines represent the absorbance at 600 nm. The red and green lines represent quorum sensing strains, the purple strain has a reporter plasmid only. For both strains every value displayed is the average of at least three technical replicates, and for each the line displayed is one of three biological repeats of which all show a similar pattern.</FIGCAPTION></FIGURE><P>When the cell density is high enough, the quorum sensing system ensures that more and more toxin is produced. The downside of this is that high toxin levels will likely kill the BeeT population. This is because the Cry toxin will lyse BeeT when produced in very high concentrations. It would be beneficial to divide the population of bacteria in toxin producers and non-toxin producers, to maintain a subpopulation of healthy bacteria. These cells will be able to initiate a new growth phase after death of the toxin-producing cells. This requires that cells respond to the stimuli at different times despite being genetically identical. To create such a system, we used two proteins: one encodes for the protein that inhibits the toxin expression, whereas the other promotes toxin expression. Depending on which protein is more present, toxin production is either on or off. Both proteins are encoded behind the same promoter. However, one of the proteins has a higher turnover rate. The trick is to find the “sweet spot” of the translation rates at which in some cells one protein takes the upper hand ,and in some cells the other protein. This sweet spot has been found with a mathematical <A href="https://2016.igem.org/Team:Wageningen_UR/Model#QS_combined ">model</A>. Figure 7 shows the presence of two different subpopulations as computed by the model.</P><FIGURE><FIGCAPTION>Figure 7. Two populations are visible and together they form a growing cell population. The right y-axis shows the volume of these populations. Volume oscillations correspond dividing cells. The total amount of RFP produced by the toxin is shown by the black line.</FIGCAPTION></FIGURE><P>As an alternative to quorum sensing, we added a <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#Toggleswitch ">toggle switch</A> to the system that allows BeeT to regulate toxin production even if the bacteria do not grow well in beehives. Slow growth is a limitation of the quorum sensing system: cells might not be able to grow to the density required for toxin production. Instead, the toggle switch system makes use of the earlier described riboswitch, which is not dependent on population density.  The toggle switch we created controls expression of the BeeT’s toxin between an off-state and an on-state. It is switched on by guanine or vitamin B12, and switched off by blue light. The latter is based on the optogenetic kill switch, explained later in more detail. Apart from combining multiple systems, the toggle switch ensures that the response to guanine or vitamin B12 is fast. To create this system, a new hybrid promoter was made. The hybrid promoter ensures that toxin production is only possible in the absence of light. Figure 8 shows the results of 5 different hybrid promoters controlling expression of RFP. From this we concluded that the hybrid promoter <A href="http://parts.igem.org/Part:BBa_K1913025">BBa_K1913025</A> is the most active. Although we did not have time to test the system as a whole, we expect it to work since both the riboswitches and the hybrid promoter are functional separately.</P><FIGURE><FIGCAPTION>Figure 8. For five <I>E. coli</I> cultures, each with a different hybrid promoter and all with the reporter gene RFP, the fluorescence divided by OD 600 is shown. The cultures were grown overnight in the dark. As a negative control, the same <I>E. coli </I> with the corresponding hybrid promoter but lacking the light sensor was used.</FIGCAPTION></FIGURE></SECTION><SECTION id="section3"><P>BeeT is intended to use in beehives, where bee’s fly in and out continuously. This means BeeT can be spread by the bee’s throughout the environment. Since we cannot be sure about its effect on existing ecosystems, BeeT must be engineered to die if it leaves the beehive. To accomplish this we made use of an optogenetic kill switch and a Cas9 kill switch.</P><P>The optogenetic kill switch is the unification of two different genetic systems: a toxin-antitoxin system native to <I>E. coli</I> (MazEF), and an artificially-created promoter system activated by light (pDawn). The toxin MazF is only expressed in the presence of light, because MazF is regulated via pDawn. The antitoxin MazE is constitutively expressed, to protect the cell against leaky expression of MazF. This means that in the darkness of the beehive, where blue-light irradiance is close to zero, no toxin is produced. This allows the cells to remain stable. However, in sunlight toxin production takes the upper hand and the cell dies. Figure 9 demonstrates that the pDawn promoter system works. Alongside pDawn we tested pDusk, a promoter system activated in the absence of light. This promoter system did not provide a strong enough response to be useful for our intended purpose.</P><FIGURE><FIGCAPTION>
Figure 9. Response of pDawn- and pDusk-expressing <I>E. coli</I> to intense blue light (equivalent to the component in direct sunlight) and total darkness. Left: cell pellets. Right: fluorescence over OD600. Fluorescent protein mCherry is used as a reporter. 
</FIGCAPTION></FIGURE><P>The artificially-created promoter systems pDusk and pDawn were modelled in Matlab together with the MazEF toxin-antitoxin system. We fitted the model to literature data and can conclude that our model describes the system’s behaviour in the wet-lab well for pDusk and pDawn. Within our parameter estimation procedure for the extended pDusk + const. mazF and pDawn + const. mazE systems, we found two parameter sets which satisfy the conservative constraints. This is described in the <A href="https://2016.igem.org/Team:Wageningen_UR/Model#light">optogenetic kill switch modelling section</A>. The results from these two sets can be seen in the animated Figure 10. With increasing light intensities, the response of MazE and MazF is plotted. This gives us an indication on where in parameter space our focus should be for future studies and how the model should be extended with further wet-lab experimental data. In addition, Figure 10 indicates that it takes a few hours for the MazF toxin to take the upper hand in the pDawn system. Backed up by literature data, we can assume that the beekeepers can open their beehives during work, without immediately destroying BeeT.</P><FIGURE><FIGCAPTION>Figure 10. Response of the two parameter sets which satisfy the conservative constraints. The light intensity is changing each moving step. The legend in pDusk is also valid for pDawn. The letter 'e' resembles the antitoxin MazE, whereas 'f' resembles the toxin MazF</FIGCAPTION></FIGURE><P>We added an additional <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#SAA ">kill switch</A>, to reinforce our biocontainment strategy. As a chassis for BeeT we wanted to use a bacterial strain developed by Mandell and colleagues (2014)<SUP><A href="#res5" id="refres5">5</A></SUP>. This “biocontainment strain” is auxotrophic for a synthetic amino acid, para-L-biphenylalanine (BipA). We aimed to complement this strain by adding a measure to prevent horizontal gene transfer.  Our objective was to cleave heterologous DNA with a modified Cas9 as soon a BeeT runs out of BipA. When BipA is present, the synthetic amino acid should be built into the active site of Cas9, making it catalytically dead. However, in the absence of BipA, the native amino acid is incorporated, partially restoring cleaving activity. This active Cas9 will cut heterologous DNA. We managed to incorporate BipA in Cas9, which is shown in Figure 11.</P><FIGURE><FIGCAPTION>Figure 11. SDS-PAGE of fractions after FPLC purification of Cas9 with incorporated BipA (a) and a negative control (b). The expected size of Cas9 is 156 kDa. Marker: Precision Plus protein ladder (Bio-Rad).</FIGCAPTION></FIGURE></SECTION><SECTION id="section4"><H1><B>Testing BeeT in <STRIKE> a Beehive</STRIKE>  Beehave</B></H1><P>Ideally we want to test BeeT in a beehive. This is, however, not a feasible  option for this iGEM project. Allowing genetically modified organisms to be present in the environment is far from responsible, moreover forbidden. Because of this we had to find an alternative way to test BeeT. First we proved in an experiment and with a model that BeeT can survive in the sugar water, the medium used to apply BeeT to the bees. Secondly, we modeled the influence of BeeT in an open source model called beehave. We adapted the model in a way that it could predict what the effect of BeeT on virus epidemiology, mite population dynamics, and bee population dynamics is.</P><P>Using <A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic ">Flux Balance Analysis</A> we describe the relationship between the metabolism of <I>E. coli</I> and the osmotic pressure of sugar water. From this we can predict how different thresholds of minimal cell-water tolerance will affect the relationship between the survival time and the maximum ATP available for survival (Figure 12). Our model predicts an infinite survival time beyond 90 minutes. We’ve <A href="https://2016.igem.org/Team:Wageningen_UR/ecoli_survival ">proven in the lab</A> that <I>E. coli</I> can survive at least 24 hours in sugar concentrations that are similar to sugar water for bees. Taking the model into account, we assume that <I>E. coli</I> will survive indefinitely in sugar water. This is taken into account in the beehave model.</P><FIGURE><FIGCAPTION>Figure 12. The relationship between the maximum ATP available for survival for an <I>E. coli</I> in a sugar-water environment and the theoretical survival time, given a constant water efflux over this time and a starting volume of 2.8e<SUP>-13</SUP> grams. The various coloured lines indicate water tolerance thresholds for the <I>E. coli</I></FIGCAPTION></FIGURE><P>We modelled the behavior in <A href="https://2016.igem.org/Team:Wageningen_UR/Model#MM_model ">beehave </A> mainly because we are interested in how BeeT can best be applied given certain <A href="https://2016.igem.org/Team:Wageningen_UR/Model#MM_intro">assumptions</A>. If no functional BeeT is applied to the hive, the bee population dynamics will follow the trend as shown in Figure 13a. In other words, the bee colony will collapse after four to five years. If functional but not 100% effective BeeT is applied, the bee population will shrink, and reach an equilibrium with the mite population. (Figure 13b) If effective BeeT is applied to the hive, the mite population dies (Figure 13c). </P><FIGURE><FIGCAPTION>
Figure 13. The honey bee population is shown in blue and the <I>Varroa</I> mite population in red. A: Colony rapidly declines when no BeeT is present. Starting population is 20 <I>Varroa</I> B: Colony barely survives <I>Varroa</I> mite infestation. Shows <I>Varroa</I> mite in red and worker bee population in blue. Starting population is 20 <I>Varroa</I>. C: Colony thrives regardless of <I>Varroa</I> mite infestation. Starting population is 20 <I>Varroa</I> mites. D: Colony thrives regardless of heavy <I>Varroa</I> mite infestation. Starting population is 10.000 <I>Varroa</I> mites.
</FIGCAPTION></FIGURE><P>Furthermore, beehave predicted the most effective time and method to apply BeeT. As the results in Table 1 show, it is more effective to give BeeT-containing sugar water in spring rather than in autumn. Secondly, the model showed that application of BeeT is even more effective using a <I>Lactobacillus</I> species as chassis. This would allow application of BeeT via artificial ‘beebread’.</P><FIGURE><FIGCAPTION> Table 1. Three parameter sets per treatment, representing: colony death, survival, and thriving. If colonies can survive and thrive with higher degradation of BeeT (in-hive and outside the hive) and a lower effect of BeeT on <I>Varroa</I> mite mortality, it indicates a more effective treatment.
</FIGCAPTION></FIGURE><TABLE><TBODY><TR><TH>
  			 Period and treatment
   			</TH><TH>
  			 Colony death
   			</TH><TH>
   			Colony survival
   			</TH><TH>
			Colony thriving
			</TH></TR><TR><TH>
  			Sugar water, spring
   			</TH><TD>
  			 6,6%
   			</TD><TD>
  			 80,6%
   			</TD><TD>
			12,8%
			</TD></TR><TR><TH>
     			Bee bread, spring
    			</TH><TD>
     			0%
    			</TD><TD>
			2,9%
			</TD><TD>
			97,1%
			</TD></TR><TR><TH>
   			Sugar water, winter
   			</TH><TD>
   			15,1%
   			</TD><TD>
			80,7%
</TD><TD>
  			4,2%
   			</TD></TR><TR><TH>
     			Bee bread, winter
    			</TH><TD>
     0
    </TD><TD>
			57,6%
			</TD><TD>
			42,4%
			</TD></TR></TBODY></TABLE></SECTION><SECTION id="references"><H1>References</H1><OL class="references"><A id="res1" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1857359/">1.</A>Bravo, A., Gill, S. S., &amp; Soberon, M. (2007). Mode of action of Bacillus thuringiensis Cry and Cyt toxins and their potential for insect control. Toxicon, 49(4), 423-435.<A href="#refres1" title="Jump back to footnote 1 in the text.">↩</A><A id="res2" href="http://www.jbc.org/content/284/27/18401.full.pdf">2.</A> Jeff Fabrick, Cris Oppert, Marce´ D. Lorenzen, Kaley Morris, Brenda Oppert, and Juan Luis Jurat-Fuentes. A Novel Tenebrio molitor Cadherin Is a Functional Receptor for Bacillus thuringiensis Cry3Aa Toxin. The Journal of Biological Chemistry VOL. 284, NO. 27, pp. 18401–18410, July 3, 2009. 
<A href="#refres2" title="Jump back to footnote 2 in the text.">↩</A><A id="res3" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC26133/">3.</A> Rajamohan F, Alzate O, Cotrill JA, Curtiss A, Dean DH. Protein engineering of Bacillus thuringiensis δ-endotoxin: Mutations at domain II of CryIAb enhance receptor affinity and toxicity toward gypsy moth larvae. Proceedings of the National Academy of Sciences of the United States of America. 1996;93(25):14338-14343. 
<A href="#refres3" title="Jump back to footnote 3 in the text.">↩</A><A id="res4" href="http://www.biosecurity.govt.nz/files/pests/varroa/control-of-varroa-guide.pdf">4.</A> Goodwin, M., &amp; Van Eaton, C. (2001). Control of <I>Varroa</I>. A guide for New Zealand Beekeepers. New Zealand Ministry of Agriculture and Forestry (MAF). Wellington, New Zealand.
<A href="#refres4" title="Jump back to footnote 4 in the text.">↩</A><A id="res5" href="http://www.nature.com/nature/journal/v518/n7537/abs/nature14121.html">5.</A> Mandell, D. J., Lajoie, M. J., Mee, M. T., Takeuchi, R., Kuznetsov, G., Norville, J. E., ... &amp; Church, G. M. (2015). Biocontainment of genetically modified organisms by synthetic protein design. Nature, 518(7537), 55-60.
<A href="#refres5" title="Jump back to footnote 5 in the text.">↩</A></OL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2016. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV></DIV></BODY></HTML>