<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Wageningen_UR/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><TITLE>Wageningen UR iGEM 2016 </TITLE><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><BUTTON class="hamburger">☰</BUTTON><BUTTON class="cross">˟</BUTTON><DIV class="mobilemenu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><B>Overview</B></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Team</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Notebook</A></LI></UL></DIV><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="#header">Notebook</A></H4></DIV><LI class="menu-item"><A href="#general">General Protocols</A></LI><LI class="menu-item"><A href="#shower">Mite Shower</A></LI><LI class="menu-item"><A href="#engineering">Protein Engineering</A></LI><LI class="menu-item"><A href="#toxassay"><I>In Vitro</I> Assay</A></LI><LI><A href="#assay1">Preperation BBMVs</A></LI><LI><A href="#assay2">Incorporation of CF</A></LI><LI><A href="#assay3">TEM</A></LI><LI><A href="#assay4">DLS</A></LI><LI><A href="#assay5">Cry3Aa protein extraction</A></LI><LI><A href="#assay6">CF leakage experiments</A></LI><LI class="menu-item"><A href="#isolates"><I>Varroa</I> Isolates</A></LI><LI><A href="#isolates1">Finding Isolates</A></LI><LI><A href="#isolates2"><I>In Vivo</I> toxicity</A></LI><LI><A href="#isolates3">16s rRNA PCR</A></LI><LI><A href="#isolates4">LC-MS/MS</A></LI><LI class="menu-item"><A href="#cas">Cas9 kill switch</A></LI><LI><A href="#cas1">Mutagenesis PCR</A></LI><LI><A href="#cas2">Yeast assembly</A></LI><LI><A href="#cas3">gRNA cloning</A></LI><LI><A href="#cas4"><I>in vitro</I> gRNA transcription + isolation</A></LI><LI><A href="#cas5">Cas9 expression</A></LI><LI><A href="#cas6">Cas9 <I>in vitro</I> assay</A></LI><LI class="menu-item"><A href="#ecoli"><I>E. coli</I> survival</A></LI><LI class="menu-item"><A href="#light-exposure">Light sensor response assay</A></LI><LI class="menu-item"><A href="#references">References</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design">Applied Design</A></LI></UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"><DIV class="ca-content">WetLab</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Assay"><I>In vitro</I> assay</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#ToxinEngineering">Toxin Engineering</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates" class="up_menu_fix"><I>Varroa</I> Isolates</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#DetectingMites" class="up_menu_fix">Detecting Mites</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing" class="up_menu_fix"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#Toggleswitch" class="up_menu_fix">Toggle Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/ecoli_survival"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#Lightswitch" class="up_menu_fix">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#SAA" class="up_menu_fix">Cas9 Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model"><DIV class="ca-content">Model</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#light">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#beehave">Beehave</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">Toxin Scanner</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Vision">Our Vision</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#collaboration">Collaboration</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#events">Events</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"><FONT size="1">Integrated Practices</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/contact">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#environment">Environment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#laboratory">Laboratory</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Journals</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments">Protocols</A></LI></DIV></DIV></DIV><DIV class="center_content"><SECTION id="general"><H1><B>General Protocols</B></H1><P>All protocols used by the team are listed on this page or explained in PDF files. We have used standard iGEM protocols and protocols that we derived from literature or designed ourselves. </P><UL><LI><A href="https://static.igem.org/mediawiki/2016/2/27/T--Wageningen_UR--Polymerase_Chain_Reaction.pdf">Polymerase Chain Reaction</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/f/f8/T--Wageningen_UR--Colony_PCR.pdf">Colony PCR</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/0/0a/T--Wageningen_UR--Gel_Electrophoresis.pdf">Gel Electrophoresis</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/b9/T--Wageningen_UR--Gel_Extraction_of_DNA.pdf">Gel Extraction of DNA</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/a/ac/T--Wageningen_UR--Restriction_Enzyme_Digestion.pdf">Restriction Enzyme Digestion</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/7/76/T--Wageningen_UR--Ligation.pdf">Ligation</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/0/06/T--Wageningen_UR--Transformation_of_E.pdf">Transformation of <I>E. coli</I></A></LI><LI><A href="https://static.igem.org/mediawiki/2016/e/ee/T--Wageningen_UR--E.pdf"><I>E. coli</I> Cell Culture</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/bd/T--Wageningen_UR--MiniPrep.pdf">MiniPrep</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/d/d7/T--Wageningen_UR--SDS_Page_Gel_Electrophoresis.pdf">SDS Page Gel Electrophoresis</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/1/15/T--Wageningen_UR--Measurement.pdf">Measure the fluorescence curve</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/1/18/T--Wageningen_UR--HiFi_Gibson_Assembly.pdf">HiFi Gibson Assembly</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/9/9c/T--Wageningen_UR--Preparing_and_transforming_chemically_competent_cells_using_the_BacGen_protocol.pdf">Preparing chemically competent cells</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/0/00/T--Wageningen_UR--Preparing_and_transforming_DH5%CE%B1_electrocompetent_cells.pdf">Preparing DH5α electrocompetent cells</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/d/d4/T--Wageningen_UR--PCR_product_clean.pdf">PCR product clean-up</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/e/ef/T--Wageningen_UR--Protein_Extraction.pdf">Protein Extraction</A></LI></UL></SECTION><SECTION id="shower"><H1><B>Mite Shower</B></H1><P> In a regular beehive, most <I>Varroa</I> mites will be inside the closed brood cells. This makes it difficult to gather them for laboratory assays. Therefore, Tjeerd Blacquìere and Delphine Panziera (Wageningen UR, in personal communication) taught us how to manage a mite shower<SUP><A href="#mi1" id="refmi1">1</A></SUP>. This method of managing beehives makes sure that more mites are present on the honeybees and can be collected. Figure 1 provides an overview of a mite shower. 
<FIGURE><FIGCAPTION>Figure 1. A schematic overview of a mite shower. The numbers indicate different steps required to manage the mite shower. A queen excluder is a type of roster used to keep the queen restricted to the top half of the beehives. A more detailed explanation of this figure is in the paragraph below.</FIGCAPTION></FIGURE></P><P>
For a mite shower, two colonies are necessary: the companion colony and the mite shower colony. The companion colony produces brood infested with mites, while the mites can be gathered from the honeybees of the mite shower colony. </P><P>
During step 1, young and open brood from the top of the companion colony is moved to the bottom half of the colony. This brood is infested with <I>Varroa</I> mites. The queen excluder is used to keep the queen in the top half of the hive; extra care needs to be taken to ensure she is not moved downwards together with the young brood. This is done to prevent the queen from laying more eggs around the young brood. </P><P>Step 2 takes place a week later; most of the open brood will now have been capped by the nurse bees. <I>Varroa</I> mites are trapped inside. If there is too much open brood on the frame, it should not be moved. If it is mostly or only closed brood, it can be moved to the bottom half of the mite shower colony. This is also the frame from which you can gather larvae to feed collected <I>Varroa</I> mites. </P><P>Step 3 is not vital, but simply necessary to compensate for the frames taken out in other steps. Empty frames can be used to replace the ones that were swapped in steps 1, 2 and 4. </P><P>Step 4 is necessary to make sure that there is no open brood present in the mite shower colony. If there is open brood, the mites will mostly go in the brood cells instead of staying on the honeybees. This will make it harder to collect them. </P><P>These steps need to be repeated once every week at the same day to prevent mites from entering brood cells. If done correctly, it will make the powdered sugar method<SUP><A href="#mi2" id="refmi2">2</A></SUP> more effective. 
The powdered sugar method can be used to detach <I>Varroa</I> mites from honeybees. For this method, the following items are needed:
<UL><LI>Powdered sugar</LI><LI>A jar with a mesh big enough to keep honeybees in, but let mites out</LI><LI>Brush</LI><LI>A mite-tight container</LI></UL></P><P>
A frame from the bottom half of the mite shower colony can be shaken over the jar, so as many honeybees as possible are trapped in the jar. After the jar has been closed, a tablespoon of sugar should be poured over the bees and the jar should be shaken thoroughly. When the bees and sugar are properly mixed, the sugar should be shaken from the jar. The mites should come off along with the powdered sugar. </P><P>The brush, when wetted, is highly suitable for picking up the mites without harming them. It can be used to move the mites to the mite-tight container. To keep the mites in a good condition, it is necessary to keep them out of sunlight and wash them as soon as possible. Being covered in the powdered sugar will kill them rapidly.  Additionally, they will only survive 24 hours without larvae to feed on, so make sure to gather some larvae as well. </P><P>Finally, return the honeybees to the companion colony. As the companion colony will only have a few young honeybees, the sugary honeybees can be used to replenish their population. </P><FIGURE><VIDEO width="720" height="400" controls="" style="display:block;margin-left:auto;margin-right:auto;"><SOURCE src="https://static.igem.org/mediawiki/2016/9/9e/T--Wageningen_UR--hivemaintenance.mp4" type="video/mp4"><P>Your browser does not support this video. Please upgrade your browser.</P></SOURCE><FIGCAPTION>This video shows how we checked our beehive frames for the presence of the queen before we swapped the frames.</FIGCAPTION></VIDEO></FIGURE><SECTION id="engineering"><H1><B>Protein Engineering</B></H1><H2><B>Targeted Mutagenesis/Randomization of Binding Sites</B></H2><P>To create a library of Cry3Aa toxins randomly mutated at one of the three binding sites, three different primer pairs have been obtained. These bound next to the binding site and one of them had an overhang of the corresponding amount of Ns to bridge the binding site. This primer has also been phosphorylated. A PCR has been performed, amplifying the whole plasmid (except the binding site). After purification, the DNA fragments have been ligated with T4 ligase and then transformed.</P><FIGURE><FIGCAPTION>Figure 2. Schematic overview of the process of obtaining random binding sites.</FIGCAPTION></FIGURE><H2><B>Protein Expression in 96-well Plates and Protein Extraction</B></H2><P>Proteins were expressed in 96-well plates in a way adapted from Knaust <I>et al.</I> 2001. The mutants have been grown and protein expression has been induced in the following way:</P><P>
1)	Inoculation of single colonies in 200 µL LB containing corresponding antibiotics. Incubate overnight at 37 °C.
2)	Add 1 mL LB per well containing corresponding antibiotics. Incubate at 37 °C for 1 h.
3)	Induce with 1 % arabinose (120 µL 10% arabinose solution). Incubate at 37 °C for 3 h.</P><P>
The water-soluble proteins have then been extracted using B-PER:</P><P>
4)	Transfer 1 mL culture to 1.5 mL Eppendorf reaction tubes. Spin down at 5000xg for 10 min.
5)	Discard the supernatant and add 4 mL of B-PER per g of bacterial cells. (In this case, the mean weight of 9 samples has been evaluated and the proper amount of B-PER has been added to every sample. This amount was 75 µL.
6)	Vortex to totally dissolve the pellet. Incubate at room temperature for 15 min.
7)	Centrifuge at 15000xg for 5 min.
8)	Take supernatant and discard of the pellet.</P><H2><B>SM Buffer</B></H2><P>5.8 g/l NaCl
2 g/l MgSO4.7H2O
50ml 1MTris, pH 7.5
Filter sterilize

</P><H2><B>Feeding Experiments</B></H2><H3><B>Mites</B></H3><P>Living mites were obtained from bee hives and kept on bee larvae. Bee larvae were injected with 1 mM fluorescine, 10 mM fluorescine, or tap water. The mites were placed directly on the larvae and incubated at 35 °C overnight.
Every single mite has been washed in 1 mL PBS buffer and then ground up and homogenized with 0.5 ml PBS buffer. 100 µL of the resulting mixture was mixed with 100 mL ddH2O containing NaOH. The fluorescence was measured in a platereader.
</P><H3><B>Mealworms</B></H3><P>To feed mealworms, a 1 cm³ piece of boiled carrot has been injected with 10 mM, 1 mM and 0mM fluorescine. The mealworms were kept in a big plate with flour, oat flakes, and the piece of carrot at 37 °C. At three points in time, a mealworm was removed from the plate and its gut was homogenized with 1 mL of PBS. The measurement was the same as described in the mite protocol above.</P><H2><B>Phage Tittering</B></H2><P>
To determine the amount of plaque-forming units inside of a phage solution, the solutions were titrated. For this, a serial dilution of the phage library up to a dilution of 10-12 in SM buffer was prepared. 100 µL of each dilution was mixed with 3 mL of 0.7 % LB top agar and 100 µL of a fresh host culture (E.coli K12 ER2738). This was poured onto an IPTG/X-GAL LB Agar plate (1.5%) and incubated at 37 °C over night.
On the resulting plates, the plaques with the right morphology (blue) were counted. From these, the titre can be calculated.
</P><FIGURE><FIGCAPTION>Figure 3. Formula used for obtaining the phage titre.</FIGCAPTION></FIGURE><H2><B><I>In vivo</I> Phage Display</B></H2><H3><B>Mites</B></H3><P>Mites were obtained freshly from bee hives and kept on bee larvae until the experiments. To feed them phages, bee larvae have been injected with a phage containing solution in excess (the volume varied, phages were injected until the solution exited the larvae’s body again). 
The mites were kept on the injected bee larvae overnight at RT in darkness. Afterwards, they were isolated from the larvae and homogenized with 1000 µL of SM-buffer. The resulting solution was spun down in a micro-centrifuge for 15 min at max speed. After that, the supernatant was filtered with a 0,2 µm filter. The resulting phage containing solution was then used for determination of the phage titre and for the next round of phage display.

</P><H3><B>Mealworms</B></H3><P>Mealworms have been treated similar to mites. To feed them the phage library, a 1 cm² piece of boiled carrot has been injected with a phage solution. Per round and tested library, 5 mealworms were kept with flour, oat flakes and the piece of carrot in a large petri dish. They were kept at RT overnight in darkness. 
To obtain the phages for the next round, the mealworms have been dissected and the guts of 5 mealworms have been homogenized with 1,5 mL of SM buffer. The resulting solution was spun down in a micro-centrifuge for 15 min at max speed. After that, the supernatant was filtered with a 0,2 µm filter. The resulting phage containing solution was then used for determination of the phage titre and for the next round of phage display.
</P><H2><B><I>In vitro</I> Phage Display</B></H2><P>For the <I>in vitro</I> phage binding display, the protocol of Giordano <I>et al</I><SUP><A href="#lm1" id="reflm1">3</A></SUP>. has been adapted. 
1) 20 µL of vesicles were incubated at ice for 3 h with 5 µL of phages, resulting in a start titre of 10^8 PFU. 
2) The vesicles were added into an aquaeous phase of LB with 1 % BSA inside of an organic phase consisting of (1:9) cyclohexane:dibutylphthalate inside of a 15 ml tube.
3) The tubes were spun down at 4000xg for 1h at 4°C.
4) The tubes were incubated in dry ice for 30 min.
5) The frozen aqueous phase was separated from the organic phase.
6) The pelleted vesicles were resuspended in SM buffer.</P></SECTION><SECTION id="toxassay"><H1><B><I>In Vitro</I> Assay</B></H1><H2 id="assay1"><B>Preparation of Brush Border Membrane Vesicles</B></H2><P>For preparation of brush border membrane vesicles (BBMVs) from <I>T. molitor</I>, <I>Tenebrio molitor</I> larvae were dissected and the midguts were removed. The obtained midguts were shortly washed in dissection buffer (300mM mannitol, 5mM EGTA, 20mM mercaptoethanol, 2mM EDTA, 10mM HEPES, 2 tablets/100mL cOmplete<SUP>TM</SUP> protease inhibitor cocktail pH 7.5) and immediately frozen at -80⁰C. 5 Frozen midguts from <I>T.molitor</I> were placed in a 2mL eppendorf tube and used in the next step. For preparation of BBMVs from <I>Varroa destructor</I>, 15-20 <I>Varroa</I> mites were placed in a 2mL eppendorf tube and stored at -80⁰C.During the following steps, the samples were kept on ice as much as possible. 250µL of homogenization buffer (200mM mannitol, 10mM ascorbic acid, 5mM EDTA, 10 mM HEPES, 1% HEPES, 2 tablets/100mL cOmplete<SUP>TM</SUP> protease inhibitor cocktail tablets, 2 mM DTT pH 7.4) was added to the frozen samples. Hereafter followed grinding of the samples with a glass rod. Additionally, the samples were sonicated for 10 seconds at 50W. After addition of 250 µL of 24 mM MgCl2, the samples were incubated for 10 minutes on ice. Afterwards, the samples were centrifuged for 10 minutes at 6000xg at 4⁰C. The supernatant was further centrifuged for 35 minutes at 21130xg at 4⁰C. The supernatant was discarded and the pellet was resolved in 30 µL solution buffer (200 mM mannitol, 1 mM HEPES, 1 mM Tris, 1 mM DTT pH 7.4). The samples, now containing brush border membrane vesicles, were stored at -80⁰C until further use.</P><FIGURE><VIDEO width="720" height="400" controls="" style="display:block;margin-left:auto;margin-right:auto;"><SOURCE src=" https://static.igem.org/mediawiki/2016/b/b0/T--Wageningen_UR--mealworms.ogg " type="video/mp4"><P>Your browser does not support this video. Please upgrade your browser.</P></SOURCE><FIGCAPTION>This video shows how the guts from <I>T. molitor</I> molitor were obtained.</FIGCAPTION></VIDEO><H2 id="assay2"><B>Incorporation of 6-Carboxyfluorescein</B></H2><P>To 30 µL BBMVs, made as described in the protocol for preparation of BBMV, 30 µL of 100 mM 6-carboxyfluorescein and 140 µL of solution buffer (200 mM mannitol, 1 mM HEPES, 1 mM Tris, 1 mM DTT pH 7.4) was added. The mixture was sonicated 3 times for 30 seconds at 50 W. To prevent overheating of the samples, the samples were kept on ice as much as possible. Hereafter, 250 µL of 24 mM MgCl2 was added and the samples were incubated for 10 minutes on ice. Afterwards, the samples were centrifuged for 35 minutes at 21130xg at 4⁰C. The supernatant was discarded and the pellet was dissolved in 400 µL solution buffer. Again, the samples were centrifuged for 35 minutes at 21130xg at 4⁰C. Repeating this resolving and centrifugation step was done 1 or 2 more times to remove more 6-carboxyfluorescein free in solution. Finally the pellet was dissolved in 200 µL. The samples, now containing BBMVs incorporated with 6-carboxyfluorescein, were stored at -20⁰C until use.</P><H2 id="assay3"><B>Transmission Electron Microscopy</B></H2><P>For making the images of BBMVs, HAAFD TEM (high angle annular dark field imaging transmission electron microscopy) was used. The following protocol for negative staining was followed before placing the sample in the microscope:

1.       Take plasma cleaned 400 mesh carbon grid.2.       Put 3 µl of sample on the carbon grid and incubate 1 min.3.       Absorb sample with a filter.4.       Wash the sample 3x with MQ, absorbing the MQ every time with a filter.5.       Add 3µl 3% of uranyl acetate and incubate for 30 seconds.6.       Remove stain with filter.</P><H2 id="assay4"><B>Performing Dynamic Light Scattering on BBMVs</B></H2><P>
Before analysing the BBMVs samples with Dynamic Light Scattering (DLS), 10 µL of sample was diluted to a final volume of 1 mL. This was placed in a glass tube. The glass tube was thereafter placed in the DLS machine. Laser light of 532 nm was used to analyse the sample and the scattering pattern was measured under an angle of 90⁰C. The lit of the laser was only opened after placing the sample in the holder and the gain was adjusted before starting of the measurement. Every <I>T. molitor</I> BBMVs sample was measured 5 times for 20 seconds and every sample <I>V. destructor</I> BBMVs sample was measured 20 times for 20 seconds to gain more reliable data. Afterwards the data was analyzed with the program AfterALV.</P><H2 id="assay5"><B>Extraction of Cry3Aa from B.thuringiensis</B></H2><P><I>B. thuriengensis</I> was grown for three days at 37⁰C in 5mL LB medium with sporulation salts (0.14 mM CaCl<SUB>2</SUB>, 0.20 mM MgCl<SUB>2</SUB>, 0.01 mM MnCl<SUB>2</SUB>). The proteins from the cell culture were extracted as written in the general protocol for protein extraction. However not the protein extract itself, but the pellet was further used. The pellet was dissolved in 6 mL carbonate buffer pH. The suspension was incubated for 2 hours at 37⁰C while shaking. The suspension was spun down. The Cry protein was expected to be in the supernatant.
</P><H2 id="assay6"><B>Carboxyfluorescein (CF) Leakage Experiments</B></H2><H3>Protocol for CF leakage measurement with 1% (v/v) SDS (Short)</H3><P>200 µL of solution buffer (200 mM mannitol, 2mM HEPES, 2mM Tris, pH 7.4) with 0.08 % (V/V) BBMVs (from <I>T. molitor</I>, as prepared in the protocol for incorporation of 6-carboxyfluorescein into brush border membrane vesicles) or 0.2 % (V/V) BBMVs (from <I>V. destructor</I>, as prepared in the protocol for incorporation of 6-carboxyfluorescein into brush border membrane vesicles) was put into a well of a 96-well cell culture black microplate. Into another well 200 µL of a solution of 6-carboxyfluorescein in solution buffer was brought. The plate was read in the plate reader. The program with the following settings was started: Duration: 5 minutes. Interval between measurements: 2 seconds. Excitation wavelength: 489 nm. Emission wavelength: 523 nm. Slit: 9.0 nm. gain: 50. After the measurements to both wells 1% (v/v) SDS was added and mixed by pipetting up and down. The plate was read again with the same settings.</P><H3>Protocol for CF leakage measurement with 1% (v/v) SDS (Overnight)</H3><P>200 µL of solution buffer (200 mM mannitol, 2mM HEPES, 2mM Tris, pH 7.4) with 0.08 % (V/V) BBMVs (from <I>T. molitor</I>, as prepared in the protocol for incorporation of 6-carboxyfluorescein into brush border membrane vesicles) or 0.2 % (V/V) BBMVs (from <I>V. destructor</I>, as prepared in the protocol for incorporation of 6-carboxyfluorescein into BBMVs) was put into a well of a 96-well cell culture black microplate. This was done for two wells. To one of the wells 1% (v/v) SDS was added and mixed by pipetting up and down. The plate was read in the plate reader. The program with the following settings was started: Duration: 480 minutes. Interval between measurements: 6 minutes. Excitation wavelength: 489 nm. Emission wavelength: 523 nm. Slit: 9.0 nm. gain: 50.</P><H3>Protocol for CF leakage measurement with Cry3Aa obtained from <I>B.thuringiensis</I></H3><P>10 µL of sample prepared as described in the protocol for Cry3Aa extraction from <I>B.thuringiensis</I> was put into a well of a 96-well cell culture black microplate. 10 µL of pH 10 carbonate buffer was placed into another well. To a measurement buffer (200 mM mannitol, 2mM HEPES, 2mM Tris, pH 7.4) 0.08 % (V/V) BBMVs (from <I>T. molitor</I>, as prepared in the protocol for incorporation of 6-carboxyfluorescein into BBMVs) was added. From this solution, 190 µL was added to the 10 µL samples present in the wells. Directly thereafter the plate was placed in a microplate reader. The program with the following settings was started: Shaking: 10 seconds. Duration: 30 minutes. Interval between measurements: 10 seconds. Excitation wavelength: 489 nm. Emission wavelength: 523 nm. Slit: 9.0 nm. gain: 50.</P><H3>Protocol for CF leakage measurement with Cry3Aa obtained from <I>E.coli</I></H3><P>25 µL of sample, prepared as described in the protocol for expression of Cry3Aa-HIS by <I>E.coli</I> BL21 pSB1A3_Cry3Aa_TEV_HIS, was put into a well of a 96-well cell culture black microplate. As a negative control 25 µL of sample obtained in the same way as the previous mentioned sample, however from an <I>E.coli</I> BL21 without additional construct, was placed into another well. In order to compare measurements taken on different days, to another well 25  µL of Bper was added. To a measurement buffer (200 mM mannitol, 2mM HEPES, 2mM Tris, pH 7.4) 0.09 % (V/V) BBMVs (from <I>T. molitor</I>, as prepared in the protocol for incorporation of 6-carboxyfluorescein into brush border membrane vesicles) or 0.3 % (V/V) BBMVs (from <I>V. destructor</I>, as prepared in the protocol for incorporation of 6-carboxyfluorescein into brush border membrane vesicles) was added. From this solution, 175 µL was added to the 10 µL sample present in the wells. Directly thereafter the plate was placed in a microplate reader. The program with the following settings was started: Shaking: 10 seconds. Duration: 5 to 6 hours. Interval between measurements: 30 seconds. Excitation wavelength: 489 nm. Emission wavelength: 523 nm. Slit: 9.0 nm. gain: 50.</P></FIGURE><SECTION id="isolates"><H1><B><I>Varroa</I> Isolates</B></H1><H2 id="isolates1"><B>Identifying Isolates from <I>Varroa destructor</I></B></H2><P>
One of the methods we used to find our own <I>Bacillus thuringiensis</I> isolate was based on a combination of the procedure used by Rampersad <I>et al.</I><SUP><A href="#fi1" id="reffi1">4</A></SUP>. and Alquisira-Ramírez <I>et al.</I><SUP><A href="#fi2" id="reffi2">5</A></SUP>. It combines the genus’s ability to form spores with a Coomassie stain that allows for visualization of Cry toxins and protein-rich parasporal bodies. This protocol was used for these <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates1">results</A>.</P><P>The most important aspect of this protocol is a source of fresh <I>Varroa</I> mites. The longer they have been dead, the more dessicated they will be. This will complicate the sterilisation step. The easiest way to handle the dead mites is with a fine, wetted brush. This will prevent damage to the mites.</P><P>To promote sporulation of the isolates, sporulation plates were prepared. These are regular LB agar plates with the following salts added:
<UL><LI>0.14 mM CaCl2</LI><LI> 0.20 mM MgCl2</LI><LI> 0.01 mM MnCl2</LI></UL></P><P><B> Instructions</B><UL><LI>Transfer 10 mites to a microcentrifuge tube</LI><LI> Sterilize with 1 mL of a 2.5% bleach solution – only immerse for 5 seconds </LI><LI>Transfer the mites to a sterile microcentrifuge tube</LI><LI> Wash with 1 mL of sterile tap water </LI><LI>Replace the water with 500 µL LB </LI><LI>Macerate the mites in the LB with a pipette tip or a glass rod</LI><LI>Heat the tubes in a heatblock for 15 minutes at 65°C</LI><LI>Take 200 µL of LB and plate on sporulation plates</LI><LI>Spin down the remaining LB, pour off and resuspend – plate the resuspension</LI></UL></P><P>The plates should be incubated at 30°C for at least 2 days, preferably longer. Any appearing colonies with <I>Bacillus cereus</I>-like colony morphology (round, white colonies) should be streaked on sporulation plates and grown for at least 2 days. </P><P>Not all white, round colonies with the ability to form spores will have rod-shaped cells. The following stain stains vegetative cells, spore cell walls and protein-rich bodies such as Cry toxins and parasporal inclusions. 
<UL><LI>50% methanol</LI><LI>40% demineralized water</LI><LI>10% acetic acid</LI><LI>0.1g Coomassie Brilliant Blue R per 10 mL staining solution</LI></UL></P><P>
The staining solution and the destaining solution are the same except for the addition of Coomassie Brilliant Blue R. </P><P>To stain the isolates, the following instructions should be followed. Remember to work in a fumehood as the staining and destaining solutions are both toxic. 
<UL><LI>Scrape a small amount of colony with an inoculation loop</LI><LI>Mix colony with a drop of water on a microscopy slide</LI><LI>Let the slides air dry, then heat fix by passing over a flame 3 times</LI><LI>Immerse the slides in staining solution for 45 seconds</LI><LI>Immerse the slides in destaining solution for 60 seconds, agitate them for the first 10 seconds</LI><LI>Pour off the solution; let air dry</LI><LI>If the slides are too wet, gently place them upside down on tissue paper to dry </LI><LI>The slides are now ready for brightfield microscopy!</LI></UL></P><P>When imaging the colonies, look for colonies with large, rod-shaped cells. If you look carefully, you may be able to see parasporal inclusion bodies or Cry toxins. It is a good idea to get a control strain that produces Cry toxins to have a clear idea of what you are looking for. 
</P><H2 id="isolates2"><B><I>In Vivo</I> toxicity</B></H2><P>
The toxicity assay to determine <I>in vivo</I> toxicity was based on the assay performed by Alquisira-Ramírez <I>et al</I><SUP><A href="#fi2" id="reffi2">5</A></SUP>. Protein extracts were diluted to 100 µg/mL in a 0.1% Tween-80 Tris-HCL buffer (pH 7.5). Per sample, 5 mites were dipped in the protein solution for 5 seconds, then sieved with sterile filter paper. They were then put on an <I>Apis mellifera</I> pupa and mortality was observed every 24 hours. As a negative control, Tween-80 buffer without protein extract was used as well as <I>Bacillus subtilis</I> protein extracts. As a positive control, <I>T. molitor</I> larvae were used in combination with protein extracts from <I>B. thuringiensis tenebrionis</I>. </P><H2 id="isolates3"><B>16s rRNA PCR</B></H2><P>16s rRNA PCR can be used to identify bacterial isolates. The 16s rRNA region consists of highly conserved as well as variable regions; primers anneal to the conserved regions, so the variable regions can be sequenced and used to identify an isolate to the genus level. </P><P>The following primers were used to amplify the 16s rRNA region:
<UL><LI><B>27f</B>: 5’ AGRGTTTGATCMTGGCTAG </LI><LI><B>1492r</B>: 5’ TACGGYTACCTTGTTAYGACTT </LI></UL>
OneTaq was used for the PCR reaction. 
<TABLE><TBODY><TR><TH>Ingredient</TH><TH>Volume (µL)</TH></TR><TR><TD>dH<SUB>2</SUB>O </TD><TD>36.75</TD></TR><TR><TD>5x OneTaq buffer</TD><TD>10</TD></TR><TR><TD>dNTP mix (10 mM)</TD><TD>1</TD></TR><TR><TD>27f primer (10 µM)</TD><TD>1</TD></TR><TR><TD>1492r primer (10 µM)</TD><TD>1</TD></TR><TR><TD>OneTaq polymerase</TD><TD>0.25</TD></TR></TBODY></TABLE></P><P>A sterile pipette tip was used to transfer some cells from the single colonies to the PCR tubes. This was done in a UV cabinet with UV-sterilized gloves and tools to prevent contamination. Water was used as a negative control, <I>Escherichia coli</I> as a positive control. The following program was used with 35 cycles: 
<TABLE><TBODY><TR><TH>Step</TH><TH>Temperature in °C</TH><TH>Time</TH></TR><TR><TD>Predenaturation</TD><TD>95</TD><TD>10 minutes</TD></TR><TR><TD>Denaturation</TD><TD>95</TD><TD>40 seconds</TD></TR><TR><TD>Annealing</TD><TD>55</TD><TD>40 seconds</TD></TR><TR><TD>Extension</TD><TD>68</TD><TD>1 minute</TD></TR><TR><TD>Final Extension</TD><TD>68</TD><TD>5 minutes</TD></TR></TBODY></TABLE></P><P>The PCR products were run on a 1% TAE gel for 30 minutes at 100V to verify the length. After PCR clean-up, samples were sent to GATC for LightRun sequencing.</P><H2 id="isolates4"><B>LC-MS/MS</B></H2><P>
LC-MS/MS was performed on bands containing a 100 kDa protein. For the LC-MS/MS, the following protocol was followed<SUP><A href="#ms1" id="refms1">6</A></SUP>. 
<UL><LI>Prepare an SDS-PAGE gel with the protein of interest</LI><LI>When handling the columns and peptides, use wetted nitrile gloves to prevent keratin contamination</LI><LI>Cut out bands at the height of the protein, as well as some distance above it</LI><LI>Treat the gel bands with 10 mM DTT at pH 8 for 1 hour at 60°C</LI><LI>Treat with 20 mM iodoacetamide at pH8 for 1 hour at room temperature</LI><LI>Cut bands into 1 mm<SUP>2</SUP> pieces</LI><LI>Digest for 2 hours at 45°C with 5 ng/µL trypsin solution</LI><LI>Extract peptides by elution in a 10% trifluoro-acetic acid solution</LI><LI>Clean peptides with a C18 µColumn</LI><UL><LI>Prepare the column as follows:</LI><LI>Cut 1.6 mM disk from a C18 Empore disk</LI><LI>Transfer the disk to a 200 µL pipette tip</LI><LI>Pipet 200 µL methanol on the disk</LI><LI>Pipet a 50% slurry of Lichroprep C18 column material into the methanol</LI><LI>Wash once with 100 µL methanol </LI><LI>Equilibrate with 100 µL 1 mL/L formic acid in water </LI><LI>Prevent the column from running dry or being obstructed by air bubbles</LI></UL><LI>Dissolve samples in 100 µL 1 mL/L formic acid and run through a column</LI><LI>Wash the columns once more with formic acid</LI><LI>Elute samples in 50 µL 50% acetonitrile and 50% 1 mL/L formic acid into a low-binding microcentrifuge tube</LI><LI>Concentrate samples with a vacuum concentrator to remove acetonitrile</LI><LI>Your samples are ready for LC-MS/MS!</LI></UL></P></SECTION><SECTION id="cas"><H1><B>Cas9 kill switch</B></H1><H2 id="cas1"><B>Mutagenesis PCR</B></H2><P>
To make mutations in dCas9, mutagenesis PCR was used according to the <A href="http://www.agilent.com/cs/library/usermanuals/Public/200518.pdf">Quikchange protocol</A>. The adapted protocol that worked best for us is shown here. 

Primers with mismatches to the template on the site of the mutation were designed using their <A href="http://www.genomics.agilent.com/primerDesignProgram.jsp">online tool</A>. 
The following PCR mixture was made: 
</P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Amount</TH></TR><TR><TD>5x Q5 buffer</TD><TD>5 µL</TD></TR><TR><TD>dNTP mix (10 mM)</TD><TD>0.5 µL</TD></TR><TR><TD>Q5 polymerase</TD><TD>0.25 µL</TD></TR><TR><TD>fwd primer</TD><TD>100 ng</TD></TR><TR><TD>rev primer</TD><TD>100 ng</TD></TR><TR><TD>template</TD><TD>10-100 ng</TD></TR><TR><TD>dH<SUB>2</SUB>O </TD><TD>to 25 µL</TD></TR></TBODY></TABLE><P>
Most of the time, 100 ng template worked best for us. Additionally, 5 µL of 5x GC enhancer was needed to run the Ala10 → TAG PCR reaction successfully.

Then, the following cycling program was used to amplify the DNA:
</P><TABLE><TBODY><TR><TH>Step</TH><TH>Temperature in °C</TH><TH>Time</TH></TR><TR><TD>1</TD><TD>98</TD><TD>30 seconds</TD></TR><TR><TD>2</TD><TD>98</TD><TD>30 seconds</TD></TR><TR><TD>3</TD><TD>55</TD><TD>1 minute</TD></TR><TR><TD>4</TD><TD>72</TD><TD>9 minutes</TD></TR><TR><TD>5</TD><TD>4</TD><TD>∞</TD></TR></TBODY></TABLE><P>
Step 2-4 were repeated 30x.

PCR products were checked by gel electrophoresis. If amplification was successful, PCR products were cleaned using the Zymo PCR cleanup kit according to the accompanying protocol and eluted in water. The cleaned samples were digested for I.5 hours with DpnI, cleaned again, and 5 µL of the eluted sample (in water again) was used to transform electrocompetent cells. Resulting colonies were <A href="https://static.igem.org/mediawiki/2016/b/bd/T--Wageningen_UR--MiniPrep.pdf">miniprepped</A>
 and verified by sequencing. 
</P><H2 id="cas2"><B>Yeast assembly</B></H2><P>
To construct pEVOL-Asp, a construct containing an aminoacyl synthetase (aaRS) and tRNA<SUB>CUA</SUB> (ref) suitable for incorporating Aspartate in response to the TAG stopcodon, yeast assembly was used. In this method, linear DNA fragments are transformed into <I>Saccharomyces cerevisiae</I>. To select for positive clones, one of the fragments is a part of the yeast expression vector pYES2. pYES2 contains the <I>URA3</I> gene, which releases auxotrophy for uracil, as well as a 2µ <I>ori</I> for high copy maintenance in yeast. The other fragments included two IDT gBlocks containing the aaRS and tRNA<SUB>CUA</SUB>, as well as part of pEVOL-pAzF<SUP><A href="#bp7" id="refbp7">7</A></SUP> containing the arabinose promoter and CAT resistance gene.

Primers to generate fragments for yeast assembly were made using the <AHREF>NEBuilder for Gibson assembly, with 40 bp overlap between fragments. PCR was performed as usual (<A href="https://static.igem.org/mediawiki/2016/2/27/T--Wageningen_UR--Polymerase_Chain_Reaction.pdf">see the protocol</A>), fragments were checked by gel electrophoresis and PCR products were purified using the PCR cleanup kit from Zymo. 

Homemade heat-shock competent <I>S. cerevisiae</I> cells were incubated at RT for 10 minutes prior to mixing with ~400 ng of each PCR product and 100 µg sheared salmon sperm DNA (Sigma Aldrich). 700 µL of 1x LiAc/ 40% PEG-3350/ 1x TE was mixed in and solutions were incubated at 30°C for 30 minutes. Next, 88 µL DMSO was added, samples were mixed and heat shocked at 42°C for 7 minutes. Cells were pelleted in a microcentrifuge, washed with 1 ml 1x TE, resuspended in 100 µL 1x TE and plated on YPD plates. Plates were incubated over the weekend at 30°C.

To isolate the DNA from possible correct clones, a modified miniprep protocol was followed. 

Colonies were inoculated in 1.5 ml YPD medium and grown overnight while shaking at 30°C. Cells were harvested in a microcentrifuge (13000 rpm, 5 minutes) and the pellet was resuspended in 100 µL STET buffer. To lyse the cells, they were disrupted in a bead beater, another 100 µL STET was added and samples were boiled for 3 minutes in a heat block. Samples were spinned down again in a microcentrifuge, and the supernatants were used to proceed with miniprep as usual (hyperlink). This gave fairly low plasmid yields (~30 ng/ µL), but enough to do PCR and transform into <I>E. coli</I>. 

Correct construction of the plasmid was confirmed by PCR and sequencing. 
<B>YNB-UDM medium/plates</B></AHREF></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Concentration</TH></TR><TR><TD>YNB (Sigma Aldrich)</TD><TD>6.7 g/L</TD></TR><TR><TD>UDM (Yeast Synthetic Drop-out Medium Supplements w/o Uracil; Sigma Aldrich)</TD><TD>1.92 g/L</TD></TR><TR><TD>Glucose</TD><TD>20 g/L</TD></TR><TR><TD>Agar</TD><TD>20 g/L (for plates)</TD></TR></TBODY></TABLE><P><B>10x TE</B></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Concentration</TH></TR><TR><TD>Tris</TD><TD>10 mM</TD></TR><TR><TD>EDTA</TD><TD>1 mM</TD></TR></TBODY></TABLE><P>
pH was adjusted to 7.5.
<B>10x LiAc</B></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Concentration</TH></TR><TR><TD>Lithium Acetate</TD><TD>1 M</TD></TR></TBODY></TABLE><P>
pH was adjusted to 7.5 and filter sterilized. 
<B>STET buffer</B></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Concentration</TH></TR><TR><TD>sucrose</TD><TD>8%</TD></TR><TR><TD>Tris pH 8</TD><TD>50 mM</TD></TR><TR><TD>EDTA</TD><TD>50 mM</TD></TR><TR><TD>Triton X-100</TD><TD>5%</TD></TR></TBODY></TABLE><H2 id="cas3"><B>Cloning of gRNA plasmids using annealed oligos</B></H2><P>
Cloning was performed in the same way as normal <A href="https://static.igem.org/mediawiki/2016/a/ac/T--Wageningen_UR--Restriction_Enzyme_Digestion.pdf">digestion</A>-<A href="https://static.igem.org/mediawiki/2016/7/76/T--Wageningen_UR--Ligation.pdf">ligation</A> cloning, but annealed oligonucleotides were used as insert. 

Oligos were annealed as in Jao <I>et al</I><SUP><A href="#bp8" id="refbp8">8</A></SUP>, but using 1x ligase buffer instead of NEBuffer 3.

Two complementary oligos corresponding to the target sequence were diluted to 10 µM each in T4 ligase buffer (NEB) to a final volume of 20 µL. To anneal them, the following protocol was used: 

5 min denaturation at 95ºC, cooling to 50ºC at -0.1ºC per second, remaining at 50ºC for 10 minutes, followed by cooling to 4ºC at -1ºC per second. 

The annealed oligos were cloned into pT7-gRNA (Addgene plasmid # 46759). pT7-gRNA was digested with BsmBI and BglII for 1 hour. Cut plasmids were checked by gel electrophoresis, and digested bands were purified from the gel (hyperlink). 

Next, we proceeded with  <A href="https://static.igem.org/mediawiki/2016/7/76/T--Wageningen_UR--Ligation.pdf">ligation</A> as usual, diluting the annealed oligos 16x to add 1 µL to 25 ng digested pT7-gRNA. Ligation mixtures were diluted 2x, and 1 µL was used for <A href="https://static.igem.org/mediawiki/2016/0/00/T--Wageningen_UR--Preparing_and_transforming_DH5%CE%B1_electrocompetent_cells.pdf">electroporation</A>.
</P><H2 id="cas4"><B><I>in vitro</I> RNA transcription and purification</B></H2><P><B>Note: always use gloves and filtered tips when working with RNA, as RNAses can contaminate your samples</B><B><I>in vitro</I> transcription</B>
Prior to transcriptions, pT7-gRNA plasmids were digested with BamHI and purified. RNA guides were transcribed <I>in vitro</I> using the HiScribe high yield RNA synthesis kit from NEB. Reactions were assembled using the following protocol:
</P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Amount</TH></TR><TR><TD>10x reaction buffer</TD><TD>1.5 µL</TD></TR><TR><TD>NTP</TD><TD>1.5 µL each</TD></TR><TR><TD>Digested template DNA</TD><TD>1 µg</TD></TR><TR><TD>T7 RNA polymerase mix</TD><TD>1.5 µL</TD></TR><TR><TD>dH<SUB>2</SUB>O </TD><TD>to 20 µL</TD></TR></TBODY></TABLE><P>
Mixtures were incubated at 37°C for 4 hours and digested with 2 µL DNAseI for 15 minutes at 37°C.
<B>Checking transcription products on urea gel</B><I>In vitro</I> 5 µL of the transcription products, as well as 1 µL low range ssRNA ladder, were mixed with 2x RNA loading dye, incubated at 90°C for 5 minutes and put on ice immediately thereafter. 

1.5mm urea gels were poured and left to polymerize for &gt;30 minutes. Gels were assembled and prerun for 15 minutes (15 mA for 1 gel, 35 mA for two gels). As a running buffer, 0.5x TBE was used. Before loading the transcription products, wells were flushed with running buffer to remove urea from the wells. 

Samples were loaded and run for ~1 hour. 

To visualize the RNA on the gels, they were stained with SybrGold (Thermo Fischer Scientific) in 100 mL 0.5x TBE for ~20 minutes. After staining, bands on the gel can be visualized with UV light. The RNA transcripts might appear as big, uncolored blobs; in that case, there is so much RNA that the dye does not penetrate the gel completely. 

If gels showed RNA bands of the right size, the remainder of the transcription products was loaded on the gel for subsequent purification. 
<B>RNA purification from acrylamide gel</B>
RNA bands of the correct size were cut from the gel, cut in small pieces with a syringe and incubated overnight in 1 ml RNA elution buffer. 

The next day, the elution buffer was divided over 3 eppies and 990 µL of pre-chilled (-20°C) 100% EtOH was added. Samples were incubated for 1 hour at -20°C, spun down in a microcentrifuge for 20 minutes, 13000 rpm, at RT. At this point, a pellet can be seen that is usually translucent, but in our case whitish and flakey. Pellets were washed with 1 ml 70% EtOH (combining them again), spun down again for 5 minutes, 13000. The majority of EtOH was removed, and the remainder was evaporated at 55°C in a heatblock. Pellets were resuspended in 20 µL MQ, after which purity and concentration was measured with Nanodrop. 
<B>5x TBE buffer</B></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>amount</TH></TR><TR><TD>Tris base</TD><TD>54 g</TD></TR><TR><TD>Boric acid</TD><TD>27.5 mL</TD></TR><TR><TD>EDTA (0.5 M, pH8)</TD><TD>20 mL</TD></TR><TR><TD>MQ</TD><TD>to 1 L</TD></TR></TBODY></TABLE><P>
pH should be ~8.3. 
<B>7M urea 10% PAGE gel (for two pieces, 1.5 mm)</B></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Amount</TH></TR><TR><TD>Urea</TD><TD>10.5 g</TD></TR><TR><TD>MQ</TD><TD>6.25 mL</TD></TR><TR><TD>40% acrylamide</TD><TD>6.25 mL</TD></TR><TR><TD>5x TBE</TD><TD>5 mL</TD></TR></TBODY></TABLE><P>
This solution was filter sterilized, 0.45 µm. 0.125 mL 10% APS and 0.025 mL TEMED were added subsequently, after which polymerization starts.
<B>2X RNA loading dye</B></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Amount (mL)</TH></TR><TR><TD>Formamide</TD><TD>9.5</TD></TR><TR><TD>EDTA (200mM)</TD><TD>0.25</TD></TR><TR><TD>10% SDS</TD><TD>0.025</TD></TR><TR><TD>2.5% BPB</TD><TD>0.1</TD></TR><TR><TD>2.5% CFF</TD><TD>0.1</TD></TR></TBODY></TABLE><P><B>RNA elution buffer</B></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Concentration</TH></TR><TR><TD>NaCl</TD><TD>0.5 M</TD></TR><TR><TD>MgCl<SUB>2</SUB></TD><TD>10 mM</TD></TR><TR><TD>EDTA</TD><TD>1 mM</TD></TR><TR><TD>SDS</TD><TD>0.1%</TD></TR></TBODY></TABLE><P>
pH was adjusted to ~7. This needs to be done very carefully by the drop. If a precipitate is visible, incubate at 37°C before measuring pH. 
</P><H2 id="cas5"><B>Cas9 expression and FPLC purification</B></H2><P><I>Day 1</I>
Cas9 was expressed in a recoded <I>E. coli</I> strain lacking TAG stopcodons and release factor 1(C321dA; ref), but higher expression can probably be achieved using a bacterial strain more suited to protein expression. 

10 ml overnight cultures were prepared of C321dA transformed with Cas9-expresso, dCas9-expresso, Ala10TAG-expresso + pEVOL, or Ala840TAG-expresso + pEVOL. Cultures were grown in LB in the presence of 35 µg/mL kanamycin (expresso constructs) and 25 µg/mL chloramphenicol (pEVOL constructs) when appropriate. 
<I>Day 2</I>
Overnight cultures were diluted 100x in 0.5 L LB, with appropriate antibiotics, and divided over two 1L erlenmeyers. Cultures were were grown to an OD600 of about 0.8 at 37ºC while shaking. Next, cultures were put at 20ºC shaking for 30 minutes (cold shock) to induce expression of chaperones that aid in protein stability. 

Cas9 expression was induced by adding rhamnose to 0.2%. In cultures that contained a pEVOL vector, expression of the aminoacyl synthetase was induced by adding arabinose to 0.2%. When appropriate, at this time also the synthetic amino acid was added to a concentration ranging from 200 µM to 1 mM. Cultures were shaken overnight at 20ºC.
<I>Day 3</I>
All centrifugation steps were performed at 4 ºC, and samples were kept on ice. 

Cells were harvested by centrifugation at 4000 x g for 15 minutes. Supernatant was removed, and cells were washed once in 50 ml Tris-HCL buffer, pH7. After spinning down the cells again, they were resuspended in 10 ml cold His buffer A. Cells were lysed using a chilled French press cell (pressure ~30,000 psi) and lysed samples were spinned at 16000 x g for 10 minutes. Supernatants were collected, filtered (0.45 µm filter) and purified using FPLC.

Since C321dA is not optimized for protein expression, we aimed to reduce protein breakdown by adding <A href="https://lifescience.roche.com/webapp/wcs/stores/servlet/ProductDisplay?partNumber=3.2.7.1.40.1">cOmplete protease inhibitor</A> to our samples before lysis. However, high yield of Cas9 was also observed when this protease inhibitor was omitted. 

Using the expresso vector for Cas9 expression introduced a C-terminal his-tag to our Cas9 constructs, which was used for protein purification. An AKTA Purifier system was used, with a 1 mL HisTrap HP column (GE Healthcare Life Sciences) that was equilibrated with His buffer A . The cell-free extracts were loaded on the column, and the column was washed by addition of 2% His buffer B to already remove some unspecifically bound proteins. Elution was performed with a gradient of His buffer B. 
<B>His buffer A</B></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Concentration</TH></TR><TR><TD>KPi</TD><TD>50 mM</TD></TR><TR><TD>NaCl</TD><TD>300 mM</TD></TR></TBODY></TABLE><P>
pH was adjusted to 8.0.
<B>His buffer B</B></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Concentration</TH></TR><TR><TD>KPi</TD><TD>50 mM</TD></TR><TR><TD>NaCl</TD><TD>300 mM</TD></TR><TR><TD>Imidazole</TD><TD>500 mM</TD></TR></TBODY></TABLE><P>
pH was adjusted to 8.0.
</P><H2 id="cas6"><B><I>in vitro</I> Cas9 cleavage assay</B></H2><P>
Cleavage assays were performed according to the <A href="https://www.neb.com/protocols/2014/05/01/in-vitro-digestion-of-dna-with-cas9-nuclease-s-pyogenes-m0386">NEB protocol</A>, but with our own Cas9 samples and homemade 5x Cas9 assay buffer. 

The following reaction mixtures were assembled:
</P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Amount</TH></TR><TR><TD>5x Cas9 assay buffer</TD><TD>6 µL</TD></TR><TR><TD><I>Streptococcus pyogenes</I> Cas9 (1µM)</TD><TD>1 µL</TD></TR><TR><TD>gRNA</TD><TD>~90 ng/guide</TD></TR><TR><TD>dH<SUB>2</SUB>O , nuclease free</TD><TD>to 28 µL</TD></TR></TBODY></TABLE><P>
This mixture was allowed to incubate for 10 minutes at 25°C, followed by addition of 3nM (final concentration) substrate DNA (in our case, 200 ng of a 4100 bp PCR product). The, it was incubated for 1-1.5 hours at 37°C. 

Cleavage was assessed by gel electrophoresis on agarose gels, but stained after running with SybrGold (Thermo Fisher Scientific) for ~25 minutes instead of SybrSafe. 
<B>5x Cas9 assay buffer</B></P><TABLE><TBODY><TR><TH>Ingredient</TH><TH>Concentration</TH></TR><TR><TD>HEPES</TD><TD>100 mM</TD></TR><TR><TD>NaCl</TD><TD>500 mM</TD></TR><TR><TD>MgCl<SUB>2</SUB></TD><TD>25 mM</TD></TR><TR><TD>EDTA</TD><TD>0.1 mM</TD></TR></TBODY></TABLE><P>
pH was adjusted to 6.5.
</P></SECTION><SECTION id="ecoli"><H1><I>Escherichia coli</I> survival</H1><P>The bacteria grew overnight in LB medium, shaking at 37° C. Then a small amount of this is pipetted into sucrose solutions of varying concentrations. The solutions were then sampled after various timesteps, inoculated into 0.2 mL sterile LB medium and left to grow overnight. </P><P>Materials used: <UL><LI>Pure sucrose</LI><LI>Tap water</LI><LI>(Multi-)Pipettes</LI><LI><I>E. coli K12 MG1655 </I> strain, upon which the harvard strain and the iJO1366 model are based.</LI><LI>2x 96 wells plate (well volume ~200 μL)</LI><LI>Plate-reader</LI></UL></P><P>protocols applied:</P><P><UL><LI>On the day before, inoculate a bottle of LB with <I>E. coli</I>, then let that solution grow overnight at optimal conditions (12 hours). This is our saturated LB solution.</LI><LI>Under sterile conditions:</LI><LI>For experiment 1, fill 3 wells from a 96-well plate with 200μL of 0 g/L sucrose solution in sterilized tap water. Do the same for a 100, 200, and 400 g/L sucrose solution. </LI><LI>For experiment 2, fill 4 wells from a 96-well plate with 200μL of 312.5 g/L sucrose solution in sterilized tap water. Do th same for a 625 g/L sucrose solution.</LI><LI>For experiment 3, fill 4 wells from a 96-well plate with 200μL of 312.5 g/L sucrose solution in sterilized tap water. Do th same for a 625 g/L sucrose solution.</LI><LI>Put 200μL LB in each well of an new 96-well plate. Close and keep sterile.</LI><LI>For experiment 1, innoculate the four different sucrose solutions (three replicates) with bacteria.</LI><LI>After time steps of 30, 50, 90, and 120 minutes, inoculate the sterile LB-filled wells of with 5 μL of the earlier inoculated sucrose solutions. The final row was inoculated with sterilized sugar water as negative control.</LI><LI>For experiment 2, innoculate the two different sucrose solutions (four replicates) with bacteria.</LI><LI>After time steps of 1, 2, 3, 4, 5, and 6 hours, inoculate the sterile LB-filled wells of with 5 μL of the earlier inoculated sucrose solutions. The final row was inoculated with sterilized sugar water as negative control.</LI><LI>For experiment 3, innoculate the two different sucrose solutions (four replicates) with bacteria.</LI><LI>After time step of 24 hours, inoculate the sterile LB-filled wells of with 5 μL of the earlier inoculated sucrose solutions. The final row was inoculated with sterilized sugar water as negative control.</LI><LI>For experiment 1, a plate reader was used to compare overnight growth curves in the LB. For the other experiments bacterial growth was estimated by eye.</LI></UL></P><SECTION id="light-exposure"><H1><B>Light sensor response assay</B></H1><P>
The response of optogenetic systems pDawn and pDusk to different light levels was tested in a procedure similar to that used by the systems' original creators, Möglich <I>et al</I>. Plasmids of both systems, expressing fluorescent protein mCherry as a reporter, were transformed seperately into <I>E. coli</I>. The resulting colonies were picked into 5 ml of LB medium in clear plastic Greiner tubes (50 ml). The tubes were then fixed in place in a plate incubator with a clear plastic lid. An LED panel with an emission wavelength of 470 nm was aimed at the incubator, providing an irradiance of 110 µW/cm<SUP>2</SUP>. These are saturating conditions, ensuring the maximum possible response from the promoter systems. The cultures were grown overnight at 37 °C. They were subsequently diluted into new tubes of LB (two for each system, OD<SUB>600</SUB> = 0.004). One tube for each system was covered in a coat of aluminum foil, blocking all incoming light. The new cultures were then placed in the illuminated incubator and grown for 17 hours. The OD<SUB>600</SUB> and fluorescence (excitation: 576 nm, emission: 601 nm) were measured in a 96-well cell culture black microplate.
</P></SECTION><SECTION id="references"><H1>References</H1><OL class="references"><A id="mi1">1.</A> Panziera, D., Van Langevelde,  F., Blacquière, T. (2016). In preparation.<A href="#refmi1" title="Jump back to footnote 1 in the text.">↩</A><A id="mi2" href="http://www.tandfonline.com/doi/abs/10.3896/IBRA.1.52.1.09">2.</A> Dietemann, V., Nazzi, F., Martin, S. J., Anderson, D. L., Locke, B., Delaplane, K. S., ... &amp; Rosenkranz, P. (2013). Standard methods for varroa research. Journal of Apicultural Research, 52(1), 1-54. <A href="#refmi2" title="Jump back to footnote 2 in the text.">↩</A><A id="lm1" href="http://www.nature.com/nm/journal/v7/n11/full/nm1101-1249.html">3.</A> Ricardo J. Giordano, Marina Cardó-Vila, Johanna Lahdenranta, Renata Pasqualini &amp; Wadih Arap (2001). Biopanning and rapid analysis of selective interactive ligands. Nature Medicine.
<A href="#reflm1" title="Jump back to footnote 3 in the text.">↩</A><A id="fi1" href="http://bmcmicrobiol.biomedcentral.com/articles/10.1186/1471-2180-5-52">4.</A> Rampersad, J., &amp; Ammons, D. (2005). A <I>Bacillus thuringiensis</I> isolation method utilizing a novel stain, low selection and high throughput produced atypical results. BMC microbiology, 5(1), 1.
<A href="#reffi1" title="Jump back to footnote 4 in the text.">↩</A><A id="fi2" href="http://link.springer.com/article/10.1007/s13592-014-0288-z">5.</A> Neethu, K. Alquisira-Ramírez, E. V., Paredes-Gonzalez, J. R., Hernández-Velázquez, V. M., Ramírez-Trujillo, J. A., &amp; Peña-Chora, G. (2014). <I>In vitro</I> susceptibility of <I>varroa destructor</I> and <I>Apis mellifera</I> to native strains of <I>Bacillus thuringiensis</I>. Apidologie, 45(6), 707-718.
<A href="#reffi2" title="Jump back to footnote 5 in the text.">↩</A><A id="ms1" href="http://www.wur.nl/upload_mm/8/9/3/a8d3cdec-353b-4936-82b2-c44c0cccc811_Sample%20preparation%20for%20proteomics%20by%20MS_sept2019b.pdf">6.</A> Boeren, S. Sample preparation for proteomics by MS.
<A href="#refms1" title="Jump back to footnote 6 in the text.">↩</A><A id="bp7" href="http://pubs.acs.org/doi/abs/10.1021/ja027007w">7.</A>Chin, J. W., Santoro, S. W., Martin, A. B., King, D. S., Wang, L., &amp; Schultz, P. G. (2002). Addition of p-Azido-l-phenylalanine to the Genetic Code of <I>Escherichia coli</I>. Journal of the American Chemical Society, 124(31), 9026-9027. 
<A href="#refbp7" title="Jump back to footnote 7 in the text.">↩</A><A id="bp8" href="http://www.pnas.org/content/110/34/13904.short">8.</A> Jao, L. E., Wente, S. R., &amp; Chen, W. (2013). Efficient multiplex biallelic zebrafish genome editing using a CRISPR nuclease system. Proceedings of the National Academy of Sciences, 110(34), 13904-13909.
<A href="#refbp8" title="Jump back to footnote 8 in the text.">↩</A></OL></SECTION></SECTION></SECTION></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2016. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV></DIV></BODY></HTML>