Team:Wageningen UR/Notebook/OptogeneticKillSwitch
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
★ Design
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
AWARDS
★ Entrepreneurship
★ Hardware
★ Software
★  Measurement
★ Model
Wageningen UR iGEM 2016
☰
˟
Overview
Description
Results
Achievements
BioBricks
Specificity
Regulation
Biocontainment
Model
Outreach
Team
Notebook
 
 
 
Optogenetic
kill switch
May 16 - June 6: Obtaining Parts
4 July - 1 August: Initial Construction
8 August - 3 October: Further Construction & Testing
References
Overview
Description
Results
Achievements
BioBricks
Applied Design
WetLab
Specificity
In vitro assay
Toxin Engineering
Varroa Isolates
Regulation
Detecting Mites
Population Dynamics
Toggle Switch
E. coli survival
Biocontainment
Light Kill Switch
Cas9 Kill Switch
Collaborations
Model
Population Dynamics
Light Kill Switch
E. coli survival
Beehave
Toxin Scanner
Outreach
Our Vision
Design
Collaboration
Events
Media
Integrated Practices
Team
Members
Students
Supervisors
Gallery
Attributions
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
These experiments were performed primarily by Marijn Uitenbroek, with significant contributions by Tianhe Wang during the final phase. Unless otherwise indicated, protocols were done according to the Protocols page.
May 16 - June 6: Obtaining Parts
Both MazE and MazF have been registered as BioBricks (parts BBa_K1096001 and BBa_K302033 , respectively). However, as of the creation period of the kill switch, neither are available from the iGEM registry. For this reason, we chose to isolate both genes directly from E. coli genomic DNA via PCR amplification. In order to simplify the construction process, the forward primers we synthesized included BBa_B0034 , an RBS, between the iGEM prefix and the coding sequence.
Primers used:
MazE Forward: 5' ACACGAATTCGCGGCCGCTTCTAGaaagaggagaaaATGATCCACAGTAGCGTAAAGC
MazE Reverse: 5' GTGTCTGCAGCGGCCGCTACTAGTATTATTACCAGACTTCCTTATCTTTCG
MazF Forward: 5' ACACGAATTCGCGGCCGCTTCTAGAAAGAGGAGAAAATGGTAAGCCGATACGTACC
MazF Reverse: 5' GTGTCTGCAGCGGCCGCTACTAGTATTATTACCCAATCAGTACGTTAATTTTGG
In addition, the following BioBricks were prepared from iGEM registry kits:
BBa_K592006 : FixK2 (Promoter triggered by YF1-FixJ)
BBa_K592016 : YF1-FixJ (Light sensor)
BBa_B0015 : tt (Double terminator
BBa_J23108 : CP (Constituitive promoter of the Anderson family. Mid-level expression strength, arbitrarily chosen.)
BBa_K516030 : mRFP (Includes RBS and terminators.)
Accounting for the possibility that natively produced MazE could interfere with the kill switch, we also obtained a MazEF-deficient strain (MC4100ΔmazEFrelA+, or ΔmazEF for short) 1 for potential use as a chassis organism. A culture of these cells was made chemically competent.
4 July - 1 August: Initial construction
Our target constructs were put together piece by piece via standard 3A ligation. Transformant colonies were identified through colony PCR, and isolated constructs were confirmed through Sanger sequencing.
CP-YF1-FixJ-tt: Light sensing part, including promoter and terminator.
FixK2-mRFP: Reporter (intended to assess exact response)
FixK2-MazE-tt: Light-deactivated antitoxin
CP-MazF-tt: Constituitively expressed toxin (could not be created)
CP-MazF-tt-FixK-MazE-tt: Toxin-antitoxin cassette (could not be created)
8 August - 3 October: Further Construction & Testing
The greatest obstacle to creating a working kill switch was the fact that it proved impossible to create a part actively expressing MazF; the ligation-transformation process for this part failed consistently. The possibility was considered that the MazF produced under a constitutive promoter was simply too much for even non-deficient cells to handle properly. We therefore created a cloning strain that overproduces MazE. In order to do so, we made use of the ASKA cloning library 2
, a collection of E. coli strains that overexpress specific native genes when induced by IPTG. We grew a culture of the ASKA MazE strain (registered in the library under its synonym, chpR), then isolated the chloramphenicol-resistance plasmid and transformed it into a DH5α cloning strain as per standard protocols. The resulting strain (AE) was made chemically competent. During the final growth phase, the culture was induced by 0.1 mM IPTG. The ligation mixture of CP-MazF was transformed into these competent cells, which were then grown on IPTG-containing plates. However, no colonies containing the desired plasmid were found.
During this period, we also tested the response of our constructed pDusk-RFP by way of the light response protocol detailed on the Protocols page. A complete lack of distinguishable expression from this part prompted us to make use of the original pDawn- and pDusk-mCherry parts as created by Möglich et al. Results indicated that pDawn, rather than pDusk, forms the more suitable light sensor for this kill switch.
References
1. Sat, B., Reches, M., Engelberg-Kulka, H. (2003)  The Escherichia coli mazEF Suicide Module Mediates Thymineless Death. Journal of Bacteriology vol. 185 no. 6, 1803-1807. ↩
2. Kitagawa, M., Ara, T., Arifuzzaman, M,, Ioka-Nakamichi, T., Inamoto, E., Toyonaga, H., Mori, H. (2005) Complete set of ORF clones of Escherichia coli ASKA library (a complete set of E. coli K-12 ORF archive): unique resources for biological research. DNA Research 12(5):291-9. ↩
