<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Description_Regulation skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Description/Regulation</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Wageningen_UR/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><TITLE>Wageningen UR iGEM 2016 </TITLE><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><BUTTON class="hamburger">☰</BUTTON><BUTTON class="cross">˟</BUTTON><DIV class="mobilemenu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><B>Overview</B></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Team</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Notebook</A></LI></UL></DIV><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="#header">Regulation</A></H4></DIV><LI class="menu-item"><A href="#DetectingMites">Mite Detection</A></LI><LI><A href="#mitedevice">Mite Sensing Device</A></LI><LI><A href="#proofconc">Proof of Concept</A></LI><LI class="menu-item"><A href="#QuorumSensing"><FONT size="2">Population Dynamics</FONT></A></LI><LI><A href="#toxinproduction">Improving Toxin Production</A></LI><LI><A href="#quorumsensing">Quorum Sensing</A></LI><LI><A href="#subpop">Subpopulations</A></LI><LI><A href="#conclusions">Conclusions</A></LI><LI class="menu-item"><A href="#Toggleswitch">Toggle Switch</A></LI><LI><A href="#lightregu">Light Regulation of BeeT</A></LI><LI><A href="#hybriddesign">Hybrid Promoter Design</A></LI><LI><A href="#hybridactivation">Hybrid Promoter Activation</A></LI><LI><A href="#conclusions_tog">Conclusions</A></LI><LI class="menu-item"><A href="#references">References</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design">Applied Design</A></LI></UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"><DIV class="ca-content">WetLab</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Assay"><I>In vitro</I> assay</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#ToxinEngineering">Toxin Engineering</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates" class="up_menu_fix"><I>Varroa</I> Isolates</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#DetectingMites" class="up_menu_fix">Detecting Mites</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing" class="up_menu_fix"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#Toggleswitch" class="up_menu_fix">Toggle Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/ecoli_survival"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#Lightswitch" class="up_menu_fix">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#SAA" class="up_menu_fix">Cas9 Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model"><DIV class="ca-content">Model</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#light">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#beehave">Beehave</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">Toxin Scanner</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Vision">Our Vision</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#collaboration">Collaboration</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#events">Events</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"><FONT size="1">Integrated Practices</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/contact">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#environment">Environment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#laboratory">Laboratory</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Journals</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments">Protocols</A></LI></DIV></DIV></DIV><DIV class="center_content"><H1 style="text-align:center;">Regulation</H1><P>
A problem that is encountered when expressing Bt toxins is that overexpression can be lethal to the bacterial chassis, in our case <I>Escherichia coli</I>. We aimed to prevent premature lysis of BeeT and achieve higher toxin yield by separating the growth phase from the toxin-producing phase. Toxin expression in coupled to quorum-sensing: only when a sufficient number of bacteria are present, the toxin is produced. Furthermore, we aimed to create a subpopulation of bacteria that do not produce the toxin, even when the bacterial density is high. When the toxin-producing bacteria perish, the subpopulation survives. As the survivors are genetically identical to the rest of the population, they are able to initiate a new growth phase and subsequently new toxin production. </P><P>
The high toxin expression is needed to induce significant damage to the Varroa population. It is known that longer-term, low-dose exposure of a pesticide to the target organism can cause resistance<SUP><A href="#ri1" id="refri1">1</A></SUP>. Ideally, the toxin is only expressed when the target organism is present. To further optimize toxin expression, two measures to regulate expression were explored. BeeT was designed to sense the presence of <I>Varroa destructor</I> through the use of two riboswitches: one senses guanine, the other senses vitamin B12. Both are normally not present in beehives, but indicate the presence of the mite as guanine is a major component of mite faeces and B12 is present in the haemolymph of the bees. When a mite attaches itself to a bee to feed on the haemolymph, BeeT will be able to sense it. </P><P>
Lastly, we aimed to combine the riboswitch and part of the <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#cas1">light kill switch</A>. A hybrid promoter was designed that ensures continuous toxin production after transient sensing of the mite, but shuts off production when BeeT escapes from the hive and is exposed to light. </P><SECTION id="DetectingMites"><H1><I>Varroa</I> Mite Detection</H1><H2 id="mitedevice">Design of a Mite Sensing Device Based on Riboswitches</H2><P> To prevent toxin expression when the mite is not present, we have created a <A href="http://parts.igem.org/Part:BBa_K1913011">mite sensing device</A> that can regulate toxin expression using riboswitches. Substances that indicate the presence of the mite are guanine, since 95% of the mite faeces consist of guanine<SUP><A href="#pd21" id="refpd21">1</A></SUP> and vitamin B12, since this vitamin is present in the haemolymph of insects. <I>Varroa destructor</I> feeds on the haemolymph of the honey bee and will leave traces of haemolymph in encapsulated larvae cells and through the hive<SUP><A href="#pd21" id="refpd21">1</A></SUP>. 
</P><P>
Guanine and vitamin B12 sensitive riboswitches were used to design the system. Riboswitches are mRNA domains that can regulate gene expression depending on if it is bound to a ligand. <I>E. coli</I> possesses a vitamin B12 riboswitch that regulates the expression of the <I>btuB</I> genes<SUP><A href="#pd22" id="refpd22">2</A></SUP>. It also possesses B12 receptors on its outer membrane, allowing vitamin B12 uptake<SUP><A href="#pd23" id="refpd23">3</A></SUP>. The vitamin B12 <I>btuB</I> riboswitch will be used to design a system that will start gene expression in the presence of vitamin B12. For the guanine riboswitch the xpt-buX operon of <I>Bacillus subtilis</I><SUP><A href="#pd24" id="refpd24">4</A></SUP> has been used. <I>E.coli</I> possesses guanine receptors<SUP><A href="#pd25" id="refpd25">5</A></SUP>, so no additional transporters are needed.
</P><P>
The riboswitches regulate gene expression at the translation level<SUP><A href="#pd26" id="refpd26">6</A></SUP>, so they need to be constitutively transcribed. Therefore, a promoter of the <A href="https://2016.igem.org/Team:Wageningen_UR/Notebook/Riboswitch#may">Anderson</A> constitutive promoter family was placed in front of the riboswitch. It is the consensus constitutive promoter family of iGEM, it is well documented and the amount of expression differs 2500 fold between different promoters. It can also be easily swapped for another promoter of the same family if a different amount of expression is needed<SUP><A href="#pd27" id="refpd27">7</A></SUP>. This is convenient since the optimal expression level of the toxin gene is not known yet. 
</P><P>
Riboswitches are known to be able to both start and stop gene expression upon binding to specific ligands. Both the guanine and vitamin B12 riboswitch stop expression after binding to the metabolites<SUP><A href="#pd24" id="refpd24">4</A>,<A href="#pd25" id="refpd25">5</A></SUP>. In order for the system to work upregulating gene expression, the TetR Quad-part inverter system has been used. Quad-part inverters, or QPI, are genetic regulatory inverters that consists of a ribosome binding site, a coding region for a repressor protein, a terminator and a promoter that is regulated by the encoded repressor protein<SUP><A href="#pd27" id="refpd27">7</A></SUP>. The TetR system was used since it is widely used and characterized as a well-functioning inverter<SUP><A href="#pd28" id="refpd28">8</A></SUP>. Figure 1 shows a schematic overview of the system with the riboswitch, TetR QPI and monomeric Red Fluorescent Protein or mRFP.
</P><FIGURE><FIGCAPTION>Figure 1. Schematic overview of the mite sensing system based on riboswitch </FIGCAPTION></FIGURE><P>
An often encountered problem when creating a novel combination of genetic elements is the fact that even seemingly simple genetic functions behave differently in different settings<SUP><A href="#pd29" id="refpd29">9</A></SUP>. Therefore, the behaviour of the designed construct could be different than expected. To make the system more stable or increase the chance that the system behaves as designed, the riboswitch system will be made bicistronic. This means that two genes are under the control of one promoter. In this case, the sequence of the riboswitches has been cloned together with fifty amino acids downstream of the riboswitch in the original genomic sequence of <I>B. Subtilis</I> and <I>E. coli</I>. Adding fifty amino acids to the system means an increase of protein production for the cell which can be stressful. Therefore a ssRA protein degradation tag has been added behind the additional fifty amino acids. Any protein fused to this tag is degraded after translation. For the design of this system, the <I>E. coli ssRA tag</I><SUP><A href="#pd30" id="refpd30">10</A></SUP>, has been used. 
</P><H2 id="proofconc">Testing With Different Concentrations of Vitamin B12 And Guanine</H2><P> The designed construct was grown overnight in LB medium with different concentrations of guanine and vitamin B12. The concentration range of vitamin B12 was based on the concentrations normally used for testing bacteria that are used to determine the amount of vitamin B12 in human blood. The concentration range of guanine to test the guanine riboswitch construct were based on the concentrations used in previous experiments with guanine riboswitches from <I> B. Subtilis </I> The amount of mRFP has been measured every five minutes. Figure 2 and 3 show the mRFP over OD as from five hours of growth for the vitamin B12 and for the guanine riboswitch, respectively.</P><FIGURE><FIGCAPTION>Figure 2. Expressed mRFP over OD after five hours of growth. 
  </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 3. Expressed mRFP over OD after five hours of growth. 
  </FIGCAPTION></FIGURE><P> The expression experiment shows that gene expression is induced based on different concentrations of vitamin B12 and guanine allowing <I>E. coli </I> to sense the presence of <I> V. destructor </I>.  

In addition, as a part of our collaboration with <A href="https://2016.igem.org/Team:Groningen/Collaborations">Groningen</A>, they tested our construct in their lab as well. Their results are comparable to our vitamin B12 results. However, this is not the case for the guanine construct.
</P></SECTION><SECTION id="QuorumSensing"><H1>Population Dynamics</H1><H2 id="toxinproduction">Modifying Population Dynamics to Improve Toxin Production</H2><P>In our quest to save the honey bees from <I>Varroa destructor</I> we wanted to create a product that takes into account the hobbyist character of beekeepers.Therefore, application of BeeT should not be time consuming or require any additional expertise from beekeepers. 
Our modified <I>Bacillus thuringiensis</I> Cry toxin ensures a high level of specificity. When activated and concentrated near the surface of cell membranes, Cry toxins form pores in the membrane, lysing the cell as a result<SUP><A href="#pd1" id="refpd1">1</A></SUP>. Constitutive overexpression of such toxins in <I>Escherichia coli</I> is known to inhibit growth leading to cell death<SUP><A href="#pd2" id="refpd2">2</A></SUP>. The aim behind the population dynamics subproject is to provide a regulatory system for toxin production that does not impair bacterial growth and survival. To this end, we propose a regulatory system that enables <I>E. coli</I> to express recombinant proteins only in high cell density. Additionally we designed a system to allow <I>E. coli</I> to divide in subpopulations. This would allow separation between toxin producing cells and non-producers with increased chances of survival. </P><H2 id="quorumsensing">Quorum Sensing: Bacterial Density Regulation</H2><P>Systems that enable bacteria to regulate expression based on their density are called <B>quorum sensing </B> mechanisms. We have adopted one of the best known quorum sensing systems: the lux system originating from <I>Vibrio fischeri</I>. We demonstrated this system's functionality using a newly constructed GFP reporter. 
</P><P>The lux system consists of <B><I>luxI</I></B> and <B><I>luxR</I></B>: two genes that allow bacteria to communicate. <B>LuxI</B> is a synthase that produces acyl-homoserine lactones (AHLs). AHLs are small compounds that diffuse across cell membranes. They function as autoinducers: molecules that bacteria secrete to signal population density.</P><P>A single cell produces insufficient autoinducer molecules to start quorum sensing. However, when there is a high density of bacteria that produce AHL, the AHL concentration in the medium increases. Upon absorption of AHL by the cell, these molecules bind to LuxR. <B>LuxR</B> is the receptor protein that binds AHLs. Upon binding, LuxR then further induces expression of <I>luxI</I>. This positive feedback is the crucial component of quorum sensing. </P><FIGURE><FIGCAPTION>Figure 4. Schematic of the lux quorum sensing system as designed in this subproject. <I>luxI</I>, <I>luxR</I> and <I>GFP</I> represent protein-encoding genes. AHL represents acyl-homoserine lactones. LuxR-AHL represents the LuxR protein bound to acyl-homoserine lactones. Arrows between component indicate induction/activation/production.</FIGCAPTION></FIGURE><P>To test density dependent expression, two strains of <I>E. coli</I> DH5α were modified. <B>The first strain</B> contains two plasmids: one with <A href="http://parts.igem.org/Part:BBa_K546000">the <I>luxI</I> and <I>luxR</I> operon</A> and one with the <A href="http://parts.igem.org/Part:BBa_K1913014">GFP quorum sensing reporter</A>. <B>The second strain</B> contains one plasmid that <A href="http://parts.igem.org/Part:BBa_K1913005">combines</A> both the luxI/luxR operon and the quorum sensing reporter. </P><P>Either strain should be able to communicate their cell density and produce GFP when cell density is high enough. The bacteria were indeed found to acquire green color, preliminarily indicating functional quorum sensing and reporting. The cells that contain only the reporter plasmid did not turn green, indicating that they are not able to communicate their cell density.</P><FIGURE><FIGCAPTION>Figure 5. Fluorescence and absorbance data for E. coli quorum sensing strains. Continuous line: Fluorescence/OD<SUB>600</SUB>; Dashed line: Absorbance at OD<SUB>600</SUB>; Red: 2 plasmid quorum sensing strain; Green: 1 plasmid quorum sensing strain; Purple: reporter plasmid only strain. For both strains every value displayed is the average of all technical replicates (at least three) and for each, the line displayed is one of three biological repeats all showing a similar pattern. All Fluorescence/OD<SUB>600</SUB> data were normalized to the range of the 2 plasmid quorum sensing strain data. The Fluorescence/OD<SUB>600</SUB> data for the 1 plasmid quorum sensing strain were corrected to start at 0 (as they would otherwise have been negative). After almost 12 hours, the fluorescence from the 2 plasmid strains reached an overflow. Therefore no Fluorescence/OD<SUB>600</SUB> values after this point are displayed.
</FIGCAPTION></FIGURE><P>To investigate the dynamics of the two quorum sensing strains, plate reader measurements were used. We measured GFP fluorescence and optical density to correct total GFP production for the bacterial population (Figure 5). There is a clear difference between the responses of the quorum sensing strains and the negative control that only contains the GFP reporter plasmid. Fluorescence in the negative control barely rises above the background level. However, both quorum sensing strains show a steep response in GFP expression at comparable optical densities. This indicates that both tested systems function as expected and provide a means of regulating gene expression based on bacterial density. </P><H2 id="subpop">Subpopulation Formation</H2><P>Using the previously described system, population-wide Cry toxin overexpression is likely to kill all <I>E. coli</I> cells in the first moment of toxin expression. It would be beneficial to subdivide this population to keep healthy bacteria, as non-producers. These cells would be able to initiate a new growth phase after death of the toxin-producing cells. The critical requirement for this is that some cells can behave differently than the norm of the population, but are genetically identical. A collection of cells that acts differently from the rest of the population is called a <B>subpopulation</B>.</P><P>For bacteria to produce a subpopulation there must exist two states in which an individual cell can grow. We based our states on a two protein system. The proteins that we chose to include in the system (Figure 6) are phage λ- and phage 434 cI repressor protein. The working of the dominantly present protein determines cell behavior. <A href="http://parts.igem.org/Part:BBa_I12006">A promoter</A> has been modified previously that is regulated by the two cI proteins in opposite ways: the λ cI repressor protein induces the promoter, whereas the cI repressor protein from phage 434 represses the promoter.</P><P>To create a subpopulation, the levels of the two cI proteins, or at least the ratios between them, should vary between cells. Inspired by persister cell formation, we investigated expressing the two cI genes in one operon behind the same promoter. Differences in the turnover of the two proteins leads to switching dynamics where changes in promoter strength induce temporary changes in the ratio between the level of the proteins. Imagine a system where both cI genes are transcribed at intermediate levels. 434 cI has a much stronger ribosome binding site (RBS) than the λ cI gene and therefore is translated at a higher rate. This leads to a situation where there is more 434 cI than λ cI and 434 cI is dominant. When the promoter regulating both genes is repressed, both genes will, in time, reach a new, lower balance of protein levels. However, as 434 cI is degraded faster than λ cI, 434 cI protein levels will decrease more rapidly. This allows for brief domination of  λ cI. We hypothesize that small differences, like cell age and metabolism, between cells can determine whether λ cI indeed gets the chance to dominate the system. </P><P>The principle described here is similar to (and was inspired by) how some cells become dormant persister cells, while others continue growing<SUP><A href="#pd5" id="refpd5">5</A></SUP>. In the main model for persister cells, persisters differ from other cells in the balance between a toxin and an antitoxin. In these cells, the toxin dominates the antitoxin. This causes dormancy, allowing persisters to escape antibiotics and other environmental factors that might kill growing cells<SUP><A href="#pd6" id="refpd6">6</A></SUP>. <B> We decided against an attempt to create actual persisters out of safety reasons.</B> Persister cell formation can be expected to increase the chance of bacteria evading kill-switch mechanisms and other bio-containment measures<SUP><A href="#pd9" id="refpd9">9</A></SUP>. </P><FIGURE><FIGCAPTION>Figure 6. A schematic of the subpopulation circuit under control of the pBAD promoter. <I>λ cI</I>, <I>434 cI</I> and <I>mRFP</I> all represent protein encoding genes. Glu and Ara indicate that the promoter upstream of the <I>cI</I> is effectively induced by L-arabinose and repressed by D-glucose. The tested mRFP expression functions represents the position of the Cry toxin in the final envisioned system. Arrows between component indicate induction of expression, production or activation. The bar-headed line denotes repression.</FIGCAPTION></FIGURE><P>To test the viability of the described subpopulation system, we constructed a plasmid where λ and 434 cI encoding genes are under control of the pBAD/AraC promoter. This promoter can be induced with L-arabinose and repressed with D-glucose. This control over expression is useful to manipulate the formation of the subpopulations as envisioned in our design. Another operon was constructed with the modified λ Prm promoter controlling mRFP expression, representing expression of the Cry toxin. </P><P>We hypothesized that the system can only work when the balance between the cI proteins is in a certain ‘sweet spot’ of ratios. To improve our chances of hitting this sweet spot we incorporated a library of 18 possible RBS’s upstream of the <I>434 cI</I> gene. The library was kindly designed by Daniel Gerngross using the Redlibs algorithm<SUP><A href="#pd8" id="refpd8">8</A></SUP>. This provided us with a library of limited size representing a linear increase in predicted translation rates. This way we varied the translation rates of 434 cI between clones, resulting in different ratios between the λ and 434 cI protein levels for these clones. </P><P>DH5α cells transformed with the plasmid - containing both operons and the RBS library - yielded clones with varying intensities of red coloring on agar plates without L-arabinose and D-glucose (Figure 7). The variation in the red intensity is seen very clearly between the top three circled colonies in Figure 7.This diversity is a first indication that the assembled system works as intended: variation in the 434 cI RBS should cause different ratios between λ and 434 cI protein levels, leading to differences in mRFP expression. </P><FIGURE><FIGCAPTION>Figure 7. Close-up picture of culture plate with colonies containing the subpopulation two-operon plasmid including the RBS library. The blue circled and faintly red circled colonies were selected to be used in separate plate reader experiments.</FIGCAPTION></FIGURE><P>To investigate the working of this subpopulation system we analysed RFP fluorescence in cultures from 95 colonies of the RBS library transformation. From these we obtained four colonies that showed either a unique or a representative RFP response to glucose for further testing. By comparing the population-wide RFP response in the presence and absence of glucose. Based on this comparison we selected two clones to measure with flow cytometry.</P><P>In an effort to quantify the effect glucose has on the formation of subpopulations we performed a flow cytometry experiment. This analysis was done for two of the clones: B6 and C10. This method clearly revealed that glucose addition has little or no effect on the formation of subpopulations in these cells (Figure 8). </P><FIGURE><FIGCAPTION>Figure 8. Flow cytometry results for clone B6 and the constitutive mRFP expression control. The images on the left are histograms that show counts of bacteria with specific values of forward light scatter (FSC-H). The histograms on the right show counts of cells with fluorescence corresponding to mRFP. The mRFP fluorescence was measured with an excitation at 561nm, measuring emission at 620nm with 10nm bandwidth. </FIGCAPTION></FIGURE><P>Although flow cytometry shows that two clones do not form subpopulations, this is no reason to assume that none of the clones are able to do so. We chose to include an 18-mer RBS library on purpose. We expect that if the system described here can lead to subpopulations, this ability relies on the RBS strength. As we have not been able to measure all RBS inserts, we cannot accept nor reject the possibility that the system described here works as intended.</P><P>For future research, we <A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum">modeled</A> the subpopulation system in an effort to identify the most promising clones. The modeling is based on predicted translation rates that correspond with the transformed RBS library. To be able to relate the modeling results to the bacterial clones, we selected seven different clones to send for sequencing of the RBS insert. </P><H2 id="conclusions">Integration and Conclusion</H2><P>The two systems, quorum sensing, and subpopulation formation, were designed to work in conjunction (Figure 9). When high amounts of the luxR-AHL complex are present, those would repress the 434 and λ cI operon instead of glucose. This requires the promoter of the 434 and λ cI operon to be changed to the <A href="”http://parts.igem.org/Part:BBa_R0063”">BBa_R0063</A> promoter. We were unable to construct and test this total combined system due to time constraints. However, in an attempt to predict the population dynamics that result from such a system, we modeled first the separate systems and then combined both models. To read about the modeling of these systems <A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum"><B>click here</B></A>.</P><FIGURE><FIGCAPTION>Figure 9. Schematic of the total system designed to improve toxin expression through changing population dynamics of <I>E. coli</I>. This system combines quorum sensing and the system designed to form subpopulations. <I>luxI</I>, <I>luxR</I> and <I>GFP</I>, <I>λ cI</I>, <I>434 cI</I> and <I>mRFP</I> represent protein-encoding genes. AHL represents acyl-homoserine lactones. LuxR-AHL represents the LuxR protein bound to acyl-homoserine lactones. Arrows between component indicate induction of expression, production or activation. Bar-headed lines denote repression.</FIGCAPTION></FIGURE><P>Experimentally, we have shown the dynamics of quorum sensing with a newly constructed <A href="http://parts.igem.org/Part:BBa_K1913014">reporter</A>. Quorum sensing can be used to change the expression pattern of Cry toxins in time to minimize negative effects of Cry toxin toxicity to <I>E. coli</I>.
In addition, we propose a system for subpopulation formation to further decrease effects of Cry toxin toxicity on BeeT. Despite our efforts to test the system, we cannot draw a definite conclusion on whether the system indeed is able to induce subpopulation formation. We submitted all mentioned constructs to the registry to allow future teams to use them.</P></SECTION><SECTION id="Toggleswitch"><H1>Toggle Switch</H1><P>Because of the importance of controlling the toxin production, we took two different approaches towards this goal.Although quorum sensing system could be a good strategy to keep BeeT’s toxin expression in a reasonable range, it is possibly difficult to apply this system into realistic beehive conditions since BeeT may not growth to a certain density in the brood cells. Therefore, we engineered a regulatory system possibly more suitable for the beehive context as a strategy to integrate mite sensing and light-control for regulating BeeT’s toxin expression. The designed system is based on a toggle switch. It consists of two repressors and allows controlling BeeT’s toxin expression under two different stimulus, guanine or vitamin B12 and blue light. This toggle switch is based on a well-known genetic toggle switch <SUP><A href="#pd10" id="refpd10">1</A></SUP>. The mutual repression of two repressor proteins results in bi-stability of the system. The two steady states are off-state and on-state. In our toggle switch, the on-state leads to the stable expression of BeeT’s toxin and it is only reached when the system detects that both BeeT is in the beehive，where light is absent, and mites are present in the brood cells, where guanine or vitamin B12 can be sensed. In conclusion, we constructed a regulatory system that could be applied into real beehive and it integrates mite sensing with light-control device, which could provide a stable output of toxin expression with fast responsiveness. </P><H2 id="lightregu">How Does Light Regulate BeeT?</H2><P>In this part, we show the design of a regulatory system that connects the toggle switch and the light kill switch by means of light (Figure 10). The toggle switch controls expression of the BeeT’s toxin between an off-state and an on-state, which can be switched with two stimulus, guanine (or vitamin B12) and blue light. Our mite sensor is based on two types of riboswitches, one causes transcription termination and the other prevents transcription initiation by locating on the 5’ UTR region of the mRNA attenuating gene translation after binding a metabolite <SUP><A href="#pd11" id="refpd11">2</A></SUP>. The light sensor makes use of YF1 and FixJ. YF1 is the fusion of the LOV (Light-oxygen-voltage-sensing ) protein with a histidine kinase <SUP><A href="#pd12" id="refpd12">3</A></SUP>. In the absence of light, YF1 can activate FixJ by phosphorylation resulting in the activation of the Fixk2 promoter. In the presence of 480 nm wavelength light, YF1 can no longer phosphorylate FixJ due to the change position of a salt bridge on the LOV domain, leading to the deactivation of the Fixk2 promoter <SUP><A href="#pd13" id="refpd13">4</A></SUP>. Changes on the guanine or vitamin B12 and light conditions result in different situations on the bee hive.</P><H3><B>Situation 1: No Mites Present</B></H3><P>Initially, in the brood cells, the expression of the BeeT’s toxin is inhibited by the LacI repressor since the phosphorylation process between YF1 and FixJ takes some time to activate Fixk2 and produce enough TetR repressor to inhibit  p<I>tet</I> (promoter driving the expression of LacI). On this situation the system remains in  the off-state (Figure 10a). At the same time, MazE is constitutively expressed from the light Kill Switch preventing the death of the cells. </P><H3><B>Situation 2: Mite Present</B></H3><P>When mites are present in the brood cells, they produce and release guanine and vitamin B12 into the microenvironment of the brood cells. The binding of these metabolites to the riboswitches on the 5’ UTR of the <I> LacI </I>mRNA will attenuate translation of LacI (Figure 10b) resulting on the stop of the inhibition of the FixK2-p<I>lac</I> hybrid promoter and initiating the expression of TetR and BeeT’s toxin (Figure 10c). At this point the system has switched to on-state and any small perturbations will not influence the stable expression of BeeT’s toxin. As cells are still in dark, MazE continues gaining over MazF and cells are alive.</P><H3><B>Situation 3: BeeT Escapes The Hive</B></H3><P>Once our BeeT escapes from the brood cell, the exposition to light would inhibit the phosphorylation process between YF1 and FixJ, resulting in the inactivation of FixJ and stopping the transcription from the hybrid promoter, FixK2 promoter on Kill Switch. TetR can no longer inhibit the ptet promoter and the system switch back to the off-state (Figure 10d). In the meantime, MazF would start being expressed, which leads to programmed cell death reducing the the environmental impact of BeeT.</P><FIGURE><FIGCAPTION>Figure 10. BeeT light and guanin/vitamin B12 regulation. (a) In dark and prior to sensing of guanine/vitamin B12, LacI is repressing the p<I>lacI</I>-Fixk2 hybrid promoter and the toggle switch is on the off- state (no BeeT’s toxin production). Dark keeps a certain level of lambda cI preventing MazF expression and keeping the cells alive. (b) After guanine/vitamin B12 perception, the riboswitch on the <I>LacI</I> mRNA attenuates its translation starting the transition of the toggle switch from the OFF to the ON state. The light kill switch remains on the same state. (c) The decrease of LacI initiates TetR and BeeT’s toxin production (on-state). The light kill switch remains on the same state. (d) Exposure to light inhibits phosphorylation between YF1 and FixJ gradually reducing the inhibition of TetR and switching the system back to the OFF state. On the light kill switch, light exposure relieves MazF expression from the inhibition of lambda cI repressor. </FIGCAPTION></FIGURE><H2 id="hybriddesign">Hybrid Promoter Design</H2><P>In order to make Fixk2 promoter as an inducible promoter for the toggle switch, which could also be compatible with the light kill switch, we had to add additional repressor operator sequences into the Fixk2 sequence. Hence, the whole structure of Fixk2 should be elucidated. However, we could not find any literature demonstrating the definite structure of the Fixk2 promoter <A href="http://parts.igem.org/Part:BBa_K592006">(BBa_K592006)</A>from the iGEM parts registry and even the original Fixk2 promoter from pDawn and pDusk system <SUP><A href="#pd14" id="refpd14">5</A></SUP> does not have a detailed structural analysis. So we decided to construct five different FixK2 hybrid promoters (<A href="http://parts.igem.org/Part:BBa_K1913022">BBa_K1913022</A>, <A href="http://parts.igem.org/Part:BBa_K1913023">BBa_K1913023,</A><A href="http://parts.igem.org/Part:BBa_K1913024">BBa_K1913024</A>, <A href="http://parts.igem.org/Part:BBa_K1913025">BBa_K1913025</A>, <A href="http://parts.igem.org/Part:BBa_K1913026">BBa_K1913026</A>) based on the sequence of the wild type FixK2 promoter of <I>Bradyrhizobium japonicum</I><SUP><A href="#pd15" id="refpd15">6</A></SUP>. Nellen-Anthamatten <I>et al.</I> made a definite sequence structure analysis of this Fixk2, showing the presence of two FixJ boxes between the -40 and -70 region and a -10 to -35 core element (Figure 11a). According to this sequence structure, we added two additional LacI operators, both upstream and downstream the Fixk2 sequence (Fig 2, b), which could be bonded by tetrameric Lac repressor resulting in the formation of a DNA loop and consequently on transcription repression <SUP><A href="#pd16" id="refpd16">7</A></SUP>. We also designed a ptet-Fixk2 hybrid promoter by inserting two TetR operators into the core element region so that Tet repressors could form a dimmer and bond to these operators, resulting in transcription repression <SUP><A href="#pd17" id="refpd17">8</A></SUP>.However, according to some previous iGEM projects (<A href="https://2013.igem.org/Team:UNITN-Trento">UNITN-Trento 2013</A>, <A href="https://2013.igem.org/Team:INSA_Toulouse">INSA-Toulouse 2013</A>), the transcription activity of the wild type Fixk2 promoter is so weak that they all added an inverter part to control their target gene expression. Even the original pDusk system in darkness has only 5 times expression levels than in light conditions. Therefore, we decided to enhance the transcriptional activity of the Fixk2 promoter by changing the core element region of the wild type Fixk2 by adding a strong constitutive promoter (<A href="http://parts.igem.org/Part:BBa_J23106">BBa_J23106</A>) from iGEM part registry and by adding two typical FixJ boxes <SUP><A href="#pd18" id="refpd18">9</A></SUP><SUP><A href="#pd19" id="refpd19">10</A></SUP>. into the -40 to -70 region (Figure 11ab) In addition, we designed an additional plac-Fixk2 as a backup (Figure 11d) via changing core element into -10 to -35 region of ompC promoter from another two component system <SUP><A href="#pd20" id="refpd20">11</A></SUP>, because we couldn’t guarantee that FixJ boxes could be compatible with the core element of a constitutive promoter.</P><FIGURE><FIGCAPTION>Figure 11. Structure of wild type Fixk2 promoter and two derivative hybrid promoters BBa_K1913025, BBa_K1913026. (a) Wild type Fixk2 promoter contains two FixJ binding boxes (in red) upstream the core promoter (blue) and a xnts sequence downstream it. (b) BBa_K1913025 is a p<I>lac</I>-Fixk2 hybrid promoter that contains two lacI operators on both sides of the wild type Fixk2. (c) BBa_K1913026 is a p<I>tet</I>-Fixk2 hybrid promoter that contains two TetR operators integrated in the core element region.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 12. Structure of three synthetic hybrid promoters BBa_K1913022, BBa_K1913023, BBa_K1913024. (a) BBa_K1913022  is a p<I>lac</I>-Fixk2 synthetic promoter that contains two typical FixJ boxes (in green) upstream the core element of the constitutive promoter BBa_J23106 (dark red) as well as two extra lacI operators flanking the promoter (b) BBa_K1913023 is a p<I>tet</I>-Fixk2 synthetic promoter that contains two TetR operators in the core element region of BBa_J23106. (c) BBa_K1913024 is a plac-Fixk2 promoter with the <I>ompC</I> promoter core element region (in purple).</FIGCAPTION></FIGURE><H2 id="hybridactivation">Hybrid Promoters Are Activated in the Dark in Presence of YF1-FixJ</H2><P>In order to test the hybrid promoters, we constructed five composite parts with mRFP gene as reporter (<A href="http://parts.igem.org/Part:BBa_K1913027">BBa_K19103027</A>, <A href="http://parts.igem.org/Part:BBa_K1913028">BBa_K19103028</A>, <A href="http://parts.igem.org/Part:BBa_K1913029">BBa_K19103029</A>, <A href="http://parts.igem.org/Part:BBa_K1913030">BBa_K19103030</A>, <A href="http://parts.igem.org/Part:BBa_K1913031">BBa_K19103031</A>). These composite parts were co-transformed with the light sensor part (<A href="http://parts.igem.org/Part:BBa_K1913034">BBa_K19103034</A>) into <I>E.coli</I> strain BL21, cultured under dark for 24h and their fluorescence was tested. We included as controls cells that only contained the Fixk2 composite parts. The results (Figure 13) illustrated that hybrid promoter BBa_K1913025 has 5 times significant different compared to control, which suggested that this promoter is the most sensitive one for being induced by FixJ. Hybrid promoters BBa_K1913023 and BBa_K1913024 showed relative higher florescence values than others, suggesting that they have certain leaky expression. Whereas there were no signifi-cant difference between these promoters and their controls, this means that these two promoters have a low sensitivity to induction by FixJ compared to the other promoters.</P><FIGURE><FIGCAPTION>Figure 13.  Ratio of fluorescence value and absorbance of each Fixk2 composite. Each Fixk2 composite and control was cultured under dark condition over 24h. Emission and excitation wavelength of mRFP are 607 and 584 nm respectively. Error bars show the standard error among three technical replicates. </FIGCAPTION></FIGURE><H2 id="conclusions_tog">Conclusion</H2><P>In conclusion, we designed a toggle switch that could coordinate mites sensing, BeeT's toxin production and light sensing with responsiveness. Although we have not tested the whole toggle switch system, we supposed that this system could work as expected as we described above since our guanine and vitamin B12 riboswitches and hybrid promoters are both functional.</P></SECTION><H1>References</H1><SECTION id="references"><OL class="references"><A id="ri1" href="http://www.nature.com/nbt/journal/v31/n6/abs/nbt.2597.html">1.</A> Tabashnik, B. E., Brévault, T., &amp; Carrière, Y. (2013). Insect resistance to Bt crops: lessons from the first billion acres. Nature biotechnology, 31(6), 510-521.
<A href="#refri1" title="Jump back to footnote 1 in the text.">↩</A><H2>References for Mite Detection</H2><A id="pd21" href="http://www.pugetsoundbees.org/wp-content/uploads/2014/06/Varroa-reproductive-biology-PSBA-2014.pdf">1.</A> Huang, Z. (2012). Varroa mite reproductive biology. American Bee Culture http://www. extension. org/pages/65450/varroa-mite-reproductivebiology.<A href="#refpd21" title="Jump back to footnote 1 in the text.">↩</A><A id="pd22" href="http://www.ncbi.nlm.nih.gov/pubmed/12869542&quot;">2.</A> Rodionov, D. A., Vitreschak, A. G., Mironov, A. A., &amp; Gelfand, M. S. (2003). Comparative genomics of the vitamin B12 metabolism and regulation in prokaryotes. Journal of Biological Chemistry, 278(42), 41148-41159.<A href="#refpd22" title="Jump back to footnote 2 in the text.">↩</A><A id="pd23" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC236181/">3.</A> Bradbeer, C, Woodrow, M.L &amp; Khalifah, L.I. (1976). Transport of vitamin B12 in Escherichia coli: common receptor system for vitamin B12 and bacteriophae BF23 on the outer membrane of the cell envelope. Journal of Bacteriology, 125(3), 1032 - 1039.<A href="#refpd23" title="Jump back to footnote 3 in the text.">↩</A><A id="pd24" href="http://nar.oxfordjournals.org/content/35/16/5568.short">4.</A> Mulhbacher, J &amp; Lafontaine, D.A. (2007). Ligand recognition determinants of guanine riboswitches. Nucleic Acids Research, 35(16), 5568–5580.<A href="#refpd24" title="Jump back to footnote 4 in the text.">↩</A><A id="pd25" href="http://www.jbc.org/content/288/52/36827.short">5.</A> Papakostas, K, Botou, M &amp; Frillingos, S. . (2013). Functional Identification of the Hypoxanthine/Guanine Transporters YjcD and YgfQ and the Adenine Transporters PurP and YicO of Escherichia coli K-12. The journal of biological chemistry, 288, 36827-36840<A href="#refpd25" title="Jump back to footnote 5 in the text.">↩</A><A id="pd26" href=" http://www.nature.com/nrg/journal/v8/n10/abs/nrg2172.html ">6.</A> Serganov, A &amp; Patel, D.J. (2007). Ribozome, riboswitches and beyond: regulation of gene expression without proteins. Nature Reviews Genetics, 8(October), 776-790<A href="#refpd26" title="Jump back to footnote 6 in the text.">↩</A><A id="pd27" href="http://jbioleng.biomedcentral.com/articles/10.1186/1754-1611-3-4">7.</A> Kelly, J. R., Rubin, A. J., Davis, J. H., Ajo-Franklin, C. M., Cumbers, J., Czar, M. J., ... &amp; Endy, D. (2009). Measuring the activity of BioBrick promoters using an in vivo reference standard. Journal of biological engineering, 3(1), 1.<A href="#refpd27" title="Jump back to footnote 7 in the text.">↩</A><A id="pd28" href="http://www.nature.com/nsmb/journal/v7/n3/abs/nsb0300_215.html">8.</A> Orth, P., Schnappinger, D., Hillen, W., Saenger, W., &amp; Hinrichs, W. (2000). Structural basis of gene regulation by the tetracycline inducible Tet repressor–operator system. Nature Structural &amp; Molecular Biology, 7(3), 215-219.<A href="#refpd28" title="Jump back to footnote 8 in the text.">↩</A><A id="pd29" href="http://www.nature.com/nmeth/journal/v10/n4/abs/nmeth.2404.html">9.</A> Mutalik, V. K., Guimaraes, J. C., Cambray, G., Lam, C., Christoffersen, M. J., Mai, Q. A., ... &amp; Endy, D. (2013). Precise and reliable gene expression via standard transcription and translation initiation elements. Nature methods, 10(4), 354-360.<A href="#refpd29" title="Jump back to footnote 9 in the text.">↩</A><A id="pd30" href="http://www.nature.com/nbt/journal/v32/n12/abs/nbt.3053.html">10.</A>Cameron, D. E., &amp; Collins, J. J. (2014). Tunable protein degradation in bacteria. Nature biotechnology, 32(12), 1276-1281.<A href="#refpd30" title="Jump back to footnote 10 in the text.">↩</A><H2>References for Toggle Switch</H2><A id="pd10" href="http://www.nature.com/nature/journal/v403/n6767/full/403339a0.html">1.</A> Gardner T S, Cantor C R, Collins J J. Construction of a genetic toggle switch in Escherichia coli[J]. Nature, 2000, 403(6767): 339-342. <A href="#refpd10" title="Jump back to footnote 1 in the text.">↩</A><A id="pd11" href="http://www.nature.com/nrm/journal/v5/n6/abs/nrm1403.html">2.</A> Mandal, M., &amp; Breaker, R. R. (2004). Gene regulation by riboswitches. Nature Reviews Mo-lecular Cell Biology, 5(6), 451-463.<A href="#refpd11" title="Jump back to footnote 1 in the text.">↩</A><A id="pd12" href="http://www.sciencedirect.com/science/article/pii/S0022283608015088">3.</A> Möglich A, Ayers R A, Moffat K. Design and signaling mechanism of light-regulated histidine kinases[J]. Journal of molecular biology, 2009, 385(5): 1433-1444.<A href="#refpd12" title="Jump back to footnote 1 in the text.">↩</A><A id="pd13" href="http://pubs.acs.org/doi/abs/10.1021/bi026978l">4.</A> Crosson S, Rajagopal S, Moffat K. The LOV domain family: photoresponsive signaling mod-ules coupled to diverse output domains[J]. Biochemistry, 2003, 42(1): 2-10.<A href="#refpd13" title="Jump back to footnote 1 in the text.">↩</A><A id="pd14" href="http://www.sciencedirect.com/science/article/pii/S0022283612000113">5.</A> Ohlendorf R, Vidavski R R, Eldar A, <I>et al.</I> From dusk till dawn: one-plasmid systems for light-regulated gene expression[J]. Journal of molecular biology, 2012, 416(4): 534-542.<A href="#refpd14" title="Jump back to footnote 1 in the text.">↩</A><A id="pd15" href="http://jb.asm.org/content/180/19/5251.short">6.</A> Nellen-Anthamatten D, Rossi P, Preisig O, <I>et al.</I> Bradyrhizobium japonicumFixK2, a Crucial Distributor in the FixLJ-Dependent Regulatory Cascade for Control of Genes Inducible by Low Oxygen Levels[J]. Journal of bacteriology, 1998, 180(19): 5251-5255.
<A href="#refpd15" title="Jump back to footnote 1 in the text.">↩</A><A id="pd16" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC551766/">7.</A> Oehler S, Eismann E R, Krämer H, <I>et al.</I> The three operators of the lac operon cooperate in repression[J]. The EMBO journal, 1990, 9(4): 973.<A href="#refpd16" title="Jump back to footnote 1 in the text.">↩</A><A id="pd17" href="http://nar.oxfordjournals.org/content/25/6/1203.short">8.</A> Lutz R, Bujard H. Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements[J]. Nucleic acids research, 1997, 25(6): 1203-1210.
.<A href="#refpd17" title="Jump back to footnote 1 in the text.">↩</A><A id="pd18" href="http://jb.asm.org/content/187/10/3329.short">9.</A> Mesa S, Ucurum Z, Hennecke H, <I>et al.</I> Transcription activation in vitro by the Bradyrhizobium japonicum regulatory protein FixK2[J]. Journal of bacteriology, 2005, 187(10): 3329-3338.<A href="#refpd18" title="Jump back to footnote 1 in the text.">↩</A><A id="pd19" href="http://onlinelibrary.wiley.com/doi/10.1016/S0014-5793(02)02618-2/full">10.</A> Ferrières L, Kahn D. Two distinct classes of FixJ binding sites defined by in vitro selection[J]. FEBS letters, 2002, 517(1-3): 185-189.<A href="#refpd19" title="Jump back to footnote 1 in the text.">↩</A><A id="pd20" href="http://jb.asm.org/content/168/1/86.short">11.</A> Mizuno T, Mizushima S. Characterization by deletion and localized mutagenesis in vitro of the promoter region of the Escherichia coli ompC gene and importance of the upstream DNA domain in positive regulation by the OmpR protein[J]. Journal of bacteriology, 1986, 168(1): 86-95.<A href="#refpd20" title="Jump back to footnote 1 in the text.">↩</A><H2>References for Population Dynamics</H2><A id="pd1" href="http://doi.org/10.1111/j.1751-7915.2012.00342.x">1.</A> Bravo, A., Gómez, I., Porta, H., García-Gómez, B. I., Rodriguez-Almazan, C., Pardo, L., &amp; Soberón, M. (2013). Evolution of Bacillus thuringiensis Cry toxins insecticidal activity. Microbial Biotechnology, 6(1), 17–26. 
 <A href="#refpd1" title="Jump back to footnote 1 in the text.">↩</A><A id="pd2" href="http://doi.org/10.1007/BF00299149">2.</A> Douek, J., Einav, M., &amp; Zaritsky, A. (1992). Sensitivity to plating of Escherichia coli cells expressing the cryA gene from Bacillus thuringiensis var. israelensis. MGG Molecular &amp; General Genetics, 232(1), 162–165.
 <A href="#refpd2" title="Jump back to footnote 2 in the text.">↩</A><A id="pd3" href="http://doi.org/10.2174/138920306775474095">3.</A> Saida, F., Uzan, M., Odaert, B., &amp; Bontems, F. (2006). Expression of Highly Toxic Genes in E. coli: Special Strategies and Genetic Tools. Current Protein and Peptide Science, 7(1), 47–56. 
 <A href="#refpd3" title="Jump back to footnote 3 in the text.">↩</A><A id="pd4" href="http://doi.org/10.4051/ibc.2011.3.3.0010">4.</A> Pearson, B., Lau, K. H., DeLoache, W., Penumetcha, P., Rinker, V. G., Allen, A., … Campbell, A. M. (2011). Bacterial Hash Function Using DNA-Based XOR Logic Reveals Unexpected Behavior of the LuxR Promoter. Interdisciplinary Bio Central, 3(3), 1–8. 
 <A href="#refpd4" title="Jump back to footnote 4 in the text.">↩</A><A id="pd5" href="http://doi.org/10.1371/journal.pgen.1000437">5.</A> Van Melderen, L., &amp; Saavedra De Bast, M. (2009). Bacterial Toxin–Antitoxin Systems: More Than Selfish Entities? PLoS Genetics, 5(3), e1000437.
 <A href="#refpd5" title="Jump back to footnote 5 in the text.">↩</A><A id="pd6" href="http://doi.org/10.1128/AEM.02636-13">6.</A> Wood, T. K., Knabel, S. J., &amp; Kwan, B. W. (2013). Bacterial persister cell formation and dormancy. Applied and Environmental Microbiology, 79(23), 7116–21.
 <A href="#refpd6" title="Jump back to footnote 6 in the text.">↩</A><A id="pd7" href="http://doi.org/10.1038/nrmicro1557">7.</A> Lewis, K. (2007). Persister cells, dormancy and infectious disease. Nature Reviews Microbiology, 5(1), 48–56. 
 <A href="#refpd7" title="Jump back to footnote 7 in the text.">↩</A><A id="pd8" href="http://doi.org/10.1038/ncomms11163">8.</A> Jeschek, M., Gerngross, D., Panke, S., Canton, B., Labno, A., Endy, D., Perezmartin, J. (2016). Rationally reduced libraries for combinatorial pathway optimization minimizing experimental effort. Nature Communications, 7, 11163.
 <A href="#refpd8" title="Jump back to footnote 8 in the text.">↩</A><A id="pd9" href="http://doi.org/10.1371/journal.pgen.0020135">9.</A> Engelberg-Kulka, H., Amitai, S., Kolodkin-Gal, I., &amp; Hazan, R. (2006). Bacterial Programmed Cell Death and Multicellular Behavior in Bacteria. PLoS Genetics, 2(10), e135. 
 <A href="#refpd9" title="Jump back to footnote 9 in the text.">↩</A></OL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2016. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV></DIV></BODY></HTML>