Team:Wageningen UR/Notebook/Chitinases
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
★ Design
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
AWARDS
★ Entrepreneurship
★ Hardware
★ Software
★  Measurement
★ Model
Wageningen UR iGEM 2016
☰
˟
Overview
Description
Results
Achievements
BioBricks
Specificity
Regulation
Biocontainment
Model
Outreach
Team
Notebook
 
 
 
Introduction
July
August
September
Overview
Description
Results
Achievements
BioBricks
Applied Design
WetLab
Specificity
In vitro assay
Toxin Engineering
Varroa Isolates
Regulation
Detecting Mites
Population Dynamics
Toggle Switch
E. coli survival
Biocontainment
Light Kill Switch
Cas9 Kill Switch
Collaborations
Model
Population Dynamics
Light Kill Switch
E. coli survival
Beehave
Toxin Scanner
Outreach
Our Vision
Design
Collaboration
Events
Media
Integrated Practices
Team
Members
Students
Supervisors
Gallery
Attributions
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
These experiments were performed by Lisa Röttjers.
Introduction to Chitinases
Both honeybees and Varroa destructor mites have a chitin exoskeleton. As chitin is one of the most ubiquitous biopolymers found in nature, plenty of organisms possess the ability to degrade it. A well-described example of such an organism is Serratia marcescens, which is considered a model organism for chitinolytic activity 1 .  Chitinases isolated from Serratia marcescens GEI strain have been shown to have miticidal activity, while honeybee workers appear to be insensitive to them 2 . Additionally, chitinases are not harmful to humans, so they are a safer alternative to organic acids. Therefore, we made BioBricks out of these chitinases as a back-up strategy.
July
Week 1
This week, we received our IDT order; this contained chitinase A and B from Serratia marcescens GEI strain. The chitinases were optimized for BioBrick use, as is described in their respective BioBrick pages: chiA and chiB . Initially, I tried to do a PCR to amplify the synthetic genes, but this resulted in a too small product for chiB (Figure 1). The PCR was done with Q5 polymerase and the program from Table 1 was used.
Table 1: PCR program for chitinase amplification with BioBrick-f and BioBrick-r primers and Q5 polymerase.
Step
Temperature in °C
Time
Predenaturation
98
30 seconds
Denaturation
98
7 seconds
Annealing
60
20 seconds
Extension
72
60 seconds
Final Extension
72
5 minutes
Figure 1. Photo of a 1% TAE gel loaded with chiA and chiB PCR products, BBa_J04450 as a positive control and water as a negative control. The expected size for chiA is approximately 1750 basepairs, for chiB approximately 1560 basepairs. The gel was run for 30 minutes at 100V.
This did not work, so I also tried raising the annealing temperature to 70 degrees Celsius, diluting the templates to 1 ng/μL. That failed as well, so a gradient PCR was performed with OneTaq and 2% DMSO(see Figure 2).
Table 2: PCR program for chitinase gradient PCR with BioBrick-f and BioBrick-r primers and OneTaq polymerase.
Step
Temperature in °C
Time
Predenaturation
94
30 seconds
Denaturation
94
30 seconds
Annealing
Gradient of 60+-5
60 seconds
Extension
68
120 seconds
Final Extension
68
5 minutes
Table 2: PCR program for chitinase gradient PCR with BioBrick-f and BioBrick-r primers and OneTaq polymerase.
Figure 2. Photo of a 1% TAE gel loaded with chiA gradient PCR products, BBa_I0500 as a positive control and water as a negative control. The expected size for chiA is approximately 1750 basepairs. The gel was run for 30 minutes at 100V.
Figure 3. Photo of a 1% TAE gel loaded with chiB gradient PCR products, BBa_I0500 as a positive control and water as a negative control. The expected size for chiB is approximately 1560 basepairs. The gel was run for 30 minutes at 100V.
As Q5 PCR seemed to fail, both chitinases were digested with EcoRI and PstI and cloned into pSB1C3 with heat shock transformation. Both the PCR products were used as well as the synthetic genes as supplied by IDT. The transformation worked well. Colony PCR to confirm insert size failed, so a plasmid purification was done for 5 colonies from each transformation. These were analysed with a OneTaq PCR with VF2 and VR primers. The same program was used as in Table 2, except the annealing temperature was 51 degrees Celsius and the extension time was 210 seconds.
Figure 4. Photo of a 1% TAE gel loaded with chiA and chiB PCR products, BBa_J04450 as a positive control and water as a negative control. The expected size for chiA is 2007 basepairs, for chiB 1816 basepairs. The gel was run for 30 minutes at 100V.
I picked the first colonies for both transformants which had been transformed with digested IDT oligo's. They were verified through sequencing. I also worked with the fifth chiB colony for some time, which turned out to have a 500 bp deletion.
Week 2
I tried to add RBS BBa_B0032 to the chitinases with reverse primer VR in combination with the following forward primers and Q5 polymerase:
chiA: 5' CCGATGAATTCGCGGCCGCTTCTAGtcacacaggaaagtaCTAGATGCGCAAGTTCAATAAACC
chiB: 5'CCGgTGAATTCGCGGCCGCTTCTagAtcacacaggaaagtaCTAGATGTCCACACGTAAAGCCGTTATTG
An annealing temperature of 62 degrees Celsius was used. The PCR failed, only the positive control showed bands.
To figure out why the PCR was not working, I did another gradient PCR with OneTaq polymerase. The same program as in Table 2 was used, except the extension time was set to 4 minutes and the annealing temperate was 47 +- 5 degrees Celsius. Figure 5 also shows the wrong size of chiB, but at this point I did not quite realize it.
Figure 5. Photo of a 1% TAE gel loaded with chiA and chiB PCR products, BBa_J04450 as a positive control and water as a negative control. The expected size for chiA is 1913 basepairs, for chiB 1722 basepairs. The gel was run for 30 minutes at 100V.
Though the OneTaq gradient PCR clearly worked, I had to make a PCR work with a polymerase that has proofreading activity. I did one PCR with Pfu polymerase, but that failed as even my positive control showed no band.  I repeated the Q5 PCR to add BBa_B0032 one more time, but now with an annealing temperature at 52 degrees Celsius, 10 degrees below the NEB recommended temperature. The results are shown in Figure 6.
Figure 5. Photo of a 1% TAE gel loaded with chiA and chiB PCR products, BBa_J04450 as a positive control and water as a negative control. The expected size for chiA is 1913 basepairs, for chiB 1722 basepairs. The gel was run for 30 minutes at 100V.
Week 3
This week, the PCR products with added RBS were cloned into pSB1C3 with BBa_B0015. Purified plasmids digested with XbaI and PstI were checked for the correct lengths (Figure 6). While chiA had the correct length, it was not digested correctly; I figured out that my RBS primers were wrong and did not include the AG nucleotides for the XbaI site in the prefix. Additionally, I realized that chiA was too short. The plasmids were sequenced and my suspicions were confirmed.
Figure 6. Photo of a 1% TAE gel loaded with digested plasmids containing chiA and chiB with RBS and terminator. The gel was run for 30 minutes at 100V.
Week 4
Worked on isolates from Varroa destructor.
August
Week 5
I got a new primer and used this with reverse primer VR to correct the RBS.
5' atctaGAATTCGCGGCCGCTTCTAGAGtcacacaggaaagtaCTAG
Figure 7 shows the results of the Q5 PCR with this primer set. The PCR was done according to Table 1, but with an annealing temperature of 52 degrees Celsius and an extension time of 90 seconds. It went wrong once; afterwards, I repeated it with and without DMSO and incubated the PCR mix without polymerase but with template and primers at 10 minutes for 42 degrees Celsius.
Figure 7. Photo of a 1% TAE gel loaded with PCR products containing chiA and chiB with old RBS. The chiB with old RBS was necessary to correct for the 500 basepair deletion, while chiA was included as a positive control. For chiA, the 3A assembly with terminator was also used as a template to test the new primer. The gel was run for 30 minutes at 100V.
I needed one extra PCR to correct the chiB RBS; this worked, but not for chiA with only the coding sequence. Therefore, I purified chiA with terminator as this had the fixed RBS. The chitinases were ligated into pSB1K3 and transformed.
Week 6
Did 3A assembly of the chitinases; chiB + BBa_B0032 was ligated to BBa_I0500, chiA + BBa_B0032 + BBa_B0015 to BBa_I0500.
Week 7
Week 6 transformants were used for plasmid purification, sequenced and another 3A assembly was done: chiB with RBS and promoter was ligated to chiA with terminator and to terminator only (BBa_B0015). They were cloned into pSB4K5. I also tried to clone them into pSB1C3 right away, but the vector had not been linearized correctly. The transformants were checked with colony PCR, but that was not too conclusive; plasmids were digested with EcoRI and PstI and loaded on gel to confirm size (Figure 8).
Figure 8. Photo of a 1% TAE gel loaded with chiA and chiB devices inserted in pSB4K5. The plasmids were digested with EcoRI and PstI. The gel was run for 45 minutes at 100V.
The chiA device should have fragments of 3110 and 3378 basepairs, the chiB device 2917 and 3378 basepairs and the chiBA device should have a 4639 basepair insert. While chiA looks fine, the other ones are the wrong sizes. I picked 16 more colonies for the chiB device; the very last one turned out to be correct. They were verified through sequencing.
Week 8
chiA and chiB devices were cloned into E. coli strain BL21 and I repeated the colony PCR for the chiBA device with Q5 polymerase as OneTaq may not be able to amplify such a short insert. The colony PCR showed transformants with the right insert size, so these were used for plasmid purification, plasmids were verified through sequencing and also cloned into BL21.
All devices were cloned into pSB1C3 and verified through sequencing. Additionally, they were digested with EcoRI and PstI and ran on gel to verify insert sizes (Figure 9). Only the chiBA device could not be sequenced for unknown reasons; I performed plasmid purification twice and sent it for sequencing, but both times the sequencing failed.
Figure 9. Photo of a 1% TAE gel loaded with chiA and chiB devices inserted in pSB4K5 or pSB1C3. The plasmids were digested with EcoRI and PstI. The gel was run for 45 minutes at 100V.
September
Week 9
Made an SDS-PAGE gel from extracts BL21 cultures with the chitinase devices, induced with 0%, 0.2% or 1% arabinose. The SDS-PAGE gel is shown in Figure 10. I could not clearly observe the chitinase bands.
Figure 10. Photo of a 10% BioRad MiniProtean gel loaded with protein extracts from BL21 constructs with chitinase devices. Device BBa_K1913002 is supposed to express a 81.5 kDa protein, while BBa_K1913003 should express a 76 kDa protein and BBa_K1913004 should express both. They were induced with 0%, 0.2% or 1% arabinose. The gel was run for 38 minutes at 50 mA together with a gel for the cellular debris (not shown).
Week 10-11
I tried to adapt an in vitro assay to measure chitinase activity. This assay is based on a reaction with 3-methyl-2-benzothiazolinone hydrazine (MBTH) and was developed by Horn et al. 3 Colloidal chitin was prepared according to Murthy et al. 4 The MBTH method yielded no results, as even the calibration curve with N-acetylglucosamine showed no increase in absorbance at 620 nm. The experiment was repeated twice but no conclusion on chitinase activity was reached.
Week 12
I made colloidal chitin plates according to Murthy et al. 3 and pipetted BL21 cultures with the chitinase devices onto them, as well as Bacillus thuringiensis positive controls. The colloidal chitin density was not high enough and the chitin formed flakes. No halo could be observed that would confirm chitinase activity.
Figure 11. Photo of a colloidal chitin plate with cultures from BL21 with a chiB device induced with 0%, 0.2% or 1% arabinose. No halo could be observed.
References
1. Vaaje‐Kolstad, G., Horn, S. J., Sørlie, M., & Eijsink, V. G. (2013). The chitinolytic machinery of Serratia marcescens–a model system for enzymatic degradation of recalcitrant polysaccharides. FEBS Journal, 280(13), 3028-3049. ↩
2. Tu, S., Qiu, X., Cao, L., Han, R., Zhang, Y., & Liu, X. (2010). Expression and characterization of the chitinases from Serratia marcescens GEI strain for the control of Varroa destructor, a honey bee parasite. Journal of invertebrate pathology, 104(2), 75-82. ↩
3. Horn, S. J., & Eijsink, V. G. (2004). A reliable reducing end assay for chito-oligosaccharides. Carbohydrate Polymers, 56(1), 35-39. ↩
4. Murthy, N. K. S., & Bleakley, B. H. (2012). Simplified method of preparing colloidal chitin used for screening of chitinase producing microorganisms. Internet J Microbiol, 10(2), e2bc3. ↩
iGEM Wageningen UR 2016. Send us an email or follow us on Facebook !
For more information about iGEM, check out their website and Twitter !
