<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Notebook_VarroaIsolates skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Notebook/VarroaIsolates</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Wageningen_UR/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><TITLE>Wageningen UR iGEM 2016 </TITLE><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><BUTTON class="hamburger">☰</BUTTON><BUTTON class="cross">˟</BUTTON><DIV class="mobilemenu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><B>Overview</B></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Team</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Notebook</A></LI></UL></DIV><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates">Main Results</A></H4></DIV><LI class="menu-item"><A href="#april">April</A></LI><LI class="menu-item"><A href="#may">May</A></LI><LI class="menu-item"><A href="#june">June</A></LI><LI class="menu-item"><A href="#july">July</A></LI><LI class="menu-item"><A href="#august">August</A></LI><LI class="menu-item"><A href="#september">September</A></LI><LI class="menu-item"><A href="#october">October</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design">Applied Design</A></LI></UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"><DIV class="ca-content">WetLab</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Assay"><I>In vitro</I> assay</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#ToxinEngineering">Toxin Engineering</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates" class="up_menu_fix"><I>Varroa</I> Isolates</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#DetectingMites" class="up_menu_fix">Detecting Mites</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing" class="up_menu_fix"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#Toggleswitch" class="up_menu_fix">Toggle Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/ecoli_survival"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#Lightswitch" class="up_menu_fix">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#SAA" class="up_menu_fix">Cas9 Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model"><DIV class="ca-content">Model</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#light">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#beehave">Beehave</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">Toxin Scanner</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Vision">Our Vision</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#collaboration">Collaboration</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#events">Events</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"><FONT size="1">Integrated Practices</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/contact">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#environment">Environment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#laboratory">Laboratory</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Journals</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments">Protocols</A></LI></DIV></DIV></DIV><DIV class="center_content"><P>These experiments were performed by Lisa Röttjers.</P><SECTION id="april"><H1><B>April</B></H1><H2><B>Week 1</B></H2><P>Sporulation salts were made and used to make sporulation plates. Gridded microscopy plates were prepared using a wax pencil. Dead <I>Varroa</I> mites were gathered from the Unifarm location at Grebbedijk, Wageningen. These beehives were not treated against mites and contained a high number of them. The mites were not completely fresh.</P><H2><B>Week 2</B></H2><P>It was not known how many colonies could be isolated from one mite; therefore, microcentrifuge tubes were prepared with 1, 2, or 3 mites, 2 replicates each. The <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#isolates1">protocol</A> for <I>Bacillus</I> isolation was followed, except the sterilization step was with 1% halamid solution and lasted 10 seconds.</P><P>Initial plates showed no colonies; the experiment was repeated with a ~2.5% bleach washing step.</P><P>The second set of plates also showed no colonies. After the heat treatment, the LB + mites was incubated for 24 hours and streaked with an inoculation loop. </P><P>Again, no colonies: a mite was dissected and its gut rubbed on a plate. Due to the small size of the mite, it was difficult to isolate the gut. Additionally, the same protocol as before was followed, except only 0.1 mL of the LB was heat treated and the rest was plated without a heat treatment. </P><P>This week, the protocol failed to yield any colonies. I also tried to plate mite samples without a sterilization step. During dissection of the dead mite, I was able to observe dessication; this indicated that the mites were too dry to avoid sucking up bleach during the sterilization step. Their guts were probably also sterile, which explains the lack of colonies.</P><P>To test this hypothesis, I repeated the isolation protocol without a sterilization step and only washed with sterile tap water. This yielded 3 colonies, but now I would be unable to exclude environmental contamination. </P><P><FIGURE><FIGCAPTION>Figure 1. Photo of the first plate with colonies (no. 15). These mites were not sterilized, so this was probably contamination.</FIGCAPTION></FIGURE></P><H2><B>Week 3</B></H2><P>More colonies grew on the plates from Week 2. They were streaked, even though plate 13's colony was yellow and plates 14 and 15 only had transparent colonies. Plate 17 had small and large white round colonies.  Prepared more plates; also without sterilization step, but they were washed carefully. Up to plate 39 was prepared this week. I tried a different heat treatment: 3 minutes at 80 degrees Celsius. I observed the colonies with a stereo microscope, but images were unclear and largely useless.</P><H2><B>Week 4</B></H2><P>Prepared up to plate 49. A colony from plate 46 looked like <I>Bacillus thuringiensis</I> HD350!
I prepared 16 samples for brightfield microscopy. <I>B. thuringiensis</I> HD350 (also referred to as Bt HD350) was used as a positive control; this strain was ordered from DSMZ (no. DSM-6030) and grown according to the supplied protocol. All samples were stained with Coomassie Brilliant Blue R according to the <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#isolates1">protocol</A>. I used a Zeiss Axio Scope.A1 for imaging with the 100x oil immersion objective. Very little spore formation was observed. </P><P><FIGURE><FIGCAPTION>Figure 2. <I>Bacillus thuringiensis</I> HD350 and isolates 15, 17, 18, 46 and 47 stained with Coomassie Brilliant Blue R. 1000x magnification.</FIGCAPTION></FIGURE></P></SECTION><SECTION id="may"><H1><B>May</B></H1><H2><B>Week 5</B></H2><P>The streaks were now grown for 3 days to promote sporulation before they were imaged.
<FIGURE><FIGCAPTION>Figure 3. <I>Bacillus thuringiensis</I> HD350 and isolates 46 and 47 stained with Coomassie Brilliant Blue R. 1000x magnification. The last image was phase-contrast microscopy instead of brightfield microscopy. In both cases, the Cry toxins are visible.</FIGCAPTION></FIGURE></P><P>
I made cell-free extracts of my isolates and Bt HD350. The protein content was quantified with the Bradford method, but in all cases, this amount came out very low or negative; the extraction failed. 
I also did <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#isolates3">16s rRNA PCR</A> on isolates 46 and 47; Bt HD350 and <I>E. coli</I> were used as positive controls, water as a negative control.</P><FIGURE><FIGCAPTION>Figure 4. PCR products from 16s rRNA PCR run on a 1% TAE gel for 30 minutes at 100V.</FIGCAPTION></FIGURE><P>The cleaned PCR products were sent to GATC for LightRun sequencing.</P><H2><B>Week 6</B></H2><P>I inoculated 100 mL LB in 500 mL erlenmeyers with Bt HD350, V46, V47 and <I>E. coli</I> strains expressing Cry1 and Cry2 (courtesy of Ruud de Maagd). These were grown for 3 days at 30 degrees Celsius. A French cell press was used for the protein extraction. The first SDS-PAGE failed, so it was repeated. </P><P>Supernatant, protein extract and the cellular debris were analysed with SDS-PAGE; however, the gel with cellular debris came out completely empty. Figure 5 and 6 show the two gels and the putative Cry toxin bands.</P><P><FIGURE><FIGCAPTION>Figure 5. Protein extracts of Bt HD350, <I>E. coli</I> with Cry1 and Cry2 and isolates V46, V47. The red arrow indicates a band that is probably Cry1. The ladder is a BioRad Precision Plus Protein Standard, the gel is a MiniProtean TGX 12%.</FIGCAPTION></FIGURE></P><FIGURE><FIGCAPTION>Figure 6. Supernatant of the same cultures as in Figure 5. The red arrow indicates a band that is probably Cry1 (133 kDa). The ladder is a BioRad Precision Plus Protein Standard, the gel is a MiniProtean TGX 12%.</FIGCAPTION></FIGURE><H2><B>Week 7</B></H2><P>I repeated the SDS-PAGE from Week 6, because the bands from 46 and 47 were unclear and I could not observe Cry1Aa from Bt HD350. The SDS-PAGE failed as the machine was interrupted by leaking buffer. Additionally, I started working on cloning Cry1 and Cry2 from the <I>E. coli</I> strains. This did not work out in the end.</P><H2><B>Week 8</B></H2><P>I performed another SDS-PAGE to observe Cry1Aa production. Additionally, we received <I>Bacillus thuringiensis tenebrionis</I> (also referred to as Bt tenebrionis) from DSMZ, a strain which produces Cry3Aa. </P><P><FIGURE><FIGCAPTION>Figure 7. Cellular debris of <I>E. coli</I> BL21, the failed Cry1 and Cry2 clones, <I>Bacillus subtilis</I>, Bt HD350 and Bt tenebrionis, as well as isolates V46 and V47. The red and blue arrrow indicate bands which are probably Cry1Aa (133 kDa) and Cry3Aa (73 kDa). The ladder is a BioRad Precision Plus Protein Standard, the gel is a MiniProtean TGX 12%.</FIGCAPTION></FIGURE></P><P><FIGURE><FIGCAPTION>Figure 8. Protein extract of <I>E. coli</I> BL21, the failed Cry1 and Cry2 clones, <I>Bacillus subtilis</I>, Bt HD350 and Bt tenebrionis. The ladder is a BioRad Precision Plus Protein Standard, the gel is a MiniProtean TGX 12%.</FIGCAPTION></FIGURE></P><P>
The SDS-PAGE gel shows that the Cry toxins are mostly present in the cellular debris and are not very soluble. </P><P>I went to the Duurzame Bij in Veenendaal. These beekeepers do not treat their honeybees with pesticides, so they have a lot of <I>Varroa</I> mites. Dead mites were taken from the mite drawers below the hives with the help of Henk Kok. I filled 7 eppendorfs with approximately 100 mites each. With these fresh mites, I was able to perform the protocol as <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#isolates1">described</A> and still get suitable colonies. I prepared up to plate 62. Plates 50, 55, 57, 58, 59, 60, 61 and 62 had colonies with the right colony morphology. Brightfield microscopy with the Coomassie stain showed that only isolate 60 had rod-shaped cells, but when I checked the streaked plate again, the plate turned out to be contaminated. </P></SECTION><SECTION id="june"><H1><B>June</B></H1><H2><B>Week 9-12</B></H2><P>Unable to do lab work due to the laboratory moving to a new building.</P></SECTION><SECTION id="july"><H1><B>July</B></H1><H2><B>Week 13</B></H2><P>Streaked out plates again, made some new glycerol stocks.</P><H2><B>Week 14</B></H2><P>I performed the isolation protocol and prepared up to plate 107. Plates 64, 68, 69, 71, 75, 81, 82, 87, 88, 90, 92, 93, 94, 95, 96, 97, 98, 99, 101, 103, 104, 105, 106 and 107 showed colonies; each of the colonies were streaked (2 to 3 per plate). Figure 9 shows the streaked plates.</P><P><FIGURE><FIGCAPTION>Figure 9. Streaked colonies which were isolated from dead <I>Varroa destructor</I>.</FIGCAPTION></FIGURE></P><H2><B>Week 15</B></H2><P>I checked all of Week 14's isolates with brightfield microscopy. Instead of the toxic Coomassie Brilliant Blue R stain, I tried to stain with a premade BioRad Coomassie stain. This has the advantage of being far less toxic. However, the quality of the images was worse as is shown in Figure 10. I also grew cultures for protein extraction on <I>Bacillus subtilis</I>, <I>Bacillus thuringiensis tenebrionis</I>and HD350, as well as isolates 46, 47, 60, 81, 82, 88 and 106. </P><P><FIGURE><FIGCAPTION>Figure 10. Rod-shaped isolates from <I>Varroa destructor</I>, observed at 1000x magnification. The isolates are stained with a premade BioRad Coomassie stain.</FIGCAPTION></FIGURE></P><H2><B>Week 16</B></H2><P>Last week's cultures had very few spores, so I streaked them and grew them for 2 days to promote sporulation before doing microscopy (Figure 11). </P><P><FIGURE><FIGCAPTION>Figure 11. Rod-shaped isolates from <I>Varroa destructor</I>, observed at 1000x magnification. The isolates are stained with a premade BioRad Coomassie stain.</FIGCAPTION></FIGURE></P><P>
Additionally, I used the sonicator to perform protein extraction; 6 pulses of 30 seconds at 20% power. In between each pulse, the cells were chilled down in ice water for 30 seconds. I tried different power settings as the cells did not lyse well. In the end, I incubated each sample with 10 mg/mL hen egg lysozyme for 1 hour at 37 degrees Celsius. Then I repeated the sonication procedure. The sonicated samples were centrifuged and the cell-free extract was frozen in aliquots. No protease inhibitors were added, as these can be toxic to insects.</P><P>I used the protein extracts to do a <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#isolates2">toxicity test</A>. 35 mites were collected; 5 mites were used per protein extract. Bee pupae were also gathered as a food source for the mites. Most pupae died when they were gathered, but the mites feed on dead bees and larvae as well. I had enough mites to test the following six protein extracts: <I>Bacillus subtilis</I>, <I>Bacillus thuringiensis tenebrionis</I>, <I>Bacillus thuringiensis tenebrionis</I> pellet, <I>Bacillus thuringiensis</I> HD350 and isolates V46, V47. As a positive control, mealworms and <I>Bacillus thuringiensis tenebrionis</I> were used. </P><P>Initially, I tried to keep the mites in a 6-wells plate. They escaped. When the plate was sealed with parafilm, some were still able to get stuck in the parafilm. The mites and pupae were transferred to sterile microcentrifuge tubes with a hole poked in the top and wetted cotton in the bottom to maintain humidity. They were incubated at 35 degrees Celsius. All mites died after 24 hours, including the <I>Bacillus subtilis</I> negative control.</P><P><FIGURE><FIGCAPTION>Figure 12. PCR products from 16s rRNA PCR run on a 1% TAE gel for 30 minutes at 100V.</FIGCAPTION></FIGURE></P></SECTION><SECTION id="august"><H1><B>August</B></H1><H2><B>Week 17</B></H2><P>I prepared protein extracts of my isolates and loaded them onto an SDS-PAGE gel. Figure 13 shows the protein extract, Figure 14 the pellet.</P><P><FIGURE><FIGCAPTION>Figure 13. SDS-PAGE gel of the protein extracts from <I>Bacillus subtilis</I>, <I>Bacillus thuringiensis</I> HD350, <I>Bacillus thuringiensis tenebrionis</I> and isolates 46, 47, 81, 82 and 88. The ladder is a BioRad Precision Plus Protein Standard. The red arrow indicates a band that could be Cry1Aa, which is 133 kDa. The blue arrow indicates a band which could be Cry3Aa, a 73 kDa toxin. </FIGCAPTION></FIGURE></P><P><FIGURE><FIGCAPTION>Figure 14. SDS-PAGE gel of the cellular debris from <I>Bacillus subtilis</I>, <I>Bacillus thuringiensis</I> HD350, <I>Bacillus thuringiensis tenebrionis</I> and isolates 46, 47, 81, 82 and 88. The ladder is a BioRad Precision Plus Protein Standard. </FIGCAPTION></FIGURE></P><P>Isolate V106 failed to grow, so I grew it again and did another protein extraction. Additionally, I tried to do an <I>in vivo</I> toxicity test again. 46 mites were gathered and I left them overnight on a food source before I used them for the toxicity test; only 18 mites survived the night. I only tested a solution without protein extract, Bt tenebrionis and Bt HD350. All mites had died the next day.  </P><H2><B>Week 18-19</B></H2><P>Only worked on chitinases these weeks.</P><H2><B>Week 20</B></H2><P>Tried to make new microscopy images with the original Coomassie Brilliant Blue R stain, but I let the cells dry out too much so the images were very bad. </P></SECTION><SECTION id="september"><H1><B>September</B></H1><H2><B>Week 21</B></H2><P>I decided to investigate isolate V82 further based on its similarity to <I>Lysinibacillus sphaericus</I> and its overexpression of a 100 kDa protein. <I>Lysinibacillus</I> species are unable to grow well on most sugars, so I grew it for 3 days at 30 degrees Celsius on LB + sporulation salts only and on LB + sporulation salts with D-mannitol, D-fructose or glycerol. The colony morphology of V82 was different when it was grown on plates with an additional carbon source, as it became brownish in hue. This matches the <I>Lysinibacillus sphaericus</I> description. To confirm that I still had identical isolates, I performed 16s rRNA PCR on each of the plates with different carbon sources (Figure 15). All V82 PCR products were confirmed to belong to the same isolate. Additionally, I did protein extraction on the liquid cultures (without lysozyme) and loaded the extracts as well as the pellets on an SDS-PAGE gel (Figure 16). 
The consensus 16s sequence was as follows:</P><P>gnrNRysnnnbacntsmyrnstrannWSSrnbsmaRNAgnartnastncTGCTCAGGACGAACGCTGGCGGCGTGCCcAATACATGCAATCCaACCGGgcgTTACTTTGgTGCTTGCACCGAAGTAACACTAGCGGCGgACGGGTGAGTAACACGTGGGCAACCTACCTTgTAGATGGGGATAACTCCGGGAAACCGgtGCTAATACCGAATGATACTTTGAAACACATGCTTCGAAGTTGAAAGATGGTTCTGCTATCGCTACAGGATGGGCCCGCGGCGCATTAGCTAGTTGGTGAGGTAACGGCTCACCAAGGCGACGATGCGTAGCCGACCTGAGAGGGTGATCGGCCACACTGGGACTGAGACACGGCCCAGACTCCTACGGGAGGCAGCAGTAGGGAATCTTCCACAATGGGCGAAAGCCTGATGGAGCAACGCCGCGTGAGTGAAGAAGGTTTTCGGATCGTAAAACTCTGTTGTAAGGGAAGAACAAGTACAGTAGTAACTGGCTGTACCTTGACGGTACCTTATTAGAAAGCCACGGCTAACTACGTGCCAGCAGCCGCGGTAATACGTAGGTGGCAAGCGTTGTCCGGAATTATTGGGCGTAAAGCGCGCGCAGGCGGTTCTTTAAGTCTGATGTGAAAGCCCCCGGCTCAACCGGGGAGGGTCATTGGAAACTGGGGAACTTGAGTGCAGAAGAGGAAAGTGGAATTCCAAGTGTAGCGGTGAAATGCGTAGAGATTTGGAGGAACACCAGTGGCGAAGGCGACTTTCTGGTCTGTAACTGACGCTGAGGCGCGAAAGCGTGGGGAGCAAACAGGATTAGATACCCTGGTAGTCCACGCCGTAAACGATGAGTGCTAAGTGTTAGGGGGTTTCCGCCCCTTAGTGCTGCAGCTAACGCATTAAGCACTCCGCCTGGGGAGTACGGTCGCAAGACTGAAACTCAAAGGAATTGACGGGGGCCCGCACAAGCGGTGGAGCATGTGGTTTAATTCGAAGCAACGCGAAGAACCTTACCAGGTCTTGACATCCCATTGACCATTATGGAGACATAGTTTTCCCTTCGGGGACAATGGTGACAGGTGGTGCATGGTTGTCGTCAGCTcCGTGTCGTGAGATGTTGGGTTAAGTCCCGCAACGAGCGCAACCCTTnGATCTTAGTTGCCATCATTTAGTTGGGCACTCTAAGGTGACTGCCGGTGATAAACCGGAGGAAGGTGGGGATGACGTCAAATCATCATGCCCCTTATGACCTGGGCTACACACGTGCTACAATGGACGGTACAAACGGTTGCCAACCCGCGAGGGGGAGCTAATCCGATAAAACCGTTCTCAGTTCGGATTGTAGGCTGCAACTCGCCTACATGAAGCCGGAATCGCTAGTAATCGTGGATCAGCATGCCACGGTGAATACGTTCCCGGGCCTTGTACACACCGCCCGTCACACCACGAGAGTTTGTAACACCCGAAGTCGGTGAGGTAACCTTTTgGGAGCCAGCCGCCGAAgGTGGGATAGATGATTGGGGTGAATTCGAAGCAAGGTAGCCG</P><P><FIGURE><FIGCAPTION>Figure 15. PCR products from 16s rRNA PCR run on a 1% TAE gel for 30 minutes at 100V. LB refers to LB medium with sporulation salts.</FIGCAPTION></FIGURE></P><P><FIGURE><FIGCAPTION>Figure 16. SDS-PAGE gel of the supernatant, protein extract and cellular debris from isolate V82 grown on LB with sporulation salts and with additional carbon sources D-fructose, glycerol or D-mannitol. The ladder is a BioRad Precision Plus Protein Standard. </FIGCAPTION></FIGURE></P><H2><B>Week 22</B></H2><P>
This week, <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#isolates4">LC-MS/MS</A> was done on three cellular debris bands from Week 21's SDS-PAGE. The bands were cut out at 100 kDa and approximately 1 cm above this, control bands were cut out.<FIGURE><FIGCAPTION>Figure 17.  LC-MS/MS spectra of 3 control samples (top) and 3 samples extracted from a ~100 kDa band on the SDS-PAGE gel with V82 protein. Click the figure for the full-resolution image.</FIGCAPTION></FIGURE></P><P>Genomic DNA was extracted from isolate V82 with the GeneJet Genomic DNA Extraction kit and sent for sequencing.</P></SECTION><SECTION id="october"><H1><B>October</B></H1><P>We got sequencing results the day before wiki freeze! I gave them to Ronald so he could make contigs and throw them through his <A href="https://2016.igem.org/Team:Wageningen_UR/Software">toxin scanner.</A></P></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2016. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV></DIV></BODY></HTML>