<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Wageningen_UR/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><TITLE>Wageningen UR iGEM 2016 </TITLE><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><BUTTON class="hamburger">☰</BUTTON><BUTTON class="cross">˟</BUTTON><DIV class="mobilemenu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><B>Overview</B></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Team</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Notebook</A></LI></UL></DIV><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="#header">Modeling</A></H4></DIV><LI class="menu-item"><A href="#quorum">Population Dynamics</A></LI><LI><A href="#QS_intro">Quorum Sensing Circuit</A></LI><LI><A href="#QS_subpop">Subpopulation System</A></LI><LI><A href="#QS_combined">Combined System</A></LI><LI class="menu-item"><A href="#light">Light Kill Switch</A></LI><LI><A href="#lighton">Light On</A></LI><LI><A href="#lightonbeetoff">Light On, BeeT Off</A></LI><LI><A href="#insilicoswitch">In Silico Kill Switch</A></LI><LI class="menu-item"><A href="#metabolic">Metabolic Model</A></LI><LI><A href="#MM_model">Survival Model</A></LI><LI><A href="#MM_discussion">90 Minutes</A></LI><LI class="menu-item"><A href="#beehave">Beehave</A></LI><LI><A href="#beetmodule">BeeT Module</A></LI><LI><A href="#modelcons">Model Considerations</A></LI><LI><A href="#howcanhoneybeessurvive">Honeybee Survival</A></LI><LI><A href="#beetbehave">BeeT Beehave</A></LI><LI class="menu-item"><A href="#references">References</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design">Applied Design</A></LI></UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"><DIV class="ca-content">WetLab</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Assay"><I>In vitro</I> assay</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#ToxinEngineering">Toxin Engineering</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates" class="up_menu_fix"><I>Varroa</I> Isolates</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#DetectingMites" class="up_menu_fix">Detecting Mites</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing" class="up_menu_fix"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#Toggleswitch" class="up_menu_fix">Toggle Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/ecoli_survival"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#Lightswitch" class="up_menu_fix">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#SAA" class="up_menu_fix">Cas9 Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model"><DIV class="ca-content">Model</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#light">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#beehave">Beehave</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">Toxin Scanner</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Vision">Our Vision</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#collaboration">Collaboration</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#events">Events</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"><FONT size="1">Integrated Practices</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/contact">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#environment">Environment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#laboratory">Laboratory</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Journals</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments">Protocols</A></LI></DIV></DIV></DIV><DIV class="center_content"><H1 style="text-align:center;">Modeling Overview</H1><P>In light of our guiding principles (<A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">specificity</A>, <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">regulation</A> and <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">biocontainment</A>), 
we modeled four different aspects of BeeT. The modeling work can inform and aid the improvement 
of wet-lab experiments. Another facet is to assess the optimal application strategy for BeeT in real
 world applications. We asked ourselves; what parts of our system can benefit the most from an interplay 
between modeling and experimental work? These considerations led us to ask the following questions:
</P><UL><LI>How can we assure optimal toxin production using <A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum">quorum sensing and subpopulations</A>?</LI><LI>What are important parameters for the <A href="https://2016.igem.org/Team:Wageningen_UR/Model#light">optogenetic killswitch</A> to function optimally?</LI><LI>Will BeeT be capable of surviving in <A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic">sugar water</A>?</LI><LI>What is the best <A href="https://2016.igem.org/Team:Wageningen_UR/Model#beehave">application strategy</A> for BeeT?</LI></UL><SECTION id="quorum"><H1>Population Dynamics</H1><P>For the final product, BeeT, we intend to use toxins produced by <I>Bacillus thuringiensis</I>, also called Cry toxins. However, these toxins are also harmful to our chassis, resulting in a reduction of toxin production when it is needed to kill the mites. To counteract this effect we envision the use of quorum sensing that activates Cry toxin production only when there is a large quantity of BeeT present. Ideally, BeeT is able to produce Cry toxin over a long period to improve its effectiveness against mites. However, if an entire population of BeeT is synchronized, we hypothesize that only a single burst of Cry toxin will take place before both the BeeT and mites are killed. Thus, this system will not be maximally effective over long time periods. To accomplish this we need multiple subpopulations of BeeT: some producing BeeT while others are recuperating. To help understand this complex system we use dynamic modeling.</P><H2 id="QS_intro">Quorum Sensing Circuit</H2><P> Quorum sensing is a cell-cell communication system. The detection of chemical molecules allows the bacteria to sense the presence of other bacteria.
 In this way the bacteria control gene expression in response to changes in cell number<SUP><A href="#fn1" id="ref1">1,</A></SUP><SUP><A href="#fn4" id="ref4">4</A></SUP>. 
 This process is achieved through the production and release of an Acylated Homoserine Lactone (AHL)
 <A class="tooltip"> autoinducer <SPAN class="tooltiptext" style="width:300px;"> An autoinducer is a signal molecule related to cell density. </SPAN></A>. 
 The AHL can diffuse from one cell to the other. There are many different types of autoinducers in quorum sensing systems. When sensed, the autoinducer can trigger other cells to produce more autoinducers. The quorum sensing system consists of two proteins: LuxI and LuxR. LuxR is constitutively expressed, together with AHL it forms a complex which activates the transcription of a toxin. In our case we use the detection protein Green Fluorescence Protein (GFP) to follow the behavior of the system<SUP><A href="#fn2" id="ref2">2</A></SUP>. LuxI encodes an AHL synthase. AHL is a molecule that can diffuse freely through the cell membrane, and in this way travels from cell to cell. LuxR encodes for a protein and together they form a complex. In a natural system the LuxR-AHL complex controls transcription of LuxI, forming a positive feedback loop that increases the amount of AHL in the system<SUP><A href="#fn3" id="ref3">3</A></SUP>. When there is more AHL in the system, AHL is more likely to bind to the LuxR protein. In Figure 1 you can see a schematic representation of the system.</P><FIGURE><FIGCAPTION>Figure 1: Schematic of the quorum sensing system. The arrows indicate activation of a part and the flat bars indicate inhibition of a part. The quorum sensing system consists of two parts: LuxR and LuxI. LuxR constitutively produces a transcriptional factor that binds to AHL and LuxI produces an AHL autoinducer that diffuses 
freely through the cell membrane, from cell to cell. </FIGCAPTION></FIGURE><P>According to the 2011 team from Davidson College and Missouri Western State University<SUP><A href="#fn5" id="ref5">5</A></SUP>, LuxR protein represses transcription of the luxR gene in the absence of AHL, establishing a negative feedback. We investigated whether this negative feedback can be used to create different populations. As you can see in Figure 3 different states of GFP responses can occur when the production rate of luxR and rate of complex formation are changed. However, when we remove the feedback in the system and change the same parameters, we get similar responses of the system as shown in Figure 2.</P><FIGURE><FIGCAPTION>Figure 2: Quorum sensing without negative feedback. In the left graphs you see the production of GFP when
 the production rate of LuxR and complex formation parameters are different in each cell. Every line represents one cell, the system is simulated with 10 cells. Five of the 10 cells have a value of 
 0.01 for the production rate of LuxR and the complex formation parameters, and the other half of the cells have a value of 
 1.0 for these parameters. In the right graphs you see the responses of the GFP where the parameter values are the same for each cell.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 3: Quorum sensing with negative feedback. In the left graphs you see the production of GFP when the production rate of LuxR and complex 
formation parameters are different in each cell. Every line represents one cell, the system is simulated with 10 cells. Five of the 10 cells have a value of 0.01 for the production rate of LuxR and the complex formation parameters, and 
the other half of the cells have a value of 1.0 for these parameters. 
In the right graphs you see the responses of the GFP where the parameter values are the same for each cell. </FIGCAPTION></FIGURE><P>This shows that the feedback has a minimal influence on the system to create different subpopulations. Also when there is no negative feedback in the system, we found more parameter sets that give 
a high amount of GFP (see Figures 4 and 5).</P><FIGURE><FIGCAPTION>Figure 4. In this graph you see the amounts of GFP we got from the different parameter sets for the system without negative feedback. 
Most of the parameter sets give a low GFP response, but a few give high ones. Those we use for further analysis. </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 5. In this graph you see the amounts of GFP we got from the different parameter sets for the
 system with negative feedback. Most of the parameter sets give a low GFP response, but a
 few give high ones. Those we use for further analysis. </FIGCAPTION></FIGURE><P>The figures are simulated with the parameter sets obtained from the quorum sensing model. We are using 
the parameter sets that give the best response according to the confidence intervals 
(see Method section). In the presence and absence of negative feedback, the system behaves similarly by assuming the production rate of luxR and the formation rate of the complex are different in each cell.
The quorum sensing system can simulate different responses with the same cells and different parameters, if we could control the LuxR production rate. However, we could not envision a method of doing this experimentally. 
We aim to make cells that are genetically identical but still can respond differently. Therefore, we designed another system that could help us create this response downstream of the quorum sensing mechanism. 
The two systems were tested in the <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing">lab</A> parallel with the modeling.</P><H2 id="QS_subpop"> Why Include a Subpopulation System?</H2><P>Since the quorum sensing mechanism could not provide the subpopulations we desired using genetically identical cells, we created a system that could act downstream of quorum signalling 
to provide the response we needed. Population-wide Cry toxin overexpression is likely to kill all <I>E. coli</I> cells in the first moment of toxin expression. Since this would only affect one of the subpopulations, the second subpopulation would be able to initiate a new growth phase after death of the toxin-producing cells. The critical requirement for this is that cells respond at different times to the quorum stimuli despite being genetically identical. A small part of the population that acts differently from the rest of the population is called a subpopulation. The subpopulation system consist of two genes; the first encodes for the protein that inhibits the systems expression, the other encodes for the corresponding activation protein of the system. The 434 cI inactivates the λ cI directly, or prevents the translation of the λ cI protein. 434 cI has a higher turnover rate than λ cI. This subpopulation system is based on a natural system for persister cell formation
 <SUP><A href="#fn6" id="ref6">6</A></SUP>.</P><FIGURE><FIGCAPTION>Figure 6: Schematic of the subpopulation system. In this system,
 AraC inhibits the promoter activity of the subpopulation part. AraC is inhibited by
 Arabinose, and when glucose is added to the system Arabinose is inhibited by glucose.
 AraC inhibits the constitutive promoter of 434 cI and λ cI, where 434 cI 
 inhibits the promotion of λ cI. The production of λ cI leads to production of Red 
 Fluorescence Protein(RFP) as the reporter protein.</FIGCAPTION></FIGURE><P>As you can see in Figure 6; glucose has a suppressing function on the system, arabinose has an activating function on the system. We used the model to predict what will happen when we add glucose in different amounts. The assumptions we made for the subpopulation system are the following: 434 cI degrades faster than λ cI, cell division happens when the cell size has doubled, and two cell populations exist with one growing differently than another. We tested a number of parameter sets and picked the parameter set that gave the biggest RFP response based on the strength of glucose. To see how the model behaved when we changed the ratios of the initial amounts of λ cI and 434 cI, we investigated the effect of starting conditions on the outcome of the system. The initial conditions were varied between 1 and 10. Within the heatmap you can see in which ratios the initial amounts of λ cI and 434 cI in the system are needed to get high RFP production (here glucose and arabinose have a fixed concentration).</P><FIGURE><FIGCAPTION>Figure 7. Heatmap of λ cI and 434 cI against RFP production. The right side bar indicates the amount of RFP, where red indicate a high amount and black a low amount.
 This heatmap shows you the amount of RFP produced given the balance between the two genes initial conditions.</FIGCAPTION></FIGURE><P> As seen in Figure 7, when λ cI and 434 cI are present in similar amounts we have a subpopulation system. But if they are present in different amounts there will be no subpopulations. This can be expected when you look at the subpopulation system. The system is inhibited by 434 cI, which represses the RFP production, and λ cI activates the RFP production. In Figure 7 you can see the balance between the 434 cI and λ cI amounts that are present for the output of RFP. To understand how changing the Ribosomal Binding Site (RBS) impacts RFP responses, we simulated the system for different combinations of transcription rates. From these simulations we found: </P><FIGCAPTION><P><B>Table 1: RBS library of the 434 cI gene with different predicted translation rates.</B></P></FIGCAPTION><TABLE><TBODY><TR><TH> λ cI </TH><TH>434 cI </TH><TH> RFP responses</TH></TR><TR><TD>1.94</TD><TD>135.6</TD><TD>7.00</TD></TR><TR><TD>1.94</TD><TD>108.8</TD><TD>8.51</TD></TR><TR><TD>1.94</TD><TD>942.5</TD><TD>1.48</TD></TR><TR><TD>1.94</TD><TD>726</TD><TD>1.77</TD></TR><TR><TD>1.94</TD><TD>628.6</TD><TD>1.962</TD></TR><TR><TD>1.94</TD><TD>577.1</TD><TD>2.09</TD></TR><TR><TD>1.94</TD><TD>529</TD><TD>2.23</TD></TR><TR><TD>1.94</TD><TD>40.3</TD><TD>20.1</TD></TR><TR><TD>1.94</TD><TD>384.9</TD><TD>2.87</TD></TR><TR><TD>1.94</TD><TD>268.5</TD><TD>3.87</TD></TR><TR><TD>1.94</TD><TD>257.8</TD><TD>4.01</TD></TR><TR><TD>1.94</TD><TD>256.7</TD><TD>4.02</TD></TR><TR><TD>1.94</TD><TD>73.2</TD><TD>12.1</TD></TR><TR><TD>1.94</TD><TD>100.9</TD><TD>9.09</TD></TR></TBODY></TABLE><P>To confirm that our model represents biological behavior we analysed the RBS library. 
We can conclude from these results that the RFP concentration qualitatively corresponds to the Figure 7 Because as you can see in the Table 1 the highest RFP responses are when the λ cI and 434 cI are present in the same amount. The lowest responses of RFP when there is a is a big difference between λ cI and 434 cI. This is also shown in the heatmap we got form the model. </P><H2 id="QS_combined">Combined System</H2><FIGURE><FIGCAPTION>Figure 8: Schematic of the quorum sensing system attached to the subpopulation network. The arrows indicate activation of a part and the flat bars indicate inhibition of a part. 
The quorum sensing system consists of two parts: LuxR constitutively produces a transcriptional factor that binds to AHL, and LuxI produces an AHL autoinducer that diffuses  freely through 
the cell membrane, from cell to cell. LuxR forms a complex with AHL that inhibits the subpopulation system, where 434 cl and λ cl are under the same promoter. This leads to either GFP production, 
or RFP production depending on the same strength of quorum sensing.</FIGCAPTION></FIGURE><P> Based on the modeling we hypothesized the following: when there are more cells present in the system, more AHL-LuxR complex is formed. The complex inhibits the subpopulation promoter. 
When the promoter is inhibited production of 434 cI will be suppressed and production of λ cl will be activated. At a certain time point λ cl takes control over the system, because 434 cI has a higher turnover rate than λ cl.
 In this case more λ cl results in more RFP. </P><P>In a later stage, to get the desired response, the quorum sensing system and the subpopulation system were combined. As shown in Figure 8, you can see how we intend to generate different cell populations.
With this extended network we try to generate a model which predicts the behavior of the subpopulations. In Figure 9 you can see the increasing RFP over time after many cell divisions, indicating an increasing cell population.</P><FIGURE><FIGCAPTION>Figure 9. Volume oscillations correspond to the dividing cells. Together they form a growing cell population. On the left y-axis you see the RFP, and on the right y-axis you see the volume. The x-axis is the time. The total amount of RFP produced by the toxin is shown by the red line.</FIGCAPTION></FIGURE><H3>Methods</H3><P>During the research Matlab version R2016a has been used. 
Because there was no data from the wet lab we assumed that all parameters exist within a biologically reasonable range between zero and one. For the analyses we used 100,000 parameter sets, 10 cells, random initial conditions between zero and one and 60 timesteps. Tuneable parameters are used, each parameter set can produce dramatically different population dynamics. To determine which of these parameters produce the best system response we used <A class="tooltip"> Latin Hypercube<SPAN class="tooltiptext" style="width:700px;">Latin hypercube sampling is a statistical method to sample the parameter space equally. It divides the parameter space according to the number of parameters and sample size and picks a random number from each parameter subspace. </SPAN></A> sampling.</P><P>With the Latin Hypercube sampling we made parameter sets that are obtained from a 
<A class="tooltip"> lognormal distribution <SPAN class="tooltiptext" style="width:400px;"> 
a lognormal distribution assigns probabilities to all positive values. However the distributions is skewed to favour smaller values. Biological reaction rates have been found to follow this distribution</SPAN></A><SUP><A href="#fn8" id="ref8">8</A></SUP> with a parameter estimation based on Raue <I>et al</I><SUP><A href="#fn7" id="ref7">7</A></SUP>. In Figure 10 you can see an example.</P><FIGURE><FIGCAPTION>Figure 10. Here you see the lognormal distribution of the 19 parameters in the quorum sensing system. 
The red line indicates the mean of the parameter. </FIGCAPTION></FIGURE><P>With these lognormal distributions and the accessory confidence intervals the best parameter sets could be chosen. </P><P>In a later stage, to get the desired response, the quorum sensing system and the subpopulation system were combined. As shown in Figure 8, you can see how we think to generate different cell populations.
With this extended part we try to generate a system which predicts the behavior of the subpopulations. In figure 9 you can see the increasing RFP over time after many cell divisions, indicating an increasing cell population.</P><FIGURE><FIGCAPTION>Click the button to see the equations!</FIGCAPTION></FIGURE><H3>Conclusion</H3><P>Quorum sensing ensures that the toxin is only produced when the density of
 bacteria is high enough, thus standardising the amount of toxin produced by the bacteria
 population. The subpopulation system was coupled to the quorum sensing system. 
 Together, quorum sensing and formation of subpopulations allow 
 bacteria to produce waves of toxin. 
  
 Different parameters in the quorum sensing system elicit different GFP 
 responses. However, we can not expect to be able to make genetically identical cells with varying values for these parameters
With the subpopulation system we may be able to predict which RBSs of the library have the highest chance to successfully create subpopulations. The library  can be tested and used in further research. </P></SECTION><SECTION id="light"><H1>Optogenetic Kill Switch Model</H1><P>One important aspect of our project is the <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">biocontainment</A> of BeeT to the hive, preventing it from spreading into the environment. In collaboration with the wet-lab, we designed an optogenetic kill switch based on the blue light sensing pDusk and pDawn systems as described by Ohlendorf <I>et al</I><SUP><A href="#mb1" id="refmb1">1</A></SUP>. The cell death is inflicted by the mazEF toxin-antitoxin (TA) system. The latter functions by the antitoxin (mazE) preventing the toxin (mazF) from cleaving mRNAs which ultimately kills the cell. With the underlying biology of the system in mind, we used synthetic bioengineering principles to mathematically describe our design and explore the dynamics of the system. 
Our results show that we have a working mathematical model which describes the data of Ohlendorf <I>et al</I><SUP><A href="#mb1" id="refmb1">1</A></SUP>. With this model, we now know where to look for the relative kinetic rates in the parameter space to give suggestions to the wet-lab on how to build a functioning optogenetic kill switch. Our differential equations of pDusk and pDawn can also be used by future iGEM teams to mathematically describe and build new optogenetic tools.</P><H2 id="lighton"> Light On</H2><H3>Modeling The Optogenetic Tool</H3><P>For the first part of our analysis, we optimized the mathematical models of pDusk and pDawn using published data<SUP><A href="#mb1" id="refmb1">1</A></SUP>. For now, we ignored the toxin-antitoxin system. This gave us the option to construct our scoring function to evaluate both systems simultaneously with a weighted means approach.
By constructing an accurate model of the optogenetic part of our overall kill switch system, we are confident that we are mathematically representing the behavior of the microorganism in the lab. Results from the model can then be used as a reference point for building the toxin-antitoxin system into the kill switch model. We will search the parameter space to identify parameter sets which fit the data given by Ohlendorf <I>et al</I><SUP><A href="#mb1" id="refmb1">1</A></SUP>. In addition, we will select the best fit to data based on a scoring function as described by Raue <I>et al</I><SUP><A href="#mb4" id="refmb4">4</A></SUP>. Lastly, we provide a mathematical model which can be used by future iGEM teams to build new optogenetic tools.</P><H3>System Design</H3><P>We first studied the pDusk system in more detail to derive simplified equations which can describe the system behavior. The genetic switches are based on the interactions between the light sensing protein YF1 and the response regulator FixJ as described by Möglich <I>et al</I><SUP><A href="#mb2" id="refmb2">2</A></SUP>. In darkness YF1 phosphorylates FixJ, which in turn activates the promoter pFixK<SUB>2</SUB>. This activation leads to the expression of the desired genes. Under light conditions YF1 enters an excited light-state, which inhibits the phosphorylation of FixJ. In addition, it prevents the activation of the pFixK<SUB>2</SUB> promoter and thus represses the target genes in Figure 11 for pDusk.</P><FIGURE><FIGCAPTION>Figure 11. Simplified representation of the pDusk and pDawn system with an RFP reporter gene. The arrows indicate activation of a part and the flat bars indicate inhibition of a part.</FIGCAPTION></FIGURE><P>The pDawn system functions in the same way as the pDusk system, with the addition of λ cI being expressed depending on the activity of the pFixK<SUB>2</SUB> promoter. The protein form of λ cI inhibits the expression of the RFP reporter gene in Figure 11 for pDawn.</P><H3>Assumptions</H3><P>To derive mathematical equations describing the above systems, we made some simplifying assumptions. </P><UL><LI><P>We neglect the effects of dilution/ growth. We thus assume the cell’s volume to be constant over time.</P></LI><LI><P>We assumed that the components in our system are neither affected by nor affect other cellular mechanisms.</P></LI><LI><P>As there is no additional information gained from modeling transcription and translation for the dark state of YF1(y<SUB>DD</SUB>) and the inactive form of FixJ (j<SUB>i</SUB>), we lumped these processes and assumed that they can be described by a single parameter.</P></LI><LI><P>We also assumed the rates of production, binding, and degradation of mRNAs and proteins to be linear which results in a first order differential equations system</P></LI></UL><P>
To model the influence of light on the system, we assumed light activation to occur similar to the model of phytochrome B dimerization described in Klose <I>et al</I><SUP><A href="#mb3" id="refmb3">3</A></SUP> as <MATH><MI>N</MI><MO>⋅</MO><MSUB><MI>k</MI><MN>2</MN></MSUB></MATH>. In addition, we consider in our system the different stages of YF1 (y<SUB>DD</SUB>, y<SUB>DL/LD</SUB>, and y<SUB>LL</SUB>) as described by Möglich <I>et al</I><SUP><A href="#mb2" id="refmb2">2</A></SUP>. Translated to our system, this means that with increasing light intensities (N), the pool of activated YF1 (y<SUB>DL/LD</SUB> and y<SUB>LL</SUB>) would deplete the dark state of YF1 (y<SUB>DD</SUB>), which is responsible for the phosphorylation and thus activation from inactive FixJ (j<SUB>i</SUB>) to active FixJ (j<SUB>a</SUB>). As all stages of YF1 form a dimer, we assumed dimerization to occur quickly and be the dominant and relevant form for our system. Lastly, we assumed a quasi steady-state approximation (Michaelis-Menten kinetics) for the effect of activated FixJ (j<SUB>a</SUB>) on the expression downstream of the RFP (RFP<SUB>m</SUB>) reporter gene. A detailed system design is provided <A href="https://static.igem.org/mediawiki/2016/c/c2/T--Wageningen_UR--Detailed_Methods_Kill_Switch.pdf">here.</A></P><H3>Results</H3><P>With our mathematical model we can describe the behavior, as shown in Figure 12, of both systems as tested in the lab by Ohlendorf <I>et al</I><SUP><A href="#mb1" id="refmb1">1</A></SUP>. We can thus say that we built a simplified mathematical representation of the pDusk and pDawn system which can describe the systems’ behavior.</P><FIGURE><FIGCAPTION>

Figure 12. Comparison of computed model results to the data of Ohlendorf <I>et al</I><SUP><A href="#mb1" id="refmb1">1</A></SUP>. In both pDusk and pDawn the black dots ± standard deviation represent the fluorescence over OD600 of the reporter gene RFP as tested in the lab by Ohlendorf <I>et al</I><SUP><A href="#mb1" id="refmb1">1</A></SUP>. The blue circles represent the computed RFP responses per light intensity normalized to the highest light intensity in each graph. The red line is given to show the trend of the behavior of the system.
</FIGCAPTION></FIGURE><P>Preliminary results from the lab showed that the RFP<SUB>p</SUB> response after 17 hours incubation time is weak using the pDusk system. The experiment was conducted in darkness and under <MATH><MN>1.41</MN><MO>⋅</MO><MSUP><MN>10</MN><MN>4</MN></MSUP><MROW><MO>[</MO><MO>µ</MO><MI>m</MI><MI>o</MI><MI>l</MI><MO>⋅</MO><MSUP><MI>m</MI><MROW><MO>-</MO><MN>2</MN></MROW></MSUP><MO>⋅</MO><MSUP><MI>h</MI><MROW><MO>-</MO><MN>1</MN></MROW></MSUP><MO>]</MO></MROW></MATH> light intensity and the results are shown in Table 2.</P><FIGURE><FIGCAPTION>
Table 2: Preliminary Lab Results on pDusk/ pDawn System
</FIGCAPTION></FIGURE><TABLE><TBODY><TR><TH>System</TH><TH>Dark</TH><TH>Light</TH></TR><TR><TH>pDusk</TH><TH><MATH><MN>45.43</MN><MO>±</MO><MN>37.83</MN><MROW><MO>[</MO><MFRAC><MROW><MI>F</MI><MI>L</MI></MROW><MROW><MI>O</MI><MI>D</MI><MN>600</MN></MROW></MFRAC><MO>]</MO></MROW></MATH></TH><TH><MATH><MN>36.79</MN><MO>±</MO><MN>18.35</MN><MROW><MO>[</MO><MFRAC><MROW><MI>F</MI><MI>L</MI></MROW><MROW><MI>O</MI><MI>D</MI><MN>600</MN></MROW></MFRAC><MO>]</MO></MROW></MATH></TH></TR><TR><TH>pDawn</TH><TH><MATH><MN>126.54</MN><MO>±</MO><MN>25.19</MN><MROW><MO>[</MO><MFRAC><MROW><MI>F</MI><MI>L</MI></MROW><MROW><MI>O</MI><MI>D</MI><MN>600</MN></MROW></MFRAC><MO>]</MO></MROW></MATH></TH><TH><MATH><MN>5356.75</MN><MO>±</MO><MN>78.55</MN><MROW><MO>[</MO><MFRAC><MROW><MI>F</MI><MI>L</MI></MROW><MROW><MI>O</MI><MI>D</MI><MN>600</MN></MROW></MFRAC><MO>]</MO></MROW></MATH></TH></TR></TBODY></TABLE></SECTION><P>This data from the lab suggests that there is not a high change in RFP for the pDusk system between dark and light conditions. When plotting the computed response of RFP for increasing light intensity in Figure 13 we see, however, that pDusk has a steep decline in RFP levels from complete darkness to very low light intensities. There could be two potential reasons for such a result. First, despite all efforts in creating dark conditions, the experiment could have been contaminated with low light intensities. This would explain why there is no significant change visible in the experimental data between different conditions. To confirm this, additional experiments would need to be conducted. Secondly, a more in-depth parameter sensitivity and robustness analysis could also reveal potential causes of the discrepancies between model simulations and experiments conducted under darkness. This could lead to an understanding of how to engineer the promoters in pDusk to amplify the wanted the response.</P><FIGURE><FIGCAPTION>
Figure 13: RFP Response in pDusk and pDawn per Light Intensity.
</FIGCAPTION></FIGURE><H3>Methods</H3><P>After studying the pDusk and pDawn system, we constructed a parameter estimation procedure by picking random parameter sets based on the <A class="tooltip">latin hypercube<SPAN class="tooltiptext" style="width:700px;">Latin hypercube sampling is a statistical method to sample the parameter space equally. It divides the parameter space according to the number of parameters and sample size and picks a random number from each parameter subspace.</SPAN></A> sampling (LHS) method in Matlab 2015b (used for the whole study). We simulated 10,000 parameter sets and assumed the initial conditions of only y<SUB>DD</SUB> and j<SUB>a</SUB> being in the on state, i.e. we assigned the value one. All other initial conditions were kept at zero. In addition, we needed to transform those parameters to a more biologically feasible distribution, as LHS generates uniformly distributed parameters. For those parameters where there was no information available, we assumed a lognormal distribution that is representative of measured biological rates<SUP><A href="#mb7" id="refmb7">7</A></SUP>. We transformed the two parameters, k<SUB>2</SUB> and k<SUB>3</SUB>, for which experimental data was available according to their known distribution. We then loaded the RFP response data ± standard deviation from Ohlendorf <I>et al</I>.<SUP><A href="#mb1" id="refmb1">1</A></SUP> for both pDusk and pDawn and transformed the light intensities from
<MATH><MROW><MO>[</MO><MO>µ</MO><MI>W</MI><MO>⋅</MO><MI>c</MI><MSUP><MI>m</MI><MROW><MO>-</MO><MN>2</MN></MROW></MSUP><MO>]</MO></MROW></MATH> to <MATH><MROW><MO>[</MO><MO>µ</MO><MI>mol</MI><MO>⋅</MO><MSUP><MI>m</MI><MROW><MO>-</MO><MN>2</MN></MROW></MSUP><MO>⋅</MO><MSUP><MI>h</MI><MROW><MO>-</MO><MN>1</MN></MROW></MSUP><MO>]</MO></MROW></MATH>. For a more detailed explanation see the <A href="https://2016.igem.org/Team:Wageningen_UR/Notebook/OptogeneticKillSwitchModel">notebook.</A></P><P>We then simulated both systems according to our ordinary differential equations (ODEs) with the same parameter sets, and stored the response at 17 hours for later evaluation in the scoring function. We simulated the system for 17 hours, as this is the duration Ohlendorf <I>et al</I>.<SUP><A href="#mb1" id="refmb1">1</A></SUP> used in their study. For the next step, we normalized the computed RFP<SUB>p</SUB> response with the measured response at the highest light intensity used by Ohlendorf <I>et al</I><SUP><A href="#mb1" id="refmb1">1</A></SUP>. A more detailed description of how we constructed the scoring function and the ODEs for both systems can be found here:</P><FIGURE><FIGCAPTION>Click the button to see the detailed methods!</FIGCAPTION></FIGURE><H2 id="lightonbeetoff">Light On, BeeT Off</H2><H3>Modeling The Optogenetic Kill Switch</H3><P>After building the mathematical system for pDusk and pDawn, we were now able to simulate dark and light conditions. We have an understanding of how the optogenetic tool functions and can now explore the parameter space to find parameter sets which describe the wanted behavior and regulation of the toxin-antitoxin system. As there was no experimental data available in literature to score the combined system, we used conservative constraints which are known from literature<SUP><A href="#mb5" id="refmb5">5,</A></SUP><SUP><A href="#mb6" id="refmb6">6</A></SUP> to construct our model for the MazE and MazF kinetics.
The objective is to find sets of parameters which give a response within the conservative constraints and thus show us where in parameter space to search in further analysis to describe the toxin-antitoxin system. As the Biobrick <A href="http://parts.igem.org/Part:BBa_K081007:Design">BBa_K081007</A> is not only used in pDawn, but also in the <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing">quorum sensing system</A>, we wanted to have a tool to decide whether we can also use the <A class="tooltip">less sensitive<SPAN class="tooltiptext" style="width:700px;">Based on data from Ohlendorf <I>et al</I>. (2012), pDusk shows a smaller change in RFP<SUB>p</SUB> ratio than pDawn. For the kill switch this translates into a smaller change in antitoxin over toxin (A/T) ratio and thus a smaller likelihood to kill the cell effectively.</SPAN></A>, but theoretically feasible pDusk system in the kill switch design.
</P><H3>System Design</H3><P>To build the kill switch we first studied the mazEF system and its functionality. The mazEF module is an toxin-antitoxin (TA) system native in <I>E. coli</I>. For the purpose of building a mathematical model, we describe the relevant aspects of the system here. For more background on the mazEF system, please go to <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#Lightswitch">wet-lab part</A>.
In the TA system, mazE represents the antitoxin which forms a dimer and binds two dimers of mazF to built an inactive complex under non-lethal conditions<SUP><A href="#mb5" id="refmb5">5</A></SUP>. The functionality of the system in nature is given by a fast degrading antitoxin and a slower degrading toxin. Once expression of the TA module is terminated, mazE will degrade faster and mazF will start cleaving mRNAs which ultimately leads to cell death. For the kill switch, we will consider either “under”-production of mazE in pDusk or overproduction of mazF in pDawn both under light inducing conditions as shown in Figure 14. </P><FIGURE><FIGCAPTION>Figure 14. Simplified representation of the pDusk + constitutive mazF and pDawn + constitutive mazE system. The arrows indicate activation of a part and the flat bars indicate inhibition of a part.</FIGCAPTION></FIGURE><H3>Constraints</H3><P>For the parameter estimation of the optogenetic kill switch, all the assumptions mentioned in the above sections hold. Further, we assumed a lumped mazEF complex formation, where we disregard potential intermediate stages of complex formation. In addition, we defined the following constraints:
</P><P>As we have literature data supporting that mazF degrades faster than mazE<SUP><A href="#mb5" id="refmb5">5</A></SUP>, we exploited this as a constraint in our parameter generation procedure.</P><LI><P>Another constraint was added to the system as the ratio of free A/T is assumed to be bigger than one<SUP><A href="#mb6" id="refmb6">6</A></SUP> under non-inducing conditions.</P></LI><LI><P>Lastly, we added a logical constraint which describes the underlying biology of the system. Under the highest simulated light intensity, the A/T ratio has to be smaller than one to show that toxin wins over antitoxin and kills the cell.</P></LI><H3>Results</H3><P>We can show that our parameter estimation procedure found two parameter sets, out of 1,000 sampled sets, which satisfy these constraints as shown in Figure 15 and 16.</P><FIGURE><FIGCAPTION>Figure 15. Response of the two parameter sets which satisfy the conservative constraints. The legend in pDusk is also valid for pDawn.</FIGCAPTION></FIGURE><P>The two parameter sets give us an indication where in parameter space we will have to focus on for further analysis. These two sets also both describe a functioning pDusk and pDawn based optogenetic kill switch. Although, one might argue, that set number one (blue and red lines) is to be favoured as the change in mazE in pDusk compared to the change of mazE of set two (yellow and purple lines) in pDusk is higher and thus more likely to function and kill the cell. With this information, an indication can be given on which relative kinetic parameters are needed to make the system work.</P><FIGURE><FIGCAPTION>Figure 16. A/T ratios per light intensity of the two parameter sets which satisfy the constraints. Where blue = set 1 and red = set 2.</FIGCAPTION></FIGURE><P>When plotting the A/T ratio against the increasing light intensities, it becomes more evident that parameter set 1 has a higher change in the A/T ratio within pDusk between different light intensity conditions and is thus to be favoured. To further evaluate these findings, more data from the lab is needed.</P><H3>Methods</H3><P>Adding the mazEF module to the pDusk/ pDawn system requires another 10 parameters to describe the system. We constructed our parameter estimation similar to the parameter estimation of the previously described procedure and evaluated the system based on the constraints described above. It should be noted, that this is a qualitative evaluation and is not backed by statistical real world data. Nonetheless, it can be used as an indication.
We then took the best parameter set from the pDusk/ pDawn evaluation and added the equations describing the interactions with the MazEF system to the set of ODEs. As we noticed that we have to discard around half of the parameter sets because they do not satisfy the constraint that mazE degrades faster than mazF, we built this constraint into the parameter sampling procedure. In addition, we considered the native mazEF system of <I>E. coli</I>. To prevent any potential interaction between the engineered and the native system, a mazEF deficient strain was created in the lab. This is, why we do not consider a native mazEF system in the model.
</P><FIGURE><FIGCAPTION>Click the button to see the detailed methods!</FIGCAPTION></FIGURE><H2 id="insilicoswitch">In Silico Kill Switch</H2><P>We have built a mathematical model of pDusk and pDawn which describes the system behavior. Initial results from the lab, did not show a significant change in response for the pDusk system. However, the experimental results are in agreement with the computed data in the pDawn system. This leads us to suspect that either there was an irregularity with the dark conditions in the lab, or the model does not fully reflect the underlying biology of the system in the lower light intensity regions. We suggest that a parameter sensitivity and robustness analysis could be conducted to find the most likely cause of the mismatch to the data. 
We further conclude, that we achieved a good fit in Figure 12 to the data available in literature<SUP><A href="#mb1" id="refmb1">1</A></SUP>.
We then studied the mazEF toxin-antitoxin integration to design our optogenetic kill switch. After analysis of this system, we found two parameter sets which satisfy the constraints. These parameters give us an idea on how to search the parameter space for further analysis to select functioning kinetic rates in collaboration with the wet-lab.
In addition, future work could show the effect of parameter sensitivity such as leaky promoters and their effect on the system to engineer the pDusk system, adjusting it to suit our needs. We also made the <A href="http://github.com/marioisbeck/iGEM_Wageningen_UR_2016">Matlab code</A> available for future iGEM teams to build optogenetic tools.</P><SECTION id="metabolic"><H1 id="MM_intro">Metabolic Modeling</H1><P>In order to assess the real world viability of BeeT, we evaluated the proposed system of application by making a model of the entire system. To do this we used Flux Balance Analysis (FBA) to model the chassis. The

<A class="tooltip"> chassis <SPAN class="tooltiptext" style="width:300px;">The chassis-organism is the framework that is modified for use in synthetic biology experiments.</SPAN></A>

of BeeT is a variant of <I>​ Escherichia coli</I>, for which it is known that it does not grow overnight in high osmotic pressure environments of <MATH xmlns="http://www.w3.org/1998/Math/MathML"><MN>1</MN><MFRAC><MROW><MI>m</MI><MI>o</MI><MSUB><MI>l</MI><MROW><MI>s</MI><MI>u</MI><MI>c</MI><MI>r</MI><MI>o</MI><MI>s</MI><MI>e</MI></MROW></MSUB></MROW><MI>L</MI></MFRAC></MATH> or above<SUP><A href="#mm1" id="refmm1">1</A></SUP>. 
 
Supplementing an <I>Apis mellifera</I> (honey bee) colony with sugar water is a well established practice amongst beekeepers<SUP><A href="#mm4" id="refmm4">2</A></SUP>. This is usually done with about <MATH xmlns="http://www.w3.org/1998/Math/MathML"><MN>1</MN><MI>k</MI><MI>g</MI></MATH> of table sugar for each liter of <A class="tooltip">table sugar<SPAN class="tooltiptext" style="width:300px;">Chemically speaking this is pure sucrose: C<SUB>12</SUB>H<SUB>22</SUB>O<SUB>11</SUB></SPAN></A>. 
After heating and stirring, this ends up as a concentration of <MATH xmlns="http://www.w3.org/1998/Math/MathML"><MN>625</MN><MFRAC><MSUB><MI>g</MI><MROW><MI>s</MI><MI>u</MI><MI>c</MI><MI>r</MI><MI>o</MI><MI>s</MI><MI>e</MI></MROW></MSUB><MROW><MSUB><MI>L</MI><MROW><MI>W</MI><MI>a</MI><MI>t</MI><MI>e</MI><MI>r</MI></MROW></MSUB></MROW></MFRAC></MATH>. This concentration can also be defined as <MATH xmlns="http://www.w3.org/1998/Math/MathML"><MN>1.82</MN><MFRAC><MROW><MI>m</MI><MI>o</MI><MSUB><MI>l</MI><MROW><MI>S</MI><MI>u</MI><MI>c</MI><MI>r</MI><MI>o</MI><MI>s</MI><MI>e</MI></MROW></MSUB></MROW><MI>L</MI></MFRAC></MATH>, which is almost twice the threshold value at which <I>E. coli</I> would no longer grow. However, in this project we only need the <I>​E. coli</I> to <U>survive</U> in the sugar water. This only needs to be as long as the worker bees require to pick the BeeT-cells up and bring them to where the <I>Varroa destructor</I> are. The question we try to answer is: does our chassis survive in the sugar water, and if so, for how long? </P><H2 id="MM_model">Modeling <I>E. coli</I> Survival</H2><P>In order to make this model a technique called <A href="http://www.nature.com/nbt/journal/v28/n3/full/nbt.1614.html"> Flux Balance Analysis</A> (FBA) was used to describe the relationship between the metabolism of the <I>E. coli</I> and the osmotic pressure of the sugar water. FBA is a mathematical method that can be used for predicting reaction fluxes and optimal growth rates of species using genome-scale reconstructions of their metabolic networks. In this case we used it to specifically predict the effect of water efflux on the total amount of ATP the cell has available for maintenance. The model used as a base for this analysis was <A href="http://bigg.ucsd.edu/models/iJO1366"> iJO1366</A><SUP><A href="#mm3" id="refmm3">3</A></SUP> from the <A href="http://bigg.ucsd.edu/"> BIGG database</A>. We modified the model to represent an <I>E. coli</I> in a sugar water-like environment using 
<A href="http://cobrapy.readthedocs.io/">CobraPy</A><SUP><A href="#mm5" id="refmm5">4</A></SUP>.
We also changed the objective of the optimization such that the ATP-maintenance reaction should be maximized. Then we ran the model 1000 times whilst sequentially increasing the lower bound of the water efflux exchange reaction, the results of which can be seen in figure 17. </P><FIGURE><FIGCAPTION>Figure 17: The relationship between the maximum ATP available for survival for an <I>E. coli</I> in a sugar-water environment and the incrementally increased water efflux. </FIGCAPTION></FIGURE><P>The relationship between maximum ATP available for survival and water efflux is shown in Figure 17 the results from the metabolic model suggest that there is a linear relationship between ATP availability and water efflux. This implies that if no water is available for ATP used for maintenance, the cell will die. When the model runs without any modification, <I>i.e.</I> in an environment where it is in the exponential growth phase, an ATP Maintenance flux of <MATH xmlns="http://www.w3.org/1998/Math/MathML"><MN>3.15</MN><MO> </MO><MFRAC><MROW><MI>m</MI><MI>m</MI><MI>o</MI><MI>l</MI></MROW><MROW><MI>g</MI><MI>D</MI><MI>W</MI><MO>⋅</MO><MI>h</MI></MROW></MFRAC></MATH> is given as output by the model.</P><P>We do not know the exact amount of ATP needed for survival in sugar water conditions, but because of the results from Figure 17 we can start looking at the relationship between survival time and water efflux.</P><FIGURE><FIGCAPTION>Figure 18: The relationship between the maximum ATP available for survival for an <I>E. coli</I> in a sugar-water environment and the theoretical survival time, given a constant water efflux over this time and a starting volume of 2.8e-13 grams<SUP><A href="#mm2" id="refmm2">5</A></SUP>. The various coloured lines indicate water tolerance thresholds for the <I>E. coli</I>.
</FIGCAPTION></FIGURE><P> In Figure 18 we can see not only the relationship of survival time against max ATP available for survival, but also how different thresholds of minimal cell-water tolerance would affect this relationship. The minimal cell-water tolerance threshold gives the value at which percentage of the remaining cell-water reaches a state that will cause cell-death. This has a drastic effect on survival time, changing 20 minutes of maximum survival time to a mere ~90 seconds in the worst case scenario.</P><H2 id="MM_discussion">Beyond 90 Minutes</H2><P>Figure 17 shows us that osmotic pressure alone can indeed have an effect on cell regulation and cell death, and from Figure 18 it appears that the minimal water allowance threshold has a high impact on range of possible times. Our model is limited in that it predicts an infinite survival time beyond 90 minutes. This suggests that our model may be missing some form of regulation that allows for longer survival times. To understand this effect, we conducted an experiment to test this prediction. This experiment can be found <A href="igem.2016.org/Team:Wageningen_UR/ecoli_survival"> here</A>. What we can say is that if the cells can survive for longer outside of this period, then they must have enough ATP available for basic maintenance. If cell death occurs,  other processes than pure water-efflux must be the cause of that. This could be due to a combination of lacking nutrients and water-efflux, or over production of osmolytes to keep the balance. </P></SECTION><SECTION id="beehave"><H1>Beehave</H1><P> Due to regulatory and experimental hurdles it is difficult to test the effectiveness of BeeT in combating <I>Varroa destructor</I>. We would still like to be able to give advice to local beekeepers on the ideal application strategy of BeeT, based on several scenarios. To accomplish this, the previously published model beehave was adapted to include the effects of BeeT on <I>Varroa</I> mite and virus dynamics in simulated colonies. Beehave is an open-source agent-based model, which can be used to examine the multifactorial causes of Colony Collapse Disorder<SUP><A href="#rh1" id="refrh1">1,</A><A href="#rh2" id="refrh2">2</A></SUP>.
It consists of several modules which control aspects like foraging, <I>Varroa</I> mite dynamics, and colony growth as shown in Figure 19, and was extensively tested for robustness and realism<SUP><A href="#rh1" id="refrh1">1</A></SUP>. 

BeeT can be transported to the hive by adding it in sugar water or bee bread. Both these sources have their advantages and disadvantages. In this project we have used <I>Escherichia coli</I> as our model chassis, which can be applied to <A href="https://2016.igem.org/Team:Wageningen_UR/Model#MM_discussion">sugar water</A>. Sugar water is supplemented to support a colony during honey harvest before winter, as a substitute for nectar<SUP><A href="#rh3" id="refrh3">3,</A></SUP><SUP><A href="#rh4" id="refrh4">4</A></SUP>.  
Supplementing honey bees with sugar water is a well established and familiar practice among beekeepers<SUP><A href="#rh3" id="refrh3">3</A></SUP>. Alternatively, reconstructing the same system in <I>Lactobacillus</I> species would allow us to use BeeT in bee bread. Bee bread has some advantages over sugar water, as it it is more frequently transported to larvae. Bee bread is a combination of pollen, regurgitated nectar, and glandular secretions. To protect against harmful micro-organisms honey bees also inoculate bee bread with fermenting bacteria<SUP><A href="#rh5" id="refrh5">5</A></SUP>. The fermentation process reduces the pH 
. 
</P><H2 id="beetmodule">BeeT Module</H2><P> Beehave is an open-source GNU licensed agent-based model utilizing  <A class="tooltip"> NetLogo <SPAN class="tooltiptext" style="width:300px;"> NetLogo is an open source, agent-based modeling platform widely used in education and research. </SPAN></A>, and consists of several interlocking modules which each model different aspects of the beehive. It models the wide variety of stresses affecting honey bees<SUP><A href="#rh1" id="refrh1">1</A></SUP>. 
As such, it is the ideal basis for our investigation into the effects of BeeT on <I>Varroa</I> mites and honey bee dynamics. Beehave has several modules covering colony dynamics, foraging, and a <I>Varroa</I> mite model as depicted in Figure 19. Two viruses are also included in the model: deformed wing virus (DMV), and acute paralysis virus (APV), both for which <I>Varroa</I> mites are a vector. Our BeeT module, which runs parallel to beehave, is capable of modeling transport of BeeT into the hive using sugar water or bee bread. It also calculates how much BeeT is transported to larvae based on consumption of honey and pollen stores. The <I>Varroa</I> mite mortality is determined by the amount of BeeT near larvae when <I>Varroa</I> mites emerge from brood cells. This in turn affects <I>Varroa</I> mite population levels in the hive, reducing virus loads in the hive, and allowing colony survival.
</P><FIGURE><FIGCAPTION>
Figure 19: The honey bee colony model includes <I>Varroa</I> mite dynamics and virus epidemiology, agent-based foraging behavior with either pre-defined landscape definitions or a representation of local floral patterns. It is also possible to include weather patterns to more accurately model local conditions. Note that the model includes various interdependent mortalities and other parameters which are not included in this figure. Figure from Becher et al.<SUP><A href="#rh1" id="refrh1">1,</A></SUP>.
</FIGCAPTION></FIGURE><H2 id="modelcons">Model Considerations</H2><P>
Not everything is known about the BeeT model and as such, it was necessary to make certain assumptions. Some of these are related to BeeT in general, while others are specific to the sugar water or bee bread applications. We will first discuss general limitations of our model, and then address each of the applications separately.</P><P><B>General Assumptions:</B> several properties of BeeT and its chassis are currently unknown. This includes the degradation rate of BeeT, both inside and outside the hive. With both treatments BeeT encounters different environments while being transported to larvae. For sugar water,  BeeT is exposed to the honey stomach of bees, honey stores, and brood cells, whereas bee bread encounters pollen and brood cells. We do not know the effects of these different environments on degradation rates of BeeT. For this reason we assumed that degradation rates are stable inside and outside the hive. Another uncertainty is what the effect of BeeT is on <I>Varroa</I> mite mortality. We modeled this by using a <A class="tooltip"> saturating <SPAN class="tooltiptext" style="width:300px;"> The chosen saturating function smoothly approaches one and is used to calculate mite mortality.</SPAN></A> function. We also assumed that the effect of BeeT on <I>Varroa</I> mite mortality is entirely determined by the amount of BeeT present at larvae when a brood cell is <A class="tooltip"> capped <SPAN class="tooltiptext" style="width:300px;"> Nursing bees cover up brood cells to protect larvae, also called capping. </SPAN></A>.

Finally, the beehave model is only able to model a single virus: DMV or APV. We are unable to model the combined effects of both viruses on a bee colony. For all analyses we used the DWV virus, as it is more harmful to honey bee colony survival than APV<SUP><A href="#rh8" id="refrh8">8</A></SUP>. </P><P><B>Sugar Water:</B> we know that, based on the <A href="https://2016.igem.org/Team:Wageningen_UR/ecoli_survival">experimental results</A> obtained by metabolic modeling of <I>E. coli</I>, and it is able to survive in sugar water for at least 48 hours. Additionally, <A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic">modeling</A> predicts that if <I>E. coli</I> is capable of surviving 90 minutes, it can survive indefinitely. 

Additionally, we estimate that in sugar water there are approximately <MATH xmlns="http://www.w3.org/1998/Math/MathML"><MSUP><MN>10</MN><MN>6</MN></MSUP><MO> </MO><MFRAC><MROW><MI>c</MI><MI>e</MI><MI>l</MI><MI>l</MI><MI>s</MI></MROW><MROW><MI>m</MI><MI>l</MI></MROW></MFRAC></MATH>. This estimate is based on the assumption that we add saturated medium (<MATH xmlns="http://www.w3.org/1998/Math/MathML"><MSUP><MN>10</MN><MN>8</MN></MSUP><MO> </MO><MFRAC><MROW><MI>c</MI><MI>e</MI><MI>l</MI><MI>l</MI><MI>s</MI></MROW><MROW><MI>m</MI><MI>l</MI></MROW></MFRAC></MATH>) containing BeeT to sugar water and diluted this by a factor of 100. The dilution is performed to avoid rejection of sugar water by bees, since it would be too contaminated to consume.

We based the hourly uptake rate of sugar water on a paper by Avni <I>et al</I><SUP><A href="#rh9" id="refrh9">9</A></SUP>, but it is possible that this is not the maximal uptake rate of sugar water. In the measured time period all of the sugar water was taken up by the colony, suggesting that the uptake rate is theoretically greater than the value proposed by Avni <I>et al</I><SUP><A href="#rh9" id="refrh9">9</A></SUP>.</P><P><B>Bee Bread:</B> we assume that the amount of <I>Lactobacillus</I> cells in bee bread is roughly equal to the number of <MATH xmlns="http://www.w3.org/1998/Math/MathML"><MFRAC><MROW><MI>C</MI><MI>F</MI><MI>U</MI></MROW><MROW><MI>m</MI><MI>l</MI></MROW></MFRAC></MATH> in yoghurt
<SUP><A href="#rh10" id="refrh10">
10
</A></SUP>.
Artificial bee breads can be made using yoghurt and are readily accepted by bees<SUP><A href="#rh10">10</A></SUP>. Additionally, we assume that the removal rate of artificial bee bread is equal to <MATH xmlns="http://www.w3.org/1998/Math/MathML"><MN>22.7</MN><MFRAC><MROW><MI>g</MI></MROW><MROW><MI>d</MI><MI>a</MI><MI>y</MI></MROW></MFRAC></MATH><SUP><A href="#rh4">4</A></SUP>.  
Based on Sumpter <I>et al.</I> we know that the uptake rate depends on the manner in which bee bread is applied<SUP><A href="#rh9">9</A></SUP>.
We chose to base the uptake rate of bee bread on Brodschneider <I>et al.</I>, since their experimental setup is similar to Avni <I>et al</I><SUP><A href="#rh4">4,</A></SUP><SUP><A href="#rh10">10</A></SUP>.</P><H2 id="howcanhoneybeessurvive">How Can Honeybees Survive?</H2><P>
The functionality of the BeeT module is primarily based on literature research and as such, requires a relatively low amount of parametrization. The main unknown quality is how BeeT will behave, namely degradation and its effectiveness at combating <I>Varroa</I> mites nearby larvae. We are interested in how BeeT can best be applied given certain assumptions. As such we are interested in the best period to apply BeeT, either in autumn or during spring, so we can give recommendations to beekeepers. We also examine the difference in effectiveness between BeeT in sugar water and in bee bread for both periods. This can inform future work on whether it is beneficial to adapt BeeT for functionality in bee bread. There are three additional parameters related to BeeT that can be varied and upon which effectiveness relies. These are: degradation of BeeT in the hive, outside the hive, and the effect of BeeT on <I>Varroa</I> mite mortality. Each is varied across a range of values. Additionally, for every combination we have five replicates to show the variance in our results due to stochastic processes. We divided the analyses into four treatments, each with the same three parameter ranges and five replicates per combination of parameters. This can be seen in Table 3. </P><FIGURE><FIGCAPTION>
Table 3: Structure of BeeT module analysis.
</FIGCAPTION></FIGURE><TABLE><TBODY><TR><TH>
       Period
        </TH><TH>
       Treatment
        </TH><TH>
        Range
        </TH></TR><TR><TH rowspan="2">
      1 April -  15 June Spring
        </TH><TD>
       Sugar water
        </TD><TD><I>Varroa</I> mite mortality, degradation in-hive and degradation outside-hive
        </TD></TR><TR><TD>
          Bee bread
        </TD><TD><I>Varroa</I> mite mortality, degradation in-hive and degradation outside-hive
        </TD></TR><TR><TH rowspan="2">
        1 Sep. - 15 Nov.Autumn
        </TH><TD>
        Sugar Water
        </TD><TD><I>Varroa</I> mite mortality, degradation in-hive and degradation outside-hive
  </TD></TR><TR><TD>
          Bee bread
        </TD><TD><I>Varroa</I> Mite Mortality, degradation in-hive and degradation outside-hive
   </TD></TR></TBODY></TABLE><P>
Each combination of period and treatment has the same 5169 parameter set per ranges. We compared the autumn period with the spring period for the different ranges and treatments. In each case the spring period was significantly more effective in combating <I>Varroa</I> mites than the autumn period. For further analysis we continued with the spring period. 

Three scenarios were identified: the colony dies due to virus load, the colony survives at a lower population level, or the colony thrives. Every year the beehave model checks the population after winter. If the population is below 4000 worker bees the colony is presumed dead and the population is set to zero. To differentiate between the scenarios we looked at the mean population of worker bees after winter averaged over 10 years. Included in this average are the years where the bee population is zero. If the average is below 3000, we assume that the colony dies within a few years and consider this colony death. If the mean population after winter is between 3000 and 5000, the colony survives but its population is reduced due to <I>Varroa</I> mite pressure. Above 5000 worker bees, the colony is thriving and the initial infestation of <I>Varroa</I> mites rapidly dies off. Each parameter combination for sugar water and bee bread falls within one of these three scenarios. For both treatments we chose a parameter set which is representative for a certain scenario. From this we divide parameter sets in three categories: death, survival, and thriving. As can be seen in Table 4, the spring period is more effective for both sugar water and bee bread. Using bee bread is advisable when BeeT has a limited effect on mite mortality. If BeeT has a large impact on mite mortality it is more beneficial to use sugar water. This is because beekeepers are already comfortable applying sugar water to hives which reduces the risk of mistakes. </P><FIGURE><FIGCAPTION> Table 4: Parameter sets are divided into three categories: colony death, survival, and thriving. If colonies can survive and thrive with higher degradation of BeeT (in-hive and outside the hive) and a lower effect of BeeT on <I>Varroa</I> mite mortality, it indicates a more effective treatment.</FIGCAPTION></FIGURE><TABLE><TBODY><TR><TH>
         Period and treatment
        </TH><TH>
         Colony death
        </TH><TH>
        Colony survival
        </TH><TH>
      Colony thriving
      </TH></TR><TR><TH>
        Sugar water, spring
        </TH><TD>
         6,6%
        </TD><TD>
         80,6%
        </TD><TD>
      12,8%
      </TD></TR><TR><TH>
          Bee bread, spring
          </TH><TD>
          0%
          </TD><TD>
      2,9%
      </TD><TD>
      97,1%
      </TD></TR><TR><TH>
        Sugar water, autumn
        </TH><TD>
        15,1%
        </TD><TD>
      80,7%
</TD><TD>
        4,2%
        </TD></TR><TR><TH>
          Bee bread, autumn
          </TH><TD>
     0%
    </TD><TD>
      57,6%
      </TD><TD>
      42,4%
      </TD></TR></TBODY></TABLE><P>
In the case of sugar water a far lower fraction of BeeT ends up at larvae, and most is consumed by worker bees. This is likely the main reason for the difference in effectiveness for sugar water and bee bread.

For sugar water, parameter sets were chosen which fall in the categories survival, and thriving. These are further analysed by examining the population dynamics of worker bees and <I>Varroa</I> mites over time. In Figure 20B and Figure 20C these are shown, respectively, for a colony barely surviving a <I>Varro</I> mite infestation,  and a thriving colony, regardless of the presence of <I>Varroa</I> mites at the start of the simulations. When the colony is barely surviving we can see that worker bee and <I>Varroa</I> mite populations reach a dynamic equilibrium. For comparisons sake we also show in figure 20A with population dynamics when BeeT.
</P><FIGURE><FIGCAPTION>
Figure 20: The honey bee population is shown in blue and the <I>Varroa</I> mite population in red. A: Colony rapidly declines when no BeeT is present. Starting population is 20 <I>Varroa</I> B: Colony barely survives <I>Varroa</I> mite infestation. Shows <I>Varroa</I> mite in red and worker bee population in blue. Starting population is 20 <I>Varroa</I>. C: Colony thrives regardless of <I>Varroa</I> mite infestation. Starting population is 20 <I>Varroa</I> mites and rapidly declines. D: Colony thrives regardless of heavy <I>Varroa</I> mite infestation. Starting population is 10,000 <I>Varroa</I> mites.
</FIGCAPTION></FIGURE><P>
When the colony is thriving, the starting <I>Varroa</I> mite population rapidly declines and disappears as shown in Figure 19C. In the Figure you can see that the <I>Varroa</I> mite population is never able to establish itself. This indicates that the treatment is effective for a low population of initial <I>Varroa</I> mites. We further analysed a situation with a starting population of 10,000 <I>Varroa</I> mites which is shown in Figure 20D. Even with a higher starting population and a high initial virus load, we can see that the treatment is successful in reducing <I>Varroa</I> mite populations and that the colony recovers.</P><H3>How Was The Model Analysed?</H3><P>The beehave model utilizes multiple number generators to calculate mortality amongst <I>Varroa</I> mites and bees. These generators increase the variance inherent in the model and, as such, it is advisable to use multiple replicates per parameter set. To determine the error bars we ran a simulation where we varied the <I>Varroa</I> mite mortality, and plot the mean worker bee population over 10 years with 10 replicates per value of <I>Varroa</I> mite mortality. This was done for the first version of the beehave model, which governs year round <I>Varroa</I> mite mortality when they emerge from a brood cell.
<FIGURE><FIGCAPTION>
Figure 21: Mean worker bee population over 10 years plotted against year round <I>Varroa</I> mite mortality at emergence from brood cell.
</FIGCAPTION></FIGURE></P><P>
With 10 replicates the variance is relatively low, particularly when the mean worker bee population is high. To analyse the full BeeT module we chose to use 5 replicates. This is a compromize between <A class="tooltip"> simulation time <SPAN class="tooltiptext" style="width:300px;"> Over 110,000 parameter sets were simulated. </SPAN></A> and reduction of variance.  

We first separated the results of the analyses of the full BeeT module based on the three scenarios. The results can be seen in Figure 22. Within this 2D graph the axis are either degradation in the hive or degradation outside the hive, and impact of BeeT on <I>Varroa</I> mite mortality. Additionally, the mean population of worker bees surviving winter is color coded. We can see that the highest mean population is in the bottom right corner with the scaling factor affecting the results strongly. Additionally, degradation inside the hive has a greater impact on mean population than degradation outside the hive. This can be seen by comparing the top right corner of both panels. </P><FIGURE><FIGCAPTION>
Figure 22: 2D graph with either degradation in the hive or degradation outside the hive, and impact of BeeT on <I>Varroa</I> mite mortality on its axes. Results are color coded with mean population of bees after winter. The left panel has degradation outside the hive on its y-axis, degradation inside the hive is averaged out for each point. The right panel has degradation inside the hive on its y-axis, degradation outside the hive is averaged out for each point.
</FIGCAPTION></FIGURE><H2 id="beetbehave">BeeT Beehave - Discussion and Conclusions</H2><P>
Currently it is unclear how effective BeeT will be at combating <I>Varroa</I> mites since, primarily, toxicity is difficult to measure in the lab and deploying BeeT in the field has many regulatory issues. For these reasons, we built a BeeT module for the well known beehave model to assess the minimum effectiveness for certain treatments. We also examined two different periods for administering the BeeT: autumn and during spring. We chose autumn to analyse the effect of BeeT on the <I>Varroa</I> mite population since honey bee populations are at their most fragile during the winter following this period. This way we strengthen the honey bees since winter is coming. Additionally, it is normal for current beekeeper practices to supplement hives with sugar water during autmn and, thus, applying BeeT at this time would provide a minimal amount of disturbance for beekeepers. 
The advantage of administering BeeT during spring is that this is the period at which most of the <I>Varroa</I> mite population is present at larvae. By administering during this period we can ensure that BeeT is able to affect a large percentage of the <I>Varroa</I> mite population. Beekeepers also visit hives regularly during this period as they harvest honey in spring. Recent papers indicate that feeding of artificial bee bread during this period can have significant positive effects on honey bee health<SUP><A href="#rh6">6,</A><A href="#rh7">7</A></SUP>. For both sugar water and bee bread the ideal period for administering BeeT is during the spring due to the large difference in effectiveness between the two periods. In a future application of BeeT this can be given as a recommendation to beekeepers. This is a practice which can be easily adopted since beekeepers already visit hives regularly in the spring period. In particular, supplementing sugar water at this time is already an accepted practice among beekeepers.

The main difference between sugar water and bee bread is that a larger fraction of total in-hive BeeT is given to larvae for bee bread. This is not surprising as honey stores are primarily used to support energetic needs of worker bees<SUP><A href="#rh1">1</A></SUP>. As such only a small fraction is transported to larvae. On the other hand, the protein contained in bee bread is crucial for the development of larvae and, therefore is mainly given to larvae to support their growth<SUP><A href="#rh1">1</A></SUP>. Based on our results we expect that BeeT will be effective using sugar water but greater effectiveness can be accomplished by engineering BeeT for compatibility with bee bread.
</P><H3>Modeling Methods</H3><P>
We examined the effect of year round mite mortality by adding the following code to beehave:</P><P>which calculates the mortality of <I>Varroa</I> mites when they emerge from the brood cell (mortalityMites). In beehave the mortality of <I>Varroa</I> mites is encapsulated in the fecundity of female <I>Varroa</I> mites. This means that mortalityNoBeeT is zero. When the mite mortality ScalingVariable is large, the mortalityMites function approaches one. 
The amount of <I>Varroa</I> mites emerging from a cell is calculated with the variable ‘healthyMitesInSingleCell’. Using a Poisson distribution, some of the <I>Varroa</I> mites emerging from the brood cell die. Both versions of the BeeT module only affect the <I>Varroa</I> mite population when <I>Varroa</I> mites emerge from a brood cell. This was done to reduce the chance of introducing bugs into the original beehave model.
The initial code for mortalityMites was changed to:</P><P>so that it is dependent on the concentration of BeeT in the cell at the time of capping. Additionally, we included a procedure BeeTProc which is called daily. It handles BeeT moving from outside to inside the hive at a set rate per hour. It also handles degradation: both hourly degradation outside the hive and daily degradation inside the hive. When <I>Varroa</I> mites invade, the amount of BeeT present at larvae is saved for each invading <I>Varroa</I> mite in the variable BeeTLarvaeInvasion. Whenever a <I>Varroa</I> mite emerges from a cell its mortality is calculated based on BeeTcell, which is dependent on BeeTLarvaeInvasion. The total amount of BeeT moving to larvae is based on the amount of sugar water and pollen consumed by larvae. If BeeT is consumed by worker bees it is considered lost and is removed from the BeeT in-hive store. 
All variables added to beehave can be found below, including how they are calculated, if applicable what their initial value is, units, in what procedure they are used, and any comments or references. The original variables of the beehave model can be found <A href="https://static.igem.org/mediawiki/2016/e/e1/T--Wageningen_UR--BEEHAVE_variables.pdf">here</A>.</P><FIGURE><FIGCAPTION>Click the button to see the detailed methods!</FIGCAPTION></FIGURE></SECTION><H1>References</H1><SECTION id="references"><OL class="references"><H2>References for Population Dynamics</H2><A id="fn1" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3485958/">1.</A> Eri Nasuno, Nobutada Kimura, Masaki J. Fujita, Cindy H. Nakatsu, Yoichi Kamagata, and  Satoshi Hanada (2012). Phylogenetically Novel LuxI/LuxR-Type Quorum Sensing Systems Isolated Using a Metagenomic Approach
 Vol, 78, number 22. <A href="#ref1" title="Jump back to footnote 1 in the text.">↩</A><A id="fn2" href="http://www.sciencedirect.com/science/article/pii/S0009250912007075">2.</A> Nazanin Saeidi, Mohamed Arshath, Matthew Wook Chang, Chueh Loo Poh, Characterization of a quorum sensing device for synthetic biology design: Experimental and modeling validation. Volume 103, 15 November 2013, Pages 91–99 <A href="#ref2" title="Jump back to footnote 2 in the text.">↩</A><A id="fn3" href="http://www.ibc7.org/article/journal_v.php?sid=265">3.</A> Pearson, B. Bacterial Hash Function Using DNA-Based XOR Logic Reveals Unexpected Behavior of the LuxR Promoter, IBC 2011, vol. 3, article no. 10, pp. 1-10
<A href="#ref3" title="Jump back to footnote 3 in the text.">↩</A><A id="fn4" href="http://www.sciencedirect.com/science/article/pii/S1878778911000275">4.</A> Sergi Abadal Ian F. Akyildiz (2011). Automata modeling of Quorum Sensing for nanocommunication networks, Volume 2, Issue 1, Pages 74–83 <A href="#ref2" title="Jump back to footnote 4 in the text.">↩</A><A id="fn5" href="http://parts.igem.org/wiki/index.php/Part:BBa_K199052">5.</A> Designed by: Kin Lau Group: iGEM09_MoWestern_Davidson   (2009-07-22) <A href="#ref5" title="Jump back to footnote 5 in the text.">↩</A><A id="fn6" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3697643/">6.</A> Bokinsky, Edward E. K. Baidoo, Swetha Akella, Helcio Burd, Daniel Weaver, Jorge Alonso-Gutierrez, Héctor García-Martín, Taek Soon Lee, and Jay D. Keasling(2011).HipA-Triggered Growth Arrest and β-Lactam Tolerance in Escherichia coli Are Mediated by RelA-Dependent ppGpp Synthesis 2013 Jul; 195(14): 3173–3182. <A href="#ref6" title="Jump back to footnote 6 in the text.">↩</A><A id="fn7" href="http://bioinformatics.oxfordjournals.org/content/25/15/1923.full.pdf">7.</A> Raue et al. Structural and practical identifiability analysis of partially observed dynamical models by exploiting the profile likelihood. 
Vol. 25 no. 15 2009, pages 1923–1929 <A href="#ref7" title="Jump back to footnote76 in the text.">↩</A><A id="fn8" href="http://www.qpcrupdate.info/bengtsson-singlecell-2005.pdf">8.</A> Bengtsson et al. Gene expression profiling in single cells from the pancreatic islets of Langerhans reveals lognormal distribution of mRNA levels volume 15 page 1388–1392(2005) <A href="#ref8" title="Jump back to footnote 8 in the text.">↩</A><H2>References for Optogenetic Kill Switch</H2><A id="mb1" href="http://www.sciencedirect.com/science/article/pii/S0022283612000113">1.</A> Ohlendorf, Robert, Roee R. Vidavski, Avigdor Eldar, Keith Moffat, and Andreas Möglich. 2012. “From Dusk till Dawn: One-Plasmid Systems for Light-Regulated Gene Expression.&quot; Journal of Molecular Biology 416(4):534–42. Retrieved (http://dx.doi.org/10.1016/j.jmb.2012.01.001) <A href="#refmb1" title="Jump back to footnote 1 in the text.">↩</A><A id="mb2" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3527124/">2.</A> Möglich, Andreas, Rebecca A. Ayers, and Keith Moffat. 2009. “Design and Signaling Mechanism of Light-Regulated Histidine Kinases.&quot; Journal of Molecular Biology 385(5):1433–44. Retrieved (http://dx.doi.org/10.1016/j.jmb.2008.12.017). <A href="#refmb2" title="Jump back to footnote 2 in the text.">↩</A><A id="mb3" href="http://www.nature.com/articles/nplants201590">3.</A> Klose, Cornelia <I>et al</I>. 2015. “Systematic Analysis of How Phytochrome B Dimerization Determines Its Specificity.&quot; Nature Plants 1(July):15090.<A href="#refmb3" title="Jump back to footnote 3 in the text.">↩</A><A id="mb4" href="http://dx.doi.org/10.1038/nplants.2015.90\n10.1038/nplants.2015.90">4.</A> Rausenberger, Julia <I>et al</I>. 2010. “An Integrative Model for Phytochrome B Mediated Photomorphogenesis: From Protein Dynamics to Physiology.&quot; PLoS ONE 5(5).<A href="#refmb4" title="Jump back to footnote 4 in the text.">↩</A><A id="mb5" href="http://jcs.biologists.org/content/118/19/4327">5.</A> Engelberg-Kulka, Hanna, Ronen Hazan, and Shahar Amitai. 2005. “mazEF: A Chromosomal Toxin-Antitoxin Module That Triggers Programmed Cell Death in Bacteria.&quot; J. Cell Sci. 118:4327–32.<A href="#refmb5" title="Jump back to footnote 5 in the text.">↩</A><A id="mb6" href="http://www.ncbi.nlm.nih.gov/pubmed/23781105">6.</A> Limpert, Eckhard, Werner a. Stahel, and Markus Abbt. 2001. “Log-Normal Distributions across the Sciences: Keys and Clues.&quot; BioScience 51(5):341.<A href="#refmb6" title="Jump back to footnote 6 in the text.">↩</A><A id="mb7" href="http://bioscience.oxfordjournals.org/content/51/5/341.extract">7.</A> Fasani, Rick a and Michael a Savageau. 2013. “Molecular Mechanisms of Multiple Toxin-Antitoxin Systems Are Coordinated to Govern the Persister Phenotype.&quot; Proceedings of the National Academy of Sciences 110(27):E2528–37.<A href="#refmb7" title="Jump back to footnote 7 in the text.">↩</A><H2>References for Metabolic Modeling</H2><A id="mm1" href="http://www.microbiology.ubc.ca/sites/default/files/roles/drupal_ungrad/JEMI/15/JEMI15_15-21.pdf">1.</A> Cheng, Y. L., Hwang, J., &amp; Liu, L. (2011). The Effect of Sucrose-induced Osmotic Stress on the Intracellular Level of cAMP in Escherichia coli using Lac Operon as an Indicator. Journal of Experimental Microbiology and Immunology (JEMI) Vol, 15, 15-21. 
<A href="#refmm1" title="Jump back to footnote 1 in the text.">↩</A><A id="mm4" href="http://link.springer.com/article/10.1007%2FPL00001720">2.</A> DeGrandi-Hoffman, G., &amp; Hagler, J. (2000). The flow of incoming nectar through a honey bee (Apis mellifera L.) colony as revealed by a protein marker. Insectes Sociaux, 47(4), 302-306. 
<A href="#refmm4 " title="Jump back to footnote 2  in the text.">↩</A><A id="mm3" href="http://www.ncbi.nlm.nih.gov/pubmed/21988831">3.</A> Orth, J. D., Conrad, T. M., Na, J., Lerman, J. A., Nam, H., Feist, A. M., &amp; Palsson, B. Ø. (2011). A comprehensive genome‐scale reconstruction of Escherichia coli metabolism—2011. Molecular systems biology, 7(1), 535.
<A href="#refmm3 " title="Jump back to footnote 3  in the text.">↩</A><A id="mm5" href="http://cobrapy.readthedocs.io/">4.</A> Ebrahim, A., Lerman, J. A., Palsson, B. O., &amp; Hyduke, D. R. (2013). COBRApy: constraints-based reconstruction and analysis for python. BMC systems biology, 7(1), 74.
<A href="#refmm5" title="Jump back to footnote 4  in the text.">↩</A><A id="mm2" href="http://bionumbers.hms.harvard.edu/bionumber.aspx?&amp;id=100011">5.</A> Neidhardt F.C. Escherichia coli and Salmonella: Cellular and Molecular Biology. Vol 1. pp. 15, ASM Press 1996
<A href="#refmm2" title="Jump back to footnote 5 in the text.">↩</A><H2>References for Beehave</H2><A id="rh1" href="http://onlinelibrary.wiley.com/doi/10.1111/1365-2664.12222/epdf"> 1.</A> M. A. Becher, V. Grimm, P. Thorbek, J. Horn, P. J. Kennedy, and J. L. Osborne, (2014). BEEHAVE: A systems model of honeybee colony dynamics and foraging to explore multifactorial causes of colony failure, Journal of Applied Ecology., vol. 51, no. 2, 470–482. <A href="#refrh1" title=" Jump back to footnote 1 in the text.">↩</A><A id="rh2" href="http://onlinelibrary.wiley.com/doi/10.1111/oik.02636/epdf">2.</A> J. Horn, M. A. Becher, P. J. Kennedy, J. L. Osborne, and V. Grimm (2015), “Multiple stressors: Using the honeybee model BEEHAVE to explore how spatial and temporal forage stress affects colony resilience,&quot; Oikos, no. September 2015, 1001–1016. <A href="#refrh2" title="Jump back to footnote 2 in the text.">↩</A><A id="rh3" href="http://journal.unaab.edu.ng/index.php/JNSET/article/view/1378/1271">3.</A> Fasasi, K A (2011) &quot;Cumulative effect of sugar syrup on colony size of honeybees, <I> Apis mellifera adansonii </I> (Hymenoptera : apidae) in artificial beehives &quot; Journal of Natural Sciences, Engineering and Technology ed. 10: 33-43.<A href="#refrh3" title="Jump back to footnote 3 in the text.">↩</A><A id="rh4" href="http://www.apidologie.org/articles/apido/pdf/2010/03/m09120.pdf">4.</A> Robert Brodschneider, Karl Crailsheim (2010) &quot;Nutrition and health in honey bees&quot; Apidologie ed. 41: 278-294<A href="#refrh4" title="Jump back to footnote 4 in the text.">↩</A><A id="rh5" href="http://www.apidologie.org/articles/apido/pdf/2010/03/m09120.pdf">5.</A> Ellis, Amanda M. Hayes, Jr, G. W. (2009) &quot;An evaluation of fresh versus fermented diets for honey bees (<I>Apis mellifera</I>).&quot; Journal of Apicultural Research 48: 215-216<A href="#refrh5" title="Jump back to footnote 5  in the text.">↩</A><A id="rh6" href="http://onlinelibrary.wiley.com/doi/10.1111/1758-2229.12169/epdf">6.</A> Basualdo, Marina Barragán, Sergio Antúnez, Karina (2014) &quot;Bee bread increases honeybee haemolymph protein and promote better survival despite of causing higher <I>Nosema ceranae</I> abundance in honeybees&quot; Environmental Microbiology Reports 6: 396-400<A href="#refrh6" title="Jump back to footnote 63  in the text.">↩</A><A id="rh7" href="http://www.researchgate.net/publication/250278863_Pollen_substitutes_increase_honey_bee_haemolymph_protein_levels_as_much_as_or_more_than_does_pollen">7.</A> De Jong, David (2009) &quot;Pollen substitutes increase honey bee haemolymph protein levels as much as or more than does pollen&quot; Journal of Apicultural Research Reports 48: 34-37<A href="#refrh7" title="Jump back to footnote 7  in the text.">↩</A><A id="rh8" href="http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2656.2004.00776.x/pdf"> 8.</A> Sumpter, D. J T, Martin, S. J., (2004). The dynamics of virus epidemics in <I>Varroa</I>-infested honey bee colonies, Journal of Animal Ecology., vol. 73, is. 1, 51-63. <A href="#refrh8" title="Jump back to footnote 8  in the text.">↩</A><A id="rh9" href="http://onlinelibrary.wiley.com/doi/10.1111/1365-2664.12222/epdf"> 9.</A> Sumpter, D. J T, Martin, S. J., (2004). The effect of surface area of pollen patties fed to honey bee (<I>Apis mellifera</I>) colonies on their consumption, brood production and honey yields., Journal of Apicultural Research., vol. 48, is. 1, 23-28. <A href="#refrh9" title="Jump back to footnote 9  in the text.">↩</A><A id="rh10" href="http://onlinelibrary.wiley.com/doi/10.1111/1365-2664.12222/epdf"> 10.</A> Ellis, Amanda M. Hayes, Jr, G. W., (2009). An evaluation of fresh versus fermented diets for honey bees (<I>Apis mellifera</I>)., Journal of Apicultural Research., vol. 48, is. 3, 215-216. <A href="#refrh10" title="Jump back to footnote 10  in the text.">↩</A></OL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2016. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV></DIV></BODY></HTML>