<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Notebook_QuorumSensing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Notebook/QuorumSensing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Wageningen_UR/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><TITLE>Wageningen UR iGEM 2016 </TITLE><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><BUTTON class="hamburger">☰</BUTTON><BUTTON class="cross">˟</BUTTON><DIV class="mobilemenu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><B>Overview</B></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Team</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Notebook</A></LI></UL></DIV><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="#0">Quorum Sensing</A></H4></DIV><LI class="menu-item"><A href="#1">General</A></LI><LI class="menu-item"><A href="#2">May 9 - July 3</A></LI><LI class="menu-item"><A href="#3">July 4 - July 17</A></LI><LI class="menu-item"><A href="#4">July 25 - Aug. 7</A></LI><LI class="menu-item"><A href="#5">Aug. 8th -  Aug. 21st</A></LI><LI class="menu-item"><A href="#6">Aug. 22th -  Sep. 4th</A></LI><LI class="menu-item"><A href="#7">Sep. 5th -  Sep. 18th</A></LI><LI class="menu-item"><A href="#8">Sep. 19th -  Oct. 2nd</A></LI><LI class="menu-item"><A href="#9">Oct. 3th -  Oct. 5th</A></LI><LI class="menu-item"><A href="#10">Primer list</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design">Applied Design</A></LI></UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"><DIV class="ca-content">WetLab</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Assay"><I>In vitro</I> assay</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#ToxinEngineering">Toxin Engineering</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates" class="up_menu_fix"><I>Varroa</I> Isolates</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#DetectingMites" class="up_menu_fix">Detecting Mites</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing" class="up_menu_fix"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#Toggleswitch" class="up_menu_fix">Toggle Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/ecoli_survival"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#Lightswitch" class="up_menu_fix">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#SAA" class="up_menu_fix">Cas9 Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model"><DIV class="ca-content">Model</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#light">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#beehave">Beehave</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">Toxin Scanner</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Vision">Our Vision</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#collaboration">Collaboration</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#events">Events</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"><FONT size="1">Integrated Practices</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/contact">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#environment">Environment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#laboratory">Laboratory</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Journals</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments">Protocols</A></LI></DIV></DIV></DIV><DIV class="center_content"><SECTION id="1"><H1>Quorum sensing</H1><P>When cells or bacteria are described in the population dynamics notebook, those bacteria were <I>E. coli</I> DH5alpha.
Unless stated otherwise, all actions performed for the wetlab construction and testing of the population dynamics subproject were done by Thomas.
Where Arabinose and/or Glucose are mentioned, we refer to L-Arabinose and D-Glucose respectively in concentrations of 2 g/L. 
Where sequencing of DNA is mentioned we refer to GATC lightrun tube sanger sequencing. Where it is mentioned that samples were sent for sequencing - unless stated otherwise - the sequencing results corresponded to the expected sequence. </P><H2>Plate reader measurements: </H2><P>The machine used for plate reader experiments is BioTek’s SynergyMx. Gen5 2.01 software was used to acquire the data and export these to microsoft excel files which were further organized with Microsoft Excel 2011. All plate reader measurements were done in black 96-wells plates with a transparent bottom. During measurements the lid was on the plate in all cases to prevent mixing of sample due to shaking during incubation. All reported experiments were performed at 37℃ with fast continuous shaking. Measurements described were always at least 10 minutes apart to allow for proper shaking of the samples and allow for sufficient aeration. GFP activity was measured with fluorescence measurements with an excitation at 485nm and emission at 510nm. mRFP activity was measured for excitation at 584nm and emission at 607nm. Bandwidth for fluorescence was always 9nm. Wells contained 200µl cultures. Absorbance was measured at both 600nm and 660nm to assess bacterial growth. Absorbance data for bacterial cultures were corrected with the absorbance of medium only wells. This caused absorbance to start at 0 for all cultures, the first measurements notwithstanding. Fluorescence values were corrected for the fluorescence of medium only wells. This sometimes caused negative values for the fluorescence of cultures at low OD<SUB>600</SUB> values. This of course it not realistic and was considered an artifact caused by fluctuations in the measurement of low level fluorescence. </P><H2>Flow Cytometry</H2><P>The machine used for flow cytometry in Wageningen is the Flow cytometry sorter - BD FACS AriaTM III. This machine is in principle also able to sort cells, but we did not make use of this function. Marcel H. Tempelaars, Specialist Flow cytometry sorter - BD FACS AriaTM III at Wageningen UR operated the controls while Thomas was responsible for sample handling and insertion. Marcel meanwhile explained the functioning of the technique. </P><P>During the flow cytometry analysis we found that the choice of mRFP as reporter was a good one: we observed a clear difference in fluorescence between the mRFP producing cells and cells that did not have a <I>mRFP</I> gene. Differentiation between GFP expressing cells and cells unable to produce GFP is known to be more troublesome. Therefore we advise future iGEM team planning to do flow cytometry of FACS to (if at all possible) use mRFP as reporter rather than GFP.</P><H2>Fluorescence Microscopy</H2><P>The microscope used for fluorescence microscopy of the subpopulation clones is an Olympus BX41. The laser used to excite fluorophores is Excelitas Technologies X-cite series 120. A lumenera infinity 3 was used to capture the images. Steven Aalvink, technician at Wageningen UR molecular ecology, instructed me in the use of the microscope and other hardware.</P></SECTION><SECTION id="2"><H1>May 9 - July 3:</H1><H2>General</H2><P>Belwina, Marijn and Thomas transformed biobrick parts from the registry to home-made chemically competent DH5alpha cells. They spread the transformations on LB agar plates with antibiotics corresponding to the resistance in the backbone of each plasmid. Transformed parts (table 1). We isolated (Machery-Nagel nucleospin kit) the plasmids from liquid cultures of these transformation plates.  </P><FIGURE><FIGCAPTION>Table1. Biobrick transformations</FIGCAPTION></FIGURE></SECTION><SECTION id="3"><H1>July 4 - July 17:</H1><H2>General</H2><P>We verified the length (through taq PCR) and sequence (through GATC lightrun sanger sequencing) of the biobrick isolated (May 9 - July 3) plasmid inserts. We also made -80℃ glycerol stocks for cells containing the parts.</P><H2>434- and λ cI repressor operon construction</H2><P>We designed primers with overhangs for amplification of biobricks in Benchling for subsequent Gibson assembly (GA) and ordered those primers from NEB:
Primers designed for parts: BBa_B0015, BBa_C0052, BBa_I0500, BBa_K081007 and pSB1C3. 
Primers:897 - 908. </P><P>We used primers 897 - 906 for Q5 PCR to amplify the parts for GA. We was unable to amplify pSB1C3 with the flanked primers through numerous attempts. We then realized that (because of the overhangs in the other fragments) the fragments all already contain 20 + bp overlap with each other required for GA. We chose to Q5 PCR amplify pSB1C3 with primers suffix-forward and prefix-reverse and use this PCR product with the previously mentioned PCR products for the other fragments. </P><P>We did a first attempt to Gibson assemble BBa_K1913007 using (unpurified) PCR reactions of BBa_B0015, BBa_C0052, BBa_I0500, BBa_K081007 and pSB1C3 mentioned before. We used NEB GA mastermix and followed the corresponding protocol. We used 0.2 to 2.0 µl of 50 ng/µl dilutions of the PCR product to use 75 ng of pSB1C3 and 2.5 as much moles of the other fragments, except for B0015 where 5 times the backbone concentration was used. After assembly, we transformed the mixture to electro competent cells prepared beforehand by Belwina. The plates were kindly transferred from 37℃ to 4℃ by Alex (supervisor). </P><P>We did a colony taq PCR of 48 of the resulting colonies to screen for successful assembly clones. We found one clone (‘colony 35’) that showed a substantially bigger insert than the background found in the other colonies and in the negative control. </P><FIGURE><FIGCAPTION>Figure 1. Colony PCR gel 434 GA (1st try) transformation.</FIGCAPTION></FIGURE><P>We made -80℃ glycerol stocks and isolated the plasmid from ‘colony 35’ using the Machery-Nagel plasmid isolation kit. We did a verification PCR on the isolated plasmid and found that the insert size is shorter than it should be. </P></SECTION><SECTION id="4"><H1>July 25 - Aug. 7 </H1><H2>434- and λ cI repressor operon construction</H2><P>We sent ‘colony 35’ for GATC lightrun Sanger sequencing. The sequencing revealed that all fragments except BBa_K081007 were assembled in pSB1C3. We designed (and ordered from NEB) primers with overhangs for amplification for a second step of GA to include BBa_K081007 in the construct. 
Primers: 109 - 110.</P><P>We used the primers for Q5 PCR to amplify BBa_K081007 and the plasmid from ‘colony 35’ for GA. Both reactions yielded band of the expected sizes in agarose gel electrophoresis. We purified the reactions using the Machery-Nagel kit for PCR purification and eluted in milli-Q water. </P><P>We used the purified PCR reaction in GA with NEB HiFi mastermix according to the corresponding protocol. We saved part of the reaction after 15m at 50℃ and left the rest to react for 60m 50℃. We purified the GA reaction with the Zymo Research DNA clean and concentrator kit. We transformed 5µl the 15m reaction to NEB chemically competent cells according to the manufacturer's protocol. Lisa transferred the plates from 37℃ to 4℃.</P><H2>mRFP cI protein balance reporter construction</H2><P>We designed primers with overhangs for amplification of biobricks in Benchling for subsequent GA and ordered those primers from NEB:
Primers designed for parts: BBa_I12006, BBa_K081007, BBa_K081014 and pSB1C3. 
Primers: 113 - 118. </P><P>We used the primers for Q5 PCR to amplify the fragments for GA. All reactions yielded band of the expected sizes in agarose gel electrophoresis. pSB1C3 again could only successfully be amplified with primers suffix-forward and prefix-reverse. We purified the reactions using the Machery-Nagel kit for PCR purification and eluted in milli-Q water. </P><P>We used the purified PCR reaction in GA with NEB HiFi mastermix according to the corresponding protocol. We saved part of the reaction after 15m at 50℃ and left the rest to react for 60m 50℃. We purified the GA reaction with the Zymo Research DNA clean and concentrator kit. We transformed 5µl the 60m reaction to NEB chemically competent cells according to the manufacturer's protocol. Lisa transferred the plates from 37℃ to 4℃.</P><H2>Quorum sensing GFP reporter construction</H2><P>We designed two flanked forward primers to elongate (in two PCR steps) the 5’ end of BBa_K081012 with the sequence of promoter BBa_R0062. We ordered the primers from IDT. </P><P>We did Q5 PCR on BBa_K081012 with the primers 111 and VR, got a band of the expected size (around 1kb). We purified the PCR product with the Machery-Nagel PCR purification kit. We then used a dilution hereof for a seconds step with purification with primers 112 and VR. We also amplified (Q5 PCR) pSB1C3 with primers suffix-forward and prefix-reverse to linearize the backbone. Both PCRs yielded bands of the expected sizes in agarose gel electrophoresis. We then purified both PCR products with the Machery-Nagel kit and protocol. </P><H2>Quorum sensing 2 plasmid system construction</H2><P>We Q5 PCR amplified BBa_K546000 and pSB4K5 with primers VF2 + VR and suffix-f + prefix-r. Both PCRs yielded bands of the expected size on agarose gel. We purified both PCR products with the Machery-Nagel PCR purification kit and corresponding protocol. </P><P>We, then started digestion of 1µg of both purified products with EcoRI-HF and PstI-HF, both from NEB and according to the NEB digestion protocol 15m at 37℃. We purified both digestion products with the Machery-Nagel PCR purification kit and protocol.</P><P>We started ligation with NEB T4 according to the NEB T4 ligase protocol. We used 50ng of pSB4K5 and 3 times as much moles of the BBa_K546000 insert. Ligation took place overnight at 16℃. We then purified the ligation reaction with the Machery-Nagel PCR purification kit according to the manufacturer's protocol.</P><P>The ligation was transformed using homemade electrocompetent cells. The transformations were spread on LB agar plates with Kanamycin. The transformation yielded plates with moderate amount of colonies. </P></SECTION><SECTION id="5"><H1>Aug. 8th -  Aug. 21st</H1><H2>434- and λ cI repressor operon construction</H2><P>Transformation of the GA for assembly of the 434- and λ cI repressor operon plasmid yielded many colonies. We screened for clones with the correct insert through taq colony PCR with primers biobrick-forward and biobrick-reverse. Prior to colony PCR, the colonies were grown in 40µl LB medium inside a 96-wells plate for 30m. 1µl of the medium was used as template for the colony PCR. All screened colonies were found to contain an insert of the correct size. </P><FIGURE><FIGCAPTION>Figure 2. Colony PCR gel 434- and λ cI repressor operon GA (1st try) and mRFP cI protein balance reporter GA transformations.</FIGCAPTION></FIGURE><P>We chose two colonies, grew those overnight and isolated the plasmid with the Machery-Nagel PCR plasmid isolation kit and corresponding protocol. We again verified the presence an insert with the correct length through taq PCR on a dilution of the isolated plasmids with primers biobrick-forward and biobrick-reverse. We also send the isolated plasmid to GATC for sequencing. The sequencing revealed that the insert was indeed correct although there was no coverage of a small part of the insert. </P><H2>mRFP cI protein balance reporter construction</H2><P>Transformation of the GA for assembly of the mRFP cI protein balance reporter operon plasmid yielded many colonies. We screened for clones with the correct insert through taq colony PCR with primers biobrick-forward and biobrick-reverse. Prior to colony PCR, the colonies were grown in 40µl LB medium inside a 96-wells plate for 30m. 1µl of the medium was used as template for the colony PCR. Almost all colonies displaying red color on the agar plate, were found to contain an insert of the correct size.</P><H2>Quorum sensing 2 plasmid system construction</H2><P>1µg of the purified GFP reporter and pSB1C3 PCR products were digested with NEB enzymes  EcoRI-HF and PstI-HF, both from NEB and according to the NEB digestion protocol 20m at 37℃. We purified both digestion products with the Machery-Nagel PCR purification kit and protocol.</P><P>We then used the purified digestions in a ligation with NEB T4 according to the NEB T4 ligase protocol. We used 50ng of pSB1C3 and 3 times as much moles of the of the GFP reporter insert. Ligation was done for 10m at room temperature. The ligation mixture was purified with the Machery-Nagel PCR purification kit and protocol.</P><P>The purified ligation was used to transform NEB chemically competent cells according to the manufacturer’s protocol. The transformations were plated on LB agar plates with chloramphenicol. These plates yielded moderate amounts of colonies. Taq Colony PCR identified two colonies as containing a plasmid with the correct insert size. </P><P>The plasmids of these two colonies were isolated from 10ml LB cultures using the Machery-Nagel plasmid isolation kit and protocol. The isolated plasmids were verified to contain an insert of the correct size with taq PCR with primer combination biobrick-forward and biobrick-reverse. </P><P>We also isolated the plasmid from overnight cultures of BBa_K546000 - pSB4K5 with the Machery-Nagel plasmid isolation kit and protocol. Taq PCR of the prepped plasmid confirmed the presence of an insert of the size corresponding with the BBa_K546000 biobrick. </P><P>We made electrocompetent cells from a 100ml culture of the GFP reporter transformant. These cells were used to transform the BBa_K546000 - pSB4K5 plasmid. These cells, containing both the quorum sensing system and reporter on separate plasmids are from here called the ‘2 plasmid quorum sensing system’. Transformation through electroporation. This transformation yielded colonies on the agar plate that are green to the naked eye. </P></SECTION><SECTION id="6"><H1>Aug. 22th -  Sep. 4th</H1><H2>Quorum sensing 2 plasmid system testing</H2><P>Started an overnight plate reader experiment for the 2 plasmid quorum sensing system cells. Cells were grown in LB with the appropriate selection antibiotic(s). Measurements were done for wells with only the medium, medium with the reporter only cells, medium with BBa_K546000 - pSB4K5 cells and for cells with the 2 plasmid quorum sensing system. For all bacterial samples, three serial dilutions were measured and for each dilution 4 technical replicate wells were measured. We measured absorbance and GFP fluorescence. This experiment was performed three times on different days with different starting cultures to provide biological replicates. All replicates showed the same general pattern of sharp fluorescence/OD<SUB>600</SUB> increase when the OD<SUB>600</SUB> increased. We confirmed the presence of both BBa_K546000 and the GFP reporter BBa_K1913014 through taq PCR with primers VF2 and VR. After gel electrophoresis, the PCR product showed 2 bands of the lengths corresponding to the envisioned inserts. </P></SECTION><SECTION id="7"><H1>Sep. 5th -  Sep. 18th</H1><H2>Subpopulation plasmid construction</H2><P>Q5 PCR of 434, lambda cI plasmid with primers VF2 and VR. This yielded a strong specific band of the expected size. Subsequent purification of the PCR product with the Zymo clean-up kit. Subsequent digestion with NEB enzymes, according to NEB protocol. 
434, lambda cI PCR product: EcoRI-HF and SpeI-HF
mRFP cI protein balance reporter plasmid: XbaI and EcoRI-HF
Reaction of 2h at 37℃. Subsequent Zymo DNA purification. Ligated the purified digestions (3:1) ratio according to NEB T4 ligase protocol. 16℃ overnight. Zymo DNA purification. Transformation of NEB chemically competent DH5alpha cells with 5µl ligation mix. Transformation according to NEB protocol and plated on Cm plates. </P><P>This resulted in the growth of light-red colonies on the plates. Taq colony PCR of 16 red colonies with primers p899 and p696 revealed several colonies with inserts of the expected size. Three of these colonies were selected to be grown overnight and subsequent plasmid isolation with the Thermo Fisher kit for plasmid isolation. </P><P>‘Round the horn PCR (Q5) on the 434, lambda cI, mRFP reporter plasmid to include the RBS library and construct the complete subpopulation plasmid. Primers 907 (degenerate primer) and p433. 25µl reactions. 61℃ annealing, 5m30s elongation. PCR yielded a specific band of the expected size. The PCR product was purified with the Zymo DNA purification kit. T4 blunt end ligation (the primers are phosphorylated) overnight at room temperature. </P><P>Transformation of blunt end ligation with NEB chemically competent DH5alpha cells. This yielded over 300 colonies, some differing in the intensity of red coloring. </P><H2>Quorum sensing 1 plasmid system construction</H2><P>Q5 PCR of the quorum sensing reporter (BBa_K1913014) with primers VF2 &amp; VR. The PCR yielded a specific band. Zymo DNA purification of the PCR product. Digested using NEB enzymes. 
Quorum sensing reporter BBa_K1913014: EcoRI-HF and SpeI.
BBa_K546000 in pSB4K5: EcoRI-HF and XbaI.
3h incubation at 37℃. The digestions were then purified with the Zymo DNA purification kit. Subsequently, the purified digestions were ligated with T4 ligase according to the NEB protocol. Ligation at 16℃ overnight. The ligation mix was transformed to NEB chemically competent cells. The transformation were plated on Kan plates.</P><P>The transformation (now BBa_K1913005 in pSB4K5) yielded green (and some white) colonies. Taq colony PCR indicated that all tested green colonies contain an insert of the correct insert. Selected 3 green colonies for overnight growth and subsequent plasmid isolation with the Thermo Fisher kit for plasmid isolation. The plasmids were sent for GATC lightrun tube sanger sequencing.</P></SECTION><SECTION id="8"><H1>Sep. 19th -  Oct. 2nd</H1><H2>Subpopulation testing</H2><P>96 colonies from the transformed subpopulation plasmid were re streaked on one agar plate in a structured way.</P><FIGURE><FIGCAPTION>Figure 3. Close-up picture of culture plate with colonies containing the subpopulation two-operon plasmid (including the RBS library). The blue circled and (faintly) red circled colonies were selected to be used in separate plate reader experiments.</FIGCAPTION></FIGURE><P>We started a preliminary plate reader experiment with 95 of the subpopulation plasmid containing colonies. The plate was grown in LB with Arabinose overnight at 20℃ while shaking overnight. Glucose was added to 2 g/L to each well. We then measured OD<SUB>600</SUB> and RFP fluorescence. </P><P>The data from this experiment show that the majority of the colonies display roughly the same pattern of mRFP production during growth after Glucose addition. Roughly 4 hours after Glucose addition an increase in fluorescence/OD<SUB>600</SUB> is seen for almost all colonies. </P><P>We chose to further analyze a selection of the colonies that either showed a unique response or represented a general response found in many colonies. For these colonies we compared the development of mRFP activity between samples grown in only LB with the selection antibiotic, on the same medium with L-Arabinose and on medium with L-Arabinose and Glucose. </P><P>According to our expectation, glucose repression should lead to a subset of cells starting to produce substantial levels of mRFP. Therefore, addition of glucose should increase the total mRFP activity in the population. This is not clearly found in our results, but the development of mRFP activity in colonies B6, C10, E11 and H9 preliminarily suggests an increase in fluorescence in samples with Glucose starting after approximately 11 hours relative to the samples lacking Glucose. </P><P>Up to now, we only described experiments that measure the population-wide response in mRFP activity. Of course, subpopulations could only really be observed when fluorescence is assessed for individual cells. To this end we performed fluorescence microscopy on colonies B6, C10, E11 and H9, comparing samples grown in LB (with selection antibiotics) with L-Arabinose to samples where we also added D-Glucose. In an effort to visualize the number of cells that show fluorescence among the other cells, we chose to make pictures where the cells were exposed to mRFP excitation with laser light, but also to low levels of white-light. Pictures were taken after 10m and 4h20m after glucose addition. From the investigated sample, clone H9 seemed to give more mRFP producing cells after glucose addition, but this difference is already seen after 10m, a time too short to allow for the envisioned mechanism to change protein levels. Therefore we have little faith in the validity of these preliminary results.</P><FIGURE><FIGCAPTION>Figure x. Fluorescence microscopy ‘overlay’ images 4.5 hours after glucose addition. Shown is fluorescence imaging of microscopy slides covered with a cover glass.</FIGCAPTION></FIGURE><P>Attempted to measure the subpopulations with flow cytometry in Leiden with Paul van Heusden, Assistant professor Institute of Biology, Leiden. Unfortunately we were unable to differentiated between medium only and bacteria grown in medium, even using other <I>E. coli</I> samples present in the laboratory in Leiden. They have no experience using flow cytometry on bacteria, but are experienced in analysis of yeasts. We thank Paul van Heusden for his effort and time.



</P><H2>Quorum sensing 1 plasmid system construction and testing</H2><P>Digested BBa_K1913005 in pSB4K5 with EcoRI-HF and PStI-HF and used the same enzymes to digest pSB1C3. Used NEB enzymes and protocol. Purified digestion with Zymo kit for DNA purification. Ligated with T4 according to NEB protocol. Again purified using Zymo kit. Belwina transformed (2 days later) the mixture to NEB chemically competent cells according to the NEB protocol and plated on Cm agar plates. </P><P>Four green colonies from the transformation plates were used to test the functioning of the plasmid in an overnight plate reader experiment: results show in ‘population dynamics’ section of the wetlab results. The results indicated that 3 of the 4 colonies showed the expected response to population density. After sequence confirmation of the insert, we submitted a plasmid isolation of 1 colony to the registry (BBa_K1913005).</P></SECTION><SECTION id="9"><H1>Oct. 3th -  Oct. 5th</H1><H2>Subpopulation testing</H2><P>For flow cytometry in Wageningen I was allowed to bring a limited number of samples. Therefore I chose to use 2 subpopulation clones B6 and C10 as well as 2 controls: one constitutive mRFP expresser and one strain of DH5alpha unable of mRFP production.
The samples were inoculated in the morning in LB with arabinose and Cm as selection antibiotic. At 21:00 the same day, 1-5µl of the samples were transferred to fresh LB with Cm35, arabinose and both with and without glucose. The volume used to refresh the samples differed to provide roughly equal starting densities. </P><P>The next day, refreshed the cultures in fresh medium with the same composition as 21:00 the day before. We analyzed both samples for each: the samples from 21:00 and the refreshed cultures for those. We were easily able to differentiate between background in the medium and cells in the actual cultures. We related different values to each other including forward scatter, side scatter, fluorescence intensity and cell count. We found that the size of the cells correlated to the red fluorescence. The constitutive mRFP expressors - used as positive control - show a high mean fluorescence. The cells containing the subpopulation system have a lower, but substantial mean fluorescence per cell. At first glance it seem that the subpopulation system causes a more diverse population in mRFP expression, but this could well be an artifact because of the logarithmic scale on the x-axis. 
</P></SECTION><SECTION id="10"><H1>Primer List</H1><FIGURE><FIGCAPTION>Table 2. Primer list for the population dynamics subproject. All primers are displayed with the 5’-end on the left and the 3’-end on the right. The codes in the 2nd column are referred to in the text.</FIGCAPTION></FIGURE></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2016. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV></DIV></BODY></HTML>