<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Description_Specificity skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Description/Specificity</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Wageningen_UR/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><TITLE>Wageningen UR iGEM 2016 </TITLE><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><BUTTON class="hamburger">☰</BUTTON><BUTTON class="cross">˟</BUTTON><DIV class="mobilemenu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><B>Overview</B></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Team</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Notebook</A></LI></UL></DIV><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="#header">Specificity</A></H4></DIV><LI class="menu-item"><A href="#Assay"><I>In Vitro</I> Assay</A></LI><LI><A href="#invitro1">High throughput method</A></LI><LI><A href="#invitro2">Analysis of BBMVs</A></LI><LI><A href="#invitro3">Fluorophore Leaking</A></LI><LI><A href="#invitro4">Obtaining Kinetic Values</A></LI><LI class="menu-item"><A href="#ToxinEngineering">Toxin Engineering</A></LI><LI><A href="#engineering1">Binding site mutagenesis</A></LI><LI><A href="#engineering2">Phage display</A></LI><LI class="menu-item"><A href="#Isolates"><I>Varroa</I> Isolates</A></LI><LI><A href="#Isolates1">Staining Isolates</A></LI><LI><A href="#Isolates2"><I>In Vivo</I> Toxicity</A></LI><LI><A href="#Isolates3">16s + SDS-PAGE</A></LI><LI><A href="#Isolates4">Proteomics</A></LI><LI class="menu-item"><A href="#references">References</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design">Applied Design</A></LI></UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"><DIV class="ca-content">WetLab</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Assay"><I>In vitro</I> assay</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#ToxinEngineering">Toxin Engineering</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates" class="up_menu_fix"><I>Varroa</I> Isolates</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#DetectingMites" class="up_menu_fix">Detecting Mites</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing" class="up_menu_fix"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#Toggleswitch" class="up_menu_fix">Toggle Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/ecoli_survival"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#Lightswitch" class="up_menu_fix">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#SAA" class="up_menu_fix">Cas9 Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model"><DIV class="ca-content">Model</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#light">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#beehave">Beehave</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">Toxin Scanner</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Vision">Our Vision</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#collaboration">Collaboration</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#events">Events</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"><FONT size="1">Integrated Practices</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/contact">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#environment">Environment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#laboratory">Laboratory</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Journals</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments">Protocols</A></LI></DIV></DIV></DIV><DIV class="center_content"><H1 style="text-align:center;">Specificity</H1><P> To improve on existing methods, BeeT should affect <I>Varroa</I> mites only. This means it should be harmless to bees, humans and other organisms. In our search for a suitable toxin we looked into Cry toxins, also named Bt toxins, which are produced by <I>Bacillus thuringiensis</I>. Due to interaction with receptors in the gut membrane, these toxins are known to be very species-specific. To find a suitable Cry toxin we tried isolating <I>B. thuringiensis</I> from dead mites, as well as mutating an existing Cry toxin. For testing the potential toxins that were obtained, we employed an <I>in vitro</I> toxicity assay with brush border membrane vesicles made from <I>Varroa</I> mites. The vesicles contain fluorophores. When a Cry toxin is applied that is toxic to <I>Varroa</I> mites it binds the receptors on the vesicles, rupturing them and releasing the fluorophores.  </P><SECTION id="Assay"><H1><I>In Vitro</I> Assay</H1><H2 id="invitro1"><B>A High Throughput Method to Test Cry Toxin Activity
</B></H2><P>Cry toxins have been found to have activity towards insects of different phylogenetic orders. In short, a functional Cry protein is only active when specific binding occurs to the gut membrane of the target insect. In more detail, these receptors have been found in the gut epithelium cells, where the Cry proteins polymerize binding to its membrane receptor forming pores. After the formation of these ion channels, the osmotic pressure is disturbed resulting in cell death of the target host<SUP><A href="#jz1" id="refjz1">1,</A></SUP><SUP><A href="#jz2" id="refjz2">2</A></SUP>.Although a number of Cry toxins have been (partly) characterized, a toxin specific to <I>Varroa destructor</I> has yet to be found. For this reason, we developed an <I>in vitro</I> toxicity test in order to be able to efficiently screen Cry toxins on their activity against <I>V. destructor</I>. From the <I>V. destructor</I>, Brush Border Membrane Vesicles (BBMV's) were made and loaded with the fluorophore 6-carboxyfluorescein to test the pore formation ability of Cry proteins, see Figure 1a. As a proof of principle, BBMV's from the gut of <I>Tenebrio molitor</I> were made and loaded with 6-carboxyfluorescein to test the pore formation ability of Cry3Aa, which is known to be toxic to <I>T. molitor</I> larvae<SUP><A href="#jz3" id="refjz3">3,</A></SUP><SUP><A href="#jz4" id="refjz4">4</A></SUP>.
<FIGURE><FIGCAPTION>Figure 1. Schematic overview of the function of the <I>in vitro</I> assay to the activity of Cry proteins. (a) When active Cry toxins are mixed with BBMV's incorporated with fluorophores , (b) they will bind to their receptors and create pores into the membrane. (c) The thereby induced leaking of fluorophores results in an increase in fluorescence.
</FIGCAPTION></FIGURE></P><P><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#assay1">BBMV's</A> are composed of phospholipid double layers originating from midgut membrane cells. This membrane therefore should also contain the membrane proteins of the midgut membrane.  After <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#assay2">incorporation of 6-carboxyfluorescein</A> into the BBMV's the presence of an active Cry toxin can principally be detected as followed: the Cry toxins will bind to their highly specific receptors<SUP><A href="#jz5" id="refjz5">5</A></SUP> and consequently create holes into the phospholipid double membrane layer, see Figure 1b. After this event, the fluorophores leak out of the vesicles due to diffusion, which can be measured as an increase in fluorescence, see Figure 1c. This method has been proven to work for the activity of Cry3Aa on <I>Leptinotarsa decemlineata</I><SUP><A href="#jz6" id="refjz6">6</A></SUP>. <I>Rausell et al.</I> only demonstrated how the magnitude of the fluorescence increase links to toxicity, but this method has its limitations. BBMV's have the disadvantage of being unstable and undergoing degradation<SUP><A href="#jz7" id="refjz7">7</A></SUP>. In other words, measurements on BBMV's can be influenced by spontaneous breaking of the vesicles. In our project, we took this into account by relating toxicity activity to a kinetic value.The increase of fluorescence due to fluorophore leaking can be attributed to <A class="tooltip"> dequenching<SPAN class="tooltiptext" style="width:300px;"> Dequenching refers to any process which increases the fluorescence intensity of a given substance. </SPAN></A>. 6-Carboxyfluorescein is a self-quenching fluorophore. At a high concentration 6-carboxyfluorescein forms dimers. Because of dimer-dimer and dimer-monomer interactions, the fluorescence of a carboxyfluorescein molecule decreases drastically<SUP><A href="#jz8" id="refjz8">8</A></SUP>. The vesicles created in our study contain a high concentration of fluorophores, which results in self-quenching behavior. When the fluorophores leak out of the vesicles, the local concentration of fluorophores decreases. Hence, dequenching occurs and the fluorescence increases. </P><H2 id="invitro2"><B>Analysis of Brush Border Membrane Vesicles</B></H2><P>To visualize the presence of BBMV's, images were made with the use of <A class="tooltip"> TEM<SPAN class="tooltiptext" style="width:300px;">Transmission Electron Microscopy </SPAN></A>. This was <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#assay3">done</A> in collaboration with the iGEM team from Delft <FIGURE><FIGCAPTION>Figure 2. TEM images of BBMV's. The images of <I>V. destructor</I> vesicles were made by the iGEM team from Delft.(a) <I>T. molitor</I>        (b) <I>V. destructor</I></FIGCAPTION></FIGURE></P><P>Figure 2 shows the vesicles - round shapes with sizes between the 100 nm and 300 nm. To gain more information about the composition of the samples that are visualized above, <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#assay4">dynamic light scattering</A> was performed. The dynamic light scattering results (Figure 3) give information about the size distribution of the particles that are present in the solution.</P><FIGURE><FIGCAPTION>Figure 3. The size distribution of particles in solution of BBMV's obtained from <I>T. molitor</I> and <I>Varroa destructor</I> calculated with data obtained by dynamic light scattering.</FIGCAPTION></FIGURE><P> Looking at both transmission electron microscopy results and dynamic light scattering results, the <I>T. molitor</I> samples have a large population of vesicles that vary their size around 200 nm. Looking at the <I>V. destructor</I> samples, the population around this size is smaller. To learn more about the proteins in these samples, click <A href=" https://2016.igem.org/Team:Wageningen_UR/Notebook/InVitroAssay#appendices">here</A>.</P><H2 id="invitro3"><B>Fluorophore Leaking Experiments</B></H2><P>The hypothesis is that due to addition of a detergent, the phospholipid bilayer will rearrange. This process will subsequently give the fluorophores the chance to escape their enclosed space and disperse in the solution. The dispersion of the fluorophores into the solution would then cause an increase in fluorescence as described previously.To determine whether these BBMV's were capable of releasing fluorophores and whether this release could be measured, the following was done: to BBMV's containing carboxyfluorescein, 1 volume percent of SDS (sodium dodecyl sulphate) was added to disturb their stability. Meanwhile the fluorescence was <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#assay6">measured</A> over time. Looking at Figure 4, after addition of SDS (detergent), an increase in fluorescence is indeed visible. However, the same increase in fluorescence can be observed over time with vesicles that are in an environment without SDS. Two possible explanations for this phenomenon are that BBMV's are already porous or are not stable under the circumstances during measurement. Both processes would result in an increase in fluorescence over time. In order to compare induced fluorophore leaking from standard fluorophore leaking, the speed of the carboxyfluorescein leaking should be calculated and compared.</P><FIGURE><FIGCAPTION> Figure 4. (a) The fluorescence of a solution with BBMV's incorporated with fluorophores and a solution with free fluorophores was measured over time. After 330 seconds, detergent was added. (b) The fluorescence of a two solutions with BBMV's incorporated with fluorophores was measured over time. To one sample detergent was added. Both graphs were made from data obtained from BBMV's from <I>T. molitor</I>. For both graphs a similar effect was visible with <I>V. destructor</I> BBMV's.
</FIGCAPTION></FIGURE><H2 id="invitro4"><B>Obtaining Kinetic Values</B></H2><P> To determine whether a specific Cry protein can induce fluorophore release, Cry3Aa was <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#assay5">isolated</A> from <I>B. thuringiensis</I> and tested on BBMV's obtained from <I>T. molitor</I>. Activated Cry3Aa gives a clear band on a gel of a 68 kDa protein after SDS-PAGE was performed, as described by Rausell <I>et al</I><SUP><A href="#jz6" id="refjz6">6</A></SUP>. The SDS-PAGE gel of the protein extract from <I>B. thuringiensis</I>, shown in Figure 5, shows such a band of approximately this size.</P></SECTION><FIGURE><FIGCAPTION>Figure 5. SDS-PAGE gel. M represents the marker lane. In the other three lanes, protein extracts of <I>B</I>. thuringiensis are loaded. Lane 1 has an undiluted sample, lane 2 a 10x diluted sample and lane 3 a 100x diluted sample.</FIGCAPTION></FIGURE><P><I>T. molitor</I> BBMV's were incubated with and without Cry3Aa and the fluorescence was <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#assay6">measured</A> over time(Figure 6). From this graph it can be concluded that the presence of Cry3Aa results in a faster increase in fluorescent and therefore in a faster fluorophore release. To determine how large the role of induced leaking is, the kinetic properties of both measurements have to be compared. In order to compare these properties, it should be known which kinetic rules the fluorophore leaking follows. Plotting the natural logarithm of the concentration of fluorophores entrapped in vesicles against time, gives a straight line, (Figure 6). The concentration of fluorophores entrapped in vesicles is expressed as the difference of the maximum fluorescence and the fluorescence at a certain time point. The observed straight line in this graph tells us that first order reaction rules can be applied on the measurement<SUP><A href="#jz9" id="refjz9">9</A></SUP>. This means that equation 1 can be applied to the obtained data.</P><P>The reaction rate constant, k<SUB>f</SUB>, determines how fast the reaction goes. This constant can be used to compare one reaction with another. To find this constant, we plot equation 2 directly through the obtained data.
The equation is also shown in Figure 6.</P><FIGURE><FIGCAPTION>Figure 6. The fluorescence of a two solutions with BBMV's obtained from <I>T. molitor</I> incorporated with fluorophores was measured over time. One in the presence and one in the absence of Cry3Aa.  (a) shows the fluorescence against time, whereas (b) shows the natural logarithm of the maximum fluorescence minus the fluorescence at a certain time point against the time.</FIGCAPTION></FIGURE><P> For both samples, 6 technical replicates were measured. Representative measurements of each sample are shown in Figure 6. All data can be found <A href="https://2016.igem.org/Team:Wageningen_UR/Notebook/InVitroAssay#september">here</A>. For all measurements, the reaction rate constants were determined as explained above and are shown in Figure 7. This figure shows that in the presence of Cry3Aa, the k<SUB>f</SUB> values are higher and thus the fluorophores are released faster. Overall, fluorophore leaking in the presence of Cry3Aa is accelerated with a factor of 1.59 compared to fluorophore leaking in the absence of Cry3Aa. Unless one of the other few proteins still present in the sample would unexpectedly interfere with the experiment, Cry3Aa induces leakage of carboxyfluorescein from <I>T. molitor</I> BBMV's. </P><FIGURE><FIGCAPTION>Figure 7. A boxplot shows the k<SUB>f</SUB> values that represent the velocity at which fluorophore release occurs for BBMV's obtained from <I>T. molitor</I> in the presence and absence of Cry3Aa.</FIGCAPTION></FIGURE><P>A library of Cry3Aa proteins with mutations in the three different binding sites was <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#ToxinEngineering ">made</A> to create a Cry toxin that is potentially toxic to <I>V. destructor</I>. When one of these mutant Cry3Aa proteins is toxic to <I>V. destructor</I>, the toxin should induce leakage of carboxyfluorescein from <I>V. destructor BBMV's</I>, and therefore have a higher k<SUB>f</SUB> value than the negative control. The influence of Cry3Aa on <I>T. molitor</I> vesicles can be used as a positive control. The process of optimization of the positive control can be found in the <A href="https://2016.igem.org/Team:Wageningen_UR/Notebook/InVitroAssay#appendices">notebook</A>. It should be noted, however, that the relationship between the effect of the Cry toxin on vesicles and the Cry toxin on the whole organism might be different for each organism.  To know with certainty whether a certain Cry toxin is lethal for <I>V. destructor</I>, an <I>in vivo</I> test should be performed. </P><SECTION id="ToxinEngineering"><H1><B>Toxin Engineering</B></H1><P>
As mentioned above, no specific toxin to <I>Varroa</I> mites is known. To overcome this limitation, we tried to engineer one. We used the Cry3Aa toxin as a starting point.This toxin is known to be effective against a number of Coleoptera as the Colorado potato beetle <I>Leptinotarsa decemlineata</I> and the mealworm <I>T. molitor</I>. This toxin was also shown to be active against 4 different types of mites (<I>Acarus siro L., Tyrophagus putrescentiae, Dermatophagoides farinae Hughes, and Lepidoglyphus destructor</I>)<SUP><A href="#lm1" id="reflm1">1</A></SUP>. This engineering focused on two different strategies: </P><OL><LI>Modification of the previously described motifs within the Cry3Aa structure –Random mutagenesis of the binding sites, and</LI><LI>Finding specific binding motifs to the receptors within the mite gut – Phage Display</LI></OL><H2 id="engineering1"><B>Binding Site Mutagenesis</B></H2><H3><B>Design</B></H3><P>In 1996, Rajamohan <I>et al.</I> demonstrated that mutations in the binding sites of Cry toxins can both decrease and enhance the specificity of a toxin towards its target organism<SUP><A href="#lm2" id="reflm2">2</A></SUP>. Knowing this, the 3D structure of the Cry3Aa toxin has been analysed. Three putative binding sites have been identified. These are shown in in the following table.
</P><FIGURE><FIGCAPTION>
Table 1: Binding regions of the Cry3Aa protein. </FIGCAPTION><TABLE><TBODY><TR><TH>Region Name</TH><TH>Amino Acid Region</TH><TH>Amino Acid Sequence</TH></TR><TR><TD>3.1</TD><TD>312-317</TD><TD>NNLRGY</TD></TR><TR><TD>3.2</TD><TD>349-355</TD><TD>GYYGND</TD></TR><TR><TD>3.3</TD><TD>410-416</TD><TD>VWPSAVY</TD></TR></TBODY></TABLE></FIGURE><P>To mutate these binding sites, the plasmid pSB1A3_Cry3Aa_TEV_HIS has been used as a template. This plasmid contains the gene encoding wildtype Cry3Aa toxin (<I>B. thuringiensis var. tenebrionis</I>, Taxonomy ID: 1444). Its expression is regulated by the arabinose inducible promotor pBAD/araC (<A href="http://parts.igem.org/Part:BBa_I0500">Bba_I0500</A>).
From this plasmid, three different plasmids were <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#engineering">created</A> with randomized sequences within the binding sites. By this we expect that each individual colony harbouring this plasmid containing an individual variants, in other words, a library of Cry3Aa proteins with randomly binding sites to be screened.
</P><H3><B>Screening</B></H3><P>The mutants have been screened following the hightroughput procedure developed and described <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Assay">above</A>. In short, random colonies were picked from each of the binding mutants (n=144) for toxin expression and from these (n=24) were selected for <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#engineering">activity testing</A>. As a negative control, cell free extract from empty expression strain, BL21(DE3) has been used. The <I>in vitro</I> toxicity assay results for these 24 tested mutants compared to the Cry3Aa expressing strain and the negative control are shown in Figure 8. Figure 9 shows a heatmap of the tested mutants' activities.</P><FIGURE><FIGCAPTION>Figure 8. relative k-values of 24 different mutants compared to Cry3Aa and the empty expression strain.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 9. Heatmap of the relative k values. A high relative k value indicates a higher toxicity and specificity of the tested toxin. As the legend shows, the lighter the colour, the higher the activity.</FIGCAPTION></FIGURE><P>From these results, it can be concluded that the third binding site (amino acids 410-416) seems to be a good candidate for future engineering and specificity adaptation of this particular Cry toxin. Due to the relatively high deviation in reaction speed for the toxins 3.3.3 and 3.3.7, these should not be taken into account, as they are rather inconclusive. This leaves us with one proper candidate – mutant Cry3.3.5. This mutant has both a higher mean reaction velocity compared to the reference value and a relatively narrow range of measured relative k-values obtained for the 9 tested samples. Furthermore, the negative control shows that other proteins present in the cell free extract or cell debris (e.g membrane residue) interfere with our reading. Nevertheless, our preliminary results show that the mutant revokes a stronger response than the negative control, suggesting a specific binding to the Varroa vesicles.
</P><H2 id="engineering2"><B>Phage Display</B></H2><P>Due to difficulty to obtain mites, the phage display experiments were set up in a model organism where a Cry toxin has already been described: <I>T. molitor</I> (Cry3Aa). We expected to prove the applicability of this technique for specific protein engineering towards toxicity for <I>V. destructor</I>.
We used the filamentous bacteriophage M13 and two different libraries; 7-mer flanked by a pair of cysteine residues (The Ph.D.™-C7C Phage Display Peptide Library) and the 12-mer (The Ph.D.™-12 Phage Display Peptide Library).

</P><H3><B>Display of M13KE-PIII Libraries</B></H3><P>Two different strategies for phage display were used, <I>in vivo</I> (conventional phage display) and <I>in vitro</I> (panning). For the <I>in vitro</I> display, a modified biopanning protocol, the Biopanning and Rapid Analysis of Selective Interactive Ligands (BRASIL) method, was used. </P><H4><B><I>in vivo</I></B></H4><P>First as a proof of concept feeding experiments were <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#engineering">performed</A> with fluorescein on mealworms and mites (Figure 10). In short, the mites and mealworms were fed with either a piece of carrot soaked with fluorescein or a bee larvae soaked with fluorescein. After washing with PBS buffer, the insect were ground up and fluorescence was measured.
</P><FIGURE><FIGCAPTION>Figure 10. Intensity of fluorescence of several mites after the fluorescein feeding experiment. Only 3/18 of mites do take up fluorescein overnight.</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION>Figure 11. Results of the mealworm feeding assay. The mealworms take up notable amounts of food containing fluorescein overnight.</FIGCAPTION></FIGURE><P>
After seeing that the mites and mealworms could take up fluorescein, the phage libraries were fed to <I>T. molitor</I> and <I>V. destructor</I> and the bound phages <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#engineering">recovered</A> for further use. Due to a lack of mites, only two rounds of mite <I>in vivo</I> display could be performed. For the mealworms, the titer (the amount of plaque forming phages per mL) decreased drastically per round Figure 12. Thus, a second run with propagation of phages between the round has been performed. </P><FIGURE><FIGCAPTION>Figure 12. Phage titre over three rounds of phage display.</FIGCAPTION></FIGURE><P>

During this second round, an unexpected incident caused us to stop the <I>in vivo</I> phage display. While tittering a phage library obtained from a mealworm, no plaques with the expected morphology (blue, lysogenic) showed up on the plate. Instead of M13 plaques, several lytic virus were recovered. The experiments have thus been stopped due to a risk of contamination in the samples. To investigate the source of the contamination, all possible sources have been looked at: Used buffer, used top-agar, and the plates. The source of the unexpected plaques turned out to be the mealworm gut. Kelly <I>et al.</I> found that phages can be found in mealworms naturally<SUP><A href="#lm3" id="reflm3">3</A></SUP>. When using guts of mealworms that have not been fed any phages prior to the process, still a lot of plaques of different kinds could be seen on the titer plates. From this, we could conclude that there are naturally some bacteriophages that <I>E. coli</I> can be a host for in the gut of <I>T. molitor</I>. While those phages could be of interest for general bacteriophage studies, or further specific binding studies, they do interfere with our phage display experiments in two ways:</P><OL><LI>Knowing that phages are already inside of the gut (in relatively high amounts) they might block the receptors for our phage libraries.</LI><LI>As the phages and their properties are not known, working with them might not be safe
As there was a shortage of living mites for the <I>in vivo</I> display anyway, we decided to switch to an <I>in vitro </I>phage display method.</LI></OL><P>Nevertheless, the results obtained up to then in the mite phage display experiments have been analyzed and the following binding site motif was found: </P><FIGURE><FIGCAPTION>Figure 13. Consensus sequence of recovered phages in <I>in vivo</I> Phage Display in <I>V. destructor</I>. The legend shows which aminoacid has which properties. The letters &quot;N&quot; and &quot;C&quot; in the graph indicate the N-terminus and the C-terminus, respectively.</FIGCAPTION></FIGURE><H4><B><I>in vitro</I></B></H4><P>For an <I>in vitro</I> assay, vesicles of the <I>T. molitor</I> midgut and <I>V. destructor</I> have been produced, according to the <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#toxassay">protocol</A>. In short, cells were lysed, lipids were separated and then left for reassembly. These vesicles should have no phages attached on the outside, due to the way they were prepared. The only chance of having phages that are naturally occurring in mealworms or <I>Varroa</I> mites was thus that they had been trapped inside of the vesicle during the preparation steps and so relatively low.
Unfortunately, the titer of phages bound to the vesicles prepared from <I>V. destructor</I> decreased dramatically after the first round of display and dropped to 0 after the second round. This indicates that the phages did not bind to the vesicles in a way that is strong enough to keep them attached during the separation step of vesicles with attached phages from phages that have not bound. The reasons for this might be investigated and the protocol used for this phage display might still be optimized.


</P></SECTION><SECTION id="Isolates"><H1><B><I>Varroa</I> Isolates</B></H1><H2 id="Isolates1">Staining Isolates</H2><P> To provide an alternative to currently available pesticides, we wanted to adapt Cry toxins to kill <I>V. destructor</I>. Cry toxins, also known as Bt toxins, have been used commercially for decades and can limit the environmental impact of pesticide use<SUP><A href="#vi1" id="refvi1">1</A></SUP>. Cry toxins are produced by the insect pathogen <I>B. thuringiensis</I>. Their mode of action can make them highly specific to certain species of insects.</P><P>While over 800 Cry toxins are known, research on the workings of these toxins mainly focuses on conventional pests such as the Colorado potato beetle, cabbage moths and mosquitoes. Field experiments on non-target invertebrates show that <A class="tooltip">Acari<SPAN class="tooltiptext" style="width:400px;">The scientific name for mites and ticks.</SPAN></A> are not significantly affected by <A class="tooltip"> Bt crops<SPAN class="tooltiptext" style="width:400px;"> Transgenic crops which express Cry toxins, such as Bt cotton or Bt maize.</SPAN></A><SUP><A href="#vi3" id="refvi3">3</A></SUP>, but these use only a few known Cry toxins. Other studies have shown that there are strains of <I>B. thuringiensis</I> which are harmful to Acari. While most of these strains or toxins were tested against herbivorous species of mites<SUP><A href="#vi4" id="refvi4">4,</A><A href="#vi5" id="refvi5">5,</A><A href="#vi6" id="refvi6">6,</A><A href="#vi7" id="refvi7">7</A></SUP>, some are known to kill species of ticks<SUP><A href="#vi8" id="refvi8">8,</A><A href="#vi9" id="refvi9">9,</A><A href="#vi10" id="refvi10">10,</A><A href="#vi11" id="refvi11">11</A></SUP>. In fact, two strains have been discovered which show toxicity to <I>V. destructor</I><SUP><A href="#vi12" id="refvi12">12,</A><A href="#vi13" id="refvi13">13</A></SUP>!  Unfortunately, they did not identify a particular Cry toxin. </P><P>To find our own Cry toxin, we <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#isolates1">adapted a procedure</A> from Rampersad <I>et al.</I><SUP><A href="#vi14" id="refvi14">14</A></SUP> and Alquisira-Ramírez <I>et al</I><SUP><A href="#vi12" id="refvi12">12</A></SUP>. We gathered over 800 dead mites from the Dutch foundation &quot;De Duurzame Bij&quot;<SUP><A href="#vi15" id="refvi15">15</A></SUP>, Bennekom, The Netherlands.
<FIGURE><FIGCAPTION>Figure 14. Photo of the beehives from  De Duurzame Bij. We gathered most of our dead mites from these beehives.</FIGCAPTION></FIGURE></P><P>Our method of sample preparation selected for spore formation, but not the <I>B. thuringiensis</I> rod-shaped morphology. Therefore, 106 colonies from  the mites were investigated with brightfield microscopy.  They were stained with Coomassie Brilliant Blue to make Cry toxins more visible. Five isolates were identified as <I>Bacillus</I>-like species. Some of them, including  <I>B. thuringiensis</I> HD-350, are shown in Figure 15. </P><FIGURE><FIGCAPTION>Figure 15.  Microscopy images of Coomassie-stained isolates, 1000x magnification with a Zeiss Axio Scope.A1 brightfield microscope. (a) <I>B. thuringiensis</I> HD350. The red arrow points to a Cry toxin, the green arrow to a spore and the yellow arrow to a vegetative cell. (b) Isolate 62, a coccus. Most isolates had this morphology. (c) Isolate 82, showing <I>Bacillus</I>-like morphology.</FIGCAPTION></FIGURE><H2 id="Isolates2"><I>In Vivo</I> Toxicity</H2><P>We wanted to test the strains we identified for their toxicity to <I>V. destructor</I>. Protein extracts and SDS-PAGE were <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#header">performed</A>. The <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#isolates2">toxicity assay</A> was based on the assay used by Alquisira-Ramírez <I>et al.</I><SUP><A href="#vi12" id="refvi12">12</A></SUP>, but we faced one major problem: the assay, including the negative control, killed all of the <I>Varroa</I> mites. An alternative would be to use an assay where we dip honeybee larvae in protein extracts from the strains; a <I>Varroa</I> mite which feeds on the larvae should then ingest any toxins. Unfortunately, <I>Varroa</I> mites tend to share feeding sites and use them repeatedly<SUP><A href="#vi16" id="refvi16">16</A></SUP>, which means that not all mites would consume the protein extract. </P><P>A sample-size calculation<SUP><A href="#vi17" id="refvi17">17</A></SUP> was performed in R with the following code to assess the feasibility of such an assay. 
<CODE>
&gt; p=0.66

&gt; p0=0.5

&gt; alpha=0.05

&gt; beta=0.1 

&gt; (n=p*(1-p)*((qnorm(1-alpha)+qnorm(1-beta))/(p-p0))^2)
</CODE>
Here, p is the alternative hypothesis, p0 the null hypothesis, alpha the type I error and beta the type II error. The values of these variables were based on the following assumptions:
<UL><LI>The null hypothesis is 50% mite mortality. Natural mite mortality is very high in the laboratory. </LI><LI> Linea Muhsal estimated during <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#engineering2"> feeding experiments</A> that 3 out of 18 mites ingested the fluorophores she had dipped honeybee larvae in.</LI><LI>If an effective Cry toxin is ingested, we expect to see 100% mite mortality. This is an optimistic assumption.</LI><LI>16% of mites eat a toxin which causes 100% mite mortality, so if there is a toxin, we should see 66% mite mortality. </LI></UL></P><P>
With these assumptions, a sample size of 75 was calculated. We were generally able to gather a maximum of 40 mites per week. Gathering enough mites to test all our samples was unfeasible. Therefore, pursuit of this type of toxicity assay was cancelled in favour of further development of an <A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Assay"><I>in vitro</I> assay.</A></P><H2 id="Isolates3">16s rRNA sequencing and SDS-PAGE results</H2><P>
We <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#isolates3">sequenced the 16s rRNA</A> for all five isolates with <I>Bacillus</I>-like morphology. A search of NCBI’s 16S ribosomal RNA sequences database resulted in the following alignments, each with approximately 99% identity of their best match. The isolates and their matches are displayed in Table 1. 
<FIGURE><FIGCAPTION>Table 1. Best matches of a BLAST search of the isolates' 16s rRNA sequence against NCBI's 16 ribosomal RNA sequence database.</FIGCAPTION><TABLE><TBODY><TR><TH>Isolate</TH><TH>Best alignment</TH></TR><TR><TD>V46</TD><TD><I>Bacillus licheniformis</I></TD></TR><TR><TD>V47</TD><TD><I>Paenibacillus amylolyticus</I></TD></TR><TR><TD>V82</TD><TD><I>Lysinibacillus meyeri</I></TD></TR><TR><TD>V88</TD><TD><I>Paenibacillus chitinolyticus</I></TD></TR><TR><TD>V106</TD><TD><I>Bacillus sp.</I></TD></TR></TBODY></TABLE></FIGURE></P><P>While it narrows down the number of candidates, 16s rRNA sequencing is not a reliable method to distinguish between <I>Bacillus</I> species<SUP><A href="#vi18" id="refvi18">18</A></SUP>. Therefore, an SDS-PAGE gel of the protein extracts was prepared to further analyse them. This gel is shown in Figure 16; <I>Bacillus subtilis</I> and <I>Bacillus thuringiensis</I> were included as controls.
<FIGURE><FIGCAPTION>Figure 16.  SDS-PAGE gel of the protein extracts from <I>Bacillus subtilis</I>, <I>B. thuringiensis</I> HD350,  <I>B. thuringiensis</I> tenebrionis and isolates 46, 47, 81, 82 and 88. The ladder is a BioRad Precision Plus Protein Standard.  The red arrow indicates a band that could be Cry1Aa, which is 133 kDa. The blue arrow indicates a band which could be Cry3Aa, a 73 kDa toxin. The yellow arrow indicates a band that was investigated with protein MS/MS – see Figure 17.  </FIGCAPTION></FIGURE></P><H2 id="Isolates4">Proteomics analysis of isolate V82<P>
Based on the previous results, isolate V82 had the most potential to be a putative mite pathogen. It is closely related to a known insect pathogen, <I>Lysinibacillus sphaericus</I>, which is known to produce a 100 kDa mosquitocidal toxin<SUP><A href="#vi19" id="refvi19">19</A></SUP>. Our isolate also showed a strong band in the 100 kDa region (Figure 16). A new SDS-PAGE gel was prepared with independently grown V82 cultures and bands were cut out for peptide extraction, so the 100 kDa protein could be analysed with <A href="”https://2016.igem.org/Team:Wageningen_UR/Experiments#isolates4&quot;">LC-MS/MS</A>. Negative control bands were cut out approximately ~1 cM above the 100 kDa bands. MaxQuant<SUP><A href="#vi20" id="refvi20">20</A></SUP> was used to analyze the data. 

<FIGURE><FIGCAPTION>Figure 17.  LC-MS/MS spectra of 3 control samples (top) and 3 samples extracted from a ~100 kDa band on the SDS-PAGE gel with V82 protein. Click the figure for the full-resolution image.</FIGCAPTION></FIGURE></P><P><FIGURE><FIGCAPTION>Figure 18.  Volcano plot of LC-MS/MS results. The colours indicate whether the difference between the amount of protein in the 100 kDa bands and the control bands was significant, or whether the proteins were recognized as contaminants by the MaxQuant software.</FIGCAPTION></FIGURE></P><P>
Analysis of the LC-MS/MS results is shown in Figure 18, a volcano plot. The x-axis plots the protein abundance ratio between the 100 kDa band and its respective control band, while the y-axis plots the p-value of this difference in protein abundance. Therefore, a protein in the left half of the plot is more abundant in the control bands, while the opposite goes for proteins in the right half of the plot. The higher it is on the y-axis, the more significant this difference is. </P><P>
Contaminations such as keratins and the trypsin used to digest the samples should not differ significantly between the samples. This is also shown in Figure 18, as most contaminants (the orange data points) are present in the bottom half of the volcano plot.
The relative label-free quantification (LFQ) intensity of the significant proteins shown in Figure 18 was very low, in all cases lower than the LFQ intensity of trypsin. From a biological perspective, this makes sense; there is no reason for an organism to produce mass amounts of alanine tRNA ligase, which was one of the significant peptide matches. 

However, this and the fact that only 3% of peptides could be matched indicates that the 100 kDa protein was not present in the prepared <I>Lysinibacillus</I> sequence database. Therefore, the peptides were also matched against a <I>Bacillus</I> database. If the 16s rRNA sequence was wrong, this could be a more suitable database. However, the expanded database yielded similar results. Genomic DNA was extracted from V82 and sent for sequencing so the LC-MS/MS data could be run against the isolate's own DNA. This genomic DNA was also analyzed with the <A href="https://2016.igem.org/Team:Wageningen_UR/Software">Toxin Scanner</A>. 
</P></H2></SECTION><P>We were able to isolate <I>Bacillus</I>-like strains from dead <I>Varroa</I> mites and we could show that one of them overexpressed a protein at a size of interest. This isolate was sent for genomic sequencing; <A href="https://2016.igem.org/Team:Wageningen_UR/Software#TS_results">analysis of the ~100 kDa proteins was performed with the Toxin Scanner.</A></P><SECTION id="references"><H1>References</H1><OL class="references"><H2>References for <I>in vitro</I> essay</H2><A id="jz1" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1857359/">1.</A> Alejandra Bravo, Sarjeet S. Gill, Mario Soberón. Mode of action of <I>Bacillus thuringiensis</I> Cry and Cyt toxins and their potential for insect control. Toxicon. 2007 Mar 15; 49(4): 423–435. <A href="#refjz1" title="Jump back to footnote 1 in the text.">↩</A><A id="jz2" href="http://base.dnsgb.com.ua/files/book/Agriculture/Pesticides/Pesticide-Chemistry-Crop-Protection.pdf">2.</A> Hideo Ohkawa,Hisashi Miyagawa,Phillip W. Lee. Pesticide Chemistry: Crop Protection, Public Health, Environmental Safety. Wiley-VHC Verlag GmbH &amp; Co KGaA Weinheim. 2007. p.193 <A href="#refjz2" title="Jump back to footnote 2 in the text.">↩</A><A id="jz3" href="http://www.jbc.org/content/284/27/18401.full.pdf">3.</A> Jeff Fabrick, Cris Oppert, Marce´ D. Lorenzen, Kaley Morris, Brenda Oppert, and Juan Luis Jurat-Fuentes. A Novel <I>Tenebrio molitor</I> Cadherin Is a Functional Receptor for <I>Bacillus thuringiensis</I> Cry3Aa Toxin. The Journal of Biological Chemistry VOL. 284, NO. 27, pp. 18401–18410, July 3, 2009. <A href="#refjz3" title="Jump back to footnote 2 in the text.">↩</A><A id="jz4" href="https://www.ncbi.nlm.nih.gov/pubmed/22640634">4.</A>Oppert B1, Martynov AG, Elpidina EN.Bacillus thuringiensis Cry3Aa protoxin intoxication of <I>Tenebrio molitor</I> induces widespread changes in the expression of serine peptidase transcripts. Comp Biochem Physiol Part D Genomics Proteomics. 2012 Sep;7(3):233-42. <A href="#refjz4" title="Jump back to footnote 2 in the text.">↩</A><A id="jz5" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC282293/pdf/pnas00300-0023.pdf">5.</A>C. Hofmann, H. Vanderbruggen, H. Höfte, J. van Rie, S. Jansens, H. Van MallaertSpecificity of <I>Bacillus thuringiensis</I> δ-endotoxins is correlated with the presence of high affinity binding sites in the brush border membranes of target insect midguts Proc. Natl. Acad. Sci. U. S. A., 85 (1988), pp. 7844–7848 <A href="#refjz5" title="Jump back to footnote 2 in the text.">↩</A><A id="jz6" href="http://ac.els-cdn.com/S0005273603003547/1-s2.0-S0005273603003547-main.pdf?_tid=fa9548dc-8e15-11e6-ae8d-00000aab0f6c&amp;acdnat=1476013753_d2b355e5a9a51318749179f82d01af53">6.</A> C. Rausell, I. García-Robles, J. Sánchez, C. Muñoz-Garay, A.C. Martínez-Ramírez, M.D. Real, A. Bravo Role of toxin activation on binding and pore formation activity of the <I>Bacillus thuringiensis</I> Cry3 toxins in membranes of <I>Leptinotarsa decemlineata</I> (Say) Biochim. Biophys. Acta, 1660 (2004), pp. 99–105 <A href="#refjz6" title="Jump back to footnote 2 in the text.">↩</A><A id="jz7" href="http://www.jlr.org/content/37/11/2405.full.pdf">7.</A> G. Schulthess, S. Compassi, D. Boffelli, M. Werder, F. E. Weber, and H. Hauser. A comparative study of sterol absorption in different small-intestinal brush border membrane models. Journal of Lipid Research Volume 37, 1996, pp 2405-2419 <A href="#refjz7" title="Jump back to footnote 2 in the text.">↩</A><A id="jz8" href="http://www.sciencedirect.com/science/article/pii/0003269788904125">8.</A> Raymond F. Chen, Jay R. Knutson. Mechanism of fluorescence concentration quenching of carboxyfluorescein in liposomes: Energy transfer to nonfluorescent dimers. Analytical Biochemistry Volume 172, Issue 1, July 1988, Pages 61-77 <A href="#refjz8" title="Jump back to footnote 2 in the text.">↩</A><A id="jz9" href="http://www.ijbbb.org/papers/161-E005.pdf">9.</A> Huang., Y. Tippmann, J. Becker, T. Kinetic Modeling of Hop Acids during Wort Boiling. International Journal of Bioscience, Biochemistry and Bioinformatics, Vol. 3, No. 1, January 2013 <A href="#refjz9" title="Jump back to footnote 2 in the text.">↩</A><H2>References for Toxin engineering</H2><A id="lm1" href="https://www.ncbi.nlm.nih.gov/pubmed/19381844">1.</A> Erban, T., Nesvorna, M., Erbanova, M. et al. Exp Appl Acarol (2009) 49: 339. doi:10.1007/s10493-009-9265-z <A href="#reflm1" title="Jump back to footnote 1 in the text.">↩</A><A id="lm2" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC26133/">2.</A> 
Rajamohan F, Alzate O, Cotrill JA, Curtiss A, Dean DH. Protein engineering of <I>Bacillus thuringiensis</I> δ-endotoxin: Mutations at domain II of CryIAb enhance receptor affinity and toxicity toward gypsy moth larvae. Proceedings of the National Academy of Sciences of the United States of America. 1996;93(25):14338-14343. <A href="#reflm2" title="Jump back to footnote 2** in the text.">↩</A><A id="lm3" href="http://jgv.microbiologyresearch.org/content/journal/jgv/10.1099/0022-1317-42-1-95">3.</A> Kelly D, Ayres M, Lescott T, Robertson J, Happ G. J.  A Small Iridescent Virus (Type 29) Isolated from <I>Tenebrio molitor</I>: a Comparison of its Proteins and Antigens with Six Other Iridescent Viruses. Gen. Virol. 42(1):95-105 doi:10.1099/0022-1317-42-1-95 <A href="#reflm3" title="Jump back to footnote 1 in the text.">↩</A><H2>References for <I>Varroa</I> isolates</H2><A id="vi1" href="http://www.sciencedirect.com/science/article/pii/S0022201115001342">1.</A> Lacey, L. A., Grzywacz, D., Shapiro-Ilan, D. I., Frutos, R., Brownbridge, M., &amp; Goettel, M. S. (2015). Insect pathogens as biological control agents: back to the future. Journal of invertebrate pathology, 132, 1-41. <A href="#refvi1" title="Jump back to footnote 1 in the text.">↩</A><A id="vi2" href="http://www.sciencedirect.com/science/article/pii/S0168952501022375">2.</A> de Maagd, R. A., Bravo, A., &amp; Crickmore, N. (2001). How <I>Bacillus thuringiensis</I> has evolved specific toxins to colonize the insect world.TRENDS in Genetics, 17(4), 193-199. <A href="#refvi2" title="Jump back to footnote 2 in the text.">↩</A><A id="vi3" href="http://science.sciencemag.org/content/316/5830/1475">3.</A> Marvier, M., McCreedy, C., Regetz, J., &amp; Kareiva, P. (2007). A meta-analysis of effects of Bt cotton and maize on nontarget invertebrates.science, 316(5830), 1475-1477. <A href="#refvi3" title="Jump back to footnote 3 in the text.">↩</A><A id="vi4" href="http://www.tandfonline.com/doi/abs/10.1080/01647954.2015.1109707">4.</A> Ahmed, N., Wang, M., &amp; Shu, S. (2016). Effect of commercial <I>Bacillus thuringiensis</I> toxins on <I>Tyrophagus putrescentiae</I> (Schrank) fed on wolfberry (<I>Lycium barbarum</I> L.). International Journal of Acarology, 42(1), 1-6. <A href="#refvi4" title="Jump back to footnote 4 in the text.">↩</A><A id="vi5" href="http://onlinelibrary.wiley.com/doi/10.1111/j.1439-0418.1991.tb00305.x/abstract">5.</A> Chapman, M. H., &amp; Hoy, M. A. (1991). Relative toxicity of <I>Bacillus thuringiensis</I> var. tenebrionis to the two‐spotted spider mite (<I>Tetranychus urticae</I> Koch) and its predator <I>Metaseiulus occidentalis</I> (Nesbitt)(Acari, Tetranychidae and Phytoseiidae). Journal of Applied Entomology, 111(1‐5), 147-154. <A href="#refvi5" title="Jump back to footnote 5 in the text.">↩</A><A id="vi6" href="http://parasitesandvectors.biomedcentral.com/articles/10.1186/s13071-015-0890-6">6.</A> Dunstand-Guzmán, E., Peña-Chora, G., Hallal-Calleros, C., Pérez-Martínez, M., Hernández-Velazquez, V. M., Morales-Montor, J., &amp; Flores-Pérez, F. I. (2015). Acaricidal effect and histological damage induced by <I>Bacillus thuringiensis</I> protein extracts on the mite <I>Psoroptes cuniculi</I>. Parasites &amp; vectors, 8(1), 1. <A href="#refvi6" title="Jump back to footnote 6 in the text.">↩</A><A id="vi7" href="http://link.springer.com/article/10.1007/s10493-009-9265-z">7.</A> Erban, T., Nesvorna, M., Erbanova, M., &amp; Hubert, J. (2009). <I>Bacillus thuringiensis</I> var. tenebrionis control of synanthropic mites (Acari: Acaridida) under laboratory conditions. Experimental and Applied Acarology, 49(4), 339-346. <A href="#refvi7" title="Jump back to footnote 7 in the text.">↩</A><A id="vi8" href="http://jinsectscience.oxfordjournals.org/content/10/1/186.abstract">8.</A> Fernández-Ruvalcaba, M., Peña-Chora, G., Romo-Martínez, A., Hernández-Velázquez, V., de la Parra, A. B., &amp; De La Rosa, D. P. (2010). Evaluation of <I>Bacillus thuringiensis</I> pathogenicity for a strain of the tick,<I> Rhipicephalus microplus</I>, resistant to chemical pesticides. Journal of Insect Science, 10(1), 186. <A href="#refvi8" title="Jump back to footnote 8 in the text.">↩</A><A id="vi9" href="http://link.springer.com/article/10.1007/s004360050235">9.</A> Hassanain, M. A., Garhy, M. E., Abdel-Ghaffar, F. A., El-Sharaby, A., &amp; Megeed, K. N. A. (1997). Biological control studies of soft and hard ticks in Egypt. Parasitology research, 83(3), 209-213. <A href="#refvi9" title="Jump back to footnote 9 in the text.">↩</A><A id="vi10" href="http://onlinelibrary.wiley.com/doi/10.1111/jen.12235/pdf">10.</A> Neethu, K. B., Priji, P., Unni, K. N., Sajith, S., Sreedevi, S., Ramani, N., ... &amp; Benjamin, S. (2015). New <I>Bacillus thuringiensis</I> strain isolated from the gut of Malabari goat is effective against <I>Tetranychus macfarlanei</I>. Journal of Applied Entomology. <A href="#refvi10" title="Jump back to footnote 10 in the text.">↩</A><A id="vi11" href="http://jme.oxfordjournals.org/content/36/6/900.abstract">11.</A> Zhioua, E., Heyer, K., Browning, M., Ginsberg, H. S., &amp; LeBrun, R. A. (1999). Pathogenicity of <I>Bacillus thuringiensis</I> variety kurstaki to<I> Ixodes scapularis</I> (Acari: Ixodidae). Journal of medical entomology, 36(6), 900-902. <A href="#refvi11" title="Jump back to footnote 11 in the text.">↩</A><A id="vi12" href="http://link.springer.com/article/10.1007/s13592-014-0288-z">12.</A> Neethu, K. Alquisira-Ramírez, E. V., Paredes-Gonzalez, J. R., Hernández-Velázquez, V. M., Ramírez-Trujillo, J. A., &amp; Peña-Chora, G. (2014). <I>In vitro</I> susceptibility of <I>varroa destructor</I> and <I>Apis mellifera</I> to native strains of <I>Bacillus thuringiensis</I>. Apidologie, 45(6), 707-718. <A href="#refvi12" title="Jump back to footnote 12 in the text.">↩</A><A id="vi13" href="http://link.springer.com/article/10.1023/B:BILE.0000019563.92959.0e">13.</A> Tsagou, V., Lianou, A., Lazarakis, D., Emmanouel, N., &amp; Aggelis, G. (2004). Newly isolated bacterial strains belonging to Bacillaceae (<I>Bacillus</I> sp.) and Micrococcaceae accelerate death of the honey bee mite, <I>varroa destructor</I> (<I>V. jacobsoni</I>), in laboratory assays. Biotechnology letters, 26(6), 529-532. <A href="#refvi13" title="Jump back to footnote 13 in the text.">↩</A><A id="vi14" href="http://bmcmicrobiol.biomedcentral.com/articles/10.1186/1471-2180-5-52">14.</A> Rampersad, J., &amp; Ammons, D. (2005). A <I>Bacillus thuringiensis</I> isolation method utilizing a novel stain, low selection and high throughput produced atypical results. BMC microbiology, 5(1), 1. <A href="#refvi14" title="Jump back to footnote 14 in the text.">↩</A><A id="vi15" href="http://www.duurzamebij.nl/">15.</A> This is the homepage of &quot;De Duurzame Bij&quot;. They want to stop treating <I>Varroa</I> with pesticides. Loosely translated, &quot;Duurzame Bij&quot; means &quot;sustainable bee&quot;. <A href="#refvi15" title="Jump back to footnote 15 in the text.">↩</A><A id="vi16" href="http://geneticsmr.com/articles/190">16.</A> Kanbar, G., &amp; Engels, W. (2005). Communal use of integumental wounds in honey bee (<I>Apis mellifera</I>) pupae multiply infested by the ectoparasitic mite<I> Varroa destructor</I>. Genetics and Molecular Research, 4(3), 465-472. <A href="#refvi16" title="Jump back to footnote 16 in the text.">↩</A><A id="vi17" href="http://onlinelibrary.wiley.com/doi/10.1002/wics.155/full">17.</A> Chow, S. C. (2011). Sample size calculations for clinical trials. Wiley interdisciplinary reviews: Computational statistics, 3(5), 414-427. <A href="#refvi17" title="Jump back to footnote 17 in the text.">↩</A><A id="vi18" href="http://jcm.asm.org/content/42/8/3711.short">18.</A> Bavykin, S. G., Lysov, Y. P., Zakhariev, V., Kelly, J. J., Jackman, J., Stahl, D. A., &amp; Cherni, A. (2004). Use of 16S rRNA, 23S rRNA, and gyrB gene sequence analysis to determine phylogenetic relationships of <I>Bacillus cereus</I> group microorganisms. Journal of clinical microbiology, 42(8), 3711-3730. <A href="#refvi18" title="Jump back to footnote 18 in the text.">↩</A><A id="vi19" href="http://www.sciencedirect.com/science/article/pii/S0022201111002461">19.</A> Berry, C. (2012). The bacterium, <I>Lysinibacillus sphaericus</I>, as an insect pathogen. Journal of invertebrate pathology, 109(1), 1-10. <A href="#refvi19" title="Jump back to footnote 19 in the text.">↩</A><A id="vi20" href="http://www.nature.com/nbt/journal/v26/n12/abs/nbt.1511.html">20.</A> Cox, J., &amp; Mann, M. (2008). MaxQuant enables high peptide identification rates, individualized ppb-range mass accuracies and proteome-wide protein quantification. Nature biotechnology, 26(12), 1367-1372. <A href="#refvi20" title="Jump back to footnote 20 in the text.">↩</A></OL></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2016. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV></DIV></BODY></HTML>