Team:Wageningen UR/Notebook/Riboswitch
MENU ‚ñ§
HOME
TEAM
Team
‚òÖ  Collaborations
PROJECT
‚òÖ  Description
‚òÖ Design
Experiments
‚òÖ Proof of Concept
‚òÖ Demonstrate
Results
Notebook
PARTS
Parts
‚òÖ Basic Parts
‚òÖ Composite Parts
‚òÖ Part Collection
SAFETY
‚òÖ  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
‚òÖ Silver
‚òÖ Gold
‚òÖ Integrated Practices
‚òÖ Engagement
AWARDS
‚òÖ Entrepreneurship
‚òÖ Hardware
‚òÖ Software
‚òÖ  Measurement
‚òÖ Model
Wageningen UR iGEM 2016
‚ò∞
Àü
Overview
Description
Results
Achievements
BioBricks
Specificity
Regulation
Biocontainment
Model
Outreach
Team
Notebook
¬†
¬†
¬†
Main Results
May
June
July
August
September
October
Overview
Description
Results
Achievements
BioBricks
Applied Design
WetLab
Specificity
In vitro assay
Toxin Engineering
Varroa Isolates
Regulation
Detecting Mites
Population Dynamics
Toggle Switch
E. coli survival
Biocontainment
Light Kill Switch
Cas9 Kill Switch
Collaborations
Model
Population Dynamics
Light Kill Switch
E. coli survival
Beehave
Toxin Scanner
Outreach
Our Vision
Design
Collaboration
Events
Media
Integrated Practices
Team
Members
Students
Supervisors
Gallery
Attributions
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
May
Design considerations
A promoter of the Anderson constitutive promoter family was placed in front of the riboswitch. It is the consensus constitutive promoter family of iGEM, it is well documented and the amount of expression differs 2500 fold between different promoters. It can also be easily swapped for another promoter of the same family if a different amount of expression is needed 7 . This is convenient since the optimal expression level of the toxin gene is not known yet.
02 - 12 May
First attempt in constructing the inverter part out of biobricks Bba_P0140 and Bba_I13521. This has been done via the standard digestion, ligation and transformation protocol. The transformation resulted in one red colony. This colony has been checked by placing the constitutive promoter J23119 in front of it via another 3A assembly. If the construct is correct, the result should be white colonies.
However, this was not the case. Sequencing and PCR results showed that the construct only contained Bba_P0140.
13 - 24 May
Creation of the riboswitch part out of Bascillis subtilis and Escherichia coli. Extraction of genomic DNA from B. subtilis and E. coli was done with the use of the GeneJet genomic DNA purification kit and an overnight culture of B. subtilis strain 168 that has grown in B. subtilis medium and an overnight culture of E. coli strain DH5ùõÇ in normal LB medium. The extraction was done according to the supplied Gram-positive and Gram-negative bacteria standard protocol.   The table shows the primers used to retrieve the riboswitches.
Bribo_F
CTGAAGTCTCGAATTCGCGG
CCGCTTCTAGAGTTGACGGCTAGCTCAGTCC
TAGGTACAGTGCTAGCTACTAGAGgccggtcctgtgagttaatag
Forward primer for amplifying vitamin b12 riboswitch from E. coli while adding promoter and ssRA-tag.
Bribo_R
gttagggtctcctgcagcggccg
ctactagtattattAAGCAGCCAGAGCGTAGTTTT
CGTCGTTAGCAGCACCGTCTTTTGCAAACCTAG
Reverse primer for amplifying¬† vitamin b12 riboswitch from E. coli while adding promoter and ssRA-tag.
Gribo_F
CTGAAGTCTCGAATTCGCGGCCGCTT
CTAGAGTTGACGGCTAGCTCAGTCCTAGGTACA
GTGCTAGCTACTAGAGactcatataatcgcgtggatatg
Forward primer for amplifying guanine riboswitch from B. subtillis while adding promoter and ssRA-tag.
Gribo_R
Gttagggtctcctgcagcggccgc
tactagtattattAAGCAGCCAGAGCGTAGTTTTC
GTCGTTAGCAGCACCGTCTTTTGCAAACCTAG
Reverse primer for amplifying guanine riboswitch from B. subtillis while adding promoter and ssRA-tag.
For the PCR the standard protocol of New England Biolab has been used. For the retrieval of the vitamin b12 riboswitch out of Escherichia coli DH5alpha an annealing temperature of 66¬∞ was used and for the guanine riboswitch out of Bascillis subtilis an annealing temperature of 63¬∞. PCR and sequencing results showed that both constructs were correctly retrieved.
Via 3A-assembly the riboswitch part and the inverter part have been assembled. This resulted in three types of colonies: white, pink and red. The correct construct should be around 2500 base pairs.
PCR amplification and sequencing results showed that the pink colonies where the correct construct.
Cloning the mite sensing system based on riboswitches
The cloning of the system has been done with the use of a 3A-assembly out of two constructs. The first part is called B/Gribo and contains the riboswitch and constitutive promoter. The second part is the inverter part, consisting out of the TetR QPI and mRFP as a reporter gene.
Figure 1. Schematic overview showing that the system has been constructed via a 3A-assembly out of two constructs: one containing the riboswitch and one containing the TetR QPI and a reporter gene.
Creating riboswitch part with PCR and special primers
A part containing the riboswitch has been created with the use of special primers that copies the correct riboswitch from genomic DNA. During the PCR, the primer added the constitutive promoter and ssRA-tag. This part has been dubbed B-ribo and G-ribo (of vitamin B12 and guanine riboswitch).
Figure 2. GRibo schematic overview
Creating the inverter part out of bio bricks
The other part of the construct is the inverter-part, containing the TetR gene, a promoter that is inhibited by TetR and the reporter gene mRFP. This part is called the Inv-part. The Inv-part has been created out of standard part BBa_p0140 and BBa_I13521 from the iGEM kit as can be seen in figure 3.
Figure 3. The BBa_p0140 part consists of a ribosome binding site, tetracycline producing gene and two terminators.  The BBa_I13521 part consists of a tetracycline inducible promoter, a ribosome binding site, a mRFP producing gene and two terminators.
The PCR of the Inv-part shows a band between 2000 and 3000 basepairs where a band of 2122 base pairs is expected, as can be seen in figure 4.
Figure 4. The PCR of the Inv-part shows a band between 2000 and 3000 basepairs where a band of 2122 base pairs is expected.
Combing the riboswitch part and the inverter part to clone the designed construct
Out of the ribo-part and the Inv-part, the designed system B/GRInv has been assembled via a 3a-assembly. The transformation of B/GRinv out of B/Gribo and Inv-part in psB1C3 has resulted in three different types of colonies: white, pink and red ones.
Figure 5. The transformation of B/GRinv out of B/Gribo and Inv-part in psB1C3 has resulted in three different types of colonies: white, pink and red. PCR results show that only the pink colonies have the expected size of the B/GRInv
If no guanine or vitamin b12 are present, a colony containing the R/GRInv construct should have a white colour since TetR is expressed when no metabolite is present to bind to the riboswitch. TetR in its turn, will inhibit the promoter that normally expresses mRFP. However, in LB agar, small amounts of vitamin and guanine are present and therefore it is not possible to be sure which colony is the right one based on its colour: white, pink or red.  PCR results show that only the pink colonies have the expected size of the B/GRInv construct as can be seen in figure 5. Sequencing results confirm that the pink colonies contain the right construct.
05 - 20 July
To provide a basic proof of principle of the designed construct, a colony containing the B/GRinv plasmid was grown overnight in 10 mL LB medium. The concentration range of guanine to test the guanine riboswitch construct were based on the concentrations used in previous experiments with guanine riboswitches from B. Subtilis. The concentration range of vitamin b12 was based on the concentrations normally used for testing bacteria that are used to determine the amount of vitamin b12 in human blood. Five tubes containing a concentration of f 0, 200 ng L-1 , 2 Œºg L-1, 20 Œºg L-1, 200 Œºg L-1 and 2 mg L-1 vitamin b12 and 0, 0.01, 0.05, 0.1, 0.5 and 1 mg mL-1 guanine were used.
These cultures were spin down at 4700 RPM for 3 minutes in order to compare the colouration of the pellets.
05 September - 05 October
For the second proof of principle experiment, agar plates were overgrown with E. coli containing the B/GRinv. In the middle of the plate a small scoop (0,5 cm in diameter) was taken out and filled with 100 ¬µL of a 50 mg mL-1 guanine solution or 100 ¬µL of a 200 mL L-1 vitamin b12 solution to create a decreasing gradient from the middle. To provide clear images showing the expressed mRFP, pictures were taking using the G:Box gel imager of Syngene. The RFP filter protocol  provided with the software programme was used.
iGEM Wageningen UR 2016. Send us an email or follow us on Facebook !
For more information about iGEM, check out their website and Twitter !
