Team:Wageningen UR/Notebook/Cas9
MENU ▤
HOME
TEAM
Team
★  Collaborations
PROJECT
★  Description
★ Design
Experiments
★ Proof of Concept
★ Demonstrate
Results
Notebook
PARTS
Parts
★ Basic Parts
★ Composite Parts
★ Part Collection
SAFETY
★  ATTRIBUTIONS
HUMAN PRACTICES
Human Practices
★ Silver
★ Gold
★ Integrated Practices
★ Engagement
AWARDS
★ Entrepreneurship
★ Hardware
★ Software
★  Measurement
★ Model
Wageningen UR iGEM 2016
☰
˟
Overview
Description
Results
Achievements
BioBricks
Specificity
Regulation
Biocontainment
Model
Outreach
Team
Notebook
 
 
 
Cas9-based
kill switch
May 15 - May 25
May 26 - June 6
June 7 - June 29
June 30 - July 31
Aug 1 - Aug 31
Sept 1 - Okt 10
No Cas9 BioBrick?
References
Overview
Description
Results
Achievements
BioBricks
Applied Design
WetLab
Specificity
In vitro assay
Toxin Engineering
Varroa Isolates
Regulation
Detecting Mites
Population Dynamics
Toggle Switch
E. coli survival
Biocontainment
Light Kill Switch
Cas9 Kill Switch
Collaborations
Model
Population Dynamics
Light Kill Switch
E. coli survival
Beehave
Toxin Scanner
Outreach
Our Vision
Design
Collaboration
Events
Media
Integrated Practices
Team
Members
Students
Supervisors
Gallery
Attributions
Contact
Safety
Environment
Laboratory
Notebook
Journals
Protocols
Unless indicated otherwise, all experiments were performed by Belwina
May 15 - may 25
Moving iGEM Cas9 to another backbone
Belwina, Marijn and Thomas transformed biobrick parts and backbones from the registry to home-made chemically competent DH5alpha cells. Transformations were plated on LB agar plates with antibiotics corresponding to the resistance in the backbone of each plasmid.
Transformed parts:
The plasmids were isolated from liquid cultures inoculated from colonies of transformation plates. Also, glycerol stocks were made.  The length of the dCas9 part (5080 bp) was verified using PCR with the VF2 and VR primer.
Moving dCas9 to pET26B
E. coli glycerol stocks containing pdCas9 (Addgene plasmid # 46569) and pET26B were streaked on LB plates with the appropriate antibiotic, liquid cultures were inoculated and the plasmids were isolated .
Q5 PCR was performed on these plasmids, to make fragments with cleaving sides for PacI and SpeI:
pdCas9:
Fwd primer: Bel-cas9 Cas9 fwd (acacacTTAATTAATGACAGCTTATCATCGATAAGCT)
Rev primer: Bel-cas9 Cas9 rev (acacacACTAGTTCAGTCACCTCCTAGCTGAC)
Annealing T: 60ºC, elongation time 1.5 minutes.
Expected band size: 5340 bp
pET26B:
Fwd primer: Bel-cas9 pET fwd (acacacactagtgcgcaacgcaattaatgtaag)
Rev primer: Bel-cas9 pET rev (acacacttaattaaatggatatcggaattaattcggatc)
Annealing T: 60ºC, elongation time 1.5 minutes.
Expected band size: 3737 bp
Fragments were checked on 1% agarose TAE gels, 100V, 30 minutes. The PCR for pET26B worked, whereas the one for pdCas9 did not.
Collection and construction of pEVOL plasmids
The following plasmids were received from Peter Schultz:
pEVOL-pAzF (Addgene plasmid # 31186)
pEVOL-pBpF (Addgene plasmid # 31190)
As well as the recoded strain C321ΔA-exp, from George Church (Bacterial strain #49018) All were delivered as agar stabs, that were streaked on LB plates with the appropriate antibiotic.
May 26 - June 6
General
Heat-shock competent cells were made according to the BacGen protocol . After testing by transforming with PUC19, transformation efficiency turned out to be 10^4.
Moving iGEM Cas9 to another backbone
iGEM Cas9, pSB1K3, pSB6A1 and pSB4K5 were digested with EcoRI-HF and PstI-HF. Fragments were checked on agarose gels. Estimated band size:
iGEM Cas9, 5080 bp
pSB1K3, 2204 bp
pSB6A1, 4022 bp
pSB4K5, 3419 bp
pSB1K3 certainly did not have the correct length, and the other two backbones were unclear, as they should have a different size but they did not. The digested iGEM Cas9 looked fine.
iGEM Cas9, pSB1K3 and pSB4K5 were cut from the gel and purified (Machery-Nagel nucleospin kit). Samples were kept in the freezer.
After a few days, I tried ligating iGEM Cas9 in pSB6A1 and pSB4K5 according to the standard protocol, transformed 2 µL into heat-shock competent cells (the ones mentioned above).
The ligation of Cas9 in pSB4K5 gave 7 colonies, but Colony PCR (primers: VF2 and VR) revealed there was no insert (expected band: 5080).
Another approach was was PCR amplifying Cas9 and backbones with VF2 and VR primers. Only the pSB1K3 backbone and iGEM Cas9 (5380 bp) gave a convincing band, so these PCR products were purified (hyperlink), digested with EcoRI-HF and PstI-HF, ligated and transformed into electrocompetent cells (made by Linea).
Moving dCas9 to pET26B
pdCas9 was verified using digestion with SacI-HF & SalI-HF, as well as digestion with SacI-HF and XbaI.
The PCR for pdCas9 was repeated under various conditions; a range of annealing temperatures, addition of 5% DMSO, and varying template concentrations. Nothing worked. So I looked closer at the primers, and they turned out to be suboptimal in terms of stability of the 3’ vs 5’ end.
My supervisor helped me design better primers:
BelCas9-cas9Fw2 (GCCTTAATTAATGACAGCTTATCATCGATAAGCTTTAATG)
BelCas9-cas9Rev2 (GCCACTAGTAATTGCATCAACGCATATAGCGCTAGCAG)
Q5 PCR was performed on pdCas9 using these primers:
Annealing T: gradient ranging from 60-72ºC, elongation time: 1.5 minutes
Expected band size: 5349 bp
And this time it worked.
PCR products of pdCas9 and pET26B were purified (hyperlink), digested with PacI and SpeI, ligated and transformed into electrocompetent cells (made by Linea).
Result of transformation of dCas9 and iGEM Cas9 ligations
Basically all transformations had colonies, also control transformations with only backbone. Only transformations without ligase were clean, indicating there was probably some back-ligation of the backbones.
Colony PCR s with VF2, VR, BelCas9-cas9Fw2 and BelCas9-cas9Rev2 (for iGEM ligations and own ligations, respectively) revealed there were no clones with the correct insert.
I decided to focus on my own Cas9 construct from now on, and drop the iGEM construct.
pT7-gRNA construction
A glycerol stock of E. coli containing pT7-gRNA was streaked on LB agar with ampicillin and allowed to grow overnight. A colony was picked for making a glycerol stock, as well as plasmid isolation (Machery-Nagel nucleospin kit).
June 7 - June 29
No labwork due to moving of the lab.
June 30 - July 31
General
Electrocompetent E. coli cells were made according to the protocol . Transformation efficiency was very high (could not count the colonies, really).
Moving dCas9 to pET26B
New PCR products were made of pdCas9 and pET26B. I proceeded with purification and digestion as usual, but added a step with alkaline phosphatase (CIP, NEB).
Performed ligation and electroporation of ligation. This time cloning was more successful: some colonies on ligation mixture, no colonies on backbone only control.
Colony PCR revealed that some colonies probably contained the correct construct. This was verified by sequencing.
Collection and construction of pEVOL plasmids
We received the biocontainment strains from Harvard, from which pEVOL-BipA was extracted (Machery-Nagel nucleospin kit).
Construction of pT7-gRNA plasmids The following primers were annealed as inserts:
pBbS5a (RFP) 1766-1785 FWD fwd 5’- TAGGgtggtccgctgccgttcgct-3’
pBbS5a (RFP) 1766-1785 FWD rev 5’- AAACagcgaacggcagcggaccac-3’
pBbS5a (RFP) 1739-1758 REV fwd 5’- TAGGaactttcagtttagcggtct -3’
pBbS5a (RFP) 1739-1758 REV rev 5’- AAACagaccgctaaactgaaagtt -3’
pBbS5a (RFP) 1821-1840 FWD fwd 5’- TAGGcaaagcttacgttaaacacc -3’
pBbS5a (RFP) 1821-1840 FWD rev 5’- AAACggtgtttaacgtaagctttg -3’
pBbS5a (RFP) 1803-1822 REV fwd 5’- TAGGtggaaccgtactggaactgc -3’
pBbS5a (RFP) 1803-1822 REV rev 5’- AAACgcagttccagtacggttcca -3’
We tried at first constructing the pT7-gRNA plasmids using protocol described in Jao et al. (2014), but this gave a lot of false positive colonies (on plates transformed without any insert).
Still, some plasmids were isolated from colonies of plates with insert, and digested with SalI-HF and ScaI-HF. Any positive clones should not be cut by SalI, because this restriction site is only present in the original backbone. Expected bands: 759 bp and 1782 bp. No positive clones were found.
The next strategy was to digest with BsmBI and SalI, isolate the linearized plasmid from gel , and proceed with a href="https://static.igem.org/mediawiki/2016/7/76/T--Wageningen_UR--Ligation.pdf">ligation.
This did give us some positive clones.
Procedure was repeated for gRNA 3 and 4. Results were later confirmed by sequencing (however, it turned out that gRNA 3 was not correct after all. It took another round of picking colonies/digestion/sequencing before we also got that one right).
Mutagenesis of dCas9-pET26B
The Ala10TAG and Ala840TAG mutations were introduced by mutagenesis PCR , using the following primers:
dCas9 Ala10TAG fwd 5'-ggcaaaaatggataagaaatactcaataggcttatagatcggcacaaatagcgtc-3'
dCas9 Ala10TAG rev 5'-gacgctatttgtgccgatctataagcctattgagtatttcttatccatttttgcc-3'
dCas9 Ala840TAG fwd 5'-taatcgtttaagtgattatgatgtcgattagattgttccacaaagtttccttaaagacg-3'
dCas9 Ala840TAG rev 5'-cgtctttaaggaaactttgtggaacaatctaatcgacatcataatcacttaaacgatta-3'
First PCRs revealed that only the Ala840TAG PCR was successful, as was revealed by gel electrophoresis (expected band size for both: 9077 bp).
The Ala10TAG mutation worked after addition of GC enhancer to the PCR mixture.
Mutations were verified after sequencing of isolated plasmids.
Aug 1 - Aug 31
Expression of Cas9-pET26B in C321ΔA
Cas9-pET26B constructs as well as iGEM-Cas9 and the original pdCas9 were transformed in E. coli C321ΔA as described in Lajoie et. al (2013) protocol 1 for electroporation, successfully.  Later, also, pEVOL-BipA, pEVOL-pAzF and pEVOL-pBpF were transformed into C321ΔA, both with and without Cas9-pET26B constructs.
A first expression experiment was done with 50 ml overnight cultures of C321ΔA + Cas9 construct, in LB with the appropriate antibiotic.
Cells were spun down, resuspended in 10 ml lysis buffer (50mM Tris-HCL, 250 mM NaCl, 1mM EDTA) and lysed by sonication (4x15 sec, 25Am).
Protein concentrations were measured with a Bradford assay.
20 ug of each extract was loaded on SDS to check for Cas9 expression. The expected weight of Cas9 is 156 kDa, of dCas9-Ala10TAG is 1 kDa (can’t be seen anyways), and of dCas9-Ala840TAG it is 97 kDa. No such bands could be observed (possible also due to a background band of the same size)
because for the original iGEM construct, the band seemed to be a bit more pronounced, I grew new cultures and repeated the experiment. This time, it was really obvious that expression levels were too low.
in vitro transcription of guide RNAs By the time of transcription, guide 3 had not been verified by sequencing yet, so only guide 1, 2 and 4 were transcribed and purified, according to the protocol .
I only have a picture of the gel after cutting the RNA bands, but they were present.
After purification, guide 2 and 4 had decent concentrations of ~350 ng/uL. Guide 1 had only 35 ng/uL.
Cloning of Cas9 variants in expresso vector
After discussion with an employee in the departement who has experience expressing Streptococcus pyogenes Cas9, it was decided that expression from pdCas9 is probably too low to visualize on SDS-PAGE, and perhaps not suitable for further in vitro testing.
So it was decided to clone Cas9 in the Expresso c-rham vector system .
First, Cas9 variants and the Expresso vector were amplified by PCR . The following reactions were performed:
Expresso
fwd: CATCATCACCACCATCACTAATAG
Rev: CATATGTATATCTCCTTCTTATAGTTAAAC
Annealing T: 59ºC, elongation time 2.5 minutes.
Expected band size: 2275 bp
iGEM Cas9
Fwd: gtttaactataagaaggagatatacatatgGATAAGAAATACTCAATAGGCTTAGATATC
Rev: gccgctctattagtgatggtggtgatgatgGTCACCTCCTAGCTGACTCAAATC
Annealing T: 64ºC, elongation time 2.5 minutes.
Expected band size: 4088 bp
dCas9 & Ala840TAG:
Fwd: gtttaactataagaaggagatatacatatgGATAAGAAATACTCAATAGGCTTAGCTATC
Rev: gccgctctattagtgatggtggtgatgatgGTCACCTCCTAGCTGACTCAAATC
Annealing T: 66ºC, elongation time 2.5 minutes.
Expected band size: 4088 bp
Ala10TAG:
Fwd: gtttaactataagaaggagatatacatatgGATAAGAAATACTCAATAGGCTTATAGATC
Rev: gccgctctattagtgatggtggtgatgatgGTCACCTCCTAGCTGACTCAAATC
Annealing T: 64ºC, elongation time 2.5 minutes.
Expected band size: 4088 bp
Positive control: Some ~2000 bp thing from Thomas with iGEM prefix and suffix primers.
Fragments were checked by gel electrophoresis .
PCR products were cleaned with Zymo kit (hyperlink), eluted in water and assembled by Gibson Assembly . A vector : insert ratio of 1 : 2 was used, with 100 ng vector. 1 uL of Gibson mixtures were transformed in 25uL commercial competent cells (NEB) according to the accompanying protocol and plated on LB plates with kanamycin.
Colonies that came up were verified with Colony PCR .
Primers that were used:
Fwd: TTGAAGGGTAGTCCAGAAG
Rev: CATATGTATATCTCCTTCTTATAGTTAAAC
Annealing T: 46ºC, elongation time 3 minutes.
Expected band size: 2647 bp.
PCRs were verified using gel electrophoresis . It seemed that there were a lot of positive colonies.
Correct clones were confirmed by sequencing.
Collection and construction of pEVOL plasmids
pEVOL-Asp was constructed according to the yeast assembly protocol .
pEVOL-pAzF
Fwd: ACTAGTGCATGCTCGAGCAG
Rev: CCTCCTGTTAGCCCAAAAAAACGGGTATG
Annealing T: 68ºC, elongation time 2 minutes.
Expected band size: ~3320 bp
pYES2
Fwd: gagcaggcttttttactagtACTCTTCCTTTTTCAATGGG
Rev: aaagcaaattcgaccctgagctgctcgagcatgcactagtAAATATTTGCTTATACAATCTTCC
Annealing T: 56ºC, elongation time 2 minutes.
Expected band size: 2667 bp
gBlock1
Fwd: gagcaggcttttttactagtACTCTTCCTTTTTCAATGGG
Rev: ACAGGGTATTGCTTACGTACCAACTC
Annealing T: 66ºC, elongation time 2 minutes.
Expected band size: 1203 bp
gBlock2
Fwd: TTGCTCATGAAATTGAGTTGGTACGTAAG
Rev: CCCATTGAAAAAGGAAGAGTACTAG
Annealing T: 64ºC, elongation time 2minutes.
Expected band size: 1230 bp
PCR products were verified using gel electrophoresis .
PCR products were cleaned up using the Zymo kit (hyperlink).
Then, yeast assembly was performed using the protocol , with competent yeast cells received from a supervisor.
From the resulting colonies, 6 were picked for plasmid isolation . Only 3 of them had some plasmid yield, which were checked for correct assembly using OneTaq PCR .
PCR reactions that were performed:
pEVOL fwd and gBlock 1 rev primers, annealing T: 56ºC, elongation 4 minutes. Expected fragment size: 4469 bp.
gBlock 1 fwd and gBlock 2 rev primers, annealing T: 50ºC, elongation 4 minutes. Expected fragment size: 2318 bp.
gel electrophoresis reveiled fragments of the right size for colony 2 and 3.
Eventually, the plasmid from colony 3 was transformed successfully in E. coli, miniprepped and sent for sequencing. The following mutations were present: Tryp156Cys, Gly321Val and Gly525Cys. Because there was no time to check other clones, I continued with this plasmid anyways.
Expression of Cas9-expresso constructs in C321ΔA
Both the acquired Cas9-expresso constructs as well as pEVOL-Asp were transformed into C321ΔA as described in Lajoie et al. (2013) 1 .
Sept 1 - Okt 10
Expression of Cas9-expresso constructs in C321ΔA
An expression experiment was performed with 3 ml cultures induced overnight with rhamnose, arabinose and synthetic amino acids when applicable. This yielded no visible Cas9 bands.  The same happened when 5 ml cultures where induced for 4 hours. What worked, was the protocol with bigger volumes followed by Ni-NTA purification (the majority of the actual work with the FPLC was performed by a supervisor)
First, samples were purified as described, but with addition of DNAse. This gave good yields, but DNAse remained in the purified fractions as was later found out during in vitro Cas9 assays. However, without DNAse also a good yield was obtained.
Click the figure for the full-resolution image.
The green line indicates the amount of His buffer B that is passed through the column.
SDS-PAGE was performed on different fractions after purification. a) Cas9. b) dCas9. c) dCas9-Ala10BipA. d) dCas9-Ala10Asp. e)dCas9-Ala10TAG, no synthetic amino acid.
in vitro Cas9 cleaving assays
Assays were performed with all produced guide RNAs, according to the protocol . Substrate for cleaving was a PCR product including the gene encoding RFP, which is targeted at the N-terminal side, both on the template strand (guideRNA 2 and 4) and the non-template strand (gRNA 1). Size of the uncleaved PCR product is 4140 bp, cleaving generates a 3100 bp and a 1040 bp fragment.
No Cas9 biobrick?
When I started making the constructs for the Cas9 kill switch, two approaches were taken: taking the Cas9 that is available in the iGEM registry (BBa_K1218011) and pdCas9 (Addgene plasmid # 46569) as a starting point for making mutations and expressing Cas9. I did not manage to transfer BBa_K1218011 to another backbone. Furthermore, when cultures transformed with BBa_K1218011 were checked for Cas9 expression with SDS-PAGE, no convincing Cas9 band could be observed. Because of time limitations I decided to continue working with the Addgene construct for making the mutations, and chose an established system for protein expression. For that reason I did not submit any Cas9-biobricks. Furthermore, the pEVOL construct containing an aminoacyl-synthetase and a tRNA for introducing BipA in response to the TAG stopcodon were isolated from a strain kindly received from George Church (described in Mandel et al., 2015 ). The MTA that was signed to receive the strain does not allow for redistribution.
References
1. Lajoie, M. J., Rovner, A. J., Goodman, D. B., Aerni, H. R., Haimovich, A. D., Kuznetsov, G., ... & Rohland, N. (2013). Genomically recoded organisms expand biological functions. Science, 342(6156), 357-360. ↩
iGEM Wageningen UR 2016. Send us an email or follow us on Facebook !
For more information about iGEM, check out their website and Twitter !
