<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Notebook_InVitroAssay skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Notebook/InVitroAssay</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:Wageningen_UR/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design"> ★ Design </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Proof"> ★ Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Basic_Part"> ★ Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Composite_Part"> ★ Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Part_Collection"> ★ Part Collection </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Human_Practices"> Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Silver">★ Silver </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">★ Gold </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"> ★ Integrated Practices </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Engagement">★ Engagement </A></LI><LI class="menu_item"> AWARDS 
			</LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Entrepreneurship"> ★ Entrepreneurship </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Hardware"> ★ Hardware </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">★ Software </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Measurement">★  Measurement </A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">★ Model </A></LI></DIV><DIV class="content_wrapper"><TITLE>Wageningen UR iGEM 2016 </TITLE><DIV id="wrapper"><DIV id="innerwrap"><DIV id="left"><BUTTON class="hamburger">☰</BUTTON><BUTTON class="cross">˟</BUTTON><DIV class="mobilemenu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><B>Overview</B></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Team</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Notebook</A></LI></UL></DIV><P> </P><DIV id="inner-left"><P> </P><DIV class="sidebar-container"><P> </P><SIDENAV><DIV class="menu-head"><H4><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook/InVitroAssay">Logbook</A></H4></DIV><LI class="menu-item"><A href="#july">July</A></LI><LI class="menu-item"><A href="#august">August</A></LI><LI class="menu-item"><A href="#september">September</A></LI><LI class="menu-item"><A href="#oktober">Oktober</A></LI><LI class="menu-item"><A href="#appendices">Appendices</A></LI><LI class="menu-item"><A href="#references">References</A></LI></SIDENAV></DIV></DIV></DIV><DIV id="center"><DIV class="navmenubkg"><DIV id="nav_wrapper"><DIV id="ca-menu"><UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR"><DIV class="ca-content">Overview</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Parts">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Design">Applied Design</A></LI></UL><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description"><DIV class="ca-content">WetLab</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity">Specificity</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Assay"><I>In vitro</I> assay</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#ToxinEngineering">Toxin Engineering</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Specificity#Isolates" class="up_menu_fix"><I>Varroa</I> Isolates</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation">Regulation</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#DetectingMites" class="up_menu_fix">Detecting Mites</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#QuorumSensing" class="up_menu_fix"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Regulation#Toggleswitch" class="up_menu_fix">Toggle Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/ecoli_survival"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment">Biocontainment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#Lightswitch" class="up_menu_fix">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Description/Biocontainment#SAA" class="up_menu_fix">Cas9 Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model"><DIV class="ca-content">Model</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#quorum"><FONT size="1">Population Dynamics</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#light">Light Kill Switch</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#metabolic"><I>E. coli</I> survival</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Model#beehave">Beehave</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Software">Toxin Scanner</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold"><DIV class="ca-content">Outreach</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Vision">Our Vision</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#collaboration">Collaboration</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#events">Events</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/HP/Gold#media">Media</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Integrated_Practices"><FONT size="1">Integrated Practices</FONT></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members"><DIV class="ca-content">Team</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members">Members</A></LI><LI class="up_menu_fix"><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#students">Students</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/team/members#supervisors">Supervisors</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/gallery">Gallery</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/contact">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety"><DIV class="ca-content">Safety</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#environment">Environment</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Safety#laboratory">Laboratory</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook"><DIV class="ca-content">Notebook</DIV></A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Notebook">Journals</A></LI><LI><A href="https://2016.igem.org/Team:Wageningen_UR/Experiments">Protocols</A></LI></DIV></DIV></DIV><DIV class="center_content"><P>This experiment was performed Jaccoline Zegers and she was helped by Linea Muhsal and Lisa Röttjers.</P><SECTION><H1><B>Notebook <I>In Vitro</I> Assay</B></H1></SECTION><SECTION id="july"><H2><B>July</B></H2></SECTION><H3><B>Week 1. (04/07-10/07)</B></H3><P>An experimental plan was written. Lb medium, LB Agar medium, SOC medium, ampicillin stocks, plate stocks were prepared. </P><H3><B>Week 2. (11/07-17/07)</B></H3><P>A Q5-PCR on pSB1A3 was performed with the prefix reverse and suffix forward primers to obtain linearized plasmid. The expected size of linearized pSB1A3 is 2155 kB. The DNA was cleaned-up afterwards.
Solution Buffer, Homogenization Buffer, Dissection Buffer, 24 mM MgCl2 and DTT stocks were prepared. Mealworms were dissected. The first BBMVs from the midgut of mealworms were made. TEM pictures of the first samples were made. From this could be concluded that all fat on the outside of the mealworm gut should be removed before making BBMVs.</P><H3><B>Week 3. (18/07-24/07)</B></H3><P>More mealworms were dissected. 12 different samples of BBMVs with variations in the protocols were made. An emission spectra, excitation spectra, and calibration curve of fluorescein were made. <I>Varroa</I> mites were dissected. Obtaining the guts of <I>varroa</I> mites did fail, because the <I>varroa</I> mites were too small. 12 mites were stored at -80⁰C until further use. DLS measurements were performed on the 12 BBMVs samples from T. molitor.</P><H3><B>Week 4. (25/07-31/07)</B></H3><P><I>B. thuringiensis</I> genomic DNA was isolated and a Q5 colony PCR on <I>B. thuringiensis</I> was performed to get the Cry3Aa gene (with RBS, HIS-tag, TEV sequence to remove the HIS-tag, right restriction sites, and overhangs) with the following primers:
5’_CGGGGCCCGCGGCCGCTCTAGAGAAAGATAGGAGACACTAGATGATAAGAAAGGGAGGAAGA_3’  5’_CGGGCCACTAGTTTATTAGTGATGATGGTGGTGATGGTGATGCCCCTGGAAGTACAAGTTCTCATTCACTGGAATAAATTCAAT_3’
Restriction digestion was performed with the linearized pSB1A3 plasmid, the Cry gene and the promoter BBa-I0500. The first time wrong restriction enzymes were used. Repeated with right restriction enzymes. Ligation and transformation in <I>E. coli</I> DH5a. Only three recombinants were created, while the positive control had 52 colonies. Colony PCR on the recombinants was done and shows only empty pSB1A3 without insert is present.</P><SECTION id="august"><H2><B>August</B></H2><H3><B>Week 5. (01/08-07/08)</B></H3><P>Measurement buffer was prepared and with this the first CF leakage experiment was performed with tritonX-100. Only two points in time were measured: nothing could be concluded from this.
Restriction digestion of pSB1A3 with an insert was performed in order to use this for the pSB1A_Cry3Aa_TEV_HIS construct. An attempt to create this construct was made by ligation and transformation into <I>E.coli</I> DH5alpha. this yielded many colonies. A colony PCR was done with 90 colonies, however none appeared to have the right insert. All plasmids ligated back with the original insert or without any insert at all. 
More mealworms were dissected. From this BBMVs were made and incorporated with fluorophores. More CF leakage experiments were performed with TritonX-100. This compound did not give a large and always inconsequent change in fluorescence when added to BBMVs. Furthermore, it changed the fluorescence of carboxyfluorescein itself. 
Chloramphenicol and Ampicillin plates were poured.</P><H3><B>Week 6. (08/08-14/08)</B></H3><P><I>E. coli</I> with the plasmid pSB1A3_RFP, kindly provided by Carina, were grown. The plasmid was miniprepped and restriction digested. This failed, because the yield of DNA was too low with the miniprep. Leakage of fluorophore from vesicles was tried with different detergents. Tween-80 and dish soap influenced the fluorescence of CF itself negatively. Buttermilk handsoap had a very small effect on the fluorescence of carboxyfluorescein itself and could induce a fluorophore release.</P><H3><B>Week 7. (15/08-21/08)</B></H3><P>No lab work was done this week</P><H3><B>Week 8. (22/08-28/08)</B></H3><P><I>B. thuringiensis</I> was grown on sporulation salts and incubated with normal BBMVs from <I>T. molitor</I>. Addition of this sample to BBMVs incorporated with fluorophores had no effect. pSB1A3_RFP was miniprepped again, then digested and treated with alkaline phosphatase. All fragments to create pSB1A3_Cry3Aa_TEV-HIS were ligated and transformed. No colonies were present. The cells were probably not competent, since the positive control did not give a single colony. SDS-PAGE with BBMVs from T.molitor was performed and showed the presence of different proteins. Fluorophore leaking of triton X-100 again showed to not work.</P></SECTION><SECTION id="september"><H2><B>September</B></H2><H3><B>Week 9. (29/08-04/09)</B></H3><P>More mealworms were dissected. BBMVs from both mealworms and varroa mites were made. Vesicles were tested whether they would break in the presence of SDS. They did. An SDS-PAGE of all vesicles and protein extracts from different  strains <I>Bacillus</I>, kindly provided by Lisa, was performed. Transformation was performed with new competent cells and ligation mix made a week earlier. This resulted in colonies.</P><H3><B>Week 10. (05/09-11/09)</B></H3><P>A colony PCR with verification primers was performed on seven colonies. 6 colonies did not have an insert. One colony had two bands: one with the length that correspondents for no insert and one that correspondents for the right insert. The bacteria used for this PCR were streaked out on a plate and again a colony PCR was performed with verification primers on 6 colonies. 4 Colonies had an insert with the right length and 2 contained an empty plasmid.
An overnight measurement of vesicles breaking of BMMVs from mealworms and varroa mites was performed in the presence and absence of SDS. Kinetic values were calculated for standard breaking. Addition the protein extracts from different <I>Bacillus</I> strains (including the Cry3Aa strain) did not change the values.
Cry3Aa was dissolved under alkaline conditions out of the protein extract. This was analyzed with SDS-PAGE.</P><H3><B>Week 11. (12/09-18/09)</B></H3><P>More BBMVs out of <I>varroa</I> mites were made. These were analyzed with DLS. A glycerol stock of <I>E. coli</I>DH5alpha with the Cry Gene was made. The plasmid was miniprepped. More solution buffer was made. Negative controls for fluorophore leaking experiments were made. Experiments with Cry3Aa (obtained by dissolving the protein extract under alkaline conditions) were performed and showed positive results. However, graphs start looking weird after a certain time. Bacteria from Delft were inoculated. Later from these bacteria, samples were prepared for fluorescence measurement and this measurement was performed.</P><H3><B>Week 12. (19/09-25/09)</B></H3><P>Sequencing of pSB1A3_Cry3Aa_TEV_HIS was performed with VR, VF2, and CRY-F primer. Leaking experiment from last week was repeated, however, mixing the samples this time better beforehand. This resulted in better graphs. Results were still positive.</P><LI><A href=" https://static.igem.org/mediawiki/2016/6/6b/T--Wageningen_UR--RawDataBBMVs.pdf
 ">Link to raw data.</A></LI><P>More LB medium was made. <I>E.coli</I> DH5aplha with pSB1A3_Cry3Aa_TEV_HIS was induced with arabinose and proteins were extracted. The protein extraction was run on a gel, but the negative controls were forgotten. Therefore, no conclusions could be made. The Cry3Aa protein was concentrated with the help of an amplicon. However, the concentration increases are doubtful. Leaking experiments with this protein sample did not give better results than obtained earlier. pSB1A3_Cry3Aa_TEV_HIS was sent for sequencing with the primers:
5’_gtctacactttatgtgtc_3’
5’_tcggcaaacaaattctcgt_3’
</P><H3><B>Week 13. (26/09-02/10)</B></H3><P><I>E. coli</I> DH5aplha with pSB1A3_Cry3Aa_TEV_HIS was induced with arabinose. The cell pellet was dissolved in Bper. Leaking experiments with this supernatant did not give any positive results. SDS-PAGE was performed. Together with Linea, for <I>E. coli</I> DH5aplha with pSB1A3_Cry3Aa_TEV_HIS and <I>E. coli</I> BL 21 pSB1A3_mutant_Cry3Aa_TEV_HIS, the protocol for expression of Cry3Aa-HIS by <I>E.coli</I> BL21 pSB1A3_Cry3Aa_TEV_HIS and the protocol for carboxyfluorescein leakage measurement with Cry3Aa obtained from <I>E. coli</I> were followed. On top of that, the pellets were also added to the measurement. SDS PAGE with some protein extract was performed, sadly enough some lanes were forgotten. The pSB1A3_Cry3Aa_TEV_HIS was successfully transformed to <I>E.coli</I> BL21. Three more plates with different Cry proteins were measured following the protocols mentioned above. One failed because the quality of the vesicles dropped after thawing multiple times. More solution buffer was made.</P></SECTION><SECTION id="oktober"><H2><B>Oktober</B></H2><H3><B>Week 14. (03/10-09/10)</B></H3><P>Together with Linea, three more plats were measured following the protocols for expression of Cry3Aa-HIS by <I>E. coli</I> BL21 pSB1A3_Cry3Aa_TEV_HIS and the protocol for carboxyfluorescein leakage measurement with Cry3Aa obtained from <I>E. coli</I>. All with mite vesicles.  One more SDS-gel of protein extract was made and the positive control experiment was repeated.</P><H3><B>Week 15. (10/10-16/10)</B></H3><P>Linea did a PCR on pSB1A3_Cry3Aa_TEV_HIS to remove one of the four forbidden restriction sites. Gel electrophoresis showed that it failed, because only primer dimers were formed. </P></SECTION><SECTION id="appendices"><H2><B>Appendices</B></H2><H3><B>Protein in Brush Border Membrane Vesicles</B></H3><P> To visualize the proteins in the BBMVs, SDS-PAGE was performed. The results, shown in Figure A1, show the presence of a variety of proteins. From the results of the SDS-PAGE we were not able to conclude which proteins are membrane proteins. </P><FIGURE><FIGCAPTION>Figure A1. SDS-PAGE gel. M is the marker lane. In the other lanes the following samples are visualized: Lane 1, protein extract from <I>T. molitor</I> midgut 100x diluted. Lane 2, BBMVs from <I>T. molitor</I> 2x diluted. Lane3, BBMVs from <I>T. molitor</I> 6x diluted. Lane 4, BBMVs from <I>V. destructor</I> 2x diluted. Lane 5, protein extract from <I>V. destructor</I>. Lane 6, protein extract from <I>V. destructor</I> undiluted.</FIGCAPTION></FIGURE><H3><B>The non-biobrick <I>cry3Aa</I> with araC/pBAD </B></H3><P>We intended to create the mutant <I>cry3Aa</I> mutant library as biobricks. To be able to quickly, in other words before this library was finished, test the BBMVs with a Cry toxin, the non-biobrick <I>cry3Aa</I> with araC/pBAD was made. The construct can be seen in Figure A2. This construct has an HIS-tag and a TEV-sequence for better purification. Another advantage of this construct is that it can be made faster, since the four forbidden restriction sites, which naturally occur in our <I>cry3Aa</I> gene, do not have to be removed. When in a later stage of our research, creating the mutant <I>cry3Aa</I> library kept failing, we decided to use the non-biobrick <I>cry3Aa</I> with araC/pBAD. With mutagenesis PCR we adapted the binding motifs in the <I>cry3Aa</I> gene on this construct. This was finished merely a few weeks before the wiki freeze. We intended to remove all the forbidden restriction sites, so we would be able to submit it as a real biobrick. Even though we ordered the primers to perform the four mutagenenis PCR's, we were not able to finish perform those in time. Since future iGEM teams still may want to use the gene, we did sent it for shipping to the iGEM headquarters.</P><FIGURE><FIGCAPTION>
Figure A2. Schematic overview of elements contained in this non-BioBrick.
</FIGCAPTION></FIGURE><H3><B>A Positive Control for Testing the Cry3Aa Mutant Library</B></H3><P>Taking a closer look at the protocol of the carboxyfluorescein leaking experiment performed earlier (link to protocol), the pH of the environment of the BBMVs was increased prior to the measurement itself. The increase of the pH influences the fluorescence of 6-carboxyfluorescein<SUP><A href="#jz8" id="refjz8">1</A></SUP> and could also influence the speed at which standard fluorophore leaking occurs. Hence, it will be difficult to compare data obtained at a normal pH compared to data obtained at a higher pH. On top of that, comparisons becomes even more difficult, since the protein concentrations are not known.In order to be be able to compare the influence of Cry3Aa on <I>T. molitor</I> BBMVs and the influence of Cry3Aa with mutations in their binding sites on <I>V. destructor</I> BBMVs, the following construct, see Figure A3, was made by restriction digestion and ligation, creating pSB1A3. The plasmid was transformed into <I>E. coli</I> BL21(DE3).Sequencing results (link to sequencing results) and a colony PCR showed that this construct was correct for colonies 1 to 4.</P></SECTION><FIGURE><FIGCAPTION>Figure A3. Colony PCR of transformed <I>E.coli</I> with the verification primers. Lane 1 to lane 6 are transformants. Lane 7 is the positive control (pSB1A3 with RFP) and lane 8 is the negative control.</FIGCAPTION></FIGURE><P>The proteins expressed by this bacteria were extracted and tested on BBMVs obtained from <I>T. molitor</I>. As a negative control the proteins expressed by the same strain but without the pSB1A3_Cr3Aa_TEV_HIS construct were extracted and simultaneously tested on brush border membrane vesicles obtained from <I>T. molitor</I>. 8 measurements were performed for both Cry3Aa and a blank. However, the obtained k<SUB>f</SUB>-values did not show a significant difference.Expression of the protein was visualized with SDS-PAGE. Both supernatant and pellet of the protein extracts were run with SDS-PAGE. A picture of the gel can be found in Figure A4. The predicted size of the Cry protein with HIS-tag is 75 kDa. This band is not visible in the supernatant, but a small band around this size can be found in the pellet. An explanation for this is the formation of inclusion bodies around the Cry proteins, which is known to happen with the expression of Cry proteins in <I>E. coli</I>. The inclusion bodies can be solubilized in an alkaline solution<SUP><A href="#jz1" id="refjz1">1</A></SUP>. For the carboxyfluorescein leakage experiments, only the supernatant data could be analysed,  because the pellet contained too much contaminants, such as lipids, to have a reliable measurement. The absence of a band probably explains that the concentration of Cry protein is too low to influence the carboxyfluorescein leakage measurement noticeably.</P><FIGURE><FIGCAPTION>Figure A4. SDS-PAGE gel. M represents the marker lane. In lane 1 and 2 is the supernatant of the protein extract is visible and lane 3 and 4 represent the pellet. Lane 2 and 4 are the negative controls, whereas the protein extracts from lane 1 and 3 originate from transformant. The expected size of the Cry3Aa_TEV_HIs protein is 76 kDa.</FIGCAPTION></FIGURE><P>From this can be concluded that the current method <A href="https://2016.igem.org/Team:Wageningen_UR/Experiments#engineering ">method </A> that is used to extract the Cry proteins from E.coli is perhaps not the best method when testing Cry proteins on BBMVs. It is not expected that Cry3Aa proteins with mutations in their binding sites obtained by this method will result in accelerated fluorophore leakage within V. destructor BBMVs. If this happens nevertheless, this might be due to two reasons. Either there were other substances in the samples present that caused this, or a functional Cry toxin for <I>V. destructor</I>, with a higher efficacy than Cry3Aa on <I>T. molitor</I>, was present. To confirm this, the protein should first be produced in higher concentrations and preferably purified. The latter is possible due to the presence of a HIS-tag. The measurements then need to be repeated. For now, all results, both negative and positive, obtained with the Cry toxins produced by <I>E. coli</I>, are preliminary indications.</P><SECTION id="references"><H2><B>References</B></H2><P><A id="jz1" href="http://www.microbiologyresearch.org/docserver/fulltext/micro/141/3/mic-141-3-629.pdf?expires=1476183703&amp;id=id&amp;accname=guest&amp;checksum=353BF085DA839D919115CEFF2276A2D6">1.</A> Wabiko, Hiroetsu. Yasuda, Eriko. <I>Bacillus thuringiensis</I> protoxin : location of toxic border and requirement of non-toxic domain for high-level <I>in vivo</I> production of active toxin. Microbiology (1995), 141, 629-639. <A href="#refjz1" title="Jump back to footnote 2 in the text.">↩</A></P></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="footer"><P>iGEM Wageningen UR 2016. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A>!
			
			For more information about iGEM, check out their <A class="soft_link" href="https://igem.org">website</A> and <A class="soft_link" href="http://twitter.com/igem">Twitter</A>!</P></DIV></DIV></BODY></HTML>