<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Valencia_Biocampus_ResultsUpdated"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Valencia_Biocampus/ResultsUpdated">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Valencia_Biocampus/ResultsUpdated&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ResultsUpdated&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ResultsUpdated&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Valencia_Biocampus/ResultsUpdated" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Valencia Biocampus/ResultsUpdated</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="header_wrapper"><DIV id="top_menu_wrapper"><UL id="top_menu" class="container"><LI><A href="https://2013.igem.org/Team:Valencia_Biocampus" id="menu_a_home" class="active">Home</A></LI><LI><A href="https://2013.igem.org/Team:Valencia_Biocampus/Team" id="menu_a_team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Valencia_Biocampus/Project" id="menu_a_project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Valencia_Biocampus/Results" id="menu_a_results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Valencia_Biocampus/Modeling" id="menu_a_modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Valencia_Biocampus/Devices" id="menu_a_devices">Devices</A></LI><LI><A href="https://2013.igem.org/Team:Valencia_Biocampus/HP" id="menu_a_HP">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:Valencia_Biocampus/Achievements" id="menu_a_achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Valencia_Biocampus/Attributions" id="menu_a_attributions">Attributions</A></LI></UL></DIV></DIV><DIV class="container"><P><UL class="nav nav-tabs" id="slider-id-nav-ul"><LI class="active tab1"><A href="#the_riding">The Riding</A></LI><LI class="tab2"><A href="#the_calling">The Calling</A></LI><LI class="tab3"><A href="#the_clumping">The Clumping</A></LI><LI class="tab4"><A href="#the_building">The Building</A></LI><LI class="tab5"><A href="#pre-field_test">Pre-field test</A></LI></UL></P><DIV class="tab-content" id="results_section"><DIV class="tab-pane fade active in" id="the_riding"><H1 class="title2 riding-color">The Riding</H1><P>
                In this synthetic symbiosis<B>, <I>C. elegans</I></B>acts as a transport for engineered bacteria <B>(<I>Pseudomonas putida</I></B><I>) </I>in order to
                take them to the hotspot of interest, because bacteria are not able to move fast through solid or semi-solid substrates but they are very
            interesting from a biotechnological point of view. This is the reason why we thought about this innovative mode of transport: the regulated <B> formation of a biofilm</B>.
            </P><P>
                To achieve our goal, we constructed a BioBrick (see part: <U><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1112001">BBa_K1112001</A></U>)
                consisting of the coding sequence of the <B>hmsHFRS operon</B>, an adhesion operon natural from <I>Xenorhadbus nematophila</I> which allows the formation
            of a biofilm on the nematode <I>S. carpocapsae</I>, under the control of a nitrogen sensitive promoter (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K763003"><B>pGlnA, </B>characterized by the    <B> 2012 Valencia Biocampus iGEM team</B></A>). (<B>Fig.1</B>)
            </P><H3>Controlling the mechanism</H3>
            Riding is a regulated process: with <B>low nitrogen </B>in the media, the promoter is activated and <I>hms</I> genes are expressed, triggering the    <B>formation of the biofilm </B>over <I>C. elegans</I>; in contrast, with <B>high nitrogen </B>concentrations, such as the ones found in nutrient-rich hotspots, the promoter is repressed, so bacteria can <B>“get off</B>” of the nematode <B>(Fig. 2)</B>.
            <P style="text-align:center">
            The manufacturing of the plates can be seen in <A href="http://www.youtube.com/watch?v=5MIEVW6NedE">this video</A>.
            </P><H3>Biofilm formation in genetically-engineered bacteria</H3>

                Our original idea was to introduce the Biobrick in <B><I>Pseudomonas putida </I></B>, a bacterial species with wide applications in biotechnology. To do
                so, we cloned the construction (<B>Fig.1)</B> in the <B>pIZ1016 vector</B>, which has a replication origin compatible with <I>Pseudomonas</I>. We
                successfully performed the cloning <B>(Fig.3)</B>, but the efficiency of the transformation was too low, so we have not been able to obtain <I>P. putida </I>
                transformants as of yet. This is probably a consequence of the length of the construction, 6,5 kb, which decreases transformation efficiency.
            <P>
                But far from being disheartened, we decided to express the construction with <B><I>E. coli.</I></B>We cloned the construction in the pUC57 vector,
            obtained transformant <I>E. coli</I>, and then grew them in medium with low nitrogen (0.6 g/L) in order to induce the formation of the biofilm. <I>C. elegans </I>was fed with these induced bacteria, and then several worms were isolated in order to check biofilm formation with    <B>scanning electron microscopy (SEM) </B>imaging. As you can see in <B>Fig.4</B>, it actually worked! We observed a formation of an <I>E. coli </I>biofilm
                over the nematode!
            </P></DIV><DIV class="tab-pane fade" id="the_calling"><H1 class="title2">The Calling</H1><H3>Looking for an attractant</H3>
            When we were considering setting up a transport for bacteria, we thought it was necessary to have a 'destination', a place to go. That destination would be a    <B>'hot spot' on a heterogeneous substrate</B> where <B><I>Caenorhabditis</I><I>elegans</I></B> should carry the bacteria to.
                
            Leveraging the powerful smell of the nematode, it was decided to try a number of attractants from various lists from the web <A href="http://www.wormbook.org">www.wormbook.org</A> that could work as 'hot spots' in our experiment. Thus, using <I> C. elegans</I><B>chemotaxis, </B>we would be able to direct transport.
   <H3>The attractants experiment</H3>

                The test was carried out by creating our own plates in which half plate would be unmodified NGM and the other half  would include soluble compounds before
                solidifying, or after solidification (in the case of volatiles). The list of modifications can be found in <B>Fig. 1</B>.
            <P>
                To place <B><I>C. elegans</I></B>on the plate, different cuts were made on a fresh plate of NGM and the resulting small pieces were placed in the exact
                center of the 50% - 50% plates to determine which side of the plate the nematode preferred, one per Petri plate.
            </P><P>
                The results after counting 2 replicates per attractant can be seen in <B>Fig. 2</B>.
            </P><P>
                From <B>Table 2 </B> many of the attractants that were thought viable were discarded.
            </P><P>
                Volatile attractants were not a good choice because they evaporate quickly (which would also be a limitation for the field experiments).
            </P><P>
                Another impediment arose. Once we had already performed the experiments, we decided that the promoter that would control the production of interference RNA
                in <I>Escherichia coli</I> would be regulated by nitrogen, so amino acids had to be discarded as attractants; because it would modify the controlled expression of E.coli.
            </P><P>
                That left MgSO<SUB>4</SUB> and hypoosmotic media as potential attractants.
            </P><H3>Attractant final choice</H3>
            Because the hyposmotic medium could interfere with the proper growth of bacteria (our nematode's food), the final decision was to choose MgSO<SUB>4</SUB> as our attractant.
            <P>
                The question at that time was: <I>How can we multiply the amount of MgSO<SUB>4</SUB> to increase the efficiency?</I></P><H3>MgSO<SUB>4</SUB> efficiency</H3>

            Once we had selected the most feasible attractant for the ‘transport’ during our experiments, we needed to know which could be the largest concentration of MgSO<SUB>4</SUB>, in order to optimize the nematode's attraction.
            
             To choose the concentration, attraction was tested in a battery of increasing concentrations (regarding the initial medium of 1 ml / L). Bearing in mind the results of
                the x2 factor, we decided to test other factor concentrations: x3, x4, x5, x8 and x10; covering a wide range that did not exceed the concentration that
                would affect the life of <I>C. elegans</I> or bacteria.
            
                Moreover, experiments with <I>E. coli</I> and <I>Pseudomonas putida</I> were carried out. These trials were performed in order to obtain the final media: half plate with
                non-altered NGM and half with the medium for PHA production by <I>Pseudomonas</I> and interference RNA production by <I>E. coli</I>.
            
                Not knowing which fatty acid could activate the transcription process better, we tested two possibilities: oleic acid (named in the tables as PHAol) and octanoic acid
                (PHAoc). The concentration for each fatty acid was tested in <I>E. coli</I> and we obtained best results with:
            <UL><LI>1.28 µl/ml of Octanoic acid.</LI><LI>2.58 µl/ml of Oleic acid.</LI></UL><P>
                You can see our results in<B> figures 3, 4, 5 and 6</B>. The first two counts were made after 3 hours and the next ones after 6 hours. We supposed that after that time, the worms were already capable of selecting their &quot;favourite&quot; side of the plate, get to the desired side and keep moving in the same part of the petri dish. We prepared control plates with the
                same composition but without MgSO<SUB>4</SUB> in PHA media.
            </P><H3>Final concentration choice</H3>
                Once the tests were performed and after observing the results obtained with the different selected fatty acids, we then observed a scattered result(there was
                probably a repellent effect at high concentrations in the medium with oleic but reversed in the octanoic) and in the end, we decided to use the following MgSO<SUB>4</SUB> concentrations
                for each medium:
            <UL><LI><B>If oleic acid is used</B><B>:</B>Best results with 4ml/L MgSO<SUB>4</SUB>.
            </LI><LI><B>If octanoic acid is used </B><B>:</B>Best results with 10 ml/L of MgSO<SUB>4</SUB>.
            </LI></UL>
                Octanoic was discarded as a transcriptional activator of the iRNA of clumping after other results for <I>E. coli</I> were obtained, so finally we selected the PHA medium with
                oleic acid only.
            
                The biggest problem while trying to have an effective attractant was knowing <B>how effective could an attractant,  in the presence of <I>E. coli</I>, be</B>. It was therefore necessary
                to test the tradeoff between the value of 4ml / L MgSO<SUB>4</SUB> and pair it with different concentrations of bacteria, high enough to feed the nematode
                but low enough to allow the attractant effect of MgSO<SUB>4</SUB>.
            
                To find this point of commitment we prepared experiments in which the concentration of 4ml/L of MgSO<SUB>4</SUB> was faced against different ODs from serial
                dilutions of a preculture of E. coli DH5a.
                <B>Table 7 </B>
                shows the results obtained. The count took place 3h after the passing of fresh nematodes.
            <H3>Best E. coli OD choice</H3>
                An OD of 1 (minimum concentration of cells / volume) gives an attractive effect even better than expected (subsequent experiments try to check if there is
                synergy between <I>E. coli </I>and attractive factors MgSO4).
                
                To improve the approximation we decided to repeat the experiment with MgSO<SUB>4</SUB> concentration and the chosen bacterial OD showing that the system
                worked correctly. The results can be seen in <B>figure 8.</B></DIV><DIV class="tab-pane fade" id="the_clumping"><H1 class="title2 clumping-color">The Clumping</H1><P>
    The main microorganism of this part is our engineered <I>E.coli</I> with a biobrick that is capable of changing the worm’s behaviour.
</P><P>
    Our<I><B>C. elegans</B></I> strain <B>(N2 strain</B>) has two feeding behaviours: <B>Social and solitary feeding</B>. Social feeding is known as clumping. You can watch it in this video: <A href="http://www.youtube.com/watch?v=jCNsmcVVUVY">http://www.youtube.com/watch?v=jCNsmcVVUVY</A></P><P>
    Clumping is known to be induced under some conditions <B>like temperature or starvation</B>, something that we had to consider during the entire project.
    However, it is also known how clumping is controlled from a genetic perspective and the main genes involved have already been described. Therefore, we thought about using
    this genetic approximation to control clumping under specific and controlled conditions. <I>(</I><A href="http://129.85.244.162/bard/pdf week_07_de_bono_bargmann_natural_variation_in_receptor_modifies_social_behavior_and_food_response_cell_1998.pdf
"><I>view “</I>Natural variation in a neuropeptide Y receptor
homolog modifies social behavior and food response in C. elegans.<B>”</B> Bono M, Bargmann CI</A>).
</P><H3>How do we induce clumping?</H3>
    Our initial thought was to interfere the principal gene involved in this route: <B>NPR-1</B>. It is known that mutations in <B>NPR1</B> convert a solitary
    strain into a social strain. Another option (the one we finally chose) was <B>FLP-21</B>, a gene that positively regulates <B>NPR-1</B>. We selected this one instead
because it is not involved in so many vital processes. <B>FLP-21</B> encodes a single FMRF amide-related neuropeptide that serves as a ligand for<B>Npr-1</B>, a G protein-coupled receptor that regulates social versus solitary feeding behavior in several <I>Caenorhabditis </I>species (see <B>fig. 1</B>).
<P>
    Therefore, this would be the final process: engineered<B><I>E.coli</I></B>that expresses <B>FLP-21</B>, whose <B>RNA would interfere </B>with the one codified by
    the worm because of the ingestion of bacteria. The formation of a complex of dsRNA would induce the elimination of <B>Flp-21 </B>transcripts by the RNA
    silencing pathway and the final result would be the induction of clumping (<B>see figure 2)</B>. We also had to take into account that the induction of clumping by
    this mechanism is almost immediate, in comparison with natural clumping which takes longer depending on the external factors.
</P><H3>Why do we induce clumping?</H3>

    The main objective of the synthetic symbiosis that we have designed is to detect hotspots of interest in irregular substrates and the induction of clumping was a good tool to do so. With this natural mechanism <B>worms are kept in the desired places</B>, then <B>bacteria transported by <I>C. elegans </I></B>(<I>Pseudomonas putida)</I><B>are concentrated </B>where their action is needed, in order to generate value-added products such as bioplastic PHA and the increase of worms in a specific location <B>improves the image-based detection </B>mechanism (<A href="https://2013.igem.org/Team:Valencia_Biocampus/Devices">go to the Devices section</A>)
<H3>Controlling the mechanism</H3>

    The social feeding behaviour needs to be controlled; it can be induced by the mentioned conditions of interest as a tool for detecting hotspots in irregular substrates. We decided that <B>fatty acids </B>were going to be the inductors of the promoter (<B>fadBp</B>), which regulated the expression in the FLP-21
    antisense sequence, in order to induce RNA interference (<B>Fig.3)</B>. We decided to use this regulation, because fatty acids are also used for the PHA production, our biotechnological process of interest.
<H3>Choosing the right fatty acid</H3><B><I>E. coli’s </I>biobrickpromoter is induced by fatty acids</B>, but these also induce the promoter of <I>Pseudomonas putida </I>in order to
    produce PHA. Some investigations show that the induction <B>of fadBp promoter </B>by <B>oleic acid </B>is the most efficient (<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC216235/?page=1"><I>view “</I>Regulation of
fatty acid degradation in Escherichia coli: analysis by operon fusion” Clark D. et al</A>) but this is a long chained compound and that could affect the production of bioplastic (PHA). We
    decided to test the growth of <I>E. coli </I>and the production of bioplastic by <B><I>P. putida</I></B>using PHA media with oleic acid and
    octanoic acid in order to get the best results in both activities.
<DIV style="overflow:auto;height:auto;clear:both"><DIV style="float:left;width:550px;"><P><B>Fig. 4. </B>
    Effect of chain length upon induction. Strain DC530 0 (fadA-lacZ) was grown in minimal medium with acetate as the carbon source and with various fatty
    acids present at 2 mM. The induction ratio is the β-galactosidase activity, relative to the control culture containing no fatty acid. Fatty acids are
    designated by “n:k”, where “n” is the number of carbon atoms and “k” is the number of double bonds.
</P><P><B>OLE</B>, Oleate; <B>CVC</B>, cis-vaccenate; <B>16:1</B>, palmitelaidate.
</P><P>
    Figure taken from: Clark, 1981.
</P></DIV></DIV><P>
    Both species of microorganisms where grown in PHA production media with different concentrations of oleic and octanoic acids, maintaining a global
    concentration of fatty acids of 8mM, because it was the one where we found the highest growth of both <I>E. coli </I>and <I>P. putida. </I><B>Table 2 </B>
    (below) summarizes the assays done.
</P><P>
    The results of this assay showed that <I>E. coli </I>growth was better on PHA production media + 8mM oleic acid and the <I>P. putida </I>production of
    bioplastic was also acceptable in these conditions. However, the highest production of bioplastic was on PHA production media + 8mM octanoic acid. So then, we
    made a <B>Colonization assay </B>and finally we determined it was better to keep using the PHA production media + 8mM oleic acid. Furthermore,
the ODs for plating both organisms were established as a consequence of this experiment. The “<B>Choosing the right fatty acid” </B>and the “<B>Colonization” </B>assays are explained at <A data-toggle="tab" href="#the_building"><B>Building the bioplastic </B>section.</A></P><H3>  Observing clumping induced by fatty acid rich media </H3>

We could observe how clumping was induced by the presence of <B>8mM of oleic acid </B>(2.58 µl/ml) which regulated the <B>FadBp promoter </B>of    <B><I>E. coli</I></B>and as a consequence the expression of the complementary sequence to the worms’ <B>FLP-21 </B>transcript, producing interference
    by RNA. In the plates, there was a high concentration of <I>E. coli </I>from the centrifugation of a preculture of 4ml <B>XL1-Blue</B>, 2 min at maximum rpm
    (2 times). The following images show the change in the behaviour of <I>C. elegans, </I> first eating alone and then in groups, just as we planned <B>(Fig. 5-7).</B><P>
    We confirmed that the clumping was induced because of the expression of the iRNA of engineered <I>E. coli. </I>Two populations of worms were fed with
    engineered <I>E. coli. </I>The control population ate <I>E. coli </I>that was grown in a culture medium without fatty acids. The experimental population
    was fed with <I>E. coli </I>that was grown in PHA media (culture media with fatty acids). The ingestion of <I>E. coli </I>expressing FLP-21 iRNA did result
    in a clumping behaviour in comparison with the control, where one could not observe this immediate social feeding response, due to the expression of Flp-21 (<B>Fig. 8</B>)
</P></DIV><DIV class="tab-pane fade" id="the_building"><H1 class="title2 building-color">The Building</H1><P>
    We decided to include a regulated production of a value-added product in the hotspots of interest (where C. elegans would be attracted to) in the mechanism design. Therefore, we took advantage of the natural capacity of Pseudomonas putida of producing bioplastic PHA (polyhydroxyalkanoates) by bacterial fermentation of
    sugar or lipids (fatty acids in this case). We found this material interesting due to its easy
    detection (complexes of PHA with Nile red are fluorescent) and its bright future in the field of biomaterials.
</P><H3>The mechanism</H3>

    A cluster of genes is responsible for the production of PHA, the <B>phaC operon </B>of <B><I>P. putida </I></B>which consists of 4 ORFs that are transcribed
    in the same direction: phaC1 and phaC2 genes, codify for two PHA synthases; phaZ gene, codifies for a depolymerase; and phaD gene, that codifies for a
    protein of the TetR family. In the opposite direction, two genes that codify for fasinas (phaF and phaI) and structural proteins are found <B>(Fig.1). </B><H3>The role of the operon in our project</H3>

Our 2013 project is a proof of concept of the benefits that an artificial synthetic symbiosis between bacteria and nematodes can offer. <I>Pseudomonas</I> play basically two roles: firstly, they have been engineered to “ride” on <I>C. elegans</I> by the formation of a biofilm, but not satisfied with that, we stimulated the production of PHA (a value-added product) under the promoter glnA. As mentioned before, PHA has several uses and applications such as:
<UL><LI>
    For short disposable packaging items (personal hygiene products, surgical clothes).
</LI><LI>
    In upholstery.
</LI><LI>
    In the photographic and printing industry
</LI><LI>
    For textile industry since PHA can be processed into fibers.
</LI><LI>
    For biofuel production, from PHA obtained from sewage sludge. This would combine two major advantages: the wastewater treatment and the generation of energy.
</LI></UL><H3>Choosing the right media</H3>

    On the plates, where we were going to have <I>P. putida </I>, <I>E. coli </I>and their promoters’, response towards fatty acids was different, so we had to create a media where <B><I>E. coli </I></B>could induce<B> clumping </B>and <B><I>P. putida </I></B>could <B>produce bioplastic</B><I>. </I>In <B><I>E. coli, </I></B>thebriobrick promoter for the expression of <B>FLP-21 iRNA </B>engineered in order to induce the social feeding
    behavior of<I> C. elegans </I><B>(Clumping) </B><B>is induced by fatty acids</B>, but not all the fatty acids are able to induce the same level of expression
    (<B>Fig. 2</B>); <B>oleic acid </B>produces the highest activation of the promoter.
<P>
Whereas the <B>standard medium </B>for the <B>production of PHA </B>by <B><I>Pseudomonas putida</I></B>(composition reflected in <B>Fig.3</B>) employs    <B>octanoic acid </B>(a short-chain fatty acid) as inducer. This fact made necessary to check if the modification of the fatty acid used, octanoic by
    oleic, would modify the bioplastic (PHA) production.
</P><P>
    Otherwise, the medium for PHA production is tuned for the development of <I>Pseudomonas putida</I>, but not for the <I>E.coli</I>, so we had to test if
    these bacteria were able to grow in it. Once we had checked this, we realized that the development of these bacteria did not occur in the PHA production
media. However, with the addition of acetate (carbon source), the removal of the iron salt (added to the media to avoid the synthesis of a <SPAN class="siderophore-tooltip">siderophore</SPAN> by <I>Pseudomonas sp.</I>) and (due to serendipity) with double the
    concentration of trace elements, we obtained the media for the enteric bacteria growth.
</P><H3>Choosing the right fatty acid</H3>
    Firstly, we tried to grow <I>E. coli </I>in the new PHA production media with different concentrations of octanoic acid. The same assays were carried out in
    parallel with <I>P. putida </I>in order to check which were the right conditions where both microorganisms could grow and where <I>P. putida </I>could produce enough bioplastic.
<P>
After overnight incubation, the cultures were washed using sterile PBS and ODs<SUB>600</SUB> were measured (<B>Fig.4.</B>). We also <B>checked the PHA production </B>by centrifuging the precultures, resuspending the pellet on PBS and adding Nile Red dissolved with DMSO. We could then observe the
    emission of fluorescence due to the interaction between the Nile Red and the PHA through an UV transilluminator <B>(Fig.5)</B>. According to results of the
    assays we chose to work with a <B>8mM</B> concentration of fatty acids (<B>Fig.4</B>).
</P><P>
    At this point we found that <I>E. coli </I>and <I>P. putida </I>could grow at 8mM of octanoic acid and that bioplastic (PHA) was also correctly produced. However,
    because <I>E. coli </I>grows better in <B>oleic acid </B>and we wanted to check if <B><I>P. putida </I></B>could produce PHA if its phaC operon was induced
    by it, we grew both bacteria in <B>different concentrations of octanoic and oleic acid but maintaining a constant concentration of fatty acids of 8mM</B>.
    The assays we carried out are explained in the table below (<B>Fig.6</B>).
</P><P>
    We saw that <I>E. coli </I>had grown better with the increase of oleic acid, getting at its maximum on experiment 9, even so, the quantity of pellet
    corresponding to 1 mL of preculture was scarce (and not visible at experiments 1 to 5). <I>E. coli </I>needed more time to grow in the PHA production
    media.
</P><P><I>P. Putida</I>
    had grown too, however we wanted to measure the production of bioplastic, we measured the ODs<SUB>600</SUB> from the precultures of <I>P. putida </I>by
    centrifugating 2 mL of them and then washing them on 1 mL of PBS and resuspending the pellet in 500 μ<B>l </B>of PBS<B>. </B> Then, dilutions on PBS were
    adjusted to the lowest OD<SUB>600</SUB><SUB>,</SUB>corresponding to the experiment number 9 (8mM oleic acid). The growth of <I>P. putida </I> was lower in PHA
    production media with a concentration of 8mM of oleic acid (<B>Fig. 7-8)</B>.
</P><P>
    The results showed that the growth of <B><I>E. coli </I></B>on 8mM of oleic acid was better than in any other condition, then <B>experiment 9 </B>
    conditions would be the best ones for the induction of <B>clumping. </B>However, even <B><I>P. putida </I></B>had produced bioplastic at all oleic acid
    concentrations, the highest production levels corresponded to <B>experiment 1 </B>conditions (that is, in the absence of oleic acid), just on 8mM of octanoic
    acid. Because of that, following assays were done under the conditions of <B>experiments 1 and 9</B> (<B>Fig.9</B>).
</P><P>
    After the optimization of the media, kinetics of the production of PHA and biomass generation were obtained in relation to time (<B>Fig.10</B>).
</P></DIV><DIV class="tab-pane fade" id="pre-field_test"><H1 class="title2 finalresult-color">Pre-field test</H1><H3>Final Test: PHA Production in an Heterogeneous Substrate (soil)</H3>
      Finally, we designed an assay in which all the activities presented till this moment were carried out by the synthetic symbiosis that we have been working
with. In this experiment we achieve the goal of our project<B>: the detection of hotspots of interest in irregular substrates and the production of bioplastic (PHA) in those places</B>.    <U>Our project works in a pre-field test!</U><P>
    A soil ground was prepared with an artificially distributed substrate-rich medium in discrete points (<B>Fig.1</B>). Once the medium was ready, we
    inoculated the soil with a <B><I>C. elegans-Pseudomonas </I></B>suspension and with transformed <B><I>E. coli </I></B>expressing the iRNA of interest. As a
    control, we took a picture of the soil ground the first day the microorganisms were added by exposing it to UV light (<B>Fig.2</B>). After a day we added
    Nile red to reveal the PHA production and to see if the experiment had finally worked (<B>Fig. 3</B>). It is possible to see bioplastic nodules which are
    red when combined with Nile red.
</P><P>
    In addition, we took an electron microscopy image of one of the discrete points of medium (<B>Fig. 4</B>) to study which organisms were actually near the
hotspots. More electron microscopy images were taken to prove that <I>C. elegans</I> dragged <I>Pseudomonas</I> to the points of interest (<B>Fig. 4)</B>.
</P><P>
We were finally able to conclude that our synthetic consortium worked: Worms managed to reach the hotspots of interest, the worms dragged    <I>Pseudomonas </I>with them and <I>Pseudomonas</I> finally made bioplastic, detected by the added Nile Red.
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Valencia_Biocampus/ResultsUpdated">http://2013.igem.org/Team:Valencia_Biocampus/ResultsUpdated</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Valencia_Biocampus/ResultsUpdated" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Valencia_Biocampus/ResultsUpdated" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ResultsUpdated&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Valencia_Biocampus/ResultsUpdated&amp;oldid=352054" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>