<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Uppsala_Project_Sensing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Uppsala/Project_Sensing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Uppsala/Project_Sensing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Uppsala/Project_Sensing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Uppsala/Project_Sensing&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Uppsala/Project_Sensing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Uppsala/Project Sensing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="master"><DIV id="header"><DIV id="imgwrapNH"><DIV id="ste"><SPAN>Stephanie Herman</SPAN></DIV><DIV id="ter"><SPAN>Teresa Reinli</SPAN></DIV><DIV id="joa"><SPAN>Joakim Hellner</SPAN></DIV><DIV id="ale"><SPAN>Alexander Virtanen</SPAN></DIV><DIV id="jen"><SPAN>Jennifer Rosenius</SPAN></DIV><DIV id="marc"><SPAN>Marcus Hong</SPAN></DIV><DIV id="mir"><SPAN>Miranda Stiernborg</SPAN></DIV><DIV id="tim" onclick="darkLord()"><SPAN>Tim Hagelby Edström</SPAN></DIV><DIV id="vik"><SPAN>Viktor Blomkvist</SPAN></DIV><DIV id="meg"><SPAN>Megha Biradar</SPAN></DIV><DIV id="nik"><SPAN>Niklas Handin</SPAN></DIV><DIV id="jon"><SPAN>Jonas Mattisson</SPAN></DIV><DIV id="ari"><SPAN>Arina Gromov</SPAN></DIV><DIV id="nil"><SPAN>Nils Anlind</SPAN></DIV><DIV id="eri"><SPAN>Eric Sandström</SPAN></DIV><DIV id="gun"><SPAN>Gunta Celma</SPAN></DIV><DIV id="oli"><SPAN>Oliver Possnert</SPAN></DIV><DIV id="mar"><SPAN>Martin Friberg</SPAN></DIV><DIV id="kir"><SPAN>Kira Karlsson</SPAN></DIV><DIV id="chr"><SPAN>Christoffer Andersson</SPAN></DIV><DIV id="lau"><SPAN>Laura Pacoste</SPAN></DIV><DIV id="and"><SPAN>Andries Willem Boers</SPAN></DIV></DIV></DIV><DIV id="menu_container"><DIV id="menu"><UL><LI><A href="https://2014.igem.org/Team:Uppsala/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Project_Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Project_Targeting">Targeting</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Project_Killing">Killing</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Project_Adhesion">Adhesion</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Project_Notebook">Notebook</A></LI></UL><LI><A href="https://2014.igem.org/Team:Uppsala/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Modeling_CellCellInteraction">Cell-Cell Interaction</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Modeling_PopulationLevel">Population Level</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/InterlabStudy">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Parts">Parts</A></LI></DIV><DIV id="menu_right"><UL><LI><A href="https://2014.igem.org/Team:Uppsala/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Team_SensingGroup">Sensing Group</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Team_TargetingGroup">Targeting Group</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Team_KillingGroup">Killing Group</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Team_AdhesionGroup">Adhesion Group</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Team_ModelingGroup">Modeling Group</A></LI></UL><LI><A href="https://2014.igem.org/Team:Uppsala/PolicyPractices">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/PolicyPractices_MicrobialDesigns">Microbial Designs</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/PolicyPractices_Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/PolicyPractices_BusinessPlan">Business Plan</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Attributions">Attributions</A></LI></DIV></DIV><DIV id="main_content"><DIV id="title"><H1 id="currentH"> Home </H1></DIV><DIV><DIV id="tracking_system"><P id="tra">Failed to load tracking. JS is probably not enabled</P></DIV><DIV id="left_overview"><DIV class="background"><A href="#ref_point" style="display:block; width:100%; height: 100%; text-decoration:none !important;"><H2 class="overview">Background</H2><P class="box_text">In our project we developed a system to fight pathogens in the gut, which is not a unique idea. Except we wanted our systems key attribute to be specificity. To do so, we needed to find a unique quorum sensing language that we could hijack and utilize. We found the Yen system, from <I>Yersinia enterocolitica</I>. A homologous sytem to the well known Lux system, from <I>Vibrio fischeri</I>.</P></A></DIV></DIV><DIV id="right_overview"><DIV id="system_design"><A href="#ref_point3" style="display:block; width:100%; height: 100%; text-decoration:none !important;"><H2 class="overview">System design</H2><P class="box_text">From the Yen system we stole the activator YenR and the recognition region, the yenbox, fused with a wild type promoter. YenR was synthesized with the RBS B0034 and codon optimized for <I>E. coli</I>. A customized version of the yenbox-promoter was also created.</P></A></DIV><DIV id="result"><A href="#ref_point2" style="display:block; width:100%; height: 100%; text-decoration:none !important;"><H2 class="overview">Result</H2><P class="box_text">We managed to get a five-fold induction relative to the base level of expression. The cells that were used during the measurements were double transformed with the yenbox-promoter complex with GFP and a plasmid producing the activator YenR.</P></A></DIV></DIV><DIV id="parts"><A href="#ref_point4"><H2 class="overview">Parts</H2></A><A href="http://parts.igem.org/Part:BBa_K1381000">BBa_K1381000</A><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381001">BBa_K1381001</A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381002">BBa_K1381002</A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381004">BBa_K1381004</A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381005">BBa_K1381005</A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381006">BBa_K1381006</A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381007">BBa_K1381007</A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381008">BBa_K1381008</A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381009">BBa_K1381009</A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381010">BBa_K1381010</A></LI></DIV><DIV id="p_containerR"><TABLE style="border-radius: 25px;"><TBODY><TR><TD><P>The assembly plan of the Sensing system.</P></TD></TR></TBODY></TABLE></DIV><H2>Background</H2><H3> Communication using quorum sensing</H3><P>Quorum sensing (QS) is the communication tool of bacteria. It is usually done through secretion and detection of small signal molecules or proteins. Depending on the bacterial density, these molecules can trigger a change of behaviour, i.e. invoke a function only when there are enough bacteria present in that one area. Quorum sensing can control several things, from biofilm formation to bioluminescence. The most commonly known and most explored QS system is the Lux system, which was originally isolated from <I>Vibrio fischeri</I>, where it controls the production of luciferase, an enzyme that aids in the production of a yellow light.</P><H3>Using the pathogens own quorum sensing system to detect and track its source</H3><P>When we decided that we wanted to fight pathogens it became obvious that we needed a way to detect and track them. After doing some research, we decided that this could be  done by hijacking the pathogen’s own quorum sensing. Naturally, the first thought that came to our minds was to use the Lux system from <I>V. fischeri</I>. However, this had been done before in 2008 by Heidelberg and it also did not provide the specificity that we wanted for our system. We decided to dig deeper and see if we could find a more unique quorum sensing system. Among some potential candidates was <I>Yersinia enterocolitica</I>, which uses a quorum sensing system called the Yen system.</P><H3>The Yen system </H3><P>As it turns out <I>Y. enterocolitica</I> has a homologous quorum sensing system to the famous Lux system, the Yen system. From this system we chose to steal two parts. A recognition region, the yenbox, fused together with a promoter and an activator, YenR, that can recognise and interact with the yenbox. When YenR binds to the yenbox it induces the expression level of the promoter fused to the yenbox. Later, in the presence of <I>Y. enterocolitica</I>, its signaling molecules, 3-oxo-hexanoyl homoserine lactone (OHHL), will start flowing into our system, interacting with YenR. When binding occurs between OHHL and YenR, YenR will lose its active shape and thereby its ability to interact with the yenbox. The induction will then be lost and the expression level will return to its base level. <SUP><A href="#reference1">[1]</A></SUP>By BioBricking and characterizing this system, we wish to make it possible to get a <I>Y. enterocolitica</I> density triggered response. Our goal with this system is to control our two other systems, the Targeting and the Killing system.</P><H2>System Design</H2><P>While stealing the yenbox together with the wild type promoter fused with it, we had no idea about the strength of the wild type promoter or if it would even work at all. Because of this, we created an alternative version where we replaced the wild type promoter with a standardised one (J23113). In the wild type version there is an overlap between the wild type promoter and the yenbox. Hence, we mimicked the same while creating our customized version where we had an overlap between the promoter and the yenbox. Since the strength of the promoter would correspond to the leakage in our system, we wanted to have a weak promoter to minimize the leakage. We chose the constitutive promoter J23113 (<A href="http://parts.igem.org/Part:BBa_J23113">BBa_J23113</A>) from the Anderson library. Unfortunately, the promoter J23113 did not begin with the same two bases as the end of the yenbox. We were left with the option of either changing the two bases in the yenbox sequence or changing the two bases in the sequence of the promoter J23113. In the article by Ching-Sung Tsai and Stephen C. Winanas <SUP><A href="#reference1">[1]</A></SUP> they discovered that the binding between the activator YenR and the recognition region of the yenbox is not dependent on the entire sequence of the yenbox. Depending on which part of the yenbox that is changed or replaced, YenR binds to the yenbox with different strengths. However, it still interacts with the yenbox and induces the strength of the promoter. Based on this fact, together with the knowledge that the Anderson promoters are very sequence dependent, we chose to change two bases in the sequence of the yenbox.We also stole the coding sequence <SUP><A href="#reference2">[2]</A></SUP> of the activator YenR, from <I>Y. enterocolitica</I>, which we codon optimized for <I>E. coli</I> using a IDTs web tool <SUP><A href="#reference3">[3]</A></SUP> and synthesized it together with the RBS B0034 (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>). Since we always want production of the activator YenR, it was coupled to three different constitutive promoters from the Anderson promoter library with three different strengths.</P><H2>Result</H2><P>For characterisation we created the constructs yenbox_WT-B0034-GFP (<A href="http://parts.igem.org/Part:BBa_K1381008">BBa_K1381008</A>) and yenbox_J23113-B0034-GFP (<A href="http://parts.igem.org/Part:BBa_K1381009">BBa_K1381009</A>), where the yenbox fused with a promoter was coupled to the green fluorescent protein (GFP). These constructs were then cloned into the backbones pSB1C3 and pSB3C17 and transformed into competent <I>E. coli</I> cells already containing one of the YenR constructs J23110-B0034-YenR (<A href="http://parts.igem.org/Part:BBa_K1381005">BBa_K1381005</A>), J23102-B0034-YenR (<A href="http://parts.igem.org/Part:BBa_K1381006">BBa_K1381006</A>) or J23101-B0034-YenR (<A href="http://parts.igem.org/Part:BBa_K1381007">BBa_K1381007</A>) on the backbone pSB1K3. The double transformed cells were then streaked on plates containing both the antibiotic Kanamycin and Chloramphenicol and left it overnight to grow. Cells containing only the constructs yenbox_promoter-B0032-GFP were also streaked and left to grow.The following day, overnight cultures were prepared and left for 16 h to grow into stationary phase. After that, 10 µL of the overnight culture was put into 500 µL of PBS solution and left for one hour for stabilization. The green fluorescence production was then measured using a flow cytometer. The results of the test is shown below.



Graph 1 above, shows the induction relative to the base level of expression. In graph 1.A. we can observe the induction but cannot predict to what extent. This is because these cells are transformed with two high copy plasmids containing the same ori (origin of replication). The consequence will be that the cells will confuse the two plasmids with each other and have no control of in what amount the two respective plasmids are present. It will only ensure that it is 100-300 plasmids present in total, regardless of which plasmid it is. This is why we also cloned the yenbox containing constructs into a low copy plasmid with a different ori than the high copy ones.In graph 1.B., it can be seen that the amount of YenR that is produced is correlated, as expected, to the rate of the induction. When coupled to the strongest of the three promoters, J23102 (<A href="http://parts.igem.org/Part:BBa_J23102">BBa_J23102</A>), the production is increased up to five folds.In graph 1.C., we did not observe any induction  This implies that our customized version of the yenbox fused with the promoter J23113 did not work as intended. It seems like the activator YenR could not recognise nor interact with the yenbox due to the changes made in the last two bases of the yenbox. It should be tested to redesign this part so that the yenbox is not fused to the promoter but just simply put beside it without any overlap. If the induction is not extremely distance dependent, that might work better. Otherwise, you could try to find other promoters, were the first two bases match with the two last bases of the yenbox, so that we don’t have to modify the yenbox.

These kind of constructs were also constructed with a blue fluorescence protein (BFP) that were also transformed into cells already containing one of the YenR constructs J23110-B0034-YenR (<A href="http://parts.igem.org/Part:BBa_K1381005">BBa_K1381005</A>), J23102-B0034-YenR (<A href="http://parts.igem.org/Part:BBa_K1381006">BBa_K1381006</A>) or J23101-B0034-YenR (<A href="http://parts.igem.org/Part:BBa_K1381007">BBa_K1381007</A>) on the backbone pSB1K3. They were then plated and restreaked together with a non-fluorescence containing reference and a base level reference without any YenR. These plates were used to see if the induction could be seen in UV light, with the naked eye. Unfortunately, as you can see in fig. 1 below, this was not the case. The color could be seen, but there was barely any difference in the intensity of the different cells.

It should also be mentioned that we did an attempt to create bigger constructs containing the characterisation constructs with the yenbox coupled to GFP and the YenR constructs (yenbox_promoter-B0034-GFP-terminator-promoter-B0034-YenR). But for some reason these assemblies did not work. After some troubleshooting we came to the conclusion that it was too stressful for our bacteria to produce both BFP and YenR on the same high copy plasmid or that these constructs were not stable in <I>E. coli</I> and got rejected or mutated by the cell.</P><H2>Parts</H2><TABLE id="partsT" style="width:100%"><TBODY><TR><TH>Fav.</TH><TH>BioBrick code</TH><TH>Type</TH><TH>Construct</TH><TH>Description</TH><TH>Designers</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381000">BBa_K1381000</A></TD><TD>Regulatory</TD><TD>yenbox_WT</TD><TD>The Yen systems recognition region fused with a wildtype promoter</TD><TD>Sensing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381001">BBa_K1381001</A></TD><TD>Regulatory</TD><TD>yenbox_J23113</TD><TD>The Yen systems recognition region fused with the promoter J23113</TD><TD>Sensing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381002">BBa_K1381002</A></TD><TD>Regulatory</TD><TD>yenbox_J23101</TD><TD>The Yen systems recognition region fused with the promoter J23101</TD><TD>Sensing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381004">BBa_K1381004</A></TD><TD>Coding</TD><TD>B0034-YenR</TD><TD>YenR, the activator in the Yen system</TD><TD>Sensing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381005">BBa_K1381005</A></TD><TD>Regulatory</TD><TD>J23110-B0034-YenR</TD><TD>The activator YenR coupled to the promoter J23110</TD><TD>Sensing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381006">BBa_K1381006</A></TD><TD>Regulatory</TD><TD>J23102-B0034-YenR</TD><TD>The activator YenR coupled to the promoter J23102</TD><TD>Sensing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381007">BBa_K1381007</A></TD><TD>Regulatory</TD><TD>J23101-B0034-YenR</TD><TD>The activator YenR coupled to the promoter J23101</TD><TD>Sensing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381008">BBa_K1381008</A></TD><TD>Measurement</TD><TD>yenbox_WT-B0032-GFP</TD><TD>A characterisation construct for the yenbox with the wildtype promoter</TD><TD>Sensing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381009">BBa_K1381009</A></TD><TD>Measurement</TD><TD>yenbox_J23113-B0032-GFP</TD><TD>A characterisation construct for the yenbox with the promoter J23113</TD><TD>Sensing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381010">BBa_K1381010</A></TD><TD>Measurement</TD><TD>yenbox_J23101-B0032-GFP</TD><TD>A characterisation construct for the yenbox with the promoter J23101</TD><TD>Sensing Group</TD></TR></TBODY></TABLE><UL class="reference"><LI><A id="reference1">[1]</A> Ching-Sung Tsai and Stephen C. Winanas, The quorum-hindered transcription factor YenR of <I>Yersinia enterocolitica</I> inhibits pheromone production and promotes motility via a small non-coding RNA, 2011, Molecular Microbiology 80[2], 556-571</LI><LI><A id="reference2">[2]</A>Yersinia enterocolitica subsp. enterocolitica 8081 chromosome, complet - Nucleotide - NCBI [WWW Document], n.d. URL http://www.ncbi.nlm.nih.gov/nuccore/123440403/?from=1798138&amp;to=1798872 (accessed 10.17.14).</LI><LI><A id="reference3">[3]</A>Codon Optimization [WWW Document], n.d. URL http://eu.idtdna.com/CodonOpt (accessed 10.17.14).</LI></UL></DIV></DIV><DIV class="back-to-top"><A>Back to Top</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Uppsala/Project_Sensing">http://2014.igem.org/Team:Uppsala/Project_Sensing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Uppsala/Project_Sensing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Uppsala/Project_Sensing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Uppsala/Project_Sensing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Uppsala/Project_Sensing&amp;oldid=345711" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>