<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Uppsala_Project_Killing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Uppsala/Project_Killing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Uppsala/Project_Killing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Uppsala/Project_Killing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Uppsala/Project_Killing&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Uppsala/Project_Killing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Uppsala/Project Killing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="master"><DIV id="header"><DIV id="imgwrapNH"><DIV id="ste"><SPAN>Stephanie Herman</SPAN></DIV><DIV id="ter"><SPAN>Teresa Reinli</SPAN></DIV><DIV id="joa"><SPAN>Joakim Hellner</SPAN></DIV><DIV id="ale"><SPAN>Alexander Virtanen</SPAN></DIV><DIV id="jen"><SPAN>Jennifer Rosenius</SPAN></DIV><DIV id="marc"><SPAN>Marcus Hong</SPAN></DIV><DIV id="mir"><SPAN>Miranda Stiernborg</SPAN></DIV><DIV id="tim" onclick="darkLord()"><SPAN>Tim Hagelby Edström</SPAN></DIV><DIV id="vik"><SPAN>Viktor Blomkvist</SPAN></DIV><DIV id="meg"><SPAN>Megha Biradar</SPAN></DIV><DIV id="nik"><SPAN>Niklas Handin</SPAN></DIV><DIV id="jon"><SPAN>Jonas Mattisson</SPAN></DIV><DIV id="ari"><SPAN>Arina Gromov</SPAN></DIV><DIV id="nil"><SPAN>Nils Anlind</SPAN></DIV><DIV id="eri"><SPAN>Eric Sandström</SPAN></DIV><DIV id="gun"><SPAN>Gunta Celma</SPAN></DIV><DIV id="oli"><SPAN>Oliver Possnert</SPAN></DIV><DIV id="mar"><SPAN>Martin Friberg</SPAN></DIV><DIV id="kir"><SPAN>Kira Karlsson</SPAN></DIV><DIV id="chr"><SPAN>Christoffer Andersson</SPAN></DIV><DIV id="lau"><SPAN>Laura Pacoste</SPAN></DIV><DIV id="and"><SPAN>Andries Willem Boers</SPAN></DIV></DIV></DIV><DIV id="menu_container"><DIV id="menu"><UL><LI><A href="https://2014.igem.org/Team:Uppsala/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Project_Sensing">Sensing</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Project_Targeting">Targeting</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Project_Killing">Killing</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Project_Adhesion">Adhesion</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Project_Notebook">Notebook</A></LI></UL><LI><A href="https://2014.igem.org/Team:Uppsala/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Modeling_CellCellInteraction">Cell-Cell Interaction</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Modeling_PopulationLevel">Population Level</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/InterlabStudy">Interlab Study</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Parts">Parts</A></LI></DIV><DIV id="menu_right"><UL><LI><A href="https://2014.igem.org/Team:Uppsala/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Team_SensingGroup">Sensing Group</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Team_TargetingGroup">Targeting Group</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Team_KillingGroup">Killing Group</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Team_AdhesionGroup">Adhesion Group</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Team_ModelingGroup">Modeling Group</A></LI></UL><LI><A href="https://2014.igem.org/Team:Uppsala/PolicyPractices">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/PolicyPractices_MicrobialDesigns">Microbial Designs</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/PolicyPractices_Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/PolicyPractices_BusinessPlan">Business Plan</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Uppsala/Attributions">Attributions</A></LI></DIV></DIV><DIV id="main_content"><DIV id="title"><H1 id="currentH"> Home </H1></DIV><DIV><DIV id="tracking_system"><P id="tra">Failed to load tracking. JS is probably not enabled</P></DIV><DIV id="left_overview"><DIV class="background"><A href="#ref_point" style="display:block; width:100%; height: 100%; text-decoration:none !important;"><H2 class="overview">Background</H2><P class="box_text">We wanted to use a bacteriocin to design a new form of toxic that, unlike general antibiotics, only kills specific pathogenes, thereby leaving our gut flora intact. The goal was to express the bacteriocin colicin Fy in <I>E. coli</I>. Colicin Fy is specific against <I>Yersinia enterocolitica</I>, a gut pathogen. We also wanted to include an on and off system with a silencing sRNA together with the Sensing system. </P></A></DIV></DIV><DIV id="right_overview"><DIV id="system_design"><A href="#ref_point3" style="display:block; width:100%; height: 100%; text-decoration:none !important;"><H2 class="overview">System design</H2><P class="box_text">Two different constructs were needed to get the Killing system working. First the secretion tag, USP45, with the bacteriocin, colicin Fy, coupled to it. Secondly, we need a silencing sRNA system for inhibition of colicin Fy production when not in proximity to <I>Y. enterocolitica</I>.</P></A></DIV><DIV id="result"><A href="#ref_point2" style="display:block; width:100%; height: 100%; text-decoration:none !important;"><H2 class="overview">Result</H2><P class="box_text">We managed to prove the existance of the bacteriocin, colicin Fy on an SDS-page. By adding lysate that contains the colicin Fy to living <I>Y. enterocolitica</I> cultures we could show that it works for killing the actual pathogen as well.
</P></A></DIV></DIV><DIV id="parts"><A href="#ref_point4"><H2 class="overview">Parts</H2></A><A href="http://parts.igem.org/Part:BBa_K1381014">BBa_K1381014 </A><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381015">BBa_K1381015 </A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381016">BBa_K1381016 </A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381017">BBa_K1381017 </A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381018">BBa_K1381018 </A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381019">BBa_K1381019 </A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381020">BBa_K1381020 </A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381021">BBa_K1381021 </A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381022">BBa_K1381022 </A></LI><LI class="mparts"><A href="http://parts.igem.org/Part:BBa_K1381023">BBa_K1381023 </A></LI></DIV><DIV id="p_containerR"><TABLE style="border-radius: 25px;"><TBODY><TR><TD><P>The assembly plan of the Killing system.</P></TD></TR></TBODY></TABLE></DIV><H2>Background</H2><H3>Bacteriocin</H3><P>With the growing problem of antibiotic resistance, we decided to use bacteriocins as our antimicrobial agent. A bacteriocin is a peptide that is produced naturally by certain bacteria and targets its close relatives. Unlike general antibiotics, the targets of bacteriocins are very specific. The bacteriocin, colicin Fy is produced by <I>Y. frederiksenii</I> and it should mostly target other yersinia species.  Bacteriocins will either attack the cell membrane or a mechanism inside  the cell, such as gene expression or protein production.<SUP><A href="#reference1">[1]</A></SUP> One of colicins Fy main targets is <I>Y. enterocolitica</I>. It kills <I>Y. enterocolitica</I> by creating pores in its cell membrane. <I>Y. enterocolitica</I> is also one among the common pathogens that infects the gut and causes some serious symptoms<SUP><A href="#reference2">[2]</A></SUP>. These were some of the reasons to choose colicin Fy as the bacteriocin to express.
</P><H3>Spot42 sRNA</H3><P>Our goal was to design a seek and kill system. This implies that we only want to express the colicin Fy when our bacteria is close to the target. We chose this strategy because the cell would stress out to express colicin Fy all the time, even when not in close proximity to Yersinia. In order to make our bacteria express the colicin Fy only when we want it to, we have designed an sRNA system. We have based our design on the system that Team Uppsala iGEM 2012 implemented.
. The sRNA, spot42, consists of an antisense region that recognizes a specific RNA sequence to interact with as well as another region that recruits the protein Hfq. The Hfq protein blocks and prevents mRNA from binding to the ribosome. We have redesigned the antisense region so it will specifically recognize our USP45. This will make it bind to our USP45 and recruit the Hfq protein to stick to that region.<SUP><A href="#reference3">[3]</A></SUP> USP45 is the secretion tag that we have coupled to the colicin Fy. The sRNA will stick to Hfq and bind to the mRNA of USP45-Colicin Fy gene and that will block the translation of colicin Fy.
When our bacteria is in proximity of <I>Y. entercolitica</I> we want it to express the colicin Fy. Therefore, the Sensing group have been working on a Yen system which will inactivate the promoter that regulates the spot42 when it is close to <I>Y. enterocolitica</I>.
</P><P><I>Figure 1. The spot42 system</I></P><H2>System design</H2><P>The aim for the killing group was to design two different constructs - one with a secretion tag and the bacteriocin, colicin Fy, and the other with an sRNA system for inhibition of the toxin when not in proximity of <I>Y. enterocolitica</I>. However, at first we wanted to insert our final construct into <I>Lactobacillus</I> via a shuttle vector. Since <I>Lactobacillus</I> is a probiotic we could use our system in a real scenario if one gets a <I>Y. enterocolitica</I> infection in the gut. The final construct would consist of J23106-B0034-USP45-CFY and a construct for the sRNA spot42. The sRNA spot42 already includes a promoter. 
Two different secretion tags were used. PelB for <I>E. coli</I> and USP45 for both <I>Lactobacillus</I> and <I>E. coli</I>. Two Anderson promoters were chosen, J23106 (strong) and J23116 (weak) for the colicin Fy because we wanted to find a balance, where <I>E. coli</I> would not die from stress (due to too high expression) or its self-produced toxin. In order to characterize our colicin Fy we wanted to purify our sample so we could see the protein on an SDS-page. To do this, a His-Tag was required for an IMAC column purification. Therefore we also built the constructs with a 6x His-Tag.

We designed the colicin Fy according to the Freiburg assembly standard, biobrick 25. This would allow us to fuse our protein with the secretion tag without having to consider the stop codon from the scar of SpeI and XbaI from the biobrick 10 standard.
</P><H3>Silencing RNA system design</H3><P>The silencing RNA system or sRNA system was also one of the primary aims for the killing group. We designed and modified the sRNA system from Team Uppsala iGEM 2012, called spot42. The spot42 was redesigned to sense our USP45 sequence and bind to it. We call our modified part, Spot42_USP45. The Spot42_USP45 system consists of an antisense region that will recognize our USP45 and it also has the part that recruits the Hfq protien that will block and inhibit the mRNA translation. The Hfq protein will stick to the antisense region; this will block the RBS from binding to the mRNA and therefore inhibits the translation. We have designed three different antisense regions for our Spot42_USP45. They are all based on USP45 but with different lengths, 10, 15 and 20bp. This is to ensure that it is specific to only USP45. Otherwise it might bind to another region and inhibit a process that could be vital for the bacteria. We chose to have USP45 as our primary secretion tag since our initial goal was to build our system in <I>Lactobacillus</I>. To help us with that, we used knowledge from the Uppsala iGEM 2013 team, who worked with <I>Lactobacillus</I> and USP45 among other things.   
</P><H2>Result</H2><H3>Characterization</H3><P>Our initial plan was to have an secretion tag together with our colicin Fy allowing it to secret from the living cells. Unfortunately we did not manage to get the assembly of any of the constructs to fully work. Our enzymes for the Freiburg standard did not seem to work so we had to use SpeI and XbaI causing a stop codon that we did not managed to mutagenise away. 

We had however succeeded in creating construct without secretion tag. The expression of colicin Fy with x6 His-tag was characterized via an purification of the protein with IMAC followed by an SDS-page to confirm lenght, <A href="https://2014.igem.org/Team:Uppsala/Project_Notebook">[Link to protocol]</A>. 11 tubes of eluent from the IMAC was saved. The protein concentration in each tube was analyzed with a nanodrop. Tube 4 and 6 had higher concentrations than the other tubes and were run on SDS-page.

The results from the characterization experiments can be seen in fig. 2 and 3. According to calculations based on the nucleotide sequence, the mass of colicin Fy is about 49,6 kDa. The protein ladder we used was Thermo Scientifics PageRuler Unstained Protein Ladder #26614 (fig. 2). The thickest band is 50 kDa so our bands should be at about the same height.
</P><TABLE><TBODY><TR><TD>Figure 2: Thermo Scientifics PageRuler Unstained Protein Ladder #26614</TD></TR><TR><TD><I style="margin-left: 20px;">Figure 3: The gel contains the following from the left to the right
<UL style="margin-left: 20px;"><LI>1. Protein ladder</LI><LI>2. Eluent 1 for lysate produced by colicin Fy producing bacteria</LI><LI>3. Eluent 3 for lysate produced by colicin Fy producing bacteria</LI><LI>4. Eluent 2 for lysate produced by colicin Fy producing bacteria</LI><LI>5. Final eluent test tube 4 for lysate produced by colicin Fy producing bacteria</LI><LI>6. Final eluent test tube 6 for lysate produced by colicin Fy producing bacteria</LI><LI>7. Eluent 2 for lysate produced by the negative control</LI><LI>8. Eluent 3 for lysate produced by the negative control</LI><LI>9. Final eluent test tube 4 for lysate produced by the negative control</LI><LI>10. Final eluent test tube 6 for lysate produced by the negative control</LI></UL></I></TD></TR></TBODY></TABLE><P>Several IMAC and SDS-pages were run, see (Read more). In all cases eluent 1 was found to be empty. This is not surprising because the eluent volume is so small that the proteins will not come out until eluent 2. Because of this we decided to skip eluent 1 for the negative control in the presented SDS in fig. 3. Eluent 2 consists of several proteins; every protein that does not bind to the IMAC should come out in this step. Eluent 3 seems to be where our colicin Fy is eluted, since it shows a band at the correct length, 50kDa. The negative control did not show any band in 50kDa and expression could therefore be confirmed. Since the colicin was eluted at elution 3 no band can be seen at elution 4.
</P><DIV class="panel"><H2>Read more/Hide</H2><DIV class="panelcontent"><P>Appendix 1 displays an earlier SDS-page that was run with more constructs. Since there were more constructs we could not try all elutes, since they did not fit on one gel. The existence of colicin Fy in the construct with J23106 could be seen. Due to this we continued to try further characterization with this construct (SDS-page gel before).
</P><TABLE><TBODY><TR><TD><I>Appendix 1: The gel contains the following from the right to the left:
<UL><LI>1. Protein ladder</LI><LI>2. Final eluent 1 with CP 11 promoter</LI><LI>3. Final eluent 2 with CP 11 promoter</LI><LI>4. Final eluent 1 with J23106 promoter</LI><LI>5. Final eluent 2 with J23106 promoter</LI><LI>6. Final eluent 1 with J23113 promoter</LI><LI>7. Final eluent 2 with J23113 promoter</LI><LI>8. Final eluent 1 with DH5-alpha as negative control</LI><LI>9. Final eluent 2 with DH5-alpha as negative control</LI><LI>10. Stage 3 eluent with promoter J23106</LI></UL></I></TD></TR></TBODY></TABLE><P>A in the appendix 1 is pointing towards the 50 kDa band on the ladder. B is pointing towards the band that can be seen and that likely is the colicin Fy while C is another eluent with the same promoter also showing the colicin Fy existence. This shows that the promotor J23106 can be used to produce the colicin, while CP 11 and J23113 can not. 
We did manage to get some indication that the CP11 promoter might have worked but the band was not strong enough to guarantee that it was actually there. There can be quite a few reasons to why the band is not stronger than it is. There is a possibility that the protein was eluted out of the system too early which also seems to be the case to some extent. Appendix 1 lane 3 shows a band in the right position. This was not observed for the other tests but since that eluent is in a falcon tube containing about 40 ml of liquid it was probably very diluted and weak. It could also be some leakage from the well next to it.</P></DIV></DIV><P>
The IMAC and SDS-page managed to provide a strong indication that we did produce the colicin Fy. There existed bands at the right places and the negative control did not have any bands at all. All experiments were conducted in the same manner and if it was another protein that by chance bonds to the IMAC gel it should have been in the negative control as well. 

Most bacteriocins are not fully studied so there might always be unknown problems with them. One possibility is that they somehow hurt the bacteria in which they are produced. Since the colicin described here is supposed to only react to a specific receptor that some <I>Yersinia</I> species have on the cell surface the probability for the colicin to hurt the producing bacteria is very low. A bigger problem could be that many bacteriocins are short lived in certain conditions and might denature or break before they are characterized.</P><H3>Inhibition test</H3><P>To be able to prove that our system is working we needed to test it on the actual <I>Y. enterocolitica</I>. Since <I>Y. enterocolitica</I> is a class two bacteria we had to do it in another lab. We managed to get in contact with Livsmedelsverket (the Swedish authority of food safety) and they allowed us to test our colicin Fy in one of their labs on one of their pathogenic strains of <I>Y. enterocolitica</I>.

To examine the inhibition of colicin Fy an overnight culture of bacteriocin producing bacteria in 200ml LB with no antibiotic was prepared. As negative control, DH5-alpha was grown under the same conditions. No antibiotic was added to ensure that the antibiotics would not affect the results. The cells were then lysed and the supernatant was collected according to <A href="https://2014.igem.org/Team:Uppsala/Project_Notebook">protocol</A>. The supernatant was added to a final concentration of 1%(v/v) in LB with 10^4 CFU/ml <I>Y. enterocolitica</I> added. The culture was put to grow in 37d C with 100rpm shaking. OD measurements were taken every hour,  but due to <I>Y. enterocolitica</I> ability to aggregate the results were inconclusive. Instead 100microL was plated on BHI agar plates after 0h, 2h, 4h and 24h. Immediately growth difference could be spotted and after 4 hours the results were very clear, fig. 4.  More figures of the plating can be found in the appendix.</P><TABLE><TBODY><TR><TD><I>Figure 4: Tests to compare colicin Fy lysate on the right plate compared to the N/C on the left. Both plates are diluted to the same concentration and grown for the same amount of time. For more figures, see &quot;Read more&quot;</I></TD></TR></TBODY></TABLE><DIV class="panel"><H2>Read more/Hide</H2><DIV class="panelcontent"><P>On all of the following nine pictures the plate to the right is the negative control while the bacterias with colicin Fy added is to the left. The first three pictures shows 3 different dilutions, all done at time zero. They are in the order, not diluted, diluted 10 times and diluted 100 times. The three pictures after that are diluted in the order of 10, 100 and lastly 1000 times , they are all plated after 2 hours. The last 3 plates are also diluted, 10, 100 and lastly 1000 times and are </P><I>Right: Negative control, Left: Colicin Fy, time 0 hour, not diluted</I><I>Right: Negative control, Left: Colicin Fy, time 0 hour, diluted 1:10</I><I>Right: Negative control, Left: Colicin Fy, time 0 hour, diluted 1:100</I><I>Right: Negative control, Left: Colicin Fy, time 2 hour, diluted 1:10</I><I>Right: Negative control, Left: Colicin Fy, time 2 hour, diluted 1:100</I><I>Right: Negative control, Left: Colicin Fy, time 2 hour, diluted 1:1000</I><I>Right: Negative control, Left: Colicin Fy, time 4 hour, diluted 1:10</I><I>Right: Negative control, Left: Colicin Fy, time 4 hour, diluted 1:100</I><I>Right: Negative control, Left: Colicin Fy, time 4 hour, diluted 1:1000</I><P>As seen in all the pictures the bacterias with colicin Fy are less or about the same amount of bacteria in all the pictures.</P><P>The following 4 pictures are all Y.enterocoliticas plated on Yersinia specific plates to make sure that it isn´t a contamination that we see. All the plates in the first three pictures have all been dilluted 10-1while the plates on the last picture was diluted 10-4. </P><I>Plates at time zero. The two plates to the left are J23106-B0034-CFY on different backbones and the last one to the right is a negative control.</I><I>Plates after two hours. The two plates to the left are J23106-B0034-CFY on different backbones and the last one to the right is a negative control</I><I>Plates after 4 hours. The two plates to the left are J23106-B0034-CFY on different backbones and the last one to the right is a negative control</I><I>Plates after 26 hour. The two plates to the left are J23106-B0034-CFY on different backbones and the last one to the right is a negative control</I><P>In conclusion colicin Fy inhibits the growth of yersinias very fast and the effects lasts for a long while acording to the results, the negative control always grows much more.</P></DIV></DIV><H3>Colicin Fy specificity test</H3><P>To see if the Colicin Fy really is as specific as we thought, we had to try it on other bacteria. Apart from testing them on <I>Y. enterocolitica</I> we also managed to get a hold of risk class one versions of <I>Pseudomonas</I>, <I>Enterobacter</I>, <I>Klebsiella</I> as well as <I>Lactobacillus plantarum</I> and the standard <I>E. coli</I> DH5-alpha. We did multiple tests with varying results.  The tests were performed in the same way as the inhibition test above. However OD600 measurements were made here since no aggregations could be seen in most of our cultures, see table A. The table indicates that the colicin Fy is probably specific, but something seems to have gone wrong with the plating, see figures E, and that result was inconclusive.</P><H3>OD value after 2H</H3><TABLE id="partsT" style="width:100%;"><TBODY><TR><TH>DH5-alpha</TH><TH>Pseudomonas</TH><TH>Enterobacter</TH><TH>Klebsiella</TH><TH>Lactobacillus</TH></TR><TR><TD>Control</TD><TD>0.390</TD><TD>0.085</TD><TD>0.620</TD><TD>0.563</TD><TD>1.100*</TD></TR><TR><TD>Colicin Fy</TD><TD>0.378</TD><TD>0.098</TD><TD>0.618</TD><TD>0.578</TD><TD>0.109*</TD></TR></TBODY></TABLE><H3>OD value after 4h</H3><TABLE id="partsT" style="width:100%;"><TBODY><TR><TH>DH5-alpha</TH><TH>Pseudomonas</TH><TH>Enterobacter</TH><TH>Klebsiella</TH><TH>Lactobacillus</TH></TR><TR><TD>Control</TD><TD>0.520</TD><TD>0.137</TD><TD>0.639</TD><TD>0.669</TD><TD>1.747*</TD></TR><TR><TD>Colicin Fy</TD><TD>0.503</TD><TD>0.139</TD><TD>0.658</TD><TD>0.748</TD><TD>1.815*</TD></TR></TBODY></TABLE><H3>OD valueafter 24H</H3><TABLE id="partsT" style="width:100%;"><TBODY><TR><TH>DH5-alpha</TH><TH>Pseudomonas</TH><TH>Enterobacter</TH><TH>Klebsiella</TH><TH>Lactobacillus</TH></TR><TR><TD>Control</TD><TD>1.003**</TD><TD>1.903</TD><TD>1.151**</TD><TD>1.012</TD><TD>2.589**</TD></TR><TR><TD>Colicin Fy</TD><TD>1.464**</TD><TD>1.990</TD><TD>1.825**</TD><TD>1.035</TD><TD>2.540*</TD></TR></TBODY></TABLE><P><I> *The broth used to grow the lactobacillus bacteria was not available to calibrate the spectrophotometer. Therefore the values will be wrong but the comparison between the control and the colicin Fy should still be valid.
**The bacteria had created precipations which made the measurings unreliable.
</I></P><I>Figure 5: Specificity inhibition test. Top row is control plates, bottom row is with colicin. From left to right <I>Lactobacillus Plantarum</I>, <I>Klebsiella</I>, <I>E. coli</I>,  <I>Pseudomonas</I>, <I>Enterobacter</I>.</I><P>Since the experiment did not give any clear results we did the experiment again with new lysates, one with colicin Fy and one negative control. At first it seemed as if the colicin Fy killed most of them but at a closer look we saw two types of colonies on some of the plates, probably contamination. When plating out the lysates that should be sterile, some bacteria still grew, but they grew much more in the controls. If the contaminated plates are ignored the colicin Fy seems specific, but more experiments need to be done to be sure. Due to lack of time this was not possible.</P><I>Figure 6: The E on the plates stands for Enterobacter, the C stands for control lysate added while B for bacteriocin (colicin Fy) added. the numbers 10, 100 and 300 stands for the amount of µl of lysate added.</I><I>Figure 7: Controls of the lysate to see if they were contaminated or not, to the left is the control lysate while the bacteriocin is to the right.</I><P>As seen in fig. 7 the control lysate is much more contaminated than the bacteriocin, this explains why the control plates in all cases have more bacteria on the plates plated with 300 µl of lysate added compared with all the others. Different types of colonies can also be seen on most of them. 

In conclusion there are some indications that colicin Fy is specific but more tests need to be done.
</P><DIV class="panel"><H2>Read more/Hide</H2><DIV class="panelcontent"><I>Appendix 2: The K on the plates stands for klebsiella, the C stands for control lysate added while B for bacteriocin (colicin Fy) added. the numbers 10, 100 and 300 stands for the amount of µl of lysate added.</I><I>Appendix 3: The D on the plates stands for DH5-alpha (E. coli), the C stands for control lysate added while B for bacteriocin (colicin Fy) added. The numbers 10, 100 and 300 stands for the amount of µl of lysate added.</I><I>Appendix 4: The P on the plates stands for pseudomonas, the C stands for control lysate added while B for bacteriocin (colicin Fy) added. The numbers 10, 100 and 300 stands for the amount of µl of lysate added.</I></DIV></DIV><H2>Parts</H2><TABLE id="partsT" style="width:100%"><TBODY><TR><TH>Fav.</TH><TH>BioBrick code</TH><TH>Type</TH><TH>Construct</TH><TH>Description</TH><TH>Designers</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381014">BBa_K1381014</A></TD><TD>Regulatory</TD><TD>J23101-spot42_USP45_10bp</TD><TD>Silencing sRNA with affinity to USP45</TD><TD>Killing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381015">BBa_K1381015</A></TD><TD>Regulatory</TD><TD>J23101-spot42_USP45_15bp</TD><TD>Silencing sRNA with affinity to USP45</TD><TD>Killing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381016">BBa_K1381016</A></TD><TD>Regulatory</TD><TD>J23101-spot42_USP45_20bp</TD><TD>Silencing sRNA with affinity to USP45</TD><TD>Killing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381017">BBa_K1381017</A></TD><TD>Coding</TD><TD>CFY-X6 his</TD><TD>The gene coding for the bacteriocin colicin Fy</TD><TD>Killing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381018">BBa_K1381018</A></TD><TD>Tag</TD><TD>B0034-USP45</TD><TD>The secretion tag USP45 with the RBS B0034</TD><TD>Killing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381019">BBa_K1381019</A></TD><TD>Device</TD><TD>B0034-pelB-CFY-X6 his</TD><TD>The bacteriocin colicin Fy coupled to the the secretion tag pelB</TD><TD>Killing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381020">BBa_K1381020</A></TD><TD>Generator</TD><TD>J23106-B0034-pelB-CFY-X6 his</TD><TD>Colicin Fy coupled to the the secretion tag pelB with the promoter J23106</TD><TD>Killing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381021">BBa_K1381021</A></TD><TD>Generator</TD><TD>J23116-B0034-pelB-CFY-X6 his</TD><TD>Colicin Fy coupled to the the secretion tag pelB with the promoter J23116</TD><TD>Killing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381022">BBa_K1381022</A></TD><TD>Device</TD><TD>B0034-USP45-CFY-X6 his</TD><TD>The bacteriocin colicin Fy coupled to the the secretion tag USP45</TD><TD>Killing Group</TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K1381023">BBa_K1381023</A></TD><TD>Generator</TD><TD>J23106-B0034-CFY-X6 his</TD><TD>Colicin Fy coupled to the promoter J23106</TD><TD>Killing Group</TD></TR></TBODY></TABLE><UL class="reference"><LI><A id="reference1">[1]</A> (2013) &quot;Bacteriocins — a viable alternative to antibiotics?&quot; Paul D. Cotter et al</LI><LI><A id="reference2">[2]</A> (2012) &quot;Novel Colicin FY of Yersinia frederiksenii Inhibits Pathogenic Yersinia Strains via YiuR-Mediated Reception, TonB Import, and Cell Membrane Pore Formation&quot; J.Bosak et al</LI><LI><A id="reference3">[3]</A> (2012) https://2012.igem.org/Team:Uppsala_University/Project</LI></UL></DIV></DIV><DIV class="back-to-top"><A>Back to Top</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Uppsala/Project_Killing">http://2014.igem.org/Team:Uppsala/Project_Killing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Uppsala/Project_Killing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Uppsala/Project_Killing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Uppsala/Project_Killing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Uppsala/Project_Killing&amp;oldid=357349" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>