"
Page
Discussion
View source
History
teams
Log in
 
Team:Uppsala/Project Sensing
From 2014.igem.org
Stephanie Herman
Teresa Reinli
Joakim Hellner
Alexander Virtanen
Jennifer Rosenius
Marcus Hong
Miranda Stiernborg
Tim Hagelby Edström
Viktor Blomkvist
Megha Biradar
Niklas Handin
Jonas Mattisson
Arina Gromov
Nils Anlind
Eric Sandström
Gunta Celma
Oliver Possnert
Martin Friberg
Kira Karlsson
Christoffer Andersson
Laura Pacoste
Andries Willem Boers
Project
Sensing
Targeting
Killing
Adhesion
Notebook
Modeling
Cell-Cell Interaction
Population Level
Interlab Study
Parts
Team
Sensing Group
Targeting Group
Killing Group
Adhesion Group
Modeling Group
Policy & Practices
Microbial Designs
Outreach
Business Plan
Safety
Attributions
Home
Failed to load tracking. JS is probably not enabled
Background
In our project we developed a system to fight pathogens in the gut, which is not a unique idea. Except we wanted our systems key attribute to be specificity. To do so, we needed to find a unique quorum sensing language that we could hijack and utilize. We found the Yen system, from Yersinia enterocolitica. A homologous sytem to the well known Lux system, from Vibrio fischeri.
System design
From the Yen system we stole the activator YenR and the recognition region, the yenbox, fused with a wild type promoter. YenR was synthesized with the RBS B0034 and codon optimized for E. coli. A customized version of the yenbox-promoter was also created.
Result
We managed to get a five-fold induction relative to the base level of expression. The cells that were used during the measurements were double transformed with the yenbox-promoter complex with GFP and a plasmid producing the activator YenR.
Parts
BBa_K1381000
BBa_K1381001
BBa_K1381002
BBa_K1381004
BBa_K1381005
BBa_K1381006
BBa_K1381007
BBa_K1381008
BBa_K1381009
BBa_K1381010
The assembly plan of the Sensing system.
Background
Communication using quorum sensing
Quorum sensing (QS) is the communication tool of bacteria. It is usually done through secretion and detection of small signal molecules or proteins. Depending on the bacterial density, these molecules can trigger a change of behaviour, i.e. invoke a function only when there are enough bacteria present in that one area. Quorum sensing can control several things, from biofilm formation to bioluminescence. The most commonly known and most explored QS system is the Lux system, which was originally isolated from Vibrio fischeri, where it controls the production of luciferase, an enzyme that aids in the production of a yellow light.
Using the pathogens own quorum sensing system to detect and track its source
When we decided that we wanted to fight pathogens it became obvious that we needed a way to detect and track them. After doing some research, we decided that this could be  done by hijacking the pathogen’s own quorum sensing. Naturally, the first thought that came to our minds was to use the Lux system from V. fischeri. However, this had been done before in 2008 by Heidelberg and it also did not provide the specificity that we wanted for our system. We decided to dig deeper and see if we could find a more unique quorum sensing system. Among some potential candidates was Yersinia enterocolitica, which uses a quorum sensing system called the Yen system.
The Yen system
As it turns out Y. enterocolitica has a homologous quorum sensing system to the famous Lux system, the Yen system. From this system we chose to steal two parts. A recognition region, the yenbox, fused together with a promoter and an activator, YenR, that can recognise and interact with the yenbox. When YenR binds to the yenbox it induces the expression level of the promoter fused to the yenbox. Later, in the presence of Y. enterocolitica, its signaling molecules, 3-oxo-hexanoyl homoserine lactone (OHHL), will start flowing into our system, interacting with YenR. When binding occurs between OHHL and YenR, YenR will lose its active shape and thereby its ability to interact with the yenbox. The induction will then be lost and the expression level will return to its base level. [1]
By BioBricking and characterizing this system, we wish to make it possible to get a Y. enterocolitica density triggered response. Our goal with this system is to control our two other systems, the Targeting and the Killing system.
System Design
While stealing the yenbox together with the wild type promoter fused with it, we had no idea about the strength of the wild type promoter or if it would even work at all. Because of this, we created an alternative version where we replaced the wild type promoter with a standardised one (J23113). In the wild type version there is an overlap between the wild type promoter and the yenbox. Hence, we mimicked the same while creating our customized version where we had an overlap between the promoter and the yenbox. Since the strength of the promoter would correspond to the leakage in our system, we wanted to have a weak promoter to minimize the leakage. We chose the constitutive promoter J23113 ( BBa_J23113 ) from the Anderson library. Unfortunately, the promoter J23113 did not begin with the same two bases as the end of the yenbox. We were left with the option of either changing the two bases in the yenbox sequence or changing the two bases in the sequence of the promoter J23113. In the article by Ching-Sung Tsai and Stephen C. Winanas [1] they discovered that the binding between the activator YenR and the recognition region of the yenbox is not dependent on the entire sequence of the yenbox. Depending on which part of the yenbox that is changed or replaced, YenR binds to the yenbox with different strengths. However, it still interacts with the yenbox and induces the strength of the promoter. Based on this fact, together with the knowledge that the Anderson promoters are very sequence dependent, we chose to change two bases in the sequence of the yenbox.
We also stole the coding sequence [2] of the activator YenR, from Y. enterocolitica, which we codon optimized for E. coli using a IDTs web tool [3] and synthesized it together with the RBS B0034 ( BBa_B0034 ). Since we always want production of the activator YenR, it was coupled to three different constitutive promoters from the Anderson promoter library with three different strengths.
Result
For characterisation we created the constructs yenbox_WT-B0034-GFP ( BBa_K1381008 ) and yenbox_J23113-B0034-GFP ( BBa_K1381009 ), where the yenbox fused with a promoter was coupled to the green fluorescent protein (GFP). These constructs were then cloned into the backbones pSB1C3 and pSB3C17 and transformed into competent E. coli cells already containing one of the YenR constructs J23110-B0034-YenR ( BBa_K1381005 ), J23102-B0034-YenR ( BBa_K1381006 ) or J23101-B0034-YenR ( BBa_K1381007 ) on the backbone pSB1K3. The double transformed cells were then streaked on plates containing both the antibiotic Kanamycin and Chloramphenicol and left it overnight to grow. Cells containing only the constructs yenbox_promoter-B0032-GFP were also streaked and left to grow.
The following day, overnight cultures were prepared and left for 16 h to grow into stationary phase. After that, 10 µL of the overnight culture was put into 500 µL of PBS solution and left for one hour for stabilization. The green fluorescence production was then measured using a flow cytometer. The results of the test is shown below.
Graph 1 above, shows the induction relative to the base level of expression. In graph 1.A. we can observe the induction but cannot predict to what extent. This is because these cells are transformed with two high copy plasmids containing the same ori (origin of replication). The consequence will be that the cells will confuse the two plasmids with each other and have no control of in what amount the two respective plasmids are present. It will only ensure that it is 100-300 plasmids present in total, regardless of which plasmid it is. This is why we also cloned the yenbox containing constructs into a low copy plasmid with a different ori than the high copy ones.
In graph 1.B., it can be seen that the amount of YenR that is produced is correlated, as expected, to the rate of the induction. When coupled to the strongest of the three promoters, J23102 ( BBa_J23102 ), the production is increased up to five folds.
In graph 1.C., we did not observe any induction  This implies that our customized version of the yenbox fused with the promoter J23113 did not work as intended. It seems like the activator YenR could not recognise nor interact with the yenbox due to the changes made in the last two bases of the yenbox. It should be tested to redesign this part so that the yenbox is not fused to the promoter but just simply put beside it without any overlap. If the induction is not extremely distance dependent, that might work better. Otherwise, you could try to find other promoters, were the first two bases match with the two last bases of the yenbox, so that we don’t have to modify the yenbox.
These kind of constructs were also constructed with a blue fluorescence protein (BFP) that were also transformed into cells already containing one of the YenR constructs J23110-B0034-YenR ( BBa_K1381005 ), J23102-B0034-YenR ( BBa_K1381006 ) or J23101-B0034-YenR ( BBa_K1381007 ) on the backbone pSB1K3. They were then plated and restreaked together with a non-fluorescence containing reference and a base level reference without any YenR. These plates were used to see if the induction could be seen in UV light, with the naked eye. Unfortunately, as you can see in fig. 1 below, this was not the case. The color could be seen, but there was barely any difference in the intensity of the different cells.
It should also be mentioned that we did an attempt to create bigger constructs containing the characterisation constructs with the yenbox coupled to GFP and the YenR constructs (yenbox_promoter-B0034-GFP-terminator-promoter-B0034-YenR). But for some reason these assemblies did not work. After some troubleshooting we came to the conclusion that it was too stressful for our bacteria to produce both BFP and YenR on the same high copy plasmid or that these constructs were not stable in E. coli and got rejected or mutated by the cell.
Parts
Fav.
BioBrick code
Type
Construct
Description
Designers
BBa_K1381000
Regulatory
yenbox_WT
The Yen systems recognition region fused with a wildtype promoter
Sensing Group
BBa_K1381001
Regulatory
yenbox_J23113
The Yen systems recognition region fused with the promoter J23113
Sensing Group
BBa_K1381002
Regulatory
yenbox_J23101
The Yen systems recognition region fused with the promoter J23101
Sensing Group
BBa_K1381004
Coding
B0034-YenR
YenR, the activator in the Yen system
Sensing Group
BBa_K1381005
Regulatory
J23110-B0034-YenR
The activator YenR coupled to the promoter J23110
Sensing Group
BBa_K1381006
Regulatory
J23102-B0034-YenR
The activator YenR coupled to the promoter J23102
Sensing Group
BBa_K1381007
Regulatory
J23101-B0034-YenR
The activator YenR coupled to the promoter J23101
Sensing Group
BBa_K1381008
Measurement
yenbox_WT-B0032-GFP
A characterisation construct for the yenbox with the wildtype promoter
Sensing Group
BBa_K1381009
Measurement
yenbox_J23113-B0032-GFP
A characterisation construct for the yenbox with the promoter J23113
Sensing Group
BBa_K1381010
Measurement
yenbox_J23101-B0032-GFP
A characterisation construct for the yenbox with the promoter J23101
Sensing Group
[1] Ching-Sung Tsai and Stephen C. Winanas, The quorum-hindered transcription factor YenR of Yersinia enterocolitica inhibits pheromone production and promotes motility via a small non-coding RNA, 2011, Molecular Microbiology 80[2], 556-571
[2] Yersinia enterocolitica subsp. enterocolitica 8081 chromosome, complet - Nucleotide - NCBI [WWW Document], n.d. URL http://www.ncbi.nlm.nih.gov/nuccore/123440403/?from=1798138&to=1798872 (accessed 10.17.14).
[3] Codon Optimization [WWW Document], n.d. URL http://eu.idtdna.com/CodonOpt (accessed 10.17.14).
Back to Top
Retrieved from " http://2014.igem.org/Team:Uppsala/Project_Sensing "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
