"
Page
Discussion
View source
History
teams
Log in
 
Team:Uppsala/Project Adhesion
From 2014.igem.org
Stephanie Herman
Teresa Reinli
Joakim Hellner
Alexander Virtanen
Jennifer Rosenius
Marcus Hong
Miranda Stiernborg
Tim Hagelby Edström
Viktor Blomkvist
Megha Biradar
Niklas Handin
Jonas Mattisson
Arina Gromov
Nils Anlind
Eric Sandström
Gunta Celma
Oliver Possnert
Martin Friberg
Kira Karlsson
Christoffer Andersson
Laura Pacoste
Andries Willem Boers
Project
Sensing
Targeting
Killing
Adhesion
Notebook
Modeling
Cell-Cell Interaction
Population Level
Interlab Study
Parts
Team
Sensing Group
Targeting Group
Killing Group
Adhesion Group
Modeling Group
Policy & Practices
Microbial Designs
Outreach
Business Plan
Safety
Attributions
Home
Failed to load tracking. JS is probably not enabled
Background
To maximize the specificity in targeting Yersinia enterocolitica, we wanted to design a construct consisting of a membrane protein complex that would bind to the surface of the pathogen. This could be done by fusing an anchor protein with the colicin Fy that aimed at the membrane of Y. enterocolitica. This complex would be expressed on the surface of our probiotic, making it adhere to its target.
System design
We made two constructs, using a synthesized gene coding for pgsA. In the first one we fused pgsA with colicin Fy and added a promoter aswell as a RBS called B0034. In the other, BFP was put instead of colicin Fy to test the expression of pgsA.
Result
Mutagenesis of B0034-BFP into RFC25 was successful, verified by sequencing. All other constructs were not successful. Mainly because we had trouble with the RFC25-assemblies. The removal of TACTAG was also not successful.
Parts
BBa_K1381024
BBa_K1381025
Background
Purpose of making an adhesion system between Y.enterocolitica and the Bactissile
While creating a biological machine that can efficiently kill specific pathogens without disturbing other cells in its environment, an adhesion system could play a vital role. Getting our Bactissile to attach to its target, in our case Yersinia enterocolitica , could give important advantages in target specificity and delivering medicine in a way that ensures it reaches Y. enterocolitica in a concentration as high as possible.
Technical Approach
The idea was to manipulate our Bactissiles bonding to any of Y. enterocoliticas surface structures and thus preventing the pathogen from ever attaching to the gut wall. This could be done with a DNA construct coding for a membrane protein that either resembles the bonding that Y. enterocolitica makes to our cells or one that itself binds to Y. enterocolitica.
We studied the ways of infection of Y. enterocolitica and could determine that its main strategies of entering through the gut wall is by using its diverse membrane proteins YadA, invasin and Ail. YadA and invasin work together to bind to our cell membranes. While YadA binds to collagen, invasin structurally resembles fibronectin, that is present in the extracellular matrix, which is recognized by the fibronectin binding membrane protein integrin alpha-5-beta-1 in our cell membranes. [1] Y. enterocolitica uses our own cell-to-cell binding mechanism to find a target cell and trick it to endocytose by binding to the integrin and to the collagen outside and in between the cells. [2] At first glance we thought of manipulating our probiotics to also use the same bioorganic systems. That could be done by expressing collagen-like proteins and fusing them with an anchor protein that could transport the whole construct to the outer membrane and keep the collagen attached to the probiotic surface. [3]
By using collagen, and especially putting it on the surface of a bacterial outer membrane that is meant to enter the human body, is an extremely risky procedure. “As antibodies would be produced to target the collagen and anchor protein complex this could lead to an autoimmune reaction as they would also react on the collagen naturally present in our body” says Lars Hellman, a professor in immunology at Uppsala University. More about problems with Y. enterocolitica-adhesion and the development of autoimmune  disease can be found in this text .
Instead we took inspiration from our teams killing system which secretes a bacteriocin, CFY, which specifically binds to and ruptures the outer cell membrane of Y. enterocolitica. [4] The main focus of the adhesion system drastically changed from expressing human like proteins to expressing the bacteriocin fused with the anchor protein.
If the bacteriocin would survive degradation in stomach like environments and later on find Y. enterocolitica, the bacteriocin, while attaching to the membrane, would  fix the probiotic to its target. Even though this connection would be much weaker than a system with collagen, it is safe and still an advantage to the Bactissile.
System Design
The Anchor
Since we want our system to be usable in a probiotic bacteria we looked for an anchor motif that works both in our well characterised prototype bacteria (E. coli) and in a well known probiotic bacteria like Lactobacillus. We also wanted the anchor motif to be usable with a wide range of protein types since this would be a great addition to the iGEM registry, since no construct for cell surface display could be found by us at the start of our project. In the end we found the gene coding for PgsA from Bacillus subtilis that had been used successfully in both E.coli and a Lactobacillus species. To prevent interaction with the fused domain we added a long (20aa) rigid linker [BBa_J176131] to the C-terminal of the anchor.
The fusion
To make the fusion protein of anchor and colicin Fy (PgsA-CFY) we chose to synthesize the genes in RFC25 standard (Freiburg). This adds two additional restriction enzyme sites (NgoMIV and AgeI) to the four standard ones, thus enabling simple assembly of the two proteins so that they are translated as one in the same reading frame.
Back up plan
We used normal assembly (RFC10) with re-ligation of XbaI and SpeI sites that leaves a TACTAG-scar where TAG can stop translation and therefore disable production of a fusion protein. A solution would then be to perform PCR-mutagenesis and remove the TACTAG-scar via point mutation.
Characterising the anchor
To test that the anchor attaches to the membrane, we tried to fuse a reporter, the blue fluorescence protein BFP, to the anchor. We chose to mutagenise B0034-BFP [BBa_K592024] to make it compatible with RFC25. It was designed to get an ATG followed by an NgoMIV site upstream of the BFP CDS. The original ATG from the BFP CDS was removed. In the end we would still have to mutagenise BFP, because we never got the RFC25 assembly to work.
Results
Mutagenesis of BFP into RFC25 was successful, verified by sequencing. All other constructs were not successful. Mainly because we had trouble with the RFC25-assemblies. The mutagenesis to remove TACTAG also failed because the reverse primer annealed to a region upstream of the desired site causing a frameshift verified by sequencing. The repeat-like sequence of the 60bp linker was probably one of the reasons for this.
Parts
Fav.
BioBrick code
Type
Construct
Description
Designers
BBa_K1381024
Tag
B0034-pgsA-(EAAAR)x4
The anchor tag pgsA with the RBS B0034 and a linker
Adhesion Group
BBa_K1381025
Reporter
B0034-BFP (RFC25)
A mutagenised version of B0034-BFP, compatible with RFC25
Adhesion Group
[1] (2012) “Bacterial Cell Surface Structures of Yersinia”, N. Bialas
[2] (2011) “Unique Cell Adhesion and Invasion Properties of Yersinia enterocolitica” O:3, the Most Frequent Cause of Human Yersiniosis” by F. Piscano et.al.
[3] (2001) “YadA, the multifaceted Yersinia adhesin” by J. El Tahir, M. Skurnik.
[4] (2012) “Novel Colicin FY of Yersinia frederiksenii Inhibits Pathogenic Yersinia Strains via YiuR-Mediated Reception, TonB Import, and Cell Membrane Pore Formation“ by J. Bosak et al.
Back to Top
Retrieved from " http://2014.igem.org/Team:Uppsala/Project_Adhesion "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
