"
Page
Discussion
View source
History
teams
Log in
 
Team:SJTU-BioX-Shanghai/Project/CRISPRi/Introdution
From 2013.igem.org
Menu Search
Loading
Home
Project
Overview
Motivation
Black Box
Inside the BOX
Luminous System
Software
Devices
Light Sensors
Integration of Three lights
Red
Green
Blue
CRISPRi
Introdution to CRISPRi
Stay on Targets
Prospect
Prospect
Results
Tests
Luminous System
Verification of CRISPRi
Theory basis of three light systems
Overall System
Applications
Versatile
One Example - Fatty Acid Synthesis
Notebook
Protocol
Lab Log
Modeling
Light sensor characteristic
Report gene prediction
Metabolic optimization
Parts
Parts
Human Practice
Subject Cross
Research on Subject Cross
Biocraft
Communication
Safety
Safety Form
Security
Environment
Team
Our Team
Members & Attributions
Introduction to CRISPRi
From CRISPR to CRISPRi
CRISPR
CRISPR Genome Editing (Jinek et al., 2012)
CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats) and CRISPR-associated system (Cas) widely exist in Bacteria and Archea, endowing the cell with adaptive immunity to foreign DNAs. The immune process could be divided into three stages:
Adaptive Stage.    Fragments of foreign DNA (named protospacers) is incorporated into the proximal end of CRISPR.
Expression Stage.    Precursor CRISPR RNAs (pre-crRNAs) are transcribed from CRISPR. Pre-crRNAs are then cleaved to produce crRNAs.
Interference Stage.    crRNA complements with target DNA (protospacers), forming a complex that is recognized and bound by Cas nuclease (Csn). Csn excises the target, leaving the target vulnerable to successive degradation.
Three types of CRISPR system have already been revealed, of which Type II CRISPR system is the simplest – one sole protein, Cas9 (formerly named Csn1) is capable of all tasks once crRNA is complemented by trans-activating crRNA (tracrRNA). Due to this simplicity, Type II CRISPR/Cas can be ectopically expressed as a tool for sequence-specific genome editing (Jinek et al., 2012). And to further simplify the tool, Jinek et al. linked crRNA and tracr RNA together, successfully creating small guide RNA (sgRNA) as an equivalent (Jinek et al., 2012).
CRISPRi
CRISPRi Mechanism
With its endonuclease activity crippled, Cas9 can be used to hinder RNA polymerase and transcript elongation. Together with a proper sgRNA, Cas9 possessing such defect (named dCas9) can be used as an effective tool in expression interrogation – CRISPR interference, abbreviated as CRISPRi (Qi et al., 2013).
CRISPRi is promising tool for expression regulation, versatile and easy-to-use. Generally, a small shift in sgRNA would be enough for a new target.
Key Parts of CRISPRi
There are only 2 parts to be expressed -- dCas9 and sgRNA. Only 2!
dCas9
As the signature protein of the type II CRISPR/Cas, Cas9, does not show any detectable similarity to any proteins in Type I and Type III systems, in that it is sufficient both to generate crRNA and to cleave the target DNA. This large protein (about 1000 amino acids) contains two predicted nuclease domains -- the N-terminal RuvC-like nuclease (RNAse H fold) and the HNH (McrA-like) nuclease domain that is located in the middle of the protein (Makarova et al., 2011), each of which would cleave one DNA strand. To acquire dCas9, one point mutation is conducted on each nuclease, respectively, namely, D10A and H841A (Qi et al., 2013).
File:DCas9.fasta.txt
sgRNA
Structure of sgRNA
If dCas9 acts as an executive, than sgRNA is the director of CRISPRi. An sgRNA constitutes three parts:
· Base-Pairing Region: A 20 nt complementary region for specific DNA binding.
· dCas9 Handle: A 42 nt hairpin for Cas9 binding
· A terminator: derived from S. pyogenes, 42nt
Therefore, the sequence of gRNA can be represented like this:
5'-N20GUUUUAGAGCUAGAAAUAGCAAGUUAAAAUAAGGCUAGUCCGUUAUCAACUUGAAAAAGUGGCACCGAGUCGGUGCUUUUUU-3’
See the base-paring sequence
Design Criteria (Qi et al., 2013)
· Binding specificity is determined by both sgRNA-DNA base pairing and a protospacer adjacent motif (PAM), NGG, upstream of target.
· Generally, the optimal length of the complementary region is 20nt.
· If a sgRNA targets coding sequence, it has to be complementary to the non-template (NT) strand. But if a sgRNA targets the promoter, either strand would be acceptable.
· If a sgRNA targets coding sequence, generally CRISPRi would work better if the complementary region is closer to the promoter.
· The first 7 nt might be a "seed region" for binding, since any single mutation of  dramatically decreased repression.
References
LEVSKAYA, A., CHEVALIER, A. A., TABOR, J. J., SIMPSON, Z. B., LAVERY, L. A., LEVY, M., DAVIDSON, E. A., SCOURAS, A., ELLINGTON, A. D., MARCOTTE, E. M. & VOIGT, C. A. 2005. Synthetic biology: engineering Escherichia coli to see light. Nature, 438, 441-2.
TABOR, J. J., LEVSKAYA, A. & VOIGT, C. A. 2011. Multichromatic control of gene expression in escherichia coli. Journal of Molecular Biology, 405, 315-324.
Bio-X Institutions Shanghai Jiao Tong University 800, Dongchuan Road 200240 Shanghai, China
igemsjtu2013@gmail.com
Retrieved from " http://2013.igem.org/Team:SJTU-BioX-Shanghai/Project/CRISPRi/Introdution "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
