"
Page
Discussion
View source
History
teams
Log in
 
Team:SJTU-BioX-Shanghai/Notebook/Lab log
From 2013.igem.org
Menu Search
Loading
Home
Project
Overview
Motivation
Black Box
Inside the BOX
Luminous System
Software
Devices
Light Sensors
Integration of Three lights
Red
Green
Blue
CRISPRi
Introdution to CRISPRi
Stay on Targets
Prospect
Prospect
Results
Tests
Luminous System
Verification of CRISPRi
Theory basis of three light systems
Overall System
Applications
Versatile
One Example - Fatty Acid Synthesis
Notebook
Protocol
Lab Log
Modeling
Light sensor characteristic
Report gene prediction
Metabolic optimization
Parts
Parts
Human Practice
Subject Cross
Research on Subject Cross
Biocraft
Communication
Safety
Safety Form
Security
Environment
Team
Our Team
Members & Attributions
July
Week1
1.Template Plasmid Amplification from Kit Plate
We amplified BBa_K592006, BBa_K592016, BBa_K592018, BBa_J23100, BBa_B0015, BBa_I15009, BBa_I15008 and BBa_J23119 from kit plate for further construction.
2.Construct consititutional backbone plasmid
PCR to add SpeⅠand Sal I restriction sites upstream of and Pst I downstream of BBa_B0015 terminator part.
Use spe I and Pst I  to digest the PCR product and the BBa_K880005（promotor and rbs) part and then use T4 ligase to connect PCR product and backbone.
Identify by monoclonol colony PCR to find positive clone.
Sequencing results showed accurate construction of the consititutional backbone plasmid.
3.Construct luciferase plasmid.
PCR to connect luciferase into pETDuet. (Adding restriction enzyme cutting sites at both ends, Nco I and Xho I)
Recombination of luciferase (Nco I and Xho I)and pETDuet (Nco I and Xho I). Digestion 6 hours and ligation 3 hours.
Transform constructed plasmid to competent cell DH5α. Culturing overnight.
Week2
1.Construct luciferase plasmid.
Picking colonies and culturing overnight.
Extract the plasmid through miniprep. Identification by PCR.
2.Construct dCas9, pcyA and Ho1 plasmid
PCR to connect  dCas9, pcyA and Ho1 into pSB1C3. (Adding restriction enzyme cutting sites at both ends,Xba I and Sal I)
Digest with Xba I and Sal I, and pSB1C3 backbone with Spe I and Sal I. 6 hours.
Ligation and transformation (3 hours, DH5α). Culturing overnight.
Picking colonies and culturing overnight.
Identify by monocolonal colony PCR and send 2 copies of positive colony each of Ho1 and PcyA.
No positive dCas9 colony identified.
File:13igemFig3.jpg
PcyA Identification
3.Construct Blue Sensor plasmid
Add suitable enzyme cutting site though PCR
Digest and ligate, to get the constitutive operon of blue light sensor.
4.Construct Red Sensor plasmid.
PCR to connect cph8 into pSB1C3. (Adding restriction enzyme cutting sites at both ends,Xba I and Sal I)
Digest cph8 with Xba I and Sal I, and pSB1C3 with Spe I and Sal I. 6 hours.
Ligation and transformation (3 hours, DH5α). Culturing overnight.
Picking colonies and culturing overnight.
Week3
1.Construct luciferase plasmid.
Plasmid is confirmed by sequencing.
2.Construct Red Sensor plasmid.
Plasmid extraction and identification of Cph8-PSB1C3. Identification by PCR, using primers of cph8 and standard primer on pSB1C3, VR&VF2).
Concentration of plasmid is low for sequencing. Culture colonies with positive results in large scale for 24 hours.  Plasmid extraction and identification by sequencing.
3.Construct dCas9, pcyA and Ho1 plasmid
Sequencing reseult showed that the plasmid we got was the model plasmid.
The new backbone and original sequence model are both chlorampenicol resistant.
Reconstruct constitutive pcyA and Ho1 plasmid using DpnI to digest plasmid model.
Get 2 plasmids of pcyA to send sequencing.
Again didn't get positive colony of dCas9 and Ho1.
4.Construct Blue Sensor plasmid onto another Backbone
After we get the right sequencing result, we begin to use a pair of new primers doing PCR to add a new restriction enzyme site on the PCR pieces.
Cut and Paste, trying to get another plasmid.
Week4
1.Construct dCas9, pcyA and Ho1 plasmid
Constitutive pcyA-pRSF plasmid construction
PCR to add restriction enzyme cutting sites at both ends of consititutive pcyA operon, Xho1 and BamH1)
Digest pRSF-duet1 plasmid and PCR product with Xho1 and BamH1. Use elecrophoresis and gel extraction to purify pcr product and backbone sequence.
Ligation and transformation (3 hours, DH5α).
Picking colonies and culturing overnight.
Identify by monocolonal colony pcr.
Constitutive dCas9 and Ho1 pSB1C3 plasmid construction
Reconstruct Constitutive dCas9 and Ho1 pSB1C3 plasmid.
Got 1 positive colony of Ho1 and sent sequencing.
Again didn't get positive clone of dCas9.
2.Construct Red Sensor plasmid.
PCR to connect cph8 into pSB1C3.
Digest cph8 with Xba I and Sal I, and pSB1C3 with Spe I and Sal I. 6 hours. Purification pSB1C3 by gel extraction.
Ligation and transformation (3 hours, DH5α). Culturing overnight.
Picking colonies and culturing 24 hours.
3.Blue Sensor Plasmid Faces with Challenge
After several tests the result hasn't been got by us.
Repeat and try to find the smallest mistakes.
August
Week1
1.Construct dCas9, PcyA and Ho1 plasmid.
PcyA-pRSF plasmid construction
Sequencing results showed both 2 plasmids had been successfully constructed
Ho1-pSB1C3 plasmid construction
Sequencing results showed frameshift mutation in the sequence.
Reconstruct and got 2 extra positive clones and sent sequencing.
dCas9-pSB1C3 plasmid construction
Reconstruct and got 1 positive clone and sent sequencing
2.Construct Red Sensor plasmid.
Plasmid is confirmed by sequencing.
Transform pCDFDuet plasmid to DH5αin large scales for 24 hours.
Extract the plasmid through miniprep.
PCR to get cph8 sequence combined with promoter and terminator(adding restriction enzyme cutting sites, Xba I and Hind III).
Digest pro-cph8-ter sequence and pCDFDuet with Xba I and Hind III, 6 hours. Purification pCDFDuet with gel extraction.
Ligation and transformation. (3 hours, DH5α). Culturing overnight.
3.Point Mutation of Blue Sensor
Luckily we got a right clone from the plate
Sadly it has a point mutation that ends the transcription of our protein
Time to perform point mutation.
Week2
1.Construct dCas9, PcyA and Ho1 plasmid
individual dCas9-pRSF plasmid construction
Sequencing result showed accurate construction of dCas9-pSB1C3 plasmid.
Add 2 digestion sites of XbaI and XhoI outside dCas9 operon sequence by PCR.
Digest pRSF-duet1 plasmid and PCR product with XbaI and XhoI.
Ligation and transformation.
No positive colonies identified.
Constitutive ho1-pSB1C3 plasmid construction
sequencing results showed mutation in ho1 operon.
Contacted the ho1 part designer and knew that this protein has some kind of toxicity, which may influence the survival of E.coli.
Considering that Ho1 is the first enzyme in phycocyanobillin production and may product some toxic intermediates.So we decided to connect PcyA and Ho1 first on pRSF-duet1 and then use point mutation.
2.Construct Red Sensor plasmid.
Picking colonies and culturing 24 hours.
Extract the plasmid through miniprep. Identification by PCR, using primers of cph8 and primers on pCDFDuet).
Picking samples with positive results, identification by digestion(Xba I and Hind III).
Sequencing results shows that promoter and terminator have been misconnected. Intended to shorten digestion time of Hind III.
3.Still Point Mutation of Blue Sensor
Due to the low efficiency of the first pair of point mutation primers we designed, we cannot get the right one.
Still doing point mutation
Week3
1.Construct dCas9 ,pcyA and ho1 plasmid
pcyA and ho1 ligation
No positive clone identified .Concerning that the protein may be really toxic, we decided to construct T7-lac inducible ho1 plasmid.
Constitutive dCas9-pRSF plasmid construction
Reconstruct dCas9-pRSF plasmid and got 2 positive clones and sent sequencing.
2.Construct Red Sensor plasmid.
PCR to get cph8 sequence combined with promoter and terminator.
Digest pro-cph8-ter sequence and pCDFDuet with Xba I, 6 hours and Hind III, 1 hours. Purification pCDFDuet with gel extraction.
Ligation and transformation. (3 hours, DH5α). Culturing overnight.
3.Result of Point Mutation
Finally we get 2 clones, after sequencing we find there are 24 tedious bps, the other, 3 tedious bps.
Sad but maybe the second can be used.
Week4
1.Construct dCas9， pcyA and Ho1 plasmid
Constitutive dCas9-pRSF plasmid construction
Sequencing results showed dCas9-pRSF plasmid accurately constructed.
This plasmid can be used in blue light system.
T7-lac inducible pcyA, Ho1 and dCas9 plasmid construction
Using new primers to amplify PcyA sequencing （adding BamHI and PstI restriction sites ）
Digest pRSF-duet1 and PCR product and purify.
Ligasing the plasmid and pcr product and transduct.
Picking colonies and identify with PCR.
Send 2 positive clones to get sequenced.
2.Construct Red Sensor plasmid.
Picking colonies and culturing 24 hours.
Extract the plasmid through miniprep. Identification by PCR, using primers of cph8 and primers on pCDFDuet).
Picking samples with positive results, identification by digestion(Xba I and Hind III).
We found that when plasmids duplicate in pCDFDuet, they could get higher concentration and purity only on longer than 12 hours. Intend to shorten culturing time.
3.Construct green Sensor plasmid.
Insert sgRNA which points to RFP in downstream of Ccas in opposite direction.
3.Meanwhile Constructing a T7 clone
Lots of efforts have been put into the constitutively expressed genes and maybe cloning a T7 one is easier.
PCR, cut and paste
September
Week1
1.Construct t7-lac inducible dCas9,pcyA and Ho1 plasmid
Sequencing result showed both plasmids were accurately constructed.
Amplify ho1 sequence using new primers to add NdeI and XhoI restriction sites.
Digest T7-pcyA-pRSFDUET plasmid and PCR product and then purify.
Ligation and transformation. (3 hours, DH5α). Culturing overnight.
Pick colonies and identify.
No positive results showed because of contaminants.
2.Construct Red Sensor plasmid.
Considering low copy number of pCDFDuet, we change the constitute promoter into T7 promoter, an inducible one.
PCR from constructed cph8-pSB1C3 plasmid to get cph8 sequence combined with terminator (adding restriction enzyme cutting sites, Nco I and Hind III).
Digest cph8-ter sequence and pCDFDuet with Nco I, 6 hours and Hind III, 1 hour.
Ligation and transformation. (3 hours, DH5α). Culturing overnight.
Plasmid of cph8-pCDFDuet of constitutive promoter is confirmed by sequencing.
Overlap PCR to get sgRNA sequence.
3.Construct green Sensor plasmid.
Sequence of green sensors on pCDFDuet dis confirmed by sequencing.
Inserted sgRNA is confirmed by PCR and digestion.
Co-transformation: RFP, green sensor with sgRNA, dCas, segment.
4.Get the Blue sensing part
After getting the blue sensing part, next we will link the sgRNA to it
Week2
1.Construct Red Sensor plasmid.
Picking colonies of T7 promoter and culturing 12 hours.
Extract the plasmid through miniprep. Identification by PCR, using primers of cph8 and primers on pCDFDuet).
Picking samples with positive results, identification by digestion(Kpn I and Hind III).
Digest cph8-pCDFDuet and sgRNA with Kpn I, 6 hours and Hind III, 1 hour.
Ligation and transformation. (3 hours, DH5α). Culturing overnight.
Picking colonies of constitutive promoter and culturing 12 hours.
2.Keep Ligating
Strange things keep happening and we have to ligate the sgRNA to it. So harsh.
Week3
1.Construct Red Sensor plasmid.
Extract the plasmid through miniprep. Identification by PCR, using primers of cph8 and primers of sgRNA).
Picking samples with positive results, identification by digestion(Xba I and Hind III).
No positive results, ligation and transformation again.
2.Sensor+sgRNA of blue light controlled gene regulating system
Finally we get the right plasmid, next we need to test its function. Ready to set case group and control group.
Week4
1.The First Group of Data Collection
Thanks to the wonderful and successful construction of our measuring box, we are able to collect data. The very first group of exciting practical result was got just after the wiki froze event and before we left our school for Asia Jamboree.
October
Week2
1.Quantative Data Collection
We enjoyed the regional jamboree, and at the same time some of our teammates were focusd on the quantative data collection of blue light controlled sgRNA system and the target gene is RFP. Fortunately we got the quantative data before the
Bio-X Institutions Shanghai Jiao Tong University 800, Dongchuan Road 200240 Shanghai, China
igemsjtu2013@gmail.com
Retrieved from " http://2013.igem.org/Team:SJTU-BioX-Shanghai/Notebook/Lab_log "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
