"
Page
Discussion
View source
History
teams
Log in
Â 
Team:SJTU-BioX-Shanghai/Project/CRISPRi/Why
From 2013.igem.org
Menu Search
Loading
Home
Project
Overview
Motivation
Black Box
Inside the BOX
Luminous System
Software
Devices
Light Sensors
Integration of Three lights
Red
Green
Blue
CRISPRi
Introdution to CRISPRi
Stay on Targets
Prospect
Prospect
Results
Tests
Luminous System
Verification of CRISPRi
Theory basis of three light systems
Overall System
Applications
Versatile
One Example - Fatty Acid Synthesis
Notebook
Protocol
Lab Log
Modeling
Light sensor characteristic
Report gene prediction
Metabolic optimization
Parts
Parts
Human Practice
Subject Cross
Research on Subject Cross
Biocraft
Communication
Safety
Safety Form
Security
Environment
Team
Our Team
Members & Attributions
Stay on Targets
Recently, it has been reported that CRISPR/Cas exhibits noticible off-target effects (Fu et al., 2013). Although it is still under exploration and debate, the topic has no wonder raised concern of all those biologists who wish to utilize CRISPR in their research.
And for us, though we are still convinced that CRISPRi is a best choice for sequence-specific projects. It is true that, with each ribonucleotide in the gRNA as a recognizing module, recognizing one deoxyribonucleotide of the target, the accuracy of CRISPR may not be as good as Transcription Activtor-like effectors (TALEs), in which each peptide segment recognizes one pair of deoxyribonucleotides. However, the modularity of ribonucleotides are much better than peptides. Therefore, to target a 20bp DNA, we still prefer CRISPRi.
Despite all these comparison, we still take all measures to reduce potential "off-targets"(Carroll, 2013), ever from the beginning of our project. For each gRNA we design, we will check its potential mismatches, BLASTing them against the genome of Escherichia coli BL21(DE3).
We would like to share the gRNA sequence used in our project (only the base-pairing region is shown here):
To get rid of the labor of manually BLASTing, we wrote a simple Python script to help us with the work. By far the programe is only able to conduct BLAST against a local database of E. coli genome. But we plan to offer a public WEB CGI (Common Gateway Interface) that can search NCBI directly in the future.
References
FU, Y., FODEN, J. A., KHAYTER, C., MAEDER, M. L., REYON, D., JOUNG, J. K. & SANDER, J. D. 2013. High-frequency off-target mutagenesis induced by CRISPR-Cas nucleases in human cells. Nature Biotechnology, 31, 822-826.
CARROLL, D. 2013. Staying on target with CRISPR-Cas. Nature Biotechnology, 31, 807-809.
Bio-X Institutions Shanghai Jiao Tong University 800, Dongchuan Road 200240 Shanghai, China
igemsjtu2013@gmail.com
Retrieved from " http://2013.igem.org/Team:SJTU-BioX-Shanghai/Project/CRISPRi/Why "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
