Team:Hawaii/Notebook
HOME
PROJECT
EXPERIMENTS
NOTEBOOK
PARTS
TEAM
MORE
SAFETY
COLLABORATIONS
HUMAN PRACTICES
MODELLING
DEMONSTRATE
ATTRIBUTIONS
CHARACTERIZATION
CALENDAR
May 14 - May 18
May 21 - May 25
May 28 - Jun 1
Jun 4 - Jun 8
Jun 11 - Jun 15
Jun 18 - Jun 22
Jun 25 - Jun 29
Jul 2 - Jul 6
Jul 9 - Jul 13
Jul 16 - Jul 20
Jul 23 - Jul 27
Jul 30 - Aug 3
Aug 6 - Aug 10
Aug 13 - Aug 17
Aug 20 - Aug 24
Aug 27 - Aug 31
Sep 3 - Sep 7
Sep 10 - Sep 14
Sep 17 - Sep 21
Sep 24 - Sep 28
Oct 1 - Oct 5
Oct 8 - Oct 12
Oct 15 - Oct 19
MAY 14 - MAY 18
Bacteria media, milli Q water, buffers and LB agar plates were restocked in preparation for the upcoming intensity of our lab activities. While our undergraduates were preparing for final exams, iGEM graduate student Ryan designed and ordered the CR2 Gag-Protease-RT construct for our first experiment - determining the cleavage sites of the aspartyl-like protease.
HIGHLIGHTS
Designing the CR2 Gag-Protease-RT construct
Diagnostic digest and plasmid extraction
Designing the CR2 Gag-Protease-RT construct
The CR2 Gag-Protease-RT construct consists of a consensus sequence produced from 171 sequences of this Zea mays centromere retrotransposon (CR) subfamily and includes the conserved gag, protease, and reverse transcriptase domains (Conserved Domain Basic Local Alignment Search Tool). The sequences were gathered and assembled by Dan Laspisa. The construct contains N-terminal and C-terminal 8-histidine tags followed by a TEV cleavage site for later nickel resin purification and was cloned into the pET14b vector (GenScript) using NcoI and BamHI. Ryan performed transformations into DH5-alpha and BL21 cells for long term plasmid storage and expression respectively.
CR2 Gag-Protease-RT Construct
Amplicon Size (bp)
2342
Protein Size (kDa)
90
5' RE Site
NcoI
3' RE Site
BamHI
Purification
C + N-terminal 8-His and TEV cleavage site
Vector
pET14b
Diagnostic digest and plasmid extraction
A restriction enzyme digest of our CR2 Gag-Protease-RT construct was conducted to confirm plasmid uptake for two of our colonies. A plasmid extraction of the DH5-alpha colonies with this insert and a BL21(DE3) pLysSs transformation was performed.
MAY 21 - MAY 25
Colony PCR reactions and gel runs were performed throughout the week to confirm the presence of our insert in DH5-alpha and BL21 cells. Plasmids were extracted for sequence verification and a small scale 4 hour induction was performed on our BL21 cells that revealed successful expression of the full polyprotein at 28C. Meanwhile, Gina is starting to work on the Wiki and the team is researching optimized buffer conditions for aspartyl proteases in preparation for dialysis after purification.
Highlights
Confirming the presence of our CR2 Gag-Protease-RT insert
Verifying the sequence of the CR2 Gag-Protease-RT insert
Expression of the full CR2 Gag-Protease-RT polyprotein at 28C
Confirming the presence of our CR2 Gag-Protease-RT insert
A colony PCR of the DH5a CR2 Gag-Protease-RT construct containing cells initially performed by Ryan revealed a puzzling gel image. Emily and Gina performed a colony PCR of the BL21 CR2 Gag-Protease-RT construct containing cells and ran it on a 1.5% agarose gel. However, there were still no indicative bands of the insert. Emily and Gina then conducted another set of PCR reactions with leftover bacteria in the PCR tubes (invisible to the eye) in thousand-fold and million-fold dilutions and modified the PCR parameters. The gel run this time revealed our insert band at ~2 kb.
Colony PCR Reaction Mixture
Reagent
Amount (ul)
Template
11 ul
5X OneTaq Buffer
3 ul
dNTP Mix
0.3 ul
T7 Forward
0.3 ul
T7 Reverse
OneTaq
Total
15 ul
Parameters for Colony PCR
Initial PCR Parameters
Modified PCR Parameters
Temperature (C)
Time
Temperature (C)
Time
94
3 min.
94
3 min.
94
20 sec.
94
20 sec.
65
20 sec
*50
20 sec
68
1 min. 10 sec.
68
*3 min.
Verifying the sequence of the CR2 Gag-Protease-RT insert
A plasmid extraction was performed by Shelby, Fernanda, and John from CR2 Gag-Protease-RT BL21 cells that were grown overnight to send to sequencing with primers flanking our insert and internal primers. The sequencing machine was expected to yield ~500-800 readable bases, thus, internal primers were ordered to cover the gaps. Initially, an incorrectly labelled buffer was added during plasmid extraction so DNA concentration was low (5 ng/ ul). However, there was enough supernatant to try it again. After Nanodrop Quantification, a much higher DNA concentration (70-90 ng/ul) was obtained. PCR tubes were then prepared with DNA, primers (T7F, T7R, Internal GagF, Internal GagR), and PCR grade H20 and submitted it to the sequencing lab on campus. We set up overnight cultures of colonies 1, 2, and 3 for induction the next morning.
Expression of the full CR2 Gag-Protease-RT polyprotein at 28C
A 4-hour induction at 28C or 37C with 0.5 mM or 1 mM IPTG was performed by Gina and later run on a 15% SDS-PAGE gel by Emily. The 28C gel revealed higher intensity bands slightly under the 100 kDa ladder band which corresponded to the ~90 kDa full polyprotein. At 37C, little to no polyprotein expression could be differentiated between uninduced and induced cells.
MAY 28 - JUN 1
Bands corresponding to partial degradation fragments after protease cleavage were extracted from 15% SDS-PAGE gels and sent to a mass spectrometry facility to reveal protease cut sites. A soluble purification was performed on the induced BL21 cells and all fractions (total protein, wash, and elution) were run on a 15% SDS-PAGE gel.
Highlights
Analysis of polyprotein expression at 28C, 32C, and 35C
Soluble purification confirms absence of polyprotein in soluble fraction
Sending bands corresponding to our polyprotein fragments to Taplin
Analysis of polyprotein expression at 28C, 32C, and 35C
To determine the effect of temperature on protease activity, a 6 hour induction was performed at 32C or 35C. Induced cells were run on a 15% SDS-PAGE gel at 70 V for stacking and 100 V for the resolving gel. A gel was run with cells from all different temperatures so far, however, unclear separation in the gel prevented us from coming to a definite conclusion about the effect of temperature on protease activity.
Soluble purification confirms absence of polyprotein in soluble fraction
Although we expected to find our polyprotein in the insoluble fraction, a soluble purification of induced cells were performed to confirm this. During soluble purification, cells were nutated in wash (Lysis Equilibrium Wash buffer) and elution buffers. All fractions were run on a 15% SDS-PAGE gel.
Sending bands corresponding to our polyprotein fragments to Taplin
~90 kDa, ~75 kDa, and ~50 kDa bands were extracted from three different 15% SDS-PAGE gels containing 6 hour induced BL21 cells at 32 C and sent to the Taplin Mass Spectrometry Facility to reveal protease cut sites. Ryan and Vishal analyzed bands to extract from and conducted the gel extraction.
JUN 4 - JUN 8
We explored our iGEM Distribution Kit and transformed E. coli to express red fluorescent protein (RFP). Various insoluble protein purifications were performed on our induced CR2 Gag-Protease-RT containing cells and run on 15% SDS-PAGE gels.
Highlights
Expression of RFP in E.coli
Insoluble purification reveals CR2 Gag-Protease-RT polyprotein in inclusion bodies
Expression of RFP in E.coli
Shelby performed a transformation of E.coli to express RFP using the guides provided by iGEM. The RFP BioBrick from the iGEM kit plate #3 (part BBa_E1010) was used. The first transformation using previous transformation protocols from our lab notebook did not yield visible colonies. However, the second transformation using transformation protocols specified by iGEM yielded red colonies in the control plate. Shelby plans to make a working stock of transformed E.coli using the positive control colonies.
Insoluble purification reveals CR2 Gag-Protease-RT polyprotein in inclusion bodies
Gina and Jon performed an insoluble protein purification of CR2 Gag-Protease-RT containing cells using nickel-resin, Lysis Equilibrium Wash buffer, and elution buffer. A 15% SDS-PAGE gel revealed ~90 kDa, ~75 kDa, and ~50 kDa fragments corresponding to the expected degradation fragments from our polyprotein. An insoluble protein purification without lysozyme was performed by Gina as lysozyme was observed to have a similar molecular weight to the protease. The fractions of this purification were run on a tricine gel in the hopes of being able to observe the smaller ~10-23 kDa fragments. However, this was the first time our lab performed a tricine gel run and experimental protocols were used.             The Tricine gel experiment was unsuccessful in separating the protease and the smaller RT protein. This may have been due to the modifications made to Haider et al., 2012’s protocols, such as running the gel at 70 V stacking and 150 V running rather than the continuous 125-150 V.  We decided that we will determine the RT and protease sequence via mass spectrometry and trypsin digest rather than redoing another tricine gel experiment.
JUN 11 - JUN 15
A 20C induction was performed to further test protease activity at various temperatures and cells induced at 20C, 28C, and 35C were run on the same 15% SDS-PAGE gel. Attempts at verifying the sequence of our insert and Shelby’s RFP project are continued.
Highlights
Inducing cultures at 20C, 28C, and 35C to determine the effect of temperature on protease activity
Sequence verification attempts of BL21 and DH5-alpha cells
Colony PCR of transformed E.coli with RFP
Inducing cultures at 20C, 28C, and 35C to determine the effect of temperature on protease activity
A 6 hour induction of our CR2 Gag-Protease-RT BL21 cells at 20C was performed to investigate the effect of even lower temperatures on protease activity. A 6 hour induction of cells at 28C and 35C was performed a few days later to run all induced and uninduced cells at 3 different temperatures. Unfortunately the bands did not separate well but slightly show partial degradation products at all induced temperatures. The gel will be rerun next week.
Sequence verification attempts of BL21 and DH5-alpha cells
Our previous internal primers could not completely cover the full length CR2 Gag-Protease-RT sequence. New internal primers were ordered to sequence our entire ~2 kb insert and used in our sequencing reactions. Initial overnight cultures of DH5-alpha yielded low concentration of DNA, thus, another plasmid prep was run the next day. Plasmid extraction of BL21 cells were sent to sequencing with the new primers. Unfortunately, chromatogram results did not yield the ~500-800 bp expected from the sequencing machine. In order to continue the team’s efforts and ensure better results, Jon took on the task of verifying sequences and will investigate the reason for our poor sequencing results.
Colony PCR of transformed E.coli with RFP
A colony PCR to verify the insert was performed with transformed E.coli expressing the RFP.
Reaction Mix (1 Reaction)
Reagent
Amount (ul)
Template
11
5X OneTaq Buffer
3
dNTP mix
0.3
VF2 Primer
0.3
VR Primer
0.3
OneTaq
0.1
Total
15
Parameters for Colony PCR (35 cycles)
Temperature (C)
Time
94
3 min.
94
20 sec.
55
20 sec.
68
45 sec.
68
5 min.
4
Infinite (Storage)
JUN 18 - JUN 22
The 20C, 28C, and 35C gels from last week was rerun to obtain a better resolution and a PCR was conducted off of leftover plasmid to send in for another attempt at sequencing. Shelby continues her work on the RFP transformed cells and starts overnight cultures.
Highlights
Analysis of protease activity at 20C, 28C, and 35C
PCR of DH5a plasmids to amplify insert for later sequencing
Analysis of protease activity at 20C, 28C, and 35C
Another SDS-PAGE gel was made to run induced cultures from last week. The resolution obtained was much better, possibly meaning that the person who made the gel last week may have added incorrect reagent amounts. In this gel, a ~50 kDa band appears to be present in 20C and 28C induced cultures, while a stronger ~90 kDa band is observed in the 28C induced cultures. At 35C, there seems to be no difference between induced and uninduced cultures. This may be due to nonspecific cleavage or higher activity of the protease at this temperature.
PCR of DH5a plasmids to amplify insert for later sequencing
PCR and PCR cleanup (QIA-quick PCR purification protocol) was run to perform sequencing reactions until the insert sequence can be successfully verified. 250 ng of DNA was used for each reaction.
#
Template
DNA conc. (ng/ul)
1
CR2 Gag-Protease-RT (DH5a) colony 1
97.1
2
CR2 Gag-Protease-RT (DH5a) colony 2
49.6
PCR Reaction Mixture
Template
2.57
5.04
5X OneTaq Buffer
10
10
10 mM dNTP
1
1
10 mM T7 Forward
1
1
10 mM T7 Reverse
1
1
PCR grade H2O
34.18
31.71
Total
50
50
PCR Thermocycle Parameters
94
1 min.
94
20 sec.
50
20 sec.
68
3 min.
68
5 min.
4
Infinite (Storage)
JUN 25 - JUN 29
Mass spectrometry results came in from Taplin! Fragments were annotated on the sequence file and revealed a protease cleavage site downstream of the nucleocapsid. Unfortunately, we were unable to identify the second protease cleavage site through this data. However, the team decided to move forward into the next major experiment - determining the presence of VLPs in various polyproteins. These polyproteins would consist of combinations of the domains with both determined and putative protease cut sites.  The team is beginning to seriously consider dynamic light scattering, split proteins, and other equipment for VLP assessment and debated Gibson vs. Biobrick for the initial assembly of parts. Jon and Fernanda sent the PCR products to sequencing and had a discussion with the sequencer to investigate why sequencing chromatograms were not coming out as expected.
Highlights
Mass spectrometry results reveal protease cleavage site downstream of the nucleocapsid
Discussing combinatorial assay for VLP formation
Analysis of previous teams working with virus like particles (VLP)
Mass spectrometry results reveal protease cleavage site downstream of the nucleocapsid
Tryptic fragments for each band were mapped onto the CR2 Gag-Protease-RT GenScript construct using fragment data copied from the Taplin results page into an Excel sheet and sorted by position. The 100 kDa fragments covered 90.04 kDa, the 75 kDa fragments covered 56.76 kDa, and the 50 kDa fragments covered 47.88 kDa of the full construct. These fragments confirmed a protease cleavage site downstream of the nucleocapsid in the form of an 18 bp gap between the end of the last 50 kDa fragment (6 bp upstream of the 3’ end of the CCHC Zinc Finger domain) and the beginning of the next 75 kDa and 100 kDa fragments (12bp after the 3’ end of the CCHC Zinc Finger). The second cleavage site between the protease and reverse transcriptase domains could not be identified as the 75 kDa fragments did not exceed 56.76 kDa, with the most upstream protease cut site observed being located well within the protease domain. This possibly suggests that either the wrong band was excised from the PAGE gel or that the protease has auto digestive activity under the conditions used.
Discussing combinatorial assay for VLP formation
With only one cut site confirmed from the mass spectrometry results, the team decided to proceed with expressing variations of our CR2 Gag-Protease-RT construct with differences in presence and order of the domains, purification tags, and/or reporter sequences. This week, the team researched possible split reporter proteins and conducted a literature search on potential colorimetric assays.
Analysis of previous teams working with virus like particles (VLP)
The team conducted a quick search on previous iGEM teams that worked with VLPs. We found great resources from the 2012 Wageningen Wiki with VLP assembly and disassembly protocols. While surveying other teams, we also stumbled on split reporter parts from the 2017 Wageningen Wiki and sent an inquiry to Abigail, our iGEM HQ representative, about how we could obtain the parts.
Gibson vs. Biobrick assembly considerations
After considering what the combinatorial polyprotein assay would entail, the team decided to proceed with Gibson assembly to put together the initial parts. Afterwards, we would clone the successful parts into a Biobrick for submittal.
Investigation of unsatisfactory sequencing results reveal too much DNA as the cause
Following Jon and Fernanda’s discussion with the sequencer on campus, they re-submitted one tube with less DNA per his suggestion. The sequencing results came out much better! They then diluted and prepared the rest of the tubes for submittal. These results allowed us to confirm that we had the correct insert sequence in our DH5a colony.
JUL 2 - JUL 6
A same day induction at 20C, 28C, and 35C with subsequent 15% SDS-PAGE gel run confirmed protease activities at different temperatures. The team finalized the initial part combinations and ordered the first set of primers (and Gibson primers) for PCR amplification and assembly next week. In addition, the team surveyed all 342 team wikis to discover any teams with similar interests to collaborate on.
Highlights
Confirmation of protease activities at 20C, 28C, and 35C
Wiki Survey
Polyprotein combinations and primer ordering
Confirmation of protease activities at 20C, 28C, and 35C
Previous induction and gel runs for the protease experiment were performed in parts by various team members on different days of the week. This may have contributed to our unclear results on the effect of induction temperature on protease activity. To eliminate these variables, a 200 mL 20 C, 28 C, and 35 C induction and same day SDS-PAGE run was performed by one team member. Dr. Robert Paull graciously granted the team access to a third shaking incubator to perform our three inductions all at the same time. Three autoclaved 500 mL flasks covered with layers of aluminum foil were used for induction of the 200 mL cultures and 50 mL falcon tubes were used to grow the uninduced cells. The bands visible on the 15% SDS-PAGE gel conducted on the same day of the induction confirms that 20C is best for producing the full length gag protein (50 kDa),  28C is best for producing the full length CR2 Gag-Protease-RT polyprotein and 35C produces less protein than at any other temperature we have induced with so far.
Wiki Survey
The team divided the 342 team wikis into ~50 wikis to scan per person. We found that some teams were working on nanocompartments and using capsid proteins, but no other team was working with retrotransposons or looking to express virus-like particles (VLPs). Some teams were thinking of using split reporter systems, however, many of the descriptions were still slightly vague. Although we were not able to find anything extremely similar to our own project, It was interesting to see all the different ideas that other teams came up with to tackle their problem.
Polyprotein combinations and primer ordering
After filtering through a list of all possible combinations of domains, purification tags, and reporter sequences that would make up our polyprotein, the team decided on an initial list of polyproteins to assemble. We decided to amplify this off of the existing CR2 Gag-Protease-RT construct. Thus, C-terminal purification tag constructs would need to be amplified in fragments, then ligated through Gibson Assembly. We then spent an afternoon together creating regular and Gibson primers for PCR amplification of our pieces and subsequent Gibson assembly next week.
List of Initial Polyprotein Constructs
1
Construct
Amplicon (bp)
2
HisTEV-CA-NC
~1300
3
HisTEV-CA1
~950
4
HisTEV-CA2
~1100
5
CA-NC-TEVHis
~1300
6
CA1-TEVHis
~950
7
CA2-TEVHis
~1100
Jul 9 - Jul 13
In preparation for our polyprotein assembly, we cloned and digested the pET14b plasmid and amplified the parts using primers. A 1.5% agarose gel confirmed the sizes of our amplicons.
Highlights
Digesting the pET14b plasmid with NcoI and BamHI
Amplifying polyprotein pieces
Growing DH5a with the Gag-Prot-RT plasmid
Cloning and digesting the pET14b plasmid
pET14b was obtained from a glycerol stock of transformed DH5-a cells. A 100 ml overnight culture was made, and a maxi-prep was done to isolate the plasmid. A double digest was then conducted using the NcoI and BamHI restriction enzymes and a gel was run on the both the whole and digested plasmid to confirm that the plasmid was cut and linearized. Our linearized plasmid showed a band size of about 8-10 kb, which was larger than expected (pET14b is 4.6 kb).
Amplifying polyprotein pieces
A PCR was conducted to amplify all of our construct fragments. For C-terminal purification tag constructs, two fragments were amplified: one larger fragment that included the capsid (~900-1000 bp), and a smaller fragment that included the TEV cleavage site and histidine tags (~60 bp).
PCR Reaction Mix
Reagent
Amount (ul) (1 Reaction)
5X OneTaq Buffer
9
dNTP mix
0.9
OneTaq
0.1
H2O
30
Template (diluted 0.42 ng/ul)
1
Total Volume
41
Primers Added to Each Tube
Respective Forward primer
0.9
Respective Reverse primer
0.9
PCR Thermocycle Parameters (Larger fragments)
94
3 min.
94
20 sec.
61
20 sec.
68
1 min. 15 sec.
68
5 min.
4
Infinite (Storage)
PCR Thermocycle Parameters (Smaller fragments)
94
3 min.
94
20 sec.
60
20 sec.
68
20 sec.
68
5 min.
4
Infinite (Storage)
Growing DH5a with the Gag-Prot-RT plasmid
A 100ml overnight culture was done on transformed DH5-a cells containing the Gag-Prot-RT construct (GPR). A maxi-prep was done on the culture to isolate and purify the plasmid. A gel was done on the purified GPR construct with the plasmid to confirm plasmid presence. Nanodrop quantification revealed a very low yield from the first maxi-prep (~12 ng/µl) in our 20 µl sample, so a second overnight culture was prepared and a second maxi prep was done. The second maxi-prep yielded a 20 µl sample that was 136 ng/µl.
JUL 16 - JUL 20
Bacterial cultures containing the CR2 Gag-Prot-RT plasmid were grown overnight from our DH5a glycerol stocks. The pET14b plasmids and PCR amplicons from last week were digested, cleaned, then ligated. Multiple ligation reactions were created with varying amounts of vector. Transformations into commercial and homemade DH5a and BL21 cells were conducted later in the week. The second transformation initially appeared to be successful.
Highlights
Gel extraction and EtOH precipitation of vector
RE digestion and MinElute Cleanup
Ligation reactions with varying amounts of vector
Transformation in DH5a and BL21 with constructs 1, 2, 3
Gel extraction and EtOH precipitation of vector
In order to obtain more pET14b vectors, we grew overnight cultures of bacteria containing pET14b plasmids with our CR2 Gag-Protease-RT insert, digested them with NcoI and BamHI, then conducted a gel extraction of the 4671 bp vector. Following EtOH precipitation and Nanodrop Quantification, we were able to use the vector for our ligation reactions and yielded enough vector for ~30 more ligation or Gibson Assembly reactions.
RE digestion and MinElute Cleanup of PCR amplicons
PCR amplicons from last week and empty pET14b plasmids were digested with NcoI and BamHI primers, then cleaned using the MinElute cleanup kit. After cleanup, we were able to ligate these to our pET14b vector and store them at -4C overnight for transformation the next day.
Ligation reactions with varying amounts of vector
We conducted 3 sets of ligation reactions this week with different amounts of vector in a 1:3 vector:insert molar ratio. We ligated partial Gibson Assembly fragments to the vector in preparation for Gibson Assembly next week.
Ligation Reaction Mix #1 for Constructs 1 - 5a
Reagent
1 (ul)
2 (ul)
3 (ul)
4a (ul)
5a (ul)
PCR grade H2O
5.5
5.5
5.5
5.5
5.5
10X ligation buffer
1
1
1
1
1
Vector (8.4 ng)
1.5
1.5
1.5
1.5
1.5
Insert
1
1
1
1
1
T4 DNA Ligase
1
1
1
1
1
Total
10
10
10
10
10
Ligation Reaction Mix #2 for Constructs 1 - 5a
Reagent
1 (ul)
2 (ul)
3 (ul)
4a (ul)
5a (ul)
PCR grade H2O
5.39
6.17
5.73
4.11
5.68
10X ligation buffer
1
1
1
1
1
Vector (21.5 ng)
1
1
1
1
1
Insert
1.61
0.83
1.27
2.89
1.32
T4 DNA Ligase
1
1
1
1
1
Total
10
10
10
10
10
Ligation Reaction Mix #3 for Constructs 1 - 3
Reagent
1 (ul)
2 (ul)
3 (ul)
PCR grade H2O
3.69
5.20
4.39
10X ligation buffer
1
1
1
Vector (44.1 ng)
1
1
1
Insert
3.31
1.80
2.61
T4 DNA Ligase
1
1
1
Total
10
10
10
Transformation in DH5a and BL21 with constructs 1, 2, 3
We conducted one set of transformations in DH5a and BL21 and another of DH5a on both commercial and homemade competent cells. The first transformation used ligation reactions with 8.4 ng of vector and the second transformation used ligation reactions with 21.5 ng of vector. Our observations were as follows.
Transformation 1 - BL21 and DH5a with ligation reactions containing 8.4 ng of vector
2 large white colonies on positive control plate with commercial BL21 cells
Numerous small white scattered colonies on positive control plate with commercial DH5a cells
Few white colonies with smaller satellite colonies on positive control plate with homemade DH5a cells
No colony growth on negative controls
Conclusions
Plates contained the correct antibiotics
Homemade DH5a cells are competent, but much less efficient than commercial cells
8.4 ng may be too little vector to ligate with
Transformation 2 - DH5a with ligation reactions containing 21.5 ng of vector
~105 small white colonies scattered across the commercial DH5a plate with part 1
~135 small white colonies scattered across the commercial DH5a plate with part 2
~100 small white colonies scattered across the commercial DH5a plate with part 3
~5-7 small white colonies in a cluster on the homemade DH5a plate with part 1
~5-7 small white colonies in a cluster on the homemade DH5a plate with part 2
~5-7 small white colonies in a cluster on the homemade DH5a plate with part 3
Conclusions
21.5 ng of vector in ligation reactions seem to be enough to yield colonies scattered across the plates of competent cells
Transformation for DH5a (H=homemade, C=commercial)
Reagents
1 (H) (ul)
2 (H) (ul)
3 (H) (ul)
1 (C) (ul)
2 (C) (ul)
3 (C) (ul)
(+) (ul)
(-) (ul)
DH5a Competent Cells
40
40
40
40
40
40
40
40
pUC19
-
-
-
-
-
-
5
-
Ligated Vector (2 fold TE dilution)
5
5
5
5
5
5
-
-
Total
45
45
45
45
45
45
45
40
Transformation for BL21 (H=homemade, C=commercial)
Reagents
1 (H) (ul)
2 (H) (ul)
3 (H) (ul)
1 (C) (ul)
2 (C) (ul)
3 (C) (ul)
(+) (ul)
(-) (ul)
Bl21 Competent Cells
40
40
40
40
40
40
40
40
23 mM B-ME
1
1
1
1
1
1
1
1
pUC19
-
-
-
-
-
-
5
-
Ligated Vector (2 fold TE dilution)
5
5
5
5
5
5
-
-
Total
46
46
46
46
46
46
46
41
JUL 23 - JUL 27
Colony PCR and sequencing revealed that our transformation was unsuccessful and steps throughout the way may have introduced contaminants.
Highlights
Colony PCR of DH5α transformants with constructs 1, 2, 3
Sequencing of DH5a transformants reveal incorrect bacterial insert
Colony PCR of DH5α transformants with constructs 1, 2, 3
A colony PCR was run to verify that colonies contain correct insert sizes for constructs 1, 2, and 3. The first 8 colonies were run on a 1.5% agarose gel. Only one colony per construct seemed to contain an insert ~200-300 bp lower than the expected size. The gel appeared to contain aberrations that may have contributed to the unexpected result. Colonies containing our possible insert were grown overnight for sequencing the next morning.
Sequencing of DH5a transformants reveal incorrect bacterial insert
A protein BLAST search of the sequences received for the colonies grown reveal a bacterial insert. The top hits were Beta-lactamase and ATP synthase for E.coli. This suggests that our ligation reactions may have been contaminated with the original insert present in the empty vector which may not have been cleaned with our spin columns.
JUL 30 - AUG 3
Several attempts of PCRs were made to obtain amplicons corresponding to constructs 1, 2, and 3. However, we consistently received no yield of amplicon 1 despite previously having success with the same thermocycler parameters and reagent amounts. Throughout the week, we tested different parameters and finally saw better yield when using OneTaq GC Reaction Buffer and GC Enhancer. While these GC enhancements did produce amplicon 1, the production of amplicons 2 and 3 seemed less enhanced with these reagents. A successful transformation with construct 2 was also conducted and the colony was sequence verified by the end of the week.
Highlights
PCR with and without GC enhancements
First successfully transformed DH5a cells with construct 2
PCR with and without GC enhancements
The use of GC Buffer instead of regular Reaction buffer seemed to help produce stronger bands for our high GC content (55-60%) amplicons.
First successfully transformed DH5a cells with construct 2
We obtained 1 colony that contained the correct insert following Colony PCR and sequencing. This colony was then grown overnight and transformed into BL21.
AUG 6 - AUG 10
One N-terminal purification tagged construct is pushed forward and placed in BL21 for induction. Due to our unsuccessful transformations, we investigated possible reasons for our poor results. In addition, Gibson Assembly for the formation of our C-terminal tagged constructs is tested.
Highlights
Transformation and Induction of BL21 cells with construct 2
Optimizing heat shock for homemade competent cells
Protein purification of BL21 cells with construct 2
Gibson Assembly of C-terminal purification tag constructs
2-Step PCR of C-terminal purification tag constructs
Transformation and Induction of BL21 cells with construct 2
A sequence verified BL21 colony with our construct 2 insert was induced at 20C and 28C with 0.5 mM IPTG. At both temperatures, the 37 kDa protein was successfully expressed. However, at 28C, much more of our protein was produced. A large scale 100 mL induction at 28C was performed the next day for subsequent protein purification.
Optimizing heat shock for homemade competent cells
Three transformations of DH5a were conducted using the same amount of plasmid, but with varying heat shock durations at 20 seconds, 30 seconds, and 45 seconds. Only 2 colonies were present with the 45 second transformation, while 100+ colonies appeared on the 30 second and 40+ colonies appeared on the 20 second plate. These findings suggest that our transformations may have yielded so few colonies due to our use of the 45 second heat shock specified in our protocol.
Protein purification of BL21 cells with construct 2
Soluble and insoluble fractions of the induced cultures run on a SDS PAGE gel revealed that the protein was in the insoluble fraction. An insoluble purification with 700 mM NaCl and 30 mM imidazole washes was conducted and concentrated using the Pierce protein concentrator.
Gibson Assembly of C-terminal purification tag constructs
A Gibson Assembly of C-terminal purification tagged constructs was conducted for 15 minutes and 60 minutes with the positive control (NEBuilder) provided. The samples were run on a 1.5% agarose gel as the manual indicated that an efficient reaction will show assembled products of the correct size and disappearance of fragments. However, the larger of the two fragments that was supposed to be assembled was observed on the gel. The positive control did not show up on a gel, however, the reaction consisted of all commercially obtained reagents. We decided to contact NEB and inquire about our results next week.
2-Step PCR of C-terminal purification tag constructs
A 2-step PCR was conducted to add on the C-terminal TEV-8xHis tag. Amplicons were extracted from a low melt gel, digested, cleaned, then run on another agarose gel to confirm the amount of DNA present in each extract.
AUG 13 - AUG 17
Additional modifications were made to our protocols to increase our transformation yield. Our lack of results after running the Gibson Assembly sample on an agarose gel lead us to contact the supplier.
Highlights
New findings about Gibson Assembly after speaking with NEB
Gibson Assembly confirmation through transformation into DH5a
Optimizing ligation and transformation
New findings about Gibson Assembly after speaking with NEB
After speaking with a product manager at NEB, we found that the positive control provided for Gibson Assembly would not necessarily show up if run on a gel. Instead, we would have to transform it to confirm the success of our assembly.
Gibson Assembly confirmation through transformation into DH5a
A transformation was conducted with the Gibson samples, however, no colonies were observed on all plates. This may be due to the transformation protocol itself rather than the Gibson Assembly. Another transformation using modified transformation procedures below was conducted, yet still yielded no colonies.
Optimizing ligation and transformation
Throughout the week we faced difficulties in obtaining colonies when transforming into our homemade DH5a competent cells, even after optimization of the heat shock duration. We tried the following new techniques which seemed to produce more colonies.
Ligation
Run insert samples on an agarose gel to estimate DNA amount instead of using Nanodrop Quantification
Conduct a "melting ice" ligation
Transformation
Thaw competent cells on ice
Use 1 ul of 23 mM beta-mercaptoethanol
30 second heat shock instead of 45 seconds
AUG 20 - AUG 24
The optimized protocols allowed us to transform all N-terminal purification tagged constructs. However, efforts were still being made to transform the C-terminal purification tagged constructs. Our campus is closed from Thursday through the weekend due to the path of Hurricane Lane which delays our lab efforts.
Highlights
Continued efforts to transform C-terminal purification tagged constructs
Transformation of all N-terminal purification tagged constructs into BL21
Hurricane Lane delays lab efforts
Continued efforts to transform C-terminal purification tagged constructs
PCR, ligation, and transformation with quantification of DNA on an agarose gel was continued throughout the week. No colonies are observed on any of our plates, indicating that the transformation protocol was conducted incorrectly, ligation was unsuccessful, the PCR produced insufficient amplicons, or the digestion did not work. We previously verified that our RE digest indeed cuts correctly. Ligation and transformation were previously optimized and produced colonies on other constructs. This led us to investigate the PCR itself. The PCR for these constructs is conducted in two steps to add the C-terminal purification tag through the primer.
Transformation of all N-terminal purification tagged constructs into BL21
All N-terminal tagged constructs were sequence verified from DH5a plasmids, then transformed into BL21. Colony PCR revealed that correct insert sizes were present in a majority of colonies.
Hurricane Lane delays lab efforts
The onset of Hurricane Lane closed our campus facilities from Thursday through Sunday, pushing our induction and C-terminal transformation plans to the next week.
AUG 27 - AUG 31
Induction of all N-terminal purification tagged constructs at 28C were carried out for subsequent purification, dialysis and VLP assembly. In addition, PCR amplicons were sent to sequencing to verify that the correct ends and restriction sites were being formed.
Highlights
Induction of N-terminal purification tagged constructs at 28C
Confirming PCR amplicons of C-terminal tagged constructs
Induction of N-terminal purification tagged constructs at 28C
Induction at 28C for four hours produced proteins of the correct molecular weight. 50 mL cultures were spun down and pellets were stored at -80C until ready for purification.
Confirming PCR amplicons of C-terminal tagged constructs
Previous efforts to transform C-terminal purification tagged constructs into DH5a continued to be unfruitful. We decided to send the amplicons to sequencing and received poor results.
SEP 3 - SEP 7
The pellets from our induction of N-terminal purification tagged proteins were purified and stored at -80C in a general dialysis buffer. We determined which buffers we would try for VLP assembly and planned and prepared the materials we needed.
Highlights
N-terminal purification tagged protein purification and planning for VLP assembly
N-terminal purification tagged protein purification and planning for VLP assembly
We purified N-terminal purification tagged proteins and obtained nanodrop readings of 0.66 mg/ml - 1.44 mg/ml for all purified proteins. We will begin to assemble VLPs using these proteins by subjecting them to dialysis buffers of different compositions. Our assays will test for the effects of salt concentration and pH on VLP assembly and if VLP formation is affected by the different CA cut sites or by the addition of a few extra amino acids at the end of our proteins.
SEP 10 - SEP 14
Our first VLP assembly produces virus-like particles viewable under the EM. We attempt to assemble VLPs with the rest of our N-terminal purification tagged constructs. We had to perform another transformation of BBa_E1010 into DH5a and received an accurate sequencing result.
Highlights
VLP assembly and our first look at virus-like particles
Follow up VLP assembly assay
Cloning and sequencing of BBa_E1010 into DH5a
VLP assembly and our first look at virus-like particles
VLPs are viewed under the EM by Ryan on our full Gag construct. The particle diameter ranges from ~30 - 50 nm. Cleavage of the His tag appears to form more consistent, spherical, and closed particles. Meanwhile, particles with the His tag still attached seem to have more variability in size with some particles seemingly opened or deflated. Furthermore, we saw more VLPs under acidic assembly buffer conditions than basic conditions, suggesting that pH may play a role in VLP assembly.
Follow up VLP assembly assay
We decided to follow up on the last VLP experiment by observing the VLPs and intermediates at different points during assembly. We used our full Gag protein for this time assay. We also sought statistical data on VLP sizes to compare which was larger, tagged or untagged Gag. Unfortunately, we could not gather statistical data on VLP sizes as very little VLPs formed from this assembly. Our positive control, a VLP we assembled previously, did not produce as many particles as expected. This suggests that something may have gone wrong in the assembly process.
Cloning and sequencing of BBa_E1010 into DH5a
DH5a bacterial colonies that were transformed with BBa_E1010 during the summer were not preserved, thus we had to perform the transformation again and send the plasmid to sequencing. Sequencing confirmed that we had the correct RFP sequence with the prefix and suffix.
SEP 17 - SEP 21
Efforts to create the Gag-RFP and Gag biobrick are continued. Sequences were amplified for subsequent digestion and ligation into the biobrick (pSB1C3) and expression (pET14b) vectors.
Highlights
Amplification for expression and Biobrick creation
Amplification for expression and Biobrick creation
The Gag and RFP were amplified off of our Gag construct and BBa_E1010 sequence in two different ways. For ligation into the pSB1C3 vector for biobrick submittal, the Gag and RFP were amplified with primers consisting of the prefix and suffix. For ligation into the pET14b vector for expression, the Gag and RFP were amplified to incorporate the NcoI and BamHI restriction enzyme sites upstream of the Gag and downstream of the RFP.
SEP 24 - SEP 28
Amplicons for biobrick and expression were digested, cleaned, quantified, and ligated into their respective vectors.
Highlights
Transformation of DH5a with the Gag insert in pSB1C3
Transformation of DH5a with the Gag, RFP, and pSB1C3 / pET14b insert
Transformation of DH5a with the Gag insert in pSB1C3
The Gag with prefix and suffix primers was digested with EcoRI and PstI for a melting ice ligation with the digested pSB1C3 vector. Transformation of competent DH5a cells with the Gag insert in the biobrick submittal plasmid appeared successful after colony PCR and sequencing.
Transformation of DH5a with the Gag, RFP, and pSB1C3 / pET14b insert
The ligation with multiple inserts (Gag and RFP) was something we tried for the first time. We followed the PCR-after-ligation method (An et. al, 2010) and attempted to conduct a PCR on the ligation product with primers flanking our insert (VF2 and VR primers) to obtain the ligated Gag-RFP. The PCR did not show indicative bands of our ~2.1 kb insert. We thus switched gears and attempted to use our Gag ligated to pSB1C3 as the vector backbone.
OCT 1 - OCT 5
We digested the sequenced Gag in pSB1C3 with SpeI and PstI (RE sites downstream of the Gag) to ligate the RFP digested with XbaI and PstI. This would allow us to obtain the Gag-RFP biobrick and use it as template for amplifying the sequence with the correct RE sites for ligation into pET14b.
Highlights
Digestion and ligation of the RFP into pSB1C3-Gag
Preparation of biobricks for submittal to iGEM
Digestion and ligation of the RFP into pSB1C3-Gag
The ligation was conducted with a 3:1 insert to vector ratio. Ligation product diluted in TE buffer was used to transform competent DH5a cells. A colony PCR of the Gag-RFP with flanking insert primers (VF2 and VR) revealed a band corresponding to the approximately 2.1 kb insert.
Preparation of biobricks for submittal to iGEM
We begin to prepare the biobricks for submittal by performing plasmid extractions and following iGEM’s protocol for drying the samples. Unfortunately we did not have time to sequence the Gag-RFP part before submitting.
OCT 8 - OCT 12
The Gag-RFP in pSB1C3 was used as template for the amplification of the Gag-RFP with primers that would add the NcoI and BamHI RE sites for digestion and ligation into pET14b. However, odd sequencing results of our plasmids after transformation suggest we may have the wrong insert.
Highlights
Amplification of the Gag-RFP for ligation into pET14b
Odd sequencing results of pET14b containing the Gag-RFP suggest an incorrect insert
Expression of BL21 cells with large insert reveal no indicative bands on a SDS-PAGE gel
Amplification of the Gag-RFP for ligation into pET14b
The Gag-RFP was amplified with primers to add on the NcoI site upstream of the Gag and the BamHI site downstream of the RFP. The amplicons were gel extracted and digested using the NcoI and BamHI restriction enzymes. Following spin column cleanup, the Gag-RFP was ligated into digested pET14b in a 3:1 insert to vector ratio.
Odd sequencing results of pET14b containing the Gag-RFP suggest an incorrect insert
Transformation of DH5a and BL21 with the Gag-RFP insert in pET14b appeared to be successful after colony PCR. However, sequencing results of the plasmid with VF2 and VR primers reveal the beginning and end of the Gag. The Gag itself is ~1.3 kb, which does not match up with the insert size observed on the gel.
Expression of BL21 cells with large insert reveal no indicative bands on a SDS-PAGE gel
BL21 cells with the supposed Gag-RFP insert are expressed with 0.5 mM IPTG for 2 and 4 hours. No band differences are observed between the induced and uninduced cultures.
Oct 15 - Oct 19
The investigation continues to understand what the insert could possibly be, if not the Gag-RFP fusion protein. Sequencing is conducted with additional primers and steps are redone to try to transform BL21 and DH5a cells with the correct Gag-RFP insert.
Highlights
Sequencing with the VR and RFP reverse primer gives mixed results
Next steps to produce the Gag-RFP fusion protein
Sequencing with the VR and RFP reverse primer gives mixed results
Another ligation of the pSB1C3-Gag and the RFP was conducted. VF2 and VR primers were used to amplify the insert. This amplicon was sent to sequencing with the VF2, VR, and RFP reverse primer with the BamHI restriction site. The sequencing result from the VF2 revealed the beginning of the Gag as expected. However, the VR and RFP reverse primer gave different results. The RFP reverse primer revealed the end of the RFP as expected, while the VR primer, which also should have revealed the end of the RFP, displayed the end of the Gag. After looking to see if anyone had similar problems on iGEM’s website, we saw documentation that the VR may misprime on certain biobricks. We checked our Gag-RFP sequence against the VR primer and found 5 sites where the VR primer could possibly anneal with 9-12 base pairs complementarity out of its 20 base pairs.
Next steps to produce the Gag-RFP fusion protein
We continued through with this sequenced VF2 and VR amplicon and proceeded to digest and ligate it into pET14b. Colony PCR again revealed the rather large, ~2.1 kb insert. However, expression in BL21 cells did not display our protein of interest. We plan to backtrack and analyze at which step something could have gone wrong to clone our fusion protein by the time we present at the Giant Jamboree.
While the notebook is temporarily frozen due to the Wiki Freeze, our team will still be busy in the lab! Stay tuned for updates on our final Gag-RFP fusion protein.
CONTACT
uhigem@hawaii.edu
LOCATE
University of Hawaii at Manoa
1955 East West Road
Honolulu, HI 96822
NAVIGATE
Home Project Notebook
Parts Human Practices Safety
Collaboration Modelling Awards
