Team:Hawaii/Characterization
HOME
PROJECT
EXPERIMENTS
NOTEBOOK
PARTS
TEAM
MORE
SAFETY
COLLABORATIONS
HUMAN PRACTICES
MODELLING
DEMONSTRATE
ATTRIBUTIONS
CHARACTERIZATION
CALENDAR
PART CHARACTERIZATION
The 2018 Hawaii iGEM team performed an internal priming screening for all of the most frequently used BioBricks against the VF2 and VR primers flanking the insert in pSB1C3. BioBrick usage ranged from as little as 49 to as many as 4,572 uses. The various functionalities of these BioBricks include protein coding regions, promoters, ribosome binding sites, and terminators. As these BioBricks are frequently used and have a range of functionalities it is important to know if there is any internal priming using the VF2 and VR primers. The VF2 and VR primers are used during the sequence verification of BioBricks in the pSB1C3 vector. The purpose of this screening is to ensure that future iGEM teams are able to effectively and successfully sequence verify BioBricks they create or plan on using in their experimentation and are able to use the primers to amplify the part for downstream applications.
The 2018 Hawaii iGEM team experienced the issue of possible internal priming during the sequence verification process of our own BBa_K2574001 BioBrick and in the cloning process to express the part as a fusion protein. BBa_K2574001 is a composite part containing a VLP forming Gag protein sequence attached to a frequently used RFP part (BBa_E1010). We conducted a PCR amplification of the Gag-RFP insert using the VF2 and VR primers on the ligation product (pSB1C3 ligated to the Gag + RFP). This amplicon would serve as template for another PCR where we would add the NcoI and BamHI restriction enzyme sites through new primers for ligation into pET14b and subsequent induced expression. Despite gel confirming a rather large, approximately 2.1 kb insert band, our sequencing results with the VR primer and BamHI RFP reverse primer gave mixed results. Both should have displayed the end of the RFP, but the VR primer revealed the end of the Gag. Analysis of the VR primer on the Gag-RFP sequence revealed several sites where the VR primer could have annealed with ~9 - 12 bp of complementarity.  Internal priming of forward and reverse primers can be detrimental to an iGEM project because you can never be sure if the desired construct was correctly inserted into the BioBrick plasmid without a successful sequence verification.
We evaluated the 40 most frequently used BioBricks and ran them through an internal priming screening process that we developed below. Out of the 40 BioBricks we evaluated, 10 of them showed possible internal priming of either the VF2 or VR primers and sometime even both. Table 1 depicts the 10 frequently used BioBricks that show this possible internal priming. There are 3 frequently used BioBricks that show possible internal priming for both VF2 and VR primers. The data set has a range of sequence lengths from as small as 12 bases to as large as 1,210 bases. Figure 1 depicts the relationship between the length of BioBrick sequences and the number of BLAST hits a given BioBrick has for either VF2 or VR primers.
Table 1. A table depicting the internal priming screening analysis of BioBricks with primer VF2 (in blue) and VR (in green) [Note: Some BioBricks are seen in both colors indicating that there is internal priming for both VF2 and VR primers]. The alignment location refers to the location on both the primer and the BioBrick where there is a significant alignment [Note: Some BioBricks have multiple locations where the primer can align]. Another term used to describe a significant alignment is a hit. These hits are read from top to bottom, therefore the top is the primer (subject) and the bottom is the BioBrick (query). The bit score is a numerical value that describes the overall quality of an alignment and gives an indication of how good the alignment is; the higher the score, the better the alignment. In general terms, this score is calculated from a formula that takes into account the alignment of similar or identical residues, as well as any gaps introduced to align the sequences. The E-value is a parameter that describes the number of hits one can "expect" to see by chance when searching a database of a particular size so the lower the E-value, or the closer it is to zero, the more "significant" the match is. The strand description refers to the directionality of the alignment for that specific hit. The plus represents the 5’ to 3’ direction while the minus represents the 3’ to 5’ direction. For an example if it says Plus/Minus then that can be interpreted as the primer in the 5’ to 3’ direction aligns with the BioBrick in the 3’ to 5’ direction.
Figure 1. A graph depicting the relationship between the length of BioBrick sequences and the number of hits a given BioBrick has for both VF2 and VR primers. The Y-axis represents the individual number of BioBricks that have a certain number of hits (as indicated by the 5 different colors). The X-axis represents different ranges of sequence lengths found in the ‘Frequently Used’ data set. For an example, in the 0-20 sequence length range there are 8 BioBricks from the data set that have no hits (internal priming). As shown in the graph there are more BioBrick parts in this data set that have no internal priming than BioBrick parts that do. The smaller the BioBrick sequence the less of chance there is for internal priming and as you increase in sequence size you also increase the chances of internal priming.
INTERNAL PRIMING SCREENING PROTOCOL
Determine which BioBrick you wish to analyze.
Retrieve the FASTA formatted BioBrick sequence from the iGEM registry.
Retrieve the FASTA formatted primer sequence you wish to align with the BioBrick.
Access the Basic Local Alignment Search Tool (BLAST) program through the National Institute of Health (NIH).
Click on the desired database whether it be BLASTN for nucleotide sequences or BLASTP for protein sequences.
Indicate that you want to align two or more sequences.
Insert the desired primer sequence in the subject line.
Insert the desired BioBrick sequences in the query line. [Note: You are able to run multiple sequences in the query.]
Adjust the algorithm parameters based on the size of the queries. [Note: Keep in mind that the algorithm must be catered to either a large sequence (> 20 bases) or short sequences (< 20 bases).]
Click the BLAST button on the bottom of the page to run the alignment.
Determine whether or not there are any significant alignments. [Note: This can be adjusted in the algorithm parameters but for this screening we used a range of word sizes from 7 to 11, which is pretty significant given that the VF2 and VR primer sequences are fairly short.]
Determine where on the BioBrick sequence does internal priming occur.
CONTACT
uhigem@hawaii.edu
LOCATE
University of Hawaii at Manoa
1955 East West Road
Honolulu, HI 96822
NAVIGATE
Home Project Notebook
Parts Human Practices Safety
Collaboration Modelling Awards
