<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hawaii_Project_Meetings skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hawaii/Project Meetings</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#May"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">May</SPAN></A></LI><LI class="toclevel-2"><A href="#5.2F14.2F2018"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">5/14/2018</SPAN></A></LI><LI class="toclevel-2"><A href="#5.2F15.2F2018"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">5/15/2018</SPAN></A></LI><LI class="toclevel-2"><A href="#5.2F17.2F2018"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">5/17/2018</SPAN></A></LI><LI class="toclevel-2"><A href="#5.2F18.2F2018"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">5/18/2018</SPAN></A></LI><LI class="toclevel-2"><A href="#5.2F22.2F2018_.2B_5.2F23.2F2018"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">5/22/2018 + 5/23/2018</SPAN></A></LI><LI class="toclevel-2"><A href="#5.2F23.2F2018"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">5/23/2018</SPAN></A></LI><LI class="toclevel-2"><A href="#5.2F24.2F2018"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">5/24/2018</SPAN></A></LI></UL></DIV><H1><SPAN class="mw-headline" id="May">May</SPAN></H1><P> This week marked the start of our initial laboratory experiments. Plasmids were prepped and bacteria were transformed with CR2 constructs. Plasmids were also sent in for sequencing to verify that the constructs were valid. </P><H2><SPAN class="mw-headline" id="5.2F14.2F2018">5/14/2018</SPAN></H2><P> Restocked necessary lab materials such as bacteria media (LB), milli Q H2O, buffers/solutions.
    Completed By: Ryan </P><H2><SPAN class="mw-headline" id="5.2F15.2F2018">5/15/2018</SPAN></H2><P> Conducted a plasmid extraction of CR2 Gag-Prot-RT,CR2 integrase, and CR5 integrase and prepared/sent to sequencing lab.
    Completed By: Ryan </P><H2><SPAN class="mw-headline" id="5.2F17.2F2018">5/17/2018</SPAN></H2><P> Diagnostic restriction enzyme digest of CR2 Gag-Prot-RT construct to determine plasmid uptake for two colonies.
 
    Materials: 
<TABLE style="width:75%"><TBODY><TR><TH> RE Digest </TH><TH> ~ 1 Reaction </TH></TR><TR><TD> DNA </TD><TD> ~1 µg </TD></TR><TR><TD> 10x Cut Smart Buffer </TD><TD> 5 µL </TD></TR><TR><TD> H20 </TD><TD> to 48 µL </TD></TR><TR><TD> Nco1 </TD><TD> 1 µL </TD></TR><TR><TD> BamH1 </TD><TD> 1 µL </TD></TR><TR><TD> 50 µL total </TD></TR></TBODY></TABLE></P><P> Methods:
 <OL><LI> Add all reagents to a 1.5 mL Eppendorf tube
 </LI><LI> Mix by pipetting and one touch spin down
 </LI><LI> Incubate 1 hour at 37 degrees Celsius
 </LI><LI> Run 1.5% agarose gel in TAE for 30 min
 </LI></OL></P><P> Gag 1 and Gag 2 Pipette Scheme:
<TABLE style="width:75%"><TBODY><TR><TH> RE Digest </TH><TH> NO RE </TH><TH> Nco1 and BamH1 </TH></TR><TR><TD> DNA </TD><TD> 6 µL </TD><TD> 6 µL </TD></TR><TR><TD> 10x Cut Smart Buffer </TD><TD> 5 µL </TD><TD> 5 µL </TD></TR><TR><TD> H20 </TD><TD> 39 µL </TD><TD> 37 µL </TD></TR><TR><TD> Nco1 </TD><TD> --- </TD><TD> 1 µL </TD></TR><TR><TD> BamH1 </TD><TD> --- </TD><TD> 1 µL </TD></TR><TR><TD> 50 µL total </TD><TD> 50 µL total </TD></TR></TBODY></TABLE></P><P> Results: No bands were present when gag 1 and gag 2 were run on 1.5% agarose gels. May be due to the low concentration of DNA for both samples (500 ng). </P><P> Completed By: Emily, John, Gina, Ryan </P><H2><SPAN class="mw-headline" id="5.2F18.2F2018">5/18/2018</SPAN></H2><P><I> Extracted plasmid for subsequent transformation in BL21 from confirmed DH5-A colonies. </I></P><P><STRONG> Bacterial Lysate Prep </STRONG><OL><LI> Pellet 1-15 mL on culture by spinning at 4000 RPM for 20 minutes.
 </LI><LI> Discard supernatant.
 </LI><LI> Resuspend pellet with 250 µL P1 buffer (resuspension buffer) and gently vortex
 </LI><LI> Transfer resuspended bacteria into 1.5 mL Eppendorf tube.
 </LI><LI> Add 250 µL P2 buffer (lysis buffer) and invert 8 times
 </LI><LI> Add 350 µL N3 buffer (neutralization buffer) and invert 8 times.
 </LI><LI> Centrifuge for 10 min at 10,000 RPM.
 </LI></OL></P><P><STRONG> Column Purification </STRONG><OL><LI> Add supernatant to QIAprep column and spin for 60s at 10,000 RPM. Discard flow through.
 </LI><LI> Add 500 µL PB buffer (binding buffer) to column and spin for 60s at 10,000 RPM. Discard flow through.
 </LI><LI> Add 750 µL PE buffer (wash buffer) to column and spin for 60s at 10,000 RPM. Discard flow through.
 </LI><LI> Spin again for 60s at 10,000 RPM.
 </LI><LI> Place column in 1.5 mL centrifuge tube.
 </LI><LI> Add appropriate volume of H20 or TE (we used 7.5 µL EB buffer).
 </LI><LI> Let column stand for 60s then spin for 60s at 10,000 RPM.
 </LI><LI> Repeat step 6.
 </LI><LI> Repeat step 7.
 </LI></OL></P><P><STRONG> DNA Quantification w/ Nanodrop </STRONG><OL><LI> Clean Nanodrop w/ 2 µL EB buffer (used as solvent).
 </LI><LI> Blank with elution medium (EB buffer).
 </LI><LI> Measure 2 µL of sample.
 </LI><LI> Clean Nanodrop with 2 µL dH20.
 </LI><LI> Repeat steps as necessary for each sample.
 </LI></OL></P><P> Results: Obtained a DNA concentration of 140.4 ng/µL. Completed by: Emily, Gina, Ryan </P><P><I> Transformed BL21 competent cells with CR2 Gag-Prot-RT ligated vector. </I></P><P> Materials:
<TABLE style="width:75%"><TBODY><TR><TH> BL21 Transformation </TH><TH> Sample </TH><TH> (+) control </TH></TR><TR><TD> BL21 competent cells </TD><TD> 40 µL </TD><TD> 40 µL </TD></TR><TR><TD> 23 mM B-mercaptoethanol </TD><TD> 1 µL </TD><TD> 1 µL </TD></TR><TR><TD> pUC19 control </TD><TD> --- </TD><TD> 1 µL </TD></TR><TR><TD> Ligated vector (2 fold TE dilution) </TD><TD> To 50 ng </TD><TD> --- </TD></TR><TR><TD> Total Volume </TD><TD> 42-45 µL </TD><TD> 42 µL </TD></TR></TBODY></TABLE></P><P> *Ligated vector
50 ng = 1 µL/140.4 ng = 0.356 µL of ligated vector
 <OL><LI> Incubate on ice for 30 min.
 </LI><LI> Heatshock for 45 sec in 42 C water bath.
 </LI><LI> Incubate on ice for 2 min.
 </LI><LI> Add 250 µL SOC media preheated to 37 C.
 </LI><LI> Incubate for 1 hour at 37 C at 225 RPM.
 </LI><LI> Plate 50-100 µL of transformants.
 </LI><LI> Incubate at 37 C overnight.
 </LI></OL></P><P> Results: Two LB-Amp-Chloramph CR2 Gag-Prot-RT BL21 plates with colonies (1:1 and 1:1+) were obtained.
 Completed By: Emily, Gina, Ryan </P><H2><SPAN class="mw-headline" id="5.2F22.2F2018_.2B_5.2F23.2F2018">5/22/2018 + 5/23/2018</SPAN></H2><P> Determined presence or absence of DNA in plasmid construct and amplified CR2 Gag-Prot-Rt insert.</P><P>Materials/Methods:
 <TABLE style="width:75%"><TBODY><TR><TH> Colony PCR Reaction </TH><TH> 1 Reaction </TH><TH> 9 Reaction </TH></TR><TR><TD> Template </TD><TD> 11 µL </TD><TD> --- </TD></TR><TR><TD> 5X One TaqBuffer </TD><TD> 3 µL </TD><TD> 27 µL </TD></TR><TR><TD> dNTP mix </TD><TD> 0.3 µL </TD><TD> 2.7 µL </TD></TR><TR><TD> T7 Forward </TD><TD> 0.3 µL </TD><TD> 2.7 µL </TD></TR><TR><TD> T7 Reverse </TD><TD> 0.3 µL </TD><TD> 2.7 µL </TD></TR><TR><TD> OneTaq </TD><TD> 0.1 µL </TD><TD> 0.9 µL </TD></TR><TR><TD> Total Volume </TD><TD> 15 µL </TD><TD> 36 µL </TD></TR></TBODY></TABLE></P><P><OL><LI> Add 11 µL of PCR grade H20 to each PCR tube.
 </LI><LI> BL21 colonies are located in 4 C fridge. On the colony PCR plate, create a grid with 16 blocks.
 </LI><LI> Use a 10 µL pipette to dab at a single white colony (avoid colonies that seem blobbed together). If you think the tip touched another colony, dispose tip and do a new one. Dab 2-3 times on the cell in the colony PCR grid. Eject the tip into a PCR tube. Once you've done 8 (to fill 8 PCR tubes), continue to streak all 16 cells in the grid.
 </LI><LI> Prepare the PCR reaction master mix into an Eppendorf tube. Mix and centrifuge.
 </LI><LI> Remove the tips from the PCR tubes w/ a pipette and expel any liquid from the tip.
 </LI><LI> Add 4 µL of master mix to each PCR tube, mix by pipetting.
 </LI><LI> Spin down in mini-centrifuge (next to PCR machine)
 </LI></OL></P><P><TABLE style="width:75%"><TBODY><TR><TH> PCR Temperature </TH><TH> Time </TH></TR><TR><TD> 94 C </TD><TD> 3 min </TD></TR><TR><TD> 94 C, 35 cycles </TD><TD> 20s </TD></TR><TR><TD> 65 C, 35 cycles </TD><TD> 20s </TD></TR><TR><TD> 68 C, 35 cycles </TD><TD> 1 min 10s </TD></TR><TR><TD> 68 C </TD><TD> 5 min </TD></TR><TR><TD> 4 C </TD><TD> Store until ready to use </TD></TR></TBODY></TABLE></P><P><TABLE style="width:50%"><TBODY><TR><TH> Gel Parameters </TH></TR><TR><TD> 0.75% Agar </TD></TR><TR><TD> 50 mL TAE </TD></TR><TR><TD> Run @ 85V. ~30 min </TD></TR></TBODY></TABLE></P><P> *Note: We conducted another PCR reaction with bacteria that were left in the PCR tubes by adding 15 µL from a 120 µL master mix and running it with modified PCR parameters. </P><P><STRONG> Modified PCR Parameters </STRONG></P><P><TABLE style="width:75%"><TBODY><TR><TH> PCR Temperature </TH><TH> Time </TH></TR><TR><TD> 94 C </TD><TD> 3 min </TD></TR><TR><TD> 94 C, 35 cycles </TD><TD> 20s </TD></TR><TR><TD> 50 C, 35 cycles </TD><TD> 20s </TD></TR><TR><TD> 68 C, 35 cycles </TD><TD> 3 min </TD></TR><TR><TD> 68 C </TD><TD> 5 min </TD></TR><TR><TD> 4 C </TD><TD> store until use </TD></TR></TBODY></TABLE></P><P> Results: Two gel images retrieved, refer to gel images. Completed By: Emily, John, Jon, Shelby, Fernanda, Gina </P><H2><SPAN class="mw-headline" id="5.2F23.2F2018">5/23/2018</SPAN></H2><P> More bacteria were prepared with the insert for subsequent experiments. </P><P> Materials/Methods:
 *We prepared 2 tubes per colony and used colonies 3,4, and 7 from BL21 CR2 Gag-Prot-RT Colony PCR plate stored in the -4 C freezer. For each colony:
 <OL><LI> Fill one falcon tube with:
   </LI><LI>30 mL LB (Luria Broth)
   </LI><LI>30 µL Ampicillin (100 µg/mL conc.)
   </LI><LI>30 µL Chlorampheltcol (50 µg/mL conc.)
   </LI><LI> Mix thoroughly.
 </LI><LI> Pour 15 mL into the 2nd tube.
 </LI><LI> Inoculate by touching the respective colony with a 10 µL pipette tip and dipping (swirling into the falcon tube).
 </LI><LI> Place falcon tubes in the shaking incubator at 37 C, 225 RPM.
 </LI></OL></P><P> Results: We incubated overnight on 5/23 from ~ 2:25 pm Completed By: Fernanda, Gina </P><H2><SPAN class="mw-headline" id="5.2F24.2F2018">5/24/2018</SPAN></H2><P> Resuspended internal Gag-Prot-RT primers and prepared working solution from stock. </P><P> Materials/Methods:
<STRONG> Primer Resuspension </STRONG><OL><LI> Look on primer tube for XX nanomol number. Multiply this number by 10.
 </LI><LI> Add that much TE buffer in µL (if 39.6 nanomol, add 396 µL TE).
 </LI><LI> Briefly vortex.
 </LI><LI> Place on ice for 5 minutes.
 </LI><LI> Briefly vortex.
 </LI><LI> Store in -20 C (stock solution).
 </LI></OL></P><P><STRONG> Working Solution </STRONG></P><OL><LI> Take 10 µL stock solution.
 </LI><LI> Add 90 µL TE buffer.
 </LI><LI> Place on ice while working with it.
 </LI><LI> Store in -20 C afterwards.
 </LI></OL><P>Notes: We received 2 primers today (Gag 12 and Gag 14) that are internal primers to help sequence the middle section of our ~2.3kb insert. We only were able to sequence ~500 terminal nt of our Gag-Prot-RT construct so the middle primers will be used to sequence the middle portions. Based on Dr.Presting's analysis of the CR2 Gag-Prot-RT alignments in Ryan's folder, we determined that Gag 12 is the forward and Gag 14 is the reverse primer.
</P>
Completed By: Gina </DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>