<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tec-Monterrey_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tec-Monterrey/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM MTY 2015</TITLE></P><DIV class="research-pg"><HEADER><NAV class="navbar"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#menu-collapse" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="menu-collapse"><UL><LI><A href="https://2015.igem.org/Team:Tec-Monterrey">Main</A></LI><LI><A href="https://2015.igem.org/Team:Tec-Monterrey/Description">Research</A></LI><LI><A href="https://2015.igem.org/Team:Tec-Monterrey/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Tec-Monterrey/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Tec-Monterrey/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Tec-Monterrey/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Tec-Monterrey/Practices">Policies and Practices</A></LI><LI><A href="https://2015.igem.org/Team:Tec-Monterrey/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Tec-Monterrey/Achievements">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Tec-Monterrey/Attributions">Team</A></LI></UL></DIV></DIV></NAV><NAV class="submenu"><UL><LI><A data-scroll="" href="#background">Background</A></LI><LI><A data-scroll="" href="#objectives">Objectives</A></LI><LI><A data-scroll="" href="#Strategies">Strategies</A></LI><LI><A data-scroll="" href="#biobricks">BioBricks</A></LI><LI><A data-scroll="" href="#protocols">Protocols</A></LI><LI><A data-scroll="" href="#results">Results</A></LI><LI><A data-scroll="" href="#discussion">Discussion</A></LI><LI><A data-scroll="" href="#conclusions">Conclusions</A></LI><LI><A data-scroll="" href="#references">References</A></LI></UL></NAV></HEADER><DIV class="container"><SECTION class="main tr"><ARTICLE class="col-xs-12 col-sm-12 col-md-10 col-md-offset-1" id="background"><H1>Background</H1><P align="justify">In the past twenty years, the use of insect cells has grown enormously; the establishment of more than 500 insect cell lines from different organisms and tissues is proof of this (Smagghe et al, 2009). However, two cell lines are considered as
          the basic models, Sf9 cell from <I>Spodoptera frugiperda</I> and High Five™ cells (officially named BTI-TN-5B1-4) derived from <I>Trichopulsia</I> ni.
           Nowadays the most common way to work with the previous cell lines is by using the baculovirus expression vector systems (BEVS). This method consists in the use of a baculovirus, normally the multicapsid nuclear polyhedrosis virus Autographa
          california (AcMNPV), as a vector to infect the cells in order to produce protein. The gene of interest is introduced into the viral genome via homologous recombination with a transfer vector (Greene, 2004).</P></ARTICLE><ARTICLE class="col-xs-12 col-sm-12 col-md-10 col-md-offset-1"><H1>Objectives</H1><P align="justify">The aim of our project is to expand the use of insect cells in synthetic biology. We believe that we have the opportunity to enrich the part registry with biobricks designed for an insect chassis. To accomplish our goal, our team is going to introduce various biobricks that contain basic functional parts necessary to build your own expression cassette for insect cells. In addition, we want to validate the use of the CRISPR/Cas9 System in the Sf9 cells for specific genome editing. </P></ARTICLE><ARTICLE class="col-xs-12 col-sm-12 col-md-10 col-md-offset-1" id="Strategies"><H1>Strategies</H1><DIV class="row" style="text-align:center;"><H2>BEVS</H2><P align="justify">Characterization of the widely used polyhedrin promoter (<A href="http://parts.igem.org/Part:BBa_K173400">BBa_K1734000</A>) and the confirmation of two secretion signals (<A href="http://parts.igem.org/Part:BBa_K1734001">BBa_K1734001</A>, <A href="http://parts.igem.org/Part:BBa_K1734002">BBa_K1734002</A>). All the work was confirmed by using the reporter gene Nanoluc (<A href="http://parts.igem.org/Part:BBa_K1734004">BBa_K1734004</A>)</P></DIV><DIV class="row" style="text-align:center;"><H2>STABLE</H2><P align="justify">Random genome integration to generate a stable cell line mediated by zeocin antibiotic resistance by selective pressure. The promoter OpIE2 (<A href="http://parts.igem.org/Part:BBa_K1734001">BBa_K1734001</A>) was used to test protein production.</P></DIV><DIV class="row" style="text-align:center;"><H2>CRISP/Cas9</H2><P align="justify">To prove the function of the CRISPR/Cas9 in the Sf9, we developed two constructs of our gRNA (<A href="http://parts.igem.org/Part:BBa_K1734012">BBa_K1734012</A>, <A href="http://parts.igem.org/Part:BBa_K1734013">BBa_K1734013</A>) to attenuate the Nanoluc’s luminescence in the stable cell line. These gRNAs are in the same plasmid that produces the Cas9 protein and a GFP protein as a fluorescent marker, having two separate plasmids. We will work with both pathways of CRISPR: nonhomologous end joining and homology-directed repair.</P></DIV></ARTICLE><ARTICLE class="col-xs-12 col-sm-12 col-md-10 col-md-offset-1"><H1>Biobricks</H1><STRONG><H3>Polyhedrin promoter (pPH) <A href="http://parts.igem.org/Part:BBa_K1734000">(BBa_K1734000)</A></H3></STRONG><P align="justify">Natively, this promoter activates the transcription of the polyhedrin gene (polh), a major occlusion-body matrix protein expressed during the very late phase of infection.</P><STRONG><H3>OpIE2 promoter <A href="http://parts.igem.org/Part:BBa_K1734001">(BBa_K1734001)</A></H3></STRONG><P align="justify">The immediate-early 2 promoter from the multicapsid nucleopolyhedrosis virus Orgyia pseudotsugata (OpIE2) is a constitutive promoter. Usually is used in nonlytic gene expression systems, but can be used with the baculovirus expression vector system (BEVS) methodology as well.</P><STRONG><H3>Polyhedrin promoter + Mellitin’s signal Peptide <A href="http://parts.igem.org/Part:BBa_K1734002">(BBa_K1734002)</A></H3></STRONG><P align="justify">The biobrick contains the polyhedrin promoter from the baculovirus Autographa californica. After the promoter there is a secretion signal sequence of the protein Mellitin from the organism Abis mellifera (Honeybee).</P><STRONG><H3>Polyhedrin promoter + Lysozyme’s signal peptide <A href="http://parts.igem.org/Part:BBa_K1734003">(BBa_K1734003)</A></H3></STRONG><P align="justify">The biobrick contains the polyhedrin promoter from the baculovirus Autographa californica. After the promoter there is a secretion signal sequence of the protein lysozyme from the organism Gallus gallus (chicken).</P><STRONG><H3>Nanoluc (Codon optimized for Sf9 cells) <A href="http://parts.igem.org/Part:BBa_K1734004">(BBa_K1734004)</A></H3></STRONG><P align="justify">Nluc luciferase is an enzyme that was engineered to work as a luminescent reporter weighing 19.1 kDa. This protein uses fumarizine as novel substrate to produce a high intensity glow-type luminescence. These reactions are ATP-independent, trying to obtain the maximal assay sensitivity.</P><STRONG><H3>6xHis-T24-Nanoluc <A href="http://parts.igem.org/Part:BBa_K1734006">(BBa_K1734006)</A></H3></STRONG><P align="justify">This part encodes a sequence capable to add a purification tag (6X His) at the 3’ terminus of a desired protein. Furthermore, by using the reporter gene (nanoluc) it’s possible to quantify indirectly the protein concentration. The T2A sequence cleaves the protein, producing the protein of interest tagged with HisTag and nanoluc separately. This part has been codon optimized for Spodoptera frugiperda (Sf9).</P></ARTICLE><ARTICLE class="col-xs-12 col-sm-12 col-md-10 col-md-offset-1"><H2>Protocols</H2><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
                  Protocol 1: Calcium Competent Cells
                </A></H4></DIV><DIV id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body"><H5>Objective</H5><P>To obtain a resilient bacteria against a transformation procedure in order to insert a plasmid on it.</P><H5>Approximate Time:</H5><P>1680 min = 28 hours</P><H5>Material</H5><UL><LI>0.250 L LB </LI><LI>0.5 L CaCl2 0.1M </LI><LI>6 centrifuge tubes</LI><LI>2 Erlenmeyer flasks</LI><LI>Lots of microfuge tubes</LI><LI>50 ml glycerol</LI></UL><H5>Equipment and Apparatus:</H5><P>The day prior to this protocol, things to were:</P><OL><LI>Autoclave:
                    </LI><LI>0.250 L LB medium </LI><LI>0.5 L CaCl2 0.1M</LI><LI>6 centrifuge tubes </LI><LI>2 Erlenmeyer flasks of 250 ml capacity</LI><LI>Microfuge flask</LI><LI>50 ml glycerol </LI><LI>Tips of 1 ml</LI><LI>Tips of 200 l </LI><LI>PCR Tubes</LI><LI>For the step 1 in this protocol:
                    </LI><LI>Put 20 ml LB medium in 2 sterile centrifuge tubes. </LI><LI>Inoculate using 10 ul of E. coli stock or take a single colony from an LB plate. </LI><LI>Gtr culture overnight at 37°C in a shaker. </LI></OL><LI>Store at 4°C CaCl2 solution, tips for micropipette and PCR tubes. </LI><H4>Related Protocols:</H4><H5>Transformation:</H5><OL><LI>Incubate a pre-inoculum in 2 centrifuge tubes using 20 ml of LB medium in each tube overnight at 37°C on a shaker at 250 rpm. </LI><LI>Inoculate 200ml of LB medium. Measure the OD600 to get 0.1.</LI><LI>Incubate at 37°C, 250 rpm until OD600 gets between 0.4 and 0.6 (Measure the OD600 every hour, this step takes 1.5 to 2 hours).</LI><LI>Split 200 ml culture in 4 centrifuge tubes, pouring 50 ml in each one of them. Label properly every tube. </LI><LI>Starting from this step for the remainder of the procedure, it is totally important to keep cells at 4°C. The cells, and any bottles or solutions that they come in contact with, must be pre-chilled to 4°C.</LI><LI>Keep these tubes with cells in suspension in ice for 30 minutes.</LI><LI>Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C. </LI><LI>Decant the supernatant and gently resuspend the pellet in 1 ml of ice cold CaCl2. Once dissolved, fill every tube reaching a 20 ml final volume with ice cold 0.1M CaCl2. </LI><LI>Keep these tubes with cells in suspension in ice for 20 minutes. </LI><LI>Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C. </LI><LI>Decant the supernatant and gently resuspend the pellet in 1 ml of ice cold CaCl2. Once dissolved, fill every tube reaching a 20 ml final volume with ice cold 0.1M CaCl2. </LI><LI>Keep these tubes with cells in suspension in ice for 15 minutes. </LI><LI>Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C. </LI><LI>Decant the supernatant and gently resuspend the pellet in 425 l of ice cold CaCl2 and 75l of glycerol. </LI><LI>Aliquot 70 l into sterile PCR tubes. Store frozen cells in the -80°C freezer. </LI></OL><H5>Waste Disposal:</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD><TD>What to avoid</TD></TR><TR><TD>
                      Supernatant
                    </TD><TD>Sink after adding bleach</TD></TR><TR><TD>Pipette tips </TD><TD>Trash after autoclave sterilization</TD></TR></TBODY></TABLE><H5>Safety notes</H5><UL><LI>Do all this procedure in a hood.</LI><LI>Wear gloves.</LI><LI>Wear lab coat.</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
                  Protocol 2: Calcium Competent Cells
                </A></H4></DIV><DIV id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><DIV class="panel-body"><H5>Objective:</H5><P>To obtain a resilient bacteria against a transformation procedure, in order to insert a plasmid into it</P><H5>Approximate Time:</H5><P>1680 min = 28 hours</P><H5>Materials:</H5><UL><LI>0.250 L label</LI><LI>0.5 L CaCl2 0.1M</LI><LI>6 centrifuge tubes </LI><LI>2 Erlenmeyer flask </LI><LI>Lots of microfuge tubes </LI><LI>50 ml glycerol</LI></UL><H5>Equipment and apparatus</H5><UL><LI>Spectrophotometer</LI><LI>Centrifuge</LI><LI>-80º C freezer</LI></UL><H5>Previous steps:</H5><P>The day prior to this protocol, things to do were:</P><OL><LI><P>Autoclave:</P></LI><LI>0.250 L LB medium </LI><LI>0.5 L CaCl2 0.1M </LI><LI>6 centrifuge tubes </LI><LI>2 Erlenmeyer flask 250 ml</LI><LI>Microfuge flask </LI><LI>50 ml glycerol </LI><LI>Tips of 1 ml </LI><LI>Tips of 200 l </LI><LI>PCR Tubes</LI><LI>For step 1 in this protocol:
                    </LI><LI>Put in 2 sterile centrifuge tubes 20 ml LB medium in each tube. </LI><LI>Inoculate using 10 ul of E. coli stock or take a single colony from LB plate.</LI><LI>Grow culture at 37°C in shaker overnight.</LI></OL><LI>Store at 4°C CaCl2 solution, tips for micropipette and PCR tubes. </LI><H4>Related protocols:</H4><H5>Transformation:</H5><OL><LI>Incubate a pre-inoculum 2 centrifuge tubes using 20 ml of LB medium in each tube overnight at 37°C on a shaker at 250 rpm.</LI><LI>Inoculate 200ml of LB medium. Measure the OD600 to get 0.1.</LI><LI>Incubate at 37°C, 250 rpm until OD600 gets between 0.4 and 0.6 (Measure the OD600 every hour, this step takes 1.5 to 2 hours).</LI><LI>Split 200 ml culture in 4 centrifuge tubes, pouring 50 ml in each one of them. Label properly every tube. </LI><LI>Starting from this step for the remainder of the procedure, it is totally important to keep cells at 4°C. The cells, and any bottles or solutions that they come in contact with, must be pre-chilled to 4°C.</LI><LI>Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C. </LI><LI>Decant the supernatant and gently resuspend the pellet in 1 ml of ice cold CaCl2. Once dissolved, fill every tube reaching a 20 ml final volume with ice cold 0.1M CaCl2. </LI><LI>Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C.</LI><LI>Decant the supernatant and gently resuspend the pellet in 1 ml of ice cold CaCl2. Once dissolved, fill every tube reaching a 20 ml final volume with ice cold 0.1M CaCl2. </LI><LI>Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C.</LI><LI>Decant the supernatant and gently resuspend the pellet in 425 l of ice cold CaCl2 and 75l of glycerol. </LI><LI>Aliquot 70 l into sterile PCR tubes. Store frozen cells in the -80°C freezer. </LI></OL><H5>Waste disposal:</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD><TD>What to avoid</TD></TR><TR><TD>Supernatant</TD><TD>Sink after adding bleach</TD></TR><TR><TD>Pipette tips</TD><TD>Trash after auclave sterilization</TD></TR></TBODY></TABLE><H5>Safety notes</H5><UL><LI>Do all this procedure in a hood</LI><LI>Wear gloves</LI><LI>Wear lab coat</LI></UL></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingThree"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree">
                  Protocol 3: Digestion with restriction enzymes
                </A></H4></DIV><DIV id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree"><DIV class="panel-body"><H5>Objective:</H5><P>Cut DNA fragments for ligation.</P><P>Analyze DNA sanmples through the use of electrophoresis</P><H5>Approximate time:</H5><P>120 min</P><H5>Materials:</H5><UL><LI>Nuclease free distilled water</LI><LI>PCR Tubes</LI><LI>Digestion buffer (NEB)</LI><LI>Restriction enzymes (NEB)</LI><LI>Pipettes and tips</LI><LI>DNA sample</LI></UL><H5>Equipment and Apparatus</H5><UL><LI>Thermocycler (BioRad)</LI><LI>Nanodrop (Thermo)</LI></UL><H5>Previous steps:</H5><P>Measure the concentration of DNA in the sample using the Nanodrop.</P><P>Depending on the concentration of DNA in the sample and the volume of the reaction, make the calculation for the required volumes of the components. Normally, 1 unit of enzyme is put per µg of DNA; however, the provider recommends using
                  10 units per µg of DNA to be digested. Our typical reaction was of 20 µL </P><TABLE class="table"><TBODY><TR><TD>Reaction volume</TD><TD>Resctriction enzyme</TD><TD>DNA</TD><TD>Buffer</TD></TR><TR><TD>10 μL</TD><TD>1 unit</TD><TD>0.1 µg</TD><TD>1 µL </TD></TR><TR><TD>25 μL</TD><TD>5 units</TD><TD>0.5 µg </TD><TD>2.5 μL</TD></TR><TR><TD>50 μL</TD><TD>10 units</TD><TD>1 µg</TD><TD>5 μL</TD></TR></TBODY></TABLE><H4>Related protocols:</H4><P>Gel electrophoresis, miniprep, ligation</P><H5>Steps:</H5><OL><LI>After labeling the PCR tubes, pour with the micropipette the nuclease free distilled water.</LI><LI>Pour the buffer to a concentration of 1X.</LI><LI>Introduce the DNA sample to the PCR tube to the required volume.</LI><LI>Pour the enzymes.
                    <STRONG>The enzymes must be introduced lastly to the reaction.</STRONG> Handle them carefully, as they are temperature sensitive
                    <STRONG>(handle with ice and return to refrigerator as soon as possible).</STRONG></LI><LI>Program thermocycler at 37°C 1 hour for the digestion and 80°C for 20 minutes to inactivate the enzymes.</LI><LI>Analyze digestion results by agarose electrophoresis. </LI></OL><H5>Waste disposal:</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD><TD>What to avoid</TD></TR><TR><TD>Digestion waste in tube</TD><TD>Place in trash container</TD></TR></TBODY></TABLE><H5>Safety notes:</H5><UL><LI>In order to minimize the possibility of nuclease action, gloves must be worn at all times.</LI><LI>The enzymes must be handled with care.</LI></UL><H5>References:</H5><P>https://www.neb.com/protocols/2014/05/07/double-digest-protocol-with-standard-restriction-enzymes </P><P>Digestion Protocol for BioBrick Assembly Kit (E0546) https://www.neb.com/protocols/1/01/01/digestion-protocol-e0546 </P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingFour"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour">
                  Protocol 4: Agarose based electrophoresis
                </A></H4></DIV><DIV id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour"><DIV class="panel-body"><H5>Objective:</H5><P>Analyze DNA fragments based on their movement in a uniform electric field.</P><H5>Approximate time</H5><P>120 min</P><H5>Materials:</H5><UL><LI>Agarose (BioRad)</LI><LI>TAE Buffer (Promega)</LI><LI>Loading dye (Promega)</LI><LI>10 kb and 1 kb weight stair (Promega)</LI><LI>Ethidium bromide (BioRad) </LI><LI>Pipettes and tips </LI><LI>DNA sample</LI></UL><H5>Equipment and Apparatus</H5><UL><LI>Electrophoresis chamber (Fisher Scientific) </LI><LI>Voltage source (BioRad)</LI><LI>Transiluminator (Thermo) </LI></UL><H5>Previous steps:</H5><UL><LI>Digestion: Usually this protocol precedes the electrophoresis </LI><LI>Check if there is TAE available. A stock 50X can be made from:
                    </LI><LI>Tris-base: 242 g</LI><LI>Acetate (100% acetic acid): 57.1 ml</LI><LI>EDTA: 100 ml 0.5M sodium EDTA, </LI><LI>Add dH2O up to one litre.</LI></UL><H5>Related protocols:</H5><P>Digestion, enzyme restriction analysis</P><H5>Steps:</H5><OL><LI>The amount of agarose depends on the size of the DNA. (A guide can be seen in the table below)
                    <TABLE class="table"><TBODY><TR><TD><STRONG>Agarose Concentration (g/100mL)</STRONG></TD><TD><STRONG>Optimal DNA Resolution (kb)</STRONG></TD></TR><TR><TD>0.5 </TD><TD>1 - 30</TD></TR><TR><TD>0.7</TD><TD>0.8 - 12 </TD></TR><TR><TD>1.0</TD><TD>0.5 - 10</TD></TR><TR><TD>1.2</TD><TD>0.4 - 7</TD></TR><TR><TD>1.5 </TD><TD>0.2 - 3</TD></TR></TBODY></TABLE></LI><LI>Measure out the appropriate mass of agarose into a beaker with the appropriate volume of buffer. The volume in the lab gel box is 50mL.</LI><LI>Microwave until the agarose is fully melted and cool down a few minutes. (Microwave little by little because the solution might rise out of the beaker) </LI><LI>Pour the agarose solution into the gel box. Carefully pop or shove to the side any bubbles (with the help of a pipette tip), put the comb in place, and let it cool for about 15-30 minutes until the gel solidifies (cover with foil to
                    avoid contamination). </LI><LI>Mix the DNA samples with 5X loading dye solution to make it reach 1X. (Except for 1 µL of sample you can use 1µL of Loading dye) </LI><LI>Pour the mixture in the wells.
                    <STRONG> Be careful with the bubbles and with breaking the gel with the pipette.</STRONG></LI><LI>Fill the chamber with TAE (about 300-450 mL) and run the gel 70 min at 90 V and ~180 mA </LI><LI>Stain the gel soaking it in EtBr, be careful not to break it. </LI><LI>Reveal using the trans illuminator, first focus using the white light lamp and then use the UV to make the bands visible.</LI></OL><H5>Waste disposal</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD><TD>What to avoid</TD></TR><TR><TD>TAE</TD><TD>Throw through the sink, the school has a water treatment plant for all the facilities.</TD><TD>NA</TD></TR><TR><TD>Gel</TD><TD>The gel has EtBr, which is very mutagenic, gloves must be used to handle the gel when you soak it in the solution. The gel must be disposed in the EtBr container.</TD><TD>Do not touch the gel or the EtBr solution with your bare hands.</TD></TR><TR><TD>EtBr solution</TD><TD>Every once in a while the solution must be changed, and the discarded solution must be?</TD><TD>See above.</TD></TR><TR><TD>Gloves</TD><TD>You must discard the glove you use when you handle the gel and the computer. They must be thrown in the EtBr labeled container.</TD><TD>Make sure to dispose your gloves</TD></TR></TBODY></TABLE><H5>Safety notes:</H5><UL><LI>Remember to use the EtBr at the delimited area in the lab. </LI><LI>Dispose your gloves after handling the gel.</LI><LI>If you accidentally touch a surface of EtBr designed area, remember to soak yourself with ethanol 70% to inactivate the EtBr. Seriously,
                    <STRONG>this shouldn’t happen.</STRONG></LI></UL><H5>References:</H5><P>Agarose Gel Electrophoresis http://www.addgene.org/plasmid-protocols/gel-electrophoresis/</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingFive"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseFive" aria-expanded="false" aria-controls="collapseFive">
                  Protocol 5: Bacmid Miniprep
                </A></H4></DIV><DIV id="collapseFive" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFive"><DIV class="panel-body"><H5>Objective:</H5><P>Isolate recombinant bacmid DNA from E. coli culture (DH10Bac)</P><H5>Approximate time:</H5><P>100-120 min</P><H5>Materials:</H5><UL><LI>Eppendorf tubes</LI><LI>Solution 1 (15 mM Tris-HCl (pH: 8), 10mM EDTA (pH 8)) (Filtered)</LI><LI>Solution 2 (0.2 M NaOH, 1% w/v SDS) (Filtered)</LI><LI>Solution 3 (Sodium acetate 3M (pH 5.5)) (Sterile)</LI><LI>Isopropanol</LI><LI>EtOh 70%</LI><LI>TE 1X</LI></UL><H5>Equipment &amp; Apparatus</H5><UL><LI>Centrifuge</LI><LI>Safety cabinet</LI></UL><H5>Previous steps:</H5><UL><LI>Transformation for recombinant bacteria.</LI><LI>Prepare starter culture cells.</LI></UL><H5>Related protocols:</H5><UL><LI>Transfection for insect cells.</LI></UL><H5>Steps:</H5><OL><LI>Centrifuge starter culture at 3000 g for 5 min.</LI><LI>Decant supernatant and resuspend pellet in 1 mL of LB medium.</LI><LI>Pass cells to Eppendorf tube.</LI><LI>Centrifuge at 14000 rpm for 1 min. </LI><LI>Decant supernatant.</LI><LI>Resuspend (pippeting up and down) pellet in 300 μL of Solution 1. </LI><LI>Add 0.5 μl of RNAse to each tube</LI><LI>Add 300 μL of Solution II.</LI><LI>Invert tube carefully 5 times.</LI><LI>Incubate at room temperature for 5 min.</LI><LI>Add 300 µL of Solution III. </LI><LI>Mix by inverting several times.</LI><LI>Place the sample on ice for 10 min.</LI><LI>Centrifuge tubes at 14000 rpm for 10 min. Meanwhile, label another eppendorf tube and add 800 µL of isopropanol to it.</LI><LI>Collect supernatant in the new tube with isopropanol (around 700 µL). </LI><LI>Mix by gently inverting tube a few times and place on ice for 5 to 10 min.</LI><LI>Centrifuge the sample for 15 min at 14,000 g at room temperature. </LI><LI>Remove the supernatant and add 0.5 µl 70% ethanol to each tube.</LI><LI>Invert the tube several times to wash the pellet.</LI><LI>Centrifuge for 5 min at 14,000 g at room temperature.</LI><LI>Remove as much of the supernatant as possible.</LI><LI>Air dry the pellet briefly, 20 min.</LI><LI>Dissolve the DNA in 40 µl of TE. </LI><LI>Store DNA at -20°C.</LI></OL><H5>Waste disposal</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD><TD>What to avoid</TD></TR><TR><TD>Supernatant</TD><TD>Sink after adding bleach.</TD></TR><TR><TD>Pipet tips</TD><TD>Trash after adding bleach.</TD></TR><TR><TD>Eppendorf tubes</TD><TD>Trash after adding bleach.</TD></TR></TBODY></TABLE><H5>Safety notes:</H5><OL><LI>Every step must be done in the safety cabinet to avoid contamination.</LI><LI>Gloves must be worn at all times.</LI><LI>Use lab coat.</LI></OL><H5>References</H5><P>Made by: Carolina Elizondo</P><P>Date: 29/08/15</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingSix"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseSix" aria-expanded="false" aria-controls="collapseSix">
                Protocol 6: Miniprep
              </A></H4></DIV><DIV id="collapseSix" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingSix"><DIV class="panel-body"><H5>Objective:</H5><P>Isolate plasmid from E. coli culture</P><H5>Approximate time:</H5><P>180-240 min</P><H5>Materials:</H5><UL><LI>Eppendorf tubes</LI><LI>Solution 1 (50 mM Glucose, 25 mM Tris-HCl (pH: 8), 10mM EDTA (pH 8)</LI><LI>Solution 2 (0.2 M NaOH, 1% w/v SDS) </LI><LI>Solution 3 (Sodium acetate 3M, acetic acid 2M) </LI><LI>EtOH 100%</LI><LI>EtOH 70%</LI><LI>DNAse free water</LI></UL><H5>Equipment and apparatus:</H5><UL><LI>Centrifuge</LI></UL><H5>Previous steps:</H5><UL><LI>Transformation for recombinant bacteria. </LI><LI>Prepare starter culture cells.</LI></UL><H5>Related Protocols:</H5><UL><LI>Transformation for recombinant bacteria</LI><LI>Digestion</LI><LI>Electrophoresis</LI></UL><H5>Steps:</H5><OL><LI>Centrifuge starter culture at 3000 g for 5 min.</LI><LI>Decant supernatant and resuspend pellet in 1 mL of LB medium.</LI><LI>Transfer cells to Eppendorf tube. </LI><LI>Centrifuge at 14000 rpm for 2 min.</LI><LI>Decant supernatant.</LI><LI>Resuspend pellet in 100 µL of Solution 1.</LI><LI>Vortex solution for 2 min. until all bacteria are fully resuspended.</LI><LI>Add 200 µL of Solution II.</LI><LI>Invert tube carefully 5 times. </LI><LI>Incubate on ice for 5 min. </LI><LI>Add 150 µL of Solution III.</LI><LI>Mix by inverting several times. </LI><LI>Incubate on ice for 5 min.</LI><LI>Centrifuge tubes at 14000 rpm for 5 min.</LI><LI>Collect supernatant in new tube (around 400 µL). </LI><LI>Add 0.5 µL of 20 mg/mL RNase A to supernatant in new tube. </LI><LI>Incubate at 37°C for 5 min.</LI><LI>Add 1 mL of EtOh 100%.</LI><LI>Incubate at -80°C for 30 min or at -20°C for 1 hour.</LI><LI>Centrifuge solution at 14000 rpm for 15 min. (If possible at 4°C).</LI><LI>Pour out supernatant. </LI><LI>Open and invert tubes on a paper towel to drain them out.</LI><LI>Add 500 µL of EtOH 70%. </LI><LI>Centrifuge solution at 14000 rpm for 5 min.</LI><LI>Pour supernatant in sink. </LI><LI>Dry by inverting over paper towel 5-20 min. (Until pellet is completely dry). </LI><LI>Resuspend DNA with 20-40 µL of DNAse free water.</LI><LI>Measure concentrations at NanoDrop. </LI><LI>Store DNA at 4°C.</LI></OL><H5>Waste disposal:</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD><TD>What to avoid</TD></TR><TR><TD>Supernatant</TD><TD>Sink after adding bleach.</TD></TR><TR><TD>Pipette tips </TD><TD>Trash after adding bleach.</TD></TR><TR><TD>Eppendorf tubes</TD><TD>Trash after adding bleach.</TD></TR></TBODY></TABLE><H5>Safety notes:</H5><UL><LI>Wear gloves.</LI><LI>Use lab coat.</LI></UL><H5>References:</H5><P>Made by: Ana Karen Tello</P><P>Date: 29/08/15</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingSeven"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseSeven" aria-expanded="false" aria-controls="collapseSeven">
                    Protocol 7: Miniprep
                </A></H4></DIV><DIV id="collapseNumber" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingSevenr"><DIV class="panel-body"><H5>Objective:</H5><P>Cheap DNA extraction for E. coli </P><H5>Approximate time</H5><P>180 - 240 min</P><H5>Materials:</H5><UL><LI>Eppendorf tubes</LI><LI>Solution 1 (Tris-HCl 0.1181 g, Cilantro extract 130 µL, distilled water 29.87 µL)</LI><LI>Solution 2 (NaOH 24 g, Axion detergent 300 µL, distilled water 29.7 µL) </LI><LI>Solution 3 (Sodium bicarbonate 5g, Lemon 8 mL)</LI><LI>EtOH 100%</LI><LI>EtOH 70%</LI><LI>DNAse free water</LI></UL><H5>Equipment and apparatus</H5><UL><LI>Centrifuge</LI></UL><H5>Previous steps</H5><UL><LI>Transformation for recombinant bacteria.</LI><LI>Prepare starter culture cells.</LI></UL><H5>Related Protocols:</H5><UL><LI>Transformation for recombinant bacteria</LI><LI>Digestion</LI><LI>Electrophoresis</LI></UL><H5>Steps</H5><OL><LI>Centrifuge starter culture at 3000 g for 5 min. </LI><LI>Decant supernatant and resuspend pellet in 1 mL of LB medium. </LI><LI>Pass cells to Eppendorf tube.</LI><LI>Centrifuge at 14000 rpm for 1 min.</LI><LI>Decant supernatant.</LI><LI>Add 200 µL of Solution 1 </LI><LI>Vortex solution for 2 min. until all bacteria are fully resuspended. </LI><LI>Add 200 µL of Solution II.</LI><LI>Invert tube carefully 5 times.</LI><LI>Incubate at 37°C for 5 min.</LI><LI>Add 200 µL of Solution III.</LI><LI>Mix by inverting several times.</LI><LI>Incubate at 37°C for 5 min.  </LI><LI>Centrifuge tubes at 14000 rpm for 10 min. </LI><LI>Collect supernatant in new tube (around 500 µL).</LI><LI>Incubate at 37°C for 5 min.</LI><LI>Add 1 mL of EtOh 100% (-20°C). </LI><LI>Incubate at -20°C (10 minutes – 2 hours)</LI><LI>Centrifuge solution at 14000 rpm for 10 min. </LI><LI>Pour out supernatant.</LI><LI>Open and invert tubes on a paper towel to drain them out.</LI><LI>Add 200 µL of EtOH 70%. (-20°C) </LI><LI>Centrifuge solution at 14000 rpm for 5 min.</LI><LI>Pour supernatant in sink.</LI><LI>Dry pellet at 37°C for 5-20 min. (Until pellet is completely dry). </LI><LI>Add 30 µL of DNAse free water </LI><LI>Resuspend with vortex</LI><LI>Incubate at 37°C for 20 min.</LI><LI>Measure concentrations at Nanodrop.</LI></OL><H5>Waste disposal</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD><TD>What to avoid</TD></TR><TR><TD>Supernatant</TD><TD>Sink after adding bleach. </TD></TR><TR><TD>Pipet tips </TD><TD>Trash after adding bleach. </TD></TR><TR><TD>Eppendorf tubes</TD><TD>Trash after adding bleach.</TD></TR></TBODY></TABLE><H5>Safety notes:</H5><UL><LI>Wear globes</LI><LI>Use lab coat</LI></UL><H5>References:</H5><P>Made by: Ana Karen Tello</P><P>Date: 29/08/15</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingEight"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseEight" aria-expanded="false" aria-controls="collapseEight">
                  Protocol 8: Bacmid Transformation
                </A></H4></DIV><DIV id="collapseEight" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingEight"><DIV class="panel-body"><H5>Objective</H5><P>Generate recombinant bacmid by transforming DH10Bac E.coli</P><H5>Approximate Time</H5><P>300 min</P><H5>Material</H5><UL><LI>S.O.C. Medium</LI><LI>LB agar plates containing kanamycin, gentamicin, tetracycline, Bluo-gal and IPTG.</LI><LI>15 ml round-bottom polypropylene tubes.</LI></UL><H5>Equipment and Apparatus</H5><P>Name</P><UL><LI>37ºC incubator with shaker.</LI><LI>Water bath</LI></UL><H5>Previous Steps</H5><UL><LI>Water bath</LI><LI>Autoclave LB-agar media</LI><LI>Equilibrate a water bath to 42°C.</LI><LI>Warm selective plates at 37°C for 30 minutes.</LI><LI>Warm the S.O.C. Medium to room temperature.</LI><LI>Pre-chill one 15 ml round-bottom polypropylene tube for each transformation.</LI></UL><H5>Related Protocols</H5><P>Miniprep</P><H5>Steps</H5><OL><LI>For each transformation, gently mix and transfer 100 μl of the DH10Bac cells into the 15 ml tubes.</LI><LI>Add the appropriate amount of plasmid DNA to the cells and mix gently.
                      </LI><LI>Construct: 5 μl</LI><LI>Control: 5 μl</LI><LI>Incubate cells on ice for 30 minutes.</LI><LI>Heat-shock the cells for 45 seconds at 42°C without shaking</LI><LI>Immediately transfer the tubes to ice and chill for 2 minutes.</LI><LI>Add 900 μl of room temperature S.O.C. Medium.</LI><LI>Shake tubes at 37°C at 225 rpm for 4 hours. </LI><LI>Prepare dilutions of the cells (10-1, 10-2, 10-3) with S.O.C. medium.</LI><LI>Plate 100 μl of the transformation onto a LB agar plate containing the appropriate antibiotics. </LI><LI>Incubate plates for 48 hours at 37°C. </LI></OL><H5>Waste disposal</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD></TR><TR><TD>Petri dishes with culture. </TD><TD>Place in a bag to sterilize. </TD></TR><TR><TD>Liquid LB with culture. </TD><TD>Place in a container and with bleach then sterilize. </TD></TR></TBODY></TABLE><H5>References</H5><P>Invitrogen. Bac-to-Bac® Baculovirus Expression System: An efficient site-specific transposition system to generate baculovirus for high-level expression of recombinant proteins. (2004, April 6).</P><H5>Made by</H5><P>Carolina Elizondo</P><H5>Date</H5><P>08/29/2015</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingNine"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseNine" aria-expanded="false" aria-controls="collapseNine">
                  Protocol 9: Transformation
              </A></H4></DIV><DIV id="collapseNine" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingNine"><DIV class="panel-body"><H5>Objective</H5><P>Insert plasmid DNA in calcium competent cells.</P><H5>Approximate Time</H5><P>90 min.</P><H5>Material</H5><UL><LI>Liquid LB Media.</LI><LI>Petri dishes.</LI></UL><H5>Equipment and Apparatus</H5><P>Name:</P><UL><LI>Incubator with shaker.</LI><LI>Laminar flow safety cabinet.</LI><LI>Water bath.</LI></UL><H5>Previous steps</H5><UL><LI>Autoclave LB liquid media and LB-agar media.</LI></UL><H5>Related protocols</H5><P>Miniprep</P><H5>Steps</H5><OL><LI>Take competent cells from -80ºC freezer.</LI><LI>Turn on water bath at desired temperature.</LI><LI>Make sure you have 70 μl of competent cells on a eppendorf tube. Keep them on ice.</LI><LI>Add 50ng of DNA (1-2 μl).</LI><LI>Place the competent cell/DNA mixture on ice for 20min.</LI><LI>Heat shock each transformation tube by placing the bottom of the tube into a 42°C water bath for 45 seconds (time varies depending on the competent cells you are using).</LI><LI>Put the tubes back on ice for 2 min.</LI><LI>Add 500μl LB media (without antibiotic) and grow in 37°C shaking incubator for 1 hour. </LI><LI>Plate 100 μl of the transformation onto a LB agar plate containing the appropriate antibiotic.</LI><LI>Incubate plates at 37°C overnight.</LI></OL><H5>Waste Disposal</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD></TR><TR><TD>Petri dishes with culture.</TD><TD>Place in a bag to sterilize</TD></TR><TR><TD>Liquid LB with culture.</TD><TD>Place in a container and wit bleach then sterilize.</TD></TR></TBODY></TABLE><H5>Safety Notes</H5><OL><LI>Every step must be done in the safety cabinet to avoid contamination. </LI><LI>Gloves must be worn at all times.</LI><LI>Use lab coat.</LI></OL><H5>References</H5><P>Seidman, C. E., Struhl, K., Sheen, J., &amp; Jessen, T. (2005). “Basic Protocol 1: Transformation Using Calcium Chloride.” Introduction of plasmid DNA into cells. Current protocols in molecular biology, 1-8.</P><H5>Made by</H5><P>Carolina Elizondo</P><H5>Date</H5><P>08/29/15</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTen"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseTen" aria-expanded="false" aria-controls="collapseTen">
                  Protocol 10: Passaging Suspension Cells
              </A></H4></DIV><DIV id="collapseTen" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTen"><DIV class="panel-body"><H5>Objective</H5><P>To continue providing cell culture enough nutrients and space for their proper growth.</P><H5>Approximate time</H5><P>0-60 minutes</P><H5>Material</H5><UL><LI>Culture vessels containing your suspension cells.</LI><LI>Shaker flasks without baffles.</LI><LI>Complete growth medium, pre-warmed to 27°C.</LI><LI>Reagents and equipment to determine viable and total cell counts.</LI></UL><H5>Equipment and Apparatus</H5><P>Name</P><UL><LI>37°C incubator with humidified atmosphere of 5% CO2</LI><LI>Shaking platform</LI></UL><H5>Previous steps</H5><UL><LI>Growth of cells in shaker flasks before reaching confluency.</LI></UL><H5>Related protocols</H5><P>Cell counting</P><H5>Steps</H5><OL><LI>When the cells are ready for passaging (i.e., log-phase growth before they reach confluency), remove the flask from the shaking incubator, and take a small sample from the culture flask using a sterile pipette. If cells have settled
                      down before taking the sample, swirl the flask to evenly distribute the cells in the medium.</LI><LI>From the sample, determine the total number of cells and percent viability using a hemocytometer, cell counter, and Trypan Blue exclusion or an automated cell counter.</LI><LI>Calculate the volume of media that you need to add to dilute the culture down to the recommended seeding density.</LI><LI>Aseptically add the appropriate volume of pre-warmed growth medium into the culture flask. You may split the culture to multiple flasks if needed.</LI><LI>Loosen the caps of the culture flasks one full turn to allow for proper gas exchange (or use a gas-permeable cap), and return the flasks to the shaking incubator.</LI></OL><H5>Waste Disposal</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD></TR><TR><TD>Cell waste</TD><TD>Sink after adding bleach</TD></TR></TBODY></TABLE><H5>Safety Notes</H5><OL><LI>Wear gloves.</LI><LI>Use lab coat.</LI></OL><H5>References</H5><P>Cell Culture Basics Handbook Thermo-Fischer Scientific</P><H5>Made By</H5><P>Dania Bazúa Gerez</P><H5>Date</H5><P>08/29/2015</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingEleven"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseEleven" aria-expanded="false" aria-controls="collapseEleven">
                  Protocol 11: Cell Counting with a Hemocytometer
              </A></H4></DIV><DIV id="collapseEleven" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingEleven"><DIV class="panel-body"><H5>Objective</H5><P>Determine the amount of media needed for passaging of suspension cells. </P><H5>Approximate time</H5><P>45-60 minutes </P><H5>Material</H5><UL><LI>Hemocytometer </LI><LI>Trypan blue</LI><LI>Eppendorf tube</LI><LI>Balanced salt solution (i.e. DPBS solution) </LI></UL><H5>Equipment and Apparatus</H5><P>Name</P><UL><LI>37°C incubator with humidified atmosphere of 5% CO2</LI><LI>Shaking platform</LI></UL><H5>Previous steps</H5><UL><LI>Growth of cells in shaker flasks before reaching confluency.</LI></UL><H5>Related protocols</H5><P>Passaging suspension cells</P><H5>Steps</H5><OL><LI>Ensure the hemocytometer is clean using 70% ethanol. </LI><LI>Make sure the cell suspension to be counted is well mixed by either gentle agitation of the flask containing the cells. </LI><LI>Before the cells have a chance to settle take out about 1 mL of cell suspension using a serological pipette and place in an Eppendorf tube. </LI><LI>Prepare a cell suspension in a balanced salt solution (i.e. DPBS solution). Add 0.05 mL of salt solution and 0.05 mL of cells. Add 0.01 mL of trypan blue to the solution (dilution factor=4).</LI><LI>Draw up some cell suspension containing trypan blue. Carefully fill the hemocytometer by gently resting the end of the micropipette tip at the edge of the chambers. Take care not to overfill the chamber.</LI><LI>Focus on the grid lines of the hemocytometer using the 10X objective of the microscope and focus on one set of 16 corner squares.</LI><LI>Count the number of cells in this area of 16 squares. When counting, always count only live cells that look healthy (unstained by trypan blue). Count cells that are within the square and any positioned on the right hand or bottom boundary
                      line. </LI><LI>Dead cells stained blue with trypan blue can be counted separately for a viability count.</LI><LI>Move the hemocytometer to another set of 16 corner squares and carry on counting until all 4 sets of 16 corner squares are counted. Adittionally you can count the center square of 25 squares. </LI><LI>Use formulas to perform appropriate calculations:
                       CELLS PER ml = the average count per square *dilution factor *10**4 (count 10 squares)
                       TOTAL CELLS = cells per ml* the original volume of fluid from which cell sample was removed.
                       CELL VIABILITY (%) = total viable cells
                       (unstained) ) / total cells (stained and unstained) *100.</LI></OL><H5>Waste disposal</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD><TD>What to avoid</TD></TR><TR><TD>Solution from hemocytometer</TD><TD>Appropriate container for cancergenic/mutagenic waste</TD><TD>Contact</TD></TR></TBODY></TABLE><H5>Safety Notes</H5><OL><LI>Wear gloves.</LI><LI>Use lab coat.</LI></OL><H5>References</H5><P>Sigma “Use of Trypan Blue Stain and the Hemocytometer to Determine Total Cell Counts and Viable Cell Number” Abcam “Counting cells using a hemocytometer” </P><H5>Made by</H5><P>Dania Bazúa Gerez</P><H5>Date</H5><P>08/29/2015</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingTwelve"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseTwelve" aria-expanded="false" aria-controls="collapseTwelve">
                  Protocol 12: Transfection of Insect Cells
              </A></H4></DIV><DIV id="collapseTwelve" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwelve"><DIV class="panel-body"><H5>Objective</H5><P>Insertion of the recombinant bacmid DNA in the Sf9 cells for the production of the recombinant baculovirus.</P><H5>Approximate time</H5><P>5 hours (including incubation time)</P><H5>Material</H5><UL><LI>Purified recombinant bacmid DNA construct (500 ng/µL in TE buffer, pH 8.0).</LI><LI>Sf9 cells.</LI><LI>Cellfectin II Reagent (store at 4° until use).</LI><LI>Grace’s Insect Cell Medium, Unsupplemented (no supplements, FBS, antibiotics).</LI><LI>24-well tissue culture plates.</LI><LI>1.5-mL sterile microcentrifuge tubes.</LI><LI>Complete growth medium for culturing insect cells (Sf-900 II SFM).</LI></UL><H5>Equipment and Apparatus</H5><UL><LI>Bacteriological Hood.</LI><LI>Humidified Incubator at 27°C.</LI></UL><H5>Previous steps</H5><UL><LI>Cell counting/Subculturing to assure Sf9 cells are in the log phase (1.5–2.5 × 106 cells/mL) with greater than 95% viability</LI><LI>Bacmid MiniPrep to have the DNA extracted and quantified for the transfection</LI></UL><H5>Related protocols</H5><UL><LI>Isolation of P1 viral stock.</LI><LI>Amplification of baculoviral stock.</LI><LI>Viral Plaque Assay.</LI></UL><H5>Steps</H5><OL><LI>Verify that the Sf9 cells are in the log phase (1.5–2.5 × 106cells/mL) with greater than 95% viability.</LI><LI>f the cell density is in range of 1.5 × 106–2.5 × 106 cells/mL and the culture is without antibiotics, proceed to step 2a. If the cell density is not in this range or the cell culture contains antibiotics, follow steps 2b–2c:
                      </LI><LI>Add 500 μL of Grace’s Insect Medium, Unsupplemented (without antibiotics and serum) in each well. Seed 8 × 105 Sf9 cells from Step 1 per well. Do not change medium or wash the cells. The medium carried over will enhance the transfection
                          efficiency. Allow cells to attach for 15 minutes at room temperature in the hood. Proceed to step 3.</LI><LI>Prepare 10 mL plating medium by mixing 1.5 mL Supplemented Grace’s Insect Medium containing 10% FBS (without antibiotics) and 8.5 mL Grace’s Insect Medium, Unsupplemented (without FBS and antibiotics).</LI><LI>Plate 8 × 105 Sf9 cells from Step 1 per well. Allow cells to attach for 15 minutes at room temperature in the hood. Remove the medium. Add 550 μL plating medium from step 2b per well. Proceed to step 3.</LI></OL><LI>For each transfection sample, prepare complexes as follows:
                      </LI><LI>Mix Cellfectin® II before use, and dilute 2 μL in 25 μL Grace’s Medium, Unsupplemented (without antibiotics and serum). Vortex briefly to mix. Note: You may leave this mixture at room temperature for up to 30 minutes.</LI><LI>Dilute 1 μL baculovirus DNA in 25 μL Grace’s Medium, Unsupplemented (without antibiotics and serum). Mix gently.</LI><LI>Combine the diluted DNA with diluted Cellfectin® II (total volume ~50 μL). Mix gently and incubate for 15–30 minutes at room temperature.</LI><LI>Add ~50 μL DNA-lipid mixture or transfection mixture (Step 3c)
                      <STRONG>dropwise</STRONG> onto the cells from Step 2. Incubate cells at 27°C for 3–5 hours.</LI><LI>Remove the transfection mixture and replace with 550 μL of complete growth medium (e.g., Grace’s Insect Medium,Supplemented and 10% FBS). Using antibiotics is optional.</LI><LI>Incubate cells at 27°C for 72 hours or until you see signs of viral infection</LI><H5>Waste disposal</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD><TD>What to avoid</TD></TR><TR><TD>Growth Media</TD><TD>Bleach and dispose in the sink</TD><TD>N/A</TD></TR><TR><TD>Transfection Mixture </TD><TD>Bleach and dispose in the sink</TD><TD>N/A</TD></TR></TBODY></TABLE><H5>Safety notes</H5><UL><LI>Every step must be done in the hood to avoid contamination.</LI><LI>Gloves must be worn at all times.</LI></UL><H5>References</H5><P>Life Technologies Corporation. (2013). Bac-to-Bac® Baculovirus Expression System . Carlsbad, CA, USA: Life Technologies</P><H5>Made by</H5><P>Claudia Nallely Alonso Cantú </P><H5>Date</H5><P>August 29, 2015</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingThirteen"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseThirteen" aria-expanded="false" aria-controls="collapseThirteen">
                  Protocol 13: Viral Plaque Assay
              </A></H4></DIV><DIV id="collapseThirteen" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThirteen"><DIV class="panel-body"><H5>Objective</H5><P>Perform a viral plaque assay in order to watch the presence of recombinant baculovirus and calculate the titer. </P><H5>Approximate time</H5><P>418 - 490 min</P><H5>Material</H5><P>Name (Supplier)</P><UL><LI>15-ml tubes.</LI><LI>12-well plate.</LI><LI>50-ml tube.</LI><LI>Sf-900 Medium 1X.</LI><LI>Agarose gel.</LI></UL><H5>Equipment and Apparatus</H5><P>Name:</P><OL><LI>Humidified incubator.</LI><LI>Laminar flow cabinet.</LI><LI>Inverted microscope.</LI><LI>Water bath (Thermo Scientific).</LI></OL><H5>Previous steps</H5><UL><LI>Transfection of Sf9 cells.</LI><LI>Viral stock isolation or amplification of baculovirus stock.</LI></UL><H5>Related protocols</H5><UL><LI>Transfection of Sf9 cells.</LI><LI>Amplification of baculovirus stock.</LI><LI>Viral stock isolation.</LI><LI>Plaquing medium.</LI><LI>Calculating the titer of baculoviral stock.</LI></UL><H5>Steps</H5><OL><LI>On the day of infection, harvest Sf9 cells and prepare a 15 ml cell suspension 5×105 cells / ml in Sf-900 II SFM. </LI><LI>Aliquot 0.67 ml of cell suspension into each well of two 12-well plates, including a negative control.</LI><LI>Allow the cells to settle to the bottom of the plate and incubate, covered, at room temperature for 1 hour.</LI><LI>Prepare the plaquing medium. Melt 4% agarose gel in 10 ml by placing it on 50-ml in a 70ᵒC water bath for 20 to 30 minutes or heating the agarose in a microwave oven.</LI><LI>While the 4% agarose gel is melting, place the following in the 40ᵒC water bath:
                      </LI><LI>Empty, sterile 50-ml tube.</LI><LI>Sf-900 Medium (1X).</LI></OL><LI>Once the 4% agarose gel has liquefied, move the agarose gel, medium, and empty 50-ml tube to a sterile hood. </LI><LI>Working quickly, prepare the plaquing medium as follows:
                      </LI><LI>Combine 30 ml of Sf-900 Medium (1X) and 10 ml of melted 4% agarose gel in the empty 50-ml tube and mix gently.</LI><LI>Return the bottle of plaquing medium to the 40ᵒC water bath until use. </LI><LI>After 1 hour, observe the cell monolayers using an inverted microscope. Sf9 cells should be attached and at 50% confluence.</LI><LI>Prepare an 8-log serial dilution 10-1 to 10-8 of the clarified baculoviral stock in Sf900 II SFM, diluting 0.25 ml of the baculoviral stock or previous dilution in 2.25 ml of medium in 15-ml tubes.</LI><LI>Move the 12-well plates containing Sf9 cells and the tubes of diluted virus to the sterile hood.</LI><LI>Label the plates as follows: no virus (negative control), 10-4, 10-5, 10-6, 10-7, 10-8.</LI><LI>Remove the medium from each well, discard, and immediately replace with 0.67 ml of the appropriate virus dilution. As a negative control, add the appropriate medium without virus.</LI><LI>Incubate cells with virus for 1 hour at room temperature.</LI><LI>Move the cells and the tube of plaquing medium from the 40ᵒC water bath to a sterile hood.</LI><LI>Sequentially starting from the highest dilution to the lowest dilution, remove the medium containing the virus from the wells and replace with 0.67 ml of plaquing medium. Work quickly to avoid dessication of the cell monolayer.</LI><LI>Allow agarose overlay to harden for 1 hour at room temperature before moving the plates.</LI><LI>Incubate the cells in a 27ᵒC humidified incubator for 7-10 days until plaques are visible and ready to count.</LI><LI>OPTIONAL: Stain plaques to facilitate counting and calculate the titer.</LI><H5>Waste disposal</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD></TR><TR><TD>Supernatant</TD><TD>Trash after autoclaving.</TD></TR><TR><TD>Pipet tips</TD><TD>Trash after adding bleach.</TD></TR><TR><TD>Eppendorf tubes </TD><TD>Trash after adding bleach. </TD></TR><TR><TD>Plaquing Medium </TD><TD>Trash.</TD></TR><TR><TD>Dilutions </TD><TD>Trash after autoclaving. </TD></TR></TBODY></TABLE><H5>Safety notes</H5><OL><LI>Wear gloves.</LI><LI>Use lab coat.</LI></OL><H5>References</H5><P>Bac-to-Bac® Baculovirus Expression System (Invitrogen by LifeTechnologies).</P><H5>Made by</H5><P>Ana Karen Carrizales</P><H5>Date</H5><P>08/29/15</P></DIV></DIV></DIV><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="heading14"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse14" aria-expanded="false" aria-controls="collapse14">
                  Protocol 14: Viral stock isolation
              </A></H4></DIV><DIV id="collapse14" class="panel-collapse collapse" role="tabpanel" aria-labelledby="heading14"><DIV class="panel-body"><H5>Objective</H5><P>Isolate P1 viral stock from transfected Sf9.</P><H5>Approximate time</H5><P>20 min.</P><H5>Material</H5><UL><LI>Eppendorf tubes.</LI><LI>Aluminium foil.</LI></UL><H5>Equipment and Apparatus</H5><P>Name</P><UL><LI>Centrifuge</LI><LI>Refrigerator (-20 and -80ᵒC)</LI><LI>Laminar flow cabinet</LI></UL><H5>Previous steps</H5><UL><LI>Transfection of Sf9 cells.</LI></UL><H5>Related protocols</H5><UL><LI>Transfection of Sf9 cells. </LI><LI>Amplification of baculovirus stock.</LI><LI>Viral plaque assay. </LI><LI>Plaquing medium. </LI><LI>Calculating the titer of baculoviral stock.</LI></UL><H5>Steps</H5><OL><LI>Look if transfected cells demonstrate signs of late stage infection (24 - 72 hours post-transfection). </LI><LI>Collect the medium containing virus from each well (0.5 ml) and transfer to sterile 1.5-ml tubes. </LI><LI>Centrifuge the tubes at 500 x g for 5 minutes to remove cells and large debris. </LI><LI>Transfer the clarified supernatant to fresh 1.5-ml tube.</LI><LI>Store at 4ᵒC, protected from light.</LI></OL><H5>Waste disposal</H5><TABLE class="table"><TBODY><TR><TD>Waste</TD><TD>Place to dispose</TD></TR><TR><TD>Supernatant </TD><TD>Sink after adding bleach.</TD></TR><TR><TD>Pipet tips </TD><TD>Trash after adding bleach.</TD></TR><TR><TD>Eppendorf tubes</TD><TD>Trash after adding bleach.</TD></TR></TBODY></TABLE><H5>Safety notes</H5><OL><LI>Wear gloves.</LI><LI>Use lab coats.</LI></OL><H5>References</H5><P>Bac-to-Bac® Baculovirus Expression System (Invitrogen by LifeTechnologies).</P><H5>Made by</H5><P>Ana Karen Carrizale</P><H5>Date</H5><P>08/29/2015</P></DIV></DIV></DIV></DIV></ARTICLE><ARTICLE class="col-xs-12 col-sm-12 col-md-10 col-md-offset-1"><H1>Discussion of Results</H1><P>The plasmid C2 is composed of a pPH promoter, a secretion signal Honey Bee Melittin (HBM), two reporter proteins, Nanoluc and RFP divided by the T2A cleavage peptide for bicistronic vectors.
These results confirm the activity of the pPH promoter at 72 h post-transfection. They also confirm the activity of the secretion peptide HBM that was fused to the reporter protein Nanoluc.
The observed data indicates the predominant presence of Nanoluc in the supernatant due to the secretion signal. Also we found luminescence in the interior of the cells but the signal was 86 times lower. We estimate a percentage of secretion efficiency of 98.85%</P><DIV class="row" style="text-align: center"><TABLE class="table"><TBODY><TR><TH>Experiment</TH><TH>Luminescence of supernatant</TH><TH>Luminescence of lysate</TH><TH>Efficiency secretion</TH></TR><TR><TD>1 with C2</TD><TD>102006.667</TD><TD>1176.667</TD><TD>98.85%</TD></TR><TR><TD>2 with C2</TD><TD>122766.667</TD><TD>1434.667</TD><TD>98.85%</TD></TR></TBODY></TABLE><STRONG>Table 2. Efficiency of secretion signal HBM.</STRONG></DIV><P>
Since the fluorescence of RFP wasn’t detected, we assume that the T2A is not working in its cleavage mechanism which means that the RFP probably continued to be fused with Nanoluc by the T2A. We conclude that the T2A doesn’t work in Sf9 cells but we’ll do more experiments to confirm the size of the complex/fused protein. We’d also continue to study this mechanism because the hydrolysis of the peptidyl:glycyl-tRNA ester linkage was not completely understood.</P></ARTICLE><ARTICLE class="col-xs-12 col-sm-12 col-md-10 col-md-offset-1"><H1>References</H1><OL style="list-style:decimal;font-size:14px;"><LI>Drugmand, J.-C., Schneider, Y.-J., &amp; Agathos, S. N. (2012). Insect cells as factories for biomanufacturing. Biotechnology Advances, 30(5), 1140-1157. doi:http://dx.doi.org/10.1016/j.biotechadv.2011.09.014</LI><LI>Fernandes, F., Vidigal, J., Dias, M. M., Prather, K. L. J., Coroadinha, A. S., Teixeira, A. P., &amp; Alves, P. M. (2012). Flipase-mediated cassette exchange in Sf9 insect cells for stable gene expression.Biotechnology and Bioengineering, 109(11), 2836-2844. doi:10.1002/bit.24542</LI><LI>Greene, J. (2004). Host Cell Compatibility in Protein Expression. En P. Balbás, &amp; A. Lorence, Recombinant Gene Expression (págs. 3-14). USA: Humana Press</LI><LI>Invitrogen. (2015). Guide to Baculovirus Expression Vector System (BEVS) and Insect Cell Culture Techniques. Obtenido de Invitrogen: https://tools.thermofisher.com/content/sfs/manuals/bevtest.pdf</LI><LI>Jardin, B. A., Montes, J., Lanthier, S., Tran, R., &amp; Elias, C. (2007). High cell density fed batch and perfusion processes for stable non-viral expression of secreted alkaline phosphatase (SEAP) using insect cells: Comparison to a batch Sf-9-BEV system. Biotechnology and Bioengineering, 97(2), 332-345. doi:10.1002/bit.21224</LI><LI>Kempf, J., Snook, L. A., Vonesch, J.-L., Dahms, T. E. S., Pattus, F., &amp; Massotte, D. (2002). Expression of the human μ opioid receptor in a stable Sf9 cell line. Journal of Biotechnology, 95(2), 181-187. doi:http://dx.doi.org/10.1016/S0168-1656(02)00008-1
Shen, X., Hacker, D. L., Baldi, L., &amp; Wurm, F. M. (2014). Virus-free transient protein production in Sf9 cells. Journal of Biotechnology, 171, 61-70. doi:http://dx.doi.org/10.1016/j.jbiotec.2013.11.018</LI><LI>magghe, G., Goodman, C., &amp; Stanley, D. (2009). Insect cell culture and applications to research and pest management. In Vitro Cellular &amp; Developmental Biology - Animal, 45(3-4), 93-105. doi:10.1007/s11626-009-9181-x</LI></OL></ARTICLE></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>