Team:Tec-Monterrey/research.html
iGEM MTY 2015
Toggle navigation
Main
Research
Notebook
Modeling
Outreach
Collaborations
Policies and Practices
Safety
Achievements
Team
Background
Objectives
BioBricks
Protocols
Results
Discussion
Conclusions
References
Background
In the past twenty years, the use of insect cells has grown enormously; the establishment of more than 500 insect cell lines from different organisms and tissues is proof of this (Smagghe et al, 2009). However, two cell lines are considered as           the basic models, Sf9 cell from Spodoptera frugiperda and High Five™ cells (officially named BTI-TN-5B1-4) derived from Trichopulsia ni.
Nowadays the most common way to work with the previous cell lines is by using the baculovirus expression vector systems (BEVS). This method consists in the use of a baculovirus, normally the multicapsid nuclear polyhedrosis virus Autographa           california (AcMNPV), as a vector to infect the cells in order to produce protein. The gene of interest is introduced into the viral genome via homologous recombination with a transfer vector (Greene, 2004) , see Figure 1.
Figure 1. Generation of recombinant baculoviruses and gene expression in Sf9 cells. Figure from (Invitrogen, 2015)
Objectives
The aim of our project is to expand the use of insect cells in synthetic biology. We believe that we have the opportunity to enrich the part registry with biobricks designed for an insect chassis. To accomplish our goal, our team is going to introduce various biobricks that contain basic functional parts necessary to build your own expression cassette for insect cells. In addition, we want to validate the use of the CRISPR/Cas9 System in the Sf9 cells for specific genome editing.
Biobricks
Polyhedrin promoter (pPH) (BBa_K1734000)
Natively, this promoter activates the transcription of the polyhedrin gene (polh), a major occlusion-body matrix protein expressed during the very late phase of infection.
OpIE2 promoter (BBa_K1734001)
The immediate-early 2 promoter from the multicapsid nucleopolyhedrosis virus Orgyia pseudotsugata (OpIE2) is a constitutive promoter. Usually is used in nonlytic gene expression systems, but can be used with the baculovirus expression vector system (BEVS) methodology as well.
Polyhedrin promoter + Mellitin’s signal Peptide (BBa_K1734002)
The biobrick contains the polyhedrin promoter from the baculovirus Autographa californica. After the promoter there is a secretion signal sequence of the protein Mellitin from the organism Abis mellifera (Honeybee).
Polyhedrin promoter + Lysozyme’s signal peptide (BBa_K1734003)
The biobrick contains the polyhedrin promoter from the baculovirus Autographa californica. After the promoter there is a secretion signal sequence of the protein lysozyme from the organism Gallus gallus (chicken).
Nanoluc (Codon optimized for Sf9 cells) (BBa_K1734004)
Nluc luciferase is an enzyme that was engineered to work as a luminescent reporter weighing 19.1 kDa. This protein uses fumarizine as novel substrate to produce a high intensity glow-type luminescence. These reactions are ATP-independent, trying to obtain the maximal assay sensitivity.
6xHis-T24-Nanoluc (BBa_K1734006)
This part encodes a sequence capable to add a purification tag (6X His) at the 3’ terminus of a desired protein. Furthermore, by using the reporter gene (nanoluc) it’s possible to quantify indirectly the protein concentration. The T2A sequence cleaves the protein, producing the protein of interest tagged with HisTag and nanoluc separately. This part has been codon optimized for Spodoptera frugiperda (Sf9).
Protocols
Protocol 1: Calcium Competent Cells
Objective
To obtain a resilient bacteria against a transformation procedure in order to insert a plasmid on it.
Approximate Time:
1680 min = 28 hours
Material
0.250 L LB
0.5 L CaCl2 0.1M
6 centrifuge tubes
2 Erlenmeyer flasks
Lots of microfuge tubes
50 ml glycerol
Equipment and Apparatus:
The day prior to this protocol, things to were:
Autoclave:
0.250 L LB medium
0.5 L CaCl2 0.1M
6 centrifuge tubes
2 Erlenmeyer flasks of 250 ml capacity
Microfuge flask
50 ml glycerol
Tips of 1 ml
Tips of 200 l
PCR Tubes
For the step 1 in this protocol:
Put 20 ml LB medium in 2 sterile centrifuge tubes.
Inoculate using 10 ul of E. coli stock or take a single colony from an LB plate.
Gtr culture overnight at 37°C in a shaker.
Store at 4°C CaCl2 solution, tips for micropipette and PCR tubes.
Related Protocols:
Transformation:
Incubate a pre-inoculum in 2 centrifuge tubes using 20 ml of LB medium in each tube overnight at 37°C on a shaker at 250 rpm.
Inoculate 200ml of LB medium. Measure the OD600 to get 0.1.
Incubate at 37°C, 250 rpm until OD600 gets between 0.4 and 0.6 (Measure the OD600 every hour, this step takes 1.5 to 2 hours).
Split 200 ml culture in 4 centrifuge tubes, pouring 50 ml in each one of them. Label properly every tube.
Starting from this step for the remainder of the procedure, it is totally important to keep cells at 4°C. The cells, and any bottles or solutions that they come in contact with, must be pre-chilled to 4°C.
Keep these tubes with cells in suspension in ice for 30 minutes.
Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C.
Decant the supernatant and gently resuspend the pellet in 1 ml of ice cold CaCl2. Once dissolved, fill every tube reaching a 20 ml final volume with ice cold 0.1M CaCl2.
Keep these tubes with cells in suspension in ice for 20 minutes.
Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C.
Decant the supernatant and gently resuspend the pellet in 1 ml of ice cold CaCl2. Once dissolved, fill every tube reaching a 20 ml final volume with ice cold 0.1M CaCl2.
Keep these tubes with cells in suspension in ice for 15 minutes.
Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C.
Decant the supernatant and gently resuspend the pellet in 425 l of ice cold CaCl2 and 75l of glycerol.
Aliquot 70 l into sterile PCR tubes. Store frozen cells in the -80°C freezer.
Waste Disposal:
Waste
Place to dispose
What to avoid
Supernatant
Sink after adding bleach
Pipette tips
Trash after autoclave sterilization
Safety notes
Do all this procedure in a hood.
Wear gloves.
Wear lab coat.
Protocol 2: Calcium Competent Cells
Objective:
To obtain a resilient bacteria against a transformation procedure, in order to insert a plasmid into it
Approximate Time:
1680 min = 28 hours
Materials:
0.250 L label
0.5 L CaCl2 0.1M
6 centrifuge tubes
2 Erlenmeyer flask
Lots of microfuge tubes
50 ml glycerol
Equipment and apparatus
Spectrophotometer
Centrifuge
-80º C freezer
Previous steps:
The day prior to this protocol, things to do were:
Autoclave:
0.250 L LB medium
0.5 L CaCl2 0.1M
6 centrifuge tubes
2 Erlenmeyer flask 250 ml
Microfuge flask
50 ml glycerol
Tips of 1 ml
Tips of 200 l
PCR Tubes
For step 1 in this protocol:
Put in 2 sterile centrifuge tubes 20 ml LB medium in each tube.
Inoculate using 10 ul of E. coli stock or take a single colony from LB plate.
Grow culture at 37°C in shaker overnight.
Store at 4°C CaCl2 solution, tips for micropipette and PCR tubes.
Related protocols:
Transformation:
Incubate a pre-inoculum 2 centrifuge tubes using 20 ml of LB medium in each tube overnight at 37°C on a shaker at 250 rpm.
Inoculate 200ml of LB medium. Measure the OD600 to get 0.1.
Incubate at 37°C, 250 rpm until OD600 gets between 0.4 and 0.6 (Measure the OD600 every hour, this step takes 1.5 to 2 hours).
Split 200 ml culture in 4 centrifuge tubes, pouring 50 ml in each one of them. Label properly every tube.
Starting from this step for the remainder of the procedure, it is totally important to keep cells at 4°C. The cells, and any bottles or solutions that they come in contact with, must be pre-chilled to 4°C.
Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C.
Decant the supernatant and gently resuspend the pellet in 1 ml of ice cold CaCl2. Once dissolved, fill every tube reaching a 20 ml final volume with ice cold 0.1M CaCl2.
Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C.
Decant the supernatant and gently resuspend the pellet in 1 ml of ice cold CaCl2. Once dissolved, fill every tube reaching a 20 ml final volume with ice cold 0.1M CaCl2.
Harvest the cells by centrifugation at 3000xg, for 5 minutes at 4°C.
Decant the supernatant and gently resuspend the pellet in 425 l of ice cold CaCl2 and 75l of glycerol.
Aliquot 70 l into sterile PCR tubes. Store frozen cells in the -80°C freezer.
Waste disposal:
Waste
Place to dispose
What to avoid
Supernatant
Sink after adding bleach
Pipette tips
Trash after auclave sterilization
Safety notes
Do all this procedure in a hood
Wear gloves
Wear lab coat
Protocol 3: Digestion with restriction enzymes
Objective:
Cut DNA fragments for ligation.
Analyze DNA sanmples through the use of electrophoresis
Approximate time:
120 min
Materials:
Nuclease free distilled water
PCR Tubes
Digestion buffer (NEB)
Restriction enzymes (NEB)
Pipettes and tips
DNA sample
Equipment and Apparatus
Thermocycler (BioRad)
Nanodrop (Thermo)
Previous steps:
Measure the concentration of DNA in the sample using the Nanodrop.
Depending on the concentration of DNA in the sample and the volume of the reaction, make the calculation for the required volumes of the components. Normally, 1 unit of enzyme is put per µg of DNA; however, the provider recommends using                   10 units per µg of DNA to be digested. Our typical reaction was of 20 µL
Reaction volume
Resctriction enzyme
DNA
Buffer
10 μL
1 unit
0.1 µg
1 µL
25 μL
5 units
0.5 µg
2.5 μL
50 μL
10 units
1 µg
5 μL
Related protocols:
Gel electrophoresis, miniprep, ligation
Steps:
After labeling the PCR tubes, pour with the micropipette the nuclease free distilled water.
Pour the buffer to a concentration of 1X.
Introduce the DNA sample to the PCR tube to the required volume.
Pour the enzymes. The enzymes must be introduced lastly to the reaction. Handle them carefully, as they are temperature sensitive (handle with ice and return to refrigerator as soon as possible).
Program thermocycler at 37°C 1 hour for the digestion and 80°C for 20 minutes to inactivate the enzymes.
Analyze digestion results by agarose electrophoresis.
Waste disposal:
Waste
Place to dispose
What to avoid
Digestion waste in tube
Place in trash container
Safety notes:
In order to minimize the possibility of nuclease action, gloves must be worn at all times.
The enzymes must be handled with care.
References:
https://www.neb.com/protocols/2014/05/07/double-digest-protocol-with-standard-restriction-enzymes
Digestion Protocol for BioBrick Assembly Kit (E0546) https://www.neb.com/protocols/1/01/01/digestion-protocol-e0546
Protocol 4: Agarose based electrophoresis
Objective:
Analyze DNA fragments based on their movement in a uniform electric field.
Approximate time
120 min
Materials:
Agarose (BioRad)
TAE Buffer (Promega)
Loading dye (Promega)
10 kb and 1 kb weight stair (Promega)
Ethidium bromide (BioRad)
Pipettes and tips
DNA sample
Equipment and Apparatus
Electrophoresis chamber (Fisher Scientific)
Voltage source (BioRad)
Transiluminator (Thermo)
Previous steps:
Digestion: Usually this protocol precedes the electrophoresis
Check if there is TAE available. A stock 50X can be made from:
Tris-base: 242 g
Acetate (100% acetic acid): 57.1 ml
EDTA: 100 ml 0.5M sodium EDTA,
Add dH2O up to one litre.
Related protocols:
Digestion, enzyme restriction analysis
Steps:
The amount of agarose depends on the size of the DNA. (A guide can be seen in the table below)
Agarose Concentration (g/100mL)
Optimal DNA Resolution (kb)
0.5
1 - 30
0.7
0.8 - 12
1.0
0.5 - 10
1.2
0.4 - 7
1.5
0.2 - 3
Measure out the appropriate mass of agarose into a beaker with the appropriate volume of buffer. The volume in the lab gel box is 50mL.
Microwave until the agarose is fully melted and cool down a few minutes. (Microwave little by little because the solution might rise out of the beaker)
Pour the agarose solution into the gel box. Carefully pop or shove to the side any bubbles (with the help of a pipette tip), put the comb in place, and let it cool for about 15-30 minutes until the gel solidifies (cover with foil to                     avoid contamination).
Mix the DNA samples with 5X loading dye solution to make it reach 1X. (Except for 1 µL of sample you can use 1µL of Loading dye)
Pour the mixture in the wells. Be careful with the bubbles and with breaking the gel with the pipette.
Fill the chamber with TAE (about 300-450 mL) and run the gel 70 min at 90 V and ~180 mA
Stain the gel soaking it in EtBr, be careful not to break it.
Reveal using the trans illuminator, first focus using the white light lamp and then use the UV to make the bands visible.
Waste disposal
Waste
Place to dispose
What to avoid
TAE
Throw through the sink, the school has a water treatment plant for all the facilities.
NA
Gel
The gel has EtBr, which is very mutagenic, gloves must be used to handle the gel when you soak it in the solution. The gel must be disposed in the EtBr container.
Do not touch the gel or the EtBr solution with your bare hands.
EtBr solution
Every once in a while the solution must be changed, and the discarded solution must be?
See above.
Gloves
You must discard the glove you use when you handle the gel and the computer. They must be thrown in the EtBr labeled container.
Make sure to dispose your gloves
Safety notes:
Remember to use the EtBr at the delimited area in the lab.
Dispose your gloves after handling the gel.
If you accidentally touch a surface of EtBr designed area, remember to soak yourself with ethanol 70% to inactivate the EtBr. Seriously, this shouldn’t happen.
References:
Agarose Gel Electrophoresis http://www.addgene.org/plasmid-protocols/gel-electrophoresis/
Protocol 5: Bacmid Miniprep
Objective:
Isolate recombinant bacmid DNA from E. coli culture (DH10Bac)
Approximate time:
100-120 min
Materials:
Eppendorf tubes
Solution 1 (15 mM Tris-HCl (pH: 8), 10mM EDTA (pH 8)) (Filtered)
Solution 2 (0.2 M NaOH, 1% w/v SDS) (Filtered)
Solution 3 (Sodium acetate 3M (pH 5.5)) (Sterile)
Isopropanol
EtOh 70%
TE 1X
Equipment & Apparatus
Centrifuge
Safety cabinet
Previous steps:
Transformation for recombinant bacteria.
Prepare starter culture cells.
Related protocols:
Transfection for insect cells.
Steps:
Centrifuge starter culture at 3000 g for 5 min.
Decant supernatant and resuspend pellet in 1 mL of LB medium.
Pass cells to Eppendorf tube.
Centrifuge at 14000 rpm for 1 min.
Decant supernatant.
Resuspend (pippeting up and down) pellet in 300 μL of Solution 1.
Add 0.5 μl of RNAse to each tube
Add 300 μL of Solution II.
Invert tube carefully 5 times.
Incubate at room temperature for 5 min.
Add 300 µL of Solution III.
Mix by inverting several times.
Place the sample on ice for 10 min.
Centrifuge tubes at 14000 rpm for 10 min. Meanwhile, label another eppendorf tube and add 800 µL of isopropanol to it.
Collect supernatant in the new tube with isopropanol (around 700 µL).
Mix by gently inverting tube a few times and place on ice for 5 to 10 min.
Centrifuge the sample for 15 min at 14,000 g at room temperature.
Remove the supernatant and add 0.5 µl 70% ethanol to each tube.
Invert the tube several times to wash the pellet.
Centrifuge for 5 min at 14,000 g at room temperature.
Remove as much of the supernatant as possible.
Air dry the pellet briefly, 20 min.
Dissolve the DNA in 40 µl of TE.
Store DNA at -20°C.
Waste disposal
Waste
Place to dispose
What to avoid
Supernatant
Sink after adding bleach.
Pipet tips
Trash after adding bleach.
Eppendorf tubes
Trash after adding bleach.
Safety notes:
Every step must be done in the safety cabinet to avoid contamination.
Gloves must be worn at all times.
Use lab coat.
References
Made by: Carolina Elizondo
Date: 29/08/15
Protocol 6: Miniprep
Objective:
Isolate plasmid from E. coli culture
Approximate time:
180-240 min
Materials:
Eppendorf tubes
Solution 1 (50 mM Glucose, 25 mM Tris-HCl (pH: 8), 10mM EDTA (pH 8)
Solution 2 (0.2 M NaOH, 1% w/v SDS)
Solution 3 (Sodium acetate 3M, acetic acid 2M)
EtOH 100%
EtOH 70%
DNAse free water
Equipment and apparatus:
Centrifuge
Previous steps:
Transformation for recombinant bacteria.
Prepare starter culture cells.
Related Protocols:
Transformation for recombinant bacteria
Digestion
Electrophoresis
Steps:
Centrifuge starter culture at 3000 g for 5 min.
Decant supernatant and resuspend pellet in 1 mL of LB medium.
Transfer cells to Eppendorf tube.
Centrifuge at 14000 rpm for 2 min.
Decant supernatant.
Resuspend pellet in 100 µL of Solution 1.
Vortex solution for 2 min. until all bacteria are fully resuspended.
Add 200 µL of Solution II.
Invert tube carefully 5 times.
Incubate on ice for 5 min.
Add 150 µL of Solution III.
Mix by inverting several times.
Incubate on ice for 5 min.
Centrifuge tubes at 14000 rpm for 5 min.
Collect supernatant in new tube (around 400 µL).
Add 0.5 µL of 20 mg/mL RNase A to supernatant in new tube.
Incubate at 37°C for 5 min.
Add 1 mL of EtOh 100%.
Incubate at -80°C for 30 min or at -20°C for 1 hour.
Centrifuge solution at 14000 rpm for 15 min. (If possible at 4°C).
Pour out supernatant.
Open and invert tubes on a paper towel to drain them out.
Add 500 µL of EtOH 70%.
Centrifuge solution at 14000 rpm for 5 min.
Pour supernatant in sink.
Dry by inverting over paper towel 5-20 min. (Until pellet is completely dry).
Resuspend DNA with 20-40 µL of DNAse free water.
Measure concentrations at NanoDrop.
Store DNA at 4°C.
Waste disposal:
Waste
Place to dispose
What to avoid
Supernatant
Sink after adding bleach.
Pipette tips
Trash after adding bleach.
Eppendorf tubes
Trash after adding bleach.
Safety notes:
Wear gloves.
Use lab coat.
References:
Made by: Ana Karen Tello
Date: 29/08/15
Protocol 7: Miniprep
Objective:
Cheap DNA extraction for E. coli
Approximate time
180 - 240 min
Materials:
Eppendorf tubes
Solution 1 (Tris-HCl 0.1181 g, Cilantro extract 130 µL, distilled water 29.87 µL)
Solution 2 (NaOH 24 g, Axion detergent 300 µL, distilled water 29.7 µL)
Solution 3 (Sodium bicarbonate 5g, Lemon 8 mL)
EtOH 100%
EtOH 70%
DNAse free water
Equipment and apparatus
Centrifuge
Previous steps
Transformation for recombinant bacteria.
Prepare starter culture cells.
Related Protocols:
Transformation for recombinant bacteria
Digestion
Electrophoresis
Steps
Centrifuge starter culture at 3000 g for 5 min.
Decant supernatant and resuspend pellet in 1 mL of LB medium.
Pass cells to Eppendorf tube.
Centrifuge at 14000 rpm for 1 min.
Decant supernatant.
Add 200 µL of Solution 1
Vortex solution for 2 min. until all bacteria are fully resuspended.
Add 200 µL of Solution II.
Invert tube carefully 5 times.
Incubate at 37°C for 5 min.
Add 200 µL of Solution III.
Mix by inverting several times.
Incubate at 37°C for 5 min.
Centrifuge tubes at 14000 rpm for 10 min.
Collect supernatant in new tube (around 500 µL).
Incubate at 37°C for 5 min.
Add 1 mL of EtOh 100% (-20°C).
Incubate at -20°C (10 minutes – 2 hours)
Centrifuge solution at 14000 rpm for 10 min.
Pour out supernatant.
Open and invert tubes on a paper towel to drain them out.
Add 200 µL of EtOH 70%. (-20°C)
Centrifuge solution at 14000 rpm for 5 min.
Pour supernatant in sink.
Dry pellet at 37°C for 5-20 min. (Until pellet is completely dry).
Add 30 µL of DNAse free water
Resuspend with vortex
Incubate at 37°C for 20 min.
Measure concentrations at Nanodrop.
Waste disposal
Waste
Place to dispose
What to avoid
Supernatant
Sink after adding bleach.
Pipet tips
Trash after adding bleach.
Eppendorf tubes
Trash after adding bleach.
Safety notes:
Wear globes
Use lab coat
References:
Made by: Ana Karen Tello
Date: 29/08/15
Protocol 8: Bacmid Transformation
Objective
Generate recombinant bacmid by transforming DH10Bac E.coli
Approximate Time
300 min
Material
S.O.C. Medium
LB agar plates containing kanamycin, gentamicin, tetracycline, Bluo-gal and IPTG.
15 ml round-bottom polypropylene tubes.
Equipment and Apparatus
Name
37ºC incubator with shaker.
Water bath
Previous Steps
Water bath
Autoclave LB-agar media
Equilibrate a water bath to 42°C.
Warm selective plates at 37°C for 30 minutes.
Warm the S.O.C. Medium to room temperature.
Pre-chill one 15 ml round-bottom polypropylene tube for each transformation.
Related Protocols
Miniprep
Steps
For each transformation, gently mix and transfer 100 μl of the DH10Bac cells into the 15 ml tubes.
Add the appropriate amount of plasmid DNA to the cells and mix gently.
Construct: 5 μl
Control: 5 μl
Incubate cells on ice for 30 minutes.
Heat-shock the cells for 45 seconds at 42°C without shaking
Immediately transfer the tubes to ice and chill for 2 minutes.
Add 900 μl of room temperature S.O.C. Medium.
Shake tubes at 37°C at 225 rpm for 4 hours.
Prepare dilutions of the cells (10-1, 10-2, 10-3) with S.O.C. medium.
Plate 100 μl of the transformation onto a LB agar plate containing the appropriate antibiotics.
Incubate plates for 48 hours at 37°C.
Waste disposal
Waste
Place to dispose
Petri dishes with culture.
Place in a bag to sterilize.
Liquid LB with culture.
Place in a container and with bleach then sterilize.
References
Invitrogen. Bac-to-Bac® Baculovirus Expression System: An efficient site-specific transposition system to generate baculovirus for high-level expression of recombinant proteins. (2004, April 6).
Made by
Carolina Elizondo
Date
08/29/2015
Protocol 9: Transformation
Objective
Insert plasmid DNA in calcium competent cells.
Approximate Time
90 min.
Material
Liquid LB Media.
Petri dishes.
Equipment and Apparatus
Name:
Incubator with shaker.
Laminar flow safety cabinet.
Water bath.
Previous steps
Autoclave LB liquid media and LB-agar media.
Related protocols
Miniprep
Steps
Take competent cells from -80ºC freezer.
Turn on water bath at desired temperature.
Make sure you have 70 μl of competent cells on a eppendorf tube. Keep them on ice.
Add 50ng of DNA (1-2 μl).
Place the competent cell/DNA mixture on ice for 20min.
Heat shock each transformation tube by placing the bottom of the tube into a 42°C water bath for 45 seconds (time varies depending on the competent cells you are using).
Put the tubes back on ice for 2 min.
Add 500μl LB media (without antibiotic) and grow in 37°C shaking incubator for 1 hour.
Plate 100 μl of the transformation onto a LB agar plate containing the appropriate antibiotic.
Incubate plates at 37°C overnight.
Waste Disposal
Waste
Place to dispose
Petri dishes with culture.
Place in a bag to sterilize
Liquid LB with culture.
Place in a container and wit bleach then sterilize.
Safety Notes
Every step must be done in the safety cabinet to avoid contamination.
Gloves must be worn at all times.
Use lab coat.
References
Seidman, C. E., Struhl, K., Sheen, J., & Jessen, T. (2005). “Basic Protocol 1: Transformation Using Calcium Chloride.” Introduction of plasmid DNA into cells. Current protocols in molecular biology, 1-8.
Made by
Carolina Elizondo
Date
08/29/15
Protocol 10: Passaging Suspension Cells
Objective
To continue providing cell culture enough nutrients and space for their proper growth.
Approximate time
0-60 minutes
Material
Culture vessels containing your suspension cells.
Shaker flasks without baffles.
Complete growth medium, pre-warmed to 27°C.
Reagents and equipment to determine viable and total cell counts.
Equipment and Apparatus
Name
37°C incubator with humidified atmosphere of 5% CO2
Shaking platform
Previous steps
Growth of cells in shaker flasks before reaching confluency.
Related protocols
Cell counting
Steps
When the cells are ready for passaging (i.e., log-phase growth before they reach confluency), remove the flask from the shaking incubator, and take a small sample from the culture flask using a sterile pipette. If cells have settled                       down before taking the sample, swirl the flask to evenly distribute the cells in the medium.
From the sample, determine the total number of cells and percent viability using a hemocytometer, cell counter, and Trypan Blue exclusion or an automated cell counter.
Calculate the volume of media that you need to add to dilute the culture down to the recommended seeding density.
Aseptically add the appropriate volume of pre-warmed growth medium into the culture flask. You may split the culture to multiple flasks if needed.
Loosen the caps of the culture flasks one full turn to allow for proper gas exchange (or use a gas-permeable cap), and return the flasks to the shaking incubator.
Waste Disposal
Waste
Place to dispose
Cell waste
Sink after adding bleach
Safety Notes
Wear gloves.
Use lab coat.
References
Cell Culture Basics Handbook Thermo-Fischer Scientific
Made By
Dania Bazúa Gerez
Date
08/29/2015
Protocol 11: Cell Counting with a Hemocytometer
Objective
Determine the amount of media needed for passaging of suspension cells.
Approximate time
45-60 minutes
Material
Hemocytometer
Trypan blue
Eppendorf tube
Balanced salt solution (i.e. DPBS solution)
Equipment and Apparatus
Name
37°C incubator with humidified atmosphere of 5% CO2
Shaking platform
Previous steps
Growth of cells in shaker flasks before reaching confluency.
Related protocols
Passaging suspension cells
Steps
Ensure the hemocytometer is clean using 70% ethanol.
Make sure the cell suspension to be counted is well mixed by either gentle agitation of the flask containing the cells.
Before the cells have a chance to settle take out about 1 mL of cell suspension using a serological pipette and place in an Eppendorf tube.
Prepare a cell suspension in a balanced salt solution (i.e. DPBS solution). Add 0.05 mL of salt solution and 0.05 mL of cells. Add 0.01 mL of trypan blue to the solution (dilution factor=4).
Draw up some cell suspension containing trypan blue. Carefully fill the hemocytometer by gently resting the end of the micropipette tip at the edge of the chambers. Take care not to overfill the chamber.
Focus on the grid lines of the hemocytometer using the 10X objective of the microscope and focus on one set of 16 corner squares.
Count the number of cells in this area of 16 squares. When counting, always count only live cells that look healthy (unstained by trypan blue). Count cells that are within the square and any positioned on the right hand or bottom boundary                       line.
Dead cells stained blue with trypan blue can be counted separately for a viability count.
Move the hemocytometer to another set of 16 corner squares and carry on counting until all 4 sets of 16 corner squares are counted. Adittionally you can count the center square of 25 squares.
Use formulas to perform appropriate calculations:
CELLS PER ml = the average count per square *dilution factor *10**4 (count 10 squares)
TOTAL CELLS = cells per ml* the original volume of fluid from which cell sample was removed.
CELL VIABILITY (%) = total viable cells
(unstained) ) / total cells (stained and unstained) *100.
Waste disposal
Waste
Place to dispose
What to avoid
Solution from hemocytometer
Appropriate container for cancergenic/mutagenic waste
Contact
Safety Notes
Wear gloves.
Use lab coat.
References
Sigma “Use of Trypan Blue Stain and the Hemocytometer to Determine Total Cell Counts and Viable Cell Number” Abcam “Counting cells using a hemocytometer”
Made by
Dania Bazúa Gerez
Date
08/29/2015
Protocol 12: Transfection of Insect Cells
Objective
Insertion of the recombinant bacmid DNA in the Sf9 cells for the production of the recombinant baculovirus.
Approximate time
5 hours (including incubation time)
Material
Purified recombinant bacmid DNA construct (500 ng/µL in TE buffer, pH 8.0).
Sf9 cells.
Cellfectin II Reagent (store at 4° until use).
Grace’s Insect Cell Medium, Unsupplemented (no supplements, FBS, antibiotics).
24-well tissue culture plates.
1.5-mL sterile microcentrifuge tubes.
Complete growth medium for culturing insect cells (Sf-900 II SFM).
Equipment and Apparatus
Bacteriological Hood.
Humidified Incubator at 27°C.
Previous steps
Cell counting/Subculturing to assure Sf9 cells are in the log phase (1.5–2.5 × 106 cells/mL) with greater than 95% viability
Bacmid MiniPrep to have the DNA extracted and quantified for the transfection
Related protocols
Isolation of P1 viral stock.
Amplification of baculoviral stock.
Viral Plaque Assay.
Steps
Verify that the Sf9 cells are in the log phase (1.5–2.5 × 106cells/mL) with greater than 95% viability.
f the cell density is in range of 1.5 × 106–2.5 × 106 cells/mL and the culture is without antibiotics, proceed to step 2a. If the cell density is not in this range or the cell culture contains antibiotics, follow steps 2b–2c:
Add 500 μL of Grace’s Insect Medium, Unsupplemented (without antibiotics and serum) in each well. Seed 8 × 105 Sf9 cells from Step 1 per well. Do not change medium or wash the cells. The medium carried over will enhance the transfection                           efficiency. Allow cells to attach for 15 minutes at room temperature in the hood. Proceed to step 3.
Prepare 10 mL plating medium by mixing 1.5 mL Supplemented Grace’s Insect Medium containing 10% FBS (without antibiotics) and 8.5 mL Grace’s Insect Medium, Unsupplemented (without FBS and antibiotics).
Plate 8 × 105 Sf9 cells from Step 1 per well. Allow cells to attach for 15 minutes at room temperature in the hood. Remove the medium. Add 550 μL plating medium from step 2b per well. Proceed to step 3.
For each transfection sample, prepare complexes as follows:
Mix Cellfectin® II before use, and dilute 2 μL in 25 μL Grace’s Medium, Unsupplemented (without antibiotics and serum). Vortex briefly to mix. Note: You may leave this mixture at room temperature for up to 30 minutes.
Dilute 1 μL baculovirus DNA in 25 μL Grace’s Medium, Unsupplemented (without antibiotics and serum). Mix gently.
Combine the diluted DNA with diluted Cellfectin® II (total volume ~50 μL). Mix gently and incubate for 15–30 minutes at room temperature.
Add ~50 μL DNA-lipid mixture or transfection mixture (Step 3c) dropwise onto the cells from Step 2. Incubate cells at 27°C for 3–5 hours.
Remove the transfection mixture and replace with 550 μL of complete growth medium (e.g., Grace’s Insect Medium,Supplemented and 10% FBS). Using antibiotics is optional.
Incubate cells at 27°C for 72 hours or until you see signs of viral infection
Waste disposal
Waste
Place to dispose
What to avoid
Growth Media
Bleach and dispose in the sink
N/A
Transfection Mixture
Bleach and dispose in the sink
N/A
Safety notes
Every step must be done in the hood to avoid contamination.
Gloves must be worn at all times.
References
Life Technologies Corporation. (2013). Bac-to-Bac® Baculovirus Expression System . Carlsbad, CA, USA: Life Technologies
Made by
Claudia Nallely Alonso Cantú
Date
August 29, 2015
Protocol 13: Viral Plaque Assay
Objective
Perform a viral plaque assay in order to watch the presence of recombinant baculovirus and calculate the titer.
Approximate time
418 - 490 min
Material
Name (Supplier)
15-ml tubes.
12-well plate.
50-ml tube.
Sf-900 Medium 1X.
Agarose gel.
Equipment and Apparatus
Name:
Humidified incubator.
Laminar flow cabinet.
Inverted microscope.
Water bath (Thermo Scientific).
Previous steps
Transfection of Sf9 cells.
Viral stock isolation or amplification of baculovirus stock.
Related protocols
Transfection of Sf9 cells.
Amplification of baculovirus stock.
Viral stock isolation.
Plaquing medium.
Calculating the titer of baculoviral stock.
Steps
On the day of infection, harvest Sf9 cells and prepare a 15 ml cell suspension 5×105 cells / ml in Sf-900 II SFM.
Aliquot 0.67 ml of cell suspension into each well of two 12-well plates, including a negative control.
Allow the cells to settle to the bottom of the plate and incubate, covered, at room temperature for 1 hour.
Prepare the plaquing medium. Melt 4% agarose gel in 10 ml by placing it on 50-ml in a 70ᵒC water bath for 20 to 30 minutes or heating the agarose in a microwave oven.
While the 4% agarose gel is melting, place the following in the 40ᵒC water bath:
Empty, sterile 50-ml tube.
Sf-900 Medium (1X).
Once the 4% agarose gel has liquefied, move the agarose gel, medium, and empty 50-ml tube to a sterile hood.
Working quickly, prepare the plaquing medium as follows:
Combine 30 ml of Sf-900 Medium (1X) and 10 ml of melted 4% agarose gel in the empty 50-ml tube and mix gently.
Return the bottle of plaquing medium to the 40ᵒC water bath until use.
After 1 hour, observe the cell monolayers using an inverted microscope. Sf9 cells should be attached and at 50% confluence.
Prepare an 8-log serial dilution 10-1 to 10-8 of the clarified baculoviral stock in Sf900 II SFM, diluting 0.25 ml of the baculoviral stock or previous dilution in 2.25 ml of medium in 15-ml tubes.
Move the 12-well plates containing Sf9 cells and the tubes of diluted virus to the sterile hood.
Label the plates as follows: no virus (negative control), 10-4, 10-5, 10-6, 10-7, 10-8.
Remove the medium from each well, discard, and immediately replace with 0.67 ml of the appropriate virus dilution. As a negative control, add the appropriate medium without virus.
Incubate cells with virus for 1 hour at room temperature.
Move the cells and the tube of plaquing medium from the 40ᵒC water bath to a sterile hood.
Sequentially starting from the highest dilution to the lowest dilution, remove the medium containing the virus from the wells and replace with 0.67 ml of plaquing medium. Work quickly to avoid dessication of the cell monolayer.
Allow agarose overlay to harden for 1 hour at room temperature before moving the plates.
Incubate the cells in a 27ᵒC humidified incubator for 7-10 days until plaques are visible and ready to count.
OPTIONAL: Stain plaques to facilitate counting and calculate the titer.
Waste disposal
Waste
Place to dispose
Supernatant
Trash after autoclaving.
Pipet tips
Trash after adding bleach.
Eppendorf tubes
Trash after adding bleach.
Plaquing Medium
Trash.
Dilutions
Trash after autoclaving.
Safety notes
Wear gloves.
Use lab coat.
References
Bac-to-Bac® Baculovirus Expression System (Invitrogen by LifeTechnologies).
Made by
Ana Karen Carrizales
Date
08/29/15
Protocol 14: Viral stock isolation
Objective
Isolate P1 viral stock from transfected Sf9.
Approximate time
20 min.
Material
Eppendorf tubes.
Aluminium foil.
Equipment and Apparatus
Name
Centrifuge
Refrigerator (-20 and -80ᵒC)
Laminar flow cabinet
Previous steps
Transfection of Sf9 cells.
Related protocols
Transfection of Sf9 cells.
Amplification of baculovirus stock.
Viral plaque assay.
Plaquing medium.
Calculating the titer of baculoviral stock.
Steps
Look if transfected cells demonstrate signs of late stage infection (24 - 72 hours post-transfection).
Collect the medium containing virus from each well (0.5 ml) and transfer to sterile 1.5-ml tubes.
Centrifuge the tubes at 500 x g for 5 minutes to remove cells and large debris.
Transfer the clarified supernatant to fresh 1.5-ml tube.
Store at 4ᵒC, protected from light.
Waste disposal
Waste
Place to dispose
Supernatant
Sink after adding bleach.
Pipet tips
Trash after adding bleach.
Eppendorf tubes
Trash after adding bleach.
Safety notes
Wear gloves.
Use lab coats.
References
Bac-to-Bac® Baculovirus Expression System (Invitrogen by LifeTechnologies).
Made by
Ana Karen Carrizale
Date
08/29/2015
Results
Discussion
Conclusions
References
