"
Page
Discussion
View source
History
teams
Log in
 
Team:Tec-Monterrey/Geneticframe
From 2010.igem.org
Tec de Monterrey
  Home
  Team
Undergraduates
Instructors/Advisors
Attributions
Headquarters
  Project
Introduction
Genetic Frame
Parts
Research
  Documentation
Journal
Protocols
Safety & Ethics
Human Practices
  Bonus
Gallery
Wiki Wiki
  Sponsors
Platinum
Gold
Silver
Special Thanks
Development of the genetic frame for an “Intelligent Biosensor”
Phage Activators
The Proteins
The Promoters
The Sensitivity Tuners
Characterization
MuC sensitivity tuner (BBa_K427005)
MuMor sensitivity tuner (BBa_K427006)
Transfer Function Parameters
References
Development of the genetic frame for an “Intelligent Biosensor”
After deciding we wanted to make a concentration sensitive biosensor, we began to investigate the different ways we could achieve our goal. After an initial research concerning biological reporters and the mechanisms currently used to make them, we began to read previous iGEM projects to look for useful BioBricks that could be applied to our work. Two past iGEM projects were of our particular interest: British Columbia’s 2009 project and Cambridge’s 2007 project.
British Columbia’s project
In the past iGEM competition The University of British Columbia team developed a project that resembled our initial proposition. They wanted to produce an E. coli that expressed different Fluorescent Proteins depending on the concentration of arabinose in the medium. Basically, they proposed a system that used three different sensitive promoters to activate a “Lock and Key” mechanism that enabled the expression of the Fluorescent Protein. There were 3 promoters: the wild type pBAD promoter that is sensitive to arabinose, and two others called pBAD weak and pBAD strong that were designed by the team via mutagenesis. As their name implies, the weak promoter is sensitive only to higher concentrations of arabinose and the strong promoter is sensitive to lower concentrations of arabinose compared to the wild type. Finally, to prevent the expression of all the fluorescent proteins when there was enough arabinose in the medium to activate all three promoters, they developed a “jammer” that consisted in placing a reverse promoter on the other side of the fluorescent protein. Thus, when both promoters were activated, the production of fluorescent protein ceased.
(For more information about this project, you can consult their wiki )
Cambridge’s project
In 2007, the University of Cambridge iGEM team worked in developing both intracellular and intercellular communication pathways as well as a Gram-positive platform for synthetic biology. We were particularly interested in the intracellular project. It consisted of a PoPS amplifier made with transcriptional activators and promoters that were obtained from bacteriophage components. The Cambridge team decided to use activators from bacteriophage components because they are not common in the cellular signals of bacteria. In this particular case, they used activator and promoters from the bacteriophage PSP3 and fR73.
(For more information about this project, you can consult their wiki )
After reading about these projects we decided that we could adapt our idea and build upon the achievements of those two previous teams. We wanted to contribute in a significant way, so we began by analyzing their results and identifying points that could use more work. We found that, in general, British Columbia’s proposal was complicated and apparently ineffective. The problem seemed to be in the “lock and key” mechanism, which didn’t seem to behave as expected. On the other hand, we found that their “jammers” could be very useful and they seemed to work as expected.
So we began to design a different genetic frame to create “intelligent biosensors”, one that preferably didn’t contain a “lock and key” mechanism. Our solution is composed by a combination of parts of the two previously mentioned projects. We incorporated the “jammer” mechanism in addition to PoPS amplification systems in order to create a flexible genetic construction for concentration sensitive bio-reporters. Our proposed solution can be seen in Figure 1 (click on it to enlarge):
Figure 1
In theory, this genetic construction is able to detect different concentrations of a compound provided that you have three different promoters that are sensitive to different concentrations of the substance. Based on the amount of compound detected, the bacteria will produce one of three reporter proteins. For our project, we decided to use the same sensors and reporter proteins that the University of British Columbia team used in their 2009 iGEM project. Therefore we began with 3 promoters: pBAD wild type, pBAD weak and pBAD strong; each promoter was able to detect arabinose at different concentrations. Our reporter proteins were GFP, YFP, and RFP (Cherry). We arranged the constructions so that GFP is produced when the cells were induced with low concentrations of arabinose, YFP is produced when induced with medium concentrations and RFP (Cherry) is produced when there is a high concentration. The mechanism that enables this behavior will be explained in two parts, the first part involves the detection of the compound and the second part concerns the production of the reporter protein.
The first part of the genetic construction is responsible for the detection of arabinose. This is achieved by using a series of promoters from the pBAD family that were developed and characterized by British Columbia. Each promoter is activated at a different concentration of arabinose, the strong promoter requires a higher concentration of arabinose than the weak promoter. When the promoter is activated, it enables the expression of a bacteriophage activator that works as a PoPS amplifier. The first amplifier we used was the one provided by the Cambridge 2007 team.  However, for our construction to work, we required two additional amplifiers. So we did some more research on bacteriophage activators and constructed two new PoPS amplifier systems in order to finish our construction. We consider these new amplifiers to be a valuable addition to the Registry of Standard Biological Parts, because they come from different bacteriophage families, this means that they produce different activator proteins and they can be used at the same time without causing interference.
Eventually we realized that if we didn’t have a mechanism that disabled the expression of the PoPS amplifiers, when the cells were induced with high concentrations of arabinose, all three reporter proteins would be produced. Once again, incorporating the previous advances of the 2009 British Columbia Team, we were able to overcome this obstacle. The solution consisted in implementing their “jammer” systems, in order to suppress the expression of the PoPS amplifiers as the concentration of arabinose in the medium increases. The full mechanism is explained in the following figures:
Figure 2: shows the sensor part of the final construct
Figure 2
Figure 3: shows the sensor when there is a low concentration of arabinose
Figure 3
Figure 4: shows the sensor when there is a medium concentration of arabinose
Figure 4
Figure 5: shows the sensor when there is a high concentration of arabinose
Figure 5
The second part of the construction is in charge of producing the reporter proteins. According to our research on biological sensors, the reporter molecule must be easy to detect and it must be a substance that is not naturally produced by the organism. Fluorescent proteins are great reporter proteins because they fulfill both of these requirements. The team from British Columbia University used three different fluorescent proteins in their project in the 2009 competition: Green Fluorescent Protein (GFP), Yellow Fluorescent Protein (YFP), and the Cherry variety of the Red Fluorescent Protein (RFP Cherry). We ultimately decided to use the same three molecules because they have been widely characterized and there is plenty of information regarding their fluorescence spectrum and behavior.
When the sensing part of our construction produces one of the bacteriophage activators, this molecule enables the expression of one of the reporter proteins. Because these promoters produce very high PoPS numbers, the amount of reporter molecule produced is elevated, even though the arabinose concentration isn’t. The following figure is a detail that explains the mechanism behind the production of reporter proteins.
Figure 6: shows the reporter part of the final construct.
Figure 6
Phage Activators
The most basic bio-bricks created in our project were sequences from the genome of the Mu bacteriophage that attacks E. coli. We needed some sort of sensitivity tuner for our biosensor and the registry had only one family of promoters and protein activators that could be used for the construction of these devices. After some research we came to Mu bacteriophage, where we found two different families of promoters and protein activators: Mor protein – Pm promoter and C protein - Plys, Pi, Pp and Pmom promoters. We took the reported sequences of Mor, Pm, C (Kumaraswami & Howe, 2004) and Pmom (Jiang, 2008) . The proteins sequences were optimized for their expression in E. coli with Mr. Gene’s optimization algorithm; we added an Elowitzribosome binding site and finally the BioBrick prefix and suffix to create the bio-bricks that we used. We only added the prefix and suffix to the promoters sequences.
The Proteins
Mor and C are both proteins encoded in the genome of Mu bacteriophage, and they have similar characteristics. Both are proteins that bind to DNA in a specific sequence, both have about the same size, sequence and have similar secondary structures and domains.  Figure 7 (Mathee, 1990) shows the amino acid sequence alignment of Mor and C proteins.
Figure 7
In that image we can see the similarity between these two proteins, but we can also notice that the helix-turn-helix motifs are quite different. It is hypothesized that these domains are the ones that recognize and interact with the DNA sequence. According to Kumaraswami (2004) and Jiang (2008) two domains can be observed on each of the proteins a Helix-Turn-Helix (HTH) domain and a dimerization domain, which suggests that the proteins can only activate their promoters when in dimers.
The final bio-bricks were: Mor ( BBa_K427002 ) and C ( BBa_K427001 )
The Promoters
The first family of activators that we used was the Mor-Pm family. Pm promoter is the middle promoter of the Mu bacteriophage, where it initiates the transcription of C protein. The second family had four promoters: Plys, Pi, Pp and Pmom. We chose Pmom for our sensitivity tuners because out of the four promoters it is the most characterized in literature. It is normally activated by the C protein.
The activation of these promoters requires the holoenzyme σ70, which normally binds to the -35 and -10 consensus sequences. These promoters have the -10 but do not have the -35 sequences. Instead, they have another sequence approximately at -51 where the activator protein binds. Once the protein is located there, the holoenzyme σ70 recognizes the promoter and transcription begins.
The final bio-bricks were: Pm ( BBa_K427003 ), Pmom ( BBa_K427004 )
The Sensitivity Tuners
These devices were designed with two purposes in mind: increase the PoPS output of the pBad promoter (any promoter would work) and change its almost linear production into an almost Boolean one. These parts have the following structure: activator protein, transcriptional stop and phage promoter. The sensitivity tuners can be used to achieve both purposes on any construction, they just have to be inserted between the desired promoter and the desired coding sequence.
In our project we used three sensitivity tuners: one made by Cambridge in previous iGEM competitions; this one is built with phiR73 delta activator ( BBa_I746352 ), a bidirectional transcriptional terminator ( BBa_B0014 ) and Po phage promoter ( BBa_I746361 ). The other two were built by us using the Mor and C families. They are MuC sensitivity tuner ( BBa_K427005 ) and MuMor sensitivity tuner ( BBa_K427006 ). The first one includes MuC protein activator ( BBa_K427002 ), bidirectional terminator ( BBa_B0014 ) and Pmom promoter ( BBA_K427004 ). The other includes MuMor protein activator ( BBa_K427001 ), bidirectional terminator ( BBa_B0014 ) and Pmom promoter ( BBA_K427003 ).
Figure 8: MuC sensitivity tuner ( BBa_K427005 )
Figure 8
Figure 9: MuMor sensitivity tuner ( BBa_K427006 )
Figure 9
Characterization
We characterized both sensitivity tuners using Hill equations:
In order to do this we had to build another two parts which were only used to characterize the sensitivity tuners. They consist of a bidirectional terminator ( BBa_B0014 ), PBad Weak ( BBa_K206001 ), the sensitivity tuner ( BBa_K427005 or BBa_K427006 ) and GFP reporter ( BBa_E0840 ). These parts are BBa_K427007 for the MuC sensitivity tuner and BBa_K427008 for the MuMor sensitivity tuner.
All the characterizations were made in the BW27783 strain of E. coli. This strain was modified to internalize arabinose so that the concentration inside the cell is the same as the concentration outside. Another modification is that BW27783 does not use or degrade arabinose.The graphs show the data used to obtain the coefficients reported in the table. The protocol we used is described in the Protocol section of our wiki.
MuC sensitivity tuner (BBa_K427005)
MuMor sensitivity tuner (BBa_K427006)
Transfer function parameters
References
Jiang, Y. a. (2008). Regional mutagenesis of the gene encoding the phage Mu late gene activator C identifies two separate regions important for DNA binding. NucleicAcidsResearch, 6396–6405.
Kumaraswami, M., & Howe, M. a.-W. (2004). Crystal Structure of the Mor Protein of Bacteriophage Mu, a Member of the Mor/C Family of Transcription Activators. The Journal of biological Chemistry, 16581–16590.
Mathee, K. a. (1990). Identification of a Positive Regulator of the Mu Middle Operon. Journal of bacteriology, 6641-6650.
Retrieved from " http://2010.igem.org/Team:Tec-Monterrey/Geneticframe "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
