"
Page
Discussion
View source
History
teams
Log in
 
Team:Peking S/lab/notebook/shd
From 2011.igem.org
Handuo.Shi
Dingqiao.Wen
Yiwei.Chen
Yanrong.Ma
Hanchi.Lin
Wenhan.Luo
Jingchen.Feng
Jingyi.Xi
Xiao.Qin
Rui.Sun
Linna.Deng
Weiyue.Ji
Home
Template:Https://2011.igem.org/Team:Peking S/bannerhidden
无标题文档
css r corner
Handuo Shi's Notebook
Summary
Concentrated on a feedback gene network, I have constructed it as well as demonstrated its effectiveness.  Moreover, I helped in the modeling of our system and fulfilled an algorithm so that we can construct our boolean logic gates in the simplest way.
Contents
June, 2011
July, 2011
August, 2011
September, 2011
October, 2011
June
Mon
Tue
Wed
Thu
Fri
Sat
Sun
-
-
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
-
-
[ TOP ]
6.22 - 6.25
Design primers.
6.26 - 6.27
Culture streptomycin griseus strain, which is the source of afsA, arpA, padpA.
6.28
afsA, arpA PCR using different annealing temperature, with gradient 4 ℃.
Identify them by 1% agarose gel electrophoresis.
Excise the gel slice and extract the fragments.
Second PCR using the products of gel extraction.
Electrophoresis PCR reaction system in 1.5% agarose gel.
Double digest the products of gel extraction by EcoR1-HF and Xba1.
Double digest the plasmid of B0015 by EcoR1-HF and Spe1.
Ligase afsA and arpA to the vector  of B0015.
6.29
Transform the ligation reaction system.
Sythesis the genes of CAI-1 system using DNAWorks assembly and PadpA PCR.
Identify them by 1% agarose gel electrophoresis.
Excise the gel slice of pqrr, J23106+B0034+luxU, PadpA and extract the fragments.
ligase the products of gel extraction to the vector pEASY-Blunt.
6.30
Transform the ligation reaction system.
Pick six clones each plate and shave at 37℃ to amplify the bacteria.
July
Mon
Tue
Wed
Thu
Fri
Sat
Sun
-
-
-
-
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
[ TOP ]
7.1
Miniprep of the plasmids which contain pqrr, J23106+B0034+luxU, padpA gene respectively and send for sequencing.
7.3
Get the result of sequencing and the sequence of J23106+B0034+luxU has a mutation which need to be corrected. Design primers for J23106+B0034+luxU mutation.
7.5
Get mutation primers and mutate the plasmid using TaKaRa MutanBEST Kit.
7.6
Pick six clones each plate and shave at 37℃ to amplify the bacteria. Miniprep of the plasmids.
7.7
Send the plasmid of J23106+B0034+luxU mutants for sequencing.
7.8 - 7.15
As for the initial results of DNAWorks assembly for cqsS, cqsA, luxO were not good, we changed the reaction conditions and tried many other alternative method. However, the consequence still remain disappointing. So we decide to get the strain of Vibrio cholerae and use the method of PCR:
cqsS, cqsA, luxO PCR using different annealing temperature, with gradient 4 ℃.
Identify them by 1% agarose gel electrophoresis.
Excise the gel slice of cqsS, cqsA, luxO and extract the fragments.
ligase the products of gel extraction to the vector pEASY-Blunt.
Transformation.
7.16
Send the plasmid of  cqsS, cqsA, luxO for sequencing.
Since there are some EcoR1, Pst1, Xba1 restriction sites in the original sequence of cqsS, cqsA, luxO, We design primers for point mutatiion to eliminate the restriction sites.
7.18
Get mutation primers and mutate the plasmid using TaKaRa MutanBEST Kit.
7.19
Pick five clones each plate and shave at 37℃ to amplify the bacteria.
Miniprep of the plasmids.
7.20
Send the plasmid of cqsS, cqsA, luxO mutants for sequencing.
7.22
Get the result of sequencing.
Make the second point mutation in cqsS.
7.23
Pick five clones in cqsS mutants plate and shave at 37℃ to amplify the bacteria.
Miniprep of the plasmids.
7.24
Send the plasmid of cqsS mutants for sequencing.
August
Mon
Tue
Wed
Thu
Fri
Sat
Sun
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
-
-
-
-
[ TOP ]
7.25 - 8.12
1.Put the double digested fragments of J23106+B0034+luxU, cqsS, luxO in the ligation reaction system in 16℃ for 2-4 hours, then add the EcoR1 and Pst1 double-digested vector pSB1C3 for 4 hours.
Transform the ligation reaction system.
16-18 hours later, pick six to ten clones in the plate and shave at 37℃ to amplify the bacteria. Miniprep of the plasmids.
2. Construct pBAD+B0034+cqsA+B0015 plasmid in pSB1AC3 backbone.
3. Construct  Pqrr+B0030+GFP+ssrA degredation tag in pSB4A5 backbone.
8.13 - 8.17
Characterize CAI-1 system
8.18
Transform of luxR(1-8O, 936bp, I0462), plux'(1-14P, 30bp, R0061), luxI(3-14A, 711bp, K081015)
8.19
Retransform of plux', luxI; pc+luxR
8.20
Ligation of plux'+T7ptag(2673 bp)+terminator, pT7 (I719005) +luxI.
8.21
CAI-1 induce of the CAI-1 system.
8.22
Ligation of pc+luxR, transform of luxI(1-14C, 661bp, C0261)
8.23
pT7+luxI 1,3,5 sequencing right.  Digestion of luxI(1-14C)
8.24
Ligation of pT7+luxI(no terminator), (pBAD+supD)+(plux'+T7ptag (6))+4K5.
8.25
pc+luxR(1-8O) was wrong in part!
Transform of luxR(2-4O,799bp,J37033)
8.26
CAI-1 induce of the CAI-1 system.
8.27
pT7+luxI(no term). No. 1, 3 sequencing right.
Ligation of pT7+luxI + GFP(ssrA tag) for sequencing.
Colony PCR, (1) 1,3,4,5 (3)1,2,3,4.
PCR for 1-4O, 1-8O(in 2009 Distribution).
Digest 1-2M(RBS).
Transform of pc(J23100, 1-18C), luxR(1-4O).
8.28
Ligation of pc+1-8O.
(pBAD+supD)+(plux'+T7ptag (6))+4K5 sequencing wrong!  Digest (pBAD+supD) again.
8.29
pT7+luxI+GFP(ssrA tag) sequencing wrong (No luxI?!)
Ligation again for 3A (psb1C3).  Ligation for RBS+luxR(1-4O).
Ligation  (pBAD+supD)+(plux'+T7ptag (6))+4K5 again. (btw T7ptag sequencing right).
8.30
(pBAD+supD)+(plux'+T7ptag (6))+4K5 16, 32 for sequencing.
pT7+luxI+GFP(ssrA tag) colony PCR: (1). 7, 8, 10; (3). 1, 3, 4, 5, 6, 7, 9, 10.
Digest RBS+luxR(1-4O).
8.31
pT7+luxI+GFP(ssrA tag) Digest check: (1). 7, 8, 10; (3). 4, 6, 7, 9. sent for sequencing.
Ligation of pC+RBS+luxR send for sequencing.
September
Mon
Tue
Wed
Thu
Fri
Sat
Sun
-
-
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
-
--
[ TOP ]
9.1
RBS+luxR sequencing right in the end!
Religation of (pBAD+supD)+(plux'+T7ptag (6))+4K5 and (pBAD+supD)+(plux'+T7ptag (6))+3C5, doubt whether (pBAD+supD) has been right, redigest.
9.2
pT7+luxI+GFP(ssrA tag) sequencing right.
Ligation of (pT7+luxI+GFP)+(pC+RBS+luxR) psB1A3.
Religation of (pBAD+supD)+(plux'+T7ptag (6))+4K5 and (pBAD+supD)+(plux'+T7ptag (6))+3C5.
(pBAD+supD)+(plux'+T7ptag (6))+4K5 (43) sequencing right!
9.3
(pT7+luxI+GFP)+(pC+RBS+luxR) psB1A3 colony PCR right.
Double transform for induce.
9.4
Induce of the feedback circuit.
9.5
Induce of the feedback circuit.
(pT7+luxI+GFP)+(pC+RBS+luxR) psB1A3 sent for sequencing.
9.6 - 9.10
Mutate feedback circuit to select the one works best in our system.
9.11 - 9.30
Characterize feedback loop.
October
Mon
Tue
Wed
Thu
Fri
Sat
Sun
-
-
-
-
1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
-
-
-
-
-
-
[ TOP ]
10.1
10.3
10.4
10.5
10.7
10.8
10.9
10.10
10.11
10.12
10.13
10.15
10.16-10.21
10.21-10.25
 
Retrieved from " http://2011.igem.org/Team:Peking_S/lab/notebook/shd "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
