<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SKLMT-China_Model_Process skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SKLMT-China/Model Process</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="banner-content"><H2 class="title"> PROCESS</H2><P class="content">Details of the modelling work</P></DIV><DIV class="container"><DIV class="row"><OL class="breadcrumb"><LI><A href="https://2018.igem.org/Team:SKLMT-China"><I class="fa fa-home">Home</I></A></LI><LI><A href="Model"><I class="fa fa-rocket">Model</I></A></LI><LI class="active dropdown"><DIV class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><I class="fa fa-code">Process
                            </I></DIV></LI><LI><A href="#s1" class="scrolly">Background</A></LI><LI><A href="#s2" class="scrolly">Assumptions</A></LI><LI><A href="#s3" class="scrolly">Modelling Process</A></LI><LI><A href="#s4" class="scrolly">Model Improvement--Algorithm optimization</A></LI><LI><A href="#s5" class="scrolly">Model analysis and evaluation</A></LI><LI><A href="#s6" class="scrolly">References</A></LI></OL></DIV></DIV><DIV class="container"><DIV class="paragraph shadow"><H2 class="title" id="s1">Background</H2><P>We have built a promoter library of inner promoters with various strength in <LATIN>pf-5</LATIN>. We characterized the strength by a ﬂuorescent reporter gene, <LATIN>firefly luciferase</LATIN>. Having gotten so many statistics, we thought about determine element for the strength of promoters. As we all know, strength composite promoters usually have high affinity with RNA polymerase related to the UP elements like −35 sequences and −10 sequences. We wonder the connection between the promoter sequence and its strength. So we use the resulting fluorescence data to construct a model with the promoter sequence. </P><P>1.To facilitate the promoter transformation of pseudomonas fluorescence in new site;</P><P>2.To reveal the correlation between promoter weights and their strength;</P><P>4.To improve the parameter setting of the promoter strength prediction in the species outside the large intestine with the method of position weight;</P><P>5.To provide reference for other microbe promoter strength prediction modeling.</P><H2 class="title" id="s2">Assumptions</H2><P>1. It is assumed that the promoter strength is only related to the core DNA sequence.In other words, the DNA sequence can uniquely determine the promoter strength.</P><P>2. It is assumed that the laboratory environment has no impact on the test results.In other words, our test data can fully reflect the promoter strength.</P><P>3.It is assumed that there is a simple monotonic relationship between the obtained fluorescence data and the promoter strength: the larger the fluorescence value is, the stronger the promoter can be considered.</P></DIV><DIV class="paragraph shadow"><H2 class="title" id="s3">Modelling Process</H2><H3 class="title">DNA Sequence Encoding</H3><P>The 5’-3’ region includes 70bp DNA information in each promoter. Currently, we have got 25 promoter sequences. We need to encode these 70 bases in order to establish a mathematical model. Notice that there’re only four types of bases(A,T,C and G),and calculating the numbers of each type is not enough to reflect interaction between adjacent bases.<SUP>[2]</SUP> Thus, we consider 3 continuous bases <LATIN>(Fig 1)</LATIN>.</P><P>We encode each three-base sequence with quaternary number(A,T,C,G represents quaternary number 0,1,2,3 ), and by running a C++ program (the whole code is shown in appendix), we get a vector with 64 dimensions from AAA to GGG, which calculates the frequency of appearance for each type in DNA sequence.</P><P>After encoding, all DNA sequences are transformed into a vector with 64 dimensions. The entire data is stored in the document <LATIN>“data.xlsx”</LATIN>. For example, the vectors of promoters ampC and araA are shown as follows:</P><P>At the end of the encoding step, we get an 25*64 matrix which reflects the features of these DNA sequences of each promoters. This encoding method has deep internal connection with the characteristics of DNA sequence, and it simultaneously contains the sequential relationship and details of DNA sequence. With this matrix, a mathematical model can be established.</P><H3 class="title">Model establishment and solution</H3><P>In order to simplify the problem, we assume that the connect between matrix and promoter strength is linear, in other words, the vector   \({x}\) exists when equation \({y=Ax}\) is satisfied.</P><P>Now, through a lot of experiments, we have measured a total of 24 values, and all twenty-two valid values are shown below. Each data is the average of the results of three parallel experiments.</P><P>The gap in original data is too wide, so we take the natural log of the original data as the value of the vector . Now, what we're going to do is to invert the linear relationship vector   \({x}\) by taking the linear transformation matrix   \({A}\) and the resulting vector  \({y}\).\({A,x}\) and  \({y}\)satisfy the following equation:  \({y=Ax}\),or  \({x=A^{-1}y}\).</P><P>This is a linear mapping of 64 variables but there are only 22 data points. In other words, this is an underdetermined equation which cannot be solved accurately by solving inhomogeneous linear equations. The sparse solution algorithm of underdetermined equations based on iteration3 will be adopted in the following part.</P><P>This is an effective algorithm to restore deterministic signals with fewer premeasured values.Our goal is to improve the precision of signal reduction as much as possible. Consider the following optimization problem:</P>
            $$min \left \| x \right \|_{0}$$
            $$s.t.\; \; \;   y=Ax$$
            <P> \(\left \| x \right \|_{0}\)is called   \(l^{0}\)norm, which represents the number of nonzero components in vector x in mathematics. This value is equivalent to this expression:</P>
   $$\lim_{\varepsilon \rightarrow 0}\sum_{i=1}^{64}\frac{x_{i}^{2}}{x_{i}^{2}+\varepsilon }, because\frac{x_{i}^{2}}{x_{i}^{2}+\varepsilon }= \begin{cases}
 &amp; \text{ 0, } x_{i}=0 \\ 
 &amp; \text{ 1, } x_{i}\neq 0 
\end{cases}
(\varepsilon \rightarrow 0)$$
            <P>The following iterative steps are obtained from the perspective of numerical calculation:</P><P>⑴At first , set the initial \({x^{(0)}=0}\),\({\varepsilon ^{(0)}=1}\)value</P><P>⑵After that, loop to calculate the current \({x^{(n+1)} }\),\({\varepsilon ^{(n+1)}}\)value  based on the previous value   \({x^{(n+1)} }\),\({\varepsilon ^{(n+1)}}\) with this formula:</P>

            $$x^{(n+1)}=arg\, min\left \{ \sum_{i=1}^{64}\frac{x_{i}^{2}}{(x_{i}^{(n)})^{2}+\varepsilon^{(0)} } \right\},(y=Ax);$$
            $$\varepsilon ^{(0)}=min\left \{ \varepsilon ^{(n)} ,\frac{min(x_{i}^{（n）})}{N}\right \}$$
            <P>⑶Finally, if   is less than a given bound such as 1e-6, abort this program, and regard current  \({x^{(n)}}\) as the solution vector of this underdetermined equation.</P><P>The explicit iterative form of the above steps is given below:</P>
            $$ f(x,\lambda )=\sum_{i=1}^{64}\frac{x_{i}^{2}}{{(}x_{i}^{(n)})^{2}+\varepsilon ^{(n)}}=\sum_{i=1}^{64}\frac{x_{i}^{2}}{{(}x_{i}^{(n)})^{2}+\varepsilon ^{(n)}}+\lambda (Ax-y)$$
            <P>we use Lagrange multiplier method to solve the minimum variable. Vector  \({\lambda }\)with 22 dimensions is introduced. So calculate the partial derivative to each component of  \({x}\),then solve this equation:</P>
            $$\frac{\partial f}{\partial x_{i}}=0$$
            <P>We find that the vector  \({x}\)can be iterated by this formula:</P>

            $$x^{(n+1)}=D^{(n)}{A}'(AD^{(n)}{A}')^{-1}y.$$
            <P>where we construct matrix   \({D^{(n)} }\)    as follow:</P>
$$D^{(n)}_{ij}= \begin{cases}
 &amp; \text{ 0, } i\neq j \\ 
 &amp; \text (x_{i}^{(n)})^{2}+\varepsilon ^{n},i=j 
\end{cases}$$

            <P>With the help of <LATIN>MATLAB</LATIN>, we wrote the program to find the final  \({x}\)value (the whole script and vector  \({x}\)is shown in appendix), and the corresponding y and measured values are compared as follows：</P><P>Part of the value without bold is used to solve the equation, so it is a perfect match; From the comparison of data in bold, it can be seen that the result of data fitting is good, with the average fitting degree higher than 95%.</P><P>The solution results of the matrix are good, which confirms the basic reliability of the assumed linear relationship. Therefore, we will use the neural network model to further improve the accuracy and stability of the model prediction.</P></DIV></DIV><DIV class="container" id="section-2"><DIV class="paragraph shadow"><H2 class="title" id="s4">Model Improvement--Algorithm optimization </H2><P>By solving the matrix sparse solution algorithm, we can conclude that there is indeed a linear relationship between the total 64 data of AAA-GGG and the promoter strength. Notice that we used merely 22 sets of data to approximately solve the sparse solutions of 64 equations with a fitting precision higher than 95%, therefore, when we provide as much data as possible, the fitting precision of the model will greatly increase.</P><P>However, there are still many problems with this algorithm:</P><P>(1) All known data cannot be reused.  The sparse solution is uniquely calculated as the data is determined, so the data is not very well utilized, and it is difficult to improve the accuracy of the original results as well.</P><P>(2) This algorithm is not well adapted to the extension of existing data sets. Every time a new set of data is added, all matrix operations change apparently, so it's not very stable when data updates.</P><P>On the basis of the linear structure of the original algorithm, we add more parameters to make the algorithm more stable and reliable. This algorithm adopts the neural network structure and improves the utilization rate of data by supervised learning.The construction of neural network is shown in the Figure 2:</P><P>There are 6 neurons in the hidden layer, and each neuron is connected with 32 input terminals. The corresponding relation is as follows: The binary number of integer 1~63 is exactly 6 bits, and the deficiency is added. Input and hidden neuron  \({j}\)is connected when the   \({ j^{th}}\) number of the binary number is 1. In particular, we consider port 64 separately.And then we give a certain weight  \({\omega _{ij}}\) to each edge between Input \({i}\)and hidden neuron  \({j}\),and give   to each edge between hidden neuron \({ p^{i}}\)and output terminal.</P><P>The final output can be obtained by the following formula:</P>
            $$y_{output}=\sum_{i=1}^{6}y_{i}p_{i},y_{i}=\sum_{j=1}^{64}\omega _{ji}x_{j}\, \; .$$
            <P> \({y_{i}}\)represents the input(output) data of hidden neuron \({i}\).</P><P>We use gradient descent method to determine the learning direction. The objective function is the error of output and ideal values:</P>
            $$e=\frac{1}{2}(y_{output}-\bar{y})^{2}$$
            <P>Our goal is to minimize the value of this function by generations of iterations. Difference between vector , \({ \omega =(\omega _{16},...,\omega _{n6})}\),\({p=(p_{1},...,p_{n}) }\)and its next generation is calculated by this formula:</P>

            $$y_{output}=\sum_{i=1}^{6}y_{i}•p_{i},y_{i}=\sum_{j=1}^{64}\omega _{ji}x_{j}.$$
            <P>  \({y_{i}}\)represents the input(output) data of hidden neuron \({i}\).</P><P>We use gradient descent method to determine the learning direction. The objective function is the error of output and ideal values:</P>

            $$e=\frac{1}{2}(y_{output}-\bar{y})^{2}$$
            <P>Our goal is to minimize the value of this function by generations of iterations. Difference between vector ,  \omega =(\omega _{16},...,\omega _{n6}),p=(p_{1},...,p_{n}) and its next generation is calculated by this formula:</P>

            $$ \Delta \omega =-v\cdot grad\omega ,\Delta p=-v\cdot gradp$$
            <P>where  \({v}\)represents learning speed. We use the chain rule to calculate the specific value of the gradient: </P>
$$\begin{cases}
 &amp; \text{ if } \frac{\partial e}{\partial \omega_{ij}  }= (y_{output}-\bar{y})\frac{\partial y_{output}}{\partial \omega _{ij}}=(y_{output}-\bar{y})x_{i}p_{j}\\ 
 &amp; \text{ if } \frac{\partial e}{\partial p_{i}}= (y_{output}-\bar{y})\frac{\partial y_{output}}{\partial p_{i}}=(y_{output}-\bar{y})y_{i}
\end{cases}$$

            <P>At the same time, a reasonable learning speed is designed for the program, which can accelerate the convergence speed while ensuring the convergence. After several convergence tests, we set the learning speed as\({0.00045\cdot \left | y_{output}-\bar{y} \right |.}\)That's where our mathematical model comes in.</P></DIV></DIV><DIV class="container" id="section-2"><DIV class="paragraph shadow"><H2 class="title" id="s5">Model analysis and evaluation</H2><P>After improvement of the above algorithm, the advantages of the model are as follows:</P><P>1. High stability of the model. This algorithm adopts the mode of deep learning, which shortens the step length of vector change and expands the relevant linear parameters to 206, ensuring the precision of model fitting and greatly improving the stability of the model.</P><P>2. The input of the model can be extended conveniently. When the data of the promoter library is expanded, it only needs to input the latest measured data into our system, which is automatically imported into the database, and the model can also use this data for training immediately.</P><P>3. The method of this model can be used for reference by other subjects. The coding method and mathematical treatment of the model are in line with the effect of DNA sequence on the promoter strength and can be transplanted to other similar studies such as <LATIN>E.coli</LATIN>.</P><P>At the same time, there are also many deficiencies in this model that need to be improved:</P><P>1. The model is not robust to error data. When a group of incorrect fluorescent values are given in the input data, the fitting precision of the whole program will be greatly affected, and this interference will gradually weaken as the number of correct data increases.</P><P>2. Convergence speed and accuracy still need to be improved. If the number of training rounds is too high, the running speed of the program is too slow. When the number of rounds is too low, the results presented are often unsatisfactory. We need to find a balance between speed and accuracy.</P></DIV><DIV class="paragraph shadow ref"><H2 class="title" id="s6">References</H2><P>[1]Lim, C. K.Hassan, K. A.Tetu, S. G.Loper, J. E.Paulsen, I. T. The effect of iron limitation on the transcriptome and proteome of Pseudomonas fluorescens Pf-5. PLoS One. 2012, June 2012 | Volume 7 | Issue 6 | e39139,2-18</P><P>[2] Xuan Zhou, Xin Zhou, Zhaojie Zhong, Prediction of start-up strength of E. coli promoter, Computers and Applied Chemistry, Jan.28,2014|Volume 31| Issue 1,101-103</P><P>[3]Xiaoliang Cheng, Xuan Zheng, Weimin Han, An algorithm for solving sparse solutions of undetermined linear equations,College journal of applied mathematics,2013:28(2):235-248</P></DIV></DIV><NAV class="navbar navbar-default navbar-transparent navbar-fixed dark bootsnav" style="top: 11px"><DIV class="container"><DIV class="navbar-header"><DIV class="navbar-brand"><A href="https://2018.igem.org/Team:SKLMT-China">SKLMT</A></DIV></DIV><DIV class="collapse navbar-collapse" id="navbar-menu"><UL class="nav navbar-nav navbar-right" data-in="fadeInDown" data-out="fadeInUp"><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Team" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Sponsor">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Judging">Judging</A></LI></UL><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Project" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Project_Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Red_ET">Red/ET</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Demonstrate">Results &amp; Demonstration</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/InterLab">InterLab</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Parts" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Part_Collection">Part Collection</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Improve">Improved part</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Model" class="dropdown-toggle" data-toggle="dropdown">Model</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Model_Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Model_Process">Process</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Model_Results">Results</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Software" class="dropdown-toggle" data-toggle="dropdown">Software</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/SoftWare_Overview">Introduction</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Principle">Principle</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Human_Practices" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/HP_Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Integrated_Gold">Integrated &amp; Gold</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Notebook" class="dropdown-toggle" data-toggle="dropdown">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Notebook_Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Notebook_Protocol">Protocol</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Material">Material</A></LI></DIV></DIV></NAV><SECTION id="page-footer"><DIV class="container"><DIV class="row"><DIV class="col-md-12" style="padding-top: 2.5em;"><H2><STRONG>SKLMT</STRONG></H2></DIV></DIV><DIV class="row"><DIV class="col-md-4"><H3>Useful <STRONG> Links</STRONG></H3><P><A href="https://igem.org/Main_Page">Main Page</A></P><P><A href="https://igem.org/Special:SpecialPages">Special pages</A></P></DIV><DIV class="col-md-4"><H3><STRONG>Contact </STRONG> us</H3><I class="fa fa-envelope-o fa-lg">  SKLMT.iGEM.China@gmail.com</I></DIV><DIV class="col-md-4"><H3><STRONG>Address</STRONG></H3><I class="fa fa-map-marker fa-lg">  Qingdao, Shandong266200 Shandong University</I></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>