<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SKLMT-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SKLMT-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="banner-content"><H2 class="title">Design</H2><P class="content">How do we make our project possible?</P></DIV><DIV class="container"><DIV class="row"><OL class="breadcrumb"><LI><A href="https://2018.igem.org/Team:SKLMT-China"><I class="fa fa-home">Home</I></A></LI><LI><A href="Project"><I class="fa fa-file">Project</I></A></LI><LI class="active dropdown"><DIV class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><I class="fa fa-code">Design
                            </I></DIV></LI><LI><A href="#s1" class="scrolly">Design</A></LI></OL></DIV></DIV><DIV class="container"><DIV class="paragraph shadow"><H2 class="title" id="s1">Design</H2><H3 class="title">outline</H3><P>Nowadays, the focus in metabolic engineering research is shifting from massive overexpression and inactivation of genes towards the model-based fine tuning of gene expression. This year team SKLMT-China wants to deal with the environmental problem using a new extraordinary chassis bacteria, <LATIN>Pseudomonas fluorescence pf-5</LATIN>. It is a kind of biocontrol bacteria which can be used in environmental protection. Compared with E.coli, the developed organisms, the toolkit for <LATIN>P. fluorescence </LATIN>seems hasn’t been exploited well. In our project, the construction of a library of synthetic promoters of <LATIN>P. fluorescence</LATIN> as a useful tool for fine tuning gene expression is established. The strength of different promoters will be characterized by a reporter gene, <LATIN>firefly luciferase</LATIN>. Based on this data, we did a brilliant modelling work and built a useful software tool to profoundly exploit this promoter system and guide further experiments. </P><H3 class="title">Promoter llibrary construction</H3><P>From the very beginning of our project, we have to find the sequence of proper constitutive. All the promoter sequence(70bp) is taken from the upstream region of a coding sequence (exactly upstream of its ATG base).And then, construct plasmid containing a promoter sequence and the reporter gene. </P><P>In order to obtain an artificially built promoter library suitable for cloning, promoter sequence was added a chloramphenicol selection marker. The linear DNA fragment is prepared through serious PCR, by which the chloramphenicol combined with 70bp standard promoter sequence is flanking with homology arms of the vector. We constructed the plasmids by LCHR.</P><P>The strength of different promoter was characterized by a reporter gene, <LATIN>firefly luciferase</LATIN>.</P><P> Since predecessors have less research on the promoter collection and characterization of <LATIN>P. fluorescence </LATIN>we have done an innovative work and set a standard for the promoter strength in<LATIN> P.fluorescence</LATIN>. based on the data, we established a modelling work to finger out the link between promoter sequence and strength.</P><H3 class="title">Nicotine degradation</H3><P>In addition to the formation of the promoter library, we also constructed a nicotine-degrading engineered bacteria using pf-5 as the chassis.</P><P>We completed the whole molecular clone process by Red/ET homologous reconbination technology. First of all, digeste the genome of<LATIN> Pseudomonas putida S16 </LATIN>with SpeI restriction enzyme to get nicotine degradation segment,<LATIN>nic</LATIN>. Then, transfer the <LATIN> nic </LATIN>into Arabinose-induced <LATIN>E.coli</LATIN>GB05DirTrfA cells harboring pSC101-BAD-ETgA-tet with p15A PCR purification vector through electroporation. The produced RecE/T enzyme will mediate liner-liner recombination and construct the  plasmid---p15A-cm-tetR-tetO-<LATIN>nic</LATIN>.</P><P>Second, in order to get this plasmid replicated in <LATIN>P.fluorescences</LATIN>, we have to change the p15A-vector to a kind of shuttle vector that can replicate in <LATIN>P.fluorescences</LATIN>,. Therefore, we digested pBBR1-km-ccdB-hyg plasmid with BamHI to get the shutter vector---pBBR1-km vector. After digestion, we got the shuttle vector and then we transferred p15A-cm-tetR-tetO-<LATIN>nic</LATIN> plasmid into <LATIN>E.coli</LATIN>GB05red through electroporation after which we transferred pBBR1-km into <LATIN>E.coli</LATIN>GB05red harboring p15A-cm-tetR-tetO-<LATIN>nic</LATIN> which had been induced with 10%Arabinose to mediate liner-circular recombination. After that, construct pBBR1-km-tetR-<LATIN>nic</LATIN>.</P><P>Next, we’ve found there is a part of chlorampheol <LATIN>nic</LATIN> resistance gene sequence about 300bps between pBBR1 orign of replication and the kanamycin resistance gene segment which will cause a lot of trouble in the next recombination. After disscussion, we decided to replace it with ampicillin resistance gene. We amplified ampicillin resistance gene from pR6K-amp-ccdB plasmid with PCR using PAGE-purified oligonucleotides containing 40-nt homology arms flanking the target pR6K-amp-ccdB plasmid and 19-nt standard PCR primers at 3’end. Then, we transferred the purified ampicillin resistance segment into <LATIN>E.coli</LATIN>GB05red harboring p15A-cm-tetR-tetO-<LATIN>nic</LATIN> which had been induced with 10%Arabinose to mediate liner-circular recombination. The second day, we get a new plasmid---pBBR1-amp-km-tetR-<LATIN>nic</LATIN>.</P><P>Next, we’ve found there is a part of chlorampheol <LATIN>nic</LATIN> resistance gene sequence about 300bps between pBBR1 orign of replication and the kanamycin resistance gene segment which will cause a lot of trouble in the next recombination. After decision, we decided to replace it with ampicillin resistance gene. We amplified ampicillin resistance gene from pR6K-amp-ccdB plasmid with PCR using PAGE-purified oligonucleotides containing 40-nt homology arms flanking the target pR6K-amp-ccdB plasmid and 19-nt standard PCR primers at 3’end. Then, we transferred the purified ampicillin resistance segment into <LATIN>E.coli</LATIN>GB05red harboring p15A-cm-tetR-tetO-<LATIN>nic</LATIN> which had been induced with 10%Arabinose to mediate liner-circular recombination. The second day, we get a new plasmid---pBBR1-amp-km-tetR-<LATIN>nic</LATIN>.</P><P>Finally, we got our promoter4,5and11 segments through PCR using PAGE-purified oligonucleotides containing 40-nt homology arms flanking the target pSB1C3-promoter-cm plasmids and 19-nt standard PCR primers at 3’end. At last, we transferred these three promoter segments into <LATIN>E.coli</LATIN>GB05red harboring pBBR1-amp-km-tetR-<LATIN>nic</LATIN> which had been induced with 10%Arabinose to mediate liner-circular recombination and we got our final plasmid---pBBR1-km-amp-cm-promoter-<LATIN>nic</LATIN> which can replicate itself in<LATIN> P. fluorescences</LATIN>.</P></DIV></DIV><NAV class="navbar navbar-default navbar-transparent navbar-fixed dark bootsnav" style="top: 11px"><DIV class="container"><DIV class="navbar-header"><DIV class="navbar-brand"><A href="https://2018.igem.org/Team:SKLMT-China">SKLMT</A></DIV></DIV><DIV class="collapse navbar-collapse" id="navbar-menu"><UL class="nav navbar-nav navbar-right" data-in="fadeInDown" data-out="fadeInUp"><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Team" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Sponsor">Sponsors</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Judging">Judging</A></LI></UL><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Project" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Project_Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Red_ET">Red/ET</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Demonstrate">Results &amp; Demonstration</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/InterLab">InterLab</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Parts" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Part_Collection">Part Collection</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Improve">Improved part</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Model" class="dropdown-toggle" data-toggle="dropdown">Model</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Model_Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Model_Process">Process</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Model_Results">Results</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Software" class="dropdown-toggle" data-toggle="dropdown">Software</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/SoftWare_Overview">Introduction</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Principle">Principle</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Human_Practices" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/HP_Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Integrated_Gold">Integrated &amp; Gold</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:SKLMT-China/Notebook" class="dropdown-toggle" data-toggle="dropdown">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Notebook_Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Notebook_Protocol">Protocol</A></LI><LI><A href="https://2018.igem.org/Team:SKLMT-China/Material">Material</A></LI></DIV></DIV></NAV><SECTION id="page-footer"><DIV class="container"><DIV class="row"><DIV class="col-md-12" style="padding-top: 2.5em;"><H2><STRONG>SKLMT</STRONG></H2></DIV></DIV><DIV class="row"><DIV class="col-md-4"><H3>Useful <STRONG> Links</STRONG></H3><P><A href="https://igem.org/Main_Page">Main Page</A></P><P><A href="https://igem.org/Special:SpecialPages">Special pages</A></P></DIV><DIV class="col-md-4"><H3><STRONG>Contact </STRONG> us</H3><I class="fa fa-envelope-o fa-lg">  SKLMT.iGEM.China@gmail.com</I></DIV><DIV class="col-md-4"><H3><STRONG>Address</STRONG></H3><I class="fa fa-map-marker fa-lg">  Qingdao, Shandong266200 Shandong University</I></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></BODY></HTML>