<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SKLMT-China_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SKLMT-China/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:SKLMT-China"><DIV class="menu_item direct_link">
                HOME
            </DIV></A><DIV class="menu_item">
            TEAM
        </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:SKLMT-China/Team"><DIV class="submenu_item">
                    Team Members
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Collaborations"><DIV class="submenu_item">                Collaborations
                </DIV></A></DIV><DIV class="menu_item">
            PROJECT
        </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:SKLMT-China/Description"><DIV class="submenu_item">
                    Description
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Design"><DIV class="submenu_item">
                    Design
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Experiments"><DIV class="submenu_item">
                    Experiments
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Notebook"><DIV class="submenu_item">
                    Notebook
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/InterLab"><DIV class="submenu_item">
                    InterLab
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Model"><DIV class="submenu_item">
                    Model
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Results"><DIV class="submenu_item">
                    Results
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Demonstrate"><DIV class="submenu_item">
                    Demonstrate
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Improve"><DIV class="submenu_item">
                    Improve
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Attributions"><DIV class="submenu_item">                   Attributions 
                </DIV></A></DIV><DIV class="menu_item">
            PARTS
        </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:SKLMT-China/Parts"><DIV class="submenu_item">
                    Parts Overview
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Basic_Part"><DIV class="submenu_item">
                    Basic Parts
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Composite_Part"><DIV class="submenu_item">
                    Composite Parts
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Part_Collection"><DIV class="submenu_item">
                    Part Collection
                </DIV></A></DIV><A href="https://2018.igem.org/Team:SKLMT-China/Safety"><DIV class="menu_item direct_link">
                SAFETY
            </DIV></A><DIV class="menu_item"> 
            HUMAN PRACTICES 
        </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:SKLMT-China/Human_Practices"><DIV class="submenu_item">
                    Human Practices
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Public_Engagement"><DIV class="submenu_item">
                    Education &amp; Engagement
                </DIV></A></DIV><DIV class="menu_item"> 
            AWARDS
        </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:SKLMT-China/Applied_Design"><DIV class="submenu_item">
                    Applied Design
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Entrepreneurship"><DIV class="submenu_item">
                    Entrepreneurship
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Hardware"><DIV class="submenu_item">
                    Hardware
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Measurement"><DIV class="submenu_item">
                    Measurement
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Model"><DIV class="submenu_item">
                    Model
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Plant"><DIV class="submenu_item">
                    Plant
                </DIV></A><A href="https://2018.igem.org/Team:SKLMT-China/Software"><DIV class="submenu_item">
                    Software
                </DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=SKLMT-China"><DIV class="menu_item direct_link">
                JUDGING FORM ⇗ 
            </DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size highlight decoration_B_full" style="text-align: center"><H1>Description</H1><P style="text-align:center">Construct a standard promoter library and use it to achieve expression regulation of gene in bacteria by rec/ET recombination</P></DIV><DIV class="column two_thirds_size"><H3> Abstract </H3><P>Pseudomonas fluorescens Pf-5(pf5) is the only safety bacteria using in Chinese agriculture approved by Ministry of agricultural and rural affair of China . Besides ,this strain has been confirmed to be non-pathogenic to mammals. Compared with E.coli, the developed organisms , the toolkit for Pseudomonas fluorescens seems not exploit well. We are going to build a promoter library of inner promoters with various strength in pf5. We characterize the strength by a ﬂuorescent reporter gene, firefly. An additional vector is that We hope to engineer our bacteria with the nicotine degradation gene cluster from Agrobacterium S33 into Pseudomonas fluorescens Pf-5 by rec/ET recombination , so that Pseudomonas fluorescens Pf-5 could degrade nicotine. These bacteria will be used to handle nicotine in cigarette butts or in the environment. At the same time, we would apply the promoter library to control nicotine degradation efficiency.</P></DIV><DIV class="column two_thirds_size"><H3> 1 Heterologous expression of nicotine degradation gene cluster derived from Agrobacterium S33 </H3><P>We are the first year to join the International Genetically Engineered Machine Competition(IGEM) representing State Key Laboratory of Microbial Technology of Shandong University. This year we focus on environment pollution by nicotine .As we know , nicotine is toxic. An average cigarette yields about 2 mg of absorbed nicotine; in lesser doses of that order, the substance acts as a stimulant in mammals, while high amounts (50–100mg)can be harmful.4We can also notice the second hand smoke filter membrane or cartridge filter behind the fan in the smoking area of public to prevent the air contamination. What concerns us is how to solve the polluted filter because no relative reference talking about disposing nicotine residue in filter. So our project is about an engineering bacteria having the ability to degrade nicotine. It would make a brilliant difference in solving environment problem. We design the experiment:</P></DIV><DIV class="column full_size"><H3> Restriction digestion of genomic Dna to release the target gene cluster </H3><P>High-quality genomic DNA is crucial to the successful direct cloning of large DNA segments by rec/ET recombination. DNA should be carefully extracted from lysed cells by phenol—chloroform and ethanol precipitation. [p] We digest the gene cluster of nicotine degradation of Agrobacterium S33 by the proper enzyme. Unfortunately ,we didn’t find appropriate enzyme site which we can cut the whole cluster one time. We chose the enzyme xbaⅠ including as large sequence range as possible to release the cassette from genome .PCR directly can gain another fragment easily as long as designing the suitable primer.(Figure 1)</P></DIV><DIV class="column two_thirds_size"><H3> Preparation of linear vectors for direct cloning </H3><P>We designed two primers carried homologue arm on each end of nicotine degradation cluster and an enzyme site respectively. The p15A-cm-tetR vector derived from E.coli will be the template , which can conveys chloramphenicol and tetracycline resistance. The sufficient amounts of DNA vector will have constructed after PCR.(Figure 2a)</P></DIV><DIV class="column full_size"><H3> Preparation of electro-competent E. coli cells </H3><P>The recombination system exist in a plasmid pSC101 of E.coli GB05Dir built by Youming Zhang.et2 .The RecE and RecT system from Rac prophage  can mediate highly efficient linear–linear homologous recombination.1</P><P>Electroporation of genomic nic DNA and the linear vector into E. coli cells incubate the plate at 37 °C overnight with chloramphenicol resistance . Based on homologue recombination, the recombinant can be screened by the resistance marker. After restriction analysis, we can gain the correct recombinant plasmid harboring nic segment sequence. (Figure 2b)
</P></DIV><DIV class="column full_size"><H3> Insert Reverse screening maker  </H3><P>In order to facilitate the follow-up work, we plan to insert another screening maker ccdB. The toxin CcdB will kill the host E.coli with CcdA or the gyrAArg462Cys mutation once expresses. Amplify the line vector Pamp-amp-ccdB with PCR using standard PCR primers containing homology arms flanking the target DNA segment at the 5′ end . The step is the same as  preceding part of the p15A-cm-tetR vector .</P><P>Then the line vector Pamp-amp-ccdB together with the constructed plasmid in last step was electroporated in E.coliGB05Red-gyA462 to acquire a new recombinant.E.coliGB05Red-gyA462 carries out line-circle recombination when induce the expression of Redα/Redβ/Redγ/RecA recombination system. But this time, Ampicillin and chloramphenicol resistance gene can be used as a selectable marker. Digest the plasmid with BamH Ⅰ, run agarose gel , observe the electrophoresis bands and check for release of the correct size fragment. The plasmid p15A-ccdB-nic is our Target product in this step. (Figure 2c)</P></DIV><DIV class="column full_size"><H3> Gain the complete nicotine degradation genome </H3><P>Next repeat the hold protocal from last step. The residue nic PCR product in step 1 will be the line vector like Pamp-amp-ccdB and the plasmid p15A-ccdB-nic will act as the circle cassette. Transfer the colonies that have grown up to another plate several times. The correct recombinant will finally survive by ccdB counterselection .We splice the whole genome successfully in vitro! (Figure 2d) However, it is not stable enough for plasmid cassette to store our large gene cluster. In order to add nic gene cluster in genome of pf5 ,which can contribute to the survival of gene clusters in replication, we insert the conjugation–transposition/site-specific- integration cassette and plasmid p15A-ccdB-nic simultaneously in pf5. Having induced the cassette to express transposition system, we could screen the correct recombinant. Thus far, we obtain a new pf5 with nicotine degradation gene cluster ,called pf5-nic.</P><P>This is what we are experimentalizing now.</P></DIV><DIV class="column full_size"><H3> 2  Construct a standard promoter library </H3><UL><LI style="font-size: 120%">Find various strength promoters</LI><P style="font-size: 100%">Next repeat the hold protocal from last step. The residue nic PCR product in step 1 will be the line vector like Pamp-amp-ccdB and the plasmid p15A-ccdB-nic will act as the circle cassette. Transfer the colonies that have grown up to another plate several times. The correct recombinant will finally survive by ccdB counterselection .We splice the whole genome successfully in vitro! (Figure 2d) However, it is not stable enough for plasmid cassette to store our large gene cluster. In order to add nic gene cluster in genome of pf5 ,which can contribute to the survival of gene clusters in replication, we insert the conjugation–transposition/site-specific- integration cassette and plasmid p15A-ccdB-nic simultaneously in pf5. Having induced the cassette to express transposition system, we could screen the correct recombinant. Thus far, we obtain a new pf5 with nicotine degradation gene cluster ,called pf5-nic.</P><LI style="font-size: 120%">Measurement </LI><P style="font-size: 100%">Next repeat the hold protocal from last step. The residue nic PCR product in step 1 will be the line vector like Pamp-amp-ccdB and the plasmid p15A-ccdB-nic will act as the circle cassette. Transfer the colonies that have grown up to another plate several times. The correct recombinant will finally survive by ccdB counterselection .We splice the whole genome successfully in vitro! (Figure 2d) However, it is not stable enough for plasmid cassette to store our large gene cluster. In order to add nic gene cluster in genome of pf5 ,which can contribute to the survival of gene clusters in replication, we insert the conjugation–transposition/site-specific- integration cassette and plasmid p15A-ccdB-nic simultaneously in pf5. Having induced the cassette to express transposition system, we could screen the correct recombinant. Thus far, we obtain a new pf5 with nicotine degradation gene cluster ,called pf5-nic.</P><LI style="font-size: 120%">Expression regulation of gene by orthonormal promoters. </LI><P style="font-size: 100%">Synthesize the promoter sequence with oligonucleotides. Amplify the circular vector carried firefly with multiple PCR splicing using homology arms flanking firefly gene and chloramphenicol resistance gene segment together with promoter sequence. Electroporate the linear vector prepared by PCR along with the circular vector carried firefly in E.coliGB05Red-gyA462. 
When getting the recombinational plasmid, we transfer the plasmid into pf5 to test the expression of firefly.
</P><LI style="font-size: 120%">Expression regulation of gene by orthonormal promoters. </LI><P style="font-size: 100%">The statistics we obtain from the previous step can be referenced for standardize the promoter vectors. We can sort the promoters from strong to weak ordination to set up the library. Electroporate the standardize promoter linear vectors into pf5-nic (talk in first step). Culture the pf5-nic overnight with nicotine as only carbon source, test the metabolism component by HPLC to verify the promoter we build can work as we expected.
</P></UL></DIV><DIV class="column full_size"><DIV class="highlight decoration_B_full" style="text-align: center"><UL><LI style="text-align: left">1 Hailong Wang, et al. RecET direct cloning and Redab recombineering of biosynthetic gene clusters, large operons or single genes for heterologous expression. Nat.protocal.11,1175–1190 (2016).</LI><LI style="text-align: left">2 Youming Zhang, et al. Gene Bridges – Direct cloning-proficient E. coli Strain GB05-dir, Version 1.1 (May 2014)</LI><LI style="text-align: left">3 Joshua R. Elmore, et al. Development of a high eﬃciency integration system and promoter library for rapid modiﬁcation of Pseudomonas putida KT2440. Metabolic Engineering Communications. 5, 1–8(2017).</LI><LI style="text-align: left">4  Mayer B. &quot;How much nicotine kills a human? Tracing back the generally accepted lethal dose to dubious self-experiments in the nineteenth century&quot;. Archives of Toxicology. 88,5–7 (2014).</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>