Team:SKLMT-China/Red ET
Red/ET
Introduction of our recombination technology
Home
Project
Red/ET
Introduction
What is Recombineering?
How does Recombineering work?
Expression of Proteins for Recombineering
Applications
introduction
The improvement of DNA recombination technologies has greatly enhanced the possibilities and convenience of mouse model systems usage. This improvement involves the construction of recombinant DNA molecules by recombineering in
E.coli
, the integration of this molecules into eucaryotic cells by homologous recombination or transposon techniques and the employment of site-specific recombinases to integrate recombinant DNA to alter those molecules.
What is Recombineering?
The original methods for the production of recombinant DNA along with oligonucleotide synthesis and PCR are limited by size due to their usage of restriction enzymes and DNA ligase to "cut-and-paste" DNA. However, recombineering is a revolutionary method for DNA engineering using homologous recombination. Recombineering allows unlimited cloning, subcloning, and modification of DNA at any chosen position. It permits precise engineering of DNA molecules of any size, including very large ones such as BACs or the
E.coli
chromosome.
Advantages over conventional methods:
Independent of restriction sites
No size limits
No unwanted mutations
Rapid
Recombineering was developed in A. Francis Stewart's Laboratory at the EMBL. This technical breakthrough is precise and independent of the presence of restriction sites and the size of the DNA molecule to be modified. Therefore, problems previously incurRed by traditional DNA engineering techniques (Cohen et al., 1973; Mullis et al., 1986) have come to an end (Zhang et al., 1998; Zhang et al., 2000) . Furthermore this innovative technology is significantly less laborious than traditional DNA manipulation techniques and therefore has large savings on time and cost (Muyerset al., 1999) .             In Red/ET Recombineering, also referRed to as \({\lambda}\)-mediated recombination, target DNA molecules are precisely alteRed by homologous recombination in strains of E.coli that express phage-derived protein pairs, either RecE/RecT from the Racprophage, or Red\({\alpha}\)/Red\({\beta}\) from the \({\lambda}\) phage (Muyerset al., 2000). These protein pairs are functionally and operationally equivalent. RecE and Red\({\alpha}\) are 5´-3´ exonucleases, and RecT and Red\({\beta}\) are DNA annealing proteins. A functional interaction between RecE and RecT, or between Red\({\alpha}\) and Red\({\beta}\) is also requiRed in order to catalyse the homologous recombination reaction.
Since the sequence of the homology regions can be chosen freely, any position on a target molecule can be specifically alteRed. Homologous recombination allows the exchange of genetic information between two DNA molecules in a precise, specific and faithful manner, qualities that are optimal for DNA engineering regardless of size.
One of the biggest advantages of recombineering is that it is not just limited to its unique ability to clone and subclone large DNA molecules but also it presents a variety of new ways to simplify conventional DNA engineering exercises (Muyerset al., 1999; Muyerset al., 2000). Recombineering allows every type of DNA modification possible. It becomes trivial to diRecT changes to a chosen DNA sequence or to introduce point mutations at any chosen site of a target DNA molecule regardless of its size.
The potential to use homologous recombination for DNA engineering had long been recognised, and methods which harnessed the endogenous homologous recombination system (termed RecA-dependent recombination) in
E.coli
,the premier cloning host, were developed. However this endogenous system has many limitations. It is impossible to use linear DNA molecules as they become rapidly degraded. Furthermore, recombination of circular molecules require relatively long homology regions and the ratio of intended to unwanted recombination products is extremely low. None of these limitations are encounteRed with recombineering.
How does Recombineering work?
Recombination occurs through homology regions, which are stretches of DNA shaRed by the two molecules that recombine. The recombination is further assisted by γ-encoded Gam protein, which inhibits the RecBCD exonuclease activity of
E.coli
. Unlike in yeast, linear dsDNA is unstable in
E.coli
because of the activity of RecBCD.
A double-stranded break repair (DSBR) is initiated by the recombinase protein pairs, RecE/RecT or Red\({\alpha}\)/Red\({\beta}\). First Red\({\alpha}\) (or RecE) digests one strand of the DNA from the DSB, leaving the other strand as a 3’ ended, single-stranded DNA overhang. Then Red\\({\beta}\)(or RecT) binds and coats the single strand (Erleret al., 2009). The protein-nucleic acid filament aligns with homologous DNA. Once aligned, the 3'end becomes a primer for DNA replication (Muyerset al., 2000; Marescaet al., 2010).
Expression of Proteins for Recombineering
The \({\gamma}\)-recombination functions can be expressed from a defective prophage integrated into the
E.coli
chromosome (e.g. Zhang et al. 2000) or from a plasmid (see below). In the latter case RedET recombination is transferable to the host strain in which the BAC resides, thereby alleviating the need to retransform the BAC into a special strain.
One plasmid will be used in this course. The plasmid is carrying the Red \({\alpha}\), \({\beta}\), \({\gamma}\) genes of the \({\lambda}\) phage together with the recA gene in a polycistronic operon (Wang et al., 2006) under the control of an inducible BAD-promoter. The pBAD promoter is both positively and negatively regulated by the product of the araC gene (Schleif, 1992). AraC is a transcriptional regulator that forms a complex with L-arabinose. The recombination window is therefore limited by the transient nature of the induced expression of Red\({\alpha}\) and Red\({\beta}\), which are both strictly requiRed for Red/ET recombination to occur. Thus, the risk of unwanted intramolecular rearrangement is minimized, allowing recombinants that contain no other unintended changes to be recoveRed efficiently.
While constitutive expression of the red \({\gamma}\)gene has a toxic effect in DH10B (recA-) cells under some conditions, thus limiting the efficiency of recombination, tightly regulated expression of the \({\gamma}\) gene together with simultaneous expression of the Red\({\alpha}\) and Red\({\alpha}\) genes allows efficient homologous recombination between linear DNA fragments and BACs resident in such cells as DH10B.
The plasmid is a derivative of the temperature-sensitive pSC101 replicon which is a low copy number plasmid, depending on the oriR101. The RepA protein encoded by plasmid pSC101 is required for plasmid DNA replication and the partitioning of plasmids to daughter cells at divisions. Since a temperature-sensitive (Ts) RepA protein is expressed, cells have to be cultured at 30℃, pSC101 derivatives are easily curable at 37℃ to 43℃.
Experiments have shown that after 2h of cell growth at a temperature non-permissive for replication of this plasmid (i.e. 42℃), the average plasmid copy number is sharply deceased by about 80% during four generations of bacterial cell growth at 42℃. After return of the cultures to 30℃, approximately the same number of generations of bacterial cell growth is required for the copy number of the plasmids to return to the level observed before.
Since the plasmid is depending on the oriR101 it can be propagated in
E.coli
together with most ColE1-derived plasmids. The temperature-sensitivity of the RepA proteins is leading to a rapid loss of the plasmid in the absence of continued selection pressure.
Applications
subcloning by gap repair (Zhang et al., 2000), e.g. for subcloning from genomic DNA or BACs (Fu et al., 2010)
insertion of selectable fragments or/and deletion, e.g. BAC modification (Ciottaet al., 2010; Testaet al., 2003; Valenzuela et al., 2003; Wu et al., 2008; Yang and Seed, 2003)
oligonucleotide-diRecTed mutagenesis (Ellis et al., 2001)
Those applications have also been successfully adapted to high-throughput DNA engineering methods.
Combined and sequential use of those applications enables a variety and modularity of DNA modifications. As an example we show our current method to make targeting constructs starting from a BAC.
Reference
Anastassiadis, K., Schnutgen, F., von Melchner, H. and Stewart, A. F. (2013). "Gene Targeting and Site-Specific Recombination in Mouse ES Cells." Methods Enzymol 533: 133-55.
Ejsmont, R. K., Ahlfeld, P., Pozniakovsky, A., Stewart, A. F., Tomancak, P. and Sarov, M. (2011)."Recombination-mediated genetic engineering of large genomic DNA transgenes." Methods MolBiol 772: 445-58.
von Melchner, H. and Stewart, A. F. (2011). "Methods for Extracting Function from the Mammalian Genome."Methods.
Fu, J., Teucher, M., Anastassiadis, K., Skarnes, W. and Stewart, A. F. (2010). "A Recombineering Pipeline to Make Conditional Targeting Constructs." Methods Enzymol 477C: 125-144.
Hofemeister, H., Ciotta, G., Fu, J., Seibert, P. M., Schulz, A., Maresca, M., Sarov, M., Anastassiadis, K. and Stewart, A. F. (2011). "Recombineering, transfection, Western, IP and ChIP methods for protein tagging via gene targeting or BAC transgenesis."Methods.
Ciotta, G., Hofemeister, H., Sarov, M., Anastassiadis, K. and Francis Stewart, A. (2010)."Recombineering BAC transgenes for protein tagging."Methods.
Anastassiadis, K., Glaser, S., Kranz, A., Berhardt, K. and Stewart, A. F. (2010). "A Practical Summary of Site-Specific Recombination, Conditional Mutagenesis, and Tamoxifen Induction of CreERT2." Methods Enzymol 477C: 109-123.
Muyrers, J. P., Zhang, Y., Benes, V., Testa, G., Rientjes, J. M. and Stewart, A. F. (2004). "ET recombination: DNA engineering using homologous recombination in E. coli." Methods MolBiol 256: 107-21.
Testa, G., Vintersten, K., Zhang, Y., Benes, V., Muyrers, J. P. and Stewart, A. F. (2004)."BAC engineering for the generation of ES cell-targeting constructs and mouse transgenes." Methods MolBiol 256: 123-39.
Muyrers, J. P., Zhang, Y. and Stewart, A. F. (2001). "Techniques: Recombinogenic engineering - new options for cloning and manipulating DNA."Trends BiochemSci 26: 325-31.
Muyrers, J. P., Zhang, Y. and Stewart, A. F. (2000). "ET-cloning: think recombination first." Genet Eng (N Y) 22: 77-98.
Wang H, Bian X, Xia L, Ding X, Müller R, Zhang Y, Fu J, Stewart AF. (2013) "Improved seamless mutagenesis by recombineering using ccdB for counterselection." Nucleic Acids Res.
Hu, S., Fu, J., Huang, F., Ding, X., Stewart, A.F., Xia, L. and Zhang, Y. (2013) "Genome engineering of Agrobacterium tumefaciens using the lambda Red recombination system." ApplMicrobiolBiotechnol.
Bian, X., Huang, F., Stewart, F. A., Xia, L., Zhang, Y. and Muller, R. (2012)."Direct Cloning, Genetic Engineering, and Heterologous Expression of the Syringolin Biosynthetic Gene Cluster in E. coli through Red/ET Recombineering."Chembiochem 13: 1946-52.
Fu, J., Bian, X., Hu, S., Wang, H., Huang, F., Seibert, P., Plaza, A., Xia, L., Müller, R., Stewart, A. F. and Zhang, Y. (2012) "Full-length RedE enhances linear-linear homologous recombination and facilitates direct cloning for bioprospecting.” Nat Biotechnol 30: 440-446.
Sarov, M., Murray, J. I., Schanze, K., Pozniakovski, A., Niu, W., Angermann, K., Hasse, S., Rupprecht, M., Vinis, E., Tinney, M., Preston, E., Zinke, A., Enst, S., Teichgraber, T., Janette, J., Reis, K., Janosch, S., Schloissnig, S., Ejsmont, R. K., Slightam, C., Xu, X., Kim, S. K., Reinke, V., Stewart, A. F., Snyder, M., Waterston, R. H. and Hyman, A. A. (2012). "A Genome-Scale Resource for In Vivo Tag-Based Protein Function Exploration in C. elegans." Cell 150:855-66.
Nedelkova, M., Maresca, M., Fu, J., Rostovskaya, M., Chenna, R., Thiede, C., Anastassiadis, K., Sarov, M. and Stewart, A. F. (2011)."Targeted isolation of cloned genomic regions by recombineering for haplotype phasing and isogenic targeting."Nucleic Acids Res 39:e137.
Bird, A. W., Erler, A., Fu, J., Hériché, J. K., Maresca, M., Zhang, Y., Hyman, A. A. and Stewart, A. F. (2011). "High efficiency counterselectionrecombineering for site-directed mutagenesis in bacterial artificial chromosomes."Nat Methods 9:103-109.
Maresca, M., Erler, A., Fu, J., Friedrich, A., Zhang, Y. and Stewart, A. F. (2010). "Single-stranded heteroduplex intermediates in lambda Red homologous recombination." BMC MolBiol 11: 54.
Kranz, A., Fu, J., Duerschke, K., Weidlich, S., Naumann, R., Stewart, A. F. and Anastassiadis, K. (2010). "An improved Flpdeleter mouse in C57Bl/6 based on Flporecombinase." Genesis.
Anastassiadis, K., Fu, J., Patsch, C., Hu, S., Weidlich, S., Duerschke, K., Buchholz, F., Edenhofer, F. and Stewart, A. F. (2009). "Drerecombinase, like Cre, is a highly efficient site-specific recombinase in E. coli, mammalian cells and mice." Dis Model Mech 2: 508-15.
Erler, A., Wegmann, S., Elie-Caille, C., Bradshaw, C. R., Maresca, M., Seidel, R., Habermann, B., Muller, D. J. and Stewart, A. F. (2009). "Conformational adaptability of Redbeta during DNA annealing and implications for its structural relationship with Rad52."J Mol Biol 391: 586-98.
Testa, G., Schaft, J., van der Hoeven, F., Glaser, S., Anastassiadis, K., Zhang, Y., Hermann, T., Stremmel, W. and Stewart, A. F. (2004). "A reliable lacZ expression reporter cassette for multipurpose, knockout-first alleles." Genesis 38: 151-8.
Testa, G., Zhang, Y., Vintersten, K., Benes, V., Pijnappel, W. W., Chambers, I., Smith, A. J., Smith, A. G. and Stewart, A. F. (2003). "Engineering the mouse genome with bacterial artificial chromosomes to create multipurpose alleles." Nat Biotechnol 21: 443-7.
Muyrers, J. P., Zhang, Y., Benes, V., Testa, G., Ansorge, W. and Stewart, A. F. (2000)."Point mutation of bacterial artificial chromosomes by ET recombination." EMBO Rep 1: 239-43.
Zhang, Y., Muyrers, J. P., Testa, G. and Stewart, A. F. (2000)."DNA cloning by homologous recombination in Escherichia coli."Nat Biotechnol 18: 1314-7.
Muyrers, J. P., Zhang, Y., Buchholz, F. and Stewart, A. F. (2000). "RecE/RecT and Redalpha/Redbeta initiate double-stranded break repair by specifically interacting with their respective partners." Genes Dev 14: 1971-82.
Zhang, Y., Buchholz, F., Muyrers, J. P. and Stewart, A. F. (1998). "A new logic for DNA engineering using recombination in Escherichia coli." Nat Genet 20: 123-8.
Muyrers, J. P., Zhang, Y., Testa, G. and Stewart, A. F. (1999)."Rapid modification of bacterial artificial chromosomes by ET-recombination." Nucleic Acids Res 27: 1555-7.
SKLMT
Team
Members
Attributions
Sponsors
Judging
Project
Overview
Design
Red/ET
Results & Demonstration
Safety
InterLab
Parts
Basic Parts
Composite Parts
Part Collection
Improved part
Model
Overview
Process
Results
Software
Introduction
Principle
Human Practices
Overview
Integrated & Gold
Collaborations
Notebook
Overview
Protocol
Material
SKLMT
Useful Links
Main Page
Special pages
Contact us
SKLMT.iGEM.China@gmail.com
Address
Qingdao, Shandong
266200 Shandong University
