Team:SKLMT-China/Notebook Overview
Overview
A busy but meaningful summer
Home
Notebook
week1-4
Team registration and brainstorm
During this time, we registrated the team SKLMT-China in iGEM. In order to verify the feasibility of our project, we discussed with professors and experts and did some human practice work.
week5
Find promoters with various strength in the light of transcriptom references. Take the sequence from the upstream of a specific gene.
Number
Promoter
Sequence 5'-3'
Length
01
PampC
ACGCCGGGGCCGCAAAGCCGCTGGGACAAACGCGGCGTACATAACCAGAACCCAGGGAAACCAACCAATC
70
02
ParaA
CTACGACGGCTTCCTGACCCTATAGCTCTAGCAACACCCCCGATTGTTTTGTCTCTGGAGGATCACAGTA
70
03
PfabD
TGTCAGTTTCTTGCGTCCACCACGGTGTGACGTCAATTCTCCGACGACAAGATCATTAGGGGCTTGTTAC
70
04
PccmD
AGGTGCTCAAGCGTGAAGCCCGGACCAGTTGGGTCAAGGCCGAAGTCCAACGCAGCCTGGAGGCCGCCCG
70
05
PfliA
CGGCGAACCCACGCGGGCATCTGGAGTTTTTTGTCGAGCGGCTAGTGCATCAAACAGCGGGACCGGTGCT
70
06
PltR
GGCTGTCATGATCTATTTGTAATTTGCCTTTACAAAAATAAGACACTAAAAATTCTAAAAGGATTTAGGA
70
07
PproC
GCGCTGTTTGGTGCCCGCGACTACAGTCAGTCCTGATGCGACTGACCCCGACCTCTTGTAAGGACCTGTC
70
08
PrecA
GCTGTGGAATAATACTGGCTACTTATACAGGTGTTGGCCGTCAGGCCTTATTGATTACGTGAGGACTTTA
70
09
PrpoD
TGAGGTCATAGCTCGGGTATAATCCTCGGCTTGTTTTTTGCCCGCCAAGACCTTCAGTGGATAGGGTGTT
70
10
PrpoS
AGGCTCCAGCGTTGCCAGGGATAAAGGCGCCGCTTGAGCCTGAGGTCGAACTCACCAAAGGACTATAACA
70
11
PdnaA
TAAGTGGGCGGCTGATCGCTACAATGGCCGCTTGTTTTTGCCTCACCGGCTTTCAACTTAGGGGATATCC
70
12
Ppol
ACCTTGTCCCTTTGTCGTCAACGGTCCCGCTATTCTAGCGCCGGACCCGCCGAAAAGGTTGATCCCGCTC
70
13
P16s
CGAGAAATCAAAGATGTAACCAACGATTGCTGAGCCAAGTTTAGGGTTTTCTCAAAACCCAAAGATGTTT
70
14
Psig
TGCCAAATCGATAAAGCCTCAATGACAATGCGACTAATTATCATTAGTGACACTCAGGCAGTCCGCGTCC
70
15
Patf
GTCGCCGGGCGTAAAGTGATCGCCGTTTGCGCGGCCCGATTGTCCGGCCGCCCGAGTAAGGAGTCTTATC
70
16
Pfme
ATGACCGTCTTGCAAAGGCTTGCTACGGCAAGCCTTTTTCTTTTCCGCATGTTTAACATCGAGAATTCCC
70
17
Pmem
GGTTAATCCGGGGTTTGTCGTTTCTGCCCGCCCCGCTATCATTCGCCGCATCTTTCGAGGGGTATGACCG
70
18
Pfat
CGTTTACGGCTAGCGCCCGGAGCCGGGGCTCGGGGCAGGATGCACACATCACTGACCAGAGGATGGAATC
70
19
Pami
GGTTTGGGGTTGCGCACTAACTTGCGCAGTGGCGATATAAGCGCCCTATAACAATTCCAGGGGTCGTTGT
70
20
Prib
TTCGCGCCCTAATTCGTGCGGTATTCAACAATAGTGTTGGGTGGCGGCACGCTAGCCTGAGGAATACACC
70
21
Pabc
CCCGCAAATAAAATTGCCGGCCCAGGCCGGCAGGCGCCCGGGGTCGGGCGACTTACAAGGAGCTTGTTGG
70
22
Pedo
TGAAGGCGCGCTGAGCGTGGCTGACCAGTTGATCGCCAAAGAGCGCGCAGCCAAGTAAGCGCCCGCCGCC
70
23
Ptox
AAAGACGCGGGCCTTGAGTGCTCTAACACCCAAGACCCGCTAACCACAAGCAGCTAAACAGGAGCTGAAT
70
24
Pdiv
GAGCTGACCCGGCGCACCCTCAAGCAAGGTCTGGAACTGCTGGGCCTCAAAGTCCTGGAGCAAATGTAAG
70
25
Plip
AGGTTCTGATGTTTTCCGGGCAAGGCCTGAAGTAAAAAAACCGGGGCTTCGGGCCCACGGGAGAAAAATA
70
week6
design the PAGE-purified oligonucleotides used to amplify the chloramphenicol
week7&8
Amplify the chloramphnicol and promoter segment and add vector homology arms via series PCR. Purify the PCR products using the TianGen PCR Purification Kit and elute into 30ul of autoclaved ddH2O. Quantify the elution using a NanoDrop UV spectrophotometer, store them at 4℃。
In order to obtain an promoter library suitable for cloning, promoter sequence flanked by homology arms was added a chloramphenicol selection marker. This sequence was synthesized by Sangon.
TABLE 2 Oligonucleotides used for PCR of the DNA segment containing the promoter and chloramphenicol selective marker
Primer
Sequence 5'-3'
length
Syn-primer 1
AATGAATTACAACAGTTTTTATGCAGATATCAATTAATTTGGTTATGTGTGGGAGGGCTA
60
ampC-2
TGGGTTCTGGTTATGTACGCCGCGTTTGTCCCAGCGGCTTTGCGGCCCCGGCGTATTAATGACGTTGATCGGCACGTAAG
80
araA-2
AGACAAAACAATCGGGGGTGTTGCTAGAGCTATAGGGTCAGGAAGCCGTCGTAGATTAATGACGTTGATCGGCACGTAAG
80
fabD-2
GATCTTGTCGTCGGAGAATTGACGTCACACCGTGGTGGACGCAAGAAACTGACAATTAATGACGTTGATCGGCACGTAAG
80
ccmD-2
TGCGTTGGACTTCGGCCTTGACCCAACTGGTCCGGGCTTCACGCTTGAGCACCTATTAATGACGTTGATCGGCACGTAAG
80
fliA-2
TTTGATGCACTAGCCGCTCGACAAAAAACTCCAGATGCCCGCGTGGGTTCGCCGATTAATGACGTTGATCGGCACGTAAG
80
pltR-2
AATTTTTAGTGTCTTATTTTTGTAAAGGCAAATTACAAATAGATCATGACAGCCATTAATGACGTTGATCGGCACGTAAG
80
proC-2
AGGTCGGGGTCAGTCGCATCAGGACTGACTGTAGTCGCGGGCACCAAACAGCGCATTAATGACGTTGATCGGCACGTAAG
80
recA-2
TCAATAAGGCCTGACGGCCAACACCTGTATAAGTAGCCAGTATTATTCCACAGCATTAATGACGTTGATCGGCACGTAAG
80
rpoD-2
AAGGTCTTGGCGGGCAAAAAACAAGCCGAGGATTATACCCGAGCTATGACCTCAATTAATGACGTTGATCGGCACGTAAG
80
rpoS-2
TGAGTTCGACCTCAGGCTCAAGCGGCGCCTTTATCCCTGGCAACGCTGGAGCCTATTAATGACGTTGATCGGCACGTAAG
8f0
dnaA-2
GAAAGCCGGTGAGGCAAAAACAAGCGGCCATTGTAGCGATCAGCCGCCCACTTAATTAATGACGTTGATCGGCACGTAAG
80
pol-2
TTCGGCGGGTCCGGCGCTAGAATAGCGGGACCGTTGACGACAAAGGGACAAGGtATTAATGACGTTGATCGGCACGTAAG
80
16s-2
TGAGAAAACCCTAAACTTGGCTCAGCAATCGTTGGTTACATCTTTGATTTCTCGATTAATGACGTTGATCGGCACGTAAG
80
sig-2
AGTGTCACTAATGATAATTAGTCGCATTGTCATTGAGGCTTTATCGATTTGGCAATTAATGACGTTGATCGGCACGTAAG
80
atf-2
CGGGCGGCCGGACAATCGGGCCGCGCAAACGGCGATCACTTTACGCCCGGCGACATTAATGACGTTGATCGGCACGTAAG
80
fme-2
AAACATGCGGAAAAGAAAAAGGCTTGCCGTAGCAAGCCTTTGCAAGACGGTCATATTAATGACGTTGATCGGCACGTAAG
80
mem-2
AAGATGCGGCGAATGATAGCGGGGCGGGCAGAAACGACAAACCCCGGATTAACCATTAATGACGTTGATCGGCACGTAAG
80
fat-2
CAGTGATGTGTGCATCCTGCCCCGAGCCCCGGCTCCGGGCGCTAGCCGTAAACGATTAATGACGTTGATCGGCACGTAAG
80
ami-2
TTGTTATAGGGCGCTTATATCGCCACTGCGCAAGTTAATGCGCAACCCCAAACCATTAATGACGTTGATCGGCACGTAAG
80
rib-2
TAGCGTGCCGCCACCCAACACTATTGTTGAATACCGCACGAATTAGGGCGCGAAATTAATGACGTTGATCGGCACGTAAG
80
abc-2
AAGTCGCCCGACCCCGGGCGCCTGCCGGCCTGGGCCGGCAATTTTATTTGCGGGATTAATGACGTTGATCGGCACGTAAG
80
edo-2
TTGGCTGCGCGCTCTTTGGCGATCAACTGGTCAGCCACGCTCAGCGCGCCTTCAATTAATGACGTTGATCGGCACGTAAG
80
tox-2
GCTGCTTGTGGTTAGCGGGTCTTGGGTGTTAGAGCACTCAAGGCCCGCGTCTTTATTAATGACGTTGATCGGCACGTAAG
80
div-2
GACTTTGAGGCCCAGCAGTTCCAGACCTTGCTTGAGGGTGCGCCGGGTCAGCTCATTAATGACGTTGATCGGCACGTAAG
80
lip-2
GCCCGAAGCCCCGGTTTTTTTACTTCAGGCCTTGCCCGGAAAACATCAGAACCTATTAATGACGTTGATCGGCACGTAAG
80
Set up the PCR reaction using pR6K-cm-ccdB as a template. Typically, set up the PCR reaction (50ul in total) to obtain sufficient amounts of DNA.
Component
Amount(ul)
Final
Autoclaved ddH2O
22
2×primeSTAR max premix
25
1×
Template, 50ng ul-1
1
50 ng
Syn-promoter-1 Oligo(10uM)
1
30 pmol
promoter-2 Oligo(10uM)
1
30 pmol
Total
50
Carry out the PCR in a thermal cycle following the instructions below:
Cycle number
Denaturation
Annealing
Termination
1
94℃,4min
2-31
98℃, 15s
59℃, 15s
72℃, 15s
32
72℃, 4min
33
10℃, hold
·Add vector homology arms to the PCR products purified.
·Use the purified PCR products from last step as a template. Use the PAGE-purified oligonucleotides ,Syn-promoter-3 and promoter-4 (Table3), to amplify the cm-promoter.
TABLE 3 Oligonucleotides used for PCR of the DNA segment containing the promoter and chloramphenicol selective marker
Primer
Sequence 5'-3'
length
Syn-primer 3
AATGAATTACAACAGTTTTTAT
22
ampC-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATGATTGGTTGGTTTCCCTGGGTTCTGGTTATGTACGC
76
araA-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATTACTGTGATCCTCCAGAGACAAAACAATCGGGGGTG
76
fabD-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATGTAACAAGCCCCTAATGATCTTGTCGTCGGAGAATT
76
ccmD-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATCGGGCGGCCTCCAGGCTGCGTTGGACTTCGGCCTTGA
76
fliA-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATAGCACCGGTCCCGCTGTTTGATGCACTAGCCGCTCG
76
pltR-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATTCCTAAATCCTTTTAGAATTTTTAGTGTCTTATTTT
76
proC-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATGACAGGTCCTTACAAGAGGTCGGGGTCAGTCGCATC
76
recA-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATTAAAGTCCTCACGTAATCAATAAGGCCTGACGGCCA
76
rpoD-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATAACACCCTATCCACTGAAGGTCTTGGCGGGCAAAAA
76
rpoS-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATTGTTATAGTCCTTTGGTGAGTTCGACCTCAGGCTCA
76
dnaA-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATggatatcccctaagttgaaagccggtgaggcaaaaa
76
pol-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATgagcgggatcaaccttttcggcgggtccggcgctag
76
16s-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATaaacatctttgggttttgagaaaaccctaaacttgg
76
sig-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATggacgcggactgcctgagtgtcactaatgataatta
76
atf-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATgataagactccttactcgggcggccggacaatcggg
80
fme-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATgggaattctcgatgttaaacatgcggaaaagaaaaa
76
mem-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATcggtcatacccctcgaaagatgcggcgaatgatagc
76
fat-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATgattccatcctctggtcagtgatgtgtgcatcctgc
76
ami-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATacaacgacccctggaattgttatagggcgcttatat
76
rib-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATggtgtattcctcaggctagcgtgccgccacccaaca
76
abc-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATccaacaagctccttgtaagtcgcccgaccccgggcg
76
edo-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATggcggcgggcgcttacttggctgcgcgctctttggc
76
tox-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATattcagctcctgtttagctgcttgtggttagcgggt
76
div-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATcttacatttgctccaggactttgaggcccagcagtt
76
lip-4
ATGGCGCCGGGCCTTTCTTTATGTTTTTGGCGTCTTCCATtatttttctcccgtgggcccgaagccccggtttttt
76
·Use the purified PCR products from step2 as a template. set up the PCR reaction (50ul in total) to obtain sufficient amounts of DNA
Component
Amount(ul)
Final
Autoclaved ddH2O
22
2×primeSTAR max premix
25
1×
Template, 50ng ul-1
1
50 ng
Syn-promoter-3 Oligo(10uM)
1
30 pmol
promoter-4 Oligo(10uM)
1
30 pmol
Total
50
Carry out the PCR in a thermal cycle following the instructions below:
Cycle number
Denaturation
Annealing
Termination
1
94℃,4min
2-31
98℃, 15s
52~59℃, 15s, 15s
72℃, 15s
32
72℃, 4min
33
10℃, hold
·Purify the PCR products using the TianGen PCR Purification Kit and elute into 30ul of autoclaved ddH2O. Quantify the elution using a NanoDrop UV spectrophotometer.
week9
Transform the plasmid pBBR1-kan-amp-ccdb-BAD-GFP-firefly into GB05 red gyrA462 via electroporation. Do Mini-Prep and screen correct plasmid by restriction analysis
week10
construct the plasmid pBBR1-kan-cm-promoter-firefly via liner circular recombination in GB05 red gyrA462.
Do Mini-Prep and screen correct recombinants by restriction analysis.
week11
Transform the plasmid pBBR1-kan-amp-ccdb-BAD-GFP-firefly into
P.fluorescence pf-5
via electroporation.
·Use a syringe needle (25-Gauge 5/8, 0.5 mm × 16 mm) to puncture the cap of a 1.5-ml microcentrifuge tube for aeration and add 1.0 ml LB. Inoculate the medium in the 1.5-ml tube with the several colonies of
P. fluorescence pf-5
cells and incubate at 30 °C overnight with shaking at 950 r.p.m. in an Eppendorf thermomixer.
·Puncture the cap of a 1.5-ml microcentrifuge tube, add 1.4 ml of LB inoculating with 40 ul of the overnight culture.
·Incubate the mixture at 30 °C for 2.5h with shaking at 950 r.p.m. in an Eppendorf thermomixer. At this stage, the OD600 should be between 0.35 and 0.40.
· Put the cells on ice (and keep them on the ice whenever possible).
·Spin the cells down at 9,000 rpm for 30 sec in the cooling centrifuge at 2°C.
·Discard the supernatant by decanting. Discard as much supernatant as possible.
· Resuspend the pellet in 1 ml of ice-cold 10% sucrose buffer (containing 2mM HEPES) by pipetting.
· Spin the cells at 10,000 rpm for 30 sec in the cooling centrifuge at 2°C.
·Discard the supernatant by decanting. Discard as much supernatant as possible.
· Resuspend the pellet in 1 ml of ice-cold sucrose buffer (containing 2mM HEPES) by pipetting.
· Spin the cells down at 11,000 rpm for 30 sec in the cooling centrifuge at 2°C.
· Discard the supernatant by decanting and leave about 30 µl.
· Add DNA to the cells and pipette the mixture into the chilled 1 mm electroporation cuvette.
· Set the electroporator to 1250 V, 10 µF, 600 Ohms. (This setting belongs to an Eppendorf® Electroporator 2510 using an electroporation cuvette with a gap of 1 mm. Other devices can be used, but the voltage has to be fixed at 1350V and the length of the pulse should be 5 ms.)
·Carefully knock the cuvette on the table to remove air bubbles and dry the metallic sides of the cuvette with a tissue. Do not touch the metallic sides with your hands.
·Place the cuvette into the holder of the electroporator, insert, and push the “pulse” button twice.
·Add 1 ml LB medium without antibiotics to the cuvette. Resuspend the cells carefully by pipetting up and down and pipette back into the reaction tube (avoid air bubbles in the suspension).
·Recover the culture at 37 °C in an Eppendorf thermomixer with shaking at 950 r.p.m. for 1 h 40min.
· Spin down the cells by centrifugation at 8,000g for 1 min at RT.
· Discard the supernatant by decanting, leaving about 100 ul of LB medium. Resuspend the cells by pipetting up and down.
· Streak the entire mixture with a sterile 10ul loop on one LB agar plate containing 15 ug ml−1 chloramphenicol and 10ug ml−1kanamycin.
week12
Chracterize the strength of different promoters by luciferase firefly assay
preparation of the overnight p.fluorescence culture transformed successfully
Inoculate the medium in the 1.5-ml tube with the several colonies of p.fluorescence cells that has been transformed successfully and incubate at 30 °C overnight with shaking at 950 r.p.m. in an Eppendorf thermomixer.
·Puncture the cap of a 1.5-ml microcentrifuge tube, add 1.8 ml of LB containing 5ug ml−1 chloramphenicol and inoculate with 150 ul of the overnight culture.
·Incubate the mixture at 30 °C for 2h with shaking at 950 r.p.m. in an Eppendorf thermomixer.
·Adjust the mixture to OD600 0.2
Preparation of bacterial cell lysate
·Add 10ul of 1M K2HPO4(Ph7.8) and 20mM EDTA to 90ul of transformed bacteria in a new 1.5ml microcentrifuge tube.
·Quick-freeze on dry ice, and then equilibrate to room temperature by placing the tube in room temperature water.
·Add 300ul freshly prepared lysis mix (1 volume of freshly prepared lysozyme and 2volumes of 2×CCLR with 5mg/ml BSA). Mix and incubate for 10 minutes at room temperature.
· Mix 20ul of cell lysate with 100ul of luciferase assay reafent and measure the light produced.
Measrue the light produced using GloMaxTM 96 Microplate Luminometer
Colleborate with seven Chinese iGEM teams to characterize the promoter strength under differdent conditions.
week13
modeling
Digest Pseudomonas putida S16 genome with SpeI restriction enzyme.
The purpose of this work is to achieve nicotine degradation in Pseudomonas fluorescences with the nicotine degradation gene cluster of Pseudomonas putida S16. Besides, we want to test if our promoter library could really work in regulating gene expression. The expression of nicotine degration gene cluster will be detected by the change of nicotine’s content through HPLC.
·Digest 50ug of S16 genome DNA with SpeI restriction enzyme in 400ul and incubate it at 37℃. Mix the following in a 1.5-ml microcentrifuge tube:
·Incubate the mix in 37℃ for 3h
·Add 400ul of phenol-chloroform-isoamyl alcohol mixture(25:24:1)to the 400ul digestion mixture from Step.1, mix it thoroughly by inversion to create an emulsion and centrifuge it at 9400r for 10min at RT.
·Using an end-cut wide-bore 1ml tip, gently pipette 300ul of the aqueous phase(top layer)into a new 1.5ml microcentrifuge tube.
·Add 20ul(1/15 volumes) of 3M sodium acetate(PH 7.5)and gently invert to mix.
· Add 800ul(2.5 volumes) of absolute ethanol and gently invert to mix.
· Centrifuge at 9400r for 2min at RT to pellet the genomic DNA.
·Discard the supernatant and invert the tube on a paper towl to let all of the liquid drain off the pellet.
· Wash the pellet by carefully adding 1ml of 70%(vol/vol)ethanol and centrifuge it at 9400r for 1min at RT.
·Discard the supernatant and invert the tube on a paper towl to let all of liquid drain off the pellet.
·Dry the pellet without shaking on an Eppendorf thermomixer at 45℃ for 10min until no liquid is left.
·Resuspend the DNA pellet in 12ul of autoclaved ddH2O and use 2ul to measure the concentration using the NanoDrop UV spectrophotometer. The concentration is 1ug/ul.
· Keep the DNA pellet at 4℃.
week14
Preparation of the p15A-Cm-Vector
·the PAGE-purified oligonucleotides used to amplify the P15A-vector were P15A-primer-F and P15A-primer-R
TABLE 2 Oligonucleotides used for PCR of the DNA segment
·Set up the PCR reaction using p15A-cm-tetR-tetO-hyg as a template. Typically, set up the PCR reaction (50ul in total) to obtain sufficient amounts of DNA.
· Carry out the PCR in a thermal cycle following the instructions below:
·Purify the PCR products using the TianGen PCR Purification Kit and elute into 30ul of autoclaved ddH2O. Quantify the elution using a NanoDrop UV spectrophotometer.
week15
Preparation of the pBBR1-km-Vector
·In order to make sure this DNA segment could replicate and be delivered to the next generation in Pseudomonas fluorescences, we need to transfer this gene cluster into a kind of shuttle vector called pBBR1-km-vector.
·Digest 50ug of pBBR1-km-hyg-ccdB plasmid with BamhI restriction enzyme in 400ul and incubate it at 37℃. Mix the following in a 1.5-ml microcentrifuge tube:
·Incubate the mix in 37℃ for 3h
·Add 400ul of phenol-chloroform-isoamyl alcohol mixture(25:24:1)to the 400ul digestion mixture from Step.1, mix it thoroughly by inversion to create an emulsion and centrifuge it at 9400r for 10min at RT.
·Using an end-cut wide-bore 1ml tip, gently pipette 300ul of the aqueous phase(top layer)into a new 1.5ml microcentrifuge tube.
· Add 20ul(1/15 volumes) of 3M sodium acetate(PH 7.5)and gently invert to mix.
· Add 800ul(2.5 volumes) of absolute ethanol and gently invert to mix.
·Centrifuge at 9400r for 2min at RT to pellet the genomic DNA.
·Discard the supernatant and invert the tube on a paper towl to let all of the liquid drain off the pellet.
· Wash the pellet by carefully adding 1ml of 70%(vol/vol)ethanol and centrifuge it at 9400r for 1min at RT.
·Discard the supernatant and invert the tube on a paper towl to let all of liquid drain off the pellet.
·Dry the pellet without shaking on an Eppendorf thermomixer at 45℃ for 10min until no liquid is left.
· Resuspend the DNA pellet in 12ul of autoclaved ddH2O and use 2ul to measure the concentration using the NanoDrop UV spectrophotometer. The concentration is 1ug/ul.
·Keep the DNA pellet at 4℃.
week16
Preparation of the amp-segment
In order to eliminate the influence of a segment similar to a part of our promoter sequence in liner-circular-recombination and two homology arms was designed. This sequence was synthesized by sangon .
The PAGE-purified oligonucleotides used to amplify the amp-biotics segment were amp-primer-F and amp-primer-R
Set up the PCR reaction using pR6k-amp-ccdB as a template. Typically, set up the PCR reaction (50ul in total) to obtain sufficient amounts of DNA.
Carry out the PCR in a thermal cycle following the instructions below:
Purify the PCR products using the TianGen PCR Purification Kit and elute into 30ul of autoclaved ddH2O. Quantify the elution using a NanoDrop UV spectrophotometer.
week17
Preparation of the promoter segment
In order to test if our promoter could regulate gene expression in Pseudomonas fluorescences. Therefore we put promoter 4、5、11 in the beginning of NicA2 gene and six homology arms was designed. This sequence was synthesized by sangon..
1)the PAGE-purified oligonucleotides used to amplify the promoter segment were primer promoter4F、promoter4R、promoter5F、promoter5R、promoter11F and promoter11R .
primer
Sequence(5’-3’)
promoter4F
ttgtttttgttttatcactcatCGGGCGGCCTCCAGGCTGCGTTGGACTTCGGCCTTGA
promoter4R
ggtttacaactactacgacctaactagaaggcgccgggacTTACGCCCCGCCCTGCCAC
promoter5F
 
promoter5R
 
promoter11F
tgcggctaaagccttcatttgtttttgttttatcactcatcacggatatcccctaagtt
promoter11R
ggtttacaactactacgacctaactagaaggcgccgggacTTACGCCCCGCCCTGCCAC
Set up the PCR reaction using pSB1C3-promoter4-cm、pSB1C3-promoter5-cm and pSB1C3-promoter11-cm as a template. Typically, set up the PCR reaction (50ul in total) to obtain sufficient amounts of DNA.
﻿﻿
Component
Amount(ul)
Final
Autoclaved ddH2O
22
 
2×primeSTAR  max premix
25
1×
Template,50ng ul-1
1
50 ng
Promoter XF(10uM)(X=4、5、11)
1
30 pmol
Promoter XR(10uM)
1
30 pmol
 
Total
50
 
Carry out the PCR in a thermal cycle following the instructions below:
﻿﻿
Cycle number
Denaturation
Annealing
Extension
Termination
1
94℃,4min
 
 
 
2-31
98℃, 15s
60℃, 15s
72℃, 10s
 
32
 
 
72℃, 4min
 
33
 
 
 
10℃, hold
Purify the PCR products using the TianGen PCR Purification Kit and elute into 30ul of autoclaved ddH2O. Quantify the elution using a NanoDrop UV spectrophotometer.
week18
Improve pBBR1 vector with ampicillin resistance gene, and insert promoter sequence into the beginning of NicA2 gene in order to get the final plasmid---pBBR1-km-amp-cm-promoter-nic.
Set up the restriction digestion mixture and incubate at the suggested temperature for 2 h.
component
Amount(ul)
final
Autoclaved ddH2O
4.8
 
1×
10× Digestion  buffer
1
DNA
4
Enzyme , 20 U ul−1
0.2
0.4 U ul−1
Total
10
 
Load the entire digestion mixture plus 2 ul of 6× loading dye in a 0.8% (wt/vol) agarose gel submerged in 1× TAE buffer. Run the gel using a field strength of ~4 V cm−1 for 30min.
Make a photo using the digital gel imaging system and check for release of the correct size fragment.
week19
Transfer the constructed plasmids----pBBR1-km-amp-cm-promoter-nic into Pseudomonas fluorescences and test their function with SDS-PAGE and Confocal Microscope.
week20-25
Standard part construction
1.Transforme the plasmid backbone, pSB1C3, into E.coli GB05red- gyrA462 .
2.Amplify linear amp-ccdB vector via PCR using PAGE-purified oligonucleotides containing 80-nt homology arms flanking the target DNA segment, R6K-amp-ccdB, and 20-nt standard PCR primers at the 3’ end.
Name
Sequence (5’→3’)
Length
R6K-amp-ccdb -1
AAAGGGCAGGGTGGTGACACCTTGCCCTTTTTTGCCGGACTGCAGCGGCCGCTACTAGTATTATATTCCCCAGAACATCA
 
80bp
 
R6K-amp-ccdb -2
TAAAAAAAATCCTTAGCTTTCGCTAAGGATGATTTCTGGAATTCGCGGCCGCTTCTAGAGTTTGTTTATTTTTCTAAATA
 
80bp
 
Set the PCR system as foolws:
Component
Amount(ul)
Final
Autoclaved ddH2O
22
 
2×primeSTAR max premix
25
1×
Template,50ng ul-1
1
50 ng
Syn-promoter-1  Oligo(10uM)
1
30 pmol
promoter-2 Oligo(10uM)
1
30 pmol
 
Total
50
 
3.Transform the linear fragment into E. coli GB05red- gyrA462thathas containe the plasmid pSB1C3-cm, where the recombination will be achieved be Redab after arabinose induction.
4.Prepare the pSB1C3-cm-amp-ccdb through mini-prep.
5.Screen correct recombinants by restriction analysis.
Component
Amount
ddH20
4.8ul
Cust smart
1ul
DNA
4ul
Restriction enzyme
0.2ul
Total
10ul
6.Amplify linear pSB1C3-cm-promoter vector via PCR using PAGE-purified oligonucleotides containing 40-nt homology arms flanking the target DNA segment, pSB1C3-cm-amp-ccdb, and 20-nt standard PCR primers containing part of promoter sequence at the 3’ end. What’s more, each pair of primers have 10-nt homology arms.
Oligonucleotides used in this step are as below:
﻿﻿
Name
Sequence  (5’→3’)
Length
09 SB1C3-PproD-1
CAAAAAACAAGCCGAGGATTATACCCGAGCTATGACCTCACTCTAGAAGCGGCCGCGAA
59
09 SB1C3-PproD-2
TTGTTTTTTGCCCGCCAAGACCTTCAGTGGATAGGGTGTTTACTAGTAGCGGCCGCTGC
59
11 SB1C3-PdnaA-1
CAAAAACAAGCGGCCATTGTAGCGATCAGCCGCCCACTTACTCTAGAAGCGGCCGCGAA
59
11 SB1C3-PdnaA-2
CTTGTTTTTGCCTCACCGGCTTTCAACTTAGGGGATATCCTACTAGTAGCGGCCGCTGC
59
12 SB1C3-Ppol-1 
CGCTAGAATAGCGGGACCGTTGACGACAAAGGGACAAGGTCTCTAGAAGCGGCCGCGAA
59
12 SB1C3-Ppol-2
TATTCTAGCGCCGGACCCGCCGAAAAGGTTGATCCCGCTCTACTAGTAGCGGCCGCTGC
59
13 SB1C3-P16s-1 
ACTTGGCTCAGCAATCGTTGGTTACATCTTTGATTTCTCGCTCTAGAAGCGGCCGCGAA
59
13 SB1C3-P16s-2
TGAGCCAAGTTTAGGGTTTTCTCAAAACCCAAAGATGTTTTACTAGTAGCGGCCGCTGC
59
14 SB1C3-Psig-1 
TAATTAGTCGCATTGTCATTGAGGCTTTATCGATTTGGCACTCTAGAAGCGGCCGCGAA
59
14 SB1C3-Psig-2 
CGACTAATTATCATTAGTGACACTCAGGCAGTCCGCGTCCTACTAGTAGCGGCCGCTGC
59
15 SB1C3-Patf-1 
ATCGGGCCGCGCAAACGGCGATCACTTTACGCCCGGCGACCTCTAGAAGCGGCCGCGAA
59
15 SB1C3-Patf-2
GCGGCCCGATTGTCCGGCCGCCCGAGTAAGGAGTCTTATCTACTAGTAGCGGCCGCTGC
59
17 SB1C3-memp-1
GATAGCGGGGCGGGCAGAAACGACAAACCCCGGATTAACCCTCTAGAAGCGGCCGCGAA
59
17 SB1C3-memp-2
CCCCGCTATCATTCGCCGCATCTTTCGAGGGGTATGACCGTACTAGTAGCGGCCGCTGC
59
19 SB1C3-Pami-1 
TTATATCGCCACTGCGCAAGTTAGTGCGCAACCCCAAACCCTCTAGAAGCGGCCGCGAA
59
19 SB1C3-Pami-2
GGCGATATAAGCGCCCTATAACAATTCCAGGGGTCGTTGTTACTAGTAGCGGCCGCTGC
59
21 SB1C3-Pabc-1
CGGGCGCCTGCCGGCCTGGGCCGGCAATTTTATTTGCGGGCTCTAGAAGCGGCCGCGAA
59
21 SB1C3-Pabc-2
CAGGCGCCCGGGGTCGGGCGACTTACAAGGAGCTTGTTGGTACTAGTAGCGGCCGCTGC
59
 
 
 
 
 
22 SB1C3-Pedo-1
TTTGGCGATCAACTGGTCAGCCACGCTCAGCGCGCCTTCACTCTAGAAGCGGCCGCGAA
59
22 SB1C3-Pedo-2
GATCGCCAAAGAGCGCGCAGCCAAGTAAGCGCCCGCCGCCTACTAGTAGCGGCCGCTGC
59
23 SB1C3-Ptox-1
GCGGGTCTTGGGTGTTAGAGCACTCAAGGCCCGCGTCTTTCTCTAGAAGCGGCCGCGAA
59
23 SB1C3-Ptox-2
CAAGACCCGCTAACCACAAGCAGCTAAACAGGAGCTGAATTACTAGTAGCGGCCGCTGC
59
24 SB1C3-Pdiv-1
GCAGTTCCAGACCTTGCTTGAGGGTGCGCCGGGTCAGCTCCTCTAGAAGCGGCCGCGAA
59
24 SB1C3-Pdiv-2
CTGGAACTGCTGGGCCTCAAAGTCCTGGAGCAAATGTAAGTACTAGTAGCGGCCGCTGC
59
25 SB1C3-Plip-1
TTTTTTTACTTCAGGCCTTGCCCGGAAAACATCAGAACCTCTCTAGAAGCGGCCGCGAA
59
25 SB1C3-Plip-2
AGTAAAAAAACCGGGGCTTCGGGCCCACGGGAGAAAAATATACTAGTAGCGGCCGCTGC
59
PCR process is as below:
Cycle number
Denaturation
Annealing
Extension
Termination
1
94℃,4min
 
 
 
2-31
98℃, 15s
59℃, 15s
72℃, 15s
 
32
 
 
72℃, 4min
 
33
 
 
 
10℃, hold
Set the PCR system as below:
Component
Amount(ul)
Final
Autoclaved ddH2O
22
 
2×primeSTAR max premix
25
1×
Template,50ng ul-1
1
50 ng
Syn-promoter-1 Oligo(10uM)
1
30 pmol
promoter-2 Oligo(10uM)
1
30 pmol
 
Total
50
 
Electroporate linear pSB1C3-cm-promoter vector into E. coli GB05dir cells, and enhance linear-linear homologous recombination.
Screen correct recombinants by restriction analysis.
SKLMT
Team
Members
Attributions
Sponsors
Judging
Project
Overview
Design
Red/ET
Results & Demonstration
Safety
InterLab
Parts
Basic Parts
Composite Parts
Part Collection
Improved part
Model
Overview
Process
Results
Software
Introduction
Principle
Human Practices
Overview
Integrated & Gold
Collaborations
Notebook
Overview
Protocol
Material
SKLMT
Useful Links
Main Page
Special pages
Contact us
SKLMT.iGEM.China@gmail.com
Address
Qingdao, Shandong
266200 Shandong University
