Team:SKLMT-China/Notebook Protocol
Protocol
Who are we?
Home
Notebook
Standard Protocol for Making Cells Electrocompetent
• use your fresh E.coli culture
• Put the cells on ice (and keep them on ice whenever possible).
• Spin the cells down at 9,000 rpm for 30 sec in the cooling centrifuge at 2°C.
• Discard the supernatant by decanting. Discard as much supernatant as possible.
• Resuspend the pellet in 1 ml of ice cold water by pipetting.
• Spin the cells at 10,000 rpm for 30 sec in the cooling centrifuge at 2°C.
• Discard the supernatant by decanting. Discard as much supernatant as possible.
• Resuspend the pellet in 1 ml of ice cold water by pipetting.
• Spin the cells down at 11,000 rpm for 30 sec in the cooling centrifuge at 2°C.
• Discard the supernatant by decanting and leave about 30 µl.
• Use the cells immediately.
Standard Protocol for Electroporation
• Add DNA to the cells and pipette the mixture into the chilled 1 mm electroporation cuvette.
• Set the electroporator to 1350 V, 10 µF, 600 Ohms. (This setting belongs to an Eppendorf® Electroporator 2510 using an electroporation cuvette with a gap of 1 mm. Other devices can be used, but the voltage has to be fixed at 1350V and the length of the pulse should be 5 ms.)
• Carefully knock the cuvette on the table to remove air bubbles and dry the metallic sides of the cuvette with a tissue. Do not touch the metallic sides with your hands.
• Place the cuvette into the holder of the electroporator, insert, and push the “pulse” button twice.
• Add 1 ml LB medium without antibiotics to the cuvette. Resuspend the cells carefully by pipetting up and down and pipette back into the reaction tube (avoid air bubbles in the suspension).
• Recover the cultures on an shaking incubator.
Standard Protocol for Mini-Prep
• (For future experiments in your lab, prepare a backup plate or backup tubes before starting, buffers are from Qiagen)
• Spin down the cells at 13,200 rpm for 1 minute.
• Discard the supernatant (remove as much as possible).
• Add 200 µl of P1 buffer (keep P1 at 4°C because of the RNaseA in the buffer).
• Resuspend the pellet by vortexing or use a pipette.
• Add 200 µl of P2 buffer (do not place P2 on ice, doing so will cause precipitation).
• Mix the tubes by inversion 5 times.
• Do not allow the lysis to proceed for more than 5 minutes.
• Add 200 µl of P3 buffer (keep P3 at 4°C).
• Mix by inversion 5 times.
• Spin the tubes at 13,200 rpm for 20 min.
• Prepare a fresh labeled 1.4 ml tube for each sample and add 500 µl of isopropanol .
• After centrifugation take out the supernatant with a pipette- avoiding white precipitate- and put it into the tube containing isopropanol.
• Mix by shaking the tubes vigorously.
• Spin the tubes at 13,200 rpm for 20 min.
• Discard the supernatant, invert the tube on a tissue.
• Carefully add 500 µl of 70% ethanol (centrifuge again for 5 min if pellet is only loosely attached to the wall of the tube).
• Discard the ethanol, invert the tube on a tissue.
• You may spin the tube again and remove remaining ethanol with a pipette.
• Dry the pellet for 10-15 min on a heating block (42C), do not overdry it (it is hard to dissolve then).
• For Plasmids: Dissolve the pellet in water (10-40 µl). For BACs: Dissolve the pellet in the digestion mix or dissolve in 8 µl water to send for sequencing.
Standard Protocol for getting Recombineering Proficient Cells
• Transform the expression plasmid pSC101-BAD-gbaA-tet into the E.coli strain, in which you want to perform recombineering.
• To prepare DNA of the expression plasmid, using any commercial plasmid preparation kit will result in very low yield due to the low copy number of the pSC101 plasmid. Please use our plasmid preparation protocol. To check your prepared expression plasmid rather perform a function test than a digestion analysis.
• To start overnight cultures, pick colonies from the respective plate.
• Inoculate 1.5 ml reaction tubes containing 1.0 ml LB medium plus the appropriate antibiotics (select for BAC or plasmid and for expression plasmid)
• Puncture the cap for aeration.
• Incubate the cultures with shaking at 30°C and 950 rpm o/n.
• Set up punctured reaction tubes containing 1.4 ml fresh LB medium supplemented with the same antibiotics as in the ON culture.
• Inoculate each with 40µl fresh overnight culture.
• Incubate the tubes at 30°C, 950 rpm for 2 hours.
• Add 20 µl 10% L-Arabinose to a final concentration of 0.1%-0.2%. This will induce the expression of the proteins necessary for recombineering.
• Leave some tubes without induction as negative controls
• Incubate at 37°C, 950 rpm for 40 min.
• Use 10% L-arabinose in dH2O, fresh or frozen in small aliquots at -20C
• Use 20 l stock solution per 1.4 ml LB for induction of recombination protein expression from pSC101-BAD-A-tet.
Standard Protocol for Designing Oligos for Recombineering
To target your BAC/genome/plasmid at the site(s) you choose, the homologous region must be included in the two oligonucleotides for amplification of a drug selectable cassette. Therefore each oligonucleotide consists of two (or, if desired, three) parts:
• Required Part A (A´ for the other oligonucleotide) is the homology region shared by the target molecule and the linear molecule. Choose the way that you want to engineer your target. Often, you want to delete a section of your target. This is accomplished by replacing this section with the selectable marker. The homology regions are the about 50 bp immediately either side of the deleted section. You can delete from 0bp (i.e. make an insertion) to > 100kb. The exact sequence of the homology regions can be chosen freely, according to which position on the target molecule will be modified.
• Optional Part B (B´ for the downstream oligonucleotide): This part of the oligonucleotide allows useful sequences, such as HA-tags, Myc-tags, His-tags, or restriction sites, multiple cloning sites, site-specific recombination target sites, etc., to be incorporated into the recombinant product B and/or B´. By design, these will be incorporated into the recombinant product exactly where desired. If the introduction of such operational sequences is not needed, this piece can simply be omitted from the oligonucleotide design.
• Required Part C (C´ for the downstream oligonucleotide): This piece, usually 20 to 24 nucleotides long, primes the PCR amplification of the linear molecule from the provided template.
The most efficient way to design primers for modifying BACs or chromosomes using the Red/ET system is to make the electronic map of the final construct by pasting in the desired sequence into the gene to be modified. Any program like Gene Construction Kit, DNA Strider, Gene Inspector etc. will do. Based on the final construct, copying and pasting the inserted sequence plus 50 bp immediately either side of the insertion in a new file gives the electronic map of the PCR product for recombineering. Copy the upstream primer and send the sequence to synthesizer. Copy the downstream primer and make the reverse complement.
We have no data to tell the relationship of the nucleotide context of the homologous to the recombineering efficiency. With primer design there is sometimes no choice of homology region, if for example you need to change one base pair of a particular gene. But many times you want to knock out a gene and there are no strict rules about where you must do it. Or sometimes you just want to have a marker in a particular region of a chromosome and don’t care precisely where it is. In these cases a 50 bp region that is roughly 50% GC in content is preferred. A long stretch of a single nucleotide should better be avoided.
It is a good idea to take the non-template directed extra A, that Taq polymerase puts on to the end of PCR products, into consideration. It is best to design the oligos in a way that recombination works with both the precise PCR product and the template extended-A products.
Standard protocol for separating a PCR-Product from its Template
Recombineering is often utilizes cassettes with selectable markers. Unwanted background selection is usually caused by residual intact plasmid that was used as PCR template. To reduce or eliminate recombineering background from the PCR template, you can use one of the following 5 methods or combine two of them if necessary.
Template digestion
Perform template plasmid digestion with one or more restriction enzymes with recognition sites, that are present in the plasmid backbone but absent in the drug selectable cassette to be amplified. The digested plasmid DNA needs to be checked by electrophoresis to verify efficient digestion. Furthermore, the digested DNA should be diluted to very low concentration to further reduce the background, since practically 100% digestion is always hard to achieve, especially when an inefficient restriction enzyme has to be used.
Template fragment isolation
Perform template plasmid digestion with one or more restriction enzymes with recognition sites, that are present in the plasmid backbone but absent in the drug selectable cassette to be amplified. Run the digested plasmid DNA on an agarose gel and isolate the fragment needed as template for PCR according to standard protocols.
Fragment isolation
After PCR amplification, separate the PCR-amplified targeting molecule from the template DNA by agarose gel electrophoresis. Isolate the PCR-amplicon using standard procedures. Commercially available fragment-isolation kits can be used according to manufacturer’s instructions.
DpnI digestion
The restriction enzyme DpnI has a 4-basepair recognition site (GATC), and only digests methylated DNA. Since a PCR product (i.e. linear targeting DNA) is not methylated, and template DNA usually is (upon growth in most commonly used bacterial cloning strains [dam+ strain]), DpnI only digests the template DNA and not the linear targeting molecule.
Non-replicable PCR template
Alternatively, suicide plasmids can be used as PCR template. As an example we use plasmids containing the R6K origin of replication. The pir gene encoded protein is essential to initiate the replication of the R6K plasmid. The R6K plasmid without pir gene can only replicate in a pir+-strains such as BW116, EPI100D-pir or EPI100D-pir116 cells, where the pir or the pir116 gene are present on the chromosome. Using R6K plasmids as the PCR template, consequently a pir--strain should be used for recombineering. Because the R6K plasmids cannot replicate in the recombineering strain (pir-), zero background selection is achieved without any effort of removal of the PCR template.
Standard protocol for degestion a plasmid
MasterMix
Cut Smart buffer
(n + 1) x
0.2 µl
PvuII-HF
(n + 1) x
0.2 µl
dH2O
(n + 1) x
4.8
Total volume
(n + 1) x
6.0 µl
• Add 6.0 µl of the Mastermix to 4.0 µl of the dissolved plasmid DNA
• Incubate for at least 1.5 h at 37°C.
• Load the whole mixture with loading buffer on a 0.7% agarose gel (containing EtBr) and perform electrophoresis. Take a picture.
Standard protocol for Firefly Luciferase assay
preparation of the overnight p.fluorescence culture transformed successfully
Inoculate the medium in the 1.5-ml tube with the several colonies of p.fluorescence cells that has been transformed successfully and incubate at 30 °C overnight with shaking at 950 r.p.m. in an Eppendorf thermomixer.
Puncture the cap of a 1.5-ml microcentrifuge tube, add 1.8 ml of LB containing 5ug ml−1 chloramphenicol and inoculate with 150 ul of the overnight culture.
Incubate the mixture at 30 °C for 2h with shaking at 950 r.p.m. in an Eppendorf thermomixer.
Adjust the mixture to OD600 0.2
Preparation of bacterial cell lysate
Add 10ul of 1M K2HPO4(Ph7.8) and 20mM EDTA to 90ul of transformed bacteria in a new 1.5ml microcentrifuge tube.
Quick-freeze on dry ice, and then equilibrate to room temperature by placing the tube in room temperature water.
Add 300ul freshly prepared lysis mix (1 volume of freshly prepared lysozyme and 2volumes of 2×CCLR with 5mg/ml BSA). Mix and incubate for 10 minutes at room temperature.
mix 20ul of cell lysate with 100ul of luciferase assay reafent and measure the light produced.
measrue the light produced using GloMaxTM 96 Microplate Luminometer
Standard protocol for HPLC
Preparation of samples
(1) Add 0.5ml sample into 2.0ml microcentrifuge tube;
(2) Add 1ml absolute ethyl alcohol to the tube and mix it with sample for 2min;
(3) Let the mixture rest 2-3 hours at room temperature;
(4) Spin down at 12,000 rpm for 20min;
(5) Filter the supernatant as sample with0.22mm organic filter membrane;
(4)Store the filtered liquid in a tube at 4℃.
Setup of HPLC
(1) Agilent 1100 Series High pressure liquid chromatograph with VWD, Hypersil ODS column and Automatic sampling system;
(2) Composition of mobile phase: Methanol: 1mM Sulfuric acid(5:95(v/v));
(3) Velocity of flow: 0.5ml/min;
(4) UV wavelength: 210nm；
(5) Temperature: 30℃；
(6) Sample size: 5ul
Make standard curve between target substrate concentration and corresponding peak area.
Standard protocol for SDS-PAGE
Sample preparation
(1) Prepare 2X loading buffer following Table :
Component
Volume
SDS
1g
Glycerol
5ml
Bromophenlo blue
50mg
Mercaptoethanol
2.5ml
Upper buffer
5ml
ddH2O
Total
50ml
(7) Carefully introduce solution into gel sandwich using a pipette, immediately after Solution D and TEMED are added, and avoid bubbles; (8) Carefully insert comb into gel sandwich until bottom of teeth reach top of front plate, and avoid bubbles; (9) Wait 30 min for gel to polymerize, remove the comb carefully; (10) Place the container in the canber.
Standard protocol for gel electrophoresis
(1) Add electrophoresis buffer to inner and outer reservoir, making sure that both top and bottom of gel are immersed in buffer;
(2) Introduce sample solution into well using a Hamilton syringe, and wash the syringe with ddH2O every time one sample is loaded;
(3) Attach electrode plugs to proper electrodes;
(4) Turn on power supply to 200V to start constant voltage electrophoresis;
(5) Turn off power supply when the dye front migrate to 1cm from the bottom of the gel;
(6) Dye with Coomassie Bule R-250 and then destain overnight.
Standard protocol for QPCR
Extraction of pure RNA
(1) Spin down at 12,000 rpm for 1min, discard the supernatant;
(2) Wash the pellet with ddH2O twice, spin down at 12,000 rpm for 1min, discard the supernatant;
(3) Add 1mlTrizol(Invitrogen) solution, blow and mix well, and add it into 1.5mlRNase-free-EP tube to make the cells lyse sufficiently, resting at room temperature for 5min;
(4) Add 200ul chloroform, shake vigorously and mix 30s, rest at room temperature 3-5min;
(5)Spin down at 14,000 rpm for 15min, transfer the upper aqueous phase to another new RNase-free-EP tube;
(6) Add equal volume isopropanol and mix thoroughly (gently 6-8 times). Rest at room temperature for 10min.
(7) Spin down at 14,000 rpm for 10min at 4℃, discard the supernatant;
(8) Wash the pellet with 75% ethanol for two times (12,000rpm centrifuge for 5min);
(9) Add appropriate amount of DEPC water (at least 15ul) to dissolve the precipitates.
Digest the rest DNA
component
amount
RNA
30ul
DNaseI
20ul
10xBuffer
10ul
ddH2O
39.5ul
RNasin
0.5ul
Total
100ul
Incubate at 37℃ for 30min
Inactive DNaseI at  62℃for 10min
(1) Add the same volume of phenol/chloroform, mix it upside down, place it at room temperature for 5 minutes, then centrifuge it for 15 minutes at 14,000 rpm, take the supernatant;
(2) Add equal volume of chloroform, mix it up and down, then centrifuge it for 15 minutes at 14,000 rpm, take the supernatant;
(3) Add equal volume of isopropanol, mix it gently (invert 6-8 times), and rest at -20℃ for 15min;
(4) Spin down at 14,000 rpm for 15min at 4℃, discard the supernatant;
(5) Wash the pellet with 75% alcohol for two times (centrifuge at 12000rpm for 5min);
(6) Add appropriate amount of DEPC water (at least 15ul) to dissolve the precipitates.
Reverse transcription
(1) Dispose the ddH2O and RNA solutions in the RNase free PCR tube :
(2)Blow the solution evenly and place it at 85℃for 5min to make RNA denatured;
(3)Dispose the following solutions in the same RNase free PCR tube:
component
amount
Oligo
0.5ul
Random primer
0.5ul
10mM dNTP
2.0ul
RNase inhibitor
0.5ul
5×buffer
4.0ul
M-MLV
0.5ul
Total
8.0u;
(2)Incubate the reaction solution at 30℃ for 10min;
(3)Incubate the reaction solution at 42℃ for 50min;
(4)Incubate the reaction solution at 85℃ for 10min;
(5)Store it at -20℃.
PCR
component
amount
ddH2O
4ul
SYBR Green PCR Master Mix
10ul
Upstream Primer
0.5ul
Downstream Primer
0.5ul
Total
15ul
SKLMT
Team
Members
Attributions
Sponsors
Judging
Project
Overview
Design
Red/ET
Results & Demonstration
Safety
InterLab
Parts
Basic Parts
Composite Parts
Part Collection
Improved part
Model
Overview
Process
Results
Software
Introduction
Principle
Human Practices
Overview
Integrated & Gold
Collaborations
Notebook
Overview
Protocol
Material
SKLMT
Useful Links
Main Page
Special pages
Contact us
SKLMT.iGEM.China@gmail.com
Address
Qingdao, Shandong
266200 Shandong University
