<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_CDI skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/CDI</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position:absolute;margin-left:-680px;left:50%;width:1360px;min-width:1360px;min-height:100%;background-color:white;padding-bottom:10px;"><DIV style="position:fixed;width:1360px;min-width:1360;height:70px;z-index:20;margin-top:0px;"><DIV style="display:table;min-width:1360px;height:100%;"><DIV style="display:table-row;"><DIV style="display:table-cell;background:#E0E0E0;padding-left:20px;"><NAV id="nav"><UL id="navigation" style="margin-top:9px;"><LI><A href="https://2015.igem.org/Team:Marburg/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Background">Fact Sheets</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Minicells">NUTRInity - Provide</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Curli">NUTRInity - Pick up</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/CDI">NUTRInity - Cut off</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Design">Future Application</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/InterLab">InterLab Study</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Measurement">Measurement Study</A></LI></UL><LI><A href="https://2015.igem.org/Team:Marburg/iGeneration">iGeneration</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Hessentag">Hessentag</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Killswitch">KillSwitch Statistics</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Medal Fulfillment">Medal Fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Meetup">MeetUp</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Gameofcells">Game of Cells</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Labbook">Lab Book</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Bibliography">Bibliography</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Killswitch">KillSwitch Statistics</A></LI></NAV></DIV></DIV></DIV></DIV><DIV><DIV style="position:relative;text-align:justify;z-index:1;background:white;font-size:13pt;line-height:150%;min-width:100%;max-width:100%;padding-top:132px; padding-left:235px; padding-right:80px;padding-bottom:70px;box-sizing: border-box;"><CENTER><TABLE><TBODY><TR><TD><DIV class="Anchor"><A href="#Aim" class="class1">Aim</A></DIV></TD><TD><DIV class="Anchor"><A href="#Design" class="class1">Project Design</A></DIV></TD><TD><DIV class="Anchor"><A href="#R" class="class1">Results</A></DIV></TD><TD><DIV class="Anchor"><A href="#O" class="class1">Outlook</A></DIV></TD><TD><DIV class="Anchor"><A href="#back" class="class1">Background</A></DIV></TD></TR></TBODY></TABLE></CENTER><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;margin-top:20px;"><H1>Aim</H1><P>
With our NUTRInity-cut off project, we want to develop a system that enables therapeutic bacteria to stably colonize the human gut. We aim to use the contact-dependent inhibition (CDI) system to <SPAN style="color:#FF8F45;"><B>cut off</B></SPAN> part of the natural gut population, thus creating a niche for our synthetic sub-population. In the context of NUTRInity, we also want to exploit the fact that CDI specifically inhibits growth of proteobacteria which have a higher occurrence in overweight people. By balancing the gut microbiota in this way, we want to fight obesity and associated diseases.
</P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Project Design</H1><P>
We have designed an <I>E.coli</I> system to create and establish a niche in the human gut microbiome targeting proteobacteria. 
Therefore, we created different constructs using the contact dependent growth inhibition system (CDI). This system inhibits growth of proteobacteria due to contact between CdiA and the receptor Protein BamA by which the inhibiting sub-domain from CdiA is transported into the cytoplasm of the target cell. The Cdi cells are protected by the CdiI protein, which disables the inhibitory C-terminus of CdiA. This sub-domain targets different important structures of proteobacteria, for instance the membrane or DNA. To do so, we chose a system with an inducible T5 Promoter and lac operator sites to control the expression of our growth inhibition system. To reduce leaky expression, we cloned additional lacI controlled by BBa_B0034 promoter into all our constructs to make it available for every strain. We decided to transform this construct into G10 Hicontrol strain, which is characterized by multiple lacI coding sequences. In a next step, we decided to change from the high copy plasmid pSB1C to the low copy plasmid backbone pSB4C5 in order to reduce the metabolic burden caused by the large CdiA protein (300 kDa). 

For further experiments, we cloned a red fluorescent protein into two constructs. In the first one, it is controlled by an inducible promoter while in the second it is controlled by a constitutive promoter, which enables us to distinguish between <I>E.coli</I> cells with and without the Cdi-system. For our target cells, we used G10 Hicontrol and additionally, we transformed the interlab study’s construct BBa_K1650001 encoding a green fluorescent protein into this strain in order to highlight them even more. 

For further applications, we want to use the Cdi-system to cut off a part of natural gut proteobacteria. In the future, we are aiming to implement metabolic pathways in our cells carrying the CDI-system which would enable us to produce beneficial compounds after creating a niche inside the human gut. <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 1:</B> 
Presentation of the six step cut off project design. 
</FIGCAPTION></FIGURE></P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Results</H1><P>
We constructed the CDI-system functionally in our <I>E. coli</I> lab strain and are able to induce the expression using IPTG. Proper functioning has been proven by co-cultivation of CDI<SUP>+</SUP> and CDI<SUP>-</SUP> cells in which the change of ratio over time has been observed (fig. 2).  <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 2:</B> 
Effect of the CDI expression in batch.The x-axis shows the time in coculture of CDI<SUP>+</SUP> and target cells and the y-axis shows the corresponding normalized cell count. The induced CDI<SUP>+</SUP> cells inhibit the target cells and get with the CDI system a growth advantage. 
</FIGCAPTION></FIGURE>
Additionally, we designed and built a functional prototype and showed that the CDI cells can also be used when filled in a capsule.
We used construct BBa_K1650001, which was utilized for the Interlab study for the engineering of target cells as well as construct BBa_B0007 for the engineering of inhibitor cells. Both plasmids were transformed into G10 Hi-control, a strain that expresses 60 fold LacI, to avoid leaky expression. 
If grown in separate wells of a 24 well plate, the CDI cells grow slightly slower due to a metabolic burden caused by the huge proteins of the CDI operon (fig. 3). <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 3:</B> 
Growth of induced CDI<SUP>+</SUP> cells (red dots) and of CDI<SUP>-</SUP> target cells (grey dots). The x-axis shows the time and the y-axis shows the corresponding OD of the culutres. The OD was mesured using the platereader Tecan infinite M1000 pro.
</FIGCAPTION></FIGURE>
The ability of CDI<SUP>+</SUP> cells to inhibit the growth of CDI<SUP>-</SUP> target cells was tested in an inhibition assay. The target and inhibitor cells were mixed and grown for 22 hours in the exponential phase. Constant conditions are accomplished by regular dilution using fresh medium. <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 4:</B> 
Scatterplot of the coculture of CDI<SUP>+</SUP> cells and GFP expressing target cells (data generated with flow cytometry). The x-axis shows the GFP intensity in arbitrary units and the y-axis shows the RFP intensity in arbitrary units. The experiment was done for about 20 hours three timepoints 0,10 and 24 hours are pictured. We observed a decreasing green fluorescent population over time (target cells) and an increasing red fluorescent population (CDI <SUP>+</SUP> cells). After induction we also observed a double postive population which represent CDI <SUP>+</SUP> cells in contact with target cells.
</FIGCAPTION></FIGURE>
The percentage of RFP labelled CDI<SUP>+</SUP> inhibitor and of GFP labelled CDI<SUP>-</SUP> cells was measured using a flow cytometer. In this experiment, we compared the change in CDI<SUP>+</SUP>/CDI<SUP>-</SUP> ratio upon IPTG induction. When expression of the CDI operon is induced, a distinct increase of the fraction of the CDI<SUP>+</SUP> cells occurs (fig. 4). This increase originates in growth inhibition of the target cells. So, the growth disadvantage caused by metabolic burden is compensated by the ability to actively decrease the growth rate of rivals (fig. 5). In a control experiment the uninduced CDI <SUP>+</SUP> cells were outcompeted by the target cells (data not shown). <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 5:</B> 
Effect of CDI expression on the ratio of CDI<SUP>+</SUP> cells to target cells. The x-axis shows the time in coculture of CDI<SUP>+</SUP> and target cells and the y-axis shows the corresponding normalized cell count. Over time the ratio for induced CDI<SUP>+</SUP> cells increases and decreases for uninduced CDI+ cells. The used method was flow cytometry.
</FIGCAPTION></FIGURE>
In addition to flow cytometry, we wanted to observe the growth inhibitory effect on a cellular level. Cells were grown on agar pads and pictures were taken using fluorescence microscopy. The images visually confirm the results of the measurements obtained by flow cytometry (fig. 6). <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 6:</B> 
 Growth inhibitory effect CDI carrying <I> E.coli</I> compared to a control strain without CDI. In subfigure A and B we visualized the growth inhibition of the CDI system (subfigure A shows the starting point of growth after mixing; subfigure B shows the growth state after two hours). In the subfigures C and D we visualized the growth of control strain (subfigure C shows the starting point of growth after mixing; subfigure D shows the growth state after two hours).   
</FIGCAPTION></FIGURE></P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Outlook</H1><P>
When we manage to establish our synthetic subpopulation in the human gut these cells represent a platform to produce any desired compound directly on-site.One interesting compound to test is <I>N</I>-acylphosphatidylethanolamine (NAPE). NAPEs are appetite-suppressing lipids that have shown the ability to reduce weight gain when added to the drinking water of mice (Chen <I>et al.</I> 2014). Once the bacteria are not supplied with the drinking water anymore the effect vanishes. Using the metabolic pathway to produce NAPES in cells that carry the CDI system would grant us the possibility to permanently produce NAPES thus reducing the danger of obesity. Concluding, a synthetic subpopulation gives us the possibility to create a platform for the implementation of various approaches to solve malnutrition and symptoms of obesity.
</P><P>
Additional to use the CDI cells to inhibit growth of gut bacteria and thus, creating a niche for genetically engineered bacteria, we want to use the basic principle to deliver any protein of interest into target cells. The CDI system showed to be modular. Inhibitory domains are variable, can be exchanged even among different species of bacteria and are similar in size. So, as a next project phase in the lab we want to replace the inhibitory domain of CdiA with an activator domain, for example a transcription factor, to make contact-dependent activation (CDA) possible. In the future we want to use this system to re-program cells of the gut microbiome to control the production of enzymes of native bacteria that are related to digestion and nutrient uptake and therefore ensure a healthy intestine. 
</P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;margin-bottom:30px;"><H1>Background</H1><P>
Approximately 10<SUP>14</SUP> cells inhabit the human gut, 10 times more than we have cells in our body. The gut microbiome is made up of 300 to 1000 different species. The composition of the bacteria in the gut is unique for every individual. The most abundant bacteria are bacteroidetes with 64.4 % followed by the firmicutes (24.4 %) and the proteobacteria. The latter have been shown to play a crucial role in obesity (L. R. Lopetuso <I>et al.</I>, 2014; N. R. Shin <I>et al.</I>, 2015). A healthy person has a relative abundance of 4.5 % proteobacteria, a person with a gastric bypass has 9.7 %. A person with metabolic disorders has 13.2 % and a person with inflamed and cancerous intestines has 14.9 %. The relatively recent finding that the gut microbiota is associated with disease and obesity is one of the “hot topics” in biosciences and is being intensively investigated (e.g. B. J. Marsland <I>et al.</I>, 2015; J. Walter, 2015).
</P><P>
Cdi (contact dependent inhibition) is a modular system in different strains of bacteria to deliver toxic subunits into target cells (Aoki <I>et al.</I>, 2010). The Cdi<SUP>+</SUP> cells inhibit the growth of the target cells thus, gain a growth advantage (Aoki <I>et al.</I>, 2005). The Cdi system is organized in an operon consisting of three genes: <UL style="list-style-type:square;"><LI>CdiB is a pore forming protein with a β-barrel structure and might be needed for secretion and assembly of CdiA (Aoki <I>et al.</I>, 2011).</LI><LI>CdiA is the toxin delivering protein. The C-terminus (CT) of CdiA acts as toxic domain.CDI loci can contain several orphan CdiA-CT and respective CdiI units behind the operon (Poole <I> et al.</I>, 2011).</LI><LI>The third gene, <I>cdiI</I>, the inhibitor to protect the cells against their own toxins.</LI></UL><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 7:</B> 
 Schematic of a CDI locus. <I>cdiB</I> (yellow), <I>cdiA</I> (green)and <I>cdiI</I>(blue) are organized in an operon. Various numbers of orphan CTs are located downstream of the operon (marked with asterisk). Adapted from Hayes et al. 2014.
</FIGCAPTION></FIGURE>
The structure of this operon differs from organism to organism and similar systems are found in various classes of  proteobacteria. The genes <I>cdiB</I> and <I>cdiA</I> are highly conserved up to a sequence coding for the amino acids VENN, followed by a variable sequence of the toxic CdiA-CT and <I>cdiI</I> which codes for the inhibitor of the preceding toxin. The toxin/inhibitor pairs are surprisingly diverse and many of them can be located downstream of the operon. These so-called orphan CdiA-CT and CdiI genes code for different toxins and inhibitors, respectively. By a yet unknown mechanism the orphan sequences can be swapped in and out of the operon so that a different toxin/inhibitor pair can be active at a given time. CDI<SUP>+</SUP> cells can thus possess an entire arsenal of different toxins. 
On a functional level, CdiA-CTs have been described that act as DNases, RNases, and pore-forming proteins.The natural system is only expressed in logarithmic growth state of <I>E.coli</I> while growth inhibition can occur in log-phase as well as in stationary phase.  
</P><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 8:</B> 
The contact dependent growth inhibition system. Only Target cells are inhibited by CDI<SUP>+</SUP> cells. The CDI<SUP>+</SUP> cell do not inhibit each other caused by the CdiI protein.  
</FIGCAPTION></FIGURE><P>After direct cell to cell contact, the CDI operon will be transcribed and translated. CdiB presumably plays a role in the CdiA assembly and the current hypothesis is that the CdiA-CT is not functional until secretion, cleavage and transport into the target cell. After CdiB forms a pore in the membrane of the host cell, the 320kDa CdiA protein is secreted and exposed on the outer membrane. Upon binding to the target cell’s BamA receptor in the outer membrane, the toxic domain is cleaved off of CdiA and is delivered via ArcB transporter to the cytoplasm (Beck <I> et al.</I>,2014; Webb <I> et al.</I> 2013). The transport mechanism of the toxin through the inner and outer membrane is still unclear (Aoki <I> et al.</I>,2008). If <I>E.coli</I> cells with the same Cdi system get into contact while exposing CdiA on their surface, the toxic sub-unit is nonetheless transported into the cytoplasm but gets inactivated by CdiI (Morse <I> et al.</I>,2012). The outer membrane receptor BamA is ubiquitous in most proteobacteria and determines the CDI system's specificity. </P><P>To conclude, the CdiA/CdiB mediated growth inhibition system is highly modular and specific and has the potential to be developed into a contact-dependent transport system from virtually any low Dalton proteins.</P></DIV></DIV><DIV style="position:absolute;text-align:center;z-index:2;background:#e0e0e0;line-height:150%;bottom:0px;padding-top:3px;margin-bottom:0px; padding-left:235px; padding-right:80px;min-height:60px;min-width:100%;max-width:100%;box-sizing:border-box;clear:both;"><SPAN style="font-size:8pt; color:white;"> iGEM Marburg - ZSM Karl-von-Frisch-Straße 16, D - 35043 Marburg</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>