<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_InterLab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/InterLab</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position:absolute;margin-left:-680px;left:50%;width:1360px;min-width:1360px;min-height:100%;background-color:white;padding-bottom:10px;"><DIV style="position:fixed;width:1360px;min-width:1360;height:70px;z-index:20;margin-top:0px;"><DIV style="display:table;min-width:1360px;height:100%;"><DIV style="display:table-row;"><DIV style="display:table-cell;background:#E0E0E0;padding-left:20px;"><NAV id="nav"><UL id="navigation" style="margin-top:9px;"><LI><A href="https://2015.igem.org/Team:Marburg/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Background">Fact Sheets</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Minicells">NUTRInity - Provide</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Curli">NUTRInity - Pick up</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/CDI">NUTRInity - Cut off</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Design">Future Application</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/InterLab">InterLab Study</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Measurement">Measurement Study</A></LI></UL><LI><A href="https://2015.igem.org/Team:Marburg/iGeneration">iGeneration</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Hessentag">Hessentag</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Killswitch">KillSwitch Statistics</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Medal Fulfillment">Medal Fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Meetup">MeetUp</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Gameofcells">Game of Cells</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Labbook">Lab Book</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Bibliography">Bibliography</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Killswitch">KillSwitch Statistics</A></LI></NAV></DIV></DIV></DIV></DIV><DIV><DIV style="position:relative;text-align:justify;z-index:1;background:white;font-size:13pt;line-height:150%;min-width:100%;max-width:100%;padding-top:132px; padding-left:235px; padding-right:80px;padding-bottom:70px;box-sizing: border-box;"><CENTER><TABLE><TBODY><TR><TD><DIV class="Anchor" style="width:130px;"><A href="#Aim" class="class1">Aim</A></DIV></TD><TD><DIV class="Anchor" style="width:130px;"><A href="#Design" class="class1">Design</A></DIV></TD><TD><DIV class="Anchor" style="width:160px;"><A href="#M&amp;M" class="class1">Materials &amp; Methods</A></DIV></TD><TD><DIV class="Anchor" style="width:140px;"><A href="#Me" class="class1">Measurement</A></DIV></TD><TD><DIV class="Anchor" style="width:140px;"><A href="#R" class="class1">Results</A></DIV></TD><TD><DIV class="Anchor" style="width:150px;"><A href="#D" class="class1">Discussion</A></DIV></TD></TR></TBODY></TABLE></CENTER><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;margin-top:20px;"><H1>Aim</H1><P>This year, the iGEM Team Marburg is participating together with many other teams in the InterLab Study. The aim of the InterLab Study is to investigate the reproducibility of BioBrick characterization by collecting data of the same experiment from iGEM Teams all around the globe. All teams are measuring the fluorescence of three different genetic devices constitutively expressing GFP differing in their promoter strength. These data will allow a comparison of different protocols and lab techniques regarding the variety of the three parts. We transformed the three devices into E. coli cells, diluted to an optical density (OD) of 0.5 at 600 nm using both a spectrophotometer and a plate reader, the fluorescence was measured with a plate reader. The obtained results showed that the promoter J23101 was the strongest, the promoter J23106 was weaker and the devices with promoter J23117 showed fluorescence barely above background.</P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Design</H1><P>According to the InterLab Study requirements we measured the three BioBrick parts with promoters of different strengths. The first device consists of the promoter J23101, the GFP coding sequence E0040 and the terminator B0015. The promoter J23101 was exchanged for the promotor J23106 and J23117 for the device 2 and 3, respectively. The promoters are part of a constitutive promoter family that has been introduced to iGEM by the Berkeley Team 2006. The promoters all have the same length and only differ in their sequence, which induces different strength of promoter activity. We used BBa_I20270 with the promoter J23151 as positive control and BBa_R0040 as a negative control, an empty plasmid with pTetR. E. coli DH5alpha was used as a chassis for characterization and as a control for the cells’ auto-fluorescence.

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 1:</B> Structure of InterLab Study Constructs </FIGCAPTION></FIGURE></P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Materials and Methods</H1><P>The parts for the three constructs used in the InterLab study (J23101, J23106, J23117 and I13504,) were taken from the iGEM distribution plates and transformed into NEB turbo cells. Afterwards, the plasmids were recovered through miniprep from an overnight culture. The plasmids were amplified via PCR and purified by gel extraction. The final constructs were then assembled using CPEC cloning (Circular Polymerase Extension Cloning). The correct identity of the resulting constructs was confirmed by analytical digest with PvuII and sequencing. DH5alpha was transformed with the final plasmid. Inoculation and cultivation was done following the InterLab Protocol. We prepared our devices and the controls by streaking them out on agar plates from glycerol stocks. We incubated the plates 18h overnight at 37°C. For that we used LB Agar from Roth without antibiotic for the empty DH5alpha cells and LB Agar from Roth with Chloramphenicol (c=34 µg/µl) for the remaining strains carrying the constructs and controls. We inoculated our strains carrying the devices and controls in 5 mL LB with Chloramphenicol in a 15 mL test tube. The empty DH5alpha cells were inoculated in 5 mL LB without antibiotic. For each sample we set up three biological replicas. The samples were incubated at 37°C at 300rpm for 16 h.</P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Measurement</H1><P>For the plate reader measurement we measured the OD600 of our overnight cultures and diluted them 1:10. As a reference we used pure LB plus Chloramphenicol and LB without antibiotic for the empty DH5alpha cells. We measured our construct strains and the control strains with both the Spectrometer Ultrospec 10 from Amersham Bioscience and the Microplate Reader from Tecan. For the measurement of the fluorescence in the plate reader, the samples were diluted to an OD600 of 0.5 within a deviation of 5%. In order to generate significant results of each device, 3 biological replica were measured. For the extra credit of the InterLab Study we also measured our samples three times in a row, in order to get technical replica.</P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Results</H1><P>Data of biological and technical replicas for each of the devices are summarized in Fig. 2. We observed a strong difference in fluorescence intensity between the tested promoters. The promoter J23101 was 17-fold higher in fluorescence intensity than promoter J23106, while J23117 was the lowest.

The fluorescence data show a strong difference between the expression profiles of the promoters. The promoter J23101 shows a 17-fold stronger fluorescence signal than promoter J23106. The weakest promoter is J23117.

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 2:</B> Platereader results. Initial OD measured in platereader, culture was then diluted for the final measurement and OD was verified in platereader. Construct 1 (J23101), Construct 2 (J23106), Construct 3 (J23117), Positive Constrol, Negative Constrol (ptet promoter in BB), Empty Strain. Each bar represents one technical replica. Each technical replica consisted of three biological replica.(</FIGCAPTION></FIGURE><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 3:</B> Platereader results. Initial OD measured in cuvette, culture was then diluted for the final measurement and OD was verified in cuvette and afterwards fluorescence was measured in the platereader. Construct 1 (J23101), Construct 2 (J23106), Construct 3 (J23117), Positive Constrol, Negative Constrol (ptet promoter in BB), Empty Strain. Each bar represents one technical replica. Each technical replica consisted of three biological replica.</FIGCAPTION></FIGURE></P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;margin-bottom:30px;"><H1>Discussion</H1>
By using the plate reader for measuring optical densities and fluorescence intensity, we obtained consistent data for all three biological replicates. There was no bleaching effect due to repetitive fluorescence measurements in technical replicas. In our early submission, the positive control exhibited a very low fluorescence barely above background fluorescence. We retransformed the positive control for our later studies in the measurement section (LINK) and observed higher fluorescence levels, similar to the ones from construct 2.
The InterLab study has been established within iGEM over the past years and aims to standardize and characterize BioBricks in more detail. In order to optimize the InterLab Study protocol we suggest some improvements. First, in order to avoid differences between devices, e.g. platereader and photometer, a normalization standard could be introduced in the distribution kit, similarly to the calibration solution you can buy from photometer distributors. The same can be applied to the fluorescence measurement. Second, the InterLab Worksheet should state if fluorescence data should be submitted as raw and/or normalized data (fluorescence intensity/OD). For increasing the transparency data policy raw data should preferably be included. Third, as we have shown in our own study <A href="https://2015.igem.org/Team:Marburg/Measurement" class="class2">Measurement Study</A> an internal fluorescent marker would be beneficial. Measuring the fluorescence from a diluted stationary phase culture shows a higher noise in gene expression compared to mid-log phase. Additional fluorescence measurement at an earlier time point may improve comparability between labs.

 


</DIV></DIV><DIV style="position:absolute;text-align:center;z-index:2;background:#e0e0e0;line-height:150%;bottom:0px;padding-top:3px;margin-bottom:0px; padding-left:235px; padding-right:80px;min-height:60px;min-width:100%;max-width:100%;box-sizing:border-box;clear:both;"><SPAN style="font-size:8pt; color:white;"> iGEM Marburg - ZSM Karl-von-Frisch-Straße 16, D - 35043 Marburg</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>