<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/Measurement</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position:absolute;margin-left:-680px;left:50%;width:1360px;min-width:1360px;min-height:100%;background-color:white;padding-bottom:10px;"><DIV style="position:fixed;width:1360px;min-width:1360;height:70px;z-index:20;margin-top:0px;"><DIV style="display:table;min-width:1360px;height:100%;"><DIV style="display:table-row;"><DIV style="display:table-cell;background:#E0E0E0;padding-left:20px;"><NAV id="nav"><UL id="navigation" style="margin-top:9px;"><LI><A href="https://2015.igem.org/Team:Marburg/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Background">Fact Sheets</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Minicells">NUTRInity - Provide</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Curli">NUTRInity - Pick up</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/CDI">NUTRInity - Cut off</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Design">Future Application</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/InterLab">InterLab Study</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Measurement">Measurement Study</A></LI></UL><LI><A href="https://2015.igem.org/Team:Marburg/iGeneration">iGeneration</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Hessentag">Hessentag</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Killswitch">KillSwitch Statistics</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Medal Fulfillment">Medal Fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Meetup">MeetUp</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Gameofcells">Game of Cells</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Labbook">Lab Book</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Bibliography">Bibliography</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Killswitch">KillSwitch Statistics</A></LI></NAV></DIV></DIV></DIV></DIV><DIV><DIV style="position:relative;text-align:justify;z-index:1;background:white;font-size:13pt;line-height:150%;min-width:100%;max-width:100%;padding-top:132px; padding-left:235px; padding-right:80px;padding-bottom:70px;box-sizing: border-box;"><CENTER><TABLE><TBODY><TR><TD><DIV class="Anchor" style="width:125px;"><A href="#Aim" class="class1">Aim</A></DIV></TD><TD><DIV class="Anchor"><A href="#Design" class="class1">Project Design</A></DIV></TD><TD><DIV class="Anchor"><A href="#R" class="class1">Results</A></DIV></TD><TD><DIV class="Anchor"><A href="#O" class="class1">Outlook</A></DIV></TD><TD><DIV class="Anchor"><A href="#back" class="class1">Background</A></DIV></TD><TD><DIV class="Anchor"><A href="#learn" class="class1">Recommendation</A></DIV></TD></TR></TBODY></TABLE></CENTER><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;margin-top:20px;"><H1>Aim</H1><P>This year, iGEM invited all the competing teams around the world to measure fluorescence from the same three genetic devices for GFP expression. Teams all around the world measured the fluorescence of the devices at on OD of 0.5. The reason for this study is to show the variability between labs and to create a so defined standard for constructs used in synthetic biology. Since standardization is one of the most important parts in synthetic biology, we wanted to show a way to make an overall characterization of devices. For that we extended the InterLab Study in our Measurement Study and characterized the promoters with diverse methods and in different constructs. We used plate reader, flow cytometry, proteomics and microscopy to measure the fluorescence in single cells and in populations. We introduced the devices in two different strains: DH5alpha and the wild type MG1655. We expressed RFP instead of GFP and normalized our data not only over the OD but in our double constructs also over the expressed RFP. We also showed the impact of evolution that can be seen in protein expression over time and quantified the noise of gene expression.  </P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Project Design</H1><P>We extended the Interlab Study constructs (Fig. 1) in order to be able to better characterize the promoter, and the expression on both single cell and population level. In order to see whether the coding sequence of a gene has an influence on the expression level, we build the construct <A href="http://parts.igem.org/Part:BBa_K1650003" class="class2">4</A> and <A href="http://parts.igem.org/Part:BBa_K1650004" class="class2">5</A>, where the GFP is replaced by an RFP (Fig. 2). Additionally we wanted to normalize not only to the OD, but also to an internal standard. Therefore we placed in two constructs a second fluorescent gene (RFP) with a strong promoter upstream of our GFP InterLab constructs 1 and 2 (Fig. 3). They are called constructs <A href="http://parts.igem.org/Part:BBa_K1650005" class="class2">6</A> and <A href="http://parts.igem.org/Part:BBa_K1650005" class="class2">7</A>. One of these double promoter-fluorescence gene construct was also used to measure the noise of our expression system. As not only the constructs themselves can lead to different expression levels but also the chassis, we used two common lab strains, Dh5alpha and MG1655 in order test our constructs. The expression levels were measured with a variety of different techniques and will allow a more comprehensive characterization and description of our constructs.
<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 1:</B> Structure of InterLab Study Constructs that were further characterized in our Measurement Study. They are composed of a constitutive promoter of different strengths, driving the expression of GFP.</FIGCAPTION></FIGURE><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 2:</B> Structure of extended InterLab Study constructs - replacement of coding sequence to RFP to test its influence on expression</FIGCAPTION></FIGURE><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 3:</B> Structure of double promoter and double fluorescence constructs to normalize over fluorescence and measure noise.Used were the dirst two promoters to express GFP and a very stron one to express RFP.</FIGCAPTION></FIGURE></P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Results</H1>

To get a more comprehensive characterization, we analyzed our InterLab study constructs with more techniques and additionally build constructs, that will give as more information about the behavior of the promoter. For the different methods, only the constructs were used that will provide us with additional insight for an improved part characterization.
<H2>Platereader</H2><P> The platereader results from the stationary phase of our constructs in MG1655 and DH5alpha for the fluorescence intensities of GFP are comparable to the ones we obtained for the Interlab Study. Like previously described, we diluted an over night culture to an OD of 0.5 and depending on the strain measured the fluorescence intensity of GFP(Fig. 4), RFP (Fig. 5) or both. When comparing the expression levels of GFP and RFP, it can be seen that the coding sequence influences those levels. Construct 1 and 4 have the same promoter as well as 2 and 5 but the expression levels are varying. Aditionally we observed that in most cases, the normalized promoter activity was higher in MG1655 than in DH5alpha. Taken together, we show that the coding sequence as well as the chassis play an important role for the characterization of a construct. In order to provide a comprehensive characterization we used both, commonly used fluorescent proteins like GFP and RFP as well as the widely used cloning strains DH5alpha and the wildtype strain MG1655 for our studies. 


</P><DIV style="padding-top:10px;"><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 4:</B> Fluorescence of GFP in stationary phase of constructs 1, 2, 3, 6 and 7 and the positive control used in the InterLab Study in MG1655 und DH5alpha.
</FIGCAPTION></FIGURE></DIV><DIV style="padding-top:10px;"><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 5:</B> Fluorescence of RFP in stationary phase in MG1655 und DH5alpha for the constructs 4, 5, 6 and 7.
</FIGCAPTION></FIGURE></DIV>
The timelapse measurement of our constructs reveals that fluorescence intensity increases over time. The used fluorescent proteins are not fused to a degradation tag, which may have led to an accumulation of GFP (Fig. 6) and RFP (Fig. 7) and hence to an increase of fluorescence intensity.  In accordance with the one-point measurements in the plate reader of cells in stationary phase,  we observed a variation between the two different chassis. Also the expression levels and the ratios of fluorescence intensities are consistent to previous results.

<DIV style="padding-top:10px;"><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 6:</B> Left Part: Green fluorescence over time in construct 1, 2, 3, 6, and 7 in DH5alpha. Right Part: Green fluorescence of same constructs in MG1655.</FIGCAPTION></FIGURE></DIV><DIV style="padding-top:10px;"><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 7:</B> Left Part: Red fluorescence over time in construct 4, 5, 6, and 7 in DH5alpha. Right Part: Red fluorescence of same constructs in MG1655.</FIGCAPTION></FIGURE></DIV>
Usually, platereader results are normalized over the OD. A more prize method is the normalization of the fluorescence intensities over an internal standard. With the internal standard we can reduce the variation of metabolic state of the cells, cell age and plasmid copy number of our fluorescent output. Therefore we constructed our double constructs 6 and 7, which have both the promoters of construct 1 and 2 for the expression of GFP and an additional promoter from the same promoter family, expressing RFP. The results for this normalization can be found in Fig. 8. Construct 6 showed reduced fluorescence of RFP, therefore we only focused on construct 7. This construct is a composite part and can be used as a standard for promoter characterization, as it allows a more precise normalization of fluorescence intensities of the promoter of interest and generate more reliable data. You can find some of our recommendations below. Here we show as an example the differences in normalization over OD and an internal standard for construct 7.  <DIV style="padding-top:10px;"><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 8:</B> Left Part: Green fluorescence of construct 7 normalized over RFP. Right part: Green fluorescence of construct 7 normalized over OD.</FIGCAPTION></FIGURE></DIV><H2>Flow Cytometry</H2><P> Although the platereader analysis allows us to quickly screen our constructs, we cannot analyze the expression level at the single cell level. To get more insights into the cell-to-cell variation within a population, we used flow cytometry. In accordance to our previous results, the single cell measurement of the fluorescence intensities of GFP (Fig. 9) show similar ratios between the different constructs. Construct 4 and 5 don't carry a green fluorescent. Their levels of fluorescent intensity are similar to an empty strain and hence show only background fluorescence. 

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 9:</B> Green fluorescence measured in Flow Cytometry of all our constrcuts in both MG1655 and DH5alspha.</FIGCAPTION></FIGURE></P><H2>Noise</H2><P> As previously mentioned, expression levels of proteins within a population may vary over time or within a cell population. The variation can be quantified by noise analysis. We therefore designed two constructs that carry our promoter of interest fused to RFP (strong and weak promoter, construct 3 and 4, respectively) in addition to a second strong constitutively active promoter fused to GFP. Flow cytometry was used to analyze the expression on single cell level over time (Fig. 10). From the obtained data we calculated the mean expression as well as the coefficient of variation (CV). The CV provides information about the variation of fluorescence intensity of the population -the lower the CV, the lower the noise. Our results reveal that the CV is the lowest during early-log phase and increases with higher ODs. This leads us to conclude that measuring expression levels of different constructs is optimal in log-phase, which may improve the InterLab Study Protocol. 

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 10:</B> Left part: Fluorescence intensity of GFP at different time points. Middle section: Cell count at different time points. Right part: Coefficient of variation for the expression of noise that shows the noise levels at the different time points and hence during growth.</FIGCAPTION></FIGURE></P><H2>Microscopy</H2><P> Single cell microscopy pictures gave us both an expression on the variability of gene expression levels as well as on the shape of the cells. If the cell shape varies from the empty strain, this usually gives a hint on the burden of a construct. The cells of our construct 1 are elongated and in some cases stopped dividing (Fig. 11). This is usually a sign that the cells are really stressed. This can be observed from this strong promoter. The weaker promoters of construct 2 (Fig. 12) and 3 (Fig. 13) don’t result in a variance in cell size and shape in comparison to the emtpy strain (Fig. 14). Additionally we used a computer algorithm, microbeTracker, in order to quantitatively analyze the fluorescence intensity of the cells. The results are showed as histograms. For construct 1 the microscopy settings were not optimal, as they were not adapted to the intensity and are hence saturated. The histogram gives as only an estimation of the real values. For construct 2 we see a high variation in fluorescence intensities and that a high population exhibits lower fluorescence intensity. Construct 3 that carries the weakest promoter shows a nice Gaussian distribution of the intensities, as we would have expected it for all the constructs. Even though this microscopy analysis can be optimized and should repeated, it still gives us important information about the cell state and can additionally be used to analyze the single cell fluorescence and to compare the data with flow cytometry.

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 11:</B> Fluorescence intensity histogram and Microscopy Picture of Construct 1.</FIGCAPTION></FIGURE></P><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 12:</B> Fluorescence intensity histogram and Microscopy Picture of Construct 2.</FIGCAPTION></FIGURE><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 13:</B> Fluorescence intensity histogram and Microscopy Picture of Construct 3.</FIGCAPTION></FIGURE><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 14:</B>  Microscopy Picture of Empty Cells.</FIGCAPTION></FIGURE><H2>Evolution</H2><P> Evolution is an aspect that is usually not taken into account when characterizing parts and constructs as constructs are normally used for short-term experiments. But as constructs might also be used for different purposes, long-term robustness of genetic devices needs to be ensured. We hence cultured our strains carrying construct 1-3 for 48h and measured the fluorescene intensity by flow cytometry every 12h and inoculated the strains in new flasks. The results exhibit that the stronger the promoter, the more and faster the fluorescence intensity decreases. We observed four replica for every construct and for construct 1 (Fig. 15), with the strongest promoter, two replica significantly loose expression of fluorescence proteins. For construct 2 (Fig. 16), one colony is reduced in fluorescence. The data for the last construct are not shown, as the fluorescence intensity didn’t change over our measurement period. In order to unravel the underlying reason for the decrease in fluorescence we sent extracted plasmids from each construct and replica for sequencing. The promoter sequence of the constructs, that display reduced fluorescence, mutated, which most probably lead to the reduction of fluorescence protein expression. The strains still expression at previously observed levels didn’t show any mutations in the promoter region. From the results of our evolution study we can conclude that the stronger a promoter, the higher is the probability of loss of function of the device. For long-term expression of genetic devices, design and engineering principles need to be incorporated (Sleight et al., 2010).

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 15:</B> Evolution study of Construct 1. Strain carrying construct 1 was cultivated over 48h and reinoculated and measured every 12h with flow cytometry. SHown are four different replica. The x-achses shows the fluorescence intensity. Below each replica are the sequencing results of each promoter. Highlighted sequence in the bottom part shows similarity with initial sequence. If not highlighted in black, sequence is mutated. </FIGCAPTION></FIGURE><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 16:</B> Evolution study of Construct 2. Strain carrying construct 1 was cultivated over 48h and reinoculated and measured every 12h with flow cytometry. SHown are four different replica. The x-achses shows the fluorescence intensity. Below each replica are the sequencing results of each promoter. Highlighted sequence in the bottom part shows similarity with initial sequence. If not highlighted in black, sequence is mutated. .</FIGCAPTION></FIGURE></P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;margin-bottom:30px;"><H1>Outlook</H1><P>In our follow up studies, we want to establish a standardization pipeline from construction until the provision of a data sheet. The registry of biological parts is the biggest collection of its kind, but many researchers that we have spoken to, hesitate using it. The reason for that is the lack in characterization and therefore a lack of quality. All iGEM Teams have to face this challenge and see it as a contribution to the field and the community of synthetic biology to provide a good characterization of their parts and constructs. However, there is a strong need for standardization and characterization facilities in synthetic biology. As teams submit the BioBricks around the world, there should be a synthetic biology standardization facility in each continent, so that the standardization effort becomes an international call in synthetic biology. We as the iGEM Team Marburg will continue our work on part characterization. We want to extend our efforts further, for example on the RNA level, to have indications on all levels of protein biosynthesis using fluorescent proteins as expression markers. Much more work is needed to fullfill our goal of perfect characterization of standard biological parts. </P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;margin-bottom:30px;"><H1>Background</H1><P>In synthetic biology, it is important that part characterization is consistent between different labs to be able to create well-defined standard parts whose behavior is predictable. One of the fundamental principles in Synthetic Biology is engineering. But different from electrical or mechanical engineering, Biology engineering makes use of life itself. Our biological constructs are self-replicating and there is an interaction between our circuits and the chassis that we choose to express them in.  Also most biological Parts are not as well defined and characterized as in other engineering disciplines. Because of that, one of the most challenging parts in the transition from science to an engineering field is to define standards. Every engineering discipline is very fond of standardization. In the classical mechanical engineering, standards are used to provide sufficient information about a part and the defined standards lead to an abstraction of an element’s behavior and the simplification of the design. Hence parts can be treated as a black box, which can easily be combined with others. The most common example for standardization in synthetic biology is the BioBrick standard generated by the iGEM competition. Another example is the recently introduced Standard European Vector Architecture. But as we can already see with this example, there is no uniform concept of DNA part standards. Besides the introduction of a common standard also the characterization of parts should be standardized. Both the BioBricks and the Standard European Vector Architecture are only construction standards. Developing a characterization standard is even harder to reach, as Biology is a complex science and each part and construct behaves differently in the different chassis. It is hence hard to define characterization procedures and protocols and to be able to compare them. One way of approaching the characterization is to test the different behaviors of a construct. In our measurement project we tried to follow a holistic approach in order characterize a part as complete as possible and to introduce a new standard of characterisation.</P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;margin-bottom:30px;width:991px;"><H1>Recommendation</H1><P>During our Measurement Study we learned important lessons that we want to share with the iGEM community to improve measurement and standardization of BioBricks. With our construct 7 (two promoters constitutively expressing both GFP and RFP) we submitted a BioBrick that is ideal for a promoter characterization. When you change the promoter that controls the GFP you can normalize your promoter by an internal standard. This reduces the variation of plasmid number in the cell (Karen et al., 2013). Also we showed the strong effect of the chassis on the expression level. It is hence important to not only characterize a part in one strain. For many BioBrick, such a characterization doesn’t exist or it isn’t even mentioned, which chassis was used. In order to fully characterize a part, there should be information about the state of the cell as well as how strong the promoter reduces the growth rate of the chassis. This is a good indication for the metabolic burden and therefore for the evolutionary stability. To this point we also haven’t found any characterization in the part registry that takes noise into account.
To make the measurement and characterization more successful and uniform, we, the iGEM Team Marburg 2015 suggest following points for improvement of standardization of BioBricks:
Set Standards that need to be followed:
<UL style="list-style-type:decimal;"><LI>Create a traffic light system (green – very good characterized, yellow – characterization needs to be improved, red – poor or no characterization) for each (new) BioBrick</LI><LI>Give more credit for the characterization of red parts</LI><LI> Mark Parts that have been proven 3 times to be not functional</LI><LI> Provide a standard characterization plasmid like our construct 7 for promoter characterization</LI><LI>Only parts with at least one type of characterization can be submitted to the registry</LI><LI> Establish a standards of how to characterise parts and constructs 
</LI></UL></P></DIV></DIV><DIV style="position:absolute;text-align:center;z-index:2;background:#e0e0e0;line-height:150%;bottom:0px;padding-top:3px;margin-bottom:0px; padding-left:235px; padding-right:80px;min-height:60px;min-width:100%;max-width:100%;box-sizing:border-box;clear:both;"><SPAN style="font-size:8pt; color:white;"> iGEM Marburg - ZSM Karl-von-Frisch-Straße 16, D - 35043 Marburg</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>