<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_Minicells skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/Minicells</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position:absolute;margin-left:-680px;left:50%;width:1360px;min-width:1360px;min-height:100%;background-color:white;padding-bottom:10px;"><DIV style="position:fixed;width:1360px;min-width:1360;height:70px;z-index:20;margin-top:0px;"><DIV style="display:table;min-width:1360px;height:100%;"><DIV style="display:table-row;"><DIV style="display:table-cell;background:#E0E0E0;padding-left:20px;"><NAV id="nav"><UL id="navigation" style="margin-top:9px;"><LI><A href="https://2015.igem.org/Team:Marburg/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Background">Fact Sheets</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Minicells">NUTRInity - Provide</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Curli">NUTRInity - Pick up</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/CDI">NUTRInity - Cut off</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Design">Future Application</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/InterLab">InterLab Study</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Measurement">Measurement Study</A></LI></UL><LI><A href="https://2015.igem.org/Team:Marburg/iGeneration">iGeneration</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Hessentag">Hessentag</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Killswitch">KillSwitch Statistics</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Medal Fulfillment">Medal Fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Meetup">MeetUp</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Gameofcells">Game of Cells</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Labbook">Lab Book</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Bibliography">Bibliography</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Killswitch">KillSwitch Statistics</A></LI></NAV></DIV></DIV></DIV></DIV><DIV><DIV style="position:relative;text-align:justify;z-index:1;background:white;font-size:13pt;line-height:150%;min-width:100%;max-width:100%;padding-top:132px; padding-left:235px; padding-right:80px;padding-bottom:70px;box-sizing: border-box;"><CENTER><TABLE><TBODY><TR><TD><DIV class="Anchor"><A href="#Aim" class="class1">Aim</A></DIV></TD><TD><DIV class="Anchor"><A href="#Design" class="class1">Project Design</A></DIV></TD><TD><DIV class="Anchor"><A href="#R" class="class1">Results</A></DIV></TD><TD><DIV class="Anchor"><A href="#O" class="class1">Outlook</A></DIV></TD><TD><DIV class="Anchor"><A href="#back" class="class1">Background</A></DIV></TD></TR></TBODY></TABLE></CENTER><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;margin-top:20px;"><H1>Aim</H1><P>
Within <SPAN style="color:#FF8F45;"><B>NUTRInity-provide</B></SPAN>, we aim to establish a shuttle system with countless applications. In our case, we want to produce micronutrients directly where the absorption in the human body takes place – in the small intestine. However, we do not want to release genetically modified microorganisms into our guts if it is not really necessary. Therefore, we want to find a system that can produce valuable compounds but is no longer able to divide. For this purpose, we want to establish a minicell production <I>E.coli</I> chassis and show that we can produce micronutrients like β-carotene. Furthermore, we built a modular system to transform any bacterial chassis into a minicell producer.
</P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Project Design</H1><P><FIGURE style="float:right;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:float:right;line-height:110%"><B>Figure 1:</B> Animation of the project design </FIGCAPTION></FIGURE>
In order to get a functional production shuttle in the human gut, we used the minicell-producing <I>E.coli</I> strain TB43 whose min-gene system was deleted.
We decided to utilize this kind of system to reduce risks of biosafety as far as possible for the minicells are not able to propagate. As comparison, we also tested a strain from the Keio collection with a deletion of <I>MinC</I>. We examined both strains via flow cytometry to evaluate minicell production. Originally, flow cytometry was developed for the examination of fluorescence in cells, which is why we had to optimize the measurement in order to differentiate between minicells and their parent cells. We decided to use both, size and complexity as distinctive features.
For the determination of the isolated minicells’ characteristics, we established a novel purification protocol using different filtration systems to get rid of chromosomal DNA containing cells. This was an important point in maintaining biosafety in terms of possible applications. Furthermore, we had to make sure that the minicells would not start dividing all of a sudden. For this reason, we performed a time laps experiment using fluorescence microscopy over several hours.
To see whether micronutrients can be produced by our chosen system, we transformed a plasmid encoding the four necessary genes (<I>crtY</I>, <I>crtE</I>, <I>crtI</I> and <I>crtB</I>) for the biosynthesis of β-carotene into the TB43 strain. β-carotene is a vital compound, which is necessary for the production of vitamin A in the human body. A lack can lead to a weak immune system or blindness which is still common in many developing countries.
After successful production of this nutrient, we thought of a way to make this non-living chassis suitable for as many applications as possible. For this purpose, we needed to knock down one part of the <I>Min</I>-system.  So, we designed a modular sRNA-encoding construct to target the minC-mRNA to generate the needed knock down strain.  
</P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Results</H1><H2> Proof of non-propagating minicell production</H2><P><FIGURE style="float:right;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:float:right;line-height:110%"><B>Figure 2:</B>   Two populations of cells shown in a size scatter plot</FIGCAPTION></FIGURE>
We proved, that the TB43 strain that has a deleted Min operon is producing minicells and that these minicells are not propagating. Further more, we show the round small phenotype of the minicells. This leads us to the conclusion that minicells reach all our set aims for a shuttle system. </P><P> 
By flow cytometry and by fluorescence microscopy we can show that minicells are produced (Fig. 2,3).
The minicells can be seen as a separated population in the side scatter that shows the complexity of the cells.
Minicells have a lower complexity as they don’t contain any chromosomal DNA. To determine their shape and to get a better understanding of their phenotype, we additionally examined the minicells by scanning electron microscopy (Fig.4). Using strains with a fusion of HU, a histone-like protein, with the fluorescence protein GFP, we can show that the minicells indeed do not contain any chromosomal DNA (Fig.3). 
Performing microscopic time-laps experiments we can confirm on the one hand the expression of fluorescent proteins and on the other hand that minicells do not propagate anymore.
This is a crucial aspect for the establishment of minicells as a tool with an application in the human gut. 
</P><FIGURE style="float:right;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:float:right;line-height:110%"><B>Figure 4:</B> SEM image of the formation of minicells  </FIGCAPTION></FIGURE><FIGURE style="text-align:float:left;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:float:left;line-height:110%"><B>Figure 3:</B> Producing of minicells in a time-laps experiment with  TB43 HU-GFP. Additional proof of non propagating of the minicells </FIGCAPTION></FIGURE><H2>Purification</H2><P> 

We were able to separate minicells from the parental strain with a purification of 98.8%. After non-successful efforts to separate the minicells by differential centrifugation, we developed our own purification method. We established a protocol with combination of different filter systems – first to exclude parental cells followed by a concentration process. Plating our concentrated cells indicates that there is only a slight background contamination with the parental cells. We will further improve this method in the future.

<FIGURE style="float:right;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:float:right;line-height:110%"><B>Figure 5:</B> Ratio of minicells to normal cells before and after purification  </FIGCAPTION></FIGURE><FIGURE style="text-align:float:left;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:float:left;line-height:110%"><B>Figure 6:</B> Amount of minicells in the sample measured by  flow cytometry  in the overnight culture and after the purification. </FIGCAPTION></FIGURE></P><H2>Production of nutrients</H2><P><FIGURE style="float:right;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:float:right;line-height:110%"><B>Figure 7:</B> Production of β-carotene  </FIGCAPTION></FIGURE>
Here we can show that the minicell producing strain TB43 can be altered synthesize β-carotene. To determine the production of nutrients in our strain TB43, we successfully optimized the plasmid, encoding the necessary genes for the biosynthesis of β-carotene (<A href="http://parts.igem.org/Part:BBa_K1650010">BBa_K1650010</A>).  In Fig.7 we show that β-carotene, which is of orange color, is expressed, when a culture of TB43- β-carotene is plated. This leads us to the assumption that also the minicells produced by the TB43-β-carotene strain contain the wanted micronutrients. </P><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 8:</B> Design of the biosynthesis </FIGCAPTION></FIGURE><H2>Induction of minicells</H2><P>We can prove that the minicells can react to an inducer with the production of a fluorescent protein. By using an inducible promoter (pLAC) that controls a fluorescent protein (RFP), we could follow the production of RFP in time-laps fluorescent microscopy (Fig. 9). Our results show that non-fluorescent minicells became fluorescent after induction. This result demonstrates that minicells are can perform protein-biosynthesis and are able respond upon induction.  Although most of the minicells only became fluorescent after induction, some cells constitutively expressed RFP, likely due to stochastic distribution of repressor molecules.</P><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 9:</B> Inducing minicells in a time-laps micoscopy. Top: point of time: 0 Bottom: point of time: 1 h </FIGCAPTION></FIGURE><H2>Modular device to transform to minicell producer</H2><P>We were able to develop a tool for producing minicells out of any desired strain. For that a plasmid, which expresses the modular sRNA anti-minC, with a complementary sequence to the mRNA of minC, was constructed based on BBa_K1137009.  NEB Turbo cells were successfully transformed with the sRNA plasmid(<A href="http://parts.igem.org/Part:BBa_K1650011">BBa_K1650011</A>). Upon binding of the sRNA to the mRNA, the mRNA couldn’t be processed by ribosomes anymore, thus, leading to a knock down in <I>MinC</I>. Without <I>MinC</I> cell division is not located at midcell anymore and minicells are formed. The formation of minicells was confirmed by flow cytometry, where we used non-transformed cells grown to mid-log phase as a negative control (Fig.10) </P><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 10:</B> Left: NEB Turbo cells transformed with anti-minC sRNA. Right: normal NEB Turbo </FIGCAPTION></FIGURE></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;"><H1>Outlook</H1><P>As the chemical synthesis of many compounds is still a problem due to inefficient reactions, the need of chemicals produced by microorganisms has been aimed for over the last decade. Especially, the production of pharmaceuticals is an important aspect. Microorganisms might also produce dietary supplements in the near future.

We want to deliver the minicells in form of a pill, which is resistant to stomach acid and dissolves as soon as it gets into the duodenum. There, the minicells will produce the micronutrients and excrete them into the intestines. We think that this is a better method for a successful supply of micronutrients in comparison to an increase of the concentration in food as the absorption of for example dietary β-carotene out of food is not very efficient. This is caused by strong binding to many different biomolecules due to the inflexibility of its conjugated structure.

To ensure biosafety for both, the consumer and the environment, purification of the minicells has to be conducted. This prevents settlement of engineered bacteria in the human gut and their propagation after excretion. We mainly use different filters for the purification to get rid of ordinary cells on the one hand and to increase the concentration of the minicells on the other.

We want to make minicells suitable for everyone as they have the potential to act as a chassis with a lower risk compared to ordinary bacteria. Thus, they fit for multiple applications. Our idea is a sRNA encoding plasmid which silences MinC. With this plasmid, every strain could be brought to the production of minicells by a simple transformation.

To achieve all the described future goals, further experiments have to be conducted. Most of these experiments were planed but haven’t been achieved so far. The first step is to use FACS with a cell sorter to get a 100% separation of minicells from cells containing chromosomes. This needs to be ensured so that our further analytics are not contaminated by non-minicells. Our facility will get a cell sorter soon, which will allow us to proceed with this improved purification. The next step is to analyze the metabolic activity of the minicells using metabolic approaches. For the first proof, we are going to test if we can observe glycolysis in the minicells. Furthermore, we are going to test if we can induce whole metabolic pathways e.g. the one for β-carotene in the purified minicells. If these tests are successful, we can use the minicells even for the production for growth inhibiting products as no cell proliferation is needed within the minicells.</P></DIV><DIV style="position:relative;display:inline-block;padding:15px;background:#F8F8F8;border-style:solid;border-color:#FF8F45;border-width:2px;border-radius:10px;text-align:justify;font-size:13pt;line-height:150%;margin:10px;margin-bottom:30px;"><H1>Background</H1><P>Bacterial minicells are produced by asymmetric cell division, which is caused by a knock out of the <I>MinCDE</I>-system. Naturally, this system regulates symmetric cell division [Jivrajani <I>et al.</I><B>2013</B>]. The <I>Min</I>-proteins bind to the cell poles and oscillate between them. <I>MinC</I> functions as inhibitor for <I>FtsZ</I>, which forms the Z-ring and ultimately initiates division of the cell. This symmetric division ensures that each cell gets a copy of the chromosome and all other cell components and thus, is able to live on its own. If this system is disturbed, cell division occurs randomly leading to the formation of minicells.

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 11:</B> Normal functional min-system </FIGCAPTION></FIGURE>

As only a <I>MinC</I> knock down is needed to get minicells, we looked for a simple and modular solution. The knock down of gene expression through a small regulatory RNA (sRNA) system was already shown in iGEM by the Team Paris Bettencourt 2013 which was a great success for knock downs of target genes and was based on the Na <I>et al.</I> 2013 publication. The sRNA is processed by the <I>Hfq</I> protein and then, can bind to the mRNA where it induces its degradation and stops the protein biosynthesis. Our system was designed according to the guidelines of the paper: a sRNA that binds with 24 bp to the <I>MinC</I> mRNA (Figure 11).

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:centered;line-height:110%"><B>Figure 12:</B> Normal functional min-system </FIGCAPTION></FIGURE>

Some characteristics of minicells are – as the name suggests – the smaller size compared to their parent cells and the lack of chromosomal DNA. Thus, they are not able of further propagation [Adler <I>et al.</I><B>1967</B>]. Apart from that, they contain all other parts of the parent cells [Jivrajani <I>et al.</I><B>2013</B>]. 
Previous applications of minicells include drug delivery systems for the treatment of tumor cells to avoid inflammatory reactions which occur often when the drug is injected intravenously [MacDiarmid <I>et al.</I><B>2007</B>]. Also, the usage in vaccines as antigen delivery systems were tested successfully [Carleton <I>et al.</I><B>2013</B>]. We did not find any evidence for toxicity or unwanted immunoreactions. This led us to the conclusion that the system could be FDA approved in the future. 
To prevent infections as far as possible, minicells have to be isolated from their parent cells for these medical applications. Over the last decades, several methods were introduced. Chronologically, the first used technique was density gradient centrifugation [Adler <I>et al.</I><B>1967</B>]. Because of limitations in sample sizes and low yields, antibiotic treatments of the minicell-producing strains were performed [Levy <B>1970</B>]. Many antibiotics, e.g. Penicillin, kill bacteria during cell division, which is why they do not affect minicells. Lately, the treatment with antibiotics was combined with filtration to increase purification efficiency [Jivrajani <I>et al.</I><B>2013</B>]. In the future, we will include aspects of those techniques in our purification methods to get less “normal” cells when we are selecting for minicells. 
</P></DIV></DIV><DIV style="position:absolute;text-align:center;z-index:2;background:#e0e0e0;line-height:150%;bottom:0px;padding-top:3px;margin-bottom:0px; padding-left:235px; padding-right:80px;min-height:60px;min-width:100%;max-width:100%;box-sizing:border-box;clear:both;"><SPAN style="font-size:8pt; color:white;"> iGEM Marburg - ZSM Karl-von-Frisch-Straße 16, D - 35043 Marburg</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>