<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Marburg_Labbook_CDI skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Marburg/Labbook/CDI</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV style="position:absolute;margin-left:-680px;left:50%;width:1360px;min-width:1360px;min-height:100%;background-color:white;padding-bottom:10px;"><DIV style="position:fixed;width:1360px;min-width:1360;height:70px;z-index:20;margin-top:0px;"><DIV style="display:table;min-width:1360px;height:100%;"><DIV style="display:table-row;"><DIV style="display:table-cell;background:#E0E0E0;padding-left:20px;"><NAV id="nav"><UL id="navigation" style="margin-top:9px;"><LI><A href="https://2015.igem.org/Team:Marburg/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Background">Fact Sheets</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Minicells">NUTRInity - Provide</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Curli">NUTRInity - Pick up</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/CDI">NUTRInity - Cut off</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/InterLab">InterLab Study</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Measurement">Measurement Study</A></LI></UL><LI><A href="https://2015.igem.org/Team:Marburg/iGeneration">iGeneration</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Hessentag">Hessentag</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Killswitch">KillSwitch Statistics</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/FutureApplication">Future Application</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Medal Fulfillment">Medal Fulfillment</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Meetup">MeetUp</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Gameofcells">Game of Cells</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Labbook">Lab Book</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Bibliography">Bibliography</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Members">Members</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Gallery">Gallery</A></LI><LI><A href="https://2015.igem.org/Team:Marburg/Killswitch">KillSwitch Statistics</A></LI></NAV></DIV></DIV></DIV></DIV><DIV><DIV style="position:relative;text-align:justify;z-index:1;background:white;font-size:13pt;line-height:150%;min-width:100%;max-width:100%;padding-top:132px; padding-left:235px; padding-right:80px;padding-bottom:70px;box-sizing: border-box;"><H1>15/05/18</H1><H2><A href="https://2015.igem.org/Team:Marburg/Protocols/Protocol13">PCR</A></H2><TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>PCR </TD><TD>PCR Mix 1 </TD><TD>PCR Mix 2</TD></TR><TR><TD class="col">GXL buffer</TD><TD class="col">10 uL </TD><TD class="col">10 uL  </TD></TR><TR><TD class="col">dNTPs (from GXL kit</TD><TD class="col">4 uL</TD><TD class="col">4 uL</TD></TR><TR><TD class="col">GXL polymerase</TD><TD class="col">2uL</TD><TD class="col">2uL</TD></TR><TR><TD class="col">ICD001 (10 uM)</TD><TD class="col">0.5uL</TD><TD class="col">1uL</TD></TR><TR><TD class="col">ICD007/ICD008 (10 uM)</TD><TD class="col">0.5uL</TD><TD class="col">1uL</TD></TR><TR><TD class="col">EC93 genomic DNA (15 ng/uL)</TD><TD class="col">0.5 uL</TD><TD class="col">1 uL</TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">32.5uL</TD><TD class="col">29.5uL</TD></TR></TBODY></TABLE><TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>Program </TD><TD> time </TD></TR><TR><TD class="col">Start cycle (30x)</TD></TR><TR><TD class="col">98 °C </TD><TD class="col">10 s </TD></TR><TR><TD class="col">60 °C</TD><TD class="col">15 s</TD></TR><TR><TD class="col">68 °C</TD><TD class="col">2 min</TD></TR><TR><TD class="col">close cycle</TD></TR><TR><TD class="col"> 8 °C</TD><TD class="col">store</TD></TR></TBODY></TABLE><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 1:</B> 

Ladder, PCR 1, PCR 2

</FIGCAPTION></FIGURE><P>
Ran of analytical gel. No bands were observed.
</P><H1>15/05/19</H1><H2><A href="https://2015.igem.org/Team:Marburg/Protocols/Protocol13">PCR</A></H2><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 2:</B> 
Ladder, PCR 1, PCR 2

</FIGCAPTION></FIGURE><P>
Bands visible! Lanes 1 and 2 right of the ladder should have a 10506 bp band. Lanes 3 and 4 right of the ladder should have a 11414 bp band (gel description should read EC93 instead of EC90).

PCR cleanup (according to Thermo Scientific protocol), elution in 21 ul.

Concentrations according to Nanodrop: EC93 w/ 001+007: 92 ng/ul; EC93 w/ 001+008: 57 ng/ul
</P><H1>15/05/21</H1><H2><A href="https://2015.igem.org/Team:Marburg/Protocols/Protocol13">PCR</A></H2><P>
Primers ICD014 and ICD015 arrived! These will be used to amplify the Pick up backbone with overlaps for the full-length Cdi operon. PCR should yield a 3391 bp fragment. 
<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>PCR </TD><TD>PCR Mix 1 </TD></TR><TR><TD class="col">GXL buffer</TD><TD class="col">10 uL </TD></TR><TR><TD class="col">dNTPs (from GXL kit</TD><TD class="col">4 uL</TD></TR><TR><TD class="col">GXL polymerase</TD><TD class="col">2uL</TD></TR><TR><TD class="col">ICD0014 (10 uM)</TD><TD class="col">1 uL</TD></TR><TR><TD class="col">ICD0015 (10 uM)</TD><TD class="col">1uL</TD></TR><TR><TD class="col">1:10 diluted plasmid X</TD><TD class="col">1 uL</TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">31 uL</TD></TR></TBODY></TABLE><TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>Program </TD><TD> time </TD></TR><TR><TD class="col">Start cycle (30x)</TD></TR><TR><TD class="col">98 °C </TD><TD class="col">10 s </TD></TR><TR><TD class="col">60 °C</TD><TD class="col">15 s</TD></TR><TR><TD class="col">68 °C</TD><TD class="col">35 s</TD></TR><TR><TD class="col">close cycle</TD></TR><TR><TD class="col"> 8 °C</TD><TD class="col">store</TD></TR></TBODY></TABLE></P><P>
Turned out that the template was wrong. Inoculated 3 ml LB+chloramphenicol for overnight culture of correct Curly clone (clone No. 3) for Miniprep tomorrow.
</P><H1>15/05/22</H1><H2>PCR, Gibson Assembly and Transformation </H2><P>
Miniprep of 1.5 ml of Curly clone No. 3 according to Thermo Scientific Protocol. 

PCR with primers ICD014 and ICD015 and freshly prepped Curly plasmid to amplify the backbone for the CDI plasmid (pICD001).
Reaction mix and PCR program see 15-05-21.

Analytical gel: Correct size! (3391 bp)

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 3:</B> 
Ladder, - , PCR 

</FIGCAPTION></FIGURE>
Proceeded with PCR clean up with subsequent Nanodrop measurement: 

Gibson assembly of Curly plasmid/ICD014+015 PCR fragment and CDI operon (EC93 genomic DNA with ICD001 +008)
G.A. reaction mix:

<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>Gibson mix </TD><TD> volume </TD></TR><TR><TD class="col">Insert</TD><TD class="col">2.65ul (= 151ng = 0.02 pmol) </TD></TR><TR><TD class="col">Backbone</TD><TD class="col">0.54ul (= 45ng = 0.02 pmol) </TD></TR><TR><TD class="col">G.A. master mix </TD><TD class="col">10 uL</TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">6.81 uL</TD></TR></TBODY></TABLE>
G.A program:
<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>temperature </TD><TD> time </TD></TR><TR><TD class="col">50 °C</TD><TD class="col">15 min</TD></TR><TR><TD class="col">16 °C</TD><TD class="col">store </TD></TR></TBODY></TABLE>

Electroporation of Gibson assembly mix with NEB turbo 
	• Thaw NEB turbo cells on ice 
	• Dilute G.A. mix 1:3 (5ul mix + 10ul ddH2O)
	• Add 2ul of dilution to 25ul of NEB turbo cells 
	• Transfer the cells to electroporation cuvette 
	• Electroporate and add 975ul SOC medium immediately 
	• Recover 1h at 37°C 
	• Spin down briefly and discard supernatant 
	• Plate on chloramphenicol plate </P><H1>15/05/26</H1><H2>PCR, Gibson Assembly and Transformation </H2><P>

On 15-05-22 DpnI digestion has been forgotten, so the transformed cells probably carry the Curly plasmid instead of the Gibson assembly product. So we repeated the procedure including DpnI digestion. 
PCR of Curly backbone: See 15-05-21 
DpnI digestion program:<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>temperature </TD><TD> time </TD></TR><TR><TD class="col">37 °C</TD><TD class="col">30 min</TD></TR><TR><TD class="col">16 °C</TD><TD class="col">store </TD></TR></TBODY></TABLE>
(No enzyme deactivation was carried out)

Gibson assembly of Curly plasmid/ICD014+015 PCR fragment and CDI operon (EC93 genomic DNA with ICD001 +008)
G.A. reaction mix:<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>Gibson mix </TD><TD> volume </TD></TR><TR><TD class="col">Insert</TD><TD class="col">2.65ul (= 151ng = 0.02 pmol) </TD></TR><TR><TD class="col">Backbone</TD><TD class="col">0.51ul (= 45ng = 0.02 pmol) </TD></TR><TR><TD class="col">G.A. master mix </TD><TD class="col">10 uL</TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">6.84 uL</TD></TR></TBODY></TABLE>

G.A program:
<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>temperature </TD><TD> time </TD></TR><TR><TD class="col">50 °C</TD><TD class="col">15 min</TD></TR><TR><TD class="col">16 °C</TD><TD class="col">store </TD></TR></TBODY></TABLE>

Electroporation of Gibson assembly mix with NEB turbo 
	• Thaw NEB turbo cells on ice 
	• Add 2ul of Gibson assembly product to25ul of NEB turbo cells 
	• Transfer the cells to electroporation cuvette 
	• Electroporate and add 975ul SOC medium immediately 
	• Recover 1h at 37°C 
	• Spin down briefly and discard supernatant 
	• Plate on chloramphenicol plate (300µL of 0,4% Glucose added to the plate to reduce transcription of Lac promoter) 
Gibson mix was not diluted 1:3 in contrast to first attempt on 15-05-22 </P><H1>15/05/27</H1><H2>Plasmidisolation</H2><P>

Colonies are grown wich hopefully carry CDI plasmid. To check this, 12 colonies have been picked and cultivated in 3mL LB broth with additional 0.4% glucose. 

In the evening, 1.5ml of cell suspension were pelleted and Miniprep was carried out with the Qiagen Robot.</P><H1>15/05/28</H1><H2>Plasmidisolation, SacI <A href="https://2015.igem.org/Team:Marburg/Protocols/Protocol6">Digest</A></H2><P>

The concentration of all the 12 colonies was measured and noted as follows: <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>plasmid </TD><TD> concentration ng/uL </TD></TR><TR><TD class="col">#1</TD><TD class="col">77</TD></TR><TR><TD class="col">#2</TD><TD class="col">30 </TD></TR><TR><TD class="col">#3</TD><TD class="col">90</TD></TR><TR><TD class="col">#4</TD><TD class="col">88</TD></TR><TR><TD class="col">#5</TD><TD class="col">88</TD></TR><TR><TD class="col">#6</TD><TD class="col">86</TD></TR><TR><TD class="col">#7</TD><TD class="col">97</TD></TR><TR><TD class="col">#8</TD><TD class="col">74</TD></TR><TR><TD class="col">#9</TD><TD class="col">84</TD></TR><TR><TD class="col">#10</TD><TD class="col">88</TD></TR><TR><TD class="col">#11</TD><TD class="col">62</TD></TR><TR><TD class="col">#12</TD><TD class="col">81</TD></TR></TBODY></TABLE>

A mastermix for all plasmids was made:
<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> Digest </TD><TD> volume</TD></TR><TR><TD class="col">DNA template</TD><TD class="col">10 uL</TD></TR><TR><TD class="col">Cutsmart buffer</TD><TD class="col">2 uL</TD></TR><TR><TD class="col">Distilled water </TD><TD class="col">7 uL</TD></TR><TR><TD class="col">SacI</TD><TD class="col">1 uL</TD></TR><TR><TD class="col">total</TD><TD class="col">20 uL</TD></TR></TBODY></TABLE>
all the colonies were then incubated for 1hr at 37 °C on an thermoblock 

Analytic gel ; correct size abt 6700 and 8000 <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 4:</B> 
Ladder,#1, #2, #3, #4, #5, #6, #7, #8, #9, #10, #11, #12, Ladder  

</FIGCAPTION></FIGURE>
Clones 1 to 11 show the expected band pattern. We send out the abt 50ul of  the 1st and 3rd colony for sequence reading. 
The DNA sample was send to determine the sequence and it was positive. We actually got the sequence which we expected. </P><H1>15/05/31</H1><H2>Overnightcultures</H2><P>

Inoculation of 
	• four BBa_I13522 (GFP) clones in 3ml LB+CAM overnight 
	• Olga's W3110 strain with integrated RFP and Kann resistance in 3ml LB+Kann 
	• W3110 wildtype in 3 ml LB as host for our GFP construct </P><H1>15/06/1</H1><H2>Transformation into chem. comp. cells</H2><P>

Plasmid purification (four BBa_I13522 (GFP)) was realized using the &quot;Thermo Scientific GeneJET Plasmid Miniprep Kit&quot; 

Tansformation: 
			GFP_1 Plasmid into E.coli (strain W3110)
			pICD001-1 Plasmid into E.coli (strain W3110 with integrated RFP) 
			pICD001-3 Plasmid into E.coli (strain W3110 with integrated RFP)

Transformation protocol: 
	1. Dilute 1:100 overnight culture (Max 15-05-31) 
	2. Grow 10 mL day culture (2.5 h) at 37 °C 
	3. Take 1 mL, spin down full speed (1 min, 17.000 rpm), put on ice 
	4. Add to pellet 100 micro liter of TSS, resuspend on ice. 
	5. Add 1 micro liter of Plasmid (GFP_1 ; CDI_1; CDI_3) 
	6. Incubate 45 Min on ice 
	7. Put at 42 °C for 1 min. 
	8. Incubate 15 min on ice 
	9. Add 1 mL LB Medium with 0.4 % Glucose, resuspend, incubate 2 h incubator at 37 °C, 220 rpm 
	10. Spin down, resuspend in 50 micro liter residual media, plate on LB-Kann, incubate at 37 °C over night (step 10 done by Max) </P><H1>15/06/2</H1><H2>Sequencing results</H2><P>

Colonies on all three transformation plates! <UL><LI>GFP plasmid (BBa_I13522): Colonies beyond count </LI><LI>pICD001-1 Plasmid into E.coli (strain W3110 with integrated RFP): 23 colonies </LI><LI>pICD001-3 Plasmid into E.coli (strain W3110 with integrated RFP): 26 colonies </LI></UL>
	
Finally the sequencing of pICD001-1 and pICD001-3 is done! Correct Gibson assembly sequences were checked by sequenceing with standard primers iGEM130-fwd and iGEM160-rev and 100ng/ul pICD001 plasmids. Both plasmids are correct, so pICD001-1 will become pICD001 and pICD001-3 can be trashed. 
As a result, the W3110/RFP strain transformed with pICD001-1 will become strain CDI002 (a.k.a. the killer strain). The W3110 wildtype transformed with the GFP plasmid (BBa_I13522) will become strain CDI003 (a.k.a. the Opfer strain). 

Inoculation of CDI002 and CDI003 in 3 ml LB CAM +1%glucose </P><H1>15/06/3</H1><H2>IPTG dose response</H2><P>

Dose-response curve of CDI002 at different induction levels --&gt; if Cdi operon is expressed, we expect a higher metabolic burden at high induction compared to low induction. The used broth was LB with 1:1000 Chloramphenicol and 1 % glucose.
The concentrations of IPTG were gained by dilution series.

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 5:</B> 
IPTG dose response curve 

</FIGCAPTION></FIGURE></P><H1>15/06/5</H1><H2>IPTG dose response, killing assay</H2><P>
Repetition of Dose-response assay with higher IPTG concentrations. At first the overnight cultures of 4 strains (W3310, W3310-RFP, CDI 002, CDI 003) were diluted to an OD600 of 0,08 to a volume of 10mL LB-medium
<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 6:</B> 
IPTG dose response curve 
</FIGCAPTION></FIGURE>

Abnormalities/interpretation: 
	• Flask with CDI003 with 1,6µM IPTG fell over. So last measurement was after 190min.
	• CDI003 (GFP strain) grew fast at the beginning. After 3,5h the cells clumped together and the medium cleared up.  
	• The influence of different IPTG concentrations on the growth rate is very low but a slight trend is noticable. </P><P>
Preparing of daycultures for flow cytometry in different broths.
Broths: 
LB = LB-medium with 0.8 mM IPTG 
LB-CAM =  LB-medium with 0.8 mM IPTG and 1:1000 Chloramphenicol <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> culture (broth, Time [min]</TD><TD> 0 min</TD><TD> 60 min</TD><TD> 90 min</TD></TR><TR><TD class="col">LB, W3110 RFP</TD><TD class="col">0.1</TD><TD class="col">0.25</TD><TD class="col">0.4 </TD></TR><TR><TD class="col">LB, W3110</TD><TD class="col">0.1</TD><TD class="col">0.41</TD><TD class="col">0.7 </TD></TR><TR><TD class="col">LB-CAM, CDI002 </TD><TD class="col">0.1</TD><TD class="col">0.21</TD><TD class="col">0.46 </TD></TR><TR><TD class="col">LB-CAM, CDI003</TD><TD class="col">0.1</TD><TD class="col">0.32</TD><TD class="col">0.59 </TD></TR><TR><TD class="col">LB,CDI002</TD><TD class="col">0.1</TD><TD class="col">0.23</TD><TD class="col">0.45 </TD></TR><TR><TD class="col">LB,CDI003</TD><TD class="col">0.1</TD><TD class="col">0.31</TD><TD class="col">0.65 </TD></TR></TBODY></TABLE>
The different cultures were diluted to 0.4 [OD-600] and IPTG was added to a constant overall concentration. 

Different cultures were mixed (2 mL/ 2mL) to a total volume of 4 mL, incubated at 37 °C, 250 rpm for 3 hours. An Aliqout of 0.5 mL was mesured every 60 Minutes. <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> number</TD><TD> mixed cultures </TD><TD> ratio</TD><TD> broth</TD></TR><TR><TD class="col">1</TD><TD class="col">CDU002+CDI003</TD><TD class="col">1:10</TD><TD class="col">LB</TD></TR><TR><TD class="col">2</TD><TD class="col">CDU002+CDI003</TD><TD class="col">1:1</TD><TD class="col">LB</TD></TR><TR><TD class="col">3</TD><TD class="col">CDU002+CDI003</TD><TD class="col">10:1</TD><TD class="col">LB</TD></TR><TR><TD class="col">4</TD><TD class="col">CDU002+CDI003</TD><TD class="col">1:10</TD><TD class="col">LB-CAM</TD></TR><TR><TD class="col">5</TD><TD class="col">CDU002+CDI003</TD><TD class="col">1:1</TD><TD class="col">LB-CAM</TD></TR><TR><TD class="col">6</TD><TD class="col">CDU002+CDI003</TD><TD class="col">10:1</TD><TD class="col">LB-CAM</TD></TR><TR><TD class="col">7</TD><TD class="col">W3110 RFP+CDI003</TD><TD class="col">1:10</TD><TD class="col">LB</TD></TR><TR><TD class="col">8</TD><TD class="col">W3110 RFP+CDI003</TD><TD class="col">1:1</TD><TD class="col">LB</TD></TR><TR><TD class="col">9</TD><TD class="col">W3110 RFP+CDI003</TD><TD class="col">10:1</TD><TD class="col">LB</TD></TR><TR><TD class="col">10</TD><TD class="col">Control CDI002 only</TD><TD class="col">LB</TD></TR><TR><TD class="col">11</TD><TD class="col">Control CDI003 only</TD><TD class="col">LB</TD></TR><TR><TD class="col">12</TD><TD class="col">Control W3110</TD><TD class="col">LB</TD></TR><TR><TD class="col">13</TD><TD class="col">Control W3110 RFP</TD><TD class="col">LB</TD></TR><TR><TD class="col">14</TD><TD class="col">Control CDI002 only</TD><TD class="col">LB-CAM</TD></TR></TBODY></TABLE>
Observations: 
The victim strain CDI003 is overgrown by CDI002, but also does the W3110 RFP strain. 
The victim strain CDI003 starts shrinking after reaching of an OD-600 of 1.2.  
Aggregates in samples 1,7,11 and 8 
Optical whitening in samples 11 and 12 

FACS results (data not shown) were not as expected so we will repeat the experiment. 
Experiment will be repeated with changed parameters: lower density, lower flowrate, other broth, change of intensity, change of voltage, different FACS parameters. </P><H1>15/06/9</H1><H2>PCR, DpnI Digest, Overnightcultures</H2><P>
PCR of pCDI001 (full length CDI operon) with Primers iCD003 and iCD007. Result is linearised plasmid withour CdiA-CT and CdiI. 
PCR progam and Mix with GXL Polymerase are listed in the protocol section.

<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 7:</B> 
ladder, pcr product 1 

</FIGCAPTION></FIGURE>
A DnpI digest was done afterwards. 
Inoculation of 10 W3110-RFP clones in 3 ml LB over night to check with FACS the next day (last FACS measurement showed three populations in mCherry histogram. We wanted to check, if this is clone-specific or the same for different clones). 

Inoculation of W3110-WT, W3110-RFP, CDI002 and CDI003 in 3 ml LB and TB to compare overnight growth tomorrow.</P><H1>15/06/10</H1><H2>PCR Purfication, Bluntend cloning</H2><P>
Sample Purification 
The Sample DNA was purified with the purification kit of GeneJET PCR 
 to the 49ul volume of sample was added 49ul of binding puffer, mixed, centrifugated, 700ul wash Buffer added ,centrifugated for one minute twice to assure the  removal of  any residual of wash buffer.
50ul Elution Buffer was then added to the column containing the DNA sample and centrifuged. A  concentration of 65,51ug/ml was measured at the nanodrop.
  
Blunt-end cloning 
Blunt-end cloning of the purified PCR fragment was done as follows and at 37°C incubated for 30 min 
17ul sample DNA
2ul T4 Ligase Puffer 
1ul Kinase (T4-PNK)
20min inactivation of T4-PNK at 65°C

Made LB plates with CAM and 0.8 mM IPTG

<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>strain</TD><TD> LB media </TD><TD> TB media</TD></TR><TR><TD class="col">W3110-WT</TD><TD class="col">6.6</TD><TD class="col">3.1</TD></TR><TR><TD class="col">W3110-RFP</TD><TD class="col">5.8</TD><TD class="col">2.8</TD></TR><TR><TD class="col">CDI002</TD><TD class="col">4.1</TD><TD class="col">1.9</TD></TR><TR><TD class="col">CDI003</TD><TD class="col">4.1</TD><TD class="col">1.4</TD></TR></TBODY></TABLE>
-&gt; Strains grow roughly twice as dense in LB than in TB 
-&gt; Strains harboring a high copy plasmid (CDI002 and 003) grow only half to 2/3rd as dense as strains without plasmid

FACS result of yesterday's o.n. cultures: All show the high mCherry peak in histogram, not the intermediate one or the &quot;negative&quot; peak. The occurence of different mCherry populations seems to be cell cycle specific and not clone-specific.



</P><H1>15/06/11</H1><H2>swarming assay; Transformation</H2><P>

To get a better negative control than the W3110-RFP for our swarming plates and flow cytometry mesurements, we transformed blunt-end-cloned fragment (plasmid without CdiA-CT and CdiI (pICD002)) (made by Daniel 15-6-09) into W3110-RFP strain.
Following Protocol was used: 
Transformation of non-competent E.coli cells  
To see how much Plasmid we need to get colonies on LB+CAM plates we used different amounts of Plasmid for the Transformation Protocol. 
<UL><LI>1. 0.1 µL Plasmid pCDI001 without CdiA-CT and CdiI (pICD002)</LI><LI>2. 1 µL Plasmid pCDI001 without CdiA-CT and CdiI (pICD002) </LI><LI>3. 10 µL Plasmid pCDI001 without CdiA-CT and CdiI (pICD002) </LI></UL></P><H1>15/06/12</H1><H2>swarming assay; Transformation</H2><P>
On all LB+CAM plates colonies were observed. Most colonies resulted from the third Transformation with 10 µL Plasmid.
Six colonies were picked for overnight cultures to perform Miniprep the next day. 

Swarming assay 
A swarming assay was performed to check if growth inhibition can be proven. LB-Plates (CAM, 0.8mM IPTG) with 0,27% Agar were used to give the bacteria the possibility to swarm across the plate. 
After 3 days incubation at 30°C the plates were put on a blue light screen to visualise fluorescence.<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 8:</B> 
swarming assay
</FIGCAPTION></FIGURE>
The result is that the bacteria stopped to grow a short distance before getting in contact with each other (Green: no fluorescence: &quot;killer strain&quot; CDI002; Green: &quot;target strain&quot; CDI003). This might be due to quorum sensing or nutrient depletion. </P><H1>15/06/13</H1><H2>Plasmid isolation </H2>

Miniprep of the six pICD002 candidates according to Thermo Scientific protocol. 
Undigested pICD002 candidates ran on analytical gel

<H1>15/06/16</H1><H2>Overnight cultures, sequencing  </H2>
Overnight cultures of CDI002, CDI003, W3110, NEBTurbo with pICD001 
Plate CDI002, CDI003 and CDI004 each on a third of a LB-Plate 

Sent pICD002-1 and pICD002-6 for sequencing with reverse primer 160 that binds downstream of CdiI and the CdiA-CT that we want to kick out.

<H1>15/06/17</H1><H2>growth analysis, SDS-Page  </H2>

Diluting of all overnight cultures to an OD600 of 0,05 to start a day culture
OD600 with time: <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>strain</TD><TD> T 0 </TD><TD> 2 hours</TD><TD> 5 hours</TD></TR><TR><TD class="col">W3110</TD><TD class="col">0.05</TD><TD class="col">0.69</TD><TD class="col">3.1</TD></TR><TR><TD class="col">CDI002</TD><TD class="col">0.05</TD><TD class="col">0.10</TD><TD class="col">0.20</TD></TR><TR><TD class="col">CDI003</TD><TD class="col">0.05</TD><TD class="col">0.07</TD><TD class="col">0.17</TD></TR><TR><TD class="col">CDI004</TD><TD class="col">0.05</TD><TD class="col">0.14</TD><TD class="col">0.64</TD></TR></TBODY></TABLE>
After 5 hours the growth of CDI002, CDI003 and CDI004 was to weak to start competition assay so no co-culture assay was started.

To find out why day cultures did not grow properly we checked the overnight cultures under the microscope with no result. 
Day cultures are diluted 1:100 and plated on LB-Agar plates to have single colonies on the plates. 

SDS-PAGE was started to see if CdiA, CdiB and CdiI are expressed. All steps are done with CDI002 and W3110 as control.
Sepperating gel: <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>Chemicals</TD><TD> volume</TD><TD> volume</TD></TR><TR><TD class="col">Acrylamide percentage</TD><TD class="col">6</TD><TD class="col">15</TD></TR><TR><TD class="col">water</TD><TD class="col">5.2 mL</TD><TD class="col">2.2 mL</TD></TR><TR><TD class="col">Acrylamide_Bis-acrylamide (30%;0.8%)</TD><TD class="col">2 mL</TD><TD class="col">5 mL</TD></TR><TR><TD class="col">1.5M Tris (pH=8.8)</TD><TD class="col">2.6 mL</TD><TD class="col">2.6 mL</TD></TR><TR><TD class="col">10% SDS</TD><TD class="col">0.1 mL</TD><TD class="col">0.1 mL</TD></TR><TR><TD class="col">10% ammoniumpersulfat</TD><TD class="col">0.1 mL</TD><TD class="col">0.1 mL</TD></TR><TR><TD class="col">TEMED</TD><TD class="col">0.01 mL</TD><TD class="col">0.01 mL</TD></TR></TBODY></TABLE>

Stacking gel:
<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>Chemicals</TD><TD> volume</TD></TR><TR><TD class="col">water</TD><TD class="col">2.975 mL</TD></TR><TR><TD class="col">0.5 M Tris-HCl, pH6.8</TD><TD class="col">1.25 mL</TD></TR><TR><TD class="col">10% SDS </TD><TD class="col">0.05 mL</TD></TR><TR><TD class="col">Acrylamide/Bis-acrylamide (30%; 0.8%)</TD><TD class="col">0.67 mL</TD></TR><TR><TD class="col">10% Ammoniumpersulfat</TD><TD class="col">0.05 mL</TD></TR><TR><TD class="col">TEMED</TD><TD class="col">0.005 mL</TD></TR></TBODY></TABLE>
Neither CdiA nor CdiI nor CdiB could be observed on the SDS page (data not shown).  

1mL of cells are centrifuged and supernatant is discarded. 

Cells were prepared in different ways: <UL><LI>1. Pellet is resuspended in 50µL of sample buffer. 
	Samples are mixed by pipetting up and down for 2 minutes to break CDI A if present on the surface of the cell, suspension is centrifugated and supernatant is used for gel </LI><LI>2. Cells are used whole. 
	65µL of sample buffer is used to resuspend the pellet, cells are incubated for 10 minutes at 96°C, centrifugation for 3 minutes at max speed. </LI><LI>3. Cells 50µL of lysis buffer is used to resuspend the pellet, 2µL of protease inhibitor is added, incubation at room temperature for 10 minutes to lyse the cells, centrifugation for 10 minutes at max speed and 4°C 
		</LI><LI>a. Supernatant is used with 16µL SDS-buffer </LI><LI>b. Pellet is used with 65µL sample buffer </LI></UL>

We found out that the W3110-RFP strain that we were supplied with still harbors a plasmid that contains GFP and kan-resistance! We have to make strain CDI002 lose that plasmid before we do any other assay: Plate 6 clones on fresh CAM-plate every day and transfer that culture onto kann-plate as negative control. Only if there is no more growth on the kan-plate we can continue! All FACS data acquired so far is useless, because with CDI002 being positive for GFP and mCherry we cannot make a proper analysis. 

Sequencing results of pICD002-1 and pICD002-6: Nice sequence up to the blunt-end cloning site! Then multiple sequences seem to overlap. Cloning needs to be repeated!

<H1>15/06/18</H1><H2>growth analysis, SDS-Page  </H2>

Repeated the PCR for pICD001 backbone without CdiA-CT and CdiI (see 15-06-09) and ran analytical gel: <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 9:</B> 
PCR analytical gel run
</FIGCAPTION></FIGURE>

DpnI digest of the PCR mix to get rid of methylated pICD001 template DNA 
(This time DpnI was heat inactivated, just to be sure that it does not interfere with the following steps) 

PCR cleanup using Thermo Scientific Cleanup kit. We eluted in 30 ul pre-warmed elution buffer (recommended for fragments &gt;10 kb). 

Nanodrop: 99ng/ul DNA  <H1>15/06/19</H1><H2>blunt end cloning; ligation  </H2>
5'-phosphorylation of PCR fragment with polynucleotide kinase (for subsequent blunt-end cloning).
PNK mix:
<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>Chemicals</TD><TD> volume</TD></TR><TR><TD class="col">DNA template</TD><TD class="col">17 uL (0.184 pmol)</TD></TR><TR><TD class="col">T4 DNA ligase buffer</TD><TD class="col">2 uL</TD></TR><TR><TD class="col">PNK</TD><TD class="col">1 uL</TD></TR></TBODY></TABLE>
Program: 
<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>temperature</TD><TD>time</TD></TR><TR><TD class="col">37 °C</TD><TD class="col">30 min</TD></TR><TR><TD class="col">65 °C</TD><TD class="col">20 min</TD></TR></TBODY></TABLE>
Use 0.02 pmol DNA for ligation --&gt; 2.56 ul of PNK mix
Ligation mix:
<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>chemicals</TD><TD>volume</TD></TR><TR><TD class="col">ddH20</TD><TD class="col">14.44 uL</TD></TR><TR><TD class="col">PNK mix</TD><TD class="col">2.56 uL</TD></TR><TR><TD class="col">T4 DNA ligase buffer</TD><TD class="col">2 uL</TD></TR><TR><TD class="col">T4 DNA ligase</TD><TD class="col">1 uL</TD></TR></TBODY></TABLE>
Program: 
<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>temperature</TD><TD>time</TD></TR><TR><TD class="col">RT</TD><TD class="col">2 h</TD></TR><TR><TD class="col">65 °C</TD><TD class="col">10 min</TD></TR></TBODY></TABLE><H1>15/06/24</H1><H2>plasmidisolation; Digest  </H2>
Miniprep of the six pICD002 candidates according to Thermo Scientific protocol. 
Elution in 30µL elution buffer 

We wanted to check if all pICD002 candidates got the right plasmid formation. So we did an  Restriction assay with the six pICD002 candidates: <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>Restriction assay</TD><TD>pICD002_1</TD><TD>pICD002_2</TD><TD>pICD002_3</TD><TD>pICD002_4</TD><TD>pICD002_5</TD><TD>pICD002_6</TD></TR><TR><TD class="col">DNA</TD><TD class="col">1.35 mL</TD><TD class="col">1.63 mL</TD><TD class="col">3.33 mL</TD><TD class="col">2.77 mL</TD><TD class="col">2.13 mL</TD><TD class="col">1.36 mL</TD></TR><TR><TD class="col">NEB: cutsmart buffer</TD><TD class="col">2 mL</TD><TD class="col">2 mL</TD><TD class="col">2 mL</TD><TD class="col">2 mL</TD><TD class="col">2 mL</TD><TD class="col">2 mL</TD></TR><TR><TD class="col">BSTZ17I (restriction enzym)</TD><TD class="col">0.5 mL</TD><TD class="col">0.5 mL</TD><TD class="col">0.5 mL</TD><TD class="col">0.5 mL</TD><TD class="col">0.5 mL</TD><TD class="col">0.5 mL</TD></TR><TR><TD class="col">EcoRI (restriction enzym)</TD><TD class="col">0.5 mL</TD><TD class="col">0.5 mL</TD><TD class="col">0.5 mL</TD><TD class="col">0.5 mL</TD><TD class="col">0.5 mL</TD><TD class="col">0.5 mL</TD></TR><TR><TD class="col">water</TD><TD class="col">15.65 mL</TD><TD class="col">15.37 mL</TD><TD class="col">13.67 mL</TD><TD class="col">14.23 mL</TD><TD class="col">14.87 mL</TD><TD class="col">15.64 mL</TD></TR></TBODY></TABLE><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 9:</B> 
Restriction assay
</FIGCAPTION></FIGURE>
Candidates 1 and 6 show the expected band pattern (although we forgot to load a marker). Those DNA samples were send to determine the sequence using following mix:
Volume: 15 µL 
100 ng DNA/µL 
2 µL pICD002_X  (1 and 6)  
2µL Primer  
11 µL Water <H1>15/06/25</H1><H2>growth experiment  </H2>

Due to our results this far we decided to go mesure a new growth curve (OD600 ) with CDI001 (W3110 + pCDI001) and W3110:

used browth: LB + 0.2 % Glucose and Chloramphenicol for CDI001 to hold the selection pressure <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD>time/strain</TD><TD>W3110</TD><TD>CDI001</TD></TR><TR><TD class="col">0 min</TD><TD class="col">0.1 </TD><TD class="col">0.1</TD></TR><TR><TD class="col">40 min</TD><TD class="col">0.2 </TD><TD class="col">0.12</TD></TR><TR><TD class="col">90 min</TD><TD class="col">0.71 </TD><TD class="col">0.18</TD></TR><TR><TD class="col">120 min</TD><TD class="col">1.13 </TD><TD class="col">0.24</TD></TR><TR><TD class="col">180 min</TD><TD class="col">1.0 </TD><TD class="col">0.43</TD></TR><TR><TD class="col">230 min</TD><TD class="col">1.1 </TD><TD class="col">0.72</TD></TR><TR><TD class="col">270 min</TD><TD class="col">1.0 </TD><TD class="col">1.0</TD></TR><TR><TD class="col">320 min</TD><TD class="col">1.2 </TD><TD class="col">0.6</TD></TR><TR><TD class="col">360 min</TD><TD class="col">1.2 </TD><TD class="col">0.43</TD></TR><TR><TD class="col">420 min</TD><TD class="col">1.2 </TD><TD class="col">0.43</TD></TR><TR><TD class="col">480 min</TD><TD class="col">1.3 </TD><TD class="col">0.44</TD></TR></TBODY></TABLE>

After 120 minutes we add IPTG to a total concentration of 0.5 mM (flask with W3110) 
After 270 minutes we add IPTG to a total concentration of 0.5 mM (flask with CDI001) 

After an OD-600 of 1.0 was observed we aliqout the equivalent of 1 mL of cells at OD600  = 1.0 in a 1.5 mL microfuge tube (t=0) 
Next step was induction with IPTG (0.5 mM) and continued shaking at 250 rpm for serveral hours 

Aliquots were extracted every hour. 
SDS Gel data here not shown.Growth of strains harboring plasmids is still very poor. We came to the conclusion that maybe not the plasmids (and the metabolic burden that is connected with them) are the problem, but the medium. Maybe the wrong antibiotic was added to LB. --&gt; Prepared fresh medium for growth curves next week. <H1>15/06/29</H1><H2>low copy plasmids </H2>

Sequencing result of blunt-end ligation: pICD002-6 has 3 deletions that destroy the stop codon that was supposed to be introduced, so this cannot be used. PICD002-1 also has a two-base pair deletion at the site of ligation but that does not destroy the stop codon, nor does it destroy the SpeI site of the iGEM suffix sequence. Not an ideal result but something that is good to work with! --&gt; pICD002-1 becomes pICD002. 

To reduce metabolic burden for the cells, we plan to switch to low copy plasmids, namely pSB3C5 and pSB4C5. They were both available in the Spring 2015 distribution with the same insert (BBa_J04450, a lac-inducible RFP gene). --&gt; both were transformed with DH5alpha (see: Transformation into chemocompetent cells). <H1>15/06/29</H1><H2>plans </H2>
Planned growth curve experiment was aborted due to growth problems of cultures with plasmid.
<H1>15/07/01</H1><H2>platereader </H2>

To check if the growth problems are depending on the antibiotic concentration or if the growth is different if the bacteria are taken from plate or from an overnight culture a growth curve was performed in a 24 well plate.
1,5mL of Medium (either with or without chloramphenicole (see table)) is inocculatet with a colonie from plate or with 10µL overnight culture (OD around 7,00) 
The results are shown in the following growthcurve:
<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 10:</B> 
Platereader experiment
</FIGCAPTION></FIGURE><H1>15/07/02</H1><H2>sequencing </H2>
Sequencing results:
PSB3C5-1 and -2: Both correct! --&gt; pSB3C5-1 will be used from now on 
PSB4C5-1: Forward primer sequencing failed, reverse seqencing looks correct except for point mutation in lac promoter BBa_R0010€€ 
PSB4C5-2: All correct except for same point mutation as in clone 1: A to G in position 117 of BBa_R0010. Since both clones have the mutation we assume that it was already in the registry plate and will use pSB4C5-2 for further experiments. <H1>15/07/03</H1><H2>clean streak</H2>

Clean streak of pSB3C5 and pSB4C5: 
Under the magnifier we saw colonies that were half red and half white. Due to lacI on a bacterial artificial chromosome all colonies were expected to be white. 

<H1>15/07/06</H1><H2>cloning</H2>
We want to improve our pICD001.To accomplish this goal we decided to put 6 nucleotides between the ribosome binding site and the start of the CDI_A.

For the PCR conditions and DpnI digest see protocols. 
PCR product purification was accomplished with  QiAquickPCR Purification Kit (50) 
To create a new plasmid we did a Gibson assembly with the linear PCR product: 
used DNA 100 ng   --&gt; 3 µL 
add 10 µL of Gibson Assembly Mastermix (2x) 
add dd Water to a total volume of 20 µL 
incubate 37 °C for 20 min 
The product was transformed into electrocompetent cells (see protocols). <H1>15/07/07</H1><H2>growth problems, cloning and platereader</H2>
Transformation was succesful. To check if the additional 6 bp were added between promoter and ribosome binding site, pICD007 was prepped with Thermo Scientific miniprep kit and eluted in 30µL. 

One more idea concerning our growth problems is that the chloramphenicole molecules are degraded during the first hours of an overnight culture and the plasmids are lost at the end of the overnight culture. So finally there could be a lot of living cells but only a few carry the plasmid. If this culture is used to inocculate the day culture with new chloramphenicole most of the cells die immediately. 

To check this theory, we started a new platereader experiment using different concentrations of chloramphenicole. The next day we plan to plate these overnight cultures each to LB and to LB with chloramphenicole to check if the cells are dead after overnight or if they just lost the plasmid 

We want to change our CDI operon from a high copy plasmid into a mid and low copy plasmid (5 and 15 copys per cell) therefore we used PCR to amplify the CDI operon, the plasmid backbone from pSB3C5 and pSB4C5 (5 and 15 copys per cell) with 3850 basepairs and 4300 basepairs. Also we amplified lacI with 1500 basepairs. In another experiment we wanted to create an polycistronic plasmid with mCherry therefore we amplifyed most of the backbone from pSB3C5 (3800bp), pSB4C5(4300bp) and again the CDI operon without RBS. 
We used the following primers and templates:
<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> fragments</TD><TD>fragment 1</TD><TD>fragment 2</TD><TD>fragment 3</TD><TD>fragment 4</TD><TD>fragment 5</TD><TD>fragment 6</TD><TD>fragment 7</TD></TR><TR><TD class="col">template (1:100)</TD><TD class="col">pSB3C5-J04450</TD><TD class="col">pSB4C5-J04450</TD><TD class="col">pICD001</TD><TD class="col">pICD001</TD><TD class="col">pSB4C5-J04450</TD><TD class="col">pSB4C5-J04450</TD><TD class="col">pICD001</TD></TR><TR><TD class="col">forward primer (10 uM)</TD><TD class="col">ICD0034</TD><TD class="col">ICD0034</TD><TD class="col">ICD001</TD><TD class="col">lacI, AaII rev</TD><TD class="col">ICD0036</TD><TD class="col">ICD0036</TD><TD class="col">ICD0038</TD></TR><TR><TD class="col">reverse primer (10 uM)</TD><TD class="col">ICD0035</TD><TD class="col">ICD0035</TD><TD class="col">ICD008</TD><TD class="col">ICD0033</TD><TD class="col">ICD0037</TD><TD class="col">ICD0037</TD><TD class="col">ICD0039</TD></TR><TR><TD class="col">desired fragment length</TD><TD class="col">3850 bp</TD><TD class="col">4300 bp</TD><TD class="col">11500 bp</TD><TD class="col">1500 bp</TD><TD class="col">3800 bp</TD><TD class="col">4300 bp</TD><TD class="col">11500 bp</TD></TR></TBODY></TABLE>

Protocol Standard PCR and DpnI digest. 

<H1>15/07/08</H1><H2>gibson assembly, plating of platereader wells</H2>

The cultures of wells A2, A3, A4, D2, D3, D4 are diluted 1:100 twice.
20µL are plated each on LB and LB+CAM Agar plates 

For the gibson assembly to creat our new plasmids we need to purificate our PCR products.
Purification was accomplished with QiAquickPCR Purification Kit (50) 

Transformation 
Plating of cultures from plate reader: 
The cultures of wells A2, A3, A4, D2, D3, D4 are diluted 1:100 twice (--&gt; 1:10000). 
20µL are plated each on LB and LB+CAM Agar plates. This way we are trying to find out how many cells are actually alive in our overnight culture and how many of those still have our plasmid. Only if we have roughly the same number of colonies on the LB+CAM plate as on the LB-only plate, we can be sure that all cells still harbor the plasmid. 

G.A. reaction:

<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> fragments</TD><TD>volume</TD></TR><TR><TD class="col">Fragment 1</TD><TD class="col">0.76 uL (=36.7 ng = 0.14 pmol)</TD></TR><TR><TD class="col">Fragment 3</TD><TD class="col">1.88 uL (=100 ng = 0.14 pmol)</TD></TR><TR><TD class="col">Fragment 4 </TD><TD class="col">0.32 uL (=13.8 ng = 0.14 pmol)</TD></TR><TR><TD class="col">G.A. master mix</TD><TD class="col">10 uL</TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">7.04 uL</TD></TR></TBODY></TABLE><TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> fragments</TD><TD>volume</TD></TR><TR><TD class="col">Fragment 2</TD><TD class="col">0.68 uL ( 40.9 ng = 0.14 pmol)</TD></TR><TR><TD class="col">Fragment 3</TD><TD class="col">1.88 uL (=100 ng = 0.14 pmol)</TD></TR><TR><TD class="col">Fragment 4 </TD><TD class="col">0.32 uL (=13.8 ng = 0.14 pmol)</TD></TR><TR><TD class="col">G.A. master mix</TD><TD class="col">10 uL</TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">7.12 uL</TD></TR></TBODY></TABLE><TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> fragments</TD><TD>volume</TD></TR><TR><TD class="col">Fragment 5</TD><TD class="col">0.60 uL (=36.7 ng = 0.14 pmol)</TD></TR><TR><TD class="col">Fragment 7</TD><TD class="col">2.22 uL (=100 ng = 0.14 pmol)</TD></TR><TR><TD class="col">G.A. master mix</TD><TD class="col">10 uL</TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">6.84 uL</TD></TR></TBODY></TABLE><TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> fragments</TD><TD>volume</TD></TR><TR><TD class="col">Fragment 6</TD><TD class="col">0.99 uL (=40.9 ng = 0.14 pmol)</TD></TR><TR><TD class="col">Fragment 7</TD><TD class="col">2.22 uL (=100 ng = 0.14 pmol)</TD></TR><TR><TD class="col">G.A. master mix</TD><TD class="col">10 uL</TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">6.84 uL</TD></TR></TBODY></TABLE>

used Protocol Gibson assembly.


<H1>15/07/09</H1><H2>results from platereader</H2>

Counting of the colonies from the overnight plate reader experiment: 
Numbers are: Colonies per CAM plate / colonies per LB plate 
(The CAM-plates had the SAME antibiotics concentration (34 ug/ml) ONLY the media in the plate reader experiment had varying concentrations.) 

Outcome: 
	• CDI002 (harbors high copy plasmid, ca. 200 copies per cell) needs high CAM concentration to keep up the selection pressure. Otherwise cells lose the plasmid. 
	• DH5alpha with pSB3C5 (10 - 12 copies per cell) counterintuitively seems to lose the plasmid at high chloramphenicol concentrations, whereas the cells seem to keep it at low concentrations 
	• Same applies to DH5alpha harboring pSB4C5 (ca. 5 copies per cell). This however grows much worse. <H1>15/07/14</H1><H2>SDS-Page</H2>

Preparing of SDS-Page samples for pCDI007:
We wanted to see the expression of the CDI operon with the new spacer sequence between RBS and startcodon.
Result of the SDS page (data not shown): no signals form any Cdi operon protein can be found. Maybe we need to change the strain or use a lowcopy plasmid in oder to decrease the metabolic burden. 
Samples of the overnight culture were plated out on LB Agar and LB Agar + Chloramphenicol plates. 
To see if our E.coli cells lose our Plasmid. On both plates we counted equal colonies. 

Test Digest of pCDI003_1, pCDI003_2, pCDI003_3, pCDI004_1, pCDI004_2, pCDI004_3, pCDI005_1, pCDI005_2, pCDI005_3, pCDI006_1, pCDI006_2,pCDI006_3. 
Result of this test digest (picture not shown) all plasmids show signals in not exspected areas. To gain the new constructs we will do the Gibson assemblys again.  
Gibson assembly: Protocoll 15-07-08.

<H1>15/07/15</H1><H2>SDS-Page</H2>

The transformants (pICD003=reaction 1, pICD004=reaction 2, pICD005=reaction 3, pICD006=reaction4) of the Gibson assembly are streaked out on Agar-Plates and from the same colonies some cells are used to inocculate overnight cultures for Miniprep. 

We wanted to see if our SDS-Pages from pCDI007, pCDI002  and the wildtyp differ from each other. We obtain no difference between the aliquots of the different constructs . So far we only can see expression of the chloramphenicol resistance protein. <H1>15/07/16</H1><H2>Plasmidisolation </H2>
Miniprep of pICD003, pICD004, pICD005 and pICD006 according to Thermo Scientific protocol. 
Elution in 30µL elution buffer 

Test digest: To see with which plasmid we want to work form here on we used a test digest to get further information about our gained plasmids. 
We used the restriction enzyms EcoRI and fragments with 6 kBp and 8 kBps were expected. 
So far fragments were obtained as exspected in a next step we will sequence our new constructs. 
So far fragments were obtained as exspected in a next step we will sequence our new constructs.
Overnight cultres for pCDI007 in W3110-RFP and W3110-RFP were inoculated.

<H1>15/07/17</H1><H2>Plasmidisolation </H2>
We wanted to prepare another SDS Page with pCDI007 in W3110-RFP and to compare this result we also prepared W3110-RFP as a control. 

The experiment could not be finished because grow problems ocured. 
Another Test Digest of the samples from pCDI005 and pCDI006 with pICD001 as a control was prepared.
So far fragments were obtained as exspected in a next step we will sequence our new constructs.

<H1>15/07/20</H1><H2>sequencing</H2>
The sequenced constructs were not as exspected. The experimental design have to change to get better cloning results.      
<H1>15/07/21</H1><H2>PCR, overnight culutres </H2>
PCR of 15-07-07 was repeated because the Gibson Assembly did not work. 

Overnight cultures of CDI003, CDI007, W3110 and W3110-RFP are grown to start a killing assay and an SDS PAGE the next day

<H1>15/07/22</H1><H2>SDS-Page, killing assay</H2>

Day cultures of CDI003, CDI007, W3110 and W3110-RFP are diluted to an OD600 of 0,1 and induced with 0,5mM IPTG and additional 1% glucose. 

Samples of W3110-RFP and CDI007 for SDS PAGE are taken at different times. The amount of taken samples is calculated to 1mL of OD 1,0 

For W3110-RFP and W3110 new day cultures are grown to have them at the same OD as the CDI003, CDI007.
At an OD of about 0,4 the killing assay was started by mixing CDI003 with CDI007 and CDI003 with W3110.
The result at the FACS is that most of the CDI003 cells lost GFP and some cells of CDI007 lost the RFP. So the assay was aborted. 

The PCR products of 15-07-21 were put to preperative gel electrophoresis. 
The gel extraction was performed with Qiagen gel extraction kit. 

Only for fragments 1,2 and 4 sufficient amount of DAN was achieved. 

PCR of fragments 3,5,6 and 7 is repeated <H1>15/07/23</H1><H2>Gel extraction, PCR</H2>

The PCR products of 15-07-22 were put to preperative gel electrophoresis.
Because there are very weak bands for fragments 3 and 7, the PCR is repeated with the original template and DNA from an earlyer PCR for those fragments. 

Gel extraction is performed according to the modified gel extraction protocol.
Sufficient DNA was achieved for fragments 5 and 6. <H1>15/07/24</H1><H2>plasmidisolation</H2>
New plasmid preparation of pICD001 due to strange digestion results on gel. 

PCR of fragments 3 and 7 of old and new plasmid DNA <H1>15/07/25</H1><H2>plasmidisolation, CPEC</H2>

The PCR fragments of 15-07-24 are used for gel extraction. 
Gel extraction is performed according to the modified gel extraction protocol. 
Sufficient DNA was achieved for fragment 3, for fragment 7 the amount was not completely sufficient but could be used for CPEC anyway.
CPEC:

<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> fragments/volume</TD><TD>fragment 3 (2.78 uL)</TD><TD>fragment 1 (1.29 uL) </TD><TD>fragment 4 (1.50 uL) </TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">4.39 uL</TD></TR><TR><TD class="col">mastermix</TD><TD class="col">40 uL</TD></TR><TR><TD class="col">program</TD><TD class="col">denaturation 98 °C 10 s</TD><TD class="col">annealing 55 °C 30 s</TD><TD class="col">elongation 68 °C 168 s</TD></TR></TBODY></TABLE><TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> fragments/volume</TD><TD>fragment 3 (2.78 uL)</TD><TD>fragment 2 (1.16 uL) </TD><TD>fragment 4 (1.50 uL) </TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">4.68 uL</TD></TR><TR><TD class="col">mastermix</TD><TD class="col">40 uL</TD></TR><TR><TD class="col">program</TD><TD class="col">denaturation 98 °C 10 s</TD><TD class="col">annealing 55 °C 30 s</TD><TD class="col">elongation 68 °C 173 s</TD></TR></TBODY></TABLE><TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> fragments/volume</TD><TD>fragment 7 (4.39 uL)</TD><TD>fragment 5 (1.8 uL/1:100) </TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">3.814 uL</TD></TR><TR><TD class="col">mastermix</TD><TD class="col">40 uL</TD></TR><TR><TD class="col">program</TD><TD class="col">denaturation 98 °C 10 s</TD><TD class="col">annealing 55 °C 30 s</TD><TD class="col">elongation 68 °C 153 s</TD></TR></TBODY></TABLE><TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> fragments/volume</TD><TD>fragment 7 (2.78 uL)</TD><TD>fragment 6 (2.2 uL 1:100) </TD></TR><TR><TD class="col">ddH2O</TD><TD class="col">4.39 uL</TD></TR><TR><TD class="col">mastermix</TD><TD class="col">40 uL</TD></TR><TR><TD class="col">program</TD><TD class="col">denaturation 98 °C 10 s</TD><TD class="col">annealing 55 °C 30 s</TD><TD class="col">elongation 68 °C 158 s</TD></TR></TBODY></TABLE><H1>15/07/26</H1><H2>CPEC, analytical gel</H2>


CPEC products are run on analytical gel
<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 10:</B> 
analytical gel: From left to right reactions 1,2,3,4
</FIGCAPTION></FIGURE>
Bands are visible at about 20kb. Are a little bit higher than expected but that could be due to the cirulized form of the plasmid. 

Chemocompetent cells of W3110 are prepared according to protocol &quot;Competent E. coli cells (RbCl)&quot; 
CPEC product 1 and 2 and additonally pILS4, pILS5 and pILS6 are transformed to this strain. CPEC product 3 and 4 are tranformed to the hicontrol g10 E. coli strain. 

Transformation of CPEC products
Volume cells	200µL (100µL of hicontrol g10)   
Volume DNA	5µL CPEC products / 1µL pILS4-6  
30 min 	On ice 
45s 	42°C 
2min 	On ice 
Volume LB	800µL (900µL for hicontrol g10) 
1h	37°C in shaker 
Centrifugation	4min 4000rpm 
Discard supernatant	Use remaining supernatant for resuspendation 
Plate whole volume 	<H1>15/07/27</H1><H2>transformation</H2>

Transformation did not work for all four constructs. 
For pILS4, pILS5 and pILS6 many colonys were on the plates.  
6 clones each are used to be streaked out on agar plates.

One clone from each construct was grown in an overnight culture to study to growing behaviour compared to the old GFP strain CDI003. 

Chemical transformation of the CPEC reactions 1-3 were repeated with chemocompetent top 10 E. coli cells. 

In case a new CPEC will have to be started, the PCR for fragment 7 is repeated to get sufficient amount for a new reaction. <H1>15/07/28</H1><H2>transformation, growth experiment </H2>
In case the CPEC did not work instead of the transformation the CPEC reaction of 15-07-25 was repeated.  

The cultures of the three GFP strains pILS4,5 and 6 are diluted to OD600 of 0,1 and grown over day. 
pILS5 (medium promoter) and pILS6 (weakest promoter) grow approximately as W3110. pILS4 (strongest promoter) grows extremely slow comparably to CDI003. <H1>15/07/29</H1><H2>plasmidisolation,digest </H2>
Electroporation is done for CPEC reaction 1-3 in NEB TURBO E. coli cells 
 
Miniprep of the 8 pICD006 candidates accrording to thermo scientific miniprep kit. Elution in 30µL Elution Buffer. 

Digestion with EcoR1 of the 8 pICD006 candidates and the old and new pICD001 <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> concentration/volume</TD><TD> chemicals </TD></TR><TR><TD class="col">1000 ng</TD><TD class="col">DNA</TD></TR><TR><TD class="col">2 uL</TD><TD class="col">Cut smart buffer</TD></TR><TR><TD class="col">upt to 19 uL total</TD><TD class="col">ddH2O</TD></TR><TR><TD class="col">1 uL</TD><TD class="col">EcoRI</TD></TR></TBODY></TABLE>
Whole preparation is loaded to a gel <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 10:</B> 
analytical gel: From left to right: pICD006-1 - pICD008, old pICD001, new pICD001
</FIGCAPTION></FIGURE>
The bands for the pICD006 candidates do not look like they are expected so all samples are discarded.  

The old pICD001 looks completely strange but the newly prepared pICD001 looks as expected to this will be used for further experiments. <H1>15/07/30</H1><H2>transformation,digest </H2>
Transformation worked for all clones but white and red colonies are visible.
Miniprep of the transformant of CPEC reactions 1-3 (pICD003, pICD004, pICD005) according to thermo scientific protocol. Elution in 30µL. 

Digestion with EcoR1 (protocol see 15-07-29) and ran on a gel. <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 10:</B> 
analytical gel: From left to right: pICD003-1 - pICD003-6, pICD004-1 -pICD004-6, pICD005-1 - pICD005-2

</FIGCAPTION></FIGURE><H1>15/07/31</H1><H2>sequencing,transformation,digest </H2>
pICD003_1, pICD003_2, pICD004_2, pICD004_3, pICD005_1 and pICD005_4 were send to sequencing. 

Overnight cultures of pICD006 six different colonies (part of the colonies were plated out on Chloramphenicol Agar plates). 
Electroporation is done for CPEC reaction 4 in NEB TURBO E. coli cells. <H1>15/08/04</H1><H2>plasmid isolation, test digest </H2>
Miniprep of the transformant of pICD006 according to thermo scientific protocol. Elution in 30µL

Growth was obtained on the Chloramphenicol-Agar plates. 

Test Digest from pICD006 failed caused by running problems from each aliquot. 

<H1>15/08/05</H1><H2>sequencing </H2>
Sequencing results of CPEC constructs: 
pICD003-1 and pICD004-2 are correct! Sequencing of pICD005 and 6 clones failed.

<H1>15/08/10</H1><H2>plasmidisolation, analytical gel   </H2>

Miniprep of pICD003-1,pICD004-2, pICD005-7 - pICD005-14 and pICD006-2 - pICD006-9 according to Macherey Nagel Miniprep Kit. Elution in 30µL. 

Digestion with EcoR1 of the prepped plasmids (see protocols) <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 11:</B> 
analytical gel: from left to right: pICD003-1, pICD004-2, pICD005-7 - pICD005-14, pICD006-2 - pICD006-8
</FIGCAPTION></FIGURE>
pICD003-1 and pICD004-2 are as expected but the bands for all other samples do not look as expected. <H1>15/08/11</H1><H2>digest, analytical gel, growth experiments   </H2>
To test if the plasmids are all wrong the digest is repeated with SpH1 (see protocols). <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 11:</B> 
analytical gel: from left to right: pICD001,  pICD005-7 - pICD005-14, pICD006-2 - pICD006-8
</FIGCAPTION></FIGURE>

Again no fragments at the desired size so all samples are discarded. 

Growth curve of pICD003_1 and pICD004_2 in NEB turbo: 
We wanted to check which conditions are needed to express the CDI operon and how growth of NEB turbo is affected by the CDI operon plasmids in different copy numbers (10-12 copys per cell pICD004 and 5-8 copys per cell pICD003). Expression shall be seen during RFP wich is cloned directly behind the CDI operon. 

The overnightcultres had an OD-600 of 4.1 for pICD003_1 and 5.2 for pICD004_2 
We diluted them to an OD-600 of 0.05 (10 mL) and recorded a growth curve: <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> strain</TD><TD> 0 min</TD><TD> 20 min</TD><TD> 50 min</TD><TD> 87 min</TD><TD> 117 min</TD><TD> 150 min</TD><TD> 180 min</TD><TD> 215 min</TD><TD> 275 min</TD><TD> 335 min</TD></TR><TR><TD class="col">pICD003_1</TD><TD class="col">0.05</TD><TD class="col">0.1</TD><TD class="col">0.18</TD><TD class="col">0.27</TD><TD class="col">0.43</TD><TD class="col">0.74</TD><TD class="col">1.3</TD><TD class="col">1.9</TD><TD class="col">3.1</TD><TD class="col">4.1</TD></TR><TR><TD class="col">pICD004_2</TD><TD class="col">0.05</TD><TD class="col">0.06</TD><TD class="col">0.1</TD><TD class="col">0.13</TD><TD class="col">0.2</TD><TD class="col">0.36</TD><TD class="col">0.7</TD><TD class="col">0.9</TD><TD class="col">2.0</TD><TD class="col">3.2</TD></TR></TBODY></TABLE>

We induced pICD003_1 after 87 minutes and pICD004_2 after 117 minutes with 0.5 mM IPTG.
After one day the aliqouts of pICD003_1 and pICD004 were slightly red. As a result we concluded that NEB turbo dont express proteins properly during log phase. Further information shall be gathered with FACS. So far we can not be sure if only RFP is expressed or RFP and the CDi operon. 
Cryo stocks of this two strains (piCD003_1 in NEB turbo [SiGEM 34]and pICD004_2 in NEB turbo [SiGEM 35]) were created (see protocols).    

<H1>15/08/12</H1><H2>FACS mesurement</H2>
Growth curve with NEB Turbo, pICD003, pICD004 (both in NEB Turbo) to take samples for SDS PAGE
Cultures are induced with 0,5mM IPTG <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> 9:20 </TD><TD> 10:20 </TD><TD> 11:40 </TD><TD> 12:20 </TD><TD> 13:10 </TD><TD> 15:30 </TD><TD> 18:00</TD></TR><TR><TD class="col">NEB_Turbo</TD><TD class="col">0.05</TD><TD class="col">0.18</TD><TD class="col">0.59 (induction t=0)</TD><TD class="col">1.32 (t=1)</TD><TD class="col">1.92 (t=2)</TD><TD class="col">4.50 (t=3)</TD><TD class="col">4.90 (t=4)</TD></TR><TR><TD class="col">pICD003</TD><TD class="col">0.05</TD><TD class="col">0.09</TD><TD class="col">0.15</TD><TD class="col">0.33 (induction t=0)</TD><TD class="col">0.56 (t=1)</TD><TD class="col">2.90 (t=2)</TD><TD class="col">4.60 (t=3)</TD></TR><TR><TD class="col">pICD004</TD><TD class="col">0.05</TD><TD class="col">0.12</TD><TD class="col">0.33 (induction t=0)</TD><TD class="col">0.69 (t=1)</TD><TD class="col">1.14 (t=2)</TD><TD class="col">3.30 (t=3)</TD><TD class="col">4.10 (t=4)</TD></TR></TBODY></TABLE>
Another Growth curve with NEB Turbo, pICD003 in NEB Turbo and pICD004 in NEB Turbo was created this time 0.2 % Glucose was added to the LB broth with Chloramphenicol. 

NEB Turbo was here used as a controll. pICD003 in NEB Turbo and pICD004 in NEB Turbo were induced after two hours with 0.5 mM IPTG. <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> strain</TD><TD> 0 min </TD><TD> 100 min </TD><TD> 125 min </TD></TR><TR><TD class="col">pICD003_1</TD><TD class="col">0.05</TD><TD class="col">0.11</TD><TD class="col">0.14</TD></TR><TR><TD class="col">pICD004_2</TD><TD class="col">0.05</TD><TD class="col">0.16</TD><TD class="col">0.26</TD></TR><TR><TD class="col">NEB_turbo</TD><TD class="col">0.05</TD><TD class="col">0.31</TD><TD class="col">0.56</TD></TR></TBODY></TABLE>
Before and after induction each sample was tested with FACS. The mesurments was done every hour for seven hours. The last sample was mesured with FACS on 15-08-13. 
Result of this FACS mesurements: <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> strain</TD><TD> t=0 </TD><TD> t=1 </TD><TD> t=2 </TD><TD> t=3 </TD><TD> t=4 </TD><TD> t=5 </TD><TD> t=6 </TD><TD> t=7 </TD><TD> t=7.2 </TD></TR><TR><TD class="col">NEB_turbo</TD><TD class="col">238</TD><TD class="col">262</TD><TD class="col">237</TD><TD class="col">223</TD><TD class="col">183</TD><TD class="col">181</TD><TD class="col">182</TD><TD class="col">227</TD><TD class="col">223</TD></TR><TR><TD class="col">pICD003</TD><TD class="col">14482</TD><TD class="col">13883</TD><TD class="col">10539</TD><TD class="col">7423</TD><TD class="col">4285</TD><TD class="col">3325</TD><TD class="col">5191</TD><TD class="col">off limits</TD><TD class="col">43151</TD></TR><TR><TD class="col">pICD004</TD><TD class="col">7439</TD><TD class="col">14075</TD><TD class="col">7248</TD><TD class="col">2393</TD><TD class="col">1604</TD><TD class="col">1813</TD><TD class="col">2482</TD><TD class="col">off limits</TD><TD class="col">13920</TD></TR></TBODY></TABLE>

Parameters in Voltage for the first seven mesurements: 
FSC: 350, SSC: 385, Blue C 530/30: 350, Yellow Green B 610/20: 650. 

Parameters in Voltage for the mesurment 7.2: 
FSC: 350, SSC: 385, Blue C 530/30: 350, Yellow Green B 610/20: 500. 


The absorption for RFP before induction decreased a bit. 
After the induction the absorption for RFP decreased further. 
4 hours after the induction the absorption for RFP rises again. 13 hours after the induction the RFP absorption can not be mesured. So the parameters were adjusted and a high level of RFP absorption was obtained. 
We concluded that we have leaky expression, but 0.5 mM IPTG leads to a significant rise in the RFP absorption.
Also we concluded that after four hours the expression rate of our Proteins (only seen RFP) is higher than the cell division rate. Also the cell division rate could be lower than the expression rate of our Protein.  

Transformation of pICD004 and PICD003 into G10 HI high controll strain (protocols). <H1>15/08/13</H1><H2>transformation,SDS-Gels</H2>
Transformation of psB4C5 and psB3C5 into G10 HI high control strain (protocols). Plating out of G10 Hi high control stain on LB agar. 
Preparing of SDS Gels.

<H1>15/08/15</H1><H2>pICD007 with His tag</H2>
Construction of pICD008 (pICD007+His Tag): 

PCR of whole Plasmid pICD007 using Primers with His Tag and overlap for Gibson assambly (see protocols). 

PCR program: 
30 cycles like shown below:

38 °C          10 sec 
60 °C           15 sec 
68 °C          170 sec 
after this 30 cycles the PCR product was stored at 4 °C 

Analytical Gel <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 12:</B> 
analytical gel: The two PCR products are right to the ladder. A pale band is visible for product 2 at the desired size.

</FIGCAPTION></FIGURE>

As a next step to recieve only our PCR product we used the enzyme DpnI to destroy the template dna. So we incubated the PCR product at 37 °C for half an hour and denature the restriction enzym afterwards with 80 degree Celsius for 20 minutes. 

In the following the PCR product was cleaned up with the Nucleo Spin Gel and PCR Clean up Kit. 

In the following we proceeded with a gibson assembly from the PCR product and transformed it into electro competent G10 Hicontrol E.coli cells. <H1>15/08/19</H1><H2>cloning</H2>

The transformation of the Gibson assembly was only successful for pICD007 with an additional sequence of six Histidins behind CdiA. 

So we did the Polymerase chain reaction again for pICD003, pICD004 and pICD007 as a control for the right mastermix. 
<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 13:</B> 
analytical gel
</FIGCAPTION></FIGURE>

The PCR was successful for pICD007 and pICD004 (one of two PCR reactions) and was unsuccessful for pICD003. Maybe we change the conditions of this PCR reaction next time. 

After this we purificated the PCR product and used Gibson assembly to get a circular plasmids form pICD004 and pICD007 again. The Gibson Assembly was transformed into chemical competent E. coli cells (NEB turbo). Here as a control we transformed pICD007 and the pICD004. 

The transformed constructs from Erlangen grew on the Agar plates and single colonys were picked and put into overday cultures ( 3 mL LB broth with CAM and a single picked colony). 

The plasmids out of the constructs from Erlangen were isolated with a Plasmid isolation kit and concentrations were mesured. <H1>15/08/20</H1><H2>Erlangen</H2>
We got colonys on the plates of pICD007 let them grow and isolated the plasmids.    

The constructs from Erlangen were sent to Eurofins for sequencing. <H1>15/08/20</H1><H2>Erlangen</H2>
Sequencing results: 
pWH403-P5(L9A) with iGEM125fwd: correct sequence! 
pWH527-5 and -6 with primers AA537 and AA544: Both clones correct! 
BBa_K592008 with iGEM125fwd: Correct sequence! <H1>15/08/21</H1><H2>building new constructs</H2>

Preparation of Electrocompetent NEB Turbo according to protocol &quot;Competent E.Coli cells (Electroporation)&quot; starting with 300mL culture. 
Transformation of pICD007+His-5 and pICD007+His-6 is done to check the quality of the Electrocompetent cells (Result: Excellent) 
Additionally the GA reaction to create pICD004+His is transformed (Result: A lot of colonies. 6 clones are picked and will be sequenced) 

PCR to prepare new constructs having an RBS upstream of every CDI gene and with a new promoter because the promoter in the already existing constructs has a point mutation <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> template</TD><TD> primer </TD><TD> size of the fragment </TD></TR><TR><TD class="col">pICD001,pICD007,pICD003,pICD004</TD><TD class="col">iCD043, iCD045</TD><TD class="col">13kb, 13kb, 15kb, 15kb</TD></TR><TR><TD class="col">pICD007</TD><TD class="col">iCD044,iCD046</TD><TD class="col">1829 bp</TD></TR></TBODY></TABLE>

Additionally the new promoter K592008 is digested with EcoR1 and Spe1. <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 14:</B> 
analytical gel
</FIGCAPTION></FIGURE>

For the fragments created with primers iCD043 and iCD045 additional bands are visible.
The fragments for pICD007 look fine. 

The desired fragments are excised and extracted using the Qiagen Gel extraction kit.

Nanodrop results after gel extraction: 
	• PCR pICD007 with iCD044 and 046: 26.7ng/µL (lane 4, bottom gel) --&gt; 0.022 pmol/µL

<H1>15/08/22</H1><H2>building new constructs</H2>

PCR is repeated using the excised DNA 
<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 15:</B> 
analytical gel
</FIGCAPTION></FIGURE>
The gel was run too long so the Promoter has run out of the gel.
No bands at the desired size for the repeated PCR. 

Preparation of Electrocompetent NEB Turbo and G10 Hicontrol according to protocol &quot;Competent E.Coli cells (Electroporation)&quot; starting with 2L culture each.


<H1>15/08/23</H1><H2>building new constructs</H2>
Digest of K592008 is repeated with doubled volume. And put to preparative gel.
The band of K592008 is excised and extracted using the Qiagen Gel extraction kit.
<FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 16:</B> 
analytical gel
</FIGCAPTION></FIGURE>

The band of K592008 is excised and extracted using the Qiagen Gel extraction kit.
Transformation of pICD007+His-5 is done in NEB Turbo and G10 Hicontrol to check the quality of the Electrocompetent cells prepared on 15-08-22 (Result: Amazing)

<H1>15/08/24</H1><H2>plasmid isolation, sequencing</H2>

Isolation of the plasmid  pICD004 + His-5, and the constructs from Erlangen which were retransformed to get a higher amout of plasmid. 
Sequencing of pICD004 + His-5. 
Transformation of pICD004 + His-5 into G10 Hicontrol. We want to check in the next days if we can prove the presence of our CdiA protein with western blotting.   

Another Digest of K592008 to get a higher amout of the d.n.a sequence which codes for the new promoter we want to use for CdiA. 

Overnight cultures for pICD007 + His and G10 Hicontrol for the SDS page and western blot tomorrow. 

Repeated PCR of pICD004 and 007 with primer ICD043 and 045 (see 15-08-21)
Fragment of the desired size is excised and extracted using the &quot;modified gel extraction protocol&quot;

<H1>15/08/25</H1><H2>Western Blot, sequencing</H2>

Sequencing results: 
	• PICD004-His with primer iGEM160-rev: Clone 3 is correct, clone 1 has a point mutation destroying the CdiI start codon --&gt; Clone 3 will be used and becomes pICD012 
	• pWH1413 with AA537 and AA544: tetR sequence correct! 
	• pWH527 with AA537 and AA544: tetR sequence correct!
	• pICD007-His with primer iGEM160-rev: Both clones, 5 and 6, are correct! --&gt; Clone 5 becomes pICD013 

Excised bands from yesterday's PCRs --&gt; gel extraction (Zymo kit, which is more suitable for long fragments), eluted in 15 ul
	• PICD004 PCR: 85.5 ng/ul, fragment length 14767bp --&gt; 0.009 pmol/µl 
	• PICD007 PCR: 73.1 ng/ul, fragment length 12897 bp --&gt; 0.009 pmol/µL 

With pICD0013 we want to do an Western Blot with Histidin (6x) monoclonal antibodys. So we prepared dayculutres of pICD0013 and G10 Hicontrol as a negative control. We dont have a positive control. 

Absorbance at 600 nm for the overnight culutres was: pICD0013 = 2.7 and G10 Hicontrol = 3.6 

Both culutures were diluted to an OD = 0.05 in a 200 mL culture. LB Broth without additional glucose was used. At an OD of 0.2 for pICD0013 and an OD of 0.43 for G10 Hicontrol both were induced with 0.5 mM IPTG. An aliquot of equivalent 1 mL of cells at OD-600 = 1.0 was collected and spined down at maximum speed for one minute. We removed the supernatant and store the cells as t = 0 at -20 degree Celsius. This t = 0 was collected after 100 minutes of growth. 

The let both culutures grow for 8 hours at 37 °C degree celsius and 220 rpm. After this 8 hours of induced cell growth we harvested both culutres. The OD-600 mesured value of pICD0013 was 3.0 and of G10 Hicontrol was 4.2. From both culutres an aliquot  of equivalent 1 mL of cells at OD-600 = 1.0 was collected and spined down at maximum speed for one minute. We removed the supernatant and store the cells as t = 0 at -20 degree Celsius. This sample was named t = 1. 

t = 0 and t =1 were resuspend each in 65 µL of 1 x SDS-Page sample buffer ( 900 µL 4 x SDS sample buffer + 100 µL Beta-Mercaptoethanol). 

We also wanted to test for solubility: 
So we pellet 200 mL of each culture and resuspended in 2 mL Phosphat Buffer and added 0.1 mg/µL Lysozym. After the incubation time of half an hour at 37 degree celsius we lyse the cells with sonication ( Amplitude 90 %, 1 s pulse, 2 x 10 minutes). Soluable and Insoluable were seperated by spinning down at 10000 g at 4 °C for 20 minutes. 20 µL of Supernatant were added to 20 µL 1x SDS Page Buffer (Solubale fraction). The pellet was resuspended in Phosphat buffer pH 7.2 and 20 µL of this were added to 20 µL of 1x SDS Page Buffer (insoluable fraction). 


SDS Gel and Western Blot were continued next day.   

<H1>15/08/26</H1><H2>Western Blot, sequencing</H2>

Gel extraction of the new Promoter K592008 does not work neither with Qiagen Gel extraction kit nor following the &quot;Modified gel extraction protocol&quot; 

So Gibson assembly is done with unpurified and just digested plasmid in hope that only the correct fragment will be able to assemble. 
PCR of 15-08-21 for the CDIB fragment from backbone pICD007 and primers iCD044 and iCD066 and purified using the Thermo scientific PCR clean up kit. 

Gibson is performed using one of the backbones pICD004 and pICD007 each of them containing the CDI operon without CDIB, promoter and CDIB. 

The two GA reactions are transformed into G10 Hicontrol electrocompetent cells. 

Running an SDS Page for Western Blot: 

Gel: pICD0013 was used on the lanes2-5 and G10 Hicontrol as a negative control on the lanes 7-10
Lanes: 1 Ladder; 2 cell lysat before induction; 3 cell lysat after 8 hours of induction, 4 supernatant, 5 insoluable part, 6 ladder, 7 cell lysat before induction; 8 cell lysat after 8 hours of induction, 9 supernatant, 10 insoluable part 

We prepared the Western Blot after a standart protocol. 

<H1>15/08/27</H1><H2> transformation, failed western blot </H2>

Colonies of transforemed cells are picked to do miniprep the following day. 

We use a monoclonal antibody to mark His-6 tags. But no signal could be obtained. Western Blot failed! <H1>15/08/28</H1><H2> Killing assay, transformation, failed western blot </H2>

The picked candidates of the GA are used for miniprep using the Macherey Nagel miniprep kit. 

The plasmids are digested with Sac1 and put to analytical gel. 
The fragments have a size equal to the size of digested plasmid carrying the promoter K592008 so the GA did not work and undigested backbone was transformed. 
Killing Assay 

Overnight cultures of G10 hicontrol, pSB4C5, pICD004, pILS5 were prepared on 15-08-27. 

OD was measured of the overnight cultures and during the day 
Induction at OD of about 0,2 with 0,5mM IPTG <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> strain</TD><TD> overnight culture </TD><TD> 9:30 </TD><TD> 10:30 </TD><TD> 11:05 </TD><TD> 12:40 </TD><TD> 13:50 </TD><TD> 14:50 </TD><TD> 16:10 </TD></TR><TR><TD class="col">G10 uninduced</TD><TD class="col">3.6</TD><TD class="col">0.05</TD><TD class="col">0.12</TD><TD class="col">0.23 (induction at 11:20)</TD><TD class="col">0.67</TD><TD class="col">0.79</TD><TD class="col">1.04</TD></TR><TR><TD class="col">G10 induced</TD><TD class="col">0.61</TD><TD class="col">0.81</TD><TD class="col">1.00</TD></TR><TR><TD class="col">pICD004 uninduced</TD><TD class="col">1.8</TD><TD class="col">0.05</TD><TD class="col">0.14</TD><TD class="col">0.23 (induction at 11:20)</TD><TD class="col">0.46</TD><TD class="col">0.84</TD><TD class="col">1.14</TD></TR><TR><TD class="col">pICD004 induced</TD><TD class="col">0.47</TD><TD class="col">0.84</TD><TD class="col">1.10</TD></TR><TR><TD class="col">pSB4C5 uninduced</TD><TD class="col">2.7</TD><TD class="col">0.05</TD><TD class="col">0.06</TD><TD class="col">0.09</TD><TD class="col">0.09</TD><TD class="col">0.19 (induction at 13:50)</TD><TD class="col">0.27</TD></TR><TR><TD class="col">pSB4C5 induced</TD><TD class="col">0.26</TD></TR><TR><TD class="col">pILS5 uninduced</TD><TD class="col">3.5</TD><TD class="col">0.05</TD><TD class="col">0.12</TD><TD class="col">0.24 (induction at 11:20)</TD><TD class="col">0.46</TD><TD class="col">0.8</TD><TD class="col">1.04</TD></TR><TR><TD class="col">pILS5 induced</TD><TD class="col">0.47</TD><TD class="col">0.83</TD><TD class="col">0.98</TD></TR></TBODY></TABLE>
As pILS5 and pICD004 grow very similar, the growth in coculture should be similar as well to see an effect of growth inhibition. 
Cultures are grown in an incubator with 37°C and 200rpm until induction. After induction the flasks were put in an incubator without shaking. The flasks are shaked every 30 minutes by hand. 

Cultures of pICD004 and pILS5 are mixed directly after induction in a ratio of 1:1 and 10:1                          (10*piCD004: 1*pILS5) induced as well as uninduced (as control)

<TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> well</TD><TD> 1 </TD><TD> 2 </TD><TD> 3 </TD><TD> 4 </TD><TD> 5</TD><TD> 6 </TD><TD> 7 </TD></TR><TR><TD class="col">sample</TD><TD class="col">G10 uninduced</TD><TD class="col">G10 induced</TD><TD class="col">pICD004 uninduced</TD><TD class="col">pICD004 induced</TD><TD class="col">pILS5 uninduced</TD><TD class="col">pILS5 induced</TD><TD class="col">pSB4C5-RFP uninduced</TD></TR></TBODY></TABLE><TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> well</TD><TD> 8 </TD><TD> 9 </TD><TD> 10 </TD><TD> 11 </TD><TD> 12 </TD></TR><TR><TD>sample</TD><TD class="col">pSB4C5-RFP induced</TD><TD class="col">pICD004+pILS5 1:1 uninduced</TD><TD class="col">pICD004+pILS5 10:1 uninduced</TD><TD class="col">pICD004+pILS5 1:1 induced</TD><TD class="col">pICD004+pILS5 10:1 induced</TD></TR></TBODY></TABLE>

Samples are measured in the FACS, directly after induction (t0, 1:4 dilution), after 2h (t1, 1:10 dilution), after 3h (t2, 1:20 dilution), after 4h (t3, 1:20 dilution) and on the folloPCRwing day (t4, 1:20 dilution). 
results: wait for our awesome presentation.


<H1>15/08/29</H1><H2> Gibson assembly, PCR, sequencing </H2>

Gibson assembly is done to clone synthesized CDIA-CT into pICD004 

PCR with primers ICD047 and ICD016 to amplify the whole plasmid except the CT
After PCR purification 29,5ng/µL 


Sequenced CTs are used  3 fold molar. <H1>15/08/30</H1><H2>Overnight culutre </H2>

Colonies on all plates. 6 clones each are picked and grown over night to do miniprep the following day. <H1>15/08/31</H1><H2> plasmid isolation, PCR, digest </H2>
Miniprep of all clones using the macherey nagel kit. Elution in 30µL elution buffer. 

Test PCR to test if old or new CT is in the plasmid. 
digest: 
iCD031 and 160 rev would give a band if new CT is in the plasmid.
iCD032 and 160 rev would give a band if new CT is in the plasmid 

Bands visible for pICD009-2, pICD010-4, pICD010-6, pICD011-6.  
Sent to sequencing using primers iCD031 and 160 rev. 

More clones are picked for pICD008, pICD009 and pICD010 in case that sequencing results are wrong. <H1>15/09/01</H1><H2>sequencing, overnight culture</H2>

Sequencing results:
PICD009-2 with primers ICD031 and 160rev: Entire sequence correct! --&gt; becomes pICD009 
PICD010-4 with same primers as above: Missing A in CdiA gene --&gt; frameshift!
PICD010-6 with same primers as above: GTG to GTC in base pair position 8340 of CdiA --&gt; Silent mutation. We decided to continue with this plasmid --&gt; becomes pICD010 
PICD011-6 with same primers as above: TTC to TCC in CdiA gene! Results in F to S mutation. Cannot be used! 


New minipreps of pICD008 and pICD011 candidates are prepared. <H1>15/09/02</H1><H2>sequencing</H2>
Sequencing results: 
PICD008-12 with primers ICD031 and 160rev: Sequence correct! --&gt; becomes pICD008 
PICD011-10 with same primers as above: point mutation, will not be used 
PICD011-7 with same primers as above: All correct! --&gt; becomes pICD011 

PCR to create fragments for two new target strains? <H1>15/09/03</H1><H2>sequencing</H2>
Sequencing results: 
pSB1C3-K1493504 (GFP with Ptet) sequenced with iGEM125for: All correct! <H1>15/09/04</H1><H2>pICD010</H2>

More colonies of pICD010 are picked and sent to sequencing: -&gt; 10-17 is correct and becomes pICD010. <H1>15/09/05</H1><H2>Gibson assembly pICD016, pICD017</H2>
PCR to create fragments for GA to create pICD016 (pTet+CDI+TetR) and pICD017 (CDI+GFP) for further killing/activation assays. <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> Fragment</TD><TD> primer </TD><TD> template </TD><TD> exspected size </TD></TR><TR><TD>fragment 1</TD><TD>ICD007, ICD054</TD><TD>pICD007</TD><TD>10648 bp</TD></TR><TR><TD>fragment 2</TD><TD>ICD048, ICD047</TD><TD>pSB4C5</TD><TD>3225 bp</TD></TR><TR><TD>fragment 3</TD><TD>ICD050, ICD049</TD><TD>pWH527</TD><TD>851 bp</TD></TR><TR><TD class="col">fragment 4</TD><TD class="col">ICD052, ICD053</TD><TD class="col">pSB1C5 K1493504</TD><TD class="col">956 bp</TD></TR><TR><TD class="col">fragment 5</TD><TD class="col">ICD055, ICD053</TD><TD class="col">pILS5</TD><TD class="col">932 bp</TD></TR></TBODY></TABLE><FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 17:</B> 
analytical gel
</FIGCAPTION></FIGURE>

All fragments are at the expected size. Gel extraction according to &quot;Modified gel extraction&quot; protocol. 

Gibson Assembly pICD016: <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> Fragments</TD><TD> ato molar </TD></TR><TR><TD class="col">fragment 1</TD><TD class="col">14.47</TD></TR><TR><TD class="col">fragment 2</TD><TD class="col">28.94</TD></TR><TR><TD class="col">fragment 3</TD><TD class="col">43.41</TD></TR><TR><TD class="col">fragment 4</TD><TD class="col">43.41</TD></TR></TBODY></TABLE>


Gibson Assembly pICD017: <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> Fragments</TD><TD> ato molar </TD></TR><TR><TD class="col">fragment 1</TD><TD class="col">14.47</TD></TR><TR><TD class="col">fragment 2</TD><TD class="col">28.94</TD></TR><TR><TD class="col">fragment 5</TD><TD class="col">43.41</TD></TR></TBODY></TABLE>
Transformation with electrocompetent G10 E. coli cells.

<H1>15/09/06</H1><H2>timelapse</H2>
Timelapse microscope movie of mixed culture of pILS5 and pICD004 on TB agar. 
Growth not optimal. The experiment will be repeated with LB agar and M8 agar.

Colonies for pICD0016 and pICD0017 are picked and grown over night. <H1>15/09/07</H1><H2>pICD010</H2>

The picked colonies are picked using the macherey nagel plasmid prep kit. And digested using
Xba 1. For pICD016 two fragments (1100kb and 4000 kb) are expected. For pICD0017 one fragment (17000bp) is expected. <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 18:</B> 
analytical gel: 1-6 are pICD016 2-12 are pICD017
</FIGCAPTION></FIGURE>
The bands do not look like expected so more clones are picked 

As an preexperiment for the gel movie the endpoint is analysed. 
Different dilutions are put to the agar pads. 1:10, 1:100 and 1:1000 
1:1000 to few cells. 1:10 and 1:100 will be used for further experiments. LB agar is the best choice for a movie 

To create CDI plasmid without RFP and lacI pICD004 is cut with Xba1. <H1>15/09/08</H1><H2>sequencing, overlap PCR</H2>
Sequencing results: 
pICD010-17 with primers ICD031 and 160rev: Sequence is correct and even does not have the point mutation in cdiA at position 8340 (see 15-09-01). This will be tested for induction of red fluorescence in the flow cytometer. 

pICD010-6 did not show induction of red fluorescence and was trashed. Prepared new pICD010 clones for sequencing. 

Cultures are grown again and put to agar pads. The movie shows that cells have to be induced before they are mixed and put to the agar pads. 

The clones picked the day before are prepped and digested with Xba1. <FIGURE style="text-align:center;"><FIGCAPTION style="margin-top:5px;font-size:11pt;color:#606060;text-align:center;line-height:110%"><B>Figure 19:</B> 
analytical gel:Left to the ladder are pICD016 clones. Rigth to the ladder are pICD017 clones.
</FIGCAPTION></FIGURE>

No clone looks like expected. 

Overlap extension PCR is done to combine the PCR fragments. 

Fragment 2, 3 and 4 are combined using primer iCD048 and iCD053
Fragments 2,5 are combined using primers iCD048 and iCD053

Gibson of PCR overlap product 1 (2,3 and 4) with fragment 1 and PCR overlap product 2 (2,5) with fragment 1.
Product is transformed into electrocompetent G10 E. coli cells. 

Additionally a plasmid has to be prepared with constitutive RFP to be able to better compare the induced and uninduced killer cells. 
pICD004 is cut with Xba1 and dephosphorilatet with phosphonukletid kinase. Ligation with a part containing RFP with constitutive promoter. Ligation with 2µL ligation buffer and 1µL T4 ligase. 

Product is transformed into electrocompetent G10 E. coli cells. <H1>15/09/10</H1><H2>timelapse movie</H2>

The movie is finally done on LB agar pads with preinduced cells. The experiment was measured for 6 hours and photos are taken every 15 minutes. <H1>15/09/11</H1><H2>fadbatch experiment</H2>

Prearrangement for a new killing assay. The setup will be a fat bitch fermentation to prove if a small founder population containing the cdi operon is able to establish in a big population of target cells. 
Overnight cultures are started for pICD004 (induced and uninduced) pILS5 (induced and uninduced) G10 (induced and uninduced) 

<H1>15/09/12</H1><H2>fatbatch experiment</H2>

Fadbatch experiment is started. Cultures are inocculated to an OD of 0,05 and grown to 0,2. Cocultures are started as well as monocultures as control. The cocultures are started in triplicates to have biological replicates. <TABLE id="layouttable" style="border:2px;border-style:solid;border-color:#F0F0F0;"><TBODY><TR style="background-color:#f66300;"><TD> G10</TD><TD> pICD004 </TD><TD> pILS5 </TD><TD> 1:1 uninduced </TD><TD> 1:1 induced </TD><TD> 1:10 induced </TD><TD> 1:100 induced </TD><TD> 1:1000 induced </TD></TR></TBODY></TABLE>

The cultures are shaked every 30 minutes for a 5 seconds at 100rpm. 
After 2h the cultures are diluted to an OD of 0,2 and samples for FACS measurements are taken. </DIV><DIV style="position:absolute;text-align:center;z-index:2;background:#e0e0e0;line-height:150%;bottom:0px;padding-top:3px;margin-bottom:0px; padding-left:235px; padding-right:80px;min-height:60px;min-width:100%;max-width:100%;box-sizing:border-box;clear:both;"><SPAN style="font-size:8pt; color:white;"> iGEM Marburg - ZSM Karl-von-Frisch-Straße 16, D - 35043 Marburg</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>