<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Braunschweig_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Braunschweig/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Braunschweig/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Braunschweig/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Braunschweig/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Braunschweig/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Braunschweig/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="menu"><UL id="menu"><LI><A href="https://2013.igem.org/Team:Braunschweig"> Start </A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Team"> Team </A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Team">Members</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/TU Braunschweig">TU Braunschweig</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Braunschweig">Official Team Profile</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Attributions">Attributions</A></LI></UL><LI><A href="https://2013.igem.org/Team:Braunschweig/Project/Description"> Project </A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Project/Description">Project Outline</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Project/Engineering_Principles">Engineering Principles</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Project/Appoach">Approach</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Project/Impact">Potential Impact</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Parts">Parts submitted to the registry</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Outreach">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Outreach">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Outreach/Activities">Activities</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Outreach/Communication_strategy">Guide Part 1 - Communication strategy</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Outreach/Implementation">Guide Part 2 - Implementation</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Outreach/Coverage_Report">Guide Part 3- Media Coverage Report</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Outreach/SocialMediaAnalysis">Guide Part 4 - Social Media Analysis</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Cooperations/iGEM Teams">Cooperations</A></LI><LI><A href="https://2013.igem.org/Team:Braunschweig/Safety"> Safety </A></LI><LI><A href="https://igem.org/2013_Judging_Form?id=1073"> Judging </A></LI></DIV><DIV id="contents"><H1>Labjournal</H1><P>
This is the documentation of our lab work. Achievements of each week are summerized followed by a daily discription of our experiments.
An overview on how we approached this project can be found under <B><A href="https://2013.igem.org/Team:Braunschweig/Project/Appoach">Approach</A></B>.
For detailed protocols of certain procedures please refer to <B><A href="https://2013.igem.org/Team:Braunschweig/Protocols">Protocols</A></B>. Attributions are given for each day, however please check our<B><A href="https://2013.igem.org/Team:Braunschweig/Attributions">Attributions section</A></B> for efforts beyond the lab work.</P><DIV class="weeks"><DIV id="Week1" class="menuSection"><H2><A href="#Week1">Week 1: May 19 - May 26, 2013</A></H2><P style=" margin-left:5px; margin-right:5px; margin-bottom:0px;">We set up our labspace and started some preparatory work.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; padding:0px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, May 21, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px; padding:0px;">
We finally moved in our lab. A bit of dust here and some rubbish to dispose there, but after a few hours of combined strength our lab was ready to go. Wet experiments can begin!</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, May 23, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Kerstin, Laura</B>
We prepared some chemically competent <I>E. coli</I> XL1Blue MRF' cells for all the transformations we are going to perform during the project.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px">Friday, May 24, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Kerstin, Laura</B>
Today we transformed our freshly prepared competent cells for test purposes. We used the plasmids pUC18 and pUC19 to calculate the transformation efficiency. 100 pg of DNA were used for each transformation with 50 µl of compentent cells. 
Additionally, competent cells were plated on agar plates containing ampicillin, kanamycin and chloramphenicol respectively to ensure that the strain is not carrying any corresponding resistances. The plates were incubated at 37 °C overnight.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Saturday, May 25, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Kerstin, Laura</B>
Cells from yesterday's transformation only grew on agar plates containing ampicillin. The plates with the additional antibiotics were empty, thus we can conclude that our competent <I>E. coli</I> XL1 Blue MRF' cells do not carry other antibiotic resistances.
111 colonies were counted for pUC18 transformation, 50 for pUC19 transformation on ampicillin containing agar plates. According to these numbers the transformation Efficiency was calculated.
Transformation efficiency: 
pUC18: ~ 10<SUP>6</SUP> µg<SUP>-1</SUP>
pUC19: ~5x10<SUP>5</SUP> µg<SUP>-1</SUP></P></DIV><DIV id="Week2" class="menuSection"><H2><A href="#Week2">Week 2: May 27 - June 1, 2013</A></H2><P style="margin-left:5px; margin-right:5px;">The distribution kit arrived and we started with the actual labwork. 19 BioBricks had to be transformed into our <I>E. coli</I> XL1 Blue MRF' strain to secure the parts. Additionally we developed the cloning strategy for the next weeks.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, May 27, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px;"><B>Investigators: All</B>
Today we developed our cloning strategy. Details can be found in the <A href="https://2013.igem.org/Team:Braunschweig/Project/Appoach">Approach section</A></P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, May 28, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Tabea, Jan, Laura</B>
The distribution kit arrived!
We resuspended the DNA of 19 BioBricks, measured the DNA concentrations via NanoDrop and transformed each BioBrick. This is the full list of BioBricks we planned on using in our project:</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, May 29, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Oliver, Jan</B>
No growth could be observed on plates with the BioBricks
B0012
B0010
C0060
C0062
J23106
J23100

We repeated the transformation of all the above listed BioBricks. However, we noticed that many of these parts had an <I>ampR</I> backbone.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, May 30, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigator: Jan</B>
Unfortunately, some plates still remained empty:
B0010
J23106
J23100</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, May 31, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Kevin, Roman, Tabea</B>
We did a colony PCR of the parts we secured so far. We got bands for all parts except for BioBricks C0061 and C0062.
</P></DIV><DIV id="Week3" class="menuSection"><H2><A href="#Week3">Week 3: June 2 - June 8, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">

This week we successfully cloned and transformed some of our first combination Bricks. We also managed to obtain some BioBricks that could not be transformed via Phusion-PCR directly from resuspended DNA.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Sunday, June 2, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Roman, Laura</B>
Since the transformations of the BioBricks B0010, C0060, C0061, C0062, C0070, J23100 and J23106 were not successful, these Bricks were amplified via Phusion-PCR directly from the resuspended DNA from the distribution kit instead.
We prepared 5 mL liquid cultures of Bricks B0012, B0032, B1009, C0060, C0061, C0062, C0070, C0071, C0078, C0079, E0240, E0430, J06702, R0062, R0071 in 2xYT medium containing respective antibiotics in order to miniprep the plasmid DNA tomorrow.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, June 3, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Oliver, Laura, Kerstin</B>We miniprepped the plasmids containing Bricks B0012, B0032, B1009, C0060, C0061, C0062, C0070, C0071, C0078, C0079, E0240, E0430, J06702, R0062 and R0071. As stated earlier the BioBricks C0061 and C0062 showed no visible bands in the colony PCR but we still prepped them. Furthermore we prepared glycerol stocks of all strains for further use.
Our gel electrophoresis of the PCR-amplified BioBricks showed a suitable bands for BioBricks C0060, C0061, C0070, J23100, J23106 which were recovered successfully from the gel. The BioBricks B0010, C0062 showed no bands on the gel and therefore could not be isolated. </P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, June 4, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin</B>

We received the new set of BioBricks we ordered from the registry: B0015 (this one is going to be our new terminator, replacing the combination of B0010 and B0012), B0017, E0420, J23100, J23106. These were transformed via heatshock in <I>E. coli</I> XL1 Blue MRF' cells. We also prepared following of the miniprepped BioBricks for sequencing:  C0061, B0012, B1009. The results are outlined in the following table: 
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, June 5, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura, Kevin, Oliver</B>
Since we were not able to transform the BioBricks B0015, J23100 and J23106 we decided to obtain these Bricks via Phusion-PCR directly from resuspended DNA from the distribution kit.
A gel electrophoresis was conducted with the freshly acquired PCR products of B0015, J23100  and J23106 as well as C0060, C0061, C0070, J23100 and J23106 from our last batch of PCR amplificates. Besides J23100 from today's PCR all BioBricks showed bands of the expected size and therefore were isolated via gel extraction.
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, June 6, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Oliver, Laura</B>
We prepared to clone some of our first and essential parts. The BioBricks were digested according to the table below. 
Parts labeled as vector were dephosphorylated to prevent religation and then purified.
In order to obtain the inserts a gel electrophoresis was conducted and the appropriate band was isolated by gel extraction. The ligations were conducted according to the following table.
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Friday, June 7, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Tabea, Oliver, Laura, Kevin</B>
We transformed our ligations from yesterday using our competent glycerol stocks without prior heat inactivation of T4-ligase. Transformed cells were plated on agar plates containing glucose and chloramphenicol and were grown at 37 °C overnight.
To test the success of our ligation beforehand we conducted a colony PCR (extension time 30 s) using 1 µL of our untransformed ligation mix of C0061+B0015 and B0032+J23100 as template. The conducted gel electrophoresis was visualized and showed a band of the expected size for B0032+J23100. The band for C0061+B0015 was too small. Further investigation revealed the chosen extension was too short.
We also send some of the BioBricks miniprepped on June 3, 2013 for sequencing (primers VR and VF2).</P></DIV><DIV id="Week4" class="menuSection"><H2><A href="#Week4">Week 4: June 9 - June 15, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">This week we confirmed the successful cloning of the constitutive promoter + RBS. Furthermore we confirmed the addition of the double terminator to the autoinducer synthases and the lactonase. More BioBrick sequences were verified. However, we observed some point mutations in the prefix/suffix sequences of some bricks as well as major annotation differences in the BioBrick C0061.
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Sunday, June 9, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin </B>
We ran a colony PCR of two clones of each ligation (transformation from June 6, 2013) and the biobricks P1002 and K091117 (primers VF2 and VR).
We also inoculated overnight cultures for plasmid preparations the next day.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, June 10, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin </B>
We analyzed yesterday’s colony PCR gel electrophoresis for the expected fragment sizes. Looking at the gel all clones seem to be positive, although clone 1 of J23100+B0032 and J23106+B0032 did not contain the promoters as later shown by DNA sequencing.

We isolated the plasmid DNA from all clones with a miniprep kit following the manufacturer’s instructions.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, June 11, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura </B>
We transformed the chemically competent <I>E. coli</I> XL1 Blue MRF’ with the ligation C0061+B0015 (prepared on June 6, 2013) and the biobrick E0420 (eCFP).
The plasmid DNA isolated the day before was sent to GATC for sequencing.
The DNA sequences from June 7, 2013 were evaluated by sequence alignment with the vector maps.
The biobricks C0079, C0078, C0070 and B0032 were sequence verified. However, the biobrick C0061 did not match the annotated sequence. The prefix was missing and there was an additional oligonucleotide sequence in the 5’ region of the biobrick. This led to the assumption that C0061 might be a composite part.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, June 12, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Tabea, Oliver</B>
The biobrick B0015 was amplified by PCR using the resuspended DNA from the distribution kit as template. The PCR was successful (expected fragment size was 443 bp).
A colony PCR of two clones of each of yesterday’s transformations (E0420 and C0061+B0015) was run (primers VF2 and VR). Overnight cultures of the analyzed clones were inoculated.
The remaining DNA sequences (from June 7,2013) were evaluated by sequence alignment. While the BioBricks J06702, B0012, C0071, R0062 and R0071 were sequence verified, the BioBrick C0062 did not match the sequence, the DNA sequencing for clone 2 failed. Furthermore the sequence alignment of E0240 revealed additional 25 bp after the EcoRI restriction site. The DNA sequence of E0430 was confirmed except for a point mutation (T to A between Not/I and XbaI restriction site) as well as the BioBrick J23112 which contained a point mutation in the prefix sequence (T to A).
 


</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, June 13, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin, Jan</B>
Today we received the NEB iGEM Support Kit. This will keep the labwork rolling ;-)
Yesterday’s colony PCR(E0420 and C0061+B0015) was analyzed on a 1% agarose gel.The amplified fragment for E0420 matched the expected fragment size of 1192 bp. The PCR for C0061+B0015 failed. A faint PCR product of 6-7kb could be detected for clone 2 indicating (again) an additional oligonucleotide sequence 5’ of the BioBrick. This additional DNA sequence may have up to 6kb. We isolated the plasmid DNA from all clones of E0420 and C0061 + B0015 with a miniprep kit following the manufacturer’s instructions to send them to GATC.
In order to clone the inducible promoters Plux (R0062), Prhl (R0071) and Plas (K091117) 5’ of the RBS (B0032), we digested them with EcoRI and SpeI. At the same time the lactonase (C0060) and lactonase + TT were digested with XbaI and PstI to clone them 3’ of a RBS. The autoinducer synthases + TT (C0078+B0015, C0070+B0015, C0061+B0015) were digested with XbaI and PstI for cloning. Also we did a test restriction of C0061 since the DNA sequence indicated it contained an additional nucleotide sequence.
The expected fragments of the promoters are 82, 80 and 153 bp.  We were not able to detect the fragments on the agarose gel. The restriction digest of C0060, C0078+B0015 and C0070+B0015 was not complete. However, the fragments matched the expected sizes, so they were extracted from the agarose gel.
For C0061, the restriction failed. The size of the linearized vector is about 10kb confirming differences in the Registry of Standard Parts annotation and the BioBrick.
The sequence data from June 11, 2013 was analyzed by sequence alignments. J23100+B0032 and J23105+B0032 did not contain the promoters. J23112+B0032, C0078+B0015, C0070+B0015, C0060+B0015 was sequence verified. Unfortunately the sequencing failed for K091117 and P1002.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Friday, June 14, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin, Oliver</B>
Since cloning of the constructs J23100+B0015 and J23106+B0015 failed previously and gel extraction of the promoters is difficult due to their small fragment sizes, we tried a copy &amp; paste cloning approach:
The promoters J23100 and J23106 were PCR amplified using resuspended DNA from the distribution kit as a template. The inducible promoters R0062, R0071 and K091117 were PCR amplified using plasmid DNA (miniprep) as a template.
The endonuclease digest on June 13, 2013  was repeated from scratch. This time the digest was successful and the fragments were extracted. The vectors were dephoshorylated.</P></DIV><DIV id="Week5" class="menuSection"><H2><A href="#Week5">Week 5: June 16 - June 22, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">This week was dominated by a lot of cloning. We equipped our constitutive and inducible promoters with ribosome binding sites, terminators, an ampicillin resistance gene and a lactonase (not all of them got each of these elements). Additionally, we separated the ampicillin resistance gene from its native promoter in order to make its expression inducible. Although cloning was mostly successful, we were struggling to amplify and purified the <I>luxR</I> brick (C0062), which cost us a lot of time and effort. Unfortunately, it also wasn’t rewarded with success in the end.
Another important step was to adapt our cloning strategy towards new chromoproteins which we received from iGEM team Uppsala. These allow us to evaluate our cultivation experiments solely with own equipment, rather than having to find a cell sorter with suitable excitement lasers for all fluorescent proteins.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, June 17, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Oliver, Jan </B>
Today, we digested the inducible promoters, followed by purification and clean-up. Then we equipped the inducible and previously digested constitutive promoters with a RBS (B0032) by ligating respective parts. Subsequently, these ligated constructs were used to transform competent bacteria.
We also inoculated overnight suspension cultures with B0015- and B0032-transformed <I>E. coli</I> XL1 Blue MRF' cells from glycerol stocks for DNA preparation.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, June 18, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Jan</B>
We performed a colony PCR to test if cloning from the previous day was successful. Positively tested clones were used to inoculate 2xYT liquid cultures for DNA preparation. Overnight suspension cultures of B0015- and B0032-transformed cells were used for DNA preparation.
In order to separate the ampicillin resistance gene from its native promoter we performed a Phusion-PCR with appropriate primers on the P1002 brick.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, June 19, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Roman, Laura, Kerstin</B>
In order to harvest the successfully ligated bricks, DNA  of liquid cultures from positively tested clones was prepared for sequencing.
We also repeated the colony PCR of the last four ligations as they showed questionable restriction patterns.
After the second colony PCR was also found to be negative, these parts were once again digested, including gel extraction and purification. Furthermore, the previously amplified <I>ampR</I> gene was purified by gel electrophoresis. However, gel extraction did not yield enough DNA to work with.
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, June 20, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Oliver</B>
First, we performed another Phusion-PCR on <I>ampR</I> (P1002 brick) since the first PCR did not yield enough DNA. We also tried to amplify the <I>luxR</I> gene (C0062) through Phusion-PCR from pSB1A2 plasmid. However, not enough DNA was obtained and digestion of the purified PCR product did not show the expected bands.
More digestions were set up to harvest the separated ampicillin resistance gene and prepare DNA for the next cloning round. Additionally, lactonase (C0060) was ligated with a RBS (B0032), the <I>ampR</I> gene was cloned behind our inducible promoters and we ligated each autoinducer synthetase with a RBS (B0032) overnight.
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Friday, June 21, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Jan, Oliver, Roman</B>
The overnight ligation was followed by transformation of competent <I>E. coli</I> XL1Blue MRF cells. In parallel, PCR for C0062 from pSB1A2 was done using Phusion and Q5 polymerase. However, both preparations were negative, so we purified the other two repressor/activator genes (C0071, C0079) by gel extraction. Subsequently, we ligated these bricks with a constitutive promoter. As usual, we transformed competent bacteria with the ligated vectors.
Furthermore, Phusion- and Q5-PCR was repeated with different annealing temperatures, which were also all negative.
We also changed our cloning strategy as we thankfully received new chromoproteins from iGEM team Uppsala.
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Saturday, June 22, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin</B>
Today was a PCR day. We performed a colony PCR of 10 clones of each ligation we set up yesterday, resulting in 100 PCRs! We had to use all electrophoresis chambers that we could find in order to screen them all at the same time.
</P></DIV><DIV id="Week6" class="menuSection"><H2><A href="#Week6">Week 6: June 23 - June 29, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
The focus of this week’s work was cloning further constructs and the evaluation of their quality by colony PCR and sequencing. Gel extraction was supposed to yield a functional structural gene for a beta lactamase. Additionally we took soil samples for the iGEM Team Norwich.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Sunday, June 23, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigator: Laura </B>
The ligations of J23100-B0032-C0071/C0079 and B0032-C0070/C0078-B0015 were positive in the colony PCR, so that we inoculated cultures for biomass production of the clones. We then conducted a colony PCR with ten clones of the B0032-C0061-B0015 construct to identify a correctly ligated clone. Cells with R0071/R0062/K091117-B0032-<I>ampR</I> were inoculated on agar plates with ampicillin supplemented media since we had evidence that our promoters might be leaky.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, June 24, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Roman, Laura </B>
Yesterday’s colony PCR of B0032-C0061-B0015 unfortunately was negative for all ten clones, as can be seen on the gel. Roman made minipreps of J23100-B0032-C0071/C0079 and B0032-C0070/C0078-B0015. These parts and furthermore C0061 and C0062 were prepared and sent to be sequenced. Since purification of ligated <I>ampR</I> in a column was not successful we tried to obtain it by gel extraction. The ligations of R0071-B0032-<I>ampR</I>, R0062-B0032-<I>ampR</I> and K091117-B0032-<I>ampR</I> were successful as indicated by colony PCR, therefore we inoculated cultures with bacteria carrying this construct for further processing.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, June 25, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigator: Jan, Kerstin</B>
We made minipreps and glycerol stocks of R0071/R0062/K091117-B0032-<I>ampR</I>. For validation of K091117’s sequence (2 basepairs differed from the sequence given in the registry) Kerstin conducted another Phusion-PCR. After ligation the product was evaluated by colony PCR. Together with R0071/R0062/K091117-B0032-<I>ampR</I>, K091117 was prepared and sent for sequencing. K091117-B0032-<I>ampR</I> was a double clone. 
We also took soil samples for the iGEM Team Norwich.
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, June 25, 2013 to Friday, June 28, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px">
The iGEM Team Braunschweig went to Berlin to join the Strategy Congress Biotechnology 2020+ of the Federal Ministry of Education and Research.</P></DIV><DIV id="Week7" class="menuSection"><H2><A href="#Week7">Week 7: June 30 - July 6, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
In week 7 we started our first experiments with our inducible promotors. Unfortunately we had trouble caused by the leakiness of two of our three promotors. We did further experiments but could not find a solution yet. 
Good news is that we received the chromoproteins from Uppsala! So we started working on our new cloning strategy by combining the chromoprotein DNA with our constitutive promotor-RBS construct. We hope to see nice and colorful colonies next week! 
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, July 1, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Roman, Laura </B>In order to combine lactonase and our lactonase-terminator construct as well as LasI und the LasI-terminator construct with the ribosome binding site and a constitutive promotor we digested B0032, C0060, C0061-B0015, J23112-B0032 and C0060-B0015. Afterwards gel extraction was performed for the inserts C0060, C0061-B0015 and C0060-B0015 and DNA purification for the vectors. Inserts and vectors were ligated using T4-DNA Ligase (NEB) overnight at 16°C.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, July 2, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura </B>
Yesterday's ligations transformed into <I>E. coli</I> XL1 MRF' by heatshock and plated on 2xYT agar containing glucose and Chloramphenicol. 
Since the chromoprotein DNA from Uppsala iGEM Team 2011 arrived today we now switched to our new cloning strategy starting with resuspending and transforming the newly arrived DNA into <I>E. coli</I> XL1 Blue MFR' by heatshock. Cells were as well plated on 2xYT agar containing glucose and chloramphenicol.</P><P>
Even if we switched to our new strategy we still kept working on the fluorescence markers. Therefore we digested the BioBricks E0420 (eCFP), E0430 (YFP) and J06702 (mCherry) as well as our promotor-RBS-LasR and promotor-RBS-RhlR-constructs. Still lacking the construct containing LuxR we decided to proceed with the promotor-RBS-construct instead to ligate it to the YFP-Brick. Gel extraction was performed for the inserts and DNA purification for the vector parts. Inserts and bricks were ligated overnight using T4 DNA ligase (NEB) at 16°C.
Since we now have our first constructs containing the inducible promotors as well as the the ampicillin resistence gen we started our first leakiness experiments. All constructs (Plas, Prhl, Plux in combination with the RBS and <I>ampR</I>) were cultivated overnight in 2xYT medium containing various ampicillin concentrations.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, July 3, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin </B>
Before we started working on our own chromoprotein-constructs, we screened for the optimum cultivation conditions. We cultivated <I>E. coli</I> XL1 Blue MRF' containing a new construct from Uppsala iGEM encoding the device J23110-B0034-aeBlue. We tested expression of the blue chromoprotein at different temperatures, oxygen supply and rpm. These experiments led to the conclusion that low oxygen supply and a temperature of 37°C result in higher expression rates.
The evaluation of the leakiness of the inducible promotors showed that only the Prhl is not leaky. Plux as well as Plas were leaky and showed growth of <I>E. coli</I> XL1 Blue MRF' at all tested ampicillin concentrations. Therefore we repeated the experiment using higher ampicillin conentrations.
The bricks containing the fluorescencw markers ligated yesterday were transformed in <I>E. coli</I> XL1 Blue MRF' by heatshock.Colony PCR of the constructs containing lactonase and LuxI which were transformed yesterday was performed. Since all screened colonies showed religated vectors we decided to try an alternative restriction strategy to combine the BioBricks by using the endonuclease NcoI.  
B0032, C0060 and C0061-B0015 were digested with NcoI and corresponding SpeI and XbaI. Gel extraction was performed for the vector DNA as well as the inserts.
Overnight liquid cultures were inoculated with <I>E. coli</I> XL1 Blue MRF' containing the chromoprotein in 2xYT supplemented with chloramphenicol.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, July 4, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin </B>
Ligation of the constructs B0032-C0060 (containing lactonase) and B0032-C0061-B0015 (containing LuxI) was performed for 30 minutes at room temperature using T4 DNA Ligase (NEB). Constructs were transformed in <I>E. coli</I> XL1 Blue MRF' by heatshock and plated on agar plates.
As we were still having trouble with the brick C0062 (LuxR) we amplified the brick from the device F2620 using Q5 Polymerase (NEB). Gelextraction was performed for the PCR products.The chromoprotein cultures were miniprepped and DNA was directly used for digestion to combine the chromoproteins with our promotor-RBS construct. The insert parts were extracted from an agarose gel while the vector part was dephosphorylated and purified using DNA clean-up columns. 
Colony PCR was conducted for the constructs with the fluorescent proteins which were transformed yesterday. We were able to identify colonies with the right construct size for all tested constructs.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Friday, July 5, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura, Jan </B>
When we checked the agar plates for transformed colonies with the LuxI and lactonase containing constructs we did not have any colonies. 
The digested chromoprotein DNA was ligated with the RBS and the promotor-RBS construct at room temperature for 30 minutes. The ligated DNA was transformed in <I>E. coli</I> XL1 Blue MRF' by heatshock and plated on agar plates. We cannot wait to see colorful colonies on Sunday :-)
Since we almost ran out off our competent cells it was time for new ones. New compentent cells were made and transformation efficiency was tested.
A new idea to cope with the leakiness of the Plas and Prhl was to try a different antibiotic which cannot be metabolised. We used carbenicillin instead of ampicillin at different concentrations in liquid culture with 2xYT medium. Unfortunatly no effect could be observed on the leakyness of both inducible promotors.</P></DIV><DIV id="Week8" class="menuSection"><H2><A href="#Week8">Week 8: July 7 - July 13, 2013</A></H2><P>
This week we cultivated  the chromoprotein expression cassetts for the first time in liquid culture. It was a great motivation to observe our constructs to work as expected. Everyone is really excited about the colorful cultures!  The leakiness problem we still try to get rid of is less motivating but we still did not give up trying to find a solution this week.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Sunday, July 7, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura</B>
Today, 2xYT liquid cultures were inoculated with colonies of <I>E. coli</I> XL1 Blue MRF' transformed with the constructs containing the blue, red and yellow chromoproteins (aeBlue, eforRed and amilGFP respectively) controlled by constitutive promoters (J23100) and equipped with a ribosome binding site (B0032). Cultures are grown overnight at 37°C and 250 rpm</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, July 8, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura</B>
As the cultures were prepared yesterday, plasmid DNA containing the chromoprotein expression cassettes were minipreped and glyercol cell stocks of cells were prepared.
Also, transformations of the ligated DNA of lactonase expression cassette and Lux  autoinducer synthase LuxI (C0061) flanked by a ribosome binding (B0032) site and a double terminator (B0015) in <I>E. coli</I> XL1 Blue MRF' were performed by heat shock.
Last Friday we observed that there is no effect on the leakiness of the inducible promoters by using carbenicillin as a selection marker. We assumed that there was a contamination either in the medium or the glycerol sotck, so we repeated the leakiness tests with tetracycline and ampicillin insteadt in 2xYT liquid cultures overnight.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday 9, July 17, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin, Kevin</B>
The results of the leakiness experiment: The growth of cells equipped with the Plas inducible ampicillin resistance cassette, probably a double clone, was firstly limited at ampicillin concentration of 400µg/ml. The Plas inducible ampicillin resistance cassette and the positive control grew at all tested concentrations of ampicillin (100µg/ml-1 mg/ml). 
To show that the transformation of our chromoprotein expression cassettes with and without strong constitutive promoters was successful, colony PCR was performed.
The constitutive RhlR and LasR transcription factor expression cassettes were digested with   appropriate restriction enzymes and desired fragments were extracted from gel after gelelectrophoresis. Lastly, the DNA was purified. Transformation of the Plas inducible ampicillin resistance cassette was performed. However, after prepping the DNA and sending it for sequencing the sequence revealed that this construct was a double clone.
The earlier mentioned transcription factor expression cassettes for RhlR and LasR were then ligated and incubated overnight. Also, 2xYT liquid cultures of all chromoprotein expression cassettes were prepared and incubated overnight.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, July 10, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Melanie, Laura, Kerstin</B>
Today, we transformed the ligated DNA of our transcription factors RhlR and LasR expression cassettes (containing strong constitutive promoter, RBS and double terminator) into <I>E. coli</I> XL1 Blue MRF' by heat shock and plated the samples on 2xYT agar containing chloramphenicol. The colony PCR with GoTaq of these ligations did not show fragments of expected size for any of the selected clones.
All chromoprotein containing constructs (expression cassettes as well as chromoprotein encoding genes combined with RBS) were minipreped and sent for sequencing.
Digestion of our trouble-makingautoinducer synthase LuxI and  transcription factor LuxR cassettes was done with endonuclease NcoI as an alternative to standard restriction enzymes. As a next step the DNA was ligated and transformed into <I>E.Coli</I> XL1 Blue MRF'. Extraction of DNA from gel was only performed for the ribosome binding site (B0032). 
Furthermore, we digested the chromoprotein expression parts for aeBlue and eforRed, ran a gel electrophoresis, extracted desired fragments from the gel and subsequently purified the DNA.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, July 11, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Anna, Kevin</B>
Today, we ligated the extracted and purified DNA of our constitutive chromoprotein expression cassettes encoding aeBlue and eforRed. The products were our combined constructs of Las- or Rhl-autoinducer synthase LasI and RhlI and constitutive eforRed chromoprotein and aeBlue expression cassettes respectively.
A PCR of the terminator (B0015) was performed with Q5 polymerase in order to gain more DNA material for digestion and ligation. 
The ligations of our combined constructs of autoinducer synthases LasI (flanked by RBS and double terminator) and constitutive eforRed expression cassette as well as RhlI (flanked by RBS and double terminator) and constitutive aeBlue expression cassette and the LuxR transcription factor expression cassette were transformed into <I>E. coli</I> XL1 Blue MRF'. 
The last action for today was to inoculate an overnight liquid culture of our ampicillin resistance cassettes equipped with either Rhl-, Lux- or Las-dependent promoter  in LB Medium with different ampicillin concentrations (100 µg/ml, 250 µg/ml, 500 µg/ml and 1 mg/ml) to test for potential leakiness in a different medium. Cultures were grown overnight at 37°C and 250 rpm.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Friday, July 12, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura</B>
Today, our aim was to combine the chromoprotein expression cassettes with the double terminator (B0015). Thus, the chromoprotein expression cassettes were digested, a gel electrophoresis was run and insert fragments were extracted from gel while the vector part was dephosphorylated. Afterwards purification of vector and insert parts was performed and the DNA was ligated according to our protocol. Subsequently, the ligated DNA was transformed into <I>E. coli</I> XL1 Blue MRF' by heat shock. 
Yesterday‘s test for leakiness under different conditions showed that the use of LB Medium wouldn’t solve the problem of the leaky promoters since liquid cultures showed high cell density for all ampicillin concentrations. We still need to find a solution for this problem in order to get our concept to work. 
Furthermore, we performed new colony PCRs and agarose ge of the transcription factor LuxR, the combined Rhl-autoinducer synthase RhlI and aeBlue expression cassette as well as the Plas induced ampicillin resistance cassette from colonies on agar plates.</P></DIV><DIV id="Week9" class="menuSection"><H2><A href="#Week9">Week 9: July 14 - July 20, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
The highlight of this week was our first visit at the DSMZ (German collection of microorganisms an cell cultures). We got the chance to analyze our cells containing different chromoprotein expression cassettes with a FACS (Fluorescent Activated Cell Sorter) and found out that it is difficult to differentiate our cells them like this.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, July 15, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Anna, Kerstin, Laura, Melanie</B>
In order to construct  the RhlR and LasR expression cassettes with an upstream double terminator both expression cassettes and the double terminator were restricted and extracted from gel and purified respectively. After ligation a transformation was performed.
Furthermore the problem with the leakiness of our promoters was discussed again. Therefore we tested the expression cassettes containing the inducible promoters in different minimal media. We observed leakiness for the inducible promoter of the las quorum sensing system, but no leakiness for the inducible promoter from the rhl quorum sensing system. Thus, the minimal media did not solve our problems with the promoter leakiness.
In the afternoon Kerstin, Laura, Anna and Melanie visited the DSMZ in Braunschweig  and tested the fluorescence characteristica of cells carrying a single chromoprotein and mixtures of cell carrying different chromoproteins in a FACS (Fluorescent Associated Cell Sorter). When measuring cells containing the same chromoprotein expression cassette we saw that it might be possible to differentiate cells carrying the yellow amilGFP and the pink eforRed chromoprotein by thier fluorescence cause they showed significantly different signals. However, when we measured mixed cultures no differentiation was possible without any obvious reason. A few days later we observed a mixture of cells under the microscope and found out that the expression of the chromoprotein may lead to stickiness of the cells causing them to build big clusters, thus unfortunately no differentiation and sorting of the cells in the FACS is possible in order to determine mixed culture composition.
Back at our lab, a PCR of the inducible promoter expression cassette of the las quorum sensing system and RBS-luxI-TT device was performed.
The inducible ampicillin resistance cassettes of the las und rhl qourum sensing system were transformed in <I>E. coli</I> Top10 F' cells to get rid of the problem with promoter leakiness.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, July 16, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Melanie</B>
After performing PCR and gel electrophoresis of the LasR and RhlR expression cassettes combined with an upstream double terminator no clones showed a right band size on the gel.
Minipreperation of the las expression cassette and RBS-<I>luxI</I>-TT device was carried out and prepared for sequencing.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, July 17, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Laura</B> Once more a colony PCR of the LasR and RhlR expression cassettes with the combinded upstream double terminator as well as of the three inducible promoter expression cassettes was performed.
Furthermore, 2xYT liquid cultures of <I>E. coli</I> XL1 Blue MRF'  mCherry, YFP and eCFP fluorescence expression cassettes and the LasR and RhlR expression cassettes with an upstream double were inoculated. 
The ampicillin resistance was extracted from gel.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, July 18, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Roman, Melanie</B>
The plasmid DNA encoding mCherry, YFP and eCFP fluorescenze expression cassettes and the LasR and RhlR expression cassettes with an upstream double were prepped from overnight cultures and prepared for sequencing. Glycerol stocks of every Brick were made.
For the ligation of the LuxR expression cassette the promoter with RBS and the <I>luxR</I> gene were restricted and purified.</P></DIV><DIV id="Week10" class="menuSection"><H2><A href="#Week10">Week 10: July 21 - July 27, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
This week we managed to construct our N-butyryl-HSL (Rhl system) inducible ampicillin resistance cassette (R0071-B0032-AmpR-B0015-J23100-B0032-C0071) and our N-3-oxododecanoyl-HSL (Las system) inducible ampicillin resistance cassette (K091117-B0032-AmpR-B0015-J23100-B0032-C0079). Furthermore, the constructs of the amilGFP cassette (J23100-B0032-K592010-B0015), the N-3-oxododecanoyl-HSL synthase LasI and eforRed cassette (B0032-C0076-B0015-J23100-B0032-K592012-B0015), the N-butyryl-HSL synthase RhlI and aeBlue cassette (B0032-C0070-B0015-J23100-B0032-K864401-B0015) and terminator-modified N-butyryl-HSL (Rhl system) inducible Amp-resistance cassette (B0015-R0071-B0032-AmpR-B0015-J23100-B0032-C0071) are nearly finished.
Additionally we started our first test runs with the new chromoproteins.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, July 22, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Kerstin, Laura</B>
We digested the N-butyryl-HSL/RhlR induced ampicillin resistance cassette (R0071-B0032-<I>ampR</I>) , the Rhl transcription factor (RhlR) cassette (B0015-J23100)-B0032-C0071), the N-3-oxododecanoyl-HSL/LasR induced ampicillin resistance cassette (K091117-B0032-<I>ampR</I>) and the Las-transcription factor (LasR) cassette (B0015-J23100-B0032-C0079) to ligate them to new constructs. The N-butyryl-HSL/RhlR inducible ampicillin resistance cassette (R0071-B0032-<I>ampR</I>-B0015-J23100-B0032-C0071) is supposed to be combined with the RhlR cassettte while the the N-3-oxododecanoyl-HSL/LasR inducible ampicillin resistance cassette (K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) is intended to be combined with the LasR expression cassette. We successfully performed a gel extraction of the RhlR cassette (B0015-J23100-B0032-C0071) and the LasR cassette (B0015-J23100-B0032-C0079). Additionally we measured the absorption spectra emission of our newly received chromoproteins.
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, July 23, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin</B>
Today we transformed yesterday’s ligations of the N-butyryl-HSL/RhlR inducicle ampicillin resistance cassette (R0071-B0032-<I>ampR</I>-B0015-J23100-B0032-C0071) and the N-3-oxododecanoyl-HSL/LasR inducible ampicillin resistance cassette (K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079).
Furthermore, we inoculated overnight cultures of the <I>E. coli</I> XL1 Blue MRF' expressing the chromoproteins for further spectrum measurements. Cultures were grown in 2xYT containing chloramphenicol at 37°C and 250 rpm overnight.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, July 24, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Kerstin</B>
A colony PCR of the successfully transformed N-butyryl-HSL/RhlR inducible ampicillin resistance cassette (R0071-B0032-<I>ampR</I>-B0015-J23100-B0032-C0071) and N-3-oxododecanoyl-HSL/LasR inducible ampicillin resistance cassette (K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) was performed. We also extracted the chromoproteins by performing a cell disruption and measured their emission spectra.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, July 25, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin</B>
We confirmed that our N-butyryl-HSL/RhlR inducicle ampicillin resistance cassette and RhlR cassette (R0071-B0032-<I>ampR</I>-B0015-J23100-B0032-C0071) as well as our N-3-oxododecanoyl-HSL/LasR inducible ampicillin resistance cassette and LasR Cassette (K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) were correctly ligated by gel electrophoresis. Both contructs showed PCR fragments of expected size. In order to obtain material for sequencing an overnight culture of 2xYT containing chloramphenicol was inoculated with clones bearing the right contructs.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Friday, July 26, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin</B>
The N-butyryl-HSL/RhlR inducicle ampicillin resistance cassette (R0071-B0032-<I>ampR</I>-B0015-J23100-B0032-C0071) and the N-3-oxododecanoyl-HSL/LasR inducible ampicillin resistance cassette (K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) were minipreped for sequencing.
The constructs of the amilGFP cassette (J23100-B0032-K592010-B0015), the N-3-oxododecanoyl-HSL synthase LasI and eforRed cassette (B0032-C0076-B0015-J23100-B0032-K592012-B0015), the N-butyryl-HSL synthase RhlI and aeBlue cassette (B0032-C0070-B0015-J23100-B0032-K864401-B0015) and the terminator-modified N-butyryl-HSL/RhlR inducible ampicillin resistance cassette (B0015-R0071-B0032-<I>ampR</I>-B0015-J23100-B0032-C0071) were digested, ligated and transformed.
As we were not able to transform the pSB1C3 plasmid of BBa_B0015, we performed a Q5 PCR of the part. The fragment was extracted from a gel and purified for further use.</P></DIV><DIV id="Week11" class="menuSection"><H2><A href="#Week11">Week 11: July 28 - August 3, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
This week was all about cloning our final constructs – unfortunately without success – and fighting our leaky promotors. We managed to modify the Las-inducible ampicillin resistance cassette with a terminator. A test in liquid culture to confirm if this makes the promotor less leaky was undermined by contamination. We also modified our constructs by adding new reporters.

</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, July 29, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura</B>We transformed our inducible ampicillin resistance into a new <I>E. coli</I> strain (JM109) in order to test whether the promotor being leaky is strain specific. 
We performed a colony PCR to test the result of our latest cloning experiments for the amilGFP cassette (J23100-B0032-K592010-B0015), the inducible ampicillin resistance and LasR transcription factor cassette (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079), the N-3-oxododecanoyl-HSL synthase with eforRed reporter cassette (B0032-C0076-B0015-J23100-B0032-K592012) and the N-butyryl-HSL synthase with aeBlue reporter cassette (B0032-C0070-B0015-J23100-B0032-K864401-B0015).
The amilGFP cassette (J23100-B0032-K592010-B0015), the N-3-oxododecanoyl-HSL synthase LasI with eforRed reporter cassette (B0032-C0076-B0015-J23100-B0032-K592012) and the N-butyryl-HSL synthase RhlI with aeBlue reporter cassette (B0032-C0070-B0015-J23100-B0032-K864401-B0015) showed the expected bands on the gel and were prepped. The inducible ampicillin resistance and LasR transcription factor cassette (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) was prepped as well despite a second colony PCR showing no positive results.
We also tested the inducibility of our final Prhl construct in 2xYT liquid culture with ampicillin overnight, which showed normal growth without being induced, indicating that the promotor is still leaky.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, July 30, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura</B>
The amilGFP cassette (J23100-B0032-K592010-B0015), the inducible ampicillin resistance and LasR transcription factor cassette (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079), LasI with eforRed reporter cassette (B0032-C0076-B0015-J23100-B0032-K592012) and RhlI with aeBlue reporter cassette (B0032-C0070-B0015-J23100-B0032-K864401-B0015) were sent for sequencing.
The sequencing revealed the following: The amilGFP cassette showed a point mutation in prefix A → T in front of XbaI. The sequence of the inducible ampicillin resistance and LasR transcription factor cassette (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) was verified. The LasI combined with eforRed reporter cassette (B0032-C0076-B0015-J23100-B0032-K592012) was sequence verified as well as the RhlI combined with aeBlue reporter cassette (B0032-C0070-B0015-J23100-B0032-K864401-B0015). 
We also cloned the PCR amplificates of BioBricks C0062 and C0061 + B0015 into the sequence verified pSB1C3.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, July 31, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Roman, Kevin, Anna</B>
We started a restriction digest of several bricks to construct three new parts: The Inducible promoter with Rhl expression cassette (R0071-B0032-<I>ampR</I>-B0015-J23100-B0032-C0071) and the LasI with eforRed reporter cassette (B0032-C0076-B0015-J23100-B0032-K592012) were combined to our first complete construct (fianl Prhl construct). 
The RhlI with amilGFP reporter cassette (B0032-C0070-B0015-J23100-B0032-K592010-B0015) was derived from the RhlI expression cassette (B0032-C0070-B0015) and the amilGFP cassette (J23100-B0032-K592010-B0015) as the blue chromoprotein was not usable for fluorescence detection.
The second final construct (BBa_K1073034), which is N-3-oxododecanoyl/LasR inducible, was constructed from the inducible ampicillin resistance and LasR transcription factor cassette (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) and the N-butyryl-HSL synthase RhlI with aeBlue reporter cassette (B0032-C0070-B0015-J23100-B0032-K864401-B0015).
Parts E0420 and J23100 were combined as another alternative for fluorescence experiments.
We tested the results of yesterday's cloning experiments via colony PCR. C0062 and C0061-B0015 did not show bands of expected size.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, August 1, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Roman, Melanie</B>
We continued yesterday's cloning experiment by purifying the insert DNA via gel extraction and dephosphorylation of vector DNA followed by ligation and transformation (see protocoll).</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Friday, August 2, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Anna, Melanie</B>In order to test the success of the last days’ cloning experiment we conducted a colony PCR. Unfortunately all clones of both our supposed final Prhl constructs were negative. Therefore the PCR was repeated later with new colonies. 
The N-butyryl-HSL synthase RhlI with amilGFP reporter cassette (B0032-C0070-B0015-J23100-B0032-K592010-B0015) and the eCFP cassette (J23100-E0420) showed both clones bearing the correct construct.
We also conducted a test in 5 mL 2xYT liquid cultures with different concentrations of ampicillin and autoinducers to test if our new constructs were leaky. Cells bearing the inducible ampicillin resistance and LasR transcription factor (B0015-K091117-B0032-ampR-B0015-J23100-B0032-C0079) were tested with ampicillin supplements between 1 and 8 µg/mL. We expected no growth as the ampicillin resistance was not induced.
Cells carrying the inducible ampicillin resistance with RhlR expression cassette (R0071-B0032-B0015-J23100-B0032-C0071) were incubated in 2xYT medium containing ampicillin in concentrations between 1 and 2 µg/mL and additionally 5 to 50 µmol/L N-butyryl-HSL (Rhl autoinducer). As we induce the ampicillin resistance we expect some of the cultures to grow, depending on which concentration is optimal.
Cells carrying the constructs inducible ampicillin resistance combined with LasR transcription factor (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) and the inducible ampicillin resistance with RhlR expression cassette (R0071-B0032-B0015-J23100-B0032-C0071) respectively were grown in media supplemented with chloramphenicol were prepared as positive control. A culture carrying the inducible ampicillin resistance with RhlR expression cassette (R0071-B0032-B0015-J23100-B0032-C0071) was cultivated in 2xYT medium with different ampicillin concentrations as negative control.
The next day, all cultures were heavily grown indicating a contamination in our medium.</P></DIV><DIV id="Week12" class="menuSection"><H2><A href="#Week12">Week 12: August 4 - August 10, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
This week was overshadowed by contaminated medium interfering with our experiments. We managed to transform the final Prhl inducible construct in <I>E. coli</I> Top10F’. As the promoter seemed to be not leaky in this strain we successfully conducted a growth curve experiment showing a difference in growth between induced and non-induced cultures. We are still missing our final Plas inducible construct.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, August 5, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Anna, Melanie</B>
The successful cloning of our final Prhl inducible construct was finally confirmed via colony PCR. Unfortunately the liquid cultures for the minipreps were contaminated. We decided to repeat the entire colony PCR as last PCR’s replated colonies showed almost no coloration.
We also adapted a new cloning strategy for our Plas inducible construct by swapping vector and insert. As we discovered later, the insert was extremely hard to recover via gel extraction because of almost identical band sizes. We still tried it and proceeded with ligation.
We prepared liquid cultures for a miniprep of the final Prhl inducible construct, the N-butyryl-HSL synthase RhlI with amilGFP reporter cassette (B0032-C0070-B0015-J23100-B0032-K592010-B0015) and the eCFP cassette (J23100- E0420). These were, again, contaminated the next day.
In order to find a setup where our promotors are not leaky, we conducted an experiment with liquid cultures of cells bearing the Prhl inducible ampicillin resistance with RhlR transcription factor cassette (R0071-B0032-B0015-J23100-B0032-C0071) and the Plas inducible construct ampicillin resistance combined with LasR transcription factor cassette (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) in which we varied the concentration of ampicillin and the corresponding inducer. The next day these cultures were contaminated like the rest of today's experiments. This was a serious problem!
In order to test if the leaky promotor is caused by the <I>E. coli</I> XL1 Blue MRF' strain we transformed the inducible ampicillin resistance in <I>E. coli</I> Top10F’.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, August 6, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Anna</B>
We managed to test our inducible resistances without contamination interfering with the experiment:
– Cells bearing the the Prhl inducible ampicillin resistance combined with LasR transcription factor and added terminator (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) were tested with ampicillin supplements between 1 and 8 µg/mL. We expected no growth as the ampicillin resistance was not induced.
– Cells carrying the Prhl inducible ampicillin resistance combined with LasR transcription factor and added terminator (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) were cultivated on medium containing ampicillin supplements between 1 and 8 µg/mL and additionally 10 µmol/ml N-3-oxododecanoyl-homoserine lactone. As we induced the ampicillin resistance we expected some of the cultures to grow, depending on which concentration is optimal.
– Cells bearing the Prhl inducible ampicillin resistance with RhlR expression cassette (R0071-B0032-B0015-J23100-B0032-C0071) were tested with ampicillin supplements between 1 and 8 µg/mL. We expected no growth as the ampicillin resistance was not induced.
– Cells bearing the Prhl inducible ampicillin resistance with RhlR expression cassette (R0071-B0032-B0015-J23100-B0032-C0071) were cultivated on medium containing ampicillin supplements between 1 and 8 µg/mL and additionally 10 µmol/mL N-butyryl-homoserine lactone. As we induce the ampicillin resistance we expect some of the cultures to grow, depending on which concentration is optimal.
– Cells carrying the inducible ampicillin resistance combined with LasR transcription factor cassette (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) and the inducible ampicillin resistance with RhlR expression cassette (R0071-B0032-B0015-J23100-B0032-C0071) respectively were grown in media supplemented with chloramphenicol as positive control. Medium supplemented with ampicillin only and medium supplemented with each inducer only were prepared as negative controls.
The next day all but the negative controls showed normal growth confirming that our promotor is still leaky.

A colony PCR confirmed that transformation of the Plas inducible ampicillin resistance with LasR transcription factor cassette (B0015-K091117-B0032-ampR-B0015-J23100-B0032-C0079) and the Prhl inducible construct ampicillin resistance with RhlR expression cassette (R0071-B0032-<I>ampR</I>-B0015-J23100-B0032-C0071) into Top10F’ was successful.</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px">New liquid cultures for prepping the final Prhl inducible construct (K1073035), the N-butyryl-HSL synthase RhlI with amilGFP reporter cassette (B0032-C0070-B0015-J23100-B0032-K592010-B0015) and the eCFP cassette (J23100-E0420) in <I>E. coli</I> TOP10F’ and <I>E. coli</I> XL1 BlueMRF' were inoculated. </P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, August 7, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan, Kevin, Laura</B>
We retried to clone the final Plas inducible construct containing the aeBlue chromoprotein and added a new construct to our list as an alternative to the final Prhl inducible construct containing the eforRed chromoprotein because the aeBlue chromoprotein is not usable for our planned fluorescence experiments.
The final Plas inducible construct was derived from the N-butyryl-HSL synthetase RhlI with aeBlue reporter cassette (B0032-C0070-B0015-J23100-B0032-K864401-B0015) (vector) and the Plas inducible ampicillin resistance with LasR transcription factor cassette (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) (insert).
The final Plas inducible construct with amilGFP reporter (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079-B0032-C0070-B0015-J23100-B0032-K592010-B0015) was derived from the N-butyryl-HSL synthetase RhlI with amilGFP reporter cassette (B0032-C0070-B0015-J23100-B0032-K592010-B0015) (vector) and the Las-inducible ampicillin resistance with LasR transcrip-tion factor (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079) (insert).

To test whether our promotor is still leaky in <I>E. coli</I> Top10F' we cultivated our newly transformed strains in 2xYT medium supplemented with 1 µg/mL ampicillin. Normal growth in all cultures showed that the promotor is leaky in this strain as well.

We send our freshly prepped bricks for sequencing. As a result the sequences of the final Prhl the N-butyryl-HSL synthetase RhlI with amilGFP reporter cassette (B0032-C0070-B0015-J23100-B0032-K592010-B0015) was confirmed, but the prepped DNA were contaminated.   
The sequence of the eCFP cassette (J23100-E0420) was confirmed, but the prepped DNA was also contaminated as well as the DNA of the Plas inducible ampicillin resistance with RhlR expression cassette (R0071-B0032-B0015-J23100-B0032-C0071).  </P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, August 8, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura</B>A colony PCR showed the expected bands for the final Plas inducible construct (K1073034) and the final Plas inducible construct with amilGFP reporter (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079-B0032-C0070-B0015-J23100-B0032-K592010-B0015). Hopefully our final constructs are ready to use now. The clones were prepped for sequencing.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Friday, August 9, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura</B>
The final Prhl inducible construct (K1073035) was retransformed into <I>E. coli</I> Top10F'.
The prepped DNA of the final Plas inducible construct (K1073034) and the Plas inducible amilGFP cassette (B0015-K091117-B0032-<I>ampR</I>-B0015-J23100-B0032-C0079-B0032-C0070-B0015-J23100-B0032-K592010-B0015) was sequenced. While the sequence of the Plas inducible amilGFP cassette could not be verified, the sequence of the final Plas inducible construct containing aeBlue expression cassette was finally verified!
We prepared liquid cultures of the final Prhl construct (K1073035) in different <I>E. coli</I> strains (XL1 Blue MRF’, Top10F') for a growth curve experiment.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Saturday, August 10, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura</B>Some of the liquid culture was used for prepping and sequencing the Prhl inducible construct in <I>E. coli</I> Top10F' cells. The sequence was confirmed.
We measured our first successful growth curve showing the difference in growth between induced and non-induced versions of the Prhl inducible construct. The eforRed expression cassette in <I>E. coli</I> Top10F' was miniprepped and glycerol stocks of this strain were made.
The main cultures (induced and not induced) of the final PRhl inducible construct were inoculated to a start OD520 of 0.05 in 75 ml 2xYT containing ampicillin with cells of the pre-culture. In order to induce the expression of ampR N-3-butyryl homoserine lactone was added to a final concentration of 10 µM. Samples were taken at appropriate times depending on the growth phase until the induced culture reached the stationary phase. OD was determined at 520 nm to avoid absorptions by chromoproteins.
</P></DIV><DIV id="Week13" class="menuSection"><H2><A href="#Week13">Week 13: August 11 - August 17, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
This week was dominated by testing our constructs in continuous as well as in batch culture for inducibility and regulation. Unregulated growth in chloramphenicol containing medium and regulated growth in ampicillin containing medium was tested in continuous culture. The growth kinetics observed for our Rhl regulated could be verified in a second growth curve for induced and uninduced growth. Further we received <I>E. coli</I> JM109 strains containing a reporter plasmid expressing <I>luxCDABE</I> in the presence of specific activating  N-acyl homoserine lactones (AHSLs) (Winsen, M.K. et al. 1998) resulting in bioluminescence of these reporter strains. We will be using these strains in order to verify the production of the specific homoserine lactones by our constructs.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Sunday, August 11, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Roman </B>
We prepared pre-cultures of our chromoprotein expression cassettes (promoter-RBS-chromoprotein) in <I>E. coli</I> XL1 Blue MRF'. For each chromoprotein (amilGFP, eforRed, aeBlue) 50 ml 2xYT containing chloramphenicol were inoculated from -80°C glycerol stocks and grown at 37°C and 250 rpm over night.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, August 12, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin, Laura </B>
Pre-cultures of chromoprotein constructs were mixed in one main culture in order to see how they behave during cultivation over several hours. OD<SUB>520</SUB> of pre-cultures was measured in order to inoculate the main culture with  33% of each strain to a final OD<SUB>520</SUB>=0.3. For the main culture, 25 ml 2xYT containing chloramphenicol were inoculated with the three different strains and grown at 37°C and 250 rpm in a non-baffled flask. Samples from the culture were taken at several time points, diluted and plated on 2xYT agar-plates containing chloramphenicol. Agar plates were incubated at 37°C over night.
Today we also observed our cells containing the chromoprotein expression cassette under the fluorescence microscope.

During the day we noticed that the 2xYT medium used for the pre-cultures yesterday showed contamination. Thus, there is a high possibility  of contamination in the pre-culture as well as in the main culture. Therefore we decided to repeat the experiment. New pre-cultures of the three chromoprotein expression cassettes where inoculated in 50 ml 2xYT containing chloramphenicol directly from -80°C.
In order to have our constructs available in different <I>E. coli</I> strains for the fluorescence microscopy experiments, the eforRed expression cassette was transformed into <I>E. coli</I> Top10F' by electroporation. Transformed cells were plated on 2xYT agar containing chloramphenicol and incubated over night at 37°C.
A continuous cultivation of the Prhl inducible in <I>E. coli</I> Top10F' was prepared, including preparation of pre-cultures of our inducible construct in 50 ml 2xYT containing chloramphenicol and grown at 37°C and 250 rpm over night.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, August 13, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin, Melanie, Judith, Jan, Roman </B>
Unfortunately, the agar plates from yesterday’s mixed culture didn’t show any colour which might be due to contamination. Therefore we did not evaluate these plates any further.  We also had some problems with the shaker and lost the culture containing the aeBlue construct. A test was run on how to dilute a mixed culture of the remaining eforRed and amilGFP constructs in order to be able to count colonies of each colour on large agar plates.

The transformation of the eforRed construct did not work (no colonies on agar plate) and has to be repeated.

The first continuous cultivations were carried out in order to get to know the routines and problems of a continuous cultivation. Due to problems with the regulation of pumps no valuable results were produced but the setting was improved based on these experiences.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, August 14, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Anna, Kevin </B>
Transformation of the eforRed expression cassette in <I>E. coli</I> Top10F' was performed by electroporation. 

The reporter strains for the Las and Rhl systems (<I>E. coli</I> JM109 pSB1075 and <I>E. coli</I> JM109 pSB406 respectively) arrived today. Liquid cultures in 2xYT containing ampicillin were inoculated and grown at 37°C and 250 rpm overnight. 

Pre-cultures of <I>E. coli</I> Top10F' containing final Prhl inducible construct and <I>E. coli</I> XL1 Blue MRF' containing final Plas inducible construct in 50 ml 2xYT containing chloramphenicol for continuous cultures were grown at 37°C and 250 rpm overnight.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, August 15, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin, Roman, Kevin, Jan, Judith </B>
Pre-culture of erforRed expression cassette in <I>E. coli</I> Top10F' in 50 ml 2xYT containing chloramphenicol was inoculated from the  agar plate and grown at 37°C and 250 rpm overnight.

For the reproduction of the growth curve a pre-culture of the Prhl inducible construct in <I>E. coli</I> Top10F' in 30 mL 2xYT containing chloramphenicol and incubated at 37°C overnight. 

To test the production of homoserine lactones (HSL) by our final constructs we prepared a pre-culture of each reporter strain in 30 ml 2xYT containing ampicillin and incubated them overnight at 37°C and 250 rpm. We also made glycerol cell stocks of the reporter strains for further use.
Cells containing the finale constructs were grown in 30 ml 2xYT containing ampicillin as well at 37°C and 250 rpm. We inoculated these cultures with a high cell density as growth depends on the leakiness of the inducible promoters. These cultures were grown over 24 h in order to reach a high HSL concentration in the culture broth. As negative controls strains bearing constructs with the inducible promoters but not the HSL synthesis were grown in 30 ml 2xYT over night at 37°C as well.

The first attempt to cultivate regulated and unregulated mixed cultures in continuous culture was made today. For regulated growth ampicllin was added to the medium, for unregulated growth chloramphenicol was used as selection marker. Samples of each culture were taken at several time points. OD<SUB>520</SUB> was measured and dilutions of samples were plated on 2xYT agar plates containing chloramphenicol. 
We had problems to find the right dilution of cell suspension for the agar plates (too many or too little colonies on plates). The results of these cultivations were therefore not statistically relevant and no conclusions about the regulation of growth by our constructs could be drawn.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Friday, August 16, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin</B>
The eforRed expression cassette in <I>E. Coli</I> TOP10F’ was miniprepped and glycerol stocks of this strain were made.

The main cultures (induced and not induced) of the final Prhl inducible construct were inoculated to a start OD<SUB>520</SUB> of 0.05 in 75 ml 2xYT containing ampicillin with cells of the pre-culture. In order to induce the expression of <I>ampR</I> N-3-butyryl homoserine lactone was added to a final concentration of 10 µM. Samples were taken at appropriate times depending on the growth phase until the induced culture reached the stationary phase. OD was determined at 520 nm to avoid absorptions by chromoproteins.
To verify production of HSLs by our constructs the pre-cultures containing the finale constructs as well as the negative controls were centrifuged for 10 min at 6000 rpm and 4°C. Supernatant was transferred to a new Falcon tube and sterilized by filtration. 
Dilution series of the supernatants and the synthetic HSLs as standards were pipetted in 96-well microtiter plates. Wells were inoculated with the corresponding reporter strain and grown for 3 h at 37°C. Bioluminescence produced by the <I>luxCDABE</I> of the reporter strains was detected by a microplate reader. We were able to show that our constructs produced the specific HSLs. However due to the high background especially in the N-3-oxododecanoyl-HSL producing strain we might need to modify the experimental layout in order to get a stronger signal.</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>References:</B><B>Winson, M. K., S. Swift, L. Fish, J. P. Throup, F. Jørgensen, S. R. Chhabra, B. W. Bycroft, P. Williams, and G. S. A. B. Stewart.</B><I>1998.</I> Construction and analysis of luxCDABE-based plasmid sensors for investigating N-acyl homoserine lactone-mediated quorum sensing. FEMS Microbiol. Lett. <B>163</B>:185–192.</P></DIV><DIV id="Week14" class="menuSection"><H2><A href="#Week14">Week 14: August 18 - August 24, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
The focus of this week’s work was the preparation of the final constructs in order to take pictures under the fluorescence microscope. Furthermore we started the second continuous bioreactor culture.
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, August 19, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin, Kevin, Roman, Jan, Judith</B>
We made glycerol stocks of J23100-B0032-eforRed-cassette and the final inducible constructs in different strains (<I>E. coli</I> Top10 F' and <I>E. coli</I> JM109). Furthermore we transformed J23100-B0032-aeBlue into strain JM109. The functionality of the final Plas construct in strain <I>E. coli</I> JM109 was evaluated by growing it on media supplemented with ampicillin and the inducer N-3-oxododecanoyl homoserine lactone. The strain grew and developed a blue color. This observation led to the conclusion that the construct worked in<I>E. coli</I> JM109. 
Furthermore we miniprepped the final Plas contruct in <I>E. coli</I> JM109 and revised the sequencing results of the final eforRed-construct in <I>E. coli</I> Top10 F’. The final Prhl construct was sequence verified.
We also conducted regulated and unregulated continuous bioreactor cultivations of mixed cultures of <I>E. coli</I> TOP10F'bearing final Prhl construct and <I>E. coli</I> JM109 containing the final Plas construct. The cultures were inoculated with a 70 % : 30 % ratio of <I>E. coli</I> Top10F' (Prhl construct)/<I>E. coli</I>(Plas construct) and samples were taken every hour. In order to determine the ratios of the strains during the cultivation, the samples were spread out on agar plates and incubated at 37°C. The cultivation was stopped after 25 h. </P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, August 20, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kerstin</B>
The final Plas construct in <I>E. coli</I> JM109 was prepared for sequencing. Additionally we inoculated cultures of <I>E. coli</I> Top10F' carrying J23100-B0032-aeBlue, J23100-B0032-eforRed and the final Plas and Prhl constructs, respectively, with and without the corresponding inducer in order to gain material for fluorescence microscopy experiments the next day.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, August 21, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kerstin</B>
Glycerol stocks of the J23100-0032-aeBlue construct in <I>E. coli</I> Top10F' were made. Today we also visited the DSMZ to take fluorescence microscopy pictures of bacteria bearing the J23100-0032-aeBlue, J23100-0032-eforRed and the final Plas and Prhl) constructs. Unfortunately due to problems with the autofocus of the mircroscope the obtained pictures were unusable.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, August 22, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Laura, Kevin, Jan, Roman, Judith</B>
We made glycerol stocks of the final Prhl (<I>E. coli</I> Top10F'::K1073035) and Plas (<I>E. coli</I> JM109::K1073034) constructs from agar plates of the reactor cultivation. 
Furthermore 10 mM stock solutions of the homoserine lactones (autoinducers) of the rhl and las quorum sensing system (1.7 mg N-butyryl  homoserine lactone + 1 ml DMSO and 2.8 mg N-3-oxododecanoyl  homoserine lactone +  933 µl DMSO, respectively) were prepared.
Additionally we repeated the continuous bioreactor cultivation we conducted Monday.</P></DIV><DIV id="Week15" class="menuSection"><H2><A href="#Week15">Week 15: August 25 - August 31, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
Preparation for exams kept us really busy so just a few experiments were done this week. We unsuccessfully tried to find the source of the mysterious white colonies, which we observed in samples taken from previous continuous cultivation experiments.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, August 27, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin</B> Today we checked one pink, one blue and six white colonies from a spread out sample that we took during our previous continuous cultivation for successful integration of our cloned vectors. This was primarily done to reveal the unknown source of white cultures on our agar plates. The blue colony and three of the white colonies were positive for integration of the final Plas construct. These colonies later turned blue. For some reason, all colonies showed integration of final Prhl construct, including the blue control.
Furthermore, we inoculated liquid cultures of final Plas and final Prhl constructs in <I>E. coli</I> JM109 and <I>E. coli</I> Top10 F' cells and also the autoinducer detecting strains <I>E. coli</I> JM109 pSB1075 and <I>E. coli</I> JM109 pSB406 which are used to detect the production ofautoinducers by our constructs.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, August 28, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Kevin</B>On Wednesday, we tried to resolve the puzzling results from the colony PCR that we performed on Tuesday, this time with different primer/annealing temperature/DNA polymerase combinations but with the same colonies. However, PCRs didn’t clarify the previous results. Using GoTaq polymerase, the red and the blue colony were positive for aeBlue amplification, while white colonies were negative. PCR with Q5 polymerase indicated surprising integration patterns. Unfortunately, extraction of these bands did not yield enough DNA for sequencing.</P></DIV><DIV id="Week16" class="menuSection"><H2><A href="#Week16">Week 16: September 1 - September 7, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">Since we were all busy with our exams the lab stayed empty this week.</P></DIV><DIV id="Week17" class="menuSection"><H2><A href="#Week17">Week 17: September 8 - September 14, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">We did some unseccessful growth curve experiments, but gain a lot of new infomation about our final constructs. The experiments showed again that bacteria carrying the final constructs can grow on a low level on medium containing ampicillin without inducing the ampicillin resistance by respective HSLs. But we figured out how to deal with this problem: We have to add the beta-lactamase inhibitor clavulanic acid to the culture broth to inhibit the antibiotic resistance just enough to allow for bacteria with induced resistance to grow, but prevents growth of non-induced cultures. Thus the next step for us is to determine the right beta-lactamase inhibitor concentration for our experiments. </P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Tuesday, September 10, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan, Anna, Melanie</B>
We prepared pre-cultures of <I>E. coli</I> JM109 carrying the final Plas construct for the growth curve experiments tomorrow. 2xYT medium containing chloramphenicol was inoculated with cells directely from glycerol stock and incubated at 37°C and 250 rpm over night.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, September 11, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan, Anna, Melanie</B>
We conducted a cultivation experiment with the <I>E. coli</I> JM109 bearing the final Plas construct to test the growth in presence and absence of synthetic N-3-oxododecanoyl-HSL autoinducer added to the medium. Cultivation was carried out in four 500 mL non-baffled flasks with 75 mL 2xYT medium with Ampicillin. In two flasks N-3-oxododecanoyl-HSL was added. Ideally  the flask without N-3-oxododecanoyl-HSL would not show any growth in the beginning until the antibiotic has been depleted. The N-3-oxododecanoyl-HSL in the other flask should induce the quorum sensing controlled ampicillin resistance. Unfortunately all cultures were growing equally fast from the start so the experiment was aborted.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, September 12, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan, Anna, Melanie</B>
We repeated the growth curve experiment with our <I>E. coli</I> JM109 bearing final Plas to see if the unexpected growth was caused by a resistant contamination. Cultivation was again carried out in four 500 mL non-baffled flasks with 75 mL 2xYT medium with ampicillin. Again, in two out of the four flasks N-3-oxododecanoyl-HSL was added.  The result was the same as yesterday, all culture showed normal growth. We needed a new strategy for future experiments.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, September 13, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan, Anna, Melanie</B>
We figured that maybe our ampicillin resistance was too potent because even the basal expression allowed for normal growth in medium with antibiotic.
We prepared 5 mL liquid cultures of <I>E. coli</I> JM109 bearing final Plas and added varying amounts of a beta-lactamase inhibitor (clavulanic acid) to suppress the background expression. Tubes with 1 µg/µL to 100 µg/µL clavulanic acid showed no growth while tubes with 0.001 µg/µL to 0.1 µg/µL clavulanic acid showed normal growth.</P></DIV><DIV id="Week18" class="menuSection"><H2><A href="#Week18">Week 18: September 15 - September 21, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
This week we were happy to find a workaround for a big troublemaker: the background expression of beta-lactamase in non-induced state. We determined the critical concentration of beta-lactamase inhibitor clavulanic acid to be around 1 µg/ml. Furthermore, we directly applied it to experiments which previously were problematic because of the resistance to ampicillin under non-induced conditions.
Furthermore, we added our constructs to the iGEM Registry. Therefore our final contructs are from now on referred to as <A href="http://parts.igem.org/Part:BBa_K1073034">K1073034</A> and <A href="http://parts.igem.org/Part:BBa_K1073035">K1073035</A> for final Plas construct and final Prhl construct respectively.
</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Monday, September 16, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan</B>This week started with a quite important experiment: Since we experienced background expression of beta-lactamase, our transformed bacteria were resistant on a low level to ampicillin, although cultivated under non-inducing conditions.  We prepared 5 mL liquid cultures of <I>E. coli</I> JM109::K1073034 and added varying amounts of a beta-lactamase inhibitor (clavulanic acid) to suppress the background expression. Tubes with 1 µg/µL to 100 µg/µL clavulanic acid showed no growth while tubes with 0.001 µg/µL to 0.1 µg/µL clavulanic acid showed normal growth.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, September 18, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan</B>
On Wednesday, we repeated the experiment we did on Monday with a different scale of concentration steps to more precisely determine the best concentration of clavulanic acid for future experiments. Unfortunately, all samples showed growth so no further distinction could be made.
Hence, we went ahead with the 1 µg/ml of clavulanic acid we determined on Monday to remove background expression of beta-lactamase. We applied this information to a certain growth behavior of <I>E. coli</I> JM109 bacteria transformed with our final construct (K1073034). Therefore, we cultivated this strain in 2x YT media with ampicillin and clavulanic acid. Out of four cultures, only two contained the inducer necessary for successful induction of ampicillin resistance.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, September 19, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan</B>Liquid cultures of 2x YT media with ampicillin and 1 µg/ml clavulanic acid were inoculated at different ratios (based on OD<SUB>520</SUB>) of <I>E. coli</I> JM109::K1073034 and <I>E. coli</I> Top10F'::K1073035. Each culture was additionally performed with chloramphenicol instead of ampicillin as unregulated control. Unfortunately the concentration of clavulanic acid was too high and no growth was observed.</P></DIV><DIV id="Week19" class="menuSection"><H2><A href="#Week19">Week 19: September 22 - September 28, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">This week started with the last continuous cultivation, which in contrast to earlier ones was performed with added beta-lactamase inhibitor (clavulanic acid). On Wednesday, we repeated last weeks experiment to investigate the ratio of bacterial growth if both strains (<I>E. coli</I> JM109::K1073034 and <I>E. coli</I> Top10F'::K1073035) are cultivated together under different conditions and with varying inoculation ratios. This was a really tough last week, no wounder that people going crazy at the lab.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Thursday, September 22, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan</B>
Overnight cultures of <I>E. coli</I> JM109::K1073035 and <I>E. coli</I> Top10F'::K1073035 were inoculated in 2xYT for the continuous cultivation on Monday. Furthermore, we set up the reactors and made sure we are ready to start early next day.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Wednesday, September 23, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan, Anna, Melanie, Kevin, Roman, Judith</B>
This time we inoculated the reactors with a 70/30 ratio of <I>E. coli</I> JM109::K1073034/<I>E. coli</I> Top10F'::K1073035 and cultivated for 22h under constant observation to make sure we didn’t miss potentially interesting growth behavior. Samples were taken every hour. After plating a diluted sample on agar plates with chloramphenicol, OD<SUB>520</SUB> was measured for monitoring the growth behavior. In comparison to earlier reactor cultivations, this time we added 5 µg/L clavulanic acid to the cultivation media (2xYT + either ampicillin in first reactor or chloramphenicol in second reactor) to suppress the background expression of beta-lactamase.</P><P style="font-size:14px; font-weight:bold; text-decoration:none; border:none; color:#be1e3c; margin-left:5px; margin-right:5px; margin-top:0px; margin-bottom:0px">Friday, September 27, 2013</P><P style="margin-left:5px; margin-right:5px; margin-bottom:0px; margin-top:0px"><B>Investigators: Jan</B>
We conducted another ratio test on Wednesday. Therefore 2xYT media with ampicillin and 0.01 µg/mL were inoculated with different ratios of <I>E. coli</I> Top10F’::K1073035  and <I>E. coli</I> JM109::K1073034 and cultivated for 14 hours. Additionally a control on chloramphenicol was cultivated. Unfortunately this time the clavulanic acid concentration was too low and the cells grew as if cultivated without clavulanic acid. </P></DIV><DIV id="Week20" class="menuSection"><H2><A href="#Week20">Week 20: September 29 - October 4, 2013</A></H2><P style=" margin-left:5px; margin-right:5px;">
This week we spent time evaluating our last continuous cultivation of mixed <I>E. coli</I> JM109 and Top10F' containing our final constructs. Other than that we prepared our data and ourselves for the upcoming regional jamboree.</P></DIV></DIV></DIV><DIV id="sponsors"><H1>Our sponsors</H1></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Braunschweig/Notebook">http://2013.igem.org/Team:Braunschweig/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Braunschweig/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Braunschweig/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Braunschweig/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Braunschweig/Notebook&amp;oldid=336973" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>