<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Paris_Saclay_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Paris Saclay/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container header"><NAV id="menups"><UL class="nav"><LI class="{{{team}}}"><A href="https://2015.igem.org/Team:Paris_Saclay/Team">Team</A></LI><LI class="active"><A href="https://2015.igem.org/Team:Paris_Saclay/Description">Project</A></LI><LI class="{{{labwork}}}"><A href="https://2015.igem.org/Team:Paris_Saclay/Notebook">Lab work</A></LI><LI class="{{{interlab}}}"><A href="https://2015.igem.org/Team:Paris_Saclay/Measurement">Interlab Study</A></LI><LI class="{{{practices}}}"><A href="https://2015.igem.org/Team:Paris_Saclay/Practices">Human Practices</A></LI><LI class="{{{achievements}}}"><A href="https://2015.igem.org/Team:Paris_Saclay/Achievements">Achievements</A></LI></UL></NAV></DIV><DIV class="container submenu"><NAV id="submenups"><UL><LI><A data-target="#overview" href="https://2015.igem.org/Team:Paris_Saclay/Description#overview">Overview</A></LI><LI><A data-target="#survey" href="https://2015.igem.org/Team:Paris_Saclay/Description#survey">iGEM Survey</A></LI><LI><A data-target="#contamination" href="https://2015.igem.org/Team:Paris_Saclay/Description#contamination">Environment Contamination</A></LI><LI><A data-target="#thermal" href="https://2015.igem.org/Team:Paris_Saclay/Description#thermal">Thermal control</A></LI><LI><A data-target="#physical" href="https://2015.igem.org/Team:Paris_Saclay/Description#physical">Physical containment</A></LI><LI><A data-target="#perspectives" href="https://2015.igem.org/Team:Paris_Saclay/Description#perspectives">Perspectives</A></LI></UL></NAV></DIV><DIV class="container content"><H1>Project</H1><H2>Overview</H2><ARTICLE class="column"><P>Very often, the risks related to the safety of a genetically modified organism (GEO), except for a possible toxicity or pathogenicity of the organism, correspond to the spread of GEO. Most laws, &quot;guidelines&quot; and associations related to the Biosafety agree in trying to prevent the spread of any living modified organism. They suggest a <B>precautionary approach</B> whose main purpose is the protection of the environment and human health. This is clearly stipulated in the <A href="https://bch.cbd.int/protocol" target="_blank">Cartagena protocol on Biosafety</A> (international Convention on Biological Diversity): &quot;<I>The Protocol seeks to protect biological diversity from the potential Risks Posed by living modified organisms (LMOs) resulting and from modern biotechnology</I>&quot;. Synthetic biology fits especially into the definition of &quot;<B>modern biotechnology</B>&quot;. Indeed, biological machines designed in the iGEM competition, and more generally  in synthetic biology, do not exist in nature and can be <B>randomly introduced</B> into existing natural biomass through <B>undesired cross interactions</B>.</P><P>This is why the scientific community is unanimous in saying that the &quot;<B>biosafety mechanisms</B>&quot; are essential and will ensure the <B>neutralization</B> of the living device created outside the laboratory conditions under which it was generated. Thus, any alleged dissemination would be impossible.</P><P>These mechanisms can be of several types, the most famous examples are the suicide genes (<A href="https://2012.igem.org/Team:Paris_Bettencourt" target="_blank">encoding RNAse, DNase</A>) and/or the establishment of a nutrient dependency, where the selected nutrient does not exist in the external environment.</P><P>We reviewed previous iGEM projects over the last decade and found that despite the <B>crucial aspect</B> of <B>biosafety in synthetic biology</B> and iGEM, </P></ARTICLE><ARTICLE class="column right"><P>especially for projects registered in &quot;Environment&quot; and &quot;Health / Medicine&quot; track, often <B>no biosafety mechanism is designed, or even considered</B>. This may be due to a lack of time or resources on the part of teams preferring to focus on developing their basic idea by putting safety into the background. This reason is understandable given the complexity of the <B>challenge of Biosafety</B>.</P><H4>Our solution</H4><P>In this context, the 2015 iGEM Paris Saclay team decided to focus its project entirely on <B>Biosafety</B>. Our goals were multiple. First, we wanted to provide the iGEM community with a system able to prevent, or at least <B>significantly reduce</B>, the main risk: the <B>spread of GEO</B>. This system would include <B>biological containment</B> with parts that can be integrated in the genome of any <I>Escherichia coli</I> chassis and called <B>SafetE.coli</B>. We hoped this system could be used by all future iGEM teams, which would be a great time-saver and would give an easy access to requirements in terms of safety to all iGEMers. In addition to this system, our project also includes <B>physical containment</B> with a brand new encapsulation device based on silica beads also intended to prevent the escape of Genetically Engineered Coli. Finally, as our survey showed us that in 2015 as for the previous years, biosafety did not appear as a high priority, we thought we could also help the community by providing a <B>biosafety guide</B> for future iGEM teams. The idea was to help iGEMers to think about biosafety issues their project may raise early in the conception of their projects. By this way, parts would be designed and the chassis chosen with biosafety in mind.</P></ARTICLE><H2>iGEM Survey</H2>
We wanted to design a <B>chassis</B> that could be used <B>as broadly as possible</B> in iGEM, so we asked in our survey to iGEM teams which organism they chose as chassis. Among the 37 teams (representing 12% of all teams) that answered our <A href="https://docs.google.com/forms/d/1VSC9fclmluGj695NOOeS8QLasHjYptxQQtQYiTynio8/viewform?c=0&amp;w=1" target="_blank">survey</A>, <B>29</B> worked with, at least, <B><I>E. coli</I> as chassis</B> (5 teams worked with <I>E. coli</I> and with one or more other chassis). Eight teams worked with other chassis (<I>S. cerevisiae</I>, HEK293…).<CENTER><I>Figure 1: Results of the survey</I></CENTER><P><B>The wide use of this bacterium encouraged us to consider <I>E. coli</I>  to build our safe chassis</B>. <I>E. coli</I> is often chosen because of it is easy to handle, fast growing and the high level of knowledge is available for this organism.

In our survey, we also asked teams whether they had considered the possible effects of the release of their organism in the environment. We had a wide range of responses, from a simple &quot;No&quot; to &quot;we designed a kill-switch mechanism&quot;. We were surprised to note that <B>8 of the 29</B> teams who worked with <I>E. coli</I> thought that strains commonly used in the laboratory <B>could not survive in external environment</B>. This led us to design an experiment to test this claim and to determine whether <I>E. coli</I> could survive or not outside of laboratory conditions.

Some other teams did not seem to worry about the accidental release of their organisms in the environment (&quot;we hope that there is no accidental spill&quot;, &quot;we do not anticipate this happening&quot; or &quot;this organism is intended to be used in a lab or clinical setting&quot;).A few envisaged a physical containment of their organisms (&quot;we would keep our organisms contained in a bioreactor&quot;, &quot;we made some alginate beads&quot;).

We had a few teams that gave detailed answers to this question, showing that they indeed anticipated this potential issue in the design of their project. However on the whole, we had the feeling that in many cases, biosafety was either not considered or taken into account only after the design and maybe the execution of the project. This led us to undertake the <B>writing of a guide</B> for the use of future iGEM teams that will help them to take into account potential biosafety issues at the beginning of the design of their project.
</P><H2>Environment Contamination Evaluation</H2><P>The main goal of our project is to <B>prevent accidental dissemination in the environment of GEO</B>, and more specifically, of Escherichia coli, the most widely used chassis in iGEM. But before elaborating sophisticated physical and biological containment systems to prevent dissemination, we thought it would be interesting to study the <I>E. coli</I> survival in different natural environments.</P><H3>Protocols</H3><ARTICLE class="column"><H4>Environments</H4><P>Our experiment aims at simulating the accidental contamination of various environments with an <I>E. coli</I> culture grown in LB medium. For this purpose, we chose to test different environments:</P><UL><LI>Soil, collected around the lab (GPS coordinates: 48.703073, 2.172946)</LI><LI>Freshwater, collected on the University campus (GPS coordinates: 48.702986, 2.168050)</LI><LI>Sea water, which came from the Atlantic Ocean (GPS coordinates: 47.227968, -2.176851)</LI><LI>Filtered sea water, which came from the Atlantic Ocean</LI><LI>Filtred freshwater</LI><LI>Isotonic water</LI><LI>Sterile water</LI><LI>LB</LI></UL><P>We chose <B>soil, freshwater and seawater to test <I>E. coli</I> survival</B> because we thought that they are most common environments that can be found in nature. As a positive control, we chose to measure the growth of <I>E. coli</I> in LB medium in same experimental conditions.</P><P> For the CFU count, we diluted our samples in isotonic water before plating them. To make sure that <I>E. coli</I> cells will not die in this medium because of a potential osmotic shock,  we chose to examine the survival of our strains in isotonic water(MgSO4 10^-2 M). To determinate whether the presence of other microorganisms in the natural water environments could have an influence on the survival of our strains, we chose to also examine samples of  filtered freshwater or seawater.  .</P></ARTICLE><ARTICLE class="column right"><H4>Strains</H4><P>We next needed to choose the <I>E. coli</I> strain that will be used to contaminate the chosen environments. We first thought about using a strain of <I>Escherichia coli</I> carrying a plasmid expressing a fluorescent protein. This strain would have allowed us to evaluate easily the number of living <I>E. coli</I> bacteria in our samples. However, using such strain had also several drawbacks.
<UL><LI>First of all, we must be able to differentiate our fluorescent <I>E. coli</I> from other auto-fluorescent microorganisms or molecules in our samples. Even if we compare our samples to negative controls, our experiment might not be very precise.</LI><LI>As no selective pressure would be applied to keep the plasmid, some bacteria may lose it during the experiment.</LI></UL></P><P>To address these problems, we chose to use <B>three <I>E. coli</I> strains</B>, each with a different <B>antibiotic resistance gene</B> integrated in its genome. These three strains are mutants from the <I>Escherichia coli</I> strain <B>MG1655</B>:</P><UL><LI>PhB1320 = MG1655 clpP::Cm – resistant to Chloramphenicol (Cm)</LI><LI>PhB1693 = MG1655 Z1(lacR tetR SpR) (Spectino)</LI><LI>PhB1696 = MG1655 zad220::Tn10 – resistant to Tetracyclin (Tetra)</LI></UL><P>We used three different strains as inserting a resistance gene in the genome of a strain may impact the fitness of this strain in the environment. Repeating the experiment with three different strains will allow us to compare the behavior of the three strains in a given environment and identify a possible influence of the gene inserted on the strain survival.</P></ARTICLE><ARTICLE id="column"><H4>Medium</H4><P>To count our <I>E. coli</I> strains, we chose to use the <B>Mac-Conkey medium</B>. This medium is <B>very selective for Gram negative enteric bacilli, so <I>Escherichia coli</I></B>. It contains bile salt and crystal violet dye to inhibit Gram positive strains, and neutral red dye which turns pink all colonies able to fermentate the carbon source (for the first test it was lactose, and after sorbitol).</P><P>Thus, by adding the appropriate antibiotic to this medium, we thought we would be able to only select our strains, although we needed to check this hypothesis.</P><H4>Preliminary Study</H4><ARTICLE class="column"><P>Our experiment relies on the hypothesis that we will be able to <B>detect selectively our <I>E. coli</I> strain</B> when plating a sample taken from our tested environments on MacConkey plates supplemented with the appropriate antibiotic. We therefore needed to test this hypothesis and verify that the <B>natural microorganisms found in our environment samples would grow or not on our selective plates</B>. We also decided to grow our samples on LB medium as this medium is very rich and allows the growth of many microorganisms. We wanted to see if there was a difference with the MacConkey (MCK) medium, and if we could see the growth of the natural &quot;inhabitants&quot; of our samples.</P><P><B>We chose to analyze the soil, seawater and freshwater samples</B>. We prepared 5 dilutions (10-2 to 10-6) of a sample of our natural environment and plated 100 µl on the following media:</P><UL><LI>LB</LI><LI>LB + Spectinomycin (Spectino : 100ng/μL)</LI><LI>LB + Tetracycline (Tetra : 10ng/μL)</LI><LI>LB + Chloramphenicol (Cm : 20ng/μL)</LI><LI>MCK</LI><LI>MCK + Spectinomycin  (Spectino : 100ng/μL)</LI><LI>MCK + Tetracyclin (Tetra : 10ng/μL)</LI><LI>MCK + Chloramphenicol (Cm : 20ng/μL)</LI></UL><P>We next incubated the plates overnight at 37°C and observed them the following day.</P><P>For the soil experiment <A href="#preliminary-study-A">Figure 2.1</A>, we observed the <B>growth of microorganisms on LB medium for the 10-2 dilution</B>. These micro-organisms, however, appear to be <B>sensitive</B> to three <B>antibiotics</B> tested (chloramphenicol, spectinomycin and tetracycline). They are also <B>unable to grow on the MacConkey medium</B>. No growth of micro-organisms was observed when samples of freshwater <A href="#preliminary-study-B">Figure 2.2</A> or sea water <A href="#preliminary-study-C">Figure 2.3</A> were plated on either LB or MacConkey media.  This may have been expected for the seawater medium, as micro-organisms from this environment may require high salt media for growth.</P></ARTICLE><ARTICLE class="column right"><CENTER><I>protocol pre-test survival of E. coli</I></CENTER></ARTICLE><P>On the whole, we did <B>not observe growth of “natural” microorganisms on the MacConkey + antibiotics plates</B>, demonstrating that in the conditions tested, the only micro-organisms that we should observed are <B>our specific lab bacteria strains</B>.</P><CENTER><I><B>Figure 2.1</B> - Preliminary study for soil environment - Test of soil natural microorganisms growth on LB or MCK with or without Chloramphenicol, Spectinomycin and Tetracycline. Soil samples diluted 10<SUP>-2</SUP></I></CENTER><CENTER><I><B>Figure 2.2</B> - Preliminary study for freshwater environment - Test of freshwater natural micro-organisms growth on LB or MCK with or without Chloramphenicol, Spectinomycin and Tetracycline. Freshwater samples diluted 10<SUP>-1</SUP></I></CENTER><CENTER><I><B>Figure 2.3</B> - Preliminary study for seawater environment - Test of seawater natural micro-organisms growth on LB or MCK with or without Chloramphenicol, Spectinomycin and Tetracycline. Seawater samples diluted 10<SUP>-1</SUP></I></CENTER><H4>Study</H4><ARTICLE class="column"><P>To simulate an environmental contamination by GEOs, we introduced a <B>bacterial culture grown in LB without any antibiotic</B> in our environment samples. Indeed, in case of an accidental dissemination, what could happen to our strain if we spill a bacterial culture into the wild? We chose to use the same protocol for each environment, except an additional step of water dilution for the soil sample.</P>
Protocol experiment : survival of <I>E. coli</I>:<OL><LI>We prepared a 15mL culture of our three different strains (<I>E. coli</I> 1320, 1693 and 1696), in LB medium, without antibiotic the day before starting the experiment.</LI><LI>The first day (D0), we measured the optical density at 600 nm (OD600) to evaluate “concentration” in bacteria , and we plated on MCK with the appropriate antibiotic a sample of each culture to count the  CFU.</LI><LI>We put then 1,5mL of each of the three cultures in each of the eight experimental environments (15mL for liquid environment, 15g for soil), and shake it to make a homogeneous mix.  Negative controls (-) are environmental samples without culture contamination. All experimental samples were left on the bench, at room temperature, in a 50mL closed falcon tube.</LI><LI>Each day, we took a 1mL sample of each contaminated liquid experimental environment and prepared dilutions of this sample from 10<SUP>-1</SUP> to 10<SUP>-8</SUP> in isotonic water. For the soil environment, we took 1g of soil, which we diluted in 5mL of sterile water, and after one step of shaking and of 15 min decantation, we pipetted 50 µL of the supernatant and diluted it in 950 µL of isotonic water. This allowed us to obtain the 10<SUP>-2</SUP> dilution. We prepared then the 10<SUP>-3</SUP> to 10<SUP>-9</SUP> dilutions as described for the liquid environments.</LI><LI>We spotted 5 µL of each dilution on the corresponding MCK plate + antibiotic (no antibiotic for the control).</LI><LI>We incubated plates overnight at 37°C.</LI><LI>The next day, we counted the CFU to estimate our strain survival.</LI></OL></ARTICLE><ARTICLE class="column right">protocol experiment : survival of <I>E. coli</I></ARTICLE><H3>Results</H3><P>It took us time to develop a proper protocol to follow the survival of our strains in the various environments. It was a trial and error process and we had to adjust the protocol after each trial.</P><H4>Preliminary - experiment</H4><ARTICLE class="column"><P>First of all, we made a small-scale preliminary experiment<B> during three days to determine whether our strains were able to survive</B> in our three major environments: soil, freshwater and saltwater, and to determine what would be the <B>optimal concentration of bacteria</B> in our contaminating culture for our experiment. We used 5g of soil, and 5mL of freshwater or salt water. We contaminated them with 1mL of overnight bacterial culture not diluted or diluted 10 or 100 times. For these preliminary experiments, the samples to be plated were prepared as described above, but rather than spotting 5 µl on plates, <B>we plated 100 µl of the dilutions on the adequate MCK plates</B> and incubated the plates at 37°C overnight. Unfortunately, because of some problems with the medium, the results for the experiments with freshwater and saltwater could not be interpreted. The results for the soil experiment are presented in the table 1</P><P>On the whole, we observed that after three days of contaminations only, there were no or very few surviving <I>E. coli</I> bacteria in the soil when we diluted our contaminated <I>E. coli</I> culture 10 or 100 times. Moreover, the same decrease trend is observed for each of the three <I>E. coli</I> strains tested, suggesting that the integration of each of the three resistance genes in the genome of the <I>E. coli</I> MG1655 strain is not introducing a bias in our experiment. From this experiment, we concluded that it was more interesting for us to use an <B>undiluted bacterial culture to contaminate our environment</B> in our following experiments.</P></ARTICLE><ARTICLE class="column right">table 1
<TABLE border="0" cellspacing="0" cellpadding="0" class="wikitable"><TBODY><TR><TD width="80"><STRONG>D0</STRONG></TD><TD width="80"><STRONG>D1</STRONG></TD><TD width="80"><STRONG>D2</STRONG></TD><TD width="80"><STRONG>D3</STRONG></TD></TR><TR><TD><STRONG>CM 1</STRONG></TD><TD>more than 1000</TD><TD>186</TD><TD>111</TD><TD>141</TD></TR><TR><TD><STRONG>CM0,1</STRONG></TD><TD>1744</TD><TD>35</TD><TD>13</TD><TD>2</TD></TR><TR><TD><STRONG>CM 0,01</STRONG></TD><TD>272</TD><TD>1</TD><TD>0</TD><TD>0</TD></TR><TR><TD><STRONG>Spe 1</STRONG></TD><TD>more than 1000</TD><TD>490</TD><TD>424</TD><TD>276</TD></TR><TR><TD><STRONG>Spe 0,1</STRONG></TD><TD>more than 1000</TD><TD>67</TD><TD>75</TD><TD>88</TD></TR><TR><TD><STRONG>Spe 0,01</STRONG></TD><TD width="60" valign="top">441</TD><TD>0</TD><TD>1</TD><TD>2</TD></TR><TR><TD><STRONG>Tet 1</STRONG></TD><TD width="60" valign="top">more than 1000</TD><TD>1308</TD><TD>524</TD><TD>416</TD></TR><TR><TD><STRONG>Tet 0,1</STRONG></TD><TD width="60" valign="top">more than 1000</TD><TD>196</TD><TD>35</TD><TD>62</TD></TR><TR><TD><STRONG>Tet 0,01</STRONG></TD><TD width="60" valign="top">414</TD><TD>0</TD><TD>0</TD><TD>0</TD></TR><TR><TD><STRONG>control</STRONG></TD><TD>0</TD><TD>1</TD><TD>3</TD><TD>4</TD></TR></TBODY></TABLE></ARTICLE><H4>First Experiment</H4><P>We used the same protocol than before, but with 60g of soil and 5mL of water. We made this experiment in triplicate.</P><H5>Soil</H5><P>We started a new experiment with more soil (60g) to analyze the contamination evolution during a longer period of time. We planned to do it for a few weeks but after three days, <B>we observed contaminations on our negative control</B>. After multiple trials, we concluded that this was due to the MCK medium used. Thus we changed the MCK stock and did tests from which we concluded that adding sorbitol to the MCK was more selective than adding lactose. Consequently, we decided to start another time the experiment for all the different contaminated media using <B>Sorbitol-MCK rather than lactose-MCK</B>.</P><H5>Seawater and Freshwater</H5><P>The first time we put our strains in 5mL of water but we encountered two main problems. The first one was that it was very difficult to spread the 100µL sample on plates. The samples seemed to slide on the medium. And the next day, we can only observe some “lines” of colonies. The second problem appeared on the second day: the medium, normally red, turned into a yellow color. This was probably due to a change of pH in the water sample, most likely due to the metabolism of some microorganisms when adding LB in the environment.</P><H4>Final experiment</H4><P>After those pre-tests, we finally had established a protocol that should allow us to carry out our experiment. We chose to test 15g (soil) or 15mL (seawater and freshwater) of our samples contaminated by 1,5mL of our bacterial cultures. We used the following protocol.</P><P>We performed CFU analyses each day by depositing 5µL spots of contaminated media on specific MCK plates. The results are presented bellow.</P><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Controls"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Controls</SPAN></A></LI><LI class="toclevel-1"><A href="#Experiment"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Experiment</SPAN></A></LI><LI class="toclevel-1"><A href="#Conclusions"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Conclusions</SPAN></A></LI><LI class="toclevel-1"><A href="#The_Temperature-Based_System_for_E._coli:_SafetE.coli"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">The Temperature-Based System for <I>E. coli</I>: SafetE.coli</SPAN></A></LI><LI class="toclevel-2"><A href="#Overview"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI></UL></DIV><H4><SPAN class="mw-headline" id="Controls">Controls</SPAN></H4><P><B>Figure 2.1</B> - Negative Control
</P><P>Due to the lack of time, <B>we were not able to conduct these experiments in triplicates</B> and for a longer period of time, which would be required to obtain solid results from which conclusions can be drawn. Nonetheless, these experiments allowed us to formulate some preliminary conclusions. First, in our negative control (corresponding to samples of uncontaminated environments plated on MCK plates without antibiotics), we observed a very limited number of bacteria in all our samples except for the freshwater sample. This number also appeared to be stable in time, suggesting that the population of the micro-organisms that grew on our plates remained stable during the time of our experiment.The important bacterial growth observed for the non-filtrated freshwater is surprising and inconsistent with the results obtained for the other samples. It will be necessary to repeat this experiment to check whether this is observed reproducibly.</P><P>Before interpreting the results of our contamination experiments, it is important to remember that we did not observed the growth of any micro-organisms from the various environments tested when antibiotics were added to the MacConkey medium.</P><CENTER><I><B>Figure 2.2</B> - Positive Control</I></CENTER><P>We used LB medium for our positive control. We can see that during the first few days, the <I>E. coli</I> population is growing, before stabilizing. It should be noticed that for Day 9, we have only one measure, because of manipulation errors (wrong dilutions) with the samples LB+ PhB1393 and LB+PhB1396.
</P><CENTER><I><B>Figure 2.3</B> - Isotonic water</I></CENTER><CENTER><I><B>Figure 2.4</B> - Sterile water</I></CENTER><P>Our controls of <I>E. coli</I> strains in isotonic water or sterile water shows that they survive perfectly well in these environments. 
</P><CENTER><I><B>Figure 2.5</B> - Filtered fresh water</I></CENTER><CENTER><I><B>Figure 2.6</B> - Filtered sea water</I></CENTER><P>When we filtered our water samples, we eliminated all the micro-organisms present in these environments. We observed that on the whole, the number of laboratory <I>E. coli</I> bacteria in these two environments remain stable for the length of the experiment.
</P><H4><SPAN class="mw-headline" id="Experiment">Experiment</SPAN></H4><P>All the environmental samples contained various organisms (as demonstrated by the pre-test), so these three experiments are close to the environmental conditions (as the temperature or light fluctuation).
</P><CENTER><I><B>Figure 2.7</B> - Soil</I></CENTER><P>We observed a large decrease of the number of living laboratory <I>E. coli</I> bacteria with time (100 to 1000 fold decrease between day 1 and day 9), for the three tested strains.
</P><CENTER><I><B>Figure 2.8</B> - Fresh water</I></CENTER><CENTER><I><B>Figure 2.9</B> - Sea water</I></CENTER><P>For the freshwater and seawater environments, we observed a small decrease or a stability of the population of living laboratory <I>E. coli</I> bacteria with time. The results for the seawater environment is surprising, as it is has often be described that <I>E. coli</I> populations rapidly decline in this environment, although in some experiments, a lag time before the decline of the population of  up to 20 days has been observed. (Vaccaro <I>et al.</I>, Am J Public Health Nations Health. 1950 Oct; 40(10): 1257–1266). 
</P><H4><SPAN class="mw-headline" id="Conclusions">Conclusions</SPAN></H4><P>The experiments we conducted showed that during at least 9 days, laboratory <I>E. coli</I> bacteria can survive in environments as varied as the soil, the water of a pond or ocean water. We were careful in designing our experiments to try to limit as much as possible the factors that could bias the results  of our study (three  <I>E. coli</I> strains tested)  and included all the control we could think of. We also noticed that in all the liquid conditions tested, a biofilm was visible from day 3. The fact that these biofilms appeared in conditions such as sterile water or filtered water led us to the conclusion that our <I>E. coli</I> strains can form biofilms, a property that is thought to be important for the survival in the environment (Vogeleer P <I>et al.</I>, Frontiers in Microbiology, vol5, 2014). 
We did not study here if DNA fragments could be exchanged between organisms, which could be a real issue, especially in case of a prolonged survival in the environment. Altogether, this experiment confirmed the necessity of elaborating physical and biological containments for GEO that could be released in the environment, either accidentally or for a specific application.</P><P>A special thank to Purificación López-García's team to explain the limit of our experiment and help in the process of making it. We also thanks the Philippe Bouloc's team (Signaling and Regulatory Networks in Bacteria) for the MCK and the help.
</P><H2><SPAN class="mw-headline" id="The_Temperature-Based_System_for_E._coli:_SafetE.coli">The Temperature-Based System for <I>E. coli</I>: SafetE.coli</SPAN></H2><H3><SPAN class="mw-headline" id="Overview">Overview</SPAN></H3><P>To protect the environment from a possible Genetically Engineered Bacteria (GEB) contamination, we created a two-level protection system. First a <B>biological containment</B> was built, which could be improved with the addition of a &lt;/html&gt;<A rel="nofollow" class="external text" href="https://2015.igem.org/Team:Paris_Saclay/Description#physical">physical containment system</A> level  depending applications. We choose to build a system which the <I>E. coli</I> viability was restricted to <B>a narrow range of temperatures</B>. Such system should then be combined with the physical containment system.</P><H3>Problematic</H3><P>Nowadays, a large number of thermosensitive systems are commonly used in the biology field. Two points appeared to be important when we started to design our device: first, it must be durably functional in time to avoid the spread of GEB. Secondly, the system must be flexible and adaptable to the users’ needs.</P><H3>Details</H3><H4>Why our device is safe and stable in time</H4><ARTICLE class="column"><P>In order to create a safe system, we choose to control the survival of bacteria by modulating the expression of several essential genes.</P><P>The targeted essential genes were chosen in accordance with the literature on the subject. They have to control a vital function in the cell, be well-characterized, have a constitutive expression, and not be redundant in the bacterial chromosome. Taking into account these various parameters, adenylate kinase (adk), alanyl-tRNA synthetase (alaS), DNA polymerase III subunit delta (holB), methionyl-tRNA synthetase (metG), phosphoglycerate kinase (pgk) and tyrosyl-tRNA synthetase (tyrS) should for instance be good candidates. The more of these target genes are controlled by our system, the less likely is the bacterium to live outside the lab.</P><P>The aim of our system is to control the expression of these chosen essential genes.</P></ARTICLE><ARTICLE class="column right"><P>To do this, a replacement of their natural promoter by homologous recombination  has to be made. The repressible Tet Promoter, which is controlled by the Tet Repressor, will replace the native gene promoter. The repressor gene with a specifically engineered promoter will also be inserted on the chromosome. For this project, the classical Tet system will be modified by placing theTet Repressor gene under the control of a thermosensible repressor, this repressor being downstream a thermoinducible promoter. Thereby the expression of the essential gene is indirectly determined by the outside temperature.</P><P>The thermal control part of the system (described below) could be implemented on a plasmid or onto an other localization of the <I>E. coli</I> chromosome. If this plasmid is lost by the bacterium, the Tet Repressor will constitutively inhibit the transcription of the downstream essential gene, and the bacterium will die. The same scenario will happen if one of the system elements undergone a mutation.</P></ARTICLE><CENTER><I><B>Figure 3.1</B> - Chromosomal essential genes control - <I>E. coli chromosome</I></I></CENTER><H4>How our system could be adapted to biologists needs</H4><P>In order to answer the problematic of system adaptability, we choose to use molecules that undergo structural changes depending on the temperature.</P><P>To achieve this goal, two molecules have been selected: an RNA thermosensor and a thermosensitive repressor. These two molecules define a high and a low temperature limitation. They can be introduced in a plasmid or introduced inside of the chromosome, under the control of a constitutive promoter.</P><CENTER><I><B>Figure 3.2</B> - Thermal control device - <I>Plasmid or chromosome</I></I></CENTER><ARTICLE class="column"><P>The first part of the system consists in a thermosensitive variant of transcriptional repressor cI from bacteriophage lambda.  cI repressor protein acts as a dimer to bind to promoters and repress downstream gene expression. For cI(Ts), a cI protein variant, the dimerization happen only if the temperature is under 42 degrees, since at high temperatures cI(Ts) is denatured and cannot dimerize. Consequently the cI(Ts) protein is not functional above 42 degrees. Other variants of the cI repressor can be created by inducing mutations near the dimerization site, in order to change the denaturation temperature.</P></ARTICLE><ARTICLE class="column right">Behind the thermosensitive cI variant, instead of its native RBS, we used an RNA thermometer which is based on a ROSE RNA thermometer retrieved from <I>Bradyrhizobium japonicum</I>. The temperature sensitive hairpin of this RNA thermometer that contains the Shine Dalgarno (SD) sequence is closed when the temperature is under 32°C, that structurally prevent the ribosome binding to the downstream RNA sequence, and so it cannot start to translate RNA. The translation can only be initiated when the temperature is around 32°C, when the hairpin is opened and so the SD sequence is available for the ribosome binding. The switch temperature of the thermometer RNA is easily tunable by mutations in order to stabilize or destabilize its Shine Dalgarno hairpin.</ARTICLE><H4>The global thermal system functioning</H4><P>Overall, our device consists in an inhibition of an inhibitor to allow the essential genes expression. This situation happens only if the environment temperature is comprised between 32 and 42 degrees because it is the only range of temperature that allows the production of a functional dimer of repressor cI. Indeed, from 32°C the thermometer RNA allows the ribosome to translate the RNA. And secondly, with a temperature below 42°C, the cI protein is well-folded and can form a homodimer. This cI pair then binds to the cI promoter, thus inhibiting the Tet Repressor and allowing thereby the expression of the essential genes under its control. Finally, at temperatures comprised between 32 and 42°C, our system allows the production of the proteins essential for the bacterium’s survival.</P><CENTER><I><B>Figure 3.3</B>- First scenario : Permissive temperature range </I></CENTER><P>On the contrary, if the temperature is under 32°C, the hairpin of the thermometer RNA is hybridized on itself and masks the SD sequence, inhibiting the translation of the cI protein. Thereby the Tet Repressor is constitutively produced and inhibits the essential genes expression, leading to the bacterium’s death.</P><CENTER><I><B>Figure 3.4</B>- Second scenario : Lower non-permissive temperature range</I></CENTER><P>If the temperature is over of 42°C, even if the thermometer RNA hairpin is linearized and allows the ribosome reading of the cI RNA sequence, the cI inhibitor cannot form homodimers and is not functional anymore. Thus the Tet Repressor is also produced and inhibits the essential protein production leading to the bacterium’s death.</P><CENTER><I>'''Figure 3.5''' - Third scenario : Higher non-permissive temperature range </I></CENTER><H3>Results</H3><P>In order to design a safety E. coli chassis, numerous test constructs have been made. Each of it is intended to check the functionality of one part of our final system. 
</P><CENTER><I>Figure 3.6 – Final system on E.coli chromosome</I></CENTER>
To functionally characterize our parts, a fluorescent reporter: a Cyan Fluorescent Protein with a ssRA tag has been introduced downstream, to permit a fast destruction of the reporter and so visualize the fluorescent switch rapidly. 
Finally 38 BB have been constructed, but due to a lack of time, only 2 biobricks have been properly characterized by fluorescence microscopy: the BBa_K1707026 which correspond to the ssRA tag test and the BBa_K1707028 which is a cI promoter test.
<CENTER><I>Figure 3.7 – System components test</I></CENTER>

First, to check the efficacy of the ssRA tag to destabilize the CFP, the intensity of fluorescence of two BB have been confronted: BBa_I13600 (Figure 3.9) and BBa_I13602 (Figure 3.10). Both are CFP under control of a Tet promoter, but only the BBa_I13602 has a ssRA tag. Through fluorescence microscopy we can see that the bacteria transformed with CFP (BBa_I13600) are more fluorescent than the bacteria transformed with CFP-ssRA (BBa_I13602). Our negative control, DH5α bacteria (Figure 3.8 ) don’t show significant fluorescence. In conclusion the ssRA tag seems to permit a destabilization of the CFP protein, reducing its lifetime. This experiments allowed the first characterization of the BBa_I13602.
<CENTER><I>Figure 3.8 – DH5α microscopy observation (Negative control)
A) Differential interference contrast
B) excitation with led 437 nm/17nm (CFP), 200 ms of exposure</I></CENTER><CENTER><I>Figure 3.9 – BBa_I13600 microscopy observation (Positive control)
A) Differential interference contrast
B) excitation with led 437 nm/17nm (CFP), 100 ms of exposure</I></CENTER><CENTER><I>Figure 3.10 – BBa_I13602 microscopy observation (Positive control)
A) Differential interference contrast
B) excitation with led 437 nm/17nm (CFP), 100 ms of exposure</I></CENTER>

In order to test the strength of two other promoters used in our system, a CFP-ssRA has been introduced downstream a constitutive promoter (BBa_K1707026 Figure 3.11) and a cI promoter (BBa_K1707028 Figure 3.12). Under fluorescence microcopy, the intensity of fluorescence visualized for these two new BB seems to be equivalent to the results obtained with the BBa_I13602. In conclusion, we observed a minor intensity of fluorescence for these two strong promoters with a ssRA tagged CFP, in comparison to a medium promoter (Ptet promoter) with classic CFP (see Figure 3.9).
<CENTER><I>Figure 3.11 – BBa_K1707026 characterization
A) Differential interference contrast
B) excitation with led 437 nm/17nm (CFP), 100 ms of exposure</I></CENTER><CENTER><I>Figure 3.12  – BBa_K1707028 characterization
A) Differential interference contrast
B) excitation with led 437 nm/17nm (CFP), 100 ms of exposure</I></CENTER><H3>Conclusion and perspectives</H3><P>To conclude, we designed a biosafety system based on thermal control. We could imagine tailoring this system for different temperature ranges to meet the user’s needs. The cI repressor could be mutated at different positions to try to modify the transition temperature, as could be the switch temperature of the RNA thermometer. In addition, our project is designed to provide the iGEM community with biobricks that could easily be integrated in the chromosome of any Escherichia coli chassis with a safety system on it. In principle, it could be adapted to other bacterial chassis. However, one biological containment system is likely to be insufficient, and to reduce the likelihood of an environmental contamination, it would be better to multiply the biological (auxotrophic selection, toxin/antitoxin system) or physical (such as the silica beads presented below) security systems. Nevertheless, an increasing number of bacterial modifications may seriously impact the bacterium behavior.   </P><H2>Physical containment</H2><ARTICLE class="column"><P>Considering the non-negligible risk of escape of a GEO, we wanted to add an extra layer of safety in our project.  Therefore, we devised a way to physically contain <I>E. coli</I> and other bacteria without hindering it to perform its main function. Indeed, the system we designed is a porous glass beam in which bacteria can grow, survive and carry out their function. The nutrients of the external medium  (sugars, amino acids, etc...) can penetrate inside the devise but the bacteria cannot escape from it.</P><P>Thus, the <B>SafetE.coli</B> chassis can be in a safer manner in contact with natural environments, which is an important aspect when the project competes in iGEM competition tracks such as <A href="https://2015.igem.org/Tracks/Environment" target="_blank">Environment</A> or Health and Medicine.</P></ARTICLE><ARTICLE class="column right"><P>Some teams include in their project a devise, more or less sophisticated, in which the GEO or their products will perform its function. Here we can cite the 2014 project of <A href="https://2014.igem.org/Team:NCTU_Formosa/project" target="_blank">NCTU Formosa</A> where we can found an example of such devise: <A href="https://2014.igem.org/Team:NCTU_Formosa/project#Device" target="_blank">The Pyramid Trap</A> containing PBAN (Pheromone Biosynthesis Activating Neuropeptide) produced by a Genetically Engineered Escherichia coli. However, other teams do not have the time or means to do as well.</P><P>Thus, the 2015 Paris Saclay team project offers to all iGEM teams a fast, efficient and inexpensive way to build an impenetrable shelter for bacteria.<SUP>1</SUP></P></ARTICLE><H4>The system principle</H4><P>The aim of the containment is to create a physical barrier which should prevent accidental spreading of bacterial culture. The idea is to make a containment where bacteria can grow but which does not allow them to be in contact with the environment. Bacteria will be contained inside silica monoliths. To enable cell growth, we found a protocol which describes how to make cavities inside silica monoliths possible.</P><P>First, cells are encapsulated in an alginate gel. The alginic acid polysaccharide is a linear copolymer of β-D-mannuronic and α-L-guluronic acid extracted from brown algae or bacteria.</P><P><CENTER><I>Alginic acid polysaccharide</I></CENTER></P><P>The bacterial culture is mixed with sodium alginate and the solution is dropped in a chloride calcium solution. At neutral pH, carboxylic acid functions are deprotonated so that the polymer bears a global negative charge, usually compensated by sodium ions. Addition of divalent cations such as Ca<SUP>2+</SUP> induces cross-linking of the polymer, and therefore gel formation.</P><P>Then, silica monoliths are created around the beads obtained by a sol-gel process which allows the formation of silica gels at room temperature from aqueous precursors (Brinker and Scherrer 1990). The reactions involved in this process are presented in the figure below. </P><P><CENTER><I>Polymerization reactions involved in the sol-gel process</I></CENTER></P><P>Finally, the gel is dissolved by using an acid which chelates Ca<SUP>2+</SUP> so that cells have a cavity where they can grow and which can be filled with a defined growth medium.</P><H4>Protocol</H4><P>The first one is performed by dropwise addition of a 1.5% (w/w) sodium alginate cells suspension in a 0.1 M CaCl<SUB>2</SUB> solution. After 10 min stirring, about 3 mm diameter beads are easily collected by filtration. The calcium alginate polymer prevents cell contact with synthesis precursors. The second step consists of silicate polymerization in the presence of commercial silica nanoparticles (Ludox HS40 from Aldrich), leading to a nanoporous monolithic structure. Monoliths are prepared at room temperature by mixing 2 volumes of 1.25 M sodium silicate with 1 volume of colloidal silica and 1 volume of succinic acid (5 wt %) into a recipient containing the alginate-cells bead. Once the sol-gel polymerization reaction is completed, the stiff monolith obtained is left in contact with 0.05% potassium citrate during three hours. To provide necessary nutrients to the immobilized cells, potassium citrate solution is further replaced by LB medium according to encapsulated cell strain requirements.<SUP>1,2</SUP></P><P><CENTER><I>Physical containment protocol</I></CENTER></P><H4>Results</H4><P>We obtained nice beads which are properly jellified and measure about 3 mm as expected.</P><P><CENTER><I>Alginate beads of bacterial culture</I></CENTER></P><P>The silica monolith was obtained only when we let it in contact with air.</P><P><CENTER><I>Alginate beads with few silicate at their surfaces containing bacterial culture</I></CENTER></P><H4>Conclusion and Perspectives</H4><P>We have succeeded in creating a physical containment for bacteria which is cheap, fast to build and should prevent unintended contacts between GEOs and the environment. Further experiments are recquired in order to test such system. It will be interesting to see if bacteria do not escape from the silica monolith and if they can survive inside it. <A href="https://2015.igem.org/Team:Aachen" target="_blank">The Aachen team</A> suggested us different devices to measure the growth rate. The first one can be used to determine biomass concentration using capacity measurements.<SUP>3</SUP> The second one measures growth rate using an optical detection system that is based on measuring the intensity of back-scattered light from bacterial cells suspended in liquid culture.<SUP>4</SUP>
A <A href="https://2015.igem.org/Team:Paris_Saclay/Modeling" target="_blank">modeling</A> of the physical containment has been made. 

</P><H4><I>References</I></H4>
(1) Mercedes Perullini, Mat´ıasJobba´gy, Galo J. A. A. Soler-Illia, and Sara A. Bilmes, <I>Chem. Mater</I>. <B>2005</B>, <I>17</I>, 3806-3808
(2) Mercedes Perullini, Frédéric Orias, Claude Durrieu, Matias Jobbagy, Sara A. Bilmes, <I>Biotechnology Reports</I>, <B>2014</B>, <I>4</I>, 147-150
(3) Noll, T., &amp; Biselli, M., <I>Journal of biotechnology</I>, <B>1998</B>, <I>63</I>, 187-198
(4) Toprak, E., Veres, A., Yildiz, S., Pedraza, J. M., Chait, R., Paulsson, J., &amp; Kishony, R., <I>Nature protocols</I>, <B>2013</B>, <I>8</I>, 555-567<H2>Perspectives</H2><P>Our project is entirely dedicated to improving biosafety in iGEM and synthetic biology projects. First, we chose to focus on designing biosafety systems for <I>E. coli</I> chassis according to the results of the survey that we engaged. As some iGEMers seemed to consider that releasing <I>E. coli</I> in the environment was not a problem in this <A href="https://2015.igem.org/Team:Paris_Saclay/Description#survey">survey</A>, as bacteria will die, we decided to investigate whether this was indeed the case. The <A href="https://2015.igem.org/Team:Paris_Saclay/Description#contamination">contamination experiments</A> we conducted showed that during at least 9 days, laboratory <I>E. coli</I> bacteria can survive in environments as varied as the soil, the water of a pond or ocean water. To complete this experiment, it could be interesting to work with other <I>E. coli</I> strain (K12, DH5α) and moreover than made triplicates for each strain, follow during a larger scale than 9 days.</P>
Unfortunately, this experiment confirmed the necessity of elaborating biological and physical containments for GEO that could be released in the environment, either accidentally or for a specific application.<P>We conceived a biosafety system based on <A href="https://2015.igem.org/Team:Paris_Saclay/Description#thermal">thermal control</A> adaptable to the user’s needs by using variants of the two key molecules of our system. However, one biological containment system is likely to be insufficient for some particular applications justifying the conception of a second containment level, which is a physical protection.</P><P>We have succeeded in creating a <A href="https://2015.igem.org/Team:Paris_Saclay/Description#physical">physical containment</A> for bacteria which is cheap, fast to build and should prevent unintended contacts between GEOs and the environment. Further experiments are required in order to test such system. The <A href="https://2015.igem.org/Team:Paris_Saclay/Modeling">modeling</A> of this aspect that we maid confirm the fact that this physical containment is not universal and that depending of the application different parameters have to be adjusted, as beads size for instance.</P><P>Finally, we wanted the future iGEM teams to consider biosafety as an integral part of their projects. We therefore wrote a <A href="https://static.igem.org/mediawiki/2015/0/05/Paris_Saclay-biosafety_guide_rouge_final.jpg">biosafety guide</A>, which aims at helping iGEM teams to take into account potential biosafety issues at the beginning of the design of their project, rather than after the conception of their system(s).</P></ARTICLE></DIV><DIV class="container footer"><SMALL class="contact"><B style="color:white;">Contact</B><B>e</B>igem.parissaclay <I>AT</I> gmail <I>DOT</I> com<B>t</B><A href="http://www.twitter.com/iGEMParisSaclay" target="_blank">@iGEMParisSaclay</A><B>a</B>Université Paris Sud<B>a</B>91405 Orsay, France</SMALL><SMALL class="envers copyright">Copyright © 2015 iGEM Paris Saclay. <STRIKE>All rights reserved</STRIKE> All inspirations encouraged.*This wiki was conceived during long nights of work and a great coffee experience in the beautiful Orsay, France.</SMALL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>