<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Amsterdam_Project_Synthetic_biology_Dependecies skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Amsterdam//Project/Synthetic biology/Dependecies</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Amsterdam 2015</TITLE></P><DIV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Main_Page">iGEM</A></DIV><DIV class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:Amsterdam">Home</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Overview">Project </A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Stability">Stable romance</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Eng_rom">Engineering Romance </A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Eng_rom/Photosyn_car">Photosynthetic Carbon production</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Eng_rom/Dependecies">Dependent <I>Synechocystis</I></A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Software">Algorithms</A></LI></UL><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Simula_rom">Simulating Romance </A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Modeling">Simulations</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Phy_param">Physiological parameters </A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Phy_param/Synechocysytis"><I>Synechocysytis</I></A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Phy_param/Ecoli"><I>E. coli</I></A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/emulsions">Evolving Romance</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Design">Testing Romance</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Drylab/Algorithms">Algorithms</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Parts">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Medal_fulfilment">Medal fulfilment</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/eng_carbon"> Engineering Carbon production</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/syn_dependence"> Engineering Dependencies</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/Modelling">Dry lab</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/emulsions">Emulsions</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/phys_ecoli"><I>E. coli</I> physiology</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/phys_syn"><I>Synechocystis</I> physiology</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Practices">Human Practices </A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Human_practices/RIVM">RIVM</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Human_practices/App_scenario">Application Scenarios</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Human_practices/techno_vignette">Techno-moral vignette</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><DIV id="contentContainer"><SECTION id="project_overview_header" class="wrapper style2"><HEADER class="major"><H2>Dependent <I>Synechocystis</I></H2><P>Some subtitle</P></HEADER></SECTION><SECTION id="Overview" class="wrapper style4"><HEADER class="major"><H2>Overview</H2></HEADER><DIV class="container"><DIV class="row"><DIV class="3u" style="width:20%;"><SECTION><H4><A href="#bg">Background</A></H4><P>Background into the rational behind this module</P></SECTION></DIV><DIV class="3u" style="width:20%;"><SECTION><H4><A href="#aim">Aim</A></H4><P>Engineer an auxotrophic Synechocystis.</P></SECTION></DIV><DIV class="3u" style="width:20%;"><SECTION><H4><A href="#methods">Methods</A></H4><P>How to create an auxotrophic Synechocystis</P></SECTION></DIV><DIV class="3u" style="width:20%;"><SECTION><H4><A href="#results">Results</A></H4><P>What was achieved</P></SECTION></DIV><DIV class="3u" style="width:20%;"><SECTION><H4><A href="#parts">Parts</A></H4><P>List of created parts.</P></SECTION></DIV></DIV></DIV></SECTION><A name="aim"><SECTION id="solution" class="wrapper style2"><DIV class="container"><DIV class="row"><DIV class="6u"><HEADER><H2>Background</H2></HEADER><P align="justify">One can construct a synthetic consortium between two species with a variety of methods. <I>Synechocystis</I> could simply be engineered to export a carbon source (such as glucose) for <I>E. coli</I> in a glucose-absent medium. Although such a commensal relationship would definitely under certain conditions be considered stable, <I>E. coli</I> alone is the benefactor - there is no drive for <I>Synechocystis</I> to produce this carbon source. Increased stability is possible by creating a reciprocal relationship between the two species. Therefore, how can Cyanobacteria benefit or depend on <I>E. coli</I>’s presence? Cyanobacteria manufacturers it’s own food and has grown for centuries requiring only sunlight and water, in addition to inorganic elements. In fact, <I>E. coli</I>’s presence actually harms Cyanobacteria by limiting access to precious sunlight. Herein lies this module’s focus: <B><I>Genetic engineering of an E. Coli dependent Synechocystis for the formation of stable synthetic consortia.</I></B></P><P align="justify">A basic requirement of consortia requires ‘communication’ between the species. Here we will attempt to construct a mutualistic relationship in which survival of one species is obligatory and beneficial for the survival of the other. The mode of communication will be the mutual production and exchange of essential metabolites. Since <I>Synechocystis</I> is an autotroph, one way to realize such a co-dependency is through the engineering of an auxotrophic bacterial strain. <I>E. coli</I>, could then in exchange produce the nutrient <I>Synechocystis</I> has been engineered to require. Ultimately, <I>Synechocystis</I> will produce a carbon source necessary for survival of <I>E. coli</I>, while <I>E. coli</I> produces a nutrient necessary for survival of <I>Synechocystis</I> - an engineered interdependence pathway.
                    </P><SECTION class="special"><FIGURE class="image"><FIGCAPTION> Figure 1. -  Synechocystis Auxotroph being supplied nutrients from nutrient producing E. coli</FIGCAPTION></FIGURE></SECTION></DIV><DIV class="6u"><HEADER><H2>Aim</H2></HEADER><P align="justify"> This module will aim to engineer an auxotrophic <I>Synechocystis</I>. It also aims to test this auxotroph in the presence of an <I>E. coli</I> engineered to produce the nutrient that <I>Synechocystis</I> has been engineered to be deficient in. This is visualized in Figure 1. One might ponder what is the added benefit of this feedback system in our consortia. Indeed, as stated before, stability can be achieved with just <I>Synechocystis</I> feeding <I>E. coli</I>. However, with this feedback loop, we gain numerous benefits over the aforementioned system.</P><OL style="font-family: 'Montserrat', sans-serif"><LI>We consider potential outbreak risks.  An <I>E.coli</I> dependent <I>Synechocystis</I> would not be able to survive outside the lab, thus the risk of environmental contamination would diminish.</LI><LI>An essential part of our project involves the creation of an emulsion based protocol to test potential consortia. If <I>Synechocystis</I> did not require <I>E. coli</I>, testing out the most effective and stable consortia in this manner would result in simply selecting against the presence of <I>E. coli</I>, as <I>Synechocystis</I> would do whatever it takes to increase it’s own growth rate. By having a feedback loop, we are therefore able to select consortia in which the members work well together.</LI><LI>Assuming that <I>E. coli</I> could produce Arginine faster than <I>Synechocystis</I> can itself. By knocking out the gene to encode for the enzyme to produce arginine, <I>Synechocystis</I> does not need to waste energy in making these enzymes. </LI></OL><HEADER><H4>Auxotroph Criteria</H4></HEADER><P>There are several criteria that to be upheld when making the choice of what sort of auxotroph should be made. In our modeling attempts, an algorithm was developed to searched the metabolic map of <I>Synechocystis</I> and output candidate nutrients that fulfilled the following criteria.</P><OL style="font-family: 'Montserrat', sans-serif"><LI>Simple. We want to choose a nutrient that can be relatively simple to knock out. Thus in our search, we focused on nutrients that only required the knock out of one gene to produce the desired phenotype.  </LI><LI>Loss of the gene encoding the reaction to produce the nutrient would result in non-growth. If the nutrient is not in the medium, <I>Synechocystis</I> should not grow.
                        Reintroduction of the nutrient to the medium would result in growth of <I>Synechocystis</I>.  </LI><LI>The nutrient should be able to be produced and secreted by <I>E. coli</I>.</LI></OL></DIV></DIV><DIV class="row"><DIV class="6u"><HEADER><H4>Auxotroph Targets</H4></HEADER><P>With the aforementioned criteria in mind, the auxotroph finder that was developed by one of our team members was able to find a variety of different suitable nutrients for which would be suitable candidates. Of these, two amino acids were chosen to be knocked out: Arginine and Proline. </P><P>Arginine was a suitable target as we found a strain of <I>E. coli</I> that was able to synthesize it. In addition it has been shown that <I>Synechocystis</I> cells can grow faster in the presence of Arginine. The enzyme responsible for Arginine production in <I>Synechocystis</I> is L-arginosuccinate lyase (ASL) enzyme. This is encoded by the slr1133: ArgH gene. ASL drives the reaction from argininosuccinate into arginine and fumarate in the Urea Cycle. Figure 2 displays the mechanism behind Arginine production.</P><P>In the case of Proline, a strain of Salmonella and <I>E. coli</I> capable of producing Proline was procured. The enzyme responsible for Proline production in <I>Synechocystis</I> is pyrroline-5-carboxylate reductase (P5C reductase). This is encoded by the slr0661: ProC gene. Figure three displays the mechanism behind Proline production in <I>Synechocystis</I>.</P></DIV><DIV class="6u"><SECTION class="special" style="width:75%;"><FIGURE class="image fit"><FIGCAPTION>    
                                Figure 2. - Production of Arginine in <I>Synechocystis </I> is associated with the Urea cycle. ASL, circled in red, is the gene of the enzyme  whose loss results in a Arginine auxotrophic <I>Synechocystis</I></FIGCAPTION></FIGURE></SECTION></DIV></DIV></DIV></SECTION></A><DIV class="6u"><HEADER><H2>Methods</H2></HEADER><HEADER><H4>Markerless Knock Out Procedure</H4></HEADER><P>To conduct an auxotrophic <I>Synechocystis</I>, we followed the markerless knock-out protocol developed by Albers et al. This method involves a two step transformation of <I>Synechocystis</I>. During the first transformation, a cassette including a gene encoding antibiotic resistance is inserted into the genomic DNA, replacing the gene of interest. In the second transformation, counter selection under nickel yields colonies who have lost the inserted cassette. Overall resulting in a markerless transformation of <I>Synechocystis</I> to be deficient of the gene of interest. Figure 4 displays the overall mechanism of this protocol. Two plasmids were used. The plasmid shown in 4a contains the mazF/aphII casette. aphII is a gene encoding for Kanamycin resistance. nrsR--nrsS detects nickel. PnrsB is a nickel induced promoter. mazF is a protein synthesis inhibitor. In the presence of nickel, nrsR-nrsS detects the nickel, induces the PnrsB promoter which expresses mazF - inhibiting cell growth. In the presence of Kanamycin, cells without this cassette will die. Plasmid shown in 4b) is Ampicilin resistant plasmid and contains the up and downstream homologous regions to the gene of interest.</P></DIV><DIV class="6u"><SECTION class="special"><FIGURE class="image fit"><FIGCAPTION> Figure 3. - Production of Proline in. P5C reductase , circled in red, is the gene of the enzyme whose loss results in a Proline auxotrophic <I>Synechocystis</I></FIGCAPTION></FIGURE></SECTION></DIV></DIV><DIV class="row"><DIV class="4u"><P>Figure 5: Overall Mechanism of counter selection protocol. B) Plasmid containing up/downstream homologous region and mazF/aphII cassette is transformed into <I>Synechocystis</I>. Selection on Kanamycin leads to the gene of interest to be replaced by the mazF/aphII cassette in the genome. C) <I>Synechocystis</I> is then transformed again with the plasmid containing up/downstream homologous regions to the gene of interest. Counter selection on nickel leads to the loss of the mazF/aphII cassette. </P><HEADER><H4>Synechocystis Transformation Protocol</H4></HEADER><P><I>Synechocystis</I> grows on BG11. For all transformations used in this module, BG11 was supplemented with 50 ug/ml of Kanamycin.</P><P><I>Synechocystis</I> has 12 copies of its genome contained within one chromosome. Therefore after <I>Synechocystis</I> transformation, it is necessary to verify that the insert/deletion is present in all genomic copies. If not, the WT phenotype could easily take over. If the insert/deletion is not present in all genomic copies, a process called 'segregation' is used to add further selection pressure to <I>Synechocystis</I>. During segregation, <I>Synechocystis</I> is grown in the presence of Kanamycin over a long period of time. Frequent colony PCR checks are used to check the genomic status of the transformation.</P></DIV></DIV><DIV class="row"><HEADER><H2>Results</H2></HEADER></DIV><DIV class="row"><DIV class="6u"><HEADER><H4>Arginine KO Results</H4></HEADER><P>To create a strain of <I>Synechocystis</I> auxotrophic in Arginine, we attempted to knock-out the argH gene. We were able to make both knock-out constructs depicted in Figure 4. Verification of the constructs via Colony PCR of the transformed <I>E. coli</I> colonies are shown in Figure 9.</P><P>After the constructs were made, transformation of <I>Synechocystis</I> was attempted using BG11 supplemented with 1 mM Arginine and 50 ug/ml Kanamycin. Two strains of <I>Synechocystis</I> were transformed: WT strain and the Δacs acetate producing strain. [See Carbon producing module] After 2.5 weeks, pcr of resulting colonies suggested no presence of the mazF/aphII cassette (Figure 10) for either strain in the genomic DNA.</P></DIV><DIV class="6u"><HEADER><H4>Proline KO Results</H4></HEADER><P>Although we were able to amplify the downstream homologous region of ProC gene using a DNA polymerase with proofreading (Hercules), the upstream homologous region proved more difficult to amplify. Various troubleshooting attempts were implemented but ultimately only the Taq polymerase was capable of amplifying the region as shown. Fusion was then conducted with this Taq amplified upstream fragment and the Hercules amplified downstream fragment, but this failed. One possible reason for the inability to use Hercules polymerase to amplify this region might be due to the choice of primers. Due to the issues with amplifying this fragment, Proline was put on hold in favor of focusing cloning activities on the ArgH knockout. </P></DIV></DIV><DIV class="row"><DIV class="6u"><SECTION class="special"><FIGURE class="image fit"><FIGCAPTION> Figure 6: Gels of argH KO construct verification. For both gels, ArgH upstream forward primers and downstream reverse primers were used to conduct a colony PCR of the transformed <I>E. coli</I> colonies.  a) Verification of first construct from figure 4a. Both homologous regions together equal 2 kb band - shown here. b) Verification of second construct from figure 4b. Insert of 3.7 kb mazF/aphII cassette yields a total of 5.7 kb fragment, shown here with colony 5 and 11. </FIGCAPTION></FIGURE></SECTION><P>Because the expected bands were not present in the colony pcr verification and the gene was still very much in the genome, additional checks were conducted. <I>Synechocystis</I> shouldn't have been able to survive with the kanamycin added to the medium, so specific primers especially for the mazF/aphII cassette were used to check whether it was inserted. This check proved that mazF/aphII was transformed into the genome but still did not knock out the required gene. A check for single crossover was conducted that yielded negative. The same transformation was repeated again with similar results.</P><P>A different transformation was employed to improve the probability of success of <I>Synechocystis</I> to lose the argH gene. Cells were grown on BG11 medium with or without nitrate added, in addition to supplementation with 5 mM of Arginine. The rationale behind growing with and without nitrate is so that for <I>Synechocystis</I> cells grown without nitrate - Arginine would be the only nitrogen source, therefore forcing the cells in a position that it is more favourable for them to lose the argH gene as there is plenty of Arginine in the medium. In addition to these changes, the protocol was altered such that during all stages of transformation Arginine 5mM would be present in the medium, including when inoculating cells with the KO construct. These changes were all implemented and used to transform both WT and the  Δacs acetate producing strains. Only the WT strain grown on nitrate resulted in single pickable colonies. Both Δacs strains grown with and without nitrate in the BG11 as well as the WT strain grown without nitrate resulted in overgrowth of the culture such that no colonies were pickable. This was a strange result that given the time - would have been more strongly investigated. Nevertheless, conducting the transformation in this way yielded much more promising colonies in terms of insertion of the mazF/aphII cassette. Figure 8 shows the results of the WT transformation under these conditions. </P></DIV><DIV class="6u"><SECTION class="special"><FIGURE class="image fit"><FIGCAPTION> Figure 7: Gel of colony PCR after transformation of <I>Synechocystis</I>. Upstream forward and downstream reverse homologous region argH primers were used. WT untransformed genome is shown on first well as a control. Size of the argH gene is 1.5 kb. 3.5 kb band seen in all wells signifies the presence of the gene (1.5 kb + 2kb homologous region). 5.7 kb band indicating the presence of mazF/aphII cassette is not seen in any of the colonies </FIGCAPTION></FIGURE></SECTION><SECTION class="special"><FIGURE class="image fit"><FIGCAPTION> Figure 8: Gel of colony PCR after altered protocol transformation of <I>Synechocystis</I>. Upstream forward and downstream reverse homologous region argH primers were used. WT untransformed genome is shown on first well as a control. Size of the argH gene is 1.5 kb. 3.5 kb band seen in all wells signifies the presence of the gene (1.5 kb + 2kb homologous region). 5.7 kb band indicating the presence of mazF/aphII cassette is seen in many of the colonies </FIGCAPTION></FIGURE></SECTION></DIV></DIV><DIV class="row"><DIV class="8u"><HEADER><H2>Next Steps</H2></HEADER><P>
                        A KO of an essential gene from <I>Synechocystis</I> took more time than expected. However, with more time this is definitely possible. The colonies that looked most promising from our gel would undergo segregation such that for all copies of genome, argH gene would be removed. After this is confirmed, the second step of transformation via nickel selection would be performed. Removal of mazF/aphII cassette would be verified via colony PCR. Segregation under intense nickel pressure would also be performed if necessary. Once a suitable markerless knockout strain has been verified, testing would be conducted to confirm that this strain would be unable to grow on media not containing Arginine. In addition we would be able to fully test our consortia and verify that E. coli alone is capable of providing enough Arginine for an ΔargH mutant <I>Synechocystis</I>. 
                    </P></DIV><DIV class="4u"><HEADER><H2><A name="parts">Parts</A></H2></HEADER><P>
                            Parts submitted to the registry include the construct used for step two of the transformation protocol. This removes the mazF/aphII cassette from the genomic DNA such that there is a markerless knockout of argH in <I>Synechocystis</I></P></DIV></DIV><DIV class="row"><HEADER><H2>References</H2></HEADER><P>
                    Malcolm Watford, Glutamine Metabolism and Function in Relation to Proline Synthesis and the Safety of Glutamine and Proline Supplementation, American Society for Nutrition, 2008
                </P><P>  
                    Cheah, Y. E., Albers, S. C. and Peebles, C. A. M. (2013), A novel counter-selection method for markerless genetic modification in <I>Synechocystis </I>sp. PCC 6803. Biotechnol Progress, 29: 23–30. doi: 10.1002/btpr.1661
                </P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>