<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Amsterdam_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Amsterdam/Software</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Amsterdam 2015</TITLE></P><DIV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="https://2015.igem.org/Main_Page">iGEM</A></DIV><DIV class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:Amsterdam">Home</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Overview">Project </A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Stability">Stable romance</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Eng_rom">Engineering Romance </A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Eng_rom/Photosyn_car">Photosynthetic Carbon production</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Eng_rom/Dependecies">Dependent <I>Synechocystis</I></A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Software">Algorithms</A></LI></UL><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Simula_rom">Simulating Romance </A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Modeling">Simulations</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Phy_param">Physiological parameters </A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Phy_param/Synechocysytis"><I>Synechocysytis</I></A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/Phy_param/Ecoli"><I>E. coli</I></A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Project/emulsions">Evolving Romance</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Design">Testing Romance</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Drylab/Algorithms">Algorithms</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Parts">Achievements</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Medal_fulfilment">Medal fulfilment</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/eng_carbon"> Engineering Carbon production</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/syn_dependence"> Engineering Dependencies</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/Modelling">Dry lab</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/emulsions">Emulsions</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/phys_ecoli"><I>E. coli</I> physiology</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Notebook/phys_syn"><I>Synechocystis</I> physiology</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Practices">Human Practices </A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Human_practices/RIVM">RIVM</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Human_practices/App_scenario">Application Scenarios</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Human_practices/techno_vignette">Techno-moral vignette</A></LI><LI><A href="https://2015.igem.org/Team:Amsterdam/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><DIV id="contentContainer"><SECTION id="drylab_header" class="wrapper style7"><HEADER class="major"><H2>Algorithms</H2><H4>Trying to find a match</H4></HEADER></SECTION><SECTION id="algorithms_header" class="wrapper style6"><HEADER class="major"><H3>Overview</H3></HEADER><DIV class="container"><DIV class="row"><DIV class="3u"><SECTION><H3>Background</H3><P>In our project we focussed on the <A href="https://2015.igem.org/Team:Amsterdam/Project/Stability">stability </A> in different ways. One way is genetic stability, which we needed to engineer a stable producing strain. Another was the stability of our consortium itself. We needed an auxotrophic organism in order to be able to create a strong interdependent consortium.
</P><H3>Aim</H3><P>We need to find out what genes we could knock out to create a <U>stable</U> carbon producer. We also need to know what genes we should knock out in order to make an auxotrophic organism.
</P></SECTION></DIV><DIV class="3u"><SECTION><H3>Approach</H3><P>Here we present to you two novel algorithms which work with genome-scale FBA models. They can be used for any organism for which a genome-scale FBA model is available. One algorithm, the Stable Compound Generator, searches for ways to make a strain genetically stable produce carbon compounds. The second algorithm searches for ways to create an auxotrophic strain. Therefore we called it the Auxotrophy Sniper.
</P></SECTION></DIV><DIV class="3u"><SECTION><H3>Results</H3><UL style="font-family: 'Montserrat', sans-serif"><LI>We have created the Stable Compound Generator which find ways to stably produce a carbon compound in any organism.</LI><LI>We have created the Auxotrophy Sniper, which is able to find ways to crate an auxotrophic organism.</LI><LI>With the first algorithm we found a list products which could be stably produced by <I>Synechocystis</I>. We chose acetate from the list.</LI><LI>With the second algorithm we found out that it is possible to make a <I>Synechocystis</I> strain dependent on argenine, thus creating an auxotroph. </LI></UL></SECTION></DIV><DIV class="3u"><SECTION><H3>Connections</H3><P>Sometimes modellers tend to be the lone wolfs in a project. We didn't want this to happen, so there are some clear connections between the tools we created with modelling and the wet lab. Initially the need to search for compounds which could be produced genetically stable, came from the wet lab, where we saw that most producing strains are <A href="https://2015.igem.org/Team:Amsterdam/Project/Phy_param/Synechocysytis">unstable</A>. Before we even started <A href="https://2015.igem.org/Team:Amsterdam/Project/Eng_rom/Photosyn_car"> engineering <I>Synechocystis</I></A>, we wanted to find out whether we could produce a compound genetically <A href="https://2015.igem.org/Team:Amsterdam/Project/Stability">stable</A>. This is where the Stable Compound Generater comes in. We also needed to <A href="https://2015.igem.org/Team:Amsterdam/Project/Eng_rom/Dependecies">engineer an auxotroph</A> in order to to use serial propagations of consortia in <A href="https://2015.igem.org/Team:Amsterdam/Project/emulsions">emulsions</A> to find a more robust consortium. Both algorithms provided information which was really used in the lab.</P></SECTION></DIV></DIV></DIV></SECTION></DIV><SECTION id="algorithms_aim" class="wrapper style2"><HEADER class="major"><H3>Aim</H3></HEADER><DIV class="container"><DIV class="row"><DIV class="6u"><SECTION><A href="drylab.html#drylab_kinetic"><H4>Stable production is key for sustainability</H4></A><P>
There is a major problem in biotechnology and it is called <A href="https://2015.igem.org/Team:Amsterdam/Project/Stability">genetic instability</A>. In most bioreactors there is a constant selection pressure on growth rate. This means that when a mutant has a slightly higher growth rate it will soon take over the culture. In biotechnology and especially in synthetic biology, we tend to engineer micro-organisms in such a way that they produce a product and excrete it out of the cell. We then envision large scale applications and a more sustainable world for everybody. In reality, however, a lot of these applications don't work and this has a lot to do with genetic instability. Usually the production and excretion of a compound will lead to a lower growth rate. If an organism can easily lose the production (and sometimes this is as easy as the occurrence of a single mutation in the promoter region) the mutant will very fast take over an entire culture. Not only does this decrease economic yields, but it will also have a negative effect on the sustainability of the bioreactor, since a lot of resources will be used when a bioreactor has to be emptied and re-inoculated.
A stable production of compound will lead not only to a more stable consortium, it will also lead to more sustainability. We want to tackle the problem of sustainability, therefore we wrote an algorithm, which searches for ways to create a stable carbon compound producer.


</P></SECTION></DIV><DIV class="6u"><SECTION><A href="drylab.htmldrylab_kinetic"><H4>Reliance on each other</H4></A><P>
Auxotrophs are organisms which need a certain compound which they can take up from their environment to be able to grow. They can be very useful in synthetic biology, as an auxotrophy can regulate the growth of a certain organism. It is also useful synthetic consortium, as it can create a dependency of one species on another. In our project we wanted to make the chemoheterotroph dependent on the cyanobacterium by the carbon compound the cyanobacteria produces, because this will be part of the flux of CO<SUB>2</SUB> into product. However, we also wanted to create dependency of the cyanobacterium on the chemoheterotroph, because it can stabilize a consortium if, for example, the chemoheterotoph has a lower growth rate than the photoautotroph on the carbon compound. Other important reasons are that this dependency will decrease the risk of the cyanobacterium surviving on its own in the environment in case of an outbreak. Some chemoheterotrophs are very efficient in the production of a certain compound. If the photoautotroph does not have to synthesize the compound itself, it may increase the growth rate of the photoautotroph. This increases the rate at which carbon is fixated and carbon compound is produced, which in its turn benefits the chemoheterotroph. It is also suggested that this interdependency may create a more robust system. 
So the second question that we had in the lab is how can we create an auxotroph? To answer this question we also created an algorithm, the Auxotrophy Sniper.
</P></SECTION></DIV></DIV></DIV></SECTION><SECTION id="algorithms_approach" class="wrapper style4"><HEADER class="major"><H3>Approach</H3></HEADER><DIV class="6u"><DIV class="container"><DIV class="row"><P>
For some questions we had (see Aim) we decided to look for an answer based on algorithms for genome scale flux balance analysis (FBA) models. We have used genome scale FBA that already existed of <IT>Synechocystis</IT> and PySCeS CBMpy, a tool for <B>C</B>onstraint <B>B</B>ased <B>M</B>odeling using <B>Py</B>thon <B>S</B>imulator for <B>Ce</B>llular <B>S</B>ystems (<A href="http://pysces.sourceforge.net/">Olivier, 2014-2015</A>). In flux balance analysis (FBA) you assume that all reactions in a cell are in steady state and then you can then represent them in a set of linear equations. The objective function then optimizes something given a set of boundary conditions about the flux through each reaction. The objective function is often the formation of biomass. If a cell is producing biomass, means it grows.</P><DIV class="container"><DIV class="row"><DIV class="6u"><SECTION><A href="drylab.html#drylab_kinetic"><H4>Stable Compound Generator</H4></A><P>
In previous section we emphasized we wanted to create <SPAN class="textbf">stable</SPAN> carbon compound producing cyanobacterial strain. 
But how do we do this? We created an algorithm which finds ways to make a stable producer. 
It is based on two ideas:

<OL><LI>It takes a lot more evolutionary time to re-create a whole new gene than to accumulate a loss-of-function mutation.
</LI><LI>If the gene responsible for the production of the compound is expressed only when the organism grows (growth coupled production), the organism cannot simply stop expressing the pathway.
</LI></OL>
Below follows a general outline of the algorithm.
The algorithm makes a list of carbon compounds associated with the production of biomass in the cell.
For each of these compounds, the algorithm does the following:
<UL><LI>Find sources reactions of the compound in the extracellular space and set the boundaries of these reactions to zero.
</LI><LI>Find all reactions associated with the compound, as the model only contains reactions with gene associations and no genes which directly influence flux. We will call these reactions primary reactions
</LI><LI>Find the genes associated to the reactions.
</LI><LI>For each of these genes, find all reactions which have the gene of interest in their gene association. We will call these reactions secondary reactions
</LI><LI>Make a list of combinations of the genes associated to the production of a compound.
</LI><LI>For each of the genes involved each combination, set flux boundaries of these reactions to zero. In that way we simulate the knock-out of the gene.
</LI><LI>Since accumulation of compound is not possible in FBA, while in reality most cells are leaking, we create sinks for compounds which would otherwise possibly accumulate. These sinks prevent the model from going to a non-growing steady state, because the compound would otherwise accumulate.
</LI><LI>Do a flux balance analysis on the model en check if biomass is formed.
</LI><LI>Check the value of the sink of the compound of interest. 
</LI></UL>
If there is still formation of biomass (growth) and the sink of the compound of interest is used to export the compound out of the cell, the knock-out of the combination of genes is a good candidate for making a stable producer.

</P></SECTION></DIV><DIV class="6u"><SECTION><A href="drylab.htmldrylab_kinetic"><H4>Auxotrpophy Sniper</H4></A><P>
We wanted to create an auxotrophic <SPAN class="textit">Synechocystis</SPAN> strain as part of our consortium. But how can we make an auxotroph out of <SPAN class="textit">Synechocystis</SPAN>? To get the answer to this question we decided to create an algorithm that works on genome scale FBA models. The general idea is that an auxotroph can be created by knocking out a combination of genes involved in the production of the compound we want to make a dependency on. The organism should not be able to grow if there if these genes are knocked out and there is no source for the compound in the extracellular space. When this is the case we create a source in the extracellular space (add compound to the medium) and after that the organism should be able to grow again.
Here follows an overview of how the Auxotrophy Sniper works: 
The algorithm takes a list of compounds of which you want to make a synthesis deficiency in the organism of choice. This list can contain for example vitamins, or amino acids. 
For each metabolite on the list it does the following: 

<UL><LI>Set flux boundaries of source reaction in the extracellular space (if there is already one present) to zero.
</LI><LI>Find all primary reactions associated to the compound we want to make the organism dependent on.
</LI><LI>For each primary reaction, find genes which are associated to the primary reactions.
</LI><LI>For each gene find all reactions which have the gene in their gene association (secondary reactions).
</LI><LI>Make a list of possible combinations of genes which can be knocked out.
</LI><LI>Go over the combinations one by one, for each gene in a combination turn off primary and secondary reactions.
</LI><LI>Per combination, do a flux balance analysis and check for biomass formation (growth).
</LI><LI>If it does <SPAN class="textbf">not</SPAN> form any biomass, a source reaction of the compound is added to the model
</LI><LI>Another flux balance analysis of the model is done and if there is biomass formation now, the combination of genes are good candidates. If this combination of genes is knocked out the organism will probably become an auxotroph.
</LI></UL></P></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="algorithms_aim" class="wrapper style8"><HEADER class="major"><H3>results</H3></HEADER><DIV class="container"><DIV class="row"><DIV class="8u"><SECTION><P>
Here we developed two algorithms which further increase the arsenal of constraint-based modeling techniques. Their output was validated experimental and they were both implemented in an open-source modeling software platform (link to PysCes). Further, the source code is being made available in the following public repository {link}.  

</P><H4>Outlook</H4>
The modeling tools developed here, albeit “developed for” and “validated with” Synechocystis, are fully generic and may be applied to any other organism for which a GSM has been developed. We foresee that this tool will be very useful in many model guided synthetic biology projects and ultimately in the development of (sustainable) biotechnological processes.
</SECTION></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>