<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPF_Lausanne_Notebook_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPF Lausanne/Notebook/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE></P><DIV class="container"><NAV class="collapse navbar-collapse navHeaderCollapse" role="navigation"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Design">Prototype Design</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Interlab">Interlab Study</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievements </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Judging">Judging</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet Lab </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Ecoli">E. coli notebook</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Yeast">S. cerevisiae notebook</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Practices">Human Practices</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Team/Meet">Meet us</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Collaborations">Collaborations</A></LI><LI><A href="https://igem.org" target="_blank">iGEM</A></LI></NAV></DIV><DIV class="classy-bar"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H2>Protocols</H2></DIV></DIV></DIV></DIV><DIV class="prot-section"><DIV class="row"><DIV class="col-md-2 scrollspy"><UL id="nav" class="nav hidden-xs hidden-sm" data-spy="affix"><LI class="active"><A href="#agarosegel">Agarose Gel</A></LI><LI><A href="#competentcells">Competent Cell Preparation</A></LI><LI><A href="#dephosphorylation">5' Dephosphorylation</A></LI><LI><A href="#gelextraction">Gel Extraction</A></LI><LI><A href="#gibsonassembly">Gibson Assembly</A></LI><LI><A href="#glycerolstock">Glycerol Stock</A></LI><LI><A href="#lbmedium">Lysogeny Broth (LB) Medium</A></LI><LI><A href="#lbagar">Lysogeny Broth (LB) Agar Plates</A></LI><LI><A href="#miniprep">Miniprep</A></LI><LI><A href="#pcr">Polymerase Chain Reaction (PCR)</A></LI><LI><A href="#colonypcr">Colony PCR</A></LI><LI><A href="#phusionpcr">Phusion PCR</A></LI><LI><A href="#taqpcr">Taq PCR</A></LI><LI><A href="#Q5pcr">Q5 PCR</A></LI><LI><A href="#pcrpurification">PCR Product Purification</A></LI></UL><LI><A href="#restriction">Restriction Digest</A></LI><LI><A href="#yeast">S. cerevisiae Specific Protocols</A></LI><LI><A href="#aminoacidsolution">Amino Acid Solution</A></LI><LI><A href="#integration">Integration</A></LI><LI><A href="#pegliac">PEG/LiAc Solution</A></LI><LI><A href="#sdmedium">SD Medium</A></LI><LI><A href="#ypdmedium">YPD Medium</A></LI><LI><A href="#mutagenesis">Site-directed mutagenesis</A></LI><LI><A href="#t4ligation">T4 ligation</A></LI><LI><A href="#tae">Tris-Acetate-EDTA (TAE) buffer 50X</A></LI><LI><A href="#transformation">Transformation</A></LI></DIV><DIV class="col-md-offset-1 col-md-9"><SECTION id="agarosegel" class="panel"><H1>Agarose Gel</H1><H2>Materials</H2><UL><LI>1X TAE</LI><LI>Agarose</LI><LI>Gel Red</LI><LI>DNA samples</LI><LI>6X loading dye</LI><LI>Nuclease-free water</LI></UL><H2>Procedure</H2><OL><LI>Prepare 1.2% agarose gel for small fragments and 3% agarose gel for large fragments</LI><LI>Mix 50 mL 1X TAE and 0.6 g (1.2%) or 1.5 g (3%) agarose</LI><LI>Melt in microwave until agarose has melted (about 50 seconds)</LI><LI>Add 1.3 μL (1.2%) or 1.5 μL (3%) Gel Red</LI><LI>Pour solution into agarose gel mold with comb</LI><LI>Let set for 20 minutes or until solid</LI><LI>Place gel in 1X TAE and remove comb</LI><LI>Load samples of 200 ng (or 2  μL) DNA mixed with 2  μL 6X loading dye and nuclease-free water up to 12  μL</LI><LI>Run gel at 100-120 Volts for 40-50 minutes (1.2%) or 80 Volts for 2 hours (3%)</LI><LI>Take a picture of the gel at the UV detector</LI></OL></SECTION><SECTION id="competentcells" class="panel"><A href="http://openwetware.org/wiki/Gill:Calcium_chloride_competent_cells"><H1>Competent Cell Preparation<SMALL><U>Open Wet Ware Protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>Bacterial overnight liquid culture</LI><LI>Lysogeny broth (LB) medium</LI><LI>CaCl2 solution, ice cold: 60 mM CaCl2, 15% glycerol, 10 mM PIPES, pH 7, filter sterilize and store at room temperature</LI></UL><H2>Procedure</H2><OL><LI>Subculture overnight culture 1:100 in LB medium</LI><LI>Incubate at 37°C with shaking until culture reaches an OD600 of 0.375</LI><LI>Aliquot 20 mL if the culture into chilled 50 mL tubes</LI><LI>Leave tubes on ice for 5 – 10 minutes</LI><LI>Centrifuge cells at 1600 g for 7 minutes at 4°C</LI><LI>Discard supernatant and resuspend pellet in 4 mL ice cold CaCl2 solution</LI><LI>Centrifuge cells at 1100 g for 5 minutes at 4°C</LI><LI>Discard supernatant and resuspend pellet in 4 mL ice cold CaCl2 solution</LI><LI>Keep on ice for 30 minutes</LI><LI>Centrifuge cells at 1100 g for 5 minutes at 4°C</LI><LI>Discard supernatant and resuspend pellet in 800 μL ice cold CaCl2 solution</LI><LI>Aliquot 100 μL of this suspension into microcentrifuge tubes</LI><LI>Freeze in liquid nitrogen and store at -80°C</LI></OL></SECTION><SECTION id="dephosphorylation" class="panel"><A href="https://www.neb.com/protocols/1/01/01/vector-dephosphorylation-protocol"><H1>5' Dephosphorylation<SMALL><U>NEB Protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>1-5 µg cut DNA </LI><LI>10X Antarctic Phosphatase Reaction Buffer</LI><LI>Antarctic Phosphatase</LI></UL><H2>Procedure</H2><OL><LI>Add 1/10 volume of 10X Antarctic Phosphatase Reaction Buffer to 1-5 µg of DNA cut with any restriction endonuclease in any buffer.</LI><LI>Add 1 µl of Antarctic Phosphatase (5 units) and mix</LI><LI>Incubate for 15 minutes at 37°C for 5´ extensions or blunt-ends, 60 minutes for 3´ extensions</LI><LI>Heat inactivate (or as required to inactivate the restriction enzyme) for 5 minutes at 70°C</LI><LI>Proceed with ligation</LI></OL></SECTION><SECTION id="gelextraction" class="panel"><A href="https://www.qiagen.com/ch/resources/resourcedetail?id=f4ba2d24-8218-452c-ad6f-1b6f43194425&amp;lang=en"><H1>Gel extraction<SMALL><U>QIAGEN protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>Agarose gel </LI><LI>QIAquick Gel Extraction Kit </LI></UL><H2>Procedure</H2><OL><LI>Excise the DNA fragment from the agarose gel with a clean, sharp scalpel</LI><LI>Weigh the gel slice in a colorless tube. Add 3 volumes Buffer QG to 1 volume gel (100 mg = 100µL). For &gt;2% agarose gels, add 6 volumes Buffer QG</LI><LI> Incubate at 50ºC for 10 minutes (or until the gel slice has completely dissolved). Vortex the tube every 2-3 minutes to help dissolve the gel</LI><LI>After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10µL 3M sodium acetate, pH5.0, and mix. The color of the mixture will turn yellow</LI><LI>Add 1 gel volume of isopropanol to the sample and mix</LI><LI>Place a QIAquick spin column in a provided 2 mL collection tube</LI><LI>To bind DNA, apply the sample to QIAquick column and centrifuge for 1 min. Discard the flow-through and place the QIAquick column back into the same tube. For sample volumes of &gt;800 µL, load and spin again</LI><LI>To wash, add 0.75 mL Buffer PE to QIAquick column and centrifuge for 1 min. Discard the flow-through and place the QIAquick column back into the same tube</LI><LI>Centrifuge the QIAquick column once more in the provided 2 ML collection tube for 1 min at 13'000 rpm to remove residual buffer</LI><LI> Place QIAquick column into a clean 1.5 mL microcentrifuge tube</LI><LI> To elute DNA, add 50 µL Buffer EB or water to the center of the QIAquick membrane and centrifuge the column for 1 min. For increased DNA concentration, add 30 µL Buffer EB to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min. After the addition of Buffer EB to the QIAquick membrane, increasing the incubation time to up to 4 min can increase the yield of purified DNA.</LI></OL></SECTION><SECTION id="gibsonassembly" class="panel"><A href="https://www.neb.com/protocols/2012/12/11/gibson-assembly-protocol-e5510"><H1>Gibson Assembly<SMALL><U>NEB Protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>DNA fragments</LI><LI>2X Gibson Assembly Mater Mix (NEB)</LI><LI>2X NEBuilder Positive Control (NEB)</LI><LI>Deionized water</LI></UL><H2>Procedure</H2><OL><LI>Set up following reactions on ice, adding Gibson Assembly Master Mix last:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Component</TH><TH>2 – 3 Fragments Assembly</TH><TH>4 – 6 Fragments Assembly</TH><TH>Positive Control</TH></TR></THEAD><TBODY><TR><TD>Total Amount of Fragments</TD><TD>0.02 – 0.5 pmols</TD><TD>0.2 – 1 pmols</TD><TD>10 μL</TD></TR><TR class="even"><TD>2X Gibson Assembly Master Mix</TD><TD>10 μL</TD><TD>10 μL</TD><TD>10 μL</TD></TR><TR><TD>Deionized water  </TD><TD>to 20 μL</TD><TD>to 20 μL</TD><TD>0 μL</TD></TR></TBODY></TABLE><P><SMALL>Optimized efficiency for 50 – 100 ng of vectors and 2 – 3 fold of excess inserts</SMALL></P><LI>Incubate samples at 50°C for 15 minutes (2 – 3 fragments) or for 60 minutes (4 – 6 fragments)</LI><LI>Store samples on ice or at -20°C until transformation</LI><LI>Transform competent cells following the Transformation Protocol</LI></OL></SECTION><SECTION id="glycerolstock" class="panel"><A href="https://www.addgene.org/plasmid-protocols/create-glycerol-stock/"><H1>Glycerol Stock<SMALL><U>Addgene Protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>50% Glycerol</LI><LI>Bacterial overnight liquid culture</LI><LI>Liquid Nitrogen</LI></UL><H2>Procedure</H2><OL><LI>Mix 0.5 mL 50% glycerol and 0.5 mL bacterial culture</LI><LI>Freeze in liquid nitrogen and store at -80°C</LI></OL></SECTION><SECTION id="lbmedium" class="panel"><H1>Lysogeny Broth (LB) Medium</H1><H2>Materials</H2><UL><LI>Tryptone</LI><LI>Yeast extract</LI><LI>NaCl</LI><LI>Deionized water</LI><LI>NaOH</LI><LI>If necessary: antibiotics</LI></UL><H2>Procedure (for 1 L)</H2><OL><LI>Dissolve 10 g tryptone, 5 g yeast extract and 10 g NaCl in 950 mL deionized water</LI><LI>Adjust pH to 7 using 1 M NaOH and bring volume to 1 L</LI><LI>Autoclave</LI><LI>If necessary: let medium cool to 55°C and add antibiotic</LI><LI>Store at room temperature</LI></OL></SECTION><SECTION id="lbagar" class="panel"><H1>Lysogeny Broth (LB) Agar Plates</H1><H2>Materials</H2><UL><LI>Tryptone</LI><LI>Yeast extract</LI><LI>NaCl</LI><LI>Deionized water</LI><LI>NaOH</LI><LI>Agar</LI><LI>If necessary: antibiotics</LI><LI>Petri dishes</LI></UL><H2>Procedure (for 1 L, ie. about 50 plates)</H2><OL><LI>Dissolve 10 g tryptone, 5 g yeast extract and 10 g NaCl in 950 mL deionized water</LI><LI>Adjust pH to 7 using 1 M NaOH and bring volume to 1 L</LI><LI>Add 15 g agar
                    </LI><LI>Autoclave</LI><LI>If necessary: let medium cool to 55°C and add antibiotic</LI><LI>Pour into petri dishes (about 20 mL per dish) and let set</LI><LI>Invert and store at 4°C</LI></OL></SECTION><SECTION id="miniprep" class="panel"><A href="https://www.qiagen.com/ch/resources/resourcedetail?id=331740ca-077f-4ddd-9e5a-2083f98eebd5&amp;lang=en"><H1>Miniprep<U><SMALL>QIAGEN Protocol</SMALL></U></H1></A><H2>Materials</H2><UL><LI>Bacterial overnight liquid cultures (1 - 5 mL) </LI><LI>QIAprep Spin Miniprep Kit </LI></UL><H2>Procedure</H2><OL><LI>Pellet 1 -5 mL bacterial culture by centrifugation at more than 8000 rpm for 3 minutes</LI><LI>Resuspend pelleted bacterial cells in 250 μL P1 buffer and transfer to a microcentrifuge tube</LI><LI> Add 250 μL P2 buffer and mix by inverting tube 4 – 6 times</LI><LI>Add 350 μL N3 buffer and mix by inverting tube 4- 6 times</LI><LI>Centrifuge for 10 min at 13000 rpm</LI><LI>Apply supernatant to the QIAprep spin column by pipetting, centrifuge for 30 – 60 seconds and discard flow-through</LI><LI>Wash the QIAprep spin column by adding 0.5 mL PB buffer, centrifuge for 30 – 60 seconds and discard flow-through</LI><LI>Wash the QIAprep spin column by adding 0.75 mL PE buffer, centrifuge for 30 – 60 seconds and discard flow-through</LI><LI>Centrifuge for 1 minute to remove residual wash buffer</LI><LI> Elute DNA by placing QIAprep column in a clean 1.5 mL microcentrifuge tube and adding 50 μL EB buffer or water (or less for higher concentration). Let stand for 1 minute and centrifuge for 1 minute</LI></OL><H2>Comments</H2><P>With low copy plasmid it is often difficult to obtain high concentrations (needed for sequencing). In these cases it's possible to proceed as follows: culture bacteria in 5 x 7mL tubes of LB medium and perform the miniprep for each tube separately until step 9. Then elute DNA by applying two times step 10 to each tube with the same 50µl of EB buffer.</P></SECTION><SECTION id="pcr" class="panel"><H1>Polymerase Chain Reaction (PCR)</H1><SECTION id="colonypcr" class="panel"><H1>Colony PCR</H1><H2>Materials</H2><UL><LI>Materials for Taq PCR (except template plasmid DNA)</LI><LI>Petri dish with transformed colonies</LI></UL><H2>Procedure</H2><OL><LI>Prepare 25 μL reactions as in Taq PCR Protocol without template DNA</LI><LI>With a sterile tip, under the flame, scrape part of a single colony and add to PCR tubes</LI></OL></SECTION><LI>Mix by pipetting up and down or flicking the reactions </LI><LI>Put tubes in thermocycler with cycling conditions as described in Taq PCR Protocol with a longer initial denaturation (2 - 5 minutes)</LI></SECTION><SECTION id="phusionpcr" class="panel"><A href="https://www.neb.com/protocols/1/01/01/pcr-protocol-m0530"><H1>Phusion PCR<SMALL><U>NEB Protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>5X Phusion HF or GC Buffer</LI><LI>dNTPs</LI><LI>Forward and Reverse Primers</LI><LI>Template plasmid DNA</LI><LI>Phusion DNA polymerase</LI><LI>Nuclease-free Water</LI></UL><H2>Procedure</H2><OL><LI>Prepare following reaction in 0.5 mL PCR tubes on ice, adding polymerase last:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Component</TH><TH>20 μL reaction</TH><TH>50  μL reaction</TH></TR></THEAD><TBODY><TR><TD>5X Phusion HF or GC Buffer</TD><TD>4 μL</TD><TD>10 μL</TD></TR><TR class="even"><TD>10 mM dNTPs</TD><TD>0.4 μL</TD><TD>1 μL</TD></TR><TR><TD>10 μM Forward Primer</TD><TD>1 μL</TD><TD>2.5 μL</TD></TR><TR class="even"><TD>10 μM Reverse Primer</TD><TD>1 μL</TD><TD>2.5 μL</TD></TR><TR><TD>Template plasmid DNA</TD><TD>1 pg – 10 ng</TD><TD>1 pg – 10 ng</TD></TR><TR class="even"><TD>Phusion DNA Polymerase</TD><TD>0.2 μL</TD><TD>0.5 μL</TD></TR><TR><TD>Nuclease-free Water</TD><TD>to 20 μL</TD><TD>to 50 μL</TD></TR></TBODY></TABLE><P><SMALL>Usually 100 pg – 1 ng of template DNA is sufficient</SMALL></P><LI>Mix by pipetting up and down or flicking the reactions
                    </LI><LI>Put tubes in thermocycler (with a pre-heated lid) with following cycling conditions:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR class="even"><TD>25 – 35 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>5 - 10 seconds</TD></TR><TR><TD>Annealing</TD><TD>45 – 72°C</TD><TD>10 – 30 seconds</TD></TR><TR class="even"><TD>Extension</TD><TD>72°C</TD><TD>15 -30 seconds per kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>5 -10 minutes</TD></TR><TR class="even"><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE></OL></SECTION><SECTION id="taqpcr" class="panel"><A href="https://www.neb.com/protocols/1/01/01/taq-dna-polymerase-with-standard-taq-buffer-m0273"><H1>Taq PCR<SMALL><U>NEB Protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>10X Standard Taq Reaction Buffer</LI><LI>dNTPs</LI><LI>Forward and Reverse Primers</LI><LI>Template plasmid DNA</LI><LI>Taq DNA polymerase</LI><LI>Nuclease-free Water</LI></UL><H2>Procedure</H2><OL><LI>Prepare following reaction in 0.5 mL PCR tubes on ice, adding polymerase last:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Component</TH><TH>25 μL reaction</TH><TH>50 μL reaction</TH></TR></THEAD><TBODY><TR><TD>10X Standard Taq Reaction Buffer</TD><TD>2.5 μL</TD><TD>5 μL</TD></TR><TR class="even"><TD>10 mM dNTPs</TD><TD>0.5 μL</TD><TD>1 μL</TD></TR><TR><TD>10 μM Forward Primer</TD><TD>0.5 μL</TD><TD>1 μL</TD></TR><TR class="even"><TD>10 μM Reverse Primer</TD><TD>0.5 μL</TD><TD>1 μL</TD></TR><TR><TD>Template plasmid DNA</TD><TD>1 pg – 1 ng</TD><TD>1 pg – 1 ng</TD></TR><TR class="even"><TD>Taq DNA Polymerase</TD><TD>0.125 μL</TD><TD>0.25 μL</TD></TR><TR><TD>Nuclease-free Water</TD><TD>to 25 μL</TD><TD>to 50 μL</TD></TR></TBODY></TABLE><P><SMALL>Usually 100 pg – 1 ng of template DNA is sufficient</SMALL></P><LI>Mix by pipetting up and down or flicking the reactions
                    </LI><LI>Put tubes in thermocycler (with a pre-heated lid) with following cycling conditions:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>30 seconds</TD></TR><TR class="even"><TD>25 – 35 cycles</TD><TD>Denaturation</TD><TD>95°C</TD><TD>15 – 30 seconds</TD></TR><TR><TD>Annealing</TD><TD>45 – 68°C</TD><TD>15 – 60 seconds</TD></TR><TR class="even"><TD>Extension</TD><TD>68°C</TD><TD>1 minutes per kb</TD></TR><TR><TD>Final Extension</TD><TD>68°C</TD><TD>5 minutes</TD></TR><TR class="even"><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE></OL></SECTION><SECTION id="Q5pcr" class="panel"><A href="https://www.neb.com/protocols/2013/12/13/pcr-using-q5-high-fidelity-dna-polymerase-m0491"><H1>Q5 PCR<SMALL><U>NEB Protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>5X Q5 Reaction Buffer</LI><LI>dNTPs 10µM</LI><LI>Forward and Reverse Primers</LI><LI>Template plasmid DNA</LI><LI>Q5 High Fidelity DNA polymerase</LI><LI>Nuclease-free Water</LI></UL><H2>Procedure</H2><OL><LI>Prepare following reaction in 0.5 mL PCR tubes on ice, adding polymerase last:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Component</TH><TH>25 μL reaction</TH><TH>50 μL reaction</TH></TR></THEAD><TBODY><TR><TD>5X Q5 Reaction Buffer</TD><TD>5 μL</TD><TD>10 μL</TD></TR><TR class="even"><TD>10 mM dNTPs</TD><TD>0.5 μL</TD><TD>1 μL</TD></TR><TR><TD>10 μM Forward Primer</TD><TD>1.25 μL</TD><TD>2.5 μL</TD></TR><TR class="even"><TD>10 μM Reverse Primer</TD><TD>1.25 μL</TD><TD>2.5 μL</TD></TR><TR><TD>Template plasmid DNA</TD><TD>1 &lt; 1,000 ng</TD><TD>1 &lt; 1,000 ng</TD></TR><TR class="even"><TD>Q5 DNA Polymerase</TD><TD>0.25 μL</TD><TD>0.5 μL</TD></TR><TR><TD>Nuclease-free Water</TD><TD>to 25 μL</TD><TD>to 50 μL</TD></TR></TBODY></TABLE><P><SMALL>Usually 100 pg – 1 ng of template DNA is sufficient</SMALL></P><LI>Mix by pipetting up and down or flicking the reactions
                    </LI><LI>Put tubes in thermocycler (with a pre-heated lid) with following cycling conditions:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR class="even"><TD>25 – 35 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>5 – 10 seconds</TD></TR><TR><TD>Annealing</TD><TD>50 – 72°C</TD><TD>10 – 30 seconds</TD></TR><TR class="even"><TD>Extension</TD><TD>72°C</TD><TD>20 - 30 seconds per kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR class="even"><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE></OL></SECTION><SECTION id="pcrpurification" class="panel"><A href="https://www.qiagen.com/ch/resources/resourcedetail?id=390a728a-e6fc-43f7-bf59-b12091cc4380&amp;lang=en"><H1>PCR Product Purification<SMALL><U>QIAGEN Protocol</U></SMALL></H1><H2>Materials</H2><UL><LI>PCR products</LI><LI>QIAquick PCR Purification Kit</LI></UL><H2>Procedure</H2><OL><LI>Add 5 volumes PB buffer to 1 volume of PCR product and mix</LI><LI>Place QIAquick column in 2 ml collection tube</LI><LI>Apply samples to QIAquick column and centrifuge for 30 – 60 seconds, discard flow-through</LI><LI>Wash by adding 750 μL PE buffer to QIAquick column and centrifuge fo 30 – 60 seconds, discard flow-through</LI><LI>Centrifuge QIAquick column for 1 minutes to remove residual wash buffer</LI><LI>Elute DNA by adding 30 or 50 μL EB buffer or water to the center of the QIAquick column. Let stand for 1 minutes and centrifuge for 1 minute</LI></OL></A></SECTION><SECTION id="restriction" class="panel"><A href="https://www.neb.com/protocols/2012/12/07/optimizing-restriction-endonuclease-reactions"><H1>Restriction Digest<SMALL><U>NEB Protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>Restriction Enzyme(s)</LI><LI>DNA</LI><LI>10X NEBuffer (Appropriate buffer for used enzyme)</LI><LI>Water</LI></UL><H2>Procedure</H2><OL><LI>Prepare following reaction in 0.5 mL PCR tubes, adding enzyme(s) last:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Component</TH><TH>20 μL reaction</TH><TH>50 μL reaction</TH></TR></THEAD><TBODY><TR><TD>Restriction enzyme(s)</TD><TD>1 μL (for each enzyme)</TD><TD>1 μL (for each enzyme)</TD></TR><TR class="even"><TD>DNA</TD><TD>100 ng - 1 μg</TD><TD>100 ng - 1 μg</TD></TR><TR><TD>10X NEBuffer</TD><TD>2 μL</TD><TD>5 μL</TD></TR><TR class="even"><TD>Water</TD><TD>to 20 μL</TD><TD>to 50 μL</TD></TR></TBODY></TABLE><P><SMALL>20 μL reactions are sufficient for restriction enzyme analysis, larger volumes are usefull if product is used for cloning</SMALL></P><LI>Incubate at temperature and for duration appropriate for used enzyme (typically 37°C for 15 minutes or 1 hour)</LI><LI>Optional: Inactivate enzyme by incubating reaction at temperature and for duration appropriate for used enzyme (typically 65°C for 20 minutes)</LI></OL></SECTION><SECTION id="yeast" class="panel"><H1>S. cerevisiae Specific Protocols</H1><SECTION id="aminoacidsolution" class="panel"><H1>Amino acid solution</H1><H2>Materials</H2><UL><LI>Histidine-Hcl</LI><LI>Uracil</LI><LI>Leucine</LI><LI>Tryptophan</LI></UL><H2>Procedure</H2><TABLE cellspacing="0" style="width:90%"><THEAD><TR><TH>Stock concentration</TH><TH>Final concentration</TH><TH>Total quantity for 50 mL</TH></TR></THEAD><TBODY><TR><TD>100 mM Histidine-Hcl (209 g/mol)</TD><TD> 20.9 g/L</TD><TD> 1.045 g</TD></TR><TR class="even"><TD>20 mM Uracil (112 g/mol)</TD><TD> 2.24 g/L</TD><TD> 0.112 g</TD></TR><TR><TD>100 mM Leucine (131 g/mol)</TD><TD> 13.1 g/L</TD><TD> 0.655 g</TD></TR><TR class="even"><TD>40 mM Tryptophan (204 g/mol)</TD><TD> 8.16 g/L</TD><TD> 0.408 g</TD></TR></TBODY></TABLE><OL><LI>Filter and sterilize solutions</LI><LI>Add 8 mL per liter of selective medium or spread 500 μL on a selective plate</LI></OL></SECTION><SECTION id="integration" class="panel"><H1>Yeast integration</H1><H2>Procedure</H2><OL><LI>Start a pre-culture of 50mL YPD or SD + Glucose with one colony. Growing at 30°C with shaking (250 rpm) for 16h to 18h untill stationary phase, OD600 &gt; 1,5.</LI><LI>Seed yeast in 500 mL YPD or SD + Glucose. OD should be 0,2. Grow at 30°C in shaker for 3 hours untill OD 0,4 - 0,6.</LI><LI>Spin yeast down 5 min at 800 x g (1800 rpm approx).</LI><LI>Pour off medium and check that the medium is clear (if not, you may have a bacterial contamination).</LI><LI>Wash yeast with 25 mL of water (dH2O). Resuspend and pool cells into one tube.</LI><LI>Spin yeast down, 5 min at 1800 rpm. Decant supernatant.</LI><LI>Resuspend yeasts in freshly prepared TE/LiAc solution (approx 25 mL)</LI><LI>Spin yeast down, 5 min at 1800 rpm. Decant supernatant.</LI><LI>Resuspend yeast in TE/LiAc solution according to the following formula: OD600 x culture volume/60 = vol in which to resuspend (around 2 mL)</LI><LI>Add 10% vol of ssDNA (boiled at 100°C for 5 min and cooled on ice).</LI><LI>Add linearized vectors to the yeast (1 μg).</LI><LI>Add 1 ml of TE/lithium acetate/PEG solution, and resuspend carefully.</LI><LI>Incubate 30 min at 30°C with shaking 200 rpm.</LI><LI>Heat-shock the cells for exactly 20’ at 42°C (in a water bath).</LI><LI>Centrifuge the tubes for 5’’ in a microfuge at full speed at room temperature; remove the supernatant</LI><LI>Resuspend the cells in 50µl of sterile water, and plate onto the appropriate selective media plates using glass beads. Need 2-3 days to grow, at 30°C.</LI></OL></SECTION><SECTION id="pegliac" class="panel"><H1>PEG/LiAc Solution</H1><H2>Materials</H2><UL><LI>50% PEG  (Polyethylene glycol) prepared with sterile deionized water</LI><LI>10X TE buffer: 0.1 M Tris-Hcl, 10 mM EDTA, ph 7.5, autoclaved</LI><LI>10X LiAc: 1 M lithium acetate, pH 7.5 adjusted with dilute acetic acid, autoclaved</LI></UL><H2>Procedure</H2><OL><LI>Prepare PEG/LiAc solution as follows:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Stock concentration</TH><TH>Final concentration</TH><TH>Total quantity for 10 mL solution</TH></TR></THEAD><TBODY><TR><TD>50% PEG</TD><TD>40% PEG</TD><TD>8 mL</TD></TR><TR class="even"><TD>10X TE buffer</TD><TD>1X TE buffer</TD><TD>1 mL</TD></TR><TR><TD>10X LiAc</TD><TD>1X LiAc</TD><TD>1 mL</TD></TR></TBODY></TABLE></OL></SECTION><SECTION id="sdmedium" class="panel"><H1>Sd Medium</H1><H2>Materials</H2><UL><LI>Amino Acid Powder</LI><LI>Yeast Nitrogen Base</LI><LI>Ammonium Sulphate</LI><LI>Adenine Sulphate</LI><LI>Water</LI><LI>NaOH</LI><LI>Agar</LI><LI>Glucose</LI></UL><H2>Procedure</H2><OL><LI>Place stirrer bar in 2 L Erlenmeyer</LI><LI>Add 2.6 g amino acid powder, 3.4 g yeast nitrogen base, 10 g ammonium sulphate, 1 g adenine sulphate and 950 mL water</LI><LI>Adjust pH to 5.9 by adding a few drops of 10 M NaOH</LI><LI>In an other Erlenmeyer, add 35 g agar and 900 mL water</LI><LI>Autoclave both bottles</LI><LI>Transfer the content of first bottle to the agar-containing bottle</LI><LI>Cool to 55°C</LI><LI>Add 100 ml 40% glucose and 16 ml of the required amino acids</LI><LI>Pour plates</LI></OL></SECTION><SECTION id="ypdmedium" class="panel"><A href="http://www.clontech.com/xxclt_ibcGetAttachment.jsp?cItemId=17602"><H1>ypd medium<SMALL><U>Clontech Protocol</U></SMALL></H1></A><H2>Procedure</H2><LI>Prepare YPD mixture as follows : </LI><UL><LI>20 g/L peptone</LI><LI>10 g/L Yeast extract</LI><LI>20 g/L Agar (for plates only)</LI><LI>[Optional] For adenine-supplemented YPD (YPDA), add 15 ml of a 0.2% adenine hemisulfate solution per liter of medium (final concentration is 0.003%, in addition to the trace amount of Ade that is naturally present in YPD). Adenine hemisulfate tolerates autoclaving.</LI><LI>Add H2O to 950 ml. Adjust the pH to 6.5 if necessary, then autoclave. Allow medium to cool to ~ 55°C and then add dextrose (glucose) to 2% (50 ml of a sterile 40% stock solution). Adjust the final volume to 1 L if necessary.</LI><LI>[Optional] For kanamycin-containing medium, prepare YPD or YPDA as above. After autoclaved medium has cooled to 55°C, add 0.2–0.3 ml of 50 mg/ml kanamycin (final concentration 10–15 mg/L)</LI></UL></SECTION></SECTION><SECTION id="mutagenesis" class="panel"><A href="https://www.neb.com/protocols/2013/01/26/q5-site-directed-mutagenesis-kit-protocol-e0554"><H1>Site-directed mutagenesis<SMALL><U>NEB Protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>Q5 Hot Start High-Fidelity 2X Master Mix (NEB)</LI><LI>Forward and reverse primers</LI><LI>Template DNA</LI><LI>Nuclease-free water</LI></UL><H2>Procedure</H2><OL><LI>Prepare following reaction in 0.5 mL PCR tubes on ice, adding Mater Mix last:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Component</TH><TH>25 μL reaction</TH></TR></THEAD><TBODY><TR><TD>Q5 Hot Start High-Fidelity 2X Master Mix</TD><TD>12.5 μL</TD></TR><TR class="even"><TD>10 μM Forward Primer</TD><TD>1.25 μL</TD></TR><TR><TD>10 μM Reverse Primer</TD><TD>1.25 μL</TD></TR><TR class="even"><TD>Template DNA (1-25 ng/μL)</TD><TD>1 μL</TD></TR><TR><TD>Nuclease-free Water</TD><TD>9 μL</TD></TR></TBODY></TABLE><LI>Put tubes in thermocycler (with a pre-heated lid) with following cycling conditions:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR class="even"><TD>25 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD></TR><TR><TD>Annealing</TD><TD>50 – 72°C</TD><TD>10 – 30 seconds</TD></TR><TR class="even"><TD>Extension</TD><TD>72°C</TD><TD>20 - 30 seconds per kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR class="even"><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE><LI>Assemble following reagents:</LI><TABLE cellspacing="0" style="width:75%"><THEAD><TR><TH>Component</TH><TH>Volume</TH></TR></THEAD><TBODY><TR><TD>PCR Product</TD><TD>1 μL</TD></TR><TR class="even"><TD>2X KLD Reaction Buffer</TD><TD>5 μL</TD></TR><TR><TD>10X KLD Enzyme Mix</TD><TD>1 μL</TD></TR><TR class="even"><TD>Nuclease-free Water</TD><TD>3 μL</TD></TR></TBODY></TABLE><LI>Mix by pipetting up and down and incubate at room temperature for 5 minutes</LI><LI>Thaw completent E. coli cells on ice</LI><LI>Add 5 μL of KLD mix to the tube of thawed cells. Flick tube 4-5 times to mix.</LI><LI>Place mixture on ice for 30 minutes</LI><LI>Heat shock at 42°C for 30 seconds</LI><LI>Place on ice for 5 minutes</LI><LI>Pipette 950 μL of room temperature SOC into the mixture</LI><LI>Incubate at 37°C for 60 minutes with shaking</LI><LI>Mix thouroughly by flicking the tube and inverting, then spread 50-100 μL on a plate with appropriate antibiotics and incubate overnight at 37°C</LI></OL></SECTION><SECTION id="t4ligation" class="panel"><H1>Rapid ligation using T4 ligase</H1><SMALL> Based on Invitrogen protocol (for cohesive ends)</SMALL><H2>Materials</H2><UL><LI>5X ligase reaction buffer</LI><LI>vector DNA</LI><LI>insert DNA</LI><LI>autoclaved distilled water</LI><LI>T4 DNA ligase</LI></UL><H2>Procedure</H2><OL><LI>In a final volume of 20 µL mix 4 µL 5X ligase reaction buffer, 3 to 30 fmol vector DNA, 9 to 90 fmol insert DNA, 1 µL T4 ligase and complete with water</LI><LI>Mix gently. Centrifuge briefly to bring the contents to the bottom of the tube</LI><LI>Incubate at room temperature for 5 min</LI><LI>Use 2 μl of the ligation reaction to transform competent cells</LI></OL></SECTION><SECTION id="tae" class="panel"><H1>Tris-Acetate-EDTA (TAE) buffer 50X</H1><H2>Materials</H2><UL><LI>Acetate 100%</LI><LI>Tris base</LI><LI>EDTA 0,5M</LI></UL><H2>Procedure (1L)</H2><OL><LI>Add 100mL of 0,5M EDTA (pH 8.0)</LI><LI>Add 242g of Tris base </LI><LI>Add 57,1mL of glacial acetic acid (100%)</LI><LI>Fill up to 1L with water (adjust pH to ~8.3)</LI><LI>Send to autoclave</LI><P><SMALL>To prepare 1L of 1X TAE dilute 20mL of 50X TAE in 980mL of water.</SMALL></P></OL></SECTION><SECTION id="transformation" class="panel"><A href="https://www.neb.com/protocols/2012/12/11/gibson-assembly-transformation-protocol-e5510"><H1>Transformation<SMALL><U>NEB Protocol</U></SMALL></H1></A><H2>Materials</H2><UL><LI>Competent cells</LI><LI>DNA</LI><LI>SOC medium (SOB + Glucose)</LI><LI>Petri dish with appropriate antibiotic resistance</LI></UL><H2>Procedure</H2><OL><LI>Thaw competent cells on ice</LI><LI>Add 2 μL DNA to the competent cells, mix by pipetting up and down or flicking the tube 4-5 times</LI><LI>Place mixture on ice for 30 minutes</LI><LI>Heat shock at 42°C for 30 seconds</LI><LI>Transfer tubes to ice for 2 minutes</LI><LI>Add 950 μL room-temperature SOC media</LI><LI>Incubate at 37°C for 60 minutes with shaking</LI><LI>Spread 100 μL cells onto selection plates (warm plates to 37°C prior to this step for increased efficiency)</LI><LI>Incubate overnight at 37°C</LI></OL></SECTION></DIV></DIV></DIV><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE></DIV></DIV></DIV></DIV></BODY></HTML>