<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPF_Lausanne_Project_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPF Lausanne/Project/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE></P><DIV class="container"><NAV class="collapse navbar-collapse navHeaderCollapse" role="navigation"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Design">Prototype Design</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Interlab">Interlab Study</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievements </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Judging">Judging</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet Lab </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Ecoli">E. coli notebook</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Yeast">S. cerevisiae notebook</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Practices">Human Practices</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Team/Meet">Meet us</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Collaborations">Collaborations</A></LI><LI><A href="https://igem.org" target="_blank">iGEM</A></LI></NAV></DIV><DIV class="classy-bar"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H3>Project description</H3></DIV></DIV></DIV></DIV><DIV class="container"><UL class="nav nav-tabs nav-justified" id="myTab"><LI class="active"><A data-toggle="tab" href="#BioLOGIC">Bio LOGIC</A></LI><LI><A data-toggle="tab" href="#design-ecoli">Design in E. coli</A></LI><LI><A data-toggle="tab" href="#design-yeast">Design in S. cerevisiae</A></LI><LI><A data-toggle="tab" href="#dis">Gate designs</A></LI><LI><A data-toggle="tab" href="#background">Background</A></LI></UL><DIV class="tab-content background-section"><DIV id="dis" class="tab-pane fade"><H1>Gate designs</H1><H3>Logic Gate</H3><P>The first step in creating a biological circuit is to imitate the way the electrical signal is processed. A low voltage (or absence of signal) corresponds to the “0” signal and a high voltage (or presence of signal) to the “1” signal. In the Bio LOGIC design, there are no “low” or “high” signals. Both the “0” and the “1” signals are represented by specific gRNAs and it is promoters that work as transistors.</P><P>Logic gates have the ability to receive an input (for instance a “1” and a “0” signal) and to produce a specific output (for instance signal “1”) using transistors.</P><P>We explain here how our three Bio LOGIC NAND gate designs (simple, medium and complex) make this processing possible using biologic components: dCas9 (the wires), promoters (the transistors) and gRNAs (signals “0” or “1”).</P><TABLE cellspacing="0" style="width:20%"><THEAD><TR><TH>\(A_\text{input}\)</TH><TH>\(B_\text{input}\)</TH><TH>\(C_\text{output}\)</TH></TR></THEAD><TBODY><TR><TD>\(A_0\)</TD><TD>\(B_0\)</TD><TD>\(C_1\)</TD></TR><TR class="even"><TD>\(A_0\)</TD><TD>\(B_1\)</TD><TD>\(C_1\)</TD></TR><TR><TD>\(A_1\)</TD><TD>\(B_0\)</TD><TD>\(C_1\)</TD></TR><TR class="even"><TD>\(A_1\)</TD><TD>\(B_1\)</TD><TD>\(C_0\)</TD></TR></TBODY></TABLE><H3>Gate simple</H3><P>Let us begin with the first NAND gate design: the simple design. The input gRNAs are A and B, and they can represent either a “1” signal (A1, B1) or a “0” signal (A0, B0). The output gRNAs is either C1 (for an output signal “1”)  or C0 (for an output signal  “0”). In order to make the gate testable, we replaced gRNA C0 by a RFP and gRNA C1 by a GFP.</P><P>We will go step by step through all four lines of the truth table on the right.</P><P>First line: if  A0 and B0 are expressed, promoter 1 is activated, therefore expressing C1, but promoter 2 is repressed. Therefore C0 is not produced, and the ouput is C1.</P><P>Second line: if A0 and B1 are expressed, promoter 1 is activated thus producing C1. But promoter 2 is repressed, because repression prevails on activation (A R results in the same state as R). Therefore C0 is not produced, and the output is C1.</P><P>Third line: A1 does not bind anywhere on promoters 1 or 2, and therefore does nothing. But B0 activates promoter 1, thus producing C1: the final output.</P><P>Fourth line: A1 does not bind anywhere on promoter 1 and 2, and therefore does nothing. But B1 activates promoter 2, thus producing C0: the final output.</P><P>We just went through the truth table of a biological NAND gate.</P><H3>Gate medium</H3><P>Now let’s continue with the second gate design: the medium design. This time, the signal is processed by three promoters (or transistors) instead of two in the simple design. Again, we will go through all lines of the truth table.</P><P>First line: if A0 and B0 are produced, promoter 1 is repressed, promoter 2 is repressed (because repression prevails over activation) and promoter 3 is activated, thus producing C1: the final signal.</P><P>Second line: if A0 and B1 are produced, promoter 1 is repressed (because repression prevails over activation), promoter 2 is activated, thus producing C1 and promoter 3 is repressed. Therefore, the final signal is C1.</P><P>Third line: A1 does not bind anywhere on promoters 1, 2 or 3 and therefore does not do anything. B0 represses promoter 2 and activates promoter 3, which produces C1: the final signal.</P><P>Fourth line: A1 does not bind anywhere on promoters 1, 2 and 3 and therefore does not do anything. B1 activates promoter 1, thus producing C0 and represses promoter 3. Therefore, C0 is the final signal.</P><H3>Gate complex</H3><P>The third design, the most complex of the three designs avoids basal expression levels. Indeed, in the the simple and the medium design, the gRNA specific sequence A1 is not present on any of the promoters. Therefore had to assume that the basal level of the promoters will not produce a strong enough output to compete with the intended output.</P><P>The complex design theoratically solves this problem, as all promoters are always either activated or repressed.</P><P>Let’s go one last time through the four lines of the truth table.</P><P>First line: if A0 and B0 are produced, promoter 1 is repressed, promoter 2 is repressed but promoter 3 is activated, thus producing C1: the final signal.</P></DIV><DIV id="BioLOGIC" class="tab-pane fade in active"><H1>Bio LOGIC in 12 questions</H1><H3>What does Bio LOGIC stand for?</H3><P>It stands for <B>&quot;Bio Logic Orthogonal gRNA-Implemented Circuit”</B>. In a few words, we are working on implementing digital-like circuits in cells using dCas9.</P><H3>Don’t biological circuits already exist?</H3><P>Yes. However, difficulties in the multiplication and chaining of logic elements has hindered the complexification of these circuits. To overcome these limitations, an ideal in vivo logic element should be modular, reusable and orthogonal - i.e. avoiding cross-talk with its host organism and the other elements of the circuit.</P><DIV class="row"><DIV class="col-md-8"><H3>So, what’s different about your system?</H3><P>We can avoid some of these issues by making a <B>completely synthetic biological circuit</B>. This is what we are doing by using the newly discovered <B>dCas9</B> as a <B>synthetic transcription factor</B>.</P><H3>How can dCas9 be used as a transcription factor?</H3><P>Well, you know <B>CRISPR-Cas9</B>, the RNA-guided DNA endonuclease, right? (Check out the <A href="#background" data-toggle="tab">Background tab</A> for more information about CRISPR-Cas9.) We are using <B>dCas9</B>, the catalytically dead version of Cas9, which lacks the ability to cleave DNA, but can still bind it. We fused dCas9 to a <B>RNA Polymerase (RNAP) recruiting element</B>. Depending on where it binds, it will either activate or inhibit transcription.</P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><H3>How does this activation/inhibition system work?</H3><P>When dCas9 binds at an optimal distance upstream from a promoter, the RNAP recruiting element with which it is fused will, in fact, recruit RNAP, thus <B>activating the transcription</B> of the gene that is controlled by this promoter. However, when dCas9 binds close to the transcription start site, it will <B>sterically hinder RNAP from binding</B> at the transcription starting site, thus <B>inhibiting the transcription</B> of the gene.</P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><H3>How do you guide the dCas9 to activating/inhibiting region?</H3><P>dCas9 works just like Cas9, meaning it is RNA-guided. <B>Guide RNA (gRNA)</B> and dCas9 can form a complex. This complex will bind tightly to a DNA sequence which is complementary to the gRNA specificity determinant sequence. So we can guide dCas9 to activating or inhibiting regions of a promoter by producing adequate gRNAs.</P></DIV></DIV><H3>Can a gRNA-dCas9 complex activate one region and inhibit another in the same cell?</H3><P>Yes, a gRNA-dCas9 complex will bind to any sequence that is complementary to the gRNA. So, if the activating region and the inhibiting region of different promoters have the same sequence, two identical gRNA-dCas9 complexes can bind both at the same time. This is also why we have to be careful not to target regions that are present in the genome of the host organism, in order to avoid interfering with other cellular functions.</P><DIV class="row"><DIV class="col-md-8"><H3>What happens if the activating region and the inhibiting region of the same promoter are bound by dCas9 at the same time?</H3><P>That is a very good question! It was shown that inhibition is dominant in S. cerevisiae (yeast) [1]. This means that, if both activating and inhibiting regions are bound,  transcription will be inhibited, RNAP is recruited but steric hindrance still prevents its binding. One goal of our project is to find out if this is also the case in E. coli (bacteria). We will also test our system in yeast.</P></DIV></DIV><H3>How are you going to use this to make biological circuits?</H3><P>Because of time constraints, we were not able to make a real biologic circuit. We aim to make and characterize a biological equivalent to the simplest element in a digital circuit, a <B>transistor</B>. Transistors function like controllable switches for electric current. Our <B>bio-transistors</B> work like switches for the transcription of a gene and can be assembled to form biological circuits.

      </P><H3>What does your bio-transistor look like?</H3><P>The <B>bio-transistor</B> is a synthetic promoter with a gene output, but regulated in a novel way. We insert this synthetic sequence in a cell, along with a gene producing <B>dCas9 fused to a RNAP recruiting element</B> and a sequence that produces a gRNA complementary to either the activating or the inhibiting region of the promoter. dCas9 and the gRNA are produced, they form a complex which <B>binds to the activating or inhibiting region</B> of the promoter, thus “turning the gene on or off”. Because the &quot;off&quot; state prevails over the &quot;on&quot; state, its behaviour mimicks that of an electronic PNP transistor. </P><H3>How do you make biological circuits from bio-transistors?</H3><P>Well, digital circuits are made out of <B>logic gates</B>, elements that perform basic logic functions (AND, OR, NOT, NOR, NAND, XOR, etc.), and logic gates are made out of transistors. Our idea is to assemble our bio-transistors into logic gates. By linking the output of one logic gate to the input of another, we can make <B>biological circuits</B> that function in the same way as digital circuits.</P><H3>Sounds cool! But what can this be used for?</H3><P>A single transistor is not very useful. However, by assembling a certain number of bio-transistors, we could make more <B>complex biological circuits</B> that would have a wide range of outputs depending on many inputs. For example, we could make <B>complex biosensors</B> by building a circuit that is activated by the presence of a specific combination of molecules, or that has a different response for different combinations of molecules. This is only one example among the <B>many applications of biological circuits</B>.</P><P>Find out more about how we implemented our system in <A data-toggle="tab" href="#design-ecoli">E. coli</A> and in <A data-toggle="tab" href="#design-yeast">S. cerevisiae</A> or about <A data-toggle="tab" href="#background">what’s already been done with biological circuits and the publications that inspired us</A>, or check out our <A href="https://2015.igem.org/Team:EPF_Lausanne/Results">results</A>.</P><H3>References</H3><P>[1] Farzadfard, F., Perli, S. D., Lu, T. K. (2013). Tunable and Multifunctional Eukaryotic Transcription Factors Based on CRISPR/Cas. ACS Synth. Biol., 2 (10), pp 604–613.</P></DIV><DIV id="design-ecoli" class="tab-pane fade"><DIV class="row"><P>The story we are composing is the imitation of an electronic circuit, made of biological elements. Every biological element of this system has a role to play. Here are the members of the cast:</P></DIV><TABLE cellspacing="0" style="width:80%"><THEAD><TR><TH>Electronic transistor</TH><TH>Bio LOGIC in E. coli</TH></TR></THEAD><TBODY><TR><TD><B>Connector</B></TD><TD>Electrical wires</TD><TD>dCas9-ω protein</TD></TR><TR class="even"><TD><B>Transmitted information</B></TD><TD>Electrical voltage either high or low</TD><TD>gRNAs either activating or repressing</TD></TR><TR><TD><B>Receptor of information</B></TD><TD>Entry of next transistor, the base</TD><TD>Promoter J23117</TD></TR></TBODY></TABLE><H2>dCas9-ω as a connector</H2><DIV class="row"><DIV class="col-md-8"><P>We will use the DNA-binding activity of the catalytically 'dead' version of Cas9, <B>dCas9</B>, to regulate genes.</P><P>For activation to be possible, dCas9 needs to be fused to a <B>RNA polymerase (RNAP) recruiting element</B>. We fused the <B>ω (omega) subunit of RNAP</B> to dCas9 [1]. The ω subunit works as a RNAP recruiting element in E. coli when working in a strain in which RNAP lacks the ω subunit. We used JEN202, &quot;an E. coli MG1655 mutant in which rpoZ, encoding for the ω subunit of RNAP, was replaced by a Spectinomycin resistance gene&quot; [1], for all fluorescence measurements.</P></DIV></DIV><H2>sgRNAs as the transmitted information</H2><DIV class="row"><P>We used <B>single guide RNAs (sgRNA)</B> to guide dCas9-ω to targeted regions flanked by a PAM sequence. An sgRNA &quot;comprises a complementary domain that binds to the DNA followed by a “handle” that is bound by dCas9&quot; [3]: in our case dCas9-ω. dCas9-ω and an sgRNA form a complex which will tightly bind a DNA sequence complementary to the 'complementary domain' of the sgRNA.</P><P><B>Find out how we regulate genes with dCas9-ω below!</B></P></DIV><H2>The bio-transistor</H2><DIV class="row"><DIV class="col-md-8"><P>The <B>bio-transistor for E. coli</B> consists of a gene controlled by a synthetic promoter.</P><P>We synthesized promoters based on the constitutive promoter J23117 [1]. This promoter consists of <A href="http://parts.igem.org/Part:BBa_J23117">BBa_J233117</A>, preceded by a protospacer-adjacent motif (PAM) rich upstream regulating sequence (URS) [4].</P><P>To show that the bio-transistor works with different promoter sequences, we tested our system for the promoters J23117 and J23117_alt. The sequence for J23117_alt was randomly generated, except for the -35 and -10 regions that were conserved from J23117. Some of our team members wrote a program in C++ and then in Python that generated this type of random sequence while conserving part of an original sequence.To observe the activity of a single bio-transistor, we used a gene encoding for green fluorescent protein (GFP) as a reporter gene.</P></DIV></DIV><DIV class="row"><DIV class="col-md-8"><P>If dCas9-ω guided by an sgRNA binds on the promoter close to the transcription start site, the binding of RNAP to DNA is sterically inhibited, and <B>transcription is repressed</B>. We chose to use sgRNAs complementary to sequences 14 or 18 bp upstream from the transcription start site (TSS). We will call these the inhibiting sites -14 and -18.If dCas9-ω guided by an sgRNA binds at an optimal distance upstream from the promoter, the ω subunit recruits RNAP and <B>transcription is activated</B>. We chose to use sgRNAs complementary to the sequence 71 bp upstream from the TSS. [1]Note that the activating sgRNA and the -18 inhibiting sgRNA are complementary to the bottom strand of the promoter, whereas the -14 inhibiting sgRNA is complementary to the top strand.When generating sequences for the sgRNAs and the binding sites, it is important that they are not present in the host organism's genome to avoid regulating genes other than the targeted one.</P><P>If both one inhibiting and the activating sgRNA for the same promoter are present in the same cell, we foresaw 2 possible situations. We simulated both with our <B><A href="https://2015.igem.org/Team:EPF_Lausanne/Modeling">model</A></B>: either dCas9-ω can bind both the activating and the inhibiting sites, in this case the model predicts that inhibition will tend to be stronger, or dCas9-ω cannot bind both sites due to steric hindrance in which case the overall effect should tend to be activating.<B>For more information on these situations, check out our <A href="https://2015.igem.org/Team:EPF_Lausanne/Modeling"><U>modeling page</U></A>, or see how this turned out experimentally in our <A href="https://2015.igem.org/Team:EPF_Lausanne/Results"><U>results</U></A>!</B></P><P>We constructed plasmid pdCas9-ω, encoding for <B>dCas9-ω</B> regulated by a Tetracylcine-inducible promoter, from pdCas9-bacteria [2] and pWJ66 [1]. We synthesized '<B>sgRNA expressing cassettes</B>' (IDT) controlled by the constitutive promoter pBad and inserted these into pdCas9-ω. We inserted either 1 or 2 sgRNAs (combinations of 1 activating and 2 inhibiting, affecting the same promoter).We used pWJ89 [1], GFP controlled by the J23117 promoter, [1] as a '<B>reporter-transistor</B>'. We constructed a 2nd 'reporter-transistor' called pWJ89_alt, GFP controlled by the J23117_alt promoter, from pWJ89 and J23117_alt (synthesized by IDT).<B>Find out more about the construction of these plasmids in our <A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Ecoli"><U>Lab Notebook</U></A>!</B></P><P>We transformed JEN202 cells with one of the 'reporter-transistors' and pdCas9-ω with sgRNAs complementary to the activating and/or inhibiting regions of the promoter of the transistor, and measured the fluorescence of these cells with a <B>plate reader</B> or by <B>flow cytometry</B>. After trying several concentrations of Anhydrotetracycline (ATc), we decided to do all further experiments with 1 ng/mL ATc.<B>Take a look at our <A href="https://2015.igem.org/Team:EPF_Lausanne/Results"><U>results here</U></A>!</B></P></DIV></DIV><H2>Inducible bio-transistors</H2><DIV class="row"><DIV class="col-md-8"><P>In a biological circuit, the production of the sgRNAs would most likely be induced, instead of constitutive. To test this with our bio-transistors, we changed the promoter of the 'sgRNA expressing cassette' to make it <B>inducible</B>. We did this by inserting a 'sgRNA expressing cassette', controlled by constitutive promoter <B>pBad</B>, in a plasmid containing <B>AraC</B>, placing the pBad promoter next to AraC. When placed next to one another, AraC is a repressor of pBad, and pBad can be induced by adding Arabinose to the medium [5]. Thus, the production of this sgRNAs becomes inducible with Arabinose.</P><P>We constructed three plasmids in this fashion with sgRNAs complementary to the J23117_alt promoter: one with the activating sgRNA controlled by pBad/AraC, one with one of the inhibiting sgRNAs controlled by pBAd/AraC, and one with the same inhibiting sgRNA controlled by pBad/Arac and the activating sgRNA controlled by pBad (constitutive).<B>Find out more about the construction of these plasmids in our <A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Ecoli"><U>Lab Notebook</U></A>!</B></P></DIV></DIV><P>We transformed JEN202 cells with the 'reporter-transistor' with J23117_alt promoter, pdCas9-ω and one of the pBad/AraC constructs, and (again) measured the fluorescence of these cells with a <B>plate reader</B> or by <B>flow cytometry</B>. We kept ATc levels at 1 ng/mL and tested 0 mM, 0.1 mM, 1 mM, 10 mM Arabinose.<B>See how this worked out on our <A href="https://2015.igem.org/Team:EPF_Lausanne/Results"><U>results page</U></A>.</B></P><H2>Linking bio-transistors</H2><DIV class="row"><DIV class="col-md-6"><P>To make biological circuits with our bio-transistors, we will have to <B>link several bio-transistors</B>. To test whether the &quot;signal&quot; is strong enough for this to be possible with our system, we modified the transistor with promoter J23117 to express the sgRNA complementary to the activating site of J23117_alt, instead of GFP. In this way, when the first transistor (J23117) is activated by dCas9-ω bound to the corresponding sgRNA, another sgRNA will be expressed. This sgRNA, in complex with dCas9-ω, will bind the activating site of the second transistor (J23117_alt) which will activate the transcription of GFP. In an ideal situation, we would like to see that fluorescence levels obtained like this are close to levels obtained when simply activating one transistor.</P><P>We constructed this in one plasmid that we called pWJ89_alt_Z4-to-X4 by inserting the J23117 promoter followed by an 'sgRNA expressing cassette' (synthesized by IDT) into pWJ89_alt, the plasmid that contains GFP controlled by J23117_alt.<B>You can find out more about the construction of this plasmid in our <A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Ecoli"><U>Lab Notebook</U></A>!</B></P></DIV></DIV><P>We transformed JEN202 cells with pWJ89_alt_Z4-to-X4 and pdCas9-ω with different 'sgRNA expressing cassettes', notably the one expressing the sgRNA that will activate J23117. We kept ATc levels at 1 ng/mL.<B>See the <A href="https://2015.igem.org/Team:EPF_Lausanne/Results"><U>results here</U></A></B>.</P><P><B>Note that in the Lab Notebook and in the figures the sgRNAs (and the corresponding binding sites on the promoters) have specific names:</B>sgRNA activating J23117 and J23117_alt respectively: Z4 and X4. sgRNA -14 inhibiting J23117 and J23117_alt respectively: Z0 and X0. sgRNA -18 inhibiting J23117 and J23117_alt respectively: Z35 and X35</P><H2>The logic gate</H2><DIV class="row"><DIV class="col-md-10"><P>In digital circuits, transistors are assembled to form <B>logic gates</B> [6], which can then be linked to form complex circuits. Based on this, we decided to assemble our bio-transistors to form a <B>(bio)logic gate</B>. Below is our design for the <B>NAND gate</B> [7]. It corresponds to the &quot;gate simple&quot; design described in the &quot;Gate designs&quot; tab.</P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><P>The design of our NAND gate contains 3 bio-transistors. The inputs, A and B, are in the form of sgRNAs called A0, A1, B0 and B1.Note that in our model the sequence A1 is not present in the transistors. dCas9-ω will bind to A1 but it will not find a binding site on the DNA.The output, C, is the transcription of GFP for C=0, and RFP for C=1.</P><P>We did the following design with the hypothesis that when activating and inhibiting sites of the same promoter are bound by dCas9-ω, the overall effect will be inhibition. We also suppose that, when neither the activating nor the inhibiting sites are bound, the basal transcription level is similar to the inhibited level.Note that many other designs of this gate are possible, and that with similar systems we could also reproduce other gates, such as NOT, AND, OR, NOR, XOR, etc.</P></DIV></DIV><P>Let's go through the first line of the <B>truth table</B> together. Note that all lines of the truth table is illustrated below.Let's start with A=0 and B=0. dCas9-ω will bind the inhibitory site of the 1st transistor. Transcription of GFP, C=0, will be at level <B>'i' (inhibited)</B>. dCas9-ω will bind both the inhibiting and the activating site of the 2nd transistor. Transcription of RFP(1), C=1, will be at level <B>'a/i' (activated and inhibited at the same time)</B>, which we suppose is equivalent to level 'i'. dCas9-ω will also bind to the activating site of the 3rd transistor. Transcription of RFP(2), C=1, will be at level <B>'a' (activated)</B>. If we consider 'i' levels of transcription to be negligible, we have a final result of an 'a' level of RFP transcription, ie. we have C=1 just like in the truth table.You can find a summary of the transcriptional response of the NAND gate in the 3rd column of the truth table below.</P><DIV class="row"><DIV class="col-md-6"><FIGURE><FIGCAPTION><B>A=0 B=0: C=1 (RFP)</B></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>A=1 B=0: C=1 (RFP)</B></FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><FIGURE><FIGCAPTION><B>A=0 B=1: C=1 (RFP)</B></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>A=1 B=1: C=0 (GFP)</B></FIGCAPTION></FIGURE></DIV></DIV><TABLE><TBODY><TR><TH>Input A</TH><TH>Input B</TH><TH>Output: [GFP]-&gt;C=0   [RFP1, RFP2]-&gt;C=1</TH><TH>Output C</TH></TR><TR><TD>0</TD><TD>0</TD><TD>[i]   [a/i, a]≈[i, a]</TD><TD>1</TD></TR><TR><TD>0</TD><TD>1</TD><TD>[a/i]≈[i]   [a, i]</TD><TD>1</TD></TR><TR><TD>1</TD><TD>0</TD><TD>[b]≈[i]   [i, a]</TD><TD>1</TD></TR><TR><TD>1</TD><TD>1</TD><TD>[a]   [b, i]≈[i, i]</TD><TD>0</TD></TR></TBODY></TABLE><P>Due to time constraints, we were not able construct this gate and test it in the wet lab. However, <B>modeling</B> allowed us to qualitatively assess the functionality of logic gates in silico. After tuning some constants in order to reproduce basic activation and inhibition of a single transistor, we were able to reproduce more complex experiments (chaining two transistors, and simultaneous activation and inhibition). Finally, we were able to study the response of full logic gate, particularly the NAND gate.<B>To find out more, take a look at our <A href="https://2015.igem.org/Team:EPF_Lausanne/Modeling"><U>modeling page</U></A> and our <A href="https://2015.igem.org/Team:EPF_Lausanne/Results"><U>results page</U></A>!</B></P><H2>References</H2><P>[1] Bikard, D., Jiang, W., Samai, P., Hochschild, A., Zhang, F., &amp; Marraffini, L. A. (2013). Programmable repression and activation of bacterial gene expression using an engineered CRISPR-Cas system. Nucleic acids research, 41(15), 7429-7437.[2] Qi, L. S., Larson, M. H., Gilbert, L. A., Doudna, J. A., Weissman, J. S., Arkin, A. P., &amp; Lim, W. A. (2013). Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression. Cell, 152(5), 1173-1183.[3] Alec AK Nielsen &amp; Christopher A Voigt (2014). Multi-input CRISPR/Cas circuits that interface host regulatory network. Molecular systems biology, 10(11), 763.[4] <A href="https://www.addgene.org/CRISPR/guide/#PAM">Addgene about protospacer-adjacent motif (PAM)</A>[5]<A href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>: inducible pBad/AraC promoter[6] <A href="https://en.wikipedia.org/wiki/Logic_gate">Wikipedia article on Logic gates</A>[7]<A href="https://en.wikipedia.org/wiki/NAND_gate">Wikipedia article on the NAND gate</A></P></DIV><DIV id="design-yeast" class="tab-pane fade"><DIV class="row"><P>The story we are composing is the imitation of an electronic circuit, made of biological elements. Every biological element of this system has a role to play. Here are the members of the cast:</P></DIV><TABLE cellspacing="0" style="width:80%"><THEAD><TR><TH>Electronic transistor</TH><TH>Bio LOGIC in S. cerevisiae</TH></TR></THEAD><TBODY><TR><TD><B>Connector</B></TD><TD>Electrical wires</TD><TD>dCas9-VP64 protein</TD></TR><TR class="even"><TD><B>Transmitted information</B></TD><TD>Electrical voltage either high or low</TD><TD>gRNAs either activating or repressing</TD></TR><TR><TD><B>Receptor of information</B></TD><TD>Entry of next transistor, the base</TD><TD>Promoter CYC</TD></TR></TBODY></TABLE><H2>dCas9-VP64 as a connector</H2><P>The dCas9 protein fused to a RNA Polymerase recruiting element, here VP64, can be used to activate or repress gene expression [1]. This regulation depends on the region of the promoter that dCas9-VP64 binds to.</P><H2>gRNAs as the transmitted information</H2><DIV class="row"><DIV class="col-md-5"><P>Each gRNA «cassette» is designed as in fig.1 :
                     5' - DsRed2 – polyA - HH ribozyme – gRNA SDS – gRNA scaffold – HDV ribozyme – 3'.</P><P><UL><LI>DsRed2 is a fluorescent protein. It acts as a reporter gene for the production of gRNAs [1].</LI><LI> The polyA tail is a sequence of 50 adenines. This particular sequence aims to replace the MATLAT1 triple helix in mammalian cells [3]. It increases translational efficiency [4] and contributes to efficient mRNA progression away from the gene [5]. </LI><LI> The hammerhead (HH) ribozyme and the hepatitis delta virus (HDV) ribozyme both carry out self-cleavage after transcription, [2] and [3].</LI></UL></P><P>This design, under promoter ADH1, allows to recruit RNA polymerase II. This RNA polymerase allows production of multiple gRNAs from a single transcript, contrary to RNA polymerase III [3].</P></DIV><DIV class="col-md-7"><FIGURE><FIGCAPTION>Fig.1 - gRNA design adapted from [2] and [3]. </FIGCAPTION></FIGURE></DIV></DIV><H2>Bio-transistor</H2><DIV class="row"><DIV class="col-md-12 text-justify"><P>The gRNAs are sequences composed of a 20 nucleotides Specificity Determinant Sequence (SDS) and a 76 nucleotides scaffold. The 20 nucleotides SDS is complementary to a specific region of the promoter. It guides dCas9-VP64 to this specific region.</P><CENTER><FIGURE><FIGCAPTION>Fig.2 - CYC promoter adapted from [1]. TATA: TATA box. TSS: Transcription Start Site. KS: Kozak Sequence. </FIGCAPTION></FIGURE></CENTER><P>Gene expression is enhanced when the site targeted by dCas9-VP64 is located at the beginning or before the promoter, i.e. 25 nucleotides upstream from the TATA box and 50 nucleotides upstream from the transcrition start site. Gene expression is repressed when the targeted site is located on the promoter and in particular on the TATA box or the TSS region [1]. </P><P>We use the region of strongest activation and the region of strongest repression.</P><P>The region of strongest activation is c3. Binding of dCas9-VP64 to region c3 produces a threefold increase of fluorescence [1]. </P><CENTER><FIGURE><FIGCAPTION>Fig.3 - Activation in S. cerevisiae. </FIGCAPTION></FIGURE></CENTER><P>The strongest repression is obtained with c6 and c7 simultaneously. The reason is that a stronger inhibition is observed when two gRNAs bind the promoter. Binding of dCas9-VP64 to c6 and c7 produces a sevenfold decrease of fluorescence [1]. </P><CENTER><FIGURE><FIGCAPTION>Fig.4 - Repression in S. cerevisiae. </FIGCAPTION></FIGURE></CENTER><P>Since we focus on three regions of the promoter, we use these three gRNA SDS : c3, c6 and c7. Each gRNA has four different exemplaries: c3_0, c3_1, c3_2, c3_3 for gRNA c3. The gRNAs c6 and c7 have the same notation. Each of these 20 nucleotides sequences are randomly generated by two programs, coded by us, in order to respect two conditions. First, the human blaster program ensures that gRNAs don't bind anywhere on the human genome for safety reasons. Second, another blaster program ensures that gRNAs don't bind anywhere in the genome of S. cerevisiae. We do not want our synthetic gRNAs to bind anywhere else than on the desired promoter.</P><P>A key challenge in engineering transcriptional networks is to design orthogonal transcription factors or promoters. This means gRNAs must not interfere with one another. It has been shown that a single base-pair mismatch between the gRNA SDS and the promoter region is sufficient to prevent the binding of dCas9-VP64 [1]. We ensured to avoid cross-talking issues by synthesizing gRNA SDS that are very different from one another, differing by at least 10 nucleotides out of 20. </P></DIV></DIV><H2>Linking bio-transistor</H2><DIV class="row"><DIV class="col-md-12 text-justify"><P>Building a bio-transistor consists in activating or repressing the expression of a reporter gene, GFP here. Linking the bio-transistors means replacing the expression of GFP by the expression of another gRNA. </P><P>This other gRNA will then bind the next synthetic CYC promoter, and will activates or repress the expression of GFP. This is the first level of linking. </P><CENTER><FIGURE><FIGCAPTION>Fig.5 - First level of linking. In this example, the link between the two CYC promoter is an activation (c3). It can also be a repression (c6 &amp; c7), or activation and repression simultaneously (c3 &amp; c6 &amp; c7). </FIGCAPTION></FIGURE></CENTER></DIV></DIV><H2>Logic gate</H2><DIV class="row"><DIV class="col-md-12 text-justify"><P>The first level of linking allows to build one logic gate.For example, this is the NAND logic gate implemented in S. cerevisiae. We designed two more complex logic gates as seen under the tab &quot;Gate designs&quot; and in the section «Modeling». The design described below corresponds to the &quot;Gate simple&quot; design in &quot;Gate designs&quot;.</P><CENTER><FIGURE><FIGCAPTION>Fig.6 - NAND logic gate in S. cerevisiae. </FIGCAPTION></FIGURE></CENTER></DIV></DIV><H4>References</H4><P>[1] Farzadfard F, Perli SD, Lu TK. Tunable and Multifunctional Eukaryotic Transcription Factors Based on CRISPR/Cas. ACS Synth Biol. 2013 Sep 11. 10.1021/sb400081r PubMed 23977949
          [2] Gao Y, Zhao Y. Self-processing of ribozyme-flanked RNAs into guide RNAs in vitro and in vivo for CRISPR-mediated genome editing. J Integr Plant Biol. 2013 Dec 30. doi: 10.1111/jipb.12152. 10.1111/jipb.12152 PubMed 24373158
          [3] Nissim L, Perli SD, Fridkin A, Perez-Pinera P, Lu TK. Multiplexed and Programmable Regulation of Gene Networks with an Integrated RNA and CRISPR/Cas Toolkit in Human Cells. Mol Cell. 2014 May 14. pii: S1097-2765(14)00355-4. doi: 10.1016/j.molcel.2014.04.022. 10.1016/j.molcel.2014.04.022 PubMed 24837679
          [4] [4] Preiss T, Muckenthaler M, Hentze MW. Poly(A)-tail-promoted translation in yeast: Implications for translational control. 1998 Nov;4(11):1321-31. PubMed : 9814754.
          [5] Dower K1, Kuperwasser N, Merrikh H, Rosbash M. A synthetic A tail rescues yeast nuclear accumulation of a ribozyme-terminated transcript. RNA. 2004 Dec;10(12):1888-99. PubMed 15547135.

          </P></DIV><DIV id="background" class="tab-pane fade"><H1>Background</H1><H2>Programmable cells - Promises and Limitations</H2><DIV class="row"><DIV class="col-md-8"><P align="justify">The ability to process information is fundamental to life. Cells use complex gene regulatory networks to effectively respond to the myriad of signals they receive from both the outside environment and their internal metabolism. This information processing capability enables them to move around, communicate, reproduce - in one word, survive.</P><P align="justify">Since the early 2000s, researchers have sought to harness this capability by re-engineering cellular signal processing pathways for various biotechnology applications. By implementing rational, controllable logic elements in cells, researchers aim to transform living systems into engineered &quot;machines&quot; that may perform functions ranging from industrial production to biomedical therapies, bioremediation and energy production [1].</P><P align="justify">Up to date, various strategies and genetic parts have been used to implement information processing systems in cells. Many of these designs use the regulation of DNA expression as the adjustable signal in the circuit. A good example is the â€œrepressilatorâ€ built by Elowitz et al. in 2000 that uses three orthogonal repressor-promoter pairs : LacI, tetR and Î» cI to produce an oscillating signal [2]. Since then, multiple other repressors and activators (zinc-fingers, TALEs) have been used to target specific DNA sequences. Alternative strategies to engineer synthetic biological circuits are based on the control of RNA stability, RNA translation or protein-protein interactions as the basis for signal transmission.</P><P align="justify">The challenge today is to transition from the creation of small cellular genetic â€œdevicesâ€ to cells that efficiently perform logical functions observed in electronic circuits. This issue calls for innovative strategies and rational design [3]. The engineering-driven approach of synthetic biology which includes parts standardization, modular components, modeling and systematic strategies to create biological circuits with reliable and predictable behaviors - holds part of the solution. The other key to the problem is to increase the number of orthogonal genetic components that may be used for signal processing [4]. The characterization of new parts that do not cross-talk with the cellular machinery nor with other parts of the genetic circuit promises to improve the robustness and efficiency of future circuits.</P><P align="justify">Retroactivity is not the only limitation of current cellular signal processing circuits. Such circuits are slow: their response time can be measured in hours or even days. Furthermore, they are often unreliable and offer low signal-to-noise ratios. Low output signal may make connecting multiple circuits impossible. Another limiting factor is the fact that the transcription factors used in the circuit may be toxic for the cell and that the circuit itself may monopolize the cellâ€™s resources. Finally, circuit parts may behave unexpectedly in new genetic contexts [5]. These issues underline the huge task at hand when one dreams of transferring computing from a world of silicon into the realm of biology.</P></DIV><DIV class="col-md-4"><FIGURE><FIGCAPTION>a) Representation of the repressilator that was built by Elowitz and his collaborators. The construction was based on two independent plasmids: one containing the circuit and the other one the fluorescent reporter.b) Modeled oscillation (left side) and observed results (right side).Retrieved from Elowitz &amp; Liebler 2000. The original source is: <A href="https://commons.wikimedia.org/wiki/File:Repressilator_(representation_based_on_Elowitz_%26_Liebler_2000).png">Nature. 2000 Jan 20;403(6767):335-8.</A></FIGCAPTION></FIGURE></DIV></DIV><H2>dCas9 to the rescue!</H2><DIV class="row"><DIV class="col-md-10"><P align="justify">CRISPR stands for clustered, regularly interspaced, short palindromic repeat arrays. It plays the role of an immune system for bacteria by targeting and degrading foreign DNA. The CRISPR systems uses the Cas9 (CRISPR-associated) nuclease to introduce double-strand breaks to DNA sequences that are complementary to its guide RNA (gRNA). Catalytically dead Cas9 (dCas9) lacks the ability to cleave DNA and may act as programmable transcription regulator - by either preventing the binding of the RNA polymerase (RNAP) to the targeted DNA - or as an activator when it is fused to a RNAP recruiting element (the omega subunit of RNAP in E. coli and VP64 in Yeast) [7].</P><P align="justify">The main advantage of this system is that many different orthogonal pairs of synthetic promoters and targeting gRNAs can be designed. In addition, synthetic promoters may be built with numerous regulating binding sites in order to receive a variety of inputs, both activating and inhibiting. These properties make CRISPR a useful addition to the synthetic biologistâ€™s toolbox. Its versatility may help build more complex and extendable genetic circuits, as has been already shown in recent publications. Indeed, this technology has been used to create simple logic circuits - such as NOR gates and 3-gate circuits [8].</P></DIV><DIV class="col-md-2"><FIGURE><FIGCAPTION>dCas9-Ï‰ structure</FIGCAPTION></FIGURE></DIV></DIV><H2>Biological Transistors</H2><P align="justify">Given the numerous advantages of this system, we decided to attempt to represent binary signals with the two 'elementary' CRISPR/Cas9- dependent operations described above. A system composed of Cas9 with appropriate sets of gRNAs and promoters should be able to act as an ON/OFF switch. In electronic circuits, the switches that control the flow of electricity are named transistors.  In our circuit, the output, represented by the transcription of specific genes, is controlled by gRNA-targeted dCas9, where gRNA is the input. The promoter followed by a specific gene represents a transistor.Transistors are the fundamental building block of electronic circuits. Assembled into logic gates, they enable logic operations to be performed (the computation of true/false answers). Logic gates can be linked together to form complex digital circuits.</P><P align="justify">dCas9 may be used to build logic circuits that are orthogonal, modular and reusable. Such circuits should enable scientists to implement more complex logic functions in biological systems. We hope to explore the viability of dCas9 circuits as the transformative tool to rewire genetic networks it may well prove to be.</P><H2>References</H2><P>[1] Brophy, J. A., &amp; Voigt, C. A. (2014). Principles of genetic circuit design. Nature methods, 11(5), 508-520.[2] A Synthetic Oscillatory Network of Transcriptional Regulators; Michael Elowitz and Stanislas Leibler; Nature. 2000 Jan 20;403(6767):335-8.[3] Purnick, P. E., &amp; Weiss, R. (2009). The second wave of synthetic biology: from modules to systems. Nature reviews Molecular cell biology, 10(6), 410-422.[4] Bradley, R. W., &amp; Wang, B. (2015). Designer cell signal processing circuits for biotechnology. New biotechnology.[5] Brophy, J. A., &amp; Voigt, C. A. (2014). Principles of genetic circuit design. Nature methods, 11(5), 508-520.[6] Sashital, D.G., Wiedenheft, B. &amp; Doudna, J.A. Mechanism of foreign DNA selection in a bacterial adaptive immune system. Mol. Cell 46, 606â€“615 (2012)[7] Bikard, D., Jiang, W., Samai, P., Hochschild, A., Zhang, F., &amp; Marraffini, L. A. (2013). Programmable repression and activation of bacterial gene expression using an engineered CRISPR-Cas system. Nucleic acids research, 41(15), 7429-7437.[8] Nielsen, A. A., &amp; Voigt, C. A. (2014). Multiâ€input CRISPR/Cas genetic circuits that interface host regulatory networks. Molecular systems biology, 10(11), 763.</P></DIV></DIV></DIV><CENTER><BUTTON type="button" class="btn btn-primary btn-lg outline"><A href="https://2015.igem.org/Team:EPF_Lausanne/Design">Continue the tour!</A></BUTTON></CENTER><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE></DIV></DIV></DIV></DIV></BODY></HTML>