<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPF_Lausanne_PolicyAndPractices skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPF Lausanne/PolicyAndPractices</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-header"><BUTTON class="navbar-toggle" type="button" data-toggle="collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="main-menu"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2015.igem.org/Team_EPF_Lausanne/Project/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Modelling">Modelling</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievements </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Vivo">In vivo</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Silico">In silico</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Judging">Judging</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Policy and Practices </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Policy/Ethic">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Policy/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Policy/Human">Human Practices</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Ecoli">E. Coli</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Yeast">Yeast</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Team/Meet">Meet us</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Timeline">Timeline</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Team/Acknowledgements">Acknowledgements</A></LI></DIV><DIV class="page-h promotion"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H3>Human Practices</H3></DIV></DIV></DIV></DIV><DIV class="prot-section"><DIV class="row"><DIV class="col col-md-3"><NAV id="affix-nav" class="sidebar hidden-sm hidden-xs"><UL class="nav nav-pills nav-stacked" data-spy="affix" data-offset-top="200" data-offset-bottom="600"><LI class="active"><A href="#ethics">Ethics</A></LI><UL><LI><A href="#safety">Safety</A></LI><LI><A href="#legal">Legal</A></LI><LI><A href="#security">Security</A></LI><LI><A href="#public">Public</A></LI></UL><LI><A href="#pdcas9wsgrna">Assemble pdCas9-w-sgRNAs</A></LI><UL><LI><A href="#openpdcas9w">Open pdCas9-w by restriction digest</A></LI><LI><A href="#pcrsgrna">Amplify sgRNA fragments by PCR</A></LI><LI><A href="#gibsonpdcas9wsgrna">Assemble pdCas9-w-sgRNA constructs by Gibson assembly</A></LI></UL><LI><A href="#">Assemble pWJ89alt1</A></LI></UL></NAV></DIV><DIV class="col-sm-9"><DIV id="pdcas9w" class="panel"><H1>Assemble pdCas9-w</H1><P><SMALL>pdCas9-w was constructed using PCR and Gibson assembly from the following plasmids:</SMALL></P><UL><LI>pdCas9 (pdCas9-bacteria, plasmid #44249, Addgene): dCas9 under a Tetracyclin inducible promoter + Chlorampheicol resistance gene</LI><LI>pWJ66 (pWJ66, plasmid #46570, Addgene): tracrRNA + w-dCas9 (dCas9 fused at its C-terminal to rpoZ, which encodes for the w subunit of RNA polymerase) + CRISPR array</LI></UL><P><SMALL>pdCas9 was opened by PCR and the w subunit was extracted from pWJ66 by pCR. The w subunit was then fused to the C-terminal of dCas9 by Gibson assembly, the final product is pdCas9-w.A site-directed mutagenesis was done on this plasmid to eliminate an EcoRI restriction site. This will allow us to submit pdCas9-w as a BioBrick.</SMALL></P></DIV><DIV id="PCRpdcas9" class="panel"><H1><SMALL>Assemble pdCas9-w</SMALL>Open pdCas9 by PCR</H1><P><SMALL>We received plasmid pdCas9 in bacteria and did a Miniprep (cf. Protocols) on overnight cultures to isolate it. This step consists of linearizing pdCas9 by PCR.</SMALL></P><H2>Materials and method</H2><UL><LI>20 µl Phusion PCR (cf. Protocols) of pdCas9 with primers f_Gbs_pdCas9 and r_Gbs_pdCas9</LI><LI>PCR product purification (cf. Protocols)</LI><LI>Agarose gel electrophoresis of purified PCR products</LI></UL><H2>Results</H2><DIV id="divright1"><P><SMALL>Linearized pdCas9-w is expected to be 6705 bp.The first try of this PCR was unsuccessful (gel not shown here). For our second try, we tested many parameters: HF vs. GC buffer, different annealing temperatures and different extension times. This time, many, but not all, of our samples were successfully amplified (cf. figure 1). The difficulty of this PCR is probably due to the fact that the size of the ampicon is very long, almost 7 kb.For next steps, sample from lane 1 (cf. figure 1) was used.</SMALL></P></DIV></DIV><DIV id="PCRpwj66" class="panel"><H1><SMALL>Assemble pdCas9-w</SMALL>Extract w subunit from pWJ66 by PCR</H1><P><SMALL>We received plasmid pWJ66 in bacteria and did a Miniprep (cf. Protocols) on overnight cultures to isolate it. This step consists of extracting rpoZ, the w subunit, from pW66 by PCR.</SMALL></P><H2>Materials and method</H2><UL><LI>20 µl Phusion PCR (cf. Protocols) of pWJ66 using primers f_Gbs_w and r_Gbs_w.</LI><LI>PCR product purification (cf. Protocols)</LI><LI>Agarose gel electrophoresis of purified PCR products</LI></UL><H2>Results</H2><DIV id="divleft2"><P><SMALL>Successful PCR reactions are expected to yield 340 bp fragments.As seen on gel, PCR was succesful for sample in lane 1.</SMALL></P></DIV></DIV><DIV id="gibsonpdcas9w" class="panel"><H1><SMALL>Assemble pdCas9-w</SMALL>Gibson assembly of pdCas9-w</H1><P><SMALL>This step consists of fusing the w subunit to dCas9 by Gibson assembly, using the PCR products from previous steps. We transformed cells with the Gibson assembly product and tested for colonies that contain the construct by colony PCR, restriction digest and sequencing.</SMALL></P><H2>Materials and method</H2><UL><LI>Gibson assembly (cf. Protocols) with purified PCR products:</LI><UL><LI>Open pdCas9: 0.02 pmol = 95 ng</LI><LI>w subunit extracted from pWJ66: 0.06 pmol = 12.5 ng</LI></UL><LI>Transformation (cf. Protocols) of ultra-competent DH5a cells (NEB) with Gibson assembly product, spreading on Chloramphenicol plates</LI><LI>Colony PCR (cf. Protocols) of colonies from plate used for culture of transformed cells with primers f_Gbs_w and r_Scq_pdCas9_w_sgRNA primers.</LI><LI>Miniprep (cf. Protocols) of overnight liquid cultures in 5 mL LB + Chloramphenicol of colonies from plate used for culture of transformed cells to isolate pdCas9-w plasmids</LI><LI>Restriction digest (Cf. Protocols) of pdCas9-w plasmids with BamHI and KpnI seperately</LI><LI>Sequencing (Microsynth) of one colony for which Gibson assembly worked according to colony PCR and restriction analysis</LI></UL><H2>Results</H2><H3>Colony PCR</H3><DIV id="divleft3"><P><SMALL>Primers were placed such as amplicons are 666 bp if Gibson assembly worked and 396 bp if the plasmid self-ligated.Lane &quot;C&quot; is a negative control: PCR was run with all components except template DNA, ie. cells. It is empty, this means there is no contamination.Gibson assembly seems to have worked for some of our samples (cf. figure 3 (a, b, c)).To avoid working with too many samples, we kept the ones from lanes 16, 22, 25, 31, 37 and 43 for the Minipreps and restriction digest. We did an overnight liquid culture of these colonies.</SMALL></P></DIV><H3>Restriction digest</H3><DIV id="divright1"><P><SMALL>pdCas9-w samples from different colonies are present on gels in triplicates: in the following order:</SMALL></P><OL><LI>Undigested: expected to yield 7 kb circular plasmid (migrates faster than linear fragments of the same size)</LI><LI>Digested by BamHI: expected to yield two fragments of 6147 bp and 834 bp if insert is present or one 6705 bp fragment if it is not</LI><LI>Digested by KpnI: expected to yield two fragments of 45334 bp and 2447 bp if insert is present or one 6705 bp fragment if it is not</LI></OL><P><SMALL>BamHI and KpnI are both unique cutters in pdCas9 (without the insert) and double cutters in pdCas9-w (with the insert).Too much ladder was loaded so it is difficult to estimate the size of the fragmnts. The smaller fragments are also very difficult to see.By looking at the relative heights, we can say that all colonies seem to have the w subunit insert. There is only the undigested sample for colony 16 that is not visible, probably due to an error while loading the gel. Colony 22 was kept for next steps, it was stored in a glycerol stock (c.f. Protocols).</SMALL></P></DIV><H3>Sequencing</H3><P><SMALL>As dCas9-w is very long, only part of it was sequenced (the w subunit and its surrounding base pairs). No mutations were detected.</SMALL></P></DIV><DIV id="mutagenesis" class="panel"><H1><SMALL>Assemble pdCas9-w</SMALL>Site-directed mutagenesis of dCas9-w</H1><P><SMALL>It was noticed after assembly of pdCas9-w that dCas9 contains an EcoRI restriction site, which makes it &quot;un-BioBrick-able&quot;. Since it is an important part for our project, we decided to remove the restriction site by site-directed mutagenesis to be able to submit it as a functional BioBrick.</SMALL></P><H2>Materials and method</H2><P><SMALL>Coming soon</SMALL></P><H2>Procedure</H2><P><SMALL>Coming soon</SMALL></P></DIV></DIV></DIV></DIV><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE></DIV></DIV></DIV></DIV></BODY></HTML>