<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPF_Lausanne_Ari_Notebook_Ecoli skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPF Lausanne/Ari/Notebook/Ecoli</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="site-header"><DIV class="container"><DIV class="row"><NAV class="navbar navbar-default navbar-static-top"><DIV class="navbar-header"><BUTTON class="navbar-toggle" type="button" data-toggle="collapse" data-target="#main-menu"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="main-menu"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2015.igem.org/Team_EPF_Lausanne/Project/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Modelling">Modelling</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievements </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Vivo">In vivo</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Silico">In silico</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Judging">Judging</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Parts">Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Policy and Practices </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Policy/Ethic">Ethics</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Policy/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Policy/Human">Human Practices</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Ecoli">E. Coli</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Yeast">Yeast</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Team/Meet">Meet us</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Timeline">Timeline</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Team/Aknowledgements">Aknowledgements</A></LI></DIV></NAV></DIV></DIV></DIV><DIV class="page-h promotion"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H3>E. Coli Laboratory Notebook</H3></DIV></DIV></DIV></DIV><DIV class="prot-section"><DIV class="row"><DIV class="col col-md-3"><NAV id="affix-nav" class="sidebar hidden-sm hidden-xs"><UL class="nav nav-pills nav-stacked" data-spy="affix" data-offset-top="200" data-offset-bottom="600"><LI class="active"><A href="#PCRpdcas9">Assemble pdCas9-w</A></LI><UL><LI><A href="#PCRpdcas9">Open pdCas9 by PCR</A></LI><LI><A href="#PCRpwj66">Extract w subunit from pWJ66 by PCR</A></LI><LI><A href="#gibsonpdcas9w">Gibson assembly of pdCas9-w</A></LI></UL><LI><A href="#">Assemble pdCas9-w-sgRNAs</A></LI><LI><A href="#">Assemble pWJ89alt1</A></LI></UL></NAV></DIV><DIV class="col-sm-9"><DIV id="PCRpdcas9" class="panel"><H1>Assemble pdCas9-w: Open pdCas9 by PCR<P><SMALL>We received plasmid pdCas9 in bacteria. We did a Miniprep (cf. Protocols) on overnight cultures to isolate it.pdCas9 is a plasmid containing dCas9 under a Tetracyclin inducible promoter and a Chloramphenicol resistance gene, available on Addgene under the name pdCas9-bacteria.We opened this plasmid by PCR to be able use later on in a Gibson assembly.</SMALL></P></H1><H2>Materials and method</H2><UL><LI>20 µl Phusion PCR (cf. Protocols) of 1 ng pdCas9 with appropriate primers, testing parameters such as HF vs. GC buffer, annealing temperatures and extension times</LI><LI>PCR product purification (cf. Protocols)</LI><LI>Agarose gel electrophoresis of purified PCR products</LI></UL><H2>Results</H2><P><SMALL>Linearized pdCas9-w is expected to be 6705 bp.After testing many parameters, we were able to linearize the plasmid succesfully, as seen on gel above.For further uses, sample from lane 1 was used. It was prepared with HF buffer and the following thermocycling settings:</SMALL></P><TABLE width="70%" align="center"><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>40 seconds</TD></TR><TR><TD>25 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>15 seconds</TD></TR><TR><TD>Annealing</TD><TD>59°C</TD><TD>22 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>2 minutes 30 seconds</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>7 minutes</TD></TR><TR><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE></DIV><DIV id="PCRpwj66" class="panel"><H1>Assemble pdCas9-w: Extract w subunit from pWJ66 by PCR</H1><P><SMALL>We received plasmid pWJ66 in bacteria. We did a Miniprep (cf. Protocols) on overnight cultures to isolate it.pWJ66 is a plasmid containing dCas9 fused at its C-terminal to the w subunit of RNA polymerase as well as a tracrRNA gene and a CRISPR array gene, available on Addgene.We extracted the w subunit to fuse it to our own dCas9 by Gibson assembly.</SMALL></P><H2>Materials and method</H2><UL><LI>20 µl Phusion PCR (cf. Protocols) with 1 ng pWJ66 using appropraite primers, HF buffer and following thermocycling settings:</LI><TABLE width="70%" align="center"><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR><TD>30 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD></TR><TR><TD>Annealing</TD><TD>62°C</TD><TD>15 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>15 seconds</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>7 minutes</TD></TR><TR><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE><LI>PCR product purification (cf. Protocols)</LI><LI>Agarose gel electrophoresis of purified PCR products</LI></UL><H2>Results</H2><P><SMALL>Successful PCR reactions are expected to yield 340 bp fragments.As seen on gel, PCR was succesful for sample in lane 1.</SMALL></P></DIV><DIV id="gibsonpdcas9w" class="panel"><H1>Assemble pdCas9-w: Gibson assembly of pdCas9-w</H1><P><SMALL>Using PCR products from experiments described above, we fused the w subunit of DNA polymerase to dCas9 by Gibson assembly.</SMALL></P><H2>Materials and method</H2><UL><LI>Gibson assembly (cf. Protocols) with purified PCR products:</LI><UL><LI>Open pdCas9: 0.02 pmol = 95 ng</LI><LI>w subunit extracted from pWJ66: 0.06 pmol = 12.5 ng</LI></UL><LI>Transformation (cf. Protocols) of ultra-competent DH5a cells (NEB) with Gibson assembly product, spreading on Chloramphenicol plates</LI><LI>Colony PCR (cf. Protocols) of colonies from plate used for culture of transformed cells with appropriate primers and following thermocycling settting:</LI><TABLE width="70%" align="center"><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>30 seconds</TD></TR><TR><TD>30 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR><TD>Annealing</TD><TD>56°C</TD><TD>35 seconds</TD></TR><TR><TD>Extension</TD><TD>68°C</TD><TD>80 seconds</TD></TR><TR><TD>Final Extension</TD><TD>68°C</TD><TD>5 minutes</TD></TR><TR><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE><LI>Isolate pdCas9-w plasmids by Miniprep (cf. Protocols) of overnight liquid cultures in 5 mL LB + Chloramphenicol of colonies from plate used for culture of transformed cells</LI><LI>Restriction digest (Cf. Protocols) of pdCas9-w plasmids</LI><LI>Sequencing (Microsynth) of one colony for which Gibson assembly worked according to colony PCR and restriction analysis</LI></UL><H2>Results</H2><H3>Gibson assembly and transformation</H3><P><SMALL>Almost as many colonies grew in our sample as in the self-ligation control (Gibson protocol with pdCas9 but no w subunit). This may mean that most of the colonies in our sample are actually self-ligation and that the Gibson assembly did not work.However, the transformation control (pUC19, NEB) worked very well.</SMALL></P><H3>Colony PCR</H3><P><SMALL>Primers were placed such as amplicons are 666 bp if Gibson assembly worked and 396 bp if the plasmid self-ligated.Lane &quot;C&quot; is a negative control, PCR was run with all components except template DNA, ie. cells. It is empty, this means there is no contamination.As seen on gels, Gibson assembly may have worked for samples in lanes 5, 6, 16, 22, 25, 26, 27, 28, 29, 30, 31, 37, 38, 40, 41 and 44.To avoid working with too many samples, we kept the ones from lanes 16, 22, 25, 31, 37 and 43 for the Minipreps and restriction digest. We did an overnight liquid culture of the corresponding colonies.</SMALL></P><H3>Restriction digest</H3><P><SMALL>Coming soon</SMALL></P></DIV></DIV></DIV></DIV><DIV class="fourth-section"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H2>Still under construction</H2></DIV></DIV></DIV></DIV><FOOTER class="footer-distributed"><DIV class="footer-center"><DIV><P><SPAN>Route Cantonale</SPAN> Lausanne, Suisse</P></DIV><DIV><P><A href="mailto:iGEM2015dreamteam@groupes.epfl.ch">Contact us</A></P></DIV></DIV><DIV class="footer-right"><P class="footer-company-about"><SPAN>About us</SPAN>
                    Team of a dozen biotechnology students that will attempt to change the world
                </P></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>