<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPF_Lausanne_Notebook_Yeast skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPF Lausanne/Notebook/Yeast</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE></P><DIV class="container"><NAV class="collapse navbar-collapse navHeaderCollapse" role="navigation"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Design">Prototype Design</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Interlab">Interlab Study</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievements </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Judging">Judging</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet Lab </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Ecoli">E. coli notebook</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Yeast">S. cerevisiae notebook</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Practices">Human Practices</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Team/Meet">Meet us</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Collaborations">Collaborations</A></LI><LI><A href="https://igem.org" target="_blank">iGEM</A></LI></NAV></DIV><DIV class="classy-bar"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H3>saccharomyces cerevisiae</H3></DIV></DIV></DIV></DIV><DIV class="prot-section"><DIV class="row"><DIV class="col col-md-2 scrollspy"><UL id="nav" class="nav hidden-xs hidden-sm" data-spy="affix"><LI class="active"><A href="#express_dCas9-VP64">Express dCas9-VP64</A></LI><LI><A href="#integrate_pTPGI_dCas9_VP64">Integrate pTPGI_dCas9_VP64</A></LI></UL><LI><A href="#assemble_reporter_plasmid">Assemble reporter plasmid</A></LI><LI><A href="#linearise_reporter_plasmid">Linearise the plasmid by PCR</A></LI><LI><A href="#synthesize_promoters">Synthesize promoters</A></LI><LI><A href="#PCRoverlaps_prom">Add Gibson overlaps to CYC promoters by PCR</A></LI><LI><A href="#gibsonprom">Gibson assembly of promoters in reporter plasmid</A></LI><LI><A href="#Assemble_gRNAs">Assemble gRNAs</A></LI><LI><A href="#Linearize_p405ADH1">Linearize p405ADH1</A></LI><LI><A href="#synthesize_gRNAs">Synthesize gRNA cassettes</A></LI><LI><A href="#PCRoverlaps_gRNAs">Add Gibson overlaps by PCR</A></LI><LI><A href="#gibson_gRNAs">Gibson assembly of gRNAs</A></LI><LI><A href="#activation-repression">Test activation-repression</A></LI><LI><A href="#integrate">Integration</A></LI><LI><A href="#test_activ_inhib">Measures</A></LI><LI><A href="#logic_gate">Build the logic gate</A></LI></DIV><DIV class="col-md-offset-1 col-md-9"><P><I>Note that all experiments were done by EPFL iGEM team members.</I></P><SECTION id="integrate_pTPGI_dCas9_VP64" class="panel"><H1><SMALL>Express dCas9-VP64</SMALL>Integrate pTPGI_dCas9_VP64</H1><P>We received plasmid pTPGI_dCas9_VP64 from Addgene. The plasmid was found from the article &quot;Tunable and multifunctional eukaryotic transcription factors based on Crispr/Cas&quot;. After glycerol stocks and miniprep, we performed a restriction analysis to check the identity of our plasmid. We linearised the plasmid, in order to integrate it into yeast genome. Only our fourth trial to integrate the plasmid was successful.</P><H2>Materials and methods</H2><UL><LI>Glycerol stocks</LI><LI>Miniprep</LI><LI>Restriction analysis</LI><LI>Polymerase Chain Reaction</LI><LI>Yeast integration</LI><LI>For more details about our procedures, see <A target="_blank" href="https://static.igem.org/mediawiki/2015/3/30/EPF_Lausanne_Journal_integrate_dCas9-VP64.pdf">here</A>.</LI></UL><H2>Results</H2><DIV class="row"><DIV class="col-md-6"><FIGURE><FIGCAPTION>Fig.1a - Restriction analysis of pTPGI_dCas9_VP64.</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><P>We used four different sets of enzymes for the restriction analysis. Linearized pTPGI_dCas9_VP64 is expected to be 10'987 bp. We observe that the gel (fig.1a) corresponds to the <A target="_blank" href="https://static.igem.org/mediawiki/2015/2/2b/EPF_Lausanne_Expected_Gel_dCas9.pdf">expected one</A>.
                        The plasmid was linearised both with EagI HF and NotI HF prior to integration. We integrated each linearised plasmid to obtain two different yeast strains.
                        </P></DIV></DIV></SECTION><SECTION id="linearise_reporter_plasmid" class="panel"><H1><SMALL>Integrate reporter plasmid</SMALL>Linearise reporter plasmid</H1><P>We received the plasmid pCYC1m_yeGFP from Addgene. The plasmid was found from the article &quot;Tunable and multifunctional eukaryotic transcription factors based on Crispr/Cas&quot;.</P><H2>Materials and methods</H2><UL><LI><B>PCR</B> (cf. Protocols). After glycerol stocks and minipreps. we linearised the plasmid by PCR according to the following <A target="_blank" href="https://static.igem.org/mediawiki/2015/d/d3/EPF_Lausanne_Linearise_pCYC_yeGFP.pdf">procedure</A>.</LI></UL><H2>Results</H2><DIV class="row"><DIV row="col-md-6"><P>Linearized pCYC1m_yeGFP is expected to be 5'485 bp. Running an agarose gel electrophoresis allowed to verify that we had linearised the plasmid.</P></DIV></DIV></SECTION><SECTION id="synthesize_promoters" class="panel"><H1><SMALL>Integrate reporter plasmid</SMALL>Synthesize promoters</H1><P>From the article &quot;Tunable and multifunctional eukaryotic transcription factors based on Crispr/Cas&quot;, we learnt that different regions on the promoters could lead to activation or inhibition when dCas9-VP64 was bound. We chose to modify the region of strongest activation, named c3, and the two regions that lead to the strongest inhibition, c6 and c7. We synthesized promoters CYC_0, CYC_1, CYC_2 and CYC_3. They only differ between one another by the three regions c3, c6 and c7. The promoter CYC_0 is the original promoter, already present in the plasmid pCYC1m_yeGFP.</P><H2>Results</H2><DIV class="row"><DIV class="col-md-6"><FIGURE><FIGCAPTION>Fig. 2 - Synthesized CYC promoters</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><P>Four different promoters according to Fig. 2.</P></DIV></DIV></SECTION><SECTION id="PCRoverlaps_prom" class="panel"><H1><SMALL>Integrate reporter plasmid</SMALL>Add Gibson overlaps by PCR</H1><P>We amplified the promoters CYC_0, 1, 2, 3 by PCR adding the Gibson overlaps in order to assemble each fragment in the linearised plasmid pCYC1m_yeGFP.</P><H2>Materials and methods</H2><UL><LI><B>PCR</B> (cf. Protocols). The PCR was performed according to the following <A target="_blank" href="https://static.igem.org/mediawiki/2015/7/74/EPF_Lausanne_Add_Gibson_overlaps_to_CYC-1%2C2%2C3.pdf">procedure</A>. The PCR for CYC_3 was repeated a <A target="_blank" href="https://static.igem.org/mediawiki/2015/3/3e/EPF_Lausanne_NotebookYeast_AddGibsonOverlapsCYC3_2nd.pdf">second time</A>, and a <A target="_blank" href="https://static.igem.org/mediawiki/2015/9/9a/EPF_Lausanne_NotebookYeast_AddGibsonOverlapsCYC3_3rd.pdf">third time</A>, due to a frameshift detected by sequencing.</LI></UL><H2>Results</H2><DIV class="row"><DIV class="col-md-6"><FIGURE><FIGCAPTION>Fig.3 - PCR of CYC promoters</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><P>On the gel electrophoresis that we ran after PCR, we observe the three CYC fragments at the right height (Fig.3). These fragments are then used for the Gibson assembly in the plasmid pCYC_yeGFP.</P></DIV></DIV></SECTION><SECTION id="gibsonprom" class="panel"><H1><SMALL>Integrate reporter plasmid</SMALL>Gibson assembly of CYC promoters</H1><P>The promoters CYC_0, 1, 2, 3 have overlaps from previous PCR. They were assembled in the linearised plasmid pCYC1m_yeGFP.</P><H2>Materials and methods</H2><UL><LI><B>Gibson assembly</B> was performed according to the following <A target="_blank" href="https://static.igem.org/mediawiki/2015/7/74/EPF_Lausanne_NotebookYeast_GibsonAssemblyCYC.pdf">procedure</A>. The Gibson assembly for CYC_3 was repeated a <A target="_blank" href="https://static.igem.org/mediawiki/2015/4/41/EPF_Lausanne_NotebookYeast_GibsonAssemblyCYC3_2nd.pdf">second time</A>, and a <A target="_blank" href="https://static.igem.org/mediawiki/2015/c/c4/EPF_Lausanne_NotebookYeast_GibsonAssemblyCYC3_3rd.pdf">third time</A>.</LI><LI><B>Colony PCR </B> was performed according to the following <A target="_blank" href="https://static.igem.org/mediawiki/2015/e/e4/EPF_Lausanne_NotebookYeast_colonyPCR1.pdf">procedure</A>. The colony PCR was repeated a <A target="_blank" href="https://static.igem.org/mediawiki/2015/5/53/EPF_Lausanne_NotebookYeast_colonyPCR2.pdf">second time</A> for CYC1, a <A target="_blank" href="https://static.igem.org/mediawiki/2015/a/a2/EPF_Lausanne_NotebookYeast_colonyPCR3.pdf">third</A>, <A target="_blank" href="https://static.igem.org/mediawiki/2015/b/b9/EPF_Lausanne_NotebookYeast_colonyPCR4.pdf">fourth</A>, <A target="_blank" href="https://static.igem.org/mediawiki/2015/a/aa/EPF_Lausanne_NotebookYeast_colonyPCR5.pdf">fifth</A> and <A target="_blank" href="https://static.igem.org/mediawiki/2015/4/48/EPF_Lausanne_NotebookYeast_colonyPCR6.pdf">sixth</A> time for CYC3. </LI></UL><H2>Results</H2><DIV class="row"><DIV class="col-md-6"><FIGURE><FIGCAPTION>Fig.4 - 1st colony PCR of CYC promoters</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><P>The results of the Gibson assembly are given by a colony PCR. The amplicon should be 308 bp since we used primers f_Gbs_CYC and r_Gbs_CYC. We observe on Fig. 4 that CYC2 on lane 5 and CYC3 on lanes 8 and 9 migrated at the right height. These colonies can be sent to sequencing. Colony PCR was repeated to find CYC1.
                    After colony PCR, sequencing indicated a frameshift in the CYC3 construct. This is why we repeated the whole process, from the start, for CYC3. </P></DIV></DIV></SECTION><SECTION id="Linearize_p405ADH1" class="panel"><H1><SMALL>Integrate and express gRNAs</SMALL>Linearize p405ADH1</H1><P>In order to test the effects of single gRNAs, we opted for a strong constitutive promoter - ADH1 (coming from our backbone plasmid, p405ADH1) - to be placed just before the gRNA expressing cassette.</P><H2>Materials and methods</H2><UL><LI><B>Polymerase Chain Reaction</B> (cf. Protocols). After glycerol stocks and minipreps, the PCR with primers f_Gbs_ADH1 and r_Gbs_ADH1 was performed according to the following <A target="_blank" href="https://static.igem.org/mediawiki/2015/9/94/EPF_Lausanne_NotebookYeast_linearADH1.pdf">procedure</A>.</LI></UL><H2>Results</H2><DIV class="row"><DIV class="col-md-6"><FIGURE><FIGCAPTION>Fig.6 - ADH1_Linearized</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><P><SMALL> The size of p405ADH1 plasmid is 7.2 kbp. Altough it is not in frame with the ladder, a different migration rate is observed between the PCR products (p405ADH1 linearized) and the circular plasmid on the 10th lane. Therefore, we assumed that the PCR was successful.</SMALL></P></DIV></DIV></SECTION><SECTION id="synthesize_gRNAs" class="panel"><H1><SMALL>Integrate and express gRNAs</SMALL>Synthesize gRNA-expressing cassette</H1><DIV class="col-md-6"><FIGURE><FIGCAPTION>Fig.7 - Diagram from a reference paper </FIGCAPTION></FIGURE></DIV><P>The design of the gRNA-expressing cassettes is inspired by the article &quot;Multiplexed and programmable regulation of gene networks with an integrated RNA and CRISPR/Cas toolkit in human cells&quot;. The activating gRNA, c3, and the two repressing gRNAs, c6 and c7, were synthesized. Each gRNA is fused to the HH ribozyme on the 5' end, and to the HDV ribozyme on the 3' end.</P><H2>Results</H2><DIV class="row"><DIV class="col-md-6"><FIGURE><FIGCAPTION>Fig.8 - Synthesized gRNA-expressing cassettes</FIGCAPTION></FIGURE></DIV><DIV class="col-md-6"><P>According to Fig.8, we synthesized the activating gRNA cassette c3 and the two repressing gRNA cassettes c6 and c7. Each of these gRNAs: c3, c6, c7, is in four exemplaries. The numbering 0, 1, 2, 3, corresponds to promoter CYC_0, CYC_1, CYC_2, CYC_3.
                    </P></DIV></DIV></SECTION><SECTION id="PCRoverlaps_gRNAs" class="panel"><H1><SMALL>Integrate and express gRNAs</SMALL>Add Gibson overlaps by PCR</H1><P>The gRNA-expressing cassettes, c3_0, c3_1, c3_2, c3_3, c6_0, c6_1, c6_2, c6_3 and c7_0, c7_1, c7_2, c7_3, were synthesized. In this step, overlaps for Gibson assembly were added by PCR. In order to verify the expression of the gRNAs cassettes, we used fluorescent protein DsRed2 as a reporter gene. In parallel to the gRNA cassettes, the DsRed2 gene was PCR-extracted from plasmid CMVp-dsRed2-Triplex-HHRibo-gRNA1-HDVRibo-pA from Addgene. Overlaps for Gibson assembly were added by PCR. A synthetic polyA tail followed by a part of the Hammerhead ribozyme sequence was included in the reverse primer.</P><H2>Materials and methods</H2><UL><LI><B>Polymerase Chain Reaction</B> (cf. Protocols). Gibson overlaps of DsRed2, c3, c6 and c7 cassettes were added according to the following <A target="_blank" href="https://static.igem.org/mediawiki/2015/2/21/EPF_Lausanne_NotebookYeast_PCRoverlaps_DsRed2_c3_c6_c7.pdf">procedure</A>. As we can see from the gel electrophoresis (Results), we observe no amplicon for c3_0, c3_3 and DsRed2. This PCR had to be repeated (several times) for the three missing fragments.</LI><LI>A second PCR was necessary to add specific overlaps to c3 and c6. <A target="_blank" href="https://static.igem.org/mediawiki/2015/e/eb/EPF_Lausanne_NotebookYeast_2ndPCRoverlaps_DsRed2_c3_c6_c7.pdf">Here</A> is the procedure for this second PCR.</LI></UL><H2>Results</H2><DIV class="row"><DIV class="col-md-6"><FIGCAPTION>Fig.9 - 1st PCR of gRNA cassettes and DsRed2 in order to add Gibson overlaps</FIGCAPTION></DIV><DIV class="col-md-6"><P>From the gel electrophoresis, we observe that almost every fragment migrated at the right height (207 bp for c3, 208 bp for c6, 308 bp for c7); except c3_0, c3_3 and DsRed2 whose PCR will be repeated.
                        The second PCR produced the amplicons of c3 and c6, that are ready for the Gibson assembly.</P></DIV></DIV></SECTION><SECTION id="gibson_gRNAs" class="panel"><H1><SMALL>Integrate and express gRNAs</SMALL>Gibson assembly of gRNA cassettes</H1><P>The gRNA cassettes c3_0, c3_1, c3_2, c6_0, c6_1, c6_2, c7_0, c7_1, c7_2 have overlaps from previous PCR. They were assembled in the linearised plasmid p405ADH1.

            </P><H2>Materials and methods</H2><UL><LI><B>Gibson assembly</B> was performed according to the following <A target="_blank" href="https://static.igem.org/mediawiki/2015/1/1b/EPF_Lausanne_NotebookYeast_gRNAsGbsAssembly.pdf">procedure</A>. The Gibson assembly was repeated a <A target="_blank" href="https://static.igem.org/mediawiki/2015/8/89/EPF_Lausanne_NotebookYeast_gRNAsGbsAssembly_2nd.pdf">second time</A> for the missing A0, A2 and AI2.</LI><LI><B>Colony PCR </B> was performed according to the following <A target="_blank" href="https://static.igem.org/mediawiki/2015/b/b4/EPF_Lausanne_NotebookYeast_colonyPCRgRNAs1.pdf">procedure</A>. The colony PCR was repeated a <A target="_blank" href="https://static.igem.org/mediawiki/2015/4/40/EPF_Lausanne_NotebookYeast_colonyPCRgRNAs2.pdf">second time</A>, a <A target="_blank" href="https://static.igem.org/mediawiki/2015/3/3a/EPF_Lausanne_NotebookYeast_colonyPCRgRNAs3.pdf">third</A>, <A target="_blank" href="https://static.igem.org/mediawiki/2015/9/96/EPF_Lausanne_NotebookYeast_colonyPCRgRNAs4.pdf">fourth</A>, <A target="_blank" href="https://static.igem.org/mediawiki/2015/2/27/EPF_Lausanne_NotebookYeast_colonyPCRgRNAs5.pdf">fifth</A>, a <A target="_blank" href="https://static.igem.org/mediawiki/2015/a/a1/EPF_Lausanne_NotebookYeast_colonyPCRgRNAs6.pdf">sixth</A> time and several other times. </LI></UL><H2>Results</H2><DIV class="row"><DIV class="col-md-6"><DIV class="row">
                    The results of the Gibson assembly were analysed by colony PCR. Subsequently, the sequencing confirmed we had the desired constructs.
                    A0 stands for DsRed2-polyA-HH ribozyme-c3_0-HDV ribozyme.
                    </DIV><DIV class="row">I0 stands for DsRed2-polyA-HH ribozyme-c6_0-HDV ribozyme-HH ribozyme-c7_0-HDV ribozyme.
                    </DIV><DIV class="row">AI0 stands for DsRed2-polyA-HH ribozyme-c3_0-HDV ribozyme-HH ribozyme-c6_0-HDV ribozyme-HH ribozyme-c7_0-HDV ribozyme.</DIV></DIV></DIV></SECTION><SECTION id="integrate" class="panel"><H1 id="activation-repression"><SMALL>Measure activation - repression </SMALL>Integrate CYC promoters and gRNAs</H1><P>Each reporter sequence CYC_0, CYC_1, CYC_2, CYC_3 was integrated together with :
                    <UL><LI> the corresponding A (activation), I (inhibition), AI (activation and inhibition) among exemplary 0, 1, 2, 3. This was to test activation and inhibition of the appropriate promoter CYC.</LI><LI> any gRNA that doesn't correspond to the appropriate promoter, for example CYC_1 and A_0. This was to test orthogonality.</LI><LI> no gRNA. This was to measure the basal level of the promoters. </LI></UL></P><P><FONT color="red">F</FONT><FONT color="ff3399">U</FONT><FONT color="9900cc">N</FONT><FONT color="3333cc">F</FONT><FONT color="00cc66">A</FONT><FONT color="009900">C</FONT><FONT color="ff9900">T</FONT>: on September 8, ten days before wiki freeze, we changed our strain of yeast. The strain we used previously, YM4271, had a mutation on GAL4 gene, preventing its growth in galactose supplemented medium and preventing induction of GAL1 promoter.The new strain is W303.</P><P>We present our methods, both for the former strain and the more recent one. </P><H2>Materials and methods</H2><UL><LI>Integration (cf. Protocols)</LI><OL><LI>Plasmids were linearized with restriction enzyme EagI-HF, according to a <A target="_blank" href="https://static.igem.org/mediawiki/2015/0/0f/EPF_Lausanne_NotebookYeast_integ_linear1.pdf">first</A>, <A target="_blank" href="https://static.igem.org/mediawiki/2015/0/01/EPF_Lausanne_NotebookYeast_integ_linear2.pdf">second</A> and <A target="_blank" href="https://static.igem.org/mediawiki/2015/e/ef/EPF_Lausanne_NotebookYeast_integ_linear3.pdf">third</A> procedure.
                        With the former strain YM4271, we executed three different integrations. Here are the descriptions of the <A target="_blank" href="https://static.igem.org/mediawiki/2015/4/47/EPF_Lausanne_NotebookYeast_integ1.pdf">first</A> integration of CYC_0, the<A target="_blank" href="https://static.igem.org/mediawiki/2015/0/00/EPF_Lausanne_NotebookYeast_integ2.pdf"> second</A> for CYC_1 and CYC_2, and the <A target="_blank" href="https://static.igem.org/mediawiki/2015/b/b0/EPF_Lausanne_NotebookYeast_integ3.pdf"> third</A> for CYC_3.
                        After integration, the yeasts were spread on selection plates. Single colonies were prepared for analysis at the plate reader, untill we realized there was a galactose mutation in the yeast strain.</LI><LI>With strain W303, plasmids were linearized (<A target="_blank" href="https://static.igem.org/mediawiki/2015/6/6c/EPF_Lausanne15_NotebookYeast_integW303linear.pdf">procedure</A>). All the integration were as described <A target="_blank" href="https://static.igem.org/mediawiki/2015/3/34/EPF_Lausanne15_NotebookYeast_integW303.pdf"> here</A>.</LI></OL></UL><H2>Results</H2><DIV class="row"><DIV class="col-md-6"><P>After integration, the yeasts were spread on selection plates.</P></DIV></DIV></SECTION><SECTION id="integrate" class="panel"><H1 id="test_activ_inhib"><SMALL>Measure activation - repression </SMALL>Test Activation/Inhibition</H1><P>Once each construct was integrated, we prepared the 96-well plate for the measurement at the plate reader. The first step was to centrifuge the cells at 1800 rpm for 5 minutes. Once the supernatant was thrown, we washed it with water and centrifuge at 1800 rpm for 5 minutes again. The pellet was finally resuspended in 10 ml yeast minimal media (SD medium) and 100 µl was put in each well (the disposition of the plate is shown <A href="https://static.igem.org/mediawiki/2015/3/3d/EPF_Lausanne_protocol_measure.pdf" target="blank">here</A>. Anhydro tetracycline (aTc) was added at different concentrations to induce the production of dCas9.</P></SECTION><SECTION id="integrate" class="panel"><H1 id="test_activ_inhib">Build logic gate</H1><P>Now that transistors were created and characterized, we can link them to build a (bio)-logical gate. The assembly is separated in two steps: the building of the inputs, and the building of the gates. The design of the NAND gate is shown here:</P><CENTER><FIGURE><FIGCAPTION>NAND gate design with three biological transistors.</FIGCAPTION></FIGURE></CENTER><H3>Assembly of the gate </H3><P>The building blocks of the gate (gRNAs under the control of CYC) were ordered on IDT and assemble using  <A href="https://static.igem.org/mediawiki/2015/9/96/EPF_Lausanne_Gibson_gates.pdf" target="blank">Gibson assembly</A>. We then made a colony PCR to find the gate and achieve to have a band at the desired height (1745 bp with primers f_Seq_gate and r_Seq_gate). Once the sequencing confirmed that the plasmid contains every fragments assembled, we integrated the gate into the yeast genome and focused on the inputs building.</P><H3>Assembly of the inputs</H3><P>As the NAND gate requires 2 inputs, we will need to change the selection marker for the ADH1 plasmid in order to have the same backbone with different markers. We chose to insert the KanMX marker (resistance to geneticin) in  <A href="https://static.igem.org/mediawiki/2015/f/f7/EPF_Lausanne_Gibson_Kana.pdf" target="blank">p405 ADH1 plasmid</A>. After PCRs and <A href="https://static.igem.org/mediawiki/2015/b/bd/EPF_Lausanne_Gibson_inputs.pdf" target="blank">Gibson assemblies</A> for the inputs, we did a colony PCR to find every expected inputs (as we planned to build two gates, the NAND and the NOR, we have 4 different inputs named as A1, A2, A1I2 and A2I1. A refers to an activation, I to an inhibition and 1/2 refers to the promoter CYC1/2). After this step, the inputs were integrated into the yeast genome. Unfortunately, due to a lack of time, we didn’t achieve to measure the ratios of the gate).</P></SECTION></DIV></DIV></DIV><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE></DIV></DIV></DIV></DIV></BODY></HTML>