<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPF_Lausanne_Test_Scroll skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPF Lausanne/Test Scroll</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE></P><DIV class="navbar-header"><BUTTON class="navbar-toggle" type="button" data-toggle="collapse"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="main-menu"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2015.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2015.igem.org/Team_EPF_Lausanne/Project/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Applications">Applications</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Modelling">Modelling</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Interlab">Interlab Study</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievements </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Vivo">In vivo</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Silico">In silico</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Judging">Judging</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Practices">Human Practices</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Ecoli">E. Coli</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Yeast">Yeast</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Protocols">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Team/Meet">Meet us</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Timeline">Timeline</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Team/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI></DIV><DIV class="page-h promotion"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H3>Protocols</H3></DIV></DIV></DIV></DIV><DIV class="prot-section"><DIV class="row"><DIV class="col col-md-3"><NAV id="affix-nav" class="sidebar hidden-sm hidden-xs"><UL class="nav nav-pills nav-stacked" data-spy="affix" data-offset-top="200" data-offset-bottom="600"><LI class="active"><A href="#agarosegel">Agarose Gel</A></LI><LI><A href="#aminoacidsolution">Amino Acid Solution</A></LI><LI><A href="#colonypcr">Colony PCR</A></LI><LI><A href="#competentcells">Competent Cell Preparation</A></LI><LI><A href="#gibsonassembly">Gibson Assembly</A></LI><LI><A href="#glycerolstock">Glycerol Stock</A></LI><LI><A href="#medium">Lysogeny Broth (LB) Medium</A></LI><LI><A href="#lbagar">Lysogeny Broth (LB) Agar Plates</A></LI><LI><A href="#miniprep">Miniprep</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Polymerase Chain Reaction (PCR)</A></LI><LI><A href="#phusionpcr">Phusion PCR</A></LI><LI><A href="#taqpcr">Taq PCR</A></LI><LI><A href="#Q5pcr">Q5 PCR</A></LI></UL><LI><A href="#pcrpurification">PCR Product Purification</A></LI><LI><A href="#pegliac">PEG/LiAc Solution</A></LI><LI><A href="#restriction">Restriction Digest</A></LI><LI><A href="#mutagenesis">Site-directed mutagenesis</A></LI><LI><A href="#sdmedium">SD Medium</A></LI><LI><A href="#tae">Tris-Acetate-EDTA (TAE) buffer 50X</A></LI><LI><A href="#transformation">Transformation</A></LI></NAV></DIV><DIV class="col-sm-9"><DIV id="agarosegel" class="panel"><H1>Agarose Gel</H1><H2>Materials</H2><UL><LI>1X TAE</LI><LI>Agarose</LI><LI>Gel Red</LI><LI>DNA samples</LI><LI>6X loading dye</LI><LI>Nuclease-free water</LI></UL><H2>Procedure</H2><OL><LI>Prepare 1.2% agarose gel for small fragments and 3% agarose gel for large fragments</LI><LI>Mix 50 mL 1X TAE and 0.6 g (1.2%) or 1.5 g (3%) agarose</LI><LI>Melt in microwave until agarose has melted (about 50 seconds)</LI><LI>Add 1.3 μL (1.2%) or 1.5 μL (3%) Gel Red</LI><LI>Pour solution into agarose gel mold with comb</LI><LI>Let set for 20 minutes or until solid</LI><LI>Place gel in 1X TAE and remove comb</LI><LI>Load samples of 200 ng (or 2  μL) DNA mixed with 2  μL 6X loading dye and nuclease-free water up to 12  μL</LI><LI>Run gel at 100-120 Volts for 40-50 minutes (1.2%) or 80 Volts for 2 hours (3%)</LI><LI>Take a picture of the gel at the UV detector</LI></OL></DIV><DIV id="aminoacidsolution" class="panel"><H1>Amino acid solution</H1><H2>Materials</H2><UL><LI>Histidine-Hcl</LI><LI>Uracil</LI><LI>Leucine</LI><LI>Tryptophan</LI></UL><H2>Procedure</H2><TABLE width="90%" align="center"><TBODY><TR><TH>Stock concentration</TH><TH>Final concentration</TH><TH>Total quantity for 50 mL</TH></TR><TR><TD>100 mM Histidine-Hcl (209 g/mol)</TD><TD> 20.9 g/L</TD><TD> 1.045 g</TD></TR><TR><TD>20 mM Uracil (112 g/mol)</TD><TD> 2.24 g/L</TD><TD> 0.112 g</TD></TR><TR><TD>100 mM Leucine (131 g/mol)</TD><TD> 13.1 g/L</TD><TD> 0.655 g</TD></TR><TR><TD>40 mM Tryptophan (204 g/mol)</TD><TD> 8.16 g/L</TD><TD> 0.408 g</TD></TR></TBODY></TABLE><OL><LI>Filter and sterilize solutions</LI><LI>Add 8 mL per liter of selective medium or spread 500 μL on a selective plate</LI></OL></DIV><DIV id="colonypcr" class="panel"><H1>Colony PCR</H1><H2>Materials</H2><UL><LI>Materials for Taq PCR (except template plasmid DNA)</LI><LI>Petri dish with transformed colonies</LI></UL><H2>Procedure</H2><OL><LI>Prepare 25 μL reactions as in Taq PCR Protocol without template DNA</LI><LI>With a sterile tip, under the flame, scrape part of a single colony and add to PCR tubes</LI><LI>Mix by pipetting up and down or flicking the reactions </LI><LI>Put tubes in thermocycler with cycling conditions as described in Taq PCR Protocol with a longer initial denaturation (2 - 5 minutes)</LI></OL></DIV><DIV id="competentcells" class="panel"><H1>Competent Cell Preparation<SMALL>Based on Open Wet Ware Protocol</SMALL></H1><H2>Materials</H2><UL><LI>Bacterial overnight liquid culture</LI><LI>Lysogeny broth (LB) medium</LI><LI>CaCl2 solution, ice cold: 60 mM CaCl2, 15% glycerol, 10 mM PIPES, pH 7, filter sterilize and store at room temperature</LI></UL><H2>Procedure</H2><OL><LI>Subculture overnight culture 1:100 in LB medium</LI><LI>Incubate at 37°C with shaking until culture reaches an OD600 of 0.375</LI><LI>Aliquot 20 mL if the culture into chilled 50 mL tubes</LI><LI>Leave tubes on ice for 5 – 10 minutes</LI><LI>Centrifuge cells at 1600 g for 7 minutes at 4°C</LI><LI>Discard supernatant and resuspend pellet in 4 mL ice cold CaCl2 solution</LI><LI>Centrifuge cells at 1100 g for 5 minutes at 4°C</LI><LI>Discard supernatant and resuspend pellet in 4 mL ice cold CaCl2 solution</LI><LI>Keep on ice for 30 minutes</LI><LI>Centrifuge cells at 1100 g for 5 minutes at 4°C</LI><LI>Discard supernatant and resuspend pellet in 800 μL ice cold CaCl2 solution</LI><LI>Aliquot 100 μL of this suspension into microcentrifuge tubes</LI><LI>Freeze in liquid nitrogen and store at -80°C</LI></OL></DIV><DIV id="gibsonassembly" class="panel"><H1>Gibson Assembly<SMALL> Based on NEB Gibson Assembly Protocol</SMALL></H1><H2>Materials</H2><UL><LI>DNA fragments</LI><LI>2X Gibson Assembly Mater Mix (NEB)</LI><LI>2X NEBuilder Positive Control (NEB)</LI><LI>Deionized water</LI></UL><H2>Procedure</H2><OL><LI>Set up following reactions on ice, adding Gibson Assembly Master Mix last:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Component</TH><TH>2 – 3 Fragments Assembly</TH><TH>4 – 6 Fragments Assembly</TH><TH>Positive Control</TH></TR><TR><TD>Total Amount of Fragments</TD><TD>0.02 – 0.5 pmols</TD><TD>0.2 – 1 pmols</TD><TD>10 μL</TD></TR><TR><TD>2X Gibson Assembly Master Mix</TD><TD>10 μL</TD><TD>10 μL</TD><TD>10 μL</TD></TR><TR><TD>Deionized water  </TD><TD>to 20 μL</TD><TD>to 20 μL</TD><TD>0 μL</TD></TR></TBODY></TABLE><P><SMALL>Optimized efficiency for 50 – 100 ng of vectors and 2 – 3 fold of excess inserts</SMALL></P><LI>Incubate samples at 50°C for 15 minutes (2 – 3 fragments) or for 60 minutes (4 – 6 fragments)</LI><LI>Store samples on ice or at -20°C until transformation</LI><LI>Transform competent cells following the Transformation Protocol</LI></OL></DIV><DIV id="glycerolstock" class="panel"><H1>Glycerol Stock</H1><H2>Materials</H2><UL><LI>50% Glycerol</LI><LI>Bacterial overnight liquid culture</LI><LI>Liquid Nitrogen</LI></UL><H2>Procedure</H2><OL><LI>Mix 0.5 mL 50% glycerol and 0.5 mL bacterial culture</LI><LI>Freeze in liquid nitrogen and store at -80°C</LI></OL></DIV><DIV id="lbmedium" class="panel"><H1>Lysogeny Broth (LB) Medium</H1><H2>Materials</H2><UL><LI>Tryptone</LI><LI>Yeast extract</LI><LI>NaCl</LI><LI>Deionized water</LI><LI>NaOH</LI><LI>If necessary: antibiotics</LI></UL><H2>Procedure (for 1 L)</H2><OL><LI>Dissolve 10 g tryptone, 5 g yeast extract and 10 g NaCl in 950 mL deionized water</LI><LI>Adjust pH to 7 using 1 M NaOH and bring volume to 1 L</LI><LI>Autoclave</LI><LI>If necessary: let medium cool to 55°C and add atibiotic</LI><LI>Store at room temperature</LI></OL></DIV><DIV id="lbagar" class="panel"><H1>Lysogeny Broth (LB) Agar Plates</H1><H2>Materials</H2><UL><LI>Tryptone</LI><LI>Yeast extract</LI><LI>NaCl</LI><LI>Deionized water</LI><LI>NaOH</LI><LI>Agar</LI><LI>If necessary: antibiotics</LI><LI>Petri dishes</LI></UL><H2>Procedure (for 1 L, ie. about 50 plates)</H2><OL><LI>Dissolve 10 g tryptone, 5 g yeast extract and 10 g NaCl in 950 mL deionized water</LI><LI>Adjust pH to 7 using 1 M NaOH and bring volume to 1 L</LI><LI>Add 15 g agar
                                </LI><LI>Autoclave</LI><LI>If necessary: let medium cool to 55°C and add atibiotic</LI><LI>Pour into petri dishes (about 20 mL per dish) and let set</LI><LI>Invert and store at 4°C</LI></OL></DIV><DIV id="miniprep" class="panel"><H1>Miniprep<SMALL>With QIAprep Spin Miniprep Kit (QIAGEN)</SMALL></H1><H2>Materials</H2><UL><LI>Bacterial overnight liquid cultures (1 - 5 mL) </LI><LI>QIAprep Spin Miniprep Kit </LI></UL><H2>Procedure</H2><OL><LI>Pellet 1 -5 mL bacterial culture by centrifugation at more than 8000 rpm for 3 minutes</LI><LI>Resuspend pelleted bacterial cells in 250 μL P1 buffer and transfer to a microcentrifuge tube</LI><LI> Add 250 μL P2 buffer and mix by inverting tube 4 – 6 times</LI><LI>Add 350 μL N3 buffer and mix by inverting tube 4- 6 times</LI><LI>Centrifuge for 10 min at 13000 rpm</LI><LI>Apply supernatant to the QIAprep spin column by pipetting, centrifuge for 30 – 60 seconds and discard flow-through</LI><LI>Wash the QIAprep spin column by adding 0.5 mL PB buffer, centrifuge for 30 – 60 seconds and discard flow-through</LI><LI>Wash the QIAprep spin column by adding 0.75 mL PE buffer, centrifuge for 30 – 60 seconds and discard flow-through</LI><LI>Centrifuge for 1 minute to remove residual wash buffer</LI><LI> Elute DNA by placing QIAprep column in a clean 1.5 mL microcentrifuge tube and adding 50 μL EB buffer or water (or less for higher concentration). Let stand for 1 minute and centrifuge for 1 minute</LI></OL><H2>Comments</H2><P>With low copy plasmid it is often difficult to obtain high concentrations (needed for sequencing). In these cases it's possible to proceed as follows: culture bacteria in 5 x 7mL tubes of LB medium and perform the miniprep for each tube separately until step 9. Then elute DNA by applying two times step 10 to each tube with the same 50µl of EB buffer.</P></DIV><DIV id="phusionpcr" class="panel"><H1>Phusion PCR<SMALL> Based on NEB Phusion PCR Protocol</SMALL></H1><H2>Materials</H2><UL><LI>5X Phusion HF or GC Buffer</LI><LI>dNTPs</LI><LI>Forward and Reverse Primers</LI><LI>Template plasmid DNA</LI><LI>Phusion DNA polymerase</LI><LI>Nuclease-free Water</LI></UL><H2>Procedure</H2><OL><LI>Prepare following reaction in 0.5 mL PCR tubes on ice, adding polymerase last:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Component</TH><TH>20 μL reaction</TH><TH>50  μL reaction</TH></TR><TR><TD>5X Phusion HF or GC Buffer</TD><TD>4 μL</TD><TD>10 μL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.4 μL</TD><TD>1 μL</TD></TR><TR><TD>10 μM Forward Primer</TD><TD>1 μL</TD><TD>2.5 μL</TD></TR><TR><TD>10 μM Reverse Primer</TD><TD>1 μL</TD><TD>2.5 μL</TD></TR><TR><TD>Template plasmid DNA</TD><TD>1 pg – 10 ng</TD><TD>1 pg – 10 ng</TD></TR><TR><TD>Phusion DNA Polymerase</TD><TD>0.2 μL</TD><TD>0.5 μL</TD></TR><TR><TD>Nuclease-free Water</TD><TD>to 20 μL</TD><TD>to 50 μL</TD></TR></TBODY></TABLE><P><SMALL>Usually 100 pg – 1 ng of template DNA is sufficient</SMALL></P><LI>Mix by pipetting up and down or flicking the reactions
                                </LI><LI>Put tubes in thermocycler (with a pre-heated lid) with following cycling conditions:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR><TD>25 – 35 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>5 - 10 seconds</TD></TR><TR><TD>Annealing</TD><TD>45 – 72°C</TD><TD>10 – 30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>15 -30 seconds per kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>5 -10 minutes</TD></TR><TR><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE></OL><H2>Guidelines</H2><P>To be completed</P></DIV><DIV id="taqpcr" class="panel"><H1>Taq PCR<SMALL> Based on NEB Taq PCR Protocol</SMALL></H1><H2>Materials</H2><UL><LI>10X Standard Taq Reaction Buffer</LI><LI>dNTPs</LI><LI>Forward and Reverse Primers</LI><LI>Template plasmid DNA</LI><LI>Taq DNA polymerase</LI><LI>Nuclease-free Water</LI></UL><H2>Procedure</H2><OL><LI>Prepare following reaction in 0.5 mL PCR tubes on ice, adding polymerase last:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Component</TH><TH>25 μL reaction</TH><TH>50 μL reaction</TH></TR><TR><TD>10X Standard Taq Reaction Buffer</TD><TD>2.5 μL</TD><TD>5 μL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.5 μL</TD><TD>1 μL</TD></TR><TR><TD>10 μM Forward Primer</TD><TD>0.5 μL</TD><TD>1 μL</TD></TR><TR><TD>10 μM Reverse Primer</TD><TD>0.5 μL</TD><TD>1 μL</TD></TR><TR><TD>Template plasmid DNA</TD><TD>1 pg – 1 ng</TD><TD>1 pg – 1 ng</TD></TR><TR><TD>Taq DNA Polymerase</TD><TD>0.125 μL</TD><TD>0.25 μL</TD></TR><TR><TD>Nuclease-free Water</TD><TD>to 25 μL</TD><TD>to 50 μL</TD></TR></TBODY></TABLE><P><SMALL>Usually 100 pg – 1 ng of template DNA is sufficient</SMALL></P><LI>Mix by pipetting up and down or flicking the reactions
                                </LI><LI>Put tubes in thermocycler (with a pre-heated lid) with following cycling conditions:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>30 seconds</TD></TR><TR><TD>25 – 35 cycles</TD><TD>Denaturation</TD><TD>95°C</TD><TD>15 – 30 seconds</TD></TR><TR><TD>Annealing</TD><TD>45 – 68°C</TD><TD>15 – 60 seconds</TD></TR><TR><TD>Extension</TD><TD>68°C</TD><TD>1 minutes per kb</TD></TR><TR><TD>Final Extension</TD><TD>68°C</TD><TD>5 minutes</TD></TR><TR><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE></OL><H2>Guidelines</H2><P>To be completed</P></DIV><DIV id="Q5pcr" class="panel"><H1>Q5 PCR<SMALL> Based on NEB Q5 PCR Protocol</SMALL></H1><H2>Materials</H2><UL><LI>5X Q5 Reaction Buffer</LI><LI>dNTPs 10µM</LI><LI>Forward and Reverse Primers</LI><LI>Template plasmid DNA</LI><LI>Q5 High Fidelity DNA polymerase</LI><LI>Nuclease-free Water</LI></UL><H2>Procedure</H2><OL><LI>Prepare following reaction in 0.5 mL PCR tubes on ice, adding polymerase last:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Component</TH><TH>25 μL reaction</TH><TH>50 μL reaction</TH></TR><TR><TD>5X Q5 Reaction Buffer</TD><TD>5 μL</TD><TD>10 μL</TD></TR><TR><TD>10 mM dNTPs</TD><TD>0.5 μL</TD><TD>1 μL</TD></TR><TR><TD>10 μM Forward Primer</TD><TD>1.25 μL</TD><TD>2.5 μL</TD></TR><TR><TD>10 μM Reverse Primer</TD><TD>1.25 μL</TD><TD>2.5 μL</TD></TR><TR><TD>Template plasmid DNA</TD><TD>1 &lt; 1,000 ng</TD><TD>1 &lt; 1,000 ng</TD></TR><TR><TD>Q5 DNA Polymerase</TD><TD>0.25 μL</TD><TD>0.5 μL</TD></TR><TR><TD>Nuclease-free Water</TD><TD>to 25 μL</TD><TD>to 50 μL</TD></TR></TBODY></TABLE><P><SMALL>Usually 100 pg – 1 ng of template DNA is sufficient</SMALL></P><LI>Mix by pipetting up and down or flicking the reactions
                                </LI><LI>Put tubes in thermocycler (with a pre-heated lid) with following cycling conditions:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR><TD>25 – 35 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>5 – 10 seconds</TD></TR><TR><TD>Annealing</TD><TD>50 – 72°C</TD><TD>10 – 30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>20 - 30 seconds per kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE></OL><H2>Guidelines</H2><P>To be completed</P></DIV><DIV id="pcrpurification" class="panel"><H1>PCR Product Purification<SMALL>With QIAquick PCR Purification Kit (QIAGEN)</SMALL></H1><H2>Materials</H2><UL><LI>PCR products</LI><LI>QIAquick PCR Purification Kit</LI></UL><H2>Procedure</H2><OL><LI>Add 5 volumes PB buffer to 1 volume of PCR product and mix</LI><LI>Place QIAquick column in 2 ml collection tube</LI><LI>Apply samples to QIAquick column and centrifuge for 30 – 60 seconds, discard flow-through</LI><LI>Wash by adding 750 μL PE buffer to QIAquick column and centrifuge fo 30 – 60 seconds, discard flow-through</LI><LI>Centrifuge QIAquick column for 1 minutes to remove residual wash buffer</LI><LI>Elute DNA by adding 30 or 50 μL EB buffer or water to the center of the QIAquick column. Let stand for 1 minutes and centrifuge for 1 minute</LI></OL></DIV><DIV id="pegliac" class="panel"><H1>PEG/LiAc Solution</H1><H2>Materials</H2><UL><LI>50% PEG  (Polyethylene glycol) prepared with sterile deionized water</LI><LI>10X TE buffer: 0.1 M Tris-Hcl, 10 mM EDTA, ph 7.5, autoclaved</LI><LI>10X LiAc: 1 M lithium acetate, pH 7.5 adjusted with dilute acetic acid, autoclaved</LI></UL><H2>Procedure</H2><OL><LI>Prepare PEG/LiAc solution as follows:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Stock concentration</TH><TH>Final concentration</TH><TH>Total quantity for 10 mL solution</TH></TR><TR><TD>50% PEG</TD><TD>40% PEG</TD><TD>8 mL</TD></TR><TR><TD>10X TE buffer</TD><TD>1X TE buffer</TD><TD>1 mL</TD></TR><TR><TD>10X LiAc</TD><TD>1X LiAc</TD><TD>1 mL</TD></TR></TBODY></TABLE></OL></DIV><DIV id="restriction" class="panel"><H1>Restriction Digest<SMALL>&quot;Typical&quot; Restriction Digest based on NEB Protocol</SMALL></H1><H2>Materials</H2><UL><LI>Restriction Enzyme(s)</LI><LI>DNA</LI><LI>10X NEBuffer (Appropriate buffer for used enzyme)</LI><LI>Water</LI></UL><H2>Procedure</H2><OL><LI>Prepare following reaction in 0.5 mL PCR tubes, adding enzyme(s) last:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Component</TH><TH>20 μL reaction</TH><TH>50 μL reaction</TH></TR><TR><TD>Restriction enzyme(s)</TD><TD>1 μL (for each enzyme)</TD><TD>1 μL (for each enzyme)</TD></TR><TR><TD>DNA</TD><TD>100 ng - 1 μg</TD><TD>100 ng - 1 μg</TD></TR><TR><TD>10X NEBuffer</TD><TD>2 μL</TD><TD>5 μL</TD></TR><TR><TD>Water</TD><TD>to 20 μL</TD><TD>to 50 μL</TD></TR></TBODY></TABLE><P><SMALL>20 μL reactions are sufficient for restriction enzyme analysis, larger volumes are usefull if product is used for cloning</SMALL></P><LI>Incubate at temperature and for duration appropriate for used enzyme (typically 37°C for 15 minutes or 1 hour)</LI><LI>Optional: Inactivate enzyme by incubating reaction at temperature and for duration appropriate for used enzyme (typically 65°C for 20 minutes)</LI></OL></DIV><DIV id="mutagenesis" class="panel"><H1>Site-directed mutagenesis<SMALL>Based on NEB Q5 Site-directed Mutagensis Kit Protocol</SMALL></H1><H2>Materials</H2><UL><LI>Q5 Hot Start High-Fidelity 2X Master Mix (NEB)</LI><LI>Forward and reverse primers</LI><LI>Template DNA</LI><LI>Nuclease-free water</LI></UL><H2>Procedure</H2><OL><LI>Prepare following reaction in 0.5 mL PCR tubes on ice, adding Mater Mix last:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Component</TH><TH>25 μL reaction</TH></TR><TR><TD>Q5 Hot Start High-Fidelity 2X Master Mix</TD><TD>12.5 μL</TD></TR><TR><TD>10 μM Forward Primer</TD><TD>1.25 μL</TD></TR><TR><TD>10 μM Reverse Primer</TD><TD>1.25 μL</TD></TR><TR><TD>Template DNA (1-25 ng/μL)</TD><TD>1 μL</TD></TR><TR><TD>Nuclease-free Water</TD><TD>9 μL</TD></TR></TBODY></TABLE><LI>Put tubes in thermocycler (with a pre-heated lid) with following cycling conditions:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Step</TH><TH>Temperature</TH><TH>Time</TH></TR><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD></TR><TR><TD>25 cycles</TD><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD></TR><TR><TD>Annealing</TD><TD>50 – 72°C</TD><TD>10 – 30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>20 - 30 seconds per kb</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR><TD>Hold</TD><TD>4°C </TD></TR></TBODY></TABLE><LI>Assemble following reagents:</LI><TABLE width="90%" align="center"><TBODY><TR><TH>Component</TH><TH>Volume</TH></TR><TR><TD>PCR Product</TD><TD>1 μL</TD></TR><TR><TD>2X KLD Reaction Buffer</TD><TD>5 μL</TD></TR><TR><TD>10X KLD Enzyme Mix</TD><TD>1 μL</TD></TR><TR><TD>Nuclease-free Water</TD><TD>3 μL</TD></TR></TBODY></TABLE><LI>Mix by pipetting up and down and incubate at room temperature for 5 minutes</LI><LI>Thaw completent E. coli cells on ice</LI><LI>Add 5 μL of KLD mix to the tube of thawed cells. Flick tube 4-5 times to mix.</LI><LI>Place mixture on ice for 30 minutes</LI><LI>Heat shock at 42°C for 30 seconds</LI><LI>Place on ice for 5 minutes</LI><LI>Pipette 950 μL of room temperature SOC into the mixture</LI><LI>Incubate at 37°C for 60 minutes with shaking</LI><LI>Mix thouroughly by flicking the tube and inverting, then spread 50-100 μL on a plate with appropriate antibiotics and incubate overnight at 37°C</LI></OL></DIV><DIV id="sdmedium" class="panel"><H1>Sd Medium</H1><H2>Materials</H2><UL><LI>Amino Acid Powder</LI><LI>Yeast Nitrogen Base</LI><LI>Ammonium Sulphate</LI><LI>Adenine Sulphate</LI><LI>Water</LI><LI>NaOH</LI><LI>Agar</LI><LI>Glucose</LI></UL><H2>Procedure</H2><OL><LI>Place stirrer bar in 2 L Erlenmeyer</LI><LI>Add 2.6 g amino acid powder, 3.4 g yeast nitrogen base, 10 g ammonium sulphate, 1 g adenine sulphate and 950 mL water</LI><LI>Adjust pH to 5.9 by adding a few drops of 10 M NaOH</LI><LI>In an other Erlenmeyer, add 35 g agar and 900 mL water</LI><LI>Autoclave both bottles</LI><LI>Transfer the content of first bottle to the agar-containing bottle</LI><LI>Cool to 55°C</LI><LI>Add 100 ml 40% glucose and 16 ml of the required amino acids</LI><LI>Pour plates</LI></OL></DIV><DIV id="tae" class="panel"><H1>Tris-Acetate-EDTA (TAE) buffer 50X</H1><H2>Materials</H2><UL><LI>Acetate 100%</LI><LI>Tris base</LI><LI>EDTA 0,5M</LI></UL><H2>Procedure (1L)</H2><OL><LI>Add 100mL of 0,5M EDTA (pH 8.0)</LI><LI>Add 242g of Tris base </LI><LI>Add 57,1mL of glacial acetic acid (100%)</LI><LI>Fill up to 1L with water (adjust pH to ~8.3)</LI><LI>Send to autoclave</LI><P><SMALL>To prepare 1L of 1X TAE dilute 20mL of 50X TAE in 980mL of water.</SMALL></P></OL></DIV><DIV id="transformation" class="panel"><H1>Transformation<SMALL> Based on NEB Transformation Protocol</SMALL></H1><H2>Materials</H2><UL><LI>Competent cells</LI><LI>DNA</LI><LI>SOC medium (SOB + Glucose)</LI><LI>Petri dish with appropriate antibiotic resistance</LI></UL><H2>Procedure</H2><OL><LI>Thaw competent cells on ice</LI><LI>Add 2 μL DNA to the competent cells, mix by pipetting up and down or flicking the tube 4 -5 times</LI><LI>Place mixture on ice for 30 minutes</LI><LI>Heat shock at 42°C for 30 seconds</LI><LI>Transfer tubes to ice for 2 minutes</LI><LI>Add 950 μL room-temperature SOC media</LI><LI>Incubate at 37°C for 60 minutes with shaking</LI><LI>Spread 100 μL cells onto selection plates (warm plates to 37°C prior to this step for increased efficiency)</LI><LI>Incubate overnight at 37°C</LI></OL></DIV></DIV></DIV></DIV><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE><DIV class="sponsors"><DIV><H1>Sponsors</H1></DIV></DIV><FOOTER class="footer-distributed"><DIV class="container"><DIV class="row"><DIV class="col-lg-6"><DIV class="footer-right"><P class="footer-company-about"><SPAN>About us</SPAN>
                        Team of a dozen biotechnology students that will attempt to change the world
                        </P></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>