<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPF_Lausanne_Notebook_Ecoli skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPF Lausanne/Notebook/Ecoli</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE></P><DIV class="container"><NAV class="collapse navbar-collapse navHeaderCollapse" role="navigation"><UL class="nav navbar-nav"><LI><A href="https://2015.igem.org/Team:EPF_Lausanne">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Project/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Design">Prototype Design</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Software">Software</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Interlab">Interlab Study</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Achievements </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Achievements/Parts">Parts</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Judging">Judging</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Wet Lab </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Ecoli">E. coli notebook</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Yeast">S. cerevisiae notebook</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Practices">Human Practices</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Team/Meet">Meet us</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:EPF_Lausanne/Collaborations">Collaborations</A></LI><LI><A href="https://igem.org" target="_blank">iGEM</A></LI></NAV></DIV><DIV class="classy-bar"><DIV class="container"><DIV class="row"><DIV class="col-md-12 text-center"><H3>E. Coli Laboratory Notebook</H3></DIV></DIV></DIV></DIV><DIV class="prot-section"><DIV class="row"><DIV class="col col-md-2 scrollspy"><UL id="nav" class="nav hidden-xs hidden-sm" data-spy="affix"><LI class="active"><A href="#pdcas9w">Construction of pdCas9-w</A></LI><LI><A href="#PCRpdcas9">Open pdCas9 by PCR</A></LI><LI><A href="#PCRpwj66">Extract w subunit from pWJ66 by PCR</A></LI><LI><A href="#gibsonpdcas9w">Gibson assembly of pdCas9-w</A></LI><LI><A href="#mutagenesis">Site-directed mutagenesis of pdCas9-w</A></LI></UL><LI><A href="#pdcas9wsgrna">Construction of pdCas9-w-sgRNAs</A></LI><LI><A href="#part1">Part 1: Assemble pdCas9-w + sgRNA</A></LI><LI><A href="#openpdcas9w">Open pdCas9-w by restriction digest</A></LI><LI><A href="#pcrsgrna">Amplify sgRNA cassettes by PCR</A></LI><LI><A href="#gibsonpdcas9wsgrna">Assemble pdCas9-w-sgRNA constructs</A></LI><LI><A href="#part2">Part 2: Assemble pdCas9-w-sgRNA + sgRNA</A></LI><LI><A href="#pcrpdcas9sgrna">Extract sgRNA from pdCas9-w-sgRNA and open pdCas9-w-sgRNA by Q5 PCR</A></LI><LI><A href="#gibsonpdcas9wsgrna2">Assemble pdCas9-w-sgRNA-sgRNA constructs</A></LI><LI><A href="#pwj89alt">Construction of pWJ89alt</A></LI><LI><A href="#pcrpwj89">PCR pWJ89</A></LI><LI><A href="#pcrj23117alt">PCR J23117alt promoter</A></LI><LI><A href="#gibsonpwj89alt">Gibson assembly of pWJ89alt</A></LI><LI><A href="#arac">Construction of pAraC-sgRNA</A></LI><LI><A href="#pcrparac">PCR pAraC and sgRNA</A></LI><LI><A href="#gibsonparac">Assemble pAraC-sgRNA constructs</A></LI><LI><A href="#plink">Construction of pWJ89alt_Z4-to-X4</A></LI><LI><A href="#openpwj89alt">Open pWJ89alt by restriction digest</A></LI><LI><A href="#pcrz4-to-x4">PCR Z4-to-X4</A></LI><LI><A href="#gibsonplink">Gibson assembly pWJ89alt_Z4-to-X4</A></LI><LI><A href="#activityassay">Transistor activity assays</A></LI><LI><A href="#primers">Primer table</A></LI><LI><A href="#ref">References</A></LI></DIV><DIV class="col-md-offset-1 col-md-9"><P><I>Note that all experiments were done by EPFL iGEM team members.</I></P><SECTION id="pdcas9w" class="panel"><H1>Construction of pdCas9-w<SMALL>07.05.2015 - 01.06.2015 + 28.07.2015 - 05.08.2015</SMALL></H1><P>pdCas9-w contains a gene that produces dCas9 fused to the w subunit of RNA polymerase (RNAP), which recruits RNAP, controlled by a Tetracycline inducible promoter.</P><H2>Materials and method</H2><UL><LI><B>Open pdCas9-bacteria by Phusion PCR</B> (cf. Protocols) with primers f_Gbs_pdCas9 and r_Gbs_pdCas9 + PCR product purification (cf. Protocols) + agarose gel electrophoresis of 2 µL purified PCR products. pdCas9-bacteria was a gift from Stanley Qi (Addgene plasmid # 44249) [1].</LI><LI><B>Extract w subunit from pWJ66 by Phusion PCR</B> (cf. Protocols) with primers f_Gbs_w and r_Gbs_w + PCR product purification (cf. Protocols) + agarose gel electrophoresis of 2 µL purified PCR products. pWJ66 was a gift from Luciano Marraffini (Addgene plasmid # 46570) [2].</LI><LI><B>Gibson assembly (cf. Protocols) of pdCas9-w</B> with purified PCR products of pdCas9-bacteria and w subunit + transformation (cf. Protocols) of ultra-competent DH5a cells (NEB). Control with colony PCR (cf. Protocols) with primers f_Gbs_w and r_Scq_pdCas9_w_sgRNA primers + agarose gel electrophoresis of 2 µL PCR products <B>+</B> restriction digest (cf. Protocols) of pdCas9-w (isolated by Miniprep (cf. Protocols) with BamHI and KpnI separately + agarose gel electrophoresis of 2 µL digested products <B>+</B> sequencing (Microsynth)</LI><LI><B>Site-directed mutagenesis (cf. Protocols) of dCas9-w</B> with primers f_Mt_A2080C_pdCas9-w and r_Mt_A2080C_pdCas9-w (necessary for pdCas9-w to be BioBrick compatible). Control with restriction digest (cf. Protocols) of mutated pdCas9-w-mut (isolated by Miniprep (cf. Protocols)) with EcoRI <B>+</B> sequencing (Microsynth)</LI></UL><H2>Results</H2><DIV id="PCRpdcas9" class="panel"><H3>Open pdCas9 by PCR<SMALL>07.05.2015, 10.05.2015</SMALL></H3><DIV class="row"><DIV class="col-md-7"><P>Linearized pdCas9-w is expected to be 6705 bp.We tested many parameters: HF vs. GC buffer, different annealing temperatures and different extension times. All samples visible on gel were successfully amplified (cf. Fig.1).For next steps, sample from lane 1 (cf. Fig.1) was used.</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.1</B> - Gel of pdCas9 linearized by PCR, successful for all 5 samples. <SMALL>Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="PCRpwj66" class="panel"><H3>Extract w subunit from pWJ66 by PCR<SMALL>07.05.2015</SMALL></H3><DIV class="row"><DIV class="col-md-7"><P>Successful PCR reactions are expected to yield 340 bp fragments.PCR was successful for sample visible on gel. (cf. Fig.2)</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.2</B> - Gel of w subunit extracted from pWJ66 by PCR, successful for sample in lane 1. <SMALL>Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="gibsonpdcas9w" class="panel"><H3>Gibson assembly of pdCas9-w<SMALL>Gibson assembly, transformation: 19.05.2015. Colony PCR: 29.05.2015. Restriction analysis: 01.06.2015.</SMALL></H3><DIV class="row"><DIV class="col-md-7"><P>Colony PCR yields 666 bp amplicons if Gibson assembly worked and 396 bp amplicons if the plasmid self-ligated.Lane &quot;C&quot; is a negative control: PCR was run with all components except template DNA. It is empty which means there is no contamination (cf. Fig3d).Colony PCR shows that Gibson assembly worked for some samples (cf. Fig.3). To avoid working with too many samples, we kept the ones from lanes 16, 22, 25, 31, 37 and 43 for the next steps.</P><P>pdCas9-w was analyzed by restriction digest with BamHI and KpnI separately, both are single cutters in pdCas9 and double cutters in pdCas9-w. Digestion by BamHI yields fragments of 6147 bp and 834 bp, digestion by KpnI yields fragments of 4534 bp and 2447 bp (cf. Fig.4). Too much ladder was loaded so it is difficult to estimate the size of the fragments. The smaller fragments are also very difficult to see. By looking at the relative heights, we can see that all colonies seem to have the w subunit insert. There is only the undigested sample for colony 16 that is not visible, probably due to an error while loading the gel.</P><P>Sequencing confirmed that sample 22 is in fact pdCas9-w. We used this sample for the next steps and stored it as a glycerol stock (c.f. Protocols).</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.3</B> - Gels of colony PCR of dCas9-w Gibson assembly products. Gibson assembly worked for samples that have 666 bp amplicons: lanes 5, 6, 16, 22, 25-28, 30, 31, 37, 41, 43. All other lanes are self-ligation (396 bp amplicons). <SMALL>Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Fig.4</B> - Gels of restriction digest of pdCas9-w Gibson assembly products. Each colony is present on gels in triplicates: undigested (7kb but migrates faster than linear DNA), digested by BamHI (6147 bp + 834 bp) and digested by KpnI (4534 bp + 2447 bp). All digestions worked and show that plasmids contain the w subunit. Undigested plasmid for sample 16 is not visible due to a technical error. <SMALL>Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="mutagenesis" class="panel"><H3>Site-directed mutagenesis of dCas9-w<SMALL>Site-directed mutagenesis: 28.07.2015, 29.07.2015. Restriction digest: 05.08.2015</SMALL></H3><DIV class="row"><DIV class="col-md-7"><P>The first restriction digest analysis of many pdCas9-w-mut colonies was inconclusive, so we chose one colony to do a second restriction digest analysis (cf. Fig.5). First, note that the ladder did not work well so we will ignore it and only look at the relative heights and number of bands. Also, lanes 1, 4 and 6 concerned another experiment so we will ignore them.</P><P>pdCas9-w and pdCas9-w-mut undigested (cf. Fig.5 lanes 2 and 8) should both contain the 7 kb circular plasmid. pdCas9-w digested with BsrBI (cf. Fig.5 lane 7) and pdCas9-w-mut digested with EcoRI or BsrBI separately (cf. Fig.5 lanes 10 and 11) should contain the linearized 7 kb plasmid. pdCas9-w digeted with EcoRI (cf. Fig.5 lane 5) should contain two fragments of 5.6 kb and 1.4 kb. pdCas9-w digested with EcoRI and BsrBI (cf. Fig.5 lane 3) should contain three fragments of 3.2 kb, 2.4 kb and 1.4 kb, pdCas9-w-mut digested with the same enzymes (cf. Fig.5 lane 9) should contain two fragments of 4.6 kb and 2.4 kb.We can see that the pattern on the gel is correct so the mutagenesis worked. Notice than the circular plasmids migrated less than the linear plasmids and also that the undigested plasmids seem to contain a second smaller fragment which is probably contamination.</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.5</B> - Gel of restriction digest of mutagenesis product (note that lanes 1, 4 and 6 can be ignored)Lane 2: pdCas9-w undigested. Lane 3: pdCas9-w digested with BsrBI and EcoRI. Lane 5: pdCas9-w digested with EcoRI. Lane 7: pdCas9-w digested with BsrBI. Lane 8: pdCas9-w-mut undigested. Lane 9: pdCas9-w-mut digested by BsrBI and EcoRI. Lane 10: pdCas9-w-mut digested with EcoRI. Lane 11: pdCas9-w-mut digested with BsrBI. <SMALL>Ladder: 1kb Generuler (did not work well)</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV></SECTION><SECTION id="pdcas9wsgrna" class="panel"><H1>Construction of pdCas9-w-sgRNAs - 08.07.2015 - 02.09.2015</H1><P>pdCas9-w-sgRNAs contain a gene producing dCas9-w controlled by a Tetracycline inducible promoter and sgRNA producing cassette (constitutive promoter).</P><DIV class="row"><DIV class="col-md-7"><P>These experiments consist of inserting one or two sgRNA producing cassettes (cf. Fig.6) into pdCas9-w. In <B>part 1</B>, one sgRNA (X0, X4, X35, Z0, Z4 or Z35) was inserted into pdCas9-w by Gibson assembly. For technical reasons, we were unable to make most constructs with two inserted cassettes using this method. In <B>part 2</B>, we extracted sgRNA cassettes from existing constructs and inserted them in other constructs that already contained one other sgRNA cassette.</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.6</B> - Schematics of structure of sgRNA cassettes</FIGCAPTION></FIGURE></DIV></DIV><H2>Materials and method</H2><H4>Part 1: Assemble pdCas9-w + sgRNA</H4><UL><LI><B>Open pdCas9-w by restriction digest</B> (cf. Protocols) with BsrBI (blunt ends) + agarose gel electrophoresis of 2 µL of digested and undigested product</LI><LI><B>Amplify sgRNA cassettes by Phusion or Q5 PCR</B> (cf. Protocols) with primers f_Gbs_sgRNA-A and r_Gbs_sgRNA-B + agarose gel electrophoresis of 2 µL PCR products + PCR product purification (cf. Protocols).</LI><LI><B>Gibson assembly pdCas9-w-sgRNA</B> (cf. Protocols) with linearized pdCas9-w and purified sgRNA PCR products + transformation (cf. Protocols) of ultra-competent DH5a cells (NEB). Control with colony PCR (cf. Protocols) with primers f_ColPCR_sgRNAs and r_ColPCR_sgRNAs + agarose gel electrophoresis of 2 µL PCR products <B>+</B> sequencing (Microsynth)</LI></UL><H4>Part 2: Assemble pdCas9-w-sgRNA + sgRNA</H4><UL><LI><B>Extract sgRNA from pdCas9-w-sgRNA by Q5 PCR</B> (cf. Protocols) with primers f_Gbs_sgRNAinPd and r_Gbs_sgRNAinPd + agarose gel electrophoresis of 2 µL PCR products + PCR product purification (cf. Protocols)</LI><LI><B>Open pdCas9-w-sgRNA by Q5 PCR</B> (cf. Protocols) with primers f_opn_d9w and r_opn_d9w + agarose gel electrophoresis of 2 µL PCR products + PCR product purification (cf. Protocols)</LI><LI><B>Gibson assembly of pdCas9-w-sgRNA</B>, as indicated in table below + transformation (cf. Protocols) of ultra-competent DH5a cells (NEB). Control with colony PCR (cf. Protocols) with primers f_Sq_CoP_pdCas92ndIns and r_Sq_CoP_pAraC2ndInsert + agarose gel electrophoresis of 2 µL PCR products <B>+</B> sequencing (Microsynth)</LI></UL><TABLE><TBODY><TR><TH>Final constructs</TH><TH>sgRNA insert</TH><TH>sgRNA in pdCas9-w-sgRNA (backbone)</TH></TR><TR><TD>pdCas9-w-Z0-Z4</TD><TD>Z4</TD><TD>pdCas9-w-Z0</TD></TR><TR><TD>pdCas9-w-Z0-Z35</TD><TD>Z35</TD><TD>pdCas9-w-Z0</TD></TR><TR><TD>pdCas9-w-Z4-Z35</TD><TD>Z35</TD><TD>pdCas9-w-Z4</TD></TR><TR><TD>pdCas9-w-X0-X4</TD><TD>X4</TD><TD>pdCas9-w-X0</TD></TR><TR><TD>pdCas9-w-X0-X35</TD><TD>X35</TD><TD>pdCas9-w-X0</TD></TR><TR><TD>pdCas9-w-X4-X35</TD><TD>X35</TD><TD>pdCas9-w-X4</TD></TR></TBODY></TABLE><H2>Results</H2><DIV id="part1" class="panel"><H3>Part 1: Assemble pdCas9-w + sgRNA<SMALL>08.07.2015 - 07.08.2015</SMALL></H3><DIV id="openpdcas9w"><H4>Open pdCas9-w by restriction digest<SMALL>16.07.2015, 19.07.2015, 26.07.2015, 02.08.2015 (repeats)</SMALL></H4><DIV class="row"><DIV class="col-md-7"><P>The digested and undigested plasmids have the same size. Circular fragments migrates differently (generally faster) than linear fragments of the same size. This means that the undigested and digested plasmid will migrate differently. Even though the ladder is not clear, we see that the digested sample migrated more slowly than the undigested sample (cf. Fig.7). The restriction digest was successful.Digestion was repeated at least 4 times, we needed it for many PCRs.</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.7</B> - Gel of pdCas9-w digested by BamHI (lane 1) and undigested (lane 2). Digested plasmid is lower on gel, which means digestion worked. <SMALL>Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="pcrsgrna"><H4>Amplify sgRNA cassettes by PCR<SMALL>08.07.2015, 09.07.2015, 10.07.2015, 13.07.2015, 14.07.2015, 15.07.2015, 16.07.2015, 02.08.2015, 03.08.2015, 04.08.2015, 06.08.2015</SMALL></H4><DIV class="row"><DIV class="col-md-6"><P>Amplified sgRNA cassettes are expected to be about 370 bp.sgRNA cassette Z0 was amplified with Q5 PCR, both samples were successfully amplified (cf. Fig.8a). sgRNA cassette Z4 was amplified with Phusion PCR, samples 3 and 4 were successfully amplified (cf. Fig.8b). sgRNA cassette Z35 was successfully amplified with Phusion PCR, it is hardly visible on the gel (cf. Fig.8c). sgRNA cassette X0 was successfully amplified with Phusion PCR (cf. Fig.8d). sgRNA cassette X4 was successfully amplified with Phusion PCR (cf. Fig.8e). sgRNA cassette X35 was successfully amplified with Phusion PCR (cf. Fig.8f).</P><P>Many PCRs were necessary to obtain all sgRNAs because of technical problems, such as PCR reactions not working and contamination of most of our tubes. Figures show only the samples that were used in the next steps.</P></DIV><DIV class="col-md-6"><FIGURE><FIGCAPTION><B>Fig.8: Gels of sgRNA cassettes amplified by PCR</B><SMALL><B>Fig.8a</B> - Gel of sgRNA cassette Z0 amplicon (lanes 1-2), successful for both samples<B>Fig.8b</B> - Gel of sgRNA cassette Z4 amplicon, successful for samples in lanes 3 and 4<B>Fig.8c</B> - Gel of sgRNA cassette Z35 amplicon, successful for sample in lane 1<B>Fig.8d</B> - Gel of sgRNA cassette X0 amplicon, successful for sample<B>Fig.8e</B> - Gel of sgRNA cassette X4 amplicon<B>Fig.8f</B> - Gel of sgRNA cassette X35 amplicon, successful for sampleLadder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="gibsonpdcas9wsgrna"><H4>Assemble pdCas9-w-sgRNA constructs<SMALL>Gibson assembly, transformation: 16.07.2015, 17.07.2015, 27.07.2015, 28.07.2015, 29.07.2015, 04.08.2015, 05.08.2015 Colony PCR: 17.07.2015, 26.07.2015, 27.07.2015, 28.07.2015, 29.07.2015, 30.07.2015, 05.08.2015, 06.08.2015, 07.08.2015</SMALL></H4><DIV class="row"><DIV class="col-md-7"><P>In colony PCR, primers were placed such as amplicons are 445 bp if the plasmid self-ligated, 723 bp if Gibson assembly of 1 sgRNA cassette worked and 1120 bp if Gibson assembly of 2 sgRNA cassettes worked.</P><P>pdCas9-w-Z0 was obtained by a faulty assembly of pdCas9-w-Z0-Z4 that only took up 1 of the 2 sgRNA cassettes. Sample 20 of the colony PCR (cf. Fig.9a) was confirmed by sequencing to be pdCas9-w-Z0. Sample 13 seems to be pdCas9-w-Z0-Z4 according to colony PCR (cf. Fig.9a), but sequencing showed that it is pdCas9-w-X0-Z0, probably due to contamination of a tube. Colony PCR showed that all colonies tested for pdCas9-w-Z4 have the inserted sgRNA cassette (cf. Fig.9b). Sequencing confirmed that sample 5 is in fact pdCas9-w-Z4. Colony PCR showed that the Gibson assembly for sample 5 of pdCas9-w-Z35 seems to have worked (cf. Fig.9c) and sequencing confirmed that is the case. Colony PCR and sequencing showed that Gibson assembly was successful for sample 5 of pdCas9-w-X0 (cf. Fig.9d). Colony PCR of pdCas9-w-X4 showed many potentially good samples (cf. Fig.9e). Sequencing of sample 10 confirmed that it is in fact pdCas9-w-X4. Colony PCR and sequencing showed that Gibson assembly was successful for the colony of pdCas9-w-X35 (cf. Fig.9d).</P><P>Sequencing showed some mutations and/or deletions in the promoter and/or terminator for some of these samples. Because of time constraints, we decided to keep working with these and see the effect of the mutations during the activity assay.</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.9: Gels of colony PCR of pdCas9-w-sgRNA Gibson assembly products</B><SMALL><B>Fig.9a</B> - Gel of colony PCR of pdCas9-w-Z0-Z4 (lanes 1-10) and self-ligation control pdCas9-w (lane &quot;C&quot;). Lane 2 seems to be pdCas9-w-Z0-Z4 and lanes 5, 8 and 9 seem to only have 1 inserted sgRNA cassette, being either pdCas9-w-Z0 or pdCas9-w-Z4.<B>Fig.9b</B> - Gel of colony PCR of pdCas9-w-Z4 (lanes 1-11). Gibson assembly seems to have worked for all colonies<B>Fig.9c</B> - Gel of colony PCR of pdCas9-w-Z35 (lanes 1-8), seems to have worked for lane 5, self-ligation for all others<B>Fig.9d</B> - Gel of colony PCR of pdCas9-w-X0 (lanes 1-5) and pdCas9-w-X35 (lane 6). Gibson assembly of pdCas9-w-X0 seems to have worked for lanes 2-5, but sample in lane 5 looks slightly better than the others. The faint band in lane 1 is self-ligation. Gibson assembly of pdCas9-w-X35 seems to have worked.<B>Fig.9e</B> - Gel of colony PCR of pdCas9-w-X4 (lanes 1-11). All samples seems to have the sgRNA insert, even though the bands are not very precise.Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV><DIV id="part2" class="panel"><H3>Part 2: Assemble pdCas9-w-sgRNA + sgRNA<SMALL>24.08.2015 - 02.09.2015</SMALL></H3><DIV id="pcrpdcas9sgrna"><H4>Extract sgRNA from pdCas9-w-sgRNA and open pdCas9-w-sgRNA by Q5 PCR<SMALL>24.08.2015, 25.08.2015, 28.08.2015, 31.08.2015</SMALL></H4><DIV class="row"><DIV class="col-md-6"><P>Amplified sgRNA cassettes (Z4, Z35, X4 and X35) are expected to be 400 bp. All were successfully amplified (cf. Fig.10a and Fig.10b).Amplified pdCas9-w-sgRNA (Z0, Z4, X0 and X4) are expected to be about 7000 bp. All were successfully amplified (cf. Fig.10a and Fig.10b).</P></DIV><DIV class="col-md-6"><FIGURE><FIGCAPTION><B>Fig.10</B> - Gels of PCR products of pdCas9-w-sgRNA and sgRNA cassettes<SMALL><B>Fig.10a</B> - Gel of PCR products for the construction of pdCas9-w-sgRNA: pdCas9-w-Z0 (lane 1), pdCas9-w-Z4 (lane 2), sgRNA Z4 (lane 3), pdCas9-w-X0 (lane 4), pdCas9-w-X4 (lane 5) and sgRNA X4 (lane 6), all were successfully amplified.<B>Fig.10b</B> - Gel of PCR products sgRNA Z35 (lane 7) and sgRNA X35 (lane 8), both successfully amplified. (Lanes 1-6 are for the construction of pAraC-sgRNA, see below.)Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV><DIV id="gibsonpdcas9wsgrna2"><H4>Assemble pdCas9-w-sgRNA-sgRNA constructs<SMALL>Gibson assembly, transformation: 26.08.2015, 31.08.2015. Colony PCR: 31.08.2015, 02.09.2015</SMALL></H4><DIV class="row"><DIV class="col-md-6"><P>Colony PCR primers are placed such as amplicons are 724 bp if Gibson assembly worked and are 390 bp if it did not (self-ligation). All necessary constructs were successfully obtained. Following colonies were kept for our constructs: colony #1 for pdCas9-w-Z0-Z4 (cf. Fig.11a), colony #4 (2nd attempt) for pdCas9-w-Z0-Z35 (cf. Fig.11c), colony #6 for pdCas9-w-Z4-Z35 (cf. Fig.11b), colony #4 for pdCas9-w-X0-X4 (cf. Fig.11c), colony #2 for pdCas9-w-X0-X35 (cf. Fig.11d) and colony #1 for pdCas9-w-X4-X35 (cf. Fig.11e).</P></DIV><DIV class="col-md-6"><FIGURE><FIGCAPTION><B>Fig.11</B> - Gels of colony PCR of pdCas9-w-sgRNAs<SMALL><B>Fig.11a</B> - Gel of colony PCR of pdCas9-w-Z0-Z4 (lanes 1-2), successful for colony #1, and colony PCR of pdCas9-w-Z0-Z35, self-ligation for all colonies. Ladder: 1kb Generuler.<B>Fig.11b</B> - Gel of colony PCR of pdCas9-w-Z4-Z35, successful for all colonies. Ladder: 2-log DNA ladder.<B>Fig.11c</B> - Gel of colony PCR of pdCas9-w-Z0-Z35 (2nd attempt), successful for colonies #1 and #4, and colony PCR of pdCas9-w-X0-X4 (part 1), successful for colonies #2 and #4. Ladder: 2-log DNA ladder.<B>Fig.11d</B> - Gel of colony PCR of pdCas9-w-X0-X4 (part 2), maybe successful for colonies #8, #9 and #10 (gel unclear), and colony PCR of pdCas9-w-X0-X35, successful for colony #2. Ladder: 2-log DNA ladder.<B>Fig.11e</B> - Gel of colony PCR of pdCas9-w-X4-X35, successful for colony #1 (maybe others, gel unclear). Ladder: 2-log DNA ladder.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION id="pwj89alt" class="panel"><H1>Construction of pWJ89alt<SMALL>05.08.2015 - 13.08.2015</SMALL></H1><P>pWJ89alt contains GFP controlled by the promoter J23117alt, which can be activated by dCas9-w bound to sgRNA X4 and inhibited by dCas9-w bound to sgRNA X0 or X35.</P><H2>Materials and method</H2><UL><LI><B>Phusion and Q5 PCR (cf. Protocols) pWJ89 without its J23117 promoter</B> with primers f_Rmv_J23117_of_pWJ89 and r_Rmv_J23117_of_pWJ89 + agarose gel electrophoresis of 2 µL PCR products + PCR product purification (cf. Protocols). pWJ89 was a gift from David Bikard.</LI><LI><B>Phusion and Q5 PCR J23117alt promoter</B> with primers f_G_J23117Alt1IDT and r_G_J23117Alt1IDT + agarose gel electrophoresis of 2 µL PCR products + PCR product purification (cf. Protocols). J23117alt was synthesized by IDT.</LI><LI><B>Gibson assembly of pWJ89alt</B> with purified PCR products pWJ89 (without J23117) and J23117alt promoter + transformation (cf. Protocols) of ultra-competent DH5a cells (NEB). Control with colony PCR (cf. Protocols) with primers f_Cl_pWJ89 and r_Sq_J23117alt primers + agarose gel electrophoresis of 2 µL PCR products <B>+</B> sequencing (Microsynth)</LI></UL><H2>Results</H2><DIV id="pcrpwj89" class="panel"><H3>PCR pWJ89<SMALL>05.08.2015</SMALL></H3><DIV class="row"><DIV class="col-md-7"><P>Successful PCR reactions are expected to yield 4400 bp amplicons.Both negative control lanes (cf. Fig.12) are empty, which means there is no contamination. Phusion PCR did not work, but Q5 PCR did (cf. Fig.12). We will work with this sample for next steps.</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.12</B> - Gel of pWJ89 amplified without J23117 promoter by PCR. Lane 1: Phusion PCR, lane 2: Phusion PCR negative control, lane 3: Q5 PCR, lane 4: Q5 PCR negative control.<SMALL>Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="pcrj23117alt" class="panel"><H3>PCR J23117alt promoter<SMALL>05.08.2015</SMALL></H3><DIV class="row"><DIV class="col-md-7"><P>Successful PCR reactions are expected to yield 340 bp amplicons.Both negative control lanes (cf. Fig.13) are empty, which means there is no contamination. Both Phusion and Q5 PCR worked (cf. Fig.13). We will work with the sample amplified by Q5 PCR.</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.13</B> - Gel of J23117alt promoter amplified by PCR. Lane 1: Phusion PCR, lane 2: Phusion PCR negative control, lane 3: Q5 PCR, lane 4: Q5 PCR negative control. <SMALL>Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="gibsonpwj89alt" class="panel"><H3>Gibson assembly of pWJ89alt<SMALL>Gibson assembly, transformation: 10.08.2015, Colony PCR: 11.08.2015</SMALL></H3><P>Colony PCR showed two colonies for which Gibson assembly worked. However, images of gels of colony PCR were lost for technical reasons.Sequencing confirmed that both colonies were in fact pWJ89alt and that one had a deletion but the other one did not have any mutations. We kept the mutation-free colony and stored it in a Glycerol stock (cf. Protocols).</P></DIV></SECTION><SECTION id="arac" class="panel"><H1>Construction of pAraC-sgRNA<SMALL>25.08.2015 - 03.09.2015</SMALL></H1><P>We made 3 pAraC-sgRNA constructs: pAraC-Z0 and pAraC-Z4 contain sgRNA Z0 or Z4 producing cassettes controlled by promoter pBAD-AraC (positively regulated by arabinose) and pAraC-Z0-Z4 contains sgRNA Z0 producing cassette controlled by promoter pBAD-AraC and sgRNA Z4 producing cassette controlled by promoter pBAD (constitutively active).</P><H2>Materials and method</H2><UL><LI><B>Linearize pAraC by Q5 PCR</B> (cf. Protocols) with primers f_opn_pAraC and r_opn_pAraC + agarose gel electrophoresis of 2 µL PCR products + PCR product purification (cf. Protocols). pAraC, called pFIP on Addgene, was a gift from Jay Keasling (Addgene plasmid #17565) [3].</LI><LI><B>Extract sgRNA from pdCas9-w-sgRNA by Q5 PCR</B> (cf. Protocols) with primers as indicated in table below  + agarose gel electrophoresis of 2 µL PCR products + PCR product purification (cf. Protocols)</LI><LI><B>Gibson assembly of pAraC-sgRNA constructs</B> with purified PCR products pAraC and sgRNA + transformation (cf. Protocols) of ultra-competent DH5a cells (NEB). Control with colony PCR (cf. Protocols) with primers f_Sqc2_CoP_pAraCsgRNAs_sg1 and r_CoP_pAraCsgRNAs primers + agarose gel electrophoresis of 2 µL PCR products <B>+</B> sequencing (Microsynth)</LI></UL><TABLE><TBODY><TR><TH>Construct</TH><TH>sgRNA cassette(s)</TH><TH>Forward primer(s) / Reverse primer(s)
          </TH></TR><TR><TD>pAraC-Z0</TD><TD>Z0</TD><TD>f_Gbs_sgRNAinpAraC / r_Gbs_sgRNAinpAraC</TD></TR><TR><TD>pAraC-Z4</TD><TD>Z4</TD><TD>f_Gbs_sgRNAinpAraC / r_Gbs_sgRNAinpAraC</TD></TR><TR><TD>pAraC-Z0-Z4</TD><TD>Z0Z4</TD><TD>f_Gbs_sgRNAinpAraC / r_Gbs_sgRNAwoAraf_Gbs_sgRNAwoAra_VS / r_Gbs_sgRNAinpAraC</TD></TR></TBODY></TABLE><H2>Results</H2><DIV id="pcrparac" class="panel"><H3>PCR pAraC and sgRNA<SMALL>25.08.2015, 26.08.2015</SMALL></H3><DIV class="row"><DIV class="col-md-7"><P>Successfully amplified PCR products should be 5612 bp for pAraC and 370 bp for the sgRNAs. pAraC was successfully amplified and we will use sample #1 for the next steps (cf. Fig.14b). sgRNAs Z0 and Z4 were successfully amplified with different sets of primers, as indicated in table above (cf. Fig.14a).</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.14</B> - Gels of pArac and sgRNA PCR products<SMALL><B>Fig.14a</B> (same as Fig.10b) - Gel of PCR products pAraC (lanes 1-2), unsuccessfully amplified, and sgRNA Z0 and Z4 (1) for pAraC-Z0 and pAraC-Z4 (lanes 3-4), sgRNA Z0 and Z4 (2) for pAraC-Z0-Z4 (lanes 5-6), successfully amplified. (Lanes 7-8 are for the construction of pdCas9-w-sgRNA, see above.)<B>Fig.14b</B> - Gel of pAraC PCR product, both successfully amplified but sample #1 is clearer.Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="gibsonparac" class="panel"><H3>Assemble pAraC-sgRNA constructs<SMALL>Gibson assembly and transformation: 28.08.2015, 01.09.2015, Colony PCR: 31.08.2015, 02.09.2015</SMALL></H3><P>Colony PCR (pictures of gels are were not very conclusive) and sequencing confirmed Gibson assembly was successful for one colony for each construct.</P></DIV></SECTION><SECTION id="plink" class="panel"><H1>Construction of pWJ89alt_Z4-to-X4<SMALL>18.08.2015 - 02.09.2015</SMALL></H1><P>pWJ89alt_Z4-to-X4 contains a cassette producing sgRNA X4 (activator for J23117alt) controlled by the promoter J23117 and GFP under the J23117alt promoter (corresponds to pWJ89alt).</P><H2>Materials and method</H2><UL><LI><B>Open pWJ89alt by restriction digest with AfeI</B> + Agarose gel electrophoresis of 2 µL of digested product + Gel extraction (cf. Protocols)</LI><LI><B>Amplify Z4-to-X4 by Q5 PCR</B> (cf. Protocols) with primers f_Gb_pWJ89alt and r_Gb_pWJ89alt + agarose gel electrophoresis of 2 µL PCR products + PCR product purification (cf. Protocols)</LI><LI><B>Gibson assembly of pWJ89alt_Z4-to-X4</B> with linearized pWJ89alt and purified PCR product of Z4-to-X4 + transformation (cf. Protocols) of ultra-competent DH5a cells (NEB). Control with colony PCR (cf. Protocols) with primers f_Cl_pWJ89 and r_Sqc_T2reporter + agarose gel electrophoresis of 2 µL PCR products <B>+</B> sequencing (Microsynth)</LI></UL><H2>Results</H2><DIV id="openpwj89alt" class="panel"><H3>Open pWJ89alt by restriction digest<SMALL>18.08.2015</SMALL></H3><DIV class="row"><DIV class="col-md-7"><P>Undigested pWJ89alt is expected to migrate faster than digested pWJ89alt. We observe both bands at the same height (cf. Fig.15). As there is a smear for the digested sample, it may have been partially digested.</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.15</B> - Gel of pWJ89alt digested by AfeI (lane 1) and undigested (lane 2).<SMALL>Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="pcrz4-to-x4" class="panel"><H3>PCR Z4-to-X4<SMALL>19.08.2015</SMALL></H3><DIV class="row"><DIV class="col-md-7"><P>Amplicons should be 425 bp. We did 4 samples with slightly different thermocycling settings. All were successfully amplified (cf. Fig.16) and we kept sample in lane 1 for next steps.</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.16</B> - Gel of Z4-to-X4 PCR products. Different samples had different thermocycling settings. PCR was successful for all samples.<SMALL>Ladder: 1kb Generuler.</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV id="#gibsonplink" class="panel"><H3>Gibson assembly pWJ89alt_Z4-to-X4<SMALL>Gibson assembly and transformation: 28.08.2015, Colony PCR: 31.08.2015</SMALL></H3><DIV class="row"><DIV class="col-md-7"><P>Colony PCR primers were placed such as colonies for which the Gibson assembly worked would yield 2000 bp amplicons and those for which the Gibson assembly did not work (self-ligation) would yield 1500 bp amplicons.</P><P>The gel of the colony PCR shows that Gibson assembly worked for 3 of the tested colonies (cf. Fig.17). However, the other lanes do not show self-ligation, it seems like PCR did not work. We did not investigate this further.Sequencing of pWJ89alt_Z4-to-X4 from one of the 3 colonies confirmed that Gibson assembly did work and did not reveal any mutations.</P></DIV><DIV class="col-md-5"><FIGURE><FIGCAPTION><B>Fig.17</B> - Gel of colony PCR products of pWJ89alt_Z4-to-X4. Colony PCR shows Gibson worked for colonies tested in lanes 3, 5 and 6. <SMALL>Ladder: 2-log DNA ladder</SMALL></FIGCAPTION></FIGURE></DIV></DIV></DIV></SECTION><SECTION id="activityassay" class="panel"><H1>Transistor activity assay</H1><P>To do activity assays, we transformed (cf. Transformation Protocol) JEN202 cells with different constructs and measured the fluorescence with a plate reader or by flow cytometry. Results can be found on our <A href="https://2015.igem.org/Team:EPF_Lausanne/Results">Results</A> page. A more detailed protocol can be found in this <A href="https://docs.google.com/spreadsheets/d/143C4c2Au-__z6s9XR80Y6LGBv4h3EFTPyevmLi3bjhs/edit#gid=1765145613"> sheet </A></P></SECTION><SECTION id="primers" class="panel"><H1>Primer table</H1><TABLE><TBODY><TR><TH>Name</TH><TH>Sequence</TH><TH>Associated part</TH></TR><TR><TD>f_Cl_pWJ89</TD><TD>gtgagcggataacaatttcacac</TD><TD>pWJ89/pWJ89alt</TD></TR><TR><TD>f_ColPCR_sgRNAs</TD><TD>GGAAGCCGCTGAATTACAAGCCG</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>f_G_J23117Alt1IDT</TD><TD>acatcctgtccgtccGGCGTATCACGAGGCCCT</TD><TD>J23117alt</TD></TR><TR><TD>f_Gb_pWJ89alt</TD><TD>caggtttgccggctgaaagcAAGCAGAGGAGCAAAAGC</TD><TD>pWJ89alt</TD></TR><TR><TD>f_Gbs_pdCas9</TD><TD>CTCGAGTAAGGATCTCCAG</TD><TD>pdCas9</TD></TR><TR><TD>f_Gbs_sgRNA-A</TD><TD>ctcgctcactgactcgctac</TD><TD>sgRNA cassettes</TD></TR><TR><TD>f_Gbs_sgRNAinpAraC</TD><TD>acaaagcgggaccaaagccatga</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>f_Gbs_sgRNAinPd</TD><TD>atttatctcttcaaatgtagcacctgaagtctcgctcactgactcgct</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>f_Gbs_sgRNAwoAra</TD><TD>TTATAACATATCGTTCCATGAAGGCCAGAAgcgggaccaaagccatga</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>f_Gbs_w</TD><TD>ACACGCATTGATTTGAGTCA</TD><TD>pWJ66</TD></TR><TR><TD>f_Mt_A2080C_pdCas9-w</TD><TD>TGACTTTTCGcATTCCTTATTATGTTG</TD><TD>pdCas9-w</TD></TR><TR><TD>f_opn_d9w</TD><TD>cagccccatacgatataagt</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>f_opn_pAraC</TD><TD>tcactatagggcgaattggag</TD><TD>pAraC</TD></TR><TR><TD>f_Sq_CoP_pdCas92ndIns</TD><TD>cctccaagccagttacctcg</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>f_Sqc2_CoP_pAraCsgRNAs_sg1</TD><TD>aacctttcattcccagcggt</TD><TD>pAraC-sgRNA</TD></TR><TR><TD>f_Rmv_J23117_of_pWJ89</TD><TD>GGATTGTGCTAGCgAATTCCT</TD><TD>pWJ89</TD></TR><TR><TD>r_G_J23117Alt1IDT</TD><TD>ttcgctagcacaatccGCTAGCACAATCCCGATATCTC</TD><TD>J23117alt</TD></TR><TR><TD>r_ColPCR_sgRNAs</TD><TD>cgcctggtatctttatagtcctgtcggg</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>r_CoP_pAraCsgRNAs</TD><TD>ggaataagggcgacacgg</TD><TD>pAraC-sgRNA</TD></TR><TR><TD>r_G_J23117Alt1IDT</TD><TD>ttcgctagcacaatccGCTAGCACAATCCCGATATCTC</TD><TD>J23117alt</TD></TR><TR><TD>r_Gb_pWJ89alt</TD><TD>gcaattctggaagaaatagcAAAAAAAGCACCGACTCG</TD><TD>pWJ89alt</TD></TR><TR><TD>r_Gbs_pdCas9</TD><TD>GTCACCTCCTAGCTGACTC</TD><TD>pdCas9</TD></TR><TR><TD>r_Gbs_sgRNA-B</TD><TD>tggcatcttccaggaaatc</TD><TD>sgRNA cassettes/td&gt;
        </TD></TR><TR><TD>r_Gbs_sgRNAinpAraC</TD><TD>cggccccagctccaattcgccctatagtgactggcatcttccaggaaatctcc</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>r_Gbs_sgRNAinPd</TD><TD>actagtaacaacttatatcgtatggggctgccgttcgtaagccatttccg</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>r_Gbs_sgRNAwoAra</TD><TD>TTCTGGCCTTCATGGAACGATATGTTATAAtggcatcttccaggaaatctcc</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>r_Gbs_w</TD><TD>atttgatgcctggagatccttactcgagTTAACGACGACCTTCAGCA</TD><TD>pWJ66</TD></TR><TR><TD>r_Mt_A2080C_pdCas9-w</TD><TD>AGATTTTTTCAATCTTCTCACG</TD><TD>pdCas9-w</TD></TR><TR><TD>r_opn_d9w</TD><TD>acttcaggtgctacatttga</TD><TD>pdCas9w-w-sgRNAS</TD></TR><TR><TD>r_opn_pAraC</TD><TD>gtttttgtcatggctttggtcc</TD><TD>pAraC</TD></TR><TR><TD>r_Rmv_J23117_of_pWJ89</TD><TD>ggacggacaggatgtatgct</TD><TD>pWJ89</TD></TR><TR><TD>r_Scq_pdCas9_w_sgRNA</TD><TD>ctgatttgagcgtcagat</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>r_Sq_CoP_pAraC2ndInsert</TD><TD>gtactgcgatgagtggcagg</TD><TD>pdCas9-w-sgRNA</TD></TR><TR><TD>r_Sq_J23117alt</TD><TD>ATATGATCTGGGTATCTCGCAAAG</TD><TD>pWJ89/pWJ89alt</TD></TR><TR><TD>r_Sqc_T2reporter</TD><TD>gaaacaggcgatgctgctta</TD><TD>pWJ89alt</TD></TR></TBODY></TABLE></SECTION><SECTION id="ref" class="panel"><H1>References</H1><P>[1] Repurposing CRISPR as an RNA-Guided Platform for Sequence-Specific Control of Gene Expression. Qi LS, Larson MH, Gilbert LA, Doudna JA, Weissman JS, Arkin AP, Lim WA. Cell. 2013 Feb 28;152(5):1173-83. doi: 10.1016/j.cell.2013.02.022. 10.1016/j.cell.2013.02.022 PubMed 23452860[2] Programmable repression and activation of bacterial gene expression using an engineered CRISPR-Cas system. Bikard D, Jiang W, Samai P, Hochschild A, Zhang F, Marraffini LA. Nucleic Acids Res. 2013 Jun 12. 10.1093/nar/gkt520 PubMed 23761437[3] A tightly regulated inducible expression system utilizing the fim inversion recombination switch. Ham TS, Lee SK, Keasling JD, Arkin AP. Biotechnol Bioeng. 2006 May 5. 94(1):1-4. 10.1002/bit.20916 PubMed 16534780</P></SECTION></DIV></DIV></DIV><TITLE>EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits</TITLE></DIV></DIV></DIV></DIV></BODY></HTML>