Team:EPF Lausanne/PolicyAndPractices
Toggle navigation
Home
Project
Background
Description
Applications
Modelling
Achievements
In vivo
In silico
Judging
Parts
Policy and Practices
Ethics
Safety
Human Practices
Notebook
E. Coli
Yeast
Protocols
Team
Meet us
Timeline
Acknowledgements
Human Practices
Ethics
Safety
Legal
Security
Public
Assemble pdCas9-w-sgRNAs
Open pdCas9-w by restriction digest
Amplify sgRNA fragments by PCR
Assemble pdCas9-w-sgRNA constructs by Gibson assembly
Assemble pWJ89alt1
Assemble pdCas9-w
pdCas9-w was constructed using PCR and Gibson assembly from the following plasmids:
pdCas9 (pdCas9-bacteria, plasmid #44249, Addgene): dCas9 under a Tetracyclin inducible promoter + Chlorampheicol resistance gene
pWJ66 (pWJ66, plasmid #46570, Addgene): tracrRNA + w-dCas9 (dCas9 fused at its C-terminal to rpoZ, which encodes for the w subunit of RNA polymerase) + CRISPR array
pdCas9 was opened by PCR and the w subunit was extracted from pWJ66 by pCR. The w subunit was then fused to the C-terminal of dCas9 by Gibson assembly, the final product is pdCas9-w.A site-directed mutagenesis was done on this plasmid to eliminate an EcoRI restriction site. This will allow us to submit pdCas9-w as a BioBrick.
Assemble pdCas9-w
Open pdCas9 by PCR
We received plasmid pdCas9 in bacteria and did a Miniprep (cf. Protocols) on overnight cultures to isolate it. This step consists of linearizing pdCas9 by PCR.
Materials and method
20 µl Phusion PCR (cf. Protocols) of pdCas9 with primers f_Gbs_pdCas9 and r_Gbs_pdCas9
PCR product purification (cf. Protocols)
Agarose gel electrophoresis of purified PCR products
Results
Linearized pdCas9-w is expected to be 6705 bp.The first try of this PCR was unsuccessful (gel not shown here). For our second try, we tested many parameters: HF vs. GC buffer, different annealing temperatures and different extension times. This time, many, but not all, of our samples were successfully amplified (cf. figure 1). The difficulty of this PCR is probably due to the fact that the size of the ampicon is very long, almost 7 kb.For next steps, sample from lane 1 (cf. figure 1) was used.
Assemble pdCas9-w
Extract w subunit from pWJ66 by PCR
We received plasmid pWJ66 in bacteria and did a Miniprep (cf. Protocols) on overnight cultures to isolate it. This step consists of extracting rpoZ, the w subunit, from pW66 by PCR.
Materials and method
20 µl Phusion PCR (cf. Protocols) of pWJ66 using primers f_Gbs_w and r_Gbs_w.
PCR product purification (cf. Protocols)
Agarose gel electrophoresis of purified PCR products
Results
Successful PCR reactions are expected to yield 340 bp fragments.As seen on gel, PCR was succesful for sample in lane 1.
Assemble pdCas9-w
Gibson assembly of pdCas9-w
This step consists of fusing the w subunit to dCas9 by Gibson assembly, using the PCR products from previous steps. We transformed cells with the Gibson assembly product and tested for colonies that contain the construct by colony PCR, restriction digest and sequencing.
Materials and method
Gibson assembly (cf. Protocols) with purified PCR products:
Open pdCas9: 0.02 pmol = 95 ng
w subunit extracted from pWJ66: 0.06 pmol = 12.5 ng
Transformation (cf. Protocols) of ultra-competent DH5a cells (NEB) with Gibson assembly product, spreading on Chloramphenicol plates
Colony PCR (cf. Protocols) of colonies from plate used for culture of transformed cells with primers f_Gbs_w and r_Scq_pdCas9_w_sgRNA primers.
Miniprep (cf. Protocols) of overnight liquid cultures in 5 mL LB + Chloramphenicol of colonies from plate used for culture of transformed cells to isolate pdCas9-w plasmids
Restriction digest (Cf. Protocols) of pdCas9-w plasmids with BamHI and KpnI seperately
Sequencing (Microsynth) of one colony for which Gibson assembly worked according to colony PCR and restriction analysis
Results
Colony PCR
Primers were placed such as amplicons are 666 bp if Gibson assembly worked and 396 bp if the plasmid self-ligated.Lane "C" is a negative control: PCR was run with all components except template DNA, ie. cells. It is empty, this means there is no contamination.Gibson assembly seems to have worked for some of our samples (cf. figure 3 (a, b, c)).To avoid working with too many samples, we kept the ones from lanes 16, 22, 25, 31, 37 and 43 for the Minipreps and restriction digest. We did an overnight liquid culture of these colonies.
Restriction digest
pdCas9-w samples from different colonies are present on gels in triplicates: in the following order:
Undigested: expected to yield 7 kb circular plasmid (migrates faster than linear fragments of the same size)
Digested by BamHI: expected to yield two fragments of 6147 bp and 834 bp if insert is present or one 6705 bp fragment if it is not
Digested by KpnI: expected to yield two fragments of 45334 bp and 2447 bp if insert is present or one 6705 bp fragment if it is not
BamHI and KpnI are both unique cutters in pdCas9 (without the insert) and double cutters in pdCas9-w (with the insert).Too much ladder was loaded so it is difficult to estimate the size of the fragmnts. The smaller fragments are also very difficult to see.By looking at the relative heights, we can say that all colonies seem to have the w subunit insert. There is only the undigested sample for colony 16 that is not visible, probably due to an error while loading the gel. Colony 22 was kept for next steps, it was stored in a glycerol stock (c.f. Protocols).
Sequencing
As dCas9-w is very long, only part of it was sequenced (the w subunit and its surrounding base pairs). No mutations were detected.
Assemble pdCas9-w
Site-directed mutagenesis of dCas9-w
It was noticed after assembly of pdCas9-w that dCas9 contains an EcoRI restriction site, which makes it "un-BioBrick-able". Since it is an important part for our project, we decided to remove the restriction site by site-directed mutagenesis to be able to submit it as a functional BioBrick.
Materials and method
Coming soon
Procedure
Coming soon
EPFL 2015 iGEM bioLogic Logic Orthogonal gRNA Implemented Circuits
