<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT">Home</A></DIV><DIV class="dropdown"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Team">Team</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Team">Team Members</A><A href="https://2018.igem.org/Team:MIT/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><DIV class="sub"><A class="active" href="https://2018.igem.org/Team:MIT/Description">Project</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Design">Design</A><A href="https://2018.igem.org/Team:MIT/Results">Results</A><A href="https://2018.igem.org/Team:MIT/InterLab">InterLab</A><A href="https://2018.igem.org/Team:MIT/Notebook">Notebook</A><A href="https://2018.igem.org/Team:MIT/Experiments">Protocols</A><A href="https://2018.igem.org/Team:MIT/Attributions"> Attributions </A></DIV></DIV><DIV class="dropdown"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Parts">Parts</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Basic_Part">Basic Parts</A><A href="https://2018.igem.org/Team:MIT/Composite_Part">Composite Parts</A></DIV></DIV><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Safety">Safety</A></DIV><DIV class="dropdown"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Human_Practices">Human Practices</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:MIT/Public_Engagement">Public Engagement</A></DIV></DIV><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Model">Model</A></DIV></DIV><DIV class="h3" style="position: absolute; top: 7vw; left: 2vw; margin-right: 4vw;">
Design
</DIV><DIV class="h4" style="position: absolute; top: 11vw; left: 2vw; margin-right: 4vw;">
System Design Overview
</DIV><DIV class="p" style="position: absolute; top: 14vw; left: 2vw; margin-right: 4vw;">
Our goal was to port the S. mutans ComCDE system into HEK293T mammalian cells. We wish to equip mammalian cells with the ability to sense and disrupt S. mutans biofilm formation. ComCDE is a two-component signaling (TCS) system, a type of bacterial quorum sensing system). The sensing system has two main components: ComD (a histidine kinase receptor protein) and ComE (a response regulator), whose native functions are described in detail in the ComCDE page. ComE binds to a promoter sequence. ComD is activated by the Competence-Stimulating Peptide (CSP). Below, we describe key design choices, including the modifications made to the native pieces of the ComCDE pathway to optimize its expression and functionality in mammalian cells. 
</DIV><DIV class="h4" style="position: absolute; top: 75vw; left: 2vw; margin-right: 4vw;">
ComD Design
</DIV><DIV class="p" style="position: absolute; top: 78vw; left: 2vw; margin-right: 4vw;">
In order to adapt ComD to mammalian cells, we made several modifications. First, we chose ComD amino acid sequences from three common strains of S. mutans native to the oral cavity (UA159, SM6, and NN2025) and used the IDT codon optimization tool to human optimize the DNA sequence of each protein. Second, we fused CD4 signaling peptide (BBa_I712009) to the 5’ prime end of comD in order to guide comD to expression on the membrane of HEK cells. Finally, we fused the coding sequence of mKO2, a fluorescent protein, 3’ prime so that we could easily track the location of the comD protein.
</DIV><DIV class="h4" style="position: absolute; top: 95vw; left: 2vw; margin-right: 4vw;">
ComE Design
</DIV><DIV class="p" style="position: absolute; top: 98vw; left: 2vw; margin-right: 4vw;">
The ComE protein has two main structural domains - the acceptor domain and the effector domain. The acceptor domain is a protein domain unique to ComE which contains the aspartic acid that is the target of phosphorylation. The effector domain binds to DNA. It is characterized as a LytR domain and is widely conserved across several subgroups of TCS response regulators but is not extremely well understood.<SUP>1</SUP>
In order to adapt ComE to mammalian cells, we made several modifications. First, we chose ComE amino acid sequences from three common strains of S. mutans native to the oral cavity (UA159, SM6, and NN2025) and used the IDT codon optimization tool to human optimize the DNA sequence of each protein. Additionally, we fused ComE C terminal to a nuclear localization sequence (NLS), VP16 activating domain, and a nuclear export sequence (NES). ComE’s native S. mutans don’t have a nucleus, so we believed that adding an NLS and NES would assist in shuttling the protein to and from the mammalian nucleus, to increase interactions with DNA (in the nucleus) and ComD (in the cell membrane). Since transcription machinery differs in prokaryotes and eukaryotes, we also fused the VP16 transcription activating domain to the ComE C terminal to ensure that ComE would activate transcription upon binding DNA. The NLS, VP16, and NES sequences were taken from Systematic Transfer of Prokaryotic Sensors and Circuits to Mammalian Cells (Weiss and Voigt et al., 2014. DOI: 10.1021/sb5002856).
</DIV><DIV class="p" style="position: absolute; top: 135vw; left: 6vw; margin-right: 4vw;"><B>Constitutively Active ComE:</B> We utilized two approaches to design constitutively active ComE control proteins, so we could experimentally validate that ComE was driving the correct promoter once ported into HEK cells. This validation isolates the ComE promoter interaction so that we could test for ComE and promoter functionality without needing CSP-ComD functional interactions. 

1.) Our first approach was to make a variation of each of the three ComE proteins with a D60E phosphomimetic amino acid substitution, in order to mimic the constitutively phosphorylated state of ComE. The 60th amino acid is the aspartic acid phosphorylation site, and glutamic acid is a typical aspartate phosphomimetic. This approach was previously used in S. mutans ComE studies.<SUP>2</SUP>
2.)  The second approach was to isolate the effector region of ComE. In some response regulators, the acceptor portion blocks the effector portion from binding DNA in the deactivated state, so isolating the effector region can create a mimic of the activated protein.
</DIV><DIV class="h4" style="position: absolute; top: 169vw; left: 2vw; margin-right: 4vw;">
Competence Stimulating Peptide (CSP) Design
</DIV><DIV class="p" style="position: absolute; top: 172vw; left: 2vw; margin-right: 4vw;">
We designed a synthetic CSP fused to a fluorophore (Alexa-fluor 405) to allow for visualization of ligand binding process with ComD. 

</DIV><DIV class="h4" style="position: absolute; top: 178vw; left: 2vw; margin-right: 4vw;">
ComE Binding Promoter Design
</DIV><DIV class="p" style="position: absolute; top: 181vw; left: 2vw; margin-right: 4vw;">
The ComE binding site typically includes 37bp with two 11bp direct repeat site with consensus sequence TCBTAAAYSGT.
</DIV><DIV class="p" style="position: absolute; top: 185vw; left: 6vw; margin-right: 4vw;">
1.)  To design a comE binding promoter, we chose 3 comE binding sites with different binding affinity from comE downstream gene’s promoter sequence. Specifically, the high, medium and low-affinity binding sequences are from comC, ftf and gtfc promoter with IC50 value 2.9 ± 0.4, 11 ± 1.0 and 15 ± 3.8 (10<SUP>-9</SUP> M) respectively<SUP>3</SUP>. In each engineered promoter, we fused three repeated comE binding sites at 5’ prime and a minCMV promoter at 5’ prime, generating 3 comE binding promoter with 3 levels of binding affinity. We included three repeated comE binding site to increase the likelihood that comE binds to the promoter because comE have to shuttle back and forth between nucleus and cytoplasm. Also, multiple binding site is needed because comE is activated as oligomer.
 MinCMV promoter is included at 3’ prime so that when comE-VP16 binds to promoter, VP16 will recruit the transcription complex to minCMV promoter and activates downstream gene transcription of actuation protein.

2.)  The second approach was to isolate the effector region of ComE. In some response regulators, the acceptor portion blocks the effector portion from binding DNA in the deactivated state, so isolating the effector region can create a mimic of the activated protein.
</DIV><DIV class="footer" style="position: absolute; top: 222vw; left: 2vw; margin-right: 4vw;">
References:

1. Nikolskaya, A.N.,  Galperin, M.Y. (2002) A novel type of conserved DNA-binding domain in the transcriptional regulators of the AlgR/AgrA/LytR family. <I>Nucleic Acids Res.</I> 30(11): 2453–2459.

2. Hung, D.C. et al. (2012) Oligomerization of the response regulator ComE from Streptococcus mutans is affected by phosphorylation. <I>J Bacteriol.</I> 194(5):1127-35.

3. Hung, D.C. et al. Characterization of DNA Binding Sites of the ComE Response Regulator from Streptococcus mutans. <I>J Bacteriol.</I></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>