<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT">Home</A></DIV><DIV class="dropdown"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Team">Team</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Team">Team Members</A><A href="https://2018.igem.org/Team:MIT/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><DIV class="sub"><A class="active" href="https://2018.igem.org/Team:MIT/Description">Project</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Design">Design</A><A href="https://2018.igem.org/Team:MIT/Results">Results</A><A href="https://2018.igem.org/Team:MIT/InterLab">InterLab</A><A href="https://2018.igem.org/Team:MIT/Notebook">Notebook</A><A href="https://2018.igem.org/Team:MIT/Experiments">Protocols</A><A href="https://2018.igem.org/Team:MIT/Attributions"> Attributions </A></DIV></DIV><DIV class="dropdown"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Parts">Parts</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Basic_Part">Basic Parts</A><A href="https://2018.igem.org/Team:MIT/Composite_Part">Composite Parts</A></DIV></DIV><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Safety">Safety</A></DIV><DIV class="dropdown"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Human_Practices">Human Practices</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:MIT/Public_Engagement">Public Engagement</A></DIV></DIV><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Model">Model</A></DIV></DIV><DIV class="h3" style="position: absolute; top: 5vw; left: 40vw;">Background</DIV><DIV class="p1" style="position: absolute; top: 13vw; left: 5vw;">
        As a chronic but non-lethal condition, dental caries are often overlooked as a target for modern therapeutics, and the quality of
        dental caries treatment has lagged behind that of more life-threatening diseases such as cancer. However, it is a well established
        fact that oral health affects overall health--for example, severe cavities can lead to systemic illnesses such
        as heart disease. Therefore, dental caries is still a prevalent and critical issue.
        
        The process of cariogenesis is highly facilitated by the growth of dense, sticky biofilm on the surface of teeth. This bioflim
    cuts off the bacteria's access to oxygen, thereby forcing them to revert to anaerobic respiration. Lactic acid is released as a
    byproduct of bacterial respiration, and leads to demineralization of the enamel and cavity formation. </DIV><DIV class="h3" style="position: absolute; top: 59vw; left: 38vw;">Biofilm Formation</DIV><DIV class="p1" style="position: absolute; top: 67vw; left: 5vw; ">


        In its natural, non-pathogenic state, <I>S. mutans</I> lives planktonically (free-floating).
        The transition from the planktonic to biofilm state requires expression of virulence genes, which is coordinated via
        quorum sensing: a mode of communication employed by many bacteria to coordinate gene expression
        and synchronous behavior across a population or species. In the case of <I>S. mutans</I>, quorum sensing results in the expression of
        genes coding for extracellular glucosyltransferase (GTF) enzymes. These enzymes synthesize adhesive glucans, which are used by
        the bacteria to adhere both to each other and to the enamel of the tooth.
        
        Most quorum-sensing systems involve small, diffusible peptides which are sent out as a signal by one bacteria and
        received by another. The peptides are detected by a receptor on the membrane or in the cytoplasm of other bacteria, which then
        initiate a signal transduction pathway that upregulates the quorum-sensing peptide, as well as other genes related to the coordinated
        behavior of the population. Once enough of the signal has been received by the majority of the population, bacterial behavior
        shifts to perform a synchronized activity.Biofilm formation is one of the synchronized behaviors initiated by quorum sensing.
        During this process, <I>S. Mutans</I> create and perpetuate the Competence Stimulating Peptide (CSP), a small molecule that is used to
        initiate the formation of dental plaque. The pathway through which the cells sense and respond to CSP is referred to the ComCDE pathway. </DIV><DIV class="h3" style="color: #410081; position: absolute; top: 120vw; left: 48vw;">Quorum Sensing in <I>S. mutans</I></DIV><DIV class="dropdown1"><DIV class="dropdown-content1"><DIV class="p2" style="position: absolute; top: 130vw; left: 42vw;">Competence Stimulating Peptide (CSP)
        
        CSP is an 18 amino acid long peptide that binds specifically to ComD, leading to activation of the TCS pathway. 

         </DIV></DIV></DIV><DIV class="dropdown1"><DIV class="dropdown-content1"><DIV class="p2" style="position: absolute; top: 130vw; left: 42vw;">ComD
                
                ComD is the transmembrane histidine kinase receptor and homo-dimeric protein. It has 6 transmembrane segments with three extracellular loops and 1 intracellular kinase domain.<SUP>1</SUP> ComD sense the S.mutans density based on the concentration of CSP (competence signaling peptide) that S.mutans population secrets into the environment. When CSP gets to a critical concentration, ComD trans-autophosphorylates between two dimer’s kinase domain histidine and then transfers the phosphate group from histidine to an aspartic acid on ComE (the response regulator protein).<SUP>2</SUP></DIV></DIV></DIV><DIV class="dropdown1"><DIV class="dropdown-content1"><DIV class="p2" style="position: absolute; top: 130vw; left: 42vw;">ComE
                
                ComE is the response regulator protein of the TCS pathway. When ComD is activated by CSP binding, a phosphate group is transferred from ComD to ComE. This phosphorylation activates ComE, enabling it to drive a ComE specific promoter. The phosphorylation likely contributes to ComE activation by enabling oligomerization of the protein, which may assist in facilitating DNA binding activity or DNA bending to allow access to RNA polymerase.<SUP>3,4</SUP></DIV></DIV></DIV><DIV class="dropdown1"><DIV class="dropdown-content1"><DIV class="p2" style="position: absolute; top: 130vw; left: 42vw;">Genetic Response
                
                In S.mutans, comE binds to multiple genes that are involved in genetic competence development, biofilm formation, and bacteriocin secretion.<SUP>5</SUP>Among the downstream genes of comE, gtfC is one of the proteins that is involved in biofilm formation by catalyzing the reaction that transforms glucose to adhesive glucan. 
            </DIV></DIV></DIV><DIV class="p3" style="position: absolute; top: 176vw; left: 3vw;"><I>(hover over each element to learn about its role)</I></DIV><DIV class="h3" style="position: absolute; top: 181vw; left: 5vw;">Our Synthetic Circuit</DIV><DIV class="p1" style="position: absolute; top: 186vw; left: 5vw; margin-right: 58vw;">
        Our goal is to port the ComCDE system into Human Embryonic Kidney (HEK) cells. This would allow our engineered cells
        to sense the presence of CSP and therefore detect when biofilm formation is about to occur. ComD and ComE will be
        expressed constitutively, while our three outputs (ScFv's P126 and P136, and Kappa casein) will be expressed under the ComE
        promoter, which is activated by the ComE response regulator protein as described above.
    </DIV><DIV class="h3" style="position: absolute; top: 222vw; left: 5vw;">Output</DIV><DIV class="p1" style="position: absolute; top: 228vw; left: 5vw; margin-right: 5vw;">
        Upon activation of our biofilm-sensing system, our cells will output proteins that have been shown to
        reduce the biofilm-forming capabilities of <I>S. mutans</I>.
    </DIV><DIV class="p2" style="position: absolute; top: 247vw; left: 5vw; margin-right: 5vw;">
        Our actuation protein of choice is kappa casein, a protein derived from bovine milk. It interferes with the adhesive power of the
        glucans post synthesis through a combination of hydrophilicity and negative charge.<SUP>6</SUP></DIV><DIV class="footer" style="position: absolute; top: 258vw; left: 2vw; margin-right: 4vw;">
References:

1. Dong, G., et al. (2016) Membrane Topology and Structural Insights into the Peptide Pheromone Receptor ComD, A Quorum-Sensing Histidine Protein Kinase of Streptococcus mutans. <I>Sci Rep.</I>
2. West, A.H., et al. (2001) Histidine kinases and response regulator proteins in two-component signaling systems. <I>Trends Biochem Sci.</I> 26(6):369-76.

3. Hung, D.C., et al. (2012) Oligomerization of the Response Regulator ComE from Streptococcus mutans Is Affected by Phosphorylation. <I>J Bacteriol.</I> 194(5): 1127–1135.

4. Boudes, M, et al. (2014) Structural insights into the dimerization of the response regulator ComE from Streptococcus pneumoniae. <I>Nucleic Acids Research.</I> Volume 42: Issue 8.

5. Hung, D.C., et al. Characterization of DNA Binding Sites of the ComE Response Regulator from Streptococcus mutans. <I>J Bacteriol.</I>
6.  Vacca-Smith, A.M., Van Wuyckhuyse, B.C., Tabak, L.A., Bowen, W.H. (1994). The effect of milk and casein proteins on the adherence of Streptococcus mutans to saliva-coated hydroxyapatite. <I>Archives of Oral Biology</I>, Volume 39: Issue 12. 1063-1069.
</DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>