<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_MIT_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:MIT/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT">Home</A></DIV><DIV class="dropdown"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Team">Team</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Team">Team Members</A><A href="https://2018.igem.org/Team:MIT/Collaborations">Collaborations</A></DIV></DIV><DIV class="dropdown"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Description">Project</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Design">Design</A><A href="https://2018.igem.org/Team:MIT/Results">Results</A><A href="https://2018.igem.org/Team:MIT/InterLab">InterLab</A><A href="https://2018.igem.org/Team:MIT/Notebook">Notebook</A><A href="https://2018.igem.org/Team:MIT/Experiments">Protocols</A><A href="https://2018.igem.org/Team:MIT/Attributions"> Attributions </A></DIV></DIV><DIV class="dropdown"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Parts">Parts</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Basic_Part">Basic Parts</A><A href="https://2018.igem.org/Team:MIT/Composite_Part">Composite Parts</A></DIV></DIV><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Safety">Safety</A></DIV><DIV class="dropdown"><DIV class="sub"><A href="https://2018.igem.org/Team:MIT/Human_Practices">Human Practices</A></DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:MIT/Human_Practices">Integrated Human Practices</A><A href="https://2018.igem.org/Team:MIT/Public_Engagement">Public Engagement</A></DIV></DIV><DIV class="sub"><A class="activemenu" href="https://2018.igem.org/Team:MIT/Model">Model</A></DIV></DIV><DIV class="h3">Modeling</DIV><P style="position: relative; top: 11vw; font-family: 'Raleway', sans-serif;; font-size: 1.3vw; color: white; margin-left: 2vw;
margin-right: 2vw; margin-bottom: 3vw;">
        We created an advanced multiscale model of the ComCDE quorum sensing system and biofilm formation via WIG synthesis in
        S. mutans. Employing the capabilities of the modelling environment Morpheus to simulate cell migration and adhesion as
        well as the the diffusion of extracellular small molecules and proteins, we modelled the logarithmic phase of bacterial
        growth in two dimensions. Our goal in modelling was to determine the most important factors in the bacterial biofilm
        initiation system in order to predict the effectiveness of various methods of inhibiting biofilm growth.
    </P><DIV style="position: absolute; top: 65vw"><BUTTON class="accordion1">1. List of Constants and Variables Used in the Model</BUTTON><DIV class="panel"><DIV class="tabular"><TABLE id="TBL-2" class="tabular" cellspacing="0" cellpadding="0"><TBODY><TR style="vertical-align:baseline;" id="TBL-2-1-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-1-1" class="td11">  Symbol   </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-1-2" class="td11">                     Meaning                                  </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-1-3" class="td11">  Value   </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-1-4" class="td11">                     Units                                  </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-2-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-2-1" class="td11"><SPAN class="cmmi-10">k</SPAN><SUB><SPAN class="cmmi-7">tx</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-2-2" class="td11">             Transcription Rate of a Gene                    </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-2-4" class="td11">           transcripts per gene per second                 </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-3-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-3-1" class="td11"><SPAN class="cmmi-10">k</SPAN><SUB><SPAN class="cmmi-7">tf</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-3-2" class="td11">  Transcription Factor Activity of Phosphorylated ComE   </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-3-3" class="td11">   0.15    </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-3-4" class="td11">     transcripts per ComEP per gene per second        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-4-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-4-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">mCSP</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-4-2" class="td11">              Decay Rate of CSP mRNA                      </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-4-4" class="td11">              transcripts per second                       </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-5-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-5-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">mComD</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-5-2" class="td11">             Decay Rate of ComD mRNA                    </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-5-4" class="td11">              transcripts per second                       </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-6-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-6-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">mComE</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-6-2" class="td11">             Decay Rate of ComE mRNA                    </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-6-4" class="td11">              transcripts per second                       </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-7-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-7-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">mGTFC</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-7-2" class="td11">            Decay Rate of mGTFC mRNA                   </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-7-4" class="td11">              transcripts per second                       </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-8-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-8-1" class="td11"><SPAN class="cmmi-10">pComC </SPAN></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-8-2" class="td11"> Number of ComC genes </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-8-3" class="td11"> 1 </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-8-4" class="td11"> genes</TD></TR><TR style="vertical-align:baseline;" id="TBL-2-9-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-9-1" class="td11"><SPAN class="cmmi-10">pComD  </SPAN></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-9-2" class="td11">               Number of ComD genes                        </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-9-3" class="td11">    1      </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-9-4" class="td11">                     genes                                  </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-10-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-10-1" class="td11"><SPAN class="cmmi-10">pComE   </SPAN></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-10-2" class="td11">               Number of ComE genes                        </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-10-3" class="td11">    1      </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-10-4" class="td11">                     genes                          </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-11-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-11-1" class="td11"><SPAN class="cmmi-10">pGTFC  </SPAN></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-11-2" class="td11">               Number of GTFC genes                        </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-11-3" class="td11">    1      </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-11-4" class="td11">                     genes                                  </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-12-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-12-1" class="td11"><SPAN class="cmmi-10">α</SPAN><SUB><SPAN class="cmmi-7">CSP</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-12-2" class="td11">               Translation Rate of CSP                       </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-12-4" class="td11">                proteins per second                        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-13-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-13-1" class="td11"><SPAN class="cmmi-10">α</SPAN><SUB><SPAN class="cmmi-7">ComD</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-13-2" class="td11">              Translation Rate of ComD                      </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-13-4" class="td11">                proteins per second                        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-14-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-14-1" class="td11"><SPAN class="cmmi-10">α</SPAN><SUB><SPAN class="cmmi-7">ComE</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-14-2" class="td11">              Translation Rate of ComE                      </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-14-4" class="td11">                proteins per second                        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-15-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-15-1" class="td11"><SPAN class="cmmi-10">α</SPAN><SUB><SPAN class="cmmi-7">GTFC</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-15-2" class="td11">              Translation Rate of GTFC                      </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-15-4" class="td11">                proteins per second                        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-16-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-16-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">CSP</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-16-2" class="td11">                 Decay Rate of CSP                           </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-16-3" class="td11"><SPAN class="cmsy-10">∣</SPAN><SPAN class="cmsy-10">∣</SPAN></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-16-4" class="td11">                proteins per second                        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-17-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-17-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">ComE</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-17-2" class="td11">                Decay Rate of ComE                          </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-17-3" class="td11"><SPAN class="cmsy-10">∣</SPAN><SPAN class="cmsy-10">∣</SPAN></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-17-4" class="td11">                proteins per second                        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-18-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-18-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">ComD</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-18-2" class="td11">                Decay Rate of ComD                         </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-18-3" class="td11"><SPAN class="cmr-10">1</SPAN><SPAN class="cmmi-10">.</SPAN><SPAN class="overline"><SPAN class="cmr-10">5</SPAN></SPAN><SPAN class="cmsy-10">×</SPAN><SPAN class="cmr-10">10</SPAN><SUP><SPAN class="cmsy-7">-</SPAN><SPAN class="cmr-7">5</SPAN></SUP></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-18-4" class="td11">                proteins per second                        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-19-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-19-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">GTFC</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-19-2" class="td11">                Decay Rate of GTFC                         </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-19-3" class="td11"><SPAN class="cmsy-10">∣</SPAN><SPAN class="cmsy-10">∣</SPAN></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-19-4" class="td11">                proteins per second                        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-20-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-20-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">CSPComDP</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-20-2" class="td11">              Decay Rate of CSPComDP                     </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-20-3" class="td11"><SPAN class="cmr-10">1</SPAN><SPAN class="cmmi-10">.</SPAN><SPAN class="overline"><SPAN class="cmr-10">5</SPAN></SPAN><SPAN class="cmsy-10">×</SPAN><SPAN class="cmr-10">10</SPAN><SUP><SPAN class="cmsy-7">-</SPAN><SPAN class="cmr-7">5</SPAN></SUP></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-20-4" class="td11">               complexes per second                       </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-21-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-21-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">ComEP</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-21-2" class="td11">         Decay Rate of Phosphorylated ComE               </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-21-4" class="td11">                proteins per second                        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-22-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-22-1" class="td11"><SPAN class="cmmi-10">k</SPAN><SUB><SPAN class="cmmi-7">b</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-22-2" class="td11">           Binding Activity of CSP to ComD                 </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-22-4" class="td11">      complexes per CSP per ComD per second          </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-23-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-23-1" class="td11"><SPAN class="cmmi-10">k</SPAN><SUB><SPAN class="cmmi-7">ub</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-23-2" class="td11">Unbinding Activity of CSP:Phosphorylated ComD Complex</TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-23-4" class="td11">      dissasociations per CSPComDP per second         </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-24-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-24-1" class="td11"><SPAN class="cmmi-10">k</SPAN><SUB><SPAN class="cmmi-7">k</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-24-2" class="td11">        Kinase Activity of Phosphorylated ComD            </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-24-4" class="td11">phosphorylations per CSPComDP per ComE per second</TD></TR><TR style="vertical-align:baseline;" id="TBL-2-25-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-25-1" class="td11"><SPAN class="cmmi-10">k</SPAN><SUB><SPAN class="cmmi-7">e</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-25-2" class="td11">              Enzyme Activity of GTFC                      </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-25-4" class="td11">    glucans formed per GTFC per Sugar per second      </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-26-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-26-1" class="td11"><SPAN class="cmmi-10">ϕ</SPAN><SUB><SPAN class="cmmi-7">CSP</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-26-2" class="td11">            Export Rate of CSP from a Cell                  </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-26-4" class="td11"><SUP class="nicefrac"><SPAN class="cmr-10">1</SPAN></SUP><SPAN class="cmmi-10">∕</SPAN><SUB class="nicefrac"><SPAN class="cmmi-10">CSP </SPAN><SPAN class="cmsy-10">×</SPAN><SPAN class="cmmi-10">second</SPAN></SUB></TD></TR><TR style="vertical-align:baseline;" id="TBL-2-27-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-27-1" class="td11"><SPAN class="cmmi-10">ϕ</SPAN><SUB><SPAN class="cmmi-7">Glucan</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-27-2" class="td11">          Export Rate of Glucans from a Cell                </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-27-4" class="td11"><SUP class="nicefrac"><SPAN class="cmr-10">1</SPAN></SUP><SPAN class="cmmi-10">∕</SPAN><SUB class="nicefrac"><SPAN class="cmmi-10">Glucan </SPAN><SPAN class="cmsy-10">×</SPAN><SPAN class="cmmi-10">second</SPAN></SUB></TD></TR><TR style="vertical-align:baseline;" id="TBL-2-28-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-28-1" class="td11"><SPAN class="cmmi-10">k</SPAN><SUB><SPAN class="cmmi-7">ab</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-28-2" class="td11">           Binding Activity of ScFv to GTFC                 </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-28-4" class="td11">      Complexes per ScFv per GTFC per second         </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-29-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-29-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">GTFCScFv</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-29-2" class="td11">          Decay Rate of GTFC:ScFv Complex                </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-29-3" class="td11"><SPAN class="cmsy-10">∣</SPAN><SPAN class="cmsy-10">∣</SPAN></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-29-4" class="td11">               complexes per second                       </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-30-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-30-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">ScFv</SPAN><SUB><SPAN class="cmmi-5">cell</SPAN></SUB></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-30-2" class="td11">                 Decay Rate of ScFv                           </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-30-3" class="td11"><SPAN class="cmsy-10">∣</SPAN><SPAN class="cmsy-10">∣</SPAN></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-30-4" class="td11">                proteins per second                        </TD></TR><TR style="vertical-align:baseline;" id="TBL-2-31-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-31-1" class="td11"><SPAN class="cmmi-10">k</SPAN><SUB><SPAN class="cmmi-7">kc</SPAN></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-31-2" class="td11">           Binding Activity of Kappa-Casein                 </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-31-3" class="td11">    1      </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-31-4" class="td11"><SUP class="nicefrac"><SPAN class="cmr-10">1</SPAN></SUP><SPAN class="cmmi-10">∕</SPAN><SUB class="nicefrac"><SPAN class="cmmi-10">KCasein </SPAN><SPAN class="cmsy-10">×</SPAN><SPAN class="cmmi-10">Glucan </SPAN><SPAN class="cmsy-10">×</SPAN><SPAN class="cmmi-10">second</SPAN></SUB></TD></TR><TR style="vertical-align:baseline;" id="TBL-2-32-"><TD style="white-space:nowrap; text-align:center;" id="TBL-2-32-1" class="td11"><SPAN class="cmmi-10">δ</SPAN><SUB><SPAN class="cmmi-7">KCasein</SPAN><SUB><SPAN class="cmmi-5">cell</SPAN></SUB></SUB></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-32-2" class="td11">             Decay Rate of Kappa-Casein                    </TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-32-3" class="td11"><SPAN class="cmsy-10">∣</SPAN><SPAN class="cmsy-10">∣</SPAN></TD><TD style="white-space:nowrap; text-align:center;" id="TBL-2-32-4" class="td11">                proteins per second                        </TD></TR></TBODY></TABLE></DIV></DIV><BUTTON class="accordion1">2. Morpheus and the Cellular Potts Model</BUTTON><DIV class="panel"><H4 class="subsectionHead"><SPAN class="titlemark">2.1   </SPAN>Cell Energy and Migration</H4><TABLE class="equation"><TBODY><TR><TD class="equation-label">(2)</TD></TR></TBODY></TABLE><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(3)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(4)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(5)</TD></TR></TBODY></TABLE></P>Morpheus is based on the Cellular Potts Model, which defines cells as spaces in a two
        or three dimensional lattice (our model used a hexagonal lattice in 2d) and
        determines changes to cell shape and size by using the Hamiltonian (Equations
        2-4). Equation 2 determines changes to the volume of a cell <SPAN class="grmn-1000">σ</SPAN>with current
        volume v<SPAN class="grmn-1000">σ</SPAN>in lattice sites and intended volume <SPAN class="cmmi-10">V</SPAN><SUB><SPAN class="cmmi-7">t</SPAN></SUB> where <SPAN class="cmmi-10">λ</SPAN><SUB><SPAN class="cmmi-7">V</SPAN></SUB> is a constant
        parameter of elasticity governing the extent to which the difference between
        the cell’s immediate and intended volume contributes to a rise in its free
        energy <SPAN class="ecti-1000">H</SPAN>. Equation 3 incorporates a similar system for cell perimeter into the
        equation.
         In addition to modelling adjustments to cells’ shape and size as they migrate, the
        Cellular Potts Model also uses the Hamiltonian to model the interactions between
        cells. Equation 4 determines the interaction energies between different cell types
        where <SPAN class="cmmi-10">τ</SPAN><SPAN class="cmr-10">(</SPAN><SPAN class="cmmi-10">σ</SPAN><SUB><SPAN class="cmmi-7">i</SPAN></SUB><SPAN class="cmmi-10">,σ</SPAN><SUB><SPAN class="cmmi-7">j</SPAN></SUB><SPAN class="cmr-10">) </SPAN>represents the cell types of two cells <SPAN class="cmmi-10">σ</SPAN><SUB><SPAN class="cmmi-7">i</SPAN></SUB> and <SPAN class="cmmi-10">σ</SPAN><SUB><SPAN class="cmmi-7">j</SPAN></SUB> and <SPAN class="ecti-1000">J </SPAN>specifies
        said energies in matrix form. In order to prevent cells from returning values
        for interactions with themselves, the term known as the Kronecker Delta
        defined in Equation 5 is included. Each update to the configuration of cells in
        the lattice as a result of equations 2-4 only occurs with a certain likelihood
        governed by the Boltzmann probability in Equation 1. This equation states
        that the cell’s chance of changing state is 100% if its change in energy <SPAN class="grmn-1000">Δ</SPAN><SPAN class="ecti-1000">H</SPAN>       added to its resistance to change <SPAN class="ecti-1000">Y </SPAN>is favorable (<SPAN class="grmn-1000">Δ</SPAN><SPAN class="ecti-1000">H</SPAN>+<SPAN class="ecti-1000">Y </SPAN>&lt; 0) and decreases
        exponentially with a rate of  where <SPAN class="ecti-1000">T </SPAN>represents the amount of unfavorable
        updates to the cell lattice, defined as modifications to the cell’s perimeter or
        volume.
        <H4 class="subsectionHead"><SPAN class="titlemark">2.2   </SPAN>Diffusion and the Cell Lattice Space</H4><TABLE class="equation"><TBODY><TR><TD class="equation-label">(6)</TD></TR></TBODY></TABLE><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(7)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(8)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(9)</TD></TR></TBODY></TABLE></P><P class="noindent"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(10)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(11)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(12)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(13)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(14)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(15)</TD></TR></TBODY></TABLE></P>   Another important component of our model was treating sugar, CSP, and our
        potential outputs not simply as cell-associated values but as scalar fields representing
        the concentration of said molecules. Fortunately Morpheus also incorporates the
        ability to overlay fields like these on simulated cell populations and allows the field to
        be updated locally based on cell activity at a specific lattice site (cells can also report
        on fields or other cells surrounding them, a function which is explained in the
        next section). Diffusion Fields were evaluated using the Central Difference
        Method to solve the 2-D Diffusion Equation (Equations 6-14). Equation 6 is a
        differential equation modelling diffusion in one dimension where the concentration
        c is a function of the X-coordinate and time T. D signifies the diffusion
        coefficient and L represents the length between nodes in (X, T) space where
        the domain of solutions is 0<SPAN class="cmsy-10">≤</SPAN>X<SPAN class="cmsy-10">≤</SPAN>L. By making the change of variables in
        Equation 7, the Diffusion Equation can be rewritten in the form of Equation
        8.



        Next, a set of approximations are made in order to solve for the first and second
        derivatives of concentration with respect to x (Equation 9). The central difference
        method is a version of the finite difference method of approximations to solve the
        Diffusion Equation, which assumes that there is a minimum distance in both
        dimensions, <SPAN class="cmmi-10">δ</SPAN>x and <SPAN class="cmmi-10">δ</SPAN>t, for which the concentration changes. This creates
        a grid in (x, t) space as shown in Figure . The central difference method
        approximates the x-derivative of concentration based on concentration values on
        either side of a point (i, j), one for each of the smallest change in x from
        the starting point: (i+1, j) and (i-1, j) (Equation 10). This is more precise
        than the forwards or backwards difference methods which only take into
        account one direction of incrementation in x. Next, the second derivative of
        concentration with respect to x is approximated using the values of the first

        derivative on a range half as large as in the previous approximation (Equation
        11).
        Using these results, an approximation of the concentration at a point 1 time step
        in the future can be obtained based on the concentrations at the surrounding points
        at the initial time (Equation 12), which can be appended with terms based on the
        approximations for the first and second derivatives in Equations 10 and 11 in order to
        obtain a more accurate approximation. Of course, in our model this was done in two
        dimensions, but the steps for the second dimension y are the same as those above and
        can be combined into Equations 13 and 14 in order to obtain values for a 2-D
        diffusion model.
          In order to sense nearby concentrations or cells in Morpheus, each cell can take
        advantage of the NeighborhoodReporter function, which maps values within a
        specified node length from a cell to an average, variance, or sum specific to that cell.
        In our model, cells employed the sum function of the NeighborhoodReporter to
        interact with the various diffusion fields. Equation 15 is the equation for a lattice sum
        in a 2-dimensional hexagonal lattice, which Morpheus approximates to write field
        values to individual cells.
        </DIV><BUTTON class="accordion1">3. Model Specifics</BUTTON><DIV class="panel">The first objective of accurately modelling a bacterial population was to model
            population growth over time. We chose to define our time step in the model as one
            second, and used the canonical doubling time of around 3800 seconds to
            determine a probability of cell division at each time step. Next, in order to reflect
            the constantly shifting environment of saliva on teeth, we programmed the
            cells with random motion defined within a realistic range for cell velocity in
            the salivary microbiome. This ensured that the cells would interact so that
            they would be affected by the modified adhesion associated with biofilm
            initiation.
            <H4 class="subsectionHead"><SPAN class="titlemark">3.1   </SPAN>Transcription and Translation of the Relevant Genes</H4><TABLE class="equation"><TBODY><TR><TD class="equation-label">(16)</TD></TR></TBODY></TABLE><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(17)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(18)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(19)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(20)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(21)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(22)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(23)</TD></TR></TBODY></TABLE></P>The biggest hurdle in creating an accurate model of biofilm formation in S. mutans
        was simulating the two-component signaling system known as ComCDE. We began
        by creating differential equations for the transcription and translation of the ComC,
        ComD, and ComE genes based on transcription rates, translation rates and decay
        rates for mRNAs and proteins from the literature. We also researched the genome
        size and operon structure of the bacteria in order to better reflect its production of
        the relevant proteins in our model. Equations 16-18 represent the transcription of
        mRNAs for the three genes we chose to study, and the first four terms in each of
        Equations 20-22 represent the translation of those mRNAs. All terms and
        constants are defined in the table at the beginning of this section. Now that we
        had the cells generating values for these variables at each time step, we
        could add terms to model the kinetics of the two-component sensing system
        itself.
        <H4 class="subsectionHead"><SPAN class="titlemark">3.2   </SPAN>Two-Component Sensing System: Ligand Binding, Response Regulator
            Phosphorylation and DNA Binding</H4><TABLE class="equation"><TBODY><TR><TD class="equation-label">(24)</TD></TR></TBODY></TABLE><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(25)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(26)</TD></TR></TBODY></TABLE></P>We used the Law of Mass Action to create differential equations reflecting the
        kinetics of ComD binding its ligand CSP, the autophosphorylation of ComD, the
        phosphorylation of ComE by ComD, and finally the DNA binding and transcription
        factor activity of phosphorylated ComE. For a full explanation of the way the
        two-component signalling system works in live bacteria, please refer to our project
        page, as this section assumes knowledge of the relevant components and their
        functions. The first step in the signalling system is the binding of CSP to
        ComD. In order to initiate this activity, we used the NeighborhoodReporter
        function to update the variable CSPout at each time step. ComD binds
        CSP at the rate kb, at which point the two are considered one complex,
        CSPComDP for autophosphorylated ComD:CSP, and the amount of CSP and
        ComD decrease accordingly (Equations 21 and 24). CSPComDP has an
        unbinding rate kub which decreases the amount of ComD:CSP complexes while
        increasing both CSPout and ComD by the same amount. In order to simplify
        the system of equations, the assumption was made that once CSPComDP
        phosphorylates its response regulator ComE, the complex dissociates and
        returns to separate ComD and CSPout. kk represents the rate of the complex
        phosphorylating ComE (Equation 25). Once ComE is phosphorylated, it is considered
        a different variable, ComEP for phosphorylated ComE. Phosphorylated ComE
        upregulates the ComC and Glycosyltransferase genes with the rate ktf in order
        to create a positive feedback loop within the population characteristic of
        quorum-sensing systems and produce the necessary enzymes for biofilm formation.
        Once again, the assumption was made that after ComEP binds DNA it is
        dephosphorylated by available phosphatases and returns to its initial state,
        hence the amount of ComEP decreases with the rate ktf for each gene it
        upregulates and ComE increases by the same amount (Equations 22 and
        26).
        <H4 class="subsectionHead"><SPAN class="titlemark">3.3   </SPAN>Water-Insoluble Glucan Synthesis via Glucosyltransferase Activity</H4><TABLE class="equation"><TBODY><TR><TD class="equation-label">(27)</TD></TR></TBODY></TABLE>Biofilm formation in the model depends on two main factors: Sugar available to the
        cell and the amount of Glycosyltransferase enzymes a cell possesses (GTFC). mRNAs
        for GTFC (Equation 19) are only transcribed when ComEP binds the gene to
        upregulate it, and GTFC proteins are translated according to Equation 23.
        Sugar concentration (<SPAN class="cmmi-10">Sugar</SPAN><SUB><SPAN class="cmmi-7">field</SPAN></SUB>) is given an initial value as a diffusion field
        in the model, which was defined as a homogeneous diffusion field due to
        sugar’s high solubility in saliva. <SPAN class="cmmi-10">Sugar</SPAN><SUB><SPAN class="cmmi-7">cell</SPAN></SUB> is defined at each time step via the
        NeighborhoodReporter function based on the current concentration of the
        <SPAN class="cmmi-10">Sugar</SPAN><SUB><SPAN class="cmmi-7">cell</SPAN></SUB>.
         The final differential equation based on the Law of Mass Action in the most basic
        form of the model, Equation 27, governs the production of water-insoluble
        glucans from available sugar. The enzymatic activity of GTFC is denoted
        ke, and the glucans become the basis of the extracellular biofilm by being
        incorporated into the extracellular field of the same name at the rate <SPAN class="cmmi-10">ϕ</SPAN><SUB><SPAN class="cmmi-7">Glucan</SPAN></SUB>. In
        order to simulate the transition of S. mutans from free-floating planktonic
        cells to biofilm-bound immobilized cells which grow in localized colonies,
        each cell was programmed to also use the NeighborhoodReporter to return
        a constantly updated value for how concentrated biofilm-forming glucans
        are around that cell. Once the local biofilm reaches a threshold value of
        5 arbitrary units, cells are considered to be adhered and their movement
        ceases.
        <H4 class="subsectionHead"><SPAN class="titlemark">3.4   </SPAN>Extracellular Diffusion Fields</H4><TABLE class="equation"><TBODY><TR><TD class="equation-label">(28)</TD></TR></TBODY></TABLE><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(29)</TD></TR></TBODY></TABLE></P><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(30)</TD></TR></TBODY></TABLE></P>Equations 28-30 are evaluated at each time step at each point in the lattice in order
        to affect the changes individual cells make to field values. Both the biofilm and CSP
        field equations increase at an export rate times the number of proteins a cell at a
        given point has, and the Sugar decreases as GTFC transforms it into glucans. Sugar
        undergoes homogeneous or well-mixed diffusion, CSP was given a diffusion
        constant of <SPAN class="ecti-1000">D </SPAN>= 1, and the Biofilm was given the diffusion constant <SPAN class="ecti-1000">D </SPAN>=
        0.001, both in as employed in Equation 6 and its subsequent approximate
        solutions.
        </DIV><BUTTON class="accordion1">4. Results</BUTTON><DIV class="panel">Our model proved quite successful in qualitatively reproducing multiple aspects of
        the formation of live <SPAN class="ecti-1000">S. mutans </SPAN>biofilms. The simulated cells form clear microcolonies
        in the locations where glucan concentration is the highest which expand
        from there. CSP increases everywhere throughout the simulation as the
        time steps modelled only encompass the phase of growth when that is the
        case.

        <H4 class="subsectionHead"><SPAN class="titlemark">4.1   </SPAN>Effect of Sugar Concentration on Biofilm Formation</H4>The first parameter we varied was sugar by modifying the initial value of the
        diffusion field. After fifteen thousand time steps, there is a clear difference in biofilm
        density in plots of the cells themselves. Graphs of data collected from a
        representative cell in each condition show a clear distinction in glucan production
        between sugar concentrations. For the time period simulated, limiting sugar
        concentration was between 10 and 25 arbitrary units, at which point the cells were
        unable to generate enough glucans to adhere and create microcolonies in a
        biofilm. These results reflect our experiments with live <SPAN class="ecti-1000">S. mutans </SPAN>in which
        we varied sucrose concentration and measured its effect on biofilm growth
        through image analysis of of colony-forming units as well as crystal violet
        staining. As expected based on our differential equations, the concentration
        of sugar decreases exponentially over time, whereas the total amount of
        glucans produced increases exponentially over time. The rates of exponential
        growth and decay also depend on the sugar available, with the difference
        in glucans produced by the end of the simulation being around one order
        of magnitude less than the difference in sugar available in all cases. This
        data was useful in helping us match the arbitrary units in the model to real
        experiments.
        <H4 class="subsectionHead"><SPAN class="titlemark">4.2   </SPAN>Inhibition of GTFC Activity by Single-Chain Variable Fragments</H4><H5 class="subsubsectionHead"><SPAN class="titlemark">4.2.1   </SPAN>Additional Cellular Equations Governing ScFv Activity</H5><TABLE class="equation"><TBODY><TR><TD class="equation-label">(31)</TD></TR></TBODY></TABLE><P class="nopar"><TABLE class="equation"><TBODY><TR><TD class="equation-label">(32)</TD></TR></TBODY></TABLE></P><H5 class="subsubsectionHead"><SPAN class="titlemark">4.2.2   </SPAN>Additional Field Equations Governing ScFv Activity</H5><TABLE class="equation"><TBODY><TR><TD class="equation-label">(33)</TD></TR></TBODY></TABLE><H5 class="subsubsectionHead"><SPAN class="titlemark">4.2.3   </SPAN>Modified Equations Integating ScFv Activity</H5><TABLE class="equation"><TBODY><TR><TD class="equation-label">(34)</TD></TR></TBODY></TABLE>The primary purpose of our model was to preemptively compare the effectiveness of
        our potential outputs by modelling their inhibition of different parts of the pathway
        and its effects on overall biofilm growth. We began by implementing the above
        differential equations (Equations 31-33), and modifying Equation 23 into equation 34.
        The amount of ScFvs affecting the cell is determined via the NeighborhoodReporter
        function, and the ScFvs bind GTFC at the rate kab. Once the ScFv binds a GTFC, it
        forms a GTFC:ScFv complex represented in Equation 31, decreasing both
        the cell’s GTFC and ScFv count accordingly (Equations 32 and 34). The
        complex degrades quickly based on a rate from the literature, and once it has
        degraded both the ScFv and GTFC bound have been eliminated from the
        simulation.
         The effects of the ScFv on biofilm formation are immediately obvious from plots
        of the cells at the end of the simulation, with only 0.075 arbitrary units of
        concentration to start with being enough to prevent the bacteria from forming
        adherent microcolonies throughout the time modelled. The critical point
        at which the amount of GTFC: ScFv complexes goes from increasing to
        decreasing determines how soon the cells are able to begin initiating biofilm
        formation, assuming they are still in the logarithmic growth phase where CSP
        production is increasing. Since GTFC production is decreased, everything
        downstream of it is also affected: cells consume less sugar, produce fewer
        glucans, and create less of a biofilm field in the presence of ScFvs, as shown
        in the following figures. In equation 33, the decrease of the ScFv field is
        divided by 100 due to the membrane resolution being set to 100 for the
        simulation.
        <H4 class="subsectionHead"><SPAN class="titlemark">4.3 </SPAN>Inhibition of Glucan Activity by Kappa-Casein</H4><H5 class="subsubsectionHead"><SPAN class="titlemark">4.3.1   </SPAN>Additional Cellular Equations Governing K-Casein Activity</H5><TABLE class="equation"><TBODY><TR><TD class="equation-label">(35)</TD></TR></TBODY></TABLE><H5 class="subsubsectionHead"><SPAN class="titlemark">4.3.2   </SPAN>Additional Fiels Equations Governing K-Casein Activity</H5><TABLE class="equation"><TBODY><TR><TD class="equation-label">(36)</TD></TR></TBODY></TABLE><H5 class="subsubsectionHead"><SPAN class="titlemark">4.3.3   </SPAN>Modified Equations Integating K-Casein Activity</H5><TABLE class="equation"><TBODY><TR><TD class="equation-label">(37)</TD></TR></TBODY></TABLE>Next, we sought to create a similar set of equations to model the effect of
        Kappa-Casein on biofilm formation. Equation 35 governs the effect of K-Casein on
        an individual cell. Cells use the NeighborhoodReporter to determine the
        amount of K-Casein molecules affecting it, and those molecules bind glucans
        at a rate of kkc. Rather than creating a new variable for the complex of
        K-Casein and a glucan, both are removed from the simulation immediately.
        Due to being outside the cell, the binding rate of K-Casein was set to 1.
        Equation 37 is a modification of equation 27 integrating the decrease in
        glucans.
         It became clear from varying the initial K-Casein concentration that the
        equations were successful in modelling the decrease in biofilm formation due to a
        decrease in effective adherent glucans surrounding the cells. However, despite the
        K-Casein being given a dramatically higher binding rate than the ScFvs, a much
        higher concentration of K-Casein was required to limit the formation of a biofilm
        during the time simulated. In fact, ten times as high a concentration of
        K-Casein was required to achieve the same effects as ScFvs. The K-Casein
        also affected fewer aspects of the overall system of equations because its
        influence was only effective on one of the most downstream components of the
        simulation.
        </DIV><BUTTON class="accordion1">5. Conclusions</BUTTON><DIV class="panel">The most obvious takeaway from the results of our computational model was that
            ScFvs were more effective than K-Casein at inhibiting biofilm formation, or, more
            broadly, inhibiting GTFC is a better method for limiting the virulence of S. mutans
            than reducing the glucans themselves. This result had a profound effect on our
            experimental design and our project as a whole. While Kappa-Casein can be readily
            purchased and implemented in live experiments, the ScFvs we planned to
            express and implement were not available for purchase anywhere, would be
            very expensive to get synthesized, and would require additional training
            and lab techniques to isolate from mammalian cells expressing the protein
            itself. However, thanks to data from the model, we were able to confirm the
            superiority of the ScFvs for our purposes. Based on these results, we decided to
            move further ahead with characterization experiments for GTFC-inhibiting
            ScFvs.

            </DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>