Team:MIT/Human Practices
Home
Team
Team Members Collaborations
Project
Design Results InterLab Notebook Protocols Attributions
Parts
Basic Parts Composite Parts
Safety
Human Practices
Integrated Human Practices Public Engagement
Model
Human Practices
This year Team MIT decided to follow the AREA guidelines to make sure that the project idea was not only scientifically     possible but also socially, economically, politically, and commercially viable. The AREA guidelines walk a research team     through the four basic steps of responsive science:
1) Anticipate
2) Reflect
3) Engage
4) Act
In order to Anticipate and Reflect, Team MIT carried out several social analyses (i.e. Stakeholder/Value Matrix & a SWOT analysis)     during the research design phase. This helped the team assess how we should channel our effort, time, and resources towards something     that would be impactful, not just in our lab, but also beyond it. We then spent a significant amount of time mitigating weak points the     SWOT test pinpointed as well as brainstorming approaches to external threats the project may encounter.
Additionally, the Stakeholder/Value Matrix provided insight on who will be affected by the project. By organizing meetings with the     identified stakeholders to hear concerns about the technology (specifically related to the the practicality of our project), the team     was able to modify these values based on real input. The final version of the matrix represents the project's positive and negative     impacts on various communities.
Our next step was to Engage with these identified stakeholders and Act on the feedback provided to us on our project. Team MIT had some interesting dialogues that helped to reflect on and shape the experimental design in order to be more responsible as well as practical. Read more about these conversations below!
Conversations with the Synthetic Biology Community
Ross Jones, Graduate Student, MIT Synthetic Biology Center
Early June the team met with him to ask about preliminary DNA design for comE, comD, and comE driving promoters.             Ross described a promoter he had engineered for the Envz/OpR system at the first meeting and explained in detail             how each component works together. This helped Team MIT better understand how to engineer a new comE inducible             promoter with comE binding sites and a cytomegalovirus (CMV) minimal promoter.
Ross also explained ways to promote activation upon ComE binding, how to create a constitutively active ComE,             the structural differences between activation domains VP16 and VP64, and showed us an example design for linking             a protein with an activating domain. In order to create constitutively active ComE, Team MIT had planned to change             the aspartic acid at the phosphorylation site to a glutamic acid. Another way to ensure comE binding is by truncating             only the effector region of the response regulator. The team decided to also incorporate the effector truncation of             comE into the experiments. This would hopefully provide another positive control for comE binding to the engineered promoter.             At the end of this meeting Ross made another note about the promoter driving the histidine kinases.             In his previous experiments, it was found that one needs very little histidine kinase to have an effect on the response             regulator. He gave us the name of a constitutive promoter with lower expression than hef1a and the team decided to move             forward with Ross's promoter to drive comD expression in our experiments.
The second meeting in late July was more specifically about protein localization. For the histidine kinase             receptor to sense the Competence-Stimulating Peptide (CSP), it needed to be on the cell membrane. The team hoped             to direct the receptor to the membrane by fusing the CD4 membrane signalling peptide to ComD, but were confused             because past research was able to localize the receptor correctly without fusing any signalling peptide to histidine             kinases. Based on his past experience with membrane localization, Ross went through some standard membrane signalling             peptides and talked about different ways to check co-localization as well as a microscope recommendation. The team             decided to first attempt localization of comD using CD4.
The other main question was about the response regulator nuclear localization and export. ComE fused with VP64 is             still small enough to fit through the nuclear pores, but there are concerns about its speed of diffusion (since in             the first meeting with Ross he had mentioned that many response regulators have a tendency to spontaneously             dephosphorylate after activation). It wasnâ€™t clear whether to include both nuclear localization and nuclear export             sequences (NLS and NES) to promote shuttling of comE between the nucleus and comD. Ross agreed that NLS and NES sequences             might help counter spontaneous spontaneous dephosphorylation, so the team decided to include the signalling sequences in             the first iteration of our DNA design.
Ron Weiss, Principal Investigator, MIT Synthetic Biology Center
After the initial brainstorming and design phase, team MIT presented our project to Dr. Ron Weiss and the SBC, in             order to gain feedback about the scientific validity of our work so far. There were some interesting questions raised             in these sessions, many of which concerned our mode of delivery, which was quite "hand-wavy" and seemed impractical.             Our initial mode of delivery was stem cell therapy and we, as a team had decided to go for this "envisioned" output method             without giving it much thought. Our conversation with synthetic biologists in the SBC made us realize that we needed to             have a concrete mode of delivery, that had previously worked in similar syn-bio applications. At this point, the idea of             cell encapsulation as our mode of delivery was born. One of the members of the SBC explained to us this particular technology             and showed us successful implemented models such as Viacyte. With this feedback in mind, we dedicated a new sub-team just to             research on this more practical mode of delivery, which was not the focus of our research but was important to be included             in our presentation as a viable future prospect.
Kobi Benenson, ETH Zurich
Our project idea was based on a research paper by Kobi Benenson, who had successfully transplanted a bacterial TCS into a mammalian         cell. We wanted to replicate his transplantation but essentially, with a different TCS system and a different application in mind.         We reached out to him in order to gain some expert knowledge on the experimentation that we would need to carry out but unfortunately,         we were informed that because his research work was in the patenting process he could not share a lot of information. In order to         better understand this predicament and the IP process, we decided to talk to an expert.
Conversations with the Law and Policy Community
Nevin Summers, Executive Director, MIT Synthetic Biology Center
Nevin is the Executive Director of the Synthetic Biology center at MIT and does a lot of work for the group             in relation to patents and intellectual property. Even though, we knew about Mr. Benenson's patent in regards to his             research, we felt that the specific transplantation of the ComDE system in S.mutans, into mammalian cells was something             that had never been done before and the team wondered if this technique was patentable. Nevin explained how patents were             usually framed using over-arching claims to include a lot of different things and not specific systems. With his help the             team learned that the lab whose research was so foundational to our project, ETH Zurich, had already patented the transplantation             of a two-component system from bacterial cells to mammalian cells. Furthermore, to be able to use our devised system in a             commercial product, Team MIT would need permission from ETH Zurich to make an impact beyond the bench.
Besides interesting dialogue about the scientific patenting process, Nevin also spoke about open-access research in MIT's             Synthetic Biology Center. With Nevin's help, the team was also able to gauge the transparency of our research and what areas             could be improved by filling out the Centre for Open Science checklist. The ultimate goal is to present clean, open-access, and             reproducible data that may contribute to dental as well as biosynthetic research.
Conversations with Oral Cavity Experts
Dr. Kimberly Weiss, DDS, Pediatric Dental Associates
Kimberly Weiss, DDS is a dentist at Pediatric Dental Associates (PDA) in Reading, Massachusetts. Besides over eighteen             years of experience in general dentistry, Dr. Weiss is an instructor and advisor to dental students at the Harvard School             of Dental Medicine. Team MIT visited her at the clinic to better understand the dental community's perspective on the project.
One of the major takeaways was re-evaluating how dentists would feel about a technology that could potentially drastically             reduce their customer base. The team was very surprised when she replied, "Would an oncologist be upset if you cured cancer?"             To her, any dentist or a doctor who focused more on the number of patients they got and less on the community's health was in             the wrong field. She explained to us that dentistry is a vast and varied medical discipline, and even with less patients having             caries there will always be a need for her profession.
Dr. Weiss also provided interesting insight into the patient's mindset. Based on her feedback, the team decided to do a cost             breakdown to determine whether the technology was more economically feasible than traditional methods of preventing dental             caries (i.e. brushing one's teeth). In terms of accessibility, she highlighted how this would be potentially more useful for             high cavity risk groups as well as people in nursing homes who suffer dental caries and tooth loss despite traditional dental             care.
Dr. Bruce Paster, Forsyth Institute (and selfie expert!)
Bruce Paster, a Senior Member of the Staff at the Forsyth Institute, is a local professor of oral medicine, infection, and             immunity. He specializes in microarray technology at the Harvard School of Dental Medicine.The team met with Bruce to learn             more about the oral microbiome and whether the proposed outputs and delivery methods would be effective. Specifically, Team             MIT wanted to better understand the dynamics between different bacterial species in the microbiome and the vitality of the             role S. mutans plays in cariogenicity.
Dr. Paster explained how cariogenic bacteria use a variety of pathways to adhere to tooth enamel. S. mutans rely on             glucan-mediated adhesion so in theory the proposed output would only be effective in preventing S. mutans-induced caries.             While they are the primary contributor to dental caries, S. mutans are not the only cariogenic bacteria! In fact, 15-20% of             caries cases do not involve S. mutans!
The three main classes of caries in terms of pathogenic composition are S. mutans, a mixture of S. mutans with other             Streptococci, Veillonella, plus Bifidobacterium, and Lactobacillus. Therefore eliminating S. mutans would very likely             just result in another species taking its place. He told us about the glucosyltransferase vaccine team received the same             criticism.
An important thing for us to consider in terms of mode of delivery is proximity to the site of action. Mr. Paster explained             that while the proposed target area (gingiva) could be convenient due to the mechanics of saliva/crevicular fluid, it might             not be as effective as a varnish on the tooth enamel. Any experiments the team conducted would be proof of principle/circuit.             However, to claim that our system could prevent caries, clinical trials are necessary. Clinical trials are the only way to             determine the threshold of reduction (i.e. is reducing S. mutans by 50% going to be enough?) required to inhibit caries.
Another key obstacle that Mr. Paster highlighted was that of Saliva. Saliva contains a variety of molecules such as             proline-rich proteins and statherins that can block both the attachment of planktonic cells as well as receptors that promote             binding. We realized that whatever material used to simulate the tooth enamel will have to be incubated in saliva in order for             the experiments to be realistic/accurate.
In terms of quantification methods, Dr. Paster recommended using crystal violet staining methods over CFU. Staining would be                 more accurate and overall quicker. To study the effectiveness/accuracy of the current CFU-based method, he suggested doing a             microscopic analysis to see if chain formation is enhanced. Some potential mechanical models include microfluidic devices and             chemostats. The team should also look into incubating S. mutans with output and hydroxyapatite. Wash beads, quantify bacteria             that became dislodged from the beads (in the supernatant), and use subtraction to calculate the number of bacteria that remained             bound to the beads. He also recommended to take dilutions of S. mutans and plot a standard curve for assay reference. Also,             because S. mutans is not the only cariogenic bacteria, the team should also look into experiments involving other bacterial                 species that may take the place of S. mutans after its elimination.
Repeating the same characterization assays for other species beside S. mutans can help the team understand the outputs' effects             on the oral microbiome. Because of Dr. Paster, the team will be incorporating saliva into the experiments and doing more             literature research into other cariogenic bacteria. He recommended looking into Jeff Hillman's work on S. mutans and encouraging             attachment of beneficial bacterial species.
Dr. Xuesong He, Forsyth Institute, PhD and DDS
Dr Xuesong is an expert in the oral microbiome and the role that S. mutans plays in it. As someone who has studied the oral             microbiome extensively, including the contribution of different species to cariogenesis, we felt that Dr. He would be able to             give valuable insights into whether or not our project would be successful. Our main goal was to get his take on the concerns             voiced by Dr. Paster (regarding how eliminating S. mutans would just lead to something else taking its place).
Dr. He was overall very supportive of our project. He said that because our project is a proof of concept of a unique host-based                 caries prevention system, we do not need to have our solution be modular towards targeting other cariogenic bacteria as well. Other justifications he gave for our project were that: 1. Caries treatments focused on the host rather than directly targeting             the bacteria themselves is very new and complicated, and delving into this field requires us to start on a smaller scale. 2.             S. mutans is a good organism to study: it's the top contributor to caries, the top acid-producing bacteria, and it's the most             funded cariogenic pathogen.  According to Dr. He, "If you can solve S. mutans, you will have solved 90% of the caries problem!"
He informed us of various treatment methods to help us think of even more viable future prospects, such as bacteria-focused caries             treatment methods (antimicrobial peptide that only targets S. mutans,  or an acid-activated antimicrobial peptide: only activated in             low pH, allows for targeting of all acid-producing bacteria), and host-focused caries treatment ideas (much more rare!) such as stopping             acid production of bacteria: biofilm formation is less of an issue than acid production-bacteria will be in the mouth no matter what.             Colonization requires a lot of gene function from S. mutans (i.e. GTFs). (Acid production is on a case-by-case basis, but S. mutans             is always number one). He also told us that Much work has already been done in terms of interfering with CSP signaling             (i.e. competitive inhibition of CSP receptor)
Since we met Dr. He in mid-August, we realized that it was too late to make any drastic changes in our experimental design.             However, Dr. He's insights will be extremely helpful to us in explaining and justifying our project and the envisioned mode             of delivery.. He was able to address some of the most prevalent concerns voiced by people we've presented to. He also offered             to put us in contact with some hydrogel experts he knew so we could devise a more concrete plan for our mode of delivery.
Conversations with Cell Encapsulation Experts
After some initial research into cell encapsulation, we decided to meet with some experts about the cell encapsulation             technique and how compatible this mode of delivery would be to our envisioned technology. We reached out to the Langer Lab,             which is currently carrying out cutting-edge research in hydrogels and cell encapsulation techniques. We got into contact             with two researchers at Langer Lab, namely Suman Bose and Derfogail Delcassian.
Suman Bose, PostDoc, Langer Lab
Our conversation with Suman lead to us to discover the technique of microencapsulation and how we could potentially devise             injectable capsules with sizes as small as 300 microns. The validation and extra literature provided to us by Suman really             helped us along to start designing potential experiments that could illustrate how we could encapsulate and inject our             engineered cells into the gingiva.
Derfogail Delcassian, PostDoc, Langer Lab
Our conversation with Derfogail allowed us to become aware of various hydrogels that are biocompatible with the oral cavity,             as well as start thinking of a semi-permeable membrane that would allow CSP(~18 amino acids), our sensing molecule, to enter             and our output protein (~190 amino acids) to exit the capsule. In the future, we plan to do a simple encapsulation experiment             under Derfogail's supervision to verify if the sensing and output efficiency is the same as before (i.e. the encapsulated system             reduces biofilm concentration the same as the non-encapsulated system).
Conversations with iGEM Experts
Dr. Traci Haddock-Angelli
The team met with Traci Haddock-Angelli, director of the 2018 iGEM competition, to clarify the necessary requirements             for part acceptance by iGEM headquarters.  During the conversation, Dr. Angelli showed us past work and clarified exactly             which cloning methods are necessary to prepare parts for registry submission. The team has been investigating the Loop and             Phytobricks assembly method to better craft a system suited for both the team and iGEM's needs.  Creating a library of parts             that would allow for further integration of the Golden Gate system into the iGEM registry, while still maintaining traditional             iGEM requirements (such as BioBrick assembly) is one of the team's goals. Team MIT will continue to investigate these systems             while following iGEM's standards.
Hassnain Qasim Bokhari
Team MIT wanted to reach out to the iGEM Human Practices (HP) committee and get some feedback on our initiatives so far.             The team contacted one of the members of the HP committee and an iGEM Ambassador, Hassnain. In the meeting he put emphasis             on having a vision for all the initiatives and not doing things just because other people had done it. Continuing past efforts,             especially those that have inspired us, is also a critical part of a great human practice initiative. Based on this feedback the             team decided to make a formal responsible science curriculum that could be used for future iGEM teams as well as other             institutions such as LabCentral and pgED, both of whom had conversations with us and gave us some feedback on the devised             curriculum. We also planned to make a HP How-To and build on the efforts of the Williams and Mary database of iGEM HP             initiatives so far. But we have thus far not finished that project.
