"
Page
Discussion
View source
History
teams
Log in
 
Team:Tokyo-NoKoGen/oscillator
From 2013.igem.org
Team:Tokyo-NoKoGen - 2013.igem.org
Introduction
Objective
Method
Design
Parts construction
Evaluation
TR-HHR
TR-HHR-taRNA
Future work
RNA oscillator
RNAScaffold
Light sensor
Modeling
Improving a BioBrick part - Rhodopsin
RNA oscillator
Introduction
Protein oscillator
The oscillating network was built by using three transcriptional repressor systems in which the first repressor protein suppresses the expression of second repressor gene, whose protein suppresses the transcription of the third gene, and the protein the third gene code suppresses the first gene expression.
Hummer headed ribozyme
Hummerheaded ribozymes (HHRs) are found in many species and known to have a capability to self-cleave. Recently, engineered HHRs responding to trans-acting RNAs (taRNAs) in E.coli were reported. In absence of taRNA, the HHRs can self-cleave, but in presence of taRNA, their capability of self-cleavage is inhibited due to hybridize with taRNA and change their structures.
We developed the new oscillator system that is composed only RNAs not using repressor gene and protein. We created three taRNA connected hummerheaded ribozymes (HHR-taRNA) that sense to another taRNA. The same way as the protein oscillator, three HHR-taRNAs inhibits other one’s self-cleavage such as three-way standoff. When the first HHR-taRNA is self-cleaved, connected the first taRNA that binds to the second HHR-taRNA is free and suppresses the second HHR-taRNA self-cleavage. When second HHR-taRNA is self-cleaved, the second taRNA that binds to the third HHR-taRNA is free and suppresses the third HHR-taRNA. Finally, when third HHR-taRNA is self-cleaved, the third taRNA that binds to the first HHR-taRNA is free and suppresses the first HHR-taRNA, completing cycle.
Objective
Construction of Three of orthogonal taRNA generator which repressed by another taRNA.
taRNA responsive HHR (TR-HHR) was fused with taRNA. By using orthogonal riboregulator sequences, we designed three of taRNA generators.
Method
-Design
TR-HHR was designed by Benedikt Klauser and Jo¨ rg S. Hartig. Their TR-HHR were inspired by the riboregulatory system previously reported by Collins and coworkers. They made use of sequence domains of the riboregulator RR12. And, orthogonal variants of riboregulator RR12 (RR12y, RR42) were reported by Collins and coworkers. So, by using the sequences of these orthogonal riboregulators, we designed three orthogonal TR-HHRs.
-Parts construction
1. Construction of TR-HHR
2. Construction of TR-HHR variants
3. Construction of taRNA variants
4. Construction of TR-HHR-taRNA
1. Construction of TR-HHR [BBa_K1053000]
We used the TR-HHR developed by Klauser et al.[1] Two overlap primers designed, and PCR was performed to connect them. The PCR product was ligated into pSB1C3 vector.
2．Construction of TR-HHR variants
By inverse PCR using pSB1C3-TR-HHR as a template, we could obtain three TR-HHRs, TR(12)-HHR, TR(12y)-HHR, and TR(42)-HHR.
3．Construction of taRNA variants
By inverse PCR using pSB1A3-taR12-DT as a template, we could obtain three taRNAs, taR12, taR12y, taR42.
4．Construction of TR-HHR-taRNA
[BBa_K1053006][BBa_K1053015][BBa_K1053016]
TR-HHR-taRNAs were constructed by overlap PCR.
Evaluation
I. Evaluation of TR-HHR
II. Evaluation of TR-HHR-taRNA
I. Evaluation of TR-HHR
In order to evaluate the regulation of HHR’s self-cleavage activity by taRNA, GFP gene was connected with the HHR or inactive HHR (*HHR), which has a mutation lose the activity.[BBa_K1053004][BBa_K1053005]
These fragments were subcloned into conventional biobrick standard vector pSB3C5.
1) E. coli TOP10 strain was co-transformed by two plasmids listed bellow,
A)
pSB1A3-taR12-DT
pSB3C5-TR(12)-HHR -GFP-DT
B)
pSB1A3-empty
pSB3C5- TR(12)-HHR -GFP-DT
C)
pSB1A3-taR12-DT
pSB3C5-TR(12)- * HHR-GFP-DT
D)
pSB1A3 empty
pSB3C5- TR(12)- * HHR -GFP-DT
E)
pSB1A3-empty
pSB3C5-empty
2) Bacterial cultures were incubated in LB medium at 140 rpm and 37℃ for 12 hour.
3) Then, cultures was diluted to an OD595 of 0.0125 and induced with or without 0.1% arabinose.
4) Cultures were incubated with shaking at 140 rpm and 37℃ for 8 hours in 1 mL of LB medium.
5) OD595 and GFP fluorescence intensity were measured.
All experiments were performed using three cultures per each sample.
Result and Discussion
In the presence of taR12, GFP fluorescence value divided by OD value is higher than one in absence of taR12. This result was not expected. HHR has the ability to self-cleave, but its ability is suppressed when taRNA hybridizes to HHR. Therefore, in the presence of taRNA, self-cleavage of HHR should not happen. So then, why we get such result? One factor can be due to the exposure of RBS sequence caused by the structural change of HHR when it hybridizes to taRNA
II.Evaluation of TR-HHR-taRNA
We examined whether TR-HHR-taRNA’s output can be repressed by corresponding taRNA.
1) The plasmids as shown below was used to transform E. coli Top10.
A)
pSB1A3-Pbad -TR(42)HHR-taR12-DT
pSB3C5-crR12-GFP
B)
pSB1A3-Pconst(H)-taR42-Pbad-TR(42)HHR-taR12-DT
pSB3C5-crR12-GFP-DT
C)
pSB1A3-Pbad-taR12-DT
pSB3C5-crR12-GFP
D)
pSB1A3-empty
pSB3C5-crR12-GFP
E)
pSB1A3-empty
pSB3C5-empty
2) The transformants were pre-cultured in LB medium at 37 degrees celcius for 8 hours.
3) Then, cultures was diluted to an OD595 of 0.0125 and induced with or without 0.1% arabinose.
4) Cultures were incubated with shaking at 140 rpm and 37℃ for 8 hours in 1 mL of LB medium.
5) OD595 and GFP fluorescence intensity were measured.
Result and Discussion
In order to evaluate a function of TR(42)-HHR-taR12, we used crR12-GFP as a reporter. taRNA is complementary to the sequence of crRNA, and by the binding of taRNA to crRNA, RBS is exposed and gene expression is activated. In this result, the GFP fluorescence value divided by OD value of (2) was lower than that of (3). It shows that taR42 suppressed self-cleavage of TR(42)-HHR and caused less GFP expression. Therefore, we prove that the self-cleavage activity of TR(42)-HHR and amount of taR12 can be controlled by taR42.
・Future work
We must confirm whether taR12 and TR(12)-HHR set is available for RNA oscillator.
The second experiment result showed high background, then we should try to use lower promoter to transcribe crR12-GFP.
Reference
[1]  An engineered small RNA-mediated genetic switch  based on a ribozyme expression platform  Benedikt Klauser and Jo¨ rg S. Hartig  Nucleic Acids Research, 2013
[2]  Callura,J.M., Cantor,C.R. and Collins,J.J. (2012) Genetic  switchboard for synthetic biology applications. Proc. Natl Acad.  Sci. USA, 109, 5850–5855
Retrieved from " http://2013.igem.org/Team:Tokyo-NoKoGen/oscillator "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
