Team:Washington/Description
Home
UW 2015
UW 2014
UW 2013
UW 2012
UW 2011
UW 2010
UW 2009
UW 2008
iGEM Homepage
Wet Lab
Introductions
Methods
Results
Conclusions
BioBricks
Dry Lab
Introductions
Design
Hardware
Demonstration
Conclusions
Human Practices
Collaborations
Engagement
Integrated Practices
Silver
Gold
Protocols
Experiments
Safety
Team
Members
Attributions
Judging Form
Sitemap
Project Description
Project Abstract
Managing cultures is a vital task in synthetic biology, but constantly measuring and adjusting culture conditions is both tedious and labor intensive. Our project aims to reduce the amount of time and effort needed to maintain cultures through the creation of an affordable image analysis system that autonomously reads visual data to measure the current state of a culture and then determines whether to release inducer chemicals based on user input. Our project utilizes the violacein pathway to simulate other metabolic pathways with colored signals. By regulating gene expression in this gene set with two different inducible promoters, we are able to yield up to four different color outputs. These outputs are then measured by an open-sourced Raspberry Pi setup, which captures visual data via camera, measures the culture’s RGB value, and then directs the gradual release of inducer chemicals to maintain or change the culture’s color over time.
Motivations
Adaptive and active manipulation of bacteria is important for maintaining homeostasis of cultures. Biotechnology companies use bacteria to produce important drugs, industrial chemicals, and other products; a prime example of this is the antimalarial drug artemisinin. Chemicals like these are produced through a series of enzymatic reactions within the cell, and are becoming more widespread as synthetic biology is applied to problems in a wide range of areas.
From previous projects, it is clear that detecting the level of metabolites in different cultures, including yeast cultures, is inconvenient. Traditional methods of analysing chemical compounds are expensive and labour intensive. For example, an HPLC (High Performance Liquid Chromatography) instrument can be tens of thousands of dollars to buy, and runs from ranges from 35.00–90.00 dollars per hour to use. For labs and industries that require this service to be run frequently, this is a significant financial burden.
In order to achieve the highest rate of production of the desired metabolites, steps along the pathway should be monitored in order to minimize "bottleneck" steps.
Workflow
Our model system for a metabolic pathway is the violacein pathway used in yeast. This pathway produces four distinct chemicals from five distinct enzymes of which only two contribute to the variety in color changes. We developed a strain of yeast that use the violacein chemicals and are responsive to different chemical inputs due to inducible promoters. When these inducers are added, a color result can be observed.
Figure 1. Violacein genetic pathway.
In order to make observed color changes quantifiable, we decided to build a low-cost instrument for measuring the color of the culture, processing this color using RGB values, and interpret this color as "desired" or "undesired." Then, the flow rates of inducers into the culture are autonomously adjusted to optimize the system towards the desired color. This system uses inexpensive and easily-accessible parts, with a Raspberry Pi for computation, such that it would be easy and affordable for other labs or start-ups to implement on a low budget.
Figure 2. Flow diagram of our image processing and culture management system.
Gold Medal Parts Requirement
In order to meet gold medal requirements, we worked to improve the function of three BioBricks submitted by previous iGEM teams. We codon optimized them for yeast, which should result in a quicker response time to chemical inputs when implemented in our system. We also removed illegal restriction sites in order to make them more suitable for BioBrick assembly. More detailed information can be found on our BioBricks page.
Violacein C Gene Codon Optimized for S. Cerevisiae
This part is a codon optimized version for yeast of the Violacein C gene submitted by the Alberta team in 2013 as K1161003. VioC is one of five genes originally from the Violacein metabolic pathway in Chromobacterium Violaceum that together produce the purple pigmented antibiotic, Violacein, when expressed in yeast.
Violacein D Gene Codon Optimized for S. Cerevisiae
This part is a codon optimized version for yeast of the Violacein D gene submitted by the Alberta team in 2013 as K1161004. VioD is one of five genes derived from the Violacein metabolic pathway in Chromobacterium Violaceum that together produce the purple pigmented antibiotic, Violacein, when expressed in yeast.
CUP1 Yeast Inducible Promoter with RFC 10 Illegal Restriction Sites Removed
CUP1 is a commonly used inducible transcription promoter for yeast that leads to enhanced expression of its respective gene(s) in the presence of cupric ions. The documented natural variants of the CUP1 promoter, including part K945002 in the registry, contain RFC 10 illegal restriction sites that have been eliminated in this part so that it may be used in other BioBricks.
Wet Lab
Dry Lab
Human Practices
Protocols
Team
Introductions Methods Results Conclusions BioBricks
Introductions Design Hardware Demonstration Conclusions
Collaborations Engagement Integrated Practices Silver Gold
Experiments Safety
Members Attributions Judging Form
