<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Washington_Project1 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Washington/Project1</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="nav"><UL><LI><A href="https://2016.igem.org/Team:Washington">Home</A></LI><LI><A href="https://2015.igem.org/Team:Washington">UW 2015</A></LI><LI><A href="https://2014.igem.org/Team:Washington">UW 2014</A></LI><LI><A href="https://2013.igem.org/Team:Washington">UW 2013</A></LI><LI><A href="https://2012.igem.org/Team:Washington">UW 2012</A></LI><LI><A href="https://2011.igem.org/Team:Washington">UW 2011</A></LI><LI><A href="https://2010.igem.org/Team:Washington">UW 2010</A></LI><LI><A href="https://2009.igem.org/Team:Washington">UW 2009</A></LI><LI><A href="https://2008.igem.org/Team:University_of_Washington">UW 2008</A></LI><LI><A href="https://2016.igem.org/Main_Page">iGEM Homepage</A></LI></UL><LI><A href="https://2016.igem.org/Team:Washington/Project1">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Project1#Introductions">Introductions</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Project1#Methods">Methods</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Project1#Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Project1#Conclusions">Conclusions</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Project1#Biobricks">BioBricks</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Drylab">Dry Lab</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Drylab#Introductions">Introductions</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Demonstrate">Demonstration</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Drylab#Conclusions">Conclusions</A></LI><LI><A href="https://2016.igem.org/Team:Washington/HP">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Washington/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:Washington/HP/Gold">Gold</A></LI><LI><A>Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Protocols">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Washington/Judging">Judging Form</A></LI><LI><A href="#footer">Sitemap</A></LI></DIV><H1 class="title first" style="background: url(https://static.igem.org/mediawiki/2016/9/95/T--Washington--Background_Centrifuge.png) center no-repeat;">Wet Lab</H1><DIV class="container"><H2 id="Introductions">Introductions</H2><DIV class="image" style="float: right; padding-left: 40px; padding-bottom: 30px; width: 500px;"><P>Figure 1. The violacein pathway. Post-pigment decay products are not shown. (Lee 2013; Kim 2016)</P></DIV><P>Violacein is a 4 to 6 step pathway whose final products (before decay) are a series of pigments. Originally produced in various chromobacterium, we worked to set up a recombinant yeast strain as a proof-of-concept for a viable and organism-versatile indicator or a cell culture’s status. The pathway works much like a two-bit operator. The pathway has two inputs – two promoters each enabling the production of its respective enzyme, VioC and VioD. The status of these inputs display four different pigments: Green, Purple, Pink, and Teal. Assuming VioA, VioB, and VioE are expressed (these produce the precursors necessary for VioC and VioD), the following pathways are possible: only VioC produces pink, only VioD produces teal, neither VioC nor VioD produces green, and both VioC and VioD produces purple.</P><P>In our example, violacein can either serve as either a model of another color-producing pathway or as a coupled pathway via utilizing identical inducers. The idea is that these pigments will be able to indicate the state in which a cell culture is in. Possibilities range from the simplest (ex. indication of the presence of two inducers in solution) to the complex, (ex. indication of varying hormone levels through gradient-sensitive promoter pathways).</P><P>Violacein is known to express as expected in “<A href="http://link.springer.com/article/10.1007/BF02092154" target="_blank">a wide range of Gram-negative bacteria</A> (Pemberton, 2007),” including chromobacterium and E. coli, and yeast; however, it is known to display many <A href="http://onlinelibrary.wiley.com/doi/10.1042/BA20070115/full" target="_blank">anti-pathogenic characteristics</A>, which could limit feasibility in certain model organisms. In high dosage, violacein has been shown to cause DNA damage and cytotoxicity which could be detrimental to experimental control if not taken into consideration (Durán, 2007).</P><H2 id="Methods">Methods</H2><P>We’ve dedicated a page to our experimental procedures <A href="https://2016.igem.org/Team:Washington/Protocols" target="_blank">here</A>.</P><P>Our end goal for wet lab is automating the production of metabolites. The steps our team took in order to achieve this was by choosing a pathway that produced metabolites whose colors were observable using the naked eye. Genes VioC and VioD were then modified by attaching inducible Copper (Cup1) and Galactose (Gal1) promoters respectively. This was successfully done through overlap-extension PCR. Originally the promoters were constitutively activated; but with the new promoters we are able to produce four different colour outputs. Next, we attempted to create a plasmid consisting of the Vio gene cluster (VioA, B, and E with constitutive promoters and VioB and D with their respective inducible ones) and the vector backbone. And then assemble all the parts via a four part Gibson Assembly.</P><P>After repeated attempts, we were unable to successfully produce a singular plasmid for the entire violacein system. We stepped back and redesigned our plan. We decided a more efficient course of action would be to utilize a three plasmid system. We decided to introduce a new region of overlap into the backbone plus VioB fragment and the VioA plus VioB fragment from our previous attempts and utilize gibson assembly to produce a chassis strain. This chassis, which has the enzymes necessary to create protodeoxyviolaceinic acid (the latest precursor for producing all four pigments), would then be transformed with two additional plasmids: Cup1+VioC and Gal1+VioD. At this point in the project, the codon-optimized versions of VioC and VioD were prepared, pGal1 was available, and pCup1 in its standard biobrick assembly form was on the way. We opted to produce these through biobrick assembly. While there are potential complications to using a multi plasmid system, such as the required variety of selectable markers or a potential loss of a plasmid, this avoids some of the difficulty of producing a 16 kilobase plasmid.</P><H2 id="Results">Results</H2><P>While we successfully extract violacein from the original strain we received from the University of Washington’s DAWGMA group, we were not successful in producing a strain with our desired promoters. There are a few possible sources of this difficulty. We may have been unable to elute properly during purification due to either issues with DNA length or procedural inaccuracies. Primer design may have room for improvement. If the length of the primers were too long, spontaneous formation of hairpins could have formed along the length of the primer due to a high ΔG. Conversely, if the length of the primers were too short, the primers may have nonspecific binding. </P><P>We assembled the four fragments used to make the plasmid: Cup1+VioC, Gal1+VioD, VioABE, and our vector component. However the final plasmid size was 16 kilobases. A four part, 16 kbp Gibson Assembly proved challenging and consistently had low yields and unsuccessful transformations. Although the assembly of this plasmid was unsuccessful, the violacein genes used to make the plasmid came from the biobrick part BBa_K274002, which was built using the well characterized pPSX vio+ plasmid from the Pemberton Group at the University of Queensland. The results of pPSX vio+ plasmid indicated stability without antibiotics for a hundred generations or more (Sarovich 2007). Furthermore, additional characterization of pPSX vio+ performed by the 2009 Cambridge iGEM Team indicated further stability as the violacein pigments remained in the bacteria and did not bleed into the media even after prolonged periods of time (Violacein 2009). Therefore, based on these two prior characterizations we have a firm belief that our plasmid, as a derivative of pPSX vio+, would have shown promising results in pigment stability. We continue to work towards the production of a suitable Violacein strain using our three-plasmid strategy.</P><H2 id="Conclusions">Conclusions</H2><P>Our team’s original goal was to reconstruct the violacein plasmid with inducible promoters to replace the current constituent promoters that are naturally present. We did this so that we could establish the parameters of two inputs with four color signal outputs needed to automate our system. The plasmid we were building was quite large and therefore the Gibson Assemblies repeatedly turned out unsuccessful. Our team then changed our plasmid building methods and decided to make three smaller plasmids and then do a double/tripple transformation. Since our plasmid is derived from the commonly used violacein plasmid ,pPSX vio+, we believe that our end products will be highly pigmented color signals that can be used to automate our system.
</P></DIV><H1 class="title" id="Biobricks" style="background: url(https://static.igem.org/mediawiki/2016/5/55/T--Washington--Background_Bricks.png) center no-repeat;">BioBricks</H1><DIV class="container"><P>For 2016, the University of Washington's iGEM team has worked to develop four biobricks. These are as follows:</P><OL><LI><A href="http://parts.igem.org/Part:BBa_K2165000" target="_blank">BBa_K2165000</A>: VioC (Codon Optimized for S. cerevisiae)</LI><LI><A href="http://parts.igem.org/Part:BBa_K2165001" target="_blank">BBa_K2165001</A>: VioD (Codon Optimized for S. cerevisiae)</LI><LI><A href="http://parts.igem.org/Part:BBa_K2165005" target="_blank">BBa_K2165004</A>: pCup1 (Biobrick Standard Assembly Compatible)</LI><LI><A href="http://parts.igem.org/Part:BBa_K2165002" target="_blank">BBa_K2165002</A>: pCup1 + VioC (Codon Optimized) + ADH1</LI><LI><A href="http://parts.igem.org/Part:BBa_K2165003" target="_blank">BBa_K2165003</A>: Gal1 + VioD (Codon Optimized) + Terminator</LI></OL><P>It should be stressed that without the other components of the Violacein pathway (VioA, VioB, and VioE, see part <A href="http://parts.igem.org/Part:BBa_K726015" target="_blank">BBa_K726015</A> as an example), the substrate for VioC and VioD will not be produced and no pigment will be visible.</P><H2><A href="http://parts.igem.org/Part:BBa_K2165000" target="_blank">BBa_K2165000</A>: VioC (Codon Optimized for S. cerevisiae)</H2><DIV class="image" style="float: right; padding-left: 40px; padding-bottom: 30px; width: 250px;"><P>Figure 2. The proposed structure for VioC (Helmetag 2009). Rendered with PyMOL.</P></DIV><P>VioC is the final (fourth, or fifth if VioD also present) enzyme in the violacein pathway when expressed. VioC, alongside VioA, VioB, and VioE, processes its substrate protodeoxyviolaceinic acid into deoxyviolaceinic acid. Deoxyviolaceinic acid then decays, producing a pink-color pigment. When VioD is also expressed, protoviolaceinic is instead used as VioC's substrate, producing an intermediate which subsequently degrades into a dark, purple pigment called volacein.</P><P>VioC was previously available as a standalone part as <A href="http://parts.igem.org/Part:BBa_K539513" target="_blank">BBa_K539513</A>, produced by <A href="https://2011.igem.org/Team:NCTU_Formosa">NCTU Formosa's 2011 iGEM team</A>; however, BBa_K539513 is not codon optimized for yeast. The part developed by the University of Washington's iGEM team have been codon optimized for yeast in hopes to improve the efficiency of its translation. The CDS contained in this BioBrick was designed by running the VioC gene sent to the University of Washington by the <A href="http://dueberlab.berkeley.edu/" target="_blank">Dueber Laboratory</A> at University of California-Berkley through IDT's Codon Optimization Tool for S. cerevisiae.</P><H2><A href="http://parts.igem.org/Part:BBa_K2165001" target="_blank">BBa_K2165001</A>: VioD (Codon Optimized for S. cerevisiae)</H2><DIV class="image" style="float: left; padding-right: 40px; padding-bottom: 30px; width: 250px;"><P>Figure 3. The proposed structure for VioD (Forouhar n.d.). Rendered in PyMOL.</P></DIV><P>VioD is the fourth (and final if VioC is not also present) enzyme in the violacein pathway when expressed. VioD, alongside VioA, VioB, and VioE, processes its substrate protodeoxyviolaceinic acid into protoviolaceinic acid. When VioC is unavailable, this degrades into proviolacein, a teal-colored pigment. When VioC is present this precursor is processed into an intermediate which subsequently degrades into a dark, purple pigment called volacein.</P><P>VioD was previously available as a standalone part as <A href="http://parts.igem.org/Part:BBa_K539413" target="_blank">BBa_K539413</A>, produced by NCTU Formosa's 2011 iGEM team; however, BBa_K539513 is not codon optimized for yeast. The part developed by the University of Washington's iGEM team have been codon optimized for yeast in hopes to improve the efficiency of its translation. The CDS contained in this BioBrick was designed by running the VioD gene sent to the University of Washington by the Dueber Laboratory at University of California-Berkley through IDT's Codon Optimization Tool for S. cerevisiae.</P><H2><A href="http://parts.igem.org/Part:BBa_K2165004" target="_blank">BBa_K2165004</A>: pCup1 (Biobrick Standard Assembly Compatible)</H2><P>The Cup1 promoter enables binding of RNA Polymerase II and the subsequent transcription of downstream DNA to mRNA. It is activated by ACE1, a transcription factor which binds to copper ions. It was previously available as a standalone part as <A href="http://parts.igem.org/Part:BBa_K945002" target="_blank">BBa_K945002</A>, produced by Tec-Monterrery’s 2012 iGEM team; however, the original part contained numerous illegal restriction sites preventing its usability in standard biobrick assembly. Hard information of the Greenfield 2011 team’s <A href="http://parts.igem.org/Part:BBa_K586000" target="_blank">BBa_K586000</A> claims the part is the Cup1 regulatory region, but based on the sequence and its twin we believe it to be mislabeled and that it is instead the CupI CDS. </P><P>We acquired our starting sequence from the Vicker’s Lab at the Australian Institute for Bioengineering and Nanotechnology<A href="http://www.addgene.org/83555/" target="_blank">via Addgene</A>, subsequently removed or altered bases corresponding to illegal cut sites, added a biobrick prefix and suffix, and finally ordered the unit as a geneblock from IDT.</P><H2><A href="http://parts.igem.org/Part:BBa_K2165002" target="_blank">BBa_K2165002</A>: pCup1 + VioC (codon optimized) + ADH1</H2><P>This biobrick utilizes our Cup1 promoter. Following the binding of a heavy metal to the Cup1 promoter then VioC transcription is upregulated. The effects on the transcription of the VioC protein can be seen above in the VioC section. The biobrick ends with the ADH1 terminator <A href="http://parts.igem.org/Part:BBa_K801012" target="_blank">BBa_K801012</A> for yeast that was submitted by the TU Munich team in 2012.</P><P>This biobrick was designed to characterize the codon optimized version of VioC. The codon optimized VioC should have higher translation rates than the previously made VioC, and this plasmid construct will be used to determine the change in the expression levels for the pathway. For more information on the role of VioC in the violacein pathway see the above VioC.</P><H2><A href="http://parts.igem.org/Part:BBa_K2165003" target="_blank">BBa_K2165003</A>: Gal1 + VioD + Terminator</H2><P>The promoter for the biobrick uses the Gal1 promoter sequence, which is activated in the presence of galactose but repressed in the presence of glucose. The sequence for the Gal1 promoter that was used for this biobrick is the 2006 Silver lab’s <A href="http://parts.igem.org/Part:BBa_J63006" target="_blank">BBa_J63006</A>. The effect on the transcription of the VioD protein can be seen above in the VioC section. The biobrick ends with the ADH1 terminator BBa_K801012 for yeast that was submitted by the TU Munich team in 2012.</P><P>This biobrick was designed to characterize the codon optimized version of VioD. The codon optimized VioD should have higher translation rates than the previously made VioD, and this plasmid construct will be used to determine the change in the expression levels for the pathway.  For more information on the role of VioD in the violacein pathway see the above VioD.</P><H3>Works cited</H3><P>Durán, N. et al. Violacein : properties and biological activities. 133, 127–133 (2007).</P><P>Forouhar, F., Neely, H., Seetharaman, J., Janjua, H., Xiao, R., Maglaqui, M., Wang, H., Baran, M.C., Acton, T.B., Montelione, G.T., Hunt, J.F., Tong, L., Northeast Structural Genomics Consortium. Crystal structure of vioD hydroxylase in complex with FAD from Chromobacterium violaceum. PDB ID: 3C4A</P><P>Helmetag, V., Samel, S. A., Thomas, M. G., Marahiel, M. A. &amp; Essen, L. O. Structural basis for the erythro-stereospecificity of the l-arginine oxygenase VioC in viomycin biosynthesis. FEBS J. 276, 3669–3682 (2009).</P><P>Kim, S. et al. PubChem substance and compound databases. Nucleic Acids Research 44, (2016).</P><P>Lee, M. E., Aswani, A., Han, A. S., Tomlin, C. J. &amp; Dueber, J. E. Expression-level optimization of a multi-enzyme pathway in the absence of a high-throughput assay. Nucleic Acids Res. 41, 10668–10678 (2013).</P><P>Pemberton, J., Vincent, K. &amp; Penfold, R. Cloning and heterologous expression of the violacein biosynthesis gene cluster fromChromobacterium violaceum. Curr. Microbiol. 22, 355–358 (1991).</P><P>Sarovich, D. S. &amp; Pemberton, J. M. pPSX: A novel vector for the cloning and heterologous expression of antitumor antibiotic gene clusters. Plasmid 57, 306–313 (2007).</P><P>Violacein Pigments. iGEM Cambridge 2009 (2009). Available at: https://2009.igem.org/Team:Cambridge/Project/Violacein.</P></DIV><DIV id="footer"><TABLE><TBODY><TR><TD><A href="https://2016.igem.org/Team:Washington/Project1">Wet Lab</A></TD><TD><A href="https://2016.igem.org/Team:Washington/Drylab">Dry Lab</A></TD><TD><A href="https://2016.igem.org/Team:Washington/HP">Human Practices</A></TD><TD><A>Protocols</A></TD><TD><A href="https://2016.igem.org/Team:Washington/Team">Team</A></TD></TR><TR><TD><A href="https://2016.igem.org/Team:Washington/Project1#Introductions">Introductions</A><A href="https://2016.igem.org/Team:Washington/Project1#Methods">Methods</A><A href="https://2016.igem.org/Team:Washington/Project1#Results">Results</A><A href="https://2016.igem.org/Team:Washington/Project1#Conclusions">Conclusions</A><A href="https://2016.igem.org/Team:Washington/Project1#Biobricks">BioBricks</A></TD><TD><A href="https://2016.igem.org/Team:Washington/Drylab#Introductions">Introductions</A><A href="https://2016.igem.org/Team:Washington/Design">Design</A><A href="https://2016.igem.org/Team:Washington/Hardware">Hardware</A><A href="https://2016.igem.org/Team:Washington/Demonstrate">Demonstration</A><A href="https://2016.igem.org/Team:Washington/Drylab#Conclusions">Conclusions</A></TD><TD><A href="https://2016.igem.org/Team:Washington/Collaborations">Collaborations</A><A href="https://2016.igem.org/Team:Washington/Engagement">Engagement</A><A href="https://2016.igem.org/Team:Washington/Integrated_Practices">Integrated Practices</A><A href="https://2016.igem.org/Team:Washington/HP/Silver">Silver</A><A href="https://2016.igem.org/Team:Washington/HP/Gold">Gold</A></TD><TD><A href="https://2016.igem.org/Team:Washington/Protocols">Experiments</A><A href="https://2016.igem.org/Team:Washington/Safety">Safety</A></TD><TD><A href="https://2016.igem.org/Team:Washington/Team">Members</A><A href="https://2016.igem.org/Team:Washington/Attributions">Attributions</A><A href="https://2016.igem.org/Team:Washington/Judging">Judging Form</A></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>