<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SJTU-BioX-Shanghai_project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SJTU-BioX-Shanghai/project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SJTU-BioX-Shanghai/project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SJTU-BioX-Shanghai/project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SJTU-BioX-Shanghai/project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="contentarea"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Project_overview:_Synthetic-biological_Approaches_to_Osteoarthritis_.28OA.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project overview: Synthetic-biological Approaches to Osteoarthritis (OA)</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Background"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#The_eukaryotic_approach"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">The eukaryotic approach</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Introduction"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Design"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-4 tocsection-6"><A href="#Detector"><SPAN class="tocnumber">1.2.2.1</SPAN><SPAN class="toctext">Detector</SPAN></A></LI><LI class="toclevel-4 tocsection-7"><A href="#Supervisor"><SPAN class="tocnumber">1.2.2.2</SPAN><SPAN class="toctext">Supervisor</SPAN></A></LI><LI class="toclevel-4 tocsection-8"><A href="#Actuator"><SPAN class="tocnumber">1.2.2.3</SPAN><SPAN class="toctext">Actuator</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#Future"><SPAN class="tocnumber">1.2.2.4</SPAN><SPAN class="toctext">Future</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-10"><A href="#The_prokaryotic_approach"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">The prokaryotic approach</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Introduction_2"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Design_2"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-4 tocsection-13"><A href="#Detector_2"><SPAN class="tocnumber">1.3.2.1</SPAN><SPAN class="toctext">Detector</SPAN></A></LI><LI class="toclevel-4 tocsection-14"><A href="#Supervisor_2"><SPAN class="tocnumber">1.3.2.2</SPAN><SPAN class="toctext">Supervisor</SPAN></A></LI><LI class="toclevel-4 tocsection-15"><A href="#Actuator_2"><SPAN class="tocnumber">1.3.2.3</SPAN><SPAN class="toctext">Actuator</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Future_2"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Future</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Conclusion"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#References"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Project_overview:_Synthetic-biological_Approaches_to_Osteoarthritis_.28OA.29">Project overview: Synthetic-biological Approaches to Osteoarthritis (OA)</SPAN></H2><H3><SPAN class="mw-headline" id="Background">Background</SPAN></H3><OL><LI>Gene therapy has received much attention in recent years. Despite a wide diversity of strategies, such as the use of gene transfer, RNAi, engineered zinc finger nuclease, in the future, maybe IPS, the basic and core concept of gene therapy is very simple——introduce the gene, and its products should cure or slow down the progression of the disease. In the beginning, the delivery vehicles needed to ferry the genetic material into the cell represent the &quot;Achilles' heel&quot; of gene therapy. And now, the emphasis is on regulation, cell- or tissue- specific expression and safety of gene therapy.
</LI><LI>Osteoarthritis, also called degenerative arthritis, is the most common arthritis and the leading cause of disability. Most of people with ages over 65 suffer from this disease, more or less. OA is characterized by the cartilage matrix degradation and the remodeling of underlying bone, and these will cause the stiffness of joint and pain. Its widespread influence and its severe consequence result in the increasing number of OA researches and some conclusions have been made. Those studies indicate that the only cells in cartilage, chondrocytes, play a pivotal role. In the late of OA, chondrocytes differentiate into a hypertrophic state rather a rest-state in normal cartilage. Those hypertrophic cells synthesize not only cartilage matrix but also Type X collagen (Col10), the specific cytokine of OA and MMP-13, a collagenase that is directly responsible for the matrix degradation.
</LI></OL><P>
Click to jump to:<UL class="navlist"><LI style="width:24%;"><A href="#euk">Eukaryotic part</A></LI><LI style="width:24%;"><A href="#prok">Prokaryotic part</A></LI><LI style="width:24%;"><A href="#conc">Conclusion</A></LI><LI style="width:24%;"><A href="#ref">References</A></LI></UL></P><H3><SPAN class="mw-headline" id="The_eukaryotic_approach">The eukaryotic approach</SPAN></H3><P>Click to jump to:<UL class="navlist"><LI><A href="#eukint">Introduction</A></LI><LI><A href="#eukdes">Design</A></LI><LI><A href="#eukdct">Detector</A></LI><LI><A href="#euksup">Supervisor</A></LI><LI><A href="#eukact">Actuator</A></LI><LI><A href="#eukfut">Future</A></LI></UL></P><H4><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
In recent years, significant progress in the area of tissue- or cell- specific expression has been made. Two main strategies have been generally pursued. Both of them focused on vector development. The first strategy involves engineering viral proteins responsible for binding the cellular receptors that subsequently mediate viral entry. The second strategy relies on adaptor molecules that bridge the virus and the cell. In our system, the core components are produced from a tissue-specific promoter, this can enable spatial control of gene expression.
<P>What's more, we suspect that regulated expression of the inserted gene will be important in gene therapy. An ideal regulatory system should have some good attributes. Such as, the induction should be reversible, the inducer must be non-toxic. In our design, the regulatory system is stimulated by blue light because a photosensitive ion channel is used. (ChR2, which will be mentioned later)
</P><P>Furthermore, the components of the regulatory system should not be immunogenic in the host. The biggest challenge all gene therapies face is the immune response of the host. The host immune system will recognize the viral vector, the inserted gene, the regulatory system and the transgenic products as foreign. To minimize the immune response, most scientists dedicate to design safe vector, and they do make progress. While the SJTU-BioX-Shanghai team has constructed a system in which most of the devices in use are endogenous, and external stimuli have its effect on the cell through existing cellular signaling pathways.
</P><H4><SPAN class="mw-headline" id="Design">Design</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV><H5><SPAN class="mw-headline" id="Detector">Detector</SPAN></H5><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
The first thing we must face in our approaches to OA is how to detect cells and tissues where hypertrophic differentiation takes place. Considering the eukaryotic method will be applied by inserting the curing circuit into the chromosome of mammalian cells, what we need to do is to find out the molecular markers of OA whose expression level will increase in the case of hypertrophic differentiation.
<P>The type X collagen gene (Col10a1) is such a specific molecular marker of hypertrophic chondrocytes. Multiple cis-elements and trans-acting factors have been reported to regulate Col10a1 expression both in vitro and in vivo and studies have been made on the 5’ flanking region of Col10a1 gene. Scientists found out that Col10a1 promoter locates between 4410 upstream of the initial of transcription and 643 downstream. Further studies showed that when six copies of a 90bp fragment from a enhancer site between -4.4kb to -3.8kb is aligned with Col10 basal promoter (-120 to +1 bp), the chimeric promoter showed high tissue specificity (high efficiency of initiating transcription in hypertrophic chondrocytes).
</P><P>In approaches of SJTU-BioX-Shanghai, we use the 90bp enhancer fragment (2X) and align it with synthetic JeT core promoter. The chimeric promoter we constructed is supposed to have high tissue specificity and it would initiate transcription only when it is transfected into mammalian cells. The tissue specific promoter restricts the expression of our Supervisor device in hypertrophic chondrocytes. And prevent the inserting genes from harming other normal tissues.
</P><P>Growth differentiation factor 5 (GDF5) is also a molecular maker of chondrocytes and it is known to be involved in joint formation. Studies have been made on the 5’ flanking region of GDF5 gene and researchers found out GDF5 promoter locates between 1101 upstream of the transcription initiating point and 367 downstream.
</P><P>As can be seen from Figure 2, the third DNA fragment (which is 769bp long) has the highest luciferase activity. Therefore, we insert our Supervisor device just downstream of the 769bp GDF5 promoter.
</P><H5><SPAN class="mw-headline" id="Supervisor">Supervisor</SPAN></H5><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
After the detection of tissue where OA takes place, the Supervisor begins to work. Its job is to control the expression of the curing system (Actuator) under the instruction of external signals.
<P>Channelrhodopsin-2 (ChR2) is a light gate ion channel serve as sensory photoreceptors in unicellular green algae2. In the project of team SJTU-BioX-Shanghai this year, we use ChR2 (courtesy of Dr. Weidong Li) as a photosensitive calcium channel which introduces calcium signals into the cell when it is exposed to blue light.
</P><P>After the Ca2+ enters the cell, the SJTU-BioX-Shanghai team modified cellular signal pathways which cross-talks with calcium signal. As a result, the cells begin to response to external light signal and initiate the transcription of the Actuator device.
</P><P>We found and modified two cellular pathways which are related to Ca2+. One is calcium dependent T Cell Receptor (TCR) signaling pathway and the other calcium-dependent mitogen-activated protein kinase (MAPK) pathway.
</P><P>Calcium dependent TCR pathway:
</P><P>The gene regulatory network can be characterized via three pathways’ synergic effect on the gene expression with a cascade of interactions between protein modules involved. The key nodes of the network are CAM, MEF2-Cabin1 complex, and MEF2-NFAT complex. (Further discussion of the three pathways’ synergic work will be showed in the modeling part.)
</P><P>The effect of the cellular signal transduction is showed in figure 4 below:
</P><P>In the absence of calcium, Mitogen Enhancer Factor 2 (MEF2) transcription factor binds to specific DNA sequence upstream of core promoter and recruits cytokine Cabin1, who further recruits Histone deacetylase (HDAC) and blocks transcription. While the cell is stimulated with blue light, ChR2 will introduce more Ca2+, and calcium influx will result in the release of Cabin1-mSin3-HDACs complex. Meanwhile, MEF2 will recruit p300 Histone acetyltransferase (HAT) instead. And transcription is initiated at that point.
</P><P>In the design of team SJTU-BioX-Shanghai, we aligned MEF2 enhancer with synthetic JeT promoter in order to make the curing system under the control of calcium signals.
</P><P>Calcium dependent MAPK pathway:
</P><P>Cellular MAPK pathway is one of the best studied signaling pathways scientists have focused on. The pathway begins from the activation of membrane-bonded Ras protein and through the activation of extracellular signal-regulated kinase (ERK), the extracellular signal is transducted into the nucleus.
</P><P>While calcium dependent MAPK pathway is mainly studied in brain and it controls synaptic plasticity. The reason for which Ca2+ could cross-talk with MAPK pathway in brain is that the membrane-bonded Ras protein is activated by Ras-GRF1, a Ca2+/calmodulin-dependent Ras-guanine-nucleotide-releasing factor, which is highly brain specific. Ras-GRF1 interact with NMDA subtype of glutamate receptors (NMDAR) and tranduces signals of Ca2+ influx from the ion channel to the activation of Ras protein.
</P><P>After the MAPK pathway is activated, activated ERK enters the nucleus and further activates transcription factors such as cyclic AMP response element (CRE)-binding protein (CRE-BP) family. (shown in the figure below)
</P><P>In the design of team SJTU-BioX-Shanghai, we fused ChR2 ion channel and Ras-GRF1 (courtesy of Prof. Grigory Krapivinsky) so that Ras-GRF1 is sensitive to local increase of Ca2+ introduced by ChR2. And in order to detect the signals from MAPK pathway, we build synthetic promoter with CRE enhancer aligned with JeT core promoter. And our design can be shown below:
</P><H5><SPAN class="mw-headline" id="Actuator">Actuator</SPAN></H5><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
Osteoarthritis (OA) has two major characters, joint matrix is degraded and chondrocytes undergo disordered and hypertrophic differentiation.
In order to reverse the hypertrophic differentiation, the team SJTU-BioX-Shanghai comes up with the idea of induced pluripotent stem cells (iPS cells).If we could reverse the hypertrophic chondrocytes into iPS cells, we might be able to have further stimulation on these cells and turn them into normal chondrocytes again. Although most of these designs are based on our hypothesis, we are eager to have them done.
When it comes to the degradation of joint matrix, our solution is much simpler. We cloned the gene col2a1, which is used to replenish joint matrix, and make its expression under the control of our Supervisor.
<H5><SPAN class="mw-headline" id="Future">Future</SPAN></H5><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
To test their design, the SJTU-BioX-Shanghai team has focused on experiments on mammalian cell lines this summer. However, the project will be more promising if we could have the whole system tested on animals. To do this, we have to design viral vectors which will insert the whole system into the cell chromosome and test whether it will work as expected in vivo.
There will be a long way before our design could be demonstrated useful new approach to treat OA. While we could imagine the advantages this project will bring if it is used practically: first of all, we could just use a minimally invasive surgery which introduces a thin optical fiber into the joint cavity and the whole system starts work under the control of our Supervisor; Furthermore, the system has least effect on other tissues and cells because of the Detector, which contains tissue specific promoter and restricts the expression in hypertrophic chondrocytes only. Moreover, the expression of the curing system is under our control because of the Supervisor. We can stop the curing process as soon as possible in case it has negative effect on the patient.
<H3><SPAN class="mw-headline" id="The_prokaryotic_approach">The prokaryotic approach</SPAN></H3><P>Click to jump to:<UL class="navlist"><LI><A href="#prokint">Introduction</A></LI><LI><A href="#prokdes">Design</A></LI><LI><A href="#prokdct">Detector</A></LI><LI><A href="#proksup">Supervisor</A></LI><LI><A href="#prokact">Actuator</A></LI><LI><A href="#prokfut">Future</A></LI></UL></P><H4><SPAN class="mw-headline" id="Introduction_2">Introduction</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
Our prokaryotic team designs a genetically engineered E.coli as a treatment to OA. We hope this genetic machine could be injected into cartilage and it could “know the occurrence of OA” and then excrete Oct4 to reverse the disorderly hypertrophic differentiation.
However, trying to use this bacteria therapy, we encounter three problems: How to sense OA, How to make Oct4 excreted and How to control the side effects of bacteria therapy. How could bacteria in cartilage sense OA? OA-specific cytokines and transcription factors are hard to be detected by bacteria for they are either macromolecules or inside chondrocytes. Here, we find papers reported that the oxygenic pressure in cartilage is rather low, esp. when OA happens and the appearance of nitric oxide (NO) could be detected in OA. Therefore we choose the co-occurrence of these two factors as the signal of OA. 
<P>Then, how to deal with OA? Our prokaryotic design shares a same curing part as the eukaryotic one.
</P><P>Finally, what strategy should be used to control side effects? The side effects are caused by two problems: the over-population of bacteria and the existence of bacteria after curing OA. Here, we employ quorum sensing system and RNase Barnase, a cell death peptide, to control the population of bacteria and clean existing genetically modified bacteria.
</P><H4><SPAN class="mw-headline" id="Design_2">Design</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
The genetic circuit consists of three parts: Detector, Actuator and Supervisor, just the same as the eukaryotic design. Detector aims to sense OA; Actuator consists of two curing genes, Oct4 and Col2a1; Supervisor is used to control the population of bacteria and clean residual bacteria.
<H5><SPAN class="mw-headline" id="Detector_2">Detector</SPAN></H5><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
We choose Hypoxia and the existence of NO as the signal of OA. In order to detect hypoxia, we use the fdhF promoter (PfdfF). This promoter could be taken as a switch—on in hypoxia and off in normal oxigenic pressure. Except for PfdhF to sense hypoxia, we employ the soxRS system in E.coli to detect NO. The product of gene soxR, SoxR protein could be activated by NO and this active form of SoxR will activate soxS promoter (PsoxS), in which case the curing genes, oct4 and col2a1, express. These two elements combine to form a “And” gate device. NO and hypoxia could switch it on.
<H5><SPAN class="mw-headline" id="Supervisor_2">Supervisor</SPAN></H5><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
To reduce the side effect, we employ a Quorum sensing system based population control method, which is similar as one in: As the the population of bacteria is over a limit (threshold), the quorum sensing system will activate the expression of a cell-death gene, RNase Barnase. The expression of Barnase will cut RNA in bacteria, which will lead to its death but not result in lyses. Thus, the potential severe inflammation, caused by bacteria lyses, is avoided. Besides, we also design a clean system to eliminate existing engineered bacteria after OA is cured. Arabinose-induce promoter (Pbad) combine with Barnase forms the whole system. As the outside signal (Arabinose) is added, the engineered bacteria will be killed or, more accurately commit suicide. Thus, residual bacteria will be cleaned.
<H5><SPAN class="mw-headline" id="Actuator_2">Actuator</SPAN></H5><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
Detailed in the eukaryotic project is the mechanism of Oct4 to reverse the disorderly hypertrophic differentiation of chondrocytes and the Col2a1 to replenish degraded cartilage matrix. Following, just the functions of OmpA excrete peptide (a 21AA fragment of the Outer Membrane Protein A, OmpA) and the 11 arginine(11R) PTD(protein transduction domain) will be describe. To solve the problem of excrete Oct4, we fuse a OmpA signal peptide to the N-terminus of Oct4 and to make it able to get through the cells membrane and go into the nucleus, a 11 arginine PTD is fused in its C-terminus just as do. These two peptides coordinately endow Oct4 the ability to be excreted by our engineered bacteria and then go into chondrocytes’ nucleus, in which case Oct4 could be of effect. As for Col2a1, it is fused with only OmpA in order to make it in the cartilage.
<H4><SPAN class="mw-headline" id="Future_2">Future</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
E.coli is a well-characterized model organism. It means we can, to a large extent, control and predict its behavior. This is the foundation of making biomachine based on E.coli. Besides, many new genes’ discovery and cloned will expand functions could be achieved by biomachine. Furthermore, inventions of artificial devices also offer many choices. Therefore, a promising future of prokaryotic machine could be expected.
<P>However, some problems remain to be solved, esp. when using E.coli as a medicine carrier. First question come into mind is the safety problem: how to ensure the safety of this therapy and how to deal with the conflict between E.coli and the immune system? The safety problem could be . However, it could not be solved completely. As for the immune problem, rearrangement of its membrane to remove LPS could relieve its immunogenicity. Second question will be the tissue-specific or specificity problem: how to find the target? In our prokaryotic project, we could not detect the appearance of macromolecule and circumvent it by sensing small chemicals and environment. However, in this case the specificity is reduced. Protein-fusing may be the answer for it. For example, a cyanobacterial photoreceptor and an E. coli intracellular histidine kinase domain fuse to form a light-sensitive switch. In our case, we could fuse the receptor of MMP-13, a OA-specific cytokine and a E.coli intracellular kinase. This chimerical protein is likely to sense MMP-13 and this is a direction to improve our project.
</P><H3><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H3><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
The SJTU-BioX-Shanghai team has conducted two related projects, one eukaryotic and the other prokaryotic. Therefore, it is easy to come to the question that which one is better, at least in the field of human health.
<P>In detecting where the specific disease takes place, eukaryotic approach has its advantages over the prokaryotic one because it is easier to sense cellular signals such as cytokines, inflammatory and transcription factors. In our project this year, although the Detector device of the prokaryotic approach is twice as complex as that of the eukaryotic one (which only has a tissue specific promoter), the prokaryotic one is less accurate because it can only detect extracellular signals such as NO.
</P><P>When it comes to the effectors of the curing system, eukaryotic method also has its superiority because the curing genes can have their effect in the cell where it is produced. While in the prokaryotic method, we need to attach signal peptides to the curing proteins in order to enable their exocytosis and endocytosis.
</P><P>However, eukaryotic cells are less controllable because of the complex signal transduction pathways and interaction between different pathways. In our design this year, we have to consider all the possible signaling pathways and have them calculated in a model before experiment. The Supervisor of prokaryotic design is quite straight forward in comparison.
</P><P>Therefore, the eukaryotic synthetic biology in curing disease has its advantages in detecting the disease and curing it. While the whole system is hard to control in eukaryotic cells, compared with using prokaryotic synthetic biology.
</P><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV><UL><LI>[1] Qiping Zheng et al. Localization of the Cis-Enhancer Element for Mouse Type X Collagen Expression in Hypertrophic Chondrocytes In Vivo. JOURNAL OF BONE AND MINERAL RESEARCH. Volume 24, Number 6, 2009
</LI></UL><UL><LI>[2] Yoshinari Miyamoto, Akihiko Mabuchi et al. A functional polymorphism in the 5' UTR of GDF5 is associated with susceptibility to osteoarthritis. NATURE GENETICS. 25 March 2007
</LI></UL><UL><LI>[3] Leopoldo Petreanu et al. The subcellular organization of neocortical excitatory connections. NATURE. Vol 457 26 February 2009
</LI></UL><UL><LI>[4] Jun O. Liu et al. Cabin1 Represses MEF2-Dependent Nur77 Expression and T Cell Apoptosis by Controlling Association of Histone Deacetylases and Acetylases with MEF2. Immunity. Vol. 13, 85–94, July, 2000
</LI></UL><UL><LI>[5] Gareth M. Thomas, Richard L. Huganir. MAPK cascade signalling and synaptic plasticity. NATURE REVIEWS NEUROSCIENCE. VOLUME 5 MARCH 2004
</LI></UL><UL><LI>[6] Grigory Krapivinsky, Luba Krapivinsky, Yunona Manasian. et al. The NMDA Receptor Is Coupled to the ERK Pathway by a Direct Interaction between NR2B and RasGRF1. Neuron. Vol. 40, 775–784, November 13, 2003
</LI></UL><UL><LI>[7] Charles L. Farnsworth. et al. Calcium activation of Ras mediated by neuronal exchange factor Ras-GRF. NATURE. Aug 10, 1995
</LI></UL><UL><LI>[8] Huibin Yang. et al. Phosphorylation of the Ras-GRF1 Exchange Factor at Ser916/898 Reveals Activation of Ras Signaling in the Cerebral Cortex. THE JOURNAL OF BIOLOGICAL CHEMISTRY. Vol. 278, No. 15, Issue of April 11
</LI></UL><UL><LI>[9] Adam J. Shaywitz, Michael E. Greenberg CREB: A STIMULUS-INDUCED TRANSCRIPTION FACTOR ACTIVATED BY A DIVERSE ARRAY OF EXTRACELLULAR SIGNALS. ANNU. REV. BIOCHEM. 1999
</LI></UL><UL><LI>[10] Mary B. Goldring and Steven R. Goldring, Osteoarthritis. Journal of Cellular Physiology Cellular Physiology, 2007, 626-634
</LI></UL><UL><LI>[11] Zhou H, Wu S, Joo JY, Zhu S, Han DW, Lin T, Trauger S, Bien G, Yao S, Zhu Y, et al. Generation of Induced Pluripotent Stem Cells Using Recombinant Proteins. Cell Stem Cell. 2009
</LI></UL><UL><LI>[12] Biniecka M, Kennedy A, Fearon U, et al. Oxidative damage in synovial tissue is associated with in vivo hypoxic status in the arthritic joint. Ann Rheum Dis. Published Online First: 24 August 2009
</LI></UL><UL><LI>[13] Anderson JC, Clarke EJ, Arkin AP, Voigt CA (2005) Environmentally controlled invasion of cancer cells by engineered bacteria. J Mol Biol 355: 619–627
</LI></UL><UL><LI>[14] H. Ding and B. Demple, Direct nitric oxide signal transduction via nitrosylation of iron-sulfur centers in the SoxR transcription activator. Proc. Natl. Acad. Sci. USA 97 (2000), pp. 5146–5150
</LI></UL><UL><LI>[15] Secretion of recombinant Bacillus hydrolytic enzymes using Escherichia coli expression systems. Journal of Biotechnology 2008
</LI></UL><UL><LI>[16] You L, Cox III RS, Weiss R, Arnold FH (2004) Programmed population control by cell–cell communication and regulated killing. Nature 428: 868–871
</LI></UL><UL><LI>[17] Levskaya, A., Chevalier, A. A., Tabor, J. J., Simpson, Z. B., Lavery, L. A., Levy, M., Davidson, E. A., Scouras, A., Ellington, A. D., Marcotte, E. M., &amp; Voigt, C. A. (2005) Engineering E. coli to see light, Nature, 438: 441-442.
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:SJTU-BioX-Shanghai/project">http://2010.igem.org/Team:SJTU-BioX-Shanghai/project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SJTU-BioX-Shanghai/project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SJTU-BioX-Shanghai/project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/project&amp;oldid=196051" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>