<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SJTU-BioX-Shanghai_result"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SJTU-BioX-Shanghai/result">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SJTU-BioX-Shanghai/result&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/result&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/result&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SJTU-BioX-Shanghai/result" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SJTU-BioX-Shanghai/result</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="contentarea"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Result_overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Result overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Parts"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Parts</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Demonstration"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Demonstration</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Eukaryotic_approach"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Eukaryotic approach</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#I._Detector"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">I. Detector</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#II._Supervisor_-_ChR2"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">II. Supervisor - ChR2</SPAN></A></LI><LI class="toclevel-4 tocsection-7"><A href="#Expression_of_ChR2"><SPAN class="tocnumber">3.1.2.1</SPAN><SPAN class="toctext">Expression of ChR2</SPAN></A></LI><LI class="toclevel-4 tocsection-8"><A href="#Function_of_ChR2_by_RT-PCR"><SPAN class="tocnumber">3.1.2.2</SPAN><SPAN class="toctext">Function of ChR2 by RT-PCR</SPAN></A></LI><LI class="toclevel-4 tocsection-9"><A href="#Function_by_Real-time_PCR"><SPAN class="tocnumber">3.1.2.3</SPAN><SPAN class="toctext">Function by Real-time PCR</SPAN></A></LI></UL><LI class="toclevel-3 tocsection-10"><A href="#II._Supervisor_-_MEF2-JeT_promoter"><SPAN class="tocnumber">3.1.3</SPAN><SPAN class="toctext">II. Supervisor - MEF2-JeT promoter</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#CRE-JeT_promoter"><SPAN class="tocnumber">3.1.4</SPAN><SPAN class="toctext">CRE-JeT promoter</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#III._Actuator"><SPAN class="tocnumber">3.1.5</SPAN><SPAN class="toctext">III. Actuator</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Prokaryotic_approach"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Prokaryotic approach</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#PfdhF.2C_the_hypoxia-inducible_promoter"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">PfdhF, the hypoxia-inducible promoter</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Characterization_of_existing_parts"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Characterization of existing parts</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#RNase_barnase"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">RNase barnase</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Result_overview">Result overview</SPAN></H2><P>After one month of brainstorming and three months of hard lab working, the SJTU-BioX-Shanghai team has successfully submitted 13 new parts, of which a large portion, such as the <A href="http://partsregistry.org/Part:BBa_K387003" class="external text" rel="nofollow">PfdhF promoter</A>, <A href="http://partsregistry.org/Part:BBa_K387001" class="external text" rel="nofollow">MEF2-JeT promoter</A> and <A href="http://partsregistry.org/Part:BBa_K387012" class="external text" rel="nofollow">CRE-JeT promoter</A>, are new parts that are demonstrated capable of working as expected. Moreover, <A href="http://partsregistry.org/Part:BBa_I716211" class="external text" rel="nofollow">RNase barnase</A> is an existing part in the Partsregistry, and we also have it characterized in order to provide more data.
</P><P>Furthermore, we have built models which correspond to most of our experiment results, meaning they are highly simulative and thus capable of providing essential support for our design.
</P><P>With the brilliant results of these main modules, our project, Synthetic-biological approaches to Osteoarthritis was proven to be highly feasible and has a great prospect to be further studied and applied. Now let us take a look at the experiment outcomes.
</P><P>Click to jump to:<UL class="navlist"><LI style="width:24%"><A href="#Parts">Parts</A></LI><LI style="width:24%"><A href="#Demonstration">Demonstration</A></LI><LI style="width:24%"><A href="#Characterization_of_existing_parts">Characterization</A></LI><LI style="width:24%"><A href="#References">References</A></LI></UL></P><H2><SPAN class="mw-headline" id="Parts">Parts</SPAN></H2><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
Go to <A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2010&amp;group=SJTU-BioX-Shanghai" class="external text" rel="nofollow">Partsregistry</A> to see our submitted parts.
<H2><SPAN class="mw-headline" id="Demonstration">Demonstration</SPAN></H2><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
Click to jump to:<UL class="navlist"><LI style="width:48%;"><A href="#Eukaryotic_approach">Eukaryotic approach</A></LI><LI style="width:48%;"><A href="#Prokaryotic_approach">Prokaryotic approach</A></LI></UL><H3><SPAN class="mw-headline" id="Eukaryotic_approach">Eukaryotic approach</SPAN></H3><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
Click to jump to:<UL class="navlist"><LI style="width:19%;"><A href="#gdf5">I. Detector - GDF5</A></LI><LI style="width:19%;"><A href="#chr2">II. Supervisor - ChR2</A></LI><LI style="width:19%;"><A href="#mef2">II. Supervisor - MEF2-JeT</A></LI><LI style="width:19%;"><A href="#cre">II. Supervisor - CRE-JeT</A></LI><LI style="width:19%;"><A href="#oct4">III. Actuator - Oct4</A></LI></UL><H4><SPAN class="mw-headline" id="I._Detector">I. Detector</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV><UL><LI>Our Detector module relies on the GDF5 tissue-specific promoter to act. Here we demonstrated the specificity of GDF5 by ligating it with ChR2-eYFP and transfect it into in-differentiated mesenchymal stem cell line C3H10. The result (data not shown) shows almost no fluorescence in C3H10 cells, which means GDF5 has controlled the expression of ChR2-eYFP in non-chondrocytic C3H10 cells.
</LI></UL><UL><LI><B>Future: We plan to transfect GDF5 into ATDC5 chondrocytes and see if it can be expressed in large amounts.</B></LI></UL><H4><SPAN class="mw-headline" id="II._Supervisor_-_ChR2">II. Supervisor - ChR2</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
ChR2 is a widely used membrane-cross molecule in neuron-related experiments. When illuminated, it could be penetrated by many cations, such as sodium ion(Na+), calcium ion(Ca2+) and even some organic molecules which are smaller than the hole formed by ChR2. ChR2 is sensitive, especially to the light ranging in a particular bandwidth, usually between 450nm-470nm and this is the region for blue light.
<H5><SPAN class="mw-headline" id="Expression_of_ChR2">Expression of ChR2</SPAN></H5><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV><UL><LI>Idea: To detect whether ChR2 is correctively expressed cross the membrane of another kind of cell, especially in mammalian somatic cells, we could transfect 293T cells with pcDNA3.1 containing the gene encoding ChR2-eYFP. If it could be expressed as the theory told us, the fusion protein--ChR2 and eYFP, will be embedded in the membrane. 
</LI></UL><UL><LI>Method: We have prepared coverslips in the wells of cell culture plate before adding 293T cells. The medium is composed by DMEM, 10% FBS and 1% Penicillin/Streptomycin. We made transfection on the next day. After another 24h-culture, we picked up the coverslips out of the wells and washed them carefully by PBS. Then we treated them with 2-(4-Amidinophenyl)-6-indolecarbamidine dihydrochloride, which is known as DAPI dihydrochloride. This reagent could stain cell nuclear into blue, so it would be easy to observe with the help of microscope.
</LI></UL><UL><LI>Result: As we expected, we find the blue points which represent cell nuclears are surrounded by green outer covering under Laser Scanning Confocal Microscope (Leica), as shown below. After reading the paper published by Stanford on Nature Neuroscience in 2005 (Millisecond-timescale, genetically targeted optical control of neural activity), we could tell the green-labeled membrane has ChR2-eYFP embedded in. 
</LI></UL><UL><LI><B>Conclusion: From the photo, we can be sure that ChR2 is expressed in 293T cell and more importantly, it has its correct position in the membrane, which is an essential precondition for its function. At this point, we proved the structural correctness of ChR2.</B></LI></UL><H5><SPAN class="mw-headline" id="Function_of_ChR2_by_RT-PCR">Function of ChR2 by RT-PCR</SPAN></H5><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV><UL><LI>Idea: Since ChR2 could be expressed structural correctively in another kind of cell, we start to seek for the answer of whether it could work normally as in neurons. As ChR2 could initiate calcium influx, we could test its function by examining the transcription level of some certain genes which are sensitive to the change of Ca2+ concentration. Luckily, we found Zif268, a gene in Mouse Genome, and it would be up-regulated when meeting with an increase of Ca2+. On the other hand, we found discontinuous illumination is better for ChR2’s penetration for cations, while continuous illumination could cause inactivation of ChR2.
</LI></UL><UL><LI>Method: We cultured ECHO cells in 35mm-plates and the medium is composed by Ham’s/F12, 10% FBS, 1% Penicillin/Streptomycin and 0.25% Zeocin. After 24 hours, we transfected ECHO cells with pcDNA3.1, the same as above, containing ChR2-eYFP, and we wrapped the plates with foil. 24h after transfection, we carried the plates into darkroom for later experiments. We used a red light source to provide enough light for us to see and blue light pulses (around 460nm) are used to illuminate the cells. Since the blue light source is continuous, we got light pulses by pressing the switch button in a given rhythm. We tested different timescale to see the change of Zif268 transcription level. Besides the semi-quantitative measurement, we later led Realtime PCR to test quantitatively.
</LI></UL><UL><LI>Result: According to the different conditions of experimental and control groups, we could see that RNA extraction was successful. From the RT-PCR results, we know the cells transfected by ChR2-eYFP (pcDNA3.1) then illuminated for 30min showed the brightest band in Zif268 expression, followed by transfected/illuminated (20min), transfected/no illumination and no transfection/no illumination. While the bands of GAPDH showed no difference among them. The pictures are below.
</LI></UL><UL><LI><B>Conclusion: The ChR2 is able to function normally, causing the increase of the expression level of ZIF268 and ARC. But this method is not so precise; we'll need to use a more precise method to prove it.</B></LI></UL><H5><SPAN class="mw-headline" id="Function_by_Real-time_PCR">Function by Real-time PCR</SPAN></H5><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV><UL><LI>Idea: Although results from semi-quantitative PCR had primarily shown that lighting cells transfected by ChR2 could cause the ascend of the expression level of ZIF268 and ARC, which indicated that Ca<SUP>2+</SUP> has entered cells, quantitative methods like Real-Time PCR should be more preciese and convincing. Therefore, we conducted real-time PCR for three times to render the result more precise. Besides, in order to quantify the ability of ChR2 to incorporate Ca<SUP>2+</SUP> into cells, we refined the design of our experiment, which is shown below:
</LI></UL><UL><LI>Method: We design two sets of experiments: one is time related, in which the Ca2+ concentration is uniform (16 mM) while different groups are lighted for different time (0min, 5min, 10min, 15min and 30min) and there are two repeats for each group; the second one is Ca2+ related, in which all groups are lighted for 30 min while having different Ca2+ concentration (0mM, 4mM, 8mM, 16mM, 32mM and 64mM) and each group has three repeats. After lighting, RNA is extracted from the cells with the method mentioned before and then reverse-transcribed into cDNA by Promega’s reverse transcript kit. By adding equal quantity of cDNA into the real-time PCR system, we assume that our target gene, zif268 and ARC, if whose quantity is different in different groups, is cause by the different expression level in the cells, not by deviation from the system. The reagent for real-time PCR is SYBR@Green Reagent. 
</LI></UL><P>By analyzing the data exported by StepOneR Software (supported by Applied Biosystems), we calculate out the ??C<SUB>T</SUB> between gene ARC and GAPDH, as well as ZIF268 and GAPDH, and the average expression level gap between different group. The result is listed below:
</P><P>
For ZIF268, when comparing with the value of ??C<SUB>T</SUB> with 0 min’s light and 0 mM Ca2+, we find that the value is dropping when lighting time or Ca2+ concentration is increasing:
</P><P>
When the intracellular Ca2+ concentration increases, the expression level of ZIF268 would increase as well in neuron. But the real-time PCR result is not the same to our expectation. As the intracellular condition between neuron cell and C3H10 cell has many differences, we cannot assure that all necessary factors for the expression of ZIF268 also exist in C3H10. Thus, we cannot conclude that ChR2 could function as it does in neuron cell.
</P><P>When we detect the quantity of gene ARC, there are some positive results. With the increase of lighting time or Ca2+ concentration, the quantity of ARC increases as well:
</P><P>
This is a powerful evidence to prove that the expression level of ARC is related to the lighting time or Ca2+ concentration. As ARC is regulated by the concentration of Ca2+, this result indicates that when lighted, ChR2 could be permeable for Ca2+, which is what we expect. Besides, from the drafts above, we find that when the concentration of Ca2+ is about 16mM, the expression level of ARC reach to the top. In addition, this experiment also proves that the expression of ARC could also be regulated by Ca2+ in C3H10, just like in neuron cell. So the expression level of ARC could act as index of the intracellular Ca2+ concentration.
</P><UL><LI><B>Conclusion: ChR2 could be permeable to Ca2+ when lighted and ARC could be a index of the intracellular Ca2+ concentration in C3H10, as well as in neuron cells.</B></LI></UL><H4><SPAN class="mw-headline" id="II._Supervisor_-_MEF2-JeT_promoter">II. Supervisor - MEF2-JeT promoter</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
Our supervisor acts as the switch for the genetic circuit via calcium-depedent intracellular signal transduction pathways. At the ends of these pathways lie two most important regulatory parts: MEF2 and CRE enhancer sequence. We then built artificial promoters by ligating MEF2 and CRE binding sequences with another artificial basal promoter JeT.
<UL><LI>Idea: In order to test whether MEF2-Jet could function as a good promoter, we set two groups, experimental and control, to verify it. We reconstructed pcDNA3.1 by connecting MEF2-Jet with the gene encoding luciferase. So, if MEF2-Jet could work normally, the expression of luciferase will gain a huge increase than that with no promoter in its upstream. 
</LI></UL><UL><LI>Method: Cultured ECHO and C3H10 cells in 24-well plate for 24h (medium for C3H10: DMEM, 10% FBS and 1% Penicillin/Streptomycin). Experimental groups were transfected with pcDNA3.1 containing MEF2-Jet-luciferase, while control groups were transfected with pGL3-Basic which contains no promoter. After another 24h, we conducted a dual-luciferase assay to measure the expression of luciferase.
</LI></UL><UL><LI>Note: We have searched related information and known the calcium concentration of the medium for C3H10 is 2mmol/L, because of the calcium ions got from DMEM and FBS.
</LI></UL><UL><LI>Result: As we expected, expression level of luciferase in experimental group transfected with MEF2-Jet-luciferase largely outgrows its counterpart with pGL3-Basic. The picture below well depicts the function of MEF2-Jet as a good promoter. However, by comparing the experimental groups varying in Ionomycin concentration, we get no favorable evidence in explaining calcium ions could regulate the enhancer-promoter. In theory, after adding 5μM Ionomycin, it would have a significant intracellular increase of Ca2+, which would show a great up-regulated expression of luciferase.
</LI></UL><UL><LI><B>Conclusion: In order to explain the result, we searched for more research and studies and found out that there is calcium in the cell culture (proximally 2mM) and that concentration might be enough to start transcription. Then we come up with the idea to reduce the calcium concentration using EGTA (ethylene glycol tetraacetic acid) a polyamino carboxylic acid, a chelating agent that is related to the better known EDTA, but with a much higher affinity for calcium. The result will be shown below.</B></LI></UL><P>2. Promoter functions without Ca2+ from medium
</P><UL><LI>Idea: As the medium we use to culture C3H10 is DMEM mixed with 10% FBS, the Ca2+ concentration in this medium is about 2mM, which is not negligible in our experiment. In order to make clear whether the Ca2+ in the medium could activate the MEF-Jet circuit or not, we need to eliminate all dissolved Ca2+ cation and then measure the activity of luciferase.
</LI></UL><UL><LI>Method: EGTA(ethylene glycol tetraacetic acid) has a high affinity for calcium so we use it as a kind of calcium chelator to eliminate the Ca2+ in the medium. In our experiment, EGTA was added into medium 8h after transfection with MEF-Jet. The experimental groups was added EGTA while the control ones not. Dual-Luciferase assay was conducted 24h after transfection. 
</LI></UL><UL><LI>Result: After treated with EGTA, the expression activity of luciferase shows slightly increase with the increase of the concentration of Ionomycin (shown in figure below column marked A, B and C). However, the increase of expression level is not as high as we have expected because column a, who is supposed to have no luciferase expression showed high level fluorescence.
</LI></UL><UL><LI><B>Conclusion: To explain the result, we need to look carefully into the mechanism of MEF2 enhancer. As is explained in the project page, MEF2 enhancer who recruits series of transcription factors regulates the activity of transcription mainly by epgenetically process (MEF2 recruits HDACs to down regulate transcription and p300 HAT to up regulate it). However, in our experiments, we use pcDNA as our vector, which will not be integrated into the cell chromosome and the system it contains will not be modified by nucleosome. Therefore, the transcription activity has less relationship with the stimuli of ionomycin. Moreover, as a synthetic core promoter, JeT promoter has relatively high level of transcription activity, that result in the luciferase expression when there is no calcium (column a, with EGTA added and no ionomycin introduced).</B></LI></UL><UL><LI><B>This problem will be solved if we use viral vectors instead of pcDNA, however, viral transfection and cell culture methods takes much more time than our current method and we did not have time to finish these experiments before the deadline when wiki freezes.</B></LI></UL><P>3. Could ChR2 and MEF-JeT work together as expected?
</P><UL><LI>Idea: In order to make our circuit sense, one of the challenges in front of us is to make sure that ChR2 and MEF-Jet could work together, which means when lighted, Ca2+ pumped into the cell could turn on or up the MEF-Jet and when no light, turn off or down. ChR2 and MEF-Jet was transfected into the cell simultaneously.
</LI></UL><UL><LI>Method: In this experiment we should transfect ChR2, MEF-Jet and RL-TK in one time and cells were incubated into 24-wells plate. 24h after transfection, cells as experimental groups were put under pulse blue light to activate ChR2 while control ones were not taken out from dark environment until lysis. Dual-luciferase assay was conducted as normal.
</LI></UL><UL><LI>Result: The figure below shows that illumination has in fact induced transcription initiation, and the result in more favorable in ECHO cells than in C3H10 cells. There are still problems that the increase in expression level is not as dramatically as we expected and the expression of luciferase started even if there is no light.
</LI></UL><UL><LI>Explanations: The SJTU-BioX-Shanghai team has got two reasons for these situations: 1. Just like the reason in the front part, the high level of expression when there is no light the result of our use of pcDNA as a vector. And MEF2 enhancer lost its epigenetical function; 2. There is no EGTA used in this experiment (because we need to have enough Ca2+ outside the cell membrane so that ChR2 could function normally), and Ca2+ in the cell culture might initiate transcription automatically.
</LI></UL><UL><LI><B>Since the MEF2 results are not as we exprected, we proceed to test the CRE-JeT promoter, which acts via a different signal pathway.</B></LI></UL><H4><SPAN class="mw-headline" id="CRE-JeT_promoter">CRE-JeT promoter</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
The thought and method for the test of CRE-Jet part is similar to that of MEF2-Jet. The only difference between them is that in this test the enhancer before Jet is replaced by CRE, so we would turn to results directly.
<P>1. Work as a good promoter
</P><UL><LI>Result: As we expected, expression level of luciferase in experimental group transfected with CRE-Jet-luciferase largely outgrows its counterpart with pGL3-Basic. The picture below well depicts the function of CRE-Jet as a good promoter. However, by comparing the experimental groups varying in Ionomycin concentration, we get no favorable evidence in explaining calcium ions could regulate the enhancer-promoter. In theory, after adding 5μM Ionomycin, it would have a significant intracellular increase of Ca2+, which would show a great up-regulated expression of luciferase.
</LI></UL><UL><LI><B>Explanation: In order to explain the result, we searched for more research and studies and find out that there is calcium in the cell culture (proximally 2mM) and that concentration might be enough to start transcription. Then we come up with the idea to reduce the calcium concentration using EGTA (ethylene glycol tetraacetic acid) a polyamino carboxylic acid, a chelating agent that is related to the better known EDTA, but with a much higher affinity for calcium. The result will be shown below.</B></LI></UL><P>2. Promoter functions without Ca2+ from medium
</P><UL><LI>Result: The figure below strongly demonstrated that CRE-JeT promoter has dramatically promotes the level of expression with the stimuli of calcium/Ionomycin, compared with the control group (whose Ca2+ concentration is reduced to almost zero with the help of EGTA).
</LI></UL><UL><LI><B>While there is still the problem that the expression level of luciferase when there is no calcium is relatively high. Similarly, we tried to figure out the reason for this phenomenon by referring to the mechanism of the CRE enhancer. As is shown in the project page, there is no research showing that the CRE enhancer has the function of blocking transcription. And the activation function is based on epigenetical mechanism as well. In that case, the unexpected transcription initiation is the result of JeT core promoter.</B></LI></UL><P>3. Could ChR2 and CRE-Jet work together as expected?
</P><UL><LI>Result: Unfortunately, there is no positive result in this section because instead of increasing, the fluorescent value decreased slightly after illumination treatment.
</LI></UL><P>In the beginning, we thought this result is hard to explain because the previous experiment has suggested that CRE-JeT promoter be sensitive to Ca2+ stimuli. However, after close look at the mechanism of Ras-MAPK signaling pathway, we figured out that this result is explainable and reliable.
</P><UL><LI><B>Future: Research by Professor Grigory Krapivinsky suggested that NMDAR (another Ca2+ channel) is coupled to ERK-MAPK pathway by interaction with Ras-GRF1. And further studies suggested that concentration increased as a result of Ca2+ influx through ion channels is localized. Researchers believed that this localized increase will have little effect on cellular MAPK pathways unless Ras-GRF1 appeared in the nearby region (such as binding to specific region of NMDAR). Therefore, without the help of Ras-GRF1, the ChR2 cannot activate the signaling pathway. While Ionomycin works as a transporter which increase calcium level in the whole cell rather than in local regions. As a result, when induced with Ionomycin in the previous experiment, the expression level does increase.</B></LI></UL><UL><LI><B>To solve this problem, we have designed fusion protein with ChR2 and Ras-GRF1. It could restrict Ras-GRF1 in the region where Ca2+ influxes through ChR2. However, the result of this experiment is not shown hear.</B></LI></UL><H4><SPAN class="mw-headline" id="III._Actuator">III. Actuator</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
The Actuator module of our design features Oct4 as well as col2 gene. Since time is limited for us, we checked the expression of Oct4 gene.
<P>Below is the western-blot image we took in the experiment to check whether our cloned Oct4-myc (with myc tag attached to the peptide chain) can be expressed correctly. You can clearly see the band of Oct4 at about 39kD, which is exactly the molecular mass of Oct4-myc.
</P><P>This photo indicates that our main part of Actuator, the Oct4 protein, is expressed successfully in ATDC5 cell line.
</P><H3><SPAN class="mw-headline" id="Prokaryotic_approach">Prokaryotic approach</SPAN></H3><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV><H4><SPAN class="mw-headline" id="PfdhF.2C_the_hypoxia-inducible_promoter">PfdhF, the hypoxia-inducible promoter</SPAN></H4><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV><A href="https://igem.org/User:biohyp" class="external text" rel="nofollow">Yupeng He</A> built and tested the hypoxia-inducible promoter, fdhF promoter (PfdfF, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K387003" class="external text" rel="nofollow">Part:BBa_K387003</A>). In this test experiment, we expected to see that, at the same OD<SUB>600</SUB>, the activity of PfdhF is higher in hypoxia than in normal oxygen pressure (20%). At the beginning, both the hypoxia group and control (20% oxygen partial pressure, i.e. in air) group were incubated with 200 microlitre suspension of bacteria with the testing part (PfdhF followed by firefly luciferase). Then, we sampled both group and measured OD<SUB>600</SUB> and value of luciferase every 30 minutes (control group) or 1 hour (hypoxia). The figure was the result of this test. Our result showed that this part worked as we expected: under hypoxia, the efficiency of this promoter was much higher than that under the condition of 20% oxygen at the same OD<SUB>600</SUB>. (One thing that needs to be noticed is that at 0.4 OD<SUB>600</SUB>, the bacteria are in log phase and the PfdhF begins to work.)
<P>Then we wanted to know the difference of PfdhF activity under these two conditions. So, we plotted another figure with OD<SUB>600</SUB> as x axis and the ratio of PfdhF activity under hypoxia to that under normal oxygen pressure as y axis. This curve showed that, during and after the log phase (OD<SUB>600</SUB>&gt;0.4), the activity of PfdhF is over 1.5 times more under hypoxia than under normal 20% oxygen pressure. Particularly, this ratio is over 3 when OD<SUB>600</SUB>&gt;0.5. Furthermore, in the normal 20% oxygen group, as the OD<SUB>600</SUB> increases, the oxygen supplies may be insufficient for <I>E. coli</I> which may lead to the increase of PfdhF activity. In this case, the activity of PfdhF under normal condition is probably overestimated. With all these data and analysis, it is evident that PfdhF is a rather good hypoxia-inducible promoter with much higher activity under hypoxia than under 20% oxygen pressure.
</P><H2><SPAN class="mw-headline" id="Characterization_of_existing_parts">Characterization of existing parts</SPAN></H2><H3><SPAN class="mw-headline" id="RNase_barnase">RNase barnase</SPAN></H3><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV>
Due to our design containing the RNase barnase as part of our prokaryotic Supervisor, we decided to go on a journey to characterize the <A href="http://partsregistry.org/Part:BBa_I716211" class="external text" rel="nofollow">part</A>.
However since we are still processing our data, we cannot list our experiment results here on the wiki before it freezes but we will place our results and analysis on <A href="http://partsregistry.org/Part:BBa_I716211" class="external text" rel="nofollow">Partsregistry</A> as soon as we finish them.
<P>Thank you for understanding us! :)
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><DIV class="gotop"><A href="#topofpage">Back to top ↑</A></DIV><UL><LI>[1] Mary B. Goldring and Steven R. Goldring, Osteoarthritis. Journal of Cellular Physiology Cellular Physiology, 2007, 626-634 
</LI></UL><UL><LI>[2]Yoshinari Miyamoto, Akihiko Mabuchi et al. A functional polymorphism in the 5' UTR of GDF5 is associated with susceptibility to osteoarthritis. NATURE GENETICS. 25 March 2007 
</LI></UL><UL><LI>[3] Jun O. Liu et al. Cabin1 Represses MEF2-Dependent Nur77 Expression and T Cell Apoptosis by Controlling Association of Histone Deacetylases and Acetylases with MEF2. Immunity. Vol. 13, 85–94, July, 2000 
</LI></UL><UL><LI>[4] Grigory Krapivinsky, Luba Krapivinsky, Yunona Manasian. et al. The NMDA Receptor Is Coupled to the ERK Pathway by a Direct Interaction between NR2B and RasGRF1. Neuron. Vol. 40, 775–784, November 13, 2003 
</LI></UL><UL><LI>[5] Charles L. Farnsworth. et al. Calcium activation of Ras mediated by neuronal exchange factor Ras-GRF. NATURE. Aug 10, 1995 
</LI></UL><UL><LI>[6] Adam J. Shaywitz, Michael E. Greenberg CREB: A STIMULUS-INDUCED TRANSCRIPTION FACTOR ACTIVATED BY A DIVERSE ARRAY OF EXTRACELLULAR SIGNALS. ANNU. REV. BIOCHEM. 1999
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:SJTU-BioX-Shanghai/result">http://2010.igem.org/Team:SJTU-BioX-Shanghai/result</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SJTU-BioX-Shanghai/result" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SJTU-BioX-Shanghai/result" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/result&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SJTU-BioX-Shanghai/result&amp;oldid=208501" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>