<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dalhousie_test12 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dalhousie/test12</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><A href="/wiki/index.php?title=Template:DalhousieNavBar&amp;action=edit&amp;redlink=1" class="new" title="Template:DalhousieNavBar (page does not exist)">Template:DalhousieNavBar</A><A href="/wiki/index.php?title=Template:Team:Dalhousie_Halifax_NS/footer&amp;action=edit&amp;redlink=1" class="new" title="Template:Team:Dalhousie Halifax NS/footer (page does not exist)">Template:Team:Dalhousie Halifax NS/footer</A></P><DIV class="jumbotron"><H1 id="title"> Experiments </H1></DIV><DIV class="container centered"><DIV class="row"><H3 id="FecalSamples" style="color:#674A8A;"> High-Throughput Sequencing </H3></DIV><DIV class="container"><DIV class="col-md-4"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/d/d6/T--Dalhousie_Halifax_NS--PowerFecalProtocol.pdf"> Fecal Sample DNA Extraction </A></H4></DIV><P>To extract environmental DNA from fecal samples. This extraction will purify genomic DNA from prokaryotic and eukaryotic cells found in the fecal sample. 
</P></DIV><DIV class="col-md-4"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/7/70/T--Dalhousie_Halifax_NS--PowerFecalConcentration.pdf"> DNA Concentration</A></H4></DIV><P>Concentration protocol to concentrate DNA isolating using the PowerFecal Extraction kit.</P></DIV><DIV class="col-md-4"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/a/ae/T--Dalhousie_Halifax_NS--16IlluminaSeq.pdf"> 16S Illumina Miseq Sequencing </A></H4></DIV><P>Illumina MiSeq Sequencing was used to identify the sequences of the 16S rRNA genes in the fecal samples. This step was undertaking by the Integrated Microbiome Resource at Dal. There protocol was provided to us.</P></DIV><DIV class="col-md-4"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/4/46/T--Dalhousie_Halifax_NS--MetagenomicSequencingIllumina.pdf"> Metagenomic Shotgun Illumina Miseq Sequencing </A></H4></DIV><P>This protocol was used by the Integrated Microbiome Resource to sequence the metagenomic DNA in our fecal samples from the porcupine, beaver, arctic wolf and coyote.
</P></DIV><DIV class="col-md-4"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/d/d1/T--Dalhousie_Halifax_NS--16SSequencingAnalysis.pdf"> 16S Sequencing Analysis 
</A></H4></DIV><P>This protocol was used for analyzing the 16S Illumina Sequencing data from the illumina sequencer. Provided by the Integrated Microbiome Resource at Dalhousie University.</P></DIV><DIV class="col-md-4"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/a/a8/T--Dalhousie_Halifax_NS--MetagenomicSequencingAnalysis.pdf"> Metagenomic Sequencing Analysis 
</A></H4></DIV><P>This protocol was used for analyzing the Metagenomic Illumina Sequencing data from the illumina sequencer. Provided by the Integrated Microbiome Resource at Dalhousie University.</P></DIV></DIV></DIV><DIV class="row"><H3 id="FecalSamples" style="color:#4A8A87;">Isolation of Bacteria</H3></DIV><DIV class="container"><DIV class="col-md-4"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/1/19/T--Dalhousie_Halifax_NS--CelluloseMediaProtocol.pdf"> Preparation of Cellulose Media </A></H4></DIV><P>Congo Red Cellulose media can be used to isolate bacteria that degrade cellulose. It does this because the only carbon source in the media is cellulose. It uses Whatman Paper and gelatin.</P></DIV><DIV class="col-md-4"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/c/c5/T--Dalhousie_Halifax_NS--ColonyPCRProtocol.pdf"> 16S Colony PCR </A></H4></DIV><P>Colony PCR has many applications including; verifying that an insert has been successfully incorporated in plasmid constructs, insert orientation, and species determination of unknown samples. The later was the reason for our use of Colony PCR. After streaking fecal sample onto cellulose plates, we picked colonies of different morphologies to find out which species or genus they were. We did this by using primers that targeted conserved sequence found in bacteria so we could amplify the DNA of almost all bacteria. </P></DIV><DIV class="col-md-4"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/6/69/T--Dalhousie_Halifax_NS--ExoSapItProtocol.pdf"> ExoSapIt and DNA Cleanup </A></H4></DIV><P>Instead of PCR clean up, we used ExoSap It. ExoSap It allows for an enzymatic clean-up of left over dNTPs and single stranded DNA. It is a very quick procedure as the only activation ExoSap It needs is a 15 minute 37°C incubation and a 15 minute 80 °C inactivation.</P></DIV></DIV></DIV><DIV class="row"><H3 id="FecalSamples" style="color:#4A4D8A;">Metagenomic Library and Transformation</H3></DIV><DIV class="container"><DIV class="col-md-6"><DIV class="row"><H4 style="text-align:center;"><A href="https://static.igem.org/mediawiki/2016/c/c3/T--Dalhousie_Halifax_NS--MetagenomicLibraryProtocol.pdf" style="text-align:center;"> Metagenomic Library Construction </A></H4></DIV><P style="text-align:center;">Protocol that takes extracted fecal DNA, prepares it for ligation and clones it into a cosmid. This cosmid can be packaged into phage particles and those can be used to place the cosmid into <EM>E. coli</EM> cells which will express environmental DNA that had been placed into the cosmid.</P></DIV><DIV class="col-md-6"><DIV class="row"><H4 style="text-align:center;"><A href="https://static.igem.org/mediawiki/2016/0/02/T--Dalhousie_Halifax_NS--Transformation.pdf">Heat-Shock Transformation</A></H4></DIV><P style="text-align:center;">Protocol for the heat-shock transformation of plasmids into chemically competent <EM>E. coli</EM> cells.</P></DIV></DIV><DIV class="row"><H3 id="FecalSamples" style="color:#4D8A4A;">Chemical Analysis</H3></DIV><DIV class="container"><DIV class="col-md-6"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/7/7a/T--Dalhousie_Halifax_NS--steamdistillationprotocol.pdf"> Steam Distillation </A></H4></DIV><P>Steam Distillation was used to separate the terpenes from pine tree oleoresin without degrading them. Through GC/MS analysis we confirmed that this procedure worked.</P></DIV><DIV class="col-md-6"><DIV class="row"><H4><A href="https://static.igem.org/mediawiki/2016/f/f1/T--Dalhousie_Halifax_NS--terpeneinhibitionprotocol.pdf"> Terpene Inhibition Experiments </A></H4></DIV><P>These experiments were used to test <EM>E. coli</EM> and <EM>S. cerevisiae</EM> tolerance to terpenes, both in pure form and the terpenes isolated from oleoresin.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>