Team:Dalhousie/Description
Home
Project
Description
Design
Requirements
Results
Interlab
Results
Team
Team
Collaborations
Attributions
Sponsors
Human Practices
Summary
Science Communication
Integrated and Gold
Public Engagement
Social Media
Lab
Parts
Safety
Experiments
Notebook
Improve
Contact us today
Project Description
WHY
As fossil fuels continue to run out across the globe, many people are looking towards alternative sources of energy that are renewable and sustainable. One of these options is biofuel, fuel made from organic matter. Biofuel is most commonly made from ethanol, or bioethanol, which can be used as a fuel for vehicles in its pure form. In the field of biofuel production, bioethanol made from cellulose continues to be the dominant form; however, harsh methods are required to be able to extract the sugars from cellulose and convert it to ethanol. In Atlantic Canada, the main export is wood, pulp, and paper. The processes of these mills have been shown to be environmentally unsound with large quantities of hazardous waste being expelled. As many of our members have seen first-hand, growing up near these pulp and paper mills. Along with hazardous waste practices, the extraction of usable materials from wood is fairly inefficient. These mills leave behind a great deal of wood waste that could be used for biofuel production if broken down and converted to ethanol. The only question is: how?
WHAT
The Dalhousie iGEM team this year is focused on using the microbiome of the porcupine to solve this conversion of wood waste to ethanol. A large part of the porcupine’s diet is made up of bark. Unable to digest the cellulose, hemi-cellulose, and lignin in the bark, the gut bacteria of the porcupine do the work instead. We hypothesized that the microbiome of the porcupine would contain enzymes that convert cellulose, hemi-cellulose, and lignin to glucose; a usable sugar. If true, we would be able to mine metagenomic sequencing for these enzymes, clone them into E.coli, creating a system that could convert cellulose to glucose. Finally, we would use this expression system in a bioreactor containing both E. coli and yeast to create ethanol from wood waste. The yeast being an integral part of the system to ferment the glucose created by the E. coli to ethanol, our biofuel.
HOW WE DID IT
Dry Lab This year’s team extended last year’s project to find a higher quantity of DNA sequences and key enzymes within the cellulose and hemicellulose degradation pathway. The team focused on bioinformatics, more specifically a metagenomic pipeline and metagenomic library. The metagenomic library was produced through the collaboration with Dr. Trevor Charles at the University of Waterloo. Along with the production of the metagenomic pipeline and library, the team planned to co-culture cellulose-degrading E. coli and yeast in a bioreactor, in order to produce ethanol from cellulose. For the synthetic metagenomic library, the team used the Illumina MiSeq data from last year’s four fecal samples: Artic Wolf, Coyote, Porcupine, and Beaver. The porcupine Illumina MiSeq data was used to discover enzymes within the cellulose and hemicellulose degradation pathways. The process of shotgun sequencing, as seen in Illumina MiSeq technology, required short inputs of DNA that were ~300 base pairs. The length of base pairs required for a full gene is much longer. Thus, the team used another program, MegaHIT, which allowed them to stitch together sequences to get a fragment yield of 1,000 base pairs. Once the larger fragments were formed the team used the program, Prodigal, to identify the open reading frames (ORFs). Prodigal is responsible for locating ribosome binding sites, through the identification of the start and stop codons. Lastly, jackHMMR was used to find protein domains, with respect to the predicted function.
Wet Lab The team focused on beta-glucosidase, endoglucanase, and beta-xylanase for the cellulose and hemicellulose degradation pathway. The genes were optimized for expression within E. coli, and once all were modified, the team submitted them for synthesis at Integrated DNA Technologies (IDT). Each were cloned into the pET26b(+) expression vector system. pET26b(+) encodes a pelB sequence at the N-terminus of the protein of interest which is responsible for localization of the protein of interest to the periplasm. From the periplasm, soluble proteins are able to diffuse into the surrounding environment or are secreted. Once successfully cloned, the team aimed to characterize beta-xylanase. Beta-xylanase, is a key enzyme in the hemicellulose degradation pathway that cleaves xylose dimers to usable xylose monomers. Using a Coomassie Blue stain for total protein expression, the team was able to show that this novel enzyme is able to be expressed in E. coli in the pET26b(+) expression vector system. The team took a step further and attempted to assess the enzymatic activity of this novel beta-xylanase using a modified version of a cellulase/xylanase activity fluorophore assay (Chen et al., 2016). Using xylobiose conjugated to a fluoro-active molecule, we were able to determine the relative activity of our novel beta-xylanase relative to pet26b(+) alone. We discovered that under the conditions we provided the enzyme was not able to function properly (ie. cleave the substrate), possibly due to the differences in oxidation states of the gut and open air or due to pH differences from the protein’s natural environment.
References Bolger, AM., Lohse, M, and Usadel, B. 2014. Trimmomatic: a flexible trimmer for Illumina sequence data. Int Soc Comp Bi. 30(15): 2114-2120. Chen, H. M., Armstrong, Z., Hallam, S. J., & Withers, S. G. (2016). Synthesis and evaluation of a series of 6-chloro-4-methylumbelliferyl glycosides as fluorogenic reagents for screening metagenomic libraries for glycosidase activity. Carbohydr. Res. 421, 33-39. EMBL-EBI. 2017. HMMER Biosequence analysis using profile hidden Markov Models [internet]. Cambridgeshire (UK): European Molecular Biology Laboratory. Available from: https://www.ebi.ac.uk/Tools/hmmer/search/jackhmmer Houghton, J., Weatherwax, S., & Ferrell, J. (2006). Breaking the biological barriers to cellulosic ethanol: a joint research agenda (No. DOE/SC--0095). EERE Publication and Product Library. Kopylova, E, Noé L, and Touzet H. 2012. SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data. Int Soc Comp Bi. 28(24), 3211-3217. Sockolosky, J. T., & Szoka, F. C. 2013. Periplasmic production via the pET expression system of soluble, bioactive human growth hormone. Protein Expr Purif. 87(2), 129-135.
