<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Dalhousie_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Dalhousie/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="nav" class="noBackground" style="width:100%; z-index:100;"><DIV class="inner"><DIV id="navWrap"><NAV><DIV class="container-fluid"><UL class="nav navbar-nav navbar-center"><LI class="active"><A href="https://2017.igem.org/Team:Dalhousie" style=" color: white;">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#" style=" color: white;">Project
        </A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Requirements">Requirements</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#" style=" color: white;">Results
        </A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Results">Results</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#" style=" color: white;">Team
        </A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Sponsors">Sponsors</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#" style=" color: white;">Human Practices
        </A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Human_Practices">Summary</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/HP/Silver">Science Communication</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/SocialMedia">Social Media</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#" style=" color: white;">Lab
        </A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Dalhousie/Improve">Improve</A></LI><UL class="nav navbar-nav navbar-right"><DIV class="navbar-header"><A class="navbar-brand" style=" color: white;">Contact us today </A></DIV></UL></DIV></NAV></DIV></DIV></DIV><DIV class="panel" id="panel1"><DIV class="inner"><DIV class="top"><DIV class="title">Results</DIV></DIV></DIV></DIV><DIV class="panel" id="panel2"><DIV class="inner"><H1><FONT color="#C1D35D">Results</FONT></H1><FONT color="#C1D35D"><H2>Metagenomic Library</H2></FONT><FONT color="#C1D35D">DNA Extraction</FONT>

We were able to confirm the size of the metagenomic DNA isolated from the porcupine fecal samples via pulse field gel electrophoresis (PFGE) (Fig.1). The remainder of the DNA was ran via PFGE and all DNA larger than 24.8 kB was excised without exposure to UV or ethidium bromide. The gel was stained after band excision for visualization (Fig. 2).<DIV><DIV class="col-md-6"><CENTER><FIGURE><FIGCAPTION>Fig1. DNA extracted from the porcupine microbiome.</FIGCAPTION></FIGURE></CENTER></DIV><DIV class="col-md-6"><CENTER><FIGURE><FIGCAPTION>Fig2. DNA &gt;24 kB excised from a PFGE gel.</FIGCAPTION></FIGURE></CENTER></DIV></DIV><FONT color="#C1D35D">Vector Preparation</FONT>

pJC8 (Fig. 3) was digested with <I>Pmi</I>I and ran on a 0.8% agarose gel. The ladder was stained, and the proper band was marked with a razor. The gel was reassembled and the proper band (11kb) was cut out. The rest of the gel was stained with ethidium bromide and visualized via UV (Fig. 4). 
<DIV><DIV class="col-md-6"><CENTER><FIGURE><FIGCAPTION>Fig3. The pJC8 cosmid map. Based off the map described previously(Neufeld et al., 2011).</FIGCAPTION></FIGURE></CENTER></DIV><DIV class="col-md-6"><CENTER><FIGURE style="PADDING-BOTTOM: 50px"><FIGCAPTION>Fig4. 11 kB pJC8 digested with <I>Pmi</I>I excised from a 0.8% agarose gel.</FIGCAPTION></FIGURE></CENTER></DIV></DIV>

Ligation efficiency of the vector was tested using 3 separated reactions: 
<OL><LI>pJC8 + T4 ligase buffer</LI><LI>pJC8+ T4 ligase + ligase buffer</LI><LI>pJC8 +T4 ligase + ligase buffer + PNK</LI></OL>

These ligation reactions were then transformed into STBL3 <I>E. coli</I> and plated on tetracycline plates.<I>Pmi</I>I is a restriction enzyme that leaves blunt ends, so theoretically pJC8 should be able to re-ligate due to the lack of 5’ phosphates. By adding in PNK, you add the 5’ phosphate groups back allowing for re-ligation. A good result would be low to no colonies on plates 1 &amp; 2, and many colonies on plate 3. 

Our transformation plates reflected that we had excellent ligation efficiency with pJC8 (Fig. 5).<DIV><CENTER><FIGURE><FIGCAPTION>Fig5. Ligation control reactions for pJC8.</FIGCAPTION></FIGURE></CENTER></DIV><FONT color="#C1D35D">Trouble-Shooting</FONT>
After packaging attempts failed three times, we set up our last packaging extract using the control bacteria host <I>E.coli</I> VSC257 and using the lambda phage DNA as the insert in the phage packaging reaction, both supplied by Agilent Gigapack XL III packaging extract.
Unfortunately, no lambda plaques formed at the suggested dilution of 10<SUP>-4</SUP>. This suggests there is an issue with the packaging extract.
After purchasing new packaging extract, a control packaging reaction was set up. This time with the recommended 10<SUP>-4</SUP> dilution of phage as well as 10<SUP>-2</SUP> and 10<SUP>0</SUP> dilutions. Unfortunately, no plaques were seen at the 10<SUP>-4</SUP> and 10<SUP>-2</SUP> dilutions, however 10 plaques were counted on the 10<SUP>0</SUP>. This was a slight success; however it is not efficient enough to go forward with our experiment.
During trouble-shooting the packaging extract, we ran our insert DNA on a 0.8% agarose gel to ensure that all the high molecular weight DNA was still intact. If intact, we would expect it to stay in the well of the 0.8% agarose gel. The result of this gel electrophoresis was that the DNA migrated down the length of the gel suggesting it was sheared and that no high molecular weight DNA remained (Fig. 6). <DIV><CENTER><FIGURE><FIGCAPTION>Fig6. DNA extracted from porcupine microbiome after blunt-end repair and DNA clean-up.</FIGCAPTION></FIGURE></CENTER></DIV><FONT color="#C1D35D">Future Directions of the Cosmid Library</FONT>

Since our troubleshooting procedures showed that both the packaging extract and the insert DNA was faulty, we have many ways to try and fix the problems in the future.<UL type="1"><UL>Packaging Issue

<OL type="a"><LI>The aliquot of packaging extract used out of the newly purchased pack had the sample all along the side of the tube. This suggests that the sample may have thawed during transit. As soon as the packaging extract thaws, the constituents of the phage begin to stick together, like magnets. It would make sense that efficiency would be much lower in a thawed sample as most of the phage would already have formed capsids, preventing accumulation of foreign DNA and functional phage. </LI><LI>Another possibility is the technique used to add the DNA to the packaging extract was imperfect. The Charles lab emphasized that this was a critical aspect of the experiment, so we were sure to take good notes. However it is possible we are missing something and could contact the company for advice if needed.</LI><UL>DNA Issue
<OL type="a"><LI>It is imperative that throughout the process we are handling the DNA with wide-mouth pipet tips. It is probably for that reason that our DNA sheared. It is also possible that an accidental vortex or vigorous shake could have randomly sheared the large and fragile DNA</LI><UL>Other Issues
<OL type="a"><LI>When dephosphorylating the vector pJC8 opt for Shrimp Alkaline Phosphatase instead of Calf Intestinal Phosphatase (CIP). Or if using CIP, do a final phenol extraction before moving forward. CIP is extremely sticky, especially for blunt ended DNA, so it’s possible that it hangs on and prevent ligation of pJC8 with the insert DNA from the porcupine microbiome.</LI><LI>The regular amount of ATP present in the NEB T4 ligation kit can inhibit proper blunt-end ligation. In the future, we plan to add our own amount of ATP (5mM) and DTT to Invitrogen’s “ReactOne” buffer which is the exact recipe as the T4 ligase buffer without the DTT and ATP.</LI><DIV id="this"><FONT color="#C1D35D"><H2>Sequencing Metagenomics and Cloning</H2></FONT><FONT color="#C1D35D"><H3>Beta-Xylanase</H3></FONT></DIV>
Beta-xylanase was synthesized <I>de novo</I> via our synthetic metagenomic pipeline, and was cloned into the pET26b(+) expression vector system, under control of the T7 promoter. The construct was then transformed into chemically competent BL21 DE3 <I>E. coli</I> and grown on LB+kanamycin media. After successful transformation, a single colony was grown in 5ml culture until an OD600 of 0.6, at which time 0.1 M IPTG was (or wasn't) added 1:1000 for a final concentration of 0.1 mM. Induction with IPTG continued for 4 hours at which point the bacteria were lysed, and proteins collected. This process was repeated for BL21 DE3 <I>E. coli</I> containing empty vector (pET26b+). Proteins were run on a 12% SDS-PAGE gel. After running, the gel was stained with Coomassie Brilliant Blue.

A protein around 50 kDa is over-expressed in cells exposed to IPTG with the construct containing beta-xylanase(Fig. 7). Since this protein is not present without induction via IPTG or without the insert, and as our protein's predicted molecular weight is ~51 kDa we conclude that this over-expressed protein is our novel beta-xylanase. This assay proves that our protein is able to be expressed via IPTG induction in BL21 DE3 <I>E. coli</I> and is not toxic to the growing bacteria, as we expected.<DIV><CENTER><FIGURE><FIGCAPTION>Fig7. Coomassie Brillant Blue stain of SDS-PAGE. Red arrow indicates over-expressed protein of interest.</FIGCAPTION></FIGURE></CENTER></DIV><FONT color="#C1D35D">Functional Assay</FONT>
	After proving that the protein is able to be expressed in <I>E. coli</I> as expected, we took a step further and sought to characterize the enzymatic activity of the protein. As before, BL21 DE3 <I>E. coli</I> were grown to an OD600 of 0.6 at which point 50 uL of bacteria were added to an opaque-walled 96 well plate. Fifty microlitres of a substrate mixture containing 200 uM of the substrate (xylobiose conjugated to fluorophore) dissolved in a 1% Triton-X100/Potassium Acetate pH 7 buffer solution was added to each well of the bacterial solution. The plate was incubated at 37 degrees Celsius for 18 hours, shaking. At 18 hours, fluorescence measurements (excitation: 365 nm, emission: 450 nm, gain: 65) were taken and relative light units for each sample were determined. Each sample was corrected to background (LB broth alone) and an unconjugated fluorophore was used as a positive control.

Our protein was not able to cleave the substrate to release the fluorophore under these experimental conditions (Fig.8). Two possible explanations for this phenomenon could be that the protein is designed to be expressed under anaerobic conditions (such as the gut of a porcupine) and thus misfolding or damage caused by oxidation caused the protein to be non-functional in an aerobic environment. Secondly, this non-functionality could be because the protein is designed to function at a pH higher or lower than pH 7, as the pH in the gut of the porcupine would vary depending on the location of the microbe from which this protein stems.<DIV><CENTER><FIGURE><FIGCAPTION>Fig8. Analysis of enzymatic activity via a fluorophore cleavage assay.</FIGCAPTION></FIGURE></CENTER></DIV>
	In the future, we hope to test this enzyme in a variety of experimental conditions to verify catalytic activity. As the gut is an anaerobic environment we hope to test this beta-xylanase under anaerobic conditions to assess if lack of oxygen is key for functionality of the enzyme. In addition, we hope to test the enzyme at a number of pH levels as the gut of the porcupine can be both more acidic, and more basic than pH 7. These different experimental conditions may allow the beta-xylanase to give a positive result for repeats of our fluorophore cleavage assay. Furthermore, we hope to develop an assay for assessing secretion/diffusion of the enzyme via the pET26b(+) expression vector system. A possibly assay would be to set up a media degradation assay or measure levels of beta-xylanase in media after induction. <FONT color="#C1D35D"><H3>Endoglucanase</H3></FONT>

In order to assess the enzymatic activity of an endoglucanase isolated from <I>Ruminiclostridium thermocellum</I> by last year’s team, this year’s team designed a media pH change assay centered around the dye Congo Red and the substrate carboxymethylcellulose (CMC). CMC, a cellulose polymer, contains a phosphate group that when cleaved by an enzyme to cellobiose, lowers the pH of the surrounding media. Congo Red is able to detect this pH decrease as a colour-changed halo around the site of pH decrease. We designed a solid media (CMC+Glucose+Congo Red <A href="https://static.igem.org/mediawiki/2017/7/76/DalCMCCC.pdf" style="color: #C1D35D">protocol</A>) to support the growth of bacteria containing endoglucanase in the pET26b(+) expression vector. We hypothesized that endoglucanase activity would induce this pH change, showing diffusion of our enzyme and positive enzyme activity.

	BL21 DE3 <I>E. coli</I> containing endoglucanase (or empty vector alone) were cultured for 12 hours in 5 mL LB Broth+Kanamycin at 37⁰C, shaking. At 12 hours, 0.1M IPTG was added 1:1000 to the culture for a final concentration of 0.1 mM and incubated at 37⁰C, shaking, for another hour. The culture was then streaked onto Congo Red plates and incubated for 24 hours at 37⁰C. Progression was measured at 24, 48, and 72 hours.
Both empty vector and endoglucanase containing <I>E. coli</I> were able to produce halos due to pH decrease at the same rate (Fig. 9). As this protocol is traditionally for organisms that can degrade CMC to glucose instead of only degrading CMC to cellobiose, our addition of glucose to the media may have caused this pH change in both samples due to glucose metabolic pathways. In the future, different carbon sources may be useful to prevent this pH change but still allow bacterial growth without the full cellulose degradation pathway (such as xylose). <DIV><CENTER><FIGURE><FIGCAPTION>Fig9. Congo Red analysis of enzymatic activity via pH change.</FIGCAPTION></FIGURE></CENTER></DIV><FONT color="#C1D35D"><H2>Project Achievements</H2></FONT>
Over the course of the 2017 iGEM season, we have had some downs, but many more ups. <FONT color="#C1D35D">Successes</FONT><UL><LI>Developed a pipeline to identify, or &quot;mine&quot;, the porcupine metagenomic sequencing to discover novel enzymes.</LI><LI>Identified 8 potentially novel enzymes with variable percent identity.</LI><LI>Synthesized 5 of those enzymes, and successfully cloned 4 of them into psB1AK3.</LI><LI>Optimized our previous biobrick Endoglucanse(<A href="http://parts.igem.org/Part:BBa_K2160000" style="color: #C1D35D">BBa_K2160000</A>)by adding a C termincal HIS-tag and N terminal PelB sequence (<A href="https://2017.igem.org/Team:Dalhousie/Improve" style="color:#C1D35D">Improve</A>).</LI><LI>Successfully completed a fluorophore cleavage assay from the Hallam lab.</LI><LI>Isolated high molecular weight DNA from porcupine fecal samples.</LI><LI>Obtained efficient ligation and digestion with pJC8 controls. </LI><LI>Produced phage plaque with the phage packaging extract lamba DNA controls. </LI></UL><FONT color="#C1D35D">Failures</FONT><UL><LI>Did not clone our biobricks into the shipping vector psB1C3</LI><LI>Was not able to achieve the right environment for our novel beta-xylanase to function</LI><LI> Was not able to design a functional media assay for our enzymes</LI><LI> Could not make a functional metagenomic library </LI><LI> Sheared our high molecular weight DNA</LI></UL><FONT color="#C1D35D">References</FONT><P class="hangingindent"><FONT color="#ffffff">Neufeld, J., Engel, K., Cheng, J., Moreno-Hagelsieb, G., Rose, D. and Charles, T. (2011). Open resource metagenomics: a model for sharing metagenomic libraries. Standards in Genomic Sciences, 5(2), pp.203-210.</FONT></P></OL></UL></OL></UL></OL></UL></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>