Team:UMaryland/Interlab
Information
About
Our Team
Acknowled- gements
Fundraising
2015 Jamboree
Project
Description
Hok/Sok
PCR Design
Results
Side Projects
Interlab Study
Parts
All Parts
Basic Parts
Composite Parts
Outreach
Human Practices
Regional Meetup
Survey
Notebook
Journal
Protocols
Safety
iGEM Main
Interlab Study
Objective
The iGEM interlab study embodies iGEMâ€™s core values of collaboration, the development of an open community, the education of aspiring young scientists, and the advancement of synthetic biology. Thus, for the 2015 interlab study, the objective is for teams from around the globe to obtain fluorescence data from 3 different genetic devices. This aggregated data will not only aid the detailed characterization of these Biobricks, but also promote further joint scientific efforts of the iGEM community in synthetic biology.
Tested Devices
All of our devices were constructed in the PSB1C3 backbone. The chassis for all of our devices was E. coli BL21.
Device 1: J23101+I13504 in PSB1C3
Device 2: J23106+I13504 in PSB1C3
Device 3: J23117+I13504 in PSB1C3
Controls
Positive Control: BBa_I20270 (previously characterized promoter::GFP) in PSB1C3
Negative Control: BBa_R0040 (previously characterized promoter) in PSB1C3
Construction of the Devices
Our team acquired the necessary Biobricks for this interlab study from the distributed iGEM part kits. These Biobricks, each housed in the pSB1C3 vector, were transformed into E. coli K12 DH5-alpha following the standard transformation protocol and incubated overnight on chloramphenicol plates. Isolated colonies were cultured overnight in LB + chloramphenicol. Clones were isolated using a Qiaprep miniprep kit and subjected to a standard double digest per NEB's protocol for EcoR1-HF and Pst1-HF. The digested plasmids were visualized on a 0.8% agarose gel using a transilluminator to confirm the size of the interlab parts.
We designed and ordered a series of primers according to NEB's primer design guidelines toward the assembly of the promoter::GFP constructs in the pSB1C3 backbone. In each case, two PCR products would be combined in a Gibson assembly to form the desired plasmid. The pSB1C3 backbone containing each promoter was amplified using the standard forward primer and a promoter-specific reverse primer. In a second PCR, rbs::GFP (I13504) was amplified with a forward primer that would introduce part of the specific promoter sequence, as well as a standard reverse primer. This allowed for sufficient overlap between the the backbone::promoter and RBS::GFP amplification products for Gibson assemblies.
The PCR cycling conditions and annealing temperatures for each reaction were designed following NEB's standard PCR protocol for Q5 polymerase. The PCR products were purified with Qiagen's PCR clean-up kit and the size of the products verified on an agarose gel. The linear promoter::backbone constructs were then each combined with the amplified I13504 using the Gibson assembly protocol from NEB. The Gibson products were then transformed into BL21 and the constructs were isolated and insert size confirmed as per above, with additional verification through gene sequencing by Genewiz. However, our team was unable to assemble the third device of the J23117 promoter and GFP. Thankfully, this DNA construct was graciously donated to us with iGEM approval from the William and Mary 2015 iGEM team. The controls for this experiment were transformed directly onto BL21. The plasmids were miniprepped and verified through gene sequencing as well.
Testing the Devices
Our team followed the 2015 InterLab protocol for testing that was posted by iGEM for this study. Three separate biological replicates for the verified constructs and controls were grown in 10 mL culture tubes with loose caps in an incubator overnight for 16-18 hours. We recorded the OD500 of the cells and the samples were diluted to within 5% of 0.5. 150 microliters of three biological replicates of each diluted sample and three technical replicates were transferred to a 96 well plate using a p-200 pipette. The fluorescence of the samples was then quantified using the SpectraMax M2 microplate reader, at an excitation wavelength of 498 nm and an emission recorded at 541 nm with the auto cutoff setting on. We are also very thankful to Dr. Bentley's lab on campus for allowing us to use their plate reader for testing. The results of the experiments were processed are reported below in arbitrary units.
Protocols
Protocols for construction of devices here.
iGEM DNA distribution kit protocol
iGEM transformation protocol
NEB Gibson assembly protocol
Qiagen miniprep protocol
Transformation
iGEM RE digest protocol
iGEM primer design procedure
NEB PCR protocol
Results
Fluorescence data in arbitrary units
