<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UMaryland_HokSok skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UMaryland/HokSok</SPAN></H1></DIV></DIV><DIV id="topkek"><DIV id="swagmenu"><UL><LI class="has-sub"><A href="https://2015.igem.org/Team:UMaryland/about"><SPAN>Information</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/about"><SPAN>About</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Team"><SPAN>Our Team</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Attributions"><SPAN>Acknowled- gements</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/fundraising"><SPAN>Fundraising</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/After"><SPAN>2015 Jamboree</SPAN></A></LI></UL><LI class="has-sub"><A href="https://2015.igem.org/Team:UMaryland/Description"><SPAN>Project</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Description"><SPAN>Description</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/HokSok"><SPAN>Hok/Sok</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Design"><SPAN>PCR Design</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Results"><SPAN>Results</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/sideprojects"><SPAN>Side Projects</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Interlab"><SPAN>Interlab Study
</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:UMaryland/Part_Collection"><SPAN>Parts</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Part_Collection"><SPAN>All Parts</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Basic_Part"><SPAN>Basic Parts</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Composite_Part"><SPAN>Composite Parts</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:UMaryland/Practices"><SPAN>Outreach</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Practices"><SPAN>Human Practices</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Collaborations"><SPAN>Regional Meetup</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Survey"><SPAN>Survey</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Team:UMaryland/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Notebook"><SPAN>Journal</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/protocols"><SPAN>Protocols</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:UMaryland/Safety"><SPAN>Safety</SPAN></A></LI><LI class="has-sub"><A href="https://2015.igem.org/Main_Page"><SPAN>
iGEM Main
</SPAN></A><UL><DIV id="cover"><DIV id="bar"><P style="font-size:64px"><B>Project Design: Hok/Sok</B></P><P style="font-size:30px">&quot;To maintain or not to maintain, that is the question.&quot; -- Every plasmid-carrying bacterium, ever</P></DIV>&lt;</DIV><DIV id="layer2"><DIV id="contentbox"><P style="font-size:32px;text-align:center;font-family:Verdana, Geneva, sans-serif;"><B>Purpose</B></P><P style="font-size:24px">When setting up our experimental design, we focused on answering three main questions:</P></DIV></DIV></UL></LI><LI>1. Can Hok-Sok maintain a plasmid over many generations? If so, how does its performance compare to classical antibiotic pressure systems?</LI><LI>2. Does the Hok-Sok cassette restrict the growth of the bacteria it resides in?</LI><LI>3. Do proximity effects occur between Hok-Sok and a neighboring expression construct? Alternatively, does the activity of Hok-Sok affect expression of a protein whose coding region is near the construct?</LI><P style="font-size:18px" align="center"><B>4. Can this cell living with Hok-Sok stay happy?</B></P><P style="font-size:24px">The ability to use plasmids as vectors to introduce genes of interest in <I>E. coli</I> is one of the most essential bioengineering tools. However, one of the limitations of transforming a bacterium with a plasmid, is that the organism will eventually eject the plasmid over time. To counter this, scientists add a positive selective pressure on <I>E. coli</I> to retain plasmids carrying resistance genes through the use of antibiotics. While this technique has proven to be reliable and effective, there are many limitations. The prevalent use of antibiotics both for medical and agricultural purposes has rapidly increased the number of pathogens that harbor antibiotic resistant genes. As a result, there is a pressing need to find an alternative to antibiotic use for plasmid maintenance to prevent the spread of antibiotic resistant genes. Many synthetic biology projects which focus of solving health or environmental issues are confined to the lab because of these limitations. The University of Maryland iGEM team seeks to solve this problem by developing an alternative plasmid maintenance system that should liberate other iGEM teams from the dependance on antibiotic usage. We hypothesized that Hok-Sok, our plasmid maintenance system, could maintain recombinant plasmids, as it does natural ones. We also hypothesized that Hok-Sok would have a slight negative effect on bacterial growth rate, in line with other alternative maintenance systems such as sRNBC (<A href="http://parts.igem.org/Part:BBa_K817015">K817015</A>), as well as the amount of protein expression due to competing parallel promoters. In order to answer these questions, we set up a variety of testing procedures, as shown below.</P><P style="font-size:24px;text-align:center;font-family:Verdana, Geneva, sans-serif;">Section Summary</P><UL><LI>1. We wanted to answer three main questions concerning the ability of Hok-Sok to maintain a plasmid.</LI><LI>2. We hypothesized that Hok-Sok would be able to replicate its natural maintenance ability for a recombinant plasmid.</LI><LI>3. We devised tests to measure maintenance over time.</LI></UL></DIV></DIV><DIV id="layer1"><DIV id="contentbox"><P style="font-size:32px;text-align:center;font-family:Verdana, Geneva, sans-serif;"><B>Hok/Sok Construct</B></P><P style="font-size:24px">Prior to experimentation, we had to insert the Hok-Sok construct into pSB1C3 in order to make it a BioBrick. We originally planned to PCR amplify the cassette out of the R1 plasmid of <I>E. coli</I>, but we were unable to find an suitable wild-type strain that was easily available. Instead, we turned to synthesizing the construct as a gBlock from IDT. As a 580 bp dsDNA fragment, it was suitable for addition via Gibson Assembly into pSB1C3. After subsequent transformation, miniprep, and confirmation sequencing, we had the first piece of our testing puzzle.</P><P style="font-size:18px" align="center"><B>Proof that at least some of our cloning worked</B></P><P style="font-size:24px;text-align:center;font-family:Verdana, Geneva, sans-serif;">Section Summary
<UL><LI> 1. Full Hok-Sok sequence taken from Gerdes et al. Sequence is originally from the R1 plasmid of <I>E. coli</I>.</LI><LI> 2. g-Block of Hok-Sok sequence ordered from Integrated DNA Technologies, then assembled into pSB1C3 using the Gibson Assembly method.</LI></UL></P></DIV></DIV><DIV id="layer2"><DIV id="contentbox"><A name="HS"><P style="font-size:32px;text-align:center;font-family:Verdana, Geneva, sans-serif;"><B>Fluorescence Studies</B></P></A><P style="font-size:24px">In order to determine if Hok/Sok was capable of maintaining a plasmid without antibiotic pressure, we decided to use a visual reporter gene to quantify the ability of Hok/Sok to maintain plasmids over many generations. We decided to use a RFP along with a degradation tag as the reporter gene. The most suitable candidate was an unstable LVA-tagged RFP that has a half-life of 1 hour. The shorter half life allows for more frequent measurements of protein production that would not aggregate over time. Therefore we combined a constitutive promoter and RBS to the LVA-tagged RFP through 3A assembly. We transformed this construct to <I>E. coli</I> DH5a to confirm the effectiveness of this reporter gene and its expression through increased fluorescence. Afterwords we ordered a g-block of our Hok/Sok+reporter construct. The expression of this reporter gene is proportional to plasmid number. Therefore, we concluded that if the cells containing a plasmid with both Hok/Sok and reporter gene could maintain fluorescence over many generations without the positive pressure of antibiotics compared to our controls, Hok/Sok can be used as a viable plasmid maintenance system. 

</P><P style="font-size:24px">We chose unstable Red Fluorescent Protein (RFP) as a marker for all our test groups to represent whether or not the inserted plasmid is still present in the bacteria. If the plasmid is maintained, the RFP is expressed and the overall fluorescence of the culture is greater. In contrast, if the bacteria does not feel enough pressure to keep the plasmid and ejects it, the measured fluorescence is on the lower end. From this data, we can gather whether or not the maintenance system is effective in preserving a plasmid in bacteria that is not beneficial to its survival, such as the aforementioned RFP. 
</P><P style="font-size:24px">The reason for using unstable RFP is that the half-life of the proteins is shorter than a stable protein, therefore we can tell in real-time, or at least more so, whether or not the plasmids are present. The RFP degrades and unless the plasmid is maintained, the fluorescence in the cells actively declines. 
</P><P style="font-size:24px">We used two <I>E. coli</I> strains for our testing. Originally, we used BL21 strain of <I>E. coli</I> because it is known to be the best for testing because the cell lacks proteases; the protein expression is optimal because the proteins are not digested by the enzymes. After testing BL21, we transitioned to the DH5a strain of <I>E. coli</I> because the cells lack recombinase. 

</P><P style="font-size:24px;text-align:center;font-family:Verdana, Geneva, sans-serif;">Section Summary</P><UL><LI>1. Hok-Sok was coupled to a constitutive generator of unstable RFP to form a larger composite part.</LI><LI>2. In order to test the ability of Hok-Sok to maintain protein expression, five sets of cultures were grown in LB media for fluorescence studies.</LI><LI>3. Cultures were grown for 20 hours overnight prior to fluorescence measurements using a plate reader. Excitation at 555 nm, emission at 584 nm. </LI></UL></DIV></DIV><DIV id="layer1"><DIV id="contentbox"><A name="PCR"><P style="font-size:32px;text-align:center;font-family:Verdana, Geneva, sans-serif;"><B>Plating Studies</B></P></A><P style="font-size:24px">While our fluorescence studies were effective at measuring protein expression over time, we wanted a second test that would more directly measure whether or not plasmids were being maintained throughout generations. Using the identical cultures as the fluorescence tests, we devised a plating protocol involving a challenge of chloramphenicol every 24 hours.</P><OL><LI>1.	Dilute each culture (1 : 10<SUP>6</SUP>) with LB media</LI><LI>2.	Plate 10 µL of each diluted culture on a LB-agar + chloramphenicol plate</LI><LI>3.	Incubate plates overnight</LI><LI>4.	Count colonies the next day</LI></OL><P style="font-size:24px">The goal in doing this was to determine how many bacteria were surviving by retaining their plasmids. We did not discriminate between the color of colonies.</P><P style="font-size:24px">For continuing generations of BL21 strain <I>E. coli</I>, we observed that on the plates for groups A and B, there was growth but no redness. If the bacteria were retaining the plasmids with the chloramphenicol resistance, the RFP gene should have been expressed and the colonies should fluoresce. We hypothesized that the chloramphenicol resistance gene was being recombined into the bacterial genome so the bacteria could therefore freely eject our inserted plasmids. As BL21 carries the gene for recombinase, it is possible. However, DH5α, as a common cloning strain, does not have recombinase. We created a new generation with every group (A, B, C, D, and E) to test whether the same plate would have similar results or once the bacteria stopped fluorescing, there would be no growth on the plates.</P></DIV></DIV><DIV id="layer2"><DIV id="contentbox"><A name="PCR"><P style="font-size:32px;text-align:center;font-family:Verdana, Geneva, sans-serif;"><B>Growth Curve</B></P></A><P style="font-size:24px">While the Hok-Sok cassette may help to maintain a plasmid, it may also impact the rate of cell growth. We were unsure of the level of stress that came from additional hok and sok translation, and we decided to measure whether or not cultures containing the Hok-Sok construct would grow at a similar rate to control. We created a growth curve of Hok/Sok in comparison to controls to test the effectiveness of the Hok/Sok system in keeping the bacteria alive. We had four groups:
<UL><LI>•	Hok/Sok without chloramphenicol</LI><LI>•	Hok/Sok with chloramphenicol</LI><LI>•	RFP without chloramphenicol</LI><LI>•	RFP with chloramphenicol</LI></UL></P><P style="font-size:24px">We started growing 250 mL cultures and monitored the OD at 600nm using a spectrophotometer over the span of 7.5 hours. </P></DIV></DIV><DIV id="layer1"><DIV id="contentbox"><A name="PCR"><P style="font-size:32px;text-align:center;font-family:Verdana, Geneva, sans-serif;"><B>Fluorescence Loss Analysis</B></P></A><P style="font-size:24px">Interested as to why our cells were losing fluorescence in the span of a week, we increased the level of protein expression in order to observe this effect on a larger scale. This was done by switching cell lines from DH5α to BL21, which is optimized for protein expression due to the removal of several proteases. We repeated plating experiments in triplicate in order to determine if fluorescence loss would be as dramatic in such a small span of time. Plates were exposed to UV light using a transilluminator in order to visually observe fluorescence loss over many generations.</P><P style="font-size:24px">In addition to our observations, we also wanted to make sure that our cultures had not been contaminated with a non-fluorescent, chloramphenicol resistant bacteria that had out-competed our intended culture. We thus performed a gram stain in order to verify that our bacteria were gram negative bacilli.</P><P style="font-size:24px;text-align:center;font-family:Verdana, Geneva, sans-serif;">Section Summary</P><UL><LI>1. We wanted to see whether fluorescence loss was independent of cell strain.</LI><LI>2. We switched to a better cell line for protein production, BL21, in order to magnify the effects of fluorescence loss.</LI><LI>3. We performed a gram stain to confirm our colonies were descendants of our initial <I>E. coli</I> colony and not contamination.</LI></UL></DIV></DIV><DIV id="layer2"><DIV id="contentbox"><A name="PCR"><P style="font-size:32px;text-align:center;font-family:Verdana, Geneva, sans-serif;"><B>Sequence Analysis</B></P></A><P style="font-size:24px">In order to find a definite answer as to the loss of fluorescence, we took minipreps of our non-fluorescent DH5α cultures, digested them to extract their inserts, and separated them using agarose gel electrophoresis in order to determine whether or not they were the proper size. For samples that contained an insert of the proper size, the miniprepped plasmid was then sequenced in order to determine the genetic reason as to the fluorescence loss.</P><P style="font-size:24px;text-align:center;font-family:Verdana, Geneva, sans-serif;">Section Summary</P><UL><LI>1. We wanted to determine a genetic reason for fluorescence loss.</LI><LI>2. We analyzed insert sizes and sequenced plasmids in order to observe changes in plasmids over time.</LI></UL></DIV></DIV><DIV id="layer1"><DIV id="contentbox"><A name="PCR"><P style="font-size:24px;text-align:center;font-family:Verdana, Geneva, sans-serif;">Parts Referenced</P></A><UL><LI><A href="http://parts.igem.org/Part:BBa_K1783001">Hok-Sok Type I Toxin-Antitoxin Cassette</A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1783002">Constitutive Promoter-RBS-Unstable RFP (LVA-Tagged)</A></LI><LI><A href="http://parts.igem.org/Part:BBa_K1783003">Unstable RFP under Hok-Sok Regulation</A></LI></UL><P><B> References </B></P><OL><LI>1. Gerdes, K., Thisted, T., &amp; Martinussen, J. (1992). Mechanism of post-segregational killing by the hok/sok system of plasmid R1: Sok antisense RNA regulates formation of a hok mRNA species correlated with killing of plasmid-free cells. <I>Molecular Microbiology</I>, 223(1), 1807-1818. doi:10.1016/0022-2836(92)90714-U</LI><LI>2. Mitsuoki Kawano (2012) Divergently overlapping cis-encoded antisense RNA regulating toxin-antitoxin systems from <I>E. coli</I>, <I>RNA Biology</I>, 9:12, 1520-1527, DOI: 10.4161/rna.22757</LI></OL></DIV></DIV></BODY></HTML>