Team:UMaryland/Notebook
Information
About
Our Team
Acknowled- gements
Fundraising
2015 Jamboree
Project
Description
Hok/Sok
PCR Design
Results
Side Projects
Interlab Study
Parts
All Parts
Basic Parts
Composite Parts
Outreach
Human Practices
Regional Meetup
Survey
Notebook
Journal
Protocols
Safety
iGEM Main
Notebook
Week 2
Week 4
Week 6
Week 8
Week 10
Week 12
Week 14
Week 16
Week 1
Project Preparations: Miraculin, SRNBC (Hok/Sok homolog), Lutein pathway genes
- Prepared a bacterial culture containing constitutive GFP (K584001) to grow overnights (E. coli in LB with chloramphenicol)
- Performed a transformation in order to replicate our DNA with the constitutive GFP, and plated half of them on the plates containing chloramphenicol and used the other half for another overnight batch
- Performed a mini-prep on our overnights with the constitutive GFP in order to isolate plasmids and verify that they were present through spectrophotometry
- Started two separate 3A assemblies in order to piece together the pBAD promoter with the miraculin gene and the constitutive GFP gene to the SRNBC gene
- Performed a transformation in order to clone our new plasmids, one containing the pBAD promoter and miraculin gene and the other containing SRNBC and the constitutive GFP
- Transformed β-cyclase, AppY, and CRTBEY genes (for lutein project) in bacteria to grow on plates overnight
- Started a new 3A assembly for the SRNBC and constitutive GFP construct along with pBAD, constitutive colicinFy, and lysostaphin
- Final products were SRNBC + constitutive GFP, SRNBC + constitutive colicinFy, and pBAD + lysostaphin
- Prepared overnights for the pBAD + miraculin construct that grew on the plates and the β-cyclase, AppY, and CRTBEY genes
- Performed minipreps on the overnights we had prepared the day before (for the β-cyclase, AppY, and CRTBEY genes) and shipped them off to be sequenced
- Performed transformations on the SRNBC + constitutive GFP, SRNBC + constitutive colicinFy, and pBAD + lysostaphin digests and let them grow overnight
Week 2
Miraculin
- RE digest on β-cyclase, AppY, CREB, and CRTBEY genes along with the pSB1C3 backbone and the pBAD + miraculin out of the pSB1A3 backbone so that we could move the genes to the pSB1C3 backbone
- Ran gel with all parts and cut out the bands
- Mini-prepped SRNBC + constitutive colicinFy and shipped them off to be sequenced
- Performed gel purification on gels from day before but they failed; they didn’t show up in the spectrophotometer
- Performed minipreps on pBAD+Miraculin in the pSB1C3 backbone as well as the const. GFP+SRNBC
- Performed RE digests on the previous mini-preps: EcoRI and PstI on pBAD + Miraculin, and XBa1 and Pst1 on SRNBC + Constitutive GFP
Designing gBlocks
- Designed gblocks for epsilon-cyclase (from arabidopsis), pyocin and Hok/Sok, and ordered them
Week 3
Miraculin
- Sequence of pBAD + Miraculin confirmed through sequencing
- Culture of pBAD + Miraculin was induced with 0.1% arabinose at OD of 1
- Attempted to purify the Miraculin out of the induction culture by first using a French Press to lyse the cells and then running it through cobalt bead column by FPLC (Fast Protein Liquid Chromatography)
- Ran the resulting 62 elutions through an SDS-PAGE
- Failed to extract Miraculin using French press, FPLC and SDS-Page
- Made overnights of pBAD + Miraculin culture to prepare for test inductions
SRNBC and Hok/Sok
- Performed a gel extraction of SRNBC, const. GFP, and pBAD which we then ligated them together to make the SRNBC construct
- Performed transformations of the constitutive GFP + SRNBC construct
- Transformations of SRNBC + Constitutive GFP failed
- Performed PCR on the pSB1C3 backbone in order to amplify it in preparation for a Gibson Assembly with the Hok/Sok gene
PCR
-  Began work on Arduino code to cycle the machine
-  Purchased Peltier units and lm35 temperature sensors
Week 4
Miraculin
- Made overnights of pBAD+Miraculin culture to prepare for test inductions tomorrow
- Inoculated 3 test cultures of pBAD+Miraculin in LB broth with Ampicillin
- Induced the tubes with 0.05%, 0.1%, and 0.2% arabinose in order to initiate production of miraculin at an OD of 0.4
- Prepared the induction samples for SDS-PAGE by separating out the media, supernatant after lysing with 200 uL SDS and then the supernatant after lysing with SDS again (pellet supernatant)
- Ran samples of 0%, 0.05%, 0.1%, and 0.2% arabinose through the gel, but no bands appeared
- Prepared to perform procedure again using bl21 cells instead of Dh5a cells
Hok/Sok
- Performed an RE digest on the pSB1C3 backbone with XbaI and SpeI and then ran it through a gel to separate it out, to prepare for the moving of Hok/Sok into pSB1C3
PCR
- Received Peltier's, lm35's and began testing the code
- 6 volt battery pack was found to be insufficient to run Peltier's
- Purchased an AC/DC power converter
Week 5
Miraculin
- transformation in BL21 strain was successful
- SDS page showed a band at roughly 25 kDa
Hok/Sok
- inserted Hok/Sok gblock into pSB1C3 backbone using Gibson assembly
- construct sent for sequencing
Interlab Study
- transformed parts into dh5a: 3 Anderson promoters, GFP, and RFP
PCR
-  Peltier units heated and cooled to proper temperatures however took 30 minutes to cycle properly
-  started construction of housing to insulate the element to cycle temperature faster and reduced time to 15 minutes
-  burned out Peltier elements and broke temperature sensor, and ordered new ones
Week 6
Hok/Sok
- transformed unstable GFP and RFP with PBAD into dh5a
- transformed unstable RFP and inducible lac promoter (k1399011) and const_promoter+RBS (K081005) into dh5a
- ligated const_promoter + RBS + RFP
Interlab
- performed a 3A assembly of each promoter + GFP in pSB1K3 (GFP in pSB1A3 and promoters in pSB1C3)
PCR
- received new Peltier unit and temperature sensor
- began milling on metal PCR tube holder
Week 7
Hok/Sok
- construct created through 3A assembly of quick degrading RFP and constitutive promoter +RBS was proven incorrect using sequencing
- Sequencing showed a 3A assembly of quick degrading RFP and a "leaky" lactose promoter
- Re- ran 3A assembly but the transformation failed
- could be due to contaminated SOC media
PCR
-  received milled PCR tube holder
- embedded temperature sensors into holder reran the machine and again burned out Peltier's
- spoke with Peltier manufacturers and Open PCR staff and found the Peltier's used for PCR were specialized and more expensive
- bought higher grade Peltier's to handle high voltages
Week 8
Hok/Sok
- const_promoter + RBS + RFP were replated from last week but produced no colonies
- re-transformed original and new 3A assembly which produced the correct sequence
- ligated const_promoter:QD-RFP in pSB1C3
- performed a 3A assembly of Hok/Sok plasmid with QD-RFP downstream in pSB1K3
- site directed mutagenesis of quick degrading GFP (Bba_K750000)  failed
- there was no change from the original sequence
Interlab study
- 3A assembly attempts for the promoters and GFP failed
- Gibson Assemblies produced a vast amount of colonies
- questionable success because the amount of colonies may indicate false positives or contamination
Lutein
-  ordered pAC-LYC which encodes 3 enzymes from E. Herbicola to produce basal levels of lycopene
PCR
- Peltier's were still in shipping however work continued on the software
- designed relay configuration to both heat and cool Peltier's with an unidirectional current flow
Week 9
Hok/Sok
- previous 3A assembly from last week showed no colony growth on kanamycin plate
- transformed RFP into C3 backbone last week
- colonies were produced that were not distinctively red but with the correct sequence
- 3A assembly with the RFP + Hok/Sok failed
- Created primers for Gibson assembly to create Hok/Sok +const_promoter::RBS::unstable RFP
Interlab Study
- replated last week’s constructs with GFP at lower concentration selected for chloramphenicol which was successful
- K08  and K13 constructs grew colonies
- Miniprepped constructs and only promoter K08+GFP had the correct sequence
PCR
- received high rated peltier units that did not break
- took ~31 minutes to complete 1 cycle but insulated heating dropped the time to 16 minutes
- Took apart a hair dryer for heating element
- heating went to >95 in ~2 seconds, cooled to 50 in ~8 seconds
- The heating element is not very precise so it resulted in a lot of overshoot in temperature
Week 10
Hok-Sok
- created construct with RFP and hok/sok
- Pcr of the construct failed so the construct will be sequenced to check
PCR
- cycle data was taken and it exhibits consistency
Week 11
Hok/Sok
- 3A assembly of H/S + RFP failed
- ran a PCR of pSB1C3+H/S+RFP using 3 different rxn buffers
- HF rxn buffer
- GC rxn buffer
- GC rxn buffer w/ DMSO which yielded product
- Ran a PCR of unstable RFP using the  3 rxn buffers above
Interlab Study
- used Phusion instead of Q5 for PCR
- ran PCR for GFP 1, GFP 3, and IL1 which produced the correct sized products
- Gibson Assembly of GFP + IL1 produced colonies
- pSB1C3 - IL3 did not have bands in gel of appropriate size
Lutein
- RE digests of pLAC-RFP in pSB1C3
- ran a Gibson Assembly of E-cyclase/C3 and E-hydroxylase/C3 which produced colonies
PCR
- rebuilt top of hair dryer housing w/ soda can
- observed random temp spikes occurring during each run
- caused by hardware issue with relays
- purchased new relays to test this week
Week 12
Hok/Sok
- Ran a Gibson Assembly of Hok/Sok + RFP
Interlab
- worked on finishing the last construct
- K13 sequenced correctly and the pcr looks good
- contacted W&M on possible collaboration for the last construct
Lutein
- ordered primers for Gibson Assembly
PCR
- replaced old relays with higher powered relays
- reduced mass by over 50% of the peltier machine
- resulted in speedier temperature changes
Week 13
Hok/Sok
- sequence of hok/sok construct created last week was incorrect
- may put a unique RE site between hok/sok and rfp when retrying 3A assembly, RE cloning and Gibson
Interlab
- Gibsons of IL3 (K13 + GFP) have all failed
- 3 trials of IL2 , IL1,+ control (const_promoter with GFP), - control were tested using the plate reader
- IL2 was a little low in fluorescence, similar to the - control
PCR
- Rewired hair dryer and changed relays to handle higher wattage
- attempted PCR cycle failed
- assumed temp sensor needed to be immersed in mineral oil to properly report temp
- denaturation temp may have been too low
- increased the denaturation temperature, attained an amplified product
Week 14
Hok/Sok
- Began testing for RFP fluorescence for Hok/Sok effectiveness as a plasmid maintenance system
- Group A: RFP coding region; constitutive promoter ; chloramphenicol in liquid culture
- Group B: RFP coding region; constitutive promoter ; no chloramphenicol in liquid culture
- Group C: RFP coding region; no promoter ; chloramphenicol in liquid culture
- Group D: H/S-RFP coding region; constitutive promoter ; chloramphenicol in liquid culture
- Group E: H/S-RFP coding region; constitutive promoter ; nochloramphenicol in liquid culture
- Group F: H/S-RFP coding region; constitutive promoter ; chloramphenicol in liquid culture
Interlab
- obtained the last construct from W&M
- tested the last construct using the plate reader
- Turned in the IL study
PCR
-began work on incubator
-incubator was proven to maintain 37 degrees Celsius
Week 15
Hok/Sok
- Tested generations alpha beta and gamma in BL21 cells
- Alpha and beta contained groups A-E
- gamma had only group F
Week 16
Hok/Sok
- Tested generation delta using DH5alpha
- contained groups A-E
