<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Kyoto_Material_and_method"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Kyoto/Material_and_method">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Kyoto/Material_and_method&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Kyoto/Material_and_method&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Kyoto/Material_and_method&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Kyoto/Material_and_method" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Kyoto/Material and method</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><A id="kyoto_alert" target="_blank" href="https://2013.igem.org/Team:Kyoto/TimeLimit">count down</A><DIV id="kyoto-main"><DIV class="texts"><DIV id="material"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Material"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Parts"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Parts</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Strains"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Strains</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#E.coli.28K12.29"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">E.coli(K12)</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Protocol"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Protocol</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#PCR"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Mutation_PCR"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Mutation PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#RNA_Extraction"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">RNA Extraction</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Making_Competent_Cells"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Making Competent Cells</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Miniprep"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Miniprep</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Ethanol_Precipitation"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Ethanol Precipitation</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Electrophoresis"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Gel_Extraction_and_PCR_Purification"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Gel Extraction and PCR Purification</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#qRT-PCR"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">qRT-PCR</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Ligation"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Restriction_Enzyme_Digestion"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">Restriction Enzyme Digestion</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Media"><SPAN class="tocnumber">2.12</SPAN><SPAN class="toctext">Media</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Buffer_TB"><SPAN class="tocnumber">2.13</SPAN><SPAN class="toctext">Buffer TB</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Solubilization_of_Antibiotics"><SPAN class="tocnumber">2.14</SPAN><SPAN class="toctext">Solubilization of Antibiotics</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Transformation"><SPAN class="tocnumber">2.15</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Sequence"><SPAN class="tocnumber">2.16</SPAN><SPAN class="toctext">Sequence</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Golden_Gate_Assembly"><SPAN class="tocnumber">2.17</SPAN><SPAN class="toctext">Golden Gate Assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#qRT-PCR_2"><SPAN class="tocnumber">2.18</SPAN><SPAN class="toctext">qRT-PCR</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Material">Material</SPAN></H1><H2><SPAN class="mw-headline" id="Parts">Parts</SPAN></H2><P>&lt;groupparts&gt;iGEM013 Kyoto&lt;/groupparts&gt;</P><H2><SPAN class="mw-headline" id="Strains">Strains</SPAN></H2><H3><SPAN class="mw-headline" id="E.coli.28K12.29"><I>E.coli</I>(K12)</SPAN></H3><UL><LI><B>JM109</B></LI><LI><B>XL10-gold</B></LI><LI><B>BL21(DE3)pLysS</B></LI><LI><B>JW0588</B></LI></UL><H1><SPAN class="mw-headline" id="Protocol">Protocol</SPAN></H1><H2><SPAN class="mw-headline" id="PCR">PCR</SPAN></H2><UL><LI>Mix the following.</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Name</TH><TH>Volume
</TH></TR><TR><TD>25mM MgSO4</TD><TD>1.5 µL
</TD></TR><TR><TD>2mM dNTPs</TD><TD>2.5 µL
</TD></TR><TR><TD>10xBuffer for KOD -Plus- ver.2</TD><TD>2.5 µL
</TD></TR><TR><TD>Template DNA</TD><TD>properly
</TD></TR><TR><TD>Primer Forward (10 µM)</TD><TD>0.75 µL
</TD></TR><TR><TD>Primer Reverse (10 µM)</TD><TD>0.75 µL
</TD></TR><TR><TD>KOD -Plus-</TD><TD>0.5 µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 25 µL
</TD></TR><TR><TD>Total</TD><TD>25 µL
</TD></TR></TBODY></TABLE><UL><LI>Put samples into Thermal Cyclers and run the following steps.
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>PreDenature</TH><TH>Denature</TH><TH>Anealing</TH><TH>Extension</TH><TH>cycle
</TH></TR><TR><TD>94 °C</TD><TD>98 °C</TD><TD>Tm-5 °C</TD><TD>68 °C</TD><TD>--
</TD></TR><TR><TD>2 min</TD><TD>10 sec</TD><TD>30 sec</TD><TD>1 min/kb</TD><TD>30 cycles
</TD></TR></TBODY></TABLE><UL><LI>Agarose Gel Electrophoresis for confirmation.</LI></UL><P><B>PCR: ToYoBo Quick Taq HS DyeMix</B></P><UL><LI>Mix the following.</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Name</TH><TH>Volume
</TH></TR><TR><TD>Template DNA</TD><TD>properly
</TD></TR><TR><TD>Primer Forward (10 µM)</TD><TD>0.2 µL
</TD></TR><TR><TD>Primer Reverse (10 µM)</TD><TD>0.2 µL
</TD></TR><TR><TD>2x Quick Taq HS DyeMix</TD><TD>5 µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 10 µ</TD></TR><TR><TD>Total</TD><TD>10 µL
</TD></TR></TBODY></TABLE><UL><LI>Put samples into Thermal Cyclers and run the following steps.
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>PreDenature</TH><TH>Denature</TH><TH>Anealing</TH><TH>Extension</TH><TH>cycle
</TH></TR><TR><TD>94 °C</TD><TD>94 °C</TD><TD>Tm-5 °C</TD><TD>68 °C</TD><TD>--
</TD></TR><TR><TD>2 min</TD><TD>30 sec</TD><TD>30 sec</TD><TD>1 min/kb</TD><TD>30 cycles
</TD></TR></TBODY></TABLE><UL><LI>Agarose Gel Electrophoresis for confirmation.</LI></UL><H2><SPAN class="mw-headline" id="Mutation_PCR">Mutation PCR</SPAN></H2><P>Use TaKaRa PrimeSTAR Mutagenesis BasaI Kit</P><UL><LI>Dilute the concentration of template DNA with MilliQ.</LI><LI>Mix the following</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>PrimeSTAR Max Premix (2x)</TD><TD>25µL
</TD></TR><TR><TD>Primer Forward </TD><TD>10pmol
</TD></TR><TR><TD>Primer Reverse</TD><TD>10pmol
</TD></TR><TR><TD>Template DNA (1ng/µL)</TD><TD>1~2µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 50µL
</TD></TR><TR><TD>Total</TD><TD>50µL
</TD></TR></TBODY></TABLE><UL><LI>30 cycles for 10s at 98℃, for 15s 55℃, and for 40sec at 72℃.</LI><LI>Agalose Gel Electrophoresis for confirmation.</LI><LI>Negative Control: Use nothing.</LI><LI>Add 2L PCR products and 20L competent cells for transformation.</LI></UL><H2><SPAN class="mw-headline" id="RNA_Extraction">RNA Extraction</SPAN></H2><P>Use　ISOGEN－LS（NIPPON　GENE,311-02621</P><UL><LI>Add 1mL ISOGEN-LS to sample and vortex.</LI><LI>Store for 5min on ice.</LI><LI>Add 250µL chloroform and shake vigorously for 15 sec.</LI><LI>Store for 3min. on ice.</LI><LI>Centrifuge 17400xg for 10min. at 4°C.</LI><LI>Transfer aqueous phase to another tube and add 0.8 volume isopropanol.</LI><LI>Store for 10min. on ice.</LI><LI>Centrifuge 17400xg for 10min. at 4°C.</LI><LI>Discard the supernatant.</LI><LI>Add 800µL 80% ethanol and vortex.</LI><LI>Centrifuge 7500xg for 5min. at 4°C.</LI><LI>Discard the supernatant.</LI><LI>Dry briefly.</LI><LI>Dissolve in nuclease-free water.</LI></UL><H2><SPAN class="mw-headline" id="Making_Competent_Cells">Making Competent Cells</SPAN></H2><UL><LI>Streak <I>E.coli</I> cells on an LB plate</LI><LI>Allow cells to grow at 37°C overnight</LI><LI>Place one colony in 3 mL LB media (+antibiotic selection if necessary), grow overnight at 37°C</LI><LI>Take 2 ml LB media and save for blank. Transfer 500&amp;micro L overnight culture into 50 mL LB media in 500 mL flask</LI><LI>Allow cell to grow at 37°C (250 rpm), until OD600= 0.4 (~2-3 hours)</LI><LI>Place cells on ice for 30 mins</LI><LI>Transfer cells to a centrifuge tube (50 mL), and centrifuge cells in High Speed refrigirated centrifuge at 4°C for 10 mins at 3,000 g.</LI><LI>Pour off media and resuspend cells in 12 mL of cold TB.</LI><LI>Centrifuge cells at 4°C for 10 mins at 3,000 g (2500 rpm)</LI><LI>Pour supernatant and resuspend cells (by pipetting) in 5 mL of cold TB and 350 µL of DMSO. Transfer 20 µL to 1.5 mL　tube</LI><LI>Ependorff tubes placed on ice. Freeze the cells in liquid nitrogen. Cells stored at -80°C can be used for transformation for up to ~6 months</LI></UL><H2><SPAN class="mw-headline" id="Miniprep">Miniprep</SPAN></H2><P>Use LaboPass Plasmid Mini Purification Kit.</P><UL><LI>Pick a single colony from a freshly streaked selective plate and inoculate a culture of about 4mL Plusgrow medium containing the appropriate selective antibiotic.</LI><LI>Incubate at 160 rpm for 8 h at 37 °C with vigorous shaking.</LI><LI>Pellet the bacterial culture by centrifugation for 1 min at 15,000 g in a tabletop centrifuge.</LI><LI>Discard the supernatant as much as possible.</LI><LI>Resuspend pelleted bacterial cells in 250 µL of Buffer S1.</LI><LI>Transfer the suspension to a new 1.5 mL tube.</LI><LI>Add 250 µL of Buffer S2 and mix by inverting the tube 4 times (do not vortex).</LI><LI>Add 350 µL of Buffer S3 and immediately mix by inverting the tube 4-6 times (do not vortex).</LI><LI>Centrifuge for 10min.</LI><LI>Transfer carefully the supernatant to a spin column and centrufuge for 1 min.</LI><LI>Remove the spin column, discard the flowthrough, and re-insert the spin column to the collection tube.</LI><LI>Apply the 750 µL of Buffer PW and centrifuge for 1 min.</LI><LI>Remove the spin column, discard the flowthrough, and re-insert the spin column to the collection tube.</LI><LI>Discard the flowthrough, and centrifuge for an additional 1 min to remove residual wash buffer.</LI><LI>Transfer the spin column to a new 1.5 mL tube.</LI><LI>Add 30 µL of MilliQ, let stand for 1 min, and centrifuge for 1 min.</LI></UL><H2><SPAN class="mw-headline" id="Ethanol_Precipitation">Ethanol Precipitation</SPAN></H2><P>Use Ethachinmate (NIPPON GENE、312-01791).</P><UL><LI>Add 3.3 µL of 3M Sodium Acetate (attached with Ethachinmate) into 100µL of DNA solution.</LI><LI>Add 1µL of Ethachinmate.</LI><LI>Vortex.</LI><LI>Add ethanol, 200-250µL.</LI><LI>Vortex.</LI><LI>Centrifuge at 12000xg for 5min.</LI><LI>Precipitation.</LI></UL><H2><SPAN class="mw-headline" id="Electrophoresis">Electrophoresis</SPAN></H2><UL><LI>Prepare 200mL of a 1.0% agarose solution:</LI><LI>Measure 2.0g agarose into a beaker.</LI><LI>Add 200mL 1xTAE buffer.</LI><LI>Wrap the top of the beaker with plastic wrap.</LI><LI>Punch a hole through the wrap with a pipette tip (To let out steam).</LI><LI>Dissolve the agarose by heating in microwave and swirling without boiling.</LI><LI>Allow the agarose to cool.</LI><LI>Pour the agarose solution into a gel tray on a gel maker.</LI><LI>If there is air bubbles, pushing them with a pipette tip.</LI><LI>Place comb in the maker.</LI><LI>Cover the maker with a plastic wrap.</LI><LI>Let stand for about 45min.</LI><LI>Remove the comb carefully.</LI><LI>Store in the Tupperware in the refrigerator.</LI><LI>Place the tray in electrophoresis chamber.</LI><LI>Cover the tray with 1xTAE buffer.</LI><LI>To prepare samples for electrophoresis, add 1µL of 10x Loading Buffer for every 9µL of DNA solution and mix well.</LI><LI>Load 6µL of the DNA solution per well.</LI><LI>Electrophoresis at 100V for about 30min until Loading Buffer have migrated approximately three-quarters of the gel.</LI><LI>Stain the gel in 0.5µg/mL ethidium bromide for 20-30min.</LI><LI>Place a plastic wrap on the transilluminator in the cabinet of Printgraph.</LI><LI>Place the gel on the transilluminator.</LI><LI>Turn on the transilluminator and confirm the position of the gel.</LI><LI>Shoot the picture.</LI><LI>Turn off the transilluminator.</LI><LI>Dispose of the gel.</LI></UL><H2><SPAN class="mw-headline" id="Gel_Extraction_and_PCR_Purification">Gel Extraction and PCR Purification</SPAN></H2><P>Use Promega Wizard SV Gel and PCR Clean-Up System.<B>Gel Extraction</B></P><UL><LI>Perform electrophoresis using an established protocol.</LI><LI>Weigh a 1.5 mlL microcentrifuge tube for each DNA fragment to be isolated and record the weight.</LI><LI>Visualize and photograph the DNA using a long-wavelength UV lamp and an intercalating dye such as ethidium bromide. To reduce nicking, irradiate the gel for the absolute minimum time possible. Excise the DNA fragment of interest in a minimal volume of agarose using a clean scalpel or razor blade.</LI><LI>Transfer the gel slice to the weighed microcentrifuge tube and record the weight. Subtract the weight of the empty tube from the total weight to obtain the weight of the gel slice.</LI><LI>Add Membrane Binding Solution at a ratio of 10 µL of solution per 10 mg of agarose gel slice.</LI><LI>Vortex the mixture and incubate at 52 °C for 10 minutes or until the gel slice is completely dissolved. Vortex the tube every few minutes to increase the rate of agarose gel melting. Centrifuge the tube briefly at room temperature to ensure the contents are at the bottom of the tube. Once the agarose gel is melted, the gel will not resolidify at room temperature.</LI><LI>Proceed to General.</LI></UL><P><B>PCR Purification</B></P><UL><LI>Amplify target of choice using standard amplification conditions.</LI><LI>Add an equal volume of Membrane Binding Solution to the PCR amplification.</LI><LI>Proceed to General.</LI></UL><P><B>General</B></P><UL><LI>Place one SV Minicolumn in a Collection Tube for each dissolved gel slice or PCR amplification.
</LI><LI>Transfer the dissolved gel mixture or prepared PCR product to the SV Minicolumn assembly and incubate for 1 minute at room temperature.</LI><LI>Centrifuge the SV Minicolumn assembly in a microcentrifuge at 16,000 g for 1 min. Remove the SV Minicolumn from the Spin Column assembly and discard the liquid in the Collection Tube. Return the SV Minicolumn to the Collection Tube.</LI><LI>Wash the column by adding 700 µL of Membrane Wash Solution, previously diluted with 95% ethanol, to the SV Minicolumn. Centrifuge the SV Minicolumn assembly for 1 min at 16,000 g.</LI><LI>Remove the SV Minicolumn assembly from the centrifuge, being careful not to wet the bottom of the column with the flowthrough. Empty the Collection Tube and recentrifuge the column assembly for 1 min with the microcentrifuge lid open (or off) to allow evaporation of any residual ethanol.</LI><LI>Carefully transfer the SV Minicolumn to a clean 1.5 mL microcentrifuge tube. Apply 30 mL of MilliQ directly to the center of the column without touching the membrane with with the pipette tip. Incubate at room temperature for 1 min. Centrifuge for 1 min at 16,000 g.</LI><LI>Discard the SV Minicolumn and store the microcentrifuge tube containing the eluted DNA at 4 °C or 20 °C</LI></UL><H2><SPAN class="mw-headline" id="qRT-PCR">qRT-PCR</SPAN></H2><P>Use QuantiTect SYBR green PCR kit Cat. No. 204143 by QIAGEN</P><UL><LI>Dilute primer to 1.5µM.</LI><LI>Dilute RT products.</LI><LI>Mix the following;</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>2x QuantiTect SYBR Green PCR Master Mix</TD><TD>22.5µL
</TD></TR><TR><TD>1/20xRT products</TD><TD>4.5µL
</TD></TR><TR><TD>MilliQ</TD><TD>9µL
</TD></TR><TR><TD>total</TD><TD>36µL
</TD></TR></TBODY></TABLE><UL><LI>Mix the reaction mix thoroughly, and dispense 36µL into 96 wells plate.</LI><LI>Add primer set 9µL.</LI><LI>Mix by inverting and voltex.</LI><LI>Dispense 10µL into 384 wells plate and centrifuge.</LI><LI>Let stand for 2min at 50°C and for 15min for 95°C.</LI><LI>40 cycles for 15sec at 95°C, for 30sec at 60°C, and for 1min at 72°C.</LI><LI>Let stand for 15sec at 95°C.</LI></UL><H2><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H2><P>Use ToYoBo Ligation High Ver.2.
</P><UL><LI>Mix the vector DNA and the insert DNA (the vector and the insert at 1 : 5-10).</LI><LI>Add a half volume of Ligation High Ver.2 to the DNA solution.</LI><LI>Incubate at 16 °C for 1 hour (or at 4 °C for overnight).</LI></UL><H2><SPAN class="mw-headline" id="Restriction_Enzyme_Digestion">Restriction Enzyme Digestion</SPAN></H2><P><B>Use EcoRI, XbaI, SpeI, PstI (TaKaRa).</B></P><UL><LI>Mix the following.
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Name</TH><TH>Volume
</TH></TR><TR><TD>Sample DNA</TD><TD>2 µg
</TD></TR><TR><TD>Restriction Enzyme</TD><TD>0.5µL
</TD></TR><TR><TD>10x Buffer</TD><TD>3 µL
</TD></TR><TR><TD>(If use XbaI,) 10x BSA</TD><TD>3 µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 30 µL
</TD></TR><TR><TD>total</TD><TD>30 µL
</TD></TR></TBODY></TABLE><UL><LI>Let stand for 1 hour at 37 °C.</LI></UL><P><B>Use EcoRI, XbaI, SpeI, PstI (Promega).</B></P><UL><LI>Mix the following.
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>Name</TH><TH>Volume
</TH></TR><TR><TD>Sample DNA</TD><TD>2 µg
</TD></TR><TR><TD>Restriction Enzyme</TD><TD>0.5µL
</TD></TR><TR><TD>10x Buffer</TD><TD>3 µL
</TD></TR><TR><TD>100x BSA</TD><TD>3 µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 30 µL
</TD></TR><TR><TD>total</TD><TD>30 µL
</TD></TR></TBODY></TABLE><UL><LI>Let stand for 1 hour at 37 °C.</LI></UL><H2><SPAN class="mw-headline" id="Media">Media</SPAN></H2><P><B>M9 medium</B></P><UL><LI>Stir Na2HPO4 6 g, KH2PO4 3 g, NaCl 0.5 g and NH4Cl 1 g with water 500 mL.</LI><LI>If you make M9 plates, add agar 15 g.</LI><LI>After autoclave, add 1 mL filter sterilized 1 M MgSO4, 5.6 mL 2 M glucose, 0.1 mL 1 M CaCl2, and 1 mL 1 % thiamine-HCl.</LI></UL><P>If you need, add 10 mL filter sterilized 20 % casamino acid.<B>LB medium</B></P><UL><LI>Stir BactoTryptone 2 g, Bacto-yeast extract 1 g, NaCl 1 g and 1M NaOH 200 µL with water 200mL.</LI><LI>If you make LB plates, add agar 2 g.</LI><LI>Autoclave.</LI></UL><P><B>SOB medium</B></P><UL><LI>Stir BactoTryptone 20 g and Bacto-yeast extract 5 g with water.</LI><LI>Add 2 mL 5 M NaCl and 840 µL 3 M KCl and add water up to 1 L.</LI><LI>After autoclave, add 10 mL filter sterilized 1 M MgSO4 and 1 M MgCl2.</LI></UL><P><B>SOC medium</B></P><UL><LI>Add 2 M glucose 1 mL to 100 mL SOB.</LI></UL><P><B>Plusgrow Medium</B></P><UL><LI>Stir plusgrow 20 g and with water 500 mL.</LI><LI>Autoclave.</LI></UL><H2><SPAN class="mw-headline" id="Buffer_TB">Buffer TB</SPAN></H2><UL><LI>Stir 0.6g PIPES and 30mg CaCl2 with 10 mL water.</LI><LI>Add 2.5mL 2M KCl.</LI><LI>Add KOH and adjust pH 6.7.</LI><LI>Add 0.218g MnCl2・4H2O.</LI><LI>Add water up to 20 mL.</LI><LI>Filter sterilize.</LI></UL><H2><SPAN class="mw-headline" id="Solubilization_of_Antibiotics">Solubilization of Antibiotics</SPAN></H2><UL><LI>Mix the following (Final concentration is 50mg/mL).</LI></UL><P><B>Ampicillin</B></P><TABLE class="wikitable"><TBODY><TR><TD>Ampicillin</TD><TD>1.0g
</TD></TR><TR><TD>MilliQ</TD><TD>20mL
</TD></TR></TBODY></TABLE><P><B>Kanamycin</B></P><TABLE class="wikitable"><TBODY><TR><TD>Kanamycin</TD><TD>0.5g
</TD></TR><TR><TD>MilliQ</TD><TD>10mL
</TD></TR></TBODY></TABLE><P><B>Chloramphenicol</B></P><TABLE class="wikitable"><TBODY><TR><TD>Chloramphenicol</TD><TD>0.5g
</TD></TR><TR><TD>99.5% EtOH</TD><TD>10mL
</TD></TR></TBODY></TABLE><UL><LI>Dispense 1.1mL of the solution into 1.5mL tubes.</LI><LI>Store in the -20°C freezer.</LI></UL><H2><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H2><UL><LI>Unfreeze conpetent cells on ice.</LI><LI>Dry a plate by letting the plate upside down and partly open in incubator.</LI><LI>Add 1~20µL DNA solution and 10~50µL competent cells to 1.5mL tube, let stand for 30min on ice. If few colonies are observed, increase the amount of the competent cells or DNA, but make the amount of DNA not to get over that of the competent cells.</LI><LI>Heatshock for 45s at 42°C.</LI><LI>Let stand for 2min on ice.</LI><LI>Culture for 1h in preculture medium (LB or SOC medium), and plate by using spreader. Do not heat spreader too much because <I>E.coli</I> will dead for heat.</LI><LI>Culture overnight at 37°C.</LI></UL><H2><SPAN class="mw-headline" id="Sequence">Sequence</SPAN></H2><P>Use Big Dye Terminator 3.1（ABI）</P><UL><LI>Mix the following</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>5xBuffer</TD><TD>1.75µL
</TD></TR><TR><TD>Primer (3.2µM)</TD><TD>0.5µL
</TD></TR><TR><TD>Template Plasmid</TD><TD>400ng
</TD></TR><TR><TD>Big Dye Terminator 3.1</TD><TD>0.5µL
</TD></TR><TR><TD>MilliQ</TD><TD>up to 10.5µL
</TD></TR></TBODY></TABLE><UL><LI>Let stand for 1min at 96°C.</LI><LI>30 cycles for 5s at 98°C, for 5s 50°C, and for 4min at 68°C.</LI><LI>Add 25µL 100% ethanol and 1µL NAOAC</LI></UL><H2><SPAN class="mw-headline" id="Golden_Gate_Assembly">Golden Gate Assembly</SPAN></H2><UL><LI>Add 100 ng of the linearized vector backbone and equimolar amounts of the other assembly pieces to a 15 µl total volume assembly reaction mixture as follows:
</LI></UL><TABLE class="wikitable"><TBODY><TR><TH>linearized vector backbone</TH><TH>each additional assembly piece</TH><TH>10x NEB T4 Buffer</TH><TH>100x BSA</TH><TH>BsaI</TH><TH>NEB T4 Ligase, 2 million cohesive end units / mL</TH><TH>dH2O
</TH></TR><TR><TD>100 ng</TD><TD>to equimolar with backbone</TD><TD>1.5 µl</TD><TD>0.15 µl</TD><TD>1µl</TD><TD>1 µl</TD><TD>to 15µl
</TD></TR></TBODY></TABLE><DL><DD>NOTE: It is essential to use a High Concentration Ligase
</DD><DD>BsaI is only 10% active at 37 °C without the addition of BSA.
</DD></DL><UL><LI>Perform the assembly reaction in a thermocycler as follows:
</LI></UL><P>either (following Engler 2009):
3 min @ 37 °C }
4 min @ 16 °C } 25 cycles
5 min @ 50 °C }
5 min @ 80 °C }    1 cycle</P><UL><LI>Transform 5 µl of the assembly reaction into 100 µl of competent <I>E. coli</I> and/or run a diagnostic agarose gel to check for successful assembly.
</LI></UL><H2><SPAN class="mw-headline" id="qRT-PCR_2">qRT-PCR</SPAN></H2><P>Use QuantiTect SYBR green PCR kit Cat. No. 204143 by QIAGEN</P><UL><LI>Dilute primer to 1.5μM.</LI><LI>Dilute RT products.</LI><LI>Mix the following;</LI></UL><TABLE class="wikitable"><TBODY><TR><TD>2x QuantiTect SYBR Green PCR Master Mix</TD><TD>22.5μL
</TD></TR><TR><TD>1/20xRT products</TD><TD>4.5μL
</TD></TR><TR><TD>MilliQ</TD><TD>9μL
</TD></TR><TR><TD>total</TD><TD>36μL
</TD></TR></TBODY></TABLE><UL><LI>Mix the reaction mix thoroughly, and dispense 36μL into 96 wells plate.</LI><LI>Add primer set 9μL.</LI><LI>Mix by inverting and voltex.</LI><LI>Dispense 10μL into 384 wells plate and centrifuge.</LI><LI>Let stand for 2min at 50°C and for 15min for 95°C.</LI><LI>40 cycles for 15sec at 95°C, for 30sec at 60°C, and for 1min at 72°C.</LI><LI>Let stand for 15sec at 95°C.</LI></UL></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Kyoto/Material_and_method">http://2013.igem.org/Team:Kyoto/Material_and_method</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Kyoto/Material_and_method" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Kyoto/Material_and_method" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Kyoto/Material_and_method&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Kyoto/Material_and_method&amp;oldid=317949" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>