<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_OLS_Canmore_Composite_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:OLS Canmore/Composite Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="ols_menubar"><UL id="ols_accordion" class="ols_accordion"><LI class="ols_menu_item ols_menubtn"><A href="https://2016.igem.org/Team:OLS_Canmore">HOME </A></LI><LI class="ols_menu_item ols_menubtn"><A href="#"> TEAM </A><DIV class="ols_submenu"><A href=" https://2016.igem.org/Team:OLS_Canmore/Team"> Team   </A><A href="https://2016.igem.org/Team:OLS_Canmore/Attributions"> Attributions </A></DIV></LI><LI class="ols_menu_item ols_menubtn"><A href="#"> PROJECT </A><DIV class="ols_submenu"><A href="https://2016.igem.org/Team:OLS_Canmore/Description"> Description </A><A href="https://2016.igem.org/Team:OLS_Canmore/Design"> Design </A><A href="https://2016.igem.org/Team:OLS_Canmore/Experiments"> Experiments </A><A href="https://2016.igem.org/Team:OLS_Canmore/Proof"> Proof of Concept </A><A href="https://2016.igem.org/Team:OLS_Canmore/Demonstrate"> Demonstrate </A><A href="https://2016.igem.org/Team:OLS_Canmore/Results"> Results </A><A href="https://2016.igem.org/Team:OLS_Canmore/Notebook"> Notebook </A></DIV></LI><LI class="ols_menu_item ols_menubtn"><A href="#"> PARTS </A><DIV class="ols_submenu"><A href="https://2016.igem.org/Team:OLS_Canmore/Parts">Parts </A><A href="https://2016.igem.org/Team:OLS_Canmore/Basic_Part"> Basic Parts </A><A href="https://2016.igem.org/Team:OLS_Canmore/Composite_Part"> Composite Parts </A></DIV></LI><LI class="ols_menu_item ols_menubtn"><A href="https://2016.igem.org/Team:OLS_Canmore/Safety"> SAFETY </A></LI><LI class="ols_menu_item ols_menubtn"><A href="#">PRACTICES</A><DIV class="ols_submenu"><A href="https://2016.igem.org/Team:OLS_Canmore/Human_Practices"> Human Practices </A><A href="https://2016.igem.org/Team:OLS_Canmore/Integrated_Practices"> Integrated Practices </A><A href="https://2016.igem.org/Team:OLS_Canmore/Engagement"> Engagement </A><A href="https://2016.igem.org/Team:OLS_Canmore/Collaborations"> Collaborations </A></DIV></LI><LI class="ols_menu_item ols_menubtn"><A href="#">AWARDS</A><DIV class="ols_submenu"><A href="https://2016.igem.org/Team:OLS_Canmore/HP/Silver"> Silver </A><A href="https://2016.igem.org/Team:OLS_Canmore/HP/Gold"> Gold </A></DIV></LI></UL></DIV><DIV class="column full_size"><H3 class="ols_header" id="ols_compositeparts"> COMPOSITE PARTS </H3><P class="ols_bodytext">
All the composite parts are composed of the same skeleton: A promoter, an R.B.S., a keratinase gene, and a double terminator.  There is a composite part that uses the KerA gene, and one that uses the KerUS gene.<B> Part name: Keratinase A Expression Cassette (BBa_K1717000) </B><I> Creator: </I> Our Lady of the Snows Catholic Academy 2016 <I> Short description: </I> Keratinase A (KerA) under control of a Lacl promoter <I> Long description: </I> Keratinases are proteolytic enzymes that are capable of breaking the disulfide bonds in many different keratin substrates, such as hair and feathers. KerA is a basic part comprised of a synthesized KerA protein coding sequence, optimized for expression in <I>E. coli</I> and is intended for the breakdown of the beta-keratin, more commonly present in feathers. 

KerA, when successfully expressed in cells, can be tested for enzymatic effectivity by looking for zones of clearing when plated on skim milk agar plates, as well observing the degradation of keratin substrates when grown with cells. <I> Design considerations: </I> Keratinases are isolated from a number of bacteria and fungi species, most notably those in the Bacillus genera. The bacterial chassis we use is a strain of <I>E. coli</I> called K-12. Like all E. coli, this strain is gram-negative. Building off of the work of previous teams, such as Chicago, Sheffield, we deduced that the signal peptides that originated from gram-positive organisms could be causing the poor enzyme secretion in these teams projects. These teams either left these gram positive signals in their sequence, or removed it entirely. However, our team decided to replace the native signal peptide sequence and replaced it with one specifically optimized for secretion in gram-negative <I>E. coli</I> (pelB, with the sequence taken from part Part BBa_J32015). This part would hopefully secrete our enzyme into the periplasm, where it could escape into the media extracellularly once the promoter was induced.  DNA sequences were compared using Clustal to ensure protein coding sequences were not altered, as well as to ensure that start and stop codons were correct. NEBCutter V2.0 was used to check for illegal cut sites. One EcoRI site and one PstI site were identified, and silent mutations were introduced in order to remove them. <I> Source: </I> The sequence for the KerA coding region was taken from the <A href="http://www.uniprot.org/uniprot/Q53521" class="ols_hyperlink"> Uniprot gene/protein sequence bank. </A><B> Part name: Keratinase US Expression Cassette (BBa_K1717173) </B><I> Creator: </I> Our Lady of the Snows Catholic Academy 2016 <I> Short description: </I> Coding sequence for Keratinase US (KerUS) <I> Long description: </I> Keratinases are proteolytic enzymes that are capable of breaking the disulfide bonds in many different keratin substrates, such as hair and feathers. Kerus is a basic part comprised of a synthesized Kerus protein coding sequence, optimized for expression in <I>E. coli</I> and is most active in breaking down alpha-keratin, more commonly found in hair. <I> Design considerations: </I> Keratinases are isolated from a number of bacteria and fungi species, most notably those in the Bacillus genera. The bacterial chassis we use is a strain of <I>E. coli</I> called K-12. Like all <I>E. coli</I>, this strain is gram-negative. Building off of the work of previous teams, such as Chicago, Sheffield, we deduced that the signal peptides that originated from gram-positive organisms could be causing the poor enzyme secretion in these teams projects. These teams either left these gram-positive signals in their sequence, or removed it entirely. However, our team decided to replace the native signal peptide sequence and replaced it with one specifically optimized for secretion in gram-negative <I>E. coli</I> (pelB, with the sequence taken from part Part BBa_J32015). This part would hopefully secrete our enzyme into the periplasm, where it could escape into the media extracellularly once the promoter was induced.  DNA sequences were compared using Clustal to ensure protein coding sequences were not altered, as well as to ensure that start and stop codons were correct. NEBCutter V2.0 was used to check for illegal cut sites.  The gene contained no illegal cut sites.<I> Source: </I> The sequence for the KERUS coding region was taken from the <A href="http://www.uniprot.org/uniprot/A0A059PWB6" class="ols_hyperlink"> Uniprot gene/protein sequence bank. </A></P><P style="text-align: center;" class="ols_bodytext"> 
Contact us at: 
https://www.facebook.com/OLeSsence/ 
@igem_canmore  
larvisais@redeemer.ab.ca
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>