<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lethbridge_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lethbridge/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Lethbridge iGEM 2016</TITLE></P><DIV id="content"><NAV class="blue-grey darken-4" role="navigation" style="margin-top: 15px;"><DIV class="nav-wrapper"><A href="https://2016.igem.org/Team:Lethbridge" class="brand-logo left" style="width:auto; left:40px;"> Lethbridge iGEM</A><UL class="right hide-on-med-and-down" style="list-style-image: none;"><LI><A class="dropdown-button" href="#!" data-activates="dropdown1" style="display: block; padding-top: 19px; padding-bottom: 19px">Project</A></LI><UL id="dropdown1" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Description" style="color:#616161;">Description</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Design" style="color:#616161;">Design</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Experiments" style="color:#616161;">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Database" style="color:#616161;">Database</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Notebook" style="color:#616161;">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Parts" style="color:#616161;">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Safety" style="color:#616161;">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/RNAi" style="color:#616161;">RNAi</A></LI></UL><LI><A class="dropdown-button" href="#!" data-activates="dropdown2" style="display: block; padding-top: 19px; padding-bottom: 19px">Practices</A></LI><UL id="dropdown2" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Integrated_Practices" style="color:#616161;">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#intro" style="color:#616161;">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#interviews" style="color:#616161;">Interviews</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#litrev" style="color:#616161;">Literature Review</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#ridealong" style="color:#616161;">Ridealongs</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#protocol" style="color:#616161;">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#sampling" style="color:#616161;">Sampling</A></LI></UL><LI><A href="https://2016.igem.org/Team:Lethbridge/Results" style="display: block; padding-top: 19px; padding-bottom: 19px">Results</A></LI><LI><A class="dropdown-button" href="#!" data-activates="dropdown3" style="display: block; padding-top: 19px; padding-bottom: 19px">Team</A></LI><UL id="dropdown3" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Team" style="color:#616161;">Members</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Collaborations" style="color:#616161;">Collab</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Attributions" style="color:#616161;">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Sponsors" style="color:#616161;">Sponsors</A></LI></UL><LI><A href="https://2016.igem.org/Team:Lethbridge/Achievements" style="display: block; padding-top: 19px; padding-bottom: 19px">Achievements</A></LI></UL><A href="#" data-activates="nav-mobile" class="button-collapse"><I class="material-icons">menu</I></A><UL id="nav-mobile" class="side-nav" style="margin-top:20px; margin-left:0px; margin-right:0px; margin-bottom:0px;"><LI class="bold"><A class="collapsible-header waves-effect waves-blue">Project</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Database">Database</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/RNAi">RNAi</A></LI></UL></DIV><LI class="bold"><A class="collapsible-header waves-effect waves-red">Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#intro">Intro</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#interviews">Interviews</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#litrev">Literature Review</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#ridealong">Ridealongs</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#sampling">Sampling</A></LI></NAV></DIV><LI class="bold"><A class="collapsible-header waves-effect" href="https://2016.igem.org/Team:Lethbridge/Results">Results</A></LI><LI class="bold"><A class="collapsible-header waves-effect">Team</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Sponsors">Sponsors</A></LI></DIV><LI class="bold"><A class="collapsible-header waves-effect" href="https://2016.igem.org/Team:Lethbridge/Achievements">Achievements</A></LI></DIV><MAIN><DIV class="row"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H2 class="thin white-text">Experiments</H2></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="one"><DIV class="col s12 blue-grey darken-2"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H3 class="thin white-text">Sampling Protocol</H3><P class="grey-text text-lighten-4">Prior to sampling, kits were prepared in house under sterile conditions. Each kit consisted of:</P><OL class="white-text"><LI> Tweezers </LI><LI> Large KimWipes</LI><LI> Nitrile Gloves </LI><LI> 70% ethanol </LI><LI> 12 micro centrifuge tubes (4 per ambulance) </LI><LI> 15 mm filter paper (1 per tube) </LI></OL><P class="grey-text text-lighten-4">In the biological safety cabinet, 500 µL of 70% ethanol was pipetted into 2mL micro centrifuge tubes. Filter paper was carefully added to each tube and the tube was labelled to indicate the ambulance sampled, sample location and sampling week. </P><P class="grey-text text-lighten-4">Sampling took place at the main Fire Station in downtown Lethbridge over the course of one month. Ambulances 1, 9 and 10 were sampled every Friday at 7:00pm and were on call at this time. Extra precautions were put in place to maintain sterility and avoid sample contamination. Nitrile gloves were worn by team members taking samples and 70% ethanol was used to sterilize gloves and tweezers, which were used to remove the filter paper from the micro centrifuge tubes after each location was sampled. Large KimWipes were used to dry the gloves before the next location was sampled.</P><P class="grey-text text-lighten-4">The first area sampled was the SPO2 finger monitor located in the heart rate monitor kit. The filter paper was removed from the micro centrifuge tube to swipe the bottom part of the finger monitor where the patients finger would be inserted. Dimensions of the area swiped was approximately 2 cm x 2 cm. The filter paper was then carefully placed back in the centrifuge tube. Gloves and tweezers were cleaned with ethanol between samples. This process was repeated for sampling of the rear interior door handle and bottom of the soft kit. The sampling area was similar in size for each location to maintain consistency throughout the sampling protocol. Following data collection, the samples were stored at -20०C until further processing. </P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="two"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H3 class="thin white-text">DNA Extraction Protocol</H3><P class="grey-text text-lighten-4">DNA extraction from sample filter paper. Adapted from [1]</P><OL class="white-text"><LI> Ethanol evaporation for approximately 2 hours on the rotary evaporator on high heat, under vacuum and high speed </LI><LI> Add 400 µL of alkaline lysis buffer (200 mM KOH, 50 mM DTT), mix by inverting several times </LI><LI> Incubate on heat block at 65 °C for 10 minutes </LI><LI> Centrifuge at 17 000 x G for 1 minute </LI><LI> Pipette liquid volume (as much as possible) into new 1.5 mL micro centrifuge tube (~300 µL) </LI><LI> Add 1/10 of the volume (~30 µL) of 3 M sodium acetate (pH 5.3) to each tube </LI><LI> Add 5 µL of 0.5% LPA (linear polyacrylamide) carrier to each tube </LI><LI> Add 750 µL of cold 100% ethanol to each tube, mix by inverting and place tubes on ice for approximately 10 minutes </LI><LI> Centrifuge at 17 000 x G for 5 minutes </LI><LI> Remove supernatant </LI><LI> Wash with 500 µL of 70% ethanol and mix gently. Pulse centrifuge and remove supernatant </LI><LI> Repeat wash step (step 11) </LI><LI>Vacuum evaporate the ethanol under high heat for 10 minutes </LI><LI> Resuspend in 20 µL of filtered dd H2O </LI></OL></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="three"><DIV class="col s12 blue-grey darken-2"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H3 class="thin white-text">DNA Amplification</H3><P class="grey-text text-lighten-4">Using universal 16S/18S ribosomal RNA gene primers to amplify [2,3]</P><P class="grey-text text-lighten-4">For a 12 µL PCR reaction:</P><UL class="white-text"><LI> 6 µL 2x Phusion MM, HF buffer</LI><LI> 0.6 µL 16S/18S Forward Primer, 10 µM </LI><LI> 0.6 µL 16S/18S Reverse Primer, 10 µM </LI><LI> 2 µL EMS sample/control DNA (10ng/ µL) </LI><LI> 2.8 µL dd H2O </LI></UL><P class="grey-text text-lighten-4">Thermocycler conditions: 98 °C initial denaturation for 2 min, [98 °C denaturation for 20s, 64/56 °C annealing for 15 sec, 72 °C extension for 30 sec] 30 cycles, 72 °C final extension for 2 min, 4 °C hold</P><P class="grey-text text-lighten-4">Followed by PCR cleanup using BioBasic Protocols, elute in 30 µL of dd H2O</P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="four"><DIV class="col s12 blue-grey darken-2"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H3 class="thin white-text">DNA Barcoding and Adapter Ligation</H3><P class="grey-text text-lighten-4">Adapted from [4] using the PCR barcoding kit from Oxford Nanopore Technologies</P><OL class="white-text"><LI> Add 7 µL of Ultra II End Prep Buffer, 3 µL of Ultra II End Prep Enzyme mix and 5 µL of DNA CS 3.6kb (positive control) or dd H2O to each sample from the DNA amplification </LI><LI> Add 60 µL resuspended AMPure XP beads at room temperature, incubate on rotator for 5 minutes, spin down and pellet on magnet. Discard supernatant. </LI><LI> Keep on magnet, wash 2x with 200 µL of 70% ethanol, do not disturb pellet </LI><LI> Pulse centrifuge, replace on magnet, pipette off residual wash. Briefly allow to dry. </LI><LI> Resuspend pellet in 31 µL of dd H2O, incubate at room temperature for 2 minutes </LI><LI> Pellet beads on a magnet, remove eluate of End-Prepped DNA and transfer to new 1.5 mL micro centrifuge tube (measure DNA concentration of DNA using the BioDrop ~ 700ng) </LI><LI> Check NEB Blunt/TA Ligase Master Mix is clear of any precipitate. </LI><LI> For ligation reaction, add 20 µL of BCA to 30 µL of End-Prepped DNA.  Remove 5 µL of the BCA/End-Prepped DNA mix and add 5 µL of ligase master mix </LI><LI> Purify DNA following BioBasic spin column protocols. Elute in 30 µL of dd H2O (measure DNA concentration using the BioDrop ~10ng/ µL)</LI><LI> Set up barcoding PCR reaction: 24 µL ligated sample, 1 µL barcode, 25 µL, Phusion Master Mix </LI><LI> Thermocycler conditions: 98 °C initial denaturation for 2 min, [98 °C denaturation for 15s, 64 °C annealing for 15s, 72 °C extension for 1 min] 25 cycles, 72 °C final extension for 2 min, 4 °C hold </LI><LI> PCR cleanup following BioBasic spin column protocol. Elute in 30 µL of dd H2O (measure DNA concentration using the BioDrop) </LI><LI> Prepare 1 µg of pooled barcoded library in 45 µL dd H2O </LI><LI> Add 7 µL of Ultra II End Prep Buffer, 3 µL Ultra II End Prep Enzyme mix, 5 µL of DNA CS 3.6 kb (positive control). Mix by inversion and pulse spin down. Incubate at 20 °C for 5 minutes and 65 °C for 5 minutes </LI><LI> Add 60 µL resuspended AMPure XP beads at room temperature, incubate on rotator for 5 minutes, spin down and pellet on magnet. Discard the supernatant </LI><LI> Keep on magnet, wash 2x with 200 µL 70% ethanol, do not disturb pellet </LI><LI> Pulse spin down, replace on magnet, pipette off residual wash, briefly allow to dry. </LI><LI> Resuspend pellet in 31 µL of dd H2O, incubate at room temperature for 2 minutes </LI><LI> Pellet beads on a magnet, remove eluate of End-Prepped DNA and transfer to fresh 1.5 mL micro centrifuge tube </LI><LI> Check NEB Blunt/TA Ligase Master Mix is clear of any precipitate </LI><LI> For ligation reaction, add 8 µL of dd H2O, 10 µL Adapter Mix, 2 µL HPA, 50 µL NEB Blunt/TA Master mix, incubate at room temperature for 10 minutes </LI><LI> Add 1 µL of HPT, mix by inversion and pulse spin down. Incubate at room temperature for 10 minutes. While incubating prepare MyOne C1 beads for purification of the adapted and tethered DNA </LI><LI> Vortex MyOne C1 beads to resuspend, transfer 50 µL of beads to 1.5 µL micro centrifuge tube, pellet on magnet, wash beads 2x with 100 µL BBB vortexing to resuspend, pellet on magnet, pipette off and discard supernatant, resuspend washed beads in 100 µL to be used for adapter ligation cleanup </LI><LI> Purify adapted DNA: add 100 µL washed MyOne C1 beads to the adapted, tethered DNA reaction on a rotator, Incubate at room temperature for 5 minutes </LI><LI> Place on magnet, pellet and pipette of the supernatant, wash beads with library bound 2x with 150 µL BBB, pipette to resuspend, pellet on magnet, pipette off any residual BBB </LI><LI> Elute the adapted library: resuspend pelleted beads in 25 µL ELB and incubate at 37 °C for 10 minutes, pellet beads on magnet, remove eluate and transfer to new 1.5 mL micro centrifuge tube (measure DNA concentration using the BioDrop) </LI></OL><P class="grey-text text-lighten-4">Thermocycler conditions: 98 °C initial denaturation for 2 min, [98 °C denaturation for 20s, 64/56 °C annealing for 15 sec, 72 °C extension for 30 sec] 30 cycles, 72 °C final extension for 2 min, 4 °C hold</P><P class="grey-text text-lighten-4">Followed by PCR cleanup using BioBasic Protocols, elute in 30 µL of dd H2O</P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="five"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H3 class="thin white-text">MinION Sequencing Protocol</H3><P class="grey-text text-lighten-4">Adapted from [4]</P><OL class="white-text"><LI> Assemble MinION and MinION flow cell </LI><LI> Set up MinKNOW to run the Platform QC – name the run and start the protocol script </LI><LI> Allow the script to run to completion and the number of active pores are reported </LI><LI> Prime the flow cell for the library to be loaded when the library preparation is complete </LI><LI> Prepare priming buffer: 500 µL of NFW and 500 µL RBF1 </LI><LI> Prime the flow cell: Open the sample port and using a 1000 µL pipette draw back the buffer in the flow cell to make sure there is a continuous buffer flow from the sample port across the sensor array </LI><LI> Load 500 µL of the priming buffer using a vertical 1000 µL pipette and tip in a continuous flow avoiding introducing air bubbles or disturbing the sensor array. Wait 10 minutes, then repeat the loading and wait another 10 minutes </LI><LI> Load the prepared library: mix the pre sequencing mix by inversion 10x, briefly spin down the library </LI><LI> Add the reagents in the following order: 75 µL RBF1, 63 µL of dd H2O and 12 µL of the pre sequencing mix kept on ice </LI><LI> Load 150 µL of the library loading mix via the sample port, close the cover and the MinION lid </LI><LI> Start the sequencing script in MinKNOW and the workflow in the Metrichor Agent </LI><LI> Return to MinKNOW, name the run, select the MAP_48Hr_Sequencing_Run_SQK_MAP006.py and start using the start in the MinKNOW dialogue box. </LI><LI> Open the Desktop Agent, select the latest version of the barcoding plus 2D basecalling, run the workflow and monitor the workflow using the visualization options in details </LI><LI> MinKNOW will report the number of pores available for the sequencing before data collection begins. These may differ from those reported in the Platform QC </LI><LI> Allow the protocol to proceed until MinKNOW reports Finished Successfully. Use the stop in the control panel to finish the protocol </LI><LI> Quit the Desktop Agent, close down MinKNOW and disconnect the MinION </LI></OL></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="six"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H3 class="thin white-text">Sequencing Data Analysis</H3><P class="grey-text text-lighten-4">Following the flow cell run, the raw data was obtained as fast5 files using MinKNOW software version 1.0.8.0 (Oxford Nanopore Technologies), The electric signals in the raw files were converted to sequence (base calling) using 2D Basecalling workflow in Metrichor Agent version 2.40.17.  Basecalled data passing quality control and filtering were downloaded and basic statistical analysis was carried out using poretools [5] program. The reads outputted as fasta files were processed for metagenomics analysis using mothur [6] and Taxonomer [7] </P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="seven"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H3 class="thin white-text">Oligo Assembly</H3><P class="grey-text text-lighten-4">1. Oligo Annealing</P><P class="grey-text text-lighten-4">Combine the Following in a Microfuge tube: </P><UL class="white-text"><LI> 2µL Forward oligo (100 µM) </LI><LI> 2µL Reverse oligo (100 µM) </LI><LI> 46µL MilliQ dH2O </LI></UL><P class="grey-text text-lighten-4">Heat Oligo mixture to 95 °C on a heat block then allow it to slowly cool down to room temperature</P><P class="grey-text text-lighten-4">2. Oligo Extension using T4 DNA polymerase </P><P class="grey-text text-lighten-4">Combine the Following in a Microfuge tube:</P><UL class="white-text"><LI> 43µL Annealed oligos (~4 µM) </LI><LI> 5µL 10X NEB Buffer 2.1 </LI><LI> 1µL 10 mM dNTPs </LI><LI> T4 DNA polymerase </LI></UL><P class="grey-text text-lighten-4">Incubate at 12 °C for 15 mins and then transfer to ice, clean up on a spin column and elute in 40 µL of MilliQ dH2O. Analyze products on a 1% TAE agarose gel or an 8% TBE Native PAGE. </P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="eight"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H3 class="thin white-text">Overlap Extension PCR Protocol</H3><P class="grey-text text-lighten-4">Reaction Mixture: </P><UL class="white-text"><LI> 25µL 2X Phusion MM, HF Buffer </LI><LI> 5µL Extension Product #1 </LI><LI> 5µL Extension Product #2 </LI><LI> 15µL MilliQ dH2O</LI></UL><P class="grey-text text-lighten-4">Thermocycler conditions: 98 °C initial denaturation for 10 sec, [98 °C denaturation for 10 sec, 67 °C annealing for 10 sec, 72 °C extension for 40 sec] 20 cycles, 72 °C final extension for 1 min, 4 °C hold </P><P class="grey-text text-lighten-4">After running the reaction analyze products on a 1% TAE agarose gel or an 8% native PAGE </P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="nine"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H3 class="thin white-text">SLIC</H3><P class="grey-text text-lighten-4">Adapted from [8] </P><P class="grey-text text-lighten-4">Step 1: Generate linear DNA fragments via polymerase chain reaction (PCR) </P><OL class="white-text"><LI> Design forward and reverse PCR primers that bind to the template DNA (~18-25 nt) with 20-30 nt 5’ extensions that overlap adjacent DNA fragments </LI><LI> For each DNA fragment to be assembled, set up individual PCR reactions consisting of: 25 μL 2 X Phusion DNA Pol Master Mix, HF buffer; 2.5 μL Forward PCR primer, 10 μM; 2.5 μL Reverse PCR primer, 10 μM;  X μL* Template DNA (10 – 100 ng); Y μL* MilliQ ddH2O (50 μL Total reaction volume) </LI><P class="grey-text text-lighten-4">* Note: adjust the volume of template DNA/MilliQ ddH2O to make up 50 μL total volume </P><LI>PCR thermocycling conditions (annealing temp, extension time, etc.*) will need to be determined for individual reactions, however use the following guidelines: Initial denaturation 98 °C, 30 sec Denaturation, 98 °C 15 sec Annealing, X °C 20 sec Extension, 72 °C Y sec (~30 sec/kb) Final extension, 72 °C 2 – 5 min Hold, 4 °C Indefinite </LI><P class="grey-text text-lighten-4">*Note: for each primer pair, determine the optimal annealing temperature using the Thermo Scientific website Tm calculator, with Phusion DNA Polymerase settings. </P><LI> Check PCR products on an agarose gel. PCR products should be a single band that runs at the expected size. PCR products that have additional non-specific amplicons should be gel-purified to isolate the desired PCR band and possibly re-amplified </LI><LI> Purify PCR products using BioBasic spin columns and the PCR cleanup protocol, according to the manufacturer’s instructions. Elute PCR products in 40 μL of nuclease-free MilliQ ddH2O and quantitate the DNA using the BioDrop apparatus </LI></OL><P class="grey-text text-lighten-4">Step 2: Chew-back DNA 3’ ends using T4 DNA Polymerase </P><OL class="white-text"><LI> For each DNA fragment, assemble the following components in a 1.5 mL microfuge tube in the following order: </LI><P class="grey-text text-lighten-4">2 μL 10 X NEBuffer 2.1; X μL DNA fragment (500 – 1000 ng total); Y μL MilliQ ddH2O; 1 μL T4 DNA Polymerase, 0.3 U/μL (Make dilution in 1 X NEBuffer 2.1); 1 μL Dpn I (10 U/μL). 20 μL Total reaction volume.
                <LI> Incubate chew-back reactions for 30 minutes at 37 °C. </LI><LI> Arrest chew-back by adding 2 uL of 10 mM dCTP and then place tubes on ice. </LI></P></OL><P class="grey-text text-lighten-4">Step 3: Assemble DNA fragments and transform into E. coli </P><OL class="white-text"><LI> Based on the amount of DNA added to the chew-back reaction and the length of DNA fragments, determine the amount of each chew-back reaction needed to make an e	quimolar mixture of each fragment that totals to at least 1 μg DNA (1,000 ηg DNA). For example, to assemble three DNA fragments of 7000 bp, 1500 bp and 900 bp at a concentration of 50 ng/uL each in chew-back reactions one would calculate (total volume = 26.8μL: </LI><TABLE><THEAD><TR><TH data-field="fragment">Fragment</TH><TH data-field="DNA size">DNA size</TH><TH data-field="rel size">Rel. Size</TH><TH data-field="rel size">Equiv. DNA Amt.</TH><TH data-field="rel size">Chew-back conc.</TH><TH data-field="rel size">Amt. chew-back</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>7000bp</TD><TD>1</TD><TD>1000ng</TD><TD>50 ng/μL</TD><TD>20μL</TD></TR><TR><TD>2</TD><TD>1500bp</TD><TD>0.21</TD><TD>210ng</TD><TD>50 ng/μL</TD><TD>4.2μL</TD></TR><TR><TD>3</TD><TD>900bp</TD><TD>0.13</TD><TD>130ng</TD><TD>50 ng/μL</TD><TD>2.6μL</TD></TR></TBODY></TABLE><LI> Incubate mixtures at 37°C for 10 minutes, then purify the DNA mixture on a BioBasic Spin column using the cleanup procedure an enzymatic reaction and elute the DNA in 40 μL MilliQ ddH2O. </LI><LI> Use 2 μL of the DNA mixture (~100 ηg) for transformation into E. coli high-efficiency chemically competent or electrocompetent cells. </LI></OL></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="ten"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H3 class="thin white-text">References</H3><P class="grey-text text-lighten-4">[1] Spits, C., et al., Nature Methods (2006) 1:4, 1966 - 1970 </P><P class="grey-text text-lighten-4">[2] Wang Y, Qian P-Y (2009) PLoS ONE 4:10, e7401 </P><P class="grey-text text-lighten-4">[3] Wang Y, et al., (2014) PLoS ONE 9:3, e90053. </P><P class="grey-text text-lighten-4">[4] Oxford Nanopore “PCR barcoding genomic DNA sequencing for the MinION device” protocol </P><P class="grey-text text-lighten-4">[5] Loman et al. Bioinformatics (2013) 30:23, 3399-3401. </P><P class="grey-text text-lighten-4">[6] Schloss et al., Applied and environmental microbiology (2009) 75:23 7537-7541. </P><P class="grey-text text-lighten-4">[7] Flygare et al., Genome Biology (2016) 17:111 </P><P class="grey-text text-lighten-4">[8] Li et al., Nature Methods (2007)  4, 25-256 </P></DIV></DIV></DIV></DIV></DIV></MAIN></DIV></DIV></DIV></BODY></HTML>