Team:Lethbridge/Results
Lethbridge iGEM 2016
Lethbridge iGEM
Project
Description
Design
Experiment
Database
Notebook
Parts
Safety
RNAi
Practices
Integrated Practices
Contact
Interviews
Literature Review
Ridealongs
Protocol
Sampling
Results
Team
Members
Collab
Attribution
Sponsors
Achievements
menu
Project
Description
Design
Experiments
Database
Notebook
Parts
Safety
RNAi
Practices
Integrated Practices
Intro
Interviews
Literature Review
Ridealongs
Protocol
Sampling
Results
Team
Members
Collaborations
Attribution
Sponsors
Achievements
Results
Microbiome
Antibodies
RNAiCare
Microbiome Analysis
Our group’s initial goal was to characterize the microbiome of emergency medical vehicles utilizing a next-generation sequencing approach with cutting-edge hardware and software from Oxford Nanopore Technologies. In order to execute this, we developed our own sampling pipeline as well as created a new method for amplifying and preparing DNA for sequencing from environmental samples. Our sample preparation pipeline was successful and we were able to successfully amplify and isolate ribosomal RNA from both prokaryotes and eukaryotes using universal gene primers.
Figure 1: 1% agarose gel demonstrating amplification of 16S and 18S rDNA from various samples obtained from emergency medical vehicles as well as control samples. A clear amplification is seen using these universal primers.
The samples were then barcoded, and specific adapters were ligated on to enable nanopore sequencing. The sequencing data confirmed that our approach isolated and amplified genomic DNA from a wide array of bacterial species. Included in this list of species are several pathogenic and opportunistic pathogenic bacteria, these data are outlined below.
Table I: List of bacterial species identified within emergency medical vehicles obtained utilizing next-generation sequencing.
The sequencing approach taken was unique and allowed for identification of species which are not readily culturable. Importantly, some bacterial species including those in the Legionella genus, Mycoplasma pneumoniae and Mycobacterium tuberculosis [1] may not be detected using routine bacterial culturing. However, utilizing next-generation sequencing, identification of these microbes in emergency medical vehicles and in the environment at large would be possible. Additionally, we were able to identify distinct bacterial species from our sampling locations not obtained in our technical control, indicating that the presence of these microbes is not an experimental artifact.
Figure 2: Illustration of species identified using Oxford Nanopore Technologies MinION. Organisms identified by 16S rDNA amplification and sequencing in samples from an SPO2 finger monitor (A), Interior Door Handle (B), and Soft Kits (C) were compared to a sampling control (D).
Single-Domain Antibody Development and Screening
The data obtained by sequencing allows first responders to identify areas of concern within their vehicles and monitor their cleaning habits over the course of several months. However, a more rapid approach is clearly required for day-to-day monitoring of cleanliness in emergency medical vehicles. In order to accomplish this, our team sought to use synthetic biology to evolve single-domain antibodies utilizing a modified bacterial-2-hybrid system. Our construction employed a two-plasmid design, with one plasmid harbouring both an RNA polymerase alpha subunit fused to a target of interest and fluorescent reporter constructs and the other containing a randomized library of single-domain antibody sequences fused to lambda cI.
Figure 3: Dual plasmid design for selection of novel single-domain antibodies. One plasmid contains lambda cI protein coding sequence fused to our generated single-domain antibody library. The second plasmid contains fluorescent reporter proteins as well as the alpha subunit of RNA polymerase fused to a target of interest. This facilitates the identification of tight antibody binding which is detected as a fluorescent signal.
Gene fragments were synthesized by Integrated DNA Technologies. Each was successfully amplified and inserted into pSB3K3 and pSB3C5. These plasmids were transformed into DH5α E. coli cells.Single domain antibody generation was informed by bioinformatics analysis and sequence alignment.We were able to identify common motifs within coding sequences for single-domain antibodies termed CDR1, CDR2, and CDR3. Each of these were synthesized by Integrated DNA Technologies and subsequently cloned into pSB3C5. All of these CDRs were introduced into an invariant single-domain antibody scaffold by overlap-extension-PCR. We were then able to overexpress the lambda cI fused to the CDR library. This overexpression is designed to facilitate the selection of a multitude of different single-domain antibody variants.
Figure 4: Library of CDR 1, 2, and 3 was successfully introduced into the single-domain antibody scaffold as illustrated by this 2% agarose gel.
Figure 5: Induced overexpression of lambda cI protein fused to single-domain antibody library. Upon introduction of IPTG to a final concentration of 1mM, increased expression of lambda cI is observed.
The CDRs confer the specificity of binding, and a large library of possible sequences facilitates panning for single-domain antibodies with specificity for any given antigen. Our library of cloned CDR1-3 variants were sequenced using the MinION fron Oxford Nanopore Technologies and the complexity of our library was analyzed. As illustrated, we have a complexity of upwards of 1011 possible variants.
Figure 6: The complexity of the single-domain antibody library as analyzed using Oxford Nanopore Technologies MinION. The library is hypothesized to contain over 1011 unique possible sequences which may convey different binding characteristics.
The single-domain antibody library was placed downstream of a lambda-cI coding sequence and utilized for selection of antibody binding to a target protein. Binding of the antibody to a target brings RNA polymerase alpha subunit in proximity to a reporter promoter. Transcription and subsequent translation of the fluorescent reporter is then assessed using fluorescence spectrometry and Fluorescence Activated Cell Sorting (FACS).
Figure 7: Ensemble measurement of Blue Fluorescent Protein (BFP) signal upon excitation with 402nm light. An emission maximum at 457nm was observed, indicating that BFP is being expressed and can be used as a suitable readout in future experimentation. Fluorescence was compared to a PBS blank (grey).
References
1. Fournier, P. E., Dubourg, G., & Raoult, D. (2014). Clinical detection and characterization of bacterial pathogens in the genomics era. Genome medicine, 6(11), 1.
