<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lethbridge_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lethbridge/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Lethbridge iGEM 2016</TITLE></P><DIV id="content"><NAV class="blue-grey darken-4" role="navigation" style="margin-top: 15px;"><DIV class="nav-wrapper"><A href="https://2016.igem.org/Team:Lethbridge" class="brand-logo left" style="width:auto; left:40px;"> Lethbridge iGEM</A><UL class="right hide-on-med-and-down" style="list-style-image: none;"><LI><A class="dropdown-button" href="#!" data-activates="dropdown1" style="display: block; padding-top: 19px; padding-bottom: 19px">Project</A></LI><UL id="dropdown1" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Description" style="color:#616161;">Description</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Design" style="color:#616161;">Design</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Experiments" style="color:#616161;">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Database" style="color:#616161;">Database</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Notebook" style="color:#616161;">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Parts" style="color:#616161;">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Safety" style="color:#616161;">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/RNAi" style="color:#616161;">RNAi</A></LI></UL><LI><A class="dropdown-button" href="#!" data-activates="dropdown2" style="display: block; padding-top: 19px; padding-bottom: 19px">Practices</A></LI><UL id="dropdown2" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Integrated_Practices" style="color:#616161;">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#intro" style="color:#78909c;">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#interviews" style="color:#78909c;">Interviews</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#litrev" style="color:#78909c;">Literature Review</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#ridealong" style="color:#ff8f00;">Ridealongs</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#protocol" style="color:#ff8f00;">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#sampling" style="color:#ff8f00;">Sampling</A></LI></UL><LI><A class="dropdown-button" href="#!" data-activates="dropdown4" style="display: block; padding-top: 19px; padding-bottom: 19px">Results</A></LI><UL id="dropdown4" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#one" style="color:#616161;">Microbiome</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#two" style="color:#616161;">Antibodies</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#three" style="color:#616161;">RNAiCare</A></LI></UL><LI><A class="dropdown-button" href="#!" data-activates="dropdown3" style="display: block; padding-top: 19px; padding-bottom: 19px">Team</A></LI><UL id="dropdown3" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Team" style="color:#616161;">Members</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Collaborations" style="color:#616161;">Collab</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Attributions" style="color:#616161;">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Sponsors" style="color:#616161;">Sponsors</A></LI></UL><LI><A href="https://2016.igem.org/Team:Lethbridge/Achievements" style="display: block; padding-top: 19px; padding-bottom: 19px">Achievements</A></LI></UL><A href="#" data-activates="nav-mobile" class="button-collapse"><I class="material-icons">menu</I></A><UL id="nav-mobile" class="side-nav" style="margin-top:20px; margin-left:0px; margin-right:0px; margin-bottom:0px;"><LI class="bold"><A class="collapsible-header waves-effect waves-blue">Project</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Database">Database</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/RNAi">RNAi</A></LI></UL></DIV><LI class="bold"><A class="collapsible-header waves-effect waves-blue">Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#intro">Intro</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#interviews">Interviews</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#litrev">Literature Review</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#ridealong">Ridealongs</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#sampling">Sampling</A></LI></NAV></DIV><LI class="bold"><A class="collapsible-header waves-effect">Results</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#one">Microbiome</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#two">Antibodies</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#three">RNAiCare</A></LI></DIV><LI class="bold"><A class="collapsible-header waves-effect">Team</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Sponsors">Sponsors</A></LI></DIV><LI class="bold"><A class="collapsible-header waves-effect" href="https://2016.igem.org/Team:Lethbridge/Achievements">Achievements</A></LI></DIV><MAIN><DIV class="row"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H2 class="thin white-text">Design</H2><H5 class="thin white-text">Our engineered design princples</H5></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="one"><DIV class="col s12 blue-grey darken-2"><DIV class="container"><DIV class="row"><DIV class="col l12 m12 s12"><H3 class="thin white-text">Introduction</H3><P class="grey-text text-lighten-4">We were approached by the Lethbridge Fire Department and EMS to investigate the microbiome of their vehicles and to provide a direct assessment of their cleaning protocols. Currently, the vehicles are cleaned on a daily basis or as needed, and undergo a more extensive deep clean once a month. Areas of the vehicle where blood, vomit and other bodily fluids are visible are cleaned thoroughly, but what about areas where there is no obvious contamination?</P><P class="grey-text text-lighten-4">Another issue brought to our attention by the EMS was that some of their disinfectant products kill only a fraction of the pathogenic material, while the remaining viable pathogens spread to other areas and possibly transfer to hospitals or the community. Therefore, EMS vehicles may be potential reservoirs of pathogens, associated with an increased risk of infection for immune-compromised patients. Using this information, as well as what we learned from the literature (link to lit review), interviews (link to interview), and ride-alongs (link to ride-alongs), we designed a two-pronged approach to evaluate these concerns. Specifically, we needed to design a system that would: 1) determine what pathogens are present in the vehicles and 2) verify the vehicle is decontaminated. This issue brought to our attention by our local Fire Department and EMS has the potential to be a national case study with regards to cleaning standards. Additionally, countries around the world may experience similar issues and could benefit from our study. </P><P class="grey-text text-lighten-4">Therefore, our engineered solution must satisfy the following design principles: <B>1) comprehensive</B>; provide a complete picture of the vehicle microbiome,<B> 2) portable</B>; a detection tool that could be transferred to different vehicles, while the overall solution could be implemented around the globe, <B>3) easy to use</B>; no additional training would be required of the first responders (i.e. “paramedic proof”), <B>4) standardized</B>; develop protocols and methods that could be duplicated by others for similar or related applications, <B>5) rapid</B>; first responders could quickly identify pathogens (i.e. quick response time) and <B>6) specific</B>; accurate in regards to the types of pathogens identified and detected. </P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="two"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l12 m12 s12"><H3 class="thin white-text">Microbiome Analysis</H3><P class="grey-text text-lighten-4">From the literature review (link back to literature review), we discovered that no sequencing approach had been used to assess the types of pathogens in ambulances. As such, to get a comprehensive picture of what is contained within these vehicles we decided on a sequencing approach. Specifically, we chose the MinION from Oxford Nanopore Technologies. This sequencing platform is well suited for our project as it routinely allows for long sequencing reads of up to 10kbp. The completed reads were then mapped to a database, allowing for the identification of different types of pathogens within a sample obtained from the EMS vehicles. Since this technology is relatively new there were no established protocols for obtaining and preparing environmental samples for Nanopore sequencing. This allowed us to develop our own novel sampling pipeline and methodology to amplify and prepare DNA. </P><P class="grey-text text-lighten-4">Sample locations were chosen based on information obtained from the literature, our observations during ride-alongs and areas of high concern indicated by first responders (link back to sampling protocol). We chose to sample the bottom of soft kits, the interior door handle, and SPO2 finger monitor. As well, we included a field control (i.e. a sample that did not physically swipe a surface). All four samples were acquired weekly from 3 ambulances over one deep clean cycle for a total of 48 samples to be sequenced (link to sampling protocol).</P><P class="grey-text text-lighten-4">The acquired samples were then prepared for the MinION, which included DNA amplification, barcoding and adapter ligation (link to sample prep protocol). We chose to amplify and isolate ribosomal RNA from both prokaryotes and eukaryotes using universal gene primers [1,2]. Using this approach, we identified several pathogenic and opportunistic pathogenic bacteria. </P><P class="grey-text text-lighten-4">Additionally, we identified species that were not readily culturable. This highlighted the benefit of our approach over more conventional cell culturing techniques which could not identify these species. The results of the sequencing analysis provided a comprehensive picture of pathogens contained within EMS vehicles. This information could be used to provide a baseline measurement of the vehicle microbiome, followed by subsequent sequencing to monitor cleaning habits over an extended period of time. </P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="three"><DIV class="col s12 blue-grey darken-2"><DIV class="container"><DIV class="row"><DIV class="col l12 m12 s12"><H3 class="thin white-text">Pathogen Detection</H3><P class="grey-text text-lighten-4"> We have described an approach to provide a comprehensive picture of the vehicle microbiome. However, a rapid method is required to monitor the cleanliness of EMS vehicles on a more frequent basis. One potential tool would be an antibody based detection method, such as an Enzyme Linked Immunosorbent Assay (ELISA). In general, antibodies could be raised against specific targets of interest (i.e. surface proteins of pathogens identified in the sequencing results) and used to develop a <B>specific</B> colorimetric assay. First responders could then swab a surface and use this assay to <B>easily</B> and <B>rapidly</B> observe a color change to identify if the pathogen was present. Given a certain threshold, in the case an opportunistic pathogen was detected, the necessary steps would need to be taken (i.e. use a different cleaning product or conduct a more thorough clean) to ensure the vehicle was acceptable to use. </P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="three"><DIV class="col s12 blue-grey darken-2"><DIV class="container"><DIV class="row"><DIV class="col l12 m12 s12"><H3 class="thin white-text">sdAb Library Design and Selection </H3><P class="grey-text text-lighten-4">One issue with using antibody based detection methods is the cost and time associated with developing antibodies to bind to specific targets of interest. As this was not feasible for our project, we decided to explore an alternative approach using single domain antibodies (sdAbs), derived from camelid heavy chain antibodies and consisting only of the variable heavy chain domain (VHH). Specifically, we chose to rationally design and select for sdAbs in E. coli, reducing the time and cost associated with production. Additionally, these synthetic libraries allow for an assortment of variants to be selected for, increasing the possibility of a strong binder to be identified. </P><P class="grey-text text-lighten-4">Prior to our library design, we developed a sdAb database to provide a repository of publicly available sequences. This database would offer a resource for researchers to easily identify sdAbs that bind to specific targets for a variety of applications (link to database). Informed by our database, we performed a bioinformatics analysis and sequence alignment using a subset of sdAb sequences (~40 sequences) [3]. The primary sequence of sdAbs is characterized by constant framework regions and variable complementarity determining regions, where the variable regions convey specificity for binding to diverse targets. From the sequence alignment, we could identify commonalities between sdAbs and identify highly conserved amino acids at specific positions in order to reduce our sequence search space. Similarly, for amino acid positions that are more variable only the residues most likely to occur at this position were tested for rather than using all possible 20 amino acids. Therefore, based on the amino acid conservation we can narrow our search space for only relevant combinations. Looking only at CDR3, which is the variable region most important for conveying specificity, there are  approximately 1011 sequence combinations. This number increases significantly when including the sequence combinations for CDR1 and CDR2, highlighting our designed library complexity. </P><P class="grey-text text-lighten-4">To select for sdAbs we chose to modify a bacterial-2-hybrid system.  Our construction employed a two-plasmid design, with one plasmid harbouring both an RNA polymerase alpha subunit and fluorescent reporter constructs, and the other containing a randomized library of single-domain antibody sequences. The fluorescent reporter construct allows for strong and weak binders to be sorted using a fluorescently activated cell sorter (FACS).</P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="three"><DIV class="col s12 blue-grey darken-2"><DIV class="container"><DIV class="row"><DIV class="col l12 m12 s12"><H3 class="thin white-text">Summary</H3><P class="grey-text text-lighten-4">The design of our project adopted a two-pronged approach for identifying and detecting various pathogens within EMS vehicles. Our approach is comprehensive as it is able to identify any bacterial or eukaryotic species present utilizing universal ribosomal DNA primers. </P><P class="grey-text text-lighten-4">Additionally, we are able to generate sdAbs to any epitope or surface protein and readily detect the presence of organisms within EMS vehicles. This approach is also portable: the MinION sequencer is more compact than a smartphone, and new developments in technology and library preparation will facilitate ad hoc sequencing and rapid detection. Additionally, cloud-based sequence analysis will enable rapid detection and tracking of the microbiome of an ambulance over time. </P><P class="grey-text text-lighten-4">Moreover, our group has adapted the generation of sdAbs into an easy-to-use, extremely rapid test kit based on an ELISA for day-to-day monitoring of the pathogen load of an EMS vehicle. This test kit can be used in any location at any time and will provide a real-time answer to first responders. Most importantly our solution is designed to be used by the individuals on the front lines, the first responders; this is why we have made our ELISA-based test system easily adaptable and ready to use without any special training required. </P><P class="grey-text text-lighten-4">Moving forward, a significant amount of value lies in the standardization of our approach. We were able to develop a new, standardized method of sample preparation for a cutting-edge sequencing platform. Our device can be utilized not only by environmental sampling groups, but also future iGEM teams around the world. Our standard parts for generating a sdAb library are also available in the iGEM registry for use by any future iGEM team. </P><P class="grey-text text-lighten-4">Given that the sequencing platform from Oxford Nanopore Technologies is an emerging technology, errors in sequencing do occur. To overcome this, and to make our approach much more accurate, our two pronged-solution includes highly specific detection based on antibody recognition. <B>Overall, we have successfully generated an approach that is robust in the detection and identification of potentially harmful pathogens which is generalizable to a multitude of situations around the world. </B></P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="three"><DIV class="col s12 blue-grey darken-2"><DIV class="container"><DIV class="row"><DIV class="col l12 m12 s12"><H3 class="thin white-text">References</H3><P class="grey-text text-lighten-4">[1] Wang Y, Qian P-Y (2009) Conservative Fragments in Bacterial 16S rRNA Genes and Primer Design for 16S Ribosomal DNA Amplicons in Metagenomic Studies. PLoS ONE 4(10): e7401. doi:10.1371/journal.pone.0007401 </P><P class="grey-text text-lighten-4">[2] Wang Y, Tian RM, Gao ZM, Bougouffa S, Qian P-Y (2014) Optimal Eukaryotic 18S and Universal 16S/18S Ribosomal RNA Primers and Their Application in a Study of Symbiosis. PLoS ONE 9(3): e90053. doi:10.1371/journal.pone.0090053</P><P class="grey-text text-lighten-4">[3] Pellis M, Pardon E, Zolghadr K, Rothbauer U, Vincke C, Kinne J, Dierynck I, Hertogs K, Leonhardt H, Messens J, Muyldermans S, Conrath K (2012) A bacterial-two-hybrid selection system for one-step isolation of intracellularly functional Nanobodies. Archives of Biochemistry and Biophysics 526, 114-123. doi:10.1016/j.abb.2012.04.023 </P></DIV></DIV></DIV></DIV></DIV></MAIN></DIV></DIV></BODY></HTML>