<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lethbridge_RNAi skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lethbridge/RNAi</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Lethbridge iGEM 2016</TITLE></P><DIV id="content"><NAV class="blue-grey darken-4" role="navigation" style="margin-top: 15px;"><DIV class="nav-wrapper"><A href="https://2016.igem.org/Team:Lethbridge" class="brand-logo left" style="width:auto; left:40px;"> Lethbridge iGEM</A><UL class="right hide-on-med-and-down" style="list-style-image: none;"><LI><A class="dropdown-button" href="#!" data-activates="dropdown1" style="display: block; padding-top: 19px; padding-bottom: 19px">Project</A></LI><UL id="dropdown1" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Description" style="color:#616161;">Description</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Design" style="color:#616161;">Design</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Experiments" style="color:#616161;">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Database" style="color:#616161;">Database</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Notebook" style="color:#616161;">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Parts" style="color:#616161;">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Safety" style="color:#616161;">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/RNAi" style="color:#616161;">RNAi</A></LI></UL><LI><A class="dropdown-button" href="#!" data-activates="dropdown2" style="display: block; padding-top: 19px; padding-bottom: 19px">Practices</A></LI><UL id="dropdown2" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Integrated_Practices" style="color:#616161;">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#intro" style="color:#78909c;">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#interviews" style="color:#78909c;">Interviews</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#litrev" style="color:#78909c;">Literature Review</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#ridealong" style="color:#ff8f00;">Ridealongs</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#protocol" style="color:#ff8f00;">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#sampling" style="color:#ff8f00;">Sampling</A></LI></UL><LI><A class="dropdown-button" href="#!" data-activates="dropdown4" style="display: block; padding-top: 19px; padding-bottom: 19px">Results</A></LI><UL id="dropdown4" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#one" style="color:#616161;">Microbiome</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#two" style="color:#616161;">Antibodies</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#three" style="color:#616161;">RNAiCare</A></LI></UL><LI><A class="dropdown-button" href="#!" data-activates="dropdown3" style="display: block; padding-top: 19px; padding-bottom: 19px">Team</A></LI><UL id="dropdown3" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Team" style="color:#616161;">Members</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Collaborations" style="color:#616161;">Collab</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Attributions" style="color:#616161;">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Sponsors" style="color:#616161;">Sponsors</A></LI></UL><LI><A href="https://2016.igem.org/Team:Lethbridge/Achievements" style="display: block; padding-top: 19px; padding-bottom: 19px">Achievements</A></LI></UL><A href="#" data-activates="nav-mobile" class="button-collapse"><I class="material-icons">menu</I></A><UL id="nav-mobile" class="side-nav" style="margin-top:20px; margin-left:0px; margin-right:0px; margin-bottom:0px;"><LI class="bold"><A class="collapsible-header waves-effect waves-blue">Project</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Database">Database</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/RNAi">RNAi</A></LI></UL></DIV><LI class="bold"><A class="collapsible-header waves-effect waves-blue">Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#intro">Intro</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#interviews">Interviews</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#litrev">Literature Review</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#ridealong">Ridealongs</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#sampling">Sampling</A></LI></NAV></DIV><LI class="bold"><A class="collapsible-header waves-effect">Results</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#one">Microbiome</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#two">Antibodies</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Results#three">RNAiCare</A></LI></DIV><LI class="bold"><A class="collapsible-header waves-effect">Team</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Sponsors">Sponsors</A></LI></DIV><LI class="bold"><A class="collapsible-header waves-effect" href="https://2016.igem.org/Team:Lethbridge/Achievements">Achievements</A></LI></DIV><MAIN><DIV class="row"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l8 s12"><H2 class="thin white-text">RNAi</H2></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="one"><DIV class="col s12 blue-grey darken-2"><DIV class="container"><DIV class="row"><DIV class="col l12 m12 s12"><H3 class="thin white-text">Background</H3><P class="grey-text text-lighten-4">Our strategy for dsRNA production is a multi-part approach. The construct is expressed dicistronically, with a His-tagged MS2 coat-protein expressed initially. Additionally, we employ the use of a Herpes Delta Virus Ribozyme (HDVR), which will be evolved through SELEX in order to generate a thermozyme. Upon an increase in temperature (22 °C to 37°C), the thermozyme undergoes a conformational change, resulting in cleavage at a specifically prescribed nucleotide. This thermozyme is placed just upstream of an MS2 coat-protein binding site. This allows us to purify only full-length RNA once it has been transcribed, as those transcripts not harbouring the binding domain will not be effectively purified. His-tagged MS2 coat-protein is then free to bind the newly transcribed binding domain. Using affinity chromatography, purification of the MS2 coat-protein and the bound RNA is possible. Upon a temperature increase, cleavage and liberation of a single stranded RNA occurs, allowing for purification of a single strand of highly pure RNA.</P><P class="grey-text text-lighten-4">By using two complementary RNA-generating sequences within the thermozyme construct, we are able to generate double stranded RNA for use in pest control simply by annealing the two resultant strands produced by our purification strategy.</P><P class="grey-text text-lighten-4">Given our chassis and ability to over-express His-tagged MS2 coat-protein, our purification strategy is poised to purify large amounts of highly specific RNA. The scalability of this platform lies in the ability of individuals to design novel pesticides for any target organism, having only requisite knowledge of the genome. New dsRNA-based pesticides will be employed cheaply and specifically without costly design and massive amounts of resources currently utilized in the development of novel pesticides. Pesticides represent a multi-billion dollar industry worldwide, and with the scalability of this synthetic biology mode of production, this project represents a readily commercializable method of producing large quantities of highly specific pesticides applicable to a wide array of pest species.</P><P class="grey-text text-lighten-4">For large scale production, our group will likely use fermenters for sufficient growth. However, as demonstrated, the concentrations of dsRNA required for gene silencing are sufficiently low that the large-scale production need not necessitate massive amounts of resources.</P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="two"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col l12 m12 s12"><H3 class="thin white-text">Human Practices</H3><P class="grey-text text-lighten-4">As a part of the design of our project, we looked over the ethical implications of our study. We looked into a variety of different farming methods and uses of pesticides. We examined off-target and non-target effects by ensuring the target sequences we selected were compared against the entire database of sequenced genomes using the NCBI BLAST program. We also spoke with various experts and held a panel discussion regarding the efficacy of our project from the lab to the field. </P><P class="grey-text text-lighten-4">We also contacted major small molecule pesticide distributors to estimate the current cost analysis. Finally, we collaborated with the Canadian Food Inspection Agency (CFIA) and Health Canada to further explore the ethical implications of our project. 
</P><H3 class="thin white-text">Risks</H3><P class="grey-text text-lighten-4"><B>RNAi: Off-Target and Non-Target Effects</B></P><P class="grey-text text-lighten-4">The RNAi is used more extensively in pesticides due to its target specificity. However, it does not come without the possibility of negative effects or errors in what RNAi sequences are targeted. Ideally, the RNAi manufactured should only target the specific sequence that causes virulence in Fusarium graminearum which would be an on-target effect. Due to sequence similarities across species and within the genome of the target pest, there could be unwanted effects. Two types of these effects are off-target and non-target.</P><P class="grey-text text-lighten-4">Off-target effects are when the processed RNAi silences a part of the genome that is downstream or upstream of the target sequence, or only part of the target sequence. RNAi technologies have been found to have a lack of sequence specificity [1]. Therefore, a virulence factor may not be targeted, allowing F. graminearum to continue infecting crops. A 100% sequence match is often required for the RNAi to work [2], any slight sequence change or mutation could render the RNAi sequence unable to disrupt the target sequence. Therefore, important to know what the exact sequences are that will cause death in the organism when down-regulated. Just because a 100% match is required for the RNAi to work doesn’t mean that it can’t interact with and affect another area within the genome that is slightly different, impairing its function as well. Further testing of the RNAi spray would be required to determine the exact effects regarding the probability of off-target effects, but it has been shown that using a lower concentration of interfering RNAs reduces the amount of off-target effects within the genome [3].</P><P class="grey-text text-lighten-4">Non-target effects are when the RNAi down-regulates a gene that is not in the intended target species. Due to the short length of interfering RNAs, there is more opportunity for sequence similarities between species. Therefore two different species could be equally likely targets of the RNAi. This means that the spray could influence the growth and function of other pests, pollinators, and crops and even cause death in these organisms. Similar to off-target effects, it is possible that due to lack of specificity a non-target species could have a sequence with a completely different function than the target which interacts with the RNAi. It is important to determine what species in agricultural communities could be affected by the chosen sequence, and what sequence similarities exist between species that are closely related to the target.</P><P class="grey-text text-lighten-4">Off-target and non-target effects could potentially occur at the same time. For example, a non-target species could have short lengths of sequence similarity with F. graminearum and have its genome disrupted by the RNAi, but due to RNAi’s lack of specificity there could be off-target silencing downstream of the matching sequence within that organism’s genome.</P><P class="grey-text text-lighten-4"><B>Sequence BLAST</B></P><P class="grey-text text-lighten-4">To examine potential off-target effects of our gene-silencing dsRNAs, we undertook an extensive comparative approach using NCBI BLAST. Areas of similarity within other genomes may result in potential silencing outside of the desired organism. We first compared our chosen sequences to the entire database of sequenced genomes. After finding minimal overlap between our sequences and any other off-target organisms, we moved on to compare specifically within humans and varietals of wheat. Again, we found no significant overlaps or risks in terms of silencing capability. Therefore, we are confident in the safety and specificity of our design platform.</P><P class="grey-text text-lighten-4">Our search for potential off-target effects represents an exemplary level of attention to the safety of consumers as well as those handling and producing the dsRNA-based pesticides.</P><P class="grey-text text-lighten-4">We tested our sequences against common species.</P><P class="grey-text text-lighten-4"><U>References</U></P><P class="grey-text text-lighten-4">Jackson A. and P. Linsley. (2010). Recognizing and avoiding siRNA off-target effects for target identification and therapeutic use. Nature, 9, 57-67.</P><P class="grey-text text-lighten-4">Subba Reddy Palli. (2014). RNA interference in Colorado potato beetle: steps toward development of dsRNA as a commercial insecticide. COIS, 3, 1-8.</P><P class="grey-text text-lighten-4">Dharmacon. (2014). Off-Target Effects: Disturbing the Silence of RNA interference (RNAi). 1-4.
  </P><H3 class="thin white-text">Stakeholders</H3><P class="grey-text text-lighten-4"><B>RNAi: Off-Target and Non-Target Effects</B></P><P class="grey-text text-lighten-4"> Wheat crops in Canada are protected from infection by Fusarium graminearum through the application of foliar fungicides and seed treatments. Foliar fungicides are sprayed onto the plant itself rather than soaking seeds as is done in a seed treatment. Seed treatments are used to prevent initial FHB infections from the dormant chlamydospores in the soil. These seed treatments are however, not as effective in prevention of the spread of FHB once the crop begins to grow and are most vulnerable [1]. A suppressive spraying method is used, meaning crops are sprayed at the point in growth that they are most vulnerable to infection by FHB. Farmers will spray once after 75% of the heads on the main stem are fully emerged, but before 50% of them have flowered [2].</P><P class="grey-text text-lighten-4"> Table 1: Product information regarding foliar fungicides that target FHB in winter wheat [3][4].</P><TABLE class="responsive-table"><THEAD><TR><TH data-field="id">Fungicide</TH><TH data-field="name">Recommended Vol. / Acre (mL)</TH><TH data-field="price">Cost/Acre</TH><TH data-field="price">Concentration (g/L)</TH></TR></THEAD><TBODY><TR><TD>Folicur EW</TD><TD>201</TD><TD>$12.13</TD><TD> 250 </TD></TR><TR><TD> Prosaro</TD><TD>324</TD><TD>$19.62</TD><TD> 250</TD></TR><TR><TD>Fuse
</TD><TD>118</TD><TD>$11.67</TD><TD> 432</TD></TR></TBODY></TABLE><P class="grey-text text-lighten-4"> Table 1 provides information about three commonly used FHB combatant foliar fungicides. The cost of spraying these products adds up when considering the millions of acres that Canadian farmers must spray to keep their flowering wheat Fusarium-free. Seed treatments can also be quite expensive. For example, a fungicidal seed treatment made for wheat called Vibrance Quattro is $346 per bushel at 69.0g/L [4]. These costs all may vary based on the retailer and composition of the fungicides. Our goal is to eventually create a way to produce RNAi that is more affordable than these conventional fungicides.</P><P class="grey-text text-lighten-4"><U>References</U></P><P class="grey-text text-lighten-4"> [1] Agriculture and Agri-Food Canada. 2010. Crop Profile for Winter Wheat in Canada, 2010. Pesticide Risk Reduction Program Pest Management Centre Agriculture and Agri-Food Canada. 1-65.</P><P class="grey-text text-lighten-4"> [2] Government of Saskatchewan. 2015. Fusarium Head Blight. Provincial Crop Protection Laboratory. http://www.agriculture.gov.sk.ca/fusarium-head-blight</P><P class="grey-text text-lighten-4"> [3] Bayer CropScience Canada. 2015. http://www.cropscience.bayer.ca/Products/Fungicides</P><P class="grey-text text-lighten-4"> [4] Syngenta Canada. 2015. http://www.syngentafarm.ca/Productsdetail/Fuse/
</P><H3 class="thin white-text">Problems with Current Pest Control Techniques</H3><P class="grey-text text-lighten-4"><B>Small Molecule Pesticides</B></P><P class="grey-text text-lighten-4">Small molecule pesticides or bio-pesticides are unconventional pesticides that use substances which make up, or are produced, by plants and animals. Examples of these bio-pesticides include pheromones, enzymes, hormones, and even micro-organisms as a method of pest control [2]. These types of pesticides are excellent alternatives to traditional chemical pesticides. Advantages of this type of pesticide include low inherent-toxicity to target and non-target organisms, low persistence in the environment, and low likelihood of selecting for pest resistance [2]. Disadvantages of bio-pesticides include a typically slower mortality rate for the pests as well as a higher susceptibility to variable environmental conditions [3]. There is also an issue of properly storing pesticides that are made out of biomolecules. One of the most well-known examples of small molecule pesticides is DDT (Dichlorodiphenyltrichloroethane). Short term studies have shown that the potential benefits outweighed the risks. However, the stability of DDT, persistence in the environment, accumulation in adipose tissue, and potential estrogenic properties raise concerns over long term use. Accumulation through the trophic levels leads to adverse side effects such as thinning of falcon eggshells, leading to population decline [4].</P><P class="grey-text text-lighten-4"><B>Biological Pest Control</B></P><P class="grey-text text-lighten-4">Biological pest control involves the use of natural predators against pests. In many cases, the pests are invasive species which have escaped from their natural ecosystem. When the natural predator of the pest is not present, the pest has the opportunity to proliferate without control [5]. Natural predators are then imported after pre- and post-release testing is done. Pre-release testing involves testing the natural predator to ensure it will not take over and proliferate in the ecosystem itself, as well as non-target testing to ensure beneficial species will not be targeted [6]. Testing is initially done in the lab; once there is enough data supporting the efficacy of the predator, field tests are done. Field tests give a better representation of what to expect post-release. Upon government approval, the predator is released into the ecosystem. Potential downfalls of biological pest control include predatory species becoming an alpha predator in the food chain, or becoming an invasive species itself [7].</P><P class="grey-text text-lighten-4"><B>Mechanical Pest Control</B></P><P class="grey-text text-lighten-4">Mechanical pest control involves the use of physical means to control pests. Some examples of mechanical pest control methods include insect traps, physical barriers, or weed pulling. These control mechanisms are advantageous in terms of minimizing off-target effects, and are generally environmentally friendly. However, these methods can be laborious and are often not feasible for implementation on commercial crops. Recent research has aimed to decrease the amount of labor required to pull weeds from commercial crops by using computer vision-guided machines [8]. Initial results with computer guided machines has shown promising results, producing weed reductions in the range of 62-87%. However, this research requires more development in order to be mainstream in commercial agriculture.</P><P class="grey-text text-lighten-4"><B>Crop Rotation</B></P><P class="grey-text text-lighten-4">Crop rotation is an agricultural method in which different species of crops are grown in the same area of land to promote nutrient renewal within the soil. In addition, crop rotation aids in crop yield, soil quality, and controls the build-up of weeds, pests, and other diseases that might be specific to the crop. However, crop rotation can limit the variety of crops a farmer is able to grow in their field due to specific rotation patterns. Each season, these crops need to be monitored in order to control the likelihood of diseases being carried over to the next crop. This may be limiting for farmers because of the constraints associated with which crops they are able to plant and at what time they can be planted. In addition, certain crops require specific machinery which may be expensive for farmers [9].</P><P class="grey-text text-lighten-4">An example of an important crop produced throughout the world is canola. It is used to make canola oil which is a key ingredient in many foods, cosmetics, and biodiesels. When canola crops are rotated with barley, oats, or wheat, the lowest number of diseases carry over from the previous crop which in turn increases yields of the canola crop. Most other crops such as corn, flax, potato, and sunflower require wait times of up to three years before they can be planted to prevent a buildup of diseases common to both the respective plant and canola [10].</P><P class="grey-text text-lighten-4"><B>Crop Burning</B></P><P class="grey-text text-lighten-4">Another method of pest control involves crop burning, which is the process of setting fire to crop residues after harvest seasons to clear large areas of land. Remaining pests on particular crops are killed in order to prepare the soil for the following season. Crop burning is considered a quick and easy method for clearing farmland, but it is usually only used by farmers when other methods for clearing pests is unsuccessful. A burning permit is required by farmers interested in crop burning, which may end up being expensive if used continuously [11]. Bigger risks that come with crop burning include air pollution, soil pollution, and an increase in soil erosion from wind and water. Crop burning also has a big impact in local ecosystems surrounding farm land [12].</P><P class="grey-text text-lighten-4"><B>Buffer Zones</B></P><P class="grey-text text-lighten-4">Buffer Zones are placed between organic crop fields and fields containing genetically modified (GM) crops to protect the organic farms from any contact or contamination, keeping organic crops as close to organic as possible. Contamination of organic crops by GM crops can occur through many different methods such as: gene transfer through cross-pollination, seed translocation, and contact with inorganic pesticides [13]. Wheat pollen has been detected to travel up to 50 meters via pollen drift [13] therefore, buffer zones must be greater than 50 meters to have an impact. A 200 meter buffer zone is not enough to stop pollen-mediated genetic antibiotic and drug resistance [15]. When commercial scale equipment is used in harvest, an extra 50 metres is added on to the required distance between the wheat trials [14]. This extra distances helps to protect organic crops from any other form of contamination that could be transferred by the machines. In order to effectively protect organic crops from contamination the distance required for isolation should be further tested and increased.</P><P class="grey-text text-lighten-4"><U>References</U></P><P class="grey-text text-lighten-4"> [1] Agriculture and Agri-Food Canda. (2014). Biopesticides Priority Setting 2014. Government of Canada, http://www.agr.gc.ca/eng/?id=1411590428019.</P><P class="grey-text text-lighten-4"> [2] Agriculture and Agri-Food Canada. (2013). Categories of Biopesticides and Related Products. Government of Canada, http://www.agr.gc.ca/eng/?id=1299080205554.</P><P class="grey-text text-lighten-4"> [3] Chandler, D., Bailey, A.S., Tatchell, G.M., Davidson, G., Greaves, J. &amp; Grant, W.J. (2011). The development, regulation and use of biopesticides for integrated pest management. Phil. Trans. R. Soc. B, doi:10.1098/rstb.2010.0390.</P><P class="grey-text text-lighten-4"> [4] Turusov, V., Rakitsky, V., &amp; Tomatis, L. (2002). Dichlorodiphenyltrichloroethane (DDT): ubiquity, persistence, and risks. Environmental health perspectives, 110(2), 125.</P><P class="grey-text text-lighten-4"> [5] Flint, M. L., Dreistadt, S. H., &amp; Clark, J. K. (1998). Natural enemies handbook: the illustrated guide to biological pest control (Vol. 3386). Univ of California Press.</P><P class="grey-text text-lighten-4">[6] Follett, P., &amp; Duan, J. J. (Eds.). (2012). Nontarget effects of biological control. Springer Science &amp; Business Media.</P><P class="grey-text text-lighten-4">[7] Louda, S. M., Pemberton, R. W., Johnson, M. T., &amp; Follett, P. (2003). Nontarget Effects-The Achilles' Heel of Biological Control? Retrospective Analyses to Reduce Risk Associated with Biocontrol Introductions*. Annual Review of Entomology, 48(1), 365-396.</P><P class="grey-text text-lighten-4">[8] Tillett, N. D., Hague, T., Grundy, A. C., &amp; Dedousis, A. P. (2008). Mechanical within-row weed control for transplanted crops using computer vision.Biosystems Engineering, 99(2), 171-178.</P><P class="grey-text text-lighten-4">[9] Iain Robson. (2013). Crop rotation definition and benefits. Web.http://myfarmeducation.ca/farming/crop-rotation-definition-and-benefits10. Canola Council of Canada. (2014).</P><P class="grey-text text-lighten-4">[10] Crop Rotation. Web. http://www.canolacouncil.org/canola-encyclopedia/field-characteristics/crop-rotation</P><P class="grey-text text-lighten-4">[11] Department of Ecology, State of Washington. (2012). Agricultural Burning. Web.</P><P class="grey-text text-lighten-4">[12] The State of Victoria. (2015). Stubble burning. Web. http://agriculture.vic.gov.au/agriculture/grains-and-other-crops/crop-production/stubble-burning</P><P class="grey-text text-lighten-4">[13] Belcher, K., Nolan, J. &amp; Phillips, P.W.B. (2005). Genetically modified crops and agricultural landscapes: spatial patterns of contamination. Ecological Economics, 53(3), 387-401.</P><P class="grey-text text-lighten-4">[14] Canadian Food Inspection Agency. (2015). Directive Dir2000-07: Conducting Confined Research Field Trials of Plant with Novel Traits in Canada. Government of Canada. http://www.inspection.gc.ca/plants/plants-with-novel-traits/applicants/directive-dir2000-07/eng/1304474667559/1304474738697.</P><P class="grey-text text-lighten-4">[15] Sirinathsinghji, E. (2012). Behind the GM Wheat Trial. Science in Society, 55, 6-7.</P></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="three"><DIV class="col s12 blue-grey darken-2"><DIV class="container"><DIV class="row"><DIV class="col s12"><H3 class="thin white-text">Wet Lab</H3><P class="grey-text text-lighten-4">

In order to generate a Herpes Delta Virus thermozyme [1], we first constructed a library of variants to be utilized in systematic evolution of ligands by exponential enrichment (SELEX). To generate our library we incorporated random nucleotides into the U1A-RBD stem loop with either N9, N10, N11 or N12 random nucleotides for a total of 1.7 X 108 potential combinations. All of these variants were then assembled into the HDVR through oligo assembly followed by overlap extension PCR. Upstream of the HDVR there is a MS2 binding domain and a T7 promoter with an XbaI cut site at the 3’ end and a SpeI cut site at the 5’ end. A second module was used in order to generate an MS2 variant that will no longer dimerize [2] and assembly into a viral capsid (BBa_K2109108). This was accomplished by using a characterized MS2 variant (V291-d1FG)2 with an N-terminal HIS tag, controlled through a T7 promoter and strong RBS followed by a double terminator (B0015). The MS2 will be overexpressed and used to saturate a Nickel Sepharose column so that the Thermozyme library can be applied to select for positive mutants. Two HDVR controls were generated (BBa_K2109109, BBa_K2109110) the first is a U1A mutant which was used as a negative control. The second HDVR mutant U1A-C75T is a negative control that can be induced to cleave in the presence of imidazole[1].</P><CENTER><P class="white-text center">HDVR ribozyme secondary structure [1]</P></CENTER><P class="white-text"><U>References: </U></P><P class="white-text">
1.	Ke, A., Zhou, K., Ding, F., Jamie, H.D., Doudna, J.A. (2004). A Conformational Switch Controls Hepatitis Delta Virus Ribozyme Catalysis. RNA, 13: 1384-1389.</P><P class="white-text">
2.	Batey, R.T., Kieft, J.S. (2007). Improved Native Affinity Purification of RNA. RNA, 429: 201-205.
</P></DIV></DIV></DIV></DIV></DIV></MAIN></DIV></DIV></BODY></HTML>