<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lethbridge_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lethbridge/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Lethbridge iGEM 2016</TITLE></P><DIV id="content"><NAV class="blue-grey darken-4" role="navigation" style="margin-top: 15px;"><DIV class="nav-wrapper"><A href="https://2016.igem.org/Team:Lethbridge" class="brand-logo left" style="width:auto; left:40px;"> Lethbridge iGEM</A><UL class="right hide-on-med-and-down" style="list-style-image: none;"><LI><A class="dropdown-button" href="#!" data-activates="dropdown1" style="display: block; padding-top: 19px; padding-bottom: 19px">Project</A></LI><UL id="dropdown1" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Description" style="color:#616161;">Description</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Design" style="color:#616161;">Design</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Experiments" style="color:#616161;">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Database" style="color:#616161;">Database</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Notebook" style="color:#616161;">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Parts" style="color:#616161;">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Safety" style="color:#616161;">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/RNAi" style="color:#616161;">RNAi</A></LI></UL><LI><A class="dropdown-button" href="#!" data-activates="dropdown2" style="display: block; padding-top: 19px; padding-bottom: 19px">Practices</A></LI><UL id="dropdown2" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Integrated_Practices" style="color:#616161;">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#intro" style="color:#616161;">Contact</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#interviews" style="color:#616161;">Interviews</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#litrev" style="color:#616161;">Literature Review</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#ridealong" style="color:#616161;">Ridealongs</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#protocol" style="color:#616161;">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#sampling" style="color:#616161;">Sampling</A></LI></UL><LI><A href="https://2016.igem.org/Team:Lethbridge/Results" style="display: block; padding-top: 19px; padding-bottom: 19px">Results</A></LI><LI><A class="dropdown-button" href="#!" data-activates="dropdown3" style="display: block; padding-top: 19px; padding-bottom: 19px">Team</A></LI><UL id="dropdown3" class="dropdown-content" style="list-style-image: none; margin-left: 0px; width: auto;"><LI><A href="https://2016.igem.org/Team:Lethbridge/Team" style="color:#616161;">Members</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Collaborations" style="color:#616161;">Collab</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Attributions" style="color:#616161;">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Sponsors" style="color:#616161;">Sponsors</A></LI></UL><LI><A href="https://2016.igem.org/Team:Lethbridge/Achievements" style="display: block; padding-top: 19px; padding-bottom: 19px">Achievements</A></LI></UL><A href="#" data-activates="nav-mobile" class="button-collapse"><I class="material-icons">menu</I></A><UL id="nav-mobile" class="side-nav" style="margin-top:20px; margin-left:0px; margin-right:0px; margin-bottom:0px;"><LI class="bold"><A class="collapsible-header waves-effect waves-blue">Project</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Database">Database</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/RNAi">RNAi</A></LI></UL></DIV><LI class="bold"><A class="collapsible-header waves-effect waves-red">Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#intro">Intro</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#interviews">Interviews</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Silver#litrev">Literature Review</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#ridealong">Ridealongs</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/HP/Gold#sampling">Sampling</A></LI></NAV></DIV><LI class="bold"><A class="collapsible-header waves-effect" href="https://2016.igem.org/Team:Lethbridge/Results">Results</A></LI><LI class="bold"><A class="collapsible-header waves-effect">Team</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Lethbridge/Sponsors">Sponsors</A></LI></DIV><LI class="bold"><A class="collapsible-header waves-effect" href="https://2016.igem.org/Team:Lethbridge/Achievements">Achievements</A></LI></DIV><DIV class="row"><DIV class="col s12 blue-grey darken-1"><DIV class="container"><DIV class="row"><DIV class="col s12"><H2 class="white-text">Notebook</H2><H5 class="white-text">The various facets of our lab work were divided between several key areas. Please explore!</H5></DIV></DIV><DIV class="row"><DIV class="col s12 m6 l6"><A class="waves-effect waves-light btn-large red darken-3 center" href="https://2016.igem.org/Team:Lethbridge/Notebook#microbiome">Nanopore Microbiome Analysis</A></DIV><DIV class="col s12 m6 l6"><A class="waves-effect waves-light btn-large blue darken-3 center" href="https://2016.igem.org/Team:Lethbridge/Notebook#pathogen">Single-domain antibody development system</A></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="microbiome"><DIV class="col s12 red darken-3"><DIV class="container"><DIV class="row"><DIV class="col s12"><H4 class="white-text left">Nanopore Microbiome Analysis</H4></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col s12"><TABLE><THEAD><TR><TH data-field="date">Date</TH><TH data-field="experiment">Experiment</TH></TR></THEAD><TBODY><TR><TD><A style="color:#424242;" href="#Entry1">2016/08/16</A></TD><TD>DNA extraction protocol</TD></TR><TR><TD><A style="color:#424242;" href="#Entry2">2016/08/16</A></TD><TD>Multiple displacement amplification of EMS - Week #1</TD></TR><TR><TD><A style="color:#424242;" href="#Entry3">2016/08/24</A></TD><TD>16S rRNA gene amplification from EMS samples (week #4)</TD></TR><TR><TD><A style="color:#424242;" href="#Entry4">2016/08/31</A></TD><TD>Optimization of PCR amplification using various rDNA gene primer pairs</TD></TR><TR><TD><A style="color:#424242;" href="#Entry5">2016/08/31</A></TD><TD>Large scale PCR amplification using 16S and 18S rRNA</TD></TR><TR><TD><A style="color:#424242;" href="#Entry6">2016/10/4</A></TD><TD>DNA extraction - Week #4 samples</TD></TR><TR><TD><A style="color:#424242;" href="#Entry7">2016/10/5</A></TD><TD>Amplify 1N4, 2N4, 3N4, Pseud., virus control, E. coli control</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="container" id="Entry1"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">DNA extraction protocol</SPAN><H6 class="grey-text text-lighten-1">2016/08/16</H6><P class="flow-text">Protocol for the extraction of DNA from samples taken from EMS ride-alongs.</P><OL><LI>Ethanol evaporation for approximately 2 hours on the rotary evaporator on high heat, under vacuum and high speed.</LI><LI>Add 400 µl of alkaline lysis buffer (200 mM KOH, 50 mM DTT).</LI><LI>Mix by inverting tube several times.</LI><LI>Incubate on heat block at 65 °C for 10 minutes.</LI><LI>Centrifuge at 17,000 x g for 1 minute.</LI><LI>Pipette liquid volume (as much as possible) into new 1.5 mL microcentrifuge tube (~300 µl)</LI><LI>Add 1/10 of the volume (~30 µl) of 3 M sodium acetate (pH 5.3) to each tube.</LI><LI>Add 5 µl of 0.5% LPA (linear polyacrylamide) carrier to each tube.</LI><LI>Add 750 µl of cold 100% ethanol to each tube, mix by inverting and place tubes on ice for approximate 10 minutes.</LI><LI>Centrifuge at 17,000 x g for 5 minutes.</LI><LI>Remove supernatant.</LI><LI>Wash with 500 µl of 70% ethanol and mix gently. Push centrifuge and remove supernatant.</LI><LI>Repeat wash step (step 12)</LI><LI>Vacuum evaporate the ethanol under high heat for 10 minutes.</LI><LI>Resuspend in 20 µl of filtered MilliQ water.</LI></OL></DIV></DIV></DIV></DIV></DIV><DIV class="row" id="pathogen"><DIV class="col s12 blue darken-3"><DIV class="container"><DIV class="row"><DIV class="col s12"><H4 class="white-text left">Single Domain Antibody Development System</H4></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col s12"><TABLE><THEAD><TR><TH data-field="date">Date</TH><TH data-field="experiment">Experiment</TH></TR></THEAD><TBODY><TR><TD><A style="color:#424242;" href="#Entry8">2016/05/01</A></TD><TD>Transformation of plasmids into NEB5α cells</TD></TR><TR><TD><A style="color:#424242;" href="#Entry9">2016/05/09</A></TD><TD>Set up of O/N cultures to isolate plasmids for SLIC</TD></TR><TR><TD><A style="color:#424242;" href="#Entry10">2016/05/13</A></TD><TD>PCR amplification of RNAPα::taget scaffold gBlock</TD></TR><TR><TD><A style="color:#424242;" href="#Entry11">2016/05/18</A></TD><TD>PCR amplification of pSB1T3 and RNAPα::target for T4 DNA polymerase chewback reaction</TD></TR><TR><TD><A style="color:#424242;" href="#Entry12">2016/05/19</A></TD><TD>SLIC assembly of pSB1T3-RNAPα::target</TD></TR><TR><TD><A style="color:#424242;" href="#Entry13">2016/05/26</A></TD><TD>PCR amplification for SLIC assembly of pSB3C5_Cl::Nb_Anti-GFP Nb</TD></TR><TR><TD><A style="color:#424242;" href="#Entry14">2016/06/1</A></TD><TD>SLIC assembly of pSB3C5_Cl::Nb_Anti-GFP Nb</TD></TR><TR><TD><A style="color:#424242;" href="#Entry15">2016/06/01</A></TD><TD>Amplification of GFP (BBa_E0040) using oNb52 and oNb47</TD></TR><TR><TD><A style="color:#424242;" href="#Entry16">2016/06/02</A></TD><TD>Colony PCR of potential pSB3C5_Cl::Nb_Anti-GFP Nb clones</TD></TR><TR><TD><A style="color:#424242;" href="#Entry17">2016/06/05</A></TD><TD>Screening of pSB1T3_RNAPα::target clones using BglII restriction digest</TD></TR><TR><TD><A style="color:#424242;" href="#Entry18">2016/06/25</A></TD><TD>Single-domain antibody library cloning</TD></TR><TR><TD><A style="color:#424242;" href="#Entry19">2016/07/25</A></TD><TD>Restriction digest of single-domain antibody (sdAb) scaffold</TD></TR><TR><TD><A style="color:#424242;" href="#Entry20">2016/08/3</A></TD><TD>Generation of CDR1-2 library by oligo assembly and PCR</TD></TR><TR><TD><A style="color:#424242;" href="#Entry21">2016/08/4</A></TD><TD>Generation of CDR3 libraries</TD></TR><TR><TD><A style="color:#424242;" href="#Entry22">2016/08/08</A></TD><TD>Ligation of CDR1-2 and sdAb scaffold</TD></TR><TR><TD><A style="color:#424242;" href="#Entry23">2016/08/10</A></TD><TD>CDR3 NgoMIV digestion</TD></TR><TR><TD><A style="color:#424242;" href="#Entry24">2016/08/11</A></TD><TD>KpnI and NgoMIV digests of sdAb scaffold and CDR1-2</TD></TR><TR><TD><A style="color:#424242;" href="#Entry25">2016/08/16</A></TD><TD>Ligation PCR of CDR3s and sdAb/CDR1-2</TD></TR><TR><TD><A style="color:#424242;" href="#Entry26">2016/08/18</A></TD><TD>CloneJet cloning of sdAb scaffold/CDR1-2_CDR3 ligation PCRs</TD></TR><TR><TD><A style="color:#424242;" href="#Entry27">2016/08/31</A></TD><TD>PCR amplification and overlap extension PCR assembly of sdAb library parts</TD></TR><TR><TD><A style="color:#424242;" href="#Entry28">2016/09/07</A></TD><TD>Overlap extension PCR assembly of sdAb library</TD></TR><TR><TD><A style="color:#424242;" href="#Entry29">2016/09/07</A></TD><TD>Overlap extension PCR of sdAb 5' CDR1-2 with CDR3 sdAb 3'</TD></TR><TR><TD><A style="color:#424242;" href="#Entry30">2016/09/26</A></TD><TD>Scale up PCR amplification of sdAb libraries</TD></TR><TR><TD><A style="color:#424242;" href="#Entry32">2016/09/27</A></TD><TD>PCR amplification of pSB3C5_Cl::Nb backbone for sdAb library SLIC assembly</TD></TR><TR><TD><A style="color:#424242;" href="#Entry33">2016/09/28</A></TD><TD>SLIC assembly of sdAb library into pSB3C5_Cl::Nb</TD></TR><TR><TD><A style="color:#424242;" href="#Entry34">2016/09/29</A></TD><TD>sdAb library SLIC assembly electroporation results</TD></TR><TR><TD><A style="color:#424242;" href="#Entry35">2016/10/07</A></TD><TD>Colony PCR screening of pSB4K5_pBAD RNAPα</TD></TR><TR><TD><A style="color:#424242;" href="#Entry36">2016/10/12</A></TD><TD>PCR amplification of pSB3K5 for pBAD-RNAPα and B2HR ver 2-1 cloning</TD></TR><TR><TD><A style="color:#424242;" href="#Entry37">2016/10/13</A></TD><TD>PCR-amplification optimization of B2HR ver 2-1 gBlock from pJET clone</TD></TR><TR><TD><A style="color:#424242;" href="#Entry38">2016/10/17</A></TD><TD>PCR-amplification of BioBrick parts for cloning into pSB1C3 and pSB3K5</TD></TR><TR><TD><A style="color:#424242;" href="#Entry39">2016/10/18</A></TD><TD>Restriction digest of B2HR ver 2.1 and RNAPα pBAD for cloning into pSB1C3 and PSB3K3</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="container" id="Entry8"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">Transformation of plasmids into NEB5α cells</SPAN><H6 class="grey-text text-lighten-1">2016/12/29</H6><P class="flow-text">
          The plasmids obtained from the registry were transformed into NEB5α cells. 
        </P><OL><LI>Resuspend DNA from kit plate in 10 µl of sterile MilliQ water</LI><TABLE><THEAD><TR><TH data-field="id">Plasmid</TH><TH data-field="name">Well</TH><TH data-field="price">Kit plate (year)</TH></TR></THEAD><TBODY><TR><TD>pSB3K3</TD><TD>6F</TD><TD>Kit plate 4 (2016)</TD></TR><TR><TD>pSB3C5</TD><TD>4D</TD><TD>Kit plate 4 (2015)</TD></TR><TR><TD>pSB1T3</TD><TD>8B</TD><TD>Kit plate 4 (2015)</TD></TR><TR><TD>pSB4K5</TD><TD>6H</TD><TD>Kit plate 4 (2016)</TD></TR></TBODY></TABLE><LI>Add 1 µl of DNA to high efficiency NEB5α cells</LI><LI>Incubate on ice for 30 minutes</LI><LI>Heatschok at 42 °C for 45 seconds. Keep reactions on ice for the following 5 minutes</LI><LI>Add 400 µl of LB media and incubate at 37 °C for 1 hour</LI><LI>Plate 200 µl of each reaction in the LB plates containing the respective antibiotic</LI></OL><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry9"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">PCR amplification of pSB3C5_Cl::Nb and Anti-GFP Nb for SLIC assembly</SPAN><H6 class="grey-text text-lighten-1">2016/05/11</H6><P class="flow-text">
          To linearize the backbone (pSB3C5_Cl::Nb) use primers BioBrick suffix forward (5' TACTAGTAGCGGCCGCTGCAG 3') and oNb46 (5' CATACTCGAGGCTGCGGCGC 3'). The amplification of anti-GFP Nb (BH-GBP3 gBlock) was done using oNb51 (5' GCGCCGCAGCCTCGAGTATGGATGTGCAACTGCAAGAATCAG 3') and BioBrick suffix reverse (5' CTGCAGCGGCCGCTACTAGTA 3').
               </P><P>
                 Table 1. Reagents and the volumes used to set up various PCR reactions
               </P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>2x Phusion MM, HF buffer</TD><TD>25</TD></TR><TR><TD>Forward primer, 10 µM</TD><TD>2.5</TD></TR><TR><TD>Reverse primer, 10 µM</TD><TD>2.5</TD></TR><TR><TD>DNA template (10 ng)</TD><TD>1</TD></TR><TR><TD>MilliQ water</TD><TD>19</TD></TR></TBODY></TABLE><P>
                  Table 2. PCR cycle used for the amplification of different DNA fragments
                <TABLE><THEAD><TR><TH data-field="Step">Step</TH><TH data-field="Temperature">Temperature (°C )</TH><TH data-field="Time">Duration (s)</TH><TH data-field="Cycles">Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98</TD><TD>30</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>15</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>58</TD><TD>20</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>45</TD><TD>30</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>300</TD><TD>1</TD></TR></TBODY></TABLE></P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV>&lt;<DIV class="container" id="Entry10"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">PCR amplification of pSB1T3 and RNAPα::target for SLIC</SPAN><H6 class="grey-text text-lighten-1">2016/05/13</H6><P> The plasmid was linearized using the BioBrick prefix forward (5' GAATTCGCGGCCGCTTCTAGAG 3') and BioBrick reverse suffix, while the gBlock (RNAPα::target) was amplified using BioBrick prefix reverse (5' CTCTAGAAGCGGCCGCGAATTC 3') and BioBrick suffix forward.
            </P>
            Table 1. Reagents and the volumes used to set up various PCR reactions
          <TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>2x Phusion MM, HF buffer</TD><TD>25</TD></TR><TR><TD>Forward primer, 10 µM</TD><TD>2.5</TD></TR><TR><TD>Reverse primer, 10 µM</TD><TD>2.5</TD></TR><TR><TD>DNA template (10 ng)</TD><TD>1</TD></TR><TR><TD>MilliQ water</TD><TD>19</TD></TR></TBODY></TABLE><P>
             Table 2. PCR cycle used for the amplification of different DNA fragments
           <TABLE><THEAD><TR><TH data-field="Step">Step</TH><TH data-field="Temperature">Temperature (°C )</TH><TH data-field="Time">Duration (s)</TH><TH data-field="Cycles">Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98</TD><TD>30</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>15</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>71.7</TD><TD>20</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>45</TD><TD>30</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>300</TD><TD>1</TD></TR></TBODY></TABLE></P><P> Note: Template used for the amplification of RNAPα::target was the wrong one. Use gBlock of pJET clone in future experiments.</P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry11"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">Large scale PCR of pSB1T3 and RNAPα::target scaffold to generate more DNA for chewback reactions.</SPAN><H6 class="grey-text text-lighten-1">2016/05/18</H6><P>
       The previous PCR did not yield enough DNA to carry out the second step of SLIC, a chewback reaction. In this step, the DNA is made single stranded by taking advantage of the exonuclease activity of T4 DNA polymerase. 
     </P>
     Table 1. Reagents and the volumes used to set up various PCR reactions
   <TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>5x HF Phusion Buffer</TD><TD>10</TD></TR><TR><TD>Phusion (2U/µl)</TD><TD>1</TD></TR><TR><TD>dNTPs, 10 mM</TD><TD>1</TD></TR><TR><TD>Reverse primer, 10 µM</TD><TD>2.5</TD></TR><TR><TD>Forward primer, 10 µM</TD><TD>2.5</TD></TR><TR><TD>DNA template  (10 ng)</TD><TD>1</TD></TR><TR><TD>MilliQ water</TD><TD>32</TD></TR></TBODY></TABLE><P>
      Table 2. PCR cycle used for the amplification of different DNA fragments
    <TABLE><THEAD><TR><TH data-field="Step">Step</TH><TH data-field="Temperature">Temperature (°C )</TH><TH data-field="Time">Duration (s)</TH><TH data-field="Cycles">Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98</TD><TD>30</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>15</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>71.7</TD><TD>20</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>45</TD><TD>30</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>300</TD><TD>1</TD></TR></TBODY></TABLE></P><P> Note: Template used for the amplification of RNAPα::target was the wrong one. Use gBlock of pJET clone in future experiments.</P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry12"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">SLIC assembly of pSB1T3_RNAPα::target.</SPAN><H6 class="grey-text text-lighten-1">2016/05/19</H6><P>
Table 1. Quantification of the purified PCR products using a BioDrop
</P><TABLE><THEAD><TR><TH data-field="sample">Sample</TH><TH data-field="DNA conc">[DNA](ng/µl)</TH><TH data-field="Absorbance">A260/280</TH></TR></THEAD><TBODY><TR><TD>pSB1T3 SLIC PCR</TD><TD>78.18</TD><TD>1.854</TD></TR><TR><TD>RNAPα::target SLIC PCR</TD><TD>50.24</TD><TD>1.844</TD></TR></TBODY></TABLE><P>Chewbackk reactions of 3' ends using T4 DNA polymerase</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>10x NEBuffer 2.1</TD><TD>2</TD></TR><TR><TD>DNA fragments (~ 1000 ng)</TD><TD>16</TD></TR><TR><TD>MilliQ water</TD><TD>0</TD></TR><TR><TD>T4 DNA polymerase (0.3 U/µl)</TD><TD>1</TD></TR><TR><TD>DpnI (10 U/µl)/td&gt;
   </TD><TD>1</TD></TR></TBODY></TABLE><P>Incubate reactions for 30 minutes in a 37 °C  waterbath</P><P>Add 2 µl of 10 mM dCTP to arrest the reaction. Inmediately transfer the tubes to an ice box</P><P>Assembly of DNA fragments</P><TABLE><THEAD><TR><TH data-field="Fragment">Fragment</TH><TH data-field="DNA size">DNA size (bp)</TH><TH data-field="Relative size">Relative size</TH><TH data-field="Equivalent DNA amount">Equivalent DNA amount (ng)</TH><TH data-field="[Chewback]">[Chewback] (ng/µl)</TH><TH data-field="Amount of chewback needed">Amount of chewback needed (µl)</TH></TR></THEAD><TBODY><TR><TD>pSB1T3 SLIC PCR</TD><TD>2497</TD><TD>1</TD><TD>1000</TD><TD>60</TD><TD>16</TD></TR><TR><TD>RNAPα::target SLIC PCR</TD><TD>1028</TD><TD>0.41</TD><TD>410</TD><TD>40</TD><TD>10.25</TD></TR></TBODY></TABLE><P>Mix the required DNA amounts in a microcentrifuge tube and incubate for 10 minutes at 37 °C in a waterbath.</P><P>Clean up reaction using the BioBasic enzymatic protocol.</P><P>Electroporate 100 ng of DNA into fresh electrocompetent E. coli BL21(DE3) gold.</P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry13"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">PCR amplification for SLIC assembly of pSB3C5_Cl::Nb_Anti-GFP Nb</SPAN><H6 class="grey-text text-lighten-1">2016/05/26</H6><P class="flow-text">More PCR product is needed to do the chewback reaction. Large scale PCRs were set up for this purpose</P>
      Table 1. Reagents and the volumes used to set up various PCR reactions
    <TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>2x Phusion MM, HF buffer</TD><TD>75</TD></TR><TR><TD>Reverse primer, 10 µM</TD><TD>7.5</TD></TR><TR><TD>Forward primer, 10 µM</TD><TD>7.5</TD></TR><TR><TD>DNA template  (10 ng)</TD><TD>3</TD></TR><TR><TD>MilliQ water</TD><TD>57</TD></TR></TBODY></TABLE><P>
       Table 2. PCR cycle used for the amplification of different DNA fragments
     <TABLE><THEAD><TR><TH data-field="Step">Step</TH><TH data-field="Temperature">Temperature (°C )</TH><TH data-field="Time">Duration (s)</TH><TH data-field="Cycles">Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98</TD><TD>30</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>15</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>58</TD><TD>20</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>45</TD><TD>30</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>300</TD><TD>1</TD></TR></TBODY></TABLE></P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry14"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">SLIC assembly of pSB3C5_Cl::Nb_GBP3</SPAN><H6 class="grey-text text-lighten-1">2016/06/01</H6><P>
                    Table 1. Quantification of the purified PCR products using a BioDrop
                  </P><TABLE><THEAD><TR><TH data-field="sample">Sample</TH><TH data-field="DNA conc">[DNA](ng/µl)</TH><TH data-field="Absorbance">A260/280</TH></TR></THEAD><TBODY><TR><TD>pSB3C5_Cl::Nb SLIC PCR</TD><TD>274.6</TD><TD>1.849</TD></TR><TR><TD>GBP3 SLIC PCR</TD><TD>178.8</TD><TD>1.829</TD></TR></TBODY></TABLE><P>
                   Chewbackk reactions of 3' ends using T4 DNA polymerase
</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>10x NEBuffer 2.1</TD><TD>2</TD></TR><TR><TD>DNA fragments (~ 1000 ng)</TD><TD>6</TD></TR><TR><TD>MilliQ water</TD><TD>10</TD></TR><TR><TD>T4 DNA polymerase (0.3 U/µl)</TD><TD>1</TD></TR><TR><TD>DpnI (10 U/µl)/td&gt;
                              </TD><TD>1</TD></TR></TBODY></TABLE><P>Incubate reactions for 30 minutes in a 37 °C  waterbath</P><P>Add 2 µl of 10 mM dCTP to arrest the reaction. Inmediately transfer the tubes to an ice box</P><P>Assembly of DNA fragments</P><TABLE><THEAD><TR><TH data-field="Fragment">Fragment</TH><TH data-field="DNA size">DNA size (bp)</TH><TH data-field="Relative size">Relative size</TH><TH data-field="Equivalent DNA amount">Equivalent DNA amount (ng)</TH><TH data-field="[Chewback]">[Chewback] (ng/µl)</TH><TH data-field="Amount of chewback needed">Amount of chewback needed (µl)</TH></TR></THEAD><TBODY><TR><TD>pSB3C5_Cl::Nb SLIC PCR</TD><TD>3823</TD><TD>1</TD><TD>1000</TD><TD>50</TD><TD>18</TD></TR><TR><TD>GBP3 PCR</TD><TD>459</TD><TD>0.12</TD><TD>120</TD><TD>50</TD><TD>2.5</TD></TR></TBODY></TABLE><P>Mix the required DNA amounts in a microcentrifuge tube and incubate for 10 minutes at 37 °C in a waterbath</P><P>Clean up reaction using the BioBasic enzymatic protocol</P><P>Electroporate 100 ng of DNA into fresh electrocompetent E. coli BL21(DE3) gold</P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry15"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">Amplification of GFP (BBa_E0040) USING oNb52 and oNb47</SPAN><H6 class="grey-text text-lighten-1">2016/06/01</H6><P class="flow-text">The GFP coding sequence was amplified from pSB2A1 using oNb52 (5' CCAGATCTGCGGCGGCAAGAATCATGCGTAAAGGAGAAGAACTTTTCACTGG 3') and oNb47 (5' GATTCTTGCCGCCGCAGATC 3')</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>5x HF Phusion Buffer</TD><TD>4</TD></TR><TR><TD>Phusion (2U/µl)</TD><TD>0.5</TD></TR><TR><TD>dNTPs, 10 mM</TD><TD>0.5</TD></TR><TR><TD>oNb52, 10 µM</TD><TD>1</TD></TR><TR><TD>oNb47, 10 µM</TD><TD>1</TD></TR><TR><TD>pSB2A1  (10 ng)</TD><TD>1</TD></TR><TR><TD>MilliQ water</TD><TD>12</TD></TR></TBODY></TABLE><P>
          Table 2. PCR cycle used for the amplification of different DNA fragments
        <TABLE><THEAD><TR><TH data-field="Step">Step</TH><TH data-field="Temperature">Temperature (°C )</TH><TH data-field="Time">Duration (s)</TH><TH data-field="Cycles">Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98</TD><TD>30</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>15</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>X</TD><TD>20</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>45</TD><TD>30</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>300</TD><TD>1</TD></TR></TBODY></TABLE></P><P> A gradient PCR was done to amplify GFP. The thermocycler was set up in a way that each well had a different temperature (51 to 61°C)</P><P>The DNA fragments obtained were approximately 750 bp, the expected size of the GFP CDS</P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry16"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">Colony PCR of potential pSB3C5_Cl::Nb_GBP3 clones</SPAN><H6 class="grey-text text-lighten-1">2016/06/02</H6><P class="flow-text">The electroporation of the pSB3C5_Cl::Nb_GBP3 was succesful. Each plate had approximately 40 CFU.</P><P> Randome colonies were selected to set up colony PCRs using BioBrick suffix reverse and oNb51 as primers, ideally a DNA fragment of the size of the GBP3 gBlock should be observed.</P><P> * Using a pipette tip, a little bit of the CFU was mixed with 50 µl of MilliQ water. Only 1 µl of the mixture was added to the PCR tube</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>10x Pfu buffer</TD><TD>2</TD></TR><TR><TD>Pfu</TD><TD>0.5</TD></TR><TR><TD>dNTPs, 10 mM</TD><TD>0.5</TD></TR><TR><TD>oNb51, 10 µM</TD><TD>1</TD></TR><TR><TD>BioBrick suffix reverse, 10 µM</TD><TD>1</TD></TR><TR><TD>Colony dilution(*)</TD><TD>1</TD></TR><TR><TD>MilliQ water</TD><TD>14</TD></TR></TBODY></TABLE><P>
          Table 2. PCR cycle used for the amplification of different DNA fragments
        <TABLE><THEAD><TR><TH data-field="Step">Step</TH><TH data-field="Temperature">Temperature (°C )</TH><TH data-field="Time">Duration (s)</TH><TH data-field="Cycles">Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98</TD><TD>120</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>30</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>58</TD><TD>30</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>30</TD><TD>30</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>300</TD><TD>1</TD></TR></TBODY></TABLE></P><P>The small fragment (~450 bp) belong to the GBP3 gBlock. Indicating that the construct was assembled</P><P>The colonies containing the DNA fragment were used to set up O/N cultures. The plasmid was isolated and sent for sequencing</P><P>The sequencing results determined that the GBP3 gBlock was not present in any of the plasmids. Repeat SLIC assembly </P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry17"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">Screening of potential pSBT3_RNAPα::target clones</SPAN><H6 class="grey-text text-lighten-1">2016/06/05</H6><P class="flow-text">BglII digest of pSB1T3_RNAPα::target clones to determine whether the contruct was assembled.</P><P>The RNAPα::target gBlock has a BglII unique cutting site on the 3' of its sequence.</P><P>The presence of a single cut indicates a positive clone.</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>BglII (ThermoFisher)/td&gt;
              </TD><TD>1</TD></TR><TR><TD>Buffer O (ThermoFisher)</TD><TD>2</TD></TR><TR><TD>plasmid DNA (100 ng)</TD><TD>5</TD></TR><TR><TD>MilliQ</TD><TD>12</TD></TR></TBODY></TABLE><P>
          Incubate for 1 hour at 37 °C
        </P><P>
          Agarose electrophoresis of results. </P><P>
            Clones 8, 9, 11, 12 cut once. The plasmids were sent for sequence analysis</P><P>
          The sequencing results showed that there is a missing region in the RNAPα::target gBlock in the 5' region.
        </P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry18"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">Single domain antibody library (sdAb) scaffold</SPAN><H6 class="grey-text text-lighten-1">2016/06/25</H6><P class="flow-text">Amplification of the sdAb scaffold via PCR</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>2x Phusion MM, HF buffer</TD><TD>50</TD></TR><TR><TD>oNb45, 10 µM</TD><TD>5</TD></TR><TR><TD>BioBrick suffix reverse, 10 µM</TD><TD>5</TD></TR><TR><TD>pSB3C5_Cl::Nb (Scaffold) gBlock(10 ng)</TD><TD>1</TD></TR><TR><TD>MilliQ water</TD><TD>39</TD></TR></TBODY></TABLE><P>PCR cycle for the amplification of the sdAb scaffold</P><TABLE><THEAD><TR><TH data-field="Step">Step</TH><TH data-field="Temperature">Temperature (°C )</TH><TH data-field="Time">Duration (s)</TH><TH data-field="Cycles">Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98</TD><TD>30</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>15</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>68</TD><TD>15</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>30</TD><TD>30</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>120</TD><TD>1</TD></TR></TBODY></TABLE><P>PCR worked well, however, several larger bands also appeared. They are likely multimers of PCR products as they occur in ~300 bp increments.</P><P>The multimers should not affect downstream digestion and cloning steps</P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry19"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">Restriction digest of sdAb scaffold</SPAN><H6 class="grey-text text-lighten-1">2016/08/02</H6><P class="flow-text">Double digest of sdAb scaffold PCR using NheI and BamHI</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>10x Fast Digest buffer</TD><TD>4</TD></TR><TR><TD>sdAb scaffold PCR (103 ng/µl)</TD><TD>30</TD></TR><TR><TD>NheI Fast Digest (ThermoScientific)</TD><TD>3</TD></TR><TR><TD>BamHI Fast Digest (ThermoScientific)</TD><TD>3</TD></TR></TBODY></TABLE><P> Incubate for 30 minutes at 37 °C</P><P>Spin column purification (BioBasic) and elute DNA in 40 µl MilliQ water</P><P>Set up NgoMIV difest using the eluted DNA</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>10x CutSmart buffer</TD><TD>10</TD></TR><TR><TD>sdAb scaffold PCR (NheI and BamHI digested)</TD><TD>40</TD></TR><TR><TD>NgoMIV (10 U/µl)</TD><TD>2</TD></TR><TR><TD>MilliQ water</TD><TD>48</TD></TR></TBODY></TABLE><P> Incubate for 30 minutes at 37 °C</P><P>Spin column purification (BioBasic) and elute DNA in 40 µl MilliQ water</P><P>The sample concentration is approximately 64 ng/µl</P><P>KpnI digestion of sdAb scaffold</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>10x KpnI buffer</TD><TD>10</TD></TR><TR><TD>sdAb scaffold PCR (NheI, BamHI, NgoMIV digest)</TD><TD>40</TD></TR><TR><TD>KpnI (10 U/µl)</TD><TD>2</TD></TR><TR><TD>MilliQ water</TD><TD>48</TD></TR></TBODY></TABLE><P> Incubate for 30 minutes at 37 °C</P><P>Spin column purification (BioBasic) and elute DNA in 40 µl MilliQ water</P><DIV class="card-action"><A href="#">Previous</A><A href="#">Next</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry20"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">Generation of CR1-2 library by oligo assembly and PCR</SPAN><H6 class="grey-text text-lighten-1">2016/08/03</H6><P class="flow-text">Oligo annealing of CDR1 and Framework region 2</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>oNb, 100 µM</TD><TD>2</TD></TR><TR><TD>oNb6, 100 µM</TD><TD>2</TD></TR><TR><TD>MilliQ water</TD><TD>46</TD></TR></TBODY></TABLE><P> Heat oligo mixture to 95°C, then slowly cool on heat block (turn heat block off and let it reach room temperature)</P><P>Extend oligos using T4 DNA polymerase</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>Annealed oligos (4 µM)</TD><TD>43</TD></TR><TR><TD>10x NEBuffer 2.1,</TD><TD>5</TD></TR><TR><TD>dNTPs, 10 mM</TD><TD>1</TD></TR><TR><TD>t4 DNA polymerase (3 U/µl)</TD><TD>1</TD></TR></TBODY></TABLE><P> Incubate for 30 minutes at 12 °C for 15 minutes on thermocycler, then place on ice, clean up on spin column and elute in 40 µl of MilliQ water</P><P>Use 2 µl extended oligos as template for PCR (10 pmol)</P><P>PCR amplification of CDR1-2 oligos</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>2x Phusion MM, HF buffer</TD><TD>25</TD></TR><TR><TD>Annealed/ Extended oligos (50 pmol)</TD><TD>10</TD></TR><TR><TD>oNb 7, 10 µM</TD><TD>1</TD></TR><TR><TD>MilliQ water</TD><TD>14</TD></TR></TBODY></TABLE><TABLE><THEAD><TR><TH data-field="Step">Step</TH><TH data-field="Temperature">Temperature (°C )</TH><TH data-field="Time">Duration (s)</TH><TH data-field="Cycles">Cycles</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98</TD><TD>120</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98</TD><TD>20</TD><TD>5</TD></TR><TR><TD>Annealing</TD><TD>72</TD><TD>20</TD><TD>5</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>120</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV class="container" id="Entry21"><DIV class="row"><DIV class="col s12 m12 l12"><DIV class="card blue-grey darken-1"><DIV class="card-content white-text"><SPAN class="card-title">Generation of CDR3 libraries</SPAN><H6 class="grey-text text-lighten-1">2016/08/03</H6><P class="flow-text">Oligo annealing of various CDR3 DNA fragments</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>Forward oligo (oNb8, 9, 10, 11, 12), 100 µM</TD><TD>2</TD></TR><TR><TD>oNb13, 100 µM</TD><TD>2</TD></TR><TR><TD>MilliQ water</TD><TD>46</TD></TR></TBODY></TABLE><P> Heat oligo mixture to 95°C, then slowly cool on heat block (turn heat block off and let it reach room temperature)</P><P>Extend oligos using T4 DNA polymerase</P><TABLE><THEAD><TR><TH data-field="Reagent">Reagent</TH><TH data-field="Volume">Volume (µl)</TH></TR></THEAD><TBODY><TR><TD>Annealed oligos (4 µM)</TD><TD>43</TD></TR><TR><TD>10x NEBuffer 2.1,</TD><TD>5</TD></TR><TR><TD>dNTPs, 10 mM</TD><TD>1</TD></TR><TR><TD>t4 DNA polymerase (3 U/µl)</TD><TD>1</TD></TR></TBODY></TABLE><P> Incubate for 30 minutes at 12 °C for 15 minutes on thermocycler, then place on ice, clean up on spin column and elute in 40 µl of MilliQ water</P><P>Phenol/chloroform extract DNA</P>\
                                                    
      </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>