Team:Lethbridge/Notebook
Lethbridge iGEM 2016
Lethbridge iGEM
Project
Description
Design
Experiment
Database
Notebook
Parts
Safety
RNAi
Practices
Integrated Practices
Contact
Interviews
Literature Review
Ridealongs
Protocol
Sampling
Results
Team
Members
Collab
Attribution
Sponsors
Achievements
menu
Project
Description
Design
Experiments
Database
Notebook
Parts
Safety
RNAi
Practices
Integrated Practices
Intro
Interviews
Literature Review
Ridealongs
Protocol
Sampling
Results
Team
Members
Collaborations
Attribution
Sponsors
Achievements
Notebook
The various facets of our lab work were divided between several key areas. Please explore!
Nanopore Microbiome Analysis
Single-domain antibody development system
Nanopore Microbiome Analysis
Date
Experiment
2016/08/16
DNA extraction protocol
2016/08/16
Multiple displacement amplification of EMS - Week #1
2016/08/24
16S rRNA gene amplification from EMS samples (week #4)
2016/08/31
Optimization of PCR amplification using various rDNA gene primer pairs
2016/08/31
Large scale PCR amplification using 16S and 18S rRNA
2016/10/4
DNA extraction - Week #4 samples
2016/10/5
Amplify 1N4, 2N4, 3N4, Pseud., virus control, E. coli control
DNA extraction protocol
2016/08/16
Protocol for the extraction of DNA from samples taken from EMS ride-alongs.
Ethanol evaporation for approximately 2 hours on the rotary evaporator on high heat, under vacuum and high speed.
Add 400 µl of alkaline lysis buffer (200 mM KOH, 50 mM DTT).
Mix by inverting tube several times.
Incubate on heat block at 65 °C for 10 minutes.
Centrifuge at 17,000 x g for 1 minute.
Pipette liquid volume (as much as possible) into new 1.5 mL microcentrifuge tube (~300 µl)
Add 1/10 of the volume (~30 µl) of 3 M sodium acetate (pH 5.3) to each tube.
Add 5 µl of 0.5% LPA (linear polyacrylamide) carrier to each tube.
Add 750 µl of cold 100% ethanol to each tube, mix by inverting and place tubes on ice for approximate 10 minutes.
Centrifuge at 17,000 x g for 5 minutes.
Remove supernatant.
Wash with 500 µl of 70% ethanol and mix gently. Push centrifuge and remove supernatant.
Repeat wash step (step 12)
Vacuum evaporate the ethanol under high heat for 10 minutes.
Resuspend in 20 µl of filtered MilliQ water.
Single Domain Antibody Development System
Date
Experiment
2016/05/01
Transformation of plasmids into NEB5α cells
2016/05/09
Set up of O/N cultures to isolate plasmids for SLIC
2016/05/13
PCR amplification of RNAPα::taget scaffold gBlock
2016/05/18
PCR amplification of pSB1T3 and RNAPα::target for T4 DNA polymerase chewback reaction
2016/05/19
SLIC assembly of pSB1T3-RNAPα::target
2016/05/26
PCR amplification for SLIC assembly of pSB3C5_Cl::Nb_Anti-GFP Nb
2016/06/1
SLIC assembly of pSB3C5_Cl::Nb_Anti-GFP Nb
2016/06/01
Amplification of GFP (BBa_E0040) using oNb52 and oNb47
2016/06/02
Colony PCR of potential pSB3C5_Cl::Nb_Anti-GFP Nb clones
2016/06/05
Screening of pSB1T3_RNAPα::target clones using BglII restriction digest
2016/06/25
Single-domain antibody library cloning
2016/07/25
Restriction digest of single-domain antibody (sdAb) scaffold
2016/08/3
Generation of CDR1-2 library by oligo assembly and PCR
2016/08/4
Generation of CDR3 libraries
2016/08/08
Ligation of CDR1-2 and sdAb scaffold
2016/08/10
CDR3 NgoMIV digestion
2016/08/11
KpnI and NgoMIV digests of sdAb scaffold and CDR1-2
2016/08/16
Ligation PCR of CDR3s and sdAb/CDR1-2
2016/08/18
CloneJet cloning of sdAb scaffold/CDR1-2_CDR3 ligation PCRs
2016/08/31
PCR amplification and overlap extension PCR assembly of sdAb library parts
2016/09/07
Overlap extension PCR assembly of sdAb library
2016/09/07
Overlap extension PCR of sdAb 5' CDR1-2 with CDR3 sdAb 3'
2016/09/26
Scale up PCR amplification of sdAb libraries
2016/09/27
PCR amplification of pSB3C5_Cl::Nb backbone for sdAb library SLIC assembly
2016/09/28
SLIC assembly of sdAb library into pSB3C5_Cl::Nb
2016/09/29
sdAb library SLIC assembly electroporation results
2016/10/07
Colony PCR screening of pSB4K5_pBAD RNAPα
2016/10/12
PCR amplification of pSB3K5 for pBAD-RNAPα and B2HR ver 2-1 cloning
2016/10/13
PCR-amplification optimization of B2HR ver 2-1 gBlock from pJET clone
2016/10/17
PCR-amplification of BioBrick parts for cloning into pSB1C3 and pSB3K5
2016/10/18
Restriction digest of B2HR ver 2.1 and RNAPα pBAD for cloning into pSB1C3 and PSB3K3
Transformation of plasmids into NEB5α cells
2016/12/29
The plasmids obtained from the registry were transformed into NEB5α cells.
Resuspend DNA from kit plate in 10 µl of sterile MilliQ water
Plasmid
Well
Kit plate (year)
pSB3K3
6F
Kit plate 4 (2016)
pSB3C5
4D
Kit plate 4 (2015)
pSB1T3
8B
Kit plate 4 (2015)
pSB4K5
6H
Kit plate 4 (2016)
Add 1 µl of DNA to high efficiency NEB5α cells
Incubate on ice for 30 minutes
Heatschok at 42 °C for 45 seconds. Keep reactions on ice for the following 5 minutes
Add 400 µl of LB media and incubate at 37 °C for 1 hour
Plate 200 µl of each reaction in the LB plates containing the respective antibiotic
Previous Next
PCR amplification of pSB3C5_Cl::Nb and Anti-GFP Nb for SLIC assembly
2016/05/11
To linearize the backbone (pSB3C5_Cl::Nb) use primers BioBrick suffix forward (5' TACTAGTAGCGGCCGCTGCAG 3') and oNb46 (5' CATACTCGAGGCTGCGGCGC 3'). The amplification of anti-GFP Nb (BH-GBP3 gBlock) was done using oNb51 (5' GCGCCGCAGCCTCGAGTATGGATGTGCAACTGCAAGAATCAG 3') and BioBrick suffix reverse (5' CTGCAGCGGCCGCTACTAGTA 3').
Table 1. Reagents and the volumes used to set up various PCR reactions
Reagent
Volume (µl)
2x Phusion MM, HF buffer
25
Forward primer, 10 µM
2.5
Reverse primer, 10 µM
2.5
DNA template (10 ng)
1
MilliQ water
19
Table 2. PCR cycle used for the amplification of different DNA fragments
Step
Temperature (°C )
Duration (s)
Cycles
Initial denaturation
98
30
1
Denaturation
98
15
30
Annealing
58
20
30
Extension
72
45
30
Final extension
72
300
1
Previous Next
<
PCR amplification of pSB1T3 and RNAPα::target for SLIC
2016/05/13
The plasmid was linearized using the BioBrick prefix forward (5' GAATTCGCGGCCGCTTCTAGAG 3') and BioBrick reverse suffix, while the gBlock (RNAPα::target) was amplified using BioBrick prefix reverse (5' CTCTAGAAGCGGCCGCGAATTC 3') and BioBrick suffix forward.
Table 1. Reagents and the volumes used to set up various PCR reactions
Reagent
Volume (µl)
2x Phusion MM, HF buffer
25
Forward primer, 10 µM
2.5
Reverse primer, 10 µM
2.5
DNA template (10 ng)
1
MilliQ water
19
Table 2. PCR cycle used for the amplification of different DNA fragments
Step
Temperature (°C )
Duration (s)
Cycles
Initial denaturation
98
30
1
Denaturation
98
15
30
Annealing
71.7
20
30
Extension
72
45
30
Final extension
72
300
1
Note: Template used for the amplification of RNAPα::target was the wrong one. Use gBlock of pJET clone in future experiments.
Previous Next
Large scale PCR of pSB1T3 and RNAPα::target scaffold to generate more DNA for chewback reactions.
2016/05/18
The previous PCR did not yield enough DNA to carry out the second step of SLIC, a chewback reaction. In this step, the DNA is made single stranded by taking advantage of the exonuclease activity of T4 DNA polymerase.
Table 1. Reagents and the volumes used to set up various PCR reactions
Reagent
Volume (µl)
5x HF Phusion Buffer
10
Phusion (2U/µl)
1
dNTPs, 10 mM
1
Reverse primer, 10 µM
2.5
Forward primer, 10 µM
2.5
DNA template  (10 ng)
1
MilliQ water
32
Table 2. PCR cycle used for the amplification of different DNA fragments
Step
Temperature (°C )
Duration (s)
Cycles
Initial denaturation
98
30
1
Denaturation
98
15
30
Annealing
71.7
20
30
Extension
72
45
30
Final extension
72
300
1
Note: Template used for the amplification of RNAPα::target was the wrong one. Use gBlock of pJET clone in future experiments.
Previous Next
SLIC assembly of pSB1T3_RNAPα::target.
2016/05/19
Table 1. Quantification of the purified PCR products using a BioDrop
Sample
[DNA](ng/µl)
A260/280
pSB1T3 SLIC PCR
78.18
1.854
RNAPα::target SLIC PCR
50.24
1.844
Chewbackk reactions of 3' ends using T4 DNA polymerase
Reagent
Volume (µl)
10x NEBuffer 2.1
2
DNA fragments (~ 1000 ng)
16
MilliQ water
0
T4 DNA polymerase (0.3 U/µl)
1
DpnI (10 U/µl)/td>
1
Incubate reactions for 30 minutes in a 37 °C  waterbath
Add 2 µl of 10 mM dCTP to arrest the reaction. Inmediately transfer the tubes to an ice box
Assembly of DNA fragments
Fragment
DNA size (bp)
Relative size
Equivalent DNA amount (ng)
[Chewback] (ng/µl)
Amount of chewback needed (µl)
pSB1T3 SLIC PCR
2497
1
1000
60
16
RNAPα::target SLIC PCR
1028
0.41
410
40
10.25
Mix the required DNA amounts in a microcentrifuge tube and incubate for 10 minutes at 37 °C in a waterbath.
Clean up reaction using the BioBasic enzymatic protocol.
Electroporate 100 ng of DNA into fresh electrocompetent E. coli BL21(DE3) gold.
Previous Next
PCR amplification for SLIC assembly of pSB3C5_Cl::Nb_Anti-GFP Nb
2016/05/26
More PCR product is needed to do the chewback reaction. Large scale PCRs were set up for this purpose
Table 1. Reagents and the volumes used to set up various PCR reactions
Reagent
Volume (µl)
2x Phusion MM, HF buffer
75
Reverse primer, 10 µM
7.5
Forward primer, 10 µM
7.5
DNA template  (10 ng)
3
MilliQ water
57
Table 2. PCR cycle used for the amplification of different DNA fragments
Step
Temperature (°C )
Duration (s)
Cycles
Initial denaturation
98
30
1
Denaturation
98
15
30
Annealing
58
20
30
Extension
72
45
30
Final extension
72
300
1
Previous Next
SLIC assembly of pSB3C5_Cl::Nb_GBP3
2016/06/01
Table 1. Quantification of the purified PCR products using a BioDrop
Sample
[DNA](ng/µl)
A260/280
pSB3C5_Cl::Nb SLIC PCR
274.6
1.849
GBP3 SLIC PCR
178.8
1.829
Chewbackk reactions of 3' ends using T4 DNA polymerase
Reagent
Volume (µl)
10x NEBuffer 2.1
2
DNA fragments (~ 1000 ng)
6
MilliQ water
10
T4 DNA polymerase (0.3 U/µl)
1
DpnI (10 U/µl)/td>
1
Incubate reactions for 30 minutes in a 37 °C  waterbath
Add 2 µl of 10 mM dCTP to arrest the reaction. Inmediately transfer the tubes to an ice box
Assembly of DNA fragments
Fragment
DNA size (bp)
Relative size
Equivalent DNA amount (ng)
[Chewback] (ng/µl)
Amount of chewback needed (µl)
pSB3C5_Cl::Nb SLIC PCR
3823
1
1000
50
18
GBP3 PCR
459
0.12
120
50
2.5
Mix the required DNA amounts in a microcentrifuge tube and incubate for 10 minutes at 37 °C in a waterbath
Clean up reaction using the BioBasic enzymatic protocol
Electroporate 100 ng of DNA into fresh electrocompetent E. coli BL21(DE3) gold
Previous Next
Amplification of GFP (BBa_E0040) USING oNb52 and oNb47
2016/06/01
The GFP coding sequence was amplified from pSB2A1 using oNb52 (5' CCAGATCTGCGGCGGCAAGAATCATGCGTAAAGGAGAAGAACTTTTCACTGG 3') and oNb47 (5' GATTCTTGCCGCCGCAGATC 3')
Reagent
Volume (µl)
5x HF Phusion Buffer
4
Phusion (2U/µl)
0.5
dNTPs, 10 mM
0.5
oNb52, 10 µM
1
oNb47, 10 µM
1
pSB2A1  (10 ng)
1
MilliQ water
12
Table 2. PCR cycle used for the amplification of different DNA fragments
Step
Temperature (°C )
Duration (s)
Cycles
Initial denaturation
98
30
1
Denaturation
98
15
30
Annealing
X
20
30
Extension
72
45
30
Final extension
72
300
1
A gradient PCR was done to amplify GFP. The thermocycler was set up in a way that each well had a different temperature (51 to 61°C)
The DNA fragments obtained were approximately 750 bp, the expected size of the GFP CDS
Previous Next
Colony PCR of potential pSB3C5_Cl::Nb_GBP3 clones
2016/06/02
The electroporation of the pSB3C5_Cl::Nb_GBP3 was succesful. Each plate had approximately 40 CFU.
Randome colonies were selected to set up colony PCRs using BioBrick suffix reverse and oNb51 as primers, ideally a DNA fragment of the size of the GBP3 gBlock should be observed.
* Using a pipette tip, a little bit of the CFU was mixed with 50 µl of MilliQ water. Only 1 µl of the mixture was added to the PCR tube
Reagent
Volume (µl)
10x Pfu buffer
2
Pfu
0.5
dNTPs, 10 mM
0.5
oNb51, 10 µM
1
BioBrick suffix reverse, 10 µM
1
Colony dilution(*)
1
MilliQ water
14
Table 2. PCR cycle used for the amplification of different DNA fragments
Step
Temperature (°C )
Duration (s)
Cycles
Initial denaturation
98
120
1
Denaturation
98
30
30
Annealing
58
30
30
Extension
72
30
30
Final extension
72
300
1
The small fragment (~450 bp) belong to the GBP3 gBlock. Indicating that the construct was assembled
The colonies containing the DNA fragment were used to set up O/N cultures. The plasmid was isolated and sent for sequencing
The sequencing results determined that the GBP3 gBlock was not present in any of the plasmids. Repeat SLIC assembly
Previous Next
Screening of potential pSBT3_RNAPα::target clones
2016/06/05
BglII digest of pSB1T3_RNAPα::target clones to determine whether the contruct was assembled.
The RNAPα::target gBlock has a BglII unique cutting site on the 3' of its sequence.
The presence of a single cut indicates a positive clone.
Reagent
Volume (µl)
BglII (ThermoFisher)/td>
1
Buffer O (ThermoFisher)
2
plasmid DNA (100 ng)
5
MilliQ
12
Incubate for 1 hour at 37 °C
Agarose electrophoresis of results.
Clones 8, 9, 11, 12 cut once. The plasmids were sent for sequence analysis
The sequencing results showed that there is a missing region in the RNAPα::target gBlock in the 5' region.
Previous Next
Single domain antibody library (sdAb) scaffold
2016/06/25
Amplification of the sdAb scaffold via PCR
Reagent
Volume (µl)
2x Phusion MM, HF buffer
50
oNb45, 10 µM
5
BioBrick suffix reverse, 10 µM
5
pSB3C5_Cl::Nb (Scaffold) gBlock(10 ng)
1
MilliQ water
39
PCR cycle for the amplification of the sdAb scaffold
Step
Temperature (°C )
Duration (s)
Cycles
Initial denaturation
98
30
1
Denaturation
98
15
30
Annealing
68
15
30
Extension
72
30
30
Final extension
72
120
1
PCR worked well, however, several larger bands also appeared. They are likely multimers of PCR products as they occur in ~300 bp increments.
The multimers should not affect downstream digestion and cloning steps
Previous Next
Restriction digest of sdAb scaffold
2016/08/02
Double digest of sdAb scaffold PCR using NheI and BamHI
Reagent
Volume (µl)
10x Fast Digest buffer
4
sdAb scaffold PCR (103 ng/µl)
30
NheI Fast Digest (ThermoScientific)
3
BamHI Fast Digest (ThermoScientific)
3
Incubate for 30 minutes at 37 °C
Spin column purification (BioBasic) and elute DNA in 40 µl MilliQ water
Set up NgoMIV difest using the eluted DNA
Reagent
Volume (µl)
10x CutSmart buffer
10
sdAb scaffold PCR (NheI and BamHI digested)
40
NgoMIV (10 U/µl)
2
MilliQ water
48
Incubate for 30 minutes at 37 °C
Spin column purification (BioBasic) and elute DNA in 40 µl MilliQ water
The sample concentration is approximately 64 ng/µl
KpnI digestion of sdAb scaffold
Reagent
Volume (µl)
10x KpnI buffer
10
sdAb scaffold PCR (NheI, BamHI, NgoMIV digest)
40
KpnI (10 U/µl)
2
MilliQ water
48
Incubate for 30 minutes at 37 °C
Spin column purification (BioBasic) and elute DNA in 40 µl MilliQ water
Previous Next
Generation of CR1-2 library by oligo assembly and PCR
2016/08/03
Oligo annealing of CDR1 and Framework region 2
Reagent
Volume (µl)
oNb, 100 µM
2
oNb6, 100 µM
2
MilliQ water
46
Heat oligo mixture to 95°C, then slowly cool on heat block (turn heat block off and let it reach room temperature)
Extend oligos using T4 DNA polymerase
Reagent
Volume (µl)
Annealed oligos (4 µM)
43
10x NEBuffer 2.1,
5
dNTPs, 10 mM
1
t4 DNA polymerase (3 U/µl)
1
Incubate for 30 minutes at 12 °C for 15 minutes on thermocycler, then place on ice, clean up on spin column and elute in 40 µl of MilliQ water
Use 2 µl extended oligos as template for PCR (10 pmol)
PCR amplification of CDR1-2 oligos
Reagent
Volume (µl)
2x Phusion MM, HF buffer
25
Annealed/ Extended oligos (50 pmol)
10
oNb 7, 10 µM
1
MilliQ water
14
Step
Temperature (°C )
Duration (s)
Cycles
Initial denaturation
98
120
1
Denaturation
98
20
5
Annealing
72
20
5
Extension
72
120
1
Generation of CDR3 libraries
2016/08/03
Oligo annealing of various CDR3 DNA fragments
Reagent
Volume (µl)
Forward oligo (oNb8, 9, 10, 11, 12), 100 µM
2
oNb13, 100 µM
2
MilliQ water
46
Heat oligo mixture to 95°C, then slowly cool on heat block (turn heat block off and let it reach room temperature)
Extend oligos using T4 DNA polymerase
Reagent
Volume (µl)
Annealed oligos (4 µM)
43
10x NEBuffer 2.1,
5
dNTPs, 10 mM
1
t4 DNA polymerase (3 U/µl)
1
Incubate for 30 minutes at 12 °C for 15 minutes on thermocycler, then place on ice, clean up on spin column and elute in 40 µl of MilliQ water
Phenol/chloroform extract DNA
\
