"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF-Lausanne/Notebook/3 August 2012
From 2012.igem.org
Introduction
Our Project
Notebook
Human Practices
Acknowledgments
Contents
1 PCR
1.1 Protocol: PCR
2 Tips
3 Digestion of pMP and pGL4.30 for ligation
3.1 Protocol: Restriction site digestion
3.2 Protocol: Gel Electrophoresis
4 CH Lab
5 BM Lab
PCR
Protocol: PCR
PCR is a reaction that makes it possible (and relatively easy) to amplify a certain region of DNA. The first step is the selection of that region (and the design of the relevant primers). Primer design can be done by hand, or by using our Primer Design Helper . Once done, order the primers (in our case, we ordered from them IDT ).
When you've received the primers, prepare them and make sure you've got your PCR kit (we used the "Phusion® High-Fidelity DNA Polymerase"). Start preparing your master mix, the composition for one tube is:
1X Mastermix 20μl reaction, add in this order
Reagent
Volume [μl]
Water
Complete to total volume of 20μl
HF-Buffer (5x)
4
DMSO (optional)
0.6
dNTPs
0.4
Forward primer (50μM)
0.2
Reverse primer (50μM)
0.2
Template (10ng/μl)
0.5
Phusion HF polymerase
0.2
Prepare one or two extra tubes-worth of reagent (you'll use some liquid on the walls of your tips).
Once you've finished, you should run the resulting products on a gel to check if everything went as planned.
Tips
Thaw the HF-Buffer, DMSO and dNTPs before making the mastermix.
Avoid taking the Phusion-HF polymerase out of the freezer (only take it out briefly when you need to add it).
If the reactions have different primers and/or template, add the polymerase right after the dNTPs, split the mastermix and add the rest.
Don't forget positive and negative controls
Primers should have similar Tms (less than 5°C).
Primer Tm calculation is a less exact science than it should be (just test several tools and compare their results). If you're not sure what the correct Tm is, consider using a gradient PCR.
Avoid primers with strong secondary structures.
PCR can introduce mutations. Don't forget to sequence your final product (this could be your final plasmid): you really don't want to lose a few weeks because of a "corrupt" plasmid.
Ran a PCR on the readouts aimed to be co-expressed with melanopsin (SEAP, TNFR, eGFP), using primers that would add the restriction sites suitable for ligation into the plasmids of interest.
The master mix for 4 tubes (3 + an excess one) was done as follows
H20: 146 µl
Buffer HF (5x): 40 µl
dNTP (10 mM): 4 µl
Then split into 3 tubes, add the following reagents to every one of them:
Primers:
Forward (50 µM): 0.5 µl
Reverse (50 µM): 0.5 µl
Template DNA: 1 µl
Phusion DNA Polymerase: 0.5 µl
The total volume in each tube should be 50 µl.
The average Tm for our primers was around 69°C.
The thermal cycling protocol follows
Initial denaturation: T = 98°C, 30 sec
Cycle through this 30 times:
Denaturation: T = 98°C, 10 sec
Annealing: T = 69°C, 30 sec
Extension: T = 72°C, 1 min
Final extension: T = 72°C, 7 min
Hold: T = 4°C, stay until pickup
Loosely based on the Phusion kit protocol.
The expected product sizes are
SEAP+polyA: 1890 bp (+ tails)
TNFR: 1384 bp (+ tails)
eGFP: 725 bp (+ tails)
Digestion of pMP and pGL4.30 for ligation
Protocol: Restriction site digestion
Look for the best pair of restriction sites, ideally with similar digestion temperatures and times.
NEBcutter for finding cutting enzymes.
Double Digest Finder for the parameters.
Calculate the amounts required of:
DNA
Buffer (usually from 10x to 1x)
BSA, if needed (usually from 100x to 1x)
Enzymes (depends on the amount of DNA)
Water
Get the recommended buffer (and BSA if needed) from the freezer and let defreeze.
Mix all the ingredients, except DNA, in a tube.
Note: Enzymes should stay no longer than a couple of minutes out of the freezer. Don't touch the bottom of the tubes! Don't vortex!
Distribute the mix in as many tubes as DNA samples and add the DNA.
Keep in the Thermomixer at the recommended temperature.
Sowmya's recommended amounts (50 µl total solution):
5 µl of 10x buffer
0.5 µl of 100x BSA
1 µl of each enzyme
5 µl of DNA
37.5 (up to 50 µl) of water.
Protocol based on what was done on July the 4th .
Prepared pMP2, pGL2 and pGL3 for ligation. This digestion removed several nucleotides from pMP and the luciferase gene from pGL.
The restriction enzyme digest mixtures were as follows:
pMP2 control for plasmid
DNA 2µL
HindIII 1µL
XbaI 1µL
N4 buffer 10x 5µL
BSA 100x 0.5µL
Water 40.5 µL
pMP2 control for NotI
DNA 2µL
NotI 1µL
N4 buffer 10x 5µL
BSA 100x 0.5µL
Water 41.5 µL
pMP2 control for SpeI
DNA 2µL
SpeI 1µL
N4 buffer 10x 5µL
BSA 100x 0.5µL
Water 41.5 µL
pMP2 for ligation
DNA 19µL
SpeI 1µL
NotI 1µL
N2 buffer 10x 5µL
BSA 100x 0.5µL
Water 23.5 µL
pGL2 for ligation
DNA 12µL
HindIII 1µL
MfeI 1µL
N4 buffer 10x 5µL
BSA 100x 0.5µL
Water 31 µL
pGL3 for ligation
DNA 16µL
HindIII 1µL
FseI 1µL
N4 buffer 10x 5µL
BSA 100x 0.5µL
Water 26.5 µL
All left for 1:30 hours at 37ºC, shaking (600 rpm).
Bands obtained in the control of DNA and enzymes. Everything seems to be correct now!
After that we ran the digestion products in a gel (1% agarose, 50 µl of digestion product + 10 µl of loading dye).
Protocol: Gel Electrophoresis
Agarose concentration depends on the size of the DNA to be run. We will mostly use 1%. VOL is the desired volume of gel in ml:
File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif
1kb ladder bands
CH Lab
Add 0.01*VOL g of agarose to a clean glass bottle.
Pour VOL/50 ml of 50xTAE in a graduated cylinder. Fill up to VOL ml with di water.
Add the resulting VOL ml of 1xTAE to the glass bottle with agarose.
Microwave, at 7, the bottle (loose cap!) until it boils.
Carefully remove bottle (can be super heated!) and check for the total absence of particles. Microwave again if needed.
Prepare a gel box, with comb, and fill it up with the agarose solution (maybe not the whole solution is needed).
Add 0.05 µl per ml of gel in the box of Red Gel (it's in the iGEM drawer) and stirr until disolved.
Wait until cold and solidified.
Carefully remove comb.
Place the box in the electrophoresis chamber.
Fill up the electrophresis chamber with 1x TAE buffer.
Add blue dye to the DNA samples (6x loading buffer, that is 10 µl in 50 µl of DNA solution).
Inject 30 µl of ladder marker in the first well (that's 1 µg of DNA).
Inject 60 µl of each DNA solution in the other wells.
Set voltage to 70-90 V and run for 30-40 min, or until the dye reaches the last 25% of the gel length (DNA travels from - to +).
Place the gel under the camera, cover, turn UV on and take photos!
Preparing the ladder:
get 1kb ladder DNA from the freezer (500 µg/ml).
for 30 charges, 30 µl per charge, we need 900 µl:
60 µl of 1kb ladder DNA
150 µl of dye (6x loading buffer)
690 µl of water
BM Lab
In this lab the gels are slightly different. The total volumes for the small, the medium and the large gel are respectively 60ml, 80ml and 90ml. As we use 0.5x TAE buffer instead of 1x, we can use higher voltages (170V seems to work fine). The gel should run 20-40 minutes, not more. As the gel is thinner, load less DNA (up to ~10ul).
Gel with digested backbones and PCR products for readouts
Top Row:
Ladder
Empty
pMP2 cut with Not+SpeI
Empty
pGL2 cut with HindIII+MfeI
Empty
pGL3 cut with HindIII+FseI
Empty
Control of HindIII with pGL3
Control of FseI with pGL3
Bottom Row:
Ladder
Control with HindIII and XbaI for pMP2
Control of NotI with pMP2
Control of SpeI with pMP2
Empty
Empty
eGFP PCR product
Empty
TNFR PCR product
Empty
SEAP PCR product
The gel fragments corresponding to the linearized backbones in wells 3, 5 and 7 on the top comb were cut out with a razor blade. The PCR products in wells 8, 10 and 12 on the bottom comb were also cut out.
The gel fragments were run through a QIAGEN gel purification kit.
Wiki Links
Recent Changes
Upload file
Special Pages
My Preferences
Privacy Policy
Disclaimers
Important pages
Wiki editing
Planning
Meetings
Primer Design Helper
Add Page
Protocol List
Pages
Home
Team
Official Team Profile
Project
Parts Submitted to the Registry
Modeling
Notebook
Safety
Attributions
Sponsors
Concept Art
Retrieved from " http://2012.igem.org/Team:EPF-Lausanne/Notebook/3_August_2012 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
