<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF-Lausanne_Notebook_9_August_2012"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF-Lausanne/Notebook/9_August_2012">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF-Lausanne/Notebook/9_August_2012&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/9_August_2012&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/9_August_2012&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF-Lausanne/Notebook/9_August_2012" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF-Lausanne/Notebook/9 August 2012</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="all"><DIV class="menu"><UL class="simple-navi"><LI class="menu-introduction"><A href="/Team:EPF-Lausanne"><SPAN>Introduction</SPAN></A></LI><LI class="menu-project"><A href="/Team:EPF-Lausanne/Project"><SPAN>Our Project</SPAN></A></LI><LI class="menu-notebook"><A href="/Team:EPF-Lausanne/Notebook"><SPAN>Notebook</SPAN></A></LI><LI class="menu-human"><A href="/Team:EPF-Lausanne/HumanPractices"><SPAN>Human Practices</SPAN></A></LI><LI class="menu-acknowledgments"><A href="/Team:EPF-Lausanne/Acknowledgments"><SPAN>Acknowledgments</SPAN></A></LI></UL></DIV><DIV class="paper"><DIV class="paper-inner row-fluid"><DIV class="page-content span8"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Plates_observation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Plates observation</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Overnight_culture"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Overnight culture</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_Prepare_Plasmid_Extraction_.28culture_for_Miniprep.29"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Protocol: Prepare Plasmid Extraction (culture for Miniprep)</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#TNFR_minipreps"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">TNFR minipreps</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_Miniprep"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Protocol: Miniprep</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#pHY42_sequencing"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">pHY42 sequencing</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#LovTAP_into_pcDNA3.1"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">LovTAP into pcDNA3.1</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Digestions"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Digestions</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_Restriction_site_digestion"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Protocol: Restriction site digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Nanodrop"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Nanodrop</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_DNA_Concentration_Measurement"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Protocol: DNA Concentration Measurement</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Digestion_of_the_TNFR_ligations"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Digestion of the TNFR ligations</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_Restriction_site_digestion_2"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Protocol: Restriction site digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Gels"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Gels</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_Gel_Electrophoresis"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Protocol: Gel Electrophoresis</SPAN></A></LI><LI class="toclevel-1"><A href="#CH_Lab"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">CH Lab</SPAN></A></LI><LI class="toclevel-1"><A href="#BM_Lab"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">BM Lab</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Gel_extraction_of_LovTAP_and_pGL"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Gel extraction of LovTAP and pGL</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_Gel_Electrophoresis_2"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">Protocol: Gel Electrophoresis</SPAN></A></LI><LI class="toclevel-1"><A href="#CH_Lab_2"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">CH Lab</SPAN></A></LI><LI class="toclevel-1"><A href="#BM_Lab_2"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">BM Lab</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#LovTAP_ligation_into_pMP"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">LovTAP ligation into pMP</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_Ligation"><SPAN class="tocnumber">15.1</SPAN><SPAN class="toctext">Protocol: Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Transformation"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_E.Coli_Transformation"><SPAN class="tocnumber">16.1</SPAN><SPAN class="toctext">Protocol: E.Coli Transformation</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Plates_observation"> Plates observation </SPAN></H2><P>We have colonies for SEAP and eGFP! 
</P><H2><SPAN class="mw-headline" id="Overnight_culture"> Overnight culture </SPAN></H2><DIV class="protocol-remove-stylePreparePlasmidExtraction"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_Prepare_Plasmid_Extraction_.28culture_for_Miniprep.29"><A href="/Team:EPF-Lausanne/Protocol/PreparePlasmidExtraction" title="Team:EPF-Lausanne/Protocol/PreparePlasmidExtraction">Protocol: Prepare Plasmid Extraction (culture for Miniprep)</A></SPAN></H4><DIV class="protocol-inner-box"><UL><LI> Select and number colonies on the plates.
</LI><LI> Prepare tubes of LB medium with the correct quantity of antibiotics (100 µg/ml for Amp, Spc or chloramphenicol).
</LI><LI> Amp can be found in the -20ºC freezer at Ecoli, labeled as &quot;stock&quot;. It is 100 µg/µl, or 1000x.
</LI><LI> The tubes to be used are the 14 ml round bottom found in front of the iGEM drawers (Falcon). Culture with cap in the first step (loose) and close to the second step after culture.
</LI></UL><LI> Touch each colony with a clean pipette tip and put it in a tube.
</LI><LI> Put in incubator.
</LI></DIV></DIV></DIV><P>Colonies were picked up from the SEAP and eGFP ligation plates. Cultures were made (23 tubes...).
</P><H2><SPAN class="mw-headline" id="TNFR_minipreps"> TNFR minipreps </SPAN></H2><DIV class="protocol-remove-styleMiniPrep"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_Miniprep"><A href="/Team:EPF-Lausanne/Protocol/MiniPrep" title="Team:EPF-Lausanne/Protocol/MiniPrep">Protocol: Miniprep</A></SPAN></H4><DIV class="protocol-inner-box"><P>
The slim tubes can be centrifuged in the machine in front of the &quot;Gel hood&quot;, at 4000 rpm for 10 min. The fatter ones, in the E. coli centrifuge by the fridge (the tip can be left inside, since it floats).
</P><P>Pellets resuspended with RNase containing buffer (Resuspension Buffer R3, from Invitrogen, equivalent to Buffer P1 from Qiagen, in Sowmya's box in the fridge).
Note: keep the buffer in ice if you are not bringing it back to the fridge for some minutes.
</P><P>We then use the QIAGEN QIAprep Spin Miniprep Kit with their <A href="http://www.qiagen.com/literature/render.aspx?id=370" class="external text" rel="nofollow">protocol</A> (page 22) and a microcentrifuge.
</P></DIV></DIV></DIV><P>10 minipreps were made from yesterday's cultures.
</P><H2><SPAN class="mw-headline" id="pHY42_sequencing"> pHY42 sequencing </SPAN></H2><P>We received results for the melanopsin sequencing (to be able to design primers for it in the future because we did not know the exact plasmid sequence).
It is a 100% match with mouse melanopsin found on the ENSEMBL database.
</P><H2><SPAN class="mw-headline" id="LovTAP_into_pcDNA3.1"> LovTAP into pcDNA3.1 </SPAN></H2><P>It was decided to try to insert LovTAP into the pcDNA3.1(+) vector (to see if the cloning goes better and to have an alternative vector).
This plasmid was part of the original design, so the restriction sites we needed were already present on the plasmid that came from direct synthesis.
</P><H2><SPAN class="mw-headline" id="Digestions"> Digestions </SPAN></H2><DIV class="protocol-remove-styleRestrictionSiteDigestion"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_Restriction_site_digestion"><A href="/Team:EPF-Lausanne/Protocol/RestrictionSiteDigestion" title="Team:EPF-Lausanne/Protocol/RestrictionSiteDigestion">Protocol: Restriction site digestion</A></SPAN></H4><DIV class="protocol-inner-box"><OL><LI> Look for the best pair of restriction sites, ideally with similar digestion temperatures and times.
</LI><LI><A href="http://tools.neb.com/NEBcutter2/" class="external text" rel="nofollow">NEBcutter</A> for finding cutting enzymes.
</LI><LI><A href="http://www.neb.com/nebecomm/DoubleDigestCalculator.asp" class="external text" rel="nofollow">Double Digest Finder</A> for the parameters.
</LI></OL><LI> Calculate the amounts required of:
</LI><LI> DNA
</LI><LI> Buffer (usually from 10x to 1x)
</LI><LI> BSA, if needed (usually from 100x to 1x)
</LI><LI> Enzymes (depends on the amount of DNA)
</LI><LI> Water
</LI><LI> Get the recommended buffer (and BSA if needed) from the freezer and let defreeze.
</LI><LI> Mix all the ingredients, except DNA, in a tube.
</LI><LI> Note: Enzymes should stay no longer than a couple of minutes out of the freezer. Don't touch the bottom of the tubes! Don't vortex!
</LI><LI> Distribute the mix in as many tubes as DNA samples and add the DNA.
</LI><LI> Keep in the Thermomixer at the recommended temperature.
</LI><P>Sowmya's recommended amounts (50 µl total solution):
</P><UL><LI> 5 µl of 10x buffer
</LI><LI> 0.5 µl of 100x BSA
</LI><LI> 1 µl of each enzyme
</LI><LI> 5 µl of DNA
</LI><LI> 37.5 (up to 50 µl) of water.
</LI></UL><P>Protocol based on what was done on <A href="/Team:EPF-Lausanne/Notebook/4_July_2012" title="Team:EPF-Lausanne/Notebook/4 July 2012"> July the 4th</A>.
</P></DIV></DIV></DIV><DL><DT> Digestion of pMA-LovTAP</DT></DL><P>The plasmid we obtained by direct synthesis was digested with EcoRI-HF and HindIII-HF, following the usual protocol.
</P><DL><DT> Digestion of the pcDNA3.1(+) backbone</DT></DL><P>The backbone was also digested with EcoRI-HF and HindIII-HF. This digestion only removed a few nucleotides.
</P><DL><DT> Digestion of the pGL backbone</DT></DL><P>The pGL4.30 backbone was redigested with FseI and HindIII for future ligations of the melanopsin experiment readouts, because we ran out of it.
</P><H2><SPAN class="mw-headline" id="Nanodrop"> Nanodrop </SPAN></H2><DIV class="protocol-remove-styleDNAConcentration"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_DNA_Concentration_Measurement"><A href="/Team:EPF-Lausanne/Protocol/DNAConcentration" title="Team:EPF-Lausanne/Protocol/DNAConcentration">Protocol: DNA Concentration Measurement</A></SPAN></H4><DIV class="protocol-inner-box"><UL><LI> Take a 6 µl aliquote of the DNA and put back the main DNA tube in the fridge.
</LI><LI> Go to the room by the E.Coli lab (LBTM, not on Friday morning!) with:
</LI><LI> The 6 µl aliquote
</LI><LI> A 10 µl pipet
</LI><LI> Optionally, the buffer you used for DNA elution (there might be some next to the machine).
</LI></UL><LI> The machine is the NanoDrop Spectrophotometer.
</LI><LI> On the computer, click on &quot;Nucleic Acid&quot;.
</LI><LI> Put a 2 µl drop of (nuclease-free) water on the machine's tip as you are asked to and measure.
</LI><LI> Clean tips (both sides) with a quarter of tissue.
</LI><LI> Add 2 µl of the buffer you use and click on &quot;Blank&quot;.
</LI><LI> Clean tips (both sides).
</LI><LI> Add 2 µl of your DNA sample and click &quot;Measure&quot;.
</LI><LI> Clean tips (both sides) with a tissue.
</LI><LI> Take 2 measurements per sample (for averaging).
</LI><LI> Print the report when you are done 
</LI><LI> Click on exit.
</LI><P>The important numbers are:
</P><UL><LI> 260/280 ratio, must be &gt; 1.8
</LI><LI> 260/230 ratio, must be &gt; 2 (too big, &gt; 2.5? , might mean too much salts)
</LI><LI> Of course the DNA concentration.
</LI></UL></DIV></DIV></DIV><P>The Nanodrop gave acceptable results for all of the TNFR minipreps (around 300 ng/µl).
The new LovTAP PCR yielded an average concentration of around 150 ng/µl.
</P><H2><SPAN class="mw-headline" id="Digestion_of_the_TNFR_ligations"> Digestion of the TNFR ligations </SPAN></H2><DIV class="protocol-remove-styleRestrictionSiteDigestion"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_Restriction_site_digestion_2"><A href="/Team:EPF-Lausanne/Protocol/RestrictionSiteDigestion" title="Team:EPF-Lausanne/Protocol/RestrictionSiteDigestion">Protocol: Restriction site digestion</A></SPAN></H4><DIV class="protocol-inner-box"><OL><LI> Look for the best pair of restriction sites, ideally with similar digestion temperatures and times.
</LI><LI><A href="http://tools.neb.com/NEBcutter2/" class="external text" rel="nofollow">NEBcutter</A> for finding cutting enzymes.
</LI><LI><A href="http://www.neb.com/nebecomm/DoubleDigestCalculator.asp" class="external text" rel="nofollow">Double Digest Finder</A> for the parameters.
</LI></OL><LI> Calculate the amounts required of:
</LI><LI> DNA
</LI><LI> Buffer (usually from 10x to 1x)
</LI><LI> BSA, if needed (usually from 100x to 1x)
</LI><LI> Enzymes (depends on the amount of DNA)
</LI><LI> Water
</LI><LI> Get the recommended buffer (and BSA if needed) from the freezer and let defreeze.
</LI><LI> Mix all the ingredients, except DNA, in a tube.
</LI><LI> Note: Enzymes should stay no longer than a couple of minutes out of the freezer. Don't touch the bottom of the tubes! Don't vortex!
</LI><LI> Distribute the mix in as many tubes as DNA samples and add the DNA.
</LI><LI> Keep in the Thermomixer at the recommended temperature.
</LI><P>Sowmya's recommended amounts (50 µl total solution):
</P><UL><LI> 5 µl of 10x buffer
</LI><LI> 0.5 µl of 100x BSA
</LI><LI> 1 µl of each enzyme
</LI><LI> 5 µl of DNA
</LI><LI> 37.5 (up to 50 µl) of water.
</LI></UL><P>Protocol based on what was done on <A href="/Team:EPF-Lausanne/Notebook/4_July_2012" title="Team:EPF-Lausanne/Notebook/4 July 2012"> July the 4th</A>.
</P></DIV></DIV></DIV><P>To check if the TNFR fragment actually correctly ligated into pGL4.30, a digestion with HindIII-HF and BamHI was run.
The digestion was incubated for 2 hours, then ran on a gel.
</P><H2><SPAN class="mw-headline" id="Gels"> Gels </SPAN></H2><DIV class="protocol-remove-styleGel"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_Gel_Electrophoresis"><A href="/Team:EPF-Lausanne/Protocol/Gel" title="Team:EPF-Lausanne/Protocol/Gel">Protocol: Gel Electrophoresis</A></SPAN></H4><DIV class="protocol-inner-box"><P>
Agarose concentration depends on the size of the DNA to be run. We will mostly use 1%.
VOL is the desired volume of gel in ml:
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif" title="File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif">File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif</A><DIV class="thumbcaption">1kb ladder bands</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="CH_Lab"> CH Lab </SPAN></H2><OL><LI> Add 0.01*VOL g of agarose to a clean glass bottle.
</LI><LI> Pour VOL/50 ml of 50xTAE in a graduated cylinder. Fill up to VOL ml with di water.
</LI><LI> Add the resulting VOL ml of 1xTAE to the glass bottle with agarose.
</LI><LI> Microwave, at 7, the bottle (loose cap!) until it boils.
</LI><LI> Carefully remove bottle (can be super heated!) and check for the total absence of particles. Microwave again if needed.
</LI><LI> Prepare a gel box, with comb, and fill it up with the agarose solution (maybe not the whole solution is needed).
</LI><LI> Add 0.05 µl per ml of gel in the box of Red Gel (it's in the iGEM drawer) and stirr until disolved.
</LI><LI> Wait until cold and solidified.
</LI><LI> Carefully remove comb.
</LI><LI> Place the box in the electrophoresis chamber.
</LI><LI> Fill up the electrophresis chamber with 1x TAE buffer.
</LI><LI> Add blue dye to the DNA samples (6x loading buffer, that is 10 µl in 50 µl of DNA solution).
</LI><LI> Inject 30 µl of ladder marker in the first well (that's 1 µg of DNA).
</LI><LI> Inject 60 µl of each DNA solution in the other wells.
</LI><LI> Set voltage to 70-90 V and run for 30-40 min, or until the dye reaches the last 25% of the gel length (DNA travels from - to +).
</LI><LI> Place the gel under the camera, cover, turn UV on and take photos!
</LI></OL><P>
Preparing the ladder:
</P><UL><LI> get 1kb ladder DNA from the freezer (500 µg/ml).
</LI><LI> for 30 charges, 30 µl per charge, we need 900 µl:
</LI><LI> 60 µl of 1kb ladder DNA
</LI><LI> 150 µl of dye (6x loading buffer)
</LI><LI> 690 µl of water
</LI></UL><H2><SPAN class="mw-headline" id="BM_Lab"> BM Lab </SPAN></H2><P>In this lab the gels are slightly different. The total volumes for the small, the medium and the large gel are respectively 60ml, 80ml and 90ml. As we use 0.5x TAE buffer instead of 1x, we can use higher voltages (170V seems to work fine). The gel should run 20-40 minutes, not more. As the gel is thinner, load less DNA (up to ~10ul).
</P></DIV></DIV></DIV><P>The digested pGL backbone and the digested LovTAP-pMA were run on a gel to excise the wanted fragments and to purify them.
</P><DL><DT> Gel picture</DT></DL><P>The gel was acceptable and could be used for a gel extraction.
</P><P>The digested TNFR ligations were ran on a gel too.
</P><DL><DT> Gel picture</DT></DL><P>The bands are of an extremely weird shape and intensity, the ladder can't even be seen. Maybe something went wrong during the gel casting. Some of them might be of the right size, but it's hard to tell. The best-looking ones are going to be used for the following experiments anyway.
</P><H2><SPAN class="mw-headline" id="Gel_extraction_of_LovTAP_and_pGL"> Gel extraction of LovTAP and pGL </SPAN></H2><DIV class="protocol-remove-styleGel"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_Gel_Electrophoresis_2"><A href="/Team:EPF-Lausanne/Protocol/Gel" title="Team:EPF-Lausanne/Protocol/Gel">Protocol: Gel Electrophoresis</A></SPAN></H4><DIV class="protocol-inner-box"><P>
Agarose concentration depends on the size of the DNA to be run. We will mostly use 1%.
VOL is the desired volume of gel in ml:
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif" title="File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif">File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif</A><DIV class="thumbcaption">1kb ladder bands</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="CH_Lab_2"> CH Lab </SPAN></H2><OL><LI> Add 0.01*VOL g of agarose to a clean glass bottle.
</LI><LI> Pour VOL/50 ml of 50xTAE in a graduated cylinder. Fill up to VOL ml with di water.
</LI><LI> Add the resulting VOL ml of 1xTAE to the glass bottle with agarose.
</LI><LI> Microwave, at 7, the bottle (loose cap!) until it boils.
</LI><LI> Carefully remove bottle (can be super heated!) and check for the total absence of particles. Microwave again if needed.
</LI><LI> Prepare a gel box, with comb, and fill it up with the agarose solution (maybe not the whole solution is needed).
</LI><LI> Add 0.05 µl per ml of gel in the box of Red Gel (it's in the iGEM drawer) and stirr until disolved.
</LI><LI> Wait until cold and solidified.
</LI><LI> Carefully remove comb.
</LI><LI> Place the box in the electrophoresis chamber.
</LI><LI> Fill up the electrophresis chamber with 1x TAE buffer.
</LI><LI> Add blue dye to the DNA samples (6x loading buffer, that is 10 µl in 50 µl of DNA solution).
</LI><LI> Inject 30 µl of ladder marker in the first well (that's 1 µg of DNA).
</LI><LI> Inject 60 µl of each DNA solution in the other wells.
</LI><LI> Set voltage to 70-90 V and run for 30-40 min, or until the dye reaches the last 25% of the gel length (DNA travels from - to +).
</LI><LI> Place the gel under the camera, cover, turn UV on and take photos!
</LI></OL><P>
Preparing the ladder:
</P><UL><LI> get 1kb ladder DNA from the freezer (500 µg/ml).
</LI><LI> for 30 charges, 30 µl per charge, we need 900 µl:
</LI><LI> 60 µl of 1kb ladder DNA
</LI><LI> 150 µl of dye (6x loading buffer)
</LI><LI> 690 µl of water
</LI></UL><H2><SPAN class="mw-headline" id="BM_Lab_2"> BM Lab </SPAN></H2><P>In this lab the gels are slightly different. The total volumes for the small, the medium and the large gel are respectively 60ml, 80ml and 90ml. As we use 0.5x TAE buffer instead of 1x, we can use higher voltages (170V seems to work fine). The gel should run 20-40 minutes, not more. As the gel is thinner, load less DNA (up to ~10ul).
</P></DIV></DIV></DIV><P>The DNA was extracted and purified.
</P><H2><SPAN class="mw-headline" id="LovTAP_ligation_into_pMP"> LovTAP ligation into pMP </SPAN></H2><DIV class="protocol-remove-styleLigation"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_Ligation"><A href="/Team:EPF-Lausanne/Protocol/Ligation" title="Team:EPF-Lausanne/Protocol/Ligation">Protocol: Ligation</A></SPAN></H4><DIV class="protocol-inner-box"><P>
Ligation is a method of combining several DNA fragments into a single plasmid. This is often the
step following a <A href="/Team:EPF-Lausanne/Protocol/PCR" title="Team:EPF-Lausanne/Protocol/PCR">PCR</A> (and a <A href="/Team:EPF-Lausanne/Protocol/PCRCleanup" title="Team:EPF-Lausanne/Protocol/PCRCleanup">PCR cleanup</A>) or a <A href="/Team:EPF-Lausanne/Protocol/GelExtraction" title="Team:EPF-Lausanne/Protocol/GelExtraction">gel extraction</A>. You can also do a &quot;dirty&quot; ligation, where you follow a certain number of <A href="/Team:EPF-Lausanne/Protocol/RestrictionSiteDigestion" title="Team:EPF-Lausanne/Protocol/RestrictionSiteDigestion">digestions</A> directly by a ligation.
</P><OL><LI> Download the following spreadsheet : <A href="/File:Team-EPF-Lausanne_Ligation.xls" title="File:Team-EPF-Lausanne Ligation.xls">File:Team-EPF-Lausanne Ligation.xls</A></LI><LI> Fill in the pink areas with the vector and fragment concentration, their size and the ratio.
</LI><LI> Add all the suggested ingredients order in a microcentrifuge tube, in the order they appear.
</LI><LI> Ligate for 2 hours at 14ºC.
</LI><LI> Immediately transform competent bacteria with the ligation product.
</LI></OL><P>Note: This protocol hasn't been optimized for blunt-end ligation (though it might still work).
</P></DIV></DIV></DIV><P>The digested backbone and the digested PCR product from the day before were ligated together. We tried every ratio twice, and made a negative control. This gave 7 ligation tubes.
</P><H2><SPAN class="mw-headline" id="Transformation"> Transformation </SPAN></H2><DIV class="protocol-remove-styleTransformation"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_E.Coli_Transformation"><A href="/Team:EPF-Lausanne/Protocol/Transformation" title="Team:EPF-Lausanne/Protocol/Transformation">Protocol: E.Coli Transformation</A></SPAN></H4><DIV class="protocol-inner-box"><OL><LI> Thaw the competent E.coli (DH5alpha) cells on ice (not in hands!)
</LI><LI> As soon as it is thawed, add 50µl of the cells to the DNA (~50-100 ng of pure plasmid, or some 2 µl usually)
</LI><LI> Let it rest on ice for 20-30 min. Meanwhile, put agar plate (with correct antibiotic) at 37°C for prewarming.
</LI><LI> Put the tube with DNA+E.coli at 42°C for 45 sec - 1 min (heat shock)
</LI><LI> Add 400 µl of LB broth and place at 37°C for 20-30 min (shaking)
</LI><LI> Spread the cells on the prewarmed plate (and let it dry)
</LI><LI> Incubate the plate upside-down at 37°C for ~14-15 hours (leaving it more than 16h decreases the plasmid quality)
</LI></OL></DIV></DIV></DIV><P>Bacteria were transformed with the ligation products, and plated on 7 plates.
</P></DIV></DIV></DIV><DIV class="footer"><DIV class="footer-inner"><DIV class="footer-inner-inner"><DIV class="row-fluid"><DIV class="span4"><H4>Wiki Links</H4><UL><LI><A href="https://2012.igem.org/Special:RecentChanges">Recent Changes</A></LI><LI><A href="https://2012.igem.org/Special:Upload?wpDestFile=Team-EPF-Lausanne_">Upload file</A></LI><LI><A href="https://2012.igem.org/Special:SpecialPages">Special Pages</A></LI><LI><A href="https://2012.igem.org/Special:Preferences">My Preferences</A></LI><LI><A href="https://2012.igem.org/2012.igem.org:Privacy_policy">Privacy Policy</A></LI><LI><A href="https://2012.igem.org/2012.igem.org:General_disclaimer">Disclaimers</A></LI></UL></DIV><DIV class="span4"><H4>Important pages</H4><UL><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/Wiki">Wiki editing</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/Planning">Planning</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/Meetings">Meetings</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/PrimerDesignHelper">Primer Design Helper</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/AddPage">Add Page</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/Protocol">Protocol List</A></LI></UL></DIV><DIV class="span4"><H4>Pages</H4><UL><LI><A href="/Team:EPF-Lausanne">Home</A></LI><LI><A href="/Team:EPF-Lausanne/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=EPF-Lausanne">Official Team Profile</A></LI><LI><A href="/Team:EPF-Lausanne/Project">Project</A></LI><LI><A href="/Team:EPF-Lausanne/Parts">Parts Submitted to the Registry</A></LI><LI><A href="/Team:EPF-Lausanne/Modeling">Modeling</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook">Notebook</A></LI><LI><A href="/Team:EPF-Lausanne/Safety">Safety</A></LI><LI><A href="/Team:EPF-Lausanne/Attributions">Attributions</A></LI><LI><A href="/Team:EPF-Lausanne/Sponsors">Sponsors</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/ConceptArt">Concept Art</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:EPF-Lausanne/Notebook/9_August_2012">http://2012.igem.org/Team:EPF-Lausanne/Notebook/9_August_2012</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF-Lausanne/Notebook/9_August_2012" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF-Lausanne/Notebook/9_August_2012" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/9_August_2012&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/9_August_2012&amp;oldid=164058" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>