<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF-Lausanne_Notebook_3_August_2012"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF-Lausanne/Notebook/3_August_2012">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF-Lausanne/Notebook/3_August_2012&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/3_August_2012&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/3_August_2012&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF-Lausanne/Notebook/3_August_2012" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF-Lausanne/Notebook/3 August 2012</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="all"><DIV class="menu"><UL class="simple-navi"><LI class="menu-introduction"><A href="/Team:EPF-Lausanne"><SPAN>Introduction</SPAN></A></LI><LI class="menu-project"><A href="/Team:EPF-Lausanne/Project"><SPAN>Our Project</SPAN></A></LI><LI class="menu-notebook"><A href="/Team:EPF-Lausanne/Notebook"><SPAN>Notebook</SPAN></A></LI><LI class="menu-human"><A href="/Team:EPF-Lausanne/HumanPractices"><SPAN>Human Practices</SPAN></A></LI><LI class="menu-acknowledgments"><A href="/Team:EPF-Lausanne/Acknowledgments"><SPAN>Acknowledgments</SPAN></A></LI></UL></DIV><DIV class="paper"><DIV class="paper-inner row-fluid"><DIV class="page-content span8"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#PCR"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_PCR"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Protocol: PCR</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Tips"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Tips</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Digestion_of_pMP_and_pGL4.30_for_ligation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Digestion of pMP and pGL4.30 for ligation</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_Restriction_site_digestion"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Protocol: Restriction site digestion</SPAN></A></LI><LI class="toclevel-2"><A href="#Protocol:_Gel_Electrophoresis"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Protocol: Gel Electrophoresis</SPAN></A></LI><LI class="toclevel-1"><A href="#CH_Lab"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">CH Lab</SPAN></A></LI><LI class="toclevel-1"><A href="#BM_Lab"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">BM Lab</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="PCR"> PCR </SPAN></H2><DIV class="protocol-remove-stylePCR"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_PCR"><A href="/Team:EPF-Lausanne/Protocol/PCR" title="Team:EPF-Lausanne/Protocol/PCR">Protocol: PCR</A></SPAN></H4><DIV class="protocol-inner-box"><P>
PCR is a reaction that makes it possible (and relatively easy) to amplify
a certain region of DNA. The first step is the selection of that region
(and the design of the relevant primers). Primer design can be done by hand, or by
using our <A href="/Team:EPF-Lausanne/PrimerDesignHelper" title="Team:EPF-Lausanne/PrimerDesignHelper">Primer Design Helper</A>. Once
done, order the primers (in our case, we ordered from them <A href="http://www.idtdna.com/" class="external text" rel="nofollow">IDT</A>).
</P><P>When you've received the primers, prepare them and make sure you've got your
PCR kit (we used the &quot;Phusion® High-Fidelity DNA Polymerase&quot;). Start preparing your
master mix, the composition for one tube is:
</P><P>1X Mastermix 20μl reaction, add in this order
</P><TABLE class="wikitable"><TBODY><TR><TH> Reagent
</TH><TH> Volume [μl]
</TH></TR><TR><TD>Water
</TD><TD> Complete to total volume of 20μl
</TD></TR><TR><TD>HF-Buffer (5x)
</TD><TD>4
</TD></TR><TR><TD>DMSO (optional)
</TD><TD>0.6
</TD></TR><TR><TD>dNTPs
</TD><TD>0.4
</TD></TR><TR><TD>Forward primer (50μM)
</TD><TD>0.2
</TD></TR><TR><TD>Reverse primer (50μM)
</TD><TD>0.2
</TD></TR><TR><TD>Template (10ng/μl)
</TD><TD>0.5
</TD></TR><TR><TD>Phusion HF polymerase
</TD><TD>0.2
</TD></TR></TBODY></TABLE><P>Prepare one or two extra tubes-worth of reagent (you'll use some liquid on the walls of your tips).
</P><P>Once you've finished, you should run the resulting products on a <A href="/Team:EPF-Lausanne/Protocol/Gel" title="Team:EPF-Lausanne/Protocol/Gel">gel</A>
to check if everything went as planned.
</P><H2><SPAN class="mw-headline" id="Tips"> Tips </SPAN></H2><UL><LI> Thaw the HF-Buffer, DMSO and dNTPs before making the mastermix.
</LI><LI> Avoid taking the Phusion-HF polymerase out of the freezer (only take it out briefly when you need to add it).
</LI><LI> If the reactions have different primers and/or template, add the polymerase right after the dNTPs, split the mastermix and add the rest.
</LI><LI> Don't forget positive and negative controls
</LI><LI> Primers should have similar Tms (less than 5°C).
</LI><LI> Primer Tm calculation is a less exact science than it should be (just test several tools and compare their results). If you're not sure what the correct Tm is, consider using a gradient PCR.
</LI><LI> Avoid primers with strong secondary structures.
</LI><LI> PCR can introduce mutations. Don't forget to sequence your final product (this could be your final plasmid): you really don't want to lose a few weeks because of a &quot;corrupt&quot; plasmid.
</LI></UL></DIV></DIV></DIV><P>Ran a PCR on the readouts aimed to be co-expressed with melanopsin (SEAP, TNFR, eGFP), using primers that would add the restriction sites suitable for ligation into the plasmids of interest.
</P><DL><DT> The master mix for 4 tubes (3 + an excess one) was done as follows</DT></DL><UL><LI> H20: 146 µl
</LI><LI> Buffer HF (5x): 40 µl
</LI><LI> dNTP (10 mM): 4 µl
</LI></UL><P>Then split into 3 tubes, add the following reagents to every one of them:
</P><UL><LI> Primers:
</LI><LI> Forward (50 µM): 0.5 µl
</LI><LI> Reverse (50 µM): 0.5 µl
</LI></UL><LI> Template DNA: 1 µl
</LI><LI> Phusion DNA Polymerase: 0.5 µl
</LI><P>The total volume in each tube should be 50 µl.
</P><P>The average Tm for our primers was around 69°C.
</P><DL><DT> The thermal cycling protocol follows</DT></DL><UL><LI> Initial denaturation: T = 98°C, 30 sec
</LI><LI> Cycle through this 30 times:
</LI><LI> Denaturation: T = 98°C, 10 sec
</LI><LI> Annealing: T = 69°C, 30 sec
</LI><LI> Extension: T = 72°C, 1 min
</LI></UL><LI> Final extension: T = 72°C, 7 min
</LI><LI> Hold: T = 4°C, stay until pickup
</LI><P>Loosely based on the Phusion kit protocol.
</P><DL><DT> The expected product sizes are</DT></DL><UL><LI> SEAP+polyA: 1890 bp (+ tails)
</LI><LI> TNFR: 1384 bp (+ tails)
</LI><LI> eGFP: 725 bp (+ tails)
</LI></UL><H2><SPAN class="mw-headline" id="Digestion_of_pMP_and_pGL4.30_for_ligation"> Digestion of pMP and pGL4.30 for ligation </SPAN></H2><DIV class="protocol-remove-styleRestrictionSiteDigestion"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_Restriction_site_digestion"><A href="/Team:EPF-Lausanne/Protocol/RestrictionSiteDigestion" title="Team:EPF-Lausanne/Protocol/RestrictionSiteDigestion">Protocol: Restriction site digestion</A></SPAN></H4><DIV class="protocol-inner-box"><OL><LI> Look for the best pair of restriction sites, ideally with similar digestion temperatures and times.
</LI><LI><A href="http://tools.neb.com/NEBcutter2/" class="external text" rel="nofollow">NEBcutter</A> for finding cutting enzymes.
</LI><LI><A href="http://www.neb.com/nebecomm/DoubleDigestCalculator.asp" class="external text" rel="nofollow">Double Digest Finder</A> for the parameters.
</LI></OL><LI> Calculate the amounts required of:
</LI><LI> DNA
</LI><LI> Buffer (usually from 10x to 1x)
</LI><LI> BSA, if needed (usually from 100x to 1x)
</LI><LI> Enzymes (depends on the amount of DNA)
</LI><LI> Water
</LI><LI> Get the recommended buffer (and BSA if needed) from the freezer and let defreeze.
</LI><LI> Mix all the ingredients, except DNA, in a tube.
</LI><LI> Note: Enzymes should stay no longer than a couple of minutes out of the freezer. Don't touch the bottom of the tubes! Don't vortex!
</LI><LI> Distribute the mix in as many tubes as DNA samples and add the DNA.
</LI><LI> Keep in the Thermomixer at the recommended temperature.
</LI><P>Sowmya's recommended amounts (50 µl total solution):
</P><UL><LI> 5 µl of 10x buffer
</LI><LI> 0.5 µl of 100x BSA
</LI><LI> 1 µl of each enzyme
</LI><LI> 5 µl of DNA
</LI><LI> 37.5 (up to 50 µl) of water.
</LI></UL><P>Protocol based on what was done on <A href="/Team:EPF-Lausanne/Notebook/4_July_2012" title="Team:EPF-Lausanne/Notebook/4 July 2012"> July the 4th</A>.
</P></DIV></DIV></DIV><P>
Prepared pMP2, pGL2 and pGL3 for ligation. This digestion removed several nucleotides from pMP and the luciferase gene from pGL.
</P><P>The restriction enzyme digest mixtures were as follows:
</P><DL><DT> pMP2 control for plasmid
</DT></DL><UL><LI>DNA 2µL
</LI><LI>HindIII 1µL
</LI><LI>XbaI 1µL
</LI><LI>N4 buffer 10x 5µL
</LI><LI>BSA 100x 0.5µL
</LI><LI>Water 40.5 µL
</LI></UL><DL><DT> pMP2 control for NotI
</DT></DL><UL><LI>DNA 2µL
</LI><LI>NotI 1µL
</LI><LI>N4 buffer 10x 5µL
</LI><LI>BSA 100x 0.5µL
</LI><LI>Water 41.5 µL
</LI></UL><DL><DT> pMP2 control for SpeI
</DT></DL><UL><LI>DNA 2µL
</LI><LI>SpeI 1µL
</LI><LI>N4 buffer 10x 5µL
</LI><LI>BSA 100x 0.5µL
</LI><LI>Water 41.5 µL
</LI></UL><DL><DT> pMP2 for ligation
</DT></DL><UL><LI>DNA 19µL
</LI><LI>SpeI 1µL
</LI><LI>NotI 1µL
</LI><LI>N2 buffer 10x 5µL
</LI><LI>BSA 100x 0.5µL
</LI><LI>Water 23.5 µL
</LI></UL><DL><DT> pGL2 for ligation
</DT></DL><UL><LI>DNA 12µL
</LI><LI>HindIII 1µL
</LI><LI>MfeI 1µL
</LI><LI>N4 buffer 10x 5µL
</LI><LI>BSA 100x 0.5µL
</LI><LI>Water 31 µL
</LI></UL><DL><DT> pGL3 for ligation
</DT></DL><UL><LI>DNA 16µL
</LI><LI>HindIII 1µL
</LI><LI>FseI 1µL
</LI><LI>N4 buffer 10x 5µL
</LI><LI>BSA 100x 0.5µL
</LI><LI>Water 26.5 µL
</LI></UL><P>All left for 1:30 hours at 37ºC, shaking (600 rpm).
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Bands obtained in the control of DNA and enzymes. Everything seems to be correct now!</DIV></DIV></DIV><P>After that we ran the digestion products in a gel (1% agarose, 50 µl of digestion product + 10 µl of loading dye).
</P><DIV class="protocol-remove-styleGel"><DIV class="protocol-outer-box"><H4 class="protocol-title"><SPAN class="mw-headline" id="Protocol:_Gel_Electrophoresis"><A href="/Team:EPF-Lausanne/Protocol/Gel" title="Team:EPF-Lausanne/Protocol/Gel">Protocol: Gel Electrophoresis</A></SPAN></H4><DIV class="protocol-inner-box"><P>
Agarose concentration depends on the size of the DNA to be run. We will mostly use 1%.
VOL is the desired volume of gel in ml:
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif" title="File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif">File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif</A><DIV class="thumbcaption">1kb ladder bands</DIV></DIV></DIV><H2><SPAN class="mw-headline" id="CH_Lab"> CH Lab </SPAN></H2><OL><LI> Add 0.01*VOL g of agarose to a clean glass bottle.
</LI><LI> Pour VOL/50 ml of 50xTAE in a graduated cylinder. Fill up to VOL ml with di water.
</LI><LI> Add the resulting VOL ml of 1xTAE to the glass bottle with agarose.
</LI><LI> Microwave, at 7, the bottle (loose cap!) until it boils.
</LI><LI> Carefully remove bottle (can be super heated!) and check for the total absence of particles. Microwave again if needed.
</LI><LI> Prepare a gel box, with comb, and fill it up with the agarose solution (maybe not the whole solution is needed).
</LI><LI> Add 0.05 µl per ml of gel in the box of Red Gel (it's in the iGEM drawer) and stirr until disolved.
</LI><LI> Wait until cold and solidified.
</LI><LI> Carefully remove comb.
</LI><LI> Place the box in the electrophoresis chamber.
</LI><LI> Fill up the electrophresis chamber with 1x TAE buffer.
</LI><LI> Add blue dye to the DNA samples (6x loading buffer, that is 10 µl in 50 µl of DNA solution).
</LI><LI> Inject 30 µl of ladder marker in the first well (that's 1 µg of DNA).
</LI><LI> Inject 60 µl of each DNA solution in the other wells.
</LI><LI> Set voltage to 70-90 V and run for 30-40 min, or until the dye reaches the last 25% of the gel length (DNA travels from - to +).
</LI><LI> Place the gel under the camera, cover, turn UV on and take photos!
</LI></OL><P>
Preparing the ladder:
</P><UL><LI> get 1kb ladder DNA from the freezer (500 µg/ml).
</LI><LI> for 30 charges, 30 µl per charge, we need 900 µl:
</LI><LI> 60 µl of 1kb ladder DNA
</LI><LI> 150 µl of dye (6x loading buffer)
</LI><LI> 690 µl of water
</LI></UL><H2><SPAN class="mw-headline" id="BM_Lab"> BM Lab </SPAN></H2><P>In this lab the gels are slightly different. The total volumes for the small, the medium and the large gel are respectively 60ml, 80ml and 90ml. As we use 0.5x TAE buffer instead of 1x, we can use higher voltages (170V seems to work fine). The gel should run 20-40 minutes, not more. As the gel is thinner, load less DNA (up to ~10ul).
</P></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel with digested backbones and PCR products for readouts</DIV></DIV></DIV><UL><LI> Top Row:
</LI><LI> Ladder
</LI><LI> Empty
</LI><LI> pMP2 cut with Not+SpeI
</LI><LI> Empty
</LI><LI> pGL2 cut with HindIII+MfeI
</LI><LI> Empty
</LI><LI> pGL3 cut with HindIII+FseI
</LI><LI> Empty
</LI><LI> Control of HindIII with pGL3
</LI><LI> Control of FseI with pGL3
</LI></UL><LI> Bottom Row:
</LI><LI> Ladder
</LI><LI> Control with HindIII and XbaI for pMP2
</LI><LI> Control of NotI with pMP2
</LI><LI> Control of SpeI with pMP2
</LI><LI> Empty
</LI><LI> Empty
</LI><LI> eGFP PCR product
</LI><LI> Empty
</LI><LI> TNFR PCR product
</LI><LI> Empty
</LI><LI> SEAP PCR product
</LI><P>The gel fragments corresponding to the linearized backbones in wells 3, 5 and 7 on the top comb were cut out with a razor blade. The PCR products in wells 8, 10 and 12 on the bottom comb were also cut out. 
</P><P>The gel fragments were run through a QIAGEN gel purification kit. 
</P></DIV></DIV></DIV><DIV class="footer"><DIV class="footer-inner"><DIV class="footer-inner-inner"><DIV class="row-fluid"><DIV class="span4"><H4>Wiki Links</H4><UL><LI><A href="https://2012.igem.org/Special:RecentChanges">Recent Changes</A></LI><LI><A href="https://2012.igem.org/Special:Upload?wpDestFile=Team-EPF-Lausanne_">Upload file</A></LI><LI><A href="https://2012.igem.org/Special:SpecialPages">Special Pages</A></LI><LI><A href="https://2012.igem.org/Special:Preferences">My Preferences</A></LI><LI><A href="https://2012.igem.org/2012.igem.org:Privacy_policy">Privacy Policy</A></LI><LI><A href="https://2012.igem.org/2012.igem.org:General_disclaimer">Disclaimers</A></LI></UL></DIV><DIV class="span4"><H4>Important pages</H4><UL><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/Wiki">Wiki editing</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/Planning">Planning</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/Meetings">Meetings</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/PrimerDesignHelper">Primer Design Helper</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/AddPage">Add Page</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/Protocol">Protocol List</A></LI></UL></DIV><DIV class="span4"><H4>Pages</H4><UL><LI><A href="/Team:EPF-Lausanne">Home</A></LI><LI><A href="/Team:EPF-Lausanne/Team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=EPF-Lausanne">Official Team Profile</A></LI><LI><A href="/Team:EPF-Lausanne/Project">Project</A></LI><LI><A href="/Team:EPF-Lausanne/Parts">Parts Submitted to the Registry</A></LI><LI><A href="/Team:EPF-Lausanne/Modeling">Modeling</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook">Notebook</A></LI><LI><A href="/Team:EPF-Lausanne/Safety">Safety</A></LI><LI><A href="/Team:EPF-Lausanne/Attributions">Attributions</A></LI><LI><A href="/Team:EPF-Lausanne/Sponsors">Sponsors</A></LI><LI><A href="https://2012.igem.org/Team:EPF-Lausanne/ConceptArt">Concept Art</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:EPF-Lausanne/Notebook/3_August_2012">http://2012.igem.org/Team:EPF-Lausanne/Notebook/3_August_2012</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF-Lausanne/Notebook/3_August_2012" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF-Lausanne/Notebook/3_August_2012" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/3_August_2012&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/3_August_2012&amp;oldid=152090" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>