"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF-Lausanne/Notebook/24 September 2012
From 2012.igem.org
Introduction
Our Project
Notebook
Human Practices
Acknowledgments
Contents
1 FACS
1.1 Protocol: FACS
2 Calcium imaging with Oregon Green
2.1 Protocol: Calcium Imaging with Oregon Green
3 mRNA extraction
3.1 Protocol: mRNA extraction
3.2 Prepare the cell culture
3.3 Prepare the lysis buffer
3.4 Prepare your work place
3.5 mRNA extraction
4 Nanodrop of mRNA
4.1 Protocol: DNA Concentration Measurement
5 LovTAP RT-PCR
5.1 Protocol: RT-PCR
5.2 Primer design
5.3 DNAse Treatment
5.4 Master Mix
5.5 Protocol
FACS
Protocol: FACS
We've used flow cytometry to check the measure the expression of LovTAP. As we had no machines in our lab, we used the FCCF's (Flow Cytometry Core Facility) machines to perform the measurements. The cells normally need to be fixed before being brought to the lab but, as our project has a biosafety level 1, we could bring in the cells alive.
The sample preparation required only a single washing step with PBS:
Centrifuge the cells at 1500rpm for 3 minutes in a centrifuge (or at 3000rpm for 3 minutes in a microcentrifuge).
Aspirate away the medium (be very careful not to disturb the pellet).
Resuspend in PBS (we used ~400µl for 400'000 cells, but the required density is different for every machine).
The cells transfected with LovTAP and dsRed at diverse ratios, as well as with dsRed only, and a seed control, were illuminated for three hours approximately with diverse light intensities (100% of the maximal LED power, 50% and 25%) while shaking in a well plate. Then they have been sampled every three hours and taken to FACS analysis at the corresponding EPFL facility. It was quite hard to keep the correct time pacing, however, because the FACS analysis on a 24-well plate takes a lot of time and the FACS machine was not always free.
Calcium imaging with Oregon Green
Protocol: Calcium Imaging with Oregon Green
Note
This protocol is specific to OGB-1/AM (Oregon Green) and may vary if you use different dyes.
Prepare your samples by measuring their PCV (or estimating the cell amount according to the doubling rate). Dilute them with PBS in order to have between 200 and 500 cells/µl. Prepare at least one well that has seed cells.
For these experiments, we usually use 200'000 cells.
1. Add 5 µl 80% DMSO, 20% pluronic acid to 50 g tube of OGB-1/AM to dissolve dye powder. Mix with 45 µl of PBS for a 1 mM solution. (This is for 10 samples.)
2. Add 5 µl 1 mM OGB-1/AM to 495 µl PBS (for one sample).
3. Add 500 µl directly to the cell culture medium and incubate for 45 minutes at 37 degrees.
4. Centrifuge cells and aspirate culture media and OGB-1/AM. Resuspend in 750 µl of PBS.
5. Centrifuge cells and aspirate PBS. Resuspend in PBS. Repeat this step twice.
6. Aspirate the PBS and add fresh culture medium to the cells.
7. Your cells are ready to go under the scope! Put them on a slide or in an optical well plate quickly to minimize the number of dead cells you observe. If the pathway you use is photosensitive, remember to keep the cells in the dark.
Seed cultures for HEK and CHO along with Cultures transfected with just melanopsin were excited with blue light and observed under a fluorescence microscope.
Some cells with melanospin apeared to have a higher calcium concentration after excitation under the microscope. The images are interpreted in the results section.
mRNA extraction
Protocol: mRNA extraction
mRNA extraction is a procedure that makes it possible to recover all the messenger RNA of a cell. Before proceeeding further, make sure you have your mRNA extraction kit (we've used the RNeasy Plus Mini Kit from Qiagen and the QIAshredder Kit from Qiagen), all the necessary reagents (β-mercaptoethanol, 70% ethanol and 100% ethanol) and RNAse inhibitor to avoir RNA degradation (RNaseZAP form Sigma-Aldrich). Once you've got all the necessary equipment, proceed according to the hnadbook of the kit.
With the RNeasy Plus Mini Kit from Qiagen the procedure takes about an hour.
Prepare the cell culture
1. Harvest 0.5*10^6 cells from the culture.
2. Centrifugate 5 min at 300g
3. Discard the aliquot, resuspend the pellet in the PBS solution
4. Centrifugate 5 minutes at 300g
5. repeat step 3.
6. repeat step 4.
7. Discard the aliquot, keep the pellet.
Please pay attention to remove all the liquid, otherwise it might inhibit cell lysis.
Prepare the lysis buffer
Step to be performed if β-mercaptoethanol haven't been added yet to the lysis buffer.
1.Pipet 594μL of lysis buffer (Buffer RLT Plus) into a 1.5 mL eppendorf
2.Add 6μL of β-mercaptoethanol
Step 2 must be done in a fume hood, since β-mercaptoethanolis toxic and nocif upon inhalation.
Prepare your work place
RNA is easily degraded by the RNAses present on the surfaces, surface of skin or just dust particles in the air. To avoid contamination with the RNAses, please clean your workplace and all the instruments with 70% ethanol, then spray them with RNAse inhibitor. Change gloves whenever you touch any common surface and spray them with RNAse inhibitor regularly. Close all the tubes, eppendorfs and tips boxes as soon as you've done with pipeting to avoid contamination from RNAses on the dust particules.
Lysis buffer and β-mercaptoethanol are both toxic to the environment and the RTL buffer cannot be bleached. Please, anticipate a storage for both of them and dispose them in according with the instructions in the handbook.
mRNA extraction
(From the Qiagen RNeasy Plus Mini Kit mini-handbook )
1. Loosen the cell pellet by flicking the tube
2. Add 600μL of the prepared lysis buffer
3. Vortex or mix vigurously
4. Pipet the lysate directly into a QIAshredder spin column placed in a 2 ml collection tube, and centrifuge for 2 min at maximum speed
5. Transfer the homogenized lysate to a gDNA Eliminator spin column placed in a 2 ml collection tube. Centrifuge for 30 s at ≥8000 x g. Discard the column, and save the flowthrough
6. Add 600μL of 70% ethanol to the flowthrough, and mix well by pipetting. Do not centrifuge.
7. Transfer 700 μL of the sample to an RNeasy spin column placed in a 2 ml collection tube. Centrifuge for 15 s at >8000 x g. Discard the flow-through. Reuse the collection tube in next step.
8. Add 700 μl Buffer RW1 to the RNeasy spin column. Centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through. Reuse the collection tube in the next step.
9. Add 500 μl Buffer RPE to the RNeasy spin column. Close the lid gently, and centrifuge for 15 s at ≥8000 x g to wash the spin column membrane. Discard the flow-through. Reuse the collection tube in in the next step.
10. Add 500 μl Buffer RPE to the RNeasy spin column. Close the lid gently, and centrifuge for 2 min at .8000 x g to wash the spin column membrane
11. Place the RNeasy spin column in a new 2 ml collection tube, and discard the old collection tube with the flowthrough. Centrifuge at full speed for 1 min.
12. Place the RNeasy spin column in a new 1.5 ml collection tube. Add 30–50 μl RNase-free water directly to the spin column membrane. Centrifuge for 1 min at ≥8000 x g to elute the RNA.
13. Recover the flow-through, place it in the RNeasy spin column atop the same collection tube. Centrifuge for 1 minute at >8000 x g to finish the elution.
14. Characterize the obtained mRNA solution by nanodropping and store it at -80 until use.
Proceeded to the mRNA extraction from a seed CHO culture and a LovTAP-transfected CHO culture.
Nanodrop of mRNA
Protocol: DNA Concentration Measurement
Take a 6 µl aliquote of the DNA and put back the main DNA tube in the fridge.
Go to the room by the E.Coli lab (LBTM, not on Friday morning!) with:
The 6 µl aliquote
A 10 µl pipet
Optionally, the buffer you used for DNA elution (there might be some next to the machine).
The machine is the NanoDrop Spectrophotometer.
On the computer, click on "Nucleic Acid".
Put a 2 µl drop of (nuclease-free) water on the machine's tip as you are asked to and measure.
Clean tips (both sides) with a quarter of tissue.
Add 2 µl of the buffer you use and click on "Blank".
Clean tips (both sides).
Add 2 µl of your DNA sample and click "Measure".
Clean tips (both sides) with a tissue.
Take 2 measurements per sample (for averaging).
Print the report when you are done
Click on exit.
The important numbers are:
260/280 ratio, must be > 1.8
260/230 ratio, must be > 2 (too big, > 2.5? , might mean too much salts)
Of course the DNA concentration.
The final amount of mRNA harvested was 40uL for both cultures, at 77 ng/uL for LovTAP culture and 141 ng/uL for the seed culture according to nanodropping. Thus a total of 3.0 uG of mRNA for LovTAP and in all and 5.6uG of mRNA for the seed cells. However, the seed cells' mRNA seemed to be contaminated with something absorbing at 220 nm.
LovTAP RT-PCR
Protocol: RT-PCR
RT-PCR is a procedure that makes it possible to compare concentration of different mRNA in the cell culture. The first step is to select the mRNA which concentration we would like to compare and to design the relevant primers. Primer design can be done with the primer design tool from the university of Massachussets medical school. Once done, order the primers (in our case, we ordered from them IDT). When  you've received the primers, prepare them by diluting with a TE buffer to a high concentration, then prepare a working solution (generally 1μM). Make sure you've got your RT-PCR kit (we used the Power SYBR-Green RNA-to-CT 1 step Kit from Applied Biosystems) and the DNAses mix (we used the TURBO DNA-free™ Kit from Life Technology). Even though the DNAse mix is not necessary, it decreases the noise from the genomic DNA. Provide yourself with the 96 plates for the PCR (we used MicroAmp® Optical 96-Well Reaction Plates from Applied Biosystems) and plate cover film (we used MicroAmp® Optical Adhesive Film from Applied Biosystems).
Primer design
The primers for the RT-PCR should have a melting temperature between 56°C and 60°C (optimal at 58°C). The optimal length for the primers is 20 nucleotides with CG content between 40 and 60%. The final product should be about 100 bp long for the optimal results.
DNAse Treatment
Add 0.1 volume of the DNAse buffer to the mRNA solution
Add 1 uL of DNAse to the mRNA solution
Leave for incubation for 30 minutes at 37°C.
Add 0.1 volume of DNAse Inactivation Reagent to the solution, mix well by flicking. Do not centrifuge.
Leave for 5 minutes.
Centrifuge at 10 000 g for 1.5 minutes and transfer RNA to a fresh tube. Pay attention not to touch the beds, since it might inhibit the PCR reaction.
Master Mix
The composition for one 10.0μL tube is:
Reagent
Volume [μL]
SYBR Green RT-PCR Mix
5.0
Forward primer (2μM)
1.0
Reverse primer (2μM)
1.0
RT enzyme
0.08
Rnase- free H20
complete to 8μL
2μL of template will be added upon addition to the 96 well plate to complete to a final volume of 10μL.
Prepare one or two extra tubes-worth of reagent (you'll use some liquid on the walls of your tips)
Note : It is a good idea to prepare for each sample a master mix without the RT Enzyme, so that we can measure the degree of contamination with the genomic DNA.
Protocol
Get the 96 well plate and drop 2μL of template directly to the wells, then complete with 8μL of Master Mix to 10μL of total solution. Complete all the wells in rows (marked by letter), without leaving any column dropped out (it is better for the analysis).
Cover the plate with the plate cover film and get rid of all the bubbles by successively centrifugating - shaking the plate and centrifugating again.
Place the plate in the RT-PCR machine. Add an incubation step: 30 minutes at 48 degrees and configure it to run for 40 PCR cycles.
Used mRNA extracted in the previous step to perform an RT-PCR according to the protocol.
Performed the following dilutions:
LovTAP mRNA: 1, 1:5, 1:10
Seed mRNA: 1, 1:5, 1:10
Each of the dilutions was placed both in an RT enzyme-containing master mix and RT-enzyme free master mix.
In order to obtain a reference and check the functionality of the primer, we've also used a plasmid containing LovTAP at 100pM, 10pM and 1pM final concentration in an RT-free buffer, without being treated by the DNAses.
The final results weren't exploitable, most likely due to the contamination of the initial samples with a DNAse.
Wiki Links
Recent Changes
Upload file
Special Pages
My Preferences
Privacy Policy
Disclaimers
Important pages
Wiki editing
Planning
Meetings
Primer Design Helper
Add Page
Protocol List
Pages
Home
Team
Official Team Profile
Project
Parts Submitted to the Registry
Modeling
Notebook
Safety
Attributions
Sponsors
Concept Art
Retrieved from " http://2012.igem.org/Team:EPF-Lausanne/Notebook/24_September_2012 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
