"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF-Lausanne/Notebook/9 August 2012
From 2012.igem.org
Introduction
Our Project
Notebook
Human Practices
Acknowledgments
Contents
1 Plates observation
2 Overnight culture
2.1 Protocol: Prepare Plasmid Extraction (culture for Miniprep)
3 TNFR minipreps
3.1 Protocol: Miniprep
4 pHY42 sequencing
5 LovTAP into pcDNA3.1
6 Digestions
6.1 Protocol: Restriction site digestion
7 Nanodrop
7.1 Protocol: DNA Concentration Measurement
8 Digestion of the TNFR ligations
8.1 Protocol: Restriction site digestion
9 Gels
9.1 Protocol: Gel Electrophoresis
10 CH Lab
11 BM Lab
12 Gel extraction of LovTAP and pGL
12.1 Protocol: Gel Electrophoresis
13 CH Lab
14 BM Lab
15 LovTAP ligation into pMP
15.1 Protocol: Ligation
16 Transformation
16.1 Protocol: E.Coli Transformation
Plates observation
We have colonies for SEAP and eGFP!
Overnight culture
Protocol: Prepare Plasmid Extraction (culture for Miniprep)
Select and number colonies on the plates.
Prepare tubes of LB medium with the correct quantity of antibiotics (100 µg/ml for Amp, Spc or chloramphenicol).
Amp can be found in the -20ºC freezer at Ecoli, labeled as "stock". It is 100 µg/µl, or 1000x.
The tubes to be used are the 14 ml round bottom found in front of the iGEM drawers (Falcon). Culture with cap in the first step (loose) and close to the second step after culture.
Touch each colony with a clean pipette tip and put it in a tube.
Put in incubator.
Colonies were picked up from the SEAP and eGFP ligation plates. Cultures were made (23 tubes...).
TNFR minipreps
Protocol: Miniprep
The slim tubes can be centrifuged in the machine in front of the "Gel hood", at 4000 rpm for 10 min. The fatter ones, in the E. coli centrifuge by the fridge (the tip can be left inside, since it floats).
Pellets resuspended with RNase containing buffer (Resuspension Buffer R3, from Invitrogen, equivalent to Buffer P1 from Qiagen, in Sowmya's box in the fridge). Note: keep the buffer in ice if you are not bringing it back to the fridge for some minutes.
We then use the QIAGEN QIAprep Spin Miniprep Kit with their protocol (page 22) and a microcentrifuge.
10 minipreps were made from yesterday's cultures.
pHY42 sequencing
We received results for the melanopsin sequencing (to be able to design primers for it in the future because we did not know the exact plasmid sequence). It is a 100% match with mouse melanopsin found on the ENSEMBL database.
LovTAP into pcDNA3.1
It was decided to try to insert LovTAP into the pcDNA3.1(+) vector (to see if the cloning goes better and to have an alternative vector). This plasmid was part of the original design, so the restriction sites we needed were already present on the plasmid that came from direct synthesis.
Digestions
Protocol: Restriction site digestion
Look for the best pair of restriction sites, ideally with similar digestion temperatures and times.
NEBcutter for finding cutting enzymes.
Double Digest Finder for the parameters.
Calculate the amounts required of:
DNA
Buffer (usually from 10x to 1x)
BSA, if needed (usually from 100x to 1x)
Enzymes (depends on the amount of DNA)
Water
Get the recommended buffer (and BSA if needed) from the freezer and let defreeze.
Mix all the ingredients, except DNA, in a tube.
Note: Enzymes should stay no longer than a couple of minutes out of the freezer. Don't touch the bottom of the tubes! Don't vortex!
Distribute the mix in as many tubes as DNA samples and add the DNA.
Keep in the Thermomixer at the recommended temperature.
Sowmya's recommended amounts (50 µl total solution):
5 µl of 10x buffer
0.5 µl of 100x BSA
1 µl of each enzyme
5 µl of DNA
37.5 (up to 50 µl) of water.
Protocol based on what was done on July the 4th .
Digestion of pMA-LovTAP
The plasmid we obtained by direct synthesis was digested with EcoRI-HF and HindIII-HF, following the usual protocol.
Digestion of the pcDNA3.1(+) backbone
The backbone was also digested with EcoRI-HF and HindIII-HF. This digestion only removed a few nucleotides.
Digestion of the pGL backbone
The pGL4.30 backbone was redigested with FseI and HindIII for future ligations of the melanopsin experiment readouts, because we ran out of it.
Nanodrop
Protocol: DNA Concentration Measurement
Take a 6 µl aliquote of the DNA and put back the main DNA tube in the fridge.
Go to the room by the E.Coli lab (LBTM, not on Friday morning!) with:
The 6 µl aliquote
A 10 µl pipet
Optionally, the buffer you used for DNA elution (there might be some next to the machine).
The machine is the NanoDrop Spectrophotometer.
On the computer, click on "Nucleic Acid".
Put a 2 µl drop of (nuclease-free) water on the machine's tip as you are asked to and measure.
Clean tips (both sides) with a quarter of tissue.
Add 2 µl of the buffer you use and click on "Blank".
Clean tips (both sides).
Add 2 µl of your DNA sample and click "Measure".
Clean tips (both sides) with a tissue.
Take 2 measurements per sample (for averaging).
Print the report when you are done
Click on exit.
The important numbers are:
260/280 ratio, must be > 1.8
260/230 ratio, must be > 2 (too big, > 2.5? , might mean too much salts)
Of course the DNA concentration.
The Nanodrop gave acceptable results for all of the TNFR minipreps (around 300 ng/µl). The new LovTAP PCR yielded an average concentration of around 150 ng/µl.
Digestion of the TNFR ligations
Protocol: Restriction site digestion
Look for the best pair of restriction sites, ideally with similar digestion temperatures and times.
NEBcutter for finding cutting enzymes.
Double Digest Finder for the parameters.
Calculate the amounts required of:
DNA
Buffer (usually from 10x to 1x)
BSA, if needed (usually from 100x to 1x)
Enzymes (depends on the amount of DNA)
Water
Get the recommended buffer (and BSA if needed) from the freezer and let defreeze.
Mix all the ingredients, except DNA, in a tube.
Note: Enzymes should stay no longer than a couple of minutes out of the freezer. Don't touch the bottom of the tubes! Don't vortex!
Distribute the mix in as many tubes as DNA samples and add the DNA.
Keep in the Thermomixer at the recommended temperature.
Sowmya's recommended amounts (50 µl total solution):
5 µl of 10x buffer
0.5 µl of 100x BSA
1 µl of each enzyme
5 µl of DNA
37.5 (up to 50 µl) of water.
Protocol based on what was done on July the 4th .
To check if the TNFR fragment actually correctly ligated into pGL4.30, a digestion with HindIII-HF and BamHI was run. The digestion was incubated for 2 hours, then ran on a gel.
Gels
Protocol: Gel Electrophoresis
Agarose concentration depends on the size of the DNA to be run. We will mostly use 1%. VOL is the desired volume of gel in ml:
File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif
1kb ladder bands
CH Lab
Add 0.01*VOL g of agarose to a clean glass bottle.
Pour VOL/50 ml of 50xTAE in a graduated cylinder. Fill up to VOL ml with di water.
Add the resulting VOL ml of 1xTAE to the glass bottle with agarose.
Microwave, at 7, the bottle (loose cap!) until it boils.
Carefully remove bottle (can be super heated!) and check for the total absence of particles. Microwave again if needed.
Prepare a gel box, with comb, and fill it up with the agarose solution (maybe not the whole solution is needed).
Add 0.05 µl per ml of gel in the box of Red Gel (it's in the iGEM drawer) and stirr until disolved.
Wait until cold and solidified.
Carefully remove comb.
Place the box in the electrophoresis chamber.
Fill up the electrophresis chamber with 1x TAE buffer.
Add blue dye to the DNA samples (6x loading buffer, that is 10 µl in 50 µl of DNA solution).
Inject 30 µl of ladder marker in the first well (that's 1 µg of DNA).
Inject 60 µl of each DNA solution in the other wells.
Set voltage to 70-90 V and run for 30-40 min, or until the dye reaches the last 25% of the gel length (DNA travels from - to +).
Place the gel under the camera, cover, turn UV on and take photos!
Preparing the ladder:
get 1kb ladder DNA from the freezer (500 µg/ml).
for 30 charges, 30 µl per charge, we need 900 µl:
60 µl of 1kb ladder DNA
150 µl of dye (6x loading buffer)
690 µl of water
BM Lab
In this lab the gels are slightly different. The total volumes for the small, the medium and the large gel are respectively 60ml, 80ml and 90ml. As we use 0.5x TAE buffer instead of 1x, we can use higher voltages (170V seems to work fine). The gel should run 20-40 minutes, not more. As the gel is thinner, load less DNA (up to ~10ul).
The digested pGL backbone and the digested LovTAP-pMA were run on a gel to excise the wanted fragments and to purify them.
Gel picture
The gel was acceptable and could be used for a gel extraction.
The digested TNFR ligations were ran on a gel too.
Gel picture
The bands are of an extremely weird shape and intensity, the ladder can't even be seen. Maybe something went wrong during the gel casting. Some of them might be of the right size, but it's hard to tell. The best-looking ones are going to be used for the following experiments anyway.
Gel extraction of LovTAP and pGL
Protocol: Gel Electrophoresis
Agarose concentration depends on the size of the DNA to be run. We will mostly use 1%. VOL is the desired volume of gel in ml:
File:Team-EPF-Lausanne-Protocols-1kb-ladder.gif
1kb ladder bands
CH Lab
Add 0.01*VOL g of agarose to a clean glass bottle.
Pour VOL/50 ml of 50xTAE in a graduated cylinder. Fill up to VOL ml with di water.
Add the resulting VOL ml of 1xTAE to the glass bottle with agarose.
Microwave, at 7, the bottle (loose cap!) until it boils.
Carefully remove bottle (can be super heated!) and check for the total absence of particles. Microwave again if needed.
Prepare a gel box, with comb, and fill it up with the agarose solution (maybe not the whole solution is needed).
Add 0.05 µl per ml of gel in the box of Red Gel (it's in the iGEM drawer) and stirr until disolved.
Wait until cold and solidified.
Carefully remove comb.
Place the box in the electrophoresis chamber.
Fill up the electrophresis chamber with 1x TAE buffer.
Add blue dye to the DNA samples (6x loading buffer, that is 10 µl in 50 µl of DNA solution).
Inject 30 µl of ladder marker in the first well (that's 1 µg of DNA).
Inject 60 µl of each DNA solution in the other wells.
Set voltage to 70-90 V and run for 30-40 min, or until the dye reaches the last 25% of the gel length (DNA travels from - to +).
Place the gel under the camera, cover, turn UV on and take photos!
Preparing the ladder:
get 1kb ladder DNA from the freezer (500 µg/ml).
for 30 charges, 30 µl per charge, we need 900 µl:
60 µl of 1kb ladder DNA
150 µl of dye (6x loading buffer)
690 µl of water
BM Lab
In this lab the gels are slightly different. The total volumes for the small, the medium and the large gel are respectively 60ml, 80ml and 90ml. As we use 0.5x TAE buffer instead of 1x, we can use higher voltages (170V seems to work fine). The gel should run 20-40 minutes, not more. As the gel is thinner, load less DNA (up to ~10ul).
The DNA was extracted and purified.
LovTAP ligation into pMP
Protocol: Ligation
Ligation is a method of combining several DNA fragments into a single plasmid. This is often the step following a PCR (and a PCR cleanup ) or a gel extraction . You can also do a "dirty" ligation, where you follow a certain number of digestions directly by a ligation.
Download the following spreadsheet : File:Team-EPF-Lausanne Ligation.xls
Fill in the pink areas with the vector and fragment concentration, their size and the ratio.
Add all the suggested ingredients order in a microcentrifuge tube, in the order they appear.
Ligate for 2 hours at 14ºC.
Immediately transform competent bacteria with the ligation product.
Note: This protocol hasn't been optimized for blunt-end ligation (though it might still work).
The digested backbone and the digested PCR product from the day before were ligated together. We tried every ratio twice, and made a negative control. This gave 7 ligation tubes.
Transformation
Protocol: E.Coli Transformation
Thaw the competent E.coli (DH5alpha) cells on ice (not in hands!)
As soon as it is thawed, add 50µl of the cells to the DNA (~50-100 ng of pure plasmid, or some 2 µl usually)
Let it rest on ice for 20-30 min. Meanwhile, put agar plate (with correct antibiotic) at 37°C for prewarming.
Put the tube with DNA+E.coli at 42°C for 45 sec - 1 min (heat shock)
Add 400 µl of LB broth and place at 37°C for 20-30 min (shaking)
Spread the cells on the prewarmed plate (and let it dry)
Incubate the plate upside-down at 37°C for ~14-15 hours (leaving it more than 16h decreases the plasmid quality)
Bacteria were transformed with the ligation products, and plated on 7 plates.
Wiki Links
Recent Changes
Upload file
Special Pages
My Preferences
Privacy Policy
Disclaimers
Important pages
Wiki editing
Planning
Meetings
Primer Design Helper
Add Page
Protocol List
Pages
Home
Team
Official Team Profile
Project
Parts Submitted to the Registry
Modeling
Notebook
Safety
Attributions
Sponsors
Concept Art
Retrieved from " http://2012.igem.org/Team:EPF-Lausanne/Notebook/9_August_2012 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
