<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Tec-Monterrey_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Tec-Monterrey/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM 2016 - Tec de Monterrey</TITLE></P><DIV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"><SPAN class="sr-only">Toggle nav</SPAN></BUTTON><DIV class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><P>Team</P></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Attributions">Attributions</A></LI></UL><LI class="dropdown"><P>Project</P></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Description">Project</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Entrepreneurship">Business</A></LI><LI class="dropdown"><P>Human Practice</P></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Human_Practices">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Integrated_Practices">Integrated Practice</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Engagement">Engagement</A></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:Tec-Monterrey/Collaborations"><P>Collaborations</P></A></LI><LI class="dropdown"><P>Journal</P></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Experiments">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Safety">Safety</A></LI><LI class="dropdown"><P>Achievements</P></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Tec-Monterrey/Medals">Medals</A></LI></DIV></DIV></DIV></DIV><DIV class="container-fluid modelText" style="margin-top:6%"><DIV class="row"><NAV class="col-lg-2" id="myScrollspy"><UL class="nav nav-pills nav-stacked"><LI class="active"><A href="#intro1">Module 1:Introduction</A></LI><LI><A href="#consid">Considerations</A></LI><LI><A href="#cassette">Modeling ofthe Cassette</A></LI><LI><A href="#enzymek">Enzyme Kinetics</A></LI><LI><A href="#intro2">Module 2:Introduction</A></LI><LI><A href="#population">Population</A></LI><LI><A href="#quorum">Quorum Sensing</A></LI><LI><A href="#gold">Gold Cyanidation</A></LI><LI><A href="#overexp">HCN SynthaseOverexpression</A></LI><LI><A href="#refs">References</A></LI></UL></NAV><DIV class="col-lg-8 col-xs-12" align="justify"><DIV id="intro1" class="offset"><H1>Module 1: Introduction</H1><P class="”indent”"><I>Acidithiobacillus ferrooxidans</I> is a bacterium who usually lives in very acid environments (1.5-2 pH), it can be found mostly in acid mine drainage, and have the amazing property of being able to convert insoluble metals to their soluble state. That is why it is usually used in a process known as industrial bioleaching to extract otherwise unobtainable metals. This happens because <I>A. ferrooxidans</I> obtains its energy by oxidizing \(Fe^{2+}\) to \(Fe^{3+}\) and producing \(SO_{4}^{2-}\) during a process of electron exchange in the membrane. In the process of bioleaching <I>A. ferrooxidans</I> can be considered a clean alternative because it does not produce toxic residues.
                    </P><P class="”indent”">
                        Because of this, our team decided to use <I>A. ferroxidans</I> to bioleach metals as a way of tackling the problem of electronic waste. To increase the efficiency of the process our team proposed a mechanism in which <I>Tetrathionate Hydrolase</I> (TetH) is produced. The kinetics of this enzyme is discussed below.
                    </P><P class="indent">
                        We will use the following parameters in the development of the model:
                    </P><TABLE><TBODY><TR><TD>\(F_{in}\)</TD><TD>Average concentration of \(Fe^{2+}\) inside <I>A. ferroxidans</I></TD></TR><TR><TD>\(F_{out}\)</TD><TD>Average concentration of \(Fe^{2+}\) outside <I>A. ferroxidans</I></TD></TR><TR><TD>\(r_f\)</TD><TD>Rate of \(Fe^{2+}\) diffusion across the membrane for each cell</TD></TR><TR><TD>\(p\)</TD><TD>Number of <I>A. ferroxidans</I> cells</TD></TR><TR><TD>\(Fur\)</TD><TD>Average Fur concentration in <I>A. ferroxidans</I></TD></TR><TR><TD>\(C_1\)</TD><TD>Average concentration of Fur complexed with \(Fe^{2+}\)</TD></TR><TR><TD>\(k_{FC}\)</TD><TD>Rate constant for the formation of \(C_1\) complex</TD></TR><TR><TD>\(k_{CF}\)</TD><TD>Rate constant for the dissociation of \(C_1\) complex</TD></TR><TR><TD>\(\alpha_0\)</TD><TD>Basal expression of the cassette (coding for Fur and LacI)</TD></TR><TR><TD>\(K_1\)</TD><TD>\(C_1\) concentration occupying half of the binding sites</TD></TR><TR><TD>\(\delta_{Fur}\)</TD><TD>Degradation parameter for Fur</TD></TR><TR><TD>\(LacI\)</TD><TD>Average LacI concentration</TD></TR><TR><TD>\(\delta_{LacI}\)</TD><TD>Degradation parameter for LacI</TD></TR><TR><TD>\(\beta_0\)</TD><TD>Basal expression of TetH</TD></TR><TR><TD>\(K_2\)</TD><TD>LacI concentration occupying half of the binding sites</TD></TR><TR><TD>\(\delta_{TetH}\)</TD><TD>Degradation parameter for TetH</TD></TR><TR><TD>\(k_{TC}\)</TD><TD>Rate constant for the formation of \(C_2\) complex</TD></TR><TR><TD>\(k_{CT}\)</TD><TD>Rate constant for the dissociation of \(C_2\) complex</TD></TR><TR><TD>\(S\)</TD><TD>Average concentration of sulfur</TD></TR><TR><TD>\(C_2\)</TD><TD>Average concentration of TetH complexed with S</TD></TR><TR><TD>\(k_{CS}\)</TD><TD>Rate constant for the formation of the products of \(C_2\)</TD></TR></TBODY></TABLE></DIV><DIV id="consid" class="offset"><H1>Considerations</H1><P class="indent">
                        We will not consider the reactions going on outside the cell in order to simplify the model. We will assume that the more TetH is produced, the more Copper will be dissolved according to the reaction:
                        \begin{equation}
                            Cu + Fe_{2}(SO_{4})^{-2}_{3}\,\rightleftharpoons\, Cu^{+2} + 2\:Fe^{+2} + 3\:SO_{4}^{-2}
                        \end{equation}
                        The amount of dissolved \(Fe^{+2}\) in the medium will also increase. We will also consider the same population model we used for <I>Chromobacterium violaceum</I> in the section <A href="#population">below</A>.
                    </P></DIV><DIV id="cassette" class="offset"><H1>Modeling of the Cassette</H1><P class="”indent”">
                        Initially, the solution will contain \(Fe^{+2}\) that will be diffused through the cell membrane according to Fick's first law of diffusion:
                        \begin{equation}
                            \frac{dF_{in}}{dt} = r_{f}\pars{F_{out}-F_{in}}
                        \end{equation}
                        \begin{equation}
                            \frac{dF_{out}}{dt} = -pr_{f}\pars{F_{out}-F_{in}}
                        \end{equation}
                        This iron will combine with the protein Fur to form a complex which we will call \(C_{1}\) according to the reaction:
                        \begin{equation}
                            F_{in} + Fur \overset{k_{FC}}{\underset{k_{CF}}{\rightleftharpoons}} C_{1}
                        \end{equation}
                        The reaction rates can be written as:
                        \begin{equation}
                            \frac{dFur}{dt}= -k_{FC}FurF_{in} + k_{CF}C_{1}
                        \end{equation}
                        \begin{equation}
                            \frac{dF_{in}}{dt} = k_{FC}FurF_{in} + k_{CF}C_{1}
                        \end{equation}
                    </P><P class="”indent”">
                        The complex \(C_{1}\) will act as an inhibitor to the expression of the proteins Fur and LacI, so their expression rates will be:
                        \begin{equation}
                            \frac{dFur}{dt} = \alpha_{0} + \frac{1}{1+\frac{C_{1}}{K_{1}}} - \delta_{Fur}Fur
                        \end{equation}
                        \begin{equation}
                            \frac{dLacI}{dt} = \alpha_{0} + \frac{1}{1+\frac{C_{1}}{K_{1}}} - \delta_{LacI}LacI
                        \end{equation}
                        LacI also acts as an inhibitor to the expression of the enzyme <I>Tetrathionate hydrolase</I>. So, the expression of TetH is given by:
                        \begin{equation}
                            \frac{dTetH}{dt} = \beta_{0} + \frac{1}{1+\frac{LacI}{K_{2}}} - \delta_{TetH}TetH
                        \end{equation}
                    </P><P class="fig-capt">
                        Figure 1: As Fur-Fe\(^{2+}\) complex appears, it inhibits LacI production and the existing LacI degradates.
                    </P></DIV><DIV id="enzymek" class="offset"><H1>Enzyme Kinetics</H1><P class="”indent”">
                        For enzyme kinetics we will use the same analysis used on <I>C. Violaceum</I>. It is known that the reaction for TetH is given by:
                        \begin{equation}
                            TetH + S \overset{k_{TC}}{\underset{k_{CT}}{\rightleftharpoons}} C_{2} \overset{k_{CS}}{\rightarrow} SO_{4}^{-2} + S_{2}O_{3}^{-2} + 2\:H^{+} + TetH
                        \end{equation}
                    </P><P class="”indent”">
                        We then write the equations for the reactions:
                        \begin{equation}
                            \dfrac{dTetH}{dt} = -k_{TC}(TetH)(S) + k_{CT}C_{2} + k_{CS}C_{2}
                        \end{equation}
                        \begin{equation}
                            \frac{dS}{dt} = -k_{TC}(TetH)(S) + k_{CT}C_{2}
                        \end{equation}
                        \begin{equation}
                            \frac{dC_{2}}{dt} = k_{TC}(TetH)(S) - k_{CT}C_{2} - k_{CS}C_{2}
                        \end{equation}
                        \begin{equation}
                            \frac{dSO_{4}}{dt} = k_{CS}C_{2}
                        \end{equation}
                        \begin{equation}
                            \frac{dS_{2}O_{3}}{dt} = k_{CS}C_{2}
                        \end{equation}
                        \begin{equation}
                            \frac{dH^{+}}{dt} = 2k_{CS}C_{2}
                        \end{equation}
                        Notice that the pH of the solution will diminish because of the production of \(H^{+}\) by TetH.
                    </P><P class="fig-capt">
                        Figure 2: As LacI concentration goes down, the expression of TetH is allowed.
                    </P></DIV><DIV id="intro2" class="offset"><H1>Module 2: Introduction</H1><P class="indent"><I>Chromobacterium violaceum</I> is defined as a saprophyte bacteria, which is normally considered to be nonpathogenic for humans, and it is found in soil and water samples of tropical and sub-tropical areas of several continents. It was firstly observed during the second half of the XIX century and it was named after its violet coloration, which is given by violacein. However it wasn’t until 1976 that it was isolated.
                    </P><P class="indent">
                        Studies have found that a number of phenotypic characteristics in <I>C. violaceum</I> such as the production of violacein, hydrogen cyanide, antibiotics and exoproteases are regulated by the endogenous <I>N</I>-hexanoyl-L-homoserine lactone through the process of quorum sensing. Hydrogen cyanide produced by <I>C. violaceum</I> is generated by the enzyme HCN synthase, whose expression is regulated by quorum sensing.
                    </P><P class="indent">
                        In order to increase HCN production with the objective of dissolving gold, a genetic modification to overexpress HCN synthase is proposed. For the regulation of the mechanism, a promoter inducible by dicyanoaurate(I) ion is used. This way, overexpression will only happen in the presence of gold.
                    </P><P class="indent">
                        We start our model by describing the population of <I>C. violaceum</I>. We follow into describing the quorum sensing mechanism that activates the expression of HCN synthase and its HCN production. Then we take a rough approach to gold dissolution, which is quite a complex process. Finally we describe the overexpression of HCN synthase. All these processes occur simultaneously (of course, some depend upon others to be in a specific state). To get a better insight of the behavior of our descriptions, we simulate the processes in Matlab<SUP>®</SUP>.
                    </P><P class="indent">
                        We will use the following parameters in the development of the model:
                    </P><TABLE><TBODY><TR><TD>\(A\)</TD><TD>Average concentration of AHL inside <I>C. violaceum</I></TD></TR><TR><TD>\(A_{ext}\)</TD><TD>Average concentration of AHL outside <I>C. violaceum</I></TD></TR><TR><TD>\(d\)</TD><TD>Diffusion parameter for the autoinducer outside the cell</TD></TR><TR><TD>\(R\)</TD><TD>Concentration of unbound CviR</TD></TR><TR><TD>\(R^{\ast}\)</TD><TD>Concentration of CviR-AHL complex (2 AHL and 2 CviR molecules)</TD></TR><TR><TD>\(R_T\)</TD><TD>Total concentration of CviR monomers</TD></TR><TR><TD>\(I\)</TD><TD>Concentration of CviI monomers</TD></TR><TR><TD>\(p\)</TD><TD>Number of <I>C. violaceum</I> cells</TD></TR><TR><TD>\(k_{QS_0}\)</TD><TD>Rate constant for AHL production due to CviI concentration</TD></TR><TR><TD>\(k_{QS_1}\)</TD><TD>Rate constant for the formation of CviR-AHL complex</TD></TR><TR><TD>\(k_{QS_2}\)</TD><TD>Rate constant for the dissociation of the CviR-AHL complex</TD></TR><TR><TD>\(r_{QS}\)</TD><TD>Rate of AHL diffusion across the membrane for each cell</TD></TR><TR><TD>\(a_0\)</TD><TD>Basal production parameter of CviI</TD></TR><TR><TD>\(a\)</TD><TD>CviI production parameter due to bound CviR-AHL to the promoter</TD></TR><TR><TD>\(\delta_I\)</TD><TD>Degradation parameter for CviI</TD></TR><TR><TD>\(K_M\)</TD><TD>CviR-AHL concentration occupying half of the binding sites</TD></TR><TR><TD>\(E_{Cell}\)</TD><TD>Concentration of HCN synthase inside <I>C. violaceum</I> (non-active)</TD></TR><TR><TD>\(E_{Memb}\)</TD><TD>Concentration of HCN synthase in <I>C. violaceum</I>’s membrane (active)</TD></TR><TR><TD>\(b_0\)</TD><TD>Basal production parameter of HCN synthase</TD></TR><TR><TD>\(b\)</TD><TD>HCN synthase production parameter due to bound CviR-AHL to the promoter</TD></TR><TR><TD>\(\delta_E\)</TD><TD>Degradation parameter for HCN synthase</TD></TR><TR><TD>\(G\)</TD><TD>Glycine concentration</TD></TR><TR><TD>\(Ac\)</TD><TD>Acceptor concentration</TD></TR><TR><TD>\(C_1\)</TD><TD>Concentration of HCN synthase complexed with glycine</TD></TR><TR><TD>\(C_2\)</TD><TD>Concentration of HCN synthase complexed with glycine and two acceptors</TD></TR><TR><TD>\(AcH_2\)</TD><TD>Reduced acceptor concentration</TD></TR><TR><TD>\(HCN\)</TD><TD>HCN concentration</TD></TR><TR><TD>\(Au\)</TD><TD>Au <I>concentration</I> (amount of solid gold in the reaction tank)</TD></TR><TR><TD>\(AuCN_{out}\)</TD><TD>\(\text{Au}\pars{\text{CN}}_2^{-}\) concentration (extracellular)</TD></TR><TR><TD>\(AuCN_{in}\)</TD><TD>\(\text{Au}\pars{\text{CN}}_2^{-}\) concentration (intracellular)</TD></TR><TR><TD>\(k_{fAuCN}\)</TD><TD>Gold cyanidation forward rate constant</TD></TR><TR><TD>\(k_{rAuCN}\)</TD><TD>Gold cyanidation reverse rate constant</TD></TR><TR><TD>\(r_{AuCN}\)</TD><TD>Rate of dicyanoaurate(I) ion diffusion across the membrane for each cell</TD></TR><TR><TD>\(\gamma\)</TD><TD>HCN synthase production parameter due to bound \(\text{Au}\pars{\text{CN}}_2^{-}\) to the promoter</TD></TR><TR><TD>\(K_E\)</TD><TD>\(\text{Au}\pars{\text{CN}}_2^{-}\) concentration occupying half of the binding sites</TD></TR></TBODY></TABLE></DIV><DIV id="population" class="offset"><H1>Population</H1><P class="indent">
                        We start by proposing a model for the population growth. Since there shouldn’t be any threats to the population during the project’s processes and the medium will be kept well nourished within the time it involves, it can be assumed that the population will grow until it reaches an equilibrium in the medium because of the limited carrying capacity. Therefore, we use a logistic model.
                    </P><P class="indent">
                        The differential equation that describes this behavior is:
                            \begin{equation}
                                \label{eq:P}
                                \frac{dP}{dt}=rP\pars{1-\frac{P}{K_P}},
                            \end{equation}
                    </P><P>
                        where \(P\) is the total <I>C. violaceum</I> concentration at a given moment, \(r\) is the growth rate and \(K_P\) is the carrying capacity. This equation will be solved numerically along with the ones presented in the following sections to facilitate their coupled solution. However, we expect population to show the following behavior:
                            \begin{equation}
                                P(t)=\frac{P_0}{\frac{P_0}{K_P}+\pars{1-\frac{P_0}{K_P}}e^{-rt}},
                            \end{equation}
                        where \(P_0\) is the initial concentration: \(P(0) = P_0\).
                    </P><H3>Regression</H3><P>
                        We were able to measure <I>C. violaceum</I>'s population density in different time points via OD\(_{600}\). Using a regression algorithm we obtained the growth rate constant \(r\) and the carrying capacity constant \(K_P\) resulting in a value of \(\text{R}^2 = 0.9956\).
                    </P><P class="fig-capt">
                        Figure 3: Experimental data and fitted growth model of <I>C. violaceum</I> with growth rate constant \(r = 0.0156\text{ min}^{-1}\) and carrying capacity \(K_P = 1.3344\) OD\(_{600}\).
                    </P></DIV><DIV id="quorum" class="offset"><H1>Quorum Sensing</H1><P class="indent"><I>C. violaceum</I> has naturally evolved to produce an enzyme that catalyzes HCN production: HCN synthase. The promoter that encodes for this enzyme is regulated by a quorum sensing mechanism: a process through which bacteria <I>sense</I> the population density in their close surroundings with the help of the autoinducer (signaling molecule) <I>N</I>-Acyl homoserine lactone (AHL). We will use the quorum sensing model for <I>Vibrio fischeri</I> due to the similarities with the chemical species present in <I>C. violaceum</I>. In <I>V. fischeri</I> the transcriptional activator proteins LuxI and LuxR regulate the mechanism along with the signaling molecule AHL 3OC6HSL. <I>C. violaceum</I> uses CviI and CviR, which are homologue to LuxI and LuxR, respectively, and it uses the signaling molecule AHL C6HSL, which is very similar to 3OC6HSL.
                    </P><P class="indent">
                        AHL is produced inside the cell by CviI and it diffuses across the membrane in order to reach an equilibrium between intra- and extracellular concentrations. Two intracellular AHL molecules bind to two CviR monomers to form a CviR-AHL complex. CviR in this complex binds to an operator that enhances CviI production and therefore generates a positive feedback loop. This results in a steep switch-like response in CviI expression.
                    </P><H3>CviR-AHL Complex</H3><P class="indent">
                        The reaction that occurs between the unbound CviR and AHL can be expressed as follows:
                        \begin{equation}
                            2\:A+2\:R \overset{k_{QS_1}}{\underset{k_{QS_2}}{\rightleftharpoons}} R^{\ast}
                        \end{equation}
                    </P><P class="indent">
                        If we consider that the total amount of CviR is fixed, the amount of unbound CviR can be given by \(R = R_T – 2 R^{\ast}\), this is evident from the previous equation because two CviR monomers are needed in order to create a CviR-AHL complex. Writing the chemical kinetics following the law of mass action, we get:
                        \begin{equation}
                            \label{eq:CviI}
                            \frac{dR^{\ast}}{dt} = k_{QS_1}A^2\pars{R_T-2R^{\ast}}^2 - k_{QS_2}R^{\ast}
                        \end{equation}
                    </P><H3>CviI Transcription</H3><P class="indent">
                        To describe the transcription from DNA to a product protein we can use the equations (Ingalls, 2012):
                        \begin{equation}
                            \frac{dm}{dt}=k_{0} - \delta_{m}m(t), \qquad \frac{d\rho}{dt}=k_{1}m(t) - \delta_{\rho}\rho(t),
                        \end{equation}
                        where \(m\) and \(\rho\) represent the concentration of mRNA molecules and product protein, \(\delta_m\) and \(\delta_\rho\) represent their degradation constants, \(k_0\) is the transcription rate from DNA to mRNA and \(k_1\) is the transcription rate from mRNA to \(\rho\).
                    </P><P class="indent">
                        Because the degradation of mRNA molecules occurs much faster than the degradation of proteins, the system quickly falls in a quasi-steady-state where \(\frac{dm}{dt}=0\) and the concentration of m at that time is:
                        \begin{equation}
                            m^{ss}=\frac{k_{0}}{\delta_{m}}
                        \end{equation}
                        Substituting back in the equation for \(\rho(t)\) we have:
                        \begin{equation}
                            \frac{d\rho}{dt}=\alpha-\delta_{\rho}\rho(t),
                        \end{equation}
                        where \(\alpha=\frac{k_{1}k_{0}}{\delta_{m}}\) is the constant expression rate. Because CviI is part of an autoregulated gene circuit with positive feedback and \(R^{\ast}\) acting as a promoter, then the transcription of CviI can be expressed as:
                        \begin{equation}
                            \label{eq:I}
                            \frac{dI}{dt}=a_0 + \frac{aR^{\ast}}{K_{M} + R^{\ast}}-\delta_{I}I
                        \end{equation}
                        Here \(\alpha\equiv a_0\) for the basal production and \(\alpha\equiv a\) for the production where \(R^{\ast}\) promotes the transcription.
                    </P><H3>AHL Diffusion</H3><P class="indent">
                        As \(A\) is going to be diffused through the cell membrane, we apply Fick’s Law of diffusion. It is also produced linearly by CviI and we need to take into account that the degradation of \(R^{\ast}\) liberates 2 molecules of \(A\). Writing the equation for \(A\) will give us:
                        \begin{equation}
                            \label{eq:A}
                            \frac{dA}{dt} = -2k_{QS_1}A^2(R_{T} - 2R^{\ast})^2 + 2k_{QS_2}R^{\ast} + k_{QS_0}I-r_{QS}(A-A_{ext})
                        \end{equation}
                    </P><P class="indent">
                        We also include an expression for \(A_{ext}\) considering diffusion through the cell. For this we add the population term \(p\) because each cell is going to be producing the molecule and contributing to the extracellular space. A term is added for concentrations that diffuse far away from the cell population and don’t contribute to the reaction:
                        \begin{equation}
                            \label{eq:Aext}
                            \frac{dA_{ext}}{dt}=pr_{QS}(A-A_{ext})-dA_{ext}
                        \end{equation}
                        Equations \eqref{eq:CviI}, \eqref{eq:I}, \eqref{eq:A}, \eqref{eq:Aext}, along with the population growth equation \eqref{eq:P}, describe the process know as quorum sensing. The switch-like behavior can be seen in Figure (4).
                    </P><P class="fig-capt">
                        Figure 4: Following the population growth model from Figure (3) and the equations described in this section, we obtained a model for the concentration of CviI, which shows the steep switch-like activation of the quorum sensing mechanism. We took into account an approximation of OD\(_{600}\) to number of cells \(p=P\times10^8\) (\(P\) being the population density in OD\(_{600}\) and described by equation \eqref{eq:P}). Parameter values: \(k_{QS_0}=3.2\times10^{-4}\) (time\(^{-1}\)), \(k_{QS_1}=0.5\) (concentration\(^{-3}\cdot\)time\(^{-1}\)), \(k_{QS_2}=0.02\) (time\(^{-1}\)), \(r_{QS}=0.6\) (time\(^{-1}\cdot\)cell\(^{-1}\)), \(d=1000\), \(R_T=0.3\) (concentration), \(a_0=0.01\) (concentration\(\cdot\)time\(^{-1}\)), \(a=10\) (concentration\(\cdot\)time\(^{-1}\)), \(\delta_I=0.07\), \(K_I=0.01\) (concentration). The timescale is only accurate to the population as the parameters have arbitrary units for demonstration purposes.
                    </P><H3>HCN Synthase</H3><P class="indent">
                        In <I>C. violaceum</I> the quorum sensing mechanism also produces an enzyme called HCN synthase, which we will call \(E\). As this enzyme is encoded in the same cassette where CviI is encoded, it is also regulated by CviR-AHL and so the production rate of can be modeled in the same way. Taking into account the basal production in the absence of CviR-AHL and the degradation of \(E\), we get:
                        \begin{equation}
                            \label{eq:E0}
                            \frac{dE}{dt} = b_0 + \frac{bR^{\ast}}{K_{M} + R^{\ast}}-\delta_{E}E
                        \end{equation}
                    </P><P class="indent">
                        The enzyme then proceeds to insert itself into the membrane where it is active and can produce HCN. Because of this, there will be some enzymes inside the cell and others will be in the membrane. We propose a behavior similar to the law of mass action on the basis that the probability of the insertion into the membrane will depend on the amount of \(E\) in the cell and that it should reach an equilibrium where the membrane gets to a <I>saturated</I> status for that specific amount of \(E\). The <I>chemical-like</I> reaction of insertion (and <I>withdraw</I>) would be the following:
                        \begin{equation}
                            E_{Cell} \overset{k_{E_{on}}}{\underset{k_{E_{off}}}{\rightleftharpoons}} E_{Memb}
                        \end{equation}
                    </P><P class="indent">             Therefore the equations that describe HCN synthase (before its overexpression) would be:
                        \begin{equation}
                            \label{eq:EMemb}
                            \frac{dE_{Memb}}{dt} = k_{E_{on}}E_{Cell} - k_{E_{off}}E_{Memb}
                        \end{equation}
                        \begin{equation}
                            \label{eq:ECell}
                            \frac{dE_{Cell}}{dt} = b_0 + \frac{bR^{\ast}}{K_{M} + R^{\ast}} - \delta_{E}E_{Cell} - k_{E_{on}}E_{Cell} + k_{E_{off}}E_{Memb}
                        \end{equation}
                        We proceed to analyze the production of HCN using the Michaelis-Menten kinetics. A glycine molecule binds to a HCN synthase in the membrane, forming a complex \(C_1\). Two acceptors then bind to this complex to form a second one \(C_2\). Finally, \(C_2\) liberates its products: HCN, CO\(_2\) and the modified acceptors. The equations that describe this transitions are:
                        \begin{equation}
                            \begin{split}
                                E_{Memb} + G &amp;\overset{k_{EC}}{\underset{k_{CE}}{\rightleftharpoons}} C_{1}\\
                                C_{1} + 2\: Ac &amp;\overset{k_{CC_1}}{\underset{k_{CC_2}}{\rightleftharpoons}} C_{2}\\
                                C_{2} \overset{k_{CP}}{\rightarrow} HCN +&amp;\: CO_{2} + 2\: AcH_2
                            \end{split}
                        \end{equation}
                        Here \(Ac\) acts as an acceptor.
                    </P><P class="fig-capt">
                        Figure 5: This graph shows the consumption of glycine and the acceptor taking into account all the equations described in this model (i.e. it follows the overexpression of HCN synthase). It demonstrates a clear dependence on both glycine and the acceptor for HCN production: as long as there is a healthy population, there will be a production of HCN given that there are enough resources.
                    </P><P class="indent">
                        The derivation of the Michaelis-Menten equation requires that the amount of synthase remains constant but this is not the case. So we re-write equation \eqref{eq:EMemb} and write the kinetic equations for \(G\), \(C_1\), \(C_2\), \(Ac\), \(HCN\) and \(AcH_2\):
                        \begin{equation}
                            \frac{dE_{Memb}}{dt} = -k_{EC}E_{Memb}G + k_{CP}C_{2}+k_{CE}C_{1} + k_{E_{on}}E_{Cell}-k_{E_{off}}E_{Memb}
                        \end{equation}
                        \begin{equation}
                            \frac{dG}{dt}=-k_{EC}E_{Memb}G+k_{CE}C_{1}
                        \end{equation}
                        \begin{equation}
                            \frac{dC_{1}}{dt}=-k_{CE}C_{1}+k_{EC}EG+k_{CC_2}C_{2}-k_{CC_1}C_{1}Ac^{2}
                        \end{equation}
                        \begin{equation}
                            \frac{dC_{2}}{dt}=k_{CC_1}C_{1}Ac^2-k_{CC_2}C_{2}-k_{CP}C_{2}
                        \end{equation}
                        \begin{equation}
                            \frac{dAc}{dt}=-2k_{CC_1}C_{1}Ac^2+2k_{CC_2}C_{2}
                        \end{equation}
                        \begin{equation}
                            \frac{dHCN}{dt}=k_{CP}C_{2}
                        \end{equation}
                        \begin{equation}
                            \label{eq:AcH2}
                            \frac{dAcH_{2}}{dt}=2k_{CP}C_{2}
                        \end{equation}
                        These equations describe the HCN production by following the intermediate steps where the enzyme binds to glycine and the acceptors. Equation \eqref{eq:AcH2} describes the concentration of reduced acceptors, however it can be ignored in the simulation as it is a final byproduct.
                    </P><P class="fig-capt">
                        Figure 6: This graph shows HCN synthase inside <I>C. violaceum</I> and in its membrane (in arbitrary units). It was elaborated by taking into account the overexpression which is described later. A change in the slope clearly demonstrates the difference between the regular state and the overexpression state.
                    </P></DIV><DIV id="gold" class="offset"><H1>Gold Cyanidation</H1><P class="indent">
                        Gold cyanidation is a complex process in which solid particles of gold are transformed into dicyanoaurate(I) ion \(Au(CN)^−_2\) in the presence of cyanide. Its mechanism is extremely complicated due to the fact that it involves interactions between liquids and solids, it’s an heterogeneous reaction. Lots of research have been conducted in order to develop mathematical models for this phenomena. Each of these models take different approaches, some use gold discs (to simplify contact area) and others use small gold pebbles.
                    </P><P class="indent">
                        Because the focus of this mathematical model is to describe the bio-bricks and gold cyanidation is a very complex field of study, the latter is beyond our scope and so we will take a simplified approach. The chemical equation describing gold cyanidation is:
                        \begin{equation}
                            4\: Au + 8\: CN^{-} + O_{2} + 2\: H_{2}O \rightleftharpoons 4\: Au(CN)_{2}^{-} + 4\: OH
                        \end{equation}
                    </P><P class="indent">
                        For the kinetics of the reaction we will consider O\(_2\) concentration as constant due to the fact that oxygen can be constantly supplied to the reactor tank and small variations have little effect on the reaction speed. We will work with a constant pH of 9 (pH inside the reactor tank can be measured and controlled by a device that liberates a base into the solution, this came from previous simulations where there were significant pH changes; equation \eqref{eq:mergeAuCyan} has protons as a product, this also suggested the team to work with a proton pump). It should be noted that <I>C. violaceum</I> produces HCN, whose dissociation equation is:
                        \begin{equation}
                            HCN \rightleftharpoons H^{+} + CN^{-}
                        \end{equation}
                    </P><P class="indent">
                        To simplify the process we combine both reactions to obtain:
                        \begin{equation}
                            \label{eq:mergeAuCyan}
                            4\: Au + 8\: HCN + O_{2} \overset{k_{fAuCN}}{\underset{k_{rAuCN}}{\rightleftharpoons}} 4\: Au(CN)_{2}^{-} + 2\: H_{2}O + 4\: H^{+}
                        \end{equation}
                        It is important to note that the combination of these equations may create some deviations against the real cyanidation process. However, as we have stated, cyanidation is not our main focus and we have found this to considerably simplify our simulations of the model while maintaining the rest of the processes intact.
                    </P><P class="indent">
                        We then use simple kinetics to model the reaction as if it were homoge- neous but with a simple modification:
                        \begin{equation}
                            \frac{dHCN}{dt}=8\pars{-k_{fAuCN}HCN^{8}Au^{4} + k_{rAuCN}AuCN_{out}^4}Au^{2/3}
                        \end{equation}
                        \begin{equation}
                            \frac{dAu}{dt}=4\pars{-k_{fAuCN}HCN^{8}Au^{4} + k_{rAuCN} AuCN_{out}^4}Au^{2/3}
                        \end{equation}
                        \begin{equation}
                            \frac{dAuCN_{out}}{dt}=4\pars{k_{fAuCN}HCN^{8}Au^{4} - k_{rAuCN}AuCN_{out}^4}Au^{2/3}
                        \end{equation}
                        As noted in the equations above, a \(Au^{2/3}\) term was added. This was derived from the consideration that the reaction speed depends upon the superficial area of solid gold. Assuming perfect spheres of solid gold with an average diameter, we concluded that the proportion of superficial area to the amount of undissolved gold would be \([Au]^{2/3}\). Adding the production of \(HCN\) and its reaction with gold, we get:
                        \begin{equation}
                            \frac{dHCN}{dt} = 8\pars{-k_{fAuCN}HCN^{8}Au^{4} + k_{rAuCN}AuCN_{out}^4}Au^{2/3} + k_{CP}C_2
                        \end{equation}
                    </P><P class="fig-capt">
                        Figure 7: Following our approximation of gold cyanidation, this graph shows the dissolution of gold pebbles by HCN produced by <I>C. violaceum</I>. Although the time scale given is accurate for the population, this only shows the expected behavior as the rates of HCN production and gold cyanidation are unknown. However, it does show the dependence upon these rates and gives us an insight into how the process can be improved.
                    </P></DIV><DIV id="overexp" class="offset"><H1>HCN Synthase Overexpression</H1><P class="indent">
                        One of the modifications made to <I>C. violaceum</I> is the addition of a cassette that overexpresses HCN synthase. The promoter for this cassette is activated by the presence of the dicyanoaurate(I) ion. For this reason, it is important that we take into consideration the diffusion of the ion across the membrane. The reverse reaction of the dissolution inside the cells can be dismissed because of the lack of nucleation sites. Following Fick’s Law of diffusion we get:
                        \begin{equation}
                            \frac{dAuCN_{in}}{dt} = r_{AuCN}\pars{AuCN_{out}-AuCN_{in}}
                        \end{equation}
                        \begin{equation}
                            \frac{dAuCN_{out}}{dt} = -r_{AuCN}\pars{AuCN_{out}-AuCN_{in}}
                        \end{equation}
                    </P><P class="indent">
                        With this description of the diffusion of the ion, we can model the over-expression of HCN synthase as we have previously done so with CviI and HCN synthase in equations \eqref{eq:I} and \eqref{eq:E0}. This time, however, there is no basal expression and the degradation constant is the same, so we only expand upon equation \eqref{eq:ECell} adding the production rate modeled by the Hill equation with Hill coefficient \(n = 1\):
                        \begin{equation}
                                \frac{dE_{Cell}}{dt} = b_0 + \frac{bR^{\ast}}{K_{M} + R^{\ast}} + \frac{\gamma AuCN_{in}}{K_{E} + AuCN_{in}} - \delta_{E}E_{Cell} - k_{E_{on}}E_{Cell} + k_{E_{off}}E_{Memb}
                        \end{equation}
                        It should be noted that the overexpression of HCN synthase is activated after the regular expression regulated by quorum sensing as it depends on the detection of dicyanoaurate(I) ion.
                    </P><P class="fig-capt">
                        Figure 8: An increased slope clearly shows the effect of the overexpression of HCN synthase. A limit is reached when there are no resources left for the enzyme to catalyze the production (i.e. lack of glycine and/or acceptors).
                    </P></DIV><DIV id="refs" class="offset"><H1>References</H1><DIV style="padding-left: 4em; text-indent: -4em;"><P>
                            Alon, U. (2007) <I>An Introduction To Systems Biology: design principles of biological circuits.</I> London, UK: Chapman &amp; Hall/CRC.
                        </P><P>
                            BRENDA Enzyme Database: <A href="http://www.brenda-enzymes.org/enzyme.php?ecno=3.12.1.B1&amp;Suchword=&amp;reference=&amp;organism%5B%5D=Acidithiobacillus+ferrooxidans&amp;show_tm=0">http://www.brenda-enzymes.org/enzyme.php?ecno=3.12.1.B1&amp;Suchword=&amp;reference=&amp;organism%5B%5D=Acidithiobacillus+ferrooxidans&amp;show_tm=0</A>.
                        </P><P>
                            Durán, N., &amp; Menck, C.F.M. (2001) Chromobacterium violaceum: A Review of Pharmacological and Industrial Perspectives. <I>Critical Reviews in Microbiology</I> 27(3):201-222.
                        </P><P>
                            Ingalls, B. (2013) <I>Mathematical Modeling in Systems Biology: An Introduction.</I> University of Waterloo, Canada: MIT Press.
                        </P><P>
                            KEGG <I>DBGET</I> Database: <A href="http://www.genome.jp/dbget-bin/www_bget?reaction+R05704+R00374">http://www.genome.jp/dbget-bin/www_bget?reaction+R05704+R00374</A>.
                        </P><P>
                            Marsden, J.O., &amp; House, C.L. (2009) <I>The Chemistry of Gold Extraction.</I> Colorado, USA: SME.
                        </P><P>
                            Nancharaiah, Y.V., Venkata-Mohan, S., &amp; Lens, P.N.L. (2016) Biological and Bioelectrochemical Recovery of Critical and Scarce Metals. <I>Trends in Biotechnology</I> 34:2. doi:10.16/j.tibtech.2015.11.003.
                        </P><P>
                            Potysz, A., Lens, P.N.L., van de Vossenberg, J., Rene, E.R., Malgorzata, G., Guiband, G., Kierzack, J., &amp; van Hullebusch, E.D. (2016) Comparison of Cu, Zn and Fe bioleaching from Cu-metallurgical slags in the presence of <I>Pseudomonas fluorescens and Acidithiobacillus thiooxidans. Applied Geochemistry.</I> 68:39-52. doi:10.1016/j.apgeochem.2016.03.006.
                        </P><P>
                            Rubisov, D.H., Papangelakis, V.G., &amp; Kondos, P.D. (1996) Fundamental Kinetic Models For Gold Ore Cyanide Leaching. <I>Canadian Metallurgical Quarterly.</I> 35(4):353-361.
                        </P><P>
                            Valdés, J., Pedroso, I., Quatrini, R., Dodson, R.J., Tettelin, H., Blake-II, R., Eisen, J.A., &amp; Holmes, D.S. (2008) Acidithiobacillus ferrooxidans metabolism: from genome sequence to industrial applications. <I>BMC Genomics</I>, 9:597. doi:10.1186/1471-2164-9-597.
                        </P></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" style="height:80px; background-color:grey; color:white" align="center"><P style="text-align:center;"><B>Brought to you by iGEM Tec-Monterrey 2016</B></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>