<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_RDFZ-China_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:RDFZ-China/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>RDFZ-China</TITLE><TITLE>RDFZ-China</TITLE></P><NAV class="site-nav"><UL><LI class="team"><A href="https://2017.igem.org/Team:RDFZ-China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Attributions">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Notebook">Notebook</A></LI></UL><LI class="project"><A href="https://2017.igem.org/Team:RDFZ-China/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Experiments">Experiments</A></LI><LI class="awards"><A href="https://2017.igem.org/Team:RDFZ-China/Applied_Design">Awards</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Entrepreneurship">Entrepreneurship</A></LI><LI class="parts"><A href="https://2017.igem.org/Team:RDFZ-China/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Composite_Part">Composite Parts</A></LI><LI class="human"><A href="https://2017.igem.org/Team:RDFZ-China/HP/Silver">HumanPractice</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/HP/Gold_Integrated">HP Gold</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Engagement">Public Engagament</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Collaboration">Collobration</A></LI><LI class="model"><A href="https://2017.igem.org/Team:RDFZ-China/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Hardware">Hardware</A></LI></NAV><MAIN><SECTION class="title"><DIV class="title-content"><H1>Introduction</H1><P>Our cellular model develops on the characterization on srfA, comA and sfp gene expression pathway as well as trying to predict the final productivity of surfactin over time. The main tool wo used to build our model is Matlab,sponsored by Mathworks. To be more specific, we use Simbiology which is a plugin toolbox that specifically designed for computational system biology. By writing deterministic differential equations of reaction inside the cell, we can easily create the pathway that resembles real-life situations. </P><H1>The signalling cascade</H1><P>To start with characterization of our gene pathway, our model has been assumed to act as a standard three-component regulation pathway. This type of modelling is well established in the past(Chen, He and Church.1999)(Ingalls, Brian, 2012)
                </P><P>In our model, we assumed the biochemical reaction take place in sequence as shown below:</P><OL><LI>srfA, comA and sfp are transcripedt and translated simultaneously.</LI><LI>srfA activator,which is coded by comA, helps in transciption of srfA which codes for the synthesis of surfactin synthethase</LI><LI>PPTase, which is coded by sfp, helps in activation of surfactin synthethase, making it become holo synthethase which will produce surfactin by binding with the amino acids.</LI></OL><H1>Transcription</H1><P>In real life situation, the transcription rate can be affected by many factors. However we found out through literature research that the major factor would be the initiation of transcription. Therefore we assumed that: </P><OL><LI>The overall transcription rate is primarily determined by the strength of the promoter in the pathway</LI><LI>The promoter strength is constant throught at maximum expression level the whole process</LI><LI>For constitutive promoter, we assume that the expression level is at the maximum rate for all time</LI></OL><P>In our final construct, the sfp is under the control of pVEG. According to our literature research, pVEG is 100 times stronger than J23101 which has been characterized in terms of relative units by Berkeley iGEM2006 team. From the database entry we can see that J23101 is 1791 times stronger than promoter J23112 which has a value of 1. Therefore our promoter, pVeg, would be 1791*100 times stronger than J23112. Now, J23101 has been previously characterized by Kelly et al. 2009, whoe found that it has a value of 0.03PoPS(polymerase per second). Therefore, by multiplying 0.03PoPS with 100, we can get 3.00PoPS for the strength of pVEG. Finally, we conclude that sfp, under the control of pVEG, has a constitutive ranscription rate of 3.00PoPS per molecule of DNA construct, and this can be translated to  </P><H1>Translation</H1><P>After the mRNAs are formed, they will then be translated into functinonal proteins consists one or more polypeptide chains. It is universally acknowledged that translation involves in three main stages which are initiation, elongation and termination in sequence. Hence the rate of translation can be affected by three main process:</P><OL><LI>The binding of the ribosome to the RBS(ribosome binding site)</LI><LI>Formation of initial complexes</LI><LI>Speed of the ribosome across the mRNA</LI></OL><P>In our model, we assume the speed of the ribosome across the mRNA to be the main factors in this process. Unfortunately, we couldn't find the parameters in <I>B.subtilis</I>, therefore, we decided to use the parameters in <I>E.coli</I> instead. According to our literature research, the translation rate in E.coli has been characterized.(Young and Bremer, 1976 and Proshkin et al. 2010.), which gives us 17.1 amino acids/second.</P><H1>Degradation</H1><P>The lifetime of mRNA is relatively short, but it is still a factor we need to concern in our simulation in order to be more precise. We take the average value(0.0028molecule per second) of mRNA degradation in E.coli since we couldn't find accurate parameters in B.subtilis.</P><H1>Parameters in Michaelis-Menten equation</H1><P>The synthesis of surfactin, a cyclic lipopeptide, involves a non-ribosomal peptide synthetase complex, including acyl carrier protein (ACP) for fatty acid synthesis and peptide carrier protein (PCP) for polypeptide synthesis. The non-ribosomal peptide synthetase stays in inactive apo-synthetase form until modified by 4′-phosphopantetheinyl transferases, which is Sfp in the case of Bacillus subtilis. The Sfp transfers the 4′-phosphopantetheinyl moiety of coenzyme A to the carrier proteins, thereby activates them. This reaction is an enzyme-substrate reaction, which is suitable to be modeled by Michaelis-Menten equation.</P><P>According to Mohammad R. Mofid (2002), the kinetics constants for Sfp and apo-PCP in Michaelis-Menten equation are determined, with a <B>K</B>m of 4.45  ± 1 μM and <B>K</B>cat of 96  ± 4/min. Based on the calculation of <B>K</B>cat = Vmax/[E], where [E] is the concentration of enzyme in solution, we can calculate the Vmax for Sfp and apo-PCP binding. Then a system model of Michaelis-Menten equation can be built for the reaction.</P><H1>Enzymic Reaction of Surfactin Synthesis</H1><P>The synthesis of surfactin involves an enzymic catalyzed reaction. Therefore we use Michaelis-Menten equation to calculate the rate of reaction between substrate amino acid and holo surfactin synthetase, when the concentration of enzyme is constant. We deduced that the overall rate of reaction is proportional to the concentration of enzyme, so we multiply the concentration of holo synthetase to the reaction rate which is affected by substrate concentration.</P><H1>Sensitivity Analysis</H1><P>Compared to the sophisticated pathways existing in nature, our model is a relatively simple simulation, but still, it leads to a model with many parameters. Apparently, those parameters do not contribute equally to the final result. By using Sensitivity Analysis in Matlab, it is possible to find out which one is the factor that has the greatest influence.</P><P>The sensitivity analysis uses complex approximation to calculate the derivative of the final out value respect to each initial individual parameters.</P><H1>Result</H1><P>The graph we obtained from model using Matlab shows that the graph, the concentration of holo synthetase grows exponentially, the concentration of amino acid decreases exponentially until it reach zero, and the concentration of surfactin increases exponentially until amino acid reaches zero: no substrate available. The reaction rate of surfactin synthesis does not decrease when substrate amino acid is at low concentration and deceasing because the concentration of  holo synthetase is increasing exponentially; however, we still reckon low concentration of substrate should be more determinant compared to high concentration of holo synthetase.</P><H1>Reference</H1><P>M. R. Mofid, R. Finking, M. A. Marahiel. (2002). <I>Recognition of Hybrid Peptidyl Carrier Proteins/Acyl Carrier Proteins in Nonribosomal Peptide Synthetase Modules by the 4′-Phophopantetheinyl Transferases AcpS and Sfp.</I> The Journal of Biological Chemistry, 2002 277: 17023-, doi:10.1074/jbc.M200120200
J. G. Owen, J. N. Copp, D. F. Ackerley. (2011). <I>Rapid and flexible biochemical assays for evaluating 4′-phosphopantetheinyl transferase activity.</I> Biochemical Journal, 436 (3) 709-717, doi: 10.1042/BJ20110321
</P><P>Chen, Ting, Hongyu He, and George Church.<I> Modeling Gene Expression With Differential Equations.</I> 1st ed. Boston: Harvard Medical School, 1999. Web. 9 Oct. 2016.</P><P>Ingalls, Brian. <I>Mathematical Modelling In Systems Biology: An Introduction. </I>Waterloo: University of Waterloo Department of Applied Mathematics, 2012. Print.</P></DIV></SECTION></MAIN></DIV></DIV></DIV></DIV></DIV></BODY></HTML>