<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_RDFZ-China_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:RDFZ-China/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>RDFZ-China</TITLE></P><NAV class="site-nav"><UL><LI class="team"><A href="https://2017.igem.org/Team:RDFZ-China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Attributions">Attribution</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Notebook">Notebook</A></LI></UL><LI class="project"><A href="https://2017.igem.org/Team:RDFZ-China/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Experiments">Experiments</A></LI><LI class="awards"><A href="https://2017.igem.org/Team:RDFZ-China/Applied_Design">Awards</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Entrepreneurship">Entrepreneurship</A></LI><LI class="parts"><A href="https://2017.igem.org/Team:RDFZ-China/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Composite_Part">Composite Parts</A></LI><LI class="human"><A href="https://2017.igem.org/Team:RDFZ-China/HP/Silver">HumanPractice</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/HP/Gold_Integrated">HP Gold</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Engagement">Public Engagament</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Collaboration">Collobration</A></LI><LI class="model"><A href="https://2017.igem.org/Team:RDFZ-China/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:RDFZ-China/Hardware">Hardware</A></LI></NAV><MAIN><SECTION class="title"><DIV class="title-content"><DIV class="slideshow-container"><DIV class="mySlides fade" style="display: none;"><DIV class="numbertext">1 / 3</DIV></DIV><DIV class="mySlides fade" style="display: none;"><DIV class="numbertext">2 / 3</DIV></DIV><DIV class="mySlides fade" style="display: block;"><DIV class="numbertext">3 / 3</DIV></DIV><A class="prev" onclick="slides.prev();" onmouseenter="slides.clearInterval();" onmouseleave="slides.setInterval();">❮</A><A class="next" onclick="slides.next();" onmouseenter="slides.clearInterval();" onmouseleave="slides.setInterval();">❯</A></DIV><H1>Assembly</H1><P>We hereby describe the assembly process involved in our project:</P><H2>Gibson Assembly</H2><P>The method can simultaneously connect up to 15 DNA fragments based on complementary ends, which contains 20-40 base pair overlap with adjacent DNA fragments. These fragments are then mixed with three enzymes, along with other buffer components for reaction.</P><P>The three required enzyme activities are: exonuclease, DNA polymerase, and DNA ligase:</P><UL><LI>The exonuclease removes several nucleotides from the 5' end, producing single-stranded regions on adjacent DNA fragments for annealing</LI><LI>The DNA polymerase incorporates nucleotides to fill in any gaps</LI><LI>* The DNA ligase joins the DNA of adjacent segments and remove nicks in the DNA.</LI></UL><P>The resulting product is all DNA fragments joined into one. Either linear or closed circular molecules can be assembled through Gibson Assembly.</P><P>We used Gibson Assembly to create our integration vector for <I>Bacillus subtilis.</I>This vector has antibiotic resistance genes on its backbone and origin of replication for <I>E. coli</I> but not <I>B. subtilis</I>, thereby making the plasmid easily lost in the latter host, allowing the plasmid to disappear after gene insertion by homologous recombination. The fragment to be inserted is flanked by two homologous arms of several hundred base pairs, allowing double recombination to take place. In this case, we chose amyE for our locus of insertion because interruption of this gene cause observable change in phenotype: loss of ability to digest amylose that can be checked with the iodine solution.</P><H2>3A Assembly</H2><P>3A Assembly (which stands for three antibiotic assembly) is a method for assembling two part samples and selecting for correct assemblies through antibiotics. 3A assembly uses the restriction sites on the prefix and suffix to assemble part samples. This new composite part will maintain the same prefix and suffix as its &quot;parents&quot; and contain a scar, where the cut and re-ligated restriction sites were stitched together.</P><P>It uses effective antibiotic selection to eliminate unwanted background colonies and eliminates the need for gel purification and colony PCR of the resulting colonies. In theory, about 97% of the colonies should be the desired assembly. Additionally, the Registry has been providing iGEM teams and Registry labs with linearized plasmid backbones to further improve this assembly method.</P><P>We used 3A assembly to put together pieces of genetic elements like promoters, ribosomal binding site and coding sequences. There’s a little bit of difference between our experiment and assembly in the picture, sfp is joined with pVeg and RBS. Then this part is joined with another RBS and the following YerP or lmrA to construct the full operon. Finally, the operon is cut from the plasmid and inserted onto the integration vector.</P><H1>Protocols</H1><H2>Sand Pack Test</H2><P>Materials:</P><OL><LI>A type of surfactant</LI><LI>Petroleum or sunflower oil</LI><LI>Soil sample</LI><LI>Tap water</LI><LI>Plastic bottles</LI><LI>Frame</LI></OL><P>Investigating the optimal concentration:</P><OL><LI>Prepare plastic bottles and place them in the frame.</LI><LI>Weigh about 200g of soil sample, and fill them into the plastic bottle.</LI><LI>Measure 20ml of oil with a measuring cylinder, and add them to each soil sample.</LI><LI>Wait for 2hrs to allow the oil to combine with the soil.</LI><LI>Prepare surfactant solutions with different concentrations.</LI><LI>Add 20ml of surfactant solutions to the oil.</LI><LI>Place a cup under each bottle. Wait for 2hrs for the emulsion to occur.</LI><LI>Carefully cover the cups and freeze them in -20℃.</LI><LI>After the liquid freezes, separate the ice and soil residue from the remaining liquid by filtering. The volume of liquid should be equal to the volume of oil eluded from the soil. Record it in the table below.</LI><LI>Calculate the percentage elusion with (V/20ml)*100%.</LI></OL><P>THERE SHOULD BE A PICTURE OF THE TABLE FOR DATA</P><H2>OEPCR(Overlap Extension PCR)</H2><P>Overlap Extension PCR involves designing two mutagenic primers, containing the mutation and partially or completely complementary to each other. Each primer is used in a separate reaction with an outer flanking primer designed to one end of the region of interest. You generate two halves of the region in this manner in two separate reactions and put them together in the next step, where they anneal in the 25-30 bp region of complementarity and prime off each other, to give you the full length product. All you need is Pfu polymerase, and the target sequence on a plasmid. </P><P>Materials:</P><OL><LI>Template for the left and right fragment</LI><LI>Primers A,B,C,D.</LI><LI>2x Pfu mastermix</LI><LI>DNA purification kit</LI></OL><P>OEPCR Steps:</P><OL><LI>Amplify desired DNA fragments on left and right template with AB and CD primer pairs correspondingly. The annealing temperature is set to 55°C. When designing the primers, B is made to be complementary to C, with 30 bases overlapping with ends of the left and right fragments to be joined. (Make Tm be as close to 60°C as possible.)</LI><LI>Directly purify PCR product with a kit. Elute DNA with ddH2O.</LI><LI>Set a new PCR reaction with specific requirements: equimolar amount of left and right fragment PCR product from step 1, combined volume of the fragments should be 1/2 to 3/4 of the total volume, no new primers is added, annealing temperature is set to 60°C. Run PCR for 15 cycles.</LI><LI>Add primers AD to the previous PCR product and set annealing temperature to 72°C and run PCR for another 15-20 cycles.</LI><LI>Electrophorese, and cut gels of correct product length and recycle OEPCR product.</LI></OL><P>(If the amount of OEPCR product is not enough for later experiments, consider amplifying the product with AD primers, but be aware of nonspecific amplification results!)</P><H2>Bacillus Subtilis Transformation</H2><P>Preparation of Bacillus subtilis competent cell:</P><OL><LI>Streak out the strain to be made competent on an LB or TBABG [Trytose blood agar base (Difco) + 0.5% glucose] agar plate as a large patch and incubate overnight at 30°C.</LI><LI>The following morning scrape the cell growth off the plate and use to inoculate fresh, pre-warmed, SpC mediuma (20 ml) to give an OD600 reading of about 0.5. </LI><LI>Incubate the culture at 37°C with vigorous aeration and take periodic OD readings (OD600) to assess cell growth. </LI><LI>When the rate of cell growth is seen to depart from exponential (i.e. no significant change in cell density over 20-30 min) inoculate 200 ml of pre-warmed, SpII mediuma with 2 ml of stationary-phase culture and continue incubation at 37°C with slower aeration </LI><LI>After 90 min incubationb, pellet the cells by centrifugation (8,000 g, 5min) at room temperature. </LI><LI>Carefully decant the supernatant into a sterile container and save.</LI><LI>Gently resuspended the cell pellet in 18 ml of the saved supernatant and add 2 ml of sterile glycerol; mix gently.</LI><LI>Aliquot the competent cell (0.5 ml) in sterile tubes, freezed rapidly in liquid nitrogen or a dry-iced/ethanol bath or ice/isopropanol bath and store -70°C.</LI></OL><P>Transformation:</P><OL><LI>Thaw competent cells rapidly by immersing frozen tubes in a 37°C water bath. </LI><LI>Immediately, add one volume of SpII + EGTA to the Thawed cells; mix gently.</LI><LI>In a sterile test tube add competent cell (0.2~0.5 ml) to the DNA solution (&lt;*0.1 ml) and  incubate in a roller drum at 37°C.</LI><LI>Dilute the transformed cells as appropriate in T base containing 0.5% glucose and plate  immediately onto selective media.</LI></OL><P>Media and Antibiotics used in two-step transformation:</P><P>T-base(per liter):</P><UL><LI>(NH4)2SO4 - 2g</LI><LI>K2HPO4·3H2O - 18.3g</LI><LI>KH2PO4 - 6g</LI><LI>trisodium citrate·2H2O - 1g</LI><P>Autoclave</P></UL><P>SpC(Made fresh on the day of use from the following sterile solution):</P><UL><LI>T base - 20ml</LI><LI>50% (w/v) glucose - 0.2ml</LI><LI>1.2% (w/v) MgSO4·3H2O - 0.3ml</LI><LI>10% (w/v) Bacto yeast extract - 0.4ml</LI><LI>1% (w/v) casamino acids - 0.5ml</LI><LI>Extra nutrients if the strain is auxotroph.</LI></UL><P>SpII(Made fresh on the day of use from the following sterile solutions):</P><UL><LI>T base - 200ml</LI><LI>50% (w/v) glucose - 2ml</LI><LI>1.2% (w/v) MgSO4·3H2O - 14ml</LI><LI>10% (w/v) Bacto yeast extract - 2ml</LI><LI>1% (w/v) casamino acids - 2ml</LI><LI>0.1 M CaCl2 - 1ml</LI><LI>Extra nutrients if the strain is auxotroph.</LI></UL><P>SpII + EGTA: SpII (200 ml) with 4 ml EGTA (0.1 M, pH 8.0) but without CaCl2. SpII + EGTA can be frozen at -20°C in small aliquots, although repeated freeze-thawing should be avoided.</P><P>Antibiotics for <I>Bacillus subtilis</I> selection:</P><UL><LI>Chloramphenicol(Cm) - 5 μg/ml, 95% ethanol</LI><LI>Erythromycin(Em) - 1 μg/ml, 95% ethanol</LI><LI>Lincomycin(Lm) - 25 μg/ml, 95% ethanol</LI><LI>Neomycin(Nm) - 5 μg/ml, dH2O</LI><LI>Kanamycin(Km) - 10 μg/ml, dH2O</LI><LI>Phleomycin(Pm) - 0.1-0.5 μg/ml, dH2O</LI><LI>Tetracycline(Tc) - 20 μg/ml, 50% ethanol</LI><LI>Spectinomycin(Spc) -100 μg/ml, dH2O</LI></UL><H2>Colony PCR</H2><P>Colony PCR is can be used after a transformation to screen colonies for the desired plasmid. Primers are used which generate a PCR product of known size. Thus, any colonies which give rise to an amplification product of the expected size are likely to contain the correct DNA sequence.</P><P>First, prepare a tube of master mix for many individual reactions. Each reaction is 20μl, prepare one extra reaction to account for loss during pipetting.</P><P>Recipe for a single reaction:</P><UL><LI>EasyTaq SuperMix (2x) - 10μl</LI><LI>Forward Primer - 1μl</LI><LI>Reverse Primer - 1μl</LI><LI>dH2O - 8μl</LI></UL><P>Setting up for PCR:</P><UL><LI>Choose a colony on a plate and then pick it up with a white pipette tip. Be careful not to stick the tip into agar or neighboring colonies,</LI><LI>Streak a short line on a new antibiotic plate to save for future experiment.</LI><LI>Pipette up and down in a PCR tube with 20μl of colony PCR master mix to transfer the remaining bacteria into the tube. (Do not leave too much bacteria.)</LI><LI>Set the first denaturing step to 15 minutes to completely degrade the bacterial cell wall to release plasmids and genomic DNA as templates for amplification.</LI><LI>Run gel to check PCR result. If nonspecific amplification occurs, go back and check primer specificity with genomic sequences, and then reduce cycle number from 30-35 to 25 and run the reaction again.</LI></UL><H2>Genomic DNA Extraction from Gram-Positive Bacteria</H2><P>Gram-postive bacteria are notorious for their thick and hard-to-break peptidoglycan cell walls in comparison to those of Gram-negative bacteria. Therefore, a more harsh treatment(lysozyme incubation) of cells must be used to lyse them and get the genomic DNA.</P><P>Recipe for our homemade lysozyme mixture(including buffer, 180μl per rxn):</P><UL><LI>Tris - 20mM</LI><LI>2Na·EDTA - 2mM</LI><LI>Lysozyme - 20mg/ml</LI></UL><P>Store the solution at -20°C in small aliquots, and avoid repeated thawing.</P><P>Then lyse the bacteria as follows:</P><OL><LI>Centrifuge at 3000rpm for 30s to softly precipitate the cells. Dispose the supernatant.</LI><LI>Add 180μl lysozyme mixture, pipette the cells well to resuspense. Digest overnight in a 37°C incubator.</LI></OL><H1>References</H1><OL><LI>Gibson DG, Young L, Chuang RY, Venter JC, Hutchison CA 3rd, Smith HO (2009). &quot;Enzymatic assembly of DNA molecules up to several hundred kilobases&quot;. Nature Methods. 6 (5): 343–345. PMID 19363495. doi:10.1038/nmeth.1318.</LI><LI>Colin R. Harwood, Simon M. Cutting(1991). &quot;Molecular Biological Methods for Bacillus&quot;. Wiley, ISBN:0471923931.</LI><LI>Nandita Mullapudi，“Overlap Extension PCR”， Retrieved from &quot;https://openwetware.org/wiki/Overlap_Extension_PCR&quot;</LI><LI>Anonymous, &quot;Colony PCR&quot;, Retrieved from &quot;https://openwetware.org/wiki/Colony_PCR&quot;</LI></OL></DIV></SECTION></MAIN></DIV></DIV></DIV></DIV></DIV></BODY></HTML>