<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Dundee_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Dundee/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Dundee/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Dundee/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Dundee/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="globalWrapper"><DIV id="contentMain" class="drop-shadow raised clearfix"><DIV id="menuDiv"><UL><LI class="active "><A href="https://2012.igem.org/Team:Dundee"><SPAN>Home</SPAN></A></LI><LI class="has-sub "><A href="https://2012.igem.org/Team:Dundee/Team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Team"><SPAN>Team Members</SPAN></A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Dundee"><SPAN>iGEM team Profile</SPAN></A></LI></UL><LI class="has-sub"><A href="https://2012.igem.org/Team:Dundee/Project"><SPAN>Project</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Project"><SPAN>The Problem</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Solution"><SPAN>Our Solution</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Biobricks"><SPAN>Biobricks</SPAN></A></LI><LI class="has-sub"><A href="https://2012.igem.org/Team:Dundee/Strategy"><SPAN>Wet Lab</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Strategy"><SPAN>Strategy</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Results"><SPAN>Experimentation</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Dry Lab</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Modelling4"><SPAN>Modelling</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Software"><SPAN>Software</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>Human Practices</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Safety"><SPAN>Safety</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Outreach"><SPAN>Outreach</SPAN></A></LI><LI class="has-sub"><A href="#"><SPAN>More</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Collaboration"><SPAN>Collaboration</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/Song"><SPAN>Song!</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Dundee/References"><SPAN>References</SPAN></A></LI><LI class="active"><A href="https://2012.igem.org/Team:Dundee/Attributions"><SPAN>Attributions</SPAN></A></LI></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week One</H3><DIV class="boxInner rounded centre" id="w1"><STRONG>Biology</STRONG>
                The first week for the biologists was a general introduction to the ways of the lab. 
We were given a health and safety briefing and attended a health and safety seminar (an experience that would terrify any undergrad new in the lab). The team was taken through all the experiments that would be crucial for the project and a general plan of attack was devised. The main achievements of this week were the production of competent DH5α E. coli cells; purification and amplification of Salmonella typhimurium LT2 chromosomal DNA and digestion of pUNI-PROM with EcoRI and BamHI or XbaI BamHI. 
<STRONG>Maths</STRONG>
The Mathematics team began by looking at common modelling methods in biology. After some research, we started to piece together a system of ordinary differential equations (ODEs) representing the populations of E. coli and C. difficile using logistic growth models. We also practiced with computer programs such as MATLAB by analysing a basic ode45 model as well as a pdepe model. 
<STRONG>Software</STRONG>
The team created a template Blog website and Twitter account. To provide a graphical representation of the project, existing software modelling options were investigated to ascertain whether an existing tool could provide suitable functionality or if a bespoke solution would be preferable. Work commenced on an Android framework for a suite of mobile utilities that will be developed over the course of the project.
            </DIV></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week Two</H3><DIV class="boxInner rounded centre" id="w2"><STRONG>Biology</STRONG>
The second week was when the project really began. The first task was to amplify the 13 type VI secretion system genes by PCR. TssA, TssE, TssF, TssM, Hcp and VgrG were all cloned individually from the prepared S. typhimurium chromosomal DNA. ClpV, VipA, VipB and TssJ, TssK (referred to hereafter at ClpV and TssJK respectively) were cloned as the original operons. Following PCR, results were analysed on a gel and showed that every gene was successfully amplified. The genes were then PCR purified and digested either BglII- HindIII or BclI- HindIII (TssM). Hcp (needle) and VgrG (needle tip) genes were cloned as fuse genes, i.e. they were cloned without a stop codon to allow the addition of another gene.  These fuse genes were digested BglII- EcoRI. Digests were strata-cleaned, ligated into pUNI-PROM and transformed into DH5α cells.
Transformed colonies were mini-prepped, digested EcoRI-HindIII to remove the insert and products were run on a gel. Clones TssA, TssE, TssJK, Hcp and Hcp Fuse appeared correct and were sent for sequencing. TssF, VgrG, VgrG fuse and ClpV failed to show an insert on the gel so were rechecked using colony PCR.
<STRONG>Maths</STRONG>
We continued to work on our interacting species model by including specific parameters, such as removal rates due to the natural processes in the colon, competition terms and the death rate of C. difficile due to the engineered type VI secretion system in E. coli.
<STRONG>Software</STRONG>
Testing of a promising software modelling tool proved disappointing due to its inability to represent large cell number interactions. After consideration, the decision was made to begin development of a bespoke C++ application that could handle realistically large colonies of cells, whilst integrating the mathematical modelling aspects into the behaviour. The aim is to provide a cross-platform, cellular automata model of the project features top provide a graphical means to explore the project’s biological dynamics. Work also began on developing the team’s official Wikipedia website.
           </DIV></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week Three</H3><DIV class="boxInner rounded centre" id="w3"><STRONG>Biology</STRONG>
From this week onwards, the Biology tasks were split between the four members of the lab team.<U>Combinatorial clone</U>
TssA was the first gene to be fully sequenced and as a result was chosen as the first gene in the combinatorial clone. The plasmid containing TssA was digested BamHI-HindIII so that the next gene could be added by suicide ligation at the Bam site. Following digestion, the plasmid was ran on a gel and then extracted. Ligations were performed adding TssJK and ClpV and transformations were completed using DH5α cells. Plates were left overnight and then liquid cultures were set up. These were digested EcoRI- HindIII to remove the insert and ran on a gel. TssA+ClpV was sent for sequencing using a primer designed from the latter part of TssA. Once this had been checked, TssA+ClpV was digested BamHI- HindIII and TssJK was added. 
<U>Site directed mutagenesis</U>
The type VI secretion system genes TssL and TssG required SDM. This was due to the presence of an EcoRI site in the TssL gene and both EcoRI and PstI sites within the TssG sequence. These genes were amplified and digested HindIII only. Plasmid pSB1C3 was prepared and also digested HindIII. TssL and TssG were ligated into pSB1C3 and transformed into DH5α cells. However, no inserts were found in the initial minipreps prepared and so patch plates were made for both genes. 
<U>Fuse genes</U>
mCherry and endolysin genes were PCR’d and  digested XbaI- HindIII ready for ligation to Hcp Fuse and VgrG Fuse genes.
<STRONG>Maths</STRONG>
We analysed our model using MATLAB to create graphs illustrating how the populations change over time. An online phase plane application was used in order to create phase plane diagrams showing the steady states of the ODE system. We first considered the model excluding the removal rates to gain an idea of how the species may interact in laboratory conditions. Once satisfied with our initial results, we then included the removal rates to encompass the real world biological behaviour.
<STRONG>Software</STRONG>
Development of the cellular automata tool continued, and now provides generation of randomly placed cells as well as pseudo-Brownian motion cell behaviour. Current testing proved to handle 250’000 interacting cells. A text-book implementation of the MVC (model-view-controller) architecture has been employed, though bottlenecks have resulted. Code has been profiled using Valgrind &amp; Callgrind to identify areas for code refactoring.
           </DIV></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week Four</H3><DIV class="boxInner rounded centre" id="w4"><STRONG>Biology</STRONG><U>Combinatorial clone</U>
TssA+ClpV+TssJK was digested Eco Hind and sent for sequencing. The plasmid was then digested (BamHI- HindIII) and TssE was added by ligation and transformation. When this was successful, the procedure was repeated and TssA+ClpV+TssJK+TssE was sent for sequencing. 
<U>Site directed mutagenesis</U>
TssL patch PCR was successful as an insert was found so this was prepared and sent for sequencing. TssG still had not worked so the PCR was repeated and the product digested as before. Also screened some more patches in case an insert had been missed but these were unsuccessful.
<U>Fuse genes</U>
Western blots were carried out using VgrG, VgrG fuse and Hcp fuse cell and supernatant samples. mCherry and endolysin PCR was repeated and ligations to VgrG and Hcp Fuse were carried out and transformed with JM1061 cells. Hcp fused with endolysin was successful.
<STRONG>Maths</STRONG>
To create a more complex and realistic model, we decided to use partial differential equations (PDEs). PDEs can be used to illustrate not only how the population of the bacteria change in time but also in space, which enabled us to model real life situations, where the C. diff would be adhered to the walls of the colon. We began by analysing these models on MATLAB and COMSOL using basic geometry as well as including diffusion terms.
<STRONG>Software</STRONG>
Analysis of the cellular automata program has identified a bottleneck that results from cell iteration being executed by both the model and view. The code was therefore refactored to exploit the graphics surface as a partial data structure, which relieves the bottleneck by updating the model and rendering new cell positions in a single iteration. This refactoring has increased cell handling capability to 300’000. Code to emulate physics behaviour has begun, and flow modelling has been integrated into the code.
           </DIV></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week Five</H3><DIV class="boxInner rounded centre" id="w5"><STRONG>Biology</STRONG><U>Combinatorial clone</U>
TssA+ClpV+TssJK+TssE was digested and ligated with TssF. Transformation was successful and the plasmid was sent for sequencing. Sequencing results showed a problem with TssF so this ligation was repeated. Following 2 further unsuccessful attempts a successful transformation was finally produced.
<U>Site directed mutagenesis</U>
TssL sequencing came back showing that it was correct and so SDM was begun on this gene, following the quickchange procedure. Retried PCR on TssG but this failed twice and so went back to check the previous digests of this gene and set up new ligations, which were transformed into DH5α cells. TssL SDM product was sent for sequencing, as after digesting some of the minipreps and running on a gel, it looked as though the procedure had been successful. 
<U>Fuse genes</U>
More western blots were carried out with the VgrG and Hcp genes in S. typhimurium and Hcp-endolysin in E. coli. VgrG fused with endolysin failed to work and so more ligations were set up.
<STRONG>Maths</STRONG>
Continuing to use COMSOL, we modified the geometry of the model by adding different domains. We allocated certain initial conditions to specific domains so that the model would coincide with the positions of the bacteria in the colon.
<STRONG>Software</STRONG>
The Android application was upgraded to provide a dilutions calculator which can run simple or serial dilution calculations. The cellular automata program was extended to encompass further physics attributes - nutrient levels (consumption and supply), cellular mitosis and tissue and mucus effects.
           </DIV></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week Six</H3><DIV class="boxInner rounded centre" id="w6"><STRONG>Biology</STRONG><U>Combinatorial clone</U>
The plasmid containing TssA+ClpV+TssJK+TssE+TssF was mini-prepped and sent for sequencing. It was then digested and ligated with TssM. TssM failed to work and so the original PCR and digestion of this gene was repeated, as it had shown quite low yield from the start.
<U>Site directed mutagenesis</U>
TssL sequencing confirmed that the SDM had been successful and so this was now ready to be cloned in with the other genes in the combinatorial clone. TssG sequencing still showed the gene to be wrong and so gel purified the digest that had been used for ligations last week again and then set up new ligations which were transformed into DH5α cells. This did not work and so decided to try a different digest of the gene (Bgl II and Hind III) and clone it into a different vector (rather than the pSB1C3). Ligations were therefore set up using PUT18 and T7KS vectors. Transformations did not work correctly though and so set up some patch plates, to screen colonies using PCR. 
<U>Fuse genes</U>
VgrG-endolysin, VgrG-mCherry and Hcp-mCherry all successfully ligated and sent for sequencing.
<STRONG>Maths</STRONG>
We altered the previously mentioned models, by using functions to set the initial conditions instead of specific domains. We also started to model the interaction between C. diff and endolysin, in order to determine the amount of endolysin required to kill one C. diff bacterium.
<STRONG>Software</STRONG>
The Android utility, now named ‘Lazy Scientist II’ was analysed for compatibility issues, and the code was refactored to allow the code to execute safely on a wider range of devices. Work began on an Android based game, with a preliminary MVC framework established ready for game logic to be added. Current application has placeholders for game logic and model aspects, and has a touch-screen handling routine that provides co-ordinates of screen presses.
           </DIV></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week Seven</H3><DIV class="boxInner rounded centre" id="w7"><STRONG>Biology</STRONG><U>Combinatorial clone</U>
Repeated ligation of TssA+ClpV+TssJK+TssE with TssM was unsuccessful. Ligations were also carried out with TssL. All successful transformations were sent to sequencing.
<U>Site directed mutagenesis</U>
TssG was PCR'd again and digested with BamHI and HindIII. This was then ligated into pUNI prom and T7KS and transformed into JM110 cells. These grew well and so minipreps were made, which were digested to check for inserts. The gene was found to have inserted this time and so SDM was carried out to remove both the EcoRI and PstI sites. TssL was stocked. 
<U>Biosensor</U>
The Ttr-RS genes were PCR'd (twice- first time was unsuccessful so changed to 100µl PCR) and digested as an EcoRI-BamHI fragment. This was ligated into pUNI PROM which had been digested in the same manner and transformed into JM110 cells. The plates grew well and so liquid cultures and minipreps were prepared. 
<U>Fuse genes</U>
S. typhimurium cells transformed with the plasmid containing VgrG were grown under various growth conditions: Glucose, Glucose formate, Glycerol DMSO, Glycerol TMAO, Glycerol nitrate and Glycerol tetrathionate. Cell and supernatant samples were western blotted. Fuse genes were PCR’d again and digested SpeI HindIII. These were then ligates to VgrG and Hcp fuse plasmids and JM110 cells were transformed. 
<STRONG>Maths</STRONG>
Both mathematicians were on holiday this week and so no progress was made on this aspect of the project.
<STRONG>Software</STRONG>
The Android game was further developed, and an existing concept was chosen and modified as the theme for the game. A game state management system was developed to allow for different rendering intents to be handled (splash screen, level up screen etc.). An audio playback routine was also developed and included within the game engine to provide sound effects during game play. The cellular automata program was enhanced by creating a file parser to initialise the initial emulation state and environment parameters.
           </DIV></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week Eight</H3><DIV class="boxInner rounded centre" id="w8"><STRONG>Biology</STRONG><U>Combinatorial clone</U>
Sequencing results showed that both ligations had failed. TssM and TssL were ligated into empty pUNI PROM to check they could work. The team think the unsuccessful ligations are because the combinatorial clone is now too big. It contains 7 genes with 5 still to be added. Ligation with Hcp was carried out; this gene is much smaller than TssM and TssL. This ligation was successful and results were sent for sequencing. Ligation of TssM and TssL with pUNI was both successful but digestion with EcoRI- HindIII showed no TssM insert. Colony PCR was carried out on 28 colony patches to check for the insert.
<U>Site directed mutagenesis</U>
TssG mutagenesis had not worked as had attempted to do both sites at once so instead set-up reactions to do the two sites separately. Quickchange was carried out to remove the EcoRI site first. The transformation after the SDM worked well and so liquid cultures were made for miniprepping. Some of the miniprepped DNA was digested with EcoRI and ran on a gel. This looked as though the EcoRI site had been removed and so the gene was sent for sequencing. Quickchange was also carried out on the samples that looked as though they no longer had an EcoRI site, to remove the PstI site.
<U>Biosensor</U>
Tried cloning the gene into different vectors but still no success and so planned to try other vectors next week.
<STRONG>Maths</STRONG>
Continued to work on the endolysin and C. diff model, as well as researching optical density values to calculate precise numbers of molecules of endolysin that are needed to lyse the bacteria. We added non-periodic flow to the COMSOL models in both domains.
<STRONG>Software</STRONG>
The Android-based game, now called ‘Lysis To Kill’ reached testing stage, and by the end of the week was deployed on two different devices for user testing. No critical issues were indentified, though several aesthetic aspects were identified and addressed. Initial work began on a desktop game concept to establish feasibility of production within timescale.
           </DIV></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week Nine</H3><DIV class="boxInner rounded centre" id="w9"><STRONG>Biology</STRONG><U>Combinational cloning</U>
Eventually, colony PCR was carried out on over 50 TssM ligation/transformation colonies and 5 were identified as being likely to contain the correct insert. These were sent for sequencing and the most successful was chosen to be the starting point for the addition of the final genes. Due to the size of TssM, an additional 6 primers were ordered to allow the entire gene to be sequenced.
<U>Site directed mutagenesis</U>
The TssG samples were prepared for sequencing after the PstI mutagenesis was complete. The sequencing results showed that PstI had finally been removed but also showed that the EcoRI site was still present and so another round of Quikchange was carried out to remove this site. The plasmid was transformed after DpnI digestion, the cells prepared and the sample sent for sequencing.
<STRONG>Maths</STRONG>
We continued to focus on models with non-periodic flow but then added periodic flow to the COMSOL models in both domains.
</DIV></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week Ten</H3><DIV class="boxInner rounded centre" id="w10"><STRONG>Biology</STRONG><U>Combinational cloning</U>
Following successful SDM, TssG and TssL could be added to the plasmid containing TssM. Finally, VgrG was added. The first plasmid will end with VgrG and a second plasmid will end with VgrG-endolysin on the end. Finally, Hcp was also cloned onto the end of the first plasmid to complete the construct.
<U>Site directed mutagenesis</U>
TssG sequencing results showed that both sites were now removed. The gene was therefore ready to be added to the combinatoral clone. 
<U>Radiolabelling</U>
The combinatorial clones were transformed into E. coli K38 cells and an overnight subculture was prepped for radiolabelling through an hours growth in 1x M9 salts lacking in both cysteine and methionine. The cells were subsequently heat shocked at 42°C allowing for the synthesis of T7 controlled genes and the antibiotic Rifampicin was added, blocking the action of RNA polymerase. Following two further growth stages at 42°C (10mins) and 30°C (20mins), 1ml of cells were radiolabelled with 1µl 35S-methionine for 15mins at 30°C. The subsequent samples were run on a protein gel and the outcome was examined through acetate and methanol.
<STRONG>Maths</STRONG>
The COMSOL models that have been discussed previously were re-run using different parameter values. We then took images and movies from all the models that were considered important. 
</DIV></DIV><DIV class="boxOuter rounded full"><H3 class="black">Week Eleven</H3><DIV class="boxInner rounded centre" id="w9"><STRONG>Biology</STRONG>
BL21 (DE3) cells were transformed with the now completed pUNI-PROM constructs. Induced and non-induced protein samples were run on SDS-PAGE gels. New bands visible in the overexpressed lanes were validated as VipA, VipB and TssM by Tryptic mass fingerprinting. 
The pACY-Duet2 plasmid was exploited to create two additional plasmids, the first containing genes TssM, TssL, TssG, VgrG-endo and Hcp whilst the second plasmid had VgrG in replace of VgrG-endo.





        </DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Dundee/Notebook">http://2012.igem.org/Team:Dundee/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Dundee/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Dundee/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Dundee/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Dundee/Notebook&amp;oldid=220447" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>