<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_XMU-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:XMU-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:XMU-China/Results - 2018.igem.org</TITLE></P><DIV id="container"><HEADER><DIV class="wrapper cf"><NAV id="main-nav"><UL class="first-nav"><LI class="Project"><A href="#" target="_blank">Project</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Parts">Parts</A></LI></UL><UL class="second-nav"><LI class="Hardware"><A href="#">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware">Overview</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware#Microfluidic_Chips">Microfluidic Chips</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware#Fluorescence_Detection">Fluorescence Detection</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware#Raspberry_Pi">Raspberry Pi</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware#Application">Application</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Applied_Design">Product Design</A></LI></UL><LI class="Model"><A href="#">Model</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Model#Summary">Summary</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Model#Thermodynamic_model">Thermodynamic Model</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Model#Fluid_dynamics_model">Fluid dynamics Model</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Model#Molecular_docking_model">Molecular Docking Model</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Model#The_dynamic_model">Derivation of Rate Equation</A></LI><LI class="Human_Practice"><A href="#">Social Works</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Human_Practices">Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Public_Engagement">Engagement</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Entrepreneurship">Entrepreneurship</A></LI><LI class="Other_Works"><A href="#">Other Works</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Measurement">Measurement</A></LI><LI class="Notebook"><A href="#">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Engineering">Engineering</A></LI><LI class="Team"><A href="#">Team</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Judging">Judging</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/After_iGEM">After iGEM</A></LI></NAV></DIV></HEADER></DIV><DIV class="header"><DIV class="nav"><DIV id="Team"><DIV class="nav-word">Team</DIV><UL><LI><A href="https://2018.igem.org/Team:XMU-China/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Judging">Judging</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/After_iGEM">After iGEM</A></LI></UL></DIV><DIV id="Notebook"><DIV class="nav-word">Notebook</DIV><UL><LI><A href="https://2018.igem.org/Team:XMU-China/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Engineering">Engineering</A></LI></UL></DIV><DIV id="Other_Works"><DIV class="nav-word">Other Works</DIV><UL><LI><A href="https://2018.igem.org/Team:XMU-China/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Measurement">Measurement</A></LI></UL></DIV><DIV id="Human_Practice"><DIV class="nav-word">Social Works</DIV><UL><LI><A href="https://2018.igem.org/Team:XMU-China/Human_Practices">Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Public_Engagement">Engagement</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Entrepreneurship">Entrepreneurship</A></LI></UL></DIV><DIV id="Model"><DIV class="nav-word">Model</DIV><UL><LI><A href="https://2018.igem.org/Team:XMU-China/Model">Summary</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Model#Thermodynamic_model">Thermodynamic Model</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Model#Fluid_dynamics_model">Fluid dynamics Model</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Model#Molecular_docking_model">Molecular Docking Model</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Model#The_dynamic_model">Derivation of Rate Equation</A></LI></UL></DIV><DIV id="Hardwork"><DIV class="nav-word">Hardware</DIV><UL><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware">Overview</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware#Microfluidic_Chips">Microfluidic Chips</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware#Fluorescence_Detection">Fluorescence Detection</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware#Raspberry_Pi">Raspberry Pi</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware#Application">Application</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Applied_Design">Product Design</A></LI></UL></DIV><DIV id="Project"><DIV class="nav-word">Project</DIV><UL><LI><A href="https://2018.igem.org/Team:XMU-China/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Parts">Parts</A></LI></UL></DIV></DIV></DIV><DIV class="description_banner"><DIV class="word">Results</DIV></DIV><NAV class="Quick-navigation"><DIV class="Quick-navigation_word"><A href="#ABCDsystem" id="Quick_A">ABCD system</A><A href="#OMVs" id="Quick_B">OMVs</A><A href="#Supporting" id="Quick_C">Supporting</A></DIV></NAV><DIV class="main"><SECTION id="ABCDsystem" class="js-scroll-step"><DIV class="headline">
                    The result of ABCD system
                </DIV><H1>What work we've done:</H1><P>Verified the concept of our design for three modules respectively.</P><P>Verified the activity of EpCAM.</P><P>Verified the successful combination between SYL3C aptamer and EpCAM.</P><P>Verified that SYL3C aptamer could combine with 10nt complementary chain successfully.</P><P>Verified the feasibility of &quot;competition method&quot; successfully.</P><P>Verified that As Cas12a could be activated.</P><P>Verified the trans-cleavage activity.</P><P>Transferred molecular signal to fluorescence signal with DNaseAlert.</P><P>Verified the transfer from protein signal to the fluorescent one successfully.</P><H1>The work hasn’t finished yet:</H1><P>Quantitative analysis of the effect of &quot;competition method&quot; and &quot;As Cas12a system&quot; respectively.</P><P>Verified the detection limit of &quot;competition method&quot;, &quot;As Cas12a system&quot;, and then our whole system.</P><P>Other designs to transfer signals for reporting devices.</P><H1>Activity of EpCAM (Flow Cytometry)</H1><P> We got EpCAM directly from ACROBiosystems<SUP>®</SUP>. In order to prove that the protein was still bio-active, we chose antibody of EpCAM with fluorescence and flow cytometry to measure it. Furnished by the research group laboratory, the fluorescent antibody was proved effective through previous experiments.</P><P>We bought EpCAM with His-tag. It's widely known that His-tag could chelate with nickel ions, so we incubated EpCAM protein with nickel beads to bind them together. Adding the antibody and then incubating, we measured them with flow cytometry. At the same time, we started two other control group experiments, with the result shown below. Get more information of our experiment, <A class="click_here" href="https://2018.igem.org/Team:XMU-China/Experiments">click here.</A></P><DIV class="table_container"><TABLE class="table"><TBODY><TR><TH>Name</TH><TH>Events</TH><TH>% Parent</TH><TH>% Grandparent</TH><TH>% Total</TH><TH>PE-A Mean</TH><TH>PE-A Geo Mean</TH></TR><TR><TD>ni-beads:P1</TD><TD>5,087</TD><TD>94.52</TD><TD>***</TD><TD>94.52</TD><TD>108</TD><TD>82</TD></TR><TR><TD>ni-beads+antiepcam:P1</TD><TD>5,208</TD><TD>91.45</TD><TD>***</TD><TD>91.45</TD><TD>2,880</TD><TD>2,626</TD></TR><TR><TD>EpCAM+Anti-EpCAM:P1</TD><TD>4,238</TD><TD>93.53</TD><TD>***</TD><TD>93.53</TD><TD>14,816</TD><TD>14,211</TD></TR></TBODY></TABLE></DIV><P class="Figure_word"><STRONG>Figure 1.</STRONG> The result of detecting the binding of EpCAM with fluorescent antibody, with flow cytometry. 
                            a) Black line: pure nickel beads; b) Blue line: nickel beads + fluorescent antibody (measure the non-specific adsorption between beads and antibody); c) Red line: nickel beads-EpCAM complex + fluorescent antibody.
                        </P><P> From the graph and data above, we could find out that the non-specific adsorption between nickel beads and fluorescent antibody does exist, but in the presence of EpCAM, the fluorescence intensity increased seven times compared with the original one, so we could prove that our bio-active EpCAM protein successfully combined with fluorescent antibody.</P><H1>Combination between SYL3C and EpCAM (Fluorescence Polarization)</H1><P> Although EpCAM and aptamer SYL3C are directly quoted in the literature <SUP>[1]</SUP>, we believed that we still needed to verify this step and that we could ensure the validity of our material.</P><P>We struggled with this step for a long period, and we suspected that our failure was caused by impure materials. We used 30 KDa ultrafiltration tube--through which unbound fluorescent aptamer could pass, but the aptamer combined with protein could not. We also used flow cytometry to directly detect whether there were binding fluorescent aptamer on EpCAM, without satisfying results in the end. </P><P>
                        Fortunately, we successfully validated this step with the fluorescence polarization method after replacing materials, with the proposal from Prof. Zhi Zhu. We incubated two groups, namely pure fluorescent aptamer as well as fluorescent aptamer with EpCAM without light at the same time, then adding binding buffer to the total volume of 200 μL. After incubation, we added 3 mL binding buffer for fluorescence polarization. Get more information of our experiment, <A class="click_here" href="https://2018.igem.org/Team:XMU-China/Experiments">click here.</A></P><P class="Figure_word"><STRONG>Figure 2.1</STRONG> Polarizing fluorescence intensity of fluorescent aptamer.</P><P class="Figure_word"><STRONG>Figure 2.2</STRONG> Polarizing fluorescence intensity of fluorescent aptamer with EpCAM, after incubation.</P><P> According to the principle of fluorescence polarization, the FP value would depend directly on the effective molecular size of the fluorescent group at fixed temperature and viscosity of solution. For small molecules (fluorescent aptamer) with fast Brownian rotation in solution, the FP value was lower; while for larger molecules (fluorescent aptamer combined with EpCAM), the FP value was higher. <SUP>[2]</SUP></P><P> The FP value could be calculated by the following formula: </P><P>Below are four maximum fluorescence values for calculating FP values:</P><P>Later, we could calculate whether the FP value of fluorescent aptamer increased with EpCAM incubation according to the principle. If the FP value increased, it showed that fluorescent aptamer did bind to EpCAM. We calculated the difference herein:</P><P>With an increase of 0.142, the difference was rather obvious, which demonstrated that the fluorescent aptamer did combine with EpCAM.</P><H1>Combination between SYL3C and the &quot;complementary strand&quot; (DNA Agarose Gel Electrophoresis)</H1><P> We've been struggling for some time due to one problem about how to prove that our aptamer and the complementary chains were combined, which would be paramount for our following experiments.</P><P> We tried SYBR Green I, a fluorescent dye that could be embedded in a double helix of DNA and thus significantly enhance fluorescence intensity, since it could distinguish ssDNA from dsDNA obviously. However, there's no significant difference according to our results. We speculated that the sequence might be not long enough, coupled with the formation of secondary structure of aptamers, leading to no difference in fluorescence intensity.</P><P> Finally, we exploited the conventional gel electrophoresis method to prove this block, and then gel electrophoresis was performed immediately after grouping and denaturing annealing. Because the sequence was too short, we did not have the appropriate DNA marker to use. Get more information of our experiment, <A class="click_here" href="https://2018.igem.org/Team:XMU-China/Experiments">click here.</A></P><P class="Figure_word"><STRONG>Figure 3.</STRONG> Gel image of aptamer combined with complementary chain.
                                            a) W1: aptamer; b) W2: 24nt complementary chain with a 9nt complementary part; c) W3: 24nt complementary chain with a 10nt complementary part; d) W4: Aptamer + 24nt complementary chain with a 9nt complementary part; e) W5: Aptamer + 24nt complementary chain with a 10nt complementary part.
                                        </P><P>We could find that there were three distinct bands between the aptamer + complementary strands with a 10nt complementary part (W5), while 9nt did not. The lane 1 and 2 acted as markers, so we could clearly identify the positions of the aptamers and complementary strands in the three bands. The last band, obviously, was the result of an increase in molecular weight, which meant that the aptamer and the complementary strand combined together successfully.</P><P>Therefore, we have proved that the aptamer could combine with its complementary strand successfully.</P><H1>The competition (Fluorescence Spectrometer)</H1><P> We used the competition method to transfer molecular signal, because we considered that the competition method was best operated automatically later, so that users did not need to have a deep understanding toward technical basis. Therefore, it's quite important for us to verify that EpCAM could compete successfully, from a theoretical prospective to support our hardware and the activation of Cas12a as well.</P><P> Below are the exact procedures of our experimental group. First, we successfully hatched the streptavidin beads, biotin-modified aptamers and fluorescent complementary strands to form the complexes, then fixed the beads with magnets and discarded the supernatant, which would still contain fluorescent complementary strands that did not form complexes. After that, we incubated the complexes with EpCAM to achieve the process of competition.</P><P> Additionally, considering the non-specific adsorption, we set up several control groups, and the variables amongst them were replaced by buffer. After competition, we fixed the magnetic beads with magnet, then extracted the supernatant and added 3 mL incubation buffer in tandem. Below is the plot about the fluorescence intensity detected in different groups. Get more information of our experiment, <A class="click_here" href="https://2018.igem.org/Team:XMU-China/Experiments">click here.</A></P><P class="Figure_word"><STRONG>Figure 4.</STRONG> Fluorescence intensity images of competition results. a) Black line: incubation buffer; b) Red line: magnetic beads-aptamer + incubation buffer; c) Blue line: magnetic beads-aptamer-complementary strand complex + incubation buffer; d) Green line: magnetic beads-aptamer-complementary strand complex + EpCAM + incubation buffer.
                                        </P><P>Comparing the four groups, we can conclude that the non-specific adsorption of complementary strands on the magnetic beads does exist, and the magnetic beads-aptamer complex, like the buffer solution, had almost no fluorescence intensity. Significantly, the fluorescence intensity of the competition group with EpCAM was about 2.5 times larger than that of the non-specific adsorption group, which demonstrated that the fluorescence complementary strand of the competition group with EpCAM was successful, and so was our competition method.</P><H1>Trans-cleavage activity of Cas12a (Microplate Reader)</H1><P>It's reported that Cas12a could be activated by ssDNA and has trans-cleavage activity this year, so we hope to use this function in our system to amplify the signal. <SUP>[3]</SUP></P><P>We operated the experiment according to the instructions from IDT. We incubated the Cas12a protein with crRNA to form a complex and then added our ssDNA to be recognized. Then, we used the commercial DNaseAlert<SUP>TM</SUP> Kit to explore its trans-cleavage activity. We drawed the fluorescence curve over time using a microplate reader, and measured the fluorescence intensity from 30 minutes to 110 minutes.</P><P>In order to control variables, we also set up a negative (DNaseAlert) and a positive (DNase I + DNaseAlert) control group. Get more information of our experiment, <A class="click_here" href="https://2018.igem.org/Team:XMU-China/Experiments">click here.</A></P><P class="Figure_word"><STRONG>Figure 5.</STRONG> Fluorescence time curve image of activation effect of Cas12a. a) Black line: negative group, DNaseAlert only; b) Red line: positive group, DNase I + DNaseAlert; c) Blue line: experimental group, activated Cas12a + DNaseAlert;
                                        </P><P>We fitted three sets of data points with Gaussian fitting curve in Figure 5. From the fitting results, we could see that the negative group and the positive group were declining, while the Cas12a protein group increased at first and then decreased. This is in line with our expectations, and the reason for the decline was, we suspected, the quenching caused by continuous measurements and the weakening of the fluorescence itself as the experiment proceeds. However, it's enough to prove that Cas12a could meet the expected result of amplifying signals.</P><H1>Signal transformation by DNaseAlert<SUP>TM</SUP> Kit (Fluorescence Spectrometer)</H1><P> Although we have used DNaseAlert to detect Cas12a, as the last of the three modules we designed, we needed to prove that DNaseAlert was indeed cut and would emit fluorescence to our expectations, so that it could be used to transfer our signals.</P><P> Here, we chose DNase I to verify directly. In the presence or absence of DNase I, we measured fluorescence intensity in the DNase alert tube. Considering the experiences we got previously that long-term measurement in the microplate reader would lead to reduction in fluorescence intensity, resulting in experimental error, we chose fluorescence spectrometer instead.</P><P> We set up the experimental group (with DNase I) and the control group (without DNase I), with the time gradient of 3, 15, 30 minutes. All groups started at the same time. We didn't operate continuous measurement, to avoid excitation that may cause partial fluorescence quenching. After incubation, 3 mL TE buffer was added, and then detect the fluorescence intensity. Get more information of our experiment, <A class="click_here" href="https://2018.igem.org/Team:XMU-China/Experiments">click here.</A></P><P class="Figure_word"><STRONG>Figure 6.</STRONG> Fluorescence intensity curve in 3rd minutes, with or without DNase I.
                                        </P><P class="Figure_word"><STRONG>Figure 7.</STRONG> Fluorescence intensity curve in 15th minutes, with or without DNase I.
                                        </P><P class="Figure_word"><STRONG>Figure 8.</STRONG> Fluorescence intensity curve in 30th minutes, with or without DNase I.
                                        </P><P>From the Figure 6-8, we could find that cutting efficiency of DNase I was very high, almost all of the DNaseAlert substrate has been cut in 3 minutes, fluorescence intensity in 15 and 30 minutes and 3 minutes almost kept the same level. As we expected, DNaseAlert could indeed be cleaved and then re-released as a fluorescent group. In other words, DNaseAlert could transfer the molecular signal to the fluorescent signal actually.</P><H1>Integrated experiment</H1><P>After making each part work effectively, we accomplished the integrated experiment finally, transferring the protein signal to fluorescent one.</P><P>Similarly, following procedures mentioned in the &quot;competition method&quot; part, we successfully hatched the streptavidin beads, biotin-modified aptamers and fluorescent complementary strands to form the complexes, then fixed the beads with magnets and discarded the supernatant, which would still contain fluorescent complementary strands that did not form complexes. After that, we incubated the complex with EpCAM to achieve the process of competition. After competition, we fixed the magnetic beads with magnet, adding the supernatant extracted into Cas12a-crRNA complex incubated previously to activate Cas12a protein. Finally, we added it into DNaseAlert, supplementing 3 mL TE buffer after 30 minutes to measure the fluorescent intensity. Below is the corresponding plot we've obtained, indicating that we made the integrated experiment successfully.</P><P class="Figure_word"><STRONG>Figure 9.</STRONG> Integrated experiment.
                                        </P><H1>References</H1><P class="reference">[1] Yanling Song, Zhi Zhu, Yuan An, Weiting Zhang, Huimin Zhang, Dan Liu, Chundong Yu, Wei Duan, Chaoyong James Yang. Selection of DNA Aptamers against Epithelial Cell Adhesion Molecule for Cancer Cell Imaging and Circulating Tumor Cell Capture. <I>Anal. Chem</I>. <STRONG>2013</STRONG>, 85: 4141-4149. 
                                        [2] DS Smith, SA Eremin, Fluorescence polarization immunoassays and related methods for simple, high-throughput screening of small molecules. <I>Analytical and bioanalytical chemistry</I>. <STRONG>2008</STRONG>, 391(5): 1499. 
                                        [3] Chen JS, Ma E, Harrington LB, Da Costa M, Tian X, Palefsky JM, Doudna JA. CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity. <I>Science</I>. <STRONG>2018</STRONG>, 360(6387): seaar6245.
                                    </P></SECTION><SECTION id="OMVs" class="js-scroll-step"><DIV class="headline">
                    The result of OMVs
                </DIV><H1>The growth curve of IPTG-induced and Arabinose-induced <I>E. coli</I> BL21</H1><P>To ensure when we should add the inducer, we measured the growth curve of BBa_K2623026 (IPTG-induced) and BBa_K2623027 (Arabinose-induced). According to figure 1a, we found that after incubation for 5 h, the OD600 of BBa_K2623026 (IPTG-induced) reached to about 0.6, which is suitable for adding IPTG to a final concentration at 0.5 mM. Figure 1b shows that after incubation for 5 h, the OD600 of BBa_K2623027 (Arabinose-induced) reached to about 0.6, which is suitable for adding Arabinose to a 0.2% final concentration.
                </P><P class="Figure_word"><STRONG>Figure 1.</STRONG> Growth curve of (a) IPTG-induced and (b) Arabinose-induced <I>E. coli</I> BL21.</P><H1>OmpA-ST could be transported to OMVs</H1><P>OmpA-SpyTag (OmpA-ST) is transported to bacterial outer membrane and therefore OMVs. However, <I>E. coli</I> could produce natural OmpA and then transport it to its outer membrane and OMVs as well. It's necessary to know how much OmpA-ST is transported to OMVs or what the ratio of the OMVs-ST is in total OMVs. To evaluate the efficiency of its transport to outer membrane and OMVs, we fused wtGFP and our OmpA-ST (BBa_K2623022 and BBa_K2623024) together in order to construct OmpA-ST-GFP (BBa_K2623021 and BBa_K2623023) (Figure 2).</P><P class="Figure_word"><STRONG>Figure 2.</STRONG> The spectrogram of wtGFP <SUP>[1]</SUP>. </P><P>We cultured bacteria transfected with these four parts respectively in 10 mL OMVs-free LB broth culture and extracted the OMVs according to our protocols, in which two forms of non-GFP OMVs were set as negative controls. Transmission electron microscopy (TEM) pictures identified what we extracted were exactly OMVs with membrane structure and 100-nm diameter (Figure 3). </P><P class="Figure_word"><STRONG>Figure 3.</STRONG> TEM figures identify our OMVs. </P><P>
                    Building upon Yan's lab-built high-sensitivity flow cytometer <SUP>[2-3]</SUP>, we successfully detected the OMVs labelled with GFP and evaluated the efficiency of its transport to OMVs (Figure 4). Herein, we'd like to extend our heartfelt thanks toward Yan's lab due to the very kind and selfless help during the experiment.
                </P><P class="Figure_word"><STRONG>Figure 4.</STRONG> HSFCM analysis of OmpA-ST-GFP inside OMVs.
                    </P><P>Figure 4 are bivariate dot-plots of GFP green fluorescence versus side scattering (SSC) for our OMVs isolates. <I>E. coli</I> transfected with BBa_K2623021 (Figure 4a) could secret more OMVs labeled with GFP than <I>E. coli</I> transfected with BBa_K2623022 (Figure 4b). Similarly, <I>E. coli</I> transfected with BBa_K2623023 (Figure 4c) would secret more OMVs labeled with GFP than <I>E. coli</I> transfected with BBa_K2623024 (Figure 4d). Moreover, we could see <I>E. coli</I> transfected with BBa_K2623021 (Figure 4a) would transport OmpA-ST to OMVs more efficiently than <I>E. coli</I> transfected with BBa_K2623023 (Figure 4c). The remarkable difference of GFP-OMVs ratio between these two parts might account for the different inserted site of ST. SpyTag (ST) in BBa_K2623021 (Figure 4a) is inserted to N-termini of OmpA, while ST in BBa_K2623023 (Figure 4c) is inserted to C-termini of OmpA. Our HSFCM data shows that BBa_K2623021 is more efficient to transport ST-OmpA-GFP to OMVs and might be a good candidate for the following study.</P><H1>References</H1><P class="reference">
                    [1] https://www.semrock.com/ 
                    [2] Zhu S, Ma L, Wang S, <I>et al</I>. Light-Scattering Detection below the Level of Single Fluorescent Molecules for High-Resolution Characterization of Functional Nanoparticles. <I>Acs Nano</I>. <STRONG>2014</STRONG>, 8(10): 10998-11006. 
                    [3] Tian Y, Ma L, Gong M, <I>et al</I>. Protein Profiling and Sizing of Extracellular Vesicles from Colorectal Cancer Patients via Flow Cytometry[J]. <I>Acs Nano</I>. <STRONG>2018</STRONG>, 12(1): 671-680.
                </P><H1>Demonstrate the OmpA-ST inside <I>E. coli</I> BL21 and OMVs</H1><P>As our HSFCM data show, OmpA-ST-GFP (BBa_K2623021 and BBa_K2623023) transported to OMVs could be detected. In our total circuit, we would use non-GFP part (BBa_K2623022 and BBa_K2623024) to establish SpyTag/SpyCatcher (ST/SC) conjugation. For this purpose, we'd like to demonstrate the OmpA-ST (BBa_K2623022 and BBa_K2623024) inside both <I>E. coli</I> BL21 and OMVs, as well as SDS-PAGE used (Figure 5).</P><P class="Figure_word"><STRONG>Figure 5.</STRONG> Lane 1: <I>E. coli</I> BL21 transfected with BBa_K2623022. Lane 2: OMVs secreted by <I>E. coli</I> BL21 transfected with BBa_K2623022. Lane 3: <I>E. coli</I> BL21 transfected with BBa_K2623024. Lane 4: OMVs secreted by <I>E. coli</I> BL21 transfected with BBa_K2623024.
                    </P><P>As our SDS-PAGE figure shows, ideal band at about 50 kDa are shown in <I>E. coli</I> BL21 transfected with BBa_K2623022 and BBa_K2623024, respectively. However, we could see neither bands nor traces in lane 2 and lane 4 loading with OMVs samples. We suppose that our sample loadings are not enough for the LOD (limit of detection) of SDS-PAGE followed by Coomassie blue staining. Because of limited time, we couldn't repeat this experiment. Whatever, our HSFCM data have shown that OmpA-ST could be encapsulated into OMVs.</P><H1>Demonstrate the SC-L7Ae inside <I>E. coli</I> BL21</H1><P>Similarly, in our total circuit, we used non-RFP part (BBa_K2623026) to fulfill L7Ae-C/D<SUB>box</SUB> linkage. It's demonstrated that the SpyCatcher (SC)-L7Ae could be expressed inside <I>E. coli</I> BL21 firstly. Under the induction of IPTG at 0.5 mM, we could detect the expression of SC-L7Ae (Figure 6).</P><P class="Figure_word"><STRONG>Figure 6.</STRONG> SDS-PAGE figure of the SC-L7Ae. Lane 1: induced-BL21. Lane 2: uninduced-BL21. The red arrow points to the band of the SC-L7Ae.
                    </P><P>Though no IPTG was added into <I>E. coli</I> BL21 as negative control (lane 2), we could still see our band at about 70.4 kDa indicated for SC-L7Ae. However, SDS-PAGE is not an ideal approach for identify our targeted protein. In our future plans, we would use anti-SC mAb to immunoblot our SC-LA7e.</P><H1>Demonstrate the ST/SC conjugation inside OMVs</H1><P>Though we have revealed the OmpA-ST inside OMVs, we are inclined to figure out the ST/SC bioconjugation inside OMVs. Hence, we fused mRFP1 to SC-L7Ae (BBa_K2623026) to build SC-L7Ae-RFP (BBa_K2623025) (Figure 7). We linked our two non-GFP OmpA-ST parts (BBa_K2623022 and BBa_K2623024) with BBa_K2623025 and BBa_K2623026 to form four new parts:</P><P class="reference">1) BBa_K2623022 &amp; BBa_K2623025: BBa_K2623028 (RFP form) 
                    2) BBa_K2623022 &amp; BBa_K2623026: BBa_K2623029 
                    3) BBa_K2623024 &amp; BBa_K2623025: BBa_K2623030 (RFP form) 
                    4) BBa_K2623024 &amp; BBa_K2623026: BBa_K2623031
                </P><P>Under the induction of IPTG, L7Ae could be expressed with SpyCatcher and RFP modification and then be encapsulated into OMVs through ST/SC conjugation. OMVs labeled with RFP will be detected through HSFCM.</P><P class="Figure_word"><STRONG>Figure 7.</STRONG> a) Genetic circuit of SC-L7Ae-RFP <SUP>[1]</SUP>. b) The spectrogram of mRFP1.
                    </P><P class="Figure_word"><STRONG>Figure 8.</STRONG> HSFCM analysis of ST/SC bioconjugation inside OMVs.
                    </P><P>Figure 8 contains bivariate dot-plots of RFP red fluorescence versus side scattering (SSC) for our OMVs isolates. <I>E. coli</I> BL21 transfected with BBa_K2623028, BBa_K2623029, BBa_K2623030, BBa_K2623031 were incubated with OMVs-free LB culture for about 5 hours to get a 0.6-0.8 OD600 respectively, in which BBa_K2623029 and BBa_K2623031 were set as negative controls. Then IPTG was added to a final concentration at 0.5 mM and nurture the bacteria overnight. OMVs were isolated according to our protocols and then analyzed by HSFCM. It’s interesting to note that <I>E. coli</I> transfected with BBa_K2623030 (Figure 3c) could secret more RFP-OMVs than BBa_K2623028 (Figure 3a), indicating that ST/SC conjugation inside OMVs is more efficient with ST at the C-termini of OmpA. This result is inconsistent with the result shown in Figure 4, in which ST at the N-termini of OmpA (BBa_K2623028) has a higher efficiency to be encapsulated inside OMVs. We propose that GFP inserted to OmpA might interfere the transport of OmpA-ST (BBa_K2623021 and BBa_K2623023) to OMVs. </P><H1>References:</H1><P class="reference">[1] https://www.semrock.com/</P><H1>Demonstrate the L7Ae-C/D<SUB>box</SUB> conjugation inside OMVs</H1><P>Our OMVs module is designed for siRNA delivery and then for cancer treatment. To demonstrate the conjugation of the L7Ae-C/D<SUB>box</SUB> inside OMVs, we linked ST/SC part with the box part (BBa_K2623027, see Figure 9) to form our total circuit (BBa_K2623032).</P><P class="Figure_word"><STRONG>Figure 9.</STRONG> Genetic circuit of the Box part (BBa_K2623027).
                    </P><P>The box part can code siRNA target for Kras in human pancreatic ductal adenocarcinoma (PDAC) with a C/D<SUB>box</SUB> RNA structure to bind with L7Ae. </P><P>Under the induction of arabinose at a 0.2% final concentration, we isolated the OMVs according to our protocols and then stained the OMVs isolates with SYTO<SUP>TM</SUP> RNASelect<SUP>TM</SUP> (selectively stain for RNA) at 10 μM (Figure 10). The stained isolates were analyzed by HSFCM (Figure 11). </P><P class="Figure_word"><STRONG>Figure 10.</STRONG> a) Schematic illustration of our total circuit. b) The spectrogram of SYTO<SUP>TM</SUP> RNASelect<SUP>TM</SUP><SUP>[1]</SUP>.
                    </P><P class="Figure_word"><STRONG>Figure 11.</STRONG> HSFCM analysis of OMVs stained with SYTO<SUP>TM</SUP> RNASelect<SUP>TM</SUP>.
                    </P><P>Figure 11 are bivariate dot-plots of green fluorescence versus side scattering (SSC) for our OMVs isolates. <I>E. coli</I> BL21 transfected with BBa_K2623029 and BBa_K2623032 respectively were cultured in 100 mL OMVs-free LB culture for about 5 hours to get a OD600 at 0.6-0.8, in which BBa_K2623029 was set as negative control. Then Arabinose was added to a final concentration at 0.2% and after incubation for another 2 hours, IPTG was added to a final concentration at 0.5 mM and then nurtured the bacteria overnight. OMVs were isolated according to our protocols and stained with SYTO<SUP>TM</SUP> RNASelect<SUP>TM</SUP>. The stain OMVs were analyzed by HSFCM. As we expected, OMVs isolated from BBa_K2623032 could secret more OMVs containing with RNA stained positively by SYTO<SUP>TM</SUP> RNASelect<SUP>TM</SUP> (Figure 11b) than BBa_K2623029, indicating that our L7Ae and C/D<SUB>box</SUB> could fulfill their function. It’s a pity that we didn’t have cell culture experiment to test our siOMVs to silence Kras in human pancreatic ductal adenocarcinoma (PDAC). In our future plans, we'll finish our cell experiment and demonstrate the function of our siOMVs.</P><H1>References</H1><P class="reference">[1] https://www.semrock.com/</P><H1>Future Plan</H1><P>It is not practical to transfer our ideas to a fully-completed project or even a portable device just after an enriched summer. We should learn more for the sake of improving our project, making it suitable for feasible usage. Therefore, we list our future plans to develop our projects.</P><P><STRONG>Design-1</STRONG>. Though we have characterized OmpA-ST through SDS-PAGE, in many cases SDS-PAGE is not an optimal method to characterize specific protein in complex samples, like bacterial lysates. For further characterizing our OmpA-ST, we would be inclined to find monoclonal antibody (mAb) specific for SpyTag and Western Blot analysis alike. Exploiting with anti-SpyTag mAb, we supposed that the band at about 50 kDa would be stained positively.</P><P><STRONG>Design-2</STRONG>. Similar to Design-1, we would also use monoclonal antibody (mAb) specific for L7Ae and SC to validate our L7Ae-SC. </P><P><STRONG>Design-3</STRONG>. For our siRNA-C/D<SUB>box</SUB>, due to poor experimental instrument, we could only characterize the promotion of RNA level inside our engineering bacteria. The problem is that we could not ensure the presence of our siRNA target for Kras<SUP>G12D</SUP>. To characterize this part better, we'd like to use qPCR, FISH (fluorescence in situ hybridization) or low-throughput miRNA target identification method<SUP>[1]</SUP> to demonstrate our siRNA could target Kras<SUP>G12D</SUP>. Through the incubation of siOMVs (OMVs containing siRNA) and human pancreatic ductal adenocarcinoma (PDAC) cell line PANC-1, the silence effect of the siOMVs could be told.</P><P><STRONG>Design-4</STRONG>. As mentioned above, our chassis is bacterium. Some researchers have demonstrated not only bacteria but also their OMVs could be pathogenetic. Though some governments have approved the usage of OMVs vaccines, the side effects of the OMVs are still ambiguous. Extracellular vesicles (EVs), originated from eukaryocytes, which are often referred to exosomes and mircovesicles, are better choices for nucleic acid delivery for its biocompatibility and homology. We could conjugate L7Ae to the C-terminus of tetraspanin CD63 and insert a C/D<SUB>box</SUB> into the 3′-untranslated region (3′-UTR) of the gene coding for siRNA<SUP>[2]</SUP>.</P><H1>References</H1><P class="reference">[1] Michael J V, Jgt W, Mao G F, <I>et al</I>. Platelet microparticles infiltrating solid tumors transfer miRNAs that suppress tumor growth. <I>Blood</I>. <STRONG>2017</STRONG>, 130(5): blood-2016-11-751099. 
                    [2] Kojima R, Bojar D, Rizzi G, <I>et al</I>. Designer exosomes produced by implanted cells intracerebrally deliver therapeutic cargo for Parkinson’s disease treatment. <I>Nature Communications</I>. <STRONG>2018</STRONG>, 9(1): 1305.
                </P></SECTION><SECTION id="Supporting" class="js-scroll-step"><DIV class="headline">The result of Supporting</DIV><H1>Verify the expression of SAHS protein</H1><P>Firstly, we used the <A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A> (mRFP) as our report gene to make sure the circuit for expressing the SAHS protein was constructed precisely. But we did not observe the distinct color on the plate as it was so weak unless the <I>E.coli</I> was cultured in a tube and centrifuged. So we had a new test by using the <A href="http://parts.igem.org/Part:BBa_K592009">BBa_K592009</A>（blue chromoprotein）as our report gene and got a distinct color on the plate.
                </P><P class="Figure_word"><STRONG>Figure 1.</STRONG> These are the pictures our fluorescently characterized plate and bacterial pellets. The report gene on the left is BBa_K592009, and the one on the right is E1010. Some colonies on the left are blue, which are the DH5α that we successfully transferred to the designed genetic loop. The leftmost EP tube in the right photo is the control, and the other two EP tubes contain DH5α which we had successfully transferred to the designed genetic loop.
                    </P><P>And then, we transformed the plasmid to the <I>E.coli</I> BL21, which is always used to express proteins with high efficiency to verify the SAHS protein was produced successfully with a small scale. Besides, whether the SAHS protein with a signal peptide could be secreted was determined by SDS-PAGE.</P><P class="Figure_word"><STRONG>Figure 2.</STRONG> The marker is on the lest, followed by our control group (the BL21 with the empty plasmid), and the third well is the concentrated supernatant.
                    </P><P>
                    As shown in the image above, we also explored the appropriate temperature for SAHS protein expression. As we expected, protein expression was be more efficient at 30 °C. However, we did not find the SAHS protein band in the supernatant even when it was concentrated by ultrafiltration. So we speculated that cell wall obstructed the secretion of protein on the basis of 2016 Peking University's working. See more information on <A class="click_here" href="https://2016.igem.org/Team:Peking/Secretion">Come form wiki</A>.
                </P><P>Therefore, we have to design a new protein purification program. To get enough protein produces for the following experiment, the gene was cloned into the vector pET-28a with high expression lever combined with <I>E.coli</I> BL21. Besides, Two HIS-TAGs on the end of N-and C-terminal was produced and allow SAHS protein to bind with Nickel column (like Ni-NTA) for purification. Meanwhile, we purified the sample with heat water bath considering the characterization of heat stability by following the reference.</P><P class="Figure_word"><STRONG>Figure 3.</STRONG> The picture on the left is a series of temperature gradient processed protein samples. The number represents the temperature (°C). And the picture on the right is the gel map of our final purified high purity SAHS protein.
                    </P><P>So, we designed a set of temperature gradients from 70°C to 90°C, to explore the appropriate condition. What's more, we chose the 85°C for 15 min finally for large scale purification. But given that there were many other protein bands, we combined heating with Nickel column for producing high quality protein. Finally, we tested two patterns, heating-Ni-NTA and Ni-NTA-heating, and found that the later one is better.</P><P>After that, we obtained a protein sample with high concentration and purity by using this protein purification method（Figure 3）.</P><H1>Verify the role of SAHS protein in preserving biological activity</H1><P>In order to verify the preservation effect of SAHS protein, we used the lyophilization method. First, we tested the concentration of purified SAHS protein, which ranged from 0.3 g/L to 0.6 g/L. Then, we concentrated the purified protein to control the protein concentration to about 1 g/L.</P><P>We selected <I>Taq</I> enzyme as the protein preserved in our experiment, using SAHS protein as protective agent, and set a series of concentration gradients: 1 g/L, 0.5 g/L, 0.1 g/L, 0.01 g/L and 0 g/L， then stored with lyophilization. In order to verify the preservation effect of SAHS protein, we resuscited the lyophilized sample and performed PCR experiments to verify the activity of <I>Taq</I> enzyme. The activity of <I>Taq</I> enzyme can be reflected by the product of PCR.</P><P class="Figure_word"><STRONG>Figure 4.</STRONG> PCR results of <I>Taq</I> enzymes preserved at different SAHS protein concentrations.
                    </P><P>We selected different PCR systems for verification. We found that some short fragments were easy to succeed, but there was no obvious gradient. However, when we replaced them with some other fragments that are difficult to succeed or very long, there would show a very obvious gradient. We think it must be that a shorter fragments were too easy to succeed, and made the base too large. The two fragments above are pET-28a, which had a relatively obvious gradient and also had a good repeatability.</P><P class="Figure_word"><STRONG>Figure 5.</STRONG> The result of the comparison of PBS and ultra-pure water. Blank is the group that the <I>Taq</I> enzyme has not been lyophilized and without SAHS protein.
                    </P><P>In addition, we also made a comparison between  PBS and ultra-pure water. We found that regardless of the presence of TDP, the lyophilized system added with PBS could not make the correct band. On the contrary, there was only one large band which was much larger than 8000bp. We think it may be a PCR mismatch. Considering that PBS contained a large amount of salt, it was likely that these salt ions affected the binding of <I>Taq</I> enzyme to Mg<SUP>2+</SUP>.</P><P class="Figure_word"><STRONG>Figure 6.</STRONG> The ruselt of the success rate of PCR with <I>Taq</I> enzymes preserved at different SAHS protein concentrations.
                    </P><P>Then, we tried some shorter fragments for PCR, and interfered with the PCR system by reducing time extension and reducing the concentration of the template in order to explore the success rate of PCR. As the figure showed, when the concentration of SAHS protein was 0.5 g/L, the success rate of PCR was the highest. We suspected that excessive concentration of SAHS protein might affect the normal function of the <I>Taq</I> enzyme and might affect the binding of the enzyme to the substrate. This reminded us of the results of TUDelft in 2017. They also got the result of that 0.5 g/L was better than 1 g/L in some cases. Eventually, they took a concentration of 1 g/L to preserve Cas13a. They discovered that while CAHS 94205 preserved Cas13a's RNase-like activity after drying, CAHS 94205 could not preserve its specificity. This is exactly the same as ours when the concentration is 1 g/L. Therefore, we believe that it is possible that they just did not find the most suitable storage concentration, and if the concentration is properly reduced, the results may be much better.</P><H1>Conclusion</H1><P>According to the experimental results of lyophilization above, we can clearly see the preservation effect of SAHS protein on protein activity. Secondly, when the concentration is too high, SAHS protein may affect the normal function of protein. Therefore, it is necessary to find the most appropriate concentration. The optimal concentration may vary slightly among different proteins, but the preservation of biological activity by SAHS proteins is undoubtable. We believe that SAHS protein is a promising bioactive protective agent for different preservative substances, as long as the appropriate preservation system is set up.</P><P>Verify the expression of CAHS protein</P><P class="Figure_word"><STRONG>Figure 7.</STRONG> The circuit of BBa_K2623014
                    </P><P>In the circuit, the Lac I was expressed constantly, and Lac O, lac repressor was bound to Lac O. Following the CAHS gene was our reporter gene mRFP (J04650). When we added IPTG in the bacterial solution, the IPTG bound to repressor and inactivated it. So the CAHS gene and mRFP would start to express.</P><P class="Figure_word"><STRONG>Figure 8.</STRONG> the <I>E.coli</I> DH5а with BBa_K2623014 and IPTG turns red.(D+ is <I>E.coli</I> DH5а with BBa_K2623014 and IPTG, D- is <I>E.coli</I> DH5а with BBa_K2623014 without IPTG, D0 is <I>E.coli</I> DH5а without BBa_K2623014)
                    </P><P>We found that the red fluorescent protein expressed very slowly. After we added IPTG in the 4mL bacterial solution, we put the bacterial solution in 25°C shaker culture. After 12 hours, we saw the centrifugal precipitation color showed red.</P><P>We also did experiment with <I>E.coli</I> BL21, but the bacteria didn't turn red.</P><P>We cultured the <I>E.coli</I> BL21 which contained the plasmid of CAHS protein (BBa_K2623014) in the fluid medium. When the value of OD600 reached about 0.6, we took 4mL bacteria solution in the glass test tubes and added 4 μL IPTG in the tubes. And then put them in 25°C shaker culture for 5 hours. Then, we took 1mL bacterial liquid to centrifugalize. We picked the precipitation and added 100 μL DDW and 20 μL SDS loading buffer. After that, we heated and boiled it for 15 minutes. And then, it was separated by SDS-PAGE and stained with Coomassie brilliant blue for 40 minutes. Ultimately, we decolorized it and observed the results.</P><P>We find the CAHS is surely expressed in the <I>E.coli</I> BL21 after adding IPTG. Here is the result.</P><P class="Figure_word"><STRONG>Figure 9.</STRONG> the CAHS protein express in SDS-PAGE. (1, 2, 3 is all with BBa_K2623014. 1 is BL21 with IPTG and pre-heat, 2 is BL21 with IPTG, 3 is BL21 without IPTG, 4 is BL21 without the BBa_K2623014.)
                    </P><P>The picture of SDS-PAGE shows the CAHS protein is surely expressed in B+ group (the <I>E.coli</I> BL21 with BBa_K2623014 with IPTG). And CAHS protein isn't expressed in the B- group (the <I>E.coli</I> BL21 with BBa_K2623014 without IPTG) and B0 group (the <I>E.coli</I> BL21 without BBa_K2623014).</P><P>We also did experiments with <I>E.coli</I> DH5а, but the SDS-PAGE showed there was no CAHS protein. Because the <I>E.coli</I> BL21 is a kind of strain which can knock out proteases while the <I>E.coli</I> DH5а not. We guess the CAHS protein in <I>E.coli</I> DH5а is degraded. So we chose the BL21 to verify the function of CAHS.</P><H1>Verify the role of CAHS protein in preserving biological activity</H1><P>The SDS-PAGE showed the CAHS protein was really expressed in BL21. We cultured the <I>E.coli</I> BL21 bacteria which contained the plasmid BBa_K2623014. We picked the <I>E.coli</I> BL21 bacterial liquid for freeze-drying. First of all, we picked 1mL bacterial liquid to centrifugalize, added 100 μL protective agent (LB+5% Sucrose), and put them in -20°C refrigerator for 2 hours. And then, we put them in -80°C for one thorough night, and then freeze-dried them (Advantage ES-53).</P><P>After lyophilization, we added 500 μL PBS in each centrifuge tube, and put them in 37°C shaker culture for 15mins. We also added 500 μL DDW in each centrifuge tube. And then, we took 100 μL the dilute bacteria liquid, added in 900 μL ddH《》, repeated the step, and carried out gradient dilution to 104, 105, 106. Then, we picked 200 μL dilute bacteria liquid, and coated the plate with chloramphenicol.</P><P class="Figure_word"><STRONG>Figure 10.</STRONG> gradient dilution.
                    </P><P>The results showed that CAHS protein could improve the preservation effect of freeze-drying bacteria significantly!</P><P>The B+ group's CFU is 100 times higher than the B0 group, 30 times better than the B- group. So we think the CAHS is leaking expression in B- group, which has some of the protection for the bacteria in freeze-drying.</P><P>After being irradiated by UV for 1min, the B0 group was nearly fully dead, and the CFU of B- group had been decrease to a very low level, while the B+ group was higher. So we think the CAHS is can provide some of certain protection against UV.</P><P class="Figure_word"><STRONG>Figure 11.</STRONG> The CFU of BL21. (B0 is <I>E.coli</I> BL21 without BBa_K2623014, B+ is <I>E.coli</I> BL21 with BBa_K2623014 and IPTG, B- is <I>E.coli</I> BL21 with BBa_K2623014, UV means the desiccant bacterial under UV for 1min.)
                    </P><P>This result shows that the CAHS can provide production for bacteria in freeze-drying. And the CAHS can also provide CAHS for bacteria suffering from UV. We think the CAHS protein is a very potential protective agent, so we can use the CAHS protein to store plasmid and bacteria. We can even design a kit for the bacteria storage with CAHS protein.</P><P>After being irradiated by UV for 1min, the B+ group was still alive. Therefore, we think the CAHS protein can provide protection for bacteria against UV and can protect the nucleic acid in the bacteria. The water bear can survive from UV, and the bacteria can also survive from UV, so it is certain the CAHS protein works.</P><P>As for the colony recovery rate, the CAHS also showed great effect. The B+ group colony recovery rate was highest, which reached about 40%. While those of the B- group and B0 group were no more than 2%. After being irradiated by UV for 1min, the B+ group was still the highest in the three group (UB_B0, UV_B+ and UV_B-).</P><P class="Figure_word"><STRONG>Figure 12.</STRONG> The colony recovery rate. (B0 is <I>E.coli</I> BL21 without BBa_K2623014, B+ is <I>E.coli</I> BL21 with BBa_K2623014 and IPTG, B- is <I>E.coli</I> BL21 with BBa_K2623014, UV means the desiccant bacterial under UV for 1min.)
                </P><P>Here are the pictures of the culture plate. We used the disposable plastic petri dishes, and appended the sealing film on the aperture. When we started, we used the glass petri dishes, but the effect was not obvious at all. All the petri dishes had chloramphenicol, and the B0 group bacteria had transformed BBa_S0100.</P><P class="Figure_word"><STRONG>Figure 13.</STRONG> the plate of colony for counting. (B0 is <I>E.coli</I> BL21 without BBa_K2623014, B+ is <I>E.coli</I> BL21 with BBa_K2623014 and IPTG, B- is <I>E.coli</I> BL21 with BBa_K2623014, UV means the desiccant bacterial under UV for 1min. And the 104 &amp; 105 means Dilution ratio)
                </P><H1>Conclusion</H1><P>Our experiment shows the great function of CAHS protein in improving the desiccation tolerance and UV tolerance. We think that the CAHS protein is a very potential protective agent for bacteria storage. Except expressing the CAHS protein with plasmid, we can also knock the CAHS protein gene into the genome of bacteria through homologous recombination. And we believe that a new way for storing the bacteria is available.</P></SECTION></DIV><DIV class="footer"><DIV class="footer_top"><UL><LI><A href="https://2018.igem.org/Team:XMU-China">Home</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Attributions">Attributions</A></LI></UL><UL><LI><A href="https://2018.igem.org/Team:XMU-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Judging">Judging</A></LI></UL><UL><LI><A href="https://2018.igem.org/Team:XMU-China/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:XMU-China/After_iGEM">After iGEM</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>