Team:XMU-China/Design
Team:XMU-China/Design - 2018.igem.org
Project
Description
Design
Results
Demonstrate
Parts
Hardware
Overview
Microfluidic Chips
Fluorescence Detection
Raspberry Pi
Application
Software
Product Design
Model
Summary
Thermodynamic Model
Fluid dynamics Model
Molecular Docking Model
Derivation of Rate Equation
Social Works
Human Practice
Engagement
Collaborations
Entrepreneurship
Other Works
InterLab
Improve
Safety
Measurement
Notebook
Notebook
Experiments
Engineering
Team
Members
Attributions
Judging
After iGEM
Team
Members
Attributions
Judging
After iGEM
Notebook
Notebook
Experiments
Engineering
Other Works
InterLab
Improve
Safety
Measurement
Social Works
Human Practice
Engagement
Collaborations
Entrepreneurship
Model
Summary
Thermodynamic Model
Fluid dynamics Model
Molecular Docking Model
Derivation of Rate Equation
Hardware
Overview
Microfluidic Chips
Fluorescence Detection
Raspberry Pi
Application
Software
Product Design
Project
Description
Design
Results
Demonstrate
Parts
Design
ABCD system
OMVs
Supporting
ABCD system
Ideas
During our brainstorming, we read the project of the 2017 Munich team , which established a system for the universal detection of RNA biomarkers. Therefore, we hope to improve the entire detection system and establish a system that can detect protein biomarkers universally.
Luckily, our ABCD system has appeared. Later, we discussed how could it be implied in synthetic biology. We found the answer ultimately in cell-free synthetic biology. In addition, after exchanging ideas with the Munich team, we were more certain that we did use engineering principles to re-edit the biological system.
We have divided our ABCD system into three modules, using engineering ideas, so that each can be "plug and play", but also can be "replaced". The three modules are "Sensor", "Amplifier" and "Reporter".
Figure 1. The whole design.
Sensor
In the Sensor, we first selected the epithelial cell adhesion molecule as the object of our detection. The epithelial cell adhesion molecule (EpCAM, also known as ESA or CD326) is a transmembrane glycoprotein that mediates epithelial cell-specific intercellular adhesion and involves cell signaling, migration, proliferation, and differentiation. EpCAM has been detected in most adenocarcinomas, and is also found in metastatic tumors, malignant effusions and cancer stem cells.
Next, we choosed the ssDNA aptamer as an intermediary for downstream signaling of proteins. It is a single-stranded DNA that specifically binds to a target molecule. Theoretically, the corresponding aptamer of any protein can be screened by SELEX technique. And we choosed EpCAM's aptamer--SYL3C. [1]
Name
Sequence
Modification
SYL3C
5'-CACTACAGAGGTTGCGTCTGTCCC                                 ACGTTGTCATGGGGGGTTGGCCTG-3'
3'-Biotin-TEG
Complementary strand
5'-CTCTGTAGTGTTTTTTTTTTTTTT-3'
3'-FITC
At the same time, we also used the competition method to facilitate our design of hardware. As shown in the figure, according to our assumption, the upstream EpCAM would compete with our designed aptamer and its partial complementary chain. After that, some complementary strands would be competed and freed, coming out and going to the next module to get the signal amplified.
Figure 2. The principle of Sensor.
For this module, we also modeled the molecular docking model of aptamer and protein, and partially predicted the number of bases in the complementary strands. Click to learn more.
Amplifier
In this module we used the Cas12a protein to amplify the signal. Cas12a (also known as Cpf1) was originally discovered as a replacement for Cas9. [2] But this year's discovery showed that it had a new function of being activated by ssDNA and realizing trans-cleavage. [3]
Figure 3. The principle of Amplifer.
As shown in the figure, when some of our upstream complementary strands are released to the Cas12a protein after competitions, recognized by the specifically designed crRNA and finally complemented with it successfully, the Cas12a protein will be activated, and non-specific cleavage will occur.
Name
Sequence
Modification
SYL3C-crRNA
5'-UAAUUUCUACUCUUGUAGAU                                         AAAAAAAAAAACACUACAGAG-3
None
Reporter
When the Cas12a protein is activated, it will realize a non-specific cleavage that functions as DNase I, so we can report our signal by using the method of detecting DNase I.
We finally used the fluorescent signal as our reporting signal, because it was easy to observe, semi-quantitative, and can be directly read and analyzed by the machine. And our signal reports used the commercially available DNaseAlertTM kit on the market. [4]
The working principle is as follows: a combination of a ground state fluorescent molecule and a quencher generates a complex by ssDNA, and the complex quenches the fluorescence. When the nuclease cleavages the ssDNA, the quenching fails and the fluorophore re-emits.
Figure 4. The principle of Reporter.
References
[1] Yanling Song, Zhi Zhu, Yuan An, Weiting Zhang, Huimin Zhang, Dan Liu, Chundong Yu, Wei Duan, Chaoyong James Yang. Selection of DNA Aptamers against Epithelial Cell Adhesion Molecule for Cancer Cell Imaging and Circulating Tumor Cell Capture. Anal. Chem. 2013, 85: 4141-4149.
[2] Zetsche B, Gootenberg JS, Abudayyeh OO, Slaymaker IM, Makarova KS, Essletzbichler P, Volz SE, Joung J, van der Oost J, Regev A, Koonin EV, Zhang F. Cpf1 is a single RNA-guided endonuclease of a class 2 CRISPR-Cas system. Cell. 2015, 163(3): 759-771.
[3] Chen JS, Ma E, Harrington LB, Da Costa M, Tian X, Palefsky JM, Doudna JA. CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity. Science. 2018, 360(6387): eaar6245.
[4] DNaseAlertTM Substrate Nuclease Detection System User Manual.
OMVs
Ideas
To utilize outer-membrane vesicles (OMVs) as our cell-free platform for RNA delivery and then for disease treatment, we designed three modules with different functions, which can be divided into OmpA, L7Ae, and Box (Figure 1).
Figure 1. a) Genetic circuit of three modules. b) Schematic illustration of our total circuit.
Module one - OmpA
Porin outer-membrane protein A, OmpA, is a kind of highly expressed transmembrane porin protein of E. coli and is therefore abundant in OMVs. [1] We inserted SpyTag to the C-termini and N-termini of a truncated form of OmpA respectively to compare the efficiency of SpyTag/SpyCatcher linkage (Figure 2).
Figure 2. a) Genetic circuit of SpyTag-OmpA part and OmpA-SpyTag part. b) Schematic illustration of module one.
Furthermore, to characterize whether our OmpA-SpyTag can localize in OMVs, we also attached GFP to both OmpA parts (Figure 3).
Figure 3. a) Genetic circuit of SpyTag-OmpA-GFP part and OmpA-SpyTag-GFP part. b) Schematic illustration of these two parts.
Module two – L7Ae
Archaeal ribosomal protein protein L7Ae owns the ability to bind with RNA with C/Dbox RNA structure. [2-4] Through the isopeptide bonds formed between SpyTag and SpyCatcher, L7Ae can be anchored with OmpA, and then secreted into OMVs. To transport our L7Ae into the periplasmic space, we inserted TorA leader sequence to the circuit (Figure 4).
Figure 4. Genetic circuit of SpyCatcher-L7Ae.
To localize our L7Ae inside OMVs, we also conjugated mRFP1 to the L7Ae-SpyCatcher. Under the induction of the IPTG, our bacteria coudld express our L7Ae parts (Figure 5).
Figure 5. Genetic circuit of SpyCatcher-L7Ae-RFP.
Module three - Box
We inserted a C/Dbox into the 3'-untranslated region (3'-UTR) of the gene coding for siRNA. [5] Under the induction of 0.2% Arabinose, our bacteria could express our Box parts (Figure 6).
Figure 6. Genetic circuit of Box part.
Under the induction of 0.2% Arabinose and then 0.5 mM IPTG, we could produce L7Ae-SpyCatcher and siRNA-C/Dbox. With the isopeptide bond formed between SpyTag and SpyCatcher, and the ability of L7Ae to be bind with C/Dbox, we could produce customizable and cell-free OMVs containing specific siRNA to target for oncogenic gene (Figure 7).
Figure 7. Schematic illustration of our total circuit.
References
[1] Acevedo R, Fernández S, Zayas C, et al. Bacterial Outer Membrane Vesicles and Vaccine Applications. Frontiers in Immunology. 2014, 5: 121.
[2] Ausländer S, Ausländer D, Müller M, et al. Programmable single-cell mammalian biocomputers. Nature, 2012, 487(7405):123-127.
[3] Saito H, Fujita Y, Kashida S, et al. Synthetic human cell fate regulation by protein-driven RNA switches. Nature Communications. 2011, 2(1): 160.
[4] Saito H, Kobayashi T, Hara T, et al. Synthetic translational regulation by an L7Ae–kink-turn RNP switch. Nature Chemical Biology. 2009, 6(1): 71.
[5] Kojima R, Bojar D, Rizzi G, et al. Designer exosomes produced by implanted cells intracerebrally deliver therapeutic cargo for Parkinson's disease treatment. Nature Communications. 2018, 9(1): 1305.
Supporting
Ideas
We selected two proteins SAHS1 and CAHS1, which have been shown with excellent effects by a Japanese team in 2012. SAHS1 could be used to preserve proteins and CAHS1 could be used to preserve bacteria.
Our genetic circuits are shown below:
Figure1. Genetic circuits of TDP.
CAHS protein is a cytoplasmic-abundant protein which can strengthen the stress tolerance of bacteria. Taking biosafety into account, we added constant promoter J23100 into the CAHS circuit to express lacI in order to inhibit lac O promoter. Once IPTG is added, the inhibition of lacI will be removed. Then CAHS and mRFP will be expressed, in which mRFP was used for screening. We intended to adopt the Red recombination method to transform the circuit into the genome of DH5α. Hence, DH5α would be improved and preserved easily.
After discussion, we decided to add mRFP or blue chromoprotein to the SAHS part for selecting E. coli transfected successfully. Since SAHS protein is an exogenous protein, it is preceded by a signal peptide, which allows it to pass through the cell membrane. We removed the LB medium by centrifugation and harvested our proteins in high concentration. As a protective agent, SAHS can prevent Cas12a from lyophilization.
Expectation
A new E.coli strain
We intend to transform the circuit into the genome of DH5α. Hence, DH5α would be improved and preserved easily. In this way, E. coli DH5α can be transported normally and easily.
A new way to preserve protein
We hope that the purified TDPs can be used to preserve proteins and other biological materials.
References
[1] Yamaguchi A, Tanaka S, Yamaguchi S, et al. Two Novel Heat-Soluble Protein Families Abundantly Expressed in an Anhydrobiotic Tardigrade. PLoS ONE, 2012, 7(8):e44209.
[2] Boothby T C, Tapia H, Brozena A H, et al. Tardigrades Use Intrinsically Disordered Proteins to Survive Desiccation. Molecular Cell, 2017, 65(6):975-984.
[3] Boothby T C, Pielak G J. Intrinsically Disordered Proteins and Desiccation Tolerance: Elucidating Functional and Mechanistic Underpinnings of Anhydrobiosis. Bioessays, 2017, 39(11).
Home
Model
Parts
Entrepreneurship
Attributions
Design
Hardware
Human Practices
Notebook
Judging
Results
Software
Collaborations
Members
After iGEM
