<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTU-Taida_Project_Stability"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTU-Taida/Project/Stability">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTU-Taida/Project/Stability&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTU-Taida/Project/Stability&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTU-Taida/Project/Stability&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTU-Taida/Project/Stability" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTU-Taida/Project/Stability</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="navbar navbar-fixed-top"><DIV class="navbar-inner"><DIV class="container"><A class="brand" href="/Team:NTU-Taida">NTU-Taida</A><DIV class="nav-collapse"><UL class="nav" id="wiki-navbar"><LI id="nav-Home"><A href="/Team:NTU-Taida">Home</A></LI><LI class="dropdown" id="nav-Project"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI id="nav-Project-Intro"><A href="/Team:NTU-Taida/Project/Introduction">Introduction</A></LI><LI id="nav-Project-Background"><A href="/Team:NTU-Taida/Project/Background">Background</A></LI><LI id="nav-Project-Overview"><A href="/Team:NTU-Taida/Project/Overview">Project Overview</A></LI><LI id="nav-Project-Effector" style="margin-left: 10px;"><A href="/Team:NTU-Taida/Project/Effector">Effector</A></LI><LI id="nav-Project-Sensor" style="margin-left: 10px;"><A href="/Team:NTU-Taida/Project/Sensor">Sensor</A></LI><LI id="nav-Project-Circuit" style="margin-left: 10px;"><A href="/Team:NTU-Taida/Project/Circuit">Circuit</A></LI><LI id="nav-Project-Stability" style="margin-left: 10px;"><A href="/Team:NTU-Taida/Project/Stability">Stability</A></LI><LI id="nav-Project-Safety" style="margin-left: 10px;"><A href="/Team:NTU-Taida/Project/Safety">Safety</A></LI><LI id="nav-Project-Future"><A href="/Team:NTU-Taida/Project/Future-Plan">Future Plan</A></LI></UL><LI class="dropdown" id="nav-Result"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Result</A></LI><LI id="nav-Result-pFadBA"><A href="/Team:NTU-Taida/Result/pFadBA">pFadBA Promoter</A></LI><LI id="nav-Result-Thermal"><A href="/Team:NTU-Taida/Result/Thermal-Promoter">Thermal Promoter</A></LI><LI id="nav-Result-clPromoter"><A href="/Team:NTU-Taida/Result/Modified-cI-promoter">Modified cI promoter</A></LI><LI id="nav-Result-Secetion-CPP"><A href="/Team:NTU-Taida/Result/Secretion-CPP">Secretion: CPP</A></LI><LI id="nav-Result-Secetion-GLP1"><A href="/Team:NTU-Taida/Result/Secretion-GLP1">Secretion: GLP-1</A></LI><LI id="nav-Result-Penetration"><A href="/Team:NTU-Taida/Result/Penetration">Penetration</A></LI><LI id="nav-Result-Stability"><A href="/Team:NTU-Taida/Result/Stability">Stability</A></LI><LI class="dropdown" id="nav-Modeling"><A class="dropdown-toggle" id="nav-model-overview" data-toggle="dropdown" href="#">Modeling</A></LI><LI id="nav-Modeling-Overview"><A href="/Team:NTU-Taida/Modeling">Overview</A></LI><LI id="nav-Modeling-Single"><A href="/Team:NTU-Taida/Modeling/Single-Cell">Single Cell Model</A></LI><LI id="nav-Modeling-FA"><A href="/Team:NTU-Taida/Modeling/FA">Fatty Acid Reaction-Absorption Model</A></LI><LI id="nav-Modeling-System"><A href="/Team:NTU-Taida/Modeling/System-Analysis">System Analysis</A></LI><LI id="nav-Modeling-Stochastic"><A href="/Team:NTU-Taida/Modeling/Stochastic-Analysis">Stochastic Analysis</A></LI><LI id="nav-Modeling-Combined"><A href="/Team:NTU-Taida/Modeling/2D-3D-Combined">2D &amp; 3D Combined Model</A></LI><LI id="nav-Modeling-Plasmid"><A href="/Team:NTU-Taida/Modeling/Plasmid-Stability">Plasmid Stability Model</A></LI><LI id="nav-Modeling-Parameters"><A href="/Team:NTU-Taida/Modeling/Parameters">Parameters</A></LI><LI id="nav-Parts"><A href="/Team:NTU-Taida/Parts">Parts</A></LI><LI id="nav-Safety"><A href="/Team:NTU-Taida/Safety">Safety</A></LI><LI id="nav-Team"><A href="/Team:NTU-Taida/Team">Team</A></LI><LI class="dropdown" id="nav-HumanPractice"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practice</A></LI><LI id="nav-HumanPractice-Overview"><A href="/Team:NTU-Taida/Human Practice/Overview">Overview</A></LI><LI id="nav-HumanPractice-DesignThinking"><A href="/Team:NTU-Taida/Human Practice/Design">Design Thinking in Synthetic Biology</A></LI><LI id="nav-HumanPractice-Symposium"><A href="/Team:NTU-Taida/Human Practice/Symposium">Symposium with High School Students</A></LI><LI id="nav-HumanPractice-Questionnaire"><A href="/Team:NTU-Taida/Human Practice/Questionnaire">Questionnaire Analysis</A></LI><LI id="nav-HumanPractice-SafetyiniGEM"><A href="/Team:NTU-Taida/Human Practice/Safety-in-iGEM">Safety in iGEM</A></LI><LI id="nav-HumanPractice-DoctorInterview"><A href="/Team:NTU-Taida/Human Practice/Doctor-Interview">Doctor Interview</A></LI><LI id="nav-HumanPractice-Collaboration"><A href="/Team:NTU-Taida/Human Practice/Collaboration">Collaboration and Other Activities</A></LI><LI id="nav-HumanPractice-InformationPlatform"><A href="/Team:NTU-Taida/Human Practice/Information-Platform">Information Platform</A></LI><LI class="dropdown" id="nav-user"><A class="dropdown-toggle" id="nav-login" role="button" data-toggle="dropdown" data-target="#" href="">
              Login
            </A></LI><LI><A href="" id="nav-edit">Edit</A></LI><LI><A href="" id="nav-history">History</A></LI><LI><A href="/Special:Upload" id="nav-upload">Upload</A></LI><LI><A href="" id="nav-logout">Logout</A></LI></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row" id="maintext"><DIV class="span3 my-sidebar"><DIV class="my-sidebar-nav"><H4 class="lead">Stability</H4></DIV></DIV><DIV class="span9"><DIV class="hero-unit" style="margin-top: 20px; margin-bottom: 0px;"><H1>Stability of Delivery System</H1><P>Every GM system that functions outside the laboratories encounters two major problems: system stability and safety! Without selective pressure, we have to deal with plasmid instability. Besides, plasmid multimers may be produced in recA+ strains we used to make our ''E. coli'' colonize bowel. Lastly, with many kinds of bacteria in intestines, our genetically modified ''E. coli'' also lives under the risk of horizontal gene transfer.</P><P>The following segment depicts our struggle against these obstacles.</P></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Abstract"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Abstract</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Stability_Issues"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Stability Issues</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#1_Multimer_Resolution_System"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">1 Multimer Resolution System</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#2_Partition_System"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">2 Partition System</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#3_Toxin-antitoxin_System"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">3 Toxin-antitoxin System</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#4_Reduce_Burden_Effect"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">4 Reduce Burden Effect</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Reference"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Abstract">Abstract</SPAN></H2><P style="text-indent: 2em;">Since our PepdEx system consists of several plasmids and will function in human gut, which lacks antibiotic selection pressure, plasmid segregation stability is critical for determining whether every single E. Coli in guts contains our original designed system after numerous generations. Inspired by natural plasmid and mobile gene element, we cope with segregation instability by incorporating three modules, namely <STRONG>partition system</STRONG>, <STRONG>Multimer resolution system</STRONG>, and <STRONG>toxin antitoxin system</STRONG> into our system. We suppose that the level of regulation would potentially benefit the entire synthetic biology community, as synthetic systems are becoming increasingly complex with fine controls (e.g. on low copy plasmids).</P><H2><SPAN class="mw-headline" id="Stability_Issues">Stability Issues</SPAN></H2><P>Problems of plamid instability mainly results from the segregational instability,the burden effect and the dimer catastrophe.
</P><OL><LI> Segregational instability <P style="text-indent: 2em;"> Plasmids are randomly (possibly uneven) distributed inside a bacterium. After cell division, one of the progenies may lose plasmids.</P></LI><LI> Burden Effect <P style="text-indent: 2em;">Cells bearing our PepdEx system have to spend extra energy/resource, so they won't grow as good as plasmid-free cells. This metabolic burden will cause our E.coli to gradually lose parts of our circuits, and the different growth rate  will eventually bring about elimination of plasmid-bearing bacteria in the population.</P></LI><LI> Dimer Catastrophe<P style="text-indent: 2em;">Homologues recombination may cause plasmid multimers which will increase segregational instability and burden<A href="#Ref">[1] </A>. This is the reason why most E.coli in lab are recA1 mutants. Since our E.coli must be able to colonize in the gut, it should be recA+ wild type strain. Therefore it's the problem we need to deal with right now!</P></LI></OL><H2><SPAN class="mw-headline" id="1_Multimer_Resolution_System">1 Multimer Resolution System</SPAN></H2><DL><DD><P style="text-indent: 2em;">To deal with multimerization, we cloned a cassette (BBa_K817018) encoding an autoregulated resolvase (serine type recombinase) from E.coli F plasmid transposon tn1000 (tn3 family)<A href="#Ref">[2] </A>. Its promoter region consists of 3 sub-sites(res site) and can process recombination<A href="#Ref">[3] </A>. This cassette can resolve multimers that are formed during replicative transposition<A href="#Ref">[4] </A>. It can also increase plasmid stability by avoiding dimer catastrophe. Multimer resolution system (MRS) has analogous functions as chromosome XerCD/dif in E.coli; however, MRS seems more efficient due to its independence of cell cycle and DNA localization.<A href="#Ref">[5] </A>.</P></DD></DL><H2><SPAN class="mw-headline" id="2_Partition_System">2 Partition System</SPAN></H2><DL><DD><P style="text-indent: 2em;">Bacterium distributes chromosomes/plasmids evenly to its progenies by a dynamic system that includes SMC like proteins, type Ia partition system and so on<A href="#Ref">[6] </A>. Type Ia partition system segregates chromosomes/plasmids in a process akin to Eukaryotic mitosis. It can be found in most eubacteria. However, E. coli is not the case.</P></DD></DL><DL><DD><P style="text-indent: 2em;">Previous studies have shown that when putting parAB of P.putida in trans and setting parS site in cis, segregation of low copy plasmids (mini F) can be stabilized in E. coli.<A href="#Ref">[7] ]</A>].The BioBrick vector pSB2K3 is a mini F plasmid, and thus we expect it can help stabilize the segregation of our system. Our team has made a modified version of pSB2K3 with parS site (BBa_K817017), and plans to use it to harbor our PepdEx system (a low copy number plasmid allows us to fine tune our systems controls e.g. with smaller number of operator sites). We have also cloned the parAB operon (BBa_K817019), and provided the registry with the short 16bp parS site (BBa_K817016) for future applications.</P></DD></DL><H2><SPAN class="mw-headline" id="3_Toxin-antitoxin_System">3 Toxin-antitoxin System</SPAN></H2><DL><DD><P style="text-indent: 2em;">No matter how well our partition system and multimer resolution system work, inevitably there are still some bacteria losing their plasmids (parts of the circuit). These cells with incomplete system will not function as expected. Thus we sentence these cells to death to solve the problem.</P></DD></DL><DL><DD><P style="text-indent: 2em;">Type I toxin-antitoxin srnBC is an ideal executor which belongs to hok/sok homologues. It expresses stable toxin encoding RNA and short-lived antitoxin RNA that can neutralize toxin RNA by RNA interaction and RNase III cleavage<A href="#Ref">[9] </A>. It acts as post segregation killing system â a bacterial cell will be killed if it loses the DNA (genomic islands, plasmids, mobile gene elements) containing the module<A href="#Ref">[10] </A>. We plan to use it to reduce plasmid-lost cells in our coli population, in the hope to make applications without antibiotic selection more feasible. We have cloned the cassette with BioBrick standard (BBa_K817015). Following pictures show the mechanism (based on hok/sok).</P></DD></DL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><P style="text-indent: 2em;">TA loci express two kinds of RNA, namely srnB and srnC, respectively. Initially, srnB is inactive and stable (half-life in the order of 30 minutes <A href="#Ref">[11] </A>) and will gradually be processed to active form producing hydrophobic toxic protein if srnC RNA does not exist; srnC is relative unstable and is degraded quickly, and it overlaps and compliments to 5âsequence of srnB.</P></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><P style="text-indent: 2em;">At normal condition (plasmid retained), srnC can pair with processed srnB and forms double stranded RNA that is susceptible to RNase III cleavage and thus prevents toxin production. If the plasmid loses, srnC will quickly disappear without replenishing and srnB will show its power<A href="#Ref">[12] </A>.</P></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="4_Reduce_Burden_Effect">4 Reduce Burden Effect</SPAN></H2><DL><DD><P style="text-indent: 2em;">Finally, beyond these modules, reducing the burden effect is generally important to the stability of complex artificial systems in an intact E. coli cell without antibiotic selection. We reason that a well designed system with a lower gene dosage would help.</P></DD></DL><P style="text-indent: 2em;">High level gene expression and gene dosage lead to burden effect. That is to say, if two systems are encoded in different plasmids with the same overall performance, bacteria with lower copy plasmids prevail over the ones with higher copy plasmids. Again if with the same overall outcome, stabilizing mRNA prevails over overexpressing the gene (unless for regulatory purposes). Put yourselves in E. coli's shoes! Reduce its metabolic burden as much as possible then it will work for you! </P><H3><SPAN class="mw-headline" id="Reference">Reference</SPAN></H3><OL id="Ref"><LI>Field, C.M. and D.K. Summers, Multicopy plasmid stability: revisiting the dimer catastrophe. Journal of Theoretical Biology, 2011. 291: p. 119-27.</LI><LI>Falvey, E. and N.D.F. Grindley, Contacts between Gamma-Delta-Resolvase and the Gamma-Delta-Res Site. Embo Journal, 1987. 6(3): p. 815-821.</LI><LI>Grindley, N.D.F., et al., Transposon-Mediated Site-Specific Recombination - Identification of 3 Binding-Sites for Resolvase at the Res-Sites of Gamma-Delta and Tn3. Cell, 1982. 30(1): p. 19-27.</LI><LI>Wells, R.G. and N.D.F. Grindley, Analysis of the Gamma-Delta Res Site - Sites Required for Site-Specific Recombination and Gene-Expression. Journal of Molecular Biology, 1984. 179(4): p. 667-687.</LI><LI>Guhathakurta, A., I. Viney, and D. Summers, Accessory proteins impose site selectivity during ColE1 dimer resolution. Molecular Microbiology, 1996. 20(3): p. 613-620.</LI><LI>Mullins, R.D., Bacterial Chromosome Segregation. Annu Rev Cell Dev Biol, 2009.</LI><LI>Godfrin-Estevenon, A.M., F. Pasta, and D. Lane, The parAB gene products of Pseudomonas putida exhibit partition activity in both P. putida and Escherichia coli. Molecular Microbiology, 2002. 43(1): p. 39-49.</LI><LI>Gerdes, K., et al., Mechanism of killer gene activation. Antisense RNA-dependent RNase III cleavage ensures rapid turn-over of the stable hok, srnB and pndA effector messenger RNAs. Journal of Molecular Biology, 1992. 226(3): p. 637-49.</LI><LI>Thisted, T. and K. Gerdes, Mechanism of post-segregational killing by the hok/sok system of plasmid R1. Sok antisense RNA regulates hok gene expression indirectly through the overlapping mok gene. Journal of Molecular Biology, 1992. 223(1): p. 41-54.</LI><LI>Szekeres, S., et al., Chromosomal toxin-antitoxin loci can diminish large-scale genome reductions in the absence of selection. Molecular Microbiology, 2007. 63(6): p. 1588-1605.</LI><LI>Faridani, O.R., et al., Competitive inhibition of natural antisense Sok-RNA interactions activates Hok-mediated cell killing in Escherichia coli. Nucleic Acids Research, 2006. 34(20): p. 5915-5922.</LI><LI>Gerdes, K. and E.G.H. Wagner, RNA antitoxins. Current Opinion in Microbiology, 2007. 10(2): p. 117-124.</LI></OL></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:NTU-Taida/Project/Stability">http://2012.igem.org/Team:NTU-Taida/Project/Stability</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTU-Taida/Project/Stability" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTU-Taida/Project/Stability" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTU-Taida/Project/Stability&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTU-Taida/Project/Stability&amp;oldid=289965" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>