<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTU-Taida_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTU-Taida/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTU-Taida/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTU-Taida/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTU-Taida/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTU-Taida/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTU-Taida/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV class="navbar navbar-fixed-top"><DIV class="navbar-inner"><DIV class="container"><A class="brand" href="/Team:NTU-Taida">NTU-Taida</A><DIV class="nav-collapse"><UL class="nav" id="wiki-navbar"><LI id="nav-Home"><A href="/Team:NTU-Taida">Home</A></LI><LI class="dropdown" id="nav-Project"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI id="nav-Project-Intro"><A href="/Team:NTU-Taida/Project/Introduction">Introduction</A></LI><LI id="nav-Project-Background"><A href="/Team:NTU-Taida/Project/Background">Background</A></LI><LI id="nav-Project-Overview"><A href="/Team:NTU-Taida/Project/Overview">Project Overview</A></LI><LI id="nav-Project-Effector" style="margin-left: 10px;"><A href="/Team:NTU-Taida/Project/Effector">Effector</A></LI><LI id="nav-Project-Sensor" style="margin-left: 10px;"><A href="/Team:NTU-Taida/Project/Sensor">Sensor</A></LI><LI id="nav-Project-Circuit" style="margin-left: 10px;"><A href="/Team:NTU-Taida/Project/Circuit">Circuit</A></LI><LI id="nav-Project-Stability" style="margin-left: 10px;"><A href="/Team:NTU-Taida/Project/Stability">Stability</A></LI><LI id="nav-Project-Safety" style="margin-left: 10px;"><A href="/Team:NTU-Taida/Project/Safety">Safety</A></LI><LI id="nav-Project-Future"><A href="/Team:NTU-Taida/Project/Future-Plan">Future Plan</A></LI></UL><LI class="dropdown" id="nav-Result"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Result</A></LI><LI id="nav-Result-pFadBA"><A href="/Team:NTU-Taida/Result/pFadBA">pFadBA Promoter</A></LI><LI id="nav-Result-Thermal"><A href="/Team:NTU-Taida/Result/Thermal-Promoter">Thermal Promoter</A></LI><LI id="nav-Result-clPromoter"><A href="/Team:NTU-Taida/Result/Modified-cI-promoter">Modified cI promoter</A></LI><LI id="nav-Result-Secetion-CPP"><A href="/Team:NTU-Taida/Result/Secretion-CPP">Secretion: CPP</A></LI><LI id="nav-Result-Secetion-GLP1"><A href="/Team:NTU-Taida/Result/Secretion-GLP1">Secretion: GLP-1</A></LI><LI id="nav-Result-Penetration"><A href="/Team:NTU-Taida/Result/Penetration">Penetration</A></LI><LI id="nav-Result-Stability"><A href="/Team:NTU-Taida/Result/Stability">Stability</A></LI><LI class="dropdown" id="nav-Modeling"><A class="dropdown-toggle" id="nav-model-overview" data-toggle="dropdown" href="#">Modeling</A></LI><LI id="nav-Modeling-Overview"><A href="/Team:NTU-Taida/Modeling">Overview</A></LI><LI id="nav-Modeling-Single"><A href="/Team:NTU-Taida/Modeling/Single-Cell">Single Cell Model</A></LI><LI id="nav-Modeling-FA"><A href="/Team:NTU-Taida/Modeling/FA">Fatty Acid Reaction-Absorption Model</A></LI><LI id="nav-Modeling-System"><A href="/Team:NTU-Taida/Modeling/System-Analysis">System Analysis</A></LI><LI id="nav-Modeling-Stochastic"><A href="/Team:NTU-Taida/Modeling/Stochastic-Analysis">Stochastic Analysis</A></LI><LI id="nav-Modeling-Combined"><A href="/Team:NTU-Taida/Modeling/2D-3D-Combined">2D &amp; 3D Combined Model</A></LI><LI id="nav-Modeling-Plasmid"><A href="/Team:NTU-Taida/Modeling/Plasmid-Stability">Plasmid Stability Model</A></LI><LI id="nav-Modeling-Parameters"><A href="/Team:NTU-Taida/Modeling/Parameters">Parameters</A></LI><LI id="nav-Parts"><A href="/Team:NTU-Taida/Parts">Parts</A></LI><LI id="nav-Safety"><A href="/Team:NTU-Taida/Safety">Safety</A></LI><LI id="nav-Team"><A href="/Team:NTU-Taida/Team">Team</A></LI><LI class="dropdown" id="nav-HumanPractice"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practice</A></LI><LI id="nav-HumanPractice-Overview"><A href="/Team:NTU-Taida/Human Practice/Overview">Overview</A></LI><LI id="nav-HumanPractice-DesignThinking"><A href="/Team:NTU-Taida/Human Practice/Design">Design Thinking in Synthetic Biology</A></LI><LI id="nav-HumanPractice-Symposium"><A href="/Team:NTU-Taida/Human Practice/Symposium">Symposium with High School Students</A></LI><LI id="nav-HumanPractice-Questionnaire"><A href="/Team:NTU-Taida/Human Practice/Questionnaire">Questionnaire Analysis</A></LI><LI id="nav-HumanPractice-SafetyiniGEM"><A href="/Team:NTU-Taida/Human Practice/Safety-in-iGEM">Safety in iGEM</A></LI><LI id="nav-HumanPractice-DoctorInterview"><A href="/Team:NTU-Taida/Human Practice/Doctor-Interview">Doctor Interview</A></LI><LI id="nav-HumanPractice-Collaboration"><A href="/Team:NTU-Taida/Human Practice/Collaboration">Collaboration and Other Activities</A></LI><LI id="nav-HumanPractice-InformationPlatform"><A href="/Team:NTU-Taida/Human Practice/Information-Platform">Information Platform</A></LI><LI class="dropdown" id="nav-user"><A class="dropdown-toggle" id="nav-login" role="button" data-toggle="dropdown" data-target="#" href="">
              Login
            </A></LI><LI><A href="" id="nav-edit">Edit</A></LI><LI><A href="" id="nav-history">History</A></LI><LI><A href="/Special:Upload" id="nav-upload">Upload</A></LI><LI><A href="" id="nav-logout">Logout</A></LI></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row" id="maintext"><DIV class="span3 my-sidebar"><DIV class="my-sidebar-nav"><H4 class="lead">Modeling Overview</H4></DIV></DIV><DIV class="span9"><DIV class="hero-unit" style="margin-top: 20px; margin-bottom: 0px;"><H1>Modeling Overview</H1><P>Since our system is designed to work in the human intestine, it depends heavily on the physiological condition in human body. There’s usually a gap between the reality and the models, especially for those regarding true physiological conditions. To solve this, we take the real conditions into consideration as much as possible, either incorporates them into the model design or adjust our model parameters to fit to the conditions, and again, evaluate if our results are compatible to reality. Our ultimate goal is to bring our simulation closer and closer to the real conditions.</P><P>The intimate interaction between our system and the physiological environment can be seen from the very beginning in our circuit design. There is in fact a baseline level of fatty acid in the gut even during the time without food intake. However, we don’t want our E.coli cells to respond to this noise. Therefore, we designed a filter in our circuit to filter out the baseline noise. After a fatty meal, the concentration of fatty acid around the cells gradually increases. When fatty acid level rises above the threshold, the circuits in the cells are turned on and express GLP-1 in response. In this way, we aim to make our cells to respond sensitively to fat intake, but remain silent to the baseline noise.</P><P>To achieve this, we have to first ensure that the filter will work. Therefore, we start our modeling effort with the single cell model to verify the filter function of our circuit.</P></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Single_Cell_Model"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Single Cell Model</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Fatty_Acid_Reaction-Absorption_Model"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Fatty Acid Reaction-Absorption Model</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#System_Analysis"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">System Analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Stochastic_Analysis"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Stochastic Analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#2D_.26_3D_Combined_Model"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">2D &amp; 3D Combined Model</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#2D_Combined_Model"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">2D Combined Model</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#2D_Cell_Population_Model"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">2D Cell Population Model</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#3D_Cell_Population_Model"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">3D Cell Population Model</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#Plasmid_Stability_Model"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Plasmid Stability Model</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Single_Cell_Model">Single Cell Model</SPAN></H2><P style="text-indent: 2em;">The single cell model is the basis of our modeling. It described each species involved in our circuit by a set of <B>ordinary differential equations</B> and shows the function of our modules. With the single cell computational model, we <B>verified the filter function of our circuit</B>, and found the <B>initial threshold</B> of the high pass filter in our circuit as <B>100 uM</B>. </P><TABLE align="center"><TBODY><TR><TD> $$X_{FadR}=FadR\times (\frac{\beta^{n2}_{FadR}}{FA^{n2}+\beta^{n2}_{FA}})$$
<P>$$\frac{\text{d}TetR_{1}}{\text{d}t}=\alpha_{TetR_{1}}\times (\frac{\beta^{n1}_{FadR}}{\beta^{n1}_{FadR}+X^{n1}_{FadR}})-\gamma_{TetR_{1}}\times TetR_{1}$$
$$\frac{\text{d}LuxI}{\text{d}t}=\alpha_{LuxI}\times (\frac{\beta^{n1}_{FadR}}{\beta^{n1}_{FadR}+X^{n1}_{FadR}})-\gamma_{LuxI}\times LuxI$$
$$\frac{\text{d}FadR}{\text{d}t}=\alpha_{FadR}-\gamma_{FadR}\times FadR$$
</P></TD></TR></TBODY></TABLE><P style="text-indent: 2em;">In addition, the single cell model helps us to have a complete view of how modules act together as a system in a single E.coli cell and provides a quantitative visualization of the <B>dynamical behavior</B> of our sensor, high pass filter and quorum sensing module.</P><P style="text-indent: 2em;">In order to have a better experimental (wet) characterization of each module, we performed functional tests for each module individually – i.e. PfadBA-RBS-mRFP (BBa_K817033) to characterize sensor, J23119 (constitutive promoter)-RBS-SP1-GLP1-xx (BBa_K817008) to characterize the effector, etc. </P><P style="text-indent: 2em;">So far, we have verified the filter function of our circuit. However, we didn’t know if the current threshold, which is about 100 uM, would be appropriate for the real <B>physiological condition</B>. Therefore, we have to investigate the true condition of the <B>environment in human gut</B>. After a series of literature review, we decided to fix the filter threshold at <B>800 uM</B>. Furthermore, we seek to construct a model to <B>simulate the concentration of fatty acid after a meal</B>.</P><H2><SPAN class="mw-headline" id="Fatty_Acid_Reaction-Absorption_Model">Fatty Acid Reaction-Absorption Model</SPAN></H2><P style="text-indent: 2em;">We created a <B>two dimensional fatty acid reaction- absorption model</B>, using the mechanical engineering software platform COMSOL, to describe the spatial-temporal change of <B>fatty acid level</B> in <B>the extracellular environment</B> after a <B>meal</B>. In this model, we consider the <B>hydrolysis of fat in the lumen</B> and the <B>absorption of fatty acid at the gut wall</B> and aims to determine if the response time falls in the <B>physiological reasonable time scale</B>.</P><H2><SPAN class="mw-headline" id="System_Analysis">System Analysis</SPAN></H2><P>Now we have the threshold suitable for real condition, which is about 800 uM. Next we have to find the way to adjust our threshold to this value. We performed system analysis to find the most suitable parameter to <B>adjust the threshold</B> of our circuit.
The system analysis explores extensively into the parameter space, and helps us to have better understanding of the <B>roles each parameter plays</B> in our single cell model.
</P><P>After the analysis, we chose the <B>production rate of FadR</B> as the critical parameter to adjust and successfully tuned the threshold <B>from 100uM to 800uM</B> to <B>fit the physiological conditions</B>.
</P><H2><SPAN class="mw-headline" id="Stochastic_Analysis">Stochastic Analysis</SPAN></H2><P>Besides the parameter space search, we performed another type of analysis to verify the <B>robustness</B> and the <B>mono-stability</B> of our system. Because gene expression is an <B>intrinsically stochastic</B> process, we performed stochastic simulations to see how our system reacts to noise and how it responds to perturbations. We were especially interested in the <B>filter function</B>, whether it is always present and whether the amount of GLP-1 produced in the cells for different concentrations of fatty acid has large fluctuations. 
</P><H2><SPAN class="mw-headline" id="2D_.26_3D_Combined_Model">2D &amp; 3D Combined Model</SPAN></H2><P>With the single cell model, fatty acid reaction-absorption model, and system analysis, we have verified the filter function of our circuit, simulated the real physiological fatty acid level at intestinal wall, and adjust our filter threshold to fit to the real condition. Next, we want to combine them together and see what would happen when we put cells in the intestine after a fatty meal. 
The combined model is composed of three different sections. 
</P><H3><SPAN class="mw-headline" id="2D_Combined_Model">2D Combined Model</SPAN></H3><P>2D combined model simulates the <B>GLP1 response of Pepdex E-coli cells after the food intake</B> in the x-y cross-sectional plane.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:400px"><DIV class="thumbcaption">Video 1</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:400px"><DIV class="thumbcaption">Video 2 closer look</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="2D_Cell_Population_Model">2D Cell Population Model</SPAN></H3><P>2D Cell population response model further considers the <B>possible uneven concentration of fatty acid along the z-axis</B> due to <B>non-uniform distribution of lipase</B> and simulates how the <B>quorum sensing system</B> assists our system to overcame this possible limitation and enhance the response.
</P><DIV class="center"><DIV class="floatnone"><A href="/File:NTU-Taida-Model-Overview-2DCell-video1.gif" title="File:NTU-Taida-Model-Overview-2DCell-video1.gif">File:NTU-Taida-Model-Overview-2DCell-video1.gif</A></DIV></DIV><H3><SPAN class="mw-headline" id="3D_Cell_Population_Model">3D Cell Population Model</SPAN></H3><P>Finally, 3D Cell population response model incorporates all the consideration to generate a <B>full model</B> to visualize the dynamic behavior of our system.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:400px"><DIV class="thumbcaption">Video 4</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:400px"><DIV class="thumbcaption">Video 5 closer look</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Plasmid_Stability_Model">Plasmid Stability Model</SPAN></H2><P>Last but not least, we performed a simulation of the partition system. As our PepdEx system consists of several plasmids and will function outside of the laboratory (human gut) which lacks of antibiotic selection pressure, plasmid segregation stability is critical that determines whether every single E. Coli cell contains the original system with the designated function. Inspired by natural plasmid &amp; mobile gene element, we cope with segregation instability by incorporating three modules, i.e. <B>partition system</B>, <B>Multimer resolution system</B> and <B>toxin antitoxin system</B>, on top of our system. Choosing which module to stabilize the system needs modeling and stability assay, at this section, we will use cooper’s model as an example.
</P><P>Overall, we think that modeling was valuable to the decision process in constructing our synthetic network. With the communication between the biologists and modelers, we will be able to, from a simulation result, guide experimentalists in choosing a ‘part’ with appropriate parameter sets (e.g. promoter strength, protein degradation rates) from the sea of iGEM registry, and provide quantitative insights to our flexible and expandable PepdEx system.
</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:NTU-Taida/Modeling">http://2012.igem.org/Team:NTU-Taida/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTU-Taida/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTU-Taida/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTU-Taida/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTU-Taida/Modeling&amp;oldid=294104" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>