<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCopenhagen_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCopenhagen/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu"><A href="https://2018.igem.org/Team:UCopenhagen"><DIV class="menu_item direct_link"> HOME
        </DIV></A><DIV class="menu_item"> TEAM
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Team"><DIV class="submenu_item">
                Team Members
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Collaborations"><DIV class="submenu_item">       Collaborations
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Events_Attended"><DIV class="submenu_item">
                iGEM Events
            </DIV></A></DIV><DIV class="menu_item"> PROJECT
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Background"><DIV class="submenu_item">
                Background
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Description"><DIV class="submenu_item">
                Description
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Gene_Construct_Design"><DIV class="submenu_item">
                Gene Construct Design 
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Experiments"><DIV class="submenu_item">
                Experiments
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Protocols"><DIV class="submenu_item">
                Protocols
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Notebook"><DIV class="submenu_item">
                Notebook
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Model"><DIV class="submenu_item">
                Model
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Results"><DIV class="submenu_item">
                Results
            </DIV></A></DIV><A href="https://2018.igem.org/Team:UCopenhagen/InterLab"><DIV class="menu_item direct_link"> INTERLAB
        </DIV></A><DIV class="menu_item"> PARTS
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Parts"><DIV class="submenu_item">
                Parts Overview
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Basic_Part"><DIV class="submenu_item">
                Basic Parts 
            </DIV></A></DIV><A href="https://2018.igem.org/Team:UCopenhagen/Safety"><DIV class="menu_item direct_link"> SAFETY
        </DIV></A><DIV class="menu_item"> HUMAN PRACTICES
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Human_Practices"><DIV class="submenu_item">
                Introduction
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Why_Colonize_Mars"><DIV class="submenu_item">
                Why Colonize Mars?
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Theraputic Proteins"><DIV class="submenu_item">
                Theraputic Proteins
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Choice_of_Protein"><DIV class="submenu_item">
                Choosing a Protein
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Protein_Production_System"><DIV class="submenu_item">
                Evaluation of system
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Public_Engagement"><DIV class="submenu_item">
                Education &amp; Engagement
            </DIV></A></DIV><A href="https://2018.igem.org/Team:UCopenhagen/Attributions"><DIV class="menu_item direct_link"> ATTRIBUTIONS
        </DIV></A><DIV class="menu_item"> AWARDS
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/NordicMeetup"><DIV class="submenu_item">
                Nordic iGEM conference
            </DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=UCopenhagen"><DIV class="menu_item direct_link"> JUDGING FORM ‚áó
        </DIV></A><DIV class="menu_item"> Miscellaneous
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Articles"><DIV class="submenu_item">
                Articles
            </DIV></A><A href="https://igem-copenhagen.github.io/awesome-iGEM/"><DIV class="submenu_item">
                Awesome iGEM
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Wiki_how_tos"><DIV class="submenu_item">
                Wiki How-tos
            </DIV></A></DIV><A href="https://medium.com/igem-copenhagen" style="padding-right: 0px;" target="_blank"><DIV class="menu_item direct_link"> Blog on Medium
        </DIV></A></DIV><DIV class="igem_2018_team_content pageContent"><DIV class="igem_2018_team_column_wrapper"><H1> Lab notebook </H1><P>The notebook is a chronological overview of what has happened in the lab. It is organized after each experiment. </P><TITLE>jQuery UI Example Page</TITLE><DIV id="tabs"><UL><LI><A href="#tabs-1">Cloning</A></LI><LI><A href="#tabs-3">Liposome charaterization</A></LI><LI><A href="#tabs-4">Protoplast experiments with plants</A></LI><LI><A href="#tabs-5">Leakiness characterization</A></LI><LI><A href="#tabs-6">Supported lipid bilayers</A></LI><LI><A href="#tabs-7">InterLab</A></LI><LI><A href="#tabs-8">Egg membrane experiment</A></LI></UL><DIV id="tabs-1"><H1>Cloning and Plasmid Construction</H1><H2>Week 25</H2><H3>Competent cell preparation 1 ‚Äì 19/06/18</H3><P>
Overnight culture of MACH1 and DH5Œ± E. coli
Inoculate MACH1 and DH5Œ± E. coli from glycerol stock into 5 ml LB broth. Incubate at 37 <SUP>‚ó¶</SUP>C with 150 rpm shaking overnight.
</P><P>
Prepare CCMB80 competent cell buffer (100ml)
</P><TABLE border="0"><TBODY><TR><TH>Components</TH><TH>Amount</TH><TH>Final concentration</TH></TR><TR><TH>1M pH 7.0 CH<SUB>3</SUB>COOK stock solution</TH><TH>1 mL</TH><TH>10 mM</TH></TR><TR><TH>CaCl<SUB>2</SUB>.2H<SUB>2</SUB>O</TH><TH>1.18 g</TH><TH>80 mM</TH></TR><TR><TH>MnCl<SUB>2</SUB>.4H<SUB>2</SUB>O</TH><TH>0.4 g</TH><TH>20 mM</TH></TR><TR><TH>MgCl<SUB>2</SUB>.6H<SUB>2</SUB>O</TH><TH>0.2 g</TH><TH>10 mM</TH></TR><TR><TH>Glycerol</TH><TH>10 mL</TH><TH>10 % v/v</TH></TR><TR><TH>MiliQ H<SUB>2</SUB>O </TH><TH>added to 100 ml</TH><TH>Final concentration</TH></TR></TBODY></TABLE><P>

    Adjust pH to 6.4 by 0.01M HCl. Sterile by filtration through 0.22 Œºm filter. Keep in 4 <SUP>‚ó¶</SUP>C
</P><H4>Nat</H4><H2>Competent cell preparation 2 ‚Äì 20/06/18</H2><P>
Protocol adjusted from iGEM <A href="http://parts.igem.org/Help:Protocols/Competent_Cells">Competent Cell Production Protocol</A>.
</P><P>
Inoculate 1 ml of MACH1 and DH5Œ± E. coli O/N culture in 100 ml of fresh LB broth. Incubate at room temperature, with 50 rpm shaking until OD600 reach 0.3 - 0.5 (3.30 hours).  
</P><OL><LI>Chill the culture on ice for 30 min.
</LI><LI>Centrifuge at 3000 g for 10 min. Discard supernatant.
</LI><LI>Resuspend cell pellet in 50 ml ice cold sterile H<SUB>2</SUB>0.
</LI><LI>Centrifuge at 3000 g for 10 min. Discard supernatant.
</LI><LI>Resuspend cell pellet gently in 20 ml ice cold CCMB80 buffer.
</LI><LI>Centrifuge at 3000 g for 10 min. Discard supernatant.
</LI><LI>Resuspend cell pellet gently in 8 ml ice cold CCMB80 buffer.
</LI><LI>Aliquot 50 ùúáL to cold 2 ml Eppendorf tube. Snap freeze in liquid N<SUB>2</SUB>.
</LI><LI> Store in -80 <SUP>‚ó¶</SUP>C</LI></OL><H4>Nat</H4><H4>Test for competency ‚Äì 21/06/18</H4><P>
Test for competency according to iGEM<A href="http://parts.igem.org/Help:2017_Competent_Cell_Test_Kit"> Competent Cell Test Kit protocol</A>. 
</P><P>
Result: 	DH5Œ± 1.5 x 10<SUP>7</SUP> CFU/Œºg
</P><P>
		MACH1 5 x 10<SUP>6 </SUP>CFU/Œºg
</P><H4>Nat</H4><H1>Week 28</H1><H2>Culture  SIEC and SIECŒîp1 <EM>E. coli</EM> ‚Äì 11/07/18</H2><P>
SIEC and SIECŒîp1<EM> E. coli </EM>strain arrived from Spain.
Streak SIEC and SIECŒîp1 <EM>E. coli</EM> from agar stab stock on LB agar plate.
</P><H4>Nat</H4><H2>Culture  SIEC and SIECŒîp1 <EM>E. coli </EM>‚Äì12/07/18</H2><P>
Pick a single colony from SIEC and SIECŒîp1 into 5 ml LB broth. Incubate at 37 <SUP>‚ó¶</SUP>C with 150 rpm shaking overnight.
</P><H4>Nat</H4><H2>Prepare SIEC and SIECŒîp1 competent cell stocks ‚Äì 13/07/18</H2><P>
Prepare competent cell stocks from SIEC and SIECŒîp1 using the same protocol mentioned in date 19/06/18 and 20/06/18.
</P><H4>Nat</H4><H1>Week 29</H1><H2>pBAD and pTac Transformation  ‚Äì 16/07/18</H2><P>
Transform pBAD (BBa_K228005) and pTac (BBa_731500) into MACH1 E. coli according to the  <A href="http://parts.igem.org/Help:Protocols/Transformation">iGEM transformation protocol</A>.
</P><P>
Primers arrived today! Dilute primer into 100 ŒºM stock and 10 ŒºM working stock.
</P><H4>Nat</H4><H2>PCR reporter genes and T3SS signal ‚Äì 17/07/18</H2><P>
PCR amplify reporter genes sfGFP (BBa_I746916), NanoLuc (BBa_K1159001), mCherry (BBa_J06504 ), and beta-lactamase (BBa_K1189008) from Biobrick distribution kit.
</P><P>
PCR assemble T3SS signal from 5 overlapping oligonucleotides. This signal encode 20 amino acid from N-terminal Map genes.
</P><TABLE><TBODY><TR><TD>Template
   </TD><TD>Primer 1
   </TD><TD>Primer 2
   </TD><TD>Product
   </TD><TD>Size (bp)
   </TD></TR><TR><TD>1. NanoLuc BBa_K1159001 
   </TD><TD>Nluc-F
   </TD><TD>Nluc-R
   </TD><TD>NanoLuc
   </TD><TD>555
   </TD></TR><TR><TD>2. sfGFP BBa_I746916 
   </TD><TD>GFP-F
   </TD><TD>GFP-R
   </TD><TD>sfGFP
   </TD><TD>748
   </TD></TR><TR><TD>3. mCherry BBa_J06504
   </TD><TD>mCH-F
   </TD><TD>mCh-R
   </TD><TD>mCherry
   </TD><TD>740
   </TD></TR><TR><TD>4. Beta-lactamase BBa_K1189008
   </TD><TD>Blac-F
   </TD><TD>Blac-R
   </TD><TD>Beta-lactamase
   </TD><TD>822
   </TD></TR><TR><TD>5. Map (no template)
   </TD><TD colspan="2">Map20-1, Map20-2, Map20-3, Map20-F2, Map20-R2 (55)
   </TD><TD> Map20
   </TD><TD>136
   </TD></TR><TR><TD>6. SIEC E. coli genomic DNA
   </TD><TD>EspD-UF
   </TD><TD>EspD-UR
   </TD><TD>EspD
   </TD><TD>1180
   </TD></TR></TBODY></TABLE><P><STRONG>Pfu-X7 PCR reaction components</STRONG></P><TABLE><TBODY><TR><TD>Components
   </TD><TD>50 ¬µl reaction
   </TD><TD>Final conc.
   </TD></TR><TR><TD>10X home-made X7 buffer
   </TD><TD>5 ¬µl
   </TD><TD>1X
   </TD></TR><TR><TD>dNTP mixture (2.5 mM)
   </TD><TD>4 ¬µl
   </TD><TD>200 ¬µM
   </TD></TR><TR><TD>Pfu-X7 polymerase (2U/¬µl)
   </TD><TD>0.5 ¬µl
   </TD><TD>1 Unit/50 ¬µl
   </TD></TR><TR><TD>Forward primer (10 ¬µM)
   </TD><TD>2.5 ¬µl
   </TD><TD>0.5 ¬µM
   </TD></TR><TR><TD>Reverse primer (10 ¬µM)
   </TD><TD>2.5 ¬µl
   </TD><TD>0.5 ¬µM
   </TD></TR><TR><TD>Template
   </TD><TD>1 ¬µl
   </TD><TD>&lt;250ng DNA
   </TD></TR><TR><TD>MiliQ H<SUB>2</SUB>O
   </TD><TD>To 50 ¬µl
   </TD><TD>-
   </TD></TR></TBODY></TABLE><P><STRONG>Pfu-X7 PCR temperature profile </STRONG></P><TABLE><TBODY><TR><TD>Step
   </TD><TD>Temperature
   </TD><TD>time
   </TD><TD>Cycle
   </TD></TR><TR><TD>Initial denaturation 
   </TD><TD>98 <SUP>‚ó¶</SUP>C
   </TD><TD>30 sec
   </TD><TD>-
   </TD></TR><TR><TD>Denaturation 
   </TD><TD>98 <SUP>‚ó¶</SUP>C
   </TD><TD>10 sec
   </TD><TD rowspan="3">35 Cycles
   </TD></TR><TR><TD>Annealing 
   </TD><TD>55 <SUP>‚ó¶</SUP>C 
   </TD><TD>15 sec
   </TD></TR><TR><TD>Extension
   </TD><TD>72 <SUP>‚ó¶</SUP>C
   </TD><TD>45 sec
   </TD></TR><TR><TD>Final Extension
   </TD><TD>72 <SUP>‚ó¶</SUP>C
   </TD><TD>5 min
   </TD><TD>-
   </TD></TR><TR><TD>Hold 
   </TD><TD>16 <SUP>‚ó¶</SUP>C
   </TD><TD>‚àû
   </TD><TD>-
   </TD></TR></TBODY></TABLE><P><STRONG>Gel result</STRONG></P><P>
1. NanoLuc Fail
2. sfGFP Fail
3. mCherry Fail
4. Beta-lactamase Fail
5. Map20 (T3SS signal) Success
</P><P>
6. EspD Success
</P><P>
The failed PCR may result from incompatible PCR polymerase enzyme or buffer. Next time change PCR enzyme to Q5 DNA polymerase.
</P><P>
Nat, Selma
</P><H2>PCR reporter genes ‚Äì 18/07/18</H2><P>
PCR amplify reporter genes sfGFP (BBa_I746916), NanoLuc (BBa_K1159001), mCherry (BBa_J06504 ), and beta-lactamase (BBa_K1189008) again using Q5 DNA polymerase.
</P><TABLE><TBODY><TR><TD>Template
   </TD><TD>Primer 1 (Tm)
   </TD><TD>Primer 2 (Tm)
   </TD><TD>Product
   </TD><TD>Size(bp)
   </TD></TR><TR><TD>1. sfGFP BBa_I746916 
   </TD><TD>GFP-F (61)
   </TD><TD>GFP-R (55)
   </TD><TD>sfGFP
   </TD><TD>748
   </TD></TR><TR><TD>2. NanoLuc BBa_K1159001 
   </TD><TD>Nluc-F (68)
   </TD><TD>Nluc-R (60)
   </TD><TD>NanoLuc
   </TD><TD>555
   </TD></TR><TR><TD>3. mCherry BBa_J06504
   </TD><TD>mCH-F (63)
   </TD><TD>mCh-R (58)
   </TD><TD>mCherry
   </TD><TD>740
   </TD></TR><TR><TD>4. Beta-lactamase BBa_K1189008
   </TD><TD>Blac-F (61)
   </TD><TD>Blac-R (58)
   </TD><TD>Beta-lactamase 
   </TD><TD>822
   </TD></TR></TBODY></TABLE><P>
Second PCR using PCR product from the first PCR as a template.
</P><TABLE><TBODY><TR><TD>Template
   </TD><TD>Primer 1 
   </TD><TD>Primer 2
   </TD><TD>Product
   </TD><TD>Size(bp)
   </TD></TR><TR><TD>1. sfGFP
   </TD><TD>GFP-F
   </TD><TD>GS-His-R
   </TD><TD>sfGFP-His
   </TD><TD>772
   </TD></TR><TR><TD>2. NanoLuc
   </TD><TD>Nluc-F 
   </TD><TD>GS-His-R
   </TD><TD>NanoLuc-His
   </TD><TD>579
   </TD></TR><TR><TD>3. mCherry
   </TD><TD>mCH-F
   </TD><TD>GS-His-R
   </TD><TD>mCherry-His
   </TD><TD>762
   </TD></TR><TR><TD>4. Beta-lactamase
   </TD><TD>Blac-F
   </TD><TD>GS-His-R
   </TD><TD>Beta-lactamase-His 
   </TD><TD>846
   </TD></TR></TBODY></TABLE><P><STRONG>Q5 PCR reaction components</STRONG></P><TABLE><TBODY><TR><TD>Components
   </TD><TD>50 ¬µl reaction
   </TD><TD>Final concentration
   </TD></TR><TR><TD>Q5 High-Fidelity 2X Master Mix
   </TD><TD>25 ¬µl
   </TD><TD>1X
   </TD></TR><TR><TD>Forward primer (10 ¬µM)
   </TD><TD>2.5 ¬µl
   </TD><TD>0.5 ¬µM
   </TD></TR><TR><TD>Reverse primer (10 ¬µM)
   </TD><TD>2.5 ¬µl
   </TD><TD>0.5 ¬µM
   </TD></TR><TR><TD>Template
   </TD><TD>1 ¬µl
   </TD><TD>&lt;250ng DNA
   </TD></TR><TR><TD>MiliQ H<SUB>2</SUB>O
   </TD><TD>To 50 ¬µl
   </TD><TD>-
   </TD></TR></TBODY></TABLE><P><STRONG>Q5 PCR temperature profile </STRONG></P><TABLE><TBODY><TR><TD>Step
   </TD><TD>Temperature
   </TD><TD>time
   </TD><TD>Cycle
   </TD></TR><TR><TD>Initial denaturation 
   </TD><TD>98 <SUP>‚ó¶</SUP>C
   </TD><TD>30 sec
   </TD><TD>-
   </TD></TR><TR><TD>Denaturation 
   </TD><TD>98 <SUP>‚ó¶</SUP>C
   </TD><TD>15 sec
   </TD><TD rowspan="3">35 Cycles
   </TD></TR><TR><TD>Annealing 
   </TD><TD>55 <SUP>‚ó¶</SUP>C 
   </TD><TD>25 sec
   </TD></TR><TR><TD>Extension
   </TD><TD>72 <SUP>‚ó¶</SUP>C
   </TD><TD>45 sec
   </TD></TR><TR><TD>Final Extension
   </TD><TD>72 <SUP>‚ó¶</SUP>C
   </TD><TD>5 min
   </TD><TD>-
   </TD></TR><TR><TD>Hold 
   </TD><TD>16 <SUP>‚ó¶</SUP>C
   </TD><TD>‚àû
   </TD><TD>-
   </TD></TR></TBODY></TABLE><P><STRONG>Gel result</STRONG></P><P>
1. sfGFP-His Success
</P><P>
2. NanoLuc-His Success
3. mCherry-His Success
4. Beta-lactamase-His Success
</P><P>
Purify sfGFP-His, NanoLuc-His, mCherry-His, Beta-lactamase-His, EspD, and Map20 PCR product using E.Z.N.A.¬Æ Cycle Pure Kit.
</P><H4>Nat, Eric</H4><H2>Restriction enzyme digestion ‚Äì 19/07/18</H2><P><STRONG>Digestion</STRONG></P><TABLE><TBODY><TR><TD>DNA
   </TD><TD>Restriction enzyme 1
   </TD><TD>Restriction enzyme 2
   </TD></TR><TR><TD>pBAD
   </TD><TD>SpeI
   </TD><TD>PstI
   </TD></TR><TR><TD>Map20
   </TD><TD>XbaI
   </TD><TD>AgeI
   </TD></TR><TR><TD>mCherry
   </TD><TD>XmaI
   </TD><TD>PstI
   </TD></TR><TR><TD>sfGFP
   </TD><TD>XmaI
   </TD><TD>PstI
   </TD></TR><TR><TD>Beta-lactamase
   </TD><TD>XmaI
   </TD><TD>PstI
   </TD></TR><TR><TD>NanoLuc
   </TD><TD>XmaI
   </TD><TD>PstI
   </TD></TR></TBODY></TABLE><P><STRONG>25 Œºl restriction enzyme digestion reaction components</STRONG></P><P>
10X Cutsmart‚Ñ¢ buffer 		2.5 Œºl
</P><P>
DNA 					250 ng
</P><P>
20U/Œºl Restriction enzyme 	0.5 Œºl
</P><P>
H<SUB>2</SUB>0 					to 25 Œºl
</P><P><STRONG>Temperature profile</STRONG></P><P>
37 <SUP>‚ó¶</SUP>C					30 min
</P><P>
80 <SUP>‚ó¶</SUP>C 					20 min
</P><P>
Digestion products are purified by E.Z.N.A.¬Æ Cycle Pure Kit.
</P><H4>Selma, Eric</H4><H2>Ligation and transformation ‚Äì 20/07/18</H2><P>
The digested plasmids and genes were ligated together in the four reactions below
</P><P><STRONG>10 Œºl Ligation reaction components</STRONG></P><P>
Vector DNA					20-50 ng
Insert DNA					2X - 4X molar ratio compared to vector
10U/Œºl T4 Ligase				0.5 Œºl
</P><P>
T4 ligase buffer				1 Œºl
H<SUB>2</SUB>O						to 10 Œºl
</P><TABLE><TBODY><TR><TD>Vector
   </TD><TD>Insert 1
   </TD><TD>Insert 2
   </TD></TR><TR><TD>Ligation I
   </TD><TD>pBAD
   </TD><TD>Map20
   </TD><TD>mCherry
   </TD></TR><TR><TD>Ligation II
   </TD><TD>pBAD
   </TD><TD>Map20
   </TD><TD>sfGFP
   </TD></TR><TR><TD>Ligation III
   </TD><TD>pBAD
   </TD><TD>Map20
   </TD><TD>Íûµ-lactamase
   </TD></TR><TR><TD>Ligation IV
   </TD><TD>pBAD
   </TD><TD>Map20
   </TD><TD>NanoLuc
   </TD></TR></TBODY></TABLE><P>
Ligation reactions are performed at 16 <SUP>‚ó¶</SUP>C for 1 hour.
</P><P>
1 Œºl of Ligation products were transformed into Mach1 E. coli competent cell according to <A href="http://parts.igem.org/Help:Protocols/Transformation">iGEM transformation protocol</A>.
</P><H4>Selma, Eric</H4><H2>Colony PCR ‚Äì 24/07/18</H2><P>
Ligation I, II, III grow only a few colonies while ligation IV give no colony. Colony PCR was performed to confirm reporter protein insertion.
</P><P><STRONG>10 Œºl Colony PCR reaction</STRONG></P><TABLE><TBODY><TR><TD>Components
   </TD><TD>10 ¬µl reaction
   </TD><TD>Final concentration
   </TD></TR><TR><TD>Q5 High-Fidelity 2X Master Mix
   </TD><TD>5 ¬µl
   </TD><TD>1X
   </TD></TR><TR><TD>Forward primer (10 ¬µM)
   </TD><TD>0.5 ¬µl
   </TD><TD>0.5 ¬µM
   </TD></TR><TR><TD>Reverse primer (10 ¬µM)
   </TD><TD>0.5 ¬µl
   </TD><TD>0.5 ¬µM
   </TD></TR><TR><TD>Template
   </TD><TD>E. coli cells
   </TD><TD>&lt;250ng DNA
   </TD></TR><TR><TD>MiliQ H<SUB>2</SUB>O
   </TD><TD>4 ¬µl
   </TD><TD>-
   </TD></TR></TBODY></TABLE><P><STRONG>Colony PCR temperature profile </STRONG></P><TABLE><TBODY><TR><TD>Step
   </TD><TD>Temperature
   </TD><TD>time
   </TD><TD>Cycle
   </TD></TR><TR><TD>Initial denaturation 
   </TD><TD>98 <SUP>‚ó¶</SUP>C
   </TD><TD>30 sec
   </TD><TD>-
   </TD></TR><TR><TD>Denaturation 
   </TD><TD>98 <SUP>‚ó¶</SUP>C
   </TD><TD>15 sec
   </TD><TD rowspan="3">35 Cycles
   </TD></TR><TR><TD>Annealing 
   </TD><TD>55 <SUP>‚ó¶</SUP>C 
   </TD><TD>25 sec
   </TD></TR><TR><TD>Extension
   </TD><TD>72 <SUP>‚ó¶</SUP>C
   </TD><TD>45 sec
   </TD></TR><TR><TD>Final Extension
   </TD><TD>72 <SUP>‚ó¶</SUP>C
   </TD><TD>5 min
   </TD><TD>-
   </TD></TR><TR><TD>Hold 
   </TD><TD>16 <SUP>‚ó¶</SUP>C
   </TD><TD>‚àû
   </TD><TD>-
   </TD></TR></TBODY></TABLE><P>
Colony PCR result
</P><P>
Note: Gel was overheated and image quality is poor. However, it is interpretable.
</P><P>
Positive results are achieved from colony I5, II1, and II2. No positive colonies from ligation III (pBAD-Map20-Íûµ-lactamase). 
</P><P>
The positive colonies I5, II1, and II2 were picked and cultured in LB broth+chloramphenicol.
</P><P>
Repeat transformation of ligation III and IV into Mach1 E. coli. The repeated transformants are called IIIB and IVB. 
</P><H4>Nat</H4><H2>Plasmid extraction ‚Äì 25/07/18</H2><P>
Plasmid extraction from cells with plasmid I5, II1, and II2 was done using E.Z.N.A.¬Æ plasmid mini kit. DNA Concentrations were measured using Nanodrop. Purified I5, II1, and II2 plasmids were sent to sequencing with Map20-F2 and GS-His-R primers. 
</P><P>
Transformant IIIB and IVB grow few colonies. Perform colony PCR on 4 colonies from IIIB and 2 colonies from IIB.
</P><P>
Colony PCR result
</P><P>
Positive results are achieved from colony IIIB2, IIIB3, and IVB1. All positive colonies were picked and culture overnight in LB broth+chloramphenicol.
</P><H4>Eirikur</H4><H2>USER cloning of EspD and chaperone cassette‚Äì 26/07/18</H2><P>
Plasmid extraction from overnight cultured IIIB2, IIIB3, and IVB1 was done using E.Z.N.A.¬Æ plasmid mini kit. DNA Concentrations were measured using Nanodrop. The purified plasmids were sent to sequencing with Map20-F2 and GS-His-R primers.
</P><P>
pBAD, pTac, synthetic CesF-CesT chaperone cassette, and EspD are PCR with USER primers.
</P><P>
PCR reactions are performed with X7 polymerase which is not inhibited by Uracil-containing primers.
</P><TABLE><TBODY><TR><TD>Template
   </TD><TD>Primer 1 
   </TD><TD>Primer 2
   </TD><TD>Product
   </TD><TD>Size(bp)
   </TD></TR><TR><TD>1.pBAD (BBa_K228005)
   </TD><TD>AraC-UF
   </TD><TD>AraC-UR
   </TD><TD>Linearized pBAD
   </TD><TD>3351
   </TD></TR><TR><TD>2. pTac (BBa_K731500)
   </TD><TD>LacO-UF 
   </TD><TD>LacO-UR
   </TD><TD>Linearized pTac
   </TD><TD>3342
   </TD></TR><TR><TD>3. SyntheticCesF-CesT chaperone cassette
   </TD><TD>CesF-UF
   </TD><TD>CesT-UR
   </TD><TD>Chaperone cassette
   </TD><TD>965
   </TD></TR><TR><TD>4. EspD
   </TD><TD>EspD-UF
   </TD><TD>EspD-UR
   </TD><TD>EspD
   </TD><TD>1180
   </TD></TR></TBODY></TABLE><P>
Gel result
</P><P>
Purify the PCR products with EZNA Cycle Pure Kit.
</P><H4>Nat</H4><H2>USER cloning and transformations of pBAD-Chaperone and pTac-EspD ‚Äì 27/07/18</H2><P>
USER cloning of pBAD + Chaperone cassette to generate pBAD-Chaperone plasmid Mach1 E. coli.
</P><P>
USER cloning of pTac + EspD to generate pTac-EspD plasmid and transform into DH5Œ± E. coli.
</P><P><STRONG>USER reaction components</STRONG></P><P>
Vector DNA 			1 Œºl
</P><P>
Insert DNA 			1 Œºl
</P><P>
10X Cutsmart buffer 	0.5 Œºl
</P><P>
DpnI 				0.3 Œºl
</P><P>
10X USER enzyme mix 	0.5 Œºl
</P><P>
H20 				0.7 Œºl
</P><P><STRONG>USER reaction temperature profile</STRONG></P><P>
37 ‚ÑÉ 				20 min
</P><P>
25 ‚ÑÉ 				30 min
</P><P>
16 ‚ÑÉ 				hold
</P><H4>Nat</H4><H2>Colony PCR pTac-EspD and pBAD-Chaperone ‚Äì 28/07/18</H2><P>
Colony PCR pTac-EspD and pBAD-Chaperone and pick the correct clone into LB broth with chloramphenicol.
</P><H4>Nat</H4><H2>pTac-EspD and pBAD-Chaperone plasmid extraction and  ‚Äì 29/07/18</H2><P>
Plasmid extraction from an overnight culture of pTac-EspD and pBAD-Chaperone was done using E.Z.N.A.¬Æ plasmid mini kit. DNA Concentrations were measured using Nanodrop. The purified plasmids were sent to sequencing with a set of sequencing primers.
</P><H4>Nat</H4><H2>Construction of reporter plasmids with chaperone cassette  ‚Äì31/07/18</H2><P>
Sequencing result of pBAD-chaperone confirms a correct sequence. Reporter genes and the Map20 signal was inserted into the pBAD-chaperone vector, using the same method as described previously in date 19/07/18 - 24/07/18.
</P><H4>Nat</H4><H2>Colony PCR Reporter plasmid with chaperone cassette ‚Äì01/08/18</H2><P>
Correct constructs of the reporter plasmids with chaperone cassette are identified by colony PCR as described previously. The correct colonies are picked into LB + chloramphenicol and incubate O/N in 37 C. 
</P><H4>Nat</H4><H2>Plasmid extraction and sequencing ‚Äì02/08/18</H2><P>
Reporter plasmids with chaperone are extracted using E.Z.N.A.¬Æ plasmid mini kit. DNA Concentrations were measured using Nanodrop. The purified plasmids were sent to sequencing with Map20-F2 and GS-His-R primers.
</P><H4>Nat</H4><H2>Design a new primer set to clone reporter genes without signal sequence ‚Äì25/08/18</H2><P>
Reporter plasmid without signal is needed. Forward primers for mCherry, sfGFP, Íûµ-lactamase, and Nanoluc are designed to contain RBS and XbaI restriction site to facilitate cloning.
</P><P>
This diagram demonstrates a plan to insert a reporter gene without signal sequence into a pBAD vector without chaperone cassette.
</P><H4>Nat</H4><H2>Construction of reporter plasmids without signal sequence ‚Äì27/08/18</H2><P>
mCherry, sfGFP, and Beta-lactamase are PCR  from previous reporter plasmid and inserted into pBAD and pBAD-Chaperone vector using restriction-ligation as designed. 
</P><H4>Nat</H4><H2>Colony PCR Reporter plasmid without signal ‚Äì29/08/18</H2><P>
Correct constructs of the reporter plasmids are identified by colony PCR as described previously. The correct colonies are picked into LB + chloramphenicol and incubate O/N in 37 C. 
</P><H4>Nat</H4><H2>Plasmid extraction and sequencing ‚Äì30/08/18</H2><P>
Reporter plasmids with chaperone are extracted using E.Z.N.A.¬Æ plasmid mini kit. DNA Concentrations were measured using Nanodrop. The purified plasmids were sent to sequencing.
</P></DIV><DIV id="tabs-3"><H1>Liposome injection experiments.</H1><H1>Week 40</H1><H2>Overnight of strains for liposome experiment 0,1 ‚Äì 4/10/18</H2><P>
GFP			I<SUP>+</SUP>S<SUP>+</SUP>C<SUP>+</SUP>, I<SUP>-</SUP>S<SUP>-</SUP>C<SUP>-</SUP>
mCherry		I<SUP>+</SUP>S<SUP>+</SUP>C<SUP>+</SUP>, I<SUP>-</SUP>S<SUP>-</SUP>C<SUP>-</SUP></P><P>
Beta-lactamase	I<SUP>+</SUP>S<SUP>+</SUP>C<SUP>+</SUP>, I<SUP>-</SUP>S<SUP>-</SUP>C<SUP>-</SUP></P><P>
Nanoluc		I<SUP>+</SUP>S<SUP>+</SUP>C<SUP>+</SUP>, I<SUP>-</SUP>S<SUP>-</SUP>C<SUP>-</SUP></P><H4>Attila</H4><H2>Liposome experiment 0,1 ‚Äì 5/10/18</H2><H4>Liposome experiment 0,1</H4><H4>Attila</H4><H2>Overnight for liposome experiment 1 ‚Äì 7/10/18</H2><P>
GFP, all combinations of strains
</P><H4>Attila.</H4><H1>Week 41 </H1><H2>Liposome experiment 1 ‚Äì 8/10/18</H2><P>
liposome experiment 1.
</P><H4>Attila &amp; nuttawat</H4><H2>Overnight cultures for liposome experiment 2 ‚Äì 10/10/18</H2><P>
mCherry, all combinations of strains.
</P><H4>Attila</H4><H2>Liposome experiment 2 ‚Äì 11/10/18</H2><P>
Liposome experiment 2
</P><P>
Attila
</P><P>
Western blot for experiment 1 &amp; 2 part 1/2.
</P><H4>Attila &amp; Nuttawat</H4><H2>Liposome experiment 2 ‚Äì 12/10/18</H2><P>
Western blot 2/2.
</P><H4>Attila &amp; Nuttawat</H4><H1>Week 42</H1><H2>Overnight for liposome experiment 3 ‚Äì 15/10/18</H2><P>
mCherry, all combinations of strains.
</P><P>
GFP, all combinations of strains
</P><H4>Attila</H4><H2>Liposome experiment 3 ‚Äì 16/10/18</H2><P>
Liposome experiment 3
</P><H4>Attila</H4><P>
Fluorescence Confocal microscopy
</P><P>
Lasse
</P><P>
Western blot part 1/2
</P><P>
Attila, Nuttawat, Lasse
</P><H2>Liposome experiment 3 ‚Äì 17/10/18</H2><P>
Western blot part 2/2
</P><H4>Lasse &amp; Nuttawat</H4></DIV><DIV id="tabs-4"><H1>Protoplast experiment with plants</H1><H2>Week 36</H2><P>
 Preparation of pilot experiment ‚Äì 06/09/18
</P><UL><LI>Picked colonies from plates with the relevant strains and started an overnight liquid culture. <EM>Note: </EM>The colonies picked were from chose three SIEC strains, one containing GFP, another mCherry and then Beta-Lactamase to serve as a negative control.
</LI><LI>Looked at the protocol which  the 2011 Hokkaido iGEM team used for their onion experiment and adjusted it to our strains.</LI></UL><I>Stephanie and Eir√≠kur</I><H3>Further preparation for pilot experiment ‚Äì 07/09/18</H3><OL><LI>Started a new overnight culture of cells (from the 6th) in induction media containing IPTG and Arabinose. 
</LI><LI>Prepared 1L of MgM-MES buffer pH5.0 using the following recipie:</LI></OL><P><STRONG><EM>MgM-MES buffer pH5.0:</EM></STRONG></P><P><EM>170mM MES</EM></P><P><EM>5mM KCl</EM></P><P><EM>75mM (NH4)2SO4</EM></P><P><EM>0.5mM K2SO4</EM></P><P><EM>8uM MgCl2</EM></P><P><EM>38mM Glycerol</EM></P><P><EM>0.1% Casaminoacids</EM></P><P>
Fill up to 1L of water and autoclave to sterilize.
</P><I>Stephanie and Eir√≠kur</I><H2>Week 37</H2><H3>Preparation of pilot experiment ‚Äì 10/09/18</H3><P>
Due to a delay because of missing mannitol, two new overnight cultures were started with the strains mentioned above. The two cultures were made differently to see if there is a difference in time of the Induction of Injectisome production.
</P><P><STRONG>Culture 1</STRONG>(mCherry,GFP, beta-lactamase strains): Prepared using LB media with Chloramphinicol and 1%arabinose(to induce the production of all the reporter proteins).
</P><P><STRONG>Culture 2:</STRONG>(mCherry,GFP, beta-lactamase strains): Prepared using LB media with Chloramphinicol ,1%arabinose(to induce the production of all the reporter proteins) and 0.1mM IPTG(To induce the production of the injectisome)
</P><I>Stephanie and Eir√≠kur</I><H3>Pilot experiment ‚Äì 11/09/18</H3><P>
Decision was made to use D-Sorbitol instead of Manitole
</P><UL><LI>The overnight cultured cells were spun down  at 3000rpm, for 10 min at 35C. Supernatant removed. 
</LI><LI>5mL MgM-MES (pH5.0), Chloramphenicol, L-arabinose and IPTG were added to the bacterial pellet and resuspended by vortexing.cells were culteured at 37C, shaking at 200rpm for 4 hours.
</LI><LI>While cultures were incubating, onion sheets were prepared using this <A href="https://static.igem.org/mediawiki/2018/8/82/T--UCopenhagen--onionProt.pdf" target="_blank">protocol</A></LI><LI>When cultures had incubated we spun down the culture at 3000rpm, for 10 min at 25C, supernatant was removed
</LI><LI>Pellet was resuspended with 2 ml of MgM-MES (pH5.0) containing 0.4 M Mannitol and spun down at  3000rpm, for 10 min at 25C, supernatant was removed. This was repeated 3 times.
</LI><LI>Pellet was resuspended with 2 ml of MgM-MES (pH5.0) containing 0.4 M Sorbitol, chloramphenicol, L-arabinose and IPTG. 
</LI><LI>Light absorbance was measured at 600nm using a spectrophotometer and the concentration of cells was adjusted to ŒîOD600 = 0.06 by diluting with the same medium mentioned above. 
</LI><LI>To infect the onion cells 500ul of the culture fluid was put onto the pre-treatd onion cell-sheets. The whole thing was left at room temperature. We made two samples for each culture, one was left for 2 hours the other one was left overnight</LI></UL><UL><LI>Cells were observed under a fluorescent microscope</LI></UL><I>Stephanie and Eir√≠kur</I><H3>Onion preparation‚Äì 25/09/18</H3><UL><LI>7 Fresh epidermal sheets of onion cells were cut out and put on microscope-slides</LI></UL><UL><LI>The onion cell sheets were attached to the microscope slide with agarose</LI></UL><UL><LI>A solution containing 1% Cellulase, 0.1%Pectolyase, 0.4M Sorbiltole-D and MgM-MES buffer, was put onto the onion sheets</LI></UL><UL><LI>1 sample was a control with no solution, the rest(6) were observed after 10min, 20min and 45min(duplicates were made for each time interval), under a microscope to see how the onion cells changed according to the incubation time of the enzymes</LI></UL><TABLE><TBODY><TR><TD><EM>t=0</EM></TD><TD><EM>T=20</EM></TD><TD><EM>t=45</EM></TD></TR></TBODY></TABLE><UL><LI>There were no conclusive results, might be because the incubation time was too short, the buffers were old or the enzymes did not work as suspected</LI></UL><H3>Further preparation of onion sheets and tobacco plant observations‚Äì 27/09/18</H3><P>
New buffers and solutions were made:
</P><UL><LI>1 liter MgM-MES buffer,
</LI><LI>500ml 0.4M Sorbitol-D solution in  MgM-MES buffer, 
</LI><LI>50ml of 1%Cellulase and 0.2% pectolyase in a 0.4M Sorbitol-D solution in MgM-MES	</LI><LI>7 Fresh epidermal sheets of onion cells were cut out and put on microscope-slides</LI></UL><UL><LI>The onion cell sheets were attached to the microscope slide with agarose</LI></UL><UL><LI>A solution containing 1% Cellulase, 0.1%Pectolyase, 0.4M Sorbiltol-D and MgM-MES buffer, was put onto the onion sheets</LI></UL><UL><LI>1 sample was a control with no solution, the rest(6) were observed after 10min, 20min and 60min(duplicates were made for each time interval), under a microscope to see how the onion cells changed according to the incubation time of the enzymes</LI></UL><UL><LI>A single tobacco plant was obtained for the purpose of trying to observe protoplasts
</LI><LI>The cuticule was scraped off as best as possible
</LI><LI>A solution containing 1% Cellulase, 0.1%Pectolyase, 0.4M Sorbiltol-D and MgM-MES buffer, was put into a petri dish, the piece of leaf without it¬¥s cuticle was put on top of the solution and left there for an hour
</LI><LI>The treatment was done on a big leaf from the plant, a smaller leaf, and a tiny leaf, to see of there was any visible difference in amount of protoplasts
</LI><LI>The leafs and its solution were observed under through a microscope which revealed burst cells and very few protoplast</LI></UL><H3>Osmolarity test on  onion sheets and tobacco plant ‚Äì 02/10/18</H3><P>
We had been using 0.4M solution of Sorbitol through the entire experiment so we decided to test different concentrations, to see if it made any difference. We decided furthermore to switch from Sorbitol to Mannitol since a new batch of mannitol just arrived
</P><UL><LI>A 12 well culture plate was obtained. Each well was filled with a solution containing MgM-MES buffer, 0.1% pectolyase, 1%cellulase and Mannitol in 9 different concentrations ranging from 0.4M-0.8M. 
</LI><LI>The cuticle of the tobacco leaves were scraped of and but downside-down into each of the wells
</LI><LI>The first 9 wells contained 0.4;0.45;0.5;0.55;0.6;0.65;0.7;0.75 and 0.8 M Mannitol,respectively, plus the MgM-MES and enzymes. The last three contained onion cell sheets in 0.6,0.7 and 0.8 M Mannitol with the solution mentioned above.
</LI><LI>Samples were taken after 3 hours and looked at under a fluorescent microscope.
</LI><LI>We saw tobacco protoplasts in most samples but the sample with the highest concentration protoplast appeared to be the one containing 0.6-0.65 M of Mannitol
</LI><LI>From looking at the onion cells, they appeared to be best treated with a 0.8M of Mannitol
</LI><LI>We decided to move forward with 0.6M Mannitol for the tobacco leaves and 0.8M for the Onion cells</LI></UL><H1>Onion and tobacco assay ‚Äì 05/10/18</H1><I>8/9/18 - Stephanie &amp; Eirikur</I><P>
Onion assay pilot experiment: 
</P><P>
Delayed due to missing Mannitol
</P><OL><LI>Spin down the cultured cells at 3000rpm, for 10 min at 35C. Remove supernatant. 
</LI><LI>Add 5mL MgM-MES (pH5.0), Chloramphenicol, L-arabinose and IPTG to the bacterial pellet. Resuspend it using vortex mixer. Culture cells at 37C, shaking at 200rpm for 4 hours. This step is required because genes encoding salmonella T3SS are expressed in acidic environment. 
</LI><LI>While cultures are incubating, prepare onion sheets:
</LI><LI>Cut an onion into four pieces.
</LI><LI>Take out a fresh sheets of epidermal cells (second or third layer from the surface of onion). 
</LI><LI>Cut the layer into rectangular shape (about 1.5cm x 3cm) 
</LI><LI>Tear off a cell sheet from the inner surface of the layer.
</LI><LI>Put the sheet onto a slide glass at back face up because the upper face deflects enzyme solution, and fix the edge of the sheet onto slide glass with 1% agarose gel. 
</LI><LI>Fill a petri dish with 1% cellulase and 0.1% pectolyase in 0.4 M Mannitol solution (pH7.0), then submerge the cell sheet fixed on slide glass into the enzyme solution. Incubate for 8 min at RT.
</LI><LI>Prepare 3 petri dishes filled with MgM-MES pH5.0 containing 0.4 M Mannitol. After processing, wash cells stepwise soaking in these petri dishes to remove enzyme solution. 
</LI><LI>Remove excess moisture, and transfer the sample to another petri dish for infection.
</LI><LI>Spin down the culture fluid at 3000rpm, for 10 min at 25C. Remove supernatant. 
</LI><LI>Resuspend the pellet with 2 ml of MgM-MES (pH5.0) containing 0.4 M Mannitol, then spin down the culture fluid at 3000rpm, for 10 min at 25C. Remove supernatant. Repeat this step 3 times. (This is done to remove the toxic substances that are produced by <EM>E. coli</EM> and adjust the osmotic pressure to that in onion cells) 
</LI><LI>Resuspend the pellet with 2 ml of MgM-MES (pH5.0) containing 0.4 M Mannitol, Tetracycline, chloramphenicol and L-arabinose. 
</LI><LI>Measure the light absorbance at 600 nm using a spectrophotometer, then adjust the concentration of cells to ŒîOD600 = 0.06 by diluting with the same medium mentioned above. 
</LI><LI>To infect onion cells with <EM>E. coli </EM>add 500 ul of the culture fluid onto pre-treated onion cell-sheets. Leave at RT in petri dish (Prevent drying) 
</LI><LI>Remove the bacterial culture fluid carefully by micropipette so that the cell-sheets are not to be torn. Observe the cells under a fluorescencent microscope.</LI></OL><H2>Week 41</H2><H3>Remaking protoplast and preparing bacteria‚Äì 10/10/18</H3><P>
We needed to remake protoplast in 5,0 pH and induce older bacteria samples to mix and check for secretion. 
</P><P><EM>Victoria </EM></P><P>
Collecting protoplasts made 09/10/18
</P><OL><LI>Identify which well contains protoplasts for onion and tobacco samples with a microscope 
</LI><LI>Add 2 ml sample to an eppendorf tube
</LI><LI>Spin down samples at low tempo for 3 min
</LI><LI>Remove supernatant 
</LI><LI>Flick tubes to mix
</LI><LI>Check new samples for protoplast using a microscope
</LI><LI>Steps 1-6 were repeated 2x, with batch number 1 containing the most protoplasts </LI></OL><H4>Inducing bacteria</H4><P>
Samples
</P><P>
mCherry: I-S-C-, I+S+C+
</P><P>
GFP: I-S-C-, I+S+C+
</P><P>
EspD: 
</P><OL><LI>bacteria  (older sample from the fridge, since no new overnight culture was started beforehand) was spun down and pellet resuspended in 5 ml fresh LB+CaM media
</LI><LI>400 microliter resuspended culture added to 10 ml LB+CaM + 500 microliter 20 % Arabinose + 2mM IPTG to induce bacteria 
</LI><LI>Incubation </LI><LI>1st batch start: 12:53
 </LI><LI>2nd batch start: 13:20</LI></OL><H4>Remaking onion and tobacco protoplasts at pH 5,0 </H4><P>
Prepared 500 ml of MgM-MES buffer pH5.0 using the following recipe:
</P><P><STRONG><EM>MgM-MES buffer pH5.0:</EM></STRONG></P><P><EM>170mM MES: 19,66 g</EM></P><P><EM>5mM KCl: 0,19 g</EM></P><P><EM>75mM (NH4)2SO4: 4,96 g</EM></P><P><EM>0.5mM K2SO4: 0,044 g</EM></P><P><EM>8uM MgCl2: 0,068 g</EM></P><P><EM>38mM Glycerol: 1,4 ml</EM></P><P><EM>0.1% Casaminoacids: 0,5 g</EM></P><OL><OL><LI>pH was adjusted with 5,0 M KOH
</LI><LI>Dilute buffer to 500 ml
</LI><LI>0,8 M and 0,6 M Mannitole buffer was prepared (250 ml 0,6 M Mannitole = 27,5, 250 ml 0,6 M Mannitole = 36,5 g )
</LI><LI>Peel inner epidermis from onion and tobacco 
</LI><LI>Leave in Mannitole buffer for min 4. hours
</LI><LI>Mix bacteria and protoplasts 
</LI><LI>Incubate overnight </LI></OL><H2>Week 41</H2><H3>Washing protoplasts to remove bacteria ‚Äì 11/10/18</H3><P>
We needed to separate bacteria from protoplasts to use a fluorescent microscope to check for secretion into protoplasts. It turned out washing by spinning down protoplasts, removing supernatant containing bacteria and resuspending pellet was an inefficient way to do this.
</P><P><EM>Victoria</EM></P><P>
Note: Only the onion protoplast samples were used
</P><OL><LI>Mix samples gently by flicking
</LI><LI>Centrifuge for 3 min at 150 RCF (or until pellet appear)
</LI><LI>Remove supernatant
</LI><LI>Resuspend pellet in 200 microliter 0,8 M mannitole buffer </LI><LI>Check samples in microrscope to ensure protoplast are stable and kept in sample
</LI><LI>All samples were washed 6 times by following step 1-4 </LI><LI>Note: This wasn't sufficient to remove bacteria entirely
 </LI><LI>Note: Protoplast seemed to shrink during the time of the experiment
</LI><LI>Check samples in fluorescent microscope</LI></OL></OL><H3> Using Lyzozyme to remove bacteria ‚Äì 11/10/18</H3><P>
We wanted to see if digesting the bacteria would be more efficient than washing the samples. This experiment showed that separating protoplast from bacteria was innificient. 
</P><P>
Note: This experiment used the tobacco samples left over from the day before
</P><P><EM>Victoria</EM></P><OL><LI>Spin tobacco 3 min 150 RCF
</LI><LI>Remove supernatant 
</LI><LI>Add 200 microliter of lyzozyme solution to samples </LI><LI>Lyzozyme solution: 10 g Lysozyme from chicken egg white,1 ml Mannitole buffer 
</LI><LI>Check samples every 30 min in microscope </LI><LI>Note: Bacteria did not get removed
<DIV id="tabs-5"><H1>Leakiness characterization</H1><H1>Week 36</H1><H2>Overnight of strains (1) ‚Äì 06/09/18</H2><P>
An overnight of the following strains was made: 
</P><P>
1: I+, S+, C+ mCherry
</P><P>
2: I+, S+, C- mCherry
</P><P>
3: I-, S+, C+ mCherry
</P><P>
4: I-, S-, C- mCherry
</P><P>
5: I-, S+, C- mCherry
</P><P>
6: I+, S-, C+ mCherry
</P><P>
7: I+, S+, C+ beta-lactamase (TEM1) (negative control for mCherry fluorescent measurement).
</P><H4>Nat</H4><H2>Induction of strains + sample taking over time (1) ‚Äì 07/09/18</H2><P>
The first experiment in relation to the examination of the injectisome leakiness was conducted with the strains from the overnight. We followed this <A href="https://static.igem.org/mediawiki/2018/1/14/T--UCopenhagen--LeakProt.jpeg">protocol</A> (protocol 1).  
</P><P>
The bacteria were induced with IPTG and arabinose and a sample was taking at T=0 for each strain. Thereafter a sample was taken every 2 hours. The samples were centrifuged and the supernatant transferred to another tube. The tubes with the pellet and the tubes with the supernatant were placed in the freezer. 
</P><H4>Attila, Nat and Lasse</H4><H2>Induction of strains + sample taking over time (1) ‚Äì 08/09/18</H2><P>
The last sample of this experiment, T=overnight, was taken for each strain. The samples were centrifuged and the supernatant transferred to another tube. The tubes with the pellet and the tubes with the supernatant were placed in the freezer. 
</P><H4>Attila, Nat and Lasse</H4><H1>Week 37</H1><H2>Overnight of strains (2) ‚Äì 13/09/18</H2><P>
An overnight of the following strains was made: 
</P><P>
1: I+, S+, C+ mCherry
</P><P>
2: I+, S+, C- mCherry
</P><P>
3: I-, S+, C+ mCherry
</P><P>
4: I-, S-, C- mCherry
</P><P>
5: I-, S+, C- mCherry
</P><P>
6: I+, S-, C+ mCherry
</P><P>
7: I+, S+, C+ beta-lactamase (TEM1) (negative control for mCherry fluorescent measurement). 
</P><H4>Nat</H4><H2>Induction of strains + sample taking over time (2) ‚Äì 14/09/18</H2><P>
The second experiment in relation to the examination of the injectisome leakiness was conducted with the strains from the overnight. We followed this <A href="https://static.igem.org/mediawiki/2018/1/14/T--UCopenhagen--LeakProt.jpeg">protocol </A>(protocol 2).
</P><P>
The bacteria were induced with IPTG and arabinose and a sample was taking at T=day0 for each strain. Thereafter the bacteria were placed at 16 <SUP>o</SUP>C. The T=day0 samples were centrifuged and the supernatant transferred to another tube. The tubes with the pellet and the tubes with the supernatant were placed in the freezer. 
</P><H4>Attila, Nat and Lasse</H4><H2>Induction of strains + sample taking over time (2) ‚Äì 15/09/18</H2><P>
A sample, T=day1, was taken from the tubes incubated at 16 C<SUP>O</SUP> for each strain. The T=day1 samples were centrifuged and the supernatant transferred to another tube. The tubes with the pellet and the tubes with the supernatant were placed in the freezer. 
</P><H4>Attila, Nat and Lasse</H4><H2>Induction of strains + sample taking over time (2) ‚Äì 16/09/18</H2><P>
A sample, T=day2, was taken from the tubes incubated at 16 C<SUP>O</SUP> for each strain. The T=day2 samples were centrifuged and the supernatant transferred to another tube. The tubes with the pellet and the tubes with the supernatant were placed in the freezer. 
</P><H4>Attila, Nat and Lasse</H4><H1>Week 38</H1><H2>Induction of strains + sample taking over time (2) ‚Äì 17/09/18</H2><P>
A sample, T=day3, was taken from the tubes incubated at 16 C<SUP>O</SUP> for each strain. The T=day3 samples were centrifuged and the supernatant transferred to another tube. The tubes with the pellet and the tubes with the supernatant were placed in the freezer. 
</P><H4>Attila, Nat and Lasse</H4><H2>Induction of strains + sample taking over time (2) ‚Äì 18/09/18</H2><P>
A sample, T=day4, was taken from the tubes incubated at 16 C<SUP>O</SUP> for each strain. The T=day4 samples were centrifuged and the supernatant transferred to another tube. The tubes with the pellet and the tubes with the supernatant were placed in the freezer. 
</P><H4>Attila, Nat and Lasse</H4><H2>Induction of strains + sample taking over time (2) ‚Äì 19/09/18</H2><P>
A sample, T=day5, was taken from the tubes incubated at 16 C<SUP>O</SUP> for each strain. The T=day5 samples were centrifuged and the supernatant transferred to another tube. The tubes with the pellet and the tubes with the supernatant were placed in the freezer. 
</P><H4>Attila, Nat and Lasse</H4><H2>Induction of strains + sample taking over time (2) ‚Äì 20/09/18</H2><P>
A sample, T=day6, was taken from the tubes incubated at 16 C<SUP>O</SUP> for each strain. The T=day6 samples were centrifuged and the supernatant transferred to another tube. The tubes with the pellet and the tubes with the supernatant were placed in the freezer. 
</P><H4>Attila, Nat and Lasse</H4><H2>Measurement of fluorescence of supernatant and pellet (2) ‚Äì 21/09/18</H2><P>
The fluorescence was measured of the samples of the supernatant and the pellet, for the experiment where samples were taken once a day for six days. The supernatant was measured directly whereas the pellet was first resuspended and diluted 10-fold using LB+CAM media.
</P><H4>Attila and Lasse</H4><H1>Week 39</H1><H2>Western blot of the samples (2) - 26/09/18</H2><P>
Western blot was conducted for the samples following this <A href="https://static.igem.org/mediawiki/2018/c/cd/T--UCopenhagen--WestAndSDS.pdf">protocol</A>.
</P><H4>Attila and Nat</H4><H1>Week 42</H1><H2>Western blot of the samples (1) - 16/10/2018</H2><P>
Western blot was conducted for the samples following this <A href="https://static.igem.org/mediawiki/2018/c/cd/T--UCopenhagen--WestAndSDS.pdf">protocol</A>.
</P><H4>Attila, Nat and Lasse</H4><H2>Western blot of the samples (1) - 17/10/2018</H2><P>
Western blot was conducted for the samples following this <A href="https://static.igem.org/mediawiki/2018/c/cd/T--UCopenhagen--WestAndSDS.pdf">protocol</A>.
</P><H4>Attila, Nat and Lasse</H4></DIV><DIV id="tabs-6"><H1>Experiments with supported lipid bilayers</H1><H2>Week 28 </H2><H3>Learning to make liposomes ‚Äì 12/07/18</H3><P>
We were introduced to the process of making liposomes by the nice people in Nikos Hatzakis lab, S√∏ren Schmidt-Rasmussen Nielsen and Mette Galsgaard Malle who gave us the <A href="https://static.igem.org/mediawiki/2018/6/6d/T--UCopenhagen--Protocol_Liposome_preparation_step_by_step_NH_lab.pdf">Liposome preparation protocol</A>. We made some tryout liposomes, that were flashed freezed and stored in -20 <STRONG>¬∞</STRONG>C.
</P><P><EM>Selma, Sofia and Attila</EM></P><H2>Week 34</H2><H3>Transforming SIEC strains with GFP plasmids ‚Äì 24/08/18</H3><P>
To visualize our bacteria in the microscope together with a membrane we needed it transformed with a GFP plasmid. We used plasmids from the distribution kit that functioned as Test Device 1 (BBa_J36400) and positive control (BBa_I20270) in the InterLab study.
</P><P>
Transformation as described in <A href="https://static.igem.org/mediawiki/parts/6/67/IGEM_Registry_-_Transformation_Protocol.pdf">Transformation protocol</A> until step 14, the following combinations of strain and plasmid was made:
</P></DIV></LI><LI>SIEC x BBa_I20270
</LI><LI>SIECŒîp1 x BBa_I20270</LI><P>
Plates were made from agar and LB with chloramphenicol (34 Œºg/mL) (CAM). Plates with bacteria spent 17 hours in the 37 <STRONG>¬∞</STRONG>C incubator before being kept in fridge.
</P><P><EM>Selma</EM></P><H2>Week 35</H2><H3>Testing transformed bacteria for fluorescence ‚Äì 27/08/18</H3><P>
To test if the transformation was successful fluorescence were measured on a plate reader. 
</P><P>
Colonies from transformation 24/08/18 was picked into 50 mL falcon tubes with 5 mL LB media (that also contained 1% arabinose). Incubated at 37 <STRONG>¬∞</STRONG>C at 130 rpm for 6 hours. Triplicates of each 100 ŒºL of culture was pipetted into wells of a 96 well-plate. Fluorescence and absorbance was measured. LB media was used as blank.
</P><P>
Glycerol stocks were made from the cultures.
</P><P><EM>Selma</EM></P><H3>Making lipid film stocks 1 ‚Äì 29/08/18</H3><P>
In order to make the liposomes for later experiments quickly and to ensure that the liposomes have the same composition, we made lipid film stocks from a lipid mastermix.
</P><P>
First, we weighed about 25 mg of powdered lipids.
</P><TABLE><TBODY><TR><TD><STRONG>Lipid</STRONG></TD><TD><STRONG>Mass (mg)</STRONG></TD></TR><TR><TD>DOPC
   </TD><TD>24,8
   </TD></TR><TR><TD>DOPS
   </TD><TD>25,2
   </TD></TR><TR><TD>SM
   </TD><TD>25*
   </TD></TR><TR><TD>Cholesterol
   </TD><TD>25*
   </TD></TR></TBODY></TABLE><P>
*lipids were obtained as powder in this amount from the manufacturer
</P><P>
Lipids were dissolved in 1 mL chloroform. 
</P><P>
To properly store the lipids, we used nitrogen flow to exchange air in the glass vials with nitrogen. Because some of the solvent evaporated during preparation due to the nitrogen flow, the lipid film stocks had to be redone 31/08/18.
</P><P>
Second, we preparred biotin stock 2. The biotin stock concentration was 0.5 Œºg/ŒºL (stock 1). 150 ŒºL of biotin stock 1 was added to 600 ŒºL chloroform to make biotin stock 2 (0.1 Œºg/ŒºL).
</P><P><EM>Selma and Lasse</EM></P><H3>Microscope tryout ‚Äì 30/08/18</H3><P>
We got an introduction to the microscope and saw our bacteria in the microscope for the first time. We used the confocal microscope in Associate Professor Nikos Hatzakis' laboratory. Introduction to the microscope was kindly performed by S√∏ren Schmidt-Rasmussen.
</P><H4>Bacteria incubation</H4><P>
We added 20 ŒºL of glycerol stock from the 27/08/18 to 5 mL LB media with CAM in 50 mL falcon tubes (SIEC and SIECŒîp1 with GFP, BBa_I20270).
</P><P>
Put in incubator at 37<STRONG>¬∞</STRONG>C at 130 rpm for 3 hours.
</P><H4>Preparation of supported lipid bilayer</H4><P>
Supported lipid bilayer was made as described in the <A href="https://www.chemistry.mcmaster.ca/moran-mirabal/resources/SUPPORTED-LIPID-BILAYER-PREPARATION-SOP-9-2015.pdf">Moran-Mirabel protocol</A>, step 8 to 12. For this first tryout, we used the liposomes made 12/07/18 (not right membrane composition). Microscopy slides were washed with Tris buffer. Membrane contained the dye Atto655.
</P><H4>Microscopy</H4><P>
Microscope slides were washed with the same media as used for cultivation of the bacteria. 100 ŒºL bacteria culture was added to a slide well. We used lasers of 488 nm and 635 for excitation. FRAP was used to see if the membrane was intact, which was not the case.
</P><P><EM>Selma, Attila, Lasse and Nat</EM></P><H3>Making lipid film stocks 2 ‚Äì 31/08/18</H3><P>
We wanted to make a mixture containing sphingomyelin (SM), dioleoylphosphatidylcholine (DOPC), dioleoylphosphatidylserine (DOPS), and cholesterol (Chl) was used for the supported lipid bilayer and liposomes in a ratio of 44:24:12:20 (SM:DOPC:DOPS:Chl).
</P><P>
Chloroform in stocks from 29/08/18 was evaporated with nitrogen flow. Masses of lipids left were weighed. 1 mL solvent was added to each lipid vial to make stock 1. 
</P><TABLE><TBODY><TR><TD><STRONG>Lipid</STRONG></TD><TD><STRONG>Mass (mg)</STRONG></TD></TR><TR><TD>DOPC
   </TD><TD>25,6*
   </TD></TR><TR><TD>DOPS
   </TD><TD>21
   </TD></TR><TR><TD>SM
   </TD><TD>24,4
   </TD></TR><TR><TD>Cholesterol
   </TD><TD>24,9
   </TD></TR></TBODY></TABLE><P>
*This number is higher than 29/08/18. This means that the concentration for DOPC is not exact. 
</P><P>
Dilutions of each lipid stock ( called &quot;Stock 2&quot;) were made:
</P><TABLE><TBODY><TR><TD><STRONG>Lipid</STRONG></TD><TD><STRONG>Volume of Stock 1 </STRONG></TD><TD><STRONG>Methanol</STRONG></TD><TD><STRONG>Chloroform</STRONG></TD></TR><TR><TD>DOPC
   </TD><TD>39 ŒºL**
   </TD><TD>0 ŒºL
   </TD><TD>960 ŒºL
   </TD></TR><TR><TD>DOPS
   </TD><TD>48 ŒºL
   </TD><TD>476 ŒºL
   </TD><TD>476 ŒºL
   </TD></TR><TR><TD>SM
   </TD><TD>41 ŒºL
   </TD><TD>480 ŒºL
   </TD><TD>480ŒºL
   </TD></TR><TR><TD>Cholesterol
   </TD><TD>40 ŒºL
   </TD><TD>0 ŒºL
   </TD><TD>960 ŒºL
   </TD></TR><TR><TD>Atto655 (0.5 Œºg/ŒºL)
   </TD><TD>20 ŒºL
   </TD><TD>0 ŒºL
   </TD><TD>980 ŒºL
   </TD></TR></TBODY></TABLE><P>
**If the mass had corresponded to what was measured 29/08/18 this would have been 40 ŒºL instead of 39 ŒºL.
</P><P>
Three mixtures were made from the stock 2.
</P><TABLE><TBODY><TR><TD><STRONG>Lipid, stock 2</STRONG></TD><TD><STRONG>Mix 1</STRONG></TD><TD><STRONG>Mix 2</STRONG></TD><TD><STRONG>Mix 3</STRONG></TD></TR><TR><TD>DOPC (1Œºg/ŒºL)
   </TD><TD>75 ŒºL
   </TD><TD>75 ŒºL
   </TD><TD>75 ŒºL
   </TD></TR><TR><TD>DOPS (1 Œºg/ŒºL)
   </TD><TD>39 ŒºL
   </TD><TD>39 ŒºL
   </TD><TD>39 ŒºL
   </TD></TR><TR><TD>SM (1 Œºg/ŒºL)
   </TD><TD>133 ŒºL
   </TD><TD>134 ŒºL
   </TD><TD>134 ŒºL
   </TD></TR><TR><TD>Cholesterol (1 Œºg/ŒºL)
   </TD><TD>31 ŒºL
   </TD><TD>31 ŒºL
   </TD><TD>31 ŒºL
   </TD></TR><TR><TD>Atto655 (0.01 Œºg/ŒºL)
   </TD><TD>27 ŒºL
   </TD><TD>27 ŒºL
   </TD><TD>0 ŒºL
   </TD></TR><TR><TD>Biotin (0.1 Œºg/ŒºL)
   </TD><TD>60 ŒºL
   </TD><TD>0 ŒºL
   </TD><TD>0 ŒºL
   </TD></TR></TBODY></TABLE><P>
Mixtures were aliquoted into glass vials to contain 0.2 Œºmol lipids:
</P><TABLE><TBODY><TR><TD><STRONG>Mix 1</STRONG></TD><TD><STRONG>Mix 2</STRONG></TD><TD><STRONG>Mix 3</STRONG></TD></TR><TR><TD>ŒºL mastermix to reach 0,2 Œºmol
   </TD><TD>37 ŒºL/vial
   </TD><TD>31 ŒºL/vial
   </TD><TD>28 ŒºL/vial
   </TD></TR></TBODY></TABLE><P>
Vials were kept in -20<STRONG>¬∞</STRONG>C.
</P><P><EM>Selma</EM></P><H2>Week 36</H2><H3>Making lipid film stocks 3 ‚Äì 05/09/18</H3><P>
To finish the lipid film stocks, solvent was evaporated with nitrogen flow for about 5 minutes. Then vials were centrifuged on a ScanVac for 15 minutes under vacuum. Vials containing lipid films were stored at -20 <STRONG>¬∞</STRONG>C.
</P><H4>Selma</H4><H3>Streaking bacteria ‚Äì 05/09/18</H3><P>
Bacteria from glycerol stock 27/08/18 (transformations 24/08/18) were streaked on LB agar plates with CAM (34 Œºg/mL). Incubated overnight and kept in fridge.
</P><P><EM>Selma</EM></P><H2>Week 37</H2><H3>Full protocol tryout 1 ‚Äì 12/09/18</H3><P>
We inoculated colonies from plates that were made the 05/09/18 in LB media with arabinose (1%), IPTG (0.1 mM) and CAM (34 Œºg/mL). The cultures were incubated for 2.5 hours at 37 <STRONG>¬∞</STRONG>C 160 rpm. Meanwhile lipid films were rehydrated, extruded through a 200 nm filter and underwent 10 cycles of freezing in nitrogen and heating in a water bath, see  <A href="https://static.igem.org/mediawiki/2018/6/6d/T--UCopenhagen--Protocol_Liposome_preparation_step_by_step_NH_lab.pdf">Liposome preparation protocol</A>. Liposomes were used to make a supported lipid bilayer on a microscope slide. The rest of the liposomes were kept in -80 <STRONG>¬∞</STRONG>C freezer.
</P><P>
After 2.5 hours of incubation, we measured OD of the bacterial culture (results below) and pipetted 100 uL of bacteria culture onto each slide with bilayer. This was incubated at 37 <STRONG>¬∞</STRONG>C in 3 hours.
</P><P>
After incubation, we looked at each slide in the microscope, taking pictures before and after washing with 100 uL media 3 times.
</P><P>See the full protocol <A href="https://static.igem.org/mediawiki/2018/4/4e/T--UCopenhagen--Protocol_SLB_Experiments_iGEM_2018.pdf">here</A></P><P><EM>Sofia and Selma</EM></P><DIV id="tabs-7"><H1>InterLab</H1><P>Throughout the interlab experiments, we followed the protocol provided by the measurement committee. The protocol can be seen <A href="https://static.igem.org/mediawiki/2018/0/09/2018_InterLab_Plate_Reader_Protocol.pdf" target="_blank">here</A>.</P><P>In the following notebook we have focused on what we have done and then linked to the interlab page on this wiki. We have chosen not to incorporate the data not used as final result however, we states the number of times we have done the experiments. If you want to read more about the different experiments you can read more here (link to wiki interlab page). </P><H2>Week 26</H2><H3>Calibration 1: LUDOX ‚Äì 27/06/18</H3><P>OD600 was measured for the four replicates, and this was performed three times for the LUDOX and one time for ddH2O. The data from one of these replicants were used to the submission as a final result. Description and final results of the experiment can be found here (2.1 LUDOX). </P><I>Selma, Attila, Eirikur and Lasse</I><H4>Calibration 2: Silica beads (1) ‚Äì 27/06/18</H4><P>OD600 was measured for the dilutions of silica beads. The results from this measurement was not used. Description and final results of the experiment can be found here (2.2 Silica beads). </P><I>Selma, Attila, Eirikur and Lasse</I><H4>Transformation of E. coli DH5 Œ± with the six test devices (1) ‚Äì 27/06/18</H4><P>Competent E. coli DH5 Œ± were transformed with six test devices obtained from the distribution kit. The protocol used for the transformation were the one produced by iGEM, see here http://parts.igem.org/Help:Protocols/Transformation. Changes from the protocol were:
- The use of LB media instead of soc in step 10. 
These transformants were not used. </P><I>Selma, Attila, Eirikur and Lasse</I><H4>Transformation of E. coli DH5 Œ± with the six test devices (2) ‚Äì 28/06/18</H4><P>Competent E. coli DH5 Œ± were transformed with six test devices obtained from the distribution kit. The protocol used for the transformation were the one produced by iGEM, see here http://parts.igem.org/Help:Protocols/Transformation. Changes from the protocol were:
- The use of LB media instead of soc in step 10. 
These transformants were used for the rest of the interlab experiments. </P><I> Selma, Attila, Eirikur and Lasse </I><H4>Calibration 3: Fluorescein (1)‚Äì 01/07/18</H4><P>
The pH of PBS buffer from common stock was adjusted to pH=7,45. 
</P><P>
The fluorescence was measured for the dilutions of fluorescein. The results from this measurement was not used. Description and final results of the experiment can be found here (2.3 Fluorescein). 
</P><I>
Selma, Attila and Eirikur
</I><H4>Calibration 2: Silica beads (2) ‚Äì 01/07/18</H4><P>
OD<SUB>600</SUB> was measured for the dilutions of silica beads. The results from this measurement was used for submission. Description and final results of the experiment can be found here (2.2 Silica beads). 
</P><I>
Selma, Attila and Eirikur
</I><H4>Cell measurement: Overnight of transformants (1) ‚Äì 01/07/18</H4><P>
An overnight of the transformants were set over in 5 mL LB + Chloramphenicol (CAM) (1:1000). 
</P><I>
Selma, Attila and Eirikur
</I><H3>Week 27</H3><H4>Cell measurement: Cell growth, sampling, and assay (1) ‚Äì 02/07/18</H4><P>
The overnight were diluted, abs<SUB>600</SUB> measured and diluted further to an abs<SUB>600</SUB> og 0,02 in a final volume of 12 mL. The samples were not measured and the experiment was therefore performed again. Description and final results of the experiment can be found here (3 Absorbance and fluorescence of transformed cells).
</P><I>
Selma, Attila, Eirikur and Lasse
</I><H4>Cell measurement: Overnight of transformants (2) ‚Äì 02/07/18</H4><P>
An overnight was made from the overnight from the same day (Cell measurement: Overnight of transformants (1) ‚Äì 01/07/18). 
</P><I>
Selma, Attila, Eirikur and Lasse
</I><H4>Cell measurement: Cell growth, sampling, and assay (2) ‚Äì 03/07/18</H4><P>
The overnight were diluted, abs<SUB>600</SUB> measured and diluted further to an abs<SUB>600</SUB> og 0,02 in a final volume of 12 mL. Abs<SUB>600</SUB> and fluorescence were measured. The results from this measurement was used for submission. Description and final results of the experiment can be found here (3 Absorbance and fluorescence of transformed cells).
</P><I>
Selma, Attila, Eirikur and Lasse
</I><H4>CFU per 0,1 OD<SUB>600</SUB>: Starting sample preparation and dilution series (1) ‚Äì 03/07/18</H4><P>
Abs<SUB>600</SUB> were measured on samples from the overnight culture (Overnight of transformants (2)). Triplicates were made after measurement of OD<SUB>600</SUB>=0,1. After the dilution series the samples were plated. Description and final results of the experiment can be found here (4 Colony Forming Units per 0.1 OD<SUB>600</SUB> E. coli cultures). 
</P><I>
Selma, Attila, Eirikur and Lasse
</I><H4>CFU per 0,1 OD<SUB>600</SUB>: CFU/mL/OD calculation (1) ‚Äì 03/07/18</H4><P>
The plates were counted, but as we figured that the triplicate was not correct, we did not use the result. Description and final results of the experiment can be found here (4 Colony Forming Units per 0.1 OD<SUB>600</SUB> E. coli cultures). 
</P><I>
Selma, Attila, Eirikur and Lasse
</I><H4>CFU per 0,1 OD<SUB>600</SUB>: Overnight culture (2) ‚Äì 03/07/18</H4><P>
An overnight culture was made from the transformants. 
</P><I>
Selma, Attila, Eirikur and Lasse
</I><H4>CFU per 0,1 OD<SUB>600</SUB>: Starting sample preparation and dilution series (2) ‚Äì 04/07/18</H4><P>
Abs<SUB>600</SUB> were measured on samples from the overnight culture. Triplicates were made and each of the triplicates were diluted to an OD<SUB>600</SUB>=0,1. After the dilution series the samples were plated. Description and final results of the experiment can be found here (4 Colony Forming Units per 0.1 OD<SUB>600</SUB> E. coli cultures). 
</P><I>
Attila and Lasse
</I><H4>CFU per 0,1 OD<SUB>600</SUB>: CFU/mL/OD calculation (2) ‚Äì 05/07/18</H4><P>
The plates were counted, but as we were not satisfied with the result, we did not use the result and decided to do it once more. Description and final results of the experiment can be found here (4 Colony Forming Units per 0.1 OD<SUB>600</SUB> E. coli cultures). 
</P><I>
Attila and Lasse
</I><H3>Week 29</H3><H4>CFU per 0,1 OD<SUB>600</SUB>: Overnight culture (3) ‚Äì 16/07/18</H4><P>
An overnight culture was made from the transformants. 
</P><I>
Attila and Lasse
</I><H4>CFU per 0,1 OD<SUB>600</SUB>: Starting sample preparation and dilution series (3) ‚Äì 17/07/18</H4><P>
Abs<SUB>600</SUB> were measured on samples from the overnight culture. Triplicates were made and each of the triplicates were diluted to an OD<SUB>600</SUB>=0,1. After the dilution series the samples were plated. Description and final results of the experiment can be found here (4 Colony Forming Units per 0.1 OD<SUB>600</SUB> E. coli cultures). 
</P><I>
Attila and Lasse
</I><H4>CFU per 0,1 OD<SUB>600</SUB>: CFU/mL/OD calculation (3) ‚Äì 18/07/18</H4><P>
The plates were counted, but as we were not satisfied with the result, we did not use the result and decided to do it once more. Description and final results of the experiment can be found here (4 Colony Forming Units per 0.1 OD<SUB>600</SUB> E. coli cultures). 
</P><I>
Attila and Lasse
</I><H4>CFU per 0,1 OD<SUB>600</SUB>: Overnight culture (4) ‚Äì 18/07/18</H4><P>
An overnight culture was made from the transformants. 
</P><I>
Attila and Lasse
</I><H4>CFU per 0,1 OD<SUB>600</SUB>: Starting sample preparation and dilution series (4) ‚Äì 19/07/18</H4><P>
Abs<SUB>600</SUB> were measured on samples from the overnight culture. Triplicates were made and each of the triplicates were diluted to an OD<SUB>600</SUB>=0,1. The calculation for the OD<SUB>600</SUB>=0,1 were made based on an average of the measurements of each of the singletons from the triplicate. After the dilution series the samples were plated. Description and final results of the experiment can be found here (4 Colony Forming Units per 0.1 OD<SUB>600</SUB> E. coli cultures). 
</P><I>
Attila and Lasse
</I><H4>CFU per 0,1 OD<SUB>600</SUB>: CFU/mL/OD calculation (4) ‚Äì 20/07/18</H4><P>
The plates were counted and The results from this measurement/calculation was used for submission. Description and final results of the experiment can be found here (4 Colony Forming Units per 0.1 OD<SUB>600</SUB> E. coli cultures). 
</P><I>
Attila and Lasse
</I><H3>Week 30</H3><H4>Calibration 3: Fluorescein (2)‚Äì 24/07/18</H4><P>
As we figured out that a wrong setting on the plate reader had been used for Calibration 3: Fluorescein (1) we performed the experiment again. 
</P><P>
The pH of PBS buffer from common stock was adjusted to pH=7,45. 
</P><P>
The fluorescence was measured for the dilutions of fluorescein. The results from this measurement was used for submission. Description and final results of the experiment can be found here (2.3 Fluorescein). 
</P><I>
Attila and Lasse
</I></DIV><DIV id="tabs-8"><H1>Egg membrane experiment</H1><H2>Up to week 35</H2><H3>Egg yolk membrane extractions </H3><UL><LI>Various attempts to cut the egg yolk and obtain the intact membrane.
</LI><LI>Experimenting with &quot;fixation&quot; of the membrane on various surfaces, integrity of membrane, etc...</LI></UL><P>
Note: fresh eggs have stronger membrane 
</P><UL><LI>Learning the &quot;technique&quot; and manufacturing appropriate tools (bend wire) for handling delicate membrane</LI></UL><I>Eric &amp; Natt</I><H2>Week 34</H2><H4>Manufacturing the chamber and testing the setup ‚Äì 24-26/08/18</H4><P>
The polystyrene plastic cup (PS) was found to be the most appropriate material for our setup. 
</P><UL><LI>Small holes were made at the bottom of the upper cup with heated nail
</LI><LI>Membrane was placed across the holes, left to settle and fixed with wax</LI></UL><P>
The next step was testing the leakiness of the setup. Cups were filled with water up to 2 cm and left for 24h. Upon any sign of leakiness the membrane was discarded.
</P><I>Eric</I><H2>Week 35</H2><H4>Manufacturing the chamber and testing the setup2 ‚Äì  1-2/09/18</H4><P>
Upon further testing the simple &quot;gluing&quot; the membrane with universal glue was found to be superior in comparison to &quot;waxing&quot;. 
</P><UL><LI>Same testing for leakines was performed with similar amount of water and therefore hydrostatic pressure. 
</LI><LI>It was found out that letting the membres dry even for 1h causes leakiness upon further testing for leakiness. All leaky membranes were discarded.
</LI><LI>4 membranes that passed all the tests were stored in a plastic box with wet tissue to preserve the humidity and prevent drying of the membranes.</LI></UL><H2>Week 36</H2><H4>Preparing the bacteria ‚Äì  6/09/18</H4><P>
For preliminary experiments the following strains of SIEC bacteria were used:
</P><P>
(All had chaperones and mCherry as a reporter protein)
</P><TABLE><TBODY><TR><TD>numbering
   </TD><TD>Genetic material
   </TD><TD>Description
   </TD></TR><TR><TD>1.
   </TD><TD>I+, S+, C+
   </TD><TD>Fully functional strain
   </TD></TR><TR><TD>2.
   </TD><TD>I-,S+,C+
   </TD><TD>Strain with disabled injectisome 
   </TD></TR><TR><TD>3.
   </TD><TD>I+,S-,C+
   </TD><TD>Strain with injectosome but without secretion signal
   </TD></TR></TBODY></TABLE><P>
Legend: 
</P><UL><LI> I = injectosome, (+) means functional injectosome, while (-) means the plasmid is lacking promoter in front DNA that in encoding injectisome.
</LI><LI>S = secretion signal, a sequence of 20 amino acids which is believed to provide selectivity when it comes to secretion via injectosome.
</LI><LI>C = chaperone. In this experiment all strains had chaperone. It is believed that they facilitate unfolding of the protein prior to injecting, however we tried to validate this in some other experiments.</LI></UL><P>
Strains were picked from the araginoze plates and and overnight culture was grown in 10mL of LB media + Chloramphenicol. 
</P><UL><LI>Incubated overnight at 28¬∞C  since the other incubator was broken (for 37¬∞C)</LI></UL><H4>Incubation‚Äì  7/09/18</H4><P>
After measuring OD600 of overnight cultures the calculated amount of arabinose and IPTG was added to get the final concentration of 1% (w/w%) for arabinose and 1mM for IPTG. After incubating for 2h at 28¬∞C (the 37¬∞C incubator was still out of function) the bacteria solution was poured over membrane (without centrifugation) so that the upper chamber was filled approximately to the half (5mL of culture).
</P><P>
In lower chamber LB was poured (approx 3mL) for easier comparison of fluorescence.
</P><H4>Measurements </H4><P>
The samples were taken after 20 and 47 hours and compared with LB media serving as a blank.
</P></DIV><FOOTER class="igem_2018_team_content footer"><DIV class="igem_2018_team_column_wrapper"><DIV class="has-text-centered"><SPAN>
                Made with ‚ù§Ô∏è in Copenhagen
            </SPAN><SPAN>
                Supported by
            </SPAN><SPAN><A href="https://medium.com/igem-copenhagen">Medium (blog)</A> |
                <A href="https://www.facebook.com/iGEMCopenhagen/">Facebook</A> |
                <A href="https://twitter.com/iGEM_Copenhagen">Twitter</A> |
                <A href="https://www.instagram.com/igem_copenhagen/">Instagram</A></SPAN></DIV></DIV></FOOTER></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>