<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCopenhagen_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCopenhagen/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu"><A href="https://2018.igem.org/Team:UCopenhagen"><DIV class="menu_item direct_link"> HOME
        </DIV></A><DIV class="menu_item"> TEAM
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Team"><DIV class="submenu_item">
                Team Members
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Collaborations"><DIV class="submenu_item">
                Collaborations
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Events_Attended"><DIV class="submenu_item">
                iGEM Events
            </DIV></A></DIV><DIV class="menu_item"> PROJECT
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Background"><DIV class="submenu_item">
                Background
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Description"><DIV class="submenu_item">
                Description
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Gene_Construct_Design"><DIV class="submenu_item">
                Gene Construct Design 
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Experiments"><DIV class="submenu_item">
                Experiments
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Protocols"><DIV class="submenu_item">
                Protocols
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Notebook"><DIV class="submenu_item">
                Notebook
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Model"><DIV class="submenu_item">
                Model
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Results"><DIV class="submenu_item">
                Results
            </DIV></A></DIV><A href="https://2018.igem.org/Team:UCopenhagen/InterLab"><DIV class="menu_item direct_link"> INTERLAB
        </DIV></A><DIV class="menu_item"> PARTS
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Parts"><DIV class="submenu_item">
                Parts Overview
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Basic_Part"><DIV class="submenu_item">
                Basic Parts 
            </DIV></A></DIV><A href="https://2018.igem.org/Team:UCopenhagen/Safety"><DIV class="menu_item direct_link"> SAFETY
        </DIV></A><DIV class="menu_item"> HUMAN PRACTICES
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Human_Practices"><DIV class="submenu_item">
                Introduction
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Why_Colonize_Mars"><DIV class="submenu_item">
                Why Colonize Mars?
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Theraputic Proteins"><DIV class="submenu_item">
                Theraputic Proteins
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Choice_of_Protein"><DIV class="submenu_item">
                Choosing a Protein
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Protein_Production_System"><DIV class="submenu_item">
                Evaluation of system
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Public_Engagement"><DIV class="submenu_item">
                Education &amp; Engagement
            </DIV></A></DIV><A href="https://2018.igem.org/Team:UCopenhagen/Attributions"><DIV class="menu_item direct_link"> ATTRIBUTIONS
        </DIV></A><DIV class="menu_item"> AWARDS
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/NordicMeetup"><DIV class="submenu_item">
                Nordic iGEM conference
            </DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=UCopenhagen"><DIV class="menu_item direct_link"> JUDGING FORM ⇗
        </DIV></A><DIV class="menu_item"> Miscellaneous
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Articles"><DIV class="submenu_item">
                Articles
            </DIV></A><A href="https://igem-copenhagen.github.io/awesome-iGEM/"><DIV class="submenu_item">
                Awesome iGEM
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Wiki_how_tos"><DIV class="submenu_item">
                Wiki How-tos
            </DIV></A></DIV><A href="https://medium.com/igem-copenhagen" style="padding-right: 0px;" target="_blank"><DIV class="menu_item direct_link"> Blog on Medium
        </DIV></A></DIV><DIV class="igem_2018_team_content pageContent"><DIV class="igem_2018_team_column_wrapper"><TITLE>jQuery UI Example Page</TITLE><H1>Experiments</H1><P>
This is the page where we describe our different experiments. 

</P><DIV id="tabs"><UL><LI><A href="#tabs-1">Cloning</A></LI><LI><A href="#tabs-3">Experiments with biomimetic membranes</A></LI><LI><A href="#tabs-4">Protoplast experiments with plants</A></LI><LI><A href="#tabs-5">Leakiness characterization</A></LI><LI><A href="#tabs-6">Egg membrane experiment</A></LI></UL><DIV id="tabs-1"><H1>Cloning of expression constructs</H1><H1>1. Introduction</H1><P>
Our experiments were based on SIEC and SIECΔp1 bacteria transformed with plasmids encoding reporter proteins, Map20 and Chaperones (read more about the different components <A href="https://2018.igem.org/Team:UCopenhagen/Gene_Construct_Design">here</A>). We used both USER cloning and restriction-ligation to assemble the expression plasmids. Protocols used for molecular cloning can be found <A href="https://2018.igem.org/Team:UCopenhagen/Protocols">here</A>.
</P><H1>2. Cloning strategies</H1><P>
The primers, that we used throughout the cloning process can be viewed in the bottom of this page.
</P><P>
The reporter genes were PCR amplified in two rounds. First to add a C-terminal 6x-His tag and secondly to add restriction sites. XmaI and PstI restriction sites were added to the reporters that were to be assembled with both Map20 the vector and XbaI and PstI restriction sites were added reporter genes that were inserted into the vector without Map20. The reporter genes were then inserted into a BBa_K228005 backbone with or without chaperones.
</P><P>
The Map20 part sequence was constructed using five different primers and assembled together in a PCR reaction. Restrictions sites for XbaI and AgeI were also added by PCR. This method gave a successful result in first try. 
</P><P>
A vector backbone with chaperones genes was also constructed. We used USER cloning to insert the CesT and CesF chaperone genes into the <EM>pBAD</EM> vector as can be seen <STRONG>figure 1.</STRONG></P><P><STRONG>Figure 1. </STRONG>Assembly of the <EM>pBAD</EM> vector with chaperones. Both the chaperone (CesF and CesT) genes and vector backbone (BBa_K228005) are first PCR amplified with uracil primers to allow for USER cloning. The figure is made using SnapGene.
</P><P>
The strategy used to assemble expression plasmids with the T3SS signal sequence Map20 can be seen on <STRONG>figure 2</STRONG>. The same approach was used for insertion into the vector with and without chaperone genes. We used 3A assembly to insert the reporter genes with Map20 signal downstream of the pBAD promoter in the vectors. The reporter genes are digested with XmaI leaving overhang complementary that of the Map20 gene digested with AgeI. This way Map20 is inserted upstream of the reporter. Additionally, the reporter is cut with PstI and Map20 with XbaI allowing them to be assembled with the vector backbone which is digested with SpeI and PstI.
</P><P>
The same strategy was used to assemble the expression plasmids with reporter without Map20 signal sequence. This time both the vector backbone and the reporter protein sequence were digested with XbaI and PstI.
</P><P><STRONG>A</STRONG></P><P><STRONG>B</STRONG></P><P><STRONG>Figure 2.</STRONG> Outline of the 3A cloning strategy used to assemble the expression plasmid with reporter and T3SS signal sequence. Plasmid backbone is either without (<STRONG>A</STRONG>) or with chaperones (<STRONG>B</STRONG>). The figure is made using SnapGene.
</P><P>
The expression plasmid with the EspD translocon protein was assembled with USER cloning as well. USER overhangs were added to the EspD genes and the BBA_K731500 vector by PCR. EspD genes and vector backbone were mixed and treated with USER™ enzyme to create single stranded overhangs on both the gene and vector. The overhangs allow for hybridization to form a circular construct. The assembly method of the EspD expression plasmid can be seen in <STRONG>figure 3</STRONG>.
</P><P><STRONG>Figure 3.</STRONG> Outline of USER cloning strategy used to assemble the expression plasmid with EspD translocon. The figure was made using SnapGene.
</P><H1>3. Primers</H1><P>
The following primers were used for cloning.
</P><P><STRONG>Map20-signal synthetic primers</STRONG></P><P>
Map20-1 5' CTAAAGAGGAGAAATACTAGATGTTTAGTCCAATGACAATGGTAGGTC 3'
Map20-2 5' AGTCCAATGACAATGGTAGGTCGTTCGTTAGCTCAGGCGGCTACACA 3'
Map20-3 5' ACCTGAGCCAGAACCACCAAGAGTTTGTGTAGCCGCCTGAGCTAAC 3'
Map20-F2 5' GGCCGCTTCTAGAGAATCTAAAGAGGAGAAATACTAGATGTTTAG 3'
Map20-R2 5' AATCACTTAACCGGTACCAGAACCTGAGCCAGAACCACCAAG 3'</P><P><STRONG>Primers for pBAD Vector</STRONG></P><P>
araC-UF 5' AGATCACTACUAGAGCCAGGCATCAAATAAA 3'
araC-UR 5' ACTTTCCTGUGTGACTCTAGTTATTAAGCTACTAAAGCGTAGT 3'
Primers for CesT/CesF chaperone
CesF-UF 5' ACAGGAAAGUACTAGATGAATGAACAA 3'
CesT-UR 5' AGTAGTGATCUACACTAGCACTATTTATTAATTG 3' </P><P><STRONG>Sequencing primers for pBAD (BBA_K228005)</STRONG></P><P>
AraC-SF 5' GACGATCAACTCTATTTCTCGCGAG 3'
rrnB-SR 5' AGTGTGACTCTAGTAGAGAGCGTTC 3'
CmR-SF 5' CATGATGAACCTGAATCGCCAG 3'
Ori-SF 5' 5' CGCTTTCTCATAGCTCACGCTGTA 3'
Ori-SR 5' GTTGGACTCAAGACGATAGTTACCG 3'
BBP-SF 5' CTGGAATTCGCGGCCGCTTCTAGA 3'
BBS-SR 5' CGGACTGCAGCGGCCGCTACTAGTA 3'
</P><P><STRONG>Primers for Reporter genes</STRONG></P><P>
NLuc-F 5' AGTAGACCCGGGATGGCCGGCGTGTTCACCCTCG 3'
NLuc-R 5' GTGATGGTGGCTACCGCTACCGGTAGCAAGGATTCTCTCG 3'
mCh-F 5' AGTAGACCCGGGATGGTGAGCAAGGGCGAGGAG 3'
mCh-R 5' GTGATGGTGGCTACCGCTACCCTTGTACAGCTCGTCCATGC 3'
GFP-F2 5' AGTAGACCCGGGATGCGTAAAGGCGAAGAGCTGT 3'
GFP-R 5' GTGATGGTGGCTACCGCTACCTTTGTACAGTTCATCCATACCATG 3'
Blac-F 5' AGTAGACCCGGGCACCCAGAAACGCTGGTGAAAG 3'
Blac-R 5' GTGATGGTGGCTACCGCTACCCCAATGCTTAATCAGTGAGGCAC 3'
GS-His-R 5' ACTGACCTGCAGCTAGTGATGATGGTGATGGTGGCTACCGCTACC 3'
</P><P><STRONG>Primers for EspD translocon expression vectors</STRONG></P><P>
EspD-UF 5' AGAGGAGAAA<STRONG>U</STRONG>ACTAGATGCTTAATGTAAATAACGATATCCA 3'
EspD-UR 5' ATGGTGATGG<STRONG>U</STRONG>GAGATCCTGAGCCAACTCGACCGCTGACAATACG 3'
LacO-UF 5' ACCATCACCA<STRONG>U</STRONG>CATCACTAATAGTACTAGTAGCGGCCGCTGCAGTC 3'
LacO-UR 5' ATTTCTCCTC<STRONG>U</STRONG>TTCTCTAGTATTTCATGAGGGAATTGTTATCCG 3'</P><P><STRONG>Primers to identify SIEC and SIEC∆P1 strain</STRONG></P><P>
pTac-F 5' TTGACAATTAATCATCGGCTCGTATAATG 3' 
Ler-F    5' AACTCATCGAAAGGTGTTTACTACCG 3' 
CesAB-R 5' TTCTTGTTTGGCTCACAATACTCATCC 3'</P><P><STRONG>Primers for reporter genes without signal</STRONG></P><P>
mCherry 5’ GCT TCT AGA GAA TCT AAA GAG GAG AAA TAC TAG ATG GTG AGC AAG GGC GA 3’  
GFP 5’ GCT TCT AGA GAA TCT AAA GAG GAG AAA TAC TAG ATG CGT AAA GGC GAA GA 3’  
Beta-lactamase 5’ GCT TCT AGA GAA TCT AAA GAG GAG AAA TAC TAG ATG GCC GGC GTG TTC AC 3’ 
NanoLuc 5’ GCT TCT AGA GAA TCT AAA GAG GAG AAA TAC TAG ATG CAC CCA GAA ACG CTG 3’ </P><H1>5. Conclusion</H1><P>
To verify successful cloning we did either colony PCR or sequencing, revealing that we managed to make 14 out of the 16 possible constructs seen in the table below.
</P><TABLE><TBODY><TR><TD>Plasmid constructed
   </TD><TD>Reporter
   </TD><TD>N-terminal Map20
   </TD><TD>CesT and CesF
   </TD></TR><TR><TD>+
   </TD><TD>sfGFP
   </TD><TD>+
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>sfGFP
   </TD><TD>+
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>sfGFP
   </TD><TD>-
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>sfGFP
   </TD><TD>-
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>mCherry
   </TD><TD>+
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>mCherry
   </TD><TD>+
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>mCherry
   </TD><TD>-
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>mCherry
   </TD><TD>-
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>nanoluc
   </TD><TD>+
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>nanoluc
   </TD><TD>+
   </TD><TD>-
   </TD></TR><TR><TD>-
   </TD><TD>nanoluc
   </TD><TD>-
   </TD><TD>-
   </TD></TR><TR><TD>-
   </TD><TD>nanoluc
   </TD><TD>-
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>β-lactamase
   </TD><TD>+
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>β-lactamase
   </TD><TD>+
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>β-lactamase
   </TD><TD>-
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>β-lactamase
   </TD><TD>-
   </TD><TD>+
   </TD></TR></TBODY></TABLE></DIV><DIV id="tabs-3"><H1>Experiments with biomimetic membranes</H1><H1>1. Introduction</H1><P>
The membrane is an important component of our protein production system as it separates the bacterial culture from the protein collection chamber. In the final protein production chamber the membrane would be an artificial biomimetic membrane. 
</P><P>
In this experiment, we investigated the use of lipid membranes using supported lipid bilayers and liposomes.
</P><H1>2. Aim</H1><P>
The first aim of this part of our project was to investigate if the SIEC strain (i.e. the <EM>E. coli </EM>strain expressing the injectisome) would bind to a biomimetic membrane and secrete
proteins through it.
</P><P>
The second aim was to investigate if the secretion signal Map20 (BBa_K2871000) would increase the amount of protein secreted into liposomes.
</P><H1>3. Membrane composition</H1><P>
A mixture containing sphingomyelin (SM), dioleoylphosphatidylcholine (DOPC), dioleoylphosphatidylserine (DOPS), and cholesterol (Chl) was used for the supported lipid bilayer and liposomes in a ratio of 44:24:12:20 (SM:DOPC:DOPS:Chl). The ratio of the membrane components was inspired from Chatterjee and colleagues [1]who showed that EspD, which is required for formation of pores in the host membrane, spontaneously bound to lipid vesicles with this lipid composition. This lipid composition is also similar to the composition of the outer leaflet of a mammalian cell plasma membrane [2]. 
</P><P>
The dye Atto655 was also included in some of the lipid mixtures to enable detection of the lipid bilayers and liposome by fluorescence microscopy.
</P><H1>4. Binding to supported lipid bilayers</H1><P>
In this experiment, binding of SIEC and SIECΔp1 to supported lipid bilayers were investigated using fluorescence microscopy. The protocols used can be found <A href="https://2018.igem.org/Team:UCopenhagen/Protocols">here</A>.The bacterial strains expressed GFP (BBa_I20270) for visualization in the microscope. We expected that the SIEC strain would bind to the membrane due to expression of the injectisome whereas SIECΔp1 would not bind to the membrane. Microscopy slides covered with supported lipid bilayers with the dye Atto655 were prepared and incubated with bacterial culture. After incubation, the microscopy slides were washed in LB media to remove unbound bacteria. An initial test where SIECΔp1 bacteria expressing GFP were incubated with supported lipid bilayers showed GFP signal after washing the microscopy slide 15 times. The integrity of the supported lipid bilayer was not investigated here, but it could be investigated using fluorescence recovery after photobleaching (FRAP). Due to time limitations, this experiment was not pursued further.
</P><H1>5. Injection into liposome experiment</H1><H2>Outline</H2><P>
The purpose of this experiment is to see if the bacteria with the right genetic constructs is able to inject a specific protein into liposomes that mimic the <A href="https://2018.igem.org/Team:UCopenhagen/Background">injectisomes</A> natural membrane-targets. By varying the presence of the injectisome, specific <A href="https://2018.igem.org/Team:UCopenhagen/Background">chaperones</A> and the <A href="https://2018.igem.org/Team:UCopenhagen/Background"><SPAN>signal peptide</SPAN></A>, the experiment is set up to give information about the effect of these genetic products concerning the secretion into liposomes. Any differences of injection that is observed when varying these components provide information about if the parts are needed for secretion or if the components increase secretion rates.
</P><P>
Bacterial strain abbreviations are <STRONG>&quot;I&quot;</STRONG>,<STRONG> &quot;S&quot;</STRONG> and <STRONG>&quot;C&quot;</STRONG> which corresponds to <STRONG>&quot;injectisome&quot;,</STRONG><STRONG>&quot;signal&quot;</STRONG> and <STRONG>&quot;Chaperones (CesT, CesF)&quot;</STRONG> respectively. For example strain <STRONG>I<SUP>+</SUP>S<SUP>+</SUP>C<SUP>-</SUP></STRONG> have all components present, except the chaperones. 
</P><P>
The strains that are used are: <STRONG>I<SUP>+</SUP>S<SUP>+</SUP>C<SUP>+</SUP>, I<SUP>+</SUP>S<SUP>+</SUP>C<SUP>-</SUP>, I<SUP>+</SUP>S<SUP>-</SUP>C<SUP>+</SUP>, I<SUP>+</SUP>S<SUP>-</SUP>C<SUP>-</SUP>, I<SUP>-</SUP>S<SUP>-</SUP>C<SUP>-</SUP>, I<SUP>-</SUP>S<SUP>+</SUP>C<SUP>-</SUP>, I<SUP>-</SUP>S<SUP>-</SUP>C<SUP>+</SUP>, I<SUP>-</SUP>S<SUP>+</SUP>C<SUP>+</SUP>.</STRONG> All constructs are available with either mCherry or GFP functioning as a reporter protein.
</P><P>
Separating bacteria, liposomes and possible media soluble reporter proteins from each other in an intact form proved troublesome. Method is still not fully developed. Centrifugation with spin columns containing a 450 µm pore size filter was used to separate bacteria and liposomes. Proteinase K was used as a method to sever the injectisome needle complex, in case of strong attachment to the liposomes, that could prevent separation. Moreover proteinase k would remove residual media soluble reporter proteins, so.
</P><H2>Experimental overview</H2><P>
Protocols can be found <A href="https://static.igem.org/mediawiki/2018/f/f2/T--UCopenhagen--Protocol_Injection_into_Liposome.pdf">here.</A></P><P><STRONG>For liposome injection experiment 1</STRONG></P><OL><LI>Overnight of strains.
</LI><LI>Rehydration of previously made and 'dried' lipids to form liposomes.
</LI><LI>Recovery of strain cultures in fresh media.
</LI><LI>Induction of injectisome and reporter protein.
</LI><LI>Incubation of bacteria and liposomes together.
</LI><LI>Separation of bacteria and liposomes.
</LI><LI>Measure the absorption and fluorescence of the liposomes.
</LI><LI>Do western blot of liposomes.</LI></OL><P><STRONG>For liposome injection experiment 2 &amp; 3</STRONG></P><OL><LI>Overnight of strains with induction.
</LI><LI>Rehydration of previously made and 'dried' lipids to form liposomes.
</LI><LI>Incubation of bacteria and liposomes together.
</LI><LI>Separation of bacteria and liposomes.
</LI><LI>Measure the absorption and fluorescence of the liposomes.
</LI><LI>Do western blot of liposomes.</LI></OL><P><STRONG>[1]</STRONG> Chatterjee, A., Caballero-Franco, C., Bakker, D., Totten, S., Jardim, A. (2015) Pore-forming Activity of the Escherichia coli Type III Secretion System Protein EspD. J Biol Chem. 290 (42) pp. 25579-25594.
</P><P><STRONG>[2] </STRONG>Zachowski, A. (1993) Phospholipids in animal eukaryotic membranes: transverse asymmetry and movement. Biochem. J. 294, pp. 1-14.
</P></DIV><DIV id="tabs-4"><H1>Protoplast experiment with plants</H1><H1>Aim</H1><P>The aim of the experiment was to check, using onion and tobacco protoplasts and <EM>E. Coli</EM> bacteria expressing GFP, Mcherry and the injectisome, if the injectisome(T3SS) was successful in injecting proteins across a membrane.</P><H1>Background and Introduction</H1><H2>The Idea</H2><P>
The inspiration for this project came from the 2011 Hokkaido iGEM team and their &quot;injection assay using onion cells&quot;. We wanted to replicate the experiment to show that the Injectisome served its function, a proof of concept of sorts. First we were focused on using onion cells but in the process of the experiment some problems arose which led us to look to other options. We decided on that the other option should be tobacco cells. 
</P><H2>Bacteria and reporter proteins</H2><P>
The<EM> E. coli</EM> strain we wanted the proof of concept for contained the injectisome, secretion signal and a reporter protein(GFP and Mcherry). We wanted to use Mcherry for the assay on the onion and GFP for the assay on the tobacco due to ease of analysis of results in a fluorescent microscope. IPTG induces the formation of the injectisome and arabinose induces the reporter protein.
</P><H2>Protoplasts</H2><P>
Tobacco protoplasts and onion protoplasts differ quite a bit. Onion cells are large and without any chlorophyll but tobacco cells are small and contain chlorophyll. Protoplasts are in essence plant cells without a cell wall. To make our protoplasts, the cells with an intact cell wall, have to be incubated with a solution with cell wall degrading enzymes, an optimal osmotic pressure and PH value, so the cells don't shrink and/or burst. The optimization of the production and handling of the protoplasts was one of the biggest barriers to overcome, during the experimental process. 
</P><P>
The reason for making the protoplast in the first place is that a plant cell with an intact cell wall might pose a problem to the attachment of the injectisome to the cell and the injection of the protein into the cell. 
</P><P><STRONG>Figure 1</STRONG> showing experimental overview
</P><H1>Methods, process and results</H1><H2>Intro</H2><P>
We used the protocol from the 2011 Hokkaido iGEM team as a start point and tried to follow their protocol as best as possible. When that didn't give useful results other than burst cells, flabs of apparent onion remnants and fluorescent bacteria floating about, we had to optimise several aspects of the protocol to our needs. 
</P><H2>Protoplast production and preperation</H2><P>
After trying effortlessly attaching onion cell sheets with agarose to a slide glass and incubating with an MgM-MES buffer(ph 5) with 0.4M Sorbitol and cell wall degrading enzymes to obtain protoplasts, we got the idea of making tobacco protoplasts instead and keep the onion to the side. 
</P><P>
We then tried incubating  a few pieces of a tobacco leafs (downside-down without their cuticle) in a culture plate with the same solution mentioned above. Looking at a sample of the solution after a few hours of incubation revealed burst protoplasts and other plant materials. the production of onion protoplast was tried again on glass slides and revealed no apparent degradation of the cell wall, it looked like the cells had shrunk.
</P><P>
After mostly getting burst or shrunk cells or protoplasts we decided to  look at the osmolarity. It was decided  to try out different concentrations of Mannitol in an MgM-MES buffer of PH 5. This time the onion cell sheets were treated the same way as the tobacco cells. both were put in a cell wall degrading enzyme solution in an MgM-MES buffer with different concentrations of Mannitol in a culture plate. We observed the solutions in a microscope after incubating the onion and tobacco cells for a couple of hours. This time we successfully produced protoplasts, both onion and tobacco. We decided from quantitative observations in the microscope to use 0.8M Mannitol for the onion cell sheets and 0.6M Mannitol for the tobacco leaves. 
</P><P><STRONG>Figure 2 </STRONG> showing experimental setup of culture plates with plant samples and enzyme buffer
</P><H2>Bacteria preperation</H2><P>
We mostly followed the Hokkaido 2011 team's protocol in preparation of our bacteria. We incubated them overnight and induced them with IPTG and arabinose. the next day the bacteria were washed and resuspended, in the solution mentioned above(The solution differs in Osmolarity between onion cells and tobacco cells).  To check that our bacteria actually emit fluorescence we looked at them in a fluorescence microscope. In the microscope we saw clearly that our strains expressing Mcherry and GFP without a signal sequence gave of clear and bright fluorescence but our strains which contained the signal sequence showed much weaker fluorescence. 
</P></DIV><DIV id="tabs-5"><H1>Leakiness characterization</H1><H2>1. Aim</H2><P>
The aim of this experiment was to examine if injectisome (T3SS)-bearing bacteria secrete proteins into the surrounding media without interaction with any target membrane. In other words we wanted to test if the <EM>E. coli </EM> expressing the injectisome was leaky and whether the leakiness was affected by presence of the reporter signal and/or co-expression of chaperones. In order to do this we needed to ensure that the signal tagged reporter protein was expressed by bacteria. 
</P><H2>2. Introduction</H2><P>
The principle of the experiment is to measure fluorescence of the media in which injectisome expressing <EM>E. coli </EM>have been grown for different time periods (0, 3 and 6 days) at 16 ℃. The experiment was conducted to investigate whether the injectisome leaks and quantify the degree of leakiness
</P><H2>3. Method </H2><P>
We used the plate reader SpectraMax M5 to measure the flourescence of the supernatant after different time periods. We also did Western Blots in order to detect presence of unfolded protein (that wouldn't be detected by the flourescence).  
</P><H2>Experimental overview</H2><P>
1) Grow bacteria 
</P><P>
2) Induce the bacteria with IPTG and arabinose to get expression of the injectisome and reporter protein
</P><P>
4) Take samples at different time intervals
</P><P>
5) Measure fluorescence of the media (supernatant) and pellet
</P><P>
6) Perform a western blot to detect non-folded proteins 
</P><P>
Strains
</P><P>
For the experiment a variety of different strains were used. We have use &quot;I&quot;, &quot;S&quot;, &quot;C&quot; to describe the different strains.
</P><P><STRONG>'I+/-' </STRONG>corresponds to presence or absence <STRONG>injectisome </STRONG>(control strain vs. strain that is induced to express the injectisome). 
</P><P><STRONG>'C+/-' </STRONG>corresponds to whether the <STRONG>chaperone </STRONG>(the CesF-CesT chaperone cassette) is present or not
</P><P><STRONG>'S+/-' </STRONG>corresponds to presence or absence of T3SS protein <STRONG>signal</STRONG> tag ( in our experiment the N-terminal Map20).
</P><P>
For example strain <STRONG>I<SUP>+</SUP>S<SUP>+</SUP>C<SUP>-</SUP></STRONG> have all components present, except the chaperones. 
</P><P><STRONG>IPTG </STRONG>is used to induce the expression of the injectisome.
</P><P><STRONG>Arabinose </STRONG>is used toinduce the expression of reporter proteins (e.g. mCherry or GFP)
</P><P>
In our experiment we examined the following strains:
</P><P><B> Figure 1 </B> Strains used in leakiness</P></DIV><DIV id="tabs-6"><H2>Introduction: Egg membrane experiment</H2><P>
The final product of PharMARSy would be a device that allows for growth of bacteria in one chamber and secretion of pure protein into another. Chambers would be divided by a biomimetic membrane, which should allow for bacteria to &quot;dock&quot;on it and use injectosome for secretion, yet it should also be robust to prevent bacteria from crossing across to other chamber. Having bacteria residues or living bacteria in collection chamber would be very dangerous for patient (could result in sepsis) if no additional purification steps are implemented.
</P><P>
It is thus quite evident that membrane is probably one of the most important component of our final product.
</P><P>
However as we couldn't obtain satisfactory biomimetic membrane we were forced to improvise and look into what nature could offer us. One of the possibilities was to use egg yolk membrane - the biggest membrane we could obtain and robust enough for experimenting. 
</P><P><STRONG>Figure 1</STRONG>: Schematic of the device on which membrane was tested.
</P><H2>Experiment: Hypothesis and setup</H2><P>
The goal was to prove that only the secreted proteins can pass the membrane and not the bacteria. We believed that bacteria would recognize the phospholipid bilayer characteristic for eukaryotic cells and secrete the protein via injectosome. 
</P><P>
Detection of increase in fluorescence from samples taken from collection chamber would mean a successful demonstration of our idea. However the solution in collection chamber has to remain sterile, otherwise we would get false positive result. (i.e. bacteria could pass to collection chamber and start producing protein which increases the fluorescence). 
</P><P>
The following illustration provides the somewhat simpler explanation of the setup of the experiment .
</P><P><STRONG>Figure 2</STRONG>: Description of the experiment. In time intervals a sample was taken from the collection chamber. Fluorescence was measured to detect if protein was being secreted (mCherry), while plating was performed to check whether bacteria crossed the membrane. Since our strain was resistant to Chloramphenicol, we used agarose plates with added Chloramphenicol. This way only bacterial strains with resistance to this antibiotic could grow on the plates. In order to prove that the colonies on the plate were indeed the strain we were working with arabinose was added since it induces production of mCherry protein in our strain. If colonies turned red then we can safely assume that the bacteria has somehow breached the membrane and that detected increase in fluorescence in collection chamber was indeed a false positive result. 
</P><H3>Technical details </H3><P>
The following experiment has probably not yet been done. The experiment therefore required a great deal of improvisation, flexibility and imagination, not mentioning the number of trials before optimizing the protocols.
</P><H3>Strains used</H3><P>
For preliminary experiments the following strains of SIEC (<EM>Synthetic injector E. coli) </EM>bacteria were used:
</P><P>
All had chaperone and mCherry as a reporter protein.
</P><P>
Legend: 
</P><UL><LI> I = injectisome, (+) means functional injectisome, while (-) means the plasmid is lacking promoter in front DNA that in encoding injectisome.
</LI><LI>S = secretion signal, a sequence of 20 amino acids which is believed to provide selectivity when it comes to secretion via injectosome.
</LI><LI>C = chaperone. In this experiment all strains had chaperone. It is believed that they facilitate unfolding of the protein prior to injecting, however we tried to validate this in some other experiments.</LI></UL><TABLE><TBODY><TR><TD>numbering
</TD><TD>Genetic material
</TD></TR><TR><TD>1.
</TD><TD>I+, S+, C+
</TD></TR><TR><TD>2.
</TD><TD>I-,S+,C+
</TD></TR><TR><TD>3.
</TD><TD>I+,S-,C+
</TD></TR></TBODY></TABLE><H3> Egg yolk extraction</H3><P>
In contrary to popular belief - it is actually possible to extract the membrane separating yolk from the white. However, some practice is needed beforehand and a specialized tools which are not part of the standard laboratory inventory. 
</P><P>
First step is to separate the white of the egg, which can be done by simply cracking the egg and carefully pouring the white of the egg out. Once the intact yolk is obtained it should be submerged into water to allow for easier manipulation afterwards.
</P><P>
Next, the membrane should be carefully cut - there are many ways that this can be done, but the best results are usually obtained with sharp scissors and a pincet. First a hole should be made into yolk carefully to serve as a starting point for cutting. The hole should be ideally on the top of the yolk to prevent excessive spilling which quickly clouds the water and makes any further operations much more difficult.
</P><P>
Starting from this hole a line can be cut in any direction. The easiest way seems to be by making (almost) 180° circle cut from the side and allowing for yolk to &quot;open&quot; and spill to the opposite side. It is worth noting that movements should be relatively slow and gentle, since we don't want to spill the yolk and cloud the water before finishing the cutting.
</P><P>
Once the desired cut is made the yolk will spill out. Careful exchange of water either with syringe or any other method will gradually return the visibility in the media to initial levels. The purification should be done gently, though, as membrane may very well rupture or be flushed out with exiting water. It is advisable to hold membrane gently with pincers where membrane is the strongest i.e. on the &quot;poles&quot; or on &quot;chalaza&quot;, the white membrane structure on the &quot;poles&quot; of the egg yolk. Chalaza can be seen on the figure 3.
</P><P><STRONG>Figure 3: </STRONG>A submerged egg yolk with visible white membrane structure (chalaza).
</P><P>
After some purification relatively pure membranes can be obtained. White like sheet can be manipulated using pincers or any similarly shaped tool with no sharp edges. 
</P><P>
The membrane should be cut to appropriate size (smaller is better).
</P><P><STRONG>Figure 4</STRONG>: Purified membrane in a petri dish.</P><H3>&quot;Fixation&quot; of the membrane</H3><P>
After several attempts a glue was found to be a better alternative to wax. The membrane had to be held tightly against the bottom of the chamber covering the holes, so that no liquid can pass between the membrane and the chamber bottom. Since any mechanical pressure system i.e. with rubber bands pressing against the membrane would inevitably result in its penetration, a glue was used instead. Dropping melted wax seemed to be a better solution at first, since it allowed for a very fine manipulation, but after it hardened it tended to separate from the base of the chamber thus destroying its purpose. The following figure (figure 5) illustrates the crude design of the chamber:
</P><P><STRONG>Figure 5</STRONG>: A schematics of the &quot;hardware&quot;. Into plastic cups the holes were made, which were later covered with membrane.
</P><H3>Tests for quality - leakiness </H3><P>
Each membrane was tested for leakines. Some water was placed in the chamber and a paper tissue was placed below. Upon any signs of water on the tissue the chamber was discarded. 
</P><H3>Final setup</H3><P>
In the final setup two chambers were separated by glued egg yolk membrane with different orientations. Initially the outer side of the membrane (that was originally facing the white of the egg) was facing the medium with bacteria.
</P><P>
The overnight culture of needed strain was incubated in 1% arabinose (w/w%) and 1mM IPTG for 2 hours prior to being poured over membrane (In next experiments this time was increased to an overnight incubation). Approximately 1 mL of overnight culture was poured into upper chamber while in collection chamber there was LB media (3-4mL) with antibiotic Chloramphenicol. Previous experiments characterizing the pBAD arabinose promoter  suggested that 1% arabinose was optimal for our setup.
</P><P><STRONG>Figure 6</STRONG>: As can be seen from the pictures improvisation was an important part of the experiment. Two plastic cups were cut and on the upper one holes were made with heated needle. After gluing the egg yolk membrane and testing for leakage, the setup was placed on the petri dish to minimize further contamination.  In the presented picture the violet red colour comes from the mCherry fluorescent protein that was produced by strain having functional injectosome and chaperone but no secretion signal sequence. 
</P></DIV></DIV></DIV></DIV></DIV></DIV><FOOTER class="igem_2018_team_content footer"><DIV class="igem_2018_team_column_wrapper"><DIV class="has-text-centered"><SPAN>
                Made with ❤️ in Copenhagen
            </SPAN><SPAN>
                Supported by
            </SPAN><SPAN><A href="https://medium.com/igem-copenhagen">Medium (blog)</A> |
                <A href="https://www.facebook.com/iGEMCopenhagen/">Facebook</A> |
                <A href="https://twitter.com/iGEM_Copenhagen">Twitter</A> |
                <A href="https://www.instagram.com/igem_copenhagen/">Instagram</A></SPAN></DIV></DIV></FOOTER></DIV></DIV></DIV></BODY></HTML>