<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCopenhagen_Gene_Construct_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCopenhagen/Gene Construct Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu"><A href="https://2018.igem.org/Team:UCopenhagen"><DIV class="menu_item direct_link"> HOME
        </DIV></A><DIV class="menu_item"> TEAM
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Team"><DIV class="submenu_item">
                Team Members
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Collaborations"><DIV class="submenu_item">
                Collaborations
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Events_Attended"><DIV class="submenu_item">
                iGEM Events
            </DIV></A></DIV><DIV class="menu_item"> PROJECT
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Background"><DIV class="submenu_item">
                Background
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Description"><DIV class="submenu_item">
                Description
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Gene_Construct_Design"><DIV class="submenu_item">
                Gene Construct Design 
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Experiments"><DIV class="submenu_item">
                Experiments
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Protocols"><DIV class="submenu_item">
                Protocols
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Notebook"><DIV class="submenu_item">
                Notebook
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Model"><DIV class="submenu_item">
                Model
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Results"><DIV class="submenu_item">
                Results
            </DIV></A></DIV><A href="https://2018.igem.org/Team:UCopenhagen/InterLab"><DIV class="menu_item direct_link"> INTERLAB
        </DIV></A><DIV class="menu_item"> PARTS
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Parts"><DIV class="submenu_item">
                Parts Overview
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Basic_Part"><DIV class="submenu_item">
                Basic Parts 
            </DIV></A></DIV><A href="https://2018.igem.org/Team:UCopenhagen/Safety"><DIV class="menu_item direct_link"> SAFETY
        </DIV></A><DIV class="menu_item"> HUMAN PRACTICES
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Human_Practices"><DIV class="submenu_item">
                Introduction
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Why_Colonize_Mars"><DIV class="submenu_item">
                Why Colonize Mars?
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Theraputic Proteins"><DIV class="submenu_item">
                Theraputic Proteins
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Choice_of_Protein"><DIV class="submenu_item">
                Choosing a Protein
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Protein_Production_System"><DIV class="submenu_item">
                Evaluation of system
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Public_Engagement"><DIV class="submenu_item">
                Education &amp; Engagement
            </DIV></A></DIV><A href="https://2018.igem.org/Team:UCopenhagen/Attributions"><DIV class="menu_item direct_link"> ATTRIBUTIONS
        </DIV></A><DIV class="menu_item"> AWARDS
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/NordicMeetup"><DIV class="submenu_item">
                Nordic iGEM conference
            </DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=UCopenhagen"><DIV class="menu_item direct_link"> JUDGING FORM ⇗
        </DIV></A><DIV class="menu_item"> Miscellaneous
    </DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:UCopenhagen/Articles"><DIV class="submenu_item">
                Articles
            </DIV></A><A href="https://igem-copenhagen.github.io/awesome-iGEM/"><DIV class="submenu_item">
                Awesome iGEM
            </DIV></A><A href="https://2018.igem.org/Team:UCopenhagen/Wiki_how_tos"><DIV class="submenu_item">
                Wiki How-tos
            </DIV></A></DIV><A href="https://medium.com/igem-copenhagen" style="padding-right: 0px;" target="_blank"><DIV class="menu_item direct_link"> Blog on Medium
        </DIV></A></DIV><DIV class="igem_2018_team_content pageContent"><DIV class="igem_2018_team_column_wrapper"><H1>Construct design</H1><P>
To characterize our system, we needed to make various plasmid constructs containing various combinations of suitable reporter proteins with and without secretion signals, unfolding chaperones and additional translocon protein.
</P><P>
These constructs will be used in combination with the Synthetic Injector <EM>E. coli</EM> (SIEC) strain, which have the 5 operons encoding the full injectisome chromosomally integrated under IPTG inducible promoters [1]. The constructs will allow us to characterize any combination of genes by combining up to two vectors with both arabinose and IPTG induction.
</P><H1>1. Components</H1><H2>1.1 Reporter proteins</H2><P>
Good reporters for proof of concept should be easily expressed, detectable and without post-translational modifications. We chose to work with fluorescent reporters, sfGFP, mCherry and a small luciferase enzyme, NanoLuc. This will allow us to easily characterize our system by measuring both expression and injection of protein through membranes by fluorescent measurement and fluorescent microscopy. We chose <A href="http://parts.igem.org/Part:BBa_I746916">BBa_I746916</A> (sfGFP) and <A href="http://parts.igem.org/Part:BBa_J06504">BBa_J06504</A> (mCherry) from the 2018 iGEM distribution kit and received <A href="http://parts.igem.org/Part:BBa_K1159001">BBa_K1159001</A> (NanoLuc) upon request from iGEM HQ.
</P><P>
In Charpentier and Oswald 2004 [2], β-lactamase was used as a substrate for secretion by the <EM>E. coli</EM> injectisome. β-lactamase is thus a proven substrate, but experimentally it is harder to measure its activity than the fluorescent reporters. So we decided to use beta-lactamase (<A href="http://parts.igem.org/Part:BBa_K1189008">BBa_K1189008</A>) from the 2018 iGEM distribution kit as one of our target protein.
</P><P>
Finally, we used only mCherry, sfGFP, β-lactamase for our experiments. 
</P><H2>1.2 Map20 secretion signal</H2><P>
E. coli injectisome system naturally secrete many effector proteins into its host cell. According to Charpentier and Oswald, 2004 [2], 20 amino acid sequence at the N-terminal domain of four T3SS effector proteins, Map, Cif, Tir, and EspF, is an essential and sufficient signal for E. coli injectisome transport. 
</P><P>
We chose to use 20 amino acid sequence from N-terminal part of Map protein as a T3SS secretion signal in our system, as it was demonstrated in Charpentier and Oswald, 2004 [2] to be the most efficient signal. We named it Map20. In our constructs we fused this signal N-terminally to the various reporter proteins.
</P><H2>1.3 Chaperones</H2><P>
For the Map20-tagged proteins to go through the injectisome, they will need to be in unfolded states. Charpentier and Oswald [2] showed that secretion of larger proteins through injectisomes from <EM>E. coli </EM> improves when chaperones are present. Therefore we chose to express the two <EM>E. coli</EM> chaperones CesT and CesF. These chaperones assist natural effector protein transport and are not present in the modified LEE region in SIEC strain. </P><H2>1.4 Translocon protein</H2><P>
EspD is a subunit of the injectisome which can be inserted into the host membrane. We plan to use it as a docking site to facilitate injectisome binding to the membrane. This creates a docking site, and will be useful in the case that the bacteria can not recognize and dock to the membrane we use. EspD has previously been used to create a pore on a lipid membrane (Chatterjee <EM>et al.</EM> 2015 [3]). EspD is already expressed in the SIEC strain, but it could be important to express more for higher binding efficiency to membranes.
</P><H1>2. Expression vectors with reporter proteins</H1><P>
We based our expression vectors on the <A href="http://parts.igem.org/Part:BBa_K228005">BBa_K228005</A> biobrick. In this vector, expression of a reporter is thus controlled by an arabinose-inducible promoter (<EM>pBAD</EM>). The plasmid also contains a gene encoding the arabinose-responsive regulatory protein araC under control of a pCat promoter and an chloramphenicol resistance cassette.
</P><P>
From BBa_K228005 we constructed two different backbones, either with or without the genes for the 6x-His tagged effector chaperones CesT and CesF, see <STRONG>figure 1.A</STRONG> and <STRONG>figure 1.B</STRONG>. The chaperone genes are positioned between the araC gene and the downstream terminator and are thus controlled by the constitutive promoter <EM>pCat</EM>. 
</P><P>
These backbones are combined with a reporter protein tagged or untagged with Map20 signal sequence, see <STRONG>figure 1.C</STRONG> and <STRONG>figure 1.D</STRONG>. The reporter proteins are in both cases 6x-His tagged and controlled by <EM>pBAD</EM>. The Map20 is linked to the reporter protein by a glycine-serine linker peptide.
</P><P><STRONG>Figure 1. </STRONG>A vector backbone is constructed based on BBa_K228005 either without (<STRONG>A</STRONG>) or with chaperones CesT and CesF, shown in dark green on figure, (<STRONG>B</STRONG>). A reporter gene (sfGFP, mCherry, NanoLuc or β-lactamase) is inserted downstream of the <EM>pBAD</EM> promoter. The reporter gene is either untagged (<STRONG>C</STRONG>) or tagged with Map20 (<STRONG>D</STRONG>), shown in red, at its N-terminal. The figure was made using SnapGene.
</P><P>
This way we have four different expression plasmids pr reporter protein. In total we constructed 14 different plasmids with reporter proteins. An overview of the constructs can be seen below.
</P><TABLE><TBODY><TR><TD>Plasmid constructed
   </TD><TD>Reporter
   </TD><TD>N-terminal Map20
   </TD><TD>CesT and CesF
   </TD></TR><TR><TD>+
   </TD><TD>sfGFP
   </TD><TD>+
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>sfGFP
   </TD><TD>+
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>sfGFP
   </TD><TD>-
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>sfGFP
   </TD><TD>-
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>mCherry
   </TD><TD>+
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>mCherry
   </TD><TD>+
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>mCherry
   </TD><TD>-
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>mCherry
   </TD><TD>-
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>nanoluc
   </TD><TD>+
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>nanoluc
   </TD><TD>+
   </TD><TD>-
   </TD></TR><TR><TD>-
   </TD><TD>nanoluc
   </TD><TD>-
   </TD><TD>-
   </TD></TR><TR><TD>-
   </TD><TD>nanoluc
   </TD><TD>-
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>β-lactamase
   </TD><TD>+
   </TD><TD>+
   </TD></TR><TR><TD>+
   </TD><TD>β-lactamase
   </TD><TD>+
   </TD><TD>-
   </TD></TR><TR><TD>+</TD><TD>β-lactamase
   </TD><TD>-
   </TD><TD>-
   </TD></TR><TR><TD>+
   </TD><TD>β-lactamase
   </TD><TD>-
   </TD><TD>+
   </TD></TR></TBODY></TABLE><P><B> Figure 2 </B>: Example of constructed plasmid with mCherry reporter protein
</P><H1>3. Expression vector with the translocon protein EspD</H1><P>
For expression of the translocon protein EspD, we used <A href="http://parts.igem.org/Part:BBa_K731500">BBa_K731500</A> as an expression vector. The EspD is inserted downstream of pTac promoter and can be induced. 
</P><P>
The plasmid, seen on <STRONG>figure 3</STRONG>, encodes 6His-tagged EspD translocon under pTac, which can be induced by IPTG. The EspD translocon protein self-assembles into a hydrophobic membrane protein complex that is sufficient for membrane penetration and pore formation. This translocon protein is normally secreted along with the EspB hydrophilic translocon to penetrate the host cell membrane and serve as a docking site for the injectisome. This 6His-tagged EspD can be purified by Ni-NTA column or TALON column. It might be used to pre-assemble a translocon on a target membrane to induce injectisome docking on an artificial membrane.
</P><P><STRONG>Figure 3.</STRONG> EspD Translocon expression plasmid. The plasmid encodes the translocon protein EspD (shown in purple on the figure) in a BBa_K731500 backbone. The figure was made using SnapGene.
</P><H1>4. Experimental plan</H1><P>
The flow charts below show the experimental plan that was the basis of our work over the summer. Read more about the cloning strategy that we used when assembling our constructs <A href="https://2018.igem.org/Team:UCopenhagen/Experiments">here</A>.
</P><P><STRONG>Figure 4.</STRONG> Flowchart describing the laboratory work concerning constructing expression vectors and finally transforming the SIEC strains with the plasmids.
</P><P><STRONG>Figure 5.</STRONG> Flow chart describing the different steps in designing and characterizing a home made chamber for protein production. We use an egg yolk membrane and test for leakiness of the system. 
</P><P><STRONG>Figure 6.</STRONG> Flow chart describing the basis of our experiments combining membranes and SIEC bacteria. Initially we had planned to work with liposomes and an artificial membrane, which was later replaced with experiments with egg yolk membrane and plant protoplasts.
</P><P><STRONG>Figure 7.</STRONG> Flow chart describing experimental troubleshooting plan for secretion signal.
</P><H1>References</H1><P><STRONG>[1] </STRONG>Ruano-Gallego, D., Álvarez, B., Fernández, L. A. (2015) Engineering the Controlled Assembly of FIlamentous Injectisomes in E. Coli K-12 for Protein Translocation into Mammalian Cells. ACS Synth. Biol. 4, pp 1030-1041.
</P><P><STRONG>[2]</STRONG> Charpentier, X., Oswald, E. (2004) Identification of the Secretion and Translocation Domain of the Enteropathogenic and Enterohemorrhagic Escherichia coli Effector Cif, Using TEM-1 β-Lactamase as a New Fluorescence-Based Reporter. J Bacteriol, 186(16), pp 5486-5495.
</P><P><STRONG>[3]</STRONG> Chatterjee, A., Caballero-Franco, C., Bakker, D., Totten, S., Jardim, A. (2015) Pore-forming Activity of the Escherichia coli Type III Secretion System Protein EspD. J Biol Chem. 290 (42) pp. 25579-25594.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>