Team:Wageningen UR/Results/Phage Display
Project
Background
Wet-lab
Results
BioBricks
Model
Cpx Kinetics
Quorum Sensing
Cpx Integration
QS Integration
Human Practices
Integrated HP
Safety & Ethics
Collaborations
Future Perspectives
Engagement
Meetup
Application
Demonstrate
Device
Roll-out Plan
Notebook
Protocols
InterLab
Lab Journal
Achievements
Team
Members
Attributions
Acknowledgements
Project
Background
Wet-lab
Model
Human Practices
Application
Notebook
Team
Wet-lab
Model
Application
Human Practices
Notebook
Team
Parts
Results
Cpx Kinetics
Quorum Sensing
Cpx integration
QS integration
Integrated HP
Safety & Ethics
Collaborations
Future Perspectives
Engagement
Meetup
Demonstrate
Device
Rollout Plan
Protocols
InterLab
Journal
Achievements
Members
Attributions
Phage display
Phage generation
Selection
Home
Wet-lab
Results
Phage Display
Phage Display
Phage display is a powerful laboratory technique used to characterize protein-ligand interactions and functions. Phage display can be used to select for proteins with the desired properties from large libraries. Using M13 phage display, we can select for the best affinity body against Chikungunya virus (CHIKV) and Human African Trypanosoma (HAT) antigens using the affinity body library we created. These affinity bodies can then be used to make the Mantis cells detect these diseases.
Generation of recombinant M13 phages
In M13 phage display, the DNA sequence encoding the protein of interest (in our case the affinity body library) is fused to a gene which codes for the M13 phage G3P minor coat protein. After correct assembly, the phage will display the affinity body on its surface, linking the genotype to the phenotype. To fuse the affinity body to the phage coat protein, two distinct strategies can be applied. The first method is to directly clone the affinity body sequence into the coding sequence of the coat protein in the phage genome. The other option is to use a small vector (phagemid vector) encoding for part of the phage G3P minor coat protein. The protein of interest can be cloned into the phagemid vector and will be expressed as a fusion with G3P (p III in Figure 1). The advantage of using phagemids is the ease of cloning into the smaller vector, as efficiency is important during library cloning. Furthermore, using a phagemid results in a monovalent display of the protein of interest compared to a polyvalent display with the phage vector [1]. This is an important difference since a polyvalent display could lead to the selection of weak-binding variants while this is circumvented with a monovalent display. However, the phagemid vector approach must be used in combination with a helper phage. In our case, we use the VCSM13 helper phage which is an M13-based phage and has a defective origin of replication. The phagemid vector has a working origin of replication and this leads to the preference for packing the phagemid vector in the phage particles. Taking this in consideration, we chose the phagemid vector approach.
Figure 1: Schematic picture M13 phage, made by iGEM Edinburgh 2011 .
Proof-of-principle
Before generating a phage library we performed a proof-of-principle with the wild type (WT) affinity body sequence. This affinity body specifically binds Immunoglobulin G (IgG) antibodies. The WT affinity body sequence was ligated into the pComb3XSS phagemid. XL1-Blue cells were transformed with the phagemid and subsequently used to create recombinant phages containing the WT affinity body (Figure 2). The phages were amplified and titrated. In order to see whether the phagemid was preserved, we infected fresh XL1-Blue cells with the recombinant phages. The infected cell suspension was plated on LB agar plates with the appropriate antibiotic. Several colonies were picked, grown, plasmids were isolated and sent for sequencing.
Figure 2: Overview of the creation of the proof-of-principle recombinant M13 phages carrying the IgG affinity body sequence.
Approach
After ligation of the wild type affinity body sequence into the pComb3XSS phagemid, the mixture was electroporated into XL1-Blue Escherichia coli (E. coli) cells. The colonies on the transformation plates were scraped together and suspended in 50 mL of 2xYT medium supplemented with 1% (w/v) glucose and 100 ug/mL ampicillin. To this culture, 250 mL of fresh medium was added and grown in 1L flasks until an OD600 of ~0.6 was reached. VCSM13 helper phages were added at an MOI of 20 and incubated at 37°C without shaking. After incubation, the medium was changed to a 2xYT medium supplemented with 1% (w/v) glucose, 100 ug/mL ampicillin, 50 ug/mL kanamycin and 0.25mM IPTG and incubated overnight at 30°C. The next day the phages were precipitated and a titration was performed to assess infectivity.
The resulting recombinant phage with wild type affinity body has an infective titer of 9.6*1011 CFU/mL (colony forming units per mL). We visualized these phages using transmission electron microscopy (TEM) (Figure 3).
Figure 3: Transmission Electron Microscopy (TEM)  images of recombinant M13 phage expressing WT affinity body. Stained using phosphotungstic acid (PTA). Pictures were taken using JEOL JEM1400.
×
Figure 4: Multiple sequence alignment of 3 random colonies after infection of fresh XL1-blue cells with the WT-IgG phages. Sequences align and proves the phagemid is preserved correctly.
Figure 4: Multiple sequence alignment of 3 random colonies after infection of fresh XL1-blue cells with the WT-IgG phages. Sequences align and prove the phagemid is preserved correctly.
Phage library
The proof-of-principle confirmed that infecting XL1-Blue cells carrying the pComb3XSS phagemid with helper phages results in recombinant phages. Therefore we proceeded with the creation of the phage library using the the affinity body library created here (Figure 5).
Figure 5: Overview of the creation of the recombinant M13 phages carrying affinity body library.
Approach
The ligation of the WT affinity body and library into the pComb3XSS vector has been described here. After ligation, the mixture was electroporated into XL1-Blue E. coli cells. The colonies on the transformation plates were scraped together and suspended in 50 mL of 2xYT medium supplemented with 1% (w/v) glucose and 100 ug/mL ampicillin. To this culture, 250 mL of fresh medium was added and grown in 1L flasks until an OD600 of ~0.6 was reached. VCSM13 helper phages were added at an MOI of 20 and incubated at 37°C without shaking. After incubation, the medium was changed to a 2xYT medium supplemented with 1% (w/v) glucose, 100 ug/mL ampicillin, 50 ug/mL kanamycin and 0.25mM IPTG and incubated overnight at 30°C. The next day the phages were precipitated and a titration was performed to assess infectivity.
The resulting recombinant phage library has an infective titer of 5.7*1015 CFU/mL (colony forming units per mL). This library can then be used to screen for the appropriate affibody against any antigen.
Selection
Antigens were produced for CHIKV, Zika virus (ZIKV) and Mayaro virus (MAYV) in the form of Virus-Like Particles (VLPs). Strep-tagged surface proteins were produced for CHIKV and HAT. In order to purify the VLPs and attach them to beads, antigens for the specific virus particles can be linked to protein-A resin beads, washed, blocked and used for phage display. However, due to the cost of the antigens and required resin beads, we chose to continue with only the strep-tagged surface proteins, attached to strep-tactin sepharose beads.
The phage library was loaded on the beads, washed and then eluted. The eluted phage was then amplified and used again for a second and third round of biopanning, increasing the stringency of the wash with each step. In the last step, the eluted phage was used to infect XL1-Blue cells. Colony PCR is then used to amplify the phagemids and sent for sequencing to confirm the sequence of the best binding affinity body (Figure 6).
Figure 6: Overview of phage display biopanning including the library creation.
Approach
The beads containing the CHIKV and HAT antigens were washed and subsequently blocked using PBS-TWEEN® with gelatin. Milk or BSA was not used for blocking due to the possible presence of immunoglobulins which is the natural target of our wild type affinity body. Next, the phage library was added and after incubation at room temperature, the unbound phages were washed away using PBS-TWEEN®. The phages that were still bound to the beads were eluted, titrated and amplified. After amplification, the phages were titrated again and the biopanning was repeated. Stringency was increased with each round of panning by increasing the TWEEN® concentration during the washing step. The TWEEN® concentrations in round 1, 2 and 3 were 0.1%, 0.3% and 0.6% respectively. The exact protocols used can be found here .
Results
After each round of biopanning, the eluted and amplified phages were titrated using the Colony Forming Units (CFU) protocol. In order to perform CFU, dilutions of the phage solution were made in 20 mM Tris-HCl from 10-8 to 10-13. These phage dilutions were then plated together with XL1 cells. When a phage with our construct infects a healthy XL1 E. coli, it transfers the ampicillin (Amp) resistance gene that is present on the pComb3XSS vector. Therefore, the selection is possible on Amp plates.
We found that during the first rounds of CFU titration, all plates showed similar amounts of colonies on each plate. As we previously had a contamination in our XL1 stock with an unknown Amp resistant bacteria we chose to repeat the CFU titration with fresh cells. However, due to time constraints, we had to continue with the biopanning rounds without knowing the titers of the elution and amplification fractions. During the second CFU titration, the results were similar to the first round. All plates showed a similar amount of colonies. We ruled out a contamination with an Amp-resistant bacteria. Taking into consideration that we loaded the initial biopanning round with 1015 phages we hypothesized that we had to extend the dilution series from 108-1013 to 1010- 1020. The next day, the plates showed again the same results, with similar amounts of colonies on every plate. As the dilutions are extremely high it is very unlikely that these colonies represent the amount of phages in our fractions.
As a troubleshooting step we performed a titration round with all buffers used in the protocol without adding our phage samples. Here, we found that we had a major contamination in one of the buffers. Due to the approaching deadlines, we could not repeat the entire biopanning experiment. The results are therefore inconclusive.
What we would have expected is an enrichment after each biopanning step with the use of the titration data. After the final biopanning step we would expect a reasonable amount of candidates with high affinity for our antigens. In our phagemid, there is an amber stop codon present at the fusion site between the G3P protein and our affinity bodies. Using an amber stop codon non-suppressor strain our affinity body candidates could be expressed. As there is also a 6xHis-tag and HA-tag present on the affinity bodies, they could be purified and used for binding assays.
Once the affinity body candidates were purified, a ForteBio Octet® could be used to determine the full kinetic profile of the biomolecular interactions of the candidate group to the purified antigens. These affinity measurements offer a real-time interaction of the molecular kinetics, better than the estimations which can be made by other methods such as ELISA. We could compare these kinetics to the antibodies commercially available for the antigens in order to select the best binding affinity body, candidate. The best affinity body that could be selected is used in the Mantis diagnostic.
Even though we were unable to select for an affinity body against the tested viruses, the library we have developed and the presented methodology will be of use for future iGEM teams and external researchers interested in creating diagnostics against particular pathogens.
References
Paschke, Matthias. "Phage display systems and their applications." Applied microbiology and biotechnology 70.1 (2006): 2-11.
iGEM Wageningen 2017. Send us an email or follow us on Facebook
