Team:Wageningen UR/Results/Affinity Bodies
Project
Background
Wet-lab
Results
BioBricks
Model
Cpx Kinetics
Quorum Sensing
Cpx Integration
QS Integration
Human Practices
Integrated HP
Safety & Ethics
Collaborations
Future Perspectives
Engagement
Meetup
Application
Demonstrate
Device
Roll-out Plan
Notebook
Protocols
InterLab
Lab Journal
Achievements
Team
Members
Attributions
Acknowledgements
Project
Background
Wet-lab
Model
Human Practices
Application
Notebook
Team
Wet-lab
Model
Application
Human Practices
Notebook
Team
Parts
Results
Cpx Kinetics
Quorum Sensing
Cpx integration
QS integration
Integrated HP
Safety & Ethics
Collaborations
Future Perspectives
Engagement
Meetup
Demonstrate
Device
Rollout Plan
Protocols
InterLab
Journal
Achievements
Members
Attributions
Results
Introduction
Results
Home
Results
Affinity Body Library
Affinity Body Library
The goal of this project was to create a large library of affinity bodies. This library was created to screen for specific binding proteins for different disease antigens with the use of phage display . The modularity and specificity of the Mantis diagnostic system is based on the use of the specific affinity bodies. The approach taken in this project proved to be a reliable way to create such a library without any apparent bias.
Introduction
Figure 1: Affinity body 3D structure of the 3 helices [4].
Affinity bodies are antibody mimetics based on staphylococcal protein A (SPA) [1]. The small, 6kDa, affinity proteins are based on the Z domain of the cell-wall anchored bacterial protein A. This protein binds immunoglobulin and contributes to evading the immune system [2]. By changing 13 amino acids on 2 helices essential for specificity, affinity bodies for a wide variety of targets can be developed (Figure 1). Since its discovery, affinity bodies have been generated to target insulin, fibrinogen, transferrin, tumor necrosis factor-a, IL-8, gp120, CD28, human serum albumin, IgA, IgE and HER2. Affinity bodies can be used for imaging, purification, detection and many therapeutic applications [3].
Approach
Construct
The vector used to make the library with is a phagemid pComb3XSS, acquired from AddGene (Figure A). The pComb3XSS vector has an origin of replication for both Escherichia coli and filamentous phage M13. By using the SacI and SpeI restriction sites, any protein of interest can be expressed, fused to the G3P protein. This protein is incorporated in the M13 helper phages upon infection of bacteria carrying this phagemid.
×
Figure A: pComb3XSS phagemid vector used for the library creation. The vector includes ampicillin resistance and origin of replications for E. coli and the M13 phage. Click for larger figure.
Figure A: pComb3XSS phagemid vector used for the library creation. The vector includes ampicillin resistance and origin of replications for E. coli and the M13 phage. Click for larger figure.
The amino acid sequence of the wild type Immunoglobin G (IgG)-binding affinity body is depicted in Figure B. The amino acid residues that are responsible for specific binding (red) will be targeted for randomization in the creation of the library.
Figure B: Amino acid sequence of wild-type IgG binding affinity body. Amino acid residues targeted for randomization in library creation in red.
The Helix 3 region of the affinity body is not responsible for the binding specificity and will not be randomized. Therefore, the Helix 3 region is ligated into the backbone before the library is integrated for an easier library ligation. The Helix 3 fragment was amplified with primers in such a way that it can be ligated into the pComb3XSS vector with the existing SacI/SpeI restriction sites. However, a type-II restriction site (BsaI) was incorporated into the fragment to allow for the library integration without leaving a scar (Figure C).
Figure C: Overview of the Helix 3 fragment with SacI/SpeI restriction sites for cloning into the pComb3XSS vector. The internal BsaI restriction site allows for scarless integration of the Helix 1/Helix 2 library.
Oligo fragments were used to create the Helix 1 and Helix 2 fragments with random nucleotides on the desired places. The oligo’s are designed to contain a NNK(K = G/T) degeneracy at the amino acid residues of interest. The NNK degeneracy improves the amount of non-sense codons produced by a NNN degeneracy and reduces the amount of stop codons as well [5]. The annealed fragments for Helix 1 (top) and Helix 2 (bottom) can be seen in Figure D.
Figure D: Top: Fragment of annealed oligo's designed for Helix 1 with a NNK degeneracy. Bottom: Fragment of annealed oligo's designed for Helix 2 with a NNK degeneracy.
The Helix 1 and Helix 2 fragments were ligated into the linearized backbone (SacI/BsaI) and the ligation product was used for the transformation of XL1-Blue cells. The XL1-Blue cells have the following genotype: recA1 endA1 gyrA96 thi-1 hsdR17 supE44 relA1 lac [F ́proAB lacI qZ∆M15 Tn10 (Tetr)]. Importantly the strain used has an F pilus, which is essential for the attachment of the M13 phages.
Results
After 25 transformations the colonies were counted and all pooled together which in total yielded an estimated library size of 110,000 affinity bodies. To check whether the library has a bias towards certain nucleotides and therefore amino acids, 96 colony PCRs were performed on one of the transformations.
The PCR products were sent for sequencing and from 86 successful PCR reactions the data is depicted in Figure 2. The rest of the PCR products had regions of bad sequencing quality and were discarded. On the X-axis all the randomized nucleotide places (39) can be seen. On the Y-axis the total amount of sequenced samples is given. So each column represents a randomized nucleotide place divided into the 4 base pairs (ATCG). As expected, in every third column there are only the G and T base pair. All the columns show a very similar pattern and indicates that there is no significant bias towards any of the base pairs overall.
Figure 2: Sequencing data from 86 colony PCRs on a transformation of the library mixture. Each column represents a randomized nucleotide position and is divided into the 4 possible base pairs (ATCG).
To further investigate the pattern that is seen, the average occurrence was plotted in Figure 3 and the data is normalized to the expected occurrence for each of the base pairs in a truly random library. Statistical analysis showed that there is no significant difference with the expected occurrence (control) and each of the base pairs (P < 0.05).
Figure 3: The average occurrence for each of the 4 basepairs, normalized to the expected occurrence (control), is plotted on the Y-axis. All columns are compared to the control and no significant correlation was found (P < 0.05).
References
Nord, Karin, et al. "Binding proteins selected from combinatorial libraries of an α-helical bacterial receptor domain." Nature biotechnology 15.8 (1997): 772-777.
Nord, Karin, et al. "A combinatorial library of an α-helical bacterial receptor domain." Protein Engineering, Design and Selection 8.6 (1995): 601-608.
Löfblom, John, et al. "Affibodies: engineered proteins for therapeutic, diagnostic and biotechnological applications." FEBS letters 584.12 (2010): 2670-2680.
Lendel, Christofer, Jakob Dogan, and Torleif Härd. "Structural basis for molecular recognition in an affibody: affibody complex." Journal of molecular biology 359.5 (2006): 1293-1304.
Hughes, Marcus D., et al. "Removing the redundancy from randomised gene libraries." Journal of molecular biology 331.5 (2003): 973-979.
iGEM Wageningen 2017. Send us an email or follow us on Facebook
