<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Results_Viral_Antigens skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Results/Viral Antigens</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="Main-Menu"><DIV class="col-lg-10 MainMenu-Center-Column MenuCenter"><DIV class="Project MenuButtonColumn"><DIV class="Project-Top TopMenuButtonColumn"><UL><LI class="project-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Project" class="nav-item" id="project">Project</A></LI></UL></DIV></DIV><DIV class="Background MenuButtonColumn"><DIV class="Background-Top TopMenuButtonColumn"><UL><LI class="background-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Background" class="nav-item active" id="background">Background</A></LI></UL></DIV></DIV><DIV class="Wet-Lab MenuButtonColumn"><DIV class="Wet-Lab-Top TopMenuButtonColumn"><UL><LI class="wet-lab-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Wet-lab" class="nav-item active" id="wet-lab">Wet-lab</A></LI></UL></DIV><DIV class="Wet-Lab-Bottom SubMenuButtonColumn"><UL id="wet-lab-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results" class="results-sub-nav-item sub-nav-item">Results</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Parts" class="parts-sub-nav-item sub-nav-item">BioBricks</A></LI></UL></DIV></DIV><DIV class="Model MenuButtonColumn"><DIV class="Model-Top TopMenuButtonColumn"><UL><LI class="model-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Model" class="model nav-item" id="model">Model</A></LI></UL></DIV><DIV class="Model-Bottom SubMenuButtonColumn"><UL id="model-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics" class="kinetics-sub-nav-item sub-nav-item">Cpx Kinetics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/QS" class="qs-sub-nav-item sub-nav-item">Quorum Sensing</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx" class="integrationcpx-sub-nav-item sub-nav-item">Cpx Integration</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS" class="integrationQS-sub-nav-item sub-nav-item">QS Integration</A></LI></UL></DIV></DIV><DIV class="Human-Practices MenuButtonColumn"><DIV class="Human-Practices-Top TopMenuButtonColumn"><UL><LI class="human-practices-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Human_Practices" class="nav-item" id="human-practices">Human Practices</A></LI></UL></DIV><DIV class="Human-Practices-Bottom SubMenuButtonColumn"><UL id="human-practices-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated" class="integrated-sub-nav-item sub-nav-item ">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Silver" class="safety-sub-nav-item sub-nav-item">Safety &amp; Ethics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Collaborations" class="collaboration-sub-nav-item sub-nav-item">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Future" class="future-sub-nav-item sub-nav-item">Future Perspectives</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Engagement" class="engagement-sub-nav-item sub-nav-item">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Meetup" class="meetup-sub-nav-item sub-nav-item">Meetup</A></LI></UL></DIV></DIV><DIV class="Application MenuButtonColumn"><DIV class="Application-Top TopMenuButtonColumn"><UL><LI class="application-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Applied_Design" class="nav-item" id="application">Application</A></LI></UL></DIV><DIV class="Application-Bottom SubMenuButtonColumn"><UL id="application-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Demonstrate" class="demo-sub-nav-item sub-nav-item">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Hardware" class="device-sub-nav-item sub-nav-item">Device</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Application/Rollout" class="rollout-sub-nav-item sub-nav-item">Roll-out Plan</A></LI></UL></DIV></DIV><DIV class="Notebook MenuButtonColumn"><DIV class="Notebook-Top TopMenuButtonColumn"><UL><LI class="notebook-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook" class="notebook nav-item" id="notebook">Notebook</A></LI></UL></DIV><DIV class="Notebook-Bottom SubMenuButtonColumn"><UL id="notebook-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols" class="protocols-sub-nav-item sub-nav-item">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/InterLab" class="interlab-sub-nav-item sub-nav-item">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Journal" class="journal-sub-nav-item sub-nav-item">Lab Journal</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Achievements" class="achievements-sub-nav-item sub-nav-item">Achievements</A></LI></UL></DIV></DIV><DIV class="Team MenuButtonColumn"><DIV class="Team-Top TopMenuButtonColumn"><UL><LI class="team-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Team" class="team nav-item" id="team">Team</A></LI></UL></DIV><DIV class="Team-Bottom SubMenuButtonColumn"><UL id="team-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Team/Members" class="members-sub-nav-item sub-nav-item">Members</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Attributions" class="attributions-sub-nav-item sub-nav-item">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Acknowledgements" class="Acknowledgements-sub-nav-item sub-nav-item">Acknowledgements</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="MobileMenu"><UL id="MainMobileMenu"><LI><BUTTON class="project-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Project'" type="button">Project</BUTTON></LI><LI><BUTTON class="background-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Background'" type="button">Background</BUTTON></LI><LI><BUTTON class="wet-lab-li-mo">Wet-lab</BUTTON></LI><LI><BUTTON class="model-li-mo">Model</BUTTON></LI><LI><BUTTON class="human-practices-li-mo">Human Practices</BUTTON></LI><LI><BUTTON class="application-li-mo">Application</BUTTON></LI><LI><BUTTON class="notebook-li-mo">Notebook</BUTTON></LI><LI><BUTTON class="team-li-mo">Team</BUTTON></LI></UL><UL id="MobileSubMenu"><LI class="wet-lab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Wet-lab'" type="">Wet-lab</BUTTON></LI><LI class="model-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model'">Model</BUTTON></LI><LI class="application-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Applied_Design'">Application</BUTTON></LI><LI class="human-practices-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Human_Practices'">Human Practices</BUTTON></LI><LI class="notebook-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook'">Notebook</BUTTON></LI><LI class="team-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team'">Team</BUTTON></LI><LI class="parts-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Parts'" type="button">Parts</BUTTON></LI><LI class="results-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Results'" type="button">Results</BUTTON></LI><LI class="kinetics-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics'" type="button">Cpx Kinetics</BUTTON></LI><LI class="qs-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/QS'" type="button">Quorum Sensing</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx'" type="button">Cpx integration</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS'" type="button">QS integration</BUTTON></LI><LI class="integrated-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated'" type="button">Integrated HP</BUTTON></LI><LI class="safety-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Silver'" type="button">Safety &amp; Ethics</BUTTON></LI><LI class="collaboration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Collaborations'" type="button">Collaborations</BUTTON></LI><LI class="future-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Future'" type="button">Future Perspectives</BUTTON></LI><LI class="engagement-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Engagement'" type="button">Engagement</BUTTON></LI><LI class="meetup-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Meetup'" type="button">Meetup</BUTTON></LI><LI class="demo-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Demonstrate'" type="button">Demonstrate</BUTTON></LI><LI class="device-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Hardware'" type="button">Device</BUTTON></LI><LI class="rollout-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Application/Rollout'" type="button">Rollout Plan</BUTTON></LI><LI class="protocols-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols'" type="button">Protocols</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/InterLab'" type="button">InterLab</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Journal'" type="button">Journal</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Achievements'" type="button">Achievements</BUTTON></LI><LI class="members-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team/Members'" type="button">Members</BUTTON></LI><LI class="attributions-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Attributions'" type="button">Attributions</BUTTON></LI></UL></DIV><DIV class="container-fluid OurContent"><DIV class="row"><DIV class="col-lg-8"><DIV class="Main-Border"><DIV id="SideMenu-Wrapper"><SIDENAV id="SideMenu"><DIV class="menu-head"><H4>Viral Antigens</H4></DIV><UL class="sidebar-nav"><LI class="menu-item"><A href="#Intro">Introduction</A></LI><LI class="menu-item"><A href="#VLP">Virus Like Particles</A></LI><LI><A href="#Surface">Surface Antigens</A></LI></UL></SIDENAV></DIV><DIV class="Main-Center-Content-Column Overview"><DIV id="breadcrumb-wrapper"><UL class="breadcrumb"><LI><A href="https://2017.igem.org/Team:Wageningen_UR">Home</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Wet-lab">Wet-lab</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI>Viral Antigens</LI></UL></DIV><SECTION id="Intro"><SECTION class="deviceintro"><DIV class="Title"><H1>Production of Viral Antigens</H1></DIV><DIV class="Textbox Results-Desc"><P>In order to safely produce viral antigens for phage display, two different approaches were taken. The first focuses on the production of enveloped Virus-Like-Particles (VLPs) for chikungunya (CHIKV), Zika (ZIKV) and Mayaro (MAYV). These particles are noninfectious and widely used as vaccines for various viral diseases. The second approach is to produce secreted Strep-tagged antigens for alphaviruses. These antigens can be purified easier and can be directly linked to Strep-tactin beads for antigen display in the <A href="https://2017.igem.org/Team:Wageningen_UR/Results/Phage_Display" target="_blank">phage-panning.</A></P></DIV></SECTION></SECTION><SECTION id="VLP"><SECTION class="qs"><DIV class="Title"><H2>Virus-Like Particles</H2></DIV><DIV class="Textbox Title"><P>
Three viruses were chosen for the production of antigens; two alphaviruses (CHIKV and MAYV) and one flavivirus (ZIKV). The VLPs were produced using the Invitrogen<SUP>TM</SUP> Bac-to-Bac<SUP>®</SUP> Baculovirus Expression System [1]. The baculovirus expression system is widely used in biotechnology for high yield recombinant protein expression by exploiting the life cycle of the virus in insect cells [2,3]. The advantage of using insect cells is the accurate post-translational modifications (phosphorylation, chaperone-assisted folding, glycosylation etc.) which cannot be achieved in <I>E. coli.</I>. To generate the CHIKV, ZIKV and MAYV VLPs, the genes encoding the viral structural proteins for the specific virus were placed downstream of the very strong polyhedrin promotor of the baculovirus, in this case <I>Autographa californica</I> multicapsid nucleopolyhedrovirus (AcMNPV). The recombinant baculovirus DNA is then used to transfect a <I>Spodoptera frugiperda (Sf-21)</I> cell line in order to produce recombinant baculoviruses expressing the VLPs. </P><P>
Wageningen University &amp; Research has constructs available for ZIKV and CHIKV [4], but not for MAYV. As the generation of such virus-like particles is similar, only the generation of MAYV VLPs will be described below.

</P></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#Approach1" aria-expanded="true" aria-controls="Approach1"><DIV class="col-xs-11">
Approach

</DIV></A></H4></DIV><DIV id="Approach1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
MAYV genomic RNA was isolated from a Trizol sample. After the integrity and yield of the MAYV RNA was checked, cDNA was generated using Superscript RTII according to the <A href="https://static.igem.org/mediawiki/2017/6/6c/Cloning_and_DNA_Manipulation.pdf" target="_blank">cDNA generation protocol.</A> Gene-specific primers were ordered to amplify the structural polyprotein gene with attB1/2 overhangs (Gateway), after which Phusion PCR was used to generate the MAYV amplicon.
A BP Gateway reaction was used to insert the MAYV amplicon in a pDONR207 vector, followed by an LR Gateway reaction to move the fragment to the pDEST8 baculovirus expression vector. In parallel the fragment was directly cloned into pFastBacDual (pFBD), another baculovirus expression vector used in the Bac-to-Bac® system. Both vectors (pFBD and pDEST8) place the MAYV polyprotein gene downstream the strong polyhedrin promotor. The plasmids were checked using restriction analysis, colony PCR and sequencing. The resulting expression clones are abbreviated MAYV-D (pDEST8 derived) and MAYV-F (pFBD derived). Both constructs were used in the Bac-to-Bac® system, the resulting 136kb bacmids (plasmids containing AcMNPV DNA) were confirmed using colony PCR and isolated using the <A href="https://static.igem.org/mediawiki/2017/3/32/VLP_Generation.pdf" target="_blank">bacmid isolation protocol.</A>
Bacmids of MAYV-D and MAYV-F were used to transfect an <I>Sf-21</I> at cell-line using ExpreS2 transfection reagent in a 6 wells plate, a healthy cell line was used as control/mock. The baculovirus infection was observed for 72 hours. The medium was then replaced, and the cells were moved to a T25 flask. After 48 hour the infection was clearly visible, as can be observed in Figure A.


</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure A: </B> Transfection of <I>Sf</I>-21 with bacmid containing MAYV structural casette. T=48 hour, prior to harvesting. 
</DIV></DIV><P>

The infected cells are enlarged, and show some degree of capsid bodies in the nucleus, which might indicate the presence of MAYV capsids. The AcMNPV-MAYV-D and AcMNPV-MAYV-F baculoviruses were harvested, titrated, and the passage 0 (P0) stock was used to infect <I>Sf21</I> cells in a T25 flask at 50% confluency with an multiplicity of infection (MOI) of 5. Six days post infection the cells and medium were harvested. 
</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure B: </B> Western blot of <I>Sf/</I>-21 cells infected with AcMNPV-MAYV, cell fraction and medium fraction, 5.5D11 Alphavirus capsid antibody. Expected size: 35kDa. Blot on right is a repead due to overloading of CHIKV lane (positive control). The coloring reaction was performed using secondary antibodies with alkaline phosphatase.
</DIV></DIV><P>
Western blot with specific antibody (<A href="https://static.igem.org/mediawiki/2017/3/32/VLP_Generation.pdf" target="_blank">Semi-dry blotting protocol</A>) was used to determine the presence of the alphavirus capsid, as seen in Figure B. Furthermore, 10 ml of medium was precipitated and VLP's were purified using ultracentrifugation (UC) accoding to the <A href="https://static.igem.org/mediawiki/2017/3/32/VLP_Generation.pdf" target="_blank">UC protocol</A>. The isolated fraction was analyzed using Transmission Electron Microscopy, as can be seen in Figure 1.
</P></DIV></DIV></DIV></DIV><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 1: </B>Transmission Electron Microscopy of MAYV VLP samples, stained with uranyl acetate. Pictures taken using JEOL JEM1400.
</DIV></DIV><P>
Figure 1 shows the particles in the purified VLP fractions. The particle are spherical and sized approximately 70nm in diameter. Immunogold staining could be used to confirm this hypothesis. Virus-Like Particles for chikungunya and Zika virus were produced in a similar fashion.
</P></SECTION></SECTION><SECTION id="Surface"><SECTION class="qs"><DIV class="Title"><H2>Strep-tagged surface proteins</H2></DIV><DIV class="Textbox Title"><P>
Next to producing VLPs, strep-tagged surface glycoproteins of CHIKV were also produced. The benefit of these proteins is the ease of purification compared to VLPs, whilst offering comparable antigenic properties. The schematic structure of the surface glycoproteins is depicted in Figure 2 [5]. The native CHIKV polyprotein is cleaved into the individual E1 and E2 glcyoproteins, which are transported to the plasma membrane. There, they are taken up by budding nucleocapsids, resulting in mature, enveloped virions [6]. The introduction of a linker between E2 and E1 as well as the instruction of a strep-tag bypasses the need for membrane anchoring and yields secreted proteins, which are easily purified. The constructs for CHIKV secreted glycosylated trimeric E-protein were produced by a MSc student at the Wageningen University &amp; Research. As with the production of the VLPs, the production of the Strep-tagged CHIKV-E uses the Bac-to-Bac<SUP>®</SUP> system as well.
</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 2: </B> Schematic overview of CHIKV-E protein membrane organization of native and recombinant surface proteins, adapted from Voss <I>et al.</I> (2010).
</DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#Approach2" aria-expanded="true" aria-controls="Approach2"><DIV class="col-xs-11">
Approach

</DIV></A></H4></DIV><DIV id="Approach2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
An AcMNPV clone containing the Strep-tagged CHIKV-E was available at the Laboratory of Virology at Wageningen University &amp; Research. <I>Sf21</I> cells were infected with a MOI of 5. The infection was observed for 7 days, after which the infection was harvested. As the CHIKV-E spikes are secreted, the medium fraction could be used directly. The binding of the CHIKV-E protein was tested using Sepharose-Strep-Tactin beads. 2ml of 50% resin beads suspension was added to 10ml harvested medium and incubated for 1 hour at 4C. The beads were washed and eluted using biotin in an empty column. Various fractions were loaded on an SDS gel and subsequently semi-dry-blotted. Western blot analysis with a specific antibody for CHIKV-E was used to detect the CHIKV-E protein.


</P></DIV></DIV></DIV></DIV><DIV class="col-lg-5 banner-column" style="float: right;"><DIV class="figure-fullwidth"><DIV class="figure-center-imagebox.Banner-box"><DIV class="figure-center-caption"><B>Figure 3: </B>Western blot of Sepharose-Strep-Tactin beads with CHIKV-E spikes. Fractions: pre, first flow through, last flow trough, last wash, elution 1-5.
</DIV></DIV></DIV></DIV><P>
As can be seen in Figure 3, the strep-tactin sepharose beads are capable of catching the strep-tagged CHIKV-E proteins. The experiment was repeated to cover a total of 6ml strep-tactin sepharose beads to be used in <A href="https://2017.igem.org/Team:Wageningen_UR/Results/Phage_Display" target="_blank">phage-panning</A> later on.
 
The production of the secreted CHIKV-E construct is defined by Voss <I>et al.</I> and can be applied to Mayaro as well, as this virus is similar to chikungunya. For Zika, a different approach can be taken where the E protein present in the surface of the virion can be tagged [7]. Due to time constraints we chose not to pursue such methods and focus on the CHIKV-E proteins.
 </P></DIV></SECTION></SECTION><SECTION class="qs"><DIV class="Textbox Citations"><H3>
References
</H3><OL><LI>Invitrogen. Bac-to-Bac<SUP>®</SUP> Baculovirus Expression System, User Guide. A.0. Life Technologies Corporation; 2010.</LI><LI>Kost TA, Condreay JP, Jarvis DL. Baculovirus as versatile vectors for protein expression in insect and mammalian cells. Nat Biotechnol. 2005;23(5):567-575. doi:10.1038/nbt1095.</LI><LI>Rohrmann GF. Baculovirus Molecular Biology. 3rd ed. Bethesda (MD): National Center for Biotechnology Information (US); 2013. doi:NBK114593.</LI><LI>Metz SW, Gardner J, Geertsema C, et al. Effective Chikungunya Virus-like Particle Vaccine Produced in Insect Cells. PLoS Negl Trop Dis. 2013;7(3). doi:10.1371/journal.pntd.0002124.</LI><LI>Voss JE, Vaney M-C, Duquerroy S, et al. Glycoprotein organization of Chikungunya virus particles revealed by X-ray crystallography. Nature. 2010;468(7324):709-712. doi:10.1038/nature09555.</LI><LI>Rashad AA, Mahalingam S, Keller PA. Chikungunya Virus : Emerging Targets and New Opportunities for Medicinal Chemistry. 2013. doi:10.1021/jm400460d.</LI><LI>Barba-Spaeth, Giovanna, et al. &quot;Structural basis of potent Zika–dengue virus antibody cross-neutralization.&quot; Nature 536.7614 (2016): 48-53.</LI></OL></DIV></SECTION></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid FooterWrapper"><DIV class="FooterImage"><DIV class="col-lg-12 FooterMainColumn"><DIV class="col-lg-8"><DIV id="Footer"><P>iGEM Wageningen 2017. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link" style="padding-right:0;">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>