<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Safetytest skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Safetytest</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="Main-Menu"><DIV class="col-lg-10 MainMenu-Center-Column MenuCenter"><DIV class="Project MenuButtonColumn"><DIV class="Project-Top TopMenuButtonColumn"><UL><LI class="project-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Project" class="nav-item" id="project">Project</A></LI></UL></DIV></DIV><DIV class="Background MenuButtonColumn"><DIV class="Background-Top TopMenuButtonColumn"><UL><LI class="background-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Background" class="nav-item active" id="background">Background</A></LI></UL></DIV></DIV><DIV class="Wet-Lab MenuButtonColumn"><DIV class="Wet-Lab-Top TopMenuButtonColumn"><UL><LI class="wet-lab-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Wet-lab" class="nav-item active" id="wet-lab">Wet-lab</A></LI></UL></DIV><DIV class="Wet-Lab-Bottom SubMenuButtonColumn"><UL id="wet-lab-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results" class="results-sub-nav-item sub-nav-item">Results</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Parts" class="parts-sub-nav-item sub-nav-item">BioBricks</A></LI></UL></DIV></DIV><DIV class="Model MenuButtonColumn"><DIV class="Model-Top TopMenuButtonColumn"><UL><LI class="model-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Model" class="model nav-item" id="model">Model</A></LI></UL></DIV><DIV class="Model-Bottom SubMenuButtonColumn"><UL id="model-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics" class="kinetics-sub-nav-item sub-nav-item">Cpx Kinetics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/QS" class="qs-sub-nav-item sub-nav-item">Quorum Sensing</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx" class="integrationcpx-sub-nav-item sub-nav-item">Cpx Integration</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS" class="integrationQS-sub-nav-item sub-nav-item">QS Integration</A></LI></UL></DIV></DIV><DIV class="Human-Practices MenuButtonColumn"><DIV class="Human-Practices-Top TopMenuButtonColumn"><UL><LI class="human-practices-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Human_Practices" class="nav-item" id="human-practices">Human Practices</A></LI></UL></DIV><DIV class="Human-Practices-Bottom SubMenuButtonColumn"><UL id="human-practices-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated" class="integrated-sub-nav-item sub-nav-item ">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Silver" class="safety-sub-nav-item sub-nav-item">Safety &amp; Ethics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Collaborations" class="collaboration-sub-nav-item sub-nav-item">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Future" class="future-sub-nav-item sub-nav-item">Future Perspectives</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Engagement" class="engagement-sub-nav-item sub-nav-item">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Meetup" class="meetup-sub-nav-item sub-nav-item">Meetup</A></LI></UL></DIV></DIV><DIV class="Application MenuButtonColumn"><DIV class="Application-Top TopMenuButtonColumn"><UL><LI class="application-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Applied_Design" class="nav-item" id="application">Application</A></LI></UL></DIV><DIV class="Application-Bottom SubMenuButtonColumn"><UL id="application-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Demonstrate" class="demo-sub-nav-item sub-nav-item">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Hardware" class="device-sub-nav-item sub-nav-item">Device</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Application/Rollout" class="rollout-sub-nav-item sub-nav-item">Roll-out Plan</A></LI></UL></DIV></DIV><DIV class="Notebook MenuButtonColumn"><DIV class="Notebook-Top TopMenuButtonColumn"><UL><LI class="notebook-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook" class="notebook nav-item" id="notebook">Notebook</A></LI></UL></DIV><DIV class="Notebook-Bottom SubMenuButtonColumn"><UL id="notebook-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols" class="protocols-sub-nav-item sub-nav-item">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/InterLab" class="interlab-sub-nav-item sub-nav-item">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Journal" class="journal-sub-nav-item sub-nav-item">Lab Journal</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Achievements" class="achievements-sub-nav-item sub-nav-item">Achievements</A></LI></UL></DIV></DIV><DIV class="Team MenuButtonColumn"><DIV class="Team-Top TopMenuButtonColumn"><UL><LI class="team-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Team" class="team nav-item" id="team">Team</A></LI></UL></DIV><DIV class="Team-Bottom SubMenuButtonColumn"><UL id="team-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Team/Members" class="members-sub-nav-item sub-nav-item">Members</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Attributions" class="attributions-sub-nav-item sub-nav-item">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Acknowledgements" class="Acknowledgements-sub-nav-item sub-nav-item">Acknowledgements</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="MobileMenu"><UL id="MainMobileMenu"><LI><BUTTON class="project-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Project'" type="button">Project</BUTTON></LI><LI><BUTTON class="background-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Background'" type="button">Background</BUTTON></LI><LI><BUTTON class="wet-lab-li-mo">Wet-lab</BUTTON></LI><LI><BUTTON class="model-li-mo">Model</BUTTON></LI><LI><BUTTON class="human-practices-li-mo">Human Practices</BUTTON></LI><LI><BUTTON class="application-li-mo">Application</BUTTON></LI><LI><BUTTON class="notebook-li-mo">Notebook</BUTTON></LI><LI><BUTTON class="team-li-mo">Team</BUTTON></LI></UL><UL id="MobileSubMenu"><LI class="wet-lab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Wet-lab'" type="">Wet-lab</BUTTON></LI><LI class="model-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model'">Model</BUTTON></LI><LI class="application-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Applied_Design'">Application</BUTTON></LI><LI class="human-practices-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Human_Practices'">Human Practices</BUTTON></LI><LI class="notebook-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook'">Notebook</BUTTON></LI><LI class="team-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team'">Team</BUTTON></LI><LI class="parts-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Parts'" type="button">Parts</BUTTON></LI><LI class="results-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Results'" type="button">Results</BUTTON></LI><LI class="kinetics-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics'" type="button">Cpx Kinetics</BUTTON></LI><LI class="qs-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/QS'" type="button">Quorum Sensing</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx'" type="button">Cpx integration</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS'" type="button">QS integration</BUTTON></LI><LI class="integrated-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated'" type="button">Integrated HP</BUTTON></LI><LI class="safety-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Silver'" type="button">Safety &amp; Ethics</BUTTON></LI><LI class="collaboration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Collaborations'" type="button">Collaborations</BUTTON></LI><LI class="future-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Future'" type="button">Future Perspectives</BUTTON></LI><LI class="engagement-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Engagement'" type="button">Engagement</BUTTON></LI><LI class="meetup-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Meetup'" type="button">Meetup</BUTTON></LI><LI class="demo-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Demonstrate'" type="button">Demonstrate</BUTTON></LI><LI class="device-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Hardware'" type="button">Device</BUTTON></LI><LI class="rollout-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Application/Rollout'" type="button">Rollout Plan</BUTTON></LI><LI class="protocols-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols'" type="button">Protocols</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/InterLab'" type="button">InterLab</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Journal'" type="button">Journal</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Achievements'" type="button">Achievements</BUTTON></LI><LI class="members-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team/Members'" type="button">Members</BUTTON></LI><LI class="attributions-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Attributions'" type="button">Attributions</BUTTON></LI></UL></DIV><DIV class="container-fluid OurContent"><DIV class="row"><DIV class="col-lg-8"><DIV class="Main-Border"><DIV class="Main-Center-Content-Column"><DIV id="breadcrumb-wrapper"><UL class="breadcrumb"><LI><A href="https://2017.igem.org/Team:Wageningen_UR">Home</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Silver">Ethics &amp; Safety</A></LI><LI>Biosafety</LI></UL></DIV><DIV class="Results-Title"><H1>Biosafety</H1></DIV><DIV class="Textbox Results-Example"><P>
                            Biosafety is defined as the prevention of the unintended exposure or accidents involving genetically modified organisms or pathogens. It is the responsibility of scientists and companies to follow working policies that will guarantee that the biological sample will not cause harm to the workers, the public or the environment. Currently there are many projects focused on developing biosafety tools for synthetic biology, mainly motivated by the concern that society, institutions and organizations have expressed, but whose aim is to create a safe technology and not to please some sectors of the population [1]. The present efforts are a reflection of the Asilomar Conference on Recombinant DNA, which was held in 1975 at a conference center at Asilomar State Beach to discuss biosafety and ethics issues related to genetic engineering.
                        </P><P>
                            On the other hand, a related confusing term to biosafety is biosecurity, which deals with the intended theft, misuse or release of harmful biological agents [2].
                        </P><P>
                            Synthetic Biology is a relatively new technology that can be still modified to make it more open and safer [3]. To achieve this, key stakeholders such as governmental institutions and the general public will have to engage in a public conversation with scientists and companies in order to develop the field.
                        </P><P>
                            Moreover, the synthetic biology tools have themselves the potential to make synthetic biology safer through the application of safety mechanisms on the biological devices developed. As synthetic biology is based on engineering principles applied to biology, the safety engineering applied in the usual technologies will be able to be applied to synthetic biology. This gives a base to develop a system to identify and eliminate hazards that will reduce the risks associated to synthetic biology. An example of a tool from safety engineering that can be applied to synthetic biology are the Fault Analysis Trees, which will be explained further below.
                        </P></DIV><H2>Biocontainment</H2><P>
                                        In current industrial applications such as fermenters, the use of closed settings that are easily controlled and proper waste management allows for the safe use of genetically modified microorganisms. However, taking these organisms for use in external situations increases the chances of an accidental release. Therefore, the existence of efficient containment mechanisms that avoid the spread of the modified bacteria will be needed in order to exploit the full potential of synthetic biology [3].
                                    </P><P>
                                        For a biocontainment mechanism to be effective, it needs to prevent the proliferation of the organisms. These situations are the mutagenic drift, the environmental supplementation and the horizontal gene transfer [4]. An analysis of the chosen biocontainment mechanism can be found below, as well as two other alternatives:
                                    </P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#BS1" aria-expanded="true" aria-controls="BS1"><DIV class="col-xs-11">Auxotrophy</DIV></A></H4></DIV><DIV id="BS1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                        The aim is to engineer organisms so that they need an essential compound for their survival. There are two ways of doing so [3]:
                                    </P><UL><LI> Removing an essential metabolic pathway that will make the bacteria dependent on one or several biomolecules. For example, deleting the gene <I>thyA</I> makes the bacteria dependent on thymidine and removing the gene <I>asd</I> makes the bacteria need diaminopimelic acid, methionine, lysine and threonine to survive.
                                        </LI><LI> Introducing a lethal gene, such as <I>gef</I> or <I>nucA</I>. This gene will be under control of a compound in such a way that the depletion of said compound will activate the lethal gene and result in consequent cell death. For example, a xylS-gef switch was developed in such a way that the bacteria was dependent on benzoate [5].
                                        </LI></UL><P>
                                        Introducing two copies or combining the two methods will help the biocontainment method to be resistant to mutagenic drift and horizontal gene transfer. However, the main limitation of this method is the environmental supplementation, as this method will only be effective as long as the natural environment does not have sources of the necessary compound's for the survival of the bacteria.
                                    </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#BS11" aria-expanded="true" aria-controls="BS11"><DIV class="col-xs-11">Induced lethality</DIV></A></H4></DIV><DIV id="BS11" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                        Also known as &quot;killswitch&quot; mechanisms. It could be considered the opposite to auxotrophy, as in this case a compound will induce the death of the modified bacteria, while it will not affect natural organisms. Among the death-inducer we can find IPTG, heat, sucrose or arabinose [3]. There is research [6] suggesting that time could be also used as an inducer. This way, the expression of a lethal protein would happen after a determined number of cell cycles.
                                    </P><P>
                                        Although this biocontainment method avoids the problem of environmental supplementation, it does not escape from mutagenic drift and horizontal gene transfer. A genetic modification inactivating the toxic protein would not have any impact to the fitness of the cell and it would greatly be selected by evolution once the toxic protein is active in other cells. Due to leakiness of regulated promoters, an inactivating mutation in the toxic gene might grant a growth advantage even in the absence of the lethality inducer.
                                    </P></DIV></DIV></DIV></DIV><H3>Orthogonal life: Synthetic Auxotrophy</H3><P>
                                        The orthogonal life will be a result of the manipulation of the genetic code (the rules by which DNA is read), that will give place to an artificial language that the synthetic organisms will not be able to share with natural organisms. These modifications can be made both at the DNA level or at the protein level.
                     </P><UL><LI> At the DNA level, artificial bases can be used, as well as artificial backbones with threose or glucose instead of deoxyribose.
                                        </LI><LI> At the protein level, synthetic tRNAs can be engineered to recognize unused codons such as TAG (non-sense codon) or to recognize quadruplet codons instead of triplets.
                                        </LI></UL><P>
                                        Although these alternatives are quite recent, there are already successful examples of using orthogonal life as a biocontainment technique. Mandell <I>et al.</I> (2015) [4] developed a mechanism called &quot;Synthetic Auxotrophy&quot; which consists of re-coding the bacterial genetic code so that the codon TAG can be assigned to an artificial amino acid, biphenylalanine (bipA). For this, a bipA-tRNA synthetase had to be engineered and introduced in the bacteria, so that bipA could be used by the cellular machinery.
                                    </P><P>
                                        Furthermore, essential proteins were engineered so they would require bipA in their amino acid sequence to be functional. The proteins were chosen so that their deficiency could not be supplemented by compounds in the environment. 
                                    </P><P>
                                        The synthetic auxotrophy has several advantages over the engineered auxotrophy and the induced lethality:
                                    </P><UL><LI> Resistance to the mutagenic drift: Engineering proteins to require bipA for functionality means that several residues will have to mutate to render a functional protein that does not need bipA. Multiply the slim chances that a single protein mutates for the three engineered proteins, and the chances of an escapee mutant are really low.
                                        </LI><LI> Protection from environmental supplementation: bipA is an artificial compound that will not be found in any natural ecosystem.
                                        </LI><LI> Resistance to horizontal gene transfer: The escape frequency will decrease as different auxotrophies are included. Moreover, the difference in the genetic code will help avoiding the horizontal gene transfer. For example, if an essential gene transferred contains the stop codon UAG, this will be translated into bipA and the gene will render an inactive protein.
                                        </LI></UL><P>
                                        The advantages and efficacy of this method make it the perfect for our chosen containment mechanism.
                                    </P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 1:</B> This figure explains basics of the chosen biocontainment system, synthetic auxotrophy. Its easy explanation makes it adequate for the general public. This figure was used next to the comic to inform the public about the safety of our project.
                        </DIV></DIV><H2>Risk Assessment</H2><P>
                                        Risk assessment is the quantitative or qualitative estimation of the risk that a specific threat generates. Performing this analysis in synthetic biology can prove difficult, as there is no certain understanding on how synthetic organisms might affect the environment.
                                    </P><P>
                                        With the development of genetic engineering, a risk assessment protocol for GMOs was developed in 2008 [7] in which a case-by-case analysis of the trait inserted, the strain used, and a release in a specific environment should be performed. However, synthetic biology handles far more complex systems that just one insertion, which will interact among themselves in a way that might affect the cell and therefore, reduce the knowledge about the risk it possesses [1].
                                    </P><P>
                                        In an attempt to develop a more suitable protocol for the release of synthetic organisms in the environment, a group of synthetic biologists and ecologists met in the Woodrow Wilson International Center for Scholars in Washington DC to discuss on how to assess the risks of synthetic biology [8]. They identified four points of risk research to evaluate the ecological impact of synthetic organisms:
                                   
                    </P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#BS21" aria-expanded="true" aria-controls="BS21"><DIV class="col-xs-11">Interaction between synthetic and natural organisms</DIV></A></H4></DIV><DIV id="BS21" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                        In the case of a release in the environment, the differences in the physiology between natural and synthetic organisms will have an important impact on the environment. Furthermore, when a modified bacteria is released in the environment, its main competitor will be unmodified bacteria of the same species or similar ones. Therefore, it is important to compare the synthetic organisms with their unmodified parent strain: growth rate, toxic effects and growth requirements.
                                </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#BS22" aria-expanded="true" aria-controls="BS22"><DIV class="col-xs-11">Effect of the synthetic organism on the environment</DIV></A></H4></DIV><DIV id="BS22" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                        In case the modified bacteria finds a niche in the environment, it may interact with other organisms as well as with the environment itself (e.g. changing the composition of the soil). The risk of a synthetic bacteria affecting the food webs and the biodiversity of the ecosystems is difficult to assess, as it will depend on the environment considered, and a lot of unknown factors will come into play (e.g. the bacterial composition of any ecosystem is still unknown). In our opinion, the best approach would be to decide the most likely environments where a spread of the synthetic organism might happen, based on two conditions: native environment of the unmodified parent strain and the environments where the synthetic cell will be applied. Once a limited number of environments have been selected, samples will be inoculated with the genetically modified bacteria in a controlled environment. Using metagenomic analysis, the samples will be studied to look for synergistic or toxic effects that the synthetic bacteria might have caused in the natural population.
                                </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#BS23" aria-expanded="true" aria-controls="BS23"><DIV class="col-xs-11">Evolution and adaptation of the synthetic organism</DIV></A></H4></DIV><DIV id="BS23" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                        Environmental supplementation happens when the environment contains a component that will render the biocontainment method useless and therefore will allow the bacteria to proliferate. Both horizontal gene transfer and mutagenic drift work at the DNA level, directly inactivating the biocontainment mechanisms, or unlocking an alternative way for the bacteria to survive. Although the environmental supplementation and horizontal gene transfer might be difficult to predict (because they depend in great extent on the specific ecosystem where the organism has been released), the mutagenic drift can be easily assessed by detecting escapees after the biocontained population has been inoculated in non-permissive media. The recommended limit for the number of escapees is 1 out of 10<SUP>8</SUP> [9].
                                    
                                </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#BS24" aria-expanded="true" aria-controls="BS24"><DIV class="col-xs-11">Gene transfer from synthetic to natural organisms</DIV></A></H4></DIV><DIV id="BS24" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                        Inactivating biocontainment mechanisms is not the only problem associated to horizontal gene transfer. Genetic material can also be transferred from synthetic organisms to natural ones, with possible threatening consequences (e.g. antibiotic resistance). Therefore, the ability of the synthetic organisms for sharing genetic material should be assessed. This could be assessed by mixing the synthetic strain with the natural one. After a time of co-culture, the genetic transfer could be identified through FISH (Fluorescence <I>In Situ</I> Hybridization), using three different markers: one for the synthetic sequence, a second one for a sequence specific in the synthetic strain and a third one for a specific sequence in the natural strain. The number of genetic transfers will then be analysed through microscopy or flow-cytometry. The recommended limit of genetic transfer is the same as the limit for mutant escapees.
                                    </P></DIV></DIV></DIV></DIV><H2>Safety Assessment</H2><P>
                                        Once the biological device is complete, the biocontainment is implemented and the possible risks have been assessed, the robustness of the device must be analysed. Although in an iGEM project it is difficult to implement, a biological device should be robust in the sense that even after failure of single parts or subsystems, the device will keep working in a safe way. The safety assessment allows to find weak spots in the biological design, as well as potential problems that may not have been taken into account previously. This information will help to redesign the biological device to make it more robust and safer, for example by implementing redundant circuits.
                                    </P><P>
                                        There are different ways of performing the safety assessment. One of the most common methods is the Tree Analysis. There is an inductive approach (Event Tree Analysis), in which possible failures are identified after considering malfunctioning of the different parts of the device, and a deductive approach (Fault Tree Analysis), in which failures are first defined and the possible causes are traced backward. We have developed an example of a Fault Tree Analysis applied to the biocontainment mechanism, which can be found below (Figure 2).
                                    </P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 2:</B> This figure shows the Fault Tree Analysis that we have developed to analyse the possible problems that might make the biocontainment method not work.
                        </DIV></DIV><DIV class="Textbox Citations"><H4>
								References
								</H4><OL><LI>Schmidt, M. (2010). Do I Understand What I Can Create? In M. Schmidt, A. Kelle, A. Ganguli-Mitra, &amp; H. Vriend (Eds.), Synthetic Biology: The technoscience and its societal consequences (pp. 81-100). Dordrecht: Springer Netherlands.</LI><LI>World Health Organization &quot;Laboratory Biosafety Manual. Third Edition.&quot;, Geneva (2004)</LI><LI>Moe-Behrens, Gerd H. G., Rene Davis, and Karmella A. Haynes. Preparing Synthetic Biology for the World. Frontiers in Microbiology 4 (2013): 5. PMC. Web. 18 Sept. 2017.</LI><LI>Mandell, D. J., Lajoie, M. J., Mee, M. T., Takeuchi, R., Kuznetsov, G., Norville, J. E.,... Church, G. M. (2015). Biocontainment of genetically modified organisms by synthetic protein design. Nature, 518(7537), 55-60.</LI><LI>Contreras, A., Molin, S., and Ramos, J. L. (1991).Conditional-suicide containment system for bacteria which mineralize aromatics. Appl. Environ. Microbiol. 57, 1504-1508.</LI><LI>Lu, T.K., Khalil, A.S., and Collins, J. J. (2009). Next-generation synthetic gene networks. Nat. Biotechnol. 27, 1139-1150.</LI><LI>CBD (2008) Risk Assessment and Risk Management (Articles 15 and 16) Conference of the Parties to the Convention on Biological Diversity Serving as the Meeting of the Parties to the Cartagena Protocol on biosafety, Hourth Meeting Bonn, 12-16 May.</LI><LI>Dana, Genya V., et al. (2012) Four steps to avoid a synthetic-biology disaster. Nature. 483,29. </LI><LI>Wilson, D.J. (1993) NIH guidelines for research involving recombinant DNA molecules. Account. Res. 3,177-185. </LI></OL></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid FooterWrapper"><DIV class="FooterImage"><DIV class="col-lg-12 FooterMainColumn"><DIV class="col-lg-8"><DIV id="Footer"><P>iGEM Wageningen 2017. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link" style="padding-right:0;">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>