<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Model_QS skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Model/QS</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="Main-Menu"><DIV class="col-lg-10 MainMenu-Center-Column MenuCenter"><DIV class="Project MenuButtonColumn"><DIV class="Project-Top TopMenuButtonColumn"><UL><LI class="project-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Project" class="nav-item" id="project">Project</A></LI></UL></DIV></DIV><DIV class="Background MenuButtonColumn"><DIV class="Background-Top TopMenuButtonColumn"><UL><LI class="background-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Background" class="nav-item active" id="background">Background</A></LI></UL></DIV></DIV><DIV class="Wet-Lab MenuButtonColumn"><DIV class="Wet-Lab-Top TopMenuButtonColumn"><UL><LI class="wet-lab-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Wet-lab" class="nav-item active" id="wet-lab">Wet-lab</A></LI></UL></DIV><DIV class="Wet-Lab-Bottom SubMenuButtonColumn"><UL id="wet-lab-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results" class="results-sub-nav-item sub-nav-item">Results</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Parts" class="parts-sub-nav-item sub-nav-item">BioBricks</A></LI></UL></DIV></DIV><DIV class="Model MenuButtonColumn"><DIV class="Model-Top TopMenuButtonColumn"><UL><LI class="model-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Model" class="model nav-item" id="model">Model</A></LI></UL></DIV><DIV class="Model-Bottom SubMenuButtonColumn"><UL id="model-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics" class="kinetics-sub-nav-item sub-nav-item">Cpx Kinetics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/QS" class="qs-sub-nav-item sub-nav-item">Quorum Sensing</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx" class="integrationcpx-sub-nav-item sub-nav-item">Cpx Integration</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS" class="integrationQS-sub-nav-item sub-nav-item">QS Integration</A></LI></UL></DIV></DIV><DIV class="Human-Practices MenuButtonColumn"><DIV class="Human-Practices-Top TopMenuButtonColumn"><UL><LI class="human-practices-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Human_Practices" class="nav-item" id="human-practices">Human Practices</A></LI></UL></DIV><DIV class="Human-Practices-Bottom SubMenuButtonColumn"><UL id="human-practices-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated" class="integrated-sub-nav-item sub-nav-item ">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Silver" class="safety-sub-nav-item sub-nav-item">Safety &amp; Ethics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Collaborations" class="collaboration-sub-nav-item sub-nav-item">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Future" class="future-sub-nav-item sub-nav-item">Future Perspectives</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Engagement" class="engagement-sub-nav-item sub-nav-item">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Meetup" class="meetup-sub-nav-item sub-nav-item">Meetup</A></LI></UL></DIV></DIV><DIV class="Application MenuButtonColumn"><DIV class="Application-Top TopMenuButtonColumn"><UL><LI class="application-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Applied_Design" class="nav-item" id="application">Application</A></LI></UL></DIV><DIV class="Application-Bottom SubMenuButtonColumn"><UL id="application-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Demonstrate" class="demo-sub-nav-item sub-nav-item">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Hardware" class="device-sub-nav-item sub-nav-item">Device</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Application/Rollout" class="rollout-sub-nav-item sub-nav-item">Roll-out Plan</A></LI></UL></DIV></DIV><DIV class="Notebook MenuButtonColumn"><DIV class="Notebook-Top TopMenuButtonColumn"><UL><LI class="notebook-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook" class="notebook nav-item" id="notebook">Notebook</A></LI></UL></DIV><DIV class="Notebook-Bottom SubMenuButtonColumn"><UL id="notebook-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols" class="protocols-sub-nav-item sub-nav-item">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/InterLab" class="interlab-sub-nav-item sub-nav-item">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Journal" class="journal-sub-nav-item sub-nav-item">Lab Journal</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Achievements" class="achievements-sub-nav-item sub-nav-item">Achievements</A></LI></UL></DIV></DIV><DIV class="Team MenuButtonColumn"><DIV class="Team-Top TopMenuButtonColumn"><UL><LI class="team-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Team" class="team nav-item" id="team">Team</A></LI></UL></DIV><DIV class="Team-Bottom SubMenuButtonColumn"><UL id="team-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Team/Members" class="members-sub-nav-item sub-nav-item">Members</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Attributions" class="attributions-sub-nav-item sub-nav-item">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Acknowledgements" class="Acknowledgements-sub-nav-item sub-nav-item">Acknowledgements</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="MobileMenu"><UL id="MainMobileMenu"><LI><BUTTON class="project-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Project'" type="button">Project</BUTTON></LI><LI><BUTTON class="background-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Background'" type="button">Background</BUTTON></LI><LI><BUTTON class="wet-lab-li-mo">Wet-lab</BUTTON></LI><LI><BUTTON class="model-li-mo">Model</BUTTON></LI><LI><BUTTON class="human-practices-li-mo">Human Practices</BUTTON></LI><LI><BUTTON class="application-li-mo">Application</BUTTON></LI><LI><BUTTON class="notebook-li-mo">Notebook</BUTTON></LI><LI><BUTTON class="team-li-mo">Team</BUTTON></LI></UL><UL id="MobileSubMenu"><LI class="wet-lab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Wet-lab'" type="">Wet-lab</BUTTON></LI><LI class="model-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model'">Model</BUTTON></LI><LI class="application-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Applied_Design'">Application</BUTTON></LI><LI class="human-practices-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Human_Practices'">Human Practices</BUTTON></LI><LI class="notebook-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook'">Notebook</BUTTON></LI><LI class="team-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team'">Team</BUTTON></LI><LI class="parts-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Parts'" type="button">Parts</BUTTON></LI><LI class="results-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Results'" type="button">Results</BUTTON></LI><LI class="kinetics-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics'" type="button">Cpx Kinetics</BUTTON></LI><LI class="qs-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/QS'" type="button">Quorum Sensing</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx'" type="button">Cpx integration</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS'" type="button">QS integration</BUTTON></LI><LI class="integrated-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated'" type="button">Integrated HP</BUTTON></LI><LI class="safety-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Silver'" type="button">Safety &amp; Ethics</BUTTON></LI><LI class="collaboration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Collaborations'" type="button">Collaborations</BUTTON></LI><LI class="future-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Future'" type="button">Future Perspectives</BUTTON></LI><LI class="engagement-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Engagement'" type="button">Engagement</BUTTON></LI><LI class="meetup-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Meetup'" type="button">Meetup</BUTTON></LI><LI class="demo-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Demonstrate'" type="button">Demonstrate</BUTTON></LI><LI class="device-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Hardware'" type="button">Device</BUTTON></LI><LI class="rollout-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Application/Rollout'" type="button">Rollout Plan</BUTTON></LI><LI class="protocols-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols'" type="button">Protocols</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/InterLab'" type="button">InterLab</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Journal'" type="button">Journal</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Achievements'" type="button">Achievements</BUTTON></LI><LI class="members-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team/Members'" type="button">Members</BUTTON></LI><LI class="attributions-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Attributions'" type="button">Attributions</BUTTON></LI></UL></DIV><DIV class="container-fluid OurContent"><DIV class="row"><DIV class="col-lg-8"><DIV class="Main-Border"><DIV id="SideMenu-Wrapper"><SIDENAV id="SideMenu"><DIV class="menu-head"><H4>QS Model</H4></DIV><UL class="sidebar-nav"><LI class="menu-item"><A href="#qsintro">Introduction</A></LI><LI class="menu-item"><A href="#Temporal">Temporal model</A></LI><LI class="menu-item"><A href="#Global">Global parameters</A></LI><LI class="menu-item"><A href="#Local">Local parameters</A></LI><LI><A href="#Spatial">Spatiotemporal model</A></LI><LI><A href="#results">AHL diffusion</A></LI><LI><A href="#aiiA">aiiA addition</A></LI><LI><A href="#LuxR">LuxR stability</A></LI></UL></SIDENAV></DIV><DIV class="Main-Center-Content-Column Rollout"><DIV id="breadcrumb-wrapper"><UL class="breadcrumb"><LI><A href="https://2017.igem.org/Team:Wageningen_UR">Home</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model">Model</A></LI><LI>Quorum Sensing Model</LI></UL></DIV><DIV class="Title"><H1>Quorum sensing and cell lysis</H1></DIV><SECTION class="abstract"><P>
                            We developed a mathematical model of the signal amplification module of the Mantis system. Using quorum sensing (QS) and a lytic mechanism, the goal of this module is to take a weak antigen signal detected only by a minority of the biosensor cells, and to turn this into a population-wide response. Thus, a signal can be detected despite a low concentration of antigens in the tested sample. We explored the role of various reaction rates on the behavior of the system. Our mathematical model shows that the QS system can spontaneously activate, be insensitive to N-Acyl homoserine lactone (AHL), or function properly, depending on the reaction rates of the system. In our most complete model, we can change a spontaneously activating system into a system with proper signaling behavior by either increasing LuxR degradation, or by adding the enzyme aiiA to the genetic circuit.
                        </P><P>Alongside the development of this model, the amplification module described here was <A href="https://2017.igem.org/Team:Wageningen_UR/Results/Quorum_Sensing"> built in the lab</A>. We described how the modeling and wet-lab projects were integrated on a <A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS">separate page</A>. The QS system depends on a <A href="https://2017.igem.org/Team:Wageningen_UR/Results/Cpx">detection module</A> that is able to detect disease antigens.
                        </P></SECTION><SECTION id="qsintro"><DIV class="Title"><H2>Introduction</H2></DIV><P>
                            The Mantis biosensor can use an optional amplification module. A detection module can be directly linked to a reporter module. When using the QS system, detection of an antigen will trigger the production of the QS signaling molecule AHL. Only when AHL levels are sufficiently high, cells will activate their reporter module. A general overview of the QS and lysis module is given in Figure 1.
                        </P><DIV class="figure-fullwidth"><DIV class="figure-center-caption"><B>Figure 1: </B> A general overview of the QS module of Mantis. Some cells will detect antigen and start producing AHL. Other cells will detect AHL and the entire population will start QS, triggering cell lysis. The release of cellular components upon lysis allows the two halves of the split fluorescent protein to generate a fluorescent signal. In green dots an entire fluorescent molecule (GFP) is depicted.
                            </DIV></DIV><P>
                            Adding an amplification module to Mantis can make it more sensitive to very low levels of antigen, making it able to diagnose diseases that are difficult to detect. As the QS system depends on transcription and translation, the QS system will make the diagnostic test slower. But different circumstances require diagnostics that are either very rapid, but not very sensitive, or are very sensitive, but take more time until a result is produced. Therefore, the amplification module can be very useful to whole-cell biosensors.
                        </P><P>
                            The signaling molecule AHL is central to QS. Together with the two proteins LuxR and LuxI, the system allows bacteria to produce a signaling molecule and activate genes when the bacteria reach a high enough cell density. AHL can freely diffuse across the cell membrane. Inside the cell, AHL can bind with LuxR to form the the LuxR-AHL complex. This complex can dimerize to become an active transcription factor (TF). The LuxR-AHL dimer can result in positive and negative regulation of genes. The pLuxR promoter is positively regulated while pLuxL is negatively regulated by the LuxR-AHL dimer. In our Mantis module (Figure 2), QS is coupled to a lytic mechanism. Initially, there will be two bacterial populations. Each population contains one part of split fluorescent protein (FP). Together, proteins from each population can generate a fluorescent signal. The cellular membranes will prevent the proteins from interacting, so initially there will be no fluorescence.
                        </P><DIV class="figure-fullwidth"><DIV class="figure-center-caption"><B>Figure 2: </B> The molecular components of the Mantis QS and lytic system. Antigen will activate QS (top left). QS will initiate cell lysis (top right). Cell lysis will result in a fluorescent signal (bottom).
                            </DIV></DIV><P>
                            Once some cells detect antigen, they will start producing AHL. This will spread to other cells within the vicinity of those cells that detect antigen, eventually causing all cells in Mantis to start QS. The active form of LuxR will activate the lytic mechanism, resulting in lysis of all cells engaging in QS. Once cell contents are lysed, the two complementary parts of the split FP will start to interact. This will result in a fluorescent signal, which will give a read-out on the Mantis device.
                        </P><P>
                            In the QS system, LuxI positively regulates its own expression. This can lead to positive feedback, where more and more AHL is produced. Cells with this positive feedback motif can therefore become activated. This is important, as the lytic mechanism should either be active or completely inactive.
                        </P><P>
                            To achieve this goal, a temporal model was first developed. The temporal model was used to sample the global parameter space. The second model that we used is the spatiotemporal model. This model was used to test hypotheses and to guide wet-lab experiments.
                        </P><P>
                            For the system to function properly, it needs the following properties.
                        </P><UL><LI><B>Bistability:</B> Upon activation by an external signal, cells must be able to switch from an OFF-state to an ON-state. This requires that the time-dependent system of ordinary differential equations (ODEs) has more than one equilibrium state solution. The bistability of switches based on the lux operon have been explored in the literature. A motif with two feedback loops has been shown to have bistability [1,2]. However, it remains unclear if the native lux box (BBa_K546000) has the ability to switch from a clear OFF-state to an ON-state [3].
                            </LI><LI><B>Cell signaling:</B> Cells in the ON-state should be able to signal cells in the OFF-state to also transition to the ON-state. This is a core property of the QS system [4].
                            </LI><LI><B>Delayed cell lysis:</B> Cells in the ON-state should lyse to release their split FP. But they should do so only after signaling cells still in the OFF-state.
                            </LI></UL><H4>Activator and receiver cells</H4><P>
                            In the Mantis system, antigen concentrations will be low and therefore only a small number of cells come into contact with antigens. For the temporal model, we assumed that 10% of all cells would come into direct contact with the antigen, allowing them to detect the antigen. These cells are the activator cells. In the model, detection of antigen is simulated by switching on LuxI and LuxR production in 10% of the cells. The receiver cells are identical to the activator cells in all respects, except for this activation of LuxR and LuxI production.
                        </P><H4>Biological components</H4><P>
                            In the activator cells, both LuxI and LuxR will be produced under control of a promoter pInd induced by antigen. LuxI can also be produced from a promoter pLuxA, which is positively regulated by LuxR-AHL. LuxR is either positively or negatively regulated by LuxR-AHL and produced from pLuxB. The lysis protein is produced from pLuxC and is always positively regulated by LuxR-AHL.
                        </P></SECTION><SECTION id="Temporal"><DIV class="Title"><H2>Temporal model</H2></DIV><DIV class="Textbox"><P>
                                The main goal of the temporal model is to assess the dynamics of the system and what kind of responses it has to the antigen input. To effectively do this, the biological picture has to be translated to mathematical equations. Furthermore, this model is more manageable and more straightforward to interpret when the number of kinetic parameters are low. To achieve this, the following simplifying assumptions have been made.
                            </P></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SimpleAssumptions" aria-expanded="true" aria-controls="SimpleAssumptions"><H4 class="panel-title"><DIV class="col-xs-11">Assumptions</DIV></H4></A></DIV><DIV id="SimpleAssumptions" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><H5>System</H5><P>
                                            The system consists of three compartments; the external medium, antigen-activated bacterial cells, and signal-receiving bacterial cells (Figure A). The equations of both cells are identical, except that the activator cells receive an external signal intended to put the cells in the ON-state at t = 240. This simulates the addition of antigen-containing blood to the bacteria and detection of this low concentration of antigen by a minority of the cells. The total size of the system is 1 000 000 volume units. The ratio of external volume to cells is 1 cell in 1000 volume units. This approximates a cell density of 1 OD600. Therefore, the total cell volume is 1000 volume units. As the volume unit is defined as the volume of a single cell, there are a total of 1000 cells in the simulated system. Out of these cells, 900 cells are receiver cells, 100 are activator cells. Both of them are represented by single cell compartments. When the compartment representing each cell exchanges species with the external medium compartment, the total amount of exchange is calculated by using the total for each cell type. This exchange of AHL allows the cells to signal across the medium, communicating the presence of antigen. This leads to cell lysis and activation of fluorescence.
                                        </P><DIV class="figure-fullwidth"><DIV class="figure-center-caption"><B>Figure A: </B>The temporal model has three different compartments, the external medium, activator cells, and receiver cells. Diffusion is assumed to be instantaneous inside each compartment allowing us to neglect spatiotemporal dynamics. Diffusion across the cell membrane is computed using a linear approximation, as indicated by the 'flux' arrows. ODEs describe the rate of change of all species inside the three compartments.
                                            </DIV></DIV><P>
                                            Each cell has concentration values for the following species: AHL, LuxI, LuxR, LuxR-AHL, split FP and lysis protein. In the external medium, there are concentration values for AHL, C-terminal split FP, N-terminal split FP, and the fused and active FP. All compartments are assumed to be well-mixed, having a homogeneous concentration. All cells are identical in size and their size is constant throughout the simulation. There is assumed to be no cell growth and therefore no dilution term.
                                        </P><H5>Kinetics</H5><P>
                                            The levels of mRNA of all genes expressed is assumed to be in a quasi-steady state. The basis for this assumption is that translation is slower than transcription. Therefore, the concentration of mRNA species in the model is assumed to be constant. Similarly, transcription factors binding to DNA are also assumed to occur quickly since binding and unbinding of a transcription factor to a promoter occurs at timescales much faster than translation. The third assumption is that the dimerization of LuxR-AHL takes place quicker than protein dynamics. Therefore, the concentration of the LuxR-AHL dimer is assumed constant in the model. To summarize, transcription factor-DNA complexes, mRNA, and LuxR-AHL dimer concentrations are implicitly modeled. All promoters are assumed to have a single transcription binding site and the transcription factor is assumed to form a dimer only, thus we use a Hill coefficient of 2.
                                        </P><P>
                                            The lysis mechanism is not explicitly modeled. Lysis is assumed to occur instantaneously the moment the lysis protein concentration exceeds a value of 1. Before the lysis protein reaches this concentration, the lysis mechanism is inert. Diffusion of AHL across the cell membrane is modeled by a linear diffusion equation depending on the concentration difference inside and outside of the cell, and a diffusion constant for diffusion of AHL across the cell membrane. This diffusion constant is assumed to be the same whether AHL diffuses into or out of the cell. As stated earlier, the cells as well as the external medium are considered to be well-mixed, which equates to assuming that both the diffusion of AHL inside each compartment is instantaneous. Similarly, the release of split FPs upon cell lysis is assumed to be instantaneous.
                                        </P><P>
                                            Protein production in the model is dependent on the expression rate of the gene. Production of AHL is dependent on LuxI. Except for the FPs in the external medium, all species also have degradation rates. The split and fused FPs are considered to be sufficiently stable so that degradation outside the cell plays no role on the timescales of our simulations. Upon lysis of the cell, only AHL and the split FPs are released into the external medium. All other model components are assumed to be destroyed.
                                        </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SimpleEquations" aria-expanded="true" aria-controls="SimpleEquations"><H4 class="panel-title"><DIV class="col-xs-11">Equations</DIV></H4></A></DIV><DIV id="SimpleEquations" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary equation"><P>The change in time of every species in the system is described by the following ordinary differential equations.
                                        </P><H5>Ordinary differential equations</H5><P class="equation">
                                            ${\mathrm{d}AHL_{cell} \over \mathrm{d}t} = \alpha_{1}LuxI + D_{AHL}(AHL_{ext}-AHL_{cell}) + k_{-1}[LuxR-AHL] - k_1 LuxR \cdot AHL -\beta_{1}AHL_{cell}$
                                        </P><P>
                                            ${\mathrm{d}AHL_{ext} \over \mathrm{d}t} = V_{ratio} \cdot D_{AHL} (AHL_{ext}-AHL_{cell}) - \beta_{2}AHL_{cell}$
                                        </P><P>
                                            ${\mathrm{d}[LuxR-AHL] \over \mathrm{d}t} = k_{1}LuxR \cdot AHL - \beta_{3}[LuxR-AHL]^{2} - \beta_{4}[LuxR-AHL]$
                                        </P><P>
                                            ${\mathrm{d}LuxI \over \mathrm{d}t} = \alpha_2 + \alpha_3 {{LuxR \cdot [LuxR-AHL]^2} \over {km_1 + [LuxR-AHL]^2}} + \alpha_4 + S \cdot \alpha_5 - \beta_5 LuxI$
                                        </P><P> Positive regulation of LuxR:
                                             ${\mathrm{d}LuxR \over \mathrm{d}t} = k_{-1}[LuxR-AHL] + \alpha_6 + \alpha_7 {{LuxR \cdot [LuxR-AHL]^2} \over {km_1 + [LuxR-AHL]^2}} + \alpha_8 + S \cdot \alpha_9 - \beta_6 LuxR$
                                        </P><P> Negative regulation of LuxR:
                                             ${\mathrm{d}LuxR \over \mathrm{d}t} = k_{-1}[LuxR-AHL] + \alpha_6 + \alpha_7 {{LuxR \cdot km_1} \over {km_1 + [LuxR-AHL]^2}} + \alpha_8 + S \cdot \alpha_9 - \beta_6 LuxR$
                                        </P><P>
                                            ${\mathrm{d}Lysis \over \mathrm{d}t} = \alpha_{10} + \alpha_{11} {{LuxR \cdot [LuxR-AHL]^2} \over {km_2 + [LuxR-AHL]^2}} - \beta_7 Lysis$
                                        </P><P>
                                            ${\mathrm{d}SplitFP_{cell} \over \mathrm{d}t} = \alpha_{12} - \beta_8 SplitFP$
                                        </P><P>
                                            ${\mathrm{d}SplitFP_{N} \over \mathrm{d}t} = -k_3 SplitFP_{N} \cdot dSplitFP_{C}$
                                        </P><P>
                                            ${\mathrm{d}SplitFP_{C} \over \mathrm{d}t} = -k_3 SplitFP_{N} \cdot dSplitFP_{C}$
                                        </P><P>
                                            ${\mathrm{d}ActiveFP \over \mathrm{d}t} = k_3 SplitFP_{N} \cdot dSplitFP_{C}$
                                        </P><TABLE class="table table-bordered Results-Table"><THEAD><TR><TH colspan="2">Species</TH></TR></THEAD><TBODY><TR><TH>$AHL_{cell}$</TH><TH>Signaling molecule AHL inside the cell</TH></TR><TR><TH>$AHL_{ext}$</TH><TH>Signaling molecule AHL inside the external medium</TH></TR><TR><TH>$LuxR$</TH><TH>Free inactive LuxR protein</TH></TR><TR><TH>$[LuxR-AHL]$</TH><TH>The LuxR-AHL complex</TH></TR><TR><TH>$LuxI$</TH><TH>The AHL-producing enzyme LuxI</TH></TR><TR><TH>$LuxR$</TH><TH>The AHL-producing enzyme LuxI</TH></TR><TR><TH>$Lysis$</TH><TH>The lysis protein responsible for the lytic mechanism</TH></TR><TR><TH>$SplitFP_{cell}$</TH><TH>The split fluorescent protein in the cell</TH></TR><TR><TH>$SplitFP_{N}$</TH><TH>The N-terminal part of the split fluorescent protein in the external medium </TH></TR><TR><TH>$SplitFP_{C}$</TH><TH>The C-terminal part of the split fluorescent protein in the external medium </TH></TR><TR><TH>$ActiveFP$</TH><TH>The active full form of the split fluorescent protein in the external medium</TH></TR></TBODY></TABLE><TABLE class="table table-bordered Results-Table"><THEAD><TR><TH colspan="2">Parameters</TH></TR></THEAD><TBODY><TR><TH>$S$</TH><TH>is 0 or 1 depending on if the cell detected antigen</TH></TR><TR><TH>$\alpha_1$</TH><TH>AHL production rate of LuxI </TH></TR><TR><TH>$\alpha_2$</TH><TH>Leaky production rate of LuxI from pIndnst </TH></TR><TR><TH>$\alpha_3$</TH><TH>Maximum production rate of LuxI from pInd</TH></TR><TR><TH>$\alpha_4$</TH><TH>Leaky production rate of LuxI from pLuxA</TH></TR><TR><TH>$\alpha_5$</TH><TH>Maximum production rate of LuxI from pLuxA</TH></TR><TR><TH>$\alpha_6$</TH><TH>Leaky production rate of LuxR from pInd</TH></TR><TR><TH>$\alpha_7$</TH><TH>Maximum production rate of LuxR from pInd</TH></TR><TR><TH>$\alpha_8$</TH><TH>Leaky production rate of LuxR from pLuxB</TH></TR><TR><TH>$\alpha_9$</TH><TH>Maximum production rate of LuxR from pLuxB</TH></TR><TR><TH>$\alpha_{10}$</TH><TH>Leaky production rate of Lysis from pLuxC</TH></TR><TR><TH>$\alpha_{11}$</TH><TH>Maximum production rate of Lysis from pLuxC</TH></TR><TR><TH>$\alpha_{12}$</TH><TH>Constitutive production of $SplitFP_{cell}$</TH></TR><TR><TH>$k_1$</TH><TH>formation rate of LuxR-AHL</TH></TR><TR><TH>$k_{-1}$</TH><TH>dissociation rate of LuxR-AHL</TH></TR><TR><TH>$k_{2}$</TH><TH>formation &amp; maturation rate of the full fluorescent protein</TH></TR><TR><TH>$km_{1}$</TH><TH>Michaelis-Menten constant of pLuxA and pLuxB for $LuxR-AHL$</TH></TR><TR><TH>$km_{2}$</TH><TH>Michaelis-Menten constant of pLuxC for $LuxR-AHL$</TH></TR><TR><TH>$D_{AHL}$</TH><TH>diffusion rate of AHL across the cellular membrane</TH></TR><TR><TH>$\beta_1$</TH><TH>degradation rate of $AHL_{cell}$</TH></TR><TR><TH>$\beta_2$</TH><TH>degradation rate of $AHL_{ext}$</TH></TR><TR><TH>$\beta_3$</TH><TH>degradation rate of $LuxR-AHL$ complex </TH></TR><TR><TH>$\beta_4$</TH><TH>degradation rate of the $LuxR-AHL$ dimer</TH></TR><TR><TH>$\beta_5$</TH><TH>degradation rate of $LuxI$</TH></TR><TR><TH>$\beta_6$</TH><TH>degradation rate of $LuxR$</TH></TR><TR><TH>$\beta_7$</TH><TH>degradation rate of $Lysis$</TH></TR><TR><TH>$\beta_8$</TH><TH>degradation rate of $SplitFP_{cell}$</TH></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SimpleMethods" aria-expanded="true" aria-controls="SimpleMethods"><H4 class="panel-title"><DIV class="col-xs-11">Methods</DIV></H4></A></DIV><DIV id="SimpleMethods" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                            The initial concentration for all species was zero. The system of ODEs was solved using the scipy.integrate.odeint module available in Python. To sample the global parameter space, 1 000 000 parameter sets were generated using the lhs() function from pyDOE. The parameters were uniformly distributed between 0 and 10. These parameter sets were simulated for a total of 480 time units, where between t=240 and t=300, the external signal S was set from 0 to 1. A value for 1 was chosen to represent enough lysis protein for cell lysis to occur. Every 0.01 timesteps, the value of lysis protein in each cell was checked. In the case that a cell has a lysis protein value above 1, the cell would be removed from the simulation and the AHL and split FP contents would be released into the external medium. Half of the cells contained the C-terminal FP, whilst the other half of the cells contained the N-terminal version.
                                        </P><H5>Scoring function</H5><P>
                                            To assess how well a parameter set of reaction rates performed, we needed to generate scoring functions that described our ideal system behavior. We want our system to produce a lot of fluorescence quickly, but only after detection of antigen. This can be quantified by calculating the area under the curve, which we approximate using the numpy.trapz() function. As an ideal parameter set generates no fluorescence before t=240 and very quickly generates a lot of fluorescence after t=240, the area under the curve before t=240 should be minimal, while after t=240, the area should be as large as possible. The area under the curve represents both how much, as well as how quickly, fluorescence is generated. Secondly, in absence of an external input, (when S = 0 during the entire simulation), no fluorescence should be generated. Fluorescence in the absence of an antigen would amount to a false positive. Therefore, each parameter set is simulated twice. Once with no external signal, and once with an external signal between t=240 and t=300. The final score of a parameter set is calculated by combining both simulation runs.
                                            We define this as</P><DIV><P>
                                                $Score_{set} = {w_1 \over { |Score_{noAntigen} - Score_{Antigen}|+ w_2} } + Score_{Antigen}$
                                            </P><P>$Score_{noAntigen}$ is the $Score_{simulation}$ computed in the absence of antigen</P><P>$Score_{Antigen}$ is the $Score_{simulation}$ computed in the presence of antigen</P><P> $w_1$ sets an upper bound to the score in case the simulations produce identical levels of fluorescence and was set to $w_1 = 1000$</P><P>$w_2$ determines the weight of the difference in fluorescent production between the negative and positive simulation and was set to $w_2 = 1$</P><P>$Score_{noAntigen}$ is the $Score_{set}$ for the simulation in absence of antigen</P><P>$Score_{Antigen}$ is the $Score_{set}$ for the simulation in the presence of antigen</P><P>$Score_{simulation} = \displaystyle\int_{t=0}^{t=240} ActiveFP \mathrm{d}t + {1 \over {w_3 + \displaystyle\int_{t=240}^{t=300} ActiveFP \mathrm{d}t}}$
                                            </P><P>$ActiveFP$ is the fluorescence at time t</P><P>$w_3$ determines the weighting of fluorescent production before and after t=240, and was set to $w_3 = 0.01$</P></DIV></DIV></DIV></DIV></DIV><H3>Results</H3><P>After initial investigation of the model, the following types of system behaviors can be categorized:
                        </P><UL><LI><B>No lysis:</B> None of the cells in the system lyse. Even the activator cells, that are directly induced by the external signal, are unable to produce enough lysis protein within the duration of the simulation.</LI><LI><B>Activator lysis only:</B> Activator cells are able to lyse, release their FPs and generate some fluorescence. But the majority of the cell population remain unaffected. There is no cell-to-cell signaling through QS.</LI><LI><B>Spontaneous lysis:</B> All cells lyse within the duration of the simulation, but the lysis is not caused by induction of the activator cells by the external signal. Spontaneous auto-activation and cell lysis are caused by the build-up of AHL over time. This build-up is caused by leaky expression of LuxR and LuxI and, in the case of a system overly sensitive to AHL, this leaky expression will lead to auto-activation. Receiver cells will thus auto-activate using AHL they themselves produce, in the absence of antigen.</LI><LI><B>Induced full lysis:</B> Activator cells produce AHL and this triggers the receiver cells to auto-activate as well. At the end of the simulation, all cells will have lysed and released their split FPs, resulting in a significant fluorescent signal. The fluorescent signal is caused by the presence of antigen and not auto-activation. The increase of fluorescence over time of a simulation with the desired behavior is displayed in Figure 3.</LI></UL><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 3:</B> The increase of fluorescence for a simulation with induced full lysis. At t=240, 10% of the cells detect antigen and they lyse at t=242, releasing split FP. At t=247, the receiver cells lyse, releasing much more split FP. <A href="https://static.igem.org/mediawiki/2017/f/f7/T--Wageningen_UR--Model-TimeSeries_Parameter.pdf">Click for parameters</A>.
                            </DIV></DIV></SECTION><SECTION id="Global"><H5>Global parameter sampling</H5><P>A total of 1 million parameter sets were sampled for the system with positively regulated LuxR. The parameter sets were scored with the scoring functions and sorted by score. In the case of negative regulation of LuxR by the LuxR-AHL dimer, the percentage of parameter sets for each behavior is displayed in Table 1.
                        </P><TABLE class="table table-bordered Results-Table"><THEAD><TR><TH colspan="4">Table 1: Results of simulating 1 million parameter sets using Latin hypercube sampling. LuxR was positively regulated by the dimer of the LuxR-AHL complex</TH></TR></THEAD><TBODY><TR><TH>Behavior</TH><TH># parameter sets</TH><TH>Percentage</TH></TR><TR><TD>No lysis</TD><TD>789430</TD><TD>78.9%</TD></TR><TR><TD>Activator lysis</TD><TD>199560</TD><TD>19.9%</TD></TR><TR><TD>Spontaneous lysis</TD><TD>10986</TD><TD>1.1%</TD></TR><TR><TD>Induced full lysis</TD><TD>21</TD><TD>0.002%</TD></TR></TBODY></TABLE><P>
                            The parameter space for the system with negative LuxR regulation was explored in an identical manner, and the results are displayed in Table 2.
                        </P><TABLE class="table table-bordered Results-Table"><THEAD><TR><TH colspan="4">Table 2: Results of simulating 1 million parameter sets using Latin hypercube sampling. LuxR was negatively regulated by the dimer of the LuxR-AHL complex</TH></TR></THEAD><TBODY><TR><TH>Behavior</TH><TH># parameter sets</TH><TH>Percentage</TH></TR><TR><TD>No lysis</TD><TD>813045</TD><TD>81.3%</TD></TR><TR><TD>Activator lysis</TD><TD>146105</TD><TD>14.6%</TD></TR><TR><TD>Spontaneous lysis</TD><TD>40842</TD><TD>4.1%</TD></TR><TR><TD>Induced full lysis</TD><TD>8</TD><TD>0.001%</TD></TR></TBODY></TABLE><P>
                            The variation in parameters for each behavior was visualized by generating a histogram showing how many parameter sets fell within a certain parameter range for each behavior (positive regulation of LuxR shown in Figure 4, negative regulation regulation of LuxR in Figure 5). This indicates that parameter sets that have low degradation values for LuxR, and all species that contain LuxR, are more likely to display spontaneous lysis. This trend is stronger in the case of positive regulation of LuxR, where the active TF is needed for high levels of LuxR expression. For negative LuxR regulation, basal levels of LuxR are high and the system can spontaneously activate more easily with higher LuxR degradation factors. The parameter distributions for the degradation of LuxI differ significantly between the cases of positive and negative regulation of LuxR by the LuxR-AHL dimer. In the case of positive LuxR regulation, levels of LuxR limit the amount of active TF. When LuxR is negatively regulated, LuxR levels are higher and the levels of AHL are stronger correlated with the total amount of active TF.
                        </P><DIV class="figure-center"><DIV class="figure-center-imagebox"><DIV id="PosLuxR-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 4:</B>The parameter distribution of the LuxR degradation rate for spontaneous lysis (blue) and no lysis (green) is compared. When there is no relationship between a parameter and the behavior of the system, the distribution is uniform. LuxR was positively regulated.
                                    </DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 4:</B> The parameter distribution of the LuxR degradation rate for spontaneous lysis (blue) and no lysis (green) is compared. When there is no relationship between a parameter and the behavior of the system, the distribution is uniform. LuxR was positively regulated.
                            </DIV></DIV><P>A similar trend is seen in the data of the negative regulation of LuxR.</P><DIV class="figure-center"><DIV class="figure-center-imagebox"><DIV id="NegLuxR-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 5: </B>The parameter distribution of the LuxR degradation rate for spontaneous lysis (blue) and no lysis (green) is compared. When there is no relationship between a parameter and the behavior of the system, the distribution is uniform. LuxR was negatively regulated.</DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 5:</B> The parameter distribution of the LuxR degradation rate for spontaneous lysis (blue) and no lysis (green) is compared. When there is no relationship between a parameter and the behavior of the system, the distribution is uniform. LuxR was negatively regulated.
                            </DIV></DIV></SECTION><SECTION id="Local"><H3>Local parameter space</H3><P>To test the robustness of parameter sets that produced the system responses we desired, we explored the local parameter space around these optimal parameter sets. By scoring the local parameter sets, set with induced full lysis could be compared to those that lysed spontaneously. While the parameter distributions for many parameters when comparing spontaneous lysis and induced full lysis were still quite similar, a different behavior could be seen for the extracellular degradation of AHL and the degradation rate of LuxR. The results for the external degradation rate of AHL and the degradation rate of LuxR in the situation that LuxR is positive regulated are displayed in Figures 6 and 7.
                        </P><DIV class="col-lg-6 inline-img-left"><DIV class="figure-fullwidth"><DIV class="figure-center-imagebox"><DIV id="PosLuxRExtAHLdeg-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 6: </B>Parameter sets with spontaneous self-activation have higher external AHL degradation rates while parameter sets with the desired behavior have lower external AHL degradation rates.</DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 6:</B> Parameter sets with spontaneous self-activation (green) have higher external AHL degradation rates. Parameter sets with the desired behavior (blue) have lower external AHL degradation rates.
                                </DIV></DIV></DIV><DIV class="col-lg-6 inline-img-right"><DIV class="figure-fullwidth"><DIV class="figure-center-imagebox"><DIV id="PosLuxRLuxRdeg-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 7: </B>Parameter sets with induced full lysis (blue) and compared with spontaneous lysis (green). The parameter sets with high values of LuxR degradation are very likely to have the desired behavior. The parameter distribution of induced full lysis appears shifted to the right compared to the distribution of spontaneous lysis.
                                        </DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 7:</B> Parameter sets with induced full lysis (blue) and compared with spontaneous lysis (green). The parameter sets with high values of LuxR degradation are very likely to have the desired behavior. The parameter distribution of induced full lysis appears shifted to the right compared to the distribution of spontaneous lysis.
                                </DIV></DIV></DIV><P>The same analysis was carried out on the parameters obtained through Latin hypercube sampling for the situation with negative regulation of LuxR (Figure 8 and 9).</P><DIV class="row"><DIV class="col-lg-6 inline-img-left"><DIV class="figure-fullwidth"><DIV class="figure-center-imagebox"><DIV id="NegLuxRExtAHLdeg-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 8: </B>Induced full lysis (blue) compared with spontaneous lysis (green). Several high blue peaks are visible for low values of external AHL degradation rates. Of the parameter sets simulated that had high external AHL degradation rates, very few of them had the desired behavior.
                                            </DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 8: </B>Induced full lysis (blue) compared with spontaneous lysis (green). Several high blue peaks are visible for low values of external AHL degradation rates. Of the parameter sets simulated that had high external AHL degradation rates, very few of them had the desired behavior.
                                    </DIV></DIV></DIV><DIV class="col-lg-6 inline-img-right"><DIV class="figure-fullwidth"><DIV class="figure-center-imagebox"><DIV id="NegLuxRLuxRdeg-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 9: </B>Induced full lysis (blue) compared with spontaneous lysis (green). The parameter sets with induced full lysis have a clear shift towards higher rates of LuxR degradation when compared to the spontaneous lysis parameter sets.
                                            </DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 9:</B>Induced full lysis (blue) compared with spontaneous lysis (green). The parameter sets induced full lysis have a clear shift towards higher rates of LuxR degradation when compared to the spontaneous lysis parameter sets.
                                    </DIV></DIV></DIV></DIV><P>
                            Both the external AHL degradation rates and LuxR degradation rates seemed to play a large role in determining if a system would spontaneously lyse, or have induced full lysis, the effect of changing these parameters was explored further. We took one spontaneously activating parameter set and in this set varied both these two parameter values from 0 to 10. The scores for these 10 000 (100 by 100) parameter sets can be visualized in a heatmap (Figure 10). This heatmap shows the previously described types of behavior and how the desired behavior is the transition area between spontaneous lysis and lysis only of the activator cells and that by reducing the sensitivity of the system to AHL, for example by increasing the degradation rate of LuxR. This heatmap also shows that the external AHL degradation controls the thickness of the transition area and that for high values the transition area of desired system behavior disappears.
                        </P><DIV class="figure-fullwidth"><DIV class="figure-center-caption"><B>Figure 10:</B> Heatmap of the scoring function when varying both the external AHL degradation rate and the LuxR degradation rate. This shows the transition area of desired behavior in between spontaneous lysis and lysis of only the activator cells.
                            </DIV></DIV><H3>Summary</H3><P>
                            The global and local parameter sensitivity of our system was explored. When parameters allow for QS to occur, spontaneous lysis is the most likely outcome. But parameters were identified that distinguish between induced full lysis and spontaneous lysis. The external AHL degradation rate and the LuxR degradation rate have a large effect on system behavior. By decreasing LuxR stability or increasing external AHL stability, a simulation outcome can change from spontaneous lysis to induced full lysis. LuxR degradation can be changed experimentally by adding a degradation tag to a protein. The external AHL degradation cannot be changed. However, it's relative value can be decreased by increasing the cellular degradation rate. Therefore, these results provide sufficient justification for exploring these strategies in the lab.
                        </P></SECTION><SECTION id="Spatial"><DIV class="Textbox Results-Title"><H2>Spatiotemporal model</H2></DIV><P>As the assumption that AHL diffuses instantaneously breaks down when the system becomes larger, or when the number of activator cells decreases, we developed a spatiotemporal model that incorporates both diffusion of AHL, but also the mixing of the split FPs. To more realistically model the biological system, we used the spatiotemporal model with the following features:
                        </P><UL class="bullet-points"><LI>Diffusion of the signaling molecule, AHL</LI><LI>Diffusion of fluorescent proteins</LI><LI>Cell movement</LI><LI>Asymmetric cell division</LI><LI>Variable growth rates</LI></UL><P>
                            In the temporal model the cell density had remained constant throughout the simulation. Now, in the spatiotemporal model, cell density will increases exponentially.</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SpatialDetails" aria-expanded="true" aria-controls="SpatialDetails"><H4 class="panel-title"><DIV class="col-xs-11">Assumptions</DIV></H4></A></DIV><DIV id="SpatialDetails" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                            This spatiotemporal model is comprised out of a 2D grid where each square is the size of a single bacterial cell. Squares are either external medium, or completely occupied by a single cell. The grid system discretizes space while at the same time gives every cell a definite position.
                                            </P><P>
                                                The finite volume method approximates component concentrations over a spatial domain by dividing the space into smaller volume elements. At every coordinate in space, the concentration value there is set equal to the average concentration over that small volume element. By computing the flux through the faces of each volume piece, the concentration change over time can then be approximated. Diffusion rates from a cell to a medium square or from one medium square to another medium square have different diffusion coefficients, where the diffusion coefficient of AHL inside the external medium is much larger. When two cells are in adjacent squares, diffusion is still possible, but at a much slower rate.
                                            </P><P>
                                                Cell movement in this system is modeled by a random movement where cells have a certain probability to move to the four adjacent squares as well as the four diagonal squares. The model therefore does not realistically model chemotaxis or other forms of bacterial movement [5]. When a cell tries to move into a square that is already occupied, movement is prevented and the cell simply stays in their current square. When a cell moves into an unoccupied square, the AHL concentration of the original square and the destination square get switched around, as squares occupied by cells have no external medium AHL values. Cells don't change in size in this model. A bacteria is always the size of exactly one square. All cells do have their own growth rate. Once the cell has become big enough to divide, the simulation simply creates a new cell in an adjacent square and distributes all cellular species among both daughter cells. Every species is independently and asymmetrically distributed by drawing a number from a normal distribution (µ=0.5, σ=0.2). Once the cells are divided, the new cell is awarded a growth rate independent of that of the parent cell and both cells have their growth rate clock reset to zero. Cells that lyse still release their AHL and FP contents the same way as in the temporal model, except in the spatiotemporal method the square the lysing cell occupies becomes a external medium square with the same concentration of extracellular AHL as had been the cellular AHL concentration of that cell upon lysis.
                                            </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SpatialEquations" aria-expanded="true" aria-controls="SpatialEquations"><H4 class="panel-title"><DIV class="col-xs-11">Equations</DIV></H4></A></DIV><DIV id="SpatialEquations" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>Most equations for the spatiotemporal model are identical to those in the temporal model. The exceptions are described here. In the spatial model, production of both N- and C-terminal split FP is now explicitly modeled and the following two equations replace the one from the temporal model.
                                        </P><P class="equation">               ${\mathrm{d}SplitFPC_{cell} \over \mathrm{d}t} = \alpha_{12} - \beta_8 SplitFPC$
                                        </P><P class="equation">
                                            ${\mathrm{d}SplitFPN_{cell} \over \mathrm{d}t} = \alpha_{12} - \beta_8 SplitFPN$
                                        </P><P>The change in AHL and split FP in a square as a result of diffusion is calculated using the following equation:</P><P class="equation">
                                            $G_{i,j} = p_{i-1,j} + p_{i,j-1} + p_{i+1,j} + p_{i,j+1} - 4 \cdot p_{i,j}$

                                             $G$ is a scalar value indicating the size and the sign of the flux gradient in the square with coordinates $i$ and $j$
                                             $p_{i,j}$ is the concentration in square the with coordinates $i$ and $j$
                                        </P><P class="equation">
                                            $G_{i,j}$ is positive when the concentration in a square is increasing. In that case, more flux enters the square than leaves. The flux out of a square is proportional to the concentration inside that square. As there are four faces through which flux can leave, this term is present four times. Flux can also flow into a square from either of the four neighbors. These represent the four positive terms. Together, this provides a scalar quantity that describes the intensity of the flux.
                                        </P><P>
                                            The change in concentration is calculated by multiplying $U$ with the diffusion constant $D$.
                                        </P><P class="equation">
                                            $f = D \cdot G \Delta t$
  $f$ is the change concentration in a square
                                             $D$ is the diffusion constant.
                                             $G$ is the gradient
                                        </P><P>
                                            With the addition of $aiiA$, the equation for $AHL_{cell}$ gains a degradation term:
                                        </P><P class="equation">
                                            ${\mathrm{d}AHL_{cell} \over \mathrm{d}t} = \alpha_{1}LuxI + D_{AHL}(AHL_{ext}-AHL_{cell}) + k_{-1}LuxR-AHL - k_1 LuxR \cdot AHL - k_3 aiiA -\beta_{1}AHL_{cell}$
                                        </P><P>
                                            And the equation describing $aiiA$ itself:
                                        </P><P class="equation">
                                            ${\mathrm{d}aiiA \over \mathrm{d}t} = (1-S) \cdot \alpha_{13} - \beta_9 aiiA$
                                        </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SpatialMethods" aria-expanded="true" aria-controls="SpatialMethods"><H4 class="panel-title"><DIV class="col-xs-11">Methods</DIV></H4></A></DIV><DIV id="SpatialMethods" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                            Simulations start with an initial value of zero for all species. Again, odeint is used to solve the system of equations. The algorithm checks for cell lysis, movement and division every 0.01 time steps. The duration of a simulation is now 120 time units, with activators detecting antigen at t=60. The diffusion constant of AHL in the external medium is 100, unless otherwise stated. The diffusion constant of split FP inside the external medium was 10. </P></DIV></DIV></DIV></DIV></SECTION><SECTION id="results"><H3>Results</H3><H5>From temporal to spatiotemporal model</H5><P>
                            After developing the spatiotemporal mode, we first explored the differences between the spatiotemporal model and the original temporal model by taking parameter sets with the desired behavior from the temporal model and simulating them in the spatiotemporal model. The main difference between the spatiotemporal and the temporal model is neighbor-to-neighbor cell signaling. In the spatiotemporal model, AHL is generated by an activator cell, generating a local cloud of higher AHL concentrations in the medium (Figure 11).
                        </P><DIV class="figure-fullwidth"><DIV class="figure-center-caption"><B>Figure 11:</B>After three cells were activated by antigen, AHL (black) is being produced and diffusing out of the cell, generating a local cloud of AHL.
                            </DIV></DIV><P>
                            This changes the sensitivity of the model to the ratio between activator cells and receiver cells. Under the right parameters, an activator cell can activate its nearest neighbors regardless of the total system size. In some simulations (Video 1), a single cell activated by antigen can signal to and activate the entire population, as a wave of activation travels through the system. This suggests that for specific parameter values, the QS system has a great capacity for signal amplification, where a single cell can start a chain reaction, activating the entire cell population.
                        </P><VIDEO controls=""><SOURCE src="https://static.igem.org/mediawiki/2017/8/8d/T--Wageningen_UR--Model-300x300signal.mp4" type="video/mp4"><SOURCE src="https://static.igem.org/mediawiki/2017/8/88/T--Wageningen_UR--Model-300x300signal.webm" type="video/webm">
 Your browser does not support the video tag.
                        </SOURCE><DIV class="figure-center-caption"><B>Video 1: </B> Fluorescence produced by the original system (top) versus the system with aiiA included (bottom). On the right side is shown the system induced with antigen. On the left, the system in the abscence of antigen. This system was simulated with positive regulation of LuxR. <A href="https://static.igem.org/mediawiki/2017/a/a4/T--Wageningen_UR--Model-LargeSystem_Parameter.pdf">Click for parameters</A>.
                        </DIV></SOURCE><SECTION id="aiiA"><H5>Inclusion of aiiA</H5><DIV class="Textbox"><P>
                                We took a spontaneously lysing parameter set from the temporal model and transferred it to the spatiotemporal model. Production and degradation terms for aiiA were added to the system of ODEs, as well as degradation of AHL by aiiA. Simulations were run both with and without antigen induction. With the right parameters associated with aiiA, a parameter set with spontaneous self-activation and lysis can be adjusted into a parameter set where cells fully activate and lyse only in response to antigen (Table 3 and 4).
                            </P></DIV><TABLE class="table table-bordered Results-Table"><THEAD><TR><TH colspan="5">Table 3: A system with positive regulation of LuxR that self-activated spontaneously was modified by adding aiiA or increasing the degradation rate of LuxR, resulting in the desired behavior.</TH></TR></THEAD><TBODY><TR><TH>Score</TH><TH>Animation, no antigen</TH><TH>Animation, antigen</TH><TH>Parameters</TH></TR><TR><TD>Original set</TD><TD>99.98</TD><TD><A href="https://static.igem.org/mediawiki/2017/0/03/T--Wageningen_UR--PosRegSpontNegShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/c/c8/T--Wageningen_UR--PosRegSpontPosShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/c/c6/T--Wageningen_UR--Model-PosRegSpont.pdf" target="_blank">link</A></TD></TR><TR><TD>Original Set &amp; aiiA</TD><TD>0.22</TD><TD><A href="https://static.igem.org/mediawiki/2017/6/64/T--Wageningen_UR--PosRegaiiANegShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/7/78/T--Wageningen_UR--PosRegaiiAPosShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/3/31/T--Wageningen_UR--Model-PosRegaiiA.pdf" target="_blank">link</A></TD></TR><TR><TD>Original Set &amp; increased LuxR deg.</TD><TD>0.24</TD><TD><A href="https://static.igem.org/mediawiki/2017/1/1e/T--Wageningen_UR--PosRegLuxRNegShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/9/99/T--Wageningen_UR--PosRegLuxRPosShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/e/ee/T--Wageningen_UR--Model-PosRegLuxR.pdf" target="_blank">link</A></TD></TR></TBODY></TABLE><TABLE class="table table-bordered Results-Table"><THEAD><TR><TH colspan="5">Table 4: A system with negatively regulation of LuxR that self-activated spontaneously was modified by adding aiiA or increasing the degradation rate of LuxR, resulting in the desired behavior.</TH></TR></THEAD><TBODY><TR><TH>Score</TH><TH>Animation, no antigen</TH><TH>Animation, antigen</TH><TH>Parameters</TH></TR><TR><TD>Original set</TD><TD>99.97</TD><TD><A href="https://static.igem.org/mediawiki/2017/c/c0/T--Wageningen_UR--NegRegSpontNegShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/b/b0/T--Wageningen_UR--NegRegSpontPosShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/f/f4/T--Wageningen_UR--Model-NegRegSpont.pdf" target="_blank">link</A></TD></TR><TR><TD>Original Set &amp; aiiA</TD><TD>0.059</TD><TD><A href="https://static.igem.org/mediawiki/2017/0/04/T--Wageningen_UR--NegRegaiiANegShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/a/ac/T--Wageningen_UR--NegRegaiiAPosShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/d/d3/T--Wageningen_UR--Model-NegRegaiiA.pdf" target="_blank">link</A></TD></TR><TR><TD>Original Set &amp; increased LuxR deg.</TD><TD>0.084</TD><TD><A href="https://static.igem.org/mediawiki/2017/4/46/T--Wageningen_UR--NegRegLuxRNegShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/b/bb/T--Wageningen_UR--NegRegLuxRPosShort.mp4" target="_blank">link</A></TD><TD><A href="https://static.igem.org/mediawiki/2017/4/42/T--Wageningen_UR--Model-NegRegLuxR.pdf" target="_blank">link</A></TD></TR></TBODY></TABLE><P>
                            By adding aiiA to our model, we can simulate a model where cells are able to communicate the presence of antigen, resulting in a population-wide signal (Video 2). But in the absence of antigen, cells remain inactive and no fluorescence is generated. This suggests that aiiA can remove enough AHL to prevent spontaneous lysis, but does not remove so much that it prevents antigen-induced cell lysis.

                        </P><VIDEO controls=""><SOURCE src="https://static.igem.org/mediawiki/2017/7/76/T--Wageningen_UR--Model-aiiAfluorescencecomp.mp4" type="video/mp4"><SOURCE src="https://static.igem.org/mediawiki/2017/1/17/T--Wageningen_UR--Model-aiiAfluorescencecomp.webm" type="video/webm"> Your browser does not support the video tag.
                        </SOURCE><DIV class="figure-center-caption"><B>Video 2: </B> Fluorescence produced by the original system (top) versus the system with aiiA included (bottom). On the right side is shown the system induced with antigen. On the left, the system in the abscence of antigen. This system was simulated with negative regulation of LuxR.
                        </DIV></SOURCE><SECTION id="LuxR"><H5>Influence of LuxR stability</H5><P>
                            Analogous to adding aiiA, simulations with increased LuxR degradation rates were also carried out. All parameters are identical, except for the LuxR degradation rate, and no aiiA is present. Here, spontaneous lysis was also prevented without the system losing the ability to lyse upon induction (Video 3). As with our temporal model, this suggests another system parameter that could be altered experimentally to improve performance.
                        </P><VIDEO controls=""><SOURCE src="https://static.igem.org/mediawiki/2017/0/0f/T--Wageningen_UR--NegRegLuxRGFP.mp4" type="video/mp4"><SOURCE src="https://static.igem.org/mediawiki/2017/2/27/T--Wageningen_UR--NegRegLuxRGFP-new.webm" type="video/webm"> Your browser does not support the video tag.
                        </SOURCE><DIV class="figure-center-caption"><B>Video 3: </B>Fluorescence produced by the original system (top) versus the system with decreased LuxR stability (bottom). On the right side is shown the system induced with antigen. On the left, the system in the abscence of antigen. This system was simulated with negative regulation of LuxR.
                        </DIV></SOURCE><SECTION id="SpatialSummary"><H5>Summary</H5><P>
                            Two strategies for tuning the system into having the desired behavior using experimentally tunable parameters have been successfully tested <I>in silico</I> with both strategies giving very comparable results.
                        </P><P>
                            In the spatiotemporal model, the system behaved similarly to the temporal model. One major difference was that the number of activator cells can be lower while still being able to activate the entire population after detecting antigen. This is because in the temporal model, activator cells have to produce enough AHL to saturate the entire system, as AHL is instantaneously diluted homogeneously across the entire system. In the spatiotemporal model, activator cells can trigger lysis of all the cells in the population by producing enough AHL to activate their direct neighbors.
                        </P><P>
                            Cell density is very important to QS. However, our proposed QS system was able to function even though the cell density was increasing during the simulation. The system can therefore be robust to changes in cell density, which is important for the application of Mantis.
                        </P><P>
                            Furthermore, two straightforward ways to change an overly sensitive system into one with the desired behavior were demonstrated. By only increasing the rate of LuxR degradation, or by only adding the aiiA enzyme, the system could be tuned to only generate fluorescence in the presence of antigen. This is important because in our experimental subproject we encountered the same problem of spontaneous auto-activation as is predicted in many of our simulations.
                        </P></SECTION><SECTION id="conclusion"><H3>Conclusion</H3><DIV class="Textbox"><P>
                                In the global parameter space, only a very small number of parameter sets have the desired behavior. This can simply be explained by the size of the parameter space. From the lab we know that the lux operon is prone to self-activation, even at low cell densities. When visualizing the entire parameter space. the desired behavior lies in between the area representing spontaneous activation and the area representing no QS (Figure 10). By lowering the sensitivity of the QS system, a system will move towards that area of the parameter space where the desired behavior is found. We found that by changing the degradation rate of LuxR or with the addition of aiiA the system could transition from either of these two types of behaviour into the desired behavior.
                            </P><P>
                                We hypothesized that negatively regulated LuxR may have a faster response time and that they would be able to switch from a state where they would take up and sense AHL to a state where they would produce and release AHL. While we did find some evidence to suggest negatively regulated systems are faster, and indeed the parameter set found that had both the desired behavior and the quickest generation of fluorescent signal was one with negative LuxR regulation.
                            </P><P>
                                To solve the problem of spontaneous auto-activation observed in the lab, we considered introducing the aiiA enzyme that degrades AHL. Therefore, we tested this hypothesis with our mathematical model and found that by introducing aiiA, the sensitivity of a system to AHL can be reduced while maintaining the ability of the system switch from an OFF-state to an ON-state, self-activate, and cell-signal to its neighboring cells. Both for positive and negative regulation of LuxR, we were able to show that only by adding aiiA, the desired functionality can be obtained. This hypothesis was tested and <A href="https://2017.igem.org/Team:Wageningen_UR/Results/Quorum_Sensing#QSM">confirmed in the lab</A>.
                            </P><P>
                                Similarly, we found that the degradation rate of LuxR is a parameter that plays a large role in determining the system properties. As the LuxR degradation is tunable in the lab, by adding a degradation tag to the protein, we also explored increasing the LuxR degradation rate. Here we find similar results, where for some spontaneously self-activating parameters, we can obtain the desired behavior by only adding aiiA to the system. This was also successful for both positive and negative regulation of LuxR.
                            </P></DIV></SECTION><SECTION id="citations"><DIV class="Textbox Citations"><H3>
											References
											</H3><OL><LI>Williams, Joshua W., et al. &quot;Robust and sensitive control of a quorum‐sensing circuit by two interlocked feedback loops.&quot; <I>Molecular systems biology</I> 4.1 (2008): 234.</LI><LI>Haseltine, Eric L., and Frances H. Arnold. &quot;Implications of rewiring bacterial quorum sensing.&quot; <I>Applied and environmental microbiology</I> 74.2 (2008): 437-445.</LI><LI>Miyashiro, Tim, and Edward G. Ruby. &quot;Shedding light on bioluminescence regulation in Vibrio fischeri.&quot; <I>Molecular microbiology</I> 84.5 (2012): 795-806.</LI><LI>Bassler, Bonnie L., and Richard Losick. &quot;Bacterially speaking.&quot; <I>Cell</I> 125.2 (2006): 237-246.</LI><LI>Sourjik, Victor, and Ned S. Wingreen. &quot;Responding to chemical gradients: bacterial chemotaxis.&quot; <I>Current opinion in cell biology</I> 24.2 (2012): 262-268.</LI><LI>Moukalled, F., L. Mangani, and M. Darwish. <I>&quot;The finite volume method in computational fluid dynamics.&quot;</I> (2016).</LI></OL></DIV></SECTION></VIDEO></SECTION></VIDEO></SECTION></VIDEO></SECTION></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid FooterWrapper"><DIV class="FooterImage"><DIV class="col-lg-12 FooterMainColumn"><DIV class="col-lg-8"><DIV id="Footer"><P>iGEM Wageningen 2017. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link" style="padding-right:0;">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>