<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Results_Cpx skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Results/Cpx</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="Main-Menu"><DIV class="col-lg-10 MainMenu-Center-Column MenuCenter"><DIV class="Project MenuButtonColumn"><DIV class="Project-Top TopMenuButtonColumn"><UL><LI class="project-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Project" class="nav-item" id="project">Project</A></LI></UL></DIV></DIV><DIV class="Background MenuButtonColumn"><DIV class="Background-Top TopMenuButtonColumn"><UL><LI class="background-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Background" class="nav-item active" id="background">Background</A></LI></UL></DIV></DIV><DIV class="Wet-Lab MenuButtonColumn"><DIV class="Wet-Lab-Top TopMenuButtonColumn"><UL><LI class="wet-lab-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Wet-lab" class="nav-item active" id="wet-lab">Wet-lab</A></LI></UL></DIV><DIV class="Wet-Lab-Bottom SubMenuButtonColumn"><UL id="wet-lab-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results" class="results-sub-nav-item sub-nav-item">Results</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Parts" class="parts-sub-nav-item sub-nav-item">BioBricks</A></LI></UL></DIV></DIV><DIV class="Model MenuButtonColumn"><DIV class="Model-Top TopMenuButtonColumn"><UL><LI class="model-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Model" class="model nav-item" id="model">Model</A></LI></UL></DIV><DIV class="Model-Bottom SubMenuButtonColumn"><UL id="model-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics" class="kinetics-sub-nav-item sub-nav-item">Cpx Kinetics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/QS" class="qs-sub-nav-item sub-nav-item">Quorum Sensing</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx" class="integrationcpx-sub-nav-item sub-nav-item">Cpx Integration</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS" class="integrationQS-sub-nav-item sub-nav-item">QS Integration</A></LI></UL></DIV></DIV><DIV class="Human-Practices MenuButtonColumn"><DIV class="Human-Practices-Top TopMenuButtonColumn"><UL><LI class="human-practices-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Human_Practices" class="nav-item" id="human-practices">Human Practices</A></LI></UL></DIV><DIV class="Human-Practices-Bottom SubMenuButtonColumn"><UL id="human-practices-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated" class="integrated-sub-nav-item sub-nav-item ">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Silver" class="safety-sub-nav-item sub-nav-item">Safety &amp; Ethics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Collaborations" class="collaboration-sub-nav-item sub-nav-item">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Future" class="future-sub-nav-item sub-nav-item">Future Perspectives</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Engagement" class="engagement-sub-nav-item sub-nav-item">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Meetup" class="meetup-sub-nav-item sub-nav-item">Meetup</A></LI></UL></DIV></DIV><DIV class="Application MenuButtonColumn"><DIV class="Application-Top TopMenuButtonColumn"><UL><LI class="application-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Applied_Design" class="nav-item" id="application">Application</A></LI></UL></DIV><DIV class="Application-Bottom SubMenuButtonColumn"><UL id="application-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Demonstrate" class="demo-sub-nav-item sub-nav-item">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Hardware" class="device-sub-nav-item sub-nav-item">Device</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Application/Rollout" class="rollout-sub-nav-item sub-nav-item">Roll-out Plan</A></LI></UL></DIV></DIV><DIV class="Notebook MenuButtonColumn"><DIV class="Notebook-Top TopMenuButtonColumn"><UL><LI class="notebook-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook" class="notebook nav-item" id="notebook">Notebook</A></LI></UL></DIV><DIV class="Notebook-Bottom SubMenuButtonColumn"><UL id="notebook-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols" class="protocols-sub-nav-item sub-nav-item">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/InterLab" class="interlab-sub-nav-item sub-nav-item">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Journal" class="journal-sub-nav-item sub-nav-item">Lab Journal</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Achievements" class="achievements-sub-nav-item sub-nav-item">Achievements</A></LI></UL></DIV></DIV><DIV class="Team MenuButtonColumn"><DIV class="Team-Top TopMenuButtonColumn"><UL><LI class="team-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Team" class="team nav-item" id="team">Team</A></LI></UL></DIV><DIV class="Team-Bottom SubMenuButtonColumn"><UL id="team-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Team/Members" class="members-sub-nav-item sub-nav-item">Members</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Attributions" class="attributions-sub-nav-item sub-nav-item">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Acknowledgements" class="Acknowledgements-sub-nav-item sub-nav-item">Acknowledgements</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="MobileMenu"><UL id="MainMobileMenu"><LI><BUTTON class="project-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Project'" type="button">Project</BUTTON></LI><LI><BUTTON class="background-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Background'" type="button">Background</BUTTON></LI><LI><BUTTON class="wet-lab-li-mo">Wet-lab</BUTTON></LI><LI><BUTTON class="model-li-mo">Model</BUTTON></LI><LI><BUTTON class="human-practices-li-mo">Human Practices</BUTTON></LI><LI><BUTTON class="application-li-mo">Application</BUTTON></LI><LI><BUTTON class="notebook-li-mo">Notebook</BUTTON></LI><LI><BUTTON class="team-li-mo">Team</BUTTON></LI></UL><UL id="MobileSubMenu"><LI class="wet-lab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Wet-lab'" type="">Wet-lab</BUTTON></LI><LI class="model-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model'">Model</BUTTON></LI><LI class="application-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Applied_Design'">Application</BUTTON></LI><LI class="human-practices-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Human_Practices'">Human Practices</BUTTON></LI><LI class="notebook-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook'">Notebook</BUTTON></LI><LI class="team-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team'">Team</BUTTON></LI><LI class="parts-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Parts'" type="button">Parts</BUTTON></LI><LI class="results-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Results'" type="button">Results</BUTTON></LI><LI class="kinetics-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics'" type="button">Cpx Kinetics</BUTTON></LI><LI class="qs-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/QS'" type="button">Quorum Sensing</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx'" type="button">Cpx integration</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS'" type="button">QS integration</BUTTON></LI><LI class="integrated-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated'" type="button">Integrated HP</BUTTON></LI><LI class="safety-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Silver'" type="button">Safety &amp; Ethics</BUTTON></LI><LI class="collaboration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Collaborations'" type="button">Collaborations</BUTTON></LI><LI class="future-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Future'" type="button">Future Perspectives</BUTTON></LI><LI class="engagement-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Engagement'" type="button">Engagement</BUTTON></LI><LI class="meetup-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Meetup'" type="button">Meetup</BUTTON></LI><LI class="demo-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Demonstrate'" type="button">Demonstrate</BUTTON></LI><LI class="device-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Hardware'" type="button">Device</BUTTON></LI><LI class="rollout-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Application/Rollout'" type="button">Rollout Plan</BUTTON></LI><LI class="protocols-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols'" type="button">Protocols</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/InterLab'" type="button">InterLab</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Journal'" type="button">Journal</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Achievements'" type="button">Achievements</BUTTON></LI><LI class="members-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team/Members'" type="button">Members</BUTTON></LI><LI class="attributions-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Attributions'" type="button">Attributions</BUTTON></LI></UL></DIV><DIV class="container-fluid OurContent"><DIV class="row"><DIV class="col-lg-8"><DIV class="Main-Border"><DIV id="SideMenu-Wrapper"><SIDENAV id="SideMenu"><DIV class="menu-head"><H4>Cpx Signal Transduction</H4></DIV><UL class="sidebar-nav"><LI class="menu-item"><A href="#AP">Approach</A></LI><LI><A href="#CON">Conclusion</A></LI><LI><A href="#DIS">Discussion</A></LI></UL></SIDENAV></DIV><DIV class="Main-Center-Content-Column Overview"><DIV id="breadcrumb-wrapper"><UL class="breadcrumb"><LI><A href="https://2017.igem.org/Team:Wageningen_UR">Home</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Wet-lab">Wet-lab</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI>Cpx Signal Transduction</LI></UL></DIV><SECTION class="CpxIntro"><DIV class="Title"><H1>Cpx Signal Transduction</H1></DIV><DIV class="Textbox Results-Desc"><P>
						The aim of this project was to create a modular bacterial membrane receptor capable of detecting extracellular antigens leading to an intracellular signal. This was done by fusing an affinity body to the Cpx system’s auxiliary inhibitor CpxP. Targets present in the medium will be bound by the fusion, leading to alleviation of the Cpx inhibition. This causes the membrane receptor system to become activated, resulting in the generation of an intracellular signal, visualized by mRFP1 fluorescence directly coupled to this system.				
						</P></DIV></SECTION><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#Background" aria-expanded="true" aria-controls="Background"><DIV class="col-xs-11">	
												Background 
											</DIV></A></H4></DIV><DIV id="Background" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
										The sensing module of the Mantis diagnostic is essential for signal generation upon recognition of a  disease antigen. Most receptor systems in bacteria are so-called Two-Component Signal Transduction Systems. Their name comes from the fact that they contain a membrane-bound sensor and an intracellular response regulator that often acts as a transcription factor. However, none of these systems can sense disease antigens, and it is still challenging with current technology to directly change substrate specificity of such receptors. Instead, we chose to engineer an inhibitor, which influences the receptor. The Cpx system (Figure A) is involved in sensing and responding to stress on the bacterial envelope. It consists of a membrane sensor CpxA and a response regulator CpxR, as well as the auxiliary inhibitor CpxP. The latter is able to inhibit CpxA by protein-to-protein interaction, causing the system to remain in an off-state [1]. This prevents the CpxR regulator from being activated by CpxA. Therefore it will not act as a transcription factor for genes involved in the alleviation of stress and reinstatement of homeostasis [2].									
										</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure A:</B> Visual representation of the Cpx system in native conditions. Signals  associated with envelope stress can activate CpxA, often by CpxP dissociation, resulting in phosphorylation of CpxR. Upon dimerization this response regulator can act as a transcription factor for genes involved in the reinstatement of homeostasis.
											</DIV></DIV></DIV></DIV></DIV></DIV><SECTION id="AP"><H2>Approach</H2><P>
										The sensing module of the Mantis diagnostic is required for detection of the antigens and subsequent signal transduction in the bacteria. We approached this by creating a recombinant receptor based on the native Cpx system. This recombinant Cpx system will detect antigens, resulting in downstream signalling. As the Cpx system is normally suppressed by the inhibitor CpxP, we aimed to introduce the antigens as a new substrate for the inhibitor. Thereby, presence of the antigen would result in alleviation of suppression by making CpxP dissociate from the membrane sensor CpxA. 
										</P><P>
										Introduction of antigens as a new substrate for CpxP was done through fusion to an affinity body. This molecule is designed to bind a specific antigen with high affinity. Since antigen specific affinity bodies are produced in
										<A href="https://2017.igem.org/Team:Wageningen_UR/Results/Affinity_Bodies">another part</A> of this project, a placeholder affinity molecule specific for Immunoglobulin G (IgG) was used in this subproject. This would allow testing of the sensing module with IgG [3] as a placeholder antigen. 
										</P><P>
										To determine whether IgG could induce the system, it is necessary to remove the outer membrane from the <I>Escherichia coli</I> cells by a method called spheroplasting. Without removing the outer membrane, the antigen place-holder would not be able to reach the Cpx system located in the periplasm[4]. However, spheroplasting is known to cause activation of the Cpx system. CpxP is normally free in the periplasm, causing them to be titrated away if cells were spheroplasted [5]. Therefore, the CpxP-affinity body is tethered to the inner membrane by a transmembrane Maltose-Binding Protein (MBP) mutant.										
										</P><P>
										Furthermore, an mRFP1 reporter is placed in the CpxR operon, resulting in red fluorescence if CpxA is activated (Figure 1). In the final version of the Mantis device activation of the Cpx system will be visualized by Bimolecular Fluorescence Complementation (BiFC) linked to CpxR dimerization (see the 
										<A href="https://2017.igem.org/Team:Wageningen_UR/Results/SpecificVisualization">Specific Visualization page</A>).
										</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 1:</B> Visual representation of the proposed recombinant Cpx system. Presence of disease antigens, such as viral particles, in the environment of the cell can activate the Cpx system by dissociation of CpxP, resulting in downstream mRFP1 gene transcription.
											</DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#Methods" aria-expanded="true" aria-controls="Methods"><DIV class="col-xs-11">	
												Methods 
											</DIV></A></H4></DIV><DIV id="Methods" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><H3> 1. CpxP – affinity body fusions</H3><P>					The CpxP gene, isolated from <I>E. coli</I> K12 genome, was fused together with a synthetic affinity body gene, specific for IgG F<SUB>c</SUB> region, at both the C– and N-terminus. Resulting constructs were named CpxP-Aff and Aff-CpxP. Under control of the tac inducible promoter (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K864400">BBa_K864400</A>), these are cloned into the pSB1C3 backbone (<A href="http://parts.igem.org/Part:BBa_K2387017">CpxP-Aff: BBa_K2387017</A> and <A href="http://parts.igem.org/Part:BBa_K2387018">Aff-CpxP: BBa_K2387018</A>), already containing the mRFP1 reporter under control of the pCpxR promoter (<A href="http://parts.igem.org/Part:BBa_K339007">BBa_K339007</A>). Furthermore, the native CpxP gene was assembled into a similar construct, to act as a control (<A href="http://parts.igem.org/Part:BBa_K2387014">BBa_2387014</A>).  Figure B gives a visual representation of the created constructs:									
										</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure B:</B> Constructs of CpxP-affinity body fusions. (A): Inserts mRFP1, under control of pCpxR, and CpxP, under control of ptac. (B): Inserts mRFP1, under control of pCpxR, and CpxP-affinity body fusion, under control of ptac. (C): Inserts mRFP1, under control of pCpxR, and affinity body-CpxP fusion, under control of ptac.
											</DIV></DIV><H3> 2. Inhibitory capacity assay for CpxP – affinity body fusions</H3><P>
										The inhibitory capacity of both CpxP – affinity body fusions were compared to that of the native CpxP gene by expressing these proteins in an <I>E. coli</I>ΔCpxP knock-out strain (JW5558-1). Strains carrying the different constructs were inoculated in 10 mL M9 cultures containing 2 g/L glucose to suppress leaky transcription. Cells were grown for 5 hours at 37°C and then induced with 0.2 mM IPTG to express CpxP-affibody fusions. After overnight growth, the cells were harvested by centrifugation at 4,700 rpm for 5 minutes. The pellets were resuspended in 1 mL PBS buffer, after which the suspensions were stored at 4°C to maturate the mRFP1 fluorophores. Finally, cells were harvested, washed in PBS buffer, and measured in the plate reader. A BioTek Synergy Mx Monochromator-based Microplate Reader was used to measure red fluorescence at excitation/emission of 580/612 and Optical Density at 600nm.
										</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure C:</B> Fluorescence/OD<SUB>600nm</SUB> resulting from expression of CpxP-Affinity body variants. <I>E. coli</I> K12 ΔCpxP strains containing CpxP, CpxP-His, CpxP-Aff, CpxP-Aff-His, Aff-CpxP and Aff-CpxP-His constructs were induced with 0.2 mM IPTG. As a positive control a strain containing only the RFP reporter was included. A strain containing CpxP which was not induced was included to determine the degree of leakyness of the ptac promoter. 
											</DIV></DIV><P>
										Resulting fluorescence data (Figure C) indicated that the Cpx system was suppressed in the presence of CpxP – affinity body fusions to the same - or higher extend as in the presence of CpxP.
										</P><H3> 3.	Membrane tethering of CpxP – affinity bodies</H3><P>
										Membrane tethered variants of the CpxP-affinity body fusions were created by C-terminal fusion to  MBP misfolder, MalE24-1. The MBP mutant was created by introducing a point mutation using PCR, causing the alanine at position 24 of the signal sequence to aspartate. This mutation results in localization of the MBP to the inner membrane [6]. The resulting triple fusions, named MalE24-1-CpxP-Aff and MalE24-1-Aff-CpxP, were cloned under control of the ptac promoter in the pSB1C3 backbone (<A href="http://parts.igem.org/Part:BBa_K2387023:Design">MalE24-1-CpxP-Aff: BBa_K2387023</A> and <A href="http://parts.igem.org/Part:BBa_K2387024">MalE24-1-Aff-CpxP: BBa_K2387024</A>), already containing the mRFP1 reporter under control of the pCpxR promoter (<A href="http://parts.igem.org/Part:BBa_K339007">BBa_K339007</A>). Furthermore, the MalE24-1-CpxP fusion was also created and assembled into this construct, to act as a control (<A href="http://parts.igem.org/Part:BBa_K2387022">BBa_K2387022</A>).  										
										</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure D:</B> Constructs of CpxP-affinity body fusions. (A): Inserts mRFP1, under control of pCpxR, and MalE24-1-CpxP fusion, under control of ptac. (B): Inserts mRFP1, under control of pCpxR, and MalE24-1-CpxP-affinity body fusion, under control of ptac. (C): Inserts mRFP1, under control of pCpxR, and MalE24-1-affinity body-CpxP fusion, under control of ptac.
											</DIV></DIV><H3> 4.	Inhibitory capacity assay for MalE24-1-CpxP-affinity body fusions in spheroplasts</H3><P>
										The inhibitory capacity assay of the MBP-CpxP-affinity body fusions  were compared to that of the native <I>CpxP</I> gene in spheroplasts. Strains carrying the constructs were grown in 10 mL LB overnight at 37°C. The next day, fresh 5 mL cultures were created with 10% inoculum. After growth at 37°C for 1.5 hours the cultures were induced with 0.2 mM IPTG to express the membrane bound CpxP – affinity body fusions. After another hour of growth the cultures were harvested by 2 minutes of centrifugation at 4,700 rpm. Pellets were spheroplasted according to the protocol of Sun <I>et al.</I> [7]: Pellets were resuspended in 500 µL 0.8 M sucrose, after which the following reagents were added in the mentioned order: 30 µL 1 M Tris-HCl (pH8); 24 µL 0.5 mg/mL lysozyme; 6 µL 5 mg/mL DNase;  6 µL 125 mM EDTA-NaOH (pH8). Suspensions were incubated for 20 minutes at room temperature, after which 100 µL STOP solution (10 mM Tris-HCl (pH8), 0.7 M sucrose, 20 mM MgCl2) was added to stop degradation of the membranes.
										</P><P>
										After the spheroplasting procedure, cells were kept on ice, spun down and resuspended in 1 mL of M9 medium. Culture samples of a 100 µL samples were added to 96 wells plates and measured in the plate reader for fluorescence (excitation/emission: 580nm/612nm) and OD<SUB>600</SUB>. It was found that that the tethered fusions, as well as the native CpxP, were able to inhibit the system in spheroplast conditions (Figure E). Differences seen in inhibitory capacity between the different fusions could possibly be explained by the conformational changes, resulting from the fusions. These protein structures of the affinity body might hinder the <I>CpxP</I> gene from properly binding the CpxA sensor, which would result in a lower inhibitory capacity. 
										</P><DIV class="figure-fullwidth"><DIV id="Inhib-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure E:</B> Growth measurements of spheroplasted <I>E. coli</I>ΔCpxP strains carrying constructs of different MalE24-1-CpxP-Affinity body fusions, as well as a control without CpxP and a native CpxP control. Expression of the tethered CpxP-Affinity body fusions and the native control were induced with 0.2 mM IPTG. The left graph shows that the positive control has high activation of the Cpx system due to absence of the inhibitor. The right graph shows the same results, without the positive control, indicating that the membrane tethered fusions can effectively suppress the Cpx system.
												</DIV></DIV></DIV><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure E:</B> Growth measurements of spheroplasted <I>E. coli</I>ΔCpxP strains carrying constructs of different MalE24-1-CpxP-Affinity body fusions, as well as a control without CpxP and a native CpxP control. Expression of the tethered CpxP-Affinity body fusions and the native control were induced with 0.2 mM IPTG. The left graph shows that the positive control has high activation of the Cpx system due to absence of the inhibitor. The right graph shows the same results, without the positive control, indicating that the membrane tethered fusions can effectively suppress the Cpx system.
											</DIV></DIV><H3> 5.	IgG sensitivity assay</H3><P>
										In this experiment, IgG was added to strains carrying the fusion protein to verify if antigens could induce the system. Similar to experiment 4, cultures were grown, induced and spheroplasted, after which samples were added to 96 wells plates together with varying concentrations of IgG (0.05  to 0.5 mg/mL). Suspensions were measured in the plate reader for fluorescence (Excitation/emission: 580nm/610nm) and OD<SUB>600</SUB> over a period of 6 hours. Even though cells only remain spheroplasted for ±5 generations [8, 9] (±2.5 hours in M9 medium [10]), it was worthwhile to measure for the extended period of time to account for maturation times of the mRFP1 fluorophores [11]. Figure 2, located in the conclusion section, indicates that no activation of the Cpx system was induced by the presence of IgG. 
										</P></DIV></DIV></DIV></DIV></SECTION><SECTION id="CON"><H2>Conclusion</H2><P>In this project a recombinant bacterial receptor was constructed to have affinity for disease antigens. By combining affinity bodies with the <I>E. coli</I>’s native Cpx system’s auxiliary inhibitor CpxP, we attempted to create an antigen sensitive suppressor. This sensor was measured to have a similar inhibitory capacity as the native CpxP protein, indicating that the fusion did in fact not hinder the Cpx functioning. An inner membrane-bound variant of this system was created by a triple fusion of misfolded MBP, CpxP and affinity body. This tethered variant was proven to suppress the Cpx system in case the outer membrane was removed, which is necessary for the disease antigens to reach the receptor. To test whether these antigens could actually induce the engineered system, a placeholder antigen IgG was added to the medium. It was shown that no induction resulted from the addition of IgG (Figure 2). Therefore we can conclude that the engineered system is not sensitive for antigens under the conditions tested. 
					
					
					</P><DIV class="figure-fullwidth"><DIV id="IgG-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 2:</B> Cpx activation levels of <I>E. coli</I>ΔCpxP strains, carrying varying MBP-CpxP-Aff constructs, in the presence of IgG, visualized by mRFP1 reporter fluorescence corrected by OD<SUB>600</SUB>. <B>Left:</B> Fluorescence in MBP-CpxP-Aff fusion carrying strain in presence and absence of IgG. <B>Right:</B> Fluorescence in MBP-Aff-CpxP fusion carrying strain in presence and absence of IgG.
							</DIV></DIV></DIV><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 2:</B> Cpx activation levels of <I>E. coli</I>ΔCpxP strains, carrying varying MBP-CpxP-Aff constructs, in the presence of IgG, visualized by mRFP1 reporter fluorescence corrected by OD<SUB>600nm</SUB>. Left: Fluorescence in MBP-CpxP-Aff fusion carrying strain in presence and absence of IgG. Right: Fluorescence in MBP-Aff-CpxP fusion carrying strain in presence and absence of IgG.
						</DIV></DIV><P>
					Despite the negative results obtained from the IgG sensitivity assay, this project was integrated with the <A href="https://2017.igem.org/Team:Wageningen_UR/Results/SpecificVisualization">Specific Visualization</A> project. Together, these two projects would hypothetically lead to signal generation, without the need for transcription, upon binding of the IgG by the recombinant receptor. To make this integration possible the CpxP-affinity body containing constructs were cloned into medium copy number plasmid pSB3T5. The setup and results of the experiment can be found on the <A href="https://2017.igem.org/Team:Wageningen_UR/Demonstrate#combination">demonstrate</A> page.

</P></SECTION><SECTION id="DIS"><H2>Discussion</H2><P>
					The outcome of this project is difficult to interpret as it can be caused by a variety of factors. First of all, it might be possible that the fusion of affinity body to the CpxP inhibitor cannot bind antigens due to steric hindrance. In this fusion, there might simply be no room for binding to the IgG antigen. Another possibility is that the binding of IgG does not cause dissociation of CpxP from the CpxA, meaning that the reporter gene is not transcribed. In this case, it might be worthwhile to introduce a linker in the fusion between the MalE24-1 and CpxP/affinity body. This linker would theoretically give the CpxP more space to dissociate in the presence of IgG. 
					</P><P>
					Furthermore, no IgG binding assay was performed on the affinity body due to time constraints. Therefore it is possible that the IgG affinity body lacks the presumed high affinity for IgG. In this case, the results obtained in this study might result from this lack of affinity, instead of the dysfunctionality of the designed system.
					</P><P>
					Another factor that makes measurements on the created system difficult, is the fragility of spheroplasted cells. These cells only remain in their conformation for ±5 generations. Since expression of the reporter requires active cells and the timeframe of measurement is limited to 90 minutes. This timeframe is not ideal for the measurement of the system, especially considering that fluorophores take time to mature. As an alternative, we would suggest adapting an outer membrane receptor system in a similar way as was done in this project. This would eliminate the trouble of removing the outer membrane and thereby creating a more robust and stable bacterial detection system. Another approach that could be taken is to convert the whole system to Gram-positive bacteria, which do only contain one membrane. 
					</P><P>
					Combined, all these factors make it difficult to identify the precise cause of negative results obtained in the measurements. During subsequent experiments, it might be possible to pinpoint and resolve problems with the setup of the experiments, leading to a signal transduction system that has potential for fast detection and visualization. 
					</P></SECTION><SECTION class="references"><DIV class="Textbox Citations"><H3>
							References
							</H3><OL><LI>Zhou, Xiaohui, et al. &quot;Structural basis for two-component system inhibition and pilus sensing by the auxiliary CpxP protein.&quot; Journal of Biological Chemistry 286.11 (2011): 9805-9814.
									</LI><LI>Raivio, Tracy L., Daniel L. Popkin, and Thomas J. Silhavy. &quot;The Cpx envelope stress response is controlled by amplification and feedback inhibition.&quot; Journal of bacteriology 181.17 (1999): 5263-5272.
									</LI><LI>Nilsson, Björn, et al. &quot;A synthetic IgG-binding domain based on staphylococcal protein A.&quot; Protein Engineering, Design and Selection 1.2 (1987): 107-113.
									</LI><LI>Muller, Sybille, et al. &quot;TransMabs: cell-penetrating antibodies, the next generation.&quot; Expert opinion on biological therapy 5.2 (2005): 237-241.
									</LI><LI>Raivio, Tracy L., et al. &quot;Tethering of CpxP to the inner membrane prevents spheroplast induction of the Cpx envelope stress response.&quot; Molecular microbiology 37.5 (2000): 1186-1197.
									</LI><LI>Fikes, JOHN D., and P. J. Bassford. &quot;Export of unprocessed precursor maltose-binding protein to the periplasm of Escherichia coli cells.&quot; Journal of bacteriology 169.6 (1987): 2352-2359.
									</LI><LI>Sun, Yen, Tzu-Lin Sun, and Huey W. Huang. &quot;Physical properties of Escherichia coli spheroplast membranes.&quot; Biophysical journal 107.9 (2014): 2082-2090.
									</LI><LI>Ranjit, Dev K., and Kevin D. Young. &quot;The Rcs stress response and accessory envelope proteins are required for de novo generation of cell shape in Escherichia coli.&quot; Journal of bacteriology 195.11 (2013): 2452-2462.
									</LI><LI>Martinac, Boris, et al. &quot;Patch clamp electrophysiology for the study of bacterial ion channels in giant spheroplasts of E. coli.&quot; Bacterial Cell Surfaces: Methods and Protocols (2013): 367-380.
									</LI><LI>Reshes, G., et al. &quot;Timing the start of division in E. coli: a single-cell study.&quot; Physical biology 5.4 (2008): 046001.
									</LI><LI>Campbell, Robert E., et al. &quot;A monomeric red fluorescent protein.&quot; Proceedings of the National Academy of Sciences 99.12 (2002): 7877-7882.
			</LI></OL></DIV></SECTION></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid FooterWrapper"><DIV class="FooterImage"><DIV class="col-lg-12 FooterMainColumn"><DIV class="col-lg-8"><DIV id="Footer"><P>iGEM Wageningen 2017. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link" style="padding-right:0;">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>