Team:Wageningen UR/Results/SpecificVisualization
Project
Background
Wet-lab
Results
BioBricks
Model
Cpx Kinetics
Quorum Sensing
Cpx Integration
QS Integration
Human Practices
Integrated HP
Safety & Ethics
Collaborations
Future Perspectives
Engagement
Meetup
Application
Demonstrate
Device
Roll-out Plan
Notebook
Protocols
InterLab
Lab Journal
Achievements
Team
Members
Attributions
Acknowledgements
Project
Background
Wet-lab
Model
Human Practices
Application
Notebook
Team
Wet-lab
Model
Application
Human Practices
Notebook
Team
Parts
Results
Cpx Kinetics
Quorum Sensing
Cpx integration
QS integration
Integrated HP
Safety & Ethics
Collaborations
Future Perspectives
Engagement
Meetup
Demonstrate
Device
Rollout Plan
Protocols
InterLab
Journal
Achievements
Members
Attributions
Specific Visualization using BiFC
Visualization Strategy
Visualization Results
Optimal Cpx Activation
Conclusions
Home
Wet-lab
Results
Specific Visualization
Specific visualization using BiFC
Figure 1: The BiFC rationale: Target proteins A and B are fused to split reporter YFP (YFPn and YFPc). Once A and B interact, YFPn and YFPc reassemble and regain fluorescence [2].
In order to generate a visual signal in Mantis upon antigen binding, we use the Cpx two-component signal transduction pathway native from Escherichia coli. We combine its protein-protein interactions with Bimolecular Fluorescence Complementation (BiFC) and show how we obtain a rapid and specific visual response upon activation!
We already described the native function of the Cpx two-component signal transduction pathway [1] , and how we can use this system to detect and bind antigens here . However, we need to create an output signal to visually show the presence of antigen.
Our approach combines the native protein-protein interactions of the Cpx pathway with a visualization method called Bimolecular Fluorescence Complementation (BiFC). BiFC is based on the association of fragments of a fluorescent reporter fused to interacting proteins [2] (Figure 1). A fluorophore can be split into two non-fluorescent fragments which reassemble into a fluorescent complex upon interaction between the aforementioned target proteins. BiFC can directly be used in living cells, and does not need addition of substrates (in contrast to luminescent proteins). Another advantage is that a simple spectrophotometer is enough to measure the signal. We use fluorescent protein eYFP, a commonly used reporter in BiFC [2].
Visualization Strategy
Upon activation of the Cpx pathway, CpxP gets titrated away from CpxA which is subsequently activated and autophosphorylates. This phosphogroup is transferred to CpxR, which can then homodimerize [3]. By fusing split reporter proteins to these Cpx-proteins, Cpx pathway activation can directly be visualized!
As a first option we link BiFC to CpxR dimerization (Figure 2A). Upon Cpx activation, CpxR is phosporylated by CpxA, after which CpxR can homodimerize. We visualize this dimerization by fusing eYFP-termini to the C-terminus of CpxR. When dimerization occurs, eYFP can reassemble and a fluorescent signal is formed.
The second possibility is based on the CpxA-CpxR interaction (Figure 2B). When Cpx is activated, CpxA autophosphorylates itself and this phospho-group is transferred to CpxR. We fused eYFP-termini to the C-termini of CpxA and CpxR. When phosphotransfer occurs, eYFP reassembles and a fluorescent signal is formed.
A third possible method uses specific cleavage via TEV protease. The TEV protease is fused to CpxR whereas eYFPn and eYFPc are fused to the cytoplasmic C-terminus of CpxA. The eYFP-termini are fused to antiparallel leucine zippers . Upon Cpx activation, CpxA and CpxR interact and the eYFP-zipper fusions are cleaved off of CpxA and they can freely move through the cytoplasm. The leucine zippers have natural affinity for each other and will act as target proteins in BiFC to facilitate eYFP complementation (Figure 2C).
Figure 2: A) eYFPn and eYFPc are fused (seperately) to CpxR. This way BiFC is used to visualize the CpxR dimerization step. B) eYFPc is fused to CpxA, and eYFPn is fused to CpxR. This way, BiFC is used to visualize the phosphorylation step of the Cpx pathway. C) TEV protease is fused to CpxR, and eYFPn and eYFPc are fused to CpxA (seperately). Upon Cpx pathway activation, the eYFP-termini are cleaved off of CpxA. Leucine zippers are fused to the eYFP-termini to enable the reassembly.
Cloning methods, promoter test and eYFP-termini affinity analysis
Cloning methods
To visualize Cpx activation using the aforementioned protein-protein interactions, several plasmids were assembled to fuse eYFP-termini to Cpx components (Figure A). CpxA and CpxR were amplified from E. coli K12 genome using Q5 Polymerase. E. coli native CpxA contained an illegal EcoRI site; a silent mutation was made to remove this site. N-Leucine zipper and C-Leucine zipper were ordered as gBlocksÂ® from IDT. eYFP was split between amino acids 154 and 155 [4]. CpxA fusions were made at its cytoplasmic C-terminus; CpxR fusions were made at its C-terminus which was found to be optimal for fluorophore reassembly [5].
All constructs were put under control of the inducible pBad/araC promoter (BBa_BI0500) to enable controlled protein expression, and a strong ribosome binding site (RBS) (BBa_B0034) was placed upstream of the created fusions. All CpxR-containing fusions were cloned into high copy number standard backbone pSB1C3 , whereas all CpxA-containing fusions were cloned into medium copy number backbone pSB3T5 . This was done to prevent overexpression and subsequent formation of inclusion bodies containing membrane protein CpxA. All parts were brought together via Golden Gate Assembly using restriction enzyme BsaI and T4 Ligase, and verified by Restriction Digest Analysis and subsequent sequencing.
Figure A: Schematic of the translational units designed to visualize Cpx activation. All units are under control of the inducible pBad/araC promoter and strong RBS BBa_B0034. A) eYFPn is fused to the C-terminus of CpxA via a flexible linker ( CpxA-eYFPn, BBa_K2387010 ); eYFPc is fused to the C-terminus of CpxR ( CpxR-eYFPc, BBa_K2387030 ). B) eYFPn and eYFPc are fused to the C-terminus of CpxR via flexible linkers ( CpxR-eYFPn-CpxR-eFYPc, BBa_K2387032 ). C) eYFPn-nZIPPER and eYFPc-cZIPPER are fused to the C-terminus of CpxA via TEV-cleavable linkers ( CpxA-eYFPn-nZIP-CpxA-cZIP-eYPFc, BBa_K2387078 ); TEV protease is fused to the C-terminus of CpxR via a flexible linker ( CpxR-TEV, BBa_K2387074 ).
Promoter test and eYFP-termini affinity analysis
To test the strength of the pBad/araC promoter combined with RBS BBa_B0034, we fused it directly to eYFP and measured fluorescence using the following protocol . Figure B shows that the expression of eYFP is strongly linked to the concentration of added L-arabinose. On top of this, we tested the fluorescence of the individual N- and C-termini of eYFP, and if these halves have natural affinity for each other, as this could lead to high levels of background fluorescence. The results in Figure B show that this is not the case!
Figure B: The pBad/araC inducible system is used to control eYFP protein expression. This way protein expression levels can be determined using eYFPs fluorescence (ex.:512 nm, em.: 528 nm). Three L-arabinose induction concentrations were used. On top of this, fluorescence of eYFP-termini (eYFPn, eYFPc) and eYFP-termini natural affinity (eYFPn-eYFPc) are tested.
Visualization Results
After confirming that the pBad/araC promoter works and assembling the necessary constructs, we can start visualizing Cpx pathway activation with BiFC. In order to simplify the experiments we activate the Cpx pathway with a known stress factor, KCl [6]. We perform all experiments in E. coli K12. We grow the cells in saltless LB and induce protein expression with a range of 0.02 - 0.2% L-arabinose. CpxA-CpxR protein interaction or CpxR dimerization and subsequent fluorescence resulting from CpxA-CpxR protein interaction or CpxR dimerization is measured over time, and the system is activated at a timepoint of 20 min with 75 mM KCl [5]. View the full protocol here .
The results clearly show a rapid increase in fluorescence after activation of the Cpx pathway when we visualize CpxR-CpxR dimerization, and we see that the signal gets stronger when CpxR-eYFP is expressed at higher levels (Figure 3A). It is clear that within two hours a strong fluorescent signal is detected. We also correctly predicted a positive CpxR dimerization result in our computer model !
Unfortunately, the CpxA-CpxR protein interactions do not show any convincing increase in fluorescence after activation (Figures 3B and 3C). We already predicted similar results in our computer models .
Figure 3: A) CpxR dimerization visualized with different L-arabinose (L-ara) concentrations over time. B) CpxA-CpxR protein interaction visualized with different L-arabinose concentrations over time. C) CpxA-CpxR protein interaction visualized with different L-arabinose concentrations over time. Upon Cpx activation, eYFP-termini are cleaved off of CpxA by TEV protease and reassemble in the cytoplasm using leucine zipper's natural affinity for each other.
Optimal Cpx Activation
The results clearly show that we should pursue CpxR dimerization (Figure 2A) as a means of visualizing Cpx pathway activation. We further investigate this system by applying different levels of stress, as to determine the range of KCl concentrations which generates a fluorescent signal. This helps us in finding the amount of antigen Mantis would need. The protocol for this experiment is the same as before and can be found here .
The results in Figure 4 show that the intensity of the fluorescent signal is heavily dependent on the activation level, but each level of activator shows signal within two hours! This shows that the system is sensitive to small amounts of stress, which can imply that low antigen concentrations could be detected.
Figure 4: CpxR dimerization visualized with L-arabinose concentration of 0.2% and different activator concentrations over time.
Cpx visualization in E. coliÎCpxR
To improve the efficiency of the CpxR dimerization visualization we transformed ( CpxR-eYFPn-CpxR-eFYPc, BBa_K2387032 ) (figure A) into E. coli K12 with a knockout of CpxR. This knockout was taken from the KEIO collection [7]. Using this knockout, theoretically the fraction of protein-protein interactions leading to fluorescent signaling is doubled (Table A).
Table A: Possible CpxR interactions in E. coli K12 wild type and K12ÎCpxR.
Possible interactions
K12 wildtype
K12 CpxR knockout
CpxR - CpxR
x
CpxR-eYFPn â CpxR
x
CpxR-eYFPc â CpxR
x
CpxR-eYFPn â CpxR-eYFPn
x
x
CpxR-eYFPc â CpxR-eYFPc
x
x
CpxR-eYFPn â CpxR-eYFPc
x
x
E. coli K12ÎCpxR was tested under the same conditions and using the same protocol as the wild type E. coli K12. The used protocol can be found here .
Figure C: CpxR dimerization visualized with a L-arabinose concentration of 0.2% and different activator concentrations over time.
Figure C shows that, even though the pattern created by different levels of activation is still the same as in the wildtype E. coli K12 (Figure 4), the signal is more chaotic and the maximal fluorescence/OD600 levels are lower than in wild type K12. A possible explanation is that the CpxR knockout poses a burden on the cell which influences protein production and subsequent reassembly and fluorescent signaling, which cannot be overcome by the increment in the fraction of protein interactions leading to fluorescent signaling. Therefore it is beneficial to use the K12 wild type rather than K12ÎCpxR.
Visualizing Cpx activation with mVenus
We aim to improve the response time of our visualization system. As stated, our model shows that this can be done by using a faster maturing fluorescent protein. During our "Fluorescent Protein" project we tested a number of fluorescent proteins, of which (at the moment of implementation) mVenus showed one of the highest levels of brightness and a high percentage of fluorescence reconstitution. Furthermore mVenus is designed to have a fast and efficient maturation time [8], exactly what we need!
Like with eYFP, mVenus N-terminus and C-terminus were fused to the C-terminus of CpxR . These fusions were put under control of the L-arabinose inducible pBad/araC promoter and strong RBS BBa_B0034 . All parts were brought together via Golden Gate Assembly using restriction enzyme BsaI and T4 Ligase, and verified by Restriction Digest Analysis and subsequent sequencing and yielded the following translational unit (Figure D).
Figure D: CpxR-mVenus[1-157] and CpxR-mVenus[158-238] under control of the inducible pBad/araC promoter BBa_K2387029 .
The results show that usage of mVenus over eYFP as a reporter protein increases the produced fluorescent signal more than five times (Figure E). Unfortunately, the background signal also increases a lot, which means we lose a lot of specificity. We hypothesize that the association rate of mVenus is too fast, which means that many non-specific interaction become irreversible, leading to high fluorescent signals, even when no activator is present. This means that mVenus is not a suitable candidate to visualize antigen binding. Through literature research this hypothesis was confirmed [9].
During this project , more reporter proteins were tested. Due to time constraints, we were unable to model these systems. At this moment, we recommend using sfGFP as a reporter for antigen binding. We found out sfGFP is thermostable and has has efficient maturation at high temperatures, while still being one of the fastest and brightest reporters we tested.
Figure E: CpxR dimerization visualized using eYFP (left) and mVenus (right), with L-arabinose concentration = 0.2% and different activator concentrations over time
Conclusions
Three strategies were tested to find the optimal visualization method of antigen binding to use in Mantis (Figure 2). In the end, it was clear that fusion of eYFP-termini to CpxR (Figure 2A) and measuring its dimerization shows clear fluorescence within two hours after activation, even with low activator concentrations! This shows that we found a potential method for rapidly and specifically measuring antigens in blood.
During this âwet-labâ project, we constantly implemented results from the computer modeling to improve the experiments performed in the lab. For example, we used the computer model to help us find the right protein levels, as well as how strongly to activate the system. As it turns out, the âwet-labâ and modeling results are very similar. Check out how we integrated the lab work with modeling here !
Furthermore, we had time to integrate this visualization module with other wet-lab projects: check out here how we combined this project with the "Fluorescent Protein" experiment, and find out here how we combined with the "Signal Transduction" project to directly visualize antigen binding.
References
P. a DiGiuseppe and T. J. Silhavy, âSignal Detection and Target Gene Induction by the CpxRA Two-Component System,â J. Bacteriol., vol. 185, no. 8, pp. 2432â2440, 2003.
T. Kerppola, âBimolecular fluorescence complementation (BiFC) analysis as a probe of protein interactions in living cells,â Annu. Rev. Biophys., vol. 37, pp. 465â87, 2008.
T. L. Raivio and T. J. Silhavy, âThe sigmaE and Cpx regulatory pathways: Overlapping but distinct envelope stress responses,â Curr. Opin. Microbiol., vol. 2, no. 2, pp. 159â165, 1999.
C. D. Hu, Y. Chinenov, and T. K. Kerppola, âVisualization of interactions among bZIP and Rel family proteins in living cells using bimolecular fluorescence complementation,â Mol. Cell, vol. 9, no. 4, pp. 789â798, 2002.
2014 iGEM EFPL Results Page
Fleischer, R., Heermann, R., Jung, K., & Hunke, S. (2007). Purification, reconstitution, and characterization of the CpxRAP envelope stress system of Escherichia coli. Journal of Biological Chemistry, 282(12), 8583â8593.
Baba, Tomoya et al. âConstruction of Escherichia Coli K-12 in-Frame , Single-Gene Knockout Mutantsâ¯: The Keio Collection.â Molecular Systems Biology 4474 (2006): 1â11
Nagai, T., Ibata, K., Park, E. S., Kubota, M., & Mikoshiba, K. (2001). A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications. Nature Biotechnology, 20, 1585â1588.
Shyu, Y. J., Liu, H., Deng, X., & Hu, C.-D. (2006). Identification of new fluorescent protein fragments for bimolecular fluorescence complementation analysis under physiological conditions. BioTechniques, 40(1), 61â66.
iGEM Wageningen 2017. Send us an email or follow us on Facebook
