Team:Wageningen UR/Notebook/Journal/Quorum Wet
Project
Background
Wet-lab
Results
BioBricks
Model
Cpx Kinetics
Quorum Sensing
Cpx Integration
QS Integration
Human Practices
Integrated HP
Safety & Ethics
Collaborations
Future Perspectives
Engagement
Meetup
Application
Demonstrate
Device
Roll-out Plan
Notebook
Protocols
InterLab
Lab Journal
Achievements
Team
Members
Attributions
Acknowledgements
Project
Background
Wet-lab
Model
Human Practices
Application
Notebook
Team
Wet-lab
Model
Application
Human Practices
Notebook
Team
Parts
Results
Cpx Kinetics
Quorum Sensing
Cpx integration
QS integration
Integrated HP
Safety & Ethics
Collaborations
Future Perspectives
Engagement
Meetup
Demonstrate
Device
Rollout Plan
Protocols
InterLab
Journal
Achievements
Members
Attributions
Home
Notebook
Journals
QS Wet-lab
Quorum Sensing Wet-lab
By: Natalia Giner Laguarda
Here you can find Natalia's work on the wet-lab experiments of the Quorum Sensing system she made. Curious about her results? Read more about it here .
May
Week 1 (1st of May - 7th of May)
Getting started, proposal writing, office decoration.
Week 2 (8th of May - 14th of May)
More proposal writing. Becoming an expert in SnapGene by watching YouTube tutorials!
Week 3 (15th of May – 21st of May)
More Snapgene. Started 3A Assembly method.
Week 4 (22nd of May – 28th of May)
Designed primers, experimental setup and applied for BioBricks.
June
Week 5 (29th of May – 4th of June)
Modified experimental setup and made illustrations of it in Illustrator. Subsequently became an expert in Illustrator by watching YouTube tutorials!
Week 6 (5th of June – 11th of June)
Finalized construct design.
Week 7 (12th of June – 18th of June)
Finally ordered primers and BioBricks.
Week 8 (19th of June – 25th of June)
Prepared media, made E. coli electrocompetent cells and prepared some agar plates.
July
Week 9 (26th of June – 2nd of July)
RFP, CRFP, NRFP, lactose promoter + RBS were transformed from iGEM plates and PCR'ed + purified. Streaked plates, made minipreps and glycerol stocks.
Week 10 (3rd of July – 9th of July)
Ligation 1 (lactose promoter+RBS+ RFP) suceeds. Made glycerol stock of it and sent sample for sequence analysis. Ligation 2 and 3 failed. Moreover, CRFP lacks the ATG start codon and has an unexpected frameshift. NRFP lacks the stop codon.
Week 11 (10th of July – 16th of July)
Design of more primers to fix both CRFP and NRFP.
Week 12 (17th of July – 23rd of July)
PCR addition of ATG to CRFP (CRFP2). PCR addition of RBS to CRFP (CRFP3). PCR addition of stop codon to NRFP (NRFP2). PCR'ed COLE7 and T4 Endolysin from their sources.
Week 13 (24th of July – 30th of July)
Ligations 2 and 3 failed 3 times in a row. Succeeded in the 4th try. Transformation, colony PCR, Q5 PCR of the new set of genes needed for experiments 2 and 3:
T4 Holin  from a requested GeneBlock
LuxI
LuxR
RBS+CFP
RBS+GFP
Promoter LuxR-AHL
Designed primers to amplify backbones pSB4K5 and pSB3T5. Preparation of more agar plates.
August
Week 14 (31rd of July – 6th of August)
Ligation 2, ligation 3, ligation 5, ligation 6 succeeded. Ligation 4, ligation 7, ligation 8, ligation 11 and ligation 10 failed. Repeated ligation 4, 7, 8, 11 and 10. Failed again. Ligation 11 succeeded.
Week 15 (7th of August - 13th of August)
Holiday! - Out of office.
Week 16 (14th of August - 20th of August)
Holiday! - Out of office. Week 17 (21st of August - 27th of August)
Made liquid cultures, which were subsequently miniprepped and colony PCR'ed. Ligations 4, 7, 8, 10 and 11 were sent for sequencing. None of them worked as all of them were empty, starting to doubt the 3A assembly method. Repeated the whole process, but to no avail.  Moved to Golden Gate assembly: everything suddenly worked on my first try. Made a change in the Quorum Sensing approach, ligation 7 and 9 are scratched from now on onwards, I'm going to take a new approach:
Fluorophore-only ligation (GFP) ---> Acts as negative control
QS ligation (GFP + LuxR + aiiA under pLac) ---> Acts as negative control
Reporter QS (GFP + Lux I + LuxR)
QS ligation 2 (GFP + Lux I + LuxR + aiiA under pLac)
Week 18 (28th of August - 3rd of September)
New set of PCRs in order to add Golden Gate overhangs to everything:
pLac vector
NRFP2 for ligation 2
NRFP2 for ligation 4
CRFP3
pBAD vector
COLE7
T4 Endolysin
T4 holin for ligation 7
T4 holin for ligation 8
Reporter QS
aiiA
Design GeneBlocks in order to construct the big QS ligation and as soon as they arrive: start ligation.
Week 19 (4th of September - 10th of September)
All the PCR products are ready and purified, but the backbones refuse to be amplified. Stamatios gave me a pair of primers annealing in the resistance of the backbone. Therefore, I moved on to a new Golden Gate assembly method in which each ligation was going to have 1 extra part (as a result of the amplification of each backbone in two parts). Both parts of both vectors (pBAD and pLac) were succesfully amplified.  I started the Quorum Sensing ligation and picked colonies until I found a good one. QS ligation (without LuxI) was never decided to be done, but LuxI contains around 100 base pairs and I never realized this while assessing the colonies (colony PCR, restriction analysis, spreading plates) that luxI was not there until I sent my 5 samples for sequencing. The reason for sending 5 samples is that QS ligation is extremelly long and hence 5 primers were needed to get the whole sequence. Everything was perfect but luxI was not there, which saddened me. Then I was happy, because this unexpectedly became the best negative control ever.  I started the rest of ligations as well:
Ligation 2 (amino)
Ligation 4 (carboxi and amino)
Ligation  5 (cole7)
Ligation 7 (endolysin + holin)
Ligation 8 (holin)
Restriction analysis, colony pcr and Minipreps of :
Ligation 2 (amino)
Ligation 4 (carboxi and amino)
Ligation 5 (cole7)
Ligation 7 (endolysin + holin)
Ligation 8 (holin)
All the sequences were fine except ligation 7,  whose colonies were either empty or contained a mutation.
Week 19 (11th of September - 17th of September)
Only reporter ligation was grown from a glycerol stock and made the QS reporter as well. Repeated Ligation 7 and picked all the possible colonies, miniprepped and analyzed them through restriction digestion and = of course I used colony PCR as well. Finally i found a colony which achieved to insert the gene. But of course, it has a mutation in the gene. It was not possible to pick any perfect colony. The conclusion was that the mechanism was that lytic that there was no possibility that bacteria was contianing the perfect double gene mechanism as it was to lytic to the cells.
Week 20 (18th of September - 24th of September)
Transformed the QS ligation and amplified pSB4K5 with the Golden Gate overhangs. Started experiment 1 (Bimolecular fluorescent recombination of RFP), it completely failed. This experiment was run with the lysis protocol of JosE. Miniprepped 8 colonies of Ligation 7 and analyzed them trough restriction digestion. Sent one for sequencing as well, but it was once again empty. Based on the fact that this super lytic ligation was done millions of times and all the possible colonies were tested, it was concluded that the system was lytic to such a degree that it would never be able to be expressed by a single bacterium. Lytic mechanisms… still working, i have them all, for Ligation 7 I have one successful but it has one mutation, which should not matter. I ran the experiment of the lytic mechanisms in pSB1C3 by inducing with L-arabinose. Results look good so far:
COLE 7 works perfect
T4 holin works perferct
T4 endolysin gives a lot of noise
Ligation 7 works, but not 100%
I decided to check the sequences again and i saw that the endolysin gene from the Repository was includes 10 nucleotides extra in front of the ATG. This means the expression of the gene was going to comprised. So I decided to fix this gene (which meant repeating L6 and L7).  QS ligation repeated, colonies picked, restriction analysis, liquid culture, miniprep and send for sequencing. I got the QS mechanism with the aiiA in the same backbone, finally its time to test it.  Designed primers in order to amplify all my lytic mechanisms in pSB4K5. Ligations, transformed and streaked plates, picked colonies, made liquid culture, made glycerol stock, performed minipreps and sent for sequencing. Succeeded: COLE 7, T4 Endolysin Failed: l7 and T4 Holin  Repeated Ligation 7 and 8 (T4 holin) again, but it only worked for Ligation 8
October
Week 21 (25th of September - 1st of October)
QS experiment done. Bimolecular fluorescent recombination experiment was repeated as well, = but with two different approaches: Sonication rather than lysis protocol Biological lysis rather than lysis protocol (for this I cotransformed all the strains with my COLE7 system), checked on gel.
Week 22 (2nd of October - 8th of October)
Decided to tackle 2 new approaches:
Quenching GFP by REACH2
Quenching GFP by the M2 fragment from influenza
Designed primers to do all the new PCRs, in order to make all the required ligations. Moreover, regarding the QS mechanism, the leackiness should be fixed and the system should be made inducible by: Moving the QS mechanism into pSB4K5 Expressing the aiiA mechanism under pBAD in pSB1C3 14 PCRs done so far, also performed oligo annealing in order to create m2 fragment.
Week 23 (9th of October - 15th of October)
Started all the 11 ligations in parallel, from transformation to cPCR and sequencing. Repeated failed sequences (GFP cleavable quencher, TEV protease (3 times), GFP quenched). I got: QS in 4K5, L6 IN 4K5, aiiA in 2C3, l6 fixed, l7 fixed. And repeated oligo annealing with a new protocol as well. Made the Zymo Mix&Go competent cells to see whether this improves my quality of life.
Week 24 (16th of October - 22th of October)
Still couldn't get the Quenchd GFP assembly to work... Repeated it once again with new electrocompetent cells, transfomed in duplo and with different concentrations. Again empty plate, but after some more time in the incubator, something grew. I got the GFP cleavable quencher and GFP M2 absorbed cleavable, but one was mutated and the other one empty. Provided that this was the 4th time i was trying those and they still didn't work, I decided to move forward with the project, as time becomes pressing.  Made liquid cultures for:
QS experiment (with inducible system)
8 lytic mechanisms in both backbones
Quenching experiment
Incubated the 3 of them, M9 medium was contaminated when i saw the results. So repeat again.  1 and 2 worked, 3 nothing, so i worked with Despoina (Prarthana's student) and we came up with a new protocol: Sonication of samples with a new lysis buffer and induction of the cut of the linker with a purified TEV protease. In parallel I ran the quenching experiment again. Data still not analyzed but so far, I've already drunk ~35 RedBull during this month.
iGEM Wageningen 2017. Send us an email or follow us on Facebook
