<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Model_QStest skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Model/QStest</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="Main-Menu"><DIV class="col-lg-10 MainMenu-Center-Column MenuCenter"><DIV class="Project MenuButtonColumn"><DIV class="Project-Top TopMenuButtonColumn"><UL><LI class="project-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Project" class="nav-item" id="project">Project</A></LI></UL></DIV></DIV><DIV class="Background MenuButtonColumn"><DIV class="Background-Top TopMenuButtonColumn"><UL><LI class="background-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Background" class="nav-item active" id="background">Background</A></LI></UL></DIV></DIV><DIV class="Wet-Lab MenuButtonColumn"><DIV class="Wet-Lab-Top TopMenuButtonColumn"><UL><LI class="wet-lab-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Wet-lab" class="nav-item active" id="wet-lab">Wet-lab</A></LI></UL></DIV><DIV class="Wet-Lab-Bottom SubMenuButtonColumn"><UL id="wet-lab-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results" class="results-sub-nav-item sub-nav-item">Results</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Parts" class="parts-sub-nav-item sub-nav-item">BioBricks</A></LI></UL></DIV></DIV><DIV class="Model MenuButtonColumn"><DIV class="Model-Top TopMenuButtonColumn"><UL><LI class="model-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Model" class="model nav-item" id="model">Model</A></LI></UL></DIV><DIV class="Model-Bottom SubMenuButtonColumn"><UL id="model-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics" class="kinetics-sub-nav-item sub-nav-item">Cpx Kinetics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/QS" class="qs-sub-nav-item sub-nav-item">Quorum Sensing</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx" class="integrationcpx-sub-nav-item sub-nav-item">Cpx Integration</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS" class="integrationQS-sub-nav-item sub-nav-item">QS Integration</A></LI></UL></DIV></DIV><DIV class="Human-Practices MenuButtonColumn"><DIV class="Human-Practices-Top TopMenuButtonColumn"><UL><LI class="human-practices-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Human_Practices" class="nav-item" id="human-practices">Human Practices</A></LI></UL></DIV><DIV class="Human-Practices-Bottom SubMenuButtonColumn"><UL id="human-practices-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated" class="integrated-sub-nav-item sub-nav-item ">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Silver" class="safety-sub-nav-item sub-nav-item">Safety &amp; Ethics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Collaborations" class="collaboration-sub-nav-item sub-nav-item">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Future" class="future-sub-nav-item sub-nav-item">Future Perspectives</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Engagement" class="engagement-sub-nav-item sub-nav-item">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Meetup" class="meetup-sub-nav-item sub-nav-item">Meetup</A></LI></UL></DIV></DIV><DIV class="Application MenuButtonColumn"><DIV class="Application-Top TopMenuButtonColumn"><UL><LI class="application-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Applied_Design" class="nav-item" id="application">Application</A></LI></UL></DIV><DIV class="Application-Bottom SubMenuButtonColumn"><UL id="application-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Demonstrate" class="demo-sub-nav-item sub-nav-item">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Hardware" class="device-sub-nav-item sub-nav-item">Device</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Application/Rollout" class="rollout-sub-nav-item sub-nav-item">Roll-out Plan</A></LI></UL></DIV></DIV><DIV class="Notebook MenuButtonColumn"><DIV class="Notebook-Top TopMenuButtonColumn"><UL><LI class="notebook-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook" class="notebook nav-item" id="notebook">Notebook</A></LI></UL></DIV><DIV class="Notebook-Bottom SubMenuButtonColumn"><UL id="notebook-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols" class="protocols-sub-nav-item sub-nav-item">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/InterLab" class="interlab-sub-nav-item sub-nav-item">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Journal" class="journal-sub-nav-item sub-nav-item">Lab Journal</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Achievements" class="achievements-sub-nav-item sub-nav-item">Achievements</A></LI></UL></DIV></DIV><DIV class="Team MenuButtonColumn"><DIV class="Team-Top TopMenuButtonColumn"><UL><LI class="team-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Team" class="team nav-item" id="team">Team</A></LI></UL></DIV><DIV class="Team-Bottom SubMenuButtonColumn"><UL id="team-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Team/Members" class="members-sub-nav-item sub-nav-item">Members</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Attributions" class="attributions-sub-nav-item sub-nav-item">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Acknowledgements" class="Acknowledgements-sub-nav-item sub-nav-item">Acknowledgements</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="MobileMenu"><UL id="MainMobileMenu"><LI><BUTTON class="project-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Project'" type="button">Project</BUTTON></LI><LI><BUTTON class="background-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Background'" type="button">Background</BUTTON></LI><LI><BUTTON class="wet-lab-li-mo">Wet-lab</BUTTON></LI><LI><BUTTON class="model-li-mo">Model</BUTTON></LI><LI><BUTTON class="human-practices-li-mo">Human Practices</BUTTON></LI><LI><BUTTON class="application-li-mo">Application</BUTTON></LI><LI><BUTTON class="notebook-li-mo">Notebook</BUTTON></LI><LI><BUTTON class="team-li-mo">Team</BUTTON></LI></UL><UL id="MobileSubMenu"><LI class="wet-lab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Wet-lab'" type="">Wet-lab</BUTTON></LI><LI class="model-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model'">Model</BUTTON></LI><LI class="application-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Applied_Design'">Application</BUTTON></LI><LI class="human-practices-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Human_Practices'">Human Practices</BUTTON></LI><LI class="notebook-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook'">Notebook</BUTTON></LI><LI class="team-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team'">Team</BUTTON></LI><LI class="parts-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Parts'" type="button">Parts</BUTTON></LI><LI class="results-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Results'" type="button">Results</BUTTON></LI><LI class="kinetics-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics'" type="button">Cpx Kinetics</BUTTON></LI><LI class="qs-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/QS'" type="button">Quorum Sensing</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx'" type="button">Cpx integration</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS'" type="button">QS integration</BUTTON></LI><LI class="integrated-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated'" type="button">Integrated HP</BUTTON></LI><LI class="safety-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Silver'" type="button">Safety &amp; Ethics</BUTTON></LI><LI class="collaboration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Collaborations'" type="button">Collaborations</BUTTON></LI><LI class="future-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Future'" type="button">Future Perspectives</BUTTON></LI><LI class="engagement-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Engagement'" type="button">Engagement</BUTTON></LI><LI class="meetup-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Meetup'" type="button">Meetup</BUTTON></LI><LI class="demo-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Demonstrate'" type="button">Demonstrate</BUTTON></LI><LI class="device-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Hardware'" type="button">Device</BUTTON></LI><LI class="rollout-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Application/Rollout'" type="button">Rollout Plan</BUTTON></LI><LI class="protocols-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols'" type="button">Protocols</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/InterLab'" type="button">InterLab</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Journal'" type="button">Journal</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Achievements'" type="button">Achievements</BUTTON></LI><LI class="members-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team/Members'" type="button">Members</BUTTON></LI><LI class="attributions-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Attributions'" type="button">Attributions</BUTTON></LI></UL></DIV><DIV class="container-fluid OurContent"><DIV class="row"><DIV class="col-lg-8"><DIV class="Main-Border"><DIV id="SideMenu-Wrapper"><DIV id="SideMenu-Anchor"><DIV id="SideMenu"><DIV class="SideMenu-Header"><H4>Results<UL class="sidebar-nav"><LI><A href="#item1">Disease Antigens</A></LI><LI><A href="#item1">Affinity Bodies</A></LI><LI><A href="#item2">Cpx system</A></LI><LI><A href="#item3">ToxR dimerization</A></LI><LI><A href="#item3">BiFC system</A></LI><LI><A href="#item3">Quorum Sensing</A></LI></UL><DIV class="Main-Center-Content-Column Rollout"><DIV id="breadcrumb-wrapper"><UL class="breadcrumb"><LI><A href="https://2017.igem.org/Team:Wageningen_UR">Home</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model">Model</A></LI><LI>Quorum Sensing Model</LI></UL></DIV></DIV></H4><H1>Quorum sensing and cell lysis</H1></DIV><DIV class="Textbox Results-Desc"><P>
                                We developed a mathematical model of the quorum sensing and cell lysis system. The goals of the mathematical model are the following. To characterize the theoretical properties of the system. To identify which kinetic parameters dictate the behavior of the system. To determine which parameter, that can be changed in the lab, can be varied to obtain a system with the correct behavior. To achieve this goal, the <I>Simple model</I> was first developed. The <I>Simple model</I> was used to sample the global parameter space. The second model that we used is the <I>Spatial Model</I>. This model was used to test hypothesis and to guide wet-lab experiments.
                                With the modeling of the quorum sensing and cell lysis system we had the following goals.</P><UL class="bullet-points"><LI>Characterize system behavior</LI><LI>Sample global parameter space</LI><LI>Identify parameters with high sensitivity</LI><LI>Find lab-tunable parameters</LI><LI>Test hypothesis</LI></UL><DIV class="Summary-Box"><DIV class="Summary-Box-Title"><H4>Summary</H4></DIV><P>
                        Our mathematical model shows that the QS system can spontaneously activate, function properly, or be insensitive to AHL. Changing the parameters of LuxR degradation and AHL cellular degradation allow a system to transition from one behavior to the other. In the diffusion model of our QS and cell lysis system, we can change a spontaneously activating system into a system with proper signaling behavior by either increasing LuxR degradation, or by adding the enzyme aiiA to the genetic circuit.

                    </P></DIV><P>
                    For the system to function properly, it needs the following behaviors.
                    <B>Bistability:</B> Cells must be able to switch from an OFF-state to an ON-state. This requires that the system of ODE equations has more than one equilibrium state solutions. The bistability of switches based on the lux operon have been explored in the literature. A motif with two feedback loops has been shown to have bistability[1,2]. However, it means unclear if the native lux box has the ability to switch from a clear OFF-state to an ON-state[3].
                    <B>Cell signaling:</B> Cells in the ON-state should be able to signal cells in the OFF-state to also transition to the ON-state. This is a core property of the quorum sensing system.[4]
                    <B>Delayed cell lysis:</B> Cells in the ON-state should lyse to release their split fluorescent protein. But they should do so only after signaling cells still in the OFF-state.
                </P></DIV><SECTION class="ODEmodel"><DIV class="Title"><H2>Simple Model</H2></DIV><DIV class="Textbox"><P>
                        The main goal of the simple model is to allow for a sampling of the parameter space of the kinetic rate constants of the model. To effectively do this, the biological picture has to be translated to mathematical equations. Furthermore, this model is more manageable and more straightforward to interpret when the number of kinetic parameters are low. To achieve this, the following assumptions have been made.
                    </P></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SimpleAssumptions" aria-expanded="true" aria-controls="SimpleAssumptions"><H4 class="panel-title"><DIV class="col-md-11">Assumptions</DIV></H4></A></DIV><DIV id="SimpleAssumptions" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><H5>System</H5><P>
                                    The system consists out of three compartments; the external medium, antigen-activated bacterial cells, and signal-receiving bacterial cells. The equations of both cells are identical, except that the activator cells receive an external signal intended to put the cells in the ON-state halfway during the simulation. This simulates the addition of antigen-containing blood to the bacteria and detecting of this low concentration of antigen by a minority of the cells. The total size of the system is 1 000 000 volume units. The ratio of external volume to cells is 1 cell volume unit in 1000 total volume units. This approximates a cell density of 1.0 OD<SUB>600</SUB>. Therefore, the total cell volume is 1000 volume units. As the volume unit is defined as the volume of a single cell, there are a total 1000 cells in this system. Out of these cells, 900 cells are receiver cells, 100 are activator cells. The two cellular compartments in the model actually represent only a single cell. When cellular compartments exchange species with the external medium compartment, the total amount of exchange is calculated by using the total for each cell type. This means that all receiver cells are identical copies of each other. The same is true for the and activator cells.
                                </P><DIV class="figure-fullwidth"><DIV class="figure-center-caption"><B>Figure 1: </B>The ODE model has three different compartments, the external medium, activator cells, and receiver cells. Diffusion is assumed to be instantaneous inside each compartment. Diffusion across the cell membrane is computed using a linear approximation, as indicated by the 'flux' arrows. ODE equations describe the rate of change of all species inside the three compartments.
                                    </DIV></DIV><P>
                                    Each cell has concentration values for the following species: AHL, LuxI, LuxR, [AHL-LuxR] complex, split fluorescent protein and lysis protein. In the external medium, there are concentration values for AHL, C-terminal split Fluorescent protein, N-terminal split fluorescent protein, and the fused and active fluorescent protein. All compartments are assumed to be well-mixed, having a homogeneous concentration. All cells are identical in size and their size is constant throughout the simulation. There is assumed to be no cell growth and therefore no dilution term. There is no spatial component to any reaction in this model.
                                </P><H5>Kinetics</H5><P>
                                    The levels of mRNA of all genes expressed is assumed to be in a quasi steady state. The basis for this assumption is that translation is much slower than transcription. Therefore, there are no concentration values for any mRNA species in the model. Similarly, transcription factors binding to DNA is also assumed to quasi steady state. The assumption here is that binding and unbinding of a transcription factor to a promoter occurs at timescales much faster than translation. The third assumption is that the dimerization of [AHL-LuxR] is also in a quasi steady state. While there has been some debate on whether [AHL-LuxR] complex formation or [AHL-LuxR] dimer formation occurs faster. We decided to go with the assumption that dimer formation is faster and in quasi steady state. Therefore, there is no concentration value for the [AHL-LuxR] dimer in the model. To summarize, transcription factor-DNA complexes, mRNA, and [AHL-LuxR] dimer concentrations are implicitly modeled. All promoters are assumed to have a single transcription binding site and the transcription factor is assumed to form a dimer only, giving a Hill coefficient of 2.
                                </P><P>
                                    The lysis mechanism is not implicitly modeled. Lysis is assumed to occur instantaneously the moment the lysis protein value exceeds a value of 1.0. Before lysis protein reaches this concentration, the lysis protein is inert. Diffusion of AHL across the cell membrane is modeled by a linear diffusion equation depending on the concentration difference inside and outside of the cell, and a diffusion constant for diffusion of AHL across the cell membrane. This diffusion constant is assumed to be the same whether AHL diffuses into out ouf of the cell. As stated earlier, the compartments are considered to be well-mixed, which equates to assuming that both the diffusion of AHL inside each compartment is instantaneous. Similarly, the split fluorescent proteins released upon cell lysis is assumed to be instantaneous.
                                </P><P>
                                    Except for the fluorescent proteins in the external medium, all species have degradation rate. The split and fused fluorescent proteins are considered to be sufficiently stable so that degradation outside the cell plays no role on simulation length timescales. There are no values of LuxR, LuxI and [AHL-luxR] inside the external medium. Upon lysis of the cell, only AHl and the split fluorescent proteins are released into the external medium. AHL bond to LuxR and LuxI inside lysing cells simply disappear from the model.
                                     Some of these assumptions may result in a model that fails to incorporate important dynamics. To accurately sample the 26-dimensional parameter space, the computational time of a single simulation can not be longer than several seconds. The spatial model, described in the second part of this page, alleviates some of the assumptions that are made here.
                                </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SimpleEquations" aria-expanded="true" aria-controls="SimpleEquations"><H4 class="panel-title"><DIV class="col-xs-11">Equations</DIV></H4></A></DIV><DIV id="SimpleEquations" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary equation"><H5>Ordinary differential equations</H5><P>

                                    ${\mathrm{d}AHL_{cell} \over \mathrm{d}t} = \alpha_{1}LuxI + D_{AHL}(AHL_{ext}-AHL_{cell})+k_{-1}RA-k_1 LuxR \cdot AHL -\beta_{1}AHL_{cell}$</P><P>
                                    ${\mathrm{d}AHL_{ext} \over \mathrm{d}t} = V_{ratio} \cdot D_{AHL} (AHL_{ext}-AHL_{cell}) - \beta_{2}AHL_{cell}$
                                </P><P>
                                    ${\mathrm{d}RA \over \mathrm{d}t} = k_{-1}LuxR \cdot AHL - \beta_{3}RA^{2} - \beta_{4}RA$
                                </P><P>
                                    ${\mathrm{d}LuxI \over \mathrm{d}t} = \alpha_2 + \alpha_3 {{LuxR \cdot RA^2} \over {km_1 + RA^2}} + \alpha_4 + S \cdot \alpha_5 - \beta_5 LuxI$
                                </P><P> Positive regulation of LuxR:
                                     ${\mathrm{d}LuxR \over \mathrm{d}t} = \alpha_6 + \alpha_7 {{LuxR \cdot RA^2} \over {km_1 + RA^2}} + \alpha_8 + S \cdot \alpha_9 - \beta_6 LuxR$
                                </P><P> Negative regulation of LuxR:
                                     ${\mathrm{d}LuxR \over \mathrm{d}t} = \alpha_6 + \alpha_7 {{LuxR \cdot km_1} \over {km_1 + RA^2}} + \alpha_8 + S \cdot \alpha_9 - \beta_6 LuxR$

                                </P><P>
                                    ${\mathrm{d}Lysis \over \mathrm{d}t} = \alpha_{10} + \alpha_{11} {{LuxR \cdot RA^2} \over {km_2 + RA^2}} - \beta_7 Lysis$
                                </P><P>
                                    ${\mathrm{d}SplitFP_{cell} \over \mathrm{d}t} = \alpha_{12} - \beta_8 SplitFP$
                                </P><P>
                                    ${\mathrm{d}SplitFP_{N} \over \mathrm{d}t} = -k_3 SplitFP_{N} \cdot dSplitFP_{C}$
                                </P><P>
                                    ${\mathrm{d}SplitFP_{C} \over \mathrm{d}t} = -k_3 SplitFP_{N} \cdot dSplitFP_{C}$
                                </P><P>
                                    ${\mathrm{d}ActiveFP \over \mathrm{d}t} = k_3 SplitFP_{N} \cdot dSplitFP_{C}$
                                </P><H5>Species</H5><P>$AHL_{cell}$ Signaling molecule AHL inside the cell</P><P>$AHL_{ext}$ Signaling molecule AHL inside the external medium</P><P>$LuxR$ Free inactive LuxR protein</P><P>$RA$ The LuxR-AHL complex</P><P>$RA_2$ The dimer of the LuxR-AHL complex and active transcription factor</P><P>$LuxI$ The AHL-producing enzyme LuxI</P><P>$LuxR$ The AHL-producing enzyme LuxI</P><P>$Lysis$ The lysis protein responsible for the lytic mechanism</P><P>$SplitFP_{cell}$ The split fluorescent protein in the cell</P><P>$SplitFP_{N}$ The N-terminal part of the split fluorescent protein in the external medium </P><P>$SplitFP_{C}$ The C-terminal part of the split fluorescent protein in the external medium </P><P>$ActiveFP$ The active full form of the split fluorescent protein in the external medium</P><H5>Parameters</H5><P>$\alpha_1$ AHL production rate of LuxI </P><P>$\alpha_2$ Leaky production rate of LuxI from pLuxA </P><P>$\alpha_3$ Maximum production rate of LuxI from pLuxA</P><P>$\alpha_4$ Leaky production rate of LuxI from pLuxB</P><P>$\alpha_5$ Maximum production rate of LuxI from pLuxB</P><P>$\alpha_6$ Leaky production rate of LuxR from pLuxB</P><P>$\alpha_7$ Maximum production rate of LuxR from pLuxA</P><P>$\alpha_8$ Leaky production rate of LuxR from pLuxB</P><P>$\alpha_9$ Maximum production rate of LuxR from pLuxB</P><P>$\alpha_{10}$ Leaky production rate of Lysis from pLuxB</P><P>$\alpha_{11}$ Maximum production rate of Lysis from pLuxB</P><P>$\alpha_{12}$ Constitutive production of $SplitFP_{cell}$</P><P>$k_1$ formation rate of [LuxR-AHL]</P><P>$k_{-1}$ dissociation rate of [LuxR-AHL]</P><P>$k_{-2}$ dissociation rate of RA</P><P>$k_{3}$ formation &amp; maturation rate of the full fluorescent protein</P><P>$km_{1}$ Michaelis-Menten constant of pLuxA for $RA$</P><P>$km_{2}$ Michaelis-Menten constant of pLuxB for $RA$</P><P>$D_{AHL}$ diffusion rate of AHL across the cellular membrane</P><P>$\beta_1$ degradation rate of $AHL_{cell}$</P><P>$\beta_2$ degradation rate of $AHL_{ext}$</P><P>$\beta_3$ degradation rate of the $RA$ dimer</P><P>$\beta_4$ degradation rate of $RA$ complex </P><P>$\beta_5$ degradation rate of $LuxI$</P><P>$\beta_6$ degradation rate of $LuxR$</P><P>$\beta_7$ degradation rate of $Lysis$</P><P>$\beta_8$ degradation rate of $SplitFP_{cell}$</P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SimpleMethods" aria-expanded="true" aria-controls="SimpleMethods"><H4 class="panel-title"><DIV class="col-xs-11">Methods</DIV></H4></A></DIV><DIV id="SimpleMethods" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                    The system of ODE equations was solved using the scipy.integrate.odeint module available in Python. To sample the global parameter space, 1 000 000 parameter sets were generated using the lhs() function from pyDOE. These parameter sets were simulated for a total of 240 time units, where between t=120 and t=180, the external signal S was set from 0 to 1. A value for 1.0 was chosen to represent enough lysis protein for cell lysis to occur. Every 0.01 timesteps, the value of lysis protein in each cell was checked. In the case that a cell has a lysis protein value above 1.0, the cell would be removed from the simulation and the AHL and split fluorescent protein contents would be released into the external medium. All released fluorescent protein was always 50-50 of both C- and N-terminal versions. This was done to avoid having to introduce random numbers.
                                </P><H5>Scoring function</H5><P>To properly score how well the parameter set resulted in the desired behavior, the following scoring scheme was used. The area under the curve of the fluorescence timeseries was approximated using the numpy.trapezoid() function. As an ideal parameter set generates no fluorescence before t=120 and very quickly generates a lot of fluorescence after t=120, the area under the curve before t=120 should be minimal, while after t=120, the area should be as large as possible. The area under the curve represents both how much, as well as how quickly, fluorescence is being generated. Secondly, in absence of an external input, so in case S = 0 during the entire simulation, there should be no fluorescence generated. Fluorescence in absence of an antigen would amount to a false positive. Therefore, each parameter set is simulated twice. Once with no external signal, and once with an external signal between t=120 and T=180. The final score of a parameter set is calculated by combining both simulation runs. As the objective function has to be minimalized. This leads to the following equations for the scoring function.
                                </P><DIV class="equation"><P>$Score_{simulation} = \displaystyle\int_{t=0}^{t=120} F \mathrm{d}t + {1 \over {\displaystyle\int_{t=120}^{t=240} F \mathrm{d}t + 0.01}}$
                                    </P><P>
                                        $Score_{set} = {900 \over { |Score_{noAntigen} - Score_{Antigen}|+ 0.9} } + 0.1 \cdot Score_{Antigen}$
                                    </P></DIV></DIV></DIV></DIV></DIV><H3>Results</H3><H5>Global parameter sampling</H5><P>A total of 1 million parameter sets were sampled for the system with positively regulated LuxR.
                </P><TABLE class="table table-bordered Results-Table"><THEAD><TR><TH colspan="4">Table 1: Results of simulating 1 million parameter sets using latin hypercube sampling. LuxR was positively regulated by the dimer of the LuxR-AHL complex</TH></TR></THEAD><TBODY><TR><TH>Behavior</TH><TH># parameter sets</TH><TH>Percentage</TH></TR><TR><TD>No lysis</TD><TD>778326</TD><TD>77.8%</TD></TR><TR><TD>Activator lysis</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Spontaneous lysis</TD><TD>10986</TD><TD>1.1%</TD></TR><TR><TD>Induced full lysis</TD><TD>21</TD><TD>0.002%</TD></TR></TBODY></TABLE><TABLE class="table table-bordered Results-Table"><THEAD><TR><TH colspan="4">Table 2: Results of simulating 1 million parameter sets using latin hypercube sampling. LuxR was negatively regulated by the dimer of the LuxR-AHL complex</TH></TR></THEAD><TBODY><TR><TH>Behavior</TH><TH># parameter sets</TH><TH>Percentage</TH></TR><TR><TD>No lysis</TD><TD>813045</TD><TD>81.3%</TD></TR><TR><TD>Activator lysis</TD><TD>146105</TD><TD>14.6%</TD></TR><TR><TD>Spontaneous lysis</TD><TD>40842</TD><TD>4.1%</TD></TR><TR><TD>Induced full lysis</TD><TD>8</TD><TD>0.001%</TD></TR></TBODY></TABLE><P>The variation in parameters for each behavior was visualized by generating a histogram showing how many parameter sets fell within a certain parameter range for each behavior.</P><DIV class="figure-center"><DIV class="figure-center-imagebox"><DIV id="PosLuxR-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 2:</B> Comparing parameter sets that lyse and those that do not lyse, low values for the degradation rates of the [LuxR-AHL] dimer, the [LuxR-AHL] complex, and LuxR are overrepresented in sets that lyse. Low degradation rates of species containing LuxR are associated with systems are more likely to lyse. The same is true for the LuxI protein, responsible for AHL production.</DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 2:</B> Comparing parameter sets that lyse and those that do not lyse, low values for the degradation rates of the [LuxR-AHL] dimer, the [LuxR-AHL] complex, and LuxR are overrepresented in sets that lyse. Low degradation rates of species containing LuxR are associated with systems are more likely to lyse. The same is true for the LuxI protein, responsible for AHL production.
                    </DIV></DIV><P>A similar trend is seen in the data of the negative regulation of LuxR.</P><DIV class="figure-center"><DIV class="figure-center-imagebox"><DIV id="NegLuxR-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 3:</B> Comparing parameter sets that lyse and those that do not lyse, low values for the degradation rates of the [LuxR-AHL] dimer, the [LuxR-AHL] complex, and LuxR are overrepresented in sets that lyse. Low degradation rates of species containing LuxR are associated with systems are more likely to lyse. The LuxI rate of parameter sets that lyse are uniformly distributed, suggesting the LuxI rate has no significant effect.</DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 3:</B> Comparing parameter sets that lyse and those that do not lyse, low values for the degradation rates of the [LuxR-AHL] dimer, the [LuxR-AHL] complex, and LuxR are overrepresented in sets that lyse. Low degradation rates of species containing LuxR are associated with systems are more likely to lyse. The LuxI rate of parameter sets that lyse are uniformly distributed, suggesting the LuxI rate has no significant effect.
                    </DIV></DIV><P>As the number of parameter sets that quorum sense and lyse in response to antigen were very low in both cases, it was impossible to say something about the difference between the parameters with the desired behavior and those that lysed spontaneosly. Therefore, the local parameter space around these parameter sets was explored. By scoring parameter sets in the local vicinity of the parameter sets discovered by the global search, parameter sets with the desired behavior could be compared to those that lysed spontaneously. While the two types of parameter sets were still quite similar, a different behavior could be seen for the extracellular degradation of AHL and the degradation rate of LuxR (figure 4 and 5).</P><DIV class="col-lg-6 inline-img-left"><DIV class="figure-fullwidth"><DIV class="figure-center-imagebox"><DIV id="PosLuxRExtAHLdeg-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 4:</B> PosReg ExtAHLdeg</DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 4:</B> PosReg ExtAHLdeg
                        </DIV></DIV></DIV><DIV class="col-lg-6 inline-img-right"><DIV class="figure-fullwidth"><DIV class="figure-center-imagebox"><DIV id="PosLuxRLuxRdeg-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 4:</B> PosReg LuxRdeg</DIV></DIV><DIV class="figure-center-caption"><B>Figure 5:</B> PosReg LuxRdeg</DIV></DIV></DIV></DIV><DIV class="col-lg-6 inline-img-left"><DIV class="figure-fullwidth"><DIV class="figure-center-imagebox"><DIV id="NegLuxRExtAHLdeg-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 6:</B> NegReg ExtAHLdeg</DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 6:</B> NegReg ExtAHLdeg
                        </DIV></DIV></DIV><DIV class="col-lg-6 inline-img-right"><DIV class="figure-fullwidth"><DIV class="figure-center-imagebox"><DIV id="NegLuxRLuxRdeg-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure 7:</B> NegReg LuxRdeg</DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure 7:</B> NegReg LuxRdeg
                        </DIV></DIV></DIV></SECTION><SECTION id="Spatialmodel"><DIV class="Textbox Results-Title"><H2>Spatial Model</H2></DIV><P>As the assumption that AHL diffuses instantaneous breaks down when the system becomes larger, or when the number of activator cells becomes lower, we developed a spatial model that incorporates both diffusion of AHL, but also of the mixing of the split fluorescent proteins.
				</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SpatialDetails" aria-expanded="true" aria-controls="SpatialDetails"><H4 class="panel-title"><DIV class="col-xs-11">Assumptions</DIV></H4></A></DIV><DIV id="SpatialDetails" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
									This spatial model is comprised out of a 2d grid where each square is the size of a single bacterial cell. Squares are either external medium, or completely occupied by a single cell. Diffusion occurs through the face that each square has with it's top, left, right, and bottom neighbor.
				
                                    </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SpatialEquations" aria-expanded="true" aria-controls="SpatialEquations"><H4 class="panel-title"><DIV class="col-xs-11">Equations</DIV></H4></A></DIV><DIV id="SpatialEquations" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                    The finite volume method was implemented in the following manner. For squares with four neighbors, the sum over all of the concentration values of the four adjacent squares mimus four times the concentration value in that square.
                                    </P><P>
                                    $U_{i,j} = p_{i-1,j} + p_{i,j-1} + p_{i+1,j} + p_{i,j+1} - 4 \cdot p_{i,j}$
                                    
                                    $U$ is the scalar value of the vector field describing the flux in square with coordinates i and j
                                    
                                    $p_{i,j}$ is the concentration in square with coordinates i and j
                                    </P><P>										
									$U_{i,j}$ is positive when the concentration in a square is increasing. In that case, more flux enters the square than leaves the square.
									As there are four faces through which flux can flow, the flux out of a square is proportional to the concentration inside that square. As there are four faces through which flux can leave, this term is present four times.
                                    Flux can also flow into a square from either of the four neighbors. These represent the four positive terms. Together, this provides a scalar that describes the d
                                    </P><P>
                                    The change in concentration is calculated by multipliying U with the diffusion constant.
                                    </P><P>
                                    $f = D \cdot U \Delta t$
                                    
                                    $f$ is the change concentration in a square
                                    
                                    D is the diffusion constant.
                                    
                                    $U$ is vector describing the flux
                                    </P><P>The divergence describes how large the gradients between squares are. THe diffusion coefficient describes how quickly flux flows along the gradient field.</P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SpatialMethods" aria-expanded="true" aria-controls="SpatialMethods"><H4 class="panel-title"><DIV class="col-xs-11">Methods</DIV></H4></A></DIV><DIV id="SpatialMethods" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>
                                    Anim pariatur cliche reprehenderit, enim eiusmod high life accusamus terry richardson ad squid. 3 wolf moon officia aute, non cupidatat skateboard dolor brunch. Food truck quinoa nesciunt laborum eiusmod. Brunch 3 wolf moon tempor, sunt aliqua put a bird on it squid single-origin coffee nulla assumenda shoreditch et. Nihil anim keffiyeh helvetica, craft beer labore wes anderson cred nesciunt sapiente ea proident. Ad vegan excepteur butcher vice lomo. Leggings occaecat craft beer farm-to-table, raw denim aesthetic synth nesciunt you probably haven't heard of them accusamus labore sustainable VHS.
                                </P></DIV></DIV></DIV></DIV><H3>Results</H3><H5>Spatial model and aiiA</H5><P>We took a spontaneously lysing parameter set from the ODE model and transfered it to the spatial model. Production and degradation terms for aiiA were added to the system of ODE's, as well as degradation of AHL by aiiA.
                Simulations were run both with and without antigen induction. With the right parameters associated with aiiA, a parameter set with spontaneous self-activation and lysis can be adjusted into a parameter set where cells fully activate and lyse only in response to antigen (table 3).
                
                <TABLE class="table table-bordered Results-Table"><THEAD><TR><TH colspan="4">Table 3: A system with positive regulation of LuxR that self-activated spontaneously was modified by adding aiiA, resulting in the desired behavior.</TH></TR></THEAD><TBODY><TR><TH>Original Set</TH><TH>Original Set &amp; aiiA</TH></TR><TR><TD>Score</TD><TD>99.98</TD><TD>0.22</TD></TR><TR><TD>Animation no antigen</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Animation antigen</TD><TD>10986</TD><TD>1.1%</TD></TR></TBODY></TABLE></P><H3>Conclusion
									</H3><P>
                    Conclusion/Discussion on the affibody experiment.
                </P><DIV class="Textbox Citations"><H3>
										References
										</H3><OL><LI>Williams, Joshua W., et al. &quot;Robust and sensitive control of a quorum‐sensing circuit by two interlocked feedback loops.&quot; <I>Molecular systems biology</I> 4.1 (2008): 234.</LI><LI>Haseltine, Eric L., and Frances H. Arnold. &quot;Implications of rewiring bacterial quorum sensing.&quot; <I>Applied and environmental microbiology</I> 74.2 (2008): 437-445.</LI><LI>Miyashiro, Tim, and Edward G. Ruby. &quot;Shedding light on bioluminescence regulation in Vibrio fischeri.&quot; <I>Molecular microbiology</I> 84.5 (2012): 795-806.</LI><LI>Bassler, Bonnie L., and Richard Losick. &quot;Bacterially speaking.&quot; <I>Cell</I> 125.2 (2006): 237-246.</LI></OL></DIV></SECTION></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid FooterWrapper"><DIV class="FooterImage"><DIV class="col-lg-12 FooterMainColumn"><DIV class="col-lg-8"><DIV id="Footer"><P>iGEM Wageningen 2017. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link" style="padding-right:0;">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>