<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Results_Fluorescent skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Results/Fluorescent</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="Main-Menu"><DIV class="col-lg-10 MainMenu-Center-Column MenuCenter"><DIV class="Project MenuButtonColumn"><DIV class="Project-Top TopMenuButtonColumn"><UL><LI class="project-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Project" class="nav-item" id="project">Project</A></LI></UL></DIV></DIV><DIV class="Background MenuButtonColumn"><DIV class="Background-Top TopMenuButtonColumn"><UL><LI class="background-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Background" class="nav-item active" id="background">Background</A></LI></UL></DIV></DIV><DIV class="Wet-Lab MenuButtonColumn"><DIV class="Wet-Lab-Top TopMenuButtonColumn"><UL><LI class="wet-lab-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Wet-lab" class="nav-item active" id="wet-lab">Wet-lab</A></LI></UL></DIV><DIV class="Wet-Lab-Bottom SubMenuButtonColumn"><UL id="wet-lab-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results" class="results-sub-nav-item sub-nav-item">Results</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Parts" class="parts-sub-nav-item sub-nav-item">BioBricks</A></LI></UL></DIV></DIV><DIV class="Model MenuButtonColumn"><DIV class="Model-Top TopMenuButtonColumn"><UL><LI class="model-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Model" class="model nav-item" id="model">Model</A></LI></UL></DIV><DIV class="Model-Bottom SubMenuButtonColumn"><UL id="model-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics" class="kinetics-sub-nav-item sub-nav-item">Cpx Kinetics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/QS" class="qs-sub-nav-item sub-nav-item">Quorum Sensing</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx" class="integrationcpx-sub-nav-item sub-nav-item">Cpx Integration</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS" class="integrationQS-sub-nav-item sub-nav-item">QS Integration</A></LI></UL></DIV></DIV><DIV class="Human-Practices MenuButtonColumn"><DIV class="Human-Practices-Top TopMenuButtonColumn"><UL><LI class="human-practices-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Human_Practices" class="nav-item" id="human-practices">Human Practices</A></LI></UL></DIV><DIV class="Human-Practices-Bottom SubMenuButtonColumn"><UL id="human-practices-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated" class="integrated-sub-nav-item sub-nav-item ">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Silver" class="safety-sub-nav-item sub-nav-item">Safety &amp; Ethics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Collaborations" class="collaboration-sub-nav-item sub-nav-item">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Future" class="future-sub-nav-item sub-nav-item">Future Perspectives</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Engagement" class="engagement-sub-nav-item sub-nav-item">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Meetup" class="meetup-sub-nav-item sub-nav-item">Meetup</A></LI></UL></DIV></DIV><DIV class="Application MenuButtonColumn"><DIV class="Application-Top TopMenuButtonColumn"><UL><LI class="application-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Applied_Design" class="nav-item" id="application">Application</A></LI></UL></DIV><DIV class="Application-Bottom SubMenuButtonColumn"><UL id="application-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Demonstrate" class="demo-sub-nav-item sub-nav-item">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Hardware" class="device-sub-nav-item sub-nav-item">Device</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Application/Rollout" class="rollout-sub-nav-item sub-nav-item">Roll-out Plan</A></LI></UL></DIV></DIV><DIV class="Notebook MenuButtonColumn"><DIV class="Notebook-Top TopMenuButtonColumn"><UL><LI class="notebook-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook" class="notebook nav-item" id="notebook">Notebook</A></LI></UL></DIV><DIV class="Notebook-Bottom SubMenuButtonColumn"><UL id="notebook-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols" class="protocols-sub-nav-item sub-nav-item">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/InterLab" class="interlab-sub-nav-item sub-nav-item">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Journal" class="journal-sub-nav-item sub-nav-item">Lab Journal</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Achievements" class="achievements-sub-nav-item sub-nav-item">Achievements</A></LI></UL></DIV></DIV><DIV class="Team MenuButtonColumn"><DIV class="Team-Top TopMenuButtonColumn"><UL><LI class="team-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Team" class="team nav-item" id="team">Team</A></LI></UL></DIV><DIV class="Team-Bottom SubMenuButtonColumn"><UL id="team-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Team/Members" class="members-sub-nav-item sub-nav-item">Members</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Attributions" class="attributions-sub-nav-item sub-nav-item">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Acknowledgements" class="Acknowledgements-sub-nav-item sub-nav-item">Acknowledgements</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="MobileMenu"><UL id="MainMobileMenu"><LI><BUTTON class="project-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Project'" type="button">Project</BUTTON></LI><LI><BUTTON class="background-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Background'" type="button">Background</BUTTON></LI><LI><BUTTON class="wet-lab-li-mo">Wet-lab</BUTTON></LI><LI><BUTTON class="model-li-mo">Model</BUTTON></LI><LI><BUTTON class="human-practices-li-mo">Human Practices</BUTTON></LI><LI><BUTTON class="application-li-mo">Application</BUTTON></LI><LI><BUTTON class="notebook-li-mo">Notebook</BUTTON></LI><LI><BUTTON class="team-li-mo">Team</BUTTON></LI></UL><UL id="MobileSubMenu"><LI class="wet-lab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Wet-lab'" type="">Wet-lab</BUTTON></LI><LI class="model-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model'">Model</BUTTON></LI><LI class="application-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Applied_Design'">Application</BUTTON></LI><LI class="human-practices-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Human_Practices'">Human Practices</BUTTON></LI><LI class="notebook-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook'">Notebook</BUTTON></LI><LI class="team-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team'">Team</BUTTON></LI><LI class="parts-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Parts'" type="button">Parts</BUTTON></LI><LI class="results-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Results'" type="button">Results</BUTTON></LI><LI class="kinetics-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics'" type="button">Cpx Kinetics</BUTTON></LI><LI class="qs-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/QS'" type="button">Quorum Sensing</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx'" type="button">Cpx integration</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS'" type="button">QS integration</BUTTON></LI><LI class="integrated-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated'" type="button">Integrated HP</BUTTON></LI><LI class="safety-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Silver'" type="button">Safety &amp; Ethics</BUTTON></LI><LI class="collaboration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Collaborations'" type="button">Collaborations</BUTTON></LI><LI class="future-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Future'" type="button">Future Perspectives</BUTTON></LI><LI class="engagement-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Engagement'" type="button">Engagement</BUTTON></LI><LI class="meetup-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Meetup'" type="button">Meetup</BUTTON></LI><LI class="demo-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Demonstrate'" type="button">Demonstrate</BUTTON></LI><LI class="device-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Hardware'" type="button">Device</BUTTON></LI><LI class="rollout-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Application/Rollout'" type="button">Rollout Plan</BUTTON></LI><LI class="protocols-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols'" type="button">Protocols</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/InterLab'" type="button">InterLab</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Journal'" type="button">Journal</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Achievements'" type="button">Achievements</BUTTON></LI><LI class="members-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team/Members'" type="button">Members</BUTTON></LI><LI class="attributions-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Attributions'" type="button">Attributions</BUTTON></LI></UL></DIV><DIV class="container-fluid OurContent"><DIV class="row"><DIV class="col-lg-8"><DIV class="Main-Border"><DIV id="SideMenu-Wrapper"><SIDENAV id="SideMenu"><DIV class="menu-head"><H4>Choosing the Best Reporter</H4></DIV><UL class="sidebar-nav"><LI class="menu-item"><A href="#Introduction">Introduction</A></LI><LI><A href="#Chromoproteins">Chromoproteins</A></LI><LI><A href="#Testing">Testing Fluorescent Proteins</A></LI><LI><A href="#Comparing">Comparing Fluorescent Proteins</A></LI><LI><A href="#Conclusions">Conclusions</A></LI></UL></SIDENAV></DIV><DIV class="Main-Center-Content-Column Overview"><DIV id="breadcrumb-wrapper"><UL class="breadcrumb"><LI><A href="https://2017.igem.org/Team:Wageningen_UR">Home</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Wet-lab">Wet-lab</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI>Fluorescent Proteins</LI></UL></DIV><SECTION id="Introduction"><DIV class="Title"><H1>Choosing the Best Reporter</H1></DIV><DIV class="Textbox Results-Desc"><P>
                                The aim of this project is to analyze different reporter proteins to choose the one with the most suitable characteristics for our device. As Mantis' signal generation is based on bimolecular complementation, the candidate reporter proteins were split and their halves fused to synthetic leucine zippers in order to promote their reassembly. The best reporter should fulfill a set of characteristics. First, the reporter protein has to show a bright signal, so our device can easily detect if the system has been activated. Second, the reassembly and maturation of the split reporter must be as fast as possible, to be able to detect a signal in a short timeframe. Last, our device will be used in tropical areas, where temperatures can be really high. Therefore, the reporter must also be able to produce a strong signal and mature fast at high temperatures. We successfully split five fluorescent proteins, which were delivered and proposed as a <A href="https://2017.igem.org/Team:Wageningen_UR/Part_Collection">Collection of Parts</A> for bimolecular fluorescence complementation analysis. We characterized the split proteins, which revealed that split sfGFP is the best-suited reporter for Mantis due to its thermostability. </P><P>The protocols for routine lab techniques used for this section can be found <A href="https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols">here</A>. The protocols specific for procedures involving the analysis of the fluorescent proteins can be found <A href="https://static.igem.org/mediawiki/2017/d/d0/Splitting_Fluorophores_and_Chromoproteins.pdf">here</A>.   </P></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#BApproach" aria-expanded="true" aria-controls="BApproach"><H4 class="panel-title"><DIV class="col-xs-11">	
											Background 
										</DIV></H4></A></DIV><DIV id="BApproach" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>Bimolecular fluorescence complementation (BiFC) is a technique used to study and detect protein-protein interactions. It is based on the development of protein fragments which will reassemble when brought together by fusion proteins. This technique has substituted others such as Förster resonance energy transfer (FRET) or two-hybrid systems due to its flexibility, lower background effects, and direct analysis. There are several criteria that a split-protein system must accomplish in order to provide a successful tool [1]. Among these, the reassembled protein should present the desired activity with sufficient intensity to be detected and measured. Besides, the individual fragments should not present any signal and there should not be any affinity between them, providing a low signal/noise ratio. This way, the split protein will only reassemble when the two halves are brought together by the fused, interacting proteins (Figure A).</P><DIV class="figure-fullwidth"><DIV class="figure-center-caption"><B>Figure A:</B> BiFC approach: The bimolecular interaction between the two interacting fusion proteins (leucine zippers) leads to the interaction between the two halves of mVenus. Once they interact, a hydrophobic pocket is formed, allowing the maturation reaction to happen. This reaction will form a chromophore inside the protein structure, which will grant fluorescence to the reassembled protein.
                                        </DIV></DIV><P>Among the different proteins that have been used in split protein systems, the fluorescent proteins are the most widely used. However, not all fluorescent proteins are equally suitable for BiFC analysis. One characteristic that is essential when using split protein reassembly is a monomeric conformation, as any other oligomeric state will render a lack of signal when a monomeric subunit is reassembled. Furthermore, a multimeric conformation might lead to unwanted interaction among the proteins, precipitation and even toxicity for the cell [2]. The ideal fluorescent protein should have a high quantum yield, high photostability and a short maturation time [3]. </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#CApproach" aria-expanded="true" aria-controls="CApproach"><H4 class="panel-title"><DIV class="col-xs-11">	
											Approach 
										</DIV></H4></A></DIV><DIV id="CApproach" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><H4>Choosing candidate reporter proteins, their split sites, and the interacting proteins</H4><P>Since <A href="https://2017.igem.org/Team:Wageningen_UR/Results/SpecificVisualization">our signal</A> is based on bimolecular complementation, we selected proteins that could be split into two parts and were able to regenerate a functional protein when coming together. Several fluorescent proteins had been previously split to be used as reporters in BiFC [4]. We chose five of them as our candidates: mRFP (Red fluorescent protein, <A href="http://parts.igem.org/Part:BBa_K2387054">BBa_K2387054</A>), sfGFP (Green fluorescent protein, <A href="http://parts.igem.org/Part:BBa_K2387047">BBa_K2387047</A>), eYFP (Yellow fluorescent protein, <A href="http://parts.igem.org/Part:BBa_K2387003">BBa_K2387003</A>), mVenus (Yellow fluorescent protein, <A href="http://parts.igem.org/Part:BBa_K2387045">BBa_K2387045</A>) and mCerulean (Cyan fluorescent protein, <A href="http://parts.igem.org/Part:BBa_K2387052">BBa_K2387052</A>). We based our decision on their availability either in the iGEM Registry or in our laboratory. In addition to fluorescent proteins, we also thought about the possibility of using split chromoproteins in Mantis so the signal could be observed with the naked eye and without using any artificial source of light. Although there is a large collection of chromoproteins in the iGEM Registry, none of them are monomeric. After a literature research study, we found three monomeric chromoproteins: Dathail [5] (Orange chromoprotein), Ultramarine [6] (Blue chromoprotein) and anm2CP [7] (Purple chromoprotein, <A href="http://parts.igem.org/Part:BBa_K2387001">BBa_K2387001</A>). </P><P>In order to choose the split sites for the fluorescent proteins and the chromoproteins we carried out a further literature research study. All the chosen fluorescent proteins had been split previously, so we used the split site found in the literature. Additionally, in the Registry we found that mRFP and mCerulean had also been split in other sites, so those sites were also chosen (Table A). However, to our knowledge, chromoproteins have never been split. Therefore, our approach in this case was to look at the structures of the chromoproteins or related proteins and to choose a split site situated in the loop between the eighth and the ninth β-strands. The structure of Dathail is available in PDB (<A href="http://www.rcsb.org/pdb/explore.do?structureId=5EXU">5EXU</A>). For Ultramarine, we looked at the structure of Rtms5 (<A href="http://www.rcsb.org/pdb/explore.do?structureId=2p4m">2P4M</A>), the multimeric chromoprotein from which it was evolved [6]. For anm2CP we looked at the structure of Killer Red (<A href="http://www.rcsb.org/pdb/explore.do?structureId=2wiq">2WIQ</A>), a phototoxic fluorescent protein derived from anm2CP [8]. The list of all the selected split sites can be found in Table A. </P><TABLE style="width:100%"><TBODY><TR><TH colspan="4">Table A: Chosen split sites.</TH></TR><TR><TH>Protein</TH><TH>Split Site(s)</TH><TH>Position</TH><TH>Reference</TH></TR><TR><TD>anm2CP</TD><TD>154-155
                                                168-169</TD><TD>Loop between β7 &amp; β8 strands
                                                Loop between β8 &amp; β9 strands</TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=2wiq">2WIQ</A></TD></TR><TR><TD>Dathail</TD><TD>165-166</TD><TD>Loop between β8 &amp; β9 strands</TD><TD><A href="https://www.rcsb.org/pdb/explore.do?structureId=5EXU">5EXU</A></TD></TR><TR><TD>Ultramarine</TD><TD>169-170</TD><TD>Loop between β8 &amp; β9 strands</TD><TD><A href="http://www.rcsb.org/pdb/explore.do?structureId=2p4m">2P4M</A></TD></TR><TR><TD>mRFP</TD><TD>154-155
                                                168-169</TD><TD>Loop between β7 &amp; β8 strands
                                                Loop between β8 &amp; β9 strands</TD><TD>iGEM Registry (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_I715022">BBa_I715022</A>&amp;<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_I715023">BBa_I715023</A>)
                                                [9] Guido, J., <I>et al.</I> (2006)</TD></TR><TR><TD>eYFP</TD><TD>154-155</TD><TD>Loop between β7 &amp; β8 strands</TD><TD>[10] Chang-Deng, H., <I>et al.</I>(2002)</TD></TR><TR><TD>mVenus</TD><TD>155-156</TD><TD>Loop between β7 &amp; β8 strands</TD><TD>[11] Shyu, Y.J., <I>et al.</I> (2006)</TD></TR><TR><TD>sfGFP</TD><TD>214-215</TD><TD>Loop between β10 &amp; β11 strands</TD><TD>[12] Zhou, Jun, <I>et al.</I> (2011)</TD></TR><TR><TD>mCerulean</TD><TD>155-156
                                                172-173</TD><TD>Loop between β7 &amp; β8 strands
                                                Loop between β8 &amp; β9 strands</TD><TD>iGEM Registry (<A href="http://partsregistry.org/Part:BBa_K157006">BBa_K157006</A>&amp;<A href="http://parts.igem.org/Part:BBa_K157005">BBa_K157005</A>)
                                                [11] Shyu, Y.J., <I>et al.</I> (2006)</TD></TR></TBODY></TABLE><P>One of the most preferred protein domains to test bimolecular complementation are leucine zippers. These are simple protein domains widely spread in nature that interact among themselves. Most leucine zippers found in nature interact in a parallel fashion, having both C-terminal ends at one side and the N-terminal ends at the other side [13]. In order to use parallel leucine zippers for bimolecular complementation, both the N-terminal and the C-terminal ends of the split fragments and the ones of the full-length protein should be proximal in the reassembled structure. Although this happens for some split sites, other split sites generate end termini on the opposite side of the protein where the ends of the full protein are found. This situation requires using antiparallel leucine zippers. The only antiparallel leucine zippers used in bimolecular complementation are in fact the ones that were used in the first BiFC experiment with GFP [14]. These leucine zippers are synthetic sequences and were used joined to the split fragments through a Gly-Ser linker to the split fragments. We used these same leucine zippers for inducing the reassembly of our split reporters.  </P><H4>Constructs</H4><P>The sequences of the full-length proteins were PCR amplified using as template Biobricks from the iGEM Distribution Kit (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_E1010">BBa_E1010</A>, <A href="http://parts.igem.org/Part:BBa_K525406">BBa_K525406</A>, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_I746916">BBa_I746916</A>), Addgene <A href="https://www.addgene.org/27794/">Plasmid #27794</A>mVenus C1 or gBlocks purchased from IDT as templates. The fragments for the split proteins were amplified from the sequence of the full-length protein. The fragments were fused to the respective leucine zippers through Golden Gate Assembly in a way that the two protein domains were fused by a glycine and a serine. The full-length proteins, as well as both halves of the split proteins fused to the leucine zippers, were cloned in pSB1C3 under the control of the araC/pBAD promoter (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_I0500">BBa_I0500</A>) and the strong RBS <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0034">BBa_B0034</A>. As an example, a plasmid containing both fragments of sfGFP can be observed in Figure B.</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure B:</B> Scheme of the plasmid expressing both halves of the split sfGFP fused to the respective leucine zippers.
                                        </DIV></DIV><P>These plasmids were transformed into <I>Escherichia coli</I> DH5α. After single colonies were selected, the plasmids were extracted and sent for sequencing to verify the sequence. Once the sequence was corroborated, plasmids were transformed into <I>E. coli</I> BL21(DE3). This strain was the one used to perform the experiments. </P></DIV></DIV></DIV></DIV></SECTION><SECTION id="Chromoproteins"><H2>Chromoproteins</H2><P>
                            A set of monomeric chromoproteins (anm2CP, Ultramarine, Dathail) and monomeric fluorescent proteins (mRFP, eYFP, mVenus, mCerulean, sfGFP) were successfully transformed and expressed in <I>E. coli</I> BL21(DE3). At the start of our project, we were considering the use of chromoproteins as a possible reporter that would allow us detecting the signal with the naked eye. Inducing the expression of the reporters using arabinose allowed us to observe potential candidates (Figure 1).
                        </P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 1: (A)</B><I>E. coli</I> BL21(DE3) expressing chromoproteins under induction with 0.2% arabinose. <B>(B)</B><I>E. coli</I> BL21(DE3) expressing anm2CP and fluorescent proteins under induction with 0.2% arabinose.
                            </DIV></DIV><P>Among the tested chromoproteins only anm2CP (<A href="http://parts.igem.org/Part:BBa_K2387001">BBa_K2387001</A>) produced colored cells. Among the fluorescent proteins, mRFP (<A href="http://parts.igem.org/Part:BBa_K2387054">BBa_K2387054</A>) produced bright pink cells and mCerulean (<A href="http://parts.igem.org/Part:BBa_K2387052">BBa_K2387052</A>) produced bright yellow cells. The absorbance spectra of these proteins and their split versions were analyzed and Table 1 shows the absorbance at the maximum absorption wavelength for each of them.</P><TABLE style="width:100%"><TBODY><TR><TH colspan="6">Table 1: Values of Absorbance at the picks maximum values.</TH></TR><TR><TH>Protein</TH><TH>Wavelength</TH><TH>Max. Absorbance (Full-length)</TH><TH>Part number (Full-length)</TH><TH>Max. Absorbance (Split)</TH><TH>Part number (Split)</TH></TR><TR><TD>anm2CP</TD><TD>570 nm</TD><TD>0.313±0.007</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387001">BBa_K2387001</A></TD><TD>-0.032±0.005</TD><TD>-</TD></TR><TR><TD>mCerulean</TD><TD>440 nm</TD><TD>0.6±0.03</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387052">BBa_K2387052</A></TD><TD>0.09±0.04</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387053">BBa_K2387053</A></TD></TR><TR><TD>mRFP</TD><TD>590 nm</TD><TD>0.59±0.05</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387054">BBa_K2387054</A></TD><TD>0.06±0.02</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387055">BBa_K2387055</A></TD></TR></TBODY></TABLE><P>The data in Table 1 shows that the absorbance of the split proteins is really low compared to their full-length version. This correlates with the fact that colonies expressing the split proteins didn’t show coloring detectable by the naked eye. For this reason, we decided to develop <A href="https://2017.igem.org/Team:Wageningen_UR/Hardware">our device</A> to measure fluorescence and we continued characterizing our successfully split fluorescent proteins! </P></SECTION><SECTION id="Testing"><H2>Testing Fluorescent Proteins</H2><P>Once all the full-length and split proteins were cloned in <I>E. coli</I> and their sequence confirmed, we tested them. To do this we measured their absorbance and fluorescence spectra. The absorbance and emission spectra for full mCerulean (<A href="http://parts.igem.org/Part:BBa_K2387052">BBa_K2387052</A>) and split mCerulean (<A href="http://parts.igem.org/Part:BBa_K2387053">BBa_K2387053</A>) can be observed in Figure 2. </P><DIV class="figure-fullwidth"><DIV class="figure-center-caption"><B>Figure 2:</B> Absorbance and emission spectra for mCerulean (left) and split mCerulean (right). The emission spectra were recorded applying an excitation wavelength centered at 433 nm with a 9 nm bandwidth. The plate reader we used (BioTek's Synergy<SUP>TM</SUP> Mx) has a monodirectional sensor, so the emission around the excitation wavelength could not be measured due to overlap.
                            </DIV></DIV><P>Comparison of the two emission spectra depicted in Figure 2 shows that the shape of the absorbance and emission spectra of mCerulean is not affected by the splitting. Similarly, the shape of the absorbance and the fluorescence spectra remained unchanged for sfGFP, mVenus, eYFP and mRFP. However, a proper recovery of the signal was not achieved for mRFP (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_I715022">BBa_I715022</A>&amp;<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_I715023">BBa_I715023</A>) and mCerulean (<A href="http://partsregistry.org/Part:BBa_K157006">BBa_K157006</A>&amp;<A href="http://parts.igem.org/Part:BBa_K157005">BBa_K157005</A>) when they were split using the sites found in the iGEM Registry (155-156 for both). The spectra for the rest of fluorescent proteins can be found in their BioBrick Pages (Table 2).</P><P>Although the <A href="https://2017.igem.org/Team:Wageningen_UR/Results/SpecificVisualization">Specific Visualization</A> system requires the proteins to interact inside the cell <I>in vivo</I>, the <A href="https://2017.igem.org/Team:Wageningen_UR/Results/Quorum_Sensing#Gen">Quorum Sensing</A> system requires an extracellular signal. Therefore, we tested if the split fluorescent proteins could reassemble <I>in vitro</I> after cells have lysed.</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#SApproach" aria-expanded="true" aria-controls="SApproach"><H4 class="panel-title"><DIV class="col-xs-11">	
											Reassembly of split proteins <I>in vitro</I></DIV></H4></A></DIV><DIV id="SApproach" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><P>To test the <I>in vitro</I> reassembly, two plasmids were constructed, each of them containing one half of the split mVenus fused to the corresponding leucine zipper. The plasmids were transformed into <I>E. coli</I> DH5α. This way, two populations were obtained, each of them expressing one of the fragments. Before lysing the cells, the two populations were mixed in the same tube. <I>E. coli</I> DH5α without any plasmid was used as negative control, and <I>E. coli</I> DH5α expressing the full mVenus was used as positive control. The lysis procedure was enzymatic, using lysozyme, DNase, and B-PER™ Bacterial Protein Extraction Reagent. Two variations of the protocol were tested: adding protease inhibitor, and not removing cell debris (not centrifuging). The lysate was measured to detect fluorescence emitted by mVenus (Figure C). </P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure C:</B> Graph showing the fluorescence intensity of the lysate after lysing empty <I>E. coli</I> (negative), <I>E. coli</I> expressing the full mVenus and two mixed populations of <I>E. coli</I> expressing the different fragments of split mVenus.
                                            </DIV></DIV><P>As can be observed, no fluorescence was detected arising from the sample with the two halves of the split mVenus, whereas fluorescence was detected in all variants of the protocol for the sample containing the full mVenus. This experiment proves that reassembly of split proteins <I>in vitro</I> does not happen (at least in the tested conditions). This was confirmed by an independent <A href="https://2017.igem.org/Team:Wageningen_UR/Results/Quorum_Sensing#TICS">experiment</A>. </P></DIV></DIV></DIV></DIV></SECTION><SECTION id="Comparing"><H2>Comparing Fluorescent Proteins</H2><P>For comparing our split fluorescent proteins, the first factor we determined was a comparison between the performance of split fluorescent proteins versus the full-length versions. For this test, we grew <I>E. coli</I> expressing split and full-length fluorescent proteins in M9 minimal medium while measuring at the same time the evolution of fluorescence. The relative fluorescence of the split fluorescent proteins in relation to the full-length proteins was calculated comparing the maximum values of fluorescence (Figure 3).
                        </P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 3:</B> Graph showing the relative fluorescence of the split fluorescent protein in comparison to their full-length versions.
                            </DIV></DIV><P>
                            As can be observed, split mVenus and split sfGFP excel in comparison to the other proteins, showing that the reassembly of the split portions of these proteins is more efficient than for other ones.
                        </P><P> Another parameter we measured to choose the best reporter was the Quantum Yields (QY) of the split fluorescent proteins, which relates the light absorbed and the emitted fluorescence. Therefore, the QY can be used as an estimation for the brightness of the fluorescent proteins. The QY must be calculated in comparison to a reference using the same wavelength for excitation. The full proteins were used as standards for the split proteins, as their QY has already been calculated previously. The calculated QY can be found in Table 2. </P><TABLE style="width:100%"><TBODY><TR><TH colspan="6">Table 2: Quantum Yields (QY) calculated for split proteins.</TH></TR><TR><TH>Protein</TH><TH>QY Full Protein</TH><TH>Reference</TH><TH>Part Number (Full-length)</TH><TH>QY Split Protein</TH><TH>Part Number (Split)</TH></TR><TR><TD>mRFP</TD><TD>0.15</TD><TD>[9] Guido, J, <I>et al.</I> (2006)</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387054">BBa_K2387054</A></TD><TD>0.06±0.04</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387055">BBa_K2387055</A></TD></TR><TR><TD>eYFP</TD><TD>0.61</TD><TD>[15] Nagai, T., <I>et al.</I> (2005)</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387003">BBa_K2387003</A></TD><TD>0.004±0.027</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387065">BBa_K2387065</A></TD></TR><TR><TD>mVenus</TD><TD>0.57</TD><TD>[15] Nagai, T., <I>et al.</I> (2005)</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387045">BBa_K2387045</A></TD><TD>0.61±0.06</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387046">BBa_K2387046</A></TD></TR><TR><TD>sfGFP</TD><TD>0.65</TD><TD>[16] Pédelacq, J., <I>et al.</I> (2004)</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387047">BBa_K2387047</A></TD><TD>1.3±0.2</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387048">BBa_K2387048</A></TD></TR><TR><TD>mCerulean</TD><TD>0.62</TD><TD>[17] Rizzo, M. A., <I>et al.</I> (2004)</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387052">BBa_K2387052</A></TD><TD>0.51±0.08</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387053">BBa_K2387053</A></TD></TR></TBODY></TABLE><P>As observed in Table 2, split mRFP and eYFP show much lower QY values than their full counterparts. Both mVenus and mCerulean show QY values similar to those of the full proteins, which indicates that these two proteins are good candidates in terms of brightness. However, the results show that the split sfGFP has a even higher QY than the full sfGFP. This may be due to background noise in the absorbance of the full sfGFP, which would alter the results. Nevertheless, although it is likely that splitting sfGFP does not increase the QY, the high value of QY for the split sfGFP show that this split protein can be considered bright.  </P><P>As mVenus and sfGFP seemed to be the best fluorescent proteins for Mantis when taking into consideration its reconstitution and brightness, we implemented mVenus in the <A href="https://2017.igem.org/Team:Wageningen_UR/Results/SpecificVisualization">Specific Visualization</A> system to reduce the signal detection time. Unfortunately, the new combination did not show the expected results (see the results of the experiment <A href="https://2017.igem.org/Team:Wageningen_UR/Demonstrate#improvement">here</A>).</P><P>The next factor we took into account was the maturation rate of the fluorescent proteins. This rate can be calculated as the half-time, which is the time at which half of the maximum fluorescence is reached. The maturation rates can be found in Table 3.
                        </P><TABLE style="width:100%"><TBODY><TR><TH colspan="6">Table 3: Maturation rates determined as t<SUB>1/2</SUB> (min).</TH></TR><TR><TH>Protein</TH><TH>Full Protein</TH><TH>Part Number (Full-length)</TH><TH>Split Protein</TH><TH>Part Number (Split)</TH></TR><TR><TD>mRFP</TD><TD>32±1</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387054">BBa_K2387054</A></TD><TD>54±3</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387055">BBa_K2387055</A></TD></TR><TR><TD>eYFP</TD><TD>18±4</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387003">BBa_K2387003</A></TD><TD>19±1</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387065">BBa_K2387065</A></TD></TR><TR><TD>mVenus</TD><TD>41±5</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387045">BBa_K2387045</A></TD><TD>59±8</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387046">BBa_K2387046</A></TD></TR><TR><TD>sfGFP</TD><TD>20.7±0.7</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387047">BBa_K2387047</A></TD><TD>25±7</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387048">BBa_K2387048</A></TD></TR><TR><TD>mCerulean</TD><TD>11.6±0.9</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387052">BBa_K2387052</A></TD><TD>22±4</TD><TD><A href="http://parts.igem.org/Part:BBa_K2387053">BBa_K2387053</A></TD></TR></TBODY></TABLE><P>According to the results, eYFP and mCerulean are the fastest maturing proteins although the difference with sfGFP is relatively small. However, we have already observed that eYFP shows a weak fluorescence intensity, so the time it takes it to reach a detectable fluorescence will be longer than the time taken for sfGFP and mCerulean.</P><P>The last characteristic we wanted our reporter to fulfill was thermostability. Our split reporter should be able to mature and produce fluorescence at high temperatures, typical of tropical areas (25°C-35°C). We tested the evolution of fluorescence for 3 hours at different temperatures: 4°C, 10°C, 20°C (Room Temperature), 30°C, 45°C (possible temperature in tropical regions) and 60°C. The results are shown in Figure 4. </P><DIV class="figure-fullwidth"><DIV class="figure-center-caption"><B>Figure 4:</B> Differences in fluorescence after incubation for three hours at different temperatures. A t-test was used to analyze differences between the values at 20°C (room temperatures) and the values at the other temperatures.
                                ns: No significance (P&gt;0.05); *: P≤0.05; **: P≤0.01; ***: P≤0.001.
                            </DIV></DIV><P>The graph shows that two of the proteins generate the most fluorescence when incubated at 45°C: mRFP and sfGFP, both in their full-length and split versions. However, for mCerulean, the full protein and its split version showed different behaviors. While full-length mCerulean matures best at 45°C, its split version shows more fluorescence at 20°C. This difference may be caused by a lower structural stability of the split fragments of mCerulean at higher temperatures. As split sfGFP has a maturation time of 25 min (at 30°C), is bright and it is thermostable at 45°C, it is our perfect candidate. We could not implement it in the <A href="https://2017.igem.org/Team:Wageningen_UR/Results/SpecificVisualization">Specific Visualization</A> system due to time constraints, but we think that the use of this reporter will improve Mantis.
                        </P></SECTION><SECTION id="Conclusions"><H2>Conclusions</H2><P>We analyzed two types of reporters: chromoproteins and fluorescent proteins. The first ones had not been used before in bimolecular complementation and we observed that they are not good reporters for being used in our application. However, we successfully split five fluorescent proteins, being able to detect fluorescence when reassembled inside the cells (<I>in vivo</I>). The fluorescent proteins tested and the split fluorescent proteins were submitted to the Registry. Both mCerulean and mRFP have been improved due to the fact that the split proteins already present in the Registry are split in the wrong site. The reassembly <I>in vitro</I> was tested with negative results. The first experiments showed promising results for mVenus and it was <A href="https://2017.igem.org/Team:Wageningen_UR/Demonstrate#improvement">implemented</A> in the <A href="https://2017.igem.org/Team:Wageningen_UR/Results/SpecificVisualization">Specific Visualization</A> system. Both sfGFP and mCerulean have been shown to be bright and have fast maturation. However, among these, only sfGFP is able to mature efficiently at 45°C and therefore is the one we choose for ideally use in our device. </P><DIV class="Textbox Citations"><H3>
								References
								</H3><OL><LI>Shekhawat, S.S. &amp; Ghosh, I., &quot;Split-protein systems: Beyond binary protein-protein interactions.&quot; <I>Current Opinion in Chemical Biology</I> 15.6 (2011): 790–797.</LI><LI>Shaner, N.C., <I>et al.</I>, &quot;A guide to choosing fluorescent proteins.&quot; <I>Nature Methods</I> 2.12 (2005): 905–909.</LI><LI>Zhang, X.E., <I>et al.</I>, &quot;Sensing of biomolecular interactions using fluorescence complementing systems in living cells.&quot; <I>Biosensors and Bioelectronics</I> 76 (2016): 243–250.</LI><LI>Miller, Kristi .E., <I>et al.</I>, &quot;Bimolecular fluorescence complementation (BiFC) analysis: advances and recent applications for genome-wide interaction studies.&quot; <I>Journal of Molecular Biology</I> 427.11 (2015): 2039–2055.</LI><LI>Langan, P.S., <I>et al.</I>, &quot;Evolution and characterization of a new reversibly photoswitching chromogenic protein, Dathail.&quot; <I>Journal of Molecular Biology</I> 428.9 (2016): 1776–1789.</LI><LI>Pettikiriarachchi, A., <I>et al.</I>, &quot;Ultramarine, a chromoprotein acceptor for forster resonance energy transfer.&quot; <I>PLoS ONE</I> 7.7 (2012): e41028.</LI><LI>Shagin, D.A., <I>et al.</I>, &quot;GFP-like Proteins as Ubiquitous Metazoan Superfamily: Evolution of Functional Features and Structural Complexity.&quot; <I>Molecular Biology and Evolution</I> 21.5 (2003): 841–850.</LI><LI>Bulina, Maria E., <I>et al.</I>, &quot;A genetically encoded photosensitizer.&quot; <I>Nature Biotechnology</I> 24.1 (2006): 95-99.</LI><LI>Guido, Jach, <I>et al.</I> &quot;An improved mRFP1 adds red to bimolecular fluorescence complementation.&quot; <I>Nature Methods</I> 13.8 (2006): 597-600.</LI><LI>Chang-Deng, Hu, <I>et al.</I> &quot;Visualization of Interactions among bZIP and Rel Family Proteins in Living Cells Using Bimolecular Fluorescence Complementation.&quot; <I>Molecular Cell</I> 9.4 (2002): 789-798.</LI><LI>Shyu, Y. John, <I>et al.</I> &quot;Identification of new fluorescent protein fragments for bimolecular fluorescence complementation analysis under physiological conditions&quot; <I>BioTechniques</I> 40.1 (2006): 61-66.</LI><LI>Zhou, Jun, <I>et al.</I> &quot;An improved bimolecular fluorescence complementation tool based on superfolder green fluorescent protein&quot; <I>Acta Biochim Biophys Sin</I> 43.3 (2011): 239-244.</LI><LI>Nikolaev, Yaroslav &amp; Pervushin, Konstantin, &quot;Rethinking Leucine Zipper – a ubiquitous signal transduction motif.&quot; <I>Nature Precedings</I>(2009)</LI><LI>Ghosh, Indraneel, <I>et al.</I>, &quot;Antiparallel Leucine Zipper-Directed Protein
Reassembly: Application to the Green Fluorescent Protein.&quot; <I>Journal of the American Chemical Society</I> 122 (2000): 5658-5659.</LI><LI>Nagai, Takeharu, <I>et al.</I> &quot;A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications&quot; <I>Nature Biotechnology</I> 20 (2002): 87-90.</LI><LI>Pédelacq, Jean-Denis, <I>et al.</I> &quot;Engineering and characterization of a superfolder green fluorescent protein&quot; <I>Nature Biotechnology</I> 24 (2004): 79-88.</LI><LI>Rizzo, Mark A., <I>et al.</I> &quot;An improved cyan fluorescent protein variant useful for FRET&quot; <I>Nature Biotechnology</I> 22 (2004): 445-449.</LI></OL></DIV></SECTION></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid FooterWrapper"><DIV class="FooterImage"><DIV class="col-lg-12 FooterMainColumn"><DIV class="col-lg-8"><DIV id="Footer"><P>iGEM Wageningen 2017. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link" style="padding-right:0;">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>