<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Wageningen_UR_Results_Affinity_Bodies skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Wageningen UR/Results/Affinity Bodies</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="Main-Menu"><DIV class="col-lg-10 MainMenu-Center-Column MenuCenter"><DIV class="Project MenuButtonColumn"><DIV class="Project-Top TopMenuButtonColumn"><UL><LI class="project-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Project" class="nav-item" id="project">Project</A></LI></UL></DIV></DIV><DIV class="Background MenuButtonColumn"><DIV class="Background-Top TopMenuButtonColumn"><UL><LI class="background-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Background" class="nav-item active" id="background">Background</A></LI></UL></DIV></DIV><DIV class="Wet-Lab MenuButtonColumn"><DIV class="Wet-Lab-Top TopMenuButtonColumn"><UL><LI class="wet-lab-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Wet-lab" class="nav-item active" id="wet-lab">Wet-lab</A></LI></UL></DIV><DIV class="Wet-Lab-Bottom SubMenuButtonColumn"><UL id="wet-lab-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results" class="results-sub-nav-item sub-nav-item">Results</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Parts" class="parts-sub-nav-item sub-nav-item">BioBricks</A></LI></UL></DIV></DIV><DIV class="Model MenuButtonColumn"><DIV class="Model-Top TopMenuButtonColumn"><UL><LI class="model-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Model" class="model nav-item" id="model">Model</A></LI></UL></DIV><DIV class="Model-Bottom SubMenuButtonColumn"><UL id="model-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics" class="kinetics-sub-nav-item sub-nav-item">Cpx Kinetics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/QS" class="qs-sub-nav-item sub-nav-item">Quorum Sensing</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx" class="integrationcpx-sub-nav-item sub-nav-item">Cpx Integration</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS" class="integrationQS-sub-nav-item sub-nav-item">QS Integration</A></LI></UL></DIV></DIV><DIV class="Human-Practices MenuButtonColumn"><DIV class="Human-Practices-Top TopMenuButtonColumn"><UL><LI class="human-practices-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Human_Practices" class="nav-item" id="human-practices">Human Practices</A></LI></UL></DIV><DIV class="Human-Practices-Bottom SubMenuButtonColumn"><UL id="human-practices-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated" class="integrated-sub-nav-item sub-nav-item ">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Silver" class="safety-sub-nav-item sub-nav-item">Safety &amp; Ethics</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Collaborations" class="collaboration-sub-nav-item sub-nav-item">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Future" class="future-sub-nav-item sub-nav-item">Future Perspectives</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Engagement" class="engagement-sub-nav-item sub-nav-item">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/HP/Meetup" class="meetup-sub-nav-item sub-nav-item">Meetup</A></LI></UL></DIV></DIV><DIV class="Application MenuButtonColumn"><DIV class="Application-Top TopMenuButtonColumn"><UL><LI class="application-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Applied_Design" class="nav-item" id="application">Application</A></LI></UL></DIV><DIV class="Application-Bottom SubMenuButtonColumn"><UL id="application-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Demonstrate" class="demo-sub-nav-item sub-nav-item">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Hardware" class="device-sub-nav-item sub-nav-item">Device</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Application/Rollout" class="rollout-sub-nav-item sub-nav-item">Roll-out Plan</A></LI></UL></DIV></DIV><DIV class="Notebook MenuButtonColumn"><DIV class="Notebook-Top TopMenuButtonColumn"><UL><LI class="notebook-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook" class="notebook nav-item" id="notebook">Notebook</A></LI></UL></DIV><DIV class="Notebook-Bottom SubMenuButtonColumn"><UL id="notebook-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols" class="protocols-sub-nav-item sub-nav-item">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/InterLab" class="interlab-sub-nav-item sub-nav-item">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Journal" class="journal-sub-nav-item sub-nav-item">Lab Journal</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Achievements" class="achievements-sub-nav-item sub-nav-item">Achievements</A></LI></UL></DIV></DIV><DIV class="Team MenuButtonColumn"><DIV class="Team-Top TopMenuButtonColumn"><UL><LI class="team-li"><A href="https://2017.igem.org/Team:Wageningen_UR/Team" class="team nav-item" id="team">Team</A></LI></UL></DIV><DIV class="Team-Bottom SubMenuButtonColumn"><UL id="team-sub"><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Team/Members" class="members-sub-nav-item sub-nav-item">Members</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Attributions" class="attributions-sub-nav-item sub-nav-item">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Acknowledgements" class="Acknowledgements-sub-nav-item sub-nav-item">Acknowledgements</A></LI></UL></DIV></DIV></DIV></DIV><DIV id="MobileMenu"><UL id="MainMobileMenu"><LI><BUTTON class="project-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Project'" type="button">Project</BUTTON></LI><LI><BUTTON class="background-li-mo" onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Background'" type="button">Background</BUTTON></LI><LI><BUTTON class="wet-lab-li-mo">Wet-lab</BUTTON></LI><LI><BUTTON class="model-li-mo">Model</BUTTON></LI><LI><BUTTON class="human-practices-li-mo">Human Practices</BUTTON></LI><LI><BUTTON class="application-li-mo">Application</BUTTON></LI><LI><BUTTON class="notebook-li-mo">Notebook</BUTTON></LI><LI><BUTTON class="team-li-mo">Team</BUTTON></LI></UL><UL id="MobileSubMenu"><LI class="wet-lab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Wet-lab'" type="">Wet-lab</BUTTON></LI><LI class="model-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model'">Model</BUTTON></LI><LI class="application-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Applied_Design'">Application</BUTTON></LI><LI class="human-practices-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Human_Practices'">Human Practices</BUTTON></LI><LI class="notebook-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook'">Notebook</BUTTON></LI><LI class="team-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team'">Team</BUTTON></LI><LI class="parts-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Parts'" type="button">Parts</BUTTON></LI><LI class="results-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Results'" type="button">Results</BUTTON></LI><LI class="kinetics-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/Cpx_Kinetics'" type="button">Cpx Kinetics</BUTTON></LI><LI class="qs-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/QS'" type="button">Quorum Sensing</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationCpx'" type="button">Cpx integration</BUTTON></LI><LI class="integration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Model/IntegrationQS'" type="button">QS integration</BUTTON></LI><LI class="integrated-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Gold_Integrated'" type="button">Integrated HP</BUTTON></LI><LI class="safety-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Silver'" type="button">Safety &amp; Ethics</BUTTON></LI><LI class="collaboration-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Collaborations'" type="button">Collaborations</BUTTON></LI><LI class="future-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Future'" type="button">Future Perspectives</BUTTON></LI><LI class="engagement-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Engagement'" type="button">Engagement</BUTTON></LI><LI class="meetup-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/HP/Meetup'" type="button">Meetup</BUTTON></LI><LI class="demo-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Demonstrate'" type="button">Demonstrate</BUTTON></LI><LI class="device-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Hardware'" type="button">Device</BUTTON></LI><LI class="rollout-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Application/Rollout'" type="button">Rollout Plan</BUTTON></LI><LI class="protocols-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Protocols'" type="button">Protocols</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/InterLab'" type="button">InterLab</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Notebook/Journal'" type="button">Journal</BUTTON></LI><LI class="interlab-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Achievements'" type="button">Achievements</BUTTON></LI><LI class="members-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Team/Members'" type="button">Members</BUTTON></LI><LI class="attributions-sub-li-mo"><BUTTON onclick="location.href='https://2017.igem.org/Team:Wageningen_UR/Attributions'" type="button">Attributions</BUTTON></LI></UL></DIV><DIV class="container-fluid OurContent"><DIV class="row"><DIV class="col-lg-8"><DIV class="Main-Border"><DIV id="SideMenu-Wrapper"><SIDENAV id="SideMenu"><DIV class="menu-head"><H4>Results</H4></DIV><UL class="sidebar-nav"><LI class="menu-item"><A href="#IN">Introduction</A></LI><LI><A href="#RE">Results</A></LI></UL></SIDENAV></DIV><DIV class="Main-Center-Content-Column Overview"><DIV id="breadcrumb-wrapper"><UL class="breadcrumb"><LI><A href="https://2017.igem.org/Team:Wageningen_UR">Home</A></LI><LI><A href="https://2017.igem.org/Team:Wageningen_UR/Results">Results</A></LI><LI>Affinity Body Library</LI></UL></DIV><SECTION class="TrypIntro"><DIV class="Title"><H1>Affinity Body Library</H1></DIV><DIV class="Textbox Results-Desc"><DIV><P>
							The goal of this project was to create a large library of affinity bodies. This library was created to screen for specific binding proteins for different disease antigens with the use of <A href="https://2017.igem.org/Team:Wageningen_UR/Results/Phage_Display">phage display</A>. The modularity and specificity of the Mantis diagnostic system is based on the use of the specific affinity bodies. The approach taken in this project proved to be a reliable way to create such a library without any apparent bias.
                                            
                            </P></DIV></DIV></SECTION><SECTION id="IN"><DIV class="Title"><H2>Introduction</H2></DIV><DIV class="col-lg-4 banner-column" style="float: right;"><DIV class="figure-fullwidth"><DIV class="figure-center-imagebox.Banner-box"><DIV class="figure-center-caption"><B>Figure 1:</B> Affinity body 3D structure of the 3 helices [4].
															</DIV></DIV></DIV></DIV><P>
											Affinity bodies are antibody mimetics based on staphylococcal protein A (SPA) [1]. The small, 6kDa, affinity proteins are based on the Z domain of the cell-wall anchored bacterial protein A. This protein binds immunoglobulin and contributes to evading the immune system [2]. By changing 13 amino acids on 2 helices essential for specificity, affinity bodies for a wide variety of targets can be developed (Figure 1). Since its discovery, affinity bodies have been generated to target insulin, fibrinogen, transferrin, tumor necrosis factor-a, IL-8, gp120, CD28, human serum albumin, IgA, IgE and HER2. Affinity bodies can be used for imaging, purification, detection and many therapeutic applications [3].

                                        </P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="headingOne"><A data-toggle="collapse" data-parent="#accordion" href="#Approach" aria-expanded="true" aria-controls="Approach"><H4 class="panel-title"><DIV class="col-xs-11">	
											Approach 
										</DIV></H4></A></DIV><DIV id="Approach" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body bg-primary"><H3>Construct</H3><DIV><P>
										The vector used to make the library with is a phagemid pComb3XSS, acquired from AddGene (Figure A). The pComb3XSS vector has an origin of replication for both <I>Escherichia coli</I> and filamentous phage M13. By using the SacI and SpeI restriction sites, any protein of interest can be expressed, fused to the G3P protein. This protein is incorporated in the M13 helper phages upon infection of bacteria carrying this phagemid. 
									</P></DIV><DIV class="figure-center"><DIV class="figure-center-imagebox"><DIV id="PosLuxR-Modal" class="modal"><SPAN class="close">×</SPAN><DIV class="caption"><B>Figure A:</B> pComb3XSS phagemid vector used for the library creation. The vector includes ampicillin resistance and origin of replications for <I>E. coli</I> and the M13 phage. Click for larger figure.  
  
                                    </DIV></DIV></DIV><DIV class="figure-center-caption"><B>Figure A:</B> pComb3XSS phagemid vector used for the library creation. The vector includes ampicillin resistance and origin of replications for <I>E. coli</I> and the M13 phage. Click for larger figure.  
  

                            </DIV></DIV><P>
									The amino acid sequence of the wild type Immunoglobin G (IgG)-binding affinity body is depicted in Figure B. The amino acid residues that are responsible for specific binding (red) will be targeted for randomization in the creation of the library.
									
									</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure B:</B> Amino acid sequence of wild-type IgG binding affinity body. Amino acid residues targeted for randomization in library creation in red.       
								</DIV></DIV><P>
								The Helix 3 region of the affinity body is not responsible for the binding specificity and will not be randomized. Therefore, the Helix 3 region is ligated into the backbone before the library is integrated for an easier library ligation. The Helix 3 fragment was amplified with primers in such a way that it can be ligated into the pComb3XSS vector with the existing SacI/SpeI restriction sites. However, a type-II restriction site (BsaI) was incorporated into the fragment to allow for the library integration without leaving a scar (Figure C).
								</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure C:</B> Overview of the Helix 3 fragment with SacI/SpeI restriction sites for cloning into the pComb3XSS vector. The internal BsaI restriction site allows for scarless integration of the Helix 1/Helix 2 library.  
								</DIV></DIV><P>
								Oligo fragments were used to create the Helix 1 and Helix 2 fragments with random nucleotides on the desired places. The oligo’s are designed to contain a NNK(K = G/T) degeneracy at the amino acid residues of interest. The NNK degeneracy improves the amount of non-sense codons produced by a NNN degeneracy and reduces the amount of stop codons as well [5]. The annealed fragments for Helix 1 (top) and Helix 2 (bottom) can be seen in Figure D.
								</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure D:</B> Top: Fragment of annealed oligo's designed for Helix 1 with a NNK degeneracy. Bottom: Fragment of annealed oligo's designed for Helix 2 with a NNK degeneracy.   
								</DIV></DIV><P>The Helix 1 and Helix 2 fragments were ligated into the linearized backbone (SacI/BsaI) and the ligation product was used for the transformation of XL1-Blue cells. The XL1-Blue cells have the following genotype: recA1 endA1 gyrA96 thi-1 hsdR17 supE44 relA1 lac [F ́proAB lacI qZ∆M15 Tn10 (Tetr)]. Importantly the strain used has an F pilus, which is essential for the attachment of the M13 phages. 
									</P></DIV></DIV></DIV></DIV></SECTION></DIV><SECTION id="RE"><DIV class="Title"><H2>Results</H2></DIV><DIV><P>
							After 25 transformations the colonies were counted and all pooled together which in total yielded an estimated library size of 110,000 affinity bodies. To check whether the library has a bias towards certain nucleotides and therefore amino acids, 96 colony PCRs were performed on one of the transformations.  </P><P>
							The PCR products were sent for sequencing and from 86 successful PCR reactions the data is depicted in Figure 2. The rest of the PCR products had regions of bad sequencing quality and were discarded. On the X-axis all the randomized nucleotide places (39) can be seen. On the Y-axis the total amount of sequenced samples is given. So each column represents a randomized nucleotide place divided into the 4 base pairs (ATCG). As expected, in every third column there are only the G and T base pair. All the columns show a very similar pattern and indicates that there is no significant bias towards any of the base pairs overall.
							</P></DIV><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 2:</B> Sequencing data from 86 colony PCRs on a transformation of the library mixture. Each column represents a randomized nucleotide position and is divided into the 4 possible base pairs (ATCG). 
								</DIV></DIV><P>
							To further investigate the pattern that is seen, the average occurrence was plotted in Figure 3 and the data is normalized to the expected occurrence for each of the base pairs in a truly random library. Statistical analysis showed that there is no significant difference with the expected occurrence (control) and each of the base pairs (P &lt; 0.05). 
							</P><DIV class="figure-center"><DIV class="figure-center-caption"><B>Figure 3:</B> The average occurrence for each of the 4 basepairs, normalized to the expected occurrence (control), is plotted on the Y-axis. All columns are compared to the control and no significant correlation was found (P &lt; 0.05). 
								</DIV></DIV></SECTION><SECTION class="qs"><DIV class="Textbox Citations"><H3>
References
</H3><OL><LI>Nord, Karin, et al. &quot;<I>Binding proteins selected from combinatorial libraries of an α-helical bacterial receptor domain.</I>&quot; Nature biotechnology 15.8 (1997): 772-777.</LI><LI>Nord, Karin, et al. &quot;<I>A combinatorial library of an α-helical bacterial receptor domain.</I>&quot; Protein Engineering, Design and Selection 8.6 (1995): 601-608.</LI><LI>Löfblom, John, et al. &quot;<I>Affibodies: engineered proteins for therapeutic, diagnostic and biotechnological applications.</I>&quot; FEBS letters 584.12 (2010): 2670-2680.</LI><LI>Lendel, Christofer, Jakob Dogan, and Torleif Härd. &quot;Structural basis for molecular recognition in an affibody: affibody complex.&quot; <I>Journal of molecular biology</I> 359.5 (2006): 1293-1304.</LI><LI>Hughes, Marcus D., et al. &quot;<I>Removing the redundancy from randomised gene libraries.</I>&quot; Journal of molecular biology 331.5 (2003): 973-979.</LI></OL></DIV></SECTION></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid FooterWrapper"><DIV class="FooterImage"><DIV class="col-lg-12 FooterMainColumn"><DIV class="col-lg-8"><DIV id="Footer"><P>iGEM Wageningen 2017. Send us an <A href="mailto:igemwageningen@gmail.com" class="soft_link" style="padding-right:0;">email</A> or follow us on <A class="soft_link" href="http://www.facebook.com/iGEMWageningen" target="_blank">Facebook</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>