Team:Wageningen UR/Results/Fluorescent
Project
Background
Wet-lab
Results
BioBricks
Model
Cpx Kinetics
Quorum Sensing
Cpx Integration
QS Integration
Human Practices
Integrated HP
Safety & Ethics
Collaborations
Future Perspectives
Engagement
Meetup
Application
Demonstrate
Device
Roll-out Plan
Notebook
Protocols
InterLab
Lab Journal
Achievements
Team
Members
Attributions
Acknowledgements
Project
Background
Wet-lab
Model
Human Practices
Application
Notebook
Team
Wet-lab
Model
Application
Human Practices
Notebook
Team
Parts
Results
Cpx Kinetics
Quorum Sensing
Cpx integration
QS integration
Integrated HP
Safety & Ethics
Collaborations
Future Perspectives
Engagement
Meetup
Demonstrate
Device
Rollout Plan
Protocols
InterLab
Journal
Achievements
Members
Attributions
Choosing the Best Reporter
Introduction
Chromoproteins
Testing Fluorescent Proteins
Comparing Fluorescent Proteins
Conclusions
Home
Wet-lab
Results
Fluorescent Proteins
Choosing the Best Reporter
The aim of this project is to analyze different reporter proteins to choose the one with the most suitable characteristics for our device. As Mantis' signal generation is based on bimolecular complementation, the candidate reporter proteins were split and their halves fused to synthetic leucine zippers in order to promote their reassembly. The best reporter should fulfill a set of characteristics. First, the reporter protein has to show a bright signal, so our device can easily detect if the system has been activated. Second, the reassembly and maturation of the split reporter must be as fast as possible, to be able to detect a signal in a short timeframe. Last, our device will be used in tropical areas, where temperatures can be really high. Therefore, the reporter must also be able to produce a strong signal and mature fast at high temperatures. We successfully split five fluorescent proteins, which were delivered and proposed as a Collection of Parts for bimolecular fluorescence complementation analysis. We characterized the split proteins, which revealed that split sfGFP is the best-suited reporter for Mantis due to its thermostability.
The protocols for routine lab techniques used for this section can be found here . The protocols specific for procedures involving the analysis of the fluorescent proteins can be found here .
Background
Bimolecular fluorescence complementation (BiFC) is a technique used to study and detect protein-protein interactions. It is based on the development of protein fragments which will reassemble when brought together by fusion proteins. This technique has substituted others such as Förster resonance energy transfer (FRET) or two-hybrid systems due to its flexibility, lower background effects, and direct analysis. There are several criteria that a split-protein system must accomplish in order to provide a successful tool [1]. Among these, the reassembled protein should present the desired activity with sufficient intensity to be detected and measured. Besides, the individual fragments should not present any signal and there should not be any affinity between them, providing a low signal/noise ratio. This way, the split protein will only reassemble when the two halves are brought together by the fused, interacting proteins (Figure A).
Figure A: BiFC approach: The bimolecular interaction between the two interacting fusion proteins (leucine zippers) leads to the interaction between the two halves of mVenus. Once they interact, a hydrophobic pocket is formed, allowing the maturation reaction to happen. This reaction will form a chromophore inside the protein structure, which will grant fluorescence to the reassembled protein.
Among the different proteins that have been used in split protein systems, the fluorescent proteins are the most widely used. However, not all fluorescent proteins are equally suitable for BiFC analysis. One characteristic that is essential when using split protein reassembly is a monomeric conformation, as any other oligomeric state will render a lack of signal when a monomeric subunit is reassembled. Furthermore, a multimeric conformation might lead to unwanted interaction among the proteins, precipitation and even toxicity for the cell [2]. The ideal fluorescent protein should have a high quantum yield, high photostability and a short maturation time [3].
Approach
Choosing candidate reporter proteins, their split sites, and the interacting proteins
Since our signal is based on bimolecular complementation, we selected proteins that could be split into two parts and were able to regenerate a functional protein when coming together. Several fluorescent proteins had been previously split to be used as reporters in BiFC [4]. We chose five of them as our candidates: mRFP (Red fluorescent protein, BBa_K2387054 ), sfGFP (Green fluorescent protein, BBa_K2387047 ), eYFP (Yellow fluorescent protein, BBa_K2387003 ), mVenus (Yellow fluorescent protein, BBa_K2387045 ) and mCerulean (Cyan fluorescent protein, BBa_K2387052 ). We based our decision on their availability either in the iGEM Registry or in our laboratory. In addition to fluorescent proteins, we also thought about the possibility of using split chromoproteins in Mantis so the signal could be observed with the naked eye and without using any artificial source of light. Although there is a large collection of chromoproteins in the iGEM Registry, none of them are monomeric. After a literature research study, we found three monomeric chromoproteins: Dathail [5] (Orange chromoprotein), Ultramarine [6] (Blue chromoprotein) and anm2CP [7] (Purple chromoprotein, BBa_K2387001 ).
In order to choose the split sites for the fluorescent proteins and the chromoproteins we carried out a further literature research study. All the chosen fluorescent proteins had been split previously, so we used the split site found in the literature. Additionally, in the Registry we found that mRFP and mCerulean had also been split in other sites, so those sites were also chosen (Table A). However, to our knowledge, chromoproteins have never been split. Therefore, our approach in this case was to look at the structures of the chromoproteins or related proteins and to choose a split site situated in the loop between the eighth and the ninth β-strands. The structure of Dathail is available in PDB ( 5EXU ). For Ultramarine, we looked at the structure of Rtms5 ( 2P4M ), the multimeric chromoprotein from which it was evolved [6]. For anm2CP we looked at the structure of Killer Red ( 2WIQ ), a phototoxic fluorescent protein derived from anm2CP [8]. The list of all the selected split sites can be found in Table A.
Table A: Chosen split sites.
Protein
Split Site(s)
Position
Reference
anm2CP
154-155
168-169
Loop between β7 & β8 strands
Loop between β8 & β9 strands
2WIQ
Dathail
165-166
Loop between β8 & β9 strands
5EXU
Ultramarine
169-170
Loop between β8 & β9 strands
2P4M
mRFP
154-155
168-169
Loop between β7 & β8 strands
Loop between β8 & β9 strands
iGEM Registry ( BBa_I715022 & BBa_I715023 )
[9] Guido, J., et al. (2006)
eYFP
154-155
Loop between β7 & β8 strands
[10] Chang-Deng, H., et al.(2002)
mVenus
155-156
Loop between β7 & β8 strands
[11] Shyu, Y.J., et al. (2006)
sfGFP
214-215
Loop between β10 & β11 strands
[12] Zhou, Jun, et al. (2011)
mCerulean
155-156
172-173
Loop between β7 & β8 strands
Loop between β8 & β9 strands
iGEM Registry ( BBa_K157006 & BBa_K157005 )
[11] Shyu, Y.J., et al. (2006)
One of the most preferred protein domains to test bimolecular complementation are leucine zippers. These are simple protein domains widely spread in nature that interact among themselves. Most leucine zippers found in nature interact in a parallel fashion, having both C-terminal ends at one side and the N-terminal ends at the other side [13]. In order to use parallel leucine zippers for bimolecular complementation, both the N-terminal and the C-terminal ends of the split fragments and the ones of the full-length protein should be proximal in the reassembled structure. Although this happens for some split sites, other split sites generate end termini on the opposite side of the protein where the ends of the full protein are found. This situation requires using antiparallel leucine zippers. The only antiparallel leucine zippers used in bimolecular complementation are in fact the ones that were used in the first BiFC experiment with GFP [14]. These leucine zippers are synthetic sequences and were used joined to the split fragments through a Gly-Ser linker to the split fragments. We used these same leucine zippers for inducing the reassembly of our split reporters.
Constructs
The sequences of the full-length proteins were PCR amplified using as template Biobricks from the iGEM Distribution Kit ( BBa_E1010 , BBa_K525406 , BBa_I746916 ), Addgene Plasmid #27794 mVenus C1 or gBlocks purchased from IDT as templates. The fragments for the split proteins were amplified from the sequence of the full-length protein. The fragments were fused to the respective leucine zippers through Golden Gate Assembly in a way that the two protein domains were fused by a glycine and a serine. The full-length proteins, as well as both halves of the split proteins fused to the leucine zippers, were cloned in pSB1C3 under the control of the araC/pBAD promoter ( BBa_I0500 ) and the strong RBS BBa_B0034 . As an example, a plasmid containing both fragments of sfGFP can be observed in Figure B.
Figure B: Scheme of the plasmid expressing both halves of the split sfGFP fused to the respective leucine zippers.
These plasmids were transformed into Escherichia coli DH5α. After single colonies were selected, the plasmids were extracted and sent for sequencing to verify the sequence. Once the sequence was corroborated, plasmids were transformed into E. coli BL21(DE3). This strain was the one used to perform the experiments.
Chromoproteins
A set of monomeric chromoproteins (anm2CP, Ultramarine, Dathail) and monomeric fluorescent proteins (mRFP, eYFP, mVenus, mCerulean, sfGFP) were successfully transformed and expressed in E. coli BL21(DE3). At the start of our project, we were considering the use of chromoproteins as a possible reporter that would allow us detecting the signal with the naked eye. Inducing the expression of the reporters using arabinose allowed us to observe potential candidates (Figure 1).
Figure 1: (A) E. coli BL21(DE3) expressing chromoproteins under induction with 0.2% arabinose. (B) E. coli BL21(DE3) expressing anm2CP and fluorescent proteins under induction with 0.2% arabinose.
Among the tested chromoproteins only anm2CP ( BBa_K2387001 ) produced colored cells. Among the fluorescent proteins, mRFP ( BBa_K2387054 ) produced bright pink cells and mCerulean ( BBa_K2387052 ) produced bright yellow cells. The absorbance spectra of these proteins and their split versions were analyzed and Table 1 shows the absorbance at the maximum absorption wavelength for each of them.
Table 1: Values of Absorbance at the picks maximum values.
Protein
Wavelength
Max. Absorbance (Full-length)
Part number (Full-length)
Max. Absorbance (Split)
Part number (Split)
anm2CP
570 nm
0.313±0.007
BBa_K2387001
-0.032±0.005
-
mCerulean
440 nm
0.6±0.03
BBa_K2387052
0.09±0.04
BBa_K2387053
mRFP
590 nm
0.59±0.05
BBa_K2387054
0.06±0.02
BBa_K2387055
The data in Table 1 shows that the absorbance of the split proteins is really low compared to their full-length version. This correlates with the fact that colonies expressing the split proteins didn’t show coloring detectable by the naked eye. For this reason, we decided to develop our device to measure fluorescence and we continued characterizing our successfully split fluorescent proteins!
Testing Fluorescent Proteins
Once all the full-length and split proteins were cloned in E. coli and their sequence confirmed, we tested them. To do this we measured their absorbance and fluorescence spectra. The absorbance and emission spectra for full mCerulean ( BBa_K2387052 ) and split mCerulean ( BBa_K2387053 ) can be observed in Figure 2.
Figure 2: Absorbance and emission spectra for mCerulean (left) and split mCerulean (right). The emission spectra were recorded applying an excitation wavelength centered at 433 nm with a 9 nm bandwidth. The plate reader we used (BioTek's SynergyTM Mx) has a monodirectional sensor, so the emission around the excitation wavelength could not be measured due to overlap.
Comparison of the two emission spectra depicted in Figure 2 shows that the shape of the absorbance and emission spectra of mCerulean is not affected by the splitting. Similarly, the shape of the absorbance and the fluorescence spectra remained unchanged for sfGFP, mVenus, eYFP and mRFP. However, a proper recovery of the signal was not achieved for mRFP ( BBa_I715022 & BBa_I715023 ) and mCerulean ( BBa_K157006 & BBa_K157005 ) when they were split using the sites found in the iGEM Registry (155-156 for both). The spectra for the rest of fluorescent proteins can be found in their BioBrick Pages (Table 2).
Although the Specific Visualization system requires the proteins to interact inside the cell in vivo, the Quorum Sensing system requires an extracellular signal. Therefore, we tested if the split fluorescent proteins could reassemble in vitro after cells have lysed.
Reassembly of split proteins in vitro
To test the in vitro reassembly, two plasmids were constructed, each of them containing one half of the split mVenus fused to the corresponding leucine zipper. The plasmids were transformed into E. coli DH5α. This way, two populations were obtained, each of them expressing one of the fragments. Before lysing the cells, the two populations were mixed in the same tube. E. coli DH5α without any plasmid was used as negative control, and E. coli DH5α expressing the full mVenus was used as positive control. The lysis procedure was enzymatic, using lysozyme, DNase, and B-PER™ Bacterial Protein Extraction Reagent. Two variations of the protocol were tested: adding protease inhibitor, and not removing cell debris (not centrifuging). The lysate was measured to detect fluorescence emitted by mVenus (Figure C).
Figure C: Graph showing the fluorescence intensity of the lysate after lysing empty E. coli (negative), E. coli expressing the full mVenus and two mixed populations of E. coli expressing the different fragments of split mVenus.
As can be observed, no fluorescence was detected arising from the sample with the two halves of the split mVenus, whereas fluorescence was detected in all variants of the protocol for the sample containing the full mVenus. This experiment proves that reassembly of split proteins in vitro does not happen (at least in the tested conditions). This was confirmed by an independent experiment .
Comparing Fluorescent Proteins
For comparing our split fluorescent proteins, the first factor we determined was a comparison between the performance of split fluorescent proteins versus the full-length versions. For this test, we grew E. coli expressing split and full-length fluorescent proteins in M9 minimal medium while measuring at the same time the evolution of fluorescence. The relative fluorescence of the split fluorescent proteins in relation to the full-length proteins was calculated comparing the maximum values of fluorescence (Figure 3).
Figure 3: Graph showing the relative fluorescence of the split fluorescent protein in comparison to their full-length versions.
As can be observed, split mVenus and split sfGFP excel in comparison to the other proteins, showing that the reassembly of the split portions of these proteins is more efficient than for other ones.
Another parameter we measured to choose the best reporter was the Quantum Yields (QY) of the split fluorescent proteins, which relates the light absorbed and the emitted fluorescence. Therefore, the QY can be used as an estimation for the brightness of the fluorescent proteins. The QY must be calculated in comparison to a reference using the same wavelength for excitation. The full proteins were used as standards for the split proteins, as their QY has already been calculated previously. The calculated QY can be found in Table 2.
Table 2: Quantum Yields (QY) calculated for split proteins.
Protein
QY Full Protein
Reference
Part Number (Full-length)
QY Split Protein
Part Number (Split)
mRFP
0.15
[9] Guido, J, et al. (2006)
BBa_K2387054
0.06±0.04
BBa_K2387055
eYFP
0.61
[15] Nagai, T., et al. (2005)
BBa_K2387003
0.004±0.027
BBa_K2387065
mVenus
0.57
[15] Nagai, T., et al. (2005)
BBa_K2387045
0.61±0.06
BBa_K2387046
sfGFP
0.65
[16] Pédelacq, J., et al. (2004)
BBa_K2387047
1.3±0.2
BBa_K2387048
mCerulean
0.62
[17] Rizzo, M. A., et al. (2004)
BBa_K2387052
0.51±0.08
BBa_K2387053
As observed in Table 2, split mRFP and eYFP show much lower QY values than their full counterparts. Both mVenus and mCerulean show QY values similar to those of the full proteins, which indicates that these two proteins are good candidates in terms of brightness. However, the results show that the split sfGFP has a even higher QY than the full sfGFP. This may be due to background noise in the absorbance of the full sfGFP, which would alter the results. Nevertheless, although it is likely that splitting sfGFP does not increase the QY, the high value of QY for the split sfGFP show that this split protein can be considered bright.
As mVenus and sfGFP seemed to be the best fluorescent proteins for Mantis when taking into consideration its reconstitution and brightness, we implemented mVenus in the Specific Visualization system to reduce the signal detection time. Unfortunately, the new combination did not show the expected results (see the results of the experiment here ).
The next factor we took into account was the maturation rate of the fluorescent proteins. This rate can be calculated as the half-time, which is the time at which half of the maximum fluorescence is reached. The maturation rates can be found in Table 3.
Table 3: Maturation rates determined as t1/2 (min).
Protein
Full Protein
Part Number (Full-length)
Split Protein
Part Number (Split)
mRFP
32±1
BBa_K2387054
54±3
BBa_K2387055
eYFP
18±4
BBa_K2387003
19±1
BBa_K2387065
mVenus
41±5
BBa_K2387045
59±8
BBa_K2387046
sfGFP
20.7±0.7
BBa_K2387047
25±7
BBa_K2387048
mCerulean
11.6±0.9
BBa_K2387052
22±4
BBa_K2387053
According to the results, eYFP and mCerulean are the fastest maturing proteins although the difference with sfGFP is relatively small. However, we have already observed that eYFP shows a weak fluorescence intensity, so the time it takes it to reach a detectable fluorescence will be longer than the time taken for sfGFP and mCerulean.
The last characteristic we wanted our reporter to fulfill was thermostability. Our split reporter should be able to mature and produce fluorescence at high temperatures, typical of tropical areas (25°C-35°C). We tested the evolution of fluorescence for 3 hours at different temperatures: 4°C, 10°C, 20°C (Room Temperature), 30°C, 45°C (possible temperature in tropical regions) and 60°C. The results are shown in Figure 4.
Figure 4: Differences in fluorescence after incubation for three hours at different temperatures. A t-test was used to analyze differences between the values at 20°C (room temperatures) and the values at the other temperatures.
ns: No significance (P>0.05); *: P≤0.05; **: P≤0.01; ***: P≤0.001.
The graph shows that two of the proteins generate the most fluorescence when incubated at 45°C: mRFP and sfGFP, both in their full-length and split versions. However, for mCerulean, the full protein and its split version showed different behaviors. While full-length mCerulean matures best at 45°C, its split version shows more fluorescence at 20°C. This difference may be caused by a lower structural stability of the split fragments of mCerulean at higher temperatures. As split sfGFP has a maturation time of 25 min (at 30°C), is bright and it is thermostable at 45°C, it is our perfect candidate. We could not implement it in the Specific Visualization system due to time constraints, but we think that the use of this reporter will improve Mantis.
Conclusions
We analyzed two types of reporters: chromoproteins and fluorescent proteins. The first ones had not been used before in bimolecular complementation and we observed that they are not good reporters for being used in our application. However, we successfully split five fluorescent proteins, being able to detect fluorescence when reassembled inside the cells (in vivo). The fluorescent proteins tested and the split fluorescent proteins were submitted to the Registry. Both mCerulean and mRFP have been improved due to the fact that the split proteins already present in the Registry are split in the wrong site. The reassembly in vitro was tested with negative results. The first experiments showed promising results for mVenus and it was implemented in the Specific Visualization system. Both sfGFP and mCerulean have been shown to be bright and have fast maturation. However, among these, only sfGFP is able to mature efficiently at 45°C and therefore is the one we choose for ideally use in our device.
References
Shekhawat, S.S. & Ghosh, I., "Split-protein systems: Beyond binary protein-protein interactions." Current Opinion in Chemical Biology 15.6 (2011): 790–797.
Shaner, N.C., et al., "A guide to choosing fluorescent proteins." Nature Methods 2.12 (2005): 905–909.
Zhang, X.E., et al., "Sensing of biomolecular interactions using fluorescence complementing systems in living cells." Biosensors and Bioelectronics 76 (2016): 243–250.
Miller, Kristi .E., et al., "Bimolecular fluorescence complementation (BiFC) analysis: advances and recent applications for genome-wide interaction studies." Journal of Molecular Biology 427.11 (2015): 2039–2055.
Langan, P.S., et al., "Evolution and characterization of a new reversibly photoswitching chromogenic protein, Dathail." Journal of Molecular Biology 428.9 (2016): 1776–1789.
Pettikiriarachchi, A., et al., "Ultramarine, a chromoprotein acceptor for forster resonance energy transfer." PLoS ONE 7.7 (2012): e41028.
Shagin, D.A., et al., "GFP-like Proteins as Ubiquitous Metazoan Superfamily: Evolution of Functional Features and Structural Complexity." Molecular Biology and Evolution 21.5 (2003): 841–850.
Bulina, Maria E., et al., "A genetically encoded photosensitizer." Nature Biotechnology 24.1 (2006): 95-99.
Guido, Jach, et al. "An improved mRFP1 adds red to bimolecular fluorescence complementation." Nature Methods 13.8 (2006): 597-600.
Chang-Deng, Hu, et al. "Visualization of Interactions among bZIP and Rel Family Proteins in Living Cells Using Bimolecular Fluorescence Complementation." Molecular Cell 9.4 (2002): 789-798.
Shyu, Y. John, et al. "Identification of new fluorescent protein fragments for bimolecular fluorescence complementation analysis under physiological conditions" BioTechniques 40.1 (2006): 61-66.
Zhou, Jun, et al. "An improved bimolecular fluorescence complementation tool based on superfolder green fluorescent protein" Acta Biochim Biophys Sin 43.3 (2011): 239-244.
Nikolaev, Yaroslav & Pervushin, Konstantin, "Rethinking Leucine Zipper – a ubiquitous signal transduction motif." Nature Precedings(2009)
Ghosh, Indraneel, et al., "Antiparallel Leucine Zipper-Directed Protein Reassembly: Application to the Green Fluorescent Protein." Journal of the American Chemical Society 122 (2000): 5658-5659.
Nagai, Takeharu, et al. "A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications" Nature Biotechnology 20 (2002): 87-90.
Pédelacq, Jean-Denis, et al. "Engineering and characterization of a superfolder green fluorescent protein" Nature Biotechnology 24 (2004): 79-88.
Rizzo, Mark A., et al. "An improved cyan fluorescent protein variant useful for FRET" Nature Biotechnology 22 (2004): 445-449.
iGEM Wageningen 2017. Send us an email or follow us on Facebook
