"
Page
Discussion
View source
History
teams
Log in
 
Team:JUIT-India/Protocol
From 2012.igem.org
Home
Team
Team
Sponsors
Project
Overview
Protocol
Parts
Modeling
Human Practices
Notebook
Experimental Design
Diary
Brainstorming
Safety
Protocols :-
Objective:
To perform restriction digestion of  DNA with EcoR I and BamHI enzymes.
Principle:
Restriction enzymes are Nucleases which can cleave the sugar-phosphate backbone of DNA, found in bacteria. As they cut within the molecule, they are commonly called restriction endonucleases. They specifically cleave the nucleic acids at specific nucleotide sequence called Restriction sites to generate a set of smaller fragments . Restriction enzymes form part of the restriction-modification system of bacterial cells that provides protection against invasion of the cell by foreign DNA – especially bacteriophage DNA. But the cells own DNA is not cleaved by these Restriction enzymes. This self protection is achieved by the help of the specific DNA methyltransferase enzyme which will methylates the specific DNA sequence for its respective restriction enzymes by transferring methyl groups to adenine or cytosine residues to produce N6-methyladenine or 5-methylcytosine.  An interesting feature of restriction endonuclease is that they commonly recognize recognition sequences that are mostly palindromes - they shows the same forward (5' to 3' on the top strand) and backward (5' to 3' on the bottom strand) sequences. In other words, they are nucleotide sequences or complimentary strands that read the same in opposite direction.
Materials Required
Microcentrifuge tubes
Vial stand
10µl pipette
Pipette tips
Beaker
Table top mini centrifuge
Incubator
Reagents
Procedure
Transfer the following solutions in a micro centrifuge tube.
Incubate the mixture at 37 o C for 1 h to overnight. Keep the tubes in -4o C freezer or in -20o C freezer, after the incubation.
Precaution
Make sure that the restriction enzyme does not exceed more than 10% of the total reaction volume, Otherwise the glycerol and the EDTA in the enzyme storage buffer may inhibit digestion process.
Differences Encountered in Real Laboratory
After performing the experiment, confirm the Digestion of DNA by running  a small amount of it in agarose gel with an undigested standard DNA.
Some restriction enzymes require BSA. In such cases make sure that, it is added to the reaction mixture. Restriction enzymes that do not require BSA for optimal activity are not adversely affected if BSA is present in the reaction.
Before performing the experiment, check whether the restriction enzymes have star activity or not.
Objectives
To understand the basic procedures involved in the isolation process of DNA from various sources such as blood, tissue, bacteria etc.
Requirements for Plasmid Isolation:
Micro centrifuge.
Water bath (37°C).
Automatic micropipettes with tips.
95-100% isopropanol Ice.
Buffers and Solutions:
1. Alkaline lysis solution I.
2.Alkaline lysis solution II.
3.Alkaline lysis solution III.
4.Antibiotic for plasmid selection.
5.Ethanol.
6.Phenol: chloroform (1:1, v/v).
7.STE.
8.TE (pH 8.0) containing 20 μg/ml RNAse A.
Media:
1.Rich medium.
Procedure:
1. Inoculate 2 ml of rich medium (LB, YT, or Terrific Broth) containing the appropriate antibiotic with a single colony of transformed bacteria. Incubate the culture overnight at 37°C with vigorous shaking.
2. Pour 1.5 ml of the culture into a microfuge tube. Centrifuge at maximum speed for 30 seconds at 4°C in a microfuge. Store the unused portion of the original culture at 4°C.
3. Remove the medium by aspiration, leaving the bacterial pellet as dry as possible.
4. Resuspend the bacterial pellet in 100 μl of ice-cold Alkaline lysis solution I by vigorous vortexing.
5. Add 200 μl of freshly prepared Alkaline lysis solution II to each bacterial suspension. Close the tube tightly, and mix the contents well by inverting the tube . Do not vortex! Store the tube in ice.
6. Add 150 μl of ice-cold Alkaline lysis solution III. Close the tube and disperse Alkaline lysis solution III through the viscous bacterial lysate by inverting the tube several times. Store the tube in ice for 3-5 minutes.
7. Centrifuge the bacterial lysate for 5 minutes at maximum speed at 4°C in a microfuge. Collect the supernatant to a fresh tube.
8. (Optional) Add equal volume of phenol: chloroform. Mix the organic and aqueous phases by vortexing and then centrifuge the emulsion at maximum speed for 2 minutes at 4°C in a microfuge. Transfer the aqueous upper layer to a fresh tube.
9. Precipitate nucleic acids from the supernatant by adding 2 volumes of ethanol at room temperature. Mix the solution by vortexing and then allow the mixture to stand for 2 minutes at room temperature.
10. Collect the precipitated nucleic acids by centrifugation at maximum speed for 5 minutes at 4°C in a microfuge.
11. Discard the supernatant by  aspiration. Stand the tube in an inverted position on a paper towel to allow all of the fluid to drain away. Use a Kim wipe or disposable pipette tip to remove any drops of fluid adhering to the walls of the tube.
12. Add 1 ml of 70% ethanol to the pellet and invert the closed tube several times. Recover the DNA by centrifugation at maximum speed for 2 minutes at 4°C in a microfuge.
13. Remove all of the supernatant by aspiration. Take care with this step, as the pellet sometimes does not adhere tightly to the tube.
14. Remove any beads of ethanol from the tube. Store the open tube at room temperature until the ethanol has evaporated and no fluid is visible in the tube (5-10 minutes).
15. Dissolve the nucleic acids in 50 μl of TE (pH 8.0) containing 20 μg/ml DNase-free RNase A (pancreatic RNase). Vortex the solution gently for a few seconds and store the DNA  at -20°C.
Recipes for Buffers, Solutions and Media:
Alkaline Lysis Solution I :
50 mM glucose.
25 mM Tris-Cl (pH 8.0).
10 mM EDTA (pH 8.0).
Prepare Solution I from standard stocks in batches of approx. 100 ml, sterilize by autoclaving and store at 4°C. (For plasmid preparation.)
Alkaline Lysis Solution II:
0.2 N NaOH (freshly diluted from a 10 N stock).
1% (w/v) SDS.
Prepare Solution II fresh and use at room temperature. (For plasmid preparation.)
Alkaline Lysis Solution III:
5 M potassium acetate, 60.0 ml.
Glacial acetic acid, 11.5 ml.
H2O, 28.5 ml.
The resulting solution is 3 M with respect to potassium and 5 M with respect to acetate. Store the solution at 4°C and transfer it to an ice bucket just before use.
(For plasmid preparation.)
EDTA:
To prepare 0.5 M  EDTA  (pH 8.0):  Dissolve 186.1 g of disodium EDTA•2H2O in 800 ml of Distilled 2H2O. Stir well on a magnetic stirrer.  EDTA will not dissolve  into solution until the pH of the solution is reached to ~ 8.0 . So  the pH should adjust to 8.0 with NaOH (~ 20 g of NaOH pellets) and make up the final volume to 1000ml with distilled water. Prepare the  aliquots and sterilize by autoclaving.
Glycerol:
To prepare a 10% (v/v) solution: Dilute 1 volume of molecular-biology grade glycerol in 9 volumes of sterile pure H2O. Sterilize the solution by passing it through a pre rinsed 0.22-μm filter. Store in 200-ml aliquots at 4°C.
LB Media
Deionized H2O, to 950 ml.
Tryptone, 10 g.
Yeast extract, 5 g.
NaCl, 10 g.
To prepare LB (Luria-Bertani) medium, shake the ingredients , mentioned above with Distilled water until the solutes have dissolved. Adjust pH to 7.0 with 5 N NaOH and make up the final volume of the solution to 1 liter with deionized H2O. Then sterilize it for 20 minutes  by autoclaving  at 15 psi .
NaCl:
To prepare 5 M  NaCl : Dissolve 292 g of NaCl in 800 ml of  sterile H2O and the volume is make up to  to 1 liter with deionized H2O. Prepare the aliquots and sterilize it by autoclaving.
NaOH:
To 800 ml of H2O,  add 400g of NaOH pellets slowly, stirring continuously.  After  dissolving the pellets,completely, make up the final volume to 1 liter with sterile H2O. Store the solution at room temperature.
Potassium Acetate:
5 M potassium acetate, 60 ml.
Glacial acetic acid, 11.5 ml.
H2O, 28.5 ml.
The resulting solution is 3 M with respect to potassium and 5 M with respect to acetate. Store the solution at room temperature
SDS:
Also called sodium lauryl sulfate. To prepare a 20% (w/v) solution, dissolve 200 g of  SDS in 900 ml of H2O. Heat to a temperature of  68°C and stir with a magnetic stirrer to help dissolution.  Adjust the volume to 1 liter with  distilled H2O. Store at room temperature. Autoclaving not necessary.
STE:
10 mM Tris-Cl (pH 8.0).
0.1 M NaCl.
1 mM EDTA (pH 8.0).
Sterilize the solution by autoclaving and store at 4°C.
TE:
100 mM Tris-Cl (desired pH).
10 mM EDTA (pH 8.0).
(10x Tris EDTA) Sterilize the buffer by autoclaving and store at room temperature.
Tris-Cl:
Dissolve 121.1 g of Tris base in 800 ml of H2O. Adjust the pH  by adding concentrated HCl , to the desired value.The volume of the solution is make up to 1 liter with distilled H2O. Prepare the aliquots and sterilize by autoclaving.
Procedure for Operating the Virtual Lab:
Check whether you have done all the steps listed below:
• Prepare the culture containing the desired plasmid.
• Incubate the culture for 24 hours at 37°C.
• Take the culture from the incubator.
• Transfer 1.5ml of the culture to a microfuge tube.
• Centrifuge the tube for 30seconds at maximum speed (4°C).
• Remove the supernatant.
• Add 100μl alkaline lysis solution I.
• Vortex the sample.
• Add 200μl of alkaline lysis solution II.
• Mix the sample by inverting the tube.
• Store in ice for 1 minute.
•  Add alkaline lysis solution III.
• Mix the contents by inverting the tube.
• Store in ice for 3-5 minutes.
• Centrifuge the solution at maximum speed (4°C) for 5 minutes .
• Collect  the supernatant to a fresh tube.
• Precipitate the nucleic acid by adding 2 volumes of ethanol.
• Mix by vortexing.
• Stand the tubes for 2 minutes.
• Centrifuge for 5 minutes.
• Collect the precipitated DNA.
• Discard the supernatant by aspiration.
• Stand the tube as inverted to drain the fluid away.
• Add 1ml 70% ethanol.
• Mix by inverting.
• Centrifuge the mixture for 2 minutes.
• Discard the supernatant by aspiration.
• Allow to dry for 3-5 minutes.
• Add TE buffer with RNAse.
• Mix by flickering.
• Detect the plasmid by doing agarose gel electrophoresis.
Objective
To separate the DNA fragments based on their Molecular weight.
Materials Required:
Buffers and Solutions:
Agarose solutions.
Ethidium bromide.
Electrophoresis buffer.
Nucleic Acids and Oligonucleotides:
DNA samples.
DNA Ladders.
(Samples of DNAs of known size are typically generated by restriction enzyme digestion of a plasmid or bacteriophage DNA of known sequence).
The equipment and supplies necessary for conducting agarose gel electrophoresis are relatively simple and include:
• An electrophoresis chamber and power supply.
• Gel casting trays, which are available in a variety of sizes and composed of UV-transparent plastic.
• Sample combs, around which molten agarose is poured to form sample wells in the gel.
• Electrophoresis buffer, usually Tris-acetate-EDTA (TAE) or Tris-borate-EDTA (TBE).
• Loading buffer, which contains something dense (e.g. glycerol) to allow the sample to "fall" into the sample wells, and one or two tracking dyes, which migrate in the gel and allow visual monitoring or how far the electrophoresis has proceeded.
• Ethidium bromide, a fluorescent dye used for staining nucleic acids.
• Transilluminator (an ultraviolet light box), which is used to visualize ethidium bromide-stained DNA in gels.
NOTE: Always wear protective eyewear when observing DNA on a Transilluminator to prevent damage to the eyes from UV light.
1.  Prepare a 50x stock solution of TAE buffer in 1000m of distilled H2O:
For this weigh 242 g of Tris base in a chemical balance. Transfer this to a 1000ml beaker.
Prepare EDTA solution (pH 8.0, 0.5M) by weighing 9.31g of EDTA and dissolve it in 40ml distilled water. EDTA is insoluble and it can be made soluble by adding sodium hydroxide pellets. Check the pH using pH meter. Make the solution 100ml by adding distilled water.
Pipette out 57.1 ml of glacial acetic acid.
Mix the Tris base, EDTA solution and glacial acetic acid and add distilled water to make the volume to 1000ml
2.  Prepare sufficient electrophoresis buffer (usually 1x TAE ) to fill the electrophoresis tank and to cast the gel:
For this we take 2ml of TAE stock solution in an Erlenmeyer flask and make the volume to 100ml by adding 98ml of distilled water. The 1x working solution is 40 mM Tris-acetate/1 mM EDTA
It is important to use the same batch of electrophoresis buffer in both the electrophoresis tank and the gel preparation.
3. Prepare a solution of agarose in electrophoresis buffer at an appropriate concentration:
For this usually 2 grams of agarose is added to 100ml of electrophoresis buffer.
Agarose Concentration in Gel (% [w/v]) Range of Separation of Linear DNA Molecules (kb)
0.3 5-60
0.6 1-20
0.7 0.8-10
0.9 0.5-7
1.2 0.4-6
1.5 0-2-3
2.0 0.1-2
4. Loosely plug the neck of the Erlenmeyer flask. Heat the slurry in a boiling-water bath or a microwave oven until the agarose dissolves. The agarose solution can boil over very easily so keep checking it. It is good to stop it after 45 seconds and give it a swirl. It can become superheated and NOT boil until you take it out whereupon it boils out all over you hands. So wear gloves and hold it at arm's length. You can use a Bunsen burner instead of a microwave
- just remember to keep watching it.
5. Use insulated gloves or tongs to transfer the flask/bottle into a water bath at 55°C. When the molten gel has cooled, add 0.5µg/ml of ethidium bromide. Mix the gel solution thoroughly by gentle swirling.
(For the preparation of ethidium bromide adds 1 g of ethidium bromide to 100 ml of H2O. Stir on a magnetic stirrer for several hours to ensure that the dye has dissolved. Wrap the container in aluminum foil or transfer the 10 mg/ml solution to a dark bottle and store at room temperature.)
6.While the agarose solution is cooling, choose an appropriate comb for forming the sample slots in the gel.
7.Pour the warm agarose solution into the mold.
(The gel should be between 3 - 5 mm thick. Check that no air bubbles are under or between the teeth of the comb.)
8. Allow the gel to set completely (30-45 minutes at room temperature), then pour a small amount of electrophoresis buffer on the top of the gel, and carefully remove the comb. Pour off the electrophoresis buffer. Mount the gel in the electrophoresis tank.
9.Add just enough electrophoresis buffers to cover the gel to a depth of approx. 1mm.
10. Mix the samples of DNA with 0.20 volumes of the desired 6x gel-loading buffer.
11. Slowly load the sample mixture into the slots of the submerged gel using a disposable micropipette or an automatic micropipettor or a drawn-out Pasteur pipette or a glass capillary tube. Load size standards into slots on both the right and left sides of the gel.
12. Close the lid of the gel tank and attach the electrical leads so that the DNA will migrate toward the positive anode (red lead). Apply a voltage of 1-5 V/cm (measured as the distance between the positive and negative electrodes). If the electrodes are 10cm apart then run the gel at 50V. It is fine to run the gel slower than this but do not run it any faster. Above 5V/cm the agarose may heat up and begin to melt with disastrous effects on your gel's resolution. If the leads have been attached correctly, bubbles should be generated at the anode and cathode.
13. Run the gel until the bromophenol blue and xylenecyanol FF have migrated an appropriate distance through the gel.
(The presence of ethidium bromide allows the gel to be examined by UV illumination at any stage during electrophoresis).
14.  The gel tray may be removed and placed directly on a transilluminator. When the UV is switched on we can see orange bands of DNA.
Procedure for operating the virtual lab:
Check whether you have done all the steps listed below:
• Prepare TAE buffer.
• Transfer 100ml of the buffer to a conical flask.
• Weigh 2grams of agarose and add to the 100ml buffer solution.
• Keep in oven.
• Take the solution from oven.
• Add ethidium bromide.
• Pour the solution to a gel caster.
• Place the comb.
• Pour the 100ml buffer solution to the electrophoretic chamber.
• Place the gel in the caster in the electrophoretic chamber.
• Connect the electrodes and switch on the current.
• Switch off the power supply.
• Remove the gel from the electrophoretic chamber.
• Place the gel in the UV Transilluminator.
• Switch on the Transilluminator.
CAUTION:
• Ethidium bromide is a mutagen and should be handled as a hazardous chemical  (so wear gloves while handling)
DIFFERENCES ENCOUNTERED IN REAL LABORATORY:
1.  Make sure that the Agarose is fully dissolved in the buffer. If it is not dissolved well, again melt it some more time to dissolve completely.
2.  Before casting the gel, the tray and comb should wipe with ethanol.
3.  Make sure that the gel in the Chamber is immersed in the TAE Buffer.
4.  Labelings should be  proper.
5.  Ensure that the connections should be proper.
6.  Before the  incubation step, ensure that the water bath is set at the correct temperature that we required or not.
Objective:
To familiarize with how cells are made competent  which is the primary step for transformation.
Materials Required:-
•  LB broth
•  Culture plates
•  Ice cold CaCl2.2H2O  (1  M)
•  Ice cold MgCl2 CaCl2  solution
•  Shaking incubator
•  Vortex mixer
•  Centrifuge
•  Water  bath
•  Inoculation loop
•  Microfuge tubes
•  Polypropylene  tubes
•  Micro pipettes and tips
Procedure:-
1. Pick a single bacterial colony  from a culture plate which is incubated for 16-20 hours at 37°C. Transfer the colony into 100 ml LB broth a 1-liter flask. Incubate the culture for 3 hours at 37°C with vigorous agitation, monitoring the growth of the culture. As a guideline, 1 OD600 of a culture of E. coli strain DH 5 alpha contains approx. 109 bacteria/ml.
2. Transfer the bacterial cells to sterile, disposable, ice-cold 50ml polypropylene tubes. Cool the cultures to 0°C by storing them on ice for 10 minutes.
3. Recover the cells by centrifugation at 2700g  at 4°C for 10 minutes .
4. Decant the medium from the cell pellets. Stand the tubes in an inverted position on a pad of paper towels for
1 minute to allow the last traces of media to drain away.
5. Resuspend each pellet by swirling or gentle vortexing in 30 ml of ice-cold MgCl2-CaCl2 solution.
6. Recover the cells by centrifugation at 2700g  at 4°C for 10 minutes .
7. Decant the medium from the cell pellets. Stand the tubes in an inverted position on a pad of paper towels for 1 minute to allow the last traces of media to drain away.
8. Resuspend the pellet in  2 ml of ice-cold 0.1 M CaCl2 (or TFB) by gentle vortexing for each 50ml of original culture. Standard TFB may be used instead of CaCl2 for most strains of E. coli.
9. At this point, either use the cells directly for transformation or dispense into aliquots and freeze at -70°C.
Objectives
1)    To develop an understanding about the transformation.
2)    Transform bacteria cells with a foreign DNA.
Materials Required
•   LB broth
•   LA- Amp  plate
•   Micropipette and tips
•   Incubator
•   Water bath
•   Microfuge tube
•   Calcium chloride treated competent cells
•   L-rod
Reagents Required
•  X Gal:   Make a 2% (w/v)  stock solution by dissolving X-gal in dimethylformamide at a concentration of 20 mg/ml solution. The X-gal tube should  wrap with  aluminium foil in order to prevent the damage caused by light and store at -20°C.
• IPTG:   Make a 20% (w/v, 0.8 M) solution of IPTG by dissolving 2 g of IPTG with  8 ml of distilled H2O. Prepare the aliquots and store them at -20°C.
Procedure
1. To transform the CaCl2- treated cells directly, transfer 200 µl of each suspension of competent cells to a sterile, chilled polypropylene   tube  using a chilled micropipette tip.
2. Add DNA (<50ng in a volume of 10 µl) to each tube. Mix the contents of the tubes by  gentle swirling. Keep the tubes in ice for 30 minutes.
3. Transfer the tubes to rack placed in a preheated 42 ºC circulating water bath. Keep the tubes in  rack for 90 seconds.
4. Transfer the tubes to an ice bath immediately. Allow the cells to chill for 1-2 minutes.
5. Add 800 µl of LB medium to each tube. Incubate the cultures for 45 minutes in a water bath at 37 ºC .
6. After incubation, add 40 µl of X –Gal and 7 µl of IPTG to the LA-Amp plate.
7. Add appropriate volume  of transformed competent cells into the plate.
8. Spread  all the contents uniformly using an L-rod. Keep the plates at room temperature until the liquid has been absorbed.
9. Invert the plates and keep for incubation at 37 ºC. Transformed colonies will  appear in 12-16 hours of incubation.
Differences Encountered in Real Laboratory
1. After taking the competent cells from the freezer, it should thaw at room temperature.
2. There should not be much time lag between adding and spreading of EDTA ,Beta-gal and Transformed cells. If so, the components will not spread uniformly in the plate.
3. Make sure that the samples are uniformly spread in to the plate(should care the spreading).
4. Make sure that the plates with transformed cells should be in inverted position while incubation. Objective
To extract specific bands of  DNA from agarose gels in which they are  separated through electrophoresis.
Materials Required
• Elution buffer
• Scalpel blade
• UV transilluminator
• Agarose
• Micro pipettes
• Micro pipette tips
• Dry bath incubator
• Microfuge tubes
• Centrifuge
• N-Butanol
• Cryo box
• Cyclomixer
• 70% Ethanol
• 95% Ethanol
• TE buffer
• -20oC freezer
• -70oC freezer
Procedure
1.    Visualize the low melting point agarose gel with DNA bands under a UV transilluminator and locate the desired DNA band to cut.
2.    Carefully cut around the desired DNA band using a scalpel blade.
3.    Transfer the gel piece into a microfuge tube.
4.    Add elution buffer into the microfuge tube until the level of buffer is just above the level of gel slice.
5.    Heat the gel slice at 65oC until it melts.
6.    Freeze the melted gel with DNA by placing in a -70oC freezer for10minuts.
7.    After freezing, centrifuge for 10minutes and transfer the supernatant into a new microfuge tube.
8.    Again add half amount of elution buffer that you added in the previous step into the pellet.
9.    Heat at 65oC until the agarose melts.
10.   Freeze the melted gel with DNA by placing in a -70oC freezer for10minuts.
11.   Centrifuge the tube again for 10 minutes and transfer (pool) the supernatant into the previous tube with supernatant.
12.   Discard the tube with pellet.
13.   Add an equal volume of n-Butanol to the supernatant and mix the contents well.
14.   Vortex the tube for 15 minutes in order to remove the Ethidium bromide.
15.   Discard the upper phase of butanol and repeat the process by adding n-butanol again for one or more times.
16.   Add 2 times volume of 95% ethanol and mix thoroughly.
17.   Keep for precipitation in -70oC freezer for 30minutes to overnight.
18.   After precipitation, centrifuge for 15 minutes.
19.   Discard the supernatant into a waste beaker and add 200µl of 70% ethanol to the pellet.
20.   Centrifuge for 5minutes and discard the supernatant again.
21.   Allow the pellets to dry well.
22.   Suspend the pellets in 20µl of TE buffer. (If you want to confirm the recovered DNA, run (1µl) it on a gel.
23.   The recovered DNA can be now used for further process of cloning otherwise can stored in -20oC freezer.
Objective:-
To perform ligation reaction using T4 DNA ligase.
Materials Required:-
• Vials and Vial stand
• Micro pipette
• pipette  tips
• Table top mini centrifuge
• Beaker
• Ice box
Reagents:-
Procedure:-
1. Take one clean fresh microfuge tube(Sample) from the rack.
2. In this microfuge tube(Vial), add 5µL water, 1µL Vector, 2.5µL insert and 1µL T4 DNA ligase buffer.
3. 0.5 µL of T4 DNA Ligase enzyme was added to the sample tube ( total reaction volume is 10 µl).
4. The vial is kept in the micro centrifuge and just spin for a few seconds.
5. Incubate the vial at room temperature (22˚C ) for 2 hours.
6. After 2 hours, the ligated mixture is taken for doing transformation.
1.REAL LAB VERSUS VIRTUAL LAB:
1.    T4 DNA Ligase Buffer contains ATP, so repeated freeze thaw cycles can degrade ATP, thereby decreasing the efficiency of Ligation.
2.    It is better to vortex or spin  the T4 DNA ligase enzyme before pipetting  to ensure that it is mixed well.
Objectives
To amplify a given region of DNA(region of interest).
Procedure:
The protocol describes how to amplify a segment of double-stranded DNA in a chain reaction catalyzed by a thermostable DNA polymerase. It is the foundation for all subsequent variations of the polymerase chain reaction.
Materials
Buffers and Solutions
10x Amplification buffer
Chloroform
dNTP solution (20 mM) containing all four dNTPs (pH 8.0)
Enzymes and Buffers
Thermostable DNA polymerase
Nucleic Acids and Oligonucleotides
Forward primer (20 μM) in H2O
Reverse primer (20 μM) in H2O
Template DNA.
Dissolve template DNA in 10 mM Tris-Cl (pH 7.6) containing a low concentration of EDTA (<0.1 mM) at the following concentrations: mammalian genomic DNA, 100 μg/ml; yeast genomic DNA, 1 μg/ml; bacterial genomic DNA, 0.1 μg/ml; and plasmid DNA, 1-5 ng/ml.
Method
1. In a sterile 0.5-ml microfuge tube, mix in the following order:
REAGENTS AMOUNT(μl)
Deionized water 37.5 μl
Taq assay buffer(10x) 5 μl
Template DNA 1μl
dNTPs mix 2 μl
Forward primer 2 μl
Reverse primer 2 μl
Taq DNA polymerase 5 μl
The table below provides standard reaction conditions for PCR. Mg2+ (1.5 mM) ;KCl(50 mM) ;dNTPs (200 μM) ;Primers(1 μM );DNA polymerase (1-5 units); Template DNA(1 pg to 1 μg ).
The amount of template DNA required varies according to the complexity of its sequence. In the case of mammalian DNA, up to 1.0 μg is used per reaction. Typical amounts of yeast, bacterial, and plasmid DNAs used per reaction are 10 ng,
1 ng, and 10 pg, respectively.
2.If the thermal cycler is not fitted with a heated lid, overlay the reaction mixtures with 1 drop (approx. 50 μl) of light mineral oil. Alternatively, place a bead of wax into the tube if using a hot start protocol. Place the tubes or the micro titer plate in the thermal cycler.
3.Amplify the nucleic acids using the denaturation, annealing, and polymerization times and temperatures listed below.
4.Withdraw a sample (5-10 μl) from the test reaction mixture and the four control reactions, analyze them by electrophoresis through an agarose gel, and stain the gel with ethidium bromide or SYBR Gold to visualize the DNA.
A successful amplification reaction should yield a readily visible DNA fragment of the expected size. The identity of the band can be confirmed by DNA sequencing, Southern hybridization and/or restriction mapping. If all is well, lanes of the gel containing samples of the two positive controls (Tubes 1& 2) and the template DNA under test should contain a prominent band of DNA of the appropriate molecular weight. This band should be absent from the lanes containing samples of the negative controls (Tubes 3 & 4).
5.  If mineral oil was used to overlay the reaction (Step 2), remove the oil from the sample by extraction with 150 μl of chloroform. The aqueous phase, which contains the amplified DNA, will form a micelle near the meniscus. The micelle can be transferred to a fresh tube with an automatic micropipette.
Do not attempt chloroform extractions in micro titer plates. The plastic used in these plates is not resistant to organic solvents.
Recipes
Amplification Buffer:
500 mM KCl.
100 mM Tris-Cl (pH 8.3 at room temperature).
15 mM MgCl2.
Autoclave the 10x buffer for 10 minutes at 15 psi (1.05 kg/cm2) on liquid cycle. Divide the sterile buffer into aliquots and store them at -20oC.
KCl
Dissolve an appropriate amount of solid KCl in H2O, autoclave for 20 minutes on liquid cycle and store at room temperature. Ideally, this 4 M solution should be divided into small (approx. 100 μl) aliquots in sterile tubes and each aliquot thereafter used one time.
Tris-Cl
Dissolve 121.1 g of Tris base in 800 ml of H2O. Adjust the pH to the desired value by adding concentrated HCl.
pH HCl
7.4 70 ml
7.6 60 ml
8.0 42 ml
(1 M) Allow the solution to cool to room temperature before making final adjustments to the pH. Adjust the volume of the solution to 1 litre with H2O. Dispense into aliquots and sterilize by autoclaving. If the 1 M solution has a yellow color, discard it and obtain Tris of better quality. The pH of Tris solutions is temperature-dependent and decreases approx. 0.03 pH units for each 1oC increase in temperature. For example, a 0.05 M solution has pH values of 9.5, 8.9, and 8.6 at 5oC, 25oC, and 37oC, respectively.
dNTP Solution
Dissolve each dNTP (deoxyribonucleoside triphosphates) in H2O at an approximate concentration of 100 mM. Use 0.05 M Tris base and a micropipette to adjust the pH of each of the solutions to 7.0 (use pH paper to check the pH). Dilute an aliquot of the neutralized dNTP appropriately, and read the optical density at the wavelengths given in the table below. Calculate the actual concentration of each dNTP. Dilute the solutions with H2O to a final concentration of 50 mM dNTP. Store each separately at 70oC in small aliquots. For polymerase chain reactions (PCRs), adjust the dNTP solution to pH 8.0 with 2 N NaOH. Commercially available solutions of PCR-grade dNTPs require no adjustment.
Base wavelength(nm) Extinction Coefficient(E) (M-1cm-1)
A 259 1.54 x 104
G 253 1.37 x 104
C 271 9.10 x 103
T 267 9.60 x 103
For a cuvette with a path length of 1 cm, absorbance = EM. 100 mM stock solutions of each dNTP are commercially available .
Precautions
Chloroform
Chloroform CHCl3 is irritating to the skin, eyes, mucous membranes, and respiratory tract. It is a carcinogen and may damage the liver and kidneys. It is also volatile.  Avoid breathing the vapours.  Wear appropriate gloves and safety glasses. Always wear a chemical fume hood.
PREPARATION OF GENOMIC DNA FROM BACTERIA
Solutions required for this protocol
• TE buffer
• 10% (w/v) sodium dodecyl sulfate (SDS)
• 20 mg/ml proteinase K
• Phenol/chloroform
• Isopropanol
• 70% ethanol
• 3M sodium acetate ph 5.2
• Phase Lock geltm (5 Prime, 3 Prime, Inc)
1. Transfer 1.5 ml to a micro centrifuge tube and spin 2 min. Decant the supernatant. Drain well onto a Kimwipe.
2. Resuspend the pellet in 467 µl TE buffer by repeated pipetting. Add 30 µl of 10% SDS and 3 µl of 20 mg/ml proteinase K, mix, and incubate 1 hr at 37°C.
3. Add an equal volume of phenol/chloroform and mix well by inverting the tube until the phases are completely mixed. CAUTION: PHENOL CAUSES SEVERE BURNS, WEAR GLOVES GOGGLES, AND LAB COAT AND KEEP TUBES CAPPED TIGHTLY. Carefully transfer the DNA/phenol mixture into a Phase Lock GelTM tube (green) and spin 2 min.
4. Transfer the upper aqueous phase to a new tube and add an equal volume of phenol/chloroform. Again mix well and transfer to a new Phase Lock GelTM tube and spin 2 min. Transfer the upper aqueous phase to a new tube.
5. Add 1/10 volume of sodium acetate.
6. Add 0.6 volumes of isopropanol and mix gently until the DNA precipitates.
7. Spool DNA onto a glass rod (or Pasteur pipet with a heat-sealed end).  Wash DNA by dipping end of rod into 1 ml of 70% ethanol for 30 sec.  Resuspend DNA in 100-200 µl TE buffer.
8. After DNA has dissolved, measure the concentration by diluting 10 µl of DNA into 1 ml of TE (1:100 dilution) and measure absorbance at 260 nm.
Concentration of original DNA solution in µg/ml = Abs x 100 x 50 µg/ml.
Protocol of Genomic DNA Isolation from a Gram-Negative Bacterium
The G NOME® kit ( http://www.qbiogene.com ) is used to quickly and efficiently isolate high molecular weight genomic DNA from Gram-negative bacterial cultures. Each preparation with the G NOME® kit yields up to 100 mg of genomic DNA.
The DNA isolated by the G NOME® procedure is suitable for restriction enzyme digestion or PCR amplification in as little as 1 hour after cell lysis.
Protocol
1. Bring cells*/tissues to a final volume of 1.85ml in Cell Suspension Solution. (Use a 15 ml clear plastic tube for efficient mixing). Mix until the solution appears homogeneous.
2. Add 50ml of RNase Mixx, mix thoroughly.
3. Add 100ml of Cell Lysis/Denaturing Solution**, mix well.
4. Incubate at 55°C for 15 minutes.
5. Add 25ml Protease Mixx, mix thoroughly. (Note: If precipitate is visible in Protease Mixx suspension, pulse spin and use 25 ml of supernatant.)
6. Incubate at 55°C for 30 to 120 minutes. (The longer time will result in minimal protein carry over and will also allow for substantial reduction in residual protease activity.)
7. Add 500ml “Salt-Out” Mixture, mix gently yet thoroughly. Divide sample into 1.5ml tubes. Refrigerate at 4°C for 10 minutes.
8. Spin for 10 minutes at maximum speed in a microcentrifuge (at least 10,000 x g). Carefully collect the supernatant, avoid the pellet. If a precipitate remains in the supernatant, spin again until it is clear. Pool the supernatants in a 15 ml (or larger) clear plastic tube.
9. To this supernatant, add 2 ml TE buffer and mix. Then add 8mls of 100% ethanol. If spooling the DNA, add the ethanol slowly and spool the DNA at the interphase with a clean glass rod. If centrifuging the DNA, add the ethanol and gently mix the solution by inverting the tube. Spin for 15 minutes at 1000-1500xg. Eliminate the excess ethanol by blotting or air drying the DNA.
10. Dissolve the genomic DNA in TE (10mM Tris pH 7.5, 1mM EDTA).
Experiment 1 : Plasmid DNA Preparation
Objective: Isolation of plasmid DNA by Alkaline Lysis with SDS: Minipreparation
Principle: Bacterial plasmids are self replicating, circular extra chromosomal, DNA molecules. The most convenient method for preparing plasmid DNA is alkaline lysis method. In this procedure, cells are lysed by SDS at high pH and then neutralized. The plasmid DNA reanneals rapidly while most of the chromosomal DNA and bacterial proteins precipitate as a protein- DNA-SDS complex.
Materials Required:
Reagents and Buffers
Alkaline Lysis Solution I
50 mM glucose
25 mM Tris-Cl (pH 8.0)
10 mM EDTA (pH 8.0)
Prepare Solution I from standard stocks in batches of approx. 100 ml, autoclave for 15 minutes at 15 psi (1.05 kg/cm2) on liquid cycle, and store at 4°C.
Alkaline Lysis Solution II
0.2 N NaOH (freshly diluted from a 10 N stock)
1% (w/v) SDS
Prepare Solution II fresh and use at room temperature.
Alkaline Lysis Solution III
5 M potassium acetate, 60.0 ml
Glacial acetic acid, 11.5 ml
H2O, 28.5 ml
The resulting solution is 3 M with respect to potassium and 5 M with respect to acetate. Store the solution at 4°C and transfer it to an ice bucket just before use.
Alkaline Lysis Solution IV
Isopropanol
Step-wise Methodology:
1. Inoculate 2 ml of rich medium (LB, YT, or Terrific Broth) containing the appropriate antibiotic with a single colony of transformed bacteria. Incubate the culture overnight at 37°C with vigorous shaking.
2. Pour 1.5 ml of the culture into a microfuge tube. Centrifuge at maximum speed for 30 seconds at 4°C in a microfuge. Store the unused portion of the original culture at 4°C.
3. Remove the medium by aspiration, leaving the bacterial pellet as dry as possible.
4. Resuspend the bacterial pellet in 100 μl of ice-cold alkaline lysis solution I by vigorous vortexing.
5. Add 200 μl of freshly prepared alkaline lysis solution II to each bacterial suspension. Close the tube tightly, and mix the contents by inverting the tube rapidly five times. Do not vortex! Store the tube on ice.
6. Add 150 μl of ice-cold alkaline lysis solution III. Close the tube and disperse alkaline lysis solution III through the viscous bacterial lysate by inverting the tube several times. Store the tube on ice for 3-5 minutes.
7. Centrifuge the bacterial lysate at maximum speed for 5 minutes at 4°C in a microfuge. Transfer the supernatant to a fresh tube.
8. (Optional) Add an equal volume of phenol:chloroform. Mix the organic and aqueous phases by vortexing and then centrifuge the emulsion at maximum speed for 2 minutes at 4°C in a microfuge. Transfer the aqueous upper layer to a fresh tube.
9. Precipitate nucleic acids from the supernatant by adding 2 volumes of ethanol at room temperature. Mix the solution by vortexing and then allow the mixture to stand for 2 minutes at room temperature.
10. Collect the precipitated nucleic acids by centrifugation at maximum speed for 5 minutes at 4°C in a microfuge.
11. Remove the supernatant by gentle aspiration as described in Step 3 above. Stand the tube in an inverted position on a paper towel to allow all of the fluid to drain away. Use a Kimwipe or disposable pipette tip to remove any drops of fluid adhering to the walls of the tube.
12. Add 1 ml of 70% ethanol to the pellet and invert the closed tube several times. Recover the DNA by centrifugation at maximum speed for 2 minutes at 4°C in a microfuge.
13. Remove all of the supernatant by gentle aspiration as described in Step 3.Take care with this step, as the pellet sometimes does not adhere tightly to the tube.
14. Remove any beads of ethanol that form on the sides of the tube. Store the open tube at room temperature until the ethanol has evaporated and no fluid is visible in the tube (5-10 minutes).
15. Dissolve the nucleic acids in 50 μl of TE (pH 8.0) containing 20 μg/ml DNase-free RNase A (pancreatic RNase). Vortex the solution gently for a few seconds. Store the DNA solution at -20°C.
Precautions:
• Dispose of used reagents according to local ordinances.
• Wear latex gloves and goggles during the procedure.
• The laboratory surfaces should be very clean during all procedures used in this activity.
• Use thoroughly clean instruments and glassware. Rinse all equipment with isopropyl alcohol or acetone.
• Ethanol is highly flammable; use caution.
Observations:  Plasmid DNA will migrate in 3 forms:  supercoiled closed circular (superciled) DNA, single-strand nicked open circle DNA, and linear DNA.  The exact mobilities of the 3 forms will vary based on conditions, but closed circular DNA will usually migrate faster than the other two because of its supercoiling.  This means that if the DNA band farthest from your well on your uncut plasmid sample is the largest, you have prepared a plasmid DNA sample without excessively damaging your DNA.  (Note that your gel may also contain a diffuse band of partially degraded RNA
– which will migrate faster than your DNA bands.)
Interpretations & Conclusions:
Experiment -2
Aim: Restriction of given plasmid or λ DNA with the restriction enzyme EcoRI and HindIIIand estimation of fragment size by comparison with 1KB ladder.
Principle: Restriction enzymes are group of enzymes isolated from bacterial species which recognize specific sequences in DNA and then cut the DNA to produce fragments, called restriction fragments. Each restriction enzyme recognizes specific sequences and cuts at specific sites Restriction enzymes play a very important role in the construction of recombinant DNA molecules, as is done in gene cloning experiments. Another application of restriction enzymes is to map the locations of restriction sites in DNA. Some Examples of restriction enzyme along with their, recognition sequence and cutting sites are given below.
Hind III       A|AGCTT
Bam HI     G|GATCC
Eco RI  G|AATTC
PstI  CTGCA|G
Eco RV GAT|ATC
Material required:  Sterile water distilled water, 10X restriction enzyme buffer, plasmid DNA or λ DNA
Procedure:
1. Add the following in a 1.5ml centrifuge tube: 15 µl Sterile distilled water, 2.0 µl, 10X enzyme buffer, 1.0 µl Restriction enzyme, 2.0 µl Plasmid DNA\ λ DNA.
2. Maintain enzyme tube and plasmid DNA on ice. Do not leave on the table.
3. Mix by tapping the tube with your finger.
4. Briefly centrifuge to remove bubbles (DNA will adhere surface and becomes inaccessible to the enzymes).
5. Incubate at 37 C in a water bath for 45 minutes.
6. If the DNA is to be used for another manipulation, Heat inactivates the restriction enzyme by incubating the tubes at 65 C for 15 minutes.
7. Load the restriction enzymes cut and uncut plasmid on an agarose gel. Electrophorase the samples at 50-100 V for 1-2 hrs
8. Record the no. of fragments and their size in restriction enzyme digested samples.
Precautions:
1. Use fresh tip after each pipetting in order to avoid cross contamination of the chemicals and if you are using enzyme twice, change the pipet tip.
2. Enzymes should always be carried in a -20 mini cooler and work quickly. Do not expose the enzyme to warm temperature any longer than necessary.
3. Always wear gloves while handling enzymes.
Observations
The EcoR I digest of λ DNA yields the following 6 discrete fragments (in base pairs): 21226*, 7421, 5804, 5643, 4878, 3530*
The Hind III digest of λ DNA yields the following 8 discrete fragments (in base pairs): 23130*, 9416, 6557, 4361*, 2322, 2027, 564, 125.
Aim-To insert the PCR product into T vector by TA-cloning
Principle
"TA cloning" is a popular method of cloning without the use of restriction enzymes; instead, the fragment to be cloned, is amplified with only Taq DNA polymerase as PCR product. TA Cloning exploits the terminal transferase activity of some DNA polymerases such as Taq polymerase. This enzyme adds a single, 3'-A overhang to each end of the PCR product. This makes it possible to clone this PCR product directly into a linearized cloning vector with single, 3'-T overhangs. The PCR products with dA overhang are mixed with this vector in high proportion. The complementary overhangs of "T" vector and PCR product will be ligated under the action of T4 DNA ligase.
Advantages
1. Eliminate any enzymatic modification of PCR product
2. Does no require the use of PCR primer tthat contain restriction sites
RBC T&A cloning kit (cat.no. RC001)
RBC TA cloning system is ideal for rapid cloning PCR product generated using a thermos table DNA polymerase, which
adds a single 3’dA nucleotide overhang. Following the ligation the mixture may be used directly to transform competent cell or purified to achieve a higher efficiency of transformation.
Product components
1. T&A (25ng/μl) cloning vector : 40μl
2. Control Insert DNA (10ng/μl) : 10μl
3. T4 DNA ligase(3U/μl) : 20μl
4. T4 DNA ligase Buffer A : 100μl
5. T4 DNA ligase Buffer B : 100μl
6. forward primer(M13-F)(10μM)(50μl)
7. reverse primer(M13-R)(10μM)(50μl)
Procedure:
1. Centrifuge T&A cloning vector and/or PCR DNA tubes to collect contents at the bottom of the tubes.
2. Vortex the ligation buffer vigorously before use.
3. Set up the following items as described below:
3. Mix the reactions by pipetting.
4. Incubate the reactions for 5 to 15 min at 22oC. Alternatively, if the maximum of transformants is required, incubate the reactions overnight at 4oC.
Suggestions
1. Avoid multiple freeze-thaw cycles and exposure to frequent temperature changes by making single-use aliquots of Ligase Buffer.
2. Pfu DNA polymerase possesses proofreading activity; it does not have the terminal transferase-like activity demonstrated by Taq DNA Polymerase. Ligation reactions using non-tailed amplified DNA resulted in no positive colonies.
Aim: Transformation of E.coli. DH5 α cells with recombinant T- vector.
Principal
Genetic transformation occurs when a host organism takes in foreign DNA and expresses the foreign gene. In this experiment, we will introduce recombinant T vector carrying genes for resistance to the antibiotic penicillin into competent cells of DH5 α which are otherwise sensitive to it. If the bacterial cells incorporate the foreign DNA, they will become penicillin resistant. The CaCl2 treated competent cell of E.coli and plasmid DNA is mixed and then briefly heated to 42C for a brief period of 45- 90 sec to give a heat shock to the cells. CaCI2treatment promotes binding of DNA to E. coli cell wall. It is taken up by the cell when temperature is raised to 42°C.  MATERIALS and Equipments: LB medium (Liq.), 1.5 ml centrifuge tube, micro tips, centrifuge, Water bath, Recombinant vector, LB plates containing penicillin and X-gall
Procedure
1. Take the competent cells stored at -80C and thaw on ice.
2. Add 10µl of recombinant T vector to the 200µl of competent cells. Mix the content by swirly the tube gently. Store the tube on ice for 30 minutes
3. Transfer the tubes to a rack placed in a preheated 42C circulating water bathwater bath for 90 seconds
4. Rapidly transfer the tubes on ice. Allow the cells to chill for 1-2 minutes
5. Add 800µl of LB medium. Incubate the culture for 2-3 hours at 37C.Transfer 200µl of heat shock given cells on LB agar plate 100ug/µl ampicillin and X-gal
6. Incubate the plates at 37C in inverted position. Transformed colonies should appear in 12-16 hours. µl
Our Title Sponsor :
Retrieved from " http://2012.igem.org/Team:JUIT-India/Protocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
