"
Page
Discussion
View source
History
teams
Log in
 
Team:KAIST Korea/Notebook Labnote/2012 8
From 2012.igem.org
KAIST Korea 2012 iGEM
HOME
Introduction
iGEM
TEAM
Official Team Profile
Members
Gallery
DATA
Description
Bio Bricks
PROJECT
Overview
Results
Future Plan
Modeling
NOTEBOOK
Labnote
Protocol
PLUS
Safety
Human Practice
Attribution
Sponsors
2012 KAIST Korea
Mail : kaist.igem.2012@gmail.com Twitter : twitter.com/KAIST_iGEM_2012 Facebook : www.facebook.com/KAISTiGEM2012
Notebook : Labnote-August
Labnote
August
August 1st 2012
PACKMAN
PCR amplification of Moth_1191, 1199 gene : decreasing annealing temperature down to 45℃
To make sure there’s no problem with unspecific binding or annealing temperature, we decreased annealing condition down to 45℃. Results
There was no band and it became certain that there are no problems with annealing procedure.
Induction of Moth_0109 gene (sampling) and running SDS-PAGE gel.
We induced Moth_0109 gene in pBAD/myc vector with different Arabinose conditions and temperature: 10mM, 30mM and 37℃, 30℃. And then, we run the sample into SDS-PAGE gel. The gel is stained with coomassie blue for 1 hour and de-stained for overnight.
PCR amplification of Moth_1191, 1199 gene with Gibson and TOPO primer
To see whether there are any problem with Gibson primers that we’ve designed, we did one more pcr amplification. Primers for Gibson assembly and TOPO cloning are mixed, and we could see any problems with forward or reverse primers. Results
Each lane indicates:
Moth_1191_1 : TOPO forward primer and Gibson reverse primer
Moth_1191_2 : Gibson forward primer and TOPO reverse primer
Moth_1199_1 : Gibson forward primer and TOPO reverse primer
Moth_1199_2 : TOPO forward primer and Gibson reverse primer
Correct size of band appeared when topo forward primer and Gibson reverse primer was used in amplification of both genes. Therefore, we can conclude that there is some unknown problem with forward primers.
Flip Flop
pPoC and pPoCpi OE PCR with changed primers
Results
→ Gel extracted
Back to the Calendar
August 2nd 2012
PACKMAN
Primer design for FDH derived from Candida boidinii mutant
Results
Discussion These primers are universial primer that can be used for insertion of this gene into both pTrcHis2A vector and pBAD/mycHisC vector.
Expression check of Moth_0109 gene on pBAD/mycHisC vector (Result)
For each samples of different induction conditions, we run SDS-PAGE gel and checked the presence of 59.94 kDa protein, formate tetrahydrofolate ligase. Results
Discussion The band of PAGE gel was not certain, so we
Flip Flop
pPoC and pPoCpi cloning - Digestion
Restriction enzyme digestion
Enzyme pre-mix / each reaction – 37℃ 1hr, 80℃ 20min
Digestion mixture – 37℃ 1hr, 80℃ 20min
Bxb1 integrase transformation
Synthesized gene arrived. Transformed into DH5α
Back to the Calendar
August 3rd 2012
Flip Flop
Bxb1 integrase-DH5α streaking
Discussions Synthesized gene was cloned into pGEM-B1 vector. It seems the vector concentration is too high so that too much colonies may appeared.
pPoC and pPoCpi cloning - Ligation
Ligation - 16℃ overnight
Transformed into DH5α chemically competent cell.
BBa_J04450 transformation into DH5α
For pSB1C3 vector amplification
Back to the Calendar
August 4th 2012
PACKMAN
1191 and 1199 forward primer design
We designed new forward primers for both 1191 and 1199 gene that are thought to solve problems appeared during PCR amplification. Results
Flip Flop
BBa_J04450 and Bxb1-pGEM colony picking and inoculation.
Made master plates and inoculated in 3mL of LB.
Back to the Calendar
August 5th 2012
No Special Event!
Back to the Calendar
August 6th 2012
Flip Flop
BBa_J04450 Plasmid miniprep
BBa_J04450(pSB1C3): 285.8ng/uL purity:2.10
Requested part arrived
Each parts were streaked on appropriate plates.
BBa_I11020
BBa_I11022
BBa_I11023
BBa_I11030
BBa_K137007
BBa_K137008
BBa_K137010
BBa_K112001
BBa_K112141
BBa_K112142
BBa_K381001(GFP) and BBa_K398005(pSB1C3) transformation
BBa_K381001(GFP): for GFP control BBa_K398005(pSB1C3): for pSB1C3 template
Back to the Calendar
August 7th 2012
PACKMAN
FDH gene PCR amplification
Results
Discussion We amplified 2 samples of formate dehydrogenase gene with the primers we designed. From gel electrophoresis picture, we can say that oligonucleotides with the proper size 1100bp are present in the sample.
Flip Flop
BBa_J04450(pSB1C3) enzyme cut check(single cut with EcoRI)
Results
Vector was double digested. The insert and vector sizes were correct
pPoC and pPoCpi OE PCR product nested PCR
Results
→ Gel extracted
BBa_K398005 Plasmid miniprep & single cut check with EcoRI
Results
Overlapping PCR
pPoC insert
HSTaq Pol MM : 25uL
DW : 1uL
templates : 22uL
primers : 2uL
pPoCpi insert
HSTaq Pol MM : 25uL
templates : 23.2uL
primers : 2uL
Results
→ Concentration after gel extraction was not sufficient for cloning
Overlapping PCR template preparation
Back to the Calendar
August 8th 2012
PACKMAN
FDH gene and pTrcHis2A, pBAD/mycHisC vector enzyme digestion
Procedure We mixed restriction enzymes and buffers with DNA samples as shown below.
After mixing the enzymes and buffer with the sample, we incubated the sample on 37℃ for 1 hour and then on 65℃ for 20minutes.
pTrcHis2A, pBAD/myc vector dephosphorylation
Procedure We mixed phosphatidase and buffer with DNA sample as shown below.
After mixing the enzyme and buffer with the sample, we incubated the sample on 37℃ for 1 hour 20minutes and then on 70℃ for 5minutes.
ligation of FDH gene with pTrcHis2A and pBAD/mycHisC vectors
Procedure We mixed ligase and buffer with DNA sample as shown below.
After mixing the enzyme and buffer with the sample, we incubated the sample on 16℃ for 16 hours.
Primer amplification of 1191 and 1199
Results
Discussion
Flip Flop
Overlapping PCR
pPoC insert
pPoCpi insert
HSTaq Pol MM : 25uL
DW : 5.2uL
templates : 19.8uL
primers : 2uL
pfu-x : 0.1uL
Results
→ Gel extraction
Overlapping PCR template preparation
Results
Back to the Calendar
August 9th 2012
PACKMAN
vector transformation into TOP10 cell
Transformation of pTrcHis2A vector into TOP10 cell was successful. So we did vector mini-preparation after the transformation. However, transformation of pBAD vector has failed, so we ligated the samples again. Results
This is gel picture to confirm that FDH is well inserted into TOP10 cells. 3 wells on the left are PCR product of
Flip Flop
Bxb1 integrase cloning into pTrcHis2a and pBADmycHisC
Bxb1 integrase: 1515bp Results
→ Gel extracted
Extracted OE PCR product cloning into pSB1C3
pSB1C3 was originated from BBa_K398005.Transformed into TOP10 Results Transformed colony was grown in LB media. No colony shows fluorescence. Discussions We think cloning failed.
Back to the Calendar
August 10th 2012
Flip Flop
Bxb1-pTrc, pBAD colony inoculation
Overlapping PCR
In this time, PCR was done first 18 cycles without any primer. After that, we add 1uL of primers. And then PCR was done 20 cycles more. Results
Discussions With this OE PCR conditions, yield was better than previous conditions.
pPoC, pPoCpi insert cloning into pSB1C3
pSB1C3 was derived from BBa_K398005
Digestion – 37℃ 1hr, 80℃ 20min
Dephosphorylation – 37℃ 30min, 80℃ 10min
Ligation – 37℃ 2hr
Ligated products were transformed into DH5α.
Transformed cell inoculation and colony PCR
Results
Discussions only pPoCpi construct cloning was completed
Back to the Calendar
August 11th 2012
PACKMAN
pTrcHis2A vector transformation into E.coli MG1655 cell
Procedure
pBAD/mycHisC-FDH transformation into TOP10
Flip Flop
Bxb1 pTrcHis2A, pBADmycHisC plasmid DNA mini-prep
Results single cut check
pPoC and pPoCpi nested PCR
Results nested PCR to prepare more insert DNA
Back to the Calendar
August 12th 2012
PACKMAN
colony PCR check of pTrcHis2A inserted FDH gene
Results
Discussion We checked the presence of FDH gene by conducting colony PCR. Correct size of band appeared at about 1115bp, which means formate dehydrogenase is inserted correctly.
pBAD FDH MG1655 electroporation
Results Fail
Back to the Calendar
August 13th 2012
PACKMAN
FDH pBAD MG1655 electroporation
Procedure We seeded cells on 7:10 AM and did cell-down on 10:00AM. And then we plated cells on 1:00PM and checked colony on 8/14 5:00AM Results
FDH pTrc MG1655 expression check (sampling and induction)
Procedure We seeded cells (500λ of cells put into 50mL LB with 50λ Ampicillin) on 7 : 30 AM and checked Optical Density on 10:30pm. According to the O.D. value, we induced 37℃ sample on 10:30PM and did sampling on 4:30pm. Also, we induced 30℃ sample on 11:30pm and did sampling on 5:30pm. (6 hour induction) Each temperature samples were induced by different conditions ; 2mM and 3mM of IPTG.
Flip Flop
pPoCpi cloning into pSB1C3
Cloning was done by remaining DNA samples.
Back to the Calendar
August 14th 2012
PACKMAN
colony PCR check of pBADmycHisC inserted FDH gene
Results
Discussion We checked the presence of FDH gene by conducting colony PCR. Correct size of band appeared at about 1115 bp, which means formate dehydrogenase is inserted correctly.
Expression check of Formate dehydrogenase on pTrcHis2A vector (Result)
Results
Discussion For each samples of different induction conditions, we run SDS-PAGE gel and checked the presence of 43.7 kDa protein, formate dehydrogenase. The band appeared significantly on the correct size.
FDH pBAD MG1655 seeding for expression check
Procedure We seeded cell on 7:00PM, so we’ll incubate cell about 16hours. (This means we have to do next seeding on 11:00AM. )
Primer design for the insertion of 1202 and 0109
Results
Discussion To eliminate uncertain spacer between genes, we inserted
Flip Flop
Bxb1 pTrcHis2A and pPoCpi ligated products transformation
The products were transformed into TOP10
Back to the Calendar
August 15th 2012
PACKMAN
FDH pBAD MG1655 expression check
Procedure We seeded cells on 11 : 30 AM and checked Optical Density on 1:30pm. According to the O.D. value, we induced 37℃ sample on 1:40pm and did sampling on 7:40pm. Also, we induced 30℃ sample on 3:30pm and did sampling on 9:30pm. (6 hour induction) Each temperature samples were induced by different conditions: 10mM of arabinose and 30mM of arabinose.
Flip Flop
pPoCpi colony PCR
Results
Among colonies #1~10, four colonies were inoculated. #1, 3, 4, 7
Bxb1 pTrcHis2A colony PCR
Results
Among colonies #1~20, three colonies were inoculated. #2, 5, 19
Back to the Calendar
August 16th 2012
PACKMAN
PCR amplification of 1201, 1202, 1203, 1198 and 1191
Results
Discussion Primers arrived, and we began to do additional PCR amplification on the preparation for Gibson assembly. Each PCR sample has the correct size of oligonucleotides except Moth 1202 gene.
1191 : 921bp
1198 : 972bp + about 400bp = 1300 bp
1201 : 1341bp + ?
1202 : 2190bp +
1203 : 2025bp + 1600bp = 3625bp
We did PCR purification on the PCR product and the concentration/purity result was as shown below.
Pre-culture of 0109
Discussion There was no vector prepared for Moth 0109, so we did 5ml pre-culture on 10ml tube. Cell was grown about 16hours after pre-culture, and we did vector mini-prep on the sample. The concentration of the mini-prepped vector is 726.8ng/ul and the purity(A260/A280) was 1.86.
Flip Flop
pPoC and pPoCpi electro-transformation into MG1655
Strain: MG1655 Plasmid source:
pPoC: from DH5α-pPoC colony #1 and TOP10-pPoC colony #1
pPoCpi: from TOP10-pPoC colony #1 and #3
Back to the Calendar
August 17th 2012
PACKMAN
Expression check of Formate dehydrogenase.
Results
Discussion For each samples of different induction conditions, we run SDS-PAGE gel and checked the presence of 40.xx kDa protein, formate dehydrogenase. The band appeared more significantly than the case of pTrcHis2A vector. Therefore, using pBAD promoter seems to produce more proteins.
PCR amplification – second trial
Results
Discussion As the band was not significant for 1203 and 1198 on previous trial and purity was low for 1191, we tried secondary PCR amplification on samples for Gibson assembly; 1202, 1203, 1198, 1191. Particularly for 1202 gene, we tried additional colony PCR, so that we could check any problem with vector mini-prep. (PCR solution for 1202 colony PCR was : 22ul
Flip Flop
MG1655-pPoC, MG1655-pPoCpi streaking
We could not distinguish single colony from plates. Therefore we scraped cells and streaked onto new plates. Discussions Wild-type E.coli showed more reddish and yellowish color than competent strains, DH5α and TOP10. The promoter BBa_J23119 may show different strength in-between strains.
TOP10-pTrcHis2A_Bxb1 expression check
TOP10 containing pTrcHis2A was cultured in 30mL of LB with 1% of Ampicillin. IPTG induced when OD = 0.4 Sampling: 8 hours after induction, 1010 cells. Only soluble fractions were PAGEed.
Back to the Calendar
August 18th 2012
PACKMAN
PCR amplification of 1202, 0109 gene with original primer and Gibson primers
Results
Each lane indicates:
Moth_1202_1 : Gibson forward + original reverse
Moth_1202_2 : Gibson reverse + original forward
Moth_0109_1 : Gibson forward + original reverse
Moth_0109_2 : Gibson reverse + original forward
There was correct size and unspecific size of band on 1202_1 sample. Although there are also unspecific band, 1202_2 sample has large band of correct size. 0109_1 sample has only unspecific band and 0109_2 sample has right size of band. Discussion When the PCR was unsuccessful, we first thought it might be problem of forward primer as in the case of 1199 and 1191. Our guess was correct, and we’re going to fix the problem with forward primers.
1202 and 0109 forward primer design
We designed new forward primer hoping no problem with it. Results
Newly designed primers contain overlapping region with former DNA fragments, artificial ribosome binding site, and overlapping region with 1202 and 0109 each.
Flip Flop
MG1655-pPoC, MG1655-pPoCpi colony inoculation
Two colonies from each plate were inoculated into 3mL of LB containing 1% of chloramphenicol. Discussions In 12 hours, cell cultures show colors compared to another common E.coli culture. When they are centrifuged, pellet shows clear and definite yellow or red color.
pTrcHis2A_Bxb1 electro-transformation into MG1655-pPoC, MG1655-pPoCpi
pTrcHis2A_Bxb1 is electro-transformed into MG1655 containing pPoC and pPoCpi. Plasmid source: TOP10-pTrcHis2A_Bxb1
Back to the Calendar
August 19th 2012
No Special Event!
Back to the Calendar
August 20th 2012
No Special Event!
Back to the Calendar
August 21st 2012
No Special Event!
Back to the Calendar
August 22nd 2012
Flip Flop
Primer design for Bxb1 integrase cloning
Primers for Bxb1 integrase cloning were designed. For controlling basal transcription level optimization, a mutant primer which has GTG rather than ATG for its start codon was also ordered. Results
Back to the Calendar
August 23rd 2012
No Special Event!
Back to the Calendar
August 24th 2012
PACKMAN
0109, 1191 PCR with new primers
Results
Discussion PCR product of Moth 0109 gene and 1191 gene has run on the gel. There was only faint band of 0109 fragment but the product band of 1191 gene was significantly shown on the gel. After gel electrophoresis, we tried PCR of 0109 once more and got significant amount of fragments.
Back to the Calendar
August 25th 2012
Flip Flop
pPoC-Bxb1 double transformants inoculation for prep & PCR check
Double transformants 1 through 24 were inoculated into 3ml of LB containing 1% of Cm and Ap. For our convenience, three colonies were inoculated in one culture tube.
Back to the Calendar
August 26th 2012
Flip Flop
pPoC-Bxb1 double transformants plasmid DNA miniprep & PCR check
Inoculated cultures were mipi-preped. And Bxb1-pTrc plasmid was checked by PCR. In case of pPoC, it would be check by color. Results Vector
Confirm PCR
Discussions In confirm PCR, all the colonies seem to have Bxb1 vector. Colony #1, 4, 7, 10, 13, 14, 16, 19 and 22 will be inoculated.
Bxb1 PCR with new primers
For cloning of Bxb1 into pBAD and mutating its start codon from ATG to GTG, we PCR pGEM-B1-Bxb1 vector as template. Samples were duplicated PCR conditions
95℃ for 3min
95℃ for 30sec
55℃ for 30sec
72℃ for 1min 30sec
Step 2 to 4: 34 cycles
72℃ for 10min
12℃ forever
Results
Duplicated samples were purified in one DNA binding column.ATG: 297.9ng/uL (purity: 1.90)GTG: purify failed. Sample lost Discussions There were multiple bands in our PCR products. We will PCR again Bxb1-GTG with different conditions. Template vector may show larger bands.
pGEM-Bxb1 template vector gel electrophoresis
To check whether the undesired bands were template vector, we diluted template vector with same conditions of PCR, 50X, and electrophoresized. Results
Discussion The band was different from undesired bands from Bxb1 ATG, GTG PCR products.
Insert DNA (Bxb1 integrase) and vector (pTrc and pBAD) cloning
Back to the Calendar
August 27th 2012
Flip Flop
Bxb1-pTrc and Bxb1-pBAD electrotransformation into MG1655
Bxb1 PCR with changed conditions
For cloning of Bxb1 into pBAD and mutating its start codon from ATG to GTG, we PCR pGEM-B1-Bxb1 vector as template. Samples were duplicated
PCR conditions #1
PCR conditions #2
1) 95℃ for 3min
1) 95℃ for 3min
2) 95℃ for 30sec
2) 95℃ for 30sec
3) 56.5℃ for 30sec
3) 58℃ for 30sec
4) 72℃ for 1min 30sec (Step 2 to 4: 34 cycles)
4) 72℃ for 1min 30sec (Step 2 to 4: 34 cycles)
5) 72℃ for 10min
5) 72℃ for 10min
6) 12℃ forever
6) 12℃ forever
Results
Discussions Still, undesired bands appeared. We do not know where they come from. We decide to extract desired DNA with gel extraction.
Bxb1, with GTG as start codon, PCR product gel extraction
PCR products were gel extracted. Results
Bxb1 GTG cond. #1 - #1: 128.2ng/uL (purity: 1.89)
Bxb1 GTG cond. #1 - #2: 79.0ng/uL (purity: 1.89)
Bxb1 GTG cond. #2 - #1: 110.7ng/uL (purity: 1.89)
Bxb1 GTG cond. #2 - #2: 66.7ng/uL (purity: 1.94)
Discussions Bxb1 GTG cond. #1 - #1 and Bxb1 GTG cond. #2 - #1 will be used to cloning.
Insert DNA (Bxb1 GTG) and vector (pTrc and pBAD) cloning
Insert DNA: Bxb1 integrase mutated by primer. Vectro DNA: pTrcHis2A, pBAD-mycHisC Prepared DNA concentration:
Bxb1 GTG cond. #1 - #1: 128.2ng/uL (purity: 1.89)
Bxb1 GTG cond. #2 - #1: 110.7ng/uL (purity: 1.89)
pTrcHis2A: 116.1ng/uL
pBAD-mycHisC: 128.7ng/uL
Enzyme digestion: 37℃ for 1hr 30min, 80℃ for 20min
Dephosphorylation: 37℃ for 30min, 70℃ for 5min
Ligation: 16℃ overnight
Bxb1, pPoC double transformants colony PCR
Colony #1, 4, 7, 10, 13, 14, 16, 19 and 22 from master plate Results
Discussions The gel showed that all the colonies have Bxb1-pTrc vector. And also they all have pPoC or pPoCpi, because they show color.
Back to the Calendar
August 28th 2012
Flip Flop
Bxb1-GTG-pTrc and Bxb1-GTG-pBAD electrotransformation into MG1655
Bxb1-pTrc and Bxb1-pBAD transformants check
Results Bxb1-pTrc: No colony Bxb1-pBAD: Colonies appeared and incoculated into 3mL of LB containing 1% of Ap
Bxb1-pTrc and Bxb1-pBAD transformants colony PCR
Results
Bxb1-pTrc and Bxb1-pBAD transformants plasmid DNA miniprep
Results Colony #2: 109.8ng/uL, purity = 1.87 Colony #3: 138.9ng/uL, purity = 1.74
mRFP, GFP 30mL culture and sampling
Results
Back to the Calendar
August 29th 2012
Flip Flop
Bxb1 pBADmycHisC prepped plasmid DNA RE cut check
mini-prepped plasmid DNAs were cut with SnaBI expected size : 5.6kb Results
PACKMAN
1202 PCR to make fragments for Gibson assembly
Results
Discussion PCR for Moth 1202 gene has failed. No band for Moth 1202 gene has appeared on the gel.
Gibson assembly – assembly of pcr fragments.
Results Among three methods to assemble DNA fragments, we chose first method, which is assembly of PCR amplified DNA fragments. 0.2umoles of each fragments are added into the reaction sample, and volume to be added for each fragments has calculated on the assumption that average MW of each base is 660Da. Table below is the result of calculation. Fragment 1
Fragment 2
Discussion As we can only assemble the fragments through Gibson Assembly method and not amplify them, there would be very small amount of template remaining in the tube. We thought that’s why we couldn’t see any band on the gel. So we did PCR purification of the fragment once more and saw if the band appears on the gel.
Back to the Calendar
August 30th 2012
PACKMAN
Gibson Assembly product – Amplification by nested PCR
Results
Discussion There was no right size of the band but just unspecific bands on the gel.  There might be several reasons why the Gibson assembly ended with failure. So we decided to change the method of Gibson assembly; we will assemble fragments in the presence of vectors, so that assembly product can become stable, circular form. To cut the cost used in experiment, we will not design primers, but digest the vector with restriction enzymes to make it linear form.
Gibson primer second design 2
Results
Discussion We designed primers again for another trial of Gibson Assembly. As stated before, We’ll use vector-involved assembly method to stabilize the assembly product and amplify them by in vivo system. Like the method we used before, we’ll assemble 5 fragments first and assemble those 5-fragment-long two fragments into full construct.
Flip Flop
Double transformants expression SDS-PAGE
Gel composition: 20% Results pPoC
pPoCpo
Results We cannot see expression level change of fluorescent protein in SDS-PAGE.
Back to the Calendar
August 31st 2012
No Special Event!
Back to the Calendar
Kaist Footer
Retrieved from " http://2012.igem.org/Team:KAIST_Korea/Notebook_Labnote/2012_8 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
