<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_University_College_London_Module_2_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:University_College_London/Module_2/Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:University_College_London/Module_2/Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:University_College_London/Module_2/Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:University_College_London/Module_2/Modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:University_College_London/Module_2/Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:University College London/Module 2/Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="ucligem"><DIV id="header" class="cover-training"><DIV id="top"><UL><LI><A href="/Team:University_College_London" title="Team:University College London">Home</A></LI><LI><A href="/Team:University_College_London/Team" title="Team:University College London/Team">Team</A></LI><LI><A href="/Team:University_College_London/Team" title="Team:University College London/Team">Team</A></LI><LI><A href="/Team:University_College_London/Press" title="Team:University College London/Press">Press</A></LI><LI><A href="/Team:University_College_London/Attributions" title="Team:University College London/Attributions">Attributions</A></LI><LI><A href="/Team:University_College_London/ContactUs" title="Team:University College London/ContactUs">Contact Us</A></LI></UL><LI><A href="/Team:University_College_London/Research" title="Team:University College London/Research">Research</A></LI><LI><A href="/Team:University_College_London/Research" title="Team:University College London/Research">Overview</A></LI><LI><A href="/Team:University_College_London/Module_1" title="Team:University College London/Module 1">1: Detection</A></LI><LI><A href="/Team:University_College_London/Module_2" title="Team:University College London/Module 2">2: Aggregation</A></LI><LI><A href="/Team:University_College_London/Module_3" title="Team:University College London/Module 3">3: Degradation</A></LI><LI><A href="/Team:University_College_London/Module_4" title="Team:University College London/Module 4">4: Buoyancy</A></LI><LI><A href="/Team:University_College_London/Module_5" title="Team:University College London/Module 5">5: Salt Tolerance</A></LI><LI><A href="/Team:University_College_London/Module_6" title="Team:University College London/Module 6">6: Containment</A></LI><LI><A href="/Team:University_College_London/Research/MarineBacteria" title="Team:University College London/Research/MarineBacteria">Marine Bacteria</A></LI><LI><A href="/Team:University_College_London/BioBricks" title="Team:University College London/BioBricks">BioBricks</A></LI><LI><A href="/Team:University_College_London/Modelling" title="Team:University College London/Modelling">Modelling</A></LI><LI><A href="/Team:University_College_London/LabSafety" title="Team:University College London/LabSafety">Safety</A></LI><LI><A href="/Team:University_College_London/HumanPractice" title="Team:University College London/HumanPractice">Human Practice</A></LI><LI><A href="/Team:University_College_London/HumanPractice" title="Team:University College London/HumanPractice">Overview</A></LI><LI><A href="/Team:University_College_London/HumanPractice/DIYbio" title="Team:University College London/HumanPractice/DIYbio">DIYbio - &quot;Public BioBrick&quot;</A></LI><LI><A href="/Team:University_College_London/HumanPractice/MOYM" title="Team:University College London/HumanPractice/MOYM">Meeting of Young Minds Debate</A></LI><LI><A href="/Team:University_College_London/HumanPractice/LegalBits" title="Team:University College London/HumanPractice/LegalBits">The Legal Bits </A></LI><LI><A href="/Team:University_College_London/HumanPractice/CrowdFunding" title="Team:University College London/HumanPractice/CrowdFunding">Crowd Funding</A></LI><LI><A href="/Team:University_College_London/gemFM" title="Team:University College London/gemFM">gemFM</A></LI><LI><A href="/Team:University_College_London/HumanPractice/SpeedDebating" title="Team:University College London/HumanPractice/SpeedDebating">Speed Debating</A></LI><LI><A href="/Team:University_College_London/HumanPractice/LivingArchitechture" title="Team:University College London/HumanPractice/LivingArchitechture" class="mw-redirect">Living Architechture</A></LI><LI><A href="/Team:University_College_London/Achievements" title="Team:University College London/Achievements">Achievements</A></LI><LI><A href="/Team:University_College_London/Notebook" title="Team:University College London/Notebook" class="mw-redirect">Diary</A></LI><LI><A href="/Team:University_College_London/Notebook" title="Team:University College London/Notebook" class="mw-redirect">Notebook</A></LI><LI><A href="/Team:University_College_London/LabBook" title="Team:University College London/LabBook" class="mw-redirect">Lab Book</A></LI><LI><A href="/Team:University_College_London/Sponsors" title="Team:University College London/Sponsors">Sponsors</A></LI><LI><A href="/Team:University_College_London/FAQ" title="Team:University College London/FAQ">FAQ</A></LI></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Module_2:_Aggregation"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Module 2: Aggregation</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Curli_synthesis"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Curli synthesis</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Aim"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Aim</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Description_of_the_model"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Description of the model</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Species"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Species</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Reactions_taking_place_in_the_model"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Reactions taking place in the model</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Results"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Module_2:_Aggregation"> Module 2: Aggregation </SPAN></H1><P><B><A href="/Team:University_College_London/Module_2" title="Team:University College London/Module 2">Description</A> | <A href="/Team:University_College_London/Module_2/Design" title="Team:University College London/Module 2/Design">Design</A> | <A href="/Team:University_College_London/Module_2/Construction" title="Team:University College London/Module 2/Construction">Construction</A> | <A href="/Team:University_College_London/Module_2/Characterisation" title="Team:University College London/Module 2/Characterisation">Characterisation</A> | <A href="/Team:University_College_London/Module_2/Shear_Device" title="Team:University College London/Module 2/Shear Device">Shear Device</A> | <STRONG class="selflink">Modelling</STRONG> | <A href="/Team:University_College_London/Module_2/Results" title="Team:University College London/Module 2/Results">Results</A> | <A href="/Team:University_College_London/Module_2/Conclusion" title="Team:University College London/Module 2/Conclusion">Conclusions</A></B></P><H2><SPAN class="mw-headline" id="Curli_synthesis"> Curli synthesis </SPAN></H2><P>Our cell model for aggregation describes the pathways through which curli cluster coding genes interact to make up  up curli fibrils which are produced on detection of microplastics. This particular curli cluster (BBa_K540000) contains five genes that later on interact and make up curli fibrils, however, we are looking into synthesis and interaction of only two of them: CsgA and CsgB due to complex nature of the system.
</P><H2><SPAN class="mw-headline" id="Aim"> Aim </SPAN></H2><P>Using this we aim to find out to which extent is curli fibrils' production is dependent on the persistent organic pollutant presence in the surrounding environment. In addition to that we would like to estimate how much fibrils are produced over time.The following diagram shows the genes, molecules and reaction involved in the synthesis of the curli fibrils, on which further description is provided below. 
</P><H2><SPAN class="mw-headline" id="Description_of_the_model"> Description of the model </SPAN></H2><P>As with our <A href="/Team:University_College_London/Module_3/Modelling" title="Team:University College London/Module 3/Modelling">degradation model</A>, this SimBiology model is divided into three compartments, with each DNA species and reaction explained in further detail below:
</P><P>
1. <B>Outside1:</B> in this compartment we see the association, adherence, and dissociation of persistent organic pollutants (POPs) from polyethylene (PE) (R1).  We rely on the POPs to induce our curli synthesis system. As it will be described later on, POPs have a tendency to adhere to the plastic 10<SUP>5</SUP> to 10<SUP>6</SUP> times more than reamin in the ocean. As curli synthesis is induced by the POPs this means that curli synthesis will be the specificity to our system.
</P><P>
2. <B>Cell:</B> NahR is a constitutively produced mRNA product (R9) of which around 3% degrades (R10). When POP diffuses into the cell (R2), it forms a complex with intact NahR (R3) which then binds to the pSal promoter (R4) to induce the curli gene cluster (R5).  This starts translation of five proteins which make up curli fibrils.  Here we focus on those controlled by the CsgAB operon, CsgA (R7) and CsgB (R6), as these are the most important in curli synthesis as described on our <A href="https://2012.igem.org/Team:University_College_London/Module_2" class="external text" rel="nofollow">research page</A>.  The CsgA and CsgB that do not degrade (R11, R12) diffuse out of the cell.   
</P><P>
3. <B>Outside2:</B> Outside of the cell the polymerization of CsgA by CsgB takes place (R8) to make a curli fibril.
</P><H2><SPAN class="mw-headline" id="Species">Species </SPAN></H2><TABLE class="bigtable"><TBODY><TR><TH> Species</TH><TH> Initial value (molecules) </TH><TH> Notes &amp; Assumptions
</TH></TR><TR><TD> PE</TD><TD> 0.044 </TD><TD> Polyethylene found in North Pacific Gyre (value per cubic metre)<SUP>1,2</SUP></TD></TR><TR><TD> POPex </TD><TD> 0.0 </TD><TD> Persistent organic pollutants (ex = extracellular) that are not adhered to plastic surface
</TD></TR><TR><TD> PEPOPex </TD><TD> 9.24E-5 </TD><TD> Persistent organic pollutants (ex = extracellular) that are adhered to the plastic surface<SUP>6</SUP></TD></TR><TR><TD> POPin </TD><TD> 0.5 </TD><TD> Persistent organic pollutants (in = intracellular) assumed from <I>E. coli</I> membrane permeability <SUP>4</SUP></TD></TR><TR><TD> mRNANahR </TD><TD> 0.0</TD><TD> NahR mRNA product
</TD></TR><TR><TD> POPinNahR </TD><TD> 0.0</TD><TD> Complex of the above two molecules
</TD></TR><TR><TD> POPinNahRpSal </TD><TD> 0.0 </TD><TD> Complex of the above molecule and pSal (promoter that induces laccase transcription)
</TD></TR><TR><TD> mRNACurli </TD><TD> 0.0 </TD><TD> Polycistronic mRNA as it codes for more than one protein, in reality curli cluster contains five or more proteins but as mentioned previously we are concentrating only on the synthesis of CsgA and CsgA therefore we assume that only two genes are present in this polycistronic mRNA.
</TD></TR><TR><TD> CsgA </TD><TD> 0.0 </TD><TD> One of the polypeptides that is coded for in curli cluster, it is a structural component secreted in subunits outside of the cell
</TD></TR><TR><TD> CsgB </TD><TD> 0.0 </TD><TD> One of the polypeptides that is coded for in curli cluster, it is secreted outside of the cell allowing polymerization of CsgA
</TD></TR><TR><TD> Fibril </TD><TD> 0.0 </TD><TD> Result of CsgA and CsgB interaction
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Reactions_taking_place_in_the_model">Reactions taking place in the model</SPAN></H2><TABLE class="bigtable"><TBODY><TR><TH> Number </TH><TH> Reaction </TH><TH> Reaction rate (molecules/sec) </TH><TH> Notes &amp; Assumptions
</TH></TR><TR><TD> R1 </TD><TD> PE + POPex â PEPOPex </TD><TD> Forward: 1000  Backward: 1 </TD><TD> Pops have 100000 to 1000000 times greater tendency to adhere to plastic than float free in the ocean<SUP>6</SUP></TD></TR><TR><TD> R2 </TD><TD> POPex â POPin </TD><TD> Forward: 0.6  Backward: 0.4 </TD><TD> Rate based on membrane permeability<SUP>4</SUP> and diffusion gradient
</TD></TR><TR><TD> R3 </TD><TD> POPin + mRNA.Nahr â POPin.mRNA.Nahr </TD><TD> Forward: 1  Backward: 0.0001 </TD><TD> Based on the assumption that the chemical structure/size of POPs is similar to salycilate<SUP>7</SUP>.  Salycilate binds to the NahR mRNA product, which complex then binds to the pSal promoter.
</TD></TR><TR><TD> R4 </TD><TD> POPinmRNANahr â POPinmRNANahr.Psal </TD><TD> Forward: 78200  Backward: 0.191 <SUP>10</SUP></TD><TD> NahR to pSal binding based on the assumption that POP-NahR binding has no effect on NahR-pSal binding
</TD></TR><TR><TD> R5 </TD><TD> POPexmRNANahr.Psal â mRNAcurli </TD><TD> 0.054 </TD><TD> Transcription rate of curli cluster in molecules/sec (for cluster size 1500bp<SUP>11</SUP>, transcription rate in E.coli 80bp/sec<SUP>9</SUP>)
</TD></TR><TR><TD> R6 </TD><TD> mRNAcurli â CsgB </TD><TD> 0.13 </TD><TD> Translation rate of CsgB in molecules/sec (for CsgB size 151aa<SUP>12</SUP>, translation rate in E.coli 20aa/sec<SUP>9</SUP>)
</TD></TR><TR><TD> R7 </TD><TD> mRNAcurli â CsgA </TD><TD> 0.13 </TD><TD> Translation rate of CsgA in molecules/sec (for CsgA size 151aa<SUP>13</SUP>, translation rate in E.coli 20aa/sec<SUP>9</SUP>)
</TD></TR><TR><TD> R8 </TD><TD> CsgB + CsgA -&gt; fibril </TD><TD> 0.003 </TD><TD> See calculations below
</TD></TR><TR><TD> R9 </TD><TD> 0 â mRNA.Nahr </TD><TD> Forward: 0.088  Backward: 0.6 </TD><TD> Transcription rate of NahR in molecules/sec (for NahR size 909 bp<SUP>8</SUP>, transcription rate in E.coli 80bp/sec<SUP>9</SUP>) under constitutive promoter control
</TD></TR><TR><TD> R10 </TD><TD> mRNA.Nahr â 0 </TD><TD> 0.03 </TD><TD> Degradation rate of NahR mRNA product<SUP>14</SUP></TD></TR><TR><TD> R11 </TD><TD> CsgA â 0 </TD><TD> 0.03 </TD><TD> Degradation rate of CsgA <SUP>14</SUP> must be taken into account due to suboptimal conditions
</TD></TR><TR><TD> R12 </TD><TD> CsgB â 0 </TD><TD> 0.03 </TD><TD> Degradation rate of CsgB <SUP>14</SUP> must be taken into account due to suboptimal conditions
</TD></TR></TBODY></TABLE><P>
We calculated the reaction rate of R8 (CsgB + CsgA -&gt; fibril) as follows:
</P><P>The average length of an amino acid is 0.8nm<SUP>15</SUP>.  The length of CsgA is 82 amino acids<SUP>16</SUP>, so a non-folded CsgA protein is 65.6 nm in length.  CsgA's tertiary structure is folded into 5<SUP>17</SUP> so the length of a CsgA protein in a curli is 13.1 nm.  A curli is several micrometers long (~3000nm)<SUP>18</SUP> so there are around 299 CsgA molecules per curli.  The production of CsgA is the limiting step in the production of curli fibrils (compared to production of CsgB) so the forward rate for this reaction is 1/299.
</P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><P>The following graph shows that fibrils are produced at a rate of around 1 every three hours.
</P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P>1. Goldstein M, Rosenberg M, Cheng L (2012) Increased oceanic microplastic debris enhances oviposition in an endemic pelagic insect. <I>Biology Letters</I> 10.1098
</P><P>2. Andrady AL (2011) Microplastics in the marine environment. <I>Marine Pollution Bulletin</I> 62: 1596-1605
</P><P>4. Kay J, Koivusalo M, Ma X, Wohland T, Grinstein S (2012) Phosphatidylserine Dynamics in Cellular Membranes. <I>Molecular Biology of the Cell</I></P><P>5. Nenninger AA, Robinson LS, Hammer ND, Epstein EA, Badtke MP, Hultgren SJ, Chapman MR (2011) CsgE is a curli secretion specificity factor that prevents amyloid fibre aggregation.  <I>Mol Microbiol</I> 81: 486-499
</P><P>6. Mato Y, Isobe T, Takada H, Kanehiro H, Ohtake C, Kaminuma T (2001) Plastic Resin Pellets as a Transport Medium for Toxic Chemicals in the Marine Environment. <I>Environ. Sci. Technol.</I> 35: 318-324
</P><P>7.  <A href="https://2011.igem.org/Team:Peking_S/project/wire/harvest" class="external free" rel="nofollow">https://2011.igem.org/Team:Peking_S/project/wire/harvest</A></P><P>8. <A href="http://www.xbase.ac.uk/genome/azoarcus-sp-bh72/NC_008702/azo2419;nahR1/viewer" class="external free" rel="nofollow">http://www.xbase.ac.uk/genome/azoarcus-sp-bh72/NC_008702/azo2419;nahR1/viewer</A></P><P>9. <A href="http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf" class="external free" rel="nofollow">http://kirschner.med.harvard.edu/files/bionumbers/fundamentalBioNumbersHandout.pdf</A></P><P>10. Park H, Lim W, Shin H (2005) In vitro binding of purified NahR regulatory protein with promoter Psal. <I>Biochimica et Biophysica Acta</I> 1775: 247-255
</P><P>11. Shala AA, Restrepo S, Gonzalez Barrios AF (2011) A network model for biofilm development in Escherichia coli K-12.  <I> Theoretical Biology and Medical Modelling </I> 8: 34 doi:10.1186/1742-4682-8-34
</P><P>12. <A href="http://www.ecogene.org//?q=gene/EG12621" class="external free" rel="nofollow">http://www.ecogene.org//?q=gene/EG12621</A></P><P>13. <A href="http://www.ecogene.org/?q=gene/EG11489" class="external free" rel="nofollow">http://www.ecogene.org/?q=gene/EG11489</A></P><P>14. Kushner S (2002) mRNA Decay in <I>Escherichia coli</I> Comes of Age. <I>J Bacteriol.</I> 184: 4658-4665
</P><P>15. <A href="http://en.wikibooks.org/wiki/Cell_Biology/Introduction/Cell_size" class="external free" rel="nofollow">http://en.wikibooks.org/wiki/Cell_Biology/Introduction/Cell_size</A></P><P>16. <A href="http://bacteria.ensembl.org/b_subtilis/Transcript/Summary?g=EBBACG00000004082;r=Chromosome:228066-228314;t=EBBACT00000004090" class="external free" rel="nofollow">http://bacteria.ensembl.org/b_subtilis/Transcript/Summary?g=EBBACG00000004082;r=Chromosome:228066-228314;t=EBBACT00000004090</A></P><P>17. Shu Q, Crick S, Pinkner J, Ford B, Hultgren S, Frieden C (2012) The E. coli CsgB nucleator of curli assembles to Î²-sheet oligomers that alter the CsgA fibrillization mechanism.  Proceedings of the National Academy of Sciences of the United States of America 109: 6502-6507.  DOI: 10.1073/pnas.1204161109
</P><P>18. Pitkanen M, Honkalampi U, von Wright A, Sneck A, Hentze H-P, Sievanen J, Hiltunen J, Hellen EKO (2010) Nanofibrillar cellulose - assessment of cytotoxic and genotoxic properties <I>in vitro</I>. International conference on nanotechnology for the forest products industry
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:University_College_London/Module_2/Modelling">http://2012.igem.org/Team:University_College_London/Module_2/Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:University_College_London/Module_2/Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:University_College_London/Module_2/Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:University_College_London/Module_2/Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:University_College_London/Module_2/Modelling&amp;oldid=200624" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>