<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_University_College_London_LabBook_Week9"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:University_College_London/LabBook/Week9">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:University_College_London/LabBook/Week9&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:University_College_London/LabBook/Week9&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:University_College_London/LabBook/Week9&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:University_College_London/LabBook/Week9" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:University College London/LabBook/Week9</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="ucligem"><DIV id="headerimg"><DIV id="top"><UL><LI><A href="/Team:University_College_London" title="Team:University College London">Home</A></LI><LI><A href="/Team:University_College_London/Team" title="Team:University College London/Team">Team</A></LI><LI><A href="/Team:University_College_London/Team" title="Team:University College London/Team">Team</A></LI><LI><A href="/Team:University_College_London/Press" title="Team:University College London/Press">Press</A></LI><LI><A href="/Team:University_College_London/Attributions" title="Team:University College London/Attributions">Attributions</A></LI><LI><A href="/Team:University_College_London/ContactUs" title="Team:University College London/ContactUs">Contact Us</A></LI></UL><LI><A href="/Team:University_College_London/Research" title="Team:University College London/Research">Research</A></LI><LI><A href="/Team:University_College_London/Research" title="Team:University College London/Research">Overview</A></LI><LI><A href="/Team:University_College_London/Module_1" title="Team:University College London/Module 1">1: Detection</A></LI><LI><A href="/Team:University_College_London/Module_2" title="Team:University College London/Module 2">2: Aggregation</A></LI><LI><A href="/Team:University_College_London/Module_3" title="Team:University College London/Module 3">3: Degradation</A></LI><LI><A href="/Team:University_College_London/Module_4" title="Team:University College London/Module 4">4: Buoyancy</A></LI><LI><A href="/Team:University_College_London/Module_5" title="Team:University College London/Module 5">5: Salt Tolerance</A></LI><LI><A href="/Team:University_College_London/Module_6" title="Team:University College London/Module 6">6: Containment</A></LI><LI><A href="/Team:University_College_London/Research/MarineBacteria" title="Team:University College London/Research/MarineBacteria">Marine Bacteria</A></LI><LI><A href="/Team:University_College_London/BioBricks" title="Team:University College London/BioBricks">BioBricks</A></LI><LI><A href="/Team:University_College_London/Modelling" title="Team:University College London/Modelling">Modelling</A></LI><LI><A href="/Team:University_College_London/LabSafety" title="Team:University College London/LabSafety">Safety</A></LI><LI><A href="/Team:University_College_London/HumanPractice" title="Team:University College London/HumanPractice">Human Practice</A></LI><LI><A href="/Team:University_College_London/HumanPractice" title="Team:University College London/HumanPractice">Overview</A></LI><LI><A href="/Team:University_College_London/HumanPractice/DIYbio" title="Team:University College London/HumanPractice/DIYbio">DIYbio - &quot;Public BioBrick&quot;</A></LI><LI><A href="/Team:University_College_London/HumanPractice/MOYM" title="Team:University College London/HumanPractice/MOYM">Meeting of Young Minds Debate</A></LI><LI><A href="/Team:University_College_London/HumanPractice/LegalBits" title="Team:University College London/HumanPractice/LegalBits">The Legal Bits </A></LI><LI><A href="/Team:University_College_London/HumanPractice/CrowdFunding" title="Team:University College London/HumanPractice/CrowdFunding">Crowd Funding</A></LI><LI><A href="/Team:University_College_London/gemFM" title="Team:University College London/gemFM">gemFM</A></LI><LI><A href="/Team:University_College_London/HumanPractice/SpeedDebating" title="Team:University College London/HumanPractice/SpeedDebating">Speed Debating</A></LI><LI><A href="/Team:University_College_London/HumanPractice/LivingArchitechture" title="Team:University College London/HumanPractice/LivingArchitechture" class="mw-redirect">Living Architechture</A></LI><LI><A href="/Team:University_College_London/Achievements" title="Team:University College London/Achievements">Achievements</A></LI><LI><A href="/Team:University_College_London/Notebook" title="Team:University College London/Notebook" class="mw-redirect">Diary</A></LI><LI><A href="/Team:University_College_London/Notebook" title="Team:University College London/Notebook" class="mw-redirect">Notebook</A></LI><LI><A href="/Team:University_College_London/LabBook" title="Team:University College London/LabBook" class="mw-redirect">Lab Book</A></LI><LI><A href="/Team:University_College_London/Sponsors" title="Team:University College London/Sponsors">Sponsors</A></LI><LI><A href="/Team:University_College_London/FAQ" title="Team:University College London/FAQ">FAQ</A></LI></DIV></DIV><DIV id="labbookmenu"><B><A href="/Team:University_College_London/LabBook/Week3" title="Team:University College London/LabBook/Week3">Week 3</A> | <A href="/Team:University_College_London/LabBook/Week4" title="Team:University College London/LabBook/Week4">Week 4</A> | <A href="/Team:University_College_London/LabBook/Week5" title="Team:University College London/LabBook/Week5">Week 5</A> | <A href="/Team:University_College_London/LabBook/Week6" title="Team:University College London/LabBook/Week6">Week 6</A> | <A href="/Team:University_College_London/LabBook/Week7" title="Team:University College London/LabBook/Week7">Week 7</A> | <A href="/Team:University_College_London/LabBook/Week8" title="Team:University College London/LabBook/Week8">Week 8</A> | <STRONG class="selflink">Week 9</STRONG> | <A href="/Team:University_College_London/LabBook/Week10" title="Team:University College London/LabBook/Week10">Week 10</A> | <A href="/Team:University_College_London/LabBook/Week11" title="Team:University College London/LabBook/Week11">Week 11</A> | <A href="/Team:University_College_London/LabBook/Week12" title="Team:University College London/LabBook/Week12">Week 12</A> | <A href="/Team:University_College_London/LabBook/Week13" title="Team:University College London/LabBook/Week13">Week 13</A> | <A href="/Team:University_College_London/LabBook/Week14" title="Team:University College London/LabBook/Week14">Week 14</A> | <A href="/Team:University_College_London/LabBook/Week15" title="Team:University College London/LabBook/Week15">Week 15</A> | <A href="/Team:University_College_London/LabBook/Week16" title="Team:University College London/LabBook/Week16">Week 16</A> | <A href="/Team:University_College_London/LabBook/Week17" title="Team:University College London/LabBook/Week17">Week 17</A></B></DIV><DIV class="experimentContent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Monday_6.7.12"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Monday 6.7.12</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Tuesday_7"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Tuesday 7</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Monday_6.7.12_2"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Monday 6.7.12</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Tuesday_7_2"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Tuesday 7</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Monday_8.8.12"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Monday 8.8.12</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Tuesday_9.8.12"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Tuesday 9.8.12</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Thursday_10.8.12"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Thursday 10.8.12</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Wednesday_8.8.12"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Wednesday 8.8.12</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Thursday_9.8.12"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Thursday 9.8.12</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Thursday_9.8.12_2"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Thursday 9.8.12</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Friday_10.8.12"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Friday 10.8.12</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Thursday_9.8.12_3"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Thursday 9.8.12</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Friday_10.8.12_2"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Friday 10.8.12</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Friday_10.8.12_3"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Friday 10.8.12</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Friday_10.8.12_4"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Friday 10.8.12</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Monday_6.7.12"> Monday 6.7.12 </SPAN></H2><P><B>Aim:</B> Inoculate the Plates that were transformed on Thursday. On Thursday we began Expt 8.5 by transforming the BioBricks, and our results on Friday indicated there was growth for all. (However, the negative control was contaminated, so we will have to be careful to assess the analytical restriction digest for correct products). The table below describes the BioBricks that were used:
</P><P>The two constitutive promoters were used because we were never able to detect our previous constitutive promoter BBa_J23119 (Expt 7.3) and our 3A ligation of BBa_J23119 and BBa_B0034 failed (Expt 8.4). This does not necessarily indicate our transformation of BBa_J23119 has failed, but we decided to transform several more in case. The RBS promoter BBa_B0030 was transformed for the same reason. The Gas Vesicle Cluster was transformed because all previous attempts have failed. 
</P><P><B>Method</B></P><DIV class="protocol protocol-ColPic">Picking Colonies Protocol</DIV><DIV class="protocolContent"><B>Step 1 – Creating culture media:</B> In a sterile environment, set up X numbers of falcons, each with 5mls of media. 
<P><B>Step 2 - Inoculating Colonies into a Selective Broth::</B> Add Yul of antibiotic to reach desired antibiotic concentration. 
</P><P>
(For Ampicillin this is 50ug/ml, For Kanamycin it is 25ug/ml, for Tetracycline it is 15ug/ml, and for Chloramphenicol  it is 25ug/ml) 
</P><P><B>Step 4 – Selecting a Colony:</B> Select a clear, isolated colony and using an inoculation hoop scoop up a colony onto the tip. Deposit in the falcon tube 
</P><P><B>Step 5 - Culture:</B>  Culture your falcon tubes overnight at a temperature of 37 oC. Leave for no longer than 16 hours.
</P></DIV><P><B>Step 2 – Inoculating Colonies into a Selective Broth:</B> The table below indicates the volume of broth and the concentration of antibiotic required for each BioBrick.
</P><TABLE class="wikitable"><TBODY><TR><TH colspan="2">  Samples </TH><TH> Volume Inoculated </TH><TH> Broth (ml) </TH><TH> Antibiotic (ug/ml)
</TH></TR><TR><TD rowspan="8">BioBrick </TD><TD rowspan="2"> BBa_J23100  </TD><TD> 10ul </TD><TD rowspan="8">Lysogeny Broth (5) </TD><TD rowspan="8"> Ampicillin(50ug/ml)
</TD></TR><TR><TD> 90ul
</TD></TR><TR><TD rowspan="2"> BBa_J23106 </TD><TD> 10ul
</TD></TR><TR><TD> 90ul
</TD></TR><TR><TD rowspan="2"> BBa_B0030  </TD><TD> 10ul
</TD></TR><TR><TD> 90ul
</TD></TR><TR><TD rowspan="2"> BBa_I750016  </TD><TD> 10ul
</TD></TR><TR><TD> 90ul
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Tuesday_7"> Tuesday 7 </SPAN></H2><P><B>Aim:</B> Results from Colony Picking
</P><P><B>Results</B>The table below indicates there was growth for all BioBricks.
</P><TABLE class="wikitable"><TBODY><TR><TH> Biobrick </TH><TH> Growth
</TH></TR><TR><TD> I750016 sample 1 </TD><TD> No Growth
</TD></TR><TR><TD> I750016 sample 2 </TD><TD> No Growth
</TD></TR><TR><TD> I750016 sample 3 </TD><TD> No Growth
</TD></TR><TR><TD> B0030 sample 1 </TD><TD> No Growth
</TD></TR><TR><TD> B0030 sample 2 </TD><TD> No Growth
</TD></TR><TR><TD> B0030 sample 3 </TD><TD> No Growth
</TD></TR><TR><TD> J23100 sample 1 </TD><TD> No Growth
</TD></TR><TR><TD> J23100 sample 2 </TD><TD> No Growth
</TD></TR><TR><TD> J23100 sample 3 </TD><TD> No Growth
</TD></TR><TR><TD> J23106 sample 1 </TD><TD> Growth
</TD></TR><TR><TD> J23106 sample 1 </TD><TD> Growth
</TD></TR><TR><TD> J23106 sample 1 </TD><TD> Growth
</TD></TR></TBODY></TABLE><P><B>Conclusion:</B> We are unsure why we have failed to get colonies for many of the BioBricks. Again, we are considering the possibility that our agar is not selective enough. Initially we thought this may be due to adding Antibiotic while the agar is too hot, leading to degradation of the sample. However this has been corrected in the protocol, but the problem is persisting. Instead we are considering running a troubleshoot by testing the selectivity of the various eppendorfs of antibiotic stocks we made at the beginning of the summer. If any show less selectivity then we can stop using them. This will be considered for later in the week. Meanwhile, we can miniprep and nanodrop BBa_J23106 (strong constitutive promoter) 
</P><DIV class="protocol protocol-Miniprep">Miniprep Protocol 1 (ANACHEM)</DIV><DIV class="protocolContent"><P><B>Step 1 - Pellet Cells:</B> Pellet 1.5-5ml bacterial culture containing the plasmid by centrifugation 
g = 6000 
</P><P>Time = 2 min 
</P><P>Temperature = (15-25oC)
</P><P><B>Step 2 - Resuspend Cells:</B> Add 250ul S1 to the pellet and resuspend the cells completely by vortexing or pipetting. Transfer the suspension to a clean 1.5ml microcentrifuge tube. 
</P><P><B>Step 3 - Puncturing Cell Membrane:</B> Add 250ul S2 gently mix by inverting the tube 4-6 times to obtain a clear lysate. Incubate on ice or at room temperature for NOT longer than 5 min.
</P><P><B>Step 4 - Neutralising S2:</B> Add 400ul Buffer NB and gently mix by inverting the tube 6-10 times, until a white precipitate forms. 
</P><P><B>Step 5 - Centrifuge:</B></P><P>g: 14000
</P><P>Time:10 minutes
</P><P>Temperature: (15-25oC)
</P><P><B>Step 6 - Centrifuge:</B> Transfer the supernatant into a column assembled in a clean collection tube (provided. Centrifuge:
</P><P>g = 10,000 
</P><P>Time: 1 minute
</P><P>Temperature: (15-25oC)
</P><P><B>Step 7 - Wash Column:</B> Discard the flow-through and wash the spin column by adding 700ul of Wash Buffer. Centrifuge:
</P><P>g - 10,000
</P><P>Time - 1 minute
</P><P>Temperature: (15-25oC)
</P><P><B>Step 8 - Remove residual ethanol:</B> Centrifuge:
</P><P>g - 10,000
</P><P>Time - 1 minute
</P><P>Temperature: (15-25oC)
</P><P><B>Step 9 - Elute DNA:</B> Place the column into a clean microcentrifuge tube. Add 50-100ul of Elution Buffer, TE buffer or sterile water directly onto column membrane and stand for 1 min. Centrifuge: 
</P><P>g - 10,000
</P><P>Time - 1 minute
</P><P>Temperature: (15-25oC)
</P><P><B>Step 10 - Storage:</B> Store DNA at 4oC or -20oC
</P></DIV><DIV class="protocol protocol-Electrophoresis">Electrophoresis Protocol</DIV><DIV class="protocolContent"><P><B>Preparing the Gel</B></P><P><B>Step 1:</B> Within a conical flask, add 3ml 50X TAE, 1.5g Agarose, and 150ml RO water
</P><P><B>Step 2:</B> Heat for 1 min in microwave. Swirl. Heat again for 30s. If solution is clear stop. Else repeat.
</P><P><B>Step 3:</B> Cool solution under running cold water. 
</P><P><B>Step 4:</B> Add 20ul ethidium bromide (normal concentration of EB solution is 500ug/ul)
</P><P><B>Step 5:</B> Pour into a sealed casting tray in a slow steady stream, ensuring there are no bubbles
</P><P><B>Running a gel</B></P><P><B>Step 6:</B> Add 1 part loading buffer to five parts of loading sample
</P><P><B>Step 7:</B> Position the gel in the tank and add TAE buffer, enough to cover the gel by several mm
</P><P><B>Step 8:</B> Add 5ul of DNA ladder to lane 1
</P><P><B>Step 9:</B> Add samples to the remaining wells
</P><P><B>Step 10:</B> Run at 100 volts for 1hour and 15 minutes
</P><P><B>Imaging the Gel</B></P><P><B>Step 11:</B> Place gel in GelDoc 2000 chamber
</P><P><B>Step 12:</B> Turn GelDoc 2000 chamber on 
</P><P><B>Step 13:</B> From computer: Quantity One &gt; Scanner &gt; Gel_Doc_Xr&gt;Manuqal Acquire
</P><P><B>Step 14:</B> Alter the exposure/settings to give a clear image.
</P><P>TAE - Tris-acetate-EDTA
</P><P>EDTA - ethylenediamine tetraacetic acid
</P></DIV><P><B>Results:</B> There was no product for BBa_J23106 on the gel, therefore either the transformation failed or the concentration is very low
</P></DIV><DIV class="experimentContent"><H2><SPAN class="mw-headline" id="Monday_6.7.12_2"> Monday 6.7.12 </SPAN></H2><P><B>Aim:</B> Repeat the inoculation of the ligation products (J23119 + B0034) done last week. Innoculation is only done for the (J23119 + B0034), but not for the (starvation promoter + B0034) because that ligation did not transform.
</P><P><B>Method</B></P><DIV class="protocol protocol-ColPic">Picking Colonies Protocol</DIV><DIV class="protocolContent"><B>Step 1 – Creating culture media:</B> In a sterile environment, set up X numbers of falcons, each with 5mls of media. 
<P><B>Step 2 - Inoculating Colonies into a Selective Broth::</B> Add Yul of antibiotic to reach desired antibiotic concentration. 
</P><P>
(For Ampicillin this is 50ug/ml, For Kanamycin it is 25ug/ml, for Tetracycline it is 15ug/ml, and for Chloramphenicol  it is 25ug/ml) 
</P><P><B>Step 4 – Selecting a Colony:</B> Select a clear, isolated colony and using an inoculation hoop scoop up a colony onto the tip. Deposit in the falcon tube 
</P><P><B>Step 5 - Culture:</B>  Culture your falcon tubes overnight at a temperature of 37 oC. Leave for no longer than 16 hours.
</P></DIV><P>Step 2 – Inoculating Colonies into a Selective Broth: The table below indicates the volume of broth and the concentration of antibiotic required for each BioBrick.
</P><TABLE class="wikitable"><TBODY><TR><TH> Ligation </TH><TH> No. innoculations </TH><TH> Broth </TH><TH> Antibiotic
</TH></TR><TR><TD> J23119 + B0034 </TD><TD> 3 </TD><TD> Lysogeny Broth </TD><TD> Kanamycin
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Tuesday_7_2"> Tuesday 7 </SPAN></H2><P><B>Aim:</B> Check results of Colony Picking
</P><P><B>Results:</B> The table below indicates whether there was growth for the BioBricks ligations
</P><TABLE class="wikitable"><TBODY><TR><TH> Ligation </TH><TH> Growth/No Growth
</TH></TR><TR><TD> J23119+B0034 sample 1 </TD><TD> Growth
</TD></TR><TR><TD> J23119+B0034 sample 2</TD><TD> Growth
</TD></TR><TR><TD> J23119+B0034 sample 3 </TD><TD> Growth
</TD></TR></TBODY></TABLE><P><B>Conclusion:</B> We can proceed now to miniprep the samples, and run them on a gel to test the presence of a band
</P><P><B>Method:</B></P><DIV class="protocol protocol-Miniprep">Miniprep Protocol 1 (ANACHEM)</DIV><DIV class="protocolContent"><P><B>Step 1 - Pellet Cells:</B> Pellet 1.5-5ml bacterial culture containing the plasmid by centrifugation 
g = 6000 
</P><P>Time = 2 min 
</P><P>Temperature = (15-25oC)
</P><P><B>Step 2 - Resuspend Cells:</B> Add 250ul S1 to the pellet and resuspend the cells completely by vortexing or pipetting. Transfer the suspension to a clean 1.5ml microcentrifuge tube. 
</P><P><B>Step 3 - Puncturing Cell Membrane:</B> Add 250ul S2 gently mix by inverting the tube 4-6 times to obtain a clear lysate. Incubate on ice or at room temperature for NOT longer than 5 min.
</P><P><B>Step 4 - Neutralising S2:</B> Add 400ul Buffer NB and gently mix by inverting the tube 6-10 times, until a white precipitate forms. 
</P><P><B>Step 5 - Centrifuge:</B></P><P>g: 14000
</P><P>Time:10 minutes
</P><P>Temperature: (15-25oC)
</P><P><B>Step 6 - Centrifuge:</B> Transfer the supernatant into a column assembled in a clean collection tube (provided. Centrifuge:
</P><P>g = 10,000 
</P><P>Time: 1 minute
</P><P>Temperature: (15-25oC)
</P><P><B>Step 7 - Wash Column:</B> Discard the flow-through and wash the spin column by adding 700ul of Wash Buffer. Centrifuge:
</P><P>g - 10,000
</P><P>Time - 1 minute
</P><P>Temperature: (15-25oC)
</P><P><B>Step 8 - Remove residual ethanol:</B> Centrifuge:
</P><P>g - 10,000
</P><P>Time - 1 minute
</P><P>Temperature: (15-25oC)
</P><P><B>Step 9 - Elute DNA:</B> Place the column into a clean microcentrifuge tube. Add 50-100ul of Elution Buffer, TE buffer or sterile water directly onto column membrane and stand for 1 min. Centrifuge: 
</P><P>g - 10,000
</P><P>Time - 1 minute
</P><P>Temperature: (15-25oC)
</P><P><B>Step 10 - Storage:</B> Store DNA at 4oC or -20oC
</P></DIV><DIV class="protocol protocol-Electrophoresis">Electrophoresis Protocol</DIV><DIV class="protocolContent"><P><B>Preparing the Gel</B></P><P><B>Step 1:</B> Within a conical flask, add 3ml 50X TAE, 1.5g Agarose, and 150ml RO water
</P><P><B>Step 2:</B> Heat for 1 min in microwave. Swirl. Heat again for 30s. If solution is clear stop. Else repeat.
</P><P><B>Step 3:</B> Cool solution under running cold water. 
</P><P><B>Step 4:</B> Add 20ul ethidium bromide (normal concentration of EB solution is 500ug/ul)
</P><P><B>Step 5:</B> Pour into a sealed casting tray in a slow steady stream, ensuring there are no bubbles
</P><P><B>Running a gel</B></P><P><B>Step 6:</B> Add 1 part loading buffer to five parts of loading sample
</P><P><B>Step 7:</B> Position the gel in the tank and add TAE buffer, enough to cover the gel by several mm
</P><P><B>Step 8:</B> Add 5ul of DNA ladder to lane 1
</P><P><B>Step 9:</B> Add samples to the remaining wells
</P><P><B>Step 10:</B> Run at 100 volts for 1hour and 15 minutes
</P><P><B>Imaging the Gel</B></P><P><B>Step 11:</B> Place gel in GelDoc 2000 chamber
</P><P><B>Step 12:</B> Turn GelDoc 2000 chamber on 
</P><P><B>Step 13:</B> From computer: Quantity One &gt; Scanner &gt; Gel_Doc_Xr&gt;Manuqal Acquire
</P><P><B>Step 14:</B> Alter the exposure/settings to give a clear image.
</P><P>TAE - Tris-acetate-EDTA
</P><P>EDTA - ethylenediamine tetraacetic acid
</P></DIV><P><B>Results:</B> There was no product on the gel, suggesting the ligation failed. 
</P></DIV><DIV class="experimentContent"><H2><SPAN class="mw-headline" id="Monday_8.8.12"> Monday 8.8.12</SPAN></H2><P><B>Aim:</B> To repeat Expt 8.2, where we undertook PCR amplification of pSB1A3, pSB1C3, pSB1K3 for 3A assembly. While we achieved bands of the correct size for all three plasmid backbones in Expt 8.2, the nanodrop concentration for pSB1A3 and pSB1C3 was very low (unusable). Therefore we decided to repeat the PCR. As the concentration of pSB1K3 was not particularly high, we decided to repeat this again also.
</P><P><B>Method:</B></P><DIV class="protocol protocol-PCR">PCR Protocol</DIV><DIV class="protocolContent"><P><B>Step 1 - Setting up PCR tubes:</B> Thaw reagents and add to PCR tubes in the proportions described in the table below
</P><TABLE class="wikitable"><TBODY><TR><TH> PCR Components </TH><TH> Volume (ul)
</TH></TR><TR><TD> 5x Reaction Buffer </TD><TD> 10
</TD></TR><TR><TD> 25mM MgCl2 </TD><TD> 4
</TD></TR><TR><TD> 10mM dNTPs </TD><TD> 1
</TD></TR><TR><TD> 10uM Forward primer </TD><TD> 5
</TD></TR><TR><TD> 10uM Reverse primer </TD><TD> 5
</TD></TR><TR><TD> DNA Polymerase </TD><TD> 0.25
</TD></TR><TR><TD> Nuclease Free Water </TD><TD> 24.25
</TD></TR><TR><TD> Template DNA </TD><TD> 0.5
</TD></TR><TR><TD> Total Volume </TD><TD> 0.5
</TD></TR></TBODY></TABLE><P><B>Step 2 - PCR program:</B> Add PCR tubes to a thermocycler and run under the following conditions. 
</P><TABLE class="wikitable"><TBODY><TR><TH> PCR conditions </TH><TH> Temp (<SUP>o</SUP>C) </TH><TH> Time (s)
</TH></TR><TR><TD> Initial Denaturation (1 cycle) </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> Denaturation/Annealing/Extension (30 cycles) </TD><TD> 95/55/72 </TD><TD> 10/25/120
</TD></TR><TR><TD> Final Extension (1 cycle) </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> Hold </TD><TD> 4 </TD><TD> ∞
</TD></TR></TBODY></TABLE></DIV><DIV class="protocol protocol-Electrophoresis">Electrophoresis Protocol</DIV><DIV class="protocolContent"><P><B>Preparing the Gel</B></P><P><B>Step 1:</B> Within a conical flask, add 3ml 50X TAE, 1.5g Agarose, and 150ml RO water
</P><P><B>Step 2:</B> Heat for 1 min in microwave. Swirl. Heat again for 30s. If solution is clear stop. Else repeat.
</P><P><B>Step 3:</B> Cool solution under running cold water. 
</P><P><B>Step 4:</B> Add 20ul ethidium bromide (normal concentration of EB solution is 500ug/ul)
</P><P><B>Step 5:</B> Pour into a sealed casting tray in a slow steady stream, ensuring there are no bubbles
</P><P><B>Running a gel</B></P><P><B>Step 6:</B> Add 1 part loading buffer to five parts of loading sample
</P><P><B>Step 7:</B> Position the gel in the tank and add TAE buffer, enough to cover the gel by several mm
</P><P><B>Step 8:</B> Add 5ul of DNA ladder to lane 1
</P><P><B>Step 9:</B> Add samples to the remaining wells
</P><P><B>Step 10:</B> Run at 100 volts for 1hour and 15 minutes
</P><P><B>Imaging the Gel</B></P><P><B>Step 11:</B> Place gel in GelDoc 2000 chamber
</P><P><B>Step 12:</B> Turn GelDoc 2000 chamber on 
</P><P><B>Step 13:</B> From computer: Quantity One &gt; Scanner &gt; Gel_Doc_Xr&gt;Manuqal Acquire
</P><P><B>Step 14:</B> Alter the exposure/settings to give a clear image.
</P><P>TAE - Tris-acetate-EDTA
</P><P>EDTA - ethylenediamine tetraacetic acid
</P></DIV><P><B>Results:</B> The image below shows a 1% Agarose Gel of an Analytical Restriction Enzyme Digest for Expt 9.1, with a 1000bp ladder. No products were detected. We expected a product corresponding to the size of the plasmid pSB1K3 (2000bp), pSB1C3 (2000bp) and pSB1A3 (2000bp) in Lanes 1-3 respectively. These are indicated by the letters A (pSB1K3), B (pSB1C3) and C (pSB1A3). 
</P><TABLE class="wikitable"><TBODY><TR><TH> Backbone </TH><TH> Expected Size
</TH></TR><TR><TD> pSB1K3 </TD><TD> 2204bp
</TD></TR><TR><TD> pSB1C3 </TD><TD> 2070bp
</TD></TR><TR><TD> pSB1A3</TD><TD> 2155bp
</TD></TR></TBODY></TABLE><P><B>Conclusion:</B>  We feel the poor nanodrop results from Expt 8.2, and the poor electrophoresis results from this Expt may be a result of the choice of polymerase. Our supervisors recommended we try Phusion Polymerase, which we will attempt tomorrow.
</P><H2><SPAN class="mw-headline" id="Tuesday_9.8.12"> Tuesday 9.8.12 </SPAN></H2><P><B>Aim:</B> To repeat PCR with Phusion Polymerase to amplify the three plasmid backbones. As we had more confidence that this PCR reaction would work, we included an extra plasmid, pSB1T3, also required for ligation. 
</P><P><B>Method</B></P><DIV class="protocol protocol-PCR">PCR Protocol</DIV><DIV class="protocolContent"><P><B>Step 1 - Setting up PCR tubes:</B> Thaw reagents and add to PCR tubes in the proportions described in the table below
</P><TABLE class="wikitable"><TBODY><TR><TH> PCR Components </TH><TH> Volume (ul)
</TH></TR><TR><TD> 5x Reaction Buffer </TD><TD> 10
</TD></TR><TR><TD> 25mM MgCl2 </TD><TD> 4
</TD></TR><TR><TD> 10mM dNTPs </TD><TD> 1
</TD></TR><TR><TD> 10uM Forward primer </TD><TD> 5
</TD></TR><TR><TD> 10uM Reverse primer </TD><TD> 5
</TD></TR><TR><TD> DNA Polymerase </TD><TD> 0.25
</TD></TR><TR><TD> Nuclease Free Water </TD><TD> 24.25
</TD></TR><TR><TD> Template DNA </TD><TD> 0.5
</TD></TR><TR><TD> Total Volume </TD><TD> 0.5
</TD></TR></TBODY></TABLE><P><B>Step 2 - PCR program:</B> Add PCR tubes to a thermocycler and run under the following conditions. 
</P><TABLE class="wikitable"><TBODY><TR><TH> PCR conditions </TH><TH> Temp (<SUP>o</SUP>C) </TH><TH> Time (s)
</TH></TR><TR><TD> Initial Denaturation (1 cycle) </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> Denaturation/Annealing/Extension (30 cycles) </TD><TD> 95/55/72 </TD><TD> 10/25/120
</TD></TR><TR><TD> Final Extension (1 cycle) </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> Hold </TD><TD> 4 </TD><TD> ∞
</TD></TR></TBODY></TABLE></DIV><DIV class="protocol protocol-Electrophoresis">Electrophoresis Protocol</DIV><DIV class="protocolContent"><P><B>Preparing the Gel</B></P><P><B>Step 1:</B> Within a conical flask, add 3ml 50X TAE, 1.5g Agarose, and 150ml RO water
</P><P><B>Step 2:</B> Heat for 1 min in microwave. Swirl. Heat again for 30s. If solution is clear stop. Else repeat.
</P><P><B>Step 3:</B> Cool solution under running cold water. 
</P><P><B>Step 4:</B> Add 20ul ethidium bromide (normal concentration of EB solution is 500ug/ul)
</P><P><B>Step 5:</B> Pour into a sealed casting tray in a slow steady stream, ensuring there are no bubbles
</P><P><B>Running a gel</B></P><P><B>Step 6:</B> Add 1 part loading buffer to five parts of loading sample
</P><P><B>Step 7:</B> Position the gel in the tank and add TAE buffer, enough to cover the gel by several mm
</P><P><B>Step 8:</B> Add 5ul of DNA ladder to lane 1
</P><P><B>Step 9:</B> Add samples to the remaining wells
</P><P><B>Step 10:</B> Run at 100 volts for 1hour and 15 minutes
</P><P><B>Imaging the Gel</B></P><P><B>Step 11:</B> Place gel in GelDoc 2000 chamber
</P><P><B>Step 12:</B> Turn GelDoc 2000 chamber on 
</P><P><B>Step 13:</B> From computer: Quantity One &gt; Scanner &gt; Gel_Doc_Xr&gt;Manuqal Acquire
</P><P><B>Step 14:</B> Alter the exposure/settings to give a clear image.
</P><P>TAE - Tris-acetate-EDTA
</P><P>EDTA - ethylenediamine tetraacetic acid
</P></DIV><P><B>Results:</B> The image below shows a 1% Agarose Gel of an Analytical Restriction Enzyme Digest for Expt 9.1, with a 1000bp ladder. In Lane 1 can be seen a product corresponding to the correct size of pSB1A3 (2155) as indicated by A. In Lane 2 is a product, shown by B, which is of the correct size for the plasmid pSB1K3 (22040bp). In Lane 3, we expect a band of size 2461bp, for the plasmid pSB1T3. This is shown by C. As it is absent, it would appear the PCR reaction failed. In Lane 4 is a product corresponding to pSB1C3 (2070bp) as shown by D. 
</P><TABLE class="wikitable"><TBODY><TR><TH> Backbone </TH><TH> Expected Size
</TH></TR><TR><TD> pSB1K3 </TD><TD> 2204bp
</TD></TR><TR><TD> pSB1C3 </TD><TD> 2070bp
</TD></TR><TR><TD> pSB1A3</TD><TD> 2155bp
</TD></TR><TR><TD> pSB1T3 </TD><TD> 2461bp
</TD></TR></TBODY></TABLE><P><B>Conclusion:</B> The PCR reaction worked for pSB1K3, pSB1C3 and pSB1A3. This would suggest the Phusion Polymerase is a better choice than Taq polymerase. These plasmids can now be nanodropped, and if their concentration is sufficient they can be used for ligation.
</P><H2><SPAN class="mw-headline" id="Thursday_10.8.12"> Thursday 10.8.12 </SPAN></H2><P><B>Aim:</B> To carry out nanodrop for the PCR reaction of plasmids with phusion polymerase
</P><P><B>Method</B></P><DIV class="protocol protocol-Nanodrop">Nanodrop Protocol</DIV><DIV class="protocolContent"><P>Software ND-1000
Model:
</P><P><B>Step 1:</B> Initialise the spectrophotometer by pipetting 1 µ of clean water onto lower optic surface, lowering the lever arm and selecting ‘initialise’ in the ND-1000 software
</P><P><B>Step 2:</B> Wipe and add elution buffer as negative control. Click blank in ND-1000 software
</P><P><B>Step 3:</B> Wipe and add 1 µl sample
</P><P><B>Step 4:</B> On the software set lambda to 260nm
</P><P><B>Step 5:</B> Lower the lever arm and click measure in ND-1000 software
</P><P><B>Step 6:</B> Take readings for concentration and purity
</P><P><B>Step 7:</B> Once measurement complete, wipe surface
</P></DIV><TABLE class="wikitable"><TBODY><TR><TH> Plasmid </TH><TH> λ260 </TH><TH> λ 280
</TH></TR><TR><TD> pSB1A3 (ng/μl) </TD><TD> 45.7 </TD><TD> 45.4
</TD></TR><TR><TD> pSB1C3 (ng/μl) </TD><TD> 52.2 </TD><TD> 48.0
</TD></TR><TR><TD> pSB1K3 (ng/μl) </TD><TD> 57.9 </TD><TD> 59.4
</TD></TR></TBODY></TABLE><P><B>Conclusion:</B> The results of the nanodrop indicate that we have usable concentrations of the plasmids, though they are not particularly high. However, we can attempt to use these for 3A ligation.
</P></DIV><DIV class="experimentContent"><H2><SPAN class="mw-headline" id="Wednesday_8.8.12"> Wednesday 8.8.12 </SPAN></H2><P><B>Aim:</B> To extract the Laccase gene for the Degradation module and the IrrE gene for the Salt Tolerance module.
</P><P><B>Method:</B></P><P>DNA for the laccase gene was extracted from cultured colonies according to the protocol below:
</P><DIV class="protocol protocol-DNAExtractionFromColonies">DNA Extraction From Colonies Protocol</DIV><DIV class="protocolContent"><P><B>Step 1 - Picking Colony:</B> Using a pipette select an isolated colony
</P><P><B>Step 2 - Deposit:</B> Dip the colony in 10ul of dH20, preferably in screw top container
</P><P><B>Step 3 - Boiling Stage:</B>Boil the dH20 100 for 5-10mins
</P><P><B>Step 4 - Centrifuge:</B></P><P>Time: 1 min
</P><P>RPM: 8000
</P><P>Temp: 18oC
</P><P><B>Step 5 - Remove Supernatant:</B> The supernatant contains the DNA template.
</P></DIV><P>DNA for IrrE was extracted by Genomic Extraction
</P><P>Protocol: Genomic Extraction
</P><DIV class="protocol protocol-PrimerDesign">Primer Design Protocol</DIV><DIV class="protocolContent"><P>Primers should be designed to: 
</P><P>1) Be 18-25 nucleotides in length
</P><P>2) Have a melting temperature that does not differ by more than 5oC between the two primers of a pair
</P><P>3) GC nucleotides should constitute 40-60%
</P><P>4) Have a 3’primer end that is not complimentary to any region of the primer pair
</P><P>5) Contain an equal distribution of the nucleotides
</P><P>6) Closely match the target sequence (and so known polymorphic sites are avoided)
</P><P>7) Have a GC clamp on either end
</P></DIV><P>The design of the primers is shown by the table below
</P><DIV class="protocol protocol-PCR">PCR Protocol</DIV><DIV class="protocolContent"><P><B>Step 1 - Setting up PCR tubes:</B> Thaw reagents and add to PCR tubes in the proportions described in the table below
</P><TABLE class="wikitable"><TBODY><TR><TH> PCR Components </TH><TH> Volume (ul)
</TH></TR><TR><TD> 5x Reaction Buffer </TD><TD> 10
</TD></TR><TR><TD> 25mM MgCl2 </TD><TD> 4
</TD></TR><TR><TD> 10mM dNTPs </TD><TD> 1
</TD></TR><TR><TD> 10uM Forward primer </TD><TD> 5
</TD></TR><TR><TD> 10uM Reverse primer </TD><TD> 5
</TD></TR><TR><TD> DNA Polymerase </TD><TD> 0.25
</TD></TR><TR><TD> Nuclease Free Water </TD><TD> 24.25
</TD></TR><TR><TD> Template DNA </TD><TD> 0.5
</TD></TR><TR><TD> Total Volume </TD><TD> 0.5
</TD></TR></TBODY></TABLE><P><B>Step 2 - PCR program:</B> Add PCR tubes to a thermocycler and run under the following conditions. 
</P><TABLE class="wikitable"><TBODY><TR><TH> PCR conditions </TH><TH> Temp (<SUP>o</SUP>C) </TH><TH> Time (s)
</TH></TR><TR><TD> Initial Denaturation (1 cycle) </TD><TD> 95 </TD><TD> 30
</TD></TR><TR><TD> Denaturation/Annealing/Extension (30 cycles) </TD><TD> 95/55/72 </TD><TD> 10/25/120
</TD></TR><TR><TD> Final Extension (1 cycle) </TD><TD> 72 </TD><TD> 600
</TD></TR><TR><TD> Hold </TD><TD> 4 </TD><TD> ∞
</TD></TR></TBODY></TABLE></DIV><P><B>Step 1 - Setting up PCR tubes</B> The table below gives the identity of the primers used for each reaction.
</P><TABLE class="wikitable"><TBODY><TR><TH> DNA Extracted </TH><TH> Function </TH><TH> Module</TH><TH> Primer Pair</TH><TH> Primer</TH><TH> Primer Sequence
</TH></TR><TR><TD rowspan="4">BBa_K729002 </TD><TD rowspan="4"> Laccase Gene</TD><TD rowspan="4">Degradation </TD><TD rowspan="2"> LR1/LF1 </TD><TD>LR1 </TD><TD>gaatacggtctttttataccg
</TD></TR><TR><TD>  LF1</TD><TD> gaaataactatgcaacgtcg
</TD></TR><TR><TD rowspan="2">REVLF2/LFTW0 </TD><TD>REVLF2</TD><TD> gtttcttcctgcagcggccgctactagtagaatacggtctttttataccg
</TD></TR><TR><TD>LFTW0 </TD><TD> gtttcttcgaattcgcggccgcttctagaggaaataactatgcaacgtcg
</TD></TR><TR><TD rowspan="4">BBa_K729001 </TD><TD rowspan="4">IrrE</TD><TD rowspan="4">Salt Tolerance </TD><TD rowspan="2"> STF1/ST2R</TD><TD>STF1 </TD><TD> atggggccaaaagctaaagctgaagcc
</TD></TR><TR><TD>   ST2R </TD><TD>tcactgtgcagcgtcctgcg
</TD></TR><TR><TD rowspan="2"> STF3/ST4R </TD><TD> STF3 </TD><TD> gtttcttcgaattcgcggccgcttctagagatggggccaaaagctaaagctgaagcc
</TD></TR><TR><TD> ST4R </TD><TD> gtttcttcctgcagcggccgctactagtatcactgtgcagcgtcctgcg
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Thursday_9.8.12"> Thursday 9.8.12 </SPAN></H2><P><B>Aim:</B> Run a gel for the PCR reaction of irrE and Laccase carried out yesterday. 
</P><P><B>Method:</B></P><DIV class="protocol protocol-Electrophoresis">Electrophoresis Protocol</DIV><DIV class="protocolContent"><P><B>Preparing the Gel</B></P><P><B>Step 1:</B> Within a conical flask, add 3ml 50X TAE, 1.5g Agarose, and 150ml RO water
</P><P><B>Step 2:</B> Heat for 1 min in microwave. Swirl. Heat again for 30s. If solution is clear stop. Else repeat.
</P><P><B>Step 3:</B> Cool solution under running cold water. 
</P><P><B>Step 4:</B> Add 20ul ethidium bromide (normal concentration of EB solution is 500ug/ul)
</P><P><B>Step 5:</B> Pour into a sealed casting tray in a slow steady stream, ensuring there are no bubbles
</P><P><B>Running a gel</B></P><P><B>Step 6:</B> Add 1 part loading buffer to five parts of loading sample
</P><P><B>Step 7:</B> Position the gel in the tank and add TAE buffer, enough to cover the gel by several mm
</P><P><B>Step 8:</B> Add 5ul of DNA ladder to lane 1
</P><P><B>Step 9:</B> Add samples to the remaining wells
</P><P><B>Step 10:</B> Run at 100 volts for 1hour and 15 minutes
</P><P><B>Imaging the Gel</B></P><P><B>Step 11:</B> Place gel in GelDoc 2000 chamber
</P><P><B>Step 12:</B> Turn GelDoc 2000 chamber on 
</P><P><B>Step 13:</B> From computer: Quantity One &gt; Scanner &gt; Gel_Doc_Xr&gt;Manuqal Acquire
</P><P><B>Step 14:</B> Alter the exposure/settings to give a clear image.
</P><P>TAE - Tris-acetate-EDTA
</P><P>EDTA - ethylenediamine tetraacetic acid
</P></DIV><P><B>Results:</B> The image belows shows that the gel failed to produce any products of the correct size. We feel the best explanation of this is that there was human error somewhere along the lines – leading to a mix up of the samples. 
</P></DIV><DIV class="experimentContent"><H2><SPAN class="mw-headline" id="Thursday_9.8.12_2"> Thursday 9.8.12 </SPAN></H2><P><B>Aim:</B> inoculation of marine bacterium. We aim to grow up a stock of the marine bacterium Oceanibulbus indolifex, which has been selected as one of the chassis we wish to investigate. 
</P><P><B>Method</B>
Oceanibulbus was ordered from NCMB and recieved as a liquid stock. 
</P><P>Step 1: Prepare agar plates
</P><P>Step 2: Dip an inoculation loop into the liquid stock of bacteria and spread over the surface of the agar
</P><P>Step 3: Leave the bacteria to grow overnight or over the weekend.
</P><H2><SPAN class="mw-headline" id="Friday_10.8.12"> Friday 10.8.12 </SPAN></H2><P><B>Aim:</B> Check the results of O.indolifex culture
</P><P><B>Results:</B> The image below indicates there was growth for both inoculations of O.indolifex. 
</P></DIV><DIV class="experimentContent"><H2><SPAN class="mw-headline" id="Thursday_9.8.12_3"> Thursday 9.8.12 </SPAN></H2><P><B>Aim:</B> To check the selectivity of our antibiotic stocks. After growth on agar plates, our transformations frequently fail to grow after inoculation into falcons. This has led us to consider the possibility that the selection has not been strong enough, leading to the growth of untransformed cells. We have designed an experiment to test whether our stocks of antibiotics are effective enough. Numerous agar plates will be made with varying antibiotic resistance, and will be inoculated with a variety of antibiotic-resistant and non-resistant bacteria. If non-resistant bacteria can grow on the antibiotic-inoculated agar, we will have some indication of whether the antibiotic activity is high enough.
</P><P><B>Method:</B>
Making plates
</P><P>
The table below indicates the plates that were made with each antibiotic. NB// we have several eppendorf stocks of antibiotics; the plate was labelled according to the eppendorf stock it came from. This is intended to identify any of our stocks which may not be of sufficient quality to create a selective agar. 
</P><TABLE class="wikitable"><TBODY><TR><TH> Antibiotic Stock </TH><TH> Presumed Concentration
</TH></TR><TR><TD> Kanamycin 01 </TD><TD> 50ug/ml
</TD></TR><TR><TD> Chloramphenicol 01 </TD><TD> 50ug/ml
</TD></TR><TR><TD> Chloramphenicol 02 </TD><TD> 50ug/ml
</TD></TR><TR><TD> Chloramphenicol 03 </TD><TD> 50ug/ml
</TD></TR><TR><TD> Tetracycline 01 </TD><TD> 50ug/ml
</TD></TR><TR><TD> Tetracycline 02</TD><TD> 50ug/ml
</TD></TR><TR><TD> Tetracycline 03 </TD><TD> 50ug/ml
</TD></TR><TR><TD> Ampicillin 01 </TD><TD> 50ug/ml
</TD></TR><TR><TD> Ampicillin 02</TD><TD> 50ug/ml
</TD></TR><TR><TD> Ampicillin 03 </TD><TD> 50ug/ml
</TD></TR><TR><TD> Ampicillin 04 </TD><TD> 50ug/ml
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Friday_10.8.12_2"> Friday 10.8.12 </SPAN></H2><P><B>Aim:</B> To set up our samples on the plates made yesterday
</P><P><B>Method:</B> the plates from yesterday have been split into half (in order to prevent wasting resources). Each side is being treated as a separate sample.
</P><P>
The following table summarises the type of plates and their contents. Top10, W3110 and pTopDsh have no antibiotic resistance, and so they are our negative controls – there should be no growth except on drug-free agar. WNu has Ampicillin resistant, and should go in the presence of Ampicillin but not other antiobiotics. pTopDsh carries Tetracycline resistance, and should grow in the presence of Tetracycline but not the other antibiotics. 
</P><TABLE class="wikitable"><TBODY><TR><TH> Antibiotic Stock </TH><TH> Sample Added </TH><TH> Expected Result </TH><TH> Result
</TH></TR><TR><TD rowspan="2"> Kanamycin 01 </TD><TD> WNu </TD><TD> No growth (WNu is Ampicillin resistant only) </TD><TD> no colony
</TD></TR><TR><TD> WNu</TD><TD> No growth (WNu is Ampicillin resistant only)</TD><TD> no colony
</TD></TR><TR><TD rowspan="2"> Chloramphenicol 01 </TD><TD> Cobalt Curli Cells  </TD><TD> Growth (Cobalt Curli cells is Chloramphenicol resistant) </TD><TD> colonies
</TD></TR><TR><TD> WNu </TD><TD> No growth (WNu is Ampicillin resistant only)</TD><TD> no colony
</TD></TR><TR><TD rowspan="2"> Chloramphenicol 02 </TD><TD>Cobalt Curli Cells </TD><TD> Growth (Cobalt Curli cells is Chloramphenicol resistant) </TD><TD> colonies
</TD></TR><TR><TD> WNu </TD><TD> No growth (WNu is Ampicillin resistant only)</TD><TD> no colony
</TD></TR><TR><TD rowspan="2"> Chloramphenicol 03 </TD><TD> Cobalt Curli Cells</TD><TD> Growth (Cobalt Curli cells is Chloramphenicol resistant) </TD><TD> colonies
</TD></TR><TR><TD>W3110 </TD><TD> No growth (W3110 has no resistance)</TD><TD> no colony
</TD></TR><TR><TD rowspan="2"> Tetracycline 01</TD><TD> WNu </TD><TD>No growth (WNu is Ampicillin resistant only)</TD><TD> no colony
</TD></TR><TR><TD>75 </TD><TD>Growth (75 is Tetracycline Resistant)</TD><TD> colonies
</TD></TR><TR><TD rowspan="2"> Tetracycline 02</TD><TD> WNu </TD><TD>No growth (WNu is Ampicillin resistant only)</TD><TD> no colony
</TD></TR><TR><TD>75 </TD><TD>Growth (75 is Tetracycline Resistant)</TD><TD> colonies
</TD></TR><TR><TD rowspan="2"> Tetracycline 03</TD><TD> WNu </TD><TD>No growth (WNu is Ampicillin resistant only)</TD><TD> no colony
</TD></TR><TR><TD>75 </TD><TD>Growth (75 is Tetracycline Resistant)</TD><TD> no colony
</TD></TR><TR><TD rowspan="2">  Ampicillin 01 </TD><TD> pTopDsh </TD><TD> No Growth (pTopDsh has no resistance)</TD><TD> no colony
</TD></TR><TR><TD>WNu</TD><TD>No growth (WNu is Ampicillin resistant only)</TD><TD> colonies
</TD></TR><TR><TD rowspan="2"> Ampicillin 02</TD><TD> WNu </TD><TD> Growth (WNu is Ampicillin resistant)</TD><TD> no colony
</TD></TR><TR><TD>W3100 </TD><TD> No Growth (W3100 has no resistance)</TD><TD> colonies
</TD></TR><TR><TD rowspan="2"> Ampicillin 03 </TD><TD> WNu </TD><TD>Growth (WNu is Ampicillin resistant)</TD><TD> no colony
</TD></TR><TR><TD> pTopDsh </TD><TD> No Growth (pTopDsh has no resistance)</TD><TD> no colony
</TD></TR><TR><TD rowspan="2">  Ampicillin 04 </TD><TD> WNu </TD><TD>Growth (WNu is Ampicillin resistant)</TD><TD> colonies
</TD></TR><TR><TD> pTopDsh </TD><TD> No Growth (pTopDsh has no resistance)</TD><TD> no colony
</TD></TR><TR><TD rowspan="5">  Control - No Drug </TD><TD> W3110 </TD><TD> Growth</TD><TD>colonies
</TD></TR><TR><TD> Top10 </TD><TD> Growth</TD><TD>‎ colonies
</TD></TR><TR><TD> 75 </TD><TD> Growth</TD><TD> colonies
</TD></TR><TR><TD> CoCurli </TD><TD> Growth</TD><TD> colonies
</TD></TR><TR><TD>pTopDsh </TD><TD> Growth</TD><TD> colonies
</TD></TR></TBODY></TABLE><P>Plates results are as expected, no problem with the activity of the antibiotics however we are not sure if the antibiotic are at the concentration stated.   
</P></DIV><DIV class="experimentContent"><H2><SPAN class="mw-headline" id="Friday_10.8.12_3"> Friday 10.8.12 </SPAN></H2><P><B>Aim:</B> To characterise the unmodified Curli BioBrick. As part of our experiment, we want to change the promoter in front of the Curli cluster of genes. This experiment will test the protocol we intend to use using the original BioBrick (BBa_K540000)
</P><P><B>Method</B></P><P>Congo Red
</P><DIV class="protocol protocol-CurliCharacterisation">Curli Characterisation Protocol</DIV><DIV class="protocolContent"><P><B>Step 1 - Prepare:</B> Set out the Congo Red plates already prepared. 
</P><P><B>Step 2 - Streak Cells:</B> Streak a sample of control cells (cells not expressing Curli) onto one antiobiotic-free congo red plate, and onto one antibiotic-inoculated congo red plate. Do the same for the curli-expressing cell line.
</P><P><B>Step 3 - Incubate:</B> Incubate cells overnight at 37oC or over the weekend at 30oC. 
</P></DIV><P><B>Step 2 - Streak Cells:</B> Streak a sample of WNu onto a Antibiotic-inoculated Congo Red plate and a Antibiotic-Free Congo Red Plate. Do the same for the CoCurli (BBa_K540000 transformed) cells. The table below indicates the results expected for each of these four plates. 
</P><TABLE class="wikitable"><TBODY><TR><TH> Cell Type </TH><TH> Expected Results on Antibiotic-free Congo Red Agar </TH><TH> Expected Results on Chloramphenicol-inoculated Congo Red Agar
</TH></TR><TR><TD>  WNu </TD><TD> Formation of White Colonies </TD><TD> No Colony Formation (WNu are not Chloramphenicol-resistant)
</TD></TR><TR><TD>  K540000 (cobalt curli) transformed cells </TD><TD> Formation of Red Colonies </TD><TD>  Formation of Red Colonies
</TD></TR></TBODY></TABLE><P><B>Step 3 - Incubate:</B> A sample of each were cultured at 30 degrees over the weekend because Curli production is expressed better at lower temperatures. 
</P></DIV><DIV class="experimentContent"><H2><SPAN class="mw-headline" id="Friday_10.8.12_4"> Friday 10.8.12</SPAN></H2><P><B>Aim:</B> To assay the nuclease activity in a test of the protocol. We will use a cell line called WNu which has native secreted nuclease activity. This, alongside a nuclease-negative control, will be cultured on agar containing DNA. Nuclease activity will be indicated by the presence of 'halos' surrounding the colonies, where nuclease activity has degraded DNA and increased the translucence of the agar.
</P><P><B>Method</B>
Prepare DNAase agar +IPTG + amp.
</P><P><B>Results</B>
The table below summarises the results from this experiment. Also included is an image of the agars, with the halo of nuclease activity indicated. 
</P><TABLE class="wikitable"><TBODY><TR><TH> Sample </TH><TH> Ampicillin + IPTG </TH><TH> Ampicillin </TH><TH> No Antibiotic
</TH></TR><TR><TD>Control BBa_K540000 transformed cells </TD><TD> Opaque </TD><TD> Opaque </TD><TD> Opaque
</TD></TR><TR><TD> WNu Cells </TD><TD> Halo Formation </TD><TD> Halo Formation </TD><TD> Halo Formation
</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:University_College_London/LabBook/Week9">http://2012.igem.org/Team:University_College_London/LabBook/Week9</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:University_College_London/LabBook/Week9" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:University_College_London/LabBook/Week9" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:University_College_London/LabBook/Week9&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:University_College_London/LabBook/Week9&amp;oldid=237983" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>