<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Saarland_4_step"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Saarland/4_step">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Saarland/4_step&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Saarland/4_step&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Saarland/4_step&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Saarland/4_step" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Saarland/4 step</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wikicontent-container"><DIV><DIV id="menu"><UL><LI><A id="team1" href="https://2014.igem.org/Team:Saarland/Team">Team</A></LI><LI><A id="members" href="https://2014.igem.org/Team:Saarland/Team">Members</A></LI><LI><A id="official" href="https://igem.org/Team.cgi">Official Team Profile</A></LI><LI><A id="press" href="https://2014.igem.org/Team:Saarland/Press">Publicity</A></LI><LI><A id="attribution" href="https://2014.igem.org/Team:Saarland/Attribution">Attributions</A></LI><LI><A id="sponsors" href="https://2014.igem.org/Team:Saarland/Sponsors">Sponsors</A></LI><LI><A id="judging" href="https://2014.igem.org/Team:Saarland/Judging">Judging</A></LI></UL><LI><A id="project3" href="https://2014.igem.org/Team:Saarland/Project">Project</A></LI><LI><A id="project2" href="https://2014.igem.org/Team:Saarland/Project">Overview</A></LI><LI><A id="1." href="https://2014.igem.org/Team:Saarland/anticancergens#thenackedmolerat">Naked mole rat</A></LI><LI><A id="1." href="https://2014.igem.org/Team:Saarland/anticancergens#hyaloronicacid">Hyaluronic acid</A></LI><LI><A id="1." href="https://2014.igem.org/Team:Saarland/anticancergens#HMM-HA">HMM-HA</A></LI><LI><A id="2." href="https://2014.igem.org/Team:Saarland/2_step">B.megaterium </A></LI><LI><A id="2." href="https://2014.igem.org/Team:Saarland/2_step#pathway">Pathway engineering </A></LI><LI><A id="3." href="https://2014.igem.org/Team:Saarland/3_step">Develop Strategy </A></LI><LI><A id="results" href="https://2014.igem.org/Team:Saarland/4_step">Results</A></LI><LI><A id="erg" href="https://2014.igem.org/Team:Saarland/4_step">Results </A></LI><LI><A id="5." href="https://2014.igem.org/Team:Saarland/5_step">Discussion </A></LI><LI><A id="5." href="https://2014.igem.org/Team:Saarland/5_step#outlook">Outlook </A></LI><LI><A id="parts" href="https://2014.igem.org/Team:Saarland/Parts">Parts</A></LI><LI><A id="modeling" href="https://2014.igem.org/Team:Saarland/Modeling">Modelling</A></LI><LI><A id="rpg" href="https://2014.igem.org/Team:Saarland/rpg">Human Resources</A></LI><LI><A id="notebook" href="https://2014.igem.org/Team:Saarland/Labbook">Notebook</A></LI><LI><A id="labbook" href="https://2014.igem.org/Team:Saarland/Labbook">Lab journal</A></LI><LI><A id="Methods" href="https://2014.igem.org/Team:Saarland/Methods">Methods</A></LI><LI><A id="Materials" href="https://2014.igem.org/Team:Saarland/Materials">Materials</A></LI><LI><A id="safety" href="https://2014.igem.org/Team:Saarland/Safety">Safety </A></LI></DIV></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H1><SPAN class="mw-headline" id="Step_4:_Build_an_army_of_clone_warriors">Step 4: Build an army of clone warriors</SPAN></H1><P><I> Skip to a chapter by clicking the corresponding marking above </I></P><H3><SPAN class="mw-headline" id="1._Design_and_Synthesis_of_hyaluronan_synthase_gene_has2">1. Design and Synthesis of hyaluronan synthase gene <I>has</I>2</SPAN></H3><P>The initial step of our project was the artificial synthesis of the <I>has</I>2 gene sequence coding for the hyaluronan synthase of the naked mole rat. The sequence was taken from the freely accessible genome database <A href="http://genome.ucsc.edu/" class="external text" rel="nofollow">UCSC Genome Browser</A>, whereby the sequence data was gained by shot-gun-sequencing (Kim <I>et al</I>., 2011). The gene sequence was adapted to the altered codon usage in <I>Bacillus megaterium</I> with <A href="http://www.jcat.de/" class="external text" rel="nofollow">JCat</A> (Grote <I>et al</I>., 2005). Additionally the open reading frame (ORF) of the <I>has</I>2 gene was modified so that prokaryotic ribosomal binding site (RBS), rho-independent transcription termination as well as restriction enzymes that are also part of the multiple cloning site (MCS) of the target plasmid pSMF2.1 were excluded. For the upcoming cloning process of the <I>has</I>2 gene the nucleotide sequence was terminally modified with the restriction sites <I>Mlu</I>Ⅰ (5`end) und <I>Sac</I>Ⅰ(3`end). Moreover a <I>B. megaterium</I> specific RBS was added downstream the <I>Mlu</I>Ⅰrestriction site for efficient protein expression. We paid attention to the fact that the amino acid sequence did not change.
Subsequently the 1693 bp long modified gene sequence <I>has</I>2 (fig. 1) was handed over to <A href="http://www.eurofinsgenomics.eu/" class="external text" rel="nofollow">Eurofins</A> who performed the gene synthesis and the cloning into the standard delivery plasmid pexK4 plasmid.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:472px;"><DIV class="thumbcaption"><B>Figure 1: Modified gene sequence of the naked mole rat`s hyaluronan synthase <I>has</I>2 (1693 bp).</B> The start codon and stop codon of the <I>has</I>2 sequence are marked (blue). At the 5`end the restriction site <I>Mlu</I>Ⅰ(light blue) and the RBS (grey) is annexed. At the 3`end the recognition site of the <I>Sac</I>Ⅰ enzyme is added (modified from <A href="http://genome.ucsc.edu/" class="external text" rel="nofollow">UCSC Genome Browser</A>).</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="2._Genomic_amplification_of_genes_that_may_optimise_the_hyaluronic_acid_production">2. Genomic amplification of genes that may optimise the hyaluronic acid production</SPAN></H3><P>The hyaluronan synthase catalyses the synthesis of hyaluronic acid by consumption of the endogenous synthesised precursor molecules UDP-D-glucuronic acid and UDP-N-acetly-D-glucosamine. The biochemical pathways for their production were already identified in streptococci of group A and C. Hence it was possible to identify genes that could have a beneficial effect on the concentration of these precursor molecules for high yield hyaluronic acid production in <I>B. megaterium</I>. <A href="https://2014.igem.org/Team:Saarland/2_step#pathway"><B> Pathway engineering </B></A>
Eventually eight relevant pathway genes, including <I>UDP-glc</I>DH, <I>glk</I>, <I>pgm</I>B, <I>gta</I>B, <I>pgi, glm</I>S,<I> glm</I>M and <I>gca</I>D, were chosen for the approach of pathway engineering. The sequences of these genes originate from <A href="http://megabac.tu-bs.de/" class="external text" rel="nofollow">MegaBac v9 database</A> and were used for primer design and amplification of genomic DNA of <I>B. megaterium.</I> Restriction sites not only at the 5` end but also at the 3` end were added for subsequent cloning into the pSB1C3 plasmid (table 1). </P><TABLE><CAPTION><B>Table 1:</B> Overview of the BioBrick length and BioBrick restriction sites.
</CAPTION><TBODY><TR><TH>Gene Name
</TH><TH>Gene Size (bp)
</TH><TH>5` Restriction Site
</TH><TH>3` Restriction Site
</TH></TR><TR><TD><I>UDP-glc</I>DH
</TD><TD>1382
</TD><TD><I>Sac</I>Ⅰ
</TD><TD><I>Sph</I>Ⅰ
</TD></TR><TR><TD><I>glk</I></TD><TD>969
</TD><TD><I>EcoR</I>Ⅰ
</TD><TD><I>Pst</I>Ⅰ
</TD></TR><TR><TD><I>pgm</I>B
</TD><TD>707
</TD><TD><I>EcoR</I>Ⅰ
</TD><TD><I>Spe</I>Ⅰ
</TD></TR><TR><TD><I>gta</I>B
</TD><TD>888
</TD><TD><I>EcoR</I>Ⅰ
</TD><TD><I>Pst</I>Ⅰ
</TD></TR><TR><TD><I>pgi</I></TD><TD>1350
</TD><TD><I>Xba</I>Ⅰ
</TD><TD><I>Pst</I>Ⅰ
</TD></TR><TR><TD><I>glm</I>S
</TD><TD>1800
</TD><TD><I>Xba</I>Ⅰ
</TD><TD><I>Pst</I>Ⅰ
</TD></TR><TR><TD><I>glm</I>M
</TD><TD>1346
</TD><TD><I>Eco</I>RⅠ
</TD><TD><I>Pst</I>Ⅰ
</TD></TR><TR><TD><I>gca</I>D
</TD><TD>1380
</TD><TD><I>Eco</I>RⅠ
</TD><TD><I>Pst</I>Ⅰ
</TD></TR></TBODY></TABLE><P>To verify the correct amplification of the genes the PCR products were loaded onto an agarose gel. As it is evident in figure 2 the bands showed the expected lengths for the genes <I>glk</I> (969 bp), <I>pgm</I>B (707 bp), <I>gta</I>B (888 bp), <I>pgi</I> (1350 bp), <I>glm</I>S (1800 bp), <I>glm</I>M (1340 bp) and <I>gca</I>D (1380 bp). Since <I>UDP-glc</I>DH was planned for cloning into pSMF2.1-<I>has</I>2 plasmid it was amplified in a separate step, described below.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 2: PCR amplification of genes coding for the enzymes that are involved in synthesis of precursor molecules for hyaluronic acid production</B>. All amplified gene sequences show the expected lengths. Lane 1: <I>glk</I> (969 bp) Lane 2: <I>pgm</I>B (707 bp) Lane 3: <I>gta</I>B (888 bp) Lane 4: <I>pgi</I> (1350 bp) Lane 5: <I>glm</I>S (1800 bp) Lane 6:<I> glm</I>M (1340 bp) Lane 7: <I>gca</I>D (1380 bp)</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="3._Test_of_restriction_enzymes">3. Test of restriction enzymes</SPAN></H3><P>Before starting with the cloning procedures it was necessary to check if the ordered restriction enzymes work properly. Therefore the pSMF2.1 plasmid was digested separately with each restriction enzyme (<I>Mlu</I>Ⅰ, <I>Sac</I>Ⅰ, <I>Sph</I>Ⅰ) that should be used for the following cloning of <I>has</I>2 and <I>UDP-glc</I>DH into the expression plasmid pSMF2.1. Displayed in Figure 3 all restriction enzymes (<I>Mlu</I>Ⅰ, <I>Sac</I>Ⅰ and <I>Sph</I>Ⅰ) linearised the plasmid, indicating proper functioning. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 3: Test of restriction enzymes <I>Mlu</I>Ⅰ, <I>Sac</I>Ⅰ and <I>Sph</I>Ⅰ.</B> All restriction enzymes work properly as the plasmid DNA is  linearised in all cases. The negative control without the adding of any restriction enzyme shows the circular plasmid.  Lane 1: <I>Mlu</I>Ⅰ restricted pSMF2.1 plasmid (6510 bp)  Lane 2: <I>Sac</I>Ⅰ restricted pSMF2.1 plasmid (6510 bp)  Lane 3: <I>Sph</I>Ⅰ restricted pSMF2.1 plasmid (6510 bp)  Lane 4: undigested pSMF2.1 plasmid (negative control) </DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="4._Cloning_of_the_hyaluronan_synthase_gene_.28has2.29_into_pSMF2.1">4. Cloning of the hyaluronan synthase gene (<I>has</I>2)  into pSMF2.1</SPAN></H3><P>A digestion with the restriction enzymes <I>Sac</I>Ⅰ und <I>Mlu</I>Ⅰwas used to extract the <I>has</I>2 gene (1693 bp) from the pexK4 plasmid (2507 bp) for the following cloning into the final pSMF2.1 plasmid (6510 bp). Agarose electrophoresis was used to detect the digestion products which showed the expected lengths (fig.4). The band of <I>has</I>2 gene was clearly visible at 1693 bp. For cloning of <I>has</I>2 into pSMF2.1 plasmid it first needed to be digested with the same restriction enzymes <I>Sac</I>Ⅰ und <I>Mlu</I>Ⅰ. Subsequently the restricted plasmid (6510 bp) was separated by agarose gel electrophoresis (fig. 4). A restriction control without the restriction enzymes was carried out as well.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 4: Restriction of pexK4-<I>has</I>2 and pSMF2.1 with <I>Mlu</I>Ⅰ/<I>Sac</I>Ⅰ.</B> The hyaluronan synthase (<I>has</I>2) artificially synthesised by <A href="http://www.eurofinsgenomics.eu/" class="external text" rel="nofollow">Eurofins</A> and cloned into the pexK4 plasmid was digested with <I>Mlu</I>Ⅰ and <I>Sac</I>Ⅰ. The digested <I>has</I>2 gene is clearly visible at 1693 bp below the digested pexK4 plasmid at 2507 bp. For cloning of the <I>has</I>2 gene into the pSMF2.1 plasmid, this plasmid was also digested with <I>Mlu</I>Ⅰ and <I>Sac</I>Ⅰ and is visible at 6510 bp. Lane 1: Restricted PexK4 plasmid (2507 bp) and<I> has</I>2 insert (1693 bp) Lane 2: Restricted pSMF2.1 (6510 bp)</DIV></DIV></DIV></DIV><P>
After reisolation the pSMF2.1 plasmid was treated with alkaline phosphatase. The treatment causes an elimination of the phosphate group at the 5` terminus of the restricted plasmid and thereby prevents the restricted plasmid from relegation.
Afterwards the digested insert <I>has</I>2 was ligated with the restricted and phosphatase treated pSMF2.1 plasmid. Figure 5 displays the control of ligation. The approach with ligase showed a high molecular band at 8203 bp indicating that insert and plasmid were successfully ligated. As expected the negative control showed only the two original bands representing digested <I>has</I>2 insert DNA and restricted pSMF2.1 plasmid.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 5: Ligation control of <I>has</I>2 and pSMF2.1.</B> The approach with ligase shows the three typical bands of restricted plasmid (6510 bp), <I>has</I>2 insert (1693 bp) and ligation product pSMF2.1-<I>has</I>2 (8210 bp). Without ligase there is no ligation product. Additionally the religation of the <I>Mlu</I>Ⅰ and <I>Sac</I>Ⅰ digested pSMF2.1 plasmid has obviously been very inefficient as the restricted plasmid is still detectable at 6510 bp. Lane 1: pSMF2.1 + <I>has</I>2 (with ligase) Lane 2: pSMF2.1 + <I>has</I>2 (without ligase) Lane 3: pSMF2.1 (without ligase)</DIV></DIV></DIV></DIV><P>
After transformation of competent <I>E. coli</I> cells with the resulting pSMF2.1-<I>has</I>2 construct and selection on ampicillin agar plates, plasmid DNA of transformants was isolated and presence of <I>has</I>2 insert DNA was checked by control digestion. Figure 6 shows the result for one representative <I>E. coli</I> transformant. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 6: Control digestion with <I>Sac</I>Ⅰ and <I>Mlu</I>Ⅰof plasmid DNA isolated from <I>E. coli </I>cells that were transformed with pSMF2.1-<I>has</I>2 construct. </B>Both pSMF2.1 plasmid DNA at 6510 bp and <I>has</I>2 insert DNA at 1693 bp are evident and indicate a successful ligation and transformation with the pSMF2.1-<I>has</I>2 construct.  Lane 1: pSMF2.1 + <I>has</I>2</DIV></DIV></DIV></DIV><P>
Obviously the pSMF2.1-<I>has</I>2 plasmid was successfully absorbed by the <I>E. coli</I> transformant as both pSMF2.1 plasmid DNA at 6510 bp and <I>has</I>2 insert DNA at 1693 bp are visible after digestion with <I>Sac</I>Ⅰ und <I>Mlu</I>Ⅰ. The isolated plasmid DNA was used in a following step for transformation of <I>B. megaterium.</I></P><H3><SPAN class="mw-headline" id="5._Cloning_of_the_UDP-glcDH_gene_into_pSMF2.1-has2_plasmid">5. Cloning of the <I>UDP-glc</I>DH gene into pSMF2.1-<I>has</I>2 plasmid</SPAN></H3><P>In the same way the UDP-glucose-6-dehdyrogenase (<I>UDP-glc</I>DH) gene was cloned into the pSMF2.1-<I>has</I>2 plasmid. However, instead of an outsourced gene synthesis like in the case of<I> has</I>2, the <I>UDP-glc</I>DH gene was amplified from the genomic DNA of <I>B. megaterium</I>. In addition modified primers were used to add the <I>Sac</I>Ⅰ restriction site at the 5` terminus and the <I>Sph</I>Ⅰ at the 3´ terminus of the PCR product. The presence and size of the PCR product was checked by agarose gel electrophoresis. pSMF2.1-<I>has</I>2 plasmid was digested with <I>Sac</I>Ⅰ and <I>Sph</I>Ⅰ for cloning <I>UDP-glc</I>DH and also checked by agarose gel electrophoresis (fig.7). A specific band could be detected at a size of 1382 bp indicating a proper PCR amplification of <I>UDP-glc</I>DH gene (fig.7). The restriction of pSMF2.1-<I>has</I>2 plasmid with <I>Sac</I>Ⅰ and<I> Sph</I>Ⅰ resulted in a linearised plasmid. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 7:  Amplification of <I>UDP-glc</I>DH from genomic DNA of <I>B. megaterium</I> and restriction of pSMF2.1-<I>has</I>2 plasmid with <I>Sac</I>Ⅰ and <I>Sph</I>Ⅰ.</B> A specific band is visible at 1382 bp. The size of this PCR product corresponds to the expected length of <I>UDP-glc</I>DH gene. The restriction of pSMF2.1-<I>has</I>2 plasmid with <I>Sac</I>Ⅰ and <I>Mlu</I>Ⅰ shows the restricted plasmid at 8203 bp. Lane 1: PCR product <I>UDP-glc</I>DH (<I>SacⅠ/Sph</I>Ⅰ) (1382 bp)  Lane 2: pSMF2.1-<I>has</I>2 (<I>Sac</I>Ⅰ/<I>Sph</I>Ⅰ)</DIV></DIV></DIV></DIV><P>
The next step was the ligation of the restricted pSMF2.1-<I>has</I>2 plasmid with the amplified and digested <I>UDP-glc</I>DH gene. As demonstrated in figure 8 the ligation was efficient because a shift in pSMF2.1-<I>has</I>2 plasmid size and a weaker insert band could be detected. However, the ratio of <I>UDP-glc</I>DH insert and pSMF2.1-<I>has</I>2 plasmid appeared not to be optimal due to a visible band at about 2750 bp resulting from a ligation of the insert <I>UDP-glc</I>DH (1382 bp) with itself.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 8: Ligation control of pSMF2.1-<I>has</I>2 and <I>UDP-glc</I>DH.</B> After addition of ligase the intensity of <I>UDP-glc</I>DH insert band at 1382 bp is weaker compared to the approach without ligase. Moreover there is a slight plasmid shift detectable up to nearly 9585 bp. Besides a third band is visible at 2600 bp indicating a partially self ligated <I>UDP-glc</I>DH insert. The negative control without ligase shows the two original bands representing the digested pSMF2.1-<I>has</I>2 plasmid at 8203 bp and <I>UDP-glc</I>DH at 1382 bp. Lane 2: pSMF2.1-<I>has</I>2 + <I>UDP-glc</I>DH (with ligase)  Lane 3: pSMF2.1-<I>has</I>2 + <I>UDP-glc</I>DH (without ligase)</DIV></DIV></DIV></DIV><P>
After transformation of the ligation product pSMF2.1-<I>has</I>2-<I>UDP-glc</I>DH into competent <I>E. coli</I> cells and selection on ampicillin agar plates, the plasmid DNA of resulting transformants was checked by control digestion. Figure 9 shows the result for one representative <I>E. coli</I> transformant. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 9: Control digestion with <I>Mlu</I>Ⅰ,<I>Sac</I>Ⅰ and<I>Sph</I>Ⅰ of plasmid DNA isolated from <I>E. coli</I> cells that were transformed with pSMF2.1-<I>has</I>2-<I>UDP-glc</I>DH plasmid.</B> Apparently the transformant did absorb the pSMF2.1-<I>has</I>2-<I>UDP-glc</I>DH plasmid and shows two additional bands of <I>UDP-glc</I>DH insert DNA at 1382 bp and <I>has</I>2 insert DNA at 1693 bp below the restricted pSMF2.1 plasmid at 6510 bp. Both negative controls with only one or no restriction enzyme did not show the specific insert bands. Lane 1: pSMF2.1-<I>has</I>2-<I>UDP-glc</I>DH (<I>Mlu</I>Ⅰ)  Lane 2: pSMF2.1-<I>has</I>2-<I>UDP-glc</I>DH (<I>Mlu</I>Ⅰ/<I>Sac</I>Ⅰ/<I>Sph</I>Ⅰ) Lane 3: pSMF2.1-<I>has</I>2-<I>UDP-glc</I>DH (undigested)</DIV></DIV></DIV></DIV><P>
Obviously the pSMF2.1-<I>has</I>2-<I>UDP-glc</I>DH plasmid was successfully absorbed by the <I>E. coli</I> transformant as all three digestion products representing the pSMF2.1 plasmid at 6510 bp, the <I>has</I>2 insert DNA at 1693 bp and the <I>UDP-glc</I>DH insert DNA at 1382 bp are visible after digestion with <I>Mlu</I>Ⅰ,<I>Sac</I>Ⅰand <I>Sph</I>Ⅰ. The isolated plasmid DNA was used in a following step for transformation of<I> B. megaterium.</I></P><H3><SPAN class="mw-headline" id="6._Transformations_of_Bacillus_megaterium">6. Transformations of <I>Bacillus megaterium</I></SPAN></H3><P>The pSMF2.1-<I>has</I>2 and the pSMF2.1-<I>has2-UDP-glc</I>DH plasmids were transformed into <I>Bacillus megaterium</I>. Transformants were lysed and the plasmid DNA was isolated. Presence of the two expression plasmids was verified by control digestion showing the same result as in figure 9. Finally the <I>B. megaterium</I> chassis is ready for the first attempt to produce the high molecular mass hyaluronic acid. </P><H3><SPAN class="mw-headline" id="7._Control_of_Protein_Expression">7. Control of Protein Expression</SPAN></H3><H4><SPAN class="mw-headline" id="7.1_Control_of_HMM-HA_synthesis_by_means_of_viscosity_measurement">7.1 Control of HMM-HA synthesis by means of viscosity measurement</SPAN></H4><P>The outstanding water binding capacity of high molecular mass hyaluronic acid (HMM-HA) should increase the viscosity of the cultivation medium. An altered viscosity can be measured with a rheometer. Samples of <I>B. megaterium</I> cultures transformed with pSMF2.1-<I>has</I>2 and pSMF2.1-<I>has</I>2-<I>UDP-glc</I>DH were prepared for viscosity measurement 24 h and 48 h after induction of protein expression. Supernatant of the cultures was used for measurement in case of secretion of HMM-HA into the medium. Cells were disrupted by sonication and lysozyme treatment in order to release the HMM-HA in case of its accumulation inside the cell. The results of the viscosity measurements demonstrated no significant differences in the viscosity between non-induced and induced samples. Even co expression of UDP-GlcDH didn´t lead to a significant increase of the viscosity. Consequently production of HMM-HA failed. Further steps comprising the characterisation of anti carcinogenic effects of HMM-HA on human cancer cell lines could not be performed.</P><H4><SPAN class="mw-headline" id="7.2_Control_of_protein_expression_and_localisation_of_a_GFP_tagged_hyaluronan_synthase">7.2 Control of protein expression and localisation of a GFP tagged hyaluronan synthase </SPAN></H4><P>Protein expression and particularly localisation of Has2 in <I>Bacillus megaterium</I> was checked by using a c-terminal GFP tagged Has2 variant. The necessary fusion gene was created by SOE-PCR
digested with <I>Mlu</I>Ⅰand <I>Sac</I>Ⅰ and inserted into pSMF2.1 plasmid. A subsequent restriction verified whether the SOE-PCR and the transformation into <I>E. coli</I> cells were effective. As displayed in figure 10 the presence of the fusion gene <I>has</I>2-<I>gfp</I> could be proofed by control digestion at 2420 bp. <A href="https://2014.igem.org/Team:Saarland/Methods#132"><B> SOE-PCR </B></A></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 10:  Control digestion with <I>Mlu</I>Ⅰ and <I>Sac</I>Ⅰof of plasmid DNA isolated from <I>E. coli </I>cells that were transformed with pSMF2.1-<I>has</I>2-<I>gfp</I>.</B> The digestion reveals a band at 2413 bp which corresponds to the length of <I>has</I>2-<I>gfp</I> fusion gene combining the 720 bp <I>gfp</I> fragment and the 1693 bp <I>has</I>2 fragment. Another band at 6510 bp shows the restricted pSMF2.1 plasmid. Lane 1: pSMF2.1-<I>has</I>2-<I>gfp</I> (<I>Mlu</I>Ⅰ/<I>Sac</I>Ⅰ)</DIV></DIV></DIV></DIV><P> 
Subsequently the pSMF2.1-<I>has</I>2-<I>gfp</I> plasmid DNA was transformed into <I>B. megaterium</I> and cultures were induced with xylose to start recombinant protein expression. Non-induced cultures served as negative controls. After 48 h of expression, corresponding samples were prepared for analysis by fluorescence microscopy (fig. 11). </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:367px;"><DIV class="thumbcaption"><B>Figure 11:  Fluorescence micrographs of <I>B. megaterium</I> cultures transformed with pSMF2.1-<I>has</I>2-<I>gfp</I> show fluorescence signals after excitation of λ<SUB>exc</SUB>= 488 nm. Exposure time 100 ms (transmitted light) / 10 s (fluorescence) at a magnification of 630.</B> The images in the upper row are transmitted light microscopic records. The images in the lower row are recorded by fluorescence microscopy. All images show liquid cultures. (C) No fluorescence signals are detectable in the non induced culture. (D) Fluorescence signals of the induced cultures are poorly localised in the cytoplasm membrane. Noticeable fluorescence dots of higher intensity are visible in the cytoplasm of the microbes.</DIV></DIV></DIV></DIV><P>
Figure 11 illustrates that fluorescence signals of induced <I>B. megaterium</I> cultures were probably detectable in the cytoplasm membrane after excitation with light at a wavelength of 488 nm. Furthermore the image shows several fluorescence dots of higher intensity that might derive from an accumulation of Has2-GFP protein in the phospholipid monolayer that surrounds insoluble spheric inclusions, also referred as poly hydroxyalkanoate (PHA) inclusion bodies. The not induced negative control does not show any fluorescence signal. 
To proof the accumulation of GFP tagged Has2 protein in the cytoplasm membrane of <I>B. megaterium </I>cells localisation studies were carried out with a laser scanning fluorescence microscope. Repeated approaches for expression of the GFP tagged Has2 protein in <I>B. megaterium</I> cultures including various incubation temperatures varying from 26 °C up to 37 °C combined with different incubation times (24 h, 48 h, 72 h) failed. Detection of corresponding fluorescence signals with the more sensitive laser scanning microscope could not be reproduced.</P><H4><SPAN class="mw-headline" id="7.3_Checking_the_expression_of_the_GFP_tagged_Has2_protein_by_SDS-Page_and_Western_Blot_analysis">7.3 Checking the expression of the GFP tagged Has2 protein by SDS-Page and Western Blot analysis</SPAN></H4><P>Recombinant protein expression of Has2 and UDP-GlcDH in <I>B. megaterium</I> cultures could not be proven by SDS-PAGE and following Coomassie Blue staining. At least the expression of the GFP tagged Has2 protein in <I>B. megaterium</I> cultures could be analysed by Western Blot with an appropriate GFP antibody. Soluble protein fraction of cell lysates didn`t show GFP tagged Has2 protein expression. Insoluble protein fraction of cell lysates including integral membrane proteins showed several unspecific protein bands but also a distinct protein band with a molecular weight of 90 kDa that likely derived from binding of the antibody to the GFP tagged Has2 protein. Cell lysates of non-induced <I>B. megaterium</I> culture served as a negative control and didn´t show GFP tagged Has2 protein expression neither in the soluble protein fraction nor in the insoluble fraction including integral membrane proteins. </P><H3><SPAN class="mw-headline" id="8_.Cloning_of_the_BioBricks_into_the_pSB1C3_plasmid">8 .Cloning of the BioBricks into the pSB1C3 plasmid</SPAN></H3><P>For cloning of the genes that may optimise the hyaluronic acid production not only the corresponding PCR products, but also the BioBrick plasmid pSB1C3 was restricted with the respective restriction enzymes (table 1). Next step was the ligation of the restricted pSB1C3 plasmid (2070 bp) with each amplified and digested gene. Controls of ligation reactions are not shown. After transformation of the ligation products into competent <I>E. coli</I> cells and selection on ampicillin agar plates, the plasmid DNA of resulting transformants was checked by control digestion. Figure 12 shows the result for one representative <I>E. coli</I> transformant of each ligation. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 12:  Digestion control of plasmid DNA isolated from<I> E. coli</I> cells that were transformed with pSB1C3-BioBrick constructs.</B> The digestions with particular restriction enzymes confirm the presence of the different BioBrick insert DNAs with correct product sizes (table 1).  Appilication scheme for (A)   Lane 1: pSBS1C3-<I>glk</I> (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ) Lane 2: pSB1C3-<I>gta</I>B (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ) Lane 3:pSB1C3-<I>glm</I>M (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ) Lane 4: pSB1C3-<I>gca</I>D (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ) Lane 5: pSB1C3-<I>pgm</I>B (<I>Eco</I>RⅠ/<I>Spe</I>Ⅰ) Application scheme for (B)   Lane 1: pSB1C3-<I>pgi</I> (<I>Xba</I>Ⅰ/<I>Pst</I>Ⅰ) Lane 2: pSB1C3-<I>glm</I>S (<I>Xba</I>Ⅰ/<I>Pst</I>Ⅰ)</DIV></DIV></DIV></DIV><P>
However genes coding for <I>pgm</I>B, <I>pgi</I>, <I>glm</I>S and <I>glm</I>M contained restriction sites for <I>Eco</I>RⅠ, <I>Xba</I>Ⅰ or <I>Pst</I>Ⅰ which are not allowed in the BioBrick registry. Thus a Quick Change PCR was necessary to mutate the restriction site without changing the amino acid sequence.
<A href="https://2014.igem.org/Team:Saarland/Methods#134"><B> QC-PCR </B></A> 
Therefore primers were designed to change the wobble position of the codon within the corresponding restriction site. The success of the QC-PCR was checked by digestion with the
previously unwanted restriction enzyme and agarose gel electrophoresis (fig.13+14). </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 13:  Digestion control of plasmid DNA isolated from <I>E. coli</I> cells that were transformed with pSB1C3-<I>pgm</I>B and pSB1C3-<I>pgi</I> after Quick Change PCR. </B>The restriction with <I>Eco</I>RⅠ/<I>Pst</I>Ⅰ and <I>Spe</I>Ⅰ of a clone transformed with pSB1C3-<I>pgm</I>B only shows a single band at 2070 bp assuming to be the restricted pSB1C3 plasmid. The clone transformed with pSB1C3-<I>pgi</I> displays two bands at 2070 bp (pSB1C3) and 1350 bp (<I>pgi</I>), evidently not containing the <I>Eco</I>RⅠ restriction site anymore. Both positive controls show two bands instead of the expected three bands indicating that there were no illegal restriction sites before QC-PCR. Lane 1: pSB1C3-<I>pgm</I>B after QC (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ/<I>Spe</I>Ⅰ) Lane 2: Positive control pSB1C3-<I>pgm</I>B without QC (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ/<I>Spe</I>Ⅰ) Lane 3: pSB1C3-<I>pgi</I> after QC (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ/<I>Xba</I>Ⅰ) Lane 4: Positive control pSB1C3-<I>pgi</I> without QC (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ/<I>Xba</I>Ⅰ)</DIV></DIV></DIV></DIV><P>
Figure 13 shows that the pSB1C3-<I>pgm</I>B clone did not feature the <I>pgm</I>B insert at 707 bp after QC-PCR. Only the restricted pSB1C3 plasmid is present at (2070 bp). The <I>pgm</I>B insert of the positive control should consist of two distinct bands since the illegal <I>Pst</I>Ⅰ restriction site should be still present. The positive control doesn´t seem to be in possession of the illegal restriction site as displayed on <A href="http://megabac.tu-bs.de/" class="external text" rel="nofollow">MegaBac v9 database</A>. QC-PCR didn`t seem to have been necessary. The presence of <I>pgi</I> without illegal <I>Eco</I>RⅠ restriction site was confirmed by sequencing of the BioBrick.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 14:  Digestion control of plasmid DNA isolated from <I>E. coli</I> cells that were transformed with pSB1C3-<I>glm</I>M and pSB1C3-<I>glm</I>S after Quick Change PCR. </B>The clone transformed with pSB1C3-<I>glm</I>M shows two bands at 2070 bp (restricted pSB1C3) and 1346 bp (<I>glm</I>M) thus indicating that the illegal restriction (<I>Xba</I>Ⅰ) site is not present. However, the positive control also shows these two fragments instead of an additional fragment deriving from the illegal restriction site that should be still present. The pSB1C3-<I>glm</I>S transformed clone demonstrates the expected result and shows two correctly sized bands for pSB1C3 plasmid at 2070 bp and <I>glm</I>S insert DNA at 1800 bp. Additionally the positive control shows the three expected bands because<I> glm</I>S insert DNA is cleaved to a 1000 bp fragment and a 800 bp due to the still present illegal <I>Eco</I>RⅠrestriction site. Lane 1: pSB1C3-<I>glmM</I> after QC (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ/<I>Xba</I>Ⅰ)  Lane 2: Positive control pSB1C3-<I>glmM</I> without QC (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ/<I>Xba</I>Ⅰ)  Lane 3: pSB1C3-<I>glm</I>S after QC (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ/<I>Xba</I>Ⅰ) Lane 4: Positive control pSB1C3-<I>glm</I>S without QC (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ/<I>Xba</I>Ⅰ)</DIV></DIV></DIV></DIV><P>
Figure 14 shows that <I>glm</I>M insert DNA and <I>glm</I>S insert DNA are no longer in possession of their illegal restriction sites after QC-PCR.
As <I>has</I>2 and <I>UDP-glc</I>DH were already cloned into the PSMF2.1 plasmid and both genes contained illegal <I>Eco</I>RⅠ restriction sites the QC PCRs to substitute these illegal restriction sites were performed in the pSMF2.1 plasmid. Afterwards the changed <I>has</I>2 and <I>UDP-glc</I>DH gene sequences were amplified by PCR and legal restriction sites for cloning into pSB1C3 were added to the termini of both genes. Next step was the ligation of the restricted pSB1C3 plasmid (2070 bp) with amplified and digested genes. Controls of ligation reactions are not shown. After transformation of the ligation products into competent <I>E. coli</I> cells and selection on ampicillin agar plates, the plasmid DNA of resulting transformants was checked by control digestion. Figure 15 shows the result for one representative <I>E. coli</I> transformant of each ligation.  </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 15: Control digestion of plasmid DNA isolated from<I> E. coli</I> transformed with pSB1C3-<I>has</I>2 and pSB1C3-<I>UDP-glc</I>DH after Quick Change PCR. </B>Digestion was carried out with <I>Eco</I>RⅠ and <I>Pst</I>Ⅰ. Only two fragments are visible for each digestion representing the <I>has</I>2 insert DNA at 1693 bp / <I>UDP-glc</I>DH at 1382 bp and the restricted pSB1C3 plasmid at 2070 bp. No illegal restriction sites within the <I>has</I>2 respectively the <I>UDP-glc</I>DH gene anymore. Lane 1: pSB1C3-<I>has</I>2 after QC (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ) Lane 2: pSB1C3-<I>UDP-glc</I>DH after QC (<I>Eco</I>RⅠ/<I>Pst</I>Ⅰ)</DIV></DIV></DIV></DIV><P>
The elimination of illegal restriction sites could be demonstrated for both genes.
 
To proof the success of restriction site mutagenesis and the correct base sequences of our constructs all plasmids were additionally sequenced by <A href="http://www.eurofinsgenomics.eu/" class="external text" rel="nofollow">Eurofins</A>. Sequencing revealed that <I>pgm</I>B and <I>glm</I>M still contained their illegal restriction site<I> Pst</I>Ⅰ respectively <I>Xba</I>Ⅰ. However it was not possible to repeat the Quick Change PCRs as the constructs were already shipped. For all other shipped BioBricks correct sequences could be confirmed.</P><P><A href="https://2014.igem.org/Team:Saarland/3_step"><B> Previous Step </B></A></P><DIV style="margin-left:765px;margin-top:-73px;"><A href="https://2014.igem.org/Team:Saarland/5_step"><B> Next Step</B></A></DIV><H3><SPAN class="mw-headline" id="References">References</SPAN></H3><P><B>Grote</B>, A., Hiller, K., Scheer, M., Munch, R., Nortemann, B., Hempel, D.C., and Jahn, D. (2005). JCat: a novel tool to adapt codon usage of a target gene to its potential expression host. Nucleic Acids Res. 33, W526–W531.<B>Kim</B>, E.B., Fang, X., Fushan, A.A., Huang, Z., Lobanov, A.V., Han, L., Marino, S.M., Sun, X., Turanov, A.A., Yang, P., et al. (2011). Genome sequencing reveals insights into physiology and longevity of the naked mole rat. Nature 479, 223–227.
</P></DIV></DIV><DIV id="social-footer"><DIV><H4><A href="https://2014.igem.org/Team:Saarland/Impressum">Impressum/Copyright</A></H4></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Saarland/4_step">http://2014.igem.org/Team:Saarland/4_step</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Saarland/4_step" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Saarland/4_step" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Saarland/4_step&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Saarland/4_step&amp;oldid=321768" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>