"
Page
Discussion
View source
History
teams
Log in
 
Team:Saarland/Modeling
From 2014.igem.org
Team
Members
Official Team Profile
Publicity
Attributions
Sponsors
Judging
Project
Overview
Naked mole rat
Hyaluronic acid
HMM-HA
B.megaterium
Pathway engineering
Develop Strategy
Results
Results
Discussion
Outlook
Parts
Modelling
Human Resources
Notebook
Lab journal
Methods
Materials
Safety
Modelling:
The hyaluronan synthase of the naked mole rat is in possession of 7 putative transmembrane domains and one large cytoplasmic loop which is supposed to be the catalytic center for HA synthesis. As there are no crystal structures of hyaluronan synthases, the corresponding protein modelling was performed using the template based RaptorX web server (Källberg et al., 2012; Ma et al., 2013; Peng and Xu, 2011; Peng and Xu, 2011b). It is shown in figure 1.
Figure 1: Protein structure of the naked mole rat's hyaluronan synthase 2 . Template: Cellulose synthase A subunit of Rhodobacter sphaeroides. (A) Lateral view. The upper 7 domains are membrane bound, the lower part that includes a β-sheet secondary structure is exposed to the cytoplasm and supposed to be the catalytic center for the HA synthesis. (B) View from above. The seven transmembrane domains are supposed to form a channel for HA secretion.
In contrast to the highly conserved catalytic domain of hyaluronan synthases in other vertebrates the naked mole rat´s HaS features two unique changes in the amino acid sequence as two asparagines have been substituted by serines. This might be the cause for the raised activity of the naked mole rat´s Has. There are two different models described for HA transport across the membrane. It´s either possible that there is a specific transporter for oligosaccharides mediating HA secretion into the medium, or that the multi transmembrane domains of the hyaluronan synthase themselves form a channel for the secretion of the nascent HA chain. Based on the protein model we think the more likely model for HA secretion across the membrane is the channel.
References:
Källberg, M., Wang, H., Wang, S., Peng, J., Wang, Z., Lu, H., and Xu, J. (2012). Template-based protein structure modeling using the RaptorX web server. Nat. Protoc. 7, 1511–1522.
Ma, J., Wang, S., Zhao, F., and Xu, J. (2013). Protein threading using context-specific alignment potential. Bioinformatics 29, i257–i265.
Peng, J., and Xu, J. (2011a). A multiple-template approach to protein threading. Proteins 79, 1930–1939.
Peng, J., and Xu, J. (2011b). Raptorx: Exploiting structure information for protein alignment by statistical inference. Proteins Struct. Funct. Bioinforma. 79, 161–171.
Impressum/Copyright
Retrieved from " http://2014.igem.org/Team:Saarland/Modeling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
