<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Saarland_Methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Saarland/Methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Saarland/Methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Saarland/Methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Saarland/Methods&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Saarland/Methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Saarland/Methods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="wikicontent-container"><DIV><DIV id="menu"><UL><LI><A id="team1" href="https://2014.igem.org/Team:Saarland/Team">Team</A></LI><LI><A id="members" href="https://2014.igem.org/Team:Saarland/Team">Members</A></LI><LI><A id="official" href="https://igem.org/Team.cgi">Official Team Profile</A></LI><LI><A id="press" href="https://2014.igem.org/Team:Saarland/Press">Publicity</A></LI><LI><A id="attribution" href="https://2014.igem.org/Team:Saarland/Attribution">Attributions</A></LI><LI><A id="sponsors" href="https://2014.igem.org/Team:Saarland/Sponsors">Sponsors</A></LI><LI><A id="judging" href="https://2014.igem.org/Team:Saarland/Judging">Judging</A></LI></UL><LI><A id="project3" href="https://2014.igem.org/Team:Saarland/Project">Project</A></LI><LI><A id="project2" href="https://2014.igem.org/Team:Saarland/Project">Overview</A></LI><LI><A id="1." href="https://2014.igem.org/Team:Saarland/anticancergens#thenackedmolerat">Naked mole rat</A></LI><LI><A id="1." href="https://2014.igem.org/Team:Saarland/anticancergens#hyaloronicacid">Hyaluronic acid</A></LI><LI><A id="1." href="https://2014.igem.org/Team:Saarland/anticancergens#HMM-HA">HMM-HA</A></LI><LI><A id="2." href="https://2014.igem.org/Team:Saarland/2_step">B.megaterium </A></LI><LI><A id="2." href="https://2014.igem.org/Team:Saarland/2_step#pathway">Pathway engineering </A></LI><LI><A id="3." href="https://2014.igem.org/Team:Saarland/3_step">Develop Strategy </A></LI><LI><A id="results" href="https://2014.igem.org/Team:Saarland/4_step">Results</A></LI><LI><A id="erg" href="https://2014.igem.org/Team:Saarland/4_step">Results </A></LI><LI><A id="5." href="https://2014.igem.org/Team:Saarland/5_step">Discussion </A></LI><LI><A id="5." href="https://2014.igem.org/Team:Saarland/5_step#outlook">Outlook </A></LI><LI><A id="parts" href="https://2014.igem.org/Team:Saarland/Parts">Parts</A></LI><LI><A id="modeling" href="https://2014.igem.org/Team:Saarland/Modeling">Modelling</A></LI><LI><A id="rpg" href="https://2014.igem.org/Team:Saarland/rpg">Human Resources</A></LI><LI><A id="notebook" href="https://2014.igem.org/Team:Saarland/Labbook">Notebook</A></LI><LI><A id="labbook" href="https://2014.igem.org/Team:Saarland/Labbook">Lab journal</A></LI><LI><A id="Methods" href="https://2014.igem.org/Team:Saarland/Methods">Methods</A></LI><LI><A id="Materials" href="https://2014.igem.org/Team:Saarland/Materials">Materials</A></LI><LI><A id="safety" href="https://2014.igem.org/Team:Saarland/Safety">Safety </A></LI></DIV></DIV></DIV><DIV id="wikicontent-container"><DIV id="wikicontent"><H1><SPAN class="mw-headline" id="Methods">Methods</SPAN></H1><P>
The following section lists the used methods in the presented iGEM Project:
<I> Skip to a chapter by clicking the corresponding marking above   </I></P><H3><SPAN class="mw-headline" id="1._Cloning">1. Cloning</SPAN></H3><H4><SPAN class="mw-headline" id="1.1_Restriction_and_phosphatase_treatment_of_plasmid_DNA">1.1 Restriction and phosphatase treatment of plasmid DNA</SPAN></H4><P>Originally found in bacteria, endonucleases belong to a class of enzymes capable of cleaving the phosphodiester bond of DNA molecules by hydrolysis. Depending on the type they recognize site specific base sequences of 4, 6 or 8 bases respectively, which are often palindromic. Treatment of different DNA molecules with the same endonucleases allows the combination of the resulting DNA fragments in a following ligation reaction. </P><P>The treatment of DNA fragments with phosphatase is beneficial if the restriction is performed with only one restriction enzyme. It can also be beneficial if the restriction with two different restriction enzymes is unbalanced and single cut plasmid DNA is left. This applies in particular to the treatment of plasmid DNA with phosphatase. As the nomenclature indicates the phosphatase cleaves the phosphate group at the 5´terminus of DNA fragments and thereby minimizes the background of mono ligated plasmid DNA. Consequently the amount of transformants with insert DNA in the plasmid construct is considerably increased.</P><P>The following components were used for a basic restriction mixture:</P><TABLE><TBODY><TR><TD>10 x FastDigest Buffer
</TD><TD>3 µl
</TD></TR><TR><TD> Restriction Enzyme 1
</TD><TD>10 U
</TD></TR><TR><TD>Restriction Enzyme 2
</TD><TD>10 U
</TD></TR><TR><TD>FastAP
</TD><TD>1 U
</TD></TR><TR><TD> Plasmid
</TD><TD>approx. 1 µg
</TD></TR><TR><TD> ddH2O
</TD><TD>to 30 µl
</TD></TR></TBODY></TABLE><P>The mixture was incubated for 10 min at 37 °C. The success of the restriction was subsequently checked in an agarose gel electrophoresis.
</P><H4><SPAN class="mw-headline" id="1.2_Restriction_of_PCR_products:">1.2 Restriction of PCR products:</SPAN></H4><P>Corresponding restriction sites were added in a PCR reaction at the 5´and 3´termini of insert DNA via modified primers. After gelextraction the following components were used for a basic restriction mixure.</P><TABLE><TBODY><TR><TD>10 x FastDigest Buffer
</TD><TD> 3 µl
</TD></TR><TR><TD> Restriction Enzyme 1
</TD><TD>10 U
</TD></TR><TR><TD>Restriction Enzyme 2
</TD><TD>10 U
</TD></TR><TR><TD>PCR Product
</TD><TD>25 µl
</TD></TR><TR><TD> ddH2O
</TD><TD>to 30 µl
</TD></TR></TBODY></TABLE><P>The mixture was incubated for 10 min at 37 °C followed by purification via QIAquick Gel Extraction Kit (Qiagen). 
</P><H4><SPAN class="mw-headline" id="1.3_Polymerase_chain_reaction_.28PCR.29">1.3 Polymerase chain reaction (PCR)</SPAN></H4><P>The PCR is a standard procedure in molecular biology that enables the amplification of distinct DNA sequences in vitro. Short flanking DNA oligomers define the DNA sequence which is to be amplified. The amplification process itself is performed by a polymerizing enzyme also referred to as DNA polymerase. Since its invention in 1983 numerous variations have been established for different applications.  </P><H4><SPAN class="mw-headline" id="1.3.1_Amplification_of_genomic_DNA:">1.3.1 Amplification of genomic DNA:</SPAN></H4><P>The following components were used for the basic amplification of genomic DNA of <I>Bacillus megaterium</I>:
</P><TABLE><TBODY><TR><TD>2x Q5 Master Mix
</TD><TD>10 µl
</TD></TR><TR><TD>10 µM Forward
</TD><TD>1 µl
</TD></TR><TR><TD>10 µM Reverse Primer
</TD><TD>1 µl
</TD></TR><TR><TD> Genomic DNA
</TD><TD>100 ng
</TD></TR><TR><TD> ddH2O
</TD><TD>to 20 µl
</TD></TR></TBODY></TABLE><P>The following speed cycle program was used:</P><TABLE><TBODY><TR><TH>Step
</TH><TH>Temperature [°C]
</TH><TH>Time [s]
</TH></TR><TR><TD>1. Initial Denaturation
</TD><TD>98
</TD><TD> 30
</TD></TR><TR><TD> 2. Denaturation
</TD><TD>98
</TD><TD>15
</TD></TR><TR><TD>3. Annealing
</TD><TD>60
</TD><TD>15
</TD></TR><TR><TD>4. Extention
</TD><TD>75
</TD><TD>20/kb
</TD></TR><TR><TD>5. Final  Extention
</TD><TD>72
</TD><TD>120
</TD></TR><TR><TD>6. storage
</TD><TD>4
</TD><TD>for ever
</TD></TR></TBODY></TABLE><DIV style="margin-top:-140px; margin-left:625px">
30 cycles
</DIV>
Presence, absence and size of amplification products were determined by agarose gel electrophoresis.
<H4><SPAN class="mw-headline" id="1.3.2_Splicing_by_overlapping_extension_.28SOE.29-PCR:">1.3.2 Splicing by overlapping extension (SOE)-PCR:</SPAN></H4><P>The SOE-PCR is a common variation of the standard PCR reaction and primarily used for the generation of fusion genes. The name refers to the special set of primers. Their single stranded endings are complementarily overlapping with the DNA sequences of the DNA fragments to be fused. Figure 1 shows the flowchart of the SOE-PCR for the generation of a hyaluronan synthase (<I>has</I>2) - green fluorescent protein (<I>gfp</I>) fusion gene.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:548px;"><DIV class="thumbcaption"><B>Figure 1: Scheme of a SOE-PCR for the generation of a hyaluronan synthase (<I>has</I>2) - green fluorescent protein (<I>gfp</I>) fusion gene.</B> First step of the SOE-PCR comprises the amplification of the individual gene sequences of <I>has</I>2 and <I>gfp</I> with primer combinations a/b and c/d. Complementary sequences of their counterparts derived from primer b and c allow the mutual annealing of the amplification products. A second amplification with the flanking primers a and d results in the generation of the respective <I>has</I>2-<I>gfp</I> fusion gene.</DIV></DIV></DIV></DIV><P>
The following components were used for the first standard PCR reactions:
<TABLE><TBODY><TR><TD> 2x Q5 Master Mix</TD><TD>10 µl </TD></TR><TR><TD>10 µM Forward Primer a or c </TD><TD>1 µl </TD></TR><TR><TD>10 µM Reverse Primer b or d </TD><TD>1 µl </TD></TR><TR><TD> Plasmid DNA</TD><TD>10 ng </TD></TR><TR><TD> ddH2O</TD><TD> to 20 µl</TD></TR></TBODY></TABLE></P><P>The following components were used for the second standard PCR reaction:
<TABLE><TBODY><TR><TD> 2x Q5 Master Mix</TD><TD>10 µl </TD></TR><TR><TD>10 µM Forward Primer a </TD><TD>1 µl </TD></TR><TR><TD>10 µM Reverse Primer d </TD><TD>1 µl </TD></TR><TR><TD>Amplification Product 1 and 2</TD><TD>1:1 Molar Ratio (approx. 10 ng) </TD></TR><TR><TD> ddH2O</TD><TD> to 20 µl</TD></TR></TBODY></TABLE></P><P>
The following speed cycle program was used: 
</P><P><TABLE><TBODY><TR><TH>Step </TH><TH>Temperature [°C] </TH><TH>Time [s] </TH></TR><TR><TD>1. Initial Denaturation	 </TD><TD>98 </TD><TD> 30</TD></TR><TR><TD> 2. Denaturation</TD><TD> 98</TD><TD>15 </TD></TR><TR><TD> 3. Annealing	</TD><TD> 60</TD><TD>15 </TD></TR><TR><TD> 4. Extention</TD><TD> 75</TD><TD> 20/kb</TD></TR><TR><TD> 5. Final  Extention</TD><TD>72 </TD><TD>120 </TD></TR><TR><TD> 6. storage</TD><TD> 4</TD><TD> for ever </TD></TR></TBODY></TABLE></P><DIV style="margin-top:-140px; margin-left:625px">
30 cycles
</DIV>
Presence, absence and size of amplification products were determined by agarose gel electrophoresis.
<H4><SPAN class="mw-headline" id="1.3.3_Colony_PCR:">1.3.3 Colony PCR:</SPAN></H4><P>After transformation of competent <I>E. coli</I> cells the colony PCR is a popular high-throughput technique in molecular biology for the quick identification of the presence, absence or orientation of insert DNA in plasmid constructs, respectively. An extended heating step at the beginning of the PCR reaction results in cell lysis and release of plasmid DNA which serves as template for the following amplification with insert specific primers. Time consuming DNA isolations and restrictions of multiple colonies are superfluous.
The following components were used for the second standard PCR reaction:</P><P><TABLE><TBODY><TR><TD>10x ThermoPol Reaction Buffer </TD><TD> 2 µl</TD></TR><TR><TD> 10 mM dNTP Mix</TD><TD>0,4 µl </TD></TR><TR><TD>10 µM Forward Primer  </TD><TD> 1 µl</TD></TR><TR><TD>10 µM Reverse Primer  </TD><TD>1 µl </TD></TR><TR><TD>Taq DNA Polymerase </TD><TD>½  U </TD></TR><TR><TD>ddH2O </TD><TD> to 20 µl</TD></TR><TR><TD>colony</TD><TD> Resuspend</TD></TR></TBODY></TABLE></P><P>
The following speed cycle program was used: 
</P><P><TABLE><TBODY><TR><TH>Step </TH><TH>Temperature [°C] </TH><TH>Time [s] </TH></TR><TR><TD>1. Initial Denaturation	 </TD><TD>95 </TD><TD> 180</TD></TR><TR><TD> 2. Denaturation</TD><TD> 95</TD><TD>30 </TD></TR><TR><TD> 3. Annealing	</TD><TD> 60</TD><TD>30 </TD></TR><TR><TD> 4. Extention</TD><TD> 68</TD><TD> 60/kb</TD></TR><TR><TD> 5. Final  Extention</TD><TD>68 </TD><TD>300 </TD></TR><TR><TD> 6. storage</TD><TD> 4</TD><TD> for ever </TD></TR></TBODY></TABLE></P><DIV style="margin-top:-140px; margin-left:625px">
34 cycles
</DIV>
Presence, absence and size of amplification products were determined by agarose gel electrophoresis. <H4><SPAN class="mw-headline" id="1.3.4_Quick_Change_.28QC.29-PCR:">1.3.4 Quick Change (QC)-PCR:</SPAN></H4><P>The QC-PCR is a common variation of the standard PCR and useful for site-directed mutagenesis of plasmid DNA. The special design of mutagenic primers allows the insertion or deletion of whole sequences, respectively, as well as the substitution of single nucleotides for applications such as restriction site mutagenesis or amino acid exchange. Figure 2 shows the experimental setup for a QC-PCR. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:572px;"><DIV class="thumbcaption"><B>Figure 2: Scheme of a QC-PCR for a single nucleotide substitution.</B> Mutagenic primers contain the mismatch for the single nucleotide substitution at the 5`terminus and bind inversely to the plasmid DNA.  The first step is an exponential amplification around the template plasmid DNA. The second step involves the incubation with DpnI for specific digestion and removal of methylated and hemi-methylated template DNA. Subsequent transformation is performed exclusively with the mutated plasmid DNA.</DIV></DIV></DIV></DIV><P>The following components were used for the standard PCR reactions:
</P><P><TABLE><TBODY><TR><TD>2x Q5 Master Mix </TD><TD>25 µl </TD></TR><TR><TD> 10 µM Forward QC Primer </TD><TD> 2,5 µl</TD></TR><TR><TD> 10 µM Reverse QC Primer </TD><TD> 2,5 µl</TD></TR><TR><TD>Plasmid DNA </TD><TD> 10 ng</TD></TR><TR><TD>ddH2O </TD><TD> to 50 µl</TD></TR></TBODY></TABLE></P><P>The following speed cycle program was used: 
</P><P><TABLE><TBODY><TR><TH>Step </TH><TH>Temperature [°C] </TH><TH>Time [s] </TH></TR><TR><TD>1. Initial Denaturation	 </TD><TD>95 </TD><TD> 30</TD></TR><TR><TD> 2. Denaturation</TD><TD> 98</TD><TD>15 </TD></TR><TR><TD> 3. Annealing	</TD><TD> 60</TD><TD>35 </TD></TR><TR><TD> 4. Extention</TD><TD> 72</TD><TD> 20/kb</TD></TR><TR><TD> 5. Final  Extention</TD><TD>72 </TD><TD>300 </TD></TR><TR><TD> 6. storage</TD><TD> 4</TD><TD> for ever </TD></TR></TBODY></TABLE></P><DIV style="margin-top:-140px; margin-left:625px">
20 cycles
</DIV>
The following components were used for digestion and removal of the template plasmid DNA : 
<P><TABLE><TBODY><TR><TD>10 x FastDigest Buffer</TD><TD>5 µl </TD></TR><TR><TD>DpnI</TD><TD>1 U </TD></TR><TR><TD>QC-PCR Batch </TD><TD> 40 µl</TD></TR><TR><TD>ddH2O</TD><TD>to 50 µl </TD></TR></TBODY></TABLE>
Presence, absence and size of amplification products were determined by agarose gel electrophoresis after digestion with DpnI.
</P><H4><SPAN class="mw-headline" id="1.4_Isolation_of_plasmid_DNA">1.4 Isolation of plasmid DNA </SPAN></H4><P>The isolation of plasmid DNA was performed with a QIAprep® Spin Miniprep Kit (Qiagen). The provided instructions were followed precisely. </P><H4><SPAN class="mw-headline" id="1.5_Isolation_of_genomic_DNA_of_Bacillus_megaterium:">1.5 Isolation of genomic DNA of<I> Bacillus megaterium</I>:</SPAN></H4><P>The isolation of genomic DNA of <I>Bacillus megaterium </I>was performed with a NexttecTM 1-Step DNA Isolation Kit for Bacteria. The provided instructions were followed precisely. </P><H4><SPAN class="mw-headline" id="1.6_Measurement_of_DNA_concentration:">1.6 Measurement of DNA concentration:</SPAN></H4><P>The measurement of DNA concentration after plasmid DNA isolation was performed with the UV2100 UV/Vis Recording Spectrophotometer (Shimadzu). The spectrum was analysed between wavelengths of 300 nm and 200 nm. The level of DNA purity was assessed with the q-value of A<SUB>260</SUB> /A<SUB>280</SUB>. 
</P><H4><SPAN class="mw-headline" id="1.7_Agarose_gel_electrophoresis_.28AGE.29">1.7 Agarose gel electrophoresis (AGE)</SPAN></H4><P>The agarose gel electrophoresis is a standard method in molecular biology primarily used for separation of DNA or RNA molecules of varying lengths, respectively. The separation effect is due to the filtering properties of the three dimensional agarose gel matrix. After applying an electric field it enables short negatively charged DNA molecules to travel faster towards the positive anode than large negatively charged DNA molecules. As a consequence the distribution pattern of DNA molecules is inversely proportional to their molecular weights. DNA fragments with identical lengths form bands inside the gel. After separation and staining with an appropriate dye (e.g. ethidium bromide) the DNA bands can be visualized under ultra violet light. 
The agarose gel electrophoresis was performed with prestained (peqGREEN) 1 % agarose gels (w/v) in 0.5 x TBE buffer. The 2-Log DNA Ladder (NEB) was used as indicator for DNA size. 120 V were applied for 50 min for appropriate separation.</P><H4><SPAN class="mw-headline" id="1.8_Gel_extraction_and_purification_of_DNA_fragments">1.8 Gel extraction and purification of DNA fragments</SPAN></H4><P>Both procedures were performed with the QIAquick Gel Extraction Kit (Qiagen). The provided instructions were followed precisely with the exception that the elution step was performed with 25 µl preheated ddH2O. 
</P><H4><SPAN class="mw-headline" id="1.9_Ligation_of_DNA_fragments">1.9 Ligation of DNA fragments</SPAN></H4><P>In the ligation process selected DNA fragments such as insert DNA and plasmid constructs are enzymatically linked to form recombinant DNA molecules. The phosphodiester bond is formed between 3`hydroxyl groups and 5`phosphoryl groups of terminal nucleotides. The energy for the reaction is provided by addition of the co-factor ATP.
The following components were used for the basic ligation mixture:<TABLE><TBODY><TR><TD> 10 x T4 Ligation Buffer </TD><TD> 2 µl</TD></TR><TR><TD>Insert DNA and Plasmid Construct </TD><TD>3:1 Molar Ratio (approx. 50 ng) </TD></TR><TR><TD> T4 DNA Ligase</TD><TD> 1 U</TD></TR><TR><TD>ddH2O </TD><TD> to 20 µl</TD></TR></TBODY></TABLE></P><P>The ligation reaction was incubated for 10 min at 22 °C. Heat inactivation was performed for 10 min at 65°C. Success of the ligation reaction was checked via agarose gel electrophoresis with 5 µl of the ligation batch. </P><H4><SPAN class="mw-headline" id="1.10_Dialysis_of_the_ligation_mixture">1.10 Dialysis of the ligation mixture</SPAN></H4><P>Dialysis of the ligation mixture was performed on MF-Membrane Filter Type VSWP02500 (Millipore) in 10 % glycerol for 1h at RT. Transformation of competent <I>E. coli</I> cells was subsequently performed with 5 µl of the dialyzed ligation batch. 
</P><H3><SPAN class="mw-headline" id="2._Transformation_and_cultivation_of_E._coli_and_B._megaterium">2. Transformation and cultivation of <I>E. coli </I>and <I>B. megaterium</I></SPAN></H3><H4><SPAN class="mw-headline" id="2.1_Measurement_of_cell_density">2.1 Measurement of cell density</SPAN></H4><P>Measurement of cell density was regularly performed after inoculation of cultures with 1 ml samples. Fresh LB medium or TB medium was used as reference for measurement of <I>E. coli</I> or <I>B. megaterium</I> density, respectively. The measurement itself was performed with a Novaspec visible spectrophotometer at a wavelength of 600 nm.</P><H4><SPAN class="mw-headline" id="2.2_Preparation_of_electro_competent_E._coli_cells">2.2 Preparation of electro competent <I>E. coli</I> cells</SPAN></H4><P><I>E. coli</I> cells belong to the class of Gram-negative bacteria and are therefore not naturally competent for absorption of DNA molecules. For a successful transformation with plasmid DNA, <I>E. coli</I> cells have to be made competent artificially. The following protocol was used for preparation of electro competent <I>E. coli</I> cells with high transformation efficiency of 2 x 108 cfu/µg pUC19 DNA:
2x 400 ml LB medium in 2000 ml Erlenmeyer flasks were inoculated with one hundredth of a fresh overnight culture of the <I>E. coli</I> cloning strain TOP10F´. The main cultures were incubated at 37 °C and 180 rpm until a cell density of A<SUB>600</SUB>=0.4. The measurement of the cell density was regularly performed over a period of time of approximately 3 hours. The flasks were subsequently chilled on ice for 30 minutes. Afterwards each culture was centrifuged at 4000 g and 4 °C for 10 min. The supernatant was discarded and each pellet resuspended in 400 ml of ice cold, sterile water. Another centrifugation step followed. The supernatant was discarded and each pellet resuspended in 200 ml of ice cold, sterile water. The centrifugation step was repeated. The supernatant was discarded and each pellet resuspended in 100 ml of ice cold, sterile water. Another centrifugation step followed. The supernatant was discarded and each pellet resuspended in 50 ml of ice cold, sterile water. The centrifugation step was repeated.  The supernatant was discarded and each pellet resuspended in 20 ml of ice cold, sterile 10 % glycerol. Another centrifugation step followed. The supernatant was discarded and each pellet resuspended in 10 ml of ice cold, sterile 10 % glycerol. The centrifugation step was repeated. The supernatant was discarded and each pellet resuspended in 2 ml of ice cold, sterile 10 % glycerol. The electro competent <I>E. coli</I> cells were immediately frozen in aliquots of 50 µl and kept at -70 °C until reconstitution.</P><H4><SPAN class="mw-headline" id="2.3_Transformation_of_electro_competent_E.coli_cells:">2.3 Transformation of electro competent <I>E.coli</I> cells:</SPAN></H4><P>In the electroporation process, the membrane of <I>E. coli</I> cells is made permeable by applying a high voltage over a short period of time. Frozen electro competent <I>E.coli</I> cells were slowly thawed on ice and mixed with up to 5 µl of prechilled, dialyzed DNA. After careful resuspension the transformation mixture was incubated for 5 minutes on ice and subsequently filled into a prechilled electroporation cuvette (gap width 0.1  cm). The transformation was performed via electroporation with the Easyject Prima (Equibio) at 1800 V. The transformation mixture was immediately mixed with 500 µl of preheated SOC medium and incubated at 37 °C and 700 rpm for 1 h. Different dilutions of the transformation mixture were grown on LB agar plates with an appropriate selection medium at 37 °C over night. Presence and size of insert DNA in plasmid constructs of transformants were checked by colony PCR.</P><H4><SPAN class="mw-headline" id="2.4_Cultivation_of_E._coli_cells">2.4 Cultivation of  <I>E. coli</I> cells</SPAN></H4><P>A 300 ml Erlenmeyer flask was filled with 50 ml of fresh LB medium and 50 µl of an ampicillin stock solution (100 mg/ml). The LB medium is inoculated with an <I>E. coli</I> colony that has been positively tested for insert DNA by colony PCR. The resulting culture was incubated at 37 °C and 180 rpm over night.</P><H4><SPAN class="mw-headline" id="2.5_Preparation_of_B._megaterium_protoplasts">2.5 Preparation of <I>B. megaterium</I> protoplasts</SPAN></H4><P><I>B. megaterium</I> belongs to the class of gram positive bacteria. Thus it is naturally competent for absorption of DNA. However, the thick surrounding peptidoglycane layer prevents the absorption of DNA and has to be degraded. The following protocol was used for preparation of protoplasts.
50 ml LB medium in a 300 ml baffeled flask was inoculated with one hundredth of a fresh overnight culture of the <I>B. megaterium</I> strain MS941. The main culture was incubated at 37 °C and 180 rpm until a cell density of A<SUB>600</SUB>=1.0. The measurement of the cell density was regularly performed over a period of time of approximately 4 hours. The culture was transferred to a 50 ml Falcon tube and centrifuged at 1280 g and 4 °C for 15 min. The supernatant was discarded, the pellet was resuspended in 5 ml of fresh SMMP and transferred to a new 15 ml Falcon tube. The protoplast formation was initiated by addition of 100 µl of lysozyme solution (10 mg/ml in SMMP). The mixture was incubated at 37 °C without shaking until 80-100 % of the cells showed the typical globular morphology of protoplasts. The progress of the reaction was checked under a microscope every 10 min. If the formation of protoplasts has not been initiated, another 100 µl of the lysozyme solution should be added. After successful protoplast formation, the mixture was centrifuged at 1280 g and RT for 10 min. The supernatant was carefully removed and the pellet was washed with 5 ml of SMMP. Both, centrifugation and washing step were repeated. Subsequently 750 µl of sterile 87 % glycerol were added to the protoplast suspension and carefully mixed by tilting the tube. Finally the protoplasts were frozen in 500 µl aliquots and kept at-70 °C until reconstitution. 
</P><H4><SPAN class="mw-headline" id="2.6_Transformation_of_B._megaterium_protoplasts">2.6 Transformation of <I>B. megaterium</I> protoplasts</SPAN></H4><P>A 15 ml Falcon tube was prepared and filled with 1.5 ml of PEG-P which serves as a carrier matrix for the DNA. The protoplast suspension was slowly thawed on ice and carefully mixed with 5-10 µg of DNA. With a micropipette in circling motions the protoplast-DNA-solution was subsequently pulled vertically through the PEG-P. The resulting mixture was mixed by tilting the tube and was incubated for 2 min at RT. Afterwards the mixture was resuspended with 5 ml of SMMP and centrifuged at 1280 g and RT for 10 min. The supernatant was immediately removed, the pellet resuspended in 500 µl SMMP and transferred into a sterile 1.5 ml reaction vessel. The regeneration of the cell wall was triggered in two steps. The first incubation step was performed at 30 °C without rotations, while the second one was performed at 30°C and 300 rpm. During these 90 min incubation time, 3.5 ml of CR5 overlay agar were prepared in a 15 ml Falcon tube and kept warm in a water bath at a temperature of 42 °C. After incubation the transformation mixture was added to the overlay agar and carefully mixed by tilting the 15 ml Falcon tube. The agar was poured onto LB plates with an appropriate selection medium. The plates were incubated over night at 37 °C. The next day single colonies were transferred to fresh LB plates with selection medium and incubated again over night at 37 °C. To avoid contamination, the following day single colonies were identified via microscopy as representatives of <I>B. megaterium</I>.</P><H4><SPAN class="mw-headline" id="2.7_Cultivation_of_B._megaterium_cells">2.7 Cultivation of <I>B. megaterium</I> cells</SPAN></H4><P>A 300 ml Erlenmeyer flask was filled with 50 ml of LB medium and 50 µl of a tetracycline stem solution (10 mg/ml). Transformed <I>B. megaterium</I> strains were taken from the corresponding glycerol stock and put into the Erlenmeyer flask. This preculture was incubated at 37 °C and 180 rpm overnight. The next day a mix of 45 ml fresh TB medium, 5 ml 10 x KPP buffer and 50 µl tetracycline stem solution was inoculated in a 300 ml baffled flask with one hundredth of the fresh overnight culture of the<I> B. magisterium</I> and incubated at 30 °C and 180 rpm until a cell density density of A<SUB>600</SUB>= 0,4. Protein expression was induced by addition of 1 ml of a xylose solution (250 mg/ml) followed by further incubation at 30 °C and 180 rpm.</P><H3><SPAN class="mw-headline" id="3._Protein_expression_analysis:">3. Protein expression analysis:</SPAN></H3><H4><SPAN class="mw-headline" id="3.1_Sample_preparation_for_SDS_Page">3.1 Sample preparation for SDS Page</SPAN></H4><P>Protein expression levels were checked after 48 h of cultivation. Cell density was adjusted to an A<SUB>600</SUB> nm = 3. Cell pellets were resuspended in 30 μl of lysis buffer and incubated for 30 min at 37 °C and 1,000 rpm. Vortexing every 10 minutes should increase cell lysis. Separation of insoluble fraction (pellet) from the soluble fraction (supernatant) was performed by centrifugation for 30 min at 4 °C and 13,000 rpm. 24 µl of the supernatant (containing soluble proteins) were mixed with 6 µl of  SDS sample buffer. Subsequently, the supernatant was completely removed. The pellet fraction was resuspended in 30 ml of 8 % urea (w/v) and centrifuged for 30 min at 4 °C and 13,000 rpm. 24 µl of the supernatant (containing insoluble proteins) were mixed with 6 µl of SDS sample buffer. Each sample was heated for 5 min at 95 °C. 7 µl of each sample were used for protein separation by SDS-PAGE.</P><H4><SPAN class="mw-headline" id="3.2_SDS-PAGE:">3.2 SDS-PAGE:</SPAN></H4><P>The samples were separated using a 12 % separating gel and a 4% stacking gel. 7 µl of each sample were loaded onto the gel. The gel electrophoresis ran for approximately 40 min at 20 mA. </P><H4><SPAN class="mw-headline" id="3.3_Western_Blot:">3.3 Western Blot:</SPAN></H4><P>The used procedure is a semi-dry blotting procedure.
The previously separated samples were transferred to a nitrocellulose membrane. The gel was put under voltage (20 V; intensity: 0,4 A; power: 8 W) during 40 min. Afterwards the membrane was incubated in blocking buffer for 60 min on a shaking device at room temperature. The nitocellulose membrane was then incubated in primary anti-GFP antibody (rabbit) overnight at 4 °C on a shaking device. The membrane was rinsed with water and washed with PBS-T (3x 5 min) to eliminate the unbound antibodies. The blot was incubated with secondary anti-rabbit peroxidase antibodies for 60 min at room temperature on a shaking device. After this step the washing procedure was repeated. Subsequently, the nitrocellulose membrane was put in a transparent foil and residual liquid was removed. The Western Lightning Enhanced Chemiluminescence substrates were mixed in equal parts and evenly spread on the membrane. After an incubation of 10 s the substrates were removed. The blot was developed on an x-ray film with developing and fixating solution in the dark room.</P><H4><SPAN class="mw-headline" id="3.4_Sample_preparation_for_fluorescence_microscopy:">3.4 Sample preparation for fluorescence microscopy:</SPAN></H4><P>Localisation studies of the GFP tagged hyaluronan synthase (Has2) were carried out after 24 h and 48 h of cultivating the corresponding <I>B. megaterium </I>strains. 1 ml sample was centrifuged for 1 min at 11000 g and the cell pellet was washed with 1 ml of ddH2O. 10 µl of the cell suspension were put on a microscope slide an air dried. </P><H3><SPAN class="mw-headline" id="4._Confirmation_of_hyaluronic_acid_production_by_rheometry">4. Confirmation of hyaluronic acid production by rheometry</SPAN></H3><P>Since hyaluronic acid is very viscous and 1 g of it can bind 6 litres of water, the viscosity of the medium should rise significantly (at least by a factor of 10-100) when hyaluronic acid is synthesised. Viscosity measurements were carried out using a parallel discs rheometer.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 3:</B> Schematic structure of a parallel discs rheometer.</DIV></DIV></DIV></DIV><P>Rheometers are generally used for measuring the (complex) viscosity of materials dependent on quantities like temperature, time or shear velocity. As shown in figure 3, a parallel disc rheometer is composed of two horizontal discs. The bottom disc is fixed and the upper disc is connected to anelectric motor. For our experiments, 1 ml of sample was used. 
With a distance of 20 nm to the lower disc, the upper disc is set into rotation and the torque needed to sustain this motion is measured. Based on the torque, shear stress τ in and thus the viscosity η of the fluid can be calculated: 
Because hyaluronic acid is a pseudoplastic fluid, meaning that its viscosity decreases with rising shear forces, the measurement is done over a range of shear forces and should result in a characteristic curve like the one shown in figure 4 if hyaluronic acid is present within the solution.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 4:</B> Viscosity of Newtonian fluids is constant for varying shear rates. The viscosity for pseudoplastic fluids such as hyaluronic acid follows a 1/x distribution.</DIV></DIV></DIV></DIV></DIV></DIV><DIV id="social-footer"><DIV><H4><A href="https://2014.igem.org/Team:Saarland/Impressum">Impressum/Copyright</A></H4></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Saarland/Methods">http://2014.igem.org/Team:Saarland/Methods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Saarland/Methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Saarland/Methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Saarland/Methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Saarland/Methods&amp;oldid=317968" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>