<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TorontoMaRSDiscovery_Bioinformatics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:TorontoMaRSDiscovery/Bioinformatics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TorontoMaRSDiscovery/Bioinformatics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TorontoMaRSDiscovery/Bioinformatics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TorontoMaRSDiscovery/Bioinformatics&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TorontoMaRSDiscovery/Bioinformatics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TorontoMaRSDiscovery/Bioinformatics</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><TABLE style="color:white;background-color:#99CCFF;" cellpadding="2" cellspacing="0" border="0" width="100%" align="center" class="menu"><TBODY><TR><TH align="center"><A href="/Team:TorontoMaRSDiscovery" title="Team:TorontoMaRSDiscovery">Home</A></TH><TH align="center"><A href="/Team:TorontoMaRSDiscovery/Team" title="Team:TorontoMaRSDiscovery/Team">The Team</A></TH><TH align="center"><A href="/Team:TorontoMaRSDiscovery/Project" title="Team:TorontoMaRSDiscovery/Project">The Project</A></TH><TH align="center"><A href="/Team:TorontoMaRSDiscovery/Parts" title="Team:TorontoMaRSDiscovery/Parts">BioBricks</A></TH><TH align="center"><A href="/Team:TorontoMaRSDiscovery/Modeling" title="Team:TorontoMaRSDiscovery/Modeling">Modelling</A></TH><TH align="center"><STRONG class="selflink">Bioinformatics</STRONG></TH><TH align="center"><A href="/Team:TorontoMaRSDiscovery/Safety" title="Team:TorontoMaRSDiscovery/Safety">Safety</A></TH><TH align="center"><A href="/Team:TorontoMaRSDiscovery/Notebook" title="Team:TorontoMaRSDiscovery/Notebook">Notebook</A></TH></TR></TBODY></TABLE><P style="font-size:18pt;">The diversity of bacterial microcompartments in nature and their potential for biotechnological applications</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Summary"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Summary</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Methods.2FResults"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Methods/Results</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#a.29_Search_for_enzyme_pairs"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">a) Search for enzyme pairs</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#b.29_Alternative_microcompartments"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">b) Alternative microcompartments</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Conclusions"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Conclusions</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Future_Work"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Future Work</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Summary">Summary</SPAN></H2><P>Spatial segregation is widely believed to be a defining organizational feature of eukaryotic cells: proteins, nucleic acids and small molecules are contained within and often actively transported between the many membrane-bound, subcellular organelles such as mitochondria, lysosomes, Golgi apparatus, etc. However, it has recently been found that a number of bacteria conditionally express proteinaceous microcompartments. These polyhedral organelles are usually 100-150 nm in cross section [1] and consist of proteinaceous outer shells, reminiscent of viral capsids, surrounding a core of enzymes[2].  It is thought that microcompartments allow bacteria to sequester specific metabolic enzymes and their substrates to enhance enzymatic efficiency (enzyme channeling) and protect cells from the toxic effects of certain intermediates. While several examples of these compartments have been reported their diversity has not been fully explored.  We wish to design new microcompartments with modified properties or novel enzymatic activities, which could result in potentially useful applications in biotechnology.  With this goal in mind, we attempted to address the following two questions using a bioinformatics approach (1) What enzymes will benefit the most from enzymatic channeling? and (2) Are there any other alternative microcompartments that could be explored for enzymatic channeling engineering?
</P><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><P>Most of our knowledge on bacterial microcompartments has been derived from the three well-studied microcompartment systems in nature.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Figure 1:  The carbon concentrating mechanism in carboxysome.</DIV></DIV></DIV><OL><LI>Carboxysomes
</LI><LI>First reported in 1956, carboxysomes were the first bacteria microcompartments to be discovered. They are often present in cyanobacteria and other chemoautotrophic bacteria [3]. They are known to play a key role in enhancing autotrophic carbon fixation in the Calvin cycle. The shell of the carboxysome encodes the enzymes carbonic anhydrase (CA) and ribulose bis-phosphate carboxylase monooxygenase (RuBisCO). CA converts bicarbonate ions into carbon dioxide, which is then converted into 3-phosphoglycerate (3-PGA) by RuBisCO. Carboxysome not only allows for co-localization of CA and RuBisCO, but also acts as a diffusion barrier to retain carbon dioxide in the immediate vicinity of RuBisCO and thus catalyzes the conversion [2]. See Figure 1.
</LI><LI><I>Pdu</I> microcompartment
</LI><LI>In 1994, homologues of carboxysome shell proteins were reported in <I>S. enterica</I>. They are amongst a cluster of genes that are involved in coenzyme B12-dependent metabolism of 1,2 propanediol [4]. The gene cluster was later termed the pdu operon and the microcompartment formed was later termed propanediol utilization microcompartment.  The proposed fuction of the pdu microcompartment is to encapsulate the enzymes that are necessary for cell to degrade propanediol and most importantly, to protect the cell from the toxic effects of propionaldehyde, an intermediate formed during the process [5].
</LI><LI><I>Eut</I> microcompartment
</LI><LI>Later, similar structures were also found in <I>E.coli</I> and <I>S. enterica</I> when they were grown using ethanolamine as energy source. These structures were named ethanolamine utilization microcompartment (encoded by eut operon) and they often display very high genetic similarity with pdu microcompartment. Eut microcompartments contain enzymes involved in the degradation of ethanolamine and protect cell from acetaldehyde [6-8].
</LI><LI><I>Generally speaking, both pdu and eut microcompartments are less uniform in size and more irregular in shape than carboxysomes[2].</I></LI></OL><P>These compartments are generally unsuitable for our purposes because a) Their shell proteins consist of several different types whose functions are not well characterized and b) It is not known how the associated enzymes are targeted to the microcompartment.  However, several other putative microcompartments have been inferred by sequence similarity.  These include 
</P><UL><LI>A microcompartment was suggested to be involved in the oxidation of ethanol by <I>Clostridium kluyveri</I>. [9].
</LI><LI>A microcompartment associated with a puryvate-formate lyase homolog is proposed to be involved in the production of ethanol from pyruvate [10].
</LI><LI>A putative microcompartment in <I>Rhodopirellula baltica</I> is proposed to be associated with a lactate dehydrogenase homologue [1].
</LI><LI>A putative microcompartment in <I>Carboxydothermus hydrogenoformans</I> is associated with an isochorismatase-family protein [1].
</LI><LI>A putative microcompartment Solibacter usitatus can be associated with a dihydrdipicolinate synthase homologue [1].
</LI></UL><P>The observation that diverse microcompartment structures are composed of proteins with homologous sequences led to the identification of a protein domain in the shell of all polyhedral bacterial microcompartments. This conserved protein domain is known as the Bacterial Microcompartment (BMC) domain (<A href="http://pfam.sanger.ac.uk/family?acc=PF00936" class="external text" rel="nofollow">Pfam00936</A>). It is approximately 84 amino acids long and can be either found as a part of a large protein or in tandem copies within the same operon. This domain is found to be present in 189 bacterial species to date. [2]
</P><P>Microcompartments typically promote the catalysis of a particular metabolic reaction by sequestering or co-localizing functionally related enzymes. The product of one enzyme will be delivered to the next enzyme at high concentration since it cannot readily diffuse. This greatly enhances enzyme efficiency.  Microcompartments can also occlude toxic intermediates that cannot be degraded by normal bacterial machinery. This was shown to be true in <I>pdu</I> and <I>eut</I> microcompartments. The degradation pathway of 1,2-propanediol and ethanolamine both proceed through aldehyde intermediates (propionaldehyde and acetaldehyde respectively). The toxicity of these aldehyde intermediates was later shown in growth assays in which cells accumulating large amounts of propionaldehyde underwent growth arrest due to propionaldehyde toxicity [5, 11]. Therefore, it is reasonable to propose that bacteria use microcompartments to mitigate the toxicity of aldehyde by encapsulating these poisonous intermediates.
</P><H2><SPAN class="mw-headline" id="Methods.2FResults">Methods/Results</SPAN></H2><H3><SPAN class="mw-headline" id="a.29_Search_for_enzyme_pairs">a) Search for enzyme pairs</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Figure 2:  Complete pathway map obtained from KEGG database with the first enzyme colored in red:.</DIV></DIV></DIV><P>In order to identify candidate enzyme pairs likely to display an observable benefit from enzyme channeling we made the following assumptions:
</P><OL><LI>Enzyme pairs with specific thermodynamic properties will likely benefit. Specifically, for two sequential reactions the second one will be thermodynamically more favorable than the first reaction. Consequently, the products from the first reaction be consumed at a faster rate than being generated, hence driving the reaction to completion. 
</LI><LI>Predictions based on free energy values: <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Scored.xls" class="new" title="Scored.xls">Scored.xls</A></LI><LI>Enzymes involved in reactions with toxic intermediates will likely benefit from enzymatic channeling since the toxin will be less likely to interact with cellular components.
</LI><LI>Enzyme pairs that possess a common metabolite that is known to be toxic: <A href="/wiki/images/4/42/ToxicPairs.xls" class="internal" title="ToxicPairs.xls">ToxicPairs.xls</A></LI><LI>Enzymes that were identified as gene fusion products in other organisms will likely benefit from enzyme channeling since fusion of multi-functional enzymes is an extreme example of colocalization.
</LI><LI>Enzyme pairs predicted to form multifunctional enzymes by the Prolinks Rosetta Stone analysis: <A href="/wiki/images/0/00/RosettaEnzymePairs.xls" class="internal" title="RosettaEnzymePairs.xls">RosettaEnzymePairs.xls</A></LI><LI>Enzyme pairs annotated to form multifunctional enzymes in SwissProt database: <A href="/wiki/images/b/b9/SwissProtPairs.xls" class="internal" title="SwissProtPairs.xls">SwissProtPairs.xls</A></LI></OL><P>These results provided us with a combined list that was further refined by removing enzymes that are not immediately adjacent to each other in their biochemical pathway based on the KEGG database (Figure 2).  In addition, enzymes with a molecular weight well beyond the microcompartment capacity (100 kDa) were also eliminated.  The remaining enzyme pairs meeting these criteria were considered candidates for enzyme channeling and have been summarized in the following table:
</P><TABLE border="1" cellpadding="5" cellspacing="0"><TBODY><TR><TD align="center"><B>Enzyme1 EC</B></TD><TD align="center"><B>Enzyme1 Size (kDa)</B></TD><TD align="center"><B>Enzyme2 EC</B></TD><TD align="center"><B>Enzyme2 Size (kDa)</B></TD><TD align="center"><B>Source</B></TD><TD align="center"><B>Pathway</B></TD><TD align="center"><B>Major Product</B></TD><TD align="center"><B>Applications</B></TD></TR><TR><TD> 2.1.3.2</TD><TD>32-125</TD><TD>6.3.4.4</TD><TD>47-96</TD><TD>Rosette</TD><TD>Alanine and aspartate metabolism</TD><TD>adenylosuccinate</TD></TR><TR><TD> 1.1.1.1</TD><TD>25-57</TD><TD>1.2.1.10</TD><TD>96.7-520</TD><TD>Swiss Prot</TD><TD>Butanoate metabolism </TD><TD>Butanoyl-coA</TD></TR><TR><TD> 1.1.1.95</TD><TD>35-250</TD><TD>2.6.1.52</TD><TD>35-96</TD><TD>Rosette</TD><TD>Glycine, serine and threonine metabolism </TD><TD>Phosphoserine</TD></TR><TR><TD> 2.3.3.9</TD><TD>52-81</TD><TD>4.1.3.1</TD><TD>14-65</TD><TD>Swiss Prot &amp;Rosette</TD><TD>Glyoxylate and dicarboxylate metabolism</TD><TD>Isocitrate</TD></TR><TR><TD> 1.5.1.15</TD><TD>32-34</TD><TD>3.5.4.9</TD><TD>30-45</TD><TD>Swiss Prot</TD><TD>One carbon pool by folate</TD><TD>Formyl-THF</TD></TR><TR><TD> 1.5.1.5</TD><TD>30-150</TD><TD>3.5.4.9</TD><TD>30-45</TD><TD>Swiss Prot</TD><TD>One carbon pool by folate</TD><TD>Formyl-THF</TD></TR><TR><TD> 1.1.1.57</TD><TD>5.3.1.12</TD><TD>Rosette</TD><TD>Pentose and glucuronate interconversions</TD><TD>D-Glucuronate</TD></TR><TR><TD> 2.2.1.1</TD><TD>100-141</TD><TD>5.1.3.1</TD><TD>45-200</TD><TD>Rosette</TD><TD>Pentose phosphate pathway</TD><TD>D-ribolose-5P</TD></TR><TR><TD> 2.2.1.2</TD><TD>35-75</TD><TD>5.3.1.9</TD><TD>27-67</TD><TD>Rosette</TD><TD>Pentose phosphate pathway</TD><TD>a-glucose-6P</TD></TR><TR><TD> 1.3.1.13</TD><TD>52-210</TD><TD>4.2.1.51</TD><TD>12-137</TD><TD>Rosette</TD><TD>Phenylalanine, tyrosine and tryptophan biosynthesis</TD><TD>Phenyl-pyruvate</TD></TR><TR><TD> 2.3.1.8</TD><TD>35-71</TD><TD>2.7.2.15</TD><TD>43</TD><TD>Rosette</TD><TD>Propanoate metabolism</TD><TD>Propanoate</TD><TD>Food preservatives
</TD></TR><TR><TD> 2.3.1.8</TD><TD>35-70</TD><TD>2.7.2.1</TD><TD>12 to 66</TD><TD>Rosette</TD><TD>Propanoate metabolism/Pyruvate metabolism</TD><TD>Propanoate/acetate</TD><TD>Energy
</TD></TR><TR><TD> 2.4.2.10</TD><TD>39-140</TD><TD>4.1.1.23</TD><TD>14-64</TD><TD>Swiss Prot</TD><TD>Pyrimidine metabolism</TD><TD>UMP</TD></TR><TR><TD> 2.4.2.1</TD><TD>45-86</TD><TD>2.4.2.3</TD><TD>32-160</TD><TD>Rosette</TD><TD>Pyrimidine metabolism</TD><TD>Uridine</TD></TR><TR><TD> 2.4.1.15</TD><TD>45-630</TD><TD>3.1.3.12</TD><TD>25-973</TD><TD>Rosette</TD><TD>Starch and sucrose metabolism</TD><TD>a,a trehalose</TD><TD>biotechnology application
</TD></TR><TR><TD> 1.2.1.41</TD><TD>41-189</TD><TD>2.7.2.11</TD><TD>47-354</TD><TD>Swiss Prot &amp;Rosette</TD><TD>Urea cycle and metabolism of amino groups</TD><TD>Glutamate</TD><TD>Flavor enhancer, Plant growth
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="b.29_Alternative_microcompartments">b) Alternative microcompartments</SPAN></H3><P>Upon further examination, most of our known microcompartments tend to consist of a large number of protein subunits with very complex structures. Because of their complicated regulatory patterns and stoichiometry, these structures are not ideal to be used in enzymatic channeling engineering.
</P><UL><LI>Carboxysome: 80-150 nm in cross section. Composed of several thousands polypeptide of 10-15 different types. Contain as many as 250 RuBisCO per carboxysome molecule[12, 13].
</LI></UL><UL><LI><I>Pdu</I> Microcomparment: 100-150 nm in cross section. Composed of about 18000 individual polypeptides of about 14-18 different types [1].
</LI></UL><UL><LI><I>Eut</I> Microcomparment: Similar to pdu microcompartment in terms of both size and protein composition.
</LI></UL><P>  
One of the putative microcompartment systems in <I>Clostridium kluyveri</I> associated with the oxidation of ethanol, however, shows promise as an alternative microcompartment for use in our system because:
</P><OL><LI>The gene cluster involved contains only seven genes: including two genes for a pair of nearly identical acetaldehyde dehydrogenases, three genes for highly similar ethanol dehydrogenases, and two genes for microcompartment proteins, which are orthologs of ethanolamine using genes (eutML) of <I>Salmonella typhimurium</I> [9].
</LI><LI>From cell extracts of <I>C. kluyveri</I> a macromolecular complex of ethanol dehydrogenase and acetaldehyde dehydrogenase can be purified by differential manganese sulfate precipitation, indicating the presence of a functional and intact microcompartment enclosing these two enzymes[9].
</LI></OL><P>In order to co-opt this stucture for our own use it is necessary to understand how the associated enzymes are targeted to the compartment so that alternative enzymes of our choosing can be substituted.  To search for a consensus targeting sequence we obtained the <I>C. kluyveri</I> compartment sequence from NCBI and performed a position specific iterative Blast search for related sequences among bacteria.  We used the following two query sequences: 
</P><TABLE border="1" cellspacing="0" width="50%"><TBODY><TR><TD><B>CKL_1072</B></TD></TR><TR><TD>153953697|ref|YP_001394462.1| microcompartment protein [Clostridium kluyveri DSM 555]
</TD></TR><TR><TD>MGQEALGMIETKGLVGAIEAADSMVKAANVALIGYEKIGSGLVTVMVRGDVGAVKAATDAGAASAKRVGE
<P>VISVHVIPRPHTDVEKILPNIG
</P></TD></TR></TBODY></TABLE><TABLE border="1" cellspacing="0" width="50%"><TBODY><TR><TD><B>CKL_1073</B></TD></TR><TR><TD>153953698|ref|YP_001394463.1| microcompartment protein [Clostridium kluyveri DSM 555]
</TD></TR><TR><TD>MNNELIEKVLGEVRKSLDLKNFDQEKLNKVVESTTEKLSDSKKEEAIKEAKPDVKVAEESKQAVVEQKAN
<P>DVKTAPTMTEFVGTAGGDTVGLVIANVDSLLHKHLGLDNTCRSIGIISARVGAPAQMMAADEAVKGTNTE
VATIELPRDTKGGAGHGIFIVLKAADVSDARRAVEIALKQTDKYLGNVYLCDAGHLEVQYTARASLIFEK
AFGAPSGQAFGIMHAAPAGVGMIVADTALKTADVKLITYGSPTNGVLSYTNEILITISGDSGAVLQSLTA
ARKAGLSILRSMGQDPVSMSKPTF
</P></TD></TR></TBODY></TABLE><P>PSI-BLAST constructs and modifies the Position Specific Scoring Matrix constantly and uses it during the next round of search. This iterative searching strategy results in increased sensitivity. The top results were saved after 5 iterations. Download the <A href="/wiki/images/8/84/1072_3_BLAST.pdf" class="internal" title="1072 3 BLAST.pdf">BLAST results</A> and <A href="/wiki/images/e/e1/Taxonomy_report.txt" class="internal" title="Taxonomy report.txt">taxonomy report</A>.A <A href="/wiki/images/b/b1/MUSCLE_alignment.pdf" class="internal" title="MUSCLE alignment.pdf">multiple alignment</A> of these sequences was constructed using CLCBio Genomic Workbench to assess the level of conservation between species and identify consensus sequence. 
</P><P>Since microcompartment proteins are known to be found adjacent to their corresponding enzymes, we searched the top hits for adjacent genes 5’ upstream and 3’ downstream of the hit gene.  We then attempted to apply multiple sequence alignment to these sequences in an effort to identify conserved motifs corresponding to a potential targeting sequence similar to the method employed by Suter <I>et al</I>.  A targeting sequence, if found, would allow translocation of enzymes into the microcompartment. Due to the incomplete annotation of certain genomes, the alignments did not yield a statistically significant targeting motif.  We are attempting further analyses using different reading frames to refine the protein candidates used during the multiple alignment.
</P><H2><SPAN class="mw-headline" id="Conclusions">Conclusions</SPAN></H2><P>We have short-listed a number of enzymes which might benefit from channeling according to several properties such as free energy, toxicity and the occurrence of multi-functional homologues.  In addition, while concluding that several classic examples of bacterial microcompartment are unsuitable for experimental manipulation, we have conducted a sequence-based search for additional protein family members corresponding to the <I>Clostridium kluyveri</I> microcompartment and searched the corresponding upstream and downstream genes for conserved motifs in an effort to find the targetting sequence for these enzymes.  The <I>Clostridium kluyveri</I> system remains under consideration as a potentially viable alternative to the encapsulin-based nanocompartment.  However, its suitability will depend on the results of further efforts to identify a targeting sequence for this compartment.
</P><H2><SPAN class="mw-headline" id="Future_Work">Future Work</SPAN></H2><P>In order to find the targeting sequence for the <I>Clostridium kluyveri</I> microcompartment, it will be necessary to fine tune the enzyme queries and potentially check out other alternative reading frames.  In addition, enzymes will be selected from those short-listed above to proceed with a proof of concept experiment and potentially commercially viable application.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>1.Cheng, S., <I>et al</I>., Bacterial microcompartments: their properties and paradoxes. Bioessays, 2008. 30(11-12): p. 1084-95.
</P><P>2.Yeates, T.O., <I>et al</I>., Protein-based organelles in bacteria: carboxysomes and related microcompartments. Nat Rev Microbiol, 2008. 6(9): p. 681-91.
</P><P>3.Shively, J.M., Inclusion bodies of prokaryotes. Annu Rev Microbiol, 1974. 28(0): p. 167-87.
</P><P>4.Chen, P., D.I. Andersson, and J.R. Roth, The control region of the pdu/cob regulon in Salmonella typhimurium. J 
Bacteriol, 1994. 176(17): p. 5474-82.
</P><P>5.Havemann, G.D., E.M. Sampson, and T.A. Bobik, PduA is a shell protein of polyhedral organelles involved in coenzyme B(12)-dependent degradation of 1,2-propanediol in Salmonella enterica serovar typhimurium LT2. J Bacteriol, 2002. 184(5): p. 1253-61.
</P><P>6.Brinsmade, S.R., T. Paldon, and J.C. Escalante-Semerena, Minimal functions and physiological conditions required for growth of salmonella enterica on ethanolamine in the absence of the metabolosome. J Bacteriol, 2005. 187(23): p. 8039-46.
</P><P>7.Penrod, J.T. and J.R. Roth, Conserving a volatile metabolite: a role for carboxysome-like organelles in Salmonella enterica. J Bacteriol, 2006. 188(8): p. 2865-74.
</P><P>8.Stojiljkovic, I., A.J. Baumler, and F. Heffron, Ethanolamine utilization in Salmonella typhimurium: nucleotide sequence, protein expression, and mutational analysis of the cchA cchB eutE eutJ eutG eutH gene cluster. J Bacteriol, 1995. 177(5): p. 1357-66.
</P><P>9.Seedorf, H., <I>et al</I>., The genome of Clostridium kluyveri, a strict anaerobe with unique metabolic features. Proc Natl Acad Sci U S A, 2008. 105(6): p. 2128-33.
</P><P>10.Wackett, L.P., <I>et al</I>., Genomic and biochemical studies demonstrating the absence of an alkane-producing phenotype in Vibrio furnissii M1. Appl Environ Microbiol, 2007. 73(22): p. 7192-8.
</P><P>11.Sampson, E.M. and T.A. Bobik, Microcompartments for B12-dependent 1,2-propanediol degradation provide protection from DNA and cellular damage by a reactive metabolic intermediate. J Bacteriol, 2008. 190(8): p. 2966-71.
</P><P>12.Yeates, T.O., <I>et al</I>., Self-assembly in the carboxysome: a viral capsid-like protein shell in bacterial cells. Biochem Soc Trans, 2007. 35(Pt 3): p. 508-11.
</P><P>13.Tanaka, S., <I>et al</I>., Atomic-level models of the bacterial carboxysome shell. Science, 2008. 319(5866): p. 1083-6.
</P><P>14.Sutter, M. <I>et al</I>. Structural basis of enzyme encapsulation into a bacterial nanocompartment. Nat Struct Mol Biol (2008).
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:TorontoMaRSDiscovery/Bioinformatics">http://2009.igem.org/Team:TorontoMaRSDiscovery/Bioinformatics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TorontoMaRSDiscovery/Bioinformatics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TorontoMaRSDiscovery/Bioinformatics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TorontoMaRSDiscovery/Bioinformatics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TorontoMaRSDiscovery/Bioinformatics&amp;oldid=167145" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>