Team:FAFU-CHINA/Notebook
侧边栏
Notebook
2015.6.8
Today was the first day that we began our project in the lab after we made a scientific plan. Firstly, we extracted the whole RNA of CSBV by Trizol. As we all know, RNA is easy to be digested in the environment. Therefore although we did this quickly, there were still some samples without RNA fragment. But we did again and the result was satisfied at that time. It was encouraging!
2015.6.9
Today we made RT-PCR with four sets of reverse specific primers and the RNA of CSBV to synthesize homologous cDNA(CSBV Helicase、Protease、VP1 and RdRp respectively). Then dsDNAs was amplified with four sets of specific primers by PCR. It went through smoothly. So far, we got the target fragments successfully. Also,we used GFP F/R to amplify GFP gene.
2015.6.10
We utilized T7 RiboMAXTM Express RNAi System kit and dsDNAs as templates to synthesize dsRNAs(dsHelicase、dsProtease、dsVP1、dsRdRp and dsGFP), but there were only dsHelicase and dsGFP we could get. We tried again and still failed. It took our whole day to find the reason. Tomorrow we will ask our advisor for some solutions and do it one more time.
2015.6.12
After figuring out the problem, we eventually got the all right dsRNAs. Another part, we began to prepare the larvae of Chinese honeybee which was offered by our institute of bees.
2015.6.13
We put dsRNAs into fodder of the larvae of the honeybee to feed them. Then after 12h, we fed them another fodder with CSBV extracting solution. We did so twice before they got normal fodder.
2015.6.17
During three days, we observed the change of percentage of pupation and collect and analyse data. Meanwhile,RT-qPCR was done to detect the effect of different dsRNAs to CSBV. According to the result, we found that dsRdRp made a remarkable influence on the replication of CSBV. So we decided to use dsRdRp to inference CSBV.
2015.6.19
We used cDNA of RdRp and specific primers(Not1-F/Pst1-R、EcoR1-F/Pst1-R) to synthesize RdRp gene. However, after agarose gel electrophoresis(AGE), we could not find our target fragment.
2015.6.20
We changed a better DNA polymerase and set the annealing temperature to 60℃. Finally we got the RdRp gene.
2015.6.22
We picked the colonies to do PCR and sent the positive sequencing(M13).
2015.6.24
Today we got the result of sequencing. After comparing with data in NCBI, we only had two 100 percent samples.  We started to do double enzyme digestion(Not1 and Pst1、EcoR1 and Pst1) to cut vector L4440、pSB1C3 and RdRp-T recombinant plasmid.
2015.6.25
We made RdRp gene link with T vector in 25℃, 15min and then transformed it into DH5α,a kind of competent cell, which was cultured in 37℃,overnight.
2015.6.26
We made it! And we immediately did gel recovery. Then we began to link RdRp with L4440 and pSB1C3 using T4 ligase, 4℃, overnight.
2015.6.27
We did transformation. However,we forgot to make LB medium with CmRR. So the transformation of RdRp-pSB1C3 could only be done tomorrow.
2015.6.29
Today there was two news. The good one was we successfully got recombinant plasmid RdRp-L4440 while the bad one was nothing grew on the plate with CmRR. We thought maybe it was because RdRp did not link with pSB1C3 successfully. Anyway we would do it again tomorrow.
2015.7.2
On the one hand, we still could not link RdRp with pSB1C3 so that there still nothing on the plate. We wonder if it was because of the efficiency of ligase or anything. On the other hand,we prepared another competent cell,HT115,which included T7 RNAP gene in its genome. It was used as engineering bacteria for expressing dsRdRp.
Also, considering the biosecurity of E.coli, we decided use yeast as the final transformation target. So we want to recombine another vector,T7 RNAP-pYES2.
2015.7.3
We transformed RdRp-L4440 into HT115,culturing in 37℃. Overnight. And T7 RNAP gene was amplified by PCR(EcoR1-F/Xho1-R).
2015.7.4
Today we used IPTG, which was divided into two groups with different concentrations(0.4mmol/ml、0.8mmol/ml), to induce the expression of dsRdRp in HT115 for 5h. Then we collected the bacteria to extract dsRdRp by CTAB. According to the result of AGE, dsRdRp had been expressed in HT115 successfully, even though its concentration was a little bit lower. It was really a good news for us!
2015.7.5
We added engineering bacteria with expressed dsRdRp, which was divided into three groups with different concentrations(low、mediate and high), into the fodder of the infected swarms. And another infected swarm was fed normal fodder. Since now, we will obverse the change of population of them, the mortality, and the number of sealed brood and collect data every weeks until we get the obvious result.
2015.7.7
Considering we have not gotten RdRp-pSB1C3 yet,we decided to link T7 RNAP gene to pSB1C3 first. So we used specific primers(EcoR1-F/Pst1-R) to synthesize T7 RNAP gene by PCR.
2015.7.8
Transformation again!
2015.7.9
As usual, pinking the positive colonies, PCR and sequencing. We hope everything goes smoothly.
2015.7.11
According to the result of sequencing, there was no sample 100 percent matching. We did transformation again.
2015.7.13
The second sequencing was completely failed because the sequencing company could not get any plasmid. We thought maybe the replication of plasmid in competent cell(DH5α) was inefficient, so we changed another one,T1,bought from Transgene company.
2015.7.14
This time we sent plasmid rather than bacteria. Although the concentration of the plasmid was a little bit lower, it was enough to do sequencing.
2015.7.15
Considering the lower fidelity of the enzyme used for T7 amplification before, we changed into a better one called KD Plus and restarted to PCR, linking and transformation.
2015.7.16
The third sequencing still showed nothing right. We were a little disappointed, but we exactly knew that failure was a normal phenomenon in the lab.
2015.7.18
Eventually,we got the matching sample after using KD Plus. Immediately, we began to do double enzyme digestion(EcoR1/Xho1、EcoR1/Pst1).
2015.7.19
After gel recovery, we linked our target fragments to relative vectors with T4 ligase, 4℃, overnight.
Meanwhile, we collected data about the index of the condition of swarms again, which was used to analyze the effect of dsRdRp by Excel later.
2015.7.20
Transformation, plate coating,37℃, overnight.
2015.7.21
Pinking the positive colonies, PCR detection, and then plasmid extraction. It was smooth! So, we got T7 RNAP-pYES2 and T7 RNAP-pSB1C3 successfully. We stored them in -20℃.
2015.7.23
Today we went to observe the condition of swarms, and it was happy to see that the experimental groups turned to a healthier development comparing with control group. It meant that using dsRdRp expressed in E.coli to prevent and cure CSBV was working. Next, we would continue to collect more data until we draw the scientific conclusion.
2015.7.26
With the development of our project, we gradually have realized that even if using yeast can solve the problem of biosecurity, there are still other troubles, such as less expression of dsRdRp in yeast, easier to loss plasmids in yeast and harder to digest the cytoderm of yeast for Chinese honeybees.
So we need a better plan to deal with these to make a more practical production. That was what we would consider deeply in future work.
2015.7.27
Today we decided to constract RdRp-pSB1C3 again, because we thought T7 RNAP-pSB1C3 may be not a new part any more. So, we did linking. We decided if it is failed just like before this time. We had to change T4 into a faster ligase..
2015.7.29
As predicted, there was still nothing on the plate.(please tell us why you cannot grow out,bacteria!)
2015.8.1
We got the preliminary result of index of swarms. Next few days we would detect the expression quantity of CSBV in offspring.
2015.8.2
We made the figure of the effect of dsRdRp on the number of sac-like larvae. It showed that the number of infected larvae which were fed with HT115-dsRdRp in mediate and high concentration was remarkably less than the control group.
2015.8.5
Other two figures were made out, which showed the effect of dsRdRp on the number of sealed brood and the population of a colony respectively. They demonstrated the similar conclusion.
2015.8.8
We detected the CSBV-carried rate of offspring by RT-PCR. After AGE, it showed that the CSBV-carried rate of first and second filial generation was 30% and 20% respectively, which was obviously declined.
2015.8.12
We detected the expression quantity of CSBV in offspring by RT-qPCR. We found with the decreasing of CSBV. The expression quantity of RdRp gene declined either, which meant that dsRdRp really could inhibit the replication of CSBV.
Last final month
Even though we were succeed in using dsRdRp expressed in proeukaryotic system to prevent and cure CSBV, we still had to take care of some problem we met. So next stage we will try to constract another plasmid with suicide gene to control the concentration of engineering bacteria. Further more, we want to use CRISPR-Cas9 system to make RdRp gene inserted into the genome of E.coli. So that we can solve the problem of biosecurity.
