<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lanzhou_Result skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lanzhou/Result</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Results</TITLE><TITLE>Team:LZU 2015</TITLE></P><DIV class="nav-wrapper #42a5f5 blue accent-3"><DIV class="container"><DIV class="row text-deep-orange"><UL id="nav-mobile" class="right hide-on-med-and-down"><LI><A href="https://2016.igem.org/Team:Lanzhou">Main</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/Project">Project</A></LI><LI class="active"><A href="https://2016.igem.org/Team:Lanzhou/Result#">Results</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/Part">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/HP/Silver">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/Notes">Notes</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/Collaborations">Collaboration</A></LI></UL></DIV></DIV></DIV><DIV class="parallax-container z-index-1"><DIV><H2 class="center-align" style="color: white;margin-top: 10%;">
            Results
          </H2></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col s8 l8"><DIV class="row" id="Surface"><H4>
              GFP-MC Surface Expression
            </H4></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                An outer membrane protein INP-N was fused to the GFP-MC gene, creating an INP-GFP-MC which was designated pET23b/INP-GFP-MC (Figure 1-A). In this display system, the MC can bind mercury to iron through immobilization on the surface of cells.
              </P><P style="text-align: justify;">
                To verify the surface localization of the GFP-MC protein by cell fractionation, INP-N-GFP-MC in the membrane fraction was also demonstrated by western blotting. This result shows that the MC has been displayed on the cell surface of E. coli, using ice nucleation protein as an anchoring motif. A protein band of 43 kDa, corresponding to the correct size of INP-N-GFP-MC, was detected from cells containing recombinant plasmid (Figure 1, lane 7). 
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P>
                Figure 1: (A) INP-GFP-MC inserted plasmid Pet23b for display of pet23b/ INP-GFP-MC. (B) Western blot analysis for subcellular location of expressed INP-N–GFP-MC fusion in total cell <EM>E. coli</EM> BL21 membrane fraction was detected with anti His-tag at a 1:1,000 dilution.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                INP-N and MC do not affect GFP expression and fluorescence. Although cells carrying pET23bNot were not fluorescent (Figure 2-B), cells carrying pET23b/INP-N-GFP-MC were brightly labeled fluorescent cells; Figure 2-A displays the fluorescence emission spectra of GFP-MC. Ice nucleation protein (INP) is an extrinsic membrane protein; since the truncated version of INP only contains the N-terminal domain, which is the anchor membrane and GFP-MC, the fusion protein was displayed on the surface of E. coli, without any adverse effects on cell growth and integrity. This strategy was adopted for targeting the GFP-MC to the surface of E. coli. Plasmid pET23b, carrying the INP-GFP-MC fusion, was used for genetic immobilization of GFP-MC. 
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                Figure 2: (A) Fluorescence images of engineering E. coli cells. (B) Cells carrying pET23bNot were not fluorescent.
              </P></DIV></DIV><DIV class="row" id="Biosensor"><H4>
              Biosensor of Mercury

            </H4></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                The fluorescence emission spectra of GFP-MC with different Hg<SUP>2+</SUP> is shown in Figure 3 (a). The addition of Hg2+ (100 μM) induced a significant decline in fluorescence emission of GFP-MC, which indicated that GFP-MC can be used as a sensor for Hg2+ detection. Furthermore, the addition of divalent sulfur (S2-) could completely restore the fluorescence of GFP-MC, implying that the interaction between GFP-MC and Hg<SUP>2+</SUP> is reversible (Figure 3-C).Similarly, there is a similar result was obtained for ethylenediaminetetraacetic acid (Figure 3-D).
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P>
                Figure 3: (A) Fluorescence spectra of IC (OD600 = 1.3), in the presence of Hg<SUP>2+</SUP>, at varying concentrations. (B) Fluorescence responses of IC to Hg<SUP>2+</SUP> at varying concentrations. (C) Fluorescence titration of IC (OD = 1.3) and Cu<SUP>2+</SUP> (50 µM) following the addition of different concentrations of S2- (0-50 µM). Excitation wavelength: 488 nm (D) Fluorescence titration of IC (OD) and Cu<SUP>2+</SUP> (50 µM) following the addition of different concentrations of EDTA (0-20 µM). Excitation wavelength: 488 nm.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                After MC binding of Hg<SUP>2+</SUP> on the surface, the fluorescence emission of GFP-MC is quenched. With the increase in the concentration of Hg<SUP>2+</SUP> from 0 to 100 μM, fluorescence of GFP-MC decreased continually. Fluorescence was almost completely quenched by 100 μM Hg<SUP>2+</SUP> (Fig. 3-A). This indicated that the MC can bind mercury effectively. From the Hg<SUP>2+</SUP> concentration-dependent calibration curve, a good linear relationship (R2 = 0.9803) in the concentrations ranging from 0 to 10 μM was obtained (Fig. 3-B), and the lowest detection limit was less than 1 µM. This whole cell was rapid and sensitive to Hg<SUP>2+</SUP> detection by a quick and high-efficiency fluorescence quench.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                To examine the specificity of the biosensor, the fluorescence responses of GFP-MC toward various metal ions, including Cd<SUP>2+</SUP>, Pb<SUP>2+</SUP>, Ba<SUP>2+</SUP>, Mo<SUP>6+</SUP>, Ni<SUP>2+</SUP>, Hg<SUP>2+</SUP>, Mn<SUP>2+</SUP>, Ca<SUP>2+</SUP>, Ag<SUP>+</SUP>, Zn<SUP>2+</SUP>, and Cu<SUP>2+</SUP>, were studied under the same conditions. The final concentration of the ions was 50 μM. It could be seen that only Hg<SUP>2+</SUP> and Cu<SUP>2+</SUP> caused dramatic fluorescence quenching, and there were no obvious fluorescence changes for the other metal ions (Figure 4-A ). Comparatively, the fluorescence quenching induced by Hg<SUP>2+</SUP> is weaker than Cu<SUP>2+</SUP>. Moreover, the coexistence of other metal ions showed no obvious interference in Hg<SUP>2+</SUP> assays (Figure 4-B). Hg<SUP>2+</SUP> and Cu<SUP>2+</SUP> show remarkable fluorescence quenching effects. Cu<SUP>2+</SUP> showed similar results. The calibration curve of the concentration dependence of Cu<SUP>2+</SUP> showed a good linear relationship (R2 = 0.9846) in the concentration ranging from 0 to 20 uM (Figure 5 A-B). However, it can be seen that only Cd<SUP>2+</SUP> increased the fluorescence of GFP-MC (Figure 6-A). From the Cd<SUP>2+</SUP> concentration-dependent calibration curve, a good linear relationship (R2 = 0.9977) in concentrations ranging from 0 to 5 μM was obtained (Fig. 6-B). Thus, GFP-MC can be used for the detection of lower concentrations of Cd2+. 
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P>
                Figure 4:(A) Selectivity of the IC sensing system. (B) Fluorescence response of IC (OD600 = 1.3) in the presence of Cu<SUP>2+</SUP> and various additional metal ions. The black bars represent the addition of an excess of the appropriate metal ion (50 µM) to a bacterial solution. The red bars represent the subsequent addition of 100 µM CuSO4 to the solution. Excitation wavelength: 488 nm. 
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P>
                Figure 5:(A) Fluorescence spectra of IC (OD600 = 1.3) in the presence of Cu<SUP>2+</SUP> with varying concentrations. (B) Fluorescence responses of IC to Cu<SUP>2+</SUP> with varying concentrations.  
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P>
                Figure 6:(A) Fluorescence spectra of IC (OD600 = 1.3) in the presence of Cd<SUP>2+</SUP> with varying concentrations. (B) Fluorescence responses of IC to Cd<SUP>2+</SUP> with varying concentrations.   
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                The addition of Hg<SUP>2+</SUP> (100 μM) induced a significant decline in the fluorescence emission of GFP-MC, which indicated that GFP-MC can be used as a sensor for Hg<SUP>2+</SUP> detection. For peptides, biosynthesizing a novel gene is much more environmental friendly, and it is less expensive than the chemical synthesis methods. In addition, engineering bacterial strains can be used as a new detection tool because they can copy continually, and are sustainable and simple to use.
              </P></DIV></DIV><DIV class="row" id="Whole"><H4>
             Whole-cell Binding of Hg<SUP>2+</SUP></H4></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                The engineered E. coli are grown in LB culture medium containing 12.5 mg/l mercury ion. Bacteria with the surface-displayed GFP-MC are able to adsorb mercury ions up to 40 mg/g cells, which is 10-fold higher than control cells (Figure 7-A). As shown in Figure 7-B, the rates of effective adsorption of GFP-MC displayed cells in the mercury ions concentration of 12.5 mg/L ranged from 24.09% to 55.72%. Furthermore, these engineered E. coli cells selectively adsorbed mercury ions from LB medium containing various heavy metals separately or mixed together. 
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P>
                Figure 7: (A) The adsorption of Hg<SUP>2+</SUP> by E. coli harboring the recombinant plasmid. (B) The effective adsorption rate of Hg<SUP>2+</SUP> by engineered E. coli.  
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                The IC can bind to mercury, which is consistent with (Glu-Cys) Gly articles. They have a strong mercury adsorption capacity. The maximum absorption capacity is up to 200 uM/g, which is close to the 230 uM/g reported in the literature. Compared with MT displayed on the cell surface, the binding efficiency of IC is higher than MT. The adsorption ability of IC is stronger than merR anchored on cell surface. Interestingly, copper, zinc, cadmium, and other ion coexistence did not affect IC adsorption of mercury. However, IC cannot bind Cd<SUP>2+</SUP>, but four peptides can bind one cadmium ion by chemical bonds (Wang, 2015). 
              </P></DIV></DIV><DIV class="row" id="Transmission"><H4>
              TEM Analysis
            </H4></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                To examine the morphology of the mercury adsorbed on the GFP-MC-displayed cells, the engineered <EM>E. coli</EM> bacteria cells were incubated with 12.5 mg/l Hg<SUP>2+</SUP> overnight. Surface adsorption was analyzed by TEM. As shown in Figure 8 (A) the GFP-MC was on the cell surface and was approximately 500 nm in size. A randomly chosen area of a cell surface confirmed the elemental composition. There were black spots on the outer membrane of the engineered E. coli (Figure 8-B), but not on control E. coli BL21 cells (Figure 8-D). An energy-dispersive X-ray spectroscopy system was used to verify the cells and it verifies that the black spots were the MC accumulated Hg<SUP>2+</SUP> (Figure 8-C). The results showed that the mercury ions were adsorbed on GFP-MC protein-displayed cells by surface display system.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                Figure 8: (A)-(B) The TEM images of E. coli BL21/INP-GFP-MC cells. Illustration of the mercury-specific metal catcher displayed on E. coli cell surface via the Inp-N. (B) Standing for the adsorbed mercury ions. (C) EDXA measurement of the box from (D) The control TEM images of E. coli BL21.
              </P></DIV></DIV><DIV class="row" id="Reduce"><H4>
              Metal Catcher in Fish
            </H4></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                It is important to investigate whether the surface-displayed E. coli is more potent for detoxification of mercury than common bacteria. The concentration of mercury in the fish of Control group 1 was 0.49 µg/g, while that in control group 2 was 1.664 µg/g and in the experimental group, it was 0.623 µg/g. Mercury content of Control 2 was 3-fold higher than Control 1. As shown in Figure 9, experimental groups fed with engineered E. coli BL21 containing IC had significantly lower mercury, close to the levels in Control 1. Mercury content in the Control 1 fish decreased by 62.5% compared to Control 2. Additionally, Table 1 indicates that the presence of engineered E. coli did not interfere with fish growth. In comparison, engineered E. coli significantly protected fish from mercury toxicity. This strategy could protect humans from mercury poisoning.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                The results show that the MC exhibits its mercury binding characteristics when it is displayed on the surface. Additionally, the presence of engineered E. coli did not affect fish growth. Fish growth was significantly influenced by the presence of 0.1 mg/l Hg2+. In comparison, engineered E. coli significantly protected fish from mercury toxicity. We believe that this strategy could protect humans from mercury poisoning. In conclusion, it is important that the fodder containing the engineered strain does not affect normal growth when fed to fish. The engineered strain can reduce the fish intestinal mercury by adsorption. . Thus, the engineered strain can be used as a feed additive agent to alleviate heavy metal contamination of fish.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P class="center-align">
                Reduced bioaccumulated heavy metals in Cyprinus carpio by gut remediation.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P>
                Table 1 Effect of administration of engineering E. coli growth performance
              </P></DIV></DIV><DIV class="row" id="Diversity"><H4>
              Diversity Analysis
            </H4></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                All sequence reads were trimmed and assigned to samples based on their barcodes. Sequences of high quality (length &gt; 150 bp, without the ambiguous base ‘N’, and an average base quality score &gt; 30) were used for downstream analysis. Sequences were clustered into operational taxonomic units (OTUs) at a 97% identity threshold. A total of 15,480 reads per sample and 1,306 OTUs were obtained from the 6 samples through Illumina Miseq system for sequencing. The rarefaction curves tended to approach the saturation plateau (Figure 10-A). Good’s coverage estimations revealed that 94% to 98% of the species were obtained in all of the samples.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                Figure 10: (A) Rarefaction analysis of the different fish. Rarefaction curves of OTUs clustered at 97% sequence identity across different environmental samples. (B) Sample Sorting analysis. Scatter plot of PCoA-score depicting variance of fingerprints derived from different fish community. Principal coordinate axes PCoA1 and PCoA2 explained 64.02% and 24.29% of the variance, respectively.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                Thirteen different phyla were identified from the 6 samples. The 6 libraries showed very dissimilar 16S rRNA profiles, even in phylum level distributions (Figure 11). LMR9 and LMR10 libraries, which included <EM>Proteobacteria</EM>, <EM>Firmicutes</EM>, <EM>Fusobacteria</EM>, and <EM>Bacteroidetes</EM>, were the most important groups and accounted for 85.13% of the reads. For C. carp composition, <EM>Bacteroidetes</EM> of LMR10 was higher than control LMR9, while <EM>Fusobacteria</EM> was lower. LMR11-14 libraries were dominated by <EM>Proteobacteria</EM>, <EM>Fusobacteria</EM>, <EM>Bacteroidetes</EM>, and <EM>Tenericutes</EM> phyla and represented over 98% of the reads. Compared to Control 1, <EM>Bacteroidetes</EM> in Control 2 increased, while <EM>Fusobacteria</EM> and Tenericutes decreased and Bacteroidetes was unchanged. In the experimental group, <EM>Proteobacteria</EM> and <EM>Bacteroidetes</EM> were increased, compared to Control 2, while <EM>Fusobacteria</EM> and <EM>Tenericutes</EM> decreased.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P class="center-align">
                Bacterial composition of the different communities. Relative read abundance of different bacterial phyla within the different communities.
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P style="text-align: justify;">
                We calculated the alpha-diversity (phylogenetic distance whole tree, chao1 estimator of richness, observed species, and Shannon’s diversity index) and beta-diversity (PCoA, UniFrac) analyses (Table 2). The principal coordinate analysis（PcoA）score plot revealed the bacterial communities of the samples. LMR13 and LMR14 samples grouped to the right of the graph along PCoA1, accounted for 64.02% of the total variations. This indicates that these two groups had no significant difference in the microflora composition. The LMR11 samples were closely related to the LMR12 samples, whereas the LMR9 and LMR10 samples were distant from the other samples along PCoA2, which represented 24.29% of the total variations (Figure 10-B). However, there were significant differences between control groups and experimental groups. Overall, the two PCoA axes explained 88.31% of the variation between the different communities. 
              </P></DIV></DIV><DIV class="row"><DIV class="col s12 l12"><P>
                Table 2 Diversity indexes and Operational taxonomic units (OTUs) are defined at 97% sequence similarity
              </P></DIV></DIV></DIV><DIV class="col s4 l4"><DIV class="col hide-on-small-only m3 l2" id="scoll"><UL class="section table-of-contents"><LI><A href="#Surface">GFP-MC Surface Expression</A></LI><LI><A href="#Biosensor">Biosensor of Mercury</A></LI><LI><A href="#Whole">Whole-cell Binding of Hg<SUP>2+</SUP></A></LI><LI><A href="#Transmission">TEM Analysis</A></LI><LI><A href="#Reduce">Metal Catcher in Fish</A></LI><LI><A href="#Diversity"> Diversity Analysis</A></LI></UL></DIV></DIV></DIV></DIV><FOOTER class="page-footer blue darken-1" style="margin: 0px;"><DIV class="container"><DIV class="row" style="margin: 0px;"><DIV class="col l6 s12"><H5 class="white-text">Lanzhou 2016</H5><P class="grey-text text-lighten-4">NO.222, TianShui South Road, Lanzhou, China, 730000</P><H6 class="white-text">Copyright 2015 Lanzhou University</H6></DIV><DIV class="col l3 s12"><H5 class="white-text">Acknowledgments</H5><UL><LI><A class="white-text" href="http://en.lzu.edu.cn/">Educational Administration Office, LZU</A></LI><LI><A class="white-text" href="http://chc.lzu.edu.cn/en/">Cuiying Honors College, LZU</A></LI><LI><A class="white-text" href="http://lifesc.lzu.edu.cn/">School of Life Sciences,LZU</A></LI></UL></DIV></DIV></DIV><DIV class="footer-copyright blue darken-3"><DIV class="container">
              Powered by <A class="brown-text text-lighten-3" href="http://materializecss.com">Materialize</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>