Team:Lanzhou/Project
Project
Main
Project
Results
Parts
Human Practices
Attributions
Safety
Notes
Collaborations
Project
Background
A Problem We Are Facing Right Now
Heavy metal contamination in the marine environment is becoming an increasingly serious problem throughout the world. There was a shocking news published by the Xinhua News Agency On August 18, 2016 about the execrable current contamination situation of the Bohai Sea in China. It revealed that the Bohai sea has almost become a dead sea and bears about 2.8 billion tons of sewage per year. And this calamity has a great passive impact on the fishing industry. People will never realize what consequences the environmental contaminations will brought to life unless they encroach our beautiful world piece by piece till the end.
(A) The Kuanhe Pump Station located in the upstream of Tianjin Distributary which will flow into the Bohai Sea is discharging sewage.
(B) There was once scads of dead fish estuary of Tianjin River, 2016.
(C) A mother brought her children to mourn the dead after the 50th anniversary of the confirmation of minamata disease in Japan.
(D) The level of mercury in the fish exceeds standard. They live in an area that was once under the minamata disease.
Heavy metal is one of the main sources of pollution. They may be a natural part on the earth but they became toxic due to so many anthropogenic activities. Heavy metals are stable and can't be broken down, which makes it easy for them to accumulate in the environment. Because of the high affinity for organic tissues, heavy metals will be bio-accumulated and biomagnified through food chains and became a threat both to ecosystem and public health.
Introduction
Heavy metal pollution in marine and coastal areas is mainly derived from oil pollution, desalination plants, industrial effluents, and sewage discharges. In particular, mercury pollution in fish is even more serious. Fish is a vital food for humans and many animals. As a main source of protein, it is now considered to be an important dietary threat of heavy metal toxicity via consumption. Mercury in fish is easily converted to methyl mercury (MeHg), which is highly toxic and can cause diseases. For example, Minamata disease caused by MeHg poisoning affects people who have ingested shellfish and fish contaminated by MeHg discharged in water. Therefore, it is essential to address the problem of mercury contamination in fish.
Common Solutions
s there nothing to do about it? The answer is definitely yes. Recently, several methods such as biosorption, ecoremedations, bioremediation, and phytoremediation have been utilized in attempts to reduce heavy metal pollution from contaminated environments. Although these approaches are used in small scale sites such as waste water plants, they are incapable of effective action in a vast marine environment. Therefore, a new strategy to address this problem needs to be developed.
An important strategy for managing pollution is finding the most effective method and technologies for heavy metal treatment. There are some techniques and methods for eliminating or reducing heavy metal pollution in the environment. However, some studies have reported levels of heavy metals in fish greater than the permissible WHO limit. Moreover, research on intestinal microorganisms is currently receiving a great deal of attention. Gut microbiota such as E. coli play an important role in maintaining health. . An effective method to reduce accumulation of heavy metals is the use of E. coli to break down the heavy metal; currently, there are few reports on this method.
A New Strategy-Gut Remediation
It has been reported that chemically synthesized peptides produced by the Tang Yu research group of Lanzhou University could bind to heavy metals; these peptides possess affinity for binding heavy metals. Thus, we attempted to use bacteria to synthesize such peptides anchored on the cell membrane. Importantly, the surface display strategy for heavy metal adsorption has an advantage over traditional bioadsorption methods due to the burden of intracellular accumulation of toxics metal ions.
Advantage
Compared with chemical synthesis, bacterial synthetic peptides are cheaper and environmental friendly. Herein, we present a new method for the selective removal of mercury by peptides, coded by a novel gene displayed on the cells, which will be able to contribute to the future. The resulting engineered E. coli, with high mercury affinity and selectivity, bind to mercury. Engineering strains of adsorption heavy metals can reduce fish intestinal mercury levels through their feed. Thus, engineering strains indirectly reduces mercury contamination in fish.
Metal Catcher
In this year, we are inspired by this idea and we designed a mercury-specific binding peptide, IC (Cys- lys- Cys- lys- Cys- lys- Cys), termed metal catcher (MC), which has a high affinity and selectivity toward mercury. The subsequent IC and GFP (Green Fluorescent Protein) were fused and displayed on Escherichia coli cell surface by using an N-terminal region ice nucleation protein (INP) anchor.
Assembling and Basic Components
We proposed a new approach named gut remediation to reduce bio-accumulated heavy metals in fish by gut microbiota. We designed polypeptides which can bind to heavy metals such as divalent mercury and copper, especially divalent mercury. Then a novel gene named Metal Catcher was designed based on the polypeptide with GFP (Green Fluorescent Protein) and N-terminal region of INP ( Ice Nucleation Protein).
There are three crucial components, INP, GFP, and Metal Catcher. INP (Ice Nucleation Protein) is an extrinsic membrane protein of E.coli and acts as an anchor. GFP acts as a reporter while metal catcher can bind heavy-metal ions. This is how we assemble the crucial elements.
The structure of INP
There are three crucial components, INP, GFP, and Metal Catcher. INP (Ice Nucleation Protein) is an extrinsic membrane protein of E.coli and acts as an anchor. GFP acts as a reporter while metal catcher can bind heavy-metal ions.
The structure of fusion protein
Experiments
Construction of the Surface Display System
The peptide sequence of the metal catcher (MC) was referred to from the literature. The peptide was named IC (Cys- lys- Cys- lys- Cys- lys- Cys). The gene sequence, encoding IC sequences, was reversely translated from the amino acid sequence of IC sequences and optimized for E. coli codon usage. The N-terminal region of the ice nucleation protein (INP-N) gene was synthesized and inserted in plasmid pUC19 by the Beijing Genomics Institute (BGI). The GFP gene was provided by our laboratory. The INP-N-GFP-MC fusion was constructed as follows. the INP-N fragment was PCR amplified from plasmid pUC19 using the primers INP-F1 and INP-R. The GFP-MC fragment was amplified with a reverse primer GFP-CR3, including gene sequence encoding IC sequences and GFP-F. INP-N-GFP-MC was fused by overlap PCR. The amplified fragment (INP-N-GFP-MC fusion) was digested with Ecor I and Xho I, and cloned into pET23b. Cultures were grown in Luria-Bertani (LB) medium with 100 mg ampicillin per L at 37°C. Finally, the plasmid was transformed into E. coli DH5α. After confirmation by sequencing, the reconstructed plasmid pET23b/INP-N-GFP-MC was transformed into E. coli BL21 DE3. pET23b plasmid without any exogenous gene was used as the control.
Mercury Selectivity Detection with the E. coli GFP-MC Sensor
For mercury detection, the E. coli BL21 DE3 containing the recombinant plasmid was cultured in LB medium until OD600 = 1.3. All cells were harvested and washed twice with 0.85% NaCl. Finally, cells were re-suspended in 0.85% NaCl. Solutions of Hg2+ and Cu2+ (1–10 μL) of different concentrations were mixed with the bacteria. The reaction system was added to a final volume of 1 mL and incubated at room temperature for 5 min. For fluorescence determination, 300 μL liquid from each sample was tested using a fluorescence spectrophotometer (PerkinElmer, USA). The excitation wavelength was 488 nm, and the emission wavelengths were in the range of 500 to 600 nm. Emission and excitation of the slit was 5 nm.
SDS-PAGE and Western Blot Analysis of the Displayed Fusion Protein
The cells were grown in LB medium containing ampicillin (100 mg/L) with overnight shaking at 37°C. After 1/100 dilution in LB medium, the cultures were grown until OD600 = 1.3. E. coli BL21 cells expressing INP-N-GFP-MC fusion were harvested, according to the cell fractionation method described by Yang et al. Samples of total cell lysate, soluble fraction, and membrane fraction were mixed with a loading buffer and boiled for 10 min. Samples were analyzed on SDS-PAGE using 12% (w/v) acrylamide and transferred to a polyvinylidene difluoride membrane. Pre-stained molecular weight markers were used to estimate protein molecular weights. Western blot analysis was performed according to the method described by Wei et al. The first antibody was His-tag and the second antibody was goat anti mouse.
Bioadsorption of mercury by GFP-MC Displayed Cells
For mercury ion adsorption, the E. coli BL21 DE3 containing the recombination plasmid was cultured in LB medium until the OD600 = 1.3. Then, an appropriate concentration of mercury ions (0, 6.5, 12.5, 25 mg/L) was added to LB medium. To measure the metal ion adsorption ability of GFP-MC displayed on E. coli, the cells were harvested from LB medium by centrifugation (5000 rpm, 3 min) and then washed twice with saline. For mercury analysis, cells were dried and digested with 70% nitric acid. Total mercury was analyzed by atomic absorption spectrophotometry (analytikjena, Germany). To measure the selectivity of the E. coli to adsorb mercury ions, 6.5 mg/l Hg2+ and Cd2+, and 12.5 mg/l Cu2+ and Zn2+ were added in place of the mercury. After the treatment, the metal ion content in the samples was measured by AAS.
TEM-EDX Analysis of GFP-MC-Displayed Cells
After mercury ion adsorption, the GFP-MC-displayed cells were washed twice with saline and dispersed in ddH2O. The sample treatment method used was adapted from Wei et al. Surface adsorption was analyzed by TEM (TECNAI, USA). Elemental analysis was performed with the energy-dispersive X-ray spectroscopy system attached to the transmission electron microscope. All transmission electron microscopy images were recorded.
Reduced Bioaccumulated Heavy Metals in Fish by Gut Remediation
A commercial fish feed containing crude protein (Min 30%), crude fat (Min 20%), crude fiber (Min 4%), moisture (Max 5%), and algae (Max 25%) was used for feeding the experimental fishes. Engineered E. coli was mixed with the basal dry feed at a level of 1 × 109 cfu/g feed (Abraham et al., 2008). After mixing with strains, the feeds were air dried at 40°C in an oven to half-dry and placed in a plastic bag at 4°C . Cyprinus carpio of size ranging from 7.483–8.348 g weight and 5.75–6.52 cm length, were introduced into each of the 8 fish tanks of 20-L capacity, which contained 6 L tap water. Each tank contained ten fish. The fish in control tank 1, containing tap water, were fed normal feed. Control tank 2 contained tap water consisting of 0.1 mg Hg2+ and the fish were fed normal feed. The fish in the experimental tanks were fed feed containing a mixture of engineered E. coli and filled with tap water containing 0.1 mg Hg2+. The experiment was carried out for 30 days in duplicates and the fish were fed twice daily. Approximately 50% of the water was exchanged every day. Mortality, external signs, and growth conditions of the fish were recorded daily (Abraham et al., 2008). Detection of mercury in fish was carried out according to “Determination of total mercury and organic-mercury in foods” (GB/T 5009.17-2003).
Bacterial community composition determined by sequencing
After 30 days of feeding, sampled fish were dissected using sterile scissors. Samples of intestines were examined utilizing methods described by Shangong Wu. DNA was extracted using TIANamp Stool DNA kit (TIANGENBIOTECH，BEIJING). Universal primer 515F and 909R with 12 nt unique barcode was used to amplify the V4 hypervariable region of 16S rRNA gene for pyrosequencing using Miseq sequencer. The purified library was diluted, denatured, re-diluted, and mixed with PhiX (equal to 30% of final DNA amount) in accordance with the Illumina library preparation protocols, and then applied to an Illumina Miseq system for sequencing with the Reagent Kit v2 2 × 250 bp according to the manufacturer’s manual. The sequence data were processed using QIIME Pipeline–Version 1.7.0 (http://qiime.org/).
Achievements
Background
Introduction
Common Solutions
A New Strategy-Gut Remediation
Assembling and Basic Components
Experiments
Achievements
Lanzhou 2016
NO.222, TianShui South Road, Lanzhou, China, 730000
Copyright 2015 Lanzhou University
Acknowledgments
Educational Administration Office, LZU
Cuiying Honors College, LZU
School of Life Sciences,LZU
Powered by Materialize
