<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Lanzhou_Notes skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Lanzhou/Notes</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Notes</TITLE><TITLE>Team:LZU 2015</TITLE></P><DIV class="nav-wrapper  blue accent-3"><DIV class="container"><DIV class="row text-deep-orange"><UL id="nav-mobile" class="right hide-on-med-and-down"><LI><A href="https://2016.igem.org/Team:Lanzhou">Main</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/Result#">Results</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/Part">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/HP/Silver">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/safety">Safety</A></LI><LI class="active"><A href="https://2016.igem.org/Team:Lanzhou/Notes">Notes</A></LI><LI><A href="https://2016.igem.org/Team:Lanzhou/Collaborations">Collaboration</A></LI></UL></DIV></DIV></DIV><DIV class="parallax-container"><DIV><H2 class="center-align" style="color: white;margin-top: 10%">Notes</H2></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col s3 l3"><DIV class=" col hide-on-small-only" id="scoll"><UL class="section table-of-contents"><LI><A href="#1">Week 1</A></LI><LI><A href="#2">Week 2</A></LI><LI><A href="#3">Week 3</A></LI><LI><A href="#4">Week 4</A></LI><LI><A href="#5">Week 5</A></LI><LI><A href="#6">Week 6</A></LI><LI><A href="#7">Week 7</A></LI><LI><A href="#8">Week 8</A></LI><LI><A href="#9">Week 9</A></LI><LI><A href="#10">Week 10</A></LI><LI><A href="#11">Week 11</A></LI><LI><A href="#12">Week 12</A></LI><LI><A href="#13">Week 13</A></LI><LI><A href="#14">Week 14</A></LI><LI><A href="#15">Week 15-16</A></LI><LI><A href="#17">Week 17-18-19</A></LI><LI><A href="#20">Week 20-21-22</A></LI><LI><A href="#22">Week 22-23-24</A></LI><LI><A href="#25">Week 25</A></LI></UL></DIV></DIV><DIV class="col s9 l9"><DIV class="row"><DIV class="section" id="1"><H5>Week 1 (Mar 28-Apr 4)</H5><P style="text-align: justify;">We got the N-terminal region of the ice nucleation protein (INP-N) gene, inserted in plasmid pUC19 by the Beijing Genomics Institute (BGI) and amplified the INP-N fragment from plasmid pUC19 by PCR, using the primers INP-F1 and INP-R.
              </P><P style="text-align: justify;">
              We got GFP gene from the template offered by our laboratory by PCR.
              </P><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">PCR system(20μL)</TH></TR></THEAD><TBODY><TR><TD>5×ps</TD><TD>2</TD></TR><TR><TD>dNTP</TD><TD>1</TD></TR><TR><TD>GFP-F</TD><TD>1</TD></TR><TR><TD>H-GFPR</TD><TD>0.25</TD></TR><TR><TD>template DNA</TD><TD>0.25</TD></TR><TR><TD>Taq E</TD><TD>0.75</TD></TR><TR><TD>ddH2O</TD><TD>14.75</TD></TR></TBODY></TABLE><P class="center-align">
                These work was done by Ning Chai, Xinyu Zhou, Jindian He and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="2"><H5>Week 2 (Apr5-Apr12)</H5><P style="text-align: justify;">We started to construct parts BBa_K1995003: HG (GFP-HG), BBa_K1995006: 2C (GFP-C) and BBa_K1995009: 2H (GFP-H).</P><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">PCR system(20μL)</TH></TR></THEAD><TBODY><TR><TD>5×ps</TD><TD>10</TD></TR><TR><TD>dNTP</TD><TD>4</TD></TR><TR><TD>GFP-F</TD><TD>1</TD></TR><TR><TD>H-GFPR</TD><TD>1</TD></TR><TR><TD>template DNA</TD><TD>1</TD></TR><TR><TD>Taq E</TD><TD>0.5</TD></TR><TR><TD>ddH2O</TD><TD>32.5</TD></TR></TBODY></TABLE><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">PCR system(20μL)</TH></TR></THEAD><TBODY><TR><TD>5×ps</TD><TD>10</TD></TR><TR><TD>dNTP</TD><TD>4</TD></TR><TR><TD>GFP-F</TD><TD>1</TD></TR><TR><TD>C-GFPR</TD><TD>1</TD></TR><TR><TD>template DNA</TD><TD>1</TD></TR><TR><TD>Taq E</TD><TD>0.5</TD></TR><TR><TD>ddH2O</TD><TD>32.5</TD></TR></TBODY></TABLE><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">PCR system(20μL)</TH></TR></THEAD><TBODY><TR><TD>5×ps</TD><TD>10</TD></TR><TR><TD>dNTP</TD><TD>4</TD></TR><TR><TD>GFP-F</TD><TD>1</TD></TR><TR><TD>H-GFPR</TD><TD>1</TD></TR><TR><TD>template DNA</TD><TD>1</TD></TR><TR><TD>Taq E</TD><TD>0.5</TD></TR><TR><TD>ddH2O</TD><TD>32.5</TD></TR></TBODY></TABLE><P style="text-align: justify;">
                After digestion by Ecor I and Not I, we measured the concentration of the amplified fragments.
              </P><P>
                Digest system:
              </P><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">Digest system(50μL)</TH></TR></THEAD><TBODY><TR><TD>EcoR I</TD><TD>2.5</TD></TR><TR><TD>Not I</TD><TD>2.5</TD></TR><TR><TD>10×H buffer</TD><TD>5</TD></TR><TR><TD>BSA</TD><TD>5</TD></TR><TR><TD>Triton X-100</TD><TD>5</TD></TR><TR><TD>DNA</TD><TD>25</TD></TR><TR><TD>ddH2O</TD><TD>5</TD></TR></TBODY></TABLE><P>
                These work was done by Ning Chai, Xinyu Zhou, Jindian He and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="3"><H5>Week 3(Apr13-Apr20)</H5><P style="text-align: justify;">Due to the unsatisfied result, we repeated the work of digestion and finally got the ideal result.We linked BBa_K1995003: HG (GFP-HG), BBa_K1995006: 2C (GFP-C) and BBa_K1995009: 2H (GFP-H) to the vector.
              </P><P style="text-align: justify;">
              Lingation:
              </P><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">Lingation system(10μL)</TH></TR></THEAD><TBODY><TR><TD>vector</TD><TD>2</TD></TR><TR><TD>insert</TD><TD>6</TD></TR><TR><TD>T4 buffer</TD><TD>1</TD></TR><TR><TD>ligase</TD><TD>1</TD></TR></TBODY></TABLE><P>
                These work was done by Zhichen Lin, Xinyu Zhou, Jindian He and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="4"><H5>Week 4(Apr21-Apr28)</H5><P style="text-align: justify;">We started to construct BBa_K1995002: PHG (INP-N-HG), BBa_K1995007: PC (INP-N-C) and BBa_K1995010: PH (INP-N-H). 
              </P><P style="text-align: justify;">
              PCR system:
              </P><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">PCR system (50μL)</TH></TR></THEAD><TBODY><TR><TD>5×ps</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>4</TD></TR><TR><TD>HG-GFPR</TD><TD>1</TD></TR><TR><TD>INP-F1</TD><TD>1</TD></TR><TR><TD>HG</TD><TD>1</TD></TR><TR><TD>INPM</TD><TD>1</TD></TR><TR><TD>taq E</TD><TD>0.5</TD></TR><TR><TD>ddH2O</TD><TD>31.5</TD></TR></TBODY></TABLE><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">PCR system (50μL)</TH></TR></THEAD><TBODY><TR><TD>5×ps</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>4</TD></TR><TR><TD>C-GFPR</TD><TD>1</TD></TR><TR><TD>INP-F1</TD><TD>1</TD></TR><TR><TD>C/td&gt;
                    </TD><TD>1</TD></TR><TR><TD>INPM</TD><TD>1</TD></TR><TR><TD>taq E</TD><TD>0.5</TD></TR><TR><TD>ddH2O</TD><TD>31.5</TD></TR></TBODY></TABLE><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">PCR system (50μL)</TH></TR></THEAD><TBODY><TR><TD>5×ps</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>4</TD></TR><TR><TD>HG-GFPR</TD><TD>2</TD></TR><TR><TD>INP-F1</TD><TD>2</TD></TR><TR><TD>H</TD><TD>2</TD></TR><TR><TD>INPM</TD><TD>2</TD></TR><TR><TD>taq E</TD><TD>0.5</TD></TR><TR><TD>ddH2O</TD><TD>27.5</TD></TR></TBODY></TABLE><P style="text-align: justify;">
                Then the amplified fragments were measured by agarose gel electrophoresis and ligated into vector. 
              </P><P>These work was done by Ning Chai, Xinyu Zhou, Jindian He and Qi Li.</P></DIV></DIV><DIV class="row"><DIV class="section" id="5"><H5>Week 5(Apr28-May5)</H5><P style="text-align: justify;">We successfully constructed BBa_K1995001: IHG (INP-N-GFP-HG), BBa_K1995005: IC (INP-N-GFP-C) and BBa_K1995011: IH (INP-N-GFP-H) by overlap PCR.We digested the amplified fragment with Ecor I and Not I, and cloned them into pET23b.Cultures were grown in Luria-Bertani (LB) medium with 100 mg ampicillin per L at 37°C.After transformation, extraction of plasmid, digestion and measurement by agarose gel electrophoresis, we selected the expected strain. 
              </P><P>
                These work was done by Ning Chai, Xinyu Zhou, Jindian He and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="6"><H5>Week 6(May6-May13)</H5><P style="text-align: justify;">We confirmed BBa_K1995002: PHG (INP-N-HG), BBa_K1995007: PC (INP-N-C), BBa_K1995010: PH (INP-N-H), BBa_K1995001: IHG (INP-N-GFP-HG), BBa_K1995005: IC (INP-N-GFP-C) and BBa_K1995011: IH (INP-N-GFP-H) by sequencing.Then we transformed reconstructed plasmid into <EM>E. coli</EM> BL21 DE3, using pET23b plasmid without any exogenous gene as control. The cells were grown in LB medium containing ampicillin (100mg/L) with overnight shaking at 37°C. 
              </P><P>
                These work was done by Ning Chai, Xinyu Zhou, Jindian He and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="7"><H5>Week 7(May14-May21)</H5><P style="text-align: justify;">We cultured the <EM>E. coli</EM> BL21 DE3 containing BBa_K1995003: HG (GFP-HG), BBa_K1995006: 2C (GFP-C) and BBa_K1995009: 2H (GFP-H) until OD600=1.3 and harvested and washed them twice with 0.85% NaCl and re-suspended them in 0.85% NaCl. Then we mixed the bacteria with solutions of Hg<SUP>2+</SUP> and Cu<SUP>2+</SUP>/Cd<SUP>2+</SUP>(1–10 μL) of different concentrations and incubated them at room temperature. We used a fluorescence spectrophotometer to detect the fluorescence.Fluorescence spectra of 2H (OD600=1.3) in the presence of Cu<SUP>2+</SUP> with varying concentrations:
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of 2H (OD600=1.3) in the presence of Hg<SUP>2+</SUP> with varying concentrations:
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of 2C (OD600=1.3) in the presence of Hg<SUP>2+</SUP> with varying concentrations:
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of 2C (OD600=1.3) in the presence of Cd<SUP>2+</SUP> with varying concentrations.
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of HG (OD600=1.3) in the presence of Cu<SUP>2+</SUP> with varying concentrations:
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of HG (OD600=1.3) in the presence of Hg<SUP>2+</SUP> with varying concentrations:
              </P><P>
                These work was done by Ning Chai, Xinyu Zhou, Jindian He and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="8"><H5>Week 8(May22-May29)</H5><P style="text-align: justify;">We successfully constructed BBa_K1995001: IHG (INP-N-GFP-HG), BBa_K1995005: IC (INP-N-GFP-C) and BBa_K1995011: IH (INP-N-GFP-H) by overlap PCR.We digested the amplified fragment with Ecor I and Not I, and cloned them into pET23b.Cultures were grown in Luria-Bertani (LB) medium with 100 mg ampicillin per L at 37°C.After transformation, extraction of plasmid, digestion and measurement by agarose gel electrophoresis, we selected the expected strain. 
              </P><P style="text-align: justify;">
                We saw the fluorescence images of engineering <EM>E.coli</EM> cells and the control group under  fluorescence microscopy to confirm.
              </P><P style="margin-bottom: -5px;">
                Fluorescence images of engineering <EM>E.coli</EM> cells:
              </P><P style="margin-bottom: -5px;">
                Cells carrying pET23b were not fluorescent:
              </P><P>
                We also finished our human practice at Lanzhou primary school.
              </P><P>
                These work was done by Ning Chai, Xinyu Zhou, Jindian He, Qi Li and Yiding Xu.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="9"><H5>Week 9 (May30-Jun6)</H5><P style="text-align: justify;">We detected the OD600 everyday and when the cultures grew to OD600=1.3, we harvested the cells. We mixed the samples of total cell lysate, soluble fraction and membrane fraction with a loading buffer and boiled for 10 min. Then we analyzed samples on SDS-PAGE using 12% (w/v) acrylamide and transferred them to a polyvinylidene difluoride membrane. Then we used pre-stained molecular weight markers to estimate protein molecular weights and performed western blot analysis, using His-tag as the first antibody and goat anti mouse as the second antibody. The correct size of samples was detected. 
              </P><P style="margin-bottom: -5px;">
                Western blot analysis for subcellular:
              </P></DIV><P>
              These work was done by Ning Chai, Xinyu Zhou, Jindian He, Ning Chai and Qi Li.
            </P></DIV><DIV class="row"><DIV class="section" id="10"><H5>Week 10(Jun7-Jun14)</H5><P style="text-align: justify;">We cultured the <EM>E. coli</EM> BL21 DE3 containing BBa_K1995001: IHG (INP-N-GFP-HG), BBa_K1995005: IC (INP-N-GFP-C) and BBa_K1995011: IH (INP-N-GFP-H) until OD600=1.3 and harvested and washed them twice with 0.85% NaCl and re-suspended them in 0.85% NaCl. Then we mixed the bacteria with solutions of Hg<SUP>2+</SUP> and Cu<SUP>2+</SUP>(1–10 μL) of different concentrations and incubated them at room temperature. We used a fluorescence spectrophotometer to detect the fluorescence.The excitation wavelength was 488 nm, and the emission wavelengths were in the range of 500 to 600 nm. Emission and excitation of the slit was 5 nm. However, the result was not ideal.
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of IHG (OD600=1.3) in the presence of Cu<SUP>2+</SUP> with varying concentrations:
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of IC (OD600=1.3) in the presence of Cu<SUP>2+</SUP> with varying concentrations:
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of IC (OD600=1.3) in the presence of Hg<SUP>2+</SUP> with varying concentrations:
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of IH (OD600=1.3) in the presence of Hg<SUP>2+</SUP> with varying concentrations:
              </P><P>
                These work was done by Anqi Wei, Xinyu Zhou, Jindian He, Ning Chai and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="11"><H5>Week 11(Jun15-Jun22)</H5><P style="text-align: justify;">We used various metal ions including Cd<SUP>2+</SUP>, Pb<SUP>2+</SUP>, Ba<SUP>2+</SUP>, Mo<SUP>6+</SUP>,Ni<SUP>2+</SUP>, Hg<SUP>2+</SUP>, Mn<SUP>2+</SUP>, Ca<SUP>2+</SUP>, Ag<SUP>+</SUP>, Zn<SUP>2+</SUP>, and Cu<SUP>2+</SUP> to measure the selectivity of the <EM>E. coli</EM> to adsorb mercury ions. After the same treatment, we used the same method to detect the fluorescence of GFP-MC. 
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of IHG (OD600=1.3) in the presence of different ions:
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of IC (OD600=1.3) in the presence of different ions:
              </P><P style="margin-bottom: -5px;">
                Fluorescence spectra of IH (OD600=1.3) in the presence of different ions:
              </P><P>
                These work was done by Anqi Wei, Xinyu Zhou, Jindian He, Ning Chai and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="12"><H5>Week 12(Jun23-Jun30)</H5><P style="text-align: justify;">We tested the influence of coexistence of other metal ion on mercury assays by detecting the fluorescence response of IC,IH and IHG (OD600=1.3) in the presence of Cu<SUP>2+</SUP> and various additional metal ions(50uM) using the same method.However, the result was not ideal.
              </P><P>
                These work was done by Anqi Wei, Xinyu Zhou, Jindian He, Ning Chai, Puxin Wang and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="13"><H5>Week 13(Jul1-Jul8)</H5><P style="text-align: justify;">We tested the influence of coexistence of other metal ion on mercury assays by detecting the fluorescence response of IC,IH and IHG (OD600=1.3) in the presence of Cu<SUP>2+</SUP> and various additional metal ions(50uM) using the same method.However, the result was not ideal.
              </P><P style="text-align: justify;">
                We repeated the experiment of testing the influence of coexistence of other metal ion on mercury assays with the same method. We got the ideal result.
              </P><P style="margin-bottom: -5px;">
                Fluorescence response of IC (OD600=1.3) in the presence of Cu<SUP>2+</SUP> and various additional metal ions:
              </P><P style="margin-bottom: -5px;">
                Fluorescence response of IHG (OD600=1.3) in the presence of Cu<SUP>2+</SUP> and various additional metal ions:
              </P><P style="margin-bottom: -5px;">
                Fluorescence response of IH (OD600=1.3) in the presence of Cu<SUP>2+</SUP> and various additional metal ions:
              </P><P>
                These work was done by Anqi Wei, Xinyu Zhou, Jindian He, Ning Chai and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="14"><H5>Week 14(Jul9-Jul16)</H5><P style="text-align: justify;">We started to construct BBa_K1995012: BA (merA coding sequence), BBa_K1995013: GA (GFP-merA), BBa_K1995014: IA (INP-N-merA), BBa_K1995016: IR (INP-N-merR), BBa_K1995017: RG (merR-GFP), BBa_K1995018: BCC (CC coding sequence), BBa_K1995019: CC (INP-N-GFP-CC), BBa_K1995020: CCW (GFP-CC+ secretion expression vector) 
              </P><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">PCR system (50μL)</TH></TR></THEAD><TBODY><TR><TD>5×ps</TD><TD>10</TD></TR><TR><TD>dNTPs</TD><TD>4</TD></TR><TR><TD>Fr</TD><TD>1</TD></TR><TR><TD>Rr</TD><TD>1</TD></TR><TR><TD>taq E</TD><TD>0.5</TD></TR><TR><TD>template DNA</TD><TD>1.5</TD></TR><TR><TD>ddH2O</TD><TD>32</TD></TR></TBODY></TABLE><TABLE class="striped" style="margin-bottom: 50px;"><THEAD><TR><TH data-field="PCR" class="center-align" style="width: 100%;">Digest system(20μL)</TH></TR></THEAD><TBODY><TR><TD>EcoR I</TD><TD>1</TD></TR><TR><TD>Xho I</TD><TD>1</TD></TR><TR><TD>10×H buffer</TD><TD>2</TD></TR><TR><TD>DNA</TD><TD>10</TD></TR><TR><TD>ddH2O</TD><TD>6</TD></TR></TBODY></TABLE><P>
              These work was done by Anqi Wei, Xinyu Zhou, Jindian He, Ning Chai and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="15"><H5>Week 15(Jul17-Jul24)</H5><P style="text-align: justify;">We successfully constructed BBa_K1995012: BA (merA coding sequence), BBa_K1995013: GA (GFP-merA), BBa_K1995014: IA (INP-N-merA), BBa_K1995016: IR (INP-N-merR), BBa_K1995017: RG (merR-GFP), BBa_K1995018: BCC (CC coding sequence), BBa_K1995019: CC (INP-N-GFP-CC), BBa_K1995020: CCW (GFP-CC+ secretion expression vector). Through transformation, extraction of plasmid, digestion and measurement by agarose gel electrophoresis, we selected the expected strain.  
              </P><P>
                These work was done by Anqi Wei, Xinyu Zhou, Jindian He, Ning Chai, Qi Li and Puxin Wang.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="16"><H5>Week 16(Jul25-Aug 1)</H5><P style="text-align: justify;">We bought fish (<EM>Cyprinus carpio</EM>) from the market in order to prepare the experiment on fish.We mixed the engineered <EM>E.coli</EM> with dry fodder in order to feed fish the engineered bacteria.  
              </P><P>
                These work was done by Anqi Wei, Jindian He, Ning Chai, Qi Li, Yiding Xu and Puxin Wang.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="17"><H5>Week 17(Aug2-Aug9)</H5><P style="text-align: justify;">We started our experiment on fish. We did preliminary test by treating fish with different concentrations of mercury (0.1mg/L,0.15mg/L and 0.2 mg/L) in order to determine the best concentration. Finally, the concentration we chose to use in experiment was 0.1mg/L. <EM>Cyprinus carpio</EM> of size ranging from 7.483–8.348 g weight and 5.75–6.52 cm length, were introduced into fish tanks of 20-L capacity, which contained 6 L tap water. Each tank contained ten fish. We fed fish with fodder containing engineered <EM>E. coli</EM> and used a commercial fish feed containing crude protein (Min 30%), crude fat (Min 20%), crude fiber (Min 4%), moisture (Max 5%), and algae (Max 25%) as control. We fed fish, changed water (containing 0.1mg Hg<SUP>2+</SUP>) and recorded mortality, external signs, and growth conditions of the fish everyday. Also, we extracted the flesh of dead fish in order to detect the mercury. 
              </P><P>
                These work was done by Zhichen Lin, Xinyu Zhou, Jindian He, Ning Chai and Qi Li.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="18"><H5>Week 18(Aug10-Aug17)</H5><P style="text-align: justify;">The experiment on <EM>Cyprinus carpio</EM> was going on.We dissected dead fish by sterile scissors, got the bacteria from gut and extracted the total DNA using TIANamp Stool DNA kit. 
              </P><P>
                These work were done by Xinyu Zhou, Puxin Wang, Yiding Xu, Ning Chai and Anqi Wei, Zhicheng Lin, Zheyuan Zhang.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="19"><H5>Week 19(Aug18-Aug25)</H5><P style="text-align: justify;">The experiment on <EM>Cyprinus carpio</EM> was going on. 
              </P><P>
                These work was done by Puxin Wang, Yiding Xu and Zheyuan Zhang.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="20"><H5>Week 20(Aug26-Sept2)</H5><P style="text-align: justify;">The experiment on Cyprinus carpio was going on.We cultured <EM>E. coli</EM> containing the recombination plasmid in LB medium and dectected the OD600 everyday.  
              </P><P>
                These work was done by Puxin Wang, Anqi Wei, Zhicheng Lin Yiding Xu and Zheyuan Zhang.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="21"><H5>Week 21(Sept3-Sept10)</H5><P style="text-align: justify;">After 30 days of feeding, we dissected all the sampled fish using sterile scissors and extract the took out intestines. Then we examined samples of intestines and extracted the total DNAusing TIANamp Stool DNA kit. We amplified the V4 hypervariable region of 16S rRNA gene using universal primer 515F and 909R with 12 nt unique barcode for pyrosequencing using Miseq sequencer. 
              </P><P style="text-align: justify;">
              When the OD600 of engineered <EM>E. coli</EM> reached 1.3, we added an appropriate concentration of mercury ions (0, 6.5, 12.5, 25 mg/L) to LB medium. In order to measure the metal ion adsorption ability of GFP-MC displayed on E. coli, we harvested cells from LB medium by centrifugation (5000 rpm, 3 min) and then washed them twice with saline. Then we analyzed total mercury by atomic absorption spectrophotometry.  
              </P><P>
                These work was done by Xinyu Zhou, Anqi Wei, Puxin Wang, Yiding Xu, Ning Chai and Zheyuan Zhang.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="22"><H5>Week 22(Sept11-Sept18)</H5><P style="text-align: justify;">We diluted, denatured, re-diluted the purified library and mixed it with PhiX (equal to 30% of final DNA amount) in accordance with the Illumina library preparation protocols, and then applied it to an Illumina Miseq system for sequencing with the Reagent Kit v22 × 250 bp according to the manufacturer’s manual. We processed the sequence data using QIIME Pipeline–Version 1.7.0.In order to measure the selectivity of the <EM>E. coli</EM> to adsorb mercury ions, we added 6.5 mg/l Hg<SUP>2+</SUP> and Cd<SUP>2+</SUP>, and 12.5 mg/l Cu<SUP>2+</SUP> and Zn<SUP>2+</SUP> in place of the mercury and measured the metal ion content in the samples by atomic absorption spectrophotometr(AAS).
              </P><P>
                These work was done by Xinyu Zhou, Anqi Wei, Zhicheng Lin, Puxin Wang, Yiding Xu, Ning Chai and Zheyuan Zhang.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="23"><H5>Week 23(Sept19-Sept26)</H5><P style="text-align: justify;">We detected the concentration of mercury in the flesh of samples of fish by AAS.We incubated the engineered <EM>E.coli</EM> bacteria cells with 12.5mg/l Hg<SUP>2+</SUP> overnight in order to exam the morphology of the mercury adsorbed on the GFP-MC-displayed cells.
              </P><P>
                These work was done by Puxin Wang, Anqi Wei, Zhicheng Lin, Qi Li, Ning Chai and Zheyuan Zhang.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="24"><H5>Week 24(Sept27-Oct4)</H5><P style="text-align: justify;">We visited the iGEM team BIT-China this week.
              </P><P style="text-align: justify;">
                After mercury ion adsorption, justify;orption, we washed the GFP-MC-displayed cells twice with saline and dispersed them in in ddH2O. Then we analyzed surface adsorption by TEM. We performed elemental analysis with the energy-dispersive X-ray spectroscopy system attached to the transmission electron microscope and recorded the electron microscopy images.
              </P><P style="margin-bottom: -5px;">
                The TEM images of <EM>E. coli</EM> BL21/INP-GFP-MC cells:
              </P><P style="margin-bottom: -5px;">
                The control TEM images of <EM>E. coli</EM> BL21:
              </P><P>
                Our team member Xinyu Zhou, Jindian He, Zhicheng Lin, Pengfei Luo, Yiding Xu and Zheyuan Zhang visited iGEM team BIT-China in Beijing.
              </P><P>
                The experiment work was done by Anqi We, Puxin Wang, Yiding Xu, Ning Chai and Zheyuan Zhang.
              </P></DIV></DIV><DIV class="row"><DIV class="section" id="25"><H5>Week 25(Sept19-Sept26)</H5><P style="text-align: justify;">We constructed all of our parts to the pSB1C3.
              </P><P>
                These work was done by Jindian He.
              </P></DIV></DIV></DIV></DIV></DIV><FOOTER class="page-footer blue darken-1" style="margin: 0px;"><DIV class="container"><DIV class="row" style="margin: 0px;"><DIV class="col l6 s12"><H5 class="white-text">Lanzhou 2016</H5><P class="grey-text text-lighten-4">NO.222, TianShui South Road, Lanzhou, China, 730000</P><H6 class="white-text">Copyright 2015 Lanzhou University</H6></DIV><DIV class="col l3 s12"><H5 class="white-text">Acknowledgments</H5><UL><LI><A class="white-text" href="http://en.lzu.edu.cn/">Educational Administration Office, LZU</A></LI><LI><A class="white-text" href="http://chc.lzu.edu.cn/en/">Cuiying Honors College, LZU</A></LI><LI><A class="white-text" href="http://lifesc.lzu.edu.cn/">School of Life Sciences,LZU</A></LI></UL></DIV></DIV></DIV><DIV class="footer-copyright blue darken-3"><DIV class="container">
              Powered by <A class="brown-text text-lighten-3" href="http://materializecss.com">Materialize</A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>