<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Westminster_Journal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Westminster/Journal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Westminster/Journal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Westminster/Journal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Westminster/Journal&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Westminster/Journal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Westminster/Journal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="wrapper"><DIV id="header"><DIV id="navigation"><UL id="navigation-list"><LI><A href="https://2012.igem.org/Team:Westminster" class="navlink">Home</A></LI><LI><A href="https://2012.igem.org/Team:Westminster/Team" class="navlink">Team</A></LI><LI class="submenu-holder"><A href="https://2012.igem.org/Team:Westminster/Overview" class="navlink active">Project</A></LI><LI class="submenu-button"><A href="https://2012.igem.org/Team:Westminster/Overview" class="navlink">Overview</A></LI><LI class="submenu-button"><A href="https://2012.igem.org/Team:Westminster/Problem" class="navlink">The Problem</A></LI><LI class="submenu-button"><A href="https://2012.igem.org/Team:Westminster/Experiments" class="navlink">Experiments</A></LI><LI class="submenu-button"><A href="https://2012.igem.org/Team:Westminster/Protocols" class="navlink">Protocols</A></LI><LI class="submenu-button"><A href="https://2012.igem.org/Team:Westminster/Results" class="navlink">Results</A></LI><LI class="submenu-button"><A href="https://2012.igem.org/Team:Westminster/Safety" class="navlink">Safety</A></LI><LI class="submenu-button"><A href="https://2012.igem.org/Team:Westminster/Journal" class="navlink active">Journal</A></LI></UL><LI><A href="https://2012.igem.org/Team:Westminster/Parts" class="navlink">Parts</A></LI><LI><A href="https://2012.igem.org/Team:Westminster/Modeling" class="navlink">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Westminster/Outreach" class="navlink">Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Westminster/Judging" class="navlink">Judging Criteria</A></LI><LI><A href="https://2012.igem.org/Team:Westminster/Attributions" class="navlink">Attributions</A></LI></DIV></DIV><DIV id="the-content"><DIV id="home-main-text"><H1>DIARY</H1><P><B><SPAN class="diary-date">May 14 </SPAN></B></P><P>Westminster iGEM 2012 ideas storm meeting. Ideas discussed
    include</P><P>1. Engineering postmodification of GFP as a detection system</P><P>2.  Engineering
    bacteria monolayers</P><P>3.  Engineering
    for reduced senescence</P><P>4.  Synthetic
    epigenetic regulator</P><P>5. Bio sensing</P><P>6. Synthetic telomeric expansion</P><P>7. Generics or Antibody production</P><P>8. Biomagnetism</P><P>9. HIV prevention based on receptor affinity</P><P>10. Exosome delivery systems</P><P>11. Ortogonal rhibosome</P><P><B><SPAN class="diary-date">May 16</SPAN></B></P><P>Westminster iGEM 2012 ideas storm meeting. Discussion about
    recruiting more members to the team. </P><P><B><SPAN class="diary-date">May 21</SPAN></B></P><P>Advertising poster made and put up around Cavendish campus
    inviting student to attend the next iGEM meeting. </P><P><B><SPAN class="diary-date">May 23</SPAN></B></P><P>Westminster iGEM 2012 ideas storm meeting. Criteria for assessing
    proposed project ideas discussed. Criteria will include:  Feasibility, WoWfactor, Application,
    Impact and Relevance to synthetic biology</P><P><B><SPAN class="diary-date">May 25</SPAN></B></P><P>Presentation of project ideas by individuals to the group.</P><P><B><SPAN class="diary-date">May 30 </SPAN></B></P><P>Evaluation of ideas presented. </P><P><B><SPAN class="diary-date">June 1</SPAN></B></P><P>Two more possible project ideas brought forward.</P><P>1. Idea for detection of proliferating bacterial cells. After
    more research and discussion it was found that the complexity of cell surface
    ligands of Gram Negative bacteria may mean that the project is not likely to be
    feasible. </P><P>2.  Designing  bacteria which dissolve chewing gum on
    pavements raised by Silvia. This idea is to be explored further.</P><P><B><SPAN class="diary-date">June 6</SPAN></B></P><P>More work being done on the chewing gum idea. Additionally,
    other members of the team have been reading more about the iGEM competition and
    biobrick assembly. </P><P><B><SPAN class="diary-date">June 16</SPAN></B></P><P>Identification of key resources still required. Need to
    narrow down idea for the group, and get a few more students involved. Need to
    get some financial support. </P><P><B><SPAN class="diary-date">June 30</SPAN></B></P><P>Project discussions.</P><P><B><SPAN class="diary-date">July 2</SPAN></B></P><P>Identification of main projects to focus on – The
    chewing gum idea. Also the possibility of doing radic’s tattoo idea. </P><P><B><SPAN class="diary-date">July 4</SPAN></B></P><P>Developing the chewing gum idea has continued. A number of
    important factors need to be determined. </P><P>1. Researching the composition of gum.</P><P>2. Assessing whether the chewing gum idea is feasible as a
    synthetic biology project. </P><P>3. Looking for parts in the iGEM registry and determining
    other components required. </P><P><B><SPAN class="diary-date">July 9</SPAN></B></P><P>Update on progress in developing the chewing gum idea. A lab
    experiment has been designed to determine if the bubble gum idea would work. </P><P><B><SPAN class="diary-date">July 13</SPAN></B></P><P>R.I.P. chewing gum idea! Lab experiment showed that the idea
    required  more of a chemical
    solution rather than biological. </P><P style="line-height:
normal;text-autospace:none">New idea
    of Cancer stem cells (CSC) was raised.  <A href="http://www.bbc.co.uk/news/health-19083685">http://www.bbc.co.uk/news/health-19083685</A></P><P style="line-height:
normal;text-autospace:none">Although the existence of CSC was
    introduced more than 40 years ago, it has only been in the last 10 years that
    the identification of Aldehyde dehydrogenase (ALDH) as a biomarker of CSC has
    allowed for these cells to be detected and isolated. The project proposal is
    therefore to create molecular constructs for the detection, isolation and
    destruction of Cancer stem cells. </P><P>2 days to deadline for project proposals! </P><P><B><SPAN class="diary-date">July 15</SPAN></B></P><P>Cancer stem cell idea voted as the Westminster iGEM project
    and submitted to iGEM. </P><P><B><SPAN class="diary-date">July 17</SPAN></B></P><P>Task plan generated. Research started by a number of team
    members. Group members also continued researching building biobricks. </P><P><B><SPAN class="diary-date">July 20</SPAN></B></P><P>Identification of the four ALDH isoforms which will be used
    in the project. </P><P>ALDH1A1 –ALDH-1 family</P><P> ALDH1A3 –
    ALDH-1 family</P><P> ALDH3A1 – AHDH
    3 family</P><P> ALDH2 – to
    be used as control</P><P>We need to identify the promoters for each of the isoforms. </P><P><B><SPAN class="diary-date">July 22</SPAN></B></P><P>Research into Cre-lox technology. Continued research on the
    project and search for ALDH promoter sequences. </P><P>Hima has looked at ALDH2 promoter sequence and has
    identified a 903bp sequence. A commercial sequence has also been found which
    has identical sequence to the PUBMED identified sequence. </P><P><A href="http://www.ncbi.nlm.nih.gov/gene/217">http://www.ncbi.nlm.nih.gov/gene/217</A></P><P>Additionally, a PstI site has been identified in the
    promoter region. This will need to be removed by site directed mutagenesis.</P><P><B><SPAN class="diary-date">July 23</SPAN></B></P><P class="MsoListParagraph" style="line-height:normal">Further feedback on ALDH promoters. Karin had found
    the promoter for ALDH1A1. There seems to be a number of restriction sites in
    the promoter. The gene codes in the reverse direction. This is important when
    designing primers. </P><P><A href="http://www.ncbi.nlm.nih.gov/gene/216">http://www.ncbi.nlm.nih.gov/gene/216</A></P><P>Silvia has looked at ALDH3A1. Promoter region has been identified
    and primers designed incorporating prefix and suffix biobrick regions. </P><P><A href="http://www.ncbi.nlm.nih.gov/gene/218">http://www.ncbi.nlm.nih.gov/gene/218</A></P><P><B>Proposed
    Promoter Sequence: </B>(position -19649444 to -19648446) = <B>998bp</B></P><P><B>Forward-Primer: </B><TT><B><SPAN style="line-height:115%;
font-family:Calibri;&quot;MS PGothic&quot;;">TGAACTTCTGACCTCAGGTGA</SPAN></B></TT></P><P><B>Reverse-Primer: </B><SPAN> </SPAN><TT><B><SPAN style="line-height:115%;font-family:Calibri;
">CGGCCTCGCTGATCTTGC</SPAN></B></TT></P><P><B><SPAN class="diary-date">July 24</SPAN></B></P><P>Further work on primer design. Clarification of prefix and
    suffix regions for building biobricks. ALDH3A1 further analysed, EcoR1 and 2
    PstI sites identified. </P><P><B><SPAN class="diary-date">July 25</SPAN></B></P><P>Need to review ALDH1A1 promoter region. Short video provided by
    Mark on using NCBI to find promoter regions has been useful to the group.  ALDH1A1 promoter region corrected by
    Karin.<A href="http://www.ncbi.nlm.nih.gov/gene/216">http://www.ncbi.nlm.nih.gov/gene/216</A><SPAN>  </SPAN>75569370 to -75568234 on negative
    strand. </P><P>Additionally, a search for biobrick parts begun. List of
    parts required has been drawn up and assigned to members. </P><P><B><SPAN class="diary-date">July 27</SPAN></B></P><P>Primers designed for ALDH1A1. </P><P><SPAN style="font-family:&quot;Times New Roman&quot;;&quot;Times New Roman&quot;">Forward-Primer                        CATCATATGACTTTTTTCAAC    </SPAN></P><P><SPAN style="font-family:&quot;Times New Roman&quot;;&quot;Times New Roman&quot;">Reverse-Primer            TTCTGATTCGGCTCCTGGAA</SPAN></P><P>More research on the Cre-lox system done. A means of
    switching on and off may be required. Parts for the cre-lox system sought from
    the parts registry. </P><P>Mammalian parts identified from 2011 DTU-Denmark iGEM team.
    These are not standard assembly parts. Need to make a decision on how these
    will be used, ie. do we go down the PlugnPlay assembly route. It may be
    necessary as standard mammalian parts have not been located in the registry. Clarification
    on using PlugnPlay has been sought from IGEM on assembly rules. </P><P><B><SPAN class="diary-date">July 29</SPAN></B></P><P>Continued  investigation into PlugnPlay assembly. </P><P><B><SPAN class="diary-date">July 30</SPAN></B></P><P>Registered for iGEM UK Team Hangout. </P><P>Karin identified EcoR1 as the only illegal site in ALDH1A1
    promoter sequence. </P><P><B><SPAN class="diary-date">July 31</SPAN></B></P><P>Following iGEM response a decision to use PlugnPlay assembly
    has been taken. BUT parts will have to be made standard for submitting to the iGEM
    registry.  In particular, it will
    still be necessary to remove illegal sites from promoter regions.</P><P><B><SPAN class="diary-date">August 1</SPAN></B></P><P>Project has been further defined and some new roles
    delegated. Discussion on mutagenisis and removing illegal sites from the
    promoter region. </P><P><B><SPAN class="diary-date">August 2</SPAN></B></P><P>Promoter sequence of ALDH1A3 determined and primers
    designed. </P><P><A href="http://www.ncbi.nlm.nih.gov/gene/220">http://www.ncbi.nlm.nih.gov/gene/220</A></P><P>Promoter sequence checked against commercially available
    sequences. No illegal sites found in the promoter region. </P><P><B><SPAN class="diary-date">August 3</SPAN></B></P><P>Search for antibiotic resistance gene for mammalian cells in
    the biobrick registry. Search for backbone for mammalian expression continues. </P><P><B><SPAN class="diary-date">August 4</SPAN></B></P><P>More discussion on the use of PlugnPlay. Using PlugnPlay
    means that we could test the un-mutated promoters, but would have to do the
    mutation to conform to parts standards. It remains uncertain how mutagenisis
    will affect the promoter activity. </P><P><B><SPAN class="diary-date">August 5</SPAN></B></P><P>Possible backbone identified. </P><P>ALDH2 promoter sequence determined and PstI site has been
    found. Primer for amplification of the promoter region have been designed.
    There is a suggestion that it may be possible to omit the region containing the
    PstI site. This needs to be looked at further. </P><P> <A href="http://www.ncbi.nlm.nih.gov/gene/217">http://www.ncbi.nlm.nih.gov/gene/217</A>.</P><P>More work on the Cre-lox system reported. More work
    presented on identifying biobrick parts. Backbone and fluorescent parts looked
    at. </P><P>Consolidation of progress and identification of next steps
    for the project. </P><P><B><SPAN class="diary-date">August 7</SPAN></B></P><P>Received very useful information from DTU regarding their
    use of PlugnPlay:</P><P><SPAN class="commentbody"><SPAN style="font-size:8.5pt;
line-height:115%;font-family:&quot;Lucida Grande&quot;;color:#333333;background:#EDEFF4">“Response
    from DTU:</SPAN></SPAN><SPAN style="font-size:8.5pt;line-height:115%;
font-family:&quot;Lucida Grande&quot;;color:#333333;background:#EDEFF4"><SPAN class="commentbody">Thank you for your interest in using our assembly
    system.</SPAN><SPAN class="commentbody">The Plug'n'Play system was created in hope that iGEM
    would see that they needed to change their assembly standard and the RFC-10,
    and that it did but it is a slow process. Most of our parts did therefore not
    comply with the RFC-10 and iGEM did not accept them. However, we had to apply
    the RFC-10 to a least one part to be regarded for the gold medal. Three parts
    was done with the RFC-10 and with the backbone of the shipping plasmid pSB1C3
    with chloramphenicol resistance and holding the restriction site of the iGEM
    assembly standard (BBa_K678002).</SPAN><SPAN class="apple-converted-space"> …”</SPAN></SPAN><B><SPAN class="diary-date">August
    8</SPAN></B></P><P>Some discrepancy found in Primers designed for some of the ALDH
    promoter amplifications when final product size was compared to commercial
    sequences. Primers have therefore been re-designed. </P><P>Design for parts assembly begun, experiments 1 and 2. </P><P style="line-height:
normal;"><U><SPAN style="font-size:11.5pt;font-family:Arial;
color:black;">Experiment 1</SPAN></U></P><TABLE border="0" cellspacing="0" cellpadding="0" width="466"><TBODY><TR><TD width="55" valign="top"><P><SPAN style="font-size:10.0pt;font-family:Arial;&quot;Times New Roman&quot;;color:black;">1.ALDH promoter</SPAN></P></TD><TD width="77" valign="top"><P><SPAN style="font-size:10.0pt;font-family:Arial;&quot;Times New Roman&quot;;color:black;">2.Yeast Kozak
          sequence (Ribosome Binding Site) BBa_K165002</SPAN></P></TD><TD width="50" valign="top"><P><SPAN style="font-size:10.0pt;font-family:Arial;&quot;Times New Roman&quot;;color:black;">3.mRFP ORF (protein
          coding domain)</SPAN></P></TD><TD width="62" valign="top"><P><SPAN style="font-size:10.0pt;font-family:Arial;&quot;Times New Roman&quot;;color:black;">4. Eukaryotic
          Terminator</SPAN></P></TD><TD width="88" valign="top"><P><SPAN style="font-size:10.0pt;font-family:Arial;&quot;Times New Roman&quot;;color:black;">5.Hygromycin
          resistance cassette</SPAN></P></TD><TD width="135" valign="top"><P><SPAN style="font-size:10.0pt;font-family:Arial;&quot;Times New Roman&quot;;color:black;">6.Plasmid backbone
          with chloramphenicol  resistance</SPAN></P></TD></TR></TBODY></TABLE><P><U>Experiment 2</U></P><TABLE border="0" cellspacing="0" cellpadding="0" width="469"><TBODY><TR><TD valign="top"><P>1.ALDH promoter</P></TD><TD valign="top"><P>2.Yeast Kozak sequence (Ribosome Binding Site) BBa_K165002</P></TD><TD valign="top"><P>3.Puromycin ORF (protein coding domain)</P></TD><TD valign="top"><P>4. Eukaryotic Terminator</P></TD><TD valign="top"><P>5.Hygromycin resistance cassette</P></TD><TD valign="top"><P>6.Plasmid backbone with chloramphenicol  resistance</P></TD></TR></TBODY></TABLE><P><B><SPAN class="diary-date">August 9</SPAN></B></P><P>Design for assembly of parts begun for experiment 3. </P><P><B><SPAN class="diary-date">August 10</SPAN></B></P><P>It has been brought to attention that direction of promoter
    sequences needs to be confirmed prior to primers being ordered. </P><P style="line-height:
normal"><SPAN style="color:#333333;background:white;
">ALDH Promoters as they appear on their respective
    genes:</SPAN></P><P style="line-height:
normal"><SPAN style="color:#333333;background:white;
">1. ALDH1A1 is on the negative strand</SPAN><SPAN style="color:#333333;"><SPAN style="background:white">2. ALDH3A1 is on the negative strand</SPAN><SPAN style="background:white">3. ALDH1A3 is on the positive strand</SPAN></SPAN></P><P style="line-height:
12.75pt;background:white"><SPAN style="
color:#333333;
">4. ALDH2 is on the positive strand</SPAN></P><P><B><SPAN class="diary-date">August 11</SPAN></B></P><P>More research into USER cloning and drawing up of parts
    list. More research into mutagenisis of illegal sites. </P><P>Parts identified thus far:</P><P><U>Experiment 1</U></P><P class="MsoListParagraphCxSpFirst"><SPAN>1.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>ALDH promoter </P><P class="MsoListParagraphCxSpMiddle"><SPAN>2.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Yeast
    Kozak sequence (Ribosome Binding Site) BBa_K165002</P><P class="MsoListParagraphCxSpMiddle"><SPAN>3.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>mRFP Open Reading Frame (protein coding domain)
    BBa_E1010 (RFC10 compatible) </P><P class="MsoListParagraphCxSpMiddle"><SPAN>4.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Eukaryotic
    Terminator BBa_J52016</P><P class="MsoListParagraphCxSpMiddle"><SPAN>5.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Hygromycin resistance cassette for selection in
    mammalian cells  DTU
    Part:BBa_K678021 (<I><U>NOT</U></I> RFC10
    compatible)</P><P class="MsoListParagraphCxSpLast"><SPAN>6.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Plasmid backbone with chloramphenicol  resistance pSB1C3 (RFC10 compatible)**</P><P><U>Experiment 2</U></P><P class="MsoListParagraphCxSpFirst"><SPAN>1.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>ALDH promoter </P><P class="MsoListParagraphCxSpMiddle"><SPAN>2.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Yeast
    Kozak sequence (Ribosome Binding Site) BBa_K165002</P><P class="MsoListParagraphCxSpMiddle"><SPAN>3.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Puromycin ORF (protein coding domain) BBa_J96012
    (RFC10 compatible) </P><P class="MsoListParagraphCxSpMiddle"><SPAN>4.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Eukaryotic
    Terminator BBa_J52016</P><P class="MsoListParagraphCxSpMiddle"><SPAN>5.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Hygromycin resistance cassette for selection in
    mammalian cells  DTU
    Part:BBa_K678021 (<I><U>NOT</U></I> RFC10
    compatible)</P><P class="MsoListParagraphCxSpMiddle"><SPAN>6.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Plasmid backbone with chloramphenicol  resistance pSB1C3 (RFC10 compatible)**</P><P><B><SPAN class="diary-date">August 11</SPAN></B></P><P class="MsoListParagraph">Parts identified for experiment 3</P><P><U>Experiment 3</U></P><P class="MsoListParagraphCxSpFirst"><SPAN>1.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Doxycycline-inducible mammalian promoter
    BBa_K415506 (<I><U>NOT</U></I> RFC10
    compatible)***</P><P class="MsoListParagraphCxSpMiddle"><SPAN>2.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Cre
    recombinase ORF BBa_J61047 (RFC10 compatible)*</P><P class="MsoListParagraphCxSpMiddle"><SPAN>3.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Eukaryotic Terminator BBa_J52016</P><P class="MsoListParagraphCxSpMiddle"><SPAN>4.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>ALDH promoter </P><P class="MsoListParagraphCxSpMiddle"><SPAN>5.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Lox 66 BBa_I718016 (RFC10 compatible)* </P><P class="MsoListParagraphCxSpMiddle"><SPAN>6.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Yeast Kozak sequence (Ribosome Binding Site)
    BBa_K165002</P><P class="MsoListParagraphCxSpMiddle"><SPAN>7.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Puromycin ORF (protein coding domain) BBa_J96012
    (RFC10 compatible) </P><P class="MsoListParagraphCxSpMiddle"><SPAN>8.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Eukaryotic Terminator BBa_J52016</P><P class="MsoListParagraphCxSpMiddle"><SPAN>9.<SPAN style="font:7.0pt &quot;Times New Roman&quot;">    </SPAN></SPAN>Lox 71 BBa_I718017 (RFC10 compatible)* </P><P class="MsoListParagraphCxSpMiddle"><SPAN>10.<SPAN style="font:7.0pt &quot;Times New Roman&quot;"> </SPAN></SPAN>Yeast
    Kozak sequence (Ribosome Binding Site) BBa_K165002</P><P class="MsoListParagraphCxSpMiddle"><SPAN>11.<SPAN style="font:7.0pt &quot;Times New Roman&quot;"> </SPAN></SPAN>mRFP
    Open Reading Frame (protein coding domain) BBa_E1010 (RFC10 compatible) </P><P class="MsoListParagraphCxSpMiddle"><SPAN>12.<SPAN style="font:7.0pt &quot;Times New Roman&quot;"> </SPAN></SPAN>Eukaryotic
    Terminator BBa_J52016</P><P class="MsoListParagraphCxSpMiddle"><SPAN>13.<SPAN style="font:7.0pt &quot;Times New Roman&quot;"> </SPAN></SPAN>Hygromycin
    resistance cassette for selection in mammalian cells  DTU Part:BBa_K678021 (<I><U>NOT</U></I> RFC10 compatible)</P><P class="MsoListParagraphCxSpLast"><SPAN>14.<SPAN style="font:7.0pt &quot;Times New Roman&quot;"> </SPAN></SPAN>Plasmid
    backbone with chloramphenicol  resistance pSB1C3 (RFC10 compatible)**</P><P>*The cre-recombinase and the two Lox sites are compatible:
    see them in BBa_K318030 the Lock Cassette which is compatible with BBa_K318000
    the Key Cassette.</P><P><B><SPAN class="diary-date">August 16</SPAN></B></P><P>Consolidation of results. Update of parts list. </P><P><B><SPAN class="diary-date">August 17</SPAN></B></P><P>Attendance and presentation at the iGEM UK Hangout at GOOGLE
    Campus. We had the opportunity to learn a lot from the other teams and returned
    from the meet newly inspired. We have a lot of work ahead of us. </P><P><B><SPAN class="diary-date">August 19</SPAN></B></P><P>Push forward with finding sponsorship. Closer look at
    judging criteria, we want to aim high – Reach for GOLD! </P><P>Consolidation of Team name – iSTEM, and further
    development on team identity and designing the team logo. </P><P>New roles taken up by team members. </P><P><B><SPAN class="diary-date">August 23</SPAN></B></P><P>Looked into promotion ideas and the team decided that a
    promotional video is required.  Time
    was spent putting forward ideas for the video.  </P><P>Push forward with finding funding and identification of
    possible sponsors. We also had a discussion about Human Practice. </P><P>Protocols for the lab are being put together. </P><P><B><SPAN class="diary-date">August 24</SPAN></B></P><P>Team photo’s taken in Regent Park. </P><P>More ideas for video have been put forward and the team had
    further discussion about Human Practice and how to proceed. </P><P>T-shirt design was discussed discussed and a few concepts
    derived. </P><P>Big push for each of the team members to deliver results is
    now on. </P><P><B><SPAN class="diary-date">August 25</SPAN></B></P><P>Started to draw up the list for reagents and consumables
    needed in the Lab and filling the order forms. </P><P><B><SPAN class="diary-date">August 26</SPAN></B></P><P>Team photo’s and profile pics selected. Karin has been
    looking into the safety and ethics part of the project. </P><P><B><SPAN class="diary-date">August 27</SPAN></B></P><P>Today we did filming for a promotional video. The footage
    needs to be edited. <SPAN style="font-size:11.5pt;line-height:115%;color:black">The
    search continues to find a sponsor for the Regional Jamoree. Requests have been
    sent to 10 companies, so fingers crossed. It is a bit late in the shedual to
    apply for funding, but we are hopeful. We are looking to interview a number of
    University lecturers involved in Cancer research. First set of primers has been
    ordered!</SPAN></P><P><B><SPAN class="diary-date">August 30</SPAN></B></P><P>Went through the work flow for lab experiments. Determined
    the next steps required to complete the assembly parts list. The designing of
    the plugnplay primers was explained and this work needs to be done soon so that
    primers can be ordered. This was recorded. </P><P>Passaging of our two cell lines was done (by Andy). This is
    in preparation for optimisation of the transfection reagent which is going to
    be used for the transfections. A recording to the passaging of the cells was
    made. </P><P>Action Points:</P><P>Need to create a TICK list</P><P>Need to delegate wiki and computer work</P><P>Need to finalise parts list – we are obtaining parts
    from DTU 2011 iGEM team. </P><P>Need to design fusion primers
    and get them ordered</P><P>Need to look into mutagenisis and design all the required
    primers and get them ordered ASAP</P><P class="diary-date">September 1</P><P>We have come across the Eukaryotic promoter Database!! They
    have the ALDH isoforms which we are using in our project. Further investigation
    has revealed that the promoter sequences on the site have been tested. This is
    encouraging news. Analysis of our own selected sequences and that of EPD show
    that ALDH1A1, ALDH2 and ALDH1A3 have significant overlap. We have designed new
    primers for ALDH1A1 avoid the restriction site. Also, ALDH3A1 has multiple
    promoter regions. We have therefore selected one of the EPD promoter sequences
    and designed new primers. Work continues on designing primers for plug-n-play. </P><P class="diary-date">September 3</P><P>We have moved into the Lab!! But  we are still waiting for primers. Work
    on wiki design and logo continues. Parts required from Serano have been
    identified, now it’s just a matter of getting them from Barcelona!Hopefully DTU
    parts will be sent tomorrow. Still waiting for primers. </P><P class="diary-date">September 5</P><P>The survey has been finalised and is to be posted soon. An
    interview with Dr Miriam Dwek has been organised and all in the team are
    looking forward to this. Andy has begun transfection optimisation using PEI.
    Traven and visa requirements are being sorted. Two team members attended the
    Societies of Westminster first meetup of the year. This was a great opportunity
    to introduce iGEM to other student groups in the university. We received great
    advice from all the groups and look forward to interacting with them over the
    year. </P><P class="diary-date">September 6</P><P>Deadline!! on  7<SUP>th</SUP>.
    We finally got conformation on the iGEM definition of BSc vs MSc students.
    Safety questions are answered and now just need to work out how to get them
    onto the wiki. We have not been able to find outside funding and the University  has generously sponsored the Registration
    fee for Amsterdam. Amsterdam, here we come!!</P><P class="diary-date">September 7</P><P>Deadline! Fees (tick), Safety (tick), Judging (tick)</P><P class="diary-date">September 10</P><P>Been busy… but so much still to do. Still waiting for
    primers so have been busy designing the lad procedures. We really are pressed
    for time now, but all in the team are still upbeat and excited to build out
    BioBricks. Linker primers have been ordered from IDT. Work has begun in
    preparing for Freshers. </P><P class="diary-date">September 11</P><P>The interview with Miriam Dwek has gone very well. Primers
    arrived late this afternoon, so we can start BioBrick building tomorrow. </P><P class="diary-date">September 12</P><P>It’s all systems go in the lab. First PCR to amplify out
    biobricks was unsuccessful. The plan is to do a crude extraction of DNA and see
    if that works. </P><P class="diary-date">September 13 </P><P>We got one!! We have managed to amplify ALDH2 promoter. The
    rest of the team continues to work on Freshers attendance and other aspects of
    the project. We now need to start thinking about site directed mutagenisis!!</P><P class="diary-date">September 16 </P><P>Been busy! Did touchdown PCR and got some bands, since then
    its been PCR-Gel-PCR-Gel-Gel extract-Transform-PCR-Gel…. Love it. </P><P class="diary-date">September 19</P><P>We’ve been busy. There just isn’t enough time in the day.
    Freshers at Marylebone was good experience. This is an arts and design campus,
    so it’s been good practice in trying to engage individuals who are not
    necessarily interested in science. Lab has had its ups and downs. </P><P class="diary-date">September 20</P><P>Freshers at Cavendish! What a great day! It’s been just the
    boost the team has needed. Up to 100 students have signed up the  our mailing list and there has been much
    enthusiastic interest in iGEM. Still waiting for DTU parts! Unfortunately we
    really have run out of time. We are all looking forward to testing plug-n-play
    though! </P><P class="diary-date">September 21</P><P>DTU parts have arrived. Lots to do now. </P><P class="diary-date">September 24</P><P>Things have been manic in the lab. Many PCR optimisations
    later and we are still not satisfied with our amplification of parts using our
    linker primers. All very frustrating. We have tried Taq to see if  we can at least amplify the parts.
    Promoter parts are ready for sending and sequencing. </P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Westminster/Journal">http://2012.igem.org/Team:Westminster/Journal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Westminster/Journal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Westminster/Journal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Westminster/Journal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Westminster/Journal&amp;oldid=229697" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>