"
Page
Discussion
View source
History
teams
Log in
 
Team:Westminster/Results
From 2012.igem.org
Home
Team
Project
Overview
The Problem
Experiments
Protocols
Results
Safety
Journal
Parts
Modeling
Outreach
Judging Criteria
Attributions
WIKI Notebook & Results
PROJECT DESIGN: WEEK 1 (16 July 16 2012)
The first criteria for the project is to identify the most prevalent ALDH isoforms which have been identified in the literature as being markers of cancer stem cells. ALDH1A1 was identified as the most prevalent isoform and ALDH1A3 and ALDH 3A1 have also been identified. ALDH2 has been selected as a control as its expression may be up-regulated by introducing alcohol.
WEEK 2
This week a protocol for identifying the promoter sequences of the ALDH isoforms has been designed.
PROTOCOL
1. Identify the start of the protein sequence (the promoter will be upstream of the start site).
2. Locate the TATA box within the region up-stream of the protein sequence.
3. Select 1000bp up-stream of the protein start site to include the TATA box.
4. Compare selected promoter sequence with commercial sequences.
5. Analyse sequences for illegal sites.
(6. Later we identified the European Promoter Database and compared selected sequences to sequences there and were able to select sequences which excluded illegal sites.
Primers to amplify the ALDH promoters were designed as follows:
pALDH1A1-FWD
GTTTCTTCGAATTCGCGGCCGCTTCTAGAGCATCATATGACTTTTTTCAAC
 
pALDH1A1-RWD
GTTTCTTCCTGCAGCGGCCGCTACTAGTATTCTGATTCGGCTCCTGGAA
 
pALDH2-FWD
GTTTCTTCGAATTCGCGGCCGCTTCTAGAGCAGAGAAGGAAGGGAGTCTTGGTTATC
 
pALDH2-RWD
GTTTCTTCCTGCAGCGGCCGCTACTAGTACTGATGCTCCAGGTGAAGAGACC
 
pALDH1A3-FWD
GTTTCTTCGAATTCGCGGCCGCTTCTAGAGATCGAGGTATAATTTATGC
 
pALDH1A3-RWD
GTTTCTTCCTGCAGCGGCCGCTACTAGTATCCCTGGCCCGAGGCGCCCTA
 
pALDH3A1-Com-FWD
GTTTCTTCGAATTCGCGGCCGCTTCTAGAGTATAGGCGTGAGCCACCG
 
pALDH3A1-RWD
GTTTCTTCCTGCAGCGGCCGCTACTAGTACGGCCTCGCTGATCTTGC
 
pALDH3A1-Com-RWD
GTTTCTTCCTGCAGCGGCCGCTACTAGTAGACAGAGAGCACCTGCAGCT
 
 
WEEK3
Search for template DNA to construct experimental parts has begun. Search of the Registry found no parts which could be used for our design. We have contacted DTU-Denmark 2011 iGEM team to obtain some of their parts directly. Parts requested from DTU are on plasmids BBa_K678049 and BBa_K678056. Serrano labs has also been identified as a source of mammalian parts and MTA transfer is being set up to allow us to receive the parts.
WEEK 4
This week was spent designing construct 3 of our project, the construct for selectively killing Cancer stem cells based on their expression of ALDH. Protocols for user cloning were also studied this week.
WEEK 5
Constructs using the DTU parts have been designed.
Friday was spent at the UK team meet-up.
WEEK 6
Protocols for the lab have been put together and orders for reagents and consumables ordered.
WEEK 7
Design of linker primers using PHUSER site ( http://www.cbs.dtu.dk/services/PHUSER/ ). Mutagenic primers were also designed for the DTU parts which we intend to use.
Linker primers are as follows:
ALDH   linker- FW
gggtttaaugtttcttcgaattcgcggccgc
ALDH   linker- RV
atggtggcgguttcttcctgcagcggccg
m-CHERRY-FW
accgccaccauggtgagcaagggcgagga
m-CHERRY-RV
agttcttguacagctcgtccatgccgc
SV-40-   FW
caagaacutgtttattgcagcttata
SV-40-RV
agacatgauaagatacattgatgagttt
HYGROMYCIN-FW
atcatgtcugccagcaggcagaagtatgcaaagc
HYGROMYCIN-RV
agtagttucttcctgcagcggccgct
pSB1C3-FW
aaactacuagtagcggccgctgcagt
pSB1C3-RV
ggtcttaauctctagaagcggccgcga
Neomycin-FW
aaacctguggaatgtgtgtcagttag
Neomycin-RV
agaacatgauaagatacattgatgagtttgg
 
WETLAB WEEK 1 (12 September 2012)
After months of planning we are finally in the lab!
Wednesday
AIM: The primers for     amplification of the ALDH promoter sequences have arrived. These primers have     the Biobrick overhangs. Primers were diluted to 100mM     and working concentrations of primers was made up at 10mM. The first set of     primers to amplify the four ALDH isoforms was set up.
PCR was set up using Pfu polymerase. Template used was whole cells (mammalian). Cells were thawed, spun     down and the supernatant removed and re-suspended in 50uL DMSO. We attempted to     amplify the following promoter sequences: ALDH1A1, ALDH2, ALDH1A3, ALDH3A1 (1)     and ALDH3A1(2).
The cycle conditions were as follows:
Initial denaturation at 980C for 30 seconds then     30 cycles with 980C for 10 seconds, 540C for 30 seconds     (annealing) and 720C for 30 seconds. Final extension was at 720C     for 10 minutes and then PCR was held at 40C.
RESULTS: No bands were seen     on the gel.
ANALYSIS: It is possible that     the DNA template (addition of whole cells) did not become available in the     reaction. Other cellular components during lysis of     the cells may also have had an impact on the efficiency of the PCR reaction. We     have located a protocol for crude extraction of mammalian DNA and will repeat     the PCR using extracted DNA.
The extraction protocol was performed. (see protocols) Sample 1 was used as DNA template in the next PCR reaction.
Nanodrop readings were as follows:
Sample 1: 42.1ng/µL     with purity ration 260/280 = 1.47
Sample 2: 50.4ng/ µL with purity ration 260/280 = 1.30
Sample 3: 46.4ng/ µL with purity ration 260/280 = 1.28
Thursday
AIM: Amplification of ALDH     promoters. A PCR reaction was carried out as before with the same reaction mix     and PCR conditions as Wednesday. amplify ALDH1A1 and     ALDH2.
RESULTS: Amplification of ALDH1A1 was     successful using the ALDH1A1-Fw and ALDH-Rv primers.     The band size was as expected. ALDH 2 was also amplified using ALDH2-Fw and     ALDH2-Rv primers. 
ANALYSIS: We are pleased with developments thus     far. Difficulties in amplification of ALDH3A1 using our the ALDH3A1(1)     and ALDH3A1(2) primers was anticipated. Initial analysis in selecting an     ALDH3A1 promoter region revealed a number of different options. Additionally,     it was difficult to find commercial sequences which matched the gene sequence     identified in NCBI. Subsequent to ordering the primers, the Eukaryotic Promoter     Database was discovered. Primers for the EPD experimentally designed ALDH3A1     promoter have been ordered and will be tested when they arrive.  ALDH1A1 was initially designed to be     quite large and also includes an illegal site. An experimentally tested ALDH1A1     promoter has been identified on EPD. There is significant overlap between this     EPD promoter sequence and the one we had designed. A new forward EPD designed     primer was ordered which would give a smaller promoter and also eliminate the     illegal site.
Friday
 We have decided to try a touchdown     PCR reaction in order to try to amplify the other parts, particularly ALDH1A1.     Touchdown PCR is a faster but a more crude method of amplification in     comparison to gradient PCR. After setting up the first PCR, we realised we had     made a mistake (Touchdown 1). A second touchdown (Touchdown 2) was set up and     run soon after with the corrections. We managed to get bands for all four     promotes. These were cut from the gel, gel extracted, cut with EP (and ES for     ALDH2), ligated and transformed into competent E coli.
Touchdown 1 cycle conditions were as follows:
Initial denaturation at 980C for 30 seconds then     10 cycles with 980C for 10 seconds, 620C for 30 seconds     (annealing) and 720C for 1 minute 10 seconds, and annealing     temperature decreasing by 10C for each cycle. Second amplification     cycle was 20 cycles of 980C for 30 seconds, 520C for 10     seconds (annealing) and 720C for 1 minute 10 seconds. Final     extension was at 720C for 10 minutes and then PCR was held at 40C.     NOTE: In the second cycle (20 cycles) denaturation time at 980C     should have been 10 seconds and annealing time should have been 30 seconds.     This correction was included in Touchdown 2. Additionally, DNA template     concentration was increased in Touchdown 2, by increasing the volume from 1µL per reaction to 5 µL per reaction
RESULTS: Spurious bands were     obtained in the touchdown 1, but ALDH1A1 and ALDH2 were amplified.  Touchdown 2 gave good results for ALDH2.     For ALDH 1A3 and number of bands were seen. A faint band was observed at the     expected size and this is the band which was cut from the gel. ALDH1A1 from gel     1, ALDH2 from gel 2 and ALDH3A1 from gel 2 were also cut from the gel and gel     purified. Extracted DNA was run on a gel to verify the gel extraction process.  Gel extracted DNA was digested with EP     and ALDH2 was restricted with ES. The backbone was restricted with EP and ES aswell. Restriction digests were saved till the next day.
GEL 1-Touchdown 1
Gel 2-Touchdown 2
WEEK 2
Monday
The linker primers and the EPD primers have arrived today.  Touchdown PCR to amplify the promoter     parts was set up. The touchdown PCR was as before. The forward ALDH1A1(EPD) primer was paired with the reverse primer     ALDH1A1-RV. The EPD designed primers for ALDH3A1 were tested.
RESULTS: It worked a dream!     We have got lovely bands for ALDH1A1(EPD), ALDH2 and     ALDH3A1 (EPD). However the band observed for ALDH1A3 was almost twice the     expected size. The size expected was around 900bp and the size seen on the gel     is around 1500bp. Parts cut from the gel were ALDH1A1 (EPD), ALDH3A1 (EPD) and     ALDH2 (Wmin). DNA was gel extracted and purified. DNA     was then run on a gel to verify the extraction process. An overnight digestion     was set up to create the Biobrick overhang parts. Only     the ALDH1A3 digest from previously was kept. The other digests were discarded     at this point.
Tuesday
Today the restricted parts were cloned to the pSB1C3     backbone and transformed to E. coli.  Parts cloned were the ALDH1A1 (EPD),     ALDH2, ALDH1A3 and ALDH3A1. Transformed E.     coli was plated to chloramphenicol plates which were incubated overnight at     370C.  More plated and     broth were made up ready for use. The parts required from DTU Denmark and Serrano     are being arranged for delivery to us.
The following promoter parts have been mad ready.
ALDH1A1 – BBa_K940000
ALDH2 – BBa_K940001
ALDH 1A3 – Bba_K940002
ALDH3A1 – BBa_K940003
Wednesday
The linker primers were hydrated ready for use. We are still     waiting for the parts from DTU and Serrano. There have been a number of issues     with delivery but we are hoping to have them sorted and the parts delivered by     Friday. Colony PCR was done to confirm the inserts ligated yesterday.
RESULTS: Only some of the colony PCR have worked. This may be due to suboptimal primer     annealing temperatures. Colony PCR was carried out as 10µL volumes an annealing     temperature was 580C using the Pfu polymerase. All colonies were inoculated to LB broth and grown overnight at 370C     in a shaking incubator.
Thursday
Two samples of each of the LB cultures were selected and     plasmid extracted using the QIAprep kit. PCR was then     done on the extracted plasmid in order to confirm DNA insert. This was done as     we had not managed to get good bands for all the colony PCR’s. The PCR results were variable. Again, this may have been due to     annealing temperatures used in PCR. The samples were analysed on the nanodrop. Nanodrop results were     as follows:
But it has now been found that the concentrations we have     are too low to send for sequencing and still have enough for sending to iGEM     parts registry. An attempt to concentrate the samples was made by putting the     samples in a spin vacuum. Disaster!!! I have mixed up the lids! Colonies were     quickly re-picked and inoculated to LB for growth overnight.
Friday
The parts have arrived from DTU Denmark. PCR has been set up     using Pfu Turbo. DNA template used was as follows:
ALDH1A1 (BBa_K940000) extracted     plasmid
mCherry (BBa_K678067) DTU
terminator (BBa_K678067) DTU
hygromycin (BBa_K678049) DTU
ALDH2 (BBa_K940000) extracted plasmid
Neomycin (BBa_K678067) DTU
Biobrick     backbone (pSB1C3) unrestricted
RESULTS: Unfortunately we did     not get any bands. Plasmid was visible on the gels which were run. This may     indicate that the plasmid concentration was too high for the PCR reaction. Plasmid     extraction was done using QIAplasmid prep kit. PCR     was done on the extracted plasmid, but again the PCR was not very successful.
NEXT STEPS: Plasmid template was diluted 1/1000     and another PCR set up using the Pfu. Hygromycin is the only part which was amplified. Mg2+  was included in the     reaction mix this time. DTU had reported success when they used Mg2+ . We     were able to amplify hygromycin.
WEEK 3
Monday
The plasmid previously extracted was analysed on the nanodrop. The samples were made ready for sequencing and     for sending to iGEM.
Nanodrop results were as follows:
ALDH1A1(1): 64.5ng/µL with purity ration 260/280 =     2.13
ALDH1A1(2): 348.5ng/µL with purity ration 260/280     =2.20
ALDH2(1): 59ng/µL with purity ration 260/280 =     1.08
ALDH2(2): 66.8ng/µL with purity ration 260/280 =     1.80
ALDH1A3(2): 166.9ng/µL with purity ration 260/280 =     2.17
ADH1A3(2): 44.5ng/µL with purity ration 260/280 =     2.20
ADH1A3(2): 174.2ng/µL with purity ration 260/280 =     2.20
ADH1A3(2): 201.4ng/µL with purity ration 260/280 =     2.22
PCR using Taq polymerase was set     up. We may be able to amplify the parts using this different polymerase.     Unfortunately such products could not be used for user cloning. Taq PCR was set up according to manufacturer protocol. PCR     was run overnight.
Tuesday
The overnight     PCR was analysed.
Promoter     parts were sent for sequencing. The parts were also packaged and sent to iGEM     registry.
ALDH1A1 – BBa_K940000
ALDH2 – BBa_K940001
ALDH 1A3 – Bba_K940002
ALDH3A1 – BBa_K940003
Wednesday
CONCLUSION:
Unfortunately we have not been able to achieve all that we set out     to do in the lab. In the two and a half weeks which we have had in the lab we were     able to amplify the promoter regions which we had designed and we have sent them to     iGEM registry. These parts are currently being sequenced.
Parts produced by the group
ALDH1A1 promoter BBa_K940000
ALDH2  promoter  BBa_K940001
ALDH1A3 promoter BBa_K940002
ALDG3A1 promoter BBa_K940003
Post European Regional in Amsterdam
In order to test the project idea, a new approach has been taken. Parts from Serrano will be used to construct the project plasmids for experiment 1 and 2. Parts will be assembled using restriction-ligation.
Retrieved from " http://2012.igem.org/Team:Westminster/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
