Team:Stanford-Brown-RISD/Experiments
HOME
TEAM
Team Members
Collaborations
PROJECT
Description
Design
Experiments
Results
Notebook
Mission Architecture
InterLab
Model
Demonstrate
Improve
Attributions
PARTS
Parts Overview
Basic Parts
Composite Parts
Part Collection
SAFETY
HUMAN PRACTICES
Human Practices
Education & Engagement
AWARDS
Applied Design
Measurement
Model
Software
JUDGING FORM ⇗
Our overall project can be broken down into four sub-projects. These are characterized as the following: Mycelium Material & Habitat Development, Mycelium Glue Project, Mycelium Filter Project, and finally the Mission Architecture that ties everything together. Below are the Experiments and Methods corresponding to each subproject.
To view the descriptions, abstracts, and/or introductions that correspond to the following Experiments, please see our Descriptions page linked here.
To view the Results and Conclusions that correspond to the following Experiments, please see our Results page linked here.
Mycelium Material & Habitat Development
Plating and growing cultures of mycelia
Mycelium is able to grow on almost any substrate, so long as the substrate contains sufficient nutrients and is not toxic to the fungus. Though various substrates were tested, a standardized substrate is required to observe the growth of mycelium in different conditions. It has also been found that the growth medium of Potato Dextrose Yeast Agar (PDYA) provides the optimal nutrients for mycelium growth [1, 2], without providing excessive nutrients that will encourage bacterial growth and contamination. The solid PDYA growth medium used in the lab consisted of 2.0% dextrose, 0.10% potato extract, 0.15% yeast extract, 1% agar, and 96.50% water. When growing the mycelium as a liquid culture, the liquid PDY growth medium is composed of 2.0% dextrose, 0.10% potato extract, 0.15% yeast extract, and 97.50% Water [3]. The mycelium can then begin to grow on the PDYA medium by extracting a small piece from the live fungus onto the plate, as shown in the sequence of images below.
Live culture of G. lucidum with fruiting bodies.
Mycelium plate of G. lucidum
Microscopic image of G. lucidum mycelium, taken by team member Leo Penny and advisor Lynn Rothschild.
The mycelium is then able to grow and completely fill the plate. An uncontaminated mycelium culture is characterised as completely white, with a fluffy texture. To prevent contamination, all procedures were formed under a sterile laminar flow hood. Below on the left (Figure 1) is a contaminated plate of mycelium, while on the right (Figure 2) is a photo of a well established culture.
Figure 1: Contaminated plate of mycelium.
Figure 2: Well established mycelium culture.
Once a culture of mycelium is well established, pieces of mycelium may be extracted from a single source plate and transferred to other plates of PDYA. This reduces the potential for contamination of mycelium plates as it reduces the chances of bacteria from the live fungus to be transferred onto the plates of PDYA.
Two forms of Ganoderma lucidum were grown; one was the monokaryon, and the other was the dikaryon. A monokaryon is a fungal cell which has only one nucleus. In contrast, a dikaryon is a fungal cell that has two genetically distinct nuclei, although they are allelically-compatible. These two species were provided courtesy of Perez. Based on the analysis done by Perez’s lab the dikaryon variant grows at greater speeds and produces composite materials with greater strength. The monokaryon was grown in addition to the dikaryon as it was easier sample to isolate a sample as it was grown in a petri dish, while in contrast the dikaryon first had to be grafted from a live mushroom that was gifted to us.
Substrate Tests
To further investigate the potential for using mycelium to grow materials and structures both on and off planet, the growth of mycelium on varied substrates were tested. Noteworthy substrates with exceptional material and environmental promises are yard waste, sawdust, used ground coffee beans, and other forms of food waste. Even more importantly, the mycelium successfully grew on lunar and martian regolith simulant with minimal nutrients added in the form of PDY. Below is a table of all the various substrates that the Ganoderma lucidum mycelium was grown on with attached images.
Martian regolith simulant
Used coffee grounds
Discarded wood chips
Yard Waste
Mixed seeds in an egg carton
Quinoa flakes
Temperature Tests
To further understand the growth properties of the mycelium and predict its growth patterns in different environments (namely the Martian surface), temperature tests were conducted. Based on already existing literature on mycelium, it has been suggested that optimal temperature conditions range from 23°C to 30°C. With this in mind, the growth rate of mycelium at the following temperatures were measured: 4°C, 23°C, 30°C, and 37°C. The surface area of each plate of mycelium in each temperature condition measured each day over the course of 20 days. Three plates of mycelium growing on PDYA were all plated on the same day and put at each temperature condition (totaling to 12 plates of mycelium growing simultaneously at different temperature conditions).
Growing Mycelia in Molds
We grew our mycelium products using a procedure we developed and standardized through running of a variety of different experiments and conducting preliminary research with DIY (Do-It-Yourself) mycelium growth kits that are sold by Ecovative on their site[4]. The kits we purchased were aspen, hemp, and kenaf.
Our procedure began with the preparation of the substrates for inoculation with mycelium. To begin, we ground the substrate into a loose particles (if it did not come in that form). We then sterilized the substrate and any containers we planned to use with the autoclave.
After the autoclave process was complete we moved the substrate, the containers, sterilized scalpels, Whirlpack bags which are supplied sterile and sealed,, and petri dishes of mycelium under the laminar flow hood. We then divided the substrate among the Whirlpack bags. We then sealed the bags and removed them out from under the hood. After removal we weighed the mass of the bags and recorded it on the front of the bag. After re-sterilizing with ethanol, the bags were placed back under the laminar flow hood. We then calculated and added a specific amount of PDY (Potato-dextrose-yeast) to the each of the bags containing the substrates (this was done to provide the mycelium with an easy substrate to break down--in the case of Ecovative they used regular flour). We experimented with different ratios between weight and PDY to find the best amount to add--while we were unable to generate a specific ratio by the end of our experimental time frame, the general consensus was the material should be damp but not soaked. Once the PDY is added to the substrate plates of mycelium were added. These plates were first divided into grids before being mixed in with the substrate in the bag. The larger the amount of substrate the larger the number of plates were added.
Once the bags were filled, they were sealed--making sure to leave plenty of air within the bag--and removed from the autoclaves. The bags were then placed in the 30°C for 1-2 weeks, depending on the rate of growth of mycelium into the substrate. Once there was a decent amount of mycelium grown the material was remixed under the laminar flow head, in order to distribute the mycelium more evenly, and then packed into molds (note that some bricks were also grown just leaving the mycelium to grow until the form was filled). There bricks were then left to fit, some in the 30oC incubator and others in a room temperature growth chamber, for another 1-2 weeks.
Once the mycelium had digested the substrate enough to form a solid block, it was removed from the mold, weighed, and baked at 120°C for several hours. Once the mycelium block was 30% of its original weight it was considered finished material.
Mycelium Glue
In designing our mycelium specific bioadhesive candidates, we looked for protein-based adhesive molecules that could be easily produced in E. coli and B. subtilis. We excluded highly glycosylated adhesive proteins as well as proteins with other significant post translational modifications (PTMs) simply because coordinating large PTMs in B. subtilis in space would be exceptionally challenging and beyond our capabilities for the summer. The last component of our criteria for identifying mycelium specific bioadhesive candidates revolved around the composition of mycelium itself. All fungus, mycelium included, contains an outer cell wall composed of chitin, which makes up the surface that would interface with the adhesive [1].
In nature, various organisms have evolved chitin-binding-domains (CBDs) that allow them to bind tightly to the fungal cell wall [2]. One of these organisms is Bacillus circulans from which we borrowed a CBD sequence [2]. Our first glue candidate uses this CBD in a fusion protein of four CBD’s with GSGGSG linkers in between each; our hope was that this could act as a mycelium specific “cross linker” in a sense. Our construct also included a six residue polyhistidine tag and a Lumio fluorescent sequence (CCPGCCGAGG) for detection with Thermo Fisher Scientific’s Lumio Detection Kit [3].
Another protein-based adhesive candidate we investigated was csgA, the major subunit of the bacterial curli system which is secreted in monomers and self-assembles into longer polymer fibers that contribute to the biofilm formation [4]. The csgA protein has been known to be highly adhesive and have amyloid-forming properties, so it was reasoned that it would be a good starting point for bioadhesive proteins [4]. Furthermore, the manner in which csgA fusion proteins express the functionality of the introduced polypeptides while retaining their self-assembly behavior has made them a molecule of extreme interest in engineering functional biomaterials [5].
Our third glue candidate was a combination of the first two: a csgA-CBD fusion protein. We hoped to take advantage of the self-assembling nature of the csgA as well as its known tendency to aggregate under isoelectric conditions [3, 6]. The rationale for this fusion protein was that csgA has a strong affinity for binding its own monomers and CBD of course has a strong affinity for binding chitin. We theorized that a fusion protein with both of these domains would bond tightly to itself and to the mycelium surface.
Lastly, we decided to test the mussel foot fusion protein fp151 developed by researchers at Pohang University of Science and Technology [7]. The fp151 fusion protein is comprised of repeats of the most adhesive segments of marine mussel adhesive proteins (MAPs) 1 and 5 [7]. MAPs are the proteins responsible for adhering marine mussels to the rocks and other hard surfaces they live on in marine ecosystems [7]. It is thought that much of MAPs adhesion properties’ can be attributed to the high number of 3,4-dihydroxyphenyl-alanine (L-DOPA) residues, which require the hydroxylation of tyrosine [7]. To facilitate this post translational modification, we ordered an additional DNA construct encoding the tyrosinase enzyme, which would be able to hydroxylate tyrosine residues in vivo. We also ordered mushroom tyrosinase which has been shown to be able to hydroxylate tyrosine in vitro post-purification [8].
Once all five constructs were ordered, we ligated the linear DNA constructs to the PSB1C3 iGEM backbone using Gibson Assembly Mastermix 2x, transformed NEB T7 competent E.coli with our new plasmids, and plated the colonies on chloramphenicol selective LB plates. The colonies were incubated at 37 ℃ overnight or until there were distinct, visible colonies and never longer than 72 hours. The existence of our DNA constructs in the colonies were confirmed using verification primers in a colony PCR. We then performed his-tag purification on crude cell extracts from our colonies using Thermo Scientific HisPur Ni-NTA spin columns. Following protein purification with the Thermo Scientific HisPur Ni-NTA spin columns, we confirmed the presence of our protein in the final elution using Thermo Fisher Scientific’s Lumio Tag Protein Detection Kit. We then performed a standard BCA Assay to determine our total protein concentration in each elution.
To test quantitatively  the strength of each purified glue candidate we performed a lap-shear test using the Instron 5565 in the Stanford Soft & Hybrid Materials Facility. To do this, we grew G. lucidum mycelium on sawdust into the specified dimensions of the ASTM D3163 rectangle, which is used as a standard for lap-shear adhesive tests [9]. We also cut cardboard into these dimensions to use as a rough control that would help us assess whether or not our glue candidates were truly specific to mycelium material. Before applying our various glue candidates to the mycelium and cardboard specimens, we incubated the purified protein solutions at their isoelectric points at 4 ℃ for 2 days, spun them down, and discarded the supernatant. This was done to allow aggregate forming proteins (csgA, csga-CBD) to form amyloid fibers and for all the proteins (CBD4x, fp151) to reach their optimum functional pH [6].
Mycelium-based Biofiltration
2.1 Design of a Copper-binding Biofilter
When designing our biofilter, we had two guiding questions in mind: how can we best bind metals on a molecular level, and how can we use synthetic biology to create a platform for functionalizing mycelial material? We also considered the advantages of our platform in comparison to previous efforts, such as flagella-based or cellulose filtration tools [10, 11]. One of the largest benefits of using mycelium material is that it leverages the concept of economies of scale, and presents an entirely feasible option of scale-up of our technology to a level that could be successfully implemented on a space mission and on earth in developing countries with poor access to clean water. Fungi are capable of displaying growth on an enormous variety of biomass types, and grow at a rate that is unparalleled by other biological agents used in synthetic biology today [12].
2.2 Chitin Binding
We sought to find ways to utilize mycotecture and enhance its properties, and in the process realized that chitin is ubiquitous in nature in organisms ranging from fungi to insects - in fact, it’s the second most abundant biopolymer on earth [13]. Methods to take advantage of this fact using synthetic biology are limited - we established a part collection using chitin binding domains as a novel platform to functionalize chitin-containing surfaces, with almost unlimited potential for modifying regions of organisms. We demonstrated the utility through designing novel biological cross-linking glues, fluorescent dyes, but most directly, filtration tools. Our use of the CBD in fusion protein production forms an important pipeline for future teams and the global scientific community for numerous applications, and here we focused on its use for metal recovery from mixed aqueous solution.
2.3 Copper Binding
We then sought to optimize the actual binding interaction between our peptides and metals. We chose copper as our proof of concept as a metal that is both biologically and economically important. When looking at past iGEM teams working with metal binding domains, we found that very few, if any, focused on optimizing the manner in which the metals were bound, instead choosing to simply obtain a binding domain from nature and attempting to implement it. We thought we could do better. A researcher in our lab, Dr. Jesica Urbina, developed a 20 amino acid long peptide (referred to as HHTC-Re (HNLGMNHVLQGNRPLVTQGC), that is a modified peptide with  substituted amino acids adapted from the peptide HHTC designed by Kozisek et al. [14]. We wondered whether we could apply this in tandem with repeats to bind multiple copper atoms to the same biomolecule, and if we could use chitin binding domains (CBD’s) to create fusion proteins that could bind copper atoms and then also bind to chitin on our mycelium material [16].
2.4 Construct Design and Modeling
To test whether we could bind multiple metal ions on the same protein and if this relationship would be monotonically increasing, we designed peptides with one, two, and three copper binding domains in tandem (1xHHTC-Re HNLGMNHVLQGNRPLVTQGC, 2xHHTC-Re HNLGMNHVLQGNRPLVTQGCHNLGMNHVLQGNRPLVTQGC, 3xHHTC-Re HNLGMNHVLQGNRPLVTQGCHNLGMNHVLQGNRPLVTQGCHNLGMNHVLQGNRPLVTQGC; see biobrick page for sequence information). These three peptides were synthesized by Elim Biopharmaceuticals (Hayward, CA), and provided as a lyophilized powder at >98% purity. They were also modified to possess N-terminal acetylation/C-terminal amidation to avoid having a charged peptide [17]. 10 mM MES pH 5.5 and the Pierce BCA assay were used for reconstitution and concentration determination, respectively [18].
To check whether the proteins would retain their respective conformations when added in tandem, we used a tool produced by the Zhang lab (University of Michigan) for de novo protein structure prediction (QUARK) [15]. Based on the structures, we were able to hypothesize that the HHTC-Re domains would each be able to bind copper atoms, even when restricted spatially by ordering them sequentially without spacing.
Figure 1: Depiction of the design of the HHTC-Re repeating peptides, along with predicted ab initio structures as per the QUARK database and algorithms. Individual HHTC-Re domains retain their conformation, even in the 3x peptide.
We also developed 3 fusion proteins: CBD-2xHHTC, CBD-3xHHTC, and CBD-6xHHTC for use in the actual filter, comprised of a CBD, and HHTC metal binding repeats interspaced with a GSGGSG flexible linker. These constructs also included a Lumio tag for downstream verification of protein production, and a 6x His tag for protein purification. A challenge incurred when designing these parts was that the His tag maintains some affinity for metals, including copper. This would interfere with downstream modeling and experimental analysis of the interaction between copper and the HHTC domain. We therefore added the Mxe GyrA Intein between the Lumio/His tag and the rest of the protein, as this intein could be cleaved through addition of 50 mM DTT through thiol-mediated cleavage after protein purification [19]. After DTT was added, we were able to obtain the fusion proteins ready for downstream application (as per Figure 2).
Figure 2: Schematic of the design of the fusion proteins (HHTC-Re x n) and the method in which they bind copper.
We also produced an RFP-CBD fusion protein to visualize the distribution of chitin on the surface of a piece of mycelium, as well as a proof of concept for aesthetic design purposes.
2.5 Wet-lab Experiments: Fusion Protein Production
Once all three constructs were ordered (IDT gBlocks), we ligated the linear DNA constructs to the PSB1C3 iGEM backbone using Gibson Assembly, transformed NEB T7 competent E. coli with our new plasmid, and plated the colonies on chloramphenicol selective LB plates. The colonies were incubated at 37℃ overnight or until there were distinct, visible colonies, and never longer than 72 hours. The existence of our DNA constructs in the colonies was confirmed using verification primers (VF2 and VR)  in a colony PCR. We then performed His-tag purification on crude cell extract from our colonies using Thermo Scientific HisPur Ni-NTA spin columns. Following protein purification with the spin columns, we confirmed the presence of our protein in the final elution using Thermo Fisher Scientific’s Lumio Tag Detection Kit [20]. The standard protein purification protocol was modified by introducing a buffer containing 50 mM DTT for on-site cleavage, so the desired fusion protein could be eluted with the His-tag removed. We then performed a BCA Assay to determine our total protein concentration in each elution.
2.6 ITC Testing
Isothermal titration calorimetry (ITC; MicroCal iTC200) was employed to determine the association equilibrium constant (Ka), enthalpy (ΔH), and the number of ions bound per ligand (n). Ka describes the affinity of a ligand for its substrate, and we used it to quantitatively characterize the interaction between our peptide and copper [21]. All binding parameters for the test were within the specifications determined by the manufacturer. We used 10 mM, 2-(N-morpholino)-ethanesulfonic acid (MES) buffer for testing because it does not cause metal ion interference, and has a stable pKa over a wide temperature and pH range. Experiments were conducted at pH 5.5 to prevent copper precipitation, and pre-made copper stock solutions of known concentration were used.
Peptides were prepared for ITC by dissolving lyophilized protein (powder) in MES buffer, and ITC experiments were run at 25°C and set to deliver 20, 0.5 – 1 µL injections of Cu at 150 second intervals. The metal solution in the syringe was titrated into the peptide solution in the cell, and interactions were measured. Raw data were corrected by subtracting the heats of dilution, and collected data were fit with a one-site binding model using the Origin-7™ software.
2.7 Prototype and Phen Green Assay for Bulk Adsorption
To test the peptides, strips of pure mycelium from the species Ganoderma lucidum were incubated in purified CBD-2xHHTC protein (suspended in 6 mL of 10 mM pH 5.5 MES buffer, with a fusion protein concentration of 0.45 mM) that had been cleaved with DTT on a flatbed shaker for 72 hours. These pieces were then cut into uniform 1 cm2 squares (assuming chitin assumes an approximately regular distribution, as this is difficult to control for and testing with RFP-CBD led us to believe this was the case). As a control, we also incubated untreated mycelium in 10 mM pH 5.5 MES buffer for the same time period. We then prepared a Cu stock solution in MES, and the Cu concentration in our starting solution was 325 (+/-25) µM. Three environments/types of trials were run: one with the treated mycelium filter, one with untreated mycelium, and one with just the Cu solution and no mycelial material. These took place in 6 mL of the Cu stock solution in sterile 15 mL falcon tubes on a flatbed shaker. All experiments were run in triplicate, for a total of 9 trials. Two hundred μL samples were collected after 0 minutes, 30 minutes, and 72 hours for surface adsorption testing.
To quantify the amount of copper in the samples that were collected, we used Phen Green SK dye, which is proportionally quenched in the presence of copper [22]. Phen Green SK dye was prepared in a stock solution in PBS (28 µM final concentration). Two hundred  µL of this stock solution was added to each well in a 96-well plate to which 25 µL of sample was added. All experiments were done in triplicate, and a standard curve was generated for Cu and PGSK to determine the amount of Cu in the tested solutions.
Mission Architecture
On another planet, mycelia will grow and expand to provide structural integrity for the house, inside which the astronauts will live. However, the mycelia requires a substrate (food) and oxygen to grow. Where will these supplies come from? Cyanobacteria.
Cyanobacteria are self-replicating, photosynthetic organisms that can convert the abundant carbon dioxide from the Martian atmosphere into oxygen. This oxygen can be used to grow the mycelia, as well as to keep astronauts healthy and alive inside the habitat. Moreover, it has been shown that mycelia can use cyanobacteria as a substrate (food) to grow (at normal Earth gas concentrations). Our goal in this experiment is to demonstrate and quantify the oxygen production capabilities of a specific strain of cyanobacteria, known as Anabaena variabilis. Achieving Earth-like percentages of oxygen (~20%) solely from the Anabaena, in combination with existing knowledge, would demonstrate the feasibility of cyano-based mycelial growth and Astronaut sustenance.
All experimentation was conducted inside an airtight, Nasco Whirl-Pak bag (total volume: 1627 mL). Two BD Gaspak Anaerobe Sachets were used to help establish an anoxic environment (down to .8% O2); Dry ice was used to establish a 90% + CO2 atmosphere, mimicking the Martian atmosphere. Wireless CO2 and O2 sensors measured the gas compositions over time, for the respective gases.
To view the Results and Conclusions that correspond to the Experiments above, please see our Results page linked here.
References
[1] Fungal cell wall - an overview | sciencedirect topics. (n.d.). https://www.sciencedirect.com/topics/immunology-and-microbiology/fungal-cell-wall
[2] Hashimoto, M., Ikegami, T., Seino, S., Ohuchi, N., Fukada, H., Sugiyama, J., … Watanabe, T. (2000). Expression and characterization of the chitin-binding domain of chitinase a1 from bacillus circulans wl-12. Journal of Bacteriology, 182(11), 3045–3054. https://doi.org/10.1128/JB.182.11.3045-3054.2000.
[3] Lumio green in-cell detection kit - thermo fisher scientific. (n.d.). https://www.thermofisher.com/order/catalog/product/12589057.
[4] Barnhart, M. M., & Chapman, M. R. (2006). Curli biogenesis and function. ?Annual Review of Microbiology, 60, 131–147. https://doi.org/10.1146/annurev.micro.60.080805.142106.
[5] Nguyen, P. Q., Botyanszki, Z., Tay, P. K. R., & Joshi, N. S. (2014). Programmable biofilm-based materials from engineered curli nanofibres. Nature Communications, 5, 4945. https://doi.org/10.1038/ncomms5945.
[6] Zhong, C., Gurry, T., Cheng, A. A., Downey, J., Deng, Z., Stultz, C. M., & Lu, T. K. (2014). Self-assembling multi-component nanofibers for strong bioinspired underwater adhesives. Nature Nanotechnology, 9(10), 858–866. https://doi.org/10.1038/nnano.2014.199.
[7] Hwang, D. S., Gim, Y., Yoo, H. J., & Cha, H. J. (2007). Practical recombinant hybrid mussel bioadhesive fp-151. Biomaterials, 28(24), 3560–3568. https://doi.org/10.1016/j.biomaterials.2007.04.039.
[8] Marumo K, Waite JH. Optimization of hydroxylation of tyrosine and tyrosine-containing peptides by mushroom tyrosinase. Biochemistry Biophysics Academy. 1986;872:98–103. https://www.ncbi.nlm.nih.gov/pubmed/3089286.
[9] Lap shear strength astm d3163 and lap shear adhesion astm d5868. (n.d.). http://www.ptli.com/testlopedia/tests/lap_shear-d3163.asp.
[10] Eckhard et al. (2017). Discovery of a proteolytic flagellin family in diverse bacterial phyla that assembles enzymatically active flagella. Nature Communications, 8(1). doi:10.1038/s41467-017-00599-0
[11] Shipovskaya et al. (2003). Physicochemical Modification of Cellulose Acetate for Manufacturing Films, Membranes, and Biofilters. Russian Journal of Applied Chemistry, 76(9), 1514–1518. doi:10.1023/b:rjac.0000012678.32530.c3
[12] Cavka et al. (2014). Comparison of the growth of filamentous fungi and yeasts in lignocellulose-derived media. Biocatalysis and Agricultural Biotechnology, 3(4), 197–204. doi:10.1016/j.bcab.2014.04.003
[13] Domard, A. (2011). A perspective on 30 years research on chitin and chitosan. Carbohydrate Polymers, 84(2), 696–703. doi:10.1016/j.carbpol.2010.04.083
[14] Kožíšek et al. (2008). Molecular Design of Specific Metal-Binding Peptide Sequences from Protein Fragments: Theory and Experiment. Chemistry - A European Journal, 14(26), 7836–7846. doi:10.1002/chem.200800178
[15] Zhang et al. (2015). Integration of QUARK and I-TASSER for Ab Initio Protein Structure Prediction in CASP11. Proteins: Structure, Function, and Bioinformatics, 84, 76–86. doi:10.1002/prot.24930
[16] Watanabe et al. (2000). Expression and Characterization of the Chitin-Binding Domain of Chitinase A1 from Bacillus circulans WL-12. Journal of Bacteriology, 182(11), 3045–3054. doi:10.1128/jb.182.11.3045-3054.2000
[17] Yi et al. (2010). A Highly Efficient Strategy for Modification of Proteins at the C Terminus. Angewandte Chemie International Edition, 49(49), 9417–9421. doi:10.1002/anie.201003834
[18] Huang et al. Competitive Binding to Cuprous Ions of Protein and BCA in the Bicinchoninic Acid Protein Assay. The Open Biomedical Engineering Journal. 2010;4:271-278. doi:10.2174/1874120701004010271.
[19] Marshall et al. (2014). An Evolved Mxe GyrA Intein for Enhanced Production of Fusion Proteins. ACS Chemical Biology, 10(2), 527–538. doi:10.1021/cb500689g
[20] Lumio green in-cell detection kit - thermo fisher scientific. (n.d.). https://www.thermofisher.com/order/catalog/product/12589057.
[21] Freyer, M. W., & Lewis, E. A. (2008). Isothermal Titration Calorimetry: Experimental Design, Data Analysis, and Probing Macromolecule/Ligand Binding and Kinetic Interactions. Methods in Cell Biology, 79–113. doi:10.1016/s0091-679x(07)84004-0
[22] Wehbe, M., Malhotra, A. K., Anantha, M., Lo, C., Dragowska, W. H., Dos Santos, N., & Bally, M. B. (2018). Development of a copper-clioquinol formulation suitable for intravenous use. Drug Delivery and Translational Research, 8(1), 239–251. http://doi.org/10.1007/s13346-017-0455-7
