<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_OUC-China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:OUC-China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_content_wrapper"><TITLE>Demonstrate</TITLE><DIV class="ouc-guide" style="height: 50px;background-color:#008EA1 ;"><DIV class="row"><DIV class="col-md-2 navbar-header" style="text-align: center; padding: 0"><BUTTON class="navbar-toggle" type="button" data-toggle="collapse" data-target=".navbar-responsive-collapse"><SPAN class="sr-only">Toggle Navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-responsive-collapse" style="padding: 0"><DIV class="col-md-1 ouc-navbar" style="padding: 0"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Team</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/Team" style="line-height: 40px; " class="ouc-navdown">Members</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Collaborations" style="line-height: 40px; " class="ouc-navdown">Collaborations</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Attributions" style="line-height: 40px; " class="ouc-navdown">Attributions</A></DIV></DIV></DIV><DIV class="col-md-1 ouc-navbar" style="padding: 0"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Project</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/Description" style="line-height: 40px; " class="ouc-navdown">Description</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Design" style="line-height: 40px; " class="ouc-navdown">Design</A></DIV><DIV class="ouc-right"><A style="line-height: 40px; font-size: 15px;" class="ouc-navdown" href="https://2017.igem.org/Team:OUC-China/proof1">Proof of concept</A><DIV class="ouc-rightnav"><A href="https://2017.igem.org/Team:OUC-China/proof1" style="display: block">Basic fermentation</A><A href="https://2017.igem.org/Team:OUC-China/proof2" style="display: block">Adhesion platform</A><A href="https://2017.igem.org/Team:OUC-China/proof3" style="display: block">MINI-GRE</A></DIV></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Demonstrate" style="line-height: 40px; " class="ouc-navdown">Demonstration</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/InterLab" style="line-height: 40px; " class="ouc-navdown">InterLab</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Improve" style="line-height: 40px; " class="ouc-navdown">Improvement</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Notebook" style="line-height: 40px; " class="ouc-navdown">Notebook</A></DIV></DIV></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Model" style="line-height: 50px;text-align: center" class="ouc-nav">Model</A></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Parts" style="line-height: 50px;text-align: center" class="ouc-nav">Parts</A></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Safety" style="line-height: 50px;text-align: center" class="ouc-nav">Safety</A></DIV><DIV class="col-md-2 ouc-navbar" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Engagement" style="line-height: 50px;text-align: center" class="ouc-nav">Human Practice</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/HumanPractice" style="line-height: 40px; " class="ouc-navdown">Overview</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/HP/Silver" style="line-height: 40px; " class="ouc-navdown">Silver</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/HP/Gold_Integrated" style="line-height: 40px; " class="ouc-navdown">Gold and integrated</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Engagement" style="line-height: 40px; " class="ouc-navdown">Engagement</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="page-header ouc-page-header"><H1><STRONG>Demonstration</STRONG></H1></DIV><H2 class="ouc-heading" id="fermentation"><STRONG>Basic fermentation</STRONG></H2><P style="font-size: 20px">
    	Our basic fermentation part derives from our local environmental problem, the outbreak of <I>Enteromorpha</I> along the coastline in Qingdao, so it is natural that we would eventually go back to the origin and try to solve the real world problem after validation of design concept in the lab. We aim to make use of <I>Enteromorpha</I> residue, where there is no trehalose left because it is easiest to extract. Therefore, all we need to do is to deal with the cellulose and hemicellulose left in the residue.  
    </P><P style="font-size: 20px">
    	We do treat our <I>Enteromorpha</I> powder with enzymes first and yeast later.(the <I>Enteromorpha</I> powder serves as the stimulation of <I>Enteromorpha</I> residue in real-world situation) The successful survival of the recombinant yeast strains that can use either xylose or cellubiose as the only carbon source can fully prove the feasibility of our designed pathway. 
    </P><P style="font-size: 20px">
    	Along with the proof of our concept, we validate the upstream pathway from real algae powder, which has exactly the same constituent as <I>Enteromorpha</I> residue, and we can say that our idea can apply to real-world problems! 
    </P><DIV class="container"><H2 class="ouc-heading" style="color: #66BCC7" id="enter"><STRONG><I>Enteromorpha</I> pretreatment</STRONG></H2><P style="font-size: 20px">
    	    We treat the Enteromorpha powder (residue) with 0.2% H<SUB>2</SUB>O<SUB>2</SUB> to remove the lignin then cellulase and xylanase to produce xylose &amp; cellubiose. 
    	</P><DIV class="row"><DIV class="col-md-4"><P style="color: gray;text-align: center;">
                    Figure 1.1 Enteromorpha Powder
                </P></DIV><DIV class="col-md-4"><P style="color: gray;text-align: center;">
					Figure 1.2 Treat the residue with 0.2% H<SUB>2</SUB>O<SUB>2</SUB>. 
                </P></DIV><DIV class="col-md-4"><P style="color: gray;text-align: center;">
                    Figure 1.3 Enzymatic hydrolysis solution of Enteromorpha fiber
                </P></DIV></DIV><H2 class="ouc-heading" style="color: #66BCC7" id="pre"><STRONG>Pretreatment validation</STRONG></H2><P style="font-size: 20px">
    		After that we detect the existence of xylose &amp; cellubiose with HPLC. The peak appears at the same point, suggesting that they are the same substance. In other words, we successfully proved that the downstream product of <I>Enteromorpha</I> powder after pretreatment contains mainly xylose and cellubiose.
    	</P><DIV class="row"><DIV class="col-md-6"><P style="color: gray;text-align: center">Figure 1.4 Result of standard cellubiose.
    			</P></DIV><DIV class="col-md-6"><P style="color: gray;text-align: center">Figure 1.6 Result of standard xylose.
    			</P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><P style="color: gray;text-align: center">Figure 1.5 Result of our sample (the doublet indicate there is impurity in our sample).
    			</P></DIV><DIV class="col-md-6"><P style="color: gray;text-align: center">Figure 1.7 Result of our sample (the smaller peak indicate there is impurity in our sample).
    			</P></DIV></DIV><H2 class="ouc-heading" style="color: #66BCC7" id="yeasta"><STRONG>Yeast A that ferments xylose</STRONG></H2><P style="font-size: 20px">
    	    After introducing the plasmid we constructed that contains gene <I>XYL1</I>,<I>XYL2</I> into <I>S.cerevisiae</I> EBY100. We creat a xylose-utilize strain. We measured the growth rate of both our recombinant strain and negative control to prove the superiority of our new strain in xylose utilization.
    	</P><P style="font-size: 20px">
			We cultivate the EBY100(XDH-XR) and the EBY100(control) in SC adding 2% xylose as the sole carbon source, adjusting initial OD<SUB>600</SUB> about 1.2,and placing in the shaking incubator of 30℃,180rpm to ferment.
		</P><P style="font-size: 20px">
    		The following result can well demonstrate that the strain that carries our plasmid grows much better than the strain that not (the stationary phase OD of EBY100(XDH-XR)is nearly 2 times that of WT EBY100)and reach the stationary phases after 40 hours’cultivation.
    	</P><P style="color: gray;text-align: center">Figure 1.8 Growth curve of strains of our recombinant strain and negative control. Error bars represent standard deviation of four biological replicates, so do the other charts in this section.
    	</P><P style="font-size: 20px">
    		For an immediate evidence, we need to know exactly how xylose content change in the medium. We use HPLC to detect the changing concentration of xylose, getting more data to support our idea that our cells can utilize the carbon sources as long as the concentration of xylose decrease with time.
    	</P><P style="font-size: 20px">
    		The following chart shows the xylose content of both our recombinant strain and negative control. It is obvious that in our xylose-utilize strain, xylose content decrease (especially rapid in the first 40h) as time goes by while for the negative strain the xylose content stays steady, indicating the disability of using xylose.
    	</P><P style="color: gray;text-align: center">Figure 1.9 Xylose content of both our recombinant strain and negative control.
    	</P><P style="font-size: 20px">
    		In addition, to finally realize our design, the yeast need to ferment on xylose only. Therefore, we use the SBA-Biosensor to detect the ethanol content in the medium, which can convert the reaction of immobilized enzyme to electrochemical signal and help make a curve reflecting the ethanol change in the culture condition.
    	</P><P style="font-size: 20px">
    		The following chart shows the ethanol content of both our recombinant strain and negative control.
    	</P><P style="font-size: 20px">
    		Gladly, the curve of xylose-consuming strain goes gradually up along with cultivating hours, and reach the plateau at around 90 hours, which is consist with the xylose consuming curve, indicating that our strain do produce ethanol on the basis of xylose, thus realizes our design and proves the concept of basic fermentation part.
    	</P><P style="color: gray;text-align: center">Figure 1.10 Ethanol content of both our recombinant strain and negative control.
    	</P><P style="font-size: 20px">
    		In the same way, we conduct a series of experiment to confirm that our cellubiose-utilizing pathway also worked.
    	</P><P style="font-size: 20px">
    		For the cellulose plasmid, we integrate cellubiose-degrading gene CDT and GH-1 into pYC230 by Gibson.
    	
    	</P><P style="color: gray;text-align: center">Fig 1.9 from lane 1 to 7 respectively:pYC230-GH1-CDT1 single enzyme digestion; pYC230-XYL1-XYL2 single enzyme digestion;marker; pYC230-GH1-CDT1 circular plasmid;pYC230-XYL1-XYL2 circular plasmid; marker 
    	</P><P style="font-size: 20px">
    		We introduce the plasmid into <I>S.cerevisiae</I> EBY100 and construct the cellubiose-utilize strain successfully and then test the growth rate of both our recombinant strain and negative control.
    	</P><P style="font-size: 20px">
    		We cultivate the EBY100(CDT-GH1) and the EBY100(control) in SC adding 2% cellubiose as the sole carbon source, adjusting initial OD<SUB>600</SUB> about 1，and placing in the shaking incubator of 30℃，180rpm to ferment.
    	</P><P style="font-size: 20px">
    		Using HPLC we detect the changing concentration of cellubiose. The result shows that the concentration of cellubiose decrease with time, indicating a well utilization of cellubiose.
    	</P><P style="font-size: 20px">
    		The following chart shows of both our recombinant strain (left) and negative control (right).
    	</P><P style="font-size: 20px">
    		To examine the production of ethanol, we use the SBA-Biosensor, the same as in the xylose pathway.
    	</P><P style="font-size: 20px">
    		The following chart shows the cellubiose content, ethanol content, and growth rate of both our cellubiose recombinant strain (left) and negative control (right).
    	</P><DIV class="row"><DIV class="col-md-6"><P style="color: gray;text-align: center">Figure 1.12 Concentration of cellobiose,ethanol as well as OD600 of biomass change in 140h for cellobiose-recombinant strain
    	        </P></DIV><DIV class="col-md-6"><P style="color: gray;text-align: center">Figure 1.13 Concentration of cellobiose,ethanol as well as OD600 of biomass change in 140h for negative control
    	        </P></DIV></DIV><P style="font-size: 20px">
    		We acquired similar trend of those curves except one, the ethanol content, which dropped from around 70 hours. Considering the difference between monosaccharide and disaccharide, we suppose that the accumulation of the glucose when making use of cellubiose might be the reason lying beneath. Therefore, we detect the glucose content of the medium after that to validate our assumption.
    	</P><DIV class="row"><DIV class="col-md-6"><P style="color: gray;text-align: center">Figure 1.14 The glucose concentration  change in 140h for recombinant EBY100(CDT-1-GH-1)  and WT EBY100
    	        </P></DIV><DIV class="col-md-6"><P style="color: gray;text-align: center">Figure 1.15 The ethanol concentration  change in 140h for recombinant EBY100(CDT-1-GH-1)  and WT EBY100 
    	        </P></DIV></DIV><P style="font-size: 20px">
    		The result shows that glucose content in the medium reach a peak at about 40 hours and then decrease rapidly, which is consist with our expectation that the glucose in yeast was transformed gradually into acetic acid. Ethanol might then participate in further metabolic procedure with acetic acid thus was turned into ethyl acetate when glucose accumulates to a particular concentration.
    	</P><P style="font-size: 20px">
    		Despite that, we can tell from the distinct difference between cellubiose-consuming strain and negative control that our approach of using yeast to transform <I>Enteromorpha</I> residue into ethanol finally worked!
    	</P><P style="font-size: 20px">
    		We successfully confirmed in lab through simulation that our idea has been turn into real-world practice!

<A href="https://2017.igem.org/Team:OUC-China/proof1">See more details in our proof page</A></P></DIV><H2 class="ouc-heading" id="mini"><STRONG>MINI-GRE</STRONG></H2><H3 style="color: #66BCC7"><STRONG>Circuit construction</STRONG></H3><P style="font-size: 20px">
    	To explore the feasibility of MINI-GRE(genetic regulatory elements) by combination of promoters and terminators as we mentioned above, we designed four promoter-terminator pairs, and constructed four different report circuits for them (fig. A)
    </P><P style="font-size: 20px">
    	For circuit 1, we pair promoter CYC1 with terminator CYC1, which are among the most commonly used native promoters and terminators and also have a relative medium strength in yeast.[4] For circuit 2, promoter CYC1 is paired with terminator MINI. The MINIp-CYC1t and MINIp-MINIt, respectively, serves as the chosen pair for circuit 3 and 4.    </P><P style="font-size: 20px">
    	For convenience, we named the“CYC1p-yECitrine-CYC1t-mStrawberry-CYC1t”as“CC”,“CYC1p-yECitrine-MINIt-mStrawberry-CYC1t”as“CM”,“MINIp-yECitrine-CYC1t-mStrawberry-CYC1t”as“MC”, and“MINIp-yECitrine-MINIt-mStrawberry-CYC1t” as “MM”,hereafter. 
    </P><P style="text-align: center; color: gray">
    	Fig. A the plasmid map of our circuit CC, CM, MC, MM. The CC circuit includes the commonly used native promoter CYC1 and terminator CYC1. The MM circuit includes the combination of MINI promoter and MINI terminator. 
    </P><H3 style="color: #66BCC7"><STRONG>Results</STRONG></H3><P style="font-size: 20px">
    	For each circuit, the strength of promoters was characterized by yECitrine, a kind of yellow fluorescent protein was used to detect the output level of particular promoter, terminator or promoter-terminator pair. And the red fluorescent signal from RFP mStrawberry can represent the relative read-through efficiency of particular terminator in the circuits including the same promoter.
    </P><P style="font-size: 20px">
          While monitoring the growth rate of four strains containing different expression regulatory devices, we also characterized the expression strength of the promoter-terminator pairs by detecting the fluorescence intensity of yECitrine in each circuit. As we can see in the bar chart, the ratio and relationship of the signals from 4 circuits become relatively stable at the early stationary phase, which hints that the expression level may reach the dynamic steady state at the time point 22 hours. And the results from 22 hours point also match the strength relationship of the two promoters in previous research, although we used another yeast strain here. [2, 5] Therefore, comparing with the mid-log phase data, we tend to believe that this results can reflect the true dynamic characteristics of the genetic regulatory devices, although we will research this phenomenon in our future work.(Fig. D)

    </P><P style="font-size: 20px">
    	So, in order to better reflect the dynamic behaviors of the circuits we tend to use data from mid-log phase during the growth process.
    </P><P style="text-align: center; color: gray">
    	Fig. B The growth curve of the four strains with different promoter- terminator pairs. Error bars represent standard deviation of three biological replicates. 

    </P><P style="text-align: center; color: gray">Fig. C The fluorescence/Abs600 in different time. Error bars represent standard deviation of three biological replicates.
    </P><P style="text-align: center; color: gray">Fig.D The fluorescence/Abs600 of strains with different promoter-terminator pairs, after being cultivating for 22 hours. Error bars represent standard deviation of three biological replicates.
    </P><H3 style="color: #66BCC7"><STRONG>Function Verification of promoters and terminators</STRONG></H3><P style="font-size: 20px">
    	Through comparing “CC” with “CM”, we can learn that the Fluorescence/Abs600 of “CM”is nearly three times of “CC”, proving that the strength of MINI terminator is higher than that of CYC1 terminator. (Fig. D)(At first we think of measuring this characteristic by comparing yECitrine fluorescence /mStrawberry fluorescence. However, considering that the fluorescence of mStrawberry is too low, we cannot expect an accurate result due to the high error rate. ) We assume that both MINI terminators and CYC1 terminators can effectively stop the transcription, so the mRNA of mStrawberry generated behind these terminators can hardly be detected, neither does the fluorescence of mStrawberry.  
 

    </P><P style="font-size: 20px">
    	What’s more, the Fluorescence/Abs600 of “MM”is also nearly 3-fold compared with “MC”, proving that the strength of MINI promoter is higher than CYC1 promoter. 


    </P><P style="font-size: 20px">
    	In addition, when we compared “MM” with “CC” which is the commonly used promoter-terminator pair, we found that the difference is nearly 6-fold. (Fig. D)
    </P><P style="font-size: 20px">
    	From our results, we confirmed that the MINI promoter and MINI terminator do be superior to CYC1 promoter and CYC1 terminator, not only in length but also in strength. (Fig. E)

    </P><P style="text-align: center; color: gray">Fig. E Length and output level of 4 different promoter–terminator combination. Error bars represent standard deviation of three biological replicates
    </P><H3 style="color: #66BCC7"><STRONG>Robustness in different host</STRONG></H3><P style="font-size: 20px">
    	In order to verify that the MINI-GRE is able to work with the same rule in a variety of yeast strains and expend the application range of these MINI regulatory devices, we invited other teams to repeat the same experiments in different yeast strains and experimental environment, which can also be an important part of our collaboration. 

    </P><P style="font-size: 20px">
    	The time point of the data collection is early stationary phase of yeast growth. The yeast strains we used with Nanjing-China were Saccharomyces cerevisiae EBY100. The yeast strain used in Tianjin was synX, the yeast strain with chemical synthetic chromosome. 
    </P><P style="font-size: 20px">
    	From the result of other colleges, we can know that our MINI-GRE can also work in other yeast strains. (Fig. F)
    </P><P style="text-align: center; color: gray">Fig. F The fluorescence/Abs600 of strains with different promoter-terminator pairs in the late  from three teams, OUC-China, Tianjin and Nanjing-China(left to right). Error bars represent standard deviation of three biological replicates.
    </P><H3 style="color: #66BCC7"><STRONG>Transcription Level Assay by qPCR</STRONG></H3><P style="font-size: 20px">
    	Moreover, we run qPCR towards corresponding protein in order for a further validation of different promoter-terminator combination’s expression on a post-transcriptional level. 

    </P><P style="font-size: 20px">
    	The result from qPCR assay shows that at the 22nd hour from setting up culture, the circuits reached the highest expression intensity and the expression level of four circuits is shown below. 

    </P><P style="text-align: center; color: gray">Fig. G The relative transcription level of yECitrine and mStrawberry at the 22nd hour from setting up culture. Error bars represent standard deviation of three biological replicates.

    </P><P style="font-size: 20px">
    	However, the transcript level of yECitrine doesn’t matched the fluorescence very well. We infer that it may be caused by the experimental error of the total mRNA concentration and the little mismatch of the primers. 

    </P><P style="font-size: 20px">
    	Luckily, we can still learn that the transcript level of mStrawberry is very low, which means the transcriptional read through of both CYC1t and MINI terminator can be overlooked; they can efficiently terminate the transcription, which confirmed the assumption before. 
    </P><H3 style="color: #66BCC7"><STRONG>Conclusion</STRONG></H3><P style="font-size: 20px">
    	To draw a conclusion, from our experiments, we confirmed the superiority of the MINI-GRE (MINI promoter-MINI terminator pair), which can be summed into three aspects: 
    </P><P style="font-size: 20px">
    	(1) Short but strong, which decreases the possibility of undesired homologous reorganization and provide significant output strength compared with commonly used promoter-terminator pair.
    </P><P style="font-size: 20px">
    	(2) Versatile,which can work in different yeast strains and provide robust function for extensive application.
    </P><P style="font-size: 20px">
    	(3) Well-modularited, has good modularity because of the low transcriptional read through efficiency from minimal terminator, which is an important characteristic in synthetic biology. 
<A href="https://2017.igem.org/Team:OUC-China/proof3">See more details in our proof page</A></P></DIV><DIV class="container-fluid ouc-reserve"><P class="text-center">
                          Contact Us : oucigem@163.com  |  ©2017 OUC IGEM.All Rights Reserved.  |  Based On Bootstrap</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>