<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_OUC-China_proof2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:OUC-China/proof2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_content_wrapper"><TITLE>proof2</TITLE><DIV class="ouc-guide" style="height: 50px;background-color:#008EA1 ;"><DIV class="row"><DIV class="col-md-2 navbar-header" style="text-align: center; padding: 0"><BUTTON class="navbar-toggle" type="button" data-toggle="collapse" data-target=".navbar-responsive-collapse"><SPAN class="sr-only">Toggle Navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-responsive-collapse" style="padding: 0"><DIV class="col-md-1 ouc-navbar" style="padding: 0"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Team</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/Team" style="line-height: 40px; " class="ouc-navdown">Members</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Collaborations" style="line-height: 40px; " class="ouc-navdown">Collaborations</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Attributions" style="line-height: 40px; " class="ouc-navdown">Attributions</A></DIV></DIV></DIV><DIV class="col-md-1 ouc-navbar" style="padding: 0"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Project</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/Description" style="line-height: 40px; " class="ouc-navdown">Description</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Design" style="line-height: 40px; " class="ouc-navdown">Design</A></DIV><DIV class="ouc-right"><A style="line-height: 40px; font-size: 15px;" class="ouc-navdown" href="https://2017.igem.org/Team:OUC-China/proof1">Proof of concept</A><DIV class="ouc-rightnav"><A href="https://2017.igem.org/Team:OUC-China/proof1" style="display: block">Basic fermentation</A><A href="https://2017.igem.org/Team:OUC-China/proof2" style="display: block">Adhesion platform</A><A href="https://2017.igem.org/Team:OUC-China/proof3" style="display: block">MINI-GRE</A></DIV></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Demonstrate" style="line-height: 40px; " class="ouc-navdown">Demonstration</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/InterLab" style="line-height: 40px; " class="ouc-navdown">InterLab</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Improve" style="line-height: 40px; " class="ouc-navdown">Improvement</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Notebook" style="line-height: 40px; " class="ouc-navdown">Notebook</A></DIV></DIV></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Model" style="line-height: 50px;text-align: center" class="ouc-nav">Model</A></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Parts" style="line-height: 50px;text-align: center" class="ouc-nav">Parts</A></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Safety" style="line-height: 50px;text-align: center" class="ouc-nav">Safety</A></DIV><DIV class="col-md-2 ouc-navbar" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Engagement" style="line-height: 50px;text-align: center" class="ouc-nav">Human Practice</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/HumanPractice" style="line-height: 40px; " class="ouc-navdown">Overview</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/HP/Silver" style="line-height: 40px; " class="ouc-navdown">Silver</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/HP/Gold_Integrated" style="line-height: 40px; " class="ouc-navdown">Gold and integrated</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Engagement" style="line-height: 40px; " class="ouc-navdown">Engagement</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="page-header ouc-page-header"><H1><STRONG>Proof of concept——Adhesion platform</STRONG></H1></DIV><H2 class="ouc-heading" id="adhesion"><STRONG>Verification of Adhesion platform</STRONG></H2><P style="font-size: 20px">
    	We use ice-nuclei to display the monomer streptavidin (mSA) on the surface of <I>E. coli</I> outer membrane and Aga1p-Aga2p system to display the biotin on the yeast cell wall. The mSA on <I>E. coli</I> surface and the biotin on the yeast cell wall form a covalent linkage and bind with other firmly, establishing an adhesion platform.
    </P><P style="font-size: 20px">
    	After sub cloning our target gene onto the vectors successfully, we transformed the plasmids into either <I>E. coli</I> or <I>S. cerevisiae</I> to test the expression and function of corresponding protein, which means they should be integrated in the out membrane or cell wall. Therefore, we conduct Western Blot so that we can prove our result specifically. After that, we need to verify that the heterogeneous cells do adhere to each other through the combination of biotin and mSA. With rhodamine- polyethylene glycol-biotin and FITC-streptavidin, we conducted immunofluorescence staining on both <I>E. coli</I> and <I>S. cerevisiae.</I> By observing through a fluorescence microscope and measuring florescent signal form two kinds of cell, we had a visual validation of parts work. Meanwhile, we explored the optimal condition for their co-culture and drew a conclusion after preliminary experiment.
    </P><P style="font-size: 20px">
    	For more solid a proof of their adhesion, we used Immunofluorescent Localization to demonstrate their relative location and took photos with transmission electron microscopy (TEM) for their microstructure. What’s more, we acquired some quantitative data such as the percentage of linkage and expression with FCM. As a result, maybe we can say that our adhesion platform was established successfully.
    </P><DIV class="container"><H3 style="color: #66BCC7" id="ecoli"><STRONG><I>E.coli</I> strain construction</STRONG></H3><P style="font-size: 20px">
    		We assume expressing monomer streptavidin on the outer membrane of <I>E.coli</I> will help us use biotin-avidin system to link <I>E.coli</I> and <I>S.cerevisiae</I>. In part registry, we find <A href="http://parts.igem.org/Part:BBa_K523013">BBa_K523013</A> constructed by team 2011_Edinburgh(http://parts.igem.org/Part:BBa_K523013) encoding INP-eYFP. We acquired INP using PCR from this part. Then we choose the constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23106">BBa_J23106</A> from part registry because it can express protein continuously so that we can have a relative steady co-culture system. The mSA part is from team 2016_Peking that gives us much help. Monomer streptavidin is much smaller than that of wild type and is easier to be used in the circuit. Based on above reasons, we designed plasmid J23106-INP-mSA-pSB1C3 (We call it JIM for short ) and transformed it into <I>E.coli</I> DH5α strain.
    	</P><P style="text-align: center;color: gray">Fig.1 Electrophoresis result of expression vector. Lane 3&amp;4 is pSB1C3-J23106-INP-mSA.  Lane1&amp;2 is pYD1-BAP, lane 3&amp;4 is pSB1C3-J23106-INP-mSA, lane5&amp;6 is pYC230-BirA
    	</P><P style="font-size: 20px">
    		The sequencing results confirmed that we successfully cloned the J23106-INP-mSA-PSB1C3 expression vectors, we transformed the target plasmids to DH5α for amplification.
    	</P><H3 style="color: #66BCC7"><STRONG>Protein Expression Assay</STRONG></H3><P style="font-size: 20px">
    		If the protein is successfully expressed, they will be displayed on the extracellular mem-brane of the <I>E.coli</I>. Therefore, we can verify protein expression by detecting whether INP-mSA exists in the total outer membrane protein.
    	</P><P style="font-size: 20px">
    		We used the Fractionation Separation to extract three components of the outer membrane protein, the intimal protein and the cytoplasmic protein of engineered <I>E.coli</I> and negative control. Further Western Blot aim to confirm the previous result specifically and We choose to use horseradish peroxidase-biotin because it has the specific ability to bind INP-mSA.
    	</P><P style="text-align: center;color: gray">Fig.2 Western blot analysis using HRP-biotin to evaluate J23106-INP-mSA(JIM) expression. lane1, the outer membrane of JIM; lane2, the intimal protein of JIM; lane3,the cytoplasmic protein of JIM; lane4, the outer membrane of DH5α; lane5, the intimal protein of DH5α; lane6, the cytoplasmic protein of DH5α.
    	</P><P style="font-size: 20px">
    		Western Blot result shows that there is a clear protein band at corresponding position with a correct size (50.2 kDa). It proved that we did express INP-mSA on the <I>E.coli</I> surface successfully .
    	</P><H3 style="color: #66BCC7"><STRONG>Protein function verification</STRONG></H3><P style="font-size: 20px">
Immunofluorescence Staining
    		After verifying the protein expression, we want to prove that the protein can function as normal. We used Rhodamine-biotin to immunofluorescence stain the engineered <I>E.coli</I>  and negative strain  DH5α to observe whether the bacteria has fluorescence under fluorescence microscope.
    	</P><P style="font-size: 20px">
    		The result proves that the function of protein was normal.
    	</P><P style="text-align: center;color: gray">Fig 2.3 Rhodamine-biotin immunofluorescence staining of engineerd <I>E.coli</I>(right) and negative strains(left).
    	</P><P style="font-size: 20px">
    		Meanwhile, we would like to quantify the expression of INP-mSA compared with DH5α by plate reader. When we carried out the experiment, we met the problem that there are different incubation conditions with immunofluorescence staining in different paper.  So we had to explore the best Incubation condition for immunofluorescence staining. Firstly, We did two sets of experiments to compare the fluorescence values of 2h incubation under two temperatures ( 0 ℃ and 30 ℃, 0℃ means incubate on ice.) 
    	</P><P style="text-align: center;color: gray">Fig 2.4 Expression level of INP-mSA fusion protein under different conditions.( I )  the single cell fluorescence values from <I>E.coli</I> DH5α cell expressing JIM fusion and DH5α without a plasmid under 0 ℃  incu-bation for 2h. ( II ) the fluorescence values of <I>E.coli</I> DH5α cell expressing JIM fusion and DH5α under 30 ℃.
    	</P><P style="font-size: 20px">
    		The image showed that <I>E. coli</I> containing the Jim plasmid successfully expressed INP-MSA fusion protein. The fluorescence value showed that after 2 hours incubation under 30 ℃ conditions the Rhodamine-biotin can better binding to mSA. We speculate that there may be an ice dye incubation that causes ice nucleation proteins to form crystals that inhibit the combination of fluorescent dyes and the mSA of fusion proteins. [2] 
    	</P><H3 style="color: #66BCC7" id="yeast"><STRONG>Yeast strain construction</STRONG></H3><P style="font-size: 20px">
    		Here we need to construct the vector of <I>BirA</I> (biotin ligase) and BAP (biotin acceptor peptide) in yeast for displaying biotin on the surface.(fig.1)
    	</P><P style="font-size: 20px">
    		We first cloned birA into the yeast-shuttle plasmid pYC230. After transformation, it enables the yeast to transfer biotin to biotin acceptor peptide（BAP);  pYD1 is another 5.0 Kbp expression circuit designed for expression, secretion, and display of proteins on the extracel-lular surface of <I>S.cerevisiae</I>. The vector contains AGA2 gene from <I>S.cerevisiae</I> encoding one subunit of the a-agglutinin receptor. By fusing our gene of interest to <I>AGA2</I>, so we can enable the yeast to display the biotin on the cell surface. 
    	</P><P style="font-size: 20px">
    		After successful construction of plasmids, we introduced pYD1 containing BAP and pYC230 containing Bira into <I>S.cerevisiae</I> EBY100. 
    	</P><H3 style="color: #66BCC7"><STRONG>Protein expression verification</STRONG></H3><P style="font-size: 20px">
    		We want to test whether biotinylated BAP is successfully integrated in the yeast cell wall. So, we dyed engineered yeast and negative control respectively with FITC-streptavidin. Using fluorescent microscopy, we are able to observe the fluorescence of both.
    	</P><P style="text-align: center;color: gray">Fig 2.7 The fluorescent staining of yeast with FITC-Streptomyces. (left) <I>S.cerevisiae</I> EBY100 without target gene; (middle) <I>S.cerevisiae</I> EBY100 with BAP+BirA though galactose induction for 12h; （right) <I>S.cerevisiae</I> EBY100 with BAP+BirA though galactose induction for 32h.
    	</P><P style="font-size: 20px">
    		 Meanwhile, we would like to  quantify the expression of biotin compared with EBY100 by plate reader, and wonder when biotin can reach the maximum expression. 
    	</P><P style="text-align: center;color: gray">Fig 2.8 Expression level of biotin in different time point compared with negative controll. The red curve represents for the fluorescence of  EBY100-BirA-BAP as time  goes  by. And  the  blue curve represents for the fluorescence  of  EBY100 control. 
    	</P><P style="font-size: 20px">
    		As a result we can see the FI/abs600 of <I>S.cerevisiae</I> EBY100 with BAP+BirA after galactose inductionis that increases as time goes by. At about 20h the expression of biotin reached maximum level，and at about 30h the accumulation of biotin reached maximum.

    	</P><P style="font-size: 20px">
    		 Further, we want to know how many biotins are expressed on the surface of a single cell. Luckily, we found that we can confirm the expression of biotin and its expressing percent under Flow cytometer. 
    	</P><P style="font-size: 20px">
    		 We use FITC-streptavidin to stain <I>Saccharomyces cerevisiae</I> and use 488nm exciting light to excite it and detect fluorescent signal at 525nm. As the result showed here, the fluorescent peak shifted towards the direction of stronger fluorescent, which means positive sample express more biotin acceptor peptide(BAP)  Compared with the results of empty sample, we can ensure that BAP is displayed on the surface of Saccharomyces cerevisiae. Above the threshold, there are 48% of Saccharomyces cerevisiae in the positive sample. While in the negative sample and empty sample, the percentage is only 28%. 
    	</P><P style="text-align: center;color: gray">Fig 2.5 the result of FCM for yeast
    	</P><P style="text-align: center;color: gray">Fig 2.5.1 the x-mean level of FCM result
  </P><P style="text-align: center;color: gray">Fig 2.5.2 the CV of FCM result
    	</P><P style="font-size: 20px">
    		 Under the same CV, the mean fluorescence of BAP-biotin expressing <I>S.cerevisiae</I>EBY100 (positive) is 1.5 times higher than that of <I>S.cerevisiae</I>EBY100 with an empty plasmid (negative).Considering the wild type <I>S.cerevisiae</I>(empty), the fluorescent of negative control is almost zero.(the slight fluorescent of it may come from the residue of PBS)  
    </P><H3 style="color: #66BCC7" id="co"><STRONG>Co-cultivation verification</STRONG></H3><P style="font-size: 20px">
    		We aimed to build an adhesion platform of <I>E. coli</I> and <I>S.cerevisiae</I>. Therefore, first we need to select a culture condition under which the heterogeneous cells can both grow happily. Taking into account the expression of the BAP in <I>S.cerevisiae</I>, galactose is required to induce protein expression. And to avoid diauxic growth of yeast when changing a new carbon source in the medium, we ultimaytely selected the YNB-CAA galactose medium. In order for the optimize expression of INP-MSA fusion protein, the in-ducing expression of BAP, and the growth rate of two heterogeneous cells, we took 28 ℃ as the culture temperature.
    	</P><P style="font-size: 20px">
    		We characterized the growth curve of <I>E. coli</I> with JIM target gene and that of <I>S.cerevisiae</I> containing BirA &amp; Bap under the mentioned culture condition. And the dry group wonder some parameters of the growth of <I>E.coli</I> and <I>S.cerevisiae</I>, as well as the consumption of galactose.
    	</P><P style="text-align: center;color: gray">Fig.8 (a) the growth curve of <I>E.coli</I> with JIM; (b) the growth curve of <I>S.cerevisiae</I> containing BirA &amp; Bap
    	</P><P style="font-size: 20px">
    		From the image we can tell, when the yeast and <I>E.coli</I> grows at 28 ℃, the culture time needs to be longer than 48h in order to make the yeast take a dominant proportion. We adjust the YNB-CAA galactose medium containing S.cerevisiae initial OD600 to 1.</P><P style="font-size: 20px">
    		Meanwhile, we used HPLC to measure the consumption of galactose.
    	</P><H3 style="color: #66BCC7" id="adhesion"><STRONG>Adhesion validation</STRONG></H3><P style="font-size: 20px">
    		In order to illustrate that in practice, the co-culture of <I>S.cerevisiae</I> and <I>E.coli</I> can truly form the collaborating platform of heterologous cells, we will cultivate the two strains together by adding <I>E.coli</I> to the galactose-induced yeast. The co-cultured samples will then be dyed with rhodamine-biotin and FITC-streptavidin.
    	</P><P style="font-size: 20px">
    		As is shown below, we can see that there are quite a few areas of overlapping red and green fluorescent in positive group where there are both <I>S.cerevisiae</I> and <I>E.coli</I> in the sample, indicating that they are very likely to have linked to each other. The negative control of <I>S.cerevisiae</I> and <I>E.coli</I> however, has either red or green fluorescent. And for the empty group, there is no fluorescent detected at all. 
    	</P><P style="text-align: center;color: gray">Fig 2.10 The observation of confocal laser scanning microscopy(Nikon A1). The rhodamine-biotin is a solid dye, so it may have impurities in this picture.  
（tips: please maximize the intensity of your screen when looking at this picture)
    	</P><P style="font-size: 20px">
    		One step further, we want to confirm the connection between <I>E. coli</I> and Saccharomyces cerevisiae by FACS analysis, because this analysis method can observe in single-cell level.Unfortunately, however, we found the Beckman flow cytometry (FCM) in our university couldn’t excite our fluorescent dye rhodamine-biotin so we were not able to use it to catch dyed <I>E. coli</I>.  
        </P><P style="font-size: 20px">
Finally, we use the scanning tunneling microscope to shoot the microscopic structure of the connecting system and get a clearer picture of the connection structure. So far, we can say our adhesion platform is successfully established.
To confirm the link between <I>E.coli</I> and <I>S. cerevisiae</I> further, we use Transmission Electron Microscopy (TEM), which can capture our strains in a microcosmic view. In the following results, the positive samples show clearly that several <I>E.coli</I> stick firmly to <I>S. cerevisiae</I>, indicating a successful adhesion between them.  In negative samples (where the yeast and bacteria have no plasmid introduced), however, there is no such connection. Actually, because of the loss of sampling point in TEM, we do not even spot the existence of <I>Saccharomyces cerevisiae</I> in negative control. We think that it may result from the dispersive distribution of <I>E.coli</I> and <I>S. cerevisiae</I>.  Due to the limited time, we are not able to repeat our experiment and acquire data that is more convincing.  In future, we will certainly try again. </P><P style="font-size: 20px">So far, the current results indicate that our first-stage adhesion platform is successfully established.
    	</P><P style="text-align: center;color: gray">Fig.12 Observation of Transmission Electron Microscopy 

    	</P>
    	Furturework
        
        If this foundation is established successfully, we will be able to use <I>E.coli</I> which can express cellulase and <I>Saccharomyces cerevisiae</I> which can utilize xylose and cellose to produce alcohol. Further the platform has more greater potential.
    	
    	
    	
    </DIV><DIV class="panel-group" id="accordion"><DIV class="panel"><DIV class="panel-heading ouc-panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse2"><STRONG>PROTOCOL</STRONG></A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="container"><H3 style="color:#E3D434"><STRONG>Cell Fractionation</STRONG></H3>
                		At first Cells cultured at 37℃ until cell density(OD600=0.6-0.8), after cultured at 25℃ for 24 h. Harvested cells were washed, and resuspended in PBS buffer containing 1 mM EDTA and lysozyme at 10 Ag/mL. After 2 hincubation, cell suspension was treated with an ultrasound sonication at 30 sec  2 cycles. To obtain total membrane fraction, whole cell lysate was pelleted by centrifugation at 20,000 rpm for 1 h using an ultracentrifuge (Optima LE-80K; Beckman, Fullerton, CA). The supernatant was regarded as soluble cytoplasmic fraction. For further outer membrane fractionation, the pellet (total membrane fraction) was resuspended with PBS buffer containing 0.01 mM MgCl2 and 2% Triton X-100 for solubilizing inner membrane and incubated at room temperature for 30 min, and then the outer membrane fraction was repelleted by ultracentrifugation. <H3 style="color:#E3D434"><STRONG>Western Blot Analysis</STRONG></H3>
                		An equal volume of each fraction (cytoplasmic and outer membrane) of the cells containing inp-mSA were mixed with SDS sample buffer (TaKaRa, 4X Protein SDS PAGE Loading Buffer), boiled for 15 min, and resolved by 12.5% (wt/vol) SDS-polyacrylamide gel electrophoresis (SDS-PAGE), followed by electrophoretic transfer to Hybond-PVDF membranes with transfer buffer (48 mM Tris-HCl, 39 mM glycine, 20% methanol, pH 9.2) by using a Trans-Blot SD Cell (Bio-Rad, Hercules, CA) at 100V for 1h30 min. After blocking for 1 h in TBS buffer (20 mM TrisHCl, 500 mM NaCl, pH 7.5) containing 5% (wt/vol) nonfat dry milk, the membrane was then incubated for 1.5 h at room temperature in antibody solution (1% (wt/vol) nonfat dry milk in TTBS (TBS with 0.05% Tween-20)) containing HRP conjugated Biotin (1:500 vol/vol) (Sangon Biotech). After successive washing twice for 5 minutes with TTBS for 2 and first for 5 minute with TBS. Color, putting the membrane protein on the face on the petri dish, and then to the above color AB drops, reaction of 2-3 minutes, then don't directly put the cassette in the dry, covered with layers of plastic wrap, draw out the film edge, so as to avoid dark room inside can't find the location of the membrane.
                		I use a petri dish to fill the film, then rinse it out and put it in the fixer.<H3 style="color:#E3D434"><STRONG>Pyd1 yeast display vector expression and display of protein fusion</STRONG></H3>
                		Grow untransformed EBY100, EBY100/pYD1, and EBY100/pYD1 containing your gene of interest as follows.
1. Inoculate a single yeast colony into 10 ml YNB-CAA containing 2% glucose and grow overnight at 30°C with shaking.
2. Read the absorbance of the cell culture at 600 nm. The OD600 should be between 2 and 5. If the OD600 is below 2 or over 5, refer to the table below.
If the OD600 is.... 
Then you may...
below 2, 
either continue growing the cells until the OD600 reaches 2 or proceed to Step 3.
over 5, 
proceed to Step 3.<STRONG>Note</STRONG>: In general, OD600 readings less than 2 will decrease the number of cells displaying fusion proteins. OD600 readings greater than 5 will delay induction of expression of the displayed protein.
3. Centrifuge the cell culture at 3000-5000 x g for 5-10 minutes at room temperature.
4. Resuspend the cell pellet in YNB-CAA medium containing 2% galactose to an OD600 of 0.5 to 1. This is to ensure that the cells continue to grow in log-phase. For example, if the OD600 is 2 from Step 2, resuspend the cells in 20 to 40 ml of medium.
5. Immediately remove a volume of cells equivalent to 2 OD600 units. For an OD600 of 0.5, remove 4 ml and place on ice. This is your zero time point.
6. Incubate the cell culture at 20-25°C with shaking. <STRONG>Note</STRONG>: In general, more cells will display the protein fusion at 20°C. However, if you do not have an incubator that maintains 20°C, try 25°C.
7. Assay the cell culture over a 48-hour time period (i.e. 0, 12, 24, 36, 48 hours) to determine the optimal induction time for maximum display. For each time point, read the OD600 and remove a volume of cells that is equivalent to 2 OD600 units (see Step 5, above). Proceed to Staining of Displayed Proteins, below.<H3 style="color:#E3D434"><STRONG>Staining of Displayed Proteins</STRONG></H3>
                		For each time point, assay untransformed EBY100, EBY100/pYD1, and EBY100/pYD1 containing your gene of interest. Time points may be processed as they are collected or placed on ice and stored at +4°C until all time points are collected. Do not freeze cells.
1. Take your time points from Steps 4 and 6, above, and centrifuge at 3000-5000 x g for 5-10 minutes at +4°C.
2. Resuspend the cells in 1X PBS and centrifuge as in Step 1.
3. Remove the PBS and resuspend the cell pellet in 250 µl of 1X PBS, 1 mg/ml BSA, and FITC-Streptavidin (1:500 vol/vol) (Solarbio) .
4. Incubate on ice for 30 minutes with occasional mixing.
5. Centrifuge the cells at 3000-5000 x g for 5-10 minutes at +4°C.
6. Wash the cells thrice with 1X PBS.
7. Resuspend the cells in 700μ 1X PBS to divided into 96 plates, and the fluorescence values were measured by the microplate reader, or to sample, biological section, and observed with a fluorescence microscope.<H3 style="color:#E3D434"><STRONG>Co-culture</STRONG></H3>
                		1.Yeast cell was cultured with YPD medium for 48 h at 30◦C.
2.Then the yeast cells was transferred into YNB-CAA Gla medium as adjusted OD=1 by YNB-CAA Gla medium and cultured for 30h at 25 ◦C.
3.Then add <I>E.coli</I> into YNB-CAA Gla medium with OD=0.4，cultured for 12 h at 28 ◦C.
4.Then the cells can be used in observation by microscopy or FACS method.<H3 style="color:#E3D434"><STRONG>FACS </STRONG></H3>
                		1.After precultivation with appropriate selective medium for 18 h at 30◦C, yeast cells were harvested and washed with phosphate buffered saline (PBS) three times. 
2.The cell density was adjusted to OD600 = 1 by PBS. 
3.Then cells were incubated with 20 g/ml of streptavidin–FITC (Solarbio)for 1 h. 
4.After washing by PBS three times, cells were observed with FACS analysis.(Beckman)
5.FACS use as ‘Cytomics FC500 MCL with CXP software Instructions For Use’<H3 style="color:#E3D434"><STRONG>Microscopy</STRONG></H3>
                       1.After precultivation with YNB-CAA Gla medium for 12 h at 28◦C, yeast cells and <I>E.coli</I>were harvested and washed with phosphate buffered saline (PBS) three times. 
2.The cell density was adjusted to OD600 = 1 by PBS. 
3.Then cells were incubated with 20 g/ml of streptavidin–FITC (Solarbio)for 1 h. 
4.Then the cells were washed with phosphate buffered saline (PBS) three times. 
5.Then cells were incubated with 20 g/ml of Biotin-Rhodamine (ToYongBio)for 1 h. 
6.After washing by PBS three times, cells were observed with microscopy. </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid ouc-reserve"><P class="text-center">                     Contact Us : oucigem@163.com  |  ©2017 OUC IGEM.All Rights Reserved.  |  Based On Bootstrap</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>