Team:OUC-China/Notebook
Notebook
Toggle Navigation
Team
Members
Collaborations
Attributions
Project
Description
Design
Proof of concept
Basic fermentation Adhesion platform MINI-GRE
Demonstration
InterLab
Improvement
Notebook
Model
Parts
Safety
Human Practice
Overview
Silver
Gold and integrated
Engagement
Notebook
Week 1 2017.3.13-2017.3.19
1.Came to Peking University and Beijing University of Chemical Technology for project consultation
2.Determined our project:the degradation and utilization of enteromorpha
3.Determined our PI
4.Started to learn basic molecular experiments
5.Prepared experimental material
Week 2 2017.3.20-2017.3.26
1.Team registration were completed
2.The circuits, parts, related reagents and equipment of our basic part were identified
3.Tried to explore the pretreatment conditions
4.Started to learn the balance analysis algorithm and programming language for modeling
5.Planned a briefing session for recruitment of next years'team
6.Set up publicity platforms
Week 3 2017.3.27-2017.4.2
1.The yeast strain was identified:W303-1A
2.Came to QIBEBT for plasmid desigening and eukaryotic operation technology
3.Came to Shandong University for project consultation
4.Lab safety training for everyone
5.The recuiting started.Related posters and other propaganda had been prepared
Week 4 2017.4.3-2017.4.9
1.Strains were identified:The yeast is BY4741,and the E.Coli is KO11
2.Held the recuiting teach-in
3.Started to make co-culture experiment
4.Got the related plasmids pYC230
Week 5 2017.4.10-2017.4.16
1.Transformed and recovered pYC230
2.Activited KO11 and BY4741
3.Inoculated,transformed and cultured E.Coli with msa
4.The concept of co-culture between E.Coli and saccharomyces cerevisiae were prooved
Week6 2017.4.17-2017.4.23
1.The first draft of contract was finished
2.Comleted the budget
3.The wiki homepage was designd
4.Sent the primer
5.Held the recruiting interview
Week7 2017.4.24-2017.4.30
1.Sent the plasmids of basic part
2.Drawn the glucose concentration standard curve
3.Recovered plasmid containing J23106 from E.Coli and cellulase from PYC230
4.Publicity brochures was designed
Week8 2017.5.1-2017.5.7
1.Investigate and survey of Seawin Biotech Group
2.The MINI-GRE was tested by PCR
3.The gene of BirA was gotten form E.Coli by PCR
Week9 2017.5.8-2017.5.14
1.The yeast backbone pYC was tested by PCR
2.The inp and msa were tested by PCR
3.The birA was sent and synthesised
4.The inp and mas were connected by Gibson assemby
5.inp-msa-J23106 was enzyme linked and transformed into TOP10
6.Bira and pYC230-TP were connected by Gibson assemby and transformed into TOP10
7. MINI-GRE was testd by PCR and connected with pYC230 by Gibson assemby
8.Prepared for national entrepreneurship projects
Week10 2017.5.15-2017.5.21
1.Our team was accepted
2.pUG6 and pSH65-CRE were transformed
Week11 2017.5.22-2017.5.28
1.4 parts of MINI-GRE were tested by PCR
2.pYC230 was tested by PCR
Week12 2017.5.29-2017.6.4
1.INP-msA was transformed and recovered
2.The strain with cellulase was in conservation
3.Path of metabolism of xylose was transformed into yeast
4.Related genes synthesising resveratrol were sent to synthesise
5.BAP and pYD1d were sent to sequencing
6.The co-culture medium was made and inoculated
7.The first circuit of MINI-GRE:Pmini-Tcyc1 was successfully connect
Week13 2017.6.5-2017.6.11
1.Designed theprimers of BAP and prepared to synthesise
2.INP-msA were successfully connected and recovered
3.The result of co-cuture was not ideal
4.Related genes of resveratrol were synthesised
Week14 2017.6.12-2017.6.18
1.Applied for visa
2.The transform of BirA failed
3.Successfully connected Pcyc1+Tmini and Pmini+Tcyc1
4.The recovering of INP-msA failed
5.The first version of homepage of wiki uploded
Week15 2017.6.19-2017.6.25
1.Detection of cellulase by SDS-PAGE successed
2.The expression of xylose metabolism circuit successe
3.The iGEM kit was received
4.The first circuit of resveratrol synthesis finished
5.The transformation of MINI-GRE into yeast successed
Week16 2017.6.26-2017.7.2
1.As a result of  preparation for the final exam that we didn't do any experiments
Week17 2017.7.3-2017.7.9
1.Attended 2017 Synthetic Biology Young Schlar Forum in Shanghai
Week18 2017.7.10-2017.7.16
1.The first circuit of resveratrol synthesis was trasformed successfully
2.Determinated resveratrol standard curve by HPLC
3.Started working with Interlab and transformed related circuits
4.BirA connected successfully by Gibson assembly
5.Started connecting OmpA and CenA
Week 19 2017.7.17-2017.7.23
1. Add the restriction site on plasmid （pUG6-Tyhomo-TAL and pUG-Tyhomo-4CL-RS）by PCR.
2. Detection of interlab plasmid.
3. Try RNA extraction pre-experiments and perform Q-PCR pre-experiments.
4. DNS method to measure xylose content, failure.
5. Successfully spliced BirA to pYC230.
Week 20 2017.7.24-2017.7.30
1. Transform the plasmid used by interlab.
2. We cultured the four EBY100 strains with four different combinations of promoters and terminators.
3. We measured the fluorescence of yECitrine and OD600. We wanted to know the approximate growth curve and the expression strength of these strains.
4. Construct the INP-mSA part by Gibson assembly.
5. PCR the INP-mSA part.
Week 21 2017.7.31-2017.8.6
1. Connected the plasmid fragments with the enzyme (failure).
2. The first extraction of RNA from MINI-GRE.
3. We began to study RNA extraction experiment and RT-PCR experiment. But at the first time, we made some mistakes resulting in the total RNA volumes had a big difference between the three biological repetitions. We learned a lot from the first try, and became more careful in the following experiments.
4. Link the INP-mSA part with pSB1C3 backbone with J23106 promoter and transform the recombine plasmid into DH5α strain.
5. By testing OD600 method to verify the xylose strain, in line with expectations, success.
Week 22 2017.8.7-2017.8.13
1. Connected the plasmid fragments with the enzyme (failure).
2. Logarithmic yeast RNA extraction from MINI-GRE.
3. Construct the BirA-pYC230 circuit by T4 DNA ligase and transform into EBY100 strain.
Week 23 2017.8.14-2017.8.20
1. Connected the plasmid fragments with the enzyme (failure).
2. Yeast RNA extraction, and Q-PCR experiments.
3. We measured the fluorescence and OD600 for the second time, and we also did RT-PCR experiments. All the results were in accord with our expectation.
4. BAP-pYD1 circuit synthesied.
5. The fermentation experiment of xylose strain was carried out and sampled regularly.
Week 24 2017.8.21-2017.8.27
1. Extract membrane protein of BirA-pYC230-BAP-pYD1-EBY100 and J23106-INP-mSA-pSB1C3-DH5α strain.
2. Do the experiment of SDS-PAGE and WesternBlot to detect the expression of mSA and biotin in membrane using HRP-biotin.
3. The xylose fermentation experiment was carried out by microplate reader, FPLC and SBA biosensor. Respectively, the successful detection of OD600, xylose and ethanol content changes.
Week 25 2017.8.28-2017.9.3
1. First interlab fluorescence measurement.
2. Began to explore the proper condition of co-culture.
Week 26 2017.9.4-2017.9.10
1. Make pUG6-Tyhomo-TAL and pUG-Tyhomo-4CL-RS become to linearized plasmids by PCR.
2. Adjust the parameters for interlab fluorescence measurements and measure the fluorescence again.
3. Acquire the genome of clostridium cellulovorans and PCR cellulase from it, however this strain’s genome doesn’t have this enzyme.
4. Fiber disaccharide gene splice, failed.
Week 27 2017.9.11-2017.9.17
1. Transfer tal, 4cl-RS linear plasmids into yeast EBY100.
2. We constructed the part “Pmini”.
3. Begin cell staining with FITC-streptavidin to show the function of biotin surface display system.
4. Fiber disaccharide circuit splicing completed half.
Week 28 2017.9.18-2017.9.24
1. Extraction of genome and PCR detection of whether integration is successful (failure) .
2. Submit interlab data to HQ and complete interlab related google form.
3. Use the Rodanmine B-biotin to ensure the mSA expression curve.
4. Fiber disaccharide gene circuit splicing completed.
5. Xylose gene standardization.
Week 29 2017.9.25-2017.10.1
1. Extraction of genome and PCR detection of whether integration is successful (failure).
2. Start building improvement-related plasmids.
3. We constructed the part “Tmini”.
4. Compare the different temperature and determine a proper temperature to do the strain staining.
5. The cellobiose plasmid is transformed into yeast.
Week 30 2017.10.2-2017.10.8
1. Fermented EBY100 contains gene 4CL-RS,and detected resveratrol by HPLC (failed).
2. Doing a FACS observation to show the percent of expression of biotin.
3. The use of yeast for cellobiose fermentation experiments.
Week 31 2017.10.9-2017.10.15
1. Fermented EBY100 contains gene 4CL-RS,and detected resveratrol by HPLC (failed).
2. The plasmid was constructed about improvement and the fluorescence test was started.
3. Do the FITC-streptavidin and Rodanmine-B-biotin staining at the same time, and use the CLSM observation to detect the connection between 2 strains.
4. Detection of cellobiose fermentation experiment using microplate reader, FPLC and SBA biosensor. Respectively, the successful detection of OD600, cellobiose, glucose and ethanol content changes. congratulations！
Week 32 2017.10.16-2017.10.22
1. Co-fermented EBY100 contain gene 4CL-RS and TAL,and detected resveratrol by HPLC (failed).
2. Improvement related work done.
3. We constructed the part “yECitrine”.
4. Do the TEM and SEM observation to have a visual result of the connection between 2 strains.
5. Try to construct the circuit OmpA-CenA-pSB1C3 but found it can not work normally.
6. Standardization of the cellobiose gene.
Week 33 2017.10.23-2017.10.29
1. Co-fermented EBY100 contain gene 4CL-RS and TAL,and detected resveratrol by HPLC (failed).
2. The standardized plasmid was dispensed into a 96-well plate and sent to the GenScript.
Contact Us : oucigem@163.com  |  ©2017 OUC IGEM.All Rights Reserved.  |  Based On Bootstrap
