<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_OUC-China_proof3 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:OUC-China/proof3</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_content_wrapper"><TITLE>proof3</TITLE><DIV class="ouc-guide" style="height: 50px;background-color:#008EA1 ;"><DIV class="row"><DIV class="col-md-2 navbar-header" style="text-align: center; padding: 0"><BUTTON class="navbar-toggle" type="button" data-toggle="collapse" data-target=".navbar-responsive-collapse"><SPAN class="sr-only">Toggle Navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-responsive-collapse" style="padding: 0"><DIV class="col-md-1 ouc-navbar" style="padding: 0"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Team</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/Team" style="line-height: 40px; " class="ouc-navdown">Members</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Collaborations" style="line-height: 40px; " class="ouc-navdown">Collaborations</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Attributions" style="line-height: 40px; " class="ouc-navdown">Attributions</A></DIV></DIV></DIV><DIV class="col-md-1 ouc-navbar" style="padding: 0"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Project</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/Description" style="line-height: 40px; " class="ouc-navdown">Description</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Design" style="line-height: 40px; " class="ouc-navdown">Design</A></DIV><DIV class="ouc-right"><A style="line-height: 40px; font-size: 15px;" class="ouc-navdown" href="https://2017.igem.org/Team:OUC-China/proof1">Proof of concept</A><DIV class="ouc-rightnav"><A href="https://2017.igem.org/Team:OUC-China/proof1" style="display: block">Basic fermentation</A><A href="https://2017.igem.org/Team:OUC-China/proof2" style="display: block">Adhesion platform</A><A href="https://2017.igem.org/Team:OUC-China/proof3" style="display: block">MINI-GRE</A></DIV></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Demonstrate" style="line-height: 40px; " class="ouc-navdown">Demonstration</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/InterLab" style="line-height: 40px; " class="ouc-navdown">InterLab</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Improve" style="line-height: 40px; " class="ouc-navdown">Improvement</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Notebook" style="line-height: 40px; " class="ouc-navdown">Notebook</A></DIV></DIV></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Model" style="line-height: 50px;text-align: center" class="ouc-nav">Model</A></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Parts" style="line-height: 50px;text-align: center" class="ouc-nav">Parts</A></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Safety" style="line-height: 50px;text-align: center" class="ouc-nav">Safety</A></DIV><DIV class="col-md-2 ouc-navbar" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Engagement" style="line-height: 50px;text-align: center" class="ouc-nav">Human Practice</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/HumanPractice" style="line-height: 40px; " class="ouc-navdown">Overview</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/HP/Silver" style="line-height: 40px; " class="ouc-navdown">Silver</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/HP/Gold_Integrated" style="line-height: 40px; " class="ouc-navdown">Gold and integrated</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Engagement" style="line-height: 40px; " class="ouc-navdown">Engagement</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="page-header ouc-page-header"><H1><STRONG>Proof of concept——Mini GRE</STRONG></H1></DIV><H3 style="color: #66BCC7"><STRONG>Circuit construction</STRONG></H3><P style="font-size: 20px">
    	To explore the feasibility of MINI-GRE by combination of promoters and terminators as we mentioned above, we designed four promoter-terminator pairs, and constructed four different report circuits for them (fig. A)
    </P><P style="font-size: 20px">
    	For circuit 1, we pair promoter CYC1 with terminator CYC1, which are among the most commonly used native promoters and terminators and also have a relative medium strength in yeast.[4] For circuit 2, promoter CYC1 is paired with terminator MINI. The MINIp-CYC1t and MINIp-MINIt, respectively, serves as the chosen pair for circuit 3 and 4.    </P><P style="font-size: 20px">
    	For convenience, we named the“CYC1p-yECitrine-CYC1t-mStrawberry-CYC1t”as“CC”,“CYC1p-yECitrine-MINIt-mStrawberry-CYC1t”as“CM”,“MINIp-yECitrine-CYC1t-mStrawberry-CYC1t”as“MC”, and“MINIp-yECitrine-MINIt-mStrawberry-CYC1t” as “MM”,hereafter. 
    </P><P style="text-align: center; color: gray">
    	Fig. A the plasmid map of our circuit CC, CM, MC, MM. The CC circuit includes the commonly used native promoter CYC1 and terminator CYC1. The MM circuit includes the combination of MINI promoter and MINI terminator. 
    </P><H3 style="color: #66BCC7"><STRONG>Results</STRONG></H3><P style="font-size: 20px">
    	For each circuit, the strength of promoters was characterized by yECitrine, a kind of yellow fluorescent protein was used to detect the output level of particular promoter, terminator or promoter-terminator pair. And the red fluorescent signal from RFP mStrawberry can represent the relative read-through efficiency of particular terminator in the circuits including the same promoter.
    </P><P style="font-size: 20px">
          While monitoring the growth rate of four strains containing different expression regulatory devices, we also characterized the expression strength of the promoter-terminator pairs by detecting the fluorescence intensity of yECitrine in each circuit. As we can see in the bar chart, the ratio and relationship of the signals from 4 circuits become relatively stable at the early stationary phase, which hints that the expression level may reach the dynamic steady state at the time point 22 hours. And the results from 22 hours point also match the strength relationship of the two promoters in previous research, although we used another yeast strain here. [2, 5] Therefore, comparing with the mid-log phase data, we tend to believe that this results can reflect the true dynamic characteristics of the genetic regulatory devices, although we will research this phenomenon in our future work.(Fig. D)

    </P><P style="font-size: 20px">
    	So, in order to better reflect the dynamic behaviors of the circuits we tend to use data from mid-log phase during the growth process.
    </P><P style="text-align: center; color: gray">
    	Fig. B The growth curve of the four strains with different promoter- terminator pairs. Error bars represent standard deviation of three biological replicates. 

    </P><P style="text-align: center; color: gray">Fig. C The fluorescence/Abs600 in different time. Error bars represent standard deviation of three biological replicates.
    </P><P style="text-align: center; color: gray">Fig.D The fluorescence/Abs600 of strains with different promoter-terminator pairs, after being cultivating for 22 hours. Error bars represent standard deviation of three biological replicates.
    </P><H3 style="color: #66BCC7"><STRONG>Function Verification of promoters and terminators</STRONG></H3><P style="font-size: 20px">
    	Through comparing “CC” with “CM”, we can learn that the Fluorescence/Abs600 of “CM”is nearly three times of “CC”, proving that the strength of MINI terminator is higher than that of CYC1 terminator. (Fig. D)(At first we think of measuring this characteristic by comparing yECitrine fluorescence /mStrawberry fluorescence. However, considering that the fluorescence of mStrawberry is too low, we cannot expect an accurate result due to the high error rate. ) We assume that both MINI terminators and CYC1 terminators can effectively stop the transcription, so the mRNA of mStrawberry generated behind these terminators can hardly be detected, neither does the fluorescence of mStrawberry.  
 

    </P><P style="font-size: 20px">
    	What’s more, the Fluorescence/Abs600 of “MM”is also nearly 3-fold compared with “MC”, proving that the strength of MINI promoter is higher than CYC1 promoter. 


    </P><P style="font-size: 20px">
    	In addition, when we compared “MM” with “CC” which is the commonly used promoter-terminator pair, we found that the difference is nearly 6-fold. (Fig. D)
    </P><P style="font-size: 20px">
    	From our results, we confirmed that the MINI promoter and MINI terminator do be superior to CYC1 promoter and CYC1 terminator, not only in length but also in strength. (Fig. E)

    </P><P style="text-align: center; color: gray">Fig. E Length and output level of 4 different promoter–terminator combination. Error bars represent standard deviation of three biological replicates
    </P><H3 style="color: #66BCC7"><STRONG>Robustness in different host</STRONG></H3><P style="font-size: 20px">
    	In order to verify that the MINI-GRE is able to work with the same rule in a variety of yeast strains and expend the application range of these MINI regulatory devices, we invited other teams to repeat the same experiments in different yeast strains and experimental environment, which can also be an important part of our collaboration. 

    </P><P style="font-size: 20px">
    	The time point of the data collection is early stationary phase of yeast growth. The yeast strains we used with Nanjing-China were Saccharomyces cerevisiae EBY100. The yeast strain used in Tianjin was synX, the yeast strain with chemical synthetic chromosome. 
    </P><P style="font-size: 20px">
    	From the result of other colleges, we can know that our MINI-GRE can also work in other yeast strains. (Fig. F)
    </P><P style="text-align: center; color: gray">Fig. F The fluorescence/Abs600 of strains with different promoter-terminator pairs in the late  from three teams, OUC-China, Tianjin and Nanjing-China(left to right). Error bars represent standard deviation of three biological replicates.
    </P><H3 style="color: #66BCC7"><STRONG>Transcription Level Assay by qPCR</STRONG></H3><P style="font-size: 20px">
    	Moreover, we run qPCR towards corresponding protein in order for a further validation of different promoter-terminator combination’s expression on a post-transcriptional level. 

    </P><P style="font-size: 20px">
    	The result from qPCR assay shows that at the 22nd hour from setting up culture, the circuits reached the highest expression intensity and the expression level of four circuits is shown below. 

    </P><P style="text-align: center; color: gray">Fig. G The relative transcription level of yECitrine and mStrawberry at the 22nd hour from setting up culture. Error bars represent standard deviation of three biological replicates.

    </P><P style="font-size: 20px">
    	However, the transcript level of yECitrine doesn’t matched the fluorescence very well. We infer that it may be caused by the experimental error of the total mRNA concentration and the little mismatch of the primers. 

    </P><P style="font-size: 20px">
    	Luckily, we can still learn that the transcript level of mStrawberry is very low, which means the transcriptional read through of both CYC1t and MINI terminator can be overlooked; they can efficiently terminate the transcription, which confirmed the assumption before. 
    </P><H3 style="color: #66BCC7"><STRONG>Conclusion</STRONG></H3><P style="font-size: 20px">
    	To draw a conclusion, from our experiments, we confirmed the superiority of the MINI-GRE (MINI promoter-MINI terminator pair), which can be summed into three aspects: 
    </P><P style="font-size: 20px">
    	(1) Short but strong, which decreases the possibility of undesired homologous reorganization and provide significant output strength compared with commonly used promoter-terminator pair.
    </P><P style="font-size: 20px">
    	(2) Can work in different yeast strains, which can provide robust function for extensive application.
    </P><P style="font-size: 20px">
    	(3) Has good modularity because of the low transcriptional read through efficiency from minimal terminator, which is an important characteristic in synthetic biology. 
    </P><H3 style="color: #66BCC7"><STRONG>Future work</STRONG></H3><P style="font-size: 20px">
    	Although we have only constructed one MINI promoter-terminator pair,  we are able to do more in the future, which includes:
    </P><P style="font-size: 20px">
    	(1)Establish a library of similar pairs on its basis
    	(2)Study the relationship between transcribtion strength and pair strength
    	(3)Enlarge its function into a toolbox.
    </P><H3 style="color: #66BCC7"><STRONG>Reference</STRONG></H3><P>
    	[1]Smanski, M. J. et al. Functional optimization of gene clusters by combinatorial design and assembly. Nat. Biotechnol. 32, 1241 (2014).
 [2]Redden H, Alper H S. The development and characterization of synthetic minimal yeast promoters[J]. Nature Communications, 2015, 6:7810.
[3]Yamanishi, M., Katahira, S., Matsuyama, T., 2011. TPS1 terminator increases mRNA and protein yield in a Saccharomyces cerevisiae expression system. Biosci. Biotechnol. Biochem. 75, 2234–2236.
[4]Curran K A, Karim A S, Gupta A, et al. Use of expression-enhancing terminators in Saccharomyces cerevisiae, to increase mRNA half-life and improve gene expression control for metabolic engineering applications[J]. Metabolic Engineering, 2013, 19:88.
[5]Curran K A, Morse N J, Markham K A, et al. Short Synthetic Terminators for Improved Heterologous Gene Expression in Yeast[J]. Acs Synthetic Biology, 2015, 4(7):824.</P><DIV class="panel-group" id="accordion"><DIV class="panel "><DIV class="panel-heading ouc-panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse3"><STRONG>PROTOCOL</STRONG></A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse "><DIV class="panel-body"><H3 style="color:#E3D434"><STRONG>Reserve transcription</STRONG></H3><STRONG>Material</STRONG>
PrimeScript<SUP>™</SUP>RT reagent Kit with gDNA Eraser (TaKaRa Code No. RR047A)<STRONG>Procedure</STRONG>
1. Genomic DNA elimination reaction
1) Prepare the genomic DNA elimination reaction solution on ice.
2) Add RNA template with the suitable amount of the master mix to a PCR tube.<TABLE class="table" style="width: 500px"><THEAD><TR><TH>Reagent</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>5X gDNA Eraser Buffer</TD><TD>2.0 μl</TD></TR><TR><TD>gDNA Eraser</TD><TD>1.0 μl</TD></TR><TR><TD>Total RNA</TD><TD>1.0 μl(Up to 1 μg of total RNA)</TD></TR><TR><TD>RNase Free dH<SUB>2</SUB>O</TD><TD>Up to 10.0 μl</TD></TR></TBODY></TABLE>
                3) Run the program: 42℃ 2 min
2. 4℃ Reverse-transcription reaction
1) Prepare the reverse-transcription reaction solution on ice.
2) Add Reaction solution from Step 1 with the suitable amount of the master mix to a PCR tube<TABLE class="table" style="width: 500px"><THEAD><TR><TH>Reagent</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Reaction solution from Step 1</TD><TD>10.0 μl</TD></TR><TR><TD>5X PrimeScript Buffer 2 (for Real
Time)</TD><TD>4.0 μl</TD></TR><TR><TD>RT Primer Mix</TD><TD>4.0 μl</TD></TR><TR><TD>PrimeScript RT Enzyme Mix I</TD><TD>1.0 μl</TD></TR><TR><TD>RNase Free dH<SUB>2</SUB>O</TD><TD>4.0 μl</TD></TR></TBODY></TABLE>
               	3) Run the program:
37℃15 min
85℃ 5 sec
4℃	<H3 style="color:#E3D434"><STRONG>Total RNA Extraction</STRONG></H3><STRONG>Material</STRONG>
RNAiso Plus(Takara Co.9109)<STRONG>Procedure</STRONG>
1. Cleavage of yeast cells.
1） Take 1 ml OD600 value of 1.5 to 2.5 liquid culture yeast to 1.5 ml Microtube and 8,000 g at 4 ° C for 2 min.
2） carefully discard the supernatant, slowly add 1 ml of ice to the precipitation of sterile water, with a pipette gently blowing, so that precipitation resuspended.
3） 8,000 g at 4 ° C for 2 min. Carefully discard the supernatant, as far as possible in addition to net liquid.
4） Add 0.4 ml of Yeast RNAprep Buffer to the precipitate and gently re-blow with a pipette to resuspend the pellet.
5） into the 30 ° C water bath for 1 hour, during which gently shake the centrifuge tube 1 or 2 times.
6） Remove the centrifuge tube from the 30 ° C water bath and centrifuge at 12,000 g for 4 minutes at 4 ° C.
7） Carefully discard the supernatant, add 1 ml of RNAiso Plus to the precipitate, gently blow with a pipette to resuspend the pellet.
8） cover the centrifuge tube cover, whirlpool oscillation 2 to 5 minutes to clarify the suspension. 12,000 g at 4 ° C for 5 min.
9） Carefully aspirate the supernatant and move into a new 1.5 ml RNase-free Microtube (do not touch the precipitate).
2.Collection
1) Collect and pipettes 1.5~2.5ml bacteria which is in the log phage(usually when OD600=1.0) into a centrifuge tube. Centrifuge tube for 8,000×g,5 minutes at 4°C.Discard supernatant and be care not to disturb the bacteria pellet.
2) Add 1ml of RNAiso Plus, pipette up and down until pellet is completely resuspended.
3) Leave at room temperature(15~30°C) for 5 minutes, isolate the RNA from the nuclear protein.
2.Extracion of total RNA
1) Add 200 ul chloroform, cap the centrifuge tube and mix until the solution becomes milky. 
2) Keep the solution at room temperature for 5 minutes. 
3) Centrifuge at 12,000×g for 15 minutes at 4°C.Centrifuging the solution will separate it into three layers; liquid top layer(contains RNA),semisolid middle layer(mostly DNA),and bottom organic solvent layer. 
4) Transfer the top liquid layer to new centrifuge tube without touching middle layer. 
5) Measure the amount of the top layer and add an equal amount or add up to 0.5 times of isopropanol of the top layer. Mix together well. Keep the mixture at room temperature for 10 minutes. 
6) Centrifuge at 12,000×g for 10 minutes at 4°C to precipitate the RNA. 
7) Cleaning RNA precipitate 
8) Carefully remove the supernatant, do not touch the pellet.  9) Add an amount of 75% cold ethanol that was equivalent to the supernatant. Clean the precipitate by vortexing.
10) Centrifuge the solution at 7,500×g for 5 minutes at 4°C and discard supernatant. Be care not to disturb the precipitate.
4.Dissolving RNA
Dry the precipitate by leaving the tube open for several minutes. After the precipitate is dry, dissolved it with appropriate amount of RNase-free water.<STRONG>Attention</STRONG>
Make sure that all the centrifuge tubes and pipettes have been treated with DEPC.   <H3 style="color:#E3D434"><STRONG>Quantitative Real-time PCR</STRONG></H3><STRONG>Material</STRONG>
SYBR® Premix Ex Taq™ (Tli RNaseH Plus) (TaKaRa(Code No. RR420A)<STRONG>Procedure</STRONG>
1. Prepare the PCR mixture shown below<TABLE class="table" style="width: 500px"><THEAD><TR><TH>Reagent</TH><TH>Volume</TH><TH>Final conc.</TH></TR></THEAD><TBODY><TR><TD>SYBR Premix Ex Taq (Tli
RNaseH Plus) (2X)</TD><TD>10 μl</TD><TD>1X</TD></TR><TR><TD>PCR Forward Primer (10 μM)</TD><TD>0.4 μl</TD><TD>0.2 μM</TD></TR><TR><TD>PCR Reverse Primer (10 μM)</TD><TD>0.4 μl</TD><TD>0.2 μM</TD></TR><TR><TD>Template (&lt; 100 ng)</TD><TD>2 μl</TD></TR><TR><TD>dH2O (sterile distilled water)</TD><TD>7.2 μl</TD></TR><TR><TD>Total</TD><TD>20 μl</TD></TR></TBODY></TABLE>
               	
               	2. Start the reaction using LightCycler 480 System
1) Denature:
95℃ 30 sec. (Ramp rate: 4.4℃/sec.)
1 cycle
2) PCR :
95℃ 5 sec. (Ramp rate: 4.4℃/sec.)
60℃ 30 sec. (Ramp rate: 2.2℃/sec.)
40 cycles
3) Melting
95℃ 5 sec. (Ramp rate: 4.4℃/sec.)
60℃ 1 min. (Ramp rate: 2.2℃/sec.)
95℃ (Ramp rate: 0.11℃/sec.)
1 cycle
4) Cooling
50℃ 30 sec. (Ramp rate: 2.2℃/sec.)
1 cycle
3. After the reaction is complete, check the amplification and melting curves and plot a standard curve if absolute quantification will be performed.
                	
                	
                </DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid ouc-reserve"><P class="text-center">
                          Contact Us : oucigem@163.com  |  ©2017 OUC IGEM.All Rights Reserved.  |  Based On Bootstrap</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>