<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_OUC-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:OUC-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_content_wrapper"><P><TITLE>ProofOfConcept</TITLE></P><DIV class="ouc-guide" style="height: 50px;background-color:#008EA1 ;"><DIV class="row"><DIV class="col-md-1 ouc-navbar"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Team</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/Team" style="line-height: 40px; " class="ouc-navdown">Members</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Collaborations" style="line-height: 40px; " class="ouc-navdown">Collaborations</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Attributions" style="line-height: 40px; " class="ouc-navdown">Attributions</A></DIV></DIV></DIV><DIV class="col-md-1 ouc-navbar"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Project</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/Description" style="line-height: 40px; " class="ouc-navdown">Description</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Design" style="line-height: 40px; " class="ouc-navdown">Design</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Results" style="line-height: 40px; font-size: 15px;" class="ouc-navdown">Proof of concept</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Demonstrate" style="line-height: 40px; " class="ouc-navdown">Demonstrate</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/InterLab" style="line-height: 40px; " class="ouc-navdown">InterLab</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Improve" style="line-height: 40px; " class="ouc-navdown">Improve</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Notebook" style="line-height: 40px; " class="ouc-navdown">Notebook</A></DIV></DIV></DIV><DIV class="col-md-1"><A href="https://2017.igem.org/Team:OUC-China/Model" style="line-height: 50px;text-align: center" class="ouc-nav">Model</A></DIV><DIV class="col-md-1"><A href="https://2017.igem.org/Team:OUC-China/Parts" style="line-height: 50px;text-align: center" class="ouc-nav">Parts</A></DIV><DIV class="col-md-1"><A href="https://2017.igem.org/Team:OUC-China/Safety" style="line-height: 50px;text-align: center" class="ouc-nav">Safety</A></DIV><DIV class="col-md-2 ouc-navbar"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Human Practice</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/HumanPractice" style="line-height: 40px; " class="ouc-navdown">Overview</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/HP/Silver" style="line-height: 40px; " class="ouc-navdown">Silver</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/HP/Gold_Integrated" style="line-height: 40px; " class="ouc-navdown">Gold and integrated</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Engagement" style="line-height: 40px; " class="ouc-navdown">Engagement</A></DIV></DIV></DIV><DIV class="col-md-2 ouc-navbar" style="font-size: 15px"><DIV class="ouc-vicedown" style="padding-top: 50px;"><LI style="list-style: none;margin: 0;padding: 0"><UL style="margin: 0;padding: 0"><A href="#fermentation"><STRONG>Basic fermentation</STRONG></A></UL><UL style="margin: 0;padding: 0"><A href="#pre">Pretreatment</A></UL><UL style="margin: 0;padding: 0"><A href="#yeasta">Yeast A</A></UL><UL style="margin: 0;padding: 0"><A href="#yeastb">Yeast B</A></UL><UL style="margin: 0;padding: 0"><A href="#adhesion"><STRONG>Adhesion platform</STRONG></A></UL><UL style="margin: 0;padding: 0"><A href="#ecoli"><I>E.coli</I> construction</A></UL><UL style="margin: 0;padding: 0"><A href="#yeast">Yeast construction</A></UL><UL style="margin: 0;padding: 0"><A href="#co">Co-cultivation</A></UL><UL style="margin: 0;padding: 0"><A href="#linkage">Linkage</A></UL><UL style="margin: 0;padding: 0"><A href="#mini"><STRONG>Mini system</STRONG></A></UL><UL style="margin: 0;padding: 0"><A href="#circuit">Circuit construction</A></UL><UL style="margin: 0;padding: 0"><A href="#function">Function verify</A></UL><UL style="margin: 0;padding: 0"><A href="#level">Transcription level</A></UL></LI></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="page-header ouc-page-header"><H1><STRONG>Proof of concept</STRONG></H1></DIV><H2 class="ouc-heading" id="fermentation"><STRONG>Basic fermentation</STRONG></H2><P style="font-size: 20px">
    	We first powdered the algae, removed the lignin and then treated remains with the enzyme solution to produce cellobiose and xylose. 
In order to engineered the yeast for utilizing cellobiose and xylose as carbon resources, we cloned relative genes to a shuttle plasmid in <I>E. coli</I> cell, then transformed it to yeast. We prepared SC medium  containing xylose and cellobiose as the only carbon resources, so that we can verify that we have already cultivated the strain which can grow with merely cellobiose and xylose as their carbon sources. If our novel strains grow significantly better than the negative control (which does not have the gene <I>GH-1, CDT-1</I>, or <I>XYL1, XYL2</I>) we can prove that our ideas are feasible and the genetic circuit works well. 
At the same time, we used HPLC to detect the concentration changing of cellobiose and xylose, and got more data to support our idea that the engineered yeast can utilize cellobiose and xylose as carbon sources. If the concentration of xylose/cellobiose decrease with time. SBA biosensor can detect the reaction process of substrate by immobilized enzyme as electrochemical signal, which supports us to describe a curve reflecting the ethanol generation under culture condition. Here, we prove that two engineered yeast strains can utilize cellobiose and xylose to produce ethanol directly.
    </P><DIV class="container"><H3 style="color: #66BCC7" id="pre"><STRONG>Pretreatment</STRONG></H3><P style="font-size: 20px">
    		1.<I>Enteromorpha</I> physical pretreatment.
2.Treat the residue with 0.2% H2O2 to remove the lignin.
3.Preparation of the enzymatic hydrolysis solution of <I>Enteromorpha</I> fiber.
4. The validation of xylose and cellobiose content after pretreatment.</P><DIV class="row"><DIV class="col-md-4" style="text-align: center;color: gray;">Fig 1.1 <I>Enteromorpha</I> Powder .
    		</DIV><DIV class="col-md-4" style="text-align: center;color: gray;">Fig 1.2 Treat the residue with 0.2% H2O2. 
    		</DIV><DIV class="col-md-4" style="text-align: center;color: gray;">Fig 1.3 Enzymatic hydrolysis solution of <I>Enteromorpha</I> fiber.
    		</DIV></DIV><H3 style="color: #66BCC7" id="yeasta"><STRONG>Yeast A--xylose fermenting strain</STRONG></H3><DIV class="container"><H4 style="color: #E3D434"><STRONG>Circuit construction</STRONG></H4><P style="font-size: 20px">
    			We used pYC230 provided by our PI, as the backbone and constructed xylosidase gene <I>XYL1</I> and <I>XYL2</I> by Gibson Assembly. Gene <I>XYL1</I> encoding xylose reductase (XR) which reduces D-xylose into xylitol, and facilitates xylose assimilation in yeast. Gene <I>XYL2</I> encoding xylitol dehydrogenase (XDH). Xylitol is oxidized to xylulose under the action of xylitol dehydrogenase, which relies on Nicotinamide adenine dinucleotide (NAD+).
    		</P><H4 style="color: #E3D434"><STRONG>Selective Medium Assay</STRONG></H4><P style="font-size: 20px">
    			We transformed the pYC230nto <I>S.  cerevisiae</I> EBY100 and got the xylose-utilizing strain successfully. We measured the growth rate of both our engineered strain and the negative control. If there is an obvious superiority of our new strain in xylose utilization, we can prove that the circuit works as our design. As a result, the curve well demonstrates that the strain with xylose-utilizing circuit has a higher OD600 than the negative control strain during the entire cultivation period. The culture of engineered yeast (EBY100 with XDH-XR) reaches stationary phases after 40 hours cultivation with an OD600 of around 2.35. While the control strain(EBY100) has an almost horizontal OD curve.
    		</P><DIV style="text-align: center; color: gray">Fig 1.4
    		</DIV><H4 style="color: #E3D434"><STRONG>Proof of function</STRONG></H4><P style="font-size: 20px">
    			We use HPLC to detect the concentration changing of xylose, getting extra data to support our idea directly that our engineered yeast can utilize the xylose as carbon sources and the concentration of xylose decrease with time in our culture system.
    		</P><P style="font-size: 20px">
    			The following chart shows the dynamic curve of xylose content of both the recombinant strain and negative control.
    		</P><DIV style="text-align: center; color: gray">Fig 1.5
    		</DIV><P style="font-size: 20px">
    			It is obvious that for our xylose-utilize strain, xylose content decreases as time goes, by while for the negative strain, the xylose content stays steady, indicating the disability of using xylose.
    		</P><P style="font-size: 20px">
    			For the final goal of our project, we ultimately detected the production of ethanol. Here, we choose to use the SBA-Biosensor. Because it can detect the reaction of substrate catalyzed by the immobilized enzyme as an electrochemical signal, which supports us to describe a curve reflecting the ethanol change in the culture condition, and is able to give the output instantly and conveniently.
    		</P><P style="font-size: 20px">
    			The following chart shows the ethanol content from both our recombinant strain and negative control.  
    		</P><DIV style="text-align: center; color: gray">Fig 1.6
    		</DIV><P style="font-size: 20px">
    			Gladly, the curve of xylose-consuming strain goes gradually up along with cultivating hours, and reach the plateau at around 90 hours, which is consist with the xylose consuming curve, indicating that our strain does produce ethanol on the basis of xylose, thus realizes our design and proves the concept of basic fermentation part.
    		</P><P style="font-size: 20px">
    			This three-in-one chart demonstrates our evidence. The left chart belongs to recombinant strain that can utilize xylose and the right chart belongs to the negative control.
    		</P><DIV class="row"><DIV class="col-md-6" style="text-align: center;color: gray;">Fig 1.7
    		</DIV><DIV class="col-md-6" style="text-align: center;color: gray;">Fig 1.8
    		</DIV></DIV></DIV><H3 style="color: #66BCC7" id="yeastb"><STRONG>Yeast B--cellobiose fermenting strain</STRONG></H3><DIV class="container"><H4 style="color: #E3D434"><STRONG>Circuit construction</STRONG></H4><P style="font-size: 20px">
    			We use pYC230 provided by our PI as our backbone and integrate cellubiose related gene <I>CDT</I> and <I>GH-1</I> through Gibson. <I>CDT</I> encoding a cellubiose transporter, which assimilate cellubiose into intracellular,and <I>GH-1</I> encoding β-glucosidase, which capable of hydrolyzing cellubiose into glucose.
    		</P><DIV style="text-align: center; color: gray">Fig 1.9
    		</DIV><P style="font-size: 20px">
    			The bright band in the AGE result shows that we successfully get the fragments of <I>GH-1</I>,<I>CDT-1</I> through PCR. The following experiment includes splicing the fragments onto the backbone pYC230.
    		</P><H4 style="color: #E3D434"><STRONG>Strain construction</STRONG></H4><P style="font-size: 20px">
    			We transformed the plasmid into <I>S.cerevisiae</I> EBY100 and got the cellubiose-utilize strain successfully then test the growth rate of the of both our recombinant strain and negative control.
    		</P><H4 style="color: #E3D434"><STRONG>Proof of function</STRONG></H4><P style="font-size: 20px">
    			Using HPLC we detect the changing concentration of cellubiose. The result shows that the concentration of cellubiose decrease with time, indicating a well utilization of cellubiose.
    		</P><P style="font-size: 20px">
    			The following chart shows of both our recombinant strain(left) and negative control(right).
    		</P><P style="font-size: 20px">
    			To examine the production of ethanol, we use the SBA-Biosensor, the same as in the xylose pathway.
    		</P><P style="font-size: 20px">
    			The following chart shows the cellubiose content, ethanol content, and  growth rate of both our cellubiose recombinant strain (left) and negative control (right).
    		</P><DIV class="row"><DIV class="col-md-6" style="text-align: center;color: gray;">Fig 1.10
    		</DIV><DIV class="col-md-6" style="text-align: center;color: gray;">Fig 1.11
    		</DIV></DIV><P style="font-size: 20px">
    			When characterizing the growth rate of recombinant cells utilizing cellubiose with OD600, we found a platform stage in the middle of the curve suggesting probably a secondary growth. We figure out a hypothesis that the accumulation of the glucose when making use of cellubiose might be the reason underlying this phenomenon because a substitution process of the carbon sources can exactly lead to such a plateau in the growth curve. [1]Therefore, we detect the glucose content of the medium after that to validate our assumption.
    		</P><DIV class="row"><DIV class="col-md-6" style="text-align: center;color: gray;">Fig 1.12
    		</DIV><DIV class="col-md-6" style="text-align: center;color: gray;">Fig 1.13
    		</DIV></DIV><P style="font-size: 20px">
   	        	The result (left) shows that glucose content in the medium reaches a peak at about 40 hours and then decrease rapidly. At around 70 hours, the content reaches the minimum and stay steady ever since. Considering the ethanol content (right) that also present a largest amount at around 80 hours, we believe that the glucose in yeast may be transformed gradually into acetic acid and ethanol participated in further metabolic procedure with acetic acid called Glucose Glycolysis, thus was turned into lipid.
   	        </P><P style="font-size: 20px">
    		And for a more visualized display, we use plate streaking to show the growth condition of our two recombinative strains and negative strains in different carbon source. We make three plates with either xylose, cellobiose or xylose plus cellobiose as sole or compound carbon source, respectively and monitor the growth of each strain in three different plate after cultivating for 32 hours at 30℃. As a result,yesat with pYC230-XYL1-XYL2 grows faster than the rest of two strains in 2% xylose as sole carbon plate, and cellobiose strain with pYC230-CDT-GH1 has a much faster growth rate than that of the other two strains in 2% cellobiose as sole carbon plate. These two recombinant strains also show a better growing condition than WT EBY100 with a medium of 2% xylose and 2% cellobiose as compound carbon source. This phenomenon indicates that we can actually utilize xylose and cellobiose from algae residue and produce ethanol with our engineered yeasts!
    	</P><P style="color: gray;text-align: center">Figure 2.1 recombinant strains with pYC230-XYL1-XYL2,pYC230-CDT-GH1 and WT EBY100 in different medium
    	</P></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel"><DIV class="panel-heading ouc-panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse1"><STRONG>PROTOCOL</STRONG></A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="container"><H3 style="color:#E3D434"><STRONG>Protocol about xylose pathway</STRONG></H3><STRONG>Experimental purpose</STRONG>
                		1.	Tested the growth difference between engineered strains which were introduced into the xylose pathway and negative strains. The utilization ability of xylose and the difference of ethanol content. Verify the availability of the line.
                		2. Calculated the growth curve，xylose change curve and ethanol change curve.of positive strains and negative strains in the xylose as the only carbon source medium.<STRONG>Materials</STRONG>
                		1.SC：0.67%YNB（2.814g+362ml）+ complete aa（42ml）  （404ml）
                        2.xylose：17ml water +8.5g   （0.5g/ml）
                        3.complete aa：10ml/100ml mother liquor
                        4.HPLC: C18 Pillars (80% PBS and 20% water as mobile phase)
                        5.BioTek Synergy H1 Automatic enzyme-linked immunosorbent assay systems;and 96-hole plate
                        6.SBAbiosensor<STRONG>The determination of xylose content</STRONG>
                        1.Use YPD to activate yeast, prepare xylose medium (2% xylose +sc=10ml+40ml)
                        2. Test OD600, the content of the xylose and the change of ethanol content 30°c 180rpm (negative, positive 4 bottles, initial od=1) 96 Hole Plate, 200ul sample/hole, 3 hole/sample; Sample, 2340rpm, 5min bacteria, 620u/times, save -80°c. Time of measurement and retention time See timetable
                        3. Using HPLC to detect the change of xylose in medium, the change of ethanol content was detected by SBA Biosensor.<STRONG>Method of adjusting OD value</STRONG>
                        1. measure the OD600 value of the bacterium liquid
                        2. Maintain the same volume as the original medium
                        3. 2340rpm, 5min, discard supernatant, repeat with sterile water or aseptic PBS to wash yeast two times
                        4. Suspension the cell with appropriate medium, then move to the appropriate medium.<H3 style="color:#E3D434"><STRONG>Protocol about cellobiose pathway</STRONG></H3><STRONG>Experimental purpose</STRONG>
                		1.	Tested the growth difference between engineered strains which were introduced into the cellobiose pathway and negative strains. The utilization ability of cellobiose and the difference of ethanol content. Verify the availability of the line.
2. Calculated the growth curve，cellobiose change curve and ethanol change curve.of positive strains and negative strains in the cellobiose as the only carbon source medium.<STRONG>Material preparation</STRONG>
1、SC：0.67%YNB（3.35g+350ml）+ complete aa（50ml）  （400ml）
2、cellobiose：83ml water +8.3g   （0.1g/ml）
3、complete aa：10ml/100ml mother liquor
                		4、HPLC：Mobile phase Na2SO4
0.1M Na2SO4 ， 0.6 mL/min ， 35℃  ，Shodex OHpak SB-804 HQ and Shodex OHpak SB-802.5 HQ column (8.0mm×300mm)
5、BioTek Synergy H1 Automatic enzyme-linked immunosorbent assay systems and 96-hole plate
6、SBAbiosensor<STRONG>Solubilty</STRONG>
cellobiose= 0.12 g/mL<STRONG>The determination of cellobiose content</STRONG>
1.Use YPD to activate yeast, prepare cellobiose medium (2% fiber two sugars +sc=10ml+40ml)
2. Test OD600, the content of the cellobiose and the change of ethanol content 30°c 180rpm (negative, positive 4 bottles, initial od=1)
96 Hole Plate, 200ul sample/hole, 3 hole/sample;
Sample, 2340rpm, 5min bacteria, 620u/times, save -80°c.
Time of measurement and retention time See timetable
3. Using HPLC to detect the change of cellobiose in medium, the change of ethanol content was detected by SBA Biosensor.<STRONG>Method of adjusting OD value</STRONG>
1, measure the OD600 value of the bacterium liquid.
2. Maintain the same volume as the original medium.
3, 2340rpm, 5min, discard supernatant, repeat with sterile water or aseptic PBS to wash yeast two times.
4. Suspension the cell with appropriate medium, then move to the appropriate medium.</DIV></DIV></DIV></DIV></DIV><H2 class="ouc-heading" id="adhesion"><STRONG>Verification of Adhesion platform</STRONG></H2><P style="font-size: 20px">
    	We use ice-nuclei to display the monomer streptavidin (mSA) on the surface of <I>E. coli</I> outer membrane and Aga1p-Aga2p system to display the biotin on the yeast cell wall. The mSA on <I>E. coli</I> surface and the biotin on the yeast cell wall form a covalent linkage and bind with other firmly, establishing a adhesion platform.
    </P><P style="font-size: 20px">
    	After sub cloning our target gene onto the vectors successfully, we transformed the plasmids into either <I>E. coli</I> or <I>S. cerevisiae</I> to test the expression and function of corresponding protein, which means they should be integrated in the out membrane or cell wall. Therefore, we conduct Western Blot so that we can prove our result specifically. After that, we need to verify that the heterogeneous cells do adhere to each other through the combination of biotin and mSA. With rhodamine- polyethylene glycol-biotin and FITC-streptavidin, we conducted immunofluorescence staining on both <I>E. coli</I> and <I>S. cerevisiae.</I> By measuring florescent signal form two kinds of cell, we had a visual validation of parts work. Meanwhile, we explored the optimal condition for their co-culture and drew a conclusion after preliminary experiment. We inoculated the yeast in the medium containing galactose and inoculated <I>E. coli</I> after 30 hours cultivation.
    </P><P style="font-size: 20px">
    	For more solid a proof of their adhesion, we used Immunofluorescent Localization to demonstrate their relative location and took photos with transmission electron microscopy (TEM) for their microstructure. What’s more, we acquired some quantitative data such as the percentage of linkage and expression with FCM. As a result, we can say that our adhesion platform was established successfully.
    </P><DIV class="container"><H3 style="color: #66BCC7" id="ecoli"><STRONG><I>E.coli</I> strain construction</STRONG></H3><P style="font-size: 20px">
    		We assume expressing monomer streptavidin on the outer membrane of <I>E.coli</I> will help us using biotin-avidin system to link <I>E.coli</I> and <I>S.cerevisiae</I>. In part registry, we find <A href="">BBa_K523013</A> constructed by team 2011_Edinburgh(http://parts.igem.org/Part:BBa_K523013) encoding INP-eYFP. We acquired INP using PCR from this part. Then we choose the constitutive promoter <A href="">J23106</A> from part registry because it can express protein continuously so that we can have a relative steady co-culture system. The mSA part is from team 2016_Peking that give us much help. Monomer streptavidin is much smaller than that of wild type and is easier to be used in circuit. Based on above reasons, we designed plasmid J23106-INP-mSA-pSB1C3 (We call it JIM for short ) and transformed it into <I>E.coli</I> DH5α strain.
    	</P><P style="text-align: center;color: gray">Fig.1 Electrophoresis result of expression vector. Lane 3&amp;4 is pSB1C3-J23106-INP-mSA.  Lane1&amp;2 is pYD1-BAP, lane 3&amp;4 is pSB1C3-J23106-INP-mSA, lane5&amp;6 is pYC230-BirA
    	</P><P style="font-size: 20px">
    		The sequencing results confirmed that we successfully cloned the J23106-INP-mSA-PSB1C3 expression vectors, we transformed the target plasmids to DH5α for amplification.
    	</P><H3 style="color: #66BCC7"><STRONG>Protein Expression Assay</STRONG></H3><P style="font-size: 20px">
    		If the protein is successfully expressed, they will be displayed on the extracellular mem-brane of the <I>E.coli</I>. Therefore, we can verify protein expression by detecting whether INP-mSA exists in the total outer membrane protein.
    	</P><P style="font-size: 20px">
    		We used the Fractionation Separation to extract three components of the outer membrane protein, the intimal protein and the cytoplasmic protein of engineered <I>E.coli</I>. Further Western Blot aim to confirm the previous result specifically and We choose to use horseradish peroxidase-biotin because it has the specific ability to bind INP-mSA.
    	</P><P style="text-align: center;color: gray">Fig.2 Western blot analysis using HRP-biotin to evaluate J23106-INP-mSA(JIM) JIM expression. lane1, the outer membrane of JIM; lane2, the intimal protein of JIM; the cytoplasmic protein of lane3; lane4, the outer membrane DH5α; lane5, the intimal protein of DH5α; lane6, the cytoplasmic protein of DH5α.
    	</P><P style="font-size: 20px">
    		Western Blot result shows that there is a clear protein band at corresponding position with a correct size and the result specifically. It prove that we do express INP-mSA on the <I>E.coli</I> surface successfully .
    	</P><H3 style="color: #66BCC7"><STRONG>Protein function verification</STRONG></H3><P style="font-size: 20px">
Immunofluorescence Staining
    		After verifying the protein expression, we want to prove that the protein can function as normal. We used Rhodamine-biotin to immunofluorescence stain the engineered <I>E.coli</I>  and negative strains  DH5α to observe whether the bacteria has fluorescence under fluorescence microscope.
    	</P><P style="font-size: 20px">
    		The result proves that the function of protein was normal.
    	</P><P style="text-align: center;color: gray">Fig 2.3 Rhodamine-biotin immunofluorescence staining of engineerd E.coli(right) and negative strains(left).
    	</P><P style="font-size: 20px">
    		Meanwhile, we would like to quantify the expression of INP-mSA compared with DH5α by plate reader. When we carried out the experiment, we met the problem that there are different incubation conditions in different paper.  So we had to explore the best culture condition for the INP-mSA protein expression. Firstly, We did two sets of experiments to compare the fluorescence values of 2h incubation under two temperatures ( 0 ℃ and 30 ℃, 0℃ means incubate on ice.) 
    	</P><P style="text-align: center;color: gray">Fig 2.4 Expression level of INP-mSA fusion protein under different conditions.( I )  the single cell fluorescence values from <I>E.coli</I> DH5α cell expressing JIM fusion and DH5α without a plasmid under 0 ℃  incu-bation for 2h. ( II ) the fluorescence values of <I>E.coli</I> DH5α cell expressing JIM fusion and DH5α under 30 ℃.
    	</P><P style="font-size: 20px">
    		The image showed that E. coli containing the Jim plasmid successfully expressed INP-MSA fusion protein. The fluorescence value increased after 2 hours incubation under 30 ℃ conditions. We speculate that there may be an ice dye incubation that causes ice nucleation proteins to form crystals that inhibit the combination of fluorescent dyes and the mSA of fusion proteins. [2] 
    	</P><H3 style="color: #66BCC7" id="yeast"><STRONG>Yeast strain construction</STRONG></H3><P style="font-size: 20px">
    		Here we need to construct the vector of <I>BirA</I> (biotin ligase) and BAP (biotin acceptor peptide) in yeast for displaying biotin on the surface.(fig.1)
    	</P><P style="font-size: 20px">
    		We first cloned birA into the yeast-shuttle plasmid pYC230. After transformation, it enables the yeast to transfer biotin to biotin acceptor peptide（BAP);  pYD1 is another 5.0 Kbp expression circuit designed for expression, secretion, and display of proteins on the extracel-lular surface of <I>S.cerevisiae</I>. The vector contains AGA2 gene from <I>S.cerevisiae</I> encoding one subunit of the a-agglutinin receptor. By fusing our gene of interest to <I>AGA2</I>, so we can enable the yeast to display the biotin on the cell surface. 
    	</P><P style="font-size: 20px">
    		After successful construction of plasmids, we introduced pYD1 containing BAP and pYC230 containing Bira into <I>S.cerevisiae</I> EBY100. 
    	</P><H3 style="color: #66BCC7"><STRONG>Protein expression verification</STRONG></H3><P style="font-size: 20px">
    		We want to test whether biotinylated BAP is successfully integrated in the yeast cell wall. So, we dyed engineered yeast and negative control respectively with FITC-streptavidin. Using fluorescent microscopy, we are able to observe the fluorescence of both.
    	</P><P style="text-align: center;color: gray">Fig 2.7 The fluorescent staining of yeast with FITC-Streptomyces. (left) <I>S.cerevisiae</I> EBY100 without target gene; (middle) <I>S.cerevisiae</I> EBY100 with BAP+BirA though galactose induction for 12h; （right) <I>S.cerevisiae</I> EBY100 with BAP+BirA though galactose induction for 32h.
    	</P><P style="font-size: 20px">
    		 Meanwhile, we would like to  quantify the expression of biotin compared with EBY100 by plate reader, and wonder when biotin can reach the maximum expression. 
    	</P><P style="text-align: center;color: gray">Fig 2.8 Expression level of biotin in different time point compared with negative controll. The red curve represents for the fluorescence of  EBY100-BirA-BAP as time  goes  by. And  the  blue curve represents for the fluorescence  of  EBY100 control. 
    	</P><P style="font-size: 20px">
    		As a result we can see the FI/abs600 of <I>S.cerevisiae</I> EBY100 with BAP+BirA is higher. At about 20h the expression of biotin is the greatest，and at about 30h the accumulation of biotin reached maximum level.
And after 30h, the expression level of biotin achieved the highest amount.
    	</P><P style="font-size: 20px">
    		 Further, we want to know how many biotins are expressed on the surface of a single cell. Luckily, we found that we can confirm the expression of biotin and its expressing percent under Flow cytometer. 
    	</P><P style="font-size: 20px">
    		 We use FITC-streptavidin to stain <I>Saccharomyces cerevisiae</I> and use 488nm exciting light to excite it and detect fluorescent signal at 525nm. As the result showed here, the fluorescent peak shifted towards the direction of stronger fluorescent, which means positive sample express more biotin acceptor peptide(BAP)  Compared with the results of empty sample, we can ensure that BAP is displayed on the surface of Saccharomyces cerevisiae. Above the threshold, there are 48% of Saccharomyces cerevisiae in the positive sample. While in the negative sample and empty sample, the percentage is only 28%. 
    	</P><P style="text-align: center;color: gray">Fig 2.5
    	</P><P style="text-align: center;color: gray">Fig 2.5.1 the x-mean level of FCM result
    	</P><P style="text-align: center;color: gray">Fig 2.5.2 the CV of FCM result
    	</P><P style="font-size: 20px">
    		 Under the same CV, the mean fluorescence of INP-mSA expressing E. coli (positive) is 1.5 times higher than that of E. coli with an empty plasmid (negative).Considering the wild type E. coli(empty), the fluorescent of negative control is almost zero.(the slight fluorescent of it may come from the residue of PBS)  
    </P><H3 style="color: #66BCC7" id="co"><STRONG>Co-cultivation verification</STRONG></H3><P style="font-size: 20px">
    		We aimed to build an adhesion platform of <I>E. coli</I> and <I>S.cerevisiae</I>. Therefore, first we need to select a culture condition under which the heterogeneous cells can both grow happily. Taking into account the expression of the BAP in <I>S.cerevisiae</I>, galactose is required to induce protein expression. And to avoid diauxic growth of yeast when changing a new carbon source in the medium, we ultimaytely selected the YNB-CAA galactose medium. In order for the optimize expression of INP-MSA fusion protein, the in-ducing expression of BAP, and the growth rate of two heterogeneous cells, we took 28 ℃ as the culture temperature.
    	</P><P style="font-size: 20px">
    		We characterized the growth curve of <I>E. coli</I> with JIM target gene and that of <I>S.cerevisiae</I> containing BirA &amp; Bap under the mentioned culture condition. And the dry group wonder some parameters of the growth of <I>E.coli</I> and <I>S.cerevisiae</I>, as well as the consumption of galactose.
    	</P><P style="text-align: center;color: gray">Fig.8 (a) the growth curve of <I>E.coli</I> with JIM; (b) the growth curve of <I>S.cerevisiae</I> containing BirA &amp; Bap
    	</P><P style="font-size: 20px">
    		From the image we can tell, when the yeast and E.coli grows at 28 ℃, the culture time needs to be longer than 48h in order to make the yeast take a dominant proportion. We adjust the YNB-CAA galactose medium containing S.cerevisiae initial OD600 to 1.</P><P style="font-size: 20px">
    		Meanwhile, we used HPLC to measure the consumption of galactose.
    	</P><H3 style="color: #66BCC7" id="adhesion"><STRONG>Adhesion validation</STRONG></H3><P style="font-size: 20px">
    		In order to illustrate that in practice, the co-culture of <I>S.cerevisiae</I> and <I>E.coli</I> can truly form the collaborating platform of heterologous cells, we will cultivate the two strains together by adding <I>E.coli</I> to the galactose-induced yeast. The co-cultured samples will then be dyed with rhodamine-biotin and FITC-streptavidin.
    	</P><P style="font-size: 20px">
    		As is shown below, we can see that there are quite a few areas of overlapping red and green fluorescent in positive group where there are both <I>S.cerevisiae</I> and <I>E.coli</I> in the sample, indicating that they are very likely to have linked to each other. The negative control of <I>S.cerevisiae</I> and <I>E.coli</I> however, has either red or green fluorescent. And for the empty group, there is no fluorescent detected at all. 
    	</P><P style="text-align: center;color: gray">Fig 2.10 The observation of confocal laser scanning microscopy(Nikon A1). The rhodamine-biotin is a solid dye, so it may have impurities in this picture.  
（tips: please maximize the intensity of your screen when looking at this picture)
    	</P><P style="font-size: 20px">
    		One step further, we want to confirm the connection between E. coli and Saccharomyces cerevisiae by FACS analysis, because this analysis method can observe in single-cell level.Unfortunately, however, we found the Beckman flow cytometry (FCM) in our university couldn’t excite our fluorescent dye rhodamine-biotin so we wereas not able to use it to catch dyed E. coli.  
        </P><P style="font-size: 20px">
Finally, we use the scanning tunneling microscope to shoot the microscopic structure of the connecting system and get a clearer picture of the connection structure. So far, we can say our adhesion platform is successfully established.
To confirm the link between <I>E.coli</I> and <I>S. cerevisiae</I> further, we use Transmission Electron Microscopy (TEM), which can capture our strains in a microcosmic view. In the following results, the positive samples show clearly that several <I>E.coli</I> stick firmly to <I>S. cerevisiae</I>, indicating a successful adhesion between them.  In negative samples (where the yeast and bacteria have no plasmid introduced), however, there is no such connection. Actually, because of the loss of sampling point in TEM, we do not even spot the existence of <I>Saccharomyces cerevisiae</I> in negative control. We think that it may result from the dispersive distribution of <I>E.coli</I> and <I>S. cerevisiae</I>.  Due to the limited time, we are not able to repeat our experiment and acquire data that is more convincing.  In future, we will certainly try again. </P><P>So far, the current results indicate that our first-stage adhesion platform is successfully established.
    	</P><P style="text-align: center;color: gray">Fig.12 Observation of Transmission Electron Microscopy 

    	</P></DIV><DIV class="panel-group" id="accordion"><DIV class="panel"><DIV class="panel-heading ouc-panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse2"><STRONG>PROTOCOL</STRONG></A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="container"><H3 style="color:#E3D434"><STRONG>Cell Fractionation</STRONG></H3>
                		At first Cells cultured at 37℃ until cell density(OD600=0.6-0.8), after cultured at 25℃ for 24 h. Harvested cells were washed, and resuspended in PBS buffer containing 1 mM EDTA and lysozyme at 10 Ag/mL. After 2 hincubation, cell suspension was treated with an ultrasound sonication at 30 sec  2 cycles. To obtain total membrane fraction, whole cell lysate was pelleted by centrifugation at 20,000 rpm for 1 h using an ultracentrifuge (Optima LE-80K; Beckman, Fullerton, CA). The supernatant was regarded as soluble cytoplasmic fraction. For further outer membrane fractionation, the pellet (total membrane fraction) was resuspended with PBS buffer containing 0.01 mM MgCl2 and 2% Triton X-100 for solubilizing inner membrane and incubated at room temperature for 30 min, and then the outer membrane fraction was repelleted by ultracentrifugation. <H3 style="color:#E3D434"><STRONG>Western Blot Analysis</STRONG></H3>
                		An equal volume of each fraction (cytoplasmic and outer membrane) of the cells containing inp-mSA were mixed with SDS sample buffer (TaKaRa, 4X Protein SDS PAGE Loading Buffer), boiled for 15 min, and resolved by 12.5% (wt/vol) SDS-polyacrylamide gel electrophoresis (SDS-PAGE), followed by electrophoretic transfer to Hybond-PVDF membranes with transfer buffer (48 mM Tris-HCl, 39 mM glycine, 20% methanol, pH 9.2) by using a Trans-Blot SD Cell (Bio-Rad, Hercules, CA) at 100V for 1h30 min. After blocking for 1 h in TBS buffer (20 mM TrisHCl, 500 mM NaCl, pH 7.5) containing 5% (wt/vol) nonfat dry milk, the membrane was then incubated for 1.5 h at room temperature in antibody solution (1% (wt/vol) nonfat dry milk in TTBS (TBS with 0.05% Tween-20)) containing HRP conjugated Biotin (1:500 vol/vol) (Sangon Biotech). After successive washing twice for 5 minutes with TTBS for 2 and first for 5 minute with TBS. Color, putting the membrane protein on the face on the petri dish, and then to the above color AB drops, reaction of 2-3 minutes, then don't directly put the cassette in the dry, covered with layers of plastic wrap, draw out the film edge, so as to avoid dark room inside can't find the location of the membrane.
                		I use a petri dish to fill the film, then rinse it out and put it in the fixer.<H3 style="color:#E3D434"><STRONG>Pyd1 yeast display vector expression and display of protein fusion</STRONG></H3>
                		Grow untransformed EBY100, EBY100/pYD1, and EBY100/pYD1 containing your gene of interest as follows.
1. Inoculate a single yeast colony into 10 ml YNB-CAA containing 2% glucose and grow overnight at 30°C with shaking.
2. Read the absorbance of the cell culture at 600 nm. The OD600 should be between 2 and 5. If the OD600 is below 2 or over 5, refer to the table below.
If the OD600 is.... 
Then you may...
below 2, 
either continue growing the cells until the OD600 reaches 2 or proceed to Step 3.
over 5, 
proceed to Step 3.<STRONG>Note</STRONG>: In general, OD600 readings less than 2 will decrease the number of cells displaying fusion proteins. OD600 readings greater than 5 will delay induction of expression of the displayed protein.
3. Centrifuge the cell culture at 3000-5000 x g for 5-10 minutes at room temperature.
4. Resuspend the cell pellet in YNB-CAA medium containing 2% galactose to an OD600 of 0.5 to 1. This is to ensure that the cells continue to grow in log-phase. For example, if the OD600 is 2 from Step 2, resuspend the cells in 20 to 40 ml of medium.
5. Immediately remove a volume of cells equivalent to 2 OD600 units. For an OD600 of 0.5, remove 4 ml and place on ice. This is your zero time point.
6. Incubate the cell culture at 20-25°C with shaking. <STRONG>Note</STRONG>: In general, more cells will display the protein fusion at 20°C. However, if you do not have an incubator that maintains 20°C, try 25°C.
7. Assay the cell culture over a 48-hour time period (i.e. 0, 12, 24, 36, 48 hours) to determine the optimal induction time for maximum display. For each time point, read the OD600 and remove a volume of cells that is equivalent to 2 OD600 units (see Step 5, above). Proceed to Staining of Displayed Proteins, below.<H3 style="color:#E3D434"><STRONG>Staining of Displayed Proteins</STRONG></H3>
                		For each time point, assay untransformed EBY100, EBY100/pYD1, and EBY100/pYD1 containing your gene of interest. Time points may be processed as they are collected or placed on ice and stored at +4°C until all time points are collected. Do not freeze cells.
1. Take your time points from Steps 4 and 6, above, and centrifuge at 3000-5000 x g for 5-10 minutes at +4°C.
2. Resuspend the cells in 1X PBS and centrifuge as in Step 1.
3. Remove the PBS and resuspend the cell pellet in 250 µl of 1X PBS, 1 mg/ml BSA, and FITC-Streptavidin (1:500 vol/vol) (Solarbio) .
4. Incubate on ice for 30 minutes with occasional mixing.
5. Centrifuge the cells at 3000-5000 x g for 5-10 minutes at +4°C.
6. Wash the cells thrice with 1X PBS.
7. Resuspend the cells in 700μ 1X PBS to divided into 96 plates, and the fluorescence values were measured by the microplate reader, or to sample, biological section, and observed with a fluorescence microscope.<H3 style="color:#E3D434"><STRONG>Co-culture</STRONG></H3>
                		1.Yeast cell was cultured with YPD medium for 48 h at 30◦C.
2.Then the yeast cells was transferred into YNB-CAA Gla medium as adjusted OD=1 by YNB-CAA Gla medium and cultured for 30h at 25 ◦C.
3.Then add E.coli into YNB-CAA Gla medium with OD=0.4，cultured for 12 h at 28 ◦C.
4.Then the cells can be used in observation by microscopy or FACS method.<H3 style="color:#E3D434"><STRONG>FACS </STRONG></H3>
                		1.After precultivation with appropriate selective medium for 18 h at 30◦C, yeast cells were harvested and washed with phosphate buffered saline (PBS) three times. 
2.The cell density was adjusted to OD600 = 1 by PBS. 
3.Then cells were incubated with 20 g/ml of streptavidin–FITC (Solarbio)for 1 h. 
4.After washing by PBS three times, cells were observed with FACS analysis.(Beckman)
5.FACS use as ‘Cytomics FC500 MCL with CXP software Instructions For Use’<H3 style="color:#E3D434"><STRONG>Microscopy</STRONG></H3>
                       1.After precultivation with YNB-CAA Gla medium for 12 h at 28◦C, yeast cells and E.coli were harvested and washed with phosphate buffered saline (PBS) three times. 
2.The cell density was adjusted to OD600 = 1 by PBS. 
3.Then cells were incubated with 20 g/ml of streptavidin–FITC (Solarbio)for 1 h. 
4.Then the cells were washed with phosphate buffered saline (PBS) three times. 
5.Then cells were incubated with 20 g/ml of Biotin-Rhodamine (ToYongBio)for 1 h. 
6.After washing by PBS three times, cells were observed with microscopy. </DIV></DIV></DIV></DIV></DIV><H2 class="ouc-heading" id="mini"><STRONG>Mini system</STRONG></H2><P style="font-size: 20px">
    	We set four different permutations for our mini promoter and mini terminator thus constructed four dif-ferent circuits. For each one, the strength level of promoters are characterized by yECitrine, a kind of yel-low fluorescent protein. And we use red fluorescent protein mStrawberry to represent the read through of terminators. 
    </P><P style="font-size: 20px">
    	In the wet lab study, we measured both the expression level of YFP and RFP to validate the expected performance of the MINI system in comparation of normal promoters and terminators whose expression level are measured as well.
    </P><P style="font-size: 20px">
    	Meanwhile, we monitored the growth rate of the recombinant yeast to specify the expression of MINI system at a particular living stage. Moreover, we conducted qPCR towards corresponding protein in or-der for a further validation of MINI system’s strong expression on a post-transcriptional level. 
    </P><P style="font-size: 20px">
    	We also expend the application range of this system by contact with other teams and test it in different yeast strains and experimental environment, which can also be an important part of our collaboration. the MINI system has a high performance ratio in driving gene expression. The obvious advantage of the MINI system is to provide a simple model for studying promoter mutations and promoter modifications.
    </P><H3 style="color: #66BCC7" id="circuit"><STRONG>Circuit construction</STRONG></H3><P style="font-size: 20px">
    	We synthesized minip, minit, cyc1p, cyc1t and built four circuit with Gibson assembly by arranging them in different orders. The plasmids were then imported into yeast EBY100. 
    </P><P style="text-align: center; color: gray">Fig 3.1 The plasmid map of our circuit. 
    </P><P style="text-align: center; color: gray">Fig 3.2 
    </P><H3 style="color: #66BCC7" id="" function=""><STRONG>Function verify</STRONG></H3><P style="font-size: 20px">
    	We characterize the strength of the MINI system by detecting the fluorescence intensity of the yeCitrine. We measured the growth curves of four strains containing different expression systems, and measured the intensity of excitation and emission of yeCitrine, respectively 502nm and 532nm. 
    </P><P style="font-size: 20px">
    	For convenience, we named the “Pmini-yECitrine-Tcyc1-mStrawberry-Tcyc1” as “mc”, “Pmini-yECitrine-Tmini-mStrawberry-Tcyc1” as “mm”, “Pcyc1-yECitrine-Tcyc1-mStrawberry-Tcyc1” as “cc” and “Pcyc1-yECitrine-Tmini-mStrawberry-Tcyc1” as “cm” hereafter.
    </P><P style="text-align: center; color: gray">Fig 3.3 The strength of the MINI system.
    </P><P style="font-size: 20px">
    	The fluorescence intensity is: mm&gt; cm&gt; mc&gt; cc, in which mm circuit has the highest expression, proving our successful construction of MINI system. To draw a conclusion, the system has a strong expression, but a very short nucleotide sequence. In this ex-ample, the combination of weak promoter + strong terminator is better than that of strong promoter + weak terminator. 
    </P><P style="font-size: 20px">
    	With the help of the other two teams, we completed the repeated testing of the MINI system. The time of the test data is the late logarithmic phase of yeast growth. The yeast strains we used with NJU-China were Saccharomyces cerevisiae EBY 100. The yeast strain used in TJU China was Synthetic yeast synX.
    </P><P style="text-align: center; color: gray">Fig 3.4 MINI system fluorescence measurements from three teams, OUC-Chian,TJU-China, NJU-China (left to right)
    </P><P style="font-size: 20px">
    	Through the fluorescence intensity map, we have not yet clear the true advantage of MINI system. But from the following figure, we can directly observe the relationship between a nucleotide base number and fluorescence intensity. The red arrows indicate the short and strong features of the mm gene circuit.
    </P><P style="text-align: center; color: gray">Fig 3.5 Promoter and terminator of the nucleotide base length and intensity. (For example, mm represents the base length of Pmini + Tmini).
    </P><P style="font-size: 20px">
    	In order to verify that the MINI system was able to work in a variety of yeast strains, we invit-ed other teams to conduct a repeat experiment. Especially in TJU-China, the use of their own synthesis of yeast to complete this experiment also verified the MINI system with versa-tility.
    </P><H3 style="color: #66BCC7" id="level"><STRONG>Validation of expression on transcription level</STRONG></H3><P style="font-size: 20px">
    	The result of qPCR shows that at 22<SUP>nd</SUP> hour the system reached the highest expression intensity and the expression of four circuits is shown below. The error bars indicate s.d. of mean of exper-iments in triplicate.
    </P><P style="text-align: center; color: gray">Fig 3.6
    </P><P style="font-size: 20px">
    	In the chart, the RNA content of yECitrine comes as the following order: mm&gt;mc&gt;cm&gt;cc, which is not completely consist with the result of protein level. mm’s RNA content is several times that of cc. Compared to Figure b, this difference in RNA content does not reflect the protein content very well, yet still, our MINI system has an obvious superiority over normal combination (cc), which confirmed our hypothesis. As shown above, generally the magni-tude of leakage is over 100 times smaller, so it can be ignored to some extent.
    </P><P style="font-size: 20px">
    	To draw a conclusion, mm has the smallest size and the strongest expression with a leakage that can almost be ignored. What’s more our system functions well in different strains and experimental conditions, which proves its potential to apply in various situations.
    </P><DIV class="panel-group" id="accordion"><DIV class="panel "><DIV class="panel-heading ouc-panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse3"><STRONG>PROTOCOL</STRONG></A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse "><DIV class="panel-body"><H3 style="color:#E3D434"><STRONG>Reserve transcription</STRONG></H3><STRONG>Material</STRONG>
PrimeScript<SUP>™</SUP>RT reagent Kit with gDNA Eraser (TaKaRa Code No. RR047A)<STRONG>Procedure</STRONG>
1. Genomic DNA elimination reaction
1) Prepare the genomic DNA elimination reaction solution on ice.
2) Add RNA template with the suitable amount of the master mix to a PCR tube.<TABLE class="table" style="width: 500px"><THEAD><TR><TH>Reagent</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>5X gDNA Eraser Buffer</TD><TD>2.0 μl</TD></TR><TR><TD>gDNA Eraser</TD><TD>1.0 μl</TD></TR><TR><TD>Total RNA</TD><TD>1.0 μl(Up to 1 μg of total RNA)</TD></TR><TR><TD>RNase Free dH<SUB>2</SUB>O</TD><TD>Up to 10.0 μl</TD></TR></TBODY></TABLE>
                3) Run the program: 42℃ 2 min
2. 4℃ Reverse-transcription reaction
1) Prepare the reverse-transcription reaction solution on ice.
2) Add Reaction solution from Step 1 with the suitable amount of the master mix to a PCR tube<TABLE class="table" style="width: 500px"><THEAD><TR><TH>Reagent</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Reaction solution from Step 1</TD><TD>10.0 μl</TD></TR><TR><TD>5X PrimeScript Buffer 2 (for Real
Time)</TD><TD>4.0 μl</TD></TR><TR><TD>RT Primer Mix</TD><TD>4.0 μl</TD></TR><TR><TD>PrimeScript RT Enzyme Mix I</TD><TD>1.0 μl</TD></TR><TR><TD>RNase Free dH<SUB>2</SUB>O</TD><TD>4.0 μl</TD></TR></TBODY></TABLE>
               	3) Run the program:
37℃15 min
85℃ 5 sec
4℃	<H3 style="color:#E3D434"><STRONG>Total RNA Extraction</STRONG></H3><STRONG>Material</STRONG>
RNAiso Plus(Takara Co.9109)<STRONG>Procedure</STRONG>
1. Cleavage of yeast cells.
1） Take 1 ml OD600 value of 1.5 to 2.5 liquid culture yeast to 1.5 ml Microtube and 8,000 g at 4 ° C for 2 min.
2） carefully discard the supernatant, slowly add 1 ml of ice to the precipitation of sterile water, with a pipette gently blowing, so that precipitation resuspended.
3） 8,000 g at 4 ° C for 2 min. Carefully discard the supernatant, as far as possible in addition to net liquid.
4） Add 0.4 ml of Yeast RNAprep Buffer to the precipitate and gently re-blow with a pipette to resuspend the pellet.
5） into the 30 ° C water bath for 1 hour, during which gently shake the centrifuge tube 1 or 2 times.
6） Remove the centrifuge tube from the 30 ° C water bath and centrifuge at 12,000 g for 4 minutes at 4 ° C.
7） Carefully discard the supernatant, add 1 ml of RNAiso Plus to the precipitate, gently blow with a pipette to resuspend the pellet.
8） cover the centrifuge tube cover, whirlpool oscillation 2 to 5 minutes to clarify the suspension. 12,000 g at 4 ° C for 5 min.
9） Carefully aspirate the supernatant and move into a new 1.5 ml RNase-free Microtube (do not touch the precipitate).
2.Collection
1) Collect and pipettes 1.5~2.5ml bacteria which is in the log phage(usually when OD600=1.0) into a centrifuge tube. Centrifuge tube for 8,000×g,5 minutes at 4°C.Discard supernatant and be care not to disturb the bacteria pellet.
2) Add 1ml of RNAiso Plus, pipette up and down until pellet is completely resuspended.
3) Leave at room temperature(15~30°C) for 5 minutes, isolate the RNA from the nuclear protein.
2.Extracion of total RNA
1) Add 200 ul chloroform, cap the centrifuge tube and mix until the solution becomes milky. 
2) Keep the solution at room temperature for 5 minutes. 
3) Centrifuge at 12,000×g for 15 minutes at 4°C.Centrifuging the solution will separate it into three layers; liquid top layer(contains RNA),semisolid middle layer(mostly DNA),and bottom organic solvent layer. 
4) Transfer the top liquid layer to new centrifuge tube without touching middle layer. 
5) Measure the amount of the top layer and add an equal amount or add up to 0.5 times of isopropanol of the top layer. Mix together well. Keep the mixture at room temperature for 10 minutes. 
6) Centrifuge at 12,000×g for 10 minutes at 4°C to precipitate the RNA. 
7) Cleaning RNA precipitate 
8) Carefully remove the supernatant, do not touch the pellet.  9) Add an amount of 75% cold ethanol that was equivalent to the supernatant. Clean the precipitate by vortexing.
10) Centrifuge the solution at 7,500×g for 5 minutes at 4°C and discard supernatant. Be care not to disturb the precipitate.
4.Dissolving RNA
Dry the precipitate by leaving the tube open for several minutes. After the precipitate is dry, dissolved it with appropriate amount of RNase-free water.<STRONG>Attention</STRONG>
Make sure that all the centrifuge tubes and pipettes have been treated with DEPC.   <H3 style="color:#E3D434"><STRONG>Quantitative Real-time PCR</STRONG></H3><STRONG>Material</STRONG>
SYBR® Premix Ex Taq™ (Tli RNaseH Plus) (TaKaRa(Code No. RR420A)<STRONG>Procedure</STRONG>
1. Prepare the PCR mixture shown below<TABLE class="table" style="width: 500px"><THEAD><TR><TH>Reagent</TH><TH>Volume</TH><TH>Final conc.</TH></TR></THEAD><TBODY><TR><TD>SYBR Premix Ex Taq (Tli
RNaseH Plus) (2X)</TD><TD>10 μl</TD><TD>1X</TD></TR><TR><TD>PCR Forward Primer (10 μM)</TD><TD>0.4 μl</TD><TD>0.2 μM</TD></TR><TR><TD>PCR Reverse Primer (10 μM)</TD><TD>0.4 μl</TD><TD>0.2 μM</TD></TR><TR><TD>Template (&lt; 100 ng)</TD><TD>2 μl</TD></TR><TR><TD>dH2O (sterile distilled water)</TD><TD>7.2 μl</TD></TR><TR><TD>Total</TD><TD>20 μl</TD></TR></TBODY></TABLE>
               	
               	2. Start the reaction using LightCycler 480 System
1) Denature:
95℃ 30 sec. (Ramp rate: 4.4℃/sec.)
1 cycle
2) PCR :
95℃ 5 sec. (Ramp rate: 4.4℃/sec.)
60℃ 30 sec. (Ramp rate: 2.2℃/sec.)
40 cycles
3) Melting
95℃ 5 sec. (Ramp rate: 4.4℃/sec.)
60℃ 1 min. (Ramp rate: 2.2℃/sec.)
95℃ (Ramp rate: 0.11℃/sec.)
1 cycle
4) Cooling
50℃ 30 sec. (Ramp rate: 2.2℃/sec.)
1 cycle
3. After the reaction is complete, check the amplification and melting curves and plot a standard curve if absolute quantification will be performed.
                	
                	
                </DIV></DIV></DIV></DIV></DIV><DIV class="container-fluid ouc-reserve"><P class="text-center">
                          Contact Us : oucigem@163.com  |  ©2017 OUC IGEM.All Rights Reserved.  |  Based On Bootstrap</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>