<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_OUC-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:OUC-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_content_wrapper"><P><TITLE>Design</TITLE></P><DIV class="ouc-guide" style="height: 50px;background-color:#008EA1 ;"><DIV class="row"><DIV class="col-md-2 navbar-header" style="text-align: center; padding: 0"><BUTTON class="navbar-toggle" type="button" data-toggle="collapse" data-target=".navbar-responsive-collapse"><SPAN class="sr-only">Toggle Navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-responsive-collapse" style="padding: 0"><DIV class="col-md-1 ouc-navbar" style="padding: 0"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Team</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/Team" style="line-height: 40px; " class="ouc-navdown">Members</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Collaborations" style="line-height: 40px; " class="ouc-navdown">Collaborations</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Attributions" style="line-height: 40px; " class="ouc-navdown">Attributions</A></DIV></DIV></DIV><DIV class="col-md-1 ouc-navbar" style="padding: 0"><A href="##" style="line-height: 50px;text-align: center" class="ouc-nav">Project</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/Description" style="line-height: 40px; " class="ouc-navdown">Description</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Design" style="line-height: 40px; " class="ouc-navdown">Design</A></DIV><DIV class="ouc-right"><A style="line-height: 40px; font-size: 15px;" class="ouc-navdown" href="https://2017.igem.org/Team:OUC-China/proof1">Proof of concept</A><DIV class="ouc-rightnav"><A href="https://2017.igem.org/Team:OUC-China/proof1" style="display: block">Basic fermentation</A><A href="https://2017.igem.org/Team:OUC-China/proof2" style="display: block">Adhesion platform</A><A href="https://2017.igem.org/Team:OUC-China/proof3" style="display: block">MINI-GRE</A></DIV></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Demonstrate" style="line-height: 40px; " class="ouc-navdown">Demonstration</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/InterLab" style="line-height: 40px; " class="ouc-navdown">InterLab</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Improve" style="line-height: 40px; " class="ouc-navdown">Improvement</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Notebook" style="line-height: 40px; " class="ouc-navdown">Notebook</A></DIV></DIV></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Model" style="line-height: 50px;text-align: center" class="ouc-nav">Model</A></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Parts" style="line-height: 50px;text-align: center" class="ouc-nav">Parts</A></DIV><DIV class="col-md-1" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Safety" style="line-height: 50px;text-align: center" class="ouc-nav">Safety</A></DIV><DIV class="col-md-2 ouc-navbar" style="padding: 0"><A href="https://2017.igem.org/Team:OUC-China/Engagement" style="line-height: 50px;text-align: center" class="ouc-nav">Human Practice</A><DIV class="ouc-down" style=" width: 100%; text-align: center"><DIV><A href="https://2017.igem.org/Team:OUC-China/HumanPractice" style="line-height: 40px; " class="ouc-navdown">Overview</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/HP/Silver" style="line-height: 40px; " class="ouc-navdown">Silver</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/HP/Gold_Integrated" style="line-height: 40px; " class="ouc-navdown">Gold and integrated</A></DIV><DIV><A href="https://2017.igem.org/Team:OUC-China/Engagement" style="line-height: 40px; " class="ouc-navdown">Engagement</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="page-header ouc-page-header"><H1><STRONG>Design</STRONG></H1></DIV><H2 class="ouc-heading" id="fermentation"><STRONG>Basic fermentation</STRONG></H2><P style="font-size: 20px">
        	The use of plant biomass for biofuel production will require efficient utilization of the sugars in lignocellulose, while in terrestrial plants the content of cellulose is quite hard to use.[1] By contrast, cellulose in macroalgae has looser texture and lower proportion, which makes it an ideal material for biofuel production.[2] As a coastal university, we have a profound theoretical foundation of the sea and of course, algae. Therefore, it is natural that we think of algae as our substrate. 
        </P><P style="font-size: 20px">
        	However, we soon meet with another problem---the main sugar used by trains of <I>Saccharomyces cerevisiae</I> currently in bioethanol production is glucose. Other saccharides such as xylose and cellobiose are relatively harder to ferment. It means a lot more energy and cost we need to donate to pretreatment. Therefore, if we can design a recombinant strain capable of using xylose or cellobiose, it will be a big step further since the yeast can take the place of chemicals and fulfill the task itself.
        </P><P style="font-size: 20px">
        	Before the construction of the plasmid, however, we need to deal with glucose repression.[3] We all know that even yeasts engineered to ferment xylose cannot utilize xylose until glucose is completely consumed, which makes the process of fermentation rather slow when in present of glucose. To overcome these bottlenecks, we decide to separate the fermentation of xylose and cellobiose into two yeast strains, which means substrates can be transported into the cell thus isolate glucose from their shared environment and avoid their mutual interference. In other words, intracellular hydrolysis of cellobiose minimizes glucose repression of xylose fermentation. What's more, we can also lighten their metabolic burden using this strategy.[4]
        </P><P style="text-align:center;color:gray">Fig 1.1 The whole diagram of basic fermentation.
</P><P style="font-size: 20px">
        	Now we can begin our project.
        </P><P style="font-size: 20px">
        	After grinding our <I>Enteromorpha</I> into powder and extracting polysaccharide we get the remains of algae which has the same content as <I>Enteromorpha</I> waste. Because in industrial manufacture, the former one can be well utilized, we mainly focus on the residue of algae containing cellulose and hemicellulose majorly. For a higher reaction efficiency and better utilization of cellulose and hemicellulose, we treat the residue with H<SUB>2</SUB>O<SUB>2</SUB> to remove the lignin.[5] H<SUB>2</SUB>O<SUB>2</SUB> is clean pollution free and low cost, which makes it a perfect reagent for pretreatment. After that, the lignin is mostly removed and the microstructure of cellulose will break down, exposing more binding site, which makes it easier to use.
        </P><P style="font-size: 20px">
        	As for the utilization of cellulose there are three main stages: dense fiber with a high content of lignin, cellodextrin, monosaccharide. Since the process from dense fiber to cellodextrin has been a proven technique, we use enzymes to treat cellulose and hemicellulose into cellobios and xylose. And our major work will be based on these two substrates. 
        </P><P style="font-size: 20px">
        	As for cellobiose and xylose, we designed two recombinant yeast strains. For yeast A, we subcloned cellobiose transporter and β-glucosidase genes into a plasmid and introduced it to <I>S.cerevisiae</I>. The cellobiose transporter can help transport cellobiose from the environment into the cell and β-glucosidase will turn cellobiose into glucose, which can then be transformed into ethanol through its own pathway.[6] In the same way, we introduced xylitol dehydrogenase and xylose reductase gene to yeast B. <I>S.cerevisiae</I> naturally has the ability to transport xylose. With xylosereductase (XR), xylose can be reduced to xylitol, which will then be oxidized to xylulose in present of xylitol dehydrogenase. Xylulokinase of wild type <I>S.cerevisiae</I> can then phosphorylate xylulose to xylulose-5-phosphate. Finally, it will join Pentose Phosphate Pathway (PPP) and produce ethanol.[7]
        </P><P style="font-size: 20px">
        	Now we have two strains that can live on either cellobiose or xylose and can produce ethanol through their own metabolic pathways, which means they can ferment cellobiose and xylose synergistically.
        </P><H2 class="ouc-heading" id="adhesion"><STRONG>Adhesion platform</STRONG></H2><P style="font-size: 20px">
        	Through a Adhesion platform between heterogeneous cells, we make <I>E.coli</I> the surface display system of <I>S.cerevisiae</I> to enrich its biofunctions and enhance their synergistic effect.
        </P><P style="font-size: 20px">
        	We select the biotin-streptavidin system to accomplish our adhesion because it is known as the strongest covalent binding system in nature. By introducing ice nucleation protein (INP) and monomer streptavidin adhesion (mSA) fusion protein to <I>E.coli</I>, we express mSA on its surface with INP, which is intergrated into the out membrane itself.[8][9][10] Meanwhile, we express biotin acceptor peptide (BAP) and biotin ligase (BirA) in the <I>S.cerevisiae</I> strain EBY100.[11] This strain of yeast contains pYD1, which has another surface-display system called Aga1p-Aga2p adhesion system. By using this strain, we will be able to express biotinylated BAP on the yeast surface. Ultimately, <I>E.coli</I> and <I>S.cerevisiae</I> can connect firmly with each other through the covalent binding of biotin and streptavidin.
        </P><P style="font-size: 20px">
        	The following question may better elucidate our designing details:
        </P><DIV class="panel-group" id="accordion"><DIV class="panel "><DIV class="panel-heading ouc-panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse1"><STRONG>Click here to see Q&amp;A</STRONG></A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse"><DIV class="panel-body"><H3 style="color: #E3D434"><STRONG>Q1. Why choose to establish an adhesion platform?</STRONG></H3><DIV class="row"><DIV class="col-md-8"><P style="font-size: 20px">
                        		At the beginning of our project, we wanted to express cellulosome in <I>S. cerevisiae</I>, where cellulase can located. Then the cellulase can degrade <I>Enteromorpha</I> into cellose and xylose. The latter two can be used by <I>S.cerevisiae</I> as the source of fermentation. Although cellulosome is a method to degrade cellulose with high efficiency, it also has a complex structure combined with multiple scaffold proteins and cellulase domains which will accrete the difficulty of experiment design and give the <I>S.cerevisiae</I> a heavier burden.[13] Obviously, too much burden can harm the ferment efficiency. 
                        	</P></DIV><DIV class="col-md-4" style="text-align: center; color: gray">Fig 2.1 Domains and enzymes on cellulosome.
							</DIV><P style="font-size: 20px">
                        		Thus, we think of <I>E.coli</I> to do the part of cellulosome. Because <I>E.coli</I> is a model strain, we can change its feature and function easily, and its plasmid has both proper copy number and size that can lead to a lighter burden than that in <I>Saccharomyces cerevisiae</I> Moreover, we can separate degradation step and fermentation step through this method, which leads to higher efficiency.
                        	</P></DIV><H3 style="color: #E3D434"><STRONG>Q2:What’s the preponderance of our platform?</STRONG></H3><P style="font-size: 20px">
                        		This platform consists of two parts, monomer streptavidin surface display system and biotin surface display system. It is known that avidin can bind with biotin firmly in nature, so we choose the biotin-avidin system to achieve the adhesion between heterogenous cells. An important advantage of this platform is that it is not a traditional co-culture system. Through the exploration of expressing condition, we break the obstacle of the pollution between heterogenous cells. At the same time, we can use the adhesion platform to build a CBP (consolidated bioprocessing ) degradation and fermentation system.
                        	</P><H3 style="color: #E3D434"><STRONG>Q3: Why choose ice nuclear protein as the surface display system of <I>E.coli</I>?</STRONG></H3><P style="font-size: 20px">
                        		INP is a kind of secretive outer membrane protein that exists in gram-negative bacteria like Pseudomonas syringae. The N-terminator domain contains abundant aspartic acid and hydrophilic serine and threonine, and can be fixed on the surface of cells through glycosyl phosphatidyl inositol(GPI), which is good for the display of macro heterogenous protein. Compared to the Lpp-OmpA fused protein, this system will not lead to an unstable growth of cell, thus, the choice of anchor unit is the key to construct the surface display system. Furthermore, INP surface display system has the following advantages: 1) it can be expressed in many kinds of gram-negative bacteria with a steady expression level. 2) INP can display large molecular protein, so it has a larger applicate range. 3) INP naturally has the ability to secrete, locate, and anchor, and can display interest protein without accessory protein. It has been confirmed useful in many reports.[9][14][15]
                        	</P><DIV class="row"><DIV class="col-md-6" style="text-align: center;color: gray;">Fig 2.2 Binding structure of mSA and biotin.
								</DIV><DIV class="col-md-6" style="text-align: center;color: gray;">Fig 2.3 INP on <I>E.coli</I> under TEM observation.
								</DIV></DIV><H3 style="color: #E3D434"><STRONG>Q4: Why choose The Yeast Display System with the a-agglutinin receptor?</STRONG></H3><P style="font-size: 20px">
                        		To stabilize cell-cell interactions during mating and to facilitate fusion between a and α haploid yeast cells, we choose this receptor. The Yeast Display System uses the a-agglutinin receptor of <I>S. cerevisiae</I> to display foreign proteins on the cell surface. The a-agglutinin receptor consists of two subunits encoded by the AGA1 and AGA2 genes. The Aga1 protein (Aga1p, 725 amino acids) is secreted from the cell and becomes covalently attached to β-glucan in the extracellular matrix of the yeast cell wall. The Aga2 protein (Aga2p, 69 amino acids) binds to Aga1p through two disulfide bonds and after secretion remains attached to the cell through its contact with Aga1p. The N-terminal portion of Aga2p is required for attachment to Aga1p, while proteins and peptides can fuse to the C-terminus for presentation on the yeast cell surface. A yeast display system is particularly useful for mammalian cell surface and secreted proteins (e.g., receptors, cytokines) that requires endoplasmic reticulum-specific posttranslational processing for efficient folding and activity.
                        	</P><P style="text-align: center; color: gray">Fig 2.4 Aga1p-Aga2p yeast display system structure.
                        	</P><H3 style="color: #E3D434"><STRONG>Q5: How to establish a functional adhesion platform?</STRONG></H3><P style="font-size: 20px">
                        		 At first, we consider using INP surface display system in <I>E.coli</I> to display mSA for binding to biotin in <I>S.cerevisiae</I> and use OmpA surface display system to display cellulase and hemicellulase, so that cellulose and hemicellulose can be degraded and the product can be utilized by <I>S.cerevisiae</I>. However, we found that OmpA-CenA (a kind of cellulase in part registry) did not have the correct function as we expect. Therefore, we decide to construct the adhesion platform.
                        	</P><P style="font-size: 20px">
                        		We use INP from BBa_K523013, BBa_J23106 and mSA from team Peking to build our part J23106-INP-mSA with a 4X linker. Similarly we acquire BirA from <I>E.coli</I> genome that can biotinylate biotin accepted peptide (BAP). In addition, we use pYD1 to display BAP on the surface of <I>S.cerevisiae</I>.
                        	</P><P style="font-size: 20px">
                        		We co-culture these two strains and explore the proper condition to culture them together. To analyze this system with mathematical model, we also characterize the growth curve of these two strains and the only consumption rate of the corresponding carbon source. With these data, we simulate a co-culture model, and then feed back to our wet lab and optimize the experiment condition so that we can get the best ethanol production.
                        	</P></DIV></DIV></DIV></DIV><H2 class="ouc-heading" id="mini"><STRONG> MINI-GRE</STRONG></H2><P style="font-size: 20px">
        In previous research, synthetic biologists have achieved minimal promoters and minimal terminators in Yeast [16,17]. However, they also find that the combination of promoter and terminator will affect the behavior of the circuit too, which means that it is difficult to predict the output level of particular circuit by the quantitative result of promoter and terminator respectively. Here, we combine the minimal promoter and minimal terminator together to test if we could get a minimal promoter-terminator pair (minimal genetic regulatory elements,MINI-GRE) with similar or higher transcription output level compared with commonly used transcriptional regulatory elements.
        </P><DIV><H3 style="color: #66BCC7"><STRONG>Background</STRONG></H3><P style="font-size:20px">The native promoters and terminators of Yeast are usually significantly longer than that of Bacteria [16,17], with quite a few unnecessary regions for engineered circuits. The big size of fungal promoters and terminators may increase the cost of synthetic circuits, the possibility of non-specific homologous recombination, and the undesired endogenous cellular interactions [16,17]. All these disadvantages limits large-scale synthetic biology efforts in yeast. </P><H3 style="color: #66BCC7"><STRONG>Mini promoters</STRONG></H3><P style="font-size:20px">In previous research, this problem of bulky yeast promoters was scarcely addressed in synthetic biology until a research work was published by Heidi Redden and Hal S. Alper, In their research, they claimed that the core element in MINI promoter is a TATA box with consensus sequence of TATAWAWR followed by a transcription start site (TSS) with consensus sequence of A(A<SUB>rich</SUB>)<SUB>5</SUB>NYAWNN(A<SUB>rich</SUB>)<SUB>6</SUB>.The UAS elements contain transcription factor-binding sites (TFBS) thought to aid in RNAP stabilization and enhance transcription rates. For Neutral AT-rich spacer, Hal S. Alper draw a conclusion that 30bp may be the minimal spacing required between the TATA box and TSS for <I>S. cerevisiae.</I>[16]</P><P style="font-size:20px">What’s more, they designed a basic scaffold for artificial minimal promoters and selected minimal promoters by high through-put technology. During their work, the length of minimal promoters is significantly shorter than general WT yeast promoters, which can provide short yeast promoters for robust function with high orthogonality.[16] For convenience, we call the minimal promoters from this outstanding work MINI promoters.  </P><P style="text-align:center;color:gray">Fig 3.1 The structure of mini promoter.
</P><H3 style="color: #66BCC7"><STRONG>Mini terminators</STRONG></H3><P style="font-size:20px"> Not until 2011, when Yamanishi et al. optimized the expression level of fluorescent protein and other heterologous protein in yeast by using the different terminators instead of the commonly used CYC1 terminator did we pay attention to the influence of terminators in gene expression[20]. This research shows that the quantitative description on function of terminators in yeast is very important for SynBio work in the future.  </P><P style="font-size:20px"> Just the same as in promoters, for yeast, terminators also have some core regions that play a crucial role in their functions, the existence of which makes it possible to optimize those lengthy terminators into shorter but stronger ones.[18]  ZIJIAN GUO et al. identified that the yeast terminator has three domains，suggesting that 39-end formation of yeast mRNAs have the following three degenerate elements: (i) the efﬁciency element, which functions by enhancing the efﬁciency of positioning elements; (ii) the positioning element, which positions the poly(A) site; and (iii) the actual site of polyadenylation.[17,18]</P><P style="font-size:20px">As suggested in Yamanishi’s research, we chose an engineered minimal expression-enhancing terminator with low leakage for standardization in our project. We call it MINI terminator.</P><P style="text-align:center;color:gray">Fig 3.2 The structure of mini terminator.
</P><H3 style="color: #66BCC7"><STRONG>Combination</STRONG></H3><P style="font-size:20px">In 2013, Kathleen A. Curran et al. published a research, in which they mentioned that different combinations of promoters and terminators can provide various effect on expression level of the corresponding gene [19]. This research shows that the modularization properties of promoters or terminators in yeast is not as good as that we desire. It is difficult to predict the behavior of a particular promoter-terminator pair based on their respective strength. </P><P style="font-size:20px">To characterize the function and relationship of promoters and terminators in yeast, we see the promoter-terminator pair as a basic genetic regulatory element here. So, what will happen if we place strong MINI promoters and strong MINI terminators from different research works together? When we define the promoter-terminator pair as genetic regulatory element, is it possible to find out the minimal genetic regulatory element (MINI-GRE) which can drive similar expression level as that of commonly used parts before in Yeast? </P><H3 style="color: #66BCC7"><STRONG>Circuit construction</STRONG></H3><P style="font-size:20px">To explore the feasibility of MINI-GRE by combining promoters with terminators as that we mentioned above, we designed four promoter-terminator pairs, and constructed four different report circuits for them (fig 3.1).</P><P style="font-size:20px">For circuit 1, we pair promoter CYC1 with terminator CYC1, which are among the most commonly used native promoters and terminators and also have a relative medium strength in yeast.[19] For circuit 2, promoter CYC1 is paired with terminator MINI. The MINIp-CYC1t and MINIp-MINIt, respectively, serves as the chosen pair for circuit 3 and 4.  </P><P style="font-size:20px">For convenience, we named the“CYC1p-yECitrine-CYC1t-mStrawberry-CYC1t”as“CC”,“CYC1p-yECitrine-MINIt-mStrawberry-CYC1t”as“CM”,“MINIp-yECitrine-CYC1t-mStrawberry-CYC1t”as“MC”, and“MINIp-yECitrine-MINIt-mStrawberry-CYC1t” as “MM”,hereafter.</P><P style="text-align: center; color: gray">
    	Fig 3.3 The plasmid map of our circuit CC, CM, MC, MM. The CC circuit includes the commonly used native promoter CYC1 and terminator CYC1. The MM circuit includes the combination of MINI promoter and MINI terminator. 
   </P></DIV><H2 class="ouc-heading"><STRONG>Reference</STRONG></H2><DIV>
[1]Klemm, D, et al. &quot;Cellulose: fascinating biopolymer and sustainable raw material. &quot; Angewandte Chemie 44.22(2005):3358.
[2]Jmel, M. A, et al. &quot;Physico-chemical characterization and enzymatic functionalization of Enteromorpha sp. cellulose. &quot; Carbohydrate Polymers 135(2016):274-279.
[3]Gancedo, Juana M. &quot;Yeast Carbon Catabolite Repression.&quot; Microbiology &amp; Molecular Biology Reviews Mmbr 62.2(1998):334.
[4]Ha, S. J., et al. &quot;Engineered Saccharomyces cerevisiae capable of simultaneous cellobiose and xylose fermentation.&quot; Proceedings of the National Academy of Sciences of the United States of America 108.2(2011):504.
[5]Liu Zhengkun. &quot;The research on transformed craft of biological ethanol of Enteromorpha prolifera&quot;Diss. Qingdao:Ocean University of China, 2011. 
[6]Fan, L. H., et al. &quot;Engineering yeast with bifunctional minicellulosome and cellodextrin pathway for co-utilization of cellulose-mixed sugars.&quot; Biotechnology for Biofuels 9.1(2016):137.
[7]Zhang, Y., et al. &quot;[Metabolic engineering for microbial production of ethanol from xylose: a review]. &quot; Chinese Journal of Biotechnology 26.10(2010):1436.
[8]Weber, P. C., et al. &quot;Structural origins of high-affinity biotin binding to streptavidin.&quot; Science 243.4887(1989):85-8.
[9]Bloois, Edwin Van, et al. &quot;Decorating microbes: surface display of proteins on Escherichia coli.&quot; Trends in Biotechnology 29.2(2011):79.
[10]Demonte, Daniel, et al. &quot;Structure‐based engineering of streptavidin monomer with a reduced biotin dissociation rate.&quot; Proteins-structure Function &amp; Bioinformatics 81.9(2013):1621.
[11]Tanaka T, Masunari S, Ishii J, et al. Displaying non-natural, functional molecules on yeast surfaces via biotin-streptavidin interaction[J]. Journal of Biotechnology, 2010, 145(1):79-83.
 [12]Park M, Jose J, Thömmes S, et al. Autodisplay of streptavidin.[J]. Enzyme &amp; Microbial Technology, 2011, 48(4):307-311.
[13]Artzi, L, E. A. Bayer, and S. Moraïs. &quot;Cellulosomes: bacterial nanomachines for dismantling plant polysaccharides.&quot; Nature Reviews Microbiology 15.2(2017):83.
[14]Fu, W., and Y. Li. &quot;Establishment of cell surface display system based on N-domain of ice nucleation protein of Xanthomonas.&quot; Chinese Journal of Applied &amp; Environmental Biology 20.03(2014):351-356.
[15]https://2016.igem.org/Team:Kyoto
[16]Redden H,Alper HS,The development and characterization of synthetic minimal yeast promoters[J],Nature Communication,2015,6 : 7810
[17]Curran K A, Morse N J, Markham K A, et al. Short Synthetic Terminators for Improved Heterologous Gene Expression in Yeast[J]. Acs Synthetic Biology, 2015, 4(7):824.
[18]Guo, Z. J., and Sherman, F. (1996) Signals sufficient for 3′-end formation of yeast mRNA. Mol. Cell. Biol. 16, 2772−2776.&quot;
[19]Curran K A, Karim A S, Gupta A, et al. Use of expression-enhancing terminators in Saccharomyces cerevisiae, to increase mRNA half-life and improve gene expression control for metabolic engineering applications[J]. Metabolic Engineering, 2013, 19:88. 
[20]Yamanishi, M., Katahira, S., Matsuyama, T., 2011. TPS1 terminator increases mRNA and protein yield in a Saccharomyces cerevisiae expression system. Biosci. Biotechnol. Biochem. 75, 2234–2236.
 </DIV></DIV></DIV><DIV class="container-fluid ouc-reserve"><P class="text-center">
                          Contact Us : oucigem@163.com  |  ©2017 OUC IGEM.All Rights Reserved.  |  Based On Bootstrap</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>