Team:UCAS-China/HP safety
HOME
PROJECT
DESCRIPTION
LIGHT TO COLOR
LIGHT TO ODOR
SOUND TO LIGHT
RESULTS
PROOF OF CONCEPT
DEMONSTRATE
SOFTWARE
HARDWARE
MODEL
IMPROVE
PARTS
BASIC & COMPOSITE PARTS
PART COLLECTION
HUMAN PRACTICE
OVERVIEW
GOLD—OVERVIEW
GOLD—ART X SCIENCE
GOLD—ORPHEUS
GOLD—GIFTS FOR THE WORLD!
SILVER—OVERVIEW
SILVER—COLLABORATIONS
SILVER—EDUCATION
SILVER—INTERVIEW
SILVER—PUBLIC ENGAGEMENT
SILVER—SAFETY & SECURITY
PEOPLE
MEET THE TEAM!
ATTRIBUTIONS
ACHIEVEMENTS
NOTEBOOK
CONTRIBUTIONS
SAFETY & SECURITY
ACKNOWLEDGEMENT
Safety & Security
The Strategies of Biocontainment of Genetically Modified Organisms
We discussed with Fankang Meng, who was a researcher in synthetic biology, about the biosafety of our project, and the general strategies of biocontainment of genetically modified organisms. We then sorted out and concluded our discussion contents into several aspects, wishing to provide suggestions on biosafety for iGEMers in the future.
To eradicated the escaping problem of genetically modified organisms and horizontal gene transfer between artificial and natural organisms, we highlighted three kinds of biocontainment strategies: traditional biocontainment strategies, the orthogonalization of central dogma and the design of complex genetic networks. [2]
1.Traditional Biocontainment strategies:
Figure 1. Established biocontainment strategies of genetically modified organisms (A)Auxotroph; (B) simple kill switches; (C) inducible gene switch to control essential gene; (D) gene flow barrier
1.1 Auxotroph:
There are some genes that regulate the produce of some molecules necessary for cells. The bacteria with these genes knocked out can survive only if they can absorb these essential molecules from the culture medium. If the bacteria escape into the environment, they will not grow and survive due to the lack of essential molecules.
1.2 Simple Kill switches:
The genome of the bacteria was modified to contain the kill switches controlled by inducible promoters. In the controlled environment exist some molecules which can repress the expression of the kill gene, and the cells can grow and reproduce. However, if the bacteria escape into the environment, the kill switch will be activated and kill themselves.
1.3 Inducible Gene Switch to Control essential gene:
Some essential genes in the genome are modified to be controlled by inducible promoters. In the controlled environment exist the molecules which activate the gene expression and provide the cell with basic substance, while in the uncontrolled environment the essential genes can not express and produce the basic substance and the bacteria cannot survive.
1.4 Gene Flow Barrier:
To avoid lateral gene transfer, the gene flow barrier is introduced into the bacteria. A plasmid containing ‘killer’ gene is transformed into the bacteria which can resist the toxin. When the genes are transferred into another cells in the uncontrolled environment, the toxin-resistance protein is not expressed and the bacteria will die.
2.The orthogonalization of central dogma
2.1 The Introduction of Non-natural Chemicals into Nucleic Acids and Proteins
Figure 2. The introduction of non-natural chemicals into the central dogma(A)Unnatural amino acid; (B) unnatural nucleotide; (C) synthetic ligand
The main strategy of engineering non-natural amino acid-dependent organisms is to use directed evolution, to introduce aminoacyl-tRNA synthetase(ssRS), which encodes a specific non-natural amino acid, into the cells. Through the specific recognition of the stop codon(UAG), the targeted insertion of non-natural amino acids with special functions in the target protein is achieved. As for non-natural nucleotide, artificial base pairs such as Dx & Ps[3], Z & P[4], d5SICS-dMMO2[5] & d5SCICS-dNAM[6], have been developed to block the information transfer among bacteria in controlled and non-controlled environment. Furthermore, as the phenomenon of allosteric regulation of enzymes by ligands is widespread in organisms, synthetic ligands are also considered to be applied to contain the genetically modified organisms. And a directed evolution platform (SliDE) to screen has been established to screen for the accessory ligands which proteins require to function normally[7].
2.2 The Orthogonalization of Macromolecular Machines
Figure 3. Orthogonalization of macromolecular machines:  (A) Orthogonal DNA polymerases, (B) Orthogonal RNA polymerases,(C) Ribo-T, (D) orthogonal ribosomes and orthogonal mRNA translation system
Through the orthogonal design and modification of macromolecular machines in central dogma, the genetic information of the orthogonal system is not only isolated from the host organisms, but also blocked from other uncontrolled organisms. The orthogonaliztion of macromolecular machines mainly include orthogonal DNA polymerases, orthogonal RNA polymerases, orthogonal ribosomes and orthogonal mRNA translation system, Ribo-T and mirror genetic system.
3.The Design of complex Genetic Networks:
Figure 4.  Complex genetic network as a strategy for biocontainment:(A)Deadman, (B)passcode
The strategies described above are mainly targeted at single target, which means that engineered microbes can escape form the system through mutation, recombination or immunity against toxin genes. By introducing redundancy into the system, genetically engineered organisms have to mutate at multiple sites to break through the biocontainment system. At the same time, due to the existence of multiple independent biocontainment systems, The time and divisions of the cells to breakthrough the biocontainment system increased significantly, and thus the escape rate is greatly reduced. As shown in Figure 5, ‘Deadman’ and ‘passcode’[8] have been developed to combine complex gene circuits to achieve biocontainment of genetically engineered organisms.
References
[1] Fernandez-Rodriguez J, Moser F, Song M, et al. Engineering RGB color vision into Escherichia coli[J]. Nature Chemical Biology, 2017, 13(7):706-708.
[2] Fankang M., Chunbo L., Research Progress in Biocontainment of Genetically Modified Organisms[J]. Chinese Journal of Organic Chemistry, 2018, 38(9):2231-2242.
[3] Kimoto M, Kawai R, Mitsui T, et al. Efficient PCR amplification by an unnatural base pair system[J]. Nucleic Acids Symp Ser, 2008, 52(52):469-470.
[4] Yang Z, Hutter D, Sheng P, et al. Artificially expanded genetic information system: a new base pair with an alternative hydrogen bonding pattern[J]. Nucleic Acids Research, 2006, 34(21):6095-6101.
[5] Kim H J, Leal N A, Hoshika S, et al. Ribonucleosides for an artificially expanded genetic information system[J]. Journal of Organic Chemistry, 2014, 79(7):3194-9.
[6] Leconte A M, Hwang G T, Matsuda S, et al. Discovery, characterization, and optimization of an unnatural base pair for expansion of the genetic alphabet.[J]. Journal of the American Chemical Society, 2008, 130(7):2336-43.
[7] Lopez G, Anderson J C. Synthetic Auxotrophs with Ligand-Dependent Essential Genes for a BL21(DE3) Biosafety Strain.[J]. Acs Synthetic Biology, 2015, 4(12):1279.
[8] Chan C T Y, Lee J W, Cameron D E, et al. “Deadman” and “Passcode” microbial kill switches for bacterial containment[J]. Nature Chemical Biology, 2016, 12(2):82-86.
