Team:UCAS-China/People/Notebook
HOME
PROJECT
DESCRIPTION
LIGHT TO COLOR
LIGHT TO ODOR
SOUND TO LIGHT
RESULTS
PROOF OF CONCEPT
DEMONSTRATE
SOFTWARE
HARDWARE
MODEL
IMPROVE
PARTS
BASIC & COMPOSITE PARTS
PART COLLECTION
HUMAN PRACTICE
OVERVIEW
GOLD—OVERVIEW
GOLD—ART X SCIENCE
GOLD—ORPHEUS
GOLD—GIFTS FOR THE WORLD!
SILVER—OVERVIEW
SILVER—COLLABORATIONS
SILVER—EDUCATION
SILVER—INTERVIEW
SILVER—PUBLIC ENGAGEMENT
SILVER—SAFETY & SECURITY
PEOPLE
MEET THE TEAM!
ATTRIBUTIONS
ACHIEVEMENTS
NOTEBOOK
CONTRIBUTIONS
SAFETY & SECURITY
ACKNOWLEDGEMENT
Notebook
Notebook can be downloaded here
Protocols
Preparation of competent cells
1. Streak competent cells on LB plates, and incubate overnight.
2.   Pick a single colony and incubate at 37°C in 4ml LB medium.
3.   Add 100l of bacteria solution to 100ml LB medium in conical flask.
4.   Shake the flask at 37 °C until the OD600 reaches 0.4-0.5, about 3h.
5.   Place the conical flask quickly on ice, violently oscillate it to cool it down quickly.
6.   Transfer the medium to a pre-cooled centrifuge tube, centrifuge at 4500 rpm for 10 min, and discard the supernatant.
7.   Add 2/3 volume of pre-cooled CaCl2-MgCl2 mixture to each tube, resuspend the cells, and ice bath for 10 min.
8.   4 degrees, centrifuge at 4500rpm for 10min, completely discard the supernatant
9.   Add 1/25 volume of pre-cooled 0.1M CaCl2 solution to each tube, resuspend the cells, and ice bath for 10 min.
10.  Add 7% volume of pre-cooled DMSO to each tube, ice bath for 10 min, dispense into 1.5 ml EP tube, dispense 100 per EP tube, and store in -80 refrigerator.
Bacteria strain preservation
1.   Mix 900 bacteria solution with 900 glycerin (60%)
2.   Store the mixture in a -80 degree refrigerator
Chemical transformation
Material:
Plasmid solution to be transformed
Competent DH5α cells
LB broth
Selection plates
Ice
1.5mL Microtubes
Methods:
1.Pipette 50µl of competent cells into pre-chilled 1.5ml tube: 50µl in a 1.5ml tube per transformation. Tubes should be labeled and pre-chilled. Keep all tubes on ice.
2.Pipette 4µl of resuspended DNA into 1.5ml tube: Gently pipette up and down a few times. Keep all tubes on ice.
3.Close 1.5ml tubes, incubate on ice for 30min.
4.Heat shock tubes at 42°C for 90 sec: 1.5ml tubes should be in a floating foam tube rack. Place in water bath to ensure the bottoms of the tubes are submerged. Timing is critical.
5.Incubate on ice for 3 min: Return transformation tubes to ice bucket.
6.Pipette 700µl LB media to each transformation.
7.Incubate at 37°C for 1 hours, shaking at 220 rpm
8.Spin down cells at 4000 rpm for 2mins and discard 550µL of the supernatant. Resuspend the cells in the remaining 200µL, and pipette each transformation onto selection plates with appropriate antibiotics. Spread with sterilized spreader or glass beads immediately.
9.Incubate transformations overnight (14-18hr) at 37°C: Incubate the plates upside down (agar side up).
10.Pick single colonies: Pick single colonies from transformations: do a colony PCR to determine whether the transformation is successful.
Colony PCR
Material:
Sterile Water
Selective LB broth
12.5 µL 2*SuperfastTaq mastermix
1 E. coli colony
1 µL of 10 µM forward primer
1 µL of 10 µM reverse primer
Methods:
1.Pick single colonies and incubate 150 µL LB broth in a 96-well plate.
2.Incubate 2 hours at 37°C, 1000rpm.
3.Combine 0.5 µL cell culture, 12.5 µL 2*SuperfastTaq mastermix, 1 µL of 10 µM forward primer, 1 µL of 10 µM reverse primer, and sterile water up to 25 µL.
4.Incubate in the thermocycler, the settings are as follows.
PCR from template
Material:
5x FastPfu buffer
10 µM forward primer
10 µM reverse primer
PCR tube
Sterile water
Template DNA
Methods:
1. In a PCR tube, combine 1 µL of plasmid DNA, 2 µL of 10 µM forward primer, 2 µL of 10 µM reverse primer, 10 µL of 5x FastPfu buffer, 1.5 µL of 10 mM dNTP mix, 0.5 µL  of FastPfu and sterile water up to 50 µL.
2. Gently mix the reaction
3. If necessary, collect the liquid to the bottom of the PCR tube by spinning briefly
4. Transfer the PCR tube from ice to a PCR machine preheated to 98°C to begin thermocycling The settings are as follows.
DNA gel electrophoresis
Material:
Agarose Powder
TAE buffer
Gel mould
GoldView
Gel Tank
DNA ladder
DNA loading dye
Methods:
1.Prepare 1% w/v solution of agarose powder in 1x TAE buffer using a conical flask
2.Heat the mixture until agarose is completely dissolved.
3. Add 0.01% GoldView to the solution. Make sure there are no bubbles in the solution.
4. Pour the solution into a gel mould
5.Allows the solution to set (approx 15-20 minutes)
6.Transfer the agarose gel to a tank, remove the comb and apply:
4 µL of the DNA ladder
4 µL of DNA samples with the corresponding amount of DNA loading dye (6X)
7.Run the gel for 14 minutes at 160V
Gibson Assembly
Material:
2x Hieff Clone MultiS Enzyme Premix
Linear DNA to be Assembled
Sterile water
Methods:
1. Combine 10 µL of 2x Hieff Clone MultiS Enzyme Premix, 30-50 ng of linear backbone vector, linear DNA fragments(nf/ nv=5:1) and sterile water up to 20 µL
2.Incubate for 1 hour at 50℃.
Fluorescence[1]
Culturing:
1*antibiotic = 125uL Spec (125mg/mL) + 50uL Trim (50mg/mL) + 50uL Kan (50mg/mL) + 50uL Amp (50mg/mL) + 34uL Cm (34mg/mL)
1. Solid LB:
The first layer: 10mL LB + 0.2* antibiotic
The second layer: 5mL LB + 0.1* antibiotic + 20uL bacteria
2. Culturing 20h under specific light in 37℃ incubator
Photographing：
Enzyme and substrate
Culturing:
1. Solid LB:
The first layer: 10mL LB + 0.2* antibiotic
The second layer: 5mL LB + 0.1* antibiotic + 20uL bacteria + 0.8mL Try (10g/L) + 50uL Rose-gluc (50mg/mL) + 50uL X-Gal (20mg/mL)
2. Culturing 20h under specific light in 37℃ incubator
Photographing：
Drawing
Culturing:
1. Solid LB: the same as that mentioned above
2. Culturing 20h under a pattern of rose producing by a projector
Photographing
the same as that mentioned above
Gene Knock-out[2]
1.General procedure of the experiment
a. Transform plasmid A(pCas) into the host, and select with plate containing kanamycin in 30℃.
b. Select positive clone and incubate in liquid LB (contain kanamycin) and prepare competent cells for electroporation. Add arabinose to 10mM induce the expression of RED 1 hour before centrifuge the culture.
c. electroporate plasmid TargetT, which contain homologous sequence of the host, or plasmid TargetF, which does not contain homologous sequence of the host, and a piece of homologous DNA into the competent cells, incubate at 30℃ for an hour, select with plate containing kanamycin and spectinomycin.
d. After incubating at 30℃ overnight, verify whether the gene is knockout by PCR.
e. Incubate positive clone in liquid LB containing kanamycin and IPTG(5mM) for 8-20 hours, and obtain monoclonal by streak culture, verify the expulsion of plasmid B ((pTargetT or pTargetF).
f. Do b to e again to bacteria who exclude plasmid B.
g. Incubate bacteria at 37 ℃ liquid LB to exclude plasmid A.
2. The construction of pTargetT (contain homologous sequence).
(Use the knockout of cadA as example)
Figure 2 Gene module for gene knockout.
The yellow part in figure changes with the target gene, but the blue part stays the same. 2.1 Design primers for the amplification of N20-sgRNA, which contains the sgRNA sequence of target gene.
Figure3 N20-sgRNAsequence use for knockout of cadA
As seen in the figure, the primers designed are as follow:
Figure 4 The gene sequence used for knockout of cadA (shown on the genome of host)
In the sequence of primer cadAspc, the N20 sequence (red part) is supposed to be a part of sequence of target gene, and in the sequence of target gene, sequence NGG is supposed to follow N20 sequence, that is to say, there is supposed to be sequence in the from of N20+NGG, and no repetition sequence.
After obtaining the product of amplification of N20+sgRNA and overlap amplification of homologous sequence (cadA-1 and cadA-2), use spel/SalⅠ restriction enzyme to cut the homologous sequence, and insert it into pTargetF.
2.2 Tips for design of primes.
a. You can put cutting site of enzyme that has the same cohesive such as xbal and ndel in the sequence of primer cadAspc.
b. It will be convenient if cutting site of sal, pstⅠ, hindIII or bglⅡ are put in the sequence of R primer for homologous sequence.
c. It is also feasible if you connect the two pieces of homologous sequence to the vetctor(pTargetT) straightforward instead of the overlap connection first.
d. It will be convenient for GIBSON connection if you design your primer a little longer for about 20bp.
3. The construction of pTargetF (does not contain homologous sequence).
Amplify pTargetF with primer as follow:
TCCTAGGTATAATACTAGT+N20+GTTTTAGAGCTAGAAATAGC         ACTAGTATTATACCTAGGACTGAGCTAGCTGTCAAG
And digest the template with DpnI, transform the product into DH5α, sequencing toverify.
Reference:
[1] Fernandez-Rodriguez J, Moser F, Song M, et al. Engineering RGB color vision into Escherichia coli[J]. Nature Chemical Biology, 2017, 13(7):706-708.
[2] Jiang W, Bikard D, Cox D, et al. RNA-guided editing of bacterial genomes using CRISPR-Cas systems[J]. Nature Biotechnology, 2013, 31(3):233-239.
