<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCAS-China_Validated_contribution skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCAS-China/Validated contribution</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI id="home-link"><A href="https://2018.igem.org/Team:UCAS-China">HOME</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT
            </A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/LightToColor">LIGHT TO COLOR</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/LightToOdor">LIGHT TO ODOR</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/SoundToLight">SOUND TO LIGHT</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">RESULTS
            </A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Validated_contribution">PROOF OF CONCEPT</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Software">SOFTWARE</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Model">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Improve">IMPROVE</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PARTS
            </A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Composite_Part">BASIC &amp; COMPOSITE PARTS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Part_Collection">PART COLLECTION</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">HUMAN PRACTICE
            </A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/Gold_overview">GOLD—OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/Gold_art-and-science">GOLD—ART X SCIENCE</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/orpheus">GOLD—ORPHEUS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/gifts_for_the_world">GOLD—GIFTS FOR THE WORLD!</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/silver_overview">SILVER—OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Collaborations">SILVER—COLLABORATIONS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/silver_education">SILVER—EDUCATION</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/silver_interview">SILVER—INTERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Public_Engagement">SILVER—PUBLIC ENGAGEMENT</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/HP_safety">SILVER—SAFETY &amp; SECURITY</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE
            </A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/People/Meet_the_Team">MEET THE TEAM!</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/People/Medal_Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/People/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Contributions">CONTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/People/safety">SAFETY &amp; SECURITY</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/People/acknowledgement">ACKNOWLEDGEMENT</A></LI></DIV></DIV><DIV class="container"><H2>Valid Contribution &amp; Proof of Concept</H2><H2>The RGB system </H2><H5>Figure 1. The RGB system composed of four subsystems(sensor array, circuits, resource allocator and actuators) is shown.</H5><P> In our project we introduced the RGB system[1] to stain and to scent our rose using light. The RGB system mainly consists of four modules: a ‘sensor array’, ‘circuits’, a ‘resource allocator’ and ‘actuators’. The ‘sensor array’ combines 3 light sensors, Cph8*(BBa_K2598006), YF1 (BBa_K2598009) and CcasR (BBa_K2598005), which can respond to lights of different wavelengths. CcaSR can sense and be switched on by green (535nm) light. Cph8* is switched off by red (650nm) light, while YF1 is switched off by blue (470nm) light. Cph8* and CcaSR are based on phytochromes in the phycocyanobilin chromophores, which are produced by pcyA and ho1(BBa_K2598018)[3], while YF1 is based on Flavin mononucleotide(FMN) chronmophore.  
</P><P> To activate gene expression, the signals from the red- and blue- light sensors need to be inverted, which is done by connecting them to NOT gates in ‘circuits’. The repressor CI turns off the promoter(P) of K1F, and red light can release the inhibition effect and induce the P promoter. Respectively, the repressor PhlF turns off the promoter(PPhlF) of T3, and blue light can induce the PPhlF. After the process of circuits, red-, green- and blue-light signals can respectively induce the P, PcpcG2-172 and PPhlF promoters.
</P><H5>Figure 2. The RGB system is encoded on 4 plasmids. The genes and genetic parts are shown.</H5><P>   The ‘resource allocator’ which connects the ‘circuits’ and ‘actuators’, is based on a split-RNA polymerase(RNAP) system[2]. In the system, a non-active ‘core’ fragment is constitutively expressed by promoter J23105. The sigma fragments(K1F, CGG, T3) regulated by P, PcpcG2-172 and PPhlF can conjugate with the core fragment to form full-functional RNA polymerases, and the RNA polymerases are then directed to three promoters and activated the expression of actuators respectively. It is worth mentioning that the sigma fragments are chosen that exhibited the least cross-talk. Furthermore, insulators (BydvJ, BBa_K2598010, and RiboJ[4], BBa_K2598014) were added to reduce the crosstalk among the three sigma fragments.   </P>.     
<P> The “actuator” module implement the biological functions that are the outputs of the RGB system. There are three output genes responding to three light sensors activated by different wavelength of light. In order to implement customized outcome, theoretically, you can change them into any genes you want. The actuators we chose here were three kinds of color protein: fluorescent protein, chromoprotein and enzyme which can produce colorful products. The fluorescent proteins mRFP(BBa_K2598065), GFP(BBa_K2598063), BFP(BBa_K2598064) were chosen to test our system and achieve our final results because fluorescent reporters can be measured easily by ELIASA (microplate reader) and flow cytometry and the expression period is much shorter than those of chromoprotein and enzyme. What’s more, We use chormoproteins amilGFP (BBa_K2598055), amilCP (BBa_K2598057) and eforred (BBa_K2598056) to prove our concept. We also constructed a plasmid containing lacZ(BBa_K2598025), bFMO(BBa_K2598029) and gusA(BBa_K2598032), which are enzymes and can generate colorful pigments on the plates. The enzyme LacZ, bFMO and gusA can react with X-gal, tryptophan and Rose-gluc, respectively. The products of these enzymes can form insoluble precipitates and color the plates. We also used enzymes to test our system under a more complexed situation, to prove our concepts.
</P><H5>Figure 3. The compositions of four plasmids we constructed, the actuators in pJFR4 could be replaced by chromoproteins and enzymes.</H5><P>  Six plasmids compose the whole system, pJFR1 (BBa_K2598049), pJFR2 (BBa_K2598050), pJFR3(BBa_K2598051), pJFR4(BBa_K2598053), pJFR5(BBa_K2598052) and pJFR6(BBa_K2598061). The ‘sensor array’ module and the core fragment was integrated into pJFR1. The ‘circuits’ and ‘resource allocator’ were integrated into pJFR2 and pJFR3. And the ‘actuators’ containing fluorescence proteins, chormoproteins and enzymes were integrated into pJFR4, pJFR5 and pJFR6, respectively. We transformed four plasmids into our E.coli and finished our artwork.(For more information, see <A href="https://2018.igem.org/Team:UCAS-China/Composite_Part">PARTS</A>）.</P><P> The actuators we chose were three kinds of color protein: fluorescent protein, chromoprotein and enzyme which can produce colorful products. The fluorescent proteins mRFP(BBa_K2598065), GFP(BBa_K2598063), BFP(BBa_K2598064) were chosen to test our system and achieve our final results because fluorescent reporters can be measured easily by ELIASA (microplate reader) and flow cytometry and the expression period is much shorter than those of chromoprotein and enzyme. What’s more, we use chormoproteins amilGFP (BBa_K2598055), amilCP (BBa_K2598057) and eforred (BBa_K2598056) to prove our concept. We also constructed a plasmid containing lacZ(BBa_K2598025), bFMO(BBa_K2598029) and gusA(BBa_K2598032), which are enzymes and can generate colorful pigments on the plates. The enzyme LacZ, bFMO and gusA can react with X-gal, tryptophan and Rose-gluc, respectively. The products of these enzymes can form insoluble precipitates and color the plates. We also used enzymes to test our system under a more complexed situation, to prove our concepts.
 </P><H5>Figure 4. The compositions of four plasmids we constructed.</H5><P>  Six plasmids composed the whole system, pJFR1 (BBa_K2598049), pJFR2 (BBa_K2598050), pJFR3(BBa_K2598051), pJFR4(BBa_K2598053), pJFR5(BBa_K2598052) and pJFR6(BBa_K2598061). The ‘sensor array’ module and the core fragment was integrated into pJFR1. The ‘circuits’ and ‘resource allocator’ were integrated into pJFR2 and pJFR3. And the ‘actuators’ containing fluorescence proteins, chormoproteins and enzymes were integrated into pJFR4, pJFR5 and pJFR6, respectively. We transformed four plasmids into our E.coli and finished our artwork.(For more information, see  <A href="https://2018.igem.org/Team:UCAS-China/Composite_Part"> PARTS</A>）.</P><H2>The concept: Mixing colors in bacteria cells</H2><P>  Although more and more fluorescent proteins and chromoproteins are edited to generate more and more colors, the number of colors produced by organisms is still limited by the number of the kinds of proteins. Once a new color is needed, researchers have to modify the chromophores of the proteins, which takes much time and effort.
</P><P> So how do we create more colors, to make our rose more colorful in a reasonable and convenient way? Here, we put forward a new concept—mixing color in bacterial cells! Unlike the mix of different bacterial cells which produce different colors as the previous iGEM teams have done[5], we used tandem expression and RGB system to control the ratios of the expression of different colors in bacterial cells, to achieve mixing color in bacterial cells, and stain our roses with more bright colors. 
</P><H2>Primary proof of our concept: the tandem expression</H2><P>To prove our concept, we firstly use the tandem expression of chromoproteins. By putting two chromoprotein RBSs and genes under one promoter, we constructed six plasmids(BBa_K2598043, BBa_K2598044, BBa_K2598045, BBa_K2598046, BBa_K2598047, BBa_K2598048) to tandem express eforred, amilCP, amilGFP and fwYellow chromoproteins.
</P><H5>Figure 5. The color spectrum built from chromoproteins and their tandem expression products.</H5><P>   As shown in the figure 4, we built a color spectrum using chromoproteins and their tandem expression products, which can show orange, pink, yellow and green colors. Furthermore, we can see that the color of the tandem expression products, is always between the colors of the two chromoproteins in the spectrum. That is to say, by applying the physical principles, we can easily mix the color we want using tandem expression of chromoproteins. 
</P><H5>Figure 6. The position of the color of GFP and the mixed color by the tandem expression of amilCP and amilGFP in the spectrum</H5><P> What’s more, using the tandem expression of amilCP and amilCP, which are blue and yellow respectively, we mixed blue and yellow together to create green color, which rarely exist in colors of chromoproteins in iGEM registry. We use color picker and Photoshop to analyze and compare the color we got and that of amilGFP. The RGB value of our color is 184-209-108, while that of amilGFP is 178-191-138. That shows our color from the tandem expression is brighter and greener. Following the physical principles of mixing color and tandem expression rather than struggling to edit the molecular structure of the proteins, we provided a new approach to getting a new color with less cost of time and effort, which will largely enrich the part registry and generate more colors for scientific research and art creation.</P><H2>Function of RGB system &amp; Further Proof of concept</H2><P>We performed simple verification of our concept using tandem expression and got well-fitted results, but the bacteria tandem express several chromoproteins can only have one color at a time, so we tried to use more convenient methods, for example light, to control and change the color of the bacteria. We decided to use the RGB system, which was introduced above, to achieve light-controlled color mix in bacteria, because light sensors on the cell membrane can response differently to the lights with different wavelengths, thus producing different outputs accordingly. By changing the wavelengths of the lights, we can control the response of red-, green- and blue-light sensors and expression of mRFP, GFP and BFP. And for precise quantitative data, we chose fluorescent proteins (GFP, BFP, mRFP) as our actuators to mix the colors. What’s more, to avoid the influence of leakage of the fluorescent proteins and get more colors, we also constructed 3 plasmids which only contained 2 output fluorescent protein genes (GFP &amp; mRFP, BFP &amp; mRFP, GFP &amp; BFP), and transformed into the E.coli with pJFR1, pJFR2, pJFR3.
</P><H5>Figure 7. The process of overlapping the fluorescent pictures, filtering the colors and get the results.</H5><P>We chose lights of 13 wavelengths from 395 nm to 660 nm, to test our system and produce colors. We firstly use light to illuminate our bacteria for hours on the plate, and then use fluorescence camera to excite the red, green and blue fluorescence and capture the photos. Then using imageJ and Photoshop, we can overlap the three pictures, filter the colors and get our results of mixing colors(Figure 6). </P><H5>Figure 8. The color spectrum built from fluorescent color mixing.</H5><H5>Figure 9.The rose we created by using the colors we mixed. </H5><P>  The figure7 demonstrated the results of our fluorescent color mixing. Using color picker to pick out the mixed colors from our pictures, we built a even wider spectrum with more colors, with which we created a colorful rose as shown in the figure 8. This result further proved that our system worked and our concept that we can use convenient method—light control—to achieve the mixing and changing the colors of the bacteria was valid.
</P><H5>Figure 10. The color spectrum built from fluorescent color mixing.</H5><P> To semi-quantitatively analyze the fluorescent plate results, we extracted the RGB value of the pictures and got curves showing the relationship between the R value, G value and B value and the wavelengths of the light. From the figure 9 we could see the trend and the peaks of the curves, that the B value reached the top at the wavelength of 490nm, the G value at 565nm, and the R value at 620nm. The R, G, B value could roughly show the expression of red, green and blue fluorescent proteins and their fluorescence intensity, greatly fitted with the literature data that the blue-light sensor was induced best at 470nm, green-light sensor at 532nm, and red-light sensor at 650nm. These curves further proved the function of our system.   </P><H5>Figure 11. The flow cytometry results shows the distribution of the cells in fluorescence intensity. BV421 represented the blue-fluorescence intensity, while the FITC-A represented the green-fluorescence intensity and Pe-TxR-A represented the red-fluorescence intensity. The horizontal axis shew the fluorescence intensity, while the vertical axis shew the number of bacteria cells.</H5><H5>Figure 12. The spectral response of the complete RGB system. “%maximum induction” is calculated as the fold change of response divided by the maximum fold change across the spectrum.
</H5><P>  For more precise control and prediction of the colors we could generate, we needed to know how the three light sensors responded to lights with different wavelengths and intensity and how the fluorescence changed with time. We firstly use flow cytometry[6] to measure the spectral response of the RGB system. From the figure 10 we could see the distribution of the cells in fluorescence intensity. BV421 represented the blue-fluorescence intensity, while the FITC-A represented the green-fluorescence intensity and Pe-TxR-A represented the red-fluorescence intensity. The horizontal axis shew the fluorescence intensity, while the vertical axis shew the number of bacteria cells. The figure demonstrated the variation of fluorescence intensity where cells were most concentrated with the wavelengths of light. The red- and blue- light data were consistent with our prediction. However, the green-light data was not so satisfying, which was probably because without NOT gate, the green-light circuit had leakage and produced fluorescent proteins even not induced by light. What’s more, the different culture and light-inducing conditions of bacteria on the plates and 96-well plates might also cause the slight inconsistency.(For more details of our optimization, please see  <A href="https://2018.igem.org/Team:UCAS-China/Hardware">HARDWARE</A>)
</P><H5>Figure 13. The curves show the relationship between the input light and fluorescent intensity.
</H5><P> From the flow cytometry results, we knew the correspondence between color and wavelengths of light. We then controlled the intensity of the input light to see the change of the fluorescent intensity over time. The figure 12 demonstrated that the stronger the input light was, the less fluorescence would be. The phenomenon was probably because that under relatively low light, the light sensor could be induced and under relatively strong light, the light can aggravate the burden of the bacteria and thus harmed the growth of the bacteria and reduced the expression of fluorescent proteins. We modelled this result for further explanation.(For more information, please see <A href="https://2018.igem.org/Team:UCAS-China/Hardware">MODEL) </A></P><P> After using fluorescent proteins, we used the plasmid pJFR5 (BBa_K2598052), which contained enzyme genes lacZ (BBa_K2598025), bFMO (BBa_K2598029) and gusA (BBa_K2598032), to created more colors using enzymes. The enzymes could generate pigments on the plates and the results of mixing color was easier to observe, which could provide a new material for artists to create artworks. From the figure 13, we could see the different colors generated by enzymes and our system also worked well. The colors were a little dark, and we thought that the phenomenon might resulted from the high concentrations of the substances and long illumination period, which could be adjusted in the future. </P><H5>Figure 14. The results of mixing color using enzyme.  </H5><P>  By controlling the wavelength and intensity of the input light and the time of exposure, we could predict and control all kinds of colors our bacteria would produce. We did not have to edit the proteins to get a new color, and all we needed to do was to change the input light. Our concept of mixing color on the bacteria cells were proved to be reasonable and provided a convenient way for scientists and artists to create new colors and artworks. Furthermore, we submitted all the parts we used (our collection kit) to the registry, and provided iGEMers a strong tool not only to achieve light-controlled multiple gene expression, but to make more creative artworks, to explore the charming junction of art and science.
</P><H2>References</H2><P>[1]Fernandez-Rodriguez J, Moser F, Song M, et al. Engineering RGB color vision into Escherichia coli[J]. Nature Chemical Biology, 2017, 13(7):706-708.

</P><P>[2] Segallshapiro T H, Meyer A J, Ellington A D, et al. A 'resource allocator' for transcription based on a highly fragmented T7 RNA polymerase.[J]. Molecular Systems Biology, 2014, 10(7):742. 

</P><P>[3] Gambetta G A, Lagarias J C. Genetic engineering of phytochrome biosynthesis in bacteria[J]. Proceedings of the National Academy of Sciences of the United States of America, 2001, 98(19):10566-10571.

</P><P>[4]Lou C, Stanton B, Chen Y J, et al. Ribozyme-based insulator parts buffer synthetic circuits from genetic context.[J]. Nature Biotechnology, 2012, 30(11):1137-1142.

</P><P>[5] https://2010.igem.org/Team:KIT-Kyoto
</P><P>[6]Davey H M, Kell D B. Flow cytometry and cell sorting of heterogeneous microbial populations: the importance of single-cell analyses[J]. Microbiol Rev, 1996, 60(4):641-696.



</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>