Team:UCAS-China/LightToOdor
HOME
PROJECT
DESCRIPTION
LIGHT TO COLOR
LIGHT TO ODOR
SOUND TO LIGHT
RESULTS
PROOF OF CONCEPT
DEMONSTRATE
SOFTWARE
HARDWARE
MODEL
IMPROVE
PARTS
BASIC & COMPOSITE PARTS
PART COLLECTION
HUMAN PRACTICE
OVERVIEW
GOLD—OVERVIEW
GOLD—ART X SCIENCE
GOLD—ORPHEUS
GOLD—GIFTS FOR THE WORLD!
SILVER—OVERVIEW
SILVER—COLLABORATIONS
SILVER—EDUCATION
SILVER—INTERVIEW
SILVER—PUBLIC ENGAGEMENT
SILVER—SAFETY & SECURITY
PEOPLE
MEET THE TEAM!
ATTRIBUTIONS
ACHIEVEMENTS
NOTEBOOK
CONTRIBUTIONS
SAFETY & SECURITY
ACKNOWLEDGEMENT
LIGHT TO ODOR
Our roses were colorful now, but only fragrance that makes them vivid, appealing and with the soul as a real rose. So there came our second part, light to odor, making our roses more perfect and real. We first tried to use CRISPR/Cas9[1] gene-editing system to knock out the original gene of E. coli producing smell to prevent E. coli from giving off a nasty and unpleasant smell. Then based on the RGB system[2], we achieved the control of multiple odors with light.
Gene knock-out using CRISPER-Cas9
Figure 1. The tnaA and L-tryptophan degradation resulted in the unpleasant odor in E. coli.
The inherent unpleasant odor of E. coli comes from indole produced naturally in the cells’ metabolic process. In the L-tryptophan degradation pathway, Tryptophanase, encoded by tnaA gene, degraded L-tryptophan of indole, which produces the odor in high concentration.[3]
.
L-tryptophan + H2O = indole + pyruvate + NH3.
(The enzyme needs Co-factor: pyridoxal 5'-phosphate)
Figure 2-3. The procedure and the mechanism of CRISPER-Cas9.
Therefore, using CRISPR/Cas9 system, we tried to knock out the tnaA gene. First, we transformed a pCas9 plasmid into the host cell. Then selecting the positive clone, we used electroporation to transform plasmid TargetF (Figure 4) with a pair of specific sgRNA, and a piece of homologous DNA (Figure 5) which consists of the two gene segments located at the either side of tnaA. Guiding by the sgRNA, Cas9, a nuclease, cut the tnaA segment, leaving a leakage between the two homologous DNA. The cell tended to keep the chromosome integrity, so there was the probability to repair the leakage with the piece of homologous DNA we transformed before, because they have two paired sequences on either side. Also, we gave higher-fold of homologous DNA to increase this probability. After the whole process, we needed to cure the exogenous DNA we introduced.
Figure 4. The PCR results of pTarget.
Figure 5. The PCR results of Homologous sequence 1, 2, 1+2.
Fragrance introducing
Then we devoted our effort to introducing diversified odor to make our roses not only vivid ones with traditional flower fragrance, but unique ones with more kinds of odor like lemon and rain. So we improved part of the iGEM project of Paris_Bettencourt in 2014 [4]by putting their parts into our RGB system and under the control of light. What’s more, we also made some adjustments to the concentrations of substrates provided by Paris_Bettencourt in 2014[5]. As tested in the LIGHT TO COLOR , the RGB system functioned well, no matter whether the actuators was fluorescent proteins or enzymes. But for further validation, we used HPLC to detect the products of our actuators.
We constructed a plasmid (BBa_K2598062) containing lims1 (BBa_K2598060), gds (BBa_K2598058), bmst1 (BBa_K2598059) as actuators, which produced the smell of lemon, rain and flower respectively. The function of the system and the actuators was shown in Table 1. Then we transformed the plasmids of RGB system into E.coli, cultured the bacteria by 96-well plates and induced the expression of odor enzymes by light. (More details, please see Hardware ).
Table 1. The function of the actuators and the wavelengths of input light in RGB system
We then made some adjustments to the concentrations of substrates provided by Paris_Bettencourt in 2014, because we found that our E. coli could not grow at the concentration of 5mM of Benzoic acid, which might be due to the burden of four plasmids and the fragility of our bacteria. Therefore, we used the concentration of 1 mM, at which our E.coli can grow normally.
.
HPLC was applied to detect our products in the fluid LB medium. However, we faced great difficulties because rare previous literature had reported the mobile phase, the temperature, and even the parameter settings of the three molecules, especially geosmin and methyl benzoate. So we chose acetonitrile-water, which is the most common mobile phase, to roughly separate the substances in the LB medium and 200nm as the detection wavelength, which was commonly used to detect Limonene[6].
Figure 6. The HPLC results of the standard samples of Limonene, Methyl benzoate and Geosmin. The retention times were 8.770min, 3.764min, 2.835min, respectively.
Figure 7. The HPLC result of the sample induced by blue light. Peaks at 2.703min, and 3.743min were found, which were closed to the peaks of methyl benzoate and geosmin. But further analysis (for example mass spectrometry, MS) were also required to determine the substances which produced these peaks.
The HPLC results of standard samples were shown in Figure 6, while that of our samples induced by Blue light was shown in figure 7. The retention times of Limonene, Methyl benzoate and Geosmin were 8.770min, 3.764min, 2.835min, respectively. The peaks were found in our HPLC results of our samples, but the peaks were not divided and further analysis (for example MS) were required to determine the substances which produced these peaks. What’s more, it required a large amount of time and effort for us to discover the right mobile phase and specific detection wavelengths of the products. Considering that our system had been tested by fluorescent proteins and enzymes and it really worked well, we did not spend more time on the HPLC analysis, but we believed that we could do further researches on our products once the conditions and the parameter settings of our products were reported.(More details of the function of our system, see LIGHT TO COLOR ).
References
[1] Cong L, Zhang F. Genome Engineering Using CRISPR-Cas9 System[J]. Methods in Molecular Biology, 2015, 1239(11):197.
[2]Fernandez-Rodriguez J, Moser F, Song M, et al. Engineering RGB color vision into Escherichia coli[J]. Nature Chemical Biology, 2017, 13(7):706-708.
[3] Li G, Young K D. Indole production by the tryptophanase TnaA in Escherichia coli is determined by the amount of exogenous tryptophan[J]. Microbiology-sgm, 2013, 159(2):402-410.
[4] https://2014.igem.org/Team:Paris_Bettencourt
[5] http://parts.igem.org/Part:BBa_K1403006
[6]Ruilun Z., Qing Z., Shiai XU. Determination of Limonene in Spearmint Oil by HPLC method[J]. Journal of Anhui Agri, 2012, 40(2):743-744.
