<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCAS-China_People_safety skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCAS-China/People/safety</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav navbar-right"><LI id="home-link"><A href="https://2018.igem.org/Team:UCAS-China">HOME</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PROJECT
            </A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Description">DESCRIPTION</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/LightToColor">LIGHT TO COLOR</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/LightToOdor">LIGHT TO ODOR</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/SoundToLight">SOUND TO LIGHT</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">RESULTS
            </A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Validated_contribution">PROOF OF CONCEPT</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Demonstrate">DEMONSTRATE</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Software">SOFTWARE</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Hardware">HARDWARE</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Model">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Improve">IMPROVE</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PARTS
            </A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Composite_Part">BASIC &amp; COMPOSITE PARTS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Part_Collection">PART COLLECTION</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">HUMAN PRACTICE
            </A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices">OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/Gold_overview">GOLD—OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/Gold_art-and-science">GOLD—ART X SCIENCE</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/orpheus">GOLD—ORPHEUS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/gifts_for_the_world">GOLD—GIFTS FOR THE WORLD!</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/silver_overview">SILVER—OVERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Collaborations">SILVER—COLLABORATIONS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/silver_education">SILVER—EDUCATION</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Human_Practices/silver_interview">SILVER—INTERVIEW</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Public_Engagement">SILVER—PUBLIC ENGAGEMENT</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/HP_safety">SILVER—SAFETY &amp; SECURITY</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">PEOPLE
            </A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/People/Meet_the_Team">MEET THE TEAM!</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Attributions">ATTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/People/Medal_Achievements">ACHIEVEMENTS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/People/Notebook">NOTEBOOK</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/Contributions">CONTRIBUTIONS</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/People/safety">SAFETY &amp; SECURITY</A></LI><LI><A href="https://2018.igem.org/Team:UCAS-China/People/acknowledgement">ACKNOWLEDGEMENT</A></LI></DIV></DIV><DIV class="container"><H1>Safety &amp; Security</H1><H2>The Safety of our project:</H2><H2>Safe Project Design</H2><P> Our project connects the four elements—sound, light, color, odor—together , and the whole process was divided to three parts: sound to light, light to color, light to odor. The sound to light module is achieved by our software, while the light to color and odor module is based on the RGB system reported by Prof. Christopher A Voigt in 2017[1]. We did some modifications and applications of the system, using the parts previously submitted by iGEM teams.
</P><H2>Safe Laboratory Work</H2><H2>Lab Safety Rules</H2><P>We have already completed and submitted the safety form. We guarantee to follow the safety rules provided by iGEM in our lab work. Besides, we have safety rules in our lab as well. We separate the experimenting area and living area apart, no experimental material is allowed in living area and no living item is allowed in experimenting area. When we do experiments, we strictly follow the protocol and are monitored by the lab manager.
</P><H2>Safe Parts</H2><P>The parts of the RGB system we used in our project have been edited to reduce the toxin by the Lab of Prof. Christopher A Voigt. The chromoprotein parts we used have been submitted to the registry by Uppsala University team in 2012, and the odor parts we used have been submitted to the registry by Paris Bettencourt in 2014.
</P><H2>Safe organism</H2><P>We used and only used E. coli in our experiments. The strain of E. coli we use are JF1(modified from K12) and DH5α, which means that all the organism we used in our lab are of group 1 risk. What’s more, the bacteria had been used and tested safe by Christopher A Voigt LAB in MIT. In addition, we separated our organisms from other organisms in the lab that are used by others. There is no phage or virus in our lab and our lab is of group 2 risk.</P><H2>Safe Shipment</H2><P>We have already sent our parts to iGEM Headquarters. Our parts are not in the select agents and toxins list. They are safe and well packed. There’s no liquid, no organism, no toxin in the package, only plasmids that are allowed to ship. Our parts are on their way to Boston, bon voyage!</P><H2>The Strategies of Biocontainment of Genetically Modified Organisms</H2><P>We discussed with Fankang Meng, who was a researcher in synthetic biology, about the biosafety of our project, and the general strategies of biocontainment of genetically modified organisms. We then sorted out and concluded our discussion contents into several aspects, wishing to provide suggestions on biosafety for iGEMers in the future.</P><P>To eradicated the escaping problem of genetically modified organisms and horizontal gene transfer between artificial and natural organisms, we highlighted three kinds of biocontainment strategies: traditional biocontainment strategies, the orthogonalization of central dogma and the design of complex genetic networks. [2]</P><H4>1.Traditional Biocontainment strategies: </H4><H5>Figure 1. Established biocontainment strategies of genetically modified organisms (A)Auxotroph; (B) simple kill switches; (C) inducible gene switch to control essential gene; (D) gene flow barrier</H5><H4>1.1 Auxotroph:</H4><P>There are some genes that regulate the produce of some molecules necessary for cells. The bacteria with these genes knocked out can survive only if they can absorb these essential molecules from the culture medium. If the bacteria escape into the environment, they will not grow and survive due to the lack of essential molecules.</P><H4>1.2 Simple Kill switches:</H4><P>The genome of the bacteria was modified to contain the kill switches controlled by inducible promoters. In the controlled environment exist some molecules which can repress the expression of the kill gene, and the cells can grow and reproduce. However, if the bacteria escape into the environment, the kill switch will be activated and kill themselves. </P><H4>1.3 Inducible Gene Switch to Control essential gene:</H4><P>Some essential genes in the genome are modified to be controlled by inducible promoters. In the controlled environment exist the molecules which activate the gene expression and provide the cell with basic substance, while in the uncontrolled environment the essential genes can not express and produce the basic substance and the bacteria cannot survive. </P><H4>1.4 Gene Flow Barrier:</H4><P>To avoid lateral gene transfer, the gene flow barrier is introduced into the bacteria. A plasmid containing ‘killer’ gene is transformed into the bacteria which can resist the toxin. When the genes are transferred into another cells in the uncontrolled environment, the toxin-resistance protein is not expressed and the bacteria will die. </P><H4>2.The orthogonalization of central dogma
</H4>2.1 The Introduction of Non-natural Chemicals into Nucleic Acids and Proteins
<H5>Figure 2. The introduction of non-natural chemicals into the central dogma(A)Unnatural amino acid; (B) unnatural nucleotide; (C) synthetic ligand</H5><P>The main strategy of engineering non-natural amino acid-dependent organisms is to use directed evolution, to introduce aminoacyl-tRNA synthetase(ssRS), which encodes a specific non-natural amino acid, into the cells. Through the specific recognition of the stop codon(UAG), the targeted insertion of non-natural amino acids with special functions in the target protein is achieved. As for non-natural nucleotide, artificial base pairs such as Dx &amp; Ps[3], Z &amp; P[4], d5SICS-dMMO2[5] &amp; d5SCICS-dNAM[6], have been developed to block the information transfer among bacteria in controlled and non-controlled environment. Furthermore, as the phenomenon of allosteric regulation of enzymes by ligands is widespread in organisms, synthetic ligands are also considered to be applied to contain the genetically modified organisms. And a directed evolution platform (SliDE) to screen has been established to screen for the accessory ligands which proteins require to function normally[7]. </P><H4>2.2 The Orthogonalization of Macromolecular Machines
</H4><H5>Figure 3. Orthogonalization of macromolecular machines:  (A) Orthogonal DNA polymerases, (B) Orthogonal RNA polymerases,(C) Ribo-T, (D) orthogonal ribosomes and orthogonal mRNA translation system</H5><P>Through the orthogonal design and modification of macromolecular machines in central dogma, the genetic information of the orthogonal system is not only isolated from the host organisms, but also blocked from other uncontrolled organisms. The orthogonaliztion of macromolecular machines mainly include orthogonal DNA polymerases, orthogonal RNA polymerases, orthogonal ribosomes and orthogonal mRNA translation system, Ribo-T and mirror genetic system.  </P><H4>3.The Design of complex Genetic Networks: </H4><H5>Figure 4.  Complex genetic network as a strategy for biocontainment:(A)Deadman, (B)passcode</H5><P>The strategies described above are mainly targeted at single target, which means that engineered microbes can escape form the system through mutation, recombination or immunity against toxin genes. By introducing redundancy into the system, genetically engineered organisms have to mutate at multiple sites to break through the biocontainment system. At the same time, due to the existence of multiple independent biocontainment systems, The time and divisions of the cells to breakthrough the biocontainment system increased significantly, and thus the escape rate is greatly reduced. As shown in Figure 5, ‘Deadman’ and ‘passcode’[8] have been developed to combine complex gene circuits to achieve biocontainment of genetically engineered organisms.</P><H2>References</H2><P>[1] Fernandez-Rodriguez J, Moser F, Song M, et al. Engineering RGB color vision into Escherichia coli[J]. Nature Chemical Biology, 2017, 13(7):706-708.</P><P>[2] Fankang M., Chunbo L., Research Progress in Biocontainment of Genetically Modified Organisms[J]. Chinese Journal of Organic Chemistry, 2018, 38(9):2231-2242.</P><P>[3] Kimoto M, Kawai R, Mitsui T, et al. Efficient PCR amplification by an unnatural base pair system[J]. Nucleic Acids Symp Ser, 2008, 52(52):469-470.</P><P>[4] Yang Z, Hutter D, Sheng P, et al. Artificially expanded genetic information system: a new base pair with an alternative hydrogen bonding pattern[J]. Nucleic Acids Research, 2006, 34(21):6095-6101.</P><P>[5] Kim H J, Leal N A, Hoshika S, et al. Ribonucleosides for an artificially expanded genetic information system[J]. Journal of Organic Chemistry, 2014, 79(7):3194-9.</P><P>[6] Leconte A M, Hwang G T, Matsuda S, et al. Discovery, characterization, and optimization of an unnatural base pair for expansion of the genetic alphabet.[J]. Journal of the American Chemical Society, 2008, 130(7):2336-43.</P><P>[7] Lopez G, Anderson J C. Synthetic Auxotrophs with Ligand-Dependent Essential Genes for a BL21(DE3) Biosafety Strain.[J]. Acs Synthetic Biology, 2015, 4(12):1279.</P><P>[8] Chan C T Y, Lee J W, Cameron D E, et al. “Deadman” and “Passcode” microbial kill switches for bacterial containment[J]. Nature Chemical Biology, 2016, 12(2):82-86.</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>