<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BIT-China_project_Fine-regulation_circuits_html skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BIT-China/project Fine-regulation circuits.html</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="bannerContainer"><H2> Add a banner to your wiki! </H2><P>You can make the image 980px  by  200px</P><P> Remember to call the file: &quot;<I>Team_BIT-China_banner.jpg</I>&quot; </P></DIV><DIV id="menuContainer"><DIV id="top_nav"><UL><LI><A href="#"><DIV><P>Project</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Description">Description</A><A href="https://2015.igem.org/Team:BIT-China/Design">Design</A><A href="https://2015.igem.org/Team:BIT-China/project_Basic_circuits.html">Basic circuits</A><A href="https://2015.igem.org/Team:BIT-China/project_Fine-regulation_circuits.html">Fine-regulation circuits</A></SPAN></LI><LI><A href="#"><DIV><P>Modeling</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/modeling_Basic_circuits.html">Basic circuits</A><A href="https://2015.igem.org/Team:BIT-China/modeling_Fine-regulation_circuits.html">Fine-regulation circuits</A></SPAN></LI><LI><A href="#" id="active"><DIV><P>Notebook</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Notebook">Laboratory Notes</A><A href="https://2015.igem.org/Team:BIT-China/Experiments">Protocols</A></SPAN></LI><LI><A href="#"><DIV><P>Judging</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Parts">Parts Submission</A><A href="https://2015.igem.org/Team:BIT-China/Achievements">Achievements</A></SPAN></LI><LI><A href=""><DIV id="active"><P>Team</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Team">Team</A><A href="https://2015.igem.org/Team:BIT-China/gallery">Gallery</A></SPAN></LI><LI><A href=""><DIV><P>Practice</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Practices">Human Practice</A><A href="https://2015.igem.org/Team:BIT-China/Collaborations">Collaboration </A></SPAN></LI><LI><A href=""><DIV><P>Safety</P></DIV></A><SPAN id="nav_second_level"><A href="https://2015.igem.org/Team:BIT-China/Safety">Safety</A><A href="https://2015.igem.org/Team:BIT-China/Questions_and_answers">Q/A</A></SPAN></LI><LI><A href="#"><DIV><P>Attributions</P></DIV></A><SPAN id="nav_second_level" style="margin-left:-100px;"><A href="https://2015.igem.org/Team:BIT-China/Attributions">Attributions</A><A href="https://2015.igem.org/Team:BIT-China/Sponsors">Sponsors</A></SPAN></LI></UL></DIV></DIV><DIV id="contentContainer"><DIV class="container" id="container_Resistance_System"><DIV id="Regulation System" class="project"><H2 style="float:left;margin-left:10px;clear:right;">Library of functional acids</H2><P style="clear:left;">
				In our project, the regulation sub-system is constructed to adjust the microenvironment pH by synthesis of acid and alkali. These chemicals can be secreted outside the bacteria and thus change the microenvironment pH. A library for acid synthesis is established and the following kinds of acid are included: lactic acid, formic acid, acetic acid, succinate, 3-hydroxypropionic acid and 3-hydroxybutyrate. Therefore, we can apply different genes in the regulation sub-system according to various situations or wide-ranging needs. 
			</P><UL class="ul_Synthesis_of_acid"><LI><H2>○ Lactic acid<SPAN>[Expand]</SPAN></H2><UL><P>
					L-Lactic acid, one of the most important chiral molecules and organic acids, is produced via pyruvate from carbohydrates in diverse microorganisms catalyzed by an NAD<SUP>+</SUP>-dependent L-lactate dehydrogenase <SUP>[1]</SUP>. The <I>ldhA</I> gene is used in the synthesis of lactic acid. This gene encodes an L-lactate dehydrogenase (L-LDH) which converts pyruvate to L-lactate <SUP>[2]</SUP>. The simple pathway of lactic acid synthesis is shown in the figure below:</P></UL></LI><LI><H2>○ Formic acid<SPAN>[Expand]</SPAN></H2><UL><P>
						Formic acid is a signature fermentation product of many obligate and facultative anaerobes <SUP>[3-5]</SUP>. One core enzyme, pyruvate formate-lyase 1<SUP>[6]</SUP>, is applied to the synthesis of Formic acid. This enzyme work only under anaerobic condition. The figure below shows a simple pathway of formic acid synthesis:
					</P></UL></LI><LI><H2>○ Acetic acid<SPAN>[Expand]</SPAN></H2><UL><P>
						In synthesis of acetic acid, two enzymes, phosphate acetyltransferase (pta) <SUP>[7]</SUP> and acetate kinase A (ackA) <SUP>[8]</SUP> are used. Phosphate acetyltransferase is an enzyme that catalyzes the chemical reaction: 
					</P><P>
						Substrates of this enzyme are acetyl-CoA and phosphate, whereas its two products are CoA and acetyl phosphate <SUP>[9-11]</SUP>. ackA can facilitate the production of acetyl-CoA by phosphorylating acetate in the presence of ATP and a divalent cation<SUP>[12-14]</SUP>. The brief pathway of acetic acid synthesis is shown in the figure below.</P></UL></LI><LI><H2>○ Succinic acid<SPAN>[Expand]</SPAN></H2><UL><P>
					Succinate has been recognized as an important platform chemical that can be produced from biomass. <SUP>[15]</SUP> There are two main methods in fermentation to synthesize succinate(Fig.1,2). One is under aerobic condition and the other is under anaerobic situation. Gene silence technology <SUP>[16]</SUP> can be employed in aerobic pathway. Genes of PoxB, Pta, sdhA and some other enzymes could be knocked out to ensure a high succinic acid yield <SUP>[15]</SUP>.
				</P><DIV style="width:100%; heigth:auto"><DIV style="float:left; width:56.5%;margin-left:0px;margin-right:5%; "><P style="text-align:center;">Fig.1 Anaerobic pathway</P></DIV><DIV style="float:left;width:35%;"><P style="text-align:center;">Fig.2 Shikimic acid pathway</P></DIV></DIV></UL></LI><LI><H2 style="clear:left;">○ 3-hydroxypropionic acid<SPAN>[Expand]</SPAN></H2><UL><P>
					3-Hydroxypropionic acid (3-HP) is an important platform chemical from which several commodity and specialty chemicals can be generated<SUP>[17]</SUP>. DhaB, a coenzyme B12-dependent glycerol dehydratase, converts glycerol to 3-hydroxypropionaldehyde (3-HPA). GdrAB is a glycerol dehydratase-reactivating factor <SUP>[18]</SUP>. GabD4, a novel aldehyde dehydrogenase, is responsible for the oxidation of aldehydes to carboxylic acids <SUP>[19]</SUP>. The brief 3-hydroxypropionic acid synthesis pathway is shown below <SUP>[20]</SUP></P></UL></LI><LI><H2>○ 3-hydroxybutyric acid<SPAN>[Expand]</SPAN></H2><UL><P>
						Poly (3-hydroxybutyrate) (P(3-HB)) is a polyester of 3-hydroxybutyrate that is synthesized by a variety of microorganisms as intracellular carbon and energy storages under unbalanced nutrients conditions <SUP>[21-22]</SUP>. An operon of three genes (organized as phaCAB) encodes the essential proteins for the production of P(3-HB) in the native producer, <I>Ralstonia eutropha</I><SUP>[23]</SUP>. The three genes of the phaCAB operon are <I>phaC</I>, which encodes the polyhydroxyalkanoate (PHA) synthase, <I>phaA</I>, which encodes a 3-ketothiolase, and <I>phaB</I>, which encodes an acetoacetyl coenzyme A (acetoacetyl-CoA) reductase.
						The synthesis pathway of 3-hydroxybutyrate is shown below.
					</P></UL></LI><LI><H2>○ Shikimic acid<SPAN>[Expand]</SPAN></H2><UL><P>
						Shikimic acid is a highly functionalised, six-carbon ring with three chiral carbons and a carboxylic acid functional group <SUP>[24]</SUP>. Shikimic acid pathway exists naturally in microorganisms and therefore they can be employed to overproduce this acid from carbon sources such as glucose. The Shikimic acid pathway is shown below(Fig.3).
					</P><P style="text-align:center;">Fig.3 Shikimic acid pathway</P></UL></LI></UL><H2>○ Results</H2><P>
			There are many kinds of acid in our acid synthesis parts, we don’t have enough time to test them one by one. So we chose gene <I>LdhA</I>, a gene coding L-lactate dehydrogenase and product L-lactic acid, as our model gene to test in our whole system.
			To verified the <I>LdhA</I> gene, we constructed our gene into pET28a (Fig.4,5), and induced its over-expression by IPTG (Isopropyl β-D-1-Thiogalactopyranoside) under 16 ℃ in BL21(DE)3.
		</P><P style="text-align:center;">Fig.4 The plasmid pET28a and gene <I>LdhA</I></P><P style="text-align:center;">Fig.5 The constructed result of <I>LdhA</I>-pET28a</P><P>
			Meanwhile, we put glucose as the substrate of glycolysis (40% glucose solution, add 2ml to 48ml culture solution). After 12 hours fermentation, we measured the pH of the culture fluid. We use the pET28a(<I>BL21(DE3)</I>) as the control. We repeated this process three times and our result is shown in Table.1.
		</P><P style="text-align:center;margin-bottom:10px;"><B>Table.1</B> The pH value of culture fluid</P><TABLE id="program_table" cellspacing="0"><TBODY><TR id="program_table_firstline"><TH>Group</TH><TH>pH value(Three times)</TH><TH>pH value(average)</TH></TR><TR><TH rowspan="3">pET28a(Aerobic condition)</TH><TD>7.40</TD><TD rowspan="3">7.45</TD></TR><TR><TD>7.53</TD></TR><TR><TD>7.42</TD></TR><TR><TH rowspan="3">pET28a(Anaerobic condition)</TH><TD>6.28</TD><TD rowspan="3">6.25</TD></TR><TR><TD>6.22</TD></TR><TR><TD>6.25</TD></TR><TR><TH rowspan="3"><I>LdhA</I>(Aerobic condition)</TH><TD>7.09</TD><TD rowspan="3">7.15</TD></TR><TR><TD>7.18</TD></TR><TR><TD>7.18</TD></TR><TR><TH rowspan="3"><I>LdhA</I>( Anaerobic condition )</TH><TD>5.25</TD><TD rowspan="3">5.27</TD></TR><TR><TD>5.30</TD></TR><TR><TD>5.26</TD></TR></TBODY></TABLE><P>
			According to the Table.1, we could easily found that there is an obvious difference between experimental group and control. Under the anaerobic condition, L-lactate dehydrogenase, coding by <I>LdhA</I>, acts much better ability than under aerobic condition. However, the final pH value of culture fluid is bigger than <I>pKa</I> of lactic acid. We suppose that the bacteria take advantage of lactic acid, due to the lack of carbon source.
		</P><H2>○ Conclusion</H2><P>
			There are too many kinds of acid, so we didn’t have enough time to verify them. However, we chose LdhA, coding lactate dehydrogenase which converts pyruvate to lactate, as our model gene to test in our whole system. According to the result, we could confirm that our acid production part is working. However, it’s not difficult to find that most of acid production genes are related to oxidative respiratory chain, and others verified them through gene silence. So we prefer to introduce CRISPR/Cas to make this part work better.The summary of acid synthesis library is shown below(Table.2).
		</P><P style="text-align:center;"><B>Table.2</B> The summary of acid synthesis library</P><DIV id="program_table_container"><TABLE id="program_table" cellspacing="0"><TBODY><TR id="program_table_firstline"><TH>Kinds of acid</TH><TH>Gene used</TH><TH>Acidity strength(Pka1)</TH></TR><TR><TH>Lactic acid</TH><TD><I>ldhA</I></TD><TD>3.86</TD></TR><TR><TH>Formic acid</TH><TD><I>pflB</I></TD><TD>3.75</TD></TR><TR><TH>Acetic acid</TH><TD><I>Pta＆ackA</I></TD><TD>4.76</TD></TR><TR><TH>Succinic acid(Aerobic pathway)</TH><TD>\</TD><TD>4.21</TD></TR><TR><TH>Succinic acid(Anaerobic pathway)</TH><TD>\</TD><TD>4.21</TD></TR><TR><TH>3-hydroxypropionic acid</TH><TD><I>DhaB</I></TD><TD>4.50</TD></TR><TR><TH>3-hydroxybutyric acid</TH><TD><I>phaC</I>, <I>phaA</I> and <I>phaB</I></TD><TD>\</TD></TR><TR><TH>Shikimic acid</TH><TD><I>aroB</I>, <I>aroD</I>~<I>aroH</I></TD><TD>3.87±0.6</TD></TR></TBODY></TABLE></DIV><H2 style="float:left;margin-left:10px;clear:right;">Error-Prone PCR</H2><H2 style="clear:left;">○ Background</H2><P style="clear:left;">
				Similar with Polymerase chain reaction(PCR), error-prone PCR is also a powerful method to produce linear DNA fragments, but results in random mutagenesis. This mutagenic PCR, which introduces random mutations by reducing the fidelity of the DNA polymerase<SUP>[25]</SUP>. This process is similar to standard PCR cloning of a target sequence except that, for constructing a diverse mutagenesis library, a much larger pool of transformants are needed to increase the diversity of the library, making the process tedious and inefficient<SUP>[26]</SUP>. As a result, various processes have been developed to streamline the methods.
			</P><H2>○ Design</H2><P>
				Step by step instructions(Fig. 6):
				1. Amplify the target gene under error-prone condition.
				2. Ligate the DNA fragment into a plasmid.
				3. Transform the plasmid into a competent cell.
				4. Pick out mutants shifting to 96-well plates for more experiments<SUP>[27][28]</SUP>.</P><P style="text-align:center;"><B>Fig.6</B> Four steps of EP-PCR</P><P>
				Conventionally, P-atp2 promoter wasn’t as well function as a mechanical device, and our machine could never use the only one alkali-induced promoter to regulate all the complex situation.  What’s more, we anticipated knowing the accurate response range of P-atp2 promoter, so we could extend it. Thus, the idea was appearing, by using error-prone PCR to mutate promoter, which has different efficiency and different ranges of response to the environment.

				The process of mutating P-atp2 is cockamamie, when it does not require excellent skill of experiment or advanced technology. Even though there are several ways to change the reaction condition, we finally chose to put manganese ion into our error-prone reaction system. 

				After the amplification, agarose gel electrophoresis analyzed the promoter DNA fragment, which was using the P-atp2 as the template of error-prone PCR. The size of P-atp2 is about 355bp. Then the right gel was extracted. And another agarose gel electrophoresis analysis was done, while the P-atp2 was linked to <I>Lac Z alpha</I> by using the OE-PCR. Then we ligated the production into pXMJ19 and transformed the plasmid into <I>BMTOP10</I>. 
				While the whole positive-bacteria couldn’t be active 100%, our promoters still needed to be verified. 
				In order to attain our goal, the strategy we have devised was measuring the activity of enzyme β-galactosidase(<I>LacZ</I>). Most recently, we picked out every other 94 bacteria of experiment groups into a 96-well cell culture plates, the rest of two wells were the negative control(NC) and the positive control(PC). As these bacteria grew normally and the OD<SUB>600</SUB> was up to about 1.5, we lysed cells, then added ONPG to start a color reaction and recorded the response time. After the sodium carbonate was added to terminate the reaction, we measured the OD<SUB>420</SUB> and OD<SUB>550</SUB>, which could be put into the formula to calculate the activity.(Table.3)</P><P style="text-align:center;"><B>Table.3</B> the formula to calculate the activity of β-galactosidase</P><DIV style="border:1px solid black;width:50%;margin:0 auto;padding-top:10px;padding-bottom:10px ;"><P style="text-align:center;">Enzyme Activity= 1000*(OD<SUB>420</SUB>-1.75*OD<SUB>550</SUB>)/(t*0.1*OD<SUB>600</SUB>)</P></DIV><H2>○ Result</H2><P>
				In the first round of experiments, we obtained over 1700 bacterial colonies, ultimately, there was about one thousand of them were positive-bacteria. In the second verification , we selected 9 typical bacteria whose promoter has strong activity.
			</P><P style="text-align:center;"><B>Table.4</B></P><TABLE id="PCR_table" style="width:80%;"><TBODY><TR id="program_table_firstline"><TH rowspan="2">Promoter</TH><TH colspan="5">Enzyme Activity</TH></TR><TR><TH>pH=5</TH><TH>pH=6</TH><TH>pH=7</TH><TH>pH=8</TH><TH>pH=9</TH></TR><TR id="pcr_pic1" style="cursor:pointer;"><TH>P-atp2-mutant-140</TH><TD>5481.93</TD><TD>2035.64</TD><TD>2265.33</TD><TD>3545.26</TD><TD>8650.00</TD></TR><TR id="pcr_pic2" style="cursor:pointer;"><TH>P-atp2-mutant-154</TH><TD>1757.99</TD><TD>2181.08</TD><TD>1918.48</TD><TD>6786.56</TD><TD>5787.04</TD></TR><TR id="pcr_pic3" style="cursor:pointer;"><TH>P-atp2-mutant-202</TH><TD>3333.33</TD><TD>2088.99</TD><TD>1335.43</TD><TD>1842.95</TD><TD>4730.26</TD></TR><TR id="pcr_pic4" style="cursor:pointer;"><TH>P-atp2-mutant-226</TH><TD>3588.87</TD><TD>2659.88</TD><TD>1913.91</TD><TD>3011.70</TD><TD>5671.05</TD></TR><TR id="pcr_pic5" style="cursor:pointer;"><TH>P-atp2-mutant-317</TH><TD>3225.00</TD><TD>3866.46</TD><TD>2764.05</TD><TD>8458.33</TD><TD>5229.01</TD></TR><TR id="pcr_pic6" style="cursor:pointer;"><TH>P-atp2-mutant-399</TH><TD>4284.27</TD><TD>3282.19</TD><TD>4856.48</TD><TD>3938.55</TD><TD>6722.07</TD></TR><TR id="pcr_pic7" style="cursor:pointer;"><TH>P-atp2-mutant-430</TH><TD>7388.89</TD><TD>5059.94</TD><TD>787.13</TD><TD>1055.39</TD><TD>969.83</TD></TR><TR id="pcr_pic8" style="cursor:pointer;"><TH>P-atp2-mutant-705</TH><TD>1750.73</TD><TD>2395.83</TD><TD>1917.76</TD><TD>4949.71</TD><TD>19923.37</TD></TR><TR id="pcr_pic9" style="cursor:pointer;"><TH>P-atp2-mutant-706</TH><TD>1062.38</TD><TD>1382.87</TD><TD>3642.15</TD><TD>9321.94</TD><TD>10627.89</TD></TR><TR><TH>Positive Control</TH><TD>1662.88</TD><TD>1892.42</TD><TD>2416.13</TD><TD>4065.80</TD><TD>3086.04 </TD></TR></TBODY></TABLE><H2 style="float:left;margin-left:10px;clear:right;">Recombinase System</H2><H2 style="clear:left;">○ Introduction</H2><P>
				Recombinase, most of which come from bacteria and fungus, is a kind of enzyme that can make gene recombine <SUP>[27][28]</SUP>. It’s usually used to change the structure of genome, regulate the expression of gene and so on. A special sign site in genome (30-40bp) can be recognized by DNA recombinase, and the DNA recombinase will catalyze the recombination of the gene between two sign sites <SUP>[29]</SUP>. This ability allows the recombinase four basic functions: gene excision and insert, inversion, translocation and cassette exchange. These functions can be applied to edit the structure of genome, and also to regulate the expression of gene directly <SUP>[30][31]</SUP>. There are many kinds of recombinase, such as Cre, Flp, RecA/RAD51, Tre, Hin and others. Different kinds recombinase have different recognized site, so they wouldn’t interfere with other recombinases’ function.
			</P><H2>○ Design</H2><P>
 				In order to fine regulate the pH of environment, we introduce the recombinase system here. We have two libraries of acid and alkali, when the different assembles of acid and alkali is used in our whole system, we need to use different methods to regulate it. So we designed three gene circuits using different kinds of recombinase to achieve the fine-regulation function.
 			</P><H4>1)<I>Cre-LoxP</I> and <I>Flp-FRT</I> circuits</H4><P>
				Cre is a kind of Tyrosine recombinase (YR) from P1 bacteriophage. It can catalyze the recombination of the gene between the two specific DNA sign sites (<I>LoxP</I> site). The <I>Cre-LoxP</I> system consists of recombinase Cre and <I>LoxP</I>. When the two <I>LoxP</I> sites are in the same direction, the gene between them would be excised. However, if two sites are in the opposite direction, the gene would be inverted (Fig.7) <SUP>[32]</SUP>. Cre does not need ATP or other cofactor to achieve its function <SUP>[33][34]</SUP>. The length of <I>LoxP</I> is 34 bp, it contains two inverted repeat sequences (13 bp) and the interval sequence (8 bp). The interval sequence also can define the direction of <I>LoxP</I>(Fig.8).
			</P><P style="text-align:center;">Fig.7 The function of the recombinase</P><P style="text-align:center;">Fig.8 The sequence of <I>LoxP</I> site</P><P>
				Flp is the other kind of recombinase found in Saccharomyces cerevisiae. It can recognize the <I>FRT</I> sign site. Flp and <I>FRT</I> form the <I>Flp-FRT</I> system. The function of <I>Flp-FRT</I> system is the same as <I>Cre-LoxP</I> system’s. <I>FRT</I> site has the similar structure with <I>LoxP</I> site (Fig.9) <SUP>[35]</SUP>.
			</P><P style="text-align:center;">Fig.9 The sequence of FRT site</P><P><I>Cre-LoxP</I> system and <I>Flp-FRT</I> system can be used in Recombinase-mediated cassette exchange (RMCE) <SUP>[36][37]</SUP>, so we use the principle of RMCE and design the first kind of our fine-regulation circuits (Fig.10).
			</P><P style="text-align:center;">Fig.10 The gene circuit of Cre/Flp-mediated recombinase circuits</P><P>
				In fermentation process, the environment usually turn acidic, so we produce alkaline substance(γ-aminobutyric acid, GABA) <SUP>[38]</SUP> in the primary state. And alkali-induced promoter P-atp2 and acid-induced promoter P-asr are employed to produce the recombinase Cre and Flp respectively. When the pH of environment reaches the P-atp2’s responsive range, Cre would express and recognize the <I>LoxP</I> sites at both sides of strong constitutive promoter J23119 and it would be inverted. Therefore the bacteria stop to produce lactate<SUP>[39]</SUP>. Similarly, when the pH of environment reaches the P-asr’s responsive range, J23119 would be inverted again. The bacteria will produce GABA again. Through the combination of multiple <I>LoxP</I> sites and <I>FTR</I> sites, we can realize multi-regulation and fine-regulation function.
				To prevent gene from wrong inverting, we designed two kinds of sRNA to inhibit <I>Cre</I> and <I>Flp</I>’s function respectively. When one of recombinases expresses, the sRNA for the other recombinase will be produced at the same time. The sRNA would inhibit the translation of the other recombinase and we can avoid the wrong inversion. sRNA consists of two parts, the scaffold structure <I>MicC</I> and a target-binding sequence. Target-binding sequence, an anti-sequence about 15~20 bp, can inhibit the translation through binding with mRNA <SUP>[40][41]</SUP>.
				These circuits is used when the functional acid and the alkali are both medium.</P><H4 id="Bxb1">2)<I>Bxb1</I> circuit</H4><P>
				The Bxb1 integrase is a DNA recombinase, more precisely a member of serine recombinase family. Bxb1 could be divided to two parts, gp35 and gp47. The gp35 part could recognize specific sequences, called <I>attB</I> and <I>attP</I>, and then integrate, invert, or excise dsDNA depending on the orientation of recognition sequences. When it inverts DNA sequence, the <I>attB</I> and <I>attP</I> sequences are changed into <I>attL</I> and <I>attR</I>, as other DNA recombinases do. Another part called Bxb1 gp47 binds to integrase-DNA complex and this complex flips DNA back into original sequence by regenerating <I>attB</I> and <I>attP</I> sequences. So the efficiency of inversion mediated by Bxb1 is half and half <SUP>[42]</SUP>.
				According to the function of <I>Bxb1</I>, we design the second type circuit to regulate the environmental pH(Fig.11)</P><P style="text-align:center;">Fig.11 The gene circuit of Bxb1-mediated recombinase system</P><P>
				This circuit is applied when the functional acid and the alkali are both strong. In the primary state, the bacteria also produce alkali. Since the alkali is strong, the pH of environment could reach the responsible range of P-atp2 easily. It can lead Bxb1 to work. The promoter J23119 could be inverted persistently until the inversion efficacy reaches half and half. In this process, the bacteria produce the acid and the alkali at the same time, it can create a wave-motion of pH. And the pH of environment will be stable finally. The final pH is influenced by the promoter sensing pH and the strength of acid and alkali.
			</P><H4>3)<I>FimE</I> circuit</H4><P>
				There is another condition. In this condition, we use the assemble of the strong acid and weak alkali, or strong alkali and weak acid. We use FimE to achieve the fine-regulation. This is our gene circuit of FimE-mediated recombinase system. (Fig.12)
			</P><P style="text-align:center;">Fig.12 The gene circuit of FimE-mediated recombinase system</P><P>
				FimE is different from other kind’s recombinase. Its recognization sites on the upstream (<I>IRL</I>) and downstream (<I>IRR</I>) are not same. It means that after the recombination mediated by FimE happens, the sign sites would be exchanged, so that the new sign sites couldn’t be recognized by FimE again. Thus the recombination mediated by FimE is irreversible.<SUP>[43]</SUP>
				Granted that the bacteria produce the strong alkali in the primary, the environment would turn alkaline quickly. So P-atp2 can correspond the change of environment and start the expression of <I>FimE</I>. FimE recognizes <I>IRL</I> and <I>IRR</I>, then catalyze the inversion of J23119. The weak acid would be generated at that time. For the environment, due to the production of the weak acid, it would turn neutral gradually.
				Since the recombinases were found and RMCE was invented, more recombinases’ functions and features were be explored. Recombinase has stick logic relationship and high robust. It fits the modern research of synthetic biology. </P><H2>○ Result</H2><P>
				To test and verify the recombinase system, we replace the <I>mLdhA</I> and <I>gadA</I> to mRFP and GFP respectively. (Fig.13) Through fluorescence, we could verify the recombinase system.
			</P><P style="text-align:center;">Fig.13 The verification circuits of recombinase system</P><P>
				We constructed our verification circuits through OE-PCR, and linked them into vector pSB1C3 (Fig.14~17). And we divided it into recombinase part and fluorescence part.
			</P><P style="text-align:center;">Fig.14 The constructed result of Bxb1, P-atp2 and P-atp2+B0034+Bxb1</P><P style="text-align:center;">(channel 1-3 are P-atp2, Bxb1 and P-atp2+B0034+Bxb1 respectively)</P><P style="text-align:center;">Fig.15 The constructed result of FimE, P-atp2 and P-atp2+B0034+FimE</P><P style="text-align:center;">(channel 1-3 are P-atp2, Bxb1 and P-atp2+B0034+Bxb1 respectively)</P><P style="text-align:center;">Fig.16 The construction result of FimE’s verification device K137058</P><P style="text-align:center;">(channel 1~6 the construction result of K137058, channel 1-3 and 6 are positive.)</P><P style="text-align:center;">Fig.17 The construction result of Bxb1’s verification device</P><P style="text-align:center;">(channel 1~6 the construction result of Bxb1’s verification device, channel 1-3 are positive.)</P><P>
				Meanwhile, we constructed the verification devices, fluorescence part, respectively, then co-transformed them into <I>BMTOP10</I>. We observed their fluorescent condition by fluorescence microscope and floe cytometry.
			</P><P style="text-align:center;">Fig.18 The fluorescence result of FimE’s verification device</P><P style="text-align:center;">Fig.19 The fluorescence result of Bxb1’s verification device</P><H2>○ Conclusion</H2><P>
				In the modern fermentation process, fine-regulation is of vital importance. So this year, in our project, we took advantage of recombinase system to come it true. Although we didn’t have enough time to verify all of them, according to the results of <I>Bxb1</I> system and <I>FimE</I> system, the induced promoter would be inverted in the alkaline environment. Due to the logic relationship and high robust of recombinase, we believed that our recombinase system could realize the fine-regulate functions.
			</P><UL class="reference_container"><LI><H2>References<SPAN>[Expand]</SPAN></H2><P>
				[1] John R P, Nampoothiri K M, Pandey A. Fermentative production of lactic acid from biomass: an overview on process developments and future perspectives[J]. Applied Microbiology &amp; Biotechnology, 2007, 74(3):524-534.
				[2] Niu D, Tian K, Prior B A, et al. Highly efficient L-lactate production using engineered Escherichia coli with dissimilar temperature optima for L-lactate formation and cell growth[J]. Microbial cell factories, 2014, 13(1): 1-11.
				[3] Beyer L. Coordination of FocA and pyruvate formate-lyase synthesis in Escherichia coli demonstrates preferential translocation of formate over other mixed-acid fermentation products.[J]. Journal of Bacteriology, 2013, 195(7):1428-1435.
				[4] Gary Sawers. The hydrogenases and formate dehydrogenases of Escherichia coli.[J]. Antonie Van Leeuwenhoek, 1994, 66(1-3):57-88.
				[5] Sawers R G. Formate and its role in hydrogen production in Escherichia coli.[J]. Biochemical Society Transactions, 2005, 33:42-46.
				[6] Lehtioe L, Leppänen V M, Kozarich J W, et al. Structure of Escherichia coli pyruvate formate-lyase with pyruvate[J]. Acta Crystallographica Section D: Biological Crystallography, 2002, 58(12): 2209-2212.
				[7] Yazdani S, Gonzalez R. Engineering Escherichia coli for the efficient conversion of glycerol to ethanol and co-products [J]. Metabolic engineering, 2008, 10(6): 340-351.
				[8] Rose I A, Grunberg-Manago M, Korey S R, et al. Enzymatic phosphorylation of acetate [J]. Journal of Biological Chemistry, 1954, 211(2): 737-756.
				[9] Bergmeyer H U, HOLZ G, KLOTZSCH H, et al. [PHOSPHOTRANSACETYLASE FROM CLOSTRIDIUM KLUYVERI. CULTURE OF THE BACTERIUM, ISOLATION, CRYSTALLIZATION AND PROPERTIES OF THE ENZYME]. [J]. Biochem Z, 1963, 338.
				[10] Stadtman E R. The purification and properties of phosphotransacetylase.[J]. Journal of Biological Chemistry, 1952, 196(2):527-534.
				[11] Klotzsch H R. [59] Phosphotransacetylase from Clostridium kluyveri[J]. Methods in Enzymology, 1969:381–386.
				[12] Grundy F J, Waters D A, Allen S H, et al. Regulation of the Bacillus subtilis acetate kinase gene by CcpA.[J]. Journal of Bacteriology, 1993, 175(22):7348-7355.
				[13] Oultram J D, Burr I D, Elmore M J, et al. Cloning and sequence analysis of the genes encoding phosphotransbutyrylase and butyrate kinase from Clostridium acetobutylicum NCIMB 8052.[J]. Gene, 1993, 131(1):107-112.
				[14] Simanshu D, Savithri H M. Crystal structures of ADP and AMPPNP-bound propionate kinase (TdcD) from Salmonella typhimurium: Comparison with members of acetate and sugar kinase/heat shock cognate 70/actin superfamily [J]. Journal of Molecular Biology, 2005, 352(4):876-892.
				[15] Thakker C, Martínez I, San K Y, et al. Succinate production in Escherichia coli.[J]. Biotechnology Journal, 2012, 7(2):213–224.
				[16] Yao R, Hirose Y, Sarkar D, et al. Catabolic regulation analysis of Escherichia coli and its crp, mlc, mgsA, pgi and ptsG mutants[J]. Microbial Cell Factories, 2011, 10(3):230-232.
				[17] Raj S M, Rathnasingh C, Jo J E, et al. Production of 3-hydroxypropionic acid from glycerol by a novel recombinant Escherichia coli BL21 strain[J]. Process Biochemistry, 2008, 43(12):1440–1446.
				[18] Kajiura H; Mori K; Tobimatsu T; Toraya T. Characterization and mechanism of action of a reactivating factor for adenosylcobalamin-dependent glycerol dehydratase[J]. Journal of Biological Chemistry, 2001, 276(39):36514-36519.
				[19] Crabb D W, Michinaga M, David C, et al. Overview of the role of alcohol dehydrogenase and aldehyde dehydrogenase and their variants in the genesis of alcohol-related pathology.[J]. Proceedings of the Nutrition Society, 2004, 63(1):49-63.
				[20] Chu H S, Kim Y S, Lee C M, et al. Metabolic engineering of 3-hydroxypropionic acid biosynthesis in Escherichia coli.[J]. Biotechnology &amp; Bioengineering, 2015, 112(2):356-364.
				[21] Chen G P M. Plastics Derived from Biological Sources: Present and Future: A Technical and Environmental Review [J]. Chemical Reviews, 2011, 112(4):2082-2099.
				[22] Wei X X, Zheng W T, Hou X, et al. Metabolic Engineering of Escherichia coli for Poly(3-hydroxybutyrate) Production under Microaerobic Condition[J]. Biomed Research International, 2015, 2015.
				[23] Hiroe A; Tsuge K; Nomura CT; Itaya M; Tsuge T. Rearrangement of gene order in the phaCAB operon leads to effective production of ultrahigh-molecular-weight poly[(R)-3-hydroxybutyrate] in genetically engineered Escherichia coli.[J]. Applied &amp; Environmental Microbiology, 2012, 78(9):3177-3184.
				[24]When the fermentation need to be proceeded in anaerobic condition, but your regulation system need to function in aerobic condition, how will you deal with it?

				［25］ Le Y, Chen H, Zagursky R, et al. Thermostable DNA ligase-mediated PCR production of circular plasmid (PPCP) and its application in directed evolution via in situ error-prone PCR[J]. DNA research, 2013: dst016.
				［26］ Pai J C, Entzminger K C, Maynard J A. Restriction enzyme-free construction of random gene mutagenesis libraries in Escherichia coli[J]. Analytical biochemistry, 2012, 421(2): 640-648.

				[27].Nern A, Pfeiffer B D, Svoboda K et al. Multiple new site-specific recombinases for use in manipulating animal genomes[J]. Proceedings of the National Academy of Sciences, 2011,108(34):14198-14203.
				[28]García-Otín A L, Guillou F. Mammalian genome targeting using site-specific recombinase[J].Front Biosci, 2006, 11(1108):C1136.
				[29]Dymecki S M, Kim J C. Molecular neuroanatomy’s “Three Gs”: a primer[J]. Neuron, 2007, 54(1):17-34.
				[30]Luan H, White B H. Combinatorial methods for refined neuronal gene targeting[J]. Current opinion in neurobiology, 2007, 17(5):572-580.
				[31]Fenno L E, Mattis J, Ramakrishnan C, et al. Targeting cells with single vectors using multiple-feature Boolean Logic[J]. Nature methods, 2014, 11(7): 763-772.
				[32]Nagy A. Cre recombinase: the universal reagent for genome tailoring[J]. genesis, 2000, 26(2):99-109.
				[33]Abremski K, Hoess R. Bacteriophage P1 site-specific recombination. Purification and properties of the Cre recombinase protein[J]. Journal of Biological Chemistry, 1984, 259(3):1509-1514.
				[34]Fan H F. Real-time single-molecule tethered particle motion experiments reveal the kinetics and mechanisms of Cre-mediated site-specific recombination[J]. Nucleic acids research, 2012, 40(13):6208-6222.
				[35]Schlake T, Bode J. Use of mutated FLP recognition target (FRT) sites for the exchange of expression cassettes at defined chromosomal loci[J]. Biochemistrty, 1994, 33(43):12746-12751.
				[36]Takata Y, Kondo S, Goda N, et al. Comparison of efficiency between FLPe and Cre for recombinase-mediated cassette exchange in vitro and in adenovirus vector production[J]. Genes to Cells, 2011, 16(7): 765-777.
				[37]Anderson R P, Voziyanov Y. Flp and Cre expressed from Flp-2A-Cre and Flp-IRES-Cre transcription units mediate the highest level of dual recombinase-mediated cassette exchange[J]. Nucleic acids research, 2012, 40(8):e62-e62.
				[38]Angela, Tramonti, Michele, De Canio, Isabel, Delany, et al. Mechanisms of Transcription Activation Exerted by GadX and GadW at the GadA and GadBC Gene Promoters of the Glutamate-Based Acid Resistance System in Escherichia coli[J]. Journal of Bacteriology, 2008, 188(23):8118-8127.
				[39]John R P, Nampoothiri K M, Pandey A. Fermentative production of lactic acid from biomass: an overview on process developments and future perspectives[J]. Applied Microbiology &amp; Biotechnology, 2007, 74(3):524-534.
				[40]Seung Min Yoo, Dokyun Na &amp; Sang Yup Lee1. Design and use of synthetic regulatory small RNAs to control gene expression in Escherichia coli[J]. Nature, 2013.
				[41]Vadama Sharma, Asami Yamamura, and Yohei Yokobayashi. Engineering Artificial small RNAs for Conditional Gene Silencing in Escherichia coli[J]. Synthetic Biology, 2012, 1, 6(13).
				[42]James G Thomason, Ronald Chan, et al. The Bxb1 recombination system demonstrates heritable transmission of site-specific excision in Arabidopsis[J]. Biotechnology, 2012, 12(9):1765-1980.
				[43] Smith SG, et al. Functional analysis of the FimE integrase of Escherichia coli K-12: isolation of mutant derivatives with altered DNA inversion preferences[J]. Molecule Microbiology, 1999 Dec. PMID 10594822.</P></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>