Team:TJU China/Experiments
<!DOCTYPE >
home
Home
Project
Description
Experiments
Notebook
Demonstrate
Parts
Parts Overview
Basic
Composite
Collection
Improve
Model
Dynamic Model
Off-target Model
Code
HP
Human Practices
Public Engagement
Team
Members
Collaborations
Attributions
Group1
Group2
Group3
Conversion
Conversion
1. Wipe the cleaned work area with 70% ethanol.
2. Thaw the competent cells on ice. A 1.5 mL microcentrifuge tube was labeled for each transformation and                 the tube was placed on ice for pre-cooling. .
3. Rotate the DNA tube from the Competent Cell Test Kit/Conversion Efficiency Kit and collect all DNA to                 the bottom of each tube before use. A quick spin of 20-30 seconds at 8,000-10,000 rpm is sufficient. Note:                 You should resuspend the DNA in each tube with 50 μL of dH2O.
4. Inhale 1 μL of DNA into each microcentrifuge tube.
5. Pipette 50 μL of competent cells into each tube. Gently mix the tube with your fingers.
6. Incubate on ice for 30 minutes.
Now preheat the bath to 42 °C.
7. Heat the cells by placing in a water bath for 90 seconds. Carefully keep the lid of the tube above the                 water level and keep the ice close.
8. Immediately transfer the tube back to the ice and incubate on ice for 5 minutes.
9. Add 950 μL of SOC medium to each tube, incubate at 37 ° C for 1 hour, and shake at 200-300 rpm.
10. Pipette 100 μL from each tube onto the appropriate plate and spread the mixture evenly over the plate.                 Incubate overnight at 37 ° C or for about 16 hours. Place the plate with the agar side on top and the lid                 on the bottom.
Kit DNA conversion
Kit DNA conversion
1. Using a pipette tip, make a hole in the foil cover and enter the corresponding hole in the part. Make sure the board is                 oriented correctly. Do not remove the foil cover as it may cause cross-contamination between the holes.
2. Pipette 10 μL of dH2O (distilled water) into the wells. Pipette up and down several times and let stand                 for 5 minutes to ensure that the dried DNA is completely resuspended. The resuspension will be red because                 the dried DNA has a cresol red dye. We recommend that you do not resuspend dry DNA with TE.
3. Convert 1 μL of resuspended DNA to the competent cells you want, transform your transformation with the                 appropriate antibiotic* and grow overnight.
4. Pick a colony and inoculate the broth (using the correct antibiotic again) and grow for 16 hours.
5. Using the resulting culture, make a small amount of DNA to make your own glycerol stock (further instructions                 on making glycerin see this page). We recommend using QC testing of micro-prepared DNA, such as restriction                 digestion and sequencing.
PCR
PCR
Single enzyme digestion
Single enzyme digestion
Take 1 ng of the verified plasmid, add BμI 1 μL, buffer 0.5 μL, add ddHO to 50 μL, place in a 37 ° C water bath                 for 1 h, then put in a 65 ° C water bath for 20 min, take out and run the gel verification.
connection:
First add a large fragment of 25ng, a small fragment of 1μM, 0.2μL (small fragment is pre-annealed by two                 complementary single-stranded oligonucleotides in a water bath in a boiling water beaker, slowly reduced                 to room temperature), then add T4 ligase 1μL 1 μL of its buffer, add ddHO to 20 μL, place in ice water and                 mix to 22 ° C in a thermos bucket, and connect overnight.
Double digestion:
Add 1 ul of enzyme, add 1 ng (or 2 ng) of plasmid, add buffer, then hydrate, 30 ul system. 37 ° C 40min                 (time can be changed) 80 ° C 5min fire
Extraction of the amplified plasmid
Extraction of the amplified plasmid from the cultured Escherichia coli using TIANprep Mini Plasmid Kit( #DP103,                 TIANgen)
1. Collect the E. coli solution into the EP tube.
2. Re-suspend pelleted bacterial cells in 250ul Buffer P1 (RNase A added, kept at 4 °C) and mix thoroughly.
3. Add Buffer 250ul P2 and gently invert the tube 6-8 times to mix.
4. Add 350μl Buffer P3 and invert the tube immediately and gently 6-8 times.
5. Centrifuge for 10 min at 12,000 rpm in a micro-centrifuge.
6. Regenerate column CP3 while centrifugation. Add 500μl Buffer BL. Centrifuge for 1 min at 12,000 rpm after                 static for 2min. Discard the flow-through.
7. Add supernatant from the EP tube to the column and put it into collection canals. Centrifuge for 1min                 at 12000rpm. Discard the flow-through.
8. Add 600μl Buffer PW(ethanol added) and centrifuge for 1min after static for 2min. Discard the flow-through.
9. Repeat step 8.
10. Centrifuging for 2min at 12000rpm to shake off the rest of the Buffer PW.
11. Place the column in a new EP tube and the opening was allowed to stand for 5 minutes, so that the ethanol                 in the PW can be sufficiently volatilized.
12. Add 70 ul sterile distilled water at 75℃ dropwise to the middle of the adsorbed film. Static for 2min.                 Then centrifuge for 2 min at 12,000 rpm to collect DNA solution in EP tube.
Agarose Gel Electrophoresis
Agarose Gel Electrophoresis (Plasmid & PCR & Cleavage product)
1. Prepare sufficient lx TAE to fill the electrophoresis tank and to cast the gel.
2. Prepare a solution of agarose in electrophoresis buffer at a concentration of 1%: Add 0.9g powdered agarose                 to 90ml of TAE in an Erlenmeyer flask.
3. Heat the slurry in a boiling-water bath or a microwave oven until the agarose dissolves.
4. Use insulated gloves or tongs to transfer the flask/bottle into a water bath at 55°C. When the molten                 gel has cooled, add ethidium bromide to a final concentration of 0.5 ug/ml. Mix the gel solution thoroughly                 by gentle swirling.
5. While the agarose solution is cooling, choose an appropriate comb for forming the sample slots in the                 gel. Position the comb 0.5-1.0 mm above the plate so that a complete well is formed when the agarose is added                 to the mold.
6. Pour the warm agarose solution into the mold.
7. Allow the gel to set completely (30-45 minutes at room temperature), then carefully remove the comb. Pour                 off the electrophoresis buffer and carefully remove the tape Mount the gel in the electrophoresis tank.
8. Add just enough electrophoresis buffer to cover the gel to a depth of ~1 mm.
9. Mix the samples of DNA with 10 ul green buffer
10. Slowly load the sample mixture into the slots of the submerged gel using a disposable micropipette, an                 automatic micro-pipettor. Load size standards into slots on both the right and left sides of the gel.
11. Close the lid of the gel tank and attach the electrical leads so that the DNA will migrate toward the                 positive anode (red lead). Apply a voltage of 1-5 V/cm (measured as the distance between the positive and                 negative electrodes). If the leads have been attached correctly, bubbles should be generated at the anode                 and cathode (due to electrolysis), and within a few minutes, the bromophenol blue should migrate from the                 wells into the body of the gel. Run the gel until the bromophenol blue and xylene cyanol FF have migrated                 an appropriate distance through the gel.
12. When the DNA samples or dyes have migrated a sufficient distance through the gel, turn off the electric                 current and remove the leads and lid from the gel tank.
Gel Extraction purification
Gel Extraction purification using TIANgen Midi Purification Kit (#DP209, TIANGEN)
1.Column equilibrium: add 500 μl equilibrium liquid BL to the adsorption column CA2 (the adsorption column into                 the collection tube) , centrifuge for 1 min at 12,000 rpm in a micro-centrifuge. Then pour out the waste                 liquid from the collection tube, and put the adsorption column back into the collection tube.
2.Cut a single DNA band from agarose gel and put it into a clean centrifuge tube to weigh.
3.Add equal volume solution PN (if gel weight is 0.1 g, add 100ul PN solution), water bath at 50 ℃, during                 which the EP tube is continuously flipped mildly up and down to ensure the full dissolution of the gel. If                 there are still undissolved rubber pieces, keep them for a few minutes or add some more sol solution until                 the block is completely dissolved.
4.The solution obtained from the previous step was added to column CA2 (the column was placed in the collection                 tube), incubate at room temperature for 2 mins, then centrifuge for 1 min at 12,000 rpm in a micro-centrifuge.                 Discard the flow-through and put the adsorption column CA2 into the collection tube.
5.Adding 600 μl PW(ethanol added) to the adsorption column CA2, then centrifuge for 1 min at 12,000 rpm in                 a micro-centrifuge. Discard the flow-through and put the adsorption column CA2 into the collection tube.
6.Repeat step 5.
7.Put the adsorption column CA2 back into the collecting tube, then centrifuge for 2 min at 12,000 rpm in                 a micro-centrifuge in order to remove the rinsing solution as much as possible. The adsorption column CA2                 was then placed at room temperature for a few minutes and dried thoroughly to prevent the residual rinsing                 solution from affecting the next step.
8.Place the adsorption column CA2 in a clean EP tube, add 40-50ul double distilled water (pH 7.0-8.5) to                 the middle of the adsorption membrane, incubate at room temperature for 2 mins. Then centrifuge for 2 min                 at 12,000 rpm in a micro-centrifuge to collect DNA solution.
Extraction of the amplified GFP and RED plasmid
Extraction of the amplified GFP and RED plasmid from the cultured Escherichia coli using TIANprep Mini Plasmid                 Kit( #DP103, TIANgen)
1. Collect the E. coli solution into the EP tube.
2. Re-suspend pelleted bacterial cells in 250ul Buffer P1 (RNase A added, kept at 4 °C) and mix thoroughly.
3. Add Buffer 250ul P2 and gently invert the tube 6-8 times to mix.
4. Add 350μl Buffer P3 and invert the tube immediately and gently 6-8 times.
5. Centrifuge for 10 min at 12,000 rpm in a micro-centrifuge.
6. Regenerate column CP3 while centrifugation. Add 500μl Buffer BL. Centrifuge for 1 min at 12,000 rpm after                 static for 2min. Discard the flow-through.
7. Add supernatant from the EP tube to the column and put it into collection canals. Centrifuge for 1min                 at 12000rpm. Discard the flow-through.
8. Add 600μl Buffer PW(ethanol added) and centrifuge for 1min after static for 2min. Discard the flow-through.
9. Repeat step 8.
10. Centrifuging for 2min at 12000rpm to shake off the rest of the Buffer PW.
11. Place the column in a new EP tube and the opening was allowed to stand for 5 minutes, so that the ethanol                 in the PW can be sufficiently volatilized.
12. Add 70 ul sterile distilled water at 75℃ dropwise to the middle of the adsorbed film. Static for 2min.                 Then centrifuge for 2 min at 12,000 rpm to collect DNA solution in EP tube.
Expression and purification of Cas12a
Expression and purification of Cas12a
Cas12a expression plasmids were transformed into E. coli BL21. For protein expression, a single clone was first cultured                 overnight in 5-mL liquid LB tubes and then 1% (v/v) inoculated into 1 L of fresh liquid LB. Cells were grown                 with shaking at 220 rpm and 37 °C until the OD600 reached 0.8, and IPTG was then added to a final concentration                 of 0.1 mM followed by further culture of the cells at 16 °C for about 16 h before the cell harvesting. Trim                 the bacteria collecting bucket, centrifuge 15min under 20°C, 4000rpm and collect the bacteria and transfer                 the bacteria into two 50ml centrifuge tubes. Cells were resuspended in 50 mL of lysis buffer (50 mM Tris-HCl                 (pH8.0), 1.5 M NaCl, 5% glycerol) and lysed by high pressure. The obtained lysate was then centrifuged twice                 at 18000 rpm for 45 min. After centrifuging, the supernatant was mixed with 5 mL of Ni-NTA beads (GE Healthcare)                 and softly shaken for 1 h at 4 °C before being loaded onto a 30-mL column. The packing was then washed with                 wash buffer (lysis buffer supplemented with 30 mM imidazole) and eluted with elution buffer (lysis buffer                 supplemented with 600 mM imidazole). The elution was dialysed with dialysis buffer 1 [50 mM Tris-HCl (pH8.0),                 1.5 M NaCl, 5% glycerol]. Before the protein solution was loaded onto an anion exchange column (HiTrapTM                 Q HP, GE Healthcare), it was diluted until the final NaCl concentration reached below 80 mM. After that,                 the column was washed and then eluted with a gradient concentration of NaCl (AKTA Pure, GE Healthcare). Fractions                 containing Cas12a proteins were verified by SDS-PAGE and then pooled for dialysis with dialysis buffer 2                 (50 mM Tris-HCl (pH8.0), 1.5 M NaCl,1 mM DTT and 5% glycerol]) overnight. Finally, protein is snap-frozen                 in liquid nitrogen and stored in aliquots at −80 °C.
In vitro digestion of DNA by crRNA and FnCas12a complex
In vitro digestion of DNA by crRNA and FnCas12a complex (fluorescence microscope)
Cleavage on hydrophobic protein substrate
Cleavage on hydrophobic protein substrate
1.Substrate Preparation
In the ventilation table, remove the substrate, put into a one-off cell culture dish (10cm).
If necessary, use water and ethanol circulation ultrasonic to wash glass, high pressure sterilization, and                 dry. Then, wrap them with tin foil in the box.
2.Place hydrophobic protein
  ①Use a special pen to wrap the liquid to prevent dispersing into too large circles. Each circle                 is dripped with 1-5ul hydrophobic protein
  ②Dry in a ventilated place dry, and use depc water to slowly rinse。
4. Mix NEB buffer 3, crRNA, FnCas12a protein and Nuclease-Free Water thoroughly and pulse-spin in a microfuge,                 pre-incubate at 25⁰C for 15 minutes.
5. Incubate for 1 hour at 37⁰C.
6. Proceed with fluorescence microscope detection.
Notice: X, Y, Z, W, M in the table above are various according to our experiment design.
Agarose Gel Electrophoresis
Agarose Gel Electrophoresis (Plasmid & PCR & Cleavage product)
1. Prepare sufficient lx TAE to fill the electrophoresis tank and to cast the gel.
2. Prepare a solution of agarose in electrophoresis buffer at a concentration of 1%: Add 0.9g powdered agarose                 to 90ml of TAE in an Erlenmeyer flask. （for the gel electrophoresis of the crRNA，the final concentration                 of the agarose is 5%.）
3. Heat the slurry in a boiling-water bath or a microwave oven until the agarose dissolves.
4. Use insulated gloves or tongs to transfer the flask/bottle into a water bath at 55°C. When the molten                 gel has cooled, add ethidium bromide to a final concentration of 0.5 ug/ml. Mix the gel solution thoroughly                 by gentle swirling.
5. While the agarose solution is cooling, choose an appropriate comb for forming the sample slots in the gel. Position the comb 0.5-1.0 mm above the plate so that a complete well is formed when the agarose is added                 to the mold.
6. Pour the warm agarose solution into the mold.
7. Allow the gel to set completely (30-45 minutes at room temperature), then carefully remove the comb. Pour                 off the electrophoresis buffer and carefully remove the tape Mount the gel in the electrophoresis tank.
8. Add just enough electrophoresis buffer to cover the gel to a depth of ~1 mm.
9. Mix the samples of DNA with 10 ul green buffer
10. Slowly load the sample mixture into the slots of the submerged gel using a disposable micropipette, an                 automatic micro-pipettor. Load size standards into slots on both the right and left sides of the gel.
11. Close the lid of the gel tank and attach the electrical leads so that the DNA will migrate toward the                 positive anode (red lead). Apply a voltage of 1-5 V/cm (measured as the distance between the positive and                 negative electrodes). If the leads have been attached correctly, bubbles should be generated at the anode                 and cathode (due to electrolysis), and within a few minutes, the bromophenol blue should migrate from the                 wells into the body of the gel. Run the gel until the bromophenol blue and xylene cyanol FF have migrated                 an appropriate distance through the gel.
12. When the DNA samples or dyes have migrated a sufficient distance through the gel, turn off the electric                 current and remove the leads and lid from the gel tank.
Extraction of the amplified plasmid
Extraction of the amplified plasmid from the cultured Escherichia coli using TIANprep Mini Plasmid Kit( #DP103,                 TIANgen)
1. Collect the E. coli solution into the EP tube.
2. Re-suspend pelleted bacterial cells in 250ul Buffer P1 (RNase A added, kept at 4 °C) and mix thoroughly.
3. Add Buffer 250ul P2 and gently invert the tube 6-8 times to mix.
4. Add 350μl Buffer P3 and invert the tube immediately and gently 6-8 times.
5. Centrifuge for 10 min at 12,000 rpm in a micro-centrifuge.
6. Regenerate column CP3 while centrifugation. Add 500μl Buffer BL. Centrifuge for 1 min at 12,000 rpm after                 static for 2min. Discard the flow-through.
7. Add supernatant from the EP tube to the column and put it into collection canals. Centrifuge for 1min                 at 12000rpm. Discard the flow-through.
8. Add 600μl Buffer PW(ethanol added) and centrifuge for 1min after static for 2min. Discard the flow-through.
9. Repeat step 8.
10. Centrifuging for 2min at 12000rpm to shake off the rest of the Buffer PW.
11. Place the column in a new EP tube and the opening was allowed to stand for 5 minutes, so that the ethanol                 in the PW can be sufficiently volatilized.
12. Add 70 ul sterile distilled water at 75℃ dropwise to the middle of the adsorbed film. Static for 2min.                 Then centrifuge for 2 min at 12,000 rpm to collect DNA solution in EP tube.
Expression and purification of S. pyogenes Cas9
E. coli Rosetta competent cells were transformed with pET-Cas9-NLS-6xHis which was purchased from Addgene. The                 plasmid encodes the S. pyogenes Cas9 fused to a C-terminal His-tag and an N-terminal NLS. The resulting single                 colony was grown in Luria-Bertani (LB) media at 37 °C overnight, and the overnight culture (5 mL) was inoculated                 into LB (1 L) in the presence of ampicillin (100μg/ml) and chloramphenicol(100μg/ml) at 37℃ for 6h. The Cas9                 protein was induced with 1 mM of isopropyl β-D-1-thiogalactopyranoside at 18℃ for 16 h. The cells were collected                 by centrifugation at 4,000rpm for 15 min at 4 ℃. The pellets were harvested, and resuspended in Buffer A(50                 mM Tris(hydroxymethyl)aminomethane hydrochloride) (Tris-HCl, pH 8.0), 1 M NaCl, 20% glycerol, 20 mM imidazole                 and 2 mM Tris(2-carboxyethyl)phosphine (TCEP). The cells were lysed and the soluble lysate was obtained by                 centrifugation at 18,000r for 40 min at 4℃. The cell lysate was incubated with His-Pure nickel-nitriloacetic                 acid (nickel-NTA) resin at 4℃ for 30 min to capture His-tagged Cas9. The resin was transferred to a 20-ml                 column and washed with 20 column volumes of Buffer A. Cas9 was eluted in Buffer B( 50 mM Tris-HCl (pH 8),                 1 M NaCl, 20% glycerol, 2 mM TCEP and 500 mM imidazole), and concentrated by Amicon ultracentrifugal filter                 (30-kDa molecular weight cut-off) to 1ml. The eluted Cas9 was loaded onto a HiTrap SP HP, 5ml and purified                 using a linear gradient of NaCl from 0.1M to 1M in buffer C (50mM Tris-HCl, pH8.0, 20% glycerol, 20 mM imidazole                 and 2mM TCEP). The collected liquid was concentrated to 1ml. The eluent, containing Cas9, was injected into                 a molecular sieve column，concentrated and exchange the buffer to Storage Buffer(300 mM NaCl, 10 mM Tris-HCl,                 0.1 mM EDTA ,1 mM DTT ,50% Glycerol ,pH 7.4 at 25℃). Finally, the Cas9 protein was quantified by Bicinchoninic                 acid (BCA) assay, snap-frozen in liquid nitrogen and stored in aliquots at −80 °C.
Assembly of the template of sgRNA by PCR
Purification of PCR product
Purification of PCR product using GeneJET PCR Purification Kit (#K0701, Thermo Scientific)
1.Add a 1:1 volume of Binding Buffer to completed PCR mixture (e.g. for every 100 µL of reaction mixture, add                 100 µL of Binding Buffer). Mix thoroughly. Check the color of the solution. A yellow color indicates an optimal                 pH for DNA binding. If the color of the solution is orange or violet, add 10 µL of 3 M sodium acetate, pH                 5.2 solution and mix. The color of the mix will become yellow.
2.for DNA ≤500 bp
Optional: if the DNA fragment is ≤500 bp, add a 1:2 volume of 100% isopropanol (e.g., 100 µL of isopropanol                 should be added to 100 µL of PCR mixture combined with 100 µL of Binding Buffer). Mix thoroughly.
3.Transfer up to 800 µL of the solution from step 1 (or optional step 2) to the GeneJET purification column.                 Centrifuge for 30-60 s. Discard the flow-through. Notes. If the total volume exceeds 800 µL, the solution                 can be added to the column in stages. After the addition of 800 µL of solution, centrifuge the column for                 30-60 s and discard flowthrough. Repeat until the entire solution has been added to the column membrane.                 Close the bag with GeneJET Purification Columns tightly after each use!
4.Add 700 µL of Wash Buffer (diluted with the ethanol as described on p. 3) to the GeneJET purification column.                 Centrifuge for 30-60 s. Discard the flow-through and place the purification column back into the collection                 tube.
5.Centrifuge the empty GeneJET purification column for an additional 1 min to completely remove any residual                 wash buffer.
6.Transfer the GeneJET purification column to a clean 1.5 mL microcentrifuge tube (not included). Add 50                 µL of Elution Buffer to the center of the GeneJET purification column membrane and centrifuge for 1 min.
7.Discard the GeneJET purification column and store the purified DNA at -20 °C.
In vitro transcription of sgRNA
In vitro transcription of sgRNA using T7 High Efficiency Transcription Kit (#JT101, TRANSGEN)
2.Mix thoroughly and incubate at 37℃ for 16h.
3.Proceed with purification.
Purification of Transcription product
Purification of Transcription product using MEGAclear™ Kit (#AM1908, Life Science)
1. Bring the RNA sample to 100 µL with Elution Solution. Mix gently but thoroughly.
2. Add 350 µL of Binding Solution Concentrate to the sample. Mix gently by pipetting.
3. Add 250 µL of 100% ethanol to the sample. Mix gently by pipetting.
4. Apply the sample to the filter
  a. Insert a Filter Cartridge into 1 of the Collection and Elution Tubes supplied.
  b. Pipet the RNA mixture onto the Filter Cartridge.
  c. Centrifuge for ~15 sec to 1 min, or until the mixture has passed through the filter. Centrifuge                 at RCF 10,000–15,000 × g (typically 10,000–14,000 rpm). Spinning harder than this may damage the filters.
  d. Discard the flow-through and reuse the Collection and Elution Tube for the washing steps.
5. Wash with 2 × 500 µL Wash Solution(ethanol added).
  a. Apply 500 µL Wash Solution. Draw the Wash Solution through the filter as in the previous step.
  b. Repeat with a second 500 µL aliquot of Wash Solution.
  c. After discarding the Wash Solution, continue centrifugation or leave the Filter Cartridge                 on the vacuum manifold for 10–30 sec to remove the last traces of Wash Solution.
6. Elute RNA from the filter with 50 µL Elution Solution.
  a. Pre-heat 110 µL of Elution Solution per sample to 95° C.
  b. Apply 50 µL of the pre-heated Elution Solution to the center of the Filter Cartridge, close                 the cap of the tube and centrifuge for 1 min at room temperature (RCF 10,000–15,000 x g) to elute the RNA.
  c. To maximize RNA recovery, repeat this elution procedure with a second preheated 50 µL aliquot                 of Elution Solution. Collect the eluate into the same Collection/Elution Tube.
In vitro digestion of DNA by sgRNA/Cas9 complex
In vitro digestion of DNA by BODIPY/RNPs complex
Agarose Gel Electrophoresis
Agarose Gel Electrophoresis (Plasmid & PCR & Cleavage product)
1. Prepare sufficient lx TAE to fill the electrophoresis tank and to cast the gel.
2. Prepare a solution of agarose in electrophoresis buffer at a concentration of 1%: Add 0.9g powdered agarose                 to 90ml of TAE in an Erlenmeyer flask.
3. Heat the slurry in a boiling-water bath or a microwave oven until the agarose dissolves.
4. Use insulated gloves or tongs to transfer the flask/bottle into a water bath at 55°C. When the molten                 gel has cooled, add ethidium bromide to a final concentration of 0.5 ug/ml. Mix the gel solution thoroughly                 by gentle swirling.
5. While the agarose solution is cooling, choose an appropriate comb for forming the sample slots in the                 gel. Position the comb 0.5-1.0 mm above the plate so that a complete well is formed when the agarose is added                 to the mold.
6. Pour the warm agarose solution into the mold.
7. Allow the gel to set completely (30-45 minutes at room temperature), then carefully remove the comb. Pour                 off the electrophoresis buffer and carefully remove the tape Mount the gel in the electrophoresis tank.
8. Add just enough electrophoresis buffer to cover the gel to a depth of ~1 mm.
9. Mix the samples of DNA with 10 ul green buffer
10. Slowly load the sample mixture into the slots of the submerged gel using a disposable micropipette, an                 automatic micro-pipettor. Load size standards into slots on both the right and left sides of the gel.
11. Close the lid of the gel tank and attach the electrical leads so that the DNA will migrate toward the                 positive anode (red lead). Apply a voltage of 1-5 V/cm (measured as the distance between the positive and                 negative electrodes). If the leads have been attached correctly, bubbles should be generated at the anode                 and cathode (due to electrolysis), and within a few minutes, the bromophenol blue should migrate from the                 wells into the body of the gel. Run the gel until the bromophenol blue and xylene cyanol FF have migrated                 an appropriate distance through the gel.
12. When the DNA samples or dyes have migrated a sufficient distance through the gel, turn off the electric                 current and remove the leads and lid from the gel tank.
Cell culture and EGFP gene disruption assay
Conversion
The COS7-EGFP cell line, a generous gift from School of Public Health, University of South China, was used as                 the model cell line which has a single copy of destabilized EGFP gene integrated into the genome. The cells                 were cultured in a 37 °C incubator under 5% CO2 and 90% humidity with full serum medium: DMEM supplemented                 with 10% (v/v) FBS. COS7-EGFP cells were seeded into glass bottom cell culture dish (~60,000 cells per well)                 one day before the transfection. When the cells reached 70% confluence, the medium was replaced with fresh                 DMEM medium containing the BODIPY/sgRNA/Cas9 (or Liposome/sgRNA/Cas9) complex (various according to the experiment                 design below). After incubation for 4h, the Cas9 containing medium was replaced with fresh full serum medium                 incubated for another two days for analyzed by laser scanning confocal microscope (LSCM).
Double enzyme digestion
Double enzyme digestion
Vector:
Take 1.5 ng of the verified plasmid, add 1 μL Xba1 , 1μL Nhe1 ,5 μL 10X Fast Digest buffer , add ddHO to                 50 μL, place in a 37 ° C water bath for 15min, then put in a 80 ° C water bath for 20 min.
Add 1μL Antarctic Phosphatase , 5.6μL Antarctic Phosphatase Buffer , then place in a 37 ° C water bath for                 30min, then put in a 70 ° C water bath for 10 min.
Take out and run the gel verification.
Insert:
Take 1 ng of the verified insert fragment, add1 μL Xba1, 1μL Nhe1 ,5 μL 10X Fast Digest buffer , add ddHO                 to 50 μL, place in a 37 ° C water bath for 1h, then put in a 80 ° C water bath for 20 min, take out and run                 the gel verification.
connection:
COX8A: First add vector 100ng, insert 75ng, then add 1μL T4 ligase, 4 μL of its buffer, add ddHO to 20 μL,                 place in 22 ° C in a thermos bucket, and place in 4℃ connect overnight.
SOD2: First add vector 100ng, insert 75ng, then add 1μL T4 ligase , 4 μL of its buffer, add ddHO to 20 μL,                 place in 22 ° C in a thermos bucket, and place in 4℃ connect overnight.
ATP5: First add vector 100ng, insert 81.25ng, then add 1μL T4 ligase , 4 μL of its buffer, add ddHO to 20                 μL, place in 22 ° C in a thermos bucket, and place in 4℃ connect overnight.
Conversion:
1. Wipe the cleaned work area with 70% ethanol.
2. Thaw the competent cells on ice. A 1.5 mL microcentrifuge tube was labeled for each transformation and                 the tube was placed on ice for pre-cooling. .
3. Inhale connection system into each microcentrifuge tube.
4. Pipette 50 μL of competent cells into each tube. Gently mix the tube with your fingers.
5. Incubate on ice for 30 minutes.
6. Heat the cells by placing in a 42℃ water bath for 90 seconds.
7. Immediately transfer the tube back to the ice and incubate on ice for 2 minutes.
8. Add 800 μL of LB medium to each tube, incubate at 37 ° C for 1 hour, and shake at 200-300 rpm.
9. Pipette bacteria from each tube onto the appropriate plate and spread the mixture
evenly over the plate. Incubate overnight or for about 16 hours at 37 ° C. Place the plate with the agar                 side on top and the lid on the bottom.
