<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TJU_China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TJU China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE &gt;<TITLE>home</TITLE></P><DIV class="nav"><UL style="margin-left:80px;" class="main"><LI><A href="https://2018.igem.org/Team:TJU_China">Home</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Description">
                    Project
                </A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Demonstrate">Demonstrate</A></LI></UL><LI><A href="https://2018.igem.org/Team:TJU_China/Parts">
                    Parts
                </A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Parts_Overview">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Basic">Basic</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Composite">Composite</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Collection">Collection</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Model#Dynamic_Model">Dynamic Model</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Model#Off-Target_Model">Off-target Model</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Model#Code">Code</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Human_Practices">
                    HP
                </A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Team">
                    Team
                </A></LI><LI style="list-style-type: none;"><A href="https://2018.igem.org/Team:TJU_China/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Attributions">Attributions</A></LI></DIV><DIV class="first_button_list"><DIV class="first_button"><BUTTON id="btn11">Group1</BUTTON></DIV><DIV class="first_button"><BUTTON id="btn12">Group2</BUTTON></DIV><DIV class="first_button"><BUTTON id="btn13">Group3</BUTTON></DIV></DIV><DIV class="group" id="group1"><DIV class="grouphead">Enhanced sensitivity of metal iron detection based on dCas9 system</DIV><DIV class="contentword">In view of the current serious pollution problems, we focus on the pollution of heavy metal ions. Only by detecting
            heavy metal ions quickly and accurately can we prevent pollution in a timely manner. To this end, we started
            with a arsenic ion, combined with the achievements of the 2006 iGEM team (iGEM2006_Edinburgh) in order to construct
            a circuit dedicated to the detection of arsenic ions, which consists of Promoter J23104, ArsR Protein, Promoter
            ArsR, smURFP. We first ligated these fragments by overlap, then ligated them to the pKM586 plasmid by double
            restriction enzymes, and then transformed them into E.coli BL21. Since we think this loop can not be completed
            to meet our requirements, we want to make this loop more sensitive. Therefore, we noticed that dcas9 in the CRISPR-Cas
            system has an enhanced transcriptional effect, thus amplifying the effect of arsenic ions on the loop. In the
            plasmid of dCas9, we need to cut the two segments of the plasmid with BsaI enzyme, then connect the spacer we
            designed to target dCas9 to the corresponding gene, and then we import it with another plasmid E.coli BL21 to
            complete the enhancement of our arsenic sensing circuit by dCas9.</DIV><DIV class="figure"><B>Figure1.</B>The result of nucleic acid gel electrophoresis of Bba-J33201 after PCR. Lane M, Marker. Lane 1-6，Bba-J33201</DIV><DIV class="figure"><B>Figure2.</B>The result of nucleic acid gel electrophoresis of smURFP after PCR.LaneM, Marker. Lane1-8, smURFP</DIV><DIV class="figure"><B>Figure3.</B>The result of nucleic acid gel electrophoresis after overlapping of J23104 and ArsR Protein. LaneM,
            Marker. Lane 1，ArsR Promoter；Lane 2-5：J23104＋ArsR Protein.</DIV><DIV class="figure"><B>Figure4.</B>The result of nucleic acid gel electrophoresis after overlapping of ArsR Promoter and smURFP. LaneM,
            Marker. Lane 1, smURFP. Lane 2-4,ArsR Promoter＋smURFP</DIV><DIV class="figure"><B>Figure5.</B>Double digestion to verify the ligation product. lane M, Marker. Lane 1, Plasmid pKM586. Lane 2,
            Plasmid pKM586 single digestion with BamHI. Lane 3, Plasmid pKM586 double digestion with AatII and BamHI. Lane
            4, Plasmid ArS. Lane 5, Plasmid ArS single digestion with BamHI. Lane 6, Plasmid ArS double gigestion with AatII
            and BamHI. Lane 7, Plasmid ArS. Lane 8, Plasmid ArS single digestion with BamHI. Lane 9, Plasmid ArS double digestion
            with AatII and BamHI.</DIV><DIV class="figure"><B>Figure6.</B>Double digestion of pKM586 with AatII and BamHI. lane M, Marker. Lane 1,Plasmid pKM586. Lane 2, single digestion with BamHI. Lane 3, Plasmid pKM586 after double enzyme digestion</DIV><DIV class="figure"><B>Figure7.</B>Double digestion of pKM586 with AatII and BamHI. LaneM, Marker. Lane 1,Plasmid pKM586. Lane 2, Plasmid
            pKM586 after double enzyme digestion</DIV><DIV class="figure"><B>Figure8.</B>Double digestion to verify the ligation product. lane M, Marker. Lane 1, Plasmid pKM586. Lane 2,
            Plasmid pKM586 single digestion with BamHI. Lane 3, Plasmid pKM586 double digestion with AatII and BamHI. Lane
            4, Plasmid ArS. Lane 5, Plasmid ArS single digestion with BamHI. Lane 6, Plasmid ArS double gigestion with AatII
            and BamHI. Lane 7, Plasmid ArS. Lane 8, Plasmid ArS single digestion with BamHI. Lane 9, Plasmid ArS double digestion
            with AatII and BamHI.</DIV></DIV><DIV class="group" id="group2" style="display:none;"><DIV class="grouphead">Targeted delivery of sgRNA/Cas9 complex</DIV><DIV class="contentword">In order to reach a new gene detection in a high-throughput technique, CRISPR-Cas12a system is modified to chips
            which have been tiled with a layer of Janus (the hydrophobic protein designed by our team Tianjin in 2015) in
            advance. After our first step on the assembly of FnCas12a and crRNA was taken，we successfully tested sequence-specific
            cleavage activity on plasmid and trans-cleavage activity on ssDNA. With tremendous trails, we optimized cleavage
            protocol of both cis and trans cleavage. Last but not least, for achieving high-throughput detection on chips,
            we dried the Janus on the chip, then incubated Cas12a protein and crRNA complex, and finally, the fluorescence
            probe was cut and detected by fluorescence microscope. With the help of Janus, we were glad that higher fluorescence
            values were detected under the condition of a smaller amount of materials.</DIV><DIV class="figure"><B>Figure1.</B>Result of protein expression and purification of FnCas12a. (A) SDS-PAGE gel of result of affinity
            chromatography (Ni-NTA) result. Lane M, marker. Lane 1, before washing by Buffer A. Lane 2, after washing by
            Buffer A. Lane 3, before elution by Buffer B. Lane 4, after elution by Buffer B. Buffer A (50 mM Tris-HCl (pH8.0),
            1.5 M NaCl, 5% glycerol, 30 mM imidazole). Buffer B (50 mM Tris-HCl (pH8.0), 1.5 M NaCl,1 mM DTT and 5% glycerol,
            600 mM imidazole). (B) SDS-PAGE gel of result of ion exchange. Lane M, marker. Lane 1, purified FnCas12a. (C)
            SDS-PAGE gel of result of gel filtration. Lane M, marker. Lane 1, purified FnCas12a. (D) The result of ion-exchange
            chromatography program. (C) The result of gel filtration program.
        </DIV><DIV class="figure"><B>Figure2.</B>Result of assembly of crRNA. Lane M, marker. Lane 1, crRNA assembled into 43 nucleotides.
        </DIV><DIV class="figure"><B>Figure3.</B>Result of sequence-specific plasmid cleavage. Lane 1, plasmid GFP. Lane 2, cleavage plasmid with
            BamH1 enzyme. Lane 3, cleavage plasmid with Fncas12a,crRNA-1 and crRNA-2. Lane 4, cleavage plasmid with Fncas12a
            and crRNA-1. Lane 5, cleavage plasmid with Fncas12a and crRNA-2.
        </DIV><DIV class="figure"><B>Figure4.</B>Optimization of cleavage protocol. (A) Line 1, GFP plasmid. Line 2, cleavage plasmid with BamH1 enzyme.
            Line3-8, cleavage according to table (B). (B) Experiment design.
        </DIV><DIV class="figure"><B>Figure5.</B>Optimization of cleavage protocol. (A) Line 1, GFP plasmid. Line 2, cleavage plasmid with BamH1 enzyme.
            Line3-7, cleavage according to tableB. (B) Experiment design.
        </DIV><DIV class="figure"><B>Figure6.</B>Result of specific-sequence cleavage after Optimization. Lane 1, GFP plasmid. Lane 2, BamH1 cleavage.
            Lane 3, cleavage according to: crRNA：FnCas12a：plasmid = 10:10:1. Lane 4, cleavage according to: crRNA：FnCas12a：Plasmid
            = 10:10:1.
        </DIV><DIV class="figure"><B>Figure7.</B> The FnCas12a trans-cleavage activity on ssDNA. Lane M, marker. Lane 1, dsDNA about 40 bps. Lane
            2, ssDNA about 40 nts. Lane 3, cleavage according to: dsDNA : ssDNA = 1:20. Lane 4, cleavage according to: dsDNA
            : ssDNA = 1:40. Lane 6, cleavage according to: dsDNA : ssDNA = 1:100. Lane 7, cleavage according to: dsDNA :
            ssDNA = 1:125.
        </DIV><DIV class="figure"><B>Figure8.</B>Optimize trans-cleavage of fluorescent probe.
        </DIV><DIV class="figure"><B>Figure9.</B>Cleavage on chip. (A) Cleavage design of experimental group and control group. (B) Result of cleavage
            according to table A with and without Janus (a kind of hydrophobic protein designed by our team TJUSLS in 2015
            and won best new application prize).
        </DIV></DIV><DIV class="group" id="group3" style="display:none;"><DIV class="grouphead">Improved sensitivity of metal iron detection based on dCas9 system</DIV><DIV class="contentword">In order to realize the targeted delivery of sgRNA/Cas9 complex into cells, we make use of BODIPY, a kind of fluorescent
            dyes, to combine with sgRNA/Cas9 complex (RNP) and to deliver them into cells, since BODIPY can target nucleus
            itself[1] and can be observed as near-infrared (NIR) dye. Firstly, we constructed the template of sgRNA and completed
            the in vitro transcription of sgRNA using T7 promoter. We also expressed and purified Cas9 protein, and then
            we successfully tested the cleavage activity of sgRNA/Cas9 complex on plasmid. Then we combined BODIPY with sgRNA/Cas9
            (BODIPY/RNP) and proved that our new complex had high in vitro cleavage efficiency. So we proceeded to the delivery
            of BODIPY/RNP into cells and found that BODIPY/RNP had better gene editing efficiency than RNP only and Liposome/RNP
            complex. After we succeeded in targeting nucleus, we tried to target not only nucleus but also mitochondrion.
            The plasmid we used was the pET-NLS-Cas9-6xHis plasmid, digested with Xba1 and Nhe1. In order to target mitochondria,
            we need to replace the NLS leader sequence on the plasmid with MTS (mitochondrial targeting sequence). We selected
            three MTS sequences, COX8a, SOD2 and ATP5. Since the MTS fragment is small and there is no suitable restriction
            site at both ends, we synthesized the MTS and the previous fragment. By annealing the two primers and then connecting
            the preceding fragment to the segment of Cas9 by overlap, we get the entire fragment. We then digest it withXba1
            and Nhe1 and connect it with the vector.</DIV><DIV class="figure"><B>Figure 1.</B> Purification of Cas9 protein. (A) Result of purification by Affinity chromatography (Ni-NTA) .
            Lane M, marker. Lane 1, before eluted by Buffer A. Lane 2, after eluted by Buffer A. Lane 3, before eluted by
            Buffer B. Lane 4, after eluted by Buffer B. Buffer A(50 mM Tris-HCl (pH 8.0), 1 M NaCl, 20% glycerol, 2 mM TCEP
            and 20 mM imidazole). Buffer B( 50 mM Tris-HCl (pH 8), 1 M NaCl, 20% glycerol, 2 mM TCEP and 500 mM imidazole).
            (B) SDS-PAGE result of ion exchange. Lane M, marker. Lane 1, Cas9 protein after ion exchange purification. (C)
            SDS-PAGE result of gel filtration. Lane M, marker. Lane 1, Cas9 protein after gel filtration purification. (D)
            Result of ion exchange program. (E) Result of gel filtration program.
        </DIV><DIV class="figure"><B>Figure 2.</B>Assembly of template of sgRNA using PCR. Lane M, marker. Lane 1, template of sgRNA.
        </DIV><DIV class="figure"><B>Figure 3.</B>In vitro transcription of sgRNA. Lane M, marker. Lane 1, 100ng sgRNA. Lane 2, 200ng sgRNA.
        </DIV><DIV class="figure"><B>Figure 4.</B>In vitro digestion of DNA with sgRNA/Cas9. Lane 1, eGFP plasmid. Lane 2, sgRNA:Cas9:DNA=10:20:1.
            Lane M, marker.
        </DIV><DIV class="figure"><B>Figure 5.</B>Characterization of sgRNA/Cas9 complex(RNP), BODIPY/RNP, Liposome/RNP, and BODIPY/Liposome/RNP.
            (A) Z-Ave of sgRNA, Cas9, RNP, Liposome/RNP, BODIPY/RNP, and BODIPY/Liposome/RNP. (B) Zeta potential of sgRNA,Cas9,RNP,BODIPY,BODIPY/RNP,
            and BODIPY/Liposome/RNP.
        </DIV><DIV class="figure"><B>Figure 6.</B>In vitro digestion of DNA with BODIPY/RNP. (A) Lane 1, eGFP plasmid. Lane 2-7 are set according
            to table B. Lane M, marker. (B) Experiment design.
        </DIV><DIV class="figure"><B>Figure 7.</B>EGFP gene disruption of COS7-GFP cell line.
        </DIV><DIV class="figure"><B>Figure 8.</B>Extraction of pET-NLS-Cas9-6xHis plasmid. Lane M, marker. Lane 1, plasmid.</DIV><DIV class="figure"><B>Figure 9.</B>The result of pET-NLS-Cas9-6xHis plasmid digestion. Lane M, marker. Lane1-2, double digestion with
            Xba1+Bmgb1.Lane 3, single digestion with Xba1. Lane 4, single digestion with Bmgb1. Lane 5, plasmid.</DIV><DIV class="figure"><B>Figure 10.</B>The result of pET-NLS-Cas9-6xHis plasmid digestion. (A) cleavage 15mins. Lane M, marker. Lane 1,
            1μg plasmid double digestion with Xba1 and Nhe1. Lane 2, 2μg plasmid double digestion with Xba1 and Nhe1. Lane
            3, 1μg plasmid single digestion with Xba1. Lane 4, 1μg plasmid single digestion with Nhe1. Lane 5, plasmid. (B)
            cleavage 60mins. Lane M, marker. Lane 1, 1μg plasmid double digestion with Xba1 and Nhe1. Lane 2, 2μg plasmid
            double digestion with Xba1 and Nhe1. Lane 3, 1μg plasmid single digestion with Xba1. Lane 4, 1μg plasmid single
            digestion with Nhe1. Lane 5, plasmid.</DIV><DIV class="figure"><B>Figure 11.</B>Amplification of the overlapped COX8a and Cas9. Lane M, marker. Lane 1, COX8a+Cas9 fragment after
            amplification. The concentration of gel extraction product is 10.8ng/μl，the volume is 200μl</DIV><DIV class="figure"><B>Figure 12.</B>Double digestion of COX8a+Cas9 fragment. Lane M, marker. Lane 1, COX8a+Cas9 fragment double digestion
            with Xba1 and Nhe1.</DIV><DIV class="figure"><B>Figure 13.</B>The result of fragments synthetic. Lane M, marker. Lane 1, COX8a fragment. Lane 2, Cas9 fragment.
            Lane 3, COX8a+Cas9 fragment.</DIV><DIV class="figure"><B>Figure 14.</B>The result of pET-NLS-Cas9-6xHis plasmid digestion. Lane M, marker. Lane 1-8, plasmid double digestion
            with Xba1 and Nhe1. Lane 9, plasmid single digestion with Xba1. Lane 10, plasmid single digestion with Xba1.
            Lane 11, plasmid.</DIV><DIV class="figure"><B>Figure 15.</B>The result of bacterial colony PCR confirmation. Lane M, marker. Lane1-4, SOD2. Lane 5-13, COX8A.
            Lane 14-17, ATP5. </DIV><DIV class="figure"><B>Figure 16.</B>The result of construction. (A) The construction of pET-NLS-Cas9-6xHis plasmid with SOD2 MTS. Lane
            1, MTS fragment. Lane 2, segment of Cas9. Lane 3, overlap of MTS and Cas9 fragments. Lane 4, re-constructed plasmid.
            (B) The construction of pET-NLS-Cas9-6xHis plasmid with ATP5 MTS. Lane 1, MTS fragment. Lane 2, segment of Cas9.
            Lane 3, overlap of MTS and Cas9 fragments. Lane 4, re-constructed plasmid.</DIV><DIV style="font-size: 30px;margin-top: 50px;
text-align: left;
margin-left: 10%;
width: 80%;
font-weight: bold;">Reference</DIV><DIV class="figure">[1]Wang, K., Xiao, Y., Wang, Y., Feng, Y., Chen, C., Zhang, J., Zhang, Q., Meng, S., Wang, Z., Yang, H. (2016). Self-assembled hydrophobin for producing water-soluble and membrane permeable fluorescent dye. Scientific Reports, 6(1). doi:10.1038/srep23061</DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>