Team:TJU China/Notebook/Group2
<!DOCTYPE >
Home
Project
Description
Experiments
Notebook
Demonstrate
Parts
Parts Overview
Basic
Composite
Collection
Improve
Model
HP
Human Practices
Public Engagement
Team
Members
Collaboration
Attributions
Mar.
Apr.
May
Jun.
Jul.
Aug.
Sep.
Oct.
March
week1
After reading a large amount of documents about Cas12 and Cas13 and searching information on the Internet or in books； we learn about the concept of loop ring, activator, Förster resonance energy transfer,                         fluorescence resonance energy transfer(FRET) and Michaelis-Menten equation, which will be our main                         theory in the future.
week2
As soon as we decide what to do, we acquire materials and all parts that we need, including the target                         and recombinant vectors; we also send e-mails to Feng Zhang for plasmids and approaches of cultivating                         protein complex, fortunately, we get them.
week3
We have done a lot of work on the comparison of cas13a protein point mutation and gene sequence. What’s                         more, we blast many sequences of crRNA , design and choose proper spacers that we might use with                         the help of Feng Zhang’s website(http://crispr.mit.edu/).
week4
We learn principles of pcr. At the same time, we designe primers that we need and come up with the thought                         of High throughput chip finally.
April
week1
We search the genotyping data(https://cancer .sanger.ac.uk/cosmic, https://www.uniprot.org/) for a variety                         of cancer , which we may use in the future. At last, we chose kras, egfp and gata for future use.                         We also search documents about how Zhang Feng deal with cfDNA.
week2
We design crRNA of the gene we had chosen last week and selected targets of fncas12a and lbcas12a. We                         understand how FQ fluorescent molecules can be used in our program and intend to do some detection                         with it. We also want to combine our proteins(fncas12a and lbcas12a) with hydrophobic proteins
week3&4
We designed a protocol of protein purification of fncas12a and lbcas12a.
May
week1&2
We send out the crRNA sequences that we designed two weeks ago and prepare buffer for storing proteins(100%                         glycerol: ddH2O=1:1). Afterwards, we manage a preservation program (avoid repeated freeze-thaw)
week3
We practice on protein purification, plasmid extraction of EGFR and RED for many times. Finally, we successfully                         purified our proteins(fncas12a and lbcas12a).
Figure 1.Protein Expression and Purification of LbCas12a and FnCas12a.Lane M, marker; lane 1,                         before washing; lane 2, after washing; lane 3, before elution; lane 4, after elution.
We also successfully extracted the plasmid with EGFP and RED.
Figure 2.The result of nucleic acid gel electrophoresis of plasmid extraction with EGFP and RED.                         Lane M, marker; lane 1，plasmid EGFP 250ng；lane 2 ，plasmid RED 250ng.
week4
We learn how to use the python software for mops rate. After that, we preserve our proteins. The plasmid                         extraction is also done. We prepare materials and protocols for later cleavage.
June
We finished the preparation of competent cell and transform our plasmid into that competent cell. One group                     member is arranged to collect the parts of biobricks that we have for now. We design our primers for                     the pcr of ssDNA that we chose before. Unfortunately, we failed our first pcr for T7 promoter.
Figure 3.The result of nucleic acid gel electrophoresis of the PCR production for T7 promoter.
July
week4
We continue to optimize the PCR process of the promoter, such as the Mg2+ optimization
Figure 4. The result of nucleic acid gel electrophoresis of the PCR production that different                         concentration gradient of Mg2+. Lane M, marker; lane 1, concentration of Mg2+ 0mM; lane 2,concentration                         of Mg2+ 1mM; lane 3,concentration of Mg2+ 1.5mM; lane 4, concentration of Mg2+ 2mM; lane 5, concentration                         of Mg2+ 2.5mM;lane 6, concentration of Mg2+ 3mM;lane 7, concentration of Mg2+ 3.5mM;lane 8, concentration                         of Mg2+ 4mM
Figure 5.The result of nucleic acid gel electrophoresis of plasmid extraction with EGFP and RED.                         Lane M, marker; lane 1 plasmid EGFP 250ng；lane2 plasmid RED 250ng.
Meanwhile, we continued to extract the plasmid and conducted the first cleaving experiment，the gelling                         was not very good, but it was cut successfully.
Figure 6.The result of nucleic acid gel electrophoresis of the cleaving experiment that different                         concentration gradient of Fncas12a protein.
August
week1
After the successful cleavage, we continued to explore other cleavage experiments, and conducted a cleavage                         experiment with two different crRNA, enzyme cleaving as control experiment and plasmid concentration                         gradient experiment successively. At the same time, we were constantly exploring how to make the                         gels run better. The gels run in this process are not so good
Cleavage with two different crRNA
Figure 7. The result of Sequence-specific plasmid cleavage.
Enzyme digestion control
Figure 8.The result of nucleic acid gel electrophoresis of the enzyme digestion. Lane M, marker;                         lane 1, plasmid EGFP 250ng ; lane 2, cleavage with no crRNA; lane 3, cleavage with crRNA：FnCas12a：plasmid                         = 10:10:1; lane 4, BamH1 cleavage
Plasmid gradients
Figure 9.The result of nucleic acid gel electrophoresis of the gradient of plasma.
week2
On fragment cutting, we used annealing short chain (DNMT) used in wang jin’s literature to perform fragment                         cutting. Since the fragment was very short, we tried to use sds-page to run the electrophoretic gel                         graph of double-stranded DNA. Similarly, the gel graph at the beginning was very ugly
Figure 10.Short sequence measured by SDS-PAGE（12%）
At the same time, we performed the cutting results with SDS-PAGE electrophoretic gel, and added single                         strand DNA to detect its non-specific cutting characteristics
Figure 11.The FnCas12a trans-cleavage activity on ssDNA
week3
Figure 12.The result of nucleic acid gel electrophoresis of plasmid extraction with EGFP .Lane                         M, marker; lane 1-lane 10 plasmid EGFP 250ng
The electrophoresis gel map demonstrated the non-specific single strand cutting characteristics of                         the FnCas12a protein.
Figure 13.The optimization of FnCas12a trans-cleavage activity on ssDNA
Later, we added the fluorescence probe connected by single strand DNA, which indirectly proved its                         single strand cutting characteristics and various influencing factors through the intensity of fluorescence
Figure 14.Optimize trans-cleavage of fluorescent probe.
Figure 15.Optimize trans-cleavage of fluorescent probe.
week4
The concept of fluorescent cutting has been reproduced. Subsequently, we conducted GATA mutant search                         and protein purification, and optimized the GFP plasmid cutting results to obtain a perfect cutting                         gel map
Figure 16. Optimize cleavage protocol.
Figure 17.Optimize cleavage protocol.
Figure 18.The result of Sequence-specific plasmid cleavage.Lane M, marker; lane 1, plasmid; lane                         2, BamH1 cleavage; lane 3, cleavage with crRNA：FnCas12a：plasmid = 10:10:1; lane 4, cleavage with                         crRNA：FnCas12a：Plasmid = 10:10:1.
In the latest literature, wang jin used 150bp fragments for cfDNA detection, not only that, but also                         realized the detection of arbitrary single-base mutation. We carefully studied wang jin's design                         method and constructed the upstream and downstream primers of GATA fragments with the same design                         principle. Then the designed primer was sent to the company for synthesis
September
week1&2
During the two weeks, we performed GATA's receptor cell transformation and multiple plasmid extraction
Figure 19.The result of nucleic acid gel electrophoresis of plasmid extraction with GATA .Lane                         M, marker; lane 1-lane 5， plasmid GATA
Figure 20.The result of nucleic acid gel electrophoresis of plasmid extraction with GATA .Lane                         M, marker; lane 1-lane 9， plasmid GATA
At the same time, we performed the first PCR of GATA wild type and mutant type fragments
Figure 21.The result of nucleic acid gel electrophoresis of GATA fragment PCR（150bp）Lane M, marker;                         lane1-15,Tm:60.8℃
Figure 22,23.The result of nucleic acid gel electrophoresis of GATA point mutant sequence PCR（150bp）.                         Lane M, marker; lane1-16，Tm：59℃
week3
Of course, our group also had the idea of fixing the Cas12a protein by hydrophobic protein, and at the                         same time to see if it would affect the cutting activity of the Cas12a protein. This week, we treated                         the hydrophobic protein with the substrate, and then let the cutting reaction occur on the substrate,                         and observed the results with a fluorescence microscope. The results showed that the fluorescence                         intensity of the hydrophobic group was much higher than that of the non-hydrophobic group.
Figure 24.Fluorescence intensity of the hydrophobic group.
Figure 25.Fluorescence intensity of the non-hydrophobic group.
week4
This week, we continued to optimize the chip cutting system and tried to reduce the system to 1ul for                         cutting reaction. The result was not very satisfactory, because the 1ul system was too small and                         the human error was too large, so we extended the system to 20ul
Figure 26.Fluorescence intensity of the hydrophobic group.
Figure 27.Fluorescence intensity of the non-hydrophobic group.
At the same time, we wanted to obtain the plasmid fragment through PCR, to verify the feasibility                         of fragment cutting in the literature, so we conducted fragment PCR for GFP
Figure 28.The result of nucleic acid gel electrophoresis of GFP fragment PCRLane M, marker; lane1-16，Tm:62℃
October
week1
Figure 29.The result of nucleic acid gel electrophoresis of plasmid extraction with EGFP. Lane                         M, marker; lane1-12,Tm:60.8℃
After the GFP and RED fragment products were obtained by PCR, we performed fragment cutting, and                         the results showed that fragment cutting could be achieved
Figure 30.The result of Sequence-specific plasmid fragment cleavage.Lane M, marker; lane 1, fragment                         of GFP; lane 2,fragment of RED; lane 3, cleavage with crRNA：FnCas12a：fragment of GFP = 10:10:1; lane                         4, cleavage with crRNA：FnCas12a：fragment of RED = 10:10:1.
At the same time, the experimental results of fluorescent chips have been unstable. We are trying                         to get stable experimental results
Figure 31.Fluorescence intensity of the hydrophobic group.
Figure 32..Fluorescence intensity of the non- hydrophobic group.
Figure 33.Fluorescence intensity of the hydrophobic group.
Figure 34.Fluorescence intensity of the non-hydrophobic group.
week2
This week, we made multiple fluorescent observations of chip cutting, and the results were gradually                         stabilizing.
Figure 35.Fluorescence intensity of the hydrophobic group. （10.9）
Figure 36.Fluorescence intensity of the non-hydrophobic group. （10.9）
Figure 37.Fluorescence intensity of the hydrophobic group. （10.12）
Figure 38.Fluorescence intensity of the non-hydrophobic group. （10.12）                         At the same time, in order to get more rigorous cutting results, we simultaneously verified the results by running glue
Figure 39. The result of Sequence-specific plasmid cleavage.                         Lane M, marker; lane 1, plasmid GFP; lane 2, cleavage with crRNA：FnCas12a：plasmid GFP= 10:10:1; lane 3, plasmid RED ;lane 4, cleavage with crRNA：FnCas12a：Plasmid RED= 10:10:1.
