<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TJU_China_Notebook_Group2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TJU China/Notebook/Group2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE &gt;</P><DIV class="nav"><UL style="margin-left:80px;" class="main"><LI><A href="https://2018.igem.org/Team:TJU_China">Home</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Description">
                    Project
                </A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Demonstrate">Demonstrate</A></LI></UL><LI><A href="https://2018.igem.org/Team:TJU_China/Parts">
                    Parts
                </A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Parts_Overview">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Basic">Basic</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Composite">Composite</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Collection">Collection</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Human_Practices">
                    HP
                </A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Team">
                    Team
                </A></LI><LI style="list-style-type: none;"><A href="https://2018.igem.org/Team:TJU_China/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Collaborations">Collaboration</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Attributions">Attributions</A></LI></DIV><DIV class="notebook_month"><DIV class="notebook_month_p">Mar.</DIV><DIV class="notebook_month_p">Apr.</DIV><DIV class="notebook_month_p">May</DIV><DIV class="notebook_month_p">Jun.</DIV><DIV class="notebook_month_p">Jul.</DIV><DIV class="notebook_month_p">Aug.</DIV><DIV class="notebook_month_p">Sep.</DIV><DIV class="notebook_month_p">Oct.</DIV></DIV><DIV class="notebook_information"><DIV id="notebook_March"><DIV class="notebook_Month_head">March</DIV><DIV class="notebook_week"><DIV><BUTTON id="3week1" class="notebook_information_button1">week1</BUTTON></DIV><DIV id="3notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p">After reading a large amount of documents about Cas12 and Cas13 and searching information on the Internet
                        or in books； we learn about the concept of loop ring, activator, Förster resonance energy transfer,
                        fluorescence resonance energy transfer(FRET) and Michaelis-Menten equation, which will be our main
                        theory in the future.</DIV></DIV><DIV><BUTTON id="3week2" class="notebook_information_button2">week2</BUTTON></DIV><DIV id="3notebook_week2" class="notebook_information_word2" style="display:none;"><DIV class="notebook_week_p">As soon as we decide what to do, we acquire materials and all parts that we need, including the target
                        and recombinant vectors; we also send e-mails to Feng Zhang for plasmids and approaches of cultivating
                        protein complex, fortunately, we get them.</DIV></DIV><DIV><BUTTON id="3week3" class="notebook_information_button3">week3</BUTTON></DIV><DIV id="3notebook_week3" class="notebook_information_word3" style="display:none;"><DIV class="notebook_week_p">We have done a lot of work on the comparison of cas13a protein point mutation and gene sequence. What’s
                        more, we blast many sequences of crRNA , design and choose proper spacers that we might use with
                        the help of Feng Zhang’s website(http://crispr.mit.edu/).</DIV></DIV><DIV><BUTTON id="3week4" class="notebook_information_button4">week4</BUTTON></DIV><DIV id="3notebook_week4" class="notebook_information_word4" style="display:none;"><DIV class="notebook_week_p">We learn principles of pcr. At the same time, we designe primers that we need and come up with the thought
                        of High throughput chip finally.</DIV></DIV></DIV></DIV><DIV id="notebook_April"><DIV class="notebook_Month_head">April</DIV><DIV class="notebook_week"><DIV><BUTTON id="4week1" class="notebook_information_button1">week1</BUTTON></DIV><DIV id="4notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p">We search the genotyping data(https://cancer .sanger.ac.uk/cosmic, https://www.uniprot.org/) for a variety
                        of cancer , which we may use in the future. At last, we chose kras, egfp and gata for future use.
                        We also search documents about how Zhang Feng deal with cfDNA. </DIV></DIV><DIV><BUTTON id="4week2" class="notebook_information_button2">week2</BUTTON></DIV><DIV id="4notebook_week2" class="notebook_information_word2" style="display:none;"><DIV class="notebook_week_p">We design crRNA of the gene we had chosen last week and selected targets of fncas12a and lbcas12a. We
                        understand how FQ fluorescent molecules can be used in our program and intend to do some detection
                        with it. We also want to combine our proteins(fncas12a and lbcas12a) with hydrophobic proteins</DIV></DIV><DIV><BUTTON id="4week3" class="notebook_information_button3">week3&amp;4</BUTTON></DIV><DIV id="4notebook_week3" class="notebook_information_word3" style="display:none;"><DIV class="notebook_week_p">We designed a protocol of protein purification of fncas12a and lbcas12a.</DIV></DIV></DIV></DIV><DIV id="notebook_May"><DIV class="notebook_Month_head">May</DIV><DIV class="notebook_week"><DIV><BUTTON id="5week1" class="notebook_information_button1">week1&amp;2</BUTTON></DIV><DIV id="5notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p">We send out the crRNA sequences that we designed two weeks ago and prepare buffer for storing proteins(100%
                        glycerol: ddH2O=1:1). Afterwards, we manage a preservation program (avoid repeated freeze-thaw)</DIV></DIV><DIV><BUTTON id="5week3" class="notebook_information_button3">week3</BUTTON></DIV><DIV id="5notebook_week3" class="notebook_information_word3" style="display:none;"><DIV class="notebook_week_p">We practice on protein purification, plasmid extraction of EGFR and RED for many times. Finally, we successfully
                        purified our proteins(fncas12a and lbcas12a).</DIV><DIV class="notebook_week_p"><B>Figure 1.</B>Protein Expression and Purification of LbCas12a and FnCas12a.Lane M, marker; lane 1,
                        before washing; lane 2, after washing; lane 3, before elution; lane 4, after elution.

                        We also successfully extracted the plasmid with EGFP and RED.</DIV><DIV class="notebook_week_p"><B>Figure 2.</B>The result of nucleic acid gel electrophoresis of plasmid extraction with EGFP and RED.
                        Lane M, marker; lane 1，plasmid EGFP 250ng；lane 2 ，plasmid RED 250ng.</DIV></DIV><DIV><BUTTON id="5week4" class="notebook_information_button4">week4</BUTTON></DIV><DIV id="5notebook_week4" class="notebook_information_word4" style="display:none;"><DIV class="notebook_week_p">We learn how to use the python software for mops rate. After that, we preserve our proteins. The plasmid
                        extraction is also done. We prepare materials and protocols for later cleavage.</DIV></DIV></DIV></DIV><DIV id="notebook_June"><DIV class="notebook_Month_head">June</DIV><DIV class="notebook_week"><DIV class="notebook_week_p">We finished the preparation of competent cell and transform our plasmid into that competent cell. One group
                    member is arranged to collect the parts of biobricks that we have for now. We design our primers for
                    the pcr of ssDNA that we chose before. Unfortunately, we failed our first pcr for T7 promoter.</DIV><DIV class="notebook_week_p"><B>Figure 3.</B>The result of nucleic acid gel electrophoresis of the PCR production for T7 promoter.</DIV></DIV></DIV><DIV id="notebook_July"><DIV class="notebook_Month_head">July</DIV><DIV class="notebook_week"><DIV><BUTTON id="7week4" class="notebook_information_button4">week4</BUTTON></DIV><DIV id="7notebook_week4" class="notebook_information_word4" style="display:none;"><DIV class="notebook_week_p">We continue to optimize the PCR process of the promoter, such as the Mg2+ optimization</DIV><DIV class="notebook_week_p"><B>Figure 4.</B> The result of nucleic acid gel electrophoresis of the PCR production that different
                        concentration gradient of Mg2+. Lane M, marker; lane 1, concentration of Mg2+ 0mM; lane 2,concentration
                        of Mg2+ 1mM; lane 3,concentration of Mg2+ 1.5mM; lane 4, concentration of Mg2+ 2mM; lane 5, concentration
                        of Mg2+ 2.5mM;lane 6, concentration of Mg2+ 3mM;lane 7, concentration of Mg2+ 3.5mM;lane 8, concentration
                        of Mg2+ 4mM</DIV><DIV class="notebook_week_p"><B>Figure 5.</B>The result of nucleic acid gel electrophoresis of plasmid extraction with EGFP and RED.
                        Lane M, marker; lane 1 plasmid EGFP 250ng；lane2 plasmid RED 250ng.</DIV><DIV class="notebook_week_p">Meanwhile, we continued to extract the plasmid and conducted the first cleaving experiment，the gelling
                        was not very good, but it was cut successfully.</DIV><DIV class="notebook_week_p"><B>Figure 6.</B>The result of nucleic acid gel electrophoresis of the cleaving experiment that different
                        concentration gradient of Fncas12a protein.</DIV></DIV></DIV></DIV><DIV id="notebook_August"><DIV class="notebook_Month_head">August</DIV><DIV class="notebook_week"><DIV><BUTTON id="8week1" class="notebook_information_button1">week1</BUTTON></DIV><DIV id="8notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p">After the successful cleavage, we continued to explore other cleavage experiments, and conducted a cleavage
                        experiment with two different crRNA, enzyme cleaving as control experiment and plasmid concentration
                        gradient experiment successively. At the same time, we were constantly exploring how to make the
                        gels run better. The gels run in this process are not so good
                        Cleavage with two different crRNA</DIV><DIV class="notebook_week_p"><B>Figure 7.</B> The result of Sequence-specific plasmid cleavage.
                        Enzyme digestion control</DIV><DIV class="notebook_week_p"><B>Figure 8.</B>The result of nucleic acid gel electrophoresis of the enzyme digestion. Lane M, marker;
                        lane 1, plasmid EGFP 250ng ; lane 2, cleavage with no crRNA; lane 3, cleavage with crRNA：FnCas12a：plasmid
                        = 10:10:1; lane 4, BamH1 cleavage
                    </DIV><DIV class="notebook_week_p">Plasmid gradients</DIV><DIV class="notebook_week_p"><B>Figure 9.</B>The result of nucleic acid gel electrophoresis of the gradient of plasma. </DIV></DIV><DIV><BUTTON id="8week2" class="notebook_information_button2">week2</BUTTON></DIV><DIV id="8notebook_week2" class="notebook_information_word2" style="display:none;"><DIV class="notebook_week_p">On fragment cutting, we used annealing short chain (DNMT) used in wang jin’s literature to perform fragment
                        cutting. Since the fragment was very short, we tried to use sds-page to run the electrophoretic gel
                        graph of double-stranded DNA. Similarly, the gel graph at the beginning was very ugly</DIV><DIV class="notebook_week_p"><B>Figure 10.</B>Short sequence measured by SDS-PAGE（12%）</DIV><DIV class="notebook_week_p">At the same time, we performed the cutting results with SDS-PAGE electrophoretic gel, and added single
                        strand DNA to detect its non-specific cutting characteristics</DIV><DIV class="notebook_week_p"><B>Figure 11.</B>The FnCas12a trans-cleavage activity on ssDNA</DIV></DIV><DIV><BUTTON id="8week3" class="notebook_information_button3">week3</BUTTON></DIV><DIV id="8notebook_week3" class="notebook_information_word3" style="display:none;"><DIV class="notebook_week_p"><B>Figure 12.</B>The result of nucleic acid gel electrophoresis of plasmid extraction with EGFP .Lane
                        M, marker; lane 1-lane 10 plasmid EGFP 250ng
                        The electrophoresis gel map demonstrated the non-specific single strand cutting characteristics of
                        the FnCas12a protein. </DIV><DIV class="notebook_week_p"><B>Figure 13.</B>The optimization of FnCas12a trans-cleavage activity on ssDNA
                        Later, we added the fluorescence probe connected by single strand DNA, which indirectly proved its
                        single strand cutting characteristics and various influencing factors through the intensity of fluorescence</DIV><DIV class="notebook_week_p"><B>Figure 14.</B>Optimize trans-cleavage of fluorescent probe.</DIV><DIV class="notebook_week_p"><B>Figure 15.</B>Optimize trans-cleavage of fluorescent probe.</DIV></DIV><DIV><BUTTON id="8week4" class="notebook_information_button4">week4</BUTTON></DIV><DIV id="8notebook_week4" class="notebook_information_word4" style="display:none;"><DIV class="notebook_week_p">The concept of fluorescent cutting has been reproduced. Subsequently, we conducted GATA mutant search
                        and protein purification, and optimized the GFP plasmid cutting results to obtain a perfect cutting
                        gel map</DIV><DIV class="notebook_week_p"><B>Figure 16.</B> Optimize cleavage protocol.</DIV><DIV class="notebook_week_p"><B>Figure 17.</B>Optimize cleavage protocol.
                    </DIV><DIV class="notebook_week_p"><B>Figure 18.</B>The result of Sequence-specific plasmid cleavage.Lane M, marker; lane 1, plasmid; lane
                        2, BamH1 cleavage; lane 3, cleavage with crRNA：FnCas12a：plasmid = 10:10:1; lane 4, cleavage with
                        crRNA：FnCas12a：Plasmid = 10:10:1.
                    </DIV><DIV class="notebook_week_p">In the latest literature, wang jin used 150bp fragments for cfDNA detection, not only that, but also
                        realized the detection of arbitrary single-base mutation. We carefully studied wang jin's design
                        method and constructed the upstream and downstream primers of GATA fragments with the same design
                        principle. Then the designed primer was sent to the company for synthesis</DIV></DIV></DIV></DIV><DIV id="notebook_September"><DIV class="notebook_Month_head">September</DIV><DIV class="notebook_week"><DIV><BUTTON id="9week1" class="notebook_information_button1">week1&amp;2</BUTTON></DIV><DIV id="9notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p">During the two weeks, we performed GATA's receptor cell transformation and multiple plasmid extraction</DIV><DIV class="notebook_week_p"><B>Figure 19.</B>The result of nucleic acid gel electrophoresis of plasmid extraction with GATA .Lane
                        M, marker; lane 1-lane 5， plasmid GATA </DIV><DIV class="notebook_week_p"><B>Figure 20.</B>The result of nucleic acid gel electrophoresis of plasmid extraction with GATA .Lane
                        M, marker; lane 1-lane 9， plasmid GATA
                        At the same time, we performed the first PCR of GATA wild type and mutant type fragments</DIV><DIV class="notebook_week_p"><B>Figure 21.</B>The result of nucleic acid gel electrophoresis of GATA fragment PCR（150bp）Lane M, marker;
                        lane1-15,Tm:60.8℃
                    </DIV><DIV class="notebook_week_p"><B>Figure 22,23.</B>The result of nucleic acid gel electrophoresis of GATA point mutant sequence PCR（150bp）.
                        Lane M, marker; lane1-16，Tm：59℃</DIV></DIV><DIV><BUTTON id="9week3" class="notebook_information_button3">week3</BUTTON></DIV><DIV id="9notebook_week3" class="notebook_information_word3" style="display:none;"><DIV class="notebook_week_p">Of course, our group also had the idea of fixing the Cas12a protein by hydrophobic protein, and at the
                        same time to see if it would affect the cutting activity of the Cas12a protein. This week, we treated
                        the hydrophobic protein with the substrate, and then let the cutting reaction occur on the substrate,
                        and observed the results with a fluorescence microscope. The results showed that the fluorescence
                        intensity of the hydrophobic group was much higher than that of the non-hydrophobic group.</DIV><DIV class="notebook_week_p"><B>Figure 24.</B>Fluorescence intensity of the hydrophobic group.
                    </DIV><DIV class="notebook_week_p"><B>Figure 25.</B>Fluorescence intensity of the non-hydrophobic group.
                    </DIV></DIV><DIV><BUTTON id="9week4" class="notebook_information_button4">week4</BUTTON></DIV><DIV id="9notebook_week4" class="notebook_information_word4" style="display:none;"><DIV class="notebook_week_p">This week, we continued to optimize the chip cutting system and tried to reduce the system to 1ul for
                        cutting reaction. The result was not very satisfactory, because the 1ul system was too small and
                        the human error was too large, so we extended the system to 20ul</DIV><DIV class="notebook_week_p"><B>Figure 26.</B>Fluorescence intensity of the hydrophobic group. </DIV><DIV class="notebook_week_p"><B>Figure 27.</B>Fluorescence intensity of the non-hydrophobic group.
                        At the same time, we wanted to obtain the plasmid fragment through PCR, to verify the feasibility
                        of fragment cutting in the literature, so we conducted fragment PCR for GFP</DIV><DIV class="notebook_week_p"><B>Figure 28.</B>The result of nucleic acid gel electrophoresis of GFP fragment PCRLane M, marker; lane1-16，Tm:62℃</DIV></DIV></DIV></DIV><DIV id="notebook_October"><DIV class="notebook_Month_head">October</DIV><DIV class="notebook_week"><DIV><BUTTON id="10week1" class="notebook_information_button1">week1</BUTTON></DIV><DIV id="10notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p"><B>Figure 29.</B>The result of nucleic acid gel electrophoresis of plasmid extraction with EGFP. Lane
                        M, marker; lane1-12,Tm:60.8℃
                        After the GFP and RED fragment products were obtained by PCR, we performed fragment cutting, and
                        the results showed that fragment cutting could be achieved</DIV><DIV class="notebook_week_p"><B>Figure 30.</B>The result of Sequence-specific plasmid fragment cleavage.Lane M, marker; lane 1, fragment
                        of GFP; lane 2,fragment of RED; lane 3, cleavage with crRNA：FnCas12a：fragment of GFP = 10:10:1; lane
                        4, cleavage with crRNA：FnCas12a：fragment of RED = 10:10:1.

                        At the same time, the experimental results of fluorescent chips have been unstable. We are trying
                        to get stable experimental results</DIV><DIV class="notebook_week_p"><B>Figure 31.</B>Fluorescence intensity of the hydrophobic group. </DIV><DIV class="notebook_week_p"><B>Figure 32.</B>.Fluorescence intensity of the non- hydrophobic group. </DIV><DIV class="notebook_week_p"><B>Figure 33.</B>Fluorescence intensity of the hydrophobic group. </DIV><DIV class="notebook_week_p"><B>Figure 34.</B>Fluorescence intensity of the non-hydrophobic group. </DIV></DIV><DIV><BUTTON id="10week2" class="notebook_information_button2">week2</BUTTON></DIV><DIV id="10notebook_week2" class="notebook_information_word2" style="display:none;"><DIV class="notebook_week_p">This week, we made multiple fluorescent observations of chip cutting, and the results were gradually
                        stabilizing.</DIV><DIV class="notebook_week_p"><B>Figure 35.</B>Fluorescence intensity of the hydrophobic group. （10.9）</DIV><DIV class="notebook_week_p"><B>Figure 36.</B>Fluorescence intensity of the non-hydrophobic group. （10.9） </DIV><DIV class="notebook_week_p"><B>Figure 37.</B>Fluorescence intensity of the hydrophobic group. （10.12）</DIV><DIV class="notebook_week_p"><B>Figure 38.</B>Fluorescence intensity of the non-hydrophobic group. （10.12）
                        At the same time, in order to get more rigorous cutting results, we simultaneously verified the results by running glue</DIV><DIV class="notebook_week_p"><B>Figure 39.</B> The result of Sequence-specific plasmid cleavage.
                        Lane M, marker; lane 1, plasmid GFP; lane 2, cleavage with crRNA：FnCas12a：plasmid GFP= 10:10:1; lane 3, plasmid RED ;lane 4, cleavage with crRNA：FnCas12a：Plasmid RED= 10:10:1.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>