<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TJU_China_Notebook_Group3 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TJU China/Notebook/Group3</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE &gt;</P><DIV class="nav"><UL style="margin-left:80px;" class="main"><LI><A href="https://2018.igem.org/Team:TJU_China">Home</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Description">
                    Project
                </A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Demonstrate">Demonstrate</A></LI></UL><LI><A href="https://2018.igem.org/Team:TJU_China/Parts">
                    Parts
                </A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Parts_Overview">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Basic">Basic</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Composite">Composite</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Collection">Collection</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Parts#Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Human_Practices">
                    HP
                </A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Team">
                    Team
                </A></LI><LI style="list-style-type: none;"><A href="https://2018.igem.org/Team:TJU_China/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Collaborations">Collaboration</A></LI><LI><A href="https://2018.igem.org/Team:TJU_China/Attributions">Attributions</A></LI></DIV><DIV class="notebook_month"><DIV class="notebook_month_p">Mar.</DIV><DIV class="notebook_month_p">Apr.</DIV><DIV class="notebook_month_p">May</DIV><DIV class="notebook_month_p">Jun.</DIV><DIV class="notebook_month_p">Jul.</DIV><DIV class="notebook_month_p">Aug.</DIV><DIV class="notebook_month_p">Sep.</DIV><DIV class="notebook_month_p">Oct.</DIV></DIV><DIV class="notebook_information"><DIV id="notebook_March"><DIV class="notebook_Month_head">March</DIV><DIV class="notebook_week_p">After a lot of document viewing and brainstorms, we finally came up with a clear idea about how we were going
                to carry out our part of project. We decided to construct the RNPs (complex of sgRNA and Cas9 protein) and
                to wrap them with delivery vector in order to improve the gene-editing efficiency of the system into the
                cells.
            </DIV></DIV><DIV id="notebook_April"><DIV class="notebook_Month_head">April</DIV><DIV class="notebook_week"><DIV><BUTTON id="4week3" class="notebook_information_button3">week3</BUTTON></DIV><DIV id="4notebook_week3" class="notebook_information_word3" style="display:none;"><DIV class="notebook_week_p">We continued to search for related documents in order to choose the appropriate target sequences of gene
                        for our project. After the final decision of making eGFP as our interested sequence of gene, we immediately
                        sent out the primers of sgRNA we designed to GENEWIZ, and we also sent lots of emails to different
                        labs to see if they can offer us a stable cell strain expressing eGFP protein.</DIV></DIV><DIV><BUTTON id="4week4" class="notebook_information_button4">week4</BUTTON></DIV><DIV id="4notebook_week4" class="notebook_information_word4" style="display:none;"><DIV class="notebook_week_p">Meanwhile, we successfully transformed the plasmid pET-NLS-Cas9-6xHis into E.coli (Rosetta), then we
                        tried to express and purify the cas9 protein according to the protocol. But the results were not
                        so ideal, Cas9 protein degraded. </DIV><DIV class="notebook_week_p"><B>Figure 1.</B>Plate of Rosetta transfected with pET-NLS-Cas9-6xHis plasmid.</DIV></DIV></DIV></DIV><DIV id="notebook_May"><DIV class="notebook_Month_head">May</DIV><DIV class="notebook_week"><DIV><BUTTON id="5week1" class="notebook_information_button1">week1</BUTTON></DIV><DIV id="5notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p">PCR and Gel Purification (TIANgel Midi Purification Kit) of the template of sgRNA, we tried different
                        Tm temperatures.</DIV><DIV class="notebook_week_p"><B>Figure 2.</B>Assembly of template of sgRNA using PCR. Lane M, marker. Lane 1, Tm=65.2℃. Lane 2, Tm=66.2℃.
                        Lane 3, Tm=66.8℃. Lane 4, Tm=67.2℃. Lane 5, Tm=68.2℃.</DIV></DIV><DIV><BUTTON id="5week2" class="notebook_information_button2">week2</BUTTON></DIV><DIV id="5notebook_week2" class="notebook_information_word2" style="display:none;"><DIV class="notebook_week_p">After two weeks of hard working, we got our first batch of Cas9 protein.</DIV><DIV class="notebook_week_p"><B>Figure 3.</B>Result of purification by Affinity chromatography (Ni-NTA) . Lane M, marker. Lane 1,
                        before eluted by Buffer A. Lane 2, after eluted by Buffer A. Lane 3, before eluted by Buffer B. Lane
                        4, after eluted by Buffer B. Buffer A(50 mM Tris-HCl (pH 8.0), 1 M NaCl, 20% glycerol, 2 mM TCEP
                        and 20 mM imidazole). Buffer B( 50 mM Tris-HCl (pH 8), 1 M NaCl, 20% glycerol, 2 mM TCEP and 500
                        mM imidazole).</DIV><DIV class="notebook_week_p"><B>Figure 4.</B>SDS-PAGE result of ion exchange. Lane M, marker. Lane 1, Cas9 protein after ion exchange
                        purification. </DIV><DIV class="notebook_week_p"><B>Figure 5.</B>SDS-PAGE result of gel filtration. Lane M, marker. Lane 1, Cas9 protein after gel filtration
                        purification.
                    </DIV><DIV class="notebook_week_p">At the same time, we tried our first transcription and preparation of sgRNA using T7 High Efficiency
                        Transcription Kit and EasyPure RNA Purification Kit (TransGene Biotech) but failed. We supposed that
                        the template might be polluted by RNase during the process.</DIV></DIV><DIV><BUTTON id="5week3" class="notebook_information_button3">week3</BUTTON></DIV><DIV id="5notebook_week3" class="notebook_information_word3" style="display:none;"><DIV class="notebook_week_p">We searched for information and knowledge of experiments dealing with RNA and then retried the PCR and
                        Gel Purification of the template according strictly to the RNA principles. However, we failed again.
                        On the other hand, we managed to get plasmid encoding eGFP by using TIANprep Mini Plasmid Kit (TIANGEN).</DIV><DIV class="notebook_week_p"><B>Figure 6.</B>Extraction of eGFP plasmid. Lane M, marker. Lane 1, eGFP plasmid.</DIV></DIV><DIV><BUTTON id="5week4" class="notebook_information_button4">week4</BUTTON></DIV><DIV id="5notebook_week4" class="notebook_information_word4" style="display:none;"><DIV class="notebook_week_p">We confirmed the best Tm temperature of PCR was 67.2℃ after several preliminary experiments. And this
                        time, there were Lanes shown on the RNA gel, but they were not clear. And the RNA marker (TAKARA)
                        didn’t appear exactly as the instruction. Then we moved forward to the In vitro digestion of DNA
                        but failed. Since that the result of the SDS-PAGE of Cas9 protein showed no problem, we considered
                        that the sgRNA might degraded.</DIV><DIV class="notebook_week_p"><B>Figure 7.</B>In vitro cleavage of plasmid. Lane M, marker. Lane 1, eGFP plasmid. Lane 2, sgRNA:Cas9:DNA=10:10:1.</DIV></DIV></DIV></DIV><DIV id="notebook_June"><DIV class="notebook_Month_head">June</DIV><DIV class="notebook_week"><DIV class="notebook_week_p">Preparation for the final-term exams. </DIV></DIV></DIV><DIV id="notebook_July"><DIV class="notebook_Month_head">July</DIV><DIV class="notebook_week"><DIV><BUTTON id="7week1" class="notebook_information_button1">week1,2,3</BUTTON></DIV><DIV id="7notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p">Most members joined a summer camp as a part of school lessons.</DIV></DIV><DIV><BUTTON id="7week4" class="notebook_information_button4">week4</BUTTON></DIV><DIV id="7notebook_week4" class="notebook_information_word4" style="display:none;"><DIV class="notebook_week_p">In order to reduce the influence of RNase on the subsequent experiment, the phenol-chloroform method
                        was used to remove RNase. And after several experiments, we confirmed that the best reaction time
                        for transcription was 16 hours.</DIV></DIV></DIV></DIV><DIV id="notebook_August"><DIV class="notebook_Month_head">August</DIV><DIV class="notebook_week"><DIV><BUTTON id="8week1" class="notebook_information_button1">week1</BUTTON></DIV><DIV id="8notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p">We tried different ways of purification of the transcription products——EasyPure RNA Purification Kit
                        and phenol-chloroform purification. And thanks to our continuous efforts, we managed to run the nucleic
                        acid gel electrophoresis of sgRNA perfectly.</DIV><DIV class="notebook_week_p"><B>Figure 8.</B> In vitro transcription of sgRNA. Lane M, marker. Lane 1, 100ng sgRNA. Lane 2, 200ng
                        sgRNA.
                    </DIV></DIV><DIV><BUTTON id="8week2" class="notebook_information_button2">week2</BUTTON></DIV><DIV id="8notebook_week2" class="notebook_information_word2" style="display:none;"><DIV class="notebook_week_p">We succeeded in the In vitro digestion of DNA but the efficiency was not as good as we expected, so we
                        began to find the reason and solutions to improve the cleavage efficiency. The experiments was carried
                        out according to the protocol (NEB, In vitro digestion of DNA with EnGen Cas9 NLS, S. pyogenes).</DIV><DIV class="notebook_week_p"><B>Figure 9.</B>The effect of different components on the cleavage reaction. Lane M, marker. Lane 1,
                        eGFP plasmid. Lane 2, plasmid+sgRNA. Lane 3, plsmid+Cas9 protein. Lane 4, plasmid+reaction buffer.</DIV></DIV><DIV><BUTTON id="8week3" class="notebook_information_button3">week3</BUTTON></DIV><DIV id="8notebook_week3" class="notebook_information_word3" style="display:none;"><DIV class="notebook_week_p">
                        We got plasmid of better quality.</DIV><DIV class="notebook_week_p"><B>Figure 10.</B>Extraction of eGFP plasmid. Lane M, marker. Lane 1, eGFP plasmid. We also designed
                        the experiments in order to improve the reaction conditions.</DIV><DIV class="notebook_week_p"><B>Figure 11.</B>In vitro digestion of eGFP plasmid. (A) Lane M, marker. Lane 1, eGFP plasmid. Lane
                        2, plasmid digested with EcoRI. Lane 3-10 are according to table B. (B) Experiment design to improve
                        the cleavage efficiency.</DIV></DIV><DIV><BUTTON id="8week4" class="notebook_information_button4">week4</BUTTON></DIV><DIV id="8notebook_week4" class="notebook_information_word4" style="display:none;"><DIV class="notebook_week_p"><B>Figure 12.</B> Improved in vitro digestion of eGFP plasmid. (A,B) Lane M, marker. Lane 1，eGFP plasmid.
                        Lane 2, plasmid digested with EcoRI. Lane 3-4, sgRNA:Cas9:DNA=10:10:1. Lane 5-6, sgRNA:Cas9:DNA=10:20:1.
                        Lane 7-8, sgRNA:Cas9:DNA=20:20:1. As Figure 12 shows, sgRNA:Cas9=1:2(mole number) has the better
                        gene editing efficiency, and the percentage of cleavage may be higher when the quantity of DNA was
                        reduced. And we also found that the DNA might be tracked in the sample holes due to Cas9 protein.</DIV></DIV><DIV><BUTTON id="8week5" class="notebook_information_button5">About Targeting mitochondrion</BUTTON></DIV><DIV id="8notebook_week5" class="notebook_information_word5" style="display:none;"><DIV class="notebook_week_p">We learned about how to design sgRNA and molecular cloning experiments, intensively studied the literatures
                        on mitochondrial editing and researched on the process of label entering mitochondrial and in-line
                        granule labeling. At the same time, we searched for various genes on mtDNA, reviewed the literature
                        on mitochondrial diseases, and examined the mutation sites of each gene in mitochondria as well as
                        their effects, sorting out the related materials of mitochondrial genome. Based on the comprehensive
                        research on the existing mitochondrial editing methods, we designed a complete experimental scheme,
                        including the plasmid construction scheme and finished designing primers.</DIV></DIV></DIV></DIV><DIV id="notebook_September"><DIV class="notebook_Month_head">September</DIV><DIV class="notebook_week"><DIV><BUTTON id="9week1" class="notebook_information_button1">week1</BUTTON></DIV><DIV id="9notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p">COS7-GFP cell Lane was offered by School of Public Health, University of South China.[1] We immediately
                        began to culture cells after receiving them and successfully observed the green fluorescence.</DIV></DIV><DIV><BUTTON id="9week2" class="notebook_information_button2">week2</BUTTON></DIV><DIV id="9notebook_week2" class="notebook_information_word2" style="display:none;"><DIV class="notebook_week_p">We explored and improved the experimental conditions for many times and finally got a better cutting
                        condition. Then we started to study the cutting efficiency of the system with carrier.</DIV></DIV><DIV><BUTTON id="9week3" class="notebook_information_button3">week3</BUTTON></DIV><DIV id="9notebook_week3" class="notebook_information_word3" style="display:none;"><DIV class="notebook_week_p"><B>Figure 13.</B>Characterization of sgRNA/Cas9 complex(RNP), BODIPY/RNP, Liposome/RNP, and BODIPY/Liposome/RNP.
                        (A) Z-Ave of sgRNA, Cas9, RNP, Liposome/RNP, BODIPY/RNP, and BODIPY/Liposome/RNP. (B) Zeta potential
                        of sgRNA,Cas9,RNP,BODIPY,BODIPY/RNP, and BODIPY/Liposome/RNP.
                    </DIV></DIV><DIV><BUTTON id="9week4" class="notebook_information_button4">week4</BUTTON></DIV><DIV id="9notebook_week4" class="notebook_information_word4" style="display:none;"><DIV class="notebook_week_p"><B>Figure 14.</B>In vitro digestion of DNA with BODIPY/RNP. (A) Lane 1, eGFP plasmid. Lane 2-7 are set
                        according to table B. Lane M, marker. (B) Experiment design.</DIV></DIV><DIV><BUTTON id="9week5" class="notebook_information_button5">About Targeting mitochondrion</BUTTON></DIV><DIV id="9notebook_week5" class="notebook_information_word5" style="display:none;"><DIV class="notebook_week_p">
                        The original fragment was synthesized by PCR and then overlapped.
                        The first batch of competent cells were made and transformed with the original plasmids to obtain
                        the cloning bacteria, from which we successfully extracted the plasmids.
                        We doubled digested the plasmids repeatedly with Xba1 and Bmgb1 enzymes. After lots of failures,
                        we did a series of double enzyme digestion pre-experiment to improve the original schemes.
                    </DIV><DIV class="notebook_week_p"><B>Figure 15.</B>Extraction of pET-NLS-Cas9-6xHis plasmid. Lane M, marker. Lane 1, plasmid.</DIV><DIV class="notebook_week_p"><B>Figure 16.</B>The result of pET-NLS-Cas9-6xHis plasmid digestion. Lane M, marker. Lane1-2, double
                        digestion with Xba1+Bmgb1.Lane 3, single digestion with Xba1. Lane 4, single digestion with Bmgb1.
                        Lane 5, plasmid.</DIV><DIV class="notebook_week_p">We change the enzymes to Xba1 and nhe1 endonuclease, redesign primers, extend the insert and complete
                        overlap and amplification of the product. After double enzyme digestion, the vector was connected
                        to insert and transformed into cell. However, we found the final construction in July contain so
                        much mutations in the vector that it can’t be used, we reconverted the original plasmid and extracted
                        the original plasmid to do reconstruction.</DIV><DIV class="notebook_week_p"><B>Figure 17.</B>The result of pET-NLS-Cas9-6xHis plasmid digestion. (A) cleavage 15mins. Lane M, marker.
                        Lane 1, 1μg plasmid double digestion with Xba1 and Nhe1. Lane 2, 2μg plasmid double digestion with
                        Xba1 and Nhe1. Lane 3, 1μg plasmid single digestion with Xba1. Lane 4, 1μg plasmid single digestion
                        with Nhe1. Lane 5, plasmid. (B) cleavage 60mins. Lane M, marker. Lane 1, 1μg plasmid double digestion
                        with Xba1 and Nhe1. Lane 2, 2μg plasmid double digestion with Xba1 and Nhe1. Lane 3, 1μg plasmid
                        single digestion with Xba1. Lane 4, 1μg plasmid single digestion with Nhe1. Lane 5, plasmid.</DIV><DIV class="notebook_week_p"><B>Figure 18.</B>Amplification of the overlapped COX8a and Cas9. Lane M, marker. Lane 1, COX8a+Cas9
                        fragment after amplification. The concentration of gel extraction product is 10.8ng/μl，the volume
                        is 200μl</DIV><DIV class="notebook_week_p"><B>Figure 19.</B> Double digestion of COX8a+Cas9 fragment. Lane M, marker. Lane 1, COX8a+Cas9 fragment
                        double digestion with Xba1 and Nhe1.</DIV><DIV class="notebook_week_p"><B>Figure 20.</B>The result of fragments synthetic. Lane M, marker. Lane 1, COX8a fragment. Lane 2,
                        Cas9 fragment. Lane 3, COX8a+Cas9 fragment.</DIV></DIV></DIV></DIV><DIV id="notebook_October"><DIV class="notebook_Month_head">October</DIV><DIV class="notebook_week"><DIV><BUTTON id="10week1" class="notebook_information_button1">week1</BUTTON></DIV><DIV id="10notebook_week1" class="notebook_information_word1" style="display:none;"><DIV class="notebook_week_p"><B>Figure 21.</B> EGFP gene disruption of COS7-GFP cell line. </DIV></DIV><DIV><BUTTON id="10week5" class="notebook_information_button5">About Targeting mitochondrion</BUTTON></DIV><DIV id="10notebook_week5" class="notebook_information_word5" style="display:none;"><DIV class="notebook_week_p">Since the previous clones were all negative, we analysed the cause of failure and redesigned the experimental
                        protocol. The Cox8a fragment was reamplified and digested by double enzymes. Also we explored the
                        conditions of dephosphorylation after double enzyme digestion adn redesign the primers to synthesize
                        the other two plasmids. After repeated vector enzyme digestion and dephosphorylation, fragment amplification
                        and double enzyme digestion and connection as well as ligation and transformation time after time,
                        we finally obtained the cloning bacteria which were proved positive by colony PCR.</DIV><DIV class="notebook_week_p"><B>Figure 22.</B>The result of pET-NLS-Cas9-6xHis plasmid digestion. Lane M, marker. Lane 1-8, plasmid
                        double digestion with Xba1 and Nhe1. Lane 9, plasmid single digestion with Xba1. Lane 10, plasmid
                        single digestion with Xba1. Lane 11, plasmid.</DIV><DIV class="notebook_week_p"><B>Figure 23.</B>The result of bacterial colony PCR confirmation. Lane M, marker. Lane1-4, SOD2. Lane
                        5-13, COX8A. Lane 14-17, ATP5. </DIV><DIV class="notebook_week_p"><B>Figure 24.</B>The result of construction. (A) The construction of pET-NLS-Cas9-6xHis plasmid with
                        SOD2 MTS. Lane 1, MTS fragment. Lane 2, segment of Cas9. Lane 3, overlap of MTS and Cas9 fragments.
                        Lane 4, re-constructed plasmid. (B) The construction of pET-NLS-Cas9-6xHis plasmid with ATP5 MTS.
                        Lane 1, MTS fragment. Lane 2, segment of Cas9. Lane 3, overlap of MTS and Cas9 fragments. Lane 4,
                        re-constructed plasmid.</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>