Team:TJU China/Notebook/Group3
<!DOCTYPE >
Home
Project
Description
Experiments
Notebook
Demonstrate
Parts
Parts Overview
Basic
Composite
Collection
Improve
Model
HP
Human Practices
Public Engagement
Team
Members
Collaboration
Attributions
Mar.
Apr.
May
Jun.
Jul.
Aug.
Sep.
Oct.
March
After a lot of document viewing and brainstorms, we finally came up with a clear idea about how we were going                 to carry out our part of project. We decided to construct the RNPs (complex of sgRNA and Cas9 protein) and                 to wrap them with delivery vector in order to improve the gene-editing efficiency of the system into the                 cells.
April
week3
We continued to search for related documents in order to choose the appropriate target sequences of gene                         for our project. After the final decision of making eGFP as our interested sequence of gene, we immediately                         sent out the primers of sgRNA we designed to GENEWIZ, and we also sent lots of emails to different                         labs to see if they can offer us a stable cell strain expressing eGFP protein.
week4
Meanwhile, we successfully transformed the plasmid pET-NLS-Cas9-6xHis into E.coli (Rosetta), then we                         tried to express and purify the cas9 protein according to the protocol. But the results were not                         so ideal, Cas9 protein degraded.
Figure 1.Plate of Rosetta transfected with pET-NLS-Cas9-6xHis plasmid.
May
week1
PCR and Gel Purification (TIANgel Midi Purification Kit) of the template of sgRNA, we tried different                         Tm temperatures.
Figure 2.Assembly of template of sgRNA using PCR. Lane M, marker. Lane 1, Tm=65.2℃. Lane 2, Tm=66.2℃.                         Lane 3, Tm=66.8℃. Lane 4, Tm=67.2℃. Lane 5, Tm=68.2℃.
week2
After two weeks of hard working, we got our first batch of Cas9 protein.
Figure 3.Result of purification by Affinity chromatography (Ni-NTA) . Lane M, marker. Lane 1,                         before eluted by Buffer A. Lane 2, after eluted by Buffer A. Lane 3, before eluted by Buffer B. Lane                         4, after eluted by Buffer B. Buffer A(50 mM Tris-HCl (pH 8.0), 1 M NaCl, 20% glycerol, 2 mM TCEP                         and 20 mM imidazole). Buffer B( 50 mM Tris-HCl (pH 8), 1 M NaCl, 20% glycerol, 2 mM TCEP and 500                         mM imidazole).
Figure 4.SDS-PAGE result of ion exchange. Lane M, marker. Lane 1, Cas9 protein after ion exchange                         purification.
Figure 5.SDS-PAGE result of gel filtration. Lane M, marker. Lane 1, Cas9 protein after gel filtration                         purification.
At the same time, we tried our first transcription and preparation of sgRNA using T7 High Efficiency                         Transcription Kit and EasyPure RNA Purification Kit (TransGene Biotech) but failed. We supposed that                         the template might be polluted by RNase during the process.
week3
We searched for information and knowledge of experiments dealing with RNA and then retried the PCR and                         Gel Purification of the template according strictly to the RNA principles. However, we failed again.                         On the other hand, we managed to get plasmid encoding eGFP by using TIANprep Mini Plasmid Kit (TIANGEN).
Figure 6.Extraction of eGFP plasmid. Lane M, marker. Lane 1, eGFP plasmid.
week4
We confirmed the best Tm temperature of PCR was 67.2℃ after several preliminary experiments. And this                         time, there were Lanes shown on the RNA gel, but they were not clear. And the RNA marker (TAKARA)                         didn’t appear exactly as the instruction. Then we moved forward to the In vitro digestion of DNA                         but failed. Since that the result of the SDS-PAGE of Cas9 protein showed no problem, we considered                         that the sgRNA might degraded.
Figure 7.In vitro cleavage of plasmid. Lane M, marker. Lane 1, eGFP plasmid. Lane 2, sgRNA:Cas9:DNA=10:10:1.
June
Preparation for the final-term exams.
July
week1,2,3
Most members joined a summer camp as a part of school lessons.
week4
In order to reduce the influence of RNase on the subsequent experiment, the phenol-chloroform method                         was used to remove RNase. And after several experiments, we confirmed that the best reaction time                         for transcription was 16 hours.
August
week1
We tried different ways of purification of the transcription products——EasyPure RNA Purification Kit                         and phenol-chloroform purification. And thanks to our continuous efforts, we managed to run the nucleic                         acid gel electrophoresis of sgRNA perfectly.
Figure 8. In vitro transcription of sgRNA. Lane M, marker. Lane 1, 100ng sgRNA. Lane 2, 200ng                         sgRNA.
week2
We succeeded in the In vitro digestion of DNA but the efficiency was not as good as we expected, so we                         began to find the reason and solutions to improve the cleavage efficiency. The experiments was carried                         out according to the protocol (NEB, In vitro digestion of DNA with EnGen Cas9 NLS, S. pyogenes).
Figure 9.The effect of different components on the cleavage reaction. Lane M, marker. Lane 1,                         eGFP plasmid. Lane 2, plasmid+sgRNA. Lane 3, plsmid+Cas9 protein. Lane 4, plasmid+reaction buffer.
week3
We got plasmid of better quality.
Figure 10.Extraction of eGFP plasmid. Lane M, marker. Lane 1, eGFP plasmid. We also designed                         the experiments in order to improve the reaction conditions.
Figure 11.In vitro digestion of eGFP plasmid. (A) Lane M, marker. Lane 1, eGFP plasmid. Lane                         2, plasmid digested with EcoRI. Lane 3-10 are according to table B. (B) Experiment design to improve                         the cleavage efficiency.
week4
Figure 12. Improved in vitro digestion of eGFP plasmid. (A,B) Lane M, marker. Lane 1，eGFP plasmid.                         Lane 2, plasmid digested with EcoRI. Lane 3-4, sgRNA:Cas9:DNA=10:10:1. Lane 5-6, sgRNA:Cas9:DNA=10:20:1.                         Lane 7-8, sgRNA:Cas9:DNA=20:20:1. As Figure 12 shows, sgRNA:Cas9=1:2(mole number) has the better                         gene editing efficiency, and the percentage of cleavage may be higher when the quantity of DNA was                         reduced. And we also found that the DNA might be tracked in the sample holes due to Cas9 protein.
About Targeting mitochondrion
We learned about how to design sgRNA and molecular cloning experiments, intensively studied the literatures                         on mitochondrial editing and researched on the process of label entering mitochondrial and in-line                         granule labeling. At the same time, we searched for various genes on mtDNA, reviewed the literature                         on mitochondrial diseases, and examined the mutation sites of each gene in mitochondria as well as their effects, sorting out the related materials of mitochondrial genome. Based on the comprehensive                         research on the existing mitochondrial editing methods, we designed a complete experimental scheme,                         including the plasmid construction scheme and finished designing primers.
September
week1
COS7-GFP cell Lane was offered by School of Public Health, University of South China.[1] We immediately                         began to culture cells after receiving them and successfully observed the green fluorescence.
week2
We explored and improved the experimental conditions for many times and finally got a better cutting                         condition. Then we started to study the cutting efficiency of the system with carrier.
week3
Figure 13.Characterization of sgRNA/Cas9 complex(RNP), BODIPY/RNP, Liposome/RNP, and BODIPY/Liposome/RNP.                         (A) Z-Ave of sgRNA, Cas9, RNP, Liposome/RNP, BODIPY/RNP, and BODIPY/Liposome/RNP. (B) Zeta potential                         of sgRNA,Cas9,RNP,BODIPY,BODIPY/RNP, and BODIPY/Liposome/RNP.
week4
Figure 14.In vitro digestion of DNA with BODIPY/RNP. (A) Lane 1, eGFP plasmid. Lane 2-7 are set                         according to table B. Lane M, marker. (B) Experiment design.
About Targeting mitochondrion
The original fragment was synthesized by PCR and then overlapped.
The first batch of competent cells were made and transformed with the original plasmids to obtain                         the cloning bacteria, from which we successfully extracted the plasmids.
We doubled digested the plasmids repeatedly with Xba1 and Bmgb1 enzymes. After lots of failures,                         we did a series of double enzyme digestion pre-experiment to improve the original schemes.
Figure 15.Extraction of pET-NLS-Cas9-6xHis plasmid. Lane M, marker. Lane 1, plasmid.
Figure 16.The result of pET-NLS-Cas9-6xHis plasmid digestion. Lane M, marker. Lane1-2, double                         digestion with Xba1+Bmgb1.Lane 3, single digestion with Xba1. Lane 4, single digestion with Bmgb1.                         Lane 5, plasmid.
We change the enzymes to Xba1 and nhe1 endonuclease, redesign primers, extend the insert and complete                         overlap and amplification of the product. After double enzyme digestion, the vector was connected                         to insert and transformed into cell. However, we found the final construction in July contain so                         much mutations in the vector that it can’t be used, we reconverted the original plasmid and extracted                         the original plasmid to do reconstruction.
Figure 17.The result of pET-NLS-Cas9-6xHis plasmid digestion. (A) cleavage 15mins. Lane M, marker.                         Lane 1, 1μg plasmid double digestion with Xba1 and Nhe1. Lane 2, 2μg plasmid double digestion with                         Xba1 and Nhe1. Lane 3, 1μg plasmid single digestion with Xba1. Lane 4, 1μg plasmid single digestion                         with Nhe1. Lane 5, plasmid. (B) cleavage 60mins. Lane M, marker. Lane 1, 1μg plasmid double digestion                         with Xba1 and Nhe1. Lane 2, 2μg plasmid double digestion with Xba1 and Nhe1. Lane 3, 1μg plasmid                         single digestion with Xba1. Lane 4, 1μg plasmid single digestion with Nhe1. Lane 5, plasmid.
Figure 18.Amplification of the overlapped COX8a and Cas9. Lane M, marker. Lane 1, COX8a+Cas9                         fragment after amplification. The concentration of gel extraction product is 10.8ng/μl，the volume                         is 200μl
Figure 19. Double digestion of COX8a+Cas9 fragment. Lane M, marker. Lane 1, COX8a+Cas9 fragment                         double digestion with Xba1 and Nhe1.
Figure 20.The result of fragments synthetic. Lane M, marker. Lane 1, COX8a fragment. Lane 2,                         Cas9 fragment. Lane 3, COX8a+Cas9 fragment.
October
week1
Figure 21. EGFP gene disruption of COS7-GFP cell line.
About Targeting mitochondrion
Since the previous clones were all negative, we analysed the cause of failure and redesigned the experimental                         protocol. The Cox8a fragment was reamplified and digested by double enzymes. Also we explored the                         conditions of dephosphorylation after double enzyme digestion adn redesign the primers to synthesize                         the other two plasmids. After repeated vector enzyme digestion and dephosphorylation, fragment amplification                         and double enzyme digestion and connection as well as ligation and transformation time after time,                         we finally obtained the cloning bacteria which were proved positive by colony PCR.
Figure 22.The result of pET-NLS-Cas9-6xHis plasmid digestion. Lane M, marker. Lane 1-8, plasmid                         double digestion with Xba1 and Nhe1. Lane 9, plasmid single digestion with Xba1. Lane 10, plasmid                         single digestion with Xba1. Lane 11, plasmid.
Figure 23.The result of bacterial colony PCR confirmation. Lane M, marker. Lane1-4, SOD2. Lane                         5-13, COX8A. Lane 14-17, ATP5.
Figure 24.The result of construction. (A) The construction of pET-NLS-Cas9-6xHis plasmid with                         SOD2 MTS. Lane 1, MTS fragment. Lane 2, segment of Cas9. Lane 3, overlap of MTS and Cas9 fragments. Lane 4, re-constructed plasmid. (B) The construction of pET-NLS-Cas9-6xHis plasmid with ATP5 MTS.                         Lane 1, MTS fragment. Lane 2, segment of Cas9. Lane 3, overlap of MTS and Cas9 fragments. Lane 4,                         re-constructed plasmid.
