Team:Mingdao/Parts
About Us
Team Members
Attribution
Motivation
Design
Experiment
Proof
Demo
Result
Notebook
Basic Parts
Composite Parts
Parts Collection
Introduction
Principal
Experiment
Future
Interlude
Lab Work
Project Work
Shipment
Q&A
Survey
Promotion
Bronze
Silver
Gold
In iGEM 2016, we designed and created a novel BLOOD ALCOHOL METER (iMeter) inspired by blood glucose meter (BGM). The glucose oxidase (GOX) was used as an oxidoreductase to catalyze the oxidation of glucose in the electrochemical analyzer. In a similar way, the alcohol oxidase (AOX) is an oxidoreductase to catalyze the oxidation of alcohol and could be applied in the alcohol measurements. This year, we proved the concept by cloning genes, protein analysis, enzyme activity assay and electrochemical modeling.
For the BioBrick parts, we’ve created and submitted 11 new parts to the Registry including AOX genes [ BBa_K1991000 , BBa_K1991001 ], Lpp-OmpA (LO) genes [ BBa_K1991004 ] and LO-AOX [ BBa_K1991005 , BBa_K1991006 ] fusion protein genes as Basic Parts, as well as a constitutive promoter [ BBa_J23101 ]and RBS [ BBa_B0034 ]-driven LO fusion protein genes [ BBa_K1991002 , BBa_K1991003 , BBa_K1991007 , BBa_K1991008 , BBa_K1991009 , BBa_K1991010 ] as Composite Parts. We have demonstrated and collected all of these parts for the application of alcohol oxidase in the blood alcohol meter.
AOX1/pSB1C3
Registry: BBa_K1991000
Source: Pichia pastoris
Codon Optimization: E. coli codon preference
DNA synthesis: gBlocks® Gene Fragments (IDT)
Notebook: PDF ; Sequencing: VF2 , VR
Primers:
1. AOX1-XbaI-F: 5’- GGCGAGTCTAGATGGCTATTCCGGAAGAGTT -3’
2. AOX1-PstI-R: 5’- AAAATTCTGCAGCGGCCGCTACTAGTATCAAAAGCGCGCAA - 3’
Cloning Procedure:
The synthesized AOX1 gene was amplified by PCR and digested by XbaI and PstI, followed by cloning onto pSB1C3 which was cut by XbaI and PstI. The part has been confirmed by sequencing.
AOX2/pSB1C3
Registry: BBa_K1991001
Source: Pichia pastoris
Codon Optimization: E. coli codon preference
DNA synthesis: gBlocks® Gene Fragments (IDT)
Notebook: PDF ; Sequencing: VF2 , VR
Primers:
1. AOX2-XbaI-F: 5’- GGAGAGTCTAGATGGCCATCCCTGAGGAATTTG -3’
2. AOX2-PstI-R: 5’- AAAAAACTGCAGCGGCCGCTACTAGTATCAGAAGCGCGCTAA -3’
Cloning Procedure:
The synthesized AOX2 gene was amplified by PCR and digested by XbaI and PstI, followed by cloning onto pSB1C3 which      was cut by XbaI and PstI. The part has been confirmed by sequencing.
Lpp-OmpA-BamHI/pSB1C3
Registry: BBa_K1991004
Source: Escherichia coli
Original Part: BBa_K1694035
Notebook: PDF ; Sequencing: VF2
Primers:
1. Lpp-XbaI-EcoRI-F: 5’- AGTCAAGAATTCGCGGCCGCTTCTAGATGAAAGCAACCAAGCTG -3’
2. OmpA-BamHI-SpeI-R: 5’- AAAGAGACTAGTAGGATCCACGTGTGGCAATTTCCGGGGTGA -3’
Cloning Procedure:
The DNA fragment of Lpp-OmpA (LO) was amplified by PCR using BBa_K1694035 as a template with a primer containing BamHI site and digested by EcoRI and SpeI, followed by cloning onto pSB1C3 which was cut by EcoRI and SpeI. The part has been confirmed by sequencing.
Existing part from NCTU-Formosa in 2015:
BBa_K1694002 : Lpp-OmpA-NcoI/pSB1C3
Improved part by Mingdao in 2016:
BBa_K1991004 : Lpp-OmpA-BamHI/pSB1C3<
BBa_K1991007 : Pcons-RBS-LO-BamHI/pSB1C3
Lpp and OmpA are outer membrane proteins of E. coli. Lpp-OmpA (LO) hybrid can direct heterologous proteins to bacterial cell surface. In 2015, NCTU-Formosa used it to display scFv (single chain fragment variable) antibodies on the surface of E. coli. They found that a fusion protein cannot be possible to be created under the standard BioBrick assembly rule, that is EcoRI(E)-XbaI(X)-GENE-SpeI(S)-PstI(P). The A part of EX-LO-SP and the B part of EX-scFV-SP, for example, are connected by cutting and ligation of SpeI plus PstI for the A part and XbaI plus PstI for the B part. The SCAR generated by XbaI/SpeI (ACTAGA) will form a stop codon just in front of the ATG start codon of the scFV protein of the B part. This situation has been officially mentioned by the BioBrick standard assembly .
Figure 5: Limitation of cloning a fusion protein by standard biobrick assembly
Therefore, in 2015, NCTU-Formosa created a novel part ( BBa_K1694002 ) putting NcoI site between the LO part and SpeI site. However, when considering cloning, we found that an extra NcoI site is present on Cm resistance gene making it difficult be a vector for gene cloning.
In 2016, Mingdao improved the part by replacing NcoI site with BamHI site ( BBa_K1991004 ). Also, we’ve confirmed and prove the function of LO directing a fusion protein to the cell surface with enzyme activity in our project.
Figure 6: Alternative methods of cloning a fusion protein on Biobrick parts designed and created by NCTU-FORMOSA in 2015 and MINGDAO in 2016
BBa_K1991004 : Lpp-OmpA-BamHI/pSB1C3 ( PDF ; VF2 ) The DNA fragment of Lpp-OmpA was amplified by PCR using BBa_K1694035 as a template with a primer containing BamHI site and digested by EcoRI and SpeI, followed by cloning onto pSB1C3 which was cut by EcoRI and SpeI. The part has been confirmed by sequencing.
BBa_K1991007 : Pcons-RBS-LO-BamHI/pSB1C3 ( PDF ; VF2 ) The DNA fragment of Pcons-RBS-Lpp-OmpA was amplified by PCR using BBa_K1694035 as a template with a primer containing BamHI site and digested by EcoRI and SpeI, followed by cloning onto pSB1C3 which was cut by EcoRI and SpeI. The part has been confirmed by sequencing.
* Contributions: we’ve added the info to the original part's main page ( BBa_K1694002 ).  “An alternative version with BamHI site: If you'd like to directly use this part as a vector/backbone for cloning a fusion protein, you should be noticed that an extra NcoI site is present on Cm resistance gene. BBa_K1991004 replaced NcoI site behind LO with BamHI site and provides an alternative choice for cloning a LO fusion protein”
LO-AOX1-His/pSB1C3
Registry: BBa_K1991005
Source: Escherichia coli & Pichia pastoris
Original Part: BBa_K1991000 , BBa_K1991004
Notebook: PDF ; Sequencing: VF2 , VR
Primers:
1. AOX1-BamHI-F: 5’- GAACGAGGATCCATGGCTATTCCGGAAGAGTT -3’
2. T7T-SpeI-PstI-R: 5’- AAGAGACTGCAGCGGCCGCTACTAGTATATAGTTCCTCCTTTCAG -3’
Cloning Procedure:
The DNA fragment of AOX1-His was amplified by PCR and digested by BamHI and PstI, followed by cloning onto LO-BamHI/pSB1C3 which was cut by BamHI and PstI. The part has been confirmed by sequencing.
LO-AOX2-His/pSB1C3
Registry: BBa_K1991006
Source: Escherichia coli & Pichia pastoris
Original Part: BBa_K1991001 , BBa_K1991004
Notebook: PDF ; Sequencing: VF2 , VR
Primers:
1. AOX2-BamHI-F: 5’- AAACGAGGATCCATGGCCATCCCTGAGGAATTTG -3’
2. T7T-SpeI-PstI-R: 5’- AAGAGACTGCAGCGGCCGCTACTAGTATATAGTTCCTCCTTTCAG -3’
Cloning Procedure:
The DNA fragment of AOX2-His was amplified by PCR and digested by BamHI and PstI, followed by cloning onto LO-BamHI/pSB1C3 which was cut by BamHI and PstI. The part has been confirmed by sequencing.
Pcons-RBS-AOX1/pSB1C3
Registry: BBa_K1991002
Source: Pichia pastoris
Original Part: BBa_J23101 , BBa_B0034 , BBa_K1991000
Notebook: PDF ; Sequencing: VF2
Primers:
1. Pcons-XbaI-EcoRI-F: 5’- AGTACAGAATTCGCGGCCGCTTCTAGAGTTTACAGCTAGCTCAG - 3’
2. RBS-SpeI-R: 5’- GGCGGCACTAGTATTTCTCCTCTTTCTCTAGTA -3’
Cloning Procedure:
The DNA fragment containing a constitutive promoter ( BBa_J23101 ) and RBS ( BBa_B0034 ) was amplified by PCR from BBa_K1694035 and digested by EcoRI and SpeI, followed by cloning onto AOX1/pSB1C3 which was cut by EcoRI and XbaI. The part has been confirmed by sequencing.
Pcons-RBS-AOX2/pSB1C3
Registry: BBa_K1991003
Source: Pichia pastoris
Original Part: BBa_J23101 , BBa_B0034 ,
BBa_K1991001
Notebook: PDF; Sequencing: VF2
Primers:
1. Pcons-XbaI-EcoRI-F: 5’- AGTACAGAATTCGCGGCCGCTTCTAGAGTTTACAGCTAGCTCAG - 3’
2. RBS-SpeI-R: 5’- GGCGGCACTAGTATTTCTCCTCTTTCTCTAGTA -3’
Cloning Procedure:
The DNA fragment containing a constitutive promoter ( BBa_J23101 ) and RBS ( BBa_B0034 ) was amplified by PCR from BBa_K1694035 and digested by EcoRI and SpeI, followed by cloning onto AOX2/pSB1C3 which was cut by EcoRI and XbaI. The part has been confirmed by sequencing.
Pcons-RBS-LO-BamHI/pSB1C3
Registry: BBa_K1991007
Source: Escherichia coli
Original Part: BBa_J23101 , BBa_B0034 , BBa_K1694035
Notebook: PDF ; Sequencing: VF2
Primers:
1. Pcons-XbaI-EcoRI-F: 5’- AGTACAGAATTCGCGGCCGCTTCTAGAGTTTACAGCTAGCTCAG - 3’
2. OmpA-BamHI-SpeI-R: 5’- AAAGAGACTAGTAGGATCCACGTGTGGCAATTTCCGGGGTGA -3’
Cloning Procedure:
The DNA fragment of Pcons-RBS-Lpp-OmpA was amplified by PCR using BBa_K1694035 as a template with a primer containing BamHI site and digested by EcoRI and SpeI, followed by cloning onto pSB1C3 which was cut by EcoRI and SpeI. The part has been confirmed by sequencing.
Pcons-RBS-LO-AOX1-His/pSB1C3
Registry: BBa_K1991008
Source: Escherichia coli & Pichia pastoris
Original Part: BBa_J23101 , BBa_B0034 , BBa_K1991005
Notebook: PDF ; Sequencing: VF2 , VR
Primers:
1. AOX1-BamHI-F: 5’- GAACGAGGATCCATGGCTATTCCGGAAGAGTT -3’
2. T7T-SpeI-PstI-R: 5’- AAGAGACTGCAGCGGCCGCTACTAGTATATAGTTCCTCCTTTCAG -3’
Cloning Procedure:
The DNA fragment of AOX1-His was amplified by PCR and digested by BamHI and PstI, followed by cloning onto Pcons-RBS-LO-BamHI/pSB1C3 which was cut by BamHI and PstI. The part has been confirmed by sequencing.
Pcons-RBS-LO-AOX2-His/pSB1C3
Registry: BBa_K1991009
Source: Escherichia coli & Pichia pastoris
Original Part: BBa_J23101 , BBa_B0034 , BBa_K1991006
Notebook: PDF ; Sequencing: VF2 , VR
Primers:
1. AOX2-BamHI-F: 5’- AAACGAGGATCCATGGCCATCCCTGAGGAATTTG -3’
2. T7T-SpeI-PstI-R: 5’- AAGAGACTGCAGCGGCCGCTACTAGTATATAGTTCCTCCTTTCAG -3’
Cloning Procedure:
The DNA fragment of AOX2-His was amplified by PCR and digested by BamHI and PstI, followed by cloning onto Pcons-RBS-LO-BamHI/pSB1C3 which was cut by BamHI and PstI. The part has been confirmed by sequencing.
To analyze the AOX gene expression, we run on a SDS-PAGE gel and observed by Coomassie Blue staining. The overnight-cultured E. coli were centrifuged and lysed with Lysis Buffer (12.5 mM Tris pH 6.8, 4% SDS). The resulting lysates were subjected to SDS-PAGE with a 10% polyacrylamide gel. The gel was stained with 0.25% Coomassie Brilliant Blue R250 for 2 hours and destained until the protein bands were clear. As the data showed in Figure 2, LO protein was expressed at around the estimated molecular weight of 17 kDa, LO-AOX fusion protein at 91 kDa and AOX protein at 79 kDa. The protein expression level was consistent with the data of GFP in Figure 1, demonstrating the low gene expression level of a LO fusion protein. However, so far we cannot confirm whether LO fusion protein is able to direct AOX or GFP proteins displayed on the cell surface of E. coli. We’re planning to do a subcellular fractionation to separate the outer membrane proteins for analysis in the future.
Figure 2: AOX protein analysis. SDS-PAG and Coomassie Blue staining were used to observe protein expression level. Lane 1: wild-type E. coli as a mock control; Lane 2: LO outer membrane protein [ BBa_K1991007 ] (17 kDa) expression in E. coli; Lane 3: LO-AOX fusion protein [ BBa_K1991009 ] (91 kDa) expression in E. coli.; Lane 4: AOX protein [ BBa_K1991003 ] (79 kDa) expression in E. coli.
After AOX gene expression was confirmed, we want to know its enzyme activity. AOX catalyzes the oxidation of alcohol in the following chemical reaction C2H5OH + O2  CH3CHO + H2O2 and generates hydrogen peroxide. We examined the activity of AOX enzyme by measuring H2O2 production from the oxidation of ethanol. Fluorimetric Hydrogen Peroxide Assay Kit of Sigma-Aldrich was used for the study. In the kit, the red peroxidase substrate is designed to react with hydrogen peroxide to generate red fluorescence signal that can be detected at Ex/Em = 540/590 nm in the microplate reader. We prepared 1ml of overnight cultured E. coli displaying LO-AOX enzyme [ BBa_K1991009 ]. The bacteria were centrifuged and resolved in Assay Buffer provided by the Sigma-Aldrich kit. Then, the resulting lysates were mixed with the increasing concentrations of ethanol from 0%, 0.025%, 0.05%, 0.075%, 0.10%, 0.15% to 0.2% for 3 minutes at room temperature. The tested alcohol concentrations were at the range from 0.03% (the alcohol law limit for safety driving) to 0.2% (people may lose consciousness at this level). The following procedure was according to the manufacture’s instruction. The data were read out at Ex/Em = 540/590 nm in the microplate reader (BioTek Microplate Spectrophotometer). As Figure 2 showed, the intensities of AOX activity were significantly and linearly correlated to the concentration of ethanol in a dose-dependent manner. The results indicated that LO-AOX enzyme we produced has a functional enzyme activity and reacted with alcohol concentration dose-dependently, implying we can use this assay to measure the unknown sample of alcohol.
Figure 2: AOX protein analysis. SDS-PAG and Coomassie Blue staining were used to observe protein expression level. Lane 1: wild-type E. coli as a mock control; Lane 2: LO outer membrane protein [ BBa_K1991007 ] (17 kDa) expression in E. coli; Lane 3: LO-AOX fusion protein [ BBa_K1991009 ] (91 kDa) expression in E. coli.; Lane 4: AOX protein [ BBa_K1991003 ] (79 kDa) expression in E. coli.
The following is the message from iGEM team NCTU-FORMOSA:
“To test the enzyme activity of AOX protein, which catalyzes the oxidation of alcohol and generates H2O2 products, produced by iGEM Team Mingdao, we performed HRP-TMB assay. TMB (3,3',5,5'-Tetramethylbenzidine) is oxidized and changed to a deep blue color during the enzymatic degradation of H2O2 by horse radish peroxidase (HRP). The concentration of H2O2 can be determined by checking the color change, indicating TMB assay can be used to examine the enzyme activity of AOX with the substrate of alcohol.
We cultivated 3ml of the E.coli DH5α carrying the Mingdao iGEM team’s biobrick (i.e., Pcons-RBS-LO-AOX/pSB1C3, BBa_K1991009 ) in the LB media supplemented with Chloramphenicol. And we prepared another one as the control which didn’t have any plasmid. The next day, we centrifuged 1mL of bacteria expressing LO-AOX enzyme followed by being suspended in PBS. The control group was adjusted to the OD values at the same level of the LO-AOX group. After mixing bacterial samples with ethanol, TMB was added as a substrate of HRP and incubated in the dark for 3 min. The color of solution was significantly changed to blue at an OD650 of 1.51 compared to 1.21 in the control group, clearly demonstrating the ethanol was oxidized by AOX. As a result, we proved the function of their BioBrick by testing the AOX enzyme activity.”
Pcons-RBS-LO-GFP-His/pSB1C3
Registry: BBa_K1991010
Source: Escherichia coli & Aequorea victoria
Original Part: BBa_J23101 , BBa_B0034 , BBa_K1694035
Notebook: PDF ; Sequencing: VF2 , VR
Primers:
1. GFP-BamHI-F: 5’- GGAGGTGGATCCCGTAAAGGAGAAGAACTTTTCA -3’
2. T7T-SpeI-PstI-R: 5’- AAGAGACTGCAGCGGCCGCTACTAGTATATAGTTCCTCCTTTCAG -3’
Cloning Procedure:
The DNA fragment of GFP-His was amplified by PCR and digested by BamHI and PstI, followed by cloning onto Pcons-RBS-LO-BamHI/pSB1C3 which was cut by BamHI and PstI. The part has been confirmed by sequencing.  BamHI and PstI, followed by cloning onto Pcons-RBS-LO-BamHI/pSB1C3 which was cut by BamHI and PstI. The part has been confirmed by sequencing.
The AOX gene were cloned onto pSB1C3 driven by a constitutive promoter ( BBa_J23101 ) and RBS ( BBa_B0034 ) and fused with bacterial outer membrane proteins, Lpp-OmpA (LO) to display protein on the cell surface of E. coli. To test whether the gene expression system works, we cloned a reporter gene (GFP) in the same context (i.e., Pcons-RBS-LO-GFP/pSB1C3). And the gene expression level was compared to Pcons-RBS-GFP/pSB1C3 ( BBa_K1694035 ) got from NCTU-Formosa, which has the same promoter and RBS but without LO. The clones of E. coli DH5α were cultured in LB media supplemented with 34 μg/ml chloramphenicol at 37°C overnight. Because overnight cultured LB medium has a background level of fluorescence, the bacterial GFP was measured in the PBS buffer ( Enzyme Microb Technol. 2001 ). As Figure 1 showed, the GFP expression was extremely high in E. coli. LO-GFP fusion protein expression was observed at low but significant level compared to wild-type E. coli or E. coli expressing LO outer membrane proteins.
Figure 1: Gene expression analysis. GFP gene expression was read at Ex/Em = 488/528 nm in BioTek Microplate Spectrophotometer. Lane 1: wild-type E. coli as a mock control; Lane 2: GFP expression in E. coli [ BBa_K1694035 ]; Lane 3: LO outer membrane protein expression [ BBa_K1991007 ] in E. coli; Lane 4: LO-GFP fusion protein expression [ BBa_K1991010 ] in E. coli.
Alcohol oxidase (AOX) was applied in biosensors which catalyzes the oxidation of alcohol followed by hydrogen peroxide production. There are two AOX genes present in a methylotrophic yeast, Pichia pastoris. The gene codon was optimized for expression in E. coli. The DNA sequences were synthesized by IDT and cloned onto pSB1C3 [ BBa_K1991000 , BBa_K1991001 ]. In addition, Lpp-OmpA outer membrane proteins, which can direct a fusion protein to display on the bacterial cell surface [ BBa_K1991004 , BBa_K1991007 ], were fused with AOX [ BBa_K1991005 , BBa_K1991006 ]. In our experimental results, the data showed the enzyme displayed on E. coli can react dose-dependently with various concentration of alcohol in the H2O2 production assay and modeling at the electrochemical analyzer [ BBa_K1991002 , BBa_K1991003 , BBa_K1991008 , BBa_K1991009 , BBa_K1991010 ]
Part Name
Description
Type
Length
Designer
BBa_K1991000 BBa_K1991000
AOX1/pSB1C3
Basic
1992 bp
Chen, Pei-En
BBa_K1991001 BBa_K1991001
AOX2/pSB1C3
Basic
1992 bp
Chen, Pei-En
BBa_K1991002 BBa_K1991002
Pcons-RBS-AOX1/pSB1C3
Composite
2053 bp
Chen, Pei-En
BBa_K1991003 BBa_K1991003
Pcons-RBS-AOX2/pSB1C3
Composite
2053 bp
Chen, Pei-En
BBa_K1991004 BBa_K1991004
Lpp-OmpA-BamHI/pSB1C3
Basic
441 bp
Chen, Pei-En
BBa_K1991005 BBa_K1991005
LO-AOX1-His/pSB1C3
Basic
2601 bp
Chen, Pei-En
BBa_K1991006 BBa_K1991006
LO-AOX2-His/pSB1C3
Basic
2601 bp
Chen, Pei-En
BBa_K1991007 BBa_K1991007
Pcons-RBS-LO-BamHI/pSB1C3
Composite
502 bp
Chen, Pei-En
BBa_K1991008 BBa_K1991008
Pcons-RBS-LO-AOX1-His/pSB1C3
Composite
2662 bp
Chen, Pei-En
BBa_K1991009 BBa_K1991009
Pcons-RBS-LO-AOX2-His/pSB1C3
Composite
2662 bp
Chen, Pei-En
BBa_K1991010 BBa_K1991010
Pcons-RBS-LO-GFP-His/pSB1C3
Composite
1384 bp
Chen, Pei-En
