<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Mingdao_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Mingdao/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="sp-menu-wrapper"><DIV id="sp-banner"><DIV class="sp-sub-menu-wrapper"><DIV class="sp-sub-menu"><UL class="sp-sub-memu-0"><LI><A href="https://2016.igem.org/Team:Mingdao/Team">About Us</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Team">Team Members</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Attributions">Attribution</A></LI></UL><UL class="sp-sub-menu-1"><LI><A href="https://2016.igem.org/Team:Mingdao/Project#motivation">Motivation</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Project#design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Project#experiment">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Project#proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Project#demo">Demo</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Project#result">Result</A></LI><LI><A href="#">Notebook</A></LI></UL><UL class="sp-sub-menu-2"><LI><A href="https://2016.igem.org/Team:Mingdao/Parts#basic_parts">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Parts#composite_parts">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Parts#parts_collection">Parts Collection</A></LI></UL><UL class="sp-sub-menu-3"><LI><A href="https://2016.igem.org/Team:Mingdao/Model#intro">Introduction</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Model#principal">Principal</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Model#experiment">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Model#future">Future</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Model#interlude">Interlude</A></LI></UL><UL class="sp-sub-menu-4"><LI><A href="https://2016.igem.org/Team:Mingdao/Safety#lab">Lab Work</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Safety#project">Project Work</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Safety#shipment">Shipment</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Safety#qa">Q&amp;A </A></LI></UL><UL class="sp-sub-menu-5"><LI><A href="https://2016.igem.org/Team:Mingdao/Survey">Survey</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Human_Practices">Promotion</A></LI></UL><UL class="sp-sub-menu-6"><LI><A href="https://2016.igem.org/Team:Mingdao/Achievement#bronze">Bronze</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Achievement#silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:Mingdao/Achievement#gold">Gold</A></LI></UL></DIV></DIV></DIV></DIV><DIV class="motivation"><DIV class="procontent"><P>You always heard from the news that drunk driving accidents killed people. The police improve the road traffic safety by the field sobriety test. The breathalyzer is the most often used tool to identify the suspects of “drunkards”. However, the method is not so ideal for some issues and we’ve researched the news about these. </P></DIV><DIV class="procontent"><H2>INDIVIDUAL METABOLIC DIFFERENCE</H2><P>Alcohol in the human body is flowing through the blood and metabolized in the liver. Some of the alcohol will diffuse out through breathing. Blood alcohol concentration (BAC) could be estimated from breath alcohol concentration (BAC) by <A href="https://en.wikipedia.org/wiki/Henry%27s_law">Henry’s Law</A> statistically. That is, the amount of alcohol in the blood concentration is 2100 times greater than that in the breath concentration. However, individual metabolic difference could hugely affect the accuracy of this evaluation by breathing method. 
  </P><P>The hypothesis was further supported by a research (Figure 1) and a breathalyzer experience activity (Figure 2) held at the campus of Mingdao High School, showing that the huge variation of alcohol concentrations between individual differences. </P></DIV><DIV class="procontent"><H2>DISEASES LIKE DIABETES OR METABOLIC DISORDERS</H2><P>Diabetes is a disease associated with a high glucose level in the blood. <A href="https://en.wikipedia.org/wiki/Diabetic_ketoacidosis">Ketoacidosis</A> is very common in diabetes, in which the body prefers burning the fatty acids and produce ketone bodies due to the lack of insulin. <A href="http://aacriminallaw.com/breathalyzer-101-fail-test-sober/">Acetone</A>, a breakdown product of ketone bodies, cannot be distinguished from ethanol by the infrared (IR) beams in an infrared-based breath analyzer. <A href="http://www.nature.com/ijo/journal/v31/n3/full/0803444a.html">Isopropanol</A> is a by-product of acetone metabolism which cannot be distinguished from ethanol in an electrochemical breathalyzer. That all causes false-positive breathalyzer test.</P></DIV><DIV class="procontent"><H2>REFUSAL</H2><P>Refusal to conduct the sobriety test is very common in the news on TV. The “suspects” were willing to pay the fine (i.e., NTD 90,000 in Taiwan) rather than blowing through the breathalyzer. Possibly they avoid being charged with a crime of drunk driving in such a way.</P></DIV><DIV class="procontent"><H2>People in coma or dead</H2><P>The police are never able to force a dead or unconscious people to breathe and run the breathalyzer.</P></DIV><DIV class="procontent"><P>Is a blood sample a good alternative you may think? According to our <A href="https://2016.igem.org/Team:Mingdao/Survey">survey</A> concerning the blood sample for sobriety test, people thought spending time and going to the hospital are the two most troublesome things they don’t want to do. In addition, to get and analyze the blood samples, you need a nurse drawing your blood </P></DIV><DIV class="procontent"><P>An alcohol biosensor project we discussed in our iGEM project brainstorming meeting. And we considered the possibility to create the biosensor designed with an ethanol-inducible promoter and the GFP reporter gene. However, some of our members said such a biosensor needs that the live bacteria grow in an optimized environment (i.e., 37 incubator) and express the reporter genes following by analyzing in an expensive microplate reader. </P></DIV><DIV class="procontent"><P>
We have to search another possible approach to overcome the problem as mentioned. One of our member told a story in her childhood about her grandma with diabetes and loss of feeling in her extremities caused by poor circulation. Take a look at the video below. </P></DIV></DIV><DIV id="design"><DIV class="procontent"><P>Sobriety test conducted by an alcohol breathalyzer met some problems as mentioned previously. Blood glucose meter is a smart design to measure the glucose concentration in the blood. We’d like to apply this technique to innovate IGEM BLOOD ALCOHOL METER (iMeter).</P></DIV><DIV class="procontent"><H2>BLOOD GLUCOSE METER</H2><P>Blood glucose meter (BGM) is a common commercialized household health care appliance. It is small, hand-held, portable. It is “smart” with an easy-to-use operation interface. Data processing is quick, accurate and reliable. You can obtain the measures within 5-8 seconds.</P></DIV><DIV class="procontent"><H2>PRINCIPLE OF BGM</H2><P>The BGM is an electronic device built with electrodes covered by the enzyme of glucose oxidase (GOX), which catalyzes the oxidation of glucose, and converts it to gluconic acid and hydrogen peroxide (H<SUB>2</SUB>O<SUB>2</SUB>). When H<SUB>2</SUB>O<SUB>2</SUB> meets electrode (e.g., Ag/AgCl), the electrons are generated and give a strength of current which can be measured in amperometry. </P></DIV><DIV class="procontent"><H2>GLUCOSE OXIDASE (GOX)</H2><P> </P><P> </P><P> </P><P> </P><P> </P><P><A href="https://en.wikipedia.org/wiki/Glucose_oxidase">Glucose oxidase (GOX)</A> is an oxido-reductase existing in several insects and fungi. The redox reaction performed by GOX can generate hydrogen peroxide as antibacterial materials when glucose and oxygen are present. GOX extracted from Aspergillus niger is widely used in glucose-related diagnostics and biotechnologies including biosensors and food industries. In the redox reaction, GOX requires FAD as electron mediators and finally transfer electrons to oxygen to generate hydrogen peroxide with the reduction potential of -97 ± 3 mV at pH 7.4 (<A href="https://www.ncbi.nlm.nih.gov/pubmed/25007396">Anal Chem. 2014</A>).</P></DIV><DIV class="procontent"><H2>ALCOHOL OXIDASE (AOX)</H2><P> </P><P> </P><P> </P><P> </P><P> </P><P><A href="https://en.wikipedia.org/wiki/Alcohol_oxidase">Alcohol oxidase (AOX)</A> is also an oxido-reductase that catalyzes the oxidation of alcohol and produce aldehyde and hydrogen peroxide. The substrates of AOX are primary alcohols including methanol and some short-chain alcohols such as ethanol. AOX is present in the methylotrophic yeast like <A href="https://en.wikipedia.org/wiki/Pichia_pastoris">Pichia pastoris</A>, which has two genes, AOX1, AOX2 under a strong methanol inducible promoter. The yeast uses AOX genes to metabolize methanol as a carbon and energy source. The AOX enzymes were used in biosensors (<A href="https://www.ncbi.nlm.nih.gov/pubmed/16023950">Biosens Bioelectron. 2005</A>) but with a limited application because its electrochemical properties including reduction potential are not well studied.  </P></DIV><DIV class="procontent"><H2>DISPLAYING AOX ENZYME ON THE CELL SURFACE OF E. COLI</H2><P>Protein induction and purification may cost much time up to 12 hours and need tricky technical skills. To simplify the procedure of AOX enzyme application, we obtained the idea from the project of an iGEM Team – <A href="https://2015.igem.org/Team:NCTU_Formosa/Design">NCTU_Formosa</A> in 2015. A protein can be displayed on the cell surface of E. coli by fusing to Lpp-OmpA. Lipoprotein (<A href="http://www.uniprot.org/uniprot/P69776">Lpp</A>) is a major outer membrane of E. coli which interacts with the peptidoglycan to maintain the structure and function of cell membrane. Another transmembrane protein called outer membrane protein A (<A href="http://www.uniprot.org/uniprot/P0A910">OmpA</A>) is involved in bacterial conjugation and phage infection. A study showed that Lpp-OmpA hybrid can direct the heterologous protein GFP to the external surface of E. coli (<A href="https://www.ncbi.nlm.nih.gov/pubmed/11118595">Enzyme Microb Technol. 2001</A>). Bacillus lipase (<A href="https://www.ncbi.nlm.nih.gov/pubmed/25163839">J Microbiol. 2014</A>) and Fungi xylanase (<A href="https://www.ncbi.nlm.nih.gov/pubmed/25691337">Curr Microbiol. 2015</A>) were demonstrated to be displayed on the cell surface of E. coli and maintained the functional enzyme activities. We’d like to display the AOX enzyme on bacterial surface by fusing with Lpp-OmpA and apply to the electrochemical analyzer by depositing on the test strip.</P></DIV><DIV class="procontent"><H2>DEVELOPING IGEM BLOOD ALCOHOL METER (iMeter)</H2><P>In order to replace GOX with bacteria displaying AOX on the cell surface to the sample test strips and apply it to the electrochemical analyzer, we’ve conducted the following experiments to demonstrate it works.</P><DIV class="procontent"><H3>GENE CLONING</H3><P>AOX genes synthesized by IDT and Lpp-OmpA got from NCTU-Formosa were amplified by PCR followed by restriction enzyme (RE) digestion, and finally cloned onto pSB1C3 based on the biobrick standard assembly rule. The recombinant vectors carrying the AOX genes were checked by colony PCR and RE, and further confirmed by sequencing.</P><H3>PROTEIN EXPRESSION </H3><P>GFP reporter gene was expressed and the fluorescence level was assayed to test our gene expression system. </P><H3>PROTEIN ANALYSIS</H3><P>Coomassie blue staining was performed on SDS-PAGE to check the expression of AOX gene.</P><H3>ENZYME ACTIVITY ASSAY</H3><P>H<SUB>2</SUB>O<SUB>2</SUB> production assay was conducted as AOX enzyme activity assay, demonstrating that AOX catalyzed the oxidation of alcohol and generated a product of hydrogen peroxide.</P><H3>MODELING </H3><P>An electrochemical simulator was utilized to study the electrochemical properties of AOX enzyme. The experiment was performed by a collaboration with the biotech company, BIONIME CORP. who is a worldwide leader in blood glucose meter innovation.</P></DIV></DIV><DIV class="procontent3"><P>This year, we’ve done all the experiment to prove the function of AOX and demonstrated the electrochemical activity of AOX with alcohol. Now we’re going to develop IGEM BLOOD ALCOHOL METER (iMeter) by creating an alcohol test strip to apply in an electrochemical analyzer (i.e., blood glucose meter). 
    </P></DIV></DIV><DIV id="experiment"><DIV class="procontent"><P>There are two Alcohol oxidase (AOX) genes, AOX1 (<A href="https://www.ncbi.nlm.nih.gov/protein/CAY72092.1">CAY72092.1</A>) and AOX2 (<A href="https://www.ncbi.nlm.nih.gov/protein/CAY71377.1">CAY71377.1</A>), on chromosome 4 in <A href="http://bioinformatics.psb.ugent.be/genomes/view/Pichia-pastoris"><I>Pichia pastoris</I></A>. AOX was utilized to provide carbon source and energy for the methylotrophic yeast by metabolizing primary alcohol like methanol. We’d like to apply AOX to alcohol meter in an electrochemical analyzer by catalyzing the oxidation of alcohol and producing hydrogen peroxide (C<SUB>2</SUB>H<SUB>5</SUB>OH + O<SUB>2</SUB> → CH<SUB>3</SUB>CHO + H<SUB>2</SUB>O<SUB>2</SUB>). AOX1 and AOX2 genes were synthesized by IDT and cloned onto pSB1C3 as BioBrick parts. The function of the BioBrick device containing a constitutive promoter and RBS in front of the genes were <A href="https://2016.igem.org/Team:Mingdao/Proof">verified</A> by protein analysis and enzyme activity assay. The application of prototype was <A href="https://2016.igem.org/Team:Mingdao/Model">demonstrated</A> by a modeling experiment in an electrochemical simulator. </P><P>We always performed the gene cloning following the standard procedure as below. To make sure the DNA sequencing is correct on the vector, our instructor, Dr. Pei-Hong Chen, developed a rule followed by Mingdao iGEMers. We’re taught and learned the skills of gene cloning based on this strict procedure. </P></DIV><DIV class="procontent"><P>We always performed the gene cloning following the standard procedure as below. To make sure the DNA sequencing is correct on the vector, our instructor, Dr. Pei-Hong Chen, developed a rule followed by Mingdao iGEMers. We’re taught and learned the skills of gene cloning based on this strict procedure. </P><DIV class="procontent"><B>Standard cloning procedure @ Mingdao Biolab</B><UL><LI>Draw a plasmid map &amp; design primers</LI><LI>DNA synthesis by IDT or plasmid extraction of Biobrick</LI><LI>PCR with KOD-plus DNA polymerase</LI><LI>Restriction enzyme (RE) digestion</LI><LI>Ligation &amp; transformation</LI><LI>Colony PCR &amp; RE check</LI><LI>Sequencing &amp; align the seq. data with DNA sequences on your DNA map </LI><LI>Make -80°C stocks &amp; recover for the future study</LI></UL></DIV><DIV class="propic2"><DIV class="promd"><P>Figure 1: A DNA plasmid map was drawn by VectorNTI software.</P></DIV><DIV class="promd"><P>Figure 2: The standard gene cloning procedure performed in Mingdao Biolab</P></DIV></DIV></DIV><DIV class="procontent"><P> </P><P>The DNA sequences of AOX1 and AOX2 genes in <A href="http://bioinformatics.psb.ugent.be/genomes/view/Pichia-pastoris"><I>Pichia pastoris</I></A> were got from the genomics database created by <A href="http://bioinformatics.psb.ugent.be/">The Bioinformatics and Evolutionary Genomics Research Group</A> at Ghent University. The gene ID and sequences were double-checked with NCBI database (AOX1: <A href="https://www.ncbi.nlm.nih.gov/protein/CAY72092.1">CAY72092.1</A> ,AOX2: <A href="https://www.ncbi.nlm.nih.gov/protein/CAY71377.1">CAY71377.1)</A></P><P>The DNA sequences were synthesized by Integrated DNA Technologies (IDT) with gBlocks® Gene Fragments method. Before submitting sequences to IDT, we optimized the nucleotide sequences for E. coli by IDT <A href="https://sg.idtdna.com/CodonOpt">Codon Optimization Tool</A> and removed the possible secondary structure analyzed by online software (called <A href="https://sg.idtdna.com/site/order/gblockentry">Test Complexity</A>) provided by IDT to prevent the difficulty of DNA synthesis.</P><DIV class="propic2"><DIV class="promd2"><P>Figure 3: Codon optimization tool of IDT</P></DIV><DIV class="promd"><P>Figure 4: gBlocks Gene Fragments Entry and Test Complexity software of IDT.
</P></DIV></DIV><P> </P><P>In gene cloning experiments, we found in many clones that there’s always one mutation at the same position on the synthesized DNA sequence of AOX1 gene. We guessed it may be caused by the process of IDT DNA synthesis. Because the length of AOX gene is near 2kb, we thought it could be possible to make errors for bigger DNA. Fortunately, the sequence of AOX2 gene is correct and corresponding to the sequence on the NCBI genomics database. Therefore, AOX2 gene was used in the following experiment and studied in our project.</P><P>To see the final synthesized DNA sequences on pSB1C3, please go to the link and find data from <A href="https://static.igem.org/mediawiki/2016/7/77/T--Mingdao--phildnaAOX1.txt">AOX1</A> and <A href="https://static.igem.org/mediawiki/2016/c/c4/T--Mingdao--phildnaAOX2.txt">AOX2.</A></P></DIV><DIV class="procontent"><P>* Please check this <A href="https://2016.igem.org/Team:Mingdao/Parts">page</A> for our biobricks in more detail.</P><P>** PDF files were provided for every gene cloning record including the cloning strategy and DNA gel data of PCR, RE, Check, etc.</P><P>*** Sequence data were also provided in .txt format. We performed sequencing by BioBrick forward seq. primer (VF2) and reverse seq. primer (VR).</P><H2>AOX GENES </H2><UL><LI><A href="http://parts.igem.org/Part:BBa_K1991000"><B>BBa_K1991000</B></A>: AOX1/pSB1C3 (<A href="https://static.igem.org/mediawiki/2016/c/c2/T--Mingdao--philAOX1.pdf">PDF</A>; <A href="https://static.igem.org/mediawiki/2016/7/70/T--Mingdao--philAOX1vf2.txt">VF2</A>, <A href="https://static.igem.org/mediawiki/2016/c/c5/T--Mingdao--philAOX1vr.txt">VR</A>) </LI><LI><A href="http://parts.igem.org/Part:BBa_K1991001"><B>BBa_K1991001</B></A>: AOX2/pSB1C3 (<A href="https://static.igem.org/mediawiki/2016/0/04/T--Mingdao--philAOX2.pdf">PDF</A>; <A href="https://static.igem.org/mediawiki/2016/b/b5/T--Mingdao--philAOX2vf2.txt">VF2</A>, <A href="https://static.igem.org/mediawiki/2016/1/13/T--Mingdao--philAOX2vr.txt">VR</A>) </LI><LI>The synthesized AOX1 and AOX2 genes were amplified by PCR and digested by XbaI and PstI, followed by cloning onto pSB1C3 which was cut by XbaI and PstI. The parts have been confirmed by sequencing. </LI><LI><A href="http://parts.igem.org/Part:BBa_K1991002"><B>BBa_K1991002</B></A>: Pcons-RBS-AOX1/pSB1C3 (<A href="https://static.igem.org/mediawiki/2016/0/07/T--Mingdao--philPRAOX1.pdf">PDF</A>; <A href="https://static.igem.org/mediawiki/2016/f/f0/T--Mingdao--philPRAOX1vf2.txt">VF2</A>)</LI><LI><A href="http://parts.igem.org/Part:BBa_K1991003"><B>BBa_K1991003</B></A>: Pcons-RBS-AOX2/pSB1C3 (<A href="https://static.igem.org/mediawiki/2016/a/a5/T--Mingdao--philPRAOX2.pdf">PDF</A>; <A href="https://static.igem.org/mediawiki/2016/0/04/T--Mingdao--philPRAOX2vf2.txt">VF2</A>)</LI><LI>The DNA fragments containing a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>) and RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>) were amplified by PCR from <A href="http://parts.igem.org/Part:BBa_K1694035">BBa_K1694035</A> and digested by EcoRI and SpeI, followed by cloning onto <B>AOX1/pSB1C3</B> and <B>AOX2/pSB1C3</B> which were cut by EcoRI and XbaI. The parts have been confirmed by sequencing.</LI></UL><H2>IMPROVING AN EXISTING PART</H2><B><P>Existing part from NCTU-Formosa in 2015:</P><UL><LI><A href="http://parts.igem.org/Part:BBa_K1694002">BBa_K1694002</A>: Lpp-OmpA-NcoI/pSB1C3</LI></UL><P>Improved part by Mingdao in 2016: </P><UL><LI><A href="http://parts.igem.org/Part:BBa_K1991004">BBa_K1991004</A>: Lpp-OmpA-BamHI/pSB1C3&lt;</LI><LI><A href="http://parts.igem.org/Part:BBa_K1991007">BBa_K1991007</A>: Pcons-RBS-LO-BamHI/pSB1C3</LI></UL></B><P>Lpp and OmpA are outer membrane proteins of E. coli. Lpp-OmpA (LO) hybrid can direct heterologous proteins to bacterial cell surface. In 2015, NCTU-Formosa used it to display scFv (single chain fragment variable) antibodies on the surface of E. coli. They found that a fusion protein cannot be possible to be created under the standard BioBrick assembly rule, that is EcoRI(E)-XbaI(X)-GENE-SpeI(S)-PstI(P). The A part of EX-LO-SP and the B part of EX-scFV-SP, for example, are connected by cutting and ligation of SpeI plus PstI for the A part and XbaI plus PstI for the B part. The SCAR generated by XbaI/SpeI (ACTAGA) will form a stop codon just in front of the ATG start codon of the scFV protein of the B part. This situation has been officially mentioned by the <A href="http://openwetware.org/wiki/The_BioBricks_Foundation:BBFRFC15#Limitations_of_Standard_Assembly">BioBrick standard assembly</A>. </P><DIV class="propic3"><P>Figure 5: Limitation of cloning a fusion protein by standard biobrick assembly</P><P> </P><P> </P></DIV><P>Therefore, in 2015, NCTU-Formosa created a novel part (<A href="http://parts.igem.org/Part:BBa_K1694002">BBa_K1694002</A>) putting NcoI site between the LO part and SpeI site. However, when considering cloning, we found that an extra NcoI site is present on Cm resistance gene making it difficult be a vector for gene cloning.</P><P>In 2016, Mingdao improved the part by replacing NcoI site with BamHI site (<A href="http://parts.igem.org/Part:BBa_K1991004">BBa_K1991004</A>). Also, we’ve confirmed and <A href="https://2016.igem.org/Team:Mingdao/Proof">prove</A> the function of LO directing a fusion protein to the cell surface with enzyme activity in our project. </P><DIV class="propic2"><P>Figure 6: Alternative methods of cloning a fusion protein on Biobrick parts designed and created by NCTU-FORMOSA in 2015 and MINGDAO in 2016</P></DIV><P><A href="http://parts.igem.org/Part:BBa_K1991004">BBa_K1991004</A>: <B>Lpp-OmpA-BamHI/pSB1C3</B> (<A href="https://static.igem.org/mediawiki/2016/8/8a/T--Mingdao--philLB.pdf">PDF</A>; <A href="https://static.igem.org/mediawiki/2016/1/19/T--Mingdao--philLBvf2.txt">VF2</A>)
The DNA fragment of Lpp-OmpA was amplified by PCR using BBa_K1694035 as a template with a primer containing BamHI site and digested by EcoRI and SpeI, followed by cloning onto pSB1C3 which was cut by EcoRI and SpeI. The part has been confirmed by sequencing.</P><P><A href="http://parts.igem.org/Part:BBa_K1991007">BBa_K1991007</A>: <B>Pcons-RBS-LO-BamHI/pSB1C3</B> (<A href="https://static.igem.org/mediawiki/2016/9/98/T--Mingdao--philPLB.pdf">PDF</A>; <A href="https://static.igem.org/mediawiki/2016/5/51/T--Mingdao--philPLBvf2.txt">VF2</A>)
The DNA fragment of Pcons-RBS-Lpp-OmpA was amplified by PCR using <A href="http://parts.igem.org/Part:BBa_K1694035">BBa_K1694035</A> as a template with a primer containing BamHI site and digested by EcoRI and SpeI, followed by cloning onto pSB1C3 which was cut by EcoRI and SpeI. The part has been confirmed by sequencing.</P><B>* Contributions: we’ve added the info to the original part's main page (<A href="http://parts.igem.org/Part:BBa_K1694002">BBa_K1694002</A>). 
“An alternative version with BamHI site:
If you'd like to directly use this part as a vector/backbone for cloning a fusion protein, you should be noticed that an extra NcoI site is present on Cm resistance gene. BBa_K1991004 replaced NcoI site behind LO with BamHI site and provides an alternative choice for cloning a LO fusion protein”</B><H2>DISPLAYING ON THE BACTERIAL CELL SURFACE</H2><UL><LI><A href="http://parts.igem.org/Part:BBa_K1991005">BBa_K1991005</A>: <B>LO-AOX1-His/pSB1C3</B> (<A href="https://static.igem.org/mediawiki/2016/e/ee/T--Mingdao--philL1.pdf">PDF</A>; <A href="https://static.igem.org/mediawiki/2016/a/a0/T--Mingdao--philL1vf2.txt">VF2</A>, <A href="https://static.igem.org/mediawiki/2016/7/76/T--Mingdao--philL1vr.txt">VR</A>)</LI><LI><A href="http://parts.igem.org/Part:BBa_K1991006">BBa_K1991006</A>: <B>LO-AOX2-His/pSB1C3</B> (<A href="https://static.igem.org/mediawiki/2016/b/ba/T--Mingdao--philL2.pdf">PDF</A>; <A href="https://static.igem.org/mediawiki/2016/a/ae/T--Mingdao--philL2vf2.txt">VF2</A>, <A href="https://static.igem.org/mediawiki/2016/d/de/T--Mingdao--philL2vr.txt">VR</A>)</LI><LI>The AOX1-His and AOX2-His DNA fragments were amplified by PCR and digested by BamHI and PstI, followed by cloning onto LO-BamHI/pSB1C3 which was cut by BamHI and PstI. The parts have been confirmed by sequencing. </LI><LI><A href="http://parts.igem.org/Part:BBa_K1991008">BBa_K1991008</A>: <B>Pcons-RBS-LO-AOX1-His/pSB1C3</B> (<A href="https://static.igem.org/mediawiki/2016/5/5b/T--Mingdao--philPL1.pdf">PDF</A>; VF2, VR)</LI><LI><A href="http://parts.igem.org/Part:BBa_K1991009">BBa_K1991009</A>: <B>Pcons-RBS-LO-AOX2-His/pSB1C3</B> (<A href="https://static.igem.org/mediawiki/2016/3/3f/T--Mingdao--philPL2.pdf">PDF</A>; <A href="https://static.igem.org/mediawiki/2016/3/3b/T--Mingdao--philPL2vf2.txt">VF2</A>, <A href="https://static.igem.org/mediawiki/2016/0/0a/T--Mingdao--philPL2vr.txt">VR</A>)</LI><LI><A href="http://parts.igem.org/Part:BBa_K1991010">BBa_K1991010</A>: <B>Pcons-RBS-LO-GFP-His/pSB1C3</B> (<A href="https://static.igem.org/mediawiki/2016/2/2e/T--Mingdao--philPLG.pdf">PDF</A>; <A href="https://static.igem.org/mediawiki/2016/9/9f/T--Mingdao--philPLGvf2.txt">VF2</A>, <A href="https://static.igem.org/mediawiki/2016/c/cc/T--Mingdao--philPLGvr.txt">VR</A>)</LI><LI>The AOX1-His, AOX2-His, GFP-His DNA fragments were amplified by PCR and digested by BamHI and PstI, followed by cloning onto Pcons-RBS-LO-BamHI/pSB1C3 which was cut by BamHI and PstI. The parts have been confirmed by sequencing.</LI></UL><H2>COMMERCIAL GENE EXPRESSION SYSTEM</H2><P>In addition to cloning and gene expression using pSB1C3 vector and a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>), we also tested the gene expression and function of AOX genes on a commercial vector, pET-29b. The genes on pET-29b are driven by T7 promoter with lac operator. To express the gene, we transformed E. coli BL21 strain and performed IPTG induction. The gene expression level is slightly better than genes on pSB1C3 under a constitutive promoter. However, the enzyme activity and electrochemical properties are similar regardless of which vector was used.  </P></DIV><DIV class="procontent"><B>LO FUSION PROTEIN VECTORS:</B><UL><LI><A href="http://parts.igem.org/Part:BBa_K1991004">BBa_K1991004</A>: Lpp-OmpA-BamHI/pSB1C3</LI><LI><A href="http://parts.igem.org/Part:BBa_K1991007">BBa_K1991007</A>: Pcons-RBS-LO-BamHI/pSB1C3</LI></UL><B>AOX GENE EXPRESSION VECTORS:</B><UL><LI><A href="http://parts.igem.org/Part:BBa_K1991000">BBa_K1991000</A>: AOX1/pSB1C3 </LI><LI><A href="http://parts.igem.org/Part:BBa_K1991001">BBa_K1991001</A>: AOX2/pSB1C3 </LI><LI><A href="http://parts.igem.org/Part:BBa_K1991002">BBa_K1991002</A>: Pcons-RBS-AOX1/pSB1C3 </LI><LI><A href="http://parts.igem.org/Part:BBa_K1991003">BBa_K1991003</A>: Pcons-RBS-AOX2/pSB1C3</LI></UL><B>AOX CELL SURFACE DISPLAYING VECTORS:</B><UL><LI><A href="http://parts.igem.org/Part:BBa_K1991005">BBa_K1991005</A>: LO-AOX1-His/pSB1C3 </LI><LI><A href="http://parts.igem.org/Part:BBa_K1991006">BBa_K1991006</A>: LO-AOX2-His/pSB1C3 </LI><LI><A href="http://parts.igem.org/Part:BBa_K1991008">BBa_K1991008</A>: Pcons-RBS-LO-AOX1-His/pSB1C3</LI><LI><A href="http://parts.igem.org/Part:BBa_K1991009">BBa_K1991009</A>: Pcons-RBS-LO-AOX2-His/pSB1C3</LI></UL><B>GFP CELL SURFACE DISPLAYING VECTOR:</B><UL><LI><A href="http://parts.igem.org/Part:BBa_K1991010">BBa_K1991010</A></LI>: Pcons-RBS-LO-GFP-His/pSB1C3</UL></DIV><DIV class="procontent"><P>AOX1 gene cloned to the vector was mutated at 349 bp (GCC → ACC) changing an amino acid of Alanine to Threonine. AOX2 gene sequence was confirmed by sequencing with the forward primer (VF2) and the reverse primer (VR). Therefore, we used the AOX2 gene in the following experiments to test the gene expression, enzyme activity and electrochemical properties with the substrate of alcohol.</P><DIV class="propic4"><DIV class="promd"><P>Figure 7: The alignment of the sequencing data and AOX1 gene in a reverse view. One T mutation is present at 1926 bp (i.e., 349 bp from the ATG start site in a forward view)</P></DIV></DIV></DIV><DIV id="proof"><DIV class="procontent"><P>The AOX gene was successfully synthesized by IDT and cloned onto pSB1C3 with a fusion protein of Lpp-OmpA (LO) which is driven by a constitutive promoter and RBS. In the following experiments to prove the concept, we tested the gene expression system by a reporter gene (GFP) and analyzed AOX gene expression by SDS-PAGE and Coomassie Blue staining. Furthermore, we examined AOX enzyme activity by H<SUB>2</SUB>O<SUB>2</SUB> assay, a product generated in the process of the oxidation of alcohol catalyzed by AOX. Finally, we designed a prototype to demonstrate the application of AOX enzyme on the alcohol meter in an electrochemical simulator. </P></DIV><DIV class="procontent"><P>The AOX gene were cloned onto pSB1C3 driven by a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>) and RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>) and fused with bacterial outer membrane proteins, Lpp-OmpA (LO) to display protein on the cell surface of E. coli. To test whether the gene expression system works, we cloned a reporter gene (GFP) in the same context (i.e., Pcons-RBS-LO-GFP/pSB1C3). And the gene expression level was compared to Pcons-RBS-GFP/pSB1C3 (<A href="http://parts.igem.org/Part:BBa_K1694035">BBa_K1694035</A>) got from NCTU-Formosa, which has the same promoter and RBS but without LO. The clones of E. coli DH5α were cultured in LB media supplemented with 34 μg/ml chloramphenicol at 37°C overnight. Because overnight cultured LB medium has a background level of fluorescence, the bacterial GFP was measured in the PBS buffer (<A href="https://www.ncbi.nlm.nih.gov/pubmed/11118595">Enzyme Microb Technol. 2001</A>). As Figure 1 showed, the GFP expression was extremely high in E. coli. LO-GFP fusion protein expression was observed at low but significant level compared to wild-type E. coli or E. coli expressing LO outer membrane proteins.</P><DIV class="propic5"><P>Figure 1: Gene expression analysis. GFP gene expression was read at Ex/Em = 488/528 nm in BioTek Microplate Spectrophotometer. Lane 1: wild-type E. coli as a mock control; Lane 2: GFP expression in E. coli [<A href="http://parts.igem.org/Part:BBa_K1694035">BBa_K1694035</A>]; Lane 3: LO outer membrane protein expression [<A href="http://parts.igem.org/Part:BBa_K1991007">BBa_K1991007</A>] in E. coli; Lane 4: LO-GFP fusion protein expression [<A href="http://parts.igem.org/Part:BBa_K1991010">BBa_K1991010</A>] in E. coli. </P></DIV></DIV><DIV class="procontent"><P>To analyze the AOX gene expression, we run on a SDS-PAGE gel and observed by Coomassie Blue staining. The overnight-cultured E. coli were centrifuged and lysed with <A href="http://openwetware.org/wiki/Endy:E._coli_Western_Blot">Lysis Buffer</A> (12.5 mM Tris pH 6.8, 4% SDS). The resulting lysates were subjected to SDS-PAGE with a 10% polyacrylamide gel. The gel was stained with 0.25% Coomassie Brilliant Blue R250 for 2 hours and destained until the protein bands were clear. As the data showed in Figure 2, LO protein was expressed at around the estimated molecular weight of 17 kDa, LO-AOX fusion protein at 91 kDa and AOX protein at 79 kDa. The protein expression level was consistent with the data of GFP in Figure 1, demonstrating the low gene expression level of a LO fusion protein. However, so far we cannot confirm whether LO fusion protein is able to direct AOX or GFP proteins displayed on the cell surface of E. coli. We’re planning to do a subcellular fractionation to separate the outer membrane proteins for analysis in the future.</P><DIV class="propic5"><P>Figure 2: AOX protein analysis. SDS-PAG and Coomassie Blue staining were used to observe protein expression level. Lane 1: wild-type E. coli as a mock control; Lane 2: LO outer membrane protein [<A href="http://parts.igem.org/Part:BBa_K1991007">BBa_K1991007</A>] (17 kDa) expression in E. coli; Lane 3: LO-AOX fusion protein [<A href="http://parts.igem.org/Part:BBa_K1991009">BBa_K1991009</A>] (91 kDa) expression in E. coli.; Lane 4: AOX protein [<A href="http://parts.igem.org/Part:BBa_K1991003">BBa_K1991003</A>] (79 kDa) expression in E. coli.</P></DIV></DIV><DIV class="procontent"><P>After AOX gene expression was confirmed, we want to know its enzyme activity. AOX catalyzes the oxidation of alcohol in the following chemical reaction C<SUB>2</SUB>H<SUB>5</SUB>OH + O<SUB>2</SUB> → CH<SUB>3</SUB>CHO + H<SUB>2</SUB>O<SUB>2</SUB> and generates hydrogen peroxide. We’d like to measure the production of hydrogen peroxide to demonstrate AOX enzyme activity. Fluorimetric Hydrogen Peroxide Assay Kit of <A href="http://www.sigmaaldrich.com/catalog/product/sigma/mak165?lang=en&amp;region=TW">Sigma-Aldrich</A> was used for the study. In the kit, the red peroxidase substrate is designed to react with hydrogen peroxide to generate red fluorescence signal that can be detected at Ex/Em = 540/590 nm in the microplate reader. First, a serial dilution of hydrogen peroxide was tested to see if the kit works in our lab. As Figure 3 showed, the three-fold serial dilutions of hydrogen peroxide from 10 to 0.014 μM were linearly correlated to red fluorescence intensity, demonstrating H<SUB>2</SUB>O<SUB>2</SUB> concentration can be measured using this kit in our lab.</P><DIV class="propic5"><P>Figure 3: H<SUB>2</SUB>O<SUB>2</SUB> assay. The concentrations (μM) of hydrogen peroxide were measured at red fluorescence intensity unit (RFU) [Ex/Em = 540/590 nm]. The serial dilutions of hydrogen peroxide were tested and correlated to red fluorescence intensity. </P></DIV><P> </P><P> </P><P>Next, we examined the activity of AOX enzyme by measuring H<SUB>2</SUB>O<SUB>2</SUB> production from the oxidation of ethanol. We prepared 1ml of overnight cultured E. coli displaying LO-AOX enzyme. The bacteria were centrifuged and resolved in Assay Buffer provided by the Sigma-Aldrich kit. Then, the resulting lysates were mixed with the increasing concentrations of ethanol from 0%, 0.025%, 0.05%, 0.075%, 0.10%, 0.15% to 0.2% for 3 minutes at room temperature. The tested alcohol concentrations were at the range from 0.03% (the alcohol law limit for safety driving) to 0.2% (people may lose consciousness at this level). The following procedure was according to the manufacture’s instruction. The data were read out at Ex/Em = 540/590 nm in the microplate reader (BioTek Microplate Spectrophotometer). As Figure 4 showed, the intensities of AOX activity were significantly and linearly correlated to the concentration of ethanol in a dose-dependent manner. The results indicated that LO-AOX enzyme we produced has a functional enzyme activity and reacted with alcohol concentration dose-dependently, implying we can use this assay to measure the unknown sample of alcohol. Although we cannot make sure whether AOX enzyme was displayed on the cell surface of E. coli or not in this moment. In addition, it is noted that according to the manufacture’s instruction, the substrate (ethanol), bacteria (E. coli) or LO-expressed E. coli has a basal level of fluorescence intensity. To count the measures clearly, you should subtract the background intensity.  </P><DIV class="propic5"><P>Figure 4: AOX enzyme activity assay. The various concentrations of ethanol were reacted with E. coli displaying AOX enzymes. The assay was performed using Fluorimetric Hydrogen Peroxide Assay Kit (Sigma-Aldrich). The readout was presented in arbitrary unit to show the enhanced AOX activity caused by alcohol concentration in a dose-dependent manner.</P></DIV><P> </P><P> </P><P>Taken together, we cloned the AOX genes, tested our expression system and performed protein analysis followed by the enzyme activity assay. Now, we’re ready to examine the chemical properties of AOX enzyme in an electrochemical experiments and apply it to the analyzer. </P></DIV></DIV><DIV id="demo"><DIV class="ptitle">*If you want to learn more, please go to our <A href="https://2016.igem.org/Team:Mingdao/Model">modeling</A> page.</DIV><DIV class="procontent"><P>Glucose oxidase (GOX) and alcohol oxidase (AOX) are oxidoreductase enzymes which catalyze the oxidation of glucose and alcohol, respectively. When applied to an electrochemical analyzer, the electrode covered with a given enzyme will generate current signal, which intensity is corresponding to the concentration of substrates. Before this application, the enzyme’s reduction potential need to be determined and set up a program to convert the current intensity to a digital number revealing the real concentration of a sample measured. For example, in the blood glucose meter, the reduction potential of GOX is -97 ± 3 mV (<A href="https://www.ncbi.nlm.nih.gov/pubmed/25007396">Anal Chem. 2014</A>) and set on the analyzer to facilitate the redox reaction between GOX enzyme and the substrate, glucose. Unfortunately, the reduction potential of AOX is not well studied. Therefore, we have to determine the reduction potential of AOX prior to applying on the blood alcohol meter we want to create.</P><DIV class="propic"><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P></DIV></DIV><DIV class="procontent"><P><A href="https://en.wikipedia.org/wiki/Cyclic_voltammetry">Cyclic voltammetry</A> is a potentiodynamic electrochemical measurement, in which the working electrode potential is ramped linearly versus time. The ramping potential move forward to the maximum and backward to the minimum, the cycle continues until the end of the experiment. In a CV experiment, electrochemical properties of a chemical molecule in the solution can be studied.</P><P>To analyze the electrochemical properties of AOX, a cyclic voltammetry experiment was performed to determine the reduction potential of AOX. 50µl of the K<SUB>2</SUB>HPO<SUB>4</SUB> solution containing the AOX enzyme displayed on the surface of E. coli was put onto the test strip. Then, 20µl of 10% alcohol was added and mixed in the solution. As shown in Figure 2, the result indicated that a current peak (the current values at 2.0E-05~2.2E-05) was induced around the voltage of -1200 mV, implying that the AOX enzyme did undergo redox reaction. But the current was not much significantly changed when responding to the different concentrations of alcohol. </P><P>To improve the redox reaction, we added 200 µl of potassium ferricyanide (K<SUB>3</SUB>[Fe(CN)<SUB>6</SUB>]) as electron mediators to facilitate the transfer of electrons to electrodes. When mixed with various concentration of ethanol, the electric currents rose significantly (i.e., the higher current intensity between 2.6E-05~2.9E-05) and dose-dependently upon the increasing concentrations of alcohol around the voltage of -1200 mV (Figure 2).</P><DIV class="propic3"><DIV class="promd"><P>Figure 3: The current strength of AOX enzyme with different concentration of ethanol in an amperometry experiment.</P></DIV></DIV></DIV><DIV class="procontent"><P>Amperometry is a way to detect the change of electric current along with the time (IT) in a given voltage. Followed the CV experiment, we’d like to know the current variation dependent on the different concentration of ethanol with the electric potential set at -1200 mV. As Figure 3 showed, the increased currents were measured along with the three different concentrations of ethanol (i.e., 0.385%, 0.741% and 1.071%), indicating the enhanced AOX enzyme activity was responding to the increasing concentration of ethanol.</P><DIV class="propic3"><DIV class="promd"><P>Figure 2: The electrochemical properties of AOX enzyme with different concentration of ethanol in a cyclic voltammetry experiment.</P></DIV></DIV></DIV><DIV class="procontent"><P>Based on the data above, we plotted a concentration calibration curve (Figure 4) with a mathematical equation, which could be applied to estimate unknown concentration of alcohol samples using current strength.</P><DIV class="propic3"><P>Figure 4: The concentration calibration curve based on the current strength reaction with ethanol.</P></DIV></DIV><DIV class="procontent"><DIV class="propic4"><P>Figure 5: The prototype of our new product – IGEM BLOOD ALCOHOL METER (iMeter)</P></DIV><P>Based on the mathematical formula obtained from the concentration calibration curve, we are ready to create a test strip with AOX to measure alcohol concentration and apply it to be a blood alcohol meter. We called it IGEM BLOOD ALCOHOL METER (iMeter).</P><P>This work was done by the collaboration with BIONIME Corp. who is an innovator working on Blood Glucose Meter design and develop. We thank the executive vice president, Dr. Wiley Chung (Shie-Shiun Jung), went to Mingdao High School to help us develop the blood alcohol meter. From him, we not only learned the electrochemical chemistry, but also completed the amazing experiments in our lab.</P></DIV></DIV><DIV id="result"><DIV class="procontent"><P>This year, we got the idea from blood glucose meter and designed a brand-new IGEM BLOOD ALCOHOL METER (iMeter) and completed all the experiments to prove that our system works well. We’ve constructed 11 biobricks, analyzed the protein expression and enzyme activity, as well as do modeling in an electrochemical simulator. Although this is just a proof-of-concept study, through the collaboration with BIONIME Corp., we believe the prototype will soon be applied to a real blood alcohol meter for the public in the near future.</P><P>Eleven biobricks were completed. A novel part of Alcohol Oxidase (AOX) [BBa_K1991001] from Pichia Pastoris was synthesized by IDT, which codon was optimized for gene expression in E. coli. We also improved an existing BioBrick Part [BBa_K1694002], where NcoI site was created between the part and SpeI site for cloning a Lpp-OmpA (LO) fusion protein. However, we found an extra NcoI site on the chloramphenicol resistance gene. And we improved it by replacing NcoI site with BamHI site [BBa_K1991004]. As a result, we used it to make another new part of fusion protein, LO-AOX [BBa_K1991006], which would be expressed and displayed on E. coli cell surface. Finally, a constitutive promoter and RBS were used to drive the gene expression.
</P></DIV><DIV class="procontent"><P>To test our gene expression system, we measured GFP level of E. coli carrying GFP and LO-GFP genes. LO-GFP fusion protein [<A href="http://parts.igem.org/Part:BBa_K1991010">BBa_K1991010</A>] was detected at a low intensity level but significantly expressed compared to wild-type E. coli or E. coli expression LO protein alone. To analyze the LO-AOX protein [<A href="http://parts.igem.org/Part:BBa_K1991009">BBa_K1991009</A>] expression, we run on a SDS-PGE gel and performed Coomassie Blue staining. The data showed the expressed proteins were observed at the estimated size (i.e., 91 kDa for LO-AOX)</P></DIV><DIV class="procontent"><P> </P><P> </P><P> </P><P>Enzyme activity of LO-AOX [<A href="http://parts.igem.org/Part:BBa_K1991009">BBa_K1991009</A>] was measured as the function of hydrogen peroxide using ethanol as substrate. The data showed that E. coli displaying AOX enzymes is able to react with increasing concentrations of ethanol in a dose-dependent manner. The intensities (readouts) are linearly correlated to the concentrations of ethanol, indicating that our system could be used to measure the concentration of an unknown sample of alcohol. </P></DIV><DIV class="procontent"><P> </P><P> </P><P> </P><P>To test whether our prototype of blood alcohol meter does work, we did modeling using an electrochemical simulator. The reduction potential of AOX with the aid of potassium ferricyanide as electron mediators was determine (around -1200 mV in our case) in the cyclic voltammetry (CV) experiment. Then, the current intensity of AOX redox reaction to various concentrations of ethanol was measured in the amperometry experiment. Finally, according to the results, we plotted a concentration calibration curve which can be utilized to calculate the unknown concentration of an alcohol sample. </P></DIV><DIV class="procontent"><P>This year, we made the impossible possible. We proved the feasibility of the blood alcohol meter prototype in the lab and in a simulator. We believe it could be applied to measure the blood alcohol in the near future, and provides an alternative method for the field sobriety test  in addition to the breathalyzer. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>