<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IISER-Mohali_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IISER-Mohali/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="preloader"><DIV class="loaded"> </DIV></DIV><HEADER id="main_menu" class="header navbar-fixed-top"><DIV class="main_menu_bg"><DIV class="container"><DIV class="row"><DIV class="nave_menu"><NAV class="navbar navbar-default" id="navmenu"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2018.igem.org/Team:IISER-Mohali">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">Project</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/History">Background</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Experiments">Protocol</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Model">Model</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">Lab</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">Community</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Applied_Design">Product Design</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Human_Practices">Integrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Entrepreneurship">Entrepreneurship</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">People</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></HEADER><SECTION id="bloginner" class="bloginner sections margin-top-120"><DIV class="container"><DIV class="row"><DIV class="col-sm-12"><DIV class="head_title text-center"><H1>Results</H1></DIV><DIV class="main_bloginner_area"><DIV class="main_bloginner_content"><DIV class="single_bloginner s_blogIner_3 margin-top-60"><H3>PCR Amplification of GSTM3</H3><DIV class="row"><DIV align="justify"><DIV class="col-sm-12 col-md-6"><P>PCR was performed in order to amplify the insert, GSTM3. It was done using DNA polymerase and the primers designed under the given PCR conditions. The agarose gel figure on the right shows the PCR amplicons run at different annealing temperatures. This was done in order to optimize the amplification condition to obtain good insert concentration. In the figure, Lane 1 represents the marker (1kb DNA Ladder), Lane 2, 3, and 4 represent the PCR products annealed at 55, 57, and 59 °celsius respectively.The results indicate that PCR performed with annealing temperature of 57°celsius gave the best results, thereby we fix the annealing temperature as 57°celsius for further PCR amplifications. The required size of the insert was obtained after the PCR amplification, whose size was found to be around 800 bp.</P></DIV></DIV></DIV><H3>Cloning of GSTM3 in Yeast expression Backbone</H3><DIV class="row"><DIV align="justify"><DIV class="col-sm-12 col-md-6"><P>In order to express the recombinant plasmid containing the insert (GSTM3) in <I>Saccharomyces cerevisiae</I>, we cloned the PCR amplified insert after digestion into the required vector(pRS313-TEF). The figure on the left shows the agarose gel picture of the double digested recombinant vector. In the figure, Lane 1 represents the run of the digested recombinant plasmid and lane 2 represents the marker (1kb DNA Ladder). From lane 1, we can see the band-size of the insert fragment at around 780 bp and the digested plasmid backbone at around 7.0 kb. Thereby the double digestion using EcoR1 and Xba1 validated the successful cloning of the insert GSTM3 in pRS313-TEF.</P></DIV></DIV></DIV></DIV><DIV class="single_bloginner s_blogIner_3 margin-top-60"><H3>Western Blot to detect the cat GST expression in yeast</H3><DIV class="row"><DIV class="col-sm-12 col-md-6"><DIV align="justify"><P>In order to check the protein expression, we performed western blot of the constitutively expressed protein GSTM3 tagged with HA tag and FLAG tag on its N- terminal and C- terminal respectively. The plasmid (pRS313-TEF) with GSTM3 and the empty plasmid (pRS313-TEF) was transformed in wild type yeast, and the 1.0 OD<SUB>600</SUB> cells were harvested and western blotting was done by using anti-HA mouse antibody to probe the HA tag. A specific band at 25 kDa was observed which corresponds to GSTM3 protein. This confirms that the cat GSTM3 was expressed in <I>Saccharomyces cerevisiae</I>.</P></DIV></DIV></DIV></DIV><H3 style="padding-top: 20px">Proposed Pathway for Felinine Synthesis in <I>Saccharomyces cerevisiae</I></H3><DIV class="container"><DIV class="row"><DIV align="justify"><P>We know that GSH and IPP conjugates to give 3-MBG in cat (Felis catus) and both these substrates are already present in the cytoplasm of Saccharomyces cerevisiae. Through bioinformatics analysis and literature mining, we found that the cat homologs of γ-GGT and cauxin are also being produced in Saccharomyces cerevisiae <A href="http://www.biochemj.org/cgi/pmidlookup?view=long&amp;pmid=11672438">(Mehdi, K., Thierie, J., &amp; Penninckx, M. J. (2001))</A>. Although the cauxin and γ-GGT homologs are found in Saccharomyces cerevisiae, they are localized in the vacuoles and perform a similar function. Therefore, we hypothesize that they will be available to act on the respective substrates and produce felinine only in vacuole. Fortunately, to aid with the same, there also exists a Glutathione conjugated pump called YCF1 found in the vacuolar membrane of Saccharomyces cerevisiae. This pump plays a crucial role in detoxification and transports glutathione conjugates that are not immediately reduced in cytosol to vacuole. Therefore the putative GST needed for the conjugation must be introduced in the cytoplasm, which can be further metabolized through the above mentioned pathway after entering the vacuole through the YCF1 pump. These properties of Saccharomyces cerevisiae coupled with the fact that their cells use pheromone based intracellular signal transduction pathway for mating made us choose Saccharomyces cerevisiae as our model organism for the production of cat pheromone.</P></DIV><DIV class="single_bloginner s_blogIner_3 margin-top-60"><H3>HPLC analysis validating the production of Felinine</H3><P>In order to perform the HPLC analysis, we evaluated the growth curve of the recombinant yeast expressing GSTM3. The growth curve of the recombinant yeast is shown below. We observed the culture reaching saturation around the third day. Thereby cell lysis was performed on the third day and sample preparation was done.</P><DIV align="justify"><UL style="list-style-type:disc"><LI><P>Figure 1 shows the HPLC plot of the standard which contains felinine for the given solvent system (acetonitrile:trifluroacetic acid:water (10:0.1:89.9 v/v)). A prominent peak was observed at 11.473 minutes.</P></LI><LI><P>Figure 2 shows the HPLC plot of the lysed wild type yeast with the same solvent system. This system serves as a negative control. </P></LI><LI><P>Figure 3 shows the HPLC plot of the lysed recombinant yeast with the GSTM3 expression with the same solvent system. It can be observed that it has peaks that matches with both the felinine standard and lysed wild type systems. Eventhough there exists a slight variation in the peaks of the Felinine standard and lysed recombinant yeast, this can  be due to the hinderance in the flow rate. Thereby we can match the 11.473 peak from Fig 1 to the peak at 10.759. In order to validate this further, we performed a spiking experiment.</P></LI><LI><P>Figure 4 shows the HPLC plot of the recombinant yeast with the GSTM3 expression spiked with the standard for the same solvent system. It can be observed that the peaks at 10.047 and 10.759 minutes from the figure 3 have raised. This can be used to validate the production of Felinine by the GSTM3 containing recombinant yeast.</P></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER style="background-color: lightgray;"><DIV class="container"><DIV class="main_footer"><DIV class="row"><DIV class="col-sm-12"><H4 style="text-align: center; margin: 2%;">Our Patrons</H4></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>