<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IISER-Mohali_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IISER-Mohali/Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="preloader"><DIV class="loaded"> </DIV></DIV><HEADER id="main_menu" class="header navbar-fixed-top"><DIV class="main_menu_bg"><DIV class="container"><DIV class="row"><DIV class="nave_menu"><NAV class="navbar navbar-default" id="navmenu"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2018.igem.org/Team:IISER-Mohali">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">Project</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/History">Background</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Experiments">Protocol</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Model">Model</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">Lab</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">Community</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Applied_Design">Product Design</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Human_Practices">Integrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Entrepreneurship">Entrepreneurship</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">People</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></HEADER><DIV class="container"><DIV class="head_title text-center"><H1>Software</H1></DIV><DIV class="main_bloginner_area"><DIV class="main_bloginner_content"><DIV class="single_bloginner s_blogIner_3 margin-top-60"><H3>Introduction to software</H3><DIV class="container"><DIV class="row"><DIV class="col-sm-12 col-md-12"><P>Softwares have changed our lives today. Synthetic biology is no different. Having the ability to use compuational tools can greatly increase both speed and efficiency in syntheic biology research. Keeping in mind the same thing, team IISER-Mohali has developed following two software tools.</P><P>
                                                EFFUSE - EFflux of Fearomone for optimal USE - This software built using Python uses Molecular Dynamics simulations to determine the rate of diffusion of small molecules out of the cell that are being produced in any compartment of the cell,given the complex environment of the cell. This approach is extrapolated including cell-cell interactions to determine the overall diffusion from an assembly of cells in a culture. </P><P>RATTEL - RAT Timidity tester from Evidence based Learning - Software based on Active Brownian Dynamics designed to study the spatio-temporal dynamics of murines exposed to Fearomone based on experimental parameters and field data. This is used to predict the best location for device deployment, given possible points of entry of murines, environmental factors, and the configuration of placed stored grains such that the aversion response seen is maximized. It also predicts the localization of multiple devices so as to direct motion of rats for effective trapping.
                                            </P><CENTER><H5>Click <A href="https://github.com/igemsoftware2018/IISER-Mohali/releases/tag/1.0.0">here </A>to download iGEM judging release (v1.0.0) of RATTEL.</H5></CENTER></DIV></DIV></DIV></DIV><DIV class="single_bloginner s_blogIner_3 margin-top-60"><DIV class="container"><H3>RATTEL-Novel Software to model Rat Behavior for synthetic biology application:</H3><DIV class="row"><P>
                                            A novel computational venture in order to widen the application of synthetic biology to complex organisms.Our project to induce fear response in rats begged us to ponder on how rats would respond to our FearOmone.Rats are complex biological species, they show fear, hunger, curiosity and have memory.Given such a setting we needed to understand how the rat would respond to external inputs.Hence, we started our journey to model the behavior of rats.</P></DIV><DIV class="row"><CENTER><CENTER><FIGCAPTION>Strategy for model development</FIGCAPTION></CENTER></CENTER></DIV><DIV class="row" style="padding-top: 20px"><P>Borrowing from modelling techniques used to model collective dynamics of microorganisms like bacteria, we started out by imagining our rats to be active brownian micro-organisms inside a closed reflective boundary.</P></DIV><DIV class="row"><H5 style="padding-top: 15px;">Basic assumptions involved in RATTEL are as follows:</H5><UL style="list-style-type:disc"><LI><P>Rats are active brownian micro-swimmers: They have an internal source of energy i.e. they can drive themselves.</P></LI><LI><P>They do not interact “talk” to each other: Since there are not many rats at the same time, we assume time ensemble is same as spatial ensemble. This allows us to save computation power.</P></LI><LI><P>They enter the arena from the boundaries and then are contained in the arena: This initial condition is reasonable as most of the entry points in real life are located toward the boundry of arena.</P></LI><LI><P>Neuman's boundry condition: Since arena is assumed to be open for gaseous exchange, this condition is justified.</P></LI><LI><P>Various parameters for potentials used. (To be justified by experimental validation)</P></LI></UL></DIV></DIV></DIV><DIV class="single_bloginner s_blogIner_3 margin-top-60"><DIV class="container"><H3>Building RATTEL using bottom-to-top approach</H3><DIV class="row"><H5>Complexity Layer - 01</H5><DIV class="col-md-6 col-sm-12"><P>We added a central static repulsive potential in order to introduce the fear rats are known to show towards open areas.</P></DIV></DIV><DIV class="row" style="padding-top: 20px"><H5>Complexity Layer - 02</H5><DIV class="col-md-6 col-sm-12"><P>Then adding a layer of interaction we use gaussian wells to mimic the attraction that rats feel towards food.</P><P>Note: The reason why we use gaussians is that they are smooth,symmetric, easy to work with and die out in a given range.</P></DIV></DIV><DIV class="row" style="padding-top: 20px"><H5>Complexity Layer - 03</H5><DIV class="col-md-6 col-sm-12"><P>We add the repulsive interaction between the rats and a field created due to a diffusing gas, emerging from a source.</P></DIV></DIV><DIV class="row" style="padding-top: 20px"><P>Now, we check if the behavior of the rats is close enough to be assumed to mimic the natural behavior of rats.</P><P style="padding-top: 20px">Upon animal testing we will be able to confirm the validity of our model.</P></DIV></DIV></DIV><DIV class="single_bloginner s_blogIner_3 margin-top-60"><DIV class="container"><H3>EFFUSE - EFflux of Fearomone for optimal USE</H3><DIV class="row"><P>Aim of EFFUSE is to quantify the diffusion of small molecules from the cell to give synthetic biologists a platform to build further experiments based on diffusion of molecules from the cell. EFFUSE consists of mainly two modules as discussed below.</P><CENTER><H5>Click <A href="https://github.com/DevangLiya/IISER-Mohali">here </A>try EFFUSE on GitHub</H5></CENTER><OL><LI><H5>Using PyMOL for molecular visualization:</H5><P>For every molecule in the database, the smiles string was generated. Based on the smiles string, the pdb was generated. A PyMOL script was then developed to find  the radius of Gyration of each molecule from the pdb file obtained.</P><P>Following registry of small molecules was compiled through literature mining.</P><CENTER><TABLE class="tableizer-table"><THEAD><TR class="tableizer-firstrow"><TH>S No</TH><TH>Type of Cell</TH><TH>Type of the Chemical</TH><TH>Name of the Chemical</TH><TH>Radius of Gyration (nm)</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>Pseudomonas aeruginosa</TD><TD>AHL</TD><TD>acyl-homoserine lactone</TD><TD>1.98</TD></TR><TR><TD>2</TD><TD>Vibrio cholerae</TD><TD>NA</TD><TD>cholera autoinducer-1 (  (S)-3-hydroxytridecan-4-one )</TD><TD>4.72</TD></TR><TR><TD>3</TD><TD>Escherichia coli</TD><TD>Autoinducer</TD><TD>AI-2/LsrR</TD><TD> </TD></TR><TR><TD>4</TD><TD>Salmonella typhimurium</TD><TD>Autoinducer</TD><TD>AI-2/LsrR</TD><TD> </TD></TR><TR><TD>5</TD><TD>Vibrio fischeri</TD><TD>AHL</TD><TD>N-(3-oxohexanoyl)-homoserine lactone</TD><TD>3.51</TD></TR><TR><TD>6</TD><TD>Pantoea stewartii</TD><TD>AHL</TD><TD>3-oxo-C6-HSL</TD><TD> </TD></TR><TR><TD>7</TD><TD>Agrobacterium tumefaciens</TD><TD>AHL</TD><TD>3-oxo-C8-HSL</TD><TD>4.22</TD></TR><TR><TD>8</TD><TD>Nitrobacter winogradskyi</TD><TD>AHL</TD><TD>7, 8-trans-N-(decanoyl) homoserine lactone (C10:1-HSL)</TD><TD> </TD></TR><TR><TD>9</TD><TD>Burkholderia mallei</TD><TD>AHL</TD><TD>C8-HSL</TD><TD>4.3</TD></TR><TR><TD>10</TD><TD>Burkholderia mallei</TD><TD>AHL</TD><TD>3-hydroxy-C8-HSL</TD><TD>4.23</TD></TR><TR><TD>11</TD><TD>Pseudomonas aeruginosa</TD><TD>AHL</TD><TD>C4-HSL</TD><TD>2.84</TD></TR><TR><TD>12</TD><TD>Pseudomonas aeruginosa</TD><TD>AHL</TD><TD>3-oxo-C12-HSL</TD><TD>5.76</TD></TR><TR><TD>13</TD><TD>Rhodopseudomonas palustris</TD><TD>AHL</TD><TD>p-coumaroyl-HSL (pC-HSL)</TD><TD>4.09</TD></TR><TR><TD>14</TD><TD>Nitrosomonas europaea</TD><TD>AHL</TD><TD>C6-HSL</TD><TD>3.56</TD></TR><TR><TD>15</TD><TD>Nitrosomonas europaea</TD><TD>AHL</TD><TD>C8-HSL</TD><TD>4.3</TD></TR><TR><TD>16</TD><TD>Nitrosomonas europaea</TD><TD>AHL</TD><TD>C10-HSL</TD><TD>5.13</TD></TR></TBODY></TABLE></CENTER></LI><H5>Developing software tool to simulate dynamics of molecules inside the cell:</H5><UL><LI><H5>Complexity layer - 01</H5><P>We started out by studying brownian motion of a small molecule in an isotropic 3 dimensional box.</P></LI><LI><H5>Complexity layer - 02</H5><P>We added compartments inside the 3 dimensional box, possessing various material properties like viscosity.These were made to mimic the various cell organelles of the cell.</P></LI><LI><H5>Complexity layer - 03</H5><P>We ran simulations of molecules of varying diffusion constants based on their radius of gyration obtained from PyMOL.</P></LI></UL></OL></DIV></DIV></DIV><DIV class="single_bloginner s_blogIner_3 margin-top-60"><DIV class="container"><H3>Future Scope</H3><DIV class="row"><DIV class="col-md-12 col-sm-12"><OL><LI>Add feedback loops of varying degrees of complexity to model Quorum sensing which has tremendous potential in biological research.</LI><LI>Add genetic parameters to control the gene expression and rate of production of molecules.</LI><LI>Simulate the diffusion outside the cell into the media to increase the accuracy of the model.</LI><LI>Extrapolate the single cell to cell colonies.</LI><LI>Include various kinds of cells.</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV><FOOTER style="background-color: lightgray;"><DIV class="container"><DIV class="main_footer"><DIV class="row"><DIV class="col-sm-12"><H4 style="text-align: center; margin: 2%;">Our Patrons</H4></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>