<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IISER-Mohali_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IISER-Mohali/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="preloader"><DIV class="loaded"> </DIV></DIV><HEADER id="main_menu" class="header navbar-fixed-top"><DIV class="main_menu_bg"><DIV class="container"><DIV class="row"><DIV class="nave_menu"><NAV class="navbar navbar-default" id="navmenu"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2018.igem.org/Team:IISER-Mohali">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">Project</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/History">Background</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Experiments">Protocol</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Model">Model</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">Lab</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Composite_Part">Composite Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">Community</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Applied_Design">Product Design</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Human_Practices">Integrated Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Entrepreneurship">Entrepreneurship</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true">People</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:IISER-Mohali/Collaborations">Collaborations</A></LI></DIV></DIV></NAV></DIV></DIV></DIV></DIV></HEADER><SECTION id="bloginner" class="bloginner sections margin-top-120"><DIV class="container"><DIV class="head_title text-center"><H1>Experiments</H1></DIV><DIV class="main_bloginner_area"><DIV class="main_bloginner_content"><DIV align="justify"><P>FearOmone seeks to exploit the innate fear of murines for cats. Our challenge is to genetically engineer <I> Saccharomyces cerevisiae</I> to produce a cat pheromone based synthetic deterrent and prepare a device capable of diffusing this cat pheromone to areas surrounding grain storage facilities, thereby keeping murines away. Our first aim is to transform our host system, <I>Saccharomyces cerevisiae</I>, with the necessary synthetic gene circuits which will result in recombineered <I> Saccharomyces cerevisiae</I> that mimic the cat nephron pathway for producing felinine. Next, we will conduct controlled experiments in the form of murine behavior assays to test the effectiveness of our synthetically derived felinine as a rat/mouse deterrent. </P></DIV><DIV class="single_bloginner s_blogIner_3 margin-top-60"><H3>Bioinformatic prediction of the putative GST that conjugates GSH which IPP</H3><DIV class="container"><DIV class="row"><DIV class="col-sm-12 col-md-7"><DIV align="justify"><P>A bioinformatics analysis was done to find out the putative <I> Saccharomyces cerevisiae</I> homologs of <I>Felis catus</I> enzymes present in its renal network, namely Glutathione-S-transferase (GST), γ-Glutamyl transferase (γ-GGT) and cauxin. The BLAST analysis revealed the existence of <I>Felis catus</I> γ-GGT and cauxin gene homologs in <I>Saccharomyces cerevisiae</I> as γ-GGT and APE1 respectively. Since these homologs were already identified in <I>Saccharomyces cerevisiae</I>, only the putative GST homolog required identification and functional characterization. We started exploring the putative GST family enzyme required for the conjugation of GSH and IPP in <I>Felis catus</I>.</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-sm-12 col-md-6"><H5>Selecting putative GST involved in GSH-IPP conjugation:</H5><DIV align="justify"><P>The criteria used for selecting putative GST involved in GSH-IPP conjugation in <I>Felis catus</I> are as follows:
Essentiality  and  expression showing profoundly in the renal network.<A href="https://doi.org/10.1111/j.1399-0012.2009.00985.x">(Clin Transplant 2009: 23: 490–498)</A>
Tissue-level expression profiles of the GSTs in comparison with the tissue content of the involved metabolites.(<A href="https://www.sciencedirect.com/science/article/pii/S1570023217316574?via%3Dihub">Futsuta et al, Journal of Chromatography B 1072 (2018) 94–99</A>)
Since conjugation can happen in an extracellular environment, and evidence exists for GSTs, GSH and IPP transport outside the cell, the putative signaling sequence may be used as an additional selecting criterion.<A href="https://www.ncbi.nlm.nih.gov/pubmed/10746931">(Toxological Sciences 54, 52–59 (2000))</A>(<A href="https://linkinghub.elsevier.com/retrieve/pii/S1096717617303749">Metabolic Engineering 47 (2018) 60–72</A>) Based on the above criteria, GSTM3 turned out to be the best candidate among all the cat GSTs. Therefore, GSTM3 was taken as the putative GST to test the GSH-IPP conjugation and all further experiments were carried out using GSTM3.<A href="https://www.sciencedirect.com/science/article/pii/S0006291X15300425?via%3Dihub">( Biochemical and Biophysical Research Communications 463 (2015) 650-655)</A></P></DIV></DIV></DIV><DIV style="padding-top: 15px"><DIV align="justify"><P>As previously stated, felinine's precursor is present in cat blood as 3-methylbutanol-glutathione (3-MBG) and is formed via a glutathione S-conjugation reaction between glutathione (GSH) and isopentenyl pyrophosphate (IPP) <A href="http://www.jbc.org/content/277/1/114.full">(J. Biol. Chem., 277 (2002), pp. 114-119)</A>. In general, glutathione S-conjugates such as 3-MBG are converted to cysteinylglycine S-conjugates (3-MBCG) by γ-glutamyl transferase in the kidney and the cysteinylglycine S-conjugates are hydrolyzed by cauxin to give rise to felinine, the precursor to cat pheromone.</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="single_bloginner s_blogIner_3 margin-top-60"><H3>Experimental approach for reconstructing the felinine pathway in Yeast</H3><OL><LI><H5>Custom synthesis and codon optimization of cat GSTM3 gene</H5><P>GSTM3 gene from <I>Catus felis</I> was custom synthesized and codon optimized for expression in <I>Saccharomyces cerevisiae</I>. The size of GSTM3 was found to be 780 bp.</P></LI><LI><H5>Primer Designing and PCR Amplification of GSTM3 gene from cat</H5><DIV align="justify"><P>Primers were designed for the amplification of the GSTM3 gene. EcoR1 and Xba1 sites were introduced in the forward and reverse primers respectively for cloning the gene in the yeast expression vector pRS313-TEF. The primers were also designed in such a way to introduce an HA ( Human influenza hemagglutinin )-tag and FLAG -tag on its N- terminus and C- terminus respectively. The final insert size was found to be around 800 bp.</P></DIV></LI><H5>PCR amplification of the GSTM3 gene and its parameters:</H5><CENTER><P> Parameters for PCR Amplification</P></CENTER><LI><H5>Genetic Circuit for cloning into Yeast expression vector</H5><P>The circuit can be divided into 3 parts. The first one is a constitutive promoter – TEF, secondly, the gene GSTM3 with an HA tag on its N terminus and a FLAG tag on its C terminus, and thirdly, the cyc terminator. </P><CENTER><P> Genetic Circuit for cloning into Yeast expression vector</P></CENTER></LI><LI><H5>Cloning of GSTM3 gene from cat into the Yeast expression vector pRS313-TEF</H5><DIV class="single_bloginner s_blogIner_3 margin-top-60"><DIV align="justify"><P>The insert band obtained via gel purification after PCR amplification was subjected to digestion with the restriction enzymes EcoR1 and Xba1. Similarly, the plasmid backbone pRS313-TEF was also digested with the same restriction enzymes. Finally the digested products were ligated at 16 °C for 16 hours (hrs) and were used for transformation in<I> E.coli</I>. 
Plasmids were isolated, and the screening of positive clones was achieved through restriction double digestion. The positive clones were sent for sequencing.</P></DIV></DIV><CENTER><P>Schematic showing cloning of GSTM3 gene from cat into the Yeast expression vector</P></CENTER><DIV class="single_bloginner s_blogIner_3 margin-top-60"><H3>Western Blot for checking the expression of cat GSTM3 in Yeast</H3><P>Plasmids containing GSTM3 were transformed in <I>Saccharomyces cerevisiae</I> under the TEF promoter. In order to check the protein expression, we performed western blot using anti-HA mouse antibody. The size of the protein was found to be 25 kDa which corresponds to the expected cat GSTM3 protein.</P><CENTER><P> Pictorial representaion of Western blot for the expression of cat GSTM3 in Yeast</P></CENTER></DIV><DIV class="single_bloginner s_blogIner_3 margin-top-60"><H3>Validation of Felinine expression in Yeast: HPLC Analysis</H3><DIV align="justify"><P>HPLC of the cell lysate was performed in order to verify the production of Felinine using appropriate solvent systems. The solvent system was chosen to be acetonitrile:trifluoroacetic acid:water (10:0.1:89.9 v/v). The cell culture was grown for 3 days at 30 °C as the culture reached saturation nearly on the third day. Cell lysis was performed on the third day and sample preparation was done as per the protocol stated below. Peaks obtained from the the cell lysate were analyzed for the production of felinine.</P></DIV><CENTER><P>A schemtaic representing the HPLC technique for the production of Felinine in Yeast</P></CENTER><DIV align="justify"><P style="padding-top: 20px">
Extraction of Felinine was carried out as described earlier <A href="https://www.sciencedirect.com/science/article/pii/S1074552106003097?via%3Dihub">(Chemistry &amp; Biology 13, 1071–1079, October 2006)</A>  with some modifications. Essentially, <I> Saccharomyces cerevisiae</I> cells were grown in 100 mL SD media supplemented with appropriate amino acids and grown at 30 °C with shaking (250 rpm). After two days, the cells were harvested and washed with deionized water and kept at −20 °C. To the frozen pellet, 3 mL of Dimethyl sulphoxide (DMSO) was added, then vortexed for 1 min and incubated at 55 °C in the water bath for 1h. 1g 0.50–0.75mm glass beads were added, and the cells were broken using a glass bead beater. The cells were centrifuged to remove cell debris. Acetone was added to the pellet, vortexed, and centrifuged. This process was repeated till the pellet became colorless. The acetone and DMSO fractions were mixed with an equal amount of Hexane, which was collected after separation of the two layers. The hexane layer was washed with distilled water and then with brine solution twice. The solvent was evaporated under a rotary evaporator (till dry) in dim light and was dissolved in 1 mL hexane for analysis by high performance liquid chromatography (HPLC). HPLC separation and quantification was performed on a Waters System using C18 − 5μm intersil ODS-P, 250×4.6mm column (LCGC) using the solvent acetonitrile:trifluroacetic acid:water (10:0.1:89.9 v/v) with flow rate 0.3 mL/min at 32 °C. 
</P></DIV></DIV></LI></OL></DIV></DIV></SECTION><FOOTER style="background-color: lightgray;"><DIV class="container"><DIV class="main_footer"><DIV class="row"><DIV class="col-sm-12"><H4 style="text-align: center; margin: 2%;">Our Patrons</H4></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>