Team:TU-Eindhoven/Scaffold Design
iGEM TU Eindhoven
Home
Team
Members
Advisors
Collaborations
Sponsors
Attributions
Project
Description
Background
Scaffold Design
Heterodimer
Tetramer
Experiments
Heterodimer
Tetramer
Timeline
Protocols
Modeling
Rosetta
Model
Results
PPI
Model
Results
Results
Heterodimer
Introduction
Functionality
Orthogonality
Conclusion
Tetramer
NanoBiT curve
Proof of Concept
Interlab Study
Biobricks
Parts
Basic Part
Achievements
Future
Policy & Practices
Outreach
Safe By Design
Applications
T-Cell Therapy
Transport Bacteria
Co-Enzym Regen.
Safety
Lab
Project
✉
Heterodimer Design
Description
Background
Scaffold Design
Heterodimer
Tetramer
Experiments
Heterodimer
Tetramer
Timeline
Protocols
Heterodimer
The first new scaffold proteins that were created consist of a mutated monomer and a wildtype monomer of the T14-3-3 protein, these form a heterodimer together. The heterodimer enables the assembly of two different protein-CT52 complexes, instead of the two equal protein-CT52 complexes that are able to assemble on the homodimeric T14-3-3.
To create these heterodimers new mutations had to be found for the T14-3-3 monomer, as well as for the CT52 protein that binds to that monomer to ensure an orthogonal binding. Orthogonality is introduced to ensure minimal interaction with the natural processes inside the cell of that organism, so that the mutated scaffold monomer only binds to the complementary mutated CT52 protein.
The Rosetta software package was used to assess which mutation besides the T14-3-3-E19R mutation would most likely work.2 This package is a molecular structure predictor that uses several algorithms for computational modelling and analysis of protein structures.
Three sets of mutations were found that would most likely create an orthogonal binding (Figure 1).  Each mutated monomer is combined with a wildtype monomer, including the monomer with the previously found E19R mutation (Figure 2) to form four different heterodimers (Table1).
Figure 1: A schematic representation of the 14-3-3 heterodimers we created and tested during out project.
Figure 2: A schematic representation of the T14-3-3 heterodimer with E19R mutation.3
Split Luciferase
The ability of the heterodimer to assemble two different proteins can be used for the activation of the split NanoLuc Luciferase system. Split NanoLuc are two non-identical fragments which are inactive on their own, but when they dimerize they form an active NanoLuc luciferase. The activated luciferase system emits light when a luminogenic substrate, furimazine, is present.
These two non-identical fragments are called SmallBiT (SsNL) and LargeBiT (LsNL). We created SsNL-CT52-I947F, SsNL-CT52-I947H, SsNL-CT52-S953K, SsNL-CT52-K943D, LsNL-CT52-I947F, LsNL-CT52-I947H, LsNL-CT52-S953K, LsNL-CT52-K943D, SsNL-CT52(wildtype), and LsNL-CT52(wildtype). These fragments linked to a CT52 protein can bind to the four different variants of the heterodimeric scaffold under influence of fusicoccin, resulting in activation of NanoLuc and leading to the emission of bioluminescence light of 460 nm.
CRISPR/Cas9
Another application of a heterodimer is the regulation of the CRISPR/Cas9 system. Currently, CRISPR/Cas9 is a hot topic in synthetic biology because it offers the possibility to edit the genome at very specific positions. In a recent study Zetsche, Volz, & Zhang1 successfully split Cas9 (sCas9) into two non-equal fragments, which are inactive on their own and are activated when they are reassembled. These two non-equal fragments are called NsCas9 and CsCas9. In their study they coupled the fragments to FK506 binding domain (FKBP) and FKBP rapamycin binding (FRB) domains. They showed that the fragments could be reassembled under influence of rapamycin binding so they created a regulation mechanism for the activation of Cas9. Yet the drawback of their findings is that the used small molecule Rapamycin has a high affinity to its binding partners2 and as a result disassembly of the split Cas9 fragments is almost impossible. So basically Zetsche, Volz, and Zhang (2015) created an on-switch for Cas9. With the heterodimeric scaffold protein we created, it is possible to bring two sCas9 together under influence of fusicoccin.
Figure 3: (Top) A schematical overview of the CRISPR/Cas9 structure. (Bottom) The split-Cas9 fragments.
NsCas9 was linked to CT52-K943D and CsCas9 was linked to CT52(wildtype). When the concentration of fusicoccin increases, both CT52-sCas9s will bind to the T14-3-3-E19R – T14-3-3(wildtype) scaffold. This results in the dimerization of sCas9 (Figure 3), followed by activation of the Cas9 protein. Both the T14-3-3 and the split Cas9 fragments can be translocated into the cell nucleus using a nuclear localisation signal2,3. When the fusicoccin concentration decreases, fusicoccin will dissociate from the scaffold3 and both CT52-sCas9 will release from the scaffold. It is possible that the dimerized Cas9 will disassemble back into two inactive sCas9, creating the off-switch of the Cas9 enzyme. Therefore, using the heterodimeric T14-3-3 scaffold we created, it is not only possible to turn Cas9 on, but also to turn it off again, increasing the control over genome editing and transcription.
pET28a Vector
The pET28a vector has a single cloning site, so the pET28a vector can be used for the expression of one gene and thus the production of one type of protein. The pET28a vector is a plasmid that is often used, it contains Kanamycin resistance and can provide the target protein with a N- and C-terminal His-tag. The pET expression system is one of the most used vector systems for cloning and expression of proteins in E. coli. This is due to the high selectivity and activity of the T7 RNA polymerase, which is utilized in the pET system. The pET vectors contain a T7 promoter and a T7 termination sequence. To start expression of the target protein, a source of T7 RNA polymerase has to be added to the host. This gives the pET system another benefit, namely the ability to maintain target genes transcriptionally silent by inducing pET vectors in hosts, who contain a T7 RNA polymerase gene that requires the precence of a small molucule for their transcription. To provide the host with the T7 RNA polymerase, IPTG is used, IPTG is a molecular mimic of the allolactose molecule. The E.coli strain that is used, contains a lac promoter that initiates the transcription of the T7 RNA polymerase gene when activated. IPTG activates this promoter resulting in the production of T7 RNA polymerase, which leads to expression of the target genes in the pET vector.
References
[1] Zetsche, B., Volz, S. E., & Zhang, F. (2015). A split-Cas9 architecture for inducible genome editing and transcription modulation. Nature biotechnology, 33(2), 139-142. doi:10.1038/nbt.3149.
[2] Banaszynski, L. A., Liu, C. W., & Wandless, T. J. (2005). Characterization of the FKBP.rapamycin.FRB ternary complex.. Jo urnal of the American Chemical Society, 127(13), 4715-4721. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/15796538
[3] Skwarczynska, M., Molzan, M., & Ottmann, C. (2013). Activation of NF-κB signalling by Fusicoccin-induced dimerization.. Proc Natl Acad Sci U S A., 110(5), 377-386. doi:10.1073/pnas.1212990110
back to top
