<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU-Eindhoven_Project_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU-Eindhoven/Project Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="Top"><DIV class="Title"><DIV class="Website_Header"> iGEM TU Eindhoven </DIV></DIV></DIV><DIV id="Balk"><NAV id="MenuBar"><UL><LI><A id="First" href="https://2016.igem.org/Team:TU-Eindhoven">Homepage</A></LI><LI><A id="cursor">Team</A></LI><LI id="First_Li"><A href="https://2016.igem.org/Team:TU-Eindhoven/Team_members">Members</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Advisors">Advisors</A></LI><LI><A href="">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Sponsors">Sponsors</A></LI><LI><A href="">Attributions</A></LI></UL><LI><A id="cursor"> Project </A></LI><LI id="First_Li"><A href="https://2016.igem.org/Team:TU-Eindhoven/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Project_Background">Background</A></LI><LI><A href="">Design</A></LI><LI><A href="">Read-out</A></LI><LI><A href="">Future</A></LI><LI><A id="cursor">Modeling</A></LI><LI id="First_Li"><A href="https://2016.igem.org/Team:TU-Eindhoven/Modeling_Approach">Modeling Approach</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Modeling_Results">Modeling Results</A></LI><LI><A href="">Under Development</A></LI><LI><A href="">Under Development</A></LI><LI><A id="cursor">Results</A></LI><LI id="First_Li"><A href="">Under Development</A></LI><LI><A href="">Under Development</A></LI><LI><A href="">Achievements</A></LI><LI><A id="cursor">Policy</A></LI><LI id="First_Li"><A href="">Under Development</A></LI><LI><A href="">Under Development</A></LI><LI><A href="">Under Development</A></LI><LI><A id="Last" href="">Biobricks</A></LI><LI id="First_Li"><A href="">Under Development</A></LI><LI><A href="">Under Development</A></LI><LI><A href="">Under Development</A></LI><LI><A href="">Under Development</A></LI><LI><A href="">Under Development</A></LI></NAV><A href="#" class="back-to-top"> Back to Top </A></DIV><DIV class="Side_Menu_Nav"><DIV class="Side_Menu_Header"> Design</DIV><DIV class="Side_Menu_List_Header"><A id="1">Heterodimer</A></DIV><DIV class="Side_Menu_List_Header"><A id="2">Tetramer
</A></DIV><DIV class="Side_Menu_List_Header"><A id="3">Vectors
</A></DIV><DIV class="Side_Menu_Item"><A id="4">pET28a
</A></DIV><DIV class="Side_Menu_Item"><A id="5">pETDuet-1</A></DIV><DIV class="Side_Menu_Item"><A id="6">pcDNA3.1 (+)</A></DIV><DIV class="Side_Menu_List_Header"><A id="Ref">  References </A></DIV><DIV class="Fold_Side_Menu">◢</DIV></DIV><DIV class="Content_Page"><DIV class="Page_Header"> Design  </DIV><DIV id="A1" class="Content_Header">Heterodimer</DIV><DIV id="Column_Full" class="Column"><P class="Inline">
The first scaffold protein we aim to create consists of two monomers of the T14-3-3 protein and is an orthogonal heterodimer. We want to create a heterodimer to enable the assembly of two different protein-CT52 complexes. 
</P><A class="Toggle"> Orthogonality </A><P class="Toggled"> 
Orthogonality within synthetic biology, refers to a biological system whose structure is so different in comparison to the natural occurring one, resulting in minimal interaction between those systems. 
</P><P class="Inline">
is introduced to ensure minimal interaction with the natural processes in that organism. Thus to create an orthogonal heterodimer we need two different monomers that have an orthogonal binding with a complementary modified CT52. A first set of mutations has already been found by Skwarczynska, Molzan, and Ottmann.<SUP class="Reference_Link">3</SUP> The mutations are: T14-3-3 E19R and CT52 K943D. Secondly we used computational design to asses which other mutations would most likely work. Three sets of mutations were found: (1)T14-3-3 S71L &amp; I72V and CT52 I947F, (2)T14-3-3 S71L and CT52 I947H and (3)T14-3-3 W237R and CT52S 953K. All monomer variants are shown in 
</P><A class="Toggle"> Table 1.</A><P class="Inline">
In order to increase the chances of successfully creating a new functional heterodimeric T14-3-3 scaffold, all monomer variants were combined and 8 different heterodimers were created (figure 1). 

		</P></DIV><DIV class="Content_Row"><DIV class="Column"><P>
The ability of the heterodimer to bring together two different proteins can be used for regulating the CRISPR-Cas9 system.  
At the moment CRISPR/Cas9 is a hot topic in synthetic biology. CRISPR/Cas9 offers the possibility to edit the genome at very specific positions. In a recent study Zetsche, Volz, &amp; Zhang<SUP class="Reference_Link">1</SUP> successfully split Cas9 (sCas9) into two inactive fragments. In their study they showed that those fragments could be reassembled under influence of rapamycin binding. Thus they created a regulation mechanism for the activation of Cas9.  Yet the drawback of their findings is that they used rapamycin. Rapamycin is a molecule that binds with high affinity to binding partners<SUP class="Reference_Link">2</SUP> and as a result disassembly of the split Cas9 is almost impossible. So basically Zetsche, Volz, and Zhang (2015) created an on-switch. With our scaffold it is possible to not only turn Cas9 on, but also off again, thus increasing control over genome editing and transcription.</P></DIV><DIV class="Column"><DIV id="Heterodimers_Caption" class="Caption"> Figure 1: Geen idee wat hier nog moet? </DIV></DIV></DIV><DIV class="Content_Row"><DIV class="Column"><VIDEO id="Crispr_Video" width="500px" height="375px" controls=""><DIV id="Crispr_Video_Caption" class="Caption"> Figure 2: video about crispr cas with our scaffold </DIV></VIDEO></DIV><DIV class="Column"><P>
With our heterodimeric scaffold protein it is possible to bring two sCas9 together under the influence of Fusicoccin. The two inactive fragments of the Cas9 are each attached to a different form of CT52 (CT52-sCas9). When the concentration of Fusicoccin increases, both CT52-sCas9s will bind to the T14-3-3 scaffold, each to their own monomer. This results in a higher local concentration of both Cas9 fragments leading to dimerization of sCas9 (figure 2), which activates the Cas9 protein. When the Fusicoccin concentration decreases, Fusicoccin will dissociate from the scaffold<SUP class="Reference_Link">3</SUP> and both  CT52-sCas9 will release from the scaffold and the dimerized Cas9 will disassemble back into two inactive sCas9.</P></DIV></DIV><DIV id="A2" class="Content_Header"> Tetramer</DIV><DIV id="Column_Full" class="Column"><P>
The second scaffold protein we seek to create is a tetrameric version of T14-3-3. A tetramer enables the assembly of four CT52 coupled proteins instead of just two. Firstly we want to create the simplest form, a tetrameric T14-3-3 consisting of two wild type T14-3-3 dimers that are linked to each other with a flexible GGS10 linker. GGS10 is a ten times repeating sequence of glycine-glycine-serine amino acids. Secondly we seek to create a tetramer with four different binding pockets, enabling the assembly of four different proteins. For this we chose the most viable mutations and applied each to a monomer, so we get: T14-3-3(1) … , T14-3-3(2) … , T14-3-3(3) … and T14-3-3(4)
</P></DIV><DIV class="Content_Row"><DIV class="Column"><P>
The ability of the tetramer to bring together four of the same proteins can be used to create a kill switch. 
Caspase 9 is a protein which is part of the caspase family, they are proteases that have been associated with apoptosis. Activated Caspase 9  cuts downstream caspases initiating a caspase cascade.<SUP class="Reference_Link">4</SUP> Caspase 9 is essential for the induction of apoptosis, utilizing this feature we want to create a kill switch by linking Caspase 9 to CT52. It is not quite understood how Caspase 9 is activated, but there is evidence that procaspases activate each other.<SUP class="Reference_Link">5</SUP> That is why our tetramer is very suitable to create a kill switch, by letting four CT52-Caspase 9  proteins assemble instead of two the local concentration of Caspase 9  is higher, likely resulting in more Caspase 9 activity.  
So when a high concentration of Fusicoccin is present, the CT52 with linked Caspase 9 will assemble on the tetramer scaffold. This will result in a high local concentration of Caspase 9 leading to active Caspase 9 (figure 3), which induces apoptosis in the cell. </P></DIV><DIV class="Column"><VIDEO id="Tetramer_Video" width="500px" height="375px" controls=""><DIV id="Tetramer_Video_Caption" class="Caption"> Figure 2: video about caspase 9 killswitch with our scaffold </DIV></VIDEO></DIV></DIV><DIV id="A3" class="Content_Header"> Vectors </DIV><DIV id="Column_Full" class="Column"><P>
The pET expression system is one of the most used vector system for cloning and expression of proteins in E. coli. This is due to the high selectivity and activity of the T7 RNA polymerase which the pET system utilizes. The pET vectors contain T7 promoters, to start expression of the target protein a source of T7 RNA polymerase has to be added to the host. This gives the pET system another benefit, namely the ability to maintain target genes transcriptionally silent by inducing pET vectors in hosts who do not contain the T7 RNA polymerase gene. So to start expression of the target gene a source of T7 RNA polymerase is needed.  
</P><P>
To provide the host with the T7 RNA polymerase, IPTG is used, IPTG is a molecular mimic of allolactose. The E.coli strain used contains a lac promoter that initiates the transcription of the T7 RNA polymerase gene when activated. IPTG activates this promoter resulting in the production of T7 RNA polymerase, which leads to expression of the target genes on the pET vector.<SUP class="Reference_Link">6</SUP><SUP>,</SUP><SUP class="Reference_Link">7</SUP></P></DIV><DIV id="A4" class="Sub_Header"> pET28a</DIV><DIV id="Column_Full" class="Column"><P>
The pET28a vector has a single cloning site, so the pET28a vector can be used for the expression of one gene and thus the production of one type of protein. The pET28a vector is a plasmid that is often used, it contains Kanamycin resistance and can provide the target protein with a N- and C-terminal His-tag.  
</P></DIV><DIV id="A5" class="Sub_Header"> pETDuet-1</DIV><DIV id="Column_Full" class="Column"><P>
The pETDuet-1 vector has two cloning sites, each with its own promotor, enabling the possibility to express two genes at once. Because of the two cloning sites the pETDuet-1 vector is able to produce two types of proteins in equal amounts. pETDuet-1 contains ampicillin resistance and has a terminator after cloning site 2.
</P></DIV><DIV id="A6" class="Sub_Header"> pcDNA3.1 (+)</DIV><DIV id="Column_Full" class="Column"><P>
pcDNA3.1 is a 5.4k base long vector and is used for protein expression in mammalian cells. It contains a CMV promotor which originates from the human Cytomegalovirus, the CMV promotor is a strong constitutive promotor which allows continual transcription of the inserted gene. We use the pcDNA3.1(+) vector in which the + refers to the orientation of the primer, + means forward and – means reverse. Also pcDNA3.1 contains Ampicillin resistance.<SUP class="Reference_Link">8</SUP></P></DIV><DIV id="A_Ref" class="Content_Header"> References </DIV><DIV class="Reference_List"><UL><LI>[1] Zetsche, B., Volz, S. E., &amp; Zhang, F. (2015). A split-Cas9 architecture for inducible genome editing and transcription modulation. Nature biotechnology, 33(2), 139-142. doi:10.1038/nbt.3149.</LI><LI>[2] Banaszynski, L. A., Liu, C. W., &amp; Wandless, T. J. (2005). Characterization of the FKBP.rapamycin.FRB ternary complex.. Jo urnal of the American Chemical Society, 127(13), 4715-4721. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/15796538</LI><LI>[3] Skwarczynska, M., Molzan, M., &amp; Ottmann, C. (2013). Activation of NF-κB signalling by Fusicoccin-induced dimerization.. Proc Natl Acad Sci U S A., 110(5), 377-386. doi:10.1073/pnas.1212990110</LI><LI>
[4] Kuida, K. (2000). Caspase-9.The international journal of biochemistry &amp; cell biology, 32(2), 121-124
</LI><LI>
[5] Srinivasula, S. M., Ahmad, M., Fernandes-Alnemri, T., &amp; Alnemri, E. S. (1998). Autoactivation of procaspase-9 by Apaf-1-mediated oligomerization. Molecular cell, 1(7), 949-957.
</LI><LI>
[6] pET Expression Systems - Details &amp; Specifications [Manual]. (n.d.). Retrieved from http://www.genomics.agilent.com/article.jsp?pageId=472&amp;_requestid=1238872
</LI><LI>
[7] pcDNA™3.1(+) pcDNA™3.1(–) User Manual [Manual]. (2010, November 10). Retrieved from https://tools.thermofisher.com/content/sfs/manuals/pcdna3_1_man.pdf  
</LI></UL></DIV><DIV class="Bottom_Navigation"><A href="https://2016.igem.org/Team:TU-Eindhoven/Project_Background"><DIV class="Previous_Page"><DIV class="Arrow_Left">➢</DIV> Background
		</DIV></A><A href="https://2016.igem.org/Team:TU-Eindhoven/Read-out"><DIV class="Next_Page">
			Read-out<DIV class="Arrow_Right">➢</DIV></DIV></A></DIV></DIV><DIV class="footer"><DIV id="Footer_Header"><DIV class="Left_Footer">Contact </DIV></DIV><DIV id="Row_Width" class="Content_Row"><DIV id="Column_30_Footer" class="Column">
				 Email: iGEM@tue.nl
			</DIV><DIV id="Column_20_Footer" class="Column">
				Phone: +3140-2475559 
			</DIV><DIV id="Column_50_Footer" class="Column">
			    Address: 5612 AJ Eindhoven, The Netherlands
			</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>