<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU-Eindhoven_Project_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU-Eindhoven/Project Background</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE> iGEM TU Eindhoven </TITLE></P><DIV class="Website_Top"><DIV class="The_Menu"><UL class="Menu_List"><LI><A class="active_1" href="https://2016.igem.org/Team:TU-Eindhoven">Home</A></LI><LI class="hover"><A class="active_2" href="https://2016.igem.org/Team:TU-Eindhoven/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Team/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Advisors">Advisors</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Sponsors">Sponsors</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Attributions">Attributions</A></LI></UL><LI class="hover"><A href="https://2016.igem.org/Team:TU-Eindhoven/Description" class="active_3">Project</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Project_Background">Background</A></LI><LI class="hover2"><A style="cursor:default">Scaffold Design</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Scaffold_Design">Heterodimer</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Scaffold_Design_Tetramer">Tetramer</A></LI><LI class="hover2"><A style="cursor:default">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Read-out">Heterodimer</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Read-out/Tetramer">Tetramer</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Timeline">Timeline</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Protocols">Protocols</A></LI><LI class="hover"><A href="https://2016.igem.org/Team:TU-Eindhoven/Modeling/Rosetta" class="active_4">Modeling</A></LI><LI class="hover2"><A style="cursor:default">Rosetta</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Modeling/Rosetta">Model</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Modeling/Rosetta_Results">Results</A></LI><LI class="hover2"><A style="cursor:default">PPI</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Modeling/PPI">Model</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Modeling/PPI_Results">Results</A></LI><LI class="hover"><A class="active_6" href="https://2016.igem.org/Team:TU-Eindhoven/Results/Introduction">Results</A></LI><LI class="hover2"><A style="cursor:default">Heterodimer</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Results/Introduction">Introduction</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Results/Functionality">Functionality</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Results/Orthogonality">Orthogonality</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Results/Conclusion">Conclusion</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Results/Tetramer">Tetramer </A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Results/NanoBit">NanoBiT curve</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Proof">Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Interlab">Interlab Study </A></LI><LI class="hover2"><A style="cursor:default">Biobricks</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Basic_Part">Basic Part</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Achievements"> Achievements</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Future">Future</A></LI><LI class="hover"><A class="active_5" href="https://2016.igem.org/Team:TU-Eindhoven/Human_Practices">Policy &amp; Practices</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/HP/Silver">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/HP/Gold">Safe By Design</A></LI><LI class="hover2"><A style="cursor:default">Applications</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/T-Cell">T-Cell Therapy</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Alzheimer">Transport Bacteria</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Industrial">Co-Enzym Regen.</A></LI><LI class="hover2"><A style="cursor:default">Safety</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Safety/Lab">Lab</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Safety/Project">Project</A></LI><DIV class="email"><A title="mailto:igem@tue.nl" id="emaillink" href="mailto:igem@tue.nl"><SPAN class="size">✉</SPAN></A></DIV></DIV></DIV><DIV class="Website"><DIV class="Header_Panel"><DIV class="Panel"><DIV class="Panel_Title">
				Background
			</DIV></DIV></DIV><DIV class="wrap"><DIV class="Sub_Menu"><UL class="Side_Menu"><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Description">Description</A></LI><LI><A class="active_Side_Menu" href="https://2016.igem.org/Team:TU-Eindhoven/Project_Background">Background</A></LI><LI class="Sub_Sub_Menu"><A>Scaffold Design</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Scaffold_Design">Heterodimer</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Scaffold_Design_Tetramer">Tetramer</A></LI></UL><LI class="Sub_Sub_Menu"><A>Experiments</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Read-out">Heterodimer</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Read-out/Tetramer">Tetramer</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Timeline">Timeline</A></LI><LI><A href="https://2016.igem.org/Team:TU-Eindhoven/Protocols">Protocols</A></LI></DIV><DIV class="The_Content"><DIV class="P_Header_Top">The Scaffold</DIV><DIV class="Column"><P>
Every mammalian cell contains about a billion proteins, about 10% of these are involved in signal transduction. With this high number of proteins, it is impressive that signaling proteins can accurately find their partners to process signaling information.<SUP class="source">1</SUP></P><P>
Over 15 years ago, scientists discovered the first protein scaffolds: proteins that regulate the assembly of other proteins. Scaffold proteins have numerous functions in the cell, one of which is mediating signaling transduction or other networks<SUP class="source">1</SUP>. These scaffold proteins bind components of a signaling pathway, to help localize signaling to a specific part of the cell or to increase the efficiency of the signaling cascade.<SUP class="source">2</SUP></P><P>
Scaffold proteins are often used in synthetic biology because of their functions. For instance, scaffolds are used because they can interfere with other proteins in order to (in)activate gene transcription. They are applied in normal and malfunctioning cells to help the world understand more about signal transduction pathways.                  </P><P>
                           One of the many scaffold proteins is the 14-3-3 protein. It is becoming more popular to use this protein in synthetic biology. The 14-3-3 proteins are a family of proteins which are well preserved in evolution and are present in all eukaryotic cells. Many organisms contain isoforms: smaller and more simplistic eukaryotes like yeast contain only two 14-3-3 genes, however, the bigger and more complex eukaryotes can contain up to fifteen different isoforms. Mammalian cells contain seven different isoforms, namely β, ε, γ, η, σ, τ, ζ.<SUP class="source">3</SUP></P></DIV><DIV id="Height_350" class="Column"><DIV id="Binding_Groove_Caption" class="Caption"> Figure 1: The T14-3-3 dimer, consisting of two monomers (light and dark grey). On both monomers the amphipathic ligand-binding groove consisting of helices H3, H5, H7 and H9 are shown in red and raspberry. On the left is a front view, on the right is a top view of the T14-3-3 dimer. </DIV></DIV><DIV class="Column"><P>
One of the 14-3-3 proteins that is used in synthetic biology is from the Nicotiana plumbaginifolia (Tobacco) plant (T14-3-3).<SUP class="source">4</SUP>  T14-3-3 proteins dimerize to form a functional scaffold and every monomer contains a bundle of nine antiparallel alfa-helices. Helices H3, H5, H7 and H9 form the “amphipathic ligand-binding groove” (figure 1), in which other proteins can bind and interact with their co-protein.<SUP class="source">3</SUP>There are different T14-3-3 variants. The variant which will be used and mentioned as T14-3-3 is the T14-3-3cΔC. 
               </P></DIV><DIV class="P_Header">Fusicoccin and CT52</DIV><DIV class="Content_Row"><DIV class="Column_50"><DIV id="Fusicoccin_Caption" class="Caption">Figure 2: The structure of fusicoccin schematically represented. <SUP class="source">11</SUP></DIV></DIV><DIV class="Column_50"><P>
Fusicoccin is a small molecule that is produced by the fungus Fusicoccum amygali.<SUP class="source">7</SUP> It is an organic compound with varying ring structures (figure 2) and because of its size able to diffuse through cell membranes. Fusicoccin is a phytotoxic molecule, meaning it has detrimental effects on plants.<SUP class="source">8</SUP> It is unknown yet if fusicoccin could have any detrimental effects on the human body. However, Skwarczynska, Molzan, and Ottmann experienced no deleterious effects of fusicoccin in HEK293T or in HeLa cells up to a concentration of 60 µM.<SUP class="source">6</SUP> One of the binding partners of T14-3-3 is plant plasma membrane H(+)-ATPase, this protein transports protons over the plasma membrane. This interaction is greatly stabilized by fusicoccin, which binds into a gap between 14-3-3 and H(+)-ATPase (figure 3).<SUP class="source">12</SUP></P></DIV></DIV><DIV class="Column"><P>
Thus the activity of the H(+)-ATPase is increased when fusicoccin is present, leading to an increase of the membrane potential and metabolic processes.<SUP class="source">9</SUP><SUP>,</SUP><SUP class="source">10</SUP> Those C-terminal regions to which T14-3-3 proteins bind have been isolated and are called CT52 (the last 52 amino acids of the C-terminal region of the H(+)-ATPase, amino acids 905 untill and including 956). CT52 is thus a 52 amino acid long protein, which needs a free C-terminal end in order to achieve a stabilized binding with T14-3-3. The N-terminus is also free, but can be used to link other proteins to. In 2013, Skwarczynska, Molzan, and Ottmann showed that it is possible to let those linked proteins dimerize on the T14-3-3 scaffold under influence of fusicoccin and designed a orthogonal binding interaction by charge reversal: glutamic acid to arginine on the 19th amino acid (E19R) of the T14-3-3 and lysine to aspartic acid on the 943rd amino acid (K943D) of the CT-52<SUP class="source">6</SUP>, offering a way to chemically regulate protein-protein interactions.                

</P></DIV><DIV class="P_Header">Interaction</DIV><DIV class="Column"><P>
Many processes in the cell are regulated and coordinated by 14-3-3, such as cell cycle progression, apoptosis, metabolism, transcription, regulation of gene expression and DNA damage repair<SUP class="source">13</SUP><SUP>,</SUP><SUP class="source">14</SUP>, which is why 14-3-3 has such a high diversity of interaction partners.                  </P><P>
The regulation and coordination of those processes can happen through different modes of binding between the 14-3-3 protein and the target protein; (1) altering the ability of the target protein to interact with other proteins; (2) modifying the target protein’s localization; (3) functioning as a scaffold protein to bring proteins together that could interact; (4) altering the intrinsic catalytic activity of the target protein; (5) protecting the target protein from other modifications.<SUP class="source">14</SUP></P><P>
In the binding between fusicoccin and CT52, the important amino acids located in the binding site of CT52 are I956 and H930 (figure 3). In the binding between fusicoccin and 14-3-3 the following important amino acids are located in the binding site of 14-3-3: K129 and K221, L50 and L225, M130, P174, I226, G178, F126, V53, D222, N49 (figure 3). These amino acids have more variation in polarity, size and load, so the binding between fusicoccin and CT52 is more specific than fusicoccin and 14-3-3.<SUP class="source">6</SUP></P><P>
Another key feature of the fusicoccin stabilization is the reversibility, in contrast to other dimerizing agents, for instance Rapamycin.<SUP class="source">13</SUP>  So for high concentrations fusicoccin the CT52 proteins will assemble on the 14-3-3, but when the concentration decreases, the CT52 will dissociate.<SUP class="source">6</SUP></P></DIV><DIV id="Height_400" class="Column"><DIV id="Fusicoccin_Binding_Caption" class="Caption">Figure 3: Fusicoccin in yellow stabilizes the binding between CT52 in blue and T14-3-3 in grey. The important amino acids for the stabilization are represented in raspberry and in red. In raspberry are the amino acids from T14-3-3 and in red are the amino acids from CT52. The stabilization for one monomer is shown, on the other side of the protein the other monomer is also stabilized by a fusicoccin molecule. </DIV></DIV><DIV class="P_Header">References</DIV><DIV class="Column"><UL class="References"><LI>[1] Good, M., Zalatan, J. and Lim, W. (2011). Scaffold Proteins: Hubs for Controlling the Flow of Cellular Information. Science, 332(6030), pp.680-686. </LI><LI>[2] Shaw, A. and Filbert, E. (2009). Scaffold proteins and immune-cell signalling. Nat Rev Immunol, 9(1), pp.47-56.</LI><LI>[3] Obsil, T. and Obsilova, V. (2011). Structural basis of 14-3-3 protein functions. Seminars in Cell &amp; Developmental Biology, 22(7), pp.663-672.</LI><LI>[4] Ottmann, C., Marco, S., Jaspert, N., Marcon, C., Schauer, N., Weyand, M., Vandermeeren, C., Duby, G., Boutry, M., Wittinghofer, A., Rigaud, J. and Oecking, C. (2007). Structure of a 14-3-3 Coordinated Hexamer of the Plant Plasma Membrane H+-ATPase by Combining X-Ray Crystallography and Electron Cryomicroscopy. Molecular Cell, 25(3), pp.427-440.</LI><LI>[5] Skwarczynska, M., Molzan, M. and Ottmann, C. (2012). Activation of NF- B signalling by fusicoccin-induced dimerization. Proceedings of the National Academy of Sciences, 110(5), pp.E377-E386. </LI><LI>[6]  Ballio, A., Chain, E.B., De Leo, P., Erlanger, B.F., Mauri, M., Tonolo, A. (1964). “Fusicoccin: a new wilting toxin produced by Fusicoccum amygdali”, Nature 203(4942): 297 </LI><LI>[7] Bury, M., Andolfi, A., Rogister, B., Cimmino, A., Mégalizzi, V., Mathieu, V., Feron, O., Evidente, A. and Kiss, R. (2013). Fusicoccin A, a Phytotoxic Carbotricyclic Diterpene Glucoside of Fungal Origin, Reduces Proliferation and Invasion of Glioblastoma Cells by Targeting Multiple Tyrosine Kinases. Translational Oncology, 6(2), pagina’s 112-123. </LI><LI>[8] Johansson, F., Sommarin, M. and Larsson, C. (1993). Fusicoccin Activates the Plasma Membrane H + -ATPase by a Mechanism Involving the C-Terminal Inhibitory Domain. The Plant Cell, 5(3), pagina 321.   </LI><LI>[9]Upadhyay, R. (2002). Advances in microbial toxin research and its biotechnological exploitation. New York: Kluwer Academic/Plenum Pub. Pagina 245 </LI><LI>[10] Giordanetto, F., Schäfer, A., &amp; Ottmann, C. (2014). Stabilization of protein–protein interactions by small molecules. Drug discovery today, 19(11), 1812-1821..  </LI><LI>[11] Svennelid, F., Olsson, A., Piotrowski, M., Rosenquist, M., Ottman, C., Larsson, C., ... &amp; Sommarin, M. (1999). Phosphorylation of Thr-948 at the C terminus of the plasma membrane H+-ATPase creates a binding site for the regulatory 14-3-3 protein. The Plant Cell, 11(12), 2379-2391. </LI><LI>[12] Banaszynski, L. A., Liu, C. W., &amp; Wandless, T. J. (2005). Characterization of the FKBP⊙ Rapamycin⊙ FRB Ternary Complex. Journal of the American Chemical Society, 127(13), 4715-4721. </LI><LI>[13] van Hemert, M. J., Steensma, H. Y., &amp; van Heusden, G. P. H. (2001). 14‐3‐3 proteins: key regulators of cell division, signalling and apoptosis. Bioessays, 23(10), 936-946.  </LI><LI>[14] Tzivion, G., Shen, Y. H., &amp; Zhu, J. (2001). 14-3-3 proteins; bringing new definitions to scaffolding. Oncogene, 20(44). </LI></UL></DIV></DIV></DIV><DIV class="Footer"><A href="#" class="back-to-top"> back to top</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>