<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCU_China_Repressilator_Repressilator_rbs skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCU China/Repressilator/Repressilator/rbs</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><DIV class="scu_overview_wrapper"><DIV class="nav nav-justified scu_nav" id="scu_welcome"><LI class="lee_nav_home"><A href="https://2017.igem.org/Team:SCU_China">Home</A></LI><LI class="dropdown lee_nav_project"><A class="dropdown-toggle" data-toggle="dropdown" href="">
                                    Project
                                </A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Repressilator">Repressilator</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Malatonion">Melatonin</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Parts">Parts</A></LI><LI class="dropdown lee_nav_modeling"><A href="https://2017.igem.org/Team:SCU_China/Model" class="dropdown-toggle" data-toggle="dropdown">
                                    Modeling
                                </A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Model/Repressilator">Repressilator</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Model/Melatonin">Melatonin</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Model/RBSModification">RBS
                                        Modification</A></LI><LI class="dropdown lee_nav_modeling"><A href="https://2017.igem.org/Team:SCU_China/HP/Silver" class="dropdown-toggle" data-toggle="dropdown">Human Practices
                                </A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Silver">Overview</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Silver/Survey">Survey</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Silver/Meetup">Meet-up</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Silver/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Collaborations">Collaboration</A></LI><LI class="dropdown lee_nav_attri"><A href="https://2017.igem.org/Team:SCU_China/Attributions" class="dropdown-toggle" data-toggle="dropdown">
                                    Attribution
                                </A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Attributions#Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Attributions#Acknowledgement">Acknowledgement</A></LI><LI class="dropdown lee_nav_gold"><A href="https://2017.igem.org/Team:SCU_China/HP/Gold_Integrated" class="dropdown-toggle" data-toggle="dropdown">
                                    Gold Medal
                                </A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Gold_Integrated">Integrated
                                        HP</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Demonstrate">Demonstrate</A></LI><LI class="lee_nav_note"><A href="https://2017.igem.org/Team:SCU_China/Notebook" class="">
                                    Notebook
                                </A></LI></DIV><DIV class="scu_nav_side" id="scu_nav_side"><DIV id="nav_tree" class="scu_panda_4"><UL class="scu_nav_tree"><LI>RBS Modification
                                        </LI><LI><A href="#design">Design</A></LI><LI><A href="#results">Results</A></LI></UL></DIV></DIV><DIV class="scu_content_wrapper"><DIV class="overview-title scu_detail_title">RBS Modification</DIV><DIV class="scu_sub_title"><I>01</I> Design</DIV><P class="scu_content_attri">      Because we want to mimic the process of melatonin
                                production in human, we need to
                                elongate the period cycle of repressilator. According to the modeling results of
                                repressilator, faster translation is helpful to the period elongation; thus, a strong
                                ribosome binding site may be required. Then, we scanned the iGEM registry, finding that
                                we only have a small number of ribosome binding sites whose strengths have been
                                determined. Therefore, we decided to modify an existing RBS in iGEM registry by
                                ourselves to obtain a series of stronger ribosome binding sites.      Random mutation of RBS will generate a huge library,
                                which will increase our workload
                                grandly with numbers of useless mutations. Thus, we turned to a bioinformatics method to
                                predict the sequence of a synthetic RBS with a target translation initiation rate on a
                                proportional scale. We designed our RBS via the modeling in the website
                                (https://salislab.net/software/doForwardRBS)1, which was based on the theory that build
                                on previous work that characterized the free energies of key molecular interactions
                                involved in translation initiation and on measurements of the sequence-dependent
                                energetic changes that occur during RNA folding and hybridization.1      According to the modeling results of RBS
                                modification, we chose three designed RBS,
                                whose translation rate is predicted to be stronger than the original part: BBa_B0034.
                                The existing Part (BBa_I13521) is a reporter device constitute of TetR repressible
                                promoter (BBa_R0040), RBS (BBa_B0034), mRFP1 (BBa_E1010) and double terminator
                                (BBa_B0015). In our project, we managed to alternate the RBS region with our designed
                                RBS and to construct new part named Part:BBa_K2276007,Part:BBa_K2276008,
                                Part:BBa_K2276010 separately.
                            </P><DIV class="scu_sub_title"><I>02</I> Results</DIV><P class="scu_content_attri">      According to the mathematical modeling result of RBS
                                prediction by the RBS Calculator,
                                the predicted translation initiation rate of BBa_B0034 was 2297.59au. And we chose three
                                designed RBS，which was predicted to be 5、10、50 times stronger than B0034，and was
                                separately designated as P1, P2,P3. (Table 1)
                            </P><DIV class="scu_table_container"><TABLE class="table table-striped table-hover table-bordered"><THEAD class="scu_table_style"><TR><TH>Synthetic RBS</TH><TH>Sequence</TH><TH>Translation Initiation Rate</TH><TH>Relative Prediction Strength</TH></TR></THEAD><TBODY><TR><TD>P1 (BBa_K2276007)</TD><TD>TATAAGGAGTAAATACC</TD><TD>11153.91 au</TD><TD>4.86</TD></TR><TR><TD>P2 (BBa_K2276008)</TD><TD>AAATAAGGAGG</TD><TD>25075.0 au</TD><TD>10.91</TD></TR><TR><TD>P3 (BBa_K2276010)</TD><TD>AAATAATAAGGGGTTTAC</TD><TD>41213.03 au</TD><TD>17.94</TD></TR></TBODY></TABLE><DIV class="scu_img_index">Table 1. Table of designed synthetic RBS. “Relative
                                    prediction strength” was the the ratio of the predicted translation initiation rate
                                    the synthetic sequence to the predicted translation initiation rate BBa_B0034.
                                </DIV></DIV><P class="scu_content_attri">      Then, we constructed the plasmid containing pTet,
                                synthetic RBS, CDS of mRFP and double
                                terminator. (Fig.1) The only difference between the different constructed plasmids and
                                the pSB1C3-BBa_I13521 plasmid is RBS.(Fig.2) And these plasmids were characterized via
                                sequencing.
                            </P><DIV class="scu_hp_image_single_mid"><DIV class="scu_img_index">Fig.1 Scheme of the device.</DIV><DIV class="scu_img_index">Fig.2 Alignment result of BBa_B0034, P1, P2 and P3.</DIV></DIV><P class="scu_content_attri">      Next, we measured the fluorescence intensity and the absorption the absorption value in
                                OD600nm of these cultures with different plasmids according to the following methods:      1. Streaked the plate with Bacterial Glycerol containing the constructed plasmid.      2. Picked single clones from the plate and incubate overnight      3. 12ml tubes containing 1ml liquid LB was inoculated with the overnight cultures using
                                a 1:100 dilution.      4. Tested the fluoresce strength every 2 hours after inoculation sampled from different
                                tubes via plate reader.      We can see that those strains have similar growth pattern，which indicated that the
                                fluorescence discrepancy had nothing to do with the quantity of the bacteria and the
                                growth conditions. (Fig.3)</P><DIV class="scu_hp_image_single_mid"><DIV class="scu_img_index">Fig.3 OD600 Curve</DIV></DIV><P class="scu_content_attri">      And the fluorecent measurement results showed that synthetic RBS P1 and P3 were obviously strong than RBS BBa_B0034. (Fig.4) In addition, we calculated the relative RBS strength via dividing (fluorescence intensity/ OD 600) by (fluorescence intensity of B0034). And this result further illustrated that the synthetic RBS (P1 and P3), especially P3, had obviously higher RBS strength than B0034; that is, the function of the original part (BBa_I13521) was indeed improved by us. (Fig.5)The drop at 16hours of P3 strain may result from the self-quenching which will happen when the concentration is too high. The cause may be the overlap of the emission spectrum and the absorb spectrum or the interaction between stimulated molecule. (Fig.4)(Fig.5)      However, the experimental results of synthetic RBS P2 was not consistent with the prediction results; that is, the synthetic RBS P2 was weaken than the original RBS. In addition, the relative strength of these three synthetic RBS were less than the prediction results (Fig.5). Nevertheless, the qualitative results was consistent between RBS prediction results and experimental results in most cases. Thus, the results convinced us that the design method was reliable so that we can anticipate if we apply the method in our project, and we could increase the translation rate of interested protein and elongate the periodic cycle.

                            </P><DIV class="scu_hp_image_single_mid"><DIV class="scu_img_index">Fig.4 Fluorescent Strength Curve</DIV></DIV><DIV class="scu_hp_image_single_mid"><DIV class="scu_img_index">Fig.5 Relative RBS strength of different RBS
</DIV></DIV></DIV></DIV><DIV class="footer"><P class="scu_footer_text">SCU_China-2017@copyright</P><P class="scu_footer_text_2">Contact Us: SCU_China2017@126.com</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>