<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCU_China_InterLab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCU China/InterLab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><DIV class="scu_interlab_wrapper"><DIV class="nav nav-justified scu_nav" id="scu_welcome"><LI class="lee_nav_home"><A href="https://2017.igem.org/Team:SCU_China">Home</A></LI><LI class="dropdown lee_nav_project"><A class="dropdown-toggle" data-toggle="dropdown" href="">
                                    Project
                                </A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Repressilator">Repressilator</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Malatonion">Melatonin</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Parts">Parts</A></LI><LI class="dropdown lee_nav_modeling"><A href="https://2017.igem.org/Team:SCU_China/Model" class="dropdown-toggle" data-toggle="dropdown">
                                    Modeling
                                </A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Model/Repressilator">Repressilator</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Model/Melatonin">Melatonin</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Model/RBSModification">RBS Modification</A></LI><LI class="dropdown lee_nav_human"><A href="https://2017.igem.org/Team:SCU_China/HP/Silver" class="dropdown-toggle" data-toggle="dropdown">Human Practices
                                </A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Silver">Overview</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Silver/Survey">Survey</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Silver/Meetup">Meet-up</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Silver/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Collaborations">Collaboration</A></LI><LI class="dropdown lee_nav_attri"><A href="https://2017.igem.org/Team:SCU_China/Attributions">
                                    Attribution
                                </A></LI><LI class="dropdown lee_nav_gold"><A href="https://2017.igem.org/Team:SCU_China/HP/Gold_Integrated" class="dropdown-toggle" data-toggle="dropdown">
                                    Gold Medal
                                </A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Gold_Integrated">Integrated HP</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/HP/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:SCU_China/Demonstrate">Demonstrate</A></LI><LI class="lee_nav_note"><A href="https://2017.igem.org/Team:SCU_China/Notebook">
                                    Notebook
                                </A></LI></DIV><DIV class="scu_nav_side" id="scu_nav_side"><DIV id="nav_tree" class="scu_smile_panda"><UL class="scu_nav_tree"><LI>InterLab
                                        </LI><LI><A href="#materials">Materials andEquipment</A></LI><LI><A href="#detail">Detail of Methods</A></LI><LI><A href="#results">Results</A></LI><LI><A href="#acknowledgement">Acknowledgement  and Collaboration</A></LI></UL></DIV></DIV><DIV class="scu_content_wrapper"><DIV class="overview-title scu_detail_title">InterLab Study</DIV><DIV><P class="scu_content_p">      We participated in this
                                year’s InterLab work and uploaded the
                                data and relative questionnaire. In this page, we will list materials and equipment used
                                in the InterLab Study, explain the details of our method, which is described a bit
                                briefly in the official protocol, show our results, and make a sincere acknowledgement
                                to who helped us.
                            </P></DIV><DIV class="scu_sub_title"><I>01</I>  Materials and Equipment</DIV><P class="scu_content_p">We did the experiment following the protocol provided on the iGEM
                                official website, including the transformation procedure. Table 1 and table 2 show the
                                details of our materials and equipment respectively.
                            </P><DIV class="scu_table_container"><TABLE class="table table-striped table-hover table-bordered"><THEAD class="scu_table_style"><TR><TH>Material Name</TH><TH>Resource</TH></TR></THEAD><TBODY><TR><TD>Competent cells</TD><TD>Trans 5 alpha form</TD></TR><TR><TD>DNA</TD><TD>2017 kit plate 7</TD></TR><TR><TD>Tryptone</TD><TD>OXOID LP0042</TD></TR><TR><TD>NaCl</TD><TD>KELONG Company</TD></TR><TR><TD>Yeast extract</TD><TD>OXOID LP0021</TD></TR><TR><TD>Water</TD><TD>Aquae sterilisata</TD></TR><TR><TD>Agar</TD><TD>Solarbio biotechnology grade</TD></TR><TR><TD>Chloramphenicol</TD><TD>Solarbio USP grade</TD></TR><TR><TD>Ethanol</TD><TD>CHRON chemicals</TD></TR><TR><TD>Parafilm</TD><TD>BEMIS</TD></TR><TR><TD>plate</TD><TD>Corning</TD></TR></TBODY></TABLE><DIV class="scu_img_index">Table 1. Details of experimental material</DIV><TABLE class="table table-striped table-hover table-bordered"><THEAD class="scu_table_style"><TR><TH>Equipment Name</TH><TH>Resource</TH></TR></THEAD><TBODY><TR><TD>Thermostatic water-bath</TD><TD>GUOHUA LKTC-B1-T</TD></TR><TR><TD>Fridge</TD><TD>Midea BCD-610WKM(E) for -20 and 4 degrees centigrade,
                                            Haier biomedical ultralow temperature freezer for -80 degrees centigrade
                                        </TD></TR><TR><TD>Biosafety cabinet</TD><TD>ThermoScientific 1300 series A2</TD></TR><TR><TD>Autoclave</TD><TD>ZEALWAY GR85DA</TD></TR><TR><TD>Centrifuge</TD><TD>ThermoScientific ligend micro 17</TD></TR><TR><TD>Thermostatic incubator</TD><TD>HENGZI SPX-150-Z</TD></TR><TR><TD>Shaker</TD><TD>ZHICHENG constant temperature incubator shaker</TD></TR><TR><TD>Plate reader/td&gt;
                                        </TD><TD>ThermoScientific Varioskan Flash</TD></TR></TBODY></TABLE><DIV class="scu_img_index">Table 2. Details of experimental equipment</DIV></DIV><DIV class="scu_sub_title"><I>02</I>  Detail of Methods</DIV><DIV class="scu_acma_subtitle">Details of Transformation</DIV><DIV><P class="scu_content_p">Aquae sterilisata was used to dissolve the DNA powder in kit.
                                After the certain DNA was dissolved, it was kept in -20 degrees centigrade fridge, as
                                well as the DNA powder. When we prepared transformation experiment, competent cells and
                                the DNA was taken out and defrosted on the ice. Mixture of DNA and competent cells was
                                done out of the biosafety cabinet while adding of liquid LB was in it. Solid LB
                                contained 2% agar and chloramphenicol was added before making plates. Parafilm was used
                                to prevent the fast evaporation of plates in incubator.
                            </P></DIV><DIV class="scu_acma_subtitle">Details of Measurement
                            </DIV><P class="scu_content_p">      When we did the measurement. The cabinet was shined
                                by ultraviolet before start for 30 minutes. One person (name C for convenience) took
                                500μl from each culture tube to a EP tube, then gave it to one person out of the cabinet
                                (name B) to test the original OD600. Then, C prepared enough 50ml tubes with 12ml liquid
                                LB in it (chloramphenicol preadded), and B calculated the dilution way according to the
                                excel provided on official website. When the calculation was done, C piped out certain
                                volume of LB and added bacterial culture, then shook to mix and took 500μl as the sample
                                of 0 hour. After the first sampling, we took out the culture from incubator (220rpm,
                                37℃) every 2 hours, with the first sampling time denoting 0 hour.      Plates with black frame and transparent, flat bottom
                                were used to do the measurement. Since we did the experiment many times resulting from
                                some accidents, we recycled the plates. Plates were cleaned by distilled water and 75%
                                ethanol. Plate reader and relative software was manipulated by a teacher due to the
                                requirement of the lab.
                            </P><DIV class="scu_acma_subtitle">Details of Preservation and Verification</DIV><P class="scu_content_p">      After picking up single colonies, the plates were
                                kept in -4 degrees centigrade fridge.
                                Bacterial glycerol stocks were prepared with the final glycerol concentration being 25%.      When we decided to use them, we first re-activated
                                them by plate streaking, then picked
                                single colonies for subsequent work. Sequencing using prime VF2 and VR was done by
                                sending bacterial culture to Chengdu Tsingke zixi Limited Company. Sequencing results
                                were aligned with sequences in iGEM Registry and all of them were right.
                            </P><DIV class="scu_sub_title"><I>03</I>  Results</DIV><P class="scu_content_p">We successfully transformed the plasmids provided in 2017 kit plate
                                7. Name of plasmids and corresponded parts name are shown in Table 3. Short for name
                                will be used to refer instead of parts name for convenience</P><DIV class="scu_table_container"><TABLE class="table table-striped table-hover table-bordered"><THEAD class="scu_table_style"><TR><TH>Plasmid Name</TH><TH>Parts Name</TH></TR></THEAD><TBODY><TR><TD>Positive control (PC)</TD><TD>J23151-B0032-E0040-B0010-B0012</TD></TR><TR><TD>Negative control (NC)</TD><TD>R0040</TD></TR><TR><TD>Test device 1 (D1)</TD><TD>J23101-B0034-E0040-B0010-B0012</TD></TR><TR><TD>Test device 2 (D2)</TD><TD>J23106-B0034-E0040-B0010-B0012</TD></TR><TR><TD>Test device 3 (D3)</TD><TD>J23117-B0034-E0040-B0010-B0012</TD></TR><TR><TD>Test device 4 (D4)</TD><TD>J23101-J364100-E0040-B0010-B0012</TD></TR><TR><TD>Test device 5 (D5)</TD><TD>J23106-J364100-E0040-B0010-B0012</TD></TR><TR><TD>Test device 6 (D6)</TD><TD>J23117-J364100-E0040-B0010-B0012</TD></TR></TBODY></TABLE><DIV class="scu_img_index">Table 3. Name of plasmids and corresponded parts name</DIV></DIV><P class="scu_content_p">      We did the measurement for
                                several times after transformation.
                                Since some mistake occurred at the beginning, we only uploaded the data of the last
                                measurement. Here we show our result in figure 1, 2, 3, 4, representing for the
                                fluorescein standard curve, curve of OD600, fluorescent strength and μM fluorescein per
                                OD600 respectively. Data were calculated in the excel provided by iGEM official website.
                                Blank control had been considered before plotting.</P><DIV class="scu_hp_image_single2"><P class="scu_img_index"> Fig. 1. Fluorescein Standard Curve and Its Log Scale 
                                    (Left) Fluorescein Standard Curve under ex485/em530; (Right) Fluorescein Standard
                                    Curve (log scale)
                                </P><P class="scu_img_index"> Fig. 2. OD600 Curve of InterLab Strains 
                                    NC and PC means the negative control and positive control respectively. Biological
                                    replicates were considered together and error bar shows the standard error of all
                                    the eight replicates.
                                </P><P class="scu_img_index"> Fig. 3. Fluorescent Strength Curve of InterLab Strains
                                    Biological replicates were considered together and error bar shows the standard
                                    error of all the eight replicates.
                   </P><P class="scu_img_index"> Fig. 3. uM Fluorescein / OD600 Curve of InterLab Strains
                                    Biological replicates were considered together and error bar shows the standard
                                    error of all the eight replicates.
                                </P></DIV><P class="scu_content_p">      According to these figures, we can conclude that in the aspect of
                                promoter strength, J23101 is greater than J23106 and J23106 is greater than J23117, and
                                in the aspect of RBS strength, B0034 is greater than J364100 (Figure 3, 4). Difference
                                in RBS strength may be related to the length of spacer between SD sequence in RBS and
                                start codon. We can also see that the growth of D1 and D4 were relatively influenced
                                (Figure 2), suggesting J23101 may have a negative effect on bacterial growth, though the
                                protein expression is relatively high.
                            </P><DIV class="scu_sub_title"><I>04</I>  Acknowledgement and Collaboration
                            </DIV><P class="scu_content_p">      Caused by our carelessness, we put the Ludox in the -20 degrees
                                centigrade and all of them precipitated. Here we sincerely acknowledge team
                                “SiCAU-China” to provide us extra Ludox.
                                      We sincerely gratitude the work done by some freshmen, Caiqin Wang, Tian Liu, Chenming
                                Zhang, Yuanhan Yang, Yanling Zhong, Changhe Li (names listed in no particular order), in
                                InterLab study.

                                      We helped Team SCU-WestChina by giving them some of our bacterial glycerol stocks, and
                                their feedback showed that our transformation was successful.
                                      When we plotted the standard curve, we found that the fluorescent strength seemed lower
                                than expected when fluorescence concentration is very high. Thus, we described this
                                phenomenon in the WeChat group of iSWU for some suggestion. Thankfully, student leader
                                of team “SiCAU-China” told us that that may be caused by the inner filter effect; that
                                is to say, when the concentration of fluorescence is very high, the excitation light
                                cannot fully excited all of them, causing the decrease of fluorescent strength. We also
                                shared this point in CCiC (Central China iGEM Consortium).
                            </P></DIV></DIV><DIV class="footer"><P class="scu_footer_text">SCU_China-2017@copyright</P><P class="scu_footer_text_2">Contact Us: SCU_China2017@126.com</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>