Team:SCU China/Model/RBSModification
Home
Project
Melatonin
Repressilator
InterLab
Parts
Modeling
Overview
Melatonin
Repressilator
RBS                                         Modification
Human Practices
Overview
Survey
Meet-up
Engagement
Collaboration
Attribution
Team
Acknowledgement
Gold Medal
Integrated                                         HP
Improve
Model
Demonstrate
Notebook
InterLab
Design
Results
Reference
RBS Modification
01  Design
      Because we want to mimic                                 the process of melatonin production in human, we need to elongate the period cycle of                                 repressilatorto 24hr. According to the modeling results of repressilator, faster                                 translation is helpful to the period elongation; thus, a strong ribosome binding site                                 may be required. Then, we scanned the iGEM registry, finding that we only have a small                                 number of ribosome binding sites whose strengths have been determined. Therefore, we                                 decided to modify an existing RBS in iGEM registry by ourselves to obtain a series of                                 stronger ribosome binding sites.
      Random mutation of RBS will generate a huge library,                                 which will increase our workload                                 grandly with numbers of useless mutations. Thus, we turned to a bioinformatics method to                                 predict the sequence of a synthetic RBS with a target translation initiation rate on a                                 proportional scale. We designed our RBS via the modeling in the website                                 (https://salislab.net/software/doForwardRBS.)1, which was based on the theory that build                                 on previous work that characterized the free energies of key molecular interactions                                 involved in translation initiation and on measurements of the sequence-dependent                                 energetic changes that occur during RNA folding and hybridization.
02  Results
      Firstly, we inputted the                                 mRNA sequence of the original device BBa_I13521, and predicted the translation                                 initiation rate of the original device BBa_I13521 with RBS BBa_B0034 by the online                                 software tool (https://salislab.net/software/reverse). According to the position of the                                 start codon of coding sequence in the transcript, we found the translation rate                                 prediction result of BBa_I13521 was 2297.59 au. (Fig.1)
Fig.1 Prediction result of BBa_B0034 within BBa_ I13521. The                                     target start codon and related predicted of translation rate were indicated by the                                     red arrow.
      Then, in order to test the consistency of the                                 predicted results with the experimental results, we predicted 3 RBS (BBa_B0029,                                 BBa_B0032, BBa_B0035) collected in the Community RBS Collection of Part Registry (                                 http://parts.igem.org/Ribosome_Binding_Sites/Prokaryotic/Constitutive/Community_Collection                                 ). According to the document in the Part Registry, only BBa_B0035 is stronger than                                 BBa_B0034, and others are weaker. Their respective measured strength documented in Part                                 Registry and predicted strength by us via the tool listed in the table 1. (Table. 1)                                 This table showed that there were a consistent trend between the predicted results and                                 the measured results.
RBS
Relative Measured Strength
Relative Predicted Strength
Predicted Strength
BBa_B0032
0.376
0.279
640.24
BBa_B0029
0.764
0.169
338.47
BBa_B0034
1
1
2297.59
BBa_B0035
1.124
1.666
3828.54
Table 1
      Thus, it convinced us that the RBS calculator could                                 guide our RBS modification. And, we inputted a nucleotide sequence (5 to 20 bp) that                                 appears before the ribosome binding site and the sequence of mRFP (CDS of the original                                 device BBa_I13521) in the online RBS calculator. Next, we entered the target translation                                 initiation rate, estimated by the prediction result of BBa_B0034 within BBa_I13521.                                 Sequentially, the RBS Calculator performed accurate calculation of the final state's                                 free energy, a modified ribosome footprint length and additional interactions between                                 the ribosomal platform and mRNA at upstream standby sites, and outputted a series of                                 synthetic RBS with target translation initiation rate. (Fig.2)
Fig.2 The result of the query which require the translation                                     initiation rate is around 10000au. The synthetic RBS sequence and related predicted                                     of translation rate were in the red boxes.
      We choose three designed                                 RBS，which was predicted to be 5、10、50 times stronger than B0034 respectively, and was                                 separately named as P1, P2,P3. (Table. 2)
Synthetic RBS
Sequence
Translation Initiation Rate
Relative Prediction Strength
P1 (BBa_K2276007)
TATAAGGAGTAAATACC
11153.91 au
4.86
P2 (BBa_K2276008)
AAATAAGGAGGTATAATA
25075.0 au
10.91
P3 (BBa_K2276010)
AAATAATAAGGGGTTTAC
41213.03 au
17.94
Table 2
      In order to characterize properties of such three designed RBS, we did some                                     experiments to substituted the BBa_B0034 in the BBa_I13521 to these synthetic RBS.                                     ( More details about experiments )
03  Reference
1. Salis, H. M., Mirsky, E. A. & Voigt, C. A. Automated design of synthetic ribosome                                 binding sites to control protein expression. Nat. Biotechnol.27, 946–50 (2009).
SCU_China-2017@copyright
Contact Us: SCU_China2017@126.com
