<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CU-Boulder_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CU-Boulder/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="w3-top" class="clear"><DIV class="w3-bar" id="myNavbar"><LI id="list" class="dropdown1"><A href="https://2018.igem.org/Team:CU-Boulder" class="w3-bar-item w3-button" style="color:black;"><B>HOME</B></A></LI><LI id="list" class="dropdown1"><A href="javascript:void(0);" class="dropbtn1" style="color:black;"><B>PROJECT</B></A><DIV class="dropdown-content1"><A href="https://2018.igem.org/Team:CU-Boulder/Description">DESCRIPTION</A><A href="https://2018.igem.org/Team:CU-Boulder/Design">DESIGN</A><A href="https://2018.igem.org/Team:CU-Boulder/Demonstrate">RESULTS</A><A href="https://2018.igem.org/Team:CU-Boulder/Model">MODELLING</A><A href="https://2018.igem.org/Team:CU-Boulder/Experiments">EXPERIMENTS</A><A href="https://2018.igem.org/Team:CU-Boulder/Notebook">NOTEBOOK</A></DIV></LI><LI id="list" class="dropdown1"><A href="javascript:void(0);" class="dropbtn1" style="color:black;"><B>TEAM</B></A><DIV class="dropdown-content1"><A href="https://2018.igem.org/Team:CU-Boulder/Team">TEAM</A><A href="https://2018.igem.org/Team:CU-Boulder/Attributions">ATTRIBUTIONS</A></DIV></LI><LI id="list" class="dropdown1"><A href="https://2018.igem.org/Team:CU-Boulder/Basic_Part" class="dropbtn1" style="color:black;"><B>PARTS</B></A></LI><LI id="list" class="dropdown1"><A href="javascript:void(0);" class="dropbtn1" style="color:black;"><B>COMMUNITY</B></A><DIV class="dropdown-content1"><A href="https://2018.igem.org/Team:CU-Boulder/Human_Practices">HUMAN PRACTICES</A><A href="https://2018.igem.org/Team:CU-Boulder/Collaborations">COLLABORATIONS</A></DIV></LI><LI id="list" class="dropdown1"><A href="https://2018.igem.org/Team:CU-Boulder/InterLab" class="dropbtn1" style="color:black;"><B>INTERLAB</B></A></LI></DIV></DIV><DIV class="bgimg-1 w3-display-container w3-opacity-min" id="home"><DIV class="container" style="white-space:nowrap;"><SPAN class="centered">RESULTS</SPAN></DIV></DIV><H2 id="top">Results</H2><BUTTON class="collapsible">Design and Cloning</BUTTON><DIV class="content"><P>With the use of protein modeling systems, a visual representation of the proteins have been made. The model includes the 
  antibody active sites attached via flexible linkers to the rapamycin complex (FKBP12 and FRB subunits) (fig. 1). The 
  flexible linkers were designed off other successful protein modeling research. The linker is optimal for flexibility and 
  length. With the designed protein model, the addition of rapamycin will induce an allosteric change resulting in the 
  antibody units to become active and properly bind to PD-1.</P><P>The created protein model was then designed for a vector system. Vectors were created using a pET42a vector system and 
  specific double stranded DNA fragments (ordered from IDT) (fig. 2 a-b). The fragments ordered contained necessary overlap 
  regions, allowing for successful incorporation into the pET42a plasmid via gibson assembly. The vectors were transformed 
  with chemical and electrical based techniques. Successful vector formation and transformation occured with the pET42a-PD1- 
  MBP vector (fig. 3).</P><P style="text-align:center;"><A href="#top"><U>Back to Top</U></A></P></DIV><BUTTON class="collapsible">Protein Identification</BUTTON><DIV class="content"><P>With the successful transformation of the pET42a-PD1-MBP plasmid, we were able to successfully transfect protein competent 
 cells. Our desired protein of PD-1 and MBP can be assumed to be produced. An SDS page gel was run to quantify the protein 
 produced. The protein band indicated by the arrow (fig. 4) is assumed to be the protein of interest with the targeted 
 molecular weight at 60.4 kDa. The band falls between 72 kDa and 55 kDa as indicated 
 by the ladder.
</P><P style="text-align:center;"><A href="#top"><U>Back to Top</U></A></P></DIV><BUTTON class="collapsible">Errors</BUTTON><DIV class="content"><P>After successful incorporation of the PD-1 and MBP insert into the pET42a vector and following the success of protein formation, the sequencing results showed two mutations in our vector (fig. 5). One mutation (fig.5a)  present in the sequence of amino acid 48 changing the third position of the codon from a G → T resulting in a missense mutation (serine → arginine).  Another mutation (fig. 5b) present in the sequence of amino acid 29 changing the third position of the codon from G → A resulting in a silent mutation (cysteine →  cysteine).</P><P>It can be assumed that these mutation arose due to the use of Ethidium Bromide, a known mutagen. Gel electrophoresis was stained using EtBr and DNA was extracted and used from these gels. This is most likely the root of the errors as the vector and insert were designed with correct sequences and the Phusion High-Fidelity polymerase is extremely robust.</P><P>A simple solution was implicated once it was found (most likely) that EtBr was causing mutations in the DNA. Agarose Gels are now visualized and stained using SYBR Green, which is noted to be safer for humans to work with and holds lower mutagenic properties comparatively to EtBr. With the use of SYBR Green, gels can be visualized with increased safety for researchers and DNA can be extracted with more security that mutations are not present.</P><P style="text-align:center;"><A href="#top"><U>Back to Top</U></A></P></DIV><BUTTON class="collapsible">Future Experiments</BUTTON><DIV class="content"><P>The mutations present in the pET42a-PD1-MBP vector caused a huge setback in our experiments. Various difficulties arose when reproducing the vector and time was limited. Future experiments will be reproducing the pET42a-PD1-MBP vector without mutations (sequencing of this vector is currently in process as of 10/15/2018). Also, the pET42a-SCFV1-FKBP12-SCFV2-FRB vector is in process of being transformed successfully. After transformation, the vectors will be transfected into protein competent cells. The protein will be purified using affinity column chromatography. The PD1-MBP and pET42a-SCFV1-FKBP12-SCFV2-FRB  will be purified in a nickel column using the HIS tag as the affinity source. Following protein purification, an ELISA assay will be used as a proof of concept. With our protein model system, our hypothesis should be confirmed of denied with the ELISA assay. If our design is successful, we should see an allosteric change in the protein with the addition of rapamycin that causes the antibody active site to bind to the PD1 antigen thus creating an allosteric “on” switch.</P><P style="text-align:center;"><A href="#top"><U>Back to Top</U></A></P></DIV><FOOTER id="footer"><P id="label" style="text-align:center;">CU-Boulder iGEM 2018 Sponsors:</P></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>