<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tsinghua-A_pr"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tsinghua-A/pr">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tsinghua-A/pr&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua-A/pr&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tsinghua-A/pr&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tsinghua-A/pr" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tsinghua-A/pr</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>wiki learned from TagSys - School of Software, Tsinghua University</TITLE></P><DIV class="navbar-wrapper"><DIV class="navbar navbar-inverse navbar-fixed-top"><DIV class="navbar-inner"><DIV class="container"><H1 class="brand"><A href="https://2013.igem.org/Team:Tsinghua-A">Tsinghua-A</A></H1><NAV class="pull-right nav-collapse collapse"><UL id="menu-main" class="nav"><LI><A title="home" href="#bac">Background</A></LI><LI><A title="team" href="#des">Project design</A></LI><LI><A title="projects" href="#res">Results and Application</A></LI></UL></NAV></DIV></DIV></DIV></DIV><SECTION class="section yellow"><UL class="ch-grid"><LI><DIV class="ch-item ch-img-2"><DIV class="ch-info"><H3><A href="https://2013.igem.org/Team:Tsinghua-A/te">Team</A></H3></DIV></DIV></LI><LI><DIV class="ch-item ch-img-3"><DIV class="ch-info"><H3><A href="https://2013.igem.org/Team:Tsinghua-A/pr">Projects</A></H3></DIV></DIV></LI><LI><DIV class="ch-item ch-img-4"><DIV class="ch-info"><H3><A href="https://2013.igem.org/Team:Tsinghua-A/md">Model</A></H3></DIV></DIV></LI><LI><DIV class="ch-item ch-img-5"><DIV class="ch-info"><H3><A href="https://2013.igem.org/Team:Tsinghua-A/test">Wetlab</A></H3></DIV></DIV></LI><LI><DIV class="ch-item ch-img-7"><DIV class="ch-info"><H3><A href="https://2013.igem.org/Team:Tsinghua-A/hm">Human Practice</A></H3></DIV></DIV></LI><LI><DIV class="ch-item ch-img-6"><DIV class="ch-info"><H3><A href="https://2013.igem.org/Team:Tsinghua-A/sf">Safety</A></H3></DIV></DIV></LI><LI><DIV class="ch-item ch-img-8"><DIV class="ch-info"><H3><A href="https://2013.igem.org/Team:Tsinghua-A">Home</A></H3></DIV></DIV></LI></UL></SECTION></DIV><SECTION id="bac" class="section green"><DIV class="container"><H4><STRONG>Synthetic gene switch shows adaptation to DNA copy number variation</STRONG></H4><DIV class="row"><P><STRONG>Background:What is Gene Regulatory Network (GRN)?</STRONG>
				Gene expression is the most basic but complicated process in living system. The interactional connection and regulatory effect among genes make up a Gene Regulatory Network (GRN). People have done great research to figure out the mechanism and rules behind gene regulatory network. Typical gene regulatory networks can achieve bistability, state-switch, biooscillation and so on. Also, some specific network structures have been designed and synthesized to realize artificial control. 
<STRONG>What is Adaptation?</STRONG>
				In order to function reliably, any network needs to keep expression efficiency and accuracy when molecular components fluctuate. So the ability to robustly function and keep the same level of expression despite the fluctuation is called adaptation. In our project, the specific fluctuation is the DNA template abundance (copy number) variation.
<STRONG>How did the idea occur to us ?</STRONG>
				Our project initially originated from medical treatment of cancer. Specific therapy to kill cancer cells greatly demands an effective method to recognize cancer cell. Hela cell is a kind of cancer cell that contains a large amount of endogenic mir-21, while the amount of this microRNA in normal cells is rather low. This remarkable difference inspired us to design a network of which the output protein value is related to the input value of mir-21. So a rational GRN is needed to sense input signal and to distinguish cancer cell from normal cell.
 
Previous research has offered some functional GRNs successfully sensing the difference between high input and low input. We hoped to find out similar network topologies that may also achieve corresponding high-low switch.  
However, there exist many complicated factors that may arouse noise during the expression of gene, which leads to instability and lack of efficiency. We noticed that the number of DNA template (copy number) has a great effect as a noise on the expression intensity. Since the actual amount of copy number involved in gene expression process is hard to measure or control precisely, we have to consider adaptation to DNA copy number variation as a key character of a network. 
So our goal is to search for a rational GRN that can sense the input signal, realize the switch function and show adaptation to copy number. 
Besides, GRN consists of different kinds of regulatory motifs that may play an important role in functioning. We are also motivated to search for certain motifs which contribute to adaptation.

<STRONG>Why are we interested ?</STRONG>
				Cells with different mir-21 amount response differently towards the input signal and their response may represent state-switch. Therefore, cells of 2 different types can be distinguished according to their response. So an adaptive and robust network topology can be regarded as a sensitive switch and a simple sorter, which is really exciting. In addition, possible functional motifs and its mechanism are also fun. 
</P></DIV></DIV></SECTION><SECTION id="des" class="section"><DIV class="container"><H4>Project design</H4><DIV class="row"><P><STRONG>What makes an ideal network?</STRONG>
				An ideal functional network is expected to express output protein at a different intensity according to the different number of input microRNA. So the network should maintain step-like input-output character to realize state-switch. Besides, adaptation to DNA template variation requires the output of ideal network to be saturated along with the increase of copy number.
<STRONG>How to search for an optimal network ?</STRONG>
				We described our regulation reactions based on Michealis-Menten Equation and Hill Equation and constructed all possible three-node network topologies (around 20,000 networks) via ODE matrix equation sets. Then we iterated the regulation process.

				During the thorough investigation of every network’s character under steady state, we finished mass amount of calculation and analysis with the help of High Performance Computing Cluster (HPCC). High-low Ratio and Interim Slope are main indexes for input-output character, and networks’ ability of saturation is considered as the key index for adaptation. 
<STRONG>How to verify our results?</STRONG>
			In order to guarantee the correctness of our results, we simulated the output expression with probability model. We firstly assumed that copy number follows Poisson distribution and studied the Overlap between the output distribution. Then a Bimodal Gaussian distribution is introduced to make comparisons on Overlap of output distribution. The value of both Overlap is rather low when ideal networks are adopted. So we have verified our optimal results to be efficient under distribution by further simulation. 

			
Besides, we transferred the corresponding functional networks (screened from simulation) and contrast unfunctional network into Hela cells in wet lab. The output intensity has clearly increased with functional networks and the trend of saturation to copy number is more obvious with functional network than with unfunctional one. The experiment results generally accord with our modeling expectation of the optimal network structure.

<STRONG>Why are we interested ?</STRONG>
				The biological networks are complex, but growing evidence shows that some core topology structures are capable of robustly executing functions, which are known as motifs. And some of them may be more efficient under certain parameter combinations. After observing the relatively functional network topologies, we are inspired to traverse all functional networks and marked the most commonly appeared motifs. 

			We did some analysis aiming at motifs such as Mutual-Inhibition, Self-Activation, Feedforward with buffer node, Feedback with buffer node and Strong Inhibition and Strong Activation. And we found that the most common motifs in adaptive networks exactly make up the 2 core optimal networks we obtained from previous results. 
</P></DIV></DIV></SECTION><SECTION id="res" class="section blue"><DIV class="container"><H4>3.	Results and Application</H4><DIV class="row"><DIV><P><STRONG>Results</STRONG>
				After complex screening and testing work, we finally got 2 optimal networks. These 2 networks are reliably functional to sense input difference and show great adaptation to DNA template abundance (copy number variation). So these networks meet our needs and goals.


Besides, we summarized the topologies of functional networks and obtained  Feedforward with a buffer node and Mutual_inhibition motifs that are significant to adaptation. 

<STRONG>Application</STRONG>According to our conclusion, we in fact realized a biomachine for sensing signal and effectively switching between high-low states. This function is also the very basis of an automatic sorting machine, which could robustly distinguish two kinds of cells. And the most important thing is that this simple sorter is stable and reliable enough in face with DNA template fluctuation. 
	
                    So our project can be adopted in the future to:
				  
					Sense signals and realize a robust bio-switch in gene circuit.
	Distinguish cancer cells from normal cells and kill cancer cells with specificity.
	Refine a network construction method to synthetize adaptive networks with the intuitionistic impression of regulatory motifs. 
	…</P></DIV></DIV></DIV></SECTION><FOOTER><DIV class="container"><DIV class="row"><DIV class="span6 offset3"><P class="copyright">©  All rights reserved.
			</P></DIV></DIV></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Tsinghua-A/pr">http://2013.igem.org/Team:Tsinghua-A/pr</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tsinghua-A/pr" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tsinghua-A/pr" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tsinghua-A/pr&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tsinghua-A/pr&amp;oldid=238581" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>