"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Munich/Glossary
From 2010.igem.org
Navigation:
Home  →  Glossary
iGEM MainPage
 
Project
Project
Project
Project
Project
Project
Project
Project
Project
Learn more about our project
What is our project about? What is the idea behind our project? This page is a good starting point if you are interested in our project and want to learn more about it.
Software
Software
Software
Software
Software
Software
Software
Software
Software
Build your own network
To demonstrate our vision about Biobrick networks, we developed a software, that enables you to interconnect Biobricks of your choice to your personal network. It will generate a DNA sequence that encodes all your logical connections. Just give it a try!
Modeling
Modeling
Modeling
Modeling
Modeling
Modeling
Modeling
Modeling
Modeling
Understand our RNA devices
On this page we show how we analyzed and estimated the switching process of our RNA devices. This is a basic requirement for the creation of better RNA devices.
Lab
Lab
Lab
Lab
Lab
Lab
Lab
Lab
Lab
Learn more about our experiments
This page provides an overview as well as detailed information about the experiments and measurements we performed. Protocols as well as our daily lab book will clarify any question about our lab proceedings.
BioBricks
BioBricks
BioBricks
BioBricks
BioBricks
BioBricks
BioBricks
BioBricks
BioBricks
Our contribution to the registry
Our project required the analysis and modification of existing Biobricks from the part registry as well as the creation of new Biobricks. This page contains more information about these steps.
Team
Team
Team
Team
Team
Team
Team
Team
Team
Get to know us
Who are we? Where are we from? What do we do in our spare time? Here you can find the answers to all these questions.
Glossary
Glossary
Glossary
Glossary
Glossary
Glossary
Glossary
Glossary
Glossary
Get the meaning of a term
To understand our project description more easily, we put together a list of terms that might be unknown or forgotten. Here you will find explanations concerning these terms.
Partners & Press
Partners & Press
Partners & Press
Partners & Press
Partners & Press
Partners & Press
Partners & Press
Partners & Press
Partners & Press
Thanks a lot to our supporters
This project would not have been possible without the help and support of various people, institutions and companies. We want to say thank you to all of them!
Beyond the Lab
Beyond the Lab
Beyond the Lab
Beyond the Lab
Beyond the Lab
Beyond the Lab
Beyond the Lab
Beyond the Lab
Beyond the Lab
iGEM is more than biology
The iGEM competition not only aims to drive the development of new Biobricks. It also encourages to think about responsibilities, safety and risks in the field of biotechnology. Furthermore, iGEM is about working together and helping other teams in various aspects. Read more how we want to contribute to these goals.
This glossary page will give you a short explanation about the following topics or terms:
Contents
1 Abbreviations
2 Antitermination
3 Aptamer
4 Attenuation
5 His-Term
6 Identity Site (bioLOGICS)
7 iGEM
8 Kinefold
9 Linker (modeling)
10 Logical circuit
11 NUPACK
12 PoPS-based devices
13 Recognition Site (bioLOGICS)
14 Riboswitch
15 Signal RNA
16 Stochastic RNA-folding simulation
17 Strand Displacement
18 Switch (bioLOGICS)
19 Target Site (bioLOGICS)
20 Terminator
21 Tiny Abortive RNA
22 Toehold
23 Transmitter (bioLOGICS)
24 Trigger Site (bioLOGICS)
25 Trp-Term
Abbreviations
iGEM TUM2010
His-Term
Histidine operon terminator, His-Switch, compare here
NUPACK
Nucleic Acid Package compare here
MPA
malachite green binding aptamer
PoPS
Polymerases per Second
RBS
Ribosome binding site
RPO
RNA Polymerase
Trp-Term
Tryptophan operon terminator, Trp-Switch, compare here
TUM
Technical University Munich, Technische Universität München
Proteins
Read more
GFP
Green Fluorescent Protein
mCherry
Not really an abbreviation: monomeric Fluorescent Protein in the color of cherries (red)
RFP
Red Fluorescent Protein
Close
For non-scientists
Read more
A
Adenine, one of the four bases in RNA and DNA
bp
base pair
C
Cytosine, one of the four bases in RNA and DNA
DNA
Deoxyribonucleic acid
E. coli
Escherichia coli
G
Guanine, one of the four bases in RNA and DNA
IPTG
Isopropyl β-D-1-thiogalactopyranoside
nt
nucleotide
PAGE
Polyacrylamide gel electrophoresis
RNA
Ribonucleic acid
T
Thymine, one of the four bases in DNA - does not occur in RNA
U
Uracil, one of the four bases in RNA - does not occur in DNA
Close
Antitermination
Antitermination exists in two different ways in bacterial cells: Protein dependent antitermination and protein-independent antiterminaiton. In general antitermination is regulatory mechanism where a Rho-independent terminator in front of a gene inhibits transcription except when a corresponding sequences binds and leads to an alternative folding conformation of the RNA, allowing transcription beyond the terminator. Protein-independent antitermination in nature is often used to control biosynthetic operons and adapt gene expression in response to environmental changes. [1] The principle our project relies on basic antitermination principles but is completely different from previously decribed cases. In May 2010 (in the middle of our iGEM project and experiments by the way), Kang and coworkers published a paper about antitermination by small RNA pieces which are formed during transcription initiation. [2] The principle described here is similiar to the theory we based our switching principle on and work described in this paper we later used to base some in vitro T7 measurements on.
Aptamer
Aptamers in our case are defined as artificially designed single stranded nucleotides designated to specifically bind ligands. Large combinatorial libraries of RNA or DNA molecules are used to isolate and evolve aptamers which achieve surprisingly high affinity and specifity. [3] Aptamers can be used for a broad range of applications: While in our case the malachitegreen-binding aptamer is only used for detection of RNA synthesis in response to an antitermination signal, other approaches use aptamers for regulating gene expression and - together with ribozymes - to design logical gates. By introducing the malachite-green binding aptamer to the Parts Registry we provide a nice tool to evaluate transcription both in vitro and in vivo. Malachite green binding aptamers were already for screening assays [4] and aptamers in general were lately tested in mammalian cells in vivo (F. Simmel, personal communication).
Attenuation
Attenuation is a very smart way of gene regulation which is known from bacterial cells using two alternatice hairpin structures. For example, E. coli only needs very little amounts of Tryptophan in its metabolism, so the amino-acyl-Synthetase for Tryptophan is only rarely synthesized. So the trp-operon contains an attenuator before the actual enzymes. If Tryptophan is absent, the rare tRNA loaded with Tryptophan will not be available at once, so the Ribosome is stalled. Sterics do not allow the formation of a certain stemloop with the ribosome attached. If there is Tryptophan available and many tRNATrp float through the cell, the ribosome can just continue, a stem loop is formed and the ribosome falls off: The transcription of the following trp-operon is terminated. [5] [6] [7]
His-Term
The nickname His-Term we used mostly in the same meaning as His-Switch. His refers to the terminator hairpin in the his-operon of Salmonella enterica. The His-terminator was one of the most thoroughly evaluated terminators we tested in our experimental work. The His-terminator was chosen because of its role in the metabolic regulation where its antitermination also leads to a read-through of genes needed in biosynthetic pathways. [8]
Identity Site (bioLOGICS)
The identity site is the part of a transmitter , that recognizes and provides the binding to a switch. Together with the [ https://2010.igem.org/transmitter can change the default state "off" of the switches which built up an AND/OR circuit to an "on" and the default state "off" in case of an NOT circuit to an "on". The identity site of the transmitter binds to the recognition site of a switch.
iGEM
i'm Gonna End up in Massachusetts... [9]
Kinefold
The Kinefold web server provides a web interface for stochastic folding simulations of RNA and DNA structures and offers the choice of renaturation or co-transcriptional folding. The folding paths are simulated at the level of helix formation and dissociation as these stochastic formation and the removal of individual helices are known to be the limiting steps of RNA folding kinetics. Kinefold is capable of handling knots and pseudoknots. The most important feature in our case is the possibility to monitor the time-frame of our RNA/RNA-interaction. The kinefold web server can be reached here . [10]
Linker (modeling)
Kinefold, the program of choice we used for modeling, can only calculate folding kinetics of individual RNA structures, but not of the interaction between two structures. Thus in our case, a linker sequence is used to connect  the switch and a corresponding signal. The linker, shown as a row of (X) can be set to a specific length and the 'X bases' do not pair. It mostly serves as a mere placeholder in this model, effects of the linker nevertheless influence the folding time. Since in reality the linker might be thought of as infinitely long, the results shown for the His-terminator in the modeling section, obtained for varying linker lengths, actually support our model: long linkers influence the folding-time while the minimal free-energy is hardly influenced by the linker length. [11]
Logical circuit
A logical circuits consists of switches, which can be connected with each other leading to a circuit which reacts to specific signals. The most common logical circuits are AND, OR and NOT. While AND/OR can be built up using switches, which have an "off" default state and are changed to "on" upon binding of a signal, must always consist of at least to switches, a NOT circuit can consist only of one switch with an "off" default state which is changed to "on" upon binding of a signal.
Logical circuits are the basic of every logical network. Further circuits like XOR and NOR may be constructed based on individual switches and corresponding signals for each individual application. [12]
NUPACK
The Nucleic Acid Package, is a non-commercial software suite both for analysis and design of nucleic acid systems. Within the scope of our project, we mainly utilized the analysis tool, which allows to determine thermodynamical properties of inter- and intramolecular interactions, both for DNA and RNA, especially in regard of the two-dimensional folding. NUPACK is able to calculate the minimum free energy for secondary structure and base-pairing propabilities making it an excellent tools to design and predict structures. Furthermore, NUPACK delivers the partition function, and equilibrium concentration for multiple interaction species. The NUPACK web server can be found here [13]
PoPS-based devices
Polymerases per Second (PoPS) is a measurement unit for transcriptional devices. It illustrates the bypass-rate of the Polymerase at a specific site. PoPS-based devices are sequences, which influence the PoPS rate. Promoters may be thought off as PoPS sources creating a steady PoPS output, while terminators are the exact opposite breaking PoPS input off. Since our switches work on the transcriptional level interfering with PoPS rate, they are PoPS-based devices as well. [14]
Recognition Site (bioLOGICS)
The recognition site is the part of a switch, which provides specifity towards the corresponding signal. While the switching element, fulfills the main duty of the switch, to change between an "on" and an "off" state and consists by definition of a terminator in the case of switches which built up AND/OR circuits and may be the same for a whole network, the recognition site provides the needed specifity for a switch to be only recognized by the corresponding signal and can consist of any sequence.
Riboswitch
While artificial aptamers are designed to specifally bind their ligands, riboswitches are developed by nature for gene regulation in prokaryotes, with some also found in eukaryotes. Riboswitches are part of mRNAs they control, located in the non-coding parts. They bind to specific metabolites and control gene expression by changing the three-dimensional RNA structures using effects like transcription elongation or translation initiation. Riboswitches typically consist of two parts, the aptamer domain and the expression platform. While the aptamer domain recognizes the metabolite specifically, the expression platforms allows transduction from metabolite binding to gene control by influencing the effciency of translation initiation, transcriptional elongation or stabilizing the mRNA in general. [15] [16] [17]
Signal RNA
see Transmitter (bioLOGICS)
Stochastic RNA-folding simulation
RNA-folding kinetics proceed through rare stochastic openings and closings of individual RNA helices. Simulating a stochastic RNA-folding path means to follow one particular stochastic trajectory within a large combinatorial space of possible helices. Each transition to a neighboring state on this trajectory occurs with a certain transition probability form state i to state j. This transition probability is the product of the average number of transitions from state i to state j per unit time and the liftime of state i which is the average time before any transition to state j occurs. [18] [19]
Strand Displacement
Polymerases are known to displace DNA and RNA pieces in order to start the reaction. The same can oocur upon binding of a RNA/DNA sequence to a complementary strain. In our case, this phenomen is used built switchable elements: The hairpin structure of the terminator is disctructed by the transmitter RNA by displacement of the complementary RNA of the hairpin. [20]
Switch (bioLOGICS)
A switch in our case consists of two major units, the recognition site and the target site. While the recognition site provides the needed specificity to bind only the corresponding transmitter and may contain of any random sequence (as long as it fits the transmitter sequence), the target site may be the same for the whole network and is by definition a terminator. In case of "on"/"off" switches, the principle is quite simple: Without the corresponding transmitter, the switch forms a stem loop, the RNA Polymerase falls off, transcription is terminated. By binding of a transmitter, antitermination occurs, the stem loop does not form, the RNA Polymerase continues transcription and the following logic gate or the desired output may be formed.
In case of a NOT switch, the switch forms a secondary structure which does not terminate by itself, resulting in an "on" as the default state. Upon binding of a transmitter, a terminator forms and transcription is stopped.
For further information of how to construct a switch, please check here for detailed information.  Individual switches can be used to built up a logical network which only consists of DNA/RNA elements for individual applications with the help of this applet .
Target Site (bioLOGICS)
The part of a switch which forms a stem loop in the case of switches with an "off" default state used for construction of AND/OR gates or an transcriptable sequence leading to the state "on" in case of a "NOT" gate. The target site is the basic component of our switching concept, to control the structure of the target site and its consequence for the RNA Polymerase by a signal is the major characteristic of a biolOGIC switch. In principle, every terminator can be utilized as a switch by addition of a recognition site. The trigger site of a signal binds to the target site of the switch to change between "on" and "off".
Terminator
In nature, two termination principles are common in prokaryotes: Rho-dependent termination, where transcription is stopped in response of a protein factor binding to a specific sequence and Rho-independent termination, where a hairpin is formed which leads for the RNA polymerase to stop after the hairpin. Switches constructed after our concept are based on Rho-independent terminators. In principle, the target site equals a terminator. [21] [22]
Tiny Abortive RNA
Tiny abortive RNA is one of the newest additions to the world of small, regulatory RNAs. In May 2010, the group of Changwon Kang from Korea Advanced Institute of Science and Technology, published a paper about antitermination by tiny abortive RNA in a T7 system. Those RNAs are formed in the process of transcription initiation at the changeover to transcription elongation. Since the transcription complex is instable at the beginning, a constant change between transcription abortion and restarting occurs, leading to the release of short RNAs. The RNA formed during this ‘abortive initiation cycling’ could not be assigned to a biological process before but was found to regulate transcription by antitermination by Kang and coworkers. Since those RNAs are in approximately the same dimension as the transmitter RNAs we used in our iGEM project, this work was a wellcomed proof of principle. We tried to reproduce parts of the experiments described in the paper using T7 in vitro transcription. [2]
Toehold
Double stranded RNA with a short overhang may have a thermodynamical advantage if it pairs with another longer RNA strand. This phenomenom is used in the construction of the switch forming a NOT gate. [23]
Transmitter (bioLOGICS)
Transmitters are what make a switch a switch. Without the corresponding transmitter, the switches of our concept are terminators. The transmitter is leading to antitermination and is setting the default "off" state in the case of a AND/OR switch to an "on". Each transmitter consists of a trigger unit and a identity site. While the trigger unit induces the antitermination, the identity site can be varied to bind to the recognition site of the switch.
Trigger Site (bioLOGICS)
The trigger unit is the part of the signal which allows interaction with the switch leading to antitermination in case of a switch which is part of a AND/OR logical gate and formation of a terminator in case of a NOT-switch. This is done by binding to the switching unit of the corresponding switch.
Trp-Term
The nickname Trp-Term we used mostly in the same meaning as Trp-Switch. Trp refers to the terminator hairpin in the trp-operon of E. coli. The Trp-terminator was one of the most thoroughly evaluated terminators we tested in our experimental work. The Trp-terminator was chosen because of its role in the metabolic regulation where its antitermination also leads to a read-through of genes needed in biosynthetic pathways. [24]
References
[1] Yanofsky, C., Transcription attenuation. Journal of Biological Chemistry, 1988. 263(2): p. 609. [2] Lee, S., H.M. Nguyen, and C. Kang, Tiny abortive initiation transcripts exert antitermination activity on an RNA hairpin-dependent intrinsic terminator. Nucleic Acids Research. [3] Weigand, J.E. and B. Suess, Aptamers and riboswitches: perspectives in biotechnology. Applied microbiology and biotechnology, 2009. 85(2): p. 229-236. [4] Rowe, W., M. Platt, and P.J.R. Day, Advances and perspectives in aptamer arrays. Integrative Biology, 2009. 1(1): p. 53-58. [5] Shimotsu, H., et al., Novel form of transcription attenuation regulates expression the Bacillus subtilis tryptophan operon. Journal of bacteriology, 1986. 166(2): p. 461. [6] Oxender, D.L., G. Zurawski, and C. Yanofsky, Attenuation in the Escherichia coli tryptophan operon: role of RNA secondary structure involving the tryptophan codon region. Proceedings of the National Academy of Sciences of the United States of America, 1979. 76(11): p. 5524. [7] Naville, M. and D. Gautheret, Transcription attenuation in bacteria: theme and variations. Briefings in Functional Genomics, 2009. [8] Gollnick, P. and P. Babitzke, Transcription attenuation. Biochimica et Biophysica Acta (BBA)-Gene Structure and Expression, 2002. 1577(2): p. 240-250. [9] Jan Schüürmann, personal communication [10] Xayaphoummine, A., T. Bucher, and H. Isambert, Kinefold web server for RNA/DNA folding path and structure prediction including pseudoknots and knots. Nucleic acids research, 2005. 33(suppl 2): p. W605. [11] http://kinefold.curie.fr [12] Lu, T.K., A.S. Khalil, and J.J. Collins, Next-generation synthetic gene networks. Nature biotechnology, 2009. 27(12): p. 1139-1150. [13] Zadeh, J.N., et al., NUPACK: Analysis and design of nucleic acid systems. Journal of Computational Chemistry. [14] http://openwetware.org/wiki/PoPS [15] Mandal, M. and R.R. Breaker, Gene regulation by riboswitches. Nature Reviews Molecular Cell Biology, 2004. 5(6): p. 451-463. [16] Mandal, M., et al., Riboswitches control fundamental biochemical pathways in Bacillus subtilis and other bacteria. Cell, 2003. 113(5): p. 577-586. [17] Nudler, E. and A.S. Mironov, The riboswitch control of bacterial metabolism. Trends in biochemical sciences, 2004. 29(1): p. 11-17. [18] Gultyaev, A.P., F.H.D. Van Batenburg, and C.W.A. Pleij, The computer simulation of RNA folding pathways using a genetic algorithm. Journal of Molecular Biology, 1995. 250(1): p. 37-51. [19] Brion, P. and E. Westhof, Hierarchy and dynamics of RNA folding. Annual review of biophysics and biomolecular structure, 1997. 26(1): p. 113-137. [20] Walker, G.T., et al., Strand displacement amplification--an isothermal, in vitro DNA amplification technique. Nucleic Acids Research, 1992. 20(7): p. 1691. [21] Jeng, S.T., J.F. Gardner, and R.I. Gumport, Transcription termination by bacteriophage T7 RNA polymerase at rho-independent terminators. Journal of Biological Chemistry, 1990. 265(7): p. 3823. [22] Lesnik, E.A., et al., Prediction of rho-independent transcriptional terminators in Escherichia coli. Nucleic acids research, 2001. 29(17): p. 3583. [23] Berg, Tymocszo, Stryer., Biochemistry, International Edition [24] Nichols, B.P., M. Blumenberg, and C. Yanofsky, Comparison of the nucleotide sequence of trpA and sequences immediately beyond the trp operon of Klebsiella aerogenes, Salmonella typhimurium and Escherichia coli. Nucleic acids research, 1981. 9(7): p. 1743.
Retrieved from " http://2010.igem.org/Team:TU_Munich/Glossary "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
