<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Munich_Lab"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Munich/Lab">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Munich/Lab&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Lab&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Lab&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Munich/Lab" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Munich/Lab</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P style="width: 154px; display: block; text-align: center; font-weight: bold;">Navigation:</P><TABLE border="0" cellpadding="0" cellspacing="0" style="background: none; width:100%"><TBODY><TR><TD align="left"><P style="margin: 0; color: #4E9D20;"><A style="color: black;" href="https://2010.igem.org/Team:TU_Munich">Home</A><SPAN style="color: black; font-size: 1.2em;"> → </SPAN>

Lab

</P></TD><TD align="right"><A href="https://2010.igem.org/" target="_blank" style="margin-right: 15px;">iGEM MainPage</A></TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Experiment_Design"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Experiment Design</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#In_vivo_Measurements"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">In vivo Measurements</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Design"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Construction_and_Cloning"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Construction and Cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Measurements_based_on_submitted_Biobricks"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Measurements based on submitted Biobricks</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Switch_evaluation_in_vivo"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Switch evaluation in vivo</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-7"><A href="#In_vitro_Translation"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">In vitro Translation</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Design_2"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Design</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Measurements"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Measurements</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#In_vitro_Transcription"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">In vitro Transcription</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#T7_RNA_polymerase"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">T7 RNA polymerase</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#E._coli_RNA_polymerase"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">E. coli RNA polymerase</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Denaturing_Polyacrylamide_gel_electrophoresis"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Denaturing Polyacrylamide gel electrophoresis</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Malachite_green_assay"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Malachite green assay</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Lab_Book"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Lab Book</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Explanations"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Explanations</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Chronological_Lab_Book"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Chronological Lab Book</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#08.04.2010"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">08.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#09.04.2010"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">09.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#15.04.2010"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">15.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#16.04.2010"><SPAN class="tocnumber">2.2.4</SPAN><SPAN class="toctext">16.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#19.04.2010"><SPAN class="tocnumber">2.2.5</SPAN><SPAN class="toctext">19.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#20.04.2010"><SPAN class="tocnumber">2.2.6</SPAN><SPAN class="toctext">20.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#21.04.2010"><SPAN class="tocnumber">2.2.7</SPAN><SPAN class="toctext">21.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#22.04.2010"><SPAN class="tocnumber">2.2.8</SPAN><SPAN class="toctext">22.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#23.04.2010"><SPAN class="tocnumber">2.2.9</SPAN><SPAN class="toctext">23.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#26.04.2010"><SPAN class="tocnumber">2.2.10</SPAN><SPAN class="toctext">26.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#27.04.2010"><SPAN class="tocnumber">2.2.11</SPAN><SPAN class="toctext">27.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#28.04.2010"><SPAN class="tocnumber">2.2.12</SPAN><SPAN class="toctext">28.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#29.04.2010"><SPAN class="tocnumber">2.2.13</SPAN><SPAN class="toctext">29.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#30.04.2010"><SPAN class="tocnumber">2.2.14</SPAN><SPAN class="toctext">30.04.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#04.05.2010"><SPAN class="tocnumber">2.2.15</SPAN><SPAN class="toctext">04.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#05.05.2010"><SPAN class="tocnumber">2.2.16</SPAN><SPAN class="toctext">05.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#06.05.2010"><SPAN class="tocnumber">2.2.17</SPAN><SPAN class="toctext">06.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#07.05.2010"><SPAN class="tocnumber">2.2.18</SPAN><SPAN class="toctext">07.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#10.05.2010"><SPAN class="tocnumber">2.2.19</SPAN><SPAN class="toctext">10.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-37"><A href="#11.05.2010"><SPAN class="tocnumber">2.2.20</SPAN><SPAN class="toctext">11.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-38"><A href="#12.05.2010"><SPAN class="tocnumber">2.2.21</SPAN><SPAN class="toctext">12.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-39"><A href="#14.05.2010"><SPAN class="tocnumber">2.2.22</SPAN><SPAN class="toctext">14.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-40"><A href="#17.05.2010"><SPAN class="tocnumber">2.2.23</SPAN><SPAN class="toctext">17.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#18.05.2010"><SPAN class="tocnumber">2.2.24</SPAN><SPAN class="toctext">18.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-42"><A href="#19.05.2010"><SPAN class="tocnumber">2.2.25</SPAN><SPAN class="toctext">19.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-43"><A href="#20.05.2010"><SPAN class="tocnumber">2.2.26</SPAN><SPAN class="toctext">20.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-44"><A href="#21.05.2010"><SPAN class="tocnumber">2.2.27</SPAN><SPAN class="toctext">21.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-45"><A href="#25.05.2010"><SPAN class="tocnumber">2.2.28</SPAN><SPAN class="toctext">25.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-46"><A href="#26.05.2010"><SPAN class="tocnumber">2.2.29</SPAN><SPAN class="toctext">26.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-47"><A href="#27.05.2010"><SPAN class="tocnumber">2.2.30</SPAN><SPAN class="toctext">27.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-48"><A href="#28.05.2010"><SPAN class="tocnumber">2.2.31</SPAN><SPAN class="toctext">28.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-49"><A href="#31.05.2010"><SPAN class="tocnumber">2.2.32</SPAN><SPAN class="toctext">31.05.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-50"><A href="#01.06.2010"><SPAN class="tocnumber">2.2.33</SPAN><SPAN class="toctext">01.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-51"><A href="#02.06.2010"><SPAN class="tocnumber">2.2.34</SPAN><SPAN class="toctext">02.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-52"><A href="#07.06.2010"><SPAN class="tocnumber">2.2.35</SPAN><SPAN class="toctext">07.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-53"><A href="#08.06.2010"><SPAN class="tocnumber">2.2.36</SPAN><SPAN class="toctext">08.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-54"><A href="#09.06.2010"><SPAN class="tocnumber">2.2.37</SPAN><SPAN class="toctext">09.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-55"><A href="#10.06.2010"><SPAN class="tocnumber">2.2.38</SPAN><SPAN class="toctext">10.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-56"><A href="#11.06.2010"><SPAN class="tocnumber">2.2.39</SPAN><SPAN class="toctext">11.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-57"><A href="#14.06.2010"><SPAN class="tocnumber">2.2.40</SPAN><SPAN class="toctext">14.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-58"><A href="#15.06.2010"><SPAN class="tocnumber">2.2.41</SPAN><SPAN class="toctext">15.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-59"><A href="#16.06.2010"><SPAN class="tocnumber">2.2.42</SPAN><SPAN class="toctext">16.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-60"><A href="#17.06.2010"><SPAN class="tocnumber">2.2.43</SPAN><SPAN class="toctext">17.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-61"><A href="#18.06.2010"><SPAN class="tocnumber">2.2.44</SPAN><SPAN class="toctext">18.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-62"><A href="#22.06.2010"><SPAN class="tocnumber">2.2.45</SPAN><SPAN class="toctext">22.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-63"><A href="#23.06.2010"><SPAN class="tocnumber">2.2.46</SPAN><SPAN class="toctext">23.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-64"><A href="#24.06.2010"><SPAN class="tocnumber">2.2.47</SPAN><SPAN class="toctext">24.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-65"><A href="#25.06.2010"><SPAN class="tocnumber">2.2.48</SPAN><SPAN class="toctext">25.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-66"><A href="#28.06.2010"><SPAN class="tocnumber">2.2.49</SPAN><SPAN class="toctext">28.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-67"><A href="#29.06.2010"><SPAN class="tocnumber">2.2.50</SPAN><SPAN class="toctext">29.06.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-68"><A href="#01.07.2010"><SPAN class="tocnumber">2.2.51</SPAN><SPAN class="toctext">01.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-69"><A href="#02.07.2010"><SPAN class="tocnumber">2.2.52</SPAN><SPAN class="toctext">02.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-70"><A href="#05.07.2010"><SPAN class="tocnumber">2.2.53</SPAN><SPAN class="toctext">05.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-71"><A href="#06.07.2010"><SPAN class="tocnumber">2.2.54</SPAN><SPAN class="toctext">06.07.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-72"><A href="#09.08.2010"><SPAN class="tocnumber">2.2.55</SPAN><SPAN class="toctext">09.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-73"><A href="#10.08.2010"><SPAN class="tocnumber">2.2.56</SPAN><SPAN class="toctext">10.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-74"><A href="#11.08.2010"><SPAN class="tocnumber">2.2.57</SPAN><SPAN class="toctext">11.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-75"><A href="#12.08.2010"><SPAN class="tocnumber">2.2.58</SPAN><SPAN class="toctext">12.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-76"><A href="#13.08.2010"><SPAN class="tocnumber">2.2.59</SPAN><SPAN class="toctext">13.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-77"><A href="#16.08.2010"><SPAN class="tocnumber">2.2.60</SPAN><SPAN class="toctext">16.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-78"><A href="#17.08.2010"><SPAN class="tocnumber">2.2.61</SPAN><SPAN class="toctext">17.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-79"><A href="#18.08.2010"><SPAN class="tocnumber">2.2.62</SPAN><SPAN class="toctext">18.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-80"><A href="#19.08.2010"><SPAN class="tocnumber">2.2.63</SPAN><SPAN class="toctext">19.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-81"><A href="#20.08.2010"><SPAN class="tocnumber">2.2.64</SPAN><SPAN class="toctext">20.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-82"><A href="#23.08.2010"><SPAN class="tocnumber">2.2.65</SPAN><SPAN class="toctext">23.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-83"><A href="#24.08.2010"><SPAN class="tocnumber">2.2.66</SPAN><SPAN class="toctext">24.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-84"><A href="#25.08.2010"><SPAN class="tocnumber">2.2.67</SPAN><SPAN class="toctext">25.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-85"><A href="#26.08.2010"><SPAN class="tocnumber">2.2.68</SPAN><SPAN class="toctext">26.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-86"><A href="#27.08.2010"><SPAN class="tocnumber">2.2.69</SPAN><SPAN class="toctext">27.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-87"><A href="#30.08.2010"><SPAN class="tocnumber">2.2.70</SPAN><SPAN class="toctext">30.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-88"><A href="#31.08.2010"><SPAN class="tocnumber">2.2.71</SPAN><SPAN class="toctext">31.08.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-89"><A href="#01.09.2010"><SPAN class="tocnumber">2.2.72</SPAN><SPAN class="toctext">01.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-90"><A href="#02.09.2010"><SPAN class="tocnumber">2.2.73</SPAN><SPAN class="toctext">02.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-91"><A href="#03.09.2010"><SPAN class="tocnumber">2.2.74</SPAN><SPAN class="toctext">03.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-92"><A href="#06.09.2010"><SPAN class="tocnumber">2.2.75</SPAN><SPAN class="toctext">06.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-93"><A href="#07.09.2010"><SPAN class="tocnumber">2.2.76</SPAN><SPAN class="toctext">07.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-94"><A href="#08.09.2010"><SPAN class="tocnumber">2.2.77</SPAN><SPAN class="toctext">08.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-95"><A href="#09.09.2010"><SPAN class="tocnumber">2.2.78</SPAN><SPAN class="toctext">09.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-96"><A href="#10.09.2010"><SPAN class="tocnumber">2.2.79</SPAN><SPAN class="toctext">10.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-97"><A href="#13.09.2010"><SPAN class="tocnumber">2.2.80</SPAN><SPAN class="toctext">13.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-98"><A href="#14.09.2010"><SPAN class="tocnumber">2.2.81</SPAN><SPAN class="toctext">14.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-99"><A href="#15.09.2010"><SPAN class="tocnumber">2.2.82</SPAN><SPAN class="toctext">15.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-100"><A href="#16.09.2010"><SPAN class="tocnumber">2.2.83</SPAN><SPAN class="toctext">16.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-101"><A href="#17.09.2010"><SPAN class="tocnumber">2.2.84</SPAN><SPAN class="toctext">17.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-102"><A href="#20.09.2010"><SPAN class="tocnumber">2.2.85</SPAN><SPAN class="toctext">20.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-103"><A href="#21.09.2010"><SPAN class="tocnumber">2.2.86</SPAN><SPAN class="toctext">21.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-104"><A href="#22.09.2010"><SPAN class="tocnumber">2.2.87</SPAN><SPAN class="toctext">22.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-105"><A href="#23.09.2010"><SPAN class="tocnumber">2.2.88</SPAN><SPAN class="toctext">23.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-106"><A href="#24.09.2010"><SPAN class="tocnumber">2.2.89</SPAN><SPAN class="toctext">24.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-107"><A href="#27.09.2010"><SPAN class="tocnumber">2.2.90</SPAN><SPAN class="toctext">27.09.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-108"><A href="#01.10.2010"><SPAN class="tocnumber">2.2.91</SPAN><SPAN class="toctext">01.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-109"><A href="#04.10.2010"><SPAN class="tocnumber">2.2.92</SPAN><SPAN class="toctext">04.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-110"><A href="#05.10.2010"><SPAN class="tocnumber">2.2.93</SPAN><SPAN class="toctext">05.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-111"><A href="#06.10.2010"><SPAN class="tocnumber">2.2.94</SPAN><SPAN class="toctext">06.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-112"><A href="#07.10.2010"><SPAN class="tocnumber">2.2.95</SPAN><SPAN class="toctext">07.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-113"><A href="#08.10.2010"><SPAN class="tocnumber">2.2.96</SPAN><SPAN class="toctext">08.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-114"><A href="#11.10.2010"><SPAN class="tocnumber">2.2.97</SPAN><SPAN class="toctext">11.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-115"><A href="#12.10.2010"><SPAN class="tocnumber">2.2.98</SPAN><SPAN class="toctext">12.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-116"><A href="#13.10.2010"><SPAN class="tocnumber">2.2.99</SPAN><SPAN class="toctext">13.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-117"><A href="#14.10.2010"><SPAN class="tocnumber">2.2.100</SPAN><SPAN class="toctext">14.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-118"><A href="#15.10.2010"><SPAN class="tocnumber">2.2.101</SPAN><SPAN class="toctext">15.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-119"><A href="#17.10.2010"><SPAN class="tocnumber">2.2.102</SPAN><SPAN class="toctext">17.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-120"><A href="#18.10.2010"><SPAN class="tocnumber">2.2.103</SPAN><SPAN class="toctext">18.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-121"><A href="#19.10.2010"><SPAN class="tocnumber">2.2.104</SPAN><SPAN class="toctext">19.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-122"><A href="#20.10.2010"><SPAN class="tocnumber">2.2.105</SPAN><SPAN class="toctext">20.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-123"><A href="#21.10.2010"><SPAN class="tocnumber">2.2.106</SPAN><SPAN class="toctext">21.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-124"><A href="#22.10.2010"><SPAN class="tocnumber">2.2.107</SPAN><SPAN class="toctext">22.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-125"><A href="#23.10.2010"><SPAN class="tocnumber">2.2.108</SPAN><SPAN class="toctext">23.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-126"><A href="#24.10.2010"><SPAN class="tocnumber">2.2.109</SPAN><SPAN class="toctext">24.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-127"><A href="#25.10.2010"><SPAN class="tocnumber">2.2.110</SPAN><SPAN class="toctext">25.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-128"><A href="#26.10.2010"><SPAN class="tocnumber">2.2.111</SPAN><SPAN class="toctext">26.10.2010</SPAN></A></LI><LI class="toclevel-3 tocsection-129"><A href="#27.10.2010"><SPAN class="tocnumber">2.2.112</SPAN><SPAN class="toctext">27.10.2010</SPAN></A></LI><LI class="toclevel-1 tocsection-130"><A href="#Protocols"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-2 tocsection-131"><A href="#Molecular_Biology"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Molecular Biology</SPAN></A></LI><LI class="toclevel-2 tocsection-132"><A href="#In_vivo_Measurement"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">In vivo Measurement</SPAN></A></LI><LI class="toclevel-3 tocsection-133"><A href="#Bacterial_Cell_Growth"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Bacterial Cell Growth</SPAN></A></LI><LI class="toclevel-3 tocsection-134"><A href="#Fluorescence_Measurement"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">Fluorescence Measurement</SPAN></A></LI><LI class="toclevel-2 tocsection-135"><A href="#In_vitro_Translation_2"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">In vitro Translation</SPAN></A></LI><LI class="toclevel-2 tocsection-136"><A href="#In_vitro_Transcription_2"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">In vitro Transcription</SPAN></A></LI><LI class="toclevel-3 tocsection-137"><A href="#Buffers"><SPAN class="tocnumber">3.4.1</SPAN><SPAN class="toctext">Buffers</SPAN></A></LI><LI class="toclevel-3 tocsection-138"><A href="#Sample_Preparation"><SPAN class="tocnumber">3.4.2</SPAN><SPAN class="toctext">Sample Preparation</SPAN></A></LI><LI class="toclevel-3 tocsection-139"><A href="#Cary_Eclipse"><SPAN class="tocnumber">3.4.3</SPAN><SPAN class="toctext">Cary Eclipse</SPAN></A></LI><LI class="toclevel-1 tocsection-140"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><TABLE border="0" cellpadding="0" cellspacing="0" width="100%" style="background: transparent;"><TBODY><TR><TD id="navtd" align="center"><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Project" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Project</P></A><A href="https://2010.igem.org/Team:TU_Munich/Project" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Project</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Project"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Learn more about our project</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">What is our project about? What is the idea behind our project? This page is a good starting point if you are interested in our project and want to learn more about it.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Software" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Software</P></A><A href="https://2010.igem.org/Team:TU_Munich/Software" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Software</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Software"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Build your own network</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">To demonstrate our vision about Biobrick networks, we developed a software, that enables you to interconnect Biobricks of your choice to your personal network. It will generate a DNA sequence that encodes all your logical connections. Just give it a try!</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Modeling" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Modeling</P></A><A href="https://2010.igem.org/Team:TU_Munich/Modeling" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Modeling</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Modeling"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Understand our RNA devices</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">On this page we show how we analyzed and estimated the switching process of our RNA devices. This is a basic requirement for the creation of better RNA devices.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Lab" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Lab</P></A><A href="https://2010.igem.org/Team:TU_Munich/Lab" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Lab</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Lab"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Learn more about our experiments</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">This page provides an overview as well as detailed information about the experiments and measurements we performed. Protocols as well as our daily lab book will clarify any question about our lab proceedings.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Parts" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">BioBricks</P></A><A href="https://2010.igem.org/Team:TU_Munich/Parts" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">BioBricks</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Parts"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Our contribution to the registry</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">Our project required the analysis and modification of existing Biobricks from the part registry as well as the creation of new Biobricks. This page contains more information about these steps.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Team" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Team</P></A><A href="https://2010.igem.org/Team:TU_Munich/Team" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Team</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Team"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Get to know us</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">Who are we? Where are we from? What do we do in our spare time? Here you can find the answers to all these questions.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Glossary" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Glossary</P></A><A href="https://2010.igem.org/Team:TU_Munich/Glossary" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Glossary</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Glossary"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Get the meaning of a term</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">To understand our project description more easily, we put together a list of terms that might be unknown or forgotten. Here you will find explanations concerning these terms.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/PartnersAndPress" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P></A><A href="https://2010.igem.org/Team:TU_Munich/PartnersAndPress" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Partners &amp; Press</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/PartnersAndPress"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Thanks a lot to our supporters</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">This project would not have been possible without the help and support of various people, institutions and companies. We want to say thank you to all of them!</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/BeyondTheLab" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P></A><A href="https://2010.igem.org/Team:TU_Munich/BeyondTheLab" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Beyond the Lab</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/BeyondTheLab"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">iGEM is more than biology</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">The iGEM competition not only aims to drive the development of new Biobricks. It also encourages to think about responsibilities, safety and risks in the field of biotechnology. Furthermore, iGEM is about working together and helping other teams in various aspects. Read more how we want to contribute to these goals.</SPAN></DIV></DIV></A></DIV></TD></TR></TBODY></TABLE><DIV class="wiki_div" style="width: 760px; margin: 10px; margin-right: 0px; margin-top: 0px; padding: 30px; padding-bottom: 65px; border: black 1px solid; border-right:0; background: url(https://static.igem.org/mediawiki/2010/b/b3/Team_TU_Munich2010_Images_WikiBackground.jpg) #EDE8E2 no-repeat;"><H1><SPAN class="mw-headline" id="Experiment_Design">Experiment Design</SPAN></H1><P>In this section we do not only want to present the experiments and results we gained but also to encourage you to evaluate your own switch based on the protocols and general procedure on how to evaluate basic parameters of a switch. In theory, every terminator can be turned into a switch with minor modifications and the right signals which are based on individual applications. While the principle of how to turn a terminator into a switch is explained in detail <A href="https://2010.igem.org/Team:TU_Munich/Project" class="external text" rel="nofollow">here</A>, experimental setups and protocols are provided in the following. Due to time and equipment limitations we could not perform all the experiments we planned but next to the hope that another iGEM team might proceed with our project we would also like to encourage you to design and test some basic switches on which you can base a complete, tightly regulated network. 

</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><P align="justified">

The complexity of our experimental setups vary, since we planned to characterize an individual switch with one exemplary signal on all relevant levels: Starting from the most general, complicated but also relevant level, <I>in vivo</I> measurements we approached to testing different switches on each smaller scale: We developed setups for <I>in vitro</I> translation which can be done without much effort following the <I>in vitro</I> measurements and also provide detailed description of <I>in vitro</I> transcription verification providing an inside to the molecular functionality of our basic idea. We do not see the methods we used here as the gold standard for bioLOGICS evaluation and encourage you to include your own ideas as well as check in our outlook section where we suggest experiments we could not do during the limited iGEM 2010 time. Together with our <A href="https://2010.igem.org/Team:TU_Munich/Parts" class="external text" rel="nofollow">Biobrick submissions</A> this year, we offer a complete set for switch evaluation on all cellular levels.

Most measurements are based on fluorescence reporters which provide easy handling, fast output and are well studied. Next to the fluorescent proteins GFP and mCherry we used <I>in vivo</I>, a malachite green binding aptamer serves as a reporter <I>in vitro</I> providing a reliable fluorescent output upon antitermination. 

Most setups up to now were only used to evaluate switches with an default state &quot;off&quot; which are applied for AND/OR devices. In principle the same methods can be used for NOT devices which are based on a switch with an default state &quot;off&quot;. Again, time limitation circumvented further tested from our team but we hope that further studies can be done in the future. 
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H2><SPAN class="mw-headline" id="In_vivo_Measurements"><I>In vivo</I> Measurements</SPAN></H2><P><I>In vivo</I> measurements have the highest complexity compared to any other experimental set-up. Different parameters and circumstances deriving from both the cellular environment as well as technical considerations like scatter have to be taken into account. Nevertheless, the measurements are essential, as our switches should finally work inside cells to fulfill our vision of an intracellular logic network. This year's submitted Biobricks provide you with a basic kit of plasmids which allow a quick beginning of the measurements. 

</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><H3><SPAN class="mw-headline" id="Design">Design</SPAN></H3><P>For the measurements <I>in vivo</I> we decided to use an expression cassette consisting of Green Fluorescent Protein (GFP) coding sequence upstream of the switch and another fluorescent protein coding sequence downstream of it. Both protein coding sequence share the same ribosome binding site allowing the usage of the GFP as an internal control in measurements. Since the spectra should not overlap and to avoid FRET as well as an pure overlap of the spectra, we settled on the usage of red fluorescent protein variants, namely mRFP1 in the first try and mCherry in an modified variant of the pSB1A10 vector. 

While the GFP fluorescence can be used to normalize the measurements, the RFP fluorescence serves as a reporter to detect and evaluate termination and antitermination. To stimulate the expression of the fluorescent proteins, we took advantage of the pBAD promoter family (sensitive towards arabinose). The signal upon which the antitermination events and therefore switching relies on were under the control of an IPTG inducible promoter. We went with this well-established pair of controllable promoters to deliver an easy setup in the beginning, like described <A href="https://2010.igem.org/Team:TU_Munich/Software" class="external text" rel="nofollow">here</A>, every sort of input may later be combined with our basic switching units. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">general measurement principle</DIV></DIV></DIV></DIV><P>The GFP internal control carries the advantage that errors in the measurement set can be detected easily. Lack of arabinose or promoter insensitivity can be recognized as well as problems with the fluorescence measurement itself. Plus, it allows normalizing measurements to compare different preparations in relation to each other.  

Upon binding of a signal to the terminator switch, termination is circumvented and the reporter protein behind the switch can be translated. In the experimental setup presented here, this will result in an RFP expression, but again, every protein or DNA-encoded element in general may be used as an output. Since the RFP fluorescence spectra do not overlap with GFP it offers an easy possibility to evaluate the effect of signal induction. Next to GFP fluorescence, RFP fluorescence will show up.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:377px;"><DIV class="thumbcaption">schematic estimated fluorescence spectra</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Construction_and_Cloning">Construction and Cloning</SPAN></H3><P>In a first try we cloned a measuring construct based on pSB1A10. The resulting plasmid, nicknamed pMonsterplasmid due to its size was tested in the fluorescent measurements described <A href="https://2010.igem.org/Team:TU_Munich/Lab#Switch_evaluation_in_vivo" class="external text" rel="nofollow">below</A>. Unfortunately after two months of cloning we had to recognize that the plasmid in use did not work for us (see also <A href="/Team:TU_Munich/Parts#Falsified_Parts" title="Team:TU Munich/Parts">pSB1A10 Falsification</A>). 
So after the first unsuccessful attempts we decided to reclone the system, substituting RFP to mCherry, a dsRED derivative with a spectrum in the far red, and adding arabinose inducible promoters in front of both fluorescent proteins to guarantee stable and comparable expression of both proteins
</P><P>
To control the expression of the switch, the particular DNA sequence itself is under the control of an IPTG dependent promoter. In the future we want our networks to be able to respond to a variety of external signals like small metabolites, ions or whatever can be found in the parts registry. For basic switch evaluation, an established and well-working system like the <I>lac</I>-operon was chosen to avoid side-effects of less well-characterized promoters.
</P><H3><SPAN class="mw-headline" id="Measurements_based_on_submitted_Biobricks">Measurements based on submitted Biobricks</SPAN></H3><P>The Biobricks BBa_K494001-BBa_K494006 are constructed for easy design of a switch-evaluation system. Detailed information can be found <A href="https://2010.igem.org/Team:TU_Munich/Parts#Plasmids" class="external text" rel="nofollow">here</A>.
</P><H3><SPAN class="mw-headline" id="Switch_evaluation_in_vivo">Switch evaluation <I>in vivo</I></SPAN></H3><P>To evaluate the switching efficiency, output with and without signal needs to be monitored. In this case, GFP fluorescence (internal control) will always appear upon arabinose induction, while RFP/mCherry fluorescence is only present upon binding of a signal and occurring antitermination. 
Upon induction with arabinose a rise of GFP expression can be seen. To monitor changes in gene expression we used a fluorimeter and measured fluorescence of whole living cells. While this approach provides easy handling and monitoring, too much scattering has to be carefully avoided: the cell density should not exceed an OD<SUB>600</SUB> of 0.05. RFP/mCherry emission should be visible only in case of a working switch or inefficient termination.

For evaluation of the measuring plasmid itself we incorporated a positive control in every measurement. A random sequence in between GFP and RFP/mCherry was chosen in a corresponding length instead of a terminator. An increase in both GFP and mCherry was detectable in comparable amounts after quantum yield correction, showing the measuring plasmid to beworking nicely. While the positive control may be the same for all evaluated devices, the negative control has to be specific for every switch or terminator, respectively. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:377px;"><DIV class="thumbcaption">Bacterial cultures after incubation of 16h</DIV></DIV></DIV></DIV><P>
The negative control contained the evaluated switch without any possibility for induction of the corresponding signal. Thereby the switch's function is limited to termination, leading to no detectable RFP/mCherry fluorescence. By definition every switch type has to be tested using a negative control without a corresponding signal, since termination efficiency may vary depending on the terminator itself, cell strain and general growth conditions. We recommend to chose your terminator of choice and evaluate it using the provided plasmids. 
In our experimental part we evaluated terminators based on the regulatory unit of the tryptophan (Trp-Term) and histidine (His-Term) operons. Those synthetic operons are regulated based on the principle of attenuation, a terminator in front of genes involved in amino acid biosynthesis avoids transcription until environmental stimulis suggest a lack of those amino acids. Since both sequences are known to be regulated by changes in secondary structure, those two attenuators became the basis for our designed switches. 
The terminators we tested can be found in the Parts Registry. With the construction of the backbone <A href="http://partsregistry.org/Part:BBa_K494001" class="external text" rel="nofollow">BBa_K494001</A>, potential switches and signals can be easily subcloned in two steps and tested. <A href="http://partsregistry.org/Part:BBa_K494002" class="external text" rel="nofollow">BBa_K494002</A> was constructed as a positive control, producing maximal mCherry fluorescence which may be used to characterize terminator and switch efficiency. <A href="http://partsregistry.org/Part:BBa_K494003" class="external text" rel="nofollow">BBa_K494003</A> and <A href="http://partsregistry.org/Part:BBa_K494004" class="external text" rel="nofollow">BBa_K494004</A> carry the His-Terminator with and without the corresponding signal, <A href="http://partsregistry.org/Part:BBa_K494005" class="external text" rel="nofollow">BBa_K494005</A> and <A href="http://partsregistry.org/Part:BBa_K494006" class="external text" rel="nofollow">BBa_K494006</A> being the same for Trp-Terminator. 
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H2><SPAN class="mw-headline" id="In_vitro_Translation"><I>In vitro</I> Translation</SPAN></H2><P>To go more into detail, the next complexity level is to study the effect of switches on a translational level. <I>In vitro</I> measurements with <I>E. coli</I> lysate make the fluorescence signals independent of cell growth and physical or biological factors like cell density or growth stadium.

</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><H3><SPAN class="mw-headline" id="Design_2">Design</SPAN></H3><P>Since the same constructs can be used both for <I>in vivo</I> and <I>in vitro</I> translation, no additional cloning effort is needed. This implements, that the Biobricks we <A href="https://2010.igem.org/Team:TU_Munich/Plasmids" class="external text" rel="nofollow">provided this year</A>, can again be used as the groundwork for constructing vectors for measurements. 
Reporter proteins GFP and mCherry are well expressed <I>in vitro</I>, the limiting factors are mostly the capacity of a kit versus the maturation time of fluorescent proteins. Since we used a fast-folding GFP variant and mCherry, which was characterized with a maturation time of 15 minutes by Tsien and Coworkers, the problem should be minimized. Alternative tags may be considered, a major advantage of measuring translation <I>in vitro</I> may be the use of non-cell permeable tags for switch evaluation.
</P><H3><SPAN class="mw-headline" id="Measurements">Measurements</SPAN></H3><P>We used the cell-free <I>E. coli</I> S30 extract system for circular DNA provided by Promega<SUP><A href="/Team:TU_Munich/Lab#ref1" title="Team:TU Munich/Lab">[1]</A></SUP>, which is prepared by modifications of the Method Zubay <I>et al.</I><SUP><A href="/Team:TU_Munich/Lab#ref2" title="Team:TU Munich/Lab">[2]</A></SUP>. The characterization of the kit can be obtained from the <A href="http://partsregistry.org/Cell-free_chassis/Commercial_E._coli_S30" class="external text" rel="nofollow">Parts Registry</A>. 
Experiments were performed at 37°C with an amount of approximately 1 µg plasmid in a reaction volume of 50 µL. Fluorescence was followed over time in a jasco fluorolog with wavelength corresponding to those used <I>in vivo</I>. 
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H2><SPAN class="mw-headline" id="In_vitro_Transcription"><I>In vitro</I> Transcription</SPAN></H2><P>To monitor transcription termination and antitermination on a the molecular level, <I>in vitro</I> transcription of individual switches and their response to signals offer an elegant way for fast and easy prove of principle. Most side effects occurring in a complex environment given in a cell or a cell lysate do not arise here. Another major advantage of <I>in vitro</I> transcription experiments is the possibility to test many signals for one switch to optimize antitermination efficiency and binding specificity without much cloning work. Data gained by <I>in vitro</I> transcription experiments can be used to improve switches and signals for <I>in vivo</I> usage. 

</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><P align="justified">

Since we are working on a totally new principle of transcriptional control, we used this approach beside the above mentioned advantages for easy variation of different variables like the length of the core unit and the switch to signal ratio. 
To study the switches on a transcriptional level offers the advantage, to reduce interference and possible artifacts to a minimum. Since we are not sure how cellular mechanisms like degradation of RNases or interacting factors as well as molecular crowding influence our systems, <I>in vitro</I> transcription was also used as a minumum system from which more complexity can derive.

Working with in vitro systems also has the advantage that an input is not needed anymore and the output can also be generated easily. We used <B>two readouts</B> with <B>two different transcription systems</B> to check and investigate our devices: First, we used an <A href="https://2010.igem.org/Team:TU_Munich/Parts" class="external text" rel="nofollow">malachitegreen-binding aptamer</A> for an fluorescence output and second, we simply put our reaction educts on an denaturing acrylamide-gel to check for RNA varying in length. As for two different transcription systems we used on the one hand <I>E. coli</I>-RNA Polymerase (RPO) based transcription since the aim is to apply the so gained results <I>in vivo</I> and on the other hand T7 based transcription which is well established through literature and delivers good RNA yields.
</P><H3><SPAN class="mw-headline" id="T7_RNA_polymerase"> T7 RNA polymerase </SPAN></H3><DIV align="justify"><P>The T7 RNA polymerase is known for satisfying RNA yields together with easy handling. In our approach we had PCR amplified, double stranded switches with an malachitegreen binding aptamer following after a T7 terminator which was constructed to function as a switch. Different signals were tested varying in length of the specificity site and the triggering unit. 
 
For in vitro expression the T7 RNA Polymerase requires a double stranded promotor region at the beginning of the DNA template but is otherwise capable of handling single stranded DNA, so a sense strain corresponding to the T7 promoter region was added. Transcription is more effective with double stranded DNA as template. Apart from that, no more requirements are needed in theory which makes the evaluation of many signals especially easy. Since we ordered the signal sequences we tested we chose the cheaper way in the beginning by using single stranded signals with corresponding sense T7 pieces and switched to double stranded constructs after narrowing down the most promising switch/signal pairs. Later on we also used double stranded signals and switches since transcription rates are higher with those. 

As a positive control, the malachite green binding aptamer right behind the T7 promoter was used. Transcription  proceeds without termination and the maximal fluorescence intensity should be gained. 
 Transcription termination can also be estimated by measuring just the switch without interfering signals. Since upon transcription of a signal sequence, less RNA Polymerase is available, the transcription rate of the switch and therefore the fluorescence output is reduced by merely adding the signal. Therefore randomly chosen short sequences in the range of the tested signals were added to the negative control. 
</P></DIV><H3><SPAN class="mw-headline" id="E._coli_RNA_polymerase"><I>E. coli</I> RNA polymerase </SPAN></H3><P>In comparison to the T7 RNA Polymerase the <I>E. coli</I> RNA Polymerase requires slightly more sophisticated proceedings when it comes to the design of switches and handling of the enzyme. The biggest in our case was to store it properly since the only -80°C fridge was in another building, so make sure you have a big supply of dry ice ready if you encounter the same problem. 
E. coli RPO was ordered saturated with σ70-factor.
</P><H3><SPAN class="mw-headline" id="Denaturing_Polyacrylamide_gel_electrophoresis"> Denaturing Polyacrylamide gel electrophoresis </SPAN></H3><DIV align="justify"><P>Polyacrylamide gel electrophoresis (PAGE) was used for evaluation of termination and switching efficiency. Gels containing 15 % acrylamide and 6 M urea were used for separation of terminated and readthrough RNAs. The same constructs as designed for the malachite green binding aptamer were used. 

Polyacrylamide gels separate RNA and DNA according to their size in an electric field. Since the negative charge equals the size of nucleotides in the RNA/DNA, the number of base pairs can be compared between two samples often with one base pair resolution. Since RNA forms three-dimensional structures, the samples are preheated and run in 6 or 7 M urea. The polyacrylamide gel is stained in SybrGold afterwards which binds to both single and double stranded DNA and RNA. A Dnase digestion was applied before running the samples to avoid confusion caused by DNA templates.

Denaturing PAGE is a simple yet elegant way to check for transcription efficiency and termination rates. Since it is a very direct way and it provides a simple yet clear readout, we used it as another method beside the more sophisticated malachitegreen binding assay to evaluate and characterize our switch. Equipment for denaturing PAGE can be found in nearly every biochemical lab, so this method also applies for an easy controlexperiment. 
</P></DIV><H3><SPAN class="mw-headline" id="Malachite_green_assay"> Malachite green assay </SPAN></H3><DIV align="justify"><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Emission spectra of malachite green; A: without signal-RNA, B: with signal-RNA</DIV></DIV></DIV><P>In this year's DNA submission we contribute the <A href="/Team:TU_Munich/Parts#Malachite_Green_Binding_Aptamer_-_BBa_K494000" title="Team:TU Munich/Parts"> malachite green binding aptamer</A>  which can be used as a transcription reporter in <I>in vitro</I> transcription experiments. 
Malachite green is a dye with a negligible fluorescence in solution but undergoes a dramatic increase about 3000 times if bound by the RNA aptamer making it an exceptional good marker. Since the binding is very specific, transcription in dependence of a signal can be monitored by measuring the fluorescence of malachite-green over time if the aptamer is located behind the switch. Transcription of the aptamer will only take place after anti-termination by a signal. An increase should be visible over time. Other triphenyldyes are also recognized with weaker effects on the fluorescence but may also serve as reporters if the emission or excitation of malachite green does not fit the experimental setup. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Description of the malachit green assay. Antitermination allows the polymerase to produce the malachite green aptamer</DIV></DIV></DIV></DIV><P>Malachite green binding can be used to follow RNA transcription over time, a rise in the fluorescence is then detectable. Fluorescence marker of specific RNA structures are still rare, so the malachite green binding aptamer provides one of the only possibilities to continuously monitor transcription reactions. In comparison to PAGE, kinetics can be taken, while with PAGE only end point estimations can be made. This makes the malachite green binding aptamer a valuable tool to study <I>in vitro</I> transcription in general and the principles underlying our switch in principle. 
</P></DIV><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H1><SPAN class="mw-headline" id="Lab_Book">Lab Book</SPAN></H1><H2><SPAN class="mw-headline" id="Explanations">Explanations</SPAN></H2><P>In the following we present an overview regarding our work in the lab. For easier understanding we summarized the work of each week using colored boxes. To get a better overview we used the following color code for the boxes:
</P><TABLE><TBODY><TR><TD><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"> </DIV> The red box represents general cloning steps that were required for our measurements. See the <A href="/Team:TU_Munich/Lab#Molecular_Biology" title="Team:TU Munich/Lab"> protocol section</A> for further details.
</TD></TR><TR><TD><DIV style="color: #0000CC; border: #0000CC 1px solid; background-color: #8888FF; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"> </DIV> The blue box indicates <I>in vivo</I> measurements which are described <A href="/Team:TU_Munich/Lab#In_vivo_Measurements" title="Team:TU Munich/Lab"> here</A>.
</TD></TR><TR><TD><DIV style="color: #00BB00; border: #00BB00 1px solid; background-color: #55FF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"> </DIV> The green box indicates <I>in vitro</I> measurements relying on <I>in vitro</I> transcription and malachite green measurements. Details can be found <A href="/Team:TU_Munich/Lab#In_vitro_Transcription" title="Team:TU Munich/Lab"> here</A>.
</TD></TR><TR><TD><DIV style="color: #BBBB00; border: #BBBB00 1px solid; background-color: #FFFF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; float: left;"> </DIV> The yellow box represents measurements done with an <I>in vitro</I> translation kit and is described in more details <A href="/Team:TU_Munich/Lab#In_vitro_Translation" title="Team:TU Munich/Lab"> here</A>.
</TD></TR></TBODY></TABLE><P>
To learn more about the work and results of a specific week, just click on the according week number. You will find detailed notes on our daily lab work. We present these notes in an unedited form as a record of our work,  for for processed results please check the <A href="/Team:TU_Munich/Project#Results" title="Team:TU Munich/Project"> results section on our project page</A>.
</P><H2><SPAN class="mw-headline" id="Chronological_Lab_Book">Chronological Lab Book</SPAN></H2><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week01
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="08.04.2010">08.04.2010</SPAN></H3><P>Flo &amp; Philipp
</P><P><B>PCR</B></P><UL><LI>samples:
</LI><LI> R0011_His
</LI><LI> R0011_Trp
</LI><LI> Control
</LI></UL><LI>protocol: <A href="/Team:TU_Munich/Lab#Molecular_Biology" title="Team:TU Munich/Lab"> protocol</A></LI><LI>templates: purified PCR products from 5.2.2010
</LI><LI>primer G1004/1005
</LI><LI>polymerase: Taq
</LI><LI>programm: igempcr
</LI><P><B>Purification of PCR products with QIAquick PCR purification Kit </B></P><UL><LI>protocol followed. exceptions: DNA-binding/unbinding with 3min 6000rpm followed by 60sec full speed
</LI></UL><P><B>2% Agarose Gel</B></P><UL><LI>in 1x TAE.
</LI></UL><H3><SPAN class="mw-headline" id="09.04.2010">09.04.2010</SPAN></H3><P>Philipp &amp; Flo
</P><P><B>Gel of PCR products from 08.04.2010</B></P><UL><LI>loaded: 10 µL sample+2 µL 6x GLD, 4/2 µL LMW standard
</LI><LI> 110 V, 90 min
</LI><LI>stained with Sybrgold, 20 min, 1:10.000 dilution in TAE
</LI><LI>Standard - Control - R0011_His - R0011_Trp - Standard(=low molecular weight (see <A href="/Team:TU_Munich/Lab#Molecular_Biology_Lab_Protocols" title="Team:TU Munich/Lab">Team:TU_Munich/Lab#Molecular_Biology       Lab_Protocols</A>))
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week02
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="15.04.2010">15.04.2010</SPAN></H3><P>Philipp &amp; Flo
<A href="http://web.e14.physik.tu-muenchen.de/igem/index.php/Team:TU_Munich/Lab#Molecular_Biology" class="external text" rel="nofollow">PCR PCR</A> of B0014 and R0011
</P><H3><SPAN class="mw-headline" id="16.04.2010">16.04.2010</SPAN></H3><P>Philipp &amp; Flo</P><UL><LI><B>Purification</B> of PCR products from <A href="/wiki/index.php?title=15.04.2010&amp;action=edit&amp;redlink=1" class="new" title="15.04.2010 (page does not exist)">15.04.2010</A> using <A href="/Team:TU_Munich/Lab#Molecular_Biology_QIAquick_purification_Kit" title="Team:TU Munich/Lab">QIA kit</A></LI></UL><UL><LI><B>Concentrations</B> measured with nanodrop:</LI></UL><TABLE width="200" cellspacing="1" cellpadding="1" border="1" align="center"><TBODY><TR><TD> B0014
</TD><TD> 2.5 ng/µL</TD></TR><TR><TD> R0011</TD><TD> 27.5 ng/µL</TD></TR></TBODY></TABLE><CENTER> --&gt; worked for R0011, not for B0014 </CENTER><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">PCR</A></B>of B0014</LI><LI>Purification with the <A href="/Team:TU_Munich/Lab#Molecular_Biology_ZYMO_RESEARCH_DNA_Clean.26Concentration_Kit" title="Team:TU Munich/Lab">Zymo Kit</A>, Elution in 20 µL H2O
</LI><LI>Concentration measured with nanodrop, 17.5 ng/µL --&gt; worked</LI></UL><UL><LI><B>Digestions</B> of:</LI></UL><TABLE width="618" cellspacing="1" cellpadding="1" border="1" align="center" style=""><TBODY><TR><TD><B>template</B></TD><TD><B>restriction enzymes (biobrick assembly)</B></TD></TR><TR><TD> B0014 (from Christoph, verified PCR products, 21 ng/uL)</TD><TD> EcoRI, PstI</TD></TR><TR><TD> R0011 (from PCR [15042010], 27.5 ng/µL</TD><TD> SpeI</TD></TR><TR><TD> HisSig (1:100 dilution)</TD><TD> XbaI</TD></TR><TR><TD> TrpSig (1:100 dilution)</TD><TD> XbaI</TD></TR><TR><TD> psB1K3 (with RFP insert, from HiWiPhilipp, 81 ng/µL)
</TD><TD> EcoRI, PstI
</TD></TR></TBODY></TABLE><CENTER>5 µL template used for each setup. <A href="/Team:TU_Munich/Lab#Molecular_Biology_Restriction" title="Team:TU Munich/Lab">protocol</A> followed</CENTER><UL><LI><B>Gel</B> for purification of the cleaved plasmid 
</LI><LI>2% Agarose in 1x TAE 
</LI><LI>120 V, 90 min 
</LI><LI><A href="/Team:TU_Munich/Lab#Molecular_Biology_stain" title="Team:TU Munich/Lab">stained</A> with SybrGold
</LI><LI>digestion, digestion, <A href="/Team:TU_Munich/Lab#Molecular_Biology_standards" title="Team:TU Munich/Lab">1 kb ladder</A></LI><LI>Digestion worked (partly). band at 2000 bp (backbone) cut</LI></UL><UL><LI><B>Purification of DNA from Gel</B></LI><LI>using the <A href="/Team:TU_Munich/Lab#Molecular_Biology_ZYMO_RESEARCH_DNA_Clean.26Concentration_Kit" title="Team:TU Munich/Lab">Zymo Kit</A></LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B> of HisSig/TrpSig with R0011in 2 reactions</LI></UL><TABLE width="617" cellspacing="1" cellpadding="1" border="1" align="center" style=""><TBODY><TR><TD><B>used Volume</B></TD><TD><B>approx. concentration*</B></TD></TR><TR><TD> HisSig</TD><TD> 6 µL</TD><TD> 7 ng/µL</TD></TR><TR><TD> TrpSig</TD><TD> 6 µL</TD><TD> 5 ng/µL</TD></TR><TR><TD> R0011</TD><TD> 3 µL</TD><TD> 6 ng/µL</TD></TR></TBODY></TABLE><PRE>* approximated from the amount used in the digestion before
</PRE><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week03
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="19.04.2010">19.04.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">PCR</A></B>of R0011-TrpSig and R0011-HisSig</LI><LI>Purification with the <A href="/Team:TU_Munich/Lab#Molecular_Biology_ZYMO_RESEARCH_DNA_Clean.26Concentration_Kit" title="Team:TU Munich/Lab">Zymo Kit</A>, Elution in 30 uL H2O
</LI><LI>Concentration measured with nanodrop: c(R0011-TrpSig)=20 ng/µL, c(R0011-HisSig)=12.5 ng/µl --&gt; worked</LI></UL><UL><LI><B>Gel</B> for analysis of ligation and PCR 
</LI><LI>2% Agarose in 1x TAE 
</LI><LI>110 V, 90 min 
</LI><LI><A href="/Team:TU_Munich/Lab#Molecular_Biology_stain" title="Team:TU Munich/Lab">stained</A> with SybrGold 1:10000 20 min
</LI><LI>pure R0011 PCR product used as control
</LI></UL><TABLE width="617" cellspacing="1" cellpadding="1" border="1" align="center" style=""><TBODY><TR><TD><B>LMW</B></TD><TD> 4 µl</TD></TR><TR><TD><B>R0011-TrpSig</B></TD><TD> 5 µL</TD></TR><TR><TD><B>R0011-HisSig</B></TD><TD> 5 µL</TD></TR><TR><TD><B>R0011</B></TD><TD> 5 µL</TD></TR></TBODY></TABLE><P>
Samples seem to have run further than the buffer/dye-Front! But: Ligation Products show bands at shorter lengths than R0011 alone --&gt; Ligation didn't work ?!?
</P><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B> of HisSig/TrpSig with R0011in 2 reactions</LI></UL><TABLE width="617" cellspacing="1" cellpadding="1" border="1" align="center" style=""><TBODY><TR><TD><B>used Volume</B></TD><TD><B>concentration</B></TD></TR><TR><TD> pSB1K3</TD><TD> 5 µL</TD><TD> 10 ng/µL (nanodrop)</TD></TR><TR><TD> B0014</TD><TD> 3 µL</TD><TD> 5 ng/µL approx.*</TD></TR></TBODY></TABLE><PRE>* approximated from the amount used in the digestion before
</PRE><H3><SPAN class="mw-headline" id="20.04.2010">20.04.2010</SPAN></H3><UL><LI><B>Gel</B> for analysis of ligation and PCR (repeat of <A href="/wiki/index.php?title=19.04.2010&amp;action=edit&amp;redlink=1" class="new" title="19.04.2010 (page does not exist)">yesterday's gel</A>)
</LI><LI>2% Agarose in 1x TAE 
</LI><LI>130 V, 75 min 
</LI><LI><A href="/Team:TU_Munich/Lab#Molecular_Biology_stain" title="Team:TU Munich/Lab">stained</A> with SybrGold 1:10000 60 min
</LI><LI>pure R0011 PCR product used as control
</LI><LI>Excision and purification of marked bands at 200 bp using QIA Kit, elution in 30 µl H2O</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">PCR</A></B>of excised and purified bands of R0011-TrpSig and R0011-HisSig</LI><LI>complete samples (30 µl) used as templates
</LI><LI>Purification with the <A href="/Team:TU_Munich/Lab#Molecular_Biology_ZYMO_RESEARCH_DNA_Clean.26Concentration_Kit" title="Team:TU Munich/Lab">Zymo Kit</A>, Elution in 30 uL H2O
</LI><LI>Concentrations of PCR-products: 0.5-1 ng/µl --&gt; Gel excision or PCR didn't work
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B> (Woehlke-Lab)
</LI><LI>8 µl of ligation product pSB1K3-B0014 to 50 µl XL-10 competent cells
</LI><LI>200 µl plated on a Kanamycin-containing Plate
</LI><LI>remaining 800 µl stored @4°C in S1-lab
</LI></UL><H3><SPAN class="mw-headline" id="21.04.2010">21.04.2010</SPAN></H3><UL><LI><B>Gel</B> for analysis of ligation and PCR (repeat of yesterday's gel) 
</LI><LI>2% Agarose in 1x TAE 
</LI><LI>110 V, 90 min 
</LI><LI><A href="/Team:TU_Munich/Lab#Molecular_Biology_stain" title="Team:TU Munich/Lab">stained</A> with SybrGold 1:10000 80 min 
</LI><LI>pure R0011 PCR product used as control 
</LI><LI>Excision and purification of marked bands at 200 bp using Zymo 5 Kit, elution in 20 µl H2O</LI></UL><P><A href="/File:TUM2010_100421beschriftet.gif%3F%3F" title="File:TUM2010 100421beschriftet.gif??">File:TUM2010 100421beschriftet.gif??</A></P><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">PCR</A></B>of excised and purified bands of R0011-TrpSig and R0011-HisSig</LI><LI>complete samples (20 µl) used as templates 
</LI><LI>Purification with the <A href="/Team:TU_Munich/Lab#Molecular_Biology_ZYMO_RESEARCH_DNA_Clean.26Concentration_Kit" title="Team:TU Munich/Lab">Zymo Kit</A>, Elution in 25 uL H2O 
</LI><LI>Concentrations of PCR-products: 
</LI><LI> R0011-TrpSig: 22.5 ng/µl
</LI><LI> R0011-HisSig: 9.5 ng/µl
</LI></UL><P>--&gt; worked!!!!!
</P><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B></LI><LI>7 Colonies picked and resuspended in 20 µl LB+Kana (each)
</LI><LI>PCR of 2 µl of each sample, 2 µl as negative control (Program: ColonyPCR, modified)
</LI><LI>15 µl of each sample mixed with 3 µl GLPn and loaded to Gel
</LI></UL><UL><LI>Overnight cultures: 
</LI><LI>remaining 18 µl of samples 1, 3, 6, and 7 added to 5 ml LB + kanamycin
</LI><LI>37°C on Shaker
</LI></UL><H3><SPAN class="mw-headline" id="22.04.2010">22.04.2010</SPAN></H3><UL><LI><B>Gel</B> for purification of PCR products R0011-TrpSig and R0011-HisSig (<A href="/wiki/index.php?title=21.04.2010&amp;action=edit&amp;redlink=1" class="new" title="21.04.2010 (page does not exist)">yesterday's result</A>) 
</LI><LI>2% Agarose in 1x TAE 
</LI><LI>110 V, 90 min 
</LI><LI><A href="/Team:TU_Munich/Lab#Molecular_Biology_stain" title="Team:TU Munich/Lab">stained</A> with SybrGold 1:10000 30 min 
</LI><LI>pure R0011 PCR product used as control 
</LI><LI>Excision and purification of marked bands at 200bp using Zymo 5 Kit, elution in 20 µl H2O</LI></UL><UL><LI>Miniprep
</LI><LI>Result: about 4 µg Plasmid
</LI></UL><H3><SPAN class="mw-headline" id="23.04.2010">23.04.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Restriction" title="Team:TU Munich/Lab">Digestion</A></B> of:</LI></UL><TABLE width="618" cellspacing="1" cellpadding="1" border="1" align="center" style=""><TBODY><TR><TD><B>template</B></TD><TD><B>template volume</B></TD><TD><B>restriction enzymes</B></TD><TD><B>Buffer</B></TD></TR><TR><TD> HisSig (1:100 dilution)</TD><TD> 5 µl</TD><TD> EcoRI, SpeI</TD><TD> NEB4</TD></TR><TR><TD> TrpSig (1:100 dilution)</TD><TD> 5 µl</TD><TD> EcoRI, SpeI</TD><TD> NEB4</TD></TR><TR><TD> psB1K3-B0014 from Miniprep (No 7, 35 ng/µl)
</TD><TD> 5 µl</TD><TD> EcoRI, XbaI
</TD><TD> NEB4</TD></TR></TBODY></TABLE><P> 
Incubated 90 min @ 37°C
</P><UL><LI><B>Gel</B> for purification of the cleaved plasmid 
</LI><LI>2% Agarose in 1x TAE (leftover from yesterday)
</LI><LI>140 V, 90 min 
</LI><LI><A href="/Team:TU_Munich/Lab#Molecular_Biology_stain" title="Team:TU Munich/Lab">stained</A> with SybrGold 40 min
</LI><LI>4 µl <A href="/Team:TU_Munich/Lab#Molecular_Biology_standards" title="Team:TU Munich/Lab">1 kb ladder</A>, 10 µl purified digestion + 2 µl GLPn, 10 µl purified digestion + 2 µl GLPn
</LI><LI>Digestion worked (partly). band at 2400 bp cut out</LI></UL><UL><LI><B>Purification of DNA from Gel</B></LI><LI>using the <A href="/Team:TU_Munich/Lab#Molecular_Biology_ZYMO_RESEARCH_Gel_DNA_Recovery_Kit" title="Team:TU Munich/Lab">Zymo Kit</A></LI><LI>elution in 25 µl H2O
</LI></UL><LI> A260/A230 and A260/A280 values were strange (see labbook)
</LI><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week04
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="26.04.2010">26.04.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of pSB1K3-B0014 with EcorI and XbaI
</LI><LI> 10 µl template (No1, 50 ng/µl)
</LI><LI> 5 µl BSA, 5 µl Buffer NEB#4
</LI><LI> 1 µl EcoRI, 1 µl XbaI
</LI><LI> 28 µl H2O
</LI><LI> 1.5 h @ 37°C
</LI><LI>Purification with Zymo5 Kit, elution in 20 µl H2O
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B> of Signals and PSB1K3-B0014
</LI><LI>3 µl of each sample, end volume 20 µl
</LI></UL><UL><LI>Preparation of Measurement Plasmid from Folder, Transformation
</LI><LI>Plate 1022, Spots 1E, 1G, 2A: pSB1A10 with different Inserts, all inserts are Zinc-finger constructs with about 1.6 kb
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B> of XL10 with Ligation Products (8 µl each) and pSB1A10 (2 µl each)
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Preparation_of_BioBricks_from_distribution_2008" title="Team:TU Munich/Lab">Preparation</A></B> of Measurement Plasmid from Folder, Transformation
</LI><LI>Plate 1022, Spots 1E, 1G, 2A: pSB1A10 with different Inserts, all inserts are Zinc-finger constructs with about 1.6 kb
</LI></UL><UL><LI>growing over night cultures of remaining PSB1K3-B0014-transformed cells
</LI><LI>2x 5 ml, 2x 1 ml
</LI></UL><H3><SPAN class="mw-headline" id="27.04.2010">27.04.2010</SPAN></H3><UL><LI>Plenty of cultures on both HisSig and TrpSig Ligation plates, but nothing on pSB1A10 plates! --&gt; repeat DNA extraction, ask Prof. Simmel for new Distribution
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B></LI><LI>7 Colonies picked and resuspended in 20 µl LB+Kana (each)
</LI><LI>PCR of 2 µl of each sample, 2 µl as negative control (Program: ColonyPCR, modified)
</LI><LI>10 µl of each sample mixed with 2 µl GLPn and loaded to Gel
</LI></UL><P>Many colonies with pSB1K3-B0014, not one with pSB1K3-Sig-B0014
</P><UL><LI>Miniprep of PSB1K3-B0014
</LI><LI>Samples I and II: 5 ml overnight cultures, centrifuged 10 min @ 3200 g, resuspended in 600 µl of the same culture
</LI><LI>Samples III and IV: 600 µl overnight cultures
</LI><LI>all samples mixed with 100 µl lysis buffer, Miniprep with Zyppy kit, each sample eluted in 50 µl H2O 
</LI><LI>Concentration measured (Nanodrop, LP=1mm, Factor 10, 4 µl sample)
</LI><LI>cI=61.5 ng/µl
</LI><LI>cII=33.5 ng/µl
</LI><LI>cIII=103 ng/µl
</LI><LI>cIV=108 ng/µl
</LI></UL><P>
--&gt; Better results for 600 µl cultures without centrifuging!!!
</P><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Preparation_of_BioBricks_from_distribution_2008" title="Team:TU Munich/Lab">Preparation</A></B> of Measurement Plasmid from Folder, Transformation
</LI><LI>Plate 1022, Spots 1E, 1F, 1G, 1H, 2A: pSB1A10 with different Inserts, all inserts are Zinc-finger constructs with about 1.6 kb
</LI></UL><H3><SPAN class="mw-headline" id="28.04.2010">28.04.2010</SPAN></H3><UL><LI>No colonies on plates from Yesterday's transformations, but on the older plates (from monday) some colonies appeared
</LI><LI>7 Colonies picked and resuspended in 20 µl LB0 (each)
</LI><LI>1&amp;2 from plate &quot;1E&quot;, 3&amp;4 from plate &quot;1G&quot;, 5,6&amp;7 from plate &quot;2A&quot;, 8 LB0
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B></LI><LI>PCR of 2 µl of each sample, 2 µl as negative control (Program: ColonyPCR, modified)
</LI><LI>15 µl of each sample mixed with 3 µl GLPn and loaded to Gel
</LI><LI>1% Agarose in 1xTAE, 95 V, after 50 minutes changed to 110 V
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of pSB1K3-B0014 with EcorI and XbaI
</LI><LI> 10 µl template (No1, 50 ng/µl)
</LI><LI> 5 µl BSA, 5 µl Buffer NEB#4
</LI><LI> 1 µl EcoRI, 1 µl XbaI
</LI><LI> 28 µl H2O
</LI><LI> 1.5 h @ 37°C
</LI><LI> heat inactivation 5min @60°C 
</LI></UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Dephosphorylation" title="Team:TU Munich/Lab">Dephosphorylation</A></B> of restricted vector
</LI><LI>Purification with Zymo5 Kit, elution in 20 µl H2O
</LI><LI>loaded on gel (with 4 µl GLPn) (Gel shown above)
</LI><LI>Gel excision with Zymo Kit
</LI><H3><SPAN class="mw-headline" id="29.04.2010">29.04.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of pSB1K3-B0014 with EcorI and XbaI
</LI><LI> 10 µl template (NoIV, 108 ng/µl)
</LI><LI> 5 µl BSA, 5 µl Buffer NEB#4
</LI><LI> 1 µl EcoRI, 1 µl XbaI
</LI><LI> 28 µl H2O
</LI><LI> 1.5 h @ 37°C
</LI><LI> heat inactivation 5min @60°C 
</LI></UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Dephosphorylation" title="Team:TU Munich/Lab">Dephosphorylation</A></B> of restricted vector
</LI><LI>Purification with Zymo5 Kit, elution in 20 µl H2O
</LI><LI>loaded on gel (with 4 µl GLPn)
</LI><P><A href="/File:X" title="File:X">File:X</A></P><UL><LI>Gel excision with Zymo Kit
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of R0011 with SpeI
</LI><LI> 10 µl template (R0011, X ng/µl)
</LI><LI> 5 µl BSA, 5 µl Buffer NEB#4
</LI><LI> 1 µl SpeI
</LI><LI> 29 µl H2O
</LI><LI> 1.5 h @ 37°C
</LI></UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B></LI><LI> 5 µl R0011 (S-digested) with 12 µl TrpSig or HisSig, respectively (X-digested)
</LI><LI> complete ligation (20 µl) loaded on Gel (with 4 µl GLPn)
</LI><UL><LI> Gel excision with Zymo Kit, eluted in 42 µl H2O
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B></LI><LI>50 µl XL-10 transformed with 2 µl of pSB1A10 prepared from IGem 2009 Distribution (13 µl left in pink Box @-20°C)
</LI></UL><H3><SPAN class="mw-headline" id="30.04.2010">30.04.2010</SPAN></H3><P>PCR R0011-HisSig and R0011-TrpSig --&gt; 13 ng/µl x 20 µl
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week05
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="04.05.2010">04.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of pSB1K3-B0014 with EcorI and XbaI
</LI><LI> 10 µl template (NoIII, 103 ng/µl)
</LI><LI> 5 µl BSA, 5 µl Buffer NEB#4
</LI><LI> 1 µl EcoRI, 1 µl XbaI
</LI><LI> 28 µl H2O
</LI><LI> 1.5 h @ 37°C
</LI><LI>Purification with Zymo5 Kit, elution in 15 µl H2O
</LI><LI>loaded on gel (with 3 µl GLPn) 
</LI></UL><UL><LI>Gel excision with Zymo Kit
</LI></UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of R0011-HisSig and R0011-TrpSig with EcorI and SpeI
</LI><LI> 10 µl template (PCR-product)
</LI><LI> 5 µl BSA, 5 µl Buffer NEB#4
</LI><LI> 1 µl EcoRI, 1 µl SpeI
</LI><LI> 28 µl H2O
</LI><LI> 1.5 h @ 37°C
</LI><LI>Purification with Zymo5 Kit, elution in 20 µl H2O
</LI><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B></LI><LI> 4 µl R0011-Signal (E/S-digested) with 4 µl pSB1K3-B0014 (E/X-digested)
</LI><LI>15 min @ RT, 20 min heat inactivation @ 65°C
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B></LI><LI>50 µl XL-10 transformed with 10 µl of Ligation mix
</LI></UL><UL><LI>Overrnight liquid cultures of pSB1A10-RFP made from
</LI><LI>1 and 2: picked clones from original plates from *<A href="/wiki/index.php?title=29.04.2010&amp;action=edit&amp;redlink=1" class="new" title="29.04.2010 (page does not exist)">Do 29.04.2010</A></LI><LI>3a: picked clone from copy plate from *<A href="/wiki/index.php?title=30.04.2010&amp;action=edit&amp;redlink=1" class="new" title="30.04.2010 (page does not exist)">Fr 30.04.2010</A></LI><LI>3b: resuspended clone N° 3 from *<A href="/wiki/index.php?title=30.04.2010&amp;action=edit&amp;redlink=1" class="new" title="30.04.2010 (page does not exist)">Fr 30.04.2010</A></LI></UL><P>--&gt; each in 600 µl LB+Carbenicillin (=Ampicillin) @37°C
</P><H3><SPAN class="mw-headline" id="05.05.2010">05.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Miniprep" title="Team:TU Munich/Lab">Miniprep</A></B> of pSB1A10; 4 samples (1, 2; 3a; 3b)
</LI><LI>eluted in 50 µl H2O each
</LI><LI>Concentrations:
</LI><LI> c1=37.5 ng/µl
</LI><LI> c2=56.5 ng/µl
</LI><LI> c3a=46.5 ng/µl
</LI><LI> c3b=30 ng/µl
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of pSB1A10 with EcorI and PstI, 4 samples (1, 2; 3a; 3b)
</LI><LI> 15 µl template 
</LI><LI> 5 µl BSA, 5 µl Buffer NEB#3
</LI><LI> 1 µl EcoRI, 1 µl PstI
</LI><LI> 23 µl H2O
</LI><LI> 1.5 h @ 37°C
</LI><LI> heat inactivation 5min @60°C 
</LI><LI>Purification with Zymo5 Kit, elution in 15 µl H2O
</LI><LI>loaded on gel (with 3 µl GLPn)
</LI></UL><P>  
Insert @ 1 kb as expected, but vector @ 2 kb and not @ 5 kb as expected!!!!
--&gt; Wrong Plasmid! Comparison to the <A href="http://partsregistry.org/cgi/assembly/plate_egel.cgi?id=615" class="external text" rel="nofollow">Gel in the registry</A> shows: The Distribution contains the wrong plasmid!
</P><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of HisTerm and TrpTerm with EcorI and PstI
</LI><LI> 5 µl template 
</LI><LI> 5 µl BSA, 5 µl Buffer NEB#3
</LI><LI> 1 µl EcoRI, 1 µl PstI
</LI><LI> 33 µl H2O
</LI><LI> 1.5 h @ 37°C
</LI></UL><UL><LI>Clones picked: 7 from each Plate (pSB1K3-R0011-TrpSig-Boo14 and pSB1K3-R0011-HisSig-Boo14)
</LI><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B></LI><LI>PCR of 2 µl of each sample, 2 µl as negative control (Program: ColonyPCR)
</LI><LI>15 µl of each sample mixed with 3 µl GLPn and loaded to Gel
</LI><LI>2% Agarose in 1xTAE, 130 V, 90 min
</LI></UL><H3><SPAN class="mw-headline" id="06.05.2010">06.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of pSB1K3 with EcorI and XbaI
</LI><LI> 20 µl template (sample III, 103 ng/µl)
</LI><LI> 5 µl BSA, 5 µl Buffer NEB#3
</LI><LI> 1 µl EcoRI, 1 µl XbaI
</LI><LI> 18 µl H2O
</LI><LI> 1.5 h @ 37°C
</LI><LI> heat inactivation 5min @60°C 
</LI><LI>loaded on gel (with 10 µl GLPn) in 4 lanes
</LI></UL><UL><LI>Gel excision with Zymo Kit (lanes 1&amp;2) and with Qiaquick Kit (lanes 3&amp;4)
</LI><LI> c1=4.5 ng/µl
</LI><LI> c2=3.5 ng/µl
</LI><LI> c3=2 ng/µl
</LI><LI> c1=7 ng/µl
</LI></UL><LI> A260/A230 and A260/A280 values were strange (see labbook)
</LI><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B></LI><LI> 4 µl R0011-Signal (E/S-digested) with 10 µl pSB1K3-B0014 (E/X-digested, from <A href="/wiki/index.php?title=23.04.2010&amp;action=edit&amp;redlink=1" class="new" title="23.04.2010 (page does not exist)">23.04.</A>)
</LI><LI>15 min @ RT, 20 min heat inactivation @ 65°C
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B></LI><LI>50 µl XL-10 transformed with 7 µl of Ligation mix
</LI></UL><H3><SPAN class="mw-headline" id="07.05.2010">07.05.2010</SPAN></H3><UL><LI>Clones picked: 7 from each Plate (pSB1K3-R0011-TrpSig-Boo14 and pSB1K3-R0011-HisSig-Boo14)
</LI></UL><P> 
---Too damn stupid to do a PCR!!!---</P><UL><LI> replated picked clones on new plates, incubated at RT
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week06
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="10.05.2010">10.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B> of picked clones from <A href="/wiki/index.php?title=07.05.2010&amp;action=edit&amp;redlink=1" class="new" title="07.05.2010 (page does not exist)">Fr 07.05.2010</A></LI><LI>PCR of 2 µl of each sample, 2 µl as negative control (Program: ColonyPCR, modified)
</LI><LI>15 µl of each sample mixed with 3 µl GLPn and loaded to Gel
</LI><LI>2% Agarose in 1xTAE, 120 V, 110 min
</LI><LI>stained in SybrSafe 50 min
</LI></UL><P>
Interpretation: 
Colonies contain an Insert with Prefix and Suffix, length is 200 bp. This is too short for the desired R0011-Signal-B0014 (245 or 247 bp) construct, but longer than B0014 (136 bp) which was the Insert in the digested vector. 
Possible explanation: Ligation worked, but not with R0011-Signal-construct but with R0011 <B>or</B> Signal. Which?
</P><TABLE cellspacing="1" cellpadding="1" border="1" align="center" width="80%"><TBODY><TR><TD> fragment
</TD><TD> length without Prefix/Suffix
</TD><TD> length with Prefix/Suffix
</TD></TR><TR><TD> R0011
</TD><TD> 55 bp
</TD><TD> 96 bp
</TD></TR><TR><TD> B0014
</TD><TD> 95 bp
</TD><TD> 136 bp
</TD></TR><TR><TD> TrpSig/HisSig
</TD><TD> 34 bp/32 bp
</TD><TD> 75 bp/73 bp
</TD></TR><TR><TD> TrpSig-B0014/HisSig-B0014
</TD><TD> 135 bp/ 133 bp
</TD><TD> 176 bp/ 174 bp
</TD></TR><TR><TD> R0011-TrpSig-B0014/R0011-HisSig-B0014
</TD><TD> 206 bp/ 204 bp
</TD><TD> 247 bp/ 245 bp
</TD></TR><TR><TD> R0011-B0014
</TD><TD> 156 bp
</TD><TD> 197 bp
</TD></TR><TR><TD> R0011-TrpSig/R0011-HisSig
</TD><TD> 95 bp/93 bp
</TD><TD> 136 bp/134 bp
</TD></TR></TBODY></TABLE><P>Prefix: 20 bp; Suffix: 21 bp; X-S-scar: 6 bp
</P><P>--&gt; it looks as if R0011 is ligated to B0014, which makes the whole construct wothless. The R0011-HisSig control looks more like R0011 alone as well.
</P><H3><SPAN class="mw-headline" id="11.05.2010">11.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B></LI><LI> 4 µl Signal (E/S-digested; from ) with 5 µl pSB1K3-B0014 (E/X-digested; from)
</LI><LI>15 min @ RT
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B></LI><LI>50 µl XL-10 transformed with 7 µl of Ligation mix
</LI></UL><H3><SPAN class="mw-headline" id="12.05.2010">12.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B> of picked clones from <A href="/wiki/index.php?title=11.05.2010&amp;action=edit&amp;redlink=1" class="new" title="11.05.2010 (page does not exist)">Tu 12.05.2010</A></LI><LI>PCR of 2 µl of each sample, 2 µl as negative control (Program: ColonyPCR, modified)
</LI><LI>15 µl of each sample mixed with 3 µl GLPn and loaded to Gel
</LI><LI>2% Agarose in 1xTAE, 120 V, 110 min
</LI><LI>stained in SybrSafe 60 min
</LI></UL><P><A href="/File:TUM2010_100512beschriftet.png" title="File:TUM2010 100512beschriftet.png">File:TUM2010 100512beschriftet.png</A></P><TABLE cellspacing="1" cellpadding="1" border="1" align="center" width="80%"><TBODY><TR><TD> fragment
</TD><TD> length without Prefix/Suffix
</TD><TD> length with Prefix/Suffix
</TD><TD> length after PCR
</TD></TR><TR><TD> R0011
</TD><TD> 55 bp
</TD><TD> 96 bp
</TD><TD> 104 bp
</TD></TR><TR><TD> B0014
</TD><TD> 95 bp
</TD><TD> 136 bp
</TD><TD> 154 bp
</TD></TR><TR><TD> TrpSig/HisSig
</TD><TD> 34 bp/32 bp
</TD><TD> 75 bp/73 bp
</TD><TD> 93 bp/91 bp
</TD></TR><TR><TD> TrpSig-B0014/HisSig-B0014
</TD><TD> 135 bp/ 133 bp
</TD><TD> 176 bp/ 174 bp
</TD><TD> 194 bp/ 192 bp
</TD></TR><TR><TD> R0011-TrpSig-B0014/R0011-HisSig-B0014
</TD><TD> 206 bp/ 204 bp
</TD><TD> 247 bp/ 245 bp
</TD><TD> 265 bp/ 263 bp
</TD></TR><TR><TD> R0011-B0014
</TD><TD> 156 bp
</TD><TD> 197 bp
</TD><TD> 215 bp
</TD></TR><TR><TD> R0011-TrpSig/R0011-HisSig
</TD><TD> 95 bp/93 bp
</TD><TD> 136 bp/134 bp
</TD><TD> 154 bp/152 bp
</TD></TR></TBODY></TABLE><P>
Prefix: 20 bp/29 bp after PCR; Suffix: 21 bp/30 bp after PCR; X-S-scar: 6 bp
</P><H3><SPAN class="mw-headline" id="14.05.2010">14.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of pSB1K3 with EcorI and XbaI
</LI><LI> 10 µl template (sample III, 103 ng/µl)
</LI><LI> 2 µl BSA, 2 µl Buffer NEB#4
</LI><LI> 1 µl EcoRI, 1 µl XbaI
</LI><LI> 4 µl H2O
</LI><LI> 1 h @ 37°C
</LI><LI>loaded on gel (with 4 µl GLPn) in 1 lane
</LI></UL><P><A href="/File:TUM2010_100514beschriftet.png" title="File:TUM2010 100514beschriftet.png">File:TUM2010 100514beschriftet.png</A></P><UL><LI>Gel excision with Zymo Kit 
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B> of HisSig and TrpSig with EcorI and SpeI
</LI><LI> 10 µl template (&quot;1:100&quot;)
</LI><LI> 2 µl BSA, 2 µl Buffer NEB#3
</LI><LI> 1 µl EcoRI, 1 µl SpeI
</LI><LI> 4 µl H2O
</LI><LI> 1.5 h @ 37°C
</LI><LI> Purification with <A href="/Team:TU_Munich/Lab#Molecular_Biology_ZYMO_RESEARCH_DNA_Clean.26Concentration_Kit" title="Team:TU Munich/Lab">Zymo 5 </A></LI><LI> or heat inactivated (20 min @ 80°C)
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B></LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B></LI><LI>50 µl XL-10 transformed with 10 µl of Ligation mix
</LI><LI>50 µl untransformed cells plated on Kana-plate as control
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week07
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="17.05.2010">17.05.2010</SPAN></H3><UL><LI>Plates from Friday:
</LI><LI>plenty colonies on control plate --&gt; XL10 cells are impure!
</LI><LI>use DH5a from now on!!!!!
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B></LI><LI>50 µl DH5a transformed with 10 µl of Friday's Ligation mix
</LI><LI>plated on Kana-Plates; Overnight @ 37°C
</LI></UL><UL><LI>DNA Isolation from BioBrick Distribution 2010
</LI><LI> 10 µl H2O added to Well 1A of plate 1 containing pSB1A10 with RFP-insert
</LI><LI> 2 µl used for <A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A> of 50 µl DH5a-cells
</LI><LI> plated on Carbenicillin (=Amp-analogon)-plates, Overnight @ 37°C
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Preparation_of_Gels" title="Team:TU Munich/Lab">Polyacrylamide Gel</A></B> prepared for tomorrow
</LI><LI> 1 big denaturing Gel with 20 pockets
</LI></UL><H3><SPAN class="mw-headline" id="18.05.2010">18.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B> of picked clones 
</LI><LI>PCR of 2 µl of each sample, 2 µl as negative control (Program: ColonyPCR)
</LI><LI>10 µl of each sample mixed with 10 µl Formamide loading buffer and loaded to Polyacrylamide Gel
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Running_of_Gels" title="Team:TU Munich/Lab">Polyacrylamide Gel</A></B></LI></UL><P>Samples: 
LMW|R0011|HisSig|TrpSig|HisSig E/S-Dig|TrpSig E/S-Dig|B0014|LMW2|Colony PCR His1|His2|Trp1|Trp2|control|HisTerm|TrpTerm|HisTerm E/P-Dig|TrpTerm E/P-Dig
</P><UL><LI>5 µl of each samples mixed with 5 µl formamide loading dye and loaded to gel(except Ladder and colonyPCR)
</LI><LI>LMW: 3 µl LMW (Korbinian) + 3 µl Formamide loading Dye
</LI><LI>LMW2: 5 µl LMW Quickload (with GLP) + 10 µl Formamide loading Dye
</LI></UL><LI>stained in SybrSafe
</LI><P><FONT color="red">Important Mistake! See below Gel! </FONT>
IMPORTANT MISTAKE: DENATURING GELS NOT USEFUL FOR dsDNA!!!
REPEAT WITH NATIVE GEL, IGNORE INTERPRETATION!!!
</P><P>(*R0011 and B0014 look normal
</P><UL><LI>ColonyPCR: bands that look like B0014 in all clones (and in control) --&gt; Religation?
</LI><LI>Signals at the wrong size: should be about 75 bp, look like 200 bp!!!
</LI><LI>terminators completely strange: should be around 100 bp!
</LI></UL><P>--&gt; are all of our sequences just wrong???
What are we going to do? Order everything new?)
</P><H3><SPAN class="mw-headline" id="19.05.2010">19.05.2010</SPAN></H3><UL><LI>  Gel from Korbinian
</LI><LI>5 µl of each samples mixed with 5 µl formamide loading dye and loaded to gel(except Ladder and colonyPCR)
</LI><LI>LMW: 3 µl LMW (Korbinian) + 3 µl Formamide loading Dye
</LI><LI>colony PCR: from Tuesday, 8 µl sample with 8 µl Formamide loading Dye
</LI></UL><LI>stained in SybrSafe 20 min
</LI><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B></LI><LI> 4 colonies picked from each Plate (Ligations from yesterday; Signal-B0014)
</LI><LI>15 µl of each Sample mixed with 3 µl GLPn and loaded to Gel:
</LI><LI> 3% Agarose in 1x TBE, 130 V
</LI></UL><P><A href="/File:TUM2010_100519beschriftet.png600px" title="File:TUM2010 100519beschriftet.png600px">File:TUM2010 100519beschriftet.png600px</A></P><H3><SPAN class="mw-headline" id="20.05.2010">20.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Digestion" title="Team:TU Munich/Lab">Digestion</A></B></LI><LI> template
</LI><LI> 2 µl BSA
</LI><LI> 2 µl Buffer 
</LI><LI> 1 µl of each enzyme
</LI><LI> water to reach 20 µl
</LI></UL><TABLE width="618" cellspacing="1" cellpadding="1" border="1" align="center" style=""><TBODY><TR><TD><B>Template</B></TD><TD><B>Enzymes</B></TD><TD><B>NEB Buffer #</B></TD></TR><TR><TD> HisTerm &amp; TrpTerm (10 µl)</TD><TD> EcoRI, PstI</TD><TD> 3
</TD></TR><TR><TD> HisSig &amp; TrpSig (10 µl)</TD><TD> EcoRI, SpeI</TD><TD> 4</TD></TR><TR><TD> B0014 (5 µl)</TD><TD> XbaI, PstI</TD><TD> 3</TD></TR><TR><TD> pSB1A10_RFP (14 µl)</TD><TD> EcoRI, PstI</TD><TD> 3
</TD></TR><TR><TD> pSB1K3_RFP (14 µl)</TD><TD> EcoRI, PstI</TD><TD> 3
</TD></TR><TR><TD> pSB1K3_B0014 N° 4 (14 µl)</TD><TD> EcoRI, XbaI</TD><TD> 4
</TD></TR></TBODY></TABLE><UL><LI> incubated @37°C for 1.5 h
</LI><LI>digested inserts heat inactivated (20 min @ 80°C)
</LI><LI>digested plasmids loaded on gel (with 4 µl GLPn) in 1 lane
</LI></UL><UL><LI>Gel excision with Zymo Kit 
</LI><LI>c(pSB1A10, I)=4.5 ng/µl
</LI><LI>c(pSB1A10, II)=1.5 ng/µl ?!?!?!
</LI><LI>c(pSB1K3 E/P)=7 ng/µl
</LI><LI>c(pSB1K3_B0014 E/X)=2.5 ng/µl
</LI></UL><UL><LI>Gel of PCR products
</LI><LI>3% Agarose in 1x TBE; 2h @130 V
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B></LI><LI> templates
</LI><LI>2 µl T4-buffer 10x
</LI><LI>1 µl T4-Ligase
</LI><LI>Water to reach 20 µl
</LI></UL><TABLE width="618" cellspacing="1" cellpadding="1" border="1" align="center" style=""><TBODY><TR><TD><B>Vector</B></TD><TD><B>Insert</B></TD></TR><TR><TD> psB1A10 (E/P; sample I) (10 µl)</TD><TD> TrpTerm (E/P) (4 µl)</TD></TR><TR><TD> psB1A10 (E/P; sample I) (10 µl)</TD><TD> HisTerm (E/P) (4 µl)</TD></TR><TR><TD> psB1K3_B0014 (E/X) (12 µl)</TD><TD> HisSig (E/S) (2 µl)</TD></TR><TR><TD> psB1K3_B0014 (E/X) (12 µl)</TD><TD> TrpSig (E/S) (2 µl)</TD></TR><TR><TD> psB1K3 (E/P) (8 µl)</TD><TD> HisSig (E/S)(2 µl) + B0014 (X/P)(1.5 µl)</TD></TR><TR><TD> psB1K3 (E/P) (8 µl)</TD><TD> TrpSig (E/S)(2 µl) + B0014 (X/P)(1.5 µl)</TD></TR></TBODY></TABLE><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B></LI><LI> 50 µl DH5a transformed with 10 µl of Ligation mix
</LI><LI> 50 µl DH5a transformed with 2 µl of pSB1K3_B0014
</LI><LI> 50 µl DH5a transformed with 2 µl of pSB1K3_RFP
</LI><LI> 50 µl DH5a transformed with 2 µl of pSB1A10_RFP
</LI></UL><H3><SPAN class="mw-headline" id="21.05.2010">21.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B></LI><LI> 4 colonies picked from each Plate (pSB1K3_HisSig_B0014, pSB1K3_TrpSig_B0014, pSB1K3_HisSig_B0014 double ligation, pSB1K3_TrpSig_B0014 double ligation)
</LI><LI> each clone resuspended in 20 µl LB0, 3 µl used as template for PCR 
</LI><LI> 15 µl of each Sample mixed with 3 µl GLPn and loaded to Gel:
</LI><LI> 3% Agarose in 1x TBE, 130 V
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week08
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="25.05.2010">25.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B></LI><LI> 4 colonies picked from each Plate 
</LI><LI>pSB1K3_HisSig_B0014
</LI><LI>pSB1K3_TrpSig_B0014
</LI><LI>pSB1K3_HisSig_B0014 double ligation
</LI><LI>pSB1K3_TrpSig_B0014 double ligation
</LI><LI>pSB1A10_TrpTerm
</LI><LI>pSB1A10_HisTerm
</LI></UL><LI> each clone resuspended in 20 µl LB0, 2 µl used as template for PCR 
</LI><LI> 15 µl of each Sample mixed with 3 µl GLPn and loaded to Gel:
</LI><LI> 3% Agarose in 1x TBE, 220 V (double Gel, 35 cm)
</LI><LI> stained in SybrSafe
</LI><P><A href="/File:TUM2010_100525beschriftet." title="File:TUM2010 100525beschriftet.">600px</A></P><UL><LI>overnight cultures made of
</LI><LI>HisSig 3, DL1, DL4
</LI><LI>TrpSig DL2, DL4
</LI><LI>HisTerm/TrpTerm 1,2,3
</LI></UL><H3><SPAN class="mw-headline" id="26.05.2010">26.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_.23ZYMO_RESEARCH_DNA_Clean.26Concentration_Kit" title="Team:TU Munich/Lab">Miniprep</A></B> of cultures set up <A href="/wiki/index.php?title=25.05.2010&amp;action=edit&amp;redlink=1" class="new" title="25.05.2010 (page does not exist)">25.05.2010</A></LI><LI>HisSig 3, DL1, DL4
</LI><LI>TrpSig DL2, DL4
</LI><LI>HisTerm/TrpTerm 1,2,3
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Restriction" title="Team:TU Munich/Lab">Restriction</A></B></LI><LI>analytical: E/P HisSig(3, DL2); TrpSig(DL2, DL4): 1.5 h 37 °C
</LI><LI>prep: E/X HisSig(3, DL2); TrpSig(DL2, DL4): 1.5 h 37 °C
</LI><LI>prep: E/S R0011: 1.5 h 37 °C, inactivation 20 min @ 80 °C
</LI><LI>total volume each 20 µl, 10 µL template
</LI></UL><LI><B>Gel</B>: 1% Agarose, TAE - 1,5 h 110 V
</LI><LI><A href="/Team:TU_Munich/Lab#Molecular_Biology_.23ZYMO_RESEARCH_Gel_DNA_Recovery_Kit" title="Team:TU Munich/Lab">bands excised</A>: all E/X cleaved vectors
</LI><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B>:
</LI><LI>HisSig_3 (E/X) with R0011 (E/S)
</LI><LI>HisSig_DL2 (E/X) with R0011 (E/S)
</LI><LI>TrpSig_DL2 (E/X) with R0011 (E/S)
</LI><LI>TrpSig_DL4 (E/X) with R0011 (E/S)
</LI><LI><B>batches</B></LI><LI>total volume 20 µL
</LI><LI>2 µL R0011 (E/S)
</LI><LI>2 µL T4 buffer
</LI><LI>1 µL T4 Ligase
</LI><LI>15 µL vector
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B> of ligations
</LI><LI> DH5a
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">PCR</A></B></LI><LI>His and TrpSig
</LI><LI>His and TrpTerm
</LI><LI>R0011
</LI><LI>B0014
</LI><LI>50 µL total volume</LI><LI>1 µL template</LI><LI>1 µl G1004</LI><LI>1 µl G1005</LI><LI>0.2 µL Taq</LI><LI>5 µl Taq standard buffer</LI><LI>rest water
</LI></UL><H3><SPAN class="mw-headline" id="27.05.2010">27.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B></LI><LI> 3 colonies picked from each Plate 
</LI><LI>pSB1K3_R0011_HisSig_B0014 (N° 3 from yesterday)
</LI><LI>pSB1K3_R0011_HisSig_B0014 (N° DL1 from yesterday)
</LI><LI>pSB1K3_R0011_TrpSig_B0014 (N° DL2 from yesterday)
</LI><LI>pSB1K3_R0011_TrpSig_B0014 (N° DL4 from yesterday)
</LI><LI>&quot;N6&quot;
</LI><LI>&quot;N15&quot;
</LI></UL><LI> each clone resuspended in 20 µl LB0, 2 µl used as template for PCR 
</LI><LI> 15 µl of each Sample mixed with 3 µl GLPn and loaded to Gel:
</LI><LI> 3% Agarose in 1x TBE, 220 V (double Gel, 35 cm)
</LI><LI> stained in SybrSafe
</LI><TABLE cellspacing="1" cellpadding="1" border="1" align="center" width="80%"><TBODY><TR><TD> fragment
</TD><TD> length without Prefix/Suffix
</TD><TD> length after PCR
</TD></TR><TR><TD> R0011
</TD><TD> 55 bp
</TD><TD> 104 bp
</TD></TR><TR><TD> B0014
</TD><TD> 95 bp
</TD><TD> 154 bp
</TD></TR><TR><TD> TrpSig/HisSig
</TD><TD> 34 bp/32 bp
</TD><TD> 93 bp/91 bp
</TD></TR><TR><TD> TrpSig-B0014/HisSig-B0014
</TD><TD> 135 bp/ 133 bp
</TD><TD> 194 bp/ 192 bp
</TD></TR><TR><TD><FONT color="red">R0011-TrpSig-B0014/R0011-HisSig-B0014
</FONT></TD><TD> 206 bp/ 204 bp
</TD><TD><FONT color="red">265 bp/ 263 bp</FONT></TD></TR><TR><TD> I712074 (&quot;N6&quot;)
</TD><TD> 46 bp
</TD><TD> 105 bp
</TD></TR><TR><TD> I719005 (&quot;N15&quot;)
</TD><TD> 23 bp
</TD><TD> 82 bp
</TD></TR></TBODY></TABLE><P>
Prefix: 29 bp after PCR; Suffix: 30 bp after PCR; X-S-scar: 6 bp
</P><UL><LI>overnight cultures made of
</LI><LI>HisSig 3_1, DL1_3
</LI><LI>TrpSig DL4_1, DL4_3
</LI><LI>N15 1&amp;2
</LI><LI>HisTerm/TrpTerm (picked colonies from yesterdays plates #2 each)
</LI></UL><UL><LI>Purification of yesterday's PCR
</LI><LI>elution in 50 µl H2O
</LI><LI> c(HisSig)=5.5 ng/µl
</LI><LI> c(TrpSig)=10 ng/µl
</LI><LI> c(HisTerm)=6.5 ng/µl
</LI><LI> c(TrpTerm)=6.5 ng/µl
</LI><LI> c(R0011)=13 ng/µl
</LI><LI> c(B0014)=12.5 ng/µl
</LI></UL><LI> 5 µl loaded on gel with 1 µl GLPn; 3% Agarose in 1x TBE, 220 V (double Gel, 35 cm)
</LI><H3><SPAN class="mw-headline" id="28.05.2010">28.05.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_ZYMO_RESEARCH_DNA_Clean.26Concentration_Kit" title="Team:TU Munich/Lab">Miniprep</A></B> of cultures from <A href="/wiki/index.php?title=27.05.2010&amp;action=edit&amp;redlink=1" class="new" title="27.05.2010 (page does not exist)">27.05.2010</A>, Elution in 50 uL nuclease free water 
</LI><LI> (1)HisSig 3_1 
</LI><LI> (2)HisSig DL1_3 
</LI><LI> (3)TrpSig DL4_1 
</LI><LI> (4)TrpSig DL4_3 
</LI><LI> (7)HisTerm#1 
</LI><LI> (8)HisTerm#2 (7 ml culture) 
</LI><LI> (9)TrpTerm#1 
</LI><LI> (10)TrpTerm#2 (7ml culture) 
</LI><LI> (5)N15-1 (=BBa_I719005) 
</LI><LI> (6)N15-2 (=BBa_I719005)
</LI><LI> ()=numbers on gel
</LI></UL><TABLE cellspacing="1" cellpadding="1" border="1" style="width: 367px; height: 271px;"><TBODY><TR><TD> sample</TD><TD> DNA concentration (ng/uL)</TD></TR><TR><TD> HisSig 3_1 </TD><TD> 6</TD></TR><TR><TD> HisSig DL1_3</TD><TD> 11</TD></TR><TR><TD> TrpSig DL4_1 </TD><TD> 16</TD></TR><TR><TD> TrpSig  DL4_3 </TD><TD> 21.5</TD></TR><TR><TD> HisTerm#1</TD><TD> 27</TD></TR><TR><TD> HisTerm#2</TD><TD> 66</TD></TR><TR><TD> TrpTerm#1</TD><TD> 34</TD></TR><TR><TD> TrpTerm#2</TD><TD> 29.5</TD></TR><TR><TD> N15-1 (=BBa_I719005)</TD><TD> 19.5</TD></TR><TR><TD> N15-2 (=BBa_I719005)</TD><TD> 18.5</TD></TR></TBODY></TABLE><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Restriction" title="Team:TU Munich/Lab">analytical digest</A></B> (E/P)</LI><LI>of all samples. total volume 20 uL, 5 uL template used for Term-constructs, 10 uL teplate for all others
</LI></UL><UL><LI><B>Agarose Gel</B></LI><LI>3% broad range agarose in TBE. Run in TBE, 140 V, 1.50 h 
</LI><LI>stained with SybrGold, 45 min
</LI><LI>signals look fine
</LI><LI>terminators also (without pre/suffix 97/104 bp)
</LI><LI>T7 promoter without pre/suffix has a length of 23 bp + cut pre/szffix ca at 60 bp --&gt;buffer?
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week09
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="31.05.2010">31.05.2010</SPAN></H3><UL><LI> Digestions
</LI><LI>15N-1 (BBa_I719005, 19.5 ng/uL) with S/P
</LI><LI>HisSig/TrpSig (6.5/10 ng/uL) with X/P
</LI></UL><DL><DD>2 h 37 °C
</DD></DL><DL><DD>heat inactivation of insert-digestions
</DD></DL><UL><LI>Gel: 1% Agarose in 1x TAE
</LI></UL><DL><DD>1 h 25 min, 115 V
</DD><DD>stained with SybrGold, 40 min
</DD></DL><DL><DD>Band at ~2100b cut und purified using the <A href="/Team:TU_Munich/Lab#Molecular_Biology_ZYMO_RESEARCH_Gel_DNA_Recovery_Kit" title="Team:TU Munich/Lab">zymo kit</A></DD></DL><UL><LI><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">ligation</A></LI><LI> HisSig (4 µL of digest) with purified plasmid (with BBa_I719005)
</LI><LI> TrpSig (2 µL of digest) with purified plasmid -=-
</LI></UL><DL><DD>reason: concentration of His Sig before digest was 1/2 of TrpSig
</DD></DL><UL><LI><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">transformation</A></LI></UL><DL><DD> of DH5a with Ligation batches, HisSig1-3, HisSig3-1, TrpSig DL4-1, TrpSig 4-3
</DD></DL><H3><SPAN class="mw-headline" id="01.06.2010">01.06.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B></LI></UL><DL><DD>of Ligations transformed into DH5a yesterday 
</DD><DD>4 Colonies of each Ligation
</DD></DL><UL><LI>T7-HisSig 
</LI><LI>T7-TrpSig
</LI></UL><UL><LI><B>Gel: 3% broad range Agarose in 1xTBE</B></LI></UL><DL><DD>1.5 h 140 V 
</DD><DD>stained with SybrGold
</DD></DL><DL><DD>calculation for the expected size of the fragments
</DD></DL><DL><DD><TABLE cellspacing="1" cellpadding="1" border="1" width="200"><TBODY><TR><TD> part
</TD><TD> size (bp)
</TD></TR><TR><TD> HisSig
</TD><TD> 32
</TD></TR><TR><TD> TrpSig
</TD><TD> 34
</TD></TR><TR><TD> T7 promoter
</TD><TD> 23
</TD></TR><TR><TD> prefix
</TD><TD> 20
</TD></TR><TR><TD> suffix
</TD><TD> 21
</TD></TR><TR><TD> X/S scar
</TD><TD> 6
</TD></TR></TBODY></TABLE></DD></DL><DL><DD>in PCR we get additional bp due to the primers - +9 at pre/suffix=+18 bp</DD></DL><DL><DD>overall size of the fragments expected to come out of the PCR: T7_HisSig: 120 bp, T7_TrpSig: 122 bp
</DD></DL><UL><LI><B>ON cultures</B></LI></UL><DL><DD>5 ml cultures of pSB1K3_R0011_HisSig_B0014 (1_3 &amp; 3_1) and pSB1K3_R0011_TrpSig_B0014 (DL4_1 &amp; DL4_3) 
</DD><DD>1 ml cultures of each colony monitored in Colony PCR
</DD></DL><H3><SPAN class="mw-headline" id="02.06.2010">02.06.2010</SPAN></H3><UL><LI>Miniprep of yesterdays cultures using Zymokit, elution by nuclease-free water 
</LI><LI>Concentration determination 
</LI><LI>analytic digestion 
</LI><LI>results on gel:
</LI></UL><UL><LI>Sequencing </LI></UL><P>JobNr. Barcode Last change Date/Time Last message / Files 6549287 AE2739 02.06.2010 / 13:51:12 HisSig 1-3-forward G1004 
</P><PRE> 	  	We just received your order. Many thanks.
</PRE><P>6549288 AE2738 02.06.2010 / 13:51:12 HisSig 3-1-forward G1004 
</P><PRE> 	  	We just received your order. Many thanks.
</PRE><P>6549289 AE2737 02.06.2010 / 13:51:12 TrpSig DL4-1-forward G1004 
</P><PRE> 	  	We just received your order. Many thanks.
</PRE><P>6549290 AE2736 02.06.2010 / 13:51:12 TrpSig DL4-3-forward G1004 
</P><PRE> 	  	We just received your order. Many thanks.
</PRE><P>6549291 AE2735 02.06.2010 / 13:51:12 HisTerm-forward G1004 
</P><PRE> 	  	We just received your order. Many thanks.
</PRE><P>6549292 AE2734 02.06.2010 / 13:51:12 TrpTerm-forward G1004 
</P><PRE> 	  	We just received your order. Many thanks.
</PRE><UL><LI>Gel 3% broad range agarose in 1x TBE
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week10
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="07.06.2010">07.06.2010</SPAN></H3><UL><LI><B>Sequenbcing results from GATC</B></LI><LI>HisSig DL1-3 is ok 
</LI><LI>HisSig 3-1 is ok 
</LI><LI>TrpSig DL4-1 is ok 
</LI><LI>TrpSig DL4-3 is ok 
</LI><LI>TrpTerm + HisTerm bad runs... --&gt; new sequencing order with Primer 100 bp upstream (within GFP)
</LI></UL><DL><DD>Files can be found stored in our <A href="/wiki/index.php?title=Zugangsdaten_f%C3%BCr_GATC&amp;action=edit&amp;redlink=1" class="new" title="Zugangsdaten für GATC (page does not exist)">GATC account</A></DD></DL><UL><LI><B>Sequencing@GATC:</B> both Term-constructs with primer pGFP-FP provided by GATC
</LI></UL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Restriction" title="Team:TU Munich/Lab">Restrictions</A></B></LI><LI>psB1A10_TrpTerm/HisTerm with Nsi1, Aat2 
</LI><LI>pSB1K3_R0011_HisSig/TrpSig_B0014 with Pst1, Aat2 
</LI><LI>T7 bb with Spe1, Pst1 
</LI><LI>PCRProducts: HisSig/TrpSig with Pst1, Xba1, 2 h @37°C
</LI></UL><DL><DD>all plasmid digests done<B> sequential</B> as enzymes do not have 100% activity in the same buffer, each reaction 1.5 h@37°C 
</DD><DD>psB1A10_TrpTerm/HisTerm and T7 bb dephosphorylated the last 30 min </DD></DL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B></LI><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Transformation" title="Team:TU Munich/Lab">Transformation</A></B></LI></UL><P> liquid culture (10 ml) of pSB1K3_R0011_HisSig/TrpSig_B0014
</P><H3><SPAN class="mw-headline" id="08.06.2010">08.06.2010</SPAN></H3><UL><LI><B>Sequencing results from GATC</B>: His/TrpTerm with pGFP-FP primer
</LI><LI>HisTerm worked
</LI><LI>TrpTerm worked
</LI></UL><DL><DD> checked with blast2seq
</DD></DL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_PCR" title="Team:TU Munich/Lab">Colony PCR</A></B></LI></UL><DL><DD> 8 colonies from each plate of T7His, T7Trp, MonsterHis, MonsterTrp as ligations resulted in many colonies
</DD></DL><UL><LI><B>3% Agarose Gel in 1xTBE</B></LI></UL><DL><DD> for all colony PCR reactions
</DD></DL><P> Interpretation/Info: The R0011_Sig_B0014 construct was cut with PstI, but not ligated into a PstI site but instead into the NsiI site --&gt; even if sticky end are compatible, the bases in the 3` direction are different --&gt; primer lags 7 bp compared to standard procedure --&gt; we didn´t expect to find the signal construct by colony PCR --&gt; control digestion tomorrow
</P><DL><DD> T7-Signal constructs seem to have worked, expected size was 23 bp (T7)+ 32 (HisSig)/34 (TrpSig) bp + 30+29 (PCRPre+Suf)=114/116 bp
</DD></DL><UL><LI>over night cultures
</LI></UL><H3><SPAN class="mw-headline" id="09.06.2010">09.06.2010</SPAN></H3><UL><LI> Miniprep of 4 Monster_His, 4 Monster_Trp, 3 T7_His and 3 T7_Trp cultures
</LI><LI> Analytical digestion of plasmids mentioned above
</LI><LI> gel of Monster_Plasmid digestion
</LI></UL><P>
gel didn´t work at all --&gt; even after &gt; 2 h, bands were not separated correctly, even the 1kb ladder was &quot;stacked&quot; in the gel-pockets, the 100 bp ladder should show EQUAL distances between the lines <A href="http://www.neb.com/nebecomm/productfiles/778/images/N3231_fig1_v1_000034.gif" class="external text" rel="nofollow">see here</A>, it looks like the gel was &quot;more dense&quot; at the pockets---&gt; no idea what happened --&gt; repeat Monster-digestion tomorrow?
</P><UL><LI> gel of T7_Plasmid digestion
</LI></UL><P>
T7_Trp E + S digestion 107 bp and T7_His 105 bp --&gt; worked for all picked colonies. (regard that there is an excess of plasmid DNA-basepairs of factor &gt;30 --&gt; thats why the inserts are much weaker than the plasmid signals.

occured trouble: 
</P><UL><LI> Ladders and loading dye´s empty --&gt; i used those of eike, BUT: eikes 1 kb ladder is different --&gt; compare <A href="http://www.neb.com/nebecomm/products/productN3272.asp" class="external text" rel="nofollow">here</A> and his loading dye was much more diluted, even if there was also 6x Sac GLP written on it -&gt; i hope this won´t cause any trouble
</LI></UL><H3><SPAN class="mw-headline" id="10.06.2010">10.06.2010</SPAN></H3><UL><LI><B>ordered</B></LI></UL><DL><DD>Promega E.coli S30 in vitro transcription/translation kit
</DD><DD>Spe1, Aat2 from NEB, 500 U each
</DD></DL><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Restriction" title="Team:TU Munich/Lab">analytical Digestion</A></B></LI></UL><DL><DD>of Ligation colonies from MonsterHis/trp 1-3 and T7His/Trp 1,2,5/1,2,3
</DD><DD>2h digestion
</DD><DD> Monster: 6 µL DNA template with Aat2/Spe1 in Buffer 4/Bsa
</DD><DD> T7-Signal: 6 µL DNA template with E/P in Buffer 3
</DD></DL><UL><LI><B>Agarose Gels</B></LI></UL><DL><DD>used standards: lmw, 2-log <A href="/Team:TU_Munich/Lab#Molecular_Biology_standards" title="Team:TU Munich/Lab">click here</A></DD><DD> Gel1: 1% Agarose in 1xTBE for Digestions of Monsterplasmid
</DD><DD> run in big chamber @ 200 V for 1 h 20 min
</DD></DL><DL><DD>Gel2: 3% Agarose (broad range) in 1xTBE for Digestions of T7-Signal
</DD><DD> run in small chamber @140 V for 1 h 35 min
</DD></DL><DL><DD>Conclusions:
</DD></DL><OL><LI> all T7-Signal ligations loaded on the gel worked
</LI><LI> monsterplasmid didn't work? bands at 800 bp, 900 bp, 1.3 kbp, 2.2 kbp, 3 kbp, we SHOULD expect to see our Insert, wich is Prefix+R0011_Signal_B0014_small Suffix, which should run around 300-400 bp...
</LI></OL><H3><SPAN class="mw-headline" id="11.06.2010">11.06.2010</SPAN></H3><UL><LI><B>Gel</B>: large 1% Agarose in TAE. Load: The rest of N/A cut Messplasmids from <A href="/wiki/index.php?title=07.06.2010&amp;action=edit&amp;redlink=1" class="new" title="07.06.2010 (page does not exist)">07.06.2010</A>. Run @220 V for 3.5 h
</LI></UL><DL><DD> fragments expected are 5087 and 176. original size of plasmid is 5263. This is a Try to differ between 5087 and 5263 bp
</DD><DD> Band @ 5000 bp of Trp_Term purified, obviously digestion was 100%. Bad point is that HisTerm includes an Nsi1 cleavage site...
</DD></DL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week11
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="color: #BBBB00; border: #BBBB00 1px solid; background-color: #FFFF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; float: left;">Promega Kit</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="14.06.2010">14.06.2010</SPAN></H3><UL><LI><B><A href="/Team:TU_Munich/Lab#Molecular_Biology_Ligation" title="Team:TU Munich/Lab">Ligation</A></B></LI></UL><DL><DD> 10 µL pSB1A10_TrpTerm Aat2/Nsi1 0.5 ng/uL
</DD><DD>  1 µL R0011_TrpSig_B0014 Aat2/Psb1 11 ng/uL
</DD><DD>  2 µL T4 ligase buffer
</DD><DD>  1 µL T4 Ligase
</DD><DD>  6 µL H2O
</DD><DD>10 min RT
</DD></DL><UL><LI><B>Transformation</B> of DH5a with
</LI></UL><DL><DD> Ligation
</DD><DD> T7His#1
</DD><DD> T7Trp#1
</DD></DL><UL><LI><B>over night cultures</B> of
</LI></UL><DL><DD> pSB1K3_R0011_TrpSig_B0014
</DD><DD> pSB1K3_R0011_HisSig_B0014
</DD><DD> pSB1A10_TrpTerm
</DD><DD> pSB1A10_HisTerm
</DD></DL><H3><SPAN class="mw-headline" id="15.06.2010">15.06.2010</SPAN></H3><UL><LI>Over night cultures 
</LI><LI>Aliquots of the Promega in vitro expressions kit from <I>E. coli</I> S30 extract:
</LI></UL><DL><DD> 40 µL with aa mix including all aa.
</DD></DL><H3><SPAN class="mw-headline" id="16.06.2010">16.06.2010</SPAN></H3><P><B>Fluoresence measurements using in vitro kit</B></P><UL><LI> in vitro kit sample 
</LI><LI> adding psBA1A10 Trp_Term --&gt; constant over time, no significant changes compared to kit alone --&gt; high efficiency of AraC
</LI><LI> adding L-(+)Arabinose (final concentration 2%) --&gt; after approx. 10 min significant GFP production --&gt; measuring for xxx min --&gt; RFP is slightly increased (to proof if correlated to GFP peak --&gt; crossdetection)
</LI><LI> adding psB1K3 R0011_TrpSig_B0014
</LI></UL><P><B>Cell culture</B>
5 ml culture for
</P><UL><LI> psBA1A10 Trp_Term/HisTerm
</LI><LI> psB1K3 R0011_TrpSig/Hissig_B0014
</LI></UL><H3><SPAN class="mw-headline" id="17.06.2010">17.06.2010</SPAN></H3><P><B> Cloning </B>
Digestion of Trp-Sig with E/P and psB1A10 Trp_Term with E/P
</P><UL><LI> Gel purification of psB1A10 Trp_Term E/P cut
</LI></UL><UL><LI> Heat inactivation of Trp-Sig E/P cut
</LI><LI> Ligation for 10 min @ RT and Transformation in DH5-a cells
</LI></UL><H3><SPAN class="mw-headline" id="18.06.2010">18.06.2010</SPAN></H3><P><B>cloning</B></P><UL><LI> Transformation (about 20 colonies) --&gt; picking 5 colonies
</LI><LI> colony PCR
</LI><LI> Gel </LI></UL><P>2 % broad range agarose, 1 h 120 V 
Sample 2, 4, 5 shows probably Trp-Signal + Pre/Suffix --&gt; send sample 2 for sequencing!
</P><UL><LI> control digestion of all 10 picked psB1A10-TrpSig in 1% broad range agarose, &gt; 3 h, 120 V
</LI></UL><P>
--&gt; digestions worked, but again, no insert can be found, despite gel was at maximum resolution (  3h 120 V, see LMW)
</P><P><B> in vitro measurements </B>
f$%&amp;&amp;§ s%§$! Again, nothing worked! Although we saw an increasing &quot;GFP signal&quot; comparable to 16.06.10, taking spectra suggested we DON'T see significant GFP-production! We used new water for preparing the samples, cleaned cuvettes with &quot;new water&quot;, used other DNA-samples etc. Somehow, it seems as if we don't express GFP (we compared Christoph's results! We should see a really significant spectrum! Next steps:
</P><UL><LI> Try in vivo measurements, just using psb1A10_xTerm without Signal (thus just measuring plasmid) to proof if kit or measuring plasmid causes this problem!
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week12
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV style="color: #0000CC; border: #0000CC 1px solid; background-color: #8888FF; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">First steps</DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="color: #BBBB00; border: #BBBB00 1px solid; background-color: #FFFF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; float: left;">Promega Kit</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="22.06.2010">22.06.2010</SPAN></H3><P><B>Transformation</B></P><UL><LI> psb1A10_HisTerm/TrpTerm into BL21 (DE3) RIL
</LI><LI> psb1A10_Monster-Trp No. 7, 8, 10 (Positiv control) into DH5-a
</LI></UL><P><B>Liquid culture</B></P><UL><LI> psb1A10_TrpSig (Positiv control) No. 2
</LI></UL><H3><SPAN class="mw-headline" id="23.06.2010">23.06.2010</SPAN></H3><UL><LI>Miniprep
</LI></UL><DL><DD> pSBN1A10_TrpSig - 40 µL, 12.5 ng/µL --&gt;very low amount of DNA...
</DD><DD> culture is slightly red? -&gt; strange because there cannot be any rfp-insert with constitutive promoter as the construct was built up from pSB1A10_TrpTerm (digested) and TrpSignal (PCR product)
</DD></DL><UL><LI>cultures
</LI><LI>5 ml cultures of DH5a 
</LI></UL><DL><DD>with MonsterTrp, #7,8,10 (Carbamp)
</DD></DL><UL><LI>50 ml cultures of BL21 (DE3) RIL
</LI></UL><DL><DD>pSB1A10_HisTerm
</DD><DD>pSB1A10_TrpTerm
</DD></DL><UL><LI>Arabinose Stock
</LI></UL><P>0.2%
</P><H3><SPAN class="mw-headline" id="24.06.2010">24.06.2010</SPAN></H3><UL><LI> Miniprep
</LI></UL><DL><DD> MonsterTrp #7,8,10
</DD><DD> positive Control
</DD><DD> concentrations are too low for sequencing --&gt; again we have to set up 5 ml cultures for tomorrow
</DD></DL><UL><LI> measurements
</LI></UL><H3><SPAN class="mw-headline" id="25.06.2010">25.06.2010</SPAN></H3><P><B>Plasmid purification and sequencing</B></P><UL><LI> Monster_Trp 7, 8, 10 and psB1A1ß_TrpSig plasmids are isolated (concentrations up to 110 ng/ul) and sent for sequencing
</LI><LI> psB1A1ß_TrpSig liquid culture was completely pink! still not clear what happend (wrong labeling of digested psB1A10_Trpterm?) --&gt; wait for sequencing details
</LI></UL><P><B>Fluorescence Measurements</B></P><UL><LI> Induction by putting Arabinose directly into cuvettes with cells IS NOT WORKING at all! Expression of GFP increases, but marginally. probably, despite stirring, oxygen is lacking?
</LI><LI> Induction on shaker work perfectly --&gt; both Trp and His showed strong GFP-signals, BUT: Probably, too high OD results in not exciting all GFP within the sample (incident beam is already scatterd enormously on the edge of the cuvette --&gt; only small volume is excited correctly). For instance, a sample showing OD of 0.7 shows a signal of 30 a.u., diluted to OD 0.35 signal falls only to 19 a.u.! Thus dilution did not result in a linear decrease of flourescence! a.u. !!! We diluted down to OD 0.1; the result: OD´s smaller than 0.4 show linear change of fluorescence signal --&gt; using OD´s up to 0.4 results in meaningful measurements!!!
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week13
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV style="color: #0000CC; border: #0000CC 1px solid; background-color: #8888FF; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Testing pSB1A10</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="28.06.2010">28.06.2010</SPAN></H3><UL><LI> Sequencing results
</LI></UL><DL><DD>psB1A10_TrpSig (Positive control) worked: Trp-Sig is inside, directly in front of RFP (without the promotor of the measurement plasmid insert <A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A>, so it seems like everything worked. Furthermore, i performed a promotor prediction with the following tool <A href="http://linux1.softberry.com/berry.phtml?topic=bprom&amp;group=programs&amp;subgroup=gfindb" class="external text" rel="nofollow">bacteria promotor prediction tool</A> to proof if our Trp-Sig in combination with the flanking regions is not forming a promotor,by mischance. According to this, there are two promotors, BUT:  one in and after the suffix (so it should be in each of our constructs), but the tools says theres is no known sigma-factor for this promotor! the second one is within the RFP and there is a sigma-factor for this one ( rpoD16). So i don´t see a explanation, why our colonies were pink in contrast to the other &quot;Messplasmids&quot;.  None of the Monsterplasmids contains the signal construct. Probably, the problem is there is no selection methods which allowed us to distinguish uncut plasmids.... --&gt; we should discuss at our next meeting, one possiblity would be connecting our construct to a resistance marker. I summed up all sequences in this document: <A href="/File:25.06.-sequenzierung.doc" title="File:25.06.-sequenzierung.doc">File:25.06.-sequenzierung.doc</A></DD></DL><UL><LI>Induction in cuvette and measuring fluorescence at the same time IS NOT WORKING! (probably cells are not growing and expressing very well, maybe lacking oxygen despite stirring. Bleaching is more unlikely)  In vivo, measuring plasmid (at least GFP) works! we optimized the paramters for fluorescence measurment! We tried different OD´s and found out that only measurments below OD 0.4 result in meaningful measurements.  as a result, in vitro expression did somehow not work, reasons are unclear, maybe too low DNA-concentrations.  positvie control psB1A10_TrpSig was pink again, we have to wait the results from GATC
</LI></UL><H3><SPAN class="mw-headline" id="29.06.2010">29.06.2010</SPAN></H3><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week14
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="color: #BBBB00; border: #BBBB00 1px solid; background-color: #FFFF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; float: left;">Invitrogen Kit</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="01.07.2010">01.07.2010</SPAN></H3><UL><LI><B>in vitro measurement</B></LI></UL><DL><DD>using Invitrogen kit
</DD><DD>measuring kinetics for 3 h @ 37°C
</DD><DD>40 µL + 5 µL pSB1A10_TrpSig (126 ng/µL) + 0.5 µL 100x L-Arabinose (=0.2%) + 4.5 µL H2O
</DD><DD>observations: GFP signal grows, after 30 min  it crashes. RFP grows
</DD><DD>emission spectra for GFP and RFP result in no spectrum
</DD><DD>looks strange, a problem might be evaporation of liquid and hence scattering of light which produces artefacts
</DD></DL><UL><LI><B>over night cultures</B></LI></UL><DL><DD>pSB1A10_TrpSig (DH5a), 5 ml for miniprep
</DD><DD>pSB1A10 XS (DH5a), 5 ml for miniprep
</DD></DL><H3><SPAN class="mw-headline" id="02.07.2010">02.07.2010</SPAN></H3><UL><LI>Miniprep
</LI></UL><DL><DD>pSB1A10_XS: 30 µL 10 ng/µL
</DD><DD>pSB1A10_TrpSig: 30 µL 10 ng/µL
</DD></DL><UL><LI>Transformation
</LI></UL><DL><DD>BL21 with pSB1A10_XS (positive control without insert and no without any bio brick site left)
</DD></DL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week15
</A></P><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="color: #0000CC; border: #0000CC 1px solid; background-color: #8888FF; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Testing pSB1A10</DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="05.07.2010">05.07.2010</SPAN></H3><UL><LI><B>over night cultures</B></LI></UL><DL><DD>250 ml of DH5a pSB1A10_XS
</DD><DD>20 ml BL21 pSB1A10_XS
</DD></DL><UL><LI><B>in vitro transcription measurement planned</B></LI></UL><DL><DD>check <A href="/wiki/index.php?title=In_vitro_Measurements&amp;action=edit&amp;redlink=1" class="new" title="In vitro Measurements (page does not exist)">In_vitro_Measurements</A></DD></DL><H3><SPAN class="mw-headline" id="06.07.2010">06.07.2010</SPAN></H3><UL><LI><B>In vivo measurement</B></LI></UL><DL><DD> BL21 pSB1A10_XS - positive control (to check the measurement plasmid...)
</DD><DD> GFP, RFP Fluorescence
</DD><DD>induced with 0.2% arabinose in (1), uninduced (2)
</DD><DD>at OD 0.15: GFP/RFP emissions spectra /100706/spectra/gfp10 and rfp10
</DD><DD>2.5 h kinetic measurement GFP/RFP /100706/kinetics/
</DD><DD>OD 0.7 (1) and 0.64 (2) after 2.5 h --&gt;  GFP/RFP emissions spectra /100706/spectra/gfp11,rfp11,gfp21,rfp21
</DD><DD>in addition for 4 h a culture at OD 0.8 induced (with 0.2% Arab), spectra taken afterwards at 1:15 dilution (OD 0.39) gpf_ku and rfp_ku
</DD></DL><DL><DD>observation: measurement plasmid is totale verarsche. RFP is not expressed at all, or this protein is not rfp. whatever.
</DD></DL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week16
</A>
The Era of Exams

</P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><P align="justified">

No Lab work this week, everybody is busy studying for their exams...
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week17
</A>
The Era of Exams

</P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><P align="justified">

No Lab work this week, everybody is busy studying for their exams...
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week18
</A>
 The Era of Exams

</P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><P align="justified">

No Lab work this week, everybody is busy studying for their exams...
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week19
</A>
 The Era of Exams

</P><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><P align="justified">

No Lab work this week, everybody is busy studying for their exams...
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week20
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Construction of new Measurement Plasmid</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="09.08.2010">09.08.2010</SPAN></H3><UL><LI>overnight cultures inoculated from Glycerolstock J06702(mCherry generator) in pSB1A2 from Christoph.
</LI></UL><H3><SPAN class="mw-headline" id="10.08.2010">10.08.2010</SPAN></H3><UL><LI>MiniPrep of pSB1A2-mCherry using ZymoKit
</LI><LI>Digestion
</LI><LI>pSB1A2-mCherry E/P
</LI><LI>pSB1A2-mCherry X/P
</LI><LI>pSB1A10-HisTerm S/P
</LI><LI>pSB1A10-TrpTerm S/P
</LI><LI>pSB1A10-HisTerm E/P
</LI></UL><LI>Purified with agarose gel (1%)
</LI><LI>Gel Doc broken =&gt; no picture
</LI><LI>Description: mCherry cut was ok, Plasmid was cut at least once (linear DNA), generally contaminated with genomic DNA
</LI><LI>Ligation
</LI><LI>50 ng Plasmid and 34 ng Insert
</LI><LI>ca. 30min @ RT
</LI><LI>Transformation of DH5a cells with ligation samples
</LI><P>(=&gt; no colonies the next day)
</P><UL><LI>overnight cultures
</LI><LI>pSB1A2-mCherry from Christoph`s stock
</LI><LI>pSB1A10-HisTerm from earlier plate
</LI><LI>pSB1A10-TrpTerm from earlier plate
</LI></UL><P>(=&gt; pSB1A10-TrpTerm and pSB1A10-HisTerm did not grow until next day)
</P><H3><SPAN class="mw-headline" id="11.08.2010">11.08.2010</SPAN></H3><UL><LI>MiniPrep of pSB1A2-mCherry using ZymoKit
</LI><LI>analytic gel of Mini preps and ligation of the previous day
</LI><LI>preps still hold genomic DNA
</LI><LI>mCherry Plasmid runs at ca. 2400 bp
</LI></UL><UL><LI>Digestion
</LI><LI>pSB1A2-mCherry E/P
</LI><LI>pSB1A2-mCherry X/P
</LI></UL><LI>Purified with agarose gel (1%)
</LI><LI>Gel Doc broken =&gt; no picture
</LI><LI>Description: mCherry cut was ok, stil contaminated with genomic DNA
</LI><LI>Ligation
</LI><LI>Plasmid (from previous day) and mCherry-Insert
</LI><LI>ca. 30 min @ RT
</LI><LI>Transformation of DH5a cells with
</LI><LI>ligation samples (=&gt; no colonies the next day)
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI><UL><LI>overnight cultures
</LI><LI>pSB1A2-mCherry from Christoph`s stock
</LI></UL><H3><SPAN class="mw-headline" id="12.08.2010">12.08.2010</SPAN></H3><UL><LI>MiniPrep of pSB1A2-mCherry using ZymoKit
</LI><LI>analytic gel of Mini preps and ligation of the previous day
</LI><LI>preps still hold genomic DNA
</LI><LI>mCherry Plasmid runs at ca. 2400 bp
</LI><LI>Gel (1%)
</LI></UL><UL><LI>Digestion
</LI><LI>pSB1A2-mCherry X/P
</LI><LI>pSB1A10-HisTerm S/P
</LI><LI>pSB1A10-TrpTerm S/P
</LI></UL><LI>Purified with agarose gel (1%)
</LI><UL><LI>Ligation
</LI><LI>Plasmid and mCherry-Insert
</LI><LI>ca. 30 min @ RT
</LI></UL><LI>Transformation of DH5a cells with
</LI><LI>ligation samples (=&gt; no colonies the next day)
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI><UL><LI>overnight cultures
</LI><LI>pSB1A2-mCherry from Christoph`s stock
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI></UL><P><B>Caution: ran out of gas =&gt; not steril?</B></P><H3><SPAN class="mw-headline" id="13.08.2010">13.08.2010</SPAN></H3><UL><LI>MiniPrep using ZymoKit
</LI><LI>pSB1A2-mCherry
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI></UL><LI>analytic gel of Mini preps and ligation of the previous day
</LI><LI>low concentration
</LI><LI>Gel (1%)
</LI><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week21
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Construction of new Measurement Plasmid</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="16.08.2010">16.08.2010</SPAN></H3><UL><LI>Concentrating MiniPrep-Samples using ZymoKit
</LI><LI>pSB1A2-mCherry
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI></UL><UL><LI>Digestion
</LI><LI>pSB1A2-mCherry E/P
</LI><LI>pSB1A2-mCherry X/P
</LI></UL><LI>Purified with agarose gel (1%)
</LI><P>
=&gt; no mCherry band!!!
</P><UL><LI>Purification using Zymo Concentrator Kit
</LI><LI>pSB1A10-HisTerm S/P
</LI><LI>pSB1A10-TrpTerm S/P
</LI></UL><UL><LI>Ligation
</LI><LI>Plasmid (from earlier date) and mCherry-Insert (<B>from 11.08</B>)
</LI><LI>ca. 30 min @ RT
</LI><LI><B>used new ligase and new ligase buffer</B></LI></UL><UL><LI>Transformation of DH5a cells with
</LI><LI>ligation samples (=&gt;<B>colonies found the next day</B>)
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI><LI>pSB1A10-RFP (BioBrick Standard)
</LI></UL><UL><LI>overnight cultures
</LI><LI>pSB1A2-mCherry from Christoph`s stock
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI></UL><H3><SPAN class="mw-headline" id="17.08.2010">17.08.2010</SPAN></H3><UL><LI>MiniPrep using ZymoKit
</LI><LI>pSB1A2-mCherry
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI></UL><UL><LI>Digestion
</LI><LI>pSB1A2-mCherry X/P
</LI><LI>pSB1A10-HisTerm S/P
</LI><LI>pSB1A10-TrpTerm S/P
</LI></UL><DL><DD> =&gt; heating block went up to 50°C
</DD></DL><UL><LI>Purified &quot;digestion&quot; samples with ZymoKit =&gt; stored for next day
</LI></UL><UL><LI>Picked 12 colonies from previous day's ligation
</LI></UL><DL><DD> =&gt; Colony PCR =&gt; Gel (2%)
</DD></DL><UL><LI>Purified with agarose gel (1%)
</LI><LI>Gel Doc broken =&gt; no picture
</LI><LI>Description: mCherry cut was ok, stil contaminated with genomic DNA
</LI></UL><LI>Ligation
</LI><LI>Plasmid (from previous day) and mCherry-Insert
</LI><LI>ca. 30 min @ RT
</LI><LI>Transformation of DH5a cells with
</LI><LI>ligation samples (=&gt; no colonies the next day)
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI><UL><LI>overnight cultures
</LI><LI>pSB1A10-RFP (plate from previous day)
</LI><LI>pSB1A2-mCherry
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI></UL><P><B>PROBLEM:</B>
mCherry has a SgrA1-cleavage site! These constructs cannot be used. Starting all over, cloning the linker sequence first...
</P><H3><SPAN class="mw-headline" id="18.08.2010">18.08.2010</SPAN></H3><UL><LI>MiniPrep using ZymoKit
</LI><LI>pSB1A2-mCherry
</LI><LI>pSB1A10-HisTerm
</LI><LI>pSB1A10-TrpTerm
</LI><LI>pSB1A10-RFP
</LI></UL><UL><LI>Analytical agarose gel (1%):
</LI></UL><UL><LI>Digestion
</LI><LI>pSB1A10-HisTerm SgrAI/PstI
</LI><LI>pSB1A10-TrpTerm SgrAI/PstI
</LI><LI>pSB1A10-RFP SgrAI/PstI
</LI></UL><DL><DD> =&gt;RFP has a SrgAI cleaving site. Discarded RFP digestion.
</DD></DL><UL><LI>preparativ agarose gel (1%):
</LI></UL><UL><LI>Soubilization of SrgAI-PstI Linker
</LI></UL><UL><LI>Ligation
</LI><LI>Plasmid His-Term(Trp-Term) and Linker
</LI><LI>ca. 30 min @ RT
</LI></UL><LI>Transformation of DH5a cells with
</LI><LI>ligation samples
</LI><UL><LI>overnight cultures
</LI><LI>pSB1A2-mCherry
</LI><LI>pSB1A10-TrpTerm
</LI></UL><H3><SPAN class="mw-headline" id="19.08.2010">19.08.2010</SPAN></H3><UL><LI>MiniPrep using ZymoKit
</LI><LI>pSB1A2-mCherry
</LI></UL><LI>analytic agarose gel (1%) from various mCherry Preps
</LI><UL><LI>Picked 6 colonies from pSB1A10-HisTerm-linker and pSB1A10-TrpTerm-linker each (previous day's ligation)
</LI></UL><DL><DD> =&gt; Colony PCR =&gt; Gel (1.5%)
</DD></DL><UL><LI>overnight cultures (600µl)
</LI><LI>pSB1A2-mCherry
</LI><LI>pSB1A2-R0011
</LI><LI>pSB1A10-HisTerm-linker (#7, 11, 12)
</LI><LI>pSB1A10-TrpTerm-linker (#1, 2, 4)
</LI></UL><H3><SPAN class="mw-headline" id="20.08.2010">20.08.2010</SPAN></H3><UL><LI>MiniPrep using ZymoKit
</LI><LI>pSB1A2-mCherry
</LI><LI>pSB1A2-R0011
</LI><LI>pSB1A10-TrpLinker (picked Colonies)
</LI><LI>pSB1A10-HisLinker (picked Colonies)
</LI></UL><UL><LI>analytical Digestion
</LI><LI>pSB1A10-HisLinker SgrAI /EcoRI
</LI><LI>pSB1A10-HisLinker NsiI
</LI><LI>pSB1A10-TrpLinker SgrAI /EcoRI</LI></UL><UL><LI>analytical agarose gel (1.5%)
</LI></UL><UL><LI>preparativ Digestions
</LI><LI>pSB1A2-mCherry EcoRI /PstI
</LI><LI>pSB1A2-mCherry XbaI /PstI
</LI><LI>pSB1A2-R0011 SpeI /PstI
</LI><LI>pSB1A10-HisLinker SpeI /PstI
</LI><LI>pSB1A10-HisLinker EcoRI /PstI
</LI><LI>pSB1A10-TrpLinker SpeI /PstI
</LI><LI>pSB1A10-TrpLinker EcoRI /PstI
</LI><LI>PCR_BB1006 XbaI /PstI
</LI></UL><UL><LI>agarose gel (1.5%):
</LI></UL><UL><LI>agarose gel (1.0%):
</LI></UL><UL><LI>agarose gel (1.0%):
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week22
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Construction of new Measurement Plasmid</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="23.08.2010">23.08.2010</SPAN></H3><UL><LI>Ligation
</LI><LI>pSB1A10-HisLinker SpeI /PstI + mCherry XbaI /PstI
</LI><LI>pSB1A10-HisLinker EcoRI /PstI + mCherry EcoRI /PstI
</LI><LI>pSB1A10-TrpLinker SpeI /PstI + mCherry XbaI /PstI
</LI><LI>pSB1A10-TrpLinker EcoRI /PstI + mCherry EcoRI /PstI
</LI><LI>pSB1A2-R0011 SpeI /PstI + PCR_BB1006 XbaI /PstI
</LI></UL><UL><LI>Transformation of DH5a cells
</LI></UL><H3><SPAN class="mw-headline" id="24.08.2010">24.08.2010</SPAN></H3><UL><LI>Picked 2 colonies per plate (previous day's ligation)
</LI><LI>R0011_B1006Sig
</LI><LI>Trp1_mCherry
</LI><LI>Trp2_mCherry
</LI><LI>His7_mCherry
</LI><LI>His11_mCherry
</LI><LI>pSB1A10_mCherry
</LI></UL><UL><LI>Colony PCR of picked colonies with prefix/suffix primers
</LI></UL><UL><LI>analytical agarose gel 1 (1.5%)
</LI></UL><UL><LI>analytical agarose gel 1 (1.5%) 
</LI></UL><P>
Trp=R0011, R0011= Trp :)</P><P>Faint bands at the correct length can be guessed. 
</P><UL><LI>overnight cultures (5 ml)</LI><LI> pSB1A10_TrpTerm_mCherry_linker
</LI><LI> pSB1A10_HisTerm_mCherry_linker
</LI><LI> pSB1A10_mCherry_linker
</LI></UL><H3><SPAN class="mw-headline" id="25.08.2010">25.08.2010</SPAN></H3><UL><LI>Miniprep using Zymo Miniprep-Classic Kit:
</LI><LI> pSB1A10_TrpTerm_mCherry_linker
</LI><LI> pSB1A10_HisTerm_mCherry_linker
</LI><LI> pSB1A10_mCherry_linker
</LI></UL><UL><LI>analytical digestions
</LI><LI> pSB1A10_TrpTerm_mCherry_linker EcoRI /PstI
</LI><LI> pSB1A10_HisTerm_mCherry_linker EcoRI /PstI
</LI><LI> pSB1A10_mCherry_linker EcoRI /PstI
</LI></UL><UL><LI>analytical agarose gel 1 (1.0%)
</LI></UL><UL><LI>Picked 2 colonies per plate (day before yesterday's ligation)
</LI><LI>R0011_B1006Sig
</LI><LI>Trp1_mCherry
</LI><LI>Trp2_mCherry
</LI><LI>His7_mCherry
</LI><LI>His11_mCherry
</LI><LI>pSB1A10_mCherry
</LI></UL><UL><LI>Colony PCR of picked colonies with prefix/suffix primers
</LI><LI> Program: colonypcr
</LI></UL><UL><LI>analytical agarose gel 1 (1.5%)
</LI></UL><UL><LI>analytical agarose gel 1 (1.5%) 
</LI></UL><UL><LI>overnight cultures (5 ml)</LI><LI> pSB1A10_TrpTerm_mCherry_linker
</LI><LI> pSB1A10_HisTerm_mCherry_linker
</LI><LI> pSB1A10_mCherry_linker
</LI></UL><H3><SPAN class="mw-headline" id="26.08.2010">26.08.2010</SPAN></H3><UL><LI>Miniprep using Zymo Miniprep-Classic Kit:
</LI><LI> pSB1A10_TrpTerm_mCherry_linker
</LI><LI> pSB1A10_HisTerm_mCherry_linker
</LI><LI> pSB1A10_mCherry_linker
</LI><LI>pSB1A2_R0011_B1006
</LI></UL><UL><LI>PCR
</LI><LI> Trp-Signal (R0011_Sig_B0014)
</LI></UL><UL><LI> His-Signal (R0011_Sig_B0014)
</LI><LI> Terminator B0014
</LI></UL><UL><LI>analytical digestions
</LI><LI> pSB1A10_mCherry_linker EcoRI /PstI
</LI></UL><LI>preparative digestion
</LI><LI> pSB1A10_TrpTerm_mCherry_linker SpeI/PstI
</LI><LI> pSB1A10_HisTerm_mCherry_linker SpeI/PstI
</LI><LI>pSB1A2_R0011_B1006 SpeI/PstI
</LI><LI> PCR Trp-Sig XbaI/PstI
</LI><LI> PCR His-Sig XbaI/PstI
</LI><LI> B0014 XbaI/PstI
</LI><UL><LI>preparative agarose gel 1 (1.0%)
</LI></UL><P>
last lane: pSB1A10_His11_mCherry SpeI/PstI
</P><UL><LI>analytical agarose gel (1.0 %)
</LI></UL><UL><LI>Ligations
</LI><LI> pSB1A10_TrpTerm_mCherry_linker + Trp-Signal (R0011_Sig_B0014)
</LI><LI> pSB1A10_HisTerm_mCherry_linker + His-Signal (R0011_Sig_B0014)
</LI><LI>pSB1A2_R0011_B1006 + Terminator B0014
</LI></UL><LI>Transformation of Ligation product in DH5alpha cells
</LI><LI>Transformation of pSB1A10_mCherry_linker in BL21
</LI><H3><SPAN class="mw-headline" id="27.08.2010">27.08.2010</SPAN></H3><UL><LI>Mini-Prep
</LI><LI> pSB1A2_R0011_B1006 </LI></UL><P>for sequencing
</P><UL><LI>Colony PCR
</LI><LI>2 colonies per plate
</LI></UL><P>
R0011=R0011_B1006!!!
</P><UL><LI>Sequencing
</LI><LI> pSB1A2_R0011_B1006 4b with primer Biobrick VR
</LI><LI> pSB1A10mod_mCherry 27b with primer GFP_FP and Biobrick VR
</LI><LI> pSB1A10mod_mCherry 32a with primer GFP_FP and Biobrick VR
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week23
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Construction of new Measurement Plasmid</DIV><DIV style="color: #0000CC; border: #0000CC 1px solid; background-color: #8888FF; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Testing new Measurement Plasmid</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="30.08.2010">30.08.2010</SPAN></H3><UL><LI>Colony PCR 
</LI><LI>picked two colonies per plate from 26.08' ligation 
</LI><LI>Program: colonypcr, modified elongation time: 1.15 instead of 1.00
</LI></UL><UL><LI>analytical agarose gel (1.5%)</LI></UL><P><U><B>Gel 1:</B></U></P><P>=&gt;<SPAN style="color: rgb(255, 0, 0);">samples named &quot;1x&quot; to &quot;8x&quot; for pSB1A2-R0011-BB1006Sig-B0014 colonies</SPAN></P><P>=&gt;<SPAN style="color: rgb(255, 0, 0);">samples named &quot;9x&quot; to &quot;12x&quot; for pSB1A10mod-TrpTerm-mCherry-TrpSig colonies</SPAN></P><P><U><B>Gel 2:</B></U></P><P>=&gt;<SPAN style="color: rgb(255, 0, 0);">samples named &quot;13x&quot; to &quot;16x&quot; for pSB1A10mod-TrpTerm-mCherry-TrpSig colonies</SPAN></P><P>=&gt;<SPAN style="color: rgb(255, 0, 0);">samples named &quot;17x&quot; to &quot;24x&quot; for pSB1A10mod-HisTerm-mCherry-HisSig colonies</SPAN></P><P><B>Interpretation for Gel1 and Gel2</B>: 
</P><P>Ligation worked for the samples 1x-6x (pSB1A2-R0011-BB1006Sig-B0014), 9x-16x (pSB1A10mod-TrpTerm-mCherry-TrpSig), 18x-24x (pSB1A10mod-HisTerm-mCherry-HisSig) 
</P><UL><LI>over night cultures: 
</LI><LI>pSB1A10mod_HisTerm_mCherry_HisSignal 
</LI><LI>pSB1A10mod_TrpTerm_mCherry_TrpSignal 
</LI><LI>PSB1A2_R0011_BB1006_B0014
</LI></UL><UL><LI>Received sequencing results from GATC. All Sequences are okay: 
</LI><LI>pSB1A10mod_mCherry (27b) and (32a) 
</LI><LI>pSB1A2_R0011_BB1006 (4b)
</LI></UL><H3><SPAN class="mw-headline" id="31.08.2010">31.08.2010</SPAN></H3><UL><LI><B>Fluorescence measurement (positive control experiment):</B></LI><LI>Settings: GFP-Excitation: 501 nm; mCherry-Excitation: 587 nm;
</LI><LI>endpoint measurements of:
</LI><LI>Timepoints of measurement: 3 h after induction and 9 h after induction (1.5 h and 7 h for 15x-sample)
</LI><LI>Samples:
</LI><LI>pSB1A10mod-mCherry (27b) in BL21 cells, induced with ca. 0.4% L-Arabinose
</LI><LI>pSB1A10mod-mCherry (27b) in BL21 cells, not induced
</LI></UL><LI>kinetic measurement of induced (0.4% L-Arabinose) BL21 cells carrying pSB1A10mod-mCherry (27b)
</LI><UL><LI><B>Results:</B></LI><LI>NO mCherry signal detected at all: The GFP signal shows a nice and strong increase; the RFP channel did not change at all.
</LI><LI>GFP signal looks perfect: strong if induced, neglectable if not!
</LI></UL><DL><DD>=&gt; System seems not capable of serving as a testing system for our switches! 
</DD></DL><UL><LI><B>Glycerol stocks</B></LI><LI>in DH5a cells:
</LI><LI>pSB1A10mod-TrpTerm-mCherry-TrpSig (9x)
</LI><LI>pSB1A10mod-TrpTerm-mCherry-TrpSig (15x)
</LI><LI>pSB1A10mod-TrpTerm-mCherry-TrpSig (10x) (sequence verified)
</LI><LI>pSB1A10mod-HisTerm-mCherry-HisSig (15x)
</LI><LI>pSB1A10mod-HisTerm-mCherry-HisSig (18x)
</LI><LI>pSB1A10mod-HisTerm-mCherry-HisSig (23x) (sequence verified)
</LI><LI>pSB1A10mod-mCherry (32a) (sequence verified)
</LI><LI>pSB1A10mod-mCherry (27b) (sequence verified)
</LI><LI>pSB1A2mod R0011-BB1006Sig-B0014 (2x) (sequence verified)
</LI><LI>pSB1A2mod R0011-BB1006Sig-B0014 (3x)
</LI><LI>pSB1A2mod R0011-BB1006Sig (4b) (sequence verified)
</LI></UL><LI>in BL21 cells:
</LI><LI>pSB1A10mod-mCherry (27b) (sequence verified)
</LI><DL><DD>&quot;x&quot; refers to Colony-PCR of 30.08.2010
</DD></DL><UL><LI>5ml <B>Over night cultures</B></LI><LI>pSB1A10mod-TrpTerm-mCherry-TrpSig (9x, 15x, 10x)
</LI><LI>pSB1A10mod-HisTerm-mCherry-HisSig (18x, 23x, 15x)
</LI><LI>pSB1A2mod R0011-BB1006Sig-B0014 (2x, 3x)
</LI></UL><H3><SPAN class="mw-headline" id="01.09.2010">01.09.2010</SPAN></H3><UL><LI><B>MiniPrep</B> using Zymo classical kit. Samples:
</LI><LI>pSB1A10mod-TrpTerm-mCherry-TrpSig (9x, 15x, 10x)
</LI><LI>pSB1A10mod-HisTerm-mCherry-HisSig (18x, 23x, 15x)
</LI><LI>pSB1A2mod R0011-BB1006Sig-B0014 (2x, 3x)
</LI></UL><UL><LI><B>Fluorescence measurement (positive control experiment):</B></LI><LI>endpoint measurements:
</LI><LI>Timepoints of measurement: 3 h after induction and 9 h after induction (1.5 h and 7 h for 15x-smaple)
</LI><LI>Settings: GFP-Excitation: 501 nm; mCherry-Excitation: 587 nm; RFP-Excitation: 584 nm
</LI><LI>Samples:
</LI><LI>pSB1A10mod-mCherry (32a) in DH5a cells, induced with ca. 0.4% L-Arabinose
</LI><LI>pSB1A10mod-mCherry (32a) in DH5a cells, not induced
</LI><LI>pSB1A10mod-mCherry (32a) in BL21 DE3 cells, induced with ca. 0.4% L-Arabinose
</LI><LI>pSB1A10mod-mCherry (32a) in BL21 DE3 cells, not induced
</LI><LI>pSB1A10-RFP, in DH5a cells, induced with ca. 0.4% L-Arabinose
</LI><LI>pSB1A10-RFP, in DH5a cells, not induced
</LI><LI>pSB1A10mod-TrpTerm-mCherry-TrpSig (15x), in DH5a cells, induced with ca. 0.4% L-Arabinose
</LI><LI>pSB1A10mod-TrpTerm-mCherry-TrpSig (15x), in DH5a cells, not induced
</LI></UL><LI><B>Results:</B></LI><LI>Very strong RFP signal in pSB1A10-RFP, induced and not induced
</LI><LI>For the first time we saw a weak but easily-detectable mCherry signal in positive control samples (pSB1A10mod-mCherry) 3 hours after induction! There was hardly no difference between the uninduced and the induced control samples for mCherry. The GFP signals was strong for induced control experiments and very weak for not induced samples! The pSB1A10mod-TrpTerm-mCherry-TrpSig sample also showd a small mCHerry signal.
</LI><LI>After 9 hours the mCherry signals were generally reduced, whereas the GFP signals were still high in all induced samples and low in all uninduced samples.
</LI><DL><DD>=&gt; Although we saw mCherry for the first time (!), the signal is to weak not reproducable! As a consequence the system can not be used to serve as a measure for our switches! Furthermore the settings of the fluorometer are ok, since we saw strong RFP signal.
</DD></DL><H3><SPAN class="mw-headline" id="02.09.2010">02.09.2010</SPAN></H3><P><B>Starting the cloning of pBAD (BioBrick I13453) downstream of GFP</B></P><UL><LI><B>Amplifing</B> the Arabinose-inducable promotor pBAD
</LI><LI>Resuspending the BioBrick I13453 with 10 µl in well 1F in the 2010 Distribution
</LI><LI>PCR using 1 µl template (programm &quot;igempcr&quot;)
</LI><LI>Purified using DNA Concentrator (ZymoKit)
</LI></UL><UL><LI>Digestion (EcoRI and SpeI) of PCR product and heat inactivation (20 min @ 80°C)
</LI></UL><UL><LI>Digestion of the target vectors using EcoRI and XbaI
</LI><LI>Samples:
</LI><LI>pSB1A10mod-TrpTerm-mCherry-TrpSig (9x, 10x)
</LI><LI>pSB1A10mod-HisTerm-mCherry-HisSig (23x, 24x)
</LI><LI>pSB1A10mod-mCherry (32a, 27b)
</LI><LI>pSB1A10mod-TrpTerm-mCherry (10b, 13b)
</LI><LI>pSB1A10mod-HisTerm-mCherry (18b, 24b)
</LI></UL><LI>Purified using 1% agarose gel:
</LI><DL><DD>=&gt;<B>Interpretation:</B> Gel is overloaded! However digestion seemed to work since the bands show correct masses.
</DD></DL><UL><LI>Extraction of bands at ca. 6000 bp
</LI></UL><UL><LI>10 µl <B>ligation</B> of 50 ng of each digested vector with 8ng insert
</LI></UL><UL><LI><B>Transformation</B> of DH5a cells using 8 µl ligation sample
</LI></UL><H3><SPAN class="mw-headline" id="03.09.2010">03.09.2010</SPAN></H3><UL><LI>Colony PCR 
</LI><LI>picked two colonies per plate from 02.09' ligation 
</LI><LI>Note: no PCR because Thermocycler was occupied
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week24
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;"><I>in vivo</I> constructs</DIV><DIV style="color: #0000CC; border: #0000CC 1px solid; background-color: #8888FF; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Testing HisTrp</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="06.09.2010">06.09.2010</SPAN></H3><UL><LI>Colony PCR 
</LI><LI>using colonies picked on 03.09.
</LI></UL><UL><LI>analytical agarose gel (1.5%)
</LI></UL><UL><LI>K1=  pSB1A10mod_HisTerm_mCherry_HisSig
</LI><LI>K2 = pSB1A10mod_TrpTerm_mCherry_TrpSig
</LI></UL><UL><LI>5ml over night cultures:
</LI><LI>positive control pSB1A10mod_Pbad_mCherry
</LI><LI>negative His control pSB1A10mod_Pbad_HisTerm_mCherry
</LI><LI>His-Switch pSB1A10mod_Pbad_HisTerm_mCherry_HisSig
</LI><LI>Trp-Switch pSB1A10mod_Pbad_TrpTerm_mCherry_TrpSig
</LI></UL><H3><SPAN class="mw-headline" id="07.09.2010">07.09.2010</SPAN></H3><UL><LI>MiniPrep using Zymo Classical kit
</LI><LI>pSB1A10mod_Pbad_mCherry
</LI><LI>pSB1A10mod_Pbad_HisTerm_mCherry
</LI><LI>pSB1A10mod_Pbad_HisTerm_mCherry_HisSig
</LI><LI>pSB1A10mod_Pbad_TrpTerm_mCherry_TrpSig
</LI></UL><UL><LI>Fluorescence measurements
</LI><LI>Induction with 0.2% L-arabinose
</LI><LI>Measurement of OD600
</LI><LI>Fluorescence measurement at 30 min, 150 min (only Pos.Control) and 4.5 h
</LI><LI>Samples:
</LI></UL><DL><DD>pSB1A10mod_Pbad_mCherry
</DD><DD>pSB1A10mod_Pbad_HisTerm_mCherry
</DD><DD>pSB1A10mod_Pbad_HisTerm_mCherry_HisSig
</DD><DD>pSB1A10mod_Pbad_TrpTerm_mCherry_TrpSig
</DD></DL><UL><LI>Transformation of BL21 (DE3)
</LI><LI>pSB1A10mod_Pbad_mCherry
</LI><LI>pSB1A10mod_Pbad_HisTerm_mCherry
</LI><LI>pSB1A10mod_Pbad_HisTerm_mCherry_HisSig
</LI><LI>pSB1A10mod_Pbad_TrpTerm_mCherry_TrpSig
</LI></UL><H3><SPAN class="mw-headline" id="08.09.2010">08.09.2010</SPAN></H3><UL><LI>Glycerolstocks
</LI></UL><DL><DD>30%Glycerol in LB_Carb
</DD></DL><UL><LI>pSB1A10mod_Pbad_mCherry
</LI><LI>pSB1A10mod_Pbad_HisTerm_mCherry
</LI><LI>pSB1A10mod_Pbad_HisTerm_mCherry_HisSig
</LI><LI>pSB1A10mod_Pbad_TrpTerm_mCherry_TrpSig
</LI></UL><UL><LI>Fluorescence measurements
</LI><LI>Induction with 0.2% L-arabinose
</LI><LI>Measurement of OD600
</LI><LI>Fluorescence measurement at 24 h and different OD600
</LI><LI>Samples:
</LI></UL><DL><DD>pSB1A10mod_Pbad_mCherry
</DD></DL><P>--&gt;OD600 0.05 reasonable for our cell measurements. Positive control works fine after 24 h induction.
</P><UL><LI>5ml cultures of BL21 cells
</LI><LI>pSB1A10mod_Pbad_mCherry
</LI><LI>pSB1A10mod_Pbad_HisTerm_mCherry
</LI><LI>pSB1A10mod_Pbad_HisTerm_mCherry_HisSig
</LI><LI>pSB1A10mod_Pbad_TrpTerm_mCherry_TrpSig
</LI></UL><UL><LI>ColonyPCR
</LI><LI>picked 4 Colonies per Neg.TrpControl from 02.09' ligation plates
</LI><LI>PCR using Program 'colonyPCR'. Elongation time modified to 1:20min
</LI></UL><H3><SPAN class="mw-headline" id="09.09.2010">09.09.2010</SPAN></H3><UL><LI>Fluoresence measurement
</LI><LI>using BL21 cells
</LI><LI>samples:
</LI><LI>Positive control (pSB1A10mod_pBAD_mCherry)
</LI><LI>Negative control (pSB1A10mod_pBAD_HisTerm_mCherry)
</LI><LI>pSB1A10mod_pBAD_HisTerm_mCherry_HisSignal
</LI><LI>pSB1A10mod_pBAD_TrpTerm_mCherry_TrpSignal
</LI></UL><LI>Induction with 0.4% L-arabinose and 1mM IPTG
</LI><LI>Timepoints: 1 h, 2 h, 4 h, 10 h, 16 h (induced the day before)
</LI><DL><DD>(OD checked seperately each time; average ODs=0.02-0.06)
</DD></DL><UL><LI>30% glycerol stocks of BL21 cells carrying:
</LI><LI>Positive control (pSB1A10mod_pBAD_mCherry)
</LI><LI>Negative control (pSB1A10mod_pBAD_HisTerm_mCherry)
</LI><LI>pSB1A10mod_pBAD_HisTerm_mCherry_HisSignal
</LI><LI>pSB1A10mod_pBAD_TrpTerm_mCherry_TrpSignal
</LI></UL><H3><SPAN class="mw-headline" id="10.09.2010">10.09.2010</SPAN></H3><UL><LI>Fluorescence measurements
</LI><LI>Induction with 0.4% L-arabinose and 1 mM IPTG
</LI><LI>Measurement of OD600
</LI><LI>Fluorescence measurement at 15 h and 26 h 
</LI></UL><LI>Samples:
</LI><DL><DD>pSB1A10mod_Pbad_mCherry - Positive Control
</DD><DD>pSB1A10mod_Pbad_HisTerm_mCherry - HisNeg. Control
</DD><DD>pSB1A10mod_Pbad_HisTerm_mCherry_HisSignal
</DD><DD>pSB1A10mod_Pbad_TrpTerm_mCherry_TrpSignal
</DD></DL><UL><LI>Conclusion:
</LI></UL><P>His-Switch DOES NOT seem to work!!! Measurements of Trp-Switch look weird, since Ara-induced culture showed strong mCherry signal than Ara+IPTG-induced culture. Maybe growing condition were not sufficient. We will use 50 ml flask next time!
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week25
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Cloning</DIV><DIV style="color: #0000CC; border: #0000CC 1px solid; background-color: #8888FF; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">HisTerm/TrpTerm</DIV><DIV style="color: #00BB00; border: #00BB00 1px solid; background-color: #55FF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Ordering Aptamer</DIV><DIV style="color: #BBBB00; border: #BBBB00 1px solid; background-color: #FFFF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; float: left;">New Measurement Plasmid</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="13.09.2010">13.09.2010</SPAN></H3><UL><LI><I>In vitro</I> Kit measurement:
</LI><LI>Test of <I>In vitro</I> kit with positive control
</LI><LI>1 h on 37°C
</LI></UL><DL><DD>--&gt; no significant mCherry signal detectable. <I>In vitro</I> Kit measurements aborted. 
</DD></DL><UL><LI>20ml cultures for Fluorescence measurements
</LI><LI>Samples:
</LI></UL><DL><DD>pSB1A10mod_Pbad_mCherry - Positive Control
</DD><DD>pSB1A10mod_Pbad_HisTerm_mCherry - HisNeg. Control
</DD><DD>pSB1A10mod_Pbad_TrpTerm_mCherry - TrpNeg. Control
</DD><DD>pSB1A10mod_Pbad_HisTerm_mCherry_HisSignal
</DD><DD>pSB1A10mod_Pbad_TrpTerm_mCherry_TrpSignal
</DD></DL><DL><DD><UL><LI>Growth on 37°C until OD600 = 0.6, then induction and growth on 25°C over night
</LI><LI>each sample uininduced, induced with 0.4% L-arabinose and 0.4% L-arabinose + 1 mM IPTG,respectively
</LI></UL></DD></DL><UL><LI> 30% Glycerol stock in LB
</LI><LI>pSB1A10mod_Pbad_TrpTerm_mCherry - TrpNeg. Control
</LI></UL><H3><SPAN class="mw-headline" id="14.09.2010">14.09.2010</SPAN></H3><UL><LI>Fluorescence measurements
</LI><LI>Induction with 0.4% L-arabinose and 1mM IPTG
</LI><LI>Measurement of OD600
</LI><LI>Fluorescence measurement at 16 h
</LI></UL><LI>Samples:
</LI><DL><DD>pSB1A10mod_Pbad_mCherry - Positive Control
</DD><DD>pSB1A10mod_Pbad_HisTerm_mCherry - HisNeg. Control
</DD><DD>pSB1A10mod_Pbad_TrpTerm_mCherry - TrpNeg. Control
</DD><DD>pSB1A10mod_Pbad_HisTerm_mCherry_HisSignal
</DD><DD>pSB1A10mod_Pbad_TrpTerm_mCherry_TrpSignal
</DD></DL><H3><SPAN class="mw-headline" id="15.09.2010">15.09.2010</SPAN></H3><UL><LI>Waiting for Mr.Gene...
</LI></UL><H3><SPAN class="mw-headline" id="16.09.2010">16.09.2010</SPAN></H3><UL><LI>Searching for Mr.Gene package...
</LI></UL><DL><DD>--&gt; our new Testsystem pMalachitApt_BB1006
</DD></DL><UL><LI>PCR and PCR purification with Zymo DNA Clean&amp;Concentrator
</LI><LI>pMalachitApt_BB1006 with terminator (Primer: Apt_For and AptFull_wT_Rev)
</LI><LI>pMalachitApt_BB1006 without terminator (Primer: Apt_For and AptPart_woT_Rev)
</LI><LI>Switches
</LI></UL><DL><DD>Trp-Switch
</DD><DD>His-Switch
</DD></DL><UL><LI>Signals
</LI></UL><DL><DD>His-Sig with Term (template: pSB1A2_R0011_HisSig_B0014)
</DD><DD>Trp-Sig with Term (template: pSB1A2_R0011_TrpSig_B0014)
</DD><DD>BB1006Sig with Term (template: pSB1A2_R0011_BB1006Sig_B0014)
</DD><DD>BB1006Sig without Term (template: pSB1A2_R0011_BB1006Sig)
</DD><DD>HisSig (ssDNA) (template: original biomers order)
</DD><DD>TrpSig (ssDNA) (template: original biomers order)
</DD></DL><H3><SPAN class="mw-headline" id="17.09.2010">17.09.2010</SPAN></H3><UL><LI>Digestion
</LI><LI>pMalachitApt_BB1006 with EcoRI/PstI
</LI><LI>pMalachitApt_BB1006 with XbaI/SpeI
</LI><LI>Switches
</LI></UL><DL><DD>Trp-Switch with EcoRI/PstI
</DD><DD>His-Switch with EcoRI/PstI
</DD></DL><UL><LI>Signals
</LI></UL><DL><DD>His-Sig with XbaI/PstI
</DD><DD>Trp-Sig with XbaI/PstI
</DD></DL><UL><LI>Purification using Qiagen Kit
</LI></UL><DL><DD>--&gt; Cut off size of Qiagen kit too high. Lost all Switches and Signals
</DD></DL><UL><LI>Purification pMalachitApt using preparative agarose gel
</LI></UL><UL><LI>new PCR:
</LI><LI>Switches
</LI></UL><DL><DD>Trp-Switch
</DD><DD>His-Switch
</DD></DL><UL><LI>Signals
</LI></UL><DL><DD>HisSig (ssDNA) (template: original biomers order)
</DD><DD>TrpSig (ssDNA) (template: original biomers order)
</DD></DL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week26
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Cloning of Malachit green constructs</DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="color: #00BB00; border: #00BB00 1px solid; background-color: #55FF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">HisTerm/TrpTerm</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="20.09.2010">20.09.2010</SPAN></H3><UL><LI>PCR Purification using Qiagen MinElute Kit
</LI><LI>Trp-Signal
</LI><LI>His-Signal
</LI><LI>Trp-Switch
</LI><LI>His-Switch
</LI></UL><UL><LI>Digestions 37°C, 2h
</LI><LI>Trp-Signal XbaI/PstI
</LI><LI>His-Signal XbaI/PstI
</LI><LI>Trp-Switch EcoRI/PstI
</LI><LI>His-Switch EcoRI/PstI
</LI></UL><DL><DD>--&gt; Heat inactivation 20 min, 80°C
</DD></DL><UL><LI>Ligation
</LI><LI>pSB1A2_R0011 SpeI/PstI + Trp-Signal XbaI/PstI
</LI><LI>pSB1A2_R0011 SpeI/PstI + His-Signal XbaI/PstI
</LI><LI>pMalachit EcoRI/PstI + Trp-Switch EcoRI/PstI
</LI><LI>pMalachit EcoRI/PstI + His-Switch EcoRI/PstI
</LI><LI>pMalachit XbaI/SpeI
</LI></UL><UL><LI>Transformation of DH5a cells with ligation
</LI></UL><H3><SPAN class="mw-headline" id="21.09.2010">21.09.2010</SPAN></H3><UL><LI>Picked 2 colonies per plate of yesterday's transformation =&gt; we call them z-Series
</LI><LI>ColonyPCR of z-Series
</LI><LI>analytical agarose gel (2.5%)
</LI></UL><DL><DD>=&gt; ColonyPCR did not work. Probably not enough template.
</DD></DL><UL><LI>Picked futher colonies
</LI><LI>2nd ColonyPCR of z-Series
</LI></UL><H3><SPAN class="mw-headline" id="22.09.2010">22.09.2010</SPAN></H3><UL><LI>analytical agarose gel (2.5%) of 2nd colonyPCR of z-Series
</LI></UL><DL><DD>pSB1A2-R0011-TrpSignal looks good. We removed all pSB1A2-R0011-HisSignal samples since there is a contamination on the gel and colonies turned redish. No conclusion regarding the other ligations of z_series can be drawn. LB control produces similar bands as to what we expected =&gt; Choose new Primers for PCR.
</DD></DL><UL><LI>3rd colonyPCR of z-Series (samples 1-16z)
</LI><LI>used Primers Apt_For and Apt_Part_woT instead of BioBrick Primers (G1005 and G1004). Bands should be about 100bp longer.
</LI></UL><UL><LI>analytical agarose gel (2%)
</LI></UL><H3><SPAN class="mw-headline" id="23.09.2010">23.09.2010</SPAN></H3><UL><LI>PCR to obtain DNA for malachite green <I>in vitro</I> measurements
</LI><LI>samples from z-Series (see <A href="/wiki/index.php?title=21.09.2010&amp;action=edit&amp;redlink=1" class="new" title="21.09.2010 (page does not exist)">21.09.2010</A>)
</LI></UL><DL><DD>2z (XbaI/SpeI religated positive control)
</DD><DD>16z (pMalachitApt_HisSig positive control)
</DD></DL><UL><LI>each samples was amplificated with two sets of primers (including and excluding the terminator BB0014):
</LI></UL><DL><DD>Apt_For and AptFull_wT_Rev
</DD><DD>Apt_For and AptPart_woT_Rev
</DD></DL><UL><LI>Purified with Qiagen MinElute Kit
</LI></UL><UL><LI>analytical digestion of z-series ligation:
</LI><LI>EcoRI/PstI:
</LI></UL><DL><DD>2z (XbaI/SpeI religated positive control)
</DD><DD>5z (pMalachitApt_TrpTerm)
</DD><DD>12z (pMalachitApt_HisTerm)
</DD><DD>16z (pMalachitApt_HisSig positive control)
</DD><DD>28z (pSB1A2_R0011_TrpSig)
</DD></DL><UL><LI>NsiI only (HisTerm contains one cutting site for NsiI, so does pMalachiteApt =&gt; we expect 2 fragments):
</LI></UL><DL><DD>12z (pMalachitApt_HisTerm)
</DD></DL><UL><LI>analytical agarose gel:
</LI></UL><UL><LI>10 µl ligations using digested samples from <A href="/wiki/index.php?title=21.09.2010&amp;action=edit&amp;redlink=1" class="new" title="21.09.2010 (page does not exist)">21.09.2010</A></LI><LI>pSB1A2_R0011 (SpeI/PstI cut) + HisSig (XbaI/PstI cut)
</LI><LI>pMalachiteApt (EcoRI/PstI cut) + TrpSwitch (EcoRI/PstI cut)
</LI><LI>pMalachiteApt (EcoRI/PstI cut) + HisSwitch (EcoRI/PstI cut)
</LI></UL><UL><LI>Transformed DH5a cell with 8µl of the above ligation samples
</LI></UL><H3><SPAN class="mw-headline" id="24.09.2010">24.09.2010</SPAN></H3><UL><LI>Fluorescence measurement: Malachite Green Assay
</LI><LI>Samples
</LI></UL><DL><DD>2z positive control (X/S religated: tac_MalachitApt) (PCR-amplified without BB0014 Terminator)
</DD><DD>2z positive control (X/S religated: tac_MalachitApt_BB0014) (PCR-amplified with BB0014 Terminator)
</DD><DD>negative control (tac_BB1006Switch_MalachitApt_BB0014) (PCR-amplified with BB0014 Terminator)
</DD><DD>tac_BB1006Switch_MalachitApt + tac_BB1006Signal (both PCR-amplified without BB0014 Terminator)
</DD></DL><UL><LI>Sample Mix:
</LI></UL><DL><DD>2 µl sigma70 satured RNA <I>E.Coli</I> Polymerase (epicenter Biozyme) (2U)
</DD><DD>ca. 1µg DNA template
</DD><DD>5 µM Malachite Green
</DD><DD>10 µM DTT
</DD><DD>4 µM NTPs
</DD><DD>40 mM Tris-HCl pH = 7.1 @ 37°C; 7.5 @ 22°C
</DD><DD>10 mM MgCl2
</DD><DD>150 mM KCl
</DD><DD>added H20 to total volume of 100 µl
</DD></DL><UL><LI>Kinetics measurement @ 37°C
</LI></UL><DL><DD><UL><LI>Excitation 630 nm
</LI><LI>Emission 655 nm
</LI></UL></DD></DL><DL><DD><UL><LI>Results: 
</LI></UL></DD><DD>no apatmer formation observed =&gt; positive controls did not work at all =&gt; assay did not work!!
</DD></DL><DL><DD><UL><LI>Scanning measurements
</LI></UL></DD><DD><UL><LI>Exitation: 630 nm
</LI><LI>Results: 
</LI></UL></DD><DD>no detactable peaks between 640nm and 800nm in any sample at any measured time (Settings were checked by detecting scattering peaks).
</DD></DL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week27
</A></P><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="color: #00BB00; border: #00BB00 1px solid; background-color: #55FF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">T7 Switch</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="27.09.2010">27.09.2010</SPAN></H3><UL><LI>Fluoresence measurements
</LI><LI>Cary Spectrometer
</LI><LI>samples:
</LI><LI>with T7 RNA Polymerase (NEB)
</LI></UL><DL><DD>T7 positive control (T7Promotor_MalachitAptamer)
</DD></DL><UL><LI>with Epicenter E coli RNA Polymerase
</LI></UL><DL><DD>2z positive control (X/S religated: tac_MalachitApt) (PCR-amplified without BB0014 Terminator)
</DD><DD>2z positive control (X/S religated: tac_MalachitApt_BB0014) (PCR-amplified with BB0014 Terminator) (NO KINETICS RECORDED)
</DD></DL><UL><LI>with in vitro kit (cell lysate)
</LI></UL><DL><DD>2z positive control (X/S religated: tac_MalachitApt) (PCR-amplified without BB0014 Terminator)
</DD><DD>2z positive control (X/S religated: tac_MalachitApt_BB0014) (PCR-amplified with BB0014 Terminator)
</DD></DL><UL><LI>Scanning measurements: excitation at 630nm
</LI><LI>Kinetics measurements: excitation at 630nm; Emission at 655nm
</LI></UL><P><B>Results:</B>
T7 Polymerase works perfectly. E coli Polymerase also produced RNA but much less (however enzyme might show reduced activity due to storage problems). In vitro (Cell lysate) kits do not work at all.
</P><H3><SPAN class="mw-headline" id="01.10.2010">01.10.2010</SPAN></H3><P><B>T7-Measurments</B></P><UL><LI> PCR of switch_phi_T7-construct to obtain dsDNA (conditions: igemPCR and 30s elongation time). Samples were purified using Qiagen MinElute.
</LI></UL><UL><LI> 1th measurment:  Maxi´s NEB buffer conditions: 40 mM Tris pH 7.4 @ RT, 40mM Mg2Cl, 5 µM Malachit-green, 4 mM NTPs, 2,5 U RNA-Polymerase All DNA templates were all added to a final concentration of 200nM.
</LI><LI>sample 1: Positive control
</LI><LI>sample 2: negative control (= switch without any signal) (<B>new DTT</B> used for this sample)
</LI><LI>sample 3: switch + SigA1a
</LI><LI>sample 4: switch + SigA1c
</LI></UL><LI>Results
</LI><LI>kinetic results </LI><LI>Spectra </LI><P>=&gt; Switches do look quite good. However DTT was not the same in all samples.
</P><UL><LI> 2th measurment:
</LI><LI>&quot;Paper´s&quot; buffer conditions: 40 mM Tris pH 7.9 @ RT, 6mM Mg2Cl, 5 µM Malachit-green, 100 mM KCl, 0.8 mM NTPs, 2,5 U RNA-Polymerase
</LI><LI>sample 1: Positive control
</LI><LI>sample 2: negative control (= switch without signal)
</LI></UL><LI>Maxi´s NEB buffer conditions: 40 mM Tris, 40 mM Mg2Cl, 10 µM Malachit-green, ph 7.4 @ RT
</LI><LI>sample 3: Positive control (<B>new DTT</B> used for this sample)
</LI><LI>sample 4: Negative control (= switch without signal)
</LI><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week28
</A></P><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="color: #00BB00; border: #00BB00 1px solid; background-color: #55FF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">T7 Switch</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="04.10.2010">04.10.2010</SPAN></H3><UL><LI>1st fluorescence measurement
</LI></UL><DL><DD><UL><LI>Samples:
</LI></UL></DD><DD><UL><LI>200 nM T7-Switch (100 nM each strand) + SignalA_1a
</LI><LI>200 nM T7-Switch (100 nM each strand) + SignalA_2a
</LI><LI>200 nM T7-Switch (100 nM each strand) + SignalA_2b
</LI><LI>200 nM T7-Switch (100 nM each strand) + SignalA_2c
</LI></UL></DD></DL><UL><LI>Buffer:
</LI></UL><DL><DD>40 mM Tris pH 7.1
</DD><DD>40 mM MgCl2
</DD><DD>10 mM DTT
</DD><DD>1.6 mM NTPs
</DD><DD>5 µM Malachite green
</DD><DD>2.5 U T7 RNA Polymerase
</DD></DL><UL><LI>Results:
</LI></UL><DL><DD>Fluorescence increased in all samples in a very similar way, suggesting all signal to work equally well.
</DD></DL><UL><LI>2nd fluorescence measurement
</LI></UL><DL><DD><UL><LI>Samples:
</LI></UL></DD><DD><UL><LI>200 nM T7-Switch only (Buffer A)
</LI><LI>200 nM T7-Switch only (Buffer B)
</LI><LI>positiv control (T7Promoter_MalchitAptamer) (Buffer A)
</LI><LI>positiv control (T7Promoter_MalchitAptamer) (Buffer B)
</LI></UL></DD></DL><UL><LI>Buffer A:
</LI></UL><DL><DD>40 mM Tris pH 7.1 @ RT
</DD><DD>40 mM MgCl2
</DD><DD>10 mM DTT
</DD><DD>1.6 mM NTPs
</DD><DD><B>10 µM</B> Malachite green
</DD><DD>2.5 U T7 RNA Polymerase
</DD></DL><UL><LI>Buffer B:
</LI></UL><DL><DD>40 mM Tris pH 7.9 @ RT
</DD><DD>6 mM MgCl2
</DD><DD>100 mM KCl
</DD><DD>10 mM DTT
</DD><DD>1.6 mM NTPs
</DD><DD><B>10 µM</B> Malachite green
</DD><DD>2.5 U T7 RNA Polymerase
</DD></DL><UL><LI>Results:
</LI></UL><DL><DD><UL><LI>Very strange results!! Both negative controls increased strongly compared to the positive controls that only slightly increased (buffer A seems to be better for transcription than buffer B).
</LI></UL></DD><DD>=&gt; Maybe something went very wrong. However, last Friday we observed a similar, strange behavior. Maybe the positive control is no good choice. It would make sence to use T7-Switch+SignalA_1a as a reference.
</DD></DL><UL><LI>Signals were generally stronger. 10 µM of malachite green seems to be quite good.
</LI></UL><H3><SPAN class="mw-headline" id="05.10.2010">05.10.2010</SPAN></H3><UL><LI> T7 <I>in vitro measurements</I></LI></UL><UL><LI> First measurement of the day: Signal, 1d, 2c, 3b
</LI><LI> Master Mix:
</LI><LI> 204 µl 2x Buffer
</LI><LI> 40.8 µl DTT
</LI><LI> 8.16 µl NTPs
</LI><LI> 21.05 µl Switch (1.94 µM)
</LI><LI> signals: # volume µl
</LI><LI> negative control - 1.213
</LI><LI> 1d - 1.290
</LI><LI> 2c - 1.450
</LI><LI> 3b - 1.376
</LI></UL><UL><LI> PCR with positive control and T7 switch
</LI><LI> Mastermix:
</LI></UL><P>PCR T7 switch/positive signal
</P><UL><LI> 32 x Mastermix: 1600 µl total volume
</LI><LI> 32 µl dNTPs
</LI><LI> 32 µl T7 forward primer
</LI><LI> 32 µl T7 reverser primer
</LI><LI> 160 µl 10x buffer
</LI><LI> 96 µl MgCl2
</LI><LI> 6.4 µl Taq Polymerase
</LI><LI> 10 µl Template (some older PCR)
</LI><LI> 1209.6 µl H2O
</LI></UL><UL><LI> Second measurement of the day: Signal negative control, nonsense signal, 1a (with 400 µM signal), 1a (with 2 µM signal)
</LI><LI> *** Master Mix:
</LI><LI> 204 µl 2x Buffer
</LI><LI> 40.8 µl DTT
</LI><LI> 8.16 µl NTPs
</LI><LI> 21.05 µl Switch (1.94 µM)
</LI><LI> signals: # volume µl
</LI><LI> negative control - 30.34
</LI><LI> 1d - 1.213
</LI><LI> 2c - 2.672
</LI><LI> 3b - 13.36
</LI></UL><UL><LI> Evaluation of different malachite green concentrations
</LI><LI> 5 µM, 10 µM, 15 µM, 25 µM chosen
</LI><LI> high temperature dependency
</LI><LI> Equilibration of samples important
</LI></UL><H3><SPAN class="mw-headline" id="06.10.2010">06.10.2010</SPAN></H3><UL><LI> PCR
</LI><LI> yesterday's PCR Purification using Zymo Clear and Concentrated
</LI><LI> yields
</LI><LI> Switch: 164 ng/µl
</LI><LI> positive control: 28 ng/µl
</LI><LI> problems with the temperature? Yield too low!
</LI></UL><UL><LI> PCR of positive control
</LI><LI> annealing temperature set to 48°C
</LI><LI> 20 x Mastermix:
</LI><LI> 20 µl dNTPs
</LI><LI> 20 µl T7 forward primer
</LI><LI> 20 µl T7 reverser primer
</LI><LI> 100 µl 10x buffer
</LI><LI> 60 µl MgCl2
</LI><LI> 4 µl Taq Polymerase
</LI><LI> 1 µl Template (some older PCR)
</LI><LI> 775 µl H2O
</LI></UL><H3><SPAN class="mw-headline" id="07.10.2010">07.10.2010</SPAN></H3><UL><LI> T7 Trancription
</LI><LI> new buffer:
</LI><LI> Stocks, volume for 20 ml, endconcentration in 1x
</LI><LI> 1M Tris/HCl, 1.6 ml, 40 mM
</LI><LI> 500 mM MgCl2, 3.2 ml, 40 mM
</LI><LI> 250 mM malachite green, 1.6 ml, 20 µM
</LI><LI> H20
</LI></UL><LI> Measurement
</LI><LI> 4.1 x, switch, switch + nonsense, switch + 1a, switch + 1c
</LI><LI> 21.16 µl switch, 1.93 µM
</LI><LI> 10.25 µl RPO
</LI><LI> 205 µl buffer
</LI><LI> 41 µl DTT 100 µM
</LI><LI> 20.5 µl rNTPs
</LI><LI> H2O
</LI><UL><LI>second measurement
</LI><LI> Measurement
</LI><LI> 4.1 x, switch, switch + nonsense, positive control + 1b, positive control + 1b
</LI><LI> 21.16 µl switch, 1.93 µM
</LI><LI> 10.25 µl RPO
</LI><LI> 205 µl buffer
</LI><LI> 41 µl DTT 100 µM
</LI><LI> 20.5 µl rNTPs
</LI><LI> H2O
</LI></UL><H3><SPAN class="mw-headline" id="08.10.2010">08.10.2010</SPAN></H3><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week29
</A></P><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="color: #00BB00; border: #00BB00 1px solid; background-color: #55FF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">T7 Switch</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="11.10.2010">11.10.2010</SPAN></H3><UL><LI> PCR T7 switch
</LI><LI> 32 x Mastermix: 1600 µl total volume
</LI><LI> 32 µl dNTPs
</LI><LI> 32 µl T7 forward primer
</LI><LI> 32 µl T7 reverser primer
</LI><LI> 160 µl 10x buffer
</LI><LI> 96 µl MgCl2
</LI><LI> 6.4 µl Taq Polymerase
</LI><LI> 10 µl Template (some older PCR)
</LI><LI> 1209.6 µl H2O
</LI></UL><UL><LI> Standart iGEM PCR program:
</LI><LI> 95°C, 2' 
</LI><LI> 1) 95°C, 0,5'
</LI><LI> 2) 58°C, 0.5'
</LI><LI> 3) 71°C, 1'
</LI><LI> 71°C, 7'
</LI></UL><UL><LI> repeat 1)-3) 35 times
</LI><LI> yield after purification: 80 µl, 1.14 µM
</LI></UL><UL><LI>Malchitegreen measurement
</LI><LI> for 4.1 x 100 µl
</LI><LI> 35.88 µl PCR product
</LI><LI> 20.05 dNTPs
</LI><LI> 41 µl DTT
</LI><LI> 10.25 µl T7 RPO
</LI><LI> 8.75 µl switch
</LI><LI> 2 µl signal per 100 µl
</LI></UL><UL><LI> measured signals: (1) nonsense, (2) 1a, (3) 1b, (4) 2a
</LI><LI> Voltage set to 1000 (maximum)
</LI><LI> 38°C, every 15 s measurement
</LI><LI> start at 5 au, end at 25 au
</LI><LI> rise visible over time in all samples, nonsense signal with the fastest rise, 1a similiar, end signal similiar in all measurements
</LI></UL><H3><SPAN class="mw-headline" id="12.10.2010">12.10.2010</SPAN></H3><UL><LI> PCR T7 switch
</LI><LI> 32 x Mastermix: 1600 µl total volume
</LI><LI> 32 µl dNTPs
</LI><LI> 32 µl T7 forward primer
</LI><LI> 32 µl T7 reverser primer
</LI><LI> 160 µl 10x buffer
</LI><LI> 96 µl MgCl2
</LI><LI> 6.4 µl Taq Polymerase
</LI><LI> 10 µl Template (some older PCR)
</LI><LI> 1209.6 µl H2O
</LI></UL><UL><LI><B>changed</B> iGEM PCR program:
</LI><LI> 95°C, 2' 
</LI><LI> 1) 95°C, 0,5'
</LI><LI> 2) 50°C, 0.5'
</LI><LI> 3) 71°C, 1'
</LI><LI> 71°C, 10'
</LI><LI> repeat 1) - 3) 35 times
</LI></UL><UL><LI> --&gt; melting temperature primers: 55/54°C!!!
</LI><LI> yield after purification: 300 ng/µl in 60 µl, 3.66 µM
</LI></UL><UL><LI> Malachitegreen measurement with preincubation of transcription stuff with signals
</LI></UL><UL><LI> for 4.1 x 100 µl
</LI><LI> 35.88 µl PCR product
</LI><LI> 20.05 dNTPs
</LI><LI> 41 µl DTT
</LI><LI> 10.25 µl T7 RPO
</LI><LI> 2 µl signal per 100 µl
</LI></UL><UL><LI> preincubation of signal with RPO for one hour, 37°C 
</LI><LI> addition of signal after one hour (2.735 µl)
</LI><LI> once done in eppis (low bind), once in cuvettes
</LI><LI> no rise in both
</LI><LI> incubation and measurement of cuvette-incubated mix over night: no rise visible
</LI></UL><UL><LI> 15 % denaturing acrylamide gels
</LI><LI> for 50 ml
</LI><LI> 15 % acrylamide: 18.75 ml 40 % acrylamide
</LI><LI> 6M urea: 18 g
</LI><LI> 1x TBE: 5 ml 10x TBE
</LI><LI> 500 µl APS
</LI><LI> 50 µl TEMED
</LI><LI> H20 till 50 ml
</LI></UL><UL><LI> only 30 ml needed for two gels
</LI><LI> big combs for a lot of sample :)
</LI><LI> glass plates cleaned with RNAseZip before pouring the gel
</LI></UL><H3><SPAN class="mw-headline" id="13.10.2010">13.10.2010</SPAN></H3><UL><LI> in vitro T7 transcription, check by polyacrylamide gel electrophoresis 
</LI><LI> 10 ml 5 x paper buffer
</LI><LI> 200 mM Tris/HCl, pH=7.85: 2 ml 1M Tris/HCl, pH=7.85
</LI><LI> 30 mM MgCl2: 600 µl 0.5 M MgCl2
</LI><LI> 500 mM KCl: 5 ml 1 M KCl
</LI><LI> 2.4 ml H2O
</LI></UL><UL><LI> for in vitro transcription
</LI><LI> 0.5 µl T7 RPO: 25 U
</LI><LI> 0.5 µl rNTPs : 20 mM
</LI><LI> 1.36 µl switch: 250 µM
</LI><LI> 1 µl signal: 250 µM
</LI><LI> 2 µl DTT: 10 µM
</LI><LI> 0.2 µl RNase inhibitor
</LI><LI> 4 µl 5x buffer
</LI><LI> 10.44 µl H2O
</LI></UL><UL><LI>5.1 times
</LI><LI> no signal, nonsense, 1a, 1c, 2a
</LI><LI> in low bind tubes
</LI><LI> 2 hour, 37°C
</LI><LI> actually 2 hours and half a hour pocket cleaning time
</LI></UL><UL><LI> for PAGE:
</LI><LI> don't try to run yourgel in the pouring device - if you do so: feel very stupid and embarrassed (I guess I've ran over 100 gels in my life yet... still too stupid...) 
</LI><LI> don't feel tempted to use the Dietz' group's 0.5 x TBE buffer - if you do so: feel very stupid and embarassed, discard buffer carefully and use 1 x TBE
</LI><LI> don't use a comb which is thinner than your spacers - if you do so: scrap gel pieces out of the pockets for half an hour
</LI><LI> cook your sample for 5 minutes, 95°C - if you do not so, feel stupid, embarassed and hope that it won't matter so much
</LI><LI> I did not cool the samples
</LI><LI> gel runs at 100 V (~ 10V/cm)
</LI><LI> 20 µl sample + 20 µl ambion loading buffer (1-2 x loading buffer - who does that?) --&gt; 40 µl fits nicely
</LI><LI> 2 µl low molecular weight marker + 20 µl loading buffer + 18 µ H2O
</LI><LI> in 1 x TBE
</LI></UL><UL><LI> M, control (same amounts switch and signals as in the samples), no signal, random, 1a, 2a, 1c, random overnight, 1a overnight, 1c overnight
</LI><LI> overnight samples: measurement with malachitegreen sample over night: no rise visible
</LI><LI> Xylene Cyanol: Comigrating with 60, Bromphenol BLue: comigrating with 15 (<A href="http://www.protocol-online.org/cgi-bin/prot/view_cache.cgi?ID=845" class="external free" rel="nofollow">http://www.protocol-online.org/cgi-bin/prot/view_cache.cgi?ID=845</A>) - Maxi: bei Hälfte ungefähr
</LI><LI> run for 1:45 hours, Bromphenoleblue at about half of the gel
</LI></UL><DL><DD>--&gt; c=control, ns=no signal, r=random
</DD><DD> --&gt; weird smear everywhere: degraded RNA?
</DD><DD> --&gt; no switch visible on the gel: 133 bp!
</DD><DD> --&gt; signal length: 29-43 bp visible
</DD><DD> --&gt; 25 bp: signal sense, between 50-25 bp: signals: 1a and 1c with approximately the same length, 2a is longer
</DD><DD> --&gt; Wie Sie sehen, sehen Sie nichts.
</DD></DL><UL><LI>PCR of switch: Biomers original used as template
</LI><LI> PCR T7 switch
</LI><LI> 32 x Mastermix: 1600 µl total volume
</LI><LI> 32 µl dNTPs
</LI><LI> 32 µl T7 forward primer
</LI><LI> 32 µl T7 reverser primer
</LI><LI> 160 µl 10x buffer
</LI><LI> 96 µl MgCl2
</LI><LI> 6.4 µl Taq Polymerase
</LI><LI> 5 µl Template (some older PCR)
</LI><LI> 1214.6 µl H2O
</LI></UL><UL><LI><B>changed</B> iGEM PCR program:
</LI><LI> 95°C, 2' 
</LI><LI> 1) 95°C, 0,5'
</LI><LI> 2) 50°C, 0.5'
</LI><LI> 3) 71°C, 1'
</LI><LI> 71°C, 10'
</LI><LI> repeat 1) - 3) 35 times
</LI></UL><H3><SPAN class="mw-headline" id="14.10.2010">14.10.2010</SPAN></H3><UL><LI> yesterday's measurement
</LI><LI> rise visible, comparable to tuesday measurement
</LI><LI> samples frozen, put on 15 % PAGE
</LI></UL><UL><LI> purification of yesterday's PCR
</LI><LI> yield: 
</LI></UL><UL><LI> in vitro T7 transcription, check by polyacrylamide gel electrophoresis 
</LI><LI> 10 ml 5 x paper buffer
</LI><LI> 200 mM Tris/HCl, pH=7.85: 2 ml 1M Tris/HCl, pH=7.85
</LI><LI> 30 mM MgCl2: 600 µl 0.5 M MgCl2
</LI><LI> 500 mM KCl: 5 ml 1 M KCl
</LI><LI> 2.4 ml H2O
</LI></UL><UL><LI> for in vitro transcription
</LI><LI> 0.5 µl T7 RPO: 25 U
</LI><LI> 0.5 µl rNTPs : 20 mM
</LI><LI> 1.36 µl switch: 250 µM
</LI><LI> 1 µl signal: 250 µM
</LI><LI> 2 µl DTT: 10 µM
</LI><LI> 0.2 µl RNase inhibitor
</LI><LI> 4 µl 5x buffer
</LI><LI> 10.44 µl H2O
</LI></UL><UL><LI>5.1 times
</LI><LI> no signal, nonsense, 1a, 1c, 2a
</LI><LI> in low bind tubes
</LI><LI> 2 hour, 37°C
</LI><LI> actually 2 hours and half a hour pocket cleaning time
</LI></UL><UL><LI> 2 % agarose gel to check previous PCR products
</LI></UL><DL><DD>--&gt; yesterdays results bad because no switch :) 
</DD></DL><UL><LI>15 % 6 M urea PAGE
</LI></UL><DL><DD>--&gt; M = low molecular weight marker (NEB)
</DD><DD>--&gt; c=control=all DNAs mixed together in used concentrations: switch and all signals, ns=no signal, r=random=nonsense
</DD><DD>--&gt; r, 1a, 2a on the left: overnight incubation with malchitegreen
</DD><DD>--&gt; r, 1a, 1c, 2a on the right: two hour in vitro transcription without malachitegreen
</DD></DL><DL><DD> --&gt; switch: 133 bp! 
</DD><DD> --&gt; signal length: 29-43 bp
</DD><DD> --&gt; T7 promoter sense: ca. 20 bp (I can't look it up right now) 
</DD></DL><DL><DD>--&gt; extra bands visible after overnight transcription (rise in malchitegreen fluorescence visible)
</DD><DD>--&gt; switch runs at a strange height: WHY? looks normal on 2 % agarose gel
</DD><DD>--&gt; upper band: RPO bound to DNA? Compare EMSA
</DD><DD>--&gt; DNA ladder, RNA bands: How to compare?
</DD></DL><DL><DD> Okay, let's try to interpret this:
</DD><DD> Denaturing conditions, everything precooked: No guarantee for double stranded DNA/RNA
</DD><DD> control: switch and signals from tubes on gel - otherwise treated the same
</DD><DD> --&gt; internal standart: switch at about 80 bp (low molecular weight standart) equals 133 bp
</DD><DD> --&gt; band seen in all samples at the height of random/1c --&gt; termination product of switch??? (expected size: about 90 bp? does not fit at all?!)
</DD><DD> --&gt; lower band visible, what is it?
</DD><DD> --&gt; What is left: NO Differences Between Random control (=nonsense) and Designed Switches...
</DD></DL><UL><LI> New malachitegreen assay (overnight in vitro transcription) with new PCR product
</LI></UL><UL><LI> for 4.1 x 100 µl
</LI><LI> 35.88 µl PCR product
</LI><LI> 20.05 dNTPs
</LI><LI> 41 µl DTT
</LI><LI> 10.25 µl T7 RPO
</LI><LI> 2 µl signal per 100 µl
</LI></UL><UL><LI> preincubation of signal with RPO at 37°C
</LI><LI> no rise during preincubation (makes sense)
</LI><LI> addition of signal after about one and a half hour
</LI></UL><H3><SPAN class="mw-headline" id="15.10.2010">15.10.2010</SPAN></H3><UL><LI> in vitro transcription - malachite green
</LI></UL><UL><LI> slight rise visible after overnight incubation after preincubation of signal
</LI><LI> only 1/4 of the intensity measured yesterday
</LI><LI> spectra fit but very weak signal, scattered spectra
</LI></UL><UL><LI>Next step: addition of DNaseI (RNase free) to overnight transcription products
</LI></UL><UL><LI>Felt ill, went home soon
</LI></UL><H3><SPAN class="mw-headline" id="17.10.2010">17.10.2010</SPAN></H3><UL><LI>Malachite-green measuring assay
</LI></UL><UL><LI> 1x:
</LI><LI> 2 µl signal, 5 µM
</LI><LI> 4.66 µl switch, 176 µM
</LI><LI> 2.5 µl RPO
</LI><LI> 50 µl Paperbuffer
</LI><LI> 10 µl DTT 100 µM
</LI><LI> 5 µl rNTPs
</LI><LI> 25.84 µl H2O
</LI></UL><UL><LI> 4.1 x
</LI><LI> 19.11 µl switch, 176 µM
</LI><LI> 10.25 µl RPO
</LI><LI> 205 µl Paperbuffer
</LI><LI> 41 µl DTT 100 µM
</LI><LI> 20.5 µl rNTPs
</LI><LI> 105.94 µl H2O
</LI></UL><UL><LI> Signals: random, 1a, 2a, 1c
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week30
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Cloning of Parts into pSB1C3</DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="color: #00BB00; border: #00BB00 1px solid; background-color: #55FF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">T7 &amp; E. coli</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="18.10.2010">18.10.2010</SPAN></H3><UL><LI>Yesterday's malachitegreen-binding assay
</LI><LI>Kinetics
</LI></UL><P>--&gt; Favourite one ever so far!
</P><UL><LI> Emission and Excitation: Exc=530 nm, Em=552 nm
</LI></UL><UL><LI>Biomers order arrived
</LI><LI> DNA resolved according to manual, 100 µM endconcentration
</LI></UL><UL><LI> PCR T7 switch/positive signal
</LI><LI> 32 x Mastermix: 1600 µl total volume
</LI><LI> 32 µl dNTPs
</LI><LI> 32 µl T7 forward primer
</LI><LI> 32 µl T7 reverser primer
</LI><LI> 160 µl 10x buffer
</LI><LI> 96 µl MgCl2
</LI><LI> 6.4 µl Taq Polymerase
</LI><LI> 10 µl Template (some older PCR)
</LI><LI> 1209.6 µl H2O
</LI></UL><UL><LI><B>changed</B> iGEM PCR program:
</LI><LI> 95°C, 2' 
</LI><LI> 1) 95°C, 0,5'
</LI><LI> 2) 50°C, 0.5'
</LI><LI> 3) 71°C, 1'
</LI><LI> 71°C, 10'
</LI><LI> repeat 1) - 3) 35 times
</LI></UL><UL><LI> yields positive control:
</LI><LI> Mr=67234,6 g/mol 
</LI><LI>230 ng/µl --&gt; 3.42 µM
</LI><LI> 198 ng/µl --&gt; 2.94 µM
</LI><LI> 209 ng/µl --&gt; used for cloning
</LI><LI> 178 ng/µl --&gt; 2.65 µM
</LI></UL><UL><LI> yields switch:
</LI><LI> Mr=82059 g/mol
</LI><LI> 170 ng/µl --&gt; 2.07 µM
</LI><LI> 204 ng/µl --&gt; 2.49 µM
</LI><LI> 179 ng/µl --&gt; 2.18 µM
</LI><LI> 160 ng/µl --&gt; 1.95 µM
</LI></UL><UL><LI> 6 M urea 15 % acrylamid PAGE
</LI><LI> DNase I digestion
</LI><LI> 1 µl 10 x DNase buffer
</LI><LI> 1 µl DNaseI 
</LI><LI> 20 µl reaction product from malachitegreen assay, 17.10.10
</LI><LI> 37°C, 90 minutes
</LI></UL><UL><LI> Gel:
</LI><LI> 20 µl loading buffer and 20 µl sample
</LI></UL><UL><LI> Malachitegreen assay with double stranded signals
</LI></UL><UL><LI> Concentration verification of sense and antisense in ng/µl
</LI><LI> diluted 1:8 (5+35 µl)
</LI><LI> signal - sense ng/µl - antisense ng/µl
</LI><LI> 1a - 1391 - 1584
</LI><LI> 2b - 1812 - 1823
</LI><LI> 1c - 2208 - 1575
</LI><LI> random2=nonsense2 - 2480 - 1903
</LI></UL><UL><LI> Concentration verification of sense and antisense in µM
</LI><LI> signal - sense µM - antisense µM
</LI><LI> 1a - 15.47 - 17.99
</LI><LI> 2b - 15.43 - 15.70
</LI><LI> 1c - 16.95 - 12.31
</LI><LI> r2 - 21.27 - 16.28
</LI></UL><UL><LI> for 10 µM of both sense and antisense and to put it together to equal 5 µM in total...
</LI><LI> signal - sense µl - antisense µl - water µl
</LI><LI> 1a - 32.32 - 27.79 - 38.89
</LI><LI> 2b - 32.40 - 31.85 - 35.75
</LI><LI> 1c - 29.50 - 40.62 - 29.88
</LI><LI> r2 - 23.51 - 30.71 - 45.78
</LI></UL><UL><LI> Measurement
</LI><LI> 1x switch
</LI></UL><UL><LI> 2 µl signal, 5 µM, double stranded!
</LI><LI> 4.17 µl switch, 2.49 µM
</LI><LI> 2.5 µl RPO
</LI><LI> 50 µl Paperbuffer
</LI><LI> 10 µl DTT 100 µM
</LI><LI> 5 µl rNTPs
</LI><LI> 26.3 µl H2O
</LI></UL><UL><LI> 3.1 x
</LI><LI> 12.93 µl switch, 176 µM
</LI><LI> 7.75 µl RPO
</LI><LI> 155 µl Paperbuffer
</LI><LI> 31 µl DTT 100 µM
</LI><LI> 15.5 µl rNTPs
</LI><LI> 81.53 µl H2O
</LI><LI> signals: r2, 1a, 1c
</LI></UL><LI> 1x positive control
</LI><LI> 2 µl signal, 5 µM, double stranded!
</LI><LI> 2.94 µl switch, 2.94 µM
</LI><LI> 2.5 µl RPO
</LI><LI> 50 µl Paperbuffer
</LI><LI> 10 µl DTT 100 µM
</LI><LI> 5 µl rNTPs
</LI><LI> 27.57 µl H2O
</LI><LI> signal: r2
</LI><UL><LI> Cloning Malachitegreen-binding aptamer into pB1C3
</LI><LI> E/P digestion
</LI><LI> 2 µl 10x NEB 4
</LI><LI> 2 µl 10x BSA
</LI><LI> 0.5 µl EcoRI
</LI><LI> 0.5 µl PstI
</LI><LI> 15 µl linearized pB1C3 (50 ng/µl)/malachitegreen binding aptamer (203 ng/µl)
</LI><LI> 37°C, 1 h
</LI><LI> purification afterwards using DNA clean and concentrated (or something)
</LI></UL><UL><LI> Concentrations
</LI><LI> backbone: 11 ng/µl
</LI><LI> insert: 
</LI></UL><UL><LI> Ligation
</LI><LI> 5 µl digested plasmid
</LI><LI> 2 µl malachitegreen binding aptamer, 1:10 diluted
</LI><LI> 2 µl T4 ligase buffer
</LI><LI> 1 µl T4 ligase
</LI><LI> 10 µl water
</LI></UL><UL><LI> no transformation not possible: no chloramphenicol plates in physic's department...
</LI></UL><H3><SPAN class="mw-headline" id="19.10.2010">19.10.2010</SPAN></H3><UL><LI> Yesterday's malachitegreen assay
</LI><LI> Kinetics:
</LI></UL><UL><LI> Emission spectra: Exc=530 nm
</LI></UL><UL><LI> Excitation spectra: Em=552 nm
</LI></UL><UL><LI> 7 M urea, 15 % PAGE
</LI></UL><UL><LI> DnaseI testdigestion
</LI><LI> plasmid digestion to test DnaseI activity: 
</LI><LI> 10 µl paper buffer
</LI><LI> 0.2 µl DTT
</LI><LI> 2 µl 10x DnaseI buffer
</LI><LI> 1 µl Dnase
</LI><LI> 1 µl plasmid (some random thing from Wuschel)
</LI><LI> 5.8 µl water
</LI><LI> 37°C, 2h
</LI></UL><UL><LI> samples for DNaseI digestion
</LI><LI> 2 µl 10x DnaseI Buffer
</LI><LI> 1 µl DnaseI
</LI><LI> 17 µl reaction product from malachitegreen assay, 18.10.10
</LI><LI> 37°C, 2 h
</LI></UL><UL><LI> 7M urea, 15 % PAGE
</LI><LI> 30 ml
</LI><LI> 11.25 ml acrylamide
</LI><LI> 12.6 g urea
</LI><LI> 3 ml 10x TBE
</LI><LI> 300 µl 10 % APS
</LI><LI> 30 µl TEMED
</LI><LI> H20 to 30 ml
</LI><LI> heat a bit and sonificate
</LI></UL><LI> gel could not be run: pockets not solid...
</LI><LI> most likely reason: too hot when radical starter were added: instant polymerization were they hit the mixture...
</LI><UL><LI> malachitegreen assay with more malachitegreen this time!
</LI><LI>  --&gt; 3x more than usual
</LI><LI> 30 mM 2x malachitegreen paper buffer
</LI><LI> 1 ml 5x paper buffer without malachitegreen
</LI><LI> 0.6 ml 250 mM malachitegreen
</LI><LI> 0.9 ml Water
</LI></UL><UL><LI> 1x switch
</LI><LI> 2 µl signal, 5 µM, double stranded!
</LI><LI> 4.17 µl switch, 2.49 µM
</LI><LI> 2.5 µl RPO
</LI><LI> 50 µl Paperbuffer
</LI><LI> 10 µl DTT 100 µM
</LI><LI> 5 µl rNTPs
</LI><LI> 26.3 µl H2O
</LI><LI> signals: 1a, r2
</LI></UL><LI> 1x positive control
</LI><LI> 2 µl signal, 5 µM, double stranded!
</LI><LI> 2.94 µl switch, 2.94 µM
</LI><LI> 2.5 µl RPO
</LI><LI> 50 µl Paperbuffer
</LI><LI> 10 µl DTT 100 µM
</LI><LI> 5 µl rNTPs
</LI><LI> 27.57 µl H2O
</LI><LI> signal: r2, none
</LI><UL><LI> Cloning Malachitegreen-binding aptamer into pB1C3
</LI></UL><UL><LI> Ligation
</LI><LI> 5 µl digested plasmid
</LI><LI> 2 µl malachitegreen binding aptamer, 1:10 diluted
</LI><LI> 2 µl T4 ligase buffer
</LI><LI> 1 µl T4 ligase
</LI><LI> 10 µl water
</LI></UL><UL><LI> Transformation
</LI><LI> borrowed plates from the Prof. Groll's department
</LI><LI> and from Prof. Becker's
</LI><LI> the one from Prof. Becker's once contained tetracyclin...
</LI><LI> in DH5alpha cells
</LI><LI> 200 µl plated
</LI><LI> overnight, 37°C
</LI></UL><H3><SPAN class="mw-headline" id="20.10.2010">20.10.2010</SPAN></H3><UL><LI> Yesterday's malachite green binding assay
</LI><LI> Kinetics: 
</LI></UL><UL><LI> Emission spectra: Exc=530 nm
</LI></UL><UL><LI> Excitation spectra: Em=552 nm
</LI></UL><UL><LI> Cloning Malachitegreen-binding aptamer into pB1C3
</LI><LI> many colonies grown
</LI><LI> Well done, Flo!
</LI><LI> Colony PCRl
</LI><LI> 10 x Mastermix: 500 µl total volume
</LI><LI> 10 µl dNTPs
</LI><LI> 10 µl G1004
</LI><LI> 10 µl G1005
</LI><LI> 50 µl 10x buffer
</LI><LI> 30 µl MgCl2
</LI><LI> 2 µl Taq Polymerase
</LI><LI> 2 µl Template per reaction
</LI><LI> H2O
</LI></UL><UL><LI> PCR program
</LI><LI> iGEM program!
</LI><LI> 58°C annealing
</LI></UL><UL><LI> 2.5 % agarose gel
</LI></UL><UL><LI> lmw: low molecular weight (ladder)
</LI><LI> 100: 100 kb (ladder)
</LI><LI> 7 clones chosen to be checked by Colony PCR
</LI><LI> LB=negative control=LB-Chloramphenicol
</LI><LI> positive control: PCR with Biomer product
</LI></UL><LI> also on the gel:
</LI><LI> S=switch=PCR of switch used for measurement
</LI><LI> P=positive control=PCR product used for measurement and cloning
</LI><UL><LI> Interpretation
</LI><LI> stupid negative control looks just the same like everything else
</LI><LI> two bands at approximately the right height, but exactly above and below positive control band
</LI><LI> shit.
</LI></UL><UL><LI> Further proceedings
</LI><LI> 5 ml cultures of clones 3, 4, 5, 7
</LI><LI> 5 ml cultures of 6 new colonies
</LI><LI> minipreps and analytical digestions tomorrow
</LI></UL><UL><LI> 7 M urea 15% PAGE
</LI></UL><UL><LI> samples from 19.10.10
</LI></UL><UL><LI> controls: 
</LI><LI> switch: PCR of switch, used in measurements
</LI><LI> P: PCR of positive control, used in measurements
</LI><LI> Plas: random test plasmid used for evaluation of DnaseI activity
</LI></UL><LI> DNase digested samples: 20 µl
</LI><LI> Plas: DnaseI digested random plasmid, for conditions check previous day
</LI><LI> P+r2: Positive control together with double stranded nonsense 2/random 2 signal after overnight transcription in paper buffer (check malachitegreen assay, previous day), DnaseI digested
</LI><LI> S+r2: Switch together with double stranded nonsense 2/random 2 signal after overnight transcription in paper buffer, DnaseI digested
</LI><LI> S+1a: Switch together with double stranded 1a signal after overnight transcription in paper buffer, DnaseI digested
</LI><LI> S+1c: Switch together with double stranded 1c signal after overnight transcription in paper buffer, DnaseI digested
</LI><LI> other: 15 µl
</LI><LI> P+r2: Positive control together with double stranded nonsense 2/random 2 signal after overnight transcription in paper buffer (check malachitegreen assay, previous day)
</LI><LI> S+r2: Switch together with double stranded nonsense 2/random 2 signal after overnight transcription in paper buffer
</LI><LI> S+1a: Switch together with double stranded 1a signal after overnight transcription in paper buffer
</LI><LI> no marker this time! Only 12 lanes :)
</LI><UL><LI> Interpretation:
</LI><LI> no real differences between DnaseI digested and not
</LI><LI> It is rather stupid to use a plasmid to check for DnaseI activity if you want to run the result on a 15 % gel. A plasmid with some kb does not seperate... But: DNA visible, without Dnase I, no DNA visible with DnaseI (stuck in the upper lane, 1)
</LI><LI> Next try with PCR product maybe
</LI><LI> weird lane in all PCR reactions at 2 in both cases and in all other gels and a bit also in agarose gels visible
</LI><LI> positive control runs totally elsewhere than switch even though they do not really vary in length (3)
</LI></UL><UL><LI> Malachitegreen binding assay
</LI><LI> Measurement
</LI><LI> 1x switch
</LI><LI> 2 µl signal, 5 µM, double stranded!
</LI><LI> 4.17 µl switch, 2.49 µM
</LI><LI> 2.5 µl RPO
</LI><LI> 50 µl Paperbuffer
</LI><LI> 10 µl DTT 100 µM
</LI><LI> 5 µl rNTPs
</LI><LI> 26.3 µl H2O
</LI></UL><UL><LI> measured: signals all doublestranded, everything in paper buffer, 10 µM malachitegreen
</LI><LI> positive control
</LI><LI> positive control with nonsense2/random2
</LI><LI> switch with nonsense2/random2
</LI><LI> switch with 1a
</LI></UL><H3><SPAN class="mw-headline" id="21.10.2010">21.10.2010</SPAN></H3><UL><LI> Yesterday's measurement
</LI></UL><UL><LI> Fluorescence spetrum, Exc=530 
</LI></UL><UL><LI> Fluorescence excitation spectrum, Em= 552
</LI></UL><UL><LI> Amplifying in vivo positive control to send for PartsRegistry
</LI><LI> Miniprep of 5 ml overnight control
</LI><LI> P1 185 ng/µl 
</LI><LI> P2 416 ng/µl
</LI><LI> check below for gel picture and further proceeding
</LI></UL><UL><LI>Cloning Malachitegreen-binding aptamer into pSB1C3 
</LI><LI> Mini-prep of 5 ml overnight cultures
</LI><LI> # - concentration in ng/µl
</LI><LI> 3 - 272 
</LI><LI> 4 - 59
</LI><LI> 5 - 100
</LI><LI> 7 - 95.56
</LI><LI> a - 86
</LI><LI> b - 165
</LI><LI> c - 114
</LI><LI> d - 131
</LI><LI> e - 232
</LI><LI> f - 144
</LI><LI> P1 - 185
</LI><LI> P2 - 416
</LI></UL><UL><LI> Control digestion
</LI><LI> 4 µl plasmid
</LI><LI> 2 µl NEB 3
</LI><LI> 0.5 µl PstI
</LI><LI> 0.5 µl EcoRI-HF
</LI><LI> 13 µl H2O
</LI><LI> 1 h, 37°C
</LI></UL><UL><LI> 2 % agarose gel
</LI></UL><UL><LI> lmw: low molecular weight ladder (NEB)
</LI><LI> 100 bp: 100 bp ladder (NEB)
</LI><LI> 3,4,5,7: yesterday's clones, checked and falsified by colony PCR
</LI><LI> a-g: yesterday picked clones
</LI><LI> P1, P2: minpreps of in vivo measurement positive control, to be sent to partsregistry
</LI></UL><UL><LI> Interpreation
</LI><LI> well, something went seriously wrong while cloning
</LI><LI> asked at physic's department: DH5alpha cells in use were pretty old: cell from old -80°C are baaaad!
</LI><LI> showed us new stock
</LI><LI> further possibilities: EcoRI nearly empty --&gt; not cutting properly anymore
</LI><LI> T4 ligation buffer: went through 4 tubes until I found one which was not totally full of precipitates
</LI><LI> Chloramphenicol-plates: got plates from Groll group, plates from 2008
</LI></UL><UL><LI> Further cloning procedure
</LI><LI> digestion of linearized pSB1C3 (from parts registry) and positive control with EcoRI-HF (full enzyme)
</LI><LI> digestion of T7 positive control
</LI><LI> 15 µl linearized pSB1C3 (25 ng/µl, damn it, I thought there are 50 ng/µl while preparing the ligation...)/5 µl of positive control, PCR product (c=173 ng/µl) + 10 µl H2O
</LI><LI> 2 µl 10x NEB 3
</LI><LI> 2 µl 10x BSA
</LI><LI> 0.5 µl EcoRI
</LI><LI> 0.5 µl PstI
</LI><LI> 37°C, 1 h
</LI></UL><UL><LI> heat inactivation
</LI><LI> 80°C, 20 minutes
</LI></UL><UL><LI> ligation
</LI><LI> 1 µl T4 ligase
</LI><LI> 2 µl T4 ligase buffer: tested three buffers, one was okay, made aliquots: what happened to all the buffers???
</LI><LI> 1.33 µl backbone (I thought 50 ng, actually 25 ng)
</LI><LI> 0.23 µl insert, 1:10 diluted
</LI><LI> 17.37 µl H2O
</LI><LI> incubated till transformation
</LI></UL><UL><LI> Extracting pSB1C3 from iGEM 2010 distribution
</LI><LI> 10 µl H20 in A3
</LI><LI> 3 µl used for transformation
</LI></UL><UL><LI> transformation
</LI><LI> in DH5alpha cells from new -80°C fridge...
</LI><LI> also in XL-1 blue: Tetracycline resistance
</LI><LI> 15 minutes in ice with ligation product
</LI><LI> 1 minute heat shock, 42°C
</LI><LI> 2 minutes on ice
</LI><LI> 500 µl LB0 added
</LI><LI> 1.5 hours at 37°C, shaking
</LI><LI> control: digested, linearized pSB1C3 into XL-1 blue
</LI><LI> reason: DpnI digestion recommended, we don't have any DpnI 
</LI><LI> on chloramphenicol agar plates from Prof. Buchner's lab --&gt; new and shiny
</LI><LI> XL-1 blue plated on Tetracycline/Chloramphenicol plates from Becker's lab: not used in Becker's lab because Tetracycline broken, does not matter for us...
</LI></UL><UL><LI> E. coli RPO malachitegreen assay
</LI><LI> E. coli RPO finally arrived
</LI><LI> called Biozym: RPO was supposed to arrive yesterday, delivery cimpany signed for yesterday
</LI><LI> called Materialausgabe two times everyday for the last week --&gt; In the end they already knew, when they heard E14
</LI><LI> nevertheless when the enzyme arrived, somebody forgot to put it into the large book of incoming stuff
</LI><LI> only one person knew, that the enzyme already arrived 
</LI><LI> stored at -20 °C meantime
</LI><LI> dry ice already vaporated...
</LI><LI> people were first mad at me, because I insisted that the package arrived although it was not written in the main book
</LI><LI> then mad at each other because somebody did not put it into the main book
</LI><LI> then I was slightly mad because we called two times a day for seven days, argued about -80°C storage and still...
</LI><LI> bought dry ice, transport into ZNN, stored at -20°C covered in dry ice (-80°C fridge is already standing in the lab but not installed yet) until measurements
</LI><LI> now stored at old -80°C in physic's department in iGEM box
</LI></UL><UL><LI> preperation of measuring constructs -  without terminator
</LI><LI> PCR of 12z (His-Terminator in plasmid)
</LI><LI> 5z (TrpTerm in plasmid)
</LI><LI> 28z (pSB1A2-R0011-TrpSig)
</LI><LI> pSB1K3-R0011-HisSig-B0014
</LI><LI> 4x 8x PCR mix
</LI><LI> 8 µl dNTPs
</LI><LI> 8 µl Apt forward
</LI><LI> 8 µl Apt reverse -  without terminator!
</LI><LI> 40 µl 10x buffer
</LI><LI> 24 µl MgCl2
</LI><LI> 1.6 µl Taq Polymerase
</LI><LI> 302.4 µl Template per reaction
</LI><LI> H2O
</LI></UL><UL><LI> recognized that wrong primers were used for the signals, thrown away after PCR
</LI><LI> protocol iGEM PCR, 52°C annealing temp.
</LI><LI> purified using Clean and Concentrator
</LI><LI> yields: 220 ng/µl TrpTerm, 178 ng/µl (or something like that) HisTerm
</LI></UL><UL><LI> Buffer preparation for E. coli RPO
</LI><LI> Instructions from Epicentre Biotechnologies Datasheet (included in package)
</LI><LI> 5x transcription buffer
</LI><LI> 2 ml Tris, pH=7.49
</LI><LI> 1 ml 500 mM MgCl2
</LI><LI> 5 µl Triton X-100
</LI><LI> 3.75 ml 2 M KCl
</LI><LI> water to 10 ml
</LI><LI> stored in fridge
</LI></UL><UL><LI> 2x transcription buffer with malachitegreen
</LI><LI> 4 ml 5x buffer
</LI><LI> 800 µl 250 mM malachitegreen stock
</LI><LI> water to 10 ml
</LI><LI> wrapped in aluminium foil, stored in fridge
</LI></UL><UL><LI> measurement with E. coli RNA Polymerase
</LI><LI> measured: 12z (TrpTerm - negative control), 5z (HisTerm, negative control), (positive control)
</LI><LI> 1x measurement
</LI><LI> 2.5 µl RPO (2.5 U)
</LI><LI> 10 µl DTT (100 mM stock, 10 mM end)
</LI><LI> 50 µl 2x buffer 
</LI><LI> 2.5 µl rNTPs (80 mM stock, 2 mM end)
</LI><LI> 1 µg DNA template
</LI><LI> water to 100 µl
</LI></UL><UL><LI> 3.1 x
</LI><LI> 7.75 µl RPO
</LI><LI> 31 µl DTT
</LI><LI> 7.75 µl rNTPs
</LI><LI> 155 µl 2x buffer
</LI><LI> 77.5µl H20
</LI><LI> 4.5 µl 5z/5.5 µl H2O
</LI><LI> 5.35 µl 12z/4.65 µl H2O
</LI><LI> 5.88 µl ??? /4.12 µl H20
</LI></UL><UL><LI> 38°C, Exc=630 nm, Em=650/655 nm
</LI></UL><UL><LI> Cloning of R0011 and B1006 in pSB1K3
</LI><LI> digestion of R0011 with EcoRi and SpeI (check above, same procedure)
</LI><LI> ligation
</LI><LI> 0.3 µl R0011
</LI><LI> 0.2 µl B1006 (signal)
</LI><LI> 5 µl pSB1K3
</LI><LI> 2 µl T4 ligation buffer
</LI><LI> 1 µl T4 ligase
</LI><LI> 11.5 µl H2O
</LI><LI> Transformation into DH5alpha cells
</LI><LI> put on Kana-Plates
</LI></UL><UL><LI> Cloning of signal B1006 in pSB1A2_R0011
</LI><LI> 1.82 µl  pSB1A2_R0011
</LI><LI> 0.2 µl Signal B1006
</LI><LI> 2 µl T4 DNA ligation buffer
</LI><LI> 1 µl T4 DNA ligase
</LI><LI> 15 µl H2O
</LI><LI> Transformation into DH5alpha cells
</LI></UL><LI> put on Amp-plates
</LI><LI> check above
</LI><H3><SPAN class="mw-headline" id="22.10.2010">22.10.2010</SPAN></H3><UL><LI> Yesterday's malachitegreen assay using E. coli RPO
</LI></UL><UL><LI> Fluorescence spetrum, Exc=630 
</LI></UL><UL><LI> Fluorescence excitation spectrum, Em= 652
</LI></UL><UL><LI> 7 M 15 % PAGE
</LI><LI> DnaseI digestion of E. coli RPO transcription
</LI><LI> control: T7 positive control
</LI><LI> 17 µl Transcription product
</LI><LI> 2 µl 10 x DnaseI buffer
</LI><LI> 1 µl Dnase
</LI></UL><UL><LI> lmw: low molecular weight ladder, NEB
</LI><LI> H: His-switch
</LI><LI> W: Trp-Switch
</LI><LI> 16z: Positive control (with TrpSignal); using Primer Apt_For+AptPart_woT_Rev
</LI></UL><UL><LI> Ladies and Gentleman, we definetly see RNA this time
</LI><LI> positive control completely transcriped, terminators terminate (finally some terminator terminate...)
</LI><LI> DnaseI digestion works. DnaseI control completely clean.
</LI><LI> I think termination products are visible
</LI><LI> I also think, that RPO/DnaseI/protein in general bound RNA is stuck in pockets
</LI><LI> 1h. 37°C, Dnase digestion conditions: RNA suffers a bit, paradise for RNases?
</LI></UL><UL><LI> Cloning Malachitegreen-binding aptamer into pSB1C3 
</LI><LI> no clones on plates
</LI><LI> clones expected to show up on A3 transformed cells --&gt; part from partsregistry!
</LI><LI> weird clones on Tetracycline/Chloramphenicol plates from Becker's department --&gt; maybe both antibiotics not working anymore
</LI><LI> talked to Moni from E 22 
</LI><LI> still old cells in new -80°C --&gt; cells from 2008, maybe dead, certainly not competent...
</LI></UL><UL><LI> repeat transformation (see yesterday's labbook)
</LI></UL><H3><SPAN class="mw-headline" id="23.10.2010">23.10.2010</SPAN></H3><UL><LI><I>In vitro</I> transcription using <I>E. coli</I> RPO 
</LI><LI> PCR of HisSig, TrpSig, HisTerm, TrpTerm, 16z (positive control)
</LI><LI> 100 µl per template, 800 µl total
</LI><LI> 16 µl dNTPs
</LI><LI> 16 µl forward primer
</LI><LI> 16 µl reverse primer
</LI><LI> 80 µl Taq-buffer Mg-free
</LI><LI> 3.2 µl Taq Polymerase
</LI><LI> 48 µl MgCl2
</LI><LI> 5 µl template per 100 µl
</LI><LI> 615.8 µl H2O
</LI></UL><LI> PCR Purification using DNA Quick and Clean
</LI><LI> yield: Template - # - concentration [ng/µl]
</LI><LI> 16z - 1 - 146
</LI><LI> 16z - 2 - 138
</LI><LI> 16z - 3 - 138
</LI><LI> 16z - 4 - 128
</LI><LI> HisTerm - 1 - 96
</LI><LI> HisTerm - 2 - 160
</LI><LI> TrpTerm - 1 - 204
</LI><LI> TrpTerm - 2 - 190
</LI><LI> HisSig - 1 - 160
</LI><LI> HisSig - 2 - 165
</LI><LI> TrpSig - 1 - 83.5
</LI><LI> TrpSig - 2 - 166
</LI><UL><LI>Measurement
</LI><LI> 16z positive control, TrpTerm, both with and without signal
</LI><LI> 4.1 x Master Mix
</LI><LI> 105 µl Buffer
</LI><LI> 41 µl DTT
</LI><LI> 10.25 µl RPO
</LI><LI> 10.25 µl rNTPs
</LI><LI> 77.9 µl H2O
</LI><LI> take 84 µl Master Mix
</LI><LI> add to: 
</LI><LI> 10.85 µl 16 z + 6.1 µl H20
</LI><LI> 10.85 µl 16 z + 6.1 µl TrpSig
</LI><LI> 10.92 µl TrpTerm + 6.1 µl H20
</LI><LI> 10.92 µl TrpTerm + 6.1 µl TrpSig
</LI></UL><P><A href="/File:TUM2010_101023_kinetik.png" title="File:TUM2010 101023 kinetik.png">600px</A>
Only the positive control without Signal increases!
</P><UL><LI> Cloning of MPA into pSB1C3
</LI><LI> Checked plates, no clones
</LI><LI> being sad
</LI><LI> said stupid plates 
</LI><LI> Repetition of ligation
</LI><LI> for 50 ng plasmid backbone
</LI><LI> 2.6 µl pBS1C3 E/P digested
</LI><LI> 0.23 µl MPA, E/P digested
</LI><LI> 2 µl T4 ligase buffer - I checked four buffers, all were precipitated, took a completely new one and aliquoted it!
</LI><LI> 1 µl T4 Ligase
</LI><LI> for 100 ng plasmid backbone
</LI><LI> 5.2 µl pBS1C3 E/P digested
</LI><LI> 0.46 µl MPA, E/P digested
</LI><LI> 2 µl T4 ligase buffer - I checked four buffers, all were precipitated, took a completely new one and aliquoted it!
</LI><LI> 1 µl T4 Ligase
</LI><LI> transformation into pSB1K3 (just in case...)
</LI><LI> 4.5 µl pSB1K3 E/P digested
</LI><LI> 0.23 µl MPA, E/P digested
</LI><LI> 2 µl T4 ligase buffer - I checked four buffers, all were precipitated, took a completely new one and aliquoted it!
</LI><LI> 1 µl T4 Ligase
</LI></UL><LI> found some pSB1C3_RFP clones in the evening, 5 ml cultures
</LI><LI> stroke over the plate with a pipette tip, 5 ml culture
</LI><H3><SPAN class="mw-headline" id="24.10.2010">24.10.2010</SPAN></H3><UL><LI>Cloning of MPA into pSB1C3
</LI><LI> Miniprep of pSB1C3_RFP and something random from the plate of no clones
</LI><LI> mixed up samples: yields were good, but I guess they don't matter
</LI></UL><LI> very tiny clones found in the morning
</LI><LI> slightly bigger ones by noon
</LI><LI> pickable clones by evening
</LI><LI> picked 10 clones, 5 ml cultures
</LI><UL><LI> BBa_K494001-BBa_K494006
</LI><LI> 2x 5 ml LB_Amp from glycerin stocks
</LI></UL><UL><LI> Measurements with T7 RPO and <I>E. coli</I> constructs
</LI><LI> 16z positive control, TrpTerm, both with and without signal
</LI><LI> 4.1 x Master Mix
</LI><LI> 205 µl Buffer
</LI><LI> 41 µl DTT
</LI><LI> 10.25 µl RPO
</LI><LI> 10.25 µl rNTPs
</LI><LI> 77.9 µl H2O
</LI></UL><UL><LI> take 76.2 µl Master Mix
</LI><LI> add to: 
</LI><LI> 10.07 µl 16 z + 13.7 µl H20
</LI><LI> 10.07 µl 16 z + 9.61 µl TrpSig + 4 µl H2O
</LI><LI> 23.67 µl HisTerm (1)
</LI><LI> 14.2 µl HisTerm (2) + 9.61 µl TrpSig
</LI></UL><P>
It looks like an increase in 3 of 4 traces, but spectr show no sign of malachite green binding:
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Week31
</A></P><DIV style="color: #FF0000; border: #FF0000 1px solid; background-color: #FFAAAA; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">Cloning of Parts into pSB1C3</DIV><DIV style="background: transparent; width: 150px; height: auto; height:1.7em; text-align:center; vertical-align:middle; margin-left: 6px; margin-right: 6px; margin-bottom: 5px; float: left;"> </DIV><DIV style="color: #00BB00; border: #00BB00 1px solid; background-color: #55FF55; width: 150px; text-align:center; vertical-align:middle; margin-left: 5px; margin-right: 5px; margin-bottom: 5px; float: left;">T7 &amp; E. coli</DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><P align="justified">

The Week of Wiki freeze!
</P><H3><SPAN class="mw-headline" id="25.10.2010">25.10.2010</SPAN></H3><UL><LI> Submitting Parts
</LI><LI> Miniprep of yesterday's 5 ml cultures using Zymo Classic
</LI><LI> MPA into pSB1C3
</LI><LI> # - concentration (ng/µl)
</LI><LI> 1 - 45.5
</LI><LI> 2 - 96.5
</LI><LI> 3 - 19.5
</LI><LI> 4 - 31
</LI><LI> 5 - 122
</LI><LI> 6 - 164
</LI><LI> 7 - 30
</LI><LI> 8 - 39.5
</LI><LI> 9 - 20.5
</LI><LI> 10 - 113
</LI></UL><UL><LI> BBa_K494001-BBa_K494006
</LI><LI> BioBrick - # - concentration in ng/µl
</LI><LI> BBa_K494001 - 1 - 179
</LI><LI> BBa_K494001 - 2 - 182
</LI><LI> BBa_K494002 - 1 - 214
</LI><LI> BBa_K494002 - 2 - 288
</LI><LI> BBa_K494003 - 1 - 160
</LI><LI> BBa_K494003 - 2 - 202
</LI><LI> BBa_K494004 - 1 - 308
</LI><LI> BBa_K494004 - 2 - 322
</LI><LI> BBa_K494005 - 1 - 290
</LI><LI> BBa_K494005 - 2 - 126
</LI><LI> BBa_K494006 - 1 - 290
</LI><LI> BBa_K494006 - 2 - 232
</LI></UL><UL><LI> control digestion of everything
</LI><LI> EcoRI/PstI
</LI><LI> MPA clones 1, 3, 4, 7, 8, 9: 8 µl
</LI><LI> all other: 4 µl
</LI><LI> 7 x for 4 µl
</LI><LI> 14 µl 10 x BSA
</LI><LI> 14 µl 10 NEB3
</LI><LI> 2.1 µl EcoRI HF
</LI><LI> 2.1 µl PstI
</LI><LI> water as needed
</LI></UL><UL><LI> 19 x for 4 µl
</LI><LI> 38 µl 10x BSA
</LI><LI> 38 µl 10x NEB3
</LI><LI> 5.7 µl EcoRI HF
</LI><LI> 5.7 µl PstI
</LI><LI> water as needed
</LI></UL><UL><LI> 37°C, 1 hour, fitted just perfectly into the heat block
</LI></UL><UL><LI> recognized later that MPA 2, 5, 6, 10 were digested without DNA
</LI><LI> NotI Digestion with --&gt; PstI empty... Time to finish :)
</LI><LI> 10 µl NEB
</LI><LI> 10 µl 10x BSA
</LI><LI> NotI
</LI></UL><UL><LI> pSB1C3_MPA (BBa_K494000) run on 2 % agarose gel
</LI></UL><UL><LI>--&gt; clone 1 picked for submission: Looked better in reality!
</LI></UL><UL><LI> BBa_K494001-BBa_K494006 run on 1.5 % agarose gel
</LI></UL><UL><LI> pSB1C3_MPA (BBa_K494000) missed preps run on 2 % agarose gel
</LI></UL><H3><SPAN class="mw-headline" id="26.10.2010">26.10.2010</SPAN></H3><UL><LI> Waited for the FedEx man from 8:00-15:00. Called at 15:15. Said, oh, they forgot to pick it up! Sent somebody immediately who arrived at 15:45. Happyness --&gt; Parts submitted, not our fault anymore :) Track: 871353522440
</LI></UL><H3><SPAN class="mw-headline" id="27.10.2010">27.10.2010</SPAN></H3><UL><LI> Ran around to shoot weird <I>E. coli</I> pics
</LI></UL><UL><LI> Not in the lab anymore... FedEx delivered our parts! 9.14 am EDT, we recognized it at 16:22 MEST! More Happyness, now back to serious working maybe...
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H1><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H1><H2><SPAN class="mw-headline" id="Molecular_Biology">Molecular Biology</SPAN></H2><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><UL><LI>PCR 
</LI></UL><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><CENTER><B>Taq Polymerase Hot Start </B></CENTER><P><B>PCR Pippeting plan: </B></P><P>1 µl template </P><P>1 µl dNTP 10 µM </P><P>1 µl G1004 (Primer) 10 µM</P><P>1 µl G1005 (Primer) 10 µM</P><P>5 µl 10x Taq-buffer  (500 mM KCl, 100 mM Tris-HCl (pH 8.3), 15 mM MgCl<SUB>2</SUB>) </P><P>0,2 µl Taq-Polymerase (add last) 5,000 U/ml</P><P>40.8 µl Water</P><P>Final volume 50µl 
</P><P><B>Processing: </B>(program saved as <B>IGEMPCR </B>)</P><UL><LI>preheating of PCR chamber to 94 °C
</LI></UL><P>   --&gt; insert sample 
</P><UL><LI>2 min at 94 °C 
</LI><LI>loop 35x:
</LI></UL><P>     - 30 sat 94°C (according to IGEM protocols) 
</P><P>     - 30 s at 56 °C 
</P><P>     - 45s at 72°C 
</P><UL><LI>7 min at 72°C 
</LI><LI>stay at 4°C </LI></UL><CENTER><B>colony PCR </B></CENTER><UL><LI>Colony PCR
</LI><LI>pick colonies and resuspend them in 20 µl LB+Antibiotic (each)
</LI><LI>PCR of 2 µl of each sample, 2 µl as negative control (Program: ColonyPCR, modified), store remaining 18 µl for overnight cultures
</LI><LI>afterwards, mix 15 µl of each PCR product with 3 µl GLPn and load to Gel
</LI><LI>make overnight cultures of positive clones by adding the remaining 18 µl to 5 ml LB+AB
</LI></UL><P><B>program:colonypcr </B></P><UL><LI>preheating of PCR chamber to 94 °C
</LI></UL><P>   --&gt; insert sample 
</P><UL><LI>5 min 30 sec at 94 °C 
</LI><LI>loop 35x:
</LI><LI>30 sat 94°C (according to IGEM protocols) 
</LI><LI>30 s at 58 °C 
</LI><LI>60s at 72°C 
</LI></UL><LI>7 min at 72°C 
</LI><LI>stay at 4°C 
</LI><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><UL><LI>DNA Purification
</LI></UL><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><CENTER><B>PCR samples </B></CENTER><P><B> ZYMO RESEARCH DNA Clean&amp;Concentration Kit </B></P><P><A href="http://www.zymoresearch.com/zrc/pdf/D4003i.pdf" class="external text" rel="nofollow">Protocol and Information</A></P><OL><LI>In a 1.5 ml microcentrifuge tube, add 2-7 volumes of DNA Binding Buffer to each volume of DNA sample (see table below). Mix briefly by vortexing.</LI></OL><TABLE cellspacing="1" cellpadding="1" border="1" align="center" width="80%"><TBODY><TR><TD> Application
</TD><TD> DNA Binding Buffer : Sample
</TD><TD> Example
</TD></TR><TR><TD> Plasmid, genomic DNA (&gt;2 kb)
</TD><TD> 2 : 1
</TD><TD> 200 μl : 100 μl
</TD></TR><TR><TD> PCR, cDNA, DNA fragment
</TD><TD> 5 : 1
</TD><TD> 500 μl : 100 μl
</TD></TR><TR><TD> ssDNA (e.g., M13 phage)
</TD><TD> 7 : 1
</TD><TD> 700 μl : 100 μl
</TD></TR></TBODY></TABLE><OL><LI>Transfer mixture to a provided Zymo-Spin™ Column1 in a Collection Tube.</LI><LI>Centrifuge at ≥10,000 x g for 30 seconds. Discard the flow-through.</LI><LI>Add 200 μl Wash Buffer to the column. Centrifuge at ≥10,000 x g for 30 seconds. Repeat wash step.</LI><LI>Add ≥6 μl water2,3 directly to the column matrix. Transfer the column to a 1.5 ml microcentrifuge tube and centrifuge at ≥10,000 x g for 30 seconds to elute the DNA.Ultra-pure DNA in water is now ready for use.</LI></OL><P><B> QIAquick purification Kit </B></P><P><A href="http://www1.qiagen.com/literature/render.aspx?id=103715" class="external text" rel="nofollow">Handbook</A></P><P>Procedure 1. Add 5 volumes of Buffer PB to 1 volume of the PCR sample and mix. It is not necessary to remove mineral oil or kerosene. For example, add 500 μl of Buffer PB to 100 μl PCR sample (not including oil). 2. If pH indicator I has beein added to Buffer PB, check that the color of the mixture is yellow. If the color of the mixture is orange or violet, add 10 μl of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn to yellow. 3. Place a QIAquick spin column in a provided 2 ml collection tube. 4. To bind DNA, apply the sample to the QIAquick column and centrifuge for 30–60 s. <B>We changed it to 3 min @ 6000rpm ! </B>5. Discard flow-through. Place the QIAquick column back into the same tube. Collection tubes are re-used to reduce plastic waste. 6. To wash, add 0.75 ml Buffer PE to the QIAquick column and centrifuge for 30–60 s. 7. Discard flow-through and place the QIAquick column back in the same tube. Centrifuge the column for an additional 1 min.<B>repeat!</B> IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation. 8. Place QIAquick column in a clean 1.5 ml microcentrifuge tube. 9. To elute DNA, add 50 μl Buffer EB (10 mM Tris·Cl, pH 8.5) or water (pH 7.0–8.5) to the center of the QIAquick membrane and centrifuge the column for 1 min. Alternatively, for increased DNA concentration, add 30 μl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge. IMPORTANT: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volume is 48 μl from 50 μl elution buffer volume, and 28 μl from 30 μl elution buffer. Elution efficiency is dependent on pH. The maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C as DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE buffer (10 mM Tris·Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions. 10. If the purified DNA is to be analyzed on a gel, add 1 volume of Loading Dye to 5 volumes of purified DNA. Mix the solution by pipetting up and down before loading the gel.</P><CENTER><B> Gel samples</B></CENTER><P><B> ZYMO RESEARCH Gel DNA Recovery Kit </B><A href="http://www.acgtinc.com/PDF_files/Sample%20Prepation_ACGT/Zymoclean%20Gel%20DNA%20Recovery%20Kit_Zymo%20Research.pdf" class="external text" rel="nofollow">Product informartion</A></P><P><B>Protocol</B></P><OL><LI>Excise the DNA fragment1 from the agarose gel using a razor blade or scalpel and transfer it to a 1.5 ml microcentrifuge tube.
</LI><LI>Add 3 volumes of ADB to each volume of agarose excised from the gel (e.g. for 100 μl (mg) of agarose gel slice add 300 μl of ADB).
</LI><LI>Incubate at 37-55 °C for 5-10 minutes until the gel slice is completely dissolved2. For DNA fragments &gt;8 kb, following the incubation step, add one additional volume (equal to that of the gel slice) of water to the mixture for better DNA recovery (e.g. 100 μl agarose, 300 μl ADB and 100 μl water).
</LI><LI>Transfer the melted agarose solution to a Zymo-SpinTM I Column in a Collection Tube.
</LI><LI>Centrifuge at ≥10,000 x g for 30-60 seconds. Discard the flow-through.
</LI><LI>Add 200 μl of Wash Buffer to the column and centrifuge at ≥10,000 x g for 30 seconds. Discard the flow-through. Repeat the wash step.
</LI><LI>Add ≥6 μl of water3,4 directly to the column matrix. Place column into a 1.5 ml tube and centrifuge ≥10,000 x g for 30-60 seconds to elute DNA.Ultra-pure DNA in water is now ready for use.</LI></OL><CENTER><B> Miniprep </B></CENTER><P><B>Protocol:</B></P><OL><LI> Add 600 μl of bacterial culture grown in LB medium to a 1.5 ml microcentrifuge tube.
</LI><LI> Add 100 μl of 7X Lysis Buffer (Blue)1 and mix by inverting the tube 4-6 times. Proceed to step 3 within 2 minutes. After addition of 7X Lysis Buffer the solution should change from opaque to clear blue, indicating complete lysis.
</LI><LI> Add 350 μl of cold Neutralization Buffer (Yellow)2 and mix thoroughly. The sample will turn yellow when the neutralization is complete and a yellowish precipitate will form. Invert the sample an additional 2-3 times to ensure complete neutralization.
</LI><LI> Centrifuge at 11,000 – 16,000 x g for 2-4 minutes.
</LI><LI> Transfer the supernatant (~900 μl) into the provided Zymo-Spin™ IIN column. Avoid disturbing the cell debris pellet.
</LI><LI> Place the column into a Collection Tube and centrifuge for 15 seconds.
</LI><LI> Discard the flow-through and place the column back into the same Collection Tube.
</LI><LI> Add 200 μl of Endo-Wash Buffer to the column. Centrifuge for 15 seconds. It is not necessary to empty the collection tube.
</LI><LI> Add 400 μl of Zyppy™ Wash Buffer2 to the column. Centrifuge for 30 seconds.
</LI><LI> Transfer the column into a clean 1.5 ml microcentrifuge tube then add 30 μl of Zyppy™ Elution Buffer3 directly to the column matrix and let stand for one minute at room temperature.
</LI><LI> Centrifuge for 15 seconds to elute the plasmid DNA.
</LI></OL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><UL><LI>Digestion
</LI></UL><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><CENTER><B> Restriction Digest </B></CENTER><TABLE width="60%" cellspacing="1" cellpadding="1" border="1"><TBODY><TR><TD> Enzyme</TD><TD> 10 units is sufficient, generally 1µl is used
</TD></TR><TR><TD> DNA
</TD><TD> 1 µg
</TD></TR><TR><TD> 10X NEBuffer</TD><TD> 5 µl (1X)
</TD></TR><TR><TD> BSA
</TD><TD> Add to a final concentration of 100 µg/ml (1X) if necessary
</TD></TR><TR><TD> Total Reaction Volume
</TD><TD> 50 µl
</TD></TR><TR><TD> Incubation Time
</TD><TD> 1 - 1.5 hour</TD></TR><TR><TD> Incubation Temperature Enzyme dependent
</TD><TD><P>XbaI, SpeI, PstI, SpeI : 37 °C 
</P></TD></TR></TBODY></TABLE><P><A href="http://www.neb.com/nebecomm/tech_reference/restriction_enzymes/buffer_activity_restriction_enzymes.asp" class="external text" rel="nofollow">activity of restriction enzymes in NEB buffers</A></P><P><B> Biobrick standard </B></P><P><A href="http://openwetware.org/wiki/Restriction_digest" class="external text" rel="nofollow">Protocols for IGEM standard digestion</A></P><CENTER><B>Dephosphorylation</B></CENTER><P>using Antarctic Phosphatase
</P><OL><LI>Add 1/10 volume of 10X Antarctic Phosphatase Reaction Buffer to 1-5 µg of DNA cut with any restriction endonuclease in any buffer.
</LI><LI>Add 1 µl of Antarctic Phosphatase (5 units) and mix.
</LI><LI>Incubate for 15 minutes at 37°C for 5´ extensions or blunt-ends, 60 minutes for 3´ extensions.
</LI><LI>Heat inactivate (or as required to inactivate the restriction enzyme) for 5 minutes at 65°C.
</LI><LI>Proceed with ligation.
</LI></OL><P><A href="http://www.neb.com/nebecomm/products/protocol76.asp" class="external text" rel="nofollow">from NEB</A></P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><UL><LI>Ligation
</LI></UL><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><P align="justified"><B> Using T4 Ligase, New England Labs </B></P><UL><LI>1 µl T4 Ligase <SPAN style="color: rgb(255, 102, 0);">(10.000 U)</SPAN></LI><LI>50 ng plasmid 
</LI><LI>3x mol(plasmid) insert 
</LI><LI>2 µl T4 Ligase 10x buffer 
</LI><LI>add H<SUB>2</SUB>O to reach final volume of 20 µl</LI></UL><UL><LI>incubation at 22°C for 1 h 
</LI><LI>storing at 16 °C for 40 min</LI></UL><P><U><B>Biobrick Standard</B></U></P><P><A href="http://parts2.mit.edu/wiki/index.php/Standard_Assembly" class="external text" rel="nofollow">Standard BioBrick assembly</A></P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><UL><LI>Transformation
</LI></UL><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><P><B>At Woehlke's S1-Lab !!!</B></P><OL><LI>Thaw competent cells on Ice
</LI><LI>Add DNA, pipette gently to mix
</LI><LI>Let sit for 30 minutes on ice
</LI><LI>Incubate cells for 45 seconds at 42°C
</LI><LI>Incubate cells on ice for 2 min
</LI><LI>Add 1 ml LB0
</LI><LI>Incubate for 1 hour at 37oC on shaker
</LI><LI>Spread 100-300 μl onto a plate made with appropriate antibiotic.
</LI><LI>Grow overnight at 37 °C.
</LI><LI>Save the rest of the transformants in liquid culture at 4 °C
</LI></OL><P>modified from <A href="http://openwetware.org/wiki/Transforming_chemically_competent_cells" class="external text" rel="nofollow">open wetware</A></P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><UL><LI>Gel electrophoresis
</LI></UL><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><CENTER><B>Agarose Gels </B></CENTER><P>usual volume needed: 80 ml
<A href="http://www.promega.com/enotes/faqspeak/fq0065.htm" class="external text" rel="nofollow">Optimum resolution according to NEB</A><A href="http://www.cwcboe.org/19992051412432550/lib/19992051412432550/Molecular%20Biology/Gel%20electrophoresis/Lab_manual_8_gel_elect.pdf" class="external text" rel="nofollow">further information on optimizing gel electrophoresis, e.g. recommanded voltage per cm2 gel</A></P><P><B>stain</B></P><UL><LI>SybrGold (<A href="http://www.invitrogen.com/site/us/en/home/References/Molecular-Probes-The-Handbook/Nucleic-Acid-Detection-and-Genomics-Technology/Nucleic-Acid-Detection-and-Quantitation-in-Electrophoretic-Gels-and-Capillaries.html" class="external text" rel="nofollow">invitrogen</A>)
</LI><LI>Cover Gel with 1x TAE
</LI><LI>Add SybrGold to a 1:10000 dilution
</LI><LI>cover with aluminium foil (light sensitive)
</LI><LI>shake&amp;incubate 20 min (for 2% Agarose Gels at least 45 min!)
</LI></UL><UL><LI>SybrSafe
</LI><LI>used just like SybrGold
</LI></UL><P><B> standards</B></P><TABLE width="600" cellspacing="1" cellpadding="1" border="1" align="center" style=""><TBODY><TR><TD> low molecular weight (NEB)</TD><TD> 1 kb standard (NEB)</TD><TD> 2-log standard (NEB)</TD></TR></TBODY></TABLE><CENTER><B>Polyacrylamide Gels</B></CENTER><P><B>Preparation of Gels</B></P><P>Recipe for denaturing gels: 
</P><TABLE cellspacing="1" cellpadding="1" border="1" align="center" width="80%"><TBODY><TR><TD> Gel type
</TD><TD> 1 big gel
</TD><TD> 2 big gels
</TD><TD> 1 small gel
</TD><TD> 2 small gels
</TD></TR><TR><TD> Urea
</TD><TD> 28.8 g
</TD><TD> 57.6 g
</TD><TD> x
</TD><TD> x
</TD></TR><TR><TD> Acrylamide 40%
</TD><TD> 22.5 ml
</TD><TD> 45 ml
</TD><TD> x
</TD><TD> x
</TD></TR><TR><TD> Buffer 10x
</TD><TD> 6 ml
</TD><TD> 12 ml
</TD><TD> x
</TD><TD> x
</TD></TR><TR><TD> End volume (reach by adding water)
</TD><TD> 60 ml
</TD><TD> 120 ml
</TD><TD> x
</TD><TD> x
</TD></TR><TR><TD> APS
</TD><TD> 600 µl
</TD><TD> 1200 µl
</TD><TD> x
</TD><TD> x
</TD></TR><TR><TD> TEMED
</TD><TD> 60 µl
</TD><TD> 120 µl
</TD><TD> x
</TD><TD> x
</TD></TR></TBODY></TABLE><UL><LI>Dissolve Urea in Acrylamide-buffer mixture (use Ultrasound bath), this may take more than an hour!
</LI><LI>Tighten the Gel chamber
</LI><LI>add water to desired end volume 
</LI><LI>Add APS, then TEMED, mix
</LI><LI>Pipette mixture into gel chamber
</LI><LI>Add desired comb
</LI><LI>let gel polymerize overnight; add buffer in the evening
</LI></UL><P><B>Running of Gels</B></P><P>mix samples 1:1 with formamide loading dye (stored @ -20°C)
carefully remove comb
blow air into pockets with a 50 µl syringe
fill samples into pockets
run the gel 
(usually about 200 V)
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H2><SPAN class="mw-headline" id="In_vivo_Measurement"><I>In vivo</I> Measurement</SPAN></H2><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><H3><SPAN class="mw-headline" id="Bacterial_Cell_Growth">Bacterial Cell Growth</SPAN></H3><P>Bacteria from over night cultures were diluted 1:50 into 20 ml culture in LBamp and incubated at 37 °C. Upon OD<SUB>600</SUB> of 0.7-0.8 the cultures were induced with 0.4% Arabinose and 0.4% Arabinose + 1mM IPTG, respectively. Subsequently Cultures were incubated at 25°C  for at least 12 h.
</P><H3><SPAN class="mw-headline" id="Fluorescence_Measurement">Fluorescence Measurement</SPAN></H3><P>Cell samples for the fluorescence measurement were diluted to OD<SUB>600</SUB>=0.03 and analyzed in a JASCO fluorimeter. eGFP excitation wavelength was set to 501 nm and mCherry fluorescence was measured with an excitation at 587 nm. Standard parameters for the fluorimeter included scanning speed of 100 nm/ min and data points every 0.2 nm as well as medium detector sensivity. The cuvette holder was temperated to 25 °C.
The resulting spectra were corrected for instrumental wavelength dependencies and quantum yield of the fluorescent proteins. A pure LBamp spectrum was subtracted and the corrected spectra were normalized using eGFP fluorescence as reference.
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H2><SPAN class="mw-headline" id="In_vitro_Translation_2"><I>In vitro</I> Translation</SPAN></H2><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><P align="justified">

The <A href="http://www.promega.com/catalog/catalogproducts.aspx?categoryname=productleaf_335&amp;ckt=1" class="external text" rel="nofollow">Promega Kit</A> is used according to the provided protocols. Further Information about this Kit can be found in the <A href="http://partsregistry.org/Cell-free_chassis/Commercial_E._coli_S30" class="external text" rel="nofollow">Parts Registry</A>.
</P><P>Fluorescence kinetics are recorded for at least 3 hours, settings are applied as in the in vitro measurement. 
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H2><SPAN class="mw-headline" id="In_vitro_Transcription_2"><I>In vitro</I> Transcription</SPAN></H2><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><H3><SPAN class="mw-headline" id="Buffers">Buffers</SPAN></H3><P>Three different buffers were used for <I>in vitro</I> transcription experiments:
</P><UL><LI> For Epicentre E. coli RNA Polymerase the recommended buffer was used
</LI><LI> For T7 RNA Polymerase experiments two buffers were tested: 
</LI><LI>T7 RPO buffer as recommended and used by members of the Simmel group 
</LI><LI>T7 RPO &quot;paper buffer&quot;, as used in the paper of XXX
</LI></UL><P>Of all buffers 2x concentrated stocks were prepared. Malachite green was usually added to the buffer stocks.
</P><TABLE cellspacing="1" cellpadding="1" border="1" align="center" width="80%"><TBODY><TR><TD> E. coli RPO buffer
</TD><TD> T7 RPO buffer
</TD><TD> T7 RPO buffer &quot;paper&quot;
</TD></TR><TR><TD> Tris
</TD><TD> 40 mM
</TD><TD> 40 mM
</TD><TD> 40 mM
</TD></TR><TR><TD> pH
</TD><TD> 7.5
</TD><TD> 7.1
</TD><TD> 7.9
</TD></TR><TR><TD> MgCl2
</TD><TD> 10 mM
</TD><TD> 40 mM
</TD><TD> 6 mM
</TD></TR><TR><TD> KCl
</TD><TD> 150 mM
</TD><TD> /
</TD><TD> 100 mM
</TD></TR><TR><TD> Triton X-100
</TD><TD> 0.01%
</TD><TD> /
</TD><TD> /
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Sample_Preparation">Sample Preparation</SPAN></H3><P>Different concentrations were tested for malachite green and DNA templates. Components of a standard experiment are listed in the table below. 
</P><TABLE cellspacing="1" cellpadding="1" border="1" align="center" width="80%"><TBODY><TR><TD> E. coli RPO buffer
</TD><TD> T7 RPO buffer
</TD><TD> T7 RPO buffer &quot;paper&quot;
</TD></TR><TR><TD> buffer
</TD><TD> 1x
</TD><TD> 1x
</TD><TD> 1x
</TD></TR><TR><TD> DTT
</TD><TD> 10 mM
</TD><TD> 10 mM
</TD><TD> 10 mM
</TD></TR><TR><TD> Malachite green
</TD><TD> 5-10 µM
</TD><TD> 5-10 µM
</TD><TD> 5-10 µM
</TD></TR><TR><TD> NTP's
</TD><TD> 1 mM
</TD><TD> 4 mM
</TD><TD> 0.8 mM
</TD></TR><TR><TD> RPO
</TD><TD> 2 U
</TD><TD> 125 U
</TD><TD> 125 U
</TD></TR></TBODY></TABLE><P>In each run up to 4 samples are measured simultaneously. Components that are the same in each of the 4 samples (buffer, DTT, NTPs, RPO, Water) are prepared as a 4.1x MasterMix in a loBind tube and split to the 4 cuvettes. Final Volume of each sample is 100 µl.
</P><H3><SPAN class="mw-headline" id="Cary_Eclipse">Cary Eclipse</SPAN></H3><P>For fluorescence measurements, a Cary Eclipse Spectrofluorimeter with a Multicellholder (4 cells) is used. Kinetics are recorded at 37° C. Excitation wavelength is 630 nm, emission is followed at 650 nm and 655 nm. After each kinetics measurement, spectra are to be recorded. 
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>[1] <A href="http://www.promega.com/catalog/catalogproducts.aspx?categoryname=productleaf_335&amp;ckt=1" class="external free" rel="nofollow">http://www.promega.com/catalog/catalogproducts.aspx?categoryname=productleaf_335&amp;ckt=1</A>[2] Zubay, G. (1980) Meth. Enzymol. 65, 856–77, Zubay, G. (1973) Ann. Rev. Genet. 7, 267–87.
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:TU_Munich/Lab">http://2010.igem.org/Team:TU_Munich/Lab</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Munich/Lab" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Munich/Lab" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Munich/Lab&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Munich/Lab&amp;oldid=208858" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>