<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Munich_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Munich/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Munich/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Munich/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Munich/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P style="width: 154px; display: block; text-align: center; font-weight: bold;">Navigation:</P><TABLE border="0" cellpadding="0" cellspacing="0" style="background: none; width:100%"><TBODY><TR><TD align="left"><P style="margin: 0; color: #4E9D20;"><A style="color: black;" href="https://2010.igem.org/Team:TU_Munich">Home</A><SPAN style="color: black; font-size: 1.2em;"> → </SPAN>

Project

</P></TD><TD align="right"><A href="https://2010.igem.org/" target="_blank" style="margin-right: 15px;">iGEM MainPage</A></TD></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Vision"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Vision</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Implementation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Implementation</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#How_to_connect_BioBricks"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">How to connect BioBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Design_and_functional_principle_of_logic_gates"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Design and functional principle of logic gates</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Switch"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Switch</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Transmitter_RNA.C2.B4s"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Transmitter RNA´s</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Putting_it_all_together:_the_switching_process"><SPAN class="tocnumber">2.2.3</SPAN><SPAN class="toctext">Putting it all together: the switching process</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#From_switches_towards_bioLOGICS_logic_gates"><SPAN class="tocnumber">2.2.4</SPAN><SPAN class="toctext">From switches towards bioLOGICS logic gates</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-9"><A href="#Network_construction"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Network construction</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Our_Objective"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Our Objective</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#In_silico_design"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">In silico design</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Evaluation_and_Measurements"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Evaluation and Measurements</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#in_silico_Design_of_Switching_and_Trigger_Unit"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">in silico Design of Switching and Trigger Unit</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Diffusion_and_RNA_Folding_Dynamics"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Diffusion and RNA Folding Dynamics</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#in_vivo_Functionality_Screening"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">in vivo Functionality Screening</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#in_vitro_Screening"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">in vitro Screening</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#in_vitro_translation"><SPAN class="tocnumber">4.4.1</SPAN><SPAN class="toctext">in vitro translation</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#in_vitro_transcription"><SPAN class="tocnumber">4.4.2</SPAN><SPAN class="toctext">in vitro transcription</SPAN></A></LI><LI class="toclevel-4 tocsection-20"><A href="#T7_System"><SPAN class="tocnumber">4.4.2.1</SPAN><SPAN class="toctext">T7 System</SPAN></A></LI><LI class="toclevel-4 tocsection-21"><A href="#E.coli_System"><SPAN class="tocnumber">4.4.2.2</SPAN><SPAN class="toctext">E.coli System</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Software"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">Software</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><TABLE border="0" cellpadding="0" cellspacing="0" width="100%" style="background: transparent;"><TBODY><TR><TD id="navtd" align="center"><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Project" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Project</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Project</P></A><A href="https://2010.igem.org/Team:TU_Munich/Project" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Project</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Project"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Learn more about our project</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">What is our project about? What is the idea behind our project? This page is a good starting point if you are interested in our project and want to learn more about it.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Software" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Software</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Software</P></A><A href="https://2010.igem.org/Team:TU_Munich/Software" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Software</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Software"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Build your own network</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">To demonstrate our vision about Biobrick networks, we developed a software, that enables you to interconnect Biobricks of your choice to your personal network. It will generate a DNA sequence that encodes all your logical connections. Just give it a try!</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Modeling" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Modeling</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Modeling</P></A><A href="https://2010.igem.org/Team:TU_Munich/Modeling" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Modeling</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Modeling"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Understand our RNA devices</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">On this page we show how we analyzed and estimated the switching process of our RNA devices. This is a basic requirement for the creation of better RNA devices.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Lab" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Lab</P></A><A href="https://2010.igem.org/Team:TU_Munich/Lab" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Lab</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Lab"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Learn more about our experiments</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">This page provides an overview as well as detailed information about the experiments and measurements we performed. Protocols as well as our daily lab book will clarify any question about our lab proceedings.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Parts" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">BioBricks</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">BioBricks</P></A><A href="https://2010.igem.org/Team:TU_Munich/Parts" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">BioBricks</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Parts"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Our contribution to the registry</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">Our project required the analysis and modification of existing Biobricks from the part registry as well as the creation of new Biobricks. This page contains more information about these steps.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Team" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Team</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Team</P></A><A href="https://2010.igem.org/Team:TU_Munich/Team" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Team</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Team"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Get to know us</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">Who are we? Where are we from? What do we do in our spare time? Here you can find the answers to all these questions.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/Glossary" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Glossary</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Glossary</P></A><A href="https://2010.igem.org/Team:TU_Munich/Glossary" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Glossary</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/Glossary"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Get the meaning of a term</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">To understand our project description more easily, we put together a list of terms that might be unknown or forgotten. Here you will find explanations concerning these terms.</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/PartnersAndPress" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Partners &amp; Press</P></A><A href="https://2010.igem.org/Team:TU_Munich/PartnersAndPress" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Partners &amp; Press</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/PartnersAndPress"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">Thanks a lot to our supporters</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">This project would not have been possible without the help and support of various people, institutions and companies. We want to say thank you to all of them!</SPAN></DIV></DIV></A></DIV><DIV class="bar" style="border-color: #C4C0BB; background-color:#EDE8E2; color: black; position: relative;"><DIV class="leftbar" style="margin: 0; cursor: hand; float: left; position: relative;"><A href="https://2010.igem.org/Team:TU_Munich/BeyondTheLab" style="color: #FFFFFF;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 2px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 4px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 2px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 4px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -2px; right: 3px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: 0px; right: 3px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 2px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 4px; font-weight: normal; color: #FFFFFF;">Beyond the Lab</P></A><A href="https://2010.igem.org/Team:TU_Munich/BeyondTheLab" style="color: #4E9D20;"><P class="barp" style="margin: 0; padding: 0; cursor: hand; position: absolute; bottom: -1px; right: 3px; font-weight: normal; color: #4E9D20;">Beyond the Lab</P></A></DIV><A href="https://2010.igem.org/Team:TU_Munich/BeyondTheLab"><DIV class="rightbar" style="text-align: left; float: right; cursor: hand; width: 72%; margin-top: 13px; margin-bottom: 2px,"><DIV style="width: 100%; border-bottom: 1px #0065BD solid;"><SPAN style="font-weight: bold; font-size: 0.97em; color: black;">iGEM is more than biology</SPAN></DIV><DIV style="text-align: justify; margin-right: 5px; margin-left: 7px; margin-top: 5px;"><SPAN style="font-size: 0.97em; line-height: 1.2em; color: black;">The iGEM competition not only aims to drive the development of new Biobricks. It also encourages to think about responsibilities, safety and risks in the field of biotechnology. Furthermore, iGEM is about working together and helping other teams in various aspects. Read more how we want to contribute to these goals.</SPAN></DIV></DIV></A></DIV></TD></TR></TBODY></TABLE><DIV class="wiki_div" style="width: 760px; margin: 10px; margin-right: 0px; margin-top: 0px; padding: 30px; padding-bottom: 65px; border: black 1px solid; border-right:0; background: url(https://static.igem.org/mediawiki/2010/b/b3/Team_TU_Munich2010_Images_WikiBackground.jpg) #EDE8E2 no-repeat;"><CENTER><FONT size="5pt" color="#000000"><B>bioLOGICS</B></FONT><FONT size="4pt" color="#000000">: Logical RNA-Devices Enabling BioBrick-Network Formation</FONT></CENTER><H1><SPAN class="mw-headline" id="Vision"> Vision</SPAN></H1><P>Until today, 13.628 biobrick sequences<SUP><A href="/Team:TU_Munich/Project#ref1" title="Team:TU Munich/Project">[1]</A></SUP> have been submitted to partsregistry, thereof 102 reporter units and 12 signaling bricks.
Since then, people are trying to arrange these single biological building blocks in such a manner that allows producing special biotechnological products (metabolic engineering), developing biological sensory circuits (biosensors) and even giving microorganisms the ability to react on multiple environmental factors and serve both as disease indicator and drug. These examples and further promising ideas were implemented on previous iGEM-competitions.<SUP><A href="/Team:TU_Munich/Project#ref2" title="Team:TU Munich/Project">[2]</A></SUP><SUP><A href="/Team:TU_Munich/Project#ref3" title="Team:TU Munich/Project">[3]</A></SUP><SUP><A href="/Team:TU_Munich/Project#ref4" title="Team:TU Munich/Project">[4]</A></SUP>
The idea of combining the outcome of several iGEM competitions to construct complex synthetic biological systems falls at the last hurdle - the fact, that each team uses a different principle how to access and functionally connect the respectively used biobricks. For example, it is a major challenge to create a system that uses several sensoring BioBricks from different iGEM-teams which in turn regulates reportering BioBricks from various teams. In order to combine and fully take advantage of these promising projects, our vision is to develop an adapter that allows interconnecting arbitrary biobricks on a functional level. Such a system easily allows to setup sensor-reporter circuits and interconnect them to complete biological chips... A further step towards artificial cells.</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><P align="justified">

Generally speaking, the above adapter has to meet the following requirements:
</P><UL><LI><B>Universality</B></LI></UL><DL><DD>The adapter has to be compatible to as many BioBricks as possible. This objective will guarantee that a large number of BioBricks can be connected.
</DD></DL><UL><LI><B>Scalability</B></LI></UL><DL><DD>Once the basic design of the system is established, the construction of the system is supposed to be automated in silico. This way it will be possible to create an adapter connecting a large amount of BioBricks.
</DD></DL><UL><LI><B>Biological orthogonality</B></LI></UL><DL><DD>Interference with cellular components has to be as low as possible in order to avoid unwanted and perturbing side effects.
</DD></DL><UL><LI><B>Logic</B></LI></UL><DL><DD>The adapter is supposed to not only associate different BioBricks, but to functionally connect BioBricks in a precisely determined manner (including operations such as AND/OR/NOT).
</DD></DL><P>
Several biological logic units, devices and circuits have been developed so far<SUP><A href="/Team:TU_Munich/Project#ref5" title="Team:TU Munich/Project">[5]</A></SUP>, but to our knowledge, none was shown to meet all requirements listed above.
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H1><SPAN class="mw-headline" id="Implementation">Implementation</SPAN></H1><P>To functionally connect BioBricks, there are several possibilities including genetic switches, riboswitches and direct protein-protein interactions. We investigated several hypothetically principles, and decided to focus our practical work on the development of a RNA-RNA interaction-based switch. These switches are capable of changing between two states, a state of antitermination and termination, and make use of highly-specific RNA-RNA interaction. In principle such a switch can fulfill all requirements mentioned previously. The following text clarifies how these switches work in detail.
</P><H2><SPAN class="mw-headline" id="How_to_connect_BioBricks">How to connect BioBricks</SPAN></H2><P>Our adapter is a system, that activates or disables BioBricks (output BioBricks) in response to the presence of other Biobricks (input Biobricks). Our approach uses a molecular network to put this into practice and consists of four major elements:

</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><TABLE><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:732px;"><DIV class="thumbcaption">The general principle how different inputs can be connect to various outputs. For details see text.Inputs (such as proteins or small molecules) are indicated on the left side. blue lines represent transmitter molecules whereas organe lines present logic gates. The type of logic gate is indicated. Green lines indicate transmitter RNA that can function as mRNA and consequently generate any output gene (indicated on the very right).</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>In order to connect different BioBricks, our network requires four major types of components:
</P><UL><LI>Input elements
</LI><LI>Transmitter molecules
</LI><LI>Logic gates
</LI><LI>Output elements
</LI></UL><DIV style="background-color: white; border: grey 1px solid; width: 45%; float: left; margin: 6px; padding: 3px; font-size:94%; line-height:1.4em;"><B>Computer vs. molecular network - and our approach</B>
Logic gates in a molecular network are often compared to transistors used in a computer, where billions of transistors are incorporated<SUP><A href="/Team:TU_Munich/Project#ref7" title="Team:TU Munich/Project">[7]</A></SUP>. The main advantage on a computer chip is, all transistors share the same functional principle, and only the way connecting them in a special sequence allows specific addressing of only a subset of other transistors by an input. However, spatially fixed connections of molecular logic gates are not possible in a living cell. The &quot;wiring&quot; within a cell relies on the specific interaction between transmitter molecule and their corresponding logic gates, for example implemented by protein-protein/ligand-protein interactions or specific ligand-riboswitch interactions.<SUP><A href="/Team:TU_Munich/Project#ref8" title="Team:TU Munich/Project">[8]</A></SUP><SUP><A href="/Team:TU_Munich/Project#ref9" title="Team:TU Munich/Project">[9]</A></SUP>  As a result, in a cell, each occurring logic gate (&quot;transistor&quot;) has to be different, at least in a special recognition site<SUP><A href="/Team:TU_Munich/Project#ref10" title="Team:TU Munich/Project">[10]</A></SUP> - for example like different transcription factors, recognizing different DNA-sites. Thanks to evolution, nature easily can invent a new transistor for each task - science achieves this only on a limited scale, and producing synthetic molecular logic gates artificially by either rational or evolutionary protein or riboswitch engineering, is limited to small circuits so far<SUP><A href="/Team:TU_Munich/Project#ref11" title="Team:TU Munich/Project">[11]</A></SUP>. Our project aims to establish a molecular switch as close as possible to a electronic transistor, thus sharing the same functional principle for all logic gates.  At the same time, we want to design a easily exchangeable recognition site, which can individually be designed by everyone! </DIV><P>These elements can be combined to build up a molecular network (see illustration). Each input molecule (such as a BioBrick) produces a unique transmitter molecule. All transmitters belong to the same type of molecule and share a common design. However, each transmitter molecule can only interact and activate a certain subset of logic gates. In other words, logic gates have to recognize as well as bind the corresponding transmitter molecules and are capable of producing a new output transmitter molecule. Depending on the type of the logic gate (AND, OR or NOT<SUP><A href="/Team:TU_Munich/Project#ref6" title="Team:TU Munich/Project">[6]</A></SUP>), an output transmitter is only created if both input transmitter molecules are present (AND), at least one of two input transmitters is present (OR) or if no input transmitter is present at all (NOT). Once a logic gate has produced a new output transmitter, these transmitters can in turn address another subset (&quot;layer&quot;) of logic gates. In theory many layers of logic gates can be connected this way allowing the creation of large networks. Until this step, various transmitter molecules might have been produced. But in order to create a Biobrick output, the last layer of logic gates finally generates transmitter molecules that will not active logic gates, but will rather interact with the cell metabolism to produce a BioBrick response. In other words, the last layer of transmitter molecules is capable of regulating BioBrick formation.
</P><P>
Summarizing, the network establishes a connection between input BioBricks and output BioBricks in a functional manner.
Having addressed the basic layout of the molecular network, the next step is to determine what type of molecules can perform the required functions. We decided to use RNA, both as transmitter molecules and for constructing logic gates. Several advantages result from the utilization of RNA as the central element:
</P><UL><LI>During the last years, many Biobricks were designed that are sensitive to various chemicals and substances. These BioBricks often function as a transcription factor that binds to a specific DNA sequence and consequently would be capable to produce a specific transmitter RNA molecule. Thus, in principle each BioBrick which involves transcription can be integrated in our network.
</LI><LI>Since all logic gates are capable of producing transmitter RNA, they can also produce functional mRNA encoding any protein. This means, each BioBrick consisting of protein or RNA can be produced as an output of our network.
</LI><LI>If RNA forms both, the transmitter molecule and the logic gates, they can specifically interact by RNA-RNA interaction, which is highly predictable compared to protein interactions. This allows to generate a library of transmitters and gates <I>in silico</I>. Such a library is essential for the creation of large networks.
</LI><LI>RNA production is fast and energy saving for a cell. Consequently, operating a network that only produces RNA rather than proteins will also be faster and more efficient for the host cell. Since our logic gates are based on transcription, translation and resource consuming protein production will only be required at the very last step. 
</LI><LI>As the half-time of RNA can be rather short, transmitter RNA will not accumulate within the cell and it is therefore less likely for the system to become saturated.
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H2><SPAN class="mw-headline" id="Design_and_functional_principle_of_logic_gates">Design and functional principle of logic gates</SPAN></H2><P>The concept introduced above provides a framework that can potentially serve as an universal adapter between different BioBricks. However, the <A href="/Team:TU_Munich/Glossary#logic_gate" title="Team:TU Munich/Glossary"> logic gates</A> have not been specified more precisely so far. This will be done in the following section.
</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><P align="justified">

Generally speaking, our logic gates are to possess the following characteristics:
</P><UL><LI>Logic gates, such as AND, OR and NOT, have to be implemented by RNA-interaction based principles (see <A href="/Team:TU_Munich/Project#How_to_connect_BioBricks" title="Team:TU Munich/Project"> How to connect BioBricks</A>).
</LI><LI>All logic gates have to recognize their corresponding <A href="/Team:TU_Munich/Glossary#Transmitter_.28bioLOGICS.29" title="Team:TU Munich/Glossary"> transmitter RNAs</A> and, in response, produce an output transmitter molecule.
</LI><LI>Logic gates should follow a basic design rule, in such a way, that their creation can be automated <I>in silico</I>.
</LI><LI>The response efficiency of a logic gate toward a transmitter molecule should be comparable for all logic gates to provide calculable robustness and sensitivity. This will ensure comparable molecular concentrations and functionality of large networks.
</LI><LI>The system has to be designed for <I>in vivo</I> utilization at the first place. As a reference we always assumed a temperature of 37 °C and an <I>E. coli</I> environment.
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV>
 
In order to build logic gates for our bioLOGICS system we will first create a simple switch. A switch can be activated by one transmitter RNA and produce an output transmitter RNA. In contrast to a logic gate, a switch does not perform logic operations. However by combining switches, logic gates can be created. The following text will first describe how the developed switch works and secondly, how logic gates such as AND/OR/NOT can be created using these switches.

<P class="trigger" style="float: left; clear: left;"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20; margin-right: 15px;">
Read more
</A></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>A</B> The basic structure of a bioLOGICS switch (left) and a transmitter molecule (right).<B>B</B>The process of switching. See the text in the close-by &quot;Read more&quot; section for details.Rectangles present the composition of our functional units on the level of DNA. Fringed lines represent RNA produced by RNA polymerase. The stem loop structure depicts the switchable terminator. Terminator and target site are illustrated in blue and turquoise, respectively. Recognition sites are indicated in different colors, in this case red for the input transmitter and green for the output transmitter.Each switch and or later logical unit has to be flanked by a promotor and another constitutive terminator, to allow RNA-production by RNA-polymerase in a proper way.</DIV></DIV></DIV></DIV><DIV class="toggle_container" style="width: 100%; clear: left;"><DIV class="block" style="width: 100%;"><H3><SPAN class="mw-headline" id="Switch">Switch</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption">The basic strcutrue of a switch (left) and a transmitter RNA (right). See text for details.</DIV></DIV></DIV><P>Roughly speaking, a switch can be regarded as an enhanced switchable transcriptional terminator. The enhancement can be described easier by dividing a switch into its functional components: 
</P><UL><LI><B>Target site</B></LI></UL><DL><DD>The target site is the functional core element of our switches, allowing a shift between an &quot;on&quot; and &quot;off&quot; state. Since we work on the level of RNA-production (transcription), a &quot;switchable&quot; transcriptional terminator is suitable for this purpose. By allowing or preventing formation of a transcriptional terminator, that is by switching between termination and antitermination it is possible to represent an &quot;off&quot; and an &quot;on&quot; state, respectively. Therefore, the target site is the 5' ending of the terminator and is required for a stable terminator formation. It should be noted that this principle was also observed in nature.
</DD><DD>To highlight and illustrate the functional principle of our switches, only the part of the terminator which is involved in interacting with a transmitter molecule and which is responsible for shifting between &quot;on&quot; and &quot;off&quot; state is called target site. The remaining terminator sequence is called terminator in the following, even if both, target site and terminator build up the terminator structure occurring in nature. 
</DD><DD>The important aspect of our switches is the fact that all switches will hold the same identical target site. Therefore having found one functional &quot;switchable&quot; terminator, will allow almost unlimited upscaling since this terminator can be used for a large library of switches. This is the main difference to previous works done on this field, which always required developing a new shifting principle for each switch.<SUP><A href="/Team:TU_Munich/Project#ref12" title="Team:TU Munich/Project">[12]</A></SUP><SUP><A href="/Team:TU_Munich/Project#ref13" title="Team:TU Munich/Project">[13]</A></SUP><SUP><A href="/Team:TU_Munich/Project#ref14" title="Team:TU Munich/Project">[14]</A></SUP> Beside this scalability, this principle provides a comparable on/off shifting rate (responds function) for all switches, avoiding complex fine tuning of molecular networks.
</DD><DD>To sum it up, the target site, allows to switch between an &quot;on&quot; and &quot;off&quot; state. But so far, the switch is not capable of performing specific interaction with transmitter molecules. This is where the recognition site comes into play.
</DD></DL><UL><LI><B>Recognition site</B></LI></UL><DL><DD>The recognition site defines which transmitter molecule can actually interact with the switch. Therefore, a unique recognition site is generated for each switch and is positioned right upstream of the target site. In principle the recognition can be any random sequence as long as it remains unique within the molecular network.
</DD></DL><P>Summing up, the recognition site allows a specific interaction between switches and transmitter molecules. Once this interaction is formed, an interaction between the transmitter and the target will actually switch the state of the terminator. This allows the specific arrangement and interconnection of numerous of these switches by transmitter molecules, without changing the target site. Comparable to wires connecting many identical transistors, our target site remains the same.
</P><H3><SPAN class="mw-headline" id="Transmitter_RNA.C2.B4s">Transmitter RNA´s</SPAN></H3><P>As desccribed above, transmitter RNAs are the input and output of bioLOGICS switches (compare <A href="/Team:TU_Munich/Project#How_to_connect_BioBricks" title="Team:TU Munich/Project"> How to connect BioBricks</A>). These transmitters are short ssRNA molecules representing the &quot;trigger&quot; to shift switches between the &quot;on&quot; and &quot;off&quot; state. To fulfill this role, they need to posses the following properties:
</P><UL><LI>A transmitter may only interact with certain switches. That is, a transmitter has to find the corresponding  recognition site of a switch.
</LI><LI>Once an interaction is established between a transmitter and a switch, a transmitter has to be capable of changing the secondary structure of a terminator and thus cause antitermination.
</LI></UL><P>Again, these two properties are fulfilled by two components of the transmitter:
</P><UL><LI><B>Identity site</B></LI></UL><DL><DD>This site is capable of forcing an interaction between the transmitter and the switch. Therefore it is complementary to the recognition site of this switch. As the recognition site is unique within a network, so is the identity site. However, the single identity site is not capable of changing the state of the switch. That is were the trigger site comes into play.
</DD></DL><UL><LI><B>Trigger site</B></LI></UL><DL><DD>Once an interaction is created by the identity site, the trigger site is capable of actually shifting the switch since it is complementary to the target site of the switch. To fulfill this role, it is placed upstream at the 5' end of the identity site. As the target site is the same for all switches, the trigger site is the same for all signals. Therefore it is important, that similar to the identity site, a trigger site cannot function on its own. That is, a single trigger site cannot shift the state of a switch without the help of an identity site.
</DD></DL><P>Summing up, we applied the principle introduced for the switches to the transmitter molecules. In contrast to previous approaches on this field <SUP><A href="/Team:TU_Munich/Project#ref12" title="Team:TU Munich/Project">[12]</A></SUP>, we introduced the described synthetic trigger site in such a manner that it is not able to change the state of the terminator on its own, but only in combination with the identity site. So the challenge is to arrange and optimize these elementary building blocks thermodynamically, that a trigger site is only able to switch in combination with its respective identity site. This was done by <I>in silico</I> design using <A href="/wiki/index.php?title=TU_Munich/Glossary&amp;action=edit&amp;redlink=1" class="new" title="TU Munich/Glossary (page does not exist)"> NUPACK</A>, presented in section <A href="/wiki/index.php?title=TU_Munich/Modeling&amp;action=edit&amp;redlink=1" class="new" title="TU Munich/Modeling (page does not exist)"> in silico design</A>.
</P><H3><SPAN class="mw-headline" id="Putting_it_all_together:_the_switching_process">Putting it all together: the switching process</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption">The basic structure of a switch (left) and a transmitter RNA (right). See text for details.</DIV></DIV></DIV>The functional principle of the designed switches is illustrated in the figure. The switch is positioned on DNA upstream of a desired output transmitter. So in the absence of a triggering transmitter molecule, transcription will be canceled by the formation of a RNA stem loop in the nascent RNA-chain. This will cause the RNA polymerase to stop transcription and fall off the DNA and consequently no output RNA will be produced. This process only relies on <A href="/Team:TU_Munich/Glossary#Termination" title="Team:TU Munich/Glossary"> rho-independent termination</A>.
<P>On the other hand, in the presence of a <A href="/Team:TU_Munich/Project#RNA_transmitters" title="Team:TU Munich/Project"> input transmitter</A>, this small functional RNA inhibits the stem loop formation by complementary base-pairing and hence avoids termination of transcription. In detail, the identity site (red part on transmitter) binds the recognition site (red part on switch) and serves as <A href="/Team:TU_Munich/Glossary#Toehold" title="Team:TU Munich/Glossary">toehold</A>, which will thermodynamically allow the trigger site (turquoise part on transmitter) to perform a <A href="/Team:TU_Munich/Glossary#Strand_Displacement" title="Team:TU Munich/Glossary"> strand displacement</A> and open up the stem loop structure. Consequently the polymerase can read all the way through and form the output RNA.Summing up, we use this concept to create a switch that can be toggled by a transmitter RNA molecule and in response, is able to produce another transmitter RNA.
</P><H3><SPAN class="mw-headline" id="From_switches_towards_bioLOGICS_logic_gates">From switches towards bioLOGICS logic gates</SPAN></H3><P>As described, each switch can be accessed by a specific RNA-transmitter molecule, representing the input. In turn, another RNA-transmitter molecule will be produced if the switch shifts its state. This output transmitter of one switch can serve as input transmitter for the next switch by meaningful selection and design of the respective recognition sites. This easily allows arranging several switches in specific sequences and faulty  wiring - the corner stone of a logical network.
To ease the building of logical networks we want to create a switch capable of Boolean logics, a common mathematical principle fundamental for computational science. Since AND/OR/NOT are basic logic operations which can be implemented with the presented switches, all remaining operations (such as XOR, NAND, ...) can be expressed by these three operators according to laws of boolean logics.
Creating logic gates is achieved by combining two switches in two different ways, as illustrated below.
</P><UL><LI>AND gate
</LI></UL><DL><DD>An AND gate can be constrcuted by positioning two switches right next to each other. For the output transmitter to be created, both input transmitter have to be present.<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Combining two switches in series creates a logic AND gate.</DIV></DIV></DIV></DIV></DD></DL><UL><LI>OR gate
</LI></UL><DL><DD>An OR gate is created by utilizing two independent switches sharing the same output transmitter. If each one of both switches is activated, an output transmitter is generated. Therefore, one input transmitter is enough to produce an output transmitter.<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Combining two switches in parallel creates a logic OR gate.</DIV></DIV></DIV></DIV></DD></DL><UL><LI>NOT gate
</LI></UL><DL><DD>A NOT gate is supposed to function as an inverter. In contrast to the gates described above, a not gate requires only one sitch. However, to meet the design rule for transmitter molecules, this switch shows some differences compared to the switches used for AND and OR gates.</DD><DD>Since the transcriptional terminator may not form if no transmitter is present. Consequently, the switch needs an internal trigger site, capable of preventing terminator formation. To allow the binding of an input transmitter molecule, the switch contains a recognition site upstream of a second target site. The additional target site is mandatory since all transmitter molecules have to carry a trigger. In the case of the NOT switch this trigger site may not bind the actual target site within the transcriptional terminator. In other words, a second target site further upstream is required to catch the trigger site of the transmitter molecule. At the same time, the identity site of the transmitter may not bind right upstream of the terminator. This is accomplished by placing an other identity site right upstream of the terminator rather than an recognition site (compare switches used for AND or OR gate). Due to these two difference, the input transmitter is forced to bind further upstream to the recognition site, displacing the internal trigger site of the switch. This will allow the RNA polymerase to read through an create the output transmitter.
</DD></DL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Structure and switching process of a NOT gate.</DIV></DIV></DIV></DIV><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H2><SPAN class="mw-headline" id="Network_construction">Network construction</SPAN></H2><P>Designing complex biological networks based on either traditional protein engineering or our new bioLOGICS is still a complex task. As described above our bioLOGICS design allows the creation and precise connection of logic gates. To illustrate how a bioLOGICS network is put together we developed a software allowing the fast construction of a custom-made network.
To read more about this, take a look at our <A href="https://2010.igem.org/Team:TU_Munich/Software" class="external text" rel="nofollow">software page</A></P><H1><SPAN class="mw-headline" id="Our_Objective">Our Objective</SPAN></H1><P>Putting the implementation described above into practice, will be a major challenge. For this year's iGEM competition our goal is to do the first step: design and build a switch that can be toggled by a RNA molecule. To be precise, we want to apply the design rules of our switch to modify a transcription terminator in such a way that it interacts with a second RNA molecule and, as a result, is no longer capable of forming a stem loop. This objective will require intensive <I>in silico</I> designing and modeling of switches based on different terminators and their corresponding transmitters. In connection to this theoretical part, we also have to test and verify the switches. For this step, we establish custom-made assays, <I>in vitro</I> and <I>in vivo</I>.

</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><P align="justified">

Once the objective mentioned above is accomplished, these basic RNA/RNA-interactions have to be modified in such a manner that the described identity/trigger site pattern for the transmitter and the complementary recognition/target site switch composition has to be established. The most important requirement is to is to optimize these modules that the transmitter is only able to switches specifically, meaning only in the presence of both, identity AND trigger site. 

Once the objective mentioned above is accomplished, the creation of an OR gate will be rather simple since it only requires two switches. However the creation of an AND or NOT gate and optimizing the logic gates to improve their responds function will remain the goal of future work. Also the creation of small networks and the correct integration of BioBricks as input and output molecules will be future challenges. Furthermore, we wanted to rather focus on the development and the testing of our structural design of the switches, rather than developing a variety of new BioBricks.
</P><H2><SPAN class="mw-headline" id="In_silico_design"><I>In silico</I> design</SPAN></H2><P>As described above, our switches are based on certain design rules. However, there still are different structural parameters that need to be tested and optimized (length of recognition site and target site, choice of terminator, etc.).
We used <A href="/Team:TU_Munich/Project#in_silico_design" title="Team:TU Munich/Project"><I>in silico</I> design</A> and <A href="/Team:TU_Munich/Modeling" title="Team:TU Munich/Modeling"> modeling</A>) to test different parameters. Furthermore we tried to use the <A href="/Team:TU_Munich/Glossary#Antitermination" title="Team:TU Munich/Glossary">antitermination principle</A> observed in nature, such as <A href="/Team:TU_Munich/Glossary#Attenuation" title="Team:TU Munich/Glossary"> attenuation</A> in <I>E. coli</I> or <A href="/Team:TU_Munich/Glossary#Tiny_Abortive_RNA.C2.B4s" title="Team:TU Munich/Glossary"> tiny abortive RNA´s</A> of T7-phage.
</P><H2><SPAN class="mw-headline" id="Evaluation_and_Measurements">Evaluation and Measurements</SPAN></H2><P>To evaluate the functionality of our molecular switches, we first had to establish several assays. Therefore, we improved an existing <A href="/Team:TU_Munich/Lab#In_vivo_Measurements" title="Team:TU Munich/Lab"><I>in vivo</I> assay</A> and developed an <A href="/Team:TU_Munich/Lab#In_vitro_Transcription" title="Team:TU Munich/Lab"><I>in vitro</I> assay</A> for this purpose. For more information please refer to the <A href="/Team:TU_Munich/Lab" title="Team:TU Munich/Lab"> lab</A> section.

Summarizing, the main challenges are 
</P><UL><LI> to find a suitable terminator construct and design a complementary trigger unit, which is only functional in combination with a specificity site - meaning an optimization of the <B>thermodynamically parameters</B> (see<A href="/Team:TU_Munich/Project#in_silico_design" title="Team:TU Munich/Project"> in silico design</A>)
</LI><LI> to investigate whether the transmitter/switch interaction reaction is on a timescale to be competitive to terminator formation - meaning an comparison of <B>kinetic parameters</B> (see <A href="/Team:TU_Munich/Modeling" title="Team:TU Munich/Modeling">Modeling page</A>)
</LI><LI> to proof antitermination can be also be caused by synthetically RNA-interaction (see <A href="/Team:TU_Munich/Glossary#Antitermination" title="Team:TU Munich/Glossary"> Antitermination in nature</A> and <A href="/Team:TU_Munich/Project#Results" title="Team:TU Munich/Project"><I>in vivo</I> and <I>in vitro</I> measurements</A> )
</LI></UL><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><P>Every network starts with a basic unit. While our declared aim is to enable networks allowing fine-tuning of gene expression beyond the regular on/off, exploring such an on/off switch/signal pair is the first step towards a functional network. We constructed several units and tested their efficiency, robustness and reproducibility <I>in vivo</I>, <I>in vitro</I> and <I>in silico</I>. Furthermore we developed a software which allows easy constructions of networks based on our designed logic gates. Conclusive elaboration of a few first RNA-based logic units is the major contribution of our iGEM team.
</P><H2><SPAN class="mw-headline" id="in_silico_Design_of_Switching_and_Trigger_Unit">in silico Design of Switching and Trigger Unit</SPAN></H2><P>As described on the <STRONG class="selflink"> project</STRONG> page, one key aspect of our switches is the idea, that a <A href="/Team:TU_Munich/Glossary#Transmitter_.28bioLOGICS.29" title="Team:TU Munich/Glossary"> RNA transmitter molecule</A> is capable to shift the state of a switch only if its <A href="/Team:TU_Munich/Glossary#Trigger_Site_.28bioLOGICS.29" title="Team:TU Munich/Glossary"> trigger site</A> is present and its <A href="/Team:TU_Munich/Glossary#Identity_Site_.28bioLOGICS.29" title="Team:TU Munich/Glossary"> identity site</A> corresponds to the <A href="/Team:TU_Munich/Glossary#Recognition_Site_.28bioLOGICS.29" title="Team:TU Munich/Glossary"> recognition site</A> of the <A href="/Team:TU_Munich/Glossary#Switch_.28bioLOGICS.29" title="Team:TU Munich/Glossary"> switch</A>. We successfully constructed several switches and their corresponding transmitter RNA <I>in silico</I> on a thermodynamical basis. We modified different transcriptional terminators in such a way, that the formation of the terminator was prevented by a transmitter molecule. As desired, this only occured if the transmitter molecule contained both, a trigger and an identity site. Analogously, we were able to design and verify a NOT gate using the same thermodynical approach.
</P><H2><SPAN class="mw-headline" id="Diffusion_and_RNA_Folding_Dynamics">Diffusion and RNA Folding Dynamics</SPAN></H2><P>We estimated the diffusion time for our constructs and modeled the folding dynamics of our bioLOGICS switches including the switching process with a stochastic RNA folding program. We were able to provide better insight in their folding dynamics and proved that they are able to interrupt termination. We also optimized the switches and the corresponding signals. Furthermore, we combined the switches what resulted in a logic gate. See our <A href="/Team:TU_Munich/Modeling" title="Team:TU Munich/Modeling">Modeling page</A> for further details.
</P><H2><SPAN class="mw-headline" id="in_vivo_Functionality_Screening"><I>in vivo</I> Functionality Screening</SPAN></H2><P>Since our logic gates are intended to function in living cells, <I>in vivo</I> measurements were essential. In a set of experiments we concentrated on two different switches based on known <A href="/Team:TU_Munich/Glossary#Attenuation" title="Team:TU Munich/Glossary">attenuators</A> from nature: the <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">HisTerm</A> and <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">TrpTerm</A>. Focusing on fluorescent proteins for quantifiable input and output we designed a functional and robust screening system. For greater detail see <A href="/Team:TU_Munich/Lab#Experiment_Design" title="Team:TU Munich/Lab">Experimental Design</A>. Unfortunately, setting up a working screening system failed twice. Only in redesigning and improving the screening plasmid pSB1A10 we succeeded, but lost precious time.
</P><P>Ultimately, the two switches displayed remarkable differences in their terminator efficiency, but neither of them responded to their corresponding signal. However, screening one transmitter signal does not disprove the basic working principle of our system. Limited by time, we hope for future teams to take up our work and to use our improved test system that we submitted to the parts registry, for performing successful in vivo measurement.

</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><P align="justified">

Considering the high complexity of <I>in vivo</I> measurements compared to other experimental challenges, a robust and easy to handle test system for <A href="/Team:TU_Munich/Glossary#PoPS-based_devices" title="Team:TU Munich/Glossary"> PoPS-based devices</A> is desirable. As described in <A href="/Team:TU_Munich/Lab#Experiment_Design" title="Team:TU Munich/Lab">Experimental design</A>, we used fluorescent proteins: RFP or mCherry to measure the amount of produced output and eGFP for normalization. Our first attempt, using the screening plasmid pSB1A10, yielded no interpretable results. Switching the fluorescent protein to mCherry did not work either, but after several experimental setups we determined a transcriptional problem causing no reporter protein expression regardless of the inserted part. Thereby we demonstrated the screening plasmid pSB1A10 to be <A href="/wiki/index.php?title=Team:TU_Munich/Biobricks&amp;action=edit&amp;redlink=1" class="new" title="Team:TU Munich/Biobricks (page does not exist)"> malfunctioning</A>. 
Finally a new design based on pSB1A10 lead to a functional and robust screening system (compare <A href="/Team:TU_Munich/Parts#Screening_system:_Backbone_BBa_K494001" title="Team:TU Munich/Parts"> Screening system: Backbone BBa_K494001</A>). A second promoter with identical induction properties inside the BioBrick cloning site enforces transcription of the PoPS-based device and the mCherry output.
</P><P>Exemplary, the graph below on the right shows the positive control, induced and uninduced at OD<SUB>600</SUB>=0.7 followed by 16 h incubation at 25 °C. Clearly visible are eGFP and mCherry fluorescence in the induced samples. The uninduced control showed no fluorescence at all, demonstrating the PBad promoter to be tight and providing very low basal transcription, what is a major advantage for the screening system. This newly designed screening approach renders the characterization of PoPS-based devices in general and switches in particular easy and robust. The low basal transcription furthermore fulfills one of the most important requirements for the designed switches, since output transmitters may only be produced in presence of an input transmitter. This helps to avoid strong &quot;background&quot; noise, which would extremely harden the successful interconnection of several switches. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Bacteria containing positive control</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:357px;"><DIV class="thumbcaption">Emission spectra of induced (green/red) and uninduced(black) positive control BBa_K494002 ; green: eGFP fluorescence ex: 501 nm, red: mCherry fluorescence ex: 587 nm</DIV></DIV></DIV></DIV><P>
Due to the time limitations of the iGEM completion we had to focus our efforts on few switches after designing the screening system. Relying on the functionality of systems occurring in nature, we choose the <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">HisTerm</A> as well as the <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">TrpTerm</A>. Both switches are based on known natural [<A href="https://2010.igem.org/Team:TU_Munich/Glossary#Attenuation|attenuators" class="external autonumber" rel="nofollow">[1]</A>]. Testing synthetic and none-naturally switchable terminators in vivo are goals for future work.
Delorme et al. reported the His-Terminator to be a remarkable effective Terminator with more than 99% termination efficiency.<SUP><A href="/Team:TU_Munich/Project#ref12" title="Team:TU Munich/Project">[12]</A></SUP> The exemplary measurement below on the right confirms the high terminator efficiency. In fact, we could not detect any mCherry fluorescence in any cells containing the <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">HisTerm</A>. Even induction of the corresponding signal transmitter RNA via IPTG did not alter the Terminator efficiency. Again time was the limiting factor and prevented us from testing more than one corresponding transmitter, although the <A href="/Team:TU_Munich/Modeling" title="Team:TU Munich/Modeling"> Modeling</A> highly suggested the necessarily of finding an optimized transmitter length. Thus, the results are insufficient either to prove or to disprove the functionality of the <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">HisTerm</A> or our concept in general.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Bacteria containing HisTerm</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:357px;"><DIV class="thumbcaption">Emission spectra of induced and uninduced screening plasmid BBa_K494002 containing HisTerm ; green: eGFP fluorescence ex: 501 nm, red: mCherry fluorescence ex: 587 nm</DIV></DIV></DIV></DIV><P>Attaining only 90% terminator efficiency, the natural Trp [<A href="https://2010.igem.org/Team:TU_Munich/Glossary#Attenuation|Attenuator" class="external autonumber" rel="nofollow">[2]</A>] is known be less effective than the <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">HisTerm</A>.<SUP><A href="/Team:TU_Munich/Project#ref13" title="Team:TU Munich/Project">[13]</A></SUP> The graph on the right depicts our designed <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">TrpTerm</A> characteristic efficiency of about 40 %, notably below the natural standard. Allowing 60% transcription in the “off” state excludes the <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">TrpTerm</A> from possible candidates for a scalable network of logic gates, due to the mentioned required &quot;yes or no&quot; function (see <A href="/Team:TU_Munich/Project#Implementation" title="Team:TU Munich/Project"> Implementation and how to connect Biobricks</A>). Thus the <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">TrpTerm</A> is inoperative as intended, but may still be useful in other contexts. Similar to the <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">HisTerm</A>, the <A href="/Team:TU_Munich/Modeling#Switch" title="Team:TU Munich/Modeling">TrpTerm</A> also did not react to the induction of the corresponding signal. Under circumstances, termination efficiencies altered by the transmitter are on a low range and not resolvable within observed 40% basal transcription. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Bacteria containing TrpTerm</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:357px;"><DIV class="thumbcaption">Emission spectra of induced and uninduced screening plasmid BBa_K494002 containing TrpTerm ; green: eGFP fluorescence ex: 501 nm, red: mCherry fluorescence ex: 587 nm</DIV></DIV></DIV></DIV><P>Making use of our improved screening system we also carried out some <I>in vivo</I> kinetic measurements in addition to the end-point measurements above. In contrast to the <I>in vitro</I> experiments we did not obtain significant results for the characterization of our switches. As the switching process is many times faster than protein synthesis our <I>in vivo</I> kinetics include the synthesis of mCherry as well as its maturation. Therefore we centered our attention on end-point experiments. For more information browse the <A href="/Team:TU_Munich/Lab#Lab_Book" title="Team:TU Munich/Lab">lab book</A>. 
Considering our <I>in vivo</I> measurements, neither of the tested switches showed any effect regarding the signal induction. But due to the small number of tested switches and signals this can hardly be regarded as disprove of concept. In particular in light of the recent findings by Sooncheol proving antitermination in principle using a T7 system.<SUP><A href="/Team:TU_Munich/Project#ref14" title="Team:TU Munich/Project">[14]</A></SUP></P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H2><SPAN class="mw-headline" id="in_vitro_Screening"><I>in vitro</I> Screening</SPAN></H2><P>To minimize the amount of disturbing factors we decided to countercheck our <I>in vivo</I> results with a set of <I>in vitro</I> measurements. While the <I>in vitro</I> systems are no doubt much less complex than living cells, the work with these set-ups proved to be quite as difficult.
Just as with the <I>in vivo</I> measurements we could prove our switching system neither right nor wrong, leaving enough work for future iGEM teams.
</P><P class="trigger"><A class="toggle_text" href="#" style="font-size: 12px; color: #4e9d20">Read more</A></P><DIV class="toggle_container"><DIV class="block"><H3><SPAN class="mw-headline" id="in_vitro_translation"><I>in vitro translation</I></SPAN></H3><P>Beside optimization of the reporter proteins in use, the major problem occuring in the experiments was the low capacity of the kit. The signal intensity was very low, which made it difficult to observe any signal intensity alterations, so no conclusion could be drawn from these measurements.
</P><H3><SPAN class="mw-headline" id="in_vitro_transcription"><I>in vitro</I> transcription</SPAN></H3><P>We used two completely independent <I>in vitro</I> systems: Using <I>E.coli</I> RNA Polymerase we analyzed the His and Trp switches that had already been tested <I>in vivo</I>. In a second set-up, we used the well-established T7 RNA Polymerase and switch based on the T7 terminator as well as several signal sequences.
</P><H4><SPAN class="mw-headline" id="T7_System">T7 System</SPAN></H4><P>In contradiction to the results of Kang and coworkers and other groups, in our <I>in vitro</I> set-up the T7 terminator did not seem to terminate at all. The negative control (Promoter_Terminator_malachite binding aptamer) showed a similar increase in fluorescence as the positive control (Promoter_random sequence_malachite binding aptamer). 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><I>in vitro</I> transcription measurement of T7 terminator with no signal(upper left), nonsense signal (upper right) and two different designed signals (below)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><I>in vitro</I> transcription measurement of positive control(upper left and T7 terminator with three different designed signals (remaining traces)</DIV></DIV></DIV><P>Furthermore denaturing Polyacrylamide Gel Electrophoresis (PAGE) confirmed that there was no observeable termination of transcription. The addition of a signaling sequence led to a significantly lower increase in fluorescence, which can be attributed to the fact that both DNA sequences, switch and signal, compete for RNA Polymerases.
However, there is almost no difference between the designed signals and random sequences, which is not a big surprise since there can be no antitermination if the terminator itself does not work.</P><P>Possible explanations for the contradiction between our results and those of Kang and coworkers might be the experimental set-up and the RNA Polymerases we used. Different variants of T7 RNA Polymerase might respond in different ways to terminator structures, and the termination might be influenced by the presence or absence of cofactors, depending on the purification methods used in producing the Polymerase.</P><P>This set-up offers a lot of possible experiments for the future, which we would have loved to conduct with a just a bit more time...
</P><H4><SPAN class="mw-headline" id="E.coli_System"><I>E.coli</I> System</SPAN></H4><P>Compared to the T7 System, the <I>E. coli</I> RPO system produced poor increases in fluorescence, indicating little RNA synthesis. It was shown that the presence of a terminator decreases, as expected, the production of downstream RNA.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><I>in vitro</I> transcription measurement of Switch TrpTerm (upper traces) and positive control (lower traces). Left side: with Trp-signal, right side: no signal</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><I>in vitro</I> transcription measurement of positive control (left), Switch TrpTerm (center) and switch HisTerm(right) </DIV></DIV></DIV><P>This result was also confirmed by denaturing PAGE. However, due to the poor changes in fluorescence we were not able to actually characterize the behaviour of our switches <I>in vitro</I>, and the small RNA concentrations did not allow a quantitative interpretation of our gels.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">denaturing polyacrylamide gel electrophoresis of DNaseI digested samples from <I>in vitro</I> transcription of positive control (16z), Switch TrpTerm (W) and switch HisTerm(H). c marks the lanes in which the DNA was injected, the last three lanes show the undigested samples</DIV></DIV></DIV></DIV><P>A major problem with this method was the low concentration of the ordered Polymerase resulting in a much weaker overall signal as comparable measurements using the T7 Polymerase. 
In future experiments we might try to work with smaller volumes in order to reach higher concentration of RPO and of the synthesized RNA molecules, so measuring in 96 well plate readers might be a good choice.  
</P><A class="toggle_close" href="#" style="font-size: 12px; color: #4e9d20" align="left">Close</A></DIV></DIV><H2><SPAN class="mw-headline" id="Software">Software</SPAN></H2><P>Although we could not show the full functionality of bioLOGICS in the lab we still want to demonstrate the potential of our approach. Hence we implemented the idea behind our logic gates in a program which illustrates how bioLOGCIS theoretically would allow the construction of complex information processing networks interconnecting BioBricks. For further details take a look at our <A href="/Team:TU_Munich/Software" title="Team:TU Munich/Software">Software page</A>.
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>[1] <A href="http://partsregistry.org/cgi/partsdb/Statistics.cgi" class="external free" rel="nofollow">http://partsregistry.org/cgi/partsdb/Statistics.cgi</A>[2] <A href="https://2009.igem.org/Team:Imperial_College_London/M1" class="external free" rel="nofollow">https://2009.igem.org/Team:Imperial_College_London/M1</A> encapsulation
[3] <A href="https://2009.igem.org/Team:TUDelft" class="external free" rel="nofollow">https://2009.igem.org/Team:TUDelft</A>[4] <A href="https://2008.igem.org/Team:Heidelberg" class="external free" rel="nofollow">https://2008.igem.org/Team:Heidelberg</A>[5] Maung Nyan Win and Christina D. Smolke, Science Oct. 2008 Vol. 322. no. 5900, pp. 456 - 460
[6] Lu, T.K., A.S. Khalil, and J.J. Collins, Next-generation synthetic gene networks. Nature biotechnology, 2009. 27(12): p. 1139-1150. 
[7] Schaller, R.R., Moore's law: past, present and future. Spectrum, IEEE, 2002. 34(6): p. 52-59.
[8] von Mering, C., et al., Comparative assessment of large-scale data sets of protein–protein interactions. Nature, 2002. 417(6887): p. 399-403.
[9] Mandal, M. and R.R. Breaker, Gene regulation by riboswitches. Nature Reviews Molecular Cell Biology, 2004. 5(6): p. 451-463. 
[10] Benner, S.A. and A.M. Sismour, Synthetic biology. Nature Reviews Genetics, 2005. 6(7): p. 533-543.
[11] Beaudry, A. and G. Joyce, Directed evolution of an RNA enzyme. Science, 1992. 257(5070): p. 635-641.
[12] Delorme, Ehrlich and Renault, Regulation of Expression of the Lactococcus lactis Histidine Operon. Journal of Bacteriology, Apr. 1999, p. 2026–2037
[13] Trun and Trempy(2003): Fundamental Bacterial Genetics, Wiley-Blackwell, Chapter 12 
[14]Sooncheol Lee, Huong Minh Nguyen and Changwon Kang, Tiny abortive initiation transcripts exert antitermination activity on an RNA hairpin-dependent intrinsic terminator. Nucleic Acids Research, 2010, 1–9
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:TU_Munich/Project">http://2010.igem.org/Team:TU_Munich/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Munich/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Munich/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Munich/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Munich/Project&amp;oldid=208106" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>