<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong-CUHK_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong-CUHK/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><BUTTON onclick="topFunction()" id="myBtn" title="Go to top">Top</BUTTON></P><DIV style="margin-top:-11.6px; height:220px; boder-style:none; background-image:url('https://static.igem.org/mediawiki/2018/5/5d/T--Hong_Kong-CUHK--HomePage.jpg'); background-size: cover; position: relative"><DIV style="background-color: rgba(128, 212, 255, 0.2); position: absolute; top: 0; left: 0; width: 100%; height: 100%;"><P style="font-size:15pt; color:floralwhite; display:inline; font-family: 'Orbitron';margin-left:10px;margin-top:12px;">
     RNA Aptamer Probe Influenza Detector
  </P><P style="font-size:16px; color:floralwhite; right:10px; bottom:0; position:absolute; font-family: 'Homemade Apple';">Proudly Presented by Team CUHK </P></DIV></DIV><DIV><NAV class="navbar navbar" data-spy="affix" data-offset-top="640" style="margin-top:0; border-style:none; height:60px; background-color:white;"><UL class="nav navbar-nav navbar-left" style="height:60px; font-size:20px; font-family: 'Lemonada';"><LI class="bar_links"><A href="https://2018.igem.org/Team:Hong_Kong-CUHK" style="color: #D98880;">HOME</A></LI><LI class="bar_links"><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/project" style="color: #C39BD3;">PROJECT</A></LI><LI class="bar_links"><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/wetlab" style="color: #85C1E9;">WET LAB</A></LI><LI class="bar_links"><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/drylab" style="color: #76D7C4;">DRY LAB</A></LI><LI class="bar_links"><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/humanpractice" style="color: #52BE80;">HUMAN PRACTICES</A></LI><LI class="bar_links"><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/Teammenu" style="color: #F7DC6F;">TEAM</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" style="color: #D35400;">AWARDS
        </A></LI><LI><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/Applied_Design">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/Model_Awards">Model</A></LI><LI><A href="https://2018.igem.org/Team:Hong_Kong-CUHK/Software">Software</A></LI></UL></NAV></DIV><DIV class="igem_2018_team_content" style="height:100%; width:100%; background-color:white; bottom:0;"><DIV class="footer_me"><DIV style="float:left;text-align:left;margin-top:25px;margin-left:0;"><P>Email:</P><P>igem18.cu@gmail.com</P></DIV><DIV style="float:left;text-align:left;margin-top:25px;margin-left:3%;"><P>Address:</P><P>CUHK, Sha Tin, NT, Hong Kong</P></DIV><DIV style="float:left;text-align:left;margin-top:25px;margin-left:3%;"><P>CUHK Igem 2018</P></DIV></DIV><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><DIV><H1>Notebook</H1></DIV><DIV class="timeline"><DIV class="container left"><DIV class="content"><H3>Sep 2017 - May 2018</H3><P>Topic Brainstorming: <I>Produce acrylamide hydrolase in E. coli, Use docked His-tags to remove heavy metal in sewage, Alter Streptococcus gordonii for oral probiotics, Using Arthrospira platensis to produce oral vaccine, Improvement of toehold switches, Produce a switchable Spinach aptamer.</I></P><P>Our initial project focused on <I>Alter Streptococcus gordonii for oral probiotics</I>. After several meetings with Professors, we decided to focus on <I>Improvement of toehold switches</I> and <I>Produce a switchable Spinach aptamer.</I></P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 1: 29 Apr - 5 May</H3><P>Frederick designed primers to make several improvements on the original toehold switch design.</P><P>Rebecca designed primers for expression of Spinach, tBroccoli, tdBroccoli and Split-Broccoli in E. coli.</P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 2: 6 May - 12 May</H3><P>Primers have arrived.</P><P>Frederick started reconstructing the toehold switch expression plasmids, however contamination on plates are abundant. He started to eliminate the cause by sanitization of the incubator, shaker, and fridge.</P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 3: 13 May - 19 May</H3><P>Frederick, Peter and Elsa are collaborating for cloning of the toehold switch expression plasmids. Frederick noticed that the cloning steps can be reduced a lot by using very long primers for overlap PCR. However, the cloning plan is unchanged.</P><P>Contamination is still noticeably abundant.</P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 4: 20 May - 26 May</H3><P>Rebecca started cloning the aptamers. She also encountered the contamination issue. She streaked plates for the competent cells and found out that 2 out of 4 batches have yeast-like contamination. Peter and Frederick made some new competent cells.</P><P>Using unconfirmed clones for fluorescence assay led to failure. The cause of this is later discovered as possibly the low pH of the M9 medium stock.</P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 5: 27 May - 2 June</H3><P>Cloning of the toehold switch plasmids kept facing failure. Sequencing results showed some fragment could not be detected. Gel electrophoresis showed that the pACYCDuet-1-based trigger plasmid has probably undergone homologous recombination. The RFP-expressing switch plasmid is showing very dim light. Primers are designed for changing the promoter from J23100 to the sfGFP promoter.</P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 6: 3 June - 9 June</H3><P>Rebecca was trying to use a “Prevent correct folding” method for the Spinach aptamer. </P><P>Frederick discovered a paper using “Destabilize folding” method, which he designed primers for expressing the probes in E. coli. The target RNA is the AmpR gene, as it already existed in pSB1A3.</P><P>No clones for the changing promoter.</P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 7: 10 June - 16 June</H3><P>Cloning of the AmpR probe to pCOLA-J23100 was successful. Clones for changing promoter had dim light at first but bright light after several days in the fridge.</P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 8: 17 June - 23 June</H3><P>pCOLA-J23100-AmpR probe and pSB1A3 are co-transformed into DH5alpha for the assay. However, there is no visible difference between the untransformed and the co-transformed cells. Frederick concluded that the expression level of AmpR is too low. The cause of this is later discovered as probably the low pH of the M9 medium stock.</P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 9: 24 June - 30 June</H3><P>InterLab duty was assigned to Daniel and Peter.</P><P>Frederick transformed the biobricks for cell assays.</P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 10:  1 July - 7 July</H3><P>Professors were urging us to have actual progress. Rebecca, Elsa, and Frederick decided to try the “Destabilizing folding” approach and designed probes for hemagglutinin and neuraminidase genes from influenza A. We decided to design them as siRNA and use the data to design better probes. Primers for E. coli expression and in vitro transcription were ordered.</P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 11: 8 July - 14 July</H3><P>In vitro transcription kits were bought. Frederick decided to try E. coli expression first. Cloning of most probes were successful, although some had single base mutations. </P><P>InterLab study was finished within the week.</P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 12: 15 July - 22 July</H3><P>The plasmids for probes and targets were transformed into E. coli C41(DE3) for screening. However, both the probe+target clones and the positive control clones did not have fluorescence. The reason was concluded to be that the ribosome binding sites in the Duet vectors were not removed, causing unfolding of the aptamers. Primers are designed for adding the T7 promoter right before the aptamer or target, which can also be used for templates for in vitro transcription.
</P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 13: 23 July - 30 July</H3><P>In vitro transcription kit arrived. In vitro transcription using linearized plasmids as templates resulted in very low yield. </P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 14: 31 July - 5 Aug</H3><P>Primers for direct T7 promoter has arrived. Cloning of the probes had once again begun. </P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 15: 6 Aug - 12 Aug</H3><P>Elsa and Frederick formed the assay cloning team, for cloning of the probes and targets into pRSFDuet-1 based plasmids for assay; Jenny, Daniel and Peter has formed the standard biobrick cloning team for cloning of the basic biobrick probes and composite biobrick T7 Promoter-Probe-T7 terminators.</P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 16: 13 Aug - 19 Aug</H3><P>Expression test of the positive control aptamer confirmed that BL21, 37C incubation results in the maximum fluorescence.</P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 17: 20 Aug - 26 Aug</H3><P>Probes with the new design were cloned and co-transformed with the target with the old design. After induction, it seems that all probes did not have significantly increased fluorescence with target expression. However, one of the N9 probes and one of the N2 probes seemed to have a little increase. Frederick decided to perform in vitro transcription for those probes using PCR product.</P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 18: 27 Aug - 2 Sep</H3><P>Elsa performed the RNA fluorescent assay of probes for H3, H7, N2, N9, and after data analysis, we found that N2 and N9 have statistically difference between probe-target complex and probe only.</P><P>We received the BL21 Star (DE3) strain from the NUS iGEM 2018 team.</P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 19: 3 - 9 Sep</H3><P>To see whether our aptamer system can perform well after nasal fluid is added, Elsa performed RNA fluorescent assay of N2 and N9 probes in different salt concentration (i.e. potassium ion, magnesium ion, calcium ion and sodium ion) with reference to the salt composition of nasal fluid. </P><P>To identify the minimum amount of target needed to have a significant positive result. Elsa also performed the limit of detection assay of N2 and N9 probe, by setting reaction of 2uM of probe with different concentration of target (i.e. 1.5uM, 1uM, 0.5uM, 0.2uM, 0.05uM) and addition of same amount of aptamer folding buffer and fluorogen DFHBI. We found that at least 0.2uM and 0.05uM were required for N9 and N2 probe-target complex to have significant on-off ratio.</P><P>To see whether the aptamers can give sufficient fluorescence in household setting, Elsa performed RNA fluorescent assay of N2 and N9 probes with condition of (1) 90 degree Celsius 5 minutes folding time, followed by room temperature 45 minutes incubation time; and (2) room temperature all the way the assay. When compared the result to the protocol (90 degree Celsius 5 minutes folding time followed by 37 degree Celsius 45 minutes incubation time), condition (1) gave a significant on-off ratio, though the fluorescence given were lower than the protocol.</P><P>New aptamers were designed and primers needed were ordered.</P><P>To see if the E. coli expression results were due to its low expression level, Frederick performed total RNA extraction with the RNA-Snap protocol and tested as the in vitro transcription probes. It seemed that the positive control has a 7-fold fluorescence recovery after RNA extraction, showing the E. coli did not have the optimal environment for the optimal folding of the aptamers. Probes did not have better performance after RNA extraction, we hypothesized that this might be due to misfolding due to the T7 terminator addition.</P><P>Primers for Improvement of a previous biobrick was ordered.</P><P>Daniel performed transformation of :RAPID-N9, RAPID-N2,T7 promoter-RAPID-N9-T7 terminator, T7 promoter-RAPID-N2-T7 terminator and iSpinach.</P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 20: 10 - 16 Sep</H3><P>Primers for previous biobricks improvement and primer for new set of aptamers were arrived. Frederick performed in vitro transcription for those probes using PCR product.</P><P>Daniel, Peter and Jenny performed Miniprep for the successful clones: T7 promoter-RAPID-N9-T7 terminator (1 success), T7 promoter-RAPID-N2-T7 terminator (3 successes), RAPID-N9(2 successes) ,checked clone by restriction enzyme digestion.</P><P>They also performed ligation and transformation of RAPID-N2, miniSpinach(P1-a5-b3), T7 promoter-miniSpinach-T7 terminator, iSpinach and colony PCR of RAPID-N2(2 successes), miniSpinach(P1-a5-b3)(1 success), T7 promoter-miniSpinach-T7 terminator(1 success), iSpinach(3 successes).They did minipreparation for the successful clones and sent to sequencing.</P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 21: 17 - 23 Sep</H3><P>Elsa performed RNA fluorescent assay of probes for H1, H3, H7, N1, N2, N9 and polymerase basic 2. For probes for H3, H7, N2, N9 and polymerase basic 2, 1 among 4 probes tested for each type of surface protein had a ‘good’ on-off ratio.</P></DIV></DIV><DIV class="container left"><DIV class="content"><H3>Week 23: 1 - 7 Oct</H3><P>Frederick and Elsa performed real-time pcr of N2 and N9, using DFHBI as fluorogen, to check for the time and optimal temperature for signal development. We found that signal develop started when the temperature reached 75 degrees Celsius on top of 95 degree Celsius 5 minutes folding time,  while 20 degree Celsius was the optimal temperature for aptamer to give fluorescence.</P></DIV></DIV><DIV class="container right"><DIV class="content"><H3>Week 24: 8 - 14 Oct</H3><P>Frederick and Elsa performed RNA fluorescent assay by mixing probes with targets from different probes to test the specificity of our RNA aptamer Probe. It was found that our probe is orthogonal and specific.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>