Team:Hong Kong-CUHK/Description
Top
RNA Aptamer Probe Influenza Detector
Proudly Presented by Team CUHK
HOME
PROJECT
WET LAB
DRY LAB
HUMAN PRACTICES
TEAM
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Software
Email:
igem18.cu@gmail.com
Address:
CUHK, Sha Tin, NT, Hong Kong
CUHK Igem 2018
Description:
Influenza Detection & its Methods
Significance of Influenza Detection
Transmissible diseases are one of the biggest public health issue faced by our society throughout the history. In Hong Kong, influenza A has the potential to be spread quickly and widely in poultry flocks and sometimes transmits to human, which often causes severe pandemic, causing the illness and deaths of many. The most notorious pandemic was the “Spanish Flu” in 1918, which killed 50 million people worldwide (1). Influenza A poses enormous societal and economic burden. It is estimated that around $90 billion US dollars and 600,000 lives are lost to influenza A every year in the States (2). Influenza A can be classified according to the types of hemagglutinin (HA) and neuraminidase (NA) on the virus surface. With 16 types of HA and 9 types of NA, 144 subtypes of Influenza A are possible. influenza A properties can vary with different subtypes. The difference in mortality rates of H5N1 and H1N1 infection, for example, shows that subtyping can be critical in monitoring a potential epidemic.
Conventional and novel methods of influenza diagnosis
To prevent the spread of pandemic, appropriate treatment is best taken within 24 hours after the onset of symptoms (3). Therefore, a rapid and accurate on-site detection method is required. However, nowadays on-site diagnostic method, such as Rapid Influenza Diagnostic Tests (RIDTs), can only identify the influenza A virus but cannot subtype it (4). Typical subtyping relies greatly on RT-PCR, which requires expertise and complicated laboratory set-up; newer on-site subtyping technology utilizing gold nanoparticles or antibodies lack design flexibility and remain too expensive for mass production. Toehold switch, an artificial RNA biosensor first developed in 2014 by Green et al, is a forward engineered mRNA secondary structure that allows the translation of downstream protein coding sequence when a specific trigger RNA binds to it (5). The trigger RNA binds to the switch region of the toehold linearizes toehold secondary structure. This then releases the Ribosomal Binding Site (RBS) from the loop, allowing a ribosome to bind on. The ribosome can then read along the coding region of the toehold switch, hence giving off a protein chosen by the user.
Fallbacks of Our Previous Project – Toehold Switches
With the recent advance of cell-free translation system expressing RNA detecting probe, a solution to the problems aforementioned is presented, but development of this technique requires further improvements to provide signal discrimination and accuracy. For example, when the iGEM 2017 CUHK team utilized red fluorescent protein (RFP) as a reporter for detecting influenza A subtypes, the development of signals took an overnight culture; when Collins et al. utilized lacZ hydrolase as a reporter, a 2-hour long RNA amplification process is required to shorten the total reporting time to 4 hour (6,7). These data present signal visualization as a main hindrance in the development of this technology. However, as the action time of protein reporters are restricted by translation and protein folding, our team aims to search for a optimal reporter by RNA instead of protein, as for a better and expertise-free detection methods, to maximise the on-site potential of this technology.
RNA Reporters
To develop a rapid, cheap, and on-site system of influenza diagnosis, we have chosen nucleic acid-based reporters, instead of protein-based (e.g. toehold switches). From the array of nucleic acid reporters, such as molecular beacons, DNAzymes, we have chosen the light-up RNA aptamers.
Aptamers refer to oligonucleotides or peptides that can specifically bind to a ligand, such as proteins, small molecules and even E. coli. Light-up RNA aptamers refer to aptamers that bind to a specific fluorogen that does not fluoresce on its own, but fluoresce upon the binding or docking of the RNA aptamer. Since the first invention of the light up RNA aptamer, malachite green aptamer, a variety of light-up RNA aptamers have been flourishing the field of live-cell imaging, DNA computation and diagnosis. The discovery of light-up RNA aptamers is achieved by a process called Systematic evolution of ligands by exponential enrichment (SELEX), which is testing the binding of randomly generated RNA molecules against a potential fluorogen and selecting the best ones as templates for next round of RNA generation. The ones which exhibit the strongest on/off ratio of fluorescence is chosen for downstream applications [8].
The Spinach RNA Aptamer
One of the most famous light-up RNA aptamers that was invented in 2011 by Paige et. al. is the Spinach aptamer. Its cognate fluorogen 3,5-difluoro-4-hydroxybenzylidene imidazolinone (DFHBI) is derived by the chromophore of green fluorescent protein (GFP) [9]. This aptamer has the brightness that could compare to its protein inspiration and can be expressed in E. coli for live-cell imaging as the organic DFHBI can directly pass across the membrane and bind to the aptamer.
The emergence of Spinach aptamer triggered the development of the alternate versions of the aptamer for various applications. By disrupting the proper folding of the aptamer and connect it to another module that is sensitive to its detecting target, such as ATP or a specific RNA sequence, the correct folding can be triggered to give off a signal; by connecting the aptamer to a self-cleaving ribozyme, it can be used to monitor progress of in vitro transcription [8].
Split Light-Up RNA Aptamers
By splitting the Spinach aptamer into 2 or more parts for contributing to its formation, a RNA AND logic gate can be constructed. In previous attempts constructing a split light-up RNA aptamer, none of the probes are proved to be truly modulable to fit its application into any RNA sequence. Split-Broccoli has a high ON/OFF ratio upon the binding of the 2 separate sequences, but it is limited to its own sequence; the universal split spinach probe can be used to detect virtually any sequence, but it involves a RNA-DNA hybrid [10,11]. Looking into the designs of previous designs, they can be classified into three categories: destabilizing folding, inducing misfolding and further splitting.
For our RNA Aptamer Probe targeting influenza sequences, we have chosen the destabilized Spinach aptamer for our design.
Using this design, we have developed several probes for the influenza A hemagglutinin, neuraminidase and polymerase basic 2 genes, which is chosen either for subtyping and influenza detection. The fluorescence data generated with these tools will allow diagnosis for personal treatment and epidemic monitoring at the same time.
Tracer – The Accompanying Tool for RNA Aptamer Probe
References
Global Influenza Programme [Internet]. World Health Organization. 2017 [cited 31 May 2018]. Available from: http://www.who.int/influenza/en/
Molinari NA, Ortega-Sanchez IR, Messonnier ML, Thompson WW, Wortley PM, Weintraub E, Bridges CB. The annual impact of seasonal influenza in the US: measuring disease burden and costs. Vaccine. 2007 Jun 28;25(27):5086-96
Landry ML. Diagnostic tests for influenza infection. Curr Opin Pediatr. 2011 Feb;23(1):91-7.
Patel P, Graser E, Robst S, Hillert R, Meye A, Hillebrand T, Niedrig M. rapidSTRIPE H1N1 test for detection of the pandemic swine origin influenza A (H1N1) virus. J Clin Microbiol. 2011 Apr;49(4):1591-3.
Green AA, Silver PA, Collins JJ, Yin P. Toehold switches: de-novo-designed regulators of gene expression. Cell. 2014 Nov 6;159(4):925-39.
Team Hong Kong-CUHK. iGEM. 2017 [cited 31 May 2017]. Available from: 2017.igem.com/Team:Hong_Kong-CUHK
Pardee K et. al. Rapid, Low-Cost Detection of Zika Virus Using Programmable Biomolecular Components. Cell. 2016 May 19;165(5):1255-66.
F. Bouhedda, A. Autour, and M. Ryckelynck, “Light-Up RNA Aptamers and Their Cognate Fluorogens: From Their Development to Their Applications,” Int. J. Mol. Sci., vol. 19, no. 1, p. 44, Dec. 2017./li>
Paige, J. S., Wu, K. Y., & Jaffrey, S. R. (2011). RNA Mimics of Green Fluorescent Protein. Science, 333(6042), 642–646.
K. K. Alam, K. D. Tawiah, M. F. Lichte, D. Porciani, and D. H. Burke, “A Fluorescent Split Aptamer for Visualizing RNA-RNA Assembly in Vivo,” ACS Synth. Biol., vol. 6, no. 9, pp. 1710–1721, 2017.
Kikuchi, N., & Kolpashchikov, D. M. (2017). A universal split spinach aptamer (USSA) for nucleic acid analysis and DNA computation. Chemical Communications, 53(36), 4977–4980.
