Team:Hong Kong-CUHK/Experiments
Top
RNA Aptamer Probe Influenza Detector
Proudly Presented by Team CUHK
HOME
PROJECT
WET LAB
DRY LAB
HUMAN PRACTICES
TEAM
AWARDS
Applied Design
Entrepreneurship
Hardware
Measurement
Model
Software
Email:
igem18.cu@gmail.com
Address:
CUHK, Sha Tin, NT, Hong Kong
CUHK Igem 2018
Experiments
Design Principle of RNA Aptamer Probe
The Spinach aptamer, along with its structural characteristics and photophysics are well-characterized. According to the crystal structures of Spinach and iSpinach-D5, the Spinach aptamer generally consists of two arms, P1 and P2, surrounding a G-quadruplex containing docking site of its fluorogen, DFHBI. While the docking site is indispensable for the formation of the Spinach-DFHBI complex, the lengths of the arms have shown to be less important by previous mutagenic studies to shorten the arms. In particular, the P1 arm length has a dramatic effect on the fluorescence level, which the single-base deletion can lead to complete loss of fluorescence in E. coli.
By adding a 11-bp arm to each side of the P1-truncated Spinach aptamer, which its sequence is complementary to a specific sequence of RNA, it can become a split light-up RNA aptamer with a modulable sequence design targeting influenza RNA. It works by having a misfolded / unfolded conformation of the Spinach aptamer on its own, but changing to a correctly folded conformation of the Spinach aptamer when hybridized to the influenza RNA. (Fig. Right)
In silico Design of RNA Aptamer Probe
As there are no known algorithms for predicting the binding of RNA to DFHBI and the resulting fluorescence level. According to previous data, shortening of the P2 arm did not have significant change in the fluorescence level of the Spinach aptamer. Based on these data, we have made 2 assumptions in our design: (1) The formation of the docking site is dependent on the correct folding of the P1 arm. (2) The P1 arm folding is optimized when the hybridization of the variable regions is the most favorable. Based on these two assumptions, we utilized the Invitrogen BLOCK-iT siRNA Designer, which can find the region of RNA with the least amount of secondary structures, as well as human genome BLAST. The web page generates the sequences of 25 bp length.
The gene sequences of influenza A were chosen as follows (Table Below). The hemagglutinin and neuraminidase genes were selected for influenza subtyping, while the region of Polymerase Basic 2 gene that is ubiquitous in most influenza A genomes were chosen for influenza detection (This was inspired by the influenza expert that we interviewed, Prof. Lee Shui-shan ). After inputting the selected sequences in BLOCK-iT designer, we have chosen the ones with the GC content around ~50% and randomly selected a 22 bp region for probe design (Fig. Above).
In vitro screening of probes
1. Production of RNA Aptamer Probes and Target RNA
To produce RNA probes and their target RNAs for assays, we used in vitro transcription kits. We designed oligos that can hybridize to each other and extend in a PCR reaction to overcome to length limit of oligo synthesis. After Phusion PCR, the product was gel purified and used as a template in NEB HiScribe T7 Quick High Yield RNA Synthesis Kit. After DNaseI treatment, the reaction was directly purified using phenol:chloroform extraction, and the RNA concentration was measured with NanoDrop 2000. RNA molar concentration was calculated using the corresponding MW according to its length.
2. Aptamer Refolding Assay
In order to investigate the effectiveness of our designed aptamers, in vitro transcribed RNA aptamers (1uM) were mixed with its target, 2X aptamer folding buffer (20mM Tris-HCl, 200mM KCl, 10mM MgCl2) and fluorophore DFHBI, inspired by Kikuchi N, 2016. Controls were set up by mixing the above components without RNA aptamer. The fluorescent signal was observed under blue light box / Chemidoc and was determined by CLARIOstar plate reader.
Characterization of Aptamer
1. Specificity
Though our aptamers were designed to target hemagglutinin or neuraminidase genes of influenza subtypes, unwanted binding between aptamer and target of two subtypes may happen in real life. 5 of our aptamers (i.e. for N9, N2, H7, H3, PB2) that performed well in previous RNA fluorescence assay were selected to investigate their cross-reactivity. They were mixed with targets from different subtypes sequence and subjected to aptamer refolding assay.
2. Detection Limit
It is important to know the minimum amount needed to distinguish the fluorescence signal under the blue light box by naked eye, in order to understand at which stage of influenza latency can be viral RNA be detected. We have mixed 2uM aptamer (N2-694 and N9-545 aptamers) with different concentration of its target (i.e.1.5uM, 1uM, 0.5uM, 0.2uM, 0.1uM and 0.05uM) and performed aptamer refolding assay as usual.
3. Ion Dependency
As the test sample will be individuals’ nasal fluid, we wondered if the addition of nasal fluid will influence the RNA aptamer folding, for instance, the ionic composition in nasal fluid.  We have selected sodium ions, potassium ions, calcium ions and magnesium ions in different concentration to test for the fluorescent signals given from our 2 probes (N2-694 and N9-545 aptamers). Ions were added to aptamer folding buffer in form of salt solution. Concentration of different ions in nasal fluid, in aptamer folding buffer are shown below.
The range of concentrations selected for each ions are shown below. Those red number indicate the original amount of ions presented in the aptamer folding buffer.
Optimization of Aptamer Folding Condition
In order to trace the change in fluorescence signal with the time and temperature, BioRad real-time PCR machine was employed for the monitoring fluorescence signal during the aptamer refolding assay using SYBR Green Mode.
E. coli Expression of RNA Aptamer Probes
Although our project aims to produce a probe outside the context of a cell, the screening of probes is cheaper if they can be expressed and tested in E. coli. Moreover, it would allow our future iGEMers to be able to use our probes in their usual contexts, which is in a chassis like E. coli. To test if this is applicable, we co-expressed the probes and targets using the Novagen Duet vector system. While we modified pRSFDuet-1 vector to generate a T7 promoter-based aptamer expression system, we also cloned the target genes into the multiple cloning site of pACYCDuet-1. The resulting constructs are as follows (Fig. Below).
While the RNA aptamer probes are immediately following the T7 promoter and followed by the T7 terminator, the influenza RNA is included in the transcripts of the MCS. After IPTG induction in co-transformed BL21 Star (DE3), they are incubated in medium containing DFHBI and their fluorescence in measured by BMG CLARIOStar microplate reader. We obtained the strain from Team NUS Singapore-A as a part of our collaboration .
Improving existing biobricks and project: RNA Reporters for Promoter Activity Measurement
Previously, the Technical University of Denmark 2014 team utilized the tRNALys3-Spinach2.1 (BBa_K1330000) for measurement of the Anderson promoter library. Due to the illegal site of the Spinach2 sequence, they designed the Spinach2.1 RNA Reporter which has lower fluorescence than the original Spinach2.
Despite the brightness of Spinach2, It has a melting temperature of 38°C. This means the characterization of heat-induced promoters is not favored by using this reporter. On the other hand, our collaborating partner NUS iGEM 2018 is trying to develop a RNA reporter connected with the chaperone promoter phtpG.
To overcome the obstacle of (1) brightness and (2) temperature sensitivity, we constructed 3 new RNA aptamers for the iGEM registry, all using the tRNALys3 scaffold of the original biobrick. The biobricks include miniSpinach, iSpinach and dimeric Broccoli. Their in vivo fluorescence are tested in E. coli.
References
W. Q. Ong, Y. R. Citron, S. Sekine, and B. Huang, “Live Cell imaging of endogenous mRNA Using RNA-Based fluorescence ‘turn-on’ probe,” ACS Chem. Biol., vol. 12, no. 1, pp. 200–205, 2017.
BLOCK-iT Designer. Invitrogen. https://rnaidesigner.thermofisher.com/rnaiexpress/.
K. K. Alam, K. D. Tawiah, M. F. Lichte, D. Porciani, and D. H. Burke, “A Fluorescent Split Aptamer for Visualizing RNA-RNA Assembly in Vivo,” ACS Synth. Biol., vol. 6, no. 9, pp. 1710–1721, 2017.
Burke W. The Ionic Composition of Nasal Fluid and Its Function. Health [Internet]. 2014 Mar 6:720-728. Available from: http://dx.doi.org/10.4236/health.2014.68093
Kikuchi N, Kolpashchikov DM. Split Spinach Aptamer for Highly Selective Recognition of DNA and RNA at Ambient Temperatures. ChemBioChem [Internet]. 2016 Jul 15;17(17):1589–92. Available from: http://dx.doi.org/10.1002/cbic.201600323
