Team:UCC Ireland/erythromycin biosensor
Home
Project
Description Biosensors Demonstrate Engagement Entrepreneurship Applied Design
Labwork
Experiments Cell Free Protocols Interlab Notebook Measurement
Modelling
Modelling Overview Hardware Software
Collaboration
Hardware
Human Practices
Silver Gold
Attributions
The Team
UCC iGEM 2017
Background
The expression of the macrolide resistance operon in the E. Coli strain Tf481A is thought to be negatively regulated at the transcriptional level by the binding of the repressor protein, MpHR(A), to the promoter of the mph(A) gene. In nature, this circuitry confers erythromycin resistance in Tf481A E.coli cells. The operon is composed of the following circuit: mph(A)- mrx- MpHR(A). The mph(A) gene codes for a 2’ macrolide-phosphotransferase, which is a strong inactivator of macrolides, such as erythromycin. In the presence of erythromycin, the binding of the repressor protein, MpHR(A) to the mph(A) promoter in inhibited and transcription of the mph(A) operon is activated.(Möhrle, Stadler and Eberz, 2007)
Fig. 1. Autoradiogram of SDS-PAGE gel adapted from (Noguchi et al., 2000). In the PTZ3517 plasmid, which contained a fragment consisting only of the mph(A) and mrx genes, high-level production of Mph(A) was recognised in both the presence (+) and absence (-) of erythromycin. However, in the PTZ3510 plasmid, which included both the mph(A) and mrx genes AND the downstream region of the mrx gene, the level of Mph(A) produced in the absence of erythromycin was much lower than that produced in the presence of erythromycin.
 
This indicated that the gene encoding for the MphR(A) regulatory protein is located downstream from the mrx gene and that the expression of Mph(A) is negatively regulated by it. Even though experiments have shown that the level of transcripts of Mph(A) are raised when cells are cultured with erythromycin…
 
The kinetics of the system and relationship between the quantity of the inducer (erythromycin) present and quantity of the protein (macrolide phosphotransferase) produced remain unclear.
 
Moreover, the results of gel mobility shift assays with purified MpHR(A) repressor protein and a DNA fragment consisting of the mph(A) promoter sequence have indicated that the repressor protein binds specifically to this promoter sequence. However…
 
The binding affinity of MphR(A) to mph(A) in the presence of various concentrations and types of macrolides needs to be further investigated.
 
Nevertheless, this research suggests that production of a visible chromoprotein, like amilCP, could be produced in the presence of erythromycin, if it was to be cloned in place of the mph(A) gene, downstream of the mph(A) promoter gene in a plasmid with a constitutively expressed MpHR(A) gene intact. The gaps in knowledge that we identified in bright blue above informed our experimental approach.
 
Experimental approach
Usage of the pJKR-H-mphR plasmid with mph(A) promoter gene (as the sensor element) and sfGFP (as the readout) was used in initial assays to answer the following questions:
Is the production of sfGFP repressed in the absence of erythromycin?
What concentrations of erythromycin induce production of sfGFP?
Is the biosensor sensitive enough to produce detectable levels of sfGFP when placed in the erythromycin concentrations in the range of the maximal residual limit in raw milk (40 micrograms/966 millilitres of raw milk)?
 
We took this approach as sfGFP is routinely used as a readout reporter protein and has been established to be sensitive enough to show detectable variations in expression, even in response to narrow ranges and minute concentrations of small molecule ligands and transcription factors. On the most fundamental level, determining that a measurable dose-response relationship exists between erythromycin induction concentrations and the production of sfGFP would serve as a proof of concept for our project. As an extension, establishing the binding affinity of the mphR(A) repressor to the mph(A) promoter in the absence of erythromycin might further explain the results of our assays using the pJKR-mpH-R plasmid (information about our assays and analysis of our experimental findings can be found below!). Simultaneously, we also constructed a basic part with AmilCP as the readout protein (Basic part: BBa_K2344055).
 
Construct Design
Here’s a simplified schematic of our erythromycin biosensor!
FIG. 2 Basic part: BBa_K2344055; Erythromycin inducible expression of Mph(A) and amilCP.
 
The MpHR(A) repressor protein is constitutively expressed (under control of the constitutive promoter: ProB). The repressor protein is thought to be bound to the Mph(A) promoter region in the absence of erythromycin. The amilCP chromoprotein gene is under control of the inducible promoter Mph(A). Therefore, inhibition of MpHR(A)-Mph(A) binding in the presence of erythromycin was expected to induce production of the amilCP chromoprotein. Our assembly was successful as confirmed by sequencing! Results from our assays with BBa_K2344055 can be found below!
 
Assays: pJKR-H-mphR plasmid!
FIG.3: pJKR-H-mphR plasmid with ProB Constitutive promoter, MphR(A) Repressor Protein, sfGFP and MphR(A) inducible promoter highlighted
 
DH5αE. Coli cells transformed with the pJKR-H-mphR plasmid were grown on ampicillin plates. Viable ampicillin-resistant colonies were selected and inoculated into overnight LB and M9 cultures. Results from periodic absorbance (OD600) measurements of 100 μl samples of the LB and M9 cultures reported slower and stunted bacterial growth rates in M9 media. Therefore, 100 μl samples of LB liquid cultures of the pJKR-H-mphR plasmid were induced with varying volumes of erythromycin and were set up in a black clear bottom 96-well plate to achieve the following erythromycin concentrations (in μM) in the pattern shown in the table below:
FIG.4: Arrangement of 100μ l bacterial cultures in 96-well plate with varying concentrations of erythromycin
 
The 96-well plate was incubated in a shaker at 37 degrees for 6 hours, with both absorbance and fluorescence in each well measured simultaneously every 30 minutes.
 
Troubleshooting: pJKR-H-mphR plasmid!
FIG.5:  Graphs showing increasing absorbance of mph(A) promoter during 6-hour incubation of the DH5αE.Coli cells transformed with pJKR-H-mphR plasmid cultured in erythromycin concentrations of 1400, 450, 150, 50, 10, 5, 1 and 0μMin LB media. Constitutively expressed GFP and Tat-HA were used as positive and negative controls respectively.
 
We noted that the growth of cells in the 150 and 1400 μMwere slower in comparison to cells cultured in lower concentrations of erythromycin. Previously, studies have shown that erythromycin was toxic to E. coli (without erythromycin resistance) at concentrations as low as 50 μM. However, with expression of the erythromycin resistance gene (eryR), only slight toxicity of pJKR-H-mphR carrying E.coli cells should be observed up till erythromycin concentrations of up to 1.4 mM. Therefore, these results were surprising. We believe that the additional metabolic strain of producing the sfGFP protein along with proteins conferring resistance i.e. macrolide 2- phosphotransferase could explain slower growth rates.
FIG. 6: Graphs showing induction of mph(A) promoter during 6-hour incubation of the DH5α E.Coli cells transformed with pJKR-H-mphR plasmid cultured in erythromycin concentrations of 1400, 450, 150, 50, 10, 5, 1 and 0μMin LB media. Constitutively expressed GFP and Tat-HA were used as positive and negative controls respectively.
 
In this assay, increasing fluorescence was noted in all cultures, especially in the 1, 5 and 10 μM erythromycin concentrations as shown in the graph below. This seemed counter-intuitive as we expected a dose-response relationship between the amount of erythromycin added and the amount of sfGFP produced. We suspected that the potential differences in fluorescence may be occurring due to varying numbers of sfGFP-producing cells in each culture (as evidenced by the differential rates of growth as in FIG. 3 above). Therefore, in order to attempt to understand the effect of erythromycin on single E.coli cells, the results of our experiment were normalised by doing the following calculation: fluorescence measurement absorbance measurement and plotting it against time as shown below:
 
FIG.7: Indirectly comparing the effect of erythromycin on a single cell carrying the pJKR-H-mphR plasmid by dividing fluorescence values by absorbance values at all time points.
 
However, even with normalisation, a clear dose-response relationship could not be established. We also monitored the growth and fluorescence of an uninduced culture and our results are shown below.
FIG. 8: Graph showing increase in absorbance and fluorescence of bacterial culture of cells transformed with pJKR-H-mphR over time, uninduced (0 μM).   Moreover, 0μMinduction concentrations of erythromycin also showed sizeable increases in fluorescence and absorbance over time as well casting doubt on the affinity of binding of the repressor to the mph(A) promoter (as shown above). Assays to investigate this relationship have been explored in the future plans section below.   Results: pJKR-H-mphR plasmid!  Suspecting that the autofluorescence of the LB media (in which our cells were incubated in) may be confounding our results, we resorted to using the IVIS and the special spectral unmixing feature to separate the fluorescent signals coming from LB and sfGFP respectively.    IVIS imaging was also used at the endpoint of the experiment at 6 hours. Spectral unmixing using the positive control (constitutively expressed GFP) and blank LB media allowed us to adjust for the effect that autofluorescent LB media has, by separating the fluorescence signals from LB and sfGFP. As seen here, fluorescence from sfGFP is seen to be higher in cells induced with higher concentrations of erythromycin.
FIG.9: sfGFP fluorescence detected by the IVIS with table showing concentrations of erythromycin in bacterial cultures in 96-well plate as a reference.
 
Assays: BBa_K2344055 Erythromycin Biosensor!
 
Our assembly was successful (as confirmed by sequencing), was transformed into Dh5αcells and grown on Kanamycin plates. Overnight cultures were then set-up. Unfortunately, amilCP production was not visible to the naked eye on growth in LB cultures of 5, 50 and 500μ M of erythromycin.
 
As an extension, we plan to grow cells transformed with BBa_K2344055 in M9 media so as to remove the confounding effect of autofluorescent LB and facilitate visibility of AmilCP to the naked eye. However, due to time restrictions and the length of time required for bacterial growth in M9 media, we were unable to carry out this assay.
 
Future Plans
More assays will need to be conducted to characterise the interactions between the different components of the system. Even in the IVIS, residual levels of sfGFP fluorescence were seen in negative control samples that were not induced with erythromycin despite controlling for the autofluorescence of LB. This suggests a defect in binding of the repressor protein to the mph(A) promoter region. Given more time, we would conduct gel mobility shift assays to ascertain a high binding affinity of mphR(A) to mph(A) in the absence of erythromycin to ensure that the biosensor will not produce false positive results, that can lead to unnecessary wastage of raw milk and unnecessary production costs for dairy farmers.
 
FIG.10: Future assays that need to be planned.
 
Ideally, assays with BBa_K2344055 in M9 media will also be conducted in the future to confirm production of amilCP in the presence of erythromcyin.
Möhrle, V., Stadler, M. and Eberz, G. (2007). Biosensor-guided screening for macrolides. Analytical and Bioanalytical Chemistry, 388(5-6), pp.1117-1125.
Noguchi, N., Takada, K., Katayama, J., Emura, A. and Sasatsu, M. (2000). Regulation of Transcription of the mph(A) Gene for Macrolide 2'-Phosphotransferase I in Escherichia coli: Characterization of the Regulatory Gene mphR(A). Journal of Bacteriology, 182(18), pp.5052-5058.
