"
Page
Discussion
View source
History
teams
Log in
 
Team:Copenhagen/Protocols
From 2012.igem.org
Home Team Project Protocols Parts Safety Human Practises Sponsors Contact Us
Cultivation of tranformed cells:
Materials:
LB Media
Antibiotics (Matching the Resistance marker gene)
Inoculation needle
Procedure:
A colony is chosen from the LB-plate, and the colony is transferred to a small tube with 20 µl water.
5 µl of each colony is transferred to 5 ml LB + 5 µl  antibiotics (in our case: chloramphenicol) with a pipet tip.
Incubate over night at 37°C in the shaking incubator.
Plating on LB plates:
Materials
LB plates (with antibiotics)
Transformed E. coli cells
Sterilized Gridalzky spatula
Procedure
The transformed cells is transferred to an LB plate containing antibiotics and dispersed with the Drigalski spatula.
The transformed cells are incubated over night at 37°C.
Next day; the plates are checked for visual colonies. These are cultivated.
Primer design for point mutation in Lux Casette
The Lux cassette of 5798 bp had a Xbal restriction site that was removed by designing to primers of 25-30 base pairs with overlapping sequences. This sequence was designed as a USER site that covered the Xbal restriction site. The chosen USER site sequence should be identical to the template strand, except for the single base pair, which is to be mutated.
Two primers to each end of the Lux cassette were also designed.
Two PCR reactions were carried out with two sets of primers. One set each containing a primer annealing to the 3’ and a primer containing the new USER site sequence. And an other set containing a primer annealing the 5’ end respectively together with the complementary primer encoding the new USER site sequence.
PCR reaction for amplification of backbone pSB1C3
The following protocol is used to amplify the pSB1C3 backbone vector.
To each PCR tube the following is added:
2.5 µl of each primer
4 µl template DNA
41 µl MasterMix
Master Mix
10 mM
X7 Buffer
10 mM
dNTP's
2 mM
X7 DNA polymerase (Produced in our lab)
The PCR programme used:
Step
Temperature/°C
Time
Cycles
Int. Denaturation
95°
120 sec
Denaturation
95°
30 sec
32
Tm
55°
12 sec
32
Elongation
68°
1.25 min
32
Final extension
68°
10 min
Hold
10°
∞
Colony PCR
20 µl H2O is added to each PCR tube. Colonies are chosen from the plates and resuspended in the PCR tubes. This serves as the template solution.
To each PCR tube the following is added:
0.5 µl of each primer
3 µl template DNA
5 µl MangoMix (5 mM) (Bioline)
MangoMix
MangoTaq DNA polymerase
Orange reference dye
MgCl2
dNTP
The PCR programme used:
Step
Temperature/°C
Time
Cycles
Int. Denaturation
96°
3 min
Denaturation
96°
20 sec
25
Tm
60°
20 sec
25
Elongation
72°
* sec
25
Final extension
72°
7 min
Hold
10°
∞
*= elongation time for MangoTaq DNA polymerase should be calculated so it is compatible with the size of the template. Taq writes 1kb/60sec.
PCR reactions for individual genes
The following protocol is used to amplify the individual genes.
To each PCR tube the following is added:
2.5 µl of each primer
0.5 µl template DNA
44.5 µl MasterMix
Master Mix
10 mM
X7 Buffer
10 mM
dNTP's
2 mM
X7 DNA polymerase (Produced in our lab)
The PCR programme used:
Step
Temperature/°C
Time/Sec
Cycles
Int. Denaturation
98°
30 sec
Denaturation
98°
10 sec
32
Tm
60°
20 sec
32
Elongation
72°
* sec
32
Final extension
72°
7 min
Hold
10°
∞
*= elongation time for X7 DNA polymerase should be calculated so it is compatible with the size of the template. X7 polymerase writes 1kb/30-45sec.
USER Cloning
Procedure:
The PCR product must be purified from a gel before (GenEluteTM HP plasmid MiniPrep kit (Sigma-Aldrich)) used in USER cloning.
The PCR products is added to Eppendorf tubes to give a total volume of 8 µl in each tube (Thus, in case of two PCR products, add 4 µl and 4 µl or 3 µl and 5 µl etc.)
The USER mix components are mixed: (in each tube)
USER Mix
1x USER Mix
NEBuffer 4 (10x diluted)
0.5 µL
BSA
0.5 µL
Dpn1
1 µL
The USER mix is transferred to each Eppendorf tube and incubated for 2 hours at 37°C.
1 µL USER enzyme is added pr. tube and the mixture is incubated for 40 minutes at 37°C and for 2 hours  min at 25°C.
Transformation: See transformation protocol.
USER enzyme: USER™ (Uracil-Specific Excision Reagent) Enzyme generates a single nucleotide gap at the location of a uracil. USER Enzyme is a mixture of Uracil DNA glycosylase (UDG) and the DNA glycosylase-lyase Endonuclease VIII (New England biolabs® Inc.).
Transformation in E. coli DH5α or E. Cloni
The preparations for the transformation can preferably be done, while the USER cloning is incubating.
Materials
E. coli DH5α (Bioline) or E. cloni-5α (Lucigen) competent cells
USER reaction
Procedure:
LB plates are taken out of the refrigerator and marked. Remember to use LB plates with the right antibiotics.
50 µL competent E. coli DH5α cells per USER reaction is taken from the -80C° freezer and place on ice. Additionally, 1,5 ml tubes are placed on ice.
5 µL USER reaction mix is added to the 50 µL competent E. coli DH5α cells. Mix well by pipetting.
The cells are placed on ice for 30 min.
The hot plate is set on 42 °C, and each transformation is heat shocked for 1 min. The cells are put directly on ice for 2 min afterwards.
200 µL recovery medium is added pr. cell tube.
The cells are placed on hot plate on 300 rpm for 30 minutes.
The hot plate is increased to 1000 rpm, and the cells are placed hereon for additional 1 hour.
Control of Lux cassette
Procedure:
One colony is chosen from the LB-plate, and transferred to a small plating tube with 2.5 ml LB-medium with antibiotics.
Incubate over night at 37°C in the shaking incubator.
The over night culture is transferred to an Erlenmeyer flask with 100 ml LB-medium and regularly checked with OD
When OD600 has reached 0.4-0.6 100 µl IPTG is added
The flask is taking into a dark room to see the immediate effect.
Cultivation
Plating on LB plates
Primer design for point mutation in Lux Casette
PCR reaction for amplification of backbone pSB1C3
Colony PCR
PCR reactions for individual genes
USER Cloning
Transformation
Control of Lux cassette
Retrieved from " http://2012.igem.org/Team:Copenhagen/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
