"
Page
Discussion
View source
History
teams
Log in
 
Team:Copenhagen/Results
From 2012.igem.org
Home Team Project Protocols Parts Safety Human Practises Sponsors Contact Us
Theory of USER Cloning
The Registry of Standard Biological Parts and most iGEM teams use the conventional 3A Standard Assembly protocol to gather BioBricks together.  However, we are using USER, uracil-specific excision reaction, cloning. Why? The method uses neither restriction enzymes nor ligase, it is simple and rapid, and it enables the simultaneous assembly of multiple target DNA sequences into a plasmid vector. In addition, it avoids any troubles with illegal restriction sites and it leaves no scars thus making it ideal for protein fusions etc.
The USER concept was introduced to the iGEM competition in 2009 by the DTU Denmark team. Two years after, the DTU-2 team took it a step further and made a standardized system based on the principle of USER cloning. They named it the Plug’n’Play system.
The recipe is simple: Mix pre-produced PCR product and backbone vector with USER enzyme and your parts are putted together.
More detailed, the template DNA is amplified using primers designed with an upstream row of 8-9 nucleotides with one deoxyuridine directly upstream of the gene. In this way, the PCR product will be flanked by linkers on both sides. To allow assembly of multiple PCR product in a specific order, each given part need to have its own unique linker sequence.
The concept is shown in the figure below where six different BioBricks are assembled.
(figure made by DTU-Denmark2 iGEM 2011)
Linker tabel provides information about each individual linker sequence and the primer design.
(figure made by DTU-Denmark2 iGEM 2011)
Upon mixing of the PCR products, the solution is treated with Dpn1 in order to remove the template DNA. The USER enzyme will afterwards remove the uracil bases creating 8-9 nucleotide overhangs in the 3’end. These sticky ends will hybridize to matching overhangs created in the assembly backbone vector. The final step is to incubate and transform the solution into E.coli, which will later grow agar plates.
The figure below illustrates the easy steps:
(figure made by DTU-Denmark2 iGEM 2011)
USER vs Gibson Assembly USER cloning and Gibson Assembly share many principle: both methods enable simultaneous assembly of multiple DNA parts, uses gel-purified PCR products and customized primers, leave no scares, and importantly, no dependence of restriction enzymes.  Gibson Assembly was envolved by Daniel G. Gibson at the J. Craig Venter Institute in 2009  (Gibson, D.D et. al., 2009. Enzymatic assembly of DNA molecules up to several hundred kilobases. Nature Methods, vol. 6, no. 5, pp. 343-47 ) Cambridge iGEM 2010 introduced an elaborated description completed with software tools to facilitate the use of Gibson Assembly (https://2010.igem.org/Team:Cambridge/Gibson/Introduction).
Gibson Assembly relies on the creation of a 40 bp overlap between two sequences, ei. A and B,  that are to joined.  The magic lies in the primer design. One primer is designed to extend sequence A downstream, the other to extend sequence B upstream. In this way, the downstream end of new PCR product A has the same 20 base pairs as the beginning of sequence B. Additionally, the upstream end of new PCR product B has the same 20 base pairs as the end of sequence A.
Here after an (isothermal T5) exonuclease chews back at the 5’-end of both dsDNA pieces leaving 3’-overhangs. A (Phusion DNA) polymerase fills in the gaps and (Taq) ligase seals the nicks, and the joined DNA is ready for transformation. Thus, the major differences between USER and Gibson lie in the size and creation of the overhangs and the enzymes involved.
(figure made by DTU-Denmark2 iGEM 2011)
ProC and YFI
We performed USER cloning and managed to assemble ProC and YF1 in the pSB1C3 backbone. This has been submitted as a BioBrick.
For further information please visit the Parts page.
Lux Cassette
After trying to amplify the Lux cassette from the BioBrick plate several times, we checked the sequence (from sequencing) in the parts registry using BLAST and found that it was an E.coli surface protein and not the Lux cassette.  We contacted Thomas Knight from the iGEM headquarter who kindly provided us with a Lux Cassette. This contained one of the “banned” Xbal restriction sites, which we have removed and the Lux cassette is submitted as a BioBrick. We have transformed the Lux cassette from Thomas Knight into E.cloni® 5α (purchased from Lucigen).  As a control we exploited that the gene cassette had a LacI promotor and we confirmed, after adding IPTG, that the Lux cassette is able to make the bacteria bioluminescent.
For further information please visit the Parts page.
Experiences with USER cloning
We have found, that in order to make USER cloning work, it is very important that the molecular concentration of the parts, that are to be assembled, is as equal as possible. This means that a small piece must be of lower concentration in order to assemble it to a larger piece so the number of each piece is approximately the same. For ProC (35 bp) and YF1 (1131bp) a x10.000 dilution of ProC was necessary in order to assemble the two pieces.  This very problem has made it hard to assemble our construct since we have pieces ranging from 35bp to almost 6000bp.
Retrieved from " http://2012.igem.org/Team:Copenhagen/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
