<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Copenhagen_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Copenhagen/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Copenhagen/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Copenhagen/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Copenhagen/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Copenhagen/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Copenhagen/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navbar-container"><DIV id="navbar"><A href="https://2012.igem.org/Team:Copenhagen ">Home</A><A href="https://2012.igem.org/Team:Copenhagen/Team">Team</A><A href="https://2012.igem.org/Team:Copenhagen/Project">Project</A><A href="https://2012.igem.org/Team:Copenhagen/Protocols">Protocols</A><A href="https://2012.igem.org/Team:Copenhagen/Parts">Parts</A><A href="https://2012.igem.org/Team:Copenhagen/Safety">Safety</A><A href="https://2012.igem.org/Team:Copenhagen/Ethics">Human Practises</A><A href="https://2012.igem.org/Team:Copenhagen/Attributions">Sponsors</A><A href="https://2012.igem.org/Team:Copenhagen/Contact">Contact Us</A></DIV></DIV></DIV><TABLE id="graa" cellpadding="20px"><TBODY><TR><TD width="660px" height="100%" valign="top"><H2>Cultivation of tranformed cells:</H2><B>Materials:</B><UL><LI>LB Media
</LI><LI>Antibiotics (Matching the Resistance marker gene)
</LI><LI>Inoculation needle
</LI></UL><B>Procedure:</B><OL><LI>A colony is chosen from the LB-plate, and the colony is transferred to a small tube with 20 µl water.
</LI><LI>5 µl of each colony is transferred to 5 ml LB + 5 µl  antibiotics (in our case: chloramphenicol) with a pipet tip.
</LI><LI>Incubate over night at 37°C in the shaking incubator.
</LI></OL><H2>Plating on LB plates:</H2><B>Materials</B><UL><LI>LB plates (with antibiotics)
</LI><LI>Transformed <I>E. coli</I> cells
</LI><LI>Sterilized Gridalzky spatula
</LI></UL><B>Procedure</B><UL><LI>The transformed cells is transferred to an LB plate containing antibiotics and dispersed with the Drigalski spatula.
</LI><LI>The transformed cells are incubated over night at 37°C.
</LI><LI>Next day; the plates are checked for visual colonies. These are cultivated.
</LI></UL><H2>Primer design for point mutation in Lux Casette</H2><UL><LI>The Lux cassette of 5798 bp had a Xbal restriction site that was removed by designing to primers of 25-30 base pairs with overlapping sequences. This sequence was designed as a USER site that covered the Xbal restriction site. The chosen USER site sequence should be identical to the template strand, except for the single base pair, which is to be mutated. 
</LI><LI>Two primers to each end of the Lux cassette were also designed.
</LI><LI>Two PCR reactions were carried out with two sets of primers. One set each containing a primer annealing to the 3’ and a primer containing the new USER site sequence. And an other set containing a primer annealing the 5’ end respectively together with the complementary primer encoding the new USER site sequence. 
</LI></UL></TD></TR></TBODY></TABLE><H2>PCR reaction for amplification of backbone pSB1C3</H2>
The following protocol is used to amplify the pSB1C3 backbone vector.
To each PCR tube the following is added:<UL><LI>2.5 µl of each primer
</LI><LI>4 µl template DNA
</LI><LI>41 µl MasterMix
</LI></UL><TABLE id="graa" align="center" border="1"><TBODY><TR><TD colspan="2"><B>Master Mix</B></TD></TR><TR><TD>10 mM</TD><TD>X7 Buffer</TD></TR><TR><TD>10 mM</TD><TD>dNTP's</TD></TR><TR><TD>2 mM</TD><TD>X7 DNA polymerase (Produced in our lab)</TD></TR></TBODY></TABLE><TABLE id="graa" align="center" border="1"><TBODY><TR><TD colspan="4"><B>The PCR programme used:</B></TD></TR><TR><TD>Step</TD><TD>Temperature/°C</TD><TD>Time</TD><TD>Cycles</TD></TR><TR><TD>Int. Denaturation</TD><TD>95°</TD><TD>120 sec</TD></TR><TR><TD>Denaturation</TD><TD>95°</TD><TD>30 sec</TD><TD>32</TD></TR><TR><TD>Tm</TD><TD>55°</TD><TD>12 sec</TD><TD>32</TD></TR><TR><TD>Elongation</TD><TD>68°</TD><TD>1.25 min</TD><TD>32</TD></TR><TR><TD>Final extension</TD><TD>68°</TD><TD>10 min</TD></TR><TR><TD>Hold</TD><TD>10°</TD><TD>∞</TD></TR></TBODY></TABLE><H2>Colony PCR</H2>
20 µl H2O is added to each PCR tube.
Colonies are chosen from the plates and resuspended in the PCR tubes. This serves as the template solution.
	To each PCR tube the following is added:<UL><LI>0.5 µl of each primer </LI><LI>3 µl template DNA</LI><LI>5 µl MangoMix (5 mM) (Bioline)</LI></UL><TABLE id="graa" align="center" border="1"><TBODY><TR><TD><B>MangoMix</B></TD></TR><TR><TD>MangoTaq DNA polymerase</TD></TR><TR><TD>Orange reference dye</TD></TR><TR><TD>MgCl2 </TD></TR><TR><TD>dNTP</TD></TR></TBODY></TABLE><TABLE id="graa" align="center" border="1"><TBODY><TR><TD colspan="4"><B>The PCR programme used:</B></TD></TR><TR><TD>Step</TD><TD>Temperature/°C</TD><TD>Time</TD><TD>Cycles</TD></TR><TR><TD>Int. Denaturation</TD><TD>96°</TD><TD>3 min</TD></TR><TR><TD>Denaturation</TD><TD>96°</TD><TD>20 sec</TD><TD>25</TD></TR><TR><TD>Tm</TD><TD>60°</TD><TD>20 sec</TD><TD>25</TD></TR><TR><TD>Elongation</TD><TD>72°</TD><TD>* sec</TD><TD>25</TD></TR><TR><TD>Final extension</TD><TD>72°</TD><TD>7 min</TD></TR><TR><TD>Hold</TD><TD>10°</TD><TD>∞</TD></TR></TBODY></TABLE>
*= elongation time for MangoTaq DNA polymerase should be calculated so it is compatible with the size of the template. Taq writes 1kb/60sec.



<H2>PCR reactions for individual genes</H2>
The following protocol is used to amplify the individual genes.
To each PCR tube the following is added:
<UL><LI>2.5 µl of each primer
</LI><LI>0.5 µl template DNA
</LI><LI>44.5 µl MasterMix
</LI></UL><TABLE id="graa" align="center" border="1"><TBODY><TR><TD colspan="2"><B>Master Mix</B></TD></TR><TR><TD>10 mM</TD><TD>X7 Buffer</TD></TR><TR><TD>10 mM</TD><TD>dNTP's</TD></TR><TR><TD>2 mM</TD><TD>X7 DNA polymerase (Produced in our lab)</TD></TR></TBODY></TABLE><TABLE id="graa" align="center" border="1"><TBODY><TR><TD colspan="4"><B>The PCR programme used:</B></TD></TR><TR><TD>Step</TD><TD>Temperature/°C</TD><TD>Time/Sec</TD><TD>Cycles</TD></TR><TR><TD>Int. Denaturation</TD><TD>98°</TD><TD>30 sec</TD></TR><TR><TD>Denaturation</TD><TD>98°</TD><TD>10 sec</TD><TD>32</TD></TR><TR><TD>Tm</TD><TD>60°</TD><TD>20 sec</TD><TD>32</TD></TR><TR><TD>Elongation</TD><TD>72°</TD><TD>* sec</TD><TD>32</TD></TR><TR><TD>Final extension</TD><TD>72°</TD><TD>7 min</TD></TR><TR><TD>Hold</TD><TD>10°</TD><TD>∞</TD></TR></TBODY></TABLE>
*= elongation time for X7 DNA polymerase should be calculated so it is compatible with the size of the template. X7 polymerase writes 1kb/30-45sec.<H2>USER Cloning</H2><B>Procedure:</B><OL><LI>The PCR product must be purified from a gel before (GenEluteTM HP plasmid MiniPrep kit (Sigma-Aldrich)) used in USER cloning. 
</LI><LI>The PCR products is added to Eppendorf tubes to give a total volume of 8 µl in each tube
(Thus, in case of two PCR products, add 4 µl and 4 µl or 3 µl and 5 µl etc.)
</LI><LI>The USER mix components are mixed: (in each tube)  
<TABLE id="graa" align="center" border="1"><TBODY><TR><TD>USER Mix</TD><TD>1x USER Mix</TD></TR><TR><TD>NEBuffer 4 (10x diluted)</TD><TD>0.5 µL</TD></TR><TR><TD>BSA</TD><TD>0.5 µL</TD></TR><TR><TD>Dpn1</TD><TD>1 µL</TD></TR></TBODY></TABLE></LI><LI>The USER mix is transferred to each Eppendorf tube and incubated for 2 hours at 37°C.
</LI><LI>1 µL USER enzyme is added pr. tube and the mixture is incubated for 40 minutes at 37°C and for 2 hours  min at 25°C.
</LI><LI>Transformation: See transformation protocol.
</LI></OL><B>USER enzyme:</B>
USER™ (Uracil-Specific Excision Reagent) Enzyme generates a single nucleotide gap at the location of a uracil. USER Enzyme is a mixture of Uracil DNA glycosylase (UDG) and the DNA glycosylase-lyase Endonuclease VIII (New England biolabs® Inc.).

<H2>Transformation in <I>E. coli</I> DH5α or E. Cloni</H2>
The preparations for the transformation can preferably be done, while the USER cloning is incubating.
<B>Materials</B><UL><LI><I>E. coli</I> DH5α (Bioline) or E. cloni-5α (Lucigen) competent cells
</LI><LI>USER reaction
</LI></UL><B>Procedure:</B><OL><LI>LB plates are taken out of the refrigerator and marked. Remember to use LB plates with the right antibiotics.
</LI><LI>50 µL competent <I>E. coli</I> DH5α cells per USER reaction is taken from the -80C° freezer and place on ice. Additionally, 1,5 ml tubes are placed on ice.
</LI><LI>5 µL USER reaction mix is added to the 50 µL competent <I>E. coli</I> DH5α cells. Mix well by pipetting.
</LI><LI>The cells are placed on ice for 30 min.
</LI><LI>The hot plate is set on 42 °C, and each transformation is heat shocked for 1 min. The cells are put directly on ice for 2 min afterwards.
</LI><LI>200 µL recovery medium is added pr. cell tube.
</LI><LI>The cells are placed on hot plate on 300 rpm for 30 minutes.
</LI><LI>The hot plate is increased to 1000 rpm, and the cells are placed hereon for additional 1 hour.
</LI></OL><H2>Control of Lux cassette</H2><B>Procedure:</B><OL><LI>One colony is chosen from the LB-plate, and transferred to a small plating tube with 2.5 ml LB-medium with antibiotics.
</LI><LI>Incubate over night at 37°C in the shaking incubator.
</LI><LI>The over night culture is transferred to an Erlenmeyer flask with 100 ml LB-medium and regularly checked with OD
</LI><LI>When OD600 has reached 0.4-0.6 100 µl IPTG is added
</LI><LI>The flask is taking into a dark room to see the immediate effect. 
</LI></OL><UL><LI><A href="https://2012.igem.org/Team:Copenhagen/Protocols#Cultivation" class="h2">Cultivation</A></LI><LI><A href="https://2012.igem.org/Team:Copenhagen/Protocols#Plate" class="h2">Plating on LB plates</A></LI><LI><A href="https://2012.igem.org/Team:Copenhagen/Protocols#Primer" class="h2">Primer design for point mutation in Lux Casette</A></LI><LI><A href="https://2012.igem.org/Team:Copenhagen/Protocols#PCR-Backbone" class="h2">PCR reaction for amplification of backbone pSB1C3</A></LI><LI><A href="https://2012.igem.org/Team:Copenhagen/Protocols#Colony-PCR" class="h2">Colony PCR</A></LI><LI><A href="https://2012.igem.org/Team:Copenhagen/Protocols#Protocol-PCR1" class="h2">PCR reactions for individual genes</A></LI><LI><A href="https://2012.igem.org/Team:Copenhagen/Protocols#USER" class="h2">USER Cloning</A></LI><LI><A href="https://2012.igem.org/Team:Copenhagen/Protocols#Transformation" class="h2">Transformation</A></LI><LI><A href="https://2012.igem.org/Team:Copenhagen/Protocols#Lux-Control" class="h2">Control of Lux cassette</A></LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Copenhagen/Protocols">http://2012.igem.org/Team:Copenhagen/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Copenhagen/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Copenhagen/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Copenhagen/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Copenhagen/Protocols&amp;oldid=220584" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>