<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania-OG_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania-OG/Measurement</SPAN></H1></DIV></DIV></DIV><TITLE>Collaborations</TITLE><DIV id="" class="clearfix"><HEADER id="header" class="page-section full-header"><DIV id="header-wrap" style="padding-left:1%"><DIV class="clearfix"><NAV id="primary-menu"><UL><LI id="homeheader"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG">Home</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Project</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design_Regulatory">Design for Regulatory Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Nucleotide_Design">Substrate Design Prospects</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Entrepreneurship">Entrepreneurship</A></LI></UL><LI id="softwareh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Gan_Introduction">Generative Adversarial Network – GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ProteinGAN">Protein GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ReactionGAN">Reaction GAN</A></LI><LI id="partsh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-basic">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-composite">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-collection">Collection</A></LI><LI id="notebookh"><A href="/pages/notebook/index.html">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/catseqguide">Using the CAT-Seq</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/labnotebook">Lab Journal</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Safety">Safety</A></LI><LI id="humanh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Collaborations">Collaborations</A></LI><LI id="peopleh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">People</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">Team</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Sponsors">Sponsors</A></LI></NAV></DIV></DIV></HEADER><SECTION id="slider" class="slider-element slider-parallax full-screen force-full-screen with-header swiper_wrapper page-section clearfix"><DIV class="slider-parallax-inner"><DIV class="swiper-container swiper-parent"><DIV class="swiper-wrapper"><DIV class="swiper-slide" style="background-image: url('https://static.igem.org/mediawiki/2018/a/ac/T--Vilnius-Lithuania-OG--measurement.jpg');"><DIV class="container clearfix"><DIV class="slider-caption slider-caption-center"><H2 data-caption-animate="fadeInUp" style="color:aliceblue">Measurement</H2></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><DIV id="page-menu"><DIV id="page-menu-wrap"><DIV class="container clearfix"><DIV class="menu-title">Explore <SPAN>MEASUREMENT</SPAN></DIV><NAV class="one-page-menu"><UL><LI><A href="#" style="font-size: 80%" data-href="#section-introduction">Introduction</A></LI><LI><A href="#" style="font-size: 80%" data-href="#section-comic">Catalytic Biomolecule activity</A></LI><LI><A href="#" style="font-size: 80%" data-href="#section-survey">Regulatory part activity</A></LI><LI><A href="#" style="font-size: 80%" data-href="#section-cross"> Cross-interaction</A></LI><LI><A href="#" style="font-size: 80%" data-href="#section-acuracy">Accuracy validation</A></LI></UL></NAV></DIV></DIV></DIV><SECTION style="overflow:hidden; position:relative; background-color: #FFF;"><DIV class="content-wrap"><DIV class="single-post nobottommargin"><DIV class="container clearfix"><SECTION id="section-introduction" class="page-section"><DIV class="tekstodydis"><DIV class="fancy-title title-border-color"><H2>Introduction</H2></DIV><P>Precise and high-throughput biological part characterization is at the very core of Catalytic Activity Sequencing (CAT-Seq) system. It is the next generation<STRONG> activity measurement system </STRONG>capable of simultaneous recording of large libraries of catalytic biomolecules or regulatory parts. The system output is simple, yet extremely useful - <STRONG>every unique biomolecule sequence receives a parameter</STRONG>, which denotes the <STRONG>activity </STRONG>of that biological part.</P><P>CAT-Seq is best explained with an example - we strongly suggest to<STRONG> read the design section first</STRONG>!</P><SECTION id="section-comic" class="page-section"><SECTION id="section-discovery" class="page-section"><DIV class="fancy-title title-border-color"><H2>Catalytic Biomolecule activity measurement</H2></DIV><P>CAT-Seq provides a way to simultaneously measure the activities of billions of catalytic biomolecules in a single experiment by recording the activity information for each unique biomolecule into its coding DNA sequence. This is achieved by quickly encapsulating single biomolecule templates in separate droplets together with special Substrate Nucleotides. The substrate nucleotides have a substrate attached to them, and they act as a target for the catalytic biomolecule. The catalytic biomolecule can then turn the Substrate Nucleotide into the Product Nucleotide. During the DNA amplification in each droplet, this activity is recorded, and by quantifying the number of product nucleotides we can assess the activity of each individual biomolecule simultaneously.</P><SECTION id="section-survey" class="page-section"><DIV class="fancy-title title-border-color"><H2>Regulatory part activity measurement</H2></DIV><P>CAT-Seq can also be effortlessly adjusted to measure the activities and cross-interactions of numerous transcriptional or translational regulatory parts instead. The amount of product nucleotide depends not only on the catalytic biomolecule activity, but also on the amount of the biomolecule.</P><P>If we would take a library, wherein all of the constructs have the exact catalytic biomolecule sequence, yet unique regulatory sequences, we would have different amounts of catalytic biomolecule depending on the regulatory sequence in each droplet. Moreover, the amount of product nucleotide produced would also be connected to the protein amount in the droplet. Finally, if we would have a stronger regulatory sequence that allows greater catalytic biomolecule expression, we would see that by an increased amount of Product Nucleotide.</P><P>Therefore, the regulatory sequence strength can be indirectly, but precisely assessed using CAT-seq.</P><SECTION id="section-cross" class="page-section"><DIV class="fancy-title title-border-color"><H2>Regulatory part cross-interaction measurement</H2></DIV><P>Finally, CAT-Seq can be used to assess the cross-interactions of different regulatory sequence combinations.</P><P>Let’s take Toehold Switches as an example. They consist of two parts - the <STRONG>Toehold Switch</STRONG> and <STRONG>activating RNA</STRONG>. The Toehold Switch forms an RNA loop on the RBS of the controlled coding sequence and in turn<STRONG> inhibits the translation</STRONG>. The expressed activating RNA can bind to the Toehold Switch RNA sequence and open RBS to the translation machinery. Testing a big number of combinations toehold switch pair combination while searching for orthogonal switches is an extremely time and resource consuming task. That is because each combination must be assessed separately.</P><P>Using CAT-Seq, the cross-interactions of large libraries of Toehold Switches and RNA activators can be recorded in a single experiment. In this example, two libraries would have to be prepared - the toehold switch and RNA activator. Then, both of these libraries are ligated next to the same catalytic biomolecule. What we then have is a new library that consists of fragments which have random toehold switch and RNA activator combinations and the same esterase.</P><P>Depending on how activating RNA affects toehold switch in each droplet, the Product Nucleotide amount will vary. After the amplification and sequencing in each fragment, we would see three main things - the specific toehold switch sequence, the specific activating RNA sequence and their performance or activity score in the form of Product Nucleotides.</P><SECTION id="section-acuracy" class="page-section"><DIV class="fancy-title title-border-color"><H2>Validating the measurement accuracy</H2></DIV><P>While ultra high-throughput is extremely important, the <STRONG>precision </STRONG>and <STRONG>accuracy </STRONG>of the system is just as critical. In order to assess if our system is measuring activities properly in high-throughput, we have first made low-throughput measurements <STRONG>using standard</STRONG> and <STRONG>well-established methods</STRONG>. Then, we have compared them with the results acquired using CAT-Seq.</P><H3 style="margin-top: 9%;">Esterase mutant activity determination</H3><P>In silico designed mutant library was subjected to catalytic activity sequencing. The DNA embedded with catalytic activity information (in a form of incorporated reference to catalytically converted nucleotide ratio) was extracted and sequenced with the nanopore. By applying data preparation and analysis pipeline, the mean methylation scores arising from reference nucleotides for each barcoded mutant DNA template were filtered and extracted. The collected data was normalized over Wild Type enzyme version and K227R mutant (it had the lowest activity).</P><DIV style="margin-top: -5%; text-align: justify; font-size: 80%; margin-bottom: 3%;"><P><STRONG>Figure 1.</STRONG> Comparison of In bulk and CAT-Seq measured esterase mutant relative activity. In silico generated Esterase mutant library was subjected to catalytic activity sequencing. The mean methylation scores for each barcoded mutant DNA template were filtered and extracted. The collected data was normalized over Wild Type CAT seq Esterase and K227R mutant (lowest activity). The relative activity, extracted from the mean methylation score of each mutant read is compared to in data gathered in standard sized reactions (in bulk).</P></DIV><P>The relative methylation score (reference nucleotide count) of each mutant read corresponds to the activity of the enzyme it encodes. The higher the activity of the expressed enzyme, the lower methylation score is assigned, due to the catalytic conversion of substrate nucleotides. The comparison of the results, gathered with CAT-Seq catalytic activity sequencing approach and standard sized reactions (Figure 1) conclude the viability of CAT-Seq as a biomolecule activity to reading method. <STRONG><EM>The activity reading, extracted from the DNA sequence correlates with the kinetic measurement data perfectly</EM></STRONG>. The activity of each Esterase mutant is measured accurately and is assigned to the corresponding DNA sequence. These results prove that CAT-Seq approach enables to screening the activity of millions of enzyme variant sequences and accurately assigns the phenotype of each variant to the genotype it arises from.</P><H3 style="margin-top: 9%;">RBS strength determination</H3><P>The constructed Ribosome Binding site library (<A href="http://parts.igem.org/Part:BBa_B0030"><STRONG>BBa_B0030</STRONG></A>, <A href="http://parts.igem.org/Part:BBa_B0032"><STRONG>BBa_B0032</STRONG></A>, <A href="http://parts.igem.org/wiki/index.php/Part:BBa_B0034"><STRONG>BBa_B0034</STRONG></A>, <A href="http://parts.igem.org/Part:BBa_K2621038"><STRONG>BBa_K2621038</STRONG></A> with a downstream CAT-Seq esterase gene <A href="http://parts.igem.org/Part:BBa_K2621000"><STRONG>BBa_K2621000</STRONG></A>) was subjected to catalytic activity sequencing method and DNA, embedded with catalytic activity information was prepared. Then mean methylation scores (reference nucleotide count) for each barcoded DNA template, housing different RBS were filtered and extracted. The activity collected data was normalized to <A href="http://parts.igem.org/wiki/index.php/Part:BBa_B0034"><STRONG>BBa_B0034</STRONG></A> data and is shown in Fig. 2.</P><DIV style="margin-top: -5%; text-align: justify; font-size: 80%; margin-bottom: 3%"><P><STRONG>Figure 2.</STRONG> Comparison of in bulk and CAT-Seq measured ribosome binding site relative strength. The catalytic activity of esterase gene, regulated by a library of ribosome binding sites was measured using cell-free expression system in bulk or CAT-Seq approach and compared side by side. The mean methylation scores for each barcoded mutant DNA template were filtered and extracted. The collected data was normalized <A href="http://parts.igem.org/wiki/index.php/Part:BBa_B0034"><STRONG>BBa_B0034</STRONG></A> corresponding to mean strength of 1.</P></DIV><P>It’s known that stronger ribosome binding sites increase the yield of translated proteins and in therefore increase the number of catalytically converted substrate nucleotides. This increase of nucleotides is inversely proportional to the assigned mean methylation score of the sequenced DNA template. Based on this fact, the activity results can be extracted from mean methylation scores (reference nucleotide count) and correspond to ribosome binding site strength. The catalytic activity sequencing results were compared to earlier, in standard sized reactions measured RBS strength results. The comparison once again concludes the viability of CAT-Seq approach. <STRONG>The ribosome binding site strength, extracted from the DNA sequence in a form of reference nucleotide count correlates with measurements made with accurate standard assays</STRONG>. These results display the validity of CAT-Seq as a method for screening the strength of regulatory sequences and its ability to assign accurate phenotype to genotype linkage.</P><H3 style="margin-top: 9%;">Toehold cross interaction assessment</H3><P>Toehold regulatory sequence library constituted of the different toehold and trigger pairs was constructed subjected to catalytic activity sequencing method. The DNA embedded with catalytic activity information was sequenced with Nanopore. The mean methylation scores (reference nucleotide count in the sequence) for each barcoded DNA template, housing different regulatory sequence were filtered and extracted.</P><DIV style="margin-top: -5%; text-align: justify; font-size: 80%; margin-bottom: 3%"><P><STRONG>Figure 3</STRONG>. The evaluation of Toehold-Trigger riboregulatory sequence orthogonality using CAT-Seq. The catalytic activity of esterase genes, regulated by different Toehold switches were measured using CAT-Seq. The mean methylation scores for each barcoded regulatory construct DNA template was filtered and essigned. Low methylation scores correspond to actively expressed protein and are only assigned when both Toehold and trigger sequences from the same group are present verifying the already measured orthogonality of regulatory parts.</P></DIV><P>The Figure 3 displays the mean methylation (reference nucleotide count) scores assigned to each barcoded toehold-trigger construct read. Based on the results, low methylation score is only assigned when both Toehold and trigger sequences from the same group are present. This means that esterase was expressed and catalyzed the conversion of substrate nucleotides. These results correlate perfectly to the standard (not in droplet) measurement results. Based on this fact, it can be concluded that CAT-Seq activity sequencing method can be utilized as a precise and accurate way to screen and assign the activity and orthogonality of regulatory sequences in a high throughput manner.</P><SECTION id="section-survey" class="page-section"><DIV class="fancy-title title-border-color"><H2>Using CAT-Seq</H2></DIV><P>We truly believe that Catalytic Activity Sequencing will be a <STRONG>valuable </STRONG>and frequently used tool by <STRONG>future </STRONG>iGEM teams, Scientists and Engineers. That is why we have created a detailed <STRONG>“Using Cat-Seq”</STRONG> protocol guide that allows users of range of background and skill level to start exploring the <STRONG>vast space</STRONG> of sequence and activity relationships right away!</P></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></DIV></SECTION></DIV></DIV></DIV></SECTION></DIV></BODY></HTML>