<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania-OG_ProteinGAN skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania-OG/ProteinGAN</SPAN></H1></DIV></DIV></DIV><TITLE>Collaborations</TITLE><DIV id="" class="clearfix"><HEADER id="header" class="page-section full-header"><DIV id="header-wrap" style="padding-left:1%"><DIV class="clearfix"><NAV id="primary-menu"><UL><LI id="homeheader"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG">Home</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Project</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design_Regulatory">Design for Regulatory Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Nucleotide_Design">Substrate Design Prospects</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Entrepreneurship">Entrepreneurship</A></LI></UL><LI id="softwareh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Gan_Introduction">Generative Adversarial Network – GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ProteinGAN">Protein GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ReactionGAN">Reaction GAN</A></LI><LI id="partsh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-basic">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-composite">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-collection">Collection</A></LI><LI id="notebookh"><A href="/pages/notebook/index.html">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/catseqguide">Using the CAT-Seq</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/labnotebook">Lab Journal</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Safety">Safety</A></LI><LI id="humanh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Collaborations">Collaborations</A></LI><LI id="peopleh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">People</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">Team</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Sponsors">Sponsors</A></LI></NAV></DIV></DIV></HEADER><SECTION id="slider" class="slider-element slider-parallax full-screen force-full-screen with-header swiper_wrapper page-section clearfix"><DIV class="slider-parallax-inner"><DIV class="swiper-container swiper-parent"><DIV class="swiper-wrapper"><DIV class="swiper-slide" style="background-image: url('https://static.igem.org/mediawiki/2018/d/d7/T--Vilnius-Lithuania-OG--protgan.jpg');"><DIV class="container clearfix"><DIV class="slider-caption slider-caption-center"><H2 data-caption-animate="fadeInUp" style="color:aliceblue; ">Protein GAN</H2></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><DIV id="page-menu"><DIV id="page-menu-wrap"><DIV class="container clearfix"><DIV class="menu-title">Explore <SPAN>ProteinGAN</SPAN></DIV><NAV class="one-page-menu"><UL><LI><A href="#" data-href="#section-introduction">Introduction</A></LI><LI><A href="#" data-href="#section-data">Data gathering</A></LI><LI><A href="#" data-href="#section-building">Building ProteinGAN</A></LI><LI><A href="#" data-href="#section-results">Results</A></LI><LI><A href="#" data-href="#section-deeper">Deeper look at ProteinGAN</A></LI></UL></NAV></DIV></DIV></DIV><SECTION style="overflow:hidden; position:relative; background-color: #FFF;"><DIV class="content-wrap"><DIV class="single-post nobottommargin"><DIV class="container clearfix"><SECTION id="section-introduction" class="page-section"><DIV class="tekstodydis"><DIV class="fancy-title title-border-color"><H2>Introducton</H2></DIV><P>After seeing multiple successful applications of GAN (generative adversarial networks) in numerous of fields, we have decided to apply them to the <STRONG>field of synthetic biology</STRONG> for the <STRONG>creation of novel biological parts</STRONG> with useful functions. More specifically, we were interest in world’s cleanest and environmentally friendly catalyzers - <STRONG>enzymes</STRONG>. For many important reactions used in research or industry <STRONG>we don’t have the appropriate enzymes</STRONG> to catalyze them, and have not other option but to use chemical catalyzers.</P><P>Thus, we have decided to build the <STRONG>world’s first Protein sequence Generative Adversarial Network</STRONG> (ProteinGAN) which would be capable to learn “what makes protein a protein”. We have started by acquiring and standardizing large number of protein sequences from public databases, which all had a specific class attributed to them.</P><P>After in-depth literature analysis and a large number of in-silico prototypes we have built the appropriate GAN architecture for protein work. Finally - we have trained the neural networks with specific classes of enzymes. We have hoped they would learn how to generate the class of enzymes they were trained for, yet also deliver unique protein sequences for that class.</P><P>All important technical details, architectural choices and detailed explanation of how ProteinGAN works can be found at the <A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ProteinGAN#section-deeper">end of the page.</A></P><P>In addition to that, we also provided <STRONG><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ReactionGAN#section-run">a short guide on how to build and train your own ProteinGAN </A></STRONG>!</P><SECTION id="section-data" class="page-section"><SECTION id="section-data" class="page-section"><DIV class="fancy-title title-border-color"><H2>Gathering the Protein Sequence data</H2></DIV><P>As an initial sequence database we used the full (reviewed + unreviewed) UniProtKB database. We selected the sequences which had a full EC number associated and filtered out the sequences which had been known to form heteromeric complexes. The resulting dataset was relatively sparse for our purpose, therefore we decided to upsample using the NCBI RefSeq non-redundant protein database (nrDB). We used MMseqs2 (Steinegger et al. 2017) to cluster our selected enzyme database using 90% sequence identity threshold and over 80% sequence coverage, which resulted in a total of 77,686 representative sequences. This step was taken to reduce the redundancy of sequence database which was used as a query database for the MMseqs2 search workflow. To upsample, we searched our clustered database against nrDB using these parameters: 30% sequence identity threshold, 80% sequence coverage, 7.5 sensitivity level; and afterwards extracted 98% identity representatives. The resulting database consisted of 409900 sequences, which were used to assemble training and test sets for ProteinGAN.</P><SECTION id="section-building" class="page-section"><DIV class="fancy-title title-border-color"><H2>Building the ProteinGAN</H2></DIV><P>ProteinGAN - generative adversarial network architecture that we developed and optimized to generate <EM>de novo</EM> proteins. Once trained, it outputs a desirable amount of proteins with the confidence level of sequences belonging to a functional class it was trained for.</P><P>A Sub-neural network called <STRONG>discriminator </STRONG>learns how proteins look like by trying to distinguish between<STRONG> real</STRONG> and <STRONG>fake (generated) sequences</STRONG>. At the same time, another Sub-neural network called <STRONG>generator </STRONG>tries to fool it, by providing realistically looking <STRONG>generated protein sequences</STRONG>. Yet, the generator <STRONG>never </STRONG>actually <STRONG>sees </STRONG>any of the protein sequences.</P><P>Over time, generator becomes so good that generated examples satisfy rules by which discriminator currently judges proteins. In order to, keep up with generator, discriminator needs to discover <STRONG>more sophisticated patterns</STRONG> in <STRONG>real data</STRONG> to able to distinguish real from fake. In turn, the generator will adjust generation of sequences to match those patterns in order to stay in competition with the discriminator.</P><P>By the end of this competition, the generator will have learned how to generate proteins that share same patterns as real ones.</P><P style="font-size:80%; margin-left: 3%; margin-right: 3%;">Figure 1: High-level design of ProteinGAN. Generator learns to create realistic sequence embeddings (digital sequence representations) from random noise, which then are compared to real sequence embedding by the Discriminator. The learning process of such a neural network is a never-ending competition between the Generator and Discriminator.</P><SECTION id="section-results" class="page-section"><DIV class="fancy-title title-border-color"><H2>ProteinGAN results</H2></DIV><P>Using the architecture described below, we have successfully trained a generative network to create enzyme sequences belonging to the DNA helicase class. After the training we asked the ProteinGAN to generate us a DNA helicase enzyme and that is what we got:</P><P>Seq1:</P><P style="word-wrap: break-word;">MIGRILGGLLEKNPPHLIVDCHGVGYEVDVPQSTFYNLPQTGEKVVLLTQQIVREDAHLLYGFGTVEERSTFRELLKISGIGARQALAVLSGMSVPELAQAVTLQDAGRLTRVPGIGKKTAERLLLELKGKLGADLGDLAGAASYSDHAIDILNALLALGYNEKEALAAIKNVPAGTGVSEGIKLALKALSKG</P><P>Remember, our ProteinGAN has never seen a protein sequence. This is the network’s <STRONG>interpretation </STRONG>of how an DNA helicase <STRONG>should look</STRONG>. Right of the bat, we saw that it successfully learnt<STRONG> that a protein sequence should start with Methionine </STRONG>- and that alone was outstanding, as it learned some rules of nature already!</P><P>The networks have also captured that enzymes can have different sequence lengths - as GAN generates all of the outputs of the same length by default, some of the generated examples had a variable number of 0's attached after sequence of amino acids. For example, these two sequences have different lengths:</P><DIV style="margin-left:3%; word-wrap: break-word; text-align: left"><P>1.<STRONG> Amino acid sequence of length 172 </STRONG>MHIIGINPGQAIVGYGIIEFQRSRVFGISYGFITTEHETANEERLRVIYNRLNEIVRHYDPDAIAVEELFFNKNVTTAYNVGHKAGVELLAALTQGLNIGEYTPMQLKQAVCGYGNANKKNVQEMVRLILGLEGIPKPDDAADAVALAICHQNTFLSELTKNIGKVRYCIGG00000000000000000000000000000000000000000000000000000000000000000000000000000000000</P><P>2.<STRONG> Amino acid sequence of length 192</STRONG> MIGRNHGVLIEKNPPRQLVDCHPVGYELDVPMSTFYTLPSLGERRSNLTQIMVREDAQILYGFATAAERQAFRMLVKISGIGARTALAVLSGMSVNEIANAVSKQDARLTRVPGVGKKTANRLLIELKGKLGADLGQAAAGNAPSDASVDILNALLALGYSEREPAEAIKLVPAGTGVSDGIKLALKTLSKG000000000000000000000000000000000000000000000000000000000000000</P></DIV><P>Furthermore, by simply inspecting the sequence, we saw that it did not contain any strange patterns, like multiple repeats of the same amino-acid.</P><DIV class="toggle toggle-bg" style="margin-top: 10%;"><DIV class="togglet"><STRONG>Some more examples of what has the ProteinGAN created</STRONG></DIV><DIV class="togglec" style="margin-left:0%; word-wrap: break-word"><P>1. MIGRITGILLEKNPPHVELDVSGVGYEIEVPMSTFYNPPLTGERVSLQTHINVNEDAHLLYGFGTAQERQAQRQLTKVGGVGPRTALGVLSGMSVNELAEAIANQEANRLIHVPGIGKKTAERILLELRGKLGADVGVTPHVVADTSADLLNALLALGYNDREAGAALKSIPQSVGVSDGIKIALKALAR</P><P>2. MLTHLKGKLVEKNPTEVVIDCQGVGYFVNLSLHTFSKIGDEESIQLFTHFNIKEDAYTLYGFNEKSERKLGRNLVGVSGIGQSTARTQLSSLTPDQIMQAIQSGDVAAVNAIKGIGARTAQRVIIDLKDKLIRLYSNDNVSTQKNNTHKEEALSALEVLGYVRKEAEKVITAIIRDQPEASVEDLLKTALKQL</P><P>3. MRLIGIEPGQANTGRSIIEVQGSRFKIVEYSAITTEANTDIPTRLLHIYNEIECLNEKYHPEEQAYEELFFNKNQKTALSVGQRRIVLLLAAKQRGIAIIERTPKQVLQAVLGYGKAQKVNVQDMVKALLSLKEIPKPDDIADALAVALYHARGSSSDSMNKG</P><P>4. MITNLRGHLVEKEPTDMVIDCHGVGYLVNISLHTFGLIPNSENVKLFTHLHVREDSHTLFGFVEATARAMFRLLIGVSGIGASTAQTMLSSIDPKILPNAIASDDVGTIQSIKGIGIKTAQRVIIDLKDKILKTYSLDEISFSNNNTNKDEALSALEVLGFVRKQSEKVVEKILKENPDASVESIIKLALKNL</P><P>5. MIRHLNGKLVLKNPNHVVIHAGGVGYHINISLSTFSQLPDMDHLKLYTYLQVKEDAHTLYGFVEKNERELFRQLLSVSGVGASLARTMLSSLDPAELMGAIAQGDVRTIQSVKGIGTKTAQRIILDLKDKVVKLNGNDDNSIGSQTNNKEDKLSALEVLGFNKKKAEKAVDKILKEEGEATVETLIKMALKRI</P><P>6. MVAHLQGKVVEKTPTQVVIESSGIGYNVNLSLHTYSLIPSMDNIKLYTKLQVREDSQTLYGFVEKSERELFHKLISVSGIGCSLARTMLSSLDPKQIIDAIASGDVGTIQSRKGIGTKTAQRVIIDLKDKVLKVYEIDEVSKQDNNTRRDDALSALEVLGFTKKTSEKVGDKIVKKNPEATVEALIKTALKHLG</P></DIV></DIV><P>We have then compared generated sequence with sequences from dataset which we used for training - <STRONG>the generated sequence was not in the dataset</STRONG>!</P><P>We used BLAST (Basic Local Alignment Search Tool) to compare generated protein with the rest of known proteins. Results were astonishing: generated sequence was not only similar to DNA Helicase enzymes, but also had identity score which ranged from 89% to 86% (Table 1).</P><TABLE class="table"><TBODY><TR><TD><P><STRONG>Blast hit number</STRONG></P></TD><TD><P><STRONG>Identity</STRONG></P></TD><TD><P><STRONG>e.value</STRONG></P></TD><TD><P><STRONG>Enzyme Class</STRONG></P></TD></TR><TR><TD><P>1</P></TD><TD><P>89%</P></TD><TD><P>2e-117</P></TD><TD><P>DNA Helicase</P></TD></TR><TR><TD><P>2</P></TD><TD><P>89%</P></TD><TD><P>2e-117</P></TD><TD><P>DNA Helicase</P></TD></TR><TR><TD><P>3</P></TD><TD><P>89%</P></TD><TD><P>1e-116</P></TD><TD><P>DNA Helicase</P></TD></TR></TBODY></TABLE><P style="text-align: center; margin-top: -5%; font-size: 90%;">Table 2: Top three hits from BLAST for Seq1 (see above)</P><DIV class="toggle toggle-bg" style="margin-top: 10%;"><DIV class="togglet"><STRONG>Sequences of the three first blast hits</STRONG></DIV><DIV class="togglec" style="margin-left:0%; word-wrap: break-word; text-align: left"><P>Blast Hit 1: MIGRIAGVLLEKNPPHLLVDCNGVGYEVDVPMSTFYNLPSTGERVVLLTQMIVREDAHLLYGFGTAEERSTFRELLKISGIGARMALAVLSGMSVHELAQTVTMQDAARLTRVPGIGKKTAERLLLELKGKIGADLGAMAGAASASDHASDILNALLALGYSEKEALAAVKNVPAGTGVSEGIKLALKALSKG</P><P>Blast Hit 2: MIGRIAGVLLEKNPPHLLIDCNGVGYEVDVPMSTFYNLPSTGERVVLLTQMIVREDAHLLYGFGTAEERSTFRELLKISGIGARMALAVLSGMSVHELAQTVTMQDAARLTRVPGIGKKTAERLLLELKGKIGADLGAMAGAASASDHASDILNALLALGYSEKEALAAVKNVPAGTGVSEGIKLALKALSKG</P><P>Blast Hit 3: MIGRIAGVLLEKNPPHLLVDCNGVGYEVDVPMSTFYNLPSTGERVVLLTQMIVREDAHLLYGFGTAEERSTFRELLKITGIGARMALAVLSGMSVHELAQTVTMQDAARLTRVPGIGKKTAERLLLELKGKIGADLGALAGAASASDHASDILNALIALGYSEKEALAAIKNVPAGTGVSEGIKLALKALSKG</P></DIV></DIV><P>This is how generated sequence looks like compared with the first BLAST hits:</P><P style="font-size:80%; margin-left: 3%; margin-right: 3%;">Figure 3. Generated sequence and top three BLAST hits alignment. Matching amino acids highlighted in red, synonymous - in yellow. Figure generated using ESPript 3.0 (Robert and Gouet, 2014).</P><P>This amount of similarity gives us confidence that generated protein could perform same function as proteins from the training set. </P><DIV class="fancy-title title-border-color"><H2>Conclusions</H2></DIV><P>We have successfully built a novel Generative Adversarial Network for novel <STRONG>Protein generation</STRONG> and then showed that our ProteinGAN is capable of <STRONG>learning the rules</STRONG> and patterns of protein sequences without ever seeing one. To our surprise - the ProteinGAN generated sequences that were not only <STRONG>unique compared to the dataset </STRONG>the networks were trained with, but at the same time being similar to the members of the same enzyme class from public databases.</P><P>Therefore, we can confidently say that <STRONG>ProteinGAN </STRONG>is a fully functional neural network that has the proven potential of <STRONG>creating novel and useful synthetic biological parts</STRONG>.</P><SECTION id="section-deeper" class="page-section"><DIV class="fancy-title title-border-color"><H2>Deeper look at ProteinGAN</H2></DIV><H3>Data preprocessing</H3><P>In order to use collected data we had to transformed it in a way that is suitable for neural networks. That meant that letters representing amino acids had to be translated to numeric representation. As a result, each amino acid has been encoded into embedding vector of 8 physicochemical attributes (Kawashima, Ogata and Kanehisa, 1998). We opted for using embedding vector due to its memory efficiency in comparison to one-hot encoding technique. In one-hot encoding, the amino acid would be turned in the vector of length 20 with 19 zeros and single 1, whereas with embedding vector, we can use vector which is 2.5 times smaller (See Figure 4). What is more, neural network also received experimentally gathered properties about each individual amino acid.
			</P><P>One-hot encoding:</P><TABLE class="table"><TBODY><TR><TD><P>A</P></TD><TD><P>1</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD><TD><P>0</P></TD></TR></TBODY></TABLE><P>Embedding:</P><P style="font-size:80%; margin-left: 3%; margin-right: 3%; text-align:center">Figure 4</P><SECTION id="section-concl" class="page-section"><DIV class="fancy-title title-border-color"><H2>Training process of ProteinGAN</H2></DIV><P>Generator receives a set of numbers from random gaussian distribution called noise which is passed through pre-defined number of filters that contain trainable parameters. These filters are arranged in blocks called residual blocks. After each block the length of the input increases until it matches the length of a real protein (analogy to increasing resolution in image generation) (Figure 2). The last layer ensures that the range of values and shape matches original data.
			</P><P style="font-size:80%; margin-left: auto; margin-right: auto;display:block; text-align:center ">Figure 5. Visualization of protein generation</P><P>Discriminator receives generated and real sequences and passes them through the network containing same size filters stacked in reversed order compared to generator (in order to not overpower one over another). At the end, discriminator arrives to a single number for each sequence that corresponds to the confidence level of discriminator that a particular resembles natural protein sequences.
				</P><P>Using the scores from discriminator, each part of the GAN is evaluated using loss function. </P><P><EM>Hinge loss</EM> where <EM>D - function of discriminator that returns a single number for each sequence, G - a function of generator that takes noise as an input and returns sequence of amino acids. N - the number of examples looked in one step. </EM></P><P>Intuitively, this means that discriminator will do well if it assigns low scores to generated examples and high to real ones. On the other hand, generator is penalized if generated examples are scored low. These scores drive the direction of how parameters are being tweaked to improve the scores of discriminator and generator separately.</P><P>As generator and discriminator have opposite goals, they compete against each other in so called mini-max game. Forward pass and backpropagation is repeated until the generated sequences are not improving further or is hardly distinguishable from real ones.</P><P>To train ProteinGAN we used Google cloud instance with NVIDIA Tesla P100 GPU (16GB). Training took approximately 60h for each class which contained at least 20000 unique proteins. Entire network contained ~4M trainable parameters. For optimization Adam optimizer has been chosen with learning rates of 0,0001 for discriminator and generator (β<SUB>1</SUB> = 0, β<SUB>2</SUB> = 0,9). Both discriminator and generator were trained the same amount of steps.</P><DIV class="fancy-title title-border-color"><H2>ProteinGAN Architecture details</H2></DIV><P>ProteinGAN networks are comprised of residual blocks that showed the best performance in ImageNet challenges. They are also a widely adopted in various implementation of GANs. Each block in discriminator contains 3 convolution layers with filter size of 3x3. The generator residual blocks consist of two deconvolution layers (transposed convolution) and one convolution block with the same filter size of 3x3 (see Figure 6).</P><P style="font-size:80%; margin-left: auto; margin-right: auto;display:block; text-align:center ">Figure 6. Residual Blocks</P><P>We have experimented with 3 up-sampling techniques: Nearest Neighbor, Transposed Convolutions (deconvolutions)  and Sub-Pixel Convolutions (Shi et al., 2016). After conducting numerous experiments, we have chosen Transposed Convolutions as the best suited for dealing with proteins.  </P><P style="font-size:80%; margin-left: auto; margin-right: auto;display:block; text-align:center ">Figure 7: Architecture of generator and discriminator</P><DIV class="fancy-title title-border-color"><H2>The addition of Dilation into ProteinGAN</H2></DIV><P>Convolution filters are very good at detecting local features, but it has limitations when it comes to long distance relationship. As a result, a lot of deep learning algorithms uses RNN (Recurrent Neural Network) approach when it comes to sequences. However, it has been showed that convolution filters with dilation outperforms RNN (Bai, Kolter and Koltun, 2018). The idea of dilation is to increase receptive field without increasing the number of parameters by introducing gaps into convolution kernels (Figure 8). Dilation rate was applied to one convolution filter in each residual block. The dilation rate was increased by 2 in each consecutive block. In this way by the last layer of the network, filters had large enough receptive field to learn long-distance relationships. </P><P style="font-size:80%; margin-left: auto; margin-right: auto;display:block; text-align:center ">Figure 8: (a) dilation rate equals 1 (standard convolution), (b) dilation rate equals 2, (c) dilation rate equals 4. Red dots are where 3x3 filter is applied, colored squares show receptive field given previous convolutions. Source: (Yu and Koltun, 2016)
									</P><DIV class="fancy-title title-border-color"><H2>Self-Attention</H2></DIV><P>Different areas of protein have different responsibilities in overall protein behaviour. In order to for network to capture this, self-attention mechanism (Zhang et al., 2018) has been implemented. To put it simply, it is a number of layers that highlights different areas of importance across the entire sequence. There are 64 such filters in <STRONG>ProteinGAN </STRONG>and <STRONG>ReactionGAN </STRONG>implementations.</P><P style="font-size:80%; margin-left: auto; margin-right: auto;display:block; text-align:center ">Figure 9: Source: (Zhang et al., 2018)
									</P><DIV class="fancy-title title-border-color"><H2>Spectrum normalization</H2></DIV><P>One of the biggest issue then implementing GAN is the stability of the training. In practise a lot of GAN implementations suffer from diminishing (very small) or exploding (enormously big) gradients. In both cases GANs are not capable to learn patterns in data successfully. To mitigate this issue,  spectrum normalization technique  (Miyato et al., 2018) was used. It is regularization method to constrain the Lipschitz constant of the weights. It has been shown that it works successfully even for a large and complex datasets such as ImageNet (Brock, Donahue and Simonyan, 2018).</P><DIV class="fancy-title title-border-color"><H2>Mode collapse</H2></DIV><P>Given original GAN formulation, there is nothing to prevent generator from generating a single, very realistic example to fool the discriminator. Such scenario is known as mode collapse. It happens when generator learns to ignore the input (random numbers). Logically, it is an efficient way for generator to start generating examples that could fool discriminator. However, it is not desirable behaviour and it eventually cripples the training as discriminator can easily remember generated examples. While working with proteins, we observed that this issue is even more severe in comparison to images. In scientific community, a lot of different approaches were proposed to address the mode collapse issue: Unrolled GAN (Metz et al., 2017), Dual Discriminator (Nguyen et al., 2017), Mini batch Discriminator (Salimans et al., 2016) to name a few. We preferred Mini Batch Discriminator approach due to its simplicity and minimal overhead. Mini Batch Discriminator works an extra layer in the network that computes the standard deviation across the batch of examples (batch contains only real, or only fake sequences). If the batch contains a small variety of examples standard deviation will be low and discriminator will be able to use this information to lower the final score for each example in the batch. ProteinGAN follows the approach proposed by authors of Progressively growing GAN (Karras et al., 2018).
				</P><P><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ReactionGAN"> Click here to find what we did next </A></P><DIV class="toggle toggle-bg" style="margin-top: 10%;"><DIV class="togglet"><STRONG>References</STRONG></DIV><DIV class="togglec"><P>Bai, S., Kolter, J. and Koltun, V. (2018). <EM>An Empirical Evaluation of Generic Convolutional and Recurrent Networks for Sequence Modeling</EM>. [online] Arxiv.org. Available at:</P><P>Brock, A., Donahue, J. and Simonyan, K. (2018). <EM>Large Scale GAN Training for High Fidelity Natural Image Synthesis</EM>. [online] Arxiv.org. Available at: https://arxiv.org/abs/1809.11096 [Accessed 14 Oct. 2018].</P><P>Karras, T., Aila, T., Laine, S. and Lehtinen, J. (2018). <EM>Progressive Growing of GANs for Improved Quality, Stability, and Variation</EM>. [online] Arxiv.org. Available at: https://arxiv.org/abs/1710.10196 [Accessed 14 Oct. 2018].</P><P>Kawashima, S., Ogata, H., &amp; Kanehisa, M. (1999). AAindex: Amino acid index database. <EM>Nucleic Acids Research</EM>, <EM>27</EM>(1), 368–369. http://doi.org/10.1093/nar/27.1.368</P><P>https://arxiv.org/abs/1803.01271 [Accessed 13 Oct. 2018].</P><P>Metz, L., Poole, B., Pfau, D. and Sohl-Dickstein, J. (2017). <EM>Unrolled Generative Adversarial Networks</EM>. [online] Arxiv.org. Available at: https://arxiv.org/abs/1611.02163 [Accessed 14 Oct. 2018].</P><P>Miyato, T., Kataoka, T., Koyama, M. and Yoshida, Y. (2018). <EM>Spectral Normalization for Generative Adversarial Networks</EM>. [online] Arxiv.org. Available at: https://arxiv.org/abs/1802.05957 [Accessed 14 Oct. 2018].</P><P>Nguyen, T., Le, T., Vu, H. and Phung, D. (2017). <EM>Dual Discriminator Generative Adversarial Nets</EM>. [online] Arxiv.org. Available at: https://arxiv.org/abs/1709.03831 [Accessed 14 Oct. 2018].</P><P>Robert, X., &amp; Gouet, P. (2014). Deciphering key features in protein structures with the new ENDscript server. <EM>Nucleic Acids Research</EM>, <EM>42</EM>(W1), 320–324. http://doi.org/10.1093/nar/gku316</P><P>Salimans, T., Goodfellow, I., Zaremba, W., Cheung, V., Radford, A. and Chen, X. (2016). <EM>Improved Techniques for Training GANs</EM>. [online] Arxiv.org. Available at: https://arxiv.org/abs/1606.03498 [Accessed 14 Oct. 2018].</P><P>Yu, F. and Koltun, V. (2016). <EM>Multi-Scale Context Aggregation by Dilated Convolutions</EM>. [online] Arxiv.org. Available at: https://arxiv.org/abs/1511.07122 [Accessed 14 Oct. 2018].</P><P>Zhang, H., Goodfellow, I., Metaxas, D. and Odena, A. (2018). <EM>Self-Attention Generative Adversarial Networks</EM>. [online] Arxiv.org. Available at: https://arxiv.org/abs/1805.08318 [Accessed 14 Oct. 2018].</P></DIV></DIV></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></DIV></SECTION></DIV></DIV></DIV></SECTION></DIV></BODY></HTML>