<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania-OG_MutationModel skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania-OG/MutationModel</SPAN></H1></DIV></DIV></DIV><TITLE>Collaborations</TITLE><DIV id="" class="clearfix"><HEADER id="header" class="page-section full-header"><DIV id="header-wrap" style="padding-left:1%"><DIV class="clearfix"><NAV id="primary-menu"><UL><LI id="homeheader"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG">Home</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Project</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design_Regulatory">Design for Regulatory Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Nucleotide_Design">Substrate Design Prospects</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Entrepreneurship">Entrepreneurship</A></LI></UL><LI id="softwareh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Gan_Introduction">Generative Adversarial Network – GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ProteinGAN">Protein GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ReactionGAN">Reaction GAN</A></LI><LI id="partsh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-basic">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-composite">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-collection">Collection</A></LI><LI id="notebookh"><A href="/pages/notebook/index.html">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/catseqguide">Using the CAT-Seq</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/labnotebook">Lab Journal</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Safety">Safety</A></LI><LI id="humanh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Collaborations">Collaborations</A></LI><LI id="peopleh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">People</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">Team</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Sponsors">Sponsors</A></LI></NAV></DIV></DIV></HEADER><SECTION id="slider" class="slider-element slider-parallax full-screen force-full-screen with-header swiper_wrapper page-section clearfix"><DIV class="slider-parallax-inner"><DIV class="swiper-container swiper-parent"><DIV class="swiper-wrapper"><DIV class="swiper-slide" style="background-image: url('https://static.igem.org/mediawiki/2018/c/c2/T--Vilnius-Lithuania-OG--ModelMutationTop2.jpeg');"><DIV class="container clearfix"><DIV class="slider-caption slider-caption-center"><H2 data-caption-animate="fadeInUp" style="color:aliceblue; font-size: 150%;">In-Silico determination of Esterase mutants</H2></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><DIV id="page-menu"><DIV id="page-menu-wrap"><DIV class="container clearfix"><DIV class="menu-title">Explore <SPAN>MODEL</SPAN></DIV><NAV class="one-page-menu"><UL><LI><A href="#" data-href="#section-introduction">Introduction</A></LI><LI><A href="#" data-href="#section-homology">Homology Model</A></LI><LI><A href="#" data-href="#section-exp">Experminental validation</A></LI><LI><A href="#" data-href="#section-concl">Conclusions</A></LI></UL></NAV></DIV></DIV></DIV><SECTION style="overflow:hidden; position:relative; background-color: #FFF;"><DIV class="content-wrap"><DIV class="single-post nobottommargin"><DIV class="container clearfix"><SECTION id="section-introduction" class="page-section"><DIV class="tekstodydis"><DIV class="fancy-title title-border-color"><H2>Introducton</H2></DIV><P>As we began to build the first prototype of CAT-Seq system, we have quickly realised, that the best way to measure system’s performance is to have a well working <STRONG>Substrate Nucleotide</STRONG> and a catalytic biomolecule pair that predictably catalyzes chemical conversion from a Substrate to a <STRONG>Product Nucleotide</STRONG>. Initially, having a predetermined system output allowed us to track CAT-Seq performance and its early issues instead of performance of substrates and catalytic biomolecules.</P><P>For this reason we have chosen a Substrate Nucleotide (<STRONG>N4-benzoyl-2'-deoxycytidine triphosphate</STRONG>) and then screened a library of esterases using standard methods to find an esterase that would successfully convert the Substrate Nucleotide into Product Nucleotide (<STRONG>2'-deoxycytidine triphosphate</STRONG>).</P><P>After finding the efficient substrate and biomolecule pair, we wanted to see how CAT-Seq system discriminates between different activities of the catalytic biomolecules. For that reason wanted to mutate our already well-working Esterase to create its mutants, which would have slightly <STRONG>different activities</STRONG> then the original Esterase. Then, we would measure the mutants using standard low-throughput methods to determine their relative. Lastly, after using the mutant library in CAT-Seq, we would compare our high-throughput system results with the results of standard methods. In this way, we can assess the accuracy and precision of CAT-Seq system.</P><P>Yet, the addition of the mutations is not an easy task, as even a single amino acid change can inactivate the catalytic biomolecule completely. Therefore, we have decided to determine the mutants of the Esterase by in-silico computational modelling.</P><SECTION id="section-homology" class="page-section"><SECTION id="section-discovery" class="page-section"><DIV class="fancy-title title-border-color"><H2>CAT-Seq Esterase homology model</H2></DIV><H3 style="margin-top: 9%;">In-silico description of CAT-Seq Esterase </H3><P>First, we wanted to determine the type of Esterase enzyme we have. For that reason, we have performed an initial sequence and literature analysis. We have recognized that our enzyme contains a 29-amino acid tannase “signature” sequence starting with “GCSTGGR”, which was described by Banerjee et al:</P><P style="margin-top: -3%; text-align: center; font-size: 80%;">Figure 1. Sequence alignment to tannase family HMM consensus (PFAM: PF07519)</P><P>Accordingly, the CAT-Seq Esterase is most similar to the currently known <A href="#myModal1" data-lightbox="inline" class="bnt-more">Tannases</A> and <A href="#myModal2" data-lightbox="inline" class="bnt-more">Feruloyl</A> Esterases.</P><DIV class="modal1 mfp-hide" id="myModal1"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;" style="overflow: hidden;"><DIV class="left" style="padding: 5%;"><P style="text-align: center; margin-top:3%;" class="tekstodydis">Appendix pic. 1</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV><DIV class="modal1 mfp-hide" id="myModal2"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;" style="overflow: hidden;"><DIV class="left" style="padding: 5%;"><P style="text-align: center; margin-top:3%;" class="tekstodydis">Appendix pic. 2</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV><DIV class="modal1 mfp-hide" id="myModal3"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;" style="overflow: hidden;"><DIV class="left" style="padding: 5%;"><P style="text-align: center; margin-top:3%;" class="tekstodydis">Appendix pic. 3</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV><DIV class="modal1 mfp-hide" id="myModal4"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;" style="overflow: hidden;"><DIV class="left" style="padding: 5%;"><P style="text-align: center; margin-top:3%;" class="tekstodydis">Appendix pic. 3</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV><H3 style="margin-top: 9%;">Identifying the Active Site</H3><P>The template search for the homology model using HHpred revealed two potential hits - two f<STRONG>eruloyl esterases</STRONG> (PDB ID: <A href="#myModal3" data-lightbox="inline" class="bnt-more">6G21</A> and <A href="#myModal4" data-lightbox="inline" class="bnt-more">3WMT</A>) which had <STRONG>43% and 40% sequence similarity</STRONG> to our Esterase. Yet, only one of the templates contained a corresponding article released by the time of our homology model generation, however as the templates are homologous, a more similar template, 6G21, was selected.</P><P>After choosing the template, initial sequence alignments were constructed using MODELLER. As the quality of homology models highly depend on the quality of their respective template-target alignments, we have performed minor adjustments to the template sequence by correcting the accidental gaps in secondary structures in order to ensure proper mapping of the target sequence to the template. Additionally, the calcium-coordinating motif cysteines C263-C280 and N-terminal C512-C534 were manually aligned to the template to ensure the disulfide bridge formation in the initial CAT-Seq Esterase homology model.</P><P style="margin-bottom: 3%;">As described in literature, 3WMT template is a serine hydrolase with an Active Site catalytic triad consisting of Serine, Aspartate and Histidine. Since all three sequences align well, we could also identify the catalytic triads for the template 6G21 and our CAT-Seq Esterase, consisting of <STRONG>Ser195</STRONG>, <STRONG>Asp429</STRONG>, <STRONG>His467.</STRONG> The sequence similarities and the positions Catalytic Triad amino acids are highlighted in the illustration below (Fig. 2).</P><P style="margin-top: -3%; margin-left: 3%; margin-right: 3%; font-size: 80%;">Figure 2. Amino acid sequence alignment of CAT-Seq Esterase and two homologous templates. The secondary structures of template 6G21 are shown above. Numbers below the sequences indicate cysteine pairs forming disulfide bridges. Boxes below the sequences indicate residues overviewed in 3WMT template: catalytic triad - red, substrate-binding residues - magenta, calcium-coordinating residues - cyan. </P><H3 style="margin-top: 9%;">Selecting the mutations</H3><P>Our esterase and our chosen template alignment was refined manually after each homology model generation using MODELLER toolkit. The final structural model of CAT-Seq Esterase was visualised with ICM Browser (Fig. 3).</P><P style="margin-top: -3%; margin-left: 3%; margin-right: 3%; font-size: 80%;">Figure 3. Homology model of CAT-Seq Esterase.
			a)<B> Superposition</B> of the generated homology <B>model</B> (light blue) and the <B>template</B> used (light yellow, PDB ID: 6G21).
			b) View of the catalytic triad in the model. Residue coloring: catalytic triad - red, mutation positions - orange, calcium-coordinating residues - cyan. Catalytic residues as well as the mutations are numbered in boxes. Hydrogen bonds in the catalytic triad represented as black dashed lines. </P><P>The residues chosen for mutation were scattered around the catalytic triad of the esterase and involved mainly polar and aromatic side chains. In literature, these side chains are assumed to form bonds stabilizing the binding of the substrate.</P><P>When selecting the mutations, we have aimed for the polar and aromatic residues in the binding pocket, and mutated them into other <STRONG>synonymous residues</STRONG> to avoid total inactivation of the enzyme, yet ensuring a change in catalytic activity.</P><P>These are the following mutations we have chosen:</P><P>N107D; G194A; W224Y; K227R; K230R; Y368F; E430D; M431L; R509K; ΔP348-H356</P><SECTION id="section-exp" class="page-section"><SECTION id="section-goals" class="page-section"><DIV class="fancy-title title-border-color"><H2 style="margin-top: 9%;">Experimental validation of the CAT-Seq Esterase Mutants</H2></DIV><P>After carefully selecting the mutations in-silico, we have synthesized and measured the Activity of those esterase mutants in the laboratory. The <STRONG>mutations have successfully altered the activity </STRONG>of the original Esterase. Also, there were no cases of complete inactivation of enzyme catalytic activity.</P><P>To read more about the Esterase Mutation experimental results, <A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Results">click here!</A></P><SECTION id="section-concl" class="page-section"><SECTION id="section-goals" class="page-section"><DIV class="fancy-title title-border-color"><H2>Conclusions</H2></DIV><P>Here we have described a <STRONG>rational </STRONG>in-silico homology modelling procedure. Our goal was to determine the specific mutations of an Esterase enzyme, which would slightly adjust the wild type Esterase enzyme’s catalytic activity. First, we have determined the possible locations of the <STRONG>binding pocket</STRONG> and <STRONG>active center</STRONG> by performing sequence alignments. Secondly, we assessed the possible <STRONG>structure of the esterase</STRONG> by comparing it to the structures of closest neighbors. Then, we have added synonymous mutations in the binding pocket in order to slightly change the catalytic activity. Finally, we have <STRONG>experimentally tested</STRONG> the esterase mutants, and showed that they indeed had <STRONG>adjusted catalytic activities</STRONG>.</P><DIV class="toggle toggle-bg"><DIV class="togglet"><DIV style="margin-left:5%;"><STRONG>References</STRONG></DIV></DIV><DIV class="togglec" id="reflistas"><OL><LI>Finn, R. D., Coggill, P., Eberhardt, R. Y., Eddy, S. R., Mistry, J., Mitchell, A. L., … Bateman, A. (2016). The Pfam protein families database: Towards a more sustainable future. <EM>Nucleic Acids Research</EM>, <EM>44</EM>(D1), D279–D285. http://doi.org/10.1093/nar/gkv1344</LI><LI>Rodríguez-Durán, L. V., Valdivia-Urdiales, B., Contreras-Esquivel, J. C., Rodríguez-Herrera, R., &amp; Aguilar, C. N. (2011). Novel strategies for upstream and downstream processing of tannin acyl hydrolase. <EM>Enzyme Research</EM>, <EM>2011</EM>(1). http://doi.org/10.4061/2011/823619</LI><LI>Banerjee, A., Jana, A., Pati, B. R., Mondal, K. C., &amp; Das Mohapatra, P. K. (2012). Characterization of tannase protein sequences of bacteria and fungi: An in silico study. <EM>Protein Journal</EM>, <EM>31</EM>(4), 306–327. http://doi.org/10.1007/s10930-012-9405-x</LI><LI>Söding, J. (2005). Protein homology detection by HMM-HMM comparison. <EM>Bioinformatics</EM>, <EM>21</EM>(7), 951–960. http://doi.org/10.1093/bioinformatics/bti125</LI><LI>Moroz, O. V., Blagova, E., Davies, G. J., &amp; Wilson, K. S. (s.a.). Crystal structure of an esterase from Aspergillus oryzae. <EM>TO BE PUBLISHED</EM>. http://doi.org/10.2210/PDB6G21/PDB</LI><LI>Suzuki, K., Hori, A., Kawamoto, K., Thangudu, R. R., Ishida, T., Igarashi, K., … Fushinobu, S. (2014). Crystal structure of a feruloyl esterase belonging to the tannase family: A disulfide bond near a catalytic triad. <EM>Proteins: Structure, Function and Bioinformatics</EM>, <EM>82</EM>(10), 2857–2867. http://doi.org/10.1002/prot.24649</LI><LI>Eswar, N., Webb, B., Marti-Renom, M. A., Madhusudhan, M. S., Eramian, D., Shen, M., … Sali, A. (2006). Comparative protein structure modeling using Modeller. <EM>Current protocols in bioinformatics</EM> (T. Chapter 5). http://doi.org/10.1002/0471250953.bi0506s15.</LI></OL></DIV></DIV></SECTION></SECTION></SECTION></SECTION></SECTION></SECTION></DIV></SECTION></DIV></DIV></DIV></SECTION></DIV></BODY></HTML>