Team:Vilnius-Lithuania-OG/MutationModel
Collaborations
Home
Project
Overview
Design
Design for Regulatory Parts
Modelling
Results
Proof of Concept
Measurement
Improve
Substrate Design Prospects
Entrepreneurship
Software
Motivation
Generative Adversarial Network – GAN
Protein GAN
Reaction GAN
Parts
Overview
Basic Parts
Composite Parts
Collection
Notebook
Using the CAT-Seq
Lab Journal
Interlab
Safety
Human practices
Human Practices
Education and Public Engagement
Collaborations
People
Team
Attributions
Sponsors
In-Silico determination of Esterase mutants
Explore MODEL
Introduction
Homology Model
Experminental validation
Conclusions
Introducton
As we began to build the first prototype of CAT-Seq system, we have quickly realised, that the best way to measure system’s performance is to have a well working Substrate Nucleotide and a catalytic biomolecule pair that predictably catalyzes chemical conversion from a Substrate to a Product Nucleotide. Initially, having a predetermined system output allowed us to track CAT-Seq performance and its early issues instead of performance of substrates and catalytic biomolecules.
For this reason we have chosen a Substrate Nucleotide (N4-benzoyl-2'-deoxycytidine triphosphate) and then screened a library of esterases using standard methods to find an esterase that would successfully convert the Substrate Nucleotide into Product Nucleotide (2'-deoxycytidine triphosphate).
After finding the efficient substrate and biomolecule pair, we wanted to see how CAT-Seq system discriminates between different activities of the catalytic biomolecules. For that reason wanted to mutate our already well-working Esterase to create its mutants, which would have slightly different activities then the original Esterase. Then, we would measure the mutants using standard low-throughput methods to determine their relative. Lastly, after using the mutant library in CAT-Seq, we would compare our high-throughput system results with the results of standard methods. In this way, we can assess the accuracy and precision of CAT-Seq system.
Yet, the addition of the mutations is not an easy task, as even a single amino acid change can inactivate the catalytic biomolecule completely. Therefore, we have decided to determine the mutants of the Esterase by in-silico computational modelling.
CAT-Seq Esterase homology model
In-silico description of CAT-Seq Esterase
First, we wanted to determine the type of Esterase enzyme we have. For that reason, we have performed an initial sequence and literature analysis. We have recognized that our enzyme contains a 29-amino acid tannase “signature” sequence starting with “GCSTGGR”, which was described by Banerjee et al:
Figure 1. Sequence alignment to tannase family HMM consensus (PFAM: PF07519)
Accordingly, the CAT-Seq Esterase is most similar to the currently known Tannases and Feruloyl Esterases.
Appendix pic. 1
Close
Appendix pic. 2
Close
Appendix pic. 3
Close
Appendix pic. 3
Close
Identifying the Active Site
The template search for the homology model using HHpred revealed two potential hits - two feruloyl esterases (PDB ID: 6G21 and 3WMT ) which had 43% and 40% sequence similarity to our Esterase. Yet, only one of the templates contained a corresponding article released by the time of our homology model generation, however as the templates are homologous, a more similar template, 6G21, was selected.
After choosing the template, initial sequence alignments were constructed using MODELLER. As the quality of homology models highly depend on the quality of their respective template-target alignments, we have performed minor adjustments to the template sequence by correcting the accidental gaps in secondary structures in order to ensure proper mapping of the target sequence to the template. Additionally, the calcium-coordinating motif cysteines C263-C280 and N-terminal C512-C534 were manually aligned to the template to ensure the disulfide bridge formation in the initial CAT-Seq Esterase homology model.
As described in literature, 3WMT template is a serine hydrolase with an Active Site catalytic triad consisting of Serine, Aspartate and Histidine. Since all three sequences align well, we could also identify the catalytic triads for the template 6G21 and our CAT-Seq Esterase, consisting of Ser195, Asp429, His467. The sequence similarities and the positions Catalytic Triad amino acids are highlighted in the illustration below (Fig. 2).
Figure 2. Amino acid sequence alignment of CAT-Seq Esterase and two homologous templates. The secondary structures of template 6G21 are shown above. Numbers below the sequences indicate cysteine pairs forming disulfide bridges. Boxes below the sequences indicate residues overviewed in 3WMT template: catalytic triad - red, substrate-binding residues - magenta, calcium-coordinating residues - cyan.
Selecting the mutations
Our esterase and our chosen template alignment was refined manually after each homology model generation using MODELLER toolkit. The final structural model of CAT-Seq Esterase was visualised with ICM Browser (Fig. 3).
Figure 3. Homology model of CAT-Seq Esterase.
a) Superposition of the generated homology model (light blue) and the template used (light yellow, PDB ID: 6G21).
b) View of the catalytic triad in the model. Residue coloring: catalytic triad - red, mutation positions - orange, calcium-coordinating residues - cyan. Catalytic residues as well as the mutations are numbered in boxes. Hydrogen bonds in the catalytic triad represented as black dashed lines.
The residues chosen for mutation were scattered around the catalytic triad of the esterase and involved mainly polar and aromatic side chains. In literature, these side chains are assumed to form bonds stabilizing the binding of the substrate.
When selecting the mutations, we have aimed for the polar and aromatic residues in the binding pocket, and mutated them into other synonymous residues to avoid total inactivation of the enzyme, yet ensuring a change in catalytic activity.
These are the following mutations we have chosen:
N107D; G194A; W224Y; K227R; K230R; Y368F; E430D; M431L; R509K; ΔP348-H356
Experimental validation of the CAT-Seq Esterase Mutants
After carefully selecting the mutations in-silico, we have synthesized and measured the Activity of those esterase mutants in the laboratory. The mutations have successfully altered the activity of the original Esterase. Also, there were no cases of complete inactivation of enzyme catalytic activity.
To read more about the Esterase Mutation experimental results, click here!
Conclusions
Here we have described a rational in-silico homology modelling procedure. Our goal was to determine the specific mutations of an Esterase enzyme, which would slightly adjust the wild type Esterase enzyme’s catalytic activity. First, we have determined the possible locations of the binding pocket and active center by performing sequence alignments. Secondly, we assessed the possible structure of the esterase by comparing it to the structures of closest neighbors. Then, we have added synonymous mutations in the binding pocket in order to slightly change the catalytic activity. Finally, we have experimentally tested the esterase mutants, and showed that they indeed had adjusted catalytic activities.
References
Finn, R. D., Coggill, P., Eberhardt, R. Y., Eddy, S. R., Mistry, J., Mitchell, A. L., … Bateman, A. (2016). The Pfam protein families database: Towards a more sustainable future. Nucleic Acids Research, 44(D1), D279–D285. http://doi.org/10.1093/nar/gkv1344
Rodríguez-Durán, L. V., Valdivia-Urdiales, B., Contreras-Esquivel, J. C., Rodríguez-Herrera, R., & Aguilar, C. N. (2011). Novel strategies for upstream and downstream processing of tannin acyl hydrolase. Enzyme Research, 2011(1). http://doi.org/10.4061/2011/823619
Banerjee, A., Jana, A., Pati, B. R., Mondal, K. C., & Das Mohapatra, P. K. (2012). Characterization of tannase protein sequences of bacteria and fungi: An in silico study. Protein Journal, 31(4), 306–327. http://doi.org/10.1007/s10930-012-9405-x
Söding, J. (2005). Protein homology detection by HMM-HMM comparison. Bioinformatics, 21(7), 951–960. http://doi.org/10.1093/bioinformatics/bti125
Moroz, O. V., Blagova, E., Davies, G. J., & Wilson, K. S. (s.a.). Crystal structure of an esterase from Aspergillus oryzae. TO BE PUBLISHED. http://doi.org/10.2210/PDB6G21/PDB
Suzuki, K., Hori, A., Kawamoto, K., Thangudu, R. R., Ishida, T., Igarashi, K., … Fushinobu, S. (2014). Crystal structure of a feruloyl esterase belonging to the tannase family: A disulfide bond near a catalytic triad. Proteins: Structure, Function and Bioinformatics, 82(10), 2857–2867. http://doi.org/10.1002/prot.24649
Eswar, N., Webb, B., Marti-Renom, M. A., Madhusudhan, M. S., Eramian, D., Shen, M., … Sali, A. (2006). Comparative protein structure modeling using Modeller. Current protocols in bioinformatics (T. Chapter 5). http://doi.org/10.1002/0471250953.bi0506s15.
