<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Vilnius-Lithuania-OG_Human_Practices skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Vilnius-Lithuania-OG/Human Practices</SPAN></H1></DIV></DIV></DIV><TITLE>Collaborations</TITLE><DIV id="" class="clearfix"><HEADER id="header" class="page-section full-header"><DIV id="header-wrap" style="padding-left:1%"><DIV class="clearfix"><NAV id="primary-menu"><UL><LI id="homeheader"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG">Home</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Project</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Overview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design_Regulatory">Design for Regulatory Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Demonstrate">Proof of Concept</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Nucleotide_Design">Substrate Design Prospects</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Entrepreneurship">Entrepreneurship</A></LI></UL><LI id="softwareh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Software</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Software">Motivation</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Gan_Introduction">Generative Adversarial Network – GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ProteinGAN">Protein GAN</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/ReactionGAN">Reaction GAN</A></LI><LI id="partsh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts">Overview</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-basic">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-composite">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Parts#section-collection">Collection</A></LI><LI id="notebookh"><A href="/pages/notebook/index.html">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/catseqguide">Using the CAT-Seq</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/labnotebook">Lab Journal</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Safety">Safety</A></LI><LI id="humanh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Public_Engagement">Education and Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Collaborations">Collaborations</A></LI><LI id="peopleh"><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">People</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/People">Team</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Sponsors">Sponsors</A></LI></NAV></DIV></DIV></HEADER><SECTION id="slider" class="slider-element slider-parallax full-screen force-full-screen with-header swiper_wrapper page-section clearfix"><DIV class="slider-parallax-inner"><DIV class="swiper-container swiper-parent"><DIV class="swiper-wrapper"><DIV class="swiper-slide" style="background-image: url('https://static.igem.org/mediawiki/2018/7/7c/T--Vilnius-Lithuania-OG--human.jpg');"><DIV class="container clearfix"><DIV class="slider-caption slider-caption-center"><H2 data-caption-animate="fadeInUp">Human Practices</H2><P class="d-none d-sm-block" data-caption-animate="fadeInUp" data-caption-delay="400">None of us is as smart as all of us</P></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><DIV id="page-menu"><DIV id="page-menu-wrap"><DIV class="container clearfix"><DIV class="menu-title">Explore <SPAN>HUMAN PRACTICES</SPAN></DIV><NAV class="one-page-menu"><UL><LI><A href="#" style="font-size: 90%" data-href="#section-overview">Overview</A></LI><LI><A href="#" style="font-size: 90%" data-href="#section-problem">Problem Identification</A></LI><LI><A href="#" style="font-size: 90%" data-href="#section-idea">Idea Generation</A></LI><LI><A href="#" style="font-size: 90%" data-href="#section-solution">Designing the System</A></LI><LI><A href="#" style="font-size: 90%" data-href="#section-building">Prototype Building</A></LI></UL></NAV></DIV></DIV></DIV><SECTION style="overflow:hidden; position:relative; background-color: #FFF;"><SECTION id="section-overview" class="page-section"><DIV class="content-wrap"><DIV class="container clearfix"><DIV class="single-post nobottommargin"><DIV class="entry clearfix"><SECTION id="section-introduction" class="page-section"><DIV class="entry-content notopmargin tekstodydis"><P>The building of CAT-Seq characterization system was a highly dynamic process in which our team and selected stakeholders <STRONG>engaged in a year long dialog</STRONG>, which has allowed us to precisely identify the most vital obstacles holding back the field of synthetic biology, helped us to ask the <STRONG>right questions</STRONG> which led to the idea of CAT-Seq and enabled the constant <STRONG>iteration and improvement </STRONG>of the system design throughout the year.</P><P>That is why we see the Human Practices as the <STRONG>most important part </STRONG>of our <STRONG>engineering </STRONG>process and overall project. Therefore, as You travel along the Human Practices <STRONG>timeline </STRONG>you will find different sections that correspond to the different engineering stages our project was at that time.</P><P>We strongly suggest to <STRONG>read the design section first</STRONG>, as it will help to fully appreciate the story of how the final design came to be.</P></DIV></SECTION></DIV></DIV></DIV><SECTION id="section-problem" class="page-section"><DIV class="section parallax" style="background-image: url('https://static.igem.org/mediawiki/2018/b/bf/T--Vilnius-Lithuania-OG--1copy.jpeg'); padding: 100px 0;" data-bottom-top="background-position:0px 100px;" data-top-bottom="background-position:0px -100px;"><DIV class="heading-block center nobottomborder nobottommargin"><H2>Identifying the problem</H2></DIV></DIV><DIV class="modal1 mfp-hide" id="121212"><DIV class="block divcenter" style="background-color: #FFF; max-width: 1500px;"><DIV class="center" style="padding: 50px;"><H3>A Simple Example of a Text Modal</H3><P class="nobottommargin">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nostrum delectus, tenetur obcaecati porro! Expedita nostrum tempora quia provident perspiciatis inventore, autem eaque, quod explicabo, ipsum, facilis aliquid! Sapiente, possimus quo!</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close this Modal</A></DIV></DIV></DIV><DIV id="part1"><SECTION class="timeline"><DIV class="container"><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 1</DIV><P>The first Overgraduate team from Lithuania was established, and we have set out to explore the most pressing issues synthetic biology field is currently facing.</P><P class="ktarpas">.</P></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 2</DIV><P>We have hosted a meeting and discussion with biotechnology experts from Lithuania to catalyse our exploration. </P><A href="#myModal01" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal01"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;" style="overflow: hidden;"><DIV class="left" style="padding: 50px;"><P>We have talked about the opportunities synthetic biosystems may offer in the future and then discussed on what currently are the biggest barriers that are slowing down the field of synthetic biology the most. These are the 4 things most of us have agreed on:</P><OL><OL><LI>Huge lack of well-characterized biological parts.</LI><LI>Hard to predict multiple part biological circuits.</LI><LI>The lack of knowledge about sequence to function relationship, which hinders the development of novel synthetic biological parts.</LI><LI>Biological circuit testing is complicated and extremely time consuming</LI></OL></OL></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 5</DIV><P>We have further discussed the current issues we have outlined during the last meeting.</P><A href="#myModal02" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal02"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;" style="overflow: hidden;"><DIV class="left" style="padding: 5%;"><P> In the end, the lack of well described biological parts and the insufficient knowledge about the biomolecular sequence to function relationship seemed like the 2 most pressing problems to solve right now. That is because basic parts are the core of synthetic biology, and if we cannot understand how the separate elements work or cannot engineer completely novel parts for our needs, it is, and will continue to be difficult task to create complex, yet useful systems that we dream synthetic biology can achieve.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 6</DIV><P>We have agreed to initially investigate what are the current state of the art methods for characterization of biological part sequence to function relationships.
													This was done in order for us to understand if our system should be based on already existing technology foundations or constructed fully from scratch.</P></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 7</DIV><P>As we have narrowed down the problem we wanted to solve, we have then <B>determined the stakeholders</B> that would enable us to solve these problems as efficient as possible and that could help us reflect how our designed solutions would affect the world.</P><P class="ktarpas">.</P></DIV></DIV><DIV class="tmlnint1"><H2 style="text-align: center; margin-bottom: 5%;">Stakeholders Of CAT-Seq</H2><P>1.  <B>Life Sciences Center in Vilnius</B> - one of the main collaborative research facilities in Lithuania, where academics from all over the Lithuania, and also the world, come together to solve various fundamental and engineering problems. We see LSC as an important stakeholder, as it can immensely influence our project in terms of helping us to forecast the right solutions to problems we are trying to solve.</P><P>2.  <B>Droplet Genomics</B> - a startup based in Lithuania specializing in droplet microfluidics and single cell research. People at droplet genomics (DG) are worldwide experts in high-throughput research using microtechnologies. As we want to innovate the throughput and precision of part characterization even further, there was no better place to start then at the company which performs state of the art high-throughput research. </P><P>3.  <B>Thermo Fisher Scientific</B> -  Thermo Fisher’s R&amp;D sector has a long history of working with directed evolution and protein engineering. We are extremely interested in navigating the space of sequence to function relationship, therefore we wanted to establish a line of communication with Thermofisher, in order for them to share their experience and help us to solve the arising engineering problems throughout the year.</P></DIV></DIV><SECTION id="section-idea" class="page-section"><DIV class="section parallax" style="background-image: url('https://static.igem.org/mediawiki/2018/b/bf/T--Vilnius-Lithuania-OG--1copy.jpeg'); padding: 100px 0;" data-bottom-top="background-position:0px 100px;" data-top-bottom="background-position:0px -100px;"><DIV class="heading-block center nobottomborder nobottommargin"><H2>Arriving at the Idea</H2></DIV></DIV><DIV id="part2"><SECTION class="timeline"><DIV class="container"><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 9</DIV><P>After an in-depth literature analyzes we have gained the knowledge there currently there are two main state of the art technological solutions of biological part characterization –<STRONG>microwell plates </STRONG>and <STRONG>microfluidics</STRONG> - have both positive, but more importantly considerable negative aspects in terms of acquiring high-throughput<STRONG> activity and sequence linked data </STRONG>and seems to require completely new approaches in order to remove those negative aspects.</P><A href="#myModal21" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal21"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P>First of all, the <STRONG>microtiter plate technology </STRONG>approach offers a way to connect biomolecule sequence to activity by physically separating the biomolecule samples in physically separated plate wells. As the location of each biomolecule is known, a robot can quickly measure the activities of biomolecules in each well. Yet, the limiting factor in this case is extracting DNA template encoding the biomolecule from each well and sequencing it, while not losing the information about what activity was recorded for that particular encoded biomolecule. Therefore, as this method offers a precise way to connect sequence and function, its throughput is rather low.</P><P>On the other hand <STRONG>microfluidic technology</STRONG> approaches offers rapid biomolecule internalization into small droplets, where each droplet then works as a separate microreactor. In this way the <STRONG>throughput is a lot higher than in microtiter approach</STRONG>. For example, libraries of 109 different biomolecules can be screened in a single experiment. Yet, the problem with such approach is that even though it offers a lot higher throughput and droplets can even be specifically sorted into few different categories, once a scientist wants to sequence the DNA molecules which are contained in those, for example, 109 droplets, the droplets need to be broken. As the water droplets are broken, different biomolecule encoding DNA is mixed together, and the activity <STRONG>information about each particular biomolecule is lost</STRONG>.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 11</DIV><P>We have decided to arrange a <STRONG>meeting with Karolis Leonavičius from Droplet Genomics</STRONG> who specializes in Microfluidic Technologies and single molecule research.</P><A href="#myModal22" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal22"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P>We talked about advantages and disadvantages of microfluidic technologies as a whole and then we went on to discuss the case of our interest – biomolecule activity screening and characterization. <STRONG>According to Karolis</STRONG>, single biomolecule encapsulation in nano-sized droplets is already highly optimized and efficient. There also is a lot of research describing efficient expression of biomolecules in those droplets.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="timeline-img-header"><H2>Week 12</H2></DIV><DIV class="date">February 2018</DIV><P>After the <STRONG>discussion session with Karolis</STRONG>, we have decided we want to use <STRONG>microfluidic technologies as the base for our system</STRONG>, as it seemed to be an extremely efficient starting point - the high-throughput is already there, we just need to <STRONG>enable it</STRONG>. Also, this discussion has led us to discover the <STRONG>main concept of the problem </STRONG>we wanted to solve with our project – the barrier to high-throughput sequence to <STRONG>function information acquisition</STRONG> is the loss of that information once all of the microfluidic droplets are broken.</P><P class="ktarpas">.</P></DIV></DIV></DIV><DIV id="part3"><SECTION class="timeline"><DIV class="container"><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 13</DIV><P><B>After a short while we have determined the main goal of our journey</B> – creating a method that enables the high-throughput biomolecule activity and sequence recording inside of each of the billion droplets and then quickly and accurately retrieving that information.</P></DIV></DIV></DIV><SECTION id="section-solution" class="page-section"><DIV class="section parallax" style="background-image: url('https://static.igem.org/mediawiki/2018/b/bf/T--Vilnius-Lithuania-OG--1copy.jpeg'); padding: 100px 0;" data-bottom-top="background-position:0px 100px;" data-top-bottom="background-position:0px -100px;"><DIV class="heading-block center nobottomborder nobottommargin"><H2>Designing the solution</H2></DIV></DIV><DIV id="part4"><SECTION class="timeline"><DIV class="container"><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 16</DIV><P>The first question we asked was - <STRONG>what is the object that we can write the activity information into</STRONG>? After hosting a discussion together with Vilnius University students and later analysing the current workflows of various microfluidic techniques we have quickly came to conclusion, that such object should be a thing that already contains half of the information we wanted to record - <STRONG>biomolecule DNA sequence</STRONG>. </P><P class="ktarpas">.</P></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 17</DIV><P>We have presented our <STRONG>earliest form of the idea</STRONG> to the researchers in the Life Science Center - the problems we’re currently facing in synthetic biology, the solutions we have right now and our idea of how we can solve those problems by building a new method based on droplet microfluidics.</P><A href="#myModal41" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal41"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P>We have received extremely positive <STRONG>comments about the idea</STRONG> we wanted to achieve. The comments were based on the future promise of what such system of activity recording could bring to the field of synthetic biology and multidisciplinary research. During the discussion, researchers have also <STRONG>shared some of their ideas</STRONG> on how it would be possible to record the activity into the DNA sequence. One of those we liked in particular - recording the activity into the DNA in the form of<STRONG> DNA modifications</STRONG>.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 19</DIV><P>After the discussion with the LSC researchers, we have explored literature and brainstormed the possible ways activity could be linked to DNA modification.In addition we have <STRONG>met with Justas Vaitekūnas</STRONG> that has in depth expertise with DNA modification research. Justas has told us about various of techniques which he uses to DNA molecules during his research. As we discussed the possibilities of this approach we have all came to conclusion, that even though we could modify DNA in some way, we still did not have the <STRONG>link mechanism </STRONG>between biomolecule activity and it‘s sequence.</P><P class="ktarpas">.</P></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 20</DIV><P>We have then decided to explore a different approach - instead of looking at the DNA molecule modifications, we have started to look at separate modified nucleotides. By using the recommendations from Vilnius University professors, we have met with <B>Rūta Stanislauskienė</B>.</P><A href="#myModal412" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal412"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P> She specializes in modified nucleotide molecular biology. During the meeting she has told us about her research concerning modified nucleotide and DNA polymerase interactions. One of the most amazing thing she told us is that there are modified nucleotide and DNA polymerase combinations, wherein the polymerase cannot incorporate the nucleotide because of its modification.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 23</DIV><P>After the last meeting we have done another round of <STRONG>literature analysis</STRONG> on modified nucleotides and their interactions with different DNA polymerases. It appeared that phi29 DNA polymerase, in many cases, refuses to incorporate nucleotides with modifications. We have then figured out the solution on how we can link the DNA sequence and the <STRONG>catalytic </STRONG>biomolecule activity - connecting the substrate for catalytic biomolecule to a nucleotide</P><P class="ktarpas">.</P></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 23</DIV><P>With the help of scientists from Life Sciences Center we have successfully have chosen a specific substrate and attached it to cytidine. Then, using standard screening methods we have selected an esterase that can remove said modification.</P><A href="#myModal43" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal43"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P>Audrius Laurynėnas suggested that we should first produce a single pair of well-working substrate nucleotide and enzyme pair before starting to build activity recording system. This was an amazing suggestion as it gave us a way to test the performance of the system we are building, while we were building it.</P><P>At that design stage, the performance of catalytic biomolecule was measured by the <STRONG>sequence length and number </STRONG>of amplicons. The amplicons had to be fragmented, because standard sequencing methods are not performing well with long read. Also, amplicons had to be barcoded in order to know that those amplicons came from the same droplet, as the amount of amplified DNA was crucial to us.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 26</DIV><P>In order to search for possible pitfalls in our current design we have met with <B>Greta Stonytė</B> from Microtechnologies Sector in Vilnius University.</P><A href="#myModal44" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal44"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P>We talked about our current, theoretical system system design. Having more then 5 year experience in microfluidic technologies she quickly pointed out what may be the negative design aspects in our system:</P><P>First of all, if we encapsulate only a single template into each of the droplets, the template concentration would be too low for protein expression to occur inside the droplets. This was based on Greta’s own research – it was nearly impossible to express detectable amounts of green fluorescent protein in droplets when only single DNA templates were used.  For this reason we have recognized another thing that requires innovation – protein expression from single templates. To see how we solved this problem, please <A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Design">click here</A> to go to Rolling Circle Transcription section in the design page.</P><P>Secondly, as our design incorporated phi29 elongation reaction to record the activity of information after protein expression <STRONG>Greta has noted that Phi29 mix might not be compatible </STRONG>with in vitro transcription-translation (IVTT) mix. Because of that we did early separate experiments to see how phi29 works with IVTT. You can see the results by clicking 
															<A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Results">here</A>
															. Phi29 amplification has only started working after IVTT mix was diluted at least 4 times. It was a life saving advice, because if we would not have caught this huge problem early on, we would not have enough time to solve it later on.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 27</DIV><P>After some more research we had the idea to add phi29 reaction mix to each droplet after protein synthesis reaction is finished by merging substrate droplets with another population of phi29 droplets. This has both allowed for IVTT reaction to be efficient as possible and later when droplets were merged, diluted the IVTT mix enough for phi29 to be functional.</P></DIV></DIV><DIV class="tmlnint1" style="margin-bottom: 10%; width: 50%;"><P style="margin-bottom: -6%;">Linas Mažutis from the Sector of Microtechnologies (Institute of Biotechnology, Vilnius University) pointed out that our design might not be efficient at this point, as it incorporates difficult and inefficient <STRONG>DNA fragmentation and barcoding</STRONG> steps. In turn, He suggested we should find alternatives to this approach. The developed method should have as little steps as possible to avoid data loss, as our goal is not only high-throughput but also high-quality data.</P></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 29</DIV><P>In order to avoid DNA fragmentation and barcoding we have thought of a solution that could solve this problem - Nanopore sequencing. It solves the problem by having the ability to sequence long strands of DNA and at the same time - detect modifications. That completely removes the need of fragmentation and barcoding, and in turn, eliminates at least 4 burdensome steps from the overall design.</P></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 30</DIV><P>After multiple design iterations based on literature and consultations with field experts we have decided it is time to start building different parts from our system.</P><P class="ktarpas">.</P></DIV></DIV></DIV><SECTION id="section-building" class="page-section"><DIV class="section parallax" style="background-image: url('https://static.igem.org/mediawiki/2018/b/bf/T--Vilnius-Lithuania-OG--1copy.jpeg'); padding: 100px 0;" data-bottom-top="background-position:0px 100px;" data-top-bottom="background-position:0px -100px;"><DIV class="heading-block center nobottomborder nobottommargin"><H2>Building CAT-Seq</H2></DIV></DIV><DIV id="part5"><SECTION class="timeline"><DIV class="container"><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 31</DIV><P>We have started to build our first activity recording system prototype and immediately found the <STRONG>first issue with our initial prototype</STRONG>: the usage of circular ssDNA template.</P><A href="#myModal51" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal51"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P>Firstly, it was extremely difficult to make circularized ssDNA templates, as ssDNA is highly unstable and the strand removal from dsDNA and circularization reactions were resulting in low yields even after weeks of various optimizations.</P><P>Secondly, the final <STRONG>elongated DNA concentration</STRONG> we got in each droplet after the prototype workflow <STRONG>was extremely low</STRONG>. Each time, it was sub-nanogram amounts of DNA. That was because initially in each droplet there was <STRONG>on average one template</STRONG> of circularized ssDNA. As phi29 elongates that ssDNA template during the amplification step, it still does not make enough of a difference to increase the DNA yield significantly, even when generating <STRONG>millions of droplets </STRONG>in a single experiment. Moreover, after phi29 elongates the ssDNA molecule it creates another long <STRONG>ssDNA </STRONG>molecule, therefore second strand synthesis needs to be performed, which is also far from 100 percent efficient. Considering that nanopore sequencing requires at least of 100 ng to 1000 ng of starting template, we immediately knew we had a serious prototype design flaw in our hands.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 32</DIV><P>In hopes of finding a quick and elegant solution to our problem stemming from the usage of single stranded DNA template we have met with <STRONG>Audrius Laurynėnas</STRONG> from the Institute of Biochemistry at Vilnius University.</P><A href="#myModal52" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal52"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P>We knew that Audrius has in-depth knowledge about Phi29 and similar isothermal polymerases. We have discussed about different ways that phi29 or any other <A href="javascript:;" data-toggle="tooltip" title="" data-original-title="Just as a reminder, we cannot use a polymerase that requires periodical changes in temperature, in order for droplets to stay stable">isothermal</A> polymerase could be applied to our case to solve our current issues.</P><P>The meeting was concluded with the idea to try and use the <STRONG>Multiple Displacement Amplification</STRONG> (MDA) reaction together with random hexamers instead of elongating one long single stranded DNA molecule from the circular ssDNA template. Firstly, this would both allow us to use <STRONG>double stranded circular DNA</STRONG> which would make the preparation for experiments much quicker and easier. Secondly, in each droplet the MDA reaction would yield a lot more DNA.</P><P><STRONG>Audrius </STRONG>also suggested that we should model our MDA reaction first in order to know what length distribution of product we should expect and what effect the substrate nucleotide might have on DNA amplification.That was an important aspect to consider, because while exploring the literature we have found that even though phi29 does not incorporate many modified nucleotides, when it tries to do that it is inhibited and falls of. That might result in large number of short fragments, which is not ideal when using Nanopore Sequencing. Therefore, we have written a mathematical description of Phi29 DNA amplification which takes into account Phi29 inhibition by substrate nucleotide, in order to find maximum amount of Substrate Nucleotides we can use without worrying about short sequencing reads. To see the model, please<A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Inhibition_Model"> click here</A>.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 33</DIV><P>We have redesigned our prototype by adding the MDA reaction. The negative aspect of turning to MDA was the usage of dsDNA as a template, as we did not know whether the Rolling Circle Transcription (RCT)  would work, because all of the literature describes RCT only using circular ssDNA template. Gladly, we have later demonstrated that RCT works well with double stranded circular DNA molecules.</P></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 34</DIV><P>Another issue we had with our first Prototype was the inefficiency of droplet merging. It is very important to merge all of the droplets containing the Product Nucleotides, as each droplet that is not merged is lost information about a specific mutant. At that time, our efficiency was lower then 1 merged droplet for every 20 to 30.</P></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 35</DIV><P>To solve this huge issue we have met with<STRONG> Physicist and Microfluidic Technology expert Valdemaras Milkus </STRONG>from the Institute of biotechnology.</P><A href="#myModal53" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal53"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P>We have hoped to receive insight about how could we effectively solve that problem. Valdemaras has told us about how he approached similar problems. In his case it was precise-size droplet generation. He solved this problem by modelling and suggested for us to do the same. According to Valdemaras - even though the microfluidic models are extremely difficult to write, it is a lot harder to experimentally find the right parameters when there are <B>multiple unknown variables</B>. It might be like searching for a needle in a haystack.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 37</DIV><P>Following Valdemaras advice we have derived a full mathematical description of the active fusion of droplets inside of a microfluidic chip. Using this model we have solved multiple parameters which we then tested experimentally. The predicted parameters have successfully provided perfect droplet merging conditions which we have then continued using in the CAT-Seq system experiments. You can find our mathematical description of droplet merging by clicking <A href="https://2018.igem.org/Team:Vilnius-Lithuania-OG/Merging_Model">this link</A></P></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 38</DIV><P>At our second meeting with Greta we have presented our results and discussed possible improvements upon our system.</P><A href="#myModal54" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal54"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P>Greta has pointed out that our system might suffer from immense sequencing bias, as our system was then based on discriminating different biomolecule activities depending on phi29 amplicon amounts, which in turn was connected to how many Product Nucleotides has the particular biomolecule produced. If the molecule variant was highly active we had a good amount of DNA to sequence. But, if the activity was low, there was a high chance to lose that information, for example, due to low read count.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInRight"><DIV class="date">Week 39</DIV><P>The meeting with Greta has pushed us to innovate our system even further. Soon after that we have came up with the idea of Reference Nucleotides.</P><A href="#myModal55" data-lightbox="inline" class="bnt-more">More</A></DIV><DIV class="modal1 mfp-hide" id="myModal55"><DIV class="block divcenter" style="background-color: #FFF; max-width: 800px;"><DIV class="left" style="padding: 5%;"><P>These were specially modified nucleotides which phi29 could incorporate into DNA easily. Also, the nanopore sequencer could detect those nucleotide modifications at the same time as reading the full sequences. We have added a specific amount of reference nucleotides to amplification droplets used for enriching droplets with amplification mix.</P><P>After the merging, each droplet now contained <STRONG>reference nucleotides</STRONG>, which have helped to balance the overall amplified DNA amount, even for the mutants which had extremely low activities.</P></DIV><DIV class="section center nomargin" style="padding: 30px;"><A href="#" class="button" onclick="$.magnificPopup.close();return false;">Close</A></DIV></DIV></DIV></DIV><DIV class="timeline-item"><DIV class="timeline-content timeline-card js--fadeInLeft"><DIV class="date">Week 41</DIV><P>We have presented our finalised design and results at Science Days event which was organized by our sponsor Thermofisher Scientific. After that, we had a session discussion about further development of this project after iGEM and have received offers to collaborate with bioinformatics from Life Sciences Center in order to start using the immense amount of function to sequence relationship data in Artificial Intelligence algorithms.</P><P class="ktarpas">.</P></DIV></DIV></DIV></SECTION></DIV></SECTION></SECTION></DIV></SECTION></SECTION></DIV></SECTION></DIV></SECTION></SECTION></DIV></SECTION></DIV></SECTION></SECTION></DIV></BODY></HTML>