<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_HeroinBiosensor skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh/HeroinBiosensor</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="custom-bootstrap-menu" class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-menubuilder"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse navbar-menubuilder"><UL class="nav navbar-nav navbar-right"><LI class="active"><A href="https://2015.igem.org/Team:Edinburgh">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">About</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Collaborations">Collaboration</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Sponsors">Sponsors</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/HeroinBiosensor">Heroin Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/DNPBiosensor">DNP Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/PMABiosensor">PMA Biosensor</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/CBD">Making it Stick</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Results">Limits of Detection</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Composite_Part">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Part_Collection">Part Collection</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Improved_Part">Improved Parts</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Characterisation_Part">Improved Characterisation</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Dry Lab</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Modeling">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Design">Design</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Software">Software</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Practices">Our Story</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Legality">Legality</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Outreach">Outreach</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">InterLab</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/InterLab">InterLab</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/InterLabBook">LabBook</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Project/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Notebook/HeroinPurity">Heroin Purity</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Notebook/PMADetection">PMA Detection</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Notebook/DNPDetection">DNP Detection</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/Notebook/FluidDynamics">Fluid Dynamics</A></LI><LI><A href="https://2015.igem.org/Team:Edinburgh/MedalCriteria">Accomplishments</A></LI></DIV></DIV></DIV><HEADER class="intro"><DIV class="intro-body"><DIV class="container"><DIV class="row"><DIV class="col-md-12"><H1 class="brand-heading">Heroin Biosensor</H1><DIV align="center"><A href="#cd-team"><SPAN class="arrowtext">Scroll down to read more</SPAN></A></DIV></DIV></DIV></DIV></DIV></HEADER><SECTION class="ss-style-bigtriangle"><P>

One third of all European citizens has tried an illicit drug with the number rising over the past few years <SUP>1</SUP>. Deaths caused by drug overdose are mostly related to heroin and its purity as drug users are unaware of its concentration (which can vary from 13.5<SUP>2</SUP> to 98%<SUP>3</SUP>, as well as adulterants and contaminants that are present in their dose<SUP>4</SUP>. This has been a great issue for drug users in the UK, with the rate 38.7 drug-induced deaths per million inhabitants, with an even higher rate in Scotland of 112.5 per million inhabitants<SUP>1</SUP>. A study in the United States calculated the costs associated with productivity losses due to absenteeism and opioid-induced deaths can reach up to $30,594 per each poisoning event<SUP>5</SUP>. Our thesis has been confirmed by a former drug user, to whom we talked to in Serenity Cafe, that focusing our efforts on heroin purity would be efficient and it would potentially reduce opioid-induced deaths. Therefore it is essential to reduce harm and prevent deaths by having a quick, precise and easy to use biosensor.

         </P></SECTION><SECTION class="color col-2 ss-style-halfcircle"><DIV class="column text"><P>
There are two key enzymes for a heroin biosensor which allow the quantity to be visualised, heroin esterase and morphine dehydrogenase. Heroin esterase, a serine esterase<SUP>6</SUP>, de-acetylates the 6-acetylester group of heroin to produce 6-acetylmorphine. This process occurs a second time on the C-6 group producing morphine. The morphine produced is then oxidised to produce morphinone by the second enzyme morphine dehydrogenase. The NADP+ to NADPH reduction allows a coupled assay. A previous heroin biosensor used FMN oxidoreductase and luciferase as the coupled assay, which produced light in the presence of heroin<SUP>7</SUP>. To make the biosensor produce colour to the visible eye, and not light, we incorporated Nitrotetrazolium Blue (NBT) instead of the FMN oxidoreductase and luciferase. NBT, when coupled with phenozine methosulfate (PMS), reacts with the NADPH produced to create an insoluble purple-blue formazan <SUP>8</SUP>. The intensity of the formazan indicates the heroin purity.
         </P></DIV></SECTION><SECTION class="col-2 ss-style-bigtriangle"><DIV class="column text"><P>
Heroin esterase was isolated from <I>Rhodococcus erythropolis</I> strain H1 in 1994 from the garden soil at Cambridge and is able to use heroin as its sole carbon source<SUP>9</SUP>. The gene <I>her</I> encodes this enzyme and has the ability to be expressed in  the chassis <I>Escherichia coli </I><SUP>6</SUP>. The sequence for our enzyme used the original sequence from Rathbone, <I>et al</I>., and was then codon optimised for <I>E. coli</I>. The RFC25 prefix and suffix were added along which required all illegal sites (EcoRI, SpeI, AgeI, NotI, NgoMIV and XbaI) to be removed. As this was a difficult sequence to make as a gBlock, it was ordered as a gene in an ampicillin backbone where it was then digested and ligated into the pSB1C3 backbone.
          </P></DIV></SECTION><SECTION class="color col-2 ss-style-halfcircle"><DIV class="column text"><P>
Morphine dehydrogenase, the second enzyme required for this biosensor, was isolated from <I>Pseudomonas putida</I> M10 in 1993. This enzyme is capable of oxidising morphine and codeine to morphinone and codeinone but not thebaine allowing high specificity<SUP>10</SUP>. The sequence was obtained from Willey, <I>et al</I>., for the structural gene <I>morA</I><SUP>11</SUP>. After codon optimizing the sequence for <I>E. coli</I>, adding the RFC25 prefix and suffix and getting rid of illegal restriction sites we were able to order the gBlock from IDT.
            </P></DIV></SECTION><SECTION class="ss-style-bigtriangle"><P>
The idea for the heroin biosensor is to put crude cell extract of the two enzymes fused to CBDs to allow the enzymes to be immobilised on the paper. The mixture of the two lysates with NBT and PMS freeze dried on the paper allows the production of a blue colour in the presence of heroin in a solution. The intensity of the blue colour depends on the concentration of heroin as it shows how much NADPH is being reduced. Morphine and codeine are two major contaminants of heroin<SUP>12</SUP> which will yield false positives on the biosensor as the morphine dehydrogenase can oxidise them thereby making the NADPH produced not derived from heroin. This means that there will actually be two biosensors for heroin purity. One will have heroin esterase and morphine dehydrogenase while the other will have just morphine dehydrogenase. This will only produce a blue colour if morphine and codeine are present on their own. The difference in colour intensity between these two should represent the heroin purity, this can be read and calculated by an app.

             </P><DIV align="center"><A href="https://2015.igem.org/Team:Edinburgh/Software" class="btn btn-primary btn-lg outline" role="button">Check out our app</A></DIV><DIV align="center"><A href="https://2015.igem.org/Team:Edinburgh/Part_Collection" class="btn btn-primary btn-lg outline" role="button">Parts</A></DIV></SECTION><SECTION class="color ss-style-bighalfcircle"><H2>References</H2><P><SUP>1</SUP>European Monitoring Centre for Drugs and Drug Addiction. (2010). The state of the drugs problem in Europe. Luxembourg: Office for Official Publications of the European Communities
<SUP>2</SUP>Fries, A., Anthony, R.W., Cseko, A., Gaither, C.C &amp; Schulman, E. (2008). The price and purity of illicit drugs: 1981-2007. IDA-P-4332. Virginia, USA: Institute for Defence Analyses.
<SUP>3</SUP>Poshyachinda, V., Ayudhya, A.S.N., Aramrattana, A., Kanato, M., Assanangkornchai, S. &amp; Jitpiromsri, S. (1957). Illicit substance supply and abuse in 200-2004: An approach to assess the outcome of the war on drug operation. Drug and Alcohol Review, 24(5), 461-466.
<SUP>4</SUP>Wells, C. (2014). Deaths related to drug poisoning in England and Wales, 2013. Office for National Statistics.
<SUP>5</SUP>Inocencio, T. J., Carroll, N. V., Read, E. J., &amp; Holdford, D. a. (2013). The Economic Burden of  Opioid-Related Poisoning in the United States. Pain Medicine (United States), 14, 1534–1547.  doi:10.1111/pme.12183
<SUP>6</SUP>Rathbone, D. A., Holt, P. J., Lowe, C. R., &amp; Bruce, N. C. (1997). Molecular analysis of the Rhodococcus sp. strain H1 her gene and characterization of its product, a heroin esterase, expressed in Escherichia coli. <I>Applied and environmental microbiology</I>, 63(5), 2062-2066.
<SUP>7</SUP>Rathbone, D. A., HOLT, P. J., Lowe, C. R., &amp; Bruce, N. C. (1996). The Use of a Novel Recombinant Heroin Esterase in the Development of an Illicit Drugs Biosensora. <I>Annals of the New York Academy of Sciences</I>, 799(1), 90-96.
<SUP>8</SUP>Mayer, K. M., &amp; Arnold, F. H. (2002). A colorimetric assay to quantify dehydrogenase activity in crude cell lysates. <I>Journal of biomolecular screening</I>, 7(2), 135-140.
<SUP>9</SUP>Cameron, G. W., Jordan, K. N., Holt, P. J., Baker, P. B., Lowe, C. R., &amp; Bruce, N. C. (1994). Identification of a heroin esterase in Rhodococcus sp. strain H1. <I>Applied and environmental microbiology</I>, 60(10), 3881-3883.
<SUP>10</SUP>Bruce, N. C., Wilmot, C. J., Jordan, K. N., Trebilcock, A. E., Stephens, L. D. G., &amp; Lowe, C. R. (1990). Microbial degradation of the morphine alkaloids: identification of morphinone as an intermediate in the metabolism of morphine by Pseudomonas putida M10. <I>Archives of microbiology</I>, 154(5), 465-470.
<SUP>11</SUP>Willey, D. L., Caswell, D. A., Lowe, C. R., &amp; Bruce, N. C. (1993). Nucleotide sequence and over-expression of morphine dehydrogenase, a plasmid-encoded gene from Pseudomonas putida M10. <I>Biochem. J</I>, 290, 539-544.
<SUP>12</SUP>Balayssac, S., Retailleau, E., Bertrand, G., Escot, M. P., Martino, R., Malet-Martino, M., &amp; Gilard, V. (2014). Characterization of heroin samples by 1 H NMR and 2D DOSY 1 H NMR. <I>Forensic science international</I>, 234, 29-38.


          </P></SECTION></DIV></DIV></DIV></DIV></BODY></HTML>