Team:Edinburgh/Improved Part
Toggle navigation
Home
About
Team
Attributions
Collaboration
Sponsors
Project
Overview
Heroin Biosensor
DNP Biosensor
PMA Biosensor
Making it Stick
Limits of Detection
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Improved Parts
Improved Characterisation
Dry Lab
Modeling
Design
Software
Human Practices
Our Story
Safety
Legality
Outreach
InterLab
InterLab
LabBook
Notebook
Protocols
Heroin Purity
PMA Detection
DNP Detection
Fluid Dynamics
Accomplishments
Improved Parts
Scroll down to read more
Laccase TVEL5 in RFC25 (BBa_K1615067)
Laccases are glycosylated polyphenol oxidases and have four copper ions per molecule1. This allows them to catalyse the reduction of O2 to 2H2O while oxidising an aromatic substrate2. This laccase is coded by the structural gene lcc in Trametes versicolor, a species of white rot fungus3.
To increase the functions of laccase, we took the sequence from iGEM12_Bielefeld-Germany laccase BBa_K863030 and codon optimised it for the chassis Escherichia coli. We then made it RFC25 compatible by adding the prefix and suffix and removing all illegal restriction sites. This laccase can now be used in protein fusions as there is an in-frame 6-nucleotide scar.
1Lontie, R. (1984). Copper proteins and copper enzymes (Vol. 2). CRC. 2Thurston, C. F. (1994). The structure and function of fungal laccases. Microbiology, 140(1), 19-26. 3Collins, P. J., & Dobson, A. (1997). Regulation of laccase gene transcription in Trametes versicolor. Applied and Environmental Microbiology, 63(9), 3444-3450.
mRFP (BBa_K1615089)
RFP is important in synthetic biology as a visualisation tool. While multiple versions of RFP exist in the registry, only one is RFC25 compatible. This RFP (BBa_K1351021) contains a Shine-Dalgarno sequence in the RFC25 prefix which precludes it from using the common lac expression cassette (BBa_K314103) which already contains a ribosome binding site and only produces RFC10 fusions. By adding the RFC25 prefix without the Shine-Dalgarno sequence we hope to improve the utility of this RFP.
We visualised RFP using multiple methods to show that it was folding and expressing.
This plate shows our RFP under the control of the lac expression cassette from iGEM10_Washington (BBa_K314103).
When grown in 10 mL of LB + chloramphenicol and spun down, a pink pellet can be visualised.
To ensure the fluorescence of our RFP we aliquoted 200 µL of the crude cell lysate into a 96 well plate and visualised it under blue light transilluminator. As you can see, it fluoresces consistently in all three samples.
We wanted to test to see how long it would take to fully saturate an isostandard Whatman 54 paper chad with RFP. This graph shows that the chad is saturated almost immediately.
We tested the dissociation of this part on Whatman 54 using isostandard chads. This involved incubating the chads in the crude cell lysate for 10 minutes and then washing them with PBS for various periods time to test the binding affinity of this CBD. The results show a gradual loss of fluorescence indicating the RFP leaving the paper.
Design: There is an existing RFC25 compatible RFP (BBa_K1351021) in the registry from iGEM14_LMU-Munich which was the basis for our sequence. We then added the RFC25 prefix without the Shine-Dalgarno sequence.
CBDcipA (BBa_K1615111)
Cellulose binding domains (CBDs) mediate the binding of enzymes to cellulose1. CBDs are divided into over a dozen families based on their sequence homology 2. Family III of CBDs is divided into a, b and c with CBDCipA belonging to family III a; the clostridial scaffoldin CBDs3. CBDCipA was identified in Clostridium thermocellum and is capable of binding to crystalline cellulose in a reversible manner4. CBDCipA includes endogenous linker sequences at both the N and C-terminals which help to prevent the CBD from interfering with the folding of any other protein it may be fused to.
To characterise the binding of CBDCipA to Whatman 54 we fused it to RFP (BBa_K1615089) at both the N and C terminals and looked at dissociation of the CBD over time. Design: In an effort to improve the CBDCipA existing in the registry from iGEM14_Imperial we used the sequence from BBa_K1321014 and made it RFC25 compatible by removing the illegal EcoRI site at 148.
1Ferreira, L. M., Durrant, A. J., Hall, J., Hazlewood, G. P., & Gilbert, H. J. (1990). Spatial separation of protein domains is not necessary for catalytic activity or substrate binding in a xylanase. Biochem. J, 269, 261-264.
2Tomme, P., Warren, R. A. J., Miller, R. C., Kilburn, D. G. & Gilkes, N. R. (1995). Enzymatic Degradation of Insoluble Polysaccharides, edited by J. M. Saddler & M. H. Penner, pp. 142±161. Washington, DC: American Chemical Society
3Shimon, L. J., Belaich, A., Belaich, J. P., Bayer, E. A., Lamed, R., Shoham, Y., & Frolow, F. (2000). Structure of a family IIIa scaffoldin CBD from the cellulosome of Clostridium cellulolyticum at 2.2 Å resolution. Acta Crystallographica Section D: Biological Crystallography, 56(12), 1560-1568.
4Bayer, E. A., Chanzy, H., Lamed, R., & Shoham, Y. (1998). Cellulose, cellulases and cellulosomes. Current opinion in structural biology, 8(5), 548-557.
