Team:Edinburgh/Project/Protocols
Toggle navigation
Home
About
Team
Attributions
Collaboration
Sponsors
Project
Overview
Heroin Biosensor
DNP Biosensor
PMA Biosensor
Making it Stick
Limits of Detection
Parts
Team Parts
Basic Parts
Composite Parts
Part Collection
Improved Parts
Improved Characterisation
Dry Lab
Modeling
Design
Software
Human Practices
Our Story
Safety
Legality
Outreach
InterLab
InterLab
LabBook
Notebook
Protocols
Heroin Purity
PMA Detection
DNP Detection
Fluid Dynamics
Accomplishments
Protocols
Scroll down to read more
1% Agarose Gel
Materials
1g Agarose
100ml 1X TAE buffer
5µl GelRed stain
Procedure
1. Mix the agarose with the 1X TAE buffer in a flask.
2. Heat the mixture until all the agarose is dissolved.
3. Swirl the flask under cold running water to cool the mixture.
4. Add the gel stain.
5. Pour into an assembled gel tray and let it cool.
Agarose Gel Electrophoresis
Materials
1% Agarose
1X TAE buffer
5X loading dye
DNA ladder
DNA samples
Procedure
1. Place gel tray into the electrophoresis apparatus.
2. Pour 1X TAE so that the gel is covered by buffer.
3. Prepare the samples by adding the appropriate amount of loading dye.
4. Load samples and DNA ladder into wells on the gel.
5. Run the gel at roughly 100V for around an hour
Culture
Materials
10ml Luria Broth (LB)
10µl Specific Antibiotic at 1000x (Chloramphenicol, Ampicillin or Kanamycin)
Loop (for picking colony)
Ethanol
Procedure
1. Pour 10ml of LB into a 50ml Falcon tube.
2. Pipette 10µl of antibiotic into the broth.
3. Dip loop in ethanol and flame to sterilise. Once it is cool, pick colony and transfer to a 50ml Falcon tube.
4. Incubate at 37°C overnight in a shaking incubator.
Gel Purification using QIAquick Gel Extraction Kit
Materials
Buffer QG
10µl 3M sodium acetate
Isopropanol
750µl Buffer PE
25µl Buffer EB
Procedure
All centrifuge steps are carried out at 13,000 rpm.
1. Excise the region of gel containing the DNA fragment using a scalpel.  Cut close to the DNA to minimise the gel volume.
2. Place the gel slice in a 1.5ml tube and weigh it. Record the volume of the gel.
3.  Add 300µl of Buffer QG for each 100mg of gel.
4. Incubate at 50°C for 10 minutes or until the gel has completely dissolved. Mix by vortexing the tube every 2 minutes during the incubation.
5. Once the gel is completely dissolved, the mixture should be yellow. If the mixture is orange or violet add 10 µl of 3M sodium acetate and mix until it turns yellow. Yellow colour indicates the solution is the optimum pH for DNA binding to the QIAquick membrane.
6. Add 1 gel volume of isopropanol to the solution and mix (1:1 volumes of isopropanol to gel slice).
7. Place a QIAquick spin column in a 2ml collection tube.
8. Pipette the sample onto the QIAquick column and centrifuge. Discard flow-through.
9. Place column back in same collection tube. Add 500µl of Buffer QG to the column and centrifuge for 1 minute to remove all traces of agarose.
10. Wash column by adding 750µl buffer PE. Let it stand for 2-5 min and then centrifuge for 1 minute.
11. Discard the flow-through. Centrifuge for 1 minute to remove the residual buffer PE.
12. Then place the column in a clean, labelled 1.5ml Eppendorf tube.
13. To elute the DNA, add 25µl of Buffer EB to the centre of the column membrane, let it stand for 1 minute and then centrifuge for 1 minute.
14. Using a pipette, transfer the flow-through back into the centre of the column. Let it stand for 1 minute and then centrifuge for 1 minute. The DNA will now be in the flow-through.
Glycerol Stock
Materials
500µl 50% glycerol solution
500µl cultured cells
Procedure
1. Add the 50% glycerol to a sterile Eppendorf tube.
2. Add 500µl of cells to the tube and vortex to mix.
3. Freeze at -80°C.
Heat Shock Transformation
Materials
2.5µl ligated DNA or 0.5µl purified plasmid
50µl chemically competent DH5α E.coli cells
250µl Luria Broth
Petri dish with LB agar and specific antibiotic (chloramphenicol or ampicillin)
Procedure
1. Thaw the competent cells on ice.
2. Add 50µl of cells to a pre-chilled, labelled microcentrifuge tube.
3. Add 2.5µl of DNA suspension to the tube. Pipette up and down to mix.
4. Incubate tube on ice for 30 min.
5. Heat shock in a water bath at 42°C for 60s.
6. Place tube back on ice for 3 min.
7. Add 250µl of LB to the tube.
8. Incubate the tube at 37°C for 1 hour.
9. Prepare two plates for each transformation, one plated with 200µl of cells and another plated with 100µl of cells.
10. Incubate at 37°C overnight (12-14 hours).
Miniprep using QIAprep Spin Miniprep Kit
Materials
10ml cell culture
250µl Buffer P1
250µl Buffer P2
350µl Buffer N3
750µl Buffer PE
50µl Buffer EB
Procedure
1. Centrifuge 10ml cell culture at 4,500 x g for 5 minutes and pour off the supernatant. Centrifuge at 4,500 x g for 2 minutes and remove the supernatant by pipetting.
2. Resuspend pelleted cells in 250 µl Buffer P1 and transfer the solution to a labelled Eppendorf tube.
3. Add 250 µl Buffer P2 and gently invert the tube 4–6 times to mix. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear. Do not allow the lysis reaction to proceed for more than 5 minutes.
4. Add 350 µl Buffer N3 and invert the tube immediately but gently 4–6 times. The solution should become cloudy.
5. Centrifuge for 10 min at 13,000 rpm in an Eppendorf tube. A compact white pellet will form.
6. Apply the supernatant to the QIAprep Spin Column by pipetting.
7. Centrifuge for 60s at 13,000 rpm. Discard the flow-through.
8. Wash QIAprep Spin Column by adding 750µl Buffer PE and centrifuging for 60s at 13,000 rpm.
9. Discard the flow-through, and centrifuge for an additional 60s at 13,000 rpm to remove residual wash buffer.
10. Place the QIAprep column in a clean, labelled Eppendorf tube. To double elute DNA, add 50 µl Buffer EB to the centre of each QIAprep Spin Column, let stand for 1 min, and centrifuge for 1 min at 13,000 rpm. Re-apply flow-through to the centre of the QIAprep Spin Column, let stand for 1 min and centrifuge for 1 min at 13,000 rpm.
11. Discard column. Flow-through contains the DNA.
Nanodrop
Materials
Buffer EB to blank
DNA sample
Procedure
1. Wipe the NanoDrop 2000 clean before pipetting 1µl of buffer EB and lowering the arm. Blank the NanoDrop spectrophotometer.
2. Clean the buffer EB from the pedestal and apply 1µl of sample. Measure and record the absorbance, and repeat for remainder of samples.
3. Clean the NanoDrop pedestal after use.
PCR Purification using QIAquick PCR Purification Kit
Materials
Buffer PB
10µl of 3M sodium acetate
750µl buffer PE
25µl buffer EB
Procedure
All centrifugation steps are carried out at 17,900 x g (13,000 rpm)
1. Add 5 volumes of buffer PB to 1 volume of PCR sample (e.g. 500µl PB to 100µl PCR sample) and mix.
2. The pH indicator in buffer PB will cause the mixture to turn yellow. If the mixture is orange or violet add 10µl of 3M sodium acetate and mix until it turns yellow.
3. Place a QIAquick spin column into a 2ml collection tube.
4. Apply the sample to the centre of the column and centrifuge for 60s.
5. Discard the flow-through and replace the column in the same collection tube.
6. Add 750µl of buffer PE to the column and centrifuge for 60s.
7. Discard the flow-through, replace the column in the collection tube and centrifuge for a further 60s to remove residual buffer.
8. Place the column in a clean 1.5ml Eppendorf tube.
9. Elute the DNA by adding 25µl buffer EB to the centre of the membrane in the column. Let it stand for 1 minute then centrifuge for 1 minute.
10. Using a pipette, transfer the flow-through back into the centre of the column. Let it stand for 1 minute and then centrifuge for 1 minute.
11. The purified DNA is now present in the flow-through.
Resuspending gBlocks
Materials
gBlocks dry gene fragment
100µl distilled water
Procedure
1. Centrifuge the tube containing the gBlocks gene fragment pellet at 3000 x g for 5s.
2. Add 100µl water to the tube.
3. Vortex to mix.
5. Incubate at 50°C for 20 mins.
6. Briefly vortex and centrifuge.
Sequencing
Materials
200-500ng DNA sample
0.32µl 10µM Vf2 and Vr primers
Water
Procedure
1. Prepare the DNA for sequencing by adding DNA, primers and water to a final volume of 6µl.
2. Prepare two samples for each DNA sequence, one containing the forward primer, and another separate sample containing the reverse primer.
3. Send the samples to be Sanger sequenced by Edinburgh Genomics using an ABI 3730 DNA analyser.
4. Edinburgh Genomics will send an email with the sequence data.
PCR using KOD Hot Start DNA Polymerase
Materials
5µl 10x Buffer for KOD Hot Start DNA Polymerase
3µl 25mM MgSO4
5µl 2mM dNTPs
32µl H2O
1.5µl Forward primer
1.5µl Reverse primer
1µl template DNA at 10ng/µl
1µl KOD polymerase
Procedure
1. Add all of the listed materials to a PCR tube to make a final volume of 50µl and mix well.
2. Place the tube in a thermocycler. Set the thermocycler at 95°C for 2 mins to activate the polymerase.
3. Run 35 cycles of denaturing at 95°C for 20secs, annealing at 60°C for 10secs and extending at 70°C for 60secs.
4. Place on a 4°C hold.
Sequencing
Materials
200-500ng DNA sample
0.32µl 10µM Vf2 and Vr primers
Water
Procedure
1. Prepare the DNA for sequencing by adding DNA, primers and water to a final volume of 6µl.
2. Prepare two samples for each DNA sequence, one containing the forward primer, and another separate sample containing the reverse primer.
3. Send the samples to be Sanger sequenced by Edinburgh Genomics using an ABI 3730 DNA analyser.
4. Edinburgh Genomics will send an email with the sequence data.
PCR using KOD Hot Start DNA Polymerase
Materials
5µl 10x Buffer for KOD Hot Start DNA Polymerase
3µl 25mM MgSO4
5µl 2mM dNTPs
32µl H2O
1.5µl Forward primer
1.5µl Reverse primer
1µl template DNA at 10ng/µl
1µl KOD polymerase
Procedure
1. Add all of the listed materials to a PCR tube to make a final volume of 50µl and mix well.
2. Place the tube in a thermocycler. Set the thermocycler at 95°C for 2 mins to activate the polymerase.
3. Run 35 cycles of denaturing at 95°C for 20secs, annealing at 60°C for 10secs and extending at 70°C for 60secs.
4. Place on a 4°C hold.
Normalised cell lysate
Materials
2 000 mL conical flask
500 mL of L. Broth
10 mL of starter cultures grown overnight
500 µL of antibiotic
500 µL IPTG
Procedure
1. Turn on a Bunsen burner and pour 500ml L. Broth into a conical flask. Add 500 µL of antibiotic and your starter cultures.
2. Place your mixture into a shaking incubator at 37 ℃ for 4 hours.
3. Add 500 µL IPTG to induce protein expression.
4. Place your mixture into a shaking incubator at 18 ℃ overnight.
5. Remove your flask from the incubator and spin them for 30 minutes, on 4000 rpm using 9.1000 rotor ID.
6. Remove supernatant and resuspend the pellet in 1x PBS buffer (30 mL).
7. Pour resuspended pellet into a 50 mL Falcon tube and spin it for 10 minutes at 4000 rcf, gently discard supernatant.
8. Weigh your pellet and add 10x the weight in volume of 1x PBS buffer.
9. Vortex pellet until fully resuspended.
10. Sonicate at 10 microns in 15 second intervals for about 3 minutes or until mixture is less viscous.
11. Spin your mixture at 20,000 rpm for 30 minutes.
12. Using a syringe and a needle, intake supernatant and using a 0.45 µm filter, filter 10 mL fractions into 15 mL Falcon tubes.
13. Flash freeze using liquid nitrogen and store in -80 ℃ until ready to use.
Chad Measurement
Aim
To find out the average mass of a dry chad
To find out the mass of water and buffer retained by an average chad
Materials
Standard hole punch
Whatman 54 filter paper
Tweezers
Shallow weight boat
Balance sensitive to at least 3 decimal places
Procedure
Be sure to wear gloves, keep the paper as free from contamination as possible, this include the transfer of protein from your fingertips.
1. Punch holes of equal size (i.e. no over lays) and collect the chads in a clean tube.
2. Weigh out 10 dry chads on a weighing boat and work out the average weight.
3. Put 10 chads in 5 mL of 1 x PBS and leave for 5 minutes.
4. Remove the chads from the PBS with a tweezer and leave to dry for 5 minutes.
5. Measure weight of chads saturated with PBS and work out the average.
6. Put 10 chads in 5 mL of water and leave for 5 minutes.
7. Remove the chads from water with a tweezer and leave to dry for 5 minutes
8.Weight out the 10 chads  and work out the average weight.
Calculations
1. Substract the weight of a chad soaked in water with the weight of a dry chad to find a mass of water absorbed by a chad.
2. Substract the weight of a chad soaked in 1 x PBS with the weight of a dry chad to find a mass of PBS absorbed by a chad.
To calculate liquid absorption by one chad per cm2, use formula A= Π r2 to calculate area of on chad. Divide it by the volume absorbed by a chad.
Sequencing
Materials
200-500ng DNA sample
0.32µl 10µM Vf2 and Vr primers
Water
Procedure
1. Prepare the DNA for sequencing by adding DNA, primers and water to a final volume of 6µl.
2. Prepare two samples for each DNA sequence, one containing the forward primer, and another separate sample containing the reverse primer.
3. Send the samples to be Sanger sequenced by Edinburgh Genomics using an ABI 3730 DNA analyser.
4. Edinburgh Genomics will send an email with the sequence data.
Back to top
© 2015 EdiGEM · Privacy · Terms
