<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong-CUHK_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong-CUHK/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-fixed-top" style="margin-top:0px; margin-left: -30px"><NAV class="navbar-inner"><DIV class="row-sm-height"><DIV class="col-md-10"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK"><SPAN>Home</SPAN></A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
 Project </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/InterLab">InterLab</A></LI></UL><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Model"><SPAN>Modelling</SPAN></A></LI><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Software"><SPAN>Software</SPAN></A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Protocol">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/HP/Silver">Engagement and Education</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Prototype">Prototype</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Collaborations">Collaboration</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Team</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Team">About Us </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Attributions">Attributions </A></LI></DIV></DIV></DIV></NAV></DIV><SECTION id="section01"><DIV class="column full_size"><DIV class="column full_size"><CENTER><H1>Overview of experiments</H1></CENTER><OL type="1"><P style="font-family: roboto;font-size:115%;"><LI><I>In silico</I> designed toehold switches to detect H7N9 and H5N1 viruses based on our modelling.</LI><LI>Constructed plasmids containing switches or triggers by DNA synthesis and standard cloning method.</LI><LI>Validated the toehold switches by co-expressing the switch and trigger plasmids in <I>E. coli</I> and cell free system.</LI><LI>Constructed cloning tools for toehold switch and trigger to allow convenient replacement of switch and trigger in the plasmids.</LI><LI>Improved one existing toehold switch in the Registry.</LI><LI>Characterized two reporter proteins (mRFP and amaJlime) in the Registry.
</LI></P></OL><H3><I>In silico</I> design of Influenza Toehold switches</H3><P style="font-family: roboto;font-size:115%;">
According to Green <I>et al.</I>(1), the optimal length of RNA to be detected by a toehold switch is around 30 bp. In other words, a target RNA with 1000 bp in length will give 970 possible switches. However, the performances of each possible switches are different, since the performance is governed by serval parameters in the target region, such as the minimum free energy of the RNA (For more information, please visit <A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Model"> modelling page</A>). To minimize the manpower on screening of the switches, we constructed an <A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Software"> online toehold switch design program </A>. Apart from the basic thermodynamic parameters, it also screens for rare codons, stop codons and RFC illegal sites along the sequence. In addition, the built-in BLAST function also automatically screen for nonspecific region to avoid false positive detection. Ultimately, the program can sort a list of “best” Toehold Switch sequence according to their free energy using the embedded function of <A href="https://www.tbi.univie.ac.at/RNA/">“Vienna RNA”</A> (2). The program facilitates the construction of toehold switch by providing a user-friendly interface with novel screening function.

To detect influenza A, Polymerase Basic Protein 2 (PB2) gene is used as a positive control as it is influenza A-specific. Further subtyping requires a subtype-specific RNA that can also fulfil the criteria for being a good toehold switch.  We downloaded the latest influenza gene sequences from the <A href="https://www.fludb.org/brc/home.spg?decorator=influenza "> Influenza Research Database</A>(3) and inputted to our program to generate switches to detect H5, H7, N1, N9 and PB2 RNAs. The sequences used are listed below (Type of flu/ region of origin/ number of lineage/ year of isolation):

</P><P style="font-family: roboto;font-size:115%;">
3 toehold switches with “good” predicted performance were chosen to target each RNA (For more information, please visit <A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Model">RNA thermodynamics modelling page</A>). For example, the three switches are named as H5-1, H5-2 and H5-3 for H5 RNA detection. The figure above shows the detection region of each toehold switch. Before constructing the toehold switches, we ensured all the switches passed our modelling criteria. 
</P><H3>Construction of toehold switch and trigger-expressing plasmid</H3><P style="font-family: roboto;font-size:115%;">
The upper picture showed the general structure of our toehold switch (For detailed structure, please visit our <A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Model"> modelling page</A>). mRFP(E1010) was chosen as the reporter of our toehold switches because it is very distinguishable by naked eyes while at the same time it can be quantified by measuring the fluorescence signal using a plate reader. The switch sequence generated by our program was linked with promoter(J23100) and reporter sequence. The trigger sequences was linked with promoter(J23100). Switches and triggers DNA were synthesized by IDT’s sponsored gBlock synthesis service. The gBlocks were used as template and amplified by PCR. The bands with correct size were gel-purified. We inserted the purified PCR products into pSB4C5 (for switch) or pSB1K3 (for trigger) using restriction cut and ligation. Sequencing results confirmed all 30 constructs were cloned. 

Due to safety and budget concern, partial sequence of the viral gene was used. Below listed the partial sequence we used:
</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P0"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P0-collapse" aria-expanded="false" aria-controls="P0-collapse"><DIV><DIV class="col-md-11">Click here to view trigger sequences </DIV></DIV></A></H4></DIV><DIV id="P0-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P0"><DIV class="panel-body"><OL><TABLE width="80%"><TBODY><TR><TH>Respective switch</TH><TH>Trigger sequence</TH></TR><TR><TD>PB2-1</TD><TD>AAACAUUGAAAAUAAGAGUACAUGAAGGAUAUGAGGAAUUCACAAUGGUUGGGCG
AAGAGCAACAGCCAUUCUAAGGAAAGCAACCAGAAGACUGAUCCAACUGAUAGUGAGUG
GGAAAGUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>PB2-2</TD><TD>CAAGGCAACCAAGAGGCUUACGGUGCUUGGGAAGGAUGCAGGUACAUUGAUGGAAG
ACCCGGACGAGGGAACAGCAGGAGUGGAAUCUGCAGUAUUGAGGGGAUUUCUGAUUCUG
GGCAAUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>PB2-3</TD><TD>AGAGUUAGUAAAAUGGGAGUAGAUGAAUAUUCCAGCACUGAGAGAGUGGUCGUGAG
UAUUGAUCGUUUCUUGAGGGUCCGAGACCAGAGGGGAAACGUACUCCUGUCUCCCGAAG
AGGUUUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>H5-1</TD><TD>UAGGGAUAAUGCAAAGGAGCUUGGUAACGGUUGUUUCGAGUUCUAUCACAGAUGUG
AUAAUGAAUGUAUGGAAAGUGUAAGAAACGGAACGUAUGACUACCCUCAAUAUUCAGAAG
AAGCUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>H5-2</TD><TD>AGUGGAGAAAAUCAAUCCAGCCAAUGACCUCUGUUAUCCAGGGAAUUUCAACGACU
AUGAAGAACUGAAACACCUAUUGAGCAGAAUAAACCAUUUUGAGAAAAUUCAGAUCAUUC
CCAAUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>H5-3</TD><TD>CAUCAUAGCAACGAGCAGGGGAGUGGGUACGCUGCAGACAAAGAAUCCACUCAAAGG
GCUAUAGAUGGAGUCACCAAUAAGGUCAAUUCGAUCAUUGACAAAAUGAACACUCAGUUU
GAGUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>N1-1</TD><TD>CAUCAUAGCAACGAGCAGGGGAGUGGGUACGCUGCAGACAAAGAAUCCACUCAAAGG
GCUAUAGAUGGAGUCACCAAUAAGGUCAAUUCGAUCAUUGACAAAAUGAACACUCAGUUU
GAGUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>N1-2</TD><TD>GUUUUCAUUUAAAUACGGCAAUGGUGUUUGGAUCGGGAGAACCAAAAGCACUAAUU
CCAGGAGCGGCUUUGAAAUGAUUUGGGACCCAAAUGGGUGGACUGGAACGGACAGUAGC
UUUUCUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>N1-3</TD><TD>AGAAGAUAAUAACCAUCGGAUCAAUCUGUAUGGUAAUUGGGAUAGCUAGCUUAAUG
UUACAAAUUGGAAACAUAAUCUCAAUAUGGAUCAGUCAUUCAAUUCAGACAGGGAACCAA
UGCCUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>H7-1</TD><TD>ACACCAGAAUGCACAGGGAGAGGGAACUGCUGCAGAUUACAAAAGCACUCAAUCGGC
AAUUGAUCAAAUAACAGGGAAAUUAAACCGGCUUAUAGCAAAAACCAACCAACAAUUUGAG
UUUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>H7-2</TD><TD>ACACAUUAACUGAAAGAGGAGUGGAAGUCGUCAAUGCAACUGAAACGGUGGAACGAAC
AAACAUCCCCCGGAUCUGCUCAAAAGGGAAAAGGACAGUUGAUCUCGGUCAAUGUGGACUC
CUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>H7-3</TD><TD>AGUGGCUACAAAGAUGUGAUACUUUGGUUUAGCUUCGGGGCAUCAUGUUUCAUACUUC
UAGCCAUUGUAAUGGGCCUUGUCUUCAUAUGUGUAAAGAAUGGAAACAUGCGGUGCACUAU
UUAGCGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>N9-1</TD><TD>UUAGUCACAAGAGAACCCUAUGUUUCAUGCAACCCAGAUGAAUGCAGGUUCUAUGCUCU
CAGCCAAGGAACAACAAUCAGAGGGAAACACUCAAACGGUACAAUACACGAUAGGUCCCAGUAG
CGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>N9-2</TD><TD>AAUAACUUAACUAAAGGGCUCUGUACUAUAAAUUCGUGGCACAUAUAUGGGAAAGACAAU
GCAGUAAGAAUUGGAGAAAGCUCGGAUGUUUUAGUCACAAGAGAACCCUAUGUUUCAUGCUAG
CGGCCGCUGCAGCUCGAG</TD></TR><TR><TD>N9-3</TD><TD>ACUACUUUAAAGAGGGGAAAAUAUUGAAAUGGGAGUCUCUGACUGGAACUGCUAAGCACAU
UGAAGAAUGCUCAUGUUACGGGGAACGAACAGGGAUUACCUGCACAUGCAAGGACAAUUUAGCG
GCCGCUGCAGCUCGAG</TD></TR></TBODY></TABLE></OL></DIV></DIV></DIV><P style="font-family: roboto;font-size:115%;">
Below is a table with the information of the backbone:
	<TABLE width="79%"><TBODY><TR><TH width="20%">Switch</TH><TH width="20%">Trigger</TH></TR><TR><TD><B>Backbone</B></TD><TD>pSB4C5</TD><TD>pSB1K3</TD></TR><TR><TD><B>Ori</B></TD><TD>pSC101 (~5 copies)</TD><TD>pMB1 (~100-300 copies)</TD></TR><TR><TD><B>Resistance</B></TD><TD>chloramphenicol</TD><TD>kanamycin</TD></TR></TBODY></TABLE></P><P style="font-family: roboto;font-size:115%;">
These two backbones with different Ori and antibiotic resistance genes were used because they will be used in the following experiments. Two co-transformed plasmids should not have the same type of origin of replication (Ori), or otherwise, they will compete for the replication machinery and affect the copy number(4). A higher copy number is chosen for the trigger plasmid to ensure trigger expression is in excess in cells. In addition, having two different antibiotic resistance genes avoid dropping out of either one of the plasmids during selection.



</P><H3><I>In vivo</I> assay: co-transformation in <I>E. coli</I></H3><P style="font-family: roboto;font-size:115%;">
Expressing our switches in <I>E. coli</I> is a cheaper and more familiar method to validate our switches and the program. The assay is done by co-transforming switch-expressing plasmid and trigger-expressing plasmid into <I>E. coli</I> BL21 (DE3). Negative control is done by co-transforming switch-expressing plasmid and empty pSB1K3. Single colonies of the transformants were picked and grown overnight starter culture. Expression of reporter mRFP was done by shaking culture for 6 hours after 1% inoculation. Cells were harvested, washed with PBS buffer, then aliquoted in 96-well plates. Fluorescence intensity was measured by BMG ClarioStar microplate reader. We later checked for difference of florescent signal between the switch-trigger co-transformants and the negative control. In theory, if the toehold switch works as expected, the florescent signal of switch-trigger co-transformants must be higher than that of the negative control.

</P><H3><I>In vitro</I> assay: Cell free system</H3><P style="font-family: roboto;font-size:115%;">
We used the Promega S30 T7 High-Yield Protein Expression System as our cell free system. After the <I>in vivo</I> test, we chose the workable switches to test in cell free system. We expressed: (1) toehold switch, (2) toehold switch and trigger pair, (3) J61002(constitutive mRFP generator as a positive control of the highest possible RFP expression), (3) luciferase positive control provided by the manufacturer, and (4) negative control where DNA is replaced with water in separate cell free reactions. We followed exactly the protocol provided by the company in our experiment. The reaction was done by mixing 2 μg DNA, S30 Premix and S30 Extract together to a reaction volume of 50 μl, followed by incubating at 37˚C for an hour. The florescent signal from the reacted mixture was then determined by BMG ClarioStar plate reader. Over-expression of protein in the reaction mixture was checked by SDS-PAGE.
</P><H3>Toehold switch and trigger cloning tools: BBa_K2254000 &amp; BBa_K2254001</H3><P style="font-family: roboto;font-size:115%;">
During our construction of switches, we realized that it would be relatively expensive to synthesis toehold switch together with the linker and reporter gene. We also want to have a convenient tool to construct and validate switches. Therefore, we constructed our toehold switch and trigger cloning tools that utilize the type IIS restriction enzyme Eco31I. Using the biobricks, user can simply construct their toehold switch or trigger by ordering 2 primer-like oligos. It also utilizes screening technique that is similar to blue/white screening. User can use this biobrick to construct their toehold switches that use pT7 as the promoter and mRFP as the reporter.

To use the biobricks to clone switches and triggers, user can just order 2 oligos (similar to primer) and insert the short DNA into the biobricks by restriction cut and ligation. For the 2 oligos (about 60 nt), one oligo should contain forward toehold switch sequence with AGGG at the 5’ end, and another one should contain reverse complement toehold switch sequence with AGTA at the 5’ end. To allow convenient screening of clones, there is a constitutive promoter (J23100) and RBS (B0034) situated between the two Eco31I sites. Digestion by Eco31I will remove them, and the subsequent insertion of switch will block the translation of mRFP, resulting in white colonies, whereas undigested plasmid or ligation of single digested plasmid will give red colonies.

The toehold switch will be linked to a flexible linker (AACCTGGCGGCAGCGCAAAAG) followed by mRFP reporter (E1010) and a double terminator (B0015). The linker is used to separate the coding sequence in the toehold switch and the reporter to prevent interference of protein folding. When the toehold switch hairpin is linearized by its orthogonal trigger RNA, RBS will be exposed, allowing the translation of downstream mRFP reporter gene. Since an XhoI site is present between the linker and the mRFP sequence, the reporter can be easily changed by restriction digestion followed by ligation.
</P><H3>Improving existing biobricks and project: Cancer switches</H3><P style="font-family: roboto;font-size:115%;">
Previously, the <A href="https://2015.igem.org/Team:CGU_Taiwan/Results"> Chang Gung University (CGU) Taiwan 2015 team</A> also worked on toehold switch. They designed toehold switches to detect biomarker of oral cancer. We investigated their project and found that their toehold switches have room for improvement.

Firstly, they use luciferase as reporter gene. We think that luciferase is not feasible to be used for on-site diagnosis since the luciferase activity need to be measured by a reader. Secondly, their toehold switches did not worked as expected because the reporter expression was suppressed when trigger was added.Thirdly, we inputted the sequences of those oral cancer biomarkers into our program and found that a better switch can be designed by choosing another region for detection (see <A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Model"> modelling page</A>). Therefore, we improved their biobricks by using <I>in silico</I> design of switch and RFP as a reporter.
</P><H3>Characterization of chromoproteins</H3><P style="font-family: roboto;font-size:115%;">
Different types of body fluid have different pH (5)(below figure). Inspired by a  <A href="https://2017.igem.org/Team:Hong_Kong-CUHK/HP/Gold_Integrated"> medical expert</A> we interviewed, we would like to investigate if the pH in body fluid can interfere with the reporter protein we used in our test, since we are going to use body fluid as sample in our influenza diagnostic test. Fluorescent signal is known to be pH-dependent because pH can change the folding and conformation of the fluorophore, and ionization states can also cause shift in the Excitation/Emission spectra (6). Therefore, we characterized the fluorescence of 2 fluorescent proteins: <A href="http://parts.igem.org/Part:BBa_E1010"> mRFP(BBa_E1010) </A> and <A href="http://parts.igem.org/Part:BBa_K1033916"> amajLime(BBa_K1033916)</A>  at different pH. We want to find out their optimum pH and see if they are suitable to be the reporter protein in our diagnostic test. 
<CENTER>pH values of commonly extracted body fluids</CENTER><CENTER><TABLE width="79%"><TBODY><TR><TH width="20%">Body Fluids</TH><TH width="20%">pH</TH></TR><TR><TD>Blood</TD><TD>7.4</TD></TR><TR><TD>Saliva</TD><TD>6.4</TD></TR><TR><TD>Ileum fluid</TD><TD>8.0</TD></TR><TR><TD>Serum</TD><TD>7.2</TD></TR><TR><TD>Stomach juice</TD><TD>1.5</TD></TR><TR><TD>Urine</TD><TD>5.8</TD></TR></TBODY></TABLE></CENTER></P><P style="font-family: roboto;font-size:115%;">
To examine the performance of mRFP and amajLime under different pH, we inserted the biobricks into pSB1A2 and expressed them in <I>E. coli</I>  C41(DE3) . mRFP and amajLime were then purified form lysed cells by anion exchange chromatography (AEC) and hydrophobic interaction chromatography (HIC). 


Below picture shows the SDS–PAGE analysis of purification of amajLime (left) and mRFP (right). 

</P><P style="font-family: roboto;font-size:115%;">
F.1 to F.3 represents chronological order of elutions in HIC. Samples (10µl) were mixed with 10 µl 2X SDS gel-loading buffer and 10 µl of the mixture were loaded on the SDS-gel. The purest fraction, F.2 in both cases, were selected to proceed to pH stability test, where purified proteins were diluted in buffers in the range of pH 2-12 to a final concentration of 100 µg/mL, and the fluorescence intensity was recorded by BMG ClarioStar plate reader. 
</P><H3>Shipping</H3>
Since iGEM requires standard backbone pSB1C3 for shipping, we later sub-cloned our biobricks into pSB1C3. Agarose electrophoresis and sequencing revealed that all biobricks were successfully inserted. Below showed our 1% agarose gel photo (our biobricks ~ 1000bp, pSB1C3 ~ 2000bp):
<H3>References:</H3>
1. Green AA, Silver PA, Collins JJ, Yin P. Toehold switches: de-novo-designed regulators of gene expression. Cell. 2014 Nov 6;159(4):925-39.

2. Lorenz, Ronny and Bernhart, Stephan H. and Höner zu Siederdissen, Christian and Tafer, Hakim and Flamm, Christoph and Stadler, Peter F. and Hofacker, Ivo L. ViennaRNA Package 2.0. Algorithms for Molecular Biology, 6:1 26, 2011, 

3. Zhang Y et. al. Influenza Research Database: An integrated bioinformatics resource for influenza virus research. Nucleic Acids Res. 2017 Jan 4;45(D1):D466-D474. 

4. Nordström K, Dasgupta S. Copy-number control of the Escherichia coli chromosome: a plasmidologist's view. EMBO Rep. 2006 May;7(5):484-9.

5. Schwalfenberg GK. The alkaline diet: is there evidence that an alkaline pH diet benefits health? J Environ Public Health. 2012;2012:727630.

6. Battad JM et al. A structural basis for the pH-dependent increase in fluorescence efficiency of chromoproteins. J Mol Biol. 2007 May 11;368(4):998-1010. 





</DIV></DIV></DIV></SECTION><FOOTER class="footer-distributed"><DIV class="footer-right">   <P> Email: igemcuhk1617@gmail.com </P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>