<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong-CUHK_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong-CUHK/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-fixed-top" style="margin-top:0px; margin-left: -30px"><NAV class="navbar-inner"><DIV class="row-sm-height"><DIV class="col-md-10"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK"><SPAN>Home</SPAN></A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
 Project </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/InterLab">InterLab</A></LI></UL><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Model"><SPAN>Modelling</SPAN></A></LI><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Software"><SPAN>Software</SPAN></A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Protocol">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/HP/Silver">Engagement and Education</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Prototype">Prototype</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Collaborations">Collaboration</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Team</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Team">About Us </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Attributions">Attributions </A></LI></DIV></DIV></DIV></NAV></DIV><SECTION id="section01"><DIV class="column full_size"><CENTER><H1>Modeling:  Designing Toehold Switch</H1></CENTER><P style="font-family: quicksand;font-size:150%;">Background </P><P style="font-family: roboto;font-size:115%;">
According to Green <I>et al.</I> (2014) [1], the optimal length of RNA to be detected by a toehold switch is around 30 bp (complementary region at below figure). In other words, a target RNA with 1000 bp in length can have 970 possible switches with different performance, which is governed by their structures and thermodynamic parameters. <STRONG>We aim at providing a workflow and platform of modeling to help users design the switches by reducing the manual processing and increasing the hit-rate of finding a good switch.</STRONG>
We adopted the toehold switch design from the original paper (below figure). Our toehold switch contains 15 nts “toehold domain”, 18 nts stem and a loop that contains the RBS B0034. A 21 nts linker sequence plus an mRFP reporter sequence is present downstream the toehold switch. The linker is used to separate the coding sequence in the toehold switch and the reporter to prevent interference of protein folding.
</P></DIV><DIV class="column full_size"><P style="font-family: quicksand;font-size:150%;">RNA thermodynamic Modelling</P><P style="font-family: roboto;font-size:125%;color:blue"><B>Assumption 1: Switch MFE (Minimum Free Energy) correlates with the expression leakage</B></P><P style="font-family: roboto;font-size:115%;">
●   Switch MFE is the minimum Gibbs free energy that a toehold switch could have among all the possible structures. 
●   Expression leakage is a phenomenon where the reporter (i.e. Red Fluorescent Protein in our project) is expressed in the absence of trigger RNA. The level of leakage can be measured as:

</P><P style="font-family: roboto;font-size:115%;">
To activate toehold switch, an amount of energy is needed to open the toehold switch hairpin. The Switch MFE reﬂects the difficulty for the toehold switch unwinding process. We assume that the more negative the Switch MFE, the harder for the unwinding to take place, and hence a lower leakage.
</P><B><P style="font-family: roboto;font-size:125%;color:blue">Assumption 2: ΔG<SUB>RBS-Linker</SUB> correlates with the duplex expression</P></B><P style="font-family: roboto;font-size:115%;">
●   ΔG<SUB>RBS-Linker</SUB>  is the Gibbs free energy of the RNA sequence starting from the RBS to the linker in the switch-trigger duplex. 
●   The duplex expression is the reporter expression of the switch-trigger duplex RNA.

After the switch RNA hairpin is unwound after binding to the trigger RNA, a switch-trigger duplex RNA would be formed. The RBS-linker region of the MFE structure of this duplex  RNA should have minimal base pairs. This makes it easier to unwind the RNA for this region, allowing ribosomes to bind to the RBS and move along the RNA for translation of the RFP reporter gene to occur. ΔG<SUB>RBS-Linker</SUB>  reﬂects the difficulty for the unwinding process of the RBS-linker region. It is assumed that the more negative the ΔG<SUB>RBS-Linker</SUB> , the harder it is for the unwinding to take place, leading to lower translation rates. Thus, the duplex expression would be reduced.
</P><P style="font-family: roboto;font-size:125%;color:blue">Assumption 3: ΔMFE correlates with the reporter expression of Switch-Trigger duplex</P><P style="font-family: roboto;font-size:115%;">
●   ΔMFE is defined as MFE of the switch-trigger duplex RNA minus (switch RNA MFE + trigger RNA MFE).

Since ΔMFE = –RTlnK, where:

R = Gas constant

T = Temperature

K = Equilibrium constant 

Therefore, we assume that the more negative the MFE difference is, the higher the switch-trigger duplex RNA concentration compared to that of the switch RNA when in equilibrium.
</P><P style="font-family: roboto;font-size:130%;"><CENTER>Switch RNA + Trigger RNA ↔ Switch-Trigger Duplex</CENTER></P><P style="font-family: roboto;font-size:115%;">
Consequently, increased equilibrium concentrations of the switch-trigger duplex RNA would provide an increased number of active mRNAs for the translation of the reporter RFP.




(THE toehold domain base pairing modelling would be covered by the switch RNA suboptimal structure modelling part)

</P></DIV><DIV class="column full_size"><P style="font-family: quicksand;font-size:150%;">Initial screening using our software</P><P style="font-family: roboto;font-size:115%;">
To minimize the manpower on screening of the switches, we constructed an online toehold switch design program. Apart from basic thermodynamic parameters, it also screens for other factors(please visit our <A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Software"> software page</A>. Ultimately, the program generated a list of possible toehold switch sequences according to many different free energy parameters using the <A href="http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/barriers.cgi">ViennaRNA package</A> [2]. The graph below shows 394 possible H5 toehold switches generated by our software. The assumptions motioned earlier stated 3 very important parameters for the selection of switch candidates with the greatest possible performances: Switch MFE, ΔG<SUB>RBS-linker</SUB>, and ΔMFE. We applied these parameters to our switch selection process: We first chose the switches that with the highest ΔG<SUB>RBS-linker</SUB> (-3.8 kcal/mol). Among those switches, we chose the 3 switches with the lowest switch MFE and the highest ΔMFE.
</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P0"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P0-collapse" aria-expanded="false" aria-controls="P0-collapse"><DIV><DIV class="col-md-11">Thermodynamic parameters of chosen influenza switches </DIV></DIV></A></H4></DIV><DIV id="P0-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P0"><DIV class="panel-body"><OL><TABLE width="100%"><TBODY><TR><TH>Switch Candidate</TH><TH>ΔG<SUB>RBS-linker</SUB></TH><TH>MFE Switch</TH><TH>ΔMFE</TH></TR><TR><TD>H5-1</TD><TD>-3.8</TD><TD>-19.1</TD><TD>+32.1</TD></TR><TR><TD>H5-2</TD><TD>-3.8</TD><TD>-21.1</TD><TD>+34.8</TD></TR><TR><TD>H5-3</TD><TD>-3.8</TD><TD>-24.0</TD><TD>+37.9</TD></TR><TR><TD>H7-1</TD><TD>-3.8</TD><TD>-24.5</TD><TD>+41.4</TD></TR><TR><TD>H7-2</TD><TD>-3.8</TD><TD>-17.8</TD><TD>+34.2</TD></TR><TR><TD>H7-3</TD><TD>-3.8</TD><TD>-16.3</TD><TD>+26.3</TD></TR><TR><TD>N1-1</TD><TD>-3.8</TD><TD>-17.1</TD><TD>+34.4</TD></TR><TR><TD>N1-2</TD><TD>-3.8</TD><TD>-17.1</TD><TD>+24.0</TD></TR><TR><TD>N1-3</TD><TD>-3.8</TD><TD>-19.4</TD><TD>+25.9</TD></TR><TR><TD>N9-1</TD><TD>-3.8</TD><TD>-22.2</TD><TD>+34.5</TD></TR><TR><TD>N9-2</TD><TD>-3.8</TD><TD>-17.9</TD><TD>+30.8</TD></TR><TR><TD>N9-3</TD><TD>-3.8</TD><TD>-21.6</TD><TD>+27.0</TD></TR><TR><TD>PB2-1</TD><TD>-3.8</TD><TD>-12.5</TD><TD>+34.6</TD></TR><TR><TD>PB2-2</TD><TD>-3.8</TD><TD>-24.7</TD><TD>+38.0</TD></TR><TR><TD>PB2-3</TD><TD>-3.8</TD><TD>-16.0</TD><TD>+28.2</TD></TR></TBODY></TABLE></OL></DIV></DIV></DIV><P style="font-family: roboto;font-size:115%;"> To improve <A href="https://2015.igem.org/Team:CGU_Taiwan/Results">Chang Gung University(CGU)’s oral cancer toehold switch</A>, we employed the screening function of our program. A toehold switch was chosen that is predicted to outperform the CGU’s switch according to the MFE RBS-Linker.</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P1"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P1-collapse" aria-expanded="false" aria-controls="P1-collapse"><DIV><DIV class="col-md-11">Thermodynamic parameters of original and improved switches from CGU</DIV></DIV></A></H4></DIV><DIV id="P1-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P1"><DIV class="panel-body"><OL><TABLE width="100%"><TBODY><TR><TH>Switch Candidate</TH><TH>ΔG<SUB>RBS-linker</SUB></TH><TH>MFE Switch</TH><TH>ΔMFE</TH></TR><TR><TD>CGU’s SAT switch</TD><TD>-9.2</TD><TD>-17.4</TD><TD>+46.8</TD></TR><TR><TD>New SAT switch</TD><TD>-4.1</TD><TD>-24.5</TD><TD>+23.1</TD></TR></TBODY></TABLE></OL></DIV></DIV></DIV><DIV class="column full_size"><P style="font-family: quicksand;font-size:150%;">Suboptimal Structure Modelling</P><P style="font-family: roboto;font-size:125%;color:blue"><B>Assumption 4: Accessibility of toehold domain correlates with the performance of switch</B></P><P style="font-family: roboto;font-size:115%;">
●   The toehold domain (first 15 nts) of the switch RNA is crucial for the binding of the trigger RNA to the switch RNA to initiate the switch-unwinding process: 


Toehold domain must have minimal paired bases in the switch RNA to ensure the successful binding of this domain with the complementary sequence in the trigger RNA, which allows the unwinding of the switch RNA and permits translation of the reporter protein, RFP, to occur. </P><P style="font-family: roboto;font-size:115%;">
Our program can only calculate the structure with the minimal free energy (MFE) for each target RNA region to reduce calculation workload. In reality, different conformations of RNAs with the same sequence co-exist in solution, and the concentrations of those populations are determined by their structures and free energy. Therefore, we manually checked the predicted structures and equilibrium concentrations of the ten suboptimal structures of each influenza switches with the lowest MFEs on the web tool developed by <A href="http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/barriers.cgi">ViennaRNA package</A> [2]. Then we predicted the performance of each influenza switches and compared with the experimental results.</P><P style="font-family: roboto;font-size:115%;">
The table below shows the different suboptimal structures of each switch RNA sequence:</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P3"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P3-collapse" aria-expanded="false" aria-controls="P3-collapse"><DIV><DIV class="col-md-11">Thermodynamic parameters of suboptimal structures of chosen influenza switches </DIV></DIV></A></H4></DIV><DIV id="P3-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P3"><DIV class="panel-body"><OL><TABLE width="100%"><TBODY><TR><TH>Switch Candidate</TH><TH>Suboptimal structures</TH><TH>Expected fluorescence</TH><TH>Experimental fluorescence(relative)</TH><TH>Conclusion</TH></TR><TR><TD>H5-1</TD><TD><FONT color="red">1, 6, 7,</FONT><FONT color="green">2</FONT></TD><TD>Low</TD><TD>11.676366 (Low)</TD><TD>True Negative</TD></TR><TR><TD>H5-2</TD><TD><FONT color="red">1, 2, 3, 4,</FONT><FONT color="green">5</FONT></TD><TD>Low</TD><TD>38.24207 (Low)</TD><TD>True Negative</TD></TR><TR><TD>H5-3</TD><TD><A href="https://static.igem.org/mediawiki/2017/c/c0/H5-3-1.png"><FONT color="red">1, </FONT></A><A href="https://static.igem.org/mediawiki/2017/f/fa/H5-3-2.png"><FONT color="red">2, </FONT></A><A href="https://static.igem.org/mediawiki/2017/a/af/H5-3-4.png"><FONT color="red">4, </FONT></A><A href="https://static.igem.org/mediawiki/2017/b/bd/H5-3-3.png"><FONT color="green">3</FONT></A></TD><TD>Low</TD><TD>13.34770467 (low)</TD><TD>True Negative</TD></TR><TR><TD>H7-1</TD><TD><FONT color="red">1, 2, 3, 4</FONT></TD><TD>Low</TD><TD>10.82038167 (Low)</TD><TD>True Negative</TD></TR><TR><TD>H7-2</TD><TD><FONT color="red">1, 3,</FONT><FONT color="green">2, 7</FONT></TD><TD>Low</TD><TD>8.577606333 (Low)</TD><TD>True Negative</TD></TR><TR><TD>H7-3</TD><TD><FONT color="green">1, 2</FONT></TD><TD>High</TD><TD>637.066 (High)</TD><TD>True Positive</TD></TR><TR><TD>N1-1</TD><TD><FONT color="green">1, 2</FONT></TD><TD>High</TD><TD>204.7757333 (High)</TD><TD>True Positive</TD></TR><TR><TD>N1-2</TD><TD><FONT color="red">1, 2</FONT></TD><TD>Low</TD><TD>9.325934 (Low)</TD><TD>True Negative</TD></TR><TR><TD>N1-3</TD><TD><FONT color="green">1, 2</FONT></TD><TD>High</TD><TD>9.148810667 (Low)</TD><TD><EM>False Positive</EM></TD></TR><TR><TD>N9-1</TD><TD><FONT color="green">1, 2, 3</FONT></TD><TD>High</TD><TD>166.49639 (High)</TD><TD>True Positive</TD></TR><TR><TD>N9-2</TD><TD><FONT color="green">1, 2, 6</FONT></TD><TD>High</TD><TD>604.2225333 (High)</TD><TD>True Positive</TD></TR><TR><TD>N9-3</TD><TD><FONT color="green">1</FONT></TD><TD>High</TD><TD>10.587811 (Low)</TD><TD><EM>False Positive</EM></TD></TR><TR><TD>PB2-1</TD><TD><FONT color="green">1, 2</FONT></TD><TD>High</TD><TD>8.756591333 (Low)</TD><TD><EM>False Positive</EM></TD></TR><TR><TD>PB2-2</TD><TD><FONT color="red">1, 3</FONT></TD><TD>Low</TD><TD>24.68932333 (Low)</TD><TD>True Negative</TD></TR><TR><TD>PB2-3</TD><TD><FONT color="green">1, 2, 3, 5</FONT></TD><TD>High</TD><TD>368.0475667 (High)</TD><TD>True Positive</TD></TR></TBODY></TABLE></OL></DIV></DIV></DIV><P style="font-family: roboto;font-size:115%;">
In the table, each suboptimal structure is labelled by numbers 1 to 10, where <EM>structure 1 has the lowest free energy (the MFE structure)</EM>, and <EM>structure 10 has the highest </EM> free energy. In the &quot;Suboptimal structures&quot; column, only suboptimal structures with non-negligible equilibrium concentrations are considered and shown. The equilibrium concentrations were calculated by the ViennaRNA web server [2].

For example, for H5-3 switch:
 

We checked the status of the toehold domain for each suboptimal structure from the ViennaRNA web server output [2] to see if it was open or closed. In the table, the structures with an open toehold domain are shown in <FONT color="green">green</FONT>, and ones with a closed toehold domain are shown in <FONT color="red">red</FONT>.

For example, for H5-3 switch:
</P><P style="font-family: roboto;font-size:115%;">
The expected fluorescence(determined by the ratio of suboptimal structures with open/closed toehold domains) exhibited by <I>E. coli</I> co-transformed with the switch and trigger RNA after 12 hours was compared with the actual observed fluorescence during the experiment.</P><P style="font-family: roboto;font-size:115%;">
From the table above, we observed that only <EM>3 out of 15 of</EM> our predictions were incorrect, showing promising accuracy of this prediction method. None of the 3 incorrect predictions were false negatives, indicating that <STRONG>the suboptimal structure prediction method should be mostly applied to filtering out switch sequences with low predicted performances</STRONG>.</P><DIV class="column full_size"><P style="font-family: quicksand;font-size:150%;">Experimental verification of the thermodynamic mode</P><P style="font-family: roboto;font-size:125%;"><U><B> Correlation of switch MFE (Minimum Free Energy) and expression leakage   </B></U></P>
The plot shows some kind of weak positive correlation  between expression leakage and  the MFE of a switch, with an outlier N9-2. This suggests that the MFE of a switch might not have a large contribution to the expression leakage.  
<P style="font-family: roboto;font-size:125%;"><U><B> Correlation of switch MFE (Minimum Free Energy) and the expression by Switch-Trigger duplex </B></U></P>
Although the switch MFE correlates with the expression leakage, this plot showed that the switches with lower MFE also have lower RFP signal in the presence of trigger. This suggests that a low switch MFE could also be a hindrance to detection that it increases the energy input to the system (activation energy), so it takes more energy for switch-trigger duplex formation to occur. This could lower the expression level of the switch-trigger duplex.
<P style="font-family: roboto;font-size:125%;"><U><B> Correlation of MFE difference and the expression by Switch-Trigger duplex  </B></U></P>
We initially thought that the larger the MFE difference between the sum of the MFE of switch and trigger and that of the switch-trigger duplex, the more favourable the duplex structure will be. However, there seems to be a weak positive correlation between the expression level of the switch-trigger duplex and the MFE difference of a switch. This could be because that a low MFE difference could indicate a more similar conformation of the switch/trigger RNA to the switch-trigger duplex RNA. This could lead to less conformational changes in the duplex formation process, making the process faster. Thus, a small MFE difference could lead to an increased expression level of the switch-trigger duplex.
<P style="font-family: roboto;font-size:125%;"><U><B> Correlation of toehold domain base pair and the expression by Switch-Trigger duplex  </B></U></P>
The plot shows some kind of negative correlation between RFP signal in the presence of trigger and the number of base pairs in the toehold domain. Some exceptions were observed, they are N1-3, N9-3, N9-1. H5-3 and N1-2. This supports the assumption in the suboptimal structure modelling that an increasing number of unpaired bases in the toehold domain promotes initial binding of a trigger RNA to the corresponding switch RNA.



References:
[1] Green AA, Silver PA, Collins JJ, Yin P. Toehold switches: De-novo-designed regulators of gene expression. <I>Cell.</I> 2014;159(4):925–39.
[2] Hofacker IL. RNA secondary structure analysis using the Vienna RNA package. <I>Curr Protoc Bioinformatics [Internet]</I>. 2009;Chapter 12:Unit12.2. Available from: 
http://www.ncbi.nlm.nih.gov/pubmed/19496057
[3]J. N. Zadeh, C. D. Steenberg, J. S. Bois, B. R. Wolfe, M. B. Pierce, A. R. Khan, R. M. Dirks, N. A. Pierce. NUPACK: analysis and design of nucleic acid systems. <I>J Compute Chem</I>, 32:170–173, 2011.

</DIV></DIV></DIV></DIV></DIV></DIV></SECTION></DIV><FOOTER class="footer-distributed"><DIV class="footer-right">   <P> Email: igemcuhk1617@gmail.com </P></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>