<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong-CUHK_InterLab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong-CUHK/InterLab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-fixed-top" style="margin-top:0px; margin-left: -30px"><NAV class="navbar-inner"><DIV class="row-sm-height"><DIV class="col-md-10"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK"><SPAN>Home</SPAN></A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
 Project </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/InterLab">InterLab</A></LI></UL><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Model"><SPAN>Modelling</SPAN></A></LI><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Software"><SPAN>Software</SPAN></A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Protocol">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/HP/Silver">Engagement and Education</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Prototype">Prototype</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Collaborations">Collaboration</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Team</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Team">About Us </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Attributions">Attributions </A></LI></DIV></DIV></DIV></NAV></DIV><SECTION id="section01"><DIV class="column full_size"><H3>Background</H3><P style="font-family: roboto;font-size:115%;">Reliable and repeatable measurement is the golden rule of engineering, and so does synthetic biology. However, most of the fluorescent measurement data generated nowadays cannot be compared because it is usually reported in relative unit, but not in absolute unit. In addition, different groups may perform measurement with different protocols, which makes it hard to reproduce. Therefore, iGEM developed a green fluorescent protein (GFP) measurement protocol in order to produce a more reliable, repeatable measurement of GFP. GFP is one of the most commonly used reporter for measurement and easily to be measured in most of laboratories. In the protocol,the unit for fluorescence data is unified so that the results can be compared. The InterLab Study protocol also unifies the measurement procedures and prevents different data processing for the measurement. 
</P><H3>The Fourth InterLab Study</H3><P style="font-family: roboto;font-size:115%;">This year, iGEM invited all teams to join the fourth InterLab Study. The aim of the study is to find out how close can the numbers be when fluorescence is measured all around the world using the same InterLab protocol and constructs. We registered and measured all the parts according to the instructions.
</P><H3>Experiments</H3><P style="font-family: roboto;font-size:115%;">iGEM headquarter provided 8 plasmids for the InterLab Study. Devices 1-6 and positive control have the same reporter gene (GFP), terminator (B0015) and backbone (pSB1C3). However, devices 1-3 share the same RBS (B0034), while devices 4-6 share another modified RBS called bicistronic device (BCD2). Different promoters are also used in different plasmids. According to the strength of promoter described by iGEM2006_Berkeley team, device 1 should have the strongest fluorescence and device 3 should have the weakest among devices 1-3, while Device 4 should have the strongest fluorescence and device 6 should have the weakest among devices 4-6.
</P><TABLE width="80%"><THEAD><TR><TH colspan="6"><U>Summary of experimental designs</U></TH></TR></THEAD><TBODY><TR><TH>Promoter (strength)</TH><TH>RBS</TH><TH>Coding Sequence</TH><TH>Terminator</TH><TH>Backbone</TH></TR><TR><TD><B>Negative control (R0040)</B></TD><TD>R0040 (N/A)</TD><TD colspan="3">N/A</TD><TD rowspan="8">pSB1C3</TD></TR><TR><TD><B>Positive control (I20270)</B></TD><TD>J23151(N/A)</TD><TD>B0032</TD><TD rowspan="7" bgcolor="#40FF00">GFP</TD><TD rowspan="7">B0015</TD></TR><TR><TD><B>Device 1 (J364000)</B></TD><TD bgcolor="#FF0000">J23101(1791)</TD><TD rowspan="3">B0034</TD></TR><TR><TD><B>Device 2 (J364001)</B></TD><TD bgcolor="#FA5858">J23106(1185)</TD></TR><TR><TD><B>Device 3 (J364002)</B></TD><TD bgcolor="#FA5858">J23117(162)</TD></TR><TR><TD><B>Device 4 (J364003)</B></TD><TD bgcolor="#FF0000">J23101(1791)</TD><TD rowspan="3">BCD2</TD></TR><TR><TD><B>Device 5 (J364004)</B></TD><TD bgcolor="#FA5858">J23106(1185)</TD></TR><TR><TD><B>Device 6 (J364005)</B></TD><TD bgcolor="#FA5858">J23117(162)</TD></TR></TBODY></TABLE><H3>What is a Bicistronic Device (BCD)?</H3><CENTER><P>Photo adopted from Mutalik, 2013</P></CENTER><P style="font-family: roboto;font-size:115%;">Bicistronic device (BCD) is a modified ribosome binding site (RBS) with another cistron. The device consists of another cistron (cistron 1) with another RBS (SD2)
between RBS (SD1) and gene of interest (cistron 2). Also, the stop codon of cistron 1 overlaps the start codon of cistron 2. Ribosome binding efficiency and translation rate will be affected after the secondary structure near the RBS has changed due to the change of gene of interest. This device can maintain the ribosome binding efficiency and translation rate even though the gene of interest has changed. Therefore, it is used to control the amount of fluorescence in this study. BCD is expected to generate a more reliable and precise gene expression.</P><H3>Method</H3><P style="font-family: roboto;font-size:115%;">We follow exactly the interlab protocol provided by iGEM(link)
</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P0"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P0-collapse" aria-expanded="false" aria-controls="P0-collapse"><DIV><DIV class="col-md-11">Click here to view iGEM Interlab Protocol </DIV></DIV></A></H4></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P1"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P1-collapse" aria-expanded="false" aria-controls="P1-collapse"><DIV><DIV class="col-md-11">Click here to view our plate reader setting </DIV></DIV></A></H4></DIV><DIV id="P1-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P1"><DIV class="panel-body"><OL>



Microplate used: Corning® clear flat bottom black 96 well plates 
Instrument used: BMG LABTECH’s CLARIOstar® <TABLE width="80%"><TBODY><TR><TH colspan="2">Instrument Settings for measuring OD600 of LUDOX and Cells:</TH></TR><TR><TH colspan="2">Endpoint settings</TH></TR><TR><TD>No. of flashes per well</TD><TD>10</TD></TR><TR><TD>Scan mode</TD><TD>orbital averaging</TD></TR><TR><TD>Scan diameter [mm]</TD><TD>3</TD></TR><TR><TH colspan="2">Optic settings</TH></TR><TR><TD>Excitation</TD><TD>600</TD></TR><TR><TH colspan="2">General settings</TH></TR><TR><TH colspan="2">Top optic used</TH></TR><TR><TD>Settling time [s]</TD><TD>0.1</TD></TR><TR><TD>Reading direction</TD><TD>bidirectional, horizontal left to right, top to bottom</TD></TR><TR><TD>Target temperature [°C]</TD><TD>25</TD></TR></TBODY></TABLE><TABLE width="80%"><TBODY><TR><TH colspan="2">Instrument Settings for measuring fluorecence of Fluorescein and GFP:</TH></TR><TR><TH colspan="2">Endpoint settings</TH></TR><TR><TD>No. of flashes per well</TD><TD>8</TD></TR><TR><TD>Scan mode</TD><TD>orbital averaging</TD></TR><TR><TD>Scan diameter [mm]</TD><TD>3</TD></TR><TR><TH colspan="2">Optic settings</TH></TR><TR><TD>Excitation:</TD><TD>470-15</TD></TR><TR><TD>Emission</TD><TD>515-20</TD></TR><TR><TD>Gain</TD><TD>500</TD></TR><TR><TD>Focal height [mm]</TD><TD>9</TD></TR><TR><TH colspan="2">General settings</TH></TR><TR><TH colspan="2">Top optic used</TH></TR><TR><TD>Reading direction</TD><TD>bidirectional, horizontal left to right, top to bottom</TD></TR><TR><TD>Target temperature [°C]</TD><TD>25
</TD></TR></TBODY></TABLE></OL></DIV></DIV></DIV><H3>Results</H3><U><P style="font-family: roboto;font-size:115%;">GFP expression in colonies</P></U><P style="font-family: roboto;font-size:115%;">
The colonies were observed from blue light box.
The controls show that the plasmid containing GFP gene will give fluorescence.
The strength of fluorescence correlates to the strength of constitutive promoters. The higher the strength of promoters, the brighter the colonies. Among the colonies with devices 4-6, the colonies with device 4 showed the highest fluorescence while those with device 6 showed the lowest.
Among the colonies with devices 1-3, the colonies with device 3 show the lowest fluorescence but no observable difference in fluorescence between the colonies with devices 1 and 2. </P><U><P style="font-family: roboto;font-size:115%;">Fluorescence standard curve</P></U><P style="font-family: roboto;font-size:115%;">
We generated the fluorescence-[fluorescein] standard curves for calibration with the iGEM protocol. The fluorescein concentration can be found with corresponding fluorescence from the graphs.
</P><P style="font-family: roboto;font-size:115%;"><U>GFP expression in culture</U></P><P style="font-family: roboto;font-size:115%;">
We measured the cell density (OD600 values) and fluorescence signal of the cell culture at 2-hour interval. We processed the data by averaging the obtained values from two flasks of culture of different colonies with the same device. The results were represented in the graph below:
</P><P style="font-family: roboto;font-size:115%;">The increasing trends of OD600 and fluorescence in all devices are shown in the first and the second graph. </P><P style="font-family: roboto;font-size:115%;">
Among the devices that share the RBS B0034 (devices 1, 2, 3), device 1 has the highest fluorescence per OD600 and device 3 has the lowest. The difference in fluorescence per OD600 is probably due to the promoter strength. Device 1 has the strongest promoter(J23101), thus it gives the highest fluorescence. Device 3 has the weakest promoter(J23117) , thus it gives the lowest fluorescence. The strength of promoter can also be deduced by comparing the fluorescence per OD600 given by the devices that share the BCD2 (devices 4, 5, 6). Device 4 has the strongest fluorescent signal because it uses the strongest promoter (J23101), while device 6 has a weakest fluorescent signal because the weakest promoter(J23117) is used.
</P><P style="font-family: roboto;font-size:115%;">
Comparing the fluorescence signal given by devices with the same promoter but different RBS, we observed that the devices that use BCD2 (devices 4,5 and 6) gave a lower fluorescence comparing with the devices that use B0034 (devices 1, 2 and 3). We may therefore conclude that BCD2 has a weaker RBS than B0034.
</P><P style="font-family: roboto;font-size:115%;">
Another interesting point is the fluorescence per cell of all devices peaked at (t=2 hour). These may because the increase in cell number is much faster than the overall expression of GFP after t=2 hour.
</P><H3>Reference</H3>
1. Mutalik VK, Guimaraes JC, Cambray G, Lam C, Christoffersen MJ, Mai QA, Tran AB, Paull M, Keasling JD, Arkin AP, Endy D. Precise and reliable gene expression via standard transcription and translation initiation elements. Nat Methods. 2013 Apr;10(4):354-60. 

</DIV></DIV></DIV></SECTION><FOOTER class="footer-distributed"><DIV class="footer-right">   <P> Email: igemcuhk1617@gmail.com </P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>