<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Hong_Kong-CUHK_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Hong Kong-CUHK/Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-fixed-top" style="margin-top:0px; margin-left: -30px"><NAV class="navbar-inner"><DIV class="row-sm-height"><DIV class="col-md-10"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK"><SPAN>Home</SPAN></A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
 Project </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/InterLab">InterLab</A></LI></UL><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Model"><SPAN>Modelling</SPAN></A></LI><LI class="menu-item"><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Software"><SPAN>Software</SPAN></A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">
Notebook </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Protocol">Protocols</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/HP/Silver">Engagement and Education</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Prototype">Prototype</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Collaborations">Collaboration</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">

                 Team</A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Team">About Us </A></LI><LI><A href="https://2017.igem.org/Team:Hong_Kong-CUHK/Attributions">Attributions </A></LI></DIV></DIV></DIV></NAV></DIV><SECTION id="section01"><CENTER><H1>Overview of Software</H1></CENTER><P style="font-family: roboto;font-size:115%;">
To reduce manpower, we wrote a program to automatically generate toehold switch sequences from target RNA input. We developed it to a website for the convenience of other iGEMers who are interested in toehold switch application. Although this website can execute many functions to reduce repetitive works for user, careful examination of thermodynamic parameters and RNA secondary structures is still encouraged to get a promising switch sequence. To help iGEMers get started, we would like to do a brief introduction on our website here.

Our web tool takes as input the trigger RNA sequence and various design parameters, such as the promoter and RBS sequences to be inserted into the toehold switch, length of the recognition part of the trigger RNA and experiment temperature. Users can also choose to produce some optional outputs, such as rare codon count, minimum free energy and numbers of paired and unpaired bases, which could be useful for ranking the resulting list of candidate toehold switches.

Upon receiving the inputs, our web tool carries out a number of steps to design the toehold switches (rightfigure). It first checks the format and validity of the inputs. After that, it uses a sliding window to consider every x consecutive bases of the trigger sequence as the potential recognition part, where x is the length of this part specified by the user. A toehold switch is constructed based on the subsequence in this window and the other input parameters. The toehold switch sequence is then subject to a sequence of tests, including the free of stop codons between the start codon and the downstream gene, and the lack of consecutive bases of the same type. If the switch can pass all these tests, free energy calculations will be performed next for the switch and trigger monomers and their interacting duplex, if the user chooses to output such information. Finally, optionally the sub-sequence in the window can be used to search the sequence database to check for highly similar off-target sequences.
</P><H1> Instructions</H1><H3>Create Your Account</H3><P style="font-family: roboto;font-size:115%;">
An account is required for every user to execute the main program, save the result and retrieve result later. A total of fifty results could be saved in one account.
</P><H3>Design your toehold switch</H3><H4><U>Input target RNA sequence</U></H4><P style="font-family: roboto;font-size:115%;">
User can input target RNA in plain text or FASTA file with numbers, space, newline, uppercase, lowercase, T or U. 
Target RNA sequences will be transformed to uppercased DNA sequence. All downstream process will use this format. Sequences less than 30 bp will be rejected. After procession of input, a page with all user inputs will prompt for user to check and confirm. Rejected sequences will also be shown.
The program will set the first nucleotide of each target RNA as the start point, generate possible switch sequences for each 30, 31, 32, 33, 34 or 35 nucleotides (nt), which is called a window sequence.  
Since the whole switch sequence could be predicted by the initial window sequence, window sequence with following features will be rejected to ensure hairpin stability:
<UL><LI>Have no C or G at the neck of hairpin </LI><LI>Have in frame stop codon after AUG </LI><LI>Have more than four consecutive A, G, C or T </LI></UL></P><P style="font-family: roboto;font-size:115%;">
Start point will move to right for one nucleotide if the previous window sequence is rejected. Switch design process will continue if the window sequence passed the first examination.
</P><H4><U>Input parameters:</U></H4><UL><LI><B>Choose your cloning method (optional) </B><P style="font-family: roboto;font-size:100%;">
User can choose to add RE site or use our toehold switch cloning tool. </P></LI><LI><B>Choose Promoter (optional) </B><P style="font-family: roboto;font-size:100%;">
User can input a custom promoter or select a promoter from iGEM registry.</P></LI><LI><B>Input linker sequence </B><P style="font-family: roboto;font-size:100%;">
The linker is used to separate the coding sequence in the toehold switch and the reporter to prevent interference of protein folding. A default flexible linker (AACCUGGCGGCAGCGCAAAAG) is provided. </P></LI><LI><B>Choose ribosomal binding site </B><P style="font-family: roboto;font-size:100%;">
User can input a custom RBS or select a RBS from iGEM registry.</P></LI><LI><B>Input trigger length</B><P style="font-family: roboto;font-size:100%;">
The downstream 120nt (or user specified length) from the start site will be copied to be trigger sequence. </P></LI><LI><B>Check RFC</B><I>RFC standard check are specifically added for iGEMers. The program can help you to eliminate switches with illegal sites.</I></LI></UL><H4><U>Output option </U></H4><UL><LI><B>Counting rare codon</B><P style="font-family: roboto;font-size:100%;">
User can choose to count the occurrence of rare codon at the start of the CDS. Rare codon at the beginning of CDS may significantly hinder translation. </P></LI><LI><B>Minimal free energy (MFE) and MFE structure calculation</B><P style="font-family: roboto;font-size:100%;">
User can choose to output the MFE and MFE structure of this switch, switch dimer, window-switch dimer and RBS-linker. We used embedded function of <A href="https://www.tbi.univie.ac.at/RNA/">“Vienna RNA”</A> (1) to do the calculation.
</P></LI><LI><B>Base pair condition</B><P style="font-family: roboto;font-size:100%;">
The base pair condition of the toehold domain of switch sequence and the RBS domain in trigger-switch dimer will be counted, and switches with excessive domain pair will be discarded. A low number of base pairing at the toehold domain and RBS is preferred.
 </P></LI><LI><B>BLASTn</B><P style="font-family: roboto;font-size:100%;">
After all target RNA sequences are processed, human EST BLASTn will be done for every legal window sequences. The program will filter out any switch that detect a RNA that is highly similar to any human RNA. This function help to avoid non-specific activation of toehold switch that gives false- positive result when detecting external RNA in human.
 </P></LI></UL><H3>Output</H3><H4><U>
Excel output</U></H4><P style="font-family: roboto;font-size:115%;">
The output result will be represented in a excel file. For example: </P><P style="font-family: roboto;font-size:115%;">
Below list out the definition of terms used in the output excel. </P><P style="font-family: roboto;font-size:115%;">
Switches that failed to pass the filter will be listed out with explanation. For example:</P><H4><U> Email output </U></H4><P style="font-family: roboto;font-size:115%;">
User will receive result files by email. If the input is only one target RNA sequence without BLASTn function, the email with contain an excel file named by user specified name. If the input is multiple sequences or BLASTn is required, the email will contain a folder named by user specified name. Within the folder, both excel files for each target RNAs and BLASTn files will be named in digit numbers.
Result files can also be retrieved in user account. The upper limit of fiel number is fifty, including excel files and BLASTn files. 


</P><H3>Reference</H3>
1. Lorenz, Ronny and Bernhart, Stephan H. and Höner zu Siederdissen, Christian and Tafer, Hakim and Flamm, Christoph and Stadler, Peter F. and Hofacker, Ivo L. ViennaRNA Package 2.0. Algorithms for Molecular Biology, 6:1 26, 2011, 

</SECTION></DIV></DIV><FOOTER class="footer-distributed"><DIV class="footer-right">   <P> Email: igemcuhk1617@gmail.com </P></DIV></FOOTER></DIV></DIV></DIV></BODY></HTML>