<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Munich_lab_results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Munich/lab/results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Munich/lab/results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/lab/results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/lab/results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Munich/lab/results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Munich/lab/results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="navmenu"><DIV id="topics"><UL class="ultopic" id="navitems"><LI class="litopic ui-corner-all" id="liproject"><A class="navtopic" id="navproject" href="https://2011.igem.org/Team:TU_Munich/project/introduction">Project</A></LI><LI class="litopic ui-corner-all" id="lilab"><A class="navtopic" id="navlab" href="https://2011.igem.org/Team:TU_Munich/lab/results">Lab</A></LI><LI class="litopic ui-corner-all" id="limodel"><A class="navtopic" id="navmodel" href="https://2011.igem.org/Team:TU_Munich/model/modeloverview">Modeling</A></LI><LI class="litopic ui-corner-all" id="lihuman"><A class="navtopic" id="navhuman" href="https://2011.igem.org/Team:TU_Munich/human/comics">Human Practice</A></LI><LI class="litopic ui-corner-all" id="liteam"><A class="navtopic" id="navteam" href="https://2011.igem.org/Team:TU_Munich/team/teamoverview">Team</A></LI><LI class="litopic ui-corner-all" id="liextra"><A class="navtopic" id="navextra" href="https://2011.igem.org/Team:TU_Munich/extra/extraoverview">Extras</A></LI></UL></DIV></DIV><DIV id="social" class="content"><DIV class="subcontent ui-corner-all sublab" style="clear:both;"><H1>Results</H1><H2>Solid Matrix</H2><P>Our idea of a 3D-printer made it necessary to immobilize our cells inside a three dimensional matrix. We figured that the matrix needed to have the following properties:</P><OL><LI>Clear substance with good transmission of light to enable us to actuate the immobilized cells precisely with light.</LI><LI>Possibility to distribute the cells evenly within the matrix.</LI><LI>Environment that ensures the survival of the bacteria.</LI></OL><P>The idea to immobilize our bacteria in something like a solid growth medium (like agar-plates) was a step in the right direction, but standard LB-agar was not the solution. In the end, we chose GELRITE as a gelling agent,  “a highly-purified, natural anionic heteropolysaccharide that forms rigid, brittle, agar-like gels”  (source: gelrite spec sheet, see <A href="2011.igem.org/Team:TU_Munich/lab/notebook?selectedTab=#ui-tabs-1">labbook/methods</A>).</P><H4>Clear substance with good transmission of light:</H4><P>The matrix should be prepared with minimal media such as M9 instead of LB, because LB shows a strong absorption in the lower range of visible light (hence its yellow color). Minimal media are clear, which is a great advantage. 
The cell density in the matrix is also an important factor, because with increasing cell number the gel becomes blurry due to refraction of light (see diagrams).</P><H4>Distribution of cells and cell survival:</H4><P>To make sure that the cells are distributed evenly within the matrix, the bacteria need to be added to the medium before gelling occurs. The gelling temperature of GELRITE depends on both the amount of GELRITE powder and the salt concentration of the used liquid. A 0.5 % GELRITE gel prepared with M9 medium solidifies at around 46 °C. This means that the cells that shall be immobilized must survive such temperatures for a few minutes.  This is why we made some experiments with the heat resistant <I>E. coli</I> strain BM28 (kindly provided by Jeanette Winter). Fluorescence microscopy showed that the cells can survive the gelling process and produce GFP or RFP upon induction. The temperature seems to be the most critical factor regarding cell survival, as minimal media, with which the gel is prepared, provide enough nutrients for the bacteria.</P><H4>Diodes</H4><P>As light sources we chose diodes, which should emit the wavelengths we need for activation of our sensors.  As the blue light sensors peak absorbance is located at 465nm, [<A href="#r4">4</A>] we chose a diode with an emission maximum at 470 nm. For the red light sensor two distinct diodes are needed. Activation of the sensor occurs at 705nm [<A href="#r1">3</A>], therefore a diode which emits red light at 710nm was used. For the inactivation of the sensor the red light sensor shows a high absorption at 650nm, so we use a diode with an emission maximum of 650nm.</P><DIV style="float:left;width:420;padding:10px;"><P style="text-align:center">Solid: inactive state - Dotted: active state. Source: [<A href="#r1">1</A>]</P></DIV><DIV style="float:left;width:420;padding:10px;"><P style="text-align:center">R: inactive state FR: active state. Source: [<A href="#r2">2</A>]</P></DIV><DIV style="clear:both"><P> The colored overlay shows the emission spectra of the respective diodes. The peaks strongly coincide with the absorption bands of the respective sensors. Since the spectra of the 710nm and the 650nm diodes overlap at in a narrow band at very low intensity, we can expect not to have any cross-excitation of the sensors by the diodes.</P></DIV><DIV style="clear:both;"><H2>Cloning of constructs</H2></DIV><P>Cloning of AND-Gate and reporter constructs were successful and plasmids have been verified via sequencing after every cloning step. Those sequences were entered into the registry. The cloning scheme for our <A href="https://static.igem.org/mediawiki/2011/d/d2/Klonierungsschema_Plasmid_1.jpg" rel="lightbox" title="Scheme Plasmid 1">AND-Gate</A> and <A href="https://static.igem.org/mediawiki/2011/3/3a/Klonierungsschema_Plasmid_2.jpg" rel="lightbox" title="Scheme Plasmid 2">reporter</A> constructs can be seen on our <A href="https://2011.igem.org/Team:TU_Munich/project/design" title="Design">Design</A> page.</P><DIV style="width:420px;float:left;padding:10px;"><P>High Fidelity PCR was performed on <A href="http://partsregistry.org/Part:BBa_K322127">BBa_K322127</A> in order to amplify the ~3.9 kbp region of PcyA (Phycocyanobilin:ferredoxin oxidoreductase) gen and cph8 genes, as well as the OmpC promotor region. This PCR-product forms new part <A href="http://partsregistry.org/Part:BBa_K568000">BBa_K568000</A>.</P></DIV><DIV style="width:420px;float:left;padding:10px;"><P>A correct clone of our final optogenetical AND-Gate <A target="_blank" href="http://partsregistry.org/Part:BBa_K568001">BBa_K568001</A> construct was successfully sequenced with VR primer into first 1000 bp of T7ptag region. Testing of this construct according to our <A href="https://2011.igem.org/Team:TU_Munich/project/design" title="Design">Design</A> notes in CP919 cells with GFP reporter part <A target="_blank" href="http://partsregistry.org/Part:BBa_K568001">BBa_I746907</A> was not possible within timescale. 
</P></DIV><DIV style="width:420px;float:left;padding:10px;"><P>Size of our cloned reporter plasmid <A target="_blank" href="http://partsregistry.org/Part:BBa_K568003">BBa_K568003</A>, containing T7 promoter with RBS and lacZ is correct (T7promoter_RBS_lacZ approx. 3kbp pSB1AK8 approx. 3.4 kbp). One clone was verified via sequencing. This part was further characterized performing Miller Assays in BL21(DE3) cells. See &quot;characterization of parts&quot; or Registry entry for detailed information.</P></DIV><DIV style="width:420px;float:left;padding:10px;"><P>Lane 1 and 2 show the succesfull digest of our constitutive promoter with GFP on pSB1C3 (<A target="_blank" href="http://partsregistry.org/Part:BBa_K568007">BBa_K568007</A>). The size of the constitutive promoter with GFP is 831 bp and from pSB1C3 2069 bp. containing T7 promoter with RBS and lacZ is correct (T7promoter_RBS_lacZ approx. 3kbp pSB1AK8 approx. 3.4 kbp). One clone was verified via sequencing. This part was further characterized performing Miller Assays in BL21(DE3) cells Both sizes fit the bands on the gel.
</P></DIV><DIV style="width:420px;float:left;padding:10px;"><P>All lanes show the expected bands at 4572 bp (red light sensor) and 2070 bp (pSB1C3 backbone) of our construct (<A target="_blank" href="http://partsregistry.org/Part:BBa_K568008">BBa_K568008</A>). The clone was verified via sequencing. 
</P></DIV><DIV style="clear:both"><H2>Characterization of parts</H2></DIV><H3>Red Light Sensor</H3><H4>Red Light Sensor with lacZ K322127</H4><P> The red light sensor used is derived from <A target="_blank" href="http://partsregistry.org/Part:BBa_K322127">BBa_K322127</A>. We used this part in two ways: At first, we used PCR to amplify the parts needed for red light detection from this preexisting part, generating <A target="_blank" href="http://partsregistry.org/Part:BBa_K568000">BBa_K568000</A>. After that we compared the original part <A target="_blank" href="http://partsregistry.org/Part:BBa_K322127">BBa_K322127</A> and our new part <A target="_blank" href="http://partsregistry.org/Part:BBa_K568000">BBa_K568000</A>. To prove the proper function of these two red light sensor parts, we did several Miller Assays. Unfortunately, this part did not produce the results we hoped for. Further testing would be needed for debugging.</P><H4>Red light sensor with GFP</H4>The part <A target="_blank" href="http://partsregistry.org/Part:BBa_K568008">BBa_K568008</A>, red light sensor with GFP was build to test <A target="_blank" href="http://partsregistry.org/Part:BBa_K568000">BBa_K568000</A>, which we use as red light sensor in our optogenetical AND-Gate <A target="_blank" href="http://partsregistry.org/Part:BBa_K568001">BBa_K568001</A>. We tested its function in JT2. The test of the assay was repeated several times. Our results indicate that the part is independed of the different light conditions active.<H3>Blue Light Sensor</H3><P>The blue light promoter was fused to lacZ and expression of lacZ was measured wih the Miller Assay. The blue light promoter can be induced at 450 nm, which we accomplished with the appropriate LED. To fully characterize this part, cultures were grown at different conditions and their lacZ expression was measured.
Comparing samples grown at different lighting conditions showed that room light decreases the activity of the blue light promoter slightly, whereas blue light of 450 nm increases its activity. </P>As literature suggested a temperature dependency, we tested samples grown at room temperature (23 °C) and 37 °C, the usual incubation temperature for bacterial cultures. This proved that the blue light promoter yields higher expression rates at this lower temperature. Blue light is another input of a response pathway to cold stress and acts upon YcgF/YcgE. Under blue light the repressor YcgF binds to YcgE which in <I>E. coli</I> represses genes for biofilm formation, which then in turn detaches from its target. The underlying part for our construct is <A target="_blank" href="http://partsregistry.org/Part:BBa_K238013">BBa_K238013</A>, we added lacZ with a preceding RBS.To analyze the three parameters of interest, temperature, light conditions and incubation time, for our blue light sensor together with two controls, we set up a very extensive test. The results were not as clear as we hoped, however confirmed first tendencies.<H3>Reporter Plasmids</H3><H4>lacZ Reporter</H4><P>We tested the function of our reporter plasmid <A target="_blank" href="http://partsregistry.org/Part:BBa_K568003">BBa_K568003</A> using a Miller Assay. The part was transformed into BL21(DE3) cells, which are able to express T7 polymerase upon stimulation with IPTG. We were able to show that the expression level of β-galactosidase increases about one hour after induction, if compared to the control incubated without IPTG. The data indicates a leaky expression of β-galactosidase, which might be due to a leaky production of T7 polymerase by the host strain. An increase in Miller Units could be shown for all tested concentrations of IPTG between 0.01 mM and 1.5 mM, which seem to be quantitative until 90 min after stimulation. <SPAN style="color:#FFFFFF"> Congratulations! You have found our easter egg!</SPAN></P><P>We repeated this experiment, applying an additional lower concenctration of 0.01 mM IPTG. For protein expression, this is a value below the normally proposed concentration. However, the experiment shows that the β-galactosidase is still strongly induced. We also tried to establish a positive control, using lacZ regulated by a constitutive promotor. This did not yield the expected results. Our cloning had not worked as expected, which we found out by sequencing our construct. There is also an additional data set at 30 min, which is presented in our Wiki. It yielded too high β-galactosidase levels, which might be due to a systematic error during this measurement. Nevertheless, the data shows most significantly that our lacZ reporter plasmid works and is activated in a quantitative way.</P><H4>GFP Reporter</H4>In order to test our AND-Gate construct in CP919 cells, we need another reporter than lacZ due to genomic ompC-lacZ fusion present in this strain. Therefore Part <A target="_blank" href="http://partsregistry.org/Part:BBa_I746907">BBa_I746907</A> has successfully been tested for proper function. With saturating concentrations of IPTG the promoter was induced with t(1/2) of ca. 200 min.<H3>Positive control-constitutive promoter with GFP</H3>The part <A target="_blank" href="http://partsregistry.org/Part:BBa_K568007">BBa_K568007</A> is used as a positive control in our GFP assays. We tested its function in JT2.<DIV style="clear:both"><H4>References:</H4><DIV id="r1"><P>1. Sudarshan Rajagopal, Jason M. Key, Erin B. Purcell, David J. Boerema and Keith Moffat. Purification and Initial Characterization of a Putative Blue Light-regulated Phosphodiesterase from <I>Escherichia coli</I>. <I>Photochemistry and Photobiology</I>, 80, 2004.</P></DIV><DIV id="r2"><P>2. Geirgios Psakis, Jo Mailliet, Christina Lang Lotte Teufel, Lars-Oliver Essen and Jon Hughes. Signaling Kintetics of Cyanobacterial Phytochrome Cph1 a Light Regulated Histidine Kinase. <I>Biochemistry</I>.</P></DIV><DIV id="r3"><P>3. Jeffrey J. Tabor, Anselm Levskaya, and Christopher A. Voigt. Multichromatic control of
gene expression in escherichia coli. <I>Journal of Molecular Biology</I>, 405(2):315 – 324, 2011.</P></DIV><DIV id="r4"><P>4. Y. Nakasone and T. Ono and A. Ishii and S. Masuda and M. Terazima . Transitional Dimerization
and Conformation Change of a BLUF Protein YcgF. <I>Journal of the American Chemical Society</I>, 129(22):7028–7035, 2007.</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:TU_Munich/lab/results">http://2011.igem.org/Team:TU_Munich/lab/results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Munich/lab/results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Munich/lab/results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Munich/lab/results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Munich/lab/results&amp;oldid=258086" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>