<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Munich_project_design"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Munich/project/design">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Munich/project/design&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/project/design&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/project/design&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Munich/project/design" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Munich/project/design</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="navmenu"><DIV id="topics"><UL class="ultopic" id="navitems"><LI class="litopic ui-corner-all" id="liproject"><A class="navtopic" id="navproject" href="https://2011.igem.org/Team:TU_Munich/project/introduction">Project</A></LI><LI class="litopic ui-corner-all" id="lilab"><A class="navtopic" id="navlab" href="https://2011.igem.org/Team:TU_Munich/lab/results">Lab</A></LI><LI class="litopic ui-corner-all" id="limodel"><A class="navtopic" id="navmodel" href="https://2011.igem.org/Team:TU_Munich/model/modeloverview">Modeling</A></LI><LI class="litopic ui-corner-all" id="lihuman"><A class="navtopic" id="navhuman" href="https://2011.igem.org/Team:TU_Munich/human/comics">Human Practice</A></LI><LI class="litopic ui-corner-all" id="liteam"><A class="navtopic" id="navteam" href="https://2011.igem.org/Team:TU_Munich/team/teamoverview">Team</A></LI><LI class="litopic ui-corner-all" id="liextra"><A class="navtopic" id="navextra" href="https://2011.igem.org/Team:TU_Munich/extra/extraoverview">Extras</A></LI></UL></DIV></DIV><DIV id="design" class="content"><DIV class="subcontent ui-corner-all"><H1>Design</H1><P> You want to know about our project idea in detail? Then this is the right page for you! Here we describe the complete and detailed implementation of our ideas into one project and working parts. </P><H2>Plasmid design</H2><P><I>Optogenetical AND-gate construct (high copy):</I>
The basic idea of the logical gate we are using was developed at UCSF in the lab of Prof. Voigt [<A href="#r1">1</A>]. It is based on amber stop-codon suppression via the non-canonical supD tRNA. We designed the part as followed:
Our plasmid insert starts with the sequences (<A href="http://partsregistry.org/Part:BBa_K322123">BBa_K322123</A> and <A href="http://partsregistry.org/Part:BBa_I15010" target="_blank">BBa_I15010</A>) needed to produce the red light sensing chromophores. Downstream, we cloned the red light sensing promotor (<A href="http://partsregistry.org/Part:BBa_R0082" target="_blank">BBa_R0082</A>) in front of the supD tRNA (<A href="http://partsregistry.org/Part:BBa_K228001" target="_blank">BBa_K228001</A>). To stop the transcription directly behind the supD tRNA we put in a double terminator sequence (<A href="http://partsregistry.org/Part:BBa_B0015" target="_blank">BBa_B0015</A>) downstream. This is followed by a blue light sensing promotor (<A href="http://partsregistry.org/Part:BBa_K238013" target="_blank">BBa_K238013</A>). In order to allow ribosome-binding upstream of the next part we introduced a ribosome binding site (<A href="http://partsregistry.org/Part:BBa_J44001" target="_blank">BBa_J44001</A>) directly downstream of the blue light sensing promotor. The AND-gate construct is completed by a T7 polymerase with the amber stop codon mutation (<A href="http://partsregistry.org/Part:BBa_K228000" target="_blank">BBa_K228000</A>). Since all standard biobrick vectors include a termination sequence after the suffix there was no need to introduce a further termination sequence.</P><P><I>Reporter construct (low copy):</I>
Our second plasmid carries the reporter construct, which can be exchanged, depending on what kind of reporter system you want to use. For simple proof of principle we used lacZ (<A href="http://partsregistry.org/Part:BBa_I732017" target="_blank">BBa_I732017</A>) as reporter gene downstream of the T7 promotor (<A href="http://partsregistry.org/Part:BBa_I712074" target="_blank">BBa_I712074</A>). Since the used lacZ part already includes a ribosome-binding site (rbs), it is not necessary to add another rbs. For the same reason as mentioned under &quot;Optogenetical AND-gate construct&quot; we did not add a terminator sequence downstream of lacZ. As alternative we also used gfp behind the T7 promoter(<A href="http://partsregistry.org/Part:BBa_I746907" target="_blank">BBa_I746907</A>). This made the testing of the other parts easier due to an assay easier to handle than the Miller assay.</P><H2>Bacteria and block matrix</H2><P>We could obtain the heat resistant <I>E. coli</I> strain BM28 (derived via directed evolution of strain MG1655 zba::kan) [<A href="#r2">2</A>], which endure temperatures up to 50°C. This is very important to us since we have chosen a matrix named GELRITE to immobilize our bacteria. It can be penetrated by light with only little refraction and it contains a minimum of nutrient to enable growth and protein synthesis but it polymerizes at 46°C. To ensure an even distribution of the bacteria we need to add them into the liquid gel which in turn means that they need to endure more than 46°C for at least 2 or 3 minutes.</P><H2>How the final construct should work</H2><P>By addition of S-Gal to the GELRITE matrix a fully functional AND-gate should lead to dark spots/colonies when hit with blue and red light. For proof of principle, we also used gfp coupled to fluorescence detection.</P><P>Since the red light sensor is active in the ground state, one needs to shut down the signalling. This can be achieved by irradiation with light of 650 nm [<A href="#r3">3</A>]. 
Afterwards, bacteria get hit by both far red light (705 nm [<A href="#r3">3</A>]) and blue light (465 nm [<A href="#r4">4</A>]) beams. Cph8 consist of the extracellular membrane domain cph1 and the intracellular part envZ. The red light induces the autophosphorylation at the cytosolic site of cph8[<A href="#r5">5</A>]. This leads to phosphorylation of ompR [<A href="#r6">6</A>] which subsequently binds to the ompC promoter and enables transcription of the supD t-RNA. 

</P><P>When YcgF senses blue light it dimerizes and binds to the repressor YcgE. The formation of the YcgE-YcgF complex leads to the unbinding of YcgE from the YcgZ promoter which activates the transcription of T7ptag (T7 polymerase with the amber stop codon mutation). However, translation of this mRNA is only possible if enough supD tRNA is available. </P><P>This AND-gate should ensure that expression of T7ptag is only induced when both wavelengths hit the bacteria. Since the reporter gene is under the control of a T7 promoter, lacZ or gfp expression is only enabled when the generated T7 polymerase binds to the T7 promoter.</P><P>The red light signal transduction pathway is supposedly also an osmolarity sensor, mainly dependent on aspartate [<A href="#r7">7</A>] concentration, a substance which is present in LB as well as other common media for <I>E. coli</I>. Hence a strain with EnvZ knockout is desireable. For this, the strain CP919 (<A href="http://partsregistry.org/Part:BBa_V1012" target="_blank">BBa_V1012</A>) can be used.</P> CP919 has a kanamycin insertion mutation into the EnvZ genomic region, therefore no kanamycin vectors can be used. Furthermore, in CP919 an additional genomic fusion of ompC-lacZ is present, which makes the usage of a lacZ reporter inappropriate in our AND-Gate system.[<A href="#r3">3</A>]<P>For testing our construct in CP919 we therefore use the GFP reporter part <A href="http://partsregistry.org/Part:BBa_I746907">BBa_I746907</A>, which we subsequently cloned into the low copy origin of replication vector <A href="http://partsregistry.org/Part:pSB6A1">pSB6A1</A>. As both plasmids need to be electroporated into the same cells, two different origins of replications and antibiotica resistances have to be choosen. </P><P>
The desired bacteria for our final system is the heat resistant <I>E. coli</I> strain BH28, in which we aim to insert a knock-out in the EnvZ genomic region in a last step. 
</P><H2>Cloning Scheme:</H2><DIV style="clear:both"><H4>References:</H4><DIV id="r1"><P>1. J Christopher Anderson, Christopher A Voigt, and Adam P Arkin. Environmental signal
integration by a modular and gate. <I>Mol Syst Biol</I>, 3, 08 2007.</P></DIV><DIV id="r2"><P>2. Birgit Rudolph, Katharina M. Gebendorfer, Johannes Buchner, and Jeannette Winter. Evolution of escherichia coli for growth at high temperatures. <I>Journal of Biological Chemistry</I>,
285(25):19029–19034, 2010.</P></DIV><DIV id="r3"><P>3. Jeffrey J. Tabor, Anselm Levskaya, and Christopher A. Voigt. Multichromatic control of
gene expression in escherichia coli. <I>Journal of Molecular Biology</I>, 405(2):315 – 324, 2011.</P></DIV><DIV id="r4"><P>4. Y. Nakasone and T. Ono and A. Ishii and S. Masuda and M. Terazima . Transitional Dimerization
and Conformation Change of a BLUF Protein YcgF. <I>Journal of the American Chemical Society</I>, 129(22):7028–7035, 2007.</P></DIV><DIV id="r5"><P>5. Julia Rausenberger, Andrea Hussong, Stefan Kircher, Daniel Kirchenbauer, Jens Timmer, Ferenc Nagy, Eberhard Schäfer, and Christian Fleck. An integrative model for phytochrome b mediated photomorphogenesis: from protein dynamics to physiology. <I>PLoS One</I>, 5(5):e10721, 2010.</P></DIV><DIV id="r6"><P>6. Oleg A Igoshin, Rui Alves, and Michael A Savageau. Hysteretic and graded responses in
bacterial two-component signal transduction. <I>Mol Microbiol</I>, 68(5):1196–215, Jun 2008</P></DIV><DIV id="r7"><P>7. R Utsumi, RE Brissette, A Rampersaud, SA Forst, K Oosawa, and M Inouye. Activation
of bacterial porin gene expression by a chimeric signal transducer in response to aspartate.
<I>Science</I>, 245(4923):1246–1249, 1989.</P></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:TU_Munich/project/design">http://2011.igem.org/Team:TU_Munich/project/design</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Munich/project/design" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Munich/project/design" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Munich/project/design&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Munich/project/design&amp;oldid=256722" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>