<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Munich_project_introduction"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Munich/project/introduction">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Munich/project/introduction&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/project/introduction&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/project/introduction&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Munich/project/introduction" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Munich/project/introduction</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="navmenu"><DIV id="topics"><UL class="ultopic" id="navitems"><LI class="litopic ui-corner-all" id="liproject"><A class="navtopic" id="navproject" href="https://2011.igem.org/Team:TU_Munich/project/introduction">Project</A></LI><LI class="litopic ui-corner-all" id="lilab"><A class="navtopic" id="navlab" href="https://2011.igem.org/Team:TU_Munich/lab/results">Lab</A></LI><LI class="litopic ui-corner-all" id="limodel"><A class="navtopic" id="navmodel" href="https://2011.igem.org/Team:TU_Munich/model/modeloverview">Modeling</A></LI><LI class="litopic ui-corner-all" id="lihuman"><A class="navtopic" id="navhuman" href="https://2011.igem.org/Team:TU_Munich/human/comics">Human Practice</A></LI><LI class="litopic ui-corner-all" id="liteam"><A class="navtopic" id="navteam" href="https://2011.igem.org/Team:TU_Munich/team/teamoverview">Team</A></LI><LI class="litopic ui-corner-all" id="liextra"><A class="navtopic" id="navextra" href="https://2011.igem.org/Team:TU_Munich/extra/extraoverview">Extras</A></LI></UL></DIV></DIV><DIV id="social" class="content"><DIV class="subcontent ui-corner-all sublab" style="clear:both;"><H1>Introduction</H1><P>During brainstorming, we brought up a lot of creative solutions, which the world should have, but from our point of view, is yet still not ready for. Hence we devoted our attention to a more present concern: tissue-engineering. In this area of expertise, a three-dimensional scaffold for the cells to attach is, amongst other things, essential. Since the origin and construction of these matrices is still a problem, new solutions are required. Currently, most of the preparations work in layers (for bone and cartilage material) or use nano fibers and textile technologies to generate a scaffold [<A href="#r1">1</A>]. </P><P>When thinking about this problem, we thought of a quite new approach on building three dimensional structures. What if we could just print tissue, bones or other stuff? How could we do that?</P><P>One of us had the idea to try something like the subsurface engraving of glassblocks, but instead of breaking small glass strucutres we wanted the bacteria to produce certain products like color [<A href="#r2">2</A>]. So how can we activate our bacteria with laser, forcing them to express a protein without bursting into thousand tiny pieces?</P><P>Because of the seemingly limitless opportunities in optogenetics, it was chosen as Method of the Year 2010 by Nature Methods and is titled as one of the breakthroughs of the decade [<A href="#r3">3</A>]. The hallmark of optogenetics is the introduction of fast light-activated channels and enzymes that allow temporally precise manipulation of electrical and biochemical events in bacteria. One can use channels derived from bacteriorhodopsin, photosynthesis associated complexes, like phycobilines in algae (for our red light sensor) or use existing light sensory domains already in <I>E.coli</I> (like our blue light promoter). Thus optogenetics provides an elegant way for us to convert a light signal into protein expression.</P><P>When thinking more carefully about the idea, we knew that we would need to immobilize the bacteria somehow to make sure that we can target certain spots. For this reason we have chosen a matrix named gelrite which can be passed by light with only little refraction. In addition to this fact, our used M9-medium only contains a minimum of (potentially light-absorbant) nutrients to enable and regulate growth and thus protein synthesis.</P><H2>Optogenetical AND-Gate</H2><P>To achieve bacterial constructed three-dimensional structures by using light induced gene expression it was necessary to come up with a precise regulation mechanism. Our most convincing approach was to design a logical AND-gate which converts two inputs in one output. In our case, it has been suitable to use two different wavelengths as inputs to induce our output - gene expression. Only when a bacterium is hit by both wavelengths it expresses a protein, e.g. a coloured molecule, to generate a three-dimensional picture inside the gel block. Our AND-Gate is built upon light sensor systems developed and optimized by Edinburgh's iGEM-Team from 2010 and on recent results of the Voigt lab at UCSF [<A href="#r4">4</A>].</P><P>This logical gate developed at UCSF is based on an amber stop-codon suppression through the non-canonical tRNA supD. A light-sensitive promoter induces the expression of mRNA with a stop-codon suppression coding for a T7-polymerase, which can only be translated by ribosomes, if the correct amber tRNA is present. The tRNA is expressed by a second light-sensitive promoter susceptible to another wavelength. Only if both signals are present, the expression of a protein under the control of a T7-promoter takes place. We chose blue and red light promoter because their wavelengths differ significantly from each other. Because of this construction we should be able to induce expression at desired spots only.</P><H2>Light Sensory Domains</H2><P> We chose the promoter that is regulated by the YcgF/E system as blue light sensor. As red light sensor we used the Cph8/phycobilin domains with the OmpR promotor. In the beginning, we also thought about using ccaR/ccaS as green-light sensor (instead of either YcgF/E or Cph8). The reason we decided to use a combination of the red and blue light sensor systems, is the great difference between the absorbtion maxima of the light sensory domains. We expect that this assembly will be the one with the lowest amount of unspecific gene expression which could occur due to overlapping of the absorption spectrums of the sensory domains.


  </P><H5>Blue light Sensor</H5><P style="padding-left:10px;">The domains YcgE and YcgF are endogenously present in <I>Escherichia coli</I>. The domains are thought to regulate the biofilm formation when <I>E.coli</I> is exposed in an aquatic environment. Blue light induces the dimerization of YcgF that then directly bind to the repressor YcgE and releases the repressor from the operator. The expression of YcgE and YcgF and therefore as well the expression of the controlled gene is increased at low temperatures. </P><H5>Red light Sensor</H5><P style="padding-left:10px;">The red light sensing domain consists of two genes. The first gene encodes the information for the phycobilin synthesis. Phycobilin is a red light sensitive chromophore which is assembled in the light sensing domain of the cph8 transmembrane protein. The chromophore is essential for the functional light sensing activity. The other gene encodes the fusion protein cph8. Cph8 is composed of two original different encoded proteins. The light sensing extracellular subunit cph1 and the intracellular envZ part. EnvZ phosphorylates and activates the ompR-regulator. The phosphorylated ompR induces the gene expression of the following gene, in our case the SupD-tRNA.




</P><H2>Outlook</H2><P> Our vision beyond the proof of principle of this project could be to fixate bacteria inside the gelrite which could express collagen and hydroxyl apatite instead of a dye. Following we can generate bone material directly in the block without any further scaffolds. After protein synthesis and secretion one could simply peel off the gel and get a bone imagined and “drawn” with the light.</P><P>Since easy artificial production of human tissue and bones is still a difficult task, we think this approach could be a way to overcome certain problems in the field of tissue engineering. Assumed that all bacteria left over have been removed or at least destroyed. We would expect less immunrejection because the material does not show any surface antigens as other mammalian material does. Subsequently, the bone could be coated with cells of the recipient and a complete bone structure could have been created. </P><H4>References:</H4><DIV id="r1"><P>1. Linda G. Griffith and Gail Naughton. Tissue Engineering - Current Challenges and Expanding Opportunities. <I>SCIENCE, </I> Vol. 295, 2002.</P></DIV><DIV id="r2"><P>2. Keming Du and Peng Shi. Subsurface precision machining of glass substrates by innovative lasers. <I>Glass Sci. Technol.,</I> 76, No. 2, 2003  </P></DIV><DIV id="r3"><P>3. <I>Nature Methods</I> 8 (1, 2011).</P></DIV><DIV id="r4"><P>4. J Christopher Anderson, Christopher A Voigt, and Adam P Arkin. Environmental signal
integration by a modular and gate. <I>Mol Syst Biol</I>, 3, 08 2007.</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:TU_Munich/project/introduction">http://2011.igem.org/Team:TU_Munich/project/introduction</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Munich/project/introduction" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Munich/project/introduction" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Munich/project/introduction&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Munich/project/introduction&amp;oldid=254633" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>