<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Munich_lab_notebook_part3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Munich/lab/notebook/part3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Munich/lab/notebook/part3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/lab/notebook/part3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/lab/notebook/part3&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Munich/lab/notebook/part3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Munich/lab/notebook/part3</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="ui-corner-all subcontent tabcontent"><H1><SPAN class="mw-headline" id="Light_sensor_systems_and_AND-Gate_cloningPart_III_Alex.2FSimon.2FBea.2FThorsten">Cloning Part III + Results</SPAN></H1><P><B>People: Alex, Simon, Bea, Thorsten, Anna</B></P><H2><SPAN class="mw-headline" id="24-08-2011">24-08-2011</SPAN></H2><P><B>People: Alex, Simon, Bea, Thorsten</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H4><P>Three ligations in total were conducted. K238013 (S,P) was quick-ligated with lacZ (X,P), rbs (S,P) with T7Pol (X,P) and T7Prom (S,P) with lacZ (X,P). </P><H4><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H4><P>The ligations were transformed into DH5alpha. Additionally, K238013 (from23-08-2011) was transformed into DH5alpha. The ligation of K238013 with lacZ is plated onto Amp and Cm, rbs with T7pol is plated onto Amp, T7Prom with lacZ onto Kan and K238013 is plated onto Amp and Cm.</P></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><P>The two cultures of K322127 in DH5alpha incubated over night with red light (ca. 630 nm) were split. One half was further incubated with red light, while the other half was incubated with sun light and  light from a table-lamp over night. </P><P><I>Test of negative controls on S-Gal:</I> T7prom (Kan) transformed into DH5alpha and K322127 (Cm) transformed into BL21 were inoculated onto S-Gal plates with the respective antibiotics. The are incubated over night, and should not yield black colonies, as they both should not be able to express functional lacZ.</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><I>Expression test of reporter plasmid:</I> The cultures with ligation of T7prom and lacZ transformed into BL21 and induced with IPTG showed black colonies on the S-Gal plates. However, the cultures, which were NOT induced also showed black colonies of the same intensity. At the current point, we cannot sufficiently explain this.</P><H4><SPAN class="mw-headline" id="Red_light_sensor_assembly_.28Bea_and_Thorsten.29">Red light sensor assembly (Bea and Thorsten)</SPAN></H4><P><I>aim:</I></P><P>Ligation of red light sensor didn't work, yet. Therefore we ligated PCR product of BBa_K322127(X and P cut) into pSB3C5(S and P cut) to verify correct sites. Subsequent amplification and recutting of this part should lead to correct ligation with SupD. </P><P>In parallel, PCR was performed again in case of failed ligations with existent PCR-products.</P><P><I>Materials:</I></P><P>digest:</P><P>pSB3C5: S and P cut by Alex on 23.08.11, CAM resistance, 2,7 kbp vector backbone</P><P>PCR product of BBa_K322127 with primers BBa_K568000, derived from Biomers, sequence see below, 3,8 kbp
(Florian 6.7.11 labelled with &quot;K322127 PCR product aus gel ausgeschnitten&quot;)</P><P>pcr:</P><P>Template: BBa_K322127 (&quot;Edinburgh part&quot;) conc: 150 ng/µl</P><P>primers (synthesized by biomers, 29.6.11, stock: 100µM):</P><DL><DD>BBa_K568000_fwd: tatatctatatcgaattcgcggccgcttctagagtttacggctagctcagtcctaggta (59bp)</DD></DL><DL><DD>BBa_K568000_rev: cgtgccggcggctgcagcggccgctactagtaagtccattctccccaaaaatg (53bp)</DD></DL><P><I>Procedure</I></P><P>digest:</P><P>10 µl PCR Product was cut with 1 µl XbaI and 1 µl PstI in 5 µl NEB4-buffer (total volume 50 µl) for 2h at 37°C with subsequent heat inactivation at 80°C for 20 minutes. Cut PCR-product was loaded on a 1% Agarose Gel in TAE buffer and run at 120 V for 1 h.</P><P>PCR:</P><P>amplification with Taq PCR Kit (NEB) according to Florian at 5.7.11. No water control was run due to lacking remaining buffer volume.</P><DL><DD>5 µl 10x standard buffer</DD></DL><DL><DD>1 µl dNTP's</DD></DL><DL><DD>0,25 µl Tag DNA polymerase</DD></DL><DL><DD>0,5 µl primers, respectively</DD></DL><DL><DD>0,4 µl Template DNA</DD></DL><DL><DD>add up to 50 µl with sterile water</DD></DL><P>PCR-program:</P><DL><DD>see 5.7.11</DD></DL><P>
Bands were cut from Gel and DNA was isolated using Squeeze N Freeze (protocol see <A href="/wiki/index.php?title=Methods" title="Methods">Methods</A>)</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P>1% Agarose Gel of new PCR-product (line 3) and cut PCR-product (line2) with 2 log ladder (line 1)</P><P>cut PCR Fragment with correct size (3,8 kbp) was frozen after isolation.
PCR yielded some short products with variable length. Used PCR-Kit is not suitable for amplification of long fragments. Experiment will be repeated tomorrow with correct Kit.</P></DIV><H2><SPAN class="mw-headline" id="25-08-2011">25-08-2011</SPAN></H2><P><B>People: Alex, Simon, Bea, Thorsten</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Digestion">Digestion</SPAN></H4><P>R2 (red writing: MiniPrep iGEM R2 31.5.11) was digested with X,P and T7prom (black writing: 17.6.11 T7) with S,P. After heat-inactivation, GLP was added to the digested DNA. The samples were applied onto a 1 % Agarose gel. The samples, along with those from the PCR (see below), were applied as follows: 2lod DNA ladder (5 ul); R2 (20 ul); R2 (20 ul); T7prom (20 ul); T7 prom (20 ul); PCR cut; positive control; PCR approach 1; PCR approach 1; PCR approach 2; PCR approach 2; PCR approaches 1 and 2 pooled.
The gel was run for 1:30 h at 110 V. The following bands were expected:  3.1 kbp for lacZ with rbs and 2 kbp for vector pSB1A2. 3.4 kbp for T7prom in pSB1AK8. The bands are excised from the gel and the DNA is recovered using freeze 'n squeeze.</P><P>The original file before cutting of the bands seems to be corrupt.</P><H4><SPAN class="mw-headline" id="Ligation_2">Ligation</SPAN></H4><P>In the next step, K238013 (from 23-08-2011; cut with S,P) was ligated with rbs&amp;lacZ (=R2; X,P), rbs (from 22-08-2011; S,P) with T7pol (from 23-08-2011; X,P) and T7prom (S,P) with rbs&amp;lacZ (X,P). Each time, ligation was performed as normal and as quick-ligation. After that, the transformed cells were plated onto LB (Amp) plates and incubated over night.</P><H4><SPAN class="mw-headline" id="New_PCR_of_K322128_using_Phusion_High-Fidelity_PCR-Kit_.28NEB.2FFinnzymes.29">New PCR of K322128 using Phusion High-Fidelity PCR-Kit (NEB/Finnzymes)</SPAN></H4><P><B>Material</B></P><P>BBa_K322127 with primers BBa_K568000, derived from Biomers, 3,8 kbp
cut vector pSB1A2 digestion with X,P from Alex on 25.8.2011
(Florian 6.7.11 labelled with &quot;K322127 PCR product aus gel ausgeschnitten&quot;)</P><P><B>Procedure</B></P><PRE>using same protocol as Florian on 11.7 </PRE><P><I>PCR-Program:</I></P><P>1) Initial Denaturation: 98°C, 4'</P><P>2) 30 cycles of:</P><PRE>   Denat.: 98°C, 30 s
   Annealing: 60°C, 15 s
   Extension: 72°C, 2´ 30 s</PRE><P>3) Final extension: 72°C, 8'</P><P>4) Hold: 4°C, oo</P><P><I>samples</I></P><P>only 2 samples were prepared using each:</P><P>35,5µl of ddH2O
10 µl5x buffer
1 µl dNTPS
1 µl of template DNA K322127 (1,5ng/µl )
1 µl of Primer fwd (10µM)
1 µl of Primer rev (10µM)</P><P>only the 10kB positive control was done:</P><P>34µl of ddH2O
10 µl5x buffer
1 µl dNTPS
1 µl of template DNA (template for positive control)
2,5 µl of Primer (10kB-Mix)</P><P>
after pcr the samples were applied on a 1% Agarose-gel</P><P>The bands are excised from the gel and the DNA is recovered using freeze 'n squeeze.</P><P>two times 10µl of the PCR product were digested with X,P</P><P>the digestion and the pcr product were stored at -20°C</P><H4><SPAN class="mw-headline" id="Ethanol_precipitation_of_digested_PCR_product">Ethanol precipitation of digested PCR product</SPAN></H4><P>Material: PCR product cut with X,P from the 24.8.2011 
protocol see <A href="/wiki/index.php?title=Methods" title="Methods">Methods</A>
the sample was applied on an agarose gel because the nanodrop indicated no DNA</P><H4><SPAN class="mw-headline" id="Result">Result</SPAN></H4><P>The gel showed that no DNA was yielded (see picture Restriction &amp; Ligation)</P><H4><SPAN class="mw-headline" id="Repeated_Digest_and_Ligation_of_PCR_Product">Repeated Digest and Ligation of PCR Product</SPAN></H4><P>Material: PCR product of BBa_K322127 with primers BBa_K568000, derived from Biomers, 3,8 kbp
cut vector pSB1A2 digestion with X,P from Alex on 25.8.2011
(Florian 6.7.11 labelled with &quot;K322127 PCR product aus gel ausgeschnitten&quot;)
Procedure: 
The PCR product was digested with X,P. After heat-inactivation, GLP was added to the digested DNA. The samples were applied onto a 0,8 % Agarose gel. The gel was run for 1:30 h at 110 V.  
The bands are excised from the gel and the DNA is recovered using freeze 'n squeeze.
Ligation of the cut fragment with the vector pSB1A2 (cut with X,P; see Restriction &amp; Ligation above)
using quick ligation protocol see <A href="/wiki/index.php?title=Methods" title="Methods">Methods</A></P><H4><SPAN class="mw-headline" id="Result_2">Result</SPAN></H4></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="SDS-PAGE">SDS-PAGE</SPAN></H4><P>1 OD_600 each from the K322127 in DH5alpha cultures from yesterday was taken. The OD_600 were: 0.835 for the first colony exposed to bright lamp (bright 1); 0.84 for the second colony exposed to bright lamp (bright 2); 1.685 for the first colony incubated with red light (red 1); 1.855 for the second colony incubated with red light (red 2). The cells were centrifuged at 13000 rcf for 8 min. After resuspension in 100 ul loading buffer, the tubes were incubated for 10 min at 95 °C. After that, the samples (5 ul and 10 ul) were loaded onto a 10 % SDS-PAGE with stacking gel. Pageruler was used as protein ladder. The gel was run for 2:00 h at 80 V, which was increased to 120 V when the buffer front reaching the separation gel. After that, the gel was stained with Coomassie blue and subsequently destained.</P><P>The gel does not show elevated intensities of the band at around 10 kDa for lacZ alpha peptide, which would have been expected.</P><H4><SPAN class="mw-headline" id="T7_promoter_in_vitro_assay">T7 promoter in vitro assay</SPAN></H4><P>As proposed yesterday, a second in vitro assay was performed to find out if the plasmids contain a T7 promoter. This time, a third sample was made for each plasmid (same plasmids as yesterday, see above). In this sample (labeled &quot;#xi&quot;) the RNase that we assume to be present were inhibited using RNA secure prior to addition of T7 polymerase and incubation. See <A href="/wiki/index.php?title=Methods" title="Methods">methods</A> for detailed procedure. The 2% agarose gel was run at 110 V, 400 mA for 1:30 hours.</P></DIV><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><H4><SPAN class="mw-headline" id="K322127_Inoculation_.26_Incubation:">K322127 Inoculation &amp; Incubation:</SPAN></H4><P>The following inoculations were performed:</P><P>- &quot;K238013 in DH5 alpha&quot;, plate date: 24.08.2011, yesterday's transformation, in LB Amp</P><P>- &quot;BM28 from Jeanette Winter&quot;, plate date: 31.05.2011, in LB Amp, Kan</P><P>- &quot;green light 2&quot;, plate date: 27.6.2011, in LB Cm</P><P>- &quot;green light 3&quot;, plate date: 27.6.2011, in LB Cm</P><P>- &quot;green light 4&quot;, plate date: 27.6.2011, in LB Cm</P><P>- &quot;K322127 in DH5 alpha, Kolonie 1, Licht-induziert&quot;, plate date: 22.08.11, in LB cm (the picked colonies were the black ones after over night incubation in light)</P><P>- &quot;K322127 in DH5 alpha, Kolonie 2, Rotlicht-deaktiviert&quot;, plate date: 22.08.11, in LB cm (the picked colonies were white ones after over night incubation in red light)</P><P>Incubation over night at 37 °C, 200 rpm.</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P>The PCR was successful (see picture Restriction &amp; Ligation), the positive control and both samples yielded DNA that is located at the right spot according to their length
positive control (10kB)
samples (3,8kB)</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><I>24.08.11</I></P><P><I>T7 promoter in vitro assay</I></P><P>The samples labeled &quot;#xi&quot; were treated with RNA secure to inhibit potentially present RNase. The gel photo shows blurry signals in all of these lanes which aren't visible in lanes &quot;#x&quot; and &quot;#xa&quot;. This proves that RNase is present in the DNA samples used in the assay. Also, the fact that the RNA signals are quite large and strong shows leads us to the conclusion that the examined plasmids all contain a T7 promoter. Sample 1 contains two plasmids (about 6.6 kb and 3.5 kb) which is the reason for the bad sequencing data. Samples 2, 3, 5 and 6 contain one 6.6 kb plasmid each. This should be our correct reporter plasmid, part 2b, containing T7 promoter and lacZ. To verify this, a sample shall be sequenced.</P><P><I>S-Gal negative Control:</I></P><P>DH5alpha cells transformed with T7prom yielded white colonies on the S-Gal plates. However, BL21 cells transformed with K322127 yielded all black colonies. Most likely, our strain of BL21 is unsuited for expression tests using S-Gal (might constitutively express lacZ).</P><P><I>Expression test of K322127</I></P><P>After over night incubation, both plates incubated in red light showed white colonies only. Of the two plates incubated under the lamp, one did not show any colonies. The other one showed black colonies. 
--&gt; First evidence that part K322127 works as expected!!! To further quantify and characterise this: Miller assay? Other methods?</P></DIV><H2><SPAN class="mw-headline" id="26-08-2011">26-08-2011</SPAN></H2><P><B>People: Alex, Simon, Bea, Thorsten</B></P><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><P><B>Part-request:</B> Parts requested from the registry for backup if the ligation of the PCR product will fail again:</P><TABLE class="wikitable"><TBODY><TR><TH>Part	</TH><TH>Team  </TH><TH>Library  </TH><TH>Plate  </TH><TH>Well  </TH><TH>Plasmid  </TH><TH>Resistance</TH></TR><TR><TD>BBa_K322123 </TD><TD>iGEM10_Edindburgh  </TD><TD>2010 Submisions Samples 001  </TD><TD>Shipment: 00694 </TD><TD>4D  </TD><TD>pSB1C3   </TD><TD>C</TD></TR><TR><TD>BBa_K322124 </TD><TD>iGEM10_Edindburgh  </TD><TD>2010 Submisions Samples 001  </TD><TD>Shipment: 00694 </TD><TD>4E  </TD><TD>pSB1C3   </TD><TD>C</TD></TR></TBODY></TABLE><P>And Gate plasmid:
PCR-Product from Edinburgh</P><P><B>New media and plates:</B>
LB, LB for Agar plates (Kan and Amp?) and Luria medium were prepared. Plates were prepared.</P><P><B>Sequencing:</B></P><P>The reporter plasmid clone 5 was handed in for sequencing at GATC-Biotech. The primer used for sequencing is &quot;Primer pSB1A2 und pSB1AK8 fwd&quot;.</P><P><B>Skype talk with iGEM Team Freiburg</B></P></DIV><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><P><B>Miniprep &amp; Glycero-Stock:</B> The cultures inoculated yesterday on LB were frozen at -80 °C as glycero-stocks. The culture of K238013 (pSB1A2) in DH5alpha was prepped. The resulting DNA yielded a concentration of approximately 300 ng/ul.</P><P><B>Repetition of transformations:</B></P><P>The transformations from yesterday (ligation using ligation protocol) were repeated using DH5alpha cells (kindly provided by Andrea). Transformations were done for reporter plasmid (part 2b) colony 5 from DATUM, K238013 with lacZ, rbs with T7pol and T7prom with lacZ. After incubation for one hour, the cells were plated on LB Amp (K238013 with lacZ; rbs with T7pol) and on LB Kan (reporter plasmid; T7prom with lacZ) and incubated over night.</P><P><B>Ligation of new PCR product with pSB1A2</B></P><P><I>Material:</I></P><P>cut PCR (X,P) product from K322127 from PCR 25.8.2011 
cut pSB1A2 (X,P)  from Alex from 25.8.</P><P><I>procedure: </I></P><P>we purified PCR product using Zymo DNA Clean +Concentrator 25</P><P><I>ligation:</I></P><P>sample 1:  15µl PCR product  5µl vector
sample 2:  10µl PCR product 10µl vector</P><P><I>Transformation of the ligation and R2 as transformation control:</I></P><P>the two samples from purifiying and the part R2( OmpR in pSB1A2) as a positive control were transformed via electroperforation in competent DH5alpha cells that were kindly supplied by Andrea Mückl.</P></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><P><B>Testing K322127</B></P><P>Material: inoculated cultures of DH5alpha with K322127 from 25.8
Procedure:  100 µl of the culture was spread on S-gal plates with Kan and amp in dark
Incubation of the plates at room temperature (30°C) under light of diode with maximum of 630nm (turn-off diode) for 2 hours
afterwards incubation at light at 37°C for 19 hours afterwards 4°C in the incubator:
one culture with aluminium TUM logo put on top 
one culture in the box under light of turn-off diode (negative control)
one culture on top of the box, light from turn-off diode from below, light from the incubator from top only on cut out TUM logo in aluminium foil</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><I>26-08-2011</I></P><P><B>Transformations:</B> The transformations from yesterday, including K238013 &amp; lacZ, rbs &amp; T7pol, T7prom &amp; lacZ as well as PCR product &amp; Vector, did not work at all. There were no colonies on the plates.</P><P><B>Sequencing:</B> More data from the sequencing from 19-08-2011 arrived. The newly ligated K238013 &amp; double terminator from 17-08-2011 does contain the double terminator. However, as witnessed before, the blue light promotor K238013, which was part of the plasmid prior to ligation, is missing. We are still awaiting the data from another sequencing from the same construct, but from another ligation.</P><P><B>Planning for next week:</B> We need to prepare new electrocompetent DH5alpha cells. EVERYONE, who has free time to spare, please come to the lab and help. We might also be able to get new BL21 (DE3) cells from Prof. Buchner. If anyone wants to get them, contact Lars Mitschke at the Lehrstuhl of Prof. Buchner and talk to him about the request, which Alex asked about. BL21 (DE3) contain a T7polymerase, which can be induced by IPTG. This would be great use in order to test our reporter plasmid, i case the ligation worked (which we should know on Monday).</P></DIV><H2><SPAN class="mw-headline" id="29-08-2011">29-08-2011</SPAN></H2><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><P><B>inoculation</B>
the following were inoculated:
T7 Promotor with lacZ
2x K322127 in DH5alpha</P><H4><SPAN class="mw-headline" id="Retransformation_of_reporter_plasmid_clone_5"> Retransformation of reporter plasmid clone 5</SPAN></H4><P>the reporter plasmid clone 5  (118ng/µl) was diluted to 50 pg/µl and to 20 ng/µl</P><P>1 µl of each dilution was transformed into DH5alpha</P><H4><SPAN class="mw-headline" id="Transformation_of_RBS">Transformation of RBS</SPAN></H4><P>0,42 µl of RBS (47,5 ng/µl) were transformed into DH5alpha</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><I>26-08-2011</I></P><P><B>transformations</B></P><P>were only successful for R2, T7 Promotor with lacZ, reporter plasmid colony 5</P><P><B>testing of K322127</B></P><P>only very small colonies on the plate with no coloring
plate that was illuminated with turn-off light and light from the incubator dried out. no black colonies could be detected. Incubation of plate with TUM pattern was continued. Plate was irradated with a light bulb.</P><P><B>sequencing of reporter plasmid colony 5</B></P><P>Bases in squared brackets are optional and belonging to another sequence. Due tu overlapping base shifts exact assignment of bases is difficult</P><P>Sequencing reveals mixed clones. Correct Sequence with T7 promotor - RBS – lacZ is probably present beneath construct with T7 promotor lacking. Preparation of RBS-LacZ vector was contaminated with uncut or religated vector. Since plasmid length was ok, construct should be present.</P><P>next steps:</P><P>Retransformation with 50 pg and 20 ng MiniPrep DNA will be performed with subsequent sequencing of 4 clones.
In parallel new construct from Alex will be sequenced.</P></DIV><H2><SPAN class="mw-headline" id="30-08-2011">30-08-2011</SPAN></H2><P><B>People: Thorsten, Bea</B></P><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><I>29.8.11</I></P><P>Re-Transformation of part 2b clone 5 with 20 ng vector was succesful. 50 pg vector didnt yield any colonies. RBS was tranformed succesfully. 5 ml LB o/n cultures with apropriate antibiotica were inoculated and incubated at 37° C shaking. Mini Prep and sequencing will be performed at 31.8.11</P></DIV><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Electrocompetent_cells">Electrocompetent cells</SPAN></H4><DL><DD>new electrocompetent cells (DH5alpha) were made by the following protocol:</DD></DL><DL><DD>a 20 ml Luria medium overnight culture was inoculated with electrocompetent DH5alpha (kindly provided Andrea Mückl) yesterday</DD></DL><DL><DD>350 ml Luria medium were inoculated with 15 ml of the overnight culture and incubated at 37°C until a OD(600) of 0,7 was reached</DD></DL><DL><DD>the medium was distributed on falcons and the cells were centrifuged at 4500 rpm for 20 min at 4°C</DD></DL><DL><DD>after discarding the supernatant 350 ml of sterile ice-cold 10% glycerol was added and the pellets were resuspended using a pipet</DD></DL><DL><DD>the resuspension was centrifuged at 4500 rpm at 4°C for 11 minutes</DD></DL><DL><DD>after discarding the supernatant the cells were resuspended again in 350 ml of ice-cold 10% glycerol ( cells kept chilled all the time)</DD></DL><DL><DD>and again centrifuged at 4500 rpm at 4°C for 11 minutes</DD></DL><DL><DD>the supernatant was discarded and the cells were resuspended in the remaining supernatant</DD></DL><DL><DD>the cells were dispensed in 40 µl aliquots and stored at -80°C for further use</DD></DL><H4><SPAN class="mw-headline" id="Planning">Planning</SPAN></H4><P>In order to get ligations working, purification steps will be performed using Promegas Wizard PCR and Gel clean up Kit. 
We are going to ligate PCR product into pSB1C3 and upstream of SupD-tRNA. In parallel, T7ptag will be ligated into AND-Gate Plasmid, which will be pSB1C3. [First step of assembly]</P><P>Then, synthesis product, which contains parts from SupDtRNA to RBS J44001, will be ligated into succesful AND-Gate Plasmid-T7ptag. The last step will be ligation of PCR-product into this construct.</P><P>Excel sheets can be found in DropBox folder &quot;protocols&quot;.</P><P>planned parts</P><P>PCR product ligated into pSB1C3 will be a new part &quot;red light sensor without lacZ&quot;.</P><P>PCR product ligated with SupD will be a new part as well.</P><P>Synthesis product is another maybe not so useful part which will be send into the registry.</P><P>Reporter plasmid will be another Part. This one was called &quot;part 2B&quot; in recent cloning steps and needs further validation (see 29.8. retransformation and sequencing)</P><H4><SPAN class="mw-headline" id="Restriction_digest">Restriction digest</SPAN></H4><P>
All digestions were complete and running on the expected lengths.</P><H4><SPAN class="mw-headline" id="Purification_of_restricion_digests_from_gel_with_Wizard_Kit">Purification of restricion digests from gel with Wizard Kit</SPAN></H4><P>the following items were purified using the Wizard Kit according to manufacturer's instructions and the concentration was determined by nanodrop:</P><P>1: 24,5 ng/µl</P><P>2: 24,5ng/µl</P><P>3: 24,5 ng/µl</P><P>5: 20,5 ng/µl</P><P>9: 20,0 ng/µl</P><P>10: 23 ng/µl</P><P>11: 21,5ng/µl</P><P>12: 29,5 ng/µl</P><P>13: 20,0 ng/µl</P><P>for more details see restriction digest table</P><H4><SPAN class="mw-headline" id="Inoculation">Inoculation</SPAN></H4><P>the following overnight cultures were inoculated:</P><P>RBS
reporter plasmid clone 1-5
K322127 from plate from 24.8.2011</P></DIV><H2><SPAN class="mw-headline" id="31-08-2011">31-08-2011</SPAN></H2><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><I>30.8.11</I></P><P>Electrocompetent cells show good transformation efficiency. 100µl 10µl and 1µl of test transformation of 30 ng BBa_K322127 were plated on LB/cam plates. 1 µl yielded ca 400 colonies which refers to a tranformationefficiency of 1.3*10^7 cfu/µg BBa_K322127 in pSB1C3. 1*10^8 cfu/µg pUC19 is expected, so 1.3*10^7 cfu/µg vector with insert seems to be ok.</P><P>cph8 from Uppsala was succesfully transformed into DH5alpha. Cells have been 1:3 diluted and 20 ng plasmid was used. Output was ca 10^6 - 10^7 cfu/µg. Plate will be stored at 4°C til inoculation of o/n culture with subsequent minipreparation.</P></DIV><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Ligation_3">Ligation</SPAN></H4><H4><SPAN class="mw-headline" id="Minipreparation_of_DNA">Minipreparation of DNA</SPAN></H4><P>The following concentrations were gained through minipreparation:</P><P>
RBS                     :  86,5 ng/µl </P><P>reporter plasmid clone 1:  135  ng/µl </P><P>reporter plasmid clone 2:  234 ng/µl </P><P>reporter plasmid clone 3:  102 ng/µl </P><P>reporter plasmid clone 4:  119 ng/µl </P><P>reporter plasmid clone 5:    96 ng/µl</P><P>
The reporter plasmid clone 1-5 were sent to be sequenced at GATC.</P></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><P>the overnight culture of K322127 was used to inoculate a 100 ml culture in LB which was incubated for 5 hours at 37°C</P><P>at an OD of 1.6 50 ml of the cells were centrifuged at 4500 rpm the supernatant discarded </P><P>the cells were resuspended in remaining 1 ml of LB</P><P>100 ml of LB media with s-gal was prepared and autoclaved according to protocol see Methods</P><P>the resuspended cells were added to the medium when it reached 40°C</P><P>the media was given in thee plates</P><P>two plates were incubated at 37°C under turn-off light (diode max. 630 nm)</P><P>the TUM cookie cutters were inserted in one plate that was then incubated under the light of a lamp</P></DIV><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><P>new chloramphenicol plates were made</P></DIV><H2><SPAN class="mw-headline" id="01-09-2011">01-09-2011</SPAN></H2><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="gel_purification_of_30.8.11_digest">gel purification of 30.8.11 digest</SPAN></H4><P>20 µl of remaining digestions (made on 30.8.11) were loaded on 1 % agarose gel with 1xTAE. Items 1,2,3,4,5,6,7,8,9,10,12 were used.
This samples were cut using 350nm UV illumination as short as possible. Cut gel slices were subsequently purified 
using Promegas Wizard Kit according to manufacturers instructions.</P><P>
Determination of concentration with Nanodrop resultet in not reliable data. Therefore quantitative 0.7 % agarose gel was run with 5 µl of each sample. 5 µl 2 log ladder was used (100 µg/ml). PCR Band (3.8 kbp) in line 2 was estimated to 75 ng. This gives aproximately 15 ng/µl. Intensities of 2 kbp band in line 4 as well as lines 5, 6, 7, 10 and 11 correspond to approx. 150 ng amount (--&gt; 30 ng/µl).</P><P>
Due to fly-by-night intensity of ladder those concentrations will be determined again on 02.09.11 for further ligations. Ligations today will be done with the following concentrations</P><H4><SPAN class="mw-headline" id="o.2Fn_ligation_with_T4_Ligase">o/n ligation with T4 Ligase</SPAN></H4><P>
P, S opened Vector containing blue light promotor was ligated with RBS-LacZ insert. Due to huge insert lengths, insert:vector ratios were choosen as can be seen in ligation table. </P><P>
Ligation was conducted at 16° C in thermocycler o/n using 20 µl sample volume and 1 µl T4 DNA ligase. Examplary calculation of I/V ratio for 6x molar excess of insert (source: <A href="http://openwetware.org/wiki/DNA_Ligation" class="external free" rel="nofollow">http://openwetware.org/wiki/DNA_Ligation</A>):</P></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><P>inoculation of two cultures with 5 µl of the glycerol stock of K322127 in DH5alpha</P><P>the cultures were incubated at turn-off light (diode max. at 630 nm) at 37°C </P><P>new transformation of K322127 (150ng/µl) 0,5 µl were used</P></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="Preparing_lacZ-ONPG_or_also_called_Miller_Assay"> Preparing lacZ-ONPG or also called Miller Assay</SPAN></H4></DIV><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><H4><SPAN class="mw-headline" id="Part_design_in_the_registry">Part design in the registry</SPAN></H4><P>edit the general information of the part : 
<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K568000&amp;action=edit" class="external free" rel="nofollow">http://partsregistry.org/wiki/index.php?title=Part:BBa_K568000&amp;action=edit</A></P><H4><SPAN class="mw-headline" id="others">others</SPAN></H4><P>the primer for the sequencing of reporter plasmid clone 1-5 from yesterday is sent to GATC</P><P>Lars Mitschke kindly provided the E. coli strain BL21 (DE3), this strain contains a T7 Polymerase which can be induced with IPTG.</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><I>31-08-2011</I></P><P><B>transformations</B></P><P>no colonies on the plates for all ligations</P><P><B>testing of K322127</B></P><P>the plate that was incubated under light showed no coloring but the plate became intransparent which indicates that the bacteria grew</P><P>plates incubated under turn-off light (diode max. 630 nm) became intransparent as well;  this plates were used for further testing ( see testing of K322127 )</P></DIV><H2><SPAN class="mw-headline" id="02-09-2011">02-09-2011</SPAN></H2><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Digest_of_T7_promoter">Digest of T7 promoter</SPAN></H4><P>
Full volume of digest was loaded onto gel (50 µl digest + 10 µl 6 x Gel loading buffer)</P><P>
Band with correct size of 3,5 kbp was cut using 365 nm illumination. Illumination time was kept as short as possible, picture was taken after cutting gel slice.</P><H4><SPAN class="mw-headline" id="Gel_purification_of_T7_promotor">Gel purification of T7 promotor</SPAN></H4><P>Band was purified using Promega's Wizard Gel purification Kit according to manufacturer's instructions. Eluted DNA was subsequently quantified with an analytical 0.7 % agarose gel (120 V for 45 min) </P><P>
Band in line 2 corresponds to 120 ng band of marker. Therefore concentration of purified T7 promoter is approx. 24 ng/µl. RBS_lacZ band is approx. 20 ng/µl and pSB1C3 band approx. 16 ng/µl</P><H4><SPAN class="mw-headline" id="Ligation_of_Reporter_Plasmid">Ligation of Reporter Plasmid</SPAN></H4><P>digested T7 promoter was ligated with RBS_lacZ from 1.9.11 purification.</P><P>Ligation was conducted o/n at 16°C.</P></DIV><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><H4><SPAN class="mw-headline" id="Agar_Plates">Agar Plates</SPAN></H4><P>500 ml of media for ampcillin plates</P><P>250 ml of media for s-gal plates 
were prepared</P><H4><SPAN class="mw-headline" id="cloning">Cloning</SPAN></H4><H4><SPAN class="mw-headline" id="Transformation_2">Transformation</SPAN></H4><P>the ligations from 01-09-2011 (1-7) were transformed into DH5alpha
as a transformation control 0,5  of R2 (233ng/) was transformed into DH5alpha</P><H4><SPAN class="mw-headline" id="testing">Testing</SPAN></H4><H4><SPAN class="mw-headline" id="preparing_ONPG_assay">preparing ONPG assay</SPAN></H4><P>ONPG solution (4mg/ml)</P><P>1 M Na2CO3 were prepared</P><H4><SPAN class="mw-headline" id="Overnight_Cultures">Overnight Cultures</SPAN></H4><P>new overnight cultures of the new transformation of K322127 (see 01-09-2011) were inoculated</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><I>01-09-2011</I></P><P><B>Testing K322127</B></P><P>no coloring on plates</P></DIV><H2><SPAN class="mw-headline" id="03-09-2011">03-09-2011</SPAN></H2><P><B>People: Bea, Thorsten, Susan, Wolfgang</B></P><H3><SPAN class="mw-headline">Other work</SPAN></H3><DIV class="otherwork"><P>Cloramphenicol stock was produced: 30 mg/ml final concentration in 100% ethanol. 153 mg + 5,1 ml 100% EtOH. 500 µl aliquotes were stored at -20°C.</P></DIV><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Precipitation_of_Ligations_and_Transformation">Precipitation of Ligations and Transformation</SPAN></H4><P>Todays Transformation were done after purifing ligation mix with Glycogen/Ethanol precipitation in order to increase transformed amount and purity of ligated vectors.</P><P>All Ligations of 02-09-11 and following items of 01-09-11 ligation were used:</P><DL><DD>1 (pos control pSB1A3single cut (10))</DD></DL><DL><DD>2 (PCR(1)-pSB1C3 linearized (3))</DD></DL><DL><DD>4 (T7ptag(1)-pSB1C3 (12))</DD></DL><DL><DD>6 (blue light(8) - lacZ(9))</DD></DL><P>
Glycogen/ethanol precipitation:</P><DL><DD>-20 µl of the ligation mixed with 2 µl NaAc (3M), 0.5 µl glycogen and 22.5 µl isopropanol</DD></DL><DL><DD>-incubation step for 1 h @ -20 °C</DD></DL><DL><DD>-15 mins @ 10.000 rpms</DD></DL><DL><DD>-withdraw supernatant</DD></DL><DL><DD>-wash pelett with 70 % EtOH ~ 10 µl</DD></DL><DL><DD>-dry pelett and resuspense in 10 µl water</DD></DL><P>
Transformation via electroporation was done with the whole cleaned 10 µl in 40 µl DH5alpha cells.</P><H4><SPAN class="mw-headline" id="Ligations_of_PCR_product_into_pSB1C3_and_pSB1A3">Ligations of PCR product into pSB1C3 and pSB1A3</SPAN></H4><P>Following ligations were examined using an excess of vector and 1 µl PCR-product (corresponds to approx 14 ng/µl).</P><P>
Items 1+2 were incubated at room temperature and items 3+4 were incubated at 37° C  for 1h each. 5 µl of ligations were subsequently electroporated into 40 µl undiluted DH5alpha cells.</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><B>02-09-2011</B></P><H4><SPAN class="mw-headline" id="testing">Testing</SPAN></H4><P>no coloring on plates</P><H4><SPAN class="mw-headline" id="cloning">Cloning</SPAN></H4><P>Single cut vector control(1) showed approx. 50 colonies. ALl ligations plated on Cam plates (2-5) showed no colonies.</P><P>Blue light - RBS_lacZ ligations (6+7) showed 5 colonies overall. Clones were inoculated into 5 ml LB/Amp and incubated at 37°C o/n.</P></DIV><H2><SPAN class="mw-headline" id="04-09-2011">04-09-2011</SPAN></H2><P><B>People: Bea, Thorsten, Flo</B></P><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><B>03-09-2011</B></P><H4><SPAN class="mw-headline" id="Transformation_of_purified_ligations_.2802-09-2011.29">Transformation of purified ligations (02-09-2011)</SPAN></H4><P>positive control showed several thousand red colonies. All ligations showed bacterial lawn, including vector control. </P><P><SPAN style="color:red">Vector prep contains uncut or single cut vector. This result is consistent with sequencing results, which indicate mixed clones. Next steps: Preperation of 46 bp T7 promoter insert with 3 % preparative agarose gel and ligation upstream of RBS_lacZ into opened RBS_lacZ vector. In parallel vector prep of T7 promoter will be repeated on 1% agarose gel with a long gel and run. Uncut and single cut vector will be loaded on gel aswell.</SPAN></P><H4><SPAN class="mw-headline" id="Transformation_of_03-09-2011_ligations">Transformation of 03-09-2011 ligations</SPAN></H4><P>showed following output:</P><P>pSB1A3 colonies were possibly red. --&gt; incubated again o/n</P><P><A href="http://partsregistry.org/Part:BBa_J04450" class="external free" rel="nofollow">http://partsregistry.org/Part:BBa_J04450</A> --&gt; red color indicates religated or uncut vector clone</P><P>
4 clones of each successful transformation were picked and inoculated into 5 ml LB medium containing the appropriate antibiotica.</P><P>Conclusion:</P><P>Further incubations should be done at RT for 1h or o/n at 16°C with subsequent gylkogen/ethanol purification and transformation of whole 10 µl resuspended pellet into 40 µl DH5alpha</P></DIV><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Reporter_Plasmid">Reporter Plasmid</SPAN></H4><P>As recent transformation of reporter plasmid yielded high vector background new preps will be done according to following scheme.</P><H5><SPAN class="mw-headline" id="digest">digest</SPAN></H5><P>long 1% preperative agarose gel and 3% preperative agarose gel have been produced and stored in a plastic bag with some 1x TAE buffer at 4°C o/n. Purification will be done tomorrow.</P><H4><SPAN class="mw-headline" id="Minipreps_of_picked_clones_of_Bluelight_lacZ_01-09-11_ligation_.28items_6.2B7.29">Minipreps of picked clones of Bluelight_lacZ 01-09-11 ligation (items 6+7)</SPAN></H4><P>[Flo]</P><H4><SPAN class="mw-headline" id="Miniprep_of_5_o.2Fn_cultures_.5Bnames_below_protocol.5D_.28Metabion_Kit.29">Miniprep of 5 o/n cultures [names below protocol] (Metabion Kit)</SPAN></H4><P>All centrifugation steps were performed at 13.000 rpm and RT</P><P>1. Pellet 2 x 1,5 ml of bacterial suspension in <B>one</B> 2 ml-Eppi, centrifuge for 1 min each</P><P>2. Add 250 ul Bact.resuspension buffer, vortex</P><P>3. Add 250 ul Cell lysis buffer, invert 5x, incub. approx. 4 min</P><P>4. Add 350 ul DNA binding buffer, invert 5x, centrifuge for 10 min</P><P>5. Transfer clear lysate on column, centrifuge for 1 min, discard supernatant</P><P>6. Add 600 ul Column Wash Buffer, centrifuge for 1 min</P><P>7. Transfer column to new 1,5 ml-Eppi</P><P>8. Add 50 ul nuclease-free water (from Promega Wizard Kit), incub. approx. 2 min, centrifuge for 1 min</P><P>9. Store at -20°C</P><P>names:</P><P>-Blue Prom + lacZ #1 (= former &quot;blue lacZ Trafo 7, Klon 2&quot;)</P><P>-Blue Prom + lacZ #2 (= former &quot;blue lacZ Trafo 7, Klon 1&quot;)</P><P>-Blue Prom + lacZ #3 (= former &quot;blue lacZ Trafo 6, Klon 1&quot;)</P><P>-Cph8 in DH5a</P><P>-K322127</P><H4><SPAN class="mw-headline" id="Preparing_glycerol_stocks_from_cultures_used_above">Preparing glycerol stocks from cultures used above</SPAN></H4><P>Prepare 50% glycerol/LB solution by mixing 2 ml 100% glycerol and 2 ml LB </P><P>Add 300 ul of 50% glycerol/LB solution to 700 ul bacterial suspension to obtain an end concentration of 15% glycerol</P><P>Store at -80°C</P><H4><SPAN class="mw-headline" id="Restriction_digest_of_5_Minipreps_from_today">Restriction digest of 5 Minipreps from today</SPAN></H4><P>Each plasmid has been cut with EcoRI HF <B>AND</B> EcoRI HF + PstI HF (= 2 digests per plasmid) as follows:</P><P>-5 ul Plasmid DNA</P><P>-2 ul 10x NEB buffer 4</P><P>-0.5 ul EcoRI HF</P><P>-(0.5 ul PstI HF)</P><P>-12 (11.5) ul ddH2O</P><P>end volume is <B>20 ul</B></P><P>mix, spin down, incub. at 37°C for 1 hour, inactivate at 80°C for 20 min</P><P>Add 4 ul 6x loading dye</P><P>Store at -20°C until tomorrow</P><PRE>--&gt; quantification on gel 05-09-11</PRE></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="ONPG-Test_-_Miller_Assay">ONPG-Test - Miller Assay</SPAN></H4><P>The assay was conducted following the protocol:</P><P>3 samples of K322127 in DH5alpha were assayed:
1 sample was incubated for 25 hours in the dark at room temperature
1 sample was incubated for 25 hours in the light at room temperature
1 sample was incubated for 25 hours in the turn-off light at room temperature</P><DL><DD>- inoculation of 4 x 2ml LB with 10 μl of overnight cultures of K322127 (see 02-09-2011)</DD></DL><DL><DD>- light induction when the cultures reach OD of around</DD></DL><DL><DD>- control with only LB </DD></DL><P>start of Assay:</P><DL><DD>- measurement of Abs(600nm)</DD></DL><DL><DD>- new eppi with 1 ml of Zbuffer + 20 μl of freshly prepared 0,1% SDS+ 40 μl of Chloroform (under fume hood)</DD></DL><DL><DD>- 100 μl of the samples are taken and transferred to the Zbuffer</DD></DL><DL><DD>- mix the solution by vortexing for 10 s</DD></DL><DL><DD>- let the chloroform settle to the ground and take  700 μl of supernatant for measurement</DD></DL><DL><DD>- initiation of assay with 140μl of ONPG (4mg/ml) NOTE START TIME</DD></DL><DL><DD>(in the paper they use 100 μl supernatant plus 20 μl of ( 4mg/ml but we need more for measurement in cuvette)</DD></DL><DL><DD>-incubation at room temperature until yellow color develops</DD></DL><DL><DD>-stop reaction with 350 μl of 1 M Na2CO3 NOTE STOP TIME  (we upscaled the paper uses 50 μl)</DD></DL><DL><DD>- measurement of Abs (420nm) and Abs(550nm)</DD></DL><P>Abs 420 nm should be less than 1 and more than 0,5</P><H4><SPAN class="mw-headline" id="Result_3">Result</SPAN></H4><P>incubation for 146 min because no visible yellow color developed;</P><P>not completely clear how to calculate the miller units:</P><P>here miller units are calculated as following:</P><P>units= 1000* (Abs 420-1,75 Abs550) /(time(min)*Abs 600/(11,6*1,7)))</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><H4><SPAN class="mw-headline" id="S-Gal_plates">S-Gal plates</SPAN></H4><P>Flo did Minipreps of cph8 from Uppsala and K322127.
Nanodrop yielded no reliable data. --&gt; quantification on gel</P></DIV><H2><SPAN class="mw-headline" id="05-09-2011">05-09-2011</SPAN></H2><P><B>People: Bea, Thorsten, Anna</B></P><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="Miller_Assay">Miller Assay</SPAN></H4><P>5 newly inoculated cultures were assayed:</P><P>after 5 hours of incubation the inducation was started and the first assay started:</P><P>induction start means:
incubation at room temperature of:
K322127 in DH5alpha in the light of lamp
K322127 in DH5alpha in the dark
K322127 in DH5alpha in the turn-off light
K322127 in BL21in the light
cpH8 in DH5 alpha in the light</P><P>a new protocol was used for this assay:</P><P>
start of Assay:
- measurement of Abs(600nm)</P><P>- new eppi with 0,5 ml of Zbuffer + 20 μl of freshly prepared 0,1% SDS+ 40 μl of Chloroform (under fume hood)</P><P>- 0,5 ml of the samples are taken and transferred to the Zbuffer</P><P>- mix the solution by vortexing for 10 s ( all samples equal vortexing time)</P><P>- initiation of assay with 200μl of ONPG (4mg/ml) NOTE START TIME</P><P>-incubation at room temperature for see table</P><P>-stop reaction with 500 μl of 1 M Na2CO3 NOTE STOP TIME  </P><P>- measurement of Abs (420nm) and Abs(550nm)</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Analytical_gel_of_Bluelight_lacZ.2C_cph8_from_Uppsala_and_K322127_testdigest">Analytical gel of Bluelight_lacZ, cph8 from Uppsala and K322127 testdigest</SPAN></H4><P>
Size of bands look fine. Expected: 3093bp(lacZ)+86bp(Bluelight)= 3179 bp  </P><P>--&gt; Send clone 2 for sequencing or load RBS_lacZ digest along with ligated construct on gel and check for 86 bp difference.</P><P>cph8(2,2kbp) and K322127(4kbp) are running on their correct sizes.</P><H4><SPAN class="mw-headline" id="Miniprep_and_testdigest_of_picked_colonies_on_04-09-2011">Miniprep and testdigest of picked colonies on 04-09-2011</SPAN></H4><P>Miniprep of yesterdays picked clones were done. Concentration was determined using Nanodrop. </P><P>T7 Pol in pSB1C3 (ligation 4 from 1.9.)samples:</P><P>clone 1 : 58,5 ng/µl   (sample 1)</P><P>clone 2 : 60,5 ng/µl   (sample 2)</P><P>clone 3 : 55,5 ng/µl   (sample 10)</P><P>clone 4 : 56,5 ng/µl   (sample 13)</P><P>PCR in linearized pSB1C3:</P><P>clone 1 : 44,0 ng/µl   (sample 4)</P><P>clone 2 : 82,0 ng/µl   (sample 7)</P><P>clone 3 : 77,5 ng/µl   (sample 5)</P><P>clone 4 : 70,0 ng/µl   (sample 14)</P><P>blue light promotor+ lacZ (ligation 6 from 1.9):</P><P>clone 1 : 79,0 ng/µl   (sample 12)</P><P>clone 2 : 67,0 ng/µl   (sample 9)</P><P>clone 3 : 55,5 ng/µl   (sample 21)</P><P>clone 4 : 65,0 ng/µl   (sample 19)</P><P>red light sensor in pSB1C3 ratio 1:3</P><P>clone 1 : 58,0 ng/µl   (sample 11)</P><P>clone 2 : 52,5 ng/µl   (sample 15)</P><P>clone 3 : 59,5 ng/µl   (sample 8)</P><P>red light sensor in pSB1C3 ratio 1:6</P><P>clone 1 : 50,5 ng/µl   (sample 18)</P><P>clone 2 : 58,5 ng/µl   (sample 3)</P><P>clone 3 : 61,5 ng/µl   (sample 6)</P><P>red light sensor in pSB1A3 (ligation 1:5 (from 1.9.)
clone 1 : 52,5 ng/µl   (sample 20)(red colony)
clone 2 : 51,0 ng/µl   (sample 16)</P><P>red light sensor in pSB1A3 (ligation 1:1 (from 3.9) (red colony):
clone 1 : 40,5 ng/µl  (sample 17)</P><P>All samples were digested except 9,21,19:</P><P>5 µl of Mini-DNA was cut using 0,5 µl E and 0,5 µl P in 2 µl NEB4 and 20 µl total volume. Digest was loaded onto 1 % agarose gel.</P><P>
Red light sensor constructs look fine (3,8kbp insert and 2 kbp vector). T7ptag constructs are wrong and will be repeated (T7ptag expected at 2,6 kbp). Blue light sensor looks fine (3kbp). Red light sensor cloning into pSB1A3 didnt yield any insertion products, mainly slightly red colonies were visible --&gt; 1kbp insert is RFP cassette. One not-red clone could be picked (sample 15), which contains a 400 bp insert of unkown origin. </P><P>--&gt; Sample 7 was choosen to be sequenced as it yielded most amount of DNA.</P><H4><SPAN class="mw-headline" id="Reporter_Plasmid:_T7_promotor_prep_and_new_RBS_LacZ_prep">Reporter Plasmid: T7 promotor prep and new RBS_LacZ prep</SPAN></H4><P>3 % agarose gel from yesterday was used at 100V for 1,5h</P><P>
No band could be detected. Nevertheless, gel slice was cut and will be purified.</P><P>
Bands were cut and purified using Wizard Kit according to manufacturer's instructions.</P><P>Concentrations was determined using Nanodrop:</P><P>T7prom E, S: 9 ng/µl</P><P>T7prom P, S: 11.5 ng/µl</P><P>RBS_lacZ E, X: 14 ng/µl</P><P>RBS_lacZ E, X: 14.5 ng/µl</P></DIV><H2><SPAN class="mw-headline" id="06-09-2011">06-09-2011</SPAN></H2><P><B>People: Thorsten, Anna, Katharina</B></P><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="Miller_Assay_2">Miller Assay</SPAN></H4><P>5 newly inoculated cultures were assayed:
difference to day before: grow in the dark (only DH5alpha)</P><P>incubation at room temperature of:</P><P>K322127 in DH5alpha in the light of lamp</P><P>K322127 in DH5alpha in the dark</P><P>K322127 in DH5alpha in the turn-off light</P><P>K322127 in BL21 in the light</P><P>cpH8 in DH5 alpha in the light</P><P>Problem 1: OD of bacterial cultures was very high, should have been around 0,7.</P><P>Problem 2: too long incubation of color reaction yielded too high ODs, samples couldn't be diluted as more blank sample (as diluent) was missing.</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Reporter_Plasmid_2">Reporter Plasmid</SPAN></H4><H5><SPAN class="mw-headline" id="Purification_of_05-09-11_gel_runs">Purification of 05-09-11 gel runs</SPAN></H5><P>Frozen gel slices were purified using Promega's Wizard Kit according to manufacturers instructions.</P><P>Subsequent determination of concentration via Nanodrop:</P><P>1: T7prom 46 bp cut: 9 ng/µl</P><P>2: T7prom vector cut: 11.5 ng/µl</P><P>3: RBS_lacZ vector cut: 14.0 ng/µl</P><P>4: RBS_lacZ insert cut: 14.5 ng/µl</P><H5><SPAN class="mw-headline" id="Ligation_of_purified_samples:_Reporter_Plasmid_and_T7ptag_into_shipping_vector_pSB1C3">Ligation of purified samples: Reporter Plasmid and T7ptag into shipping vector pSB1C3</SPAN></H5><P>
Ligation mix was incubated for 1h at RT and and then at 16°C o/n.</P></DIV><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><H4><SPAN class="mw-headline" id="Sequencing_of_clones">Sequencing of clones</SPAN></H4><P>Primer overview: <A href="http://partsregistry.org/Primers/Catalog" class="external free" rel="nofollow">http://partsregistry.org/Primers/Catalog</A></P><P>
For most biobrick plasmids the standard sequencing primers VF2 (verification forward) and VR (verification reverse) will fit. They will be synthesized at GATC and stored for 1 year. </P><P>
10 µl of Following clones were send for sequencing 300-100 ng/µl with VF2 primer (tgccacctgacgtctaagaa):</P><P>
Bluelight_lacZ in pSB1A2 110904_clone2 miniprep conc.: 49.5 ng/µl</P><P>
red light in pSB1C3 110905_sample 7 conc.: 82 ng/µl diluted 1:1 with ddH2O</P></DIV><H2><SPAN class="mw-headline" id="07-09-2011">07-09-2011</SPAN></H2><P><B>People: Thorsten, Anna, Katharina</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Reporter_Plasmid_and_T7ptag_into_pSB1C3">Reporter Plasmid and T7ptag into pSB1C3</SPAN></H4><H5><SPAN class="mw-headline" id="Glycogen.2Fethanol_precipitation_and_transformation">Glycogen/ethanol precipitation and transformation</SPAN></H5><P>- DNA precipitation with glycogen:of the ligation samples from 6-9-11</P><P>1. add 2 µl 3 M sodium acetate, 0.5 µl glycogen and 22.5 µl isopropanol to 20 µl ligation product </P><P>2. incubate the mixture at -20°C for one hour</P><P>3. centrifuge the mixture for 15 minutes at 18,000 rpm, 4°C</P><P>4. discard the supernatant</P><P>5. add 50 µl 70 % ethanol</P><P>6. centrifuge the mixture for another 10 minutes at 18,000 rpm, 4°C</P><P>7. remove supernatant with a pipet CAREFULLY. If pellet moves, centrifuge again.</P><P>8. air-dry the pellet until the ethanol is completely evaporated. Do not over-dry pellet as this makes dissolving more difficult.</P><P>9. resuspend the pellet in 10 µl nuclease-free water by pipetting carefully.</P><UL><LI>10 µl purified ligation was mixed with 40 µl competent cells and electroporated into DH5alpha. </LI><LI>1 ml SOC medium (SOB medium + 20 mM glucose) was used and incubation was carried out for 1h at 37°C, shaking at 500 rpm in 2 ml eppis.</LI><LI>Culture was then centrifuged at 4000 rpm for 5 minutes, and resolved in 100 µl medium. </LI><LI>Whole sample was plated onto LB/agar plates supplemented with appropriate antibiotic and incubated o/n @ 37°C</LI></UL><H4><SPAN class="mw-headline" id="Synthetic_construct">Synthetic construct</SPAN></H4><H5><SPAN class="mw-headline" id="Digest_of_synthetic_construct_and_PCR-product_.28in_pSB1C3.29">Digest of synthetic construct and PCR-product (in pSB1C3)</SPAN></H5><P>
Samples were loaded on preparative gel (1 %, TAE 1x, 100 V, 2 h 30 min) and subsequently cut with a new scalpel for each sample using 365 nm illumination as short as possible. Gel slices were purified using Promega's Wizard Kit according to manufacturer's instructions. Picture was taken after cutting of gel slices in order to reduce illumination.</P><H5><SPAN class="mw-headline" id="ligation_of_synthetic_construct_and_PCR-product_.28in_pSB1C3.29">ligation of synthetic construct and PCR-product (in pSB1C3)</SPAN></H5><P>
Ligation was performed at 16°C o/n</P></DIV><H2><SPAN class="mw-headline" id="08-09-2011">08-09-2011</SPAN></H2><P><B>People: Anna, Katharina</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="AND-Gate_plasmid">AND-Gate plasmid</SPAN></H4><P>- DNA precipitation with glycogen/ethanol of the ligation samples from 7-9-11 accroding to protocol 7-9-11</P><P>
- transformation via electroporation of 40 µl electro competent DH5alpha cells with the full volume of purified ligation samples from 7-9-11</P><H4><SPAN class="mw-headline" id="Reporter_plasmid_and_T7ptag">Reporter plasmid and T7ptag</SPAN></H4><P>-overnight cultures for mini prep of transformations from 7-9-2011: </P><P>four clones of plates:</P><P>2 (T7prom(2)_RBS_lacZ(4))</P><P>3 (T7prom(2)_RBS_lacZ(4))</P><P>5 (RBS_lacZ(3)_T7prom(1))</P><P>6 (pSB1C3(3)_T7pol(5))</P><P>7 (pSB1C3(3)_T7pol(5))</P><P>
one clone of plate 13 (synthetic construct in delivery vector pMA-RQ)</P></DIV><H3><SPAN class="mw-headline">Other work</SPAN></H3><DIV class="otherwork"><P>- production of LB-Agar plates with Kanamycin (500 ml), Chloramphenicol (300 ml) and Ampicillin (500 ml)</P><P>- overnight cultures of K322127 in DH5alpha (Cam, in the dark) and cpH8 inDH5alpha (Kan) for Miller Assay the next day</P></DIV><H2><SPAN class="mw-headline" id="09-09-2011">09-09-2011</SPAN></H2><P><B>People: Anna, Katharina, Thorsten</B></P><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P>Std Primer VR seems to misprime to B0015 double terminator: <A href="http://partsregistry.org/Help:Primers/Problems_using_VF2_and_VR_to_amplify_BioBrick_parts" class="external free" rel="nofollow">http://partsregistry.org/Help:Primers/Problems_using_VF2_and_VR_to_amplify_BioBrick_parts</A></P><P>Could be a problem when sequencing into our construct from downstream primer binding site as we use B0015 in synthetic construct. Because sequencing was done succesfully using this primer by others, we'll try it keeping the mispriming in mind. --&gt; B0014 would be a better terminator for next time. </P><P>pSB1C3 is using B0056 as terminator downstream of suffix.</P></DIV><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Reporter_plasmid_and_T7_polymerase">Reporter plasmid and T7 polymerase</SPAN></H4><P>- Plasmid isolation -&gt; Miniprep of the over night cultures from 8-9-11 according to manufacturers instructions from Metabion</P><P>
- measurement of the DNA concentration of the following samples:</P><P>
2a: 92 ng/µl</P><P>2b: 122 ng/µl</P><P>2c: 131 ng/µl</P><P>3a: 98 ng/µl</P><P>3b: 152 ng/µl</P><P>3c: 124 ng/µl</P><P>3d: 112 ng/µl</P><P>5a: 52.5 ng/µl</P><P>5b: 55 ng/µl</P><P>5c: 74.5 ng/µl</P><P>5d: 87 ng/µl</P><P>6a: 134 ng/µl</P><P>6b: 118 ng/µl</P><P>6c: 101 ng/µl</P><P>6d: 140 ng/µl</P><P>7a: 76.5 ng/µl</P><P>7b: 113 ng/µl</P><P>7c: 83 ng/µl</P><P>7d: 71.5 ng/µl</P><P>13: 79 ng/µl</P><P>
- control restriction digest of the 21 DNA samples (2a-d, 3a-d, 5a-d, 6a-d, 7a-d,13) with the restricition enzymes E,P</P><P>total volume 50 µl (39 µl nf H20, 5 µl DNA, 5 µl NEB4 Buffer, 0,5 µl of each enzyme)</P><P>incubation for 1 hour at 37°C, inactivation for 20 min at 80°C</P><P>
- analytical gelelectrophoresis ( 1%, 1x TAE, 12 wells, 120 V, 1 hour 30 minutes)</P><P>50 µl DNA + 10 µl loading buffer -&gt; 40 µl sample in each well</P><P>
Size of T7promotor with RBS_lacZ looks correct (RBS_lacZ approx 3kbp pSB1AK8 approx 3.4 kbp). Ligation of 46 bp T7promotor into pSB1A2_RBS_lacZ (ID 5) didnt yield any correct clones, only vector background could be picked. Samples were discarded. Transformation of sample 4 resulted in short-circuit and was lost. Ligation of T7ptag into pSB1C3 looks fine. (T7ptag approx 2.6 kbp)</P><H4><SPAN class="mw-headline" id="AND-Gate_plasmid_2">AND-Gate plasmid</SPAN></H4><P>transformation output: vector background looks fine</P><P>following clones have been picked:</P></DIV><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><H4><SPAN class="mw-headline" id="Miller_Assay_3">Next steps</SPAN></H4><P>Besides ligation of T7ptag insert, pSB1C3_T7ptag vector will be opened to ligate with red light_synthetic construct insert--&gt; see 10-09-11.</P><P>Send clones for sequencing on Monday.</P></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="Miller_Assay_3">Miller Assay</SPAN></H4><P>two cultures in LB medium: </P><P><B>K322127 in DH5alpha</B> with three incubation conditions: <B>lighted, dark and under off-light</B></P><P>cpH8 in DH5 alpha as a control</P><P>conditions of bacterial growth: 37 °C, shaken, culture K322127 in DH5alpha <B>grown in the dark</B> prior to the experiment</P><P>volume of bacterial suspension used: <B>500 µl</B></P><P>incubation time for Miller Assay: <B>10 min</B> (37 °C)</P><P>
improvement to prior experiments: start OD between 0,15 and 0,17, correct antibiotics applied, incubation time of Miller Assay only 10 min.</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><H2><SPAN class="mw-headline" id="10-09-2011">10-09-2011</SPAN></H2><P><B>People: Susan, Wolfgang, Thorsten</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="AND-Gate_plasmid_3">AND-Gate plasmid</SPAN></H4><H4><SPAN class="mw-headline" id="red_light_sensor_-_synthetic_construct_ligation_check">red_light sensor - synthetic construct ligation check</SPAN></H4><P>- Plasmid isolation -&gt; Miniprep of the over night cultures from 09-09-11 according to manufacturers instructions from Metabion. 4 ml culture was used and 1 ml culture was saved for potential glycerol stocks</P><P>
- control restriction digest of the 21 DNA samples (2a-d, 3a-d, 5a-d, 6a-d, 7a-d,13) with the restricition enzymes E,S</P><P>total volume 10 µl (7 µl nf H20, 1 µl DNA, 1 µl NEB4 Buffer, 1 µl of each enzyme)</P><P>nuke for 1 minute at 800 W (microwave) </P><P>- analytical gelelectrophoresis ( 0.7%, 1 x TAE, 12 wells, 160 V, 20 minutes)</P><P>10 µl DNA + 2 µl loading buffer -&gt; 12 µl sample in each well</P><P>
Sample ID 3c was choosen to be used for further cloning. pSB1C3 with red light sensor and synthetic construct</P><P>Nanodrop derived DNA conc. of 3c:  114 ng/µl</P><P>samples 2, 3a, 4b and 6a were discarded</P><P>sample 7b was choosen to be synthetic construct in pSB1C3 --&gt; sequencing on monday</P><H5><SPAN class="mw-headline" id="last_step:_assembly_of_redlight_sensor_synthetic_construct_in_pSB1C3_with_T7ptag">last step: assembly of redlight sensor_synthetic construct in pSB1C3 with T7ptag</SPAN></H5><P>we took 2 approaches: ligation of T7ptag as insert and vector, respectively</P><H5><SPAN class="mw-headline" id="Digestions">Digestions</SPAN></H5><P>
samples were loaded on 1 % preparative agarose gel and run for 1.5 h at 120V</P><P>
Samples were cut (365 nm illumination as short as possible, picture taken after cutting of gel slices) and purified using Promegas Wizard purification kit according to manufacturers instructions.</P><H5><SPAN class="mw-headline" id="Ligations">Ligations</SPAN></H5><P>Samples were incubated o/n at 16°C.</P></DIV><H2><SPAN class="mw-headline" id="11-09-2011">11-09-2011</SPAN></H2><P><B>People: Katharina, Flo, Anna</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Glycogen.2FEtOH-precipitation_of_ligation_samples_from_10-09-11_and_transformation">Glycogen/EtOH-precipitation of ligation samples from 10-09-11 and transformation</SPAN></H4><P>Put everything on ice before starting to work! Pre-cool centrifuge!</P><P>See protocol for precipitation/transformation from 07-09-11[See Methods]</P><P>Changes made:</P><UL><LI>all solutions were icecold</LI><LI>pellet was air-dryed for about 10 min</LI><LI>autoclaved water was used instead of nuclease-free, due to quick subsequent use in the transformation</LI><LI>resuspending of the bacterial pellet after transformation was difficult (tried pipetting and vortexing), because the wrong centrifuge has been used</LI><LI>1,5 ml-Eppis were used instead of 2 ml-Eppis for the shaking step at 37°C in SOC</LI></UL><P><B>Samples 1, 8 and 9</B>: comp. cells used were DH5a from <B>01-07-11</B></P><P><B>Samples 2 - 7</B>: comp. cells used were DH5a from <B>30-08-11</B></P><P>Antibiotic resistance: </P><UL><LI>1: Amp</LI><LI>2 - 9: Cam</LI></UL></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="Inoculation_of_o.2Fn-cultures_for_tomorrow.C2.B4s_Miller_Assay">Inoculation of o/n-cultures for tomorrow´s Miller Assay</SPAN></H4><P>10 ml LB+antibiotic in 50 ml-falcon tube</P><P>add 10 ul (= 1:1000 dilution) of yesterday´s o/n-cultures:</P><UL><LI>bluelight + lacZ in DH5a (Amp)</LI><LI>redlight + lacZ (trunc.) in DH5a (Cam)</LI><LI>neg. contr. Cph8 in DH5a (Kan)</LI></UL><P><B>Wrap falcons with aluminium foil, so growth can occur in the dark!</B></P><P>Shake o/n @ 37°C, 180 rpm</P><H4><SPAN class="mw-headline" id="Miller_Assay_with_blue_light_construct">Miller Assay with blue light construct</SPAN></H4><P>two cultures in <B>M9</B> medium: </P><P><B>blue light - lacZ on pSB1A2 in DH5alpha</B> with two incubation conditions: <B>lighted in blue light and dark</B></P><P>cpH8 in DH5 alpha as a control under room light</P><P>conditions of bacterial growth: 23,5 °C, shaken, both cultures grown under room light prior to the experiment</P><P>volume of bacterial suspension used: <B>0,1 ml</B></P><P>incubation time for Miller Assay: 5 min (37 °C)</P><P>improvement to prior experiments: start OD between 0,15 and 0,17, correct antibiotics applied</P><P>
Start of Assay:
- measurement of Abs(600nm)</P><P>- new eppi with 0,5 ml of Zbuffer + 20 μl of freshly prepared 0,1% SDS+ 40 μl of Chloroform (under fume hood)</P><P>- 0,1 ml of the samples are taken and transferred to the Zbuffer</P><P>- mix the solution by vortexing for 10 s ( all samples equal vortexing time)</P><P>- transferring of 100 μl supernatant to 96 well plate (for photometer)</P><P>- initiation of assay with 20 μl of ONPG (4mg/ml) NOTE START TIME</P><P>- incubation at 37 °C</P><P>- stop reaction with 50 μl of 1 M Na2CO3 NOTE STOP TIME  </P><P>- measurement of Abs (420nm) and Abs (550nm)</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P>Problems: 
1. cells had a party over night: three LED (blue on, red on, red off) were on alternatively
2. did cells really grow?</P></DIV><H2><SPAN class="mw-headline" id="12-09-2011">12-09-2011</SPAN></H2><P><B>People: Thorsten, Anna</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><P>Part I746907 (T7 promoter, GFP) in pSB1A2 was electroporated into DH5alpha, and plated onto a LB-Agar Amp plate. This part can be used as a reporter plasmid for our AND-Gate in e. coli CP919. </P><H4><SPAN class="mw-headline" id="AND-Gate:_Picking_of_colonies">AND-Gate: Picking of colonies</SPAN></H4><P>Transformation and ligation efficiency of control was several thousand clones. Vector controls were lawn due to usage of CAM-plasmid contaminated electorcompetent DH5 alpha. All contaminated DH5alpha from 01-07-11 have been discarded. </P><P>3 clones of each plate have been picked and inoculated in 5 ml LB/cam o/n.</P></DIV><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><H4><SPAN class="mw-headline" id="sequencing_of_clones_2">sequencing of clones</SPAN></H4><P>Sequence of Blue light promotor with downstream RBS+lacZ is correct.</P><P>Sequence of first 1000 bp of PCR-Product shows one possible silent mutation in PcyA (Phycocyanobilin:ferredoxin oxidoreductase) gen on Position 45 T-&gt;C. Resulting triplet is CCC instead of CCT, which both code for proline.</P><P>Rest of PCR product should be sequenced aswell? Two more primers would be necessary to cover whole sequence. </P><P>Or: functionality check using CP919 strain should result in comparable Miller Unit output between PCR product only and red light original part K322127 as CP919 is Φ(OmpC-lacZ)10-25</P></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="Miller_Assay_with_blue_light_construct_2">Miller Assay with blue light construct</SPAN></H4><P>two cultures in <B>LB medium</B>: </P><P><B>blue light - lacZ on pSB1A2 in DH5alpha</B> with three incubation conditions: <B>lighted in room light, dark and blue light</B></P><P>cpH8 in DH5 alpha as a control under room light</P><P>conditions of bacterial growth: 23,5 °C, shaken, both cultures <B>grown in the dark</B> prior to the experiment, two hours before the experiments: cultures were subdivided and lit as described</P><P>volume of bacterial suspension used: <B>50 µl (or dilutions with LB medium)</B></P><P>incubation time for Miller Assay: 5 min (37 °C)</P><P>
Start of Assay:
- measurement of Abs(600nm)</P><P>- new eppi with 0,5 ml of Zbuffer + 20 μl of freshly prepared 0,1% SDS+ 40 μl of Chloroform (under fume hood) in 2 ml tube</P><P>- 50 μl (as a dilution 25 µl + 25 µl LB medium or 10 μl + 40 μl LB medium) of the samples are taken and transferred to the Zbuffer</P><P>- mix the solution by vortexing for 10 s (all samples with equal vortexing time)</P><P>- transferring of 100 μl supernatant to 96 well plate (for photometer)</P><P>- initiation of assay with 20 μl of ONPG (4mg/ml) = START</P><P>- incubation at 37 °C</P><P>- stop reaction with 50 μl of 1 M Na2CO3 = STOP after exactely 5 min  </P><P>- measurement of Abs (420nm) and Abs (550nm)</P><P>- <B>when solution was too yellow (OD &gt; 2.0), stopped mixtures were diluted 1 : 7 in dd H2O</B></P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P>Problems: 
1. did cells grow up to 20 h and act normally?</P></DIV><H2><SPAN class="mw-headline" id="13-09-2011">13-09-2011</SPAN></H2><P><B>People: Simon, Anna, Alex, Thorsten</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><P>One colony from yesterday's transformation of Part I746907 in pSB1A2 was picked and inoculated into 5 ml LB Amp. Incubation o/n at 37 °C</P><H4><SPAN class="mw-headline" id="AND-Gate">AND-Gate</SPAN></H4><P>Minipreps of picked cones on 12-09-11 were done according to Metabion's instructions. Concentrations:</P><P>2a: c = 68.5 ng/µl</P><P>2b: c = 68.0 ng/µl</P><P>3a: c = 118 ng/µl</P><P>3b: c = 65.0 ng/µl</P><P>5a: c = 122 ng/µl</P><P>5b: c = 71.5 ng/µl</P><P>6a: c = 122 ng/µl</P><P>6b: c = 119 ng/µl</P><P>6c: c = 51.0 ng/µl</P><P>After restriction digest using E and P, samples were separated on a 0.8 % gel:</P><H4><SPAN class="mw-headline" id="Vector_prep_pSB1A3_and_pSB1C3">Vector prep pSB1A3 and pSB1C3</SPAN></H4><DL><DD>The restricted vectors ID10 pSB1A3 and ID12 pSB1C3 from 30-08-2011 were applied onto a 1 % agarose gel and prepped using the Wizard SV gel purification kit.</DD></DL><DL><DD>The vector pSB1A3 was single cut, while pSB1C3 was cut with E and P (from K322127). The expected lengths are about 2 kbp for pSB1C3 and about 3 kbp for pSB1A3, as it also contains a RFP cassette.</DD></DL><DL><DD>The measured concentrations were 12.5 ng/µl for pSB1A3 and 11.5 ng/µl for pSB1C3, in a resuspended volume of 50 µl.</DD></DL></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><P><B>no Miller Assay today</B></P><H4><SPAN class="mw-headline" id="Inoculation_of_o.2Fn-cultures_for_tomorrow.C2.B4s_Miller_Assay_2">Inoculation of o/n-cultures for tomorrow´s Miller Assay</SPAN></H4><P>10 ml LB+antibiotic in 50 ml-falcon tube</P><P>add 20 µl (= 1:1000 dilution) of yesterday´s o/n-cultures:</P><P>in LB medium:</P><UL><LI>bluelight + lacZ in DH5a (Amp)</LI><LI>neg. contr. Cph8 in DH5a (Kan)</LI></UL><P> 
in M9 medium:</P><UL><LI>redlight + lacZ (trunc.) in DH5a (Cam)</LI><LI>neg. contr. Cph8 in DH5a (Kan)</LI></UL><P><B>Wrap falcons with aluminium foil, so growth can occur in the dark!</B></P><P>Shake o/n @ 37°C, 180 rpm</P></DIV><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><P>1. preparation of Zbuffer and Na2CO3 solution for next day</P><P>2. Transformations:</P><DL><DD> chemical transformation of pSB1AK8 T7prom RBS lacZ (plasmid from clone 2c, 09.09.2011) in BL21 DE3. The BL21 cells we received from Lars Mitschke turned out to be chemically competent cells. Transformation was performed based on a protocol from openwetware.org. For detailed description of the protocoll see Methods.</DD></DL><DL><DD> electroporation of Part R2 (= RBS + lacZ) in pSB1A2 using today's and the old electrocompetent DH5alpha cells.</DD></DL><H4><SPAN class="mw-headline" id="Sequencing_results">Sequencing results</SPAN></H4><P>pSB1AK8_T7prom_RBS_lacZ_VF2_110909_clone_2c-GATC-VF2-545457: Sequence shows one missing T in scar between T7promoter and RBS of lacZ. Rest of sequence is fine.</P><H4><SPAN class="mw-headline" id="Electrocompetent_cells_2">Electrocompetent cells</SPAN></H4><DL><DD>new electrocompetent cells (DH5alpha) were made by the following protocol:</DD></DL><DL><DD>20 ml LB medium was inoculated with 1 µl electrocompetent DH5alpha (kindly provided by Andrea Mückl)</DD></DL><DL><DD>350 ml LB medium were inoculated with 15 ml of the overnight culture and incubated at 37°C until a OD(600) of 0,7 was reached</DD></DL><DL><DD>the medium was distributed on falcons and the cells were centrifuged at 4500 rpm for 20 min at 4°C</DD></DL><DL><DD>after discarding the supernatant 350 ml of sterile ice-cold 10% glycerol was added and the pellets were resuspended using a pipet</DD></DL><DL><DD>the resuspension was centrifuged at 4500 rpm at 4°C for 11 minutes</DD></DL><DL><DD>after discarding the supernatant the cells were resuspended again in 350 ml of ice-cold 10% glycerol (cells kept chilled all the time)</DD></DL><DL><DD>and again centrifuged at 4500 rpm at 4°C for 11 minutes</DD></DL><DL><DD>the supernatant was discarded and the cells were resuspended in the remaining supernatant</DD></DL><DL><DD>the cells were dispensed in 40 µl aliquots and stored at -80°C for further use</DD></DL></DIV><H2><SPAN class="mw-headline" id="14-09-2011">14-09-2011</SPAN></H2><P><B>People: Simon, Anna, Alex, Thorsten</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><P>The over night culture of part I746907 (pSB1A2) in DH5alpha was centrifuged and the supernatant was removed. Because of time reasons, the miniprep will be done tomorrow, the pellet was stored at -20°C.</P><P>Furthermore, parts BBa_K568002 (blue light sensor, RBS, LacZ) and BBa_K568003 (T7 promoter, RBS, LacZ) need to be transfered into shipping vector pSB1C3. Therefore, the parts are digested, the vector has already been cut with E and P.</P><H4><SPAN class="mw-headline" id="Digests">Digests</SPAN></H4><P>Blue light sensor, RBS, LacZ in pSB1A2 (tube labelled: Miniprep 04-09-2011, clone 2, c=50 ng/µl) and T7 promoter, RBS, lacZ in pSB1AK8 (miniprep from 09-09-2011, clone 2c, c=131 ng/µl) were cut with E and P.</P><P>Next, the digested DNA was loaded onto a preparative 1% gel. After the electrophoresis was complete, the correct bands were cut out on the UV transilluminator (365 nm) before the following gel photo was recorded:</P><P>The gel looks fine. Cutting out the correct band had to be done very carefully in lanes 1 and 2, because the bands for the vector and the insert were very close together (vector: 3.4 kb, insert: 3.1 kb). Lanes 3 and 4 also show the expected bands, at about 3.2 kb for the insert and about 2 kb for the vector.</P><P>Finally, the DNA was extracted and purified from the cut-out gel slices using the Promega Wizard Kit according to the kit protocol.</P><P>The following concentrations were measured (Nanodrop): </P><P>
blue light sensor, RBS, lacZ: 10 ng/µl</P><P>T7 promoter, RBS, lacZ: 10 ng/µl</P><H4><SPAN class="mw-headline" id="Ligations_2">Ligations</SPAN></H4><P>As a next step, the following ligation was performed (see table below). We used the opportunity to compare the old (NEB) and the new (Metabion) ligase. The ligation was incubated at 16 °C over night.</P></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="Miller_Assay_with_blue_light_construct_3">Miller Assay with blue light construct</SPAN></H4><P>two cultures in LB medium:</P><UL><LI>bluelight + lacZ in DH5a (Amp), two temperatures (<B>30 °C</B> and <B>RT = 22,8 °C</B>), three lightning conditions (<B>room light, dark and blue light</B>)
</LI><LI>neg. contr. Cph8 in DH5a (Kan) at 30 °C
</LI></UL><P>volume of bacterial suspension used: <B>0,1 ml</B></P><P>
Start of Assay:
- measurement of Abs(600nm)</P><P>- new eppi with 0,5 ml of Zbuffer + 20 μl of freshly prepared 0,1% SDS+ 40 μl of Chloroform (under fume hood) + sample</P><P>- 0,1 ml of the samples are taken and transferred to the Zbuffer</P><P>- mix the solution by vortexing for 10 s (all samples equal vortexing time)</P><P>- transferring of 100 μl supernatant to 96 well plate (for photometer)</P><P>- initiation of assay with 20 μl of ONPG (4mg/ml) NOTE START TIME</P><P>- incubation at 37 °C, 4 or 5 minutes</P><P>- stop reaction with 50 μl of 1 M Na2CO3 NOTE STOP TIME  </P><P>- measurement of Abs (420nm) and Abs (550nm)</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P>
Is the spectrometer linear over such a long range of OD's?</P><P><B>no Miller Assay with red light sensor as cells didn't grow:</B></P><P>in M9 medium:</P><UL><LI>redlight + lacZ (trunc.) in DH5a (Cam)
</LI><LI>neg. contr. Cph8 in DH5a (Kan)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="15-09-2011">15-09-2011</SPAN></H2><P><B>People: Simon, Anna, Alex</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Miniprep">Miniprep</SPAN></H4><P>Part I746907 (pSB1A2) was purified from yesterday's DH5alpha-pellet using the miniprep-kit according to protocol. The concentration was determined to be 63 ng/µl using Nanodrop.</P><H4><SPAN class="mw-headline" id="Glycogen.2FEthanol_precipitation_and_Transformation_2">Glycogen/Ethanol precipitation and Transformation</SPAN></H4><P>Yesterday's ligations were purified using glycogen/ethanol precipitation. The yielded 10 µl DNA-Solution were used completely for Transformation in DH5alpha. After electroporation, the cells were incubated at 37 °C for 1h in SOC-media. After this, they were plated onto LB-Agar plates with the appropriate antibiotics.</P><H4><SPAN class="mw-headline" id="Digestions_2">Digestions</SPAN></H4><P>The cloning of our last unfinished part (BBa_K568004), &quot;optogenetical AND-Gate without first promoter - choose your first input&quot; was started today. The part consists of our synthetic construct plus t7ptag.
Our synthetic construct (in pMA-RQ) was digested for preparation using E and S, while the miniprepped part I746907 was digested using E and S to check if the prepped plasmids are correct (a mistake was made during preparation of the digest, so a second digest was made at the same time. Both digests were loaded onto the analytical gel):</P><P>The product of the preparative digest was loaded onto a preparative 1% agarose gel. After electrophoresis, the desired band was cut out on the UV-transilluminator (365 nm) before the gel photo was recorded:</P><P>
The product of the preparative digest was loaded onto a preparative 1 % agarose gel. After electrophoresis, the desired band was cut out on the UV-transilluminator (365 nm) before the gel photo was recorded: :</P><P>The gel looks as expected. The cut out bands around 400 bp are the synthetic construct while the vector can be seen at 2.4 kb.</P><P>The product of the control digest was loaded onto an analytic 0.8 % agarose gel. The following gel photo was recorded:</P><P>Lane 1 (correct digest): Part BBa_I746907 is 924 bp long, the gel shows the corresponding bands. The vector pSB1A2 is 2079 bp long and can also be seen at the expected spot.</P><P>Lane 2 (wrong digest): The bands show that the digest doesn't work properly in 10x NEB4 buffer. The band around 3 kb is uncut or singly cut vector with insert.</P></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="Miller_Assay_with_reporter_construct">Miller Assay with reporter construct</SPAN></H4><P>We incubated two cultures over night in LB medium with the appropriate antibiotics:
</P><UL><LI>BBa_K568003 (T7 promotor and β-galactosidase) in BL21(DE3);
</LI><LI>Negative control BBa_I732017 (rbs and β-galactosidase) in DH5 alpha.
</LI></UL><P>The cultures were diluted to a OD_600 of about 0.7 and incubated for another 30 min. Then, the cultures were induced with IPTG at different concentrations:
</P><UL><LI>BBa_K568003: 0 mM, 0.01 mM, 0.1 mM, 1.0 mM and 1.5 mM;
</LI><LI>BBa_I732017: 0 mM and 1.0 mM.
</LI></UL><P>Before induction and every 30 min, the Miller Units were measured as follows:

</P><OL><LI>measurement of Abs(600nm) in plate reader (volume of bacterial suspension should be equal in all wells, ideally 170 µl)
</LI><LI>new eppi with 0,5 ml of Zbuffer + 20 μl of freshly prepared 0,1% SDS + 40 μl of Chloroform (under fume hood) + 500 µl sample solution (e.g. 430 µl medium + 20 µl bacterial suspension)
</LI><LI>mix the solution by vortexing for 10 s (all samples with equal vortexing time)
</LI><LI>let chloroform settle down (this takes about 5 min, if tube still contains blurred solution, centrifuge for 1 min, RT, 4000 rcf)
</LI><LI>transfer 100 μl of supernatant to 96 well plate (for photometer) 
</LI><LI>initiation of assay with 20 μl of ONPG (4mg/ml), mix well, NOTE START TIME
</LI><LI>incubation at 37 °C
</LI><LI>stop reaction with 50 μl of 1 M Na2CO3, mix well, NOTE STOP TIME  
</LI><LI>measurement of Abs (420nm) and Abs (550nm)
</LI></OL><P>The Miller Units were calculated according to the following equation:
</P><P>Miller Units = 1000 * (ABS420 - (1.75 * ABS550)) / (time [min] * volume [ml] * ABS600)
</P><P>The reaction time was 5 min and the culture volume used was 10 µl.
</P><H3><SPAN class="mw-headline">Other Work</SPAN></H3><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><P>Testing the spectrometer (at the wavelengths 420 and 600 nm, because they are used in the Miller Assay: over which range are the OD values linear?)</P><P>samples:</P><UL><LI>for 600 nm measurements: 170 µl of a bacterial culture in LB medium, blank with fresh LB medium, dilutions in LB medium (minor problem: <B>cells could still grow during the experiment</B>, especially as the plate reader heated the plate)</LI><LI>for 420 nm measurements: 130 µl of one sample of Miller Assay from 13-09-2011 (had already been developed/stopped and stored o/n at RT), blank with ddH2O, dilutions in ddH2O
</LI></UL></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><H4><SPAN class="mw-headline" id="Miller_Assay_Results">Miller Assay Results</SPAN></H4><P>The Miller Assay yielded the following data set:</P><P>The data shows that the cultures induced by isopropyl-β-D-1-thiogalactopyranoside (IPTG) yield higher Miller Units than the uninduced control after 60 min. Therefore, a higher amount of β-galactosidase was produced. This shows that the part is working as expected. Furthermore a slight quantitative dependence of the Miller Units on the concentration of IPTG can be seen. After 120 min the Miller Units of the culture induced with 1.0 mM IPTG go down again, which might be evidence for increasing of proteolysis caused by death of bacteria due to a too high concentration of IPTG. It can also be observed that the uninduced control yields higher Miller Units than the β-galactosidase negative, induced control. This can be explained by leaky biosynthesis of the genome-coded T7 polymerase. If the production of T7 polymerase is more tightly controlled, the level of expressed β-galactosidase should go down to the minimum of the negative, induced control.</P><H4><SPAN class="mw-headline" id="Testing_of_plate_reader_photometer">Testing of plate reader photometer</SPAN></H4><P><B>linear over total range tested, results of yesterday are valid</B></P><UL><LI>at 420 nm: OD = 0.15 - 3.10</LI><LI>at 600 nm: OD = 0.01 - 2.26
</LI></UL></DIV><H2><SPAN class="mw-headline" id="16-09-2011">16-09-2011</SPAN></H2><P><B>People: Simon, Anna</B></P><H3><SPAN class="mw-headline" id="Results_of_15-09-2011">Results of 15-09-2011</SPAN></H3><P>The plates of the transformation (products of the ligation from 14-09-2011) showed the following amount of colonies:</P><UL><LI>Plate 1 (positive control pSB1A3 single cut, old ligase): ca. 150 colonies</LI><LI>Plate 2 (pSB1C3 + T7promoter, RBS, lacZ, ligation ratio I:V = 1): ca. 200 colonies</LI><LI>Plate 3 (pSB1C3 + T7promoter, RBS, lacZ, ligation ratio I:V = 0.4): ca. 30 colonies</LI><LI>Plate 4 (pSB1C3 + blue light sensor, RBS, lacZ, ligation ratio I:V = 1): ca. 1500 colonies</LI><LI>Plate 5 (pSB1C3 + blue light sensor, RBS, lacZ, ligation ratio I:V = 0.4): ca. 500 colonies</LI><LI>Plate 6 (vector background pSB1C3): 5 colonies</LI><LI>Plate 7 (positive control, pSB1A3 single cut, new ligase): ca. 200 colonies</LI><LI>Plate 8 (insert background blue light sensor, RBS, lacZ): ca. 50 colonies</LI><LI>Plate 9 (insert background T7promoter, RBS, lacZ): 3 colonies
</LI></UL></DIV><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Picking_of_clones">Picking of clones</SPAN></H4><P>3 colonies were picked each from plates 2, 3 (Part BBa_K568003 in pSB1C3) and 4, 5 (Part BBa_568002 in pSB1C3) and then inoculated in 5 ml LB Cam each. The cultures were incubated at 37°C over night for miniprep tomorrow.</P><H4><SPAN class="mw-headline" id="Cloning_of_Part_BBa_K568004">Cloning of  Part BBa_K568004</SPAN></H4><P>To ligate the synthetic construct (cut yesterday with E, S) with T7ptag in pSB1C3, the following digestion was done:</P><P>After the digestion was completed, the sample was loaded onto a preparative 1 % agarose gel. After electrophoresis at 120 V, the correct bands were cut out on the transilluminator (365 nm, make sure UV lamp isn't switched on longer than absolutely necessary). Subsequently, the following gel photo was recorded:</P><P>pSB1C3 with T7ptag is 4743 bp long, which corresponds with the location of the cut-out band.</P><P>As a next step, the DNA was purified from the gel slice using the Promega Wizard Kit. The concentration of the product was determined to be 6 ng/µl using Nanodrop. This value is not reliable, but for lack of a second, quick way to measure the DNA concentration, the following ligation was perpared using the value nevertheless. Incubation of the Ligation occurred over night at 16 °C.</P><P>
The product of yesterday's ligation was purified using glycogen-ethanol-precipitation (see methods). The product was stored at -20°C for transformation tomorrow.</P></DIV><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><H4><SPAN class="mw-headline" id="Electrocompetent_cells_cp919">Electrocompetent cells cp919</SPAN></H4><DL><DD>new electrocompetent cells (cp919) were made by the following protocol: (it had to be adjusted due to circumstances)
</DD></DL><DL><DD>the day before: 20 ml LB medium was inoculated with 1 µl electrocompetent DH5alpha (from the iGEM registry)
</DD></DL><DL><DD>450 ml LB medium were inoculated with 4,8 ml of the overnight culture (OD600 = &quot;1,91&quot;, goal for start OD600 = 0,02) and incubated at 37°C until a OD(600) of 0,58 was reached
</DD></DL><DL><DD>the medium was distributed on ten falcons and the cells were centrifuged at 4500 rpm for 20 min at 4°C
</DD></DL><P><B>five falcons broke apart in the centrifuge</B></P><DL><DD>after discarding the supernatant of the five intact falcons, the pellets were resuspended in 15 ml ice-cold ddH2O and distributed on ten tubes. 
</DD></DL><DL><DD>the resuspension was centrifuged at 4000 rpm at 4°C for 10 minutes
</DD></DL><DL><DD>after discarding the supernatant the cells were resuspended again in 1 ml of ice-cold ddH2O per tube 
</DD></DL><DL><DD>the resuspension was centrifuged at 4000 rpm at 4°C for 20 minutes
</DD></DL><DL><DD>after discarding the supernatant the cells were resuspended again in 1 ml of ice-cold ddH2O per tube 
</DD></DL><DL><DD>the resuspension was centrifuged at 4500 rpm at 4°C for 11 minutes
</DD></DL><DL><DD>after discarding the supernatant the cells were resuspended again in 1 ml of ice-cold 10% glycerol per tube 
</DD></DL><DL><DD>the resuspension was centrifuged at 4500 rpm at 4°C for 10 minutes
</DD></DL><DL><DD>after discarding the supernatant the cells were resuspended again in 1 ml of ice-cold 10% glycerol per tube 
</DD></DL><DL><DD>the resuspension was centrifuged at 4500 rpm at 4°C for 10 minutes
</DD></DL><DL><DD>after discarding the supernatant the cells were resuspended again in 1 ml of ice-cold 10% glycerol per tube 
</DD></DL><DL><DD>the resuspension was centrifuged at 4500 rpm at 4°C for 10 minutes
</DD></DL><DL><DD>the cells were resuspended and pooled in a total of 700 µl ice-cold 10% glycerol
</DD></DL><DL><DD>the cells were dispensed in 40 µl aliquots and stored at -80°C for further use (18 tubes)
</DD></DL><H4><SPAN class="mw-headline" id="New_media_and_plates">New media and plates</SPAN></H4><P>LB, LB for Agar plates (Cam and Amp) and Luria medium were prepared. Plates were prepared.</P></DIV><H2><SPAN class="mw-headline" id="17-09-2011">17-09-2011</SPAN></H2><P><B>People: Simon, Anna</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Minipreps_of_parts_BBa_K568003_and_BBa_K568002">Minipreps of parts BBa_K568003 and BBa_K568002</SPAN></H4><P>Yesterday's overnight cultures were prepped today according to  protocol. A total of 12 cultures was prepped, 6 of which contained pSB1C3 with K568003 while the other 6 contained pSB1C3 with K568002.
After the miniprep, the following concentrations were measured using Nanodrop:</P><P>Part K568003 (T7prom, RBS, lacZ):</P><UL><LI> 2a: 86.5 ng/µl</LI><LI> 2b: 71.0 ng/µl</LI><LI> 2c: 63.5 ng/µl</LI><LI> 3a: 66.5 ng/µl</LI><LI> 3b: 88.5 ng/µl</LI><LI> 3c: 85.0 ng/µl
</LI></UL><P>Part K568002 (blue light promoter, RBS, LacZ):</P><UL><LI> 4a: 82.0 ng/µl
</LI><LI> 4b: 78.0 ng/µl</LI><LI> 4c: 80.0 ng/µl</LI><LI> 5a: 74.5 ng/µl</LI><LI> 5b: 97.0 ng/µl</LI><LI> 5c: 100 ng/µl  
</LI></UL><H4><SPAN class="mw-headline" id="Restriction_digest_and_control_gel">Restriction digest and control gel</SPAN></H4><P>To check whether the prepped DNA samples contained the correct parts, the following digestion was performed:</P><P>After the digest was completed, the samples were loaded onto a 0.8% agarose gel. After electrophoresis, the following gel photo was recorded:</P><P>All bands look fine. Samples 1-6 correspond to the minipreps of K568003. Here the insert is 3147 bp and the vector is 2070 bp long. The bands verify this. Samples 7 - 12 correspond to the minipreps of K568002, which has an insert of 3187 bp and the same 2070 bp vector. Again, the bands show that this is correct.</P><H4><SPAN class="mw-headline" id="Glycogen.2Fethanol_precipitation">Glycogen/ethanol precipitation</SPAN></H4><P>Yesterday's ligation was purified using glycogen/ethanol precipitation. The yielded DNA samples were used for transformation</P><H4><SPAN class="mw-headline" id="Transformations">Transformations</SPAN></H4><P>The transformations listed below were carried out. After electroporation the cells were incubated in 1 ml SOC medium for 1 h at 37 °C before plating.</P><P><B>For Redlight Sensor Characterization</B></P><P>All plasmids were electroporated into the new electrocompetent CP919 cells for characterization of the red light sensor later this week.</P><UL><LI> Plate Number 1: Part K322127 in pSB1C3 (tube label &quot;K322127 miniprep 04.09.2011&quot;). Plated onto LB Cam plate.</LI><LI> Plate Number 2: Part K568000 (tube label &quot;5.9.11, 2 vom 1.9, redlight in pSB1C3, 83 ng/µl&quot;). This is our PCR product (redlight sensor). Plated onto LB Cam plate.</LI><LI> Plate Number 3: cph8 in pSB1A3 (tube label &quot;cph8 Miniprep 4.9.11&quot;). Plated onto both LB Amp.
</LI></UL><P><B>For generation of Part BBa_K568004</B></P><P>The purified products of yesterday's ligation were electroporated  into DH5alpha and plated onto LB Cam plates.</P><UL><LI> Plate Number 4: Ligation 1 (16-09-2011), pSB1C3 T7ptag with synthetic construct, ligation ratio I:V = 5.</LI><LI> Plate Number 5: Ligation 2 (16-09-2011), pSB1C3 T7ptag with synthetic construct, ligation ratio I:V = 10.</LI><LI> Plate Number 6: Ligation 3 (16-09-2011), pSB1C3 T7ptag with synthetic construct, ligation ratio I:V = 15.</LI><LI> Plate Number 7: Ligation 4 (16-09-2011), vector background pSB1C3 T7ptag.
</LI></UL><P>The purified products of the ligation performed on 15-09-2011 were electroporated into DH5alpha, just to make sure that the part is really contained in pSB1A3 and not pSB1C3. Therefore, the transformed cells were plate onto LB Amp and LB Cam plates. The plate Number refers to the LB Amp plate as well as the LB Cam plate in all cases.</P><UL><LI> Plate Number 8: Ligation 1 (15-09-2011), positive control pSB1A3 single cut</LI><LI> Plate Number 9: Ligation 2 (15-09-2011) pSB1A3 T7ptag (purify 1.9.) with synthetic construct, ligation ratio I:V = 5.</LI><LI> Plate Number 10: Ligation 3 (15-09-2011) pSB1A3 T7ptag (purify 1.9.) with synthetic construct, ligation ratio I:V = 10.
</LI><LI> Plate Number 11: Ligation 4 (15-09-2011) pSB1A3 T7ptag (purify 1.9.) with synthetic construct, ligation ratio I:V =  15.</LI><LI> Plate Number 12: Ligation 5 (15-09-2011) vector background pSB1A3 T7ptag (purify 1.9.) 
</LI></UL><P><B>For cloning of I746907 into pSB6A1</B></P><P>We want to use I746907 as a reporter part for our optogenetical AND-gate in e. coli CP919. Therefore the part must be transformed into a low copy vector, since the AND-gate is in pSB1C3 which is a high copy vector. Because CP919 already has a Kan-Resistance, the low copy vector used for this needs to carry Amp-Resistance. That's why we chose pSB6A1, a low copy vector with Amp-Resistance.
The DNA was gained from 2011 Kit Plate 1 well 1K by resuspension in 10 µl ddH2O. This was then used for electroporation into DH5alpha which were plated onto LB Amp plates after transformation.</P><UL><LI> Plate Number 13: pSB6A1, 0.5 µl DNA used for electroporation</LI><LI> Plate Number 14: pSB6A1, 1.0 µl DNA used for electroporation</LI><LI> Plate Number 15: pSB6A1, 8.5 µl DNA used for electroporation
</LI></UL><P><B>For Testing of I746907</B></P><P>To be able to characterize part I746907, we used a chemical transformation into BL21 DE3 (see methods). 1.5 µl of the DNA were used (c = 63 ng/µl). The transformed cells were plated onto a LB Amp plate.</P><UL><LI> Plate Number 16: part I746907 in pSB1A2
</LI></UL></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><H4><SPAN class="mw-headline" id="sequencing_results_2">sequencing results</SPAN></H4><P>(Susan)
compared to our parts designed in registry and pSB1C3:</P><P>pSB1C3_opt_and-gate_110913_clone_2a_VR-GATC-VR-547731: no results, bad sequencing quality.</P><P>pSB1C3_opt_and-gate_110913_clone_6b_VR-GATC-VR-547731: part ok! End of part and plasmid backbone were sequenced.</P><P>pSB1C3_synth_construct_110910_clone_5a_VF2-GATC-VF2-545457: Part ok! inserted in backbone</P><P>pSB1C3_synth_construct_110910_clone_6b_VF2-GATC-VF2-545457: Part ok! inserted in backbone, 8 bases could not be detected, so part 5a is better, here the whole sequence is right. After looking at the raw data, sequence seems to be ok.</P></DIV><H2><SPAN class="mw-headline" id="18-09-2011">18-09-2011</SPAN></H2><P><B>People: Simon, Flo</B></P><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P>The transformations done yesterday worked quite well.</P><UL><LI> Plate 1 (Cam): lawn with a few single colonies</LI><LI> Plate 2 (Cam): lawn with a few single colonies</LI><LI> Plate 3 (Amp): ca. 20 colonies</LI><LI> Plate 4 (Cam): ca. 300 colonies</LI><LI> Plate 5 (Cam): ca. 40 colonies</LI><LI> Plate 6 (Cam): ca. 20 colonies</LI><LI> Plate 7 (Cam): 4 colonies
</LI></UL><P>No colonies grew on plates 8 through 12 with Cam. This means that the part was really cloned into pSB1A3. Hence, we use colonies from plates 4 - 6 for miniprepping of part BBa_K568004 in pSB1C3.</P><UL><LI> Plate 8 (Amp): &gt; 1000 colonies</LI><LI> Plate 9 (Amp): no colonies</LI><LI> Plate 10 (Amp): ca. 900 colonies</LI><LI> Plate 11 (Amp): ca. 900 colonies</LI><LI> Plate 12 (Amp): ca. 30 colonies</LI><LI> Plate 13 (Amp): ca. 50 colonies</LI><LI> Plate 14 (Amp): ca. 80 colonies
</LI><LI> Plate 15 (Amp): ca. 120 colonies</LI><LI> Plate 16 (Amp): ca. 300 colonies
</LI></UL></DIV><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Picking_of_clones_of_Bba_K568004_in_pSB1C3_for_miniprep">Picking of clones of Bba_K568004 in pSB1C3 for miniprep</SPAN></H4><P>3 clones each were picked from plates 4, 5 and 6 (transformation from 17-09-2011) and 5 ml LB + Cam were inoculated and vortexed shortly for equal distribution of bacterial cells.</P><P>Falcon tubes were set almost horizontally in the 37°C-incubator to maximize oxygen input and shaking occurred @ 250 rpm for 3 h and 180 rpm for another 4.5 h.</P><H4><SPAN class="mw-headline" id="Miniprep_of_3_x_3_clones_of_Bba_K568004_in_pSB1C3">Miniprep of 3 x 3 clones of Bba_K568004 in pSB1C3</SPAN></H4><P>Miniprep done according to kit instruction (Metabion &quot;mi-plasmid mini prep&quot;) [See Methods]</P><P><I>Changes:</I></P><UL><LI>All 5 ml bacterial suspension were used for the prep</LI><LI>Autoclaved water for elution was pre-warmed to ~60°C
</LI></UL><P><I>Names (according to plate # from 17-09-2011):</I></P><UL><LI> K568004_<B>4-1</B>, c = 64.0 ng/µl </LI><LI> K568004_<B>4-2</B>, c = 92.0 ng/µl</LI><LI> K568004_<B>4-3</B>, c = 62.0 ng/µl</LI><LI> K568004_<B>5-1</B>, c = 67.0 ng/µl
</LI><LI> K568004_<B>5-2</B>, c = 76.5 ng/µl</LI><LI> K568004_<B>5-3</B>, c = 74.5 ng/µl </LI><LI> K568004_<B>6-1</B>, c = 75.5 ng/µl</LI><LI> K568004_<B>6-2</B>, c = 61.5 ng/µl</LI><LI> K568004_<B>6-3</B>, c = 74.5 ng/µl
</LI></UL></DIV><H3><SPAN class="mw-headline">Other work</SPAN></H3><DIV class="otherwork"><H4><SPAN class="mw-headline" id="Picking_of_clones_2">Picking of clones</SPAN></H4><P>A clone of plate 16 (BL21 DE3 with I746907) was picked and inoculated in 5 ml LB Amp for testing tomorrow.</P><P>2 clones were picked each from plates 12 - 15 (DH5alpha containing pSB6A1) and inoculated in 5 ml Amp for miniprep tomorrow.</P><H4><SPAN class="mw-headline" id="Preparation_of_iGEM-parts_for_shipping">Preparation of iGEM-parts for shipping</SPAN></H4></DIV><H2><SPAN class="mw-headline" id="19-09-2011">19-09-2011</SPAN></H2><P><B>People: Simon, Anna</B></P><P>Our Biobricks will be picked up by FedEx today between 1 pm and 3 pm for shipping to the registry. 6 out of 7 Biobricks are ready for take-off, but BBa_K568004 has to be finished in time! </P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><H4><SPAN class="mw-headline" id="Part_BBa_K568004">Part BBa_K568004</SPAN></H4><P>The 9 prepped samples of BBa_K568004 have to be checked for correct sizes of vector and insert. The following digestion was prepared:</P><P>
Due to time reasons, 10 µl of the sample was digested for 1 min in the microwave at 800 W, while the other 10 µl of the sample were incubated at 37 °C.</P><P>The product of quick digest in the microwave was separated using electrophoresis with an analytical 0.8 %  agarose gel. This yielded the following picture:</P><P>The bands of all samples look fine. The insert BBa_K568004 is 3069 bp long, the pSB1C3 vector is 2070 bp long. All samples show these bands. We chose sample 8 for shipping, because it looked &quot;cleanest&quot; on the gel.</P><P>
The product of the &quot;normally&quot; incubated restriction digest (1 h at 37 °C) was also loaded onto an analytical 0.8 % agarose gel. The subsequent electrophoresis resulted in the following picture:</P><P>
Again, the bands are at the expected positions.</P><H4><SPAN class="mw-headline" id="Miniprep_of_pSB6A1">Miniprep of pSB6A1</SPAN></H4><P>All six overnight cultures of pSB6A1 in DH5alpha were miniprepped according to metabion's protocol. The cell pellets were pink, indicating that the cells contained the desired plasmid, which carries an RFP-coding sequence as a standard insert. Due to a mistake, the pellet of one sample was lost.</P><P>The concentrations were measured using Nanodrop:</P><UL><LI> Sample 1: c = 93.0 ng/µl</LI><LI> Sample 2: c = 79.0 ng/µl</LI><LI> Sample 3: c = 77.5 ng/µl</LI><LI> Sample 4: c = 86.5 ng/µl</LI><LI> Sample 5: c = 102 ng/µl
</LI></UL></DIV><H3><SPAN class="mw-headline">Other Work</SPAN></H3><DIV class="otherwork"><P>We prepared the miniprepped pSB1C3 with K568004 for shipping. For used amount see the table found in labbook on 18-09-2011.</P><P>Preparations for testing of red light sensor tomorrow:</P><P>1 clone was picked from each of the following plates (all plates from transformations performed on 17-09-2011):</P><UL><LI> Plate 1 (K322127), inoculated in 5 ml LB Cam</LI><LI> Plate 2 (K568000), inoculated in 5 ml LB Cam</LI><LI> Plate 3 (cph8), inoculated in 5 ml LB Amp
</LI></UL></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="GFP_Assay">GFP Assay</SPAN></H4><P>200 µl of the overnight culture of BL21 (DE3) with I746907 (T7 promotor with GFP) in pSB1A2 were inoculated into 5 ml of fresh LB Amp and incubated for 1.5 hours at 37 °C.
Before the samples for measurement in the plate reader were prepared, we made a change of medium to M63, because LB shows slight fluorescence itself.</P><P>The measurement using the plate reader occured over night, with heating the plate to 37 °C and shaking it between measurements.</P><P>Used filters:
excitation filter: 485 nm, emission filter: 520 nm</P><P><B>Problem</B>: Fluorescence was measured every 10 minutes while measurement of Absorbance occured only every 100 minutes. This led to spikes in the diagram.</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P>no concentration dependency of GFP induction (between 1.5 and 0.1 mM IPTG), as IPTG concentration was in saturation range</P><P>this table shows measured values with</P><UL><LI> calculated: Fluorescence/Absorbance</LI><LI> mean of triplicate measurement</LI><LI> adjusted to 100 %</LI><LI> corrected for outliers</LI><LI> shortened to 700 min
</LI></UL><P>ideas for new GFP Assay:</P><UL><LI> much lower IPTG concentrations</LI><LI> measurement of Absorbance every ten minutes
</LI></UL></DIV><H2><SPAN class="mw-headline" id="20-09-2011">20-09-2011</SPAN></H2><P><B>People: Simon, Anna</B></P><H3><SPAN class="mw-headline">Cloning</SPAN></H3><DIV class="cloning"><P>To be able to test the optogenetical AND-Gate in CP919, we need to transform the desired reporter part into a low copy vector.
Therefore, BBa_ I746907 will be ligated into pSB6A1.</P><H4><SPAN class="mw-headline" id="Digest_2">Digest</SPAN></H4><P>The following digest was performed:</P><P>
After this, the samples were separated by electrophoresis using a preparative 1 % agarose gel. The correct gel bands were cut out on the transilluminator at 365 nm (keep irradiation time as short as possible). After this, the following photo was recorded:</P><P>
The bands match the expectations. </P><P>Samples - : The cut out band around 4 kb is the cut vector pSB6A1, which is 4022 bp long. The band visible at around 1 kb is the RFP coding sequence (1069 bp).</P><P>Samples - : Part I746907 is 924 bp long. This corresponds to the cut out bands. The vector pSB1A2 can be seen at 2 kb (it is 2079 bp long).</P><H4><SPAN class="mw-headline" id="Ligation_5">Ligation</SPAN></H4><P>After purification of the DNA from the gel slices using the Promega Wizard Kit, the following ligation was prepared and incubated at 16 °C over night:</P></DIV><H3><SPAN class="mw-headline">Testing</SPAN></H3><DIV class="testing"><H4><SPAN class="mw-headline" id="Miller_Assay_for_testing_red_light_sensors">Miller Assay for testing red light sensors</SPAN></H4><P>Comparision of the parts:</P><UL><LI> our red light sensor K568000, the PCR product of K322127, pSB1C3 (C) in cp919</LI><LI> K322127 of the team Edinburgh 2010,  pSB1C3 (C) in cp919</LI><LI> cpH8 in pSB1A3 in cp919 as a negative control
</LI></UL><P>
three lighting options:</P><UL><LI> off light (710 nm)</LI><LI> on light (626 nm)</LI><LI> dark
</LI></UL><P>
light sensitive cultures were grown in the dark for five hours prior to the experiment</P><P>OD600 around 2.5 (of cultures before dilution at the start of the experiment), dilution in LB medium to about OD = 0.1 </P><P>Start of Assay:
- measurement of Abs(600nm)</P><P>- new eppi with 0,5 ml of Zbuffer + 20 μl of freshly prepared 0,1% SDS+ 40 μl of Chloroform (under fume hood) <B>+ 420 µl LB Medium</B> + <B>20 µl sample</B></P><P>- 0,1 ml of the samples are taken and transferred to the Zbuffer</P><P>- mix the solution by vortexing for 10 s (all samples equal vortexing time)</P><P>- <B>cenrrifugation: 1 min, RT, 4000 rcf</B></P><P>- transferring of 100 μl supernatant to 96 well plate (for photometer)</P><P>- initiation of assay with 20 μl of ONPG (4mg/ml) NOTE START TIME</P><P>- incubation at 37 °C</P><P>- stop reaction with 50 μl of 1 M Na2CO3 NOTE STOP TIME  </P><P>- measurement of Abs (420nm) and Abs (550nm)</P></DIV><H3><SPAN class="mw-headline">Results</SPAN></H3><DIV class="results"><P><B>In none of the cells with either the K568000 or the K322127 lacZ could be detected.</B></P><P>possible improvements for further experiments:</P><UL><LI> make sure, the cells are in log phase, diluting down earlier, e.g. at a OD600 of about 0.7, however cells shouldn't be in lag phase after 210 min, so at least the last values represent lacZ expression in log phase
</LI></UL></DIV><H2><SPAN class="mw-headline" id="21-09-2011">21-09-2011</SPAN></H2><P>Wiki freeze day! </P><P>Yesterday's ligation was incubated at 65 °C for 10 minutes to heat inactivate the enzymes. The samples were stored at -20 °C and can be used for transformation, as soon as we get to characterizing our optogenetical AND-Gate (BBa_568001). We still hope to generate some data to characterize this part in the near future...</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:TU_Munich/lab/notebook/part3">http://2011.igem.org/Team:TU_Munich/lab/notebook/part3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Munich/lab/notebook/part3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Munich/lab/notebook/part3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Munich/lab/notebook/part3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Munich/lab/notebook/part3&amp;oldid=166292" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>