<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_METU-Ankara"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:METU-Ankara">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:METU-Ankara&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:METU-Ankara&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:METU-Ankara&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:METU-Ankara" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:METU-Ankara</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE> Methan&lt;b&gt;E.COLIc&lt;/b&gt; : Decreasing the Greenhouse effects and Saving the workers life in one system</TITLE></P><DIV id="home" class="item"><DIV class="content2"><DIV class="text"><DIV class="writeback"><H3>Methan<B>E.COLI</B>c : Decreasing the Greenhouse effects and Saving the workers life in one system</H3><P>Firedamp explosions are frequently seen cases at all mines over the world. In Turkey every year, 50 miners lose their lives because of firedamp explosions. Firedamp is a flammable gas found in coal mines and it mainly contains methane. Beside its explosive property, methane is also the main contributor to global warming. However recent mine mechanisms release obtained methane into air. By offers of Synthetic biology, we aimed to design a device which will work on <I>E.coli</I> that provides solutions for side effects of methane. Device that we are planning to construct involves the genes of bacteria (<I>Methylococcus capsulatus</I>) and insect (<I>Drosophilia melanogaster</I>). Our compact system in <I>E.coli</I> is fabricated as sensation of methane, the conversion of methane to methanol and then entrapment of methanol to handle for biofuel and death of bacteria at 42 C by kill switch mechanism.</P><P>When project is analyzed in stepwise, there are successive 4 steps of our modelled system in modified organism. As mentioned in subtitles,  methane is sensed, then converted to methanol and methanol is entrapped and to elute methanol the cells are dead. </P></DIV><CENTER><DIV class="timer"><DIV class="box"><P><B><SPAN class="num"> t</SPAN></B></P><P><B><SPAN class="num"> t</SPAN></B></P><P><B><SPAN class="num"> t</SPAN></B></P><P><B><SPAN class="num"> t</SPAN></B></P><P><B><SPAN class="num"> t</SPAN></B></P></DIV></DIV></CENTER></DIV></DIV></DIV><DIV id="project" class="item"><DIV class="content2"><DIV class="text"><H2>Introduction</H2><P>Methan<B>E.COLIc</B> project is designed to solve one of the problems on worker security in mines of Turkey , by constructing the natural parts of organisms. The main reason for us to choose such a project is that in Turkey- and also in many countries- each year, huge numbers of  workers in mines lost their life due to deficiencies  and conditions in their working areas (mines). The release of Grizu gas leads to subsidence of mines and so death of workers.This project is also designed to solve one of universal problem; global warming. Methane gas is a potential greenhouse gas.This project focuses on sensing the ambient methane gas and converting to one of the bio-fuel sources, methanol. Methanol is entrapped by product of one of the constructs of project, and then cells are dead by kill switch device of project to elute methanol.</P><U><H3>Steps</H3></U><UL><LI><A href="#project2">Methane sensing</A></LI><LI><A href="#project3">Conversion</A></LI><LI><A href="#project4">Entrapment</A></LI><LI><A href="#project5">Kill switch</A></LI></UL></DIV></DIV></DIV><DIV id="team" class="item"><DIV class="content2"><DIV class="text"><H2>Team</H2><P><B><I>Hello everyone!We are Metu Ankara IGEM team.Our team consists of high motivated,social and enjoyable  “undergraduate  and graduate scientists”   from departments of physics, chemistry ,biology, computer engineer and computer education &amp; instructional technology. We are such a team that complete each other  so that we all experienced team  spirit in IGEM. This makes us happy  while doing experiences even in the hot summer.Our slogan is “ all for one, one for all”!!Then, who are we?</I></B></P><H3><U>Instructors</U></H3><UL><LI><A href="mailto:akkayams@metu.edu.tr">Prof. Dr. Mahinur S. Akkaya</A></LI><LI><A href="mailto:kursat@metu.edu.tr">Assoc. Prof. Dr.Kürşat Çağıltay</A></LI></UL><H3><U>Advisors</U></H3><UL><LI><A href="mailto:brkylmz@gmail.com">Burak Yilmaz</A></LI><LI><A href="mailto:okdeniz@ceng.metu.edu.tr">Gökdeniz Karadağ</A></LI></UL><DIV class="smallbox"><H2>Team leader :<A href="mailto:ceren.seref@gmail.com">Ceren Seref</A></H2><DIV class="smalltext"><P>Ceren has just finished the junior year in biological sciences department in METU. It’s her second year in iGEM competition. She’s planning to study about neuroscience-especially on Alzheimer’s disease. She believes synthetic biology approaches can be a very good way of drug development in neuroscience field. She enjoys climbing, listening to music and singing.</P></DIV></DIV><DIV class="smallbox"><H2><A href="mailto:sibelataol8@gmail.com">Sibel Ataol</A></H2><DIV class="smalltext"><P> Sibel is first semester student in Master's program of Biomedical engineering at Middle East
Technical University indeed  she is graduated from Biology Department at METU. This is the second year of her in IGEM team from METU.She is one of the experienced in team and elder sister:) As a vice team lead, planing the experiments and cloning strategies and motivating team are the responsibilities of her. Studying in bioengineering and regenerative sciences is her desire
for future.</P></DIV></DIV><DIV class="smallbox"><H2><A href="mailto:e153564@metu.edu.tr">Funda Guzey</A></H2><DIV class="smalltext"><P>She is a senior student of Biological Sciences at Middle East Technical University. This is her third year in iGEM competition. She gained  a lot of experiences  in  this summer. One of the her interests is Synthetic Biology. However, her main interest is Biomedical and Tissue Engineering. She feels lucky to have such a wonderful team and is glad to share a wonderful summer with her team.</P></DIV></DIV><DIV class="smallbox"><H2><A href="mailto:gokcanaydogan@hotmail.com">Gökcan Aydoğan</A></H2><DIV class="smalltext"><P>Gökcan has just finished the third year of the B.Sc. Chemistry degree at METU. This is his first year in IGEM team. He has been a huge fan of Synthetic Biology ever since he understands nothing about the concept. Everyday, he tries hard to get the picture. In his free time, he likes gaming, drawing cartoon and eating,  eating, eating...</P></DIV></DIV><DIV class="smallbox"><H2><A href="mailto:e160446@metu.edu.tr">Filiz ORMANCI</A></H2><DIV class="smalltext"><P>She is a fourth year student at the Department of Physics at Middle East Technical University. This is the first year of her in iGEM team. She wants to study on the usage of optical instruments in biophysics. She likes team working and sharing the success. She is always positive and happy... ehm..also a “soothsayer”. She enjoys playing musical instruments mainly violin and learning different languages.</P></DIV></DIV><DIV class="smallbox"><H2><A href="mailto:simgeyuz@gmail.com">G. Simge Yüz</A></H2><DIV class="smalltext"><P>She is a recent graduate student of Chemistry and a fourth year student of Biological Sciences. This is the first year of her in iGEM team. She wants to study on neuropharmacology in the field of neuroscience. She is glad to join to METU-Ankara-iGEM team. She enjoys dancing especially Argentine tango and swimming. She is also a member of Flashmob Ankara Society</P></DIV></DIV><DIV class="smallbox"><H2><A href="mailto:yunusalpagu@gmail.com">Yunus Alpağu</A></H2><DIV class="smalltext"><P>Yunus is a third year student in biology department at Middle East Technical University in Turkey. This is his first year in iGEM team.He is interested in regenerative medicine, stem cells, biomaterials and tissue engineering. His favourite TV series is ‘Fringe’ and he likes playing tennis and volleyball.He is glad to join to METU-Ankara-iGEM team. </P></DIV></DIV><DIV class="smallbox"><H2><A href="mailto:pelinispir@hotmail.com">Pelin İspir</A></H2><DIV class="smalltext"><P>Pelin is currently studying the biology department of Middle East Technical University in Turkey.
She is interested in stem cell, developmental biology,tissue engineering, and conservation biology.This is the first year of her in İGEM.She really enjoys with being this team and dealing with the mystery of synthetic biology.She enjoys listening latin musics and flamenco.</P></DIV></DIV><DIV class="smallbox"><H2><A href="mailto:ibrahim.ozturk_02@metu.edu.tr">İbrahim Kutay Ozturk</A></H2><DIV class="smalltext"><P>Kutay is a graduate of Molecular Biology and Genetics, and now doing his M.Sc. in Biotechnology department at METU. This is his first year in IGEM team. He is interested in plant-based drugs, plant-pathogen interaction, biofuel production and bioremediation. He enjoys playing basketball, watching movies, and listening to music.</P></DIV></DIV><DIV class="smallbox"><H2>Advisor:<A href="mailto:brkylmz@gmail.com">Burak Yılmaz</A></H2><DIV class="smalltext"><P>He is a recent graduate of METU Molecular Biology and Genetics department and now studying towards his masters degree on Molecular Bioengineering at METU. His interest in synthetic biology did start during his undergraduate years and after graduation he started up the Sentegen company which is the first biotechnology based company focused on synthetic biology in Turkey. He continues his research and training in synthetic biology while also contributing to the development of the field in his country. He needs new scientific revolutions to solve huge problems of life and emerging field of synthetic biology is best candidate for biotechnological revolution. He is interested in synthetic biology applications, along with Lab-on-a-Chip devices for molecular biology techniques, and he is designing gene synthesis chips to produce biobricks - raw materials of garage biology- faster and cheaper. he enjoys snowboarding, cycling and writing poems.</P></DIV></DIV><DIV class="team-2"><CENTER><H3><U>Software Group</U></H3></CENTER><DIV class="smallbox"><H2><A href="mailto:agca1988@gmail.com">Muhammed Akif Ağca</A></H2><DIV class="smalltext"><P>Akif is a graduate of Computer Education and Instructional Technologies department. In bachelor he has worked on Software Development, Database Management, Instructional Technologies, and Bioinformatics. 
He was software developer and team leader of BioGuide project and Synthetic Biology Online Lecture. 
Currently he is mastering in Software Engineering and developing scientific applications, he will continue to work on Science and Technology Management after becoming master of Software Engineering.
</P></DIV></DIV><DIV class="smallbox"><H2><A href="mailto:miktat_aktas@hotmail.com">Miktat Aktaş</A></H2><DIV class="smalltext"><P>He graduated from Computer Education &amp; Instructional Technology Department of Middle East Technical University last semester and he contributes this genius team as a coder for website. He wants to be a successful academician in his department at METU. He likes swimming and everything related to computers and automobiles.</P></DIV></DIV><DIV class="smallbox"><H2>Ceyhun Kerimov</H2><DIV class="smalltext"><P>Ceyhun was born in Azerbaijan. He is studying in METU Computer Engineering Department. 
He is Java and Salesforce developer now. He was in web side of the project. He tried to help the team to manupilate and store the data. 
He is working in web and cloud computing technologies.
</P></DIV></DIV><DIV class="smallbox"><H2>Huseyin Lutin</H2><DIV class="smalltext"><P>Huseyin is a senior student in METU Computer Engineering Department. 
He was helping the team during software requirements specification step of the project. 
He is working on computer graphics in this year.
</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV id="wetlab" class="item"><DIV class="content2"><DIV class="text"><H2>WET LAB</H2><P>Whole summer in lab work we aimed to design 7 new <I>E.coli</I> compatible Biobricks and  1 composite part from 2011 kit plate distributions, also we aimed to characterize each part by protein analysis and fluorescence protein measurements.</P><P>We achieved design and compatibility of parts in <I>E.coli</I>, however not all parts’ characterization is achieved. Only the genes of designed composite part from kit plate is characterized by GFP measurements.</P><P>In this section you can find out more details on all our wet lab works. You can also find our data under results subtitle and information about design of our parts-biobricks with explanations in literature. It also contains materials and protocols, and finally there is safety questions list on our safety considerations both in the lab and on a wider environmental scale.</P></DIV></DIV></DIV><DIV id="humanpractice" class="item"><DIV class="content2"><DIV class="text"><H2>Overview</H2><P>Methan<B>E.COLI</B>c projec
t arises from sustainability and conservancy of human health and safety. Therefore while we were constructing the organismic device we also wanted to inform people from all age and we wanted to increase the public awareness on our project; methane gas based (firedamp) explosions and the greenhouse effect of methane gas.</P><P>Our aim is to inform people in each age. After brainstorming and searching, we planned the content of information and activities according  to following subtitles.</P></DIV></DIV></DIV><DIV id="futureplan" class="item"><DIV class="content2"><DIV class="text"><H2>Future Plan</H2><P>Methane explosions called as grisou are one of  the main problems from the beginning of the coal mines.  All countries have tried different systems to solve this methane problem. When we joined iGEM 2011 competition, our aim is to find a solution to that problem for humanity by cheaper and energy saving way. After a long literature research, we found how seriously dangerous the methane gas is and how people deal with this big problem on coal mines. Below are the countries that have faced with serious methane explosions throughout their history.</P><OL><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#Australia" target="_blank">Australia</A></LI><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#Belgium" target="_blank">Belgium</A></LI><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#Canada" target="_blank">Canada</A></LI><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#Chile" target="_blank">Chile</A></LI><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#China" target="_blank">China</A></LI><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#France" target="_blank">France</A></LI><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#New Zealand" target="_blank">New Zealand</A></LI><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#Poland" target="_blank">Poland</A></LI><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#Russia" target="_blank">Russia</A></LI><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#United Kingdom" target="_blank">Australia</A></LI><LI><A href="http://en.wikipedia.org/wiki/Mining_accident#United States" target="_blank">United States</A></LI></OL><P>Todays companies deal with this problem by;
<I>Regenerative Thermal Oxidizer from MEGTEC capable of handling 60,000 scfm of VAM. At a methane concentration of about 0.7% a unit this size generates 50,000 carbon credits per year. This unit costs about US$ 1,5 million
Elizabeth Obediente from CO2IMPACT at a methane upgrading facility at a coal mine in Alabama. The gas is stripped of impurities and upgraded to 96% methane per volume and then compressed and then sold to the natural gas pipeline.</I><B>Ventilation Air Methane (VAM):</B> VAM units cost about $20-$30 USD per scfm.</P><P>At some mines, downtime is considerable, causing expensive losses to the mine. Ventilation cost cuts can be up to half of the original cost of the ventilation system. In one study, a mine with 400 ft3/ton of methane using vertical wells to pre-drain will result in an estimated US$ 11 million over 20 years in reduced ventilation costs, not including carbon and energy revenue[iii] (U.S. Environmental Protection Agency, 1999).
Like many solution system for methane problem, the tree examples of solution of methane problem did not satisfied us, so we created our simple and very effective way to deal with the methane.( <A href="http://www.co2impact.com/cmm.html" target="_blank">http://www.co2impact.com/cmm.html</A> )</P><P><B>According to our methane E colic project, we aimed:</B><UL type="disc"><LI>Save the coal workers’ life</LI><LI>Save the nature</LI><LI>Process a new energy</LI><LI>Obtain a cheap system</LI></UL></P><H3><U>Save the coal workers’ life</U></H3><P>We aimed to collect the methane gas from coal mines to the tanks which are filled with water. When the methane passes the legal percent that is 1%, according to our project, the gas is dissolved in water in tanks. Then, inside the tablets containing our bacteria and that are already present on the tanks the molecular process for conversion of methane to methanol begins.
Below the number of workers dead by years are given according to Prof. Dr. Mustafa ÖZTÜRK’s research,Vice President of Parliamentary Commission for the Environment in Turkey:

<CENTER><TABLE border="1" cellspacing="”2″"><TBODY><TR><TH width="100">Years</TH><TH width="300">Death workers on Turkey</TH></TR><TBODY><TR><TH>1983</TH><TH>103</TH></TR><TR><TH>1990</TH><TH>330</TH></TR><TR><TH>1995</TH><TH>40</TH></TR><TR><TH>2003</TH><TH>17</TH></TR><TR><TH>2004</TH><TH>3</TH></TR><TR><TH>2005</TH><TH>18</TH></TR><TR><TH>2006</TH><TH>21</TH></TR><TR><TH>2007</TH><TH>7</TH></TR><TR><TH>2008</TH><TH>43</TH></TR></TBODY></TBODY></TABLE></CENTER></P><H3>Methane explosions: </H3><H3><U>Save the nature</U></H3><P>Methane is one of the greenhouse gases in the atmosphere. Methane is continuously released from coal mining, livestock, landfills and natural gases to the atmosphere. The accumulation of these greenhouse gases causes one of the greatest problems of Earth: global warming. Our project aims to prevent release of methane gas by the conversion of it to methanol in bacterial system in coal mines. </P><CENTER><H3>Greenhouse gases: </H3></CENTER><H3><U>Process a new energy</U></H3><P>After the conversion of methane to methanol, methanol can be used as a precursor of bio-fuel so that accumulated methanol can be recovered. Ethanol is used as biofuel, also; however, methanol is more cheaper to obtain and use as a biofuel. According to Dr. Robert Zubrin, Methanol is cheaper than ethanol. It can also be made from a broader variety of biomass material, as well as from coal and natural gas. And methanol is the safest motor fuel.(American enterprise, February 13th, 2006).</P><H3><U>Cheap system</U></H3><P>There are many systems used to prevent or minimize methane explosions. However, they are so expensive to use and none of them are based on a biological system. The biological system that we build contains a bacterial cell having a molecular process of conversion of methane to methanol is completely harmless and cheaper since expensive equipments are not needed.</P></DIV></DIV></DIV><DIV id="partnership" class="item"><DIV class="content2"><DIV class="text"><H2>PartnerShip</H2><UL><LI><A href="#partnership2">Collaboration</A></LI><LI><A href="#partnership3">Sponsors</A></LI><LI><A href="#partnership4">METU</A></LI><LI><A href="#partnership5">Acknowledgements</A></LI><UL><DIV id="extras" class="item"><DIV class="content2"><DIV class="text"><H2>Extras</H2><CENTER><CENTER><CENTER><H3>Hands on show!</H3></CENTER><CENTER><H3>Photos</H3></CENTER></CENTER></CENTER></DIV></DIV></DIV><DIV id="project2" class="item"><DIV class="content2"><DIV class="text"><H2>Methane sensing</H2><H3>i) Introduction </H3><P>In our methanE.COLIc project, the first critical point is to sense the ambient methane gas. While in literature search, we have focused on the methane and DNA or methane and  regulator protein  interaction which binds to DNA after binding to methane. In structure, methane consists of one carbon atom  and four hydrogen atom which means it is smaller in size and weight. Therefore it is hard to find any study based on methane interaction with DNA or protein to regulate transcription. This made us search for carbon hydrogen bond interaction with any oligo or protein to trigger transcription of any metabolite product. So that we also searched the literature for any organism which utilizes  any alkane chain and alkane which regulates the transcription of organismal metabolites by  interaction with DNA or protein. Since bacteria are environmental adapting organisms, it is possible to find any alkane to regulate the transcription of degradation metabolism. Then we have found the organism, <I>Pseudomonas oleovorans</I> to analyze the alkane degradation  of organism for its carbon source.</P><H3>ii) Background </H3><P> Many microorganisms live in the environments where the conditions changing frequently and so the evolution is inevitable for mechanisms to withstand unfavorable situations.  Therefore microorganisms can use their specific and sensitive mechanisms for sensing the required nutrients for them or any pollution to affect their sustainability. Otherwise they can expose to mutations which change the gene expression and gain new functionalities.  In case they have ability to survive in such conditions. </P><P>While we were scanning the literature based on methane and alkane degrading organisms we have found some organisms that sensitive to methane presence and have mechanisms to activate transcription of related gene clusters.</P><P>We analyzed the soil bacteria, <I>Pseudomonas oleovorans</I>. This strain can assimilate the alkane for its carbon source and one of the microbial whole cell –biosensor.  They have a gene cluster which codes for degradative pathway and includes the activator which interacts especially with linear alkanes. This activator protein, AlkS, in the presence of alkanes, induces the transcription from PalkB promoter which initiates the expression of genes code for assimilation of alkanes. We have analyzed the related articles in which the promoter region  is studied and showed that it is expressed in <I>E.coli</I> correctly because of the corresponding RNA polymerase binding regions.</P><B><P>REFERENCE:</P></B><OL><LI>Canosa, I. &amp; Yuste, L. (2000). A positive feedback mechanism controls expression of AlkS, the transcriptional regulator of the Pseudomonas oleovorans alkane degradation pathway.<I> Molecular Microbiology</I>, 35 (4), 791-799.</LI></OL><H3>iii) Modelling </H3><P> Since the parts that we have found belongs to yeast organism, it was hard to us to manipulate it in bacteria <I>E.coli</I>. We designed the tests of this part by characterizing one of the subunits of methane monooxgenase (MMo). For the functional activity of monooxygenase enzyme, there are required 3 subunits as A,B and C. The A subunit of methane monooxygenase is 210 kDa protein that is supposed as the methane binding active site of monoxygenase complex. This protein consists of non-haem iron and contains p-hydroxo bridge structure. We planned the experiments of this step as characterization of protein A by PCR amplification with specific primers of full construct device to obtained the sequence of protein A. The methane gas is applied to cell culture with our modelled device then the cells were centrifuged and the gas concentration is measured from supernatant.</P></DIV></DIV></DIV><DIV id="wetlab2" class="item"><DIV class="content2"><DIV class="text"><H3>Clonning procedures</H3><OL type="1"><LI><STRONG>Getting the DNA parts</STRONG></LI><OL type="a"><LI><STRONG>For parts in kit plates</STRONG></LI><OL><LI>With a pipette tip, punch a hole through the foil cover into the corresponding well of the Biobrick<SUP>TM</SUP>-standard part that you want.</LI><LI>Pipette 10 uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully re-suspended.</LI><LI><A href="http://partsregistry.org/Help:Transformation_Protocol" target="_blank">Transform</A>
 1 or 2 uL of the re-suspended DNA into desired competent cells, plate your transformation with the appropriate antibiotic and grow overnight.</LI><LI>Pick a single colony and inoculate LB medium (again, with the correct antibiotic) and grow for 16 hours.</LI><LI>Use the resulting culture to <A href="http://openwetware.org/wiki/Miniprep/Kit-free_high-throughput_protocol" target="_blank">miniprep</A> the DNA and make your own glycerol stock.</LI></OL><LI><STRONG>For new synthesized DNA parts</STRONG></LI><OL><LI>The DNA parts are sent in nmol values in each. These data are determined for all of them individually</LI><LI>As a stock solution preparation, these DNA parts are pipetted with PCR water or TE buffer which is 100 times of DNA amount. The final solution is prepared as 100uM.</LI><LI>For each transformation procedures, the 10uM of stock DNA part in water is transformed. 1uL is taken from stock DNA solution and is added to 9uL PCR water.</LI></OL></OL><LI><STRONG>Preparation  of Competent Cells</STRONG></LI><LI>Inoculate a single colony of E. Coli cells into 50 ml  LB medium. Grow overnight at 37 C with moderate shaking (250rpm)</LI><LI>Inoculate  4 mL of the culture into 400 mL LB medium in a sterile 2-liter flask. Grow at  37 C shaking at 250 rpm to an OD590 of 0.375.
                      <P><EM>This  procedure requires that cells be growing rapidly (early- or mid-log phase).  Accordingly, it is very important that the growing cells have sufficient air.  Overgrowth of culture (beyond OD590 of 0.4) decreases the efficiency of  transformation.</EM></P></LI><LI>Aliquot culture into eight 50 mL prechilled, sterile polypropylene tubes and leave the tubes on ice 5 to 10 min.
                      <P><EM>Cells should be kept cold for all subsequent steps.
                      Larger tubes or bottles can be used to centrifuge cells if volumes of all subsequent solutions are increased in direct proportion. </EM></P></LI><LI>Centrifuge  cells 7 min at 1600 g,   4 C.</LI><P>Allow  centrifuge to decelerate without breake.<EM>We have not  attempted to determine whether deceleration without braking is critical to this  procedure. However, we do not use the breake for this step or for the  subsequent centrifugation steps. </EM></P><LI>Pour off subernant and resuspend each pellet in 10 ml ice-cold CaCl2 solution.Re-suspension should be performed very gently and all cells kept on ice.</LI><LI>Cetrifuge cells 5 min at 1100  g, 4 C. Discard supernatant and re-suspend each pellet in 10 mL ice-cold CaCl2 solution. Keep re-suspend cells on ice for 30 min.</LI><LI>Centrifuge cells 5 min at 1100 g, 4 C. Discard supernatant and re-suspend each pellet in 2 mL ice-cold CaCl2 solution.
                      <P><EM>It is important to re-suspend this final pellet well. The suspension may be left on ice for several days.</EM></P></LI><LI>Dispense cells into prechilled, sterile polypropylene tubes (250 ml aliquots are convenient). Freeze immediately at -70 C.</LI></OL><LI><STRONG>Transformation</STRONG></LI><OL><OL><LI>Aliquot  10 ng of DNA in a volume 10 to 25  mL  into a sterile 15 mL round bottom test tube and place on ice.</LI><P><EM>Plasmid DNA  can be used directly from ligation reactions. When this is done , more DNA is  usually used. However, if there is &lt;1 mg of DNA in the ligation reaction, or  if the ligation reaction is from low gelling/melting temperature agarose, it is  wise to dilute the ligation mix.</EM></P><LI>Rapidly  thaw competent cells by warming between hands and dispense 100 mL immediately  into test tubes containing DNA. Gently swirl tubes to mix, then place on ice  for 10 min.</LI><P><EM>Competent  cells should be used immediately after thawing. Remaining cells should be  discarded rather than refrozen</EM></P><LI>Heat  shock cells by placing tubes into a 42 C water bath for 2 min</LI><P><EM>Alternatively  incubate at 37 C for 5 min.</EM></P><LI>Add 1 ml  LB medium to each tube. Place each tube on a roler drum at 250 rpm for 1 hr at  37 C.</LI><LI>Plate  aliquots of transformatin culture on LB /ampicillin or other approprate  antibiotic- containing plates.</LI><P><EM>It is  convenient to plate several different dilutions of each transformation culture.  The remainder of the mixture can be stored at 4 C for subsequent platings.</EM></P><LI>When  plates are dry, incubate 12 to 16 hr at 37 C.</LI></OL></OL><LI><STRONG>Plasmid isolation</STRONG></LI><P>This procedure is performed with Fermentas GeneJET™ Plasmid Miniprep Kit</P><OL><OL><LI>Pick a single  colony</LI><LI> Inoculate  in 5 mL LB medium for high-copy or 10 mL for low-copy of LB medium supplemented  with the appropriate selection antibiotic.</LI><LI> Incubate at  225 rpm not longer than 12-14 h at 37 C .</LI><LI> Centrifuge  at 4000 rpm for 5 min at 4 C .</LI><LI> Discard the  supernatant and keep pellet.</LI><LI>  Resuspend  the pellet with 250 ul  Resuspension solution. (Bacteria should be resuspended completely by vortexing  until no cell clumps remain)</LI><LI>Transfer the  cell suspension to eppendorf.</LI><LI> Add 250 uL Lysis Solution and mix gently by  inverting the tube 4-6 times until the solution becomes slightly clearPlate  aliquots of transformatin culture on LB /ampicillin or other approprate  antibiotic- containing plates.</LI><P><EM>Do not  vortex!</EM><EM>Do not incubate for more than 5 min. (To avoid denaturation of  supercoiled plasmid DNA!)</EM></P><LI>Add 350  uL Neutralization solution and mix immediately and thoroughly by inverting the  tube 4-6 times.</LI><P><EM>(The neutralized bacterial lysate is cloudy and viscous)  (&quot;thoroughly&quot; to avoid localized precipitation of bacterial cell  debris)</EM></P><LI>Centrifuge  at 13000 rpm for 5 min to precipitate cell debris and chromosomal DNA.</LI><LI>Transfer  the supernatant to spin column (about 600 uL).</LI><P><EM>(Avoid disturbing or transferring the white precipitate)</EM></P><LI>Centrifuge  at 13000 rpm for 1 min.        </LI><LI>Discard  the flow-through.        </LI><LI> Place the  column back into same collection tube.</LI><LI> Add 500 uL  Wash solution to spin column.</LI><LI> Centrifuge  at 13000 for 1 min</LI><LI> Discard  the flow-through.</LI><LI> Repeat  this step with using 500 uL Wash solution.</LI><LI> Discard  the flow-through</LI><LI> Centrifuge  at 13000 rpm for an additional 1 min to remove residualWash solution.
          (This step is  essential to avoid residual ethanol in plasmid preps)</LI><LI> Transfer spin column into a fresh 1.5 mL epp.</LI><LI> Add 50 uL Elution buffer into center of the  spin column membrane to elute the plasmid DNA</LI><P><EM>Do not contact the membrane with pipette tip!</EM></P><LI>Incubate  for 2 min at 25 C.  </LI><P><EM>To increase  yield incubation is done for 2 min at heat block at 42 C</EM></P><LI>Centrifuge  at 13000 for 2 min.</LI><LI>Discard  the column and store the purified plasmid DNA at -20 C.</LI><P><EM>OPTIONAL= additional elution step with elution buffer or water. This  step increases the yield by 10-20 %.</EM><EM>NOTE:  For elution of plasmids ≥ 20 kb, prewarm  Elution buffer to 70 C before applying.</EM></P></OL></OL><LI><STRONG>Restriction  digestion</STRONG></LI><P>This procedure  is performed with NEB  BioBrick™ Assembly  Kit</P><UL><LI>Digest  Upstream Part with EcoRI-HF and Spel
    500 ng  Upstream Part Plasmid
    1 uL EcorI-HF
    1 uL Spe I
    5 uL  10XNE  Buffer 2
    0.5 uL 100X  BSA
    To 50 uL add  dH2O</LI><LI>Digest  downstream part will  Xbal and PstI
    500 ng  Downstream Part Plasmid
    1 ul XbaI
    5 uL 10X NE  Buffer 2
    0.5 uL 100X BSA
  To 50 uL add  dH2O</LI><LI>Digest  the destination plasmid with EcorI-HF and Pst I:</LI><P>The destination plasmid DNA should either be prepared with PCR or  contain the toxic gene (e.g, ccdB, sacB) in the cloning site to avoid the need  for gene purification.
The Destination Plasmid should also have a different antibiotic  resistance marker from both the plasmid containing the Upstream Part and the  plasmid containing the Downstream Part to avoid the need to purify the Upstream  and Downstream Parts.
500 ng  Destination Plasmid DNA
1 uL EcorI- HF
1 uL PstI
15 uL 10X NE  Buffer 2
0.5 uL 100X BSA
To 50 uL add  dH2O
 
Incubate all 3  restriction digest reactions at 37 C for 10 min and then heat inactivate at 80  C for 20 min.</P></UL><LI><STRONG>AGE (Confirmation)</STRONG></LI><H3>Background</H3><UL><LI>EtBr stains only dsDNA. You cannot see ssDNA on gel</LI><LI>Minimum amount of DNA load:</LI><P>300 ng 
  •  Each band of 1kb ladder is approximately 125 ng (2.5 ul loaded) and can be seen on gel</P></UL><H3>Preparation</H3><UL><LI>50X TAE (Tris-Acetate) Buffer

121 g/0.5 L Tris
28.6 g/0.5 L Glacial Acetic Acid

7.31 g/0.5 L EDTA

pH 8.0</LI><LI>Sterile 1.5 mL eppendorf tubes</LI><LI>Sterile long pipette tips</LI><LI>DNA standards
  </LI><LI>  Promega 1.0 kb (1.0-10Kbp), 100 bp (0.1-1.0 Kbp)</LI></UL><LI>TAE Buffer</LI><LI>EtBr sln (10 mg/mL). Purchase as solution ! Store at RT. <STRONG>Light-sensitive and carcinogenic !</STRONG></LI><LI>Sample loading buffer. Store at 4 C.</LI><H3>Protocol</H3><UL><LI>Prepare 360 ml, 1X TAE running buffer</LI><LI>Prepare  gel compartment with tape (do not use parafilm)</LI><LI>Prepare  60 ml, 1% agarose gel (0.6 g agarose <U>in 60 ml</U><U>1X TAE buffer)</U></LI><LI>1-2 min  in microwave. Check and swirl after each 30 sec. It must be dissolved completely.</LI></UL><LI>Add 1.0  ul EtBr (10 mg/ml) per 10 ml gel solution, swirl to mix</LI><LI>Add add 4  ul EtBr (10 mg/ml) per 100 ml running buffer.</LI><LI>Pour gel  into gel compartment. Put comb.</LI><LI> After gel solidifies (30 min), take  out the comb and load buffer to one well to measure its capacity.</LI><LI>Position the  gel compartment and fill gel chamber with running buffer.</LI><LI> Prepare samples and standards (ladder)</LI><LI> Prepare samples by mixing 4 ul 6X  Loading Buffer + 20 ul sample</LI><LI> Load the samples (for 24 ul sample, set  pipette 24 ul).</LI><P><EM>Do not push the tip to the bottom of the wells. Make a 45 degree angle and support the pipette tip on the edge of the well. Don’t worry, gel will not brake when you take support. Slowly release the sample. Do not use second pipet push not to risk air injection.</EM></P><LI>Make the  connections: Black (-) on sample side, Red (+) opposite side</LI><LI>Run at 60V
      5-10V/cm (distance between anode and cathode) is recommended.
      90V causes high heat generation for our unit. Use 60V maximum.
      For overnight runs: 2V is too low, causes diffusion
    35V / 4hrs / ¾ progress</LI><LI>Gel can  be stored in 4 C until next day after completion</LI><H3>Gel Photo</H3><UL><LI>Adjust zoom and position using visible light</LI><LI>Before  turning on UV load your settings file which has the following parameters:
    </LI><LI>Preview tab,  all three options checked</LI><LI> Active  image</LI><LI> Dynamic  integration, auto exposure, 10 frames</LI><LI> 50/50  brigthness/contrast</LI></UL><LI> Maximize brigthness with camera knob  (counterclockwise)</LI><LI> Turn on UV light</LI><LI>  Lower brightness from camera knob if necessary</LI><P> </P><LI><STRONG>Gel  extraction (For  Fermentas Gel Extraction Kit)</STRONG></LI><LI>Excise  gel slice containing the DNA fragment using a clean scalpel or razor blade. Cut  as close to the DNA as possible to minimize the gel volume. Place the gel slice  into a pre-weighed 1.5 ml tube and weigh. Record the weight of the gel slice.</LI><P><EM>Note: If the purified fragment will be used for cloning reactions, avoid damaging the DNA through UV light exposure. Minimize UV exposure to a few seconds or keep the gel slice on a glass or plastic plate during UV illumination.</EM></P><LI>Add 1:1 volume of Binding Buffer to the gel slice (volume: weight) (e.g., add 100 µl of Binding Buffer for every 100 mg of agarose gel)</LI>
    p&gt;<EM>Note. For gels with an agarose content greater than 2%, add 2:1 volumes of Binding Buffer to the gel slice.</EM><LI>Incubate the gel mixture at 50-60°C for 10 min or until the gel slice is completely dissolved. Mix the tube by inversion every few minutes to facilitate the melting process. Ensure that the gel is completely dissolved. Check the color of the solution. A yellow color indicates an optimal pH for DNA  binding. If the color of the solution is orange or violet, add 10 µl of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow</LI><LI><EM>Optional:  use this step only when DNA fragment is &lt;500 bp or &gt;10 kb long</EM>.
      </LI><LI>If the DNA fragment is &lt;500 bp, add a 1:2 volume of 100% isopropanol to the solubilized gel solution (e.g. 100 µl of isopropanol should be added to 100 mg gel slice solubilized in 100 µl of Binding Buffer). Mix thoroughly</LI><LI>- If the  DNA fragment is &gt;10 kb, add a 1:2 volume of water to the solubilized gel solution  (e.g. 100 µl of water should be added to 100 mg gel slice solubilized in 100 µl  of Binding Buffer). Mix thoroughly.</LI><LI>Transfer  up to 800 µl of the solubilized gel solution (from step 3 or 4) to the GeneJET™  purification column. Centrifuge for 1 min. Discard the flow-through and place  the column back into the same collection tube. </LI><P><EM>Note. If  the total volume exceeds 800 µl, the solution can be added to the column in  stages. After each application, centrifuge the column for 30-60 s and discard  the flow-through after each spin. Repeat until the entire volume has been  applied to the column membrane. Do not   exceed 1 g of total agarose gel per column.</EM></P><LI><P><EM>Optional:  use this additional binding step only if the purified DNA will be used for sequencing</EM></P><P>Add 100 µl of Binding Buffer to the GeneJET™ purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.</P></LI> 
    <LI>Add 700  µl of Wash Buffer (diluted with ethanol as described on p. 3) to the GeneJET™  purification column. Centrifuge for 1 min. Discard the flow-through and place  the column back into the same collection tube.</LI><LI>Centrifuge  the empty GeneJET™ purification column for an additional 1 min to completely  remove residual wash buffer. </LI><P><EM>Note. This  step is essential to avoid residual ethanol in the purified DNA solution. The  presence of ethanol in the DNA sample may inhibit downstream enzymatic  reactions.</EM></P><LI>Transfer  the  purification column into a clean 1.5  ml microcentrifuge tube (not included). Add 50 µl of Elution Buffer to the  center of the purification column membrane. Centrifuge for 1 min.</LI><P><EM>Note:</EM></P><EM><UL><LI>For low  DNA amounts the elution volumes can be reduced to increase DNA concentration.  An elution volume between 20-50 µl does not significantly reduce the DNA yield.  However, elution volumes less than 10 µl are not recommended.</LI><LI>If DNA  fragment is &gt;10 kb, prewarm Elution Buffer to 65°C before applying to column</LI><LI>If the  elution volume is 10 µl and DNA amount is ≤5 µg, incubate column for 1 min at  room temperature before centrifugation</LI></UL></EM><LI>Discard  the GeneJET™ purification column and store the purified DNA at -20°C</LI><LI><STRONG>Ligation</STRONG></LI><P>This procedure  is performed with  NEB BioBrick™ Assembly  Kit</P><OL><LI>Ligate  the Upstream and Downstream Parts into digested Destination Plasmid.
  </LI><LI> 2 uL Upstream  Part Digestion</LI><LI>      2 uL  Downstream Part Digestion</LI><LI>      2 uL  Desination Plasmid Digestion</LI><LI>      2 uL 10X T4  DNA ligase bufferl</LI><LI>      2 uL T4 DNA  ligase buffer</LI><LI>11 uL dH2O</LI><LI>Incubate at RT  for 10 min and then heat inactivate at 80 C for 20 min.</LI><LI>    Transform 2 uL  of the ligation product into 50 uL of competent E.Coli cells ( or other  suitable host strain). </LI><LI>Select using the antibiotic corresponding to the  Destination Plasmid.</LI></OL><LI><STRONG>Glycerol Stock Preparation</STRONG></LI><LI>Centrifuge  5 mL overnight cells at 5000 rpm for 5 min.</LI><LI>Discard 4  mL supernatant, remain 1 mL with pellet.</LI><LI>Add 176 uL 15% glycerol  and resuspend the pellet.</LI><LI>Aliquot  100 uL and immediately freeze at -70 C.</LI><LI><STRONG>Cell  imaging</STRONG></LI><H3>Checklist Procedure</H3><OL><LI>Suspension culture preparation  </LI><UL><LI>
    Dissolve a single colony in 10 mL  LB+Amp  </LI><LI>Incubate 14-15 hrs, shaking at  225 rpm</LI></UL><LI>Scale-up of the suspension culture</LI><UL><LI> Add 2-3 mL of the suspension culture to 100 mL LB+Amp </LI><LI>Incubate 5-6 hrs, shaking at 225 rpm</LI></UL><LI>Equalizing OD readings</LI><UL><LI>Take 10 mL samples from GFP/RFP  &amp; NC cultures</LI><LI>Centrifuge at 10000 rpm for 5  mins</LI><LI> Discard supernatant</LI><LI>Resuspend pellet in 500 uL 1X PBS</LI><LI>Take 250 uL &amp; dilute to 3 mL  with 1X PBS</LI><LI> Read in UV-Vis at 600 nm</LI><LI> Multiply the OD readings by 12</LI><LI>Dilute remaining 250 uL samples as  required to make all samples have the same ODvalue, final volume should also be 250 uL</LI></UL><LI>Fluorescent  signal reading</LI><UL><LI>Set  parameters
      [GFP]</LI><LI> Mode:  Emission scan</LI><LI> Config:  Digital</LI><LI> Slit: 1 turn  / 2 nm</LI><LI> Blaze: 750  nm</LI><LI> Excitation:  395 nm</LI><LI> Emission  range: 475-600 nm</LI><LI> Average: 3</LI><LI> Step size:  5 nm
        [RFP]</LI><LI> Mode:  Emission scan</LI><LI> Config:  Digital</LI><LI> Slit: 1  turn / 2 nm</LI><LI> Blaze: 750  nm</LI><LI> Excitation:  570 nm</LI><LI> Emission  range: 585-650 nm</LI><LI> Average: 3</LI><LI> Step size:  5 nm</LI><LI> Transfer  samples to small 150 uL cuvette and scan</LI></UL><LI>Data Analysis</LI><UL><LI>Emission  maxima: 
        GFP 515 nm
      RFP 605 nm</LI><LI> Record  intensity values at 509 nm for GFP &amp; 606 nm for RFP</LI><LI> Subtract NC  intensity value from GFP/RFP value</LI><LI> Result is  the fluorescent signal of the protein in the bacteria</LI></UL></OL><LI><STRONG>Protein expression</STRONG></LI><P><STRONG>MAIN STEPS/TIME TABLE</STRONG>
  - Pre-culturing/5 hours
  - Culturing/5 hours 
  - IPTG induction/15 hours<STRONG>Materials</STRONG>
  - LB medium with a suitable antibiotic
  - TB medium with a suitable antibiotic
  - IPTG
  - 2-liter flask
  - Incubator with shaker
  - Centrifuge<STRONG>Check List Procedure</STRONG>
  - Culture transformed E.coli BL21 for 5 hrs at 37 C in 10 LB medium containing antibiotic
  - Inoculate 4 ml pre-cultured cells into 400 ml of TB medium containing antibiotic in a 2-liter cultivation flask
  - Culture it for 5 hrs at 37 C with a rotary shaker at 180 rpm!
  - Add 0.5 mM IPTG in a final concentration
  - Continue the cultivation for 15 hrs at 22 C.
  - Harvest the cells with centrifuge.<STRONG>Alternative Check List Procedure</STRONG>
  - Inoculate single colony for 5 hrs at 37 C in 10 mL LB medium containing antibiotic
  - Inoculate 4 ml pre-cultured cells into 400 ml of TB medium containing antibiotic in a 2-liter cultivation flask
  - Culture it for 5 hrs at 37 C (OD600 to 0.5 - 0.6) with a rotary shaker at 180 rpm
  - Add 1 mM IPTG in a final concentration ( 0.5- 2.0 mM recomended for pT7)
  - Continue the cultivation for 3-4 h at 37 C (15 hrs at 22 C.)
  - Harvest the cells with centrifuge.<STRONG>Solutions</STRONG>
  - TB (Terrific Broth)
  Reference:<A href="http://www.embl.de/pepcore/pepcore_services/protein_expression/ecoli/" target="_blank"> http://www.embl.de/pepcore/pepcore_services/protein_expression/ecoli/</A><STRONG>SDS-PAGE
  
  MATERIALS </STRONG>   
  
  - Vertical Electrophoresis apparatus
  
  - Power supply
  
  - pH meter
  
  - Balance
  
  - Silver staining shaker platform
  
  - Transilluminator
  
  - Filter paper
  <STRONG>For Gel preparation  </STRONG>
  - Acrylamide
  
  - Bisacrylamide
  
  - Deionized Water
  
  - Tris base
  
  - SDS
  
  - APS
  
  - TEMED
  <STRONG>For Silver Stanning  </STRONG>
  - Fixer
  
  - %50 ethanol
  
  - Pretreatment solution
  
  - Silver nitrate
  
  - Devoloping solution
  
  - Stop solution
  
  - Marker: Fermentas / Page Ruler Protein Ladder SM0661 (10-200kDa)
  <STRONG>SOLUTIONS</STRONG><STRONG>Running Buffer (10L, 1X)</STRONG>
  - 25mM Tris, pH 8.3
  
  - 250mM Glycine
  
  - 0.1% SDS

  or

    
  - 60 ml 10X stock + 6 ml 10% SDS + 534 ml dH2O
 
  
  -----------------------
  
  
  - 30.3g Tris
  
  
  - 187.7g Glycine
  
  
  - 10g SDS
 
  
  - Final volume 10 L in demijon
  
  <STRONG>Staining Solution (Coomassie Blue) ( 1 L, 1 X)</STRONG>
  - 2 g brilliant blue (R-250)
  
  
  - 450 ml methanol
  
  
  - 450 ml dH2O
  
  
  - 100 ml acetic acid
  
  
  - store at 24 C
 
  <STRONG>Destaining Solution ( 4 L, 1 X)  </STRONG>
  - Methanol:Acetic Acid:dH2O 40:10:50
  
  
  - Prepare 4.0 L (1.6 L MeOH, 0.4 L AcA, 2.0 L dH2O) in 1 gallon amber bottle, cap tightly
  
  
  - store at room temp.
  
  <STRONG>4X Sample Loading Buffer  </STRONG>
  - 400 mM DTT
  
  
  - 40 mM Tris
  
  
  - 10% Glycerol
 
  
  - 4% SDS
 
  
  - 0.4% Bromophenol blue
  
  
  - prepare 15 ml
  
  
  - 925.2 mg DTT
  
  
  - 72.6 mg Tris
 
  
  - adjust pH to 6.8
   
  
  - 1500 ul Glycerol
   
  
  - 600 mg SDS
   
  
  - 6 mg Bromophenol blue
   
  
  - aliquot 50 x 600 ul
   
  
  - store at -20 C
   
    
  <STRONG>30% Acrylamide 1% Bis-acrylimide  </STRONG>
  - prepare in laminar flow
   
  
  - 30 g acrylimide and 1 g bis-acrylimide in 100 ml
   
  
  - 75 mL acrylimide solution ( 40% stock, Apllichem) and 25 mL bis-acrylimide solution  ( 2% stock, Apllichem)to final volume 100 ml 
   
  
  - store at 4 C, stable for 1 month
   
  <STRONG>Seperating Gel Buffer</STRONG>
  - 1.5 M Tris pH 8.8
   
  
  ------------------------------------------
   
  
  - for 100 mL Buffer solution
   
  
  - 18,15 g Tris pH 8.8
   
  
  - store at room temp.
   
  <STRONG>Stacking Gel Buffer</STRONG>
  - 0.5 M Tris
   
  
  ------------------------------------------
   
  
  - for 50 mL Buffer solution
   
  
  - 3 g Tris pH 6.8
   
  
  - store at room temp.
   
  <STRONG>APS (Ammonium Persulfate)  Stocks (100 mg/ml)  </STRONG>
  - dissolve 0.6 g in 6 ml dH2O
   
  
  - aliquot 10 x 600 ul and store at -20 C
   
  <STRONG>1% Bromophenol Blue  </STRONG>
  - 0.01 g in 1 ml 1M Tris, pH 7.0
   
  
  - store at room temp. in amber bottle<STRONG>CASTING 13% GEL</STRONG>
  - set heater to 100 C for sample prep step  
   <STRONG>  Seperating Gel (30 ml)</STRONG>  	
  - 13 ml 30% Acrylimide 1% Bis-acrylimide
   
   
  - 7.5 ml seperating gel buffer
   
   
  - 8,45 ml dH2O
   
   
  - 500 ul %10 SDS
   
   
  - 250 ul APS (initiator of polymerization)
   
   
  - 25 ul TEMED (catalyst of polymerization)
   
   
  - Load 5.4 ml separating gel between glasses
   
    	
   <STRONG>Stacking Gel (10 ml)  </STRONG>  	
  - 1.6 ml 30% Acrylimide 1% Bis-acrylimide
   
   
  - 2.5 ml stacking gel buffer
   
   
  - 5.85 ml dH2O
   
   
  - 100 ul %10 SDS
   
   
  - 15 ul 1% Bromophenol Blue
   
   
  - 50 ul APS (initiator of polymerization) [add after resolving gel is casted]
   
   
  - 10 ul TEMED (catalyst of polymerization) [add after resolving gel is casted]
   
   
  - Load 1.7 ml stacking gel between glasses
   
   
  - Inset the comb
   
  - make sure the comb has not been inserted in a tilted way. check from behind the apparatus
  - load seperating gel
   
  
  - add some butanol or isopropanol before resolving gel solidifies
   
  
  - make sure gel stays on flat surface while solidifies to prevent tilted surface
   
  
  - load stacking gel
   
  
  - if bubbles form in the stacking gel after polymerization, press the plates between hands to push them out
   
  <STRONG>SAMPLE PREPARATION AND LOADING  </STRONG>
  - Do not overload the the samples, purity check is difficult with overloaded samples.
   
  
  - Sample volume: 5 ul sample+ 5 ul loading buffer + 10 ul dH2O
   
  
  - vortex loading buffer before use
   
  
  - put samples in heating block (100 C) for 5 min
   
  
  - if possible, do not load into the first and last lanes
   
  
  - load 5 ul marker
   
  
  - load 17 ul samples
   
  <STRONG>PREPERATION  </STRONG>
  - check the wire on running apparatus, clean and test
   
  <STRONG>RUNNING</STRONG>
  - never terminate the run early, lighter bands dont separate
   
  
  - 600 ml running buffer is required for each run 
   
  <STRONG>Running Standards  </STRONG>
  - 5 mA &gt; 3/4 of gel &gt; 25hrs &gt;&gt;&gt; overnight running
   
  
  - max: 80 mA for both old and new gel systems
   
  
  - For Lab 103 Tankı
   
     
  - for 145 min at 30 mA 120 V through seperating gel
   
     
  - amper constant
   
  <STRONG>After Run  </STRONG>
  - wash electrophoresis unit after each use
   
  
  - weekly cleaning of power connections recommended to prevent oxidation&lt;<STRONG>STAINING    </STRONG>
  - Stain the gel for 30 min<STRONG>DESTAINING  </STRONG>
  - load the tray fully with destaining buffer 
  - do not put too many (over-destaining) or too less (under-staining) paper sheets 
  - destaining takes 2-3 hrs
 <LI><STRONG>Device Experiments</STRONG></LI></P><P><STRONG>MAIN STEPS/TIME TABLE</STRONG>
    - Pre-culturing/7 hours
    - Culturing to 250ml flasks/3 hours<STRONG>Materials</STRONG>
    - LB medium with a suitable antibiotic (Ampicillin)
    - 250 ml flasks
    - Incubator with shaker
    - Centrifuge<STRONG>Check List Procedure</STRONG>
    - Cell culture (with ROSE regulated kill switch) transformed E.coli BL21 for 7 hrs at 37 C in 10 ml LB medium containing antibiotic
    - Inoculate 100ul pre-cultured cells into 100 ml of LB medium containing antibiotic in a 250 ml cultivation flask
    - Culture it for 3 hrs at 37 C with a rotary shaker at 220 rpm up to OD reading reaches to 0.3
    - Continue taking OD readings up to 0.7 at 37C.
    - Harvest the 5 ml of cells with centrifuge for 5 min at 5000rpm.
    - Resuspend in 5ml PBS and take fluorescence readings.
</P></DIV></DIV></DIV><DIV id="humanpractice2" class="item"><DIV class="content2"><DIV class="text"><H2>Online Lecture</H2><P><B><I><U>Anybody interested and novice in Synthetic Biology can use the lecture</U></I></B>;</P><H3>Here is our animation; Industrial Revolution 2 After Synthetic Biology,  to introduce the basics of the field; </H3><H3>Here is our Online Lecture, Click the picture to go;</H3></DIV></DIV></DIV><DIV id="partnership2" class="item"><DIV class="content2"><DIV class="text"><H2>Collaboration</H2><H3>Collaboration with Turkish teams in iGEM 2011 </H3>
  Since iGEM 2007, there is a team to research and prepare a project for iGEM competition from Middle East Technical University(METU). On behalf, there are advisors and instructors who are familiar with competition and content of it. In this year there are 4 registered teams from Turkey and 3 of them are new participants. Therefore in order to come together and criticize our projects and any deficiencies or ambugity related with project. So we organized and arranged a meeting on Synthetic Biology and iGEM competition which is the first in our country.The participants of this meeting were teams;<B>Fatih Turkey, Bilkent_UNAM, METU-BIN</B> and researchers who related with Synthetic Biology and company owners in Biotechnology field also there was special guests via online talk, Drew Endy, he is an assistant Professor of Bioengineering, Stanford University and Scott Mohr Professor of Biological Chemistry, Boston University This meeting was held in September 10, that was close to Regional Jamboree in order to also check the presentation preparations of teams. The news agencies were also invited and the interviews were done with all teams publicity of teams were enhanced. 
  Except from this, we established a collaboration with <B>METU-BIN Software iGEM 2011</B> team  on their software program. This year their project, <A href="http://dayhoff.ii.metu.edu.tr:8080/m4b/" target="_blank">M4B: Mining for BioBricks</A> is on enhancement and simplification of parts registry and gene library usage to ease the wet lab researchers job. They requested us to use and try their software program. One of our parts in this year, kill switch is composite of this year distributions. By the way we tried their program to search and try on this composite. we gave them the following as feedback and also according to our comments on visual of program, we had collaborated this year.
  “We are constructing a device that is induced by IPTG and that gives lysis enzymes and GFP as the outputs because, in order to elute methanol we need to kill our modified cells that convert ambient methane to methanol and the reason why we want GFP as one output is to be sure that it works. Our modelled device has promoter induced by IPTG, RBS, gene coding GFP and lysis cassette ending with cell death and we are glad to have it and it's working how we expected. So then, we use M4B to see that if it finds the exact device or the device that can do the same job. Since the software has only one place for output, we queried IPTG as an input and GFP as an output on the software and got corresponding 117 results. These results contain devices exactly what we have one for the same operation. However, we couldn't find the lysozyme output for the IPTG input in the software.”   
  We had requested them to support us on wiki page design and same tricks on loading the links on wiki page. We had prepared the self evaluating lab safety questionnaire and they made them manipulable on wiki page  and the methane release meter on home page is their work.<H3>Helping to other new Turkish teams in the future</H3>
  Our university, METU has an online lecture application; METU OpenCourseware to support the open information source not only members of METU  and to reach people to inform in all ages. In this online application, For Synthetic Biology field, last year the sessions were loaded and this year it is upgraded. The protocols for Synthetic biology methods were downloaded and supported with tutorials of procedures that taken during experiments. We established this page and enhanced for Synthetic Biology to reach more students to meet with Synthetic Biology in any region of Turkey. We had requested  from President of METU made people to reach this webpage from universities with related departments in an official way. By the way, we believe to reach more students or researchers to iGEM competition and Synthetic Biology. <H3>Collaboration with 2011 Uppsala Sweeden iGEM Team </H3><P>We asked them to use and give feedback to our software tool on our wiki. This tool is named as BioGuide which was first studied on last year’s competition iGEM 2010 and gained silver medal. This year this team joined with wet lab team. While they were progressing the BioGuide project, this software tool also supported the web page and experiments of METU_Ankara team. We publish this software program on our wiki page to made other iGEM teams to reach and benefit from program on part  and path choices. Uppsala-Sweeden team was one of the teams that use and try the BioGuide program. They tried to reach their used parts in kit distributions and then gave us feedback on them. </P>
  They tried this software tool according to parts of their project As an input parameters they tried to type<STRONG> red light 'cph8',  blue light 'LovTAP', green light 'Ccas', IPTG(PT5-Lac), YF1 </STRONG>and as output parameters they typed <STRONG>amil GFP, amil CP and mCHERRY</STRONG>.  However they found the library of program restricted and they could not reach all parts  because BioGuide software tool library is designed and programmed on parts that are in 2010 and 2011kit plate distributions. By the way, they found this software program managable. Given part names or input and output data results with Biobrick codes of parts in parts registry library. Therefore, it made easier their job on longstanding assembly plans. For mutual collaboration, we read and evaluated their Biosafety web page and gave feedback below
  “You replied the first question having main title “materials used in your project” as bacteria strain used in your project. We suggest that you should focus on chemical or any other material to answer this question.However we also used the E.coli Top 10 as host strain and found that these strains are classified as hazard group 2 pathogen by the UK Advisory Committee on the Dangerous Pathogens (ACDP).
  In the question 3 that you explained “Institutional Biosafety Committee”, you described the threats of Synthetic biology in 3 main items based on your interview, we think you should expand content of items.
  And lastly, it would be better if you give more detail on your last question to be more clear for future iGEM teams.”
<P><A href="https://2011.igem.org/Team:Uppsala-Sweden/Feedback" target="_blank">https://2011.igem.org/Team:Uppsala-Sweden/Feedback</A></P></DIV></DIV></DIV><DIV id="software" class="item"><DIV class="content2"><DIV class="text"><H2>Software Tool</H2><P>BioGuide is a Biological System Design tool. It has been developed in 2010 by the team METU TURKEY SOFTWARE. You can check the page for detailed information about the project :<A href="https://2010.igem.org/Team:METU_Turkey_Software" target="_blank">https://2010.igem.org/Team:METU_Turkey_Software</A></P><H3>Here is rough draft and setup of BioGuide Paper you can read for technical details and download the software;</H3><P><A href="https://static.igem.org/mediawiki/2011/d/db/BioGuide.pdf" target="_blank">BioGuide.pdf</A></P><P><A href="http://sourceforge.net/projects/bioguidesetup/files/BioGuideSetup.rar/download" target="_blank">Download the Software</A></P><H3>This year;</H3><P>We have completed biological system design part of BioGuide and enabled wet lab teams to use it.</P><H3>You can watch the &quot;how to&quot; video of BioGuide to see what and how you can do with it;</H3><H3>Here is how METU ANKARA wet lab team is using BioGuide;</H3><P><I>This year we are using it while building one of our devices, kill switch composite. Kill switch composite consists of 4 main parts which are in 2011 kit plate distributions; T7 promoter, RNA thermometer, GFP and lysis casette. The upstream part of this composite is T7 promoter  (BBa_I712074) which is strong promoter from T7 bacteriophage mostly used expression system and so for strong transcription T7 promoter expression system is chosen.</I></P><P>We also tried the inducible promoters (pLac-IPTG inducer) to design and express the lysis casette. We desired to use a promoter that could be induced and enhance the expression to control and manipulate the translation of device. When we typed IPTG on input box of Bioguide we had reached the possible promoters that could be induced by IPTG. This program gave their standard biobrick codes and finally we got these parts by knowing their location in kit plate.</P><P>In downstream of promoter there is RNA thermometer(ROSE) coding region which determines the transcription of our lysis composite part. We had reached this part by coding the temperature 42 C in input box of BioGuide. RNA thermometer (BBa_K115001) is temperature sensitive at which 42 C translation initiates. </P><P>In downstream of RNA thermometer, green fluorescence protein(BBa_E0040) is ligated. This ligation is done to measure expression level and to comment on RNA thermometer efficiency. We typed GFP in output box since we wanted to measure fluorescence.</P><P>We had designed this composite and it's parts mostly by processing the BioGuide program that made our job easier in intense lab works. We reached the shortest path with our input and output parameters for this device to construct by BioGuide.</P></DIV></DIV></DIV><DIV id="project3" class="item"><DIV class="content2"><DIV class="text"><H2>Conversion</H2><H3>i) Introduction </H3><P>When examined in order, conversion is the second step of device system. While in literature searches, it is found that methanotrophs which live in either extreme conditions or in deep oceans, the process is the same, methane is converted to methanol which is the oxygenated form of hydrocarbon group. The searches directed us to organism <I>Methylococcus capsulatus</I> which is one of the mostly studied organisms among methanotrophs. We designed the conversion part of this organism and modified coding operon according to <I>E.coli</I> strain to enhance the expression. The monoxygenase coding part with  protein A, B and C products are designed for conversion step of  this project.</P><H3>ii) Background </H3><P> Methanotrophs - methane oxidizing bacteria- are ubiquitous in the environment and play an important role in mitigating global warming because they are a unique group of gram-negative bacteria that grow aerobically on methane as sole source of carbon and energy (Hanson and Hanson 1996). They are also potentially interesting for industrial applications such as production of bulk chemicals or bioremediation. Methanotrophs converts methane to methanol and subsequently to formaldehyde which is assimilated or further oxidized to CO2 for biosynthesis of cellular forms.In this biosynthesis pathway the conversion of  methane to  methanol is catalysed  by  methane monooxygenase (MMO) which is a non-specific enzyme complex that also catalyse the oxidation of  a wide range of aliphatic,  alicyclic  and aromatic compounds  (Colby  et a l . , 1977; Higgins et a l . ,1979). The first step in the oxidation of methane to precursors of organic compounds is the conversion to methanol by methane monooxygenase, the key enzyme, which exists in two forms: the cytoplasmic, soluble methane monooxygenase (sMMO) and the membrane-bound, particulate methane monooxygenase (pMMO).  sMMO components have been expressed in heterologous and homologous hosts. Therefore we searched more on soluble form of monooxygenase enzyme. When analyzed in detail, pMMo differs from sMMo by copper ion concentration. Copper ions have been shown to play a key role in regulating the expression of both MMO enzyme complexes. When its concentration increases the differentiation goes through particulate form. Here we analyzed the soluble form of methane monooxygenase to express in heterologous host.</P><H4>a) Soluble Methane Monooxygenase </H4><P> In contrast to pMMO, sMMO has extremely broad substrate specificity and can oxidise a wide range of non-growth substrates such as alkanes, alkenes and aromatic compounds thus making it the more attractive enzyme for co-oxidation reactions. sMMO is expressed only under conditions in which the copper-to-biomass ratio is low, i.e. under “low-copper” growth conditions, when copper ions are omitted from the trace elements solution of a standard mineral salts medium or cells are grown in a fermentor to high cell densities.There is also some evidence that copper ions inhibit the activity of sMMO (Jahng and Wood 1996). Like many other multi-component oxygenase systems, sMMO contains a component of approximately 16 kDa, Protein B, which serves an “effector” or regulatory role. The activity of Protein B may be regulated by proteolysis at its amino terminus (Lloyd et al. 1997). At low concentrations, Protein B converts the hydroxylase from an oxidase and stabilizes intermediates necessary for oxygen activation. Saturating amounts of Protein B dramatically increase the rates of formation of intermediates and accelerate catalysis of methane to methanol by sMMO (Lee and Lipscomb 1999). When analyzed, the most extensively characterised sMMO enzymes are those from <I>Methylococcus capsulatus</I> (Bath) and <I>Methylosinus trichosporium</I> OB3b (reviewed in Lipscomb 1994; Deeth and Dalton 1998). Therefore we preferred to study on <I>Methylococcus capsulatus</I> (Bath) for monooxygenase.The sMMO is a non-haem iron-containing enzyme complex consisting of three components. The hydroxylase consists of three subunits of 60, 45 and 20 kDa arranged in an α2 β2 γ2 configuration. sMMO genes are clustered on the chromosome of <I>Methylococcus capsulatus</I> (Bath) and <I>Methylosinus trichosporium</I> OB3b.  mmoX, mmoY and mmoZ encode the α-,  β- and  γ-subunits respectively of the hydroxylase. mmoB and mmoC code for Protein B and the reductase component. Interestingly, mmoB lies between mmoY and mmoZ;  an ORF of unknown function, designated orfY, with a coding capacity of 12 kDa, lies between mmoZ and mmoC in all genes clusters analysed to date (McDonald et al. 1997).</P><P><B>REFERENCE:</B></P><OL><LI>J. Colin Murrell · Bettina Gilbert · Ian R. McDonald, (2000), Molecular biology and regulation of methane monooxygenase Arch <I>Microbiology</I>(2000) 173 :325–33</LI><LI>Colby, J . &amp;  Dalton, H.  (1978).  Resolution  of  the methane monooxygenase  of  Methylococcus capsulatus  (Bath)  into  three components. Biochemical Journal 171,461-468</LI><LI>Hanson Te&amp;Hanson RS(1996) Methanotrophic bacteria.<I>American Society forMicrobiology</I>.Microbiol. Rev., Jun 1996, 439-471, Vol 60, No. 2</LI><LI>Charlotte, A., West, G. &amp; Horward, D. (1992). Functional expression in Escherichia coli of proteins B and C from soluble methane monooxygenase of Methylococcus capsulatus (Bath. <I>Journal of General Microbiology,</I> (138), 1301-1307.</LI></OL><H3>iii) Modelling </H3><P>For this step of MethanE.COLIc project, the experiments were planned and modelled as protein characterizations. The protein B and C were previously characterized and analyzed proteins in literature. Based on this publications we planned the experiments as purifying protein from cell extract then SDS-PAGE analysis. </P></DIV></DIV></DIV><DIV id="wetlab3" class="item"><DIV class="content2"><DIV class="text"><H2>Biobricks</H2><DIV class="biobir">   <STRONG>LUSH  protein coding device / </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593002" target="_blank">BBa_K593002</A></DIV><DIV class="biobir">   <STRONG>ROSE  regulated kill-switch / </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593000" target="_blank">BBa_K593000</A></DIV><DIV class="biobir">   <STRONG>Lush  protein / </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593003" target="_blank">BBa_K593003</A></DIV><DIV class="biobir">   <STRONG>mmo  X / </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593004" target="_blank">BBa_K593004</A></DIV><DIV class="biobir">   <STRONG>mmo  Y / </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593005" target="_blank">BBa_K593005</A></DIV><DIV class="biobir">   <STRONG>mmo  Z / </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593006" target="_blank">BBa_K593006</A></DIV><DIV class="biobir">   <STRONG>mmo  B / </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593007" target="_blank">BBa_K593007</A></DIV><DIV class="biobir">   <STRONG>mmo  C / </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593008" target="_blank">BBa_K593008</A></DIV><DIV class="biobir">   <STRONG>mmo  D / </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593010" target="_blank">BBa_K593010</A></DIV><DIV class="biobir">   <STRONG>ROSE  regulated kill switch / </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593009" target="_blank">BBa_K593009</A></DIV><DIV class="biobir">   <STRONG>ROSE  regulated  GFP generator/ </STRONG><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K593011" target="_blank">BBa_K593011</A></DIV></DIV></DIV></DIV><DIV id="humanpractice3" class="item"><DIV class="content2"><DIV class="text"><H2>Public Awareness</H2><H3>5-10 year old group</H3><P>For this age group, we prepared and modified one of the national child game. In this game there is a circle of group of child, one is not included in that circle. He/she runs around the circle and labels two of the participants in the circle. When the labelled kids realized that, they leave the circle and new kids join the circle. The video below shows our modified game in which four kids that out of circle are our restriction enzymes wearing white caps. Moreover, we have three insert groups which wear red, green and yellow caps. The blue capped ones are our backbone. During the game we explained kids what DNA, enzyme and insert are, and what we have done as a Synthetic Biology research group.</P><H3>10-17 year old group</H3><P>This group consists of high school aged children groups.Through out the workshop,  we have paired them in doubles. Firstly, we have explained  Synthetic Biolgy  and what Synthetic Biology is able to do. After these training, we explained them our project and taught them basic molecular biology procedures and what are the requirements of basic synthetic biology based experiments. We have done the experiments (cloning procedures) together and also they did individually. We also encouraged them to join iGEM High School division.</P><H3> Official Announcement To All Universities For Introduction Of Synthetic Biology and iGEM Competetion</H3><P>For the last human practice approach, we believed that we  should inform especially students from other universities about synthetic  biology as students in universities are the ones who can improve this field for  further generations. For that purpose, we published an official message from  our instructor, Prof. Dr. Mahinur Akkaya, with this message, students from  other universities –regardless of their department that they’re studying in-  will be aware of the field synthetic biology and may not only consider about  learning more but also consider about creating an iGEM team for coming years.  Here is the original official message and its brief translation for you to  read:
  Dean Office of Arts and Science Department
  Synthetic biology is a field where  natural sciences and engineering sciences go  together hand in hand. It is an emerging science that requires the literature  search, knowledge  of natural sciences  and methods of engineering sciences. Synthetic biology is the engineering of  organisms to give them desired properties, except their regular metabolic  activities. These desired properties are entegrated into genome of the organism  with molecular cloning procedures.
  There are lots of studies going on to introduce this newly  emerging science field to people from all age groups and producing an  information repository and improving it at the same time. One of this studies  is the iGEM (International Genetically Engineered Machines) competition.
  We, as METU iGEM teams, are attending this competition  for four years. For this newly emerging science field to improve more, we  believe that it should be spread out to other universities and more and more  people should attend this competiton. We are sending you our study about the  videos of lab protocols that are mainly used in synthetic biology and  references that you can search for more information.  We feel honored to tell that these sources  are available for everyone who would like to reach them. Links for the sources  are as follows:<A href="http://ocw.metu.edu.tr/course/view.php?id=137">http://ocw.metu.edu.tr/course/view.php?id=137</A><A href="http://www.youtube.com/user/Metuankara">http://www.youtube.com/user/Metuankara</A></P><H3>18-24 year old group</H3><P>This group consists of METU students. We arranged the Physics lectures in order to reach students from several departments. We explained the miracle of Synthetic Biology and our project. At the end of the presentation, we gained many volunteers to join METU-iGEM team for the following years. Also, we have an online lecture application as METU open course ware. For  this web page, we have prepare the course notes and they were uploaded to reach many students from all around  to inform and contact for further relations about Synthetic Biology.

In addition, we distributed hand outs about our project and Synthetic Biology at METU campus, when the registration of new students were done. We aimed to  inform students about our project and about Synthetic Biology to attract their attention on the issue.</P><P>Online lecture link:&amp;nbsp: <A href="http://ocw.metu.edu.tr/course/view.php?id=137" target="_blank">http://ocw.metu.edu.tr/course/view.php?id=137</A></P><H3>Above 24 year old</H3><P>For this group of people we met with group of engineers who work in mining fields in Turkey. We explained our project in details and consulted them on background of our project. What the conditions in mines are, what the general regulations for worker safety are and what the rooted controls for any gas release are the main points which we discussed on.</P><H3>List of Engineers:</H3><H3>Turkish Architects and Engineers Association members:</H3><CENTER><TABLE align="center" border="1" cellspacing="”2″"><TBODY><TR><B><TD width="50">NO</TD><TD width="200">NAME &amp; SURNAME</TD><TD width="250">INSTITUTION</TD><TD width="100">CITY</TD></B></TR><TR><TD width="50">1</TD><TD width="200">Başak GÜLEKEN</TD><TD width="250">ERKUNT DÖKÜM A.Ş.</TD><TD width="100">ANKARA</TD></TR><TR><TD width="50">2</TD><TD width="200">Saliha SİVRİ</TD><TD width="250">ETİ MADEN İŞLETMELERİ</TD><TD width="100">ANKARA</TD></TR><TR><TD width="50">3</TD><TD width="200">Aynur ODAMAN</TD><TD width="250">KOSGEB</TD><TD width="100">ANKARA</TD></TR><TR><TD width="50">4</TD><TD width="200">Mehmet Şevki TÜRÜDÜ</TD><TD width="250">MADEN TETKİK ARAMA KURUMU</TD><TD width="100">ANKARA</TD></TR><TR><TD width="50">5</TD><TD width="200">Galip Şevket YİNSEL</TD><TD width="250">MİTAŞ A.Ş</TD><TD width="100">ANKARA</TD></TR><TR><TD width="50">6</TD><TD width="200">Cemil Hakan GÜR</TD><TD width="250">ORTA DOĞU TEKNİK ÜNİVERSİTESİ</TD><TD width="100">ANKARA</TD></TR></TBODY></TABLE></CENTER></DIV></DIV></DIV><DIV id="partnership3" class="item"><DIV class="content2"><DIV class="text"><H2>Sponsors</H2><P>As METU Ankara iGEM 2011 team we would like to thank to our partners and sponsors in listed below for their supports and for their encouragement.</P><CENTER><DIV class="sponsors"><CENTER><H3>Cargill Comart</H3></CENTER></DIV><DIV class="sponsors"><CENTER><H3>Sentegen</H3></CENTER></DIV><DIV class="sponsors"><CENTER><H3>MetuTech</H3></CENTER></DIV><DIV class="sponsors"><CENTER><H3>Onkogen</H3></CENTER></DIV><DIV class="sponsors"><CENTER><H3>Sacem</H3></CENTER></DIV><DIV class="sponsors"><CENTER><H3>Metu</H3></CENTER></DIV></CENTER></DIV></DIV></DIV><DIV id="project4" class="item"><DIV class="content2"><DIV class="text"><H2>Entrapment</H2><H3>i) Introduction</H3><P>After conversion of methane to methanol, the successive step is the entrapment of methanol. Methanol is an alcohol form of hydrocarbon methane and includes hydroxyl group. Hydroxyl radical groups are highly reactive and, and so short-lived; however, they form an important part of radical chemistry. Hydroxyl groups are especially important in biological systems and their chemistry because free radicals tend to form hydrogen bonds both as donor and acceptor. This property is also related to their ability to increase hydrophilicity and water solubility. Therefore hydroxyl free radicals cause damage to oxidative cells and cellular membranes. In order to produce the methanol for further industrial manufacturing in cellular organism, the organismic conditions should be adjusted. In the light of this, for our bacterial system we transferred the protein coding sequence(named as LUSH) of <I>Drosophilia melanogaster</I> to attack to free methanols and hydroxyl group. Here is some structural information about protein LUSH.</P><H3>ii) Background</H3><P>LUSH is an alcohol-sensitive odorant binding protein expressed in the olfactory organs of <I>Drosophila melanogaster</I>, and it is used as a model system to investigate the biophysical nature of
alcohol-protein interactions at alcohol concentrations that produce intoxication in humans. In this study, by using NMR spectroscopy, they have identified the regions of  LUSH that show increased conformational stability on binding alcohols. These residues primarily line the alcohol-binding pocket. A direct measure of the degree of stability that alcohol imparts on LUSH has been provided. LUSH was originally identified as responsible for mediating an avoidance response to short-chain n-alcohols.</P><P>The general structure of odorant binding proteins consists of six α-helices surrounding a hydrophobic ligand- binding pocket which differs in size and shape between each protein. All these odorant binding proteins have a set of six cysteines that form three conserved disulfide bonds. In the study, by observing the X-ray crystal structures of LUSH-alcohol complexes, it was found that alcohol binds to a single site in the protein formed by a network of concerted hydrogen-binding residues located at one end of hyrophobic pocket. This binding site has some sequence and/or structural similarities to regions of several ligand gated ion channels (LGICs) that have previously been implicated in inferring sensitivity to alcohol. It is hypothesized in the study that the alcohol-binding site in LUSH may represent a more general structural motif for functionally relevant alcohol-binding sites in proteins. The characterization of the effects of n-alcohols on the structure and stability of LUSH is presented. Also, in the absence of ligand, LUSH exists in vitro in a partially unstructured state and binding of alcohols shifts the solution conformation to a more compact folded state which is accompanied by an increase in the overall protein stability. Those regions of the protein that show the largest changes in local dynamics on binding alcohol have been identified and it have been shown that these are predominantly associated with the residues that line the alcohol-binding pocket. The results provide a quantitative measure of the ability of short-chain alcohols to stabilize protein structure at physiological relevant concentrations. </P><P><B>REFERENCE:</B></P><OL><LI>Bucci, B. K., Kruse, S. W. &amp; Thode, A. B. (2006). Effect of n-Alcohols on the Structure and Stability of the <I>Drosophila</I> Odorant Binding Protein LUSH.<I> Biochemistry,</I> 45 1693-1701</LI><LI>Thode, A. B., Kruse, S. W. &amp; Nix, J. C. (2008). The role of multiple hydrogen bonding groups in specific alcohol binding sites in proteins: Insights from structural studies of LUSH.<I> J.Mol Biol,</I> 376 (5), 1360-1376</LI></OL><H3>iii) Modelling </H3><P>In Methan<B>E.COLI</B>c project, LUSH protein is designed to hold methanol; the conversion product of methane. Methanol is bounded to LUSH protein through its hydroxyl group. In device experiments of this part, we planned to isolate and denaturate protein and then refold in presence of methanol and we planned to analyze the alcohol bounded protein in X-ray diffraction crystallogrophy.</P></DIV></DIV></DIV><DIV id="wetlab4" class="item"><DIV class="content2"><DIV class="text"><H2>Notebook</H2></DIV></DIV></DIV><DIV id="humanpractice4" class="item"><DIV class="content2"><DIV class="text"><H2>Media Press</H2></DIV></DIV></DIV><DIV id="partnership4" class="item"><DIV class="content2"><DIV class="text"><H2>Middle East Technical University</H2><P>Middle East Technical University is a pioneering university in education, research and community service in the country. In addition to being highly preferred by students, it is internationally recognized.Our country is successfully represented by METU academic staff in numerous scientific and professional institutions and committees.The wide variety of cultural, artistic and intellectual activities, and sports opportunities that are offered by our campuses are a part of the &quot;education environment&quot; of international quality. METU has been defined in national rankings as the university which has the &quot;Richest Social Environment&quot;.Whatever your reason to visit METU is, you will find here an innovative, dynamic and exciting environment which offers diversity and variety of choices. Come and join us at METU where the most successful students of the country, the most outstanding teaching and research staff, and the most distinguished administrative-technical staff live, learn, and produce hand in hand...</P><H3> For more information, visit following link;</H3><A href="http://metu.edu.tr" target="_blank">Middle East Technical University</A></DIV></DIV></DIV><DIV id="project5" class="item"><DIV class="content2"><DIV class="text"><H2>Kill switch</H2><H3>i) Introduction </H3><P>In this project the last step is the  death of our bacteria to elute the protein(LUSH) bounded product methanol which is one of the precursors of biofuel. In our modified bacteria, the death of cells occurs by temperature dependent cell lysis device. The expression of this device takes place at 42 C since there is an RNA thermometer part in upstream of this device. At the downstream of this device there is an lysis casette. When transcription initiated at 42C holin and endolysin is expressed from this device and make cell dead.</P><H3>ii) Background </H3><P>This composite part consists of promoter, RNA thermometer, GFP and lysis casette. The upstream part of this composite is T7 promoter  (BBa_I712074) which is strong promoter from T7 bacteriophage. For strong transcription T7 promoter expression system is chosen. In downstream of promoter there is RNA thermometer(ROSE) coding region which determines the transcription of our lysis composite part. RNA thermometer (BBa_K115001) is temperature sensitive that at 42C translation initiates. That means, up to 42 C RNA thermometer is in dimer form and at 42C it becomes linear. In downstream of RNA thermometer, green fluorescence protein(BBa_E0040) is ligated. This ligation is done to measure expression level and to comment on RNA thermometer efficiency. The last ligate to this composite is bacteriophage 21 lysis casette (BBa_K124003). This lysis casette induces lysis in E.coli. This gene cassette consists of the S protein (Holin), R protein (Endolysin), and Rz protein. When translation initiates at 42 C then the following coding gene begins to produce the proteins which lyse the cell.</P><H3>iii) Modelling </H3><P>In Methan<B>E.COLI</B>c project, this composite is modelled to lyse and kill the bacteria when switched on at 42C. In experimental design, we ligated this composite without lysis casette (with promoter, ROSE and GFP) We planned two control groups. One is T7 promoter and GFP ligate, this is designed to measure expression level of T7 promoter by reading the GFP expression. The second control group is T7 promoter, ROSE and GFP ligate, this is designed to measure the expression level of T7 promoter with ROSE ligate at 37 C. The device experiment is done with T7 promoter ROSE and GFP ligate at 42 C in which  the GFP expression is expected.</P></DIV></DIV></DIV><DIV id="wetlab5" class="item"><DIV class="content2"><DIV class="text"><H2>Contact</H2><STRONG>Device Experiment Results</STRONG>
  -<STRONG>Flask experiments</STRONG>
  In characterization experiments, we choosed temperature sensitive RNA thermometer (BBa_K115001) and modelled the experiments on GFP measurements. RNA thermometer is a temperature sensitive DNA part that up to 42 C it forms a dimer. This dimer formation prevents polymerase readings that the translation is obstructed. At 42 C the linear form of part forms and translation initiates. This part is submitted to parts registry by iGEM 2008 TUDelft team and the experience of part is represented as none. We modelled the characterization of this part by planing two apart control groups.
  One pair group for characterization was T7 promoter,RNA thermometer and GFP from upstream to downstream at 37C and the same device at 42C. The aim of this control group was to control any expression of green fluorescence protein at 37 C to check RNA thermometer dimer formation. We prepared a flask experiments and from protein formation to folding range we measured the spectrophotometer and fluoresence spectrometer readings. It is showed the dimer formation of RNA thermometer at lower than 42 C caused inefficient binding of RBS to DNA. That means we measured the GFP readings at two temperatures however at 42 C the expression was observed in higher level. As seen at OD:0.6 reading there is a dramatic difference in expressions.
  Other pair group for characterization was T7 promoter with GFP at 37 C  and  T7 promoter,RNA thermometer and GFP from upstream to downstream at 42C. This control group was modelled in order to check and compare the expression levels of two devices. The expected result was to observe the similar readings. Because without RNA thermometer temperature switch off device express similar readings at 37C with RNA thermometer device at 42 C. In the data analysis we observed that there is deviation in readings between control T7 with GFP at 37C and to construct, T7 RNA thermometer and GFP at 42 C.<CENTER><H3>GFP Reading Data</H3></CENTER><P><STRONG>Assembly Results</STRONG>
  According our clonning plan, we planned to ligate the coding sequences of methane monooxygenase as subunits and to express the functional monooxygenase enzyme. Since we synthesized the long coding sequence we did PCR experiments with specific primers we extracted the parts in each the gel electrophoresis results gave validation for successfully synthesied DNA fragments, then we digested and ligated each part with related promoters and vector (pSB1C3) again gel electrophoresis data gave us validation to correctly digest and ligated form. </P><STRONG>-The device parts control experiments</STRONG><P>The synthesized methane monoxygenase construct was so long part that we had problems in synthesizing that we got genes so late. By the way the long sequence was divided into two sequences that one of the regions were splitted into two parts therefore we could not ligated the two parts of coding sequence due to unidentified restriction sites, and full construct unfortunately did not reach to us. The main methane interacting region of monooxygenase could not be expressed functionally.
  We expressed the protein B and C of methane monoxygenase encoeded from mmo B and mmo C genes in protein expression host E.coli BL21 strain. We planned to characterize the proteins in their theoretical molecular weights by SDS-PAGE analysis. However due to technical problems in gel formation we lost samples that we could not reached the data.
  We planned another part on kit plate distributions to check works or not. The bacteriophage 21 lysis casette S, R, and Rz (PVJ4) (BBa_K124003). This part was designed by 2008 iGEM Brown team which induces lysis in E.coli bacteria. We ligated this part with ROSE regulated GFP generator  to induce the lysis of bacteria at 42C. Apart from this ligate, we ligated lysis casette with plac promoter and lacZ expressing gene to observe the blue colonies on plates. However we could not observe any blue colony on plates.
  
  </P></DIV></DIV></DIV><DIV id="humanpractice5" class="item"><DIV class="content2"><DIV class="text"><H2>Fun &amp; Learn</H2><H3>Flashmob!</H3><P>In order to reach more people from several age groups, we have organized a flashmob activity which is related with our project -in molecular level-. (A flashmob is an activity done by a group of people who assemble suddenly in a public <A href="http://en.wikipedia.org/wiki/Public_place " target="_blank">place</A>, perform an unusual and sometimes seemingly pointless act for a brief time, then disperse, often for the purposes of entertainment, satire, artistic expression. Flashmob activities are organized via <A href="http://en.wikipedia.org/wiki/Telecommunications" target="_blank">telecommunications</A>, <A href="http://en.wikipedia.org/wiki/Social_media" target="_blank">social media</A>, or <A href="http://en.wikipedia.org/wiki/Viral_email " target="_blank">viral emails</A> ).One of our group member who is also a member of Flashmob society in Ankara helped us for arrangements. We use flashmob activity in order to explain what our project is and  to increase the public awareness about Synthetic Biology and especially about our project. Flashmob event took place at  one of the biggest shopping malls in Ankara where people often strolled around.  Afterwards, we had interviewed with people who watched  our sketch and informed people about nearly limitless capabilities of synthetic biology.  Indeed we prepared handouts that contains interesting statistical data about methane and its side effects, also information about iGEM and about our project.</P></DIV></DIV></DIV><DIV id="partnership5" class="item"><DIV class="content2"><DIV class="text"><H2>Acknowledgements</H2>
We are grateful to;<I><P>Gönenç Gürsoy, for helping in modelling, being our travel agent and ordering the best potato chips for us (with beer of course)...</P><P>Duygu Yıldız, for being a mother and bringing delicious food for our breaks, being our light graffiti maker...</P><P>Gence Bektaş for being a methane molecule...</P><P>Önder Alparslan, for helping in modelling and ending our desperate search for a graphic tablet :)...</P><P>Mustafa Türkkan for being the best lipid bilayer member ever :) even though being a civil engineer...</P><P>Efe Köksal for being our moviemaker...</P><P>Demir Berkay Yılmaz for our second moviemaker and being in lab with us most of the time... </P><P>Ankara Flashmob society for being with us in our presentations and making it real fun...</P><P>Tufan Öz, Burcu Tefon, Aslıhan Kurt and Çiğdem Yılmaz for their incredible patience to our questions and supplying chemicals at speed of light!</P><P>As a team, we’d like to thank our instructor Professor Dr. Mahinur S. Akkaya for her patience, love, support and all of her solutions to our unending problems. :)</P></I></DIV></DIV></DIV><DIV id="wetlab6" class="item"><DIV class="content2"><DIV class="text"><H2>Safety</H2><H3>Safety Questions</H3><P><B>1.  Would the materials used in your project and/or your final product pose:</B><B>a.  Risks to the safety and health of team members or others in the lab?</B></P><OL><LI>While in cloning steps the most  dangerous material that we used is <B>ethidium bromide</B> which is used in  agarose gel electrophoresis. Ethidium bromide is a potential mutagen because it  works by inserting itself between the two strands of double-stranded DNA. Since  the amount of ethidium bromide kept in our lab is relatively small, it does not  pose devastating effects.However it is a toxic chemical, when got in contact,  it causes eye and skin irritation. To prevent the damages that we would be face  with, we wraped it with aluminium and put in brown bottle and wore gloves while  using.</LI><LI>While device training experiments we  planned to use <B>methane gas CH4</B> and <B>methanol CH3OH</B>. Methane  is  inactive biologically and essentially  nontoxic.Methane is not listed in the IARC, NTP or by OSHA as a carcinogen or  potential carcinogen. When inhaled in high concentrations,  so as to exclude an adequate supply of  oxygen to the lungs causes dizziness, deeper  breathing due to air hunger, possible   nausea and eventual unconsciousness. In case of protection, we adjusted  the progressing the steps for methane in glove boxes in order to provide the  respiratory protection. The ventilation is   supported by hood. However due to methane gas is potent for greenhouse  effect for doing experiments, special hood system is designed that could be  used which has its own gas container for storage of gases is used. Thus,  emission of methane gas is controlled. Later on, filled gas container is taken  by department of chemistry to empty with certain procedures. We arranged the  conditions in department for possible experiments with methane gas. And  protective gloves and goggles are equiped for personal safety.Out of the  experiments for storage the gas tanks were saved in area where is cool enough  and never used open flames.</LI></OL><P>REFERENCE:<A href="http://www.isocinfo.com/DocumentRoot/13/Methane.pdf" target="_blank">http://www.isocinfo.com/DocumentRoot/13/Methane.pdf</A>
  For the steps with <B>methanol t</B>he  characterization tests for LUSH protein, methanol is        used. Methanol is hazardous in case of  skin contact, eye contact, ingestion and inhalation. It is flammable liquid.The  safety for this chemical is provided by experimenting in hood with high  ventilation and and so much exposed because it pose a health risk to anyone in  lab. 
  REFERENCE :<A href="http://www.sciencelab.com/msds.php?msdsId=9927227" target="_blank">http://www.sciencelab.com/msds.php?msdsId=9927227</A></P><P>We had studied with ANS; 1-Anilino-8-Naphthalene  Sulfonate as protein conformational tightening agent for LUSH protein folding  experiments. We searched on safety of this chemical and we found that it is non  hazardous chemical according to Directive 67/548/EE Following reference belongs  to Sigma Aldrich MSDS report.<A href="http://www.sigmaaldrich.com/catalog/DisplayMSDSContent.do" target="_blank">http://www.sigmaaldrich.com/catalog/DisplayMSDSContent.do</A></P><P><B>b. Risks to the safety and health of the general public if  released by design or accident?</B></P><OL><LI>The materials  mentioned in part a. could potentially be  dangerous to the general public. We use just enough concentration for our  experiments. We always check the amount both for methane and methanol and EtBr,  because if accidentally released, they could be dangerous to us while  experiments and to lab security.</LI></OL><P><B>c. Risks to environmental quality if released by design or  accident?</B></P><OL><LI>Since methane is a gas at normal temperature  and pressure, the inhalation of this gas pose a risk for human health and so  for environmental quality. Methane is potent greenhouse gas and compared to  carbon dioxide it has greater potential. This makes methane gas dangerous for  environmental quality in case of release by accident or design.However, as  mentioned in part a, we controlled the conditions for gas and examined in air  ventilated glove boxes. Another material which could pose risk for  environmental quality is methanol. This chemical is volatile and flammable  liquid. When this chemical release near open flame it could result with harmful  effects both for health and environment conditions.</LI></OL><P><B> d. Risks to security  through malicious misuse by individuals, groups or states?</B>Only the methane gas has risks for  security if reaches to malicious someone. Our group members have experience and  training on gas used experiments and there is no possibility on misuse indeed  malicious misuse.
  e.<B>Please explain your responses  (whether yes or no) to these questions.
    Specifically, are any parts or devices in your project associated with (or  known to cause):</B><B>- pathogenicity, infectivity, or  toxicity? </B> No<B>- threats to environmental quality? </B>None of our designed parts pose a risk to environment<B>- security concerns? </B>No</P><P>     There are no part on our experiments and our project that could be use  as treat to environment or infect to human life as individually. Only one of  the parts, subunit A of methane monooxygenase may pose a health risk while  studying since it has the region where methane interacts and the conversion  steps initialize. As host cell we used E.coli BL21 (DE3) and Top10 strains,  these strains are classified as hazard group 2 pathogen by the UK  Advisory Committee on the Dangerous Pathogens (ACDP). <B>2. If your response to any of the questions above is yes:</B><B>a. Explain how you addressed these  issues in project design and while conducting laboratory work. </B>Explained in each question<B>b. Describe and document safety,  security, health and/or environmental issues as you submit your parts to the  Registry. </B>Explained in each question<B>3. Under what biosafety provisions will / do you operate?</B><B>a. Does your institution have its  own biosafety rules and if so what are they? Provide a link to them online if  possible.</B>
  Yes, there is general laboratory biosafety  rules for lab security and researcher security obeyed in Biology and Chemistry  departments. In research laboratories the students are provided with general  lab security and rules which are compilation of general and international  procedures for individual safety. These procedures are the compilation of  several links as<A href="http://oba.od.nih.gov/oba/rac/guidelines_02/NIH_Gdlnes_lnk_2002z.pdf" target="_blank">http://oba.od.nih.gov/oba/rac/guidelines_02/NIH_Gdlnes_lnk_2002z.pdf</A></P><P> </P><P><B>b. Does your institution have an  Institutional Biosafety Committee or equivalent group? If yes, have you  discussed your project with them? Describe any concerns or changes that were  made based on this review.</B></P><P>Yes there is biosafety and ethical  research center in METU until 2000. This committee helds  many conferences about ethics, environmental  ethics in studies and the medicinal ethics based on human researchs. The below  link belongs to this committee however in Turkish language.<A href="http://www.ueam.metu.edu.tr/" target="_blank">http://www.ueam.metu.edu.tr</A></P><P>      Indeed in Biology department at METU one of our professors, Prof.Dr  Huseyin Avni Oktem was one of the members of National Biosafety Coordinating  Committee. He is our pioneer for biosafety and security issues. In the case of  any safety issues, he is the one to consult   the critical safety points in our project and also discussed the project  based on safety. </P><P><B>      c. Will / did you receive any biosafety and/or lab training before  beginning your project? If so, describe this training.</B></P><P>Before we begin to experiments, we  firstly were taught on lab security and personal safety for cases of individual  injure, indeed in emergency situations. This interval was training for us. Also  team leaders, the experienced ones in team, prepared an exam for new team  members as a part of training for general lab regulations and project details.</P><OL><LI><B>Does your country have national  biosafety regulations or guidelines? If so, provide a link to them online if  possible.</B></LI></OL><P><B> </B></P><P>In Turkey, biosafety regulations are  organized by the Biosafety Information Exchange  Mechanism of Turkey, facility of Ministry  of Agriculture and Rural Affair.<A href="http://www.tbbdm.gov.tr/en/Home.aspx" target="_blank">http://www.tbbdm.gov.tr/en/Home.aspx</A></P></DIV></DIV></DIV><DIV id="menu"><B><UL class="navmenu"><LI><DIV class="menutop menusingle panel"><A class="panel" href="#home">HOME</A></DIV></LI><LI><DIV class="menutop panel"><A class="panel" href="#project">PROJECT</A><DIV class="toggle toggle-closed">
				+</DIV></DIV></LI><LI><A class="panel" href="#project2">Methane sensing</A></LI><LI><A class="panel" href="#project3">Conversion</A></LI><LI><A class="panel" href="#project4">Entrapment</A></LI><LI><A class="panel" href="#project5">Killswitch</A></LI></UL></B><LI><B><DIV class="menutop menusingle panel"><A class="panel" href="#team">TEAM</A></DIV></B></LI><LI><DIV class="menutop panel"><A class="panel" href="#wetlab">WET LAB</A><DIV class="toggle toggle-closed">
				+</DIV></DIV></LI><LI><A class="panel" href="#wetlab2">Lab Protocols</A></LI><LI><A class="panel" href="#wetlab3">Biobricks</A></LI><LI><A class="panel" href="#wetlab4">Notebook</A></LI><LI><A class="panel" href="#wetlab5">Results</A></LI><LI><A class="panel" href="#wetlab6">Safety</A></LI><LI><DIV class="menutop panel"><A class="panel" href="#humanpractice">HUMAN PRACTICE</A><DIV class="toggle toggle-closed">
				+</DIV></DIV></LI><LI><A class="panel" href="#humanpractice2">Online Lecture</A></LI><LI><A class="panel" href="#humanpractice3">Public Awareness</A></LI><LI><A class="panel" href="#humanpractice4">Media Press</A></LI><LI><A class="panel" href="#humanpractice5">Fun &amp; Learn</A></LI><LI><DIV class="menutop menusingle panel"><A class="panel" href="#futureplan">FUTURE PLAN</A></DIV></LI><LI><DIV class="menutop panel"><A class="panel" href="#partnership">PARTNERSHIP</A><DIV class="toggle toggle-closed">
				+</DIV></DIV></LI><LI><A class="panel" href="#partnership2">Collaboration</A></LI><LI><A class="panel" href="#partnership3">Sponsors</A></LI><LI><A class="panel" href="#partnership4">METU</A></LI><LI><A class="panel" href="#partnership5">Acknowledgements</A></LI><LI><DIV class="menutop  menusingle panel"><A class="panel" href="#extras"> EXTRAS</A></DIV></LI><LI><DIV class="menutop  menusingle panel"><CENTER><A class="panel" href="#software">SOFTWARE TOOL BioGuide</A></CENTER></DIV></LI></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:METU-Ankara">http://2011.igem.org/Team:METU-Ankara</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:METU-Ankara" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:METU-Ankara" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:METU-Ankara&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:METU-Ankara&amp;oldid=236564" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>