"
Page
Discussion
View source
History
teams
Log in
 
Team:METU-Ankara/lab
From 2011.igem.org
Contents
1 Lab Protocols
1.1 Clonning procedures
1.2 Background
1.3 Preparetion
1.4 Protocol
1.5 Gel Photo
1.6 Checklist Procedure
Lab Protocols
Clonning procedures
Getting the DNA parts
For parts in kit plates
</b>
With a pipette tip, punch a hole through the foil cover into the corresponding well of the BiobrickTM-standard part that you want.
Pipette 10 uL of dH2O (distilled water) into the well. Pipette up and down a few times and let sit for 5 minutes to make sure the dried DNA is fully re-suspended.
<a href=" http://partsregistry.org/Help:Transformation_Protocol " target="_blank">Transform</a> 1 or 2 uL of the re-suspended DNA into desired competent cells, plate your transformation with the appropriate antibiotic and grow overnight.
Pick a single colony and inoculate LB medium (again, with the correct antibiotic) and grow for 16 hours.
Use the resulting culture to <a href=" http://openwetware.org/wiki/Miniprep/Kit-free_high-throughput_protocol " target="_blank">miniprep</a> the DNA and make your own glycerol stock.
For new synthesized DNA parts
The DNA parts are sent in nmol values in each. These data are determined for all of them individually
As a stock solution preparation, these DNA parts are pipetted with PCR water or TE buffer which is 100 times of DNA amount. The final solution is prepared as 100uM.
For each transformation procedures, the 10uM of stock DNA part in water is transformed. 1uL is taken from stock DNA solution and is added to 9uL PCR water.
Preparation  of Competent Cells
Inoculate a single colony of E. Coli cells into 50 ml  LB medium. Grow overnight at 37 C with moderate shaking (250rpm)
Inoculate  4 mL of the culture into 400 mL LB medium in a sterile 2-liter flask. Grow at  37 C shaking at 250 rpm to an OD590 of 0.375.
This  procedure requires that cells be growing rapidly (early- or mid-log phase).  Accordingly, it is very important that the growing cells have sufficient air.  Overgrowth of culture (beyond OD590 of 0.4) decreases the efficiency of  transformation.
Aliquot culture into eight 50 mL prechilled, sterile polypropylene tubes and leave the tubes on ice 5 to 10 min.
Cells should be kept cold for all subsequent steps.
Larger tubes or bottles can be used to centrifuge cells if volumes of all subsequent solutions are increased in direct proportion.
Centrifuge  cells 7 min at 1600 g (3000 rpm in JS -5.2),   4 C.
Allow  centrifuge to decelerate without breake.
We have not  attempted to determine whether deceleration without braking is critical to this  procedure. However, we do not use the breake for this step or for the  subsequent centrifugation steps.
Pour off subernant and resuspend each pellet in 10 ml ice-cold CaCl2 solution.Re-suspension should be performed very gently and all cells kept on ice.
Cetrifuge cells 5 min at 1100  g, 4 C. Discard supernatant and re-suspend each pellet in 10 mL ice-cold CaCl2 solution. Keep re-suspend cells on ice for 30 min.
Centrifuge cells 5 min at 1100 g, 4 C. Discard supernatant and re-suspend each pellet in 2 mL ice-cold CaCl2 solution.
It is important to re-suspend this final pellet well. The suspension may be left on ice for several days.
Dispense cells into prechilled, sterile polypropylene tubes (250 ml aliquots are convenient). Freeze immediately at -70 C.
Transformation
Aliquot  10 ng of DNA in a volume 10 to 25  mL  into a sterile 15 mL round bottom test tube and place on ice.
Plasmid DNA  can be used directly from ligation reactions. When this is done , more DNA is  usually used. However, if there is <1 mg of DNA in the ligation reaction, or  if the ligation reaction is from low gelling/melting temperature agarose, it is  wise to dilute the ligation mix.
Rapidly  thaw competent cells by warming between hands and dispense 100 mL immediately  into test tubes containing DNA. Gently swirl tubes to mix, then place on ice  for 10 min.
Competent  cells should be used immediately after thawing. Remaining cells should be  discarded rather than refrozen
Heat  shock cells by placing tubes into a 42 C water bath for 2 min
Alternatively  incubate at 37 C for 5 min.
Add 1 ml  LB medium to each tube. Place each tube on a roler drum at 250 rpm for 1 hr at  37 C.
Plate  aliquots of transformatin culture on LB /ampicillin or other approprate  antibiotic- containing plates.
It is  convenient to plate several different dilutions of each transformation culture.  The remainder of the mixture can be stored at 4 C for subsequent platings.
When  plates are dry, incubate 12 to 16 hr at 37 C.
Plasmid isolation
This procedure is performed with Fermentas GeneJET™ Plasmid Miniprep Kit
Pick a single  colony
Inoculate  in 5 mL LB medium for high-copy or 10 mL for low-copy of LB medium supplemented  with the appropriate selection antibiotic.
Incubate at  225 rpm not longer than 12-14 h at 37 C .
Centrifuge  at 4000 rpm for 5 min at 4 C .
Discard the  supernatant and keep pellet.
Resuspended  the pellet with 250 ul  Resuspension solution. (Bacteria should be resuspended completely by vortexing  until no cell clumps remain)
Transfer the  cell suspension to eppendorf.
Add 250 uL Lysis Solution and mix gently by  inverting the tube 4-6 times until the solution becomes slightly clearPlate  aliquots of transformatin culture on LB /ampicillin or other approprate  antibiotic- containing plates.
Do not  vortex!
Do not incubate for more than 5 min. (To avoid denaturation of  supercoiled plasmid DNA!)
Add 350  uL Neutralization solutionand mix immediately and throughly by inverting the  tube 4-6 times.
(The neutralized bacterial lysate is cloudy and viscous)  ("throughly" to avoid localized precipitation of bacterial cell  debris)
Centrifuge  at 13000 rpm for 5 min to precipitate cell debris and chromosomal DNA.
Transfer  the supernatant to spin column (about 600 uL).
(Avoid disturbing or transferring the white precipitate)
Centrifuge  at 13000 rpm for 1 min.
Discard  the flow-through.
Place the  column back into same collection tube.
Add 500 uL  Wash solution to spin column.
Centrifuge  at 13000 for 1 min
Discard  the flow-through.
Repeat  this step with using 500 uL Wash solution.
Discard  the flow-through
Centrifuge  at 13000 rpm for an additional 1 min to remove residual Wash solution.
(This step is  essential to avoid residual ethanol in plasmid preps)
Transfer spin column into a fresh 1.5 mL epp.
Add 50 uL Elution buffer into center of the  spin column membrane to elute the plasmid DNA
Do not contact the membrane with pipette tip!
Incubate  for 2 min at 25 C. 
To increase  yield incubation is done for 2 min at heat block at 42 C
Centrifuge  at 13000 for 2 min.
Discard  the column and store the purified plasmid DNA at -20 C.
OPTIONAL= additional elution step with elution buffer or water. This  step increases the yield by 10-20 %.
NOTE:  For elution of plasmids ≥ 20 kb, prewarm  Elution buffer to 70 C before applying.
Restriction  digestion
This procedure  is performed with NEB  BioBrick™ Assembly  Kit
Digest  Upstream Part with EcoRI-HF and Spel
500 ng  Upstream Part Plasmid
1 uL EcorI-HF
1 uL Spe I
5 uL  10*NE  Buffer 2
0.5 uL 100*  BSA
To 50 uL add  dH2O
Digest  downstream part will  Xbal and PstI
500 ng  Downstream Part Plasmid
1 ul XbaI
5 uL 10*NE  Buffer 2
0.5 uL 100*  BSA
To 50 uL add  dH2O
Digest  the destination plasmid with EcorI-HF and Pst I:
The destination plasmid DNA should either be prepared with PCR or  contain the toxic gene (e.g, ccdB, sacB) in the cloning site to avoid the need  for gene purification.
The Destination Plasmid should also have a different antibiotic  resistance marker from both the plasmid containing the Upstream Part and the  plasmid containing the Downstream Part to avoid the need to purify the Upstream  and Downstream Parts.
500 ng  Destination Plasmid DNA
1 uL EcorI- HF
1 uL PstI
15 uL 10* NE  Buffer 2
0.5 uL 100*BSA
To 50 uL add  dH2O
 
Incubate all 3  restriction digest reactions at 37 C for 10 min and then heat inactivate at 80  C for 20 min.
AGE (Confirmation)
Background
EtBr stains only dsDNA. You cannot see ssDNA on gel
Minimum amount of DNA load:
200 ng (Aysu)
•  Each band of 1kb ladder is approx. 125 ng (2.5 ul loaded) and can be seen on gel
Preparetion
50X TAE (Tris-Acetate) Buffer
121 g/0.5 L Tris
28.6 g/0.5 L Glacial Acetic Acid
7.31 g/0.5 L EDTA
pH 8.0
Sterile 1.5 mL eppendorf tubes
Sterile long pipette tips
DNA standards
[Aysu]: Promega 1.0 kb (1.0-10Kbp), 100 bp (0.1-1.0 Kbp)
TAE Buffer
EtBr sln (10 mg/mL). Purchase as solution ! Store at RT. Light-sensitive and carcinogenic !
Sample loading buffer. Store at 4 C.
Protocol
Prepare 360 ml, 1X TAE running buffer
Prepare  gel compartment with tape (do not use parafilm)
Prepare  60 ml, 1% agarose gel (0.6 g agarose in 60 ml 1X TAE buffer)
1-2 min  in microwave. Check and swirl after each 30 sec. It must be dissolved completely.
Add 1.0  ul EtBr (10 mg/ml) per 10 ml gel solution, swirl to mix
Add add 4  ul EtBr (10 mg/ml) per 100 ml running buffer.
Pour gel  into gel compartment. Put comb.
After gel solidifies (30 min), take  out the comb and load buffer to one well to measure its capacity.
Position the  gel compartment and fill gel chamber with running buffer.
Prepare samples and standards (ladder)
Prepare samples by mixing 4 ul 6X  Loading Buffer + 20 ul sample
Load the samples (for 24 ul sample, set  pipette 24 ul).
Do not push the tip to the bottom of the wells. Make a 45 degree angle and support the pipette tip on the edge of the well. Don’t worry, gel will not brake when you take support. Slowly release the sample. Do not use second pipet push not to risk air injection.
Make the  connections: Black (-) on sample side, Red (+) opposite side
Run at 60V
5-10V/cm (distance between anode and cathode) is recommended.
90V causes high heat generation for our unit. Use 60V maximum.
For overnight runs: 2V is too low, causes diffusion
35V / 4hrs / ¾ progress
Gel can  be stored in 4 C until next day after completion
Gel Photo
Adjust zoom and position using visible light
Before  turning on UV load your settings file which has the following parameters:
Preview tab,  all three options checked
Active  image
Dynamic  integration, auto exposure, 10 frames
50/50  brigthness/contrast
0/0 hue/saturation
Maximize brigthness with camera knob  (counterclockwise)
Turn on UV light
Lower brightness from camera knob if necessary
 
Gel  extraction (For  Fermentas Gel Extraction Kit)
Excise  gel slice containing the DNA fragment using a clean scalpel or razor blade. Cut  as close to the DNA as possible to minimize the gel volume. Place the gel slice  into a pre-weighed 1.5 ml tube and weigh. Record the weight of the gel slice.
Note: If the purified fragment will be used for cloning reactions, avoid damaging the DNA through UV light exposure. Minimize UV exposure to a few seconds or keep the gel slice on a glass or plastic plate during UV illumination.
Add 1:1 volume of Binding Buffer to the gel slice (volume: weight) (e.g., add 100 µl of Binding Buffer for every 100 mg of agarose gel)
p>Note. For gels with an agarose content greater than 2%, add 2:1 volumes of Binding Buffer to the gel slice.</p>
Incubate the gel mixture at 50-60°C for 10 min or until the gel slice is completely dissolved. Mix the tube by inversion every few minutes to facilitate the melting process. Ensure that the gel is completely dissolved. Check the color of the solution. A yellow color indicates an optimal pH for DNA  binding. If the color of the solution is orange or violet, add 10 µl of 3 M sodium acetate, pH 5.2 solution and mix. The color of the mix will become yellow
Optional:  use this step only when DNA fragment is <500 bp or >10 kb long.
If the DNA fragment is <500 bp, add a 1:2 volume of 100% isopropanol to the solubilized gel solution (e.g. 100 µl of isopropanol should be added to 100 mg gel slice solubilized in 100 µl of Binding Buffer). Mix thoroughly
- If the  DNA fragment is >10 kb, add a 1:2 volume of water to the solubilized gel solution  (e.g. 100 µl of water should be added to 100 mg gel slice solubilized in 100 µl  of Binding Buffer). Mix thoroughly.
Transfer  up to 800 µl of the solubilized gel solution (from step 3 or 4) to the GeneJET™  purification column. Centrifuge for 1 min. Discard the flow-through and place  the column back into the same collection tube. 
Note. If  the total volume exceeds 800 µl, the solution can be added to the column in  stages. After each application, centrifuge the column for 30-60 s and discard  the flow-through after each spin. Repeat until the entire volume has been  applied to the column membrane. Do not   exceed 1 g of total agarose gel per column.
Optional:  use this additional binding step only if the purified DNA will be used for sequencing <p>Add 100 µl of Binding Buffer to the GeneJET™ purification column. Centrifuge for 1 min. Discard the flow-through and place the column back into the same collection tube.
 
Add 700  µl of Wash Buffer (diluted with ethanol as described on p. 3) to the GeneJET™  purification column. Centrifuge for 1 min. Discard the flow-through and place  the column back into the same collection tube.
Centrifuge  the empty GeneJET™ purification column for an additional 1 min to completely  remove residual wash buffer.
Note. This  step is essential to avoid residual ethanol in the purified DNA solution. The  presence of ethanol in the DNA sample may inhibit downstream enzymatic  reactions.
Transfer  the  purification column into a clean 1.5  ml microcentrifuge tube (not included). Add 50 µl of Elution Buffer to the  center of the purification column membrane. Centrifuge for 1 min.
Note:
For low  DNA amounts the elution volumes can be reduced to increase DNA concentration.  An elution volume between 20-50 µl does not significantly reduce the DNA yield.  However, elution volumes less than 10 µl are not recommended.
If DNA  fragment is >10 kb, prewarm Elution Buffer to 65°C before applying to column
If the  elution volume is 10 µl and DNA amount is ≤5 µg, incubate column for 1 min at  room temperature before centrifugation
Discard  the GeneJET™ purification column and store the purified DNA at -20°C
Ligation
This procedure  is performed with  NEB BioBrick™ Assembly  Kit
Ligate  the Upstream and Downstream Parts into digested Destination Plasmid.
2 uL Upstream  Part Digestion
2 uL  Downstream Part Digestion
2 uL  Desination Plasmid Digestion
2 uL 10* T4  DNA ligase bufferl
2 uL T4 DNA  ligase buffer
11 uL dH2O
Incubate at RT  for 10 min and then heat inactivate at 80 C for 20 min.
Transform 2 uL  of the ligation product into 50 uL of competent E.Coli cells ( or other  suitable host strain).
Select using the antibiotic corresponding to the  Destination Plasmid.
Glycerol Stock Preparation
Centrifuge  5 mL overnight cells at 5000 rpm for 5 min.
Discard 4  mL supernatant, remain 1 mL with pellet.
Add 176 uL 15% glycerol  and resuspend the pellet.
Aliquot  100 uL and immediately freeze at -70 C.
Cell  imaging
Checklist Procedure
Suspension culture preparation
Dissolve a single colony in 10 mL  LB+Amp
Incubate 14-15 hrs, shaking at  225 rpm
Scale-up of the suspension culture
Add 2-3 mL of the suspension culture to 100 mL LB+Amp
Incubate 5-6 hrs, shaking at 225 rpm
Equalizing OD readings
Take 10 mL samples from GFP/RFP & NC cultures
Centrifuge at 10000 rpm for 5  mins
Discard supernatant
Resuspend pellet in 500 uL 1X PBS
Take 250 uL & dilute to 3 mL  with 1X PBS
Read in UV-Vis at 600 nm
Multiply the OD readings by 12
Dilute remaining 250 uL samples as  required to make all samples have the same ODvalue, final volume should also be 250 uL
Fluorescent  signal reading
Set  parameters
[GFP]
Mode:  Emission scan
Config:  Digital
Slit: 1 turn  / 2 nm
Blaze: 750  nm
Excitation:  395 nm
Emission  range: 475-600 nm
Average: 3
Step size:  5 nm
[RFP]
Mode:  Emission scan
Config:  Digital
Slit: 1  turn / 2 nm
Blaze: 750  nm
Excitation:  570 nm
Emission  range: 585-650 nm
Average: 3
Step size:  5 nm
Transfer  samples to small 150 uL cuvette and scan
Data Analysis
Emission  maxima:
GFP 515 nm
RFP 605 nm
Record  intensity values at 509 nm for GFP & 606 nm for RFP
Subtract NC  intensity value from GFP/RFP value
Result is  the fluorescent signal of the protein in the bacteria
Protein expression
Device Experiments
Retrieved from " http://2011.igem.org/Team:METU-Ankara/lab "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
