<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Delft_results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Delft/results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Delft/results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Delft/results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Delft/results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Menu</TITLE></P><CENTER><UL id="navLaksh"><LI class="on"><A href="https://2012.igem.org/Team:TU-Delft">Home</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Team2">Team</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/overview">Wetlab</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/overview" style="left">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part1" style="left">Receptor</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part2" style="left">Reporter</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part3" style="left">Snifferomyces</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/protocols" style="left">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Snifferometer" style="left">Sniffer-o-meter</A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=TU-Delft" style="left" target="_blank">Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/receptordesign" style="left">DIY Receptor Design</A></LI></UL><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling" style="left">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StructuralModeling" style="left">Structural Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/SingleCellModel" style="left">Single Cell Pathway Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Analysis" style="left">Pathway Model Analysis</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/Diffusion" style="left">Diffusion Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StochasticSensitivitySpecificityAnalysis" style="left">Sensitivity-Specificity Analysis</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/informationtheory" style="left">Information Processing Model </A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Notebook" style="left">Week Diary</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/brainstorming" style="left">Brainstorming</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/HP">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/HumanOutreach" style="left">Human Outreach</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/HP">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Publicity" style="left">Publicity</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Games" style="left">Games</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Ethics" style="left">Ethics</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Safety" style="left">Safety</A></LI><LI class="off"><A>More</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/crowdfunding" style="left">Crowdfunding</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Acknowledgements" style="left">Acknowledgments</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Collaboration" style="left">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/achievements" style="left">Achievements</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Contact" style="left">Contact</A></LI></CENTER><DIV id="contentbox" style="text-align:justify;"><H2>Making knockout strain (far1::kanmx, gpa1::URA) and preparation of knocking out atf1</H2><H3>Week of 05-07-12</H3><P>For making a functional knockout, yeast strains BY4741; Mat a; his3D1; leu2D0; met15D0; ura3D0; YJL157c::kanMX4 and BY4741; Mat a; his3D1; leu2D0; met15D0; ura3D0; YHR005c::kanMX4 were used (Euroscarf). 
	Also knockout cassette pUG72 is used (Euroscarf). The LoxP-Ura-LoxP is elongated using the pFx polymerase protocol. The PCR program and primer sequences in table 1 yielded a product which is put on a gel shown in figure 1. Here a band can be recognized between 2000 and 1500 nucleotides, which corresponds to the 1669 nucleotide PCR product.
	</P><H6>Table 1 PCR program for elongation of knockout cassette for knocking out Far1 and Gpa1 and primers used for elongation.</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD rowspan="3" id="tdunderline">5x</TD><TD id="tdunderline">95</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">2:00</TD></TR><TR><TD id="tdunderline">51</TD><TD id="tdunderline">Annealing</TD><TD id="tdunderline">1:00</TD></TR><TR><TD id="tdunderline">68</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">2:00</TD></TR><TR><TD id="tdunderline" rowspan="3">25x</TD><TD id="tdunderline">95</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">2:00</TD></TR><TR><TD id="tdunderline">61</TD><TD id="tdunderline">Annealing</TD><TD id="tdunderline">1:00</TD></TR><TR><TD id="tdunderline">68</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">2:00</TD></TR></TBODY></TABLE><TABLE id="tbtext"><TBODY><TR><TD id="tdunderline"><B>GPA ko fw</B> TTAGCATCACATCAATAATCCAGAGGTGTATAAATTGATATATTAAGGTAGGAAATAATGCAGCTGAAGCTTCGTACGC</TD></TR><TR><TD id="tdunderline"><B>GPA ko rv</B> TGCATCTTCGGAAACAGAATTTACGTATCTAAACACTACTTTAATTATACAGTTCCTTCAGCATAGGCCACTAGTGGATCTG</TD></TR><TR><TD id="tdunderline"><B>FAR1 ko fw</B> ACACAAAGTCTATAGATCCACTGGAAAGCTTCGTGGGCGTAAGAAGGCAATCTATTAATGCAGCTGAAGCTTCGTACGC</TD></TR><TR><TD id="tdunderline"><B>FAR1 ko rv</B> GAAAAAAAAAAAAGGAAAAGCAAAAGCCTCGAAATACGGGCCTCGATTCCCGAACTACTAGCATAGGCCACTAGTGGATCTG</TD></TR><TR><TD id="tdunderline"><B>GPA ko fw short</B> TAATCCAGAGGTGTATAAATTGATATATTAAGGTAGGAAATAATGCAGCTGAAGCTTCGTACGC</TD></TR><TR><TD id="tdunderline"><B>GPA ko rv short</B> AGAATTTACGTATCTAAACACTACTTTAATTATACAGTTCCTTCAGCATAGGCCACTAGTGGATCTG</TD></TR><TR><TD id="tdunderline"><B>FAR1 ko fw short</B> ATCCACTGGAAAGCTTCGTGGGCGTAAGAAGGCAATCTATTAATGCAGCTGAAGCTTCGTACGC</TD></TR><TR><TD id="tdunderline"><B>FAR1 ko rv short</B> AAAAGCAAAAGCCTCGAAATACGGGCCTCGATTCCCGAACTACTAGCATAGGCCACTAGTGGATCTG</TD></TR><TR><TD id="tdunderline"><B>ATF1 ko fw</B> gaaaataaaaaacggCACTTCATCAGTATCACAAATACCATCAATTTATCAGCTCTCATGCAGCTGAAGCTTCGTACGC </TD></TR><TR><TD id="tdunderline"><B>ATF1 ko rv</B> ggttatttacacgacatAATCATATTGTCGAATAATATCAGTCAAGCATCATGTGAGATCTAGCATAGGCCACTAGTGGATCTG </TD></TR><TR><TD id="tdunderline"><B>ATF1 ko fw short</B> CACTTCATCAGTATCACAAATACCATCAATTTATCAGCTCTCATGCAGCTGAAGCTTCGTACGC </TD></TR><TR><TD id="tdunderline"><B>ATF1 ko rv short</B> AATCATATTGTCGAATAATATCAGTCAAGCATCATGTGAGATCTAGCATAGGCCACTAGTGGATCTG  </TD></TR></TBODY></TABLE><H6>Figure 1: 1% agarose in TAE ~45 run on 80 Volts. In the picture can be seen: 1 SmartLadder, 2 Far1 short PCR product, 3 Far1 long PCR product, 4 Gpa1 short PCR product, 5 Gpa1 long PCR product, 6 Atf1 short PCR product, 7 Atf1 long PCR product.</H6><P>Gel extraction of the GPA1 PCR product is performed using the Qiagen gel extraction kit. The end concentration of this step is measured to be 167.2 ng/µl using the nanodrop nucleotide program.</P><P>The PCR reactions using FAR1 and ATF1 primers with the same conditions as table 1 but with the PCR program according to table 2 is performed yielding no result.</P><H6>Table 2 PCR program for elongation of knockout cassette for knocking out Far1 and Gpa1 and primers used for elongation.</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline" rowspan="3">30 x</TD><TD id="tdunderline">95</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">2:00</TD></TR><TR><TD id="tdunderline">60/62/66/69</TD><TD id="tdunderline">Annealing</TD><TD id="tdunderline">1:00</TD></TR><TR><TD id="tdunderline">68</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">2:00</TD></TR></TBODY></TABLE><P>Drop out medium is made and solutions for transformations in yeast strains is prepared. The formula of the dropout media can be seen in the media protocol.</P><H3>Week of 09-07-12</H3><P>Yeast strain BY4741; Mat a; his3D1; leu2D0; met15D0; ura3D0; YJL157c::kanMX4 is transformed with the GPA1 ko PCR product, according to the yeast transformation protocol. 100 ng of DNA is used for transformation. After 3 days 3 colonies could be seen on the plate. There where zero colonies on the control plate.
	To check whether the knock-out reaction was successful, colony PCR of the three colonies where performed. Primers A and D bind to 5’ upstream and 3’ downstream regions of the GPA1 gene, the B and C primers anneal in the URA gene which is knocked in. For the first reaction, only AD reaction is tried A negative control is the yeast strain without transformation. Before initiation the colonies where boiled for 10 minutes in a PCR tube in 10 µl 2 mM NaOH solution of which 2 µl is taken. First conditions according to table 3 where used, using taq Mastermix PCR solution. This yielded no results when put on gel.</P><H6>Table 3 PCR conditions of knock-out PCR check</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">2:00</TD></TR><TR><TD id="tdunderline" rowspan="3">30x</TD><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">1:00</TD></TR><TR><TD id="tdunderline">58</TD><TD id="tdunderline">Annealing </TD><TD id="tdunderline">2:00</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">2:00</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">10:00</TD></TR></TBODY></TABLE><TABLE id="tbtext"><TBODY><TR><TD id="tdunderline"><B>GPA1 A (fw)</B>TCTGCGTATTCTTCCTTGTAGAAAT</TD></TR><TR><TD id="tdunderline"><B>GPA1 D (rv) </B>GAATTCGAGATAATACCCTGTCCTT</TD></TR></TBODY></TABLE><P>
	A second run with conditions according to table 4 was performed. Now colonies where dissolved in 20 µl 2 mM NaOH of which 18 µl was taken (the maximum amount). This yielded no results when put on gel.
	</P><H6>Table 4 PCR on pUG73 cassette with Phusion polymerase summary.</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">2:00</TD></TR><TR><TD id="tdunderline" rowspan="3">25x</TD><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">1:00</TD></TR><TR><TD id="tdunderline">56</TD><TD id="tdunderline">Annealing</TD><TD id="tdunderline">1:00</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">4:00</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">10:00</TD></TR></TBODY></TABLE><TABLE id="tbtext"><TBODY><TR><TD id="tdunderline"><B>GPA1 A (fw) </B>TCTGCGTATTCTTCCTTGTAGAAAT</TD></TR><TR><TD id="tdunderline"><B>GPA1 D (rv) </B>GAATTCGAGATAATACCCTGTCCTT</TD></TR></TBODY></TABLE><H3>Week of 18-07-12</H3><P>A new PCR kit, phusion polymerase, is tried. PCR conditions and mix are shown in table 5.</P><H6>Table 5 colony PCR on WT yeast with Phusion polymerase summary.</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline">98</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">0:30</TD></TR><TR><TD id="tdunderline" rowspan="3">25x</TD><TD id="tdunderline">98</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">0:20</TD></TR><TR><TD id="tdunderline">66</TD><TD id="tdunderline">Annealing</TD><TD id="tdunderline">0:30</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">2:30</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">10:00</TD></TR></TBODY></TABLE><TABLE id="tbtext"><TBODY><TR><TD id="tdunderline"><B>GPA1 A (fw) </B>TCTGCGTATTCTTCCTTGTAGAAAT</TD></TR><TR><TD id="tdunderline"><B>GPA1 D (rv) </B> GAATTCGAGATAATACCCTGTCCTT</TD></TR><TR><TH>Phusion mix</TH></TR><TR><TD id="tdunderline">5x Phusion buffer</TD><TD>10 µl</TD></TR><TR><TD id="tdunderline">10 mM dNTP’s</TD><TD>1 µl</TD></TR><TR><TD id="tdunderline">Primer A</TD><TD>5 µl</TD></TR><TR><TD id="tdunderline">Primer D</TD><TD>5 µl</TD></TR><TR><TD id="tdunderline">DNA template (boiled)</TD><TD>2 µl</TD></TR><TR><TD id="tdunderline">DMSO</TD><TD>1.5 µl</TD></TR><TR><TD id="tdunderline">Polymerase</TD><TD>0.5 µl</TD></TR><TR><TD id="tdunderline">H<SUB>2</SUB>O</TD><TD>25 µl</TD></TR></TBODY></TABLE><P>The PCR product is put on gel, shown in figure 2. Here a band can be observed in lane 3 with a length of ~4000 nt. This PCR product is inconsistent with both the URA gene (~3000 nt) or the original gene (2000 nt). We ordered new primers, using different regions of the gene. Also primers checking the insert where redesigned.</P><H6>Figure 2: 0.8% Agarose gel in TAE, 40 minutes 80 V. In the picture can be seen:  1 DNA smartladder, 2 PCR product with DNA of colony 1, 3 PCR product with DNA of colony 2, 4 PCR product with DNA of colony 3.</H6><H3>Week of 20-07-12</H3><P>To check whether the new primers were working, colony PCR of wildtype yeast was performed. New primers were ordered for the A and D primers, called confident primers (conf). No product occurred in the negative control, PCR of wildtype yeast with the AB and CD primers was also performed. Before initiation the colonies were boiled for 10 minutes in a PCR tube in 2 mM NaOH solution. Conditions according to table 6 were used, using the Phusion polymerase. No results were seen on gel.</P><H6>Table 6 Phusion PCR reaction.</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline">98</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">0:30</TD></TR><TR><TD id="tdunderline" rowspan="3">30x</TD><TD id="tdunderline">98</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">0:10</TD></TR><TR><TD id="tdunderline">57</TD><TD id="tdunderline">Annealong</TD><TD id="tdunderline">0:30</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">1:15</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">10:00</TD></TR></TBODY></TABLE><TABLE id="tbtext"><TBODY><TR><TD id="tdunderline"><B>GPA1 A </B>TCTGCGTATTCTTCCTTGTAGAAAT</TD></TR><TR><TD id="tdunderline"><B>KO B </B>CGCCAAGGGTAGAGATCCTAAG</TD></TR><TR><TD id="tdunderline"><B>KO C </B>CTTCACGCAGGATGACAGTTC</TD></TR><TR><TD id="tdunderline"><B>GPA1 D </B>GAATTCGAGATAATACCCTGTCCTT</TD></TR><TR><TD id="tdunderline"><B>GPA1 A conf </B>CGTCCTTCTGCGTATTCTTCC</TD></TR><TR><TD id="tdunderline"><B>GPA1 D conf </B>CCGAGTATTTACCAGGGAGAAG</TD></TR></TBODY></TABLE><P>To check whether the problem was due to the treatment we performed a reaction with 18SrDNA Eukaryotic primers: EukAt &amp; Euk516GC which amplifies a region of 563 nt. First a PCR reaction with taq mastermix was performed and pretreatment of the colonies was dissolving in 10 µl 2mM NaOH and taking 3 µl of this mixture as DNA template. 0.25 µl primers were added. The PCR product was put on gel which is shown in figure 3. PCR reactions were performed on wildtype yeast. A band of ~2000 nt can be observed in lane 3 which corresponds to the length of the gene. Further bands of ~500 nt can be observed in lane 2 and 8 which correspond to the expected Euk516GC primer product. Further can be seen that the band in lane 8 is lighter. This PCR reaction was performed on a 15 minute pretreated wildtype colony. Longer pretreatment will make worse the outcome of the reaction.
	</P><H6>Table 7 Taq PCR reaction on wildtype yeast and knock-out colonies</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">4:00</TD></TR><TR><TD id="tdunderline" rowspan="3">35x</TD><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">0:30</TD></TR><TR><TD id="tdunderline">55</TD><TD id="tdunderline">Annealing </TD><TD id="tdunderline">0:40</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">0:40</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">10:00</TD></TR></TBODY></TABLE><TABLE id="tbtext"><TBODY><TR><TD id="tdunderline"><B>GPA1 A (old) </B>TCTGCGTATTCTTCCTTGTAGAAAT</TD></TR><TR><TD id="tdunderline"><B>GPA1 D (old) </B>GAATTCGAGATAATACCCTGTCCTT</TD></TR><TR><TD id="tdunderline"><B>GPA1 A conf </B>CGTCCTTCTGCGTATTCTTCC</TD></TR><TR><TD id="tdunderline"><B>GPA1 D conf </B>CCGAGTATTTACCAGGGAGAAG</TD></TR><TR><TD id="tdunderline"><B>KO B </B>CGCCAAGGGTAGAGATCCTAAG</TD></TR><TR><TD id="tdunderline"><B>KO C </B>CTTCACGCAGGATGACAGTTC</TD></TR></TBODY></TABLE><H6>Figure 3 0.8% Agarose in TAE, run 45 minutes on 80 V. In the picture can be seen:  1 DNA smartladder, 2 PCR product with DNA of colony 2 and Euk516r primers, 3 PCR product with DNA of colony 2 with GPA1 A conf and GPA1 D conf primers, 4 PCR product with DNA of colony 2 with GPA1 A and GPA 1 D primers (old), 5 negative control, GPA1 D conf added, 6 PCR product of colony 2 without boiling and Euk516 primers, 7 PCR product of colony 2 with only one of the Euk516r primers, 8 PCR product of colony 2 with 15 minutes boiling and Euk516r primers, 9 DNA smartladder, 10 Wildtype (WT) with AD primers, 11 WT with AB primers, 12 WT with CD primers, 13 WT with AD old primers, 14 Colony 1 with AD primers, 15 Colony 1 with AD old primers, 16 Colony 2 with AD primers , 17 Colony 2 with AD old primers, 18 Colony 3 with AD primers, 19 Colony 3 with AD old primers, 20 DNA smartladder.</H6><P>Colony PCR, on the KO colonies, was performed with the use of taq Mastermix DNA. Again, the pretreatment was by boiling in 2 mM NaOH. Elongation period was extended (before, conditions for Eukaryotic 18SrDNA were optimal). Gel is shown in Figure 4. Here, one can see, that In lane 5, 9, 13 and 17 a band can be seen of ~500 nt, which corresponds to the 18SrDNA PCR product. Further, on lane 7, in which colony 2 is taken, a band of ~600 nt that corresponds to the expected PCR product of the AB reaction can be seen. Further in lane 14 in which wildtype yeast is taken, a band can be seen of ~2000 nt that corresponds to the expected PCR product of AD reaction when the gene is not knocked out.</P><H6>Table 8 Taq colony PCR summary</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">4:00</TD></TR><TR><TD id="tdunderline" rowspan="3">35x</TD><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">0:30</TD></TR><TR><TD id="tdunderline">55</TD><TD id="tdunderline">Annealing </TD><TD id="tdunderline">0:40</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">2:15</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">10:00</TD></TR></TBODY></TABLE><TABLE id="tbtext"><TBODY><TR><TD id="tdunderline"><B>GPA1 A (old) </B>TCTGCGTATTCTTCCTTGTAGAAAT</TD></TR><TR><TD id="tdunderline"><B>GPA1 D (old) </B>GAATTCGAGATAATACCCTGTCCTT</TD></TR><TR><TD id="tdunderline"><B>GPA1 A conf </B>CGTCCTTCTGCGTATTCTTCC</TD></TR><TR><TD id="tdunderline"><B>GPA1 D conf </B>CCGAGTATTTACCAGGGAGAAG</TD></TR><TR><TD id="tdunderline"><B>KO B </B>CGCCAAGGGTAGAGATCCTAAG</TD></TR><TR><TD id="tdunderline"><B>KO C </B>CTTCACGCAGGATGACAGTTC</TD></TR></TBODY></TABLE><H3>Week of 24-07-12</H3><P>Cultures of colony 2 and wildtype yeast where grown in YPG (YP+2% glucose) overnight for chromosomal extraction using the MO BIO chromosomal extraction kit (with bead lysis).
Since at the time we weren’t positive of really knocking out GPA1 we wanted also to make the FAR1 knock-out DNA (to transform in a ?gpa1 strain). The primers for making a FAR1 knock-out and ATF1 knock-out by elongating LoxP-URA-LoxP on the PUG73 cassette was tried by using Pfx polymerase. Variation of annealing temperature was performed by setting annealing temperatures to 56, 60 and 65 C.  The conditions are summarized in table 9.</P><H6>Table 9 PCR on pUG73 cassette with Phusion polymerase summary.</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline" rowspan="3">5x</TD><TD id="tdunderline">95</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">2:00</TD></TR><TR><TD id="tdunderline">52</TD><TD id="tdunderline">Annealing </TD><TD id="tdunderline">1:00</TD></TR><TR><TD id="tdunderline">68</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">2:00</TD></TR><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline" rowspan="3">25x</TD><TD id="tdunderline">95</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">2:00</TD></TR><TR><TD id="tdunderline">56/60/65</TD><TD id="tdunderline">Annealing </TD><TD id="tdunderline">1:00</TD></TR><TR><TD id="tdunderline">68</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">2:00</TD></TR></TBODY></TABLE><TABLE id="tbtext"><TBODY><TR><TD id="tdunderline"><B>FAR1 ko fw short </B>ATCCACTGGAAAGCTTCGTGGGCGTAAGAAGGCAATCTATTAATGCAGCTGAAGCTTCGTACGC</TD></TR><TR><TD id="tdunderline"><B>FAR1 ko rv short </B>AAAAGCAAAAGCCTCGAAATACGGGCCTCGATTCCCGAACTACTAGCATAGGCCACTAGTGGATCTG</TD></TR><TR><TD id="tdunderline"><B>ATF1 ko fw short </B>CACTTCATCAGTATCACAAATACCATCAATTTATCAGCTCTCATGCAGCTGAAGCTTCGTACGC</TD></TR><TR><TD id="tdunderline"><B>ATF1 ko rv short </B>AATCATATTGTCGAATAATATCAGTCAAGCATCATGTGAGATCTAGCATAGGCCACTAGTGGATCTG </TD></TR></TBODY></TABLE><P>Simultaneously DNA extraction of wildtype yeast and colony 2 was performed according to the yeast DNA extraction protocol. With this product, AD and CD reactions were performed, of which the conditions are summarized in table 10.</P><H6>Table 10 Taq genomic DNA PCR</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">4:00</TD></TR><TR><TD id="tdunderline" rowspan="3">30x</TD><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">0:30</TD></TR><TR><TD id="tdunderline">55</TD><TD id="tdunderline">Annealing</TD><TD id="tdunderline">0:40</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">2:15</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">10:00</TD></TR></TBODY></TABLE><TABLE id="tbtext"><TBODY><TR><TD id="tdunderline"><B>GPA1 A conf </B>CGTCCTTCTGCGTATTCTTCC</TD></TR><TR><TD id="tdunderline"><B>GPA1 D conf </B>CCGAGTATTTACCAGGGAGAAG</TD></TR><TR><TD id="tdunderline"><B>KO C </B>CTTCACGCAGGATGACAGTTC</TD></TR></TBODY></TABLE><P>Figure 4 shows the PCR product on gel. Here a clear band can be observed in lane 2 which corresponds to the expected CD PCR product. Further, lane 5 shows a clear band corresponding to the PCR product of the AD primers with the gene. The AD primers don’t seem to work with the knock-out colony, where both combinations AB and CD worked properly. Further, lane 6 &amp; 7 show no bands, which is strange because until that moment, eukaryotic primers always have worked. 
Further Figure 4 shows bands in Lane 9 and 10, which correspond to the expected length of the elongated LoxP-URA-LoxP PCR product with FAR1 primers. The PCR program is shown to be at the optimum of 56 C (at highest). Also in lane 19 and 20 bands can be observed corresponding to the expected length of the elongated LoxP-URA-LoxP PCR product with ATF1 primers. This PCR program seems to have an optimum annealing temperature from 65 C.
</P><H6>Figure 4: 1% agarose in TAE ~45 run on 80 Volts. In the picture can be seen: 1 SmartLadder, 2 Genomic DNA of colony 2 with CD primers, 3 Genomic DNA of wildtype with CD primers, 4 Genomic DNA of colony 2 with AD primers, 5 Genomic DNA of wildtype with AD primers, 6 Genomic DNA of colony 2 with Euk516 primers, 7 Genomic DNA of wildtype with Euk516 primers, 8 DNA Smartladder, 9 &amp;10 Far1 short PCR product at 56 C, 11&amp;12 Far1 short PCR product at 60 C, 13&amp;14 Far1 short PCR product at 65 C, 15&amp;16 ATF1 short PCR product at 56 C, 17&amp;18 ATF1 short PCR product at 60 C, 19&amp;20 ATF1 short PCR product at 65 C.</H6><H3>Week of 07-08-12</H3><P>Colonies 1, 2 and 3 and a wildtype colony where grown overnight on YPG.A final PCR reaction was performed on extracted genomic DNA of the grown strains, using an annealing gradient of 45, 55 and 65 C and 15 ng or 30 ng template DNA. Reactions AD, AB and CD where performed according to the conditions summarized in table 11. PCR product is put on gel and shown in figure 5. Here can be seen that colony 2 (B) shows both expected band length in the AB and CD reactions, but no bands in the AD reaction. Colony 1 (A) shows only bands in the AD reaction, so the knock-out was unsuccessful. Colony 3 (C) shows inconsistent bands in reactions AD (lane 5 and 7), BC (lane 10) and CD (lane 14 and 15), also not of the expected length.</P><H6>Table 11 Taq PCR conditions for KO check.</H6><TABLE id="tbtext"><TBODY><TR><TH>Repeats</TH><TH>Temperature</TH><TH>Duration</TH></TR><TR><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">4:00</TD></TR><TR><TD id="tdunderline" rowspan="3">35x</TD><TD id="tdunderline">94</TD><TD id="tdunderline">Melting</TD><TD id="tdunderline">0:30</TD></TR><TR><TD id="tdunderline">45/55/65</TD><TD id="tdunderline">Annealing</TD><TD id="tdunderline">0:40</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">2:15</TD></TR><TR><TD id="tdunderline">72</TD><TD id="tdunderline">Elongation</TD><TD id="tdunderline">10:00</TD></TR></TBODY></TABLE><TABLE id="tbtext"><TBODY><TR><TD id="tdunderline"><B>GPA1 A conf </B>CGTCCTTCTGCGTATTCTTCC</TD></TR><TR><TD id="tdunderline"><B>GPA1 D conf </B>CCGAGTATTTACCAGGGAGAAG</TD></TR><TR><TD id="tdunderline"><B>KO B </B>CGCCAAGGGTAGAGATCCTAAG</TD></TR><TR><TD id="tdunderline"><B>KO C </B>CTTCACGCAGGATGACAGTTC</TD></TR></TBODY></TABLE><H6>Figure 5: 1% agarose gel in TAE, 40 min run on 100 V. PCR products of Genomic extracted DNA. Colony 1 (A), Colony 2 (B), and Colony 3 (C) are shown. In the picture can be seen: 1 DNA Smarladder, 2 AD reaction with 15 ng template and 45 C, 3 AD reaction with 15 ng template and 55 C, 4 AD reaction with 15 ng template and 65 C, 5 AD reaction with 30 ng template and 45 C, 6 AD reaction with 30 ng template and 55 C, 7 AD reaction with 30 ng template and 65 C, 8 AB reaction with 15 ng template and 45 C, 9 AB reaction with 15 ng template and 55 C, 10 AB reaction with 15 ng template and 65 C, 11 AB reaction with 30 ng template and 45 C, 12 AB reaction with 30 ng template and 55 C, 13 AB reaction with 30 ng template and 65 C, 14 CD reaction with 15 ng template and 45 C, 15 CD reaction with 15 ng template and 55 C, 16 CD reaction with 15 ng template and 65 C, 17 CD reaction with 30 ng template and 45 C, 18 CD reaction with 30 ng template and 55 C, 19 CD reaction with 30 ng template and 65 C, 20 smartladder.</H6><P>We concluded that in colony 2 GPA1 has successfully been knocked out. The following test is to transform Fus1-GFP into this ?far1 ?gpa1 yeast and to see difference in fluorescence output when compared to ?far1 yeast strain when induced by pheromone alpha. Expected is that due to lack of alpha subunit of the pheromone receptor, no or less light signal is detected.</P><H6>Written by Mark Weijers.</H6></DIV><A><DIV class="printfooter">
Retrieved from &quot;</DIV></A><A href="http://2012.igem.org/Team:TU-Delft/results">http://2012.igem.org/Team:TU-Delft/results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Delft/results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Delft/results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Delft/results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Delft/results&amp;oldid=168429" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>