<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Delft_part1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Delft/part1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Delft/part1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/part1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/part1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Delft/part1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Delft/part1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Menu</TITLE></P><CENTER><UL id="navLaksh"><LI class="on"><A href="https://2012.igem.org/Team:TU-Delft">Home</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Team2">Team</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/overview">Wetlab</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/overview" style="left">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part1" style="left">Receptor</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part2" style="left">Reporter</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part3" style="left">Snifferomyces</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/protocols" style="left">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Snifferometer" style="left">Sniffer-o-meter</A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=TU-Delft" style="left" target="_blank">Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/receptordesign" style="left">DIY Receptor Design</A></LI></UL><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling" style="left">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StructuralModeling" style="left">Structural Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/SingleCellModel" style="left">Single Cell Pathway Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Analysis" style="left">Pathway Model Analysis</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/Diffusion" style="left">Diffusion Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StochasticSensitivitySpecificityAnalysis" style="left">Sensitivity-Specificity Analysis</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/informationtheory" style="left">Information Processing Model </A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Notebook" style="left">Week Diary</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/brainstorming" style="left">Brainstorming</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/HP">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/HumanOutreach" style="left">Human Outreach</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/HP">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Publicity" style="left">Publicity</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Games" style="left">Games</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Ethics" style="left">Ethics</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Safety" style="left">Safety</A></LI><LI class="off"><A>More</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/crowdfunding" style="left">Crowdfunding</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Acknowledgements" style="left">Acknowledgments</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Collaboration" style="left">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/achievements" style="left">Achievements</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Contact" style="left">Contact</A></LI></CENTER><TITLE>Receptor</TITLE><DIV id="contentbox" style="text-align:justify;"><H2>Content</H2><A href="#P10"> Introduction</A><A href="#P8">Parts </A><A href="#P7">Results </A><A href="#P6"> Conclusions</A><A href="#P5"> Recommendations </A><A href="#P9">References</A><A name="P10"><H2>Introduction</H2></A><H3>Olfactory receptors</H3><P>
Animals sense their chemical environment through olfactory receptors (ORs). The olfactory receptors are a large group of proteins belonging to a subfamily of G protein-coupled receptors (GPCRs) that bind odorant ligands. If the receptor is activated by a ligand, the confirmation of the receptor is changed and there is an interaction with the a-subunit of the G-protein. This causes dissociation of the a-subunit from the Gßα dimer and the signal is propagated [1]. Because of the sensitivity and selectivity of the of the olfactory system it can be of value in detection of environmental toxins [2] or pharmaceutical screening. In this iGEM project we aim to investigate if the ORs can be used as a diagnostics tool for tuberculosis.</P><H3>Yeast G protein-coupled receptors</H3><P>In this project we choose to work with the budding  yeast <I>Saccharomyces cerevisiae</I>  as a host  organism because it utilizes already a GPCR pathway.  Furthermore <I>S. cerevisiae</I> has been successfully used for functional expression of GPCR’s [3,4], a lot of genomic tools are available, and it has a fully characterized genome. 
In <I>S. cerevisiae</I> two GPCR cascades have been identified: a glucose sensing pathway and a mating pathway [5]. There are two sexes of yeast cells, MATa and MATa. Whenever pheromones (small peptides) of the opposite sex are bound to the specific G-protein coupled receptors (Ste2 p or Ste3p), the MAP kinase cascade is turned on,  leading to induction of mating genes such as <I>FUS1</I> and growth arrest  mediated by the <I>FAR1</I>  promoter. This mating response can be seen in the form of a morphological change, called shmoo formation.  In figure 1 an overview of the pheromone and glucose signaling pathways in <I>S. cerevisiae</I> is shown. </P><H6>Overview of pheromone and glucose signaling in <I>S. cerevisiae</I>.  Figure adapted from <I>Versele et al.</I></H6><H3>Introduction of a new olfactory receptor</H3><P>Previously it was found that that the yeast pheromone signaling pathway can be coupled to a mammalian olfactory receptor. <I> Minic et al.</I> succeeded in functional expressing the rat 17 OR and its trafficking to the plasma membrane in <I>S. cerevisiae</I>.  Between the three GPCRs that are known in <I>S. cerevisiae</I>,  Ste2, Ste3 and Gpr1, the sequence similarity is limited. Except for pheromone receptors in <I>Schizosaccharomyces pombe</I> and <I>Kluyveromyces lactis</I>, Ste2 and Ste3 are largely unrelated in sequence to other GPCRs [5]. Nevertheless, the yeast pheromone receptors can be functionally replaced by several mammalian GPCRs so that the pheromone pathway can be activated by the corresponding ligands [4]. For localization of the receptor into the membrane and a higher affinity with the alpha-subunit we made a chimeric design of the receptor, after the research of Radhika et al. [7]. For a detailed design, and how to do this yourself, look at the <A href="https://2012.igem.org/Team:TU-Delft/receptordesign">DIY receptor design page</A>.</P><H3>Niacin olfactory receptor</H3><P>The receptors GPR109A and GPR109B are known to bind the compound nicotinic acid [8].  It was previously described that GPR109B acts a low affinity receptor for nicotinic acid and GPR109A acts as a high affinity receptor for nicotinic acid and  other compounds with related pharmacology [molecular identification of high and low affinity receptors].  The chemical compound methyl nicotinate is closely related to nicotinic acid. Because one of the compounds in the breath of tuberculosis patients is methyl nicotinate [9,10], the high affinity receptor for niacin is a good candidate for testing the ‘olfactory yeast’ as a diagnostics tool. </P><H3>Banana smell olfactory receptor</H3><P>The first iGEM team of MIT 2006 made a biobrick called the ‘banana odor generator’. With this part <I>E. coli</I> cells can generate the isoamyl acetate molecule. We aim to let yeast detect this isoamyl acetate signal with a olfactory receptor. The idea is that in future work the yeast should couple this back to the bacteria to have gaseous yeast/bacteria communication on plates.  The human receptor OR1G1 and mouse receptor Olfr154 are known to react on isoamyl acetate [11] and therefor these two receptors were used in this iGEM project. </P><A name="P8"><H2>Parts</H2></A><P>A design of the receptor construct was made with the olfactory receptors placed between the N-terminal and the C-terminal part of the rat I7 receptor. As a promoter the strong constitutive <I>GPD</I> promoter is used and as a terminator the <I>CYC1</I> terminator. The receptor can be replaced by using the restriction sites BamHI and NdeI. A FLAG tag is added upstream of the receptor sequence to look at the localization of the receptor in the membrane.
The plasmid construct for the receptor expression was obtained by restriction of the synthesized receptor construct and ligation in the pRSII415 expression vector. The following biobricks are created:<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K775000 " target="_blank">BBa_K775000</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K775001 " target="_blank">BBa_K775001</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K775002 " target="_blank">BBa_K775002</A><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K775003 " target="_blank">BBa_K775003</A></P><A name="P7"><H2>Results</H2></A><H3>Transformations</H3>
The transformations with the plasmids were done in the S288C <I>S. cerevisiae</I> strain. All the plasmids were also put in a strain with a far1::kanMX4 knock-out strain. In table 1 all the strains that we made in this subpart are shown.
<CENTER><TABLE id="tbtext" width="100%"><TBODY><TR><TH>Gpr109A</TH><TH>Olfr154</TH><TH>OR1G1</TH></TR><TR><TD id="tdunderline">Normal strain</TD><TD id="tdunderline">+NR1</TD><TD id="tdunderline">+BR1</TD><TD id="tdunderline">+BR2</TD></TR><TR><TD id="tdunderline">Δfar1 strain</TD><TD id="tdunderline">Δfar1 +NR1</TD><TD id="tdunderline">Δfar1 +BR1</TD><TD id="tdunderline">Δfar1 +BR2</TD></TR></TBODY></TABLE></CENTER><P>After transformation of the plasmids in the yeast a PCR reaction was performed in order to verify if the plasmid was correctly transformed. Since the PCR reactions were performed with single colonies we expected to obtain one PCR product with the length of the receptor part. However, for all the receptors we saw multiple PCR products on the gel; products with the length of the receptor, and products indicating that only the plasmid backbone was present (without the receptor). This indicates that during growth of the yeast a part of the plasmid was emitted. </P><H6>1% Agarose Gel run on 80 V, showing <I>S. cerevisiae</I> extracted plasmid DNA of our olfactory receptor construct. Lane 2 shows DNA smartladder. Lane 1 shows a typical bands for the <I>S. cerevisae</I> plasmid extract. The bright band at the height of 2000 nt is the expected PCR band. The secondary bands observed have DNA sizes of approximately 1200 and 400-500 nt.</H6><H3>Expression and localization of the ORs</H3><DIV class="WordSection1"><A id="FLAG_setup"><H6>Setup</H6></A>
NR1, BR1 and BR2 transformed cells and WT cells were stained with conjugated anti-FLAG antibodies according to <A href="https://2012.igem.org/Team:TU-Delft/protocols#P11">the immunofluorescence staining protocol</A> and viewed under a widefield fluorescence microscope (NR1) and a confocal microscope (BR1 and BR2) with the goal of imaging the expression of our GPCR chimeras and image their localization in the cell. The image was analyzed with ImageJ to compare the fluorescence of the cell and cell membrane to the overall fluorescence of the whole picture.
<A id="FLAG_outcome"><H6>Outcome</H6></A>
It can be seen that there is expression of the receptor: +NR1 cells are fluorescent (figure 4 below) and the reference strain is very weakly fluorescent (figure 4 top). In some of the +NR1 cells there is clear halo structure visible, which indicates localization of the receptor on the membrane. Below such a typical halo is shown (figure 4). For +BR1 and +BR2 cells receptor expression can be shown also by the fluorescent signal (figure 5, other experiment). The location of the signal indicates expression unequally distributed over the cells, indicating that localization in the membrane is not fully functioning. Contrast of these pictures are not optimal. The reference strain (figure 5, top) showed very little fluorescence. Further, a small percentage of the cells (~5 %) was found to be fluorescent, indicating that constitutive mRNA expression by the GPD promoter did not lead to constitutive protein expression in all cells.
<H6>Figure 4: FLAG immunolocalization of +NR1 yeast cells expressing a receptor with DYKDDDDK tag.</H6><H6>Figure 5: FLAG immunolocalization of +BR1 and +BR2  yeast cells expressing a receptor with FLAGtag.</H6></DIV><H3>Ligand activation</H3><H6>Setup</H6><P>The next step was to test the binding of the ligand to our new introduced receptor. If the ligand binds correctly to the receptor the downstream pathway is activated and the cells should go in growth arrest. If cells are dividing the percentage of DNA is higher then for cells that are in growth arrest. By staining the DNA of the cells with a fluorescent dye we could look at the DNA content and thus if the cells enter to growth arrest. This is done with a flow cytometer.
</P><H6>Outcome <I>Niacin receptor</I></H6><H6>Flow cytometry results of +NR1 and reference cells induced with the niacin ligand. Cells were DNA stained and measured after 4.40 hours.</H6>

As controls we used a reference strain induced with niacin, and a +NR1 receptor strain without niacin induction. On the right there is our +NR1 receptor strain induced with niacin. The graphs show on the x-axis the measured intensity and on the y-axis we see the cell count. 
We see that the controls show a peak in the same region, this peak indicates that the cells are distributed over all cell cycle phases.  We can see that the peak of the induced receptor strain is shifted to a lower fluorescent intensity. This might be an indication that the cells have a lower DNA content, and that they stopped growing. 
Despite we didn’t see a response on the methyl nicotinate molecule we were very exited to have promising results on the binding of the niacin ligand! This is also in agreement with the results of the <A href="https://2012.igem.org/Team:TU-Delft/Modeling/StructuralModeling">molecular dynamics modeling</A>.


<H6><I>Outcome Banana receptor 1 and Banana receptor 2</I></H6>
When looking at the stained +BR1 and +BR2 strains with flow cytometry the data showed unclear and strangely shifted peaks. To get an idea of the behavior of the cell under influence of ligand and DNA staining, the cells were viewed with fluorescence microscopy. There was an effect of isoamylacetate on the location of the DNA stain. For all the strains this resulted in an evenly distributed glow over the whole cell after induction of isoamylacetate. Below microscopy pictures for the +BR1 strain are shown as an example. This observations could explain the unclear flow cytometry results. Another reason for that could be that the oily isoamyl acetate that drove on the medium disturbed the measurements. 
<H6>Figure 7: OR1G1 transformed cells with isoamylacetate as inducing agent at an estimated concentration of 200mM.</H6><A name="P6"><H2>Conclusions</H2></A>
All the olfactory receptors were successfully expressed in the yeast strains. For some of the receptors we observed a halo structure with FLAG-tag localization, this points to localization of the receptor into the membrane. 
When inducing the niacin receptor strain with niacin we observed that the flow cytometry peak of the induced receptor strain was shifted to a lower fluorescent intensity. This might be an indication that the cells stopped growing and thus there was a reaction on the ligand! For the banana receptor strains we had difficulties with the DNA staining in combination with the ligand Isoamyl acetate. 

<A name="P5"><H2>Recommendations</H2></A><P>During growth of yeast cells transformed with the expression vector we observed two things: not all the cells maintain the right plasmid and the cells grew slower than wild type cells. A reason for this could be that the expression of the receptor is disadvantageously for the cells. Therefor we recommend for future work to use an inducible promoter instead of a strong constitutive promoter. In that case one can make the yeast cells expressing the receptor just before testing the strain.</P><A name="P9"><H2>References</H2></A><H6>
[1] Haiqing Zhao, Lidija Ivic, Joji M. Otaki, Mitsuhiro Hashimoto, Katsuhiro Mikoshiba, Stuart Firestein*Functional Expression of a Mammalian Odorant Receptor, Science 279, 237 (1998)
[2] Venkat Radhika, Tassula Proikas-Cezanne, Muralidharan Jayaraman, Djamila Onesime, Ji Hee Ha &amp;Danny N Dhanasekaran, Chemical sensing of DNT by engineered olfactory yeast strain Nature Chemical Biology 3 (2007) 
[3] Jasmina Minic, Marie-annick Persuy, Elodie Godel, Josiane Aioun, Ian Connerton, Roland Salesse, Functional expression of olfactory receptors in yeast and development of a bioassay for odorant screening, FEBS Journal (2005)
[4] Brown et al, Functional coupling of mammalian receptors to the yeast mating pathway using novel yeast/mammalian G protein a-subunit chimeras, Yeast (2000)
[5] Matthias Versele, Katleen Lemaire, and Johan M. Thevelein, Sex and sugar in yeast: two distinct GPCR systems, EMBO Rep. 2001
[6] Dietmar Krautwurst, King-Wai Yau, and Randall R. Reed, Identification of Ligands for Olfactory Receptors by Functional Expression of a Receptor Library, Cell (1998)
[7] Venkat Radhika, Tassula Proikas-Cezanne, Muralidharan Jayaraman, Djamila Onesime, Ji Hee Ha &amp; Danny N Dhanasekaran, Chemical sensing of DNT by engineered olfactory yeast strain, Nature Chemical biology (2007)
[8] Alan Wise, Steven M. Foord, Neil J. Fraser, Ashley A. Barnes,e Nabil Elshourbagy, Michelle Eilert,g Diane M. Ignarg Paul R. Murdock, Klaudia Steplewski,h Andrew Green,Andrew J. Brown, Simon J. Dowell, Philip G. Szekeres, David G. Hassall, Fiona H. Marshall,a, j Shelagh Wilson, and Nicholas B. Pike
Molecular Identification of High and Low Affinity Receptors for Nicotinic Acid, The journal of biological chemistry (2003) 
[9] Georgies F. Mgode Bart J. Weetjens Thorben Nawrath, Christophe Cox, Maureen Jubitana, Robert S. Machang’, Stephan Cohen-Bacrie,e Marielle Bedotto, Michel Drancourt,e Stefan Schulz and Stefan H. E. Kaufmann, Diagnosis of Tuberculosis by Trained African Giant Pouched Rats and Confounding Impact of Pathogens and Microflora of the Respiratory Tract, Journal of clinical microbiology (2011) 
[10] Mona Syhre, Stephen T. Chambers, The scent of Mycobacterium tuberculosis, Elsevier (2008)
[11 Valery Matarazzo, Olivier Clot-Faybesse, Brice Marcet, Gaelle Guiraudie-Capraz, Boriana Atanasova, Gerard Devauchelle, Martine Cerutti, Patrick Etievant and Catherine Ronin,Functional Characterization of Two Human Olfactory Receptors Expressed in the Baculovirus Sf9 Insect Cell System, Chem. Senses (2005).</H6></DIV><A><DIV class="printfooter">
Retrieved from &quot;</DIV></A><A href="http://2012.igem.org/Team:TU-Delft/part1">http://2012.igem.org/Team:TU-Delft/part1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Delft/part1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Delft/part1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Delft/part1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Delft/part1&amp;oldid=299313" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>