<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Delft_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Delft/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Delft/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Delft/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Delft/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><A href="/wiki/index.php?title=Team:TUDelft/CSSLaksh&amp;action=edit&amp;redlink=1" class="new" title="Team:TUDelft/CSSLaksh (page does not exist)">Team:TUDelft/CSSLaksh</A><TITLE>Menu</TITLE></P><CENTER><UL id="navLaksh"><LI class="on"><A href="https://2012.igem.org/Team:TU-Delft">Home</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Team2">Team</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/overview">Wetlab</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/overview" style="left">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part1" style="left">Receptor</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part2" style="left">Reporter</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part3" style="left">Snifferomyces</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/protocols" style="left">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Snifferometer" style="left">Sniffer-o-meter</A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=TU-Delft" style="left" target="_blank">Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/receptordesign" style="left">DIY Receptor Design</A></LI></UL><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling" style="left">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StructuralModeling" style="left">Structural Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/SingleCellModel" style="left">Single Cell Pathway Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Analysis" style="left">Pathway Model Analysis</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/Diffusion" style="left">Diffusion Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StochasticSensitivitySpecificityAnalysis" style="left">Sensitivity-Specificity Analysis</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/informationtheory" style="left">Information Processing Model </A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Notebook" style="left">Week Diary</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/brainstorming" style="left">Brainstorming</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/HP">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/HumanOutreach" style="left">Human Outreach</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/HP">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Publicity" style="left">Publicity</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Games" style="left">Games</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Ethics" style="left">Ethics</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Safety" style="left">Safety</A></LI><LI class="off"><A>More</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/crowdfunding" style="left">Crowdfunding</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Acknowledgements" style="left">Acknowledgments</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Collaboration" style="left">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/achievements" style="left">Achievements</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Contact" style="left">Contact</A></LI></CENTER><TITLE>Human Outreach</TITLE><DIV id="contentbox" style="text-align:justify;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Connected_Multi-level_Models"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Connected Multi-level Models</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Research_Cycle"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Research Cycle</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Structural_Model"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Structural Model</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Pathway_Model"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Pathway Model</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Diffusion_Model"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Diffusion Model</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Stochastic_Model_-_Sensitivity_.26_Specificity_Analysis"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Stochastic Model - Sensitivity &amp; Specificity Analysis</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Information_processing_Model"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Information processing Model</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Overview"> Overview </SPAN></H1><P>We decided to use the modeling expertise of the team members to achieve three key objectives for our project.
</P><UL><LI> Develop scientific understanding of the yeast pheromone response pathway.
</LI><LI> Test the effects of the changes to the system.
</LI><LI> Aid decision making in the laboratory.
</LI></UL><P>To achieve the stated objectives, we adopted a hierarchical modeling approach, in which we constructed heterogeneous models spanning
</P><UL><LI> population dynamics (diffusion model).
</LI><LI> extra-cellular interactions (structural model).
</LI><LI> intra-cellular dynamics (pathway model). 
</LI></UL><H2><SPAN class="mw-headline" id="Connected_Multi-level_Models"> Connected Multi-level Models </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:582px;"><DIV class="thumbcaption"><B>Figure 1</B>: Scheme of Multi-level models.</DIV></DIV></DIV><P>These three models interact with each other to represent the mechanism of our project in different perspectives. At the system level, <B>diffusion model</B> sheds light on the behaviour of the proposed device the Snifferometer, incorporating the observations obtained from the single cell pathway model. At a lower level, the <B>pathway model</B> is built to functionally simulate the intra-cellular biochemical reactions in the signalling pathway. Parameters in pathway were fitted to the experimental data so as to provide a good estimate of the concentration of the Green Fluoroscent Protein. At a still lower level, the <B>structural model</B> was designed to understand the ligand docking at the molecular level. It indicates how well the ligand-receptor interaction is due to hydrogen bonds, which gives an insight into the receptor activation rate in the pathway model. Further research on the ligand-receptor model may provide more knowledge about ligand affinity and G-protein release. 
</P><P>Based on these models, a practical device, Snifferometer is designed to achieve the purpose of odor detection. The model of this device is developed to give an idea of the application of our project.
</P><P>The models that were built spanned a broad spectrum of <B>techniques: PDE, MD (Molecular Dynamics), SDE and ODE, as well as alignment techniques</B>, with <B>diverse tools</B> being employed for the application of these techniques: <B>COMSOL, YASARA, MATLAB, COPASI, BLAST, ClustalO, Ensembl, PSIPRED.</B></P><H2><SPAN class="mw-headline" id="Research_Cycle"> Research Cycle </SPAN></H2><P>This research cycle is introduced in our project as an interactive boundary between the wetlab and the &quot;dry&quot; modeling to gain a hierarchical in-depth understanding of our biological system. [1]
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 2</B>: Model-Experiment Cycle.</DIV></DIV></DIV><P><I>Four basic phases</I> are sequenced to close the cycle, as shows in Figure 2.
</P><P><B>First</B>, model hypotheses for the signal transduction and the ligand-binding structure are proposed based on the biological expertise, the literature surveys on the related databases. 
</P><P><B>Second</B>, due to the inadequate descriptions of dynamics and lack of information, more analysis of the hypothesized models were done to investigate the properties of models such as the parameter sensitivities and the structural stability in pathway model. 
</P><P><B>Third</B>, The experimental design was aided by the initial analysis of the models. A 2D spatio-temporal model of the proposed device was also implemented to aid the conceptualization of the Snifferometer.
</P><P><B>Finally</B>, the measured data from &quot;wet&quot; experiment, in turn, modified the final set of parameters to provide a more accurate description of the pathway.
</P><P>As a result of the research cycle, a model validated by data was obtained which was used to predict the outcomes of the experiments, using which different conditions could be simulated by &quot;cheap&quot; computations <I>in silco</I> instead of the time-consuming experiments.
</P><DIV><H1><SPAN class="mw-headline" id="Structural_Model"> Structural Model </SPAN></H1>
In order to engineer a yeast strain that is able to detect a tuberculosis molecule (TB), its receptor should be designed <DIV class="thumb tleft"><DIV class="thumbinner" style="width:152px;"><DIV class="thumbcaption"><B>Figure 3</B>: Niacin receptor embedded in membrane.</DIV></DIV></DIV>in such a way that the molecule can act as an agonist. By modeling the olfactory receptor <I>in silico</I> by means of Molecular Dynamics simulations (MD), its biophysical and biochemical properties are investigated at the molecular level. The aim is to get a clear understanding of how a ligand binds in the receptor and how to mutate the binding niche to let it bind to methyl nicotinate more specifically. A strong correlation between <I>in vivo</I> activity of the Homo Sapiens niacin receptor 1 and the simulated activity from the model resulted, enabling the prediction of ligand characteristics in the binding niche.  
</DIV><DIV><P><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StructuralModeling" align="right">More DetailsÂ Â </A></P></DIV><DIV><H1><SPAN class="mw-headline" id="Pathway_Model"> Pathway Model </SPAN></H1>
The bio-chemical pathway model was developed based on a scheme favouring the temporal order of the processes, which involved the four fundamental modules,<DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4</B>: Schematic representation of the pathway model.</DIV></DIV></DIV><UL><LI> Receptor Activation
</LI><LI> G - Protein Cycle
</LI><LI> MAPK Cascade
</LI><LI> Gene Expression
</LI></UL><P>Three different models were used for the analysis of different aspects of the pathway. The dynamics of the system in these models were described using a set of differential equations governing the concentration changes of individual components and of complexes over time.
</P><P>Sensitivity and stability analysis were permformed to determine the sensitivity coefficients which were used to study the parametric dependence of the biological models. The results from the sensitivity analysis were then used in the parameter estimation to fit the model to the data provided by the experimentalists.
</P></DIV><DIV><P><A href="https://2012.igem.org/Team:TU-Delft/Modeling/SingleCellModel" align="right">More DetailsÂ Â </A></P></DIV><DIV><H1><SPAN class="mw-headline" id="Diffusion_Model"> Diffusion Model </SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 5</B>: The convection based model of the Snifferometer</DIV></DIV></DIV><P>One of the main objectives of the project was to synthesize a practical device for the detector This led to the design of a three layered device, The Snifferometer.
</P><P>As a first step towards achieving this goal, we built a temporal model of the system using PDE's which were simulated in matlab. A 2D reaction-diffusion system was then implemented in COMSOL multiphysics using the knowledge obtained from single cell pathway model, combining the behaviours of the which helped us get a better understanding of how such a device could be implemented and the response times involved in such a process.
</P></DIV><DIV><P><A href="https://2012.igem.org/Team:TU-Delft/Modeling/Diffusion" align="right">More DetailsÂ Â </A></P></DIV><DIV><H1><SPAN class="mw-headline" id="Stochastic_Model_-_Sensitivity_.26_Specificity_Analysis"> Stochastic Model - Sensitivity &amp; Specificity Analysis </SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 6</B>: The convection based model of the Snifferometer</DIV></DIV></DIV><P>A false positive and false negative analysis is a crucial aspect of a  binary classification test such as the snifferomenter, as it indicates the confidence level in the data that one should typically strive for. On account of the inherent stochasticity in biological functions, we built a stochastic model to analyze the Sensitivity and the Specificity of the Snifferometer.
</P></DIV><DIV><P><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StochasticSensitivitySpecificityAnalysis" align="right">More DetailsÂ Â </A></P></DIV><DIV><H1><SPAN class="mw-headline" id="Information_processing_Model"> Information processing Model </SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 7</B>: The BioBit, how much information can you process?</DIV></DIV></DIV><P>Signaling pathways and genetic circuitry have the capacity to transmit and process information about certain states in the environment. 
They are used by the cell to make decisions about whether to take certain actions to remain well adapted. Until now we have used models to describe these dynamics with the goal of eventually having enough insight into the systems so we can actually engineer them.
</P><P>If we however want to quantify the information processing capacity we will need something extra. Something like the BioBit!
</P></DIV><DIV><P><A href="https://2012.igem.org/Team:TU-Delft/informationtheory" align="right">More DetailsÂ Â </A></P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:TU-Delft/Modeling">http://2012.igem.org/Team:TU-Delft/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Delft/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Delft/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Delft/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Delft/Modeling&amp;oldid=298014" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>