<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Delft_Analysis"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Delft/Analysis">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Delft/Analysis&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/Analysis&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/Analysis&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Delft/Analysis" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Delft/Analysis</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><A href="/wiki/index.php?title=Team:TUDelft/CSSLaksh&amp;action=edit&amp;redlink=1" class="new" title="Team:TUDelft/CSSLaksh (page does not exist)">Team:TUDelft/CSSLaksh</A><TITLE>Menu</TITLE></P><CENTER><UL id="navLaksh"><LI class="on"><A href="https://2012.igem.org/Team:TU-Delft">Home</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Team2">Team</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/overview">Wetlab</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/overview" style="left">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part1" style="left">Receptor</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part2" style="left">Reporter</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part3" style="left">Snifferomyces</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/protocols" style="left">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Snifferometer" style="left">Sniffer-o-meter</A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=TU-Delft" style="left" target="_blank">Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/receptordesign" style="left">DIY Receptor Design</A></LI></UL><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling" style="left">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StructuralModeling" style="left">Structural Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/SingleCellModel" style="left">Single Cell Pathway Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Analysis" style="left">Pathway Model Analysis</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/Diffusion" style="left">Diffusion Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StochasticSensitivitySpecificityAnalysis" style="left">Sensitivity-Specificity Analysis</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/informationtheory" style="left">Information Processing Model </A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Notebook" style="left">Week Diary</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/brainstorming" style="left">Brainstorming</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/HP">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/HumanOutreach" style="left">Human Outreach</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/HP">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Publicity" style="left">Publicity</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Games" style="left">Games</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Ethics" style="left">Ethics</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Safety" style="left">Safety</A></LI><LI class="off"><A>More</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/crowdfunding" style="left">Crowdfunding</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Acknowledgements" style="left">Acknowledgments</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Collaboration" style="left">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/achievements" style="left">Achievements</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Contact" style="left">Contact</A></LI></CENTER><TITLE>Human Outreach</TITLE><DIV id="contentbox" style="text-align:justify;"><P>Stability and time-dependant sensitivity analyses were implemented for the deterministic pathway model.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Stability_analysis"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Stability analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Conclusion"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#In_the_case_of_dissipating_Ste2"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">In the case of dissipating Ste2</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#In_the_case_of_accumulating_or_constant_Ste2"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">In the case of accumulating or constant Ste2</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Sensitivity_analysis"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Sensitivity analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Motivation_of_sensitivity_analysis"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Motivation of sensitivity analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Time-dependant_sensitivity_analysis"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Time-dependant sensitivity analysis</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Sensitivity_of_rates"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Sensitivity of rates</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Sensitivity_of_initial_concentrations"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Sensitivity of initial concentrations</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Conclusion_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#References"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Stability_analysis"> Stability analysis </SPAN></H1><P>A steady-state analysis can be done using even only the network structure, without knowing the rate constants for particular reactions. In the pathway model, we can see the receptor activation is the prerequisite to activate the whole pathway. The stability of the whole pathway is decided by the state of receptor. The full structure of receptor is showed in Figure 1. 
</P><P>However, before parameter estimation by using experimental data, there are two possibilities for the evolution of receptor:  dissipation or constant, because both values of rates to synthesize and to degrade the receptor are unknown. In correspondence, two results, either stable or unstable, might be possible because of lack of information. The stability analysis for both cases is implemented in the later parts.
</P><P>Then, from the collected experimental data (see in Pathway Model, parameter estimation), we found the output GFP decreases after reaching the peak sometime, which matches the case of dissipating Ste2. 
</P><H2><SPAN class="mw-headline" id="Conclusion"> Conclusion </SPAN></H2><P>The system is stable by analysing the single cell pathway model.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 1</B>: Full structure of receptor.</DIV></DIV></DIV></DIV><DIV id="contentbox" style="text-align:justify;"><H2><SPAN class="mw-headline" id="In_the_case_of_dissipating_Ste2"> In the case of dissipating Ste2 </SPAN></H2><P>If the rate of receptor degradation is larger than the rate of receptor generation, the structure of receptor can be functionally simplified as Figure 2. The receptor will dissipate towards zero in the end. As a consequence, the states in the following pathway will recover to the initial states based on mass balance. Thus, GFP will definitely degrade to zero asymptotically.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:282px;"><DIV class="thumbcaption"><B>Figure 2</B>: Structure of dissipating receptor.</DIV></DIV></DIV><P>In this sense, there is only one steady state and no bifurcation, and the system seems to be stable. The steady states are listed in Table 1.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:282px;"><DIV class="thumbcaption"><B>Table 1</B>: Initial concentrations and steady states.</DIV></DIV></DIV><P>In order to test the stability around the steady states, the linearization method is used for this nonlinear system. 
</P><P>By computation, the eigenvalues of full Jacobian A contain both zeros and the values with negative real-part, leading to undertermined conclusion for the stability of original nonlinear system. However, because of mass balance, there are some conserved values which can be reduced. They are linear combinations of particle numbers that do not vary during the time evolution of the system. Each conservation relation can be used to eliminate one variable of the system, leading to a reduced system with a smaller number of variables. These variables are called the independent variables of the system; the dependent variables are defined as linear combinations of independent variables.Technically finding the conservation relation means finding rows in the stoichiometry matrix that can be expressed as linear combinations of other rows [1].
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:242px;"><DIV class="thumbcaption"><B>Table 2</B>: Eigenvalues of Jacobian.</DIV></DIV></DIV><P>By using COPASI which uses Householder QR factorization [2] to reduce model, the Jacobian of reduced model can be calculated and its eigenvalues are showed in Table 2. 6 out of total 17 species are conserved values including Sst2a (Sst2), Fus3a (Fus3), Ste12 (Ste12a), and GaGTP, C, D in the big cycle of combining G-protein and MAPK cascade.
</P></DIV><DIV id="contentbox" style="text-align:justify;"><H2><SPAN class="mw-headline" id="In_the_case_of_accumulating_or_constant_Ste2"> In the case of accumulating or constant Ste2 </SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:242px;"><DIV class="thumbcaption"><B>Figure 3</B>: Structure of constant receptor.</DIV></DIV></DIV><P>If the rate of receptor degradation is smaller than the rate of receptor generation, the receptor will accumulate till reaching the possible maximum value. The pathway will keep activating once activated, and GFP will accumulate without limit, so the system is unstable.
</P></DIV><DIV id="contentbox" style="text-align:justify;"><H1><SPAN class="mw-headline" id="Sensitivity_analysis"> Sensitivity analysis </SPAN></H1><H2><SPAN class="mw-headline" id="Motivation_of_sensitivity_analysis"> Motivation of sensitivity analysis </SPAN></H2><P>Sensitivity analysis is to study how system variables depend on model parameters normally including two aspects: kinetic rates and initial concentrations. There are a couple of reasons to do sensitivity analysis.
</P><UL><LI><B>Validation of model with respect to uncertainty</B>. When there is limited or no data for a conceptual model, one venue for model validation is to test the sensitivity of a given parameter fit.  Thus, a model that is touted for its qualitative behaviour without quantitative behaviour can show that these properties are conserved when there is perturbation to the presented model.</LI><LI><B>Better choice for the parameter estimation</B>. In the presence of a small data set and many parameters, responses will not change equally to similar changes in parameters: some are sensitive and some are not. The parameter space we wish to fit can be reduced by examining which parameters are sensitive and which are not.</LI><LI><B>Parameter tuning for the optimal experiment design</B>. The original parameters in the experiment may be not efficient. In order to achieve better performance, some parameter can be tuned, such as the initial concentration of Ste2, the rate of Gbc synthesis, Gabc synthesis and mRNA synthesis in our project. Sensitivity analysis can provide how the output evolve when certain parameters are changed.</LI></UL></DIV><DIV id="contentbox" style="text-align:justify;"><H2><SPAN class="mw-headline" id="Time-dependant_sensitivity_analysis"> Time-dependant sensitivity analysis </SPAN></H2><P>The output GFP is the variable of interest for our experiment, thus the sensitivity analysis is on the GFP rather than all species in the pathway model. 
</P><P>Values of the parameters are cited from related paper, which is stated in detail in the page <B> Single Cell Model</B>.
</P><P>A standard sensitivity analysis investigate the changes in the steady-state values of variables of interest with respect to the changes in the model parameters. However, time-dependant sensitivity analysis is chosen instead of steady-state analysis. This is becasue there is no multi steady states for GFP and the steady state might be not able to reach during the course of observation. Moreover, sometimes it is more important to know how the values of sensitivity evolve with the time. [3]
</P><P>V represents the variable we are interested, while p is the perturbed parameters. When initial concentrations are considered as perturbed parameters p, some of them are zero, as shows in Table 1. In order to avoid the meaningless sensitivities, the largest magnitude of each concentration is taken instead.
</P><H3><SPAN class="mw-headline" id="Sensitivity_of_rates"> Sensitivity of rates </SPAN></H3><P>By varying the input from 0.002 to 20 micromolar and the magnitude of perturbation from 20% to 300%, all of these sensitivity analysis conclude the same ranking of sensitivity, in which only 3 rates are really important to the output and the sensitivities of other rates are lower than 1%, as shows in Table 3.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><A href="/File:R2.gif" title="File:R2.gif">File:R2.gif</A><DIV class="thumbcaption"><B>Figure 4</B>: Sensitivity analysis of rates in log scale when input is 2 micromolar.(Magnitude of perturbation varies from 20%, 50%, 80%, 100% to 300%)</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><A href="/File:R20.gif" title="File:R20.gif">File:R20.gif</A><DIV class="thumbcaption"><B>Figure 5</B>: Sensitivity analysis of rates in log scale when input is 20 micromolar.(Magnitude of perturbation varies from 20%, 50%, 80%, 100% to 300%)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:149px;"><DIV class="thumbcaption"><B>Figure 6</B>: Labels for rates</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:150px;"><DIV class="thumbcaption"><B>Table 3</B>: Sensitivity ranking of sensitive rates.</DIV></DIV></DIV></DIV><DIV id="contentbox" style="text-align:justify;"><H3><SPAN class="mw-headline" id="Sensitivity_of_initial_concentrations"> Sensitivity of initial concentrations </SPAN></H3><P>By varying the input from 0.002 to 20 micromolar and the magnitude of perturbation from 20% to 300%, time-dependant sensitivities of initial concentrations are plotted in Figure 7, 8, and 9. Four of initial concentrations have very high sensitivities on output, while some of them can be even ignored because of super low sensitivities. Table 4 lists the sensitivity ranking in decrement order.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><A href="/File:I02.gif" title="File:I02.gif">File:I02.gif</A><DIV class="thumbcaption"><B>Figure 7</B>: Sensitivity analysis of initial concentrations in log scale when input is 0.002 micromolar.(Magnitude of perturbation varies from 20%, 50%, 80%, 100% to 300%)</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><A href="/File:I2.gif" title="File:I2.gif">File:I2.gif</A><DIV class="thumbcaption"><B>Figure 8</B>: Sensitivity analysis of initial concentrations in log scale when input is 2 micromolar.(Magnitude of perturbation varies from 20%, 50%, 80%, 100% to 300%)</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><A href="/File:I20.gif" title="File:I20.gif">File:I20.gif</A><DIV class="thumbcaption"><B>Figure 9</B>: Sensitivity analysis of initial concentrations in log scale when input is 20 micromolar.(Magnitude of perturbation varies from 20%, 50%, 80%, 100% to 300%)</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:426px;"><DIV class="thumbcaption"><B>Table 4</B>: Sensitivity ranking of initial concentrations.(red indicates the most sensitive ones, green the less sensitive, and the black ones are not so important)</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption"><B>Figure 10</B>: Labels for initial concentrations</DIV></DIV></DIV></DIV><DIV id="contentbox" style="text-align:justify;"><H2><SPAN class="mw-headline" id="Conclusion_2"> Conclusion </SPAN></H2><P>sensitivity rankings of rates and initial concentrations are obtained and listed in Table 3 and Table 4 respectively.
</P><UL><LI>Matlab script for Sensitivity Analysis can be found <A href="https://static.igem.org/mediawiki/igem.org/1/1c/Snifferomyces.txt" class="external text" rel="nofollow"><B>here</B></A>. 
</LI></UL></DIV><DIV id="contentbox" style="text-align:justify;"><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><H6>
1. COPASI Documentation, Version 4.6 (Build 32) 
2. R. R. Vallabhajosyula, V. Chickarmane, and H. M. Sauro. Conservation analysis of large biochemical networks. Bioinformatics. 22(3). 346 - 353. 2006 
3. Adriana Supady, Theoretical and computational analysis of negative feedback mechanisms and the dose-response alignment in the pheromone signalling pathway of the yeast Saccharomyces cerevisiae. Berlin, in September 2009.
</H6></DIV><A><DIV class="printfooter">
Retrieved from &quot;</DIV></A><A href="http://2012.igem.org/Team:TU-Delft/Analysis">http://2012.igem.org/Team:TU-Delft/Analysis</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Delft/Analysis" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Delft/Analysis" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Delft/Analysis&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Delft/Analysis&amp;oldid=294995" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>