"
Page
Discussion
View source
History
teams
Log in
 
Team:TU-Delft/receptordesign
From 2012.igem.org
Menu
Home
Team
Wetlab
Overview
Receptor
Reporter
Snifferomyces
Protocols
Sniffer-o-meter
Biobricks
DIY Receptor Design
Modeling
Overview
Structural Model
Single Cell Pathway Model
Pathway Model Analysis
Diffusion Model
Sensitivity-Specificity Analysis
Information Processing Model
Notebook
Week Diary
Brainstorming
Human Practice
Human Outreach
Human Practice
Publicity
Games
Ethics
Safety
More
Crowdfunding
Acknowledgments
Collaboration
Achievements
Contact
Receptor
Content
Chimeric receptor design
DO IT YOURSELF receptor design: What, Why and How
In silico protocol
Example
References
Chimeric receptor design
A major hindrance for functional expression of ORs has been that  the receptors did not localize in the membrane or that the downstream coupling of the receptor to the Gα did not work properly. It has been shown that the rat olfactory receptor 17 (R17) that responds to octanal can be functionally expressed in many different cell types, including S. cerevisiae [6].  Earlier research investigated on the question whether the RI7 sequence can be used to functionally express other ORs. Sequence analysis of ORs have shown that the N-termini of the receptor are involved in plasma membrane localization, whereas the C-termini generally define the specificity for G protein interaction [7]. Based on this observations Radhika et al. functionally expressed  a chimeric OR with the N-terminus and the C-terminus of the RI7 sequence. A schematic picture is shown in figure 2. In this iGEM project we use the same approach as Radhika et al. by substituting the receptor termini with the RI7 sequences.
Schematic overview of the chimeric design of the receptor.  Figure adapted from Radhika et al..
DO YOURSELF receptor design: What, Why and How
What?
Protocol for making protein chimeras with a rat G protein coupled receptor (RI7) and Your Favorite Receptor. The order of the DNA sequence looks like this: RI7-[Your Favorite Receptor]-RI7
Why?
One of the requirements for a working GPCR is that the receptor should be localized into the outside membrane of yeast cell. By replacing the N-terminal part of Your Favorite Receptor by the N-terminal ends of a receptor that is known to be localized into the outside membrane of Saccharomyces cerevisiae (R17), Your Favorite Receptor (YFR) will also be localized into the membrane.  The C-terminal part of a GPCR is the alpha subunit binding region. If this is replaced by the RI7 regions a higher affinity with the alpha subunit can be reached [1].
How?
With this step-by-step protocol we guide you trough all the in silico designing steps. After this the DNA can be transformed in yeast and you have your own olfactory yeast!
In silico protocol
What: Get your receptor protein sequence code
Why: To introduce a new receptor chimera in yeast you should start with a GPCR with at least a known sequence and preferably a known ligand.
How: By using earlier research on GPCRs. For example a nice GPCR database is http://senselab.med.yale.edu/OdorDB/. Copy the DNA sequence and the protein sequence in a text file.
What: look for the transmembrane regions
Why: Normally GPCRs have seven transmembrane regions. The N-terminal loop is important for the localization in the membrane and should be replaced by the RI7 N-terminal sequence. The C-terminal region directly after the last transmembrane part codes for the alpha-subunit binding region. If this region is replaced by the RI7 region a higher affinity with the alpha subunit can be reached.
How: Go to http://elm.eu.org/, enter the protein sequence code and find protein motifs for Saccharomyces cerevisiae and the original species. Compare the Global domain table. Ideally it finds seven transmembrane regions that all have approximately the same length (quite a conserved domains are found[2] ).  When this is not the case, investigate the hydrophobicity by a hydrophobicity index (analysis can be done by Matlab Bioinformatics tool, but this less conclusive due to multiple hydrophobicity indexes).
What: Remove protein sequences that code for the N-terminal and C-terminal regions
Why: The sequence that code for the N-terminal loop should be replaced by the RI7 sequence for better membrane localization. The C-terminal region should be replaced for a higher affinity with the alpha subunit.
How: Delete the sequence at the N-terminal end directly after the first transmembrane part (when read from N=left to C=right).
Delete the C terminus directly after the last transmembrane part, this is the subunit binding region.
What: Check what you did so far
Why: You want to know if you removed the right regions. Do I have receptor with only six transmembrane regions?
How: Check with http://elm.eu.org/, enter the protein sequence code and find protein motifs.
What: Go from protein sequence to DNA sequence
Why: For further adaptations it is easier to work with the DNA sequence
How: Enter the original full length DNA sequence in http://web.expasy.org/translate/ with the output format “Include nucleotide sequence’. Now you can easily find which nucleotides should be removed.
What: Add the RI7 N-terminal DNA code upstream of your DNA sequence and the RI7 C-terminal downstream
Why: The sequence that code for the N-terminal loop should be replaced by the RI7 sequence for better membrane localization. The C-terminal region should be replaced for a higher affinity with the alpha subunit.
How: Go to Biobrick BBa_K775000 in the Registry of Standard Biological Parts, copy the RI7 N-terminal parts and paste this upstream of YFR sequence. Copy also the RI7 N-terminal parts and paste this downstream of YFR sequence.
What: Codon optimize the sequence for Saccharomyce cerevisiae
Why: For better expression of the protein in yeast
How: Go to http://www.jcat.de/  and enter the sequence. Also specify the restriction sites that you don’t want to have in the sequence: at least the standard illegal restriction sites: EcoRI, XbaI, PsteI, SpeI.
What: add BamHI and NdeI restriction sites and other features
Why: If you have this two restriction sites you can easily clone your receptor in BBa_K775000 to have a yeast promoter and terminator.
How:  If you send it for synthesizing just add the nucleotides in your file. If you work with cDNA you can add the restriction sites with PCR.
Tip: if you send your sequence for synthesizing you can also add a Kozak sequence for better translation of the protein and A FLAGtag to analyze the localization of the protein into the membrane.
What: Final check
Why: check, check, double check!
How: Align your final protein sequence to the original sequence of your receptor and the chimeric receptors (can be done by blasting too). Also look in http://tools.neb.com/NEBcutter2/ for forbidden restriction sites.
Now you can send you sequence to a synthesizing company or work with isolated DNA.
Example
For our example we take ORL2156, found on the olfactory database of the university of Yale.
2. The Elm server had the following output on a the protein sequence of ORL2156:
Globular domains/ TM domains and signal peptide detected by the SMART server
3. When looking at the cDNA code translated by expasy we indicate the yellow as the cut-off region --> Cut out the yellow parts:
5. Now the yellow parts are removed both in the cDNA and protein code, the check is not shown.
6. Here the sequence of I7 is added. For convenience we made it a picture.
7. Codon optimization is performed.
8. A very important aspect: adding features! This is of course totally dependant on the methods you want to integrate the designed receptor... So we will leave it to your own creativity.
9. The checking:
Protein alignment with original receptor:
Protein alignment with Rat receptor:
Region detection by the elm server:
Looks good! All alpha helices have 22 amino acids through the membrane. Now we can synthesize the construct or design primer to perform PCR on the species used. Also we can run the YASARA program to optimize the binding niche for the specific ligand and then synthesize!
References
[1] Venkat Radhika, Tassula Proikas-Cezanne, Muralidharan Jayaraman, Djamila Onesime, Ji Hee Ha & Danny N Dhanasekaran, Chemical sensing of DNT by engineered olfactory yeast strain, Nature Chemical biology (2007)
[2]Janet M. Young et al. Different evolutionary processes shaped the mouse and human olfactory receptor gene families Hum. Mol. Gen. 2002, Vol. 11, No. 5
Retrieved from "
http://2012.igem.org/Team:TU-Delft/receptordesign "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
