"
Page
Discussion
View source
History
teams
Log in
 
Team:TU-Delft/protocols
From 2012.igem.org
For a new page in our wiki
Menu
Home
Team
Wetlab
Overview
Receptor
Reporter
Snifferomyces
Protocols
Sniffer-o-meter
Biobricks
DIY Receptor Design
Modeling
Overview
Structural Model
Single Cell Pathway Model
Pathway Model Analysis
Diffusion Model
Sensitivity-Specificity Analysis
Information Processing Model
Notebook
Week Diary
Brainstorming
Human Practice
Human Outreach
Human Practice
Publicity
Games
Ethics
Safety
More
Crowdfunding
Acknowledgments
Collaboration
Achievements
Contact
Your title
Protocols
Media
Yeast Transformation
Transforming One Shot® Mach1™ competent cells
Mini-prep plasmid isolation for E. coli and S. cerevisae
Freezing cell stocks
Restriction enzyme digestion
Ligation
Running DNA Gel
PCR
Gel Extraction
Immunofluorescence Staining
DNA Staining
Flow cytometry to look at DNA content
Flow cytometry to look at EGFP expression
Fluorimeter experiment
Growth curve experiment
Smell activity tests with petri dishes
Single Cell microscopy
Tips&Tricks for working with yeast
Media
DO (agar)
For BY4741 yeast strain add to mineral media:
HIS: 125 mg/liter
LEU 500 mg/liter
MET 100 mg/liter
URA 150 mg/liter
except for one which you want to select on with you auxotrophic marker.
Also add:
2% Sucrose (w/v)
2% Agarose
and autoclave
YPD (agar)
http://openwetware.org/wiki/YPD
LB (agar)
http://openwetware.org/wiki/LB
Yeast Transformation
Materials
Yeast culture
single stranded carrier DNA (200 mg Salmon sperm DNA inTE)
Lithium Acetate (1.0 M)
Polyethylene Glycol 3350 (50% w/v)
DO media plates
Dropout (DO) media, lacking an amino acid of which an auxotrophic marker is added
Water bath 42 °C
Water bath 30 °C
Protocol
Grow yeast culture overnight in 10 mL YPD on 30 °C
Put 0.5 mL – 1 mL into new flask with 20 mL YPD (for 2-4 transformations) and grow 4-5 hours until an OD of 0.8 is measured. The cells are now in exponential phase.
Spin the cells for 5 minutes on 3000 G and discard supernatant
Wash cells with water and spin down for 5 minutes on 3000 G
During centrifuging, make a transformation mix (TM): Per reaction add
240 µL 50% PEG
36 µL 1M Lithium Acetate solution
25 µL ssDNA, boiled for 5 minutes and then cooled on ice
49 µL water
Discard supernatant and resuspend in 0.2 mL 0.1 M Lithium Acetate solution
Do a quick centrifuge (10 s 13.000 rpm on a table top centrifuge), discard supernatant and add 80 µL 0.1 M Lithium Acetate solution. Vortex and do not pipette up and down
You should have ~100 µL of which 50 µL should be put in a fresh tube. This is you negative control
spin cells down again and add the mastermix
Add 1 µL target (100 ng – 1000 ng) DNA to the positive tube
Put in a 30 °C water bath for 30 minutes
Put it in a 42 °C water bath for 30 minutes
Do a quick centrifuge (10 s 13.000 rpm on a table top), remove supernatant. Add 0.2 mL water, pipet up and down and plate 150 µL and 50 µL on DO selective plates
Transforming One Shot® Mach1™ competent cells
Materials:
- Competent cells
- SOC medium (warmed to room temperature)
- Plasmid DNA or DNA ligation mix
- LB agar plates containing 15-100 ?g/mL antibiotic of choice, pre-warmed to 37 °C
- water bath at 42 °C
- shaking incubator at 37 °C.
Protocol:
Add 50-100 ng DNA into a 20 ?L competent E.coli, and mix gently. Do not mix by pipetting up and down!
Incubate tube vial on ice for 30 minutes
Heat-shocks the cells for 30 seconds at 42 °C without shaking
Immediately transfer the tubes back to ice for 2 minutes
Add 250 ?L of room temperature LB medium
Cap tube tightly and shake tube horizontally (225 rpm) at 37 °C for 1 hour
Plate from each tube 100 ?L on an agar plate containing antibiotic. Spin tube, discard supernatant to leave no more than 100 ?L, vortex and plate on an agar plate
Incubate plates overnight at 37 °C
Qiagen Mini-prep plasmid isolation for E. coli and S. cerevisae
This protocol is based on QIAGEN® Plasmid Purification Handbook.
Materials:
- bacterial or yeast culture
- Qiagen colums
- buffer P1 (100 mg/mL RNAse A, 50 mM Tris/HCl, 10 mM EDTA, pH 8.0)
- buffer P2 (200 mM NaOH, 1% SDS)
- buffer P3 (3 M KAc, pH 5.5)
- buffer PE
- milliQ pH 8.0
- centrifuge
- nanodrop
- for yeast plasmid isolation: zymolyase 5 U/µl
Protocol:
Pick a single colony from a freshly streaked selective plate and inoculate a starter culture of 2–5 mL LB medium containing the appropriate selective antibiotic or selective medium. Incubate for approximately 8 h at 37°C (bacteria) or 12 h at 30°C with vigorous shaking (approx. 300 rpm)
Harvest the 5 mL bacterial cells by centrifugation at 13,000 rpm for 1 min at 20°C (microcentrifuge tube). If you wish to stop the protocol and continue later, freeze the cell pellets at –20°C
For bacteria: Resuspend pelleted bacterial cells in 250 µL Buffer P1. Ensure that RNase A has been added to Buffer P1. No cell clumps should be visible after resuspension of the pellet.
For yeast: Resuspend cells in 250 µL Buffer P1 with 3 µL zymolyase and incubate 1 h at 37 °C
Add 250 µL Buffer P2 and mix thoroughly by inverting the tube 4–6 times. Mix gently by inverting the tube. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear. Do not allow the lysis reaction to proceed for more than 5 minutes.
Add 350 µL Buffer N3 and mix immediately and thoroughly by inverting the tube 4–6 times. To avoid localized precipitation, mix the solution thoroughly, immediately after addition of Buffer N3. Large culture volumes (e.g. =5 mL) may require inverting up to 10 times. The solution should become cloudy.
Incubate at -20 °C for 15 minutes.
Centrifuge for 10 min at 13,000 rpm in a table-top microcentrifuge. A compact white pellet will form.
Apply the supernatants from step 7 to the QIAprep spin column by decanting or pipetting.
Centrifuge for 30–60 seconds. Discard the flow-through.
Wash QIAprep spin column by adding 0.75 mL Buffer PE and centrifuging for 30–60 seconds.
Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer. Important: Residual wash buffer will not be completely removed unless the flow-through is discarded before this additional centrifugation. Residual ethanol from Buffer PE may inhibit subsequent enzymatic reactions.
Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 30 µL Buffer EB (10 mM Tris•Cl, pH 8.5) or water to the center of each QIAprep spin column, let stand for 1 minute in 50 C stove and centrifuge for 1 minute to obtain DNA.
Measure DNA concentration on the Nanodrop
Freezing cell stocks
Materials:
- bacterial culture
- Growth medium
- 80% glycerol
- centrifuge
Protocol:
Take 5 mL bacterial cells from the Erlenmeyer of a freshly grown culture and spin in a 15 mL tube for 10 minutes at 2.000 rpm (Eppendorf centrifuge)
Decant the supernatant without disturbing the pellet
Pipet on the pellet 0.5 ml of appropriate medium and 0.5 mL 80% glycerol and mix by vortexing and save in -80 °C freezer
Restriction enzyme digestion
Materials:
- plasmid DNA or PCR product
- restriction enzymes (Roche and BioLabs)
- buffer (10x)
- H2O
water bath at 37 °C
- heat block or water bath at 65 °C
Protocol:
Digestions (cutting plasmid DNA) were performed at the appropriate temperature with the appropriate buffer in the appropriate concentration, according to the supplier. With double restriction, use bigger volume (~ 50 µL) and we found out that adding BSA altered performance greatly.
Reaction for one sample:
DNA                               × µL (up to 1,0 µg)
Buffer (10×)                  × µL (1×))
Restriction enzymes   × µL (5 units/µg DNA = 1 µL) )
H2O                                 × µL
tot volume                    20-25 µL
Incubate for (at least) one hour at 37 °C.  Inactivate the restriction endonucleases by heat, incubation at 65 °C for 10 minutes and centrifuge shortly.
Used Buffers:
Buffer H (Roche):  50 mM Tris-HCl, 1 M NaCl, 100 mM MgCl2, 10 mM DTE, pH 7.5 at 37 °C
Buffer M (Roche): 100 mM Tris-HCl, 500 mM NaCI, 100 mM MgCl2, 10 mM DTE, pH 7.5 at 37 °C
Buffer 1 (BioLabs):  10 mM Bis-Tris-Propane-HCl, 10 mM MgCl2, 1 mM DTE,pH 7.0 at 25°C
Buffer 2 (BioLabs):  50 mM NaCl, 10 mM Tris-HCl, 10 mM MgCl2, 1 mM DTE, pH 7.9 at 25°C
Buffer 3 (BioLabs):  100 mM NaCl, 50 mM Tris-HCl, 10 mM MgCl2, 1 mM DTE, pH 7.9 at 25°C
Buffer 4 (BioLabs):  50 mM CH3CO2K, 20 mM TAE, 10 mM Mg(CH3COO)2, 1 mM DTE, pH 7.9 at 25°C
Note: some of the restriction enzymes of New England BioLabs required the addition of 100 µg/mL BSA)
Ligation
Materials:
- digested plasmid DNA or PCR product
- T4 ligation buffer (10x) (Fermentas)
- T4 ligase (Fermentas)
- H2O
water bath at 16 °C
Protocol: Ligations (pasting plasmid DNA) were performed at the appropriate temperature with the appropriate buffer in the appropriate concentration, according to the supplier. We encountered different tactics for ligation. Usually it comes down to keeping it ~16 °C for at least 3 hours.
Reaction for one sample:
DNA insert          × µL
DNA vector          × µL
T4 Ligation buffer (10×)         × µL (for 1×)
T4 Ligase       1.0 µL
H2O            × µL
tot. volume   10-15 µL
The final concentration is preferably ~100 ng/µL. Smaller volumes are preferred and when DNA is at low concentration, try to evaporate water using a vacuum. Incubate at 16 °C for at least 3 hours or keep in an ice box in a floatie overnight. In the morning you find eppendorf tubes floating in water which had a temperature gradient overnight. For transformation use circa half of the ligation mix.
Running a DNA gel
Materials:
- Agarose
- TAE 1x
- SybrSafe DNA stain
- Loading Dye
- DNA ladder (Smartladder)
- DNA electrophoresis machine
Protocol:
Dissolve agarose (w/v 0.6% for separating long DNA pieces (>10 kbp), 1% for separating shorter pieces) in 1x TAE by microwaving
Close sides of electrophoresis tray (scotch tape works fine) and add comb
Let solution cool and add 5 µl Sybrsafe to an empty electrophoresis tray (small gels) 10-12 µl Sybrsafe for larger gels
Pour gel until a height of ~0.5 cm. Mix and remove bubbles  with pipet tip (fast! It hardens quickly)
Put tray into electrophoresis casing and add TAE until a small layer above the gel can be seen. Remove comb
Add 1 µl loading dye to 5 µl sample, mix and load in the gel. Also add 5 µl smartladder for your reference
Run gel on 80 V (long run)- 110 V (short run, mostly for a ‘fast check’) for ~40-60 minutes, dependant of gel size, separation acquisition  and voltage.
For information about the smartladder, Smartladder specifications .
PCR
Materials:
For PCR with minimal errors, pFX polymerase is used:
Pfx polymerase (Invitrogen)
10x Pfx Buffer (Invitrogen)
enhancer (Invitrogen)
50 mM MgSO4 (Invitrogen)
10 mM dNTPs
For PCR for checks of length, Taq polymerase is used, provided in Qiagen mastermix:
Mastermix
Always needed:
primer solutions 5 mol/mL
template DNA (plasmid at 50 pg – 1 ng/µL), or plate with colonies
PCR machine
Protocol:
First make sure that there is a PCR machine available for you. Take the solutions from the freezer and thaw them on ice.
Preparation of reaction mixture:
Gently vortex and briefly centrifuge all solutions after thawing
Keep solutions on ice
Add to a thin walled PCR tube, on ice the desired reaction mixture listed below.
for PCR on colonies: prick a sterile toothpick into a colony, dip it into a PCR tube and put it in 15 mL culture tube containing growth media to grow overnight for direct culturing positives.
Make sure you keep everything cool until it enters the preheated PCR machine
For Pfx PCR the reaction mixture is:
Component
Sample
10 mM
dNTPs
1.5 μL
Enhancer (Invitrogen)
5.0 μL
10x Pfx Buffer (Invitrogen)
5.0 μL
50 mM
MgSO4 (Invitrogen)
1.0 μL
Primer 1
3.0 μL
Primer 2
3.0 μL
Pfx polymerase (Invitrogen)
0.6 μL
DNA template
1.0 μL
H2O
29.9 μL
PCR program pFX :
Step
Annealing
Temperature
Time,
min:sec
Number of cycles
Initial denaturation
95 °C
2:00
 
1
Annealing
 X °C *
1:00
Extension
68°C
1:00-2:00
(1 min/kb)
Denaturation
95    °C
1:00
 
25
Annealing
  X °C *
1:00
Extension
68    °C
1:00-2:00
(1 min/kb)
Final Extension
68    °C
10:00
1
For Taq PCR the reaction mixture is:
Component
Sample
Taq PCR   Master Mix (Qiagen
12.5 μL
Primer 1
1.5 μL
Primer 2
1.5 μL
Template   DNA (or pick a colony)
1 μL
H2O
8.5 μL
PCR program Taq polymerase:
Step
Annealing
Temperature
Time,
min:sec
Number of cycles
Initial denaturation
94 °C
10:00
 
1
Annealing
 X °C *
0:45
Extension
72°C
1:00-2:00
(1 min/kb)
Denaturation
95    °C
1:00
 
25
Annealing
  X °C *
1:00
Extension
72 °C
1:00-2:00
(1 min/kb)
Final Extension
72 °C
10:00
1
*Annealing temperature is very dependent on primer. Optimal temperature: 3x G/C + 2x A/T
Gel Extraction
This protocol is based on QIAGEN® Gel Extraction Handbook.
Materials:
QIAquick columns
buffer QG
buffer PE
isopropanol
milliQ
microcentrifuge
heat block at 50 °C
Protocol:
Excise the DNA fragment from the agarose gel with a clean, sharp scalpel. Minimize the size of the gel slice by removing extra agarose.
Weigh the gel slice in a colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100 mg ~ 100 µL). For >2% agarose gels, add 6 volumes of Buffer QG. The maximum amount of gel slice per QIAquick column is 400 mg; for gel slices >400 mg use more than one QIAquick column.
Incubate at 50°C for 10 min (or until the gel slice has completely dissolved). To help dissolve gel, mix by vortexing the tube every 2–3 min during the incubation.
IMPORTANT: Solubilize agarose completely. For >2% gels, increase incubation time.
After the gel slice has dissolved completely, check that the color of the mixture is yellow (similar to Buffer QG without dissolved agarose). If the color of the mixture is orange or violet, add 10 µL of 3 M sodium acetate, pH 5.0, and mix. The color of the mixture will turn to yellow. The adsorption of DNA to the QIAquick membrane is efficient only at pH <7.5. Buffer QG contains a pH indicator which is yellow at pH <7.5 and orange or violet at higher pH, allowing easy determination of the optimal pH for DNA binding.
Add 1 gel volume of isopropanol to the sample and mix. This step increases the yield of DNA fragments <500 bp and >4 kb. For DNA fragments between 500 bp and 4 kb, addition of isopropanol has no effect on yield. Do not centrifuge the sample at this stage.
Place a QIAquick spin column in a provided 2 ml collection tube.
To bind DNA, apply the sample to the QIAquick column, and centrifuge for 1 min.  The maximum volume of the column reservoir is 800 µL. For sample volumes of more than 800 µL, simply load and spin again.
Discard flow-through and place QIAquick column back in the same collection tube.
To wash, add 0.75 ml of Buffer PE to QIAquick column and centrifuge for 1 min.
Discard the flow-through and centrifuge the QIAquick column for an additional 1 min at 10,000 x g (~13,000 rpm).
IMPORTANT: Residual ethanol from Buffer PE will not be completely removed unless the flow-through is discarded before this additional centrifugation.
Place QIAquick column into a clean 1.5 ml microcentrifuge tube.
To elute DNA, add 50 µL of Buffer EB (10 mM Tris•Cl, pH 8.5) or H2O to the center of the QIAquick membrane and centrifuge the column for 1 min at maximum speed. Alternatively, for increased DNA concentration, add 30 µl elution buffer to the center of the QIAquick membrane, let the column stand for 1 min, and then centrifuge for 1 min.
Important: Ensure that the elution buffer is dispensed directly onto the QIAquick membrane for complete elution of bound DNA. The average eluate volume is 48 µL from 50 µL elution buffer volume, and 28 µL from 30 µL. Elution efficiency is dependent on pH. The maximum elution efficiency is achieved between pH 7.0 and 8.5. When using water, make sure that the pH value is within this range, and store DNA at –20°C as DNA may degrade in the absence of a buffering agent. The purified DNA can also be eluted in TE (10 mM Tris•Cl, 1 mM EDTA, pH 8.0), but the EDTA may inhibit subsequent enzymatic reactions.
Immunofluorescence Staining
This protocol is for immunofluorescence staining of yeast to prepare it for Flow Cytometry or fluorescence microscopy, and staining the target of the conjugated antibody of choice. The TU Delft iGEM 2012 team used a flagtag antibody. The protocol of the supplier was improved for yeast staining. IMPORTANT: Please refer to the APPLICATIONS  section on the front page of the datasheet of the antibody product to determine if this product is validated and approved for use on cultured cell lines (IF-IC).
 
Materials:
 
·         Phosphate Buffered Saline (PBS) , adjust pH to 8.0.
·         Formaldehyde, 16%, methanol free
·         Blocking Buffer: (1X PBS / 1 wt% BSA / 0.3% (v/v) Triton X-100)
·         Antibody Dilution Buffer (1X PBS / 1 wt% BSA / 0.3% (v/v) Triton X-100)
·         Primary conjugated antibody.
 
Protocol:
 
The desired culture of cell is taken for staining and the liquid is aspirated by spinning it down in appropriate tubing ( e.g. standard Eppendorf tubes )  and decanting or pipetting the supernatant.  Afterwards the container is inspected for a cell pellet of acceptable size, meaning it must be easy to spot in the container. After this the fixing of the cells can start:
 
1.       Cover cells to a depth of 10–15 mm with 4% formaldehyde in PBS and resuspend cells.
NOTE: Formaldehyde is toxic, use only in fume hood.
2.       Allow cells to fix for 15 minutes at room temperature.
3.       Aspirate fixative and rinse three times in PBS for 5 minutes each before proceeding to the staining section. ( Use the same aspirating techniques as above: Spin, Aspirate, add, resuspend. )
 
All subsequent incubations should be carried out at room temperature unless otherwise noted in a humid light-tight box or covered dish/plate to prevent drying and fluorochrome fading.
Block specimen in Blocking Buffer for 60 minutes.
While blocking, prepare primary antibody by diluting as indicated on datasheet in Antibody Dilution Buffer. ( A Cell Signaling technology antibody was used with this protocol )
Aspirate blocking solution, apply diluted primary antibody.
Incubate overnight at 4°C.
Rinse three times in PBS for 5 minutes each. NOTE: Because one is using primary antibodies directly conjugated with ( Alexa Fluor® ) fluorochromes, secondary antibodies are not needed.
For best results, examine specimens immediately using appropriate excitation wavelength. For long-term storage, store slides flat at 4°C protected from light.
DNA Staining
Materials
Your favorite yeast strain
Vybrant DyeCycle Orange from invitrogen
PBS
37°C heat block
Protocol
Centrifuge cells on a table top centrifuge for 10 s on 13.000 rpm
Take of supernatant and add PBS
Add 2 µL/mL Vybrant DyeCycle Orange and vortex until homogenuous solution
Keep for 30 minutes on 37°C and the cells are stained.
Flow cytometry to look at DNA content
Materials
Yeast culture
Vybrant® DyeCycleTM Orange stain
Ligand
Flow cytometry instrument
Protocol
Grow yeast cells at 30C overnight and re-inocculate in 10 mL medium the next morning
If cells are in exponential growth phase add the ligand (also keep a control without ligand)
After incubation stain the cells according to protocol
Transfer the stained cells in a flow cytometry tube
Analyze the samples on a flow cytometer using 488 nm excitation or 532 nm excitation and orange emission
After each measurement flush the tube with PBS buffer
Flow cytometry to look at EGFP expression
Materials
Yeast culture
Ligand
Flow cytometry instrument
Protocol
Grow yeast cells at 30C overnight and re-inocculate in 10 mL medium the next morning
If cells are in exponential growth phase add the ligand (also keep a control without ligand)
Transfer the cells in a flow cytometry tube
Analyze the samples on a flow cytometer using 488 nm excitation and  509 nm emission
After each measurement flush the tube with PBS buffer
Fluorimeter experiment
Materials
Yeast culture
Ligand
Fluorimeter plate reader
96 well plate
Protocol
Grow yeast cells at 30C overnight and re-inocculate the next morning
Measure after approx. 3h hours the OD600. Dilute the cells with medium until an OD of approx. 0.1
Transfer the cells to the 96 well plate. Don't forget to have a well with only medium.
Add the ligand.
Analyze the samples in the fluorimeter that is kept at 30C. Use filters with 488 nm excitation and  509 nm emission. At the same time measure the OD600.
Calculate the fluoresce per biomass by dewing the fluorescence by the OD600 values
Growth rate experiment
Materials
Yeast cultures (wt with and without alpha feromones and far1KO with and without alpha feromones)
Alpha feromones
Spectrophotometer
30C incubator
Protocol
Grow yeast cells at 30C overnight and re-inocculate 20 ml of medium with 500µl of grown cells the next morning
Measure OD600 every hour.
Add alpha feromones after 3hr
Measure OD600every hour, dilute if needed.
Smell activity tests with petri dishes
Materials
4 x Two yeast cultures with each having six yeast-strains
Ligands
Water 2 x  Alpha pheromone 1000x diluted 2 x  Isoamyl acetate 99%  Methyl phenylacete 99%  Niacin 0.336 g/100 ml  Methyl nicotinate 10 mM
Typhoon fluorescence scanner
8 Resealable (hard) plastic boxes of 10x10x10 cm.
Protocol
Grow yeast cells at 30C for two days.
Analyze the samples in the Typhoon fluorescence scanner before starting experiment.
Pore approx. 25 ml of every ligand (except alpha pheromone) in the box after cleaning it with alcohol / autoclaving if possible.
Place two sided tape on the inwards part of the boxlid. Gently but firmly push the petri dish on it. Remove the lid of the dish and put the dish upside down in the box. For alpha pheromone; add solution directly on all yeast strains.
After 3.5 hours remove the dish from the lid.
Analyze the samples in the Typhoon fluorescence scanner. Use filters with 488 nm excitation and  509 nm emission. Set PMT to 300. Orientation of dish to Platen.
Single Cell microscopy
If one want to get insight into the real in situ working of genetic circuits of S. Cerevisiae one must look at it at a smaller scale:  The cell scale. For this a number of techniques are available like e.g. flow cytometry. One of the drawbacks is that a certain single cell can only be measure once and that it is not possible to do analysis on a single cell in the time dimension. Therefor fluorescence microscopy can be of much help to assess the overtime state dynamics of the single cell. For this certain preparations are needed:
Materials
Fluorescence microscope
Concanavalin A type V, Sigma-Aldrich
Yeast cells
OD medium
Sterile water
Protocol
Grow yeast cells prior to the experiment for at least 15 hours in DO medium at 30 degrees. For the experiment the cells should be in exponential phase.
Incubate a glas bottom black walled 24wells plate with 500µL of Concanavalin A solution (100 µg/ml )for at least one hour.
Aspirate liquid by pipetting it out of the corner of the well.
Apply a 500µL solution of yeast cells ( OD600=0.1-0.3 ) that has been mildly sonicated and let it incubate for 10 minutes
Apply a 500µL solution of yeast cells ( OD600=0.1-0.3 ) that has been mildly sonicated and let it incubate for 10 minutes.
Wash 3 times with OD medium to remove unbound cells from the well.
Apply OD medium to the well and add ligand of choice.
Incubate the cells in a preheated microscope chamber of 30 degrees while performing the experiment.
Tips&Tricks for working with yeast
Since this is the first time that TU Delfts iGEM team is working with yeast, we faced a lot of small yeast-related 'challenges'. With this page we want to inform you about the basics and the pitfalls of working with yeast.
Basics
Yeast, Saccharomyces cerevisiae, is a simple, unicellular eukaryotic organism. This organism has been used for fermentation and baking for over 4000 years and it is probably the oldest domesticated micro-organism in the human history. An important one too, can you imagine a life without bread and beer? (Even if you can imagine it, it would most definitely be less fun!)  Nowadays the whole genomic sequence of Saccharomyces cerevisiae is known and a lot of genomic tools are available.
Auxotrophy
The main advantage from an engineering perspective is that yeast has Auxotrophic markers. In specific strains genes are knocked out which synthesize essential enzymes in the amino acid synthesis routes. By complementing these deficiencies by adding the necessary gene on your DNA this provides a nice selection procedure.
PRS huttle vectors
The name pRS415 gives an indication on the presence of a CEN/ARS replication origin. 0 means yeast integrative plasmid, 1 means that it also can be used to maintain the plasmid in circular form. pRS415 Gives an indication of the auxotrophic marker used. pRS415 Version number… not really different.
Chromosomal integration
We encountered a lot of problems with plasmids. Because we wanted our constructs to be universal (with the idea to make it suitable for ‘fast checking’) we tried maintaining a plasmid. As it turned out, yeast cells are not eager to maintain a plasmid and with our construct we suspect homologous recombination occurred. After transformation, a PCR on the transformed plasmid, obtained by isolation, showed two bands instead of the suspected single band, one being ! Integration of the plasmid is therefore advised! Checking of this can be quite gruesome optimizing the necessary PCR reactions on your transformed yeast colonies. Chromosomal isolation can therefore improve the steps.
Knock-out strains
European iGEM teams have the advantage to have Euroscarf available to order strains with knocked out ORFs. The typical nomenclature is also explained here: Euroscarf explanation .
Retrieved from "
http://2012.igem.org/Team:TU-Delft/protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
