<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU-Delft_Modeling_SingleCellModel"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU-Delft/Modeling/SingleCellModel">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU-Delft/Modeling/SingleCellModel&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/Modeling/SingleCellModel&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU-Delft/Modeling/SingleCellModel&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU-Delft/Modeling/SingleCellModel" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU-Delft/Modeling/SingleCellModel</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><A href="/wiki/index.php?title=Team:TUDelft/CSSLaksh&amp;action=edit&amp;redlink=1" class="new" title="Team:TUDelft/CSSLaksh (page does not exist)">Team:TUDelft/CSSLaksh</A><TITLE>Menu</TITLE></P><CENTER><UL id="navLaksh"><LI class="on"><A href="https://2012.igem.org/Team:TU-Delft">Home</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Team2">Team</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/overview">Wetlab</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/overview" style="left">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part1" style="left">Receptor</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part2" style="left">Reporter</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/part3" style="left">Snifferomyces</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/protocols" style="left">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Snifferometer" style="left">Sniffer-o-meter</A></LI><LI><A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2012&amp;group=TU-Delft" style="left" target="_blank">Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/receptordesign" style="left">DIY Receptor Design</A></LI></UL><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling" style="left">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StructuralModeling" style="left">Structural Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/SingleCellModel" style="left">Single Cell Pathway Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Analysis" style="left">Pathway Model Analysis</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/Diffusion" style="left">Diffusion Model</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Modeling/StochasticSensitivitySpecificityAnalysis" style="left">Sensitivity-Specificity Analysis</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/informationtheory" style="left">Information Processing Model </A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Notebook" style="left">Week Diary</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/brainstorming" style="left">Brainstorming</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/HP">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/HumanOutreach" style="left">Human Outreach</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/HP">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Publicity" style="left">Publicity</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Games" style="left">Games</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Ethics" style="left">Ethics</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Safety" style="left">Safety</A></LI><LI class="off"><A>More</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/crowdfunding" style="left">Crowdfunding</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Acknowledgements" style="left">Acknowledgments</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/Collaboration" style="left">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:TU-Delft/achievements" style="left">Achievements</A></LI><LI class="off"><A href="https://2012.igem.org/Team:TU-Delft/Contact" style="left">Contact</A></LI></CENTER><TITLE>Single Cell Model</TITLE><DIV id="contentbox" style="text-align:justify;"><P>The single-cell pathway model of our system is composed of 4 modules, which are responsible for routing the signal from the receptor to the GFP sythesis. The modules modelled in the model are,
</P><UL><LI>Receptor activation module : This module acts as the sensor, it models the ligand and the receptor which detects the presence of ligand.
</LI><LI>G-Protein cycle : It channels the sensed information from the sensor to the  MAP kinase cascade.
</LI><LI>MAP kinase cascade : The MAP kinase cascade is responsible for the activation of <I>Ste12</I> which acts as the transcription factor for the <I>FUS1</I> gene which is responsible for the production of the green fluoroscent protein.
</LI><LI>Gene Expression module : Which on the detection of the ligand produces the green fluroscent protein, which is the output of our system.
</LI></UL><P>The single cell pathway model helped us understand in detail if and how our synthetic circuit works, and also helped us understand how the input concentration was related to the concentration of the input substance. Using the model we were able to predict the time taken for the expression of the green fluoroscent protein and how the concentration of GFP varies with different input concentration.
</P><P>The mathematical models were developed on a scheme favoring the temporal order of processes, based on the current understanding of the pheromone signalling pathway from <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref1" title="Team:TU-Delft/Modeling/SingleCellModel">[1]</A><A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref2" title="Team:TU-Delft/Modeling/SingleCellModel">[2]</A><A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref3" title="Team:TU-Delft/Modeling/SingleCellModel">[3]</A> and also on the feedback received from the experimentalists on the expected behaviour of the pathway.
</P><P>In the models developed certain assumptions were made due to the lack of data available on the new modified pathway and also due to the time constraints of iGEM  which limits the range of experiments that can be carried out. The assumptions made are listed below.
</P><UL><LI> The rate constants and initial concentrations obtained from the literature are assumed to be the same for the new modified pathway.
</LI><LI> The behaviour of the Ligand is assumed to be similar to the behaviour of the alpha pheromone.
</LI><LI> The receptor is assumed to behave similar to the alpha-pheromone receptor.
</LI></UL><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Initial_Models"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Initial Models</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Model_of_the_G_-_Protein_cycle"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Model of the G - Protein cycle</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#conclusions"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">conclusions</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#limitations"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">limitations</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-5"><A href="#Model_of_the_Yeast_Pheromone_Response"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Model of the Yeast Pheromone Response</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#conclusion"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">conclusion</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#limitations_of_the_initial_models"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">limitations of the initial models</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Reduced_Model_of_the_Snifformyces_.28Modified_Yeast.29_Pathway"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Reduced Model of the Snifformyces (Modified Yeast) Pathway</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Parameter_Estimation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Parameter Estimation</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#conclusion_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Model_Predictions"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Model Predictions</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Results"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#MATLAB_Codes"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">MATLAB Codes</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Initial_Models_2"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Initial Models</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Single-Cell_Pathway_Model"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Single-Cell Pathway Model</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Parameter_Estimation_2"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Parameter Estimation</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Initial_Models"> Initial Models </SPAN></H1><P>Two intial models were studied from the literature to get an initial idea of the behaviour of the pathway. The description, analysis and the key conclusions from these studies are presented below
</P><H2><SPAN class="mw-headline" id="Model_of_the_G_-_Protein_cycle"> Model of the G - Protein cycle </SPAN></H2><P>The G - protein cycle is one of the key modules of the of the pathway as it is reponsible for the production of free G-beta-gamma dimer which is crucial for the activation of the MAP kinase cascade.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 1</B>: Reaction diagram of heterotrimeric G protein cycle <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref1" title="Team:TU-Delft/Modeling/SingleCellModel">[1]</A>.</DIV></DIV></DIV>To this end we studied the model of the G-protein cycle by Yi.et.al<A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref1" title="Team:TU-Delft/Modeling/SingleCellModel">[1]</A>.The rates and the reactions used in the model can be found <A href="http://biomodels.caltech.edu:8080/models-caltech/publ/BIOMD0000000072/BIOMD0000000072.pdf" class="external text" rel="nofollow"><B>here</B></A>.
<P>
The model was used to analyze two key effects 
</P><UL><LI> The effect of varying the ligand concentration on the concentration of the free G-beta-gamma dimer.
</LI><LI> The effect of varying the receptor concentration on the concentration of the free G-beta-gamma dimer.
</LI></UL><P>The results are presented below
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:567px;"><DIV class="thumbcaption"><B>Figure 2</B>: Effect of varying the ligand concentration on the Free G-beta-gamma concentration.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:563px;"><DIV class="thumbcaption"><B>Figure 3</B>: Effect of varying the Receptor concentration on the Free G-beta-gamma concentration.</DIV></DIV></DIV></DIV><DIV id="contentbox" style="text-align:justify;"><H3><SPAN class="mw-headline" id="conclusions"> conclusions </SPAN></H3><P>The following conclusions can be drawn from the analysis
</P><UL><LI> The increase in the Ligand concentration also leads to an increase in the Free-G-beta gamma concentration, but after it crosses a threshold(2microMolar) the output saturates which is crucial as it gives a range of inputs that needs to be used in the experiments conducted.
</LI><LI> The increase in the receptor concentration leads to an increase in the Free-G-beta gamma concentration.We are not concerned with the actual concentrations in this case as controlling the exact concentration of the receptor would have been difficult in the time scale of an iGEM project. The data suggested the use of a strong promoter which would produce high quantities of receptor.
</LI></UL><H3><SPAN class="mw-headline" id="limitations"> limitations </SPAN></H3><P>The model well represents the characteristic of the G-Protein cycle but does suffers from two crucial limitations
</P><UL><LI> It does not capture the dynamics of the MAP Kinase cascade.
</LI><LI> It did not contain the gene expression module which is the output of our system.
</LI></UL><P>These limitations motivated us to explore other models.
</P></DIV><DIV id="contentbox" style="text-align:justify;"><H2><SPAN class="mw-headline" id="Model_of_the_Yeast_Pheromone_Response"> Model of the Yeast Pheromone Response </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><A href="/File:YeastPheromonePathwayKofahlKlipp.png" title="File:YeastPheromonePathwayKofahlKlipp.png">File:YeastPheromonePathwayKofahlKlipp.png</A><DIV class="thumbcaption"><B>Figure 5</B>: Yeast Pheromone Pathway <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref2" title="Team:TU-Delft/Modeling/SingleCellModel">[2]</A>.</DIV></DIV></DIV><P>The second model we studied was the model of the yeast pheromone response by Kofahl and Klipp <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref2" title="Team:TU-Delft/Modeling/SingleCellModel">[2]</A>. This model was more detailed and captured well the dynamics of the MAP kinase cascade which the model by Yi.et al failed to do<A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref1" title="Team:TU-Delft/Modeling/SingleCellModel">[1]</A>. The activated <I>Ste12</I> concentration was crucial as it acts as the transcription factor for the synthesis of GFP. So we analyzed two different scenarios concerning this,
</P><UL><LI> The effect of different Ligand concentration on the concentration of the active <I>Ste12</I>.
</LI><LI> The experimentalists suggested the knockout of <I>Far1</I> gene to prevent the yeast mating, the effects of this were also tested using this model by analyzing the variation of the active Ste12 concentration with and without the knockout.
</LI></UL><P>The rates and reactions used in the model can be found <A href="http://www.ebi.ac.uk/biomodels/models-main/publ/BIOMD0000000032/BIOMD0000000032.pdf" class="external text" rel="nofollow"><B>here</B></A>. 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:563px;"><DIV class="thumbcaption"><B>Figure 6</B>: Effect of varying the ligand concentration on the active Ste12 concentration.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:563px;"><DIV class="thumbcaption"><B>Figure 7</B>: Effect of the Far1 gene knockout on the concentration of the active Ste12.</DIV></DIV></DIV></DIV><DIV id="contentbox" style="text-align:justify;"><H3><SPAN class="mw-headline" id="conclusion"> conclusion </SPAN></H3><P>The following conclusions can be drawn from the plots
</P><UL><LI> The increase in Ligand concentration also leads to an increase in active <I>Ste12</I> concentration, but saturates for concentrations greater than 2 microMolar, this effect was also observed in the saturation of the G-beta-gamma concentration in the analysis of the first model, which further validates the choice of the input range selected in our experiments.
</LI><LI> The linear increase in the active Ste12 concentration along with increase in the ligand meant that as we increased the ligand concentration more GFP would be produced. These observations were crucial in terms of the device that we plan to implement for TB detection.
</LI><LI> The second most crucial observation was that the <I>Far1</I> gene if knocked out would not affect the concentration of the active <I>Ste12</I>.
</LI></UL><P>These results helped determine the range of inputs that needed to be applied in the experiments and also helped conclude that the proposed knockout of the <I>Far1</I> gene would not impact the <I>GFP</I> synthesis.
</P></DIV><DIV id="contentbox" style="text-align:justify;"><H2><SPAN class="mw-headline" id="limitations_of_the_initial_models"> limitations of the initial models </SPAN></H2><P>Though the initial models were useful in providing a good understanding of the pathway and the various effects associated with it, their further use for our project suffered from the following limitations
</P><UL><LI>The model of the G-protein cycle lacked the description of MAP kinase cascade and the gene expression module.</LI></UL><UL><LI>The model of the yeast pheromone response on the other hand had a detailed description of the pathway but had two drawbacks, it did not incorporate the gene expression module and more importantly was too complex in terms of the number of parameters that were present in the model (36 Specied, 47 rate constants)
</LI></UL><UL><LI>Fitting the limited data available from  the experiments to the highly complex pathway model in <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref2" title="Team:TU-Delft/Modeling/SingleCellModel">[2]</A> we felt would not be feasible.
</LI></UL><P>
These limitations motivated us to build a reduced order model of the pathway along with incorporating the gene expression module.
</P></DIV><DIV id="contentbox" style="text-align:justify;"><H1><SPAN class="mw-headline" id="Reduced_Model_of_the_Snifformyces_.28Modified_Yeast.29_Pathway"> Reduced Model of the Snifformyces (Modified Yeast) Pathway </SPAN></H1>
To overcome the limitations suggested in the previous section, we constructed <DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 8</B>: Schematic of the modified yeast pheromone pathway</DIV></DIV></DIV> a new reduced order model of the pathway which had 18 species and 21 rate constant, which when compared to the 37 species and 47 rate constants of the model in <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref2" title="Team:TU-Delft/Modeling/SingleCellModel">[2]</A> was a significant reduction.
<P>To construct the model a mechanistic approach was taken. Combining the knowledge from <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref1" title="Team:TU-Delft/Modeling/SingleCellModel">[1]</A> and <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref2" title="Team:TU-Delft/Modeling/SingleCellModel">[2]</A> and also using  <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref3" title="Team:TU-Delft/Modeling/SingleCellModel">[3]</A> and <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref4" title="Team:TU-Delft/Modeling/SingleCellModel">[4]</A>, we constructed the new model shown in the schematic.
</P><P>The key features of the model are as follows,
</P><UL><LI> The Receptor activation &amp; the G - Protein cycle was retained from <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref1" title="Team:TU-Delft/Modeling/SingleCellModel">[1]</A>. The ligand and the receptor degradation were neglected owing to the observation that, the receptors were placed under a constitutive promoter and ligand would not be digested by yeast.
</LI></UL><UL><LI> The simplified MAP kinase cascade was adopted from <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref3" title="Team:TU-Delft/Modeling/SingleCellModel">[3]</A>. The MAPK pathway consists of a complex C to which the free G-beta-gamma dimer binds resulting in the activation of Fus3. The simplified structure captures the dynamics of the activated Fus3 in an efficient manner as can be seen in <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref3" title="Team:TU-Delft/Modeling/SingleCellModel">[3]</A>. This simplification helped a great deal in eliminating the components of the pathway which were not crucial to our experiments. Since the only key components that were under our control were the ligand concentration, the promoter of the receptor and the Fus1-GFP promoter, we decided to retain the G-protein cycle in it's entirety and reduce the MAP kinase pathways to simple two stage reaction.
</LI></UL><UL><LI> The activation of Ste12 was adopted from <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref2" title="Team:TU-Delft/Modeling/SingleCellModel">[2]</A>.
</LI></UL><UL><LI> For the gene expression module, a reaction based model was used which was based on <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref4" title="Team:TU-Delft/Modeling/SingleCellModel">[4]</A>. In this model, the mRNA synthesis is proportional to the concentration of activated Ste12 and the protein synthesis is proportional to the mRNA concentration. The protein synthesis is represented in two stages, the synthesis of nascent GFP which is proportional to the mRNA concentration and the synthesis of mature GFP which is proportional to the concentration of nascent GFP. The mRNA and mature GFP degradations are considered proportional to their concentrations.
</LI></UL><P>For the initial analysis, the rates of this model until Fus3 activation were obtained from <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref3" title="Team:TU-Delft/Modeling/SingleCellModel">[3]</A> in which the model was fit to the data from <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref5" title="Team:TU-Delft/Modeling/SingleCellModel">[5]</A>. The Ste12 activation and deactivation rates were taken from <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref2" title="Team:TU-Delft/Modeling/SingleCellModel">[2]</A>. The rates for the gene expression module were obtained from <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref4" title="Team:TU-Delft/Modeling/SingleCellModel">[4]</A>. The initial model was first interpreted using deterministic semantics. The rates for which can be found <A href="https://static.igem.org/mediawiki/2012/0/05/ReducedModelDetails.pdf" class="external text" rel="nofollow"><B>here</B></A>. Before the model could be used for prediction purposes, the parameters of the model needed to be fit to parameters from our experiments for which the analysis of the system parameters was crucial. The results for this analysis are available <A href="https://2012.igem.org/Team:TU-Delft/Analysis" class="external text" rel="nofollow"><B>here</B></A>. 
</P></DIV><DIV id="contentbox" style="text-align:justify;"><H1><SPAN class="mw-headline" id="Parameter_Estimation"> Parameter Estimation </SPAN></H1><P>Parameter estimation was done as we needed parameters which were obtained from the data generated in our lab. Using the results from the sensitivity analysis, parameter estimation was performed to find the parameters of the model in order to fit the simulated data curves to the experimental data[Fus1-Pr-EGFP WT] in which the wild type cells were stimulated with 2microMolar concentration of alpha-pheromone. The data fitting was done by using the nonlinear regression function <I>nlinfit</I> in the Matlab statistical toolbox <A href="/Team:TU-Delft/Modeling/SingleCellModel#Ref6" title="Team:TU-Delft/Modeling/SingleCellModel">[6]</A>.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:572px;"><DIV class="thumbcaption"><B>Figure 9</B>: Results of the Parameter Estimation; Data values are scaled to the peak signal measured during the stimulation with only pheromone</DIV></DIV></DIV></DIV><DIV id="contentbox" style="text-align:justify;"><H3><SPAN class="mw-headline" id="conclusion_2"> conclusion </SPAN></H3><P>The curve fitting provided a fairly faithful fit to the experimental data. The new rates can be found <A href="https://static.igem.org/mediawiki/2012/0/05/ReducedModelDetails.pdf" class="external text" rel="nofollow"><B>here</B></A>. The model was then used for making predictions about the pathway.
</P></DIV><DIV id="contentbox" style="text-align:justify;"><H1><SPAN class="mw-headline" id="Model_Predictions"> Model Predictions </SPAN></H1><P>The model with the new estimated parameters was used to make predictions on the behaviour of the pathway for different concentrations of the Ligand that were to be used in the experiment. The model was simulated using the ode15s variable order solver from Matlab. The results of these simulations are in Figure 10.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 10</B>: The response of the pathway to different input concentrations</DIV></DIV></DIV></DIV></DIV><DIV id="contentbox" style="text-align:justify;"><H3><SPAN class="mw-headline" id="Results"> Results </SPAN></H3><P>The following observations can be made from the plot about the behaviour of the pathway
</P><UL><LI><B>The GFP concentration increases with the increasing Ligand concentration but saturates for inputs greater than 2microMolar. This is a key observation since these results were also observed in the initial two models and was also verified experimentally in our project.</B></LI><LI><B>Initial traces of GFP can be noticed around 20 minutes after stimulation of the pathway.</B></LI><LI><B>The GFP concentration is at it's maximum at about 2 and a half hours.</B></LI><LI><B>The last two results were also validated experimentally the results of which can be found in the wetlab page.</B></LI></UL></DIV><DIV id="contentbox" style="text-align:justify;"><H1><SPAN class="mw-headline" id="MATLAB_Codes"> MATLAB Codes </SPAN></H1><H3><SPAN class="mw-headline" id="Initial_Models_2"> Initial Models </SPAN></H3><H3><SPAN class="mw-headline" id="Single-Cell_Pathway_Model"> Single-Cell Pathway Model </SPAN></H3><H3><SPAN class="mw-headline" id="Parameter_Estimation_2"> Parameter Estimation </SPAN></H3></DIV><DIV id="contentbox" style="text-align:justify;"><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><TABLE style="margin: 1em auto 1em auto;" class="wikitable sortable" width="90%"><TBODY><TR><TH class="unsortable">Source
</TH></TR><TR><TD style="text-align: center"><SPAN id="Ref1">[1]</SPAN></TD><TD><A href="http://www.pnas.org/content/100/19/10764.full.pdf" class="external text" rel="nofollow">Tau-Mu Yi, Hiroaki Kitano, and Melvin I. Simon,<B>A quantitative characterization of the yeast heterotrimeric G protein cycle</B>, PNAS September 16, 2003  vol. 100  no. 19</A></TD></TR><TR><TD style="text-align: center"><SPAN id="Ref2">[2]</SPAN></TD><TD><A href="http://onlinelibrary.wiley.com/doi/10.1002/yea.1122/pdf" class="external text" rel="nofollow">Kofahl B, Klipp E, <B>Modelling the dynamics of the yeast pheromone pathway</B>, Yeast 2004; 21: 831–850</A></TD></TR><TR><TD style="text-align: center"><SPAN id="Ref3">[3]</SPAN></TD><TD><A href="http://www2.hu-berlin.de/biologie/theorybp/docs/bsc_supady.pdf" class="external text" rel="nofollow">Adriana Supady, <B>Theoretical and computational analysis of negative feedback mechanisms and the dose-response alignment in the pheromone signalling pathway of the yeast Saccharomyces cerevisiae</B></A></TD></TR><TR><TD style="text-align: center"><SPAN id="Ref4">[4]</SPAN></TD><TD><A href="http://www.nature.com/nature/journal/v456/n7223/full/nature07513.html" class="external text" rel="nofollow">Richard C. Yu1, C. Gustavo Pesce1, Alejandro Colman-Lerner1,Larry Lok1, David Pincus1, Eduard Serra1, Mark Holl, Kirsten Benjamin1, Andrew Gordon1, &amp; Roger Brent1 <B>Negative feedback that improves information transmission in yeast signalling</B>, Nature 456, 755-761 (11 December 2008)</A></TD></TR><TR><TD style="text-align: center"><SPAN id="Ref5">[5]</SPAN></TD><TD><A href="http://www2.hu-berlin.de/biologie/theorybp/docs/msc_supady.pdf" class="external text" rel="nofollow">Adriana Supady <B>Mathematical modeling of synthetic logic gates in <I>S.cerevisiae</I></B></A></TD></TR><TR><TD style="text-align: center"><SPAN id="Ref6">[6]</SPAN></TD><TD><A href="http://www.mathworks.nl/help/stats/nlinfit.html" class="external text" rel="nofollow"><B>Matlab nlinfit (Nonlinear Regression)</B></A></TD></TR></TBODY></TABLE></DIV><A><DIV class="printfooter">
Retrieved from &quot;</DIV></A><A href="http://2012.igem.org/Team:TU-Delft/Modeling/SingleCellModel">http://2012.igem.org/Team:TU-Delft/Modeling/SingleCellModel</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU-Delft/Modeling/SingleCellModel" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU-Delft/Modeling/SingleCellModel" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU-Delft/Modeling/SingleCellModel&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU-Delft/Modeling/SingleCellModel&amp;oldid=299301" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>