"
Page
Discussion
View source
History
teams
Log in
 
Team:TU-Delft/Modeling/StochasticSensitivitySpecificityAnalysis
From 2012.igem.org
Team:TUDelft/CSSLaksh
Menu
Home
Team
Wetlab
Overview
Receptor
Reporter
Snifferomyces
Protocols
Sniffer-o-meter
Biobricks
DIY Receptor Design
Modeling
Overview
Structural Model
Single Cell Pathway Model
Pathway Model Analysis
Diffusion Model
Sensitivity-Specificity Analysis
Information Processing Model
Notebook
Week Diary
Brainstorming
Human Practice
Human Outreach
Human Practice
Publicity
Games
Ethics
Safety
More
Crowdfunding
Acknowledgments
Collaboration
Achievements
Contact
Single Cell Model
Biological functions are inherently stochastic in nature, which leads to a wide degree of variability not only at the population level but also at the level of individual cells, which makes it important to test the reliability of our system. Towards this, we first built a stochastic model of the pathway and then used it to assess the specificity and the sensitivity of our device.
Contents
1Stochastic Model
2Sensitivity and Specificity
3Results
4Conclusion
5MATLAB Codes
5.1Stochastic Model & False positive;False negative tests
6References
Stochastic Model
In Saccharomyces cerevisiae, stochasticity (noise) arising from transcription contributes significantly to the level of heterogeneity within a eukaryotic clonal population [1] .
Figure 1: Hybrid model of the modified yeast pathway
In order to investigate the effects of this stochasticity, we decided to build a stochastic model of the pathway using a Hybrid ODE-SDE framework (where SDE stands for stochastic differential equation). Motivated by the fact that the Pheromone signalling is robust against cell to cell variations., we assume the dynamics until the activation of the transcription factor to be deterministic rather than stochastic. As a result of the gene expression being noisy, we build a hybrid stochastic model consisting of deterministic semantics until the activation of the transcription factor Ste12 and treat it as a time varying parameter modulating the reaction based gene expression module interpreted with stochastic semantics using the stochastic differential equations approach [2] , the schematic of which is given in Figure 1.
Twenty simulations were done for different input concentrations. The results of which are in the figure below.
Figure 2: Stochastic Simulation of the System for 20nM Ligand concentration
Figure 3: Stochastic Simulation of the System for 200nM Ligand concentration
Sensitivity and Specificity
Using an input ligand concentration of 200uM (this corresponds to the quantity of Methyl Nicotinate in TB sample), we used the single cell pathway model to set the threshold to the maximum GFP output that was produced. We then used a sample population of 1000 people and a input ligand concentration varying over a range from 0 - 2uM to perform the test. The results from the tests are presented below.
Results
Figure 4: False Negative Analysis
Figure 5: False Positive Analysis
Conclusion
The results indicate that the proposed device has a sensitivity of 87% and a specificity of 74%, which shows that there is a great potential for further improvement. Towards which we are currently making use of the structural model to investigate ways by which better ligand binding affinities can be achieved.
MATLAB Codes
Stochastic Model & False positive;False negative tests
References
Source
[1]
Ozbudak, E. M., Thattai, M., Kurtser, I., Grossman, A. D. & van Oudenaarden,
Regulation of noise in the expression of a single gene,
Nature Genet. 31, 69-73 (2002)
[2]
Desmond J.Hingham
Modeling and Simulating Chemical Reactions,
SIAM Rev., 50(2), 347–368
Retrieved from " http://2012.igem.org/Team:TU-Delft/Modeling/StochasticSensitivitySpecificityAnalysis "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
