<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Amsterdam_test_max_LabJournal skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Amsterdam/test/max/LabJournal</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
      Lab Journal
     </TITLE></P><DIV class="site-wrapper"><DIV class="site-wrapper-inner"><DIV class="cover-container"><NAV class="navbar navbar-inverse"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max">
         Photosynthetic Magic
        </A></DIV><UL class="list-inline nav navbar-right"><LI class="list-inline-item dropdown"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Project">
         Project
         </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Produce">
           Production
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Export">
           Exportation
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Detect">
           Detection
          </A></LI></UL><LI class="list-inline-item"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Model">       Model
        </A></LI><LI class="list-inline-item dropdown"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Practices">
         Practices
         </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Carbon">
           Carbon Efficiency
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Silver">
           HP Silver
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Gold_Integrated">
           HP Gold
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Engagement">
           Engagement
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Safety">
           Safety
          </A></LI><LI class="list-inline-item dropdown"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Lab">
         Lab
         </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Parts">
           Parts
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/InterLab">
           Interlab Study
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/LabJournal">
           Lab Journal
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Protocol">
           Protocols
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Methods">
           Methods
          </A></LI><LI class="list-inline-item dropdown"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Team">
         Team
         </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Attributions">
           Attributions
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Collaborations">
           Collaborations
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Sponsors">
           Sponsors
          </A></LI><LI class="list-inline-item"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Achievements">
         Achievements
        </A></LI></DIV></NAV><DIV class="inner cover"><DIV class="labj-container-black"><DIV class="labj-sum-mid"><DIV class="labj-sum-text-container">
        Good science comes with good documentation - or as our PI says -
        <I>
         “one of the few mistakes that you can do in the lab is to not write down what you did”
        </I>
        . We made a record based on our weekly meetings, which is made available below. Information is grouped chronologically per module, reflecting how the project was approached and documented.
       </DIV><DIV class="labj-sum-mod-container"><DIV class="labj-sum-icon-container"><P class="labj-icon-text">
           Model
          </P></DIV><DIV class="labj-sum-icon-container"><P class="labj-icon-text">
           Production
          </P></DIV><DIV class="labj-sum-icon-container"><P class="labj-icon-text">
           Exportation
          </P></DIV><DIV class="labj-sum-icon-container"><P class="labj-icon-text">
           Detection
          </P></DIV><DIV class="labj-sum-icon-container"><P class="labj-icon-text">
           General Topics
          </P></DIV></DIV></DIV><DIV class="vertical-text-container-outer"><DIV class="vertical-text-container"><P class="vertical-text">
         Lab Journal
        </P></DIV></DIV></DIV><DIV class="labj-container-light"><DIV class="labj-mid-col"><DIV class="week-container" id="14"><DIV class="week-text-container" id="week14" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 14
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week14arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week14module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Robustness analysis. Take optimal flux of the shunt and start decreasing it, so now your mu is going to drop. reduced cost analysis. Conflicting goals of iGEM and Joeri’s project. For iGEM it is necessary to do shorter experiments with more replicates. Evolution is not interesting for iGEM. Fluctuations part might be important for iGEM. First constant light, then smooth di...regime. An easy way to increase fumarate production. Start with omics data, because this feeds into the other modules.
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont">
           ML vs ICL and ML: test if it is thermodynamically possible to make this reversible. You have clear comparison if it reversible. This should be done in the targeted module. The synthetic biology part in E. coli is similar for both projects, so we need to team up: MS by Bram and ICL &amp; MS by Yuki. We have to consider different types of  MS (E. coli, codon optimized from E. coli). The physiology part in Synechocystis is totally different for both modules. Bram uses known promoters, Yuki uses promoter library. The analyzation of the timing of the shunt should be done by Bram with the known promoters. Additional experiment: Get whole library (this is a mix), apply regime, grow until OD. The ones that are still in there perform best.
          </DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont">
           You cannot put kinetics into genome model. You can make tables with affinity (Km and Kcat) and make your choice based on this. Take the maximal predictive flux and see whether this is compatible with the affinity of the transporter. You have to make a rational choice.
          </DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont">
           Linker length is really important. On paper it is quite easy, but in the lab it might be hard. If all else fails, and this works we could present this alone and it will still be good. Calculate CO2 → fumarate efficiency (E. coli vs Synechocystis). From biosensor track to transporter track. Maybe ask a econometrist to do this part. Add a sensitivity analysis.
          </DIV></DIV><DIV class="labj-mod-holder" id="addition"><DIV class="labj-mod-cont"><B>
            Title and project description.
           </B>
           We have to think of a title. We have to be careful for the political correctness of the presentation and title. We have to think about the standards of the other countries. Examples: “Making fumarate around the clock.”  “Sustainability day and night. “
           <B>
            Biosafety
           </B>
           Strains all always outperformed by wildtype. They cannot take over the wildtype, so it is contained. Even if there is contamination, you do not put marked genes in, because our genes are markerless.
          </DIV></DIV></DIV></DIV><DIV class="week-container" id="15"><DIV class="week-text-container" id="week15" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 15
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week15arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week15module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             Max made a model using FAME software.
            </LI>
            Next:
            <LI>
             find optimal growth rate and fumarate production for MS flux.
            </LI><LI>
             look into growing on glycogen instead of glucose.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><LI>
             We use biobricks to fuse the two genes together
            </LI><LI>
             Orientation of TA cloning: by Colony PCR primer design.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Design plan for the lab with Wei.
            </LI><LI>
             Found one potential C4-dicarboxylate transporter in Synechocystis
            </LI><LI>
             Set up pipeline
            </LI><LI>
             Discussed multiple transporter families in bacteria.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             We found that previous systems did not use linkers, so we will also not do this. We will amplify dcuS from the E. coli genome with different lengths, however, both TM domains will always come from dcuS and the kinase will always entirely be envZ.
            </LI><LI>
             Do parallel experiment: …..
            </LI><LI>
             Age is designing primers now. Wei proposed to take different lengths how many different lengths to take from dcuS and how to decide different cutoffs (how much difference should there be)?
            </LI><LI>
             How can you choose the direction of your gene on the T-vector?
            </LI><LI>
             In Wei’s protocol it says ecoRI is in all T-vectors, but snapgene does not identify them as unique restriction sites (they are there but are not unique, does this matter?)
            </LI>
            Two research questions:
            <LI>
             How is fumarate transported in Synechocystis?
            </LI><LI>
             How can we increase fumarate transportation?
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="16"><DIV class="week-text-container" id="week16" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 16
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week16arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week16module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             Experiment to test the software, starting Friday over the weekend.
            </LI><LI>
             Write up on the graphs from last time, it is on the drive: location.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted approach:
            </B><LI>
             Bram and Yuki made solutions in the lab with Wenyang.
            </LI><LI>
             Bram failed his first experiment.
            </LI><LI>
             Bram and Yuki designed primers for the zwf knockout. We both have a file on the drive. The primers from the file from Yuki are already ordered by Dennis and will be used for the markerless fusion-pcr: slr1843-zwf_Yuki .
            </LI></UL><UL><B>
             Untargeted approach:
            </B><LI>
             Made a plan for the untargeted incorporation of the glyoxylate shunt: plan_yuki (if you download it you can zoom in to look at the details)
            </LI><LI>
             Made an account on Benchlink, to access all the files
            </LI><LI>
             The first step will be the transformation of mazF-Sp into ∆fumC as the receptor species.
            </LI><LI>
             Isocitrate lyase (ICL) and malate synthase (MS) need to be fused, with two different rbs in front of the ICL and MS. Patricia only uses cpc, so we need to find another rbs sequence from literature (Englund et al., 2016).
            </LI><LI>
             We need controls to make sure both rbsses are working
            </LI><LI>
             Then ICL-MS needs to be incorporated into a t-vector. Therefore, the restriction sites in ICL-MS need to be checked. Then, we can design primers to fuse ICL and MS and incorporate in into the t-vector.
            </LI></UL><UL><B>
             So the next steps for the untargeted approach will be:
            </B><LI>
             Making solutions (LB and BG-11), plates, antibiotic stock etc. - together with others?
            </LI><LI>
             Deciding what rbs to use from literature
            </LI><LI>
             Check restriction sites in ICL-MS and possibilities for the t-vector.
            </LI><LI>
             Design primers for fusion PCR of ICL-MS.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont">
           New pipeline:
           
              New appointment with Anton
             </DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Designed primers for the biosensor, there has been a slight change in plan -&gt; the Acc65I restriction site on dcuS has to be mutated first
            </LI><LI>
             Made a multiple sequence alignment of dcuS, envZ and cph1, looks well aligned (TMs, HAMPs and HK’s align), might be interesting to look at this with someone with more expertise.
            </LI><LI>
             Discussed possibilities for an intracellular sensor with Klaas (FRET) -&gt; he will ask a connection about a possible fumarate binding protein for this system
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="17"><DIV class="week-text-container" id="week17" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 17
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week17arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week17module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV></DIV></DIV><DIV class="week-container" id="18"><DIV class="week-text-container" id="week18" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 18
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week18arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week18module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             Started pilot experiment on Monday: 7 of 8 vessels are doing well, the 8th not so much
            </LI><LI>
             The 7 are currently in photonfluxostat mode but I will switch them over to the sinusoidal regime later today
            </LI><LI>
             I keep getting contamination when I plate samples on LB plates. However, I don’t think this is due to the vessels being contaminated but rather it has to do with my sampling/plating technique because 1) the contaminant colonies are usually not where I pipetted samples and 2) contaminates have “disappeared” from vessels after multiple platings
            </LI><LI>
             Worked out how much 15N we will need: (less than a gram). I think it was ordered yesterday
            </LI><LI>
             The dFBA model
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted approach
            </B><LI>
             Learning techniques in the lab. Communication with WenYang goes well.
            </LI><LI>
             PCR - amplified H1 and H2 of zwf region. Fused H1 and H2.
            </LI><B>
             Untargeted approach
            </B><LI>
             Made solutions (LB, BG-11) and selection plates.
            </LI><LI>
             Revived the Synechocystis ∆fumC and wildtype strain. They are on BG-11 plates and ready for transformation.
            </LI><LI>
             Revived the E. coli pFLNX H1H2 MazF Ω 1.1 #5.
            </LI><LI>
             Isolated the MazF plasmid and stored it for transformation into ∆fumC.
            </LI><LI>
             Transformed the plasmids that we received from other labs, including the pAQ MS-ICL with the glyoxylate shunt enzymes isocytrate lyase (ICL) and malate synthase (MS). Inoculated in and this afternoon we will perform miniprep to isolate the plasmids.
            </LI><LI>
             Working on a new approach to make the plasmid below:
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont">
           Selected genes and project overview on drive.
           
              Josine needs conclusions on which ones to use and which ones not to use. 
						Make a meeting where you present what you found so far and which transporters specifically you want to express in which strains. Critically go through what you found. Josine has to convince us.
              
               You are planning to overexpress this is in Synechocystis as a final product. First, knock this promoter out to see if it has an effect on fumarate transportation. See what the impact is.
               </DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Made the overview, working in the lab
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="19"><DIV class="week-text-container" id="week19" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 19
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week19arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week19module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             Met with Henk the other day to talk about proteomics prep procedures, 15N is on its way. Monday they will have meeting with Henk.
            </LI><LI>
             Joeri and I have a meeting with Martijs Jonker and Rob Dekker to discuss transcriptomics on Monday at 15.30. Filipe wants to be there.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted approach:
            </B><LI>
             Constructing the zwf knockout plasmid
            </LI><LI>
             Develop plan to make the 36 constructs as parallel as possible
            </LI><B>
             Untargeted approach:
            </B><LI>
             Transformed the selection cassette MazF-Spec into the fumC-knockout.
            </LI><LI>
             New approach to get the glyoxylate shunt enzymes into the H1 H2 plasmid.
            </LI><LI>
             Check if the unannotated region contains a promoter or not
            </LI><LI>
             BLAST search
            </LI><LI>
             Promoter prediction
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Primers for knocking out Sll1103 and Sll1314 are designed, checked and will be ordered. (Both TRAP transport-domains)
            </LI><LI>
             Bram will explain me the pipeline, Wei will guide me with the photoheterotrophic condition experiment, but I need supervision to start in the lab.
            </LI><LI>
             First, Parsa will help Josine for the first 1,5 month. Then Davide will come in to help.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Hopefully mutated the dcuS gene.
            </LI><LI>
             pCI148.
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="20"><DIV class="week-text-container" id="week20" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 20
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week20arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week20module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             Working with Henk tomorrow. Testing out the proteomics procedures.
            </LI><LI>
             Carbon efficiency calculation
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             Zwf-KO plasmid is at sequencing
            </LI><LI>
             Made a scheme to start with constructing the plasmid of the integration of the promoter system.
            </LI><B>
             Untargeted
            </B><LI>
             Confirmed plan with Patricia and Wei: started yesterday in the lab.
            </LI><LI>
             Growth rate will be higher, select for this. Fumarate is aligned with mu. You select on mu.
            </LI><LI>
             Library of e. Coli promoters into synechocystis.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Started in the lab → amplifying homologous regions up and downstream of the gene. Knock-out will be markerless.
            </LI><LI>
             Making a wet-lab scheme to follow the progress
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont">
           Amplified all 9 different dcuS parts successfully, working on envZ half now.
          </DIV></DIV></DIV></DIV><DIV class="week-container" id="21"><DIV class="week-text-container" id="week21" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 21
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week21arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week21module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV></DIV></DIV><DIV class="week-container" id="22"><DIV class="week-text-container" id="week22" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 22
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week22arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week22module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           All single knockouts
           
              R_RBCh: 1 O2 1 D-Ribulose 1,5-bisphosphate ==&gt; 2 H+ 1 3-Phospho-D-glycerate 1 2-Phosphoglycolate
              
               R_PGLYCP: 1 H2O 1 2-Phosphoglycolate ==&gt; 1 Phosphate 1 Glycolate
               
                  R_DDPGA - 1 4-Hydroxy-2-oxoglutarate &lt;=&gt; 1 Pyruvate 1 Glyoxylate
                  
                   R_EHGLAT - 1 2-Oxoglutarate 1 L-erythro-4-Hydroxyglutamate ==&gt; 1 L-Glutamate 1 4-Hydroxy-2-oxoglutarate
                   
                    R_PROAKGOX1r - 1 O2 1 L-Proline 1 2-Oxoglutarate ==&gt; 1 Succinate 1 CO2 1 trans-4-Hydroxy-L-proline
                    
                     R_PHCD - 1 Nicotinamide adenine dinucleotide 2 H2O 1 L-1-Pyrroline-3-hydroxy-5-carboxylate ==&gt; 1 H+ 1 Nicotinamide adenine dinucleotide - reduced 1 L-erythro-4-Hydroxyglutamate
                    </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             Still need to work on the detailed plan of Wei
            </LI><LI>
             Setback in PPP knockout. Used wron T-vector
            </LI><LI>
             WenYang never did experiments as we have planned with the CPH8 system
            </LI></UL><UL><B>
              Untargeted
             </B><LI>
              Making a detailed schedule with Patricia, including non-lab work to speed things up and get a clear overview
             </LI><LI>
              Again transformation of MazF cassette into ∆fumC
             </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont">
           H1-H2 fused for both constructs (sll1103 and sll1314), placed in the T-vector (ligated). Will start transformation procedure next week.
          </DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><B>
             Experiments in ~2 weeks, thinking about experimental conditions (too much now):
            </B><B>
             Strains:
            </B><LI>
             pCIdcuSZ 1-9 + pCI146 (9×)
            </LI><LI>
             pCIdcuSZ 1-9 (9×)
            </LI><LI>
             pCI146
            </LI><LI>
             WT
            </LI><LI>
             pCI148?
            </LI><LI>
             pCI148 + pCI146?
            </LI><B>
             Fumarate in mM:
            </B><LI>
             0, 0.01, 0.05, 0.1, 0.5, 1, 5, 10
            </LI><B>
             Compounds:
            </B><LI>
             Fumarate, succinate, malate, aspartate
            </LI><B>
             Time points:
            </B><LI>
             0, 2, 4, 6, 8
            </LI><B>
             Other:
            </B><LI>
             Continuous light vs. dark
            </LI><LI>
             LB vs BG11?
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="23"><DIV class="week-text-container" id="week23" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 23
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week23arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week23module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             Started dummy 15N experiment
            </LI><LI>
             Struggling with modelling protein synthesis/degradation with FBA in a meaningful way
            </LI><LI>
             Theoretically able to do the entire proteomics procedure by myself. Henk will be back next week and then I will test that theory
            </LI><B>
             Possible ways to incorporate proteomics data into the FBA model
            </B><LI>
             proteomics defines upper constraints
            </LI><LI>
             Colijn, C., Brandes, A., Zucker, J., Lun, D. S., Weiner, B., Farhat, M. R., … Galagan, J. E. (2009). Interpreting expression data with metabolic flux models: Predicting Mycobacterium tuberculosis mycolic acid production. PLoS Computational Biology, 5(8).
            </LI><LI>
             proteomics defines objective function, may be useful during night
            </LI><LI>
             Montezano, D., Meek, L., Gupta, R., Bermudez, L. E., &amp; Bermudez, J. C. M. (2015). Flux balance analysis with objective function defined by proteomics data-metabolism of mycobacterium tuberculosis exposed to mefloquine. PLoS ONE, 10(7), 1–19.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             Performing two approaches. Sending slow 1 for sequencing this afternoon. Sending fast one tomorrow.
            </LI><B>
             Untargeted
            </B><LI>
             Waiting for sequencing primers for the pFLXN-H2-rbs-MS-rbs-ICL-Kan-H1 to arrive before I can sequence it.
            </LI><LI>
             The transformation of the mazF omega cassette into ∆fumC did work, contrary to what Patricia and I thought (yellow dots). Colonies have been picked and regrown to check: no contamination. Colony PCR on these colonies shows that the colonies contain the mazF omega cassette.
            </LI><LI>
             Look into Electroporation for transformation.
            </LI><LI>
             Primers for segregation have to be designed: Send sequence to Wei to see if he already has them.
            </LI><LI>
             Transform ligation mixture again today to check new colonies tomorrow.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont">
           Miraculously took the undigested T-vector instead of the digested one: explains why I got the wrong colony PCR bands and why it did grow on Amp-plates. Started over (I threw away the wrong eppies). Learned a lot.
          </DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Double digestion of all 9 dcuSZ fragments and the pCI148 plasmid with Acc65I and MunI failed. Seperate digestion (after pcr purification) to check seem to show that the enzyme is working really slowly, tried digestion with only Acc65I overnight, this failed -&gt; i probably screwed up the miniprep so the plasmids are not cuttable.
            </LI><LI>
             Got some sequencing results back: what is a good/efficient way to check these?: LaserGene (was DNA Star)
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="24"><DIV class="week-text-container" id="week24" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 24
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week24arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week24module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><B>
             15N trial run is still in the first (pre-culture) phase
            </B><LI>
             analyzed fruits results
            </LI><LI>
             degradation rate on just malate synthase
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             -
            </LI><B>
             Untargeted
            </B><LI>
             Waiting on sequencing primers for my construct
            </LI><LI>
             learned that these things can unexpectedly take a lot of time - think ahead
            </LI><LI>
             Extracted genomic DNA for the first time: the ∆fumC colonies with hopefully the mazF Ω cassette.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont">
           Back to where I was, now with full back-up stocks of DNA, constructs, everything PCRred to the max.
           
            Will do ligation today.
           </DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont">
           Things are going slower than expected, ligation failed and I am out of pCI148. Going to do maxiprep today to have enough pCI148 (low copy number), I will digest the plasmid and do ligation over the weekend. I still have the dcuSZ1-9 fragments.
          </DIV></DIV></DIV></DIV><DIV class="week-container" id="25"><DIV class="week-text-container" id="week25" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 25
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week25arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week25module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont">
           Exam Week
          </DIV></DIV></DIV></DIV><DIV class="week-container" id="26"><DIV class="week-text-container" id="week26" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 26
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week26arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week26module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             almost done with the light fluctuating protocol
            </LI><LI>
             started the real 15N pre-culture
            </LI><LI>
             trial 15N is in the second phase (11 different culture conditions)
            </LI><LI>
             how exactly should I go about harvesting/mixing?
            </LI><LI>
             working through proteomics protocol mostly on my own
            </LI><LI>
             I need to take a step back from some of my iGEM responsibilities so I can write my grant proposal and pro
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             Transformed zwf-h1-h2 to Synechocystis.
            </LI><LI>
             Ordered Cph8 construct partly gene, partly gblock
            </LI><LI>
             Transformed WT and delta FumC strain WTmazf strain
            </LI><LI>
             Waiting to transform and FumCmazf untill Yuki has it finished: it is finished probably tomorrow
            </LI><B>
             Untargeted
            </B><LI>
             Sequencing results MS-ICL look good, but:
            </LI><LI>
             one primer annealed in a different place than designed - sequence this part again
            </LI><LI>
             ∆fumC mazF Ω transformation - a lot of problems amplifying the fragment
            </LI><LI>
             reinoculated into twice as much antibiotics - ready tomorrow
            </LI><LI>
             gDNA extraction protocol for high scale: high concentrations, also after cleaning with RNAse (clean on gel)
            </LI><B>
             working on improving protocols with Patricia
            </B><LI>
             partial digestion (BglII, BamHI, Sau3AI)
            </LI><LI>
             transformation by electroporation
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Sequencing results look good
            </LI><LI>
             sll1103
            </LI></UL><UL><LI>
              sll1314
             </LI></UL><UL><LI>
               MazF cassette ligated → transformed →  colony PCR: confirmed right construct.
              </LI><LI>
               Monday start with transformation to Synechocystis (now preparing strains and media).
              </LI><B>
               Start new project
              </B><LI>
               Primers designed for the overexpression of DcuB and DcuC from E.coli
              </LI><LI>
               Designed gene for synthesis (DcuC from N. Spumigena, 1.300 bp)
              </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             I am very carefully and slowly reperforming the ligation that failed twice
            </LI><B>
             Sequencing results of the T-vector containing the show one amino acid mutation:
            </B><LI>
             Valine -&gt; Isoleucine
            </LI><LI>
             25th a.a. in dcuS (TM1)
            </LI><LI>
             Similar a.a. both hydrophobic; Blast search shows 1 dcuS sequence that has this same mutation (100% similarity with sequencing result)
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="27"><DIV class="week-text-container" id="week27" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 27
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week27arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week27module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             fluctuation protocol is finished (for now)
            </LI><LI>
             current proteomics protocol is only picking up ~200 proteins
            </LI><LI>
             will talk to henk about using the protocol in:
            </LI><LI>
             Borirak, O., de Koning, L. J., van der Woude, A. D., Hoefsloot, H. C. J., Dekker, H. L., Roseboom, W., … Hellingwerf, K. J. (2015). Quantitative proteomics analysis of an ethanol- and a lactate-producing mutant strain of Synechocystis sp. PCC6803. Biotechnology for Biofuels, 8(1), 111. https://doi.org/10.1186/s13068-015-0294-z
            </LI><LI>
             and/or may try hilic chromatography
            </LI><LI>
             15N pre-culture is almost ready (OD: ~1.4), will most likely start the 2nd stage over the weekend
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             Transformed zwf-Mazf to FUMC-OmegaMazf and also to WT-Mazf-Omega.
            </LI></UL><UL><LI>
              Re-inoculate deltaFumC-zwf and WT-Omega
             </LI><LI>
              I was not able to digest the vector pFLXN with 4 enzymes. After 3 tries I found that the NedI enzyme is not cutting the plasmid. Since it is FLXN it should contain it. So now checking if a new enzyme stock works.
             </LI><LI>
              I go on holiday this saturday. I arranged a planning with Wen Yang to continue the steps in Synechocystis.
             </LI></UL><UL><B>
               Untargeted
              </B><LI>
               ∆fumC mazFΩ is ready: glycerol stocks in -80C
              </LI><LI>
               Get also evidence without marker
              </LI><LI>
               Planned to digest gDNA with Sau3AI and MS-ICL with BglII this week and ligate over the weekend -but Sau3AI is not in yet..
              </LI><LI>
               Missed part for sequencing, send again - no results yet
              </LI><LI>
               Making competent cells from different protocols to test efficiency: iGEM Kit Competent Cells test (-20C). Made an excel file that calculates the efficiency from the amount of colonies using this kit.
              </LI><B>
               Tip to check concentration:
              </B><LI>
               Load 5 uL ladder, 1 uL sample. You would at least have the same concentration of your band and your sample, because then it is five times higher than your ladder (compare bands). Filipe says your concentration must be 100.
              </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><B>
             Knock-out-project
  </B><LI>
             Transformation postponed constantly..
            </LI><B>
             Overexpression project
            </B><LI>
             Primers ordered for the overexpression of DcuB and DcuC from E.coli
            </LI><LI>
             Gene sent for synthesis (DcuC from N. Spumigena, 1.300 bp)
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Fully confirmed 7 of the 9 constructs by sequencing, missed a part for construct 3 and construct 6, send those parts for sequencing again
            </LI><LI>
             Ligation is still not working; using Wei’s approach instead of Parsa’s now (less DNA in less volume instead of more of both) -&gt; also checked previous ligation with highly competent cells from Mara, no result
            </LI><LI>
             Inserts seem good when checked on a gel
            </LI><LI>
             Only doubt could be pCI148 (there is an undefined smear on the gel) and volume size of ligation mixture
            </LI><LI>
             Redigested pCI148, checked on gel: very low concentration, can’t see an undefined smear now (can also be because of low concentration)
            </LI><LI>
             I will try to ligate this to one of the inserts in different ratios (used 1:3 up till now), extracting more pCI148 next week.
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="28"><DIV class="week-text-container" id="week28" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 28
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week28arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week28module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           2nd stage of 15N is underway. Harvested the “non-growing” cultures yesterday
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             I also tried to digest the pFL-XN plasmid containing H1H2 with four enzymes and found just like bram that NdeI did not cut. Inspection of my own sequencing results (on pFL-AN) showed that the NdeI site is actually not on the plasmid.
            </LI><LI>
             Continued with the two fragments, blunted ends, checked on gel → unexpected bands showed up after blunting?
            </LI><LI>
             Extracted correct bands from gel, I will start ligation tomorrow so bram can do transformation when he comes back.
            </LI><LI>
             For ligation: is the formation of ori-ampR-ori-ampR (instead of ori-ampR-H1H2) plasmids a big problem? (everything is in one mix)
            </LI><B>
             Untargeted
            </B><LI>
             Confirmed the sequence of pFLXN-h1h2-rbs-MS-rbs-ICL
            </LI><LI>
             primer that was designed to amplify the BglII region (where my promoters will be inserted) annealed again at a total random place - almost no match in sequence.
            </LI><LI>
             Doing a lot of preparational work - Patricia is going away for a couple of weeks
            </LI><B>
             Testing protocols to digest genomic DNA with Sau3AI:
            </B><LI>
             overnight incubation (~16 hours)
            </LI><LI>
             20 µg DNA with 3 µl enzyme
            </LI><LI>
             10 µg DNA with 2 µl enzyme
            </LI><LI>
             Question from Patricia: can we order the large tube with Sau3AI? The tube we received only has enough for 8 digestions - maybe order before Dennis goes away.
            </LI><LI>
             Designed new primers to amplify the mazFΩ fragment in ∆fumC.
            </LI><LI>
             Ligating the gDNA into pFLXN h1h2-rbs-MS-rbs-ICL over the weekend.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><B>
             Project knock-out
            </B><LI>
             Transformation to Synechocystis → now in the stove on BG11 plate (with Kanamycin)
            </LI><B>
             Project overexpress
            </B><LI>
             Amplified DcuC
            </LI><LI>
             Amplified DcuB (Fusion PCR for DcuB to mutate NdeI in the gene)
            </LI><LI>
             Ligation today over the weekend.
            </LI><LI>
             Next week transformation, colony PCR and sequencing check.
            </LI><LI>
             Have done all of this before already, so do not need supervision coming week.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             All constructs fully confirmed by sequencing
            </LI><LI>
             Checked if heat inactivation of enzymes makes a difference after digestion of pCI148; answer is no it does not seem to (at least on gel)
            </LI><LI>
             Checking if the pCI148 plasmid can be fully digested, this has not yet succeeded
            </LI><LI>
             Anniek told me plasmids of this origin are very annoying because they are naturally prepared for mobilization, this means they are often single stranded and folded in the organism, when you extract the single stranded plasmid it cannot be cut by restriction enzymes, this can often lead to ‘false positives’ in your ligation
            </LI><LI>
             Photanol does not use this plasmid anymore but Anniek still has a strain which contains a mutated version of the RSF1010 plasmid which does not get prepared for mobilization (from research by golden lab), maybe i could take this plasmid, cut everything i need out of my plasmid and put it in this one?
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="29"><DIV class="week-text-container" id="week29" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 29
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week29arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week29module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             Leo is currently running an old sample on a 2 hour gradient instead of the previously used 30 minute
            </LI><LI>
             Will also run samples using hilic separation this afternoon
            </LI><LI>
             Centrifuging works
            </LI><LI>
             Starting turbidostats next week
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><B>
              Ecoli.
             </B><LI>
              pFL-XN-H1H2-0168 without restriction sites is constructed and is send for sequencing to check the direction of the ligation.
             </LI><LI>
              IDT sequences arrived, picked them up this morning. I ordered my sequence in three parts. I will dilute them and try to clone the three parts together. (With Wei if possible)
             </LI><B>
              Synechocystis.
             </B><LI>
              Waiting for full segregation of WT-zwf
             </LI><LI>
              Waiting for full segregation of deltaFUMC-zwf-Mazf
             </LI><LI>
              WT-MazF-zwf and Fumc-zwf are not growing, so maybe try once more at perfect OD for the third time. (With Wei if possible)
             </LI></UL><UL><B>
              Untargeted
             </B><LI>
              Started segregation experiment for Synechocystis with Patricia - plate everyday (Wednesday, Thursday, Friday, Saturday) and then every other day (Monday, Wednesday etc).
             </LI><LI>
              I will make a plan with Patricia for when she is on a holiday to continue some running experiments.
             </LI><LI>
              Digested the gDNA and my plasmid with the glyoxylate shunt genes and ligated together. Going to transform today.
             </LI><LI>
              Made competent cells from Mara's protocol (XL1Blue). Tested efficiency - looked online for heat shock protocol.
             </LI><LI>
              90 sec is standard in the lab
             </LI><LI>
              45 sec was recommended
             </LI><LI>&gt;&gt; better efficiency with 45 sec heat shock
             </LI><LI>
              gDNA extraction: 10 min vortex vs 10 min beadbeater
             </LI><LI>
              Testing now
             </LI><LI>
              reinoculated ∆fumC mazFΩ into BG-11 without antibiotics to check segregation - how long should this grow?
             </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><B>
             Project knock-out
            </B><LI>
             Colonies appearing on all plates (△fumC is growing slower, only 2 colonies/plate)
            </LI><LI>
             Friday → Pick colonies and streak on new plate
            </LI><B>
             Project overexpress
            </B><LI>
             Transformation succeeded → send for sequencing yesterday (Wei, still 34 spots to fill before your samples get sequenced…)
            </LI><LI>
             Got the integrative vector with PsbAII promotor from Davide.
            </LI><LI>
             Digested with NdeI and BamHI → success, will extract from gel this afternoon.
            </LI><LI>
             Digested my constructs (Tvector + Dcu gene) with NdeI and BamHI →  failed.
            </LI><LI>
             Tried individual digestions.
            </LI><LI>
             Enzymes, incubation time and buffer are fine (they cut the integrative vector correctly)
            </LI><LI>
             NdeI does not seem to cut  → undigested plasmid (?)
            </LI><LI>
             It is the same for both sequences, making it less likely that it has to do with a sequencing error.
            </LI><LI>
             DcuC from N. Spumigena (cyanobacteria) arrived this morning
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             See new overview for current plan of digesting, ligating and transforming pCI148
            </LI><LI>
             Tips and tricks from Que:
            </LI><LI>
             Do not extract from gel but only pcr purify (however she only needed to cut in one spot) -&gt; Que had problems finding the right band
            </LI><LI>
             Ligate over night or weekend at 4oC
            </LI><LI>
             Que did not use ATP in the ligation mixture and she used T4 ligase from thermo
            </LI><LI>
             A tip Que got but she eventually did not use: Put the ligations on ice and let it melt up to RT (so at least at some point the temperature is optimal)
            </LI><LI>
             Ligations over this weekend (4oC)
            </LI><LI>
             I will each time check if the transformation worked and only take the next step if it did not work.
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="30"><DIV class="week-text-container" id="week30" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 30
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week30arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week30module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             up to 600 identified proteins using the hilic with a 30 minute gradient
            </LI><LI>
             problem with the fluctuations protocol: I forgot to push the protocol onto git before joeri left, which sucks because joeri needs to push new fixes/features through to the release version (password protected) that the computer in the MC lab uses. I’m working on a temporary work-around right now but I’m not sure if it will work.
            </LI><LI>
             If it doesn’t work I will start the the non-fluctuating turbidostat experiments
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             Project zwf knockout
            </LI><LI>
             Transformed ᐃfumC-zwf together with Wei
            </LI><LI>
             WT-zwf is almost fully segregated
            </LI><LI>
             Project construction pflxn-H1-H2-0168 without restriction sites
            </LI><LI>
             Did not manage to get the KpnI site out of the plasmid, so now I will cut with
            </LI><LI>
             KpnI &gt; blunt &gt; re-ligate
            </LI><LI>
             Project construction of synthesised genes
            </LI><LI>
             I ordered 3 separate fragments, which I want to fuse, however 1 of the genes is in a plasmid. I will cut it out with EcorI and EcorEV. To then fuse it with block 2 and then fuse block 2 and 3 with block 1. After which I will receive my synthesised construct.
            </LI><LI>
             If this approach does not work, I have ordered a primer by which I can amplify the fragment I need from this vector.
            </LI></UL><UL><B>
              Untargeted
             </B><LI>
              Segregation ∆fumC mazFΩ
             </LI><LI>
              Gel picture shows full segregation. It is a population, so there might be a 0.1% change there is a synechocystis that is not fully segregated. Discussed with Wei: the steps after (transformation of glyoxylate shunt) will even increase segregation.
             </LI><LI>
              In parallel I will still inoculate from a single fully segregated colony. Plated liquid culture - picked colonies today and plated on a grid. After 3-4 days I will do a colony PCR and inoculate the fully segregated one.
             </LI><B>
              Library construction
             </B><LI>
              I digested my gDNA with Sau3AI and my plasmid with the glyoxylate shunt enzymes (pFLXN h1h2 MS-ICL) with BglII. I ligated them and transformed them: I have a lot of colonies (&gt;1000) :D :D
             </LI><LI>
              Did ligation in different ratio's and did a colony PCR.
             </LI><LI>
              Washed colonies of plate (from protocols on internet) : this is my library in E. coili.
             </LI><LI>
              made a glycerol stock of this library
             </LI><LI>
              inoculated and grown overnight
             </LI><LI>
              Today: miniprep - this will be my library
             </LI><LI>
              Friday: transform library into Synechocystis
             </LI><B>
              Competent cells
             </B><LI>
              I made competent cells with XL1Blue. Tested efficiency with iGEM kit: transformation efficiency: 1.12 E+8. Protocol that works best: 45 seconds heat shock for 50 µl vials.
             </LI><B>
              Segregation experiment: plating
             </B><LI>
              Plated 8 days, then stopped because culture looks dead. Tycho saw results within 7 days, so it should be sufficient.
             </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><B>
             Project knock-out
            </B><LI>
             In Wildtype
            </LI><LI>
             Colony PCR on WT-ᐃsll1103 and WT-ᐃsll1314
            </LI><LI>
             → failed
            </LI><LI>
             Inoculated 3 colonies from each strain anyway.
            </LI><LI>
             In ᐃfumC
            </LI><LI>
             Picked 1 colony from ᐃfumC-ᐃsll1103 and 2 colonies from  ᐃfumC-ᐃsll1314
            </LI><LI>
             → incubating (ᐃfumC-ᐃsll1103 does not seem to grow).
            </LI><LI>
             Did transformation for both constructs in ᐃfumC again (now with Wei)
            </LI><LI>
             → incubating
            </LI></UL><UL><B>
             Project overexpression
            </B><B>
              E. coli transporters
             </B><LI>
              DcuB-ec &amp; DcuC-ec colony PCR with fwd and rev primers on my gene looked correct, but I was not able to cut with NdeI and sequencing results showed undigested T-vector.
             </LI><LI>
              Explanation: Colony PCR is on E.coli, so I amplified from genomic DNA.
             </LI><LI>
              Replated all colonies on cam-plates → 2  colonies did not grow so they probably contain my insert.
             </LI><LI>
              New colony-PCR to check, plasmid isolation, will send it for sequencing today.
             </LI><B>
              NS transporter
             </B><LI>
              (NS = Nodularia Spumigena)
             </LI><LI>
              DcuC-NS transformation to E.coli succeeded → colony PCR confirmation with primers on T-vector show the right construct. Will send it for sequencing today.
             </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Extracted half of the digestion from gel, pcr purified the other half, ligated both with dcuSZ and plated, only colonies in the pcr purified one, running colony pcr atm
            </LI><LI>
             Should i digest the other half of the pcr purified ligation mix with pstI/xhoI to get rid of the original pCI148 plasmid
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="31"><DIV class="week-text-container" id="week31" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 31
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week31arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week31module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Up to 900 identified proteins
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             ∆fumC-∆ZwF
            </LI><LI>
             Is in the first round of transformation, growing on kanamycin plate. I don’t see any colonies on the plate. The plate of Josine is inoculated at the same time and does have many colonies. Which might not be good.
            </LI><LI>
             WT-∆ZwF
            </LI><LI>
             Was contaminated. Trying to rescue it by streaking out very little amount on a kanamycin plate. I now see very little growth on this plate, so I can transform it to a new plate, after which I can transform to liquid culture. The strain is then probably fully segregated, since also this rescue-procedure is performed on kanamycin plates. So I can then start round 2 of the transformation
            </LI><LI>
             Construction of the pFLXN-H1-H2-0168-NoSites
            </LI><LI>
             The pFLXN-H1-H2-0168-NoSites does not contain KpnI, HindIII, SacI and NdeI anymore. Is stored in -80 and approximately 40 ugram of plasmid is extracted.
            </LI><LI>
             Fusion of gBlocks for the Cph8 promoter system
            </LI><LI>
             My gBlock-3 fragment arrived in a plasmid. Cut it out of the plasmid, but now the overlap is a bit low. It does not fuse very well using fusion PCR. After trying a few times. I can try to amplify this fragment from the plasmid using primers. This will lead to more chance on mutations, but does give me a better overlap for my fusion PCR with its neighbour block. A reason for the not working PCR can also be that my DNA is dissolved in TE buffer, which is not compatible with MyTaq since it reacts with Mg2+
            </LI></UL><UL><B>
             Untargeted
            </B><LI>
             I thought Patricia would be back on Monday, but I think she will be back today? This week I have been doing some analytical work and a lot outside the lab, because I have to discuss my next steps with Patricia.
            </LI><LI>
             Segregation ∆fumC mazFΩ
            </LI><LI>
             I made a glycerol stock of my fully segregated liquid culture: ∆fumC mazFΩ #4 (from population).
            </LI><LI>
             Last week I plated liquid culture of ∆fumC mazFΩ on a grid plate. I did colony PCR on them, with my new primers [Seg_H1_fw and Seg_H2_rv]
            </LI><LI>
             Conclusion: all colonies, except colony 7 have the insert. All colonies, except colony 8 are fully segregated. I will inoculate colony #4.
            </LI><LI>
             Library construction - optimizing the process
            </LI><LI>
             Digested more vector (pFLXN h1h2 rbs MS rbs ICL)
            </LI><LI>
             Checked ligation ratio: is more DNA inhibiting?
            </LI><LI>
             In my previous experiment, the plates showed that a ratio ( v:i ) of 1:5 and 1:10 were best, compared to 1:3. Therefore, I now tested the ratio's 1:7, 1:10 and 1:20:
            </LI><LI>
             1:7 2261 colonies
            </LI><LI>
             1:10 816 colonies
            </LI><LI>
             1:20 726 colonies
            </LI><LI>
             Conclusion: I will make my library with a ligation ratio of 1:7.
            </LI><LI>
             I designed a new primer to check the insert of the colonies. I will do a colony PCR today.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Project knock-out
            </LI><B>
             In Wildtype
            </B><LI>
             Almost segregated! Inoculated 10 microliter and will do second transformation when I’m back on monday.
            </LI><B>
             In ᐃfumC
            </B><LI>
             ᐃfumC-ᐃsll1103 ᐃfumC-ᐃsll1314 are growing in liquid medium.
            </LI><LI>
             → will extract DNA on monday and do PCR.
            </LI><LI>
             Did transformation for both constructs in ᐃfumC again (now with Wei)
            </LI><LI>
             → growing on plate
            </LI><B>
             Project overexpression
            </B><LI>
             E. coli transporters
            </LI><LI>
             Colonies did not seem to have correct construct → did ligation again, with newly digested T-vector → will do transformation on monday.
            </LI><B>
             NS transporter
            </B><LI>
             (NS = Nodularia Spumigena)
            </LI><LI>
             Ligated to the expression vector paa-AAH → will do transformation on monday
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             First round i had no colonies, second round i had two colonies, one of them had the construct
            </LI><LI>
             Now for the other 8:
            </LI><LI>
             I still have some of the vector I used and  I still have one of the inserts with a similar concentration as the one that worked. I will ligate this tomorrow. I also digested more pCI148 and will gel-extract it in the same way i did for the ligation that eventually worked, also digested extra insert. I will make 8 more ligations of this tomorrow.
            </LI><LI>
             I want to ask one of you to transform 5 uL of these ligations on wednesday or later, I also want to ask Yuki for some more competent cells for this transformation
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="32"><DIV class="week-text-container" id="week32" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 32
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week32arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week32module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Max is working on his graduation report
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             Constructing the zwf knockouts
            </LI><LI>
             The ∆zwf knockout is still in round 1 and growing on a plate, since it had to be rescued from contamination. It is almost ready to be inoculated in liquid culture, after which I can extract genomic DNA on monday, to see if it is fully segregated.
            </LI><LI>
             The ∆zwf ∆fumC knockout is in round 1 segregating in liquid culture, I hope to extract genomic DNA on monday as well.
            </LI><LI>
             Constructing the Cph8 promoter system
            </LI><LI>
             This week I tried to fuse the three gBlocks, which I orderer together. I did not manage to fuse the three gblocks, after using different PCR protocols with MyTaq and Herculase, therefore I loaded the gblocks on gel to check the quality. I found a problem that gBlock 2 does not run on gel. I could not figure out the problem so I emailed IDT.
            </LI><B>
             Untargeted
            </B><LI>
             Library
            </LI><LI>
             I tested multiple ligation ratio's and found out that a v:i ratio of 1:7 was working best (&gt;2000 colonies). Patricia will now also try the exact same digestion, ligation and transformation protocols for her library - hopefully this also works for her!
            </LI><LI>
             I did a large scale digestion and then ligated (1:7) over the weekend: 80 µl in total (I use 5 µl per transformation).
            </LI><LI>
             Monday I transformed with two types of cells and plated on big square plates:
            </LI><LI>
             Commercially competent cells - 1 E+9 gave: 2300 colonies (100 µl)
            </LI><LI>
             My competent cells - 1.12 E+8  gave: 1640 colonies (100 µl)
            </LI><LI>
             I did a colony PCR of 25 colonies to confirm that also with this transformation, the insert is different (I tested 10 colonies of the transformation on round plates, they were also all different):
            </LI><LI>
             I calculated that I need to do 1.5 transformations for 90% confidence interval.
            </LI><LI>
             I calculated that I need to do 2.8 transformation for 99% confidence interval.
            </LI><LI>
             Next steps:
            </LI><LI>
             collecting library (washing plates etc.)
            </LI><LI>
             transformation into ∆fumC mazFΩ to replace the mazFΩ. How do I need to do this? Multiple flasks to increase probability that we have all possible promoters in there? Or one big one?
            </LI><LI>
             evolution experiment.
            </LI><LI>
             Segregation experiment
            </LI><LI>
             I will repeat the same experiment I talked about last week (the graph: timing and concentration of antibiotics for the library transformation before adding nickel). We are discussing now the concentrations and transformation protocol to ensure we get sufficient data. Since Patricia has no library left, we will use my library-DNA from my first transformation (1:5 ligation ratio; ~500 ng/µl; I have 60 µl and glycerol stocks, so more than enough)
            </LI><LI>
             ∆fumC mazF Ω
            </LI><LI>
             I inoculated one fully segregated colony. It might be interesting to see how the segregation develops when putting it in BG-11 without antibiotics.
            </LI><LI>
             Comparing the fully segregated culture from one colony vs the culture from a population → a revertant experiment
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Project knock-out
            </LI><B>
             In Wildtype
            </B><LI>
             ᐃsll1103-nf fully segragated  → did second round of transformation this morning
            </LI><LI>
             ᐃsll1314-nf almost fully segragated → will do second round of transformation on monday
            </LI><B>
             In ᐃfumC
            </B><LI>
             ᐃfumC-ᐃsll1103 fully segragated  → did second round of transformation this morning
            </LI><LI>
             ᐃfumC-ᐃsll1314 almost fully segragated → will do second round of transformation on monday
            </LI><LI>
             Project overexpression
            </LI><LI>
             E. coli transporters
            </LI><LI>
             New ligation still shows undigested T-vector
            </LI><LI>
             16 new colonies for DcuC-ec
            </LI><LI>
             NS transporter
            </LI><LI>
             (NS = Nodularia Spumigena)
            </LI><LI>
             DcuC-NS in the expression vector paa-AAH colony PCR does not work.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont">
           I am working on Age's project while he is on holiday. Last week he managed to get a right colony for the pCI-dcuSZ7 (we had cake!!) using my transformation protocol and competent cells. So yesterday I did transformation of all other 9 in exactly the same way. I have some colonies.
           
            Next: screening if the colonies contain the right insert.
           </DIV></DIV></DIV></DIV><DIV class="week-container" id="33"><DIV class="week-text-container" id="week33" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 33
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week33arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week33module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Max is working on his graduation report
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             Constructing the ZwF knockouts:
            </LI><LI>
             The WT-∆Zwf knockout was fully segregated, so I started on Wednesday with the 2nd round transformation.
            </LI><LI>
             The ∆fumC-∆Zwf knockout was to my eye, not yet fully segregated, so I reinoculated it and will start the second round transformation on tuesday.
            </LI><LI>
             Mazf and the Zwf gene:
            </LI><LI>
             The MazEF module is composed of a toxin MazF and a antitoxin MazE, which can induce or inhibit cell death. in E.Coli the system requires a signaling peptide which is named extracellular death factor (EDF). This EDF is supposed to be produced from the Zwf enzyme, which was derived from structural analysis. When this EDF is not produced, the MazF system does not trigger in E. Coli. (Kolodkin et al. 2007, Science).
            </LI><LI>
             We use the MazF gene directly behind the Nickl-promoter, therefore the EDF might not be needed for proper function of MazF
            </LI></UL><UL><B>
             Untargeted
            </B><LI>
             The library is 'finished' in E. coli.
            </LI><LI>
             I am making a document on the drive with all the specific details and protocols for making the library, so Patricia can use/repeat this.
            </LI><LI>
             Before being able to transform into Synechocystis I need to redo the segregation experiment to know the antibiotic concentration and the timing of adding Nickel.
            </LI><LI>
             I am going on a holiday - so I planned it in a way that I can finish it before - only 1 day that needs to be covered. Maybe someone else can plate for me one day? Bram will do it
            </LI><LI>
             The ∆fumC-mazFΩ is ready and fully segregated - from fully segregated population, to single segregated colony.
            </LI><LI>
             If I have time, maybe I can do a revertant experiment?
            </LI><LI>
             what is the difference in (the rate of) reversion between a culture originated from one fully segregated colony, compared to a culture originated from a population? - I inoculated both cultures in BG-11 without antibiotic and I can extract gDNA at different time points to check segregation (compared to WT).
            </LI><LI>
             Based on the modelling results, I need the ∆fumC-∆zwf-mazFΩ to transform my library in. Is there something I can help with?
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Project knock-out
            </LI><LI>
             All constructs are fully segregated and are in the second round of transformation. If everything goes according to plan → I can start in september in the cultivators :)
            </LI><LI>
             Project overexpression
            </LI><LI>
             Change of plans:
            </LI><LI>
             Plasmid I got from Davide was not suitable for my project, so Aniek gave me this one.
            </LI><LI>
             Compatible with NdeI and BamHI.
            </LI><LI>
             Digested → still undigested plasmid after 35 min of digestion. BamHI has staractivity after 60 minutes..
            </LI><LI>
             DcuC-NS → ready for new ligation
            </LI><LI>
             DcuC-ec → ready for new ligation (  → send for sequencing)
            </LI><LI>
             DcuB-ec → tomorrow ready for new ligation (confirmed by colony PCR and digestion, will send for sequencing today)
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Bram and Yuki worked on Age's project. Yuki transformed the construct and there were colonies on the plates. However, after Bram did the screening, it appeared that there were no colonies with the right insert. We are working on it until Age comes back. Maybe a different Colony PCR can give more information (with different primers).
            </LI><LI>
             Question: The main difference between the protocol Age uses for transformation and my own protocol is that I always heat inactivate my ligation. As Age did not do this before, I also did not do this (I just repeated exactly what he did). Can inactivation of your ligation make a difference in transformation efficiency?
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="addition"><DIV class="labj-mod-cont"><UL><B>
             Podcast
            </B><LI>
             Thijs and Yuki had a meeting with Wester from VICE. He is interested in collaborating, but still some things needed to be changed (one guest per episode). Yuki is working on that now and has contact with Wester.
            </LI><LI>
             Yuki has an appointment today at 14.00 with Peter Mooij, who wrote the book &quot;de Dikke Alg&quot;. Does anyone wants to join? I think he is really nice/interesting.
            </LI><B>
             Klimaatmarathon
            </B><LI>
             Yuki is arranging a spot at the Klimaatmarathon. We thought we could interview interesting guests and maybe film this. Also, it would be nice to bring some 'cultures'. How should we arrange this?
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="34"><DIV class="week-text-container" id="week34" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 34
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week34arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week34module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Max is working on his graduation report
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>      Targeted
            </B><LI>
             ZwF knockouts
            </LI><LI>
             The WT-zwf knockout is in the second round of transformation, hope to be finished by the start of next week, then we could measure the impact of the ZwF knockout on growth rate, in continuous light, confirming the ZwF pathway is not used during day.
            </LI><LI>
             The ∆fumC-∆Zwf knockout is also in the second round of transformation, I hope that it will be finished by the end of next week, then we could measure if we indeed find a slight increase in Fumarate production, which we modelled, and we can start transforming the MazF into the this knockout.
            </LI><LI>
             Fusing gBlocks
            </LI><LI>
             The problem with gBlock 2 is still not solved, we will try to optimise the PCR-protocol to see if we can still amplify some product. What we will do if this does not work, is amplify the gBlock from the original PCI 148 plasmid and fuse it from there. This is suboptimal, since there were some changes made in the sequence, most importantly there were 2 restriction sites removed from the sequence before synthesis, these are KpnI and NcoI. Especially KpnI is essential to our project. Since this is where we introduce the shunt enzymes.
            </LI><B>
             Untargeted
            </B><LI>
             Almost on holiday, so finishing some things before.
            </LI><LI>
             Final Large Scale Library DNA is done :D
            </LI><LI>
             I have my final library plasmid DNA!! A lot of work in a little bit of water :D
            </LI><LI>
             See this file for the specific details.
            </LI><LI>
             Segregation experiment
            </LI><LI>
             Made calculations to transform my library (the old backup library from 27/7). They can be found here.
  </LI><LI>
             Based on these calculations, I have transformed my library into ∆fumC mazFΩ. Specific details can be found here.
            </LI><LI>
             I concluded that antibiotics do not influence the OD730 in the concentrations that I use.
            </LI><LI>
             I started the segregation experiment: plating everyday, based on OD and Counts/mL measured with the Casy Counter. Josine will plate and measure for me while I am on holiday [Thank you Jo!
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><B>
             Project knock-out
            </B><LI>
             Second round of transformation succeeded (so far) → I can start in two weeks in the multi-cultivators
            </LI><B>
             Project overexpression
            </B><LI>
             Ligated all constructs in the new expression vector, checked → all confirmed
            </LI><LI>
             Will do transformation of 3 constructs into fumC and Wildtype on monday
            </LI><B>
             Will result in 6 strains:
            </B><LI>
             DcuB-ec
            </LI><LI>
             DcuC-ec
            </LI><LI>
             DcuC-NS
            </LI><LI>
             ∆fumC-DcuB-ec
            </LI><LI>
             ∆fumC-DcuC-ec
            </LI><LI>
             ∆fumC-DcuC-NS
            </LI><LI>
             FINISHED IN E.COLI (PARTY!)
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             believe that I have all the constructs (based on col PCR)! With thanks to Yuki and Bram :D
            </LI><LI>
             I have ordered a primer to check the middle of each insert by sequencing so i can check if the variable part of the inserts is as I expect. I can also check if the insert with dcuS is there by digesting with SalI or NheI.
            </LI><LI>
             I have glycerol stocks and extracted plasmid of each construct, I will transform them today (together with pCI146), inoculate on monday, make glycerol stocks and start experiments (?) on tuesday.
            </LI><LI>
             For transformation:
            </LI><LI>
             molar ratio pCI146:pCI148 (in this case pCI-dcuSZ) of 3:1 works best (tested a while back)
            </LI><LI>
             Size ratio is ~1.5 (pCIdcuSZ/pCI146) so weight ratio will be 2:1
            </LI><LI>
             Experiment:
            </LI><LI>
             I would like to discuss this with Wei
            </LI><LI>
             How to continue in synecho?
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="35"><DIV class="week-text-container" id="week35" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 35
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week35arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week35module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont"><UL><LI>
             Starting a new MC experiment tomorrow (was hoping for today). Depending on if the culture is already entrained, the light regimes will either be sinusoidal/sinusoidal-fluctuating or constant/constant-fluctuating
            </LI><LI>
             The culture was taken from a very old plates (april), but still they were growing very nicely. Wei is surprised by the fact that it is still alive.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             ZwF knockouts
            </LI><LI>
             The WT-∆Zwf knockout is finished. It is now growing in liquid culture. Tomorrow or monday I will be able to make glycerol stock and extract genomic DNA.
            </LI><LI>
             The ∆fumC-∆Zwf knockout is also in the second round of transformation, I will be able to pick colonies tomorrow. However I have many colonies on my negative control, which might slow down the process. I did re-inoculate the ∆fumC-∆Zwf , So I have culture ready whenever I need to start again with the second round of transformation.
            </LI><LI>
             The colonies on the negative control plate are fully segragated WTs, which is strange but true. This would meen that the previous round was not fully segregated and so you have to screen many colonies or do another round of transformation.
            </LI><LI>
             Fusing gBlocks
            </LI><LI>       We were not able to amplify gBlock2, therefore we decided to order the gene again. We got some discount, but they could not speed up the process. The fragment is supposed to arrive at the 19th of september.
            </LI><B>
             Untargeted
            </B><LI>
             Yuki is on holidays
            </LI><LI>
             Segregation experiment
            </LI><LI>
             Patricia has taken over. Josine did day 4 and 5.
            </LI><LI>
             Cell
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Project knock-out
            </LI><LI>
             All constructs confirmed with colony PCR → inoculated a lot. Tomorrow is will do gDNA isolation, final check and pre-culture preparations. CULTIVATOR TIME :) Have to plan a meeting with Wei to plan the experiment.
            </LI><LI>
             I can do my experiments for 7 days (do not necessarily need to be 10). This will give me enough time to measure all the constructs that I have made so far in the given time.
            </LI><LI>
             Project overexpression
            </LI><LI>
             Did transformation of 3 constructs into ∆fumC and Wildtype background
            </LI><LI>
             6 new strains that will be ready in roughly 3 weeks :
            </LI><LI>
             DcuB-ec
            </LI><LI>
             DcuC-ec
            </LI><LI>
             DcuC-NS
            </LI><LI>
             ∆fumC-DcuB-ec
            </LI><LI>
             ∆fumC-DcuC-ec
            </LI><LI>
             ∆fumC-DcuC-NS
            </LI><LI>
             Have to set up the experiment for these strains as well.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Experiment delayed because 2 of the 9 e.coli sensors would not grow on plate, picked a new colony from the previous transformation (with only the pCI-dcuSZ plasmid) which did grow. I will send these pCI-dcuSZ plasmids for sequencing and replace my glycerol stock.
            </LI><LI>
             Sequencing confirmed that I have the right constructs, I will still make a nice overview to convince you of this
            </LI><LI>
             I found that I need to use anhydrotetracycline (aTc) to induce expression of the dcuSZ gene, i plan to use 100 ng/mL aTc so I’ll make a stock of 100ug/mL
            </LI><LI>
             I will use M9 media for the experiment - add CaCl2 to 5x M9 salts already or only after when making the media? (found differences in this in online protocol)
            </LI><LI>
             Really good that Age found out that there is an inducible promotor. Wei is the supervisor, but we are experts of our own project.
            </LI><LI>
             Add the CaCl2 after autoclaving, because it can bind phosphate groups.
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="36"><DIV class="week-text-container" id="week36" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 36
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week36arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week36module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Nothing to discuss
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             The ∆fumC-∆Zwf strain is finished, it will be PCR confirmed on monday, then Bram will make glycerol stocks. Yuki and Bram will then both transform the mazFΩ-0168 into the ∆fumC-∆Zwf.
            </LI><LI>
             The ∆Zwf strain is finished and will be tested with ∆fumC-∆Zwf, ∆fumC and WT in the multicultivator under a diurnal regime. We expect from Max his modelling results that the ∆fumC-∆Zwf strain will grow slightly slower but will produce more fumarate.
            </LI><LI>
             gBlocks
            </LI><LI>
             gBlock 2 is supposed to arrive at the 19th of september
            </LI></UL><UL><B>
             Untargeted
            </B><LI>
             Library is ready.
            </LI><LI>
             ∆fumC mazFΩ is ready.
            </LI><LI>
             Helping Bram to finish the ∆fumC-∆zwf as fast as possible. Transforming mazFΩ in there together next week.
            </LI><LI>
             Finishing the segregation experiment to determine at what concentration of Kanamycin I should grow my library on and at what timing I should put it on Nickel to maximize my library.
            </LI><LI>
             I have some strange results that I would like to discuss:
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><B>
             Multi-cultivator
            </B><LI>
             Started on tuesday in the multi-cultivator.
            </LI><LI>
             2 knock-out strains on ∆fumC background (triplicate) , with the ∆fumC (duplu) as a control group. Take sample to measure OD730 and fumarate every day.
            </LI><B>
             Overexpression project
            </B><LI>
             DcuB-ec appeared not to be the right construct: started again..
            </LI><LI>
             Will turn DcuB-ec into a BioBrick as well.
            </LI><LI>
             Will have to perform transformation to Syn. ∆fumC again for DcuC-NS (no colonies)
            </LI><LI>
             Will do this when DcuB-ec is ready.
            </LI><LI>
             Plan to have this ready on tuesday.
            </LI><LI>
             Think well about the design.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             I’ve been having some trouble with getting the strains to grow, they seem really slow, however currently I have dense pre-cultures of all strains in the incubator. I will start the experiment this afternoon, and start the second plate tomorrow
            </LI><LI>
             I want to start the experiment with an OD of 0.3
            </LI><LI>
             The way I have configured it now I will measure OD and Fluorescence every 15 min for up to 18h → protocols and script are ready, maybe Wei wants to double check these with me?
            </LI><LI>
             The second plate will be finished on saturday, I will not be here but the reader stops automatically. I will fetch my data on monday so I can analyse it asap next week and see if things are working
            </LI><LI>
             If some strains are working I will try to transform the system into Synechocystis asap, does it need to be integrated? Does it need to be markerless?
            </LI><LI>
             Make sure to emphasize the importance for this on the wiki page
            </LI><LI>
             At least we want to report whether it works in E.coli or not.
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="37"><DIV class="week-text-container" id="week37" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 37
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week37arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week37module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           repeat constant/constant-fluctuation experiment underway
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             The FumC zwf strain is transformed with the MazFomega cassette by both Bram and Josine in triplicate. Just to make sure it will work.
            </LI><B>
             Multicultivator experiment
            </B><LI>
             Is there one available?
            </LI><B>
             Untargeted
            </B><LI>
             Started segregation dynamics experiment: transformed my library to ∆fumC mazFΩ strain. Yesterday I did the first day of plating (day 0). I will continue for ~10 days: Wednesday 13th of September till Friday 22th of September (every day).
            </LI><LI>
             Confirmed the problem of last experiment was caused by wrong plates. Patricia does not know what went wrong still, but this will not influence my current experiment.
            </LI><LI>
             Plated all colonies on the Kan plates of last experiment to my own Kan+Nickel plates for all concentrations day 1-5. The gridplates are growing - I probably know more tomorrow. This might tell me if 5 days is enough.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Prepared all the samples for HPLC, will be analyzed (soon?)
            </LI><LI>
             DcuB-ec failed again. I don’t know why and it will be too late to test with, even in the best case. Since DcuC-ec showed a higher fumarate yield increase (Yang paper) and has as a major function ‘efflux’, I suggest to discard DcuB-ec as a target.
            </LI><LI>
             Status quo: I got many strains and there are many possibilities for cultivation experiments. Which ones give the most interesting results?
            </LI><LI>
             Re-assimilation experiments on wt under different conditions
            </LI><LI>
             Re-assimilation experiment repeated now also with zwf knock-out
            </LI><LI>
             Re-assimilation experiment including knock-out on wt background
            </LI><LI>
             Overexpression DcuC-ec and DcuC-NS on wt background (strains ready)
            </LI><LI>
             Overexpression DcuC-ec and DcuC-NS on ∆fumC background (strains ready in 2,5 weeks)
            </LI><LI>
             Re-assimilation experiment with DcuC-ec and DcuC-NS overexpression strains (transporters work in both directions)
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Plate 1 did  not show much growth or fluorescence, only a little at the end, I suspect this is because of the lag due to switching from LB to M9
            </LI><LI>
             I left plate 2 over the weekend, saw growth and fluorescence
            </LI><LI>
             Approach for analysis
            </LI><LI>
             Checked blanks
            </LI><LI>
             Sorted the data
            </LI><LI>
             Corrected for position bias
            </LI><LI>
             Checked OD of replicates separately per strain per concentration, removed data of replicates that died or showed very different behavior compared to their counterparts. I would like to discuss which replicates I should remove and which i should leave to be.
            </LI><LI>
             Led to the following result:
            </LI><LI>
             Are there better ways of analysis that you can think of?+
            </LI><LI>
             replicates are technical, not biological, I would like to repeat the experiment to be more sure about strain performance. I also have to repeat plate1. Should i preculture in  M9 or keep platereading times long?
            </LI><LI>
             Should I try a broader range of concentrations?
            </LI><LI>
             I will also test for specificity (compare with other C4-dicarboxylates → aspartate, malate, succinate). Can I eliminate strains that do not appear to work before this?
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="38"><DIV class="week-text-container" id="week38" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 38
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week38arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week38module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Will harvest my turbidostat today and tomorrow
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             The ordered genes arrived from IDT, so I am now constructing the plasmid containing my promoter system. Would it be a nice result, if I am able to test this system in E. coli?
            </LI><LI>
             Setting up the zwf experiment with Max in the multi cultivator, I would like to start on Monday? I aim to test fumarate production in a day/night cycle and the maximum growth rate in both turbidostat and batch mode for WT, ∆zwf, ∆fumC ∆zwf, ∆fumC. This way we can infer if the ∆fumC ∆zwf strain produces more fumarate and if knocking out the zwf gene impairs growth.
            </LI><LI>
             I prepared pre cultures for the multicultivator experiment and checked the sterility on LB plates. The ∆fumC is contaminated.
            </LI><LI>
             The ∆fumC ∆zwf mazFΩ omega is segregating on plate. I think on monday it would be ready to do a colony PCR. The goal with it is to introduce the library and the Shunt. Can we do something without full segregation
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Knock-out MC experiment
            </LI><LI>
             Raw fumarate data per strain
            </LI><LI>
             HPLC has a hard time analyzing samples that are taken from cultures with a high OD (timepoints after  hr). The outliers before  hour were also bad samples (problem with injection??). They will be reanalyzed today.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Started a new plate from LB pre-cultures yesterday only S8 did not grow so i left it out. Used WT instead of my previous control.
            </LI><LI>
             Started platereading yesterday at 17:00 up to now only WT is growing, looking at previous results this is not yet cause for concern. Asked laura to take out and put back my plate when she needs the platereader.
            </LI><LI>
             Made a plan and ordered primers for creation of the plasmid containing the entire system and H1 and H2.
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="39"><DIV class="week-text-container" id="week39" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 39
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week39arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week39module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Nothing to discuss
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             I did not manage to fuse the gene fragments, very annoying. I am in contact with Wei about it. Until Wei is back from holiday, I will try to fuse them  by .
            </LI><LI>
             The ∆fumC ∆zwf mazFΩ is segregating in liquid culture, I am going to test on friday if it is fully segregated.
            </LI><LI>
             I started my multi cultivator experiment with Max, Max is on holiday now, when the OD reaches ., I would like to switch to the diurnal regime. Maybe Joeri can help me with that?
            </LI><LI>
             Take a look at the data of my experiment.
            </LI><LI>
             Update: everything looks contaminated. Both on the contamination plate as in the multicultivator.
            </LI><B>
             Untargeted
            </B><LI>
             I have put my library on Nickel and Kanamycin. Now I have to wait until they turn green. I am monitoring the OD everyday. Looking at previous experiments, I think they will start turn green after the weekend (hopefully!!!!!).
            </LI><LI>
             Made pictures from the plates that are ready together with Jos.
            </LI><LI>
             I still have to edit them.
            </LI><LI>
             Depending on whether ∆fumC ∆zwf mazFΩ is fully segregated, I will transform my library in there on Monday.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Wild-type re-assimilation MC experiment
            </LI><LI>
             New fumarate data,  times diluted (now measured in UV)
            </LI><LI>
             The wild-type culture could be contaminated.. but I don’t see this now on my plates
            </LI><LI>
             this would mean that all cultures were contaminated and in the other channels, fumarate is not consumed at the same rate.
            </LI><LI>
             Knock-out strains
            </LI><LI>
             All contaminated. Most likely a little bugger in my BG, now in all my glycerol stocks and on all my plates. Tried re-streaking multiple times, but have a better plan:
            </LI><LI>
             Take colony from plate (/ second round of transformation), fully segregated and confirmed with colony-PCR.
            </LI><LI>
             Only ∆ now ready (checked for contamination)
            </LI><LI>
             Overexpression strains
            </LI><LI>
             Ready on monday:
            </LI><LI>
             Dcuc-NS
            </LI><LI>
             Dcuc-ec
            </LI><LI>
             ∆fumC-DcuC-ec
            </LI><LI>
             Planning
            </LI><LI>
             Have one MC at my disposal (% CO, % air)
            </LI><LI>
             For  week (have to talk to Aniek)
            </LI><LI>
             NOTE from fbs: The experiment to check fumarate consumption should carry along a fumC deletion strain as a negative control.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             I made new M, this time adding thiamine, proline and arginine to compensate for DHa’s auxotrophies
            </LI><LI>
             out of  strains are currently growing in this new M (only ,  and  don’t)
            </LI><LI>
             In the meantime I’ve started the experiment in LB yesterday, they have been growing fast tonight, however, up to now the FLUO/OD has only decreased, since it started with really high bg fluorescence. The experiment is still running, I can show you the data up to now, from the fluorescence data i think i might want to increase the gain when i do the experiment in LB?
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="40"><DIV class="week-text-container" id="week40" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 40
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week40arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week40module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Nothing to discuss
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             Gblocks are still not fused. Wei will give it a try now. If that doesn’t work, it might be an idea to stop this part.
            </LI><LI>
             ∆fumC∆zwf-mazFΩ is segregating, I am doing a PCR of genomic DNA right now.
            </LI><LI>
             I am waiting for the contaminated ∆fumC∆zwf &amp; ∆zwf to grow on a plate. Hope to be able to pick a colony on friday. For the rest everything is ready for the multicultivator experiment
            </LI><B>
             Untargeted
            </B><LI>
             Plate results are dubious: after day  I see no growth on my plates. Also, my library on Nickel is not growing :(  I will keep the cultures in the incubator just to be sure, maybe one of them does pick up.
            </LI><LI>
             From the plates, it is really hard to make conclusions, because of the low amount of colonies. Based on all results, together with Wei, I decided to continue with Kan and  days of segregation before putting the library on Nickel.
            </LI><LI>
             Wei and I made a very rough calculation:
            </LI><LI>
             After  days, the Kan yielded  colonies in a  µL drop, corresponding to  colonies in  mL. Because I need ~. colonies, I will transform a total of  times.
            </LI><LI>
             To increase my chances of success, I will follow two strategies:
            </LI><LI>
             On plate
            </LI><LI>
             Transform  µL OD=,
            </LI><LI>
             incubate  h in light
            </LI><LI>
             day recovery on BG- plate (no AB) by spreading  µL on membrane(s)
            </LI><LI>
             Transferring membrane on BG- + Kan plate and growing for  days
            </LI><LI>
             Transferring membrane to BG- + Kan + Nickel plate : this will be my library
            </LI><LI>
             In liquid - then plate
            </LI><LI>
             Transform  µL OD=
            </LI><LI>
             incubate  h in light
            </LI><LI>
             Inoculate  µL in  mL BG- no antibiotics -  day recovery
            </LI><LI>
             Add Kan - grow for  days
            </LI><LI>
             Spin culture down - plate on Kan+Nickel plate : this will be my library
            </LI><LI>
             To do this, I am preparing everything: growing my library, extracting plasmids, growing ∆fumC-mazFΩ. Because I need a large volume of high OD this will take some time.
            </LI><LI>
             Maybe I can do the ∆fumC∆zwf-mazFΩ in parallel.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Doing the experiment in shake flasks with S, S and WT (in duplicate; concentrations , ,  and  mM), took measurements at , , ,  and  hrs, no difference after  hrs
            </LI><LI>
             Hypothesis: by the time aTc has induced dcuSZ expression, e.coli has already taken up the extracellular fumarate, solutions:
            </LI><LI>
             Induce first, add fumarate later
            </LI><LI>
             Add more glucose in medium (currently .%)
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Wild-type re-assimilation MC experiment
            </LI><LI>
             Was contaminated.. Now see the white dots on a BG plate.
            </LI><LI>
             New experiment now running (shiny)
            </LI><LI>
             Knock-out strains
            </LI><LI>
             Took new colonies → fully segregated :) Made new glycerol stocks
            </LI><LI>
             Overexpression strains
            </LI><LI>
             Hope to start on monday in the Multicultivator with
            </LI><LI>
             ∆fumC
            </LI><LI>
             ∆fumC-DcuC-ec (fully segregated, but unhappy, yellow)
            </LI><LI>
             ∆fumC-DcuC-NS (not fully segregated, but right band confirmed)
            </LI></UL></DIV></DIV></DIV></DIV><DIV class="week-container" id="41"><DIV class="week-text-container" id="week41" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 41
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week41arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week41module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           Nothing to discuss
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             The ∆fumC∆zwf-mazFΩ strain did contain the plasmid, however it is not growing very well. I re-inoculated some of it yesterday with Wei.
            </LI><LI>
             I have picked colonies for both the ∆fumC∆zwf and the ∆zwf and replated them.They are not yet growing very fast (or I am inpatient). I will pick some more colonies today.
            </LI><B>
             Untargeted
            </B><LI>
             - Large scale library transformation into ∆fumC-mazFΩ today. Multiple approaches:
            </LI><LI>
             growing in liquid
            </LI><LI>
             final selection: on nickel plate
            </LI><LI>
             final selection: in liquid with nickel
            </LI><LI>
             growing on membrane
            </LI><LI>
             final selection: on nickel plate (with membrane)
            </LI><LI>
             Is there more I can do for the library? I do not think it will be beneficial for iGEM. Focus on iGEM?
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Knock-out strains
            </LI><LI>
             Re-assimilation of fumarate in wt and d1103 strain
            </LI><LI>
             No uptake of fumarate
            </LI><LI>
             Overexpression strains
            </LI><LI>
             Experiment #100 in MC2 now running
            </LI><LI>
             Will measure fumarate and OD730 every day
            </LI><LI>
             Note: ∆fumC-DcuC-NS is not fully segregated yet… And the culture is growing slowly and is a bit yellowish.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             I am doing the experiment with flasks for all strains, inducing with aTc and fumarate at the same time, running it for S1 and S2 and WT atm, i have some results for the 16h time point, looks fine, i have not yet subtracted bg fluo because something was wrong with my code, also have not fixed the axes yet.
            </LI><LI>
             I am also currently checking if inducing with aTc first and adding fumarate later would have been a good idea, I am not going to do it for all strains anyway but it is almost no extra effort and i can maybe put it down as a recommendation.
            </LI><LI>
             Has anyone seen sterile small flasks (50mL)?? We’re all out, using 100mL’s now, which is quite annoying.
            </LI></UL></DIV><DIV class="labj-mod-holder" id="addition"><DIV class="labj-mod-cont">
            Tomorrow the people of the Rathenau will come for their workshop synthetic biology. In between, Age will interview one of them for the podcast, and we might want to receive some feedback on the way we structured the human practices from them. The workshop will be given by Yuki, who explains the lab and Bram, who will make silly jokes (and arrange everything). We will transform the plasmids from the transformation efficiency kit we received from iGEM, and then write the name Dirk on 4 plates. then monday we will make a picture and send it to them. It will take us from 1 to 5.
           </DIV></DIV></DIV></DIV></DIV><DIV class="week-container" id="42"><DIV class="week-text-container" id="week42" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 42
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week42arrow">
           ▼
          </P></DIV></DIV></DIV><DIV class="week-container" id="43"><DIV class="week-text-container" id="week43" onclick="displayFunction(this.id)"><DIV class="week-number-container"><P class="week-text">
           Week 43
          </P></DIV><DIV class="arrow-container"><P class="arrow-text" id="week43arrow">
           ▼
          </P></DIV></DIV><DIV class="labj-main-con" id="week43module" style="display:none;"><DIV class="labj-mod-holder" id="stability"><DIV class="labj-mod-cont">
           I have something! Modeling for bram’s experiment.
          </DIV></DIV><DIV class="labj-mod-holder" id="shunt"><DIV class="labj-mod-cont"><UL><B>
             Targeted
            </B><LI>
             Segregating the ∆fumC∆zwf-mazFΩ strain, after encountering problems with the previous inoculation. I took new colonies, these are probably too late for Yuki to use, I test for full segregation today.
            </LI><LI>
             ∆fumC∆zwf and ∆zwf I try to rescue are growing, I did a contamination check and…. Monday we plan to inoculate the two MC (Turbidostat and Batch)
            </LI><LI>
             Monitor the OD very closely, because 0,65 is already enough to get OD 0,05 inoculation.
            </LI><B>
             Untargeted
            </B><LI>
             Did a large scale transformation of my library to ∆fumC-mazFΩ.
            </LI><LI>
             On Monday (day 4), I have put the library on Nickel, using different approaches.
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="transporter"><DIV class="labj-mod-cont"><UL><LI>
             Biolog Eco Microplate
  </LI><LI>
             31 substrates testing for 3 strains (wt, ∆1103, ∆1314)
            </LI><LI>
             Inoculated on tuesday and thursday
            </LI><LI>
             Take some time to show coloring
            </LI><LI>
             Overexpression strains
            </LI><LI>
             Over-expression strains look a bit yellowish. OD’s are comparable
            </LI><LI>
             No fumarate produced in any of the overexpression strains
            </LI><LI>
             Do PCR to check for fumC gene
            </LI><LI>
             Check more carefully for contamination
            </LI></UL></DIV></DIV><DIV class="labj-mod-holder" id="sensor"><DIV class="labj-mod-cont"><UL><LI>
             Results up to 16h are in for all strains, 22h hours will be this afternoon, checking way to most insightfully analyse the data
            </LI><LI>
             Questions
            </LI><LI>
             Is it good to subtract the initial fluo/OD as I did now (to have them start at 0)
            </LI><LI>
             Is there a way to have the 3rd degree fit follow my expectation more:
            </LI><LI>
             It is a logical function, not a 3rd degree
            </LI><LI>
             Do I expect to change the slope of the steep part of the curve (and thus affinity of the sensor?), I would not expect it but it does seem to happen
            </LI><LI>
             Maybe Max or any-one else from the team can prepare fumarate dilutions for Age so that he can do this on friday. It would be a very useful result for iGEM and his thesis, since there is a lot of data missing (saturation, slope, correlation, etc).
            </LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>