<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Amsterdam_test_max_Methods skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Amsterdam/test/max/Methods</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>
      Methods
     </TITLE></P><DIV class="site-wrapper"><DIV class="site-wrapper-inner"><DIV class="cover-container"><NAV class="navbar navbar-inverse"><DIV class="container-fluid"><DIV class="navbar-header"><A class="navbar-brand nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max">
         Photosynthetic Magic
        </A></DIV><UL class="list-inline nav navbar-right"><LI class="list-inline-item dropdown"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Project">
         Project
         </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Produce">
           Production
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Export">
           Exportation
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Detect">
           Detection
          </A></LI></UL><LI class="list-inline-item"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Model">
         Model
        </A></LI><LI class="list-inline-item dropdown"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Practices">
         Practices
         </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Carbon">
           Carbon Efficiency
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Silver">
           HP Silver
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Gold_Integrated">
           HP Gold
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Engagement">
           Engagement
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Safety">
           Safety
          </A></LI><LI class="list-inline-item dropdown"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Lab">
         Lab
         </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Parts">
           Parts
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/InterLab">
           Interlab Study
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/LabJournal">
           Lab Journal
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Protocol">
           Protocols
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Methods">
           Methods
          </A></LI><LI class="list-inline-item dropdown"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Team">
         Team
         </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Attributions">
           Attributions
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Collaborations">
           Collaborations
          </A></LI><LI><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Sponsors">
           Sponsors
          </A></LI><LI class="list-inline-item"><A class="nav-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Achievements">
         Achievements
        </A></LI></DIV></NAV><DIV class="inner cover"><DIV class="summary-container"><DIV class="vertical-text-container-outer"><DIV class="vertical-text-container"><P class="vertical-text">
         methods
        </P></DIV></DIV></DIV><DIV class="meth-container"><DIV class="meth-con-mid"><P class="meth-header" id="large">
        Markerless genetic modification
       </P><P class="meth-header" id="intro1">
        Introduction
       </P><P class="summary-text">
        Traditionally, the introduction of an antibiotic resistance marker accompanies gene insertions or deletions such as the ones that we planned to implement in our project. However, input collected from several sources during our
        <A class="in-text-link" href="https://2017.igem.org/Team:Amsterdam/test/max/HP/Gold_Integrated" target="_blank">
         human practices
        </A>
        activities, pointed out that it would be much better to avoid altogether the presence of resistance markers in the final production strains. Main reasons being that: (i) this makes the handling of the strains in industrial setting cheaper and easier; (ii) marker-free strains make further genetic modification independent of the availability of additional resistance markers; and (iii) the acceptance of our technology, given its added-biosafety,  is increased as our newly designed strains will never be at risk of passing on to natural environments resistance cassettes.
        
         Our team has decided to adopt the markerless genetic modification approach described by Cheah et al., 2012
         <SUP>
          1
         </SUP>
         (see Figure 10.1). This method initially mimics a classical gene insertion method as it also is based on the introduction of a positive selection marker (e.g. resistance to kanamycin or spectinomycin). However, this is done along with the insertion of a counter-selection marker (e.g. the toxic gene mazF) into the same chromosomal locus. Then, resorting to a second homologous recombination, both selection markers are replaced by a sequence of interest, resulting in a markerless gene insertion or deletion. All the production strains in this iGEM project were engineered according to this principle and are therefore, considered biosafe and ready to be further engineered or used in a real-world setting.
        </P><DIV class="method-dropdown" id="method1" onclick="displayFunction(this.id)"><P class="in-text-link" id="method1text">
         ▶  Click here to display this method
        </P></DIV><DIV class="meth-extension" id="method1extension" style="display: none"><P class="meth-header">
         Method
        </P><P class="summary-text" id="header">
         Plasmid construction
        </P><P class="summary-text" id="meth">
         For a markerless gene modification, two plasmids are required: one that contains only the upstream and downstream homologous regions flanking the gene to be deleted / introduced, and another that contains an extra selection cassette in the middle of the upstream and downstream homologous regions. To construct these two plasmids as convenient as possible, we adopted a fusion-PCR based strategy (see Figure 10.2).
         
          Each homologous region was amplified from the genomic DNA of
          <I>
           Synechocystis
          </I>
          by PCR. The two fragments were then fused together and re-amplified through
          <I>
           pfu
          </I>
          DNA Polymerase (Thermo Scientific). The resulting DNA fragment was purified through gel extraction (BIOLINE kit), and attached with one extra adenosine (“A”) of its 3’ overhang by
          <I>
           Taq
          </I>
          DNA Polymerase (Thermo Scientific). TA cloning thus enables the ligation of the fragment with extra “A” to the BioBrick “T” vector (see page on
          <A class="in-text-link" href="https://2017.igem.org/Team:Amsterdam/test/max/Methods#vector" target="_blank">
           BioBrick T-vectors
          </A>
          ), resulting in plasmid_01. Due to the additional restriction site for plasmid_01  introduced through primers between two homologous regions, selection cassette with the corresponding restriction site on both sides can be easily cut and inserted into plasmid_01, resulting in plasmid_02. All the fragments amplified during plasmid construction were sequencing confirmed at Macrogen Europe (The Netherlands).
         </P><P class="summary-text" id="header">
         Segregation
        </P><P class="summary-text" id="meth"><I>
          Synechocystis
         </I>
         can take up foreign DNA spontaneously and integrate it into the genomic DNA by homologous recombination. However, it is a polyploid, each cell containing 4 to 20 copies of the genome
         <SUP>
          2
         </SUP>
         .
         <I>
          Synechocystis
         </I>
         mutant construction takes two rounds of transformation to achieve a markerless gene-deletion. The first round is to fully integrate the selection cassette into the chromosome through homologous recombination, while second round is to completely remove it again. To transform
         <I>
          Synechocystis
         </I>
         with the plasmids, fresh cells were collected either directly from plate or from liquid culture (OD
         <SUB>
          730
         </SUB>
         ≈ 1). After being washed twice with fresh BG11 medium through centrifugation (5,000 rpm, 5 min), cells were further concentrated to a total volume of 200 µL (OD
         <SUB>
          730
         </SUB>
         ≈ 2). The desired plasmid was mixed with these cells to a concentration of 10 μg mL
         <SUP>
          -1
         </SUP>
         , and then the mixture was illuminated with white light of moderate intensity (~50 μmol photons m
         <SUP>
          -2
         </SUP>
         s
         <SUP>
          -1
         </SUP>
         ) for 4 to 6 hours. Next, the mixture was spread on a commercial membrane (Pall Corporation), resting on a BG11 agar plate (without marker selection).  After further illumination for about 16~24 hours, the membrane containing the mixture of cells was transferred to a new BG11 plate, supplemented with kanamycin (first round) or nickel sulfate (second round). After about one week, colonies were picked and streaked sequentially on a new BG11 plate with kanamycin and a plate with nickel sulfate (first round), or a new BG11 plate with nickel sulfate and a plate with kanamycin (second round). Colonies which grew on the BG11 plate with kanamycin but not on plate with nickel sulfate (first round), or on a BG11 plate with nickel sulfate but not on a plate with kanamycin (second round) were candidates for PCR confirmation. Further segregation by serial dilution in liquid culture under the same conditions was applied when necessary.
        </P><FIGURE><FIGCAPTION class="meth-images-caption"><B>
            Figure 10.1.
           </B>
           (A) Plasmid_02 containing up- and downstream homologous regions and the 2 selection markers. The total size of the
           <I>
            mazF/aphII
           </I>
           cassette is ∼3.7 kb. The sizes of the genes in the cassette are as follows: 336 bp (
           <I>
            mazF
           </I>
           ), 2.17 kb (PnrsB,
           <I>
            nrsR-nrsS
           </I>
           cluster), and 1.2 kb (aphII). The homologous regions are approximately 1 kb each. (B) The mechanism of the first homologous recombination. Selection under kanamycin or spectinomycin yields colonies that have incorporated the
           <I>
            mazF/aphII
           </I>
           cassette. (C) Plasmid_01 containing up- and downstream homologous regions. The mechanism of the counter-selection. Selection under nickel yields colonies that have lost the cassette.
          </FIGCAPTION></FIGURE><FIGURE><FIGCAPTION class="meth-images-caption"><B>
            Figure 10.2.
           </B>
           Fusion-PCR based strategy to construct two plasmids required for markerless genetic modification. H1 and H2 were amplified from the genomic DNA of
           <I>
            Synechocystis
           </I>
           and fused. One extra adenosine (“A”) was attached and the fragment was via TA cloning ligated to the BioBrick “T” vector (see page on BioBrick T-vectors), resulting in plasmid_01. Selection cassette is cut with XbaI on both sides and inserted into plasmid_01, resulting in plasmid_02.
          </FIGCAPTION></FIGURE><DIV class="method-dropdown" id="method1" onclick="displayFunction(this.id)"><A class="con-extension" href="#intro1"><DIV class="col-arr"><P class="in-text-link" id="collabarrow">
            ▲
           </P></DIV><P class="in-text-link">
           Click here to hide this method
          </P></A></DIV></DIV></DIV></DIV><DIV class="meth-container" id="dark"><DIV class="meth-con-mid" id="multicultivator"><P class="meth-header" id="large">
        Batch and Turbidostat cultivation
       </P><P class="meth-header" id="intro2">
        Introduction
       </P><P class="summary-text">
        Most of our cultivations are performed in a  Multi-Cultivator.
        <I>
         “It serves for small scale cultivation of algae, bacteria or cyanobacteria. It consists of 8 test-tubes, each holding up to ca. 85 ml of cultivated suspension. The test-tubes are immersed in a thermostated waterbath (its temperature is controlled) . Each tube is independently illuminated, which is independently adjustable for each test-tube in intensity, timing and modulation. Growth rate of the cultivated organisms may be estimated by automatic measuring of optical density - measured at 680 nm and 720 nm independently at each cultivation tube. Each cultivation tube can be bubbled with air or selected gas (optional) of different flow rate through a manually adjustable valve manifold.”
        </I><SUP>
         3
        </SUP></P><DIV class="method-dropdown" id="method2" onclick="displayFunction(this.id)"><P class="in-text-link" id="method2text">
         ▶  Click here to display this method
        </P></DIV><DIV class="meth-extension" id="method2extension" style="display: none"><P class="meth-header">
         Method
        </P><P class="summary-text" id="header">
         Cultivation Conditions
        </P><P class="summary-text" id="meth">
         Cultivations were performed in a Multi-Cultivator (MC1000-OD, PSI, Czech Republic), with light intensity controlled through a “cool-white” LED panel (PSI, CZ). BG11 is the standard medium used, together with 10 mM TES-NaOH (pH = 8.0) cultivated at 30 ℃ and bubbled a mix (v/v) of 99 % N
         <SUB>
          2
         </SUB>
         and 1% CO
         <SUB>
          2
         </SUB>
         at a flow rate of ~150 ml min
         <SUP>
          -1
         </SUP>
         and the working volume  was 60 ml. Inoculation was done such that the initial OD
         <SUB>
          720
         </SUB>
         was 0.05. Continuous light was given at fixed light intensity of 20 µmol photons m
         <SUP>
          -2
         </SUP>
         s
         <SUP>
          -1
         </SUP>
         after inoculation, and then switch to the specified light regime when OD
         <SUB>
          720
         </SUB>
         reached 0.5.
        </P><P class="summary-text" id="header">
         Turbidostat
        </P><P class="summary-text" id="meth">
         The setup used in this experiment is based on a modified Multi-Cultivator, with additional pumps (Reglo ICC, ISMATEC,Germany) transferring fresh medium to the cultures, and from there to the waste. All is controlled using in-house software package controlling the MultiCultivator hardware. This enables measurements of the cell density (720 nm) at regular intervals, and switching on, or off, the pumps to dilute the cultures when the selected OD
         <SUB>
          720
         </SUB>
         threshold is reached. For all experiments, the selected OD
         <SUB>
          720
         </SUB>
         threshold was 0.6. When the OD
         <SUB>
          720
         </SUB>
         threshold was reached, cultures were diluted by 8% (v/ v) of the total working volume with fresh BG-11. Growth rate was calculated by fitting a linear function through the natural logarithm of the OD
         <SUB>
          720
         </SUB>
         measurements during each cell ‘growth-dilution’ cycle.
        </P><DIV class="method-dropdown" id="method2" onclick="displayFunction(this.id)"><A class="con-extension" href="#method1"><DIV class="col-arr"><P class="in-text-link" id="collabarrow">
            ▲
           </P></DIV><P class="in-text-link">
           Click here to hide this method
          </P></A></DIV></DIV></DIV></DIV><DIV class="meth-container"><DIV class="meth-con-mid" id="vector"><P class="meth-header" id="large">
        Biobrick T-vectors
       </P><P class="meth-header" id="intro2">
        Introduction
       </P><P class="summary-text">
        BioBrick parts are DNA sequences which conform to a restriction-enzyme assembly standard. Several standards have been developed, using different restriction enzymes and therefore require different materials and methods. In our lab, BioBrick ‘T’ vectors are used for functional block assembling. The system is based on
        <A class="in-text-link" href="https://en.wikipedia.org/wiki/TA_cloning" target="_blank">
         TA-cloning
        </A>
        (also known as rapid cloning) and avoids the use of restriction enzymes (fig. 10.3).
       </P><DIV class="method-dropdown" id="method3" onclick="displayFunction(this.id)"><P class="in-text-link" id="method3text">
         ▶  Click here to display this method
        </P></DIV><DIV class="meth-extension" id="method3extension" style="display: none"><FIGURE><FIGCAPTION class="image-cloning-caption"><B>
            Figure 10.3.
           </B>
           The technique relies on the ability of adenine (A) and thymine (T) (complementary base pairs) on different DNA fragments to hybridize and, in the presence of ligase, become ligated together. PCR products are usually amplified using Taq DNA polymerase which preferentially adds an adenine to the 3' end of the product. Such PCR amplified inserts are cloned into linearized vectors that have complementary 3' thymine overhangs
          </FIGCAPTION></FIGURE><P class="meth-header">
         Method
        </P><P class="summary-text" id="meth">
         The BioBrick T-vectors have been constructed elsewhere
         <SUP>
          4
         </SUP>
         and can be linearalized simply by enzyme digestion using XcmI, and further gel purification (fig. 10.4). The insert (e.g. PCR product) with an additional adenine added to the 3’ end of the product can then be ligated into the linearized T-vector, without further enzyme digestion steps. The difference between each T-vector is the combination of two restriction sites from: AvrII, NheI, SpeI, and XbaI. Just like BioBricks, constructs can be taken out, combined and chained by digestion ligation using one out of two compatible enzymes (AvrII, NheI, SpeI and XbaI).
         
          Summing up the pros and cons, TA cloning is easier, faster and cheaper than traditional subcloning. This is mainly due to the absence of restriction enzymes and primers to introduce restriction sites. The major downside of TA cloning is the unidirectionality. The gene has, theoretically, 50% chance of getting cloned in the reverse direction.
         </P><FIGURE><FIGCAPTION class="meth-images-caption"><B>
            Figure 10.4.
           </B>
           BioBrick T-vectors used for functional block assembling. A vector can be prepared simply by enzyme digestion using XcmI, and further gel purification. The only difference between each T-vector is the combination of two restriction site from a total of AvrII, NheI, SpeI, and XbaI. The enzymes are compatible and allow functional block assembling.
          </FIGCAPTION></FIGURE><DIV class="method-dropdown" id="method3" onclick="displayFunction(this.id)"><A class="con-extension" href="#method2"><DIV class="col-arr"><P class="in-text-link" id="collabarrow">
            ▲
           </P></DIV><P class="in-text-link">
           Click here to hide this method
          </P></A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>