<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NJU_China_test"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NJU_China/test">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NJU_China/test&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NJU_China/test&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NJU_China/test&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NJU_China/test" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NJU China/test</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P>&lt;!DOCTYPE html&gt;</P><P> </P><DIV class="container"><UL id="nav"><LI><A href="https://2013.igem.org/Team:NJU_China">Home</A></LI><LI><A href="https://2013.igem.org/Team:NJU_China/Project">Project</A></LI><LI><A href="http://https://2013.igem.org/Team:NJU_China/Project#Overview">Overview</A></LI><LI><A https:="">Chassis</A></LI><LI><A https:="">Targeting Module</A></LI><LI><A https:="">Killing Module</A></LI><LI><A https:="">Achievement</A></LI></UL><LI><A href="https://2013.igem.org/Team:NJU_China/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:NJU_China/Human Practice">Human Practice</A></LI><LI><A href="http://https://2013.igem.org/Team:NJU_China/Project#Free debate">Free debate</A></LI><LI><A https:="">Reach the unreached</A></LI><LI><A https:="">Workshop</A></LI><LI><A https:="">Collaboration</A></LI><LI><A href="https://2013.igem.org/Team:NJU_China/Wet lab">Wet lab</A></LI><LI><A href="http://https://2013.igem.org/Team:NJU_China/Project#Protocol">Protocol</A></LI><LI><A https:="">Lab notes</A></LI><LI><A https:="">Data Page</A></LI><LI><A https:="">Parts</A></LI><LI><A href="https://2013.igem.org/Team:NJU_China/Safety">Safety</A></LI><LI><A href="http://https://2013.igem.org/Team:NJU_China/Project#Protocol">Safety form</A></LI><LI><A https:="">Virus-related safety</A></LI><LI><A href="https://2013.igem.org/Team:NJU_China/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:NJU_China/Extras">Extras</A></LI><LI><A href="http://https://2013.igem.org/Team:NJU_China/Project#Protocol">Judging criteria</A></LI><LI><A https:="">Attribution</A></LI><LI><A https:="">Acknowledgement</A></LI><LI><A href="https://2013.igem.org/Team:NJU_China/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:NJU_China/Data Page">Data Page</A></LI><LI><A href="https://2013.igem.org/Team:NJU_China/Parts">Parts</A><H2 class="ss-subtitle">Protocol</H2><DIV id="ss-links" class="ss-links"><A href="#March">1</A><A href="#April">2</A><A href="#May">3</A><A href="#June">4</A><A href="#July">5</A><A href="#August">6</A></DIV><DIV id="ss-container" class="ss-container"><DIV class="ss-row"><DIV class="ss-left"><H4 id="March">Subculturing Cells</H4></DIV><DIV class="ss-right"><H4>Plasmind DNA Transfection</H4></DIV></DIV><DIV class="ss-row ss-medium"><DIV class="ss-right"><H3><A>Subculturing Cells</A><SPAN>
Ready to passage when cell confluence to 80%, and the procedure is carried out in the super clean bench.
1.Remove present culture media. 
2.Wash cells with PBS to remove the residual media.
3.  Add 2 mL of trypsin, and let the cells sit for 2-5 minutes at 37℃ until the cells separated from the culture dish.
Note:It may be necessary to bang the culture flasks on the hood counter to remove any “sticky” cells from the flask surface. 
4.  Immediately after the 2 minutes, add room temp DMEM media (10%FBS) to inactivate the trypsin, and passing the cell several times through a pipette.
5.   Transfer cell suspension to 15 ml tube, centrifuge at 1000rpm for 5 minutes. Remove supernatant.  
6.  Add 5 mL of  PBS, mix immediately by pipetting. centrifuge at 1000rpm for 5 minutes. Remove supernatant. Add DMEM to resuspend the cells
7.  Add cell culture medium DMEM to a new flask(or dish).
8.  Add cell suspension into the new flask in a certain ratio (according to the result of   counting, see Cell Counting), and mix it well.
9.  Place culture flasks back into the 5% CO2 incubator and check daily
Cell Counting
1. Add 5 μL of cell suspension into 45 μL PBS
2. Put 10 μL of the mixture onto the counting cell slide. 
3. Count the cells in the four corner boxes and the middle box.Count border cells on only two of the borders (top and left or bottom and right etc.) 
5. The number of cells in the suspension = number of cells * 104/4 * dilution ratio per milliliter.
</SPAN></H3></DIV><DIV class="ss-right"><H3><A>Plasmind DNA Transfection</A><SPAN>
Use the following procedure to transfect DNA into mammalian cells in 24-well format. For other formats, see Scaling Up or Down Transfections. All amounts and volumes are given on a per well basis. Prepare complexes using a DNA(μg) to Lipofecctamin TM 2000(μl) ratio of 1:2 to 1:3 for most cell lines. Transfect cells at high cell density for high efficiency, high expression levels, and to minimize cytotoxicity, Optimization may be necessary(see Optimizing Plasmid DNA Transfection).
1. Adherent cells: One day before transfection, plate 0.5-2*105 cells in 500μl of growth medium without antibiotics so that cells will be 90%-95% confluent at the time of transfection.
Suspension cells: Just prior to preparing complexes, plate 4-8*105 cells in 500μl of growth medium without antibiotics.
2. For each transfection sample, prepare complexes as follows:
a. Dilute DNA in 50μl of Opti-MEM
b. Mix Lipofectamine TM2000 gently before use, then dilute the appropriate amount in 50μl of opti-MEM ⅠMedium, Incubate for 5 minutes at room temperature. Note: Proceed to Step c Within 25 mintues.
c. After the 5 minute incubation, combine the diluted DNA with diluted Lipofetcamine TM 2000 (total volume=100μl) Mix gently and incubate for 20 mintues at room temperature (solution may appear cloudy). Note: Complexes are stable for 6 hours at room temperature.
3. Add the 100μl of complexes to each well containing cells and medium. Mix gently by rocking the plate back and forth.
4. Incubate cells at 37℃ in a CO2 incubator for 18-48 hours prior to testing for transgene expression. Medium may be changed after 4-6 hours.
5. For stable cell lines: Passage cells at a 1:10 (or higher dilution) into fresh growth medium 24 hours after transfection. Add Selective medium (if desired) the following day.
Scaling Up or Down Transfections
To transfect cells in different tissue culture formats, vary the amount of Lipofectamine TM 2000, nucleic acid, cells, and medium used in proportion to the relative surface area, as shown in the table. With automated, high-throughput systems a complexing volume of 50μl is recommended for transfections in96-well plates. Note: You may perform rapid 96-well plate transfections by plating cells directly into the transfection mix. Prepare complexes in the plate and directly add cells at twice the cell density as in the basic protocol in a 100μl volume. Cells will adhere as usual in the presence of complexes.
1Surface areas may vary depending on the manufacturer.
2Volumes of dilution medium in Step 2a&amp;2b of DNA or RNAi transfection protocols.
Optimizing Plasmid DNA Transfection
To obtain the highest transfection efficiency and low cytotoxicity, optimize trans fection conditions by varying cell density as well as DNA and LipofectamineTM 2000 concentrations. Make sure that cells are greater than 90% confluent and vary DNA(μg): LipofectamineTM 2000(μl) ratio of 1:0.5 to 1:5.
</SPAN></H3></DIV></DIV><DIV class="ss-row"><DIV class="ss-left"><H4 id="April">RNA Extraction</H4></DIV><DIV class="ss-right"><H4>RT-PCR</H4></DIV></DIV><DIV class="ss-row ss-medium"><DIV class="ss-right"><H3><A>RNA Extraction</A><SPAN>
A:TRIzol Reagent (Invitrogen) _Instructions For RNA Isolation
Caution: When working with TRIzol Reagent use gloves and eye protection (shield, safety goggles). Avoid contact with skin or clothing. Use in a chemical fume hood. Aviod breathing vapor. 
Unless otherwise stated, the procedure is carried out at 15 to 30°C, and reagent are at 15 to 30°C.
Reagent required, but not supplied:
Chloroform
Isopropyl alcohol
75% Ethanol (in DEPC-treated water)
RNase-free water or 0.5% SDS solution [ To prepare RNase-free water, draw water into RNase-free galss bottles. Add diethylpyrocarbonate (DEPC) to 0.01% (v/v). Let stand overnight and autoclave. The SDS solution must be prepared using EDPC-treated, autoclaved water.]
1. HOMOGENIZATION (see notes 1-3)
　　　a.  Tissues
　　　Homogenize tissue samples in 1 ml of TRIzol Reagent per 50-100 mg of tissue using a galss-Teflon or power homogenizer (Polytron, or Tekmar’s TISSUMIZER or equivalent). The sample volume should not exceed 10% of the volume of TRIzol Reagent used for homogenization.
　　　b.  Cells Grown in Monolayer
　　　Lyse cells directly in a culture dish by adding 1 ml of TRIzol Reagent to a 3.5 cm diameter dish, and passing the cell lysate several times through a pipette. The amount of TRIzol Reagent added is based on the area of the culture dish (1 ml per 10 cm2) and not on the number of cell present. An insufficient amount of TRIzol Reagent may result in contamination of the isolated RNA with DNA.
　　　c.  Cells Grown in Suspension
　　　Pellet cells by centrifugation. Lyse cells in TRIzol Reagent by repetitive pipetting. Use 1 ml of the reagent per 5-10×106 of animal, plant or yeast cells, or per 1×107 bacterial cells. Washing cells before addition of TRIzol Reagent should be avoided as this increases the possibility of mRNA degradation. Disruption of some yeast and bacterial cells may require the use of a homogenizer.
　OPTIONAL: An additional isolation step may be required for samples with high content of proteins, fat, polysaccharides or extracellular material such as muscles, fat tissue, and tuberous parts of plants. Following homogenization, remove insoluble material from the homogenate by centrifugation at 12000×g for 10 minutes at 2 to 8°C. The resulting pellet contains extracellular membranes, polysaccharides, and high molecular weight DNA, while the supernatant contains RNA. In samples from fat tissue, an excess of fat collects as a top layer which should be removed. In each case, transfer the cleared homogenate solution to a fresh tube and proceed with chloroform addition and phase separation as described.
2. PHASE SEPARATION
　Incubate the homogenized samples for 5 minutes at 15 to 30°C to permit the complete dissociation of nucleoprotein complexes. Add 0.2 ml of chloroform per 1 ml of TRIZOL Reagent. Cap sample tubes securely. Shake tubes vigorously by hand for 15 seconds and incubate them at 15 to 30°C for 2 to 3 minutes. Centrifuge the samples at no more than 12,000×g for 15 minutes at 2 to 8°C. Following centrifugation, the mixture separates into a lower red, phenol-chloroform phase, an interphase, and a colorless upper aqueous phase. RNA remains exclusively in the aqueous phase. The volume of the aqueous phase is about 60% of the volume of TRIZOL Reagent used for homogenization.
3. RNA PRECIPITATION
　Transfer the aqueous phase to a fresh tube, and save the organic phase if isolation of DNA or protein is desired. Precipitate the RNA from the aqueous phase by mixing with isopropyl alcohol. Use 0.5 ml of isopropyl alcohol per 1 ml of TRIZOL Reagent used for the initial homogenization. Incubate samples at 15 to 30°C for 10 minutes and centrifuge at no more than 12,000×g for 10 minutes at 2 to 8°C. The RNA precipitate, often invisible before centrifugation, forms a gel-like pellet on the side and bottom of the tube.
4. RNA WASH
　Remove the supernatant. Wash the RNA pellet once with 75% ethanol, adding at least 1 ml of 75% ethanol per 1 ml of TRIZOL Reagent used for the initial homogenization. Mix the sample by vortexing and centrifuge at no more than 7,500 × g for 5 minutes at 2 to 8°C.
5. REDISSOLVING THE RNA
　At the end of the procedure, briefly dry the RNA pellet (air-dry or vacuum-dry for 5-10 minutes). Do not dry the RNA by centrifugation under vacuum. It is important not to let the RNA pellet dry completely as this will greatly decrease its solubility. Partially dissolved RNA samples have an A260/280 ratio &lt; 1.6. Dissolve RNA in RNase-free water or 0.5% SDS solution by passing the solution a few times through a pipette tip, and incubating for 10 minutes at 55 to 60°C. (Avoid SDS when RNA will be used in subsequent enzymatic reactions.) RNA can also be redissolved in 100% formamide (deionized) and stored at -70°C (5).
RNA Isolation Notes:
1.  Isolation of RNA from small quantities of tissue (1 to 10 mg) or Cell (102 to 104) Samples: Add 800 μl of TRIZOL to the tissue or cells. Following sample lysis, add chloroform and proceed with the phase separation as described in step 2. Prior to precipitating the RNA with isopropyl alcohol, add 5-10 μg RNase-free glycogen (Cat. No 10814) as carrier to the aqueous phase. To reduce viscosity, shear the genomic DNA with 2 passes through a 26 gauge needle prior to chloroform addition. The glycogen remains in the aqueous phase and is co-precipitated with the RNA. It does not inhibit first-strand synthesis at concentrations up to 4 mg/ml and does not inhibit PCR.
2.  After homogenization and before addition of chloroform, samples can be stored at -60 to -70°C for at least one month. The RNA precipitate (step 4, RNA WASH) can be stored in 75% ethanol at 2 to 8°C for at least one week, or at least one year at –5 to -20°C.
3.  Table-top centrifuges that can attain a maximum of 2,600 × g are suitable for use in these protocols if the centrifugation time is increased to 30-60 minutes in steps 2 and 3.
B:TRIzol LS Reagent_RNA Isolation Procedure
Always use the appropriate precautions to avoid RNase contamination when preparing and handling RNA.
RNA pricipitation
1.  (Optional) When precipitating RNA from small sample quantities(&lt;106 cells or &lt;10 mg tissue), add 5-10μg of RNase-free glycogen as a carrier to the aqueous phase. 
Note: Glycogen is co-precipitated with the RNA, but does not inhibit first-strand synthesis at concentrations ≤4mg/mL, and does not inhibit PCR.
2.  Add 0.5mL of 100% isopropanol to the aqueous phase, per of 0.75 mL TRIzol LS Reagent used for homogenization.
3.  Incubate at room temperature for 10 minutes.
4.  Centrifuge at 12,000×g for 10 minutes at 4°C.
Note: The RNA is often invisible prior to centrifugation, and forms a gel-like pellet on the side and bottom of the tube.
5.  Proceed to RNA wash and resuspension.
RNA wash and resuspension
1.  Remove the supernatant from the tube, leaving only the RNA pellet.
2.  Wash the RNA pellet, with 1mL of 75% ethanol per 0.75 mL of TRIzol LS Reagent used for the initial homogenization. Vortex the sample to mix.
3.  Centrifuge the sample at 7500×g for 5 minutes at 4°C, and discard the supernatant.
4.  Vacuum or air dry the RNA pellet for 5-10 minutes. Do not dry the pellet by vacuum centrifuge.
Note: Do not allow the RNA to dry completely, because the pellet can lose solubility. Partially dissolved RNA samples have an A260/280 ratio &lt;1.6.5.  Resuspend the RNA pellet in RNase-free water or 0.5% SDS solution (20-50μL) by passing the solution up and down several times through a pipette tip.
Note: Do not dissolve the RNA in 0.5% SDS if it is to be used in subsequent enzymatic reactions.
6.  Incubate in a water bath or heat block set at 55-60°C for 10-15 minutes.
7.  Proceed to downstream application, or store at -70°C.
</SPAN></H3></DIV><DIV class="ss-right"><H3><A>RT-PCR</A><SPAN>
1. According to the manufacturer’s instruction, use tubes to compound these reagent that listed below. All procedures were carried out on ice.2. Set the PCR as requested below. Then, start to reverse-transcribe the target RNA
16℃ 30minutes
42℃ 30minutes                  1 cycle
85℃ 5minutes
4℃ preserved
Note: qRT-PCR was carried out using a Taqman miRNA PCR kit (Applied Biosystems, Foster City, CA, USA) according to the manufacturer’s instructions. Briefly, 1 μl (approximate 1μg/μl) of total RNA was reverse-transcribed to cDNA using AMV reverse transcriptase (TaKaRa, Dalian, China) and the stem-loop RT primers (Applied Biosystems). Real-time PCR was performed using SYBR (Applied Biosystems) on the Applied Biosystems 7300 Sequence Detection System (Applied Biosystems). All reactions, including the no-template controls, were run in triplicate. After the reactions, the CT values were determined using the fixed threshold settings. 
Absolute Quantification: To calculate the absolute expression levels of the target siRNAs, a series of synthetic siRNA oligonucleotides (dissolved in water) of known concentrations (from 1 fM to 105 fM) were also reverse-transcribed and amplified. The absolute amount of each siRNA was then calculated by referring to the standard curve. 
Relative Quantification: To determine the relatively increasing or decreasing level of the target mRNA, U6 and β-actin were also reverse-transcribed and amplified, used for qRT-PCR analysis. The change of target mRNA was then determined by the comparison between U6 and mRNA, or between β-actin and mRNA.
</SPAN></H3></DIV></DIV><DIV class="ss-row"><DIV class="ss-left"><H4 id="May">qPCR</H4></DIV><DIV class="ss-right"><H4>Plasmids transformation</H4></DIV></DIV><DIV class="ss-row ss-medium"><DIV class="ss-right"><H3><A>qPCR</A><SPAN>
1. According to the manufacturer’s instruction, use tubes to compound these reagent that listed below. All procedures were carried out on ice.
2. Set the PCR as requested below. Then, start to reverse-transcribe the target RNA
</SPAN></H3></DIV><DIV class="ss-right"><H3><A>Plasmids transformation</A><SPAN>1. Obtain a competent cell aliquot from the -80°C freezer, keep the tubes on ice.
2. Add 40μL of ice cold H2O to each aliquot to make a 80 μL solution.
3. Separate the 80 uL solution into 2 separate tubes to make two 40uL tubes of solution.
4. Add 1μL of a Gibson Product into each tube
5. Using the Electroporator:
a. Set the electroporator to 1250 V. and press &quot;time constant&quot;.
b. Obtain a chilled cuvette, and pipet all 40 uL of a sample into the center.
c. Place the cuvette into the electroporator and press &quot;Pluse&quot; twice.
d. Immediately remove the cuvette and rescue the sample in 300-500 mL of LB.
e. Pipet the sample into a labeled microcentrifuge tube, and record the time constant. (The time constant should have a value greater than 2.5).
f. Repeat this process for the other sample. After the addition of LB, transfer the sample into another labeled microcentrifuge tube.
6. Incubate both samples for about 1 hour at 37°C.
7. After incubation, combine both samples (make sure each sample has a comparable time constant) and follow standard plating procedures.
</SPAN></H3></DIV></DIV><DIV class="ss-row"><DIV class="ss-right"><H4 id="June">Plasmids extraction</H4></DIV><DIV class="ss-right"><H4>Plasmids extraction with small quantity</H4></DIV></DIV><DIV class="ss-row ss-medium"><DIV class="ss-right"><H3><A>Plasmids extraction</A><SPAN>
Preparation
1.Column equilibration:  place a Spin Column CP5 into 50 ml collection tube (supplied in the kit) and add 2.5 ml Buffer BL to Spin Column CP5. Centrifuge for 2 min at  8,000 rpm (~8,228 × g).  Discard the flow-throw, and place Spin Column CP5 into the same collection tube.
2.Composition of solution:
   a)Solution I: 50 mM Glucose / 25 mM Tris-Cl / 10 mM EDTA，pH 8.0
   b)Solution II: 0.2 N NaOH / 1% SDS；
   c)Solution III: 3 M potassium acetate/ 2 M acetic acid

Protocol
1.Inoculate 1L LB/ampicillin (50μg/mL) medium placed in a 5 liter culture flask with E.coli carrying desired plasmid and grow at 37/C with agitation for 12-16 hr.        
2.Pellet up to 1-2L bacteria in appropriate vessels by centrifugation at 4,000 rpm for 15 min at 4°C.
3.Decant or aspirate medium and discard. Add 18 mL Solution I. Resuspend cells completely by vortexing or pipetting up and down. Complete resuspension of cell pellet is vital for obtaining good yield.
4.Add 40 mL Solution II, gently mix by inverting and rotating tube several times. Incubate 10 minutes at room temperature.
5.Add 30 mL chilled Solution III, cover, and gently mix by inverting tube several times until a flocculent white precipitate forms. Incubate on ice for 10 minutes.
6.Centrifuge at 7000rpm for 15 minutes at 4oC to pellet the cellular debris and genomic DNA. 
7.Transfer the supernatant into CS1 filter. 
8.Measure the volume of the flow-through and add 0.6 volume of the isopropyl alcohol. Mix by inverting the bottle 15-25 times. Incubate on ice for 10 minutes.
9.Centrifuge at 7000rpm for 15min. Carefully decant the supernatant without disturbing the pellet.. 
10.Resuspend pelleted bacterial cells in 8 ml Buffer P1.
Note: Ensure that RNase A has been added to Buffer P1. No cell clumps should be visible after resuspension of the pellet.
11.Add 8 ml Buffer P2 and mix thoroughly by inverting the tube 6-8 times, and incubating at room temperature for 5 min.
    Note: Mix gently by inverting the tube. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous and slightly clear. Do not allow the lysis reaction to proceed for more than 5 min.
12.Add 8 ml Buffer P4 and mix immediately and thoroughly by inverting the tube 6-8 times, incubate at room temperature for 10 min. The solution should become cloudy.
　　Note: To avoid localized precipitation, mix the solution thoroughly, immediately after addition of Buffer P4.
Centrifuge for 10 min at 8,000 rpm (~8,228 × g). A compact white pellet will form.
13.Note: If use more than 100 ml bacterial culture, prolong centrifugal time to 20-30 min.
14.Transfer the lysate into the Filtration Column CS. Gently insert the plunger into the Filtration
Column CS and filter the cell lysate into a new 50 ml tube (not supplied in the kit).
15.Add 0.3 volume isopropanol to the cleared lysate, sealed the received tube, and mix completely and then transfer all solution to the Spin Column CP5. Centrifuge for 2 min at 8,000 rpm (~8,228 × g).
　　Note:  If the volume of isopropanol–lysate mixture is larger than the capacity of the column, it can be loaded the column two times.
16. Discard the flow-through and place the Spin Column CB5 back into the same collection tube.
17. Add 10 ml Buffer PW to the column and centrifuge at 8,000 rpm (~8,228 × g) for 2 min. Discard the flow-through and place the Spin Columns CB5 back into the same collection tube.
18. Repeat step17.
19. Add 3ml 96%-100% ethanol to the Spin Column CP5 (put the CP5 in a collection  tube). Centrifuge for 2 min at  8,000 rpm (~8,228 × g). 
20. Discard the flow-through and centrifuge at 8,000 rpm (~8,228 × g) for an additional 5 min for removing residual ethanol.
　　Note: Residual ethanol will influence the subsequent enzymatic reaction and sequence. Place the column with cap open in air for several minutes to dry the membrane. 
21. To elute DNA, place the column in a clean 50 ml collection tube(supplied in the kit) and add 1-2 ml Buffer TB to the center of the membrane and incubate 5 min at room temperature, centrifuge at 8,000 rpm (~8,228 × g) for 2 min.
　　Note: Repeat step 14 to increase plasmid callback efficiency.If the volume of eluted buffer is less than 1 ml, it may affect recovery efficiency. The pH value of eluted buffer will have some influence in eluting. Buffer TB or distilled water (pH 7.0-8.5) is suggested to elute plasmid DNA. For long-term storage of DNA, eluting in Buffer TB and storing at  –20°C is recommended, since DNA stored in water is subject to acid hydrolysis.

</SPAN></H3></DIV><DIV class="ss-right"><H3><A>Plasmids extraction with small quantity</A><SPAN>Procedure
1. Inoculate10-15ml LB（with appropriate antibiotic, 50 μg/ml) medium with E.coli carrying desired plasmid isolation and grow at 37ºC with agitation for 12-16 hours.
2. Pellet bacteria in appropriate vessels by centrifugation at 5,000× g for 10 min at room temperature preferably in a swinging bucket rotor.
3. Decant or aspirate medium and discard. To the bacterial pellet add 500 μl Solution I/RNase A.  Resuspend cells completely by vortexing or pipetting up and down. Complete resuspension of cell pellet is vital for obtaining good yields.
4. Transfer the cell suspension to a new 2 ml micro-centrifuge tube. Add 500 μl
Solution II and mix gently but throughly by inverting and rotating tube 7-10 times to obtain a cleared lysate. A 2 min incubation at room temperature may be necessary. Avoid vigorous mixing as this will shear chromosomal DNA and lower plasmid purity. (Store Solution II tightly capped when not in use.)
5. Add 250 μl ice-cold Buffer N3 and mix gently but throughly by inverting tube several times until a flocculent white precipitate forms. Centrifuge at $12,000 × g for 10 minutes at room temperature (preferably at 4°C). 
Note: The Buffers must be mixed thoroughly. If the mixture appears viscous, brownish and conglobated, more mixing is required to completely neutralize the solution. Complete neutralization of the solution is vital of obtaining good yields.
6. CAREFULLY aspirate and transfer the cleared supernatant to a clean 1.5 ml
centrifuge tube. Add 0.1 volume of ETR Solution to the cleared lysate. Mix by inverting tube several times and incubate on ice for 10 minutes. Invert the tube several times during the incubation.
Note: After addition of ETR Solution, the lysate should appear turbid, but it should become clear after incubation on ice. Do not use a clean 2.0 ml centrifuge tube to collect the supernatant, because the ETR Solution will be suspend into solution when too much liquid in single 2 ml tube. 
7. Incubate the lysate at 42oC for 5 minutes. The lysate should appear turbid again. Centrifuge at 12,000 × g for 3 minutes at 25oC. The ETR Solution will form a blue layer at bottom of tube.
8. Transfer the top aqueous phase into a new 2 ml tube and add 0.5 volume of absolute ethanol (room temperature, 96-100%). Mix gently by inverting tube 6-7 times. Incubate at room temperature for 1-2 minutes. 
9. Transfer 700 μl of the mixture (from step 8) into a clean HiBind DNA Mini column II assembled in a 2 ml collection tube(provided) and Centrifuge at10,000 × g for 1 min at room temperature to pass solution through column. Discard the flow-through and re-use the collection tube in next step.
10. Repeat step 9 until all of the remaining of the mixture have been passed through the column and centrifuge as above. Discard the flow-through and re-use the collection tube.
11. Wash column with 500 μl Buffer HB and Centrifuge as above. This step ensures that residual protein contamination is removed and must be included for downstream application requiring high quality DNA.
12. Discard flow-through liquid and wash the column by adding 700 μl DNA Wash Buffer diluted with ethanol. Centrifuge as above and discard flow-through.
Note: DNA Wash Buffer Concentrate must be diluted with absolute ethanol before use. See label for directions. If refrigerated. DNA Wash Buffer must be brought to room temperature before use.
13. Repeat wash step 12 with another 700 μl DNA Wash Buffer.
14. Discard the flow-through liquid and centrifuge the empty column at maximum speed ($13,000×g ) for 3 min to dry the column matrix.  Do not skip this step-it is critical for removing ethanol from the column.
15. Place column into a new clean 1.5ml micro-centrifuge tube. Add 50-80 μl (depending on desired concentration of final product) ddH2O directly onto the column matrix and let it sit at room temperature for 2 minutes. Centrifuge at 13,000 × g for 1 min to elute DNA. This represents approximately 70-85% of bound DNA. An optional second elution will yield any residual DNA, though at a lower concentration.
16.Store purified DNA at 4°C for immediate use or at –20°C for long-term storage. Calculate DNA yield by UV absorbance at 260 nm.
</SPAN></H3></DIV></DIV><DIV class="ss-row"><DIV class="ss-left"><H4 id="July">BCA method to measure protein concentration</H4></DIV><DIV class="ss-right"><H4>Cell Freezing</H4></DIV></DIV><DIV class="ss-row ss-medium"><DIV class="ss-right"><H3><A>BCA method to measure protein concentration</A><SPAN>
1. Preparation 
(1) BCA reagent: Take 50 allots Reagent A and 1 allots Reagent B, mix up. 
(2) Standard protein solution: the concentration of protein is 0, 25, 125, 250, 500, 750, 1000, 1500, 2000 (μg/ml).
(3)Sample solution: 1μl sample + 9μl ddH2O
2. Reaction
(1)Use a cleaning 96-well plate, and add reagents (10μl protein solution+ 200μl BCA reagent per well), respectively and successively.
(2)After adding those reagents, put plate into a incubator and keep standing for 30 minutes at 37℃. Then determine absorbance at 562nm, the first tube is contrast.
3.Determination of protein concentration.
(1)Make absorbance-protein concentration calibration curve, while the standard concentration is x-axis, the absorbance is y-axis.
(2)Record the absorbance of samples. Then use the calibration curve to determine the protein concentration of samples.
</SPAN></H3></DIV><DIV class="ss-right"><H3><A>Cell Freezing</A><SPAN>　　1.For optimum results, cells should be in log phase of growth.For adherent cells, use trypsin to gently detach cells from the substrate on which they are growing. For suspended cell, there is no need to use trypsin
　　2.  Determine the desired viable cell density and calculate the required volume of frozen stock solution needed. Centrifuge cell suspension at approximately 800 to 1000 rpm for 5 minutes (Note: Centrifugation speed and duration may vary depending on cell type). 
　　3. Aseptically decant supernatant without disturbing the cell pellet. Resuspend the pellet in frozen stock solution at recommended viable cell density for specific cell type.
　　4.  Dispense aliquots of this suspension (frequently mixing to maintain a homogeneous cell suspension) into cryogenic storage vials according to the manufacturer’s specifications (i.e., 1.5 mL in a 2.0-mL cryovial). Normal freeze down procedures should take place immediately.
　　5.  Achieve cryopreservation following standard procedures (4 ˚C for several minutes then transfer into -80˚C for 3 hours or overnight).
　　6.  Transfer frozen cells to liquid nitrogen (Vapor phase).
Note: when cells were put in -80˚C, cryogenic storage vials should be perpendicular and fixed in the storage box so that the temperature would decrease slowly. Be sure that the cover of the box is tight.

Ingredients of frozen stock solution: 10％DMSO+50％FBS+40％DMEM or 10% DMSO+90％FBS
</SPAN></H3></DIV></DIV><DIV class="ss-row"><DIV class="ss-left"><H2 id="July">Cell Recovery</H2></DIV></DIV><DIV class="ss-row ss-medium"><DIV class="ss-right"><H3><A>Cell Recovery</A><SPAN>1.  Remove cells from cryopreservation storage and rapidly thaw (&lt; 1 minute) frozen vial in a 37˚C water bath.
2.  Slowly dilute frozen cells with desired amount of complete growth medium by swirling and mixing.
3.  Centrifuge cell suspension at approximately 800 to 1000 rpm for 5 minutes. (Note: Centrifugation speed and duration - may vary depending on cell type).
4.  After centrifugation, check clarity of supernatant and visibility of a complete pellet. Aseptically decant supernatant without disturbing the cell pellet.
5.  Gently resuspend cells in complete growth medium, transfer into appropriate growth vessel and into the recommended culture environment.</SPAN></H3></DIV></DIV></DIV></LI></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:NJU_China/test">http://2013.igem.org/Team:NJU_China/test</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NJU_China/test" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NJU_China/test" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NJU_China/test&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NJU_China/test&amp;oldid=234767" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>