<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Manchester_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Manchester/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><TITLE> Manchester iGEM 2016 </TITLE><NAV><UL id="nav_bar"><LI><A href="https://2016.igem.org/Team:Manchester/Team"><STRONG>Team</STRONG></A></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Team">Team</A></STRONG></LI><LI class="dropdown_content"><STRONG><A href="https://2016.igem.org/Team:Manchester/Attributions">Attributions</A></STRONG></LI></UL><LI><A href="https://2016.igem.org/Team:Manchester/Description"><STRONG>Project</STRONG></A></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Description">Overview</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Description/mechanism1">Cell-free Mechanism</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Description/mechanism2">Inducible Gene Switch</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Proof">Proof of Concept</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Notebook">Notebook</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Project/Protocols">Protocol</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Safety">Safety</A></STRONG></LI><LI class="dropdown_content"><A href="https://2016.igem.org/Team:Manchester/Achievement"><STRONG>Achievement</STRONG></A></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Achievement">Overview</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Judging">Judging Criteria</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Collaborations">Collaborations</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Parts">Parts</A></STRONG></LI><LI class="dropdown_content"><A href="https://2016.igem.org/Team:Manchester/Human_Practices"><STRONG>Human Practices</STRONG></A></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Human_Practices">Overview</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Human_Practices/Charities">Charities</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Human_Practices/Industries">Industries</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Human_Practices/Outreach">Outreach</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Human_Practices/Professional_Bodies">Professional Bodies</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Integrated_Practices">Integrated Practices</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Engagement">Engagement</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/HP/Gold">HP Gold</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/HP/Silver">HP Silver</A></STRONG></LI><LI class="dropdown_content"><A href="https://2016.igem.org/Team:Manchester/Model"><STRONG>Modelling</STRONG></A></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Model">Overview</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Model/hp">Modelling Integration</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Model/result">Results</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Model/ModelExplorer">Model Explorer</A></STRONG></LI><LI class="dropdown_content"><A href="https://2016.igem.org/Team:Manchester/Safety"><STRONG>Safety</STRONG></A></LI></NAV><DIV class="team"><H1 class="title11">Notebook</H1><DIV class="full_size"><BUTTON class="hideall">Hide All</BUTTON><BUTTON class="expandall">Expand All</BUTTON></DIV><DIV style="clear:both" class="full_size"><BUTTON class="geneswitchbutton"><A href="#week85">Inducible Gene Switch</A></BUTTON><BUTTON class="pilotbutton"><A href="#week2">Pilot Experiment</A></BUTTON><BUTTON class="cellfreebutton"><A href="#week84">Cell-free Mechanism</A></BUTTON></DIV><DIV class="full_size"><UL class="credit"><LI>* Inducible Gene Switch - All activities have been performed by Sathya, Hui Wen and Prakrithi.</LI><LI>* Cell Free Mechanism - All activities have been performed by Guada, Mala and Svetlana.</LI><LI>* Pilot Experiment - All activities have been performed by Guada, Mala and Svetlana.</LI></UL></DIV><BUTTON type="button" id="week85" class="week1button">Week 1 (29<SUP>th</SUP> June to 1<SUP>st</SUP> July)</BUTTON><DIV class="geneswitch1"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>Re-suspended DNA (constitutive promoters) from iGEM Distribution kit and transformed it into DH5α strain. </LI><LI>Growth was found on the negative control plates so re-transformed DNA</LI><LI>Inoculated transformed cells containing constitutive promoters in 10 mL of LB broth with Carbenicillin 50. </LI></UL></DIV><BUTTON type="button" id="week2" class="week2button">Week 2 (4<SUP>th</SUP> to 8<SUP>th</SUP> July)</BUTTON><DIV class="pilot2"><H1 class="subnote">Pilot Experiment</H1><UL class="list"><LI>Made stocks of our reagents: glucose solution 0.005 g/mL, glucose oxidase (GOx) solution 0.0025 g/mL, horseradish peroxidase (HRP) solution 250 μL/mL, 2,2′-Azino-bis(3-ethylbenzthiazoline-6-sulfonic acid (ABTS) 0.0025 g/mL.</LI></UL></DIV><DIV class="geneswitch2"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>Performed miniprep for overnight cultures</LI><LI>Prepared first batch of DH5α chemical competent cells using TSS (Transformation and Storage Solution) method.</LI><LI>Restriction enzyme digest of constitutive promoters and control(pUC19) for validation with EcoRI and PstI (1 μL each).</LI></UL></DIV><BUTTON type="button" class="week3button">Week 3 (11<SUP>th</SUP> to 15<SUP>th</SUP> July)</BUTTON><DIV class="pilot3"><H1 class="subnote">Pilot Experiment</H1><UL class="list"><LI>Still awaiting the arrival of the BMG Labtech FLUOstar Omega plate reader.</LI></UL></DIV><DIV class="geneswitch3"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>Prepared more DH5α chemical competent cells for transformation</LI><LI>Prepared Chloramphenicol antibiotic stock for future use.</LI><LI>Restreaked <I>alcA 1</I> (BBa_K678001) and <I>spispink</I> (BBa_K1033923, pink chromoprotein)</LI></UL></DIV><BUTTON type="button" class="week4button">Week 4 (18<SUP>th</SUP> to 22<SUP>nd</SUP>nd July)</BUTTON><DIV class="pilot4"><H1 class="subnote">Pilot Experiment</H1><UL class="list"><LI>We began by preparing master mix containing three reagents: GOx, HRP and ABTS (table 1). PBS was used as a solvent.</LI><CENTER><TABLE class="table1"><TBODY><TR><TH>Final reagent concentration, (μg/ml)</TH></TR><TR><TH>GOx</TH><TD>60</TD></TR><TR><TH>HRP</TH><TD>60</TD></TR><TR><TH>ABTS</TH><TD>100</TD></TR></TBODY></TABLE><B>Table 1.</B>Master mix concentrations.
          </CENTER><LI>Six different glucose concentrations were then made as depicted in table 2. PBS was used as a solvent.</LI><CENTER><TABLE><TBODY><TR><TH>Final glucose concentration, (μg/ml)</TH></TR><TR><TD>0.50</TD></TR><TR><TD>1.00</TD></TR><TR><TD>.25</TD></TR><TR><TD>1.50</TD></TR><TR><TD>1.75</TD></TR><TR><TD>2.00</TD></TR></TBODY></TABLE><B>Table 2.</B> Glucose concentrations. 
          </CENTER><LI>Protocol

          </LI><LI>50 μl of glucose solution was added into each well. Samples were run in triplicates.</LI><LI>Tube containing master mix was placed into spectrophotometer.</LI><LI>Spectrophotometer was then used to measure absorbance of our green coloured product-oxidised ABTS at 420 nm. Absorbance values were taken every 2 sec for 3 min once 150 μl of master mix was added into each well. </LI><LI>Results
          </LI><LI>The rate of reaction did not level off and poor colour intensity was observed.</LI><LI>We repeated the same experiment we did earlier this week. However instead of measuring absorbance for 3 min absorbance was measured every 2 sec for a 5 min period. </LI><LI>Results
              </LI><LI> Reaction rate did level off after measuring absorbance for 5 min. However, colour intensity was still very poor.</LI></UL></DIV><DIV class="geneswitch4"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI><B>Important dates:18<SUP>th</SUP> July - Resurrected DNA from iGEM DNA distribution kit</B></LI><LI>Resuspended plasmids containing required DNA(RBS, <I>amilCP</I> (BBa_K1033930, blue chromoprotein) and <I>amilGFP</I>(BBa_K1033931, yellow chromoprotein)) from iGEM Distribution kit and transformed it into DH5α strain. 
</LI><LI>Inoculated <I>alcA 1</I> and <I>spispink</I> for miniprep.</LI></UL></DIV><BUTTON type="button" class="week5button">Week 5 (25<SUP>th</SUP> to 29<SUP>th</SUP> July)</BUTTON><DIV class="pilot5"><H1 class="subnote">Pilot Experiment</H1><UL class="list"><LI>To increase the brightness of the colour change we repeated the experiment we performed on 22/07/2016 (i.e. absorbance measured every 2 sec for 5 min). However this time we doubled the concentration of each reagent that made up the master mix (table 3).</LI><CENTER><TABLE><TBODY><TR><TH>Final reagent concentration, (μg/ml)</TH></TR><TR><TH>GOx</TH><TD>120</TD></TR><TR><TH>HRP</TH><TD>120</TD></TR><TR><TH>ABTS</TH><TD>200</TD></TR></TBODY></TABLE><B>Table 3. Master mix </B></CENTER><LI style="text-align:left">Result
      </LI><LI style="text-align:left">Increasing the concentration of master mix reagents did not yield intense colour changes </LI><LI>Based on results from yesterday, further increases in in reagents concentration that make up the master mix were tested. We tested the same glucose concentration as on 22/07/2016. The same parameters for the absorbance reading were used (i.e. every 2 sec for 5 min). The only parameter that was changed this time was master mix reagent concentration (table 4).</LI><CENTER><TABLE><TBODY><TR><TH>Final reagent concentration, (μg/ml)</TH></TR><TR><TH>GOx</TH><TD>625</TD></TR><TR><TH>HRP</TH><TD>62.5</TD></TR><TR><TH>ABTS</TH><TD>625</TD></TR></TBODY></TABLE><B>Table 4. Master mix </B></CENTER><LI>Result
      </LI><LI>Strongly visible green colour was not observed following today’s experiment</LI><LI>Today it was decided to increase glucose concentration (table 5) following previous experimental conditions that did not result in appearance of intense green colour. Concentration of master mix reagents and absorbance time window was kept the same as on 22/07/2016 (i.e. absorbance measured every 2 sec for 5 min).</LI><CENTER><TABLE><TBODY><TR><TH>Final glucose concentration, (μg/ml)</TH></TR><TR><TD>0.50</TD></TR><TR><TD>1.00</TD></TR><TR><TD>.25</TD></TR><TR><TD>1.50</TD></TR><TR><TD>1.75</TD></TR><TR><TD>2.00</TD></TR></TBODY></TABLE><B>Table 5</B><LI style="text-align:left">Result
        </LI><LI style="text-align:left">At the end of our experiment we noticed very intense colour change as shown in figure 1. </LI><CENTER><B>Figure 1: Colour intensity following addition of master mix (ABTS,  HRP and H<SUB>2</SUB>O<SUB>2</SUB>)</B></CENTER></CENTER></UL></DIV><DIV class="geneswitch5"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>Ordered genes from IDT: <I>alcA2</I> and <I>alcR</I></LI><LI>Inoculated CP1 and CP3 in 10 mL of LB broth with Carbenicillin 50. </LI></UL></DIV><BUTTON type="button" class="week6button">Week 6 (1<SUP>st</SUP> to 5<SUP>th</SUP> August)</BUTTON><DIV class="pilot6"><H1 class="subnote">Pilot Experiment</H1><UL class="list"><LI>Performed the following reaction where ABTS is oxidised in the presence of H<SUB>2</SUB>O<SUB>2</SUB> (final concentration 250 μg/ml) and HRP (final concentration 250 μg/ml) (table 6).</LI><CENTER><TABLE><TBODY><TR><TH>Final ABTS concentration, (μg/ml)</TH></TR><TR><TD>1</TD></TR><TR><TD>5</TD></TR><TR><TD>10</TD></TR><TR><TD>15</TD></TR><TR><TD>20</TD></TR></TBODY></TABLE><B>Table 6</B></CENTER><LI>Measured absorbance of water  at 420 nm</LI><LI>Measured absorbance of each of the reagents at 420 nm at the following concentrations (table 7).</LI><CENTER><TABLE><TBODY><TR><TH>Concentration, (μg/ml)</TH></TR><TR><TH>HRP</TH><TD>0.25</TD></TR><TR><TH>HRP</TH><TD>2.50</TD></TR><TR><TH>H<SUB>2</SUB>O<SUB>2</SUB></TH><TD>5.08</TD></TR></TBODY></TABLE><B>Table 7</B></CENTER><LI>Set up the reaction with 3 reagents: ABTS, H<SUB>2</SUB>O<SUB>2</SUB>, HRP. Measured absorbance every 25 sec for 900 sec (table 8)</LI><CENTER><TABLE><TBODY><TR><TD>ABTS concentration (μg/ml)</TD><TD>H<SUB>2</SUB>O<SUB>2</SUB> concentration (μg/ml)</TD><TD>HRP concentration (μg/ml)</TD></TR><TR><TD>1.25</TD><TD>1.25</TD><TD>0.0625</TD></TR><TR><TD>1.875</TD><TD>1.875</TD><TD>0.0625</TD></TR><TR><TD>2.5</TD><TD>2.5</TD><TD>0.0625</TD></TR><TR><TD>3.125</TD><TD>3.125</TD><TD>0.0625</TD></TR><TR><TD>3.750</TD><TD>3.750</TD><TD>0.0625</TD></TR><TR><TD>4.375</TD><TD>4.375</TD><TD>0.0625</TD></TR><TR><TD>5.000</TD><TD>5.000</TD><TD>0.0625</TD></TR><TR><TD>5.625</TD><TD>5.625</TD><TD>0.0625</TD></TR><TR><TD>6.25</TD><TD>6.25</TD><TD>0.0625</TD></TR><TR><TD>6.875</TD><TD>6.875</TD><TD>0.0625</TD></TR><TR><TD>7.5</TD><TD>7.5</TD><TD>0.0625</TD></TR><TR><TD>8.125</TD><TD>8.125</TD><TD>0.0625</TD></TR><TR><TD>8.750</TD><TD>8.750</TD><TD>0.0625</TD></TR><TR><TD>9.375</TD><TD>9.375</TD><TD>0.0625</TD></TR><TR><TD>10</TD><TD>10</TD><TD>0.0625</TD></TR><TR><TD>10.625</TD><TD>10.625</TD><TD>0.0625</TD></TR><TR><TD>11.25</TD><TD>11.25</TD><TD>0.0625</TD></TR><TR><TD>11.875</TD><TD>11.875</TD><TD>0.0625</TD></TR><TR><TD>12.5</TD><TD>12.5</TD><TD>0.0625</TD></TR><TR><TD>13.125</TD><TD>13.125</TD><TD>0.0625</TD></TR></TBODY></TABLE></CENTER><CENTER><B>Table 8.</B></CENTER></UL></DIV><DIV class="geneswitch6"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI><B>Important dates
           2nd August- Received requested parts from iGEM HQ 
           5th August- Verified all parts from the DNA distribution kit </B></LI><LI>Restriction enzyme digest with EcorI-HF and SpeI-HF for ligation of CP2 into BBa_J61002 plasmid(backbone for CP1 and CP3) for rfp quantification as CP2 was originally in pSB1A2 vector. </LI><LI>Gel extraction of CP2 (insert) and CP1 (vector)
            Expected band size of 
            CP2 = 2056 bp and 58 bp 
            CP1 = 2925 bp and 58 bp 
            * CP2- extract smaller fragment 
            * CP1- extract larger fragment Ran samples on 1% agarose gel with 2-log ladder.  Restriction digest was successful only for CP1. CP1 was successfully extracted from the gel.

          <CENTER><B>Concentration of CP1 after gel extraction</B></CENTER></LI><LI>Q5 polymerase PCR for CP2 using protocol
            Forward primer = V2 forward from kit
            Reverse primer = VR reverse from kit
            Tm = 70C
           </LI><LI>Received parts from iGEM HQ in the form of bacteria on agar. All came in pSB1C3 (Cm resistance). Re-streaked them onto Cm plates, incubate overnight at 37°C to get single colonies next day.</LI><LI>Overnight ligation of CP2 (insert) + CP1 (vector) with T4 ligase using protocol</LI><LI>Transformation of ligated products in DH5α chemical competent cells</LI><LI>Colony PCR for CP1 and CP2 transformants

           </LI><LI>Expected colony PCR fragment size = 1142bp</LI><LI>Ran colony PCR products on 1% agarose gel to check if ligation was successful. No bands were seen indicating the ligation failed.</LI></UL><LI>Restriction digest to confirm all vectors plasmids that we need from BioBrick kit in 10 ɥL reaction (1 ɥL DNA).</LI><LI>Ran digested products on 1% agarose gel with a 2-log ladder to check band sizes </LI><LI>Ran CP2 PCR product on 4% agarose gel with 2-log DNA ladder and low molecular weight (MW) ladder to check if amplified region was correct</LI><B>Results:CP2 PCR was successful but ran out of sample stock. Inoculated CP2 PCR product in 10 mL LB broth with Carbenicillin 50</B></DIV><BUTTON type="button" id="week84" class="week7button">Week 7 (8<SUP>th</SUP> to 12<SUP>th</SUP> August)</BUTTON><DIV class="cellfree7"><H1 class="subnote">Cell-Free Mechanism</H1><UL class="list"><LI>Transformed pET28b (obtained from our supervisor Nick Weise) and pUC19 as a negative control into <I>Escherichia coli</I> DH5a cells. </LI><LI>Plated transformed cells onto LB agar plates.</LI><LI>Made overnight cultures with 50 µg/mL Kanamycin of cloned cells and miniprepped the pET28b vector (Table 1). </LI><DIV><B>Table 1.</B> Miniprep results measured with Nanodrop.</DIV><LI>Restriction digest of pET28b with NdeI and SalI for validation of the plasmid</LI><LI>This first digestion didn’t work due to short time of incubation at 37ºC.</LI><LI>Made more overnight cultures with 50 µg/mL Kanamycin of cloned cells and miniprepped pET28b (Table 2).</LI><DIV><B>Table 2.</B> Miniprep results using a Nanodrop .</DIV><LI>Restriction digest of pET28b with ClaI and EcoRI while waiting for new SalI from NEB.</LI><LI>Results:
          <CENTER><DIV><B>Figure 1.</B>1% TAE agarose gel showing restriction digest. Lane 1, pET28b cut with EcoRI, expected size 5368bp. Lane 2, pET28b cut with ClaI, expected size of 5368bp. Lane 3, pET28b cut with ClaI and EcoRI, expected sizes of 3924bp and 1444bp. Lane 4, pET28b negative control. </DIV></CENTER></LI></UL></DIV><DIV class="geneswitch7"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI><B>Important dates:12,<SUP>th</SUP> August- IDT genes arrived -- <I>alcR</I> and <I>alcA2</I></B></LI><LI>Restriction digest of <I>alcA 1</I> and <I>amilCP</I> to construct <I>alcA1</I> + <I>amilCP</I>. <I>alcA1</I> was digested with EcoRI + SpeI while <I>amilCP</I> was digested with EcoRI and XbaI. Control, pUC19 was digested with EcoRI and XbaI
Expected band sizes after digest with respective enzymes (highlighted band sizes to be gel extracted): 
Insert: alcA1 (22.4 ng/ɥL) - 866 + 2047 
Vector: amilCP (69.3 ng/ɥL) - 15 + 2742 
Control: pUC19 (48.6 ng/ɥL) - 27 + 2659
        </LI><CENTER><P style="text-align:center;">Restriction digest did not work</P></CENTER><LI>Inoculated CP1, CP2, CP3, <I>alcA1</I>, <I>amilCP</I>, <I>amilGFP</I>, <I>spisPink</I> in 10 mL LB broth with appropriate antibiotic</LI><LI>Restriction digest for plasmid verification by using different enzymes with standard 10X Cutsmart buffer</LI><P>Expected band sizes:</P><LI>Miniprep inoculated cells and measured the concentrations</LI><LI>Ethanol precipitation for CP2</LI><LI>PCR purification of CP1</LI><LI>Restriction enzyme digest to verify <I>alcA1</I> using EcoRI and EcoRV</LI><LI>Inoculated CP2_pSB1C3, CP1 and CP3 for rfp quantification</LI><LI>Miniprep inoculated samples</LI><LI>RFP quantification for CP1, CP2_pSB1C3, CP3 - Results were not comparable due to: different backbones (origin of replication) and different incubation time for cultures</LI><LI>Newly synthesised genes from IDT arrived -- <I>alcR</I> and <I>alcA2</I></LI></UL></DIV><BUTTON type="button" class="week8button">Week 8 (15<SUP>th</SUP> to 19<SUP>th</SUP> August)</BUTTON><DIV class="cellfree8"><H1 class="subnote">Cell-Free Mechanism</H1><UL class="list"><LI>AOx gene from IDT arrived! </LI><LI>Transformed pUCIDT_AOx into <I>Escherichia coli</I> DH5a cells.</LI><LI>Plated transformed cells onto LB agar plates with ampicillin. </LI><LI>Made overnight cultures of cells with 50 µg/mL ampicillin and miniprepped pUCIDT_AOx (Table 3).</LI><DIV><B>Table 3.</B> Miniprep results.</DIV><LI>Restriction digest of pUCIDT-AOx for characterisation and gel extraction.</LI><CENTER><DIV><B>Figure 2.</B> 1% LAB agarose gel showing digestion results. Lane 3, pUCIDT_AOx cut with NdeI and SalI, expected sizes of 2760bp and 2036bp.
        <LI>The digestion was not complete as a third band of 4796bp shows undigested plasmid. For this reason we could not proceed to do gel extraction as it would give a really small DNA concentration. </LI><LI>Make more overnight cultures of pET28b and miniprepped (Table 4).</LI><DIV><B>Table 4.</B> Miniprep results.</DIV></DIV><DIV class="geneswitch8"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>Restriction enzyme digest of CP1 and chromoproteins for chromoprotein quantification. Ran on gel for gel extraction.</LI><CENTER><P style="text-align: center;">Highlighted band sizes were extracted.</P></CENTER><LI>Inoculated pUC19 in 10 mL LB broth with Carbenicillin 50.</LI><LI>Re-suspended genes from IDT in 100 ɥL of sterile milli-Q for 1 hour. Transformed it into DH5α for each gene using chemical transformation protocol. We plated 20 ɥL and 200 ɥL of each sample respectively onto Carbenicilin plates. pUC19 was transformed as a control.  

Results of re-suspension of IDT genes:
        </LI><CENTER><P style="text-align: center;">Highlighted band sizes were extracted.</P></CENTER><LI>Gel extraction of digested CP1 and <I>amilCP</I>, <I>amilGFP</I> and <I>spisPink</I></LI><CENTER><P style="text-align:center;">Repeated digest as the DNA concentration was not satisfactory</P></CENTER><LI>-Inoculated CP1, <I>amilCP</I>, <I>amilGFP</I> and <I>spisPink</I> in 10 mL LB broth with its respective antibiotic to obtain more DNA</LI><LI>Transformation of genes from IDT worked. Picked single colonies and inoculated them 10 mL LB broth with the respective antibiotic.</LI><LI>Performed miniprep for the overnight cultures. Results:</LI><CENTER><P style="text-align:center;">Digested products were run on a 1% gel at 120V for 35 minutes.  <I>amilGFP</I> and <I>spisPink</I> digest did not work. CP1 and <I>amilCP</I>worked so proceeded to perform gel extraction to obtain the necessary band. </P></CENTER><CENTER><P style="text-align:center;">*all the cut bands were loaded into 1 tube</P></CENTER><LI>Redid the digest for <I>amilGFP</I> and <I>spispink</I> and control plasmid(given by Marc) using XbaI and PstI-HF. Ran on 1% gel and it worked so proceeded to gel extract. Ligation with CP1.
Concentrations of samples used:<I>amilGFP</I> - 124.4 ng/ɥL<I>spispink</I> - 136.5 ng/ɥL
control plasmid - 500 ng/ɥL<CENTER><P style="text-align: center;">*the three bands for each sample that was cut was evenly distributed into 2 tubes</P></CENTER></LI><LI>Chemical transformation of control plasmid and pUC19 (control vector) into DH5α competent cells to get more DNA. Plated onto Carbenicillin plates</LI><LI>Inoculated <I>alcA2</I>, <I>alcR</I> and pUC19. Miniprep results:</LI><LI>Made glycerol stocks of <I>alcA2</I> and <I>alcR</I> using 400 ɥL of 50% sterile glycerol + 600 ɥL inoculated culture and stored in -80°C for future use.</LI><LI>Ligated CP1 with chromoproteins and transformed into DH5α. Transformants were plated onto Carbenicillin plates and left to grow overnight in the 37°C incubator.</LI><LI>Inoculated control plasmid to make glycerol stocks.</LI><LI>Plasmid verification of alcR and alcA 2</LI></UL></DIV><BUTTON type="button" class="week9button">Week 9 (22<SUP>nd</SUP> to 26<SUP>th</SUP> August)</BUTTON><DIV class="cellfree9"><H1 class="subnote">Cell-Free Mechanism</H1><UL class="list"><LI>Restriction digest of pUCIDT_AOx for gel extraction with NdeI and SalI, and EcoRI and PstI for validation.</LI><LI>Results:
        <CENTER><DIV><B>Figure 3.</B> 1 % TAE agarose gel showing digestion results. Lane 2, pUCIDT_AOx  cut with NdeI and SalI, expected sizes of 2781bp and 2015bp. Lanes 3 and 6, pUCIDT_AOx negative control. Lane 4, pUCIDT_AOx cut with EcoRI, expected size of 4795bp. Lane 5, pUCIDT_AOx cut with PstI, expected size of 4795bp.</DIV></CENTER></LI><LI>Gel extraction of AOx 2017bp band in digested gel. </LI><LI>Restriction digest of pET28b with NdeI and SalI for PCR purification, as the fragment is &lt;100bp.</LI><CENTER><B>Figure 4.</B> Lane 2, pET28b cut with NdeI and SalI, expected sizes of around 5310bp. Lane 3, negative control.
          </CENTER><LI>PCR purification of pET28b.</LI><LI>Nanodrop results of cut AOX and PCR purified Pet28b (Table 5)</LI><DIV><B>Table 5.</B> Miniprep results using Nanodrop.</DIV><LI>Performed ligation of digested and purified AOX and Pet28b.</LI><DIV><B>Table 6.</B> Ligation of pET28b and AOx.</DIV><LI>Transformed ligated Pet28b_AOX and pUC19 as a negative control into DH5α cells for each ratio and plated onto LB agar plates with  50 μg/ml of Kanamycin.</LI><LI>Made O/N cultures of 4 colonies.</LI><LI>Prepared glycerol stocks of pET28b_AOx.</LI><LI>Miniprepped pET28b_AOx from O/N cultures.</LI><DIV><B>Table 7.</B> Miniprep results using Nanodrop.</DIV><LI>Restriction digest with NdeI and SalI to screen recombinant clones
        </LI><B>Figure 4.</B>S1 to S4, pET28b_AOx cut with NdeI and SalI, expected sizes of 5310bp and 2017bp. None of the samples showed expected sizes, suggesting ligation didn't work.
 

      </UL></DIV><DIV class="geneswitch9"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>Restriction digest &amp; gel extraction for ligationSet A = CP1/2/3 (vector, SpeI/PstI-HF) + alcR (insert, XbaI/PstI-HF/ZraI)
         Expected band size (bp):
         </LI><LI>CP1 = 887 + 2096</LI><LI>CP2 = 18 + 2096</LI><LI>CP3 = 887 + 2096</LI><LI>alcR = 565 + 2204 + 2639</LI></UL>
         Set B = alcA1/2 (insert, EcoRI-HF/SpeI) + ChP (vector, EcoRI-HF/XbaI)
         Expected band size (bp):
         <UL class="hyphen"><LI>alcA1 = 886 + 2047</LI><LI>alcA2 = 785 + 2772</LI><LI>amilCP = 15 + 2742</LI><LI>amilGFP = 15 + 2772</LI><LI>spispink = 15 + 2752</LI></UL>
         Set C =
(a) RFP (from CP1, insert, SpeI/PstI-HF) + CP2 (pSB1C3, vector, SpeI/PstI-HF)
(b) CP1 (vector, EcoRI-HF/PstI-HF) + ligated set C (a) (EcoRI-HF/PstI-HF)
Final construct = CP2+RFP in BBa backbone
Set D = Positive controls - control plasmid, puc19 vectors
Expected band size (bp):
control plasmid
         <UL class="hyphen"><LI>SpeI/PstI-HF = 89 + 1197 + 3761 </LI><LI>XbaI/PstI-HF = 16 + 2622 + 2409 </LI><LI>EcoRI-HF/SpeI = 89 + 249 + 4709</LI><LI>EcoRI-HF/XbaI = 932 + 2638 + 1477</LI></UL>
Prepared 3 vials of 25 ɥL for each sample
Set A = CP1/2/3 (vector, SpeI/PstI-HF) + alcR (insert, XbaI/PstI-HF/ZraI)
DNA concentration:
         <UL class="hyphen"><LI>CP1 = (a) 83.6ng/ɥL, (b) 323.6ng/ɥL </LI><LI>CP2 = (a) 103.9ng/ɥL, (b) 32.3ng/ɥL, (c) 22.3ng/ɥL </LI><LI> CP3 = 195.4ng/ɥL</LI><LI>alcR = 390.3ng/ɥL</LI></UL>
Per vial
        
Set B = alcA1/2 (insert, EcoRI-HF/SpeI) + ChP (vector, EcoRI-HF/XbaI)
DNA concentration:
         <UL class="hyphen"><LI>alcA 1 = 84.9ng/ɥL</LI><LI>alcA 2 = 743.3ng/ɥL </LI><LI>amilCP = 181.1ng/ɥL</LI><LI>amilGFP = 53.2ng/ɥL</LI><LI>spispink = 230.8ng/ɥL</LI></UL>
Set C (b) = CP1 (vector, EcoRI-HF/PstI-HF) + ligated set C (a) (EcoRI-HF/PstI-HF)
Set D = Positive controls - control plasmid, puc19 vectors
Prepared a 10ɥL reaction rather than a 25ɥL
DNA concentration:
         <UL class="hyphen"><LI>control plasmid = 124.6ng/ɥL</LI><LI>SpeI/PstI-HF</LI><LI>XbaI/PstI-HF</LI><LI>EcoRI-HF/SpeI</LI><LI>EcoRI-HF/XbaI</LI><LI>puc19 = 417.5ng/ɥL</LI><LI>EcoRI-HF/ZraI</LI><LI>PstI-HF/ZraI</LI></UL><LI>Inoculated CP1, CP2, BL21 cells</LI><LI>Restriction enzyme digest for alcA1, alcA2, alcR, ChP
Concentrations:
alcA 1 - 84.9 ng/ɥL
alcA 2 - 743.3 ng/ɥL
control plasmid - 124.6 ng/ɥL
Expected band size (bp):
         </LI><LI>alcA1 = 886 + 2047</LI><LI>alcA2 = 785 + 2772</LI><LI>amilCP = 15 + 2742</LI><LI>amilGFP = 15 + 2772</LI><LI>spispink = 15 + 2752</LI><LI>alcR = 565 + 2204 + 2639</LI>
        Concentrations:
        <UL class="hyphen"><LI>amilCP - 181.1 ng/ɥL</LI><LI>amilGFP - 53.2 ng/ɥL</LI><LI>spispink - 230.8 ng/ɥL</LI><LI>control plasmid - 124.6 ng/ɥL</LI></UL>
Concentrations:
       <UL class="hyphen"><LI>alcR - 390.3 ng/ɥL</LI><LI>pUC19 - 250.0 ng/ɥL</LI></UL><CENTER><P style="" text-align:center="">Result: Bands were distorted severely, most likely due to not properly dissolved agarose powder.</P></CENTER><CENTER><P style="" text-align:center="">Result: Only one band was seen instead of two.  May have missed out adding any of the reagents/problems with the DNA.  Should change a new aliquot of positive control DNA from this point onwards.</P></CENTER><LI>Gel extraction of alcA1, alcA2, alcR and digested products 
         <CENTER><P style="text-align:center;">*CP2 to be PCR purified,/p&gt;
        </P></CENTER></LI><LI>PCR purification of amilCP, amilGFP, spispink
         </LI>Re-did overnight ligation of CP1 with chromoproteins (ChP)
        
        Concentrations:
        <UL class="hyphen"><LI>amilCP - 20.2 ng/ɥL</LI><LI>amilGFP - 10.4 ng/ɥL</LI><LI>spispink - 10.7 ng/ɥL/li&gt;

        </LI></UL>
1:7 ligation DNA mass:
        <UL class="hyphen"><LI>amilCP - 59.53 ng</LI><LI>amilGFP - 62.03 ng</LI><LI>spispink - 60.36 ng/li&gt;

        </LI></UL><LI>Ligation of alcA 2 with amilCP
        </LI><LI>Re-did alcA1/alcA2/alcR digest with control plasmid and gel extracted appropriate sizes.
        <CENTER><P style="text-align:center;">*all samples were digested at 37ç for a Total of 2 hours. For alcR &amp; control plasmid, X/P was added first for 1 hour before adding ZraI and left to digest for another hour. 
*All samples were run on a 1% gel. </P></CENTER>
alcA 1: 886 + 2047
alcA 2: 785 + 2772 
alcR : 565 + 2204 + 2639<CENTER><P style="text-align:center;">Result: The correct band sizes were extracted for alcR and gel extraction was performed as per protocol. For alcA2, the wrong band size was extracted so digest would be repeated. alcA1 did not work. </P></CENTER>
After soaking in ethidium bromide, alcR bands were clearly seen and extracted</LI><LI>Transformed overnight ligated samples (CP 1 + chromoproteins (Carbenicillin), alcA2 + amilCP (CmR) ) into DH5alpha along with a pUC19 (Carbenicillin) positive control using protocol. They were plated into the appropriate Ab plates and SOC medium was used as a negative control.</LI><LI>Ligation of CP1 + alcR and CP2 + alcR using Roche ligation kit and transformed into DH5α. Plated onto Carbenicillin plates, left in 37°C incubator overnight.
            Insert: CP1, CP2Vector: alcR (482.1ng/ɥL)</LI><LI>Repeated alcA 1 &amp; alc A 2 digest using same measurements as before and used control plasmid. alcA 1 did not work again. alcA 2 worked and the correct band size (785) was extracted. Gel extraction protocol was used to obtain the DNA.
        </LI><LI>CP1 + amilCP ligation product restriction enzyme digest to verify the plasmid. pUC19 was used as a positive control.
        </LI><LI>Sent alcA 1 for sequencing due to repeated failure in restriction enzyme digest. sequencing results of alcA 1indicate that alcA1 did not contain XbaI / SpeI site.
        </LI><LI>Inoculated cultures and miniprep results
        </LI><LI>Validation of miniprep ligated product- alcA 2 + amilCP by restriction enzyme digest and sent for sequencing using VF2 primer
Expected band sizes:
control plasmid
        </LI><LI>PstI-HF/ZraI: 2867 + 2180</LI><LI>XhoI/PstI-HF: 2820 + 2040 + 187</LI>
alcA 2 + amilCP 
        <UL class="hyphen"><LI>PstI-HF/ZraI: 1921 + 1475 + 131</LI><LI>XhoI/PstI-HF: 1622 + 1013 + 892</LI></UL><LI>RE digest of all chromoproteins, constitutive promoters, alcA1, alcA2, alcR and control plasmid
        </LI></DIV><BUTTON type="button" class="week10button">Week 10 (29<SUP>th</SUP> August to 2<SUP>nd</SUP> September)</BUTTON><DIV class="cellfree10"><H1 class="subnote">Cell-Free Mechanism</H1><UL class="list"><LI>Colony PCR with T4 Taq polymerase of 16 colonies from last week’s ligation plates.</LI><DIV><B>Table 8.</B> Primers used for Colony PCR.</DIV><LI>Results:
          <CENTER><DIV><B>Figure 6.</B> 1% TAE gel showing colony PCR of pET28b_AOx, sample 3 and 13 show expected amplified sizes of 2319bp.</DIV></CENTER></LI><LI>Made O/N cultures of sample 3 in colony PCR. </LI><LI>Sample 3 in lane 4 shows a band of around 2300bp. This suggests AOX is present in the sample and thus possibly ligated. The other bands of 350 bp indicates a vector that has re-ligated to itself.</LI><LI>Made O/N of sample 3 </LI><LI>Miniprepped pET28b_AOx from O/N cultures.</LI><LI>Digestion of pET28b_AOx with Xbal and BamHI was unsuccessful (No picture available).</LI></UL></DIV><DIV class="geneswitch10"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI><B>Important dates:29<SUP>th</SUP> August-First BioBrick ready</B></LI><LI>Restriction enzyme digest
        <CENTER><P style="text-align: center;">*ZraI was added to both samples after 1 hour of incubation</P>&quot;
        </CENTER>&gt;br /&gt;
        </LI><LI>PCR purification - CP2, amilCP, amilGFP, spispink
        </LI><LI>alcA2 + amilCP sequencing results with VF2 primer → alcA2 + amilCP ligation was successful (FIRST BIOBRICK READY!)
        </LI><LI>Received alcA 1 forward and reverse primers from IDT. Added appropriate amounts of milli-Q as instructed by IDT to make a stock concentration of 100 ɥM. Did a dilution to make aliquots of 10 ɥM of primers to use for PCR.


        </LI><LI>Q5 PCR to insert XbaI/SpeI in alcA1. Also used an a negative control (empty vector from previous day’s ligation)  → failed as too much dNTP was used resulting in Mg2+ depletion
alcA1 - 349.6 ng/ɥL
        
 
        PCR reaction conditions:

        
Restriction Enzyme digest → Gel extraction →  overnight ligation of:
Insert CP3 (S/P) + Vector amilCP (X/P)
Insert CP3 (S/P) + Vector amilGFP (X/P)
Insert CP3 (S/P) + Vector spispink (X/P)
Insert CP3 (S/P) + Vector alcR (X/P/Z)

Controls: 
control plasmid (S/P) &amp; control plasmid (X/P/Z)
Concentrations:
</LI><LI>CP3 = 526.9 ng/µL</LI><LI>amilCP = 146.2 ng/µL</LI><LI>amilGFP = 273.5 ng/µL</LI><LI>spispink = 235.3 ng/µL</LI><LI>alcR = 855. 0 ng/µL</LI><LI>control plasmid= 1074.4 ng/µL</LI></UL><CENTER><P style="text-align:center">*for alcR, ZraI was added after 1 hour. Total digest time = 2 hours in a 37°C water bath.</P></CENTER><LI>Overnight ligation in 4°C cold room
        </LI><LI>noculated cultures of ligated products from 30th Aug using colonies from plate in Carbenicillin + LB. Miniprep results:
        </LI><LI>CloneAmp PCR to insert XbaI/SpeI in alcA1 as Q5 PCR has been failing.
        alcA1 - 366.0 ng/ɥL
        </LI><LI>Ran CloneAmp products (alcA1 + XbaI/SpeI)  on a 1 % gel → PCR was successful so proceeded to PCR purify the product
        </LI><LI>Restriction digest of miniprep ligation product in to verify plasmid
        </LI><LI>Restriction digest of PCR purified alcA1+ XbaI/SpeI with EcoRI-HF and SpeI-HF → Gel extract (866bp band) for ligation with spispink  (E/X)
        
Gel extraction was not successful. Repeat the PCR 
        <CENTER><P style="text-align:center">* bottom half of the gel is supposed to be verification of ligated products but there was something wrong with the gel and hence nothing could be seen. Re-do digest on Monday to verify the ligated product.</P></CENTER></LI><LI>Heat inactivation of overnight ligation product at 65C for 10 minutes. They were then transformed into DH5α competent cells and plated on Carbenicillin and left on bench over the weekend. 
        </LI>Transformed CP1 + alcR and CP2 + alcR into BL21 competent cells. Plated onto Carbenicillin and left on bench over the weekend.</DIV><BUTTON type="button" class="week11button">Week 11 (5<SUP>th</SUP> to 9<SUP>th</SUP> September)</BUTTON><DIV class="cellfree11"><H1 class="subnote">Cell-Free Mechanism</H1><UL class="list"><LI>More colonies were screened through colony PCR from ligation plates. </LI><LI>Results:
        <CENTER><DIV><B>Figure 7.</B>1% TAE agarose gel showing colony PCR of pET28b_AOx. S7 and S15 showed correct amplified size of 2044bp.</DIV></CENTER></LI><LI>Made overnight cultures of colonies 7 and 15 in Kanamycin and miniprep. </LI><LI>Restriction digest with NdeI and SalI of both samples.</LI><LI>Result from Sample 7:
        <CENTER><DIV><B>Figure 8.</B>1% TAE agarose gel showing digestion results for sample 7. Lanes 2 and 4, pET28b_AOx cut with NdeI and SalI, expected sizes of 5310bp and 2017bp. Lane 3 and 5, negative control.
        </DIV></CENTER></LI><LI>Result from Sample 15:
        <CENTER><DIV><B>Figure 9.</B>1% TAE agarose gel showing digestion results for sample 15. Lanes 2 and 3, pET28b_AOx cut with NdeI and SalI, expected sizes of 5310bp and 2017bp. Lane 4 and 5, negative control.
        </DIV></CENTER></LI><LI>Transformed samples and pUC19 into <I>Escherichia coli</I> DH5α strain. pUC19 was used as a control.</LI><LI>Plated transformed cells onto LB agar plates with 50 µg/mL Kanamycin. </LI></UL></DIV><DIV class="geneswitch11"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>Q5 PCR to insert XbaI/SpeI to alcA1 →Run on 1% gel →  PCR purify → restriction enzyme digest with EcoRI/SpeI → gel extraction → ligation with spispink (E/X) overnight
alcA1 - 366.0 ng/µL, amplified fragment side = 886 bp

        </LI><LI>PCR purification of alcA1 using protocol 
        </LI><LI>Restriction enzyme digest of PCR purified alcA1  with EcoRI-HF and SpeI. Digested at 37°C in PCR machine for 1 hour.
Band expected: alcA1 = 886bp, contorl plasmid= 4709 + 89 + 249
 
        <CENTER><P style="text-align: center;">Digest failed so repeated it again. Repated digest worked, so proceeded to gel extract the fragment.</P></CENTER></LI><LI>Gel extraction of alcA1 digested with E/S using protocol. 
        </LI><LI>	overnight ligation of alcA1 (E/S) with spispink (E/X) in 4°C (cold room)
Insert: alcA1 (E/S) - 886 bp, 92.4 ng/ɥL
Vector: spispink (E/X) - 2752 bp, 37.8 ng/ɥL

        </LI><LI>Restriction digest to verify ligated products  - CP1 + alCR, CP2 + alcR, CP2 + rfp
        <CENTER><P style="">Results: No bands were seen except for control plasmid. Re-made overnight cultures to miniprep and validate again.</P></CENTER></LI><LI>New ligation of CP + alcR and CP + Chromoproteins. Ligation was done using Roche kit for 30 mins on ice. This time transformed into Marc’s competent cells (to eliminate the possibility of contaminated competent cells) and plated onto Carbenicillin plates, left overnight. 
Ratio - 1:1

        
Ratio - 1:3
        </LI><LI> Colony PCR (with VF2 and VR primers)
Mastermix:

        
Made overnight cultures of the correct samples
        </LI><LI>Glycerol stock and miniprep overnight cultures - alcA1 + spispink
        </LI><LI>Digest alcA1 + spispink with EcoRV-HF and SphI-HF to verify plasmid. alcR used as control 
         
Sent alcA1 + XbaI/SpeI that was done using PCR for sequencing

        </LI><LI>RE digest for CP, alcR and ChP for ligation

Samples:
Eg: Vector + insert</LI><LI>CP1 (S/P) 2096bp + alcR (X/P/Z) 2639bp</LI><LI>CP2 (S/P) 2096bp + alcR (X/P/Z) 2639bp</LI><LI>CP3 (S/P) 2096bp + alcR (X/P/Z) 2639bp</LI><LI>CP1 (S/P) 2096bp + amilCP (X/P) 713bp</LI><LI>CP2 (S/P) 2096bp + amilGFP (X/P) 743bp</LI><LI>CP3 (S/P) 2096bp + spispink (X/P) 723bp</LI></UL><LI>Ligation
Conc. of alcR used = 31.4 ng/ɥL. Concentrations of the other DNA used are in G) below. 
Ratio 1:1

        
Ratio 1:3

        
Ratio 1:5
        
Controls
        </LI><LI>Gel extraction- Nanodrop concentration
        </LI></DIV><BUTTON type="button" class="week12button">Week 12 (12<SUP>th</SUP> to 16<SUP>th</SUP> September)</BUTTON><DIV class="cellfree12"><H1 class="subnote">Cell-Free Mechanism</H1><UL class="list"><LI>Sent sample 15 for sequencing.</LI><LI>Made overnight cultures of plated samples 7 and 15 pET28b_AOx from last week. </LI><LI>Transformed samples 7 and 15 of pET28b_AOx into <I>Escherichia coli</I> DH5α and BL21 strains and plate them onto LB agar plates.</LI><LI>Growth of cultures in 250 ul flasks, containing LB broth, of pET28b_AOx in <I>Escherichia coli</I> BL21 strain, empty vector pET28b and pBbESK_RFP (controls). Grown until reaching an O.D of about 0.5-0.7. </LI><LI>Induction of pET28b_AOx in BL21 E.coli cells with 0.5 mM IPTG. Controls were also IPTG induced: Empty vector pET28b was used as a negative control. pBBESK_RFP was used as a positive control. </LI><LI>Stored cultures in -20ºC overnight. </LI><LI>Simultaneous to the above work, we started working on assembling our part to the pSB1C3 backbone: </LI><LI>Restriction digest of pUCIDT_AOx with EcoRI and PstI for gel extraction.</LI><LI>Results:
        <CENTER><DIV><B>Figure 10.</B> 0.8% TAE agarose gel showing digestion results. Lane 2, pUCIDT_AOx cut with EcoRI and PstI, expected sizes of 2760 bp and 2036 bp. Picture made after gel extraction of 2036 bp band. 
        </DIV></CENTER></LI><LI>Gel extraction of band at 2036 bp corresponding to AOx.</LI><LI>Restriction digest of pSB1C3 from iGEM with EcoRI and PstI.</LI><LI>Ligation of pSB1C3 and AOx using Roche Kit.</LI><DIV><B>Table 9.</B> Ligation of pSB1C3 and AOx.</DIV><LI>Transformation of ligation of pSB1C3_AOx and plated onto LB agar plates with chloramphenicol.</LI><LI>Colony PCR of four colonies.</LI><LI>Results:
        <CENTER><P style="text-align:center"><B>Figure 11.</B>Colony PCR of pSB1C3_AOx. S1 and S4 show correct amplified size of around 2000bp.</P></CENTER></LI><LI>Restriction digest of pSB1C3_AOX with BamHI and PstI for characterisation.</LI><LI>Results:
        <CENTER><DIV><B>Figure 12.</B> 1% agarose gel showing digestion results. Lane 3 shows pSB1C3_AOx cut with BamHI and PstI showing correct sizes of 3355bp and 710bp.</DIV></CENTER></LI></UL></DIV><DIV class="geneswitch12"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>Colony PCR
Mastermix:
        </LI><LI>Miniprep CP1 + alcR (1:3)
        </LI><LI>RE digest to confirm CP1 + alcR (1:3) ligation was successful<P>Did not perform as the DNA concentration was very low</P></LI><LI>Inoculated ligated products. Miniprep results:
        </LI><LI>Ligation of CP1 and ChP
        1:3
        
        1:5
        
        Controls
        
Transformed ligated products
          </LI><LI>RE digest for verification
        </LI><LI>RE digest for ligation
        <CENTER><P style="text-align:center">Something was wrong with the gel. Will re-run gel the next day. However, we loaded all of the control plasmid  so there will be none for the re-run.</P></CENTER></LI><LI>Made DH5α competent cells </LI><LI>Ran gel for digested products
        <CENTER><P style="text-align:center">Both fragments of CP3 were gel extracted. Smaller fragment: 887 bp = rfp  
</P></CENTER><CENTER><P style="text-align:center">Bigger fragment: 2096 bp = CP3 vector  Since the CP2 S/P/A control shows that the digest has worked, CP2 was PCR purified to lose the 18bp fragment and retain the CP2 vector. 
</P></CENTER></LI><LI>PCR purification of CP2 (pSB1C3) S/P
        </LI><LI>Gel extraction of CP3 fragments (vector and rfp)
         </LI><LI>Before performing the CP2 + rfp ligation, we ran the PCR purified vector and gel extracted rfp on a gel just to confirm that the DNA is present.
        <CENTER><P style="text-align:center">The correct band sizes are seen. Proceeded to perform CP2 + rfp ligation.</P></CENTER></LI><LI>alcR restriction enzyme digest with XbaI/PstI-HF. No ZraI could be found, so we ran a 0.6% gel for a really long time hoping to be able to get the separation of the 2 bands.
        <CENTER><P style="text-align:center">Separation of bands was not seen. Repeated digest with PvuI obtained from another lab.</P></CENTER></LI><LI>CP2 (vector) + rfp (insert)  ligation
        <CENTER><P style="text-align:center">The ligation was left in the PCR machine at 24°C for 2 hours. It was then heat inactivated at 65C for 10 minutes and then transformed into DH5alpha. They were plated onto Carbenicillin plates. </P></CENTER></LI><LI>Tested our new DH5alpha competent cells and BL21 cells by transforming them with pUC19 and plated onto Carbenicillin. Left overnight at 37°C.</LI><LI>Ran gel for alcR digest from 14th Sept (F). We did not have SybrSafe, so left the gel in Ethidium Bromide for half an hour before visualising gel.
        <CENTER><P style="text-align:center">Proceeded to gel extract the appropriate alcR fragment.</P></CENTER></LI><LI>Gel extraction of alcR
        </LI><LI>Restriction digest of alcA1, alcA2 and alcR to put into pSB1C3 submission vector.
        <CENTER><P style="text-align:center;">Samples were left in PCR machine at 37°C for 1.5 hours. ZraI was added after 45 minutes.</P></CENTER></LI><LI>Gel extraction of fragments from digest
        </LI><LI>Submission vector ligation using fragments from D and pSB1C3 vector that has been digested with EcoRI-HF and PstI-HF by Mechanism 1 team. 
Concentration of pSB1C3 = 11.1 ng/ɥL

        <CENTER><P style="text-align:center">Samples were left on the bench for 1 hour before transforming into Dh5α and plated onto Cm plates. Left in 37°C incubator.</P></CENTER></LI><LI>Restriction digest of old alcA2 + amilCP product for verification
        

Samples were left in the PCR machine at 37°C for 1.5 hours. It was then run on a 1% gel.

        </LI><LI>Ligation : CP3 + chromoproteins &amp; CP + alcR
        
Controls
        </LI><LI>Transformed ligated product onto appropriate plates </LI><LI>Made 5ml overnight culture for rfp quantification for CP1 and CP3 in the Bba_J61002 vector that contains rfp. Grown in 5ml LB + CmR (50 ɥg/ml). Also grew pBbB2K (kanamycin, ATC inducible) Kiesling vector as a control with rfp.</LI><LI>alcA1+spispink restriction enzyme digest to verify the plasmid

        <CENTER><P style="text-align:center;">alcA1 + spispink is not the correct product.</P></CENTER></LI><LI>spispink restriction digest to obtain pSB1C3 vector.
        <CENTER><P style="text-align:center;">The larger fragment was extracted and proceeded to purify as per protocol.</P></CENTER></LI><LI>Colony PCR 
Mastermix:
        </LI><LI>RFP quantification for CP1 and CP3 using the overnight culture.A kiesling vector - pBb2K (Kan) was used as a control that can be induced by aTC. 50nM of aTC was added to the induced wells during mid-way log growth. RFP was measured for 24 hours. 
Stock concentration of aTC = 10mM
Concentration needed to add to 200ɥL cells in plate = 50nM

A 1:1000 dilution of the stock concentration with 100% ethanol was made and a final concentration of 1mM was obtained. 1ɥL of aTC was added to each well. Important to note that aTC is light sensitive and hence the tube has to be wrapped in aluminium foil. OD 600 and RFP were measured every 30 mins for 24 hours. The cells were shaken at 500 rpm and at 37 °C.
</LI></UL></DIV><BUTTON type="button" class="week13button">Week 13 (19<SUP>th</SUP> to 23<SUP>rd</SUP> September)</BUTTON><DIV class="cellfree13"><H1 class="subnote">Cell-Free Mechanism</H1><UL class="list"><LI>Made soluble and insoluble fraction proteins of induced cultures. </LI><LI>Performed a 12% SDS-PAGE gel of soluble and insoluble fraction of protein samples. Bands in the gel were so faint we decided to repeat it.</LI><LI>Repeat 12% SDS-PAGE gel with optimized volume of proteins.</LI><DIV><B>Table 10.</B> 12% SDS-PAGE gel with optimized volumes of protein.</DIV><LI>Results:
        <CENTER><DIV><B>Figure 12.</B> SDS-PAGE for protein expression. Lane 1: Soluble IPTG induced pET28b-AOx in BL21 cells. Lane 2: Soluble un-induced pET28b-AOx in BL21 cells. Lane 3: Soluble IPTG induced empty pET28b vector in DH5α cells. Lane 4; Soluble un-induced empty pET28b vector in DH5α cells. Lane 5: Soluble IPTG induced pBbESK_RFP vector showing RFP (27 kDa). Lane 6: Protein Standard. Lane 7: Insoluble IPTG induced pET28b-AOx in BL21 cells. Lane 8: Insoluble un-induced pET28b-AOx in BL21 cells. Lane 9: Insoluble IPTG induced empty pET28b vector in DH5α cells. Lane 10: Insoluble un-induced empty pET28b vector in DH5α cells. Lane 11; Insoluble IPTG induced pBbESK_RFP.</DIV></CENTER></LI><LI>Performed a Western Blot of transferred SDS-PAGE gel to membrane binding anti-His antibodies (sourced from Sigma-Aldrich). </LI><LI>Results:
        <CENTER><DIV><B>Figure 13. </B>Western Blot results. Only the insoluble fraction of pET28b_AOx IPTG induced is visible (lane 7), showing two major regions at around 80 kDa and 35 kDa.
        </DIV></CENTER></LI><LI> Sent pSB1C3_AOx for sequencing to check that that AOx had been correctly ligated into the submission vector.</LI></UL></DIV><DIV class="geneswitch13"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>Miniprep results for ligated products
        </LI><LI>RE digest for verification of ligated products
        </LI><LI>Inoculated CP1, CP3, CP3 + amilCP, CP3 + spispink, Kiesling vector</LI><LI>overnight ligation into pSB1C3- alcA 1, alcR, alcA 2
Vector: pSB1C3 - 31.6 ng/ɥL
Insert(all cut with E/P): alcA 1 + X/S - 46.4 ng/ɥL, alcR - 42.2 ng/ɥL, alcA 2 - 29.5 ng/ɥL
1:1
        
1:3
        
Controls
        </LI><LI>RE digest to re-verify CP3 + alcR
        <CENTER><P style="text-align:center;">Lane order: Ladder, A(P/B), B(P/B), C(P/B), control plasmid (P/B), A(E/P), B(E/P), C(E/P), amilCP(E/P), amilCP(E/P), Ladder</P></CENTER></LI><LI>Protein expression overnight culture- 5 mL culture, CARB 100, 30℃ incubator. 2 replicates of CP3+alcR, CP3+rfp and BL21 were grown. </LI><LI>Chromoprotein comparison - the same amount of cells were added onto a plate and the time taken for a visible colour change was seen.
        </LI><LI>Made overnight culture for rfp quantification - CP1, CP3 and Kiesling vector pBbB2K</LI><LI>Submission vector ligation -  alcA2, alcA1 and alcR into pSB1C3. Ligation was done in the PCR machine at 24°C for 2 hours. Heat inactivation at 65C for 10 minutes then transformed into DH5alpha and plated onto Carbenicillin. Plates were left at 37°C, overnight.
1:1
        
1:3
        
Controls
        </LI><LI>Restriction digest to obtain pSB1C3 backbone using spispink. Digest was run on a PCR machine at 37°C for 1.5 hours and then viewed on a 1% gel . since the bands were correct, we proceeded to gel extract the larger fragment (pSB1C3 backbone).
        </LI><LI>Protein expression 
Made a 1:1000 dilution of the overnight cultures in a Total volume of 25ml (in a 250 mL Flask) = 24.75 mL LB + 0.25mL culture *sterile conditions, no antibiotic
The culture was shaken at 37°C and the OD was measured at several timepoints. Some of the cultures needed dilutions to be made to enable OD 600 measurements to be taken.

        

At each OD  reading, 500ɥL of the culture was taken and placed into a 2mL tube. It was spun down at 12 000rpm, 4°C for 10 minutes. The supernatant was discarded and pellet was kept. 50 ɥL of 2X SDS-PAGE dye was used to resuspend the pellet and transferred into a PCR tube. The samples were boiled at 99°C in the PCR machine for 20 minutes. All the samples were kept in a 4°C fridge until ready to be loaded onto the SDS gel. 
SDS gel was run at 300V for 20 minutes. 
The remaining culture was transferred to a falcon tube and spun down at 10000 rpm , 4°C for 20 minutes. Supernatant was discarded and the pellet was stored in the -20°C for future use.

        </LI><LI>CP rfp quantification 
RFP quantification for CP1 and CP3 using the overnight culture.A kiesling vector - pBb2K (Kan) was used as a control that can be induced by aTC. 1ɥL of 1mM  aTC was added to the induced wells during mid-way log growth. RFP was measured for 24 hours. OD 600 and RFP were measured every 30 mins for 24 hours. The cells were shaken at 500 rpm and at 37 C.</LI><LI>SDS-PAGE did not show presence of protein. Repeated SDS-PAGE but this time using stored pelleted cells. Cells were first separated into soluble and insoluble fractions.
Pellets were first resuspended in 500ɥL Bugbuster (must be at room temperature) and transferred to a 2ml tube. They were then shaken on a rocker for 20mins before being spun down at 12 000 rpm at 4°C for 20 mins. Supernatant which contains the soluble fraction was transferred to a fresh 2ml tube.
Pellets containing the insoluble fraction were resuspended in 200ɥL Buffer SB1. In a PCR tube, 0.3ɥL of the resuspended pellet + 0.5ɥL DTT + 5ɥL 2X dye + 4.2ɥL milli-Q were added. They were then boiled on the PCR machine at 98°C for 7 minutes. Supernatant containing the soluble fraction : In a PCR tube, 2.5ɥL of the supernatant + 0.5ɥL DTT + 5ɥL 2X dye + 2ɥL milli-Q were added. They were then boiled on the PCR machine at 98°C for 7 minutes. Both the pellet and supernatant fraction samples were then loaded onto a gel and run at 300V for 25-30minutes (4 gels were running at the same time. If only 2 gels, 20 minutes is enough). alcR size = 97.7kDa, hence a pre-made 10% gel was used. </LI><LI>Rfp quantification was repeated. 
RFP quantification for CP1 and CP3 using the overnight culture.A kiesling vector - pBb2K (Kan) was used as a control that can be induced by aTC. 1ɥL of 1mM  aTC was added to the induced wells during mid-way log growth. RFP was measured for 24 hours. OD 600 and RFP were measured every 30 mins for 24 hours. The cells were shaken at 500 rpm and at 37 C. 
        </LI><LI>colony PCR to check pSB1C3 + alcR/alcA1/alcA2 transformed and plated on 21st August. 
Mastermix:
        </LI><LI>Co-transformation of CP3+alcR and alcA2+amilCP 
        </LI><LI>Ran a 1% gel for colony PCR
        </LI><LI>Co-transformed colonies
Picked 2 colonies from each plate, grow in 3mL LB with Carbenicillin and Chloramphenicol antibiotics to make cultures.
Carb50 3ɥL
CmR25 2.24ɥL
Carb25+Cmr12.5 1.5ɥL Carb + 1.12ɥL CmR
Incubate in 37°C shaker
Stop the incubation when cultures reach approximately OD=0.1
Make a 1:100 dilution for the culture with lowest OD
Dilute other cultures accordingly to normalize all ODs (in reference to the lowest OD culture)
Add different concentrations of ethanol to the cultures
Measure absorbance of chromoproteins using a plate reader</LI><LI>RE digest for composite parts (CP3+alcR, CP3+amilCP, CP3+spispink) using EcoRI/PstI

        
Result = All band sizes were correct exceptCP3+alcR. Gel extracted the correct fragments
        </LI></UL></DIV><BUTTON type="button" class="week14button">Week 14 (26<SUP>th</SUP> to 30<SUP>th</SUP> September)</BUTTON><DIV class="cellfree14"><H1 class="subnote">Cell-Free Mechanism</H1><UL class="list"><LI>Repeated induction of cells with IPTG, SDS-PAGE and western Blot from last week</LI><LI>Western Blot still shows correct protein in the insoluble fraction.</LI></UL></DIV><DIV class="geneswitch14"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>EcoRI/PStI digest for CP3 + alcR
        </LI><LI>Made cultures for co-transformation colonies with half antibiotic concentration until samples reached an OD of 0.6. Diluted the samples to normalise the OD before putting samples into the plate reader for comparison.

        </LI><LI>Re-did RE digest for verification of CP + alcR with SphI-HF and AhdI
Expected band sizes: 
CP + alcR - 2472 + 1580 + 683
control plasmid - 1254 + 2073 + 172</LI><LI>Ran co-transformed colonies in plate reader. Normalise OD of samples in the plate reader until it reaches an OD of 0.6. Then, induce cells with ethanol.
        </LI><LI>Redid RE digest of CP3 +alcR with EcorI and PstI
        


Gel extraction-not done as the band sizes for CP3 + alcR were not right. The control worked fine.</LI><LI>Miniprep overnight cultures of alcA 1, alcA 2, alcR in submission vector
        </LI><LI>Miniprep overnight cultures of alcA 1, alcA 2, alcR in submission vector
        </LI><LI>Miniprep overnight cultures of alcA 1, alcA 2, alcR in submission vector
        
Note: Concentration/ratios were weird for miniprep products, proceed to RE digest for verification regardless
          </LI><LI>Absorbance measurement using plate reader (OD600, OD588, OD540)
        </LI><LI>RE digest for CP3+alcR_pSB1C3 with EcoRI/PstI/PvuI-HF</LI><LI>pSB1C3 verification using PstI/ZraI
Expected band size:
alcR - 2762+1921
alcA1 - 1921+861+131
alcA2 - 1921+780+131
control plasmid - 2867+2180
        </LI></UL></DIV><BUTTON type="button" class="week15button">Week 15 (3<SUP>rd</SUP> to 7<SUP>th</SUP> October)</BUTTON><DIV class="cellfree15"><H1 class="subnote">Cell-Free Mechanism</H1><UL class="list"><LI>Typed up lab work for this mechanism</LI><LI>Prepared PSB1C3_AOx to be sent off to iGEM HQ</LI></UL></DIV><DIV class="geneswitch15"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI>Repeat ligation for 30/9 and transformed them following protocol
1:1
        
1:3
        
Controls
        </LI><LI>Prepared overnight cultures for rfp quantification and ethanol induction in co-transformants</LI><LI>Colony PCR
        <P style="text-align:center;">Expected band sizes:</P></LI><LI>Ligation
Vector concentration: pSB1C3 - 61.8 ng/ɥL
Insert concentration: 
</LI><LI>alcR (E/P) - 42.2 ng/ɥL</LI><LI>CP3 + alcR - 42.0 ng/ɥL</LI>
1:1
        
1:3
        
Controls
        <LI>Transformation of ligation following protocols</LI><LI>RE digest 
Band sizes: CP2 (pSB1C3) - 2029 + 76
        </LI></UL></DIV><BUTTON type="button" class="week16button">Week 16 (10<SUP>th</SUP> to 14<SUP>th</SUP> October)</BUTTON><DIV class="geneswitch16"><H1 class="subnote">Inducible Gene Switch</H1><UL class="list"><LI><B>Important dates:11<SUP>th</SUP> October- Sent alcA 2 + amilCP ( 1st composite part) to iGEM HQ</B></LI><LI>Inoculated cultures. Miniprep results:
        </LI><LI>RE digest
Expected band sizes: alcR- 2654 + 1654 + 896 + 163 + 41
        </LI><LI>Gel extraction
        </LI><LI>Ligation
Vector concentration: pSB1C3 - 134.8 ng/ɥL
Insert concentration: alcA 1- 79.1 ng/ɥL
:alcA 2 - 18.5 ng/ɥL
:CP3 + amilCP - 18.1 ng/ɥL
1:1
        
1:3
        
1:5
        
Controls
        </LI><LI>Miniprep O/N cultures
        </LI><LI>Completed gel extraction procedure. Tube containing gel was stored at 4℃ room
        </LI><LI>Submission vector ligation
Vector: pSB1C3 - 99.3 ng/ɥL
Insert: alcR - 37.5 ng/ɥL
CP3 + alcR -328.6 ng/ɥL

1:1
        
1:3
        
1:5
        
Controls
        </LI><LI>Miniprep submission vector colonies
        </LI></UL></DIV><DIV class="floatright2 project1"><A style="z-index:100; position: relative" class="projectlink" href="#top">Top ^</A></DIV><DIV class="footer" style="z-index: 99; position: relative"><H1 id="footer" style="color:white"><STRONG> Sponsors </STRONG></H1><DIV class="box"><P style="font-size:medium"><STRONG>Follow us on</STRONG></P><DIV><P><STRONG>Email: <A href="mailto:uomigem16@gmail.com">uomigem16@gmail.com</A></STRONG></P></DIV></DIV></DIV></CENTER></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>