<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Manchester_Description_mechanism2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Manchester/Description/mechanism2</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><TITLE> Manchester iGEM 2016 </TITLE><NAV><UL id="nav_bar"><LI><A href="https://2016.igem.org/Team:Manchester/Team"><STRONG>Team</STRONG></A></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Team">Team</A></STRONG></LI><LI class="dropdown_content"><STRONG><A href="https://2016.igem.org/Team:Manchester/Attributions">Attributions</A></STRONG></LI></UL><LI><A href="https://2016.igem.org/Team:Manchester/Description"><STRONG>Project</STRONG></A></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Description">Overview</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Description/mechanism1">Cell-free Mechanism</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Description/mechanism2">Inducible Gene Switch</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Proof">Proof of Concept</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Notebook">Notebook</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Project/Protocols">Protocol</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Safety">Safety</A></STRONG></LI><LI class="dropdown_content"><A href="https://2016.igem.org/Team:Manchester/Achievement"><STRONG>Achievement</STRONG></A></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Achievement">Overview</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Judging">Judging Criteria</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Collaborations">Collaborations</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Parts">Parts</A></STRONG></LI><LI class="dropdown_content"><A href="https://2016.igem.org/Team:Manchester/Human_Practices"><STRONG>Human Practices</STRONG></A></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Human_Practices">Overview</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Human_Practices/Charities">Charities</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Human_Practices/Industries">Industries</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Human_Practices/Outreach">Outreach</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Human_Practices/Professional_Bodies">Professional Bodies</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Integrated_Practices">Integrated Practices</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Engagement">Engagement</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/HP/Gold">HP Gold</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/HP/Silver">HP Silver</A></STRONG></LI><LI class="dropdown_content"><A href="https://2016.igem.org/Team:Manchester/Model"><STRONG>Modelling</STRONG></A></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Model">Overview</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Model/hp">Modelling Integration</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Model/result">Results</A></STRONG></LI><LI><STRONG><A href="https://2016.igem.org/Team:Manchester/Model/ModelExplorer">Model Explorer</A></STRONG></LI><LI class="dropdown_content"><A href="https://2016.igem.org/Team:Manchester/Safety"><STRONG>Safety</STRONG></A></LI></NAV><DIV class="team"><H1 class="nobreak grey25">Mechanism 2</H1><H1 class="nobreak gold40">Inducible Gene Switch</H1><DIV class="team1"><P id="summer" style="border-bottom: 1px black solid ;font-size:25px;text-weight:bold;display:inline-block">What have we achieved over the summer?</P><UL class="font17"><LI>Synthesised and codon optimized <I>alcR</I> (<A href="http://partsregistry.org/Part:BBa_K2092001" target="_blank">BBa_K2092001</A>) gene to be used in <I>Escherichia coli (E. coli)</I>.

</LI><LI>Successfully <A href="https://2016.igem.org/Team:Manchester/Proof#cotrans" target="_blank">assembled</A> the <A href="https://2016.igem.org/Team:Manchester/Description/mechanism2" target="_blank">AlcR</A> ethanol regulon system in <I>E. coli</I></LI><LI><A href="https://2016.igem.org/Team:Manchester/Description/mechanism2#improved" target="_blank">Improved</A> an existing <A href="http://parts.igem.org/Part:BBa_K2092002" target="_blank">P<I>alcA</I> BioBrick

</A></LI><LI>Successfully <A href="https://2016.igem.org/Team:Manchester/Description/mechanism2#promoter" target="_blank" target="_blank">characterised</A> selected constitutive promoters  (<A href="http://partsregistry.org/Part:BBa_J23100" target="_blank" target="_blank">BBa_J23100</A>, <A href="http://partsregistry.org/Part:BBa_J23110" target="_blank" target="_blank">BBa_J23110</A>) with <I><A href="https://www.ncbi.nlm.nih.gov/pubmed/12060735" target="_blank">mrfp1</A></I> as a reporter gene.

</LI><LI>Successfully <A href="https://2016.igem.org/Team:Manchester/Description/mechanism2#chromo" target="_blank">characterised</A> selected chromoproteins (<I>amilCP</I>, <A href="http://partsregistry.org/Part:BBa_K592009" target="_blank">BBa_K592009</A>; <I>spisPink</I>, <A href="http://parts.igem.org/Part:BBa_K1033932" target="_blank">BBa_K1033932</A>)

</LI><LI>Successfully demonstrated the ability of our new chromoprotein BioBrick (<A href="http://parts.igem.org/Part:BBa_K2092006" target="_blank">BBa_K2092006</A>) to <A href="https://2016.igem.org/Team:Manchester/Description/mechanism2#chromo" target="_blank">express</A> blue colour that is visible to naked eye.

</LI><LI>Successfully submitted and registered the relevant <A href="https://2016.igem.org/Team:Manchester/Parts" target="_blank">parts</A> to iGEM Registry in pSB1C3 backbone.

</LI><LI>Ongoing isolation and <A href="https://2016.igem.org/Team:Manchester/Proof#cotrans" target="_blank">testing</A> of the functionality of our <A href="https://2016.igem.org/Team:Manchester/Description/mechanism2" target="_blank">AlcR</A> ethanol regulon system.

</LI></UL><P id="how" style="border-bottom: 1px black solid ;font-size:25px;text-weight:bold;display:inline-block">How does it work?</P><CENTER><P class="width80" style="margin:auto; font-size: 13px;"><B>Figure 1.</B> Schematic representation of the plasmid constructs used in our project.</P></CENTER><H2><I>alcR</I></H2><P style="font-size:17px;" class="lineheight160"><I>alcR</I><A href="http://parts.igem.org/Part:BBa_K2092002" target="_blank">(BBa_K2092002)</A> is a positive regulatory gene in the ethanol regulon of filamentous fungus <I>Aspergillus nidulans (A.nidulans)</I>. It encodes a protein that acts as a transcription factor which would bind to its target promoters <I>alcA</I> (P<I>alcA</I>, <A href="http://parts.igem.org/Part:BBa_K2092001" target="_blank">BBa_K2092001</A>) and <I>aldA</I>. The expression of the downstream gene of P<I>alcA</I>  is strongly induced through the positive transcriptional regulator AlcR protein by various substrates such as ethanol and threonine. For our project, we were interested in the ability of the AlcR protein, under the influence of ethanol, to initiate transcription of chromoproteins by binding to specific sites on the P<I>alcA</I> (<A href="http://parts.igem.org/Part:BBa_K2092002" target="_blank">BBa_K2092002</A>) <SUP>[1]</SUP>. The chromoproteins used were from previously characterised BioBricks by 2013 iGEM Uppsala-Sweden: <I>amilCP</I> with RBS (<A href="http://parts.igem.org/Part:BBa_K1033930 " target="_blank" target="_blank">BBa_K1033930</A>) and <I>spisPink</I> with RBS (<A href="http://parts.igem.org/Part:BBa_K1033925" target="_blank" target="_blank">BBa_K1033925</A>).  

      

   Our project focuses on the ability of AlcR to bind to its binding sites on the P<I>alcA</I>. The <I>alcR</I> gene is a new BioBrick we have characterised this year (<A href="http://parts.igem.org/Part:BBa_K2092004" target="_blank">BBa_K2092004</A>).  In addition, we have also submitted another <I>alcR</I> BioBrick with a constitutive promoter (<A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>) placed upstream of the <I>alcR</I> (<A href="http://parts.igem.org/Part:BBa_K2092004" target="_blank">BBa_K2092004</A>) gene. 



    </P><H2><I>alcA</I> promoter (P<I>alcA</I>) </H2><P style="font-size:17px;" class="lineheight160">P<I>alcA</I><A href="http://parts.igem.org/Part:BBa_K678002" target="_blank">(BBa_K678002)</A> is one of the strongest inducible promoters in <I>Aspergillus nidulans (A. nidulans)</I> and is widely used to overexpress proteins <SUP>[2]</SUP>. The native P<I>alcA</I> consists of three AlcR binding sites: &quot;a&quot;, &quot;b&quot; and &quot;c&quot; (Fig. 2).  There is evidence that two sites, either in direct or inverted orientation, are necessary for full transcriptional activation of P<I>alcA</I><SUP>[3]</SUP>. Based on different publications <SUP>[1], [3]</SUP>, we concluded that the number and the position of the AlcR binding sites on P<I>alcA</I> are crucial for the strength of transcriptional activation of P<I>alcA</I>. Hence, we took the factors (number and position of AlcR binding sites) into account when deciding which P<I>alcA</I> variants to use for our system. 

      

Two of the P<I>alcA</I> variants that we used in our experiment were the native P<I>alcA</I><A href="http://parts.igem.org/Part:BBa_K678002" target="_blank">(BBa_K678002)</A> and our synthesized P<I>alcA</I> variant (P<I>alcA</I>(var); <A href="http://parts.igem.org/Part:BBa_K678003" target="_blank">BBa_K678003</A>) (Fig.2): 

<UL class="font17"><LI><B>P<I>alcA</I></B> is the native variant which is a previously characterised BioBrick by iGEM DTU-Denmark 2011 <A href="http://parts.igem.org/Part:BBa_K678001" target="_blank">(BBa_K678001)</A>. We <A href="https://2016.igem.org/Team:Manchester/Description/mechanism2#improved" target="_blank">improved this BioBrick</A> by adding several missing restriction sites on the Prefix and Suffix <A href="http://parts.igem.org/Part:BBa_K2092002" target="_blank">(BBa_K2092002)</A>.</LI><LI><B>P<I>alcA</I>(var)</B> is a new BioBrick we have characterised this year <A href="  http://parts.igem.org/Part:BBa_K2092003" target="_blank">(BBa_K2092003)</A>. P<I>alcA</I>(var) <A href="http://parts.igem.org/Part:BBa_K2092003" target="_blank">(BBa_K2092003)</A> was inspired by a research done in creating a functional, chemically inducible gene switch for monocotyledonous plant sugar cane <SUP>[3]</SUP> and <I>Escherichia coli</I><SUP>[4]</SUP>. </LI></UL></P><CENTER><DIV class="width75"><P style="font-size: 13px;"><B>Figure 2.</B> Schematic representations of PalcA variant constructs adapted from <SUP>[1]</SUP>. Binding site “a” consists of two direct tandem repeats, “b” consists of two inverted tandem repeats and “c” consists of three half sites with both direct and inverted tandem repeats. </P></DIV></CENTER><P style="font-size:17px;" class="lineheight160">P<I>alcA</I><A href="http://parts.igem.org/Part:BBa_K678002" target="_blank">(BBa_K678002)</A> and P<I>alcA</I>(var) <A href="http://parts.igem.org/Part:BBa_K678003" target="_blank">(BBa_K678003)</A> differ in the AlcR binding sites. P<I>alcA</I> has binding sites &quot;abc&quot; while P<I>alcA</I>(var) contains only binding sites &quot;bc&quot;. Binding site &quot;a&quot; contains two direct tandem repeats; binding site &quot;b&quot; is a palindromic target which contains two inverted tandem repeats while binding site &quot;c&quot; consists of three half sites with both direct and inverted tandem repeats. The binding sites &quot;abc&quot; have been previously localized in the P<I>alcA</I> by footprinting experiments and it has also been shown that each AlcR target in the P<I>alcA</I> contributes differently to the activation of the downstream protein expression <SUP>[3]</SUP>. 

    </P><P id="plasmid" style="border-bottom: 1px black solid ;font-size:25px;text-weight:bold;display:inline-block">Plasmid verification</P><P style="font-size:17px;" class="lineheight160">All <A href="https://2016.igem.org/Team:Manchester/Parts" target="_blank">parts</A> obtained from previous iGEM teams were first <A href="https://2016.igem.org/Team:Manchester/Notebook" target="_blank">transformed</A>, including:</P><UL class="font17"><LI> Constitutive promoter (CP) variant 1 (CP1, <A href="http://parts.igem.org/Part:BBa_J23100" target="_blank">BBa_J23100</A>)</LI><LI> CP variant 2 (CP2, <A href="http://parts.igem.org/Part:BBa_J23119" target="_blank">BBa_J23119</A>) </LI><LI> CP variant 3 (CP3, <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>) </LI><LI> P<I>alcA</I><A href="http://parts.igem.org/Part:BBa_K2092002" target="_blank">(BBa_2092002)</A></LI><LI><I>amilCP</I> (incl. RBS) <A href="http://parts.igem.org/Part:BBa_K1033930" target="_blank">(BBa_K1033930)</A></LI><LI><I>amilGFP</I> (incl. RBS) <A href="http://parts.igem.org/Part:BBa_K1033931" target="_blank">(BBa_K1033931)</A></LI><LI><I>spisPink</I> (incl. RBS) <A href="http://parts.igem.org/Part:BBa_K1033925" target="_blank">(BBa_K1033925)</A></LI></UL><DIV class="column half_size"><P style="font-size: 13px;"><B>Figure 3.</B> 1% TAE agarose gel showing restriction enzyme digest results. Band sizes expected can be seen in Table 1. </P></DIV><DIV class="column half_size"><P style="font-size: 13px;"><B>Table 1.</B> Table outlining the details of restriction enzyme digest in Figure 3, with samples: (i) Constitutive promoter (CP) variant 1 (CP1) (ii) CP variant 2 (CP2) (iii) CP variant 3 (CP3) (iv) P<I>alcA</I> (v) <I>amilCP</I> (incl. RBS) (vi) <I>amilGFP</I> (incl. RBS) (vii) <I>spisPink</I> (incl. RBS) and (viii) pUC19 (positive control) using restriction enzyme digest. The restriction enzymes used and band sizes expected are shown.  </P></DIV><P style="font-size:17px;" class="lineheight160">Based on the digest, it was concluded that all samples except CP2 <A href="http://parts.igem.org/Part:BBa_J23119" target="blank">(BBa_J23119)</A>and P<I>alcA</I><A href="http://parts.igem.org/Part:BBa_K2092002" target="_blank">(BBa_2092002)</A> gave us a positive conformation of the plasmid. We repeated the digest for CP2 and P<I>alcA</I> with several other enzymes and managed to get a positive verification of CP2 but not P<I>alcA</I>. 
  </P><DIV style="width:100%"><DIV class="figure2 column half_size"><CENTER><DIV class=" half_size"><P style="clear:both;font-size :13px;"><B>Figure 4.</B> 1% TAE agarose gel showing restriction enzyme digest results. Band sizes expected can be seen in Table 2.</P></DIV></CENTER></DIV><DIV class="column half_size"><P style="font-size :13px;"><B>Table 2.</B> Table outlining the details of restriction enzyme digest in Figure 4, with samples: (i) <I>amilGFP</I> (positive control) (ii) Constitutive promoter variant 2 (CP2). The restriction enzymes used and band sizes expected are shown.</P></DIV></DIV><P style="padding-top:20px;clear:both;font-size:17px;" class="lineheight160">We then proceeded to send two biological replicates of P<I>alcA</I><A href="http://parts.igem.org/Part:K678001" target="_blank">(BBa_K678001)</A> for sequencing to confirm that the part obtained is correct. The sequencing results confirmed that two restriction sites – XbaI and SpeI are absent from the <A href="http://parts.igem.org/Help:BioBrick_Prefix_and_Suffix" target="_blank">BioBrick Prefix and Suffix</A>.
   </P><CENTER><P style="font-size: 13px;"><B>Table 3.</B> Table outlining the sequencing results of 2 P<I>alcA</I> biological replicates (a) and (b). It can be concluded that restriction sites XbaI and SpeI are absent.</P></CENTER><CENTER><P class="legend"><B>Figure 5.</B> Multiple sequence alignment between the original P<I>alcA</I><A href="http://parts.igem.org/Part:K678001" target="_blank">(BBa_K678001)</A> gene sequence adapted from the iGEM Registry and two biological replicates (a, b) of P<I>alcA</I> BioBrick we received sequenced by Eurofins Laboratories Ltd. It can be concluded that restriction sites XbaI and SpeI are absent.</P></CENTER><P style="font-size:17px;" class="lineheight160">Hence, we decided to add another element to our project – <A href="https://2016.igem.org/Team:Manchester/Description/mechanism2#improved" target="_blank">an improved BioBrick</A> by adding the two missing restriction sites on the Prefix and Suffix. 
   </P><P id="promoter" style="border-bottom: 1px black solid ;font-size:25px;text-weight:bold;display:inline-block">Constitutive Promoter (CP) Characterization</P><P style="font-size:17px;">Our choice of constitutive promoters comes from the <A href="http://parts.igem.org/Part:BBa_J23110" target="blank">constitutive promoter family</A> created by iGEM Berkeley 2006. We tested two of two constitutive promoter variants, variant 1 (CP1, <A href="http://parts.igem.org/Part:BBa_J23100" target="_blank">BBa_J23100</A>) and variant 3 (CP3, <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>) that are present in the <A href="http://parts.igem.org/Part:BBa_J61002" target="_blank">BBa_J61002</A> backbone that contains <I><A href="https://www.ncbi.nlm.nih.gov/pubmed/12060735" target="_blank">mrfp1</A></I> gene downstream of the promoter, which encodes the monomeric red fluorescent protein (mRFP1).  CP2 <A href="http://parts.igem.org/Part:BBa_J23119" target="_blank">(BBa_J23119)</A>, however, comes in a different backbone <A href="http://parts.igem.org/Part:pSB1A2" target="_blank">psB1A2</A> and does not have the <I>mrfp1</I> gene downstream of the promoter.  Both of the <A href="http://parts.igem.org/Part:BBa_J61002" target="_blank">BBa_J61002</A> and <A href="http://parts.igem.org/Part:pSB1A2" target="_blank">psB1A2</A> plasmid backbones were also found to have different origin of replications (ORIs).

</P><P style="font-size: 13px;"><B>Table 4.</B> Table outlining the properties of our candidate constitutive promoter variants (CP), including: (i) variant 1 (CP1, <A href="http://parts.igem.org/Part:BBa_J23100" target="_blank">BBa_J23100</A>) (ii) variant 2 (CP2, <A href="http://parts.igem.org/Part:BBa_J23119" target="_blank">BBa_J23119</A>) (iii) variant 3 (CP3, <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>)</P><P style="font-size:17px;"> Before we characterised our constitutive promoters, we tried to clone our CP2 promoter into the same <A href="http://parts.igem.org/Part:BBa_J61002" target="_blank">BBa_J61002</A> backbone as CP1 and CP3.  However, after several unsuccessful cloning attempts we decided to proceed with our project using only CP1 and CP3.   

  </P><CENTER><DIV class=" full_size"><P style="clear:both;font-size :13px;"><B>Figure 6.</B> Schematic representation of plasmids consist of constitutive promoter (CP) variants: (a) variant 1 (CP1; <A href="http://parts.igem.org/Part:BBa_J23100" target="_blank">BBa_J23100</A>) (b) variant 2 (CP2,<A href="http://parts.igem.org/Part:BBa_J23119" target="_blank">BBa_J23119</A>) (c) variant 3 (CP3, <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">BBa_J23110</A>) adapted from SnapGene. CP1 and CP3 are present in the <A href="http://parts.igem.org/Part:BBa_J61002" target="_blank">BBa_J61002</A> backbone that contains <I><A href="https://www.ncbi.nlm.nih.gov/pubmed/12060735" target="_blank">mrfp1</A></I> gene downstream of the promoter and have ColE1 origin. CP2 <A href="http://parts.igem.org/Part:BBa_J23119" target="_blank">(BBa_J23119)</A>, is present in the <A href="http://parts.igem.org/Part:pSB1A2" target="_blank">psB1A2</A> backbone which <I>mrfp1</I> gene is absent downstream of the promoter and has pMB1 origin.</P></DIV></CENTER><P style="font-size:17px;clear:both" class="lineheight160">We performed an mRFP quantification by comparing three technical and two biological replicates of CP1 and CP3. Cultures were first normalised to an appropriate OD and then 200µl of the culture was added to each well. This was to ensure that the initial cell density is uniform for all samples. An anhydrotetracycline(aTc)-inducible Keasling vector <SUP>[7]</SUP>, <A href="https://www.addgene.org/35345/" target="_blank">pBbB2k-GFP,</A> was used as a positive control. The vector was induced at mid-log phase at OD<SUB>600</SUB> = 0.4 by adding 50nM/µl of aTC.
  </P><DIV class="singlepic"><CENTER><DIV class="half_size"><P class="legend"><B>Figure 7.</B> Image of the 96-well plate used for the mRFP quantification. Quantification was done using a FLUOstar Omega (BMG Labtech) plate reader. OD<SUB>600</SUB> and mRFP (λ= 544nm) was measured every 30 minutes over a period of 24 hours. During idle time, the plate was shaken at 500rpm. The plate reader maintained a constant temperature of 37<SUP>o</SUP>C throughout the duration of quantification.</P></DIV></CENTER></DIV><DIV class="singlepic2"><CENTER><DIV class="width80"><P class="legend"><B>Figure 8.</B> 24-hour growth curves of DH5α <I>E. coli</I> containing plasmids: (i) <A href="“http://partsregistry.org/Part:BBa_J23100”" target="blank">CP1</A> (ii) <A href="“http://partsregistry.org/Part:BBa_J23110”" target="blank">CP3</A> (iii) pBbB2k-GFP Keasling vector (KV; positive control) grown in LB broth (negative control).  KV(u) = Uninduced KV; KV(i) = 1mM aTc-induced KV, at mid-log phase, OD<SUB>600</SUB> = ± 0.4.  The samples were measured every 30 minutes at OD<SUB>600</SUB> using the BMG Labtech FLUOstar Omega spectrophotometer, under constant temperature of 37°C.  All points are the mean of 3 technical and 2 biological replicates for each plasmid, normalized to the blank with error bars = SD.    </P></DIV><P style="font-size:17px;">Fig. 8 shows that all of the <I>E. coli</I> samples with plasmids <A href="“http://partsregistry.org/Part:BBa_J23100”" target="blank">CP1</A>, <A href="“http://partsregistry.org/Part:BBa_J23110”" target="blank">CP3</A>, uninduced pBbB2k-GFP Keasling vector (KV(u)) and anhydrotetracycline(aTc)-induced pBbB2k-GFP Keasling vector (KV(i)) had normal growth curve patterns over the 24 hours period with no detectable growth in negative control.  Lag phases (T<SUB>0-1</SUB>) of all samples were identical, which was what we were expecting as the initial cell density should be uniformed (Cultures were normalized to OD<SUB>600</SUB>=0.05, 200µl of culture per well).  The exponential growth phases, however, were different for CP3 and the remaining samples.  Log phase of CP3 (T<SUB>1-6</SUB>) was comparably shorter than CP1, KV(u) and KV(i) (T<SUB>1-8.5</SUB>), suggesting that CP3 had shorter doubling time (29min) than CP1 (37min), KV(u) (56min) and KV(i) (70min).  CP3 entered the stationary phase (T<SUB>8-24</SUB>) after the transition phase at T<SUB>6-8</SUB>.  In contrast, other samples attained stationary growth (T<SUB>10-24</SUB>) 10 hours after the transition phase at T<SUB>10</SUB>.  

There was no observable differences in growth curve patterns between KV(u) and KV(i), leading us to suspect that our inducer (aTc) could be degraded during the course of experiment.  After conducting research on this subject, we found evidence that the incubation temperature (37°C) might be the main factor which led to the degradation of aTc in our case.  The decay rate of aTc was reported to be 2-fold higher at 37°C than at 30°C without taking into account the other factors <SUP>[5]</SUP>.

The major findings demonstrated that overall growth rate of CP3 was indeed higher than CP1, KV(u) and KV(i), which suggested that the following experiment on fluorescence intensity may not be comparable due to the difference in baseline cell activities.     </P><DIV class="singlepic2"><CENTER><DIV class="width80"><P class="legend"><B>Figure 9.</B> Promoter activity of <A href="“http://partsregistry.org/Part:BBa_J23100”" target="blank">CP1</A> and <A href="“http://partsregistry.org/Part:BBa_J23110”" target="blank">CP3</A> fitted to the mRFP fluorescence intensity per cell (U/OD<SUB>600</SUB>) data, with pBbB2k-GFP Keasling vector (KV) as positive control and LB broth as negative control.  KV(u) = Uninduced KV; KV(i) = 1mM aTc-induced KV, at mid-log phase, OD<SUB>600</SUB>= ± 0.4.  The samples were measured every 30 minutes at λ= 544nm using the BMG Labtech FLUOstar Omega plate reader, under constant temperature of 37°C.  All points are the mean of 3 technical and 2 biological replications for each plasmid, normalized to the blank with error bars = SD.</P></DIV><P style="font-size:17px;">Both CP1 and CP3 have the same plasmid backbone <A href="http://partsregistry.org/Part:BBa_J61002" target="blank">BBa_J61002</A>, with RFP being placed downstream of the promoters.  Thus, the promoter strength and stability of CP1 and CP3 can be inferred from the measurement of <I>mrfp1</I> gene expression dynamics, observable in the form of fluorescence intensity from the mRFP. 
 


Fig. 9 shows that promoter activities of CP1 and CP3 were similar during lag phase (T<SUB>0-2</SUB>).  Despite the lower CP3 promoter activity than CP1 at T<SUB>2</SUB>, CP3 promoter activity began to increase steadily from T<SUB>6</SUB>, which may correspond to the transition phase from lag to log.  The promoter activity of CP3 was constant during stationary phase (T<SUB>8-24</SUB>) as reported from the linear increase in fluorescence intensity.  The stable fluorescence intensity recorded in CP3 plasmid could be explained by the slow protein unfolding kinetics of mRFP1, as reported in a previous study <SUP>[6]</SUP>.  Based on this, we assumed that mRFP1 was not degraded across an incubation period of 24 hours and this should be the similar case in CP1. 



On the other hand, CP1 showed a more marked increase in promoter activity at ~T<SUB>10</SUB>, during the early stationary phase.  However, the fluorescence intensity fell significantly at ~T <SUB>15</SUB> with no supporting explanations to why mRFP1 was degraded as the expression of mRFP1 should be stable within the 24 hours experimental period.<SUP>[2]</SUP>

Similar to Fig. 8, there was no observable differences in the curve patterns between KV(u) and KV(i), probably due to the same reason - aTc could have been degraded due to the incubation temperature (37°C) <SUP>. 



Results from both of these experiments led us to proceed with our cloning procedures, using CP3 as the main constitutive promoter for <A href="https://2016.igem.org/Team:Manchester/Parts" target="blank">our BioBricks</A>: CP3+<I>amilCP</I><A href=" http://parts.igem.org/Part:BBa_K2092006" target="blank">(BBa_K2092006)</A> and CP3+alcR <A href=" http://parts.igem.org/Part:BBa_K2092008" target="blank">(BBa_K2092008)</A>.  Both of these BioBrick constructs form the very fundamental framework of our project, specifically in the <A href="https://2016.igem.org/Team:Manchester/Description/mechanism2" target="blank">Inducible Gene Switch</A> mechanism.  To verify our proof of concept for <A href=" https://2016.igem.org/Team:Manchester/Description/mechanism2" target="blank">Inducible Gene Switch</A>, the CP3+<I>amilCP</I> BioBrick <A href=" http://parts.igem.org/Part:BBa_K2092006" target="blank">(BBa_K2092006)</A> was used as a positive control for our co-transformed plasmids, which included two BioBricks: CP3+alcR <A href="http://parts.igem.org/Part:BBa_K2092008" target="blank">(BBa_K2092008)</A> and <A href="http://parts.igem.org/Part:BBa_K2092009" target="blank">P<I>alcA</I>(var)+<I>amilCP</I></A>.  </SUP></P><P style="font-size:17px;">However, for future implications, it is recommended that both of these experiments should be repeated using the same parameters due to several reasons: </P><UL class="font17"><LI>Growth is not comparable between all samples (CP1, CP3 and KV) as CP3 had higher growth rate. </LI><LI>No supporting explanation could be provided to why mRFP1 was degraded in CP1 but remained stable in CP3 as mRFP1 has low protein unfolding kinetics in nature.</LI></UL><P id="chromo" style="border-bottom: 1px black solid ;font-size:25px;text-weight:bold;display:inline-block">Chromoprotein Characterization</P><P style="font-size:17px;">The aim of chromoprotein characterization is to show that under the control of the same promoter, both of our chosen chromoproteins: <I>amilCP</I> with RBS <A href="http://parts.igem.org/Part:BBa_K1033930 " target="_blank">(BBa_K1033930)</A> and <I>spisPink</I> with RBS <A href="http://parts.igem.org/Part:BBa_K1033925" target="_blank">(BBa_K1033925)</A>, are clearly visible at the same time point. To show this, we first inserted the chromoproteins downstream of our choice of constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23110" target="_blank">(BBa_J23110)</A>. We prepared overnight cultures from a single colony and a dilution was made the next morning to normalize it to an OD<SUB>600</SUB> of 0.65. This is to ensure the samples have the same initial cell density. 20µl of each sample was then plated onto a chloramphenicol plate. Pictures were taken at various time points to monitor colour visibility. 
  </P><P style="clear:both;font-size:17px;">It can be concluded that both chromoproteins produce visible colours at the same time point and hence is suitable to use for our project.</P><P id="improved" style="border-bottom: 1px black solid ;font-size:25px;text-weight:bold;display:inline-block">Improved P<I>alcA</I> BioBrick <A href="	http://parts.igem.org/Part:BBa_K2092002" target="blank">(BBa_K2092002)</A></P><DIV class="column full_size"><CENTER><P class="legend"><B>Figure 10.</B> Schematic representation of the PCR method used to add the missing restriction sites on the BioBrick Prefix and Suffix.</P></CENTER></DIV><P style="clear:both;font-size:17px;">P<I>alcA</I><A href="http://parts.igem.org/Part:BBa_K678001" target="_blank"> (BBa_K678001)</A> is a BioBrick requested from the iGEM HQ as it was unavailable in the 2016 kit. Sequencing results confirmed the absence of two restriction sites – XbaI and SpeI on the BioBrick Prefix and Suffix. As these two sites are crucial in the assembly of our composite parts, we designed primers to add the two missing restriction sites. 
 </P><CENTER><P class="legend"><B>Table 5:</B> Table showing the primer sequences used to add the missing restrictions sites XbaI and SpeI on the P<I>alcA</I><A href="http://parts.igem.org/Part:BBa_K678001" target="blank">(iGEM DTU-Denmark 2011, BBa_K678001)</A> BioBrick Prefix and Suffix.</P></CENTER></CENTER>

We carried out PCR using the primers we designed.  To verify that we had successfully reintroduced the missing XbaI and SpeI restriction sites in P<I>alcA</I><A href="http://parts.igem.org/Part:BBa_K678001" target="blank">(iGEM DTU-Denmark 2011, BBa_K678001)</A>, we performed restriction enzyme digest using restriction enzymes EcoRI and PstI before cloning P<I>alcA</I> into the submission vector pSB1C3 and transformed into <I>E. coli</I> DH5α.  Lastly, we successfully demonstrated that XbaI and SpeI restriction sites had been added into P<I>alcA</I>  by digesting the plasmid using restriction enzymes XbaI and SpeI (Fig. 11) and submitted this improved P<I>alcA</I><A href="http://parts.igem.org/Part:BBa_K2092002 target=" blank="">(BBa_K2092002)</A> as one of our new BioBricks this year.  


<CENTER><DIV class="full_size column"><P class="legend"><B>Figure 11.</B> P<I>alcA</I> 1% agarose gel showing the results of restriction enzyme digest on plasmid P<I>alcA</I> using restriction enzymes XbaI and SpeI. Band sizes expected can be seen in the table on the right.</P></DIV></CENTER></DIV></CENTER></DIV><DIV class="team2"><UL class="navi"><LI><A href="#summer">What have we achieved over the summer?</A></LI><LI><A href="#how">How it works?</A></LI><LI><A href="#plasmid">Plasmid verification</A></LI><LI><A href="#promoter">Constitutive promoter (CP) characterization</A></LI><LI><A href="#chromo">Chromoprotein characterization</A></LI><LI><A href="#improved">Improved P<I>alcA</I> BioBrick</A></LI></UL></DIV></DIV><DIV class="team"><DIV class="referencediv"><H1 style="padding-top: 150px;" class="reference">References</H1><UL class="romanlist"><LI>Panozzo, C., Capuano, V., Fillinger, S. and Felenbok, B. (1997). The zinc binuclear cluster Activator AlcR is able to bind to single sites but requires multiple repeated sites for synergistic activation of the alcA gene in <I>Aspergillus nidulans. Journal of Biological Chemistry</I>, 272(36), 22859–22865. </LI><LI>Felenbok, B., Sequeval, D., Mathieu, M., Sibley, S., Gwynne, D.I. and Davies, R.W. (1988). The ethanol regulon in <I>Aspergillus nidulans</I>: Characterization and sequence of the positive regulatory gene <I>alcR</I>. 
<I>Gene</I>, 73(2), 385–396.</LI><LI>Kinkema, M., Geijskes, R.J., Shand, K., Coleman, H.D., De Lucca, P.C., Palupe, A., Harrison, M.D., Jepson, I., Dale, J.L. and Sainz, M.B. (2013). An improved chemically inducible gene switch that functions in the monocotyledonous plant sugar cane. <I>Plant Molecular Biology</I>, 84(4-5), 443–454. </LI><LI>Hemmati, H. and Basu, C. (2015). Transcriptional analyses of an ethanol inducible promoter in <I>Escherichia coli</I> and tobacco for production of cellulase and green fluorescent protein. <I>Biotechnology &amp; Biotechnological Equipment</I>, 29(6), 1043–1052.</LI><LI> Politi, N., Pasotti, L., Zucca, S. Casanova, M., Micoli, G., De Angelis, M. G. C. &amp; Magni, P. (2014) Half-life measurements of chemical inducers for recombinant gene expression. <I>Journal of Biological Engineering</I>, 8(5). </LI><LI> Stepanenko, O. V., Verkhusha, V. V., Vasili, I. K., Shavlovsky, M. M., Kuznetsova, I. M., Uversky, V. N. &amp; Turoverov, K. K. (2004)  Comparative studies on the structure and stability of fluorescent proteins EGFP, zFP506, mRFP1, “dimer2” and DsRed1. <I>Biochemistry</I>, 2006, 43, 14913-14923.</LI><LI> Lee, T. S., Krupa, R. A., Zhang, F., Hajimorad, M., Holtz, W. J., Prasad, N., Lee, S. K. &amp; Keasling, J. D. (2011) BglBrick vectors and datasheets: A synthetic biology platform for gene expression. <I>Journal of Biological Engineering</I>, 5:12. </LI><LI> Morgan, K. (2014). <I>Plasmids 101: Origin of Replication.</I> [online]. Available at: http://blog.addgene.org/plasmid-101-origin-of-replication [Accessed 19 October 2016].</LI></UL></DIV><DIV class="floatleft1 project"><A class="projectlink" href="https://2016.igem.org/Team:Manchester/Description">&lt;&lt; Project Overview</A></DIV><DIV class="floatright2 project"><A class="projectlink" href="#top">Top ^</A></DIV></DIV><DIV class="footer" style="z-index: 99; position: relative"><H1 id="footer" style="color:white"><STRONG> Sponsors </STRONG></H1><DIV class="box"><P style="font-size:medium"><STRONG>Follow us on</STRONG></P><DIV><P><STRONG>Email: <A href="mailto:uomigem16@gmail.com">uomigem16@gmail.com</A></STRONG></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>