Team:Manchester/Description/mechanism2
Manchester iGEM 2016
Team
Team
Attributions
Project
Overview
Cell-free Mechanism
Inducible Gene Switch
Proof of Concept
Notebook
Protocol
Safety
Achievement
Overview
Judging Criteria
Collaborations
Parts
Human Practices
Overview
Charities
Industries
Outreach
Professional Bodies
Integrated Practices
Engagement
HP Gold
HP Silver
Modelling
Overview
Modelling Integration
Results
Model Explorer
Safety
Mechanism 2
Inducible Gene Switch
What have we achieved over the summer?
Synthesised and codon optimized alcR ( BBa_K2092001 ) gene to be used in Escherichia coli (E. coli).
Successfully assembled the AlcR ethanol regulon system in E. coli
Improved an existing PalcA BioBrick
Successfully characterised selected constitutive promoters  ( BBa_J23100 , BBa_J23110 ) with mrfp1 as a reporter gene.
Successfully characterised selected chromoproteins (amilCP, BBa_K592009 ; spisPink, BBa_K1033932 )
Successfully demonstrated the ability of our new chromoprotein BioBrick ( BBa_K2092006 ) to express blue colour that is visible to naked eye.
Successfully submitted and registered the relevant parts to iGEM Registry in pSB1C3 backbone.
Ongoing isolation and testing of the functionality of our AlcR ethanol regulon system.
How does it work?
Figure 1. Schematic representation of the plasmid constructs used in our project.
alcR
alcR (BBa_K2092002) is a positive regulatory gene in the ethanol regulon of filamentous fungus Aspergillus nidulans (A.nidulans). It encodes a protein that acts as a transcription factor which would bind to its target promoters alcA (PalcA, BBa_K2092001 ) and aldA. The expression of the downstream gene of PalcA is strongly induced through the positive transcriptional regulator AlcR protein by various substrates such as ethanol and threonine. For our project, we were interested in the ability of the AlcR protein, under the influence of ethanol, to initiate transcription of chromoproteins by binding to specific sites on the PalcA ( BBa_K2092002 ) [1]. The chromoproteins used were from previously characterised BioBricks by 2013 iGEM Uppsala-Sweden: amilCP with RBS ( BBa_K1033930 ) and spisPink with RBS ( BBa_K1033925 ).
Our project focuses on the ability of AlcR to bind to its binding sites on the PalcA. The alcR gene is a new BioBrick we have characterised this year ( BBa_K2092004 ).  In addition, we have also submitted another alcR BioBrick with a constitutive promoter ( BBa_J23110 ) placed upstream of the alcR ( BBa_K2092004 ) gene.
alcA promoter (PalcA)
PalcA (BBa_K678002) is one of the strongest inducible promoters in Aspergillus nidulans (A. nidulans) and is widely used to overexpress proteins [2]. The native PalcA consists of three AlcR binding sites: "a", "b" and "c" (Fig. 2).  There is evidence that two sites, either in direct or inverted orientation, are necessary for full transcriptional activation of PalcA [3]. Based on different publications [1], [3], we concluded that the number and the position of the AlcR binding sites on PalcA are crucial for the strength of transcriptional activation of PalcA. Hence, we took the factors (number and position of AlcR binding sites) into account when deciding which PalcA variants to use for our system.
Two of the PalcA variants that we used in our experiment were the native PalcA (BBa_K678002) and our synthesized PalcA variant (PalcA(var); BBa_K678003 ) (Fig.2):
PalcA is the native variant which is a previously characterised BioBrick by iGEM DTU-Denmark 2011 (BBa_K678001) . We improved this BioBrick by adding several missing restriction sites on the Prefix and Suffix (BBa_K2092002) .
PalcA(var) is a new BioBrick we have characterised this year (BBa_K2092003) . PalcA(var) (BBa_K2092003) was inspired by a research done in creating a functional, chemically inducible gene switch for monocotyledonous plant sugar cane [3] and Escherichia coli [4].
Figure 2. Schematic representations of PalcA variant constructs adapted from [1]. Binding site “a” consists of two direct tandem repeats, “b” consists of two inverted tandem repeats and “c” consists of three half sites with both direct and inverted tandem repeats.
PalcA (BBa_K678002) and PalcA(var) (BBa_K678003) differ in the AlcR binding sites. PalcA has binding sites "abc" while PalcA(var) contains only binding sites "bc". Binding site "a" contains two direct tandem repeats; binding site "b" is a palindromic target which contains two inverted tandem repeats while binding site "c" consists of three half sites with both direct and inverted tandem repeats. The binding sites "abc" have been previously localized in the PalcA by footprinting experiments and it has also been shown that each AlcR target in the PalcA contributes differently to the activation of the downstream protein expression [3].
Plasmid verification
All parts obtained from previous iGEM teams were first transformed , including:
Constitutive promoter (CP) variant 1 (CP1, BBa_J23100 )
CP variant 2 (CP2, BBa_J23119 )
CP variant 3 (CP3, BBa_J23110 )
PalcA (BBa_2092002)
amilCP (incl. RBS) (BBa_K1033930)
amilGFP (incl. RBS) (BBa_K1033931)
spisPink (incl. RBS) (BBa_K1033925)
Figure 3. 1% TAE agarose gel showing restriction enzyme digest results. Band sizes expected can be seen in Table 1.
Table 1. Table outlining the details of restriction enzyme digest in Figure 3, with samples: (i) Constitutive promoter (CP) variant 1 (CP1) (ii) CP variant 2 (CP2) (iii) CP variant 3 (CP3) (iv) PalcA (v) amilCP (incl. RBS) (vi) amilGFP (incl. RBS) (vii) spisPink (incl. RBS) and (viii) pUC19 (positive control) using restriction enzyme digest. The restriction enzymes used and band sizes expected are shown.
Based on the digest, it was concluded that all samples except CP2 (BBa_J23119) and PalcA (BBa_2092002) gave us a positive conformation of the plasmid. We repeated the digest for CP2 and PalcA with several other enzymes and managed to get a positive verification of CP2 but not PalcA.
Figure 4. 1% TAE agarose gel showing restriction enzyme digest results. Band sizes expected can be seen in Table 2.
Table 2. Table outlining the details of restriction enzyme digest in Figure 4, with samples: (i) amilGFP (positive control) (ii) Constitutive promoter variant 2 (CP2). The restriction enzymes used and band sizes expected are shown.
We then proceeded to send two biological replicates of PalcA (BBa_K678001) for sequencing to confirm that the part obtained is correct. The sequencing results confirmed that two restriction sites – XbaI and SpeI are absent from the BioBrick Prefix and Suffix .
Table 3. Table outlining the sequencing results of 2 PalcA biological replicates (a) and (b). It can be concluded that restriction sites XbaI and SpeI are absent.
Figure 5. Multiple sequence alignment between the original PalcA (BBa_K678001) gene sequence adapted from the iGEM Registry and two biological replicates (a, b) of PalcA BioBrick we received sequenced by Eurofins Laboratories Ltd. It can be concluded that restriction sites XbaI and SpeI are absent.
Hence, we decided to add another element to our project – an improved BioBrick by adding the two missing restriction sites on the Prefix and Suffix.
Constitutive Promoter (CP) Characterization
Our choice of constitutive promoters comes from the constitutive promoter family created by iGEM Berkeley 2006. We tested two of two constitutive promoter variants, variant 1 (CP1, BBa_J23100 ) and variant 3 (CP3, BBa_J23110 ) that are present in the BBa_J61002 backbone that contains mrfp1 gene downstream of the promoter, which encodes the monomeric red fluorescent protein (mRFP1).  CP2 (BBa_J23119) , however, comes in a different backbone psB1A2 and does not have the mrfp1 gene downstream of the promoter.  Both of the BBa_J61002 and psB1A2 plasmid backbones were also found to have different origin of replications (ORIs).
Table 4. Table outlining the properties of our candidate constitutive promoter variants (CP), including: (i) variant 1 (CP1, BBa_J23100 ) (ii) variant 2 (CP2, BBa_J23119 ) (iii) variant 3 (CP3, BBa_J23110 )
Before we characterised our constitutive promoters, we tried to clone our CP2 promoter into the same BBa_J61002 backbone as CP1 and CP3.  However, after several unsuccessful cloning attempts we decided to proceed with our project using only CP1 and CP3.
Figure 6. Schematic representation of plasmids consist of constitutive promoter (CP) variants: (a) variant 1 (CP1; BBa_J23100 ) (b) variant 2 (CP2, BBa_J23119 ) (c) variant 3 (CP3, BBa_J23110 ) adapted from SnapGene. CP1 and CP3 are present in the BBa_J61002 backbone that contains mrfp1 gene downstream of the promoter and have ColE1 origin. CP2 (BBa_J23119) , is present in the psB1A2 backbone which mrfp1 gene is absent downstream of the promoter and has pMB1 origin.
We performed an mRFP quantification by comparing three technical and two biological replicates of CP1 and CP3. Cultures were first normalised to an appropriate OD and then 200µl of the culture was added to each well. This was to ensure that the initial cell density is uniform for all samples.
An anhydrotetracycline(aTc)-inducible Keasling vector [7], pBbB2k-GFP, was used as a positive control. The vector was induced at mid-log phase at OD600 = 0.4 by adding 50nM/µl of aTC.
Figure 7. Image of the 96-well plate used for the mRFP quantification. Quantification was done using a FLUOstar Omega (BMG Labtech) plate reader. OD600 and mRFP (λ= 544nm) was measured every 30 minutes over a period of 24 hours. During idle time, the plate was shaken at 500rpm. The plate reader maintained a constant temperature of 37oC throughout the duration of quantification.
Figure 8. 24-hour growth curves of DH5α E. coli containing plasmids: (i) CP1 (ii) CP3 (iii) pBbB2k-GFP Keasling vector (KV; positive control) grown in LB broth (negative control).  KV(u) = Uninduced KV; KV(i) = 1mM aTc-induced KV, at mid-log phase, OD600 = ± 0.4.  The samples were measured every 30 minutes at OD600 using the BMG Labtech FLUOstar Omega spectrophotometer, under constant temperature of 37°C.  All points are the mean of 3 technical and 2 biological replicates for each plasmid, normalized to the blank with error bars = SD.
Fig. 8 shows that all of the E. coli samples with plasmids CP1 , CP3 , uninduced pBbB2k-GFP Keasling vector (KV(u)) and anhydrotetracycline(aTc)-induced pBbB2k-GFP Keasling vector (KV(i)) had normal growth curve patterns over the 24 hours period with no detectable growth in negative control.  Lag phases (T0-1) of all samples were identical, which was what we were expecting as the initial cell density should be uniformed (Cultures were normalized to OD600=0.05, 200µl of culture per well).  The exponential growth phases, however, were different for CP3 and the remaining samples.  Log phase of CP3 (T1-6) was comparably shorter than CP1, KV(u) and KV(i) (T1-8.5), suggesting that CP3 had shorter doubling time (29min) than CP1 (37min), KV(u) (56min) and KV(i) (70min).  CP3 entered the stationary phase (T8-24) after the transition phase at T6-8.  In contrast, other samples attained stationary growth (T10-24) 10 hours after the transition phase at T10. There was no observable differences in growth curve patterns between KV(u) and KV(i), leading us to suspect that our inducer (aTc) could be degraded during the course of experiment.  After conducting research on this subject, we found evidence that the incubation temperature (37°C) might be the main factor which led to the degradation of aTc in our case.  The decay rate of aTc was reported to be 2-fold higher at 37°C than at 30°C without taking into account the other factors [5]. The major findings demonstrated that overall growth rate of CP3 was indeed higher than CP1, KV(u) and KV(i), which suggested that the following experiment on fluorescence intensity may not be comparable due to the difference in baseline cell activities.
Figure 9. Promoter activity of CP1 and CP3 fitted to the mRFP fluorescence intensity per cell (U/OD600) data, with pBbB2k-GFP Keasling vector (KV) as positive control and LB broth as negative control.  KV(u) = Uninduced KV; KV(i) = 1mM aTc-induced KV, at mid-log phase, OD600= ± 0.4.  The samples were measured every 30 minutes at λ= 544nm using the BMG Labtech FLUOstar Omega plate reader, under constant temperature of 37°C.  All points are the mean of 3 technical and 2 biological replications for each plasmid, normalized to the blank with error bars = SD.
Both CP1 and CP3 have the same plasmid backbone BBa_J61002 , with RFP being placed downstream of the promoters.  Thus, the promoter strength and stability of CP1 and CP3 can be inferred from the measurement of mrfp1 gene expression dynamics, observable in the form of fluorescence intensity from the mRFP. Fig. 9 shows that promoter activities of CP1 and CP3 were similar during lag phase (T0-2).  Despite the lower CP3 promoter activity than CP1 at T2, CP3 promoter activity began to increase steadily from T6, which may correspond to the transition phase from lag to log.  The promoter activity of CP3 was constant during stationary phase (T8-24) as reported from the linear increase in fluorescence intensity.  The stable fluorescence intensity recorded in CP3 plasmid could be explained by the slow protein unfolding kinetics of mRFP1, as reported in a previous study [6].  Based on this, we assumed that mRFP1 was not degraded across an incubation period of 24 hours and this should be the similar case in CP1. On the other hand, CP1 showed a more marked increase in promoter activity at ~T10, during the early stationary phase.  However, the fluorescence intensity fell significantly at ~T 15 with no supporting explanations to why mRFP1 was degraded as the expression of mRFP1 should be stable within the 24 hours experimental period.[2] Similar to Fig. 8, there was no observable differences in the curve patterns between KV(u) and KV(i), probably due to the same reason - aTc could have been degraded due to the incubation temperature (37°C) . Results from both of these experiments led us to proceed with our cloning procedures, using CP3 as the main constitutive promoter for our BioBricks : CP3+amilCP (BBa_K2092006) and CP3+alcR (BBa_K2092008) .  Both of these BioBrick constructs form the very fundamental framework of our project, specifically in the Inducible Gene Switch mechanism.  To verify our proof of concept for Inducible Gene Switch , the CP3+amilCP BioBrick (BBa_K2092006) was used as a positive control for our co-transformed plasmids, which included two BioBricks: CP3+alcR (BBa_K2092008) and PalcA(var)+amilCP .
However, for future implications, it is recommended that both of these experiments should be repeated using the same parameters due to several reasons:
Growth is not comparable between all samples (CP1, CP3 and KV) as CP3 had higher growth rate.
No supporting explanation could be provided to why mRFP1 was degraded in CP1 but remained stable in CP3 as mRFP1 has low protein unfolding kinetics in nature.
Chromoprotein Characterization
The aim of chromoprotein characterization is to show that under the control of the same promoter, both of our chosen chromoproteins: amilCP with RBS (BBa_K1033930) and spisPink with RBS (BBa_K1033925) , are clearly visible at the same time point. To show this, we first inserted the chromoproteins downstream of our choice of constitutive promoter (BBa_J23110) . We prepared overnight cultures from a single colony and a dilution was made the next morning to normalize it to an OD600 of 0.65. This is to ensure the samples have the same initial cell density. 20µl of each sample was then plated onto a chloramphenicol plate. Pictures were taken at various time points to monitor colour visibility.
It can be concluded that both chromoproteins produce visible colours at the same time point and hence is suitable to use for our project.
Improved PalcA BioBrick (BBa_K2092002)
Figure 10. Schematic representation of the PCR method used to add the missing restriction sites on the BioBrick Prefix and Suffix.
PalcA (BBa_K678001) is a BioBrick requested from the iGEM HQ as it was unavailable in the 2016 kit. Sequencing results confirmed the absence of two restriction sites – XbaI and SpeI on the BioBrick Prefix and Suffix. As these two sites are crucial in the assembly of our composite parts, we designed primers to add the two missing restriction sites.
Table 5: Table showing the primer sequences used to add the missing restrictions sites XbaI and SpeI on the PalcA (iGEM DTU-Denmark 2011, BBa_K678001) BioBrick Prefix and Suffix.
We carried out PCR using the primers we designed.  To verify that we had successfully reintroduced the missing XbaI and SpeI restriction sites in PalcA (iGEM DTU-Denmark 2011, BBa_K678001) , we performed restriction enzyme digest using restriction enzymes EcoRI and PstI before cloning PalcA into the submission vector pSB1C3 and transformed into E. coli DH5α.  Lastly, we successfully demonstrated that XbaI and SpeI restriction sites had been added into PalcA by digesting the plasmid using restriction enzymes XbaI and SpeI (Fig. 11) and submitted this improved PalcA (BBa_K2092002) as one of our new BioBricks this year.
Figure 11. PalcA 1% agarose gel showing the results of restriction enzyme digest on plasmid PalcA using restriction enzymes XbaI and SpeI. Band sizes expected can be seen in the table on the right.
What have we achieved over the summer?
How it works?
Plasmid verification
Constitutive promoter (CP) characterization
Chromoprotein characterization
Improved PalcA BioBrick
References
Panozzo, C., Capuano, V., Fillinger, S. and Felenbok, B. (1997). The zinc binuclear cluster Activator AlcR is able to bind to single sites but requires multiple repeated sites for synergistic activation of the alcA gene in Aspergillus nidulans. Journal of Biological Chemistry, 272(36), 22859–22865.
Felenbok, B., Sequeval, D., Mathieu, M., Sibley, S., Gwynne, D.I. and Davies, R.W. (1988). The ethanol regulon in Aspergillus nidulans: Characterization and sequence of the positive regulatory gene alcR. Gene, 73(2), 385–396.
Kinkema, M., Geijskes, R.J., Shand, K., Coleman, H.D., De Lucca, P.C., Palupe, A., Harrison, M.D., Jepson, I., Dale, J.L. and Sainz, M.B. (2013). An improved chemically inducible gene switch that functions in the monocotyledonous plant sugar cane. Plant Molecular Biology, 84(4-5), 443–454.
Hemmati, H. and Basu, C. (2015). Transcriptional analyses of an ethanol inducible promoter in Escherichia coli and tobacco for production of cellulase and green fluorescent protein. Biotechnology & Biotechnological Equipment, 29(6), 1043–1052.
Politi, N., Pasotti, L., Zucca, S. Casanova, M., Micoli, G., De Angelis, M. G. C. & Magni, P. (2014) Half-life measurements of chemical inducers for recombinant gene expression. Journal of Biological Engineering, 8(5).
Stepanenko, O. V., Verkhusha, V. V., Vasili, I. K., Shavlovsky, M. M., Kuznetsova, I. M., Uversky, V. N. & Turoverov, K. K. (2004)  Comparative studies on the structure and stability of fluorescent proteins EGFP, zFP506, mRFP1, “dimer2” and DsRed1. Biochemistry, 2006, 43, 14913-14923.
Lee, T. S., Krupa, R. A., Zhang, F., Hajimorad, M., Holtz, W. J., Prasad, N., Lee, S. K. & Keasling, J. D. (2011) BglBrick vectors and datasheets: A synthetic biology platform for gene expression. Journal of Biological Engineering, 5:12.
Morgan, K. (2014). Plasmids 101: Origin of Replication. [online]. Available at: http://blog.addgene.org/plasmid-101-origin-of-replication [Accessed 19 October 2016].
<< Project Overview
Top ^
Sponsors
Follow us on
Email: uomigem16@gmail.com
