Team:Manchester/Notebook
Manchester iGEM 2016
Team
Team
Attributions
Project
Overview
Cell-free Mechanism
Inducible Gene Switch
Proof of Concept
Notebook
Protocol
Safety
Achievement
Overview
Judging Criteria
Collaborations
Parts
Human Practices
Overview
Charities
Industries
Outreach
Professional Bodies
Integrated Practices
Engagement
HP Gold
HP Silver
Modelling
Overview
Modelling Integration
Results
Model Explorer
Safety
Notebook
Hide All
Expand All
Inducible Gene Switch
Pilot Experiment
Cell-free Mechanism
* Inducible Gene Switch - All activities have been performed by Sathya, Hui Wen and Prakrithi.
* Cell Free Mechanism - All activities have been performed by Guada, Mala and Svetlana.
* Pilot Experiment - All activities have been performed by Guada, Mala and Svetlana.
Week 1 (29th June to 1st July)
Inducible Gene Switch
Re-suspended DNA (constitutive promoters) from iGEM Distribution kit and transformed it into DH5α strain.
Growth was found on the negative control plates so re-transformed DNA
Inoculated transformed cells containing constitutive promoters in 10 mL of LB broth with Carbenicillin 50.
Week 2 (4th to 8th July)
Pilot Experiment
Made stocks of our reagents: glucose solution 0.005 g/mL, glucose oxidase (GOx) solution 0.0025 g/mL, horseradish peroxidase (HRP) solution 250 μL/mL, 2,2′-Azino-bis(3-ethylbenzthiazoline-6-sulfonic acid (ABTS) 0.0025 g/mL.
Inducible Gene Switch
Performed miniprep for overnight cultures
Prepared first batch of DH5α chemical competent cells using TSS (Transformation and Storage Solution) method.
Restriction enzyme digest of constitutive promoters and control(pUC19) for validation with EcoRI and PstI (1 μL each).
Week 3 (11th to 15th July)
Pilot Experiment
Still awaiting the arrival of the BMG Labtech FLUOstar Omega plate reader.
Inducible Gene Switch
Prepared more DH5α chemical competent cells for transformation
Prepared Chloramphenicol antibiotic stock for future use.
Restreaked alcA 1 (BBa_K678001) and spispink (BBa_K1033923, pink chromoprotein)
Week 4 (18th to 22ndnd July)
Pilot Experiment
We began by preparing master mix containing three reagents: GOx, HRP and ABTS (table 1). PBS was used as a solvent.
Final reagent concentration, (μg/ml)
GOx
60
HRP
60
ABTS
100
Table 1.Master mix concentrations.
Six different glucose concentrations were then made as depicted in table 2. PBS was used as a solvent.
Final glucose concentration, (μg/ml)
0.50
1.00
.25
1.50
1.75
2.00
Table 2. Glucose concentrations.
Protocol
50 μl of glucose solution was added into each well. Samples were run in triplicates.
Tube containing master mix was placed into spectrophotometer.
Spectrophotometer was then used to measure absorbance of our green coloured product-oxidised ABTS at 420 nm. Absorbance values were taken every 2 sec for 3 min once 150 μl of master mix was added into each well.
Results
The rate of reaction did not level off and poor colour intensity was observed.
We repeated the same experiment we did earlier this week. However instead of measuring absorbance for 3 min absorbance was measured every 2 sec for a 5 min period.
Results
Reaction rate did level off after measuring absorbance for 5 min. However, colour intensity was still very poor.
Inducible Gene Switch
Important dates:
18th July - Resurrected DNA from iGEM DNA distribution kit
Resuspended plasmids containing required DNA(RBS, amilCP (BBa_K1033930, blue chromoprotein) and amilGFP(BBa_K1033931, yellow chromoprotein)) from iGEM Distribution kit and transformed it into DH5α strain.
Inoculated alcA 1 and spispink for miniprep.
Week 5 (25th to 29th July)
Pilot Experiment
To increase the brightness of the colour change we repeated the experiment we performed on 22/07/2016 (i.e. absorbance measured every 2 sec for 5 min). However this time we doubled the concentration of each reagent that made up the master mix (table 3).
Final reagent concentration, (μg/ml)
GOx
120
HRP
120
ABTS
200
Table 3. Master mix
Result
Increasing the concentration of master mix reagents did not yield intense colour changes
Based on results from yesterday, further increases in in reagents concentration that make up the master mix were tested. We tested the same glucose concentration as on 22/07/2016. The same parameters for the absorbance reading were used (i.e. every 2 sec for 5 min). The only parameter that was changed this time was master mix reagent concentration (table 4).
Final reagent concentration, (μg/ml)
GOx
625
HRP
62.5
ABTS
625
Table 4. Master mix
Result
Strongly visible green colour was not observed following today’s experiment
Today it was decided to increase glucose concentration (table 5) following previous experimental conditions that did not result in appearance of intense green colour. Concentration of master mix reagents and absorbance time window was kept the same as on 22/07/2016 (i.e. absorbance measured every 2 sec for 5 min).
Final glucose concentration, (μg/ml)
0.50
1.00
.25
1.50
1.75
2.00
Table 5
Result
At the end of our experiment we noticed very intense colour change as shown in figure 1.
Figure 1: Colour intensity following addition of master mix (ABTS,  HRP and H2O2)
Inducible Gene Switch
Ordered genes from IDT: alcA2 and alcR
Inoculated CP1 and CP3 in 10 mL of LB broth with Carbenicillin 50.
Week 6 (1st to 5th August)
Pilot Experiment
Performed the following reaction where ABTS is oxidised in the presence of H2O2 (final concentration 250 μg/ml) and HRP (final concentration 250 μg/ml) (table 6).
Final ABTS concentration, (μg/ml)
1
5
10
15
20
Table 6
Measured absorbance of water  at 420 nm
Measured absorbance of each of the reagents at 420 nm at the following concentrations (table 7).
Concentration, (μg/ml)
HRP
0.25
HRP
2.50
H2O2
5.08
Table 7
Set up the reaction with 3 reagents: ABTS, H2O2, HRP. Measured absorbance every 25 sec for 900 sec (table 8)
ABTS concentration (μg/ml)
H2O2 concentration (μg/ml)
HRP concentration (μg/ml)
1.25
1.25
0.0625
1.875
1.875
0.0625
2.5
2.5
0.0625
3.125
3.125
0.0625
3.750
3.750
0.0625
4.375
4.375
0.0625
5.000
5.000
0.0625
5.625
5.625
0.0625
6.25
6.25
0.0625
6.875
6.875
0.0625
7.5
7.5
0.0625
8.125
8.125
0.0625
8.750
8.750
0.0625
9.375
9.375
0.0625
10
10
0.0625
10.625
10.625
0.0625
11.25
11.25
0.0625
11.875
11.875
0.0625
12.5
12.5
0.0625
13.125
13.125
0.0625
Table 8.
Inducible Gene Switch
Important dates
2nd August- Received requested parts from iGEM HQ
5th August- Verified all parts from the DNA distribution kit
Restriction enzyme digest with EcorI-HF and SpeI-HF for ligation of CP2 into BBa_J61002 plasmid(backbone for CP1 and CP3) for rfp quantification as CP2 was originally in pSB1A2 vector.
Gel extraction of CP2 (insert) and CP1 (vector)
Expected band size of
CP2 = 2056 bp and 58 bp
CP1 = 2925 bp and 58 bp
* CP2- extract smaller fragment
* CP1- extract larger fragment
Ran samples on 1% agarose gel with 2-log ladder.  Restriction digest was successful only for CP1. CP1 was successfully extracted from the gel.
Concentration of CP1 after gel extraction
Q5 polymerase PCR for CP2 using protocol
Forward primer = V2 forward from kit
Reverse primer = VR reverse from kit
Tm = 70C
Received parts from iGEM HQ in the form of bacteria on agar. All came in pSB1C3 (Cm resistance). Re-streaked them onto Cm plates, incubate overnight at 37°C to get single colonies next day.
Overnight ligation of CP2 (insert) + CP1 (vector) with T4 ligase using protocol
Transformation of ligated products in DH5α chemical competent cells
Colony PCR for CP1 and CP2 transformants
Expected colony PCR fragment size = 1142bp
Ran colony PCR products on 1% agarose gel to check if ligation was successful. No bands were seen indicating the ligation failed.
Restriction digest to confirm all vectors plasmids that we need from BioBrick kit in 10 ɥL reaction (1 ɥL DNA).
Ran digested products on 1% agarose gel with a 2-log ladder to check band sizes
Ran CP2 PCR product on 4% agarose gel with 2-log DNA ladder and low molecular weight (MW) ladder to check if amplified region was correct
Results:CP2 PCR was successful but ran out of sample stock. Inoculated CP2 PCR product in 10 mL LB broth with Carbenicillin 50
Week 7 (8th to 12th August)
Cell-Free Mechanism
Transformed pET28b (obtained from our supervisor Nick Weise) and pUC19 as a negative control into Escherichia coli DH5a cells.
Plated transformed cells onto LB agar plates.
Made overnight cultures with 50 µg/mL Kanamycin of cloned cells and miniprepped the pET28b vector (Table 1).
Table 1. Miniprep results measured with Nanodrop.
Restriction digest of pET28b with NdeI and SalI for validation of the plasmid
This first digestion didn’t work due to short time of incubation at 37ºC.
Made more overnight cultures with 50 µg/mL Kanamycin of cloned cells and miniprepped pET28b (Table 2).
Table 2. Miniprep results using a Nanodrop .
Restriction digest of pET28b with ClaI and EcoRI while waiting for new SalI from NEB.
Results:
Figure 1.1% TAE agarose gel showing restriction digest. Lane 1, pET28b cut with EcoRI, expected size 5368bp. Lane 2, pET28b cut with ClaI, expected size of 5368bp. Lane 3, pET28b cut with ClaI and EcoRI, expected sizes of 3924bp and 1444bp. Lane 4, pET28b negative control.
Inducible Gene Switch
Important dates:
12,th August- IDT genes arrived -- alcR and alcA2
Restriction digest of alcA 1 and amilCP to construct alcA1 + amilCP. alcA1 was digested with EcoRI + SpeI while amilCP was digested with EcoRI and XbaI. Control, pUC19 was digested with EcoRI and XbaI Expected band sizes after digest with respective enzymes (highlighted band sizes to be gel extracted):
Insert: alcA1 (22.4 ng/ɥL) - 866 + 2047
Vector: amilCP (69.3 ng/ɥL) - 15 + 2742
Control: pUC19 (48.6 ng/ɥL) - 27 + 2659
Restriction digest did not work
Inoculated CP1, CP2, CP3, alcA1, amilCP, amilGFP, spisPink in 10 mL LB broth with appropriate antibiotic
Restriction digest for plasmid verification by using different enzymes with standard 10X Cutsmart buffer
Expected band sizes:
Miniprep inoculated cells and measured the concentrations
Ethanol precipitation for CP2
PCR purification of CP1
Restriction enzyme digest to verify alcA1 using EcoRI and EcoRV
Inoculated CP2_pSB1C3, CP1 and CP3 for rfp quantification
Miniprep inoculated samples
RFP quantification for CP1, CP2_pSB1C3, CP3 - Results were not comparable due to: different backbones (origin of replication) and different incubation time for cultures
Newly synthesised genes from IDT arrived -- alcR and alcA2
Week 8 (15th to 19th August)
Cell-Free Mechanism
AOx gene from IDT arrived!
Transformed pUCIDT_AOx into Escherichia coli DH5a cells.
Plated transformed cells onto LB agar plates with ampicillin.
Made overnight cultures of cells with 50 µg/mL ampicillin and miniprepped pUCIDT_AOx (Table 3).
Table 3. Miniprep results.
Restriction digest of pUCIDT-AOx for characterisation and gel extraction.
Figure 2. 1% LAB agarose gel showing digestion results. Lane 3, pUCIDT_AOx cut with NdeI and SalI, expected sizes of 2760bp and 2036bp.
The digestion was not complete as a third band of 4796bp shows undigested plasmid. For this reason we could not proceed to do gel extraction as it would give a really small DNA concentration.
Make more overnight cultures of pET28b and miniprepped (Table 4).
Table 4. Miniprep results.
Inducible Gene Switch
Restriction enzyme digest of CP1 and chromoproteins for chromoprotein quantification. Ran on gel for gel extraction.
Highlighted band sizes were extracted.
Inoculated pUC19 in 10 mL LB broth with Carbenicillin 50.
Re-suspended genes from IDT in 100 ɥL of sterile milli-Q for 1 hour. Transformed it into DH5α for each gene using chemical transformation protocol. We plated 20 ɥL and 200 ɥL of each sample respectively onto Carbenicilin plates. pUC19 was transformed as a control.
Results of re-suspension of IDT genes:
Highlighted band sizes were extracted.
Gel extraction of digested CP1 and amilCP, amilGFP and spisPink
Repeated digest as the DNA concentration was not satisfactory
-Inoculated CP1, amilCP, amilGFP and spisPink in 10 mL LB broth with its respective antibiotic to obtain more DNA
Transformation of genes from IDT worked. Picked single colonies and inoculated them 10 mL LB broth with the respective antibiotic.
Performed miniprep for the overnight cultures. Results:
Digested products were run on a 1% gel at 120V for 35 minutes. amilGFP and spisPink digest did not work. CP1 and amilCP worked so proceeded to perform gel extraction to obtain the necessary band.
*all the cut bands were loaded into 1 tube
Redid the digest for amilGFP and spispink and control plasmid(given by Marc) using XbaI and PstI-HF. Ran on 1% gel and it worked so proceeded to gel extract. Ligation with CP1.
Concentrations of samples used:
amilGFP - 124.4 ng/ɥL
spispink - 136.5 ng/ɥL
control plasmid - 500 ng/ɥL
*the three bands for each sample that was cut was evenly distributed into 2 tubes
Chemical transformation of control plasmid and pUC19 (control vector) into DH5α competent cells to get more DNA. Plated onto Carbenicillin plates
Inoculated alcA2, alcR and pUC19. Miniprep results:
Made glycerol stocks of alcA2 and alcR using 400 ɥL of 50% sterile glycerol + 600 ɥL inoculated culture and stored in -80°C for future use.
Ligated CP1 with chromoproteins and transformed into DH5α. Transformants were plated onto Carbenicillin plates and left to grow overnight in the 37°C incubator.
Inoculated control plasmid to make glycerol stocks.
Plasmid verification of alcR and alcA 2
Week 9 (22nd to 26th August)
Cell-Free Mechanism
Restriction digest of pUCIDT_AOx for gel extraction with NdeI and SalI, and EcoRI and PstI for validation.
Results:
Figure 3. 1 % TAE agarose gel showing digestion results. Lane 2, pUCIDT_AOx  cut with NdeI and SalI, expected sizes of 2781bp and 2015bp. Lanes 3 and 6, pUCIDT_AOx negative control. Lane 4, pUCIDT_AOx cut with EcoRI, expected size of 4795bp. Lane 5, pUCIDT_AOx cut with PstI, expected size of 4795bp.
Gel extraction of AOx 2017bp band in digested gel.
Restriction digest of pET28b with NdeI and SalI for PCR purification, as the fragment is <100bp.
Figure 4. Lane 2, pET28b cut with NdeI and SalI, expected sizes of around 5310bp. Lane 3, negative control.
PCR purification of pET28b.
Nanodrop results of cut AOX and PCR purified Pet28b (Table 5)
Table 5. Miniprep results using Nanodrop.
Performed ligation of digested and purified AOX and Pet28b.
Table 6. Ligation of pET28b and AOx.
Transformed ligated Pet28b_AOX and pUC19 as a negative control into DH5α cells for each ratio and plated onto LB agar plates with  50 μg/ml of Kanamycin.
Made O/N cultures of 4 colonies.
Prepared glycerol stocks of pET28b_AOx.
Miniprepped pET28b_AOx from O/N cultures.
Table 7. Miniprep results using Nanodrop.
Restriction digest with NdeI and SalI to screen recombinant clones
Figure 4.S1 to S4, pET28b_AOx cut with NdeI and SalI, expected sizes of 5310bp and 2017bp. None of the samples showed expected sizes, suggesting ligation didn't work.
Inducible Gene Switch
Restriction digest & gel extraction for ligationSet A = CP1/2/3 (vector, SpeI/PstI-HF) + alcR (insert, XbaI/PstI-HF/ZraI)
Expected band size (bp):
CP1 = 887 + 2096
CP2 = 18 + 2096
CP3 = 887 + 2096
alcR = 565 + 2204 + 2639
Set B = alcA1/2 (insert, EcoRI-HF/SpeI) + ChP (vector, EcoRI-HF/XbaI)
Expected band size (bp):
alcA1 = 886 + 2047
alcA2 = 785 + 2772
amilCP = 15 + 2742
amilGFP = 15 + 2772
spispink = 15 + 2752
Set C =
(a) RFP (from CP1, insert, SpeI/PstI-HF) + CP2 (pSB1C3, vector, SpeI/PstI-HF)
(b) CP1 (vector, EcoRI-HF/PstI-HF) + ligated set C (a) (EcoRI-HF/PstI-HF)
Final construct = CP2+RFP in BBa backbone
Set D = Positive controls - control plasmid, puc19 vectors
Expected band size (bp):
control plasmid
SpeI/PstI-HF = 89 + 1197 + 3761
XbaI/PstI-HF = 16 + 2622 + 2409
EcoRI-HF/SpeI = 89 + 249 + 4709
EcoRI-HF/XbaI = 932 + 2638 + 1477
Prepared 3 vials of 25 ɥL for each sample
Set A = CP1/2/3 (vector, SpeI/PstI-HF) + alcR (insert, XbaI/PstI-HF/ZraI)
DNA concentration:
CP1 = (a) 83.6ng/ɥL, (b) 323.6ng/ɥL
CP2 = (a) 103.9ng/ɥL, (b) 32.3ng/ɥL, (c) 22.3ng/ɥL
CP3 = 195.4ng/ɥL
alcR = 390.3ng/ɥL
Per vial
Set B = alcA1/2 (insert, EcoRI-HF/SpeI) + ChP (vector, EcoRI-HF/XbaI)
DNA concentration:
alcA 1 = 84.9ng/ɥL
alcA 2 = 743.3ng/ɥL
amilCP = 181.1ng/ɥL
amilGFP = 53.2ng/ɥL
spispink = 230.8ng/ɥL
Set C (b) = CP1 (vector, EcoRI-HF/PstI-HF) + ligated set C (a) (EcoRI-HF/PstI-HF)
Set D = Positive controls - control plasmid, puc19 vectors
Prepared a 10ɥL reaction rather than a 25ɥL
DNA concentration:
control plasmid = 124.6ng/ɥL
SpeI/PstI-HF
XbaI/PstI-HF
EcoRI-HF/SpeI
EcoRI-HF/XbaI
puc19 = 417.5ng/ɥL
EcoRI-HF/ZraI
PstI-HF/ZraI
Inoculated CP1, CP2, BL21 cells
Restriction enzyme digest for alcA1, alcA2, alcR, ChP
Concentrations:
alcA 1 - 84.9 ng/ɥL
alcA 2 - 743.3 ng/ɥL
control plasmid - 124.6 ng/ɥL
Expected band size (bp):
alcA1 = 886 + 2047
alcA2 = 785 + 2772
amilCP = 15 + 2742
amilGFP = 15 + 2772
spispink = 15 + 2752
alcR = 565 + 2204 + 2639
Concentrations:
amilCP - 181.1 ng/ɥL
amilGFP - 53.2 ng/ɥL
spispink - 230.8 ng/ɥL
control plasmid - 124.6 ng/ɥL
Concentrations:
alcR - 390.3 ng/ɥL
pUC19 - 250.0 ng/ɥL
Result: Bands were distorted severely, most likely due to not properly dissolved agarose powder.
Result: Only one band was seen instead of two.  May have missed out adding any of the reagents/problems with the DNA.  Should change a new aliquot of positive control DNA from this point onwards.
Gel extraction of alcA1, alcA2, alcR and digested products
*CP2 to be PCR purified,/p>
PCR purification of amilCP, amilGFP, spispink
Re-did overnight ligation of CP1 with chromoproteins (ChP)
Concentrations:
amilCP - 20.2 ng/ɥL
amilGFP - 10.4 ng/ɥL
spispink - 10.7 ng/ɥL/li>
1:7 ligation DNA mass:
amilCP - 59.53 ng
amilGFP - 62.03 ng
spispink - 60.36 ng/li>
Ligation of alcA 2 with amilCP
Re-did alcA1/alcA2/alcR digest with control plasmid and gel extracted appropriate sizes.
*all samples were digested at 37ç for a Total of 2 hours. For alcR & control plasmid, X/P was added first for 1 hour before adding ZraI and left to digest for another hour.
*All samples were run on a 1% gel.
alcA 1: 886 + 2047
alcA 2: 785 + 2772
alcR : 565 + 2204 + 2639
Result: The correct band sizes were extracted for alcR and gel extraction was performed as per protocol. For alcA2, the wrong band size was extracted so digest would be repeated. alcA1 did not work.
After soaking in ethidium bromide, alcR bands were clearly seen and extracted
Transformed overnight ligated samples (CP 1 + chromoproteins (Carbenicillin), alcA2 + amilCP (CmR) ) into DH5alpha along with a pUC19 (Carbenicillin) positive control using protocol. They were plated into the appropriate Ab plates and SOC medium was used as a negative control.
Ligation of CP1 + alcR and CP2 + alcR using Roche ligation kit and transformed into DH5α. Plated onto Carbenicillin plates, left in 37°C incubator overnight.
Insert: CP1, CP2
Vector: alcR (482.1ng/ɥL)
Repeated alcA 1 & alc A 2 digest using same measurements as before and used control plasmid. alcA 1 did not work again. alcA 2 worked and the correct band size (785) was extracted. Gel extraction protocol was used to obtain the DNA.
CP1 + amilCP ligation product restriction enzyme digest to verify the plasmid. pUC19 was used as a positive control.
Sent alcA 1 for sequencing due to repeated failure in restriction enzyme digest. sequencing results of alcA 1indicate that alcA1 did not contain XbaI / SpeI site.
Inoculated cultures and miniprep results
Validation of miniprep ligated product- alcA 2 + amilCP by restriction enzyme digest and sent for sequencing using VF2 primer
Expected band sizes:
control plasmid
PstI-HF/ZraI: 2867 + 2180
XhoI/PstI-HF: 2820 + 2040 + 187
alcA 2 + amilCP
PstI-HF/ZraI: 1921 + 1475 + 131
XhoI/PstI-HF: 1622 + 1013 + 892
RE digest of all chromoproteins, constitutive promoters, alcA1, alcA2, alcR and control plasmid
Week 10 (29th August to 2nd September)
Cell-Free Mechanism
Colony PCR with T4 Taq polymerase of 16 colonies from last week’s ligation plates.
Table 8. Primers used for Colony PCR.
Results:
Figure 6. 1% TAE gel showing colony PCR of pET28b_AOx, sample 3 and 13 show expected amplified sizes of 2319bp.
Made O/N cultures of sample 3 in colony PCR.
Sample 3 in lane 4 shows a band of around 2300bp. This suggests AOX is present in the sample and thus possibly ligated. The other bands of 350 bp indicates a vector that has re-ligated to itself.
Made O/N of sample 3
Miniprepped pET28b_AOx from O/N cultures.
Digestion of pET28b_AOx with Xbal and BamHI was unsuccessful (No picture available).
Inducible Gene Switch
Important dates:
29th August-First BioBrick ready
Restriction enzyme digest
*ZraI was added to both samples after 1 hour of incubation
"
>br />
PCR purification - CP2, amilCP, amilGFP, spispink
alcA2 + amilCP sequencing results with VF2 primer → alcA2 + amilCP ligation was successful (FIRST BIOBRICK READY!)
Received alcA 1 forward and reverse primers from IDT. Added appropriate amounts of milli-Q as instructed by IDT to make a stock concentration of 100 ɥM. Did a dilution to make aliquots of 10 ɥM of primers to use for PCR.
Q5 PCR to insert XbaI/SpeI in alcA1. Also used an a negative control (empty vector from previous day’s ligation)  → failed as too much dNTP was used resulting in Mg2+ depletion
alcA1 - 349.6 ng/ɥL
PCR reaction conditions:
Restriction Enzyme digest → Gel extraction →  overnight ligation of:
Insert CP3 (S/P) + Vector amilCP (X/P)
Insert CP3 (S/P) + Vector amilGFP (X/P)
Insert CP3 (S/P) + Vector spispink (X/P)
Insert CP3 (S/P) + Vector alcR (X/P/Z)
Controls:
control plasmid (S/P) & control plasmid (X/P/Z)
Concentrations:
CP3 = 526.9 ng/µL
amilCP = 146.2 ng/µL
amilGFP = 273.5 ng/µL
spispink = 235.3 ng/µL
alcR = 855. 0 ng/µL
control plasmid= 1074.4 ng/µL
*for alcR, ZraI was added after 1 hour. Total digest time = 2 hours in a 37°C water bath.
Overnight ligation in 4°C cold room
noculated cultures of ligated products from 30th Aug using colonies from plate in Carbenicillin + LB. Miniprep results:
CloneAmp PCR to insert XbaI/SpeI in alcA1 as Q5 PCR has been failing.
alcA1 - 366.0 ng/ɥL
Ran CloneAmp products (alcA1 + XbaI/SpeI)  on a 1 % gel → PCR was successful so proceeded to PCR purify the product
Restriction digest of miniprep ligation product in to verify plasmid
Restriction digest of PCR purified alcA1+ XbaI/SpeI with EcoRI-HF and SpeI-HF → Gel extract (866bp band) for ligation with spispink  (E/X)
Gel extraction was not successful. Repeat the PCR
* bottom half of the gel is supposed to be verification of ligated products but there was something wrong with the gel and hence nothing could be seen. Re-do digest on Monday to verify the ligated product.
Heat inactivation of overnight ligation product at 65C for 10 minutes. They were then transformed into DH5α competent cells and plated on Carbenicillin and left on bench over the weekend.
Transformed CP1 + alcR and CP2 + alcR into BL21 competent cells. Plated onto Carbenicillin and left on bench over the weekend.
Week 11 (5th to 9th September)
Cell-Free Mechanism
More colonies were screened through colony PCR from ligation plates.
Results:
Figure 7.1% TAE agarose gel showing colony PCR of pET28b_AOx. S7 and S15 showed correct amplified size of 2044bp.
Made overnight cultures of colonies 7 and 15 in Kanamycin and miniprep.
Restriction digest with NdeI and SalI of both samples.
Result from Sample 7:
Figure 8.1% TAE agarose gel showing digestion results for sample 7. Lanes 2 and 4, pET28b_AOx cut with NdeI and SalI, expected sizes of 5310bp and 2017bp. Lane 3 and 5, negative control.
Result from Sample 15:
Figure 9.1% TAE agarose gel showing digestion results for sample 15. Lanes 2 and 3, pET28b_AOx cut with NdeI and SalI, expected sizes of 5310bp and 2017bp. Lane 4 and 5, negative control.
Transformed samples and pUC19 into Escherichia coli DH5α strain. pUC19 was used as a control.
Plated transformed cells onto LB agar plates with 50 µg/mL Kanamycin.
Inducible Gene Switch
Q5 PCR to insert XbaI/SpeI to alcA1 →Run on 1% gel →  PCR purify → restriction enzyme digest with EcoRI/SpeI → gel extraction → ligation with spispink (E/X) overnight
alcA1 - 366.0 ng/µL, amplified fragment side = 886 bp
PCR purification of alcA1 using protocol
Restriction enzyme digest of PCR purified alcA1  with EcoRI-HF and SpeI. Digested at 37°C in PCR machine for 1 hour.
Band expected: alcA1 = 886bp, contorl plasmid= 4709 + 89 + 249
Digest failed so repeated it again. Repated digest worked, so proceeded to gel extract the fragment.
Gel extraction of alcA1 digested with E/S using protocol.
 overnight ligation of alcA1 (E/S) with spispink (E/X) in 4°C (cold room)
Insert: alcA1 (E/S) - 886 bp, 92.4 ng/ɥL
Vector: spispink (E/X) - 2752 bp, 37.8 ng/ɥL
Restriction digest to verify ligated products  - CP1 + alCR, CP2 + alcR, CP2 + rfp
Results: No bands were seen except for control plasmid. Re-made overnight cultures to miniprep and validate again.
New ligation of CP + alcR and CP + Chromoproteins. Ligation was done using Roche kit for 30 mins on ice. This time transformed into Marc’s competent cells (to eliminate the possibility of contaminated competent cells) and plated onto Carbenicillin plates, left overnight.
Ratio - 1:1
Ratio - 1:3
Colony PCR (with VF2 and VR primers)
Mastermix:
Made overnight cultures of the correct samples
Glycerol stock and miniprep overnight cultures - alcA1 + spispink
Digest alcA1 + spispink with EcoRV-HF and SphI-HF to verify plasmid. alcR used as control
Sent alcA1 + XbaI/SpeI that was done using PCR for sequencing
RE digest for CP, alcR and ChP for ligation
Samples:
Eg: Vector + insert
CP1 (S/P) 2096bp + alcR (X/P/Z) 2639bp
CP2 (S/P) 2096bp + alcR (X/P/Z) 2639bp
CP3 (S/P) 2096bp + alcR (X/P/Z) 2639bp
CP1 (S/P) 2096bp + amilCP (X/P) 713bp
CP2 (S/P) 2096bp + amilGFP (X/P) 743bp
CP3 (S/P) 2096bp + spispink (X/P) 723bp
Ligation
Conc. of alcR used = 31.4 ng/ɥL. Concentrations of the other DNA used are in G) below.
Ratio 1:1
Ratio 1:3
Ratio 1:5
Controls
Gel extraction- Nanodrop concentration
Week 12 (12th to 16th September)
Cell-Free Mechanism
Sent sample 15 for sequencing.
Made overnight cultures of plated samples 7 and 15 pET28b_AOx from last week.
Transformed samples 7 and 15 of pET28b_AOx into Escherichia coli DH5α and BL21 strains and plate them onto LB agar plates.
Growth of cultures in 250 ul flasks, containing LB broth, of pET28b_AOx in Escherichia coli BL21 strain, empty vector pET28b and pBbESK_RFP (controls). Grown until reaching an O.D of about 0.5-0.7.
Induction of pET28b_AOx in BL21 E.coli cells with 0.5 mM IPTG. Controls were also IPTG induced: Empty vector pET28b was used as a negative control. pBBESK_RFP was used as a positive control.
Stored cultures in -20ºC overnight.
Simultaneous to the above work, we started working on assembling our part to the pSB1C3 backbone:
Restriction digest of pUCIDT_AOx with EcoRI and PstI for gel extraction.
Results:
Figure 10. 0.8% TAE agarose gel showing digestion results. Lane 2, pUCIDT_AOx cut with EcoRI and PstI, expected sizes of 2760 bp and 2036 bp. Picture made after gel extraction of 2036 bp band.
Gel extraction of band at 2036 bp corresponding to AOx.
Restriction digest of pSB1C3 from iGEM with EcoRI and PstI.
Ligation of pSB1C3 and AOx using Roche Kit.
Table 9. Ligation of pSB1C3 and AOx.
Transformation of ligation of pSB1C3_AOx and plated onto LB agar plates with chloramphenicol.
Colony PCR of four colonies.
Results:
Figure 11.Colony PCR of pSB1C3_AOx. S1 and S4 show correct amplified size of around 2000bp.
Restriction digest of pSB1C3_AOX with BamHI and PstI for characterisation.
Results:
Figure 12. 1% agarose gel showing digestion results. Lane 3 shows pSB1C3_AOx cut with BamHI and PstI showing correct sizes of 3355bp and 710bp.
Inducible Gene Switch
Colony PCR
Mastermix:
Miniprep CP1 + alcR (1:3)
RE digest to confirm CP1 + alcR (1:3) ligation was successful
Did not perform as the DNA concentration was very low
Inoculated ligated products. Miniprep results:
Ligation of CP1 and ChP
1:3
1:5
Controls
Transformed ligated products
RE digest for verification
RE digest for ligation
Something was wrong with the gel. Will re-run gel the next day. However, we loaded all of the control plasmid  so there will be none for the re-run.
Made DH5α competent cells
Ran gel for digested products
Both fragments of CP3 were gel extracted.
Smaller fragment: 887 bp = rfp
Bigger fragment: 2096 bp = CP3 vector  Since the CP2 S/P/A control shows that the digest has worked, CP2 was PCR purified to lose the 18bp fragment and retain the CP2 vector.
PCR purification of CP2 (pSB1C3) S/P
Gel extraction of CP3 fragments (vector and rfp)
Before performing the CP2 + rfp ligation, we ran the PCR purified vector and gel extracted rfp on a gel just to confirm that the DNA is present.
The correct band sizes are seen. Proceeded to perform CP2 + rfp ligation.
alcR restriction enzyme digest with XbaI/PstI-HF. No ZraI could be found, so we ran a 0.6% gel for a really long time hoping to be able to get the separation of the 2 bands.
Separation of bands was not seen. Repeated digest with PvuI obtained from another lab.
CP2 (vector) + rfp (insert)  ligation
The ligation was left in the PCR machine at 24°C for 2 hours. It was then heat inactivated at 65C for 10 minutes and then transformed into DH5alpha. They were plated onto Carbenicillin plates.
Tested our new DH5alpha competent cells and BL21 cells by transforming them with pUC19 and plated onto Carbenicillin. Left overnight at 37°C.
Ran gel for alcR digest from 14th Sept (F). We did not have SybrSafe, so left the gel in Ethidium Bromide for half an hour before visualising gel.
Proceeded to gel extract the appropriate alcR fragment.
Gel extraction of alcR
Restriction digest of alcA1, alcA2 and alcR to put into pSB1C3 submission vector.
Samples were left in PCR machine at 37°C for 1.5 hours. ZraI was added after 45 minutes.
Gel extraction of fragments from digest
Submission vector ligation using fragments from D and pSB1C3 vector that has been digested with EcoRI-HF and PstI-HF by Mechanism 1 team.
Concentration of pSB1C3 = 11.1 ng/ɥL
Samples were left on the bench for 1 hour before transforming into Dh5α and plated onto Cm plates. Left in 37°C incubator.
Restriction digest of old alcA2 + amilCP product for verification
Samples were left in the PCR machine at 37°C for 1.5 hours. It was then run on a 1% gel.
Ligation : CP3 + chromoproteins & CP + alcR
Controls
Transformed ligated product onto appropriate plates
Made 5ml overnight culture for rfp quantification for CP1 and CP3 in the Bba_J61002 vector that contains rfp. Grown in 5ml LB + CmR (50 ɥg/ml). Also grew pBbB2K (kanamycin, ATC inducible) Kiesling vector as a control with rfp.
alcA1+spispink restriction enzyme digest to verify the plasmid
alcA1 + spispink is not the correct product.
spispink restriction digest to obtain pSB1C3 vector.
The larger fragment was extracted and proceeded to purify as per protocol.
Colony PCR
Mastermix:
RFP quantification for CP1 and CP3 using the overnight culture.A kiesling vector - pBb2K (Kan) was used as a control that can be induced by aTC. 50nM of aTC was added to the induced wells during mid-way log growth. RFP was measured for 24 hours.
Stock concentration of aTC = 10mM
Concentration needed to add to 200ɥL cells in plate = 50nM
A 1:1000 dilution of the stock concentration with 100% ethanol was made and a final concentration of 1mM was obtained. 1ɥL of aTC was added to each well. Important to note that aTC is light sensitive and hence the tube has to be wrapped in aluminium foil. OD 600 and RFP were measured every 30 mins for 24 hours. The cells were shaken at 500 rpm and at 37 °C.
Week 13 (19th to 23rd September)
Cell-Free Mechanism
Made soluble and insoluble fraction proteins of induced cultures.
Performed a 12% SDS-PAGE gel of soluble and insoluble fraction of protein samples. Bands in the gel were so faint we decided to repeat it.
Repeat 12% SDS-PAGE gel with optimized volume of proteins.
Table 10. 12% SDS-PAGE gel with optimized volumes of protein.
Results:
Figure 12. SDS-PAGE for protein expression. Lane 1: Soluble IPTG induced pET28b-AOx in BL21 cells. Lane 2: Soluble un-induced pET28b-AOx in BL21 cells. Lane 3: Soluble IPTG induced empty pET28b vector in DH5α cells. Lane 4; Soluble un-induced empty pET28b vector in DH5α cells. Lane 5: Soluble IPTG induced pBbESK_RFP vector showing RFP (27 kDa). Lane 6: Protein Standard. Lane 7: Insoluble IPTG induced pET28b-AOx in BL21 cells. Lane 8: Insoluble un-induced pET28b-AOx in BL21 cells. Lane 9: Insoluble IPTG induced empty pET28b vector in DH5α cells. Lane 10: Insoluble un-induced empty pET28b vector in DH5α cells. Lane 11; Insoluble IPTG induced pBbESK_RFP.
Performed a Western Blot of transferred SDS-PAGE gel to membrane binding anti-His antibodies (sourced from Sigma-Aldrich).
Results:
Figure 13. Western Blot results. Only the insoluble fraction of pET28b_AOx IPTG induced is visible (lane 7), showing two major regions at around 80 kDa and 35 kDa.
Sent pSB1C3_AOx for sequencing to check that that AOx had been correctly ligated into the submission vector.
Inducible Gene Switch
Miniprep results for ligated products
RE digest for verification of ligated products
Inoculated CP1, CP3, CP3 + amilCP, CP3 + spispink, Kiesling vector
overnight ligation into pSB1C3- alcA 1, alcR, alcA 2
Vector: pSB1C3 - 31.6 ng/ɥL
Insert(all cut with E/P): alcA 1 + X/S - 46.4 ng/ɥL, alcR - 42.2 ng/ɥL, alcA 2 - 29.5 ng/ɥL
1:1
1:3
Controls
RE digest to re-verify CP3 + alcR
Lane order: Ladder, A(P/B), B(P/B), C(P/B), control plasmid (P/B), A(E/P), B(E/P), C(E/P), amilCP(E/P), amilCP(E/P), Ladder
Protein expression overnight culture- 5 mL culture, CARB 100, 30℃ incubator. 2 replicates of CP3+alcR, CP3+rfp and BL21 were grown.
Chromoprotein comparison - the same amount of cells were added onto a plate and the time taken for a visible colour change was seen.
Made overnight culture for rfp quantification - CP1, CP3 and Kiesling vector pBbB2K
Submission vector ligation -  alcA2, alcA1 and alcR into pSB1C3. Ligation was done in the PCR machine at 24°C for 2 hours. Heat inactivation at 65C for 10 minutes then transformed into DH5alpha and plated onto Carbenicillin. Plates were left at 37°C, overnight.
1:1
1:3
Controls
Restriction digest to obtain pSB1C3 backbone using spispink. Digest was run on a PCR machine at 37°C for 1.5 hours and then viewed on a 1% gel . since the bands were correct, we proceeded to gel extract the larger fragment (pSB1C3 backbone).
Protein expression
Made a 1:1000 dilution of the overnight cultures in a Total volume of 25ml (in a 250 mL Flask) = 24.75 mL LB + 0.25mL culture *sterile conditions, no antibiotic
The culture was shaken at 37°C and the OD was measured at several timepoints. Some of the cultures needed dilutions to be made to enable OD 600 measurements to be taken.
At each OD  reading, 500ɥL of the culture was taken and placed into a 2mL tube. It was spun down at 12 000rpm, 4°C for 10 minutes. The supernatant was discarded and pellet was kept. 50 ɥL of 2X SDS-PAGE dye was used to resuspend the pellet and transferred into a PCR tube. The samples were boiled at 99°C in the PCR machine for 20 minutes. All the samples were kept in a 4°C fridge until ready to be loaded onto the SDS gel.  SDS gel was run at 300V for 20 minutes.
The remaining culture was transferred to a falcon tube and spun down at 10000 rpm , 4°C for 20 minutes. Supernatant was discarded and the pellet was stored in the -20°C for future use.
CP rfp quantification
RFP quantification for CP1 and CP3 using the overnight culture.A kiesling vector - pBb2K (Kan) was used as a control that can be induced by aTC. 1ɥL of 1mM  aTC was added to the induced wells during mid-way log growth. RFP was measured for 24 hours. OD 600 and RFP were measured every 30 mins for 24 hours. The cells were shaken at 500 rpm and at 37 C.
SDS-PAGE did not show presence of protein. Repeated SDS-PAGE but this time using stored pelleted cells. Cells were first separated into soluble and insoluble fractions.
Pellets were first resuspended in 500ɥL Bugbuster (must be at room temperature) and transferred to a 2ml tube. They were then shaken on a rocker for 20mins before being spun down at 12 000 rpm at 4°C for 20 mins. Supernatant which contains the soluble fraction was transferred to a fresh 2ml tube.
Pellets containing the insoluble fraction were resuspended in 200ɥL Buffer SB1. In a PCR tube, 0.3ɥL of the resuspended pellet + 0.5ɥL DTT + 5ɥL 2X dye + 4.2ɥL milli-Q were added. They were then boiled on the PCR machine at 98°C for 7 minutes. Supernatant containing the soluble fraction : In a PCR tube, 2.5ɥL of the supernatant + 0.5ɥL DTT + 5ɥL 2X dye + 2ɥL milli-Q were added. They were then boiled on the PCR machine at 98°C for 7 minutes. Both the pellet and supernatant fraction samples were then loaded onto a gel and run at 300V for 25-30minutes (4 gels were running at the same time. If only 2 gels, 20 minutes is enough). alcR size = 97.7kDa, hence a pre-made 10% gel was used.
Rfp quantification was repeated.
RFP quantification for CP1 and CP3 using the overnight culture.A kiesling vector - pBb2K (Kan) was used as a control that can be induced by aTC. 1ɥL of 1mM  aTC was added to the induced wells during mid-way log growth. RFP was measured for 24 hours. OD 600 and RFP were measured every 30 mins for 24 hours. The cells were shaken at 500 rpm and at 37 C.
colony PCR to check pSB1C3 + alcR/alcA1/alcA2 transformed and plated on 21st August.
Mastermix:
Co-transformation of CP3+alcR and alcA2+amilCP
Ran a 1% gel for colony PCR
Co-transformed colonies
Picked 2 colonies from each plate, grow in 3mL LB with Carbenicillin and Chloramphenicol antibiotics to make cultures.
Carb50 3ɥL
CmR25 2.24ɥL
Carb25+Cmr12.5 1.5ɥL Carb + 1.12ɥL CmR
Incubate in 37°C shaker
Stop the incubation when cultures reach approximately OD=0.1
Make a 1:100 dilution for the culture with lowest OD
Dilute other cultures accordingly to normalize all ODs (in reference to the lowest OD culture)
Add different concentrations of ethanol to the cultures
Measure absorbance of chromoproteins using a plate reader
RE digest for composite parts (CP3+alcR, CP3+amilCP, CP3+spispink) using EcoRI/PstI
Result = All band sizes were correct exceptCP3+alcR. Gel extracted the correct fragments
Week 14 (26th to 30th September)
Cell-Free Mechanism
Repeated induction of cells with IPTG, SDS-PAGE and western Blot from last week
Western Blot still shows correct protein in the insoluble fraction.
Inducible Gene Switch
EcoRI/PStI digest for CP3 + alcR
Made cultures for co-transformation colonies with half antibiotic concentration until samples reached an OD of 0.6. Diluted the samples to normalise the OD before putting samples into the plate reader for comparison.
Re-did RE digest for verification of CP + alcR with SphI-HF and AhdI
Expected band sizes:
CP + alcR - 2472 + 1580 + 683
control plasmid - 1254 + 2073 + 172
Ran co-transformed colonies in plate reader. Normalise OD of samples in the plate reader until it reaches an OD of 0.6. Then, induce cells with ethanol.
Redid RE digest of CP3 +alcR with EcorI and PstI
Gel extraction-not done as the band sizes for CP3 + alcR were not right. The control worked fine.
Miniprep overnight cultures of alcA 1, alcA 2, alcR in submission vector
Miniprep overnight cultures of alcA 1, alcA 2, alcR in submission vector
Miniprep overnight cultures of alcA 1, alcA 2, alcR in submission vector
Note: Concentration/ratios were weird for miniprep products, proceed to RE digest for verification regardless
Absorbance measurement using plate reader (OD600, OD588, OD540)
RE digest for CP3+alcR_pSB1C3 with EcoRI/PstI/PvuI-HF
pSB1C3 verification using PstI/ZraI
Expected band size:
alcR - 2762+1921
alcA1 - 1921+861+131
alcA2 - 1921+780+131
control plasmid - 2867+2180
Week 15 (3rd to 7th October)
Cell-Free Mechanism
Typed up lab work for this mechanism
Prepared PSB1C3_AOx to be sent off to iGEM HQ
Inducible Gene Switch
Repeat ligation for 30/9 and transformed them following protocol
1:1
1:3
Controls
Prepared overnight cultures for rfp quantification and ethanol induction in co-transformants
Colony PCR
Expected band sizes:
Ligation
Vector concentration: pSB1C3 - 61.8 ng/ɥL
Insert concentration:
alcR (E/P) - 42.2 ng/ɥL
CP3 + alcR - 42.0 ng/ɥL
1:1
1:3
Controls
Transformation of ligation following protocols
RE digest
Band sizes: CP2 (pSB1C3) - 2029 + 76
Week 16 (10th to 14th October)
Inducible Gene Switch
Important dates:
11th October- Sent alcA 2 + amilCP ( 1st composite part) to iGEM HQ
Inoculated cultures. Miniprep results:
RE digest
Expected band sizes: alcR- 2654 + 1654 + 896 + 163 + 41
Gel extraction
Ligation
Vector concentration: pSB1C3 - 134.8 ng/ɥL
Insert concentration: alcA 1- 79.1 ng/ɥL
:alcA 2 - 18.5 ng/ɥL
:CP3 + amilCP - 18.1 ng/ɥL
1:1
1:3
1:5
Controls
Miniprep O/N cultures
Completed gel extraction procedure. Tube containing gel was stored at 4℃ room
Submission vector ligation
Vector: pSB1C3 - 99.3 ng/ɥL
Insert: alcR - 37.5 ng/ɥL
CP3 + alcR -328.6 ng/ɥL
1:1
1:3
1:5
Controls
Miniprep submission vector colonies
Top ^
Sponsors
Follow us on
Email: uomigem16@gmail.com
