Team:McMasterU/CDifficile
HOME
WET LAB
Project Description
DNAzyme Plate Experiment
C. Difficile Project
InterLab
Results
DRY LAB
Image Processing
Enzyme Model
SELEX Optimization with Machine Learning
HUMAN PRACTICES
Silver Medal
Integrated Human Practices
Public Engagement
Collaboration
TEAM
Members
Sponsors
Attributions
Introduction
We are planning to generate a cis-acting DNAzyme using in vitro selection for the detection of pathogenic strains of C. difficile. This DNAzyme will be enriched from a combinatorial sequence library of around 10^15 single-stranded DNA with central random sequences. The DNAzyme will be selected using the 61-aa (NAP1) and 65-aa (NAP7) truncated variants of C. difficile TcdC (expressed in E. coli) as the target. This target was chosen because of its low identify ratio between wild type and pathogenic strains of C. difficile. An oligonucleotide containing a fluorophore-riboadenosine-quencher motif will be ligated to the random single-stranded DNA library. DNAzymes will be selected from the library based on their ability to cleave the riboadenosine in the presence of the TcdC variants specified above. Negative selection of the library will be performed sequentially in the presence of E. coli, wild type TcdC, and B. subtilis CEM. We have cloned NAP1, NAP7, and wild type TcdC into pET15 plasmids and are currently working on expressing the genes.
Plasmid Design
Green light inducible plasmid
Red light inducible plasmid
Chromophore producing plasmid
Project Design
In-Vitro Selection
Genetic evolution by natural selection has guided life as we know it through billions of years of Earth’s harshest environments, giving rise to millions of diverse and steadfast lifeforms that we see around us today.  In the lab, we can mimic this grandiose process to our advantage on a microscale using a technique called in-vitro selection1.  By following Darwin’s principles of natural selection in a controlled environment, it is possible to artificially evolve large groups of DNA molecules from randomness and select for those with useful functions.  This is done by exposing large, “random libraries” of short DNA strands to other molecules of interest, and removing the species in the pool that don’t react to them in the way we desire.  These reactions are observed on the macroscale using the tried and true technique of polyacrylamide gel electrophoresis on the radio-labelled, denatured (untied and linearized) DNA libraries after exposure.  In the end, an assortment of DNA molecules comprising of more than a quintillion random nucleotide sequences is reduced to a handful of highly specific, functional product species.
If selected successfully to process a substrate on que, these nucleic acids are called DNAzymes.  We work with DNAzymes that have been artificially selected to serve as a detection platform for E. coli, one of the most infamous antimicrobial drug-resistant strains of bacteria in the world2,3.  Our DNAzymes react in the presence of a cocktail of molecules radiated specifically from the extracellular matrix of the E. coli bacterium.  The ensuing process results in the cleavage of an RNA unit bridging a quencher-fluorophore DNA complex, resulting in the emission of a signature green glow.  In essence, we’ve used one of nature’s oldest solutions to generate something innovative, creative, and potentially revolutionary to the field of medical point-of-care testing.
Compiled Protocols for TcdC Plasmids Design
I. C. Difficile gDNA Extraction Using Phenolchloroform
Materials
Resuspension Buffer
10 mM Tris-HCl, pH 8.0
20% (w/v) Sucrose
Lysis Buffer
10 mM Tris-HCl, pH 8.0
1mM EDTA
1% (w/v) SDS
RNase A (10mg/mL)
10 mg bovine pancreating RNase A
10 mM Tris, pH 8.0
500 mM NaCl
850 uL water
Heated to 95 C until use
Protocol
10 mL of overnight culture was pelleted at 5000 rpm for 5 minutes
Pellets were resuspended in 1 mL of resuspension buffer.
9 mL of lysis buffer was added to resuspended pellet.
Cell suspension was incubated at 37 C for 1 hour, shaking at 260 rpm.
Crude cell lysate was extracted with an equal volume of phenol chloroform.
Aqueous phase was recovered to a new tube and 25 ug RNase A was added, crude lysate was incubated at 37 C for 20 minutes.
Crude lysate was extracted with equal volume of phenolchloroform, aqueous phase removed to a new tube.
DNA was precipitated using ethanol precipitation. (see Other Protocols)
DNA was dissolved in 1.5 mL nuclease free water
II. PCR Amplification of BAA-1870, BAA-1875, and WT TcdC
Protocol
Primers were diluted to 10 uM
dNTP was diluted to 10 mM
DNA samples were diluted to 125 ng/uL
The following master mix solution was prepared
Reagent
1x(uL)
Forward Primer
2.5
Reverse Primer
2.5
dNTP
1
Phusion Buffer
10
Phusion High Fidelity Polymerase
0.5
Nuclease-free water
31.5
2 uL of DNA was added to 48 uL of the master mix
PCR reaction was run in a thermocycler using the following conditions for 35 cycles:
Step
Temperature (C)
Time
Initial Denaturing
98
10 minutes
Denaturing
98
30 seconds
Annealing
50
20 seconds
Elongation
72
20 seconds
Final elongation
72
10 minutes
Hold
4
Indefinite
The PCR reactions were run on a 1% agarose gel and purified using gel extraction (see Other Protocols)
Results:
Figure 1: PCR Amplification of NAP1, NAP7, and WT TcdC
III: Digestion of TcdC and pET15b
The following reaction was set up:
Component
50 uL Reaction
DNA
1 ug
FastDigest Buffer
5 uL
Ndel
1.0 uL
BamHI
1.0 uL
Nuclease Free Water
To 50 uL
Reactions were incubated at 37 Celsius for one hour
Digestion reaction were run on a 1% agarose gel and purified using gel extraction (see Other Protocols)
Digestion of TcdC and pET15b
Protocol
The following reaction was set up:
Component
50 uL Reaction
DNA
1 ug
FastDigest Buffer
5 uL
Ndel
1.0 uL
BamHI
1.0 uL
Nuclease free water
To 50 uL
Reactions were incubated at 37 degrees Celsius for one hour
Digestion reaction were run on a 1% agarose gel and purified using gel extraction (see Other Protocols)
Ligation of TcdC to pET15b
Protocol
The following reaction was set up:
Component
20 uL Reaction
10X T4 Ligase Buffer
2 uL
Vector DNA
70.54 ng (0.02 pmol)
Insert DNA
25.81 ng (0.06 pmol)
T4 DNA Ligase
1 uL
Nuclease-free water
To 20 uL
The ligation reactions were incubated at 16 degrees Celsius overnight.
V. Transformation Using Chemically Competent Cells
Protocol
Invitrogen DH5 alpha cells were thawed on ice for 10 minutes, 50 uL of cells were pipetted into transformation tube
5 uL of ligation reaction was added into cell mixture
Mixture was incubated on ice for 30 minutes
Mixture was heat shocked at 42 C for 30 seconds
Mixture was transferred to ice for 5 minutes.
950 uL of LB broth was added to mixture
Mixture was incubated at 37 C for 60 minutes, shaking at 250 rpm
Selection plates containing 50 ug/mL of ampicillin were warmed to 37 C
Cells were pelleted at 4000 g for 1 minute
900 uL of LB broth was removed
Cells were plated onto selection plates and incubated for 16 hours at 37 C
Colonies were picked and grown in 5 mL of LB broth for 16-18 hours
VI. Miniprep Plasmids
Promega Miniprep Kit was used.
1.5 mL of liquid culture was centrifuged at maximum speed for 30 seconds. Supernatant was discarded
An additional 1.5 mL of bacterial culture was added to the same tube and step 1 was repeated.
600 uL of nuclease-free water was used to resuspend the cells
100 uL of cell lysis buffer was added and mixed by inverting the tube 6 times
350 uL of cold neutralization solution was added
Mixture was centrifuged at maximum speed for 3 minutes
Supernatant (~900 uL) was transferred to a minicolumn without disturbing the cell debris pellet and centrifuged at maximum
speed for 15 seconds. Supernatant was discarded.
200 uL of endotoxin removal wash was added to the minicolumn. Minicolumn was centrifuged at maximum speed for 15 seconds
400 uL of column wash solution was added to the minicolumn and centrifuged for 30 seconds
Minicolumn was transferred to a clean 1.5 mL Eppendorf tube and 30 uL of elution buffer was added
After 1 minute, tubes were centrifuged for 15 seconds to elute the plasmid DNA.
Other Protocols
Ethanol Precipitation
Add 0.1 volumes of 3M NaOAc, 2.5 volumes of ice cold 100% ethanol, and mix
Precipitate at -20 C for at least one hour
Centrifuge at 15000 rpm at 4 C for 30 minutes
Wash pellet twice with 0.5 mL of ice cold 75% ethanol, spinning at 4 C for 10 minutes each time
Remove ethanol and dry the pellet in a Speed-Vac
Resuspend pellet in nuclease free water
Agarose Gel Electrophoresis
Dilute 50X TAE to 1X
Add 1% mass of agarose to TAE
Microwave until agarose has dissolved
Cool to around 50 C, then add SYBR Safe (1 uL for every 10 mL of TAE)
Pour gel into casting tray with a comb
Once gel has solidified, place the gel into a gel electrophoresis machine and add 1X TAE until the gel is fully covered
Load 5 uL of 1kb DNA ladder into the first well and DNA samples into subsequent wells
Run the gel for about 45 minutes
DNA Extraction from Agarose Gels
Nucleospin Gel and PCR Clean-up Kit was used
Use a clean razor blade to excise DNA fragment from gel
For every 100 mg of gel, add 200 uL of Buffer NTI (gel solubilization solution)
Place a binding column into a collection tube and add 700 uL of sample. Centrifuge for 30 seconds at 11,000 g. Discard flow through. Repeat if necessary
Add 700 uL of Buffer NT3 (wash buffer). Centrifuge for 30 seconds at 11,000 g. Discard flow-through. Repeat this step to minimize chaotropic salt carry-over.
Centrifuge for 1 minute at 11,000 g to remove Buffer NT3 completely. To remove residual ethanol, incubate the columns for 5 minutes at 70 C prior to elution
Place the column into a new 1.5 mL Eppendorf tube and add 30 uL of nuclease-free water. Incubate for 1 minute before centrifuging for 1 minute at 11,000 g.
Parts
<groupparts>iGEM17 McMasterU</groupparts>
[1] Wilson, D. S. & Szostak, J. W. In Vitro Selection of Functional Nucleic Acids. Annual Review of Biochemistry 68, 611–647 (1999).
[2] Aguirre, S., Ali, M., Salena, B. & Li, Y. A Sensitive DNA Enzyme-Based Fluorescent Assay for Bacterial Detection. Biomolecules 3, 563–577 (2013).
[3] Antimicrobial resistance: global report on surveillance. (World Health Organization, 2014).
Follow Us
© 2017 mGEM
