Team:McMasterU/Methodologies
HOME
WET LAB
Project Description
DNAzyme Plate Experiment
C. Difficile Project
InterLab
Results
DRY LAB
Image Processing
Enzyme Model
SELEX Optimization with Machine Learning
HUMAN PRACTICES
Silver Medal
Integrated Human Practices
Public Engagement
Collaboration
TEAM
Members
Sponsors
Attributions
DNA
The majority of the DNA was provided by iGEM, either on the Plates 1-5 or by order. These included:
1C3-pbla
T4 holin
T4 endolysin
cph8
PompC-GFP
pbad/araC
ho1
PcyA
1A3 backbone
1K3 backbone
1C3 backbone
1S3 backbone
The light sensing genes were isolated via PCR from a cassette purchased from Addgene. This cassette included the following genes:
Ccas
Ccar
Pcpcg2
PCR
Because the DNA from the cassette, when isolated via PCR, would not have the usable cut sites (E X, and S P), the primers were designed with overhangs. Annealing temperatures were calculated in two stages, using the Warren A. Kibbe OligoCalc (Kibbe WA. 'OligoCalc: an online oligonucleotide properties calculator'. (2007)  Nucleic Acids Res. 35(webserver issue): May 25.). The first stage only included the section of the forward and reverse primers that connected to the DNA in the cassette. The second stage included the entire length of the primers, as the overhangs had been formed.
Ccar PCR Thermocycle:
95°C for 2min
95°C for 30sec
45°C for 30sec
72°C for 1min
Repeat a-d for for 15 cycles
95°C for 30sec
56°C for 30sec
72°C for 1min
Repeat f-i for 15 cycles
4°C indefinitely
Ccas PCR Thermocycle:
95°C for 2min
95°C for 30sec
45°C for 30sec
72°C for 2:30min
Repeat a-d for for 15 cycles
95°C for 30sec
58°C for 30sec
72°C for 2:30min
Repeat f-i for 15 cycles
4°C indefinitely
Pcpcg2 PCR Thermocycle:
95°C for 2min
95°C for 30sec
43°C for 30sec
72°C for 45sec
Repeat a-d for for 15 cycles
95°C for 30sec
57°C for 30sec
72°C for 45sec
Repeat f-i for 15 cycles
4°C indefinitely
Cells
E. coli DH5α cells (Life Technologies / Invitrogen) were used for the entire transformation process.
Buffers and Enzymes for Digestion and Ligation
Buffers and enzymes (New England Biolabs):
CutSmart
NEBuffer 3.1
EcoRI (E)
Xbal (X)
SpeI (S)
PstI (P)
T4 DNA ligase
Through parallel tests of all enzymes in all buffers, we found that CutSmart was the best buffer for all enzymatic digestions.
Mini-Prep
Mini-prep materials were bought from Life Technologies / Invitrogen. Mini-prep procedure followed standard protocol, involving centrifugation of cell samples into columns, and removal of supernatant. Mini-prep was followed by gel extraction.
Lab Equipment and Materials for Cell Cultures
 Materials including gel dyes, Red Safe, SOC and LB media, and antibiotics (Ampicillin, Chloramphenicol, Kanamycin, and Streptomycin) were bought from Life Technologies / Invitrogen, and other materials and lab equipment were graciously provided by the allure. and McMaster undergraduate Cell Biology lab.
Gel Visualization and Gel Extraction
Gels were made using 1% agarose and Red Safe for visualization. Gel visualization was performed using the BioRad Gel Doc EZ Reader. Gel extraction was completed using standard razor blades on a UV light box.
Digestion and Ligation
The 3A assembly method was used to assemble the plasmids, as recommended by iGEM HQ, and is represented in the figure below:
Diagram representing the 3A Assembly. Source: iGEM.
Growth and Transformation
Bacterial growth was accomplished in LB either on agar plates or tubes, with their respective antibiotics. Transformations were accomplished by the standard heat-shock method at 85°C.
Sequencing of gel-verified and transformed plasmids
Sequencing was completed by the MOBIX Lab at McMaster University.
Follow Us
© 2017 mGEM
