<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_McMasterU_Genetic skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:McMasterU/Genetic</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="mainContainer"><DIV id="menuContainer" align="center"><UL><LI><A href="https://2017.igem.org/Team:McMasterU">HOME</A></LI><LI><A href="#">WET LAB
            					</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/Description">Project Description</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/DNAzyme">DNAzyme Plate Experiment</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/CDifficile">C. Difficile Project</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/Demonstrate">Results</A></LI></UL><LI><A href="#">DRY LAB
            					</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/ImageProcessing">Image Processing</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/Model">Enzyme Model</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/Genetic">SELEX Optimization with Machine Learning</A></LI><LI><A href="#">HUMAN PRACTICES
            					</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/HP/Silver">Silver Medal</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/Collaboration">Collaboration</A></LI><LI><A href="#">TEAM
            					</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/Team">Members</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/Sponsors">Sponsors</A></LI><LI><A href="https://2017.igem.org/Team:McMasterU/Attributions">Attributions</A></LI></DIV><DIV id="contentContainer"><DIV class="band"><H1>SELEX Optimization with Machine Learning</H1><P>This project was designed as a way to help the wetlab improve the efficiency of their DNAzyme.
Systematic evolution of ligands by exponential enrichment (SELEX) is an in vitro evolutionary selection process that was used to find high binding affinity sequences for a specific region in E. coli, where the DNAzyme acts to autofluoresce. Higher DNAzyme binding affinities would lead to higher observable rates of autofluorescence- serving as a potential of E. coli detection in hospitals. SELEX is an expensive, difficult, time-consuming process, with all of these increasing with each successive generation being run.</P><H2>Genetic algorithms for sequence optimization</H2><H3>Overview</H3><P>This project sought to simulate this SELEX optimization process in silico using a genetic algorithm coupled with a neural network. A genetic algorithm is an optimization process inspired by natural selection.
</P><P>Our genetic algorithm (GA) aimed to serve as a computational aid in the design of high-affinity DNA aptamers to a certain target ligand, based on the concept of the systematic evolution of ligands by exponential enrichment (SELEX). From a pool of sequenced aptamers, our GA selects a specified proportion of the most abundant aptamers and creates random cross-overs, insertions, deletions and single nucleotide polymorphisms among these parental aptamers to give rise to a new set of progeny aptamers. The number of progeny returned by our GA is user-defined. These in silico predicted aptamers can then be synthesized and have their efficacies against the target ligand assayed in vitro. Aptamers which show the greatest affinities for the target ligand will be enriched for and then sequenced. These sequencing reads can be assembled and fed back into our GA pipeline, to generate a new set of aptamers, which theoretically have better affinities to the target ligand than their parental generation. In this way, several iterations of SELEX and our GA can be conducted to yield high specificity aptamers for any target ligand.  </P><P>The code for this project, and more detailed documentation, is available in our <A href="https://github.com/mgem/mgem-drylab-2017">Github repository</A>.

</P><H3>Mechanics of the genetic algorithm</H3><P>The optimization process began with a population of solutions (DNAzyme sequences) to the problem of focus (finding high-binding affinity DNAzyme sequences). These solutions normally require string encoding, but since the solutions are DNA sequences, this has already been completed. Next the top x% (usually 10) are taken from this solution population and are bred together to get an entirely new population of solutions.
 </P><CENTER><B>Figure 1: </B>The genetic algorithm selects individuals for breeding using some fitness function.</CENTER><P> The breeding involves crossover, mutation and in our case, filtering. </P><P><U>Breeding</U> is when two members of the 10% subset are picked and the first part of one member is randomly combined with the second part of another member:</P><UL><LI> If AAAAA and TTTTTT are bred, then a random number between 1 and 5 (length of the shorter sequence) is chosen and the sequences are crossed over at that point. </LI><LI> If 3 were chosen as the random crossover point then the corresponding child sequences would be AAATTT. </LI></UL><P>Â <U>Mutations</U> may occur in the algorithm where, for each base, there is a 2% chance (can be varied) where that base will randomly switch to another base: </P><UL><LI> If AAATTT were to be mutated, then a possible outcome is AAGTTT. </LI></UL><P>The <U>filtering step</U> involved checking whether a sequence contained a hairpin (palindromic sequence that can bind to itself), as hairpins are removed from the population. Functionally, hairpins can interfere with binding as they increase the likelihood of self-binding, so sequences with hairpin structures were removed during each generation. </P><CENTER><B>Figure 2: </B>Selected sequences are bred together by crossing them over at a random point, and then mutated by changing bases at random.</CENTER><P>Running this GA, with a simple fitness function (% of bases that are A's), showed significant fitness improvements, even over a short number of generations (&lt; 10).</P><H2>Neural networks as fitness functions</H2><P> The next phase of the project was to implement a neural network. We built a LSTM network and trained it on SELEX data from the wet lab to build a model to predict the fitness of a DNA sequence. Through this method, multiple generations can be run and the top scoring individuals can be selected for breeding. </P><P> Due to the nature of long sequence data (the initial solution population had an average length of 290-300 bases), building an accurate model proved difficult. Using neural nets to complete regression on sequence data was challenging. However despite these struggles, we were able to build a functional LSTM- with limited ability to accurately predict binding affinity to a useful degree. The model was trained under multiple parameter sets, with various training datasets, but ultimately due to the nature of the problem and the quality/amount of the training data we were ultimately unable to produce a fully functional model for predicting binding affinity.
 </P><P> Despite this we were able to prove the validity of the GA + NN strategy for generating higher quality solutions. We generated a data set of short 15-30bp sequences, where fitness was calculated as abs(65-melting temperature)/65. 
Sixty-five degrees is often thought of as an ideal melting temperature for primers during PCR. As this is a very simple metric, that can easily be calculated for any sequence, we used this fitness measurement as a way to test the validity of this strategy at all. First, a dataset of 50000 random sequences was generated, along with their fitness values. 
Next, our LSTM model was trained on this data, making it able to predict melting temperature of a sequence with less than a 2% error on average. Here error was defined as abs(predicted fitness - actual fitness)*100/actual fitness, fitness being defined for melting temperature above. As the starting fitness of the initial population was rather high (80%), a 3000 sample subset was taken (avg fitness ~54%). The LSTM was integrated into the GA and allowed to run for between 1-90 generations. 
After even 10 generations, the solution populations were becoming fixed at 1-2 unique solutions (out of ~1700-2100 total solutions)- but for any number of generations there was a fitness increase of either 5% for 1 generation and 20-35% increase past 10 generations. Varying the input data set would likely result in more variation at higher generation numbers.
Taking solutions from every generation, there were 71 unique solutions, starting from a set of 3000 initial members in the solution population. </P><P>The error in predictions, while higher than when testing the LSTM on the validation level, was lower than the actual aptamer prediction error. Ultimately, this model provides a proof of concept that integrating a genetic algorithm and machine learning model can be used to optimize sequences in silico.The most challenging problem ahead is finding ways to increase the accuracy of the ML model, either by optimizing the model parameters (batch size, epochs, architecture etc.), finding ways to better extract data sets, using different techniques (random forest) or any combination. </P><P>A genetic algorithm is an excellent optimization tool, but it can only work insofar as it has an accurate fitness estimation function, provided in our case, by a neural network.</P><H2> Next Steps </H2><P>Next steps include experimenting with other machine learning techniques, as well as designing SELEX experiments to obtain data specifically tailored to serving as good training sets (high variety, high volume) Also, extracting more out of our own data, to try and expand the feature set (adding values for GC%, length, or other feature that can be extracted from a sequences) can help increase the model's accuracy, or at least have it indicate further heuristics to guide fitness estimation. </P></DIV><DIV class="band"><NAV id="menu" align="center"><H4>Follow Us</H4><H5>Â© 2017 mGEM</H5></NAV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>