<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Results</SPAN></H1></DIV></DIV></DIV><DIV class="container"><DIV id="position-menu2"><UL id="menu2"><LI><FONT size="3pt" color="#8B8378">About us</FONT></LI><LI><A href=" https://2016.igem.org/Team:Pasteur_Paris/AboutUs " href="">Our team</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Sponsors">Sponsors</A></LI></UL><LI><FONT size="3pt" color="#8B8378">Project</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Context">Context</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Overview">Overview</A></LI><LI><FONT size="3pt" color="#8B8378">Design </FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Science">Science </A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Moskit_devices">Devices</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Research">Research &amp; development</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Methodology">Methodology</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Design_Biology">Design vs Biology</A></LI><LI><FONT size="3pt" color="#8B8378">Labwork </FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Safety_Security">Safety &amp; Security</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Protocol ">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Netbook ">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Results ">Results</A></LI><LI>Human Practice</LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Collaborations ">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Scenarios">Scenarios</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Meet-up ">Meet-up</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Engagement">Communication</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Law"> Law</A></LI><LI><FONT size="3pt" color="#8B8378">Parts</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Description">Description</A></LI><LI><FONT size="3pt" color="#8B8378">Achievements</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/medals">Medals</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Prizes"> Special Prizes</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/rewards"> Awards obtained!</A></LI></DIV></DIV><H1><B>Results</B></H1><H2><B>Design of the trap </B></H2><DIV class="text1"><P>
Based on the input of specifications by <B>experts in the field</B> (entomologists, mosquito control officers, virologists..), and the impact of the <B>economy</B> and <B>sociology</B> of the places where we will apply our project, namely mostly tropical and developing countries, on the scientific process of detection (ecosystem of the mosquitoes, state of the samples containing pathogen antigens, safety,…) we were able to generate a <A href="https://2016.igem.org/Team:Pasteur_Paris/Moskit_devices">trapping device</A> with the help of <B>ideation</B>, <B>prototyping</B> and <B>3D modeling software</B>. The device is easy to use, safe and efficient in the detection of mosquito borne <B>pathogen antigens</B>. The trap was subsequently materialized through the <B>3D printing process</B>. The prototype model tested for egress of sample of mosquitoes (n=200) showed a 2% rate of escape (98% retention rate). However, capture using the Biogent® pheromone bag was not efficient as no mosquitoes were captured after 24h of exposure. This second aspect needs to be improved, by changing attraction systems including CO<SUB>2</SUB> generation. 
</P></DIV><H2><B>C1 or C2 fusion protein (Si4-CBPa-BpA)</B></H2><DIV class="text1"><P>
The <B>fusion protein</B> we designed contains the <B>silica-binding peptide</B> (Si4), the <B>cellulose-binding domain</B> of cellulose-binding protein A (CBPa), and the <B>antibody-binding B domain</B> of staphylococcal protein A (BpA). It is a 25 kDa protein (Fig. 1).</P></DIV><DIV class="text2"><P><B>Figure 1. Schematic representation of the C2 fusion protein.</B>
The C2 fusion protein is composed of the silica-binding peptide (<B>Si4</B>, in red), the cellulose-binding domain of Clostridium cellulovorans cellulose-binding protein A (<B>CBPa</B>, in green), the B domain of staphylococcal protein A (<B>BpA</B>, in blue).
</P></DIV><DIV class="text1"><P>
Once we received the sequence encoding for the protein (named construction C1 or C2, size 921bp, with a C-ter or N-ter His-Tag respectively), we amplified it by PCR by using specific primers (For, Rev, see PCR protocol) (Fig. 2) and a Taq polymerase without exonuclease activity. In lanes 1 and 4 we see that a <B>PCR product was amplified</B> with the expected size. As negative controls, neither amplification was possible with a single primer (lanes 2, 3, 5, 6), nor in the absence of primers (lane 7) or DNA template (lane 8).  </P></DIV><DIV class="text2"><P><B>Figure 2. Polymerase chain reaction of DNA sequence of the fusion protein (C1 and C2).</B>
C1 and C2 sequences (lanes 1 and 4, respectively) were amplified post-synthesis to generate sufficient material for cloning. Control reactions: single primer (lanes 2, 3, 5, 6), without primers (lane 7), without DNA template (lane 8).  MW: molecular weight marker.
</P></DIV><DIV class="text1"><P>
In order to have more DNA, we cloned it into TOPO vector (Fig. 3A), and transformed competent bacteria <I>Escherichia coli</I> TOP10, resulting in white clones (Fig. 3B). After bacteria culture and plasmid DNA extraction, we <B>verified</B> the presence of an insert by using <B>Xba I</B> and <B>Hind III</B> restriction enzymes (data not shown). After that, insert was extracted from the gel, and ligated into digested and dephosphorylated <B>pET43.1a</B>, the <B>expression vector</B> (Fig. 4A). We repeated the procedure, and we proved that our vector contained the insert by electrophoresis (Fig. 4B). Sequencing confirmed that it was the correct sequence. </P></DIV><DIV class="text2"><P><B>Figure 3. TOPO Cloning</B>
(A) TOPO cloning vectors were used to overcome cloning difficulties. Trimolecular reaction is reduced to a bimolecular one. (B) Bacteria were cultivated onto LB plates added of ampicillin and X-Gal, resulting in white clones. White/blue selection was used to identify recombinant vectors.</P></DIV><DIV class="text2"><P><B>Figure 4. Cloning into pET43.1a(+) vector</B>
(A) pET43.1a(+) vector map showing Xba I and Hind III restriction sites, ampicillin resistance gene (green arrow), and T7 promoter. (B) Plasmid DNA was extracted by Miniprep. Enzymatic digestion by Xba I and Hind III followed by electrophoresis revealed the presence of the C2 insert. 
</P></DIV><DIV class="text1"><P>
Once checked, we cloned our construct into the <I>Escherichia coli</I><B>BL21(DE3)</B> strain, a specific dedicated strain to produce high amounts of desired proteins under a T7 promoter. Bacteria were grown on large scale (4 l), and we made a growth curve (Fig. 5). Protein expression was induced with IPTG overnight at 15°C. Protein purification was achieved using the His-Tag. Owing to the <B>intrinsic affinity of C2 for cellulose</B>, we had to revert to a <B>polystyrene column for purification to work</B>. We eluted our protein using a gradient of imidazole-containing buffer, and two peaks were detected (Fig. 6). We checked the presence of proteins in the fractions by SDS-PAGE. We clearly noted the appearance of bands at about 25 kDa, the expected size of our fusion protein (24 967 Da), but also at about 50 kDa (Fig. 7). We hypothesized that it could be monomers (25 kDa) and dimers (50 kDa). Indeed, since Si4 of C2 is able to condense silicic acid, it could potentially form dimers via Si-O bonds that resist reduction by β-mercaptoethanol. </P></DIV><DIV class="text2"><P><B>Figure 5. Growth curve of pET43.1-C2-transformed BL21(DE3) bacteria.</B>
Transformed <I>E. coli</I> BL21(DE3) were grown in LB supplemented with carbenicillin (50 µg/mL). Time points were taken and OD<SUB>600 nm</SUB> was measured every 20 minutes. When OD<SUB>600 nm</SUB> was about 0.7, the culture was induced with IPTG at 0.3 mM (red arrow). </P></DIV><DIV class="text2"><P><B>Figure 6. FPLC C2 protein purification</B>
(A) Polystyrene-based Ni-NTA column (Nuvia, Biorad) was equilibrated with buffer A (Tris-Cl 50 mM pH 7.4, NaCl 150 mM). (B) Supernatant of lyzed bacteria was introduced through the column. (C) Washing with 5% of buffer B. (D) Elution by buffer B gradient (buffer A + imidazole 250 mM). UV absorbance at 280nm is shown in blue, conductivity in red, pressure in brown, temperature in cyan, and concentration of buffer B in green. Flow-rate : 0.5 ml/min. Fractions size : 1 ml.</P></DIV><DIV class="text2"><P><B>Figure 7. Protein elution profile of C2 </B>
After lysis and FPLC, fractions ran on SDS-PAGE .  SN : supernatant. FT : flowthrough. Lanes 1 to 6 : fractions. C2 monomers (25 kDa) and dimmers (50 kDa) shown by dark arrows. MW : molecular weight marker.
</P></DIV><DIV class="text1"><P>
After determining the concentration of each fraction by Bradford assay, we tested the efficiency of our protein to <B>bind to cellulose</B> and to catalyze the <B>biosilification reaction</B>. Unfortunately, the described methods to the last component, i.e. to test the ability of our protein to bind to antibodies, require high amounts of antibodies (<U>hundreds of micrograms</U>), making the experiment too expensive to perform. 
form dimers via Si-O bonds. 
</P></DIV><DIV class="text1"><P>
We first tested the ability of our protein to bind to cellulose. To do that, we used several types of cellulose: Avicell, Sigmacell, and carboxymethyl-cellulose. As described by Goldstein et al<SUP>1</SUP>, we mixed cellulose with an excess of competitor non-specific protein (BSA), and with or without our protein of interest (Fig. 8A). After washing and centrifugation, we harvested proteins from supernatant and the cellulose-based pellet in order to analyze them by SDS-PAGE. We clearly noted that the 25 kDa and 50 kDa proteins were <B>retained by cellulose</B>, instead of the non-specific BSA (Fig. 8B). Indeed, data showed that almost no monomer remained in the supernatant after the second wash, but the pellet contained most of the protein. However, some dimers seem to remain in the second washing supernatant because the initial protein concentration was too high: the binding sites were saturated. The pellet also contains a lot of the dimers. As <B>control</B>, we observed that BSA remained in the supernatant and didn’t bind to cellulose.  Therefore, we can conclude that our protein binds to cellulose. </P></DIV><DIV class="text2"><P><B>Figure 8. Cellulose-binding test for C2 protein</B>
(A) Schematic diagram of the cellulose-binding method.  Cellulose is shown in red. S1 : first supernatant. S2 : washing supernatant. P: cellulose-based pellet. (B) SDS-PAGE of extraction phases. Left panel : binding profile for 25 kDa molecules. Right panel : binding profile for 50 kDa molecules.</P></DIV><DIV class="text1"><P>Then, we investigated whether our protein was able to catalyze the <B>biosilification reaction</B>. To do that, we drew inspiration for the <A href="https://2011.igem.org/Team:Minnesota"><B>2011 Minnesota iGEM team</B></A> and their work about <B>Si4</B> to evaluate the silification process. First, we used a source of silicic acid, the tetraethyl orthosilicate (<B>TEOS</B>), which is an inactive form of silicic acid. By activating it in acidic conditions, we released the free silicic acid (Fig. 9A). After incubation with or without our fusion protein, we determined the <B>quantity of free silicic acid</B> by a spectrophotometric method, since biosilification process consumes silicic acid to form silica (Fig. 9B). We clearly observed a precipitation into the test tube, instead of the negative control (Fig. 10A). By quantifying it by <B>molybdate assay</B> using a <B>standard curve</B> (Fig. 10B), we deduced the corresponding mass of silicic acid left after silification: 33 µg. Before silification, the concentration was 208 µg/ml. The fusion protein led to the production of 175 µg of silica after 2 hours. Therefore, the silification yield after two hours is up to 84% with the C2 protein whereas the yield without the protein is 0% (Fig. 10C). We concluded that our protein worked. </P></DIV><DIV class="text2"><P><B>Figure 9. Silification process</B>
(A) Schematic representation of sol-gel process. Acidified TEOS  yields silicic acid that condenses into silica. (B) After incubation with C2, a polymer silica gel is formed and silification is evaluated by molybdate assay</P></DIV><DIV class="text2"><P><B>Figure 10. Silification test for C2 protein
</B>
 (A) Silica gel pellets recovered after 2 hours of silification and centrifugation, for the monomer (left) and the dimer (right). (B) Standard curve of molybdate assay for determination of free silicic acid. (C) Determination of free silicic acid concentration by molybdate assay. (D) Silification efficiency, calculated from C. 
</P></DIV><DIV class="text1"><P>
Taken together, our results suggest that our fusion protein is able to bind to cellulose and to catalyze biosilification. 
</P></DIV><H2><B>Prediction of the influence of silification on the mechanical properties of the patch</B></H2><DIV class="text1"><P>
Using the modeling approach described in the Protocols section (<A href="https://2016.igem.org/Team:Pasteur_Paris/Protocol">Patch mechanical properties modeling</A>), we assumed that silica could increase by 48% the <B>Young’s modulus</B> and by 75% the shear modulus of the patch. Thus, our patch would be as <B>rigid as plastics</B> after silification. However, we have to be cautious about our model since we assumed that our protein is linear, which is not the case. Moreover, the <B>rule of mixtures</B> can be applied only when the two materials have the same <B>Poisson’s ratio</B>, but we do not have any evidence of this for C2 and silified C2. Additionnaly, we assumed that all integrated proteins are silified, but we only observed 84% yield, thus generating two states in the system. 
</P></DIV><H2><B>Assembly of the composite patch</B></H2><DIV class="text1"><P><B>Composite patches</B> were obtained with a mix of cellulose and silica gel, either by mechanical mixing or produced in the <B>one pot experiment</B>. The resulting composite patches are <B>easier to handle</B> than those with cellulose alone or a mix of cellulose and water. The former resist to manipulation whereas the latter break when they are manipulated.
</P></DIV><DIV class="text1"><P>
At the time of this writing, we  have sent the patches to the <A href="https://2016.igem.org/Team:TU_Delft">iGEM TU Delft 2016 team</A> to be characterized, by electron microscopy.
</P></DIV><H2><B>Immunodetection</B></H2><DIV class="text1"><P>
Our patch aims at detecting <B>mosquito-borne pathogen antigens</B>. Since the composite patch-protein was not completely assembled, we characterized the <B>efficiency of our detection method</B> by using commercial membranes of PVDF or nitrocellulose. The detection method we designed was based on the <B>conjugation</B> of <B>antigens</B> with the Horse radish peroxydase enzyme <B>HRP</B>, and the capture of these conjugated antigens by specific antibodies.  We revealed the interactions by incubating membranes with the HRP’s substrate.
First, we tested whether our detection method was working by using <B>single purified viral protein</B> of yellow fever virus (<B>YFVp</B>). By incubating coated membranes with conjugated HRP-BSA, single EZ, or non conjugated YFVp, we didn’t observe any signal (Fig. 11). However, in presence of <B>conjugated HRP-YFVp</B>, a dark spot appeared (Fig. 11). Thus showing that we were able to <B>detect YFVp</B> by this immunodetection technique. </P></DIV><DIV class="text2"><P><B>Figure 11. Immunodetection of envelope protein of CHIKV within an excess of BSA</B>
 (A) Principle of experiment. CHIKV-specific 3E4 antibody is represented in red. (B) Photography of membranes after the interactions were revealed. 
</P></DIV><DIV class="text1"><P>
Then, in the presence of an excess of non-specific proteins such as BSA, we showed that we maintained the specificity of our detection method (data not shown). 
To determine whether mosquito proteins can interfere with the specific interaction between viral proteins and specific antibodies, we coated PVDF membranes with CHIKV-specific antibodies, and we conjugated CHIKV envelope protein in presence of an <B>excess of pathogen-free mosquito proteins</B>. Then, we incubated coated membranes with conjugated mosquito proteins in the presence or absence of CHIKV envelope protein. We noted a <B>basal level</B> of interaction between the CHIKV-specific antibody 3E4 and mosquito proteins (Fig. 12). However, we noted an important <B>increase of signal</B> when <B>CHIKV envelope protein is present</B> in the mixture. It is obvious that we have to improve the specificity of our detection method in the presence of mosquito lysate. </P></DIV><DIV class="text2"><P><B>Figure 12. Immunodetection of envelope protein of CHIKV within mosquito lysate</B>
 (A) Immunodetection of E(CHIKV)within mosquito lysate by using specific 4E3 antibody on PVDF membranes. (B) Quantitative representation of previous data. </P></DIV><DIV class="text1"><P>
In addition to the fact that these results showed that our detection method worked, we also plan to determine the sensitivity of the method by using different amounts of CHIKV envelope proteins in the mixture. 
Finally, when <B>patches became available</B> after pressure assembly of <B>cellulose-C2 protein</B>, we repeated the immuno dection assay experiment on the composite patches (cellulose-silica + fusion protein). By replacing the membrane by the patch, <B>we observed a signal</B> when CHIKV envelope protein is present, but <B>non-specific interactions</B> are clearly visible (Fig. 13). We can explain this observation by the fact that it was difficult to wash patches since they a little brittle in full liquid shaking conditions. A better packaged version, with vertical flow could improve the fluidics and stability of the experiment. Moreover, heterogeneity of degradation between the patches contributed to the fact that we cannot come to a conclusion (signal intensity was measured by mean intensity in regions of interests). In parallel, we did the same experiment to detect <B>YFV envelope protein</B> into infected mosquitoes (day 11 post-infection) (Performed by a coach with BHS clearance, see Safety conditions). As previously, no conclusion can be drawn. </P></DIV><DIV class="text2"><P><B>Figure 13. Immunodetection of envelope protein of CHIKV or YFV using the composite patch </B>
 Immunodetection of E(CHIKV) in presence of mosquito proteins onto coated-composite patches. Histograms : 1, 2, 3: mosquito lysate + E(CHIKV); 4: mosquito lysate + diluted E(CHIKV). 
Immunodetection of E(YFV) into infected mosquitoes onto coated-composite patches. Histograms : 1, 2, 3: YFV-infected mosquitoes; 4: non infected mosquitoes.</P></DIV><H2><B>Analysis kit </B></H2><DIV class="text1"><P>
Using similar approaches as in the <A href="https://2016.igem.org/Team:Pasteur_Paris/Moskit_devices">trap design</A>, a <B>prototype</B> for an <B>analysis station</B> has been <B>3D printed</B>. It allows us to visualize the analysis process and ergonomy. Sample throughput in the system remains to be tested. </P></DIV><H2><B>Scenario </B></H2><DIV class="text1"><P>
With collaborations, public <B>education</B>, public <B>opinion</B>, <B>polling</B>, we were able to generate a <A href="https://2016.igem.org/Team:Pasteur_Paris/Human_Practices"><B>scenario</B></A> that encompasses the use of our Mos(kit)o device in the real world. </P></DIV><H2><B>Open science vs Start-up model</B></H2><DIV class="text1"><P>
From our <A href="https://2016.igem.org/Team:Pasteur_Paris/Meet-up"><B>meet-ups</B></A>, discussion with other teams, and our own concern for the transition from <B>Open Science</B> to a possible start-up, we were able to generate <A href="https://2016.igem.org/Team:Pasteur_Paris/Law"><B>two document tools</B></A> that summarize and inform about these issues. </P></DIV><DIV class="text3"><H2>References: </H2>
[1] Characterization of the cellulose-binding domain of the <I>Clostridium cellulovorans</I> cellulose-binding protein A, Golstein MA et al, J. Bacteriol., 1993. </DIV></BODY></HTML>