<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Microbiology_week9 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Microbiology week9</SPAN></H1></DIV></DIV></DIV><DIV class="container"><DIV id="position-menu2"><UL id="menu2"><LI><FONT size="3pt" color="#8B8378">About us</FONT></LI><LI><A href=" https://2016.igem.org/Team:Pasteur_Paris/AboutUs " href="">Our team</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Sponsors">Sponsors</A></LI></UL><LI><FONT size="3pt" color="#8B8378">Project</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Context">Context</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Overview">Overview</A></LI><LI><FONT size="3pt" color="#8B8378">Design </FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Science">Science </A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Moskit_devices">Devices</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Research">Research &amp; development</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Methodology">Methodology</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Design_Biology">Design vs Biology</A></LI><LI><FONT size="3pt" color="#8B8378">Labwork </FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Safety_Security">Safety &amp; Security</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Protocol ">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Netbook ">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Results ">Results</A></LI><LI>Human Practice</LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Collaborations ">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Scenarios">Scenarios</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Meet-up ">Meet-up</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Engagement">Communication</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Law"> Law</A></LI><LI><FONT size="3pt" color="#8B8378">Parts</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Description">Description</A></LI><LI><FONT size="3pt" color="#8B8378">Achievements</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/medals">Medals</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Prizes"> Special Prizes</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/rewards"> Awards obtained!</A></LI></DIV></DIV><DIV id="weeks"><DIV id="weekslist"><CENTER><H1><B>click on the weeks</B></H1></CENTER></DIV></DIV><DIV><A href="https://2016.igem.org/Team:Pasteur_Paris/Microbiology"><H2><B>Microbiology Notebook</B></H2></A></DIV><DIV id="week9"><H5><B>Week 9 </B></H5><H3><B> August 1, 2016:</B></H3><H4> 120. Silification tests </H4><A href="#exp2"><H4> 121. PCR of C1 v2 and C2 v2 with Takara enzyme </H4></A><A href="#exp3"><H4> 122. Agarose gel </H4></A><A href="#exp4"><H4> 123. Electrophoresis of the PCR products (C1 v2 and C2 v2) </H4></A><A href="#exp5"><H4> 124. PCR Clean up </H4></A><H3><B> August 2, 2016:</B></H3><H4> 125. PCR of A1, A2, B1, B2, D1, D2, E1 and C2 </H4><A href="#exp7"><H4> 126. Analysis of PCR products from August 1 </H4></A><A href="#exp8"><H4> 127. PCR clean up </H4></A><A href="#exp9"><H4> 128. Ligation of PCR products with TOPO cloning </H4></A><A href="#exp10"><H4> 129. Transformation in TOP10 competent cells </H4></A><A href="#exp11"><H4> 130. PCR of A1⁄A2 and D1⁄D2  </H4></A><H3><B> August 3, 2016:</B></H3><H4> 131. Analysis of PCR products from August 2, 2016  </H4><A href="#exp13"><H4> 132. Miniprep of cultures C1 v2 and C2 v2 from August 2 </H4></A><A href="#exp14"><H4> 133. Digestion of C1 v2 and C2 v2 </H4></A><A href="#exp15"><H4> 134. Electrophoresis of C1 v2 and C2 v2 </H4></A><A href="#exp16"><H4> 135. DNA extraction from the gel </H4></A><A href="#exp17"><H4> 136. Ligation of C1.2⁄C1.5 and C2.1⁄C2.2 with pET43.1 (a+) and transformation with TOP10 competent cells </H4></A><H3><B>August 4, 2016:</B></H3><H4> 137. Miniprep of B1⁄B2 and E1⁄E2 </H4><A href="#exp19"><H4> 138. Digestion of B1⁄B2, E1⁄E2 and pET43.1 (a+) with Hind III and Xba I </H4></A><A href="#exp20"><H4> 139. Dephosphorylation of digested pET43.1 (a+) </H4></A><A href="#exp21"><H4> 140. PCR of A1⁄A2 and D1⁄D2 without MgCl<SUB>2</SUB></H4></A><A href="#exp22"><H4> 141. Preculture of TOPO – C1 v2 (2) and (5) and TOPO – C2 v2 (1) and (2) </H4></A><A href="#exp23"><H4> 142. Electrophoresis of PCR products A1⁄A2 and D1⁄D2 </H4></A><A href="#exp24"><H4> 143. Next PCR of A1⁄A2 and D1⁄D2 </H4></A><A href="#exp25"><H4> 144. Pool of inserts C1 v2 and C2 v2 </H4></A><H3> August 5, 2016:</H3><H4> 145. Digestion of B1⁄B2 and C1⁄C2  </H4><A href="#exp27"><H4> 146. Ligation of C1 and C2 with pET43.1 (a+) digested and dephosphorylated </H4></A><A href="#exp28"><H4> 147. Transformation of TOP10 competent cells </H4></A><A href="#exp29"><H4> 148. Ligation of B1⁄B2⁄C1⁄C2 in pET43.1 (a+) </H4></A><A href="#exp30"><H4> 149. Transformation of B1⁄B2⁄C1⁄C2 in TOP10 </H4></A><DIV class="lightbox" id="exp1"><FIGURE><FIGCAPTION><P><U> Aim:</U> Check if the sillification is working with different volume of HCl and TEOS. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/2/2a/Protocol-silification-assay_Pasteur_Paris2016.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• 1.5 ml Eppendorfs • Silification protein • HCl 1 M • TEOS (Tetraethyl ortho silicate, Sigma) • Shaking incubator • 15 ml Falcon • Buffer A (See protein purification protocol)• 10 μl and 200 μl pipettes <U>Method:</U><LI>1. In a 1.5 ml Eppendorf prepare a sample with the fraction that contains protein c2, add 1 ml of HCl 1mM, 65.8 μl of TEOS and let it incubate for 4 minutes in the rotating incubator</LI><LI>2. In a 15 ml Falcon, put 9 ml of buffer A and add the previous sample</LI><LI>3. Follow if silification initiates and progresses at various times :</LI></P><P><U>Table 84</U></P>Times Comments <STRONG><P> 10 minutes </P></STRONG>Ø<STRONG><P> 15 minutes </P></STRONG> Microscopic crystalline specs appear <STRONG><P> 30 minutes </P></STRONG> Microscopic crystalline specs continue to appear <STRONG><P> 50 minutes </P></STRONG> Microscopic crystalline specs continue to appears + crystalline specs grow and form a precipitate 
4. Redo this experiment with 1 ml of TEOS, 50 μl of protein and vortex: <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 85</U></P></I></CAPTION><THEAD><TR><TH>Times</TH><TH> Comments </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> 15 minutes </P></STRONG></TD><TD align="center"> Crystalline specs appear </TD></TR><TR><TD align="center"><STRONG><P> 2 hours </P></STRONG></TD><TD align="center"> Crystalline specs continue to appear </TD></TR></TBODY></TABLE>
	5. Redo the experiment with 1 ml of TEOS to see the catalytic activity of the protein : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 86</U></P></I></CAPTION><THEAD><TR><TH>Times</TH><TH> Comments </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> 1h45 </P></STRONG></TD><TD align="center">Ø</TD></TR></TBODY></TABLE>
	6. Redo the experiment with 50 μl of HCl 1 mM and 50 μl of protein
7.Redo the experiment with 50 μl of HCl 1 mM but without our protein<U>Results: </U>
When glycerol is added to the sample, as an amphiphilic modifier, an emulsion takes place which mean that glycerol act as a surfactant. 
Our protein is a catalyst
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp2"><FIGURE><FIGCAPTION><P><U> Aim:</U> Test the specificity of primers For and Rev to obtain more insert.
We also switched to the Takara EX DNA polymerase for its terminal deoxynucleotide transferase activity, which adds an deoxy-A residue at the 3' end of the PCR product<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/b/be/T--Pasteur_Paris--PCR_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• dNTP mix • MgCl<SUB>2</SUB> 25 mM • EX polymerase buffer 10X • RNAse free water • C1⁄C2 insert DNA • primers For and Rev • Takara EX DNA polymerase • PCR machine • 10 μl and 200 μl pipettes <U>Method:</U>
      1. Use a Globalmix with : </P><P><U>Table 87</U></P> Volumes (μl) <STRONG><P> dNTP </P></STRONG> 25 <STRONG><P> Mgcl<SUB>2</SUB></P></STRONG> 25 <STRONG><P> Buffer 10X </P></STRONG> 50 <STRONG><P> RNAse free water </P></STRONG> 36.5 <STRONG><P> Final </P></STRONG> 50 
2. For the next step use : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 88</U></P></I></CAPTION><THEAD><TR><TH>Tube </TH><TH> Name </TH><TH> Premix (μl) </TH><TH> C1 (μl) </TH><TH> C2 (μl) </TH><TH> Primer For (μl) </TH><TH> Primer Rev (μl) </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> 1 </P></STRONG></TD><TD align="center"> C1 insert (For +Rev) </TD><TD align="center"> 46.5 </TD><TD align="center"> 1 </TD><TD align="center">Ø</TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> 2 </P></STRONG></TD><TD align="center"> C1 For </TD><TD align="center"> 46.5 </TD><TD align="center"> 1 </TD><TD align="center">Ø</TD><TD align="center"> 1 </TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> 3 </P></STRONG></TD><TD align="center"> C1 Rev </TD><TD align="center"> 46.5 </TD><TD align="center"> 1 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> 4 </P></STRONG></TD><TD align="center"> C2 insert (For + Rev) </TD><TD align="center"> 46.5 </TD><TD align="center">Ø</TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> 5 </P></STRONG></TD><TD align="center"> C2 For </TD><TD align="center"> 46.5 </TD><TD align="center">Ø</TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> 6 </P></STRONG></TD><TD align="center"> C2 Rev </TD><TD align="center"> 46.5 </TD><TD align="center">Ø</TD><TD align="center"> 1 </TD><TD align="center">Ø</TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> 7 </P></STRONG></TD><TD align="center"> Without DNA </TD><TD align="center"> 46.5 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> 8 </P></STRONG></TD><TD align="center"> C1 without primer </TD><TD align="center"> 46.5 </TD><TD align="center"> 1 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> 9 </P></STRONG></TD><TD align="center"> C2 without primer </TD><TD align="center"> 46.5 </TD><TD align="center">Ø</TD><TD align="center"> 1 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD></TR></TBODY></TABLE>
	3. For the PCR program, start at 24°C, add 0.5 μl of Takara enzyme and proceed to 94°C. Annealing is set at 55°C.
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp3"><FIGURE><FIGCAPTION><P><U> Aim:</U> Verification of the content of the fractions obtained from the Ni-NTA column chromatography. </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp4"><FIGURE><FIGCAPTION><P><U> Aim:</U> Test the specificity of primers For and Rev to obtain more insert. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Loading buffer 6X • PCR products  • Inserts C1 v2, C2 v2  • Primer S (For) • Primer AS (Rev) • Electrophoresis chamber, and power supply  • 10 μl pipette <U>Method:</U>
      1. Add 5 μl of DNA and 1 μl of buffer 6X to each sample
2. Use 6 μl of molecular weight marker and deposit each sample in the wells :</P><P><U>Table 89</U></P> L1  L2  L3  L4  L5  L6  L7  L8  L10  L11  L12  M. weight Marker Ø C1 + insert  C1 + primer S  C1 + primer AS  C2 + insert  C2 + primer S  C2 + primer AS  Without DNA  C1 without primer  C2 without primer 
2. Start the electrophoresis at 50 V and 0.01 A for 10 minutes, then put it at 90 V and 0.03 A </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp5"><FIGURE><FIGCAPTION><P><U> Aim:</U> Get back the DNA amplified by PCR. </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp6"><FIGURE><FIGCAPTION><P><U> Aim:</U> Increase the quantity of DNA for cloning. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/b/be/T--Pasteur_Paris--PCR_protocol.pdf">link</A><U> Results :</U></P><P><U>Figure 11 :</U> PCR gel of A1/A2/B1/B2/D1/D2/E1/E2 </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp7"><FIGURE><FIGCAPTION><P><U> Aim:</U> Test the specificity of primers For and Rev to obtain more insert. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Agarose  • Ethidium bromide drops (EB)  • Molecular weight ladder (Gene ruler 1 Kb, Thermofisher) •Loading buffer 6X  • PCR products  • Electrophoresis chamber, power supply  • 10 μl pipette <U>Method:</U>
      1. Make an agarose gel at 0.7%
2. Prepare the samples with 5 μl of PCR products and 1 μl of loading buffer 6X
3. Depose each samples in the wells : </P><P><U>Table 90</U></P> L1  L2  L3  L4  L5  L6  L7  L8  L9  L10  L11  L12  L13  L14  L15  Ladder Ø A1  A2 Ø B1  B2 Ø D1 Ø D2 Ø E1  E2 Ø
4. Begin the electrophoresis at 50 V for 15 minutes, then at 90 V for 1h30 <U>Result</U>
There are streaks in lanes 3, 4, 9 and 11 which mean that the PCR did not work. But it seems to work for lanes 6, 7, 13 and 14. 
The PCR must be done another time for A1⁄A2 and D11⁄D2 with a lower temperature (1°C).
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp8"><FIGURE><FIGCAPTION><P><U> Aim:</U> Test the specificity of primers For and Rev to obtain more insert. <U>What we did in the lab:</U><U>Materials:</U>• Qiagen PCR clean up kit
• PCR products
• 200 μl pipette <U>Method:</U>
      1. Prepare the 8 samples: </P><P><U>Table 91</U></P> Samples  A1  A2  B1  B2  D1  D2  E1  E2 <STRONG><P>  Buffer A (μl) </P></STRONG>&lt; 90  90  90  90  90  90  90  90 
2. Follow the protocol of the Qiagen PCR clean up kit <U>Result</U>
We obtained 25 μl of each insert purified by PCR.
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp9"><FIGURE><FIGCAPTION><P><U> Aim:</U> Test the specificity of primers For and Rev to obtain more insert. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/6f/T--Pasteur_Paris--Ligation_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• PCR products • Salt solution • pET43.1 vector DNA • Incubator 37°C • 10 µl pipettes • 1 ml Eppendorfs <U>Method:</U>
      1 For each inserts, prepare the following mix : </P><P><U>Table 92</U></P> Volumes (μl) <STRONG><P>  PCR products </P></STRONG> 4 <STRONG><P>  Salt solution </P></STRONG> 1 <STRONG><P>  pET 43.1 </P></STRONG> 1 <STRONG><P>  Total </P></STRONG> 6 
2. Let ligate for 5 minutes at room temperature
3. Incubate for 10 minutes at 65°C.
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp10"><FIGURE><FIGCAPTION><P><U> Aim:</U> Check if our plasmid was inserted in the bacteria, and to recover larger amounts of it. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A></P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp11"><FIGURE><FIGCAPTION><P><U> Aim:</U> Increase the quality of DNA. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/b/be/T--Pasteur_Paris--PCR_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• dNTPs • MgCl<SUB>2</SUB>• Buffer 20X • RNAse free • 10 μl , 20 µl and 200 μl pipettes • 1.5 ml Eppendorfs • A1⁄A2 and B1⁄B2 inserts • PCR machine • 10 μl 20 µl and 200 μl pipettes • 1.5 ml Eppendorfs <U>Method:</U>
      1. Prepare a Globalmix with : </P><P><U>Table 93</U></P> Volumes (μl) <STRONG><P>  dNTPs </P></STRONG> 12.5<STRONG><P>  MgCl<SUB>2</SUB></P></STRONG> 12.5 <STRONG><P>  Buffer 20X </P></STRONG> 25 <STRONG><P>  RNAse free </P></STRONG> 182.5 <STRONG><P>  Primer S (For) </P></STRONG> 5 <STRONG><P>  Primer AS (Rev) </P></STRONG> 5 <STRONG><P>  Total </P></STRONG> 242.5 
2. Prepare the following samples :
                  <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 94</U></P></I></CAPTION><THEAD><TR><TH> A1 </TH><TH> A2 </TH><TH> B1 </TH><TH> B2 </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P>  V<SUB>inserts</SUB> (μl) </P></STRONG></TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> V<SUB>globalmix</SUB></P></STRONG></TD><TD align="center"> 48.5 </TD><TD align="center"> 48.5 </TD><TD align="center"> 48.5 </TD><TD align="center"> 48.5 </TD></TR></TBODY></TABLE>
3. Launch the PCR
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp12"><FIGURE><FIGCAPTION><P><U> Aim:</U> Check the efficiency of the PCR. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Agarose • Ethidium bromide drops • Molecular weight ladder (Gene ruler 1 Kb, Thermofisher) • Loading buffer 6X • PCR products • 10 μl pipette • Electrophoresis machine • 1.5 ml Eppendorfs <U>Method:</U>
      1. Make an agarose gel at 0.7%
2. Prepare the samples with 5 μl of PCR products and 1 μl of loading buffer 6X 
3. Deposit each sample in the wells : </P><P><U>Table 95</U></P> L1  L2  L3  L4  L5  L6  L7  L8  Ladder (6 μl) Ø A1 : 5 μl of DNA + 1 μl of buffer 6X  A2 : 5 μl of DNA + 1 μl of buffer 6X Ø D1 : 5 μl of DNA + 1 μl of buffer 6X  D2 : 5 μl of DNA + 1 μl of buffer 6X Ø
4. Begin the electrophoresis at 50 V for 15 minutes, then at 90 V for 1h30.
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp13"><FIGURE><FIGCAPTION><P><U> Aim:</U> Get DNA back. <U> Protocol:</U> follow in this link <U>Results</U></P><P><U>Figure 12 :</U> Digestion gel of C1 (1-8)</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp14"><FIGURE><FIGCAPTION><P><U> Aim:</U> Recombine the inserts with the plasmid. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Hind III (NEB) • Xba I 5NEB) • Buffer CutSmart 10X (NEB) • H<SUB>2</SUB>O • 1.5 Eppendorfs • C1 v2 and C2 v2 • Incubator 37°C • 20 μl and 200 μl pipettes <U>Method:</U>
      1. Prepare a Globalmix : </P><P><U>Table 96</U></P> Volumes (μl) <STRONG><P> HindIII </P></STRONG> 20 <STRONG><P>  XbaI </P></STRONG> 20 <STRONG><P>  Buffer CutSmart 10X </P></STRONG> 50 <STRONG><P>  H<SUB>2</SUB>O </P></STRONG> 10 <STRONG><P> Total </P></STRONG> 100 
2. Put 5 μl of the mix in each tube and add 20 μl of DNA C1 v2 and C2 v2 
3. Let digest for 2 hours at 37°C and inactivate enzymes for 5 minutes at 65°C
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp15"><FIGURE><FIGCAPTION><P><U> Aim:</U> Check if our inserts have been correctly digested by the enzyme. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><U>What we did in the lab:</U><U>Method:</U>
Lane 1 : </P><P><U>Table 97</U></P> 1  2  3  4  5  6  7  8  9  10  11  12  13  Ladder Ø C1.1 Ø C1.2 Ø C1.3 Ø C1.4  14  15  16  17  18  19  20  21  22  23  24  25 Ø C1.5 Ø C1.6 Ø C1.7 Ø C1.8 
Lane 2: <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 98</U></P></I></CAPTION><THEAD><TR><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4 </TH><TH> 5 </TH><TH> 6 </TH><TH> 7 </TH><TH> 8 </TH><TH> 9 </TH><TH> 10 </TH><TH> 11 </TH><TH> 12 </TH><TH> 13 </TH></TR></THEAD><TBODY><TR><TD align="center"> Ladder </TD><TD align="center">Ø</TD><TD align="center"> C2.2 </TD><TD align="center">Ø</TD><TD align="center"> C2.3 </TD><TD align="center">Ø</TD><TD align="center"> C2.4 </TD><TD align="center">Ø</TD><TD align="center"> C2.5 </TD></TR></TBODY><THEAD><TR><TH> 14 </TH><TH> 15 </TH><TH> 16 </TH><TH> 17 </TH><TH> 18 </TH><TH> 19 </TH><TH> 20 </TH><TH> 21 </TH><TH> 22 </TH><TH> 23 </TH><TH> 24 </TH><TH> 25 </TH></TR></THEAD><TBODY><TR><TD align="center">Ø</TD><TD align="center"> C2.6 </TD><TD align="center">Ø</TD><TD align="center"> C2.8 </TD><TD align="center">Ø</TD><TD align="center"> C2.9 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center">Ø</TD></TR></TBODY></TABLE></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp16"><FIGURE><FIGCAPTION><P><U> Aim:</U> Get back the DNA which was digested and purified. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4e/T--Pasteur_Paris--Gel_extraction_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials</U>• Qiagen Gel Extraction Kit <U>Method:</U>
Use Qiagen extraction kit with a final volume of 50 μl. <U>Results</U></P><P><U>Table 99</U></P> Sample  Weight of gel (g) <STRONG><P> C1.2 </P></STRONG> 1.3435 <STRONG><P> C1.4 </P></STRONG> 1.3564 <STRONG><P> C1.5 </P></STRONG> 1.3825 <STRONG><P> C1.6 </P></STRONG> 1.4318 <STRONG><P> C1.7 </P></STRONG> 1.4392 <STRONG><P> C1.8 </P></STRONG> 1.3564 <STRONG><P> C1.10 </P></STRONG> 1.3800 <STRONG><P> C2.1 </P></STRONG> 1.3179 <STRONG><P> C2.2 </P></STRONG> 1.2500 <STRONG><P> C2.3 </P></STRONG> 1.3910 <STRONG><P> C2.9 </P></STRONG> 1.3668 <STRONG><P> C2.10 </P></STRONG> 1.3934 </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp17"><FIGURE><FIGCAPTION><P><U> Aim:</U> Put the insert into the plasmid and transform the plasmid into bacteria. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/6f/T--Pasteur_Paris--Ligation_protocol.pdf">link</A><U>Results (on the August 4, 2016)</U>
Several colonies have grown but they are too small so we let them grow more. But C1.2 has much more colonies so we spread it into another Petri dish. </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp18"><FIGURE><FIGCAPTION><P><U> Aim:</U> Get back the DNA from bacteria. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A><U> Results </U></P><P><U>Figure 13 :</U> Digestion gel of B1/B2 (1-10)</P><CENTER><I><P><U>Figure 14 :</U> Digestion gel of E1/E2 (1-10)</P></I></CENTER></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp19"><FIGURE><FIGCAPTION><P><U> Aim:</U> Prepare the ligation. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials</U>• 1.5 ml Eppendorfs • DNA • Buffer CutSmart (NEB) • HindIII (NEB) • XbaI (NEB) • H<SUB>2</SUB>O • Samples B1⁄B2, E1⁄E2 • 10 μl and 20 μl pipettes <U>Method:</U>
1. Make a master mix with : </P><P><U>Table 100</U></P> Volumes (μl)  pET 43.1  B1⁄B2⁄E1⁄E2 <STRONG><P> DNA </P></STRONG> 7.5  7.5 <STRONG><P> Buffer CutSmart </P></STRONG> 2.5  7.5 <STRONG><P> HindIII </P></STRONG> 1  1 <STRONG><P> XbaI </P></STRONG> 1  1 <STRONG><P> H<SUB>2</SUB>O </P></STRONG> 1.5  0.5 <STRONG><P> Total </P></STRONG> 25  25 
	2. Distribute the volumes in each tube
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp20"><FIGURE><FIGCAPTION><P><U> Aim:</U> Avoid the religation of the plasmid with itself. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/c/c0/T--Pasteur_Paris--dephosphorylation_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials</U>• 1.5 ml Eppendorfs • Digested DNA • Dephosphorylase rSAP (recombinant Shrimp alkaline phosphatase) (NEB) • Buffer CutSmart • H<SUB>2</SUB>O • Incubator 37°C • 10 μl and 20 μl pipettes
<U>Method:</U>
1. In a 1.5 ml Eppendorf, put : </P><P><U>Table 101</U></P> Volumes (μl) <STRONG><P> Digested DNA </P></STRONG> 9 <STRONG><P> Dephosphorylase</P></STRONG> 0.2 <STRONG><P> Buffer CutSmart </P></STRONG> 2 <STRONG><P> H<SUB>2</SUB>O </P></STRONG> 8.8 <STRONG><P> Total </P></STRONG> 20 
	2. Incubate one hour at 37°C
3. Incubate 5 minutes at 65°C
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp21"><FIGURE><FIGCAPTION><P><U> Aim:</U> Increase the quantity of DNA. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/b/be/T--Pasteur_Paris--PCR_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials</U>• dNTP mix • MgCl<SUB>2</SUB> 25 mM • Buffer 10X • RNAse free water • Primer S • Primer AS • Samples A1⁄A2 and D1⁄D2 • 10 μl , 20 μl and 200 μl pipettes • PCR machine • Takara DNA polymerase enzyme • 1.5 ml Eppendorfs
<U>Method:</U>
1. Prepare the mix : </P><P><U>Table 102</U></P> Volumes (μl) <STRONG><P> dNTPs </P></STRONG> 12.5 <STRONG><P> MgCl<SUB>2</SUB></P></STRONG>Ø<STRONG><P> Buffer 10X </P></STRONG> 25 <STRONG><P> RNAse free </P></STRONG> 195 <STRONG><P> Primer S (For) </P></STRONG> 5 <STRONG><P> Primer AS(Rev) </P></STRONG> 5 <STRONG><P> Total </P></STRONG> 242.5 
	2. To put the sample into the PCR machine, use the table : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 103</U></P></I></CAPTION><THEAD><TR><TH> Tube </TH><TH> Names </TH><TH> Mix (μl) </TH><TH> A1 (μl)  </TH><TH> A2 (μl)  </TH><TH> D1 (μl)  </TH><TH> D2 (μl)  </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> 1 </P></STRONG></TD><TD align="center"> A1 </TD><TD align="center"> 48.5 </TD><TD align="center"> 1 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> 2 </P></STRONG></TD><TD align="center"> A2 </TD><TD align="center"> 48.5 </TD><TD align="center">Ø</TD><TD align="center"> 1 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> 3 </P></STRONG></TD><TD align="center"> D1 </TD><TD align="center"> 48.5 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center"> 1 </TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> 4 </P></STRONG></TD><TD align="center"> A1 </TD><TD align="center"> 48.5 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center"> 1 </TD></TR></TBODY></TABLE>
3. For hot start PCR, launch the PCR machine and once it is at 95°C, add 0.5 μl of Takara enzyme in each tube<U>Results</U><CENTER>2&quot;/&gt;
<I><P><U>Figure 15 :</U> PCR gel of A1/A2/D1/D2 without MGCl<SUB>2</SUB></P></I></CENTER></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp22"><FIGURE><FIGCAPTION><P><U> Aim:</U> Have more bacteria with the insert A. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4b/T--Pasteur_Paris--Bacterial_culture_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials</U>• 50 ml Falcon • LB medium • 20 ml and 20 μl pipettes • TOPO • C1 v2 and C2 v2 • Carbenicillin 50 mg⁄ml • Incubator • LB medium
<U>Method:</U>
1. In a 50 ml Falcon, put 15 ml of LB medium 
2. Add 15 μl of carbenicillin 
3. Add a colony in the Falcon. But for C1 v2 (2) we had to wait one more night because bacteria had not grown enough 
4. Let it incubate at 37°C
</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp23"><FIGURE><FIGCAPTION><P><U> Aim:</U> Check if the PCR is efficient. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials</U>• Agarose • A1⁄A2 and D1⁄D2 • Molecular weight marker ladder (Gene ruler 1 Kb, Thermofisher) • Loading buffer 6X • 10 μl pipette
<U>Method:</U>
1. Make a 0.7% agarose gel 
2. Add 5 μl of DNA and 1 μl of loading buffer. But we used 10 μl of DNA for the sample 4 
3. Deposit the sample for the electrophoresis according to this: </P><P><U>Table 104</U></P> L1  L2  L3  L4  L5  L6  L7  L8  L19 Ladder 6 μl Ø (1) Ø (2) Ø (3) Ø (4) <U>Results</U>
A lot of streaks appeared so it did not work as expected. We have to redo this experiment with the gradient mode
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp24"><FIGURE><FIGCAPTION><P><U> Aim:</U> Increase the quantity of DNA. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/b/be/T--Pasteur_Paris--PCR_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials</U>• dNTP mix • MgCl<SUB>2</SUB> 25 mM • Buffer 10X • RNAse free • Primer S • Primer AS • A1⁄A2 and D1⁄D2 • 200 μl pipette
<U>Method:</U>
1. Make a Globalmix with : </P><P><U>Table 105</U></P> Volumes (μl) <STRONG><P> dNTPs </P></STRONG> 72.5 <STRONG><P> MgCl<SUB>2</SUB></P></STRONG>Ø<STRONG><P> Buffer 10X </P></STRONG> 145 <STRONG><P> RNAse free </P></STRONG> 1131 <STRONG><P> Primer S </P></STRONG> 29 <STRONG><P> Primer AS </P></STRONG> 29 <STRONG><P> Total </P></STRONG> 1206.5 
	2. Prepare the samples : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 106</U></P></I></CAPTION><THEAD><TR><TH> Name </TH><TH> Samples </TH><TH> Globalmix (μl)  </TH><TH> DNA (μl)  </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> A1 </P></STRONG></TD><TD align="center"> to (7) </TD><TD align="center"> 48.5 </TD><TD align="center"> 1 of A1 </TD></TR><TR><TD align="center"><STRONG><P> A2 </P></STRONG></TD><TD align="center"> (8) to (14) </TD><TD align="center"> 48.5 </TD><TD align="center"> 1 of A2 </TD></TR><TR><TD align="center"><STRONG><P> D1 </P></STRONG></TD><TD align="center"> (15) to (21) </TD><TD align="center"> 48.5 </TD><TD align="center"> 1 of D1 </TD></TR><TR><TD align="center"><STRONG><P> D2 </P></STRONG></TD><TD align="center"> (22) to (28) </TD><TD align="center"> 48.5 </TD><TD align="center"> 1 of D2 </TD></TR></TBODY></TABLE>
3. Deposit the samples on the gel: <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 107</U></P></I></CAPTION><THEAD><TR><TH> A1 </TH><TH> A2  </TH><TH> D1</TH><TH> D2</TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> A </P></STRONG></TD><TD align="center"> (1) </TD><TD align="center"> (8) </TD><TD align="center"> (15) </TD><TD align="center"> (22) </TD></TR><TR><TD align="center"><STRONG><P> B </P></STRONG></TD><TD align="center"> (2) </TD><TD align="center"> (9) </TD><TD align="center"> (16) </TD><TD align="center"> (23) </TD></TR><TR><TD align="center"><STRONG><P> C </P></STRONG></TD><TD align="center"> (3) </TD><TD align="center"> (10) </TD><TD align="center"> (17) </TD><TD align="center"> (24) </TD></TR><TR><TD align="center"><STRONG><P> D </P></STRONG></TD><TD align="center"> (4) </TD><TD align="center"> (11) </TD><TD align="center"> (18) </TD><TD align="center"> (25) </TD></TR><TR><TD align="center"><STRONG><P> E </P></STRONG></TD><TD align="center"> (5) </TD><TD align="center"> (12) </TD><TD align="center"> (19) </TD><TD align="center"> (26) </TD></TR><TR><TD align="center"><STRONG><P> F </P></STRONG></TD><TD align="center"> (6) </TD><TD align="center"> (13) </TD><TD align="center"> (20) </TD><TD align="center"> (27) </TD></TR><TR><TD align="center"><STRONG><P> G </P></STRONG></TD><TD align="center"> (7) </TD><TD align="center"> (14) </TD><TD align="center"> (21) </TD><TD align="center"> (28) </TD></TR></TBODY></TABLE>
4. Make a 0.7% agarose gel and perform an electrophoresis with 5 µl of DNA and 1 µl of loading 6X buffer for each sample <U>Results</U><CENTER><I><P><U>Figure 16 :</U> PCR gel of A1/A2/B1/B2/D1/D2/E1/E2 </P></I></CENTER></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp25"><FIGURE><FIGCAPTION><P><U> Aim:</U> Increase the DNA. <U>What we did in the lab:</U><U>Materials</U>• C1⁄C2 • Na Acetate (NaOAc) • Ethanol (70%) • Incubator • Buffer E • 200 μl and 1000 μl pipette <U>Method:</U>
1. Calculate the final volume for each insert : For C1 :	50×7 = 350 μlFor C2 :	50×5 = 250 μl
2. Add 1⁄10 of the volume of NaOAc and 2.5x volume of ethanol : For C1 :	35 μl of NaOAc and 875 μl of ethanol For C2 :	25 μl of NaOAc and 625 μl of ethanol 
3. Mix the samples and incubate 8 minutes at -80°C 
4. Centrifuge 5 minutes at 4°C and 15000 RPM. Then, throw out the supernatant 
5. Add 1 ml of ethanol ice cold (70%) 
6. Centrifuge 15 minutes at 4°C and 15000 RPM 
7. Throw out the supernatant and let dry at room temperature 
8. Resuspend the pellet in 50 μl of buffer E.<U>Results</U></P><P><U>Figure 17 :</U> PCR gel of C1 v2 / C2 v2 with Takara enzyme </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp26"><FIGURE><FIGCAPTION><P><U> Aim:</U> Make the inserts ligate with the plasmid. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials</U>• 10 μl, 20 μl and 200 μl pipette • Hind III (NEB) • Xba I (NEB) • Buffer CutSmart • H<SUB>2</SUB>O • B1⁄B2 and C1⁄C2
<U>Method:</U>
1. Make a Globalmix with : </P><P><U>Table 108</U></P> Volumes (μl) <STRONG><P> Hind III </P></STRONG> 45 <STRONG><P> Xba I </P></STRONG> 45 <STRONG><P> Buffer CutSmart </P></STRONG> 112.5 <STRONG><P> H<SUB>2</SUB>O </P></STRONG> 67.5 <STRONG><P> Total </P></STRONG> 270 
	2. Distribute this global mix in each of our 4 tubes so they will contain : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 109</U></P></I></CAPTION><THEAD><TR><TH> Volumes (μl)  </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> DNA </P></STRONG></TD><TD align="center"> 20 </TD></TR><TR><TD align="center"><STRONG><P> Hind III </P></STRONG></TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> Xb aI </P></STRONG></TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> Buffer CutSmart </P></STRONG></TD><TD align="center"> 2.5 </TD></TR><TR><TD align="center"><STRONG><P> H<SUB>2</SUB>O </P></STRONG></TD><TD align="center"> 1.5 </TD></TR><TR><TD align="center"><STRONG><P> Total </P></STRONG></TD><TD align="center"> 26 </TD></TR></TBODY></TABLE>
3. Follow the protocol for digestion. 
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp27"><FIGURE><FIGCAPTION><P><U> Aim:</U> Prepare the plasmid for the transformation. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/6f/T--Pasteur_Paris--Ligation_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials</U>• C1⁄C2 • pET43.1 • T4 ligase • Buffer 10X • H<SUB>2</SUB>O • Incubator • 10 μl and 20 μl pipettes
<U>Method:</U>
1. In a 1.5 ml eppendorf, put : </P><P><U>Table 110</U></P> C1  C2  pET 43.1 (a+) alone <STRONG><P> C1 (μl) </P></STRONG> 15 ØØ<STRONG><P> C2 (μl) </P></STRONG>Ø 15 Ø<STRONG><P> pET 43.1a(+) (μl) </P></STRONG> 4  4  4 <STRONG><P> T4 ligase (μl) </P></STRONG> 1  1  1 <STRONG><P> Buffer 10X (μl) </P></STRONG> 2.2  2.2  2.2 <STRONG><P> H<SUB>2</SUB>O (μl) </P></STRONG>ØØ 15 <STRONG><P> Total (μl) </P></STRONG> 22.2  22.2  22.2 
	2. Incubate for 1 hour at room temperature
3. Incubate for 10 minutes at 65°C
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp28"><FIGURE><FIGCAPTION><P><U> Aim:</U> Express our plasmid in bacteria. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials</U>• Qiagen Extraction Kit
<U>Method:</U>
Use the Qiagen gel extraction kit for a final volume of 50 μl. </P><P><U>Table 111</U></P> Colonies Δm (mg) <STRONG><P> B1 colony 1 </P></STRONG> 136 <STRONG><P> B1 colony 2 </P></STRONG> 170 <STRONG><P> B2 colony 4 </P></STRONG> 166 <STRONG><P> B2 colony 7 </P></STRONG> 175 <STRONG><P> E1 colony 1 </P></STRONG> 110 <STRONG><P> E1 colony 2 </P></STRONG> 143 <STRONG><P> E2 colony 1 </P></STRONG> 108  E2 colony 2  128 &lt;</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp29"><FIGURE><FIGCAPTION><P><U> Aim:</U> Ligate our inserts with the plasmid. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/6f/T--Pasteur_Paris--Ligation_protocol.pdf">link</A><U>What we did in the lab:</U><U>Method:</U>
We use the next volumes: </P><P><U>Table 112</U></P> Volumes (μl) <STRNG><P> Insert (B1 or B2 or C1 or C2) </P> 15 <STRONG><P> pET43.1 (100 ng) </P></STRONG> 2 <STRONG><P> T4 ligase </P></STRONG> 1 <STRONG><P> Buffer 10X </P></STRONG> 2 <STRONG><P> Total </P></STRONG> 20 </STRNG></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp30"><FIGURE><FIGCAPTION><P><U> Aim:</U> Put our ligated plasmid into bacteria. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A><U>Results</U>
48 hours later, no colony had grown so the transformation did not work, it has to be redone.
</P></FIGCAPTION></FIGURE></DIV></DIV></BODY></HTML>