<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Microbiology_week11 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Microbiology week11</SPAN></H1></DIV></DIV></DIV><DIV class="container"><DIV id="position-menu2"><UL id="menu2"><LI><FONT size="3pt" color="#8B8378">About us</FONT></LI><LI><A href=" https://2016.igem.org/Team:Pasteur_Paris/AboutUs " href="">Our team</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Sponsors">Sponsors</A></LI></UL><LI><FONT size="3pt" color="#8B8378">Project</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Context">Context</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Overview">Overview</A></LI><LI><FONT size="3pt" color="#8B8378">Design </FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Science">Science </A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Moskit_devices">Devices</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Research">Research &amp; development</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Methodology">Methodology</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Design_Biology">Design vs Biology</A></LI><LI><FONT size="3pt" color="#8B8378">Labwork </FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Safety_Security">Safety &amp; Security</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Protocol ">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Netbook ">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Results ">Results</A></LI><LI>Human Practice</LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Collaborations ">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Scenarios">Scenarios</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Meet-up ">Meet-up</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Engagement">Communication</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Law"> Law</A></LI><LI><FONT size="3pt" color="#8B8378">Parts</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Description">Description</A></LI><LI><FONT size="3pt" color="#8B8378">Achievements</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/medals">Medals</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Prizes"> Special Prizes</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/rewards"> Awards obtained!</A></LI></DIV></DIV><DIV id="weeks"><DIV id="weekslist"><CENTER><H1><B>click on the weeks</B></H1></CENTER></DIV></DIV><DIV><H2><B>Microbiology Notebook</B></H2></DIV><DIV id="week11"><H5><B>Week 11</B></H5><H3><B>August 17, 2016:</B></H3><H4> 188. Miniprep of precultures B1col1/B1col2 and C2col1 </H4><A href="#exp2"><H4> 189. Digestion of the plasmid pET43.1a(+) with A1/A2/D1/D2 </H4></A><A href="#exp3"><H4> 190. Electrophoresis on agarose gel of digestion products A1/A2/D1/D2/B1 col1/B1 col2/C2</H4></A><A href="#exp4"><H4> 191. Harvest the culture with Midiprep A1/A2/D1/D2 </H4></A><H3><B>August 18, 2016:</B></H3><H4> 192. Miniprep of A1/A2/D1/D2</H4><A href="#exp6"><H4> 193. Purification of the protein </H4></A><A href="#exp7"><H4> 194. Protein gel on SDS-Page </H4></A><A href="#exp8"><H4> 195. Harvest the culture with Miniprep 4 colonies from A1, A2, D1 and D2, 2 colonies of B1 and 1 colony of C2</H4></A><H3><B>August 19, 2016:</B></H3><H4> 196. Miniprep of cultures made on the 18/08 </H4><A href="#exp10"><H4> 197. Measure the amount of DNA extracted from the miniprep of B1 and C2 </H4></A><A href="#exp11"><H4> 198. Digestion of the plasmid pET43.1a(+) with A1(0)/A1(1)/A1(3)/A1(4)/D1(3)/D1(4)/D2(2) </H4></A><A href="#exp12"><H4> 199. Electrophoresis on agarose gel of digestion products </H4></A><A href="#exp13"><H4> 200. Harvest preculture for miniprep of C2 and B1</H4></A><H3><B>August 20, 2016:</B></H3><A href="#exp14"><H4> 201. Miniprep of C2 and B1</H4></A><A href="#exp15"><H4> 202. Measure the amount of DNA extracted from the miniprep of B1 and C2</H4></A><A href="#exp16"><H4> 203. Digestion of E1(4 tubes)/E2(12tubes) and B2(14 tubes)</H4></A><A href="#exp17"><H4> 204. Electrophoresis on agarose gel of digestion products </H4></A><DIV class="lightbox" id="exp1"><FIGURE><FIGCAPTION><P><U> Aim:</U> To perform a Miniprep to isolate plasmid DNA of pET43.1a(+) with the inserts B1 col1, B1 col2 and C2 col1. The amplification method to increase the amount of plasmid is called Miniprep. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• 50 ml Falcon tube• Shaking incubator (INFORS HT)• Swing bucket centrifuge (JOUAN GR41)• QIAGEN Miniprep kit• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link)

<U>Method:</U>The protocol in step 1 ask for spinning at 6000g but we can only achieve 3500 g so we used 3500 g for 8 minutes. We will follow most of the protocol of QIAGEN Miniprep 2016 except for a few modifications, which we describe, therefore, below.
                      1.Follow QIAGEN kit steps</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp2"><FIGURE><FIGCAPTION><P><U> Aim:</U> To get back our insert from the Miniprep of our inserts with appropriate enzymes. 
We perform restriction enzyme digestion in order to recover our inserts. We choose appropriate restriction sites based on the host plasmid. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Restriction enzymes: XbaI, HindIII (New England Biolabs, NEB) • Restriction enzyme buffers • 37°C water bath• UV spectrophotometer<U>Method:</U>
                       1. Mix all the reagents and let digest during 2 hr at 37°C.  Big volumes must be added first!Beginning of digestion 12h10.</P><P><U>Table 125</U></P>ReactantsA1D1<STRONG><P>Vol<SUB>DNA</SUB></P></STRONG>20 µL 20 µL <STRONG><P>Vol<SUB>XbaI</SUB></P></STRONG>1 µL 1 µL <STRONG><P>Vol<SUB>HindIII</SUB></P></STRONG>1 µL 1 µL <STRONG><P>Vol<SUB>H<SPAN>2</SPAN>O</SUB></P></STRONG>5 µL 5 µL <STRONG><P>Vol<SUB>Buffer Cutsmart (10X)</SUB></P></STRONG>3 µL 3 µL <STRONG><P>Vol<SUB>total</SUB></P></STRONG>30 µL 30 µL <CENTER>Volumes</CENTER>

	           2. Incubate 10 min at 65°C to inactivate the enzymes. 
	           3. Store at -20°C </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp3"><FIGURE><FIGCAPTION><P><U> Aim:</U> This step check the digestion efficiency. Moreover, the inserts will be purified during this step because they will be extracted from the gel.<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Restriction enzymes: XbaI, HindIII (New England Biolabs, NEB) • Restriction enzyme buffers • 37°C water bath• UV spectrophotometer<U>Method:</U>• Electrophoresis cuve • TAE 1X • Gene ruler (Thermoscientific 1kb plus) • Loading dye • Agarose • UV table • BET 
                     Beginning of the electrophoresis at 14h30 at 100V. <U>Results:</U>The gel reveals that A1 contains the insert but the amount of DNA is too low so we will redo the experiment.</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp4"><FIGURE><FIGCAPTION><P><U> Aim:</U>  To start a culture for Midiprep. In order to obtain a large amount of plasmid, we need to grow the bacteria overnight. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4b/T--Pasteur_Paris--Bacterial_culture_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Microbiology equipement • 25 ml flasks • Carbenicillin 50 mg/ml • Chloramphenicol 34 mg/ml • LB medium <U>Method:</U>
                  2.One colony is picked from the plates and shaken in 25 ml of LB supplemented with Carbenicillin at 50 μg/ml. This step is done with the inserts A1/A2/D1/D2 and B1/B2 for sequencing. 
                  3.The flask is placed in a shaking incubator at 37°C, 150 rpm overnight. </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp5"><FIGURE><FIGCAPTION><P><U> Aim:</U> To perform a Midiprep to isolate plasmid DNA of pET43.1a(+) with A1/A2/D1/D2 and B1/B2 for sequencing.<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A>:
<U>What we did in the lab:</U><U>Materials:</U>• 50 ml Falcon tube • Shaking incubator (INFORS HT) • Swing bucket centrifuge (JOUAN GR41) • QIAGEN Midiprep kit 2016 (QiaFilter, Cat No.ID: 28704)
<U>Method:</U>
The protocol in step 1 ask for spinning at 6000 g but we can only achieve 3500 g so we used 3500 g for 20 minutes. We will follow most of the protocol of QIAGEN Midiprep 2016 except for a few modifications, which we describe, therefore, below. 

1.	Use culture from overnight (17 hr) step on June 7, 2016 
2.	Pour culture in 50 ml Falcon nd centrifuge (15 min, 3500 g, 4°C) 
3.	Discard the supernatant (in biological waste) and add 4 ml of Buffer P1 (stored on ice) to the pellet 
4.	Add 4 ml of Buffer P2 (for cell lysis) and mix by inverting the Falcon a few times. Wait 5 min at 22°C (room temperature: RT, EU). Note: The color of the solution will change to blue. 
5.	Prepare syringes with their cap and the reservoir (on 50 ml Falcon) 
6.	Add 4 ml of Buffer P3 (for neutralization) to the Falcon and mix by inverting the tube a few times. Note: The color of the solution changes to white. 
7.	Pour the content of the Falcon in the syringes and let it sit for 10 min. In the meanwhile, equilibrate the provided columns with 4 ml of OBT (equilibration buffer) 
8.	Transfer the contents from the syringe to the column and wash with 2 X 10 ml of QC buffer 
9.	Prepare 10 tubes of 2 ml to aliquot pET43.1a(+) and pSB1C3. 
Because we have only bench microfuges, we need to dispense our volume in smaller fractions. 
10.	Elution of DNA with 5 ml of QF and aliquot in 2 ml tubes 
11.	Centrifuge (30 min, 15000 g, room temperature) 
12.	Add 3.5 ml of isopropanol, mix to precipitate the DNA 
13.	Centrifuge (30 min, 15 000 g, at RT) 
14.	Remove isopropanol with pipet without taking DNA and place into chemical waste container 
15.	Add 1 ml of 70% ethanol, centrifuge again (15 min, 15 000 g, RT) and let air dry.
16.	Resuspend in 50 µL of Tris 10 mM pH 8.0, EDTA, 1 mM (TE) and store at -20°C.</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp6"><FIGURE><FIGCAPTION><P><U> Aim:</U> The previous purification shows a significant band at 30kDa for the samples 22 and 24 but also one at 70kDa. It probably left some NusA in our column so, it will be cleaned.br&gt;
 <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/0/07/T--Pasteur_Paris--FPLC_Protein_purification_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>•  Fast Purification Liquid Chromatography •  Chaotropic reagent (Guanidinium 6M) •  EDTA 0,1M •  PMSF (100mM) •  Ni 2+ solution (100mM) •  Centrifuge (labo deshmukh)
<U>Method:</U>
1.	 Melt the pellet of bacteria C2 (from 1 L culture) and resuspend it with 10 ml of buffer A 
2.	Put the column off the FPLC and wash it with 20 ml of milliQ water thanks to a fingerpit ans a syringue. 
3.	Add 20 ml of chaotropic reagent to denaturate the proteins fixed to the column 
4.	Wash the column with 20 ml of water 
5.	Add 10 ml of EDTA to clean it from nickel 
6.	Wash with 20 ml of water 
7.	Add 5ml of Ni solution to charge the column. The column turns green. 
8.	Wash with 20 ml of water 
9.	Sonicate the sample three times one minute at 60%, wait 90 seconds between each sonication, Finally, the sample is 40 ml, add 40 &amp;microL of PMSF to avoid protein denaturation. 
10.	Centrifuge 25 min at 16000 g (rotor JA 25.50) 
11.	Inject your sample in the FPLC 
12.	Get back several samples: • C= Crude extract : before centrigugation • P= Pellet • SN= Supernatant • F= Flow through (unfixed proteins) • W= Wash 5% of buffer B • Fractions (depending on the gradient) </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp7"><FIGURE><FIGCAPTION><P><U> Aim:</U> Get the size of the protein purified thanks to FPLC in order to know if it is our protein  <U>What we did in the lab:</U><U>Materials:</U><U>Materials:</U>• SDS-Page cuve • SDS-Page gel (BIORAD) • Protein migration buffer • Protein ladder • Laemmli 2X • Coomassie Blue • Microbiology equipment (Follow this link) <U>Method:</U>
1.	In 9 1.5 ml eppendorf, put 20 µL of a sample and 20 µL of Laemmli 2X. 
2.	Place the gel into the cuve and fill it with migration buffer 
3.	Follow the next deposit table: • Protein ruler 8ul • Pellet • Supernatant • Wash • Flow through • Fraction 13 • Fraction 16 • Fraction 20 • Fraction 22 • Fraction 24 
4. Launch the migration at 120V (start at 12h10 and stop at 14h10). 
5. Wash the gel three times with distilled water during 5min. 
6. Color the gel with Coomassie Blue diluted 1/5 during 30min. 
7. Wash with distilled water for 5min then let wash 15min. <U>Method:</U>
We notice a 30kDa band in the well 9 and 10 so we redo a gel with the fractions 21 to 25. Follow exactly the same protocol but with 30 &amp;microL of DNA and 30 &amp;microL of Laemmli 2X. 
Deposit table: 
	- Protein ruler 8 µl 
	- Fraction 19 
	- Fraction 20 
	- Fraction 21 
	- Fraction 22 
	- Fraction 23 
	- Fraction 24 
	- Fraction 25 
	- Fraction 26 
	- Fraction 27 
We notice a 30kDa band in the fractions 19 to 21 that may correspond to our protein and a 70kDa band due to NusA in fractions 23 to 25.</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp8"><FIGURE><FIGCAPTION><P><U> Aim:</U> To start a culture for Miniprep. 
	In order to obtain a large amount of plasmid, we need to grow the bacteria overnight. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4b/T--Pasteur_Paris--Bacterial_culture_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U><U>Materials:</U>• Microbiology equipement • 15 ml Falcon tube • Carbenicillin 50 mg/ml • LB medium <U>Method:</U>
17.	One colony is picked from the plates and shaken in 3 ml of LB supplemented with Carbenicillin at 50 µg/ml. This step is done with the inserts A1/A2/D1/D2/C2 and  two colonies B1. 
18.	The Falcon tube is placed in a shaking incubator at 37°C, 150 rpm overnight. </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp9"><FIGURE><FIGCAPTION><P><U> Aim:</U> To perform a Miniprep to isolate plasmid DNA of pET43.1a(+) with the inserts A1 (4 tubes) / B1(2 tubes) / C2 (1 tube) / D1 (2 tubes) and D2 (1 tube).  
The amplification method to increase the amount of plasmid is called Miniprep. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>•  Shaking incubator (INFORS HT) •  Swing bucket centrifuge (JOUAN GR41) •  QIAGEN Miniprep kit •  Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link) <U>Method:</U>
	  The protocol in step 1 ask for spinning at 6000 g but we can only achieve 3500 g so we used 3500 g for 8 minutes. We will follow most of the protocol of QIAGEN Miniprep 2016 except for a few modifications, which we describe, therefore, below. 

19.	Follow QIAGEN kit steps </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp10"><FIGURE><FIGCAPTION><P><U> Aim:</U> Measure the quantity of plasmid using a Nanodrop (Thermofisher) before sending for sequencing (inserts B1 and C2)<U>What we did in the lab:</U><U>Materials:</U>• Nanodrop (Thermofisher)• Elution buffer from QIAGEN kit• Microbiology equipment (Follow this link)<U>Method:</U>
Analyze absorbance at 260nm
1.	Clean the Nanodrop with water
2.	Make the blank with 1 µl of elution buffer
3.	Put 1ul of your sample on the Nanodrop
4.	Make the measure and clean the Nanodrop between each measure<U>Results:</U></P><P><U>Table 126</U></P>λ= 260 nmB1(1)B1(2)C2&gt;th&gt;
    <STRONG><P>A<SUB>DNA</SUB></P></STRONG>0.7250.7410.761<STRONG>C final</STRONG>36.3 ng/µl37.0 ng/µl38.0 ngµl<U>Absorbance</U>

5. Preparation of the samples for sequencing: put 15 µl of DNA and add 2 µl of fitted primers
6. Send the samples for sequencing</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp11"><FIGURE><FIGCAPTION><P><U> Aim:</U> To get back our insert from the miniprep with appropriate enzymes.
We perform restriction enzyme digestion in order to recover our inserts. We choose appropriate restriction sites based on the host plasmid.<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Restriction enzymes: XbaI, HindIII (New England Biolabs, NEB) • Restriction enzyme buffers • 37°C water bath • UV spectrophotometer <U>Method:</U>
4.	Mix all the reagents and let digest during 2 hr at 37°C 
 Big volumes must be added first! 


Beginning of digestion 11h45. </P><P><U>Table 127</U></P>ReactantsEach sample<STRONG><P>Vol<SUB>DNA</SUB></P></STRONG>30  µL <STRONG><P>Vol<SUB>XbaI</SUB></P></STRONG>1  µL<STRONG><P>Vol<SUB>HindIII</SUB></P></STRONG>1  µL <STRONG><P>Vol<SUB>H<SPAN>2</SPAN>O</SUB></P></STRONG>13  µL <STRONG><P>Vol<SUB>Buffer Cutsmart (10X)</SUB></P></STRONG>5  µL <STRONG><P>Vol<SUB>total</SUB></P></STRONG>50  µL 
Volumes


	2. Incubate 10 min at 65°C to inactivate the enzymes.
	3. Store at -20°C</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp12"><FIGURE><FIGCAPTION><P><U> Aim:</U> This step check the digestion efficiency of A1 (4 tubes)/D1 (2 tubes)/ D2 (1 tube). Moreover, the inserts will be purified during this step because they will be extracted from the gel.<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Electrophoresis cuve • TAE 1X • Gene ruler (Thermoscientific 1kb plus) • Loading dye • Agarose • UV table • BET <U>Method:</U> 
Each well will contain 30 µL of DNA and 6 µL of Loading Dye.  
Follow the next deposit table : 
Loading dye (6 µL ) / A1 (1) / A1 (2) / A1 (3) / A1 (4) / D1 (1) / D1 (2) / D2 (2) 
Beginning of the electrophoresis at 14h20 at 100 V.<U>Results:</U>
The gel reveals that A1 (1) and (2) contains the insert but the amount of DNA is too high to be purified. The other clones do not have an insert.
</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp13"><FIGURE><FIGCAPTION><P><U> Aim:</U> To start a culture for Midiprep of insert C2 and B1.  
	In order to obtain a large amount of plasmid, we need to grow the bacteria overnight.<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4b/T--Pasteur_Paris--Bacterial_culture_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Microbiology equipement • 25 ml flasks• Carbenicillin 50 mg/ml• LB medium<U>Method:</U>
5.	One colony is picked from the plates and shaken in 25 ml of LB supplemented with Carbenicillin at 50 µg/ml.
The flask is placed in a shaking incubator at 37°C, 150 rpm overnight.</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp14"><FIGURE><FIGCAPTION><P><U> Aim:</U> To perform a midiprep to isolate plasmid DNA pET43.1a(+) with the inserts C2 and B1.
	The amplification method to increase the amount of plasmid is called Mini or Midiprep.<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• 50 ml Falcon tube• Shaking incubator (INFORS HT)• Swing bucket centrifuge (JOUAN GR41)• QIAGEN Midiprep kit 2016 (QiaFilter, Cat No.ID: 28704)<U>Method:</U>  The protocol in step 1 ask for spinning at 6000 g but we can only achieve 3500 g so we used 3500 g for 20 minutes. We will follow most of the protocol of QIAGEN Midiprep 2016 except for a few modifications, which we describe, therefore, below.

6.	Use culture from overnight (17 hr) step on June 7, 2016 
7.	Pour culture in 50 ml Falcon nd centrifuge (15 min, 3500 g, 4°C)
8.	Discard the supernatant (in biological waste) and add 4 ml of Buffer P1 (stored on ice) to the pellet 
9.	Add 4 ml of Buffer P2 (for cell lysis) and mix by inverting the Falcon a few times. Wait 5 min at 22°C (room temperature: RT, EU). Note: The color of the solution will change to blue.
10.	Prepare syringes with their cap and the reservoir (on 50 ml Falcon)
11.	Add 4 ml of Buffer P3 (for neutralization) to the Falcon and mix by inverting the tube a few times. Note: The color of the solution changes to white.
12.	Pour the content of the Falcon in the syringes and let it sit for 10 min. In the meanwhile, equilibrate the provided columns with 4 ml of OBT (equilibration buffer)
13.	Transfer the contents from the syringe to the column and wash with 2 X 10 ml of QC buffer
14.	Prepare 10 tubes of 2 ml to aliquot pET43.1(+) and pSB1C3. 
Because we have only bench microfuges, we need to dispense our volume in smaller fractions.
1.	Elution of DNA with 5 ml of QF and aliquot in 2 ml tubes
2.	Centrifuge (30 min, 15000g, room temperature)
3.	Add 3.5 ml of isopropanol, mix to precipitate the DNA
4.	Centrifuge (30 min, 15 000g, at RT)
5.	Remove isopropanol with pipet without taking DNA and place into chemical waste container
6.	Add 1 ml of 70% ethanol, centrifuge again (15 min, 15 000 g, RT) and let air dry
7.	Resuspend in 50 &amp;microL of Tris 10 mM pH 8.0, EDTA, 1 mM (TE) and store at -20°C</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp15"><FIGURE><FIGCAPTION><P><U> Aim:</U>  Measure the quantity of plasmid using a Nanodrop (Thermofisher) before sending for sequencing (inserts B1 and C2)<U>What we did in the lab:</U><U>Materials:</U>• Nanodrop (Thermofisher)• Elution buffer from QIAGEN kit• Microbiology equipment (Follow this link)<U>Method:</U>
Analyze absorbance at 260 nm
15.	Clean the Nanodrop with water
16.	Make the blank with 1ul of elution buffer
17.	Put 1ul of your sample on the Nanodrop
18.	Make the measure and clean the Nanodrop between each measure<U>Results:</U></P><P><U>Table 128</U></P>λ= 260 nmB1(a)B1(b)C2(a)C2(b)<STRONG><P>A<SUB>260</SUB></P></STRONG>1.057/td&gt;
      1.3230.9710.148<STRONG>A<SUB>280</SUB></STRONG>0.6270.6980.5710.104<STRONG>A<SUB>260</SUB>/A<SUB>280</SUB></STRONG>1.691.891.701.43<STRONG>C final</STRONG>52.8 ng/µl66.2 ngµl48.6 ng/µl7.4 ng/µl
Absorbance
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp16"><FIGURE><FIGCAPTION><P><U> Aim:</U>To get back our insert from the miniprep with appropriate enzymes.
We perform restriction enzyme digestion in order to recover our inserts. We choose appropriate restriction sites based on the host plasmid.<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>•  Restriction enzymes: XbaI, HindIII (New England Biolabs, NEB) •  Restriction enzyme buffers •  37°C water bath •  UV spectrophotometer <U>Method:</U>
19.	Mix all the reagents and let digest during 2 hr at 37°C 
 Big volumes must be added first! Make a global mix to be more accurate</P><P><U>Table 129</U></P>ReactantsEach sampleGlobal mix<STRONG><P>Vol<SUB>DNA</SUB></P></STRONG>45 µl 0 µl <STRONG><P>Vol<SUB>XbaI</SUB></P></STRONG>2.25 µl 65.25 µl <STRONG><P>Vol<SUB>HindIII</SUB></P></STRONG>2.25 µl 65.25 µl <STRONG><P>Vol<SUB>H<SPAN>2</SPAN>O</SUB></P></STRONG>1.125 µl32.65 µl<STRONG><P>Vol<SUB>Buffer Cutsmart (10X)</SUB></P></STRONG>5.63 µl 163.12 µl <STRONG><P>Vol<SUB>total</SUB></P></STRONG>56 /µl 326.27 /µl 
Volumes


	2. Incubate 10 min at 65°C to inactivate the enzymes.
	3. Store at -20°C
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp17"><FIGURE><FIGCAPTION><P><U> Aim:</U>This step check the digestion efficiency of E1(3 tubes) / E2 (12 tubes) / B2(14 tubes). Moreover, the inserts will be purified during this step because they will be extracted from the gel. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Electrophoresis cuve • TAE 1X • Gene ruler (Thermoscientific 1kb plus) • Loading dye • Agarose • UV table • BET <U>Method:</U> Each well can contain 40 µl so we made two big gels with 20x2 lines on each.  Each sample will contain 62 µl as we add 7 µl of loading dye. They will be divided into two wells. 
Deposit table (/// means EMPTY to make the cut easier) 

Gel 1 Line 1 : 
Loading dye /// B2(2) / B2(2) /// B2(3) / B2(3) /// B2(4) / B2(4) /// B2(5) / B2(5) /// B2(1) / B2(1) /// B2(6) / B2(6) 

Gel 1 Line 2 : 
Loading dye /// B2(7) / B2(7) /// B2(8) / B2(8) /// B2(9) / B2(9) /// B2(10) / B2(10) /// B2(11) / B2(11) /// B2(12) / B2(12) 


Gel 2 Line 1 : 
Loading dye /// E1(1) / E1(1) /// E1(2) / E1(2) /// E1(3) / E1(3) /// E2(1) / E2(1) /// E2(2) / E2(2) /// E2(3) / E2(3) /// E2(4) / E2(4) /// E2(5) / E2(5) /// E2(6) / E2(6) /// E2(7) / E2(7) 

Gel 2 Line 2 : 
Loading dye /// E2(8) / E2(8) /// E2(9) / E2(9) /// E2(10) / E2(10) /// E2(11) / E2(11) /// E2(12) / E2(12) /// E2(13) / E2(13) /// B2(13) / B2(13) /// B2(14) / B2(14) /// </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp18"><FIGURE><FIGCAPTION><P><U> Aim:</U>To start a culture for Miniprep of insert A1, A2, D1 and D2. 
	In order to obtain a large amount of plasmid, we need to grow the bacteria overnight. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4b/T--Pasteur_Paris--Bacterial_culture_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Microbiology equipement • 25 ml flasks • Carbenicillin 50 mg/ml • LB medium <U>Method:</U>
1.	One colony is picked from the plates and shaken in 1.0 ml of LB supplemented with Carbenicillin at 50 μg/ml. 10 colonies are taken from each insert. 
2.	The flask is placed in a shaking incubator at 37°C, 150 rpm overnight.

Start incubation at 16h30 </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp19"><FIGURE><FIGCAPTION><P><U> Aim:</U>To start a culture for Miniprep of insert A1, A2, D1 and D2. 
	In order to obtain a large amount of plasmid, we need to grow the bacteria overnight. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4b/T--Pasteur_Paris--Bacterial_culture_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Microbiology equipement • 25 ml flasks • Carbenicillin 50 mg/ml • LB medium <U>Method:</U>
1.	One colony is picked from the plates and shaken in 1.0 ml of LB supplemented with Carbenicillin at 50 &amp;#181g/ml. 10 colonies are taken from each insert. 
2.	The flask is placed in a shaking incubator at 37°C, 150 rpm overnight.

Start incubation at 16h30 </P></FIGCAPTION></FIGURE></DIV></DIV></BODY></HTML>