<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Microbiology_week8 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Microbiology week8</SPAN></H1></DIV></DIV></DIV><DIV class="container"><DIV id="position-menu2"><UL id="menu2"><LI><FONT size="3pt" color="#8B8378">About us</FONT></LI><LI><A href=" https://2016.igem.org/Team:Pasteur_Paris/AboutUs " href="">Our team</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Sponsors">Sponsors</A></LI></UL><LI><FONT size="3pt" color="#8B8378">Project</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Context">Context</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Overview">Overview</A></LI><LI><FONT size="3pt" color="#8B8378">Design </FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Science">Science </A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Moskit_devices">Devices</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Research">Research &amp; development</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Methodology">Methodology</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Design_Biology">Design vs Biology</A></LI><LI><FONT size="3pt" color="#8B8378">Labwork </FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Safety_Security">Safety &amp; Security</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Protocol ">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Netbook ">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Results ">Results</A></LI><LI>Human Practice</LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Collaborations ">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Scenarios">Scenarios</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Meet-up ">Meet-up</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Engagement">Communication</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Law"> Law</A></LI><LI><FONT size="3pt" color="#8B8378">Parts</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Description">Description</A></LI><LI><FONT size="3pt" color="#8B8378">Achievements</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/medals">Medals</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Prizes"> Special Prizes</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/rewards"> Awards obtained!</A></LI></DIV></DIV><DIV id="weeks"><DIV id="weekslist"><CENTER><H1><B>click on the weeks</B></H1></CENTER></DIV></DIV><DIV><A href="https://2016.igem.org/Team:Pasteur_Paris/Microbiology"><H2><B>Microbiology Notebook</B></H2></A></DIV><DIV id="week8"><H5><B>Week 8</B></H5><H3><B>July 25, 2016:</B></H3><H4>92. Next step for the growth curve </H4><A href="#exp2"><H4>93. Measurements of insert quantity with Nanodrop</H4></A><A href="#exp3"><H4>94. Ligation of the new version of the inserts </H4></A><A href="#exp4"><H4>95. PCR of C1 and C2 (v2) </H4></A><H3><B> July 26, 2016:</B></H3><H4>96. Midiprep of the culture made on the 25<SUP>th</SUP> of July </H4><A href="#exp6"><H4>97. Extraction and resuspension of culture made on the 25<SUP>th</SUP> of July </H4></A><A href="#exp7"><H4>98. Extraction gel bands C1 and C2 </H4></A><A href="#exp8"><H4>99. Set up the TOPO cloning ratios, ligation of C2 and C1 (v2) </H4></A><H3><B>July 27, 2016:</B></H3><H4>100. STAB culture </H4><A href="#exp10"><H4>101. Electrophoresis of PCR amplifications with the two inserts </H4></A><A href="#exp11"><H4>102. PCR of the new versions of C1 and C2 at 63 °C </H4></A><A href="#exp12"><H4>103. Electrophoresis of PCR products from C1 and C2 </H4></A><A href="#exp13"><H4>104. PCR of the new versions of C1 and C2 </H4></A><A href="#exp14"><H4>105. PCR purification </H4></A><A href="#exp15"><H4>106. TOP 10 cells transformation with the TOPO cloning ligation product </H4></A><A href="#exp16"><H4>107. Dosage of C1 v2 and C2 v2 after PCR and purification  </H4></A><H3><B>July 28, 2016:</B></H3><H4>108. Send the recombinant vector C2.1 (from the Midiprep of the 26<SUP>th</SUP> of July) for sequencing to Eurofins-MWG </H4><A href="#exp18"><H4>109. Observation of the transformation of C1 v2 and C2 v2 and culture of one colony </H4></A><A href="#exp19"><H4>110. Preparation of Petri dishes with Xgal </H4></A><A href="#exp20"><H4>111. Dosage of C1 after PCR product purification </H4></A><A href="#exp21"><H4>112. Fast Protein Liquid Chromatography (FPLC) </H4></A><H3> July 29, 2016:</H3><H4>113. SDS PAGE gel of purified protein fractions from FPLC </H4><A href="#exp23"><H4>114. SDS PAGE gel of purified protein fractions from FPLC </H4></A><A href="#exp24"><H4>115. PCR of C1 v2 and C2 v2 </H4></A><A href="#exp25"><H4>116. Agarose gel Electrophoresis </H4></A><A href="#exp26"><H4>117. DNA extraction with TOPO cloning colony obtained on the 27<SUP>th</SUP> of July </H4></A><A href="#exp27"><H4>118. DNA digestion with Xba I and Hind III </H4></A><H3> July 30, 2016:</H3><H4>119. Protein cryoconservation </H4></DIV><DIV class="lightbox" id="exp1"><FIGURE><FIGCAPTION><P><U> Aim:</U> Based on the results of silification tests, make a larger quantity of fusion protein. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/a/a4/T--Pasteur_Paris--Protein_induction_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U><LI> 50 ml Falcon tube</LI><LI> Shaking incubator (INFORS HT)</LI><LI> Swing bucket centrifuge (JOUAN GR41)</LI><LI> Cultures of C1.2 </LI><LI> carbenicillin at 50 mg/ml </LI><LI> LB medium </LI><LI> Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link)</LI><LI>; Ultrospec 3100</LI><U>Method:</U><LI>1.Put each of the C2 1.2 cultures in a 50 ml Falcon and centrifuge 8 minutes at 3500 RPM </LI><LI>2. Wash the pellet with 20 ml of LB after throwing away the supernatant </LI><LI>3. Centrifuge again 8 minutes at 3500 RPM and throw away the supernatant </LI><LI>4. Resuspend the pellet in 20 ml of LB </LI><LI>5. Pool the two Falcons </LI><LI>6. Put in the rotating incubator 4 Erlenmeyers of 2 L containing 1 L of LB, at 37°C and 150 RPM </LI><LI>7. Add in each Erlenmeyer 10 ml of C2 1.2 culture and 1 ml of carbenicillin at 50 mg/ml</LI><LI>8. Measure the OD<SUB>600 nm</SUB> at several times : </LI></P><P><U>Table 60</U></P>TimesOD<SUB>600nm</SUB><STRONG><P>0</P></STRONG> 0.014 <STRONG><P> 0 h 30</P></STRONG> 0.018 <STRONG><P> 1 h 00 </P></STRONG> 0.020 <STRONG><P> 1 h 30 </P></STRONG> 0.035 <STRONG><P> 2 h 00 </P></STRONG> 0.067 <STRONG><P> 3 h 00 </P></STRONG> 0.360 <STRONG><P> 3 h 15 </P></STRONG> 0.451 <STRONG><P> 3 h 25 </P></STRONG> 0.515 <STRONG><P> 3 h 40 </P></STRONG> 0.701 <LI>9. Pelleted the last measured sample for 3 minutes at 7000g. Then throw away the supernatant and store the pellet at &amp;8722; 20 °C </LI><LI>10. Add 0.6 ml of iPTG in each Erlenmeyer and incubate overnight</LI></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp2"><FIGURE><FIGCAPTION><P><U> Aim:</U> DNA concentration measurement in order to find the correct ratios for ligation.<U>What we did in the lab:</U><U>Materials:</U>• Spectrophotometer • Quartz cuvettes • carbenicillin at 50 mg/ml • LB medium • B1 v2, B2 v2, C1 v2, C2 v2 DNA inserts• Shaking incubator (INFORS HT)• Microbiology equipment (37 °C incubator, Bunsen burner, water bath, etc… Follow this link)<U>Method:</U>
      		All the measurements are done twice to be more accurate
The preculture of vector is prepared with 3 ml of LB, 3 µl of carbenicillin and one colony of C2 1.2 <U>Results</U></P><P><U>Table 61</U></P> DNA  pET 43.1 dephosphorylated  B1 v2  B2 v2  C1 v2  C2 v2 <STRONG><P>A<SUB>260</SUB></P></STRONG> 0.265  0.254  0.180  0.188  0.166  0.163  0.181  0.156  0.197 Ø<STRONG><P>A<SUB>280</SUB></P></STRONG> 2.39  0.211  0.077  0.063  0.060  0.052  0.077  0.070  0.071 Ø<STRONG><P>A<SUB>260 ⁄ 280</SUB></P></STRONG> 1.11  1.20  2.35  2.97  2.76  3.16  2.35  3.04  2.79 Ø<STRONG><P>A<SUB>260⁄230</SUB></P></STRONG> 0.31  0.34  0.01  0.01  0.01  0.01  0.01  0.01  0.01  0.01 <STRONG><P>[ DNA] (ng ⁄μl) </P></STRONG> 13.3  12.7  9  9.4  8.3  8.2  9  7.8  9.8 Ø</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp3"><FIGURE><FIGCAPTION><P><U> Aim:</U> Prepare the ligation before doing the transformation with the Top10 competent  cell .<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/6f/T--Pasteur_Paris--Ligation_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link) • 1 ml Eppendorfs • B1 v2 , B2 v2 , C1 v2 , C2 v2 DNA inserts • Distilled water • Shaking incubator (INFORS HT) • T4 ligase • T4 buffer<U>Method:</U>
         1. Prepare the following samples in a 1 ml Eppendorf, finishing with the ligase : </P><P><U>Table 62</U></P> B1 v2 pET 43.1(a+) from 05 ⁄ 07 B2 v2 pET 43.1(a+) from 05 ⁄ 07  C1 v2 pET 43.1(a+) from 11 ⁄ 07  C2 v2 pET 43.1(a+) from 08 ⁄ 07 <STRONG><P>100 ng dephorphorylated ( μl)</P></STRONG> 6.7  6.7  7.7  7.7 <STRONG><P>Insert (16ng) (μl)</P></STRONG> 1.6  2  2  1.6 <STRONG><P>T4 ligase (μl)</P></STRONG> 1  1  1  1 <STRONG><P>T4 buffer (μl)</P></STRONG> 1.5  1.5  1.5  1.5 <STRONG><P> H<SUB>2</SUB>O (μl)</P></STRONG> 4.2  3.8  1.8  3.2 <STRONG><P> V<SUB>final</SUB> (μl)</P></STRONG> 15  15  15  15 
	2. Let ligate 1 hour at room temperature 
3.Incubate 10 minutes at 65°C to inactivate the enzymes
4.Store all the samples at −80°C until ready to transform
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp4"><FIGURE><FIGCAPTION><P><U> Aim:</U> Increase the amount of DNA. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/b/be/T--Pasteur_Paris--PCR_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Microbiology equipement • Mastermix 2X • Primer S (sense direction, Forward Primer, see sequence in PCR protocol section)• Primers AS (anti-sense, Reverse Primer, see sequence in PCR protocol section)• Phusion DNA polymerase • DNA template • Distilled water • PCR machine <U>Method:</U>
                1. For each insert, make 3 tubes of 0.2 ml with thin walls and prepare the following mastermix : </P><P><U>Table 63</U></P> Volume (μl)  Insert  Primer S  Primer AS <STRONG><P> Mastermix 2X </P></STRONG> 75  75  75 <STRONG><P> Primer S </P></STRONG> 3  3 Ø<STRONG><P> Primer AS </P></STRONG> 3 Ø 3 <STRONG><P> H<SUB>2</SUB>O </P></STRONG> 65.4  68.4  68.4 <STRONG><P> Phusion </P></STRONG> 0.6  0.6  0.6 
                  2. Distribute the previous mastermix in the tubes and add the template DNA to have : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 64</U></P></I></CAPTION><THEAD><TR><TH> Volume (μl) </TH><TH> Insert </TH><TH> Primer S</TH><TH> Primer AS </TH><TH> Without DNA </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> Mastermix </P></STRONG></TD><TD align="center"> 25 </TD><TD align="center"> 25 </TD><TD align="center"> 25 </TD><TD align="center"> 25 </TD></TR><TR><TD align="center"><STRONG><P> Primer S </P></STRONG></TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center">Ø</TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> Primer AS </P></STRONG></TD><TD align="center"> 1 </TD><TD align="center">Ø</TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> Template DNA diluted 1⁄10 </P></STRONG></TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> H<SUB>2</SUB>O </P></STRONG></TD><TD align="center"> 21.8 </TD><TD align="center"> 22.8 </TD><TD align="center"> 22.8 </TD><TD align="center"> 22.8 </TD></TR><TR><TD align="center"><STRONG><P> Phusion </P></STRONG></TD><TD align="center"> 0.2 </TD><TD align="center"> 0.2 </TD><TD align="center"> 0.2 </TD><TD align="center"> 0.2 </TD></TR><TR><TD align="center"><STRONG><P> Total </P></STRONG></TD><TD align="center"> 50 </TD><TD align="center"> 50 </TD><TD align="center"> 50 </TD><TD align="center"> 50 </TD></TR></TBODY></TABLE>
	     3. Launch the PCR for 24 cycles
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp5"><FIGURE><FIGCAPTION><P><U> Aim:</U> Get back plasmid DNA with the twinned insert.<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• QIAGEN plasmid Midi Kit • 2 ml Eppendorfs • Spectrophotometer • Ethanol 70%• Cultures of BL21DE3 from the 25⁄07
<U>Method:</U>
1. When isopropanol is added, spread the volume in 5 x 2 (2 ml Eppendorfs) and centrifuge 10 minutes at 13000g 
2. Add 1 ml of ethanol in each pellet and centrifuge 10 minutes at 13000g <U>Results:</U>
Make 3 measurements of the concentration (1⁄200 dilution)
</P><P><U>Table 65</U></P> Measurement 1  Measurement 2  Measurement 3 <STRONG><P> OD<SUB>260</SUB></P></STRONG> 0.046  0.44  0.073 <STRONG><P>[DNA] (μg⁄ml) </P></STRONG> 460  4.40  730 </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp6"><FIGURE><FIGCAPTION><P><U> Aim:</U> Prepare our bacteria before the protein purification.<U>What we did in the lab:</U><U>Materials:</U>• Microbiological equipment• PMSF at 100 mM• Cultures of BL21DE3• Buffer A• 50 ml Falcons
<U>Method:</U>
1.Centrifuge 4x 1 l of cultures 20 minutes at 4500g, in a Beckman JLA 8.1000 rotor in a Avanti x26-J centrifuge 
2.Throw away the supernatant 
3.Add 40 μl of PMSF in 40 ml of buffer A and add 5 ml of this solution in each pellet 
4.Scratch the pellet to resuspend it 
5.Add again 5 ml of the buffer A solution and mix the slurry until it becomes smooth, and to recover all the pellet 
6.Put each product in a clean 50 ml Falcon : store at −80°C. </P><P><U>Table 66</U></P> Tubes  1  2  3  4 <STRONG><P> Weight (g) </P></STRONG> 15  15  15.3  16.1 </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp7"><FIGURE><FIGCAPTION><P><U> Aim:</U> Extract the bands C1 and C2 amplified with 900 bp inserts, and separate non-specific bands (700 bp).<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4e/T--Pasteur_Paris--Gel_extraction_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• QIAGEN Gel extraction Kit• Buffer QG • Gel bands C1 and C2 • Isopropanol • 2 ml Eppendorfs
<U>Method:</U>
1. Calculate the amount of buffer QG for the extraction : </P><P><U>Table 67</U></P> Weight of our tubes (mg)  Weight of our extraction (mg)  Buffer QG (μl) <STRONG><P> C1 </P></STRONG> 1.0360  40.2  121 <STRONG><P> C2 </P></STRONG> 1.0154  32.8  97 
2. Add 1 volume of isopropanol for C2 (32.3 μl) and 3 volumes of isopropanol for C1 (121 μl) 
3. Recovery of DNA in 40 μl extraction buffer (20 x 2 μl)
<U>Results:</U>
Measurement of concentration with a dilution of 1⁄250 :
<TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 68</U></P></I></CAPTION><THEAD><TR><TH> C1 </TH><TH> C2 </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> OD<SUB>260 nm</SUB></P></STRONG></TD><TD align="center"> 0.003 </TD><TD align="center"> 0.003 </TD></TR><TR><TD align="center"><STRONG><P> OD<SUB>280 nm</SUB></P></STRONG></TD><TD align="center"> 0.003 </TD><TD align="center"> 0.003 </TD></TR><TR><TD align="center"><STRONG><P>[DNA] (μg⁄ml) </P></STRONG></TD><TD align="center"> 30 </TD><TD align="center"> 120 </TD></TR></TBODY></TABLE></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp8"><FIGURE><FIGCAPTION><P><U> Aim:</U> To link vectors with the inserts.<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/6f/T--Pasteur_Paris--Ligation_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Colonies C1 v2 and C2 v2 Insert bands • Salt solution • Distilled water • TOPO vector
<U>Method:</U></P><P><U>Table 69</U></P> C1 v2  C2 v2 <STRONG><P> DNA from colonies (inserts) (μl) </P></STRONG> 4  4 <STRONG><P> Salt solution (μl) </P></STRONG> 1  1 <STRONG><P> Distilled water (μl) </P></STRONG> 5  5 <STRONG><P> TOPO vector (μl) </P></STRONG> 1  1 <STRONG><P> V<SUB>final</SUB> (μl) </P></STRONG> 6  6 </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp9"><FIGURE><FIGCAPTION><P><U> Aim:</U> Save the bacteria with the recombinant vector for storage.<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/b/b1/Stab_culture_Pasteur_Paris_2016.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Microbiological equipment • Carbenicillin 50 mg⁄ml • LB medium • 50 ml Falcons • 2 ml Eppendorfs • Recombinant vector • BL21DE3 colonies 
<U>Method:</U>
1. Prepare in a falcon of 50 ml a preculture of the recombinant vector with one colony, 5 ml of LB and 5 μl of carbenicillin 50 mg⁄ml 
2. Add 5 ml of LB and centrifuge 7 minutes at 3500 RPM 
3. Discard the supernatant and add 5 ml of LB. Then, centrifuge again 7 minutes at 3500RPM 
4. Discard the supernatant and add 5 ml of LB 
5. Put 200 μl of this preparation into another Falcon of 50 ml, add 25 ml of LB and 25 μl of carbenicillin
6. Let the culture shake at 37°C for 4 hours (until the culture growth is sufficient, OD<SUB>600 nm</SUB> of approx. 0.7) and check the concentration at several times during it :</P><P><U>Table 70</U></P> Time  OD<SUB>600 nm</SUB><STRONG><P> 0 </P></STRONG> 0.007 <STRONG><P> 1h19 </P></STRONG> 0.059 <STRONG><P> 2h39 </P></STRONG> 0.458 <STRONG><P> 3h54 </P></STRONG> 0.827 
	7. In 9 Eppendorfs of 2 ml, put 1 ml of LB with AGAR and 1 μl of carbenicillin. Then mix it 
8. Dip a toothpick in the culture, and stab in the agar in each Eppendorf, and let it incubate at 37°C overnight. Afterwards store at 4°C. 
9. On the July 28, 2016 : restreaking of cultures in a Petri dish (verification of STABs)
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp10"><FIGURE><FIGCAPTION><P><U> Aim:</U> Separate the amplification bands of specific DNA (900 bp) from non-specific bands (700 bp).<U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><U>What we did in the lab:</U><U>Method:</U>
We skipped this part to concentrate on the new versions of C1 and C2 inserts.</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp11"><FIGURE><FIGCAPTION><P><U> Aim:</U> To determine if the two primers For (Forward direction) and Rev (Reverse direction) that were previously prepared are specific or not. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/b/be/T--Pasteur_Paris--PCR_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Mastermix 2X • Primer S • Primer AS • Distilled water • C1 and C2 inserts • Phusion DNA polymerase • Tubes for PCR • PCR machine
<U>Method:</U>
1. For each inserts prepare the following Globalmix : </P><P><U>Table 71</U></P> Volume (μl)  Insert  Primer S  Primer AS <STRONG><P> Mastermix 2X </P></STRONG> 75  75  75 <STRONG><P> Primer S </P></STRONG> 3  3 Ø<STRONG><P> Primer AS </P></STRONG> 3 Ø 3 <STRONG><P> H<SUB>2</SUB>O </P></STRONG> 65.4  68.4  68.4 
	2. Distribute the samples and add DNA to have : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 72</U></P></I></CAPTION><THEAD><TR><TH> Volume (μl) </TH><TH> Insert </TH><TH> Primer S </TH><TH> Primer AS </TH><TH> Without DNA </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> Globalmix </P></STRONG></TD><TD align="center"> 48.4 </TD><TD align="center"> 48.4 </TD><TD align="center"> 48.4 </TD><TD align="center"> 48.4 </TD></TR><TR><TD align="center"><STRONG><P> C1 or C2 </P></STRONG></TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> Phusion </P></STRONG></TD><TD align="center"> 0.2 </TD><TD align="center"> 0.2 </TD><TD align="center"> 0.2 </TD><TD align="center"> 0.2 </TD></TR><TR><TD align="center"><STRONG><P> Total </P></STRONG></TD><TD align="center"> 50 </TD><TD align="center"> 50 </TD><TD align="center"> 50 </TD><TD align="center"> 50 </TD></TR></TBODY></TABLE>
	3. Dispense these samples in 10 tubes and launch the PCR for 24 cycles; T<SUB>annealing</SUB>=63 ºC
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp12"><FIGURE><FIGCAPTION><P><U> Aim:</U> Check the results of the previous PCR of the new versions of C1 and C2 <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Ladder • Inserts (old) • C1 and C2 • Primer S • Buffer EX • Electrophoresis machine
<U>Method:</U>
Add 5 μl of DNA and 1 μl of lead buffer EX: </P><P><U>Table 73</U></P> Samples  L1  L2  L3  L4  L5  L6  L7  L8 <STRONG><P> Contain </P></STRONG> Ladder  C1 + insert  C1 + S  C1 + AS  C2 + insert  C2 + S  C2 + AS  C without DNA <U>Results: </U>
None of the bands of our samples can be seen because the temperature is not suitable (too hot). This experiment has to be redone another time at lower temperature. <CENTER><I><P><U>Figure 8 :</U> PCR gel of C1/C2 </P></I></CENTER></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp13"><FIGURE><FIGCAPTION><P><U> Aim:</U> PCR of the new versions of C1 and C2 at lower annealing temperature 55 ºC
Re-amplify some of the previous PCR reaction solution with fresh dNTPs, buffer and polymerase.. <U> Protocol:</U> follow this <A href="https://static.igem.org/mediawiki/2016/b/be/T--Pasteur_Paris--PCR_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Samples from the previous PCR of the 26⁄07• Tubes for PCR • PCR machine • dNTPs • MgCl<SUB>2</SUB> solution • Phusion 
<U>Method:</U>
1. Take 7 of the samples from the previous PCR  (which contains only 45 μl after the electrophoresis) 
2. With these samples, prepare the following samples by adding in order: dNTPs, MgCl<SUB>2</SUB> and Phusion enzyme : </P><P><U>Table 74</U></P> Volume (μl)  Insert  Primer S  Primer AS  Without DNA <STRONG><P> Samples </P></STRONG> 45  45  45  45 <STRONG><P> dNTPs </P></STRONG> 6  6  6  6 <STRONG><P> MgCl<SUB>2</SUB></P></STRONG> 8.4  8.4  8.4  8.4 <STRONG><P> Phusion </P></STRONG> 0.5  0.5  0.5  0.5 <STRONG><P> V<SUB>final</SUB></P></STRONG> 60  60  60  60 
3. Vortex all the samples mix and centrifuge them at room temperature at 3500 RPM 
4. Run PCR following previous protocol, except for the T<SUB>annealing</SUB>. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp14"><FIGURE><FIGCAPTION><P><U> Aim:</U> Recovering the product of our previous PCR. <U>What we did in the lab:</U><U>Materials:</U>• QIAGEN Gel extraction Kit • Products of the PCR of the 27/07 • 6X loading buffer • Molecular weight DNA Ladder • 10 µl pipette • 200 µl pipette • PCR tubes
<U>Method:</U>
1. Recovery from the agarose gel of 8 wells 
2. Take 50 µl in PCR tubes and prepare several samples by adding in each tubes: 2.a 10 µl of 6X loading buffer and mix with one pipette 2.b 6 µl of ladder
3.Determination of the weight of the gel for extraction bands of C1 with For and Rev primers :
m(tube) = 1.1258g 
m(tube+gel) = 1.2919 → m(gel) = 165 mg which correspond to 496.5 µl of buffer 
4. Follow the QIAGEN gel extraction Kit protocol <U>Results :</U></P><P>Figure 9 : PCR gel of C1/C2 </P>
1: Ladder
2: Without DNA
3: C1 insert For and Rev
4: C1 For
5: C1 Rev
6: C2 insert
7: C2 For
8: C2 Rev
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp15"><FIGURE><FIGCAPTION><P><U> Aim:</U> Transform our new plasmid (vector TOPO + C1 v2 and C2 v2). <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A><U>What we did in the lab:</U><U>Materials:</U>• Microbiological equipment • Carbenicillin at 50 mg⁄ml • LB medium • Petri dish • TOP 10 competent cells• Dimethylformamide (DMF) • Xgal 20 mg/ml in DMF
<U>Method:</U>
1. Prepare the solution of Xgal for the Petri dish with : 1.a 10 ml of DMF  1.b 200 mg of Xgal 
2. Put 100 µl of this solution on 6 Petri dishes which already contain LB and carbenicillin. Then spread the solution in an additional volume of LB (100 &amp;181#l). Let the Petri dishes dry at 37°C for approx. 1 hr.
3. Recovery C2 v2 and C1 v2 prepared on the 26⁄07 (to perform the TOP 10 cloning reaction) 
4. Put 50 µl of bacteria TOP 10 in ice and add 6 µl of C1 v2 or C2 v1 
5. Follow the protocol to transform thermally competent bacteria
</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp16"><FIGURE><FIGCAPTION><P><U> Aim:</U> Know the concentration of the inserts. <U>What we did in the lab:</U><U>Materials:</U>
See protocol
<U>Result</U></P><P><U>Table 75</U></P> C1 v2  C2 v2 <STRONG><P> V<SUB>TE</SUB> (µl) </P></STRONG> 498  498 <STRONG><P> V<SUB>DNA</SUB> (µl) </P></STRONG> 2  2 <STRONG><P> A<SUB>260 nm</SUB></P></STRONG> 0.002  0.001 <STRONG><P> A<SUB>280 nm</SUB></P></STRONG> 0.001  0.001 <STRONG><P> Ratio 260/280</P></STRONG> 2  1 <STRONG><P>[Insert] (µg⁄ml) </P></STRONG> 20  25 </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp17"><FIGURE><FIGCAPTION><P><U> Aim:</U> Send the recombinant vector C2.1 (from the Midiprep of the 26⁄07) for sequencing to Eurofins-MWG. <U> Results:</U></P><P><U>Table 76</U></P>[DNA<SUB>C2.1</SUB>] (ng⁄ml)  450 <STRONG><P> V<SUB>DNA</SUB> (µl) </P></STRONG> 3.3 <STRONG><P> V<SUB>H<SUB>2</SUB>O</SUB> (µl) </P></STRONG> 11.7 <STRONG><P> V<SUB>primer d1⁄50</SUB> (µl) </P></STRONG> 2 <STRONG><P> V<SUB>total</SUB> (µl) </P></STRONG> 17 </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp18"><FIGURE><FIGCAPTION><P><U> Aim:</U> Check if the transformation worked. <U>What we did in the lab :</U><U>Materials :</U>• Microbiological equipment • 50 ml Erlenmeyer • Incubator • LB medium • Carbenicillin at 50 mg⁄ml <U> Method </U>
1. Only one colony had grown so we put it into a 50 ml Erlenmeyer with 25 ml of LB and 25 µl of carbenicillin 
2. Put the Erlenmeyer into the incubator at 37°C with agitation overnight
</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp19"><FIGURE><FIGCAPTION><P><U> Aim:</U> Have Petri dishes ready for future transformations with TOP 10 cells. <U>What we did in the lab :</U><U>Materials :</U>• Solution from TOP 10 transformation with the ligation TOPO10 cloning product (experiment from the 27⁄07) •DMF• Xgal at 10 mg⁄ml • 200 µl micropipette • 10 ml pipette • Petri dishes <U> Method </U>
1. In a Petri dish, put 100 μl of the solution from TOPO10 transformation with the ligation TOPO cloning product 
2. Add 10 ml of DMF and 10 mg of Xgal 
3. Spread 200 µl of Xgal on the Petri dishes, and allow drying at 37°C for approximately 1 hour. 
4. Carry out the transformation according to the thermal protocol
</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp20"><FIGURE><FIGCAPTION><P><U> Aim:</U> Know the concentration of purified C1 (For + Rev). <U>What we did in the lab :</U><U>Materials :</U>• Ultrospec 3100 • Quartz cuvette • TE buffer • C1 DNA from the PCR purification (27⁄07) • Petri dishes <U> Method </U>
1. Do the blank with 500 µl of TE buffer 
2. For the sample, use 498 µl of TE buffer and 2 µl of C1 DNA from the PCR purification <U>Results</U></P><P><U>Table 77</U></P> OD<SUB>260</SUB> OD<SUB>280</SUB>[DNA<SUB>C1</SUB>] (µg⁄ml)  0.002  0.001  50 </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp21"><FIGURE><FIGCAPTION><P><U> Aim:</U> Protein purification. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/0/07/T--Pasteur_Paris--FPLC_Protein_purification_protocol.pdf">link</A><U>What we did in the lab :</U><U>Materials :</U>• FPLC machine (Fast Protein Liquid Chromatography)(Akta Prime Plus, GE Lifesciences) • 5 ml Ni-NTA His-trap column (GE lifesciences)• Buffer A (Tris 50 mM pH 7.4 ; NaCl 150 mM), Buffer B=Buffer A + Imidazole 250 mM• PMSF at 100 mM in ethanol• Branson 450 (sonicator) • JA 25.5  rotor for Beckman Avanti X26-J centrifuge
<U> Method </U>
On ice : 
1. Defrost one aliquot of C1 bacteria culture under cold running water 
2. Equilibrate the column on the FPLC machine with buffer A 
3. Once the bacteria are thawed out, add 22 ml of buffer A and 37µl of PMSF 
4. Lyse the bacteria with a sonicator 
5. Use a medium size tip, put the duty cycle at 60% and power under the limit of the micro tip setting 
6. With the sonicator machine, do 3 times 30 seconds of sonication followed-up with 1 minute of rest on ice 
7. Balance polycarbonate 50 ml tubes containing the lysate on a microbalance
8. Centrifuge the lysate at 16000 RPM in a JA 25.5 rotor in a Beckman Avanti X26-J at 4°C during 30 minutes 
9. Transfer the supernatant to a 50 ml Falcon and add 37µl of PMSF 
10. Inject 10 ml of this sample with a syringe equipped with a 0.45 µm filter in the injection port and load it on the column (Ni-NTA His Trap, 5 ml, GE Lifesciences) 
12. Wash the injection loop and column with buffer A until the UV detector output at 280 nm is close to the baseline and repeat the injection 3 times 
13. Wash the column with 5% buffer B 
14. Run a gradient from 5&amp;#37 to 100&amp;#37 buffer B over 12 column volumes and collect fractions of 1 ml
15. Wash the column with buffer A
</P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp22"><FIGURE><FIGCAPTION><P><U> Aim:</U> Detection of the proteins present in each fraction/peak. <U>What we did in the lab :</U><U>Materials :</U>• 10X TGS buffer (Biorad) • Distilled water • PAGE Ruler Protein Molecular weight ladder (Thermofisher) • 1 SDS PAGE gel, BioRad (TGS: Tris Glycine SDS) • Electrophoresis chamber , Power supply and electrodes (Biorad) • 2X SDS (Laemli buffer with β-mercapto ethanol) • 95°C Heater block • Gel Code blue stain (Pierce) with G250 Coomassie blue
<U> Method </U>
1. At the beginning, prepare 1X from 10X TGS buffer with 25 ml of buffer and 225 ml of distilled water 
2. Heat samples at 95°C for 5 minutes, cool at RT, and load 20 µl in each well; 6 µl of ladder. 
3. Run at 130 V <U>Results :</U>
1 : PAGE ruler Ladder 
2 : Well 19 
3 : Well 21 
4 : Well 25 
5 : Well 30 
6 : Well 34 
7 : Well 36 
8 : Well 37 
9 : Well 38 
10 : Well 39 

The concentration of proteins is too high so this experiment has to be redone</P><P><U>Figure 10 :</U> SDS PAGE gel of our protein </P></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp23"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Revelation of proteins in each well. <H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• TGS 10X• PAGE ruler molecular weight marker (Thermofisher) • Distilled water• Protein fraction solutions from FPLC• Buffer A• Laemli 2X• SDS PAGE gel, electrophoresis chamber, 3000 V power supply (Biorad)
<H6><U> Method :</U></H6>
1. Prepare 400 ml of TGS 1X with 40 ml of TGS 10X and 360 ml of distilled water 
2. Dilute protein solutions : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 78</U></P></I></CAPTION><THEAD><TR><TH> Fractions from FPLC </TH><TH> Dilution </TH><TH> Contain </TH></TR></THEAD><TBODY><TR><TD align="center"> 19, 21, 25</TD><TD align="center">Ø</TD><TD align="center">Ø</TD></TR><TR><TD align="center"> 30, 34, 36 </TD><TD align="center"> (1/100) in buffer A </TD><TD align="center"> 1 µl of protein with 99 µl of buffer A </TD></TR><TR><TD align="center"> 37, 38, 39 </TD><TD align="center">(1/10) in buffer A </TD><TD align="center"> 1 µl of protein with 9 µl of buffer A </TD></TR></TBODY></TABLE>
3. In each gel well, add 10 µl of diluted protein solution and 10 µl of laemli 2X. 
4. Put all samples at 95°C during 5 minutes and follow the protocol for SDS PAGE gel.
<H6><U>Results :</U></H6>
The distribution of samples in lanes is the same as the previous experiment :<CENTER><I><P><U>Figure 4 :</U> SDS PAGE gel of diluted proteins </P></I></CENTER>
1 : Ladder 
2 : Well 19
3 : Well 21
4 : Well 25
5 : Well 30
6 : Well 34
7 : Well 36 
8 : Well 37
9 : Well 38
10 :Well 39</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp24"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Test the specificity of our two primers : For and Rev. Use a different DNA polymerase<H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/b/be/T--Pasteur_Paris--PCR_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• dNTPs • Solution of MgCl<SUB>2</SUB>• RNA free • Buffer 10X pfu DNA polymerase  • Inserts C1 v2 and C2 v2  • For primer (see PCR protocol for sequence) • Rev primer  (see PCR protocol for sequence) • 2 ml Eppendorfs  • Distilled water
<H6><U> Method </U></H6>
1. Make a global mix for the 7 samples : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 79</U></P></I></CAPTION><THEAD><TR><TH> Volumes (µl) </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> dNTPs </P></STRONG></TD><TD align="center"> 22.5 </TD></TR><TR><TD align="center"><STRONG><P> MgCl<SUB>2</SUB></P></STRONG></TD><TD align="center"> 22.5 </TD></TR><TR><TD align="center"><STRONG><P> RNA free </P></STRONG></TD><TD align="center"> 327.6 </TD></TR><TR><TD align="center"><STRONG><P> Buffer 10X pfu </P></STRONG></TD><TD align="center"> 22.5 </TD></TR><TR><TD align="center"><STRONG><P> Total </P></STRONG></TD><TD align="center"> 417.6 </TD></TR></TBODY></TABLE>
2. Distribute the global mix in each tubes and add DNA : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 80</U></P></I></CAPTION><THEAD><TR><TH> Tubes </TH><TH> Name </TH><TH> V(premix) (µl) </TH><TH> V(C1 v2) (µl) </TH><TH> V(C2 v2) (µl) </TH><TH> V(primer For) (µl) </TH><TH> V(primer Rev) (µl) </TH><TH> V(H<SUB>2</SUB>O) (µl) </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> 1 </P></STRONG></TD><TD align="center"> C1 v2 insert For and Rev </TD><TD align="center"> 46.4 </TD><TD align="center"> 1 </TD><TD align="center"> 0 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 0 </TD></TR><TR><TD align="center"><STRONG><P> 2 </P></STRONG></TD><TD align="center"> C1 v2 insert For </TD><TD align="center"> 46.4 </TD><TD align="center"> 1 </TD><TD align="center"> 0 </TD><TD align="center"> 1 </TD><TD align="center"> 0 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> 3 </P></STRONG></TD><TD align="center"> C1 v2 insert Rev </TD><TD align="center"> 46.4 </TD><TD align="center"> 1 </TD><TD align="center"> 0 </TD><TD align="center"> 0 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> 4 </P></STRONG></TD><TD align="center"> C2 v2 insert For and Rev </TD><TD align="center"> 46.4 </TD><TD align="center"> 0 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 0 </TD></TR><TR><TD align="center"><STRONG><P> 1 </P></STRONG></TD><TD align="center"> C2 v2 insert For </TD><TD align="center"> 46.4 </TD><TD align="center"> 0 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 0 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> 1 </P></STRONG></TD><TD align="center"> C1 v2 insert Rev </TD><TD align="center"> 46.4 </TD><TD align="center"> 0 </TD><TD align="center"> 1 </TD><TD align="center"> 0 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> 1 </P></STRONG></TD><TD align="center"> Without DNA </TD><TD align="center"> 0 </TD><TD align="center"> 0 </TD><TD align="center"> 0 </TD><TD align="center"> 0 </TD><TD align="center"> 0 </TD><TD align="center"> 0 </TD></TR></TBODY></TABLE>
3. Put the tubes in a PCR machine heat to 94°C during 1 minute and then add the enzyme. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp25"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Test the specificity of our two primers : For and Rev. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• 100 ml Erlenmeyer• TAE 1X • Agarose gel • Ethidium bromide drops (EB) • Gel electrophoresis apparatus • Gene ruler ladder • Loading buffer 6X • DNA (C1 and C2, For and Rev)
<H6><U> Method :</U></H6>
1. Prepare a 0.7% agarose gel : 1.a In an Erlenmeyer of 100 ml, put 50 ml of TAE 1X and add 0.35 g of agarose  1.b Warm it in a microwave 2 minutes and let it cool down a little  1.c Add 2 drops of EB and mix it  1.d Transfer it on combs in a gel caster and let it solidify 
2. Put samples in the wells:<TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 81</U></P></I></CAPTION><THEAD><TR><TH> Wells </TH><TH> Samples </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> 1 </P></STRONG></TD><TD align="center"> 7 µl of gene ruler </TD></TR><TR><TD align="center"><STRONG><P> 2 </P></STRONG></TD><TD align="center"> 2 µl of 6X + 10 µl of DNA (C1 For and Rev) </TD></TR><TR><TD align="center"><STRONG><P> 3 </P></STRONG></TD><TD align="center"> 2 µl of load 6X + 10 µl of DNA (C1 For) </TD></TR><TR><TD align="center"><STRONG><P> 4 </P></STRONG></TD><TD align="center"> 2 µl of load 6X + 10 µl of DNA (C1 Rev) </TD></TR><TR><TD align="center"><STRONG><P> 5 </P></STRONG></TD><TD align="center"> 2 µl of load 6X + 10 µl of DNA (C2 For and Rev) </TD></TR><TR><TD align="center"><STRONG><P> 6 </P></STRONG></TD><TD align="center"> 2 µl of load 6X + 10 µl of DNA (C2 For) </TD></TR><TR><TD align="center"><STRONG><P> 7 </P></STRONG></TD><TD align="center"> 2 µl of load 6X + 10 µl of DNA (C2 Rev) </TD></TR><TR><TD align="center"><STRONG><P> 8 </P></STRONG></TD><TD align="center"> 2 µl of load 6X + 10 µl of distilled water </TD></TR></TBODY></TABLE><H6><U>Results :</U></H6><CENTER><I><P>Figure 5 : PCR gel of C1 v2 / C2 v2 </P></I></CENTER>
1 : Gene ruler 7 μl 
2 : C1 For + Rev 
3 : C1 For 
4 : C1 Rev 
5 : C2 For + Rev 
6 : C2 For 
7 : C2 Rev  
8 : Without DNA 
The experiments did not work. 
</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp26"><FIGURE><FIGCAPTION><P><U> Aim:</U> Extracting DNA from the colony obtained after transformation. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A><U>Materials :</U>• Miniprep QIAGEN Kit • TOPO cloning colony • Buffer EB • Spectrophotometer
<U> Method </U>
1. For the Miniprep prepare 6x 2 ml of cultures and recover it in 50 μl of buffer EB 
2. Do the prep with the QIAGEN Miniprep Kit 
3. Measure the absorbance of the products to determine the amount of DNA obtained after extraction <U>Results :</U></P><P><U>Table 82</U></P> Samples  A<SUB>260 nm</SUB> A<SUB>280 nm</SUB>[DNA] (μg⁄ml) <STRONG><P> 1 </P></STRONG> 0.009  0.001  120 → wrong <STRONG><P> 2 </P></STRONG> 0.001  0.003  10 <STRONG><P> 3 </P></STRONG> 0.001  0.003  11  </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp27"><FIGURE><FIGCAPTION><P><U> Aim:</U>Check if the insert is in the TOPO vector. <U> Protocol:</U> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><U>Materials :</U>• Biochemical equipment • TOPO vector • Xba I (NEB) • Hind III (NEB) • Buffer Cutsmart 10X (NEB) • Distilled water
<U> Method </U></P><P><U>Table 83</U></P> Volumes (μl) <STRONG><P> DNA (10 μg⁄ml) </P></STRONG> 5 <STRONG><P> Xba I </P></STRONG> 1 <STRONG><P> Hind III </P></STRONG> 1 <STRONG><P> Buffer Cutsmart 10X </P></STRONG> 4 <STRONG><P> H<SUB>2</SUB>O </P></STRONG> 29 <STRONG><P> Total </P></STRONG> 40 </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp28"><FIGURE><FIGCAPTION><P><U> Aim:</U>Protein conservation. <U>Materials :</U>• Biochemical equipment• Glycerol 80 %<U> Method </U>
To prepare glycerol 80% with a final volume of 40 ml, use 32 ml of glycerol with 8 ml of H<SUB>2</SUB>O
 (40×80)⁄100 = 32 ml
</P></FIGCAPTION></FIGURE></DIV></BODY></HTML>