Team:Pasteur Paris/Microbiology week9
About us
Our team
Attribution
Sponsors
Project
Context
Overview
Design
Science
Devices
Research & development
Methodology
Design vs Biology
Labwork
Safety & Security
Protocol
Notebook
Results
Human Practice
Integrated Practices
Collaborations
Scenarios
Meet-up
Communication
Law
Parts
Description
Achievements
Medals
Special Prizes
Awards obtained!
click on the weeks
Microbiology Notebook
Week 9
August 1, 2016:
120. Silification tests
121. PCR of C1 v2 and C2 v2 with Takara enzyme
122. Agarose gel
123. Electrophoresis of the PCR products (C1 v2 and C2 v2)
124. PCR Clean up
August 2, 2016:
125. PCR of A1, A2, B1, B2, D1, D2, E1 and C2
126. Analysis of PCR products from August 1
127. PCR clean up
128. Ligation of PCR products with TOPO cloning
129. Transformation in TOP10 competent cells
130. PCR of A1⁄A2 and D1⁄D2
August 3, 2016:
131. Analysis of PCR products from August 2, 2016
132. Miniprep of cultures C1 v2 and C2 v2 from August 2
133. Digestion of C1 v2 and C2 v2
134. Electrophoresis of C1 v2 and C2 v2
135. DNA extraction from the gel
136. Ligation of C1.2⁄C1.5 and C2.1⁄C2.2 with pET43.1 (a+) and transformation with TOP10 competent cells
August 4, 2016:
137. Miniprep of B1⁄B2 and E1⁄E2
138. Digestion of B1⁄B2, E1⁄E2 and pET43.1 (a+) with Hind III and Xba I
139. Dephosphorylation of digested pET43.1 (a+)
140. PCR of A1⁄A2 and D1⁄D2 without MgCl2
141. Preculture of TOPO – C1 v2 (2) and (5) and TOPO – C2 v2 (1) and (2)
142. Electrophoresis of PCR products A1⁄A2 and D1⁄D2
143. Next PCR of A1⁄A2 and D1⁄D2
144. Pool of inserts C1 v2 and C2 v2
August 5, 2016:
145. Digestion of B1⁄B2 and C1⁄C2
146. Ligation of C1 and C2 with pET43.1 (a+) digested and dephosphorylated
147. Transformation of TOP10 competent cells
148. Ligation of B1⁄B2⁄C1⁄C2 in pET43.1 (a+)
149. Transformation of B1⁄B2⁄C1⁄C2 in TOP10
Aim: Check if the sillification is working with different volume of HCl and TEOS.
Protocol: follow in this link
What we did in the lab:
Materials:
• 1.5 ml Eppendorfs
• Silification protein
• HCl 1 M
• TEOS (Tetraethyl ortho silicate, Sigma)
• Shaking incubator
• 15 ml Falcon
• Buffer A (See protein purification protocol)
• 10 μl and 200 μl pipettes
Method:
1. In a 1.5 ml Eppendorf prepare a sample with the fraction that contains protein c2, add 1 ml of HCl 1mM, 65.8 μl of TEOS and let it incubate for 4 minutes in the rotating incubator
2. In a 15 ml Falcon, put 9 ml of buffer A and add the previous sample
3. Follow if silification initiates and progresses at various times :
Table 84
Times Comments
10 minutes
Ø
15 minutes
Microscopic crystalline specs appear
30 minutes
Microscopic crystalline specs continue to appear
50 minutes
Microscopic crystalline specs continue to appears + crystalline specs grow and form a precipitate
4. Redo this experiment with 1 ml of TEOS, 50 μl of protein and vortex:
Table 85
Times
Comments
15 minutes
Crystalline specs appear
2 hours
Crystalline specs continue to appear
5. Redo the experiment with 1 ml of TEOS to see the catalytic activity of the protein :
Table 86
Times
Comments
1h45
Ø
6. Redo the experiment with 50 μl of HCl 1 mM and 50 μl of protein
7.Redo the experiment with 50 μl of HCl 1 mM but without our protein
Results:
When glycerol is added to the sample, as an amphiphilic modifier, an emulsion takes place which mean that glycerol act as a surfactant.
Our protein is a catalyst
Aim: Test the specificity of primers For and Rev to obtain more insert. We also switched to the Takara EX DNA polymerase for its terminal deoxynucleotide transferase activity, which adds an deoxy-A residue at the 3' end of the PCR product
Protocol: follow in this link
What we did in the lab:
Materials:
• dNTP mix
• MgCl2 25 mM
• EX polymerase buffer 10X
• RNAse free water
• C1⁄C2 insert DNA
• primers For and Rev
• Takara EX DNA polymerase
• PCR machine
• 10 μl and 200 μl pipettes
Method:
1. Use a Globalmix with :
Table 87
Volumes (μl)
dNTP
25
Mgcl2
25
Buffer 10X
50
RNAse free water
36.5
Final
50
2. For the next step use :
Table 88
Tube
Name
Premix (μl)
C1 (μl)
C2 (μl)
Primer For (μl)
Primer Rev (μl)
1
C1 insert (For +Rev)
46.5
1
Ø
1
1
2
C1 For
46.5
1
Ø
1
Ø
3
C1 Rev
46.5
1
Ø
Ø
1
4
C2 insert (For + Rev)
46.5
Ø
1
1
1
5
C2 For
46.5
Ø
1
1
Ø
6
C2 Rev
46.5
Ø
1
Ø
1
7
Without DNA
46.5
Ø
Ø
1
1
8
C1 without primer
46.5
1
Ø
Ø
Ø
9
C2 without primer
46.5
Ø
1
Ø
Ø
3. For the PCR program, start at 24°C, add 0.5 μl of Takara enzyme and proceed to 94°C. Annealing is set at 55°C.
Aim: Verification of the content of the fractions obtained from the Ni-NTA column chromatography.
Aim: Test the specificity of primers For and Rev to obtain more insert.
Protocol: follow in this link
What we did in the lab:
Materials:
• Loading buffer 6X
• PCR products
• Inserts C1 v2, C2 v2
• Primer S (For)
• Primer AS (Rev)
• Electrophoresis chamber, and power supply
• 10 μl pipette
Method:
1. Add 5 μl of DNA and 1 μl of buffer 6X to each sample 2. Use 6 μl of molecular weight marker and deposit each sample in the wells :
Table 89
L1 L2 L3 L4 L5 L6 L7 L8 L10 L11 L12 M. weight Marker Ø C1 + insert C1 + primer S C1 + primer AS C2 + insert C2 + primer S C2 + primer AS Without DNA C1 without primer C2 without primer
2. Start the electrophoresis at 50 V and 0.01 A for 10 minutes, then put it at 90 V and 0.03 A
Aim: Get back the DNA amplified by PCR.
Aim: Increase the quantity of DNA for cloning.
Protocol: follow in this link
Results :
Figure 11 : PCR gel of A1/A2/B1/B2/D1/D2/E1/E2
Aim: Test the specificity of primers For and Rev to obtain more insert.
Protocol: follow in this link
What we did in the lab:
Materials:
• Agarose
• Ethidium bromide drops (EB)
• Molecular weight ladder (Gene ruler 1 Kb, Thermofisher)
• Loading buffer 6X
• PCR products
• Electrophoresis chamber, power supply
• 10 μl pipette
Method:
1. Make an agarose gel at 0.7% 2. Prepare the samples with 5 μl of PCR products and 1 μl of loading buffer 6X 3. Depose each samples in the wells :
Table 90
L1 L2 L3 L4 L5 L6 L7 L8 L9 L10 L11 L12 L13 L14 L15 Ladder Ø A1 A2 Ø B1 B2 Ø D1 Ø D2 Ø E1 E2 Ø
4. Begin the electrophoresis at 50 V for 15 minutes, then at 90 V for 1h30
Result
There are streaks in lanes 3, 4, 9 and 11 which mean that the PCR did not work. But it seems to work for lanes 6, 7, 13 and 14.
The PCR must be done another time for A1⁄A2 and D11⁄D2 with a lower temperature (1°C).
Aim: Test the specificity of primers For and Rev to obtain more insert.
What we did in the lab:
Materials:
• Qiagen PCR clean up kit • PCR products • 200 μl pipette
Method:
1. Prepare the 8 samples:
Table 91
Samples A1 A2 B1 B2 D1 D2 E1 E2
Buffer A (μl)
< 90 90 90 90 90 90 90 90
2. Follow the protocol of the Qiagen PCR clean up kit
Result
We obtained 25 μl of each insert purified by PCR.
Aim: Test the specificity of primers For and Rev to obtain more insert.
Protocol: follow in this link
What we did in the lab:
Materials:
• PCR products
• Salt solution
• pET43.1 vector DNA
• Incubator 37°C
• 10 µl pipettes
• 1 ml Eppendorfs
Method:
1 For each inserts, prepare the following mix :
Table 92
Volumes (μl)
PCR products
4
Salt solution
1
pET 43.1
1
Total
6
2. Let ligate for 5 minutes at room temperature 3. Incubate for 10 minutes at 65°C.
Aim: Check if our plasmid was inserted in the bacteria, and to recover larger amounts of it.
Protocol: follow in this link
Aim: Increase the quality of DNA.
Protocol: follow in this link
What we did in the lab:
Materials:
• dNTPs
• MgCl2
• Buffer 20X
• RNAse free
• 10 μl , 20 µl and 200 μl pipettes
• 1.5 ml Eppendorfs
• A1⁄A2 and B1⁄B2 inserts
• PCR machine
• 10 μl 20 µl and 200 μl pipettes
• 1.5 ml Eppendorfs
Method:
1. Prepare a Globalmix with :
Table 93
Volumes (μl)
dNTPs
12.5
MgCl2
12.5
Buffer 20X
25
RNAse free
182.5
Primer S (For)
5
Primer AS (Rev)
5
Total
242.5
2. Prepare the following samples :
Table 94
A1
A2
B1
B2
Vinserts (μl)
1
1
1
1
Vglobalmix
48.5
48.5
48.5
48.5
3. Launch the PCR
Aim: Check the efficiency of the PCR.
Protocol: follow in this link
What we did in the lab:
Materials:
• Agarose
• Ethidium bromide drops
• Molecular weight ladder (Gene ruler 1 Kb, Thermofisher)
• Loading buffer 6X
• PCR products
• 10 μl pipette
• Electrophoresis machine
• 1.5 ml Eppendorfs
Method:
1. Make an agarose gel at 0.7% 2. Prepare the samples with 5 μl of PCR products and 1 μl of loading buffer 6X
3. Deposit each sample in the wells :
Table 95
L1 L2 L3 L4 L5 L6 L7 L8 Ladder (6 μl) Ø A1 : 5 μl of DNA + 1 μl of buffer 6X A2 : 5 μl of DNA + 1 μl of buffer 6X Ø D1 : 5 μl of DNA + 1 μl of buffer 6X D2 : 5 μl of DNA + 1 μl of buffer 6X Ø
4. Begin the electrophoresis at 50 V for 15 minutes, then at 90 V for 1h30.
Aim: Get DNA back.
Protocol: follow in this link
Results
Figure 12 : Digestion gel of C1 (1-8)
Aim: Recombine the inserts with the plasmid.
Protocol: follow in this link
What we did in the lab:
Materials:
• Hind III (NEB)
• Xba I 5NEB)
• Buffer CutSmart 10X (NEB)
• H2O
• 1.5 Eppendorfs
• C1 v2 and C2 v2
• Incubator 37°C
• 20 μl and 200 μl pipettes
Method:
1. Prepare a Globalmix :
Table 96
Volumes (μl)
HindIII
20
XbaI
20
Buffer CutSmart 10X
50
H2O
10
Total
100
2. Put 5 μl of the mix in each tube and add 20 μl of DNA C1 v2 and C2 v2
3. Let digest for 2 hours at 37°C and inactivate enzymes for 5 minutes at 65°C
Aim: Check if our inserts have been correctly digested by the enzyme.
Protocol: follow in this link
What we did in the lab:
Method:
Lane 1 :
Table 97
1 2 3 4 5 6 7 8 9 10 11 12 13 Ladder Ø C1.1 Ø C1.2 Ø C1.3 Ø C1.4 14 15 16 17 18 19 20 21 22 23 24 25 Ø C1.5 Ø C1.6 Ø C1.7 Ø C1.8
Lane 2:
Table 98
1
2
3
4
5
6
7
8
9
10
11
12
13
Ladder
Ø
C2.2
Ø
C2.3
Ø
C2.4
Ø
C2.5
14
15
16
17
18
19
20
21
22
23
24
25
Ø
C2.6
Ø
C2.8
Ø
C2.9
Ø
Ø
Ø
Aim: Get back the DNA which was digested and purified.
Protocol: follow in this link
What we did in the lab:
Materials
• Qiagen Gel Extraction Kit
Method:
Use Qiagen extraction kit with a final volume of 50 μl.
Results
Table 99
Sample Weight of gel (g)
C1.2
1.3435
C1.4
1.3564
C1.5
1.3825
C1.6
1.4318
C1.7
1.4392
C1.8
1.3564
C1.10
1.3800
C2.1
1.3179
C2.2
1.2500
C2.3
1.3910
C2.9
1.3668
C2.10
1.3934
Aim: Put the insert into the plasmid and transform the plasmid into bacteria.
Protocol: follow in this link
Results (on the August 4, 2016)
Several colonies have grown but they are too small so we let them grow more. But C1.2 has much more colonies so we spread it into another Petri dish.
Aim: Get back the DNA from bacteria.
Protocol: follow in this link
Results
Figure 13 : Digestion gel of B1/B2 (1-10)
Figure 14 : Digestion gel of E1/E2 (1-10)
Aim: Prepare the ligation.
Protocol: follow in this link
What we did in the lab:
Materials
• 1.5 ml Eppendorfs
• DNA
• Buffer CutSmart (NEB)
• HindIII (NEB)
• XbaI (NEB)
• H2O
• Samples B1⁄B2, E1⁄E2
• 10 μl and 20 μl pipettes
Method:
1. Make a master mix with :
Table 100
Volumes (μl) pET 43.1 B1⁄B2⁄E1⁄E2
DNA
7.5 7.5
Buffer CutSmart
2.5 7.5
HindIII
1 1
XbaI
1 1
H2O
1.5 0.5
Total
25 25
2. Distribute the volumes in each tube
Aim: Avoid the religation of the plasmid with itself.
Protocol: follow in this link
What we did in the lab:
Materials
• 1.5 ml Eppendorfs
• Digested DNA
• Dephosphorylase rSAP (recombinant Shrimp alkaline phosphatase) (NEB)
• Buffer CutSmart
• H2O
• Incubator 37°C
• 10 μl and 20 μl pipettes
Method:
1. In a 1.5 ml Eppendorf, put :
Table 101
Volumes (μl)
Digested DNA
9
Dephosphorylase
0.2
Buffer CutSmart
2
H2O
8.8
Total
20
2. Incubate one hour at 37°C
3. Incubate 5 minutes at 65°C
Aim: Increase the quantity of DNA.
Protocol: follow in this link
What we did in the lab:
Materials
• dNTP mix
• MgCl2 25 mM
• Buffer 10X
• RNAse free water
• Primer S
• Primer AS
• Samples A1⁄A2 and D1⁄D2
• 10 μl , 20 μl and 200 μl pipettes
• PCR machine
• Takara DNA polymerase enzyme
• 1.5 ml Eppendorfs
Method:
1. Prepare the mix :
Table 102
Volumes (μl)
dNTPs
12.5
MgCl2
Ø
Buffer 10X
25
RNAse free
195
Primer S (For)
5
Primer AS(Rev)
5
Total
242.5
2. To put the sample into the PCR machine, use the table :
Table 103
Tube
Names
Mix (μl)
A1 (μl)
A2 (μl)
D1 (μl)
D2 (μl)
1
A1
48.5
1
Ø
Ø
Ø
2
A2
48.5
Ø
1
Ø
Ø
3
D1
48.5
Ø
Ø
1
Ø
4
A1
48.5
Ø
Ø
Ø
1
3. For hot start PCR, launch the PCR machine and once it is at 95°C, add 0.5 μl of Takara enzyme in each tube
Results
2"/>
Figure 15 : PCR gel of A1/A2/D1/D2 without MGCl2
Aim: Have more bacteria with the insert A.
Protocol: follow in this link
What we did in the lab:
Materials
• 50 ml Falcon
• LB medium
• 20 ml and 20 μl pipettes
• TOPO
• C1 v2 and C2 v2
• Carbenicillin 50 mg⁄ml
• Incubator
• LB medium
Method:
1. In a 50 ml Falcon, put 15 ml of LB medium
2. Add 15 μl of carbenicillin
3. Add a colony in the Falcon. But for C1 v2 (2) we had to wait one more night because bacteria had not grown enough
4. Let it incubate at 37°C
Aim: Check if the PCR is efficient.
Protocol: follow in this link
What we did in the lab:
Materials
• Agarose
• A1⁄A2 and D1⁄D2
• Molecular weight marker ladder (Gene ruler 1 Kb, Thermofisher)
• Loading buffer 6X
• 10 μl pipette
Method:
1. Make a 0.7% agarose gel
2. Add 5 μl of DNA and 1 μl of loading buffer. But we used 10 μl of DNA for the sample 4
3. Deposit the sample for the electrophoresis according to this:
Table 104
L1 L2 L3 L4 L5 L6 L7 L8 L19 Ladder 6 μl Ø (1) Ø (2) Ø (3) Ø (4)
Results
A lot of streaks appeared so it did not work as expected. We have to redo this experiment with the gradient mode
Aim: Increase the quantity of DNA.
Protocol: follow in this link
What we did in the lab:
Materials
• dNTP mix
• MgCl2 25 mM
• Buffer 10X
• RNAse free
• Primer S
• Primer AS
• A1⁄A2 and D1⁄D2
• 200 μl pipette
Method:
1. Make a Globalmix with :
Table 105
Volumes (μl)
dNTPs
72.5
MgCl2
Ø
Buffer 10X
145
RNAse free
1131
Primer S
29
Primer AS
29
Total
1206.5
2. Prepare the samples :
Table 106
Name
Samples
Globalmix (μl)
DNA (μl)
A1
to (7)
48.5
1 of A1
A2
(8) to (14)
48.5
1 of A2
D1
(15) to (21)
48.5
1 of D1
D2
(22) to (28)
48.5
1 of D2
3. Deposit the samples on the gel:
Table 107
A1
A2
D1
D2
A
(1)
(8)
(15)
(22)
B
(2)
(9)
(16)
(23)
C
(3)
(10)
(17)
(24)
D
(4)
(11)
(18)
(25)
E
(5)
(12)
(19)
(26)
F
(6)
(13)
(20)
(27)
G
(7)
(14)
(21)
(28)
4. Make a 0.7% agarose gel and perform an electrophoresis with 5 µl of DNA and 1 µl of loading 6X buffer for each sample
Results
Figure 16 : PCR gel of A1/A2/B1/B2/D1/D2/E1/E2
Aim: Increase the DNA.
What we did in the lab:
Materials
• C1⁄C2
• Na Acetate (NaOAc)
• Ethanol (70%)
• Incubator
• Buffer E
• 200 μl and 1000 μl pipette
Method:
1. Calculate the final volume for each insert :
For C1 : 50×7 = 350 μl
For C2 : 50×5 = 250 μl
2. Add 1⁄10 of the volume of NaOAc and 2.5x volume of ethanol :
For C1 : 35 μl of NaOAc and 875 μl of ethanol
For C2 : 25 μl of NaOAc and 625 μl of ethanol
3. Mix the samples and incubate 8 minutes at -80°C
4. Centrifuge 5 minutes at 4°C and 15000 RPM. Then, throw out the supernatant
5. Add 1 ml of ethanol ice cold (70%)
6. Centrifuge 15 minutes at 4°C and 15000 RPM
7. Throw out the supernatant and let dry at room temperature
8. Resuspend the pellet in 50 μl of buffer E.
Results
Figure 17 : PCR gel of C1 v2 / C2 v2 with Takara enzyme
Aim: Make the inserts ligate with the plasmid.
Protocol: follow in this link
What we did in the lab:
Materials
• 10 μl, 20 μl and 200 μl pipette
• Hind III (NEB)
• Xba I (NEB)
• Buffer CutSmart
• H2O
• B1⁄B2 and C1⁄C2
Method:
1. Make a Globalmix with :
Table 108
Volumes (μl)
Hind III
45
Xba I
45
Buffer CutSmart
112.5
H2O
67.5
Total
270
2. Distribute this global mix in each of our 4 tubes so they will contain :
Table 109
Volumes (μl)
DNA
20
Hind III
1
Xb aI
1
Buffer CutSmart
2.5
H2O
1.5
Total
26
3. Follow the protocol for digestion.
Aim: Prepare the plasmid for the transformation.
Protocol: follow in this link
What we did in the lab:
Materials
• C1⁄C2
• pET43.1
• T4 ligase
• Buffer 10X
• H2O
• Incubator
• 10 μl and 20 μl pipettes
Method:
1. In a 1.5 ml eppendorf, put :
Table 110
C1 C2 pET 43.1 (a+) alone
C1 (μl)
15 Ø Ø
C2 (μl)
Ø 15 Ø
pET 43.1a(+) (μl)
4 4 4
T4 ligase (μl)
1 1 1
Buffer 10X (μl)
2.2 2.2 2.2
H2O (μl)
Ø Ø 15
Total (μl)
22.2 22.2 22.2
2. Incubate for 1 hour at room temperature 3. Incubate for 10 minutes at 65°C
Aim: Express our plasmid in bacteria.
Protocol: follow in this link
What we did in the lab:
Materials
• Qiagen Extraction Kit
Method:
Use the Qiagen gel extraction kit for a final volume of 50 μl.
Table 111
Colonies Δm (mg)
B1 colony 1
136
B1 colony 2
170
B2 colony 4
166
B2 colony 7
175
E1 colony 1
110
E1 colony 2
143
E2 colony 1
108 E2 colony 2 128 <
Aim: Ligate our inserts with the plasmid.
Protocol: follow in this link
What we did in the lab:
Method:
We use the next volumes:
Table 112
Volumes (μl)
Insert (B1 or B2 or C1 or C2)
15
pET43.1 (100 ng)
2
T4 ligase
1
Buffer 10X
2
Total
20
Aim: Put our ligated plasmid into bacteria.
Protocol: follow in this link
Results
48 hours later, no colony had grown so the transformation did not work, it has to be redone.
