<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Microbiology_week10 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Microbiology week10</SPAN></H1></DIV></DIV></DIV><DIV class="container"><DIV id="position-menu2"><UL id="menu2"><LI><FONT size="3pt" color="#8B8378">About us</FONT></LI><LI><A href=" https://2016.igem.org/Team:Pasteur_Paris/AboutUs " href="">Our team</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Attributions">Attribution</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Sponsors">Sponsors</A></LI></UL><LI><FONT size="3pt" color="#8B8378">Project</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Context">Context</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Overview">Overview</A></LI><LI><FONT size="3pt" color="#8B8378">Design </FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Science">Science </A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Moskit_devices">Devices</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Research">Research &amp; development</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Methodology">Methodology</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Design_Biology">Design vs Biology</A></LI><LI><FONT size="3pt" color="#8B8378">Labwork </FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Safety_Security">Safety &amp; Security</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Protocol ">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Netbook ">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Results ">Results</A></LI><LI>Human Practice</LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Collaborations ">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Scenarios">Scenarios</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Meet-up ">Meet-up</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Engagement">Communication</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Law"> Law</A></LI><LI><FONT size="3pt" color="#8B8378">Parts</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Description">Description</A></LI><LI><FONT size="3pt" color="#8B8378">Achievements</FONT></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/medals">Medals</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/Prizes"> Special Prizes</A></LI><LI><A href="https://2016.igem.org/Team:Pasteur_Paris/rewards"> Awards obtained!</A></LI></DIV></DIV><DIV id="weeks"><DIV id="weekslist"><CENTER><H1><B>click on the weeks</B></H1></CENTER></DIV></DIV><DIV><H2><B>Microbiology Notebook</B></H2></DIV><DIV id="week10"><H5><B>Week 10 </B></H5><H3><B>August 8, 2016:</B></H3><H4> 150. Culture for miniprep of C2 v2 and B1 v2 </H4><A href="#exp2"><H4>  151. Digestion of B2, E1 and C2 from the 4<SUP>th</SUP> of August </H4></A><A href="#exp3"><H4>  152. Electrophoresis with the results of digestion </H4></A><A href="#exp4"><H4>  153. PCR of inserts A1/A2/D1/D2 </H4></A><A href="#exp5"><H4>  154. Gel extraction of A1/A2/D1/D2 </H4></A><A href="#exp6"><H4>  155. Gel extraction of B2, E1 and E2 </H4></A><A href="#exp7"><H4>  156. Resuspension of inserts B2/E1/E2 </H4></A><H3><B>August 9, 2016:</B></H3><H4>  157. Ligation of inserts B2⁄E1⁄E2 with pET 43.1a(+) </H4><A href="#exp9"><H4>  158. Miniprep of C2 v2 (TOP 10) and B1 v2 (TOP 10 ) </H4></A><A href="#exp10"><H4>  159. Miniprep preculture of C1 v2 in pET 43.1a(+) </H4></A><A href="#exp11"><H4>  160. Preparation of 10 aliquots of carbenicillin at 50 mg⁄m </H4></A><A href="#exp12"><H4>  161. Electrophoresis of the PCR done on the 8<SUP>th</SUP> of August with A1⁄A2⁄D1⁄D2 </H4></A><A href="#exp13"><H4>  162. Transformation of A1⁄A2⁄D1⁄D2 in TOP 10 </H4></A><A href="#exp14"><H4>  163. Transformation of E1⁄E2 and B2 in TOP 10 </H4></A><H3><B>August 10, 2016:</B></H3><H4>  164. Miniprep of C1 v2 (culture from the 9 of August) </H4><A href="#exp16"><H4>  165. Digestion of C1 v2 before electrophoresis </H4></A><A href="#exp17"><H4>  166. Digestion of pET 43.1 (a+) with Xba I and Hind III </H4></A><A href="#exp18"><H4>  167. Electrophoresis and gel extraction </H4></A><A href="#exp19"><H4>  168. Electrophoresis of C1 digested </H4></A><A href="#exp20"><H4>  169. Transformation of B2 (4, 7 and 9), E1 (1 and 2) and E2 (2, 3, 4, 5, 6 and 7) </H4></A><A href="#exp21"><H4>  170. Aliquot of antibodies 462 </H4></A><A href="#exp22"><H4>  171. Culture of A1⁄A2⁄D1⁄D2 </H4></A><A href="#exp23"><H4>  172. Ligation of A1⁄A2⁄D1⁄D2 in TOPO </H4></A><A href="#exp24"><H4>  173. Transformation of A1⁄A2⁄D1⁄D2 with TOPO in TOP 10 competent cells </H4></A><A href="#exp25"><H4>  174. Transformation of B1 v2 and C2 v2 in BL21DE3 </H4></A><A href="#exp26"><H4>  175. Dosage of digested pET 43.1 (a+) </H4></A><A href="#exp27"><H4>  176. Transformation of C2 v2 and B1 v2 in pET 43.1a(+) and DH3α</H4></A><H3><B>August 11, 2016:</B></H3><H4>  177. Absorbance of precultures C2 v2(1, 2, 3) and B1 v2 (1, 2, 16) </H4><A href="#exp29"><H4>  178. Dephosphorylation of pET 43.1a(+) digested on the 10<SUP>th</SUP> of August </H4></A><A href="#exp30"><H4>  179. Miniprep of B1 v2 ⁄E1⁄E2 in TOPO </H4></A><A href="#exp31"><H4>  180. Transformation of B1 v2 colony 8 and C2 v2 colony 16 in pET 43.1 (a+) and in DH 3α</H4></A><A href="#exp32"><H4>  181. Precultures of B1 v2 and C2 v2 </H4></A><A href="#exp33"><H4>  182. Digestion of pET 43.1 (a+) with Xba I and Hind III </H4></A><H3> August 12, 2016:</H3><H4>  183. Miniprep from precultures of B2 v2/E1/E2 in TOPO </H4><A href="#exp35"><H4>  184. Digestion of inserts B2 v2/E1/E2 with XbaI and HindIII </H4></A><A href="#exp36"><H4>  185. Culture of C2 v2 and B1 v2 in 1 l of LB  </H4></A><A href="#exp37"><H4>  186. Agarose gel to analyze digestion of pET 43.1(a+) done on the 11<SUP>th</SUP> of August </H4></A><DIV class="lightbox" id="exp1"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Increase the quantity of DNA before extraction. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4b/T--Pasteur_Paris--Bacterial_culture_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• 50 ml Falcon tube• Shaking incubator (INFORS HT)• Swing bucket centrifuge (JOUAN GR41)• Colonies of C2 v2 and B1 v2 • carbenicillin at 50 mg/ml • LB medium • Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>)
<H6><U>Method :</U></H6>
         1. In a 50 ml falcon, put 48 ml of LB and 48 µl of carbenicillin. 
               2. For B1 v2 :  2.a Prepare 13 Eppendorfs of 1.5 ml in which add 1 ml of the previous mix.  2.b Take with a toothpick colonies on the petri dish.  2.c Place the toothpick in a tube.  2.d Let incubate overnight at 37°C and 150 rpm. 
                3. For C2 v2 :  3.a Prepare 20 eppendorfs of 1.5 ml with 1 ml of LB and carbenicillin mix. 3.b Take colonies and place it as previously explained. 3.c Let incubate overnight. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp2"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> As the transformations did not work (B2, E1 and E2 in pET 43.1(a+) ) with TOP 10 competent cells, we take the products of the midiprep done on the 4<SUP>th</SUP> of August and we digest before redoing the transformation.<H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• Restriction enzymes: Xba I, Hind III (New England Biolabs, NEB) • Restriction enzyme buffers • 37 °C water bath• Shaking incubator (INFORS HT)• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>)<H6><U>Method :</U></H6>
                       1. Realize a Mastermix and store it on ice : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 113</U></P></I></CAPTION><THEAD><TR><TH>Reactants</TH><TH>Volumes (µl) </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P>Xba I</P></STRONG></TD><TD align="center"> 30 </TD></TR><TR><TD align="center"><STRONG><P>Hind III</P></STRONG></TD><TD align="center"> 30 </TD></TR><TR><TD align="center"><STRONG><P>Buffer 2.1 </P></STRONG></TD><TD align="center"> 90 </TD></TR><TR><TD align="center"><STRONG><P>Distilled H<SUB>2</SUB>O</P></STRONG></TD><TD align="center"> 90 </TD></TR><TR><TD align="center"><STRONG><P>Total</P></STRONG></TD><TD align="center"> 150 </TD></TR></TBODY></TABLE>
	           2. In For each insert (B2, E1, E2), prepare 10 tubes of 1.5 ml (10 eppendorfs of miniprep products). 
	           3. In each tube, put 25 μl of DNA and 5 μl of Master mix. 
	           4. Let digest 2 hours at 37 °C, then incubate 5 minutes at 65 °C. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp3"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Check if the digestion has been done, it would mean that the plasmid contains the insert.<H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <AHREF>link) • Colonies B2, E1 and E2 • Electrophoresis chamber • TAE 1X • Ethidium bromide drops (EB) <H6><U>Method :</U></H6>
         1. Make a gel with 1.4 g of agarose, 200 ml of TAE 1X and 4 droplets of EB. 
         2. Prepare the samples with 30 µl of digested mix and 6 µl of Gene ruler (Thermofisher)ladder. 
                3. Follow the deposit table : 
	Ladder |Ø| B2 (1) | B2 (2) | B2 (3) | B2 (4) | B2 (5) | B2 (6) | B2 (7) | B2 (8) | B2 (9) | B2 (10) |Ø| E1 (1) | E1 (2)	| E1 (3) | E1 (4) | E1 (5) |Ø|Ø| ladder  | E1 (6) | E1 (7) | E1 (8) | E1 (9) | E1 (10) |Ø| E2 (1)  | E2 (2) | E2 (3) | E2 (4) | E2 (5) | E2 (6) | E2 (7) | E2 (8) | E2 (9) | E2 (10) |Ø|Ø
	         4. Launch the electrophoresis for 20 minutes at 50 V, then 45 minutes at 130 V. <H6><U>Results :</U></H6><CENTER><I><P><U>Figure 18 :</U> Gel of the results of digestion </P></I></CENTER></AHREF></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp4"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Increase the quantity of insert. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/b/be/T--Pasteur_Paris--PCR_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • 1.5 ml eppendorfs • Takara enzyme • Primers S and AS • dNTP mix 10 mM • Tak Ex Buffer 6X • Distilled H<SUB>2</SUB>O • MgCl<SUB>2</SUB><H6><U>Method :</U></H6>
                  1. Prepare the following tubes :<TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 114</U></P></I></CAPTION><THEAD><TR><TH> Mix with two primers </TH><TH> Mix with primer S only </TH><TH> Mix with primer AS only </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> Takara enzyme (µl) </P></STRONG></TD><TD align="center"> 6 </TD><TD align="center"> 6 </TD><TD align="center"> 6 </TD></TR><TR><TD align="center"><STRONG><P> Primer S (µl) </P></STRONG></TD><TD align="center"> 6 </TD><TD align="center"> 6 </TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> Primer AS (µl) </P></STRONG></TD><TD align="center"> 6 </TD><TD align="center">Ø</TD><TD align="center"> 6 </TD></TR><TR><TD align="center"><STRONG><P> MgCl<SUB>2</SUB> (µl) </P></STRONG></TD><TD align="center"> 15 </TD><TD align="center"> 15 </TD><TD align="center"> 15 </TD></TR><TR><TD align="center"><STRONG><P> dNTPs (µl) </P></STRONG></TD><TD align="center"> 15 </TD><TD align="center"> 15 </TD><TD align="center"> 15 </TD></TR><TR><TD align="center"><STRONG><P> Buffer (µl) </P></STRONG></TD><TD align="center"> 30 </TD><TD align="center"> 30 </TD><TD align="center"> 30 </TD></TR><TR><TD align="center"><STRONG><P> H<SUB>2</SUB>O (µl) </P></STRONG></TD><TD align="center"> 216 </TD><TD align="center"> 222 </TD><TD align="center"> 222 </TD></TR><TR><TD align="center"><STRONG><P> Total (µl) </P></STRONG></TD><TD align="center"> 294 </TD><TD align="center"> 294 </TD><TD align="center"> 294 </TD></TR></TBODY></TABLE>
                  2. For each mix, spread 49 µl of it in the samples. 
     3. Add 1 µl of DNA following the number of tubes : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 115</U></P></I></CAPTION><THEAD><TR><TH> Mix with two primers </TH><TH> Mix with primer S only </TH><TH> Mix with primer AS only </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> A1 </P></STRONG></TD><TD align="center"> tube 1 </TD><TD align="center"> tube 2 </TD><TD align="center"> tube 3 </TD></TR><TR><TD align="center"><STRONG><P> A2 </P></STRONG></TD><TD align="center"> tube 4 </TD><TD align="center"> tube 5 </TD><TD align="center"> tube 6 </TD></TR><TR><TD align="center"><STRONG><P> D1 </P></STRONG></TD><TD align="center"> tube 7 </TD><TD align="center"> tube 8 </TD><TD align="center"> tube 9 </TD></TR><TR><TD align="center"><STRONG><P> D2 </P></STRONG></TD><TD align="center"> tube 10 </TD><TD align="center"> tube 11 </TD><TD align="center"> tube 12 </TD></TR></TBODY></TABLE>
	     And tube 13 is the one without DNA
     4. Launch the process of PCR.
     5. Do an electrophoresis with the results of PCR </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp5"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> get back the maximum quantity of DNA.<H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4e/T--Pasteur_Paris--Gel_extraction_protocol.pdf">link</A><H6><U>What we did in the lab:</U></H6><H6><U>Materials :</U></H6>• Gel of A1⁄A2⁄D1⁄D2 • QIAGEN Extraction gel kit<H6><U>Method :</U></H6>
Follow the Qiagen gel extraction kit steps with the gel : A1 : m  = 122 mg A2 : m  = 153 mg D1 : m  = 120 mg D2 : m  = 152 mg<H6><U>Results :</U></H6><CENTER><I><P><U>Figure 19 :</U> Extraction gel of A1-A2-D1-D2 </P></I></CENTER></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp6"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Get back purified DNA.<H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4e/T--Pasteur_Paris--Gel_extraction_protocol.pdf">link</A><H6><U>What we did in the lab:</U></H6><H6><U>Materials:</U></H6>• Gel of B2/E1/E2 • QIAGEN Extraction gel kit<H6><U>Method:</U></H6>
Follow the Qiagen Extraction gel kit steps with :<TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 116</U></P></I></CAPTION><THEAD><TR><TH> DNA </TH><TH> Colonies </TH><TH> Weight of the gel (mg) </TH></TR></THEAD><TBODY><TR><TD rowspan="3;" align="center"><STRONG><P> B2 </P></STRONG></TD><TD align="center"> Colony 4 </TD><TD align="center"> 367 </TD></TR><TR><TD align="center"> Colony 7 </TD><TD align="center"> 432 </TD></TR><TR><TD align="center"> Colony 9 </TD><TD align="center"> 269 </TD></TR><TR><TD rowspan="7;" align="center"><STRONG><P> E2 </P></STRONG></TD><TD align="center"> Colony 1 </TD><TD align="center"> 300 </TD></TR><TR><TD align="center"> Colony 2 </TD><TD align="center"> 355 </TD></TR><TR><TD align="center"> Colony 3 </TD><TD align="center"> 354 </TD></TR><TR><TD align="center"> Colony 4 </TD><TD align="center"> 314 </TD></TR><TR><TD align="center"> Colony 5 </TD><TD align="center"> 299 </TD></TR><TR><TD align="center"> Colony 6 </TD><TD align="center"> 275 </TD></TR><TR><TD align="center"> Colony 7 </TD><TD align="center"> 277 </TD></TR><TR><TD rowspan="2;" align="center"><STRONG><P> E1 </P></STRONG></TD><TD align="center"> Colony 1 </TD><TD align="center"> 404 </TD></TR><TR><TD align="center"> Colony 2 </TD><TD align="center"> 321 </TD></TR></TBODY></TABLE></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp7"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Storage of the inserts.<H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• NaAc • Ethanol 70%; • Inserts B2/E1/E2 <H6><U>Method :</U></H6>
1. Use 1/10 volume of NaAc and 2.5 colume of ethanol for each insert : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 117</U></P></I></CAPTION><THEAD><TR><TH> B2 </TH><TH> E1 </TH><TH> E2 </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> NaAc (µl) </P></STRONG></TD><TD align="center"> 15 </TD><TD align="center"> 10 </TD><TD align="“center”;" valign="center"> 35 </TD></TR><TR><TD align="center"><STRONG><P> Ethanol (µl) </P></STRONG></TD><TD align="center"> 875 </TD><TD align="“center”;" valign="center"> 250 </TD><TD align="“center”;" valign="center"> 375 </TD></TR></TBODY></TABLE>
	2. Resuspend B2⁄E1⁄E2 in 15 µl of H<SUB>2</SUB>O each. 
       3. We estimated the weight of each inserts :  m(B2) = 240 ng  m(E1) =60 ng  m(E2) = 420 ng </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp8"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Prepare the transformation. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/6/6f/T--Pasteur_Paris--Ligation_protocol.pdf">link</A><H6><U>What we did in the lab:</U></H6><H6><U>Materials :</U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • Inserts B2/E1/E2 • pET 43.1(a+) • TOPO vector • Distilled water • Ligase <H6><U>Method :</U></H6>
Use the following volumes : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 118</U></P></I></CAPTION><THEAD><TR><TH> E1 </TH><TH> E2 </TH><TH> B2 </TH><TH> pET 43.1(a+) </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> E1 (µl) </P></STRONG></TD><TD align="center"> 15 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> E2 (µl) </P></STRONG></TD><TD align="center">Ø</TD><TD align="center"> 15 </TD><TD align="center">Ø</TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> B2 (µl) </P></STRONG></TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center"> 15 </TD><TD align="center">Ø</TD></TR><TR><TD align="center"><STRONG><P> pET 43.1(a+) (µl) </P></STRONG></TD><TD align="center"> 4 </TD><TD align="center"> 4 </TD><TD align="center"> 4 </TD><TD align="center"> 4 </TD></TR><TR><TD align="center"><STRONG><P> Ligase (µl) </P></STRONG></TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> TOP0  (µl) </P></STRONG></TD><TD align="center"> 2.2 </TD><TD align="center"> 2.2 </TD><TD align="center"> 2.2 </TD><TD align="center"> 2.2 </TD></TR><TR><TD align="center"><STRONG><P> H<SUB>2</SUB>O (µl) </P></STRONG></TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center">Ø</TD><TD align="center"> 15 </TD></TR><TR><TD align="center"><STRONG><P> V<SUB>total</SUB> (µl) </P></STRONG></TD><TD align="center"> 22.2 </TD><TD align="center"> 22.2 </TD><TD align="center"> 22.2 </TD><TD align="center"> 22.2 </TD></TR></TBODY></TABLE></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp9"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Increase the quantity of DNA. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • Qiagen Miniprep kit • Digestion enzyme Xba I and Hind III • Digestion buffer 2.1 • 1.5 ml Eppendorfs • Electrophoresis chamber • Distilled water
<H6><U>Method :</U></H6>
	1. Use the Qiagen kit for our cultures from the 8<SUP>th</SUP> of August :  13 Eppendorfs of B1 v2.  20 Eppendorfs of C2 v2. 
	2. Digest the plasmid with the following volumes for each sample : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 119</U></P></I></CAPTION><THEAD><TR><TH> Volumes (µl) </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> DNA </P></STRONG></TD><TD align="center"> 5 </TD></TR><TR><TD align="center"><STRONG><P> Xba I </P></STRONG></TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> Hind III </P></STRONG></TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> Buffer 2.1 </P></STRONG></TD><TD align="center"> 2 </TD></TR><TR><TD align="center"><STRONG><P> H<SUB>2</SUB>O </P></STRONG></TD><TD align="center"> 11 </TD></TR><TR><TD align="center"><STRONG><P> Total </P></STRONG></TD><TD align="center"> 20 </TD></TR></TBODY></TABLE>
	3. Analyze the digestion by electrophoresis on an agarose gel prepared with 1.05 g of agarose in 125 ml of TAE 1X. 
	4. Launch the electrophoresis, following the deposit table :
C2 | Ladder | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | Ladder 
B1 | ladder | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 |Ø|Ø|Ø| 20 | ladder</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp10"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Have different clones to know which contain the insert. <H6><U> Protocol :</U></H6><H6> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4b/T--Pasteur_Paris--Bacterial_culture_protocol.pdf">link</A></H6><H6><U>What we did in the lab:</U></H6><H6><U>Materials :</U></H6>&amp;• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • Carbenicillin at 50 mg/ml • Digestion enzyme Xba I and Hind III • LB medium • pET43.1(a+) • C1 v2 colonies • Shaking incubator (INFORS HT)<H6><U>Method :</U></H6>
	1. Prepare 20 ml of LB with 20 µl of carbenicillin. 
	2. Put 1 ml of this mix in twenty 1.5 ml Eppendorfs. 
      3. Take 20 colonies of the petri dish C1 v2 and put them in the previous Eppendorfs. 
      4. Let incubate overnight at 37°C and 150 rpm. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp11"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Create a stock of antibiotic. <H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • Carbenicillin at 50 mg/ml • 1.5 ml Eppendorfs • 15 ml Falcon • Distilled water <H6><U>Method :</U></H6>
	1. Prep: Put 500 mg of carbenicillin in a 15 ml Falcon and 10 ml of distilled water. Then, put the Falcon on ice. 
	2. Aliquot the mix in 10 Eppendorfs of 1.5 ml. 
      3. Store at -20°C </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp12"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Check if the PCR works. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><H6><U>Results :</U></H6><CENTER><I><P><U>Figure 20 :</U> Electrophoresis gel of the PCR</P></I></CENTER>
The PCR works properly since we noticed significant bands at the expected level.</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp13"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> After their ligation in TOPO cloning vector, they will be transformed into TOP 10 cells. <H6><U>Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials</U></H6>• Ligation’s products of A1/A2/D1/D2&amp;bull ; TOP 10 competent cells &amp;bull ; SOC • Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • Xgal at 10 mg/ml
<H6><U>Method</U></H6>
	1. Add 6 µl of ligation product in 50 µl of competent cells.
	2. Put the samples 30 minutes on ice, then 40 seconds at 42°C.
	3. Put the samples 3 minutes on ice.
	4. Add 150 µl of SOC.
5. Let incubate 40 minutes at 37°C and 150 rpm.
6. Take LB with carbenicillin petri plate and add Xgal to reach 40 µg/ml. As the volume is about 25 ml, add 1 mg of Xgal and as the stock solution is 10 mg/ml, spread 100 µl on the plate.
7. Spread bacteria on four distincts petri dishes (one for each insert).</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp14"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> After their ligation we must transform the inserts into bacteria. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Method</U></H6>
Add 10 µl of ligation product (to have 100 mg) at the beginning. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp15"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Get back the DNA. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A><H6><U>What we did in the lab : </U></H6><H6><U>Method</U></H6>
We use a final volume of 50 µl. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp16"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Split the insert and the plasmid. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><H6><U>What we did in the lab:</U></H6><H6><U>Materials:</U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • Qiagen Miniprep kit • Digestion enzyme Xba I and Hind III • Digestion buffer 2 X • 1.5 ml Eppendorfs • Distilled water • Shaking incubator (INFORS HT)<H6><U>Method:</U></H6>
	1. Realize a master mix with : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 120</U></P></I></CAPTION><THEAD><TR><TH> Volumes (µl) </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> Xba I </P></STRONG></TD><TD align="center"> 20 </TD></TR><TR><TD align="center"><STRONG><P> Hind III </P></STRONG></TD><TD align="center"> 20 </TD></TR><TR><TD align="center"><STRONG><P> Buffer 2X </P></STRONG></TD><TD align="center"> 40 </TD></TR><TR><TD align="center"><STRONG><P> H<SUB>2</SUB>O </P></STRONG></TD><TD align="center"> 220 </TD></TR><TR><TD align="center"><STRONG><P> Total </P></STRONG></TD><TD align="center"> 300 </TD></TR></TBODY></TABLE>
	2. Put µl of the master mix in each of the twenty 1.5 ml Eppendorfs and add 5 µl of DNA.
	3. Let incubate one hour at 37°C and 150 rpm, then 5 minutes at 65°C<H6><U>Results</U></H6> The 10<SUP>th</SUP> of August, we realize that we forgot to put Xgal, so we re-spreaded the previous mix on petri dishes with Xgal.</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp17"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> We want to produce 5 µg of dephosphorylated pET 43.1(a+) from pET 43.1(a+) at 400 ng/ml and we start with the digestion. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • Qiagen Miniprep kit • Digestion enzyme Xba I and Hind III • CutSmart buffer• 1.5 ml Eppendorfs • Distilled water • Shaking incubator (INFORS HT)<H6><U>Method :</U></H6>
	1. Put all the following reactants in a 1.5 ml Eppendorf and let digest one hour at 37°C : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 121</U></P></I></CAPTION><THEAD><TR><TH> Volumes (µl) </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> DNA </P></STRONG></TD><TD align="center"> 12.5 </TD></TR><TR><TD align="center"><STRONG><P> Xba I </P></STRONG></TD><TD align="center"> 2 </TD></TR><TR><TD align="center"><STRONG><P> Hind III </P></STRONG></TD><TD align="center"> 4 </TD></TR><TR><TD align="center"><STRONG><P> Buffer CutSmart </P></STRONG></TD><TD align="center"> 5 </TD></TR><TR><TD align="center"><STRONG><P> H<SUB>2</SUB>O </P></STRONG></TD><TD align="center"> 26.5 </TD></TR><TR><TD align="center"><STRONG><P> Total </P></STRONG></TD><TD align="center"> 50 </TD></TR></TBODY></TABLE>
	2. Inactivate the enzymes 5 minutes at 65°C. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp18"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Get back the digested and purified plasmid before dephosphorylation. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• Products from the digestion of pET 43.1(a+) with Hind III and XbaI • Agarose• Electrophoresis chamber <H6><U>Method :</U></H6>
	1. Make a 0.7% agarose gel 
	2. Prepare the chamber to do the migration at 100 V with two wells for pET43.1(a+) and one for the ladder. 
	3. Take the results and follow the kit steps of Qiagen extraction kit.<H6><U>Results :</U></H6>
We notice two bands for digested pET43.1(a+), one at 6000 bp and another between 1000 and 1500 bp. We extract the bands at 6000 bp.
We obtained :	 m1 = 0.4079 g m2 = 0 .3720 g</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp19"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Check if the digestion works properly and if we have inserts. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><H6><U>What we did in the lab:</U></H6><H6><U>Materials :</U></H6>• Products from the digestion of C1 • Agarose• Electrophoresis chamber <H6><U>Method :</U></H6>
	1. Take the 20 µl of each sample from the digestion and add 4 µl of loading buffer 6X. 
	2. Do the electrophoresis, following the deposit table : 
Ladder |Ø| 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 |Ø| Ladder 
Ladder |Ø| 17 | 18 | 19 | 20 <H6><U>Results</U></H6>
There is no insert. The colonies do not contain our insert, we must redo the experiment with new colonies. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp20"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Transform the bacterias with our recombined plasmid. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Method :</U></H6>
The volumes of insert are too small, we add 5 µl of H<SUB>2</SUB>O and diluted at 1/100. 
We performed a transformation in DH5α with 1 µl of DNA and 50 µl of competent cells. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp21"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Have more antibodies. <H6><U>Results :</U></H6> We obtained 30 µ/aliquot. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp22"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Increase the quantity of colonies containing inserts. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4b/T--Pasteur_Paris--Bacterial_culture_protocol.pdf">link</A></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp23"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Ligate the insert and the plasmid. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/6/6f/T--Pasteur_Paris--Ligation_protocol.pdf">link</A></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp24"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Transform our inserts in TOP 10 competent cells. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp25"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Have bacteria with the right plasmid to produce our protein. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A><H6><U>What we did in the lab:</U></H6><H6><U>Materials:</U></H6>• Digested B1 and C2 • BL21DE3 competent cells• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • Shaking incubator (INFORS HT)• SOC <H6><U>Method:</U></H6>
	1. Take 3 samples of C2 v2 (10, 13 and 16) and 2 samples of B1 v2 (6 and 8).
2. Put 1 µl of DNA in 99 µl of H<SUB>2</SUB>O. Then, put 1 µl of DNA (B1 v2 or C2 v2) in 50 µl of BL21DE3 competent cells.
3. Put the samples 30 minutes on ice and then 40 seconds at 42°C.
4. Put the samples 3 minutes on ice. 
5. Add 150 µl of SOC and let incubate 30 minutes at 37°C and 150 rpm.
6. Spread the mix on a petri dish with LB and carbenicillin.
7. Let incubate overnight at 37°C and 150 rpm.</FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp26"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Have the concentration of digested pET 43.1(a+) <H6><U>Results</U></H6> Measure the concentration of digested pET 43.1(a+) with a Nanodrop. For the first tube, we find a concentration of 6.8 ng/µl in 46 µl. For the second tube, we find a concentration of 8.8 ng/µl in 46 µl. Then, store the samples at -20°C. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp27"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Transform C2 v2 and B1 v2 in pET43.1a(+) and DHα . <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A><H6><U>Results</U></H6> For B2 (4, 7, 9), E1 (1, 2) and E2 (2, 3, 4, 5, 6, 7) we see small colonies so we let and 25 µl of carbenicillin.
For B1 v2 and C2 v2 in pET43.1(a+), transformations were successful.
For A1, A2 and D1, D2 there were too many bacteria growing so we took a colony, diluted it in LB and then spread it on petri dish with LB, Xgal and carbenicillin. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp28"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> To produce proteins. <H6><U>What we did in the lab:</U></H6><H6><U>Materials:</U></H6>• Spectrophotometer Ultrospec 3100• iPTG at 0.5 M
<H6><U>Method:</U></H6>
	1. Make two measurements separated of 30 minutes : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 122</U></P></I></CAPTION><THEAD><TR><TH> Sample </TH><TH> 1 </TH><TH> 2 </TH><TH> 3 </TH><TH> 4 </TH><TH> 5 </TH><TH> 6 </TH><TH> Time of addition of iPTG </TH><TH> Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> B1 (1) </P></STRONG></TD><TD align="center"> 0.022 </TD><TD align="center"> 0.062 </TD><TD align="center"> 0.152 </TD><TD align="center"> 0.344 </TD><TD align="center"> 0.557 </TD><TD align="center"> 0.694 </TD><TD align="center"> 16 h 55 </TD><TD align="center"> 0.859 </TD></TR><TR><TD align="center"><STRONG><P> B1 (2) </P></STRONG></TD><TD align="center"> 0.185 </TD><TD align="center"> 0.417 </TD><TD align="center"> 0.693 </TD><TD align="center"> 15 h 25 </TD><TD align="center"> 0.956 </TD></TR><TR><TD align="center"><STRONG><P> B1 (3) </P></STRONG></TD><TD align="center"> 0.075 </TD><TD align="center"> 0.211 </TD><TD align="center"> 0.413 </TD><TD align="center"> 0.688 </TD><TD align="center"> 15 h 55 </TD><TD align="center"> 0.920 </TD></TR><TR><TD align="center"><STRONG><P> C2 (1) </P></STRONG></TD><TD align="center"> 0.060 </TD><TD align="center"> 0.166 </TD><TD align="center"> 0.350 </TD><TD align="center"> 0.627 </TD><TD align="center"> 0.699 </TD><TD align="center"> 16 h 25 </TD><TD align="center"> 0.905 </TD></TR><TR><TD align="center"><STRONG><P> C2 (2) </P></STRONG></TD><TD align="center"> 0.044 </TD><TD align="center"> 0.119 </TD><TD align="center"> 0.296 </TD><TD align="center"> 0.510 </TD><TD align="center"> 0.698 </TD><TD align="center"> 16 h 25 </TD><TD align="center"> 0.910 </TD></TR><TR><TD align="center"><STRONG><P> C2 (16) </P></STRONG></TD><TD align="center"> 0.080 </TD><TD align="center"> 0.230 </TD><TD align="center"> 0.445 </TD><TD align="center"> 0.689 </TD><TD align="center"> 15 h 55 </TD><TD align="center"> 0.907 </TD></TR></TBODY></TABLE>
	2. When the OD<SUB>600 nm</SUB> reached 0.7 keep the last measurement and centrifuge for 3 minutes at 8000 g. 
	3. Throw away the supernatant and store at -20°C. 
	4. Add iPTG to reach 0.3 mM. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp29"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Make the future ligation easier. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/c/c0/T--Pasteur_Paris--dephosphorylation_protocol.pdf">link</A><H6><U>What we did in the lab :</U></H6><H6><U>Materials :</U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • rSAP • CutSmart buffer• 1.5 ml Eppendorfs • Distilled water • Shaking incubator (INFORS HT)<H6><U>Method :</U></H6>
	1. Start with tube 2 (8.6 ng/µl in 46 µl) and use the following mix : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 123</U></P></I></CAPTION><THEAD><TR><TH> Volumes ( µl) </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> DNA </P></STRONG></TD><TD align="center"> 46 </TD></TR><TR><TD align="center"><STRONG><P> CutSmart </P></STRONG></TD><TD align="center"> 6 </TD></TR><TR><TD align="center"><STRONG><P> H<SUB>2</SUB>O </P></STRONG></TD><TD align="center"> 6.7 </TD></TR><TR><TD align="center"><STRONG><P> rSAP  </P></STRONG></TD><TD align="center"> 1.3 </TD></TR><TR><TD align="center"><STRONG><P> Total </P></STRONG></TD><TD align="center"> 60 </TD></TR></TBODY></TABLE>
	2. Let incubate 30 minutes at 37°C then 5 minutes at 65°C. 
	3. Do the same for tube 1. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp30"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Get back the DNA. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A><H6><U> What we did in the lab </U></H6><H6><U>Materials :</U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • LB medium • Carbenicillin at 50 mg/ml • B1/E1/E2 in TOPO <H6><U>Method :</U></H6>
We do 31 precultures with a mix with 1 ml of LB and 1 µl of carbenicillin to have : 3 samples of E1  14 samples of B2  14 samples of E2 </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp31"><FIGURE><FIGCAPTION><H6><U> Aim :</U></H6> Send our insert for sequencing as the transformations in BL21DE3. <H6><U> Protocol :</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/c/ca/T--Pasteur_Paris--Transformation_protocol.pdf">link</A></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp32"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Have precultures to redo the cultures in 4 x 1 l of LB as they grow well. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/4/4b/T--Pasteur_Paris--Bacterial_culture_protocol.pdf">link</A></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp33"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Increase the quantity of plasmid for the next ligation. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp34"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Increase the quantity of DNA. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/d/d5/T--Pasteur_Paris--Miniprep_protocol.pdf">link</A></FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp35"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Check if the colonies we took contain the insert. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/a/ab/T--Pasteur_Paris--Restriction_digestion_protocol.pdf">link</A><H6><U> What we did in the lab </U></H6><H6><U> Materials </U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • Digestion enzyme Xba I and Hind III • Digestion buffer 2.1 • 1.5 ml Eppendorfs • Distilled water • Shaking incubator (INFORS HT)• Inserts B2/E1/E2 <H6><U> Method </U></H6>
1. In a 1.5 ml Eppendorf, put : <TABLE><CAPTION align="bottom" align="center"><I><P><U>Table 124</U></P></I></CAPTION><THEAD><TR><TH> Volumes (µl) </TH></TR></THEAD><TBODY><TR><TD align="center"><STRONG><P> DNA </P></STRONG></TD><TD align="center"> 5 </TD></TR><TR><TD align="center"><STRONG><P> Xba I </P></STRONG></TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> H<SUB>2</SUB>O </P></STRONG></TD><TD align="center"> 11 </TD></TR><TR><TD align="center"><STRONG><P> Hind III </P></STRONG></TD><TD align="center"> 1 </TD></TR><TR><TD align="center"><STRONG><P> Buffer 2.1 </P></STRONG></TD><TD align="center"> 2 </TD></TR><TR><TD align="center"><STRONG><P> Total </P></STRONG></TD><TD align="center"> 20 </TD></TR></TBODY></TABLE>
2. Let incubate one hour at 37°C , then 5 minutes at -20°C. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp36"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Produce the protein in higher quantity. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/a/a4/T--Pasteur_Paris--Protein_induction_protocol.pdf">link</A><H6><U> What we did in the lab </U></H6><H6><U> Materials </U></H6>• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this <A href="https://2016.igem.org/Team:Pasteur_Paris/Science">link</A>) • iPTG • Carbenicillin at 50 mg/ml
<H6><U> Method </U></H6>
1. In a 2 l Erlenmeyer, put 1 l of LB and 1 ml of carbenicillin and incubate at 37°C and 150 rpm to warm the liquid.
2. For each preculture of 25 ml, put it in a 50 ml Falcon, centrifuge 5 minutes at 3500 rpm, throw away the supernatent and resuspend the pellet in 5 ml of clean LB.
3.Centrifuge 5 minutes at 3500 rpm, throw away the supernatent and resuspend the pellet in 20 ml of clean LB.
4. In each 2 l erlenmeyer, add 5 ml of our preculture : 4 erlenmeyer for C2 and 4 for B1. 
5. Let incubate at 37 °C ann 150 rpm and measure the DO. 
6. Once the DO reaches 0.7, add 1 ml of iPTG.
7. Let incubate for 3 hours. 
8. Centrifuge the cultures. 
9. Resuspend the pellet in 10 ml of lysis buffer (See protein purification protocol) in a 50 ml Falcon of known weight.
10. Store at -20°C. </FIGCAPTION></FIGURE></DIV><DIV class="lightbox" id="exp37"><FIGURE><FIGCAPTION><H6><U> Aim:</U></H6> Check if the digestion works. <H6><U> Protocol:</U></H6> follow in this <A href="https://static.igem.org/mediawiki/2016/6/60/T--Pasteur_Paris--Gel_electrophoresis_protocol.pdf">link</A><H6><U> What we did in the lab </U></H6><H6><U> Materials </U></H6>• Agarose • Qiagen kit • Nanodrop • Electrophoresis chamber • Loading buffer 6X
<H6><U> Method </U></H6>
1. Add 10 µl of loading buffer 6X to reach 50 µl for each sample.
2. Do an electrophoresis and extract the bands of the palsmid digested. We obtaines two tubes :  Tube1 : m1 = 1.276 −  0.9964 = 131.2 mg  Tube2 : m2 = 1.1524 −  0.9994 = 153.0 mg 
3. Follow the Qiagen kit steps for a final volum of 50 µl.
4. Measure the concentration with the Nanodrop to see the results.
5. Store at -20°C.<H6><U>Results</U></H6> 
Tube 1 : 20 .2 ng/µl
Tube 2 : 24.8 ng/µl </FIGCAPTION></FIGURE></DIV></DIV></BODY></HTML>