Team:Pasteur Paris/Microbiology week10
About us
Our team
Attribution
Sponsors
Project
Context
Overview
Design
Science
Devices
Research & development
Methodology
Design vs Biology
Labwork
Safety & Security
Protocol
Notebook
Results
Human Practice
Integrated Practices
Collaborations
Scenarios
Meet-up
Communication
Law
Parts
Description
Achievements
Medals
Special Prizes
Awards obtained!
click on the weeks
Microbiology Notebook
Week 10
August 8, 2016:
150. Culture for miniprep of C2 v2 and B1 v2
151. Digestion of B2, E1 and C2 from the 4th of August
152. Electrophoresis with the results of digestion
153. PCR of inserts A1/A2/D1/D2
154. Gel extraction of A1/A2/D1/D2
155. Gel extraction of B2, E1 and E2
156. Resuspension of inserts B2/E1/E2
August 9, 2016:
157. Ligation of inserts B2⁄E1⁄E2 with pET 43.1a(+)
158. Miniprep of C2 v2 (TOP 10) and B1 v2 (TOP 10 )
159. Miniprep preculture of C1 v2 in pET 43.1a(+)
160. Preparation of 10 aliquots of carbenicillin at 50 mg⁄m
161. Electrophoresis of the PCR done on the 8th of August with A1⁄A2⁄D1⁄D2
162. Transformation of A1⁄A2⁄D1⁄D2 in TOP 10
163. Transformation of E1⁄E2 and B2 in TOP 10
August 10, 2016:
164. Miniprep of C1 v2 (culture from the 9 of August)
165. Digestion of C1 v2 before electrophoresis
166. Digestion of pET 43.1 (a+) with Xba I and Hind III
167. Electrophoresis and gel extraction
168. Electrophoresis of C1 digested
169. Transformation of B2 (4, 7 and 9), E1 (1 and 2) and E2 (2, 3, 4, 5, 6 and 7)
170. Aliquot of antibodies 462
171. Culture of A1⁄A2⁄D1⁄D2
172. Ligation of A1⁄A2⁄D1⁄D2 in TOPO
173. Transformation of A1⁄A2⁄D1⁄D2 with TOPO in TOP 10 competent cells
174. Transformation of B1 v2 and C2 v2 in BL21DE3
175. Dosage of digested pET 43.1 (a+)
176. Transformation of C2 v2 and B1 v2 in pET 43.1a(+) and DH3α
August 11, 2016:
177. Absorbance of precultures C2 v2(1, 2, 3) and B1 v2 (1, 2, 16)
178. Dephosphorylation of pET 43.1a(+) digested on the 10th of August
179. Miniprep of B1 v2 ⁄E1⁄E2 in TOPO
180. Transformation of B1 v2 colony 8 and C2 v2 colony 16 in pET 43.1 (a+) and in DH 3α
181. Precultures of B1 v2 and C2 v2
182. Digestion of pET 43.1 (a+) with Xba I and Hind III
August 12, 2016:
183. Miniprep from precultures of B2 v2/E1/E2 in TOPO
184. Digestion of inserts B2 v2/E1/E2 with XbaI and HindIII
185. Culture of C2 v2 and B1 v2 in 1 l of LB
186. Agarose gel to analyze digestion of pET 43.1(a+) done on the 11th of August
Aim :
Increase the quantity of DNA before extraction.
Protocol :
follow in this link
What we did in the lab :
Materials :
• 50 ml Falcon tube
• Shaking incubator (INFORS HT)
• Swing bucket centrifuge (JOUAN GR41)
• Colonies of C2 v2 and B1 v2
• carbenicillin at 50 mg/ml
• LB medium
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
Method :
1. In a 50 ml falcon, put 48 ml of LB and 48 µl of carbenicillin.
2. For B1 v2 :
2.a Prepare 13 Eppendorfs of 1.5 ml in which add 1 ml of the previous mix.
2.b Take with a toothpick colonies on the petri dish.
2.c Place the toothpick in a tube.
2.d Let incubate overnight at 37°C and 150 rpm.
3. For C2 v2 :
3.a Prepare 20 eppendorfs of 1.5 ml with 1 ml of LB and carbenicillin mix.
3.b Take colonies and place it as previously explained.
3.c Let incubate overnight.
Aim :
As the transformations did not work (B2, E1 and E2 in pET 43.1(a+) ) with TOP 10 competent cells, we take the products of the midiprep done on the 4th of August and we digest before redoing the transformation.
Protocol:
follow in this link
What we did in the lab :
Materials :
• Restriction enzymes: Xba I, Hind III (New England Biolabs, NEB)
• Restriction enzyme buffers
• 37 °C water bath
• Shaking incubator (INFORS HT)
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
Method :
1. Realize a Mastermix and store it on ice :
Table 113
Reactants
Volumes (µl)
Xba I
30
Hind III
30
Buffer 2.1
90
Distilled H2O
90
Total
150
2. In For each insert (B2, E1, E2), prepare 10 tubes of 1.5 ml (10 eppendorfs of miniprep products).
3. In each tube, put 25 μl of DNA and 5 μl of Master mix.
4. Let digest 2 hours at 37 °C, then incubate 5 minutes at 65 °C.
Aim :
Check if the digestion has been done, it would mean that the plasmid contains the insert.
Protocol :
follow in this link
What we did in the lab :
Materials :
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this
link)
• Colonies B2, E1 and E2
• Electrophoresis chamber
• TAE 1X
• Ethidium bromide drops (EB)
Method :
1. Make a gel with 1.4 g of agarose, 200 ml of TAE 1X and 4 droplets of EB.
2. Prepare the samples with 30 µl of digested mix and 6 µl of Gene ruler (Thermofisher)ladder.
3. Follow the deposit table :
Ladder | Ø | B2 (1) | B2 (2) | B2 (3) | B2 (4) | B2 (5) | B2 (6) | B2 (7) | B2 (8) | B2 (9) | B2 (10) | Ø | E1 (1) | E1 (2) | E1 (3) | E1 (4) | E1 (5) | Ø | Ø | ladder | E1 (6) | E1 (7) | E1 (8) | E1 (9) | E1 (10) | Ø | E2 (1) | E2 (2) | E2 (3) | E2 (4) | E2 (5) | E2 (6) | E2 (7) | E2 (8) | E2 (9) | E2 (10) | Ø | Ø
4. Launch the electrophoresis for 20 minutes at 50 V, then 45 minutes at 130 V.
Results :
Figure 18 : Gel of the results of digestion
Aim :
Increase the quantity of insert.
Protocol :
follow in this link
What we did in the lab :
Materials :
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• 1.5 ml eppendorfs
• Takara enzyme
• Primers S and AS
• dNTP mix 10 mM
• Tak Ex Buffer 6X
• Distilled H2O
• MgCl2
Method :
1. Prepare the following tubes :
Table 114
Mix with two primers
Mix with primer S only
Mix with primer AS only
Takara enzyme (µl)
6
6
6
Primer S (µl)
6
6
Ø
Primer AS (µl)
6
Ø
6
MgCl2 (µl)
15
15
15
dNTPs (µl)
15
15
15
Buffer (µl)
30
30
30
H2O (µl)
216
222
222
Total (µl)
294
294
294
2. For each mix, spread 49 µl of it in the samples.
3. Add 1 µl of DNA following the number of tubes :
Table 115
Mix with two primers
Mix with primer S only
Mix with primer AS only
A1
tube 1
tube 2
tube 3
A2
tube 4
tube 5
tube 6
D1
tube 7
tube 8
tube 9
D2
tube 10
tube 11
tube 12
And tube 13 is the one without DNA
4. Launch the process of PCR.
5. Do an electrophoresis with the results of PCR
Aim :
get back the maximum quantity of DNA.
Protocol :
follow in this link
What we did in the lab:
Materials :
• Gel of A1⁄A2⁄D1⁄D2
• QIAGEN Extraction gel kit
Method :
Follow the Qiagen gel extraction kit steps with the gel :
A1 : m = 122 mg
A2 : m = 153 mg
D1 : m = 120 mg
D2 : m = 152 mg
Results :
Figure 19 : Extraction gel of A1-A2-D1-D2
Aim :
Get back purified DNA.
Protocol :
follow in this link
What we did in the lab:
Materials:
• Gel of B2/E1/E2
• QIAGEN Extraction gel kit
Method:
Follow the Qiagen Extraction gel kit steps with :
Table 116
DNA
Colonies
Weight of the gel (mg)
B2
Colony 4
367
Colony 7
432
Colony 9
269
E2
Colony 1
300
Colony 2
355
Colony 3
354
Colony 4
314
Colony 5
299
Colony 6
275
Colony 7
277
E1
Colony 1
404
Colony 2
321
Aim :
Storage of the inserts.
What we did in the lab :
Materials :
• NaAc
• Ethanol 70%;
• Inserts B2/E1/E2
Method :
1. Use 1/10 volume of NaAc and 2.5 colume of ethanol for each insert :
Table 117
B2
E1
E2
NaAc (µl)
15
10
35
Ethanol (µl)
875
250
375
2. Resuspend B2⁄E1⁄E2 in 15 µl of H2O each.
3. We estimated the weight of each inserts :
m(B2) = 240 ng
m(E1) = 60 ng
m(E2) = 420 ng
Aim :
Prepare the transformation.
Protocol :
follow in this link
What we did in the lab:
Materials :
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• Inserts B2/E1/E2
• pET 43.1(a+)
• TOPO vector
• Distilled water
• Ligase
Method :
Use the following volumes :
Table 118
E1
E2
B2
pET 43.1(a+)
E1 (µl)
15
Ø
Ø
Ø
E2 (µl)
Ø
15
Ø
Ø
B2 (µl)
Ø
Ø
15
Ø
pET 43.1(a+) (µl)
4
4
4
4
Ligase (µl)
1
1
1
1
TOP0  (µl)
2.2
2.2
2.2
2.2
H2O (µl)
Ø
Ø
Ø
15
Vtotal (µl)
22.2
22.2
22.2
22.2
Aim :
Increase the quantity of DNA.
Protocol :
follow in this link
What we did in the lab :
Materials :
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• Qiagen Miniprep kit
• Digestion enzyme Xba I and Hind III
• Digestion buffer 2.1
• 1.5 ml Eppendorfs
• Electrophoresis chamber
• Distilled water
Method :
1. Use the Qiagen kit for our cultures from the 8th of August :
13 Eppendorfs of B1 v2.
20 Eppendorfs of C2 v2.
2. Digest the plasmid with the following volumes for each sample :
Table 119
Volumes (µl)
DNA
5
Xba I
1
Hind III
1
Buffer 2.1
2
H2O
11
Total
20
3. Analyze the digestion by electrophoresis on an agarose gel prepared with 1.05 g of agarose in 125 ml of TAE 1X.
4. Launch the electrophoresis, following the deposit table :
C2 | Ladder | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | 18 | 19 | Ladder
B1 | ladder | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | Ø | Ø | Ø | 20 | ladder
Aim :
Have different clones to know which contain the insert.
Protocol :
follow in this link
What we did in the lab:
Materials :
&• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• Carbenicillin at 50 mg/ml
• Digestion enzyme Xba I and Hind III
• LB medium
• pET43.1(a+)
• C1 v2 colonies
• Shaking incubator (INFORS HT)
Method :
1. Prepare 20 ml of LB with 20 µl of carbenicillin.
2. Put 1 ml of this mix in twenty 1.5 ml Eppendorfs.
3. Take 20 colonies of the petri dish C1 v2 and put them in the previous Eppendorfs.
4. Let incubate overnight at 37°C and 150 rpm.
Aim :
Create a stock of antibiotic.
What we did in the lab :
Materials :
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• Carbenicillin at 50 mg/ml
• 1.5 ml Eppendorfs
• 15 ml Falcon
• Distilled water
Method :
1. Prep: Put 500 mg of carbenicillin in a 15 ml Falcon and 10 ml of distilled water. Then, put the Falcon on ice.
2. Aliquot the mix in 10 Eppendorfs of 1.5 ml.
3. Store at -20°C
Aim :
Check if the PCR works.
Protocol :
follow in this link
Results :
Figure 20 : Electrophoresis gel of the PCR
The PCR works properly since we noticed significant bands at the expected level.
Aim:
After their ligation in TOPO cloning vector, they will be transformed into TOP 10 cells.
Protocol:
follow in this link
What we did in the lab :
Materials
• Ligation’s products of A1/A2/D1/D2
&bull ; TOP 10 competent cells
&bull ; SOC
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• Xgal at 10 mg/ml
Method
1. Add 6 µl of ligation product in 50 µl of competent cells.
2. Put the samples 30 minutes on ice, then 40 seconds at 42°C.
3. Put the samples 3 minutes on ice.
4. Add 150 µl of SOC.
5. Let incubate 40 minutes at 37°C and 150 rpm.
6. Take LB with carbenicillin petri plate and add Xgal to reach 40 µg/ml. As the volume is about 25 ml, add 1 mg of Xgal and as the stock solution is 10 mg/ml, spread 100 µl on the plate.
7. Spread bacteria on four distincts petri dishes (one for each insert).
Aim:
After their ligation we must transform the inserts into bacteria.
Protocol:
follow in this link
What we did in the lab :
Method
Add 10 µl of ligation product (to have 100 mg) at the beginning.
Aim:
Get back the DNA.
Protocol:
follow in this link
What we did in the lab :
Method
We use a final volume of 50 µl.
Aim:
Split the insert and the plasmid.
Protocol:
follow in this link
What we did in the lab:
Materials:
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• Qiagen Miniprep kit
• Digestion enzyme Xba I and Hind III
• Digestion buffer 2 X
• 1.5 ml Eppendorfs
• Distilled water
• Shaking incubator (INFORS HT)
Method:
1. Realize a master mix with :
Table 120
Volumes (µl)
Xba I
20
Hind III
20
Buffer 2X
40
H2O
220
Total
300
2. Put µl of the master mix in each of the twenty 1.5 ml Eppendorfs and add 5 µl of DNA.
3. Let incubate one hour at 37°C and 150 rpm, then 5 minutes at 65°C
Results
The 10th of August, we realize that we forgot to put Xgal, so we re-spreaded the previous mix on petri dishes with Xgal.
Aim :
We want to produce 5 µg of dephosphorylated pET 43.1(a+) from pET 43.1(a+) at 400 ng/ml and we start with the digestion.
Protocol :
follow in this link
What we did in the lab :
Materials :
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• Qiagen Miniprep kit
• Digestion enzyme Xba I and Hind III
• CutSmart buffer
• 1.5 ml Eppendorfs
• Distilled water
• Shaking incubator (INFORS HT)
Method :
1. Put all the following reactants in a 1.5 ml Eppendorf and let digest one hour at 37°C :
Table 121
Volumes (µl)
DNA
12.5
Xba I
2
Hind III
4
Buffer CutSmart
5
H2O
26.5
Total
50
2. Inactivate the enzymes 5 minutes at 65°C.
Aim :
Get back the digested and purified plasmid before dephosphorylation.
Protocol :
follow in this link
What we did in the lab :
Materials :
• Products from the digestion of pET 43.1(a+) with Hind III and XbaI
• Agarose
• Electrophoresis chamber
Method :
1. Make a 0.7% agarose gel
2. Prepare the chamber to do the migration at 100 V with two wells for pET43.1(a+) and one for the ladder.
3. Take the results and follow the kit steps of Qiagen extraction kit.
Results :
We notice two bands for digested pET43.1(a+), one at 6000 bp and another between 1000 and 1500 bp. We extract the bands at 6000 bp.
We obtained :
m1 = 0.4079 g
m2 = 0 .3720 g
Aim :
Check if the digestion works properly and if we have inserts.
Protocol :
follow in this link
What we did in the lab:
Materials :
• Products from the digestion of C1
• Agarose
• Electrophoresis chamber
Method :
1. Take the 20 µl of each sample from the digestion and add 4 µl of loading buffer 6X.
2. Do the electrophoresis, following the deposit table :
Ladder | Ø | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | 14 | 15 | 16 | 17 | Ø | Ladder
Ladder | Ø | 17 | 18 | 19 | 20
Results
There is no insert. The colonies do not contain our insert, we must redo the experiment with new colonies.
Aim :
Transform the bacterias with our recombined plasmid.
Protocol :
follow in this link
What we did in the lab :
Method :
The volumes of insert are too small, we add 5 µl of H2O and diluted at 1/100.
We performed a transformation in DH5α with 1 µl of DNA and 50 µl of competent cells.
Aim :
Have more antibodies.
Results :
We obtained 30 µ/aliquot.
Aim :
Increase the quantity of colonies containing inserts.
Protocol :
follow in this link
Aim:
Ligate the insert and the plasmid.
Protocol:
follow in this link
Aim:
Transform our inserts in TOP 10 competent cells.
Protocol:
follow in this link
Aim :
Have bacteria with the right plasmid to produce our protein.
Protocol :
follow in this link
What we did in the lab:
Materials:
• Digested B1 and C2
• BL21DE3 competent cells
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• Shaking incubator (INFORS HT)
• SOC
Method:
1. Take 3 samples of C2 v2 (10, 13 and 16) and 2 samples of B1 v2 (6 and 8).
2. Put 1 µl of DNA in 99 µl of H2O. Then, put 1 µl of DNA (B1 v2 or C2 v2) in 50 µl of BL21DE3 competent cells.
3. Put the samples 30 minutes on ice and then 40 seconds at 42°C.
4. Put the samples 3 minutes on ice.
5. Add 150 µl of SOC and let incubate 30 minutes at 37°C and 150 rpm.
6. Spread the mix on a petri dish with LB and carbenicillin.
7. Let incubate overnight at 37°C and 150 rpm.
Aim:
Have the concentration of digested pET 43.1(a+)
Results
Measure the concentration of digested pET 43.1(a+) with a Nanodrop. For the first tube, we find a concentration of 6.8 ng/µl in 46 µl. For the second tube, we find a concentration of 8.8 ng/µl in 46 µl. Then, store the samples at -20°C.
Aim:
Transform C2 v2 and B1 v2 in pET43.1a(+) and DHα .
Protocol:
follow in this link
Results
For B2 (4, 7, 9), E1 (1, 2) and E2 (2, 3, 4, 5, 6, 7) we see small colonies so we let and 25 µl of carbenicillin.
For B1 v2 and C2 v2 in pET43.1(a+), transformations were successful.
For A1, A2 and D1, D2 there were too many bacteria growing so we took a colony, diluted it in LB and then spread it on petri dish with LB, Xgal and carbenicillin.
Aim:
To produce proteins.
What we did in the lab:
Materials:
• Spectrophotometer Ultrospec 3100
• iPTG at 0.5 M
Method:
1. Make two measurements separated of 30 minutes :
Table 122
Sample
1
2
3
4
5
6
Time of addition of iPTG
Concentration (ng/µl)
B1 (1)
0.022
0.062
0.152
0.344
0.557
0.694
16 h 55
0.859
B1 (2)
0.185
0.417
0.693
15 h 25
0.956
B1 (3)
0.075
0.211
0.413
0.688
15 h 55
0.920
C2 (1)
0.060
0.166
0.350
0.627
0.699
16 h 25
0.905
C2 (2)
0.044
0.119
0.296
0.510
0.698
16 h 25
0.910
C2 (16)
0.080
0.230
0.445
0.689
15 h 55
0.907
2. When the OD600 nm reached 0.7 keep the last measurement and centrifuge for 3 minutes at 8000 g.
3. Throw away the supernatant and store at -20°C.
4. Add iPTG to reach 0.3 mM.
Aim :
Make the future ligation easier.
Protocol :
follow in this link
What we did in the lab :
Materials :
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• rSAP
• CutSmart buffer
• 1.5 ml Eppendorfs
• Distilled water
• Shaking incubator (INFORS HT)
Method :
1. Start with tube 2 (8.6 ng/µl in 46 µl) and use the following mix :
Table 123
Volumes ( µl)
DNA
46
CutSmart
6
H2O
6.7
rSAP
1.3
Total
60
2. Let incubate 30 minutes at 37°C then 5 minutes at 65°C.
3. Do the same for tube 1.
Aim :
Get back the DNA.
Protocol :
follow in this link
What we did in the lab
Materials :
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• LB medium
• Carbenicillin at 50 mg/ml
• B1/E1/E2 in TOPO
Method :
We do 31 precultures with a mix with 1 ml of LB and 1 µl of carbenicillin to have :
3 samples of E1
14 samples of B2
14 samples of E2
Aim :
Send our insert for sequencing as the transformations in BL21DE3.
Protocol :
follow in this link
Aim:
Have precultures to redo the cultures in 4 x 1 l of LB as they grow well.
Protocol:
follow in this link
Aim:
Increase the quantity of plasmid for the next ligation.
Protocol:
follow in this link
Aim:
Increase the quantity of DNA.
Protocol:
follow in this link
Aim:
Check if the colonies we took contain the insert.
Protocol:
follow in this link
What we did in the lab
Materials
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• Digestion enzyme Xba I and Hind III
• Digestion buffer 2.1
• 1.5 ml Eppendorfs
• Distilled water
• Shaking incubator (INFORS HT)
• Inserts B2/E1/E2
Method
1. In a 1.5 ml Eppendorf, put :
Table 124
Volumes (µl)
DNA
5
Xba I
1
H2O
11
Hind III
1
Buffer 2.1
2
Total
20
2. Let incubate one hour at 37°C , then 5 minutes at -20°C.
Aim:
Produce the protein in higher quantity.
Protocol:
follow in this link
What we did in the lab
Materials
• Microbiology equipment (type of incubator, Bunsen burner, water bath, etc… Follow this link )
• iPTG
• Carbenicillin at 50 mg/ml
Method
1. In a 2 l Erlenmeyer, put 1 l of LB and 1 ml of carbenicillin and incubate at 37°C and 150 rpm to warm the liquid.
2. For each preculture of 25 ml, put it in a 50 ml Falcon, centrifuge 5 minutes at 3500 rpm, throw away the supernatent and resuspend the pellet in 5 ml of clean LB.
3.Centrifuge 5 minutes at 3500 rpm, throw away the supernatent and resuspend the pellet in 20 ml of clean LB.
4. In each 2 l erlenmeyer, add 5 ml of our preculture : 4 erlenmeyer for C2 and 4 for B1.
5. Let incubate at 37 °C ann 150 rpm and measure the DO.
6. Once the DO reaches 0.7, add 1 ml of iPTG.
7. Let incubate for 3 hours.
8. Centrifuge the cultures.
9. Resuspend the pellet in 10 ml of lysis buffer (See protein purification protocol) in a 50 ml Falcon of known weight.
10. Store at -20°C.
Aim:
Check if the digestion works.
Protocol:
follow in this link
What we did in the lab
Materials
• Agarose
• Qiagen kit
• Nanodrop
• Electrophoresis chamber
• Loading buffer 6X
Method
1. Add 10 µl of loading buffer 6X to reach 50 µl for each sample.
2. Do an electrophoresis and extract the bands of the palsmid digested. We obtaines two tubes :
Tube1 : m1 = 1.276 − 0.9964 = 131.2 mg
Tube2 : m2 = 1.1524 − 0.9994 = 153.0 mg
3. Follow the Qiagen kit steps for a final volum of 50 µl.
4. Measure the concentration with the Nanodrop to see the results.
5. Store at -20°C.
Results
Tube 1 : 20 .2 ng/µl
Tube 2 : 24.8 ng/µl
