<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CSMU_NCHU_Taiwan_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CSMU NCHU Taiwan/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="home"><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan"><SPAN>HOME</SPAN></A></DIV><UL class="menu"><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Description">PROJECT</A></LI><LI class="dropdown-title">PROJECT</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Safety">Safety</A></LI><LI><A href=" https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Results">Results &amp; Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/InterLab">Inter Lab</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Parts">Parts</A></LI></UL></DIV></DIV><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Model">MODELING</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Software">SOFTWARE</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Notebook">NOTEBOOK</A></LI><LI class="dropdown-title">NOTEBOOK</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Notebook">Lab Book</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Experiments">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Notebook/Medal_Criteria">Medal Criteria</A></LI></DIV></DIV><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/HP/Silver">HUMAN PRACTICES</A></LI><LI class="dropdown-title">HUMAN PRACTICES</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/HP/Silver">HP-Silver</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/HP/Gold_Integrated">Gold-Integrated</A></LI><LI><A href=" https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Entrepreneurship">Entrepreneurship</A></LI></DIV><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Team">TEAM</A></LI><LI class="dropdown-title">TEAM</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Attributions">Attributions</A></LI><DIV class="hamburger"><DIV id="hamburger" class="dropdown row"><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title">PROJECT</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Safety">Safety</A></LI><LI><A href=" https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Results">Results &amp; Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Parts">Parts</A></LI></UL></DIV><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title">MODELING</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Model">Modeling</A></LI></UL></DIV><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title">SOFTWARE</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Software">Software</A></LI></UL></DIV><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title">NOTEBOOK</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Notebook">Lab Book</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Experiments">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Notebook/Medal_Criteria">Medal Criteria</A></LI></UL></DIV><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title" style="font-size: 20px;">HUMAN PRACTICES</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/HP/Silver">HP-Silver</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/HP/Gold_Integrated">Gold-Integrated</A></LI><LI><A href=" https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Entrepreneurship">Entrepreneurship</A></LI></UL></DIV><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title">TEAM</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Attributions">Attributions</A></LI></UL></DIV></DIV></DIV><TITLE>Safety</TITLE><DIV class="bg-container"><DIV class="my-main-container"><DIV class="main-content"><DIV class="text-area"><H1>Design</H1><H2>Thioredoxin fusion system</H2><P style="text-indent:2em">In our project, we used <I>E. coli</I> to express two heterogeneous enzymes from mycobacterium smegmatis. However, we didn’t know whether they were toxic to <I>E. coli</I> and whether they would become inclusion bodies because of insolubility when <I>E. coli</I> produced them. Therefore, we found the thioredoxin fusion proteins system. When the gene of thioredoxin in <I>E. coli</I> (TrxA) were co-expressed, the target proteins were inserted into the active-site loop of thioredoxin, and therefore the fusion proteins can be more soluble<SUB>[1]</SUB>.</P><P style="text-indent:2em">In addition, between the thioredoxin domain and the target gene domain, we designed some linkers to ligase two domains. The first part, which could be translated to the peptide sequence, “DDDDK”, was designed as the cleavage site of enterokinase. There were some restriction sites in the second part. The third one, which could be translated to glycine-glycine, was a flexible linker<SUB>[2]</SUB>.</P><P class="fig">Fig. 1: BBa_K2382004 showed the gene design of the Thioredoxin fusion system construction.</P><H2>Aflatoxin-degrading enzyme: F420-dependent reductase group A</H2><P style="text-indent:2em">F420-dependent reductases (FDR) can be divided into two classes (A and B) and be found in some species of bacteria<SUB>[3]</SUB>. FDR-A enzymes, has up to 100 times more activity than the other. In this class, MSMEG5998 has the best specific activity to AFB1 (10350 nmol/min/mol enzyme) and AFG1 (103210 nmol/min/mol enzyme). Therefore, we looked for the coding sequence of MSMEG5998, which was registered in NCBI in mycobacterium smegmatis and put the sequence of thioredoxin before it to form a fusion protein. For the purpose of purification through nickel-resin column, we added a 18-bp sequence which can code 6 histidines. In addition, we chose T7 promoter which contains lac operator to express this protein because it can be induced by IPTG. For the terminator, we chose BBa_B0015 because it was commonly used in <I>E. coli</I>. The gene design are shown in <FONT color="#005882">Fig. 2.</FONT></P><P class="fig">Fig. 2: BBa_K2382006 showed the gene design of the Thioredoxin-MSMEG5998 fusion protein construction.</P><H2>The activator of F420: F420-dependent glucose-6-phosphate dehydrogenase</H2><P style="text-indent:2em">F420-dependent glucose-6-phosphate dehydrogenase (FGD) can catalyze D-glucose-6-phosphate (G6P) to become D-6-phsphogluconolactone. The chemical reaction are shown in <FONT color="#005882">Fig. 3</FONT><SUB>[4]</SUB>. This enzyme can be found in many organisms. In order to coordinate with MSMEG5998 and make the two enzymes react more naturally, we chose fgd gene also in mycobacterium smegmatis (strain MC(2) 155).To increase the solubility when <I>E. coli</I> produce this enzyme, we used the same design as Thioredoxin-MSMEG5998 fusion protein to form Thioredoxin-FGD fusion protein and the design is demonstrated in <FONT color="#005882">Fig. 4</FONT>.</P><P class="fig">Fig. 3: The chemical reaction of G6P and oxidized F420 were catalyzed by FGD.</P><P class="fig">Fig. 4: BBa_K2382005 showed the gene design of the Thioredoxin-FGD fusion protein construction.</P><H2>The function of our proteins in aflatoxin-induced DNA repair pathway</H2><P style="text-indent:2em">When mammalian cells respond to DNA damage from environmental toxins or radiations, the two key signaling components, ATM and ATR were activated and therefore phosphorylated two protein kinases, Chk1 and Chk2<SUB>[5]</SUB>. All of them will reduce cyclin-dependent kinase (CDK) activity through activation of p53. Finally, inhibition of CDKs, such as p21 slows down or arrests cell-cycle progression. Therefore, we can know whether aflatoxin B1 induces DNA damage in HepG2 cells indirectly through the increasing expression of some markers in p53 pathway<SUB>[6]</SUB>. In our hypothesis, the modified aflatoxin-degrading enzyme, MSMEG_5998 can directly alleviate the genotoxicity of aflatoxin and indirectly inhibit the activation of p53 pathway by degrading the toxin when co-treatment in HepG2 cells. The model of our hypothesis is demonstrated in <FONT color="#005882">Fig. 5</FONT>.</P><P class="fig">Fig. 5: Our hypothesis were demonstrated in this figure.</P><P style="text-indent:2em">Numerous methods for determining the level of Aflatoxin B1 in food and products have been established. As is well known, high performance liquid chromatography (HPLC) using a fluorescent detector is the most widely adopted means of monitoring Aflatoxin B1. However, HPLC has many disadvantages, including its cost, the complexity of operation of the machines involved, and the extensive preparation of samples. Since a simple and efficient technique for the routine monitoring of food such as grains, peanuts and related products is in an urgent demand, we chose immunostrip as the way to detect aflatoxin.</P><P style="text-indent:2em">Traditionally, this type of test strip required a special antibody that binds to aflatoxin, called “monoclonal antibody”. However, it would take a lot of time to produce it in the processes of immunization, fusion and cloning, production of ascites, and characterization of the monoclonal antibody. What’s more, not only is the cost to keep an animal hotel high, but also hybridoma is not stable enough to maintain the quality and quantity of monoclonal antibody. Not to mention the vulnerable Van der Waals force with nanoparticle probe, if the samples are not properly treated before tested, the immunostrip will not detect anything. Therefore, to change the situations of these disadvantages, we made lots of improvement on it.</P><H2>scFv- RFP fusion system</H2><P style="text-indent:2em">In our project, we designed a fusion protein to replace the traditional monoclonal antibody applied on the immunostrip. The fusion protein that we designed is composed of three domains,scFv, rigid linker, RFP  and His-Tag <FONT color="#005882">(Fig. 1)</FONT>.</P><P class="fig">Fig. 1: Using RAPTOR X to simulate the structure and folding of fusion protein. The left side is an anti-aflatoxin scFv, and the right side is an RFP with 6X His-Tag, linked by a rigid protein linker. The simulation result demonstrates that the rigid linker can maintain the distance between two domains, and keeps them from interrupting each other or misfolding.</P><H2>scFv</H2><P style="text-indent:2em">First, we designed an scFv, Single Chain Variable Fragment, to replace the traditional full-length antibody. Compared with full-length antibody,  scFv is not actually a fragment of an antibody, but instead is a fusion protein of the variable regions of the heavy (VH) and light chains (VL) of immunoglobulins, connected with a short linker peptide of 10 to about 25 amino acids. For instance, GS linker, the most commonly used protein linker in scFv design, to maintain the flexibility and protein folding. Consequently, it prevents the obstacle for <I>E. coli</I> to produce scFv in this host, since glycosylations are required for the effector functions and are mainly located in the Fc fragment. Nevertheless, this part of the antibody is not present in scFv molecules. The scFv used here exhibited the highest sensitivity against all four major kinds of aflatoxins in a previous research<SUB>[7]</SUB>.</P><H2>EAAAK rigid linker</H2><P style="text-indent:2em">Second, at the C terminal of the scFv, we use an EAAAK rigid linker to link to maintain the distance between scFv and Red Fluorescent Protein(RFP), preventing unnecessary cross-interaction. This EAAAK rigid linker repeats amino acid “EAAAK” for three times, and it forms  α- helix in its structure<SUB>[8]</SUB>, exhibiting a rigid and stable property.</P><H2>RFP with 6X His-Tag</H2><P style="text-indent:2em">Third, right after the rigid linker is a RFP with 6X His-Tag at the end. Since the antibody itself does not exhibit colors, traditional immunostrip shows the result with the aid of gold nanoparticles<FONT color="#005882">(Fig. 2)</FONT>.However, the result could be interfered because of many reasons. We add an RFP and a 6X His-Tag in this fusion protein to make it exhibit red color and  improve purification respectively. With this newly designed fusion protein, there’s no need for the immunostrip to use gold nanoparticles, leading to a new generation of aflatoxin detection.</P><P style="text-indent:2em">We used a constitutive promoter(BBa_J23101), a common ribosome binding site(BBa_B0034) and a double terminator (BBa_B0015) constructed  in pSB1C3, to express this fusion protein in our project. And this makes of our composite part BBa_K2382010.<FONT color="#005882">(Fig. 3)</FONT></P><P style="text-indent:2em">However, only strip is not enough because samples with Aflatoxin need to be extracted first. In the lab, we have to crush sample to increase reaction surface area, and using water or organic liquid to extract Aflatoxin B1. So, we also design the kit for helping people to finish this work.</P><P class="fig">Fig. 2: The traditional immunostrip use gold nanoparticle to exhibit red color on the antibodies. The red lines on the strip indicate the position of Control line and Test  line with the aid of gold nanoparticles.</P><P class="fig">Fig. 3: The scFv compose three domains, scFv, rigid linker, RFP  and His-Tag.  This figure describes the design of BBa_K2382010, which can succefully express this fusion protein in pSB1C3.</P><H2>A small device that accelerates aflatoxin detection</H2><H2>Overview</H2><P style="text-indent:2em">Detection of aflatoxin is an important part of our AFLATOXOUT project. To make the processes of detecting aflatoxin faster, easier and cheaper, we designed a small kit to help us detect aflatoxin. By using the color changes of the test strip installed on the kit, we would know whether the food was contaminated by aflatoxin or not.</P><P style="text-indent:2em">We utilized 3D printing technology to develop our kit. The kit is printed with a Polylactic Acid (PLA), an eco-friendly plastic, and it could be degraded naturally.The graph below shows the appearance of the kit.</P><H2>Component introduction</H2><P>The kit is a composite of 4 components. The following description shows the functions of each part.</P><ALIGN:><P>1.LidThe lid works as a part applying force to drive the grinder. User can rotate the lid with the handle and make the grinder work. It also prevents food debris from spilling.2.GrindeGrinder is the place where we put the food that is going to be tested. It breaks the food into smaller particles or powder in order to improve solubility.3.SolventThe special solvent could extract aflatoxin from the food powder. It is sealed with plastic membrane. Once combined with the mixing tank, the membrane would be punctured and the solvent would flow into the tank.4.Mixing tankThe mixing tank is the place where the food powder mixed with the solvent. There’s a window on the side of the mixing tank where we can install the test strip.</P></ALIGN:></DIV><DIV class="pdf-area"><SPAN class="pdfbtn" id="design-btn-1">Reference</SPAN><DIV class="pdf-container" id="design-1"><UL><LI>1. LaVallie, E.R. and J.M. McCoy, Gene fusion expression systems in <I>Escherichia coli</I>. Current Opinion in Biotechnology, 1995. 6(5): p. 501-506.</LI><LI>2. Chen, X., J. Zaro, and W.-C. Shen, Fusion Protein Linkers: Property, Design and Functionality. Advanced drug delivery reviews, 2013. 65(10): p. 1357-1369.</LI><LI>3. Lapalikar, G.V., et al., F420H2-dependent degradation of aflatoxin and other furanocoumarins is widespread throughout the Actinomycetales. PLoS One, 2012. 7(2): p. e30114.</LI><LI>4. Oyugi, M.A., et al., Mechanistic insights into F420-dependent glucose-6-phosphate dehydrogenase using isotope effects and substrate inhibition studies. Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics, 2017.</LI><LI>5. Jackson, S.P. and J. Bartek, The DNA-damage response in human biology and disease. Nature, 2009. 461(7267): p. 1071-1078.</LI><LI>6. Boehme, K., et al., Activation of P53 in HepG2 cells as surrogate to detect mutagens and promutagens in vitro. Toxicology Letters, 2010. 198(2): p. 272-281.</LI><LI>7. Li, X., et al., Molecular characterization of monoclonal antibodies against aflatoxins: a possible explanation for the highest sensitivity. Anal Chem, 2012. 84(12): p. 5229-35.</LI><LI>8. Chen, X., Zaro, J. L., &amp; Shen, W. (2013). Fusion protein linkers: Property, design and functionality. Advanced Drug Delivery Reviews, 65(10), 1357-1369. doi:10.1016/j.addr.2012.09.039</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="path-btns" style="left:0;"><DIV class="path"><DIV class="pathWord path-word-sm"><P>Antidote</P></DIV></DIV><DIV class="path"><DIV class="pathWord path-word-sm"><P>Test Strip</P></DIV></DIV><DIV class="path"><DIV class="pathWord path-word-sm"><P>Kit</P></DIV></DIV></DIV></BODY></HTML>