<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CSMU_NCHU_Taiwan_Software_Tools skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CSMU NCHU Taiwan/Software/Tools</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="home"><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan"><SPAN>HOME</SPAN></A></DIV><UL class="menu"><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Description">PROJECT</A></LI><LI class="dropdown-title">PROJECT</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Safety">Safety</A></LI><LI><A href=" https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Results">Results &amp; Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/InterLab">Inter Lab</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Parts">Parts</A></LI></UL></DIV></DIV><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Model">MODELING</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Software">SOFTWARE</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Notebook">NOTEBOOK</A></LI><LI class="dropdown-title">NOTEBOOK</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Notebook">Lab Book</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Experiments">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Notebook/Medal_Criteria">Medal Criteria</A></LI></DIV></DIV><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/HP/Silver">HUMAN PRACTICES</A></LI><LI class="dropdown-title">HUMAN PRACTICES</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/HP/Silver">HP-Silver</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/HP/Gold_Integrated">Gold-Integrated</A></LI><LI><A href=" https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Entrepreneurship">Entrepreneurship</A></LI></DIV><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Team">TEAM</A></LI><LI class="dropdown-title">TEAM</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Attributions">Attributions</A></LI><DIV class="hamburger"><DIV id="hamburger" class="dropdown row"><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title">PROJECT</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Safety">Safety</A></LI><LI><A href=" https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Results">Results &amp; Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Parts">Parts</A></LI></UL></DIV><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title">MODELING</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Model">Modeling</A></LI></UL></DIV><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title">SOFTWARE</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Software">Software</A></LI></UL></DIV><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title">NOTEBOOK</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Notebook">Lab Book</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Experiments">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Notebook/Medal_Criteria">Medal Criteria</A></LI></UL></DIV><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title" style="font-size: 20px;">HUMAN PRACTICES</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/HP/Silver">HP-Silver</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/HP/Gold_Integrated">Gold-Integrated</A></LI><LI><A href=" https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Entrepreneurship">Entrepreneurship</A></LI></UL></DIV><DIV class="submenu col-sm-2"><UL><LI class="dropdown-title">TEAM</LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Attributions">Attributions</A></LI></UL></DIV></DIV></DIV><TITLE>software</TITLE><DIV class="bg-container" style="max-height:none;"><DIV class="my-main-container"><DIV class="main-content"><DIV class="m-text-area"><H1>Software tools</H1><DIV id="app-NCBI" class="m-block"><H2 class="m-subtitle">NCBI</H2><P style="text-align: center;">To search for protein’s sequences</P><P>This website provides information about target proteins’ gene sequences and amino
acid sequences. We are able to find the protein amino acid sequence we were
looking for using this website.For example: ThioredoxinWe understood through this website that the protein’s amino acid sequence length is 109 amino acids, the sequence is shown in the following picture:</P></DIV><DIV id="app-RaptorX" class="m-block"><H2 class="m-subtitle">RaptorX</H2><P style="text-align: center;">A web Portal for Protein Structure and Function Prediction</P><P>It is divided into four parts</P><UL><LI>A.  RaptorX Structure Prediction: template-based tertiary structure prediction </LI><LI>B.  RaptorX Property Prediction: protein structure property prediction without using templates </LI><LI>C.  RaptorX Contact Prediction </LI><LI>D.  RaptorX Binding Prediction </LI></UL><P>We utilized the part on structure prediction, helping us to express the secondary and tertiary structures.
For example: The fusion protein we designed</P><P>A.  The first part: Displays amino acid sequence</P><P>B.  The second part: Visualize the protein</P><P>C. The third part tells us information about secondary structure, where the positions of alpha helix or Beta folds are.</P></DIV><DIV id="app-Uniprot" class="m-block"><H2 class="m-subtitle">Uniprot</H2><P style="text-align: center;">Free use of the protein sequence and functions database</P><P>Including a large amount of information derived from research papers about protein’s biological function messages. We used this tool to find the active site of our protein.</P><P>For example: thioredoxin
Found this protein’s active site through this website</P></DIV><DIV id="app-Pymol" class="m-block"><H2 class="m-subtitle">Pymol</H2><P style="text-align: center;">Can visualize protein sequences.</P><P>Other than the protein’s 3D structure, the amino acid positions can be shown.For example:fusion proteinWe used this software to show the 3D structure, labelling with different colors and showing interfaces, to increase the quality of the picture.</P></DIV><DIV id="app-Swiss" class="m-block"><H2 class="m-subtitle">Swiss Model</H2><P style="text-align: center;">Protein model database to compare with the given protein</P><P> To find the most similar template or the template with highest sequence coverage. By combining different templates, we predict the desired structure of our protein.For example:fusion proteinWe used this software to show the 3D structure, labelling with different colors and showing interfaces, to increase the quality of the picture.</P></DIV><DIV id="app-PMV" class="m-block"><H2 class="m-subtitle">PMV-1.5.7rc1</H2><P style="text-align: center;">Add hydrogen bonds to the protein</P><P>Since RaptorX does not have the ability to mark hydrogen bonds of the protein structure, this tool can add hydrogen bonds to the protein derived from Raptor X. It can also allow electrons to be attached to the protein, simulating real proteins.A.Without the hydrogen bonds</P><P>B.With the hydrogen bonds</P></DIV><DIV id="app-PubChem" class="m-block"><H2 class="m-subtitle">PubChem</H2><P style="text-align: center;">Allows downloading of SDF files</P><P>Find out the substrate that we are going to combine with the active site, the website allows downloading of SDF files, which can use PyRx to simulate the 3D structure of the substrate connected with the active site.For example: aflatoxin B1</P></DIV><DIV id="app-PyRx" class="m-block"><H2 class="m-subtitle">PyRx</H2><P style="text-align: center;">Do docking to combine molecule and ligand</P><P>PyRx is a simulating tool built on AutoDock or AutoDock Vina, which can allow us to join our substrates with proteins, simulating the position where they join. PyRx is therefore a strong tool for designing drugs. It also involves the functions and simple clicking interfaces for drug designs, at the same time being a strong visualisation tool.For example:Fusion protein docking with aflatoxin B1</P><P>A.Before DOCKING</P><P>B.After DOCKING</P></DIV></DIV></DIV><DIV class="path-tags"><UL><A href="https://2017.igem.org/Team:CSMU_NCHU_Taiwan/Software"><P style="font-size:22px;font-family: 'Ubuntu'">APP</P></A><P class="tag">Software Tools</P><LI id="NCBI-btn" class="tag-btn">- NCBI</LI><LI id="RaptorX-btn" class="tag-btn">- RaptorX</LI><LI id="Uniprot-btn" class="tag-btn">- Uniprot</LI><LI id="Pymol-btn" class="tag-btn">- Pymol</LI><LI id="Swiss-btn" class="tag-btn">- Swiss</LI><LI id="PMV-btn" class="tag-btn">- PMV</LI><LI id="PubChem-btn" class="tag-btn">- PubChem</LI><LI id="PyRx-btn" class="tag-btn">- PyRx</LI></UL></DIV></DIV></DIV></BODY></HTML>