Team:CSMU NCHU Taiwan/Software/Tools
HOME
PROJECT
PROJECT
Description
Design
Safety
Results & Demonstrate
Inter Lab
Parts
MODELING
SOFTWARE
NOTEBOOK
NOTEBOOK
Lab Book
Protocols
Medal Criteria
HUMAN PRACTICES
HUMAN PRACTICES
HP-Silver
Gold-Integrated
Public Engagement
Collaborations
Entrepreneurship
TEAM
TEAM
Team
Attributions
PROJECT
Description
Design
Safety
Results & Demonstrate
InterLab
Parts
MODELING
Modeling
SOFTWARE
Software
NOTEBOOK
Lab Book
Protocols
Medal Criteria
HUMAN PRACTICES
HP-Silver
Gold-Integrated
Public Engagement
Collaborations
Entrepreneurship
TEAM
Team
Attributions
software
Software tools
NCBI
To search for protein’s sequences
This website provides information about target proteins’ gene sequences and amino acid sequences. We are able to find the protein amino acid sequence we were looking for using this website.
For example: Thioredoxin
We understood through this website that the protein’s amino acid sequence length is 109 amino acids, the sequence is shown in the following picture:
RaptorX
A web Portal for Protein Structure and Function Prediction
It is divided into four parts
A.  RaptorX Structure Prediction: template-based tertiary structure prediction
B.  RaptorX Property Prediction: protein structure property prediction without using templates
C.  RaptorX Contact Prediction
D.  RaptorX Binding Prediction
We utilized the part on structure prediction, helping us to express the secondary and tertiary structures.
For example: The fusion protein we designed
A.  The first part: Displays amino acid sequence
B.  The second part: Visualize the protein
C. The third part tells us information about secondary structure, where the positions of alpha helix or Beta folds are.
Uniprot
Free use of the protein sequence and functions database
Including a large amount of information derived from research papers about protein’s biological function messages. We used this tool to find the active site of our protein.
For example: thioredoxin
Found this protein’s active site through this website
Pymol
Can visualize protein sequences.
Other than the protein’s 3D structure, the amino acid positions can be shown.
For example:fusion protein
We used this software to show the 3D structure, labelling with different colors and showing interfaces, to increase the quality of the picture.
Swiss Model
Protein model database to compare with the given protein
To find the most similar template or the template with highest sequence coverage. By combining different templates, we predict the desired structure of our protein.
For example:fusion protein
We used this software to show the 3D structure, labelling with different colors and showing interfaces, to increase the quality of the picture.
PMV-1.5.7rc1
Add hydrogen bonds to the protein
Since RaptorX does not have the ability to mark hydrogen bonds of the protein structure, this tool can add hydrogen bonds to the protein derived from Raptor X. It can also allow electrons to be attached to the protein, simulating real proteins.
A.Without the hydrogen bonds
B.With the hydrogen bonds
PubChem
Allows downloading of SDF files
Find out the substrate that we are going to combine with the active site, the website allows downloading of SDF files, which can use PyRx to simulate the 3D structure of the substrate connected with the active site.
For example: aflatoxin B1
PyRx
Do docking to combine molecule and ligand
PyRx is a simulating tool built on AutoDock or AutoDock Vina, which can allow us to join our substrates with proteins, simulating the position where they join. PyRx is therefore a strong tool for designing drugs. It also involves the functions and simple clicking interfaces for drug designs, at the same time being a strong visualisation tool.
For example:Fusion protein docking with aflatoxin B1
A.Before DOCKING
B.After DOCKING
APP
Software Tools
- NCBI
- RaptorX
- Uniprot
- Pymol
- Swiss
- PMV
- PubChem
- PyRx
