<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Edinburgh_UG_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Edinburgh UG/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper"><TITLE>Demonstrate</TITLE><DIV id="custom-bootstrap-menu" class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><NAV class="navbar navbar-inverse navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:Edinburgh_UG">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Team</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Attribution">Attribution</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Medal_Criteria">Medal Criteria</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Ethics">Ethics</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Mary_Queen_of _Scots">Mary Queen of Scots</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Project</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Limitations">Limitations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Informatics</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Lexicon_Encoding">Lexicon Encoding</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Error_Correction">Error Correction</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Encryption">Encryption</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Files">Files</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Part_Collection">Part Collection</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Collaboration">Collaboration</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Awards</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Software">Software</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Safety/Biological Safety">Biological Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Edinburgh_UG/Plate_Reader">Plate Reader</A></LI></DIV></DIV></NAV></DIV></DIV><HEADER class="header-image"><DIV class="headline"><DIV class="container"><H1>Demonstrate</H1></DIV></DIV></HEADER><DIV class="container"><DIV class="featurette" id="about"><P class="lead"> Every iGEM project contributes towards interest and advancement in synthetic biology. However the transition from an interesting lab project to a commercial product is a big step. We believe that DNA will become a long-term storage medium, and so it is important to look to the future of this being implemented, and show how our system can be used in the real world. </P></DIV><DIV class="featurette" id="services"><H2 class="featurette-heading">Use of Biological Materials
            </H2><P class="lead">Any team considering the release of a biological product must be fully aware of rules and regulations governing their country. In Scotland, EU regulation is implemented, along with further additions by both the UK and Scottish Government.  </P><P class="lead">Our BabbleBlock generation process can be divided into three parts; the generation of BabbleBricks housed in acceptor plasmids, assembly of BabbleBlocks, and long-term storage of BabbleBlocks. The generation of BabbleBricks involves transformation of competent TOP10 E.coli cells. The assembly and storage of BabbleBricks and Blocks however is cell free.  </P><P class="lead">The distinction between these two processes is important. The use of unnatural DNA sequences outside the lab is heavily governed. Most genetically modified organism laws however cover GMOs (genetically modified organisms) or (Genetically modified micro-organisms). </P><P class="lead">“Micro-organism: This term covers bacteria, fungi and viruses, as well as cell and tissue cultures from plants, animals or humans. Naked nucleic acid, oligonucleotides, synthetic DNA, plasmids or liposomes are not considered to be micro-organisms.” - The Genetically Modified Organisms (Contained Use) Regulations 2014.     Scottish Government </P><P class="lead">In Scotland, the definition of a microorganism does not cover naked DNA. All relevant governing GM legislation in Scotland refers to GMOs and GMMs, and so exclusion of naked DNA from this allows storage of assembled BabbleBlocks outside the lab. </P><P class="lead"> Our functional proof of concept follows the steps our commercialised system would have:</P><P class="lead">1. Creation of BabbleBricks  </P><P class="lead">2. Assembly of BabbleBlocks </P><P class="lead">3. The user will store BabbleBricks and BabbleBlocks for use whenever needed. </P><P class="lead">4. The user will read back the data stored when needed.  </P><P class="lead">Barcodes on side of tubes (Figure Left)</P><P class="lead">QR codes on underside of tubes; Can be read as a plate or individually (Figure Right)</P><P class="lead">An example of this is part no. BBa_K1967000 </P></DIV><DIV class="featurette" id="contact"><H2 class="featurette-heading">Make BabbleBricks
            </H2><P class="lead">As demonstrated in Proof-of-concept (link page) we have made phytobrick compatible BabbleBricks. These plasmids are the DNA component of the BabbleBrick Library. For proof-of-concept we have made 10 BabbleBricks, however our 5bp coding regions allows for 1024 different BabbleBricks to exist in the library. This can be scaled up; for example, a coding region of 6 nucleotides would give 4096 different BabbleBricks. The BabbleBrick Library (and complete BabbleBlocks) would be stored in barcoded tubes for integration with an automated system for assembly, and catalogue system for storing the completed constructs. </P></DIV><DIV class="featurette" id="contact"><H2 class="featurette-heading">BabbleBlock Assembly
           </H2><P class="lead">BabbleBricks have been manually assembled into BabbleBlocks in the lab, as also seen in <A href="https://2016.igem.org/Team:Edinburgh_UG/Proof">proof-of-concept</A>. </P><P class="lead">We can see a constructed BabbleBlock on this gel: </P><P class="lead">However, as a commercialised product we would intend to use an automated lab so that many BabbleBlocks can be generated in parallel.The automated system would use a liquid handler to dispense all liquids. The assembly would occur in plates, rather than 1.5ml tubes. A magnet under the plate would take the place of the magnetic rack. An example set-up is shown here, modelled at The Edinburgh Genome Foundry.  </P></DIV><DIV class="featurette" id="contact"><H2 class="featurette-heading">Long-term BabbleBlock Storage
          </H2><P class="lead">We aim to produce a solution to store long-term archival data. In the space of a summer however it is not feasible to test this. Instead, we have decided to look at different methods of storing DNA. The three storage methods are DNA in elution buffer, Dried DNA, and DNA dried on filter paper.</P><P class="lead">The Edinburgh University Library are potential end users of our system. We discussed with them the type of environmental conditions long-term data is currently stored. We were informed that the library, like most other UK archives conform to British Standards Institution 5454 regulations. This standard is a set of both environmental guidelines such as temperature and humidity, as well as practical issues such as storage containers and fire protecting schemes.
This standard was something we were unable to replicate in the lab. However the library offered to store samples for us under monitored conditions. Upon risk assessment by the School of Biological Sciences safety officer this was approved.
Part BBa_K1967000 was used.  </P><P class="lead"><STRONG>Samples:</STRONG></P><P class="lead"> 5μl sample was pipetted onto blotting paper, dried and cut out. This was placed in tube 1.</P><P class="lead">5μl was put into tube 2, and left as a liquid sample.  </P><P class="lead">5μl was put in tube 3 which was left to evaporate to give a dry sample.  </P><P class="lead">All three samples were placed in a correctly labelled box, and shelved in the archives room for 2 months. The Environmental report below details these conditions. </P><P class="lead">After 2 months, the samples were retrieved, and dry samples were re-suspended. They were sent for sequencing. Our test of demonstrating use was to determine whether the sample was could be sequenced, and the encoded data could be read back using our BabblED software. </P><P class="lead">Both the dried sample, and liquid sample were read back correctly. The liquid sample has the fewest base errors, however we suspect this was due to sequencing issues. The sample dried on paper did not produce a readable sample when dissolved and sequenced, and so needed to be transformed into E.coli TOP10 cells to obtain enough DNA for sequencing.  </P><P class="lead">From these results, for cell free user experience we would not recommend using filter paper. We would recommend simply storing the DNA in tubes, or plates liquid or dry. To improve this recommendation this further we would like to test more storage conditions at varying temperatures.  </P></DIV><DIV class="featurette" id="contact"><H2 class="featurette-heading">Data Read-back
           </H2><P class="lead">Stored archival data is no use unless it can be read back when needed. Despite our best efforts to keep DNA stable, there is always the possibility of sequencing errors. While this will reduce with time it is still important to have mechanisms in place to combat this. Our read back software (link) has error identifying and correcting mechanisms to help minimise the event of unreadable data. This consists of checksums and optimal rectangular codes, and can be further explained here (link). To tests these mechanisms simulations were run. These input a variety of incorrect sequences, and allowed the software to correct them. The ability to recognise and correct errors is as follows. </P></DIV><DIV class="featurette" id="contact"><H2 class="featurette-heading">Environmental report for DNA storage
         </H2><P class="lead">This report was prepared by Katherine Richardson at Edinburgh University Library. </P><P class="lead"><STRONG>Environmental monitoring equipment</STRONG></P><P class="lead">The DNA samples were kept in a secure store at the Centre for Research Collections for two months. The climate in secure stores are controlled by close control units (CCUs).	 </P><P class="lead">Storage areas are covered by two environmental monitoring systems: </P><P class="lead">1. The first is the Estates and Buildings Department ‘Building Management System’ (BMS) – this system monitors conditions throughout the building and is linked to an alarm, which sounds if the conditions in storage areas fall outside the agreed parameters (18°C +/- 2°C and 50% RH +/- 5%).  </P><P class="lead">2. The second is the Hanwell environmental monitoring system – this system takes and logs readings for relative humidity and temperature. Data is collected via free-standing sensors located within collection stores and access areas and relayed to a base unit and remote PC.  </P><P class="lead"><STRONG>Environmental data</STRONG></P><P class="lead">The following data was recorded by the Hanwell environmental monitoring system from 05/08/2016 to 06/10/2016 </P><P class="lead">Temperature:  </P><P class="lead">* Maximum temperature: 18.2  </P><P class="lead">* Minimum temperature: 16.5</P><P class="lead">* Spread: 1.7  </P><P class="lead">* Average: 17.2  </P><P class="lead">* Temperature was below 18°C for 99.6% of the time  </P><P class="lead"> Relative Humidity </P><P class="lead">* Maximum RH%: 56.8 </P><P class="lead">* Minimum RH%: 51.8 </P><P class="lead">* Spread: 5 </P><P class="lead">* Average: 53 </P><P class="lead">* The RH% was &gt;45 and &lt;55 for 87% of the time</P><DIV class="featurette" id="contact"><H2 class="featurette-heading">References
           </H2><P class="lead">http://www.legislation.gov.uk/uksi/2014/1663/contents/made</P><P class="lead">http://www.legislation.gov.uk/uksi/2014/1663/contents/made </P><P class="lead">https://www.thenbs.com/PublicationIndex/Documents/Details?DocId=300479</P></DIV></DIV><DIV class="row"><DIV class="box"><DIV class="col-lg-12"><H2 class="intro-text text-center">Follow
                          <STRONG>Us</STRONG></H2></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>