"
Page
Discussion
View source
History
teams
Log in
 
Team:USTC-Software/Software/Overview
From 2013.igem.org
Slide
Take a gNAP before wearing your gloves! Genetic Network Analyze and Predict
The sketch and final GUI of gNAP!
We compare the result of our software with gene expression profile in literature.
We are USTC-Software!
Overview
Overview
[SOFTWARE]
BASIC FUNCTION
Our application aims to simulate genetic networks. The application analyzes the stability of genetic networks after introduction of exogenous genes. Meanwhile, given the specific purposes of the original network, the application traces the regulative process back and gives possible regulative patterns of new gene.
APPROACH & METHODOLOGY
The software is comprised of several modules as shown below:Introduction of modules:
1 Data Fetching
It is a alterable one which could be rewrite for different database document. Our model is based on the database of RegulonDB. We simulate all the TF gene regulation from the download data in this
website: http://regulondb.ccg.unam.mx/menu/download/
datasets/index.jsp There are 166 genes in the genetic network whose interactions are fetched from TF-TF Interaction file and we also search all those genes’ information in Gene Sequence file. Such as  gene identifier assigned by RegulonDB, gene left & right end position in the genome and gene sequence.
2 Alignment to Get New Regulation
We use different alignment methods to predict the regulation between new gene and genes in original network. We align gene sequence and amino acid sequence with Needleman-Wunsch algorithm, and forecast the new gene’s regulation to all the other genes. Similar percentages are output to simulation module as an influence to regulation factors. This software use these regulation patterns which consists of new gene’s regulation relationship in the target network to build up a new network.
3 Suggestion of New Gene
This software could give a suggestion inserted gene to experimenters who need to hence some specific genes’ expression via regulation. We use PSO(Particle Swarm Optimization) to find regulation factors which fit users’ requirements best. Based on those factors, our software filters obvious regulations and gives them to the experimenter as a reference of inserted gene. If our simulation of network is consonant with real organisms’ gene expression rate changing, experimenters could achieve their ideal expression through insert a new gene whose regulation is same to software’s suggestion.
4 Simulation Network
As the specific concentration value of a regulation relationship is usually unavailable, the software will adopt a random number as the coefficient of the linear equation.  To avoid extreme regulatory relationships, this module transforms comparison results to a probability offset of the regulatory factors.  It is probability to the range of random factor ,   means the probability of [0,1],   means the probability of [-1,0],   is the similarity of gene sequences and amino acids. As a result, the error of the regulation factor is minimized by random sow. Finally, we figure out a new regulatory factor matrix combine with original genetic network. Based on the matrix of regulation relationships, this software sets up the differential equations from which we could figure out the concentration of the material from one moment to the next.
(Hill’s Equation) The simulation runs until the network is stable, which means the concentration of material in the network is basically constant. In order to compare the stability between the original network and the new one, the simulation of the original network will be done as well. More random numbers will be adopted as repeating the simulation.  Based on the data from simulation above, a stability score of each combination of coefficients will be calculated. Those scores are based on the cost of stable time and the difference of two network. After the topological analysis of those scores, an overall score will be given.
PURPOSE & BACKGROUND
Synthetic biology has been working on transforming target organisms, which usually means integrating new genes with an available network to achieve a high expression level of certain compounds. Nevertheless, the new-integrated genes are always not the original parts of the target metabolic network, so it is hard to predict how the new genes will affect the network. In some cases, new genes may even lead the network to a breakdown unexpected by wet lab experimenter. On the other hand, some wet lab experimenters also expect that target organisms could increase some original gene’s expression. As a reference to those experiment, our software put a virtual gene into the network and figure out its best regulation. To achieve wet lab’s purpose, experimenter could find a specific gene based on our prediction regulation.      Lots of simulations of metabolic networks have been done with various methods. Most of them concentrate on the network itself and some of them analyze those network’s stability, robustness and Flux Balance Analysis (FBA).
SIGNIFICANCE & INNOVATION
The software provides a great model as a reference before wet lab experiments. It provides suggestions on both specific practicality and input of exogenous gene. Meanwhile, the software is comprised of separate modules, which can be customized for different database and optimized as the amount of users grows.     Previously, we could not find any specific work on the simulation and analysis of the newly introduced genes’ impact on regulation networks, and also those softwares were unavailable, which makes our software an innovation in the field. Algorithmically, the software aims to complete the simulation based on a small amount of lab data as possible.
Retrieved from " http://2013.igem.org/Team:USTC-Software/Software/Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
