"
Page
Discussion
View source
History
teams
Log in
 
Team:ETH Zurich/Mutant
From 2013.igem.org
Home
Project
Information processing
Circuit and optimization
Pre-processing: Signal diffusion
Signal processing: High pass filters
Player interaction: Design and playing
Output system: Hydrolases
Data page
Experimental results
Overview
Pre-processing: Diffusion characterization
Proof-of-principle with GFP
Signal processing engineering
Circuit optimization
Output system characterization
Flagging
Gameplay
Model
Overview
Single Cell Model
AHL reaction-diffusion model
Model Evaluation: Genetic circuit with GFP
Genetic Circuit incorporating hydrolases
Analytical Approximations
Parameters
Lab work
Triple knockout strain
Constructs
Lab journal
Methods
Materials
Safety
Human practice
Overview
Worldwide playing
Gamification
Education
Team
Students
Advisors/Instructors
Official Team Profile
Attributions & contributions
References
Achievements
Achievements
Medal criteria
Awards
Login
WIKI
Log out
My account
Edit page
History
Upload new file
Triple knockout strain
In order to use the orthogonal hydrolases as our reporter system in the sender-receiver set-up, we needed to use a strain of E. coli where the expression of native hydrolase genes was knocked out. This is to prevent background hydrolysis of our colorimetric substrates.
In our sender-receiver circuit, we use three hydrolases gusA , aes and nagZ. Hence, three hydrolase genes were knocked out of E. coli strain MG1655. The gusA hydrolase was knocked out by a scarless deletion scheme that left no scar sequences after the deletion method (1). The subsequent deletions of aes and nagZ were carried out by the P1 phage transduction by using a deletion strain from the Keio collection (2). Thus we were able to use this strain MG1655△gusA△aes△nagZ knocked out of expression of the three hydrolase genes in our project.
The triple mutant is used for the sender cells which are made to constitutively express nagZ. In the receiver cells, the hydrolases aes and gusA are expressed under the control of OHHL induced PluxR promoters which serve as highpass filters .
References
(1) Martinez-Garcia .E, de Lorenzo .V. 2012. Transposon based and plasmid based genetic tools for editing genomes of gram-negative bacteria. Methods in Molecular Biology. 813:267-283.doi: 10.1007/978-1-61779-412-4_16.
(2) Thomason L.C., Costantino .N, Court .D .L, 2007. E. coli Genome Manipulation by P1 Transduction. Current Protocols in Molecular Biology.1.17.1-1.17.8.
We thank our sponsors:
File:Basf-logo.gif
180px
100px
130px
Retrieved from " http://2013.igem.org/Team:ETH_Zurich/Mutant "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
