<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Materials"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Materials">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Materials&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Materials&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Materials&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Materials" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Materials</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: pointer">Project</A></LI><LI><A href="/Team:ETH_Zurich/Infoproc" title="Team:ETH Zurich/Infoproc">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Circuit" title="Team:ETH Zurich/Circuit">Circuit and optimization</A></LI><LI><A href="/Team:ETH_Zurich/pre_proc" title="Team:ETH Zurich/pre proc">Pre-processing: Signal diffusion</A></LI><LI><A href="/Team:ETH_Zurich/Processing_2" title="Team:ETH Zurich/Processing 2">Signal processing: High pass filters</A></LI><LI><A href="/Team:ETH_Zurich/Play" title="Team:ETH Zurich/Play">Player interaction: Design and playing</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_7" title="Team:ETH Zurich/Experiments 7">Output system: Hydrolases</A></LI></UL><LI><A href="/Team:ETH_Zurich/Data_Page" title="Team:ETH Zurich/Data Page">Data page</A></LI><LI><A name="Biology" style="cursor: pointer">Experimental results</A></LI><LI><A href="/Team:ETH_Zurich/Experimentalresults" title="Team:ETH Zurich/Experimentalresults">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_2" title="Team:ETH Zurich/Experiments 2">Pre-processing: Diffusion characterization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_6" title="Team:ETH Zurich/Experiments 6">Proof-of-principle with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_5" title="Team:ETH Zurich/Experiments 5">Signal processing engineering</A></LI><LI><A href="/Team:ETH_Zurich/Optimization" title="Team:ETH Zurich/Optimization">Circuit optimization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_3" title="Team:ETH Zurich/Experiments 3">Output system characterization</A></LI><LI><A href="/Team:ETH_Zurich/Flagging" title="Team:ETH Zurich/Flagging">Flagging</A></LI><LI><A href="/Team:ETH_Zurich/gameplayexperiment" title="Team:ETH Zurich/gameplayexperiment">Gameplay</A></LI><LI><A name="Model" style="cursor: pointer">Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Overview" title="Team:ETH Zurich/Modeling/Overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/InitialStates" title="Team:ETH Zurich/Modeling/InitialStates">Single Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Reaction_Diffusion_OOHL" title="Team:ETH Zurich/Modeling/Reaction Diffusion OOHL">AHL reaction-diffusion model</A></LI><LI><A href="/Team:ETH_Zurich/GFP" title="Team:ETH Zurich/GFP">Model Evaluation: Genetic circuit with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Genetic Circuit incorporating hydrolases</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximations" title="Team:ETH Zurich/Modeling/Analytical Approximations">Analytical Approximations</A></LI><LI><A href="/Team:ETH_Zurich/Parameter" title="Team:ETH Zurich/Parameter">Parameters</A></LI><LI><A name="LAb work" style="cursor: pointer">Lab work</A></LI><LI><A href="/Team:ETH_Zurich/Mutant" title="Team:ETH Zurich/Mutant">Triple knockout strain</A></LI><LI><A href="/Team:ETH_Zurich/Experiments" title="Team:ETH Zurich/Experiments">Constructs</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Lab journal</A></LI><LI><STRONG class="selflink">Methods</STRONG></LI><LI><A href="/Team:ETH_Zurich/Materials_2" title="Team:ETH Zurich/Materials 2">Materials</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A name="Human" style="cursor: pointer">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Practiceoverview" title="Team:ETH Zurich/Practiceoverview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/World" title="Team:ETH Zurich/World">Worldwide playing</A></LI><LI><A href="/Team:ETH_Zurich/Human" title="Team:ETH Zurich/Human">Gamification</A></LI><LI><A href="/Team:ETH_Zurich/HEducational" title="Team:ETH Zurich/HEducational">Education</A></LI><LI><A name="Team" style="cursor: pointer">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Students</A></LI><LI><A href="/Team:ETH_Zurich/Supervisors" title="Team:ETH Zurich/Supervisors">Advisors/Instructors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; contributions</A></LI><LI><A href="/Team:ETH_Zurich/References" title="Team:ETH Zurich/References">References</A></LI><LI><A name="Achievements" style="cursor: pointer">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal criteria</A></LI><LI><A href="/Team:ETH_Zurich/Awards" title="Team:ETH Zurich/Awards">Awards</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Materials" class="external text" rel="nofollow">Login</A></LI><LI><A name="WIKI" style="cursor: pointer">WIKI</A></LI><LI><A href="/wiki/index.php?title=Special:UserLogout&amp;returnto=Team:ETH_Zurich/Home;returntoquery=action%3Dedit" title="Log out">Log out</A></LI><LI><A href="https://igem.org/User_Information" title="My account">My account</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Materials&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Materials&amp;action=history" class="external text" rel="nofollow">History</A></LI><LI><A href="http://2013.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Preparation_of_stock_solutions"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Preparation of stock solutions</SPAN></A></LI><LI class="toclevel-1"><A href="#General_cloning_procedure"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">General cloning procedure</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimental_procedures"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Experimental procedures</SPAN></A></LI><LI class="toclevel-1"><A href="#We_thank_our_sponsors:"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">We thank our sponsors:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Preparation_of_stock_solutions">Preparation of stock solutions</SPAN></H1><P><B>Making LB medium</B></P><UL><LI>Dissolve 12.5g of LuriaBroth in 500ml MilliQ H<SUB>2</SUB>O and autoclave</LI></UL><P><B>Making LB-Agar</B></P><UL><LI>Dissolve 12.5g LuriaBroth in 500mL MIlliQ H<SUB>2</SUB>O,add 7.5g agar.</LI></UL><P>Before use: Heat it in the microwave and let it cool down to 50℃ before adding Antibiotics.</P><P><B>Recipe for M9 medium (500 ml)</B></P><UL><LI>100 ml MilliQ H<SUB>2</SUB>O
</LI><LI>50 µl 1M CaCl<SUB>2</SUB></LI><LI>50 ml 10x M9 salts
</LI><LI>5 ml of 40 % Glucose or Glycerol
</LI><LI>1 ml 1M MgSO<SUB>4</SUB></LI><LI>50 µl 10000x Trace elements solution
</LI><LI>Fill up to 500 ml with MilliQ H<SUB>2</SUB>O
</LI></UL><P><B>Making M9-Agar (500 ml)</B></P><UL><LI>Add 15 g of agar to 450 ml MilliQ H<SUB>2</SUB>O and autoclave, then cool to about 55°C
</LI><LI>50 µl 1M CaCl<SUB>2</SUB></LI><LI>50 ml 10x M9 salts
</LI><LI>5 ml of 40 % Glucose or Glycerol
</LI><LI>1 ml 1M MgSO<SUB>4</SUB></LI><LI>50 µl 10000x Trace elements solution
</LI><LI>Add antibiotics and pour plates
</LI></UL><P><B>Antibiotics stock solutions</B> 
Ampicillin (amp): 100mg/ml (1000X)</P><UL><LI>Dissolve 1g Ampicillin in 10ml sterile H<SUB>2</SUB>O.
</LI><LI>Aliquot and store at -20 ℃.</LI></UL><P>Kanamycin (Kan): 50mg/ml (1000X)</P><UL><LI>Dissolve 0.5g Kanamycin in 10ml sterile H<SUB>2</SUB>O.
</LI><LI>Aliquot and store at -20 ℃.</LI></UL><P>Chloramphenicol(cam): 34mg/ml (1000X)</P><UL><LI>Dissolve 0.34g Chloramphenicol in 10ml sterile H<SUB>2</SUB>O. 
</LI><LI>Aliquot and store at -20℃.</LI></UL><P>All stock solutions have to be diluted 1:1000 times when used for cultures/Plates</P><P><B>Glycerol 20%</B></P><UL><LI>Mix 20ml of Glycerol 100% and 80ml of MilliQ H<SUB>2</SUB>O, autoclave</LI></UL><P><B>Rubidium buffer 1 and 2</B></P><OL><LI>Buffer 1</LI><LI>KOAc 30nM</LI><LI>CaCl<SUB>2</SUB> 10mM</LI><LI>Glycerol 15%</LI><LI>Adjust pH to 5.8 with Acetic acid</LI><LI>RbCl<SUB>2</SUB> 100nM</LI><LI>MnCl<SUB>2</SUB>  50nM</LI><LI>Adjust pH to 5.9 with KOH</LI><LI>Mix all components, sterile filter and store at 4℃</LI><LI>Buffer 2</LI><LI>MOPS  10mM</LI><LI>RbCl<SUB>2</SUB>  10mM</LI><LI>CaCl<SUB>2</SUB>  75mM</LI><LI>Glycerol 15%</LI><LI>Mix all components and store at 4℃</LI></OL><P>Note that these are order sensitive, i.e. you have to add KOAc <I>before</I> CaCl<SUB>2</SUB>.
</P><P><B>Rubidium competent cells</B></P><UL><LI>Inoculate 100ml pre-warmed CVM with 2ml of the overnight culture and incubate under shaking (250 rpm) until OD<SUB>600</SUB> : 0.48</LI><LI>Put the culture on ice for 15 minutes and divide it in 2*50 ml falcon tubes</LI><LI>Spin at 4000rpm ,4℃ for 5 minutes</LI><LI>Discard supernatant</LI><LI>Re-suspend pellet in 15ml of ice cold buffer 1</LI><LI>keep on ice for 12 minutes </LI><LI>Spin at 4000rpm , 4℃ for 5 minutes</LI><LI>Carefully resuspend the pellet in 2mL ice cold buffer 2</LI><LI>Aliquot 10*100μl per 50ml culture in 1.5ml tubes and freeze with liquid nitrogen</LI></UL><H1><SPAN class="mw-headline" id="General_cloning_procedure">General cloning procedure</SPAN></H1><P><B>Minipreps</B></P><UL><LI>Sigma Aldrich Miniprep kit : Elute in 50μl for higher plasmid concentrations.</LI></UL><P><B>Restriction enzyme digest</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 1. iGEM suffix insertion</B></DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 2. iGEM 3A Assembly</B></DIV></DIV></DIV><UL><LI>50μl reaction Volume:</LI><LI>2ug of plasmid (max 45μl for low concentration minipreps)</LI><LI>5μl NEB buffer (which buffer for which enzyme can be looked up on the NEB website)</LI><LI>add ddH<SUB>2</SUB>O up to 50μl</LI><LI>0.5/1μl of restriction enzyme (20000U/ml/10000U/ml)</LI><LI>Keep enzymes always on cooling block and don't take them out for too long.</LI><LI>Mix through pipetting </LI><LI>1h at 37℃</LI><LI>Heat inactivation : 20min at 65℃(not always necessary)</LI><LI>Dephosphorylation of backbones only (reduces plasmid self ligation)</LI><LI>Add 1μl of Calf-intestine-phosphatase)</LI><LI>1h at 37℃</LI><LI>20 minutes heatinactivation at 65℃.</LI></UL><LI>Add 10μl of loading buffer (6X)</LI><LI>Analysis on agarose gel</LI><P><B>Ligation</B></P><UL><LI>100ng of backbone and corresponding amount of insert depending on size</LI><LI>Always do a negative control without insert</LI><LI>Add up to 10/15μl with ddH2O</LI><LI>1.1/1.65μl T4 ligation buffer</LI><LI>1μl Ligase</LI><LI>1h at room temperature</LI><LI>20 min at 65℃</LI></UL><P><B>Gel extraction</B>Sigma Aldrich Gel extraction kit</P><P><B>Transformation</B></P><UL><LI>Thaw competent cells on ice for 10 minutes</LI><LI>5μL of ligation product or 1μl of resuspended plasmid/miniprep in 50μl competent cells</LI><LI>30min on ice</LI><LI>1 min heat shock at 42℃</LI><LI>3 minutes on ice</LI><LI>add 900μl LB</LI><LI>1h shaking at 37℃s</LI><LI>Centrifuge at 12000rcf</LI><LI>remove 750μl supernatant</LI><LI>resuspend the pellet in the remaining 200μL LB</LI><LI>Plate on pre-warmed plates</LI></UL><P><B>Sequencing</B></P><UL><LI>1200ng of DNA</LI><LI>3μL sequencing primer (10μl)</LI></UL><P><B>PCR (for gene amplification or colony PCR)</B>
PCR was used for gene amplification for cloning and for colony PCR for the clone selection. For colony PCR Taq polymerase was used whereas for the gene amplification due to higher needs in accuracy Phusion<SUP>®</SUP> polymerase was used. For gene Amplification PCR the reaction volume is 50μL and for colony PCR 20μL. 1-20ng of the template plasmids were used for the amplification. For the coplony PCR in some cases NEB<SUP>®</SUP> QuickLoad 2X Taq master mix was used.</P><TABLE style="font-size:12px;font-family:verdana" class="sortable"><TBODY><TR><TH width="200px"> Equipment</TH><TH width="350px">Gene Amplification PCR (50μL reaction volume)</TH><TH width="350px" colspan="2">Colony PCR (20μL reaction volume</TH></TR><TR><TD>Volume μL</TD><TD>Volume μL</TD><TD>Volume μL</TD></TR><TR><TD>PCR Buffer 5X/10X</TD><TD>10</TD><TD>2</TD><TD>-</TD></TR><TR><TD>dNTP 10uM</TD><TD>1</TD><TD>0.5</TD><TD>-</TD></TR><TR><TD>Rev and fwd primer 10uM</TD><TD>1</TD><TD>0.75</TD><TD>0.75</TD></TR><TR><TD>Polymerase 2U/μL</TD><TD>0.5</TD><TD>0.5</TD><TD>-</TD></TR><TR><TD>Template DNA + H2O</TD><TD>36.5</TD><TD>15.5</TD><TD>10</TD></TR><TR><TD>QuickLoad 2X Taq</TD><TD>-</TD><TD>-</TD><TD>10</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Experimental_procedures">Experimental procedures</SPAN></H1><P><B>Double layer agar plate experiment</B>
The method of double layer agar experiment was adapted from bacteriphage plaque experiments used for phage transduction.<UL><LI>Pour 1.5% LB-agar (bottom layer) on a petridish. Allow to cool and solidify.</LI><LI>Add 100μl of liquid culture of receiver cells into 3ml of 0.7% agar (40℃).</LI><LI>The receiver cultures were grown to OD600 of 0.4 and then dilute into 2mL of 0.7% Agar</LI><LI>Allow the top layer to cool down and solidify. </LI><LI>1.5ul of the sender cells were then pipetted on to the two layer agar plates.</LI><LI>Incubate the plates at 37℃.</LI></UL></P><P><B>Enzyme kinetics assay</B>
This protocol was provided by Dr. Johannes Haerle
<OL><LI>Prepare buffers
</LI><LI>Lysis buffer: 10 mg/ml Lysozyme, 20 mM Tris buffer, pH 8
</LI><LI>Reaction buffer: 20 mM Tris buffer, pH 8
</LI><LI>NOTE: For other enzymes than the ones we tested (Aes,GusA,NagZ,PhoA) you might need different buffers
</LI><LI>Cell culture
</LI><LI>Inoculate bacteria in 20 mL of LB with antibiotics
</LI><LI>Let grow at 37°C shaking(200 rpm) to an OD<SUB>600</SUB> of 0.6
</LI><LI>Induce enzyme expression (100nM AHL in our case)
</LI><LI>Let grow at 37°C shaking(200 rpm) for 4-5h</LI><LI>Cell lysis
</LI><LI>Transfer to 50 mL Falcon™ tube
</LI><LI>Spin down at 4°C for 5 min with 4 rcf
</LI><LI>Resuspend in lysis buffer, 1 μL/mg pellet
</LI><LI>Transfer to eppendorf tubes
</LI><LI>Incubate at room temperature for 10 min at 220 rpm
</LI><LI>Spin down at 4°C for 10 min with max. speed
</LI><LI>Transfer the supernatant to new tubes, discard pellets
</LI><LI>Cell free extract can be stored at -20°C or continue processing
</LI><LI>Dilution
</LI><LI>The following values were provided by Johannes Haerle
</LI><LI>Aes: Dilute CFX 1:100 in reaction buffer 
</LI><LI>GusA: Dilute CFX 1:100 in reaction buffer
</LI><LI>NagZ: Use pure
</LI><LI>PhoA: Dilute CFX 1:10 in reaction buffer
</LI><LI>Hydrolysis reaction
</LI><LI>Perform this measurement in a 96 well plate or similar
</LI><LI>Perform 3 replicates for each substrate concentration
</LI><LI>Present 41.6 μL reaction buffer in each well
</LI><LI>Add 8 μL diluted CFX (the further dilution ocurring here is intended)
</LI><LI>Add 30.4 μL of corresponding substrate
</LI><LI>Detection of fluorescence in suitable plate reader (λ<SUB>Ex</SUB> 365 nm, λ<SUB>Em</SUB> 445 nm)</LI></OL></P><P><B>Microtiter plate</B><OL><LI>The experimental set-up included a 96-well plate filled with 180 μL LB media , 10uμ of receiver cells and 10μL of different OHHL concentrations, everything in triplicates. (+ blank and negative control). The experiment runs for 16 hours in order to monitor the evolution of fluorescence and later choose the steady state to create the sensitivity curve.</LI></OL></P><P><B>Facs experiments</B><OL><LI>For the real Colisweeper game we need to know about the behavior of the promoters on agar plates.We set-up an experiment which consists in pooring agar plates containing different OHHL concentration going form 10<SUP>-5</SUP>nM to 10<SUP>5</SUP>nM by 10 fold increase. The wild type and the luxR variant promoter were streaked onto those plates to analyze the sensivity to OHHL of the wild type and the promoter on an agar plate. After 15hours of incubation at 37 degree celsius some colonies were picked and resuspended in <I>Phosphate Buffered Solution</I> (PBS) for the single cell analysis , using again the BD LSRFortessa™ Flow Cytometer System.</LI></OL></P><LI>For liquid culture:From overnight cultures we inoculate 50 mL Falcon™ tubes containing 5mL LB media, 5 μL OHHL and the antibiotic. We evaluate a range of 10 different concentrations from 10<SUP>-4</SUP>nM OHHL to 10<SUP>5</SUP>nM OHHL by a 10 fold increase, according to the results of the plate reader analysis. The measurements were done in duplicates.</LI><P><B>Substrate tests</B><UL><LI>Liquid cultures
</LI><LI>5 ml M9 medium + antibiotics
</LI><LI>Add 100 ul of ON culture
</LI><LI>Grow until OD(600) 0.4 - 0.6 is reached
</LI><LI>Add substrate (see <A href="https://2013.igem.org/Team:ETH_Zurich/Materials_2#chrom_subs">table</A> for applicable end concentrations)
</LI><LI>Shake at 37°C for 10 - 20 minutes</LI></UL><UL><LI>Colonies
</LI><LI>5 ml M9 medium + antibiotics
</LI><LI>Add 100 ul of ON culture
</LI><LI>Grow until OD(600) 0.4 - 0.6 is reached
</LI><LI>Prepare M9 plates
</LI><LI>Pipette 1.5 ul of the liquid culture onto the agar plate for one colony
</LI><LI>Incubate at 37°C for at least 11 hours
</LI><LI>Add 1.5 ul of substrate onto colonies (see <A href="https://2013.igem.org/Team:ETH_Zurich/Materials_2#chrom_subs">table</A> for applicable concentrations)
</LI><LI>Wait until color develops, either at RT or 37°C (usually faster)</LI></UL></P><P><B>Spiral diffusion test with AHL</B><UL><LI>The receiver cells (J09855+GFP) were plated on LB-agar in a spiral pattern.
</LI><LI>After overnight incubation a drop of 5μL AHL was added to the center of the spiral , on the central colony.
</LI><LI>The diffusion of the AHL was recorded by the fluorescent activated colonies over time (in Dark Hood)</LI></UL></P></DIV><DIV id="eth_footer"><H1><SPAN class="mw-headline" id="We_thank_our_sponsors:">We thank our sponsors:</SPAN></H1><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Basf-logo.gif" title="File:Basf-logo.gif">File:Basf-logo.gif</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Novartis-logo.gif" title="File:Novartis-logo.gif">180px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Roche-logo.gif" title="File:Roche-logo.gif">100px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Syngenta-logo.gif" title="File:Syngenta-logo.gif">130px</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:ETH_Zurich/Materials">http://2013.igem.org/Team:ETH_Zurich/Materials</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Materials" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Materials" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Materials&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Materials&amp;oldid=328317" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>