"
Page
Discussion
View source
History
teams
Log in
 
Team:ETH Zurich/Experiments 7
From 2013.igem.org
Home
Project
Information processing
Circuit and optimization
Pre-processing: Signal diffusion
Signal processing: High pass filters
Player interaction: Design and playing
Output system: Hydrolases
Data page
Experimental results
Overview
Pre-processing: Diffusion characterization
Proof-of-principle with GFP
Signal processing engineering
Circuit optimization
Output system characterization
Flagging
Gameplay
Model
Overview
Single Cell Model
AHL reaction-diffusion model
Model Evaluation: Genetic circuit with GFP
Genetic Circuit incorporating hydrolases
Analytical Approximations
Parameters
Lab work
Triple knockout strain
Constructs
Lab journal
Methods
Materials
Safety
Human practice
Overview
Worldwide playing
Gamification
Education
Team
Students
Advisors/Instructors
Official Team Profile
Attributions & contributions
References
Achievements
Achievements
Medal criteria
Awards
Login
WIKI
Log out
My account
Edit page
History
Upload new file
Contents
1 Reporter system
2 Acetyl esterase (Aes)
3 Alkaline phosphatase (PhoA)
4 β-Galactosidase (LacZ)
5 β-Glucuronidase (GusA)
6 β-N-Acetylglucosaminidase (NagZ)
7 References
8 We thank our sponsors:
Reporter system
Colisweeper depends on the processing of molecular signals and generation of visible and distinguishable color outputs . By using the wild type PLuxR promoter and mutant promoters with varying AHL/LuxR affinities, gene expression can be induced for different concentrations of AHL. We coupled this system with reporter enzymes, called hydrolases, which catalyze hydrolysis of their colorless substrate into a colored product. Therefore, the player only triggers the colored response upon addition of a substrate.
The set of hydrolases we use include the Citrobacter Alkaline phospohatase, the Bacillus subtilis β-Glucuronidase and the Escherichia coli Acetyl esterase, β-N-Acetylglucosaminidase and β-Galactosidase. These enzymes have features that make them attractive as reporters. They are known to be relatively stable, exhibit activity under different conditions (e.g. pH and temperature), and catalyze the cleavage of various colorimetric and fluorescent substrates, which is ideal for visual screening. These hydrolases are all native to Escherichia coli and orthogonal to each other (see crosstalk experiment here ), which means that each substrate can only be processed by the corresponding specific enzyme. In order to prevent background expression of these native enzymes, we use a triple knockout strain that has three hydrolase genes knocked out: uidA (β-glucuronidase), aes (acetyl esterase) and nagZ (β-N-acetylglucosaminidase).
Addition of a multi-substrate mix by the player leads to an enzyme-susbtrate reaction which specifically cleaves the chromogenic substrates, thereby producing a visible color output. Chromogenic substrates incorporate a chromophore whose absorbance properties change after the enzyme reaction, and the color signal produced then is directly related to the enzyme-catalyzed reaction ( 1 ).
Figure 1: Colorimetric output. This image shows liquid cultures of our mutant Escherichia coli strain expressing the enzymes PhoA, NagZ, LacZ, Aes or GusA to convert specific chromogenic substrates into colored outputs.
Indican is a chromogenic glycoside hydrolase substrate which belongs to a family of natural glycosides found in plants. Cleavage of the glycosidic bond forms an unstable hydroxyindole intermediate, which dimerizes by oxidation to form indigo as a blue precipitate:
Numerous enzyme substrates have been designed following this natural product example, giving rise to many colored phenols that are used to detect enzyme activities.
Some of the hydrolases used in the Colisweeper reporter system can catalyze hydrolysis of various substrates, with different chromophores that give rise to a wide range of colors. This variety of substrates and colored outputs, additionally to the orthogonality of the enzyme-substrate reactions, enables usage of multiple hydrolases in reporter gene assays, which are becoming more important in real-time detection of cellular events associated with gene expression, regulation and signal transduction ( 2 ).
For most parts encoding the hydrolases that were sent to the registry by our team, we have conducted fluorometric assays to study and compare their kinetics. The results can be seen here , as well as in the experience section of the parts' registry sites, and the methods used are available here .
Acetyl esterase (Aes)
3D representation of the acetyl esterase from RCSB
The Escherichia coli acetyl esterase is encoded by the aes gene and has been identified as a member of the hormone-sensitive lipase family ( 3 ). Acetyl esterase interferes with the expression of the maltose system by counteracting maltose sensitivity ( 4 ). This cytosolic enzyme is a 36 kDa monomer and catalyzes hydrolysis of short chain fatty esters with acyl chain lengths of up to eight carbons ( 3 ). It is composed of 319 amino acid residues ( 5 ) and its catalytic triad is composed of Ser165, Asp262 and His292 ( 6 ). Various substrates can used for chromogenic enzyme assays with this enzyme, e.g. p-nitrophenyl acetate ( 4 ), p-nitrophenyl butanoate ( 7 ) and p-nitrophenyl butyrate ( 8 ). According to Kobayashi et al. ( 8 ), kinetic constants for the Escherichia coli acetyl esterase are Km 170 ± 41 µM and kcat 29 ± 3.0 s-1 when using p-nitrophenyl butyrate as a substrate. This enzyme displays an optimal temperature at 65°C, depicting a specific activity of 250 U/mg using pNP-butanoate as substrate, but a low half-life at the same temperature (t1/2 of inactivation = 5 minutes) ( 7 ).
This enzyme's coding region has been added to the registry by our team as BBa_K1216002 . We used this part in our final circuit and have made this our favorite part. Additionally, this part is available with an added TEV and poly-HIS tags for purification of the protein ( BBa_K1216006 ). In Colisweeper, 5-Bromo-6-Chloro-3-indoxyl butyrate is used in the multi-mix substrate. After hydrolysis of this substrate, an indigo analog precipitates, which is visible as magenta color.
Alkaline phosphatase (PhoA)
3D representation of the alkaline phosphatase from RCSB
Alkaline phosphatase is a non-specific periplasmic phosphomonoesterase that catalyzes production of free inorganic phosphate and phosphoryl transfer reaction to various alcohols. In Escherichia coli, this hydrolase is involved in the acquisition of phosphate from esters when free inorganic phosphate is scarce ( 9 ), and is also capable of catalyzing the oxidation of phosphite to phosphate and molecular H2 ( 10 ).
This enzyme is a homodimeric metalloenzyme with two Zn2+ and on Mg2+ forming the metal triplet at each active site, which is considered to be Asp101-Ser102-Ala103. It has been shown that with p-nitrophenyl phosphate (pNPP) as a substrate, this enzyme has a specific activity of 1020 µmol pNPP h-1 mg-1 at pH 8; and its kcat and Km are 13.6 s-1 and 7.4 µM, respectively ( 9 ).
In Colisweeper, the alkaline phosphatase is natively expressed in the Escherichia coli strain we used and processes pNPP of the substrate mix to produce yellow color. However, we made use of the Citrobacter phoA gene ( BBa_K1216001 ) for kinetics assays of this enzyme. Also, we have added an improved version of this part to the registry ( BBa_K1216005 ), with TEV and HIS-tags added to enable protein purification.
β-Galactosidase (LacZ)
3D representation of the β-Galactosidase from RCSB
The Escherichia coli β-galactosidase is encoded by the gene lacZ, a part of the lac operon which is inducible by lactose in the absence of glucose. β-Galactosidase catalyzes hydrolysis of glycosidic bonds between galactose and its moiety ( 11 ). It is commonly used as a reporter to indicate gene expression by using a chromogenic substrate for detection e.g. X-Gal, which forms a blue product after cleavage.
This enzyme is natively expressed in the Escherichia coli strain used for Colisweeper. Its function in this game is to catalyze hydrolysis of the flagging substrate, β-D-Green-Gal, which results in the formation of green color output.
β-Glucuronidase (GusA)
3D representation of the β-Glucuronidase from RCSB
The enzyme β-glucuronidase catalyzes the hydrolysis of D-glucuronic acids which are conjugated through a β-O-glycosidic linkage to an aglycone. The action of bacterial β-glucuronidase in the gastrointestinal tract is an important component in the enterohepatic circulation of many hydrophobic xenobiotics and endogenous waste compounds, which are conjugated to D-glucuronic acid during the main detoxification pathway of vertebrates, and are then excreted in the bile or urine ( 12 ). The reduction of these beta-d-glucuronides by β-glucuronidase activity frees aglycone residues with protective effects, such as lignans, flavonoids, ceramide and glycyrrhetinic acid ( 13 ), enabling their absorption and enterohepatic circulation.
β-Glucuronidase is commonly used as a reporter system for various experimental measurements, and has been favoured as a reporter gene in plants. In contrast to β-galactosidase, which is a widely used reporter protein, β-glucuronidase has a key advantage in its robustness and its absence in many organisms other than vertebrates and their attendant microflora, rendering the possibility of accurate measurements of β-glucuronidase activity ( 14 ).
The part we implemented in Colisweeper is gusA ( BBa_K1216000 ), a homolog of the Escherichia coli uidA, and encodes the Bacillus subtilis β-glucuronidase, a 68 kDa tetrameric and intracellular enzyme. This part is also available with added TEV and poly-HIS tags ( BBa_K1216004 ). According to Tryland and Fiksdal ( 15 ), the activity of the Bacillus spp. β-glucuronidase is similar to that of the Escherichia coli homolog, being only approximately 10-fold lower. In our game, β-glucuronidase catalyzes hydrolysis of 6-Chloro-3-indolyl-β-glucuronic acid, which gives rise to an orange-pink color.
β-N-Acetylglucosaminidase (NagZ)
3D representation of the β-N-Acetylglucosaminidase from RCSB
The β-N-Acetylglucosaminidase enzyme is a cytoplasmic hydrolase which is involved in the Escherichia coli cell wall recycling pathway. Its natural substrates are muropeptides and anhydro-muropeptides, which are released into the cytosol from cell wall murein ( 16 ). The enzyme is active against the β-1,4-glycosidic bonds and releases β-N-acetylglucosamine ( 17 ). Inhibitors of NagZ are N-acetylglucosamine and N-acetylmuramic acid at high concentrations, or N-acetylglucosaminolactone ( 17 ). Using p-nitrophenyl-β-N-acetylglucosaminide as a substrate, a kinetic assay by Vötsch and Templin ( 18 ) showed a Km 310 µM and Vmax 13.3 µmol/min mg protein at 25°C.
We have added the coding region for β-N-Acetylglucosaminidase to the registry as BBa_K1216003 and used this part for Colisweeper's final circuit in the mine cells, which produce blue color upon hydrolysis of the substrate 5-Bromo-4-Chloro-3-indoxyl-β-N-acetylglucosaminide.
References
(1) Reymond JL, Fluxà VS, Maillard N, Chem. Commun., 1, 34 – 46 (2009).
(2) Jiang T, Xing B, Rao J, Biotechnology and Genetic Engineering Reviews, 25, 41-76 (2008).
(3) Kanaya S, Koyanagi T, Kanaya E, Biochem. J., 332, 75-80 (1998).
(4) Peist R, Koch A, Bolek P, Sewitz S, Kolbus T, Boos J, J. Bacteriol., 179, 7679-7686 (1997).
(5) Blattner FR, Plunkett III G, Bloch CA, Perna NT, Burna V, Riley M, Collado-Vides J, Glasner JD, Rode CK, Mayhew GF, Gregor J, Davis NW, Kirkpatrick HA, Goeden MA, Rose DJ, Mau B, Shao Y, Science, 277, 1453-1461 (1997).
(6) Haruki M, Oohashi Y, Mizuguchi S, Matsuo Y, Morikawa M, Kanaya S, FEBS Lett., 454, 262-266 (1999).
(7) Farias T, Mandrich L, Rossi M, Manco G, Protein Pept. Lett., 14, 165-169 (2007).
(8) Kobayashi R, Hirano N, Kanaya S, Haruki M, Biosci. Biotechnol. Biochem., 76 (11), 2082-2088 (2012).
(9) Kim EE, Wyckoff HW, J. Mol. Biol., 218, 449-464 (1991).
(10) Yang K, Metcalf WW, PNAS, 101, 7919-7924 (2004).
(11) Juers DH, Heightman TD, Vasella A, McCarter JD, Mackenzie L, Withers SG, Matthews BW, Biochemistry, 40, 14781-14794 (2001).
(12) Russell WM, Klaenhammer TR, Appl. Environ. Microbiol., 67, 1253-1261 (2000).
(13) Beaud D, Tailliez P, Anba-Mondoloni J, Microbiology, 171, 2323-2330 (2005).
(14) Jefferson RA, Burgess SM, Hirsh D, Procl. Natl. Acad. Sci. USA, 83, 8447-8451 (1986).
(15) Tryland I, Fiksdal L, Appl. Environ. Microbiol., 64, 1018-1023 (1997).
(16) Cheng Q, Li H, Merdek K, Park JT, J. Bacteriol., 182, 4836-4840 (2000).
(17) Yem DW, Wu HC, J. Bacteriol., 125, 324-331 (1976).
(18) Vötsch W, Templin MF, J. Biol. Chem., 275, 39032-39038 (2000).
We thank our sponsors:
File:Basf-logo.gif
180px
100px
130px
Retrieved from " http://2013.igem.org/Team:ETH_Zurich/Experiments_7 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
