<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Experiments_7"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Experiments_7">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Experiments_7&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_7&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_7&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Experiments_7" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Experiments 7</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: pointer">Project</A></LI><LI><A href="/Team:ETH_Zurich/Infoproc" title="Team:ETH Zurich/Infoproc">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Circuit" title="Team:ETH Zurich/Circuit">Circuit and optimization</A></LI><LI><A href="/Team:ETH_Zurich/pre_proc" title="Team:ETH Zurich/pre proc">Pre-processing: Signal diffusion</A></LI><LI><A href="/Team:ETH_Zurich/Processing_2" title="Team:ETH Zurich/Processing 2">Signal processing: High pass filters</A></LI><LI><A href="/Team:ETH_Zurich/Play" title="Team:ETH Zurich/Play">Player interaction: Design and playing</A></LI><LI><STRONG class="selflink">Output system: Hydrolases</STRONG></LI></UL><LI><A href="/Team:ETH_Zurich/Data_Page" title="Team:ETH Zurich/Data Page">Data page</A></LI><LI><A name="Biology" style="cursor: pointer">Experimental results</A></LI><LI><A href="/Team:ETH_Zurich/Experimentalresults" title="Team:ETH Zurich/Experimentalresults">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_2" title="Team:ETH Zurich/Experiments 2">Pre-processing: Diffusion characterization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_6" title="Team:ETH Zurich/Experiments 6">Proof-of-principle with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_5" title="Team:ETH Zurich/Experiments 5">Signal processing engineering</A></LI><LI><A href="/Team:ETH_Zurich/Optimization" title="Team:ETH Zurich/Optimization">Circuit optimization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_3" title="Team:ETH Zurich/Experiments 3">Output system characterization</A></LI><LI><A href="/Team:ETH_Zurich/Flagging" title="Team:ETH Zurich/Flagging">Flagging</A></LI><LI><A href="/Team:ETH_Zurich/gameplayexperiment" title="Team:ETH Zurich/gameplayexperiment">Gameplay</A></LI><LI><A name="Model" style="cursor: pointer">Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Overview" title="Team:ETH Zurich/Modeling/Overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/InitialStates" title="Team:ETH Zurich/Modeling/InitialStates">Single Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Reaction_Diffusion_OOHL" title="Team:ETH Zurich/Modeling/Reaction Diffusion OOHL">AHL reaction-diffusion model</A></LI><LI><A href="/Team:ETH_Zurich/GFP" title="Team:ETH Zurich/GFP">Model Evaluation: Genetic circuit with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Genetic Circuit incorporating hydrolases</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximations" title="Team:ETH Zurich/Modeling/Analytical Approximations">Analytical Approximations</A></LI><LI><A href="/Team:ETH_Zurich/Parameter" title="Team:ETH Zurich/Parameter">Parameters</A></LI><LI><A name="LAb work" style="cursor: pointer">Lab work</A></LI><LI><A href="/Team:ETH_Zurich/Mutant" title="Team:ETH Zurich/Mutant">Triple knockout strain</A></LI><LI><A href="/Team:ETH_Zurich/Experiments" title="Team:ETH Zurich/Experiments">Constructs</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Lab journal</A></LI><LI><A href="/Team:ETH_Zurich/Materials" title="Team:ETH Zurich/Materials">Methods</A></LI><LI><A href="/Team:ETH_Zurich/Materials_2" title="Team:ETH Zurich/Materials 2">Materials</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A name="Human" style="cursor: pointer">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Practiceoverview" title="Team:ETH Zurich/Practiceoverview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/World" title="Team:ETH Zurich/World">Worldwide playing</A></LI><LI><A href="/Team:ETH_Zurich/Human" title="Team:ETH Zurich/Human">Gamification</A></LI><LI><A href="/Team:ETH_Zurich/HEducational" title="Team:ETH Zurich/HEducational">Education</A></LI><LI><A name="Team" style="cursor: pointer">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Students</A></LI><LI><A href="/Team:ETH_Zurich/Supervisors" title="Team:ETH Zurich/Supervisors">Advisors/Instructors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; contributions</A></LI><LI><A href="/Team:ETH_Zurich/References" title="Team:ETH Zurich/References">References</A></LI><LI><A name="Achievements" style="cursor: pointer">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal criteria</A></LI><LI><A href="/Team:ETH_Zurich/Awards" title="Team:ETH Zurich/Awards">Awards</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Experiments_7" class="external text" rel="nofollow">Login</A></LI><LI><A name="WIKI" style="cursor: pointer">WIKI</A></LI><LI><A href="/wiki/index.php?title=Special:UserLogout&amp;returnto=Team:ETH_Zurich/Home;returntoquery=action%3Dedit" title="Log out">Log out</A></LI><LI><A href="https://igem.org/User_Information" title="My account">My account</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Experiments_7&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Experiments_7&amp;action=history" class="external text" rel="nofollow">History</A></LI><LI><A href="http://2013.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Reporter_system"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Reporter system</SPAN></A></LI><LI class="toclevel-1"><A href="#Acetyl_esterase_.28Aes.29"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Acetyl esterase (Aes)</SPAN></A></LI><LI class="toclevel-1"><A href="#Alkaline_phosphatase_.28PhoA.29"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Alkaline phosphatase (PhoA)</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2-Galactosidase_.28LacZ.29"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">β-Galactosidase (LacZ)</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2-Glucuronidase_.28GusA.29"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">β-Glucuronidase (GusA)</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2-N-Acetylglucosaminidase_.28NagZ.29"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">β-N-Acetylglucosaminidase (NagZ)</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1"><A href="#We_thank_our_sponsors:"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">We thank our sponsors:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Reporter_system">Reporter system</SPAN></H1><P align="justify">Colisweeper depends on the <A href="https://2013.igem.org/Team:ETH_Zurich/Infoproc" class="external text" rel="nofollow">processing of molecular signals</A> and generation of visible and distinguishable <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3" class="external text" rel="nofollow">color outputs</A>. By using the wild type <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5" class="external text" rel="nofollow">P<SUB>LuxR</SUB> promoter and mutant promoters</A> with varying AHL/LuxR affinities, gene expression can be induced for different concentrations of AHL. We coupled this system with reporter enzymes, called hydrolases, which catalyze hydrolysis of their colorless substrate into a colored product. Therefore, the player only triggers the colored response upon addition of a substrate. 


The set of hydrolases we use include the <I>Citrobacter</I><B>Alkaline phospohatase</B>, the <I>Bacillus subtilis</I><B>β-Glucuronidase</B> and the <I>Escherichia coli</I><B>Acetyl esterase</B>, <B>β-N-Acetylglucosaminidase</B> and <B>β-Galactosidase</B>. These enzymes have features that make them attractive as reporters. They are known to be relatively stable, exhibit activity under different conditions (e.g. pH and temperature), and catalyze the cleavage of various colorimetric and fluorescent substrates, which is ideal for visual screening. These hydrolases are all native to <I>Escherichia coli</I> and <B>orthogonal to each other</B> (see crosstalk experiment <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3#crosstalk" class="external text" rel="nofollow">here</A>), which means that each substrate can only be processed by the corresponding specific enzyme. In order to prevent background expression of these native enzymes, we use a <A href="https://2013.igem.org/Team:ETH_Zurich/Mutant" class="external text" rel="nofollow">triple knockout strain</A> that has three hydrolase genes knocked out: <I>uidA</I> (β-glucuronidase), <I>aes</I> (acetyl esterase) and <I>nagZ</I> (β-N-acetylglucosaminidase). </P><P align="justify">Addition of a <B>multi-substrate mix</B> by the player leads to an enzyme-susbtrate reaction which specifically cleaves the chromogenic substrates, thereby producing a <B>visible color output</B>. Chromogenic substrates incorporate a chromophore whose absorbance properties change after the enzyme reaction, and the color signal produced then is directly related to the enzyme-catalyzed reaction (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">1</A>).</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:472px;"><DIV class="thumbcaption"><B>Figure 1: Colorimetric output.</B> This image shows liquid cultures of our mutant <I>Escherichia coli</I> strain expressing the enzymes PhoA, NagZ, LacZ, Aes or GusA to convert specific chromogenic substrates into colored outputs.</DIV></DIV></DIV><P align="justify">Indican is a chromogenic glycoside hydrolase substrate which belongs to a family of natural glycosides found in plants. Cleavage of the glycosidic bond forms an unstable hydroxyindole intermediate, which dimerizes by oxidation to form indigo as a blue precipitate:</P><P align="justify">Numerous enzyme substrates have been designed following this natural product example, giving rise to many colored phenols that are used to detect enzyme activities.
Some of the hydrolases used in the Colisweeper reporter system can catalyze hydrolysis of various substrates, with different chromophores that give rise to a wide range of colors. This variety of substrates and colored outputs, additionally to the orthogonality of the enzyme-substrate reactions, enables usage of multiple hydrolases in reporter gene assays, which are becoming more important in real-time detection of cellular events associated with gene expression, regulation and signal transduction (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">2</A>).



For most parts encoding the hydrolases that were sent to the registry by our team, we have conducted fluorometric assays to study and compare their kinetics. The results can be seen <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3">here</A>, as well as in the experience section of the parts' registry sites, and the methods used are available <A href="https://2013.igem.org/Team:ETH_Zurich/Materials#enzyme_kinetics">here</A>.
</P><H1><SPAN class="mw-headline" id="Acetyl_esterase_.28Aes.29">Acetyl esterase (Aes)</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">3D representation of the acetyl esterase from <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4KRX" class="external text" rel="nofollow">RCSB</A></DIV></DIV></DIV><P align="justify">The <I>Escherichia coli</I> acetyl esterase is encoded by the <I>aes</I> gene and has been identified as a member of the hormone-sensitive lipase family (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">3</A>). Acetyl esterase interferes with the expression of the maltose system by counteracting maltose sensitivity (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">4</A>). This cytosolic enzyme is a 36 kDa monomer and catalyzes hydrolysis of short chain fatty esters with acyl chain lengths of up to eight carbons (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">3</A>). It is composed of 319 amino acid residues (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">5</A>) and its catalytic triad is composed of Ser165, Asp262 and His292 (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">6</A>). Various substrates can used for chromogenic enzyme assays with this enzyme, e.g. <I>p</I>-nitrophenyl acetate (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">4</A>), <I>p</I>-nitrophenyl butanoate (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">7</A>) and <I>p</I>-nitrophenyl butyrate (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">8</A>). According to Kobayashi et al. (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">8</A>), kinetic constants for the <I>Escherichia coli</I> acetyl esterase are K<SUB>m</SUB> 170 ± 41 µM and k<SUB>cat</SUB> 29 ± 3.0 s<SUP>-1</SUP> when using <I>p</I>-nitrophenyl butyrate as a substrate. This enzyme displays an optimal temperature at 65°C, depicting a specific activity of 250 U/mg using pNP-butanoate as substrate, but a low half-life at the same temperature (t<SUB>1/2</SUB> of inactivation = 5 minutes) (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">7</A>). 

This enzyme's coding region has been added to the registry by our team as <A href="http://parts.igem.org/Part:BBa_K1216002" class="external text" rel="nofollow">BBa_K1216002</A>. We used this part in our final circuit and have made this our favorite part. Additionally, this part is available with an added TEV and poly-HIS tags for purification of the protein (<A href="http://parts.igem.org/Part:BBa_K1216006" class="external text" rel="nofollow">BBa_K1216006</A>). In Colisweeper, 5-Bromo-6-Chloro-3-indoxyl butyrate is used in the multi-mix substrate. After hydrolysis of this substrate, an indigo analog precipitates, which is visible as magenta color.</P><H1><SPAN class="mw-headline" id="Alkaline_phosphatase_.28PhoA.29">Alkaline phosphatase (PhoA)</SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">3D representation of the alkaline phosphatase from <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1ANJ" class="external text" rel="nofollow">RCSB</A></DIV></DIV></DIV><P align="justify">
Alkaline phosphatase is a non-specific periplasmic phosphomonoesterase that catalyzes production of free inorganic phosphate and phosphoryl transfer reaction to various alcohols. In <I>Escherichia coli</I>, this hydrolase is involved in the acquisition of phosphate from esters when free inorganic phosphate is scarce (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">9</A>), and is also capable of catalyzing the oxidation of phosphite to phosphate and molecular H<SUB>2</SUB> (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">10</A>).
This enzyme is a homodimeric metalloenzyme with two Zn<SUP>2+</SUP> and on Mg<SUP>2+</SUP> forming the metal triplet at each active site, which is considered to be Asp101-Ser102-Ala103. It has been shown that with <I>p</I>-nitrophenyl phosphate (pNPP) as a substrate, this enzyme has a specific activity of 1020 µmol pNPP h<SUP>-1</SUP> mg<SUP>-1</SUP> at pH 8; and its k<SUB>cat</SUB> and K<SUB>m</SUB> are 13.6 s<SUP>-1</SUP> and 7.4 µM, respectively (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">9</A>).
In Colisweeper, the alkaline phosphatase is natively expressed in the <I>Escherichia coli</I> strain we used and processes pNPP of the substrate mix to produce yellow color. However, we made use of the <I>Citrobacter</I><I>phoA</I> gene (<A href="http://parts.igem.org/Part:BBa_K1216001" class="external text" rel="nofollow">BBa_K1216001</A>) for kinetics assays of this enzyme. Also, we have added an improved version of this part to the registry (<A href="http://parts.igem.org/Part:BBa_K1216005" class="external text" rel="nofollow">BBa_K1216005</A>), with TEV and HIS-tags added to enable protein purification.</P><H1><SPAN class="mw-headline" id=".CE.B2-Galactosidase_.28LacZ.29">β-Galactosidase (LacZ)</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">3D representation of the β-Galactosidase from <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3VD5" class="external text" rel="nofollow">RCSB</A></DIV></DIV></DIV><P align="justify">The <I>Escherichia coli</I> β-galactosidase is encoded by the gene <I>lacZ</I>, a part of the <I>lac</I> operon which is inducible by lactose in the absence of glucose. β-Galactosidase catalyzes hydrolysis of glycosidic bonds between galactose and its moiety (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">11</A>). It is commonly used as a reporter to indicate gene expression by using a chromogenic substrate for detection e.g. X-Gal, which forms a blue product after cleavage.
This enzyme is natively expressed in the <I>Escherichia coli</I> strain used for Colisweeper. Its function in this game is to catalyze hydrolysis of the flagging substrate, β-D-Green-Gal, which results in the formation of green color output.</P><H1><SPAN class="mw-headline" id=".CE.B2-Glucuronidase_.28GusA.29">β-Glucuronidase (GusA)</SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">3D representation of the β-Glucuronidase from <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=3K46" class="external text" rel="nofollow">RCSB</A></DIV></DIV></DIV><P align="justify">
The enzyme β-glucuronidase catalyzes the hydrolysis of D-glucuronic acids which are conjugated through a β-O-glycosidic linkage to an aglycone. The action of bacterial β-glucuronidase in the gastrointestinal tract is an important component in the enterohepatic circulation of many hydrophobic xenobiotics and endogenous waste compounds, which are conjugated to D-glucuronic acid during the main detoxification pathway of vertebrates, and are then excreted in the bile or urine (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">12</A>). The reduction of these beta-d-glucuronides by β-glucuronidase activity frees aglycone residues with protective effects, such as lignans, flavonoids, ceramide and glycyrrhetinic acid (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">13</A>), enabling their absorption and enterohepatic circulation.
β-Glucuronidase is commonly used as a reporter system for various experimental measurements, and has been favoured as a reporter gene in plants. In contrast to β-galactosidase, which is a widely used reporter protein, β-glucuronidase has a key advantage in its robustness and its absence in many organisms other than vertebrates and their attendant microflora, rendering the possibility of accurate measurements of β-glucuronidase activity (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">14</A>). 
The part we implemented in Colisweeper is <I>gusA</I> (<A href="http://parts.igem.org/Part:BBa_K1216000" class="external text" rel="nofollow">BBa_K1216000</A>), a homolog of the <I>Escherichia coli</I><I>uidA</I>, and encodes the <I>Bacillus subtilis</I> β-glucuronidase, a 68 kDa tetrameric and intracellular enzyme. This part is also available with added TEV and poly-HIS tags (<A href="http://parts.igem.org/Part:BBa_K1216004" class="external text" rel="nofollow">BBa_K1216004</A>). According to Tryland and Fiksdal (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">15</A>), the activity of the <I>Bacillus spp.</I> β-glucuronidase is similar to that of the <I>Escherichia coli</I> homolog, being only approximately 10-fold lower. In our game, β-glucuronidase catalyzes hydrolysis of 6-Chloro-3-indolyl-β-glucuronic acid, which gives rise to an orange-pink color.</P><H1><SPAN class="mw-headline" id=".CE.B2-N-Acetylglucosaminidase_.28NagZ.29">β-N-Acetylglucosaminidase (NagZ)</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">3D representation of the β-N-Acetylglucosaminidase from <A href="http://www.rcsb.org/pdb/explore/explore.do?structureId=4GVG" class="external text" rel="nofollow">RCSB</A></DIV></DIV></DIV><P align="justify"> The β-N-Acetylglucosaminidase enzyme is a cytoplasmic hydrolase which is involved in the <I>Escherichia coli</I> cell wall recycling pathway. Its natural substrates are muropeptides and anhydro-muropeptides, which are released into the cytosol from cell wall murein (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">16</A>). The enzyme is active against the β-1,4-glycosidic bonds and releases β-N-acetylglucosamine (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">17</A>). Inhibitors of NagZ are N-acetylglucosamine and N-acetylmuramic acid at high concentrations, or N-acetylglucosaminolactone (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">17</A>). Using p-nitrophenyl-β-N-acetylglucosaminide as a substrate, a kinetic assay by Vötsch and Templin (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_reference" class="external text" rel="nofollow">18</A>) showed a K<SUB>m</SUB> 310 µM and V<SUB>max</SUB> 13.3 µmol/min mg protein at 25°C.
We have added the coding region for β-N-Acetylglucosaminidase to the registry as <A href="http://parts.igem.org/Part:BBa_K1216003" class="external text" rel="nofollow">BBa_K1216003</A> and used this part for Colisweeper's final circuit in the mine cells, which produce blue color upon hydrolysis of the substrate 5-Bromo-4-Chloro-3-indoxyl-β-N-acetylglucosaminide. 
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>(1) Reymond JL, Fluxà VS, Maillard N, <I>Chem. Commun.</I>, <B>1</B>, 34 – 46 (2009).
(2) Jiang T, Xing B, Rao J, <I>Biotechnology and Genetic Engineering Reviews</I>, <B>25</B>, 41-76 (2008).
(3) Kanaya S, Koyanagi T, Kanaya E, <I>Biochem. J.</I>, <B>332</B>, 75-80 (1998).
(4) Peist R, Koch A, Bolek P, Sewitz S, Kolbus T, Boos J, <I>J. Bacteriol.</I>, <B>179</B>, 7679-7686 (1997).
(5) Blattner FR, Plunkett III G, Bloch CA, Perna NT, Burna V, Riley M, Collado-Vides J, Glasner JD, Rode CK, Mayhew GF, Gregor J, Davis NW, Kirkpatrick HA, Goeden MA, Rose DJ, Mau B, Shao Y, <I>Science</I>, <B>277</B>, 1453-1461 (1997).
(6) Haruki M, Oohashi Y, Mizuguchi S, Matsuo Y, Morikawa M, Kanaya S, <I>FEBS Lett.</I>, <B>454</B>, 262-266 (1999).
(7) Farias T, Mandrich L, Rossi M, Manco G, <I>Protein Pept. Lett.</I>, <B>14</B>, 165-169 (2007).
(8) Kobayashi R, Hirano N, Kanaya S, Haruki M, <I>Biosci. Biotechnol. Biochem.</I>, <B>76</B> (11), 2082-2088 (2012).
(9) Kim EE, Wyckoff HW, <I>J. Mol. Biol.</I>, <B>218</B>, 449-464 (1991).
(10) Yang K, Metcalf WW, <I>PNAS</I>, <B>101</B>, 7919-7924 (2004).
(11) Juers DH, Heightman TD, Vasella A, McCarter JD, Mackenzie L, Withers SG, Matthews BW, <I>Biochemistry</I>, <B>40</B>, 14781-14794 (2001).
(12) Russell WM, Klaenhammer TR, <I>Appl. Environ. Microbiol.</I>, <B>67</B>, 1253-1261 (2000).
(13) Beaud D, Tailliez P, Anba-Mondoloni J, <I>Microbiology</I>, <B>171</B>, 2323-2330 (2005).
(14) Jefferson RA, Burgess SM, Hirsh D, <I>Procl. Natl. Acad. Sci. USA</I>, <B>83</B>, 8447-8451 (1986).
(15) Tryland I, Fiksdal L, <I>Appl. Environ. Microbiol.</I>, <B>64</B>, 1018-1023 (1997).
(16) Cheng Q, Li H, Merdek K, Park JT, <I>J. Bacteriol.</I>, <B>182</B>, 4836-4840 (2000).
(17) Yem DW, Wu HC, <I>J. Bacteriol.</I>, <B>125</B>, 324-331 (1976).
(18) Vötsch W, Templin MF, <I>J. Biol. Chem.</I>, <B>275</B>, 39032-39038 (2000).</P></DIV><DIV id="eth_footer"><H1><SPAN class="mw-headline" id="We_thank_our_sponsors:">We thank our sponsors:</SPAN></H1><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Basf-logo.gif" title="File:Basf-logo.gif">File:Basf-logo.gif</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Novartis-logo.gif" title="File:Novartis-logo.gif">180px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Roche-logo.gif" title="File:Roche-logo.gif">100px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Syngenta-logo.gif" title="File:Syngenta-logo.gif">130px</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:ETH_Zurich/Experiments_7">http://2013.igem.org/Team:ETH_Zurich/Experiments_7</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Experiments_7" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Experiments_7" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_7&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_7&amp;oldid=354319" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>