<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Experiments_3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Experiments_3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Experiments_3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_3&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Experiments_3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Experiments 3</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: pointer">Project</A></LI><LI><A href="/Team:ETH_Zurich/Infoproc" title="Team:ETH Zurich/Infoproc">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Circuit" title="Team:ETH Zurich/Circuit">Circuit and optimization</A></LI><LI><A href="/Team:ETH_Zurich/pre_proc" title="Team:ETH Zurich/pre proc">Pre-processing: Signal diffusion</A></LI><LI><A href="/Team:ETH_Zurich/Processing_2" title="Team:ETH Zurich/Processing 2">Signal processing: High pass filters</A></LI><LI><A href="/Team:ETH_Zurich/Play" title="Team:ETH Zurich/Play">Player interaction: Design and playing</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_7" title="Team:ETH Zurich/Experiments 7">Output system: Hydrolases</A></LI></UL><LI><A href="/Team:ETH_Zurich/Data_Page" title="Team:ETH Zurich/Data Page">Data page</A></LI><LI><A name="Biology" style="cursor: pointer">Experimental results</A></LI><LI><A href="/Team:ETH_Zurich/Experimentalresults" title="Team:ETH Zurich/Experimentalresults">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_2" title="Team:ETH Zurich/Experiments 2">Pre-processing: Diffusion characterization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_6" title="Team:ETH Zurich/Experiments 6">Proof-of-principle with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_5" title="Team:ETH Zurich/Experiments 5">Signal processing engineering</A></LI><LI><A href="/Team:ETH_Zurich/Optimization" title="Team:ETH Zurich/Optimization">Circuit optimization</A></LI><LI><STRONG class="selflink">Output system characterization</STRONG></LI><LI><A href="/Team:ETH_Zurich/Flagging" title="Team:ETH Zurich/Flagging">Flagging</A></LI><LI><A href="/Team:ETH_Zurich/gameplayexperiment" title="Team:ETH Zurich/gameplayexperiment">Gameplay</A></LI><LI><A name="Model" style="cursor: pointer">Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Overview" title="Team:ETH Zurich/Modeling/Overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/InitialStates" title="Team:ETH Zurich/Modeling/InitialStates">Single Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Reaction_Diffusion_OOHL" title="Team:ETH Zurich/Modeling/Reaction Diffusion OOHL">AHL reaction-diffusion model</A></LI><LI><A href="/Team:ETH_Zurich/GFP" title="Team:ETH Zurich/GFP">Model Evaluation: Genetic circuit with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Genetic Circuit incorporating hydrolases</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximations" title="Team:ETH Zurich/Modeling/Analytical Approximations">Analytical Approximations</A></LI><LI><A href="/Team:ETH_Zurich/Parameter" title="Team:ETH Zurich/Parameter">Parameters</A></LI><LI><A name="LAb work" style="cursor: pointer">Lab work</A></LI><LI><A href="/Team:ETH_Zurich/Mutant" title="Team:ETH Zurich/Mutant">Triple knockout strain</A></LI><LI><A href="/Team:ETH_Zurich/Experiments" title="Team:ETH Zurich/Experiments">Constructs</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Lab journal</A></LI><LI><A href="/Team:ETH_Zurich/Materials" title="Team:ETH Zurich/Materials">Methods</A></LI><LI><A href="/Team:ETH_Zurich/Materials_2" title="Team:ETH Zurich/Materials 2">Materials</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A name="Human" style="cursor: pointer">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Practiceoverview" title="Team:ETH Zurich/Practiceoverview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/World" title="Team:ETH Zurich/World">Worldwide playing</A></LI><LI><A href="/Team:ETH_Zurich/Human" title="Team:ETH Zurich/Human">Gamification</A></LI><LI><A href="/Team:ETH_Zurich/HEducational" title="Team:ETH Zurich/HEducational">Education</A></LI><LI><A name="Team" style="cursor: pointer">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Students</A></LI><LI><A href="/Team:ETH_Zurich/Supervisors" title="Team:ETH Zurich/Supervisors">Advisors/Instructors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; contributions</A></LI><LI><A href="/Team:ETH_Zurich/References" title="Team:ETH Zurich/References">References</A></LI><LI><A name="Achievements" style="cursor: pointer">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal criteria</A></LI><LI><A href="/Team:ETH_Zurich/Awards" title="Team:ETH Zurich/Awards">Awards</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Experiments_3" class="external text" rel="nofollow">Login</A></LI><LI><A name="WIKI" style="cursor: pointer">WIKI</A></LI><LI><A href="/wiki/index.php?title=Special:UserLogout&amp;returnto=Team:ETH_Zurich/Home;returntoquery=action%3Dedit" title="Log out">Log out</A></LI><LI><A href="https://igem.org/User_Information" title="My account">My account</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Experiments_3&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Experiments_3&amp;action=history" class="external text" rel="nofollow">History</A></LI><LI><A href="http://2013.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Enzyme-substrate_reactions"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Enzyme-substrate reactions</SPAN></A></LI><LI class="toclevel-1"><A href="#Overview_of_enzyme-substrate_reactions"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Overview of enzyme-substrate reactions</SPAN></A></LI><LI class="toclevel-1"><A href="#Acetyl_esterase_.28Aes.29"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Acetyl esterase (Aes)</SPAN></A></LI><LI class="toclevel-1"><A href="#Alkaline_phosphatase_.28PhoA.29"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Alkaline phosphatase (PhoA)</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2-Galactosidase_.28LacZ.29"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">β-Galactosidase (LacZ)</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2-Glucuronidase_.28GusA.29"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">β-Glucuronidase (GusA)</SPAN></A></LI><LI class="toclevel-1"><A href="#.CE.B2-N-Acetylglucosaminidase_.28NagZ.29"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">β-N-Acetylglucosaminidase (NagZ)</SPAN></A></LI><LI class="toclevel-1"><A href="#Crosstalk"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Crosstalk</SPAN></A></LI><LI class="toclevel-1"><A href="#Color_overlay"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Color overlay</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1"><A href="#We_thank_our_sponsors:"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">We thank our sponsors:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Enzyme-substrate_reactions">Enzyme-substrate reactions</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 1: Colorimetric output.</B> This image shows liquid cultures of our mutant <I>Escherichia coli</I> strain expressing the enzymes PhoA, NagZ, LacZ, GusA or Aes to convert specific chromogenic substrates into colored outputs.</DIV></DIV></DIV><P align="justify">To generate <B>visible output</B> by adding substrates to colonies in Colisweeper, we made use of orthogonal enzyme-substrate reactions. A set of <B>chromogenic substrates</B> was chosen to produce different colors depending on the abundant hydrolases and thereby to uncover the identity of each colony to the player.

The set of enzyme-substrate pairs chosen for the Colisweeper project, and characterizations of their reactions, are described below. Information on other possible substrates that can be used for the enzymes of the Colisweeper reporter system can be found in the <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7" class="external text" rel="nofollow">reporter system</A> section.
To characterize the hydrolases used in Colisweeper, we conducted <A href="https://2013.igem.org/Team:ETH_Zurich/Materials#substrate test">substrate tests</A> in liquid cultures as well as on colonies, <A href="https://2013.igem.org/Team:ETH_Zurich/Materials#enzyme_kinetics">enzyme kinetics</A> and detection of HIS-tagged protein by Western Blot. Additionally, <B>crosstalk</B> assays and <B>color overlay</B> tests (multiple enzyme-substrate reactions together) have been performed with the triple knockout Δ<I>aes</I>Δ<I>gusA</I>Δ<I>nagZ</I> for this project. A listing of chromogenic and fluorogenic substrates used can be found in the overview table below.
</P><H1><SPAN class="mw-headline" id="Overview_of_enzyme-substrate_reactions">Overview of enzyme-substrate reactions</SPAN></H1><TABLE style="float:left;margin-top:10px;font-size:12px;font-family:verdana"><TBODY><TR><TH width="100px">Hydrolase</TH><TH width="225px">Substrate</TH><TH width="125px">Absorption λ<SUB>max</SUB> or Excitation/Emission</TH><TH width="150px">Stock solution</TH><TH width="150px">Liquid culture: final concentration</TH><TH width="150px">Colonies:1.5 μl of substrate solution</TH><TH width="150px">Response time on <I>E. coli</I> colonies, at RT</TH></TR><TR><TD rowspan="2" align="center"><A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3#lacZ" class="external text" rel="nofollow">LacZ</A></TD><TD bgcolor="#F0FAFF">5-Bromo-4-Chloro-3-indoxyl-β-D-galactopyranoside (X-Gal)</TD><TD>Blue,615 nm</TD><TD>0.5 M in DMSO</TD><TD>1 mM</TD><TD>50 mM</TD><TD><B>~ 10 minutes</B></TD></TR><TR><TD bgcolor="#E6FFCC">N-Methyl-3-indolyl-β-D-galactopyranoside (Green-Gal)</TD><TD>Green,665 nm</TD><TD>0.1 M in DMSO</TD><TD>1 mM</TD><TD>50 mM</TD><TD><B>~ 15 minutes</B></TD></TR><TR><TD rowspan="2" align="center"><A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3#gusA" class="external text" rel="nofollow">GusA</A></TD><TD bgcolor="#FFEEEE">6-Chloro-3-indolyl-β-D-glucuronide (Salmon-Glc)</TD><TD>Salmon,540 nm</TD><TD>0.3 M in DMSO</TD><TD>1.5 mM</TD><TD>0.1 M</TD><TD><B>~ 5 minutes</B></TD></TR><TR><TD> 4-MU-β-D-Glucuronide</TD><TD>Blue (fluorescent),372 nm (λ<SUB>Ex</SUB>), 445 nm (λ<SUB>Em</SUB>)</TD><TD>50 mM in DMSO</TD><TD>100 μM</TD><TD>-</TD><TD><B>~ 5 minutes</B></TD></TR><TR><TD rowspan="3" align="center"><A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3#phoA" class="external text" rel="nofollow">PhoA</A></TD><TD bgcolor="#FFFFCC">4-Nitrophenoyl-phosphate (pNPP)</TD><TD>Yellow,405 nm</TD><TD>0.5 M in DEA</TD><TD>50 mM</TD><TD>0.5 M</TD><TD><B>~ 1 minute</B></TD></TR><TR><TD bgcolor="#F0FAFF">5-Bromo-4-Chloro-3-indolyl phosphate (BCIP)</TD><TD>Blue,615 nm</TD><TD>0.1 M in H<SUB>2</SUB>O</TD><TD>1 mM</TD><TD>50 mM</TD><TD><B>~ 30 minutes</B></TD></TR><TR><TD> 4-MU-phosphate</TD><TD>Blue (fluorescent),372 nm (λ<SUB>Ex</SUB>), 445 nm (λ<SUB>Em</SUB>)</TD><TD>50 mM in DMSO</TD><TD>50 μM</TD><TD>-</TD><TD><B>~ 5 minutes</B></TD></TR><TR><TD rowspan="3" align="center"><A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3#aes" class="external text" rel="nofollow">Aes</A></TD><TD bgcolor="#F5CCE0">5-Bromo-6-Chloro-3-indoxyl butyrate (Magenta butyrate)</TD><TD>Magenta,565 nm</TD><TD>0.5 M in Acetone</TD><TD>0.1 mM</TD><TD>20 mM</TD><TD><B>~ 2 minutes</B></TD></TR><TR><TD bgcolor="#F5CCE0">5-Bromo-6-Chloro-3-indoxyl caprylate (Magenta caprylate)
</TD><TD>Magenta,565 nm</TD><TD>0.2 M in Acetone</TD><TD>1 mM</TD><TD>0.2 M</TD><TD><B>overnight</B></TD></TR><TR><TD> 4-MU-butyrate</TD><TD>Blue (fluorescent),372 nm (λ<SUB>Ex</SUB>), 445 nm (λ<SUB>Em</SUB>)</TD><TD>50 mM in DMSO</TD><TD>150 μM</TD><TD>-</TD><TD><B>~ 5 minutes</B></TD></TR><TR><TD rowspan="3" align="center"><A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3#nagZ" class="external text" rel="nofollow">NagZ</A></TD><TD bgcolor="#FFFFCC">4-Nitrophenyl- N-acetyl-β-D-glucosaminide (pNP-GluNAc)</TD><TD>Yellow,405 nm</TD><TD>15 mM in H<SUB>2</SUB>O</TD><TD>0.01 mM</TD><TD>15 mM</TD><TD><B>~ 1 minute</B></TD></TR><TR><TD bgcolor="#F5CCE0">5-Bromo-6-Chloro-3-indolyl N-acetyl-β-D-glucosaminide (Magenta GluNAc)
</TD><TD>Magenta,565 nm</TD><TD>0.1 M in DMSO</TD><TD>1 mM, supplemented with BSA</TD><TD>50 mM, with BSA added</TD><TD><B>~ 15 minutes</B></TD></TR><TR><TD bgcolor="#F0FAFF">5-Bromo-4-Chloro-3-indolyl N-acetyl-β-D-glucosaminide (X-GluNAc)
</TD><TD>Blue,615 nm</TD><TD>0.1 M in DMSO</TD><TD>1 mM, supplemented with BSA</TD><TD>50 mM, with BSA added</TD><TD><B>~ 15 minutes</B></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Acetyl_esterase_.28Aes.29">Acetyl esterase (Aes)</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 2.</B>  Liquid cultures of the Δ<I>aes</I>Δ<I>gusA</I>Δ<I>nagZ</I> strain overexpressing Aes. Left: magenta-butyrate added; right: no substrate added.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 3.</B> Colonies of the triple knockout <I>Escherichia coli</I> strain overexpressing the Aes. Colonies produce magenta color after addition of magenta butyrate.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:287px;"><DIV class="thumbcaption"><B>Figure 5. Background activity.</B> Images taken 5 minutes, 6 hours and 12 hours after addition of various concentrations of magenta butyrate onto colonies of the triple knockout strain overexpressing (left) and not overexpressing (right) Aes.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B> Figure 4.</B> Colonies of the triple knockout <I>Escherichia coli</I> strain overexpressing the hydrolase Aes. Upon addition of magenta caprylate, a magenta color is produced after approximately 30 minutes incubation at 37°C.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:742px;"><DIV class="thumbcaption"><B>Figure 6.</B> Aes catalyzed hydrolysis reaction of magenta butyrate or magenta caprylate.</DIV></DIV></DIV><P align="justify"><B>Chromogenic assay</B>To assess color development after reaction of the enzyme with the chromogenic substrate, a liquid culture of our triple knockout <I>Escherichia coli</I> strain overexpressing Aes (<A href="http://parts.igem.org/Part:BBa_K1216002" class="external text" rel="nofollow">BBa_K1216002</A>) was grown until an OD<SUB>600</SUB> of 0.4 - 0.6 was reached before addition of 5-Bromo-6-Chloro-3-indolyl butyrate (magenta butyrate, dissolved in acetone), a magenta color yielding substrate for Aes (Figure 6), to a final concentration of 100 µM (Figure 2). To study the color development in the actual Colisweeper game setup, colonies were plated by pipetting 1.5 µl of the triple knockout <I>Escherichia coli</I> liquid culture (OD<SUB>600</SUB> of 0.4 - 0.6) on an M9 agar plate and incubated overnight. Addition of 1.5 µl of 20 mM magenta butyrate onto each colony results in color generation visible after two to three minutes at room temperature (Figure 3).
Unfortunately, substrate tests in our triple knockout strain without Aes overexpression have shown that when using the butyrate as a substrate, there is background activity observable, meaning that enzymes other than the Aes can catalyze hydrolysis of the butyrate substrate. Therefore, 5-Bromo-6-Chloro-3-indolyl caprylate (magenta caprylate) was chosen as an alternative substrate, which has shown to be more specifically cleaved by the Aes in our knockout strain. However, when using magenta caprylate as the substrate, color on colonies developed only after at least half an hour of incubation at 37°C, or overnight at room temperature (Figure 4).</P><P align="justify">Because the player of Colisweeper is not expected to incubate plates after each move, we made an assay comparing time differences of the color development in colonies of the triple knockout <I>Escherichia coli</I> strain overexpressing and not overexpressing Aes, using different concentrations of the substrates as well (Figure 5). Results have shown that with the butyrate substrate (20 mM in DMSO) and at room temperature (RT), colonies overexpressing Aes already developed color within the first few minutes, whereas colonies not overexpressing Aes did not show color until six hours after addition of the substrate. Even at twelve hours after adding the substrate, only a faint color was observed in the colonies not overexpressing Aes. Therefore, we continued using magenta butyrate for further tests and finally for the game, due to speed and intensity of color development compared to magenta caprylate.</P><P><B>Enzyme kinetics</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 7.</B> Cell lysate of the triple knockout <I>Escherichia coli</I> strain overexpressing Aes after reaction with 4-MU-butyrate.</DIV></DIV></DIV><P align="justify">For the kinetics assay, the <I>Escherichia coli</I> lysate overexpressing Aes was tested with the substrate 4-MU-butyrate which after cleavage gives a fluorescent product. The picture on the right (Figure 7) was taken with a common single lens reflex camera mounted on a dark hood at λ<SUB>Ex</SUB> 365 nm.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 8.  Enzymatic reaction of Aes with 4-MU-butyrate.</B></DIV></DIV></DIV></DIV><P align="justify">To characterize the enzyme, we conducted Michaelis Menten kinetics to obtain the K<SUB>m</SUB> value. <I>Escherichia coli</I> cells were harvested and lysed, and the cell free extract (CFX) was then collected for the fluorometric assay. The properly diluted CFX was measured on a 96 well plate in triplicates per substrate concentration. The plate reader took measurements at λ<SUB>Ex</SUB> 365 nm and λ<SUB>Em</SUB> 445 nm and determined a K<SUB>m</SUB> value of 31.5 ± 12.5 μM. The obtained data was then evaluated and fitted to Michaelis-Menten-Kinetics with SigmaPlot™:</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 9.</B>  Michaelis-Menten-Kinetics of cell lysate from <I>Escherichia coli</I> overexpressing Aes with the chromogenic substrate: plots velocity versus substrate concentration (10 μL, 30 μL, 100 μL, 200 μL, 400 μL) in 20 mM Tris buffer of pH 8. A kinetic value for K<SUB>m</SUB> obtained by fitting the raw data to standard the Michaelis Menten equation; K<SUB>m</SUB> = 31.5 ± 12.5 μM. All assays were carried out in triplicates, results are presented as means.</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Alkaline_phosphatase_.28PhoA.29">Alkaline phosphatase (PhoA)</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 10.</B>  Liquid culture of the triple knockout <I>Escherichia coli</I> strain overexpressing PhoA or PhoA-His respectively after reacting with pNPP. The negative control tube contains liquid culture without pNPP.</DIV></DIV></DIV><P align="justify"><B>Chromogenic assay</B>To test the functionality of PhoA (<A href="http://parts.igem.org/Part:BBa_K1216001" class="external text" rel="nofollow">BBa_K1216001</A>) and our improved biobrick PhoA-His (<A href="http://parts.igem.org/Part:BBa_K1216001" class="external text" rel="nofollow">BBa_K1216005</A>), a liquid culture of the triple knockout <I>Escherichia coli</I> strain overexpressing <I>phoA</I> or <I>phoA-His</I> was grown until an OD<SUB>600</SUB> of 0.4 - 0.6 was reached before addition of <I>p</I>-nitrophenyl phosphate (pNPP, dissolved in DEA) to a final concentration of 5 mM. This substrate gives rise to yellow color after hydrolysis by PhoA (Figure 10). In the Colisweeper game, substrates are pipetted onto colonies. Figure 10 shows color development on colonies five minutes after addition of 1.5 µl of a 0.5 M pNPP solution. The <I>phoA</I> gene is not a knockout in our host strain, resulting in the expression of additional chromosomal PhoA in all our cells. In our setup this is no problem, because the yellow color will be overlayed by either blue (mine), salmon and violett (mines nearby) if the cell is not a safe cell. 
However, problems with the blue color-yielding substrate for NagZ (expressed in mine colonies) have led to the idea to exchange the colors used for PhoA and NagZ. 5-Bromo-4-Chloro-3-indolyl phosphate (BCIP), which is commonly used for detection of alkaline phosphatases in biochemistry, would have been the blue color-producing alternative substrate for PhoA.</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 11.</B>  Liquid culture of the triple knockout <I>Escherichia coli</I> strain overexpressing PhoA after reacting with BCIP. The negative control tube contains liquid culture without BCIP.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 12.</B> Colonies of the triple knockout <I>Escherichia coli</I> strain overexpressing the <I>Citrobacter</I> PhoA. Colonies produce yellow color within 1 minute at RT after addition of pNPP.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 15. PhoA-HIS-tag characterization by western blotting.</B> The red stripe indicates the anti-anti-HIS antibody carrying the red fluorescent dye. The white circles indicate the PhoA-HIS protein in the SDS-PAGE gel as well as on the western blot membrane. The molecular weight of this protein is 53kDa.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 13.</B> Colonies of the triple knockout <I>Escherichia coli</I> overexpressing the <I>Citrobacter</I> PhoA. Colonies produce blue color after addition of BCIP.</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:682px;"><DIV class="thumbcaption"><B>Figure 14.</B>  Reaction of PhoA catalyzed hydrolysis of pNPP (1) and BCIP (2).</DIV></DIV></DIV></DIV><P><B>Characterization of PhoA-HIS tagged protein</B></P><P align="justify">We introduced a HIS tag to the PhoA enzyme. The HIS tag enables to purify the enzyme and therby e.g.  making a full Michalis Menten studies which can give exact values for the model. Adding of a HIS tag can lead to an unfunctional enzyme, so we first tested the functionality of both PhoA and PhoA HIS tag (see Figure 10) with our chromogenic substrate. In a second step, we confirmed the presents of the HIS tag by Western Blot: The PhoA and PhoA with HIS-tag were overexpressed in the <I>Escherichia coli</I> BL21 DE3 strain, induced by 5 μM IPTG (iso-propy-β-D-1-thiogalactopyranoside) and finally harvested in order to obtain the cell lysate by lysozyme lysis. This cell lysate of both cultures were analyzed next to each other in two SDS-PAGE gels, one for comassie staining (blue gel in the picture) and one for western blotting (black picture) with Anti-6X His tag® antibody from mouse and a second reporter goat anti mouse antibody with a IRDye 680RD. In the picture we can distinguish the PhoA His (53 kDa) on the western blot as well as on the SDS-PAGE gel (see white circles).</P><P><B>Enzyme kinetics</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 16.</B>  Cell lysate from <I>Escherichia coli</I> overexpressing PhoA-HIS (left) and PhoA (right) after reaction with 4-MU-phosphate.</DIV></DIV></DIV><P align="justify">For the kinetics assay, we tested both the <I>Citrobacter</I><I>phoA</I> and <I>phoA</I>-HIS encoded proteins with the fluorescent substrate 4-MU-phosphate. The image in Figure 16 was taken with a common single lens reflex camera mounted on a dark hood at λ<SUB>Ex</SUB> 365 nm.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:394px;"><DIV class="thumbcaption"><B>Figure 17.  Enzymatic reaction of PhoA with 4-MU-phosphate.</B></DIV></DIV></DIV></DIV><P align="justify">To characterize the enzymes, we conducted fluorometric assays to obtain K<SUB>m</SUB> values. <I>Escherichia coli</I> were harvested and lysed, and the cell free extract (CFX) was then collected for the fluorometric assay. The properly diluted CFX was measured on a 96 well plate in triplicates per substrate concentration. The plate reader took measurements at λ<SUB>Ex</SUB> 365 nm and λ<SUB>Em</SUB> 445 nm, determining a K<SUB>m</SUB> values of 105.9 ± 5.3 μM for PhoA and 72.0 ± 7.3 μM for Pho-HIS.
The obtained data was evaluated and finally fitted to Michaelis-Menten-Kinetics with SigmaPlot™:</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:928px;"><DIV class="thumbcaption"><B>Figure 18.</B>  Michaelis-Menten-Kinetics of cell lysate from <I>Escherichia coli</I> overexpressing the <I>Citrobacter</I> PhoA or PhoA-HIS: plots velocity versus substrate concentration (2.5 μL, 5 μL, 10 μL, 30 μL, 60 μL, 120 μL, 240 μL) in 20 mM Tris buffer of pH 8. A kinetic value for K<SUB>m</SUB> obtained by fitting the raw data to standard the Michaelis Menten equation; K<SUB>m</SUB> (PhoA) = 105.9 ± 5.3 μM and K<SUB>m</SUB> (PhoA-HIS) = 72.0 ± 7.3 μM. All assays were carried out in triplicates, results are presented as means.</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id=".CE.B2-Galactosidase_.28LacZ.29">β-Galactosidase (LacZ)</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 19.</B>  Liquid cultures of the triple knockout <I>Escherichia coli</I> strain; the natively expressed LacZ reacts with the added substrates Green-Gal (left) and X-Gal (middle). The negative control (right) contains liquid culture without substrates added.</DIV></DIV></DIV><P align="justify"><B>Chromogenic assay</B>The β-Galactosidase is natively expressed in the triple knockout <I>Escherichia coli</I> strain used in Colisweeper. Its commonly used chromogenic substrate is 5-Bromo-4-Chloro-3-indolyl-β-D-Galactopyranoside (X-Gal), which yields a blue-colored precipitate (Figure 19, middle). In Colisweeper, this protein catalyzes hydrolysis of the flagging substrate, which is N-Methyl-3-indolyl-β-D-Galactopyranoside (Green-β-D-Gal) and produces a green color upon cleavage of the chromophore (Figure 19, left tube). For the color assay in liquid culuture, we used <I>Escherichia coli</I> cultures grown to an OD<SUB>600</SUB> of 0.4 - 0.6, then added the substrates to a final concentration of 1 mM. On colonies, drops of 50 mM were applied, and development of color showed approximately 10 - 15 minutes after addition of the substrates.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:752px;"><DIV class="thumbcaption"><B>Figure 20.</B> Hydrolysis of Green-β-D-Gal (1) or X-Gal (2), catalyzed by LacZ.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 21.</B> Colonies of our triple knockout strain <I>Escherichia coli</I> in the hexagonal grid setup. Colonies natively produce LacZ, and addition of X-Gal gives rise to green color.</DIV></DIV></DIV><H1><SPAN class="mw-headline" id=".CE.B2-Glucuronidase_.28GusA.29">β-Glucuronidase (GusA)</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 22.</B>  Liquid culture of the triple knockout <I>Escherichia coli</I> strain overexpressing GusA; after reaction with Salmon-Gluc. The negative control (right) is a liquid culture without the substrate added.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 23.</B>  Colonies of the triple knockout <I>Escherichia coli</I> strain overexpressing the <I>Bacillus subtilis</I> GusA. Colonies produce salmon color after adding substrate.</DIV></DIV></DIV><P align="justify"><B>Chromogenic assay</B>In the Colisweeper game, GusA is expressed in colonies which are surrounded by one mine. To characterize the reaction of GusA with its substrate, a liquid culture of the triple knockout <I>Escherichia coli</I> strain overexpressing GusA (<A href="http://parts.igem.org/Part:BBa_K1216000" class="external text" rel="nofollow">BBa_K1216000</A>) was grown until an OD<SUB>600</SUB> of 0.4 - 0.6 was reached before addition of 6-Chloro-3-indolyl-β-glucuronide (Salmon-Gluc), a substrate for GusA which produces a rose/salmon color after cleavage (Figure 22). The final concentration of the substrate in liquid culture was 1 mM and the conversion of this substrate was done by incubation at 37°C. On colonies, 1.5 μl drops of the diluted substrate solution were added, resulting in production of salmon color after approximately 5 minutes at RT.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 24.</B>  Enzymatic reaction of GusA with the chromogenic substrate Salmon-Gluc.</DIV></DIV></DIV></DIV><P><B>Enzyme kinetics</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 25.</B>  Cell lysate from <I>Escherichia coli</I> overexpressing GusA, after reaction with 4-MU-β-D-Glucuronide.</DIV></DIV></DIV><P align="justify">For the kinetics assay, we tested both <I>gusA</I> and <I>gusA</I>-HIS encoded proteins with the fluorescent substrate 4-MU-β-D-Glucuronide. The picture on the right (Figure 25) was taken with a common single lens reflex camera mounted on a dark hood at λ<SUB>Ex</SUB> 365 nm.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:468px;"><DIV class="thumbcaption"><B>Figure 26.</B>  Enzymatic reaction of GusA with the fluorogenic substrate 4-MU-β-D-Glucuronide.</DIV></DIV></DIV></DIV><P align="justify">For the enzyme kinetics of GusA, we conducted fluorometric assays to obtain the K<SUB>m</SUB> value. <I>Escherichia coli</I> overexpressing GusA were harvested and lysed, and the cell free extract (CFX) was then collected for the fluorometric assay. The properly diluted CFX was measured on a 96 well plate in triplicates per substrate concentration. A plate reader took measurements at λ<SUB>Ex</SUB> 365 nm and λ<SUB>Em</SUB> 445 nm, and determined a K<SUB>m</SUB> value of 141.1 ± 5.3 μM for GusA, and 201.9 ± 51.1 μM for GusA-HIS.
The obtained data was evaluated and finally fitted to Michaelis-Menten-Kinetics with SigmaPlot™:</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:912px;"><DIV class="thumbcaption"><B>Figure 27.  Michaelis-Menten-Kinetics of <I>Escherichia coli</I> overexpressing GusA and GusA-HIS:</B> plots velocity versus substrate concentration (8 μL, 16 μL, 32 μL, 65 μL, 130 μL, 260 μL, 520 μL)) in 20 mM Tris buffer of pH 8. A kinetic value for K<SUB>m</SUB> obtained by fitting the raw data to standard the Michaelis Menten equation; K<SUB>m</SUB> (GusA) = 141.1 ± 5.3 μM and K<SUB>m</SUB> (GusA-HIS) = 201.9 ± 51.1 μM. All assays were carried out in triplicates, results are presented as means.</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id=".CE.B2-N-Acetylglucosaminidase_.28NagZ.29">β-N-Acetylglucosaminidase (NagZ)</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 28.</B>  Liquid culture of the triple knockout <I>Escherichia coli</I> strain overexpressing NagZ; after reaction with pNP-GluNAc (yellow), Magenta-GluNAc (magenta) and X-GluNAc (blue). The negative control is a liquid culture in M9 medium without any substrate added.</DIV></DIV></DIV><P align="justify"><B>Chromogenic assay</B>NagZ (<A href="http://parts.igem.org/Part:BBa_K1216003" class="external text" rel="nofollow">BBa_K1216003</A>) was proposed to be constitutively expressed in the mine colonies, processing its blue color-yielding substrate 5-Bromo-4-Chloro-3-indolyl-β-N-acetylglucosaminide (X-GluNAc). However, we could not initially manage to get color from the reaction with this enzyme-substrate pair. Using p-nitrophenyl-β-N-acetylglucosaminide (pNP-GluNAc) at an end concentration of 0.01 mM in an <I>Escherichia coli</I> liquid culture with an OD<SUB>600</SUB> of 0.4 - 0.6, development of yellow color could be observed (Figure 28), confirming that NagZ is expressed in our mine cells.
According to Orenga et al. (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3#hydrolase_reference" class="external text" rel="nofollow">1</A>), the substrate incorporating the indoxyl chromophore is less preferentially hydrolyzed by bacterial NagZ in comparison to yeast NagZ, and more preferentially hydrolyzes the alizarine-based substrate they applied. To increase hydrolysis of the indoxyl-based substrate, we added the substrate to a liquid culture supplemented with BSA for stabilization of the enzyme (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3#hydrolase_reference" class="external text" rel="nofollow">2</A>), which enhanced hydrolysis by NagZ dramatically. The end concentration in liquid culture of both X-GluNAc and 5-Bromo-6-Chloro-3-indolyl-β-N-acetylglucosaminide (Magenta-GluNAc), a magenta color-yielding substrate for NagZ, were at 1 mM (Figure 28).</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><B>Figure 29.</B>  Colonies of the triple knockout <I>Escherichia coli</I> strain overexpressing NagZ; after addition of the NagZ substrates pNP-GluNAc (yellow colonies), Magenta-GluNAc (pink colonies) and X-GluNAc (blue colonies). This image was taken only 15 minutes after the addition of substrates.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Figure 30.</B> β-N-Acetylglucosaminidase catalyzed hydrolysis of the chromogenic substrates X-β-N-acetylglucosaminide (1), Magenta-GluNAc (2) and pNP-GluNAc (3).</DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Crosstalk">Crosstalk</SPAN></H1><P align="justify">To ensure specificity of the enzyme-substrate pairs used in Colisweeper, a crosstalk test was done to make sure that all overexpressed enzymes specifically cleave their assigned substrate.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 31.</B> Liquid cultures of the Δ<I>aes</I>Δ<I>gusA</I>Δ<I>nagZ</I><I>Escherichia coli</I> strain overexpressing Aes, GusA, NagZ or none in a 96-well plate, with substrates indicated on the left added horizontally.</DIV></DIV></DIV><P align="justify">This crosstalk test was done in a 96-well plate, each well containing 200 μl from liquid cultures of our Δ<I>aes</I>Δ<I>gusA</I>Δ<I>nagZ</I><I>Escherichia coli</I> strain overexpressing either Aes, GusA, NagZ or none, each distributed among the column-wells of the plate. Horizontally, the chromogenic substrates were pipetted to the liquid cultures in the same order as their corresponding hydrolase: magenta-butyrate for Aes, Salmon-Gluc for GusA, and magenta-, X- and pNP-GluNAc for NagZ. If specificity of the chosen enzyme-substrates pairs were given, we would expect an output as shown in the figure below (Figure 32). As Figure 31 shows, the overexpressed hydrolases cleave only the substrates they were expected to.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:172px;"><DIV class="thumbcaption"><B>Figure 32. Expected outcome.</B> Added substrates should be specifically cleaved by their hydrolases.</DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Color_overlay">Color overlay</SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 33.</B> Mixture of liquid cultures of the Δ<I>aes</I>Δ<I>gusA</I>Δ<I>nagZ</I><I>Escherichia coli</I> strain overexpressing hydrolases after addition of their substrates: PhoA with pNPP in (1), PhoA with pNPP and GusA with Salmon-Gluc in (2), PhoA with pNPP, GusA with Salmon-Gluc and Aes with Magenta butyrate in (3), and PhoA with pNPP and NagZ with X-GluNAc in (4).</DIV></DIV></DIV><P align="justify">To play Colisweeper, colors for each colony identity have to be clearly distinguishable by eye. Because we are using high-pass filters to differentially process certain AHL levels, we had to make sure that a mix of cleaved product would show distinguishable colors.</P><P align="justify">For this purpose, we mixed liquid cultures of our Δ<I>aes</I>Δ<I>gusA</I>Δ<I>nagZ</I> triple knockout strain which overexpress PhoA, GusA, Aes or NagZ, and added the corresponding substrates to the cultures. Figure 33 shows the resulting colors after hydrolysis of the substrates: PhoA with pNPP in (1), PhoA with pNPP and GusA with Salmon-Gluc in (2), PhoA with pNPP, GusA with Salmon-Gluc and Aes with Magenta butyrate in (3), and PhoA with pNPP and NagZ with X-GluNAc in (4). These colors are clearly distinguishable from one another and therefore can be used together as reporters of a system incorporating high-pass filters.</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>(1) Orenga S, Roger-Dalbert C, James A, Perry J, US Patent 20090017481 (2009).
(2) Magnelli PE, Bielik A, Guthrie E, <I>Methods Mol. Biol.</I>, <B>801</B>, 189-211 (2011).
Absorption: <A href="http://www.biosynth.com/index.asp?topic_id=115&amp;g=19&amp;m=236">Biosynth</A></P></DIV><DIV id="eth_footer"><H1><SPAN class="mw-headline" id="We_thank_our_sponsors:">We thank our sponsors:</SPAN></H1><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Basf-logo.gif" title="File:Basf-logo.gif">File:Basf-logo.gif</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Novartis-logo.gif" title="File:Novartis-logo.gif">180px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Roche-logo.gif" title="File:Roche-logo.gif">100px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Syngenta-logo.gif" title="File:Syngenta-logo.gif">130px</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:ETH_Zurich/Experiments_3">http://2013.igem.org/Team:ETH_Zurich/Experiments_3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Experiments_3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Experiments_3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_3&amp;oldid=360664" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>