<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Circuit"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Circuit">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Circuit&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Circuit&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Circuit&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Circuit" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Circuit</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: pointer">Project</A></LI><LI><A href="/Team:ETH_Zurich/Infoproc" title="Team:ETH Zurich/Infoproc">Information processing</A></LI><LI><STRONG class="selflink">Circuit and optimization</STRONG></LI><LI><A href="/Team:ETH_Zurich/pre_proc" title="Team:ETH Zurich/pre proc">Pre-processing: Signal diffusion</A></LI><LI><A href="/Team:ETH_Zurich/Processing_2" title="Team:ETH Zurich/Processing 2">Signal processing: High pass filters</A></LI><LI><A href="/Team:ETH_Zurich/Play" title="Team:ETH Zurich/Play">Player interaction: Design and playing</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_7" title="Team:ETH Zurich/Experiments 7">Output system: Hydrolases</A></LI></UL><LI><A href="/Team:ETH_Zurich/Data_Page" title="Team:ETH Zurich/Data Page">Data page</A></LI><LI><A name="Biology" style="cursor: pointer">Experimental results</A></LI><LI><A href="/Team:ETH_Zurich/Experimentalresults" title="Team:ETH Zurich/Experimentalresults">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_2" title="Team:ETH Zurich/Experiments 2">Pre-processing: Diffusion characterization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_6" title="Team:ETH Zurich/Experiments 6">Proof-of-principle with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_5" title="Team:ETH Zurich/Experiments 5">Signal processing engineering</A></LI><LI><A href="/Team:ETH_Zurich/Optimization" title="Team:ETH Zurich/Optimization">Circuit optimization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_3" title="Team:ETH Zurich/Experiments 3">Output system characterization</A></LI><LI><A href="/Team:ETH_Zurich/Flagging" title="Team:ETH Zurich/Flagging">Flagging</A></LI><LI><A href="/Team:ETH_Zurich/gameplayexperiment" title="Team:ETH Zurich/gameplayexperiment">Gameplay</A></LI><LI><A name="Model" style="cursor: pointer">Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Overview" title="Team:ETH Zurich/Modeling/Overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/InitialStates" title="Team:ETH Zurich/Modeling/InitialStates">Single Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Reaction_Diffusion_OOHL" title="Team:ETH Zurich/Modeling/Reaction Diffusion OOHL">AHL reaction-diffusion model</A></LI><LI><A href="/Team:ETH_Zurich/GFP" title="Team:ETH Zurich/GFP">Model Evaluation: Genetic circuit with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Genetic Circuit incorporating hydrolases</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximations" title="Team:ETH Zurich/Modeling/Analytical Approximations">Analytical Approximations</A></LI><LI><A href="/Team:ETH_Zurich/Parameter" title="Team:ETH Zurich/Parameter">Parameters</A></LI><LI><A name="LAb work" style="cursor: pointer">Lab work</A></LI><LI><A href="/Team:ETH_Zurich/Mutant" title="Team:ETH Zurich/Mutant">Triple knockout strain</A></LI><LI><A href="/Team:ETH_Zurich/Experiments" title="Team:ETH Zurich/Experiments">Constructs</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Lab journal</A></LI><LI><A href="/Team:ETH_Zurich/Materials" title="Team:ETH Zurich/Materials">Methods</A></LI><LI><A href="/Team:ETH_Zurich/Materials_2" title="Team:ETH Zurich/Materials 2">Materials</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A name="Human" style="cursor: pointer">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Practiceoverview" title="Team:ETH Zurich/Practiceoverview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/World" title="Team:ETH Zurich/World">Worldwide playing</A></LI><LI><A href="/Team:ETH_Zurich/Human" title="Team:ETH Zurich/Human">Gamification</A></LI><LI><A href="/Team:ETH_Zurich/HEducational" title="Team:ETH Zurich/HEducational">Education</A></LI><LI><A name="Team" style="cursor: pointer">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Students</A></LI><LI><A href="/Team:ETH_Zurich/Supervisors" title="Team:ETH Zurich/Supervisors">Advisors/Instructors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; contributions</A></LI><LI><A href="/Team:ETH_Zurich/References" title="Team:ETH Zurich/References">References</A></LI><LI><A name="Achievements" style="cursor: pointer">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal criteria</A></LI><LI><A href="/Team:ETH_Zurich/Awards" title="Team:ETH Zurich/Awards">Awards</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Circuit" class="external text" rel="nofollow">Login</A></LI><LI><A name="WIKI" style="cursor: pointer">WIKI</A></LI><LI><A href="/wiki/index.php?title=Special:UserLogout&amp;returnto=Team:ETH_Zurich/Home;returntoquery=action%3Dedit" title="Log out">Log out</A></LI><LI><A href="https://igem.org/User_Information" title="My account">My account</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Circuit&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Circuit&amp;action=history" class="external text" rel="nofollow">History</A></LI><LI><A href="http://2013.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Biological_circuit_design"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Biological circuit design</SPAN></A></LI><LI class="toclevel-1"><A href="#Circuit_explanation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Circuit explanation</SPAN></A></LI><LI class="toclevel-1"><A href="#Circuit_optimization"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Circuit optimization</SPAN></A></LI><LI class="toclevel-1"><A href="#We_thank_our_sponsors:"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">We thank our sponsors:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Biological_circuit_design"> Biological circuit design</SPAN></H1><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:802px;"><DIV class="thumbcaption"><B>Figure 1: Overview of different processing states in mines and non mines that lead to different colored outputs</B></DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Circuit_explanation">Circuit explanation</SPAN></H1><P><B>Signal production by mine cells (sender cells)</B></P><P align="justify">Mine colonies constitutively express the &quot;mine-cell reporter” hydrolase <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_nagz" class="external text" rel="nofollow"><I>nagZ</I></A> that reacts with the specific substrate in the multi-substrate solution to produce a chromogenic product. The <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3" class="external text" rel="nofollow">colony turns blue</A>, indicating that the colony is a mine and hence the game ends. At the same time, the mine colony is a sender module of <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_2" class="external text" rel="nofollow">AHL</A>. LuxI is expressed constitutively and synthesizes the AHL signaling molecule. This means that the mine continuously produces AHL, that <A href="https://2013.igem.org/Team:ETH_Zurich/Modeling/Reaction_Diffusion_OOHL" class="external text" rel="nofollow">radially diffuses</A> out to neighboring colonies. The signaling molecule AHL is <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5" class="external text" rel="nofollow">processed</A> by the receiver cells (also see below).</P><P><B>Information processing in non-mine cells (receiver cells) </B>
Processing the AHL</P><P align="justify"><B>1.</B> Receiver cells respond to low AHL levels: If one mine cell is close to a non-mine colony and AHL diffusion takes place to reach the non-mine colony. Due to a high-pass filter using the LuxR system in the non-mine colony, low concentrations of AHL are detected and activate a reporter. Addition of the multi-substrate to this colony produces a salmon color, which indicates one mine is adjacent to the colony just played.The wild type promoter <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">BBa_R0062 P<SUB>LuxR</SUB></A> is used for the detection of low AHL concentrations and therefore becomes activated when one mine cell is in the vicinity of the receiver cell. Then the wild type promoter induces the expression of <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_gusa" class="external text" rel="nofollow"><I>gusA</I></A>, a hydrolase able to convert a specific substrate to a <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3" class="external text" rel="nofollow">visible red output</A>. </P><P align="justify"><B>2.</B>  Receiver cells respond to high AHL levels: If there are two or more mine colonies adjacent to a non-mine colony, a higher concentration of diffused AHL will be processed by the receiver colony. The LuxR system works as a high-pass filter again and induces the expression of another reporter to show that two or more mines surround the selected colony. Addition of the multi-substrate to this colony produces a magenta color which indicates that more than one mine is adjacent to this colony.The mutated promoter <A href="http://parts.igem.org/Part:BBa_K1216007?title=Part:BBa_K1216007" class="external text" rel="nofollow">P<SUB>LuxR</SUB> BBa_K1216007</A> was not sensitive enough for our system, therefore we did <B>partial sensitivity recovery</B> by rational design to cover all possible &quot;backmutations&quot; (7 different promoters) of the <A href="http://parts.igem.org/Part:BBa_K1216007?title=Part:BBa_K1216007" class="external text" rel="nofollow">P<SUB>LuxR</SUB> BBa_K1216007</A>. The model predicted from EC<SUB>50</SUB> that the Single Mutant 1 (SM1) would fit to the required promoter characteristics (predicted by the analytical solution for the EC<SUB>50</SUB>) and will be used in our final set-up to only respond to high concentrations of AHL and therefore be activated, when two sender cells are surrounding the receiver cell, and expressing the <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_aes" class="external text" rel="nofollow"><I>aes</I></A> hydrolase responsible for the colorimetric conversion of the specific substrate to a <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_3" class="external text" rel="nofollow">visible purple output</A>.</P><P align="justify"><B>3.</B>  Receiver cells signal with no mines in their vicinity.
If there is no mine adjacent to a non-mine colony, no AHL is processed. Only the constitutive hydrolase <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_phoa" class="external text" rel="nofollow"><I>phoA</I></A> is expressed, which upon addition of the multi-substrate will produce a chromogenic compound. This indicates that there are no mines adjacent to this colony and the game can be continued.<B>Flagging</B> Placing a flag on a mine prevents the mine from detonating and is a visual help for the player.
<B>Option 1 :</B> We  have implemented this feature of the computer game in Colisweeper. If the player is sure about a colony to be a mine, a second single substrate can be added on to this colony. All colonies express the native hydrolase <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7#hydrolase_lacz" class="external text" rel="nofollow"><I>lacZ</I></A> reporter for flagging, a green colored product indicates a flagged colony. But remember, flagging does not reveal any information about mines in neighboring colonies.<B>Option 2:</B> We implemented a second way of flagging, which is in contrast to the LacZ flagging, removable. This is done by addind a <B>Remazol blue dye</B> to the colony which immediately became blue. The color can be removed <B>within minutes</B> by adding a <B>laccase</B>.</P><H1><SPAN class="mw-headline" id="Circuit_optimization">Circuit optimization</SPAN></H1><P align="justify">
Already the data from the first experiments with GFP showed that the most important challenge in our circuit design would be to solve the problem of basal reporter expression and accumulation in the receiver cells. The reporter expression has to be minimal in the case where no mine colonies are around. Especially because we use an enzymatic reporter system with a multiple turnover of substrate this is crucial. With the <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_6" class="external text" rel="nofollow">GFP proof of concept</A> experiments we could show that the basal expression is in fact quite low already with the simple P<SUB>Lac</SUB>-LuxR-P<SUB>LuxR</SUB>-GFP receiver cell set-up. But early experiments replacing GFP with a hydrolase showed that the leakiness of the system is nevertheless too high. Hydrolase reporters are enzymatic systems. Multiple rounds of catalysis provide intrinsic signal amplification when compared to GFP. Therefore we observe a much higher sensitivity to basal expression. Moderate hydrolase expression levels can already result in total substrate conversion and signal saturation. In a first set of experiments we tried to identify the source of the leaky basal expression. Right after that we designed different variations of the first circuit that could be used to minimize the leakiness. We came up with three different strategies to approach the problem.  For details and results please see the results page <A href="https://2013.igem.org/Team:ETH_Zurich/Optimization" class="external text" rel="nofollow">here</A>. </P><P><B>1. Introduction of a positive feedback loop to reduce LuxR </B>The first idea was to replace the constitutive LuxR expression with an AHL dependent induction. Using a self-regulation motif, only a residual amount of LuxR is present in the non-induced state, minimizing its interaction with the promoter. Upon induction the LuxR protein sustains its own production allowing for a full activation of the system. To achieve autoregulation one possibility is to use a direct positive feedback loop in which the luxR gene is placed under its own P<SUB>LuxR</SUB> promoter (<A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">BBa_R0062</A>)(Figure 2, center). Only upon induction with AHL LuxR would be produced. As an alternative we could use LacI expression to inhibit LuxR production in the uninduced state. By placing the lacI (<A href="http://parts.igem.org/Part:BBa_J24679" class="external text" rel="nofollow">BBa_J24679</A>) gene under a negatively LuxR/AHL regulated promoter (P<SUB>LuxL</SUB><A href="http://parts.igem.org/Part:BBa_R0063" class="external text" rel="nofollow">BBa_R0063</A>) an indirect positive autoregulation for LuxR expression is obtained (Figure 2, right).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2: Possible strategies to reduce basal LuxR expression using positive feedback loops.</B></DIV></DIV></DIV></DIV><P><B>2. Fine-tuning of the P<SUB>Lac</SUB> driven LuxR using glucose </B>A second strategy could be to use the cells native LacI repressor to repress the P<SUB>Lac</SUB> promoter driving LuxR in our receiver construct. Addition of glucose to the pre-culture medium and/or to the agar plates could repress the expression of LuxR to a level below the one necessary for basal reporter expression, whereas in the induced state there would still be enough LuxR to activate the system.
<B>3. Destabilization of the enzymatic reporters with degradation tags </B>A third idea to minimize basal hydrolase activity would be to destabilize the reporter proteins with degradation tags. By lowering the half-life the accumulation of reporter could be reduced during incubation time. However we did not get to test this approach experimentally, but we analyzed the impact of such a degradation tag using our system model.
</P></DIV><DIV id="eth_footer"><H1><SPAN class="mw-headline" id="We_thank_our_sponsors:">We thank our sponsors:</SPAN></H1><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Basf-logo.gif" title="File:Basf-logo.gif">File:Basf-logo.gif</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Novartis-logo.gif" title="File:Novartis-logo.gif">180px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Roche-logo.gif" title="File:Roche-logo.gif">100px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Syngenta-logo.gif" title="File:Syngenta-logo.gif">130px</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:ETH_Zurich/Circuit">http://2013.igem.org/Team:ETH_Zurich/Circuit</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Circuit" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Circuit" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Circuit&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Circuit&amp;oldid=362317" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>