"
Page
Discussion
View source
History
teams
Log in
 
Team:ETH Zurich/Experiments
From 2013.igem.org
Home
Project
Information processing
Circuit and optimization
Pre-processing: Signal diffusion
Signal processing: High pass filters
Player interaction: Design and playing
Output system: Hydrolases
Data page
Experimental results
Overview
Pre-processing: Diffusion characterization
Proof-of-principle with GFP
Signal processing engineering
Circuit optimization
Output system characterization
Flagging
Gameplay
Model
Overview
Single Cell Model
AHL reaction-diffusion model
Model Evaluation: Genetic circuit with GFP
Genetic Circuit incorporating hydrolases
Analytical Approximations
Parameters
Lab work
Triple knockout strain
Constructs
Lab journal
Methods
Materials
Safety
Human practice
Overview
Worldwide playing
Gamification
Education
Team
Students
Advisors/Instructors
Official Team Profile
Attributions & contributions
References
Achievements
Achievements
Medal criteria
Awards
Login
WIKI
Log out
My account
Edit page
History
Upload new file
Contents
1 Final Circuit
2 Cloned Constructs
3 Groupparts / Biobricks
4 We thank our sponsors:
Final Circuit
For the final Colisweeper circuit we plan a four plasmid system. The mine cells constitutively express LuxI for signal generation and NagZ as identifier hydrolase. In the non-mine cells LuxR is expressed constitutively to process the AHL signal. To reduce the leakiness of the system we introduced the LacI repressor to reduce expression of LuxR in the uninduced state. At high AHL concentrations the pLuxL reporter is repressed leading to a positive feedback loop motif. PhoA as reporter for safe cells is expressed constitutively from the chromosome and is therefore not necessary as a plasmid. Aes and GusA are expressed from pLux promoters with different sensitivities. You can find all the biobricks we used and our own new biobricks in the figure below.
Figure 1. Plasmids in mine and non-mine cells: move the cursor over the separate parts to check which biobricks we used.
Cloned Constructs
To get to the circuit mentioned above we tested different versions of the circuit. For example we started our experiments using GFP as a reporter instead of the hydrolases. Then we also tested different LuxI and LuxR generating constructs. In the following table we list all the biobricks we used, the plasmids we cloned and what experiments we used them for. In general we used standard biobrick cloning techniques as described in the methods section. Whenever we used PCR gene amplification for cloning, we list the primers used in the following table. To be able to co-transform different plasmids we used backbones with compatible origins of replication and resistance genes. In the table you can find which backbone versions we used for which constructs.
Fluorescent protein reporter constructs
Description
Cloning
Maps
1
Receiver cell construct for GFP diffusion experiments
Plac-LuxR-pLuxR BBa_J09855 backbone (SpeI, PstI) and BBa_E0840 insert (XbaI, PstI)
2
Library of the Receiver cell constructs
Using the Plac-LuxR-pLuxR BBa_J09855.BBa_E0840 construct a library with mutated pLux promoters was created through site-saturation mutagenesis to screen for promoters with changed sensitivities
Primers:
5'-tatactagagacnnntaggatcgtacag
5'-gatcgtannngtttacgcaagaaaatg
5'-tagagacnnntaggatcgtannngtttacgcaagaaaatg
5'-tagagaccnntaggatcgtanangtttacgcaagaaaatg
5'-tagagacctntaggatcgtacangtttacgcaagaaaatg
Interesting versions of the promoter were sequenced and inserted into pSB1C3 backbone using custom-made oligos. They could then be used for further cloning.
3
Receiver cell construct for GFP experiments without the LuxR generating part
BBa_R0062 backbone (SpeI, PstI) and BBa_E0840 insert (XbaI, PstI)
4
Receiver cell construct for GFP experiments with positive feedback loop to reduce leakiness
Plac-LuxR-pLuxR BBa_J09855.BBa_E0840 construct where the pLac promoter was replaced with pLuxR to build a positive feedback loop. The promoter was inserted with two pairs of custom-made oligos using XbaI and HindIII restriction sites.
Oligos:
5’-ctagagacctgtaggatcgtacaggtttacgcaagaaaatggtttgttatagtcgaataaatactaga
5’-gattaaagaggagaaatactagatgaaaaacataaatgccgacgacacatacagaataattaataaaattaa
5’-aatctctagtatttattcgactataacaaaccattttcttgcgtaaacctgtacgatcctacaggtct
5’-agctttaattttattaattattctgtatgtgtcgtcggcatttatgtttttcatctagtatttctcctcttt
5
Receiver cell construct with mutated pLux promoter library and RFP reporter to test together with wild-type pLux GFP receiver cells in one cell
Library of mutated pLux promoters (see above) backbone (SpeI,PstI) and RFP (SpeI,PstI) insert from BBa_J23118
LuxI generating constructs
Description
Cloning
Maps
6
Sender cell construct with a very strong constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments
BBa_J23100 backbone (SpeI, PstI) and BBa_K805016 insert (XbaI, PstI)
7
Sender cell construct with an intermediate constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments
BBa_J23118 backbone (SpeI, PstI) and BBa_K805016 insert (XbaI, PstI)
8
Sender cell construct with an intermediate constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments
BBa_J23110 backbone (SpeI, PstI) and BBa_K805016 insert (XbaI, PstI)
9
Sender cell construct with a weak constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments
BBa_J23114 backbone (SpeI, PstI) and BBa_K805016 insert (XbaI, PstI)
LuxR generating constructs
Description
Cloning
Maps
10
constitutive LuxR generating biobrick
Plac-LuxR-pLuxR BBa_J09855
11
constitutive LuxR generating biobrick, with negative feedback-loop at high OHHL concentrations
BBa_F2621
12
negatively regulated pLuxL-LacI construct to improve the leakiness problem of the LuxR system
BBa_R0063 backbone (SpeI, PstI) and BBa_J24679 insert (XbaI, PstI)
pLuxR constructs
Description
Cloning
Maps
13
Library of pLuxR promoter variants in pSB1C3 backbone for cloning
The promoter variants were chosen after characterization and sequencing with the GFP receiver library. The sequences were ordered as custom-made oligos and inserted into pSB1C3 using EcoRI and PstI restriction sites.
Oligos:
5'-accagtaggatcgtagaggtttacgcaagaaaatggtttgttatagtcgaataaa
5'-accggtaggatcgtagaggtttacgcaagaaaatggtttgttatagtcgaataaa
5'-acccgtaggatcgtagaggtttacgcaagaaaatggtttgttatagtcgaataaa
5'-accagtaggatcgtaaaggtttacgcaagaaaatggtttgttatagtcgaataaa
5'-accagtaggatcgtataggtttacgcaagaaaatggtttgttatagtcgaataaa
Hydrolase constructs
Description
Cloning
Maps
14
Aes coding region with RBS in SB1C3 backbone
BBa_K1216002 was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs.
Primers:
5'-gtttcttcgaattcgcggccgcttctagaggtttaactttaagaaggagatatagctagc
5'-gtttcttcctgcagcggccgctactagtattattaaagctgagcggtaaagaactgag
15
Aes-His coding region with RBS in SB1C3 backbone
BBa_K1216006 was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs.
Primers:
5'-gtttcttcgaattcgcggccgcttctagaggtttaactttaagaaggagatatagctagc
5'-gtttcttcctgcagcggccgctactagtattattagtggtggtggtggtggtgctc
16
GusA coding region with RBS in SB1C3 backbone
BBa_K1216000 was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs.
Primers:
5'-gtttcttcgaattcgcggccgcttctagagtttgtttaactttaagaaggagatatacat
5'-gtttcttcctgcagcggccgctactagtattattattgtttgcctccctgctgcg
17
GusA-His coding region with RBS in SB1C3 backbone
BBa_K1216004 was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs.
Primers:
5'-gtttcttcgaattcgcggccgcttctagagtttgtttaactttaagaaggagatatacat
5'-gtttcttcctgcagcggccgctactagtattattagtggtggtggtggtggtgctc
18
PhoA coding region with RBS in SB1C3 backbone
BBa_K1216001 was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs.
Primers:
5'-gtttcttcgaattcgcggccgcttctagagtttgtttaactttaagaaggagatatacat
5'-gtttcttcctgcagcggccgctactagtattattacttcaggcccagcgccgctttc
19
PhoA-His coding region with RBS in SB1C3 backbone
BBa_K1216005 was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs.
Primers:
5'-gtttcttcgaattcgcggccgcttctagagtttgtttaactttaagaaggagatatacat
5'-gtttcttcctgcagcggccgctactagtattattagtggtggtggtggtggtgctc
20
NagZ coding region with RBS in SB1C3 backbone
BBa_K1216003 was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs.
Primers:
5'-gtttcttcgaattcgcggccgcttctagaggtttaactttaagaaggagatatagctagc
5'-gtttcttcctgcagcggccgctactagtattattactcgtgacctgctttctcttc
21
constitutive expression of PhoA for non-mine cells
constitutive promoter BBa_J23100 (SpeI, PstI) backbone and BBa_K1216001 insert (XbaI, PstI)
22
constitutive expression of NagZ for mine cells
constitutive promoter BBa_J23100 (SpeI, PstI) backbone and BBa_K1216003 insert (XbaI, PstI)
23
AHL inducible expression of GusA
Plac-LuxR-pLuxR BBa_J09855 (SpeI, PstI) backbone and BBa_K1216000 insert (XbaI, PstI)
24
AHL inducible expression of GusA with positive feedback loop for LuxR expression
Plac-LuxR-pLuxR BBa_J09855.BBa_K1216000 construct where the pLac promoter was replaced with pLuxR to build a positive feedback loop. The promoter was inserted with two pairs of custom-made oligos using XbaI and HindIII restriction sites.
Oligos:
5’-ctagagacctgtaggatcgtacaggtttacgcaagaaaatggtttgttatagtcgaataaatactaga
5’-gattaaagaggagaaatactagatgaaaaacataaatgccgacgacacatacagaataattaataaaattaa
5’-aatctctagtatttattcgactataacaaaccattttcttgcgtaaacctgtacgatcctacaggtct
5’-agctttaattttattaattattctgtatgtgtcgtcggcatttatgtttttcatctagtatttctcctcttt
25
AHL inducible expression of Aes
Plac-LuxR-pLuxR BBa_J09855 (SpeI, PstI) backbone and BBa_K1216002 insert (XbaI, PstI)
26
AHL inducible expression of Aes with mutant pLux promoter
Library of mutated pLux promoters (see above) backbone (SpeI,PstI) and BBa_K1216002 (SpeI,PstI) insert
Groupparts / Biobricks
In the following table you can find all the biobricks that were submitted by our group.
<groupparts>iGEM2013 ETH_Zurich</groupparts>
We thank our sponsors:
File:Basf-logo.gif
180px
100px
130px
Retrieved from " http://2013.igem.org/Team:ETH_Zurich/Experiments "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
