<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Experiments_5"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Experiments_5">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Experiments_5&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_5&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_5&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Experiments_5" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Experiments 5</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: pointer">Project</A></LI><LI><A href="/Team:ETH_Zurich/Infoproc" title="Team:ETH Zurich/Infoproc">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Circuit" title="Team:ETH Zurich/Circuit">Circuit and optimization</A></LI><LI><A href="/Team:ETH_Zurich/pre_proc" title="Team:ETH Zurich/pre proc">Pre-processing: Signal diffusion</A></LI><LI><A href="/Team:ETH_Zurich/Processing_2" title="Team:ETH Zurich/Processing 2">Signal processing: High pass filters</A></LI><LI><A href="/Team:ETH_Zurich/Play" title="Team:ETH Zurich/Play">Player interaction: Design and playing</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_7" title="Team:ETH Zurich/Experiments 7">Output system: Hydrolases</A></LI></UL><LI><A href="/Team:ETH_Zurich/Data_Page" title="Team:ETH Zurich/Data Page">Data page</A></LI><LI><A name="Biology" style="cursor: pointer">Experimental results</A></LI><LI><A href="/Team:ETH_Zurich/Experimentalresults" title="Team:ETH Zurich/Experimentalresults">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_2" title="Team:ETH Zurich/Experiments 2">Pre-processing: Diffusion characterization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_6" title="Team:ETH Zurich/Experiments 6">Proof-of-principle with GFP</A></LI><LI><STRONG class="selflink">Signal processing engineering</STRONG></LI><LI><A href="/Team:ETH_Zurich/Optimization" title="Team:ETH Zurich/Optimization">Circuit optimization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_3" title="Team:ETH Zurich/Experiments 3">Output system characterization</A></LI><LI><A href="/Team:ETH_Zurich/Flagging" title="Team:ETH Zurich/Flagging">Flagging</A></LI><LI><A href="/Team:ETH_Zurich/gameplayexperiment" title="Team:ETH Zurich/gameplayexperiment">Gameplay</A></LI><LI><A name="Model" style="cursor: pointer">Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Overview" title="Team:ETH Zurich/Modeling/Overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/InitialStates" title="Team:ETH Zurich/Modeling/InitialStates">Single Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Reaction_Diffusion_OOHL" title="Team:ETH Zurich/Modeling/Reaction Diffusion OOHL">AHL reaction-diffusion model</A></LI><LI><A href="/Team:ETH_Zurich/GFP" title="Team:ETH Zurich/GFP">Model Evaluation: Genetic circuit with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Genetic Circuit incorporating hydrolases</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximations" title="Team:ETH Zurich/Modeling/Analytical Approximations">Analytical Approximations</A></LI><LI><A href="/Team:ETH_Zurich/Parameter" title="Team:ETH Zurich/Parameter">Parameters</A></LI><LI><A name="LAb work" style="cursor: pointer">Lab work</A></LI><LI><A href="/Team:ETH_Zurich/Mutant" title="Team:ETH Zurich/Mutant">Triple knockout strain</A></LI><LI><A href="/Team:ETH_Zurich/Experiments" title="Team:ETH Zurich/Experiments">Constructs</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Lab journal</A></LI><LI><A href="/Team:ETH_Zurich/Materials" title="Team:ETH Zurich/Materials">Methods</A></LI><LI><A href="/Team:ETH_Zurich/Materials_2" title="Team:ETH Zurich/Materials 2">Materials</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A name="Human" style="cursor: pointer">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Practiceoverview" title="Team:ETH Zurich/Practiceoverview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/World" title="Team:ETH Zurich/World">Worldwide playing</A></LI><LI><A href="/Team:ETH_Zurich/Human" title="Team:ETH Zurich/Human">Gamification</A></LI><LI><A href="/Team:ETH_Zurich/HEducational" title="Team:ETH Zurich/HEducational">Education</A></LI><LI><A name="Team" style="cursor: pointer">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Students</A></LI><LI><A href="/Team:ETH_Zurich/Supervisors" title="Team:ETH Zurich/Supervisors">Advisors/Instructors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; contributions</A></LI><LI><A href="/Team:ETH_Zurich/References" title="Team:ETH Zurich/References">References</A></LI><LI><A name="Achievements" style="cursor: pointer">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal criteria</A></LI><LI><A href="/Team:ETH_Zurich/Awards" title="Team:ETH Zurich/Awards">Awards</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Experiments_5" class="external text" rel="nofollow">Login</A></LI><LI><A name="WIKI" style="cursor: pointer">WIKI</A></LI><LI><A href="/wiki/index.php?title=Special:UserLogout&amp;returnto=Team:ETH_Zurich/Home;returntoquery=action%3Dedit" title="Log out">Log out</A></LI><LI><A href="https://igem.org/User_Information" title="My account">My account</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Experiments_5&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Experiments_5&amp;action=history" class="external text" rel="nofollow">History</A></LI><LI><A href="http://2013.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#High_pass_filters"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">High pass filters</SPAN></A></LI><LI class="toclevel-1"><A href="#First_mutant:_Preliminary_tests_and_dose_response_curve_single_cell_analysis_of_the_wild-type_PLuxR_and_PLuxR_varaint_.28G1.29"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">First mutant: Preliminary tests and dose response curve single cell analysis of the wild-type P<SUB>LuxR</SUB> and  P<SUB>LuxR</SUB> varaint (G1)</SPAN></A></LI><LI class="toclevel-1"><A href="#Second_library_:_Back_mutation_and_single_cell_analysis_of_the_first_PLuxR_varaint_.28G1.29_by_rational_design_and_partial_sensitivity_recovery."><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Second library :  Back mutation and single cell analysis of the first P<SUB>LuxR</SUB> varaint (G1) by rational design and partial sensitivity recovery.</SPAN></A></LI><LI class="toclevel-1"><A href="#Fluorescence_data_analysis"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Fluorescence data analysis</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1"><A href="#We_thank_our_sponsors:"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">We thank our sponsors:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="High_pass_filters">High pass filters</SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Comparison of a chosen set of promoters.</B> The wild-type, P<SUB>LuxR</SUB> variant (G1, and the mutants created by rational design and partial sensitivity recovery SM1 and SM5.</DIV></DIV></DIV><P align="justify">The detection of <B>different AHL levels</B> depending on the number of mines requires filters in our detection  system. We decided to create a <B>high pass filter library</B> by doing site directed mutagenesis of the wild type <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">BBa_R0062</A> P<SUB>LuxR</SUB>. The sites for mutagenesis were chosen from literature (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5#referencespromoter" class="external text" rel="nofollow">2</A>) (see <I>Figure 1</I>). If you want to know more about the site directed saturation mutagenesis please read the theoretical part <A href="https://2013.igem.org/Team:ETH_Zurich/Processing_2" class="external text" rel="nofollow">here</A>.
We were able to isolate a promoter having a lower expression level and being less sensitive than the wild type promoter (<I>Figure 2</I>). The promoter is called P<SUB>LuxR</SUB> variant <A href="http://parts.igem.org/Part:BBa_K1216007" class="external text" rel="nofollow">BBa_K1216007</A> (G1, according to its initial position in the deep well plate). Those two different promoters were used to create two high pass filter to detect different AHL concentrations. The two promoters were analyzed not only in <B>liquid culture</B> by 96-well plate assays and single cell analysis (FACS) but also for<I> E.coli</I> on <B>agar plate</B>. Interestingly the EC<SUB>50</SUB> sensitivity (half maximal effective concentration, EC<SUB>50</SUB>) is different between cells in liquid culture and cells on agar plates. We then used the P<SUB>LuxR</SUB> variant (G1) to do <B>partial sensitivity recover</B>y by backmutating the two mutated nucleotides of the variant. <B>Rational design and oligomers</B> allow us to create a library of <B>7 different sensitive promoters</B>. See the figure on the left, and the full results in the <I>second library</I> part below.</P><H1><SPAN class="mw-headline" id="First_mutant:_Preliminary_tests_and_dose_response_curve_single_cell_analysis_of_the_wild-type_PLuxR_and_PLuxR_varaint_.28G1.29"><B>First mutant:</B> Preliminary tests and dose response curve single cell analysis of the wild-type P<SUB>LuxR</SUB> and  P<SUB>LuxR</SUB> varaint (G1)</SPAN></H1><DIV class="thumb tleft"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption"><B>Figure 1: Two sample plates used in the selection process.</B><B>a</B> Plate containing AHL (100 nM). All bright colonies have either a promoter that is sensitive to AHL or a constitutive one. <B>b</B> Plate without AHL. All bright colonies have a constitutive promoter and can be ruled out.</DIV></DIV></DIV><P><B>Positive and negative selection of the cells transformed with mutated P<SUB>LuxR</SUB></B>
In this experiment the 'luxbox' of the <A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">BBa_R0062</A> was mutated by site directed saturation muatgenisis in the P<SUB>Lac</SUB>-LuxR-P<SUB>LuxR</SUB><A href="http://parts.igem.org/Part:BBa_J09855" class="external text" rel="nofollow">BBa_J09855</A> construct using GFP as a reporter. At first we incubated the transformed cells on an AHL containing plate to rule out all AHL-insensitive mutations. All colonies expressing GFP were then restreaked on a plate which contained no AHL, thus allowing us to rule out constitutive promoters. The remaining clones were then grown in luquid cultures on 96-well plates and tested for sensitivity over a broad range of AHL concentrations. Promising candidates were then measured in a flow cytometer to check for bistable promoters. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:382px;"><DIV class="thumbcaption"><B>Figure 2: Sensivity curve of the wild type promoter and G1 mutant.</B> The AHL concentration range is plotted against the standarized fluorescence (au). For the wild type: EC<SUB>50</SUB>=5.86 nM, R<SUB>2</SUB>= 0.87,n=1.7 Fore the G1 mutant: EC<SUB>50</SUB>=1'341 nM, R<SUB>2</SUB>= 0.98,n=0.8. All assays were carried out in triplicates, results are presented as mean ± standard deviation.</DIV></DIV></DIV><P><B>Preliminary tests on microtiter plate reader screening </B></P><P align="justify">First of all we did a fast and preliminary screening of the mutated P<SUB>LuxR</SUB> promoters with the microtiter plate reader. Thus, we carried out dose response curves using the Tecan Infinite M200 plate reader. The test range was inspired from literature (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5#referencespromoter" class="external text" rel="nofollow">1</A>). Afterwards we did <B>flow cytometry</B> of the most promissing promoters to have high quality data and exclude bistable promoters.  The results are shown in Figure 2.</P><P><B>Dose response curve single cell analysis of the wild-type P<SUB>LuxR</SUB> and the first P<SUB>LuxR</SUB> variant (G1)</B><B>Liquid culture and agar plate AHL detection comparison</B></P><P align="justify">Here we employed single cell flow cytometry to obtain high quality fluorescence data of each promoter under different AHL concentrations.we started by analyzing cells grown in liquid culture, then we shifted to agar plate format in order to fit to our project. We obtained different EC<SUB>50</SUB> values between cells in liquid culture and in agar plates for each luxR promoter (see Figure 4 and 5). See <A href="https://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Materials#facs_data" class="external text" rel="nofollow">methods</A> for the protocol</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 3: Sensitivity curve of the wild type BBa_R0062 on agar plates and liquid culture</B> For the liquid culture we got: EC<SUB>50</SUB>=0.02 nM nM, R<SUP>2</SUP>=0.84, n=1.7; For the agar plates we got: EC<SUB>50</SUB>=4.45 nM, R<SUP>2</SUP>=0.80, n=1.7. All assays were carried out in duplicates, results are presented as mean ± standard deviation.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 4: Sensitivity curve of the G1 (P<SUB>LuxR</SUB> variant BBa_K1216007) on agar plates and liquid culture</B>For the liquid culture we got: EC<SUB>50</SUB>=6482 nM, R<SUP>2</SUP>=0.97, n=0.8; For the agar plates we got: EC<SUB>50</SUB>=13'704nM, R<SUP>2</SUP>=0.93, n=0.8. All assays were carried out in duplicates, results are presented as mean ± standard deviation.</DIV></DIV></DIV><P><B>In liquid culture</B></P><P align="justify">The direct plot of both sensitivity curves (Figure 5) shows the shifted sensitivity of the luxR promoter in comparison to the wild type BBa_R0062 promoter. The EC<SUB>50</SUB> shifted from 0.02nM for the wild type BBA_R0062 to 6'250 nM for the luxR variant promoter which is equal to a 300'000 fold increase.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 5: Comparison of the Wild Type P<SUB>LuxR</SUB> and the P<SUB>LuxR</SUB> mutant sensitivity curves in liquid culture</B> using the BD LSRFortessa™ Flow Cytometer SystemFor the wild type: EC<SUB>50</SUB>=0.02 nM, R<SUP>2</SUP>=0.84, n=1.7For G1 we got: EC<SUB>50</SUB>=6482 nM, R<SUP>2</SUP>=0.97, n=0.8. All assays were carried out in duplicates, results are presented as mean ± standard deviation.</DIV></DIV></DIV><P align="justify"><B>On agar plates</B>The direct plot of both sensitivity curves (Figure 6) shows the shifted sensitivity of the P<SUB>luxR</SUB> promoter in comparison to the wild type BBa_R0062 promoter. The EC<SUB>50</SUB> shifted from 4.45 nM for the wild type BBA_R0062 to 13'704 nM for the luxR variant promoter which is equal to a 2800 fold increase. Therefore, comparing with the results from cells growing in liquid culture an increase of 220 and 2 times for wild type BBa_R0062 and P<SUB>LuxR</SUB> variant respectively. If you want to know more about the methods please click <A href="https://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Materials#micro_plate" class="external text" rel="nofollow">here</A></P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 6: Comparison of the sensivity of the wild type promoter and the mutated luxR promoter on agar plates</B>For the wild type we got :EC<SUB>50</SUB>=4.45 nM, R2=0.80, n=1.7For the P<SUB>LuxR</SUB> variant we got :EC<SUB>50</SUB>=13'704 nM, R2=0.93, n=0.8. All assays were carried out in duplicates, results are presented as mean ± standard deviation.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:622px;"><DIV class="thumbcaption"><B>Figure 7: Flow cytometry dot plot of the wild type BBa_R0062 promoter vs. the P<SUB>luxR</SUB> mutant</B> The histogram plots of eGFP levels for each tested AHL concentration, culturing condition and corresponding promoter. On the top the data is the wild type luxR promoter. On the bottom the data of the luxR promoter variant.</DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Second_library_:_Back_mutation_and_single_cell_analysis_of_the_first_PLuxR_varaint_.28G1.29_by_rational_design_and_partial_sensitivity_recovery."><B> Second library : </B> Back mutation and single cell analysis of the first P<SUB>LuxR</SUB> varaint (G1) by rational design and partial sensitivity recovery.</SPAN></H1><P>Rational design and single cell analysis of additional P<SUB>LuxR</SUB> variants with partial sensitivity recovery  based on the first P<SUB>LuxR</SUB> variant (G1).
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 8: Sequence alignment of the additional luxR variants.</B></DIV></DIV></DIV><P>According to initial model predictions confirmed by successive <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_6" class="external text" rel="nofollow">experimental validation</A> the G1 promoter sensitivity is too low to drive a significant response in the concentration gradient established by our sender cells.
We need a collection of promoters with a set of EC<SUB>50</SUB> values between the wild type and G1 itself.
Since G1 has been obtained through two random mutations: 4T&gt;A and 16C&gt;G reverting one of the two, we reasoned, should result in a LuxR binding strength closer to the wild type. 
To test this hypothesis we ordered oligos encompassing all the possible combination of single position mutants (See list) and plan to experimentally characterize, and eventually deposit in the registry, all the resulting P<SUB>LuxR</SUB> variants. In parallel we used degenerate oligos to generate a small library (16 members) containing all the combination of double mutants for the two G1 key position to verify if all the double mutants show similar EC<SUB>50</SUB>.</P><P><B>Finally characterized promoter library</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:1102px;"><DIV class="thumbcaption"><B>Figure 9.Dose response curve experiment plates of the promoter library with an RFP reporter.</B> ON the picture you can see 11 plates having different AHL concentrations in the agar( 10^-5 nM, 10^-4 nM, 10^-3 nM,10^-2 nM,10^-1 nM,10^0 nM,10^1 nM,10^2 nM,10^3 nM,10^4 nM,10^5 nM.) of the backmutations SM1, SM2, SM3, SM3, S, SM5, SM6 and DM. (SM = Single mutation, DM = Double mutation.</DIV></DIV></DIV></DIV><P>As explained above we created a library of promoters originating from P<SUB>LuxR</SUB> variant (G1). We characterized the dose response curve to AHL on agar plates of all 7 promoters  by following the same protocol as for the P<SUB>LuxR</SUB> variant characterization. See the different shifted dose response curves in Figure 10. and the characteristics of the library in Table 1.
Thanks to the <B>analytical solution of the model</B> (see <A href="https://2013.igem.org/Team:ETH_Zurich/Modeling/Analytical_Approximations" class="external text" rel="nofollow">here</A>) predicting the needed EC<SUB>50</SUB> of 100 nM of the promoter to fit to our system we choose the SM1 mutant for our final set-up. (We did not choose SM2 because when we choose a promoter, according to the prediction of the model, the experiments were not done in duplicates at this time, and the characteristics changed of course as soon as we had duplicates). This promoter gets activated when two mines are nearby the receiver cell.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><B>Table 1. Characteristics of the promoter library </B></DIV></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 10. Dose response curve of the promoter library.</B> 7 different promoter were created by partial sensitivity recovery of the P<SUB>LuxR</SUB> variant. The colonies have the P<SUB>Lac</SUB>-LuxR-P<SUB>LuxR</SUB> BBa_J09855 construct with the different mutated promoter as well as a RFP reporter to analyze the dose response in the flow cytometer. The colonies were grown on 11 different plates with 11 different concentrtions as explained in Figure 9. Please see table 1 for the characteristics of the promoter library.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 11. Comparison of a choosen set of promoters.</B> The wild-type, P<SUB>LuxR</SUB> variant (G1, and the mutants created by rational design and partial sensitivity recovery SM1 and SM5.</DIV></DIV></DIV><P>
Figure 10. shows the relative fluorescence over the [AHL] of the dose-response curves of the whole library exepted the G1 and the wild type P<SUB>LuxR</SUB> promoter. Figure 11. is a comparison of a choosen set of promissing promoter we tried in different constructs for the final gameplay according to the prediction of the model.
</P><H1><SPAN class="mw-headline" id="Fluorescence_data_analysis">Fluorescence data analysis</SPAN></H1><P>The fitting of the following graphs was performed using this equation :
Output = eGFP levels [au]
Top    = maximal eGFP level [au] (&quot;full induction&quot;)
Bottom = minimal eGFP level [au] (“leakiness”)
n      = Hill coefficient (“cooperativity”)
EC<SUB>50</SUB>   = Half-maximal effective concentration (“sensitivity”)
[AHL]=AHL concentration [nM]</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><P>(1)  M Geske G.D.,<I>Evaluation of a focused library of N-aryl L-homoserine lactones reveals a new set of potent quorum sensing modulators.</I>
(2) Luis Caetano <I>A Mutational Analysis Defines Vibrio fischeri LuxR Binding Sites</I></P></DIV><DIV id="eth_footer"><H1><SPAN class="mw-headline" id="We_thank_our_sponsors:">We thank our sponsors:</SPAN></H1><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Basf-logo.gif" title="File:Basf-logo.gif">File:Basf-logo.gif</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Novartis-logo.gif" title="File:Novartis-logo.gif">180px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Roche-logo.gif" title="File:Roche-logo.gif">100px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Syngenta-logo.gif" title="File:Syngenta-logo.gif">130px</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:ETH_Zurich/Experiments_5">http://2013.igem.org/Team:ETH_Zurich/Experiments_5</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Experiments_5" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Experiments_5" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_5&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments_5&amp;oldid=362284" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>