<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><STRONG class="selflink">Home</STRONG></LI><LI><A name="Project" style="cursor: pointer">Project</A></LI><LI><A href="/Team:ETH_Zurich/Infoproc" title="Team:ETH Zurich/Infoproc">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Circuit" title="Team:ETH Zurich/Circuit">Circuit and optimization</A></LI><LI><A href="/Team:ETH_Zurich/pre_proc" title="Team:ETH Zurich/pre proc">Pre-processing: Signal diffusion</A></LI><LI><A href="/Team:ETH_Zurich/Processing_2" title="Team:ETH Zurich/Processing 2">Signal processing: High pass filters</A></LI><LI><A href="/Team:ETH_Zurich/Play" title="Team:ETH Zurich/Play">Player interaction: Design and playing</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_7" title="Team:ETH Zurich/Experiments 7">Output system: Hydrolases</A></LI></UL><LI><A href="/Team:ETH_Zurich/Data_Page" title="Team:ETH Zurich/Data Page">Data page</A></LI><LI><A name="Biology" style="cursor: pointer">Experimental results</A></LI><LI><A href="/Team:ETH_Zurich/Experimentalresults" title="Team:ETH Zurich/Experimentalresults">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_2" title="Team:ETH Zurich/Experiments 2">Pre-processing: Diffusion characterization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_6" title="Team:ETH Zurich/Experiments 6">Proof-of-principle with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_5" title="Team:ETH Zurich/Experiments 5">Signal processing engineering</A></LI><LI><A href="/Team:ETH_Zurich/Optimization" title="Team:ETH Zurich/Optimization">Circuit optimization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_3" title="Team:ETH Zurich/Experiments 3">Output system characterization</A></LI><LI><A href="/Team:ETH_Zurich/Flagging" title="Team:ETH Zurich/Flagging">Flagging</A></LI><LI><A href="/Team:ETH_Zurich/gameplayexperiment" title="Team:ETH Zurich/gameplayexperiment">Gameplay</A></LI><LI><A name="Model" style="cursor: pointer">Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Overview" title="Team:ETH Zurich/Modeling/Overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/InitialStates" title="Team:ETH Zurich/Modeling/InitialStates">Single Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Reaction_Diffusion_OOHL" title="Team:ETH Zurich/Modeling/Reaction Diffusion OOHL">AHL reaction-diffusion model</A></LI><LI><A href="/Team:ETH_Zurich/GFP" title="Team:ETH Zurich/GFP">Model Evaluation: Genetic circuit with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Genetic Circuit incorporating hydrolases</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximations" title="Team:ETH Zurich/Modeling/Analytical Approximations">Analytical Approximations</A></LI><LI><A href="/Team:ETH_Zurich/Parameter" title="Team:ETH Zurich/Parameter">Parameters</A></LI><LI><A name="LAb work" style="cursor: pointer">Lab work</A></LI><LI><A href="/Team:ETH_Zurich/Mutant" title="Team:ETH Zurich/Mutant">Triple knockout strain</A></LI><LI><A href="/Team:ETH_Zurich/Experiments" title="Team:ETH Zurich/Experiments">Constructs</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Lab journal</A></LI><LI><A href="/Team:ETH_Zurich/Materials" title="Team:ETH Zurich/Materials">Methods</A></LI><LI><A href="/Team:ETH_Zurich/Materials_2" title="Team:ETH Zurich/Materials 2">Materials</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A name="Human" style="cursor: pointer">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Practiceoverview" title="Team:ETH Zurich/Practiceoverview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/World" title="Team:ETH Zurich/World">Worldwide playing</A></LI><LI><A href="/Team:ETH_Zurich/Human" title="Team:ETH Zurich/Human">Gamification</A></LI><LI><A href="/Team:ETH_Zurich/HEducational" title="Team:ETH Zurich/HEducational">Education</A></LI><LI><A name="Team" style="cursor: pointer">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Students</A></LI><LI><A href="/Team:ETH_Zurich/Supervisors" title="Team:ETH Zurich/Supervisors">Advisors/Instructors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; contributions</A></LI><LI><A href="/Team:ETH_Zurich/References" title="Team:ETH Zurich/References">References</A></LI><LI><A name="Achievements" style="cursor: pointer">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal criteria</A></LI><LI><A href="/Team:ETH_Zurich/Awards" title="Team:ETH Zurich/Awards">Awards</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich" class="external text" rel="nofollow">Login</A></LI><LI><A name="WIKI" style="cursor: pointer">WIKI</A></LI><LI><A href="/wiki/index.php?title=Special:UserLogout&amp;returnto=Team:ETH_Zurich/Home;returntoquery=action%3Dedit" title="Log out">Log out</A></LI><LI><A href="https://igem.org/User_Information" title="My account">My account</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich&amp;action=history" class="external text" rel="nofollow">History</A></LI><LI><A href="http://2013.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><A href="http://minesweeperonline.com/" target="_Blank">Play Minesweeper here !</A><DIV style="display: table; table-layout: fixed; max-width:1200px; width: 90%; margin: 0 auto; position: relative; height: 100%; min-height:530;"><DIV style="display: table-row; height:100%;"><DIV style="display: table-cell; vertical-align: top; width: 350px; height: 100%; padding-bottom: 70px;"><UL class="bxslider" id="bxsliderText"><LI><B>Colisweeper</B> Colisweeper is an interactive, biological version of the computer game Minesweeper. The goal is to clear an agar “minefield” without detonating the mines. Genetically engineered <I>Escherichia coli</I> colonies are used as mines and non-mines. Mines secrete the signaling molecule AHL whereas non-mines process the signal. To distinguish different AHL-levels, a library of P<SUB>LuxR</SUB> promoters with various AHL sensitivities was created through site-saturation mutagenesis. High-pass filters were constructed to control the expression of different orthogonal hydrolases in non-mines, depending on the number of surrounding mines. The slideshow covers all aspects of our project. Feel free to <B>CLICK ON ANY PICTURE</B> in <B>ANY SLIDESHOW</B> to navigate to the description page. CLICK ON THE START BUTTON OF THE FIRST SLIDE TO SEE OUR VIDEO
</LI><LI><B>How to play Colisweeper</B> Initially, an agar plate is prepared with mine and non mine colonies in the hexagonal grid pattern. To play the game, the player pipettes a solution on to a colony of choice. The color developed on this colony will determine the next move for the player. If the playing solution is pipetted on a colony and it turns yellow, salmon or magenta; this indicates zero,one or two mines respectively adjacent to the played colony. In case a mine colony is encountered, the colony turns blue and the game is over. If the player decides to flag a mine colony, the flagging solution is pipetted that turns the colony green in color. This way the color logic gives the player cue for the next move in the game.

</LI><LI><B>Information Processing</B> Sender cells (mines) express LuxI protein, which catalyzes the production of a small signalling molecule, called AHL. This molecule diffuses through the agar plate and reaches the receiver cells called non-mine colonies. These colonies are designed to distinguish different concentrations of AHL and translate this analog information into the expression of different sets of reporter enzymes. The continuous signal is digitized through a set of mutated P<SUB>LuxR</SUB> promoters with  different AHL sensitivities acting as highpass filters. 
</LI><LI><B>From Minesweeper to Colisweeper </B> Mines secrete the signaling molecule AHL which diffuses through the agar and is processed by neighboring non-mine colonies. High-pass filters were constructed to control the expression of different orthogonal hydrolase enzymes in non-mines. Promoters that serve as high-pass filters were tuned to express hydrolases depending on the concentration of the AHL molecules from the surrounding mines. The colors yellow, salmon and magenta corresponds to zero, one and two mines around a colony. Additionally, the mines express their own hydrolase which when added with the multi-substrate gives blue color. The genomic expression of <I>lacZ</I> enables the flagging of both mines and non mine colonies turning the colonies green. 
</LI><LI><B>Hydrolase Reactions</B> As a reporter system we use a set of orthogonal hydrolase enzymes: alkaline phosphatase (<I>phoA</I>), β-galactosidase (<I>lacZ</I>), acetylesterase (<I>aes</I>), β-N-Acetylglucosaminidase (<I>nagZ</I>) and β-glucuronidase (<I>gusA</I>). Each hydrolase can react with its respective substrate within minutes resulting in a fast, colorful output. Quick response times and the ability to read the output without using instruments are essentials for a fast gameplay.</LI><LI><B>The Model</B>As our bio-game is based on processing the AHL concentration in the non-mine colonies, the diffusion of AHL in the agar is vital to the system. The diffusion was modeled by carrying out simulations to determine the time and distance of diffusion. We also modeled synthesis, regulation and degradation reactions of the proteins involved in our genetic circuits. To account for both processes: diffusion and reactions; we developed a spatio-temporal model in two dimensions comprised by three modules: mines, receivers, and the agar plate. Finite element methods were used to solve the system of partial differential equations (PDEs). Our model turned out to be very valuable in the circuit refinement and the design of experiments. Moreover, we continually improve out model by incorporating parameters from our own experimental data.
</LI><LI><B>Experimental Results</B> Diffusion experiments were performed to determine the time and distance of AHL diffusion between colonies in the agar mine-grid. A symbiotic relation between experiment and model proved to be beneficial. As proof-of-principle, we set up diffusion experiments using GFP as reporter. The LuxR promoter from the registry was mutated to obtain a library of P<SUB>LuxR</SUB> promoters with different sensitivities in order to distinguish different AHL levels. Initial tests suggested leaky reporter expression in the uninduced colonies. As a solution to this problem, we use glucose to shut down the P<SUB>lac</SUB>promoter and a positive feedback loop using lacI under the P<SUB>LuxL</SUB> promoter. Meanwhile we characterize the biobricks using Michealis-Menten kinetics and flow cytometry.
</LI><LI><B>Human practice</B>We analyze the concept of gamification in synthetic biology, describing many examples where common games are played in a new way or where games are used as research tools. We discuss possible consequences for synthetic biology and show how Colisweeper could be used for awareness raising. For this purpose we designed a Colisweeper Laboratory Course Kit and we propose an idea of a web-based Colisweeper platform enabling people from across the world to play against each other using a remotely controlled robot.
</LI><LI><B>Team</B>We are a team of seven highly motivated Bachelor- and Master Students at ETH Zurich pursuing various fields such as Biotechnology,  Biomedical Engineering, Neurobiology and Bioinformatics. The iGEM project is carried out at one of the youngest departments of ETHZ located in Basel-Department of Biosystems Science and Engineering - flourishing in interdisciplinary biological research. If you're around Basel, make sure to visit our team's lab to play the bio-game Colisweeper!
</LI></UL></DIV></DIV></DIV></DIV><DIV id="eth_footer"><H1><SPAN class="mw-headline" id="We_thank_our_sponsors:">We thank our sponsors:</SPAN></H1><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Basf-logo.gif" title="File:Basf-logo.gif">File:Basf-logo.gif</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Novartis-logo.gif" title="File:Novartis-logo.gif">180px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Roche-logo.gif" title="File:Roche-logo.gif">100px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Syngenta-logo.gif" title="File:Syngenta-logo.gif">130px</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:ETH_Zurich">http://2013.igem.org/Team:ETH_Zurich</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich&amp;oldid=357752" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>