<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: pointer">Project</A></LI><LI><A href="/Team:ETH_Zurich/Infoproc" title="Team:ETH Zurich/Infoproc">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Circuit" title="Team:ETH Zurich/Circuit">Circuit and optimization</A></LI><LI><A href="/Team:ETH_Zurich/pre_proc" title="Team:ETH Zurich/pre proc">Pre-processing: Signal diffusion</A></LI><LI><A href="/Team:ETH_Zurich/Processing_2" title="Team:ETH Zurich/Processing 2">Signal processing: High pass filters</A></LI><LI><A href="/Team:ETH_Zurich/Play" title="Team:ETH Zurich/Play">Player interaction: Design and playing</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_7" title="Team:ETH Zurich/Experiments 7">Output system: Hydrolases</A></LI></UL><LI><A href="/Team:ETH_Zurich/Data_Page" title="Team:ETH Zurich/Data Page">Data page</A></LI><LI><A name="Biology" style="cursor: pointer">Experimental results</A></LI><LI><A href="/Team:ETH_Zurich/Experimentalresults" title="Team:ETH Zurich/Experimentalresults">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_2" title="Team:ETH Zurich/Experiments 2">Pre-processing: Diffusion characterization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_6" title="Team:ETH Zurich/Experiments 6">Proof-of-principle with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_5" title="Team:ETH Zurich/Experiments 5">Signal processing engineering</A></LI><LI><A href="/Team:ETH_Zurich/Optimization" title="Team:ETH Zurich/Optimization">Circuit optimization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_3" title="Team:ETH Zurich/Experiments 3">Output system characterization</A></LI><LI><A href="/Team:ETH_Zurich/Flagging" title="Team:ETH Zurich/Flagging">Flagging</A></LI><LI><A href="/Team:ETH_Zurich/gameplayexperiment" title="Team:ETH Zurich/gameplayexperiment">Gameplay</A></LI><LI><A name="Model" style="cursor: pointer">Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Overview" title="Team:ETH Zurich/Modeling/Overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/InitialStates" title="Team:ETH Zurich/Modeling/InitialStates">Single Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Reaction_Diffusion_OOHL" title="Team:ETH Zurich/Modeling/Reaction Diffusion OOHL">AHL reaction-diffusion model</A></LI><LI><A href="/Team:ETH_Zurich/GFP" title="Team:ETH Zurich/GFP">Model Evaluation: Genetic circuit with GFP</A></LI><LI><STRONG class="selflink">Genetic Circuit incorporating hydrolases</STRONG></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximations" title="Team:ETH Zurich/Modeling/Analytical Approximations">Analytical Approximations</A></LI><LI><A href="/Team:ETH_Zurich/Parameter" title="Team:ETH Zurich/Parameter">Parameters</A></LI><LI><A name="LAb work" style="cursor: pointer">Lab work</A></LI><LI><A href="/Team:ETH_Zurich/Mutant" title="Team:ETH Zurich/Mutant">Triple knockout strain</A></LI><LI><A href="/Team:ETH_Zurich/Experiments" title="Team:ETH Zurich/Experiments">Constructs</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Lab journal</A></LI><LI><A href="/Team:ETH_Zurich/Materials" title="Team:ETH Zurich/Materials">Methods</A></LI><LI><A href="/Team:ETH_Zurich/Materials_2" title="Team:ETH Zurich/Materials 2">Materials</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A name="Human" style="cursor: pointer">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Practiceoverview" title="Team:ETH Zurich/Practiceoverview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/World" title="Team:ETH Zurich/World">Worldwide playing</A></LI><LI><A href="/Team:ETH_Zurich/Human" title="Team:ETH Zurich/Human">Gamification</A></LI><LI><A href="/Team:ETH_Zurich/HEducational" title="Team:ETH Zurich/HEducational">Education</A></LI><LI><A name="Team" style="cursor: pointer">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Students</A></LI><LI><A href="/Team:ETH_Zurich/Supervisors" title="Team:ETH Zurich/Supervisors">Advisors/Instructors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; contributions</A></LI><LI><A href="/Team:ETH_Zurich/References" title="Team:ETH Zurich/References">References</A></LI><LI><A name="Achievements" style="cursor: pointer">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal criteria</A></LI><LI><A href="/Team:ETH_Zurich/Awards" title="Team:ETH Zurich/Awards">Awards</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Modeling" class="external text" rel="nofollow">Login</A></LI><LI><A name="WIKI" style="cursor: pointer">WIKI</A></LI><LI><A href="/wiki/index.php?title=Special:UserLogout&amp;returnto=Team:ETH_Zurich/Home;returntoquery=action%3Dedit" title="Log out">Log out</A></LI><LI><A href="https://igem.org/User_Information" title="My account">My account</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Modeling&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Modeling&amp;action=history" class="external text" rel="nofollow">History</A></LI><LI><A href="http://2013.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Circuit_containing_hydrolases"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Circuit containing hydrolases</SPAN></A></LI><LI class="toclevel-1"><A href="#Mine_Cells"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Mine Cells</SPAN></A></LI><LI class="toclevel-1"><A href="#Agar_Plate"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Agar Plate</SPAN></A></LI><LI class="toclevel-1"><A href="#Receiver_Cells"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Receiver Cells</SPAN></A></LI><LI class="toclevel-1"><A href="#Results"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1"><A href="#Circuit_optimization"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Circuit optimization</SPAN></A></LI><LI class="toclevel-1"><A href="#We_thank_our_sponsors:"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">We thank our sponsors:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Circuit_containing_hydrolases">Circuit containing hydrolases</SPAN></H1><P align="justify"><B>A six-species model was implemented to simulate the behaviour of our multicellular sender–receiver system. The model consist of one partial differential equation for AHL and 5 ordinary differential equations with Hill functions that captured the activation of protein synthesis as a function of the concentration of the signalling molecule. </B></P><P align="center"><B>Video:</B> GusA expression levels over 24 hours, concentration in mol/m<SUP>3</SUP>. Distance between colonies: 1.5 cm.</P><P align="justify"> For the agar plate and mine cells modules, we use the system of equations and parameters set of the previous <A href="https://2013.igem.org/Team:ETH_Zurich/GFP" class="external text" rel="nofollow"><I>simulation</I></A>. </P><H1><SPAN class="mw-headline" id="Mine_Cells">Mine Cells</SPAN></H1><P align="justify"> Two proteins are produced by mine cells: (i) LuxI and (ii) <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7" class="external text" rel="nofollow">NagZ</A>. NagZ is the hydrolase responsible to reveal the mine identity, this gene is constitutively expressed and the protein is linearly degraded. The system of differential equations for the states involved in the sender module were given before (<A href="https://2013.igem.org/Team:ETH_Zurich/GFP#MineColony" class="external text" rel="nofollow">view</A>), with exception of NagZ: </P><P>\begin{align}
\frac{d[NagZ]}{dt} = \alpha_{NagZ} - d_{NagZ} \cdot [NagZ]\\
\end{align}
</P><H1><SPAN class="mw-headline" id="Agar_Plate">Agar Plate</SPAN></H1><P align="justify"> The equation for AHL in agar plate was given before (<A href="https://2013.igem.org/Team:ETH_Zurich/GFP#Plate" class="external text" rel="nofollow">view</A>). </P><H1><SPAN class="mw-headline" id="Receiver_Cells">Receiver Cells</SPAN></H1><P align="justify">Receivers are engineered to respond differently to two AHL concentration levels. Basically, cells should be capable of produce a visible response and in a reasonable amount of time, because the player will need to discriminate between the presence of 0, 1 or 2 adjacent mines. To achieve this goal, we incorporate two enzymatic reporters acting as <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5" class="external text" rel="nofollow">high-pass filters</A> (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7" class="external text" rel="nofollow">GusA</A> and  <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_7" class="external text" rel="nofollow">AES</A>). The expression of these enzymes is under the control of  P<SUB>Lux</SUB> promoters with different sensitivities, the wild type and a promoter mutant (<A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5" class="external text" rel="nofollow">G1 mutant</A>); The affinity constants for the promoters were obtained from our <A href="https://2013.igem.org/Team:ETH_Zurich/Experiments_5" class="external text" rel="nofollow">experimental data</A>. However, the affinity was determined with respect to AHL, thus we omit the complex formation and model production of the enzymes as function of AHL concentration. The assumption is not unrealistic because LuxR was constitutively produced and linearly degraded, which leads to a constant concentration.  An additional important feature of these enzymes is that they can catalyze the hydrolysis of various chromogenic compounds to give rise to a relatively quick coloured response. </P><P>
The intracellular species of interest in the receiver cells module include: LuxR, AHL and the hydrolases (GusA and AES). 
</P><P>\begin{align}
\frac{d[LuxR]}{dt}=\alpha_{[LuxR]} - d_{LuxR} [LuxR] \\
\end{align}
\begin{align}
\frac{d[AHL]}{dt}= C_{agar} \cdot D_{AHL} \nabla^{2} AHL - DF \cdot d_{AHL} \cdot [AHL] \\
\end{align}
\begin{align}
\frac{d[GusA]}{dt}=\alpha_{GusA} \cdot k_{leaky} \cdot [LuxR]+ \frac{\alpha_{GusA}\left(\frac{[AHL]}{K_{R_1}}\right)^{n_1}}{1+\left(\frac{[AHL]}{K_{R_1}}\right)^{n_1}} - d_{GusA}\cdot [GusA]\\
\end{align}
\begin{align}
\frac{d[AES]}{dt}=\alpha_{AES} \cdot k_{leaky} \cdot [LuxR] + \frac{\alpha_{AES}\left(\frac{[AHL]}{K_{R_2}}\right)^{n_2}}{1+\left(\frac{[AHL]}{K_{R_2}}\right)^{n_2}} - d_{AES}\cdot [AES]\\
\end{align}
</P><P align="justify"> In addition to new proteins incorporated to the circuit, it is important to emphasize that the grid was changed to a three neighbours setup.</P><H1><SPAN class="mw-headline" id="Results">Results</SPAN></H1><P align="justify"> 
In figure 1 and 3 are shown the results regarding GusA expression levels. From the time course plot (Fig 1.), it can be pointed out that the activation of the gene is consistent with the AHL concentration that is predicted to reach a receiver colony. Additionally, one can say that the incubations of the plates should not be longer than 17 hours, otherwise receiver cells that are more that one cell away from a mine can start synthesizing the enzyme. </P><TABLE style="border: none; background: transparent;" align="center"><TBODY><TR><TD valign="top" style="border:none;"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:512px;"><DIV class="thumbcaption"><B>Figure 1:  Averaged GusA concentration </B> for a colony with 0, 1, 2 or 3 adjacent mines colonies, respectively. Distance between colonies: 1.5 cm.</DIV></DIV></DIV></DIV></TD><TD valign="top" style="border:none;"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:492px;"><DIV class="thumbcaption"><B>Figure 2:  Averaged AHL concentration </B> sensed by a colony with 0, 1 or 2 adjacent mines colonies, respectively. The horizontal line indicates the activation coefficient by the dimer LuxR/AHL (K<SUB>R1</SUB>). Distance between colonies: 1.5 cm.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE style="border: none; background: transparent;" align="center"><TBODY><TR><TD valign="top" style="border:none;"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:512px;"><DIV class="thumbcaption"><B>Figure 3:  GusA expression profile </B> after 16 hours. Display: GusA concentration in mM. Distance between colonies: 1.5 cm.</DIV></DIV></DIV></DIV></TD><TD valign="top" style="border:none;"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:492px;"><DIV class="thumbcaption"><B>Figure 4: 1D averaged AHL concentration </B> gradient after 11 hours, radial direction</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P align="justify"> 
Regarding the expression of AES under the control of P<SUB>LuxR</SUB> variant (Fig 4.), there is not activation before 15 hours only basal expression. Should be noted that the predicted concentration of enzyme is very low if it is compared to GusA. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:1002px;"><DIV class="thumbcaption"><B>Figure 4: AES expression profile </B> after 11 (left) and 16 hours (right). Display: AES concentration in mM. Distance between colonies: 1.5 cm.</DIV></DIV></DIV></DIV><H1><SPAN class="mw-headline" id="Circuit_optimization">Circuit optimization </SPAN></H1><P><B> Protein destabilization </B></P><P align="justify"> Now, we wanted to evaluate the destabilization of proteins, by reducing the half life of GusA and set it to 2 hours. In practice, this can be done introducing degradation tags. The idea is to compensate for the accumulation of GusA at receiver colonies surrounded by two mines, which starts earlier that AES. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><B>Figure 5:</B> Solid lines:  Averaged GusA concentration over time for a colony with 0, 1, 2 or 3 adjacent mines colonies.  Dotted:  Averaged AES concentration over time for a colony with 0, 1, 2 or 3 adjacent mines colonies. Distance between colonies: 1.5 cm.</DIV></DIV></DIV></DIV><P align="justify"> According to simulation results, game should be played after 12 to 13 hours of incubation of the plates.</P><P><B> Positive feedback loop: Lowering basal expression  </B></P><P align="justify">As describe in the <A href="https://2013.igem.org/Team:ETH_Zurich/Circuit#circuitOpt" class="external text" rel="nofollow">Data page</A>, a main challenge of the circuit was to reduce the basal expression due to activation of the P<SUB>LuxR</SUB> promoter by LuxR alone. To overcome this issue we introduce a double negative feedback in the receiver module. The system of differential equations for the states involved in the receiver module were given before (<A href="https://2013.igem.org/Team:ETH_Zurich/GFP#MineColony" class="external text" rel="nofollow">view</A>), which holds with a modification in the equation for LuxR and introducing a new equation for LacI: </P><P>\begin{align}
\frac{d[LuxR]}{dt} = \frac{\alpha_{LuxR}}{1+ \left(\frac{[LacI]}{\theta_{LacI}}\right)^2} - d_{LacI} \cdot [LacI]\\
\end{align}
</P><P>\begin{align}
\frac{d[LacI]}{dt} = \frac{\alpha_{LacI}}{1+ \frac{[R]}{\theta_{R}}} - d_{LacI} \cdot [LacI]\\
\end{align}
</P><P align="justify">For practical purposes the circuit was tested with GFP as reporter. </P></DIV><DIV id="eth_footer"><H1><SPAN class="mw-headline" id="We_thank_our_sponsors:">We thank our sponsors:</SPAN></H1><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Basf-logo.gif" title="File:Basf-logo.gif">File:Basf-logo.gif</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Novartis-logo.gif" title="File:Novartis-logo.gif">180px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Roche-logo.gif" title="File:Roche-logo.gif">100px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Syngenta-logo.gif" title="File:Syngenta-logo.gif">130px</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:ETH_Zurich/Modeling">http://2013.igem.org/Team:ETH_Zurich/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling&amp;oldid=362319" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>