"
Page
Discussion
View source
History
teams
Log in
 
Team:ETH Zurich/Human
From 2013.igem.org
Home
Project
Information processing
Circuit and optimization
Pre-processing: Signal diffusion
Signal processing: High pass filters
Player interaction: Design and playing
Output system: Hydrolases
Data page
Experimental results
Overview
Pre-processing: Diffusion characterization
Proof-of-principle with GFP
Signal processing engineering
Circuit optimization
Output system characterization
Flagging
Gameplay
Model
Overview
Single Cell Model
AHL reaction-diffusion model
Model Evaluation: Genetic circuit with GFP
Genetic Circuit incorporating hydrolases
Analytical Approximations
Parameters
Lab work
Triple knockout strain
Constructs
Lab journal
Methods
Materials
Safety
Human practice
Overview
Worldwide playing
Gamification
Education
Team
Students
Advisors/Instructors
Official Team Profile
Attributions & contributions
References
Achievements
Achievements
Medal criteria
Awards
Login
WIKI
Log out
My account
Edit page
History
Upload new file
Contents
1 Synthetic Biology : Only a Game ?
2 Playing games with synthetic biology
3 Games as research tools
4 Conclusions
5 References
6 We thank our sponsors:
Synthetic Biology : Only a Game ?
Inspired by our laboratory project we analyze the relationship between synthetic biology and games, two concepts that are connected on many levels. We describe and characterize different aspects and also include our own project into our analysis. On one hand, synthetic biology can be used to play common games in a new way, possibly for educational purposes. More recently, synthetic biologists started to use games as a research tool as well, an innovative approach to make use of crowdsourcing and distributed computing. We want to present examples for these different aspects, find correlations and discuss possible consequences for the Synthetic Biology community. Despite the fact that most of these games have their origin in basic research; such experiments have the potential to be used for raising awareness in a new and original way. We see our Colisweeper game as an educational tool that could be presented for example in the context of a laboratory course. It can introduce people with only little knowledge in biology to different important topics in Synthetic Biology. Bringing science closer to people from non-scientific backgrounds may be one of the most important activities, raising understanding, and possibly acceptance for synthetic biology in public in this case.
Playing games with synthetic biology
Using an engineering approach for biological systems is an important characteristic of synthetic biology. This can not only help to change the understanding of many aspects of biology; synthetic biological systems can also be used to solve well-known computational and mathematical problems. Especially during the early days in the field, a lot of mathematical problems were approached to prove the potential of the new biological systems. There are various examples from in vitro DNA computations where classic computational tasks like the maximal clique problem [1], the traveling salesman [2] or the satisfiability problem [3] [4] were solved with DNA systems. Their large size and high computational expense made these problems especially interesting for DNA computation due to its large-scale information processing abilities. Having no direct application or profit, these experiments mainly demonstrated the possibilities and potential lying within synthetic biology.
In 2003 Milan Stojanovic and Darko Stefanovic presented an in vitro molecular automaton system that is able to play tic-tac-toe against a human opponent [5]. The network consists of 23 separate logic gates which allow the system to pursuit a perfect strategy and make it invincible to its human opponents. The gates are built using 24 deoxyribozymes reacting to input oligonucleotides through cleavage and release of a fluorescent product. The human player can choose between six input oligonucleotides, representing the six fields of the game, and add them to all wells. The automaton reacts within 15min with a change in fluorescence in the well which it “decided” to play. The authors of the paper use tic-tac-toe to demonstrate the possibility to build an interactive in vitro automaton capable of different boolean calculations. They chose tic-tac-toe because of its straightforward game tree, which had already made it a classic choice for the development of new computational paradigms in the past.
In 2010 they published another example of in vitro computation based on similar mechanisms [6]. This time, the simple game named tit-for-tat was designed specifically for this purpose. It consists of only four fields within a quadrat and the goal of the game is simply that in every round each player marks one field. Whenever the automaton manages to mark a free field and only one field at a time, it wins the game. It is assumed that the human player does not make such mistakes either. To win the game there are different strategies. In a first step, the automaton is trained to a certain strategy through addition of input oligonucleotides. Following the training, the automaton is able to pursuit the right strategy and wins. This programmable molecular automaton again illustrated the potential of synthetic biological systems.
These two in vitro examples show that logic games with a mathematical background can be used to benchmark biological systems on different characteristics like modularity, programmability or robustness that will be very important when it comes to concrete applications for synthetic biological circuits. The examples were mainly part of basic proof-of-concept research. Nowadays concrete applications and much more complex systems are in the focus of interest. Nevertheless we think that today’s research results could also be used for game implementations, something we head for with our Colisweeper project.
Games as research tools
The examples we will list here are all part of a rather recent development not only in synthetic biology, but also in other fields of research. Some problems in synthetic biology are still difficult to solve computationally, due to lack of correct algorithms or computing capacity. Dividing such tasks among a large number of people who do not necessarily have to come from a scientific background is an innovative approach referred to as crowdsourcing and distributed computing. There are a lot of examples available using different approaches. Some projects like Rosetta rely on distributing computing capacity only [7]. People can download the necessary programs from the webpage and thus allow the project to make use of the capacity of their computer whenever they are not using it. This is a passive way for people to participate and contribute to science, whereas the following examples require active participation of the people involved. The term gamification of synthetic biology is used, which means that game thinking and game mechanics can be used to engage users and solve problems from a non-game context [8]. A question or a challenge can be packed into an appealing game to motivate people to take part and help solving problems without getting rewarded directly.  We want to present some recent examples where tasks from synthetic biology have been gamified to be able to use not only the computing capacity, but also the creativity and puzzle solving abilities of a large number of people.
Figure 1. The protein folding game Foldit[11]
In 2008 Foldit, an online puzzle game about protein folding was released, initiated by the University of Washington’s Center for Game Science at the department of computer science and engineering together with the Baker group at the department of biochemistry. In the game, players try to predict the structures of proteins according to three basic rules: “pack the protein”, “hide the hydrophobics” and “clear the clashes”. The better the folded protein satisfies these objectives, the higher becomes the score which is directly based on an energy function.  No biochemistry background is needed to play the game, the most important principles are taught on introductory levels. The Foldit project addresses three major goals in parallel. By combining the use of structure prediction algorithms and human pattern recognition abilities, the game can be used to solve puzzles of proteins with unknown structures. Because the highest possible score is not known for these proteins, the players compete with each other. The predictions with the highest scores are then tested in the laboratory by scientists. Using this method, the structure of the M-PMV retroviral protease was solved in 2011 by the Baker laboratory [9]. Another long-term goal of Foldit is to improve the design process for completely new proteins. The significant improvement of a computationally designed Diels-Alderase through the Foldit community in 2012 demonstrated the successful integration of human creativity into the computational design approach [10]. Finally, players are also encouraged to code and share their strategies to find new algorithms and approaches which can be used to improve present structure prediction programs [11].
Figure 2.Screenshot of the EteRNA RNA folding game [12]
Similar to the protein folding project of Foldit, an RNA folding game called EteRNA was released in 2010 by researchers of the Carnegie Mellon University and the Stanford University. Players are confronted with a given target shape for which they have to design matching RNA sequences. The most successful designs are then synthesized and tested in the laboratory. The evaluation of folding patterns and the definition of a set of rules to eventually predict any given structures in a fast and precise manner are the main goals of this research  [12].
Figure 3.Screenshot of the Phylo sequence alignment game [13]
Also released in 2010 was Phylo, a game where sequence data from the UCSC Genome Browser needs to be aligned successfully [13] [14]. With the project, researchers from the McGill Centre for Bioinformatics at McGill University in Montreal try to complement multiple sequence alignment algorithms with human pattern recognition abilities.
Probably all of these games still need to fully prove their benefit when compared to current methods. Still they represent an interesting strategy to extend present approaches used in the respective fields. Addressing different goals like education and research at the same point makes a concrete evaluation of these gamification processes difficult and maybe future developments have to be awaited for clearer results about possible effects.
Conclusions
Although our excursion into synthetic biology games is by no means complete, we want to draw some conclusions. When trying to explain synthetic biology, especially to people without a biological background, a lot of the concepts are simplified with comparisons to Lego, not only since the Biobrick Foundation was started in 2006 [15]. The idea of interchangeable parts is crucial to the idea of introducing engineering principles into biology. Lego is nevertheless a game played mostly by children, a fact that could prove disadvantageous for the image of synthetic biology. Genetic engineering is seen as something potentially dangerous by the public, as something that certainly does not belong into the hands of children. On the other hand Lego is well-known and easily understandable. We think with our Colisweeper game we can demonstrate important mechanisms of synthetic biology in a comparably playful way, without too much reducing the complexity or the seriousness of research. We also find these qualities within the new field of citizen science. Games like Foldit and Phylo allow the public to actively take part in science and get a feeling of what the research is actually about. By taking such an explanatory route, the level of complexity as well as effort required by the audience can be adjusted to match tailored profiles.
References
[1] Ouyang, Q., et al., DNA solution of the maximal clique problem. Science, 1997. 278(5337): p. 446-9.
[2] Adleman, L.M., Molecular computation of solutions to combinatorial problems. Science, 1994. 266(5187): p. 1021-4.
[3] Sakamoto, K., et al., Molecular computation by DNA hairpin formation. Science, 2000. 288(5469): p. 1223-6.
[4] Braich, R.S., et al., Solution of a 20-variable 3-SAT problem on a DNA computer. Science, 2002. 296(5567): p. 499-502.
[5] Stojanovic, M.N. and D. Stefanovic, A deoxyribozyme-based molecular automaton. Nat Biotechnol, 2003. 21(9): p. 1069-74.
[6] Pei, R., et al., Training a molecular automaton to play a game. Nat Nanotechnol, 2010. 5(11): p. 773-7.
[7] Rosetta@home. Available from: http://boinc.bakerlab.org/rosetta/ .
[8] Gamification Synthetic Biology SB 6.0. Available from: http://sb6.biobricks.org/poster/gamifying-synthetic-biology-the-synmod-mobile-game/ .
[9] Khatib, F., et al., Crystal structure of a monomeric retroviral protease solved by protein folding game players. Nat Struct Mol Biol, 2011. 18(10): p. 1175-7.
[10] Eiben, C.B., et al., Increased Diels-Alderase activity through backbone remodeling guided by Foldit players. Nat Biotechnol, 2012. 30(2): p. 190-2.
[11] Foldit. Available from: http://fold.it/portal/ .
[12] EteRNA. Available from: http://eterna.cmu.edu/web/ .
[13] Phylo. Available from: http://phylo.cs.mcgill.ca/ .
[14] Kawrykow, A., et al., Phylo: a citizen science approach for improving multiple sequence alignment. PLoS One, 2012. 7(3): p. e31362.
[15] Biobrick Foundation. Available from: http://biobricks.org/about-foundation/ .
We thank our sponsors:
File:Basf-logo.gif
180px
100px
130px
Retrieved from " http://2013.igem.org/Team:ETH_Zurich/Human "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
