"
Page
Discussion
View source
History
teams
Log in
Â 
Team:ETH Zurich/Templates/Test
From 2013.igem.org
Home
Project
Information processing
Circuit and optimization
Pre-processing: Signal diffusion
Signal processing: High pass filters
Player interaction: Design and playing
Output system: Hydrolases
Data page
Experimental results
Overview
Pre-processing: Diffusion characterization
Proof-of-principle with GFP
Signal processing engineering
Circuit optimization
Output system characterization
Flagging
Gameplay
Model
Overview
Single Cell Model
AHL reaction-diffusion model
Model Evaluation: Genetic circuit with GFP
Genetic Circuit incorporating hydrolases
Analytical Approximations
Parameters
Lab work
Triple knockout strain
Constructs
Lab journal
Methods
Materials
Safety
Human practice
Overview
Worldwide playing
Gamification
Education
Team
Students
Advisors/Instructors
Official Team Profile
Attributions & contributions
References
Achievements
Achievements
Medal criteria
Awards
Login
WIKI
Log out
My account
Edit page
History
Upload new file
Final Circuit
For the final Colisweeper circuit we plan a four plasmid system. The mine cells constitutively express LuxI for signal generation and NagZ as identifier hydrolase. In the non-mine cells LuxR is expressed constitutively to process the OHHL signal. PhoA is expressed constitutively as well as reporter for safe cells.  Aes and GusA are expressed from pLux promoters with different sensitivities. You can find all the biobricks we used and our own new biobricks in the figure below.
Cloned Constructs
To get to the circuit mentioned above we tested different versions of the circuit. For example we started our experiments using GFP as a reporter instead of the hydrolases. Then we also tested different LuxI and LuxR generating constructs. In the following table we list all the biobricks we used, the plasmids we cloned and what experiments we used them for. In general we used standard biobrick cloning techniques as described in the methods section. Whenever we used PCR gene amplification for cloning, we list the primers used in the following table. To be able to co-transform different plasmids we used backbones with compatible origins of replication and resistance genes. In the table you can find which backbone versions we used for which constructs.
GFP constructs
Description
Cloning
Maps
1
Receiver cell construct for GFP diffusion experiments
BBa_J09855 backbone (SpeI, PstI) and BBa_E0840 insert (XbaI, PstI)
2
Library of the Receiver cell constructs
Using the BBa_J09855-E0840 construct a library with mutated pLux promoters was created through site-saturation mutagenesis to screen for promoters with changed sensitivities for OHHL:
3
Receiver cell construct for GFP experiments without the LuxR generating part
BBa_R0062 backbone (SpeI, PstI) and BBa_E0840 insert (XbaI, PstI)
LuxI generating constructs
Description
Cloning
Maps
4
Sender cell construct with a very strong constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments
BBa_J23100 backbone (SpeI, PstI) and BBa_K805016 insert (XbaI, PstI)
5
Sender cell construct with an intermediate constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments
BBa_J23118 backbone (SpeI, PstI) and BBa_K805016 insert (XbaI, PstI)
6
Sender cell construct with an intermediate constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments
BBa_J23110 backbone (SpeI, PstI) and BBa_K805016 insert (XbaI, PstI)
7
Sender cell construct with a weak constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments
BBa_J23114 backbone (SpeI, PstI) and BBa_K805016 insert (XbaI, PstI)
LuxR generating constructs
Description
Cloning
Maps
8
constitutive LuxR generating biobrick
BBa_J09855
9
constitutive LuxR generating biobrick, with negative feedback-loop at high OHHL concentrations
BBa_F2621
10
constitutive LuxR generating construct, repressible through LacI
BBa_R0010 backbone (SpeI, PstI) and BBa_I0462 insert (XbaI, PstI)
11
constitutive LuxR generating construct, with negative feedback-loop at high OHHL concentrations
BBa_R0063 backbone (SpeI,PstI) and BBa_I0462 insert (XbaI, PstI)
12
auto-inducible LuxR generating construct with positive feedback loop
BBa_R0062 backbone (SpeI, PstI) and BBa_I0462 insert (XbaI, PstI)
13
negatively regulated pLuxL-LacI construct to improve leakiness of LuxR system
BBa_R0063 backbone (SpeI, PstI) and BBa_C0012 insert (SpeI,PstI)
pLux constructs
Description
Cloning
Maps
Hydrolase constructs
Description
Cloning
Maps
We thank our sponsors:
File:Basf-logo.gif
180px
100px
130px
Retrieved from " http://2013.igem.org/Team:ETH_Zurich/Templates/Test "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
