<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Experiments"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Experiments">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Experiments&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Experiments" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Experiments</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A name="Project" style="cursor: pointer">Project</A></LI><LI><A href="/Team:ETH_Zurich/Infoproc" title="Team:ETH Zurich/Infoproc">Information processing</A></LI><LI><A href="/Team:ETH_Zurich/Circuit" title="Team:ETH Zurich/Circuit">Circuit and optimization</A></LI><LI><A href="/Team:ETH_Zurich/pre_proc" title="Team:ETH Zurich/pre proc">Pre-processing: Signal diffusion</A></LI><LI><A href="/Team:ETH_Zurich/Processing_2" title="Team:ETH Zurich/Processing 2">Signal processing: High pass filters</A></LI><LI><A href="/Team:ETH_Zurich/Play" title="Team:ETH Zurich/Play">Player interaction: Design and playing</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_7" title="Team:ETH Zurich/Experiments 7">Output system: Hydrolases</A></LI></UL><LI><A href="/Team:ETH_Zurich/Data_Page" title="Team:ETH Zurich/Data Page">Data page</A></LI><LI><A name="Biology" style="cursor: pointer">Experimental results</A></LI><LI><A href="/Team:ETH_Zurich/Experimentalresults" title="Team:ETH Zurich/Experimentalresults">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_2" title="Team:ETH Zurich/Experiments 2">Pre-processing: Diffusion characterization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_6" title="Team:ETH Zurich/Experiments 6">Proof-of-principle with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_5" title="Team:ETH Zurich/Experiments 5">Signal processing engineering</A></LI><LI><A href="/Team:ETH_Zurich/Optimization" title="Team:ETH Zurich/Optimization">Circuit optimization</A></LI><LI><A href="/Team:ETH_Zurich/Experiments_3" title="Team:ETH Zurich/Experiments 3">Output system characterization</A></LI><LI><A href="/Team:ETH_Zurich/Flagging" title="Team:ETH Zurich/Flagging">Flagging</A></LI><LI><A href="/Team:ETH_Zurich/gameplayexperiment" title="Team:ETH Zurich/gameplayexperiment">Gameplay</A></LI><LI><A name="Model" style="cursor: pointer">Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Overview" title="Team:ETH Zurich/Modeling/Overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/InitialStates" title="Team:ETH Zurich/Modeling/InitialStates">Single Cell Model</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Reaction_Diffusion_OOHL" title="Team:ETH Zurich/Modeling/Reaction Diffusion OOHL">AHL reaction-diffusion model</A></LI><LI><A href="/Team:ETH_Zurich/GFP" title="Team:ETH Zurich/GFP">Model Evaluation: Genetic circuit with GFP</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Genetic Circuit incorporating hydrolases</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Analytical_Approximations" title="Team:ETH Zurich/Modeling/Analytical Approximations">Analytical Approximations</A></LI><LI><A href="/Team:ETH_Zurich/Parameter" title="Team:ETH Zurich/Parameter">Parameters</A></LI><LI><A name="LAb work" style="cursor: pointer">Lab work</A></LI><LI><A href="/Team:ETH_Zurich/Mutant" title="Team:ETH Zurich/Mutant">Triple knockout strain</A></LI><LI><STRONG class="selflink">Constructs</STRONG></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Lab journal</A></LI><LI><A href="/Team:ETH_Zurich/Materials" title="Team:ETH Zurich/Materials">Methods</A></LI><LI><A href="/Team:ETH_Zurich/Materials_2" title="Team:ETH Zurich/Materials 2">Materials</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A name="Human" style="cursor: pointer">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Practiceoverview" title="Team:ETH Zurich/Practiceoverview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/World" title="Team:ETH Zurich/World">Worldwide playing</A></LI><LI><A href="/Team:ETH_Zurich/Human" title="Team:ETH Zurich/Human">Gamification</A></LI><LI><A href="/Team:ETH_Zurich/HEducational" title="Team:ETH Zurich/HEducational">Education</A></LI><LI><A name="Team" style="cursor: pointer">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Students</A></LI><LI><A href="/Team:ETH_Zurich/Supervisors" title="Team:ETH Zurich/Supervisors">Advisors/Instructors</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; contributions</A></LI><LI><A href="/Team:ETH_Zurich/References" title="Team:ETH Zurich/References">References</A></LI><LI><A name="Achievements" style="cursor: pointer">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/achievements" title="Team:ETH Zurich/achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal criteria</A></LI><LI><A href="/Team:ETH_Zurich/Awards" title="Team:ETH Zurich/Awards">Awards</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Experiments" class="external text" rel="nofollow">Login</A></LI><LI><A name="WIKI" style="cursor: pointer">WIKI</A></LI><LI><A href="/wiki/index.php?title=Special:UserLogout&amp;returnto=Team:ETH_Zurich/Home;returntoquery=action%3Dedit" title="Log out">Log out</A></LI><LI><A href="https://igem.org/User_Information" title="My account">My account</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Experiments&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2013.igem.org/wiki/index.php?title=Team:ETH_Zurich/Experiments&amp;action=history" class="external text" rel="nofollow">History</A></LI><LI><A href="http://2013.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Final_Circuit"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Final Circuit</SPAN></A></LI><LI class="toclevel-1"><A href="#Cloned_Constructs"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Cloned Constructs</SPAN></A></LI><LI class="toclevel-1"><A href="#Groupparts_.2F_Biobricks"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Groupparts / Biobricks</SPAN></A></LI><LI class="toclevel-1"><A href="#We_thank_our_sponsors:"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">We thank our sponsors:</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Final_Circuit">Final Circuit</SPAN></H1><P align="justify">For the final Colisweeper circuit we plan a four plasmid system. The mine cells constitutively express LuxI for signal generation and NagZ as identifier hydrolase. In the non-mine cells LuxR is expressed constitutively to process the AHL signal. To reduce the leakiness of the system we introduced the LacI repressor to reduce expression of LuxR in the uninduced state. At high AHL concentrations the pLuxL reporter is repressed leading to a positive feedback loop motif. PhoA as reporter for safe cells is expressed constitutively from the chromosome and is therefore not necessary as a plasmid. Aes and GusA are expressed from pLux promoters with different sensitivities. You can find all the biobricks we used and our own new biobricks in the figure below.</P><P><B>Figure 1. Plasmids in mine and non-mine cells: move the cursor over the separate parts to check which biobricks we used.</B></P><H1><SPAN class="mw-headline" id="Cloned_Constructs">Cloned Constructs</SPAN></H1><P align="justify">To get to the circuit mentioned above we tested different versions of the circuit. For example we started our experiments using GFP as a reporter instead of the hydrolases. Then we also tested different LuxI and LuxR generating constructs. In the following table we list all the biobricks we used, the plasmids we cloned and what experiments we used them for. In general we used standard biobrick cloning techniques as described in the methods section. Whenever we used PCR gene amplification for cloning, we list the primers used in the following table. To be able to co-transform different plasmids we used backbones with compatible origins of replication and resistance genes. In the table you can find which backbone versions we used for which constructs.</P><TABLE style="float:left;margin-top:10px;width:auto;height:auto;font-size:12px"><TBODY><TR><TH colspan="4" height="30px">Fluorescent protein reporter constructs</TH></TR><TR><TH width="475px" height="30px">Description</TH><TH width="475px" height="30px">Cloning</TH><TH width="350px" height="30px">Maps</TH></TR><TR><TD>1</TD><TD>Receiver cell construct for GFP diffusion experiments</TD><TD>Plac-LuxR-pLuxR <A href="http://parts.igem.org/Part:BBa_J09855" class="external text" rel="nofollow">BBa_J09855</A> backbone (SpeI, PstI) and <A href="http://parts.igem.org/Part:BBa_E0840" class="external text" rel="nofollow">BBa_E0840</A> insert (XbaI, PstI)</TD></TR><TR><TD>2</TD><TD>Library of the Receiver cell constructs</TD><TD>Using the Plac-LuxR-pLuxR BBa_J09855.BBa_E0840 construct a library with mutated pLux promoters was created through site-saturation mutagenesis to screen for promoters with changed sensitivitiesPrimers:5'-tatactagagac<B>nnn</B>taggatcgtacag5'-gatcgta<B>nnn</B>gtttacgcaagaaaatg5'-tagagac<B>nnn</B>taggatcgta<B>nnn</B>gtttacgcaagaaaatg5'-tagagacc<B>nn</B>taggatcgta<B>n</B>a<B>n</B>gtttacgcaagaaaatg5'-tagagacct<B>n</B>taggatcgtaca<B>n</B>gtttacgcaagaaaatgInteresting versions of the promoter were sequenced and inserted into pSB1C3 backbone using custom-made oligos. They could then be used for further cloning.</TD></TR><TR><TD>3</TD><TD>Receiver cell construct for GFP experiments without the LuxR generating part</TD><TD><A href="http://parts.igem.org/Part:BBa_R0062" class="external text" rel="nofollow">BBa_R0062</A> backbone (SpeI, PstI) and <A href="http://parts.igem.org/Part:BBa_E0840" class="external text" rel="nofollow">BBa_E0840</A> insert (XbaI, PstI)</TD></TR><TR><TD>4</TD><TD>Receiver cell construct for GFP experiments with positive feedback loop to reduce leakiness</TD><TD>Plac-LuxR-pLuxR BBa_J09855.BBa_E0840 construct where the pLac promoter was replaced with pLuxR to build a positive feedback loop. The promoter was inserted with two pairs of custom-made oligos using XbaI and HindIII restriction sites.Oligos:5â-ctagagacctgtaggatcgtacaggtttacgcaagaaaatggtttgttatagtcgaataaatactaga
5â-gattaaagaggagaaatactagatgaaaaacataaatgccgacgacacatacagaataattaataaaattaa5â-aatctctagtatttattcgactataacaaaccattttcttgcgtaaacctgtacgatcctacaggtct5â-agctttaattttattaattattctgtatgtgtcgtcggcatttatgtttttcatctagtatttctcctcttt</TD></TR><TR><TD>5</TD><TD>Receiver cell construct with mutated pLux promoter library and RFP reporter to test together with wild-type pLux GFP receiver cells in one cell</TD><TD>Library of mutated pLux promoters (see above) backbone (SpeI,PstI) and RFP (SpeI,PstI) insert from <A href="http://parts.igem.org/Part:BBa_J23118" class="external text" rel="nofollow">BBa_J23118</A></TD></TR></TBODY></TABLE><TABLE style="float:left;margin-top:10px;width:auto;height:auto;font-size:12px"><TBODY><TR><TH colspan="4" height="30px">LuxI generating constructs</TH></TR><TR><TH width="475px" height="30px">Description</TH><TH width="475px" height="30px">Cloning</TH><TH width="350px" height="30px">Maps</TH></TR><TR><TD>6</TD><TD>Sender cell construct with a very strong constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments</TD><TD><A href="http://parts.igem.org/Part:BBa_J23100" class="external text" rel="nofollow">BBa_J23100</A> backbone (SpeI, PstI) and <A href="http://parts.igem.org/Part:BBa_K805016" class="external text" rel="nofollow">BBa_K805016</A> insert (XbaI, PstI)</TD></TR><TR><TD>7</TD><TD>Sender cell construct with an intermediate constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments</TD><TD><A href="http://parts.igem.org/Part:BBa_J23118" class="external text" rel="nofollow">BBa_J23118</A> backbone (SpeI, PstI) and <A href="http://parts.igem.org/Part:BBa_K805016" class="external text" rel="nofollow">BBa_K805016</A> insert (XbaI, PstI)</TD></TR><TR><TD>8</TD><TD>Sender cell construct with an intermediate constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments</TD><TD><A href="http://parts.igem.org/Part:BBa_J23110" class="external text" rel="nofollow">BBa_J23110</A> backbone (SpeI, PstI) and <A href="http://parts.igem.org/Part:BBa_K805016" class="external text" rel="nofollow">BBa_K805016</A> insert (XbaI, PstI)</TD></TR><TR><TD>9</TD><TD>Sender cell construct with a weak constitutive promoter from the BBa_J23100 promoter library for GFP and Hydrolase experiments</TD><TD><A href="http://parts.igem.org/Part:BBa_J23114" class="external text" rel="nofollow">BBa_J23114</A> backbone (SpeI, PstI) and <A href="http://parts.igem.org/Part:BBa_K805016" class="external text" rel="nofollow">BBa_K805016</A> insert (XbaI, PstI)</TD></TR></TBODY></TABLE><TABLE style="float:left;margin-top:10px;width:auto;height:auto;font-size:12px"><TBODY><TR><TH colspan="4" height="30px">LuxR generating constructs</TH></TR><TR><TH width="475px" height="30px">Description</TH><TH width="475px" height="30px">Cloning</TH><TH width="350px" height="30px">Maps</TH></TR><TR><TD>10</TD><TD>constitutive LuxR generating biobrick</TD><TD>Plac-LuxR-pLuxR <A href="http://parts.igem.org/Part:BBa_J09855" class="external text" rel="nofollow">BBa_J09855</A></TD></TR><TR><TD>11</TD><TD>constitutive LuxR generating biobrick, with negative feedback-loop at high OHHL concentrations</TD><TD><A href="http://parts.igem.org/Part:BBa_F2621" class="external text" rel="nofollow">BBa_F2621</A></TD></TR><TR><TD>12</TD><TD>negatively regulated pLuxL-LacI construct to improve the leakiness problem of the LuxR system</TD><TD><A href="http://parts.igem.org/Part:BBa_R0063" class="external text" rel="nofollow">BBa_R0063</A> backbone (SpeI, PstI) and <A href="http://parts.igem.org/Part:BBa_J24679" class="external text" rel="nofollow">BBa_J24679</A> insert (XbaI, PstI)</TD></TR></TBODY></TABLE><TABLE style="float:left;margin-top:10px;width:auto;height:auto;font-size:12px"><TBODY><TR><TH colspan="4" height="30px"> pLuxR constructs </TH></TR><TR><TH width="475px" height="30px">Description</TH><TH width="475px" height="30px">Cloning</TH><TH width="350px" height="30px">Maps</TH></TR><TR><TD>13</TD><TD>Library of pLuxR promoter variants in pSB1C3 backbone for cloning</TD><TD> The promoter variants were chosen after characterization and sequencing with the GFP receiver library. The sequences were ordered as custom-made oligos and inserted into pSB1C3 using EcoRI and PstI restriction sites.Oligos:5'-accagtaggatcgtagaggtttacgcaagaaaatggtttgttatagtcgaataaa5'-accggtaggatcgtagaggtttacgcaagaaaatggtttgttatagtcgaataaa5'-acccgtaggatcgtagaggtttacgcaagaaaatggtttgttatagtcgaataaa5'-accagtaggatcgtaaaggtttacgcaagaaaatggtttgttatagtcgaataaa5'-accagtaggatcgtataggtttacgcaagaaaatggtttgttatagtcgaataaa</TD></TR></TBODY></TABLE><TABLE style="float:left;margin-top:10px;width:auto;height:auto;font-size:12px"><TBODY><TR><TH colspan="4" height="30px">Hydrolase constructs</TH></TR><TR><TH width="475px" height="30px">Description</TH><TH width="475px" height="30px">Cloning</TH><TH width="350px" height="30px">Maps</TH></TR><TR><TD>14</TD><TD>Aes coding region with RBS in SB1C3 backbone</TD><TD><A href="http://parts.igem.org/Part:BBa_K1216002" class="external text" rel="nofollow">BBa_K1216002</A> was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs.Primers:5'-gtttcttcgaattcgcggccgcttctagaggtttaactttaagaaggagatatagctagc5'-gtttcttcctgcagcggccgctactagtattattaaagctgagcggtaaagaactgag</TD></TR><TR><TD>15</TD><TD>Aes-His coding region with RBS in SB1C3 backbone</TD><TD><A href="http://parts.igem.org/Part:BBa_K1216006" class="external text" rel="nofollow">BBa_K1216006</A> was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs. Primers:5'-gtttcttcgaattcgcggccgcttctagaggtttaactttaagaaggagatatagctagc
5'-gtttcttcctgcagcggccgctactagtattattagtggtggtggtggtggtgctc</TD></TR><TR><TD>16</TD><TD>GusA coding region with RBS in SB1C3 backbone</TD><TD><A href="http://parts.igem.org/Part:BBa_K1216000" class="external text" rel="nofollow">BBa_K1216000</A> was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs. Primers:5'-gtttcttcgaattcgcggccgcttctagagtttgtttaactttaagaaggagatatacat5'-gtttcttcctgcagcggccgctactagtattattattgtttgcctccctgctgcg</TD></TR><TR><TD>17</TD><TD>GusA-His coding region with RBS in SB1C3 backbone</TD><TD><A href="http://parts.igem.org/Part:BBa_K1216004" class="external text" rel="nofollow">BBa_K1216004</A> was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs. Primers:5'-gtttcttcgaattcgcggccgcttctagagtttgtttaactttaagaaggagatatacat5'-gtttcttcctgcagcggccgctactagtattattagtggtggtggtggtggtgctc</TD></TR><TR><TD>18</TD><TD>PhoA coding region with RBS in SB1C3 backbone</TD><TD><A href="http://parts.igem.org/Part:BBa_K1216001" class="external text" rel="nofollow">BBa_K1216001</A> was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs. Primers:5'-gtttcttcgaattcgcggccgcttctagagtttgtttaactttaagaaggagatatacat5'-gtttcttcctgcagcggccgctactagtattattacttcaggcccagcgccgctttc</TD></TR><TR><TD>19</TD><TD>PhoA-His coding region with RBS in SB1C3 backbone</TD><TD><A href="http://parts.igem.org/Part:BBa_K1216005" class="external text" rel="nofollow">BBa_K1216005</A> was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs. Primers:5'-gtttcttcgaattcgcggccgcttctagagtttgtttaactttaagaaggagatatacat5'-gtttcttcctgcagcggccgctactagtattattagtggtggtggtggtggtgctc</TD></TR><TR><TD>20</TD><TD>NagZ coding region with RBS in SB1C3 backbone</TD><TD><A href="http://parts.igem.org/Part:BBa_K1216003" class="external text" rel="nofollow">BBa_K1216003</A> was built through touchdown PCR amplification. Biobrick prefix and suffix were added with overhangs. Primers:5'-gtttcttcgaattcgcggccgcttctagaggtttaactttaagaaggagatatagctagc5'-gtttcttcctgcagcggccgctactagtattattactcgtgacctgctttctcttc</TD></TR><TR><TD>21</TD><TD>constitutive expression of PhoA for non-mine cells</TD><TD>constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23100" class="external text" rel="nofollow">BBa_J23100</A> (SpeI, PstI) backbone and <A href="http://parts.igem.org/Part:BBa_K1216001" class="external text" rel="nofollow">BBa_K1216001</A>insert (XbaI, PstI)</TD></TR><TR><TD>22</TD><TD>constitutive expression of NagZ for mine cells</TD><TD>constitutive promoter <A href="http://parts.igem.org/Part:BBa_J23100" class="external text" rel="nofollow">BBa_J23100</A> (SpeI, PstI) backbone and <A href="http://parts.igem.org/Part:BBa_K1216003" class="external text" rel="nofollow">BBa_K1216003</A>insert (XbaI, PstI)</TD></TR><TR><TD>23</TD><TD>AHL inducible expression of GusA</TD><TD>Plac-LuxR-pLuxR <A href="http://parts.igem.org/Part:BBa_J09855" class="external text" rel="nofollow">BBa_J09855</A> (SpeI, PstI) backbone and <A href="http://parts.igem.org/Part:BBa_K1216000" class="external text" rel="nofollow">BBa_K1216000</A>insert (XbaI, PstI)</TD></TR><TR><TD>24</TD><TD>AHL inducible expression of GusA with positive feedback loop for LuxR expression</TD><TD>Plac-LuxR-pLuxR BBa_J09855.BBa_K1216000 construct where the pLac promoter was replaced with pLuxR to build a positive feedback loop. The promoter was inserted with two pairs of custom-made oligos using XbaI and HindIII restriction sites.Oligos:5â-ctagagacctgtaggatcgtacaggtttacgcaagaaaatggtttgttatagtcgaataaatactaga
5â-gattaaagaggagaaatactagatgaaaaacataaatgccgacgacacatacagaataattaataaaattaa5â-aatctctagtatttattcgactataacaaaccattttcttgcgtaaacctgtacgatcctacaggtct5â-agctttaattttattaattattctgtatgtgtcgtcggcatttatgtttttcatctagtatttctcctcttt</TD></TR><TR><TD>25</TD><TD>AHL inducible expression of Aes</TD><TD>Plac-LuxR-pLuxR <A href="http://parts.igem.org/Part:BBa_J09855" class="external text" rel="nofollow">BBa_J09855</A> (SpeI, PstI) backbone and <A href="http://parts.igem.org/Part:BBa_K1216002" class="external text" rel="nofollow">BBa_K1216002</A>insert (XbaI, PstI)</TD></TR><TR><TD>26</TD><TD>AHL inducible expression of Aes with mutant pLux promoter</TD><TD>Library of mutated pLux promoters (see above) backbone (SpeI,PstI) and <A href="http://parts.igem.org/Part:BBa_K1216002" class="external text" rel="nofollow">BBa_K1216002</A> (SpeI,PstI) insert</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Groupparts_.2F_Biobricks">Groupparts / Biobricks</SPAN></H1><P> In the following table you can find all the biobricks that were submitted by our group. </P><P>&lt;groupparts&gt;iGEM2013 ETH_Zurich&lt;/groupparts&gt;</P></DIV><DIV id="eth_footer"><H1><SPAN class="mw-headline" id="We_thank_our_sponsors:">We thank our sponsors:</SPAN></H1><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Basf-logo.gif" title="File:Basf-logo.gif">File:Basf-logo.gif</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Novartis-logo.gif" title="File:Novartis-logo.gif">180px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Roche-logo.gif" title="File:Roche-logo.gif">100px</A></DIV></DIV><DIV class="sponsors"><DIV class="floatleft"><A href="/File:Syngenta-logo.gif" title="File:Syngenta-logo.gif">130px</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:ETH_Zurich/Experiments">http://2013.igem.org/Team:ETH_Zurich/Experiments</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Experiments" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Experiments" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Experiments&amp;oldid=350327" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>