"
Page
Discussion
View source
History
teams
Log in
 
Team:ETH Zurich
From 2013.igem.org
Home
Project
Information processing
Circuit and optimization
Pre-processing: Signal diffusion
Signal processing: High pass filters
Player interaction: Design and playing
Output system: Hydrolases
Data page
Experimental results
Overview
Pre-processing: Diffusion characterization
Proof-of-principle with GFP
Signal processing engineering
Circuit optimization
Output system characterization
Flagging
Gameplay
Model
Overview
Single Cell Model
AHL reaction-diffusion model
Model Evaluation: Genetic circuit with GFP
Genetic Circuit incorporating hydrolases
Analytical Approximations
Parameters
Lab work
Triple knockout strain
Constructs
Lab journal
Methods
Materials
Safety
Human practice
Overview
Worldwide playing
Gamification
Education
Team
Students
Advisors/Instructors
Official Team Profile
Attributions & contributions
References
Achievements
Achievements
Medal criteria
Awards
Login
WIKI
Log out
My account
Edit page
History
Upload new file
Play Minesweeper here !
Colisweeper
Colisweeper is an interactive, biological version of the computer game Minesweeper. The goal is to clear an agar “minefield” without detonating the mines. Genetically engineered Escherichia coli colonies are used as mines and non-mines. Mines secrete the signaling molecule AHL whereas non-mines process the signal. To distinguish different AHL-levels, a library of PLuxR promoters with various AHL sensitivities was created through site-saturation mutagenesis. High-pass filters were constructed to control the expression of different orthogonal hydrolases in non-mines, depending on the number of surrounding mines.
The slideshow covers all aspects of our project. Feel free to CLICK ON ANY PICTURE in ANY SLIDESHOW to navigate to the description page.
CLICK ON THE START BUTTON OF THE FIRST SLIDE TO SEE OUR VIDEO
How to play Colisweeper
Initially, an agar plate is prepared with mine and non mine colonies in the hexagonal grid pattern. To play the game, the player pipettes a solution on to a colony of choice. The color developed on this colony will determine the next move for the player. If the playing solution is pipetted on a colony and it turns yellow, salmon or magenta; this indicates zero,one or two mines respectively adjacent to the played colony. In case a mine colony is encountered, the colony turns blue and the game is over. If the player decides to flag a mine colony, the flagging solution is pipetted that turns the colony green in color. This way the color logic gives the player cue for the next move in the game.
Information Processing
Sender cells (mines) express LuxI protein, which catalyzes the production of a small signalling molecule, called AHL. This molecule diffuses through the agar plate and reaches the receiver cells called non-mine colonies. These colonies are designed to distinguish different concentrations of AHL and translate this analog information into the expression of different sets of reporter enzymes. The continuous signal is digitized through a set of mutated PLuxR promoters with  different AHL sensitivities acting as highpass filters.
From Minesweeper to Colisweeper
Mines secrete the signaling molecule AHL which diffuses through the agar and is processed by neighboring non-mine colonies. High-pass filters were constructed to control the expression of different orthogonal hydrolase enzymes in non-mines. Promoters that serve as high-pass filters were tuned to express hydrolases depending on the concentration of the AHL molecules from the surrounding mines. The colors yellow, salmon and magenta corresponds to zero, one and two mines around a colony. Additionally, the mines express their own hydrolase which when added with the multi-substrate gives blue color. The genomic expression of lacZ enables the flagging of both mines and non mine colonies turning the colonies green.
Hydrolase Reactions
As a reporter system we use a set of orthogonal hydrolase enzymes: alkaline phosphatase (phoA), β-galactosidase (lacZ), acetylesterase (aes), β-N-Acetylglucosaminidase (nagZ) and β-glucuronidase (gusA). Each hydrolase can react with its respective substrate within minutes resulting in a fast, colorful output. Quick response times and the ability to read the output without using instruments are essentials for a fast gameplay.
The Model
As our bio-game is based on processing the AHL concentration in the non-mine colonies, the diffusion of AHL in the agar is vital to the system. The diffusion was modeled by carrying out simulations to determine the time and distance of diffusion. We also modeled synthesis, regulation and degradation reactions of the proteins involved in our genetic circuits. To account for both processes: diffusion and reactions; we developed a spatio-temporal model in two dimensions comprised by three modules: mines, receivers, and the agar plate. Finite element methods were used to solve the system of partial differential equations (PDEs). Our model turned out to be very valuable in the circuit refinement and the design of experiments. Moreover, we continually improve out model by incorporating parameters from our own experimental data.
Experimental Results
Diffusion experiments were performed to determine the time and distance of AHL diffusion between colonies in the agar mine-grid. A symbiotic relation between experiment and model proved to be beneficial. As proof-of-principle, we set up diffusion experiments using GFP as reporter. The LuxR promoter from the registry was mutated to obtain a library of PLuxR promoters with different sensitivities in order to distinguish different AHL levels. Initial tests suggested leaky reporter expression in the uninduced colonies. As a solution to this problem, we use glucose to shut down the Placpromoter and a positive feedback loop using lacI under the PLuxL promoter. Meanwhile we characterize the biobricks using Michealis-Menten kinetics and flow cytometry.
Human practice
We analyze the concept of gamification in synthetic biology, describing many examples where common games are played in a new way or where games are used as research tools. We discuss possible consequences for synthetic biology and show how Colisweeper could be used for awareness raising. For this purpose we designed a Colisweeper Laboratory Course Kit and we propose an idea of a web-based Colisweeper platform enabling people from across the world to play against each other using a remotely controlled robot.
Team
We are a team of seven highly motivated Bachelor- and Master Students at ETH Zurich pursuing various fields such as Biotechnology,  Biomedical Engineering, Neurobiology and Bioinformatics. The iGEM project is carried out at one of the youngest departments of ETHZ located in Basel-Department of Biosystems Science and Engineering - flourishing in interdisciplinary biological research. If you're around Basel, make sure to visit our team's lab to play the bio-game Colisweeper!
We thank our sponsors:
File:Basf-logo.gif
180px
100px
130px
Retrieved from " http://2013.igem.org/Team:ETH_Zurich "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
