<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NU_Kazakhstan_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NU Kazakhstan/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Bioremediation of Sour Crude Oil Waste using Cyanobacteria</TITLE></P><DIV class="oz-body-wrap"><HEADER class="default-header"><DIV class="container-fluid"><DIV class="header-wrap"><DIV class="header-top d-flex justify-content-between align-items-center"><DIV class="main-menubar d-flex align-items-center"><NAV class="nav navbar"><A href="https://2018.igem.org/Team:NU_Kazakhstan">Homepage</A><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Description">Overview &amp; Background</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Model">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Future_Plans">Future Plans</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Team">About us</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Project_Timeline">Project Timeline</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Public_Engagement">Public engagement and Education</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Human_Practices">Integrated human practices</A></LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Judging_form">Judging form</A></NAV></DIV></DIV></DIV></DIV></HEADER><SECTION class="generic-banner relative" style="background: url(http://highline.codal.kz/img/design.jpg); background-size: cover; "><DIV class="container"><DIV class="row height align-items-center justify-content-center"><DIV class="col-lg-10"><DIV class="col-lg-12"><DIV class="banner-content text-center"><H1 style="color: #fff; border-bottom: none; font-weight: bold!important">Design</H1></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="sample-text-area"><DIV class="container"><DIV class="row"><DIV class="col-md-12" style="padding: 0px"><P>
								Our goal is to reduce H<SUB><FONT color="black">2</FONT></SUB>S in oil wastewater using Sulfide-Quinone Reductase in genetically modified <I><FONT color="black">Synechococcus elongatus</FONT></I> PCC 7942 and use products of the reaction to produce hydrogen and utilize sulfur-rich biomass in the production of the catalytically active material.</P><P><B><I>Synechococcus elongatus</I> PCC 7942</B></P><P><I><FONT color="black">Synechococcus elongatus</FONT></I> PCC 7942 is a freshwater obligate photoautotroph [6]. Was first reliably transformed cyanobacterium [6]. One of the most studied cyanobacteria and used as a model organism with well-investigated chromosome and plasmids sequences [6].</P><P><B>PHOTOSYNTHESIS</B></P><P>Being an essential part of an organism’s metabolism photosynthetic pathways in <I><FONT color="black">S. elongatus</FONT></I> is the main subject of modification. D1 protein of Photosystem II acts as reaction center in the splitting of water and this is the main target to reach inhibition of PS II [5]. There is a natural inhibition of PSII in the minimal concentration of H<SUB><FONT color="black">2</FONT></SUB>S (60 uM)[1].</P><P><B>SULFIDE-QUINONE REDUCTASE</B></P><P>SQR catalyzes sulfide-dependent plastoquinone reduction in anaerobic conditions. In our project, SQR originates from organism <I><FONT color="black">Leptolyngbya hensonii</FONT></I> cyanobacteria that can live in sulfidic conditions and change metabolism back to oxygenic photosynthesis accordingly to conditions [2], [3], [4]. To force cyanobacteria to use SQR, Photosystem II will be inhibited by H<SUB><FONT color="black">2</FONT></SUB>S and psbA1 gene. Produced sulfur will be in the form of polysulfide and remain in the cell.</P></DIV></DIV><DIV class="row"><B>Construct assembly design</B><P>Two transcriptional units were designed so that there are sequences, which enable primers to be of good Tm (melting temperature). The primers 1-3 are used to amplify genes by PCR. Primer 1 is the same on both transcriptional units because they both have an anchor, which is derived from the phycocyanin on thylakoid membrane of Synechococcus elongatus PCC 7942 (the organism being used in our experiments). Sequences can be anything as long as they make good Tm.</P></DIV><DIV class="row"><DIV class="col-md-12 text-center"><CENTER>
Figure 1. Transcriptional units for the project.</CENTER></DIV></DIV><P>The list of designed primers
Primer 1, Forward for SQR and Supernova
5’ GATATGAAAGTGCTGGCCAGCTTTGC 3’ </P><P>Primer 2, Reverse for SQR
5’ GTATATATATGCTTTTGCTGGCAAAGGCCTGG 3’
</P><P>
Primer 3, Reverse Supernova
5’ CGCTATCTTCATCGCTGCCGATGG 3’
</P><P>
Primer design for insertion into pSyn_6 vector
pSyn_6 vector with strong psbAI promoter was ordered from Thermofisher company. The piece with a gene of interest and spectinomycin is inserted into Neutral Site 1 (NS1) of Synechococcus elongatus PCC 7942 genome. PCR amplified transcriptional units went through one more PCR reaction, where BglII from 5’ end and KpnI+ScaI from 3’ end were added into the transcriptional units. BglII and KpnI combination of digestion/ligation can be used to insert the gene of interest if V5 epitope along with 6xHis tag wants to be kept. However, if V5 epitope and 6xHis tag are not needed, BglII and ScaI combination of digestion/ligation can be used. Also, start codon is added with a forward primer and stop codon is added with a reverse primer. Primers were designed to make sure that there is no codon shift after the gene insert.
</P><CENTER>
Figure 2. Multiple cloning site of pSyn_6 vector</CENTER><CENTER>
Figure 3. pSyn_6 vector from Thermofisher
</CENTER><CENTER>
Figure 4. Design of primers for insertion into multiple cloning sites of pSyn_6 vector (SQR insertion primers as an example).
</CENTER><P>
Primer design for addition of CPEC sites in order to insert to pSB1C3  submission vector
pSB1C3 linearized backbone from submission kit was amplified by PCR reaction using primers:</P><P>
Forward for pSB1C3
5’ TCTAGAAGCGGCCGCGAATTC 3’
</P><P>
Reverse for pSB1C3
5’ TACTAGTAGCGGCCGCTGCAG 3’
</P><P>Each part was amplified using primers that add Circular Polymerase Extension Cloning (CPEC) sites (Biobrick prefix by forward and suffix by reverse primers) to the part. Each primer added the half (~11 bp) of prefix/suffix because if they were to add whole prefix/suffix at once, the Tm of primers would be too high. Therefore, each part had undergone two PCR reactions to get whole prefix/suffix flanking sites. Also, note that Supernova and SQR both have the same Forward primers as they have the same starting sequence. The following primers were used: </P><P>  1.1) SQR/Supernova Submission Forward Primer 1
5’ CGCTTCTAGAGATGAAAGTGCTGGCCAGCTTTGC 3’</P><P>  1.2) SQR/Supernova Submission Forward Primer 2
5’ GAATTCGCGGCCGCTTCTAGAGATGAAAGTGCTGG 3’</P><P>  2.1) SQR Submission Reverse Primer 1
5’ CTGCAGCGGCCGCTACTAGTACCCCTACCC 3’
</P><P>  2.2) SQR Submission Reverse Primer 2
5’ GCTACTAGTACCCCTACCCCACCGAGTG 3’</P><P>  3.1)Supernova Submission Reverse Primer 1
5’ CTGCAGCGGCCGCTACTAGTAATCTTCATCG 3’
</P><P>  3.2)Supernova Submission Reverse Primer 2
5’ CGCTACTAGTAATCTTCATCGCTGCCGATG 3’
</P><CENTER>
Figure 5. CPEC scheme for the submission of parts.</CENTER><H6>References:</H6><OL><LI>1. Cohen, Y., Jørgensen, B.B., Revsbech, N.P. and Poplawski, R., 1986. Adaptation to hydrogen sulfide of oxygenic and anoxygenic photosynthesis among cyanobacteria. Applied and Environmental Microbiology, 51(2), pp.398-407.</LI><LI>2. Encyclopedia of Life. (n.d.). Geitlerinema splendidum - Overview - Encyclopedia of Life. [online] Available at: http://eol.org/pages/919054/overview [Accessed 15 Sep. 2018].</LI><LI>3. Hamilton, T.L., Klatt, J.M., De Beer, D. and Macalady, J.L., 2018. Cyanobacterial photosynthesis under sulfidic conditions: insights from the isolate Leptolyngbya sp. strain hensonii. The ISME journal, 12(2), p.568.</LI><LI>4. Strunecky, O., Bohunicka, M., Johansen, J., Capkova, K., Raabova, L., Dvorak, P. and Komarek, J. (2017). A revision of the genus Geitlerinema and a description of the genus Anagnostidinema gen. nov. (Oscillatoriophycidae, Cyanobacteria). Fottea, 17(1), pp.114-126.</LI><LI>5. Uniprot.org. (n.d.). psbA - Photosystem II protein D1 precursor - Arabidopsis thaliana (Mouse-ear cress) - psbA gene &amp; protein. [online] Available at: https://www.uniprot.org/uniprot/P83755 [Accessed 15 Sep. 2018].</LI><LI>6. Uniprot.org. (n.d.). Synechococcus elongatus (strain PCC 7942) (Anacystis nidulans R2). [online] Available at: https://www.uniprot.org/proteomes/UP000002717 [Accessed 15 Sep. 2018].</LI></OL></DIV></SECTION><FOOTER class="section-gap"><DIV class="container"><DIV class="row pt-60"><DIV class="col-lg-6 col-sm-12"><DIV class="single-footer-widget"><H6 class="text-uppercase mb-20">Quick About</H6><P>
								SCHOOL OF SCIENCE AND TECHNOLOGY  Nazarbayev University  Astana, Kazakhstan
							</P></DIV></DIV><DIV class="col-lg-3 col-sm-12"><DIV class="single-footer-widget"><H6 class="text-uppercase mb-20">Contacts</H6><P>
								igem@nu.edu.kz
							</P></DIV></DIV><DIV class="col-lg-3 col-sm-12"><DIV class="single-footer-widget"><H6 class="text-uppercase mb-20">Social networks</H6></DIV></DIV></DIV><DIV class="footer-bottom d-flex justify-content-between align-items-center flex-wrap"><P class="foter-text m-0">Copyright © All rights reserved</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>