<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NU_Kazakhstan_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NU Kazakhstan/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Bioremediation of Sour Crude Oil Waste using Cyanobacteria</TITLE></P><DIV class="oz-body-wrap"><HEADER class="default-header"><DIV class="container-fluid"><DIV class="header-wrap"><DIV class="header-top d-flex justify-content-between align-items-center"><DIV class="main-menubar d-flex align-items-center"><NAV class="nav navbar"><A href="https://2018.igem.org/Team:NU_Kazakhstan">Homepage</A><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project </A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Description">Overview &amp; Background</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Model">Modeling</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Future_Plans">Future Plans</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Safety">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Team </A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Team">About us</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Notebook </A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Project_Timeline">Project Timeline</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Public_Engagement">Public engagement and Education</A></LI><LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Human_Practices">Integrated human practices</A></LI><A href="https://2018.igem.org/Team:NU_Kazakhstan/Judging_form">Judging form</A></NAV></DIV></DIV></DIV></DIV></HEADER><SECTION class="generic-banner relative" style="background: url(http://highline.codal.kz/img/results.jpg); background-size: cover; "><DIV class="container"><DIV class="row height align-items-center justify-content-center"><DIV class="col-lg-10"><DIV class="col-lg-12"><DIV class="banner-content text-center"><H1 style="color: #fff; border-bottom: none; font-weight: bold!important">Results</H1></DIV></DIV></DIV></DIV></DIV></SECTION><SECTION class="sample-text-area"><DIV class="container"><DIV class="row"><DIV class="col-md-12"><B>Vector construction</B></DIV></DIV><DIV class="row"><DIV class="col-md-8">
					For the vector construction, SQR gene was cloned into pSyn_6 vector, and through gel electrophoresis was checked the gene assembly. Figure 1 illustrates the bands of cloned pSyn_6 (5577 bp) in 1-4 wells between 3 and 4 ladder bands, which confirms the success of gene assembly. Also, cloned pSyn_6 plasmid was tested on a presence of SQR gene by PCR amplification using SQR primers. In Figure 2, we can see SQR bands (1271 bp) between 8 and 9 ladder bands that evidences of SQR presence in cloned pSyn_6 plasmid.  
					</DIV></DIV><DIV class="row text-center"><DIV class="col-md-6 text-center"><P>Figure 1. Agarose gel electrophoresis (1%) of cloned pSyn_6 plasmid with SQR (5577 bp).</P></DIV><DIV class="col-md-6 text-center"><P>Figure 2. Agarose gel electrophoresis (1%) of PCR amplified products using SQR primers to test its presence in cloned pSyn_6.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><B>Transformation of S. elongatus PCC 7942 with SQR</B><P>After the gene construction, all attention was focused on transformation of cyanobacteria with SQR gene </P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><P>Figure 3. Lamp with high light intensity.</P></DIV><DIV class="col-md-6"><P>Figure 4. Cyanobacteria colonies in BG-11 agar plates with spectinomycin.</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><B>Transformation verification</B><P>Confirmation of integration of the cloned pSyn_6 plasmid into genome was done by colony PCR amplification using SQR primers. In Figure 5, we can see colonies with inserted SQR genes, which were used to get liquid genetically modified cyanobacteria culture. </P></DIV></DIV><DIV class="row"><P>Figure 5. Agarose gel electrophoresis of colony PCR amplified products using SQR primers. </P></DIV><DIV class="row"><DIV class="col-md-12"><B>Validation of a Part</B><P>We inserted Sulfide Quinone Reductase into the pSyn_6 сyanobacterial protein expression vector, which performs the function of converting toxic hydrogen sulfide into elemental sulfur. Firstly, we tested the survivability of our cyanobacteria in different concentrations of Na<SUB><FONT color="black">2</FONT></SUB>S and oil. Secondly, to check the functionality of the SQR gene in cyanobacteria we conducted the assay of Na<SUB><FONT color="black">2</FONT></SUB>S reduction.

</P><B>Survival test of cyanobacteria in the oil</B><P>Na<SUB><FONT color="black">2</FONT></SUB>S used in the experiments was impure. It was obtained from the industrial waste provided by the MKA engineering company. According to their data, Na<SUB><FONT color="black">2</FONT></SUB>S content was at least 60% and the rest were impurities including heavy metals. The stock solutions of sodium sulfide were prepared assuming that the weight percent of 60%.
We conducted survival test of genetically modified and wild-type cyanobacteria in Na<SUB><FONT color="black">2</FONT></SUB>S. On the 2nd day of the experiment,  genetically modified cyanobacteria were identified to be more tolerant to the toxicity of Na<SUB><FONT color="black">2</FONT></SUB>S in the solution since the wild-type strain started to show a decrease in the growth in the 500 μM and 1 mM Na<SUB><FONT color="black">2</FONT></SUB>S solutions (figure 6).  
 </P><P>Figure 6. Survival test of cyanobacteria in different concentrations of Na<SUB><FONT color="black">2</FONT></SUB>S</P></DIV></DIV><DIV class="row"><DIV class="col-md-12"><B>Na<SUB>2</SUB>S reduction assay</B><P>
The quantitative assay was performed with two concentrations of Na<SUB><FONT color="black">2</FONT></SUB>S (0.5mM and 1mM) using Nanodrop 8000 UV-Vis spectrophotometer, which was set up at the absorbance of 230 nm [1]. The measurements were done before pH adjustment and after pH adjustment to 12. Measurements with pH-adjusted samples were needed to be done in order to minimize the effect of industrial waste content on the results of the assay. The impure Na<SUB><FONT color="black">2</FONT></SUB>S solution contains various metals, which might unfavorably react with SH-. Increasing the pH may reduce the negative effect of metals, reacting predominantly with OH<SUP><FONT color="black">-</FONT></SUP>, rather than creating the extra binding to SH<SUP><FONT color="black">-</FONT></SUP>.</P><P>

Figure 7 shows the similar absorbance at 230 nm in SQR- and SQR+ strains, however, further measurements indicated a considerable decrease of absorbance at 230 nm in SQR+ strain. The ion transport within the cells could explain the irregular trend of SQR- samples. As for the figure 8, there is no clear difference between SQR- and SQR+ cultures, which can be explained by the impact of the industrial waste composition.
</P><DIV class="row"><DIV class="col-md-6"><P>Figure 7. Absorbance values at 230 nm in 500 μM Na<SUB><FONT color="black">2</FONT></SUB>S at pH 12.</P></DIV><DIV class="col-md-6"><P>Figure 8. Average absorbance values at 230 nm in 500 μM Na<SUB><FONT color="black">2</FONT></SUB>S at not adjusted pH</P></DIV></DIV></DIV></DIV><DIV class="row"><P>In order to confirm the work of SQR and show the greater dynamics of bisulfide reduction, our team decided to increase the concentration of sodium sulfide and conduct the assay at 1 mM Na<SUB><FONT color="black">2</FONT></SUB>S.
</P><P>
As shown in Table 1, SQR- strain and SQR+ strain without Na<SUB><FONT color="black">2</FONT></SUB>S do not show any peaks at 230 nm, whereas the positive control (Bg11 with Na<SUB><FONT color="black">2</FONT></SUB>S) demonstrated the high peak at the same absorbance. Therefore, the presence of bisulfide could be indicated by the measurement at 230 nm.
</P><P>The assay demonstrated higher dynamics of bisulfide reduction during the experiment with 1 mM. Figure 9 and 10 illustrate the considerable reduction of bisulfide in SQR+ in both conditions. While, the SQR- strain indicated the fluctuating measurements. 
</P><DIV class="row"><DIV class="col-md-6"><P>Figure 9. Average absorbance values at 230 nm in 1 mM Na<SUB><FONT color="black">2</FONT></SUB>S at pH 12 </P></DIV><DIV class="col-md-6"><P>Figure 10. Average absorbance values at 230 nm in 1 mM Na<SUB><FONT color="black">2</FONT></SUB>S at not adjusted pH</P></DIV></DIV><P>Both assay cultures (SQR- and SQR+) had similar initial OD750. Difference between initial and final (2 hours) OD750 measurements of both samples indicate a different effect of Na<SUB><FONT color="black">2</FONT></SUB>S on SQR+ and SQR- cultures. As shown in Table 2 the presence of Na<SUB><FONT color="black">2</FONT></SUB>S in SQR+ sample does not inhibit the growth of cyanobacteria that is supported by the positive average difference in OD750 in table 2. Meanwhile, the OD750 of SQR- strain decreased.</P><DIV class="row"><B>Survival test of cyanobacteria in the oil</B><P>
Since the main goal of the project is the bioremediation of oil wastewater, we tested the survivability of genetically modified and wild-type strains of cyanobacteria in different concentrations of oil. The samples were constantly shaken on the agitator. After 3 days since the addition of oil (Figure 11-16), it was identified that cyanobacteria with SQR had higher survival in 0.1%, 0.5%, and 1% oil solutions compared to the wild-type strain.
</P></DIV><DIV class="row"><DIV class="col-md-3"><P>Figure 11. Survival test of cyanobacteria in 0% oil after 3 days. Control.</P></DIV><DIV class="col-md-3"><P>Figure 12. Survival test of cyanobacteria in 0.1% oil after 3 days of incubation.</P></DIV><DIV class="col-md-3"><P>Figure 13. Survival test of cyanobacteria in 0.5% oil after 3 days of incubation.</P></DIV><DIV class="col-md-3"><P>Figure 14. Survival test of cyanobacteria in 1% oil after 3 days of incubation.</P></DIV></DIV><DIV class="row"><DIV class="col-md-6"><P>Figure 15. Survival test of cyanobacteria SQR-</P></DIV><DIV class="col-md-6"><P>Figure 16. Survival test of cyanobacteria SQR+</P></DIV></DIV><DIV class="row"><P>As seen in Table 3, the survival of cyanobacteria differs between the wild-type and transformed cyanobacteria. The OD750 measurements indicate that growth is impaired in SQR- cultures. Cyanobacteria undergo survival issues to a greater extent with increasing oil content compared to the control. Although SQR+ culture demonstrates a similar trend, the decline in OD750 with an increasing oil concentration is not significant. On the contrary, SQR+ culture has shown to grow faster in 0.1% concentration compared to the control. These results imply that SQR+ are more competent to live in oily conditions with an optimal concentration of 0.1% oil content.

</P></DIV><DIV class="row"><B>References</B><P>1.Sutherland-Stacey, L., Corrie, S., Neethling, A., Johnson, I., Gutierrez, O., Dexter, R., ... &amp; Hamilton, G. (2008). Continuous measurement of dissolved sulfide in sewer systems. Water Science and technology, 57(3), 375-381.
 </P></DIV></DIV></DIV></SECTION><FOOTER class="section-gap"><DIV class="container"><DIV class="row pt-60"><DIV class="col-lg-6 col-sm-12"><DIV class="single-footer-widget"><H6 class="text-uppercase mb-20">Quick About</H6><P>
								SCHOOL OF SCIENCE AND TECHNOLOGY  Nazarbayev University  Astana, Kazakhstan
							</P></DIV></DIV><DIV class="col-lg-3 col-sm-12"><DIV class="single-footer-widget"><H6 class="text-uppercase mb-20">Contacts</H6><P>
								igem@nu.edu.kz
							</P></DIV></DIV><DIV class="col-lg-3 col-sm-12"><DIV class="single-footer-widget"><H6 class="text-uppercase mb-20">Social networks</H6></DIV></DIV></DIV><DIV class="footer-bottom d-flex justify-content-between align-items-center flex-wrap"><P class="foter-text m-0">Copyright © All rights reserved</P></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>