<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HIT-Harbin_WholeNameList"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HIT-Harbin/WholeNameList">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HIT-Harbin/WholeNameList&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HIT-Harbin/WholeNameList&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HIT-Harbin/WholeNameList&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HIT-Harbin/WholeNameList" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HIT-Harbin/WholeNameList</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV><H1 style="padding-top: 3pt;padding-left: 24pt;text-indent: 0pt;text-align: center;">2014 International Workshop and Summer School on &quot;Biological Big Bytes&quot;</H1><H1 style="padding-left: 60pt;text-indent: 0pt;text-align: center;">a Satellite Meeting of the International Workshop/Summer school on Crops, Chips &amp; Computers</H1><P class="s3" style="padding-top: 3pt;padding-left: 63pt;text-indent: 0pt;text-align: center;">Harbin Institute of Technology, Harbin, China</P><P class="s5" style="padding-top: 1pt;padding-left: 55pt;text-indent: 0pt;text-align: left;"><SPAN class="s3">August 17th to 23th, 2014</SPAN></P><H3 style="padding-top: 10pt;padding-left: 134pt;text-indent: 0pt;text-align: left;">WELCOME TO C3-B3’2014</H3><P class="s8" style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Dear Colleagues and Friends:</P><P class="s8" style="padding-left: 5pt;text-indent: 36pt;line-height: 201%;text-align: justify;">Welcome to C3-B3’2014 at the Harbin Institute of Technology, Harbin, China, August 17th-23th,2014!</P><P class="s8" style="padding-left: 5pt;text-indent: 36pt;line-height: 199%;text-align: justify;">This workshop and summer school on Biological Big Bytes (B3) is a satellite meeting of our traditional annual international workshop and summer school on &quot;Crops, Chips and Computers&quot; (C3), aiming to teach best practice in the application of state-of-the-art information technology and computational techniques to big data research in life sciences. The workshop and summer school will cover interdisciplinary science, in a rapidly evolving field, bringing together ideas from the fields of mathematics, bioinformatics, omics and computer sciences. Because of these attractions, we expect the school to be of interest to a range of students from all academic levels, from post-graduate students to postdoctoral workers to junior academics.</P><P class="s9" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">Organizers</P><P class="s10" style="padding-top: 3pt;padding-left: 41pt;text-indent: 0pt;text-align: left;">Dr. Andrew Harrison (University of Essex)</P><P class="s10" style="padding-top: 7pt;padding-left: 41pt;text-indent: 0pt;line-height: 146%;text-align: left;">Dr. Christian Klukas (Leibniz Institute  of Plant Genetics and Crop Plant Research) Dr. Dechang Xu (Harbin Institute  of Technology)</P><P class="s10" style="padding-top: 1pt;padding-left: 41pt;text-indent: 0pt;line-height: 147%;text-align: left;">Dr. Huaqin He (Fujian Agriculture  and Forestry University University) Dr. Hugh Shanahan (Royal Holloway,  Unversity of London)</P><P class="s10" style="padding-left: 41pt;text-indent: 0pt;line-height: 152%;text-align: left;">Dr. Ling-ling Chen (Huazhong Agricultural  University) Dr. Ming Chen (Zhejiang University)</P><P class="s10" style="padding-left: 41pt;text-indent: 0pt;line-height: 13pt;text-align: left;">Dr. Qingfeng Chen (Guangxi  University)</P><P class="s10" style="padding-top: 7pt;padding-left: 41pt;text-indent: 0pt;text-align: left;">Dr. Sean May (The University of Nottingham)</P><P class="s10" style="padding-top: 6pt;padding-left: 41pt;text-indent: 0pt;line-height: 152%;text-align: left;">Dr. Yongsheng Chen (Inner Mongolian University for the Nationalities) Dr. Ziding Zhang (China Agricultural University)</P><P class="s9" style="padding-top: 1pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Funding and Partners</P><P class="s10" style="padding-top: 3pt;padding-left: 5pt;text-indent: 36pt;line-height: 151%;text-align: justify;">The 2014 International Workshop and Summer School on Biological Big Bytes is sponsored by Harbin Institute of Technology. It is jointly organized by the Department of Bioinformatics, College of Life Sciences, Zhejiang University, and the Departments of Mathematical Sciences &amp; Biological Sciences, University of Essex. It is hosted by Harbin Institute  of Technology,  Harbin, China.</P><P class="s9" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">Local organizer</P><P class="s10" style="padding-top: 3pt;padding-left: 41pt;text-indent: 0pt;text-align: left;">Dr. Nanqi Ren, (Harbin Institute  of Technology),  Chair</P><P class="s10" style="padding-top: 6pt;padding-left: 41pt;text-indent: 0pt;line-height: 152%;text-align: left;">Dr. Dechang Yu (Harbin Institute  of Technology),  Co-Chair Dr. Ming Du (Harbin Institute  of Technology)</P><P class="s10" style="padding-left: 41pt;text-indent: 0pt;text-align: left;">Dr. Fei Liu (Harbin Institute  of Technology)</P><P class="s9" style="padding-top: 7pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Contact</P><P class="s10" style="padding-top: 3pt;padding-left: 41pt;text-indent: 0pt;text-align: left;">Email:   All questions  related to the C3'2014 can be sent to</P><P class="s10" style="padding-top: 6pt;padding-left: 41pt;text-indent: 0pt;line-height: 152%;text-align: left;">Dr. Dechang Xu (dcxu@hit.edu.cn)  (Harbin Institute  of Technology)  or Dr. Ming Chen (mchen@zju.edu.cn)  (Zhejiang University)  or</P><P class="s10" style="padding-left: 41pt;text-indent: 0pt;text-align: left;">Dr. Andrew Harrison (harry@essex.ac.uk)  (University of Essex)</P><P class="s10" style="padding-top: 3pt;padding-left: 41pt;text-indent: 0pt;text-align: left;">Tel:  +(0)451-86282904</P><P class="s10" style="padding-top: 7pt;padding-left: 101pt;text-indent: -60pt;line-height: 152%;text-align: left;">Address: School of Food Science &amp; Engineering, Harbin Institute of Technology, Harbin 150090, P. R. China</P><H3 style="padding-top: 2pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Travel</H3><P class="s11" style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">By Air-plane</P><P class="s10" style="padding-top: 7pt;padding-left: 44pt;text-indent: -3pt;line-height: 146%;text-align: left;">From  Airport to Harbin Institute  of Technology: You may come to Harbin via Beijing / Shanghai.</P><P class="s10" style="padding-left: 44pt;text-indent: 0pt;line-height: 146%;text-align: left;">Taxi: Taxi from airport to the 2nd campus costs about 120 RMB Airport Bus No.2: costs 20 RMB</P><P class="s11" style="padding-top: 1pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">By Train:</P><P class="s10" style="padding-top: 7pt;padding-left: 41pt;text-indent: 0pt;line-height: 146%;text-align: left;">There are three railway stations in Harbin. From Railway Station:</P><P class="s10" style="padding-left: 41pt;text-indent: 0pt;line-height: 146%;text-align: left;">Taxi: Taxi from Harbin Station to the 2nd campus costs about 20 RMB. Bus: by Bus No. 89 costs 1 RMB</P><H3 style="padding-left: 5pt;text-indent: 0pt;line-height: 20pt;text-align: left;">Hotel</H3><P class="s10" style="padding-top: 3pt;padding-left: 41pt;text-indent: 0pt;text-align: left;">7 Days Inn (Harbin Convention Center)</P><P class="s10" style="padding-top: 6pt;padding-left: 41pt;text-indent: 0pt;line-height: 14pt;text-align: left;">Shongsan Road 117, Nangang District,  Harbin</P><H3 style="padding-left: 5pt;text-indent: 0pt;line-height: 20pt;text-align: left;">Programme</H3><H4 style="padding-left: 162pt;text-indent: 0pt;line-height: 14pt;text-align: center;">Sunday 17th August 2014</H4><P class="s12" style="padding-left: 113pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Registration <SPAN class="p">Harbin Institute of Technology</SPAN></P><P style="padding-left: 5pt;text-indent: 0pt;line-height: 12pt;text-align: left;">17:30~                              <SPAN class="s13">Dinner</SPAN></P><H4 style="padding-left: 165pt;text-indent: 0pt;line-height: 14pt;text-align: center;">Monday 18th August 2014</H4><P class="s12" style="padding-left: 8pt;text-indent: 105pt;text-align: left;">Registration   <SPAN class="p">Harbin Institute of Technology Classroom (B201 Main Bldg) 08:30~09:00                    </SPAN>Introduction   <SPAN class="p">Ming Chen &amp; Andrew Harrison &amp; Dechang Xu</SPAN></P><P style="padding-left: 187pt;text-indent: 0pt;text-align: left;">An overview of what needs to be done at the school and the expected work load on each day.</P><P class="s12" style="padding-left: 115pt;text-indent: -2pt;line-height: 11pt;text-align: left;">Team grouping</P><P class="s12" style="padding-left: 115pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Section 1 (Lectures)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">09:00~10:15                     <SPAN class="s13">Lecture 1 </SPAN>history of science, technology and data</P><P class="s13" style="padding-left: 115pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Lecturer  <SPAN class="p">Andrew Harrison</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">10:30~11:45                    <SPAN class="s13">Lecture 2  </SPAN>practicalities of scientific data and the wider zeitgeist of data.</P><P class="s13" style="padding-left: 115pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Lecturer  <SPAN class="p">Andrew Harrison</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">12:00~13:30                    <SPAN class="s13">Lunch</SPAN></P><P class="s12" style="padding-left: 115pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Section 2 (Practical)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">14:30~17:30                    <SPAN class="s13">Practice 1 </SPAN>practicalities of scientific data and the wider zeitgeist of data.</P><P class="s13" style="padding-left: 115pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Lecturer  <SPAN class="p">Andrew Harrison</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">18:00~                             <SPAN class="s13">Dinner</SPAN></P><H4 style="padding-left: 162pt;text-indent: 0pt;line-height: 14pt;text-align: center;">Tuesday 19th August 2014</H4><P class="s12" style="padding-left: 116pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Section 3 (Lectures)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">09:00~10:00                    <SPAN class="s13">Lecture 3 </SPAN>Files vs Databases / SQL vs NoSQL.</P><P class="s13" style="padding-left: 113pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Lecturer  <SPAN class="p">Adam</SPAN></P><P style="padding-left: 5pt;text-indent: 0pt;text-align: left;">10:10~11:00                     <SPAN class="s13">Lecture 4 </SPAN>An introduction to SQL &amp; Database Design. An introduction to sed, awk</P><P style="padding-left: 53pt;text-indent: 0pt;line-height: 12pt;text-align: center;">and grep</P><P class="s13" style="padding-left: 113pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Lecturer  <SPAN class="p">Andrew Harrison</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">12:00~13:30                     <SPAN class="s13">Lunch</SPAN></P><P class="s12" style="padding-left: 113pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Section 4 (Practical)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">14:30~17:30                    <SPAN class="s13">Practice 2 </SPAN>File naming, versioning and backing up. Data and metadata preparation</P><P style="padding-left: 163pt;text-indent: 0pt;line-height: 12pt;text-align: left;">and publishing, persistent identifiers and citation, linking to articles etc.</P><P class="s13" style="padding-left: 113pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Practical  <SPAN class="p">Louise</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">18:00~                             <SPAN class="s13">Dinner</SPAN></P><H4 style="padding-left: 165pt;text-indent: 0pt;line-height: 14pt;text-align: center;">Wednesday 20th August 2014</H4><P class="s13" style="padding-left: 113pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Workshop Day</P><P class="s13" style="padding-left: 113pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Meeting Room  <SPAN class="p">214 Student Activity Center</SPAN></P><P class="s12" style="padding-left: 113pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Section 9 (Chair: Huaqin He)</P><P style="padding-left: 113pt;text-indent: -105pt;text-align: left;">08:30~09:10                    <SPAN class="s13">Opening Ceremony Speaker  </SPAN>President <SPAN class="s13">Group photo</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">09:10~9:40                      <SPAN class="s13">Speaker 1  </SPAN>Hongzhi Wang</P><P class="s13" style="padding-left: 113pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Title <SPAN class="p">Algorithm Design and Analysis for Big Data</SPAN></P><P style="padding-top: 3pt;padding-left: 8pt;text-indent: 0pt;text-align: left;">09:40~10:10                       <SPAN class="s13">Speaker 2  </SPAN>Adam Carter</P><P class="s13" style="padding-left: 124pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Title <SPAN class="p">The EUDAT Collaborative Data Infrastructure</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">10:10~10:30                       <SPAN class="s13">Coffee/Tea break</SPAN></P><P class="s12" style="padding-left: 124pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Section 10 (Chair: Ming Chen)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">10:30~10:55                        <SPAN class="s13">Speaker 3  </SPAN>Huaqin He</P><P class="s13" style="padding-left: 126pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Title <SPAN class="p">Genome-wide identification and function of heat-shock proteins in rice</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">10:55~11:20                        <SPAN class="s13">Speaker 4  </SPAN>Xiaobao Dong / Ziding Zhang</P><P class="s13" style="padding-left: 152pt;text-indent: -28pt;text-align: left;">Title <SPAN class="p">Revealing Shared and Distinct Network Organizations of Arabidopsis Immune Responses by Integrative Analysis</SPAN></P><P style="padding-left: 124pt;text-indent: -115pt;line-height: 11pt;text-align: left;">11:20~11:45                       <SPAN class="s13">Speaker 5  </SPAN>Andrew Harrison</P><P class="s13" style="padding-left: 124pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Title <SPAN class="p">Mind-altering microbes</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">11:45~12:10                       <SPAN class="s13">Speaker 6  </SPAN>Jingjing Wang / Ming Chen</P><P class="s13" style="padding-left: 124pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Title <SPAN class="p">The roles of cross-talking epigenetic patterns in Arabidopsis thaliana</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">12:30~13:30                        <SPAN class="s13">Lunch</SPAN></P><P class="s12" style="padding-left: 121pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Section 11 (Chair: Ziding Zhang)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">14:30~14:55                       <SPAN class="s13">Speaker 7  </SPAN>Professionalising digital data curation skills in Universities</P><P class="s13" style="padding-left: 124pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Title <SPAN class="p">Louise Corti</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">14:55~15:20                       <SPAN class="s13">Speaker 8  </SPAN>Fei Liu</P><P class="s13" style="padding-left: 124pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Title <SPAN class="p">Colored Petri Nets for Multiscale Systems Biology</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">15:20~15:45                       <SPAN class="s13">Speaker 9  </SPAN>Lifeng Xu</P><P class="s13" style="padding-left: 150pt;text-indent: -26pt;text-align: left;">Title <SPAN class="p">Simulating genotype-phenotype interactions using Functional-Structural plant models</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 11pt;text-align: left;">15:45~16:10                       <SPAN class="s13">Speaker 10  </SPAN>Bjorn Sommer</P><P class="s13" style="padding-left: 152pt;text-indent: -28pt;text-align: left;">Title <SPAN class="p">CELLmicrocosmos Workshop: Cell Modeling at Themesoscopic and Functional Level</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 11pt;text-align: left;">16:10~16:30                        <SPAN class="s13">Coffee/Tea break</SPAN></P><P class="s12" style="padding-left: 124pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Section 12: iGEM MeetUp (Chair: Dechang Xu)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">16:30~16:55                        <SPAN class="s13">Speaker 11  </SPAN>Kang Yang</P><P class="s13" style="padding-left: 126pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Title <SPAN class="p">Aspergillus niger is coming!</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">16:55~17:20                        <SPAN class="s13">Speaker 12  </SPAN>Zhiqi Liu</P><P class="s13" style="padding-left: 126pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Title <SPAN class="p">E.conan detective</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">17:20~17:45                        <SPAN class="s13">Speaker 13  </SPAN>Yifan Wu</P><P class="s13" style="padding-left: 126pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Title <SPAN class="p">Dioxin Detective Y</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">18:00~20:00                        <SPAN class="s13">Dinner &amp; Discussion</SPAN></P><H4 style="padding-left: 2pt;text-indent: 0pt;text-align: center;">Thursday 21th August 2014</H4><P class="s12" style="padding-left: 126pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Section 5 (Lectures)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">09:00~10:30                         <SPAN class="s13">Lecture 5 </SPAN>Big Science with Big Data needs Data Infrastrucutres - Data has value -</P><P style="padding-left: 171pt;text-indent: 0pt;text-align: left;">Data reuse - Data sharing - Distributed data Persistent Identifiers and Metadata (bringing out synergies with Louise's material).</P><P class="s13" style="padding-left: 129pt;text-indent: 0pt;line-height: 11pt;text-align: left;">Lecturer <SPAN class="p">Adam</SPAN></P><P style="padding-left: 129pt;text-indent: -120pt;text-align: left;">10:50~11:30                          <SPAN class="s13">Lecture 6 Lecturer</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">12:00~13:30                          <SPAN class="s13">Lunch</SPAN></P><P class="s12" style="padding-left: 130pt;text-indent: 0pt;text-align: left;">Section 6 (Practical)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">14:30~16:00                         <SPAN class="s13">Practice 3 </SPAN>Data publishing practical.</P><P class="s13" style="padding-left: 129pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Practical  <SPAN class="p">Louise</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">16:00~17:00                         <SPAN class="s13">Practice 4</SPAN></P><P style="padding-left: 5pt;text-indent: 0pt;line-height: 12pt;text-align: left;">.                                              <SPAN class="s13">Lecturer</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">18:00~                                   <SPAN class="s13">Dinner</SPAN></P><H4 style="padding-left: 173pt;text-indent: 0pt;text-align: center;">Friday 22th August 2014</H4><P class="s12" style="padding-left: 126pt;text-indent: 0pt;text-align: left;">Section 7 (Lectures)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">09:00~10:30                         <SPAN class="s13">Lecture 7  </SPAN>Models for Processing Big Data - Moving Compute to the Data -</P><P style="padding-left: 173pt;text-indent: 0pt;text-align: left;">Workflows - Data Intensive Compute Architectures Models for Processing  Big Data - MapReduce (includes practical session)</P><P class="s13" style="padding-left: 129pt;text-indent: 0pt;line-height: 11pt;text-align: left;">Lecturer <SPAN class="p">Adam</SPAN></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">10:50~11:30                          <SPAN class="s13">Lecture 8 </SPAN>Practice and Discussion.</P><P class="s13" style="padding-left: 129pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Lecturer</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">12:00~13:30                          <SPAN class="s13">Lunch</SPAN></P><P class="s12" style="padding-left: 126pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Section 8 (Practical)</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">14:30~17:30                         <SPAN class="s13">Practice 5  </SPAN>Discussion about careers in data Student presentations.</P><P class="s13" style="padding-left: 129pt;text-indent: 0pt;line-height: 12pt;text-align: left;">Practical</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">18:00~                                   <SPAN class="s13">Dinner</SPAN></P><P class="s13" style="padding-top: 6pt;text-indent: 0pt;text-align: right;">Tour</P><H4 style="padding-top: 3pt;padding-left: 18pt;text-indent: 0pt;text-align: left;">Saturday 23th August 2014</H4><H4 style="padding-top: 3pt;padding-left: 173pt;text-indent: 0pt;line-height: 14pt;text-align: center;">Sunday 24 th August 2014</H4><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: left;">9:30~11:40                            <SPAN class="s13">Close session &amp; Awards ceremony</SPAN></P><P style="padding-left: 129pt;text-indent: 0pt;text-align: left;">Science in the UK, Germany and China, and moving among countries; career structures in these countries. We expect to recruit European students who may be considering spending part of their careers in China and we hope some of the Chinese students similarly look to spend part of their careers in Europea.</P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 11pt;text-align: left;">12:00~                                    <SPAN class="s13">Lunch &amp; Farewell</SPAN></P><P class="s9" style="padding-top: 2pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Workshop abstracts</P><UL id="l1"><LI style="padding-left: 58pt;text-indent: -52pt;line-height: 17pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s15">：</SPAN><SPAN class="p">Algorithm Design and Analysis for Big Data</SPAN></P><P class="s14" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Hongzhi Wang <SPAN class="p">Assoc. Professor, </SPAN><I>School of Computer Science &amp; Technology, HIT</I></P><P class="s14" style="padding-left: 5pt;text-indent: 0pt;line-height: 149%;text-align: left;">Abstract: <SPAN class="p">Big data bring challenges for algorithm design and analysis. Algorithms for big data analysis are required to handle the volume and velocity features. Meeting these difficulties, some new algorithms design and analysis techniques are proposed. This talk shows the widely applications of big data algorithms and gives an overview of techniques for big data algorithm design and analysis including sub-linear algorithms, I/O efficient algorithms, parallel algorithms as well as crowd sourcing algorithms.</SPAN></P></LI><LI style="padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s15">：</SPAN><SPAN class="p">The EUDAT Collaborative Data Infrastructure</SPAN></P><P style="padding-top: 7pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;"><B>Adam Carter</B>, <I>EPCC</I>, <I>The University of Edinburgh</I></P><P class="s14" style="padding-top: 8pt;padding-left: 5pt;text-indent: 0pt;line-height: 148%;text-align: justify;">Abstract<SPAN class="s15">：</SPAN><SPAN class="p">EUDAT’s goal is to build a Collaborative Data Infrastructure (CDI) as a pan-European solution to the challenge of data proliferation in Europe’s scientific and research communities. The CDI will allow researchers to share data within and between communities and enable them to carry out their research effectively. Our mission is to provide a solution that will be affordable, trustworthy, robust, persistent, open and easy to use.</SPAN></P><P style="padding-left: 5pt;text-indent: 36pt;line-height: 149%;text-align: justify;">In this talk I will discuss the progress that EUDAT has made towards these goals, and I’ll describe the EUDAT services that are already up and running.</P></LI><LI style="padding-top: 6pt;padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="p">: Genome-wide identification and function of heat-shock proteins in rice</SPAN></P><P style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Yongfei Wang1, Xinhai Chen1, Shoukai Lin1,2*, Qi Song1, Huan Tao1, Jian Huang1, Yuqin Ke1, Shufu Que1,</P><P class="s14" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Huaqin He<SPAN class="p">1*</SPAN></P><P class="s16" style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;line-height: 149%;text-align: left;">1 College of Life Sciences, Fujian Agriculture and Forestry University, Fuzhou, China. 2 Putian University, Putian, Fujian, China</P><P class="s14" style="padding-left: 5pt;text-indent: 0pt;line-height: 148%;text-align: justify;">Abstract<SPAN class="s15">：</SPAN><SPAN class="p">Heat shock proteins (Hsps) have been known as a class of proteins that perform a fundamental role in protecting plants against abiotic stresses, such as drought, salinity, extremely low and high temperatures. However, rare Hsps have been reported and little consideration has been given to the role of Hsps in rice in response to abiotic stress. Therefore, we conducted this research to identify heat shock proteins and reveal the role of sHsps in rice in response to abiotic stresses.</SPAN></P><P style="padding-left: 5pt;text-indent: 36pt;line-height: 149%;text-align: justify;">First, we combined an orthology-based approach with expression association data to screen rice Hsps, the expression patterns of which strongly correlate with those of heat responsive probe-sets. Twenty-seven Hsp candidates were verified, including 12 small Hsps, six Hsp70s, three Hsp60s, three Hsp90s, and three clpB/Hsp100s. Then, using a combination of interolog and expression profile-based methods, we identified 430</P><P style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 150%;text-align: justify;">interactorsofHsp70 family members in rice, and validated the interactions by co-localization and function-based methods. Subsequently, 13 interacting domains and 28 target motifs were over-represented in Hsp70sinteractors. Twenty-four GO terms of biological processes and five GO terms of molecular functions were enriched in the positive Hsp70 interactors, which expression levels were positively associated with Hsp70s. Analyzing the interaction network of Hsp70s, we found that Hsp70s were involved in macromolecular translocation, carbohydrate metabolism, innate immunity, photosystem II repair and regulation of kinase ac tivities.</P><P style="padding-left: 5pt;text-indent: 36pt;line-height: 150%;text-align: justify;">Hsp26.7, Hsp23.2, Hsp17.9A, Hsp17.4 and Hsp16.9A were up-regulated in r ice (Orya sativa L. &amp; Nipponbare) during seedling and anthesis stages in response to heat stress. Subsequently, the expression level of five sHsps in the heat-tolerant rice cultivar, Co39, were all significantly higher than that in the heat-susceptible rice cultivar, Azucena, indicating that the expression level of these five sHsps was positive related to the ability of rice plants to avoid heat stress. Thus, the expression level of these five sHsps can be regarded as bio-markers for screening rice cultivars with different abilities to avoid heat stress. Hsp18.1, Hsp17.9A, Hsp17.7 and Hsp16.9A, in the three rice cultivars under heat stress were found to be involved in one protein complex by Native-PAGE, and the interactions of Hsp18.1 and Hsp 17.7, Hsp18.1 and Hsp 16.9A, Hsp17.7 and Hsp16.9A were further validated by yeast 2-hybridization. Pull down assay also confirmed the interaction between Hsp17.7 and Hsp16.9A in rice under heat stress. In conclusion, the up-regulation of the 5 sHsps is a key step for rice to tolerate heat stress, after that some sHsps are assembled into a large hetero-oligomeric complex.</P><P style="padding-left: 5pt;text-indent: 36pt;line-height: 149%;text-align: left;"><A href="http://bioinformatics.fafu.edu.cn/yfwang/PPIP/ppip.php" class="a" target="_blank">Moreover, we constructed a Protein-Protein Interaction Predictor (PPIP) and a Rice Gene Expression Profile database for experimental biologists. These tools could be available at </A>http://bioinformatics.fafu.edu.cn/yfwang/PPIP/ppip.php and</P><P style="padding-left: 5pt;text-indent: 0pt;text-align: left;">http://bioinformatics.fafu.edu.cn/ yfwang/array/index.php respectively.</P><P class="s14" style="padding-top: 7pt;padding-left: 5pt;text-indent: 0pt;text-align: left;">Keywords<SPAN class="p">: Rice (Oryza sativa L.), Heat shock proteins, Expression, Interactors</SPAN></P><P class="s14" style="padding-left: 5pt;text-indent: 0pt;line-height: 12pt;text-align: left;">ACKNOWLEDGMENT</P><P style="padding-left: 5pt;text-indent: 0pt;text-align: left;">This work was supported by the Natural Science Foundation of China and Fujian (grant nos. 31270454, 61163047 and 2013J01077), a grant from the Education Department of Fujian (grant no. JA10103) and the Key Program of Ecology in Fujian (grant nos. 0608507 and</P><P style="padding-left: 5pt;text-indent: 0pt;line-height: 12pt;text-align: left;">6112C0600).</P></LI><LI style="padding-top: 8pt;padding-left: 58pt;text-indent: -52pt;line-height: 143%;text-align: left;"><P class="s14" style="display: inline;">Title<SPAN class="s15">： </SPAN><SPAN class="p">Revealing Shared and Distinct Network Organizations of Arabidopsis Immune Responses by Integrative Analysis</SPAN></P><P class="s14" style="padding-left: 5pt;text-indent: 0pt;text-align: left;">Xiaobao Dong* <SPAN class="p">and Ziding Zhang</SPAN></P><P class="s16" style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;line-height: 149%;text-align: left;">State Key Laboratory of Agrobiotechnology, College of Biological Sciences, China Agricultural University, Beijing 100193, China</P></LI><LI style="padding-left: 13pt;text-indent: -8pt;text-align: left;"><P style="display: inline;">Speaker (Email: <SPAN class="s18">x.b.dong@c au.edu.cn)</SPAN></P><P class="s14" style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 149%;text-align: justify;">Abstract: <SPAN class="p">Pattern-triggered immunity (PTI) and effector-triggered immunity (ETI) are two main forms of plant immune response to counter pathogen invasion. Up to now, genome-wide regulatory network organization principles leading to quantitative differences between PTI and ETI remain elusive. We combined an advanced</SPAN></P><P style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 150%;text-align: justify;">machine learning method and modular network analysis to systematically characterize the organization principles of Arabidopsis PTI and ETI. We report our major findings from three network resolutions. At a single network node/edge level, we ranked important genes and gene interactions for immune response and successfully identified many known immune regulators for PTI and ETI, respectively. Topological analysis showed that important gene interactions tend to link network modules. At a subnetwork level, we identified a subnetwork shared by PTI and ETI, which covers 1159 genes and 1289 interactions. In addition to being enriched with interactions linking network modules, it is also a hotspot attacked by pathogen effectors. The subnetwork likely represents a core component to coordinate multiple biological processes in the transition from development to defense. Finally, we constructed modular network models for PTI and ETI to explain the quantitative differences from the global network architecture. Our results show defense modules appeared to be interdependently connected in PTI, but independently connected in ETI, providing an explanation for the robustness of ETI to genetic mutations and effector attacks. Taken together, the multiscale comparisons between PTI and ETI provide a systems biology point of view to understand plant immunity, and highlight the coordination among network modules to establish a robust immune response.</P><P class="s14" style="padding-left: 5pt;text-indent: 0pt;line-height: 149%;text-align: left;">Keywords<SPAN class="p">: pattern-triggered immunity, effector-triggered immunity, regulatory network,  subnetwork, network module</SPAN></P></LI></UL><LI style="padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s15">：</SPAN><SPAN class="p">Mind-altering microbes</SPAN></P><P class="s14" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Andrew Harrison <I>University of Essex</I></P><P class="s14" style="padding-left: 5pt;text-indent: 0pt;line-height: 148%;text-align: justify;">Abstract<SPAN class="s15">：</SPAN><SPAN class="p">We are moving through a paradigm shift in how we view the relationship between Homo Sapiens and microbes - we may be best considered as a mammalian-microbe symbiotic ecosystem. This changing view will impact upon many fields, including psychology and mental health. It is increasingly clear that the Western lifestyle is leading to a dramatic change in our microbiota. This may have causal implications for the dramatic rises we are witnessing in obesity, autoimmune diseases and mental disorders such as autism. It will further impact upon how we care for an ageing population, overcome malnutrition, and deal with infectious diseases in the post-antibiotic era.</SPAN></P></LI><LI style="padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s15">：</SPAN><SPAN class="p">The roles of cross-talking epigenetic patterns in </SPAN><I>Arabidopsis thaliana</I></P><P class="s19" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;"><SPAN class="s14">Jingjing Wang</SPAN>1,<SPAN class="s20"> 2,*</SPAN><SPAN class="p">, Xianwen Meng</SPAN>1<SPAN class="p">, Chunhui Yuan</SPAN>1<SPAN class="p">, Andrew P. Harrison</SPAN>3<SPAN class="p">and Ming Chen</SPAN>1,<SPAN class="s20"> 2</SPAN></P><P class="s21" style="padding-left: 5pt;text-indent: 0pt;line-height: 144%;text-align: left;">1<SPAN class="s16">Department of Bioinformatics, the State Key Laboratory of Plant Physiology and Biochemistry, Institute of Plant Science; College of Life Sciences, Zhejiang University, Hangzhou 310058, P.R. China, </SPAN>2<SPAN class="s16">James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou 310058, P. R. China, </SPAN>3<SPAN class="s16">Department of Mathematical Sciences and School of Biological Sciences, University of Essex, Colchester, Essex CO4 3SQ, UK</SPAN></P></LI><LI style="padding-left: 13pt;text-indent: -8pt;text-align: justify;"><P style="display: inline;">Speaker (Email: <A href="mailto:jingjingw@zju.edu.cn" class="s22" target="_blank">jingjingw @zju.edu.c n</A><A href="mailto:jingjingw@zju.edu.cn" target="_blank">)</A></P></LI><P class="s14" style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 149%;text-align: justify;">Abstract: <SPAN class="p">Epigenetic marks, including histone modifications, DNA methylation and histone variants, play a key role in determining transcriptional outcomes. The availability of the high-throughput sequencing data allows us to characterize the genome-wide distribution patterns of epigenetic marks, which have improved our understanding of the relationship between chromatin modification/structure and genes expression. More and more studies have found that epigenetic mechanisms have been implicated in many biological processes, such as Transposon Element silencing and long-range interactions within the genome. Moreover, those studies have found there are different layers of cross-talk and interplay among epigenetic marks to be involved in chromatin- dependent processes. Here, we take advantage of evidence from recent genetic, biochemical and high- throughput sequencing studies to explore the complex cross-talk between different epigenetic patterns, and address how these epigenetic marks interplay with each other and then affect gene transcription. Finally, we build a cross-talk network and analyze the characteristic of the network. Comprehensive cross-talk network of epigenetic marks will help in fully understanding the functional roles and biological impact of epigenetic regulation.</SPAN></P><P class="s14" style="padding-top: 7pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Keywords: <SPAN class="p">histone modification; DNA methylation; histone variant; cross-talk</SPAN></P><LI style="padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s15">：</SPAN><SPAN class="p">'Professionalising digital data curation skills in Universities'</SPAN></P><P class="s14" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Louise Corti <I>UK Data Archive</I></P><P class="s14" style="padding-top: 8pt;padding-left: 5pt;text-indent: 0pt;line-height: 149%;text-align: justify;">Abstract<SPAN class="s15">：</SPAN><SPAN class="p">Researchers’ responsibilities towards their research data are set to change across all domains of scientific endeavour. Research funders are increasingly mandating open access to research data; governments internationally are demanding transparency in research; the economic climate is requiring much greater re-use of data; and fear of data loss calls for more robust information security practices. All these factors mean that researchers need to improve, enhance and professionalize their research data management skills to meet the challenge of producing the highest quality shareable and reusable research outputs in a responsible and efficient way. The promotion of these skills offers a strategic contribution to the world's research capacity building programmes. How can we go about professionalising our data management skills? I take a guided tour of various efforts in UK universities to help make this agenda happen.</SPAN></P></LI><LI style="padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s23">：</SPAN><SPAN class="p">Colored Petri Nets for Multiscale Systems Biology</SPAN></P><P class="s14" style="padding-top: 7pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Fei Liu <I>Control and Simulation Center, HIT</I></P><P style="padding-top: 5pt;padding-left: 8pt;text-indent: 0pt;text-align: justify;">Email: <A href="mailto:liufei@hit.edu.cn" class="s22" target="_blank">liufei@hit.edu.c n</A></P><P class="s14" style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 149%;text-align: justify;">Abstract<SPAN class="p">: Systems biology aims to understand the behavior of a biological system at the system level by means of investigating the behavior and interactions of all the components in the system. Due to the ability to produce data of one and the same phenomenon at different scales, the modeling of biological systems is moving from single scales to multiple scales, e.g., from the molecular scale to the cell, tissue, and even the whole organism</SPAN></P><P style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 151%;text-align: justify;">level. In this report, we will present a colored Petri net pproach to modeling and analyzing multiscale systems biology.Specifically, in this report you will see what are colored Petri nets, what challenges of multiscale systems biology can be tackled by colored Petri ets, a colored Petri net framework for multiscale systems biology, and a couple of case studies.</P></LI><LI style="padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s15">：</SPAN><SPAN class="p">Simulating genotype-phenotype interactions using Functional-Structural plant models</SPAN></P><P class="s14" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;line-height: 155%;text-align: justify;">Lifeng Xu <I>College of Computer Science &amp; Technology, Zhejiang University of Technology, 310023 Hangzhou, China</I></P><P style="padding-left: 8pt;text-indent: 0pt;line-height: 12pt;text-align: justify;">Email: <A href="mailto:lfxu@zjut.edu.cn" class="s22" target="_blank">lfxu@zjut.edu.c n</A></P><P style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 150%;text-align: justify;">Crop breeders and breeding scientists presently face a multitude of problems and challenges: Cereals, namely rice, maize and wheat, constitute the basic staple foods and the demand for them is rising with an ever- increasing human population. Therefore, understanding plant growth and morphology is particularly of great importance to agronomy. Modelling efforts within the Functional-Structural Plant Model (FSPM) domain are mostly concerned with the acquisition, transport and use of matter and energy from sources to sinks through pathways dictated by plant structure, such as light, carbon, water and soil minerals, and how this affects growth and morphology of the resulting plants. Genotype-phenotype models of crops in the form of spatially explicit models of morphogenetic development, based on the interaction of compartmentalized physiological processes, and coupled with quantitative genetic information, can be used to better understand crop systems and ultimately to aid breeding of new crop plant varieties. In order to illustrate our approach of simulating genotype-phenotype interactions, an FSPM of rice will be described here. The rice model can simulate growth and morphology of rice plant from germination to seed maturity, in combination with selected ecophysiological processes including photosynthesis and sink functions based on a common assimilate pool. Furthermore, it was linked to a quantitative genetic model, thereby simulating the genetic reproduction processes to form new rice population. The rice FSPM reproduces plant architecture, morphology and the QTLs for certain traits (in this case: plant height) of rice from germination to maturity. The potential use of the model with further extension would be as a tool in domains of FSPM education and scientific research.</P></LI><LI style="padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s15">：</SPAN><SPAN class="p">SUBCELLULAR  VISUALIZATION  AND LOCALIZATION OF PROTEIN-RELATED DATA</SPAN></P><P class="s14" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Björn Sommer <I>Bio-/Medical Informatics Department, University Bielefeld, Bielefeld, Germany</I></P><P style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">E-mail: <A href="mailto:bjoem@CELLmicrocosmos.org" class="s22" target="_blank">bjoem@CELLmic roc osmos.org</A></P><P style="padding-left: 5pt;text-indent: 0pt;line-height: 150%;text-align: justify;">Molecular databases such as Reactome and KEGG support the visualization and analysis of biomedical networks [COWH11, KGSF12]. These pathways usually consist of enzymes, substrates and products, and their corresponding reaction processes.Websites of both databases provide 2D network visualizations which are often enriched with the information where specific genes or proteins are located. These images are curated and the corresponding subcellular localizations are usually reliable.</P><P style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 150%;text-align: justify;">But problems arise if biomedical networks are (semi-)automatically generated based on heterogeneous database information. For example, if a set of proteins – e.g. correlated with a specific disease – is found by combining information from different databases, consistent information regarding the subcellular localization is usually missing. To obtain the localization information, again various databases can be consulted. But the problem is now to obtain reasonable localizations in the context of the newly created biological network. For this purpose, the CELLmicrocosmos 4 PathwayIntegration was developed:</P><P style="padding-left: 5pt;text-indent: 0pt;text-align: justify;"><A href="http://cm4.cellmicrocosmos.org/" class="s24" target="_blank">http://Cm4.CELLmic roc osmos.org</A></P><P style="padding-top: 5pt;padding-left: 5pt;text-indent: 0pt;line-height: 150%;text-align: left;">Using subcellular localization charts it is possible to compare, analyze and predict the protein localization based on a number of established databases [SKDA13]. Direct links to the original sources enable the validation of the localization predictions. Moreover, the results can be mapped into a virtual cell 3D environment which is directly connected to the 2D visualization of the biomedical pathway. Based on the jSBML library, protein- related networks can be imported from various sources[DRDD11].</P><P class="s14" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">iGEM MeetUp</P></LI><LI style="padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s15">：</SPAN><SPAN class="p">Aspergillus niger is coming !</SPAN></P><P class="s14" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Kang Yang <I>Northeast Agricultural University</I></P><P class="s14" style="padding-left: 5pt;text-indent: 0pt;line-height: 150%;text-align: justify;">Abstract: <SPAN class="p">Aspergillus niger, a species of filamentous fungi, is universally acknowledged of having long been investigated for industry production of metabolites and enzymes. Meanwhile, with the growth of the biotechnology industry, it has been gradually employed in areas as diverse as medicine, agriculture, and basic science that influence our everyday lives. Taking advance of the expression system in Aspergillus niger, our team successfully accomplished “Visual operation system of continuous gene replacement” and “From Red to Cyan ”, respectively means targeted gene’s location at specific sites and transformation accelerator from lycopene to astaxanthin. The meaning of our work, on one hand, relied on that we established a new visual operation system assisting us to make sure where is the DNA. On the other hand, we designed one/two fusion protein contributing to the transformation accelerator of metabolism.</SPAN></P></LI><LI style="padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s15">：</SPAN><SPAN class="p">E.conan detective</SPAN></P><P class="s14" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Zhiqi Liu <I>Northeast Forestry University</I></P><P style="padding-left: 5pt;text-indent: 0pt;line-height: 145%;text-align: justify;"><B>Abstract</B><SPAN class="s15">：</SPAN>There is no doubt that heavy metal contamination has detrimental effects on our environment and our health. For the sake of solving this challenging problem, our team comes up with the idea of this case. In our project, we intend to construct two plasmids to carry some specific elements and flocculation gene into the E.coli host<SPAN class="s23">，</SPAN>so that we can detect whether the environment is contaminated with heavy metal ions and report the concentration of them. Owing to the flocculation gene, we can collect the reconstructed bacteria just in case</P><P style="padding-top: 3pt;padding-left: 5pt;text-indent: 0pt;line-height: 155%;text-align: justify;">of repeated pollution. We focus on trace contaminates detection and set the goal of standardizing the whole detection protocol.</P></LI><LI style="padding-left: 26pt;text-indent: -21pt;text-align: justify;"><P class="s14" style="display: inline;">Title<SPAN class="s15">：</SPAN><SPAN class="p">Dioxin Detective Y</SPAN></P></LI><P class="s14" style="padding-top: 6pt;padding-left: 5pt;text-indent: 0pt;text-align: justify;">Yifan Wu <I>Harbin Institute of Technology</I></P><P class="s14" style="padding-top: 8pt;padding-left: 5pt;text-indent: 0pt;line-height: 148%;text-align: justify;">Abstract <SPAN class="s15">： </SPAN><SPAN class="p">Dioxins, by-products of various industrial processes, are commonly regarded as highly toxic compounds that are environmental pollutants and persistent organic pollutants. Its deleterious hazards intensify with the odorless, colorless and fat-soluble properties, which enable dioxins to accumulate in vivo thus threaten almost the whole biosphere. Considering their ability of causing reproductive and developmental problems, damaging the immune system, interfering with hormones and also leading to cancer, hand in glove with the difficulty of being tested, dioxins are worthy of the name “poison of the century”.</SPAN></P><P style="padding-left: 5pt;text-indent: 36pt;line-height: 150%;text-align: justify;">So as to detect dioxins blisteringly, we have constructed device for rapid detection of dioxins inside yeast. We utilized associability between the gene for testing dioxins of MOUSE as receptor and dioxins as antibodies, together with the regulation conducted by lexA operator to downstream promoters so that mouse dioxin receptor and lexA are combined, and, striding a step further, induce downstream gene with the presence of dioxins. Eventually, we can achieve our goal of detecting dioxins rapidly through the system of positive feedback of fluorescence. Concerning the properties of yeast, like non-toxic and able to survive in hypertonic environment, our project can be put into wide application in the field like food detection, hence guarantee the food safety among the community of we human beings.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:HIT-Harbin/WholeNameList">http://2014.igem.org/Team:HIT-Harbin/WholeNameList</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HIT-Harbin/WholeNameList" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HIT-Harbin/WholeNameList" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HIT-Harbin/WholeNameList&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HIT-Harbin/WholeNameList&amp;oldid=204179" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>