"
Page
Discussion
View source
History
teams
Log in
 
Team:HIT-Harbin/Notebook
From 2014.igem.org
Project
Background
Design
Advantages
Modeling
Safety
Wetlab
Protocols
Notebook
Software
Team
Attribution
Acknowledge
Profile
Human  practices
Preliminary
Meet Up
Course
Project
Background
Design
Advantages
Modeling
Safety
Wetlab
Protocols
Notebook
Software
Team
Attribution
Acknowledge
Profile
Human  practices
Preliminary
Meet Up
Course
Notebook
2013.11
We gave the lecture to students in both the first and the second campus for new teammates recruitment, invited professor Mario ,working on synthetic biology in our institute to give the lecture to students, then he became one of our instructors.
2013.11-2014.1
We gave a series of course to the new students who wanted to join the igem team. including the introduce of synthetic biology，software，modeling and experiment .The course last about 2mouth，we totally gave 8 courses，each of the course last 2 hours. Also the course was given by studens .
2014.1-2014.3
During the winter holiday ,we discussed the detail of the qualifying for igem team in HIT. Applied for the cost，bought materials and prepared for the questions.Finally we got about 50 students joined in the competition and give them the reword on the news web of HIT.
2014.4-2014.6
We had meetings per week to discuss the project. We set the experiment pars，software part，design parts and modeling part as four department of our team this year.Firstly, we found some information about the filed we are interested in, and tured the ideas into projects, then three of
The selectable idea are followed as Magnetotactic bacterium, Gene color and dioxin sensor.finally we decided to choose the dioxin sensor as our final project because of its good efficiency in environment protection. Then we researched a lot in the detail of the project, ensured to add a memory system for better response. We designed the lab work to go and ordered the sequence we want through company.
2014.6-2014.7
Since we were waiting for the ordered sequences. We were going to write some good software for this year’s Igem. We focused on the safety and communication. We became an idea that to make the safety idea more convenient for our life so that we established the advice software based on “ Wechat plate” on mobile. For communication we also designed a APP on mobile phone for students and teachers joining in igem to get the sequences which have registry in igem but don’t stored in Igem registry.
2014.7-2014.10
We did our experiment by Gibson protocol, we designed primers and PCR for the aiming sequence then do the Gibson. Since this is the first time for the method, we met lots of problem. More details please see the following document “Lab notebook”. Meanwhile  Lab notebook
2014.7.12
1. List the parts we would use this year,including:TEF2 Promoter (K801010),lexAop_mincyc1p(K1437004),LexA DBD(k165009),EYFP(E2030),ECFP(E2020), AhR receptor in mouse(K1437006), mouse AhR detective protein 521-805(k1437007), mouse AhR detective protein 521-805 without terminator(k1437008).
2. We calculated the concentration of the A and C antibiotic.
3.we prepared the medicine for Gibson and PCR.
2014.7.13
1. Detect the competent cell.
2. transformed the sequences we wanted into competent cells.
3.prepared the LB medium.
2014.7.3
Yesterday’s Results：we got much lexAop and mdr1-805,Terminator ADH didn’t grow up.
Experiments:
1.retransformed the lexaop,mdr1-805 and Terminator ADH.
2.Remake the competent cells.
3.passaged culture and abstract the plasmids of TEF2, k165034,EYFP,ECFP.
Today’s results：
1.The competent cells didn’t grow up so we decided to buy it.
2.We can’t get a single strain in mdr1-805 and lexaop.
2014.7.4
Experiments:
1.abstract plasmids and see it with a gel(TEF2/EYFP/lexAop4/ECFP).
2.Terminator ADH activated.
3.Continue with activating competent cells.
2014.7.5
Experiments:
1.Abstract the plasmids of Terminator ADH.
2.Lower the concentration of lexaop and mdr1-805 and put them into plate.
Today’s results：
We didn’t get the aiming sequence in the gel.May be we did it in a wrong way.
2014.7.7
Experiments:
1. got a single strain from the edge of the plate.
2. Did the PCR to check the length of the fragments.
3. Abstract the plasmid of Terminator ADH again.
2014.7.8
Experiments:
1.Did the PCR to check the aiming fragments(TEF2/ECFP/EYFP).
2.Got Terminator ADH sequence and actived the strain.
2014.7.9
We got the right sequence of the aiming sequence.
Experiments:
1.  Abstracted the plasmids(Terminator ADH/k165034,mdr1-805)and run a gel.
2.  store the strains with 40 percent glycerine.
2014.7.9-7.28
We had much course in the summer semester, so we temporarily discontinued our experiments.
2014.7.28
1. retransformed the Terminator ADH ,lexA DBD and lexAop by 10 times dilution of the original kits.
2. sterilizing materials.
2014.7.29
Yesterday’s Results：
There were much single strains in the plate.
Experiments:
Activated the strains.
2014.7.30
Experiments:
1.Abstracted plasmids and did the PCR for checking .
2.Mixed the LB medium and washed glasses.
Today’s results：
We got the plasmids of LexA DBD/lexaop/Terminator ADH.
2014.7.31
1. Did the PCR to add overlap for each parts.
2014.8.1-2014.8.11
We design the primers by the software designed by Cambridge but we forgot to add the plasmids so that they were much wrong primers. By these days, we redesign the primers.
2014.8.12
Experiments:
1.We did the PCR to add overlaps for Terminator ADH both for the m1/m2/m3.
2.Ran the gel for results.
Today’s results：
The gel leakaged.
2014.8.12-2014.9.13
We did the same circle,PCR-Run gel- purifying,during these days. Finally we got the LexADBD/Mdr83-805/mdr521-805/ECFP/EYFP/TEF2 for all the sequence. But their were too much unwanted amplifies of Terminator ADH and Also because of the quality of lexaop+cyc1promoter wasn’t good either, we can’t get the good results of the two mentioned fragments. By the advice given by Prof. Mario, we decided to purify the bad quality sequences and redid the PCR.We also redesigned the primers for Terminator ADH, wishing a good result.
2014.9.13
Experiments:
1.we did double enzymes cuts for both lexAop+cyc1 promoter and Psb1c3 with E enzyme and S enzyme.
2.PCR with both the original and redesigned primers for Terminator ADH.
3.Mixed LB solid medium.
Today’s results：
Failed for the PCR.
2014.9.14
Experiments:
Redesignedhe primers to add the standard suffix and prefix of the parts we wanted to hand on.
2014.9.15
Because we can’t get a satisfied result we thought may be the plasmid we used is wrong so that we retransform the Terminator ADH and lexAop+cyc1 promoter.
2014.9.16
Yesterday’s Results：
We got lexaop+cyc1 promoter but it is not quality. We couldn’t get Terminator ADH so that we were going to do the grade of the C concentration of the plate.
Experiments:
Today’s results：
1.did the grade of the C concentration of the plate.
2.Selected more lexaop+cyc1 promoter strain and abstracted plasmids.
2014.9.17
Experiments:
1. systematically the material we had now.
2. Actived lexaop+cyc1 promoter.
3.Double enzyme cut and link the psb1c3 with lexaop+cyc1 promoter.
2014.9.7-2.14.9.20
Yesterday’s Results：
Got Psb1c3+lexaop+cyc1 promoter.
Experiments:
1.PCR the Psb1c3+lexaop+cyc1 for aiming sequences.
2.Purified Terminator m1-m4 and boiling the primers for 5minutes,redo the PCR.
Got PRS306 and PRS304 and linearized them withenzymes.
2014.9.21-2014.9.22
Experiments:
1.  Did he grade PCR for good conditions of Terminator ADH.
2.  Detected the concentrations of fragments for Gibson method.
2014.9.23
Experiments:
1. We wanted to hand on another terminator called Terminator CYC. So we add the Overlap of standard suffix and prefix by PCR.
2.Cut the Psb1c3+lexaop+cyc1 with enzymes for checking.
3.PCR TADH with standard primers for checking.
4.PCR the lexA DBD with primers linked with yeast plasmid.
2014.9.24
Yesterday’s Results：
1.We got good results for all the PCR yesterday.
Experiments:
1.Did PCR for Terminator ADH m1-m4.
2.PCR with Mdr83-805 and mdr521-805 with standard suffix and prefix overlap.
3.Cut Terminator cyc by E and P enzymes.
2014.9.25
Experiments:
1.Amplifed the  Psb1c3+lexaop+cyc1.
2.purified Tcyc+PSB1C3,linked and transformed the plasmid into E.coli.
3.Do the Gibson.
Today’s results：
1.Gibson was failed.
2014.7.26
Experiments:
1.actived the yesterday’s plate.
2.Ran gel for TADH.
Today’s results：
1.The plant with Terminator CYC grew nothing.
2.The plant with mdr521-805 and mdr83-805 grew nothing.
3.Couldn’t get aiming sequence of Terminator ADH.
2014.7.28
Because we couldn’t get the aiming sequences so we decided to use traditional method to combine sequences with standard enzyme sites.
Meanwhile the PCR for Gibson method also undergoing.
Experiments:
1.Cut TEF2 with S, P enzymes and lexA DBD with X,P enzymes.
2.Cut ECFP with S, P enzymes and Terminator ADH with X,P enzymes.
2014.9.30
Experiments:
1.Purified, linked and transformed them into E.coli.
2014.10.1
yesterday’s results：
1.Strains grew well in plate.
xperiments:
1.Actived.
2.PCR with overlap of standard suffix and prefix of MDR83-805/mdr521-805 with or without terminator codon.
3.Double cut with E and S enzymes.
4.PCR the Terminator ADH of m1-m4.
2014.10.3
1.  transformed MDR521-508 etc. But failed.
2014.10.4-2014.10.10
Prepared the T plasmids and linked the we also do the blue-white blot selection. We got the T plasmids carrying the aiming fragments. Meanwhile we linked all the other fragments without the mouse dioxin receptor genes.
2014.10.11-2014.10.17
We found the reason why we can’t cut out the plasmid because the E enzyme doesn’t work. At the same time we got all the sequences for Gibson method and we did the Gibson, finally we got success except for the memory circuits. We transformed the plasmids into yeast and get the device.
Contrast between Igem method and Gibson method
Because we do both the igem method and Gibson method this year,so that after a series of fail, we make a conclude of the features of each method.
Learn More
Click here and get more details about our project.
Back To Top
Retrieved from " http://2014.igem.org/Team:HIT-Harbin/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
