<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NAU-CHINA_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NAU-CHINA/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Document</TITLE></P><DIV id="defaultCase"><DIV class="defaultCase_auto"><DIV class="header_banner"><DIV class="banner_text"><DIV class="banner_text_background"><SPAN class="banner_bg">Model</SPAN><SPAN class="banner_china">NAU-CHINA</SPAN></DIV></DIV></DIV><DIV class="top_nav"><DIV class="nav_bg"><UL class="nav_total"><LI class="first_level"><A href="javascript:"><SPAN>OVERVIEW</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/background" target="_blank"><SPAN>BACKGROUND</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/description" target="_blank"><SPAN>DESCRIPTION</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NAU-CHINA/Demonstrate" target="_blank"><SPAN>DEMONSTRATION</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NAU-CHINA/InterLab" target="_blank"><SPAN>INTERLAB</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NAU-CHINA/Software" target="_blank"><SPAN>SOFTWARE</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/hardware" target="_blank"><SPAN>HARDWARE</SPAN></A></LI></UL><LI class="first_level"><A href="javascript:"><SPAN>PROJECT</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/sensing" target="_blank"><SPAN>SENSING</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/antifungal" target="_blank"><SPAN>ANTIFUNGAL PEPTIDES</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/mycotoxin" target="_blank"><SPAN>MYCOTOXIN DEGRADATION</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/kill" target="_blank"><SPAN>KILL SWITCH</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/promoter" target="_blank"><SPAN>PROMOTER CHARACTERIZATION</SPAN></A></LI><LI class="first_level"><A href="javascript:" target="_blank"><SPAN>HUMAN PRACTICE</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NAU-CHINA/Collaborations" target="_blank"><SPAN>COLLABORATION</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NAU-CHINA/Engagement" target="_blank"><SPAN>PUBLIC ENGAGEMENT</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NAU-CHINA/HP/Gold_Integrated" target="_blank"><SPAN>INTERGRATED HUMAN PRACTICES</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NAU-CHINA/HP/Silver" target="_blank"><SPAN>HP SILVER</SPAN></A></LI><LI class="first_level">
							HOME
							<A href="https://2017.igem.org/Team:NAU-CHINA"><SPAN>HOME</SPAN></A></LI><LI class="first_level"><A href="https://2017.igem.org/Team:NAU-CHINA/Model" target="_blank"><SPAN>MODEL</SPAN></A></LI><LI class="first_level"><A href="javascript:"><SPAN>BIOBRICKS</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/basic" target="_blank"><SPAN>BASIC PART</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/composite" target="_blank"><SPAN>COMPOSITE PART</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/improved" target="_blank"><SPAN>IMPROVED PART</SPAN></A></LI><LI class="first_level"><A href="javascript:"><SPAN>TEAM</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/members" target="_blank"><SPAN>MEMBERS</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NAU-CHINA/Attributions" target="_blank"><SPAN>ATTRIBUTIONS</SPAN></A></LI><LI class="first_level"><A href="javascript:"><SPAN>NOTEBOOK</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/safety" target="_blank"><SPAN>SAFETY</SPAN></A></LI><LI><A href="https://2017.igem.org/Template:NAU-CHINA/protocols" target="_blank"><SPAN>PROTOCOLS</SPAN></A></LI></DIV></DIV><DIV id="content-model"><DIV class="section-header"><DIV class="comwidth"><DIV class="section-header-title"><P>Model</P></DIV><DIV class="section-header-content"><DIV class="header-content-text"><P>
		                                    This year, we design a yeast system to produce antifungl
		                                    peptide which inhibits the activity of Fusarium graminearum,
		                                    the pathogen of wheat scab. To choose the suitable antifungl
		                                    peptide, we establish an evaluation model and want to know
		                                     which antimicrobial peptide gets the highest score.
		                            </P></DIV><DIV class="header-content-text"><P>
		                                    We also design a complex system to sense and degrade the
		                                    toxin produced by Fusarium graminearum. Sensing and
		                                    degrading are dynamic processes so dynamic model is a good
		                                    choice to represent this part. Since the expression of degradation
		                                    enzymes are controlled by engineered photoactivatable switch,
		                                    we consider photochemical reaction kinetics and it performs
		                                    well.
		                                    In the same system, we put in a kill switch into the yeast. Yeast
		                                    will die without galactose induction so the safety of the system
		                                    is ensured. You can see how the yeast die by our model.
		                            </P></DIV></DIV></DIV></DIV><DIV class="section-categories"><DIV class="comwidth"><DIV class="section-categories-item section-categories-app"><DIV class="section-categories-title"><P>Antifungl Peptide Part</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>
		                                        We intend to use yeast system to produce a large number
		                                        of antifungl peptides to kill Fusarium graminearum which
		                                        causes the wheat scab. Therefore, the selection of antifungl
		                                        peptide is very important.
		                                </P></DIV><DIV class="section-categories-text"><P>
		                                        We searched for some antimicrobial peptide databases like
		                                        Peptaibol Database, Defensin Knowledgebase, MiniCOPE,
		                                        APD and so on. Finally, we found 20 antimicrobial peptides
		                                        that inhibit the activity of Fusarium graminearum.
		                                </P></DIV><DIV class="section-categories-text"><DIV class="categories-text-order"><DIV class="text-bold"><P>
		                                                The ideal antifungl peptide should have the following
		                                                characteristics:
		                                        </P></DIV><P>	(1)	Being harmless to host cells.</P><P>	(2)	Being harmless to the human body.</P><P>	(3)	Having little damage on microbes in the soil.</P></DIV><DIV class="section-categories-text"><P>
		                                            Therefore, we set effective antifungl concentration, Activity
		                                            object, half-life as three indexs, and established an evaluation
		                                            model to evaluate if the peptide is suitable for our project.
		                                    </P></DIV><DIV class="section-categories-text"><P class="text-addition">
		                                        Figure 1 : The scores of different antifungl peptides
		                                    </P></DIV><DIV class="section-categories-text"><P>
		                                            Finally, we choose PDC1 and cecropin A to kill Fusarium
		                                            graminearum. They can inhibit the growth of Fusarium
		                                            graminearum in a very low concentration(1.5μM、3.6μM),
		                                            PDC1 is specific for Fusarium graminearum.
		                                    </P></DIV></DIV></DIV></DIV><DIV class="section-categories-item section-categories-sp"><DIV class="section-categories-title"><P>Sensing Part</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>
		                                        We intend to sense toxins through the yeast system. In the
		                                        sensing part, estrogen receptor is secreted by constitutive
		                                        promoter, and then binding to the membrane. ZEN comes
		                                        into the cell via the receptor-mediated endocytosis process,
		                                        the endocytosis complex will activate ERE(estrogen response
		                                        element) which incorporated with a yeast promoter and the
		                                        promoter begin to express fluorescent protein.
		                                </P></DIV><DIV class="section-categories-text"><P class="text-addition">Figure 2 : Schematic diagram of the sensing part</P></DIV><DIV class="section-categories-text"><P>
		                                        In order to observe the fluorescence intensity in different
		                                        concentrations of ZEN and the corresponding reaction time.
		                                        We set a dynamic model by some differential equations:
		                                </P></DIV></DIV></DIV><DIV class="section-categories-item section-categories-simu"><DIV class="section-categories-title"><P>Simulation</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>
		                                        In different ZEN concentrations, the system will express different
		                                        intensities of fluorescent protein. We use MATLAB GUI to
		                                        simulate the process.
		                                </P></DIV><DIV class="section-categories-text"><P class="text-addition">Figure3: Simulation result</P></DIV><DIV class="section-categories-text"><P>
		                                        You can adjust the ZEN concentration and observe the change
		                                        of protein concentration. According to the Chinese National
		                                        Standards For Food Safety, ZEN’s concentration should not
		                                        high than 0.015 mg/L. If the concentration of ZEN is higher
		                                        than the level, the device will sense the fluorescence, turn on
		                                        the photoactivatable switch and degrade the toxin.
		                                </P></DIV></DIV></DIV><DIV class="section-categories-item section-categories-var section-step"><DIV class="section-categories-title"><P>Variable</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><TABLE><TBODY><TR><TD>ER</TD><TD>Estrogen receptor</TD></TR><TR><TD>ER<SPAN class="sub">cp</SPAN></TD><TD>Complex between estrogen receptor and ZEN</TD></TR><TR><TD>FLU</TD><TD>Fluorescent protein</TD></TR><TR><TD>ER<SPAN class="sub">m</SPAN></TD><TD>Estrogen receptor mRNA</TD></TR><TR><TD>FLU<SPAN class="sub">m</SPAN></TD><TD>Fluorescent protein mRNA</TD></TR><TR><TD>∂</TD><TD>The concentration of ZEN</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="section-categories-item section-categories-para"><DIV class="section-categories-title"><P>Parameters</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><TABLE><TBODY><TR><TD>Parameters</TD><TD>Description</TD><TD>Value</TD></TR><TR><TD>β<SPAN class="sub">1</SPAN></TD><TD>Maximun transcription rate of  constitutive promoter</TD><TD>5</TD></TR><TR><TD>β<SPAN class="sub">2</SPAN></TD><TD>The maximum transcription rate of ERE promoter</TD><TD>6</TD></TR><TR><TD>α</TD><TD>Translation rate per animoacid</TD><TD>1020</TD></TR><TR><TD>k<SPAN class="sub">1</SPAN></TD><TD>Endocytosis rate</TD><TD>4</TD></TR><TR><TD>k<SPAN class="sub">2</SPAN></TD><TD>Dssociation rate of complex ER<SPAN class="sub">cp</SPAN></TD><TD>1</TD></TR><TR><TD>d<SPAN class="sub">1</SPAN></TD><TD>Degradation rate of ER</TD><TD>0.8</TD></TR><TR><TD>d<SPAN class="sub">2</SPAN></TD><TD>Degradation rate of ER<SPAN class="sub">cp</SPAN></TD><TD>0.6</TD></TR><TR><TD>d<SPAN class="sub">3</SPAN></TD><TD>Degradation rate of FLU</TD><TD>0.8</TD></TR><TR><TD>d<SPAN class="sub">mrna</SPAN></TD><TD>Degradation rate of mRNA</TD><TD>0.231</TD></TR><TR><TD>s<SPAN class="sub">ER</SPAN></TD><TD>Length of Estrongen receptor</TD><TD>596</TD></TR><TR><TD>s<SPAN class="sub">FLU</SPAN></TD><TD>Length of Fluorescent protein</TD><TD>240</TD></TR><TR><TD>ξ</TD><TD>Leakage factor of the ERE promoter</TD><TD>0.02</TD></TR><TR><TD>k</TD><TD>The concentration of ER<SPAN class="sub">cp</SPAN>when the ERE promoter reaches half of the maximum generation rate</TD><TD>200</TD></TR><TR><TD>m</TD><TD>Hills coefficient</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="section-categories-item section-categories-dp"><DIV class="section-categories-title"><P>Degrading Part</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>
		                                    We intend to degrade toxins through E.coli system. When the
		                                    fluorescence intensity is higher than threshold, our device
		                                    will sense it and turn on the LED light. In our E.coli system,
		                                    N564 Fragment and C565 Fragment are expressed by the
		                                    constitutive promoter. Induced by LED light, N564 Fragment
		                                    and C565 Fragment will complex with each other. pT7 promoter
		                                    will be induced by the complex and then expresses ZHD101 and
		                                    TRI101 which respectively modifies ZEN and DON toxin,
		                                    modification will reduce their toxicity.
		                                </P></DIV><DIV class="section-categories-text"><P class="text-addition">Figure 4 : Schematic diagram of the degrading part</P></DIV><DIV class="section-categories-text"><P>
		                                    The first part is a kind of photochemical reaction, N564 Fragment
		                                    and C565 Fragment are activated by light and react with each
		                                    other.
		                                </P></DIV><DIV class="section-categories-text"><P>The reaction process can be described as following:</P></DIV><DIV class="section-categories-text"><P>
		                                    We can get some differential equations which describes the
		                                    process:
		                                </P></DIV></DIV></DIV><DIV class="section-categories-item section-categories-simulation"><DIV class="section-categories-title"><P>Simulation</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>We use Matlab to simulate the process.</P><P class="text-addition">Figure 5 : Simulation result</P></DIV><DIV class="section-categories-text"><P>
		                                        According to our simulation result, after about 36s, the
		                                        photoactivatable switch will fully open, the complex reaching
		                                        the maximum number of molecules, 220 molecules.
		                                </P></DIV><DIV class="section-categories-text"><P>
		                                        The following part is a dynamic model, the complex activate
		                                        pT7 promoter and pT7 expresses ZHD101 and TRI101. The
		                                        reaction process can be described as following:
		                                </P></DIV></DIV></DIV><DIV class="section-categories-item section-categories-simulation"><DIV class="section-categories-title"><P>Simulation</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>We use Matlab to simulate the process.</P><P class="text-addition">Figure 6 : Simulation result</P></DIV><DIV class="section-categories-text"><P>
		                                    According to our simulation result, after 35 minutes, ZHD101
		                                    is fully produced and reaches the maximum number of molecules,
		                                    600 molecules. At the same time, 30 minutes later, TRI101
		                                    reaches the max, 500 molecules.
		                                </P></DIV></DIV></DIV><DIV class="section-categories-item section-categories-Sa"><DIV class="section-categories-title"><P>
		                                    Sensitivity analysis
		                            </P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>
		                                        We briefly evaluate the sensitivity of the model by perturbing
		                                        the value of parameters. Changes of ±10% in β, COM, ξ, d1, d2,
		                                        sZHD, sTRI and k have different effects on the variable of ZEN
		                                        and TRI, as they are showed in Figure. We can see that changes
		                                        of ±10% in β, d1 and sZHD have a greater influence for the
		                                        variable ZEN, respectively, while the changes of d2 and sTRI affect
		                                        the variable TRI greatly. As indicated in Figure, the parameter
		                                        of COM has no effect on the results.
		                                </P></DIV><DIV class="section-categories-text"><P class="text-addition">Figure 7 : Simulation result</P></DIV></DIV></DIV><DIV class="section-categories-item section-categories-var"><DIV class="section-categories-title"><P>Variables</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><TABLE><TBODY><TR><TD>N567</TD><TD>N567 Fragment</TD></TR><TR><TD>C565</TD><TD>C565 Fragment</TD></TR><TR><TD>N564</TD><TD>N564 Fragment activated by light</TD></TR><TR><TD>C565</TD><TD>C565 Fragment activated by light</TD></TR><TR><TD>Complex</TD><TD>Complex between N564 Fragment and C565 Fragment</TD></TR><TR><TD>ZHD<SPAN class="sub">m</SPAN></TD><TD>ZHD101 mRNA</TD></TR><TR><TD>TRI<SPAN class="sub">m</SPAN></TD><TD>TRI101 mRNA</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="section-categories-item section-categories-para"><DIV class="section-categories-title"><P>Parameters</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><TABLE><TBODY><TR><TD>Parameters</TD><TD>Description</TD><TD>Value</TD></TR><TR><TD>k<SPAN class="">1</SPAN></TD><TD>The 1st light reaction rate</TD><TD>1</TD></TR><TR><TD>k<SPAN class="">2</SPAN></TD><TD>The 2nd light reaction rate</TD><TD>1</TD></TR><TR><TD>k<SPAN class="">3</SPAN></TD><TD>The 3rd light reaction rate</TD><TD>0.5</TD></TR><TR><TD>k<SPAN class="">4</SPAN></TD><TD>The 4th light reaction rate</TD><TD>0.1</TD></TR><TR><TD>k<SPAN class="">5</SPAN></TD><TD>The 5th light reation rate</TD><TD>0.1</TD></TR><TR><TD>k</TD><TD>The concentration of product when pT7 promoter reaches half of the maximum generation rate</TD><TD>40</TD></TR><TR><TD>I<SPAN class="sub">a</SPAN></TD><TD>Incident light intensity</TD><TD>0.5</TD></TR><TR><TD>α</TD><TD>Translation rate per aminoacid</TD><TD>1020</TD></TR><TR><TD>β</TD><TD>Maximum transcription rate of pT7 promoter</TD><TD>24</TD></TR><TR><TD>ξ</TD><TD>Leakage factor of pT7 promoter</TD><TD>0.24</TD></TR><TR><TD>d<SPAN class="sub">1</SPAN></TD><TD>Degradation rate of ZHD</TD><TD>0.4</TD></TR><TR><TD>d<SPAN class="sub">2</SPAN></TD><TD>Degradation rate of TRI</TD><TD>0.8</TD></TR><TR><TD>d<SPAN class="sub">mrna</SPAN></TD><TD>Degradation rate of mRNA</TD><TD>0.2</TD></TR><TR><TD>s<SPAN class="sub">ZHD</SPAN></TD><TD>Length of ZHD</TD><TD>517</TD></TR><TR><TD>s<SPAN class="sub">TRI</SPAN></TD><TD>Length of TRI</TD><TD>174</TD></TR><TR><TD>m</TD><TD>Hills coefficient</TD><TD>1</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="section-categories-item section-categories-"><DIV class="section-categories-title"><P>Kill Switch Part</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>
		                                        In the final part, we design a kill switch to control our system’s
		                                        hazards to the environment. Without galactose induction to
		                                        the system, TFs will not been expressed and BI-1 will not been
		                                        induced. So the concentration of Bax is much higher than BI-1,
		                                        the yeast will die.
		                                </P></DIV><DIV class="section-categories-text"><P class="text-addition">Figure 8 : Schematic diagram of Kill Switch </P></DIV><DIV class="section-categories-text"><P>
		                                        Since Bax is expressed by a constitutive promoter, we can describe
		                                        it as following:
		                                </P></DIV></DIV></DIV><DIV class="section-categories-item section-categories-simu"><DIV class="section-categories-title"><P>Simulation</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>
		                                        We use Matlab to simulate the process.
		                                </P></DIV><DIV class="section-categories-text"><P class="text-addition">Figure 9 : Simulation result</P></DIV><DIV class="section-categories-text"><P>
		                                        According to our simulation result, produced by a constitutive
		                                        promoter, Bax is fully expressed after 80 minuutes and it’s highest
		                                        molecule number is 380.
		                                </P></DIV><DIV class="section-categories-text"><P>
		                                        BI-1 is induced by TFs which is induced by galactose, we can
		                                        describe it as following:
		                                </P></DIV></DIV></DIV><DIV class="section-categories-item section-categories-simu"><DIV class="section-categories-title"><P>Simulation</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>
		                                        In different Galactose concentrations, the system will express
		                                        different concentrations of BI-1. We use MATLAB GUI to simulate
		                                        the process.
		                                </P><P class="text-addition">Figure 10 : Simulation result</P></DIV><DIV class="section-categories-text"><P>
		                                        You can adjust the Galactose concentration and observe the
		                                        change of BI-1’s concentration. If BI-1’s concentration is higher
		                                        than Bax which simulated before, the yeast will survive. From
		                                        the simulation result, we know if galactose’s concentration is
		                                        lower than 0.02g/ml, the yeast will die.
		                                </P></DIV></DIV></DIV><DIV class="section-categories-item section-categories-var section-step"><DIV class="section-categories-title"><P>Variables</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><TABLE><TBODY><TR><TD>Bax<SPAN class="sub">m</SPAN></TD><TD>Bax protein mRNA</TD></TR><TR><TD>TFs<SPAN class="sub">m</SPAN></TD><TD>Plat-derived TFs protein mRNA</TD></TR><TR><TD>Bl1<SPAN class="sub">m</SPAN></TD><TD>BI-1 protein mRNA</TD></TR><TR><TD>Bax</TD><TD>Bax protein</TD></TR><TR><TD>TFs</TD><TD>Plant-derived TFs protein</TD></TR><TR><TD>Bl1</TD><TD>Bl-1 protein</TD></TR><TR><TD>gal</TD><TD>Galactose</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="section-categories-item section-categories-parameters section-step"><DIV class="section-categories-title"><P>Parameters</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><TABLE><TBODY><TR><TD>α</TD><TD>Translation rate per aminoacid</TD><TD>1020</TD></TR><TR><TD>β</TD><TD>Maximum transcription rate of constitutive promoter</TD><TD>1</TD></TR><TR><TD>β<SPAN class="sub">1</SPAN></TD><TD>Maximum transcription rate of TFs promoter</TD><TD>4</TD></TR><TR><TD>β<SPAN class="sub">2</SPAN></TD><TD>Maximum transcription rate of BL1 promoter</TD><TD>6</TD></TR><TR><TD>ξ<SPAN class="sub">1</SPAN></TD><TD>Leakage factor of TFs promoter</TD><TD>0.02</TD></TR><TR><TD>ξ<SPAN class="sub">2</SPAN></TD><TD>Leakage factor of BL-1 promoter</TD><TD>0.02</TD></TR><TR><TD>k<SPAN class="sub">1</SPAN></TD><TD>The concentration of galactose when TFs promoter reaches half of the maximum generation rate</TD><TD>0.02</TD></TR><TR><TD>k<SPAN class="sub">2</SPAN></TD><TD>The concentration of TFs when Bl-1 promoter reaches half of the maximum generation rate</TD><TD>200</TD></TR><TR><TD>d</TD><TD>Degradation rate of Bax</TD><TD>0.06</TD></TR><TR><TD>d<SPAN class="sub">1</SPAN></TD><TD>Degradation rate of TFs</TD><TD>0.8</TD></TR><TR><TD>d<SPAN class="sub">2</SPAN></TD><TD>Degradation rate of Bl-1</TD><TD>1</TD></TR><TR><TD>d<SPAN class="sub">mrna</SPAN></TD><TD>Degradation rate of mRNA</TD><TD>0.231</TD></TR><TR><TD>s<SPAN class="sub">Bax</SPAN></TD><TD>Length of Bax protein</TD><TD>191</TD></TR><TR><TD>s<SPAN class="sub">TFs</SPAN></TD><TD>Length of TFs protein</TD><TD>280</TD></TR><TR><TD>d<SPAN class="sub">Bl1</SPAN></TD><TD>Length of BL-1 protein</TD><TD>138</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="section-categories-item section-categories-refer"><DIV class="section-categories-title"><P>Reference</P></DIV><DIV class="section-categories-detail"><DIV class="section-categories-text"><P>
		                                        [1] Pragya Kant, Wen-Zhe Liu, K. Peter Pauls. PDC1, a corn defensin
		                                        peptide expressed in Escherichia coli and Pichia pastoris inhibits
		                                        growth of Fusarium graminearum. Peptides 30 (2009) 1593–1599.
		                                </P><P>
		                                        [2] ANTHONY J. DELUCCA, JOHN M. BLAND, THOMAS J. JACKS,
		                                        CASEY GRIMM, THOMAS E. CLEVELAND, AND THOMAS J. WALSH.
		                                        Fungicidal Activity of Cecropin A. ANTIMICROBIAL AGENTS AND
		                                        CHEMOTHERAPY,Feb. 1997, p. 481–483.
		                                </P><P>
		                                        [3] Tiyun Han, Quan Chen, and Haiyan Liu. Engineered Photo-
		                                        activatable Genetic Switches Based on the Bacterium Phage
		                                        T7 RNA Polymerase. DOI: 10.1021/acssynbio.6b00248. ACS
		                                        Synth. Biol. 2017, 6, 357−366
		                                </P><P>
		                                        [4] Gita Naseri, Salma Balazadeh, Fabian Machens, Iman Kam-
		                                        ranfar, Katrin Messerschmidt, and Bernd Mueller-Roeber. Plant
		                                        Derived Transcription Factors for Orthologous Regulation of
		                                        Gene Expression in the Yeast Saccharomyces cerevisiae.
		                                </P><P>
		                                        DOI: 10.1021/acssynbio.7b00094. ACS Synth. Biol. XXXX, XXX,
		                                        XXX−XXX
		                                </P><P>
		                                        [5] https://2016.igem.org/Team:NAU-CHINA/Model/DM
		                                </P><P>
		                                        [6] https://2013.igem.org/Team:TU-Delft/Timer-Sumo-KillSwitch
		                                </P><P>
		                                        [7] https://2012.igem.org/Team:HIT-Harbin/project/model
		                                </P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="main"><UL class="floorMenu"><SPAN class="foorMenu_come">   Orz</SPAN><LI class="active">Antifungl Peptide Part</LI><LI>Sensing Part</LI><LI>Simulation</LI><LI>Variable</LI><LI>Parameters</LI><LI>Degrading Part</LI><LI>Simulation</LI><LI>Simulation</LI><LI>Sensitivity analysis</LI><LI>Variables</LI><LI>Parameters</LI><LI>Kill Switch Part</LI><LI>Simulation</LI><LI>Simulation</LI><LI>Variables</LI><LI>Parameters</LI><LI>Reference</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>