<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Northeastern_Boston_Workflow skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Northeastern Boston/Workflow</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="wrapper"><DIV id="sidebar"><DIV id="bar_title" class="title"><A href="https://2015.igem.org/Team:Northeastern_Boston">
	      Northeastern</A></DIV><UL class="nav"><LI><A id="a-home" href="https://2015.igem.org/Team:Northeastern_Boston">Home</A></LI><LI><A id="a-team" href="https://2015.igem.org/Team:Northeastern_Boston/Attributions">Team</A></LI><LI><A id="a-project" href="https://2015.igem.org/Team:Northeastern_Boston/Description">Project</A></LI><DIV class="sub-nav__container"><LI id="a-parts" class="sub-nav__parent"><A>
		        	Parts
		        	</A></LI><DIV class="sub-nav__children hidden"><LI class="sub-nav__child"><A href="https://2015.igem.org/Team:Northeastern_Boston/BasicParts">Basic &amp; Composite</A></LI><LI class="sub-nav__child"><A href="https://2015.igem.org/Team:Northeastern_Boston/Design">Expression Plasmid</A></LI></DIV></DIV><DIV class="sub-nav__container"><LI id="a-notebook" class="sub-nav__parent"><A>
		        	Notebook 
		        	</A></LI><DIV class="sub-nav__children"><LI class="sub-nav__child"><A href="https://2015.igem.org/Team:Northeastern_Boston/notebook">Protocols</A></LI><LI class="sub-nav__child"><A class="active">Workflow</A></LI></DIV></DIV><DIV class="sub-nav__container"><LI id="a-humanpractices" class="sub-nav__parent"><A>
		        	Human Practices
		        	</A></LI><DIV class="sub-nav__children hidden"><LI class="sub-nav__child"><A href="https://2015.igem.org/Team:Northeastern_Boston/Collaborations">Collaborations</A></LI><LI class="sub-nav__child"><A href="https://2015.igem.org/Team:Northeastern_Boston/Practices">Outreach</A></LI><LI class="sub-nav__child"><A href="https://2015.igem.org/Team:Northeastern_Boston/Written_Investigation">Written Investigation</A></LI></DIV></DIV><LI><A id="a-safety" href="https://2015.igem.org/Team:Northeastern_Boston/safety">Safety</A></LI><LI><A id="a-measurement" href="https://2015.igem.org/Team:Northeastern_Boston/measurement">Interlab</A></LI></UL></DIV><DIV class="content"><DIV id="page-header__workflow" class="page-header"><H1 class="page-header__title" align="right" style="border-width:0;color:white">WORKFLOW</H1></DIV><DIV id="project_description" class="article"><H2>July 13th thru July 19th</H2><P>We’ve begun by making agar plates, competent TOP10 <I>E. coli</I>, and stock antibiotics. We’ve also transformed the 4 strains for the Interlab and miniprepped the subsequent plasmids. </P><P>We’ve begun to PCR the parts for the two planned expression plasmids above (Benchling). Each part contains primers with homology arms that will match the neighboring pieces. One plasmid is for the nucleus and the other is for the chloroplast. The plan is to produce a fluorescent protein in the nucleus, and a luminescent protein in the chloroplast. Both these &quot;test plasmids&quot;, will then validate the expression cassettes for heterologous protein production.</P><H2>July 20th thru July 26th</H2><P>Interlab plasmids were digested with their respective enzymes and then purified from the gel—after staining with ethidium bromide and UV illumination—by column purification. The three Anderson promoters were mixed and ligated with the GFP from I13504. These three ligation mixtures, and the controls reconstituted, were used to transform by heat shock, 5 strains of TOP10 <I>E. coli</I>. These strains were then plated on agar plates with Cam and, after overnight incubation at 37 degrees, subcultured into liquid broth with Cam. Resulting colonies were screened with a fluorescent microscope and colonies expressing GFP were subcultured into a 96 well plate (3x3: biologicalxtechnical). One colony from each group was subcultured overnight, then miniprepped, digested, and run thru a gel to check for bands at the appropriate sizes. Strains in the 96 well plate were grown overnight, and then read for OD600 on a multiplate reader. A dilution curve was created and the wells were diluted to within 5% of 0.5. Fluorescence was then read (485/528nm).</P><P>PCR continues for the two protein expression plasmids. Some parts have been forthcoming, while others have taken repeated attempt. Template DNA is coming from a variety of sources: ChlamyCollection.org, iGEM registry, and Mayfield. Bba_K1547005 is being used for pPsaD, and Bba_K1547001 is being used for tPsaD. The algae from ChlamyCollection, a nit-knockout strain, have been growing on TAP plates and in TAP liquid media for five days, with no signs of growth.</P><H2>July 27th thru August 2nd</H2><P>So far we have the PCR’ed Gibson-ready parts Lux, psb 5’ UTR thru 16s-atpA, and  pPsaD for the chloroplast protein expression plasmid. We have been unable to clone RbcL 3’ UTR or psbA 3’UTR thru psbH 3’ UTR yet. We are also halfway thru the nuclear expression plasmid, with the Nitrate promoter, TagBFP, Ble, and the backbone amplifying while the rest (tRBCS2, tPsaD, and pPsaD) have not. The PCR's and gels continue.</P><P>There is still no sign of growth on the nit-knockout algae plates or erlenmeyer flask. The TAP media contains ammonium, so it's concerning that the Nit- algae strain is unable to grow, since they should, according to literature, be able to. It is also unclear whether we will be able to use a gene-gun for the chloroplast transformation.</P><H2>August 3rd thru August 9th</H2><P>We've decided to hold efforts on the chloroplast plasmid, focusing exclusively on the nuclear-bound plasmid for now. There in no clear indication that we'll have access to a gene-gun and the literature suggests that transformation recovery periods are much greater for particle-bombarded microalgae relative to glass bead transformed microalgae. Furthermore, the Nit-knockout algae strain has exhibited no signs of growth over the past week and half. Therefore, we've redesigned the nuclear plasmid (as shown above). The Nit-knockout strain was dependent on an absence of reduced nitrogen in the media to drive expression. Given the slow or non-existent growth of that strain, we've ordered a Nit-competent strain thru ChlamyCollection and redesigned the plasmid as follows. It features a constitutive promoter, pPsaD, rather than an inducible promoter, Nit. Additionally, HSP70A is being used as the promoter for Zeocin resistance and will be cloned from Bba_K640001 from the iGEM repository.</P><P>Due to the rearranged nature of the new pPsaD driven expression plasmid, the pieces must be recloned with newly appropriate homology arms. We have PCR'ed and purified pPsaD, TagBFP, tRBCS2, pHSP70-RBCS2, and Ble. The Backbone and tPsaD have been less forthcoming.</P><H2>August 10th thru August 16th</H2><P>PCR attempts continue for tPsaD. The Backbone was successful, however the terminator tPsaD has not successfully cloned from any of the following: Bba_K1547010, Bba_K1547001, or Bba_K1547000. We've resorted to synthesizing the piece from IDT and are now waiting for its arrival.</P><P> The new strain, cc-400 mt+, has arrived from ChlamyCollection. It has been subcultured into a 6 well plate, each well containing 6mL of TAP, and placed on an orbital shaker at 70rpm. Using a lights timer found in the lab, we've programmed the light to a 12:4:4:4 (light:dark:light:dark) cycle, per <A href="”https://books.google.com/books?id=xTjJGV5GWY0C&amp;printsec=frontcover&amp;dq=chlamydomonas+sourcebook&amp;hl=en&amp;sa=X&amp;ved=0CBwQ6AEwAGoVChMI87uD_c_6xwIVBl4eCh3d9g2r#v=onepage&amp;q=chlamydomonas%20sourcebook&amp;f=false”" target="”_blank”">Chlamy Sourcebook.</A></P><H2>August 17th thru August 23rd</H2><P>tPsaD has arrived from IDT. A Gibson Digest was attempted with the seven pieces; however, no colonies appeared on a Cam selection plate. A second Gibson Digest was attempted with greater attention towards part ratios according to Nanodrop. ~10 colonies appeared after plating with glass beads. Colony PCR was performed on 6 of these colonies with primers corresponding to pPsaD thru Ble. Unfortunately, none of the colonies screened positively. The resulting bands were half the expected size of 3000bp. An additional Gibson Digest will be performed.</P><P>500uL of <I>Chlamydomonas reinhardtii</I> was subcultured from the 6-well plate to the Erlenmeyer flask on the orbital shaker. The same light cycle (12:4:4:4) was used and OD750 was determined (to be continued).</P><H2>August 24th thru August 31st</H2><P>Additional Gibson digests continue to yield colonies with partial-plasmid (~1,500 when screened with pPsaD thru Ble). A sequential PCR screen was performed where pieces three long were screened: pPsaD thru tRBCS2, TagBFP thru pHSP70A, and tRBCS2 thru Ble. None of the products were at expected sizes. This result was constant across biological replicates. Whatever incorrect product the Gibson is being generated by the reaction, it’s happening consistently. Benchling does not lend any clues towards improper or duplicate homology arms (each being ~20bp).</P><P>Given the dwindling time and repeated failures into a custom protein expression plasmid, it now seems worthwhile to try adapting and/or piecing together an existing expression cassette, accepting the probable lower levels of protein expression in favor of reliability. Therefore, pOpt_mVenus_Hyg has been streaked from a frozen culture, subcultured, and miniprepped. Primers have been ordered to adapt two parts from the plasmid for iGEM: a hygromycin resistance cassette (something that would have been enormously helpful) and mVenus (a YFP variant, which will be of use since iGEM lacks an independent fluorescent marker).</P><P>The adapted protein expression plasmid needs work. There is a PstI restriction site in the middle of it, and we will not want to include mVenus in the iGEM submitted part. Instead, we believe it will be most useful with an iGEM suffix after the HSP70A-RBCS2 promoter/intron1, and before the terminator. It will also incorporate a hygromycin B resistance cassette after the expression portion of the plasmid but before pSB1C3. In this way, teams will be able to insert a coding sequence for a protein and use the plasmid directly for heterologous protein.</P><H2>September 1st thru September 6th</H2><P>PCR was completed for the iGEM modified protein expression plasmid. The plasmid has been assembled by restriction digest, grown/miniprepped from <I>E. coli</I> and transformed into <I>C. reinhardtii</I> using the glass bead transformation method. Transformed microalgae has been plated onto TAP agar plates with 50ug/mL of Hygromycin B and placed beneath a light on a 12:4:4:4 hour cycle. The results of the 10 day growth curve were compiled (above). 
</P><H2>September 7th thru September 13th</H2><P>Parts for mVenus, Hyg_resistance, and the iGEM compatible expression plasmid have been subcultured, miniprepped, and added to the submission plate. We're now waiting to see whether any <I>C. reinhardtii</I> colonies grow, preferably ones expressing mVenus in addition to the Hygromycin B resistance gene (Aph7&quot;). Unfortunately, there have been no signs of growth after a single week, so an additional batch of <I>C. reinhardtii</I> were transformed by glass bead method again and plated on a new batch of TAP_Hygromycin B plates. Untransformed <I>C. reinhardtii</I> were imaged (below).</P><H2>September 14th thru September 18th</H2><P>Given the complete absence of colonies, there appears to be a complete lack of conferred Hygromycin B resistance. There are several factors that might have contributed to the lack of growth: too high a Hygromycin B concentration in the agar, too long of bead vortexing, transformation of insufficient numbers of algae, etc. While it would be nice to redo the transformations controlling for each variable, there's not enough time left before the wiki-freeze. Therefore, we're compiling data and calling it a year. Hopefully next years team has some more time to test the expression plasmid, and protein expression in general (important given the latency between transforming the Chlamy and seeing growth).</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>