<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="primary_nav_wrap" style="z-index: 2000"><UL><LI class="mobicon"><A href="javascript:void(0);" onclick="mobnavtoggle()">≡ Menu</A></LI><LI><A href="https://2017.igem.org/Team:Groningen">Home</A></LI><LI><A onclick="">Project</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/FutureResearch">Future Research</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/InterLab">InterLab</A></LI></UL><LI><A onclick="">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Overview">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/HP/Silver">Human Practices (Silver)</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/HP/Gold_Integrated">Integrated (Gold)</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/FutureOutlook">Future Outlook</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A onclick="">Team</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Team">Team Members</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Achievements">Achievements</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Attributions">Attributions</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Fun">Fun Activities</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Sponsors">Sponsors</A></LI></DIV><DIV class="marginsnav"><DIV class="main-col scrollcol"><NAV id="myScrollspy"><UL class="nav"><LI class="active"><A href="#snavprojectdesc">Introduction</A></LI><LI><A href="#snavcrisprcas">CRISPR-Cas</A></LI><LI><A href="#snavspacer">Spacer Acquisition</A></LI><LI><A href="#snavcrisprinter">CRISPR-interference</A></LI><LI><A href="#snavreporter">Reporter Plasmid</A></LI><LI><A href="#snavtoolbox">Lactis Toolbox</A></LI><LI><A href="#snavreferences">References</A></LI></UL></NAV><H1>Description</H1><H3 id="snavprojectdesc">Introduction</H3><DIV><P class="left">Our IMPACT (Integrated Modular Phage Activated CRISPR Tracer)-system contains <I>Lactococcus lactis</I> cells that are capable of detecting specific bacteriophage infections. In their native, non-infected state, the cells will emit both GFP &amp; RFP fluorescence. If our cells get infected with a certain phage they will only down-regulate GFP while RFP fluorescence will be maintained. In this section we will explain how we envision this system to work.</P><P class="left">In the animation above a general overview of the system is provided. In the first step, a hyperactive CRISPR-Cas complex will take up spacers from the invading phage DNA and implement them into the CRISPR array. In the following step a short RNA corresponding to the new spacer, the crRNA, is used by dCas9 to interfere with the expression of a reporter construct.  If a match is found, our dCas9 will target and bind to one of the corresponding target sequences that we have pre-programmed. These target sequences are located between the transcription start site and the ribosome binding site of our reporter gene. Binding of dCas9 then disrupts expression of the reporter gene and results in a detectable signal decrease. To determine which spacer(s) should be pre-programmed on our target array we have developed a model to determine the likelihood that a certain spacer will be taken up. The whole system shall be explained in more detail in the sections down below.</P></DIV><P class="left">Finally, the entire system needs to be constructed in <I>L. lactis</I>, which is a Lactic Acid Bacterium (LAB). Working with <I>L. lactis</I> requires some special attention, it is not as easily transformed as <I>Escherichia coli</I> or <I>Bacillus subtilis</I> and their promotors tend to not work in <I>L. lactis</I>. Moreover <I>L. lactis</I> is an important organism in fermentation industries, but not a lot of information or parts can be found in the iGEM database. Therefore, we created a fourth sub-project, called lactis-toolbox, in which we share the problems we encountered, our protocols and in which we created three new <I>L. lactis</I> biobrick compatible promotor parts.</P><P class="left">We will start by giving a short introduction to the most relevant aspects of CRISPR-Cas before going into more detail about our different sub-projects since our entire project involves CRISPR-Cas.</P><H5 id="snavcrisprcas"> CRISPR-Cas </H5><P class="left">In our project we will make use of the CRISPR-Cas system of <I>Streptococcus pyogenes</I>. This system is a type II-A and consists of four proteins and two RNA molecules <A class="mouseover" title="Wei et al. (2015). Cas9 function and host genome sampling in Type II-A CRISPR–Cas adaptation. Genes &amp; Development, 29(4), 356–361.">[Wei (2015)]</A>. The natural CRISPR-system is a defense mechanism against invading bacteriophages.  Often only one part of the system, namely Cas9, is used to target specific nucleotide sequences. First, a quick overview of the mechanism behind CRISPR immunity will be provided followed by a more detailed description of all components.</P><P class="left">CRISPR immunity can be seen as a three-stage process: The first stage begins with the bacteriophage inserting its DNA into the host's cell. A spacer is acquired, which is a small (~20 nt) fragment originating from the foreign DNA. This spacer is incorporated into the CRISPR array, which is a collection of multiple spacers flanked by repeat regions. This region is also responsible for the CRISPR acronym, which stands for <B>C</B>lustered <B>R</B>egularly <B>I</B>nterspaced <B>S</B>hort <B>P</B>alindromic <B>R</B>epeats.  In the second stage of defense, this array is transcribed and a pre-crRNA is produced, which is subsequently processed into separate crRNA molecules. Finally, Cas9 can utilize the processed crRNA’s together with a tracrRNA to target a specific sequence encoded by the spacer, bind to it and use it endonuclease capability to cleave. The CRISPR array offers the cell a molecular memory of infection enabling it to fend off bacteriophages that match the incorporated spacers.</P><P class="left">Four proteins comprise the <I>S. pyogenes</I> CRISPR-Cas system (Cas1, Cas2, Csn2 and Cas9) and are expressed together in a single operon including the crRNA and tracrRNA genes. In our project, we will use a plasmid constructed by <A class="mouseover" title="Heler, R. et al. Mutations in Cas9 Enhance the Rate of Acquisition of Viral Spacer
Sequences during the CRISPR-Cas Immune Response. Mol. Cell 65, 168–175 (2017)
">Heler (2017)</A> containing only the four proteins coding genes. Therefore all references to the CRISPR operon will be referring to this plasmids CRISPR version and not the natural organization. </P><P class="left">The tracrRNA is one of the two RNA molecules in the natural CRISPR system. This RNA binds to the repeat region of the crRNA (transcribed and processes from spacer array) and together they form a complex with the Cas9 protein. Another RNA molecule that is often used in combination with CRISPR-Cas is a guide RNA (gRNA). The gRNA is not a component of the natural CRISPR-system, but it is a single synthetic RNA that resembles the crRNA:tracrRNA dimer. </P><P class="left">By chemically synthesizing the part of the gRNA that corresponds to the spacer, new guides can be made to target Cas9 to almost any sequence. A whole range of variants of CRISPR-Cas9 have been developed that allow for transcription inhibition (CRISPRi), Gene knock-ins or knock-outs and many more functions. One restriction on the Cas9 target is that it needs to be flanked by a short sequence, (NGG in the natural spCas9), which is called the PAM sequence. Recognition of this sequence is a property of the Cas9 protein and is not orchestrated by the RNA molecule <A class="mouseover" title="Sternberg, H. et al. DNA interrogation by the CRISPR RNA-guided endonuclease Cas9. Nature 507, 62-67 (2014)">[Sternberg (2014)]</A>.</P><H5 id="snavspacer"> Spacer Acquisition </H5><P class="left">The first sub-project is concerned with the spacer acquisition for which we use a slightly adapted version of the <I>S. pyogenes</I> CRISPR-Cas system. Instead of using the normal CRISPR system we will use a hyperactive Cas9 (hCas9) mutant. <A class="mouseover" title="Heler, R. et al. Mutations in Cas9 Enhance the Rate of Acquisition of Viral Spacer Sequences during the CRISPR-Cas Immune Response. Mol. Cell 65, 168–175 (2017).">[Heler (2017)]
</A> described a single amino acid substitution that turns Cas9 into hCas9, which resulted in a ~100 fold increase in the spacer uptake rate. </P><P class="left"> We have decided to use this hCas9-variant to boost the relatively low spacer uptake rate of the native Cas9 protein complex. This will increase the chance that the correct spacer (for which we pre-programmed the cells) is taken up upon infection. </P><P class="left">A disadvantage of using the hyperactive Cas9 is that it is known to not only take up spacers from invading bacteriophage DNA but also plasmids and even the genome. If hCas9 is directed to these interior genetic elements, this can result in the clearing of the plasmid and the corresponding antibiotic gene, which leads to cell death. Combining the hCas9 mutation with mutations disrupting the nuclease domains of Cas9 (dCas9) would solve this but raises other problems. dCas9 is a widely used Cas9 variant due it allowing for down-regulation of gene expression in a process called CRISPR-interference (CRISPRi) <A class="mouseover" title="Larson et al. CRISPR interference (CRISPRi) for sequence-specific control of gene expression. Nature Protocols, 8(11), pp.2180-2196. (2013).">[Larson (2013)]</A>.By making use of this variant we prevent the reporter plasmid from being degraded when the system is activated and therefore activation will not lead to death. </P><P class="left"> The problem with using only dCas9 is that it would deprive our cells of the selective advantage resulting from taking up a phage spacer. In addition, this would result in high false positive rates, as spacers taken up from or near the target array on the reporter plasmid would result in dCas9 interference. 
This led us to use catalytically active hCas9 together with a separate dCas9 carrying mutations altering its PAM preference. With this setup spacer-uptake by the hCas9 from the reporter plasmid would not result in unintended activation of our reporter gene. Instead, those false positive spacers would case hCas9 to cleave the report plasmid and be lost from the population due to the severe fitness disadvantage caused by the disruption of the plasmid's antibiotic gene.</P><H5 id="snavcrisprinter"> CRISPR-interference </H5><P class="left"> To make sure that dCas9 and not the hCas9 targets the plasmid, we will introduce four mutations in dCas9 that change the PAM preference of Cas9 <A class="mouseover" title="Kleinstiver, B. P. et al. Engineered CRISPR-Cas9 nucleases with altered PAM specificities. Nature 523, 481–485 (2015).">[Kleinstiver (2015)]</A>. Our plan is to combine the PAM-altering mutations with those of dCas9 to obtain a dCas9VRER. Using this dCas9 with an altered PAM preference provides us with a great number of advantages. </P><P class="left">First of all, changing the PAM-recognition site from -NGG- to -NGCG- allows us to direct the two Cas9 variants towards different targets. Spacers that get taken up by hCas9 need to be flanked by an NGG on the bacteriophage sequence. In our target array the targeted spacers are flanked by NGCG and therefore our target array will not be targeted by the hCas9. Thus, our plasmid will not be cleaved and the cells will stay resistant to the antibiotic.</P><P class="left">Next, since the hCas9 is so much more efficient in taking up spacers than dCas9, it will be very unlikely that dCas9 will take up a spacer, which would be flanked by an NGCG sequence. As a result, it is not likely that our detector will result in a signal after a spacer, originating from the plasmid, is adapted.  If hCas9 would take up a spacer from the plasmid the cells will die, as debated in the previous section.</P><H5 id="snavreporter"> Reporter plasmid </H5><P class="left">
This sub-project is concerned with developing a system for converting spacer acquisition by the CRISPR operon into a signal. We judged that gene repression by dCas9 (CRISPRi) guided by newly acquired spacers to be the most robust and flexible system. We designed a reporter plasmid pre-programmed with target sites predicted by our <A href="https://2017.igem.org/Team:Groningen/Model">bioinformatics tool</A> to have a high rate of acquisition. These target sites are in the transcript of our reporter and match the viral genome, but are flanked by the orthogonal PAM sequence. </P><P>Crucially, an addition of the target sites does not alter the coding sequence of our reporter. They are placed downstream of the promoter, but directly upstream of the ribosome binding site and open reading frame of the reporter protein. This system allows normal transcription of the reporter until a spacer matching the target sites direct dCas9 binding and prevent normal transcription.</P>
We designed an experiment where, with just the reporter plasmid, dCas9, tracrRNA, and crRNA matching the target array, we could validate the approach. We designed a proof of concept fluorescent reporter plasmid that allows easy readout of spacer acquisition and designed a crRNA array that we could pre-load with spacers. By testing whether the addition of a spacer matching the target site reduced expression of a GFP reporter we could lay the foundations for more elaborate signaling cascades.<P>In the last phase of our project we concluded that this approach has some disadvantages. Since the immunity of our detection cells is far from perfect, a significant portion of our population will also show a decrease in fluorescence due to phage attacks or other influences. How do we distinguish cell death from an actual GFP inhibition? 
We resorted to using a <I>L. lactis</I> strain with an RFP integrated into its genome driven by a constitutive promoter. In their native, non-infected state, the cells will emit both GFP &amp; RFP fluorescence.  If the cells die, due to bacteriophage attack or other factors, both GFP &amp; RFP fluorescence will decrease proportionally. If the correct spacer gets adapted, only GFP will decrease while RFP fluorescence will be maintained. The wrong spacer will possibly lead to survival but not a decrease in fluorescence of either GFP or RFP, so it will not be registered as a signal. To explore how this will work on a population level, please refer to our <A href="https://2017.igem.org/Team:Groningen/Model#snavcompart">modeling page</A>.
</P><H5 id="snavtoolbox"> Lactis Toolbox </H5><P class="left">A fourth sub-project that is not directly involved with the project is the lactis-toolbox. Since we want to provide a suitable product for the dairy industry we require incorporation of all parts into <I>L. lactis</I> chassis. However, parts that work in <I>E. coli</I> or <I>B. subtilis</I> rarely work in <I>L. lactis</I>, promotors often need to be interchanged which requires specific cloning protocols. Although several teams have tried working in <I>L. lactis </I>before us, not a lot were successful and limited information and parts are available. Since <I>L. lactis </I>is an important food-grade bacterium we would like to improve upon this by submitting three distinctive promotors and sharing our experience as well as our protocols.</P><P class="left">The first promoter is pNisA, a nisin inducible promotor, which is one of the most widely used expression systems in Gram-positive bacteria <A class="mouseover" title="Mierau, I. et al. Optimization of the Lactococcus lactis nisin-controlled gene expression system NICE for industrial applications. Microbial cell factories  4 (16) (2005)">[Mierau (2005)]</A>. It was the first characterized <I>L. lactis</I> NZ-strains in 1996 and became a fundamental part of the <B>ni</B>sin <B>c</B>ontrolled <B>e</B>xpression system (NICE)[<A class="mouseover" title="de Ruyter, P.G. et al. Controlled gene expression systems for Lactococcus lactis with the food-grade inducer nisin. Journal of Bacteriology,  178 (12), 3662–3667. (1996)">de Ruyter (1996)</A>, <A class="mouseover" title="Kuipers, O.P. et al. Quorum sensing-controlled gene expression in lactic acid bacteria.  Journal of Biotechnology, 64 (1), 15–21. (1998)">[Kuipers (1998)]</A>]. 
The second promoter is p32, this is a constitutive promoter first characterized in 1987 from <I>Streptococcus cremoris</I><A class="mouseover" title="van der VOSSEN, J.M.B.M. et al. Isolation and characterization of Streptococcus cremoris Wg2-specific Promotors. Applied and environmental microbiology 5 (10) 2452-2457. (1987)">[van der Vossen (1987)]</A>. The last promoter is pUsp45, a promoter from <I>L. lactis</I>, first described in 1990 <A class="mouseover" title="van Asseldonk, M. et al. Cloning of usp45, a gene encoding a secreted protein from Lactococcus lactis subsp. lactis MG1363. Gene 95 (1),  155-160 (1990)">[van Asseldonk (1990)]</A>. This promoter has been characterized in the past using sfGFP(Bs) in <I>L. lactis</I><A class="mouseover" title="Overkamp, W. et al. Benchmarking various green fluorescent protein variants in Bacillus subtilis, Streptococcus pneumoniae, and Lactococcus lactis for live cell imaging. Applied Environmental Microbiology 79(20), 6481-90.(2013)">[Overkamp (2016)]</A>.  
</P><P class="left">For this sub-project we would like to thank our supervisor Patricia Alvarez Sieiro in particular since she helped us a lot with the <I>L. lactis</I> work we performed and provided us with our protocols.  </P><H5 id="snavreferences">References</H5><OL><LI style="text-align: left">van Asseldonk, M. et al. Cloning of usp45, a gene encoding a secreted protein from Lactococcus lactis subsp. lactis MG1363. Gene 95 (1),  155-160 (1990)</LI><LI style="text-align: left">Doudna Lab. CRISPR systems in prokaryotic immunity. Viewed at 24-10-2017  13:14 via http://rna.berkeley.edu/crispr.html (2012)</LI><LI style="text-align: left">Heler, R. et al. Mutations in Cas9 Enhance the Rate of Acquisition of Viral Spacer Sequences during the CRISPR-Cas Immune Response. Mol. Cell 65, 168–175 (2017).</LI><LI style="text-align: left">Kleinstiver, B.P. et al. Engineered CRISPR-Cas9 nucleases with altered PAM specificities. Nature 523, 481–485 (2015).</LI><LI style="text-align: left">Kuipers, O.P. et al. Quorum sensing-controlled gene expression in lactic acid bacteria.  Journal of Biotechnology, 64 (1), 15–21. (1998) </LI><LI style="text-align: left">Larson et al. CRISPR interference (CRISPRi) for sequence-specific control of gene expression. Nature Protocols, 8(11), pp.2180-2196. (2013).</LI><LI style="text-align: left">Mierau, I. et al. Optimization of the Lactococcus lactis nisin-controlled gene expression system NICE for industrial applications. Microbial cell factories  4 (16) (2005)</LI><LI style="text-align: left">Overkamp, W. et al. Benchmarking various green fluorescent protein variants in Bacillus subtilis, Streptococcus pneumoniae, and Lactococcus lactis for live cell imaging. Applied Environmental Microbiology 79(20), 6481-90.(2013) </LI><LI style="text-align: left">de Ruyter, P.G. et al. Controlled gene expression systems for Lactococcus lactis with the food-grade inducer nisin. Journal of Bacteriology,  178 (12), 3662–3667. (1996)</LI><LI style="text-align: left">Sternberg, H. et al. DNA interrogation by the CRISPR RNA-guided endonuclease Cas9. Nature 507, 62-67 (2014)</LI><LI style="text-align: left">van der VOSSEN, J.M.B.M. et al. Isolation and characterization of Streptococcus cremoris Wg2-specific Promotors. Applied and environmental microbiology 5 (10) 2452-2457. (1987)</LI><LI style="text-align: left">Wei, Y. et al. Cas9 function and host genome sampling in Type II-A CRISPR–Cas adaptation. Genes &amp; Development, 29(4), 356–361.(2015)</LI></OL><CENTER><A class="button" style="text-decoration: none; color:white;" href="https://2017.igem.org/Team:Groningen/Model">Next: Model</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>