<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_FutureResearch skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/FutureResearch</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="primary_nav_wrap" style="z-index: 2000"><UL><LI class="mobicon"><A href="javascript:void(0);" onclick="mobnavtoggle()">≡ Menu</A></LI><LI><A href="https://2017.igem.org/Team:Groningen">Home</A></LI><LI><A onclick="">Project</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/FutureResearch">Future Research</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/InterLab">InterLab</A></LI></UL><LI><A onclick="">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Overview">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/HP/Silver">Human Practices (Silver)</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/HP/Gold_Integrated">Integrated (Gold)</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/FutureOutlook">Future Outlook</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A onclick="">Team</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Team">Team Members</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Achievements">Achievements</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Attributions">Attributions</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Fun">Fun Activities</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Sponsors">Sponsors</A></LI></DIV><DIV class="marginsnav"><DIV class="main-col scrollcol"><NAV id="myScrollspy"><UL class="nav"><LI class="active"><A href="#snavlactis">Lactococcus lactis</A></LI><LI><A href="#snavantibio">Antibiotic Resistance</A></LI><LI><A href="#snavsecure">Security</A></LI><LI><A href="#snavnovel">Novel phages</A></LI><LI><A href="#snavmicro">Microfluidics</A></LI></UL></NAV><H1>Future Research</H1><P>We hope you have enjoyed reading about our journey developing the IMPACT system.  An efficient manner to detect bacteriophages or environmental DNA would offer huge benefits not only to the dairy industry but also, more broadly, to fermentation industries worldwide.
The ride has certainly not been without a few bumps and hiccups and here we would like to lay out what our plans are for further maturing the system.  
</P><H5 id="snavlactis"><I>Lactococcus lactis</I></H5> 
   Getting our system fully functional in <I>Lactococcus lactis</I> would require all constructs (see <A href="https://2017.igem.org/Team:Groningen/Design">our design page</A>) to be available and transformed into <I>L. lactis</I>. Besides getting all the plasmids in the cell, we expect some amount of tweaking to be required for the system to be operational. 
Plasmid copy numbers, promoters strength, and expression profiles will likely have to be adjusted to account for the toxicity of high levels of nuclease active Cas9 expression. An inducible promoter, such as <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2361009">pNis</A>, would make it easier to adjust the expression levels for the Cas9 proteins.
 
   <H5 id="snavantibio">Antibiotic resistance</H5> 
   The current system is spread over multiple plasmids which may severely impact cell viability due to the metabolic burden of expressing multiple antibiotic resistance genes (see <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4380920/">Hall et al.</A>) To avoid this, and reduce the number of resistance genes required for plasmid maintenance, we envision genomically integrating dCas9, the hCas9 operon, spacer array, and tracrRNA. This would eliminate the need for multiple antibiotic resistance genes while maintaining the modular and interchangeable nature of the detection device through the reporter plasmid.
  
   <H5 id="snavsecure">Security</H5> 
   To further strengthen the containment of our IMPACT bacteria we would like to metabolically compromise our strain. Making the strain dependent on supplementation from the media, for instance thymine or thymidine, would prevent our bacteria from escaping to the main production line. This system has been demonstrated by <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC184577/pdf/aem00088-0192.pdf">Ross et al.</A><H5 id="snavnovel">Novel Phages</H5> 
   While the detection of known phages is an important issue, detection of novel bacteriophages could also be achieved with our system. This would “only” require the hCas9 array to be sequenced and the corresponding sequence to be compared to known phage sequences and <I>L. lactis</I> DNA. Since the spacers are relatively short, identification would only be feasible by combining the results of multiple spacers. Even if only a few unknown spacers are sequenced valuable insights could still be gained when more genomic information on bacteriophages becomes available in the future. 
  
   <H5 id="snavmicro">Microfluidics</H5> 
   In its current design our system offers the possibility of detecting a limited set of pre-designed DNA sequences. To further upscale the detection and provide expanded classification an improved cartridge design with multiple detection chassis could be envisioned. Several <I>L. lactis</I> cells with different pre-designed reporter target sequences would have to be constructed and spatially separated in a microfluidic cartridge. For example, if cell A, D, and G light up, this could mean that it is most likely a 936 phage, while another set of cells indicates a different species. This scaling up should also result in a decrease in cost as the same system can be used in many applications and can be supported by bioinformatic readout, automatization (array plate), as well as an increase in sensitivity.
<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3112185/">Chung et al.</A> demonstrated an efficient manner of fixing single cells to an array but a significant amount of work still has to be invested to figure out how to ensure that composition of different cells on the array fits the envisioned one. 
 
<P>To summarize, we believe that IMPACT has a great potential that can be realized with further development. We hope that you, like us, see the innovative nature of this technology and the far-ranging applications of it, even outside the dairy industry.</P><CENTER><A class="button" style="text-decoration: none; color:white; margin-bottom:30px;" href="https://2017.igem.org/Team:Groningen/Protocols">Next: Protocols</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>