Team:Groningen/Results
≡ Menu
Home
Project
Description
Model
Design
Results
Notebook
Future Research
Protocols
InterLab
Human Practices
Overview
Human Practices (Silver)
Integrated (Gold)
Public Engagement
Applied Design
Future Outlook
Safety
Parts
Collaborations
Team
Team Members
Achievements
Attributions
Fun Activities
Sponsors
Project Achievements
dCas9
dCas9 VRER
CRISPR arrays
Lactis Toolbox
Spacer acquisition
Results
In the end, we were able to validate three of our parts:
dCas9 BBa_K2361000
dCas9VRER BBa_K2361001
pNisA promoter BBa_K2361009 .
We submitted sequence verified pUSP45 tracrRNA BBa_K2361003
a CRISPR array BBa_K2361004
a CRISPR array containing a 20 spacer BBa_K2361005
a CRISPR array containing a 21 spacer BBa_K2361006
a CRISPR array containing empty spacers BBa_K2361007 without further experimental validation
Unfortunately, sequencing revealed unsuccessful incorporation of the insert during the biobrick construction of two of our parts: p32 BBa_K2361008 and pUsp45 BBa_K2361010 .
Unfortunately, sequencing revealed that the one of the fragments in the CRISPR operon was replaced with something of approximately the same size. Therefore, we were not able to submit our hCas9 part BBa_K2361002 .
dCas9
Construction
For this part we started by making a biobrick compatible version of dCas9 called pSB1C3:dCas9QC. This part was also used for the construction of dCas9VRER. The full length of this part was sequenced (figure 2) and the results confirmed that the sequence was correct. In the notebook (week 18-09) a gel is shown from which it can be seen that the EcoRI site was successfully removed from our dCas9 part.
Although the pSB1C3:dCas9QC was already in the pSB1C3 backbone and contained all the correct restriction sites, it did not contain the suffix. To fix this we restricted the pSB1C3dCas9 with EcoRI and SpeI and ligated it into pSB1C3, which was linearized with primers G69 and G70 (see notebook) and restricted with the same enzymes. In the gel on the top right, lane 2 & 3 correspond to the isolated plasmid restricted with respectively EcorI and a combination of EcorI and PstI. The sizes correspond to the linearized (6,2 kb) and the separate backbone (2 kb) and dCas9 part (4,2 kb).
Since the chances of mutations in restriction-ligation cloning are negligible we did not fully sequence the part again. We did use pJet_Fwd and pJet-Rev to be absolutely certain that the correct part was cloned into the backbone. These results are shown in figure 3.
This part is considered as an improvement of a previously submitted dCas9 BBa_K1026001 since our part has been made biobrick compatible. Also, we sequence confirmed and experimentally validated our part, which was not done by the 2016 Warwick team, who also improved this part.
Experimental validation
See dCas9VRER  below
dCas9 VRER
Construction
This part was made from the pSB1C3:dCas9QC plasmid (see dCas9) which was completely sequenced. We replaced the final part of the dCas9QC with a gBlock containing the VRER mutations. The new construct was analyzed by sequencing the last part containing the gblock (figure 4). Also, a restriction analysis with EcoRI and PstI was performed (see the gel in dCas9 lane 4,figure 1).
Based on the validation (see below) we cannot conclude yet whether the VRER mutations have the desired effect. If we would have mixed up the
Experimental validation
This part was validated experimentally in the same experiment as dCas9. For this experiment, we have designed four gRNA's that target GFP. Two of this gRNA's bind to sequences flanked by NGG (1&2) and two bind to a sequence flanked by NGCG (3&4). These gRNA's were inserted into the plasmid pSB3C5 containing a pLacGFP construct as well. For dCas9 and dCas9VRER expression those parts were put into a pBad vector in which they are expressed behind an Arabinose inducible promoter.
In total 18 E. coli strains were produced containing pBad:dCas9, pBad:dCas9VRER or no pBad combined with pS3C5 with one of the gRNA's, pSB3C5 without a gRNA or no pSB3C5 (Table below).
min
dCas9
dCas9VRER
dH5α
dCas9:nopSB3C5
dCas9VRER
noPbad:nogRNA
dCas9:nogRNA
VRER:nogRNA
noPbad:gRNA1
dCas9:gRNA1
VRER:gRNA1
noPbad:gRNA2
dCas9:gRNA2
VRER:gRNA2
noPbad:gRNA3
dCas9:gRNA3
VRER:gRNA3
noPbad:gRNA4
dCas9:gRNA4
VRER:gRNA4
All strains were grown overnight and the next day they were diluted to an OD of 0.05 in the afternoon. At the end of the afternoon at an OD of 0.4-0.6, all cultures were induced with a final concentration of 0.01% arabinose. All induced cultures were put back into the incubator (37C 220 rpm) and grown overnight. The next morning, the OD of all cultures was measured and diluted to an OD of 0.2 in LB (in final volume of 1mL). The OD and GFP (470/510) of all samples were measured in a plate reader in quadruplicate.
The data obtained from the fluorescence measurement were converted to relative fluorescence and are shown in figure 5. To convert the fluorescence to relative fluorescence we first divided all measured fluorescence values and divided them by the measured OD's. Next the fluorescence/OD of the negative controls (samples without pSB1C3) was subtracted from the other samples. Next, all normalized fluorescence/OD values were divided by that of the positive control (no pBad:pSB3C5 without gRNA).
From figure 5 it can be seen that all fluorescence values are quite similar for the samples without gRNA. Further we can see a clear decrease in the relative fluorescence of dCas9 in combination with gRNA 2 and 3. For gRNA's 1 and 4 the relative fluorescence of the dCas9 is similar to that of the samples without the pBad vector. For dCas9VRER we showed CRISPR interference in combination with gRNA's 1,2, and 4.
From the data we cannot conclude that the VRER mutations had any effect on the PAM preference since dCas9VRER shows the strongest repression for gRNA 1 and 4 of which one has a NGG flanked target. Also, gRNA 3 whose target is flanked by NGCG seems to be repressed by dCas9 and not by dCas9 VRER. If these results are correct this would suggest that the PAM preference has not changed for dCas9VRER, however, it is quite odd that the differences between dCas9 and dCas9VRER are so big for especially gRNA 1 and gRNA4. Another explanation could be that we mixed up gRNA1 and gRNA3 in one of the steps from gBlock to double transformants. If we would switch around these data (so gRNA1-> gRNA3 and gRNA3-> gRNA1) the data would show that only dCas9VRER can be directed towards NGCG flanked targets. Unfortunately, we could not analyze the sequences of the pSB3C5 plasmids of the cultures used in this experiment before the WIKI-freeze.
Considerations for replicating the experiments
Besides sequencing the cultures used in the experiment, performing more experiments with biological replicates would increase the fidelity of the data. Also, it would be nice to design more efficient gRNA's that bind between the -35 and -10 region of the promoter. This way we can compare the efficiency of dCas9 and dCas9VRER, since this is the optimal place to target a gRNA.
CRISPR arrays
Construction
The construction of these parts all succeeded in terms of biobrick formation. As planned, the array itself was implemented into the backbone, after which the individual spacers were implemented to create the different derivatives of the array. The 20 and 21 derivatives were to be found correct after the first sequencing analysis. However, a second round of cloning was required to obtain the empty derivative of the CRISPR array.
Future plans for the project
These different derivatives create a situation in which this system can be used to detect the specific spacers but also re-instate the "normal" system by having the empty array. This allows for using the system to obtain new spacers. However, without pre-programming the reporter with complementary spacers, there won't be a change in fluorescence. Future experiments could be incorporating this with the tracrRNA production and/or a signal sequence. Ultimately, this would also help towards incorporating all the designed elements of this project into a single organism.
Lactis toolbox
With the lactis toolbox, we hope to increase the possibilities for iGEM teams to work with L. lactis in the future. Beside the physical additions of the new promoter, we also tried to obtain essential information on working with such an organism in situations with little experience. Most of our student team members did not have experience with it and although it is a bacterium, it behaves vastly different then a model organism like E. coli.
Construction
The first addition of this lactis toolbox is the new nisin inducible promoter in a biobrick format. The construction of this went quite fast, as a positive colony was quickly identified and the first round of sequencing analysis confirmed appropriate incorporation of the promoter. However, from there the work got a lot harder, as it involved working with the actual organism instead of L. lactis compatible DNA. There was quite a steep learning curve in techniques like transforming L. lactis and obtaining enough DNA for this procedure. This was all quite different from what we were used to. In our experience knowing all the little details of the protocol really had a large impact on our results. Even after all the practice we still had a low success rate in these transformations.
Experimental validation
Nevertheless, we obtained the biobrick for the nisin inducible promoter and we were able to validate this part by constructing an expression vector based on the native vector where the part was amplified from: pNZ8048. This native lactococcal plasmid was used as backbone to integrate the coding sequence for GFP behind the promoter. This enabled us to obtain data on its inducible character. This was done by inducing the cells with various amounts of nisin once the cells were in the exponential growth phase (OD600: 0.5-0.6). The graph below shows the expression of GFP over a time period of 6 hours. The values shown are a ratio of GFP emission over the absorbance at 600 nm to correct for varying growth rates.
This graph shows that the best expression of GFP was achieved with a nisin concentration of 45 ng/µl. Besides that the concentrations of 10 and 80 ng/µl seem to work sufficient, which suggests that there is a wide margin in the amount of nisin which should be added. The only risk with this are concentrations on the high end of the spectrum, like the 80ng/µl condition. During the experiments to obtain this data, the growth of the highest 80ng/µl condition seemed limited. This is in agreement with literature, as nisin is known for its anti-microbial properties in higher concentrations [Zhou (2016)] . Nevertheless, this shows the usability of this promoter in conjunction with the pNZ8048 expression vector.
Considerations for replicating the experiments
Besides these efforts, we also attempted to characterize two popular promoters from the registry in L. lactis. We successfully combined these parts with a RBS and GFP/RFP. The final step, which was ligating it together with a terminator into pIL252 was probably also successfully performed. However, we were unable to confirm this due to problems with the plasmid isolation from L. lactis.
Although we did not measure any fluorescence in our experiment we cannot be certain that this is, because the plasmid might have not been transformed successfully into L. lactis. Moreover, we learned later that measuring GFP in the regular growth medium of L. lactis does not work properly due too high background fluorescence of the medium. If you want to measure GFP you need to use a chemically defined medium as was done in the experiment described above.  We did not pursue getting data for these promoters intensify since the chances of them working in lactis are not very high. Therefore, we decided that our time was better spend developing new promoter parts and add them to the registry.
We hope that this toolbox and its information is a helpful step towards the better use of the L. lactis bacterium in the iGEM future.
Spacer acquisition
Construction
We planned to make this part from a combination of synthetic DNA(gblocks) and segments of the S. Pyogenes Cas9 operon from the plasmid pWJ40. We obtained these part by PCR reactions with the pWJ40 plasmid as template. The gblocks were used to remove the prohibited iGEM restriction sites and introduce the hCas9 mutation, I473F. All the parts were put together using gibson assembly. After transforming the assembly product, we performed several colony PCR's on the resulting colonies. We did this with the same primers that were used to construct all the separate fragments for the gibson assembly (see left gel). Based on the gel we selected two colonies for which we were most confident that they contained the correct construct. After sequencing it turned out that the I473F gblock fragment of the gene contained the wrong sequence (see figure below). Unfortunately, we did not have enough time to repeat the construction of the biobrick compatible hCas9 operon.
Besides the biobrick compatible hCas9 operon, we also wanted to create the tracrRNA. We created a tracrRNA under the constitutive pUSP45 promoter, which is also submitted as one of our parts BBA_K2361003 . We first checked it on gel (see right gel) before it was sent for sequencing for verification.
Considerations for replicating the experiments
To repeat the experiment, we would suggest to adopt a different strategy. The segment of the gene for which the I473F gblock coded could also be obtained from the pRH180 plasmid, which already contains the I473F mutation. This can be done in combination with the unaltered parts of the hCas9 operon gene. So, a gibson assembly with only two fragments would have to be performed. The prohibited EcoRI site could be removed with quick-change PCR.
Future plans for the project
First a working version of the hCas9 operon has to be created. To experimentally experimentally validate that the hCas9 operon an on plate phage assay could be performed. The pUSP45 tracrRNA could be experimentally validated in the same way, either with our own hCas9 operon or with another Cas9 operon.
Next: Notebook
