Team:Groningen/Notebook
≡ Menu
Home
Project
Description
Model
Design
Results
Notebook
Future Research
Protocols
InterLab
Human Practices
Overview
Human Practices (Silver)
Integrated (Gold)
Public Engagement
Applied Design
Future Outlook
Safety
Parts
Collaborations
Team
Team Members
Achievements
Attributions
Fun Activities
Sponsors
June ▼
June ▲
Week 23
Week 24
Week 25
Week 26
July ▼
July ▲
Week 27
Week 28
Week 29
Week 30
Week 31
August ▼
August ▲
Week 32
Week 33
Week 34
Week 35
September ▼
September ▲
Week 36
Week 37
Week 38
Week 39
October ▼
October ▲
Week 40
Week 41
Week 42
Week 43
Week 44
Primer Table
Notebook
Here you will find our notebook, in which we documented our progress during our iGEM project.
June
Week 23
Although we obtained the keys of our lab on the first of June we have not yet performed any real experiments. The entire week was spent on setting up the lab, e.g. getting all materials and machines needed. Also we made some media and standard buffers and we have produced our first batch of competent E. coli DH5α at the end of this week.
Week 24
General
This week we could finally start doing some real experiments in the lab! We started with optimizing the heat-shock time in the transformation protocol. We did this by transforming our competent cells with DNA from the competent cell test kit with various heat shock times (0-60 seconds). We found that the highest efficiency was obtained when using a heat shock of 30 seconds.
Interlab
Because we did not have fully finished the design of any of the experiments for our own project yet, we started with the Interlab study. As a result we can proudly say that we were the first team to complete the Interlab project this year!!
We have successfully transformed all test devices of the InterLab studies. Of each of the transformation plates one colony was picked, which was inoculated and grown overnight. After plasmid isolation of the overnight cultures restriction analysis with EcoRI showed that all colonies contained the correct plasmids. In the remainder of the week we have completed the measurements according the plate reader protocol. More information about the InterLab studies can be found on our InterLab page .
This and all subsequent experiments requiring plate reading/GFP/RFP were performed with Synergy Mx .
Week 25
General
We have inoculated one of the red colonies from the heatshock-test and isolated the plasmid (pSB1C3:bba_J04450) from it. This plasmid was used to confirm that the restriction enzymes, which were left over from the previous iGEM team of Groningen, worked properly. Moreover this plasmid was also used for cloning parts into the pSB1C3 backbone.
Lactis toolbox
After completing the InterLab we have worked on the lactis-toolbox. We started with two different parts of this sub-project. First was the preparation of electrocompetent L. lactis NZ9000 cells and optimization of the transformation protocols for it. The second step was the preparation of a pTet-GFP/RFP and a pLac-GFP/RFP construct to test these promoters in L. lactis. From next week progress for this part of the project will be shown under the header ‘promoter testing’. This week we produced our first batch of competent lactis cells and determined a growth curve for NZ9000 in SGM17+glycine medium (see figure below). For the second part we successfully transformed E. coli with pLac, pTet, sfGFP and the RBS (see design page for BBa numbers).
Week 26
Spacer acquisition
We obtained the gDNA of a clinical isolate of S. pyogenes from J.M. van Dijl. We tried to PCR out the entire CRISPR operon at once using primers G1 & G4 and this failed. We have tried the PCR several times including with a temperature gradient PCR. But since all attempts failed we think this isolate may have not contained the CRISPR system. The problem was solved when the plasmids of the Heler lab arrived (see next week).
dCas9
We have tried to transform our E. coli with the dCas9-Omega fusion protein several times, however all attempts failed until we had no DNA left over. To solve this problem we determined to use the pJWV-102-PL-dCas9 and make the dCas9 biobrick compatible ourselves.
Lactis toolbox
We have obtained five L. lactis vectors from our supervisor H. Karsens in different host strains (see table below). The cultures were grown overnight and the plasmids were isolated successfully.
Plasmid
Host strain
Growth medium
Size (linearized)
pNZ8048
L. lactis NZ9000
M17 + Clm
3.3 kb
pMG36E
L. lactis IL1403
M17 + Ert
3.3 kb
pMG36C (low copy)
E. coli Mg1363
LB + Clm
3.3 kb
pIL252 (low copy)
L. lactis IL1403
M17 +Ert
4.6 kb
pIL253 (High copy)
L. lactis NZ9000
M17 + Ert
4.9 kb
July
Week 27
Spacer acquisition
This week the Heler plasmids arrived so we could start with creating the different DNA fragments for the gibson assembly of the hCas9 operon. We started with PCR amplifying the DNA fragments from the pWJ40 plasmid using the primer pairs G19, G20 and G23, G24. The resulting PCR products were treated with DpnI and cleaned with a PCR clean up kit.
Lactis toolbox
The competent lactis cells that were produced in week 25, were tested by transforming them with the isolated pNZ8048 plasmid. The transformation did not work out, probably because the Chloramphenicol concentration in the plates was too high (25ug/ml). Also the pNZ8048 plasmid was linearized and the biobrick prefix and suffix were incorporated. The PCR was performed using Phusion polymerase and primers G7 & G8.
The sfGFP was cloned into the linearized vector using restriction ligation with EcoRI and PstI. The linearized pNZ8048 was not digested with DpnI, because L. lactis plasmid DNA is not methylated and therefore DpnI cannot digest the template. The ligation product was stored in the fridge.
Promotor testing
For this part we successfully transformed and isolated the RFP plasmid for the BBa number see the design page. Also we completed the first round of 3A assemblies, as shown in the table below.
Backbone
construct A
construct B
Product name
pSB1T3
pLac
sfGFP
pLac_GFP
pSB1T3
pLac
RFP
pLac_RFP
pSB1T3
pTet
RBS
pTet_RBS
The plates for pLac_GFP and pLac_RFP contained four colonies and the pTet_RBS plate contained only one colonies. All colonies were heavily overgrown since they were left at 37C for the entire weekend.
Week 28
General
We finished the first part of the design of the dCas9 and the spacer acquisition projects and we ordered the second batch of primers G15 & G37. Further we made new competent E. coli DH5α and L. lactis cells. gBlocks and Primers for the reporter project were ordered as well.
dCas9
From our supervisor C. Huang we received E.coli DH5α containing pJWV-102-PL-dCas9. These cells were plated and one colony was picked and inoculated. The plasmid was isolated from the overnight cultures and analysed by restriction analysis with PstI (see gel next week).
Promotor testing
The colonies of the 3A assemblies that were heavily overgrown were re-plated. Each plate contained many colonies and those on the pLac-GFP plate were coloured green. Restriction analysis of isolated plasmids from one of each of the colonies showed the correct size for pLac_GFP and pTet_RBS, but not for pLac_RFP. Since we only wanted to use the constructs for the function of pLac in L. lactis and we only need one marker for this we decided not to retry the pLac_RFP assembly.
Week 29
General
This week the second batch of primers arrived, so we could finally continue our work in the dCas9 and the spacer acquisition projects.
Spacer acquisition
This week the gBlocks arrived. The I473F/EcorI gBlock was amplified using the primer pair G21, G22. The other gBlock contained both the tracrRNA and a part of the hCas9 operon. It was first inserted into pSB1C3 by restriction ligation with EcorI and PstI and transformed into E. coli DH5α. We planned to seperate the tracrRNA from the hCas9 operon when we linearized the plasmid using PCR. With the primer pair G27, G28 we wanted to create a linearized tracrRNA pSB1C3 vector and with primer pair G17, G18 we wanted to create a linearized hCas9 vector, which would be the backbone for the gibson assembly.
dCas9
The isolated pJWV-102-PL-dCas9 was used to PCR out the dCas9 gene using primers G35 & G36. Since it is a large size construct (4,2 kb) we used Phusion polymerase. At the moment we thought that we had the correct PCR product, because we did not analyse the gel carefully enough. In the figure below the PCR fragment is shown together with the restricted pJWV-102-PL-dCas9 and of pSB1C3:dCas9 (see next week). From the gel it can be seen that the actual product sizes on the left are larger than they should be. On the right side of the gel the same products are shown, but this time with the correct plasmid.
Week 30
Spacer acquisition
Four colonies of the gBlock transformations of last week were checked on gel using a colony PCR with primers G15 & G16. The results are shown in on the left side of the gel image below. For the next weeks we have been trouble shooting the PCR for gBlock separation of hCas9:tracrRNA combi-gBlock with different annealing temperatures and addition of DMSO to the PCR mix.
dCas9
As said in the report of last week the PCR fragment was cloned into pSB1C3:GFP via restriction ligation with XbaI and PstI. The transformation resulted in four colonies of which the plasmids were isolated and analyzed by restriction with PstI and EcorI (figure below, right half). Colonies 2 and 3 seemed to have bands of the right size and those plasmids were used for follow-up experiments. However as explained in the previous report this dCas9 turned out to be wrong, so all other experiments using this part failed.
Week 31
General
gBlock dCas9 VRER and hCas9I473F were ligated into linearized pJET1.2 blunt end vector via blunt end ligation. The pJET vector was a gift from our supervisor C. Pohl, who also gave us two primers pJet_Fwd and pJet_Rev which bind on both the pJet backbone and the pSB1C3 backbone. These primers were used for sequencing and to confirm whether a fragment was inserted into pSB1C3.
Lactis toolbox
Transformation of lactis with pNZ8048. The transformation product was plated on plates containing different concentrations of chloramphenicol (0-15 ug/ml). 5 ug/ml was found to be the best concentration. Moreover we linearized the pIL252 and pIL 253 backbones using Phusion polymerase with primers G25 & G26.
Promoter testing
This week we successfully linearized the pNZ8048 and pIL252 and pIL253 vectors with primers pNZ Fw/pNZ Rv and G25 & G26 (for both pIL vectors). Also the pTet-RBS part was successfully fused to GFP and RFP via a 3A assembly. From the resulting transformation colonies, which were coloured green and red, the plasmids were isolated.
August
Week 32
General
This week we successfully transformed E. coli with the pJET vectors containing the gBlocks. The colonies on the plate were analysed by colony PCR with the pJet primers and for both pJET-constructs we isolated the plasmid of the colonies with a positive size of the insert. Also the gBlocks for the construction of the CRISPR array and the reporter construct came in.
Spacer acquisition
After testing even more PCR conditions we finally thought that we succeeded in creating the pSB1C3 containing the tracrRNA part. The PCR which succeeded was a temperature gradient PCR using phusion polymerase, primers G27 & G28.
Reporter
We successfully amplified the target array gBlock and the crArray gBlock using Primers G15 & 16.
Lactis toolbox
To optimize the transformation protocol of L. lactis further we tested different DNA to cell ratio’s. We used either 50 or 100 µl of competent L. lactis cells and transformed them with 50, 100 or 250 ng of pNZ8048. The volume of DNA that was added was respectively 0.5, 1 or 2,5 µl.
The results are shown in the table below. Although we find it strange that using more cells did not result in more colonies the result does show that adding more DNA does not result in more colonies. We think that this is not caused by the amount of DNA that is added, but by the volume of the DNA that is added. Adding a larger volume will dilute the competent cells further, which will lead to a decreased transformation efficiency.
Ratio (cell:DNA)
Colonies
Ratio (cell:DNA)
Colonies
50:50
700-800
100:50
200-250
50:100
100-150
100:100
100
50:250
150-200
100:250
100:150
Promoter testing
This week we developed a double terminator BBa_B0015. Also we performed the final 3A assembly of the pTet_GFP, pTet_RFP and pLac_GFP together with the double terminator into the linearized pIL252 backbone. In this ligation reaction we also ligated GFP and RFP into linearized pNZ8048 using a restriction ligation reaction with EcoRI and PstI. The ligation products were transformed successfully into L. lactis. The colonies were grown and induced with nisin (in case of pNZ8048 constructs) and after several time points the fluorescence was measured using the plate reader. We did not measure any fluorescence, but we had also difficulties with isolating the plasmids from the L. lactis cultures to analyze whether the correct plasmid was inserted. Therefore we cannot exclude the possibility that the measured colonies were false-positives.
Week 33
General
Since the PCR’s for the quickchange PCR of dCas9 and the PCR of the gBlock of hCas9 kept failing we designed and ordered new primers for them. Also we send the isolated pJET-gBlock plasmids and pSB1C3 containing the hCas9 combigblock. Unfortunately, not all samples gave reads of good lengths. Moreover the samples that did gave long reads belonged to the pJet2.1 gBlock constructs, but we could not map them well. Therefore we concluded that the blunt-end ligation didn't work properly.
dCas9
Besides ordering the new quickchange primers we also rechecked our template DNA together with the PCR fragment and the original dCas9 vector. At this point we found out that the dCas9 PCR might be too big.
Reporter
We tried to insert the gBlocks that were amplified last week into pSB1C3 backbone with restriction ligation (EcoRI, PstI). However the transformation did not gave any colonies.
Week 34
Spacer acquisition
The pUSP45 tracrRNA(see week 32) was put into an iGEM backbone and transformed into E. coli DH5α according to protocol. There were colonies visible on the plates and these were used for overnight cultures. The plasmids were isolated and send for sequencing.
dCas9
Since the PCR product of dCas9 seemed larger than the expected size we re-did the PCR to isolate dCas9 from the pJWV-102-PL-dcas9 plasmid. However the product was the same size as the one in the previous experiment.
Reporter
We tried again to put the gBlocks into the pSB1C3 backbone using varying restriction digestion time, enzyme amounts and ligation time but it still didn't work.
Week 35
Spacer acquisition
The sequencing results for the pUSP45 tracrRNA part came back and showed that the plasmids did not contain the pUSP45-tracrRNA gene. Instead it contained a GFP gene. Therefore we designed two new gBlocks, one containing only the pUsp45-tracrRNA and one containing the prefix-suffix for the hCas9 operon.
reporter
This week we made the oligomers to put into the CRISPR array. This was done via a special PCR, since we used only two primers and no template DNA. Primer combinations G41/G42, G41/G43 and G44/G45 were used to respectively produce spacer 20, spacer 21 and the 'empty' insert. Further only 3 cycles were used instead of the usually 25-30 cycles.
September
Week 36
dCas9
We discussed the odd size of the PCR fragment of dCas9 with our supervisor C. Huang. Since she had managed to a correctly sized dCas9 fragment from the plasmid we decided to redo the dCas9 PCR. This time we tested both our own purified pJWV plasmid and the plasmid C. Huang purified from the same strain. Also we tested the primers that she had used in addition to our own. This time we did get a PCR product of the right size using the template of C. Huang and our own primers G35 & G36.
We restricted both the pJWV-102-PL-dCas9 plasmid we miniprepped ourselves as the one we obtained from C. Huang and analysed the results together with the PCR fragments on a gel. From the gel (see below) it can be seen that not only the PCR fragment is bigger but also the original plasmid. Therefore we think that we had bad luck that we picked a colony in which the dCas9 plasmid under went a genetic event.
Week 37
Spacer acquisition
The new pUSP45 tracrRNA gBlock has arrived. We amplified the gBlock as backup with the G15, G16 primer pair using Phusion polymerase. The gBlock was put in the pSB1C3 backbone with restriction ligation using EcorI and PstI and was transformed in E. coli DH5α. The resulting colonies were grown overnight and the plasmids were isolated. The insert of the plasmids were checked on gel after restriction with EcorI and PstI.
dCas9
The new correctly sized dCas9 fragment was ligated into a pSB1C3 backbone via restriction ligation with XbaI and PstI. We successfully transformed E. coli with the ligated plasmid and subsequently isolated the plasmid.
To remove the illegal EcoRI site a PCR with all four combinations of the two sets of quick-change primers was performed G33 & G34 and G39 & G40. All four PCR reactions were analysed on a gel and turned out to be successful. After 15 minutes restriction at 37 C with DpnI 3µl followed by a PCR clean up. 3 µl of this product was directly used to transform E. coli the rest was ligated overnight.
Although the transformation with the ligated dCas9-QC product resulted in more colonies than that with the unligated product, we continued with the latter since already two colonies were grown overnight, the plasmid of the overnight cultures were already isolated and stored as pSB1C3:dCas9QC.
Week 38
Spacer acquisition
The new hCas9 gBlock has arrived. It was first PCR amplified with primer pair G15, G16. The gBlock was also inserted in pSB1A3 with restriction ligation using EcorI and PstI. The resulting colonies were grown overnight and the plasmids were isolated and stored.
dCas9
The isolated dCas9QC plasmids were analysed by restricting them with XbaI + PstI and with EcoRI. As a negative control the old pSB1C3:dCas9 plasmid was restricted with the same enzymes. For the first combination all samples have the same pattern (2kb + 4,2 kb bands), but for the EcorI restriction the pSB1C3:dCas9 contained two bands (4,8 kb + 1,4 kB) whereas all quickchange samples had only one band (6,2 kb). This result is as expected since the part should contain one EcoRI site in the prefix.
In order to produce dCas9VRER we restricted the amplified gBlock dCas9VRER and pSB1C3:dCas9QC with BamHI and PstI. The correct band (5,5 kb) of the plasmid digestion was extracted from the gel and the gBlock restriction was inactivated by PCR purification. Since the concentrations of the both purified products were very low the ligation and following transformation were unsuccessful.
Reporter
We finally were able to insert the amplified gBlocks into the Backbone after PCR purifying the products as well as after restriction multiple PCR products were pooled together to elevate the concentration. Ligation occurred overnight. The results were verified by colony PCR with the pJET primers.
Week 39
Spacer acquisition
The hCas9 gBlock pSB1A3 plasmid had to be linearized before it could be used in the gibson assembly. Since all the DNA fragments for the gibson assembly need a specific overhang, we decided to linearize with PCR. We did this with the primer pair G17, G18 with NEB high fidelity polymerase. The resulting product was treated with DpnI and cleaned with a PCR clean up kit.
dCas9
After a few attempts to construct dCas9VRER in the way described last week, which all failed we decided to clone the gBlock into the pJET1.2 vector using blunt end cloning. This was done to amplify the gBlock and to increase the efficiency of the restriction by increasing the length of the overhangs (which is now the entire backbone).
October
Week 40
Spacer acquisition
This week we performed the gibson assembly to create the biobrick compatible hCas9 operon. To get the hCas9 operon we had to put four parts together: the two unaltered PCR fragments from week 3-07, the amplified gBlock I473F from week 17-07 and the linearized backbone from last week. Since we had more than 3 fragments we wanted to fuse we had to put all the fragments together in equimolar amounts. The gibson assembly product was transformed in E. coli DH5α. We used a colony PCR to determine if the resulting colonies contained the desired product. We used several combinations of primers: G19 & G20, G21 & G22, G57 & G60 and G16 & G59. Two colonies contained 3 out of 4 expected fragments. These were inoculated and grown overnight.
dCas9
The plasmids pSB1C3:dCas9QC and pJET:gBlockdCas9VRER were both send for sequencing. For the first plasmid, we used primers G51 & G56 as well as pJet_Fwd and pJet_Rev. The results of sequencing came back positive and are discussed further in the Result section.
After several restriction and ligation attempts using the pSB1C3:dCas9QC and pJET:gBlockdCas9VRER plasmids we finally successfully transformed E. coli with pSB1C3dCas9VRER this week! From the plate, 7 colonies were picked, grown and mini-prepped. Restriction analysis of the isolated plasmids with PstI showed that all colonies contained a plasmid of the expected size (6,2 kb).
Reporter
We succesfully PCR'ed out the GFP fragment from pSB1C3:GFP and restricted it with BsaI and DpnI. The pSB1C3:target-array, which was isolated two weeks ago, was restricted with HindIII. The two fragments were ligated and transformed, but the transformation resulted in a lot of colonies. But also the negative plate with only restricted pSB1C3:gBlock contained a lot of colonies. This was because it could easily self-ligate. To prevent this in future experiments we first tried to keep HindIII active during the ligation to restrict all false-positive plasmids.   The pSB1C3:CRISPRarray was restricted with BsaI and the oligomeric PCR products were ligated into it. The resulting plasmids were successfully transformed into E. coli.
Week 41
dCas9
To confirm that the pSB1C3:dCas9VRER contained the right insert we sequenced one of the plasmids with pJet_Fwd and pJet_Rev. The results came back positive and the sequenced sample was stored separately and it was in the end submitted into the registry. The rest of the week we designed the gRNA gBlocks and primers for the dCas9 validation experiments. These were ordered at the start of this week and arrived in the start of next week. Also we obtained a pBad vector from our supervisor Patricia and pSB3C5:pLacGFP from our supervisor Harma.
Spacer acquisition
The biobrick compatible hCas9 operon was inserted into the pSB1A3 vector. Since we wanted to submit it as a biobrick, we had to exchange the backbone for the pSB1C3 backbone. To do this we performed a restriction ligation reaction with EcorI and PstI and transformed in E. coli DH5α. The resulting colonies were grown overnight and the plasmids were isolated. The resulting plasmid was sent for sequencing. We  digested the target array(reporter) with HindIII and the crArray with BsaI. Their respective inserts (5N5, rest of GFP & 20,21, empty,  all digested will BsaI) were added after heat inactivation. All crArrays were sent for sequencing. Construction of and analysis of potential positive colonies of tracrRNA incorporated in front of the array.
Reporter
Analyse potential correct plasmid DNA containing pSB1C3 and the whole reporter sequence. Both restriction analysis and colony pcr. Analyse Target backbone because of negative results of colony pcr.
Collaboration
We performed the analysis of the Nottingham cells as stated on the collaborations page .
Week 42
dCas9
After the primers arrived we linearized the pBad vector and the pSB3C5:pLacGFP using respectively primers G73 & G74 and G16 & G77. dCas9QC and dCas9VRER were isolated from their backbones using primers G71 & G72.
Using Gibbson cloning we successfully inserted dCas9 and dCasVRER into the pBad vector. Also all four gBlocks (not PCR amplified) were cloned into the linearized pSB3C5:pLacGFP plasmid in the same way. We successfully transformed our own E. coli cells with all separate constructs. We also tried to transform commercial competent cells with all combinations of pBad and pSB3C5 vectors, but except for two samples this resulted in empty plates.
The plasmids of all single transformants were isolated and analysed by restriction with PstI for the pSB3C5 plasmids and PstI+ EcoRI for the pBad plasmids.
Spacer acquistion
When we got the sequencing results back, it turned out that the I473F gBlock part of the gene contained a wrong sequence. The start of the I473F gBlock part of the gene contained a serie of repeats.
With the colony PCR we thought this was due to bad primer binding and since 3 PCR reactions did work out we felt confident at that point that the construct was good. We are not sure when this wrong sequence was introduced, but to be sure we would have liked to order a new gBlock. Unfortunately, this would have taken too much time. This meant that we could not submit the biobrick compatible hCas9 operon as a biobrick anymore.
Plasmid DNA crArray with spacer 20 was sent for sequencing, construction of array with empty spacer in pSB1C3 was repeated due to bad sequencing results.
Reporter
Colony pcr on potential positive pSB1C3 target array colonies was performed. We attempted to insert 5N5 into target array biobrick with restriction enzymes and alkaline phosphatase since the size of the colony PCR products was longer than expected.
Week 43
dCas9
This week we succeeded in the construction of all double transformants needed for the dCas9 & dCas9VRER validation. This we transformed our own competent cells with an equimolar mix of the pBad vector and the pSB3C5 vector. A table of all double transformant strains is shown in the result section.
In the remainder of the week we tried to prove CRISPR interference by performing overnight kinetic measurements of the OD and fluorescence level of all strains. The results of this measurements were not as useful as we hoped it to be. We saw that the fluorescence per OD only started to increase at higher OD’s and the plate reader that we used could not measure such high levels of fluorescence.
We did see some minor effects primarily in the OD at which the fluorescence was too high to measure. We think that this might be caused by CRISPR-interference, but we did a follow-up experiment to get better data.
In the next experiment we induced our cells at an OD of 0.4-0.6 and let them grow while induced overnight. The next day we measured the OD and fluorescence, which  gave some really nice results. The experiment and a more detailed description is given on the results page.
Spacer acquisition
Although we could not submit the biobrick compatible hCas9 as a biobrick anymore, we still thought it would be interesting to see if hCas9 would work. Instead of using our own construct we were planning to use the original S. pyogenes hCas9 operon and insert it into an inducible E. coli and L. lactis expression vector. To validate that hCas9 is working we also wanted to include the spacer array into the same vector. The vectors that we wanted to use the pBad vector, an arabinose inducible E. coli expression vector, and pNZ8048, a nisin inducible L. lactis expression vector. We wanted to assemble all these parts with a gibson assembly so we used overhang PCR to attach the correct homology regions.(see table). After all the parts were constructed we tried to put them together using gibson assembly. The hCas9 and backbone are approximately the same size so we decided to use equimolar amounts for those and twice as much of the smaller array insert. After transformation there were no colonies visible on the plate. After this we decided stop with the hCas9 project and focus on the dCas9 project.
Primers
Template
E. coli
 
 
hCas9
G90, G85
pRH180
array
G86, G91
Custom array 20
backbone
G88, G89
pBad
L. lactis
 
 
hCas9
G84, G85
pRH180
array
G86, G87
Custom array 20
backbone
G82, G83
pNZ8048
Week 44
Lactis toolbox
After constructing the promoter biobricks and expression vectors, the pNisA promoter was validated in a similar fashion as the cells of the InterLab study were examined. After growing an overnight culture at 30°C, the cells were diluted to an optical density of 0.1 in fresh GM17 medium. When the optical density reached a value between 0.5 and 0.6, the culture was split into 5 portions and induced with various amounts of nisin. The used concentrations were 1 ng/µl, 10 ng/µl, 45 ng/µl and 80 ng/µl and were administered from at stock dissolved in 10% glucose. The cells were induced for 6 hours at 30°C and a sample was taken every 45 minutes. The samples were stored in the dark and on ice until further use. Once all samples were collected, the cells were harvested by centrifugation and resuspended in chemically defined medium. The cell suspensions were analysed by two measurements. The first was an absorbance measurement at 600 nm to determine the optical density and therefore the amount of cells present. The second measurement determined the amount of expressed GFP by exciting at 489 nm and measuring emission at 512 nm.
Primer Table
Name
Sequence
G1_Operon_Isolation_Fw1_AflII_Operon_Isolation
GAGCTTAAGATTCTGTGATTTGGATCCTTCC
G2_Operon_Isolation_Rv1_SpeI_Operon_Isolation
GCTACTAGTATAAAACCTCTCCAACTACCTCCCC
G3_Operon_Isolation_Fw2_AflII_Operon_Isolation
TTAGCTTAAGAAACCAAACGTTTCTGGCTC
G4_Operon_Isolation_Rv2_SpeI_Operon_Isolation
GCTACTAGTACAGGAGCCTGAGCAAAATTC
G5_BioBricks_Backbone_Fw1_AflII_Operon_Isolation
GAATCTTAAGCTCTAGAAGCGGCCGCG
G6_BioBricks_Backbone_Rv1_SpeI_Operon_Isolation
TTATACTAGTAGCGGCCGCTGC
G7_pNZ8048_Backbone_Fw1_PstI_BioBrick_Compatible
ATACTGCAGTCAAGCTTTCTTTGAACCAAAATTAGAAAACC
G8_pNZ8048_Backbone_Rv1_EcoRI_BioBrick_Compatible
ATAGAATTCGTGAGTGCCTCCTTATAATTTATTTTGTAGTTCC
G9_Array_Expansion_Test_Fw1_Array_Expansion
AAAACAGCATAGCTCTAAAACG
G10_Array_Expansion_Test_Rv1_Array_Expansion
AAAACAGCATAGCTCTAAAACA
G11_Array_Expansion_Test_Rv2_Array_Expansion
AAAACAGCATAGCTCTAAAACT
G12_Array_Expansion_Test_Rv3_Array_Expansion
GGCTTTTCAAGACTGAAGTCTAG
G13_Array_Expansion_Test_Rv4_Array_Expansion
AAAACAGCATAGCTCTAAAACC
G14_Operon_exptCas9_Fw1_AflII_Operon_Isolation
AGAGCTTAAGGATGCCACTCTTATCCATCAATCC
G15_fw_prefix_Spacer_Acquisition
ATAGAATTCGCGGCCGCTTCT
G16_rv_suffix_Spacer_Acquisition
TATCTGCAGCGGCCGCTAC
G17_fw_XbaI_overhang_Spacer_Acquisition
CTGGAATTGATTAAGTCCTTAGGAG
G18_rv_prefix_overhang_Spacer_Acquisition
GCCGATATCTAAGCCTATTGAG
G19_fw_PCR1_Spacer_Acquisition
CAATAGGCTTAGATATCGGCAC
G20_rv_PCR1_Spacer_Acquisition
AGCTGAGCAATGAGATTTTCT
G21_fw_geneblock2_Spacer_Acquisition
GAAAATCTCATTGCTCAGCTC
G22_rv_geneblock2_Spacer_Acquisition
AAGCACCTTTATCGACAACTT
G23_fw_PCR2_Spacer_Acquisition
AGTTGTCGATAAAGGTGCTTC
G24_rv_PCR2_Spacer_Acquisition
CCTAAGGACTTAATCAATTCCAGG
G25_fw_pIL_lin_Plasmid_Linearization
CGAGCTGCAGATCGATACGATTTTGAAGTGG
G26_rv_pIL_lin_Plasmid_Linearization
CAAACATGAGAATTCTATTTAATCACTTTG
G27_fw_separation_XbaI_tracR+XbaI_Spacer_Acquisition
ATATCTAGACCGGGTATATTTACTAATCGCT
G28_rv_suffix_separation_XbaI_TracR_Spacer_Acquisition
AAAAGTTTCTCTAGAAGCGGC
G29_fw_suffix_separation_XbaI_TracR_Spacer_Acquisition
AACGCTCTCTACTAGTAGCG
G30_rv_separation_XbaI_tracR+SpeI_Spacer_Acquisition
ATAACTAGTATCGCACTATTTGTCTCAGC
G31_fw_pMG36_lin
GACCTGCAGGCATGCAAGC
G32_rv_pMG36_EcoRI_lin
ATAGAATTCTTCAAAATTCCTCCGAATATTTTTTTACCTACC
G33_fw_dCas9_BBcom
CGAATACCTTATTATGTTGGTCCATTGGCGC
G34_rv_dCas9_BBcom
AAGGTATTCGAAAAGTCAAGATTTTTTCAATCTTCTCACG
G35_fw_XbaI_dcas9_ChengXi
ATATCTAGAATGGATAAGAAATACTCAATAGGCTTAGC
G36_rv_dcas9_ChengXi
GTACTGCAGAATTGCCGACC
G37_fw_gblockdCas9
CTGGGATCCAAAAAAATATGGTGGTTTTG
G38_rv_seperation_XbaI_TracR_2_spacer-acquisition
ATTACTAGTTGAAAAGCCCCTGTATTACTGC
G39_rv_quickchange_2_dCas9
AAGGTATTCGAAAAGTCAAGATTTTTTCAATCTTCTCACG
G40_fw_Quickchange_2_dCas9
CGAATACCTTATTATGTTGGTCCATTGG
G41_rv_CustomSpacerRepeat_BsaI_Reporter
TCCTACGGTCTCCAAGTTATCATCGGCAATGTTGTTTTGGGACCATTCAAAACAGCATAGCTCTAAAAC
G42_fw_CustomSpacer20_BsaI_Reporter
GCTGGAGGTCTCGAAACATTTCAACCTTAATAGTAGACAACGCAGGAGTTTTAGAGCTATGC
G43_fw_CustomSpacer21_BsaI_Reporter
GCTGGAGGTCTCGAAACAGCAGTTCTGTTTAGGCGGAAGCAATACTCGTTTTAGAGCTATGC
G44_fw_EmptySpacer_BsaI_Reporter
GCGTATGGTCTCAAAACAACATTGCCGATGAT
G45_rv_EmptySpacer_BsaI_Reporter
CAGCACGGTCTCCAAGTTATCATCGGCAATGT
G46_fw_sfGFP_Insert_BsaI_Reporter
GTAGTGGTCTCAAGCTGACATTGAAGTTTATTTGC
G47_rv_sfGFP_Insert_BsaI_Reporter
CTAGCGGTCTCAAGCTCATCCATGCCGTG
G48_ChengXi_dCas9_R(pNZ8048)_136
GGCTTGAAACGTTGGATTTCCTTACGCGAAATAC
G49_ChengXi_dCas9_R(pseudo3)_00160135
AGACTGGAAGCTTGGATTTCCTTACGCGAAATAC
G50_ChengXi_dCas9_F_00150134
ATGGATAAGAAATACTCAATAGGC
G51_Seq_fw_dCas9_1
GCTTCAATGATTAAACGCTACG
G52_Seq_rv_dCas9_2
CGTAGCGTTTAATCATTGAAGC
G53_Seq_fw_dCas9_3
TGGGGACGTTTGTCTCG
G54_Seq_rv_dCas9_4
GACAAACGTCCCCAACC
G55_Seq_fw_dCas9_5
AGAAATAGGCAAAGCAACCG
G56_Seq_rv_dCas9_6
ATTTTGCGGTTGCTTTGC
G57_Seq_fw_CRISPRoperon_Cas1
GTTGGCGTACTGTTGTGG
G58_Seq_rv_CRISPRoperon_Cas1
CACAACAGTACGCCAACC
G59_Seq_fw_CRISPRoperon_Cas2
TGATTGGTCGGCTGAGG
G60_Seq_rv_CRISPRoperon_Cas2
CCGACCAATCATGGCATTG
G61_pNZ8048_Backbone_amp2.0_XbaI_rv
ACACACTCTAGAGTGCCTCCTTATAATTTATTTTGTAG
G62_pNZ8048_Backbone_amp2.0_PtsI_fw
ACACACCTGCAGCTCAAGCTTTCTTTGAACC
G63_pUSP45
CTGCAGCGGCCGCTACTAGTAGGTTCCAAAAACATATTATACTATTCC
G64_pUSP45
TATATAGAATTCGCGGCCGCTTCTAGAGC
G65_pNIS
TAGTAGTAGAATTCGCGGCCGCTTCTAGAGCTAGTCTTATAACTATACTGACAATAG
G66_pNIS
CTGCAGCGGCCGCTACTAGTAGTGAGTGCCTCCTTATAATTTATTTTGTAG
G67_p32
TATATAGAATTCGCGGCCGCTTCTAGAGGATATGATAAGATTAATAGTTTTAGCTATTAATC
G68_p32
CTGCAGCGGCCGCTACTAGTAGAATTACGAATTTTTCTGCTGAAACG
G69_pSB1C3_lin
ATACACTCTAGAAGCGGCCGCGAATTC
G70_pSB1C3_lin
ATACTTACTAGTAGCGGCCGCTGCAG
G71-Fw-dCas9-pBadComp
AATTAACCATGGATAAGAAATACTCAATAGGCTTAGCT
G72-Rv-dCas9-pBadComp
ATGGTCGACGTCAGTCACCTCCTAGCTGACTC
G73-Rv-pBad-dCas9Comp
TTATCCATGGTTAATTCCTCCTGTTAGC
G74-Fw-Pbad-Dcas9Comp
ACTGACGTCGACCATCTAGAACAAAAACTC
G75-Fw-gRNAAmp
ATTCCGCTGCAGGAGAATTC
G76-Rv-gRNAAmp
CGGCCGCTGCAGATAAAGC
G77-Fw-pSB3C5
ATTCCGCTGCAGGAGTCACTAAGG
G78-Fw-Gibbsonbackbone
GTAATACAGGGCGCTATGTACACCCGGTTG
G79-Rv-Gibbsonbackbone
TTCTTATCCATGTGAGTGCCTCCTTATAATTTATTTTGTAG
G80-Fw-GibbsonhCas9
GAGGCACTCACATGGATAAGAAATACTCAATAGGC
G81-Rv-GibbsonhCas9
TGTACATAGCGCCCTGTATTACTGCATTTATTAAGAG
G82_gib_pNZ8048_rv
TTCTTATCCATGTGAGTGCCTCCTTATAATTTATTTTGTAG
G83_gib_pNZ8048_fw
ATGGAACAAACCGCTATGTACACCCGGTTG
G84_gib_pNZ-hCas9_fw
GAGGCACTCACATGGATAAGAAATACTCAATAGGC
G85_gib_hCas9_rv
CTGCATTTATTCCCTGTATTACTGCATTTATTAAGAG
G86_gib_array_fw
GTAATACAGGGAATAAATGCAGTAATACAGGGG
G87_array-pNZ8048_rv
TGTACATAGCGGTTTGTTCCATCACACTACTC
G88_gib_pBad_rv
TTTCTTATCCATGGTTAATTCCTCCTGTTAGCCCAAAAAAC
G89_gib_pBad_fw
GATGGAACAAACTCGACCATCATCATCATCATC
G90_gib_pBad-hCas9_fw
GAGGAATTAACCATGGATAAGAAATACTCAATAGGCTTAG
G91_gib_array-pBad_rv
GATGATGGTCGAGTTTGTTCCATCACACTACTC
pJET-Fwd
AATAGGCGTATCACGAGGC
pJET-Rev
AGTCAGTGAGCGAGGAAGC
pNZ-forward
AGCTCCAAGATCTAGTCTTATAAC
pNZ-reverse
ATCAATCAAAGCAACACGTG
 
Next: Future research
