<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_test5 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/test5</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="primary_nav_wrap" style="z-index: 2000"><UL><LI class="mobicon"><A href="javascript:void(0);" onclick="mobnavtoggle()">≡ Menu</A></LI><LI><A href="https://2017.igem.org/Team:Groningen">Home</A></LI><LI><A onclick="">Project</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/FutureResearch">Future Research</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/InterLab">InterLab</A></LI></UL><LI><A onclick="">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Overview">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/HP/Silver">Human Practices (Silver)</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/HP/Gold_Integrated">Integrated (Gold)</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Engagement">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/FutureOutlook">Future Outlook</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Collaborations">Collaborations</A></LI><LI><A onclick="">Team</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Team">Team Members</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Achievements">Achievements</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Attributions">Attributions</A></LI><LI class="dir"><A href="https://2017.igem.org/Team:Groningen/Fun">Fun Activities</A></LI><LI><A href="https://2017.igem.org/Team:Groningen/Sponsors">Sponsors</A></LI></DIV><DIV class="marginsnav"><DIV class="main-col scrollcol"><NAV id="myScrollspy"><UL class="nav"><LI class="accord1 monthdown"><A href="javascript:void(0);" onclick="accord1toggle()">June ▼</A></LI><LI class="accord1 monthup"><A href="javascript:void(0);" onclick="accord1toggle()">June ▲</A></LI><LI><A href="#snav0511">Week 23</A></LI><LI><A href="#snav1218">Week 24</A></LI><LI><A href="#snav1925">Week 25</A></LI><LI><A href="#snav2602">Week 26</A></LI></UL><LI class="accord2 monthdown"><A href="javascript:void(0);" onclick="accord2toggle()">July ▼</A></LI><LI class="accord2 monthup"><A href="javascript:void(0);" onclick="accord2toggle()">July ▲</A></LI><LI><A href="#snav0309">Week 27</A></LI><LI><A href="#snav1016">Week 28</A></LI><LI><A href="#snav1723">Week 29</A></LI><LI><A href="#snav2430">Week 30</A></LI><LI><A href="#snav3106">Week 31</A></LI><LI class="accord3 monthdown"><A href="javascript:void(0);" onclick="accord3toggle()">August ▼</A></LI><LI class="accord3 monthup"><A href="javascript:void(0);" onclick="accord3toggle()">August ▲</A></LI><LI><A href="#snav0713">Week 32</A></LI><LI><A href="#snav1420">Week 33</A></LI><LI><A href="#snav2127">Week 34</A></LI><LI><A href="#snav2803">Week 35</A></LI><LI class="accord4 monthdown"><A href="javascript:void(0);" onclick="accord4toggle()">September ▼</A></LI><LI class="accord4 monthup"><A href="javascript:void(0);" onclick="accord4toggle()">September ▲</A></LI><LI><A href="#snav0410">Week 36</A></LI><LI><A href="#snav1117">Week 37</A></LI><LI><A href="#snav1824">Week 38</A></LI><LI><A href="#snav2501">Week 39</A></LI><LI class="accord5 monthdown"><A href="javascript:void(0);" onclick="accord5toggle()">October ▼</A></LI><LI class="accord5 monthup"><A href="javascript:void(0);" onclick="accord5toggle()">October ▲</A></LI><LI><A href="#snav0208">Week 40</A></LI><LI><A href="#snav0915">Week 41</A></LI><LI><A href="#snav1622">Week 42</A></LI><LI><A href="#snav2329">Week 43</A></LI><LI><A href="#snav3001">Week 44</A></LI></NAV><H1>Notebook</H1><P> Here you will find our complete labjournal. We documented our progress through the past months here.</P><H4>June</H4><H5 id="snav0511">Week 23</H5><P>Although we obtained the keys of our lab on the first of June we have not yet performed any real experiments. The entire week was spent on setting up the lab, e.g. getting all materials and machines needed. Also we made some media and standard buffers and we have produced our first batch of competent <I>E. coli</I> DH5α at the end of this week.
</P><H5 id="snav1218">Week 24</H5><H6>General</H6>
This week we could finally start doing some real experiments in the lab! We started with optimizing the heat-shock time in the transformation protocol. We did this by transforming our competent cells with DNA from the competent cell test kit with various heat shock times (0-60 seconds). We found that the highest efficiency was obtained when using a heat shock of 30 seconds.
  <H6>Interlab</H6>
Because we did not have fully finished the design of any of the experiments for our own project yet, we started with the Interlab study. As a result we can proudly say that we were the first team to complete the Interlab project this year!!
  <P>We have successfully transformed all test devices of the interlab studies. Of each of the transformation plates one colony was picked, which was inoculated and grown overnight. After plasmid isolation of the ON cultures restriction analysis with EcoRI showed that all colonies contained the correct plasmids. In the remainder of the week we have completed the measurements according the plate reader protocol. More information about the interlab studies can be found on our <A href="https://2017.igem.org/Team:Groningen/InterLab">interlab page</A>.</P> 
This and all subsequent experiments requiring plate reading/GFP/RFP were performed with <A href="”http://www.masontechnology.ie/files/documents/BIO3.pdf”">Synergy Mx</A>.
<H5 id="snav1925">Week 25</H5><H6>General</H6>
We have inoculated one of the red colonies from the heatshock-test and isolated the plasmid (pSB1C3:bba_J04450) from it. This plasmid was used to confirm that the restriction enzymes, which were left over from the previous iGEM team of Groningen, worked properly. Moreover this plasmid was also used for cloning parts into the pSB1C3 backbone.
  <H6>Lactis toolbox</H6><P>After completing the interlab we have worked on the lactis-toolbox. We started with two different parts of this sub-project. First was the preparation of electrocompetent <I>L. lactis</I> NZ9000 cells and optimization of the transformation protocols for it. The second step was the preparation of a pTet-GFP/RFP and a pLac-GFP/RFP construct to test these promoters in L. lactis. From next week progress for this part of the project will be shown under the header ‘promoter testing’.
This week we produced our first batch of competent lactis cells and determined a growth curve for NZ9000 in SGM17+glycine medium (see figure below). For the second part we successfully transformed <I>E. coli</I> with pLac (bba_), pTet (bba_), sfGFP (bba_) and a RBS (bba_).</P><H5 id="snav2602">Week 26</H5><H6>dCas9</H6>
We have tried to transform our <I>E. coli</I> with the <A href="”http://parts.igem.org/Part:BBa_K1723000”">dCas9-Omega fusion protein</A> several times, however all attempts failed until we had no DNA left over. To solve this problem we determined to use the pJWV-102-PL-dCas9 and make the dCas9 biobrick compatible ourselves.
  <H6>Spacer acquisition</H6>
We obtained the gDNA of a clinical isolate of <I>S. pyogenes</I> from J.M. van Dijl. We tried to PCR out the entire CRISPR operon at once using primers (G1-G4) failed. We have tried the PCR several time including with a temperature gradient PCR. But since all attempts failed we think this isolate may have not contained the CRISPR system. The problem was solved when the plasmids of the Heler lab arrived (see next week).
  <H6>Lactis toolbox</H6><P>We have obtained five L. lactis vectors from our supervisor H. Karsens in different host strains (see Table below). The cultures were grown overnight and the plasmids were isolated successfully.</P><TABLE class="tableizer-table"><THEAD><TR class="tableizer-firstrow"><TH>Plasmid</TH><TH>Host strain</TH><TH>Growth medium</TH><TH>Size (linearized)</TH></TR></THEAD><TBODY><TR><TD>pNZ8048</TD><TD><I>L. lactis</I> NZ9000</TD><TD>M17 + Clm</TD><TD>3.3 kb</TD></TR><TR><TD>pMG36E</TD><TD><I>L. lactis</I> IL1403</TD><TD>M17 + Ert</TD><TD>3.3 kb</TD></TR><TR><TD>pMG36C (low copy)</TD><TD><I>E. coli</I> Mg1363</TD><TD>LB + Clm</TD><TD>3.3 kb</TD></TR><TR><TD>pIL252 (low copy)</TD><TD><I>L. lactis</I> IL1403</TD><TD>M17 +Ert</TD><TD>4.6 kb</TD></TR><TR><TD>pIL253 (High copy)</TD><TD><I>L. lactis</I> NZ9000</TD><TD>M17 + Ert</TD><TD>4.9 kb</TD></TR></TBODY></TABLE><H4>July</H4><H5 id="snav0309">Week 27</H5><H6>Spacer acquisition</H6>
This week the Heler plasmids arrived so we could start with creating the different DNA fragments for the gibson assembly of the hCas9 operon.We started with PCR amplifying the DNA fragments from the pWJ40 plasmid using the primer pairs G19, G20 and G23, G24. The resulting PCR products were treated with DpnI and cleaned with a PCR clean up kit.
  <H6>Lactis toolbox</H6>
The competent lactis cells that were produced in week 25, were tested by transforming them with the isolated pNZ8048 plasmid. The transformation did not work out, probably because the Chloramphenicol concentration in the plates was too high (25ug/ml). Also the pNZ8048 plasmid was linearized and the biobrick prefix and suffix were incorporated. The PCR was performed using Phusion polymerase and primers <B> … (gel?)</B><P>The <A href="”http://parts.igem.org/Part:BBa_K1365020”">sfGFP</A>  was cloned into the linearized vector using restriction ligation with EcoRI and PstI. The linearized pNZ8048 was not digested with DpnI, because <I>L. lactis</I> plasmid DNA is not methylated and therefore DpnI cannot digest the template. The ligation product was stored in the fridge.</P><H6>Promotor testing</H6>
For this part we successfully transformed and isolated bba_, which contains the RFP plasmid. Also we completed the first round of 3A assemblies, shown in the table below.

<TABLE class="tableizer-table"><THEAD><TR class="tableizer-firstrow"><TH>Backbone</TH><TH>construct A</TH><TH>construct B</TH><TH>Product name</TH></TR></THEAD><TBODY><TR><TD>pSB1T3</TD><TD>pLac</TD><TD>sfGFP</TD><TD>pLac_GFP</TD></TR><TR><TD>pSB1T3</TD><TD>pLac</TD><TD>RFP</TD><TD>pLac_RFP</TD></TR><TR><TD>pSB1T3</TD><TD>pTet</TD><TD>RBS</TD><TD>pTet_RBS</TD></TR></TBODY></TABLE><P>The plates for pLac_GFP and pLac_RFP contained four colonies and the pTet_RBS plate contained only one colonies. All colonies were heavily overgrown since they were left at 37C for the entire weekend.</P><H5 id="snav1016">Week 28</H5><H6>General</H6>
We finished the first part of the design of the dCas9 and the spacer acquisition projects and we ordered the second batch of primers (G15-G37).
New competent <I>E. coli</I> DH5α and <I>L. lactis</I> cells were produced. gBlocks and Primers for the reporter project were ordered as well.
  <H6>dCas9</H6>
From our supervisor C. Huang we received <I>E.coli</I> DH5α containing pJWV-102-PL-dCas9. These cells were plated and one colony was picked and inoculated. The plasmid was isolated from the ON cultures and analysed by restriction analysis with PstI <B> (see gel next week).</B><H6>Promotor testing</H6>
The colonies of the 3A assemblies that were heavily overgrown were re-plated. Each plate contained many colonies and those on the pLac-GFP plate were coloured green. Restriction analysis of isolated plasmids from one of each of the colonies showed the correct size for pLac_GFP and pTet_RBS, but not for pLac_RFP. Since we only wanted to use the constructs for the function of pLac in <I>L. lactis</I> and we only need one marker for this we decided to not retry the pLac_RFP assembly.
<H5 id="snav1723">Week 29</H5><H6>General</H6>
This week the second batch of primers arrived, so we could finally continue our work in the dCas9 and the spacer acquisition projects. We PCR amplified gBlocks for the target &amp; crArray, however the PCRs failed.
  <H6>Spacer acquisition</H6>
This week the gBlocks arrived. The I473F/EcorI gblock was amplified using the primer pair G21, G22. The other gBlock contained both the tracer RNA and a part of the hCas9 operon. It was first inserted into pSB1C3 by restriction ligation with EcorI and PstI and transformed into <I>E. coli</I> DH5α. The transformation went well and the plasmids checked by a restriction analysis with EcorI. We planned to seperate the tracer RNA from the hCas9 operon when we linearized the plasmid using PCR. With the primer pair G27, G28 we wanted to create a linearized tracerRNA pSB1C3 vector and with primer pair G17, G18 we wanted to create a linearized hCas9 vector, which would be the backbone for the gibson assembly.
  <H6>dCas9</H6>
The isolated pJWV-102-PL-dCas9 was used to PCR out the dCas9 gene using primers G35 and G36. Since it is a large size construct (4,2 kb) we used Phusion polymerase. At the moment we thought that we had the correct PCR product, because we did not analyse the gel carefully enough. In the figure below the PCR fragment is shown together with the restricted pJWV-102-PL-dCas9 and of pSB1C3:dCas9 (see next week). From the gel it can be seen that the actual product sizes on the left are larger than they should be. On the right side of the gel the same products are shown, but this time with the correct plasmid.
<H5 id="snav2430">Week 30</H5><H6>Spacer acquisition</H6>
Trouble shooting PCR for gBlock separation of hCas9-tracer RNA with different annealing temperatures and addition of DMSO to the PCR mix.
  <H6>dCas9</H6>
As said in the report of last week the PCR fragment was cloned into [SB1C3:GFP (bba_) via restriction ligation with XbaI and PstI. The transformation resulted in four colonies of which the plasmids were isolated and analyzed by restriction with PstI and EcorI. <B>The results are shown in the figure below on the right.</B> Colonies 2 and 3 seemed to have bands of the right size and those plasmids were used for follow-up experiments. However as explained in the previous report this dCas9 turned out to be wrong, so all other experiments using this part failed.
<H5 id="snav3106">Week 31</H5><H6>General</H6>
gBlock dCas9 VRER and hCas9I473V were ligated into linearized pJET1.2 bluntend vector via blunt end ligation. The pJET vector was a gift from our supervisor C. Pohl, who also gave us two primers pJet_Fwd and pJet_Rev which bind on both the pJet backbone and the pSB1C3 backbone. These primers were used for sequencing and to confirm whether a fragment was inserted into pSB1C3.
  <H6>Lactis toolbox</H6>
Transformation of lactis with pNZ8048. The transformation product was plated on plates containing different concentrations of chloramphenicol (0-15 ug/ml). 5 ug/ml was found to be the best concentration.
  <P>Moreover we linearized the pIL252 and pIL 253 backbones using Phusion polymerase with primers G25, G26.</P><H6>Spacer acquisition</H6>
To separate the TracrRNA part and the prefix-suffix part from the plasmid containing the hCas9 gblock we wanted to do two PCRs. One to obtain the pSB1C3:pUsp45TracrRNA and one with the pSB1C3 containing the part used for the hCas9-operon. However this week all attempts of performing this PCR failed.
<H4>August</H4><H5 id="snav0713">Week 32</H5><H6>General</H6>
New primers for dCas9 Quickchange and separation of hCas9 combi gblock were ordered.

This week we succesfully transformed <I>E. coli</I> with the pJET vectors containing the gBlocks. The colonies on the plate were analysed by colony PCR with the pJet primers and for both pJET-constructs we isolated the plasmid of the colonies with a positive size of the insert. <B>Gel 9/8?</B><H6>Lactis toolbox</H6>
To optimize the transformation protocol of <I>L. lactis</I> further we tested different DNA to cell ratio’s. We used either 50 or 100 ul of competent <I> L. lactis</I> cells and transformed them with 50, 100 or 250 ng of pNZ8048. The volume of DNA that was added was respectively 0.5, 1 or 2,5 ul.
  <P>The results are shown in the table below. Although we find it strange that using more cells did not result in more colonies the result does show that adding more DNA does not result in more colonies. We think that this is not caused by the amount of DNA that is added, but by the volume of the DNA that is added. Adding a larger volume will dilute the competent cells further, which will lead to a decreased transformation efficiency.</P><TABLE class="tableizer-table"><THEAD><TR class="tableizer-firstrow"><TH>Ratio (cell:DNA)</TH><TH>Colonies</TH><TH>Ratio (cell:DNA)</TH><TH>Colonies</TH></TR></THEAD><TBODY><TR><TD>50:50</TD><TD>700-800</TD><TD>100:50</TD><TD>200-250</TD></TR><TR><TD>50:100</TD><TD>100-150</TD><TD>100:100</TD><TD>100</TD></TR><TR><TD>50:250</TD><TD>150-200</TD><TD>100:250</TD><TD>100:150</TD></TR></TBODY></TABLE><H6>Spacer acquisition</H6><P>After testing even more PCR conditions we finally thought that we succeeded in creating the pSB1C3 containing the tracer RNA part. The PCR which succeeded was a temperature gradient PCR using phusion polymerase, primers G27, G28.</P><H6>Reporter</H6><P>We successfully amplified the gBlocks of the target and crArray (Primer G15/16).</P><H5 id="snav1420">Week 33</H5><H6>General</H6>
Since the PCR’s for the quickchange PCR of dCas9 and also the PCR of the gblock of hCas9 kept failing we designed and ordered new primers for them.
Also we send the isolated pJET-gblock plasmids and pSB1C3 containing the hCas9 combigblock. <B> RESULT?</B><H6>dCas9</H6>
Besides ordering the new quickchange primers we also rechecked our template DNA together with the PCR fragment and the original dCas9 vector. At this point we found out that the dCas9 PCR might be too big.
<B>MART15-08 sequencing results of three samples (other samples did not work out)</B><H6>Reporter</H6>
We tried to insert the gblocks into PSB1C3 backbone with restriction ligation (EcoRI, PstI). Which turned out to be unsuccessfull.
<H5 id="snav2127">Week 34</H5><H6>Spacer acquisition</H6>
The pUSP45 tracer RNA(see week 32) was put into an iGEM backbone and transformed into <I>E. coli</I> DH5α according to protocol. There were colonies visible on the plates and these were used for ON cultures. The plasmids were isolated and send for sequencing.
  <H6>dCas9</H6>
Since the PCR product of dCas9 seemed larger than the expected size we redid the PCR to isolate dCas9 from the pJWV-102-PL-dcas9 plasmid. However the product was the same size as the one in the previous experiment.
  <H6>dCas9</H6>
Since the PCR product of dCas9 seemed larger than the expected size we redid the PCR to isolate dCas9 from the pJWV-102-PL-dcas9 plasmid. However the product was the same size as the one in the previous experiment.
  <H6>Reporter</H6>
We attempt to clone the gBlocks into the Backbon PSB1C3 varying restriction digestion time, enzyme amounts and ligation time but results did not satisfy our desires.
<H5 id="snav2803">Week 35</H5><H6>Spacer acquisition</H6><P>The sequencing results for the pUSP45 tracer RNA part came back and showed that the plasmids did not contain the pUSP45 tracer RNA gene. Instead it contained a GFP gene. Therefore we designed two new gBlocks, one containing only the pUsp45TracrRNA and one containing the prefix-suffix for the hCas9 operon. <B>sequencing results link</B></P><H4>September</H4><H5 id="snav0410">Week 36</H5><H6>dCas9</H6>
We discussed the odd size of the PCR fragment of dCas9 with our supervisor C. Huang. Since she had managed to a correctly sized dCas9 fragment from the plasmid we decided to redo the dCas9 PCR. This time we tested both our own purified PJWV plasmid and the plasmid C. Huang purified from the same strain. Also we tested the primers that she had used in addition to our own. This time we did get a PCR product of the right size using the template of C. Huang and our own primers <B>(G &amp; G)</B>.
  <P>We restricted both the pJWV-102-PL-dCas9 plasmid we miniprepped ourselves as the one we obtained from C. Huang and analysed the resuts together with the PCR fragments on a gel. From the gel (see below) can be seen that not only the PCR fragment is bigger but also the upper band of the plasmid.</P><H6>Spacer acquisition</H6>
Waiting for gblocks to arrive.
<H5 id="snav1117">Week 37</H5><H6>dCas9</H6>
The new correctly sized dCas9 fragment was ligated into a pSB1C3 backbone via restriction- ligation with XbaI and PstI. We successfully transformed <I>E. coli</I> with the ligated plasmid and subsequently isolated the plasmid.
  <P>To remove the illegal EcoRI site a PCR with all four combinations of the two sets of quick-change primers was performed <B>(G G and G G )</B>. All four PCR reactions were analysed on a gel and turned out to be successful. After 15 minutes restriction at 37 C with DpnI 3µl followed by a PCR clean up. 3 µl of this product was directly used to transform <I>E. coli</I> the rest was ligated ON.</P><P>Although the transformation with the ligated dCas9-QC product resulted in more colonies than that with the unligated product, we continued with the latter since already two colonies were grown overnight, the plasmid of the ON cultures were already isolated and stored as (pSB1C3:dCas9QC).</P><H6>Spacer acquisition</H6>
The new pUSP45 tracer RNA gblock has arrived. We amplified the gblock as backup with the G15, G16 primer pair using Phusion polymerase. The gblock was put in the pSB1C3 backbone with restriction ligation using EcorI and PstI and was transformed in <I>E. coli</I> DH5α. The resulting colonies were grown ON and the plasmids were isolated. The insert of the plasmids were checked on gel after restriction with EcorI and PstI.
<H5 id="snav1824">Week 38</H5><H6>dCas9</H6>
The isolated dCas9QC plasmids were analysed by restricting them with XbaI + PstI and with EcoRI. As a negative control the old pSB1C3:dCas9 plasmid was restricted with the same enzymes. For the first combination all samples have the same pattern (2kb + 4,2 kB bands), but for the EcorI restriction the pSB1C3:dCas9 contained two bands (4,8 kb + 1,4 kB) whereas all quickchange samples had only one band (6,2 kb). This result is as expected since the part should contain one EcoRI site in the prefix.
<B>GEL 18-09</B><P>In order to produce dCas9VRER we restricted the amplified gblock dCas9VRER and pSB1C3:dCas9QC with BamHI and PstI. The correct band (5,5 k) of the plasmid digestion was extracted from the gel and the gblock restriction was inactivated by PCR purification. Since the concentrations of the both purified products were very low the ligation and following transformation were unsuccessful.</P><H6>Spacer acquisition</H6>
The new hCas9 gblock has arrived. It was first PCR amplified with primer pair G15, G16. The gblock was also inserted in pSB1A3 with restriction ligation using EcorI and PstI. The resulting colonies were grown ON and the plasmids were isolated and stored.
  <H6>Reporter</H6>
We finally were able to insert the amplified gblocks into the Backbone after PCR purifying the products as well as after restriction multiple PCR products were pooled together to elevate the concentration. Ligation occurred overnight. The results were verified by colony PCR with pJET primers. 
<H5 id="snav2501">Week 39</H5><H6>dCas9</H6>
After a few attempts to construct dCas9VRER in the way described last week, which all failed we decided to clone the gBlock into the pJET1.2 vector using blunt end cloning. This was done to amplify the gBlock and to increase the efficiency of the restriction by increasing the length of the overhangs (which is now the entire backbone).
  <H6>Spacer acquisition</H6>
The hCas9 gblock pSB1A3 plasmid had to be linearized before it could be used in the gibson assembly. Since all the DNA fragments for the gibson assembly need a specific overhang, we decided to linearize with PCR. We did this with the primer pair G17, G18 with NEB high fidelity polymerase. The resulting product was treated with DpnI and cleaned with a PCR clean up kit.
<H4>October</H4><H5 id="snav0208">Week 40</H5><H6>dCas9</H6>
The plasmids pSB1C3:dCas9QC and pJET:gBlockdCas9VRER were both send for sequencing. For the first plasmid we used primers G51-G56 as well as pJet_Fwd and pJet_Rev. The results of sequencing came back positive and are discussed further in the Result section.
  <P>After several restriction and ligation attempts using the pSB1C3:dCas9QC and pJET:gBlockdCas9VRER plasmids we finally successfully transformed <I>E. coli</I> with pSB1C3dCas9VRER this week! From the plate 7 colonies were picked, grown and minipreppred. Restriction analysis of the isolated plasmids with PstI showed that all colonies contained a plasmid of the expected size (6,2 kb).
<B>Add gel?</B></P><H6>Spacer acquisition</H6>
This week we performed the gibson assembly to create the biobrick compatible hCas9 operon. To get the hCas9 operon we had to put four parts together: the two unaltered PCR fragments from week 3-07, the amplified gblock I473F from week 17-07 and the linearized backbone from last week. Since we had more than 3 fragments we wanted to fuse we had to put all the fragments together in equimolar amounts. The gibson assembly product was transformed in <I>E. coli</I> DH5α. We used a colony PCR to determine if the resulting colonies contained the desired product. We used several combinations of primers: G19/G20, G21/G22, G57/G60 and G16/G59. <B>(See gel)</B>. Two colonies contained 3 out of 4 expected fragments. These were inoculated and grown ON.
  <H6>Reporter</H6>
Incorporation of target array G-block into pSB1C3. <B>GEL: 28-09 colonypcrArray floor matthijs2</B><P>Construction and analysis biobrick from target array and GFP insert. Restriction ligation with HindIII and BsaI. <B>GEL [01102017 colony pcr 17 colonies potential GFP reporter transformants all negative</B></P><P>The crArray was amplified from gblocks again using G15/G16, restriction ligation was performed using XbaI and PstI. Transformation into <I>E. coli</I> DH5α and plasmid isolation.</P><H5 id="snav0915">Week 41</H5><H6>dCas9</H6>
To confirm that the pSB1C3:dCas9VRER contained the right insert we sequenced one of the plasmids with pJet_Fwd and pJet_Rev. The results came back positive and the sequenced sample was stored separately and it was in the end submitted into the registry.
The rest of the week we designed the gRNA gblocks and primers for the dCas9 validation experiments. These were ordered at the start of this week and arrived in the start of next week. Also we obtained a pBad vector from our supervisor Patricia and pSB3C5:pLacGFP (bba_ ) from our supervisor Harma
  <H6>Spacer acquisition</H6>
The biobrick compatible hCas9 operon was inserted into the pSB1A3 vector. Since we wanted to submit it as a biobrick, we had to exchange the backbone for the pSB1C3 backbone. To do this we performed a restriction ligation reaction with EcorI and PstI and transformed in <I>E. coli</I> DH5α. The resulting colonies were grown ON and the plasmids were isolated. The resulting plasmid was sent for sequencing.
  <H6>Reporter</H6>
We  digested the target array(reporter) with HindIII and the crArray with BsaI. Their respective inserts (5N5, rest of GFP &amp; 20,21, empty,  all digested will BsaI) were added after heat inactivation. All crArrays were sent for sequencing. Construction of and analysis of potential positive colonies of tracr RNA incorporated in front of the array. <B>GEL [11102017 Matthijs[2]]</B> 
Analyse potential correct plasmid DNA containing pSB1C3 and the whole reporter sequence. Both restriction analysis and colony pcr. <B>GEL hcas9 operon restiction check last two sambles</B>
Analyse Target backbone because of negative results of colony pcr. <B>GEL {13102917 target array pcr pjet primers check 732 bp}</B><H6>Collaboration</H6>
We performed the analysis of the Nottingham cells as stated on the <A href="https://2017.igem.org/Team:Groningen/Collaborations">collaborations page</A>.
<H5 id="snav1622">Week 42</H5><H6>dCas9</H6>
After the primers arrived we linearized the pBad vector and the pSB3C5:pLacGFP using respectively primers G73 &amp; G74 and G16 &amp; G77. dCas9QC and dCas9VRER were isolated from their backbones using primers G71 &amp; G72.
  <P>Using Gibbson cloning we successfully inserted dCas9 and dCasVRER into the pBad vector. Also all four gBlocks (not PCR amplified) were cloned into the linearized pSB3C5:pLacGFP plasmid in the same way. We successfully transformed our own <I>E. coli</I> cells with all separate constructs. We also tried to transform commercial competent cells with all combinations of pBad and pSB3C5 vectors, but except for two samples this resulted in empty plates.</P><P>The plasmids of all single transformants were isolated and analysed by restriction with PstI for the pSB3C5 plasmids and PstI+ EcoRI for the pBad plasmids.</P><B>Marts need to look up gels</B><H6>Spacer acquistion</H6>
When we got the sequencing results back, it turned out that the I473F gblock part of the gene contained a wrong sequence. The start of the I473F gblock part of the gene contained a serie of repeats.  
  <P>With the colony PCR we thought this was due to bad primer binding and since 3 PCR reactions did work out we felt confident at that point that the construct was good. We are not sure when this wrong sequence was introduced, but to be sure we would have liked to order a new gblock. Unfortunately, this would have taken too much time. This meant that we could not submit the biobrick compatible hCas9 operon as a biobrick anymore.</P><H6>Reporter</H6>
Plasmid DNA crArray with spacer 20 was sent for sequencing, construction of array with empty spacer in pSB1C3 was repeated due to bad sequencing results <B>GEL [20102017]</B>
Colony pcr on potential positive pSB1C3 target array colonies was performed. We attempted to insert 5N5 into target array biobrick with restriction enzymes and alkaline phosphatase since the size of the colony PCR products was longer than expected.
<H5 id="snav2329">Week 43</H5><H6>dCas9</H6>
This week we succeeded in the construction of all double transformants needed for the dCas9 &amp; dCas9VRER validation. This we transformed our own competent cells with an equimolar mix of the pBad vector and the pSB3C5 vector. A table of all double transformant strains is shown in the result section.
  <P>In the remainder of the week we try to prove CRISPR interference by performing overnight kinetic measurements of the OD and fluorescence level of all strains. The results of this measurements were not as useful as we hoped it to be. We saw that the fluorescence per OD only started to increase at higher OD’s and the plate reader that we used could not measure such high levels of fluorescence.</P><P>We did see some minor effects primarily in the OD at which the fluorescence was too high to measure. We think that this might be caused by CRISPR-interference, but we did a follow-up experiment to get better data.</P><P>In the next experiment we induced our cells at an OD of 0.4-0.6 and let them grow while induced overnight. The next day we measured the OD and fluorescence, which  gave some really nice results. The experiment and a more detailed description is given on the results page.</P><H6>Spacer acquisition</H6>
Although we could not submit the biobrick compatible hCas9 as a biobrick anymore, we still thought it would be interesting to see if hCas9 would work. Instead of using our own construct we were planning to use the original <I>S. pyogenes</I> hCas9 operon and insert it into an inducible <I>E. coli</I> and <I>L. lactis</I> expression vector. To validate that hCas9 is working we also wanted to include the spacer array into the same vector. The vectors that we wanted to use the pBad vector, an arabinose inducible <I>E. coli</I> expression vector, and pNZ8048, a nisin inducible L. lactis expression vector. We wanted to assemble all these parts with a gibson assembly so we used overhang PCR to attach the correct homology regions.(see table). After all the parts were constructed we tried to put them together using gibson assembly. The hCas9 and backbone are approximately the same size so we decided to use equimolar amounts for those and twice as much of the smaller array insert. After transformation there were no colonies visible on the plate. After this we decided stop with the hCas9 project and focus on the dCas9 project.

<TABLE class="tableizer-table"><THEAD><TR class="tableizer-firstrow"><TH>Primers </TH><TH>Template</TH></TR></THEAD><TBODY><TR><TD><I>E. coli</I></TD><TD> </TD><TD> </TD></TR><TR><TD>hCas9</TD><TD>G90, G85</TD><TD>pRH180</TD></TR><TR><TD>array</TD><TD>G86, G91</TD><TD>Custom array 20</TD></TR><TR><TD>backbone</TD><TD>G88, G89</TD><TD>pBad</TD></TR><TR><TD><I>L. lactis</I></TD><TD> </TD><TD> </TD></TR><TR><TD>hCas9</TD><TD>G84, G85</TD><TD>pRH180</TD></TR><TR><TD>array</TD><TD>G86, G87</TD><TD>Custom array 20</TD></TR><TR><TD>backbone</TD><TD>G82, G83</TD><TD>pNZ8048</TD></TR></TBODY></TABLE><H5 id="snav3001">Week 44</H5><H6>Lactis toolbox</H6>
After constructing the promoter biobricks and expression vectors, the pNisA promoter was validated in a similar fashion as the cells of the InterLab study were examined. After growing an overnight culture at 30°C, the cells were diluted to an optical density of 0.1 in fresh GM17 medium. When the optical density reached a value between 0.5 and 0.6, the culture was split into 5 portions and induced with various amounts of nisin. The used concentrations were 1 ng/µl, 10 ng/µl, 45 ng/µl and 80 ng/µl and were administered from at stock dissolved in 10% glucose. The cells were induced for 6 hours at 30°C and a sample was taken every 45 minutes. The samples were stored in the dark and on ice until further use. Once all samples were collected, the cells were harvested by centrifugation and resuspended in chemically defined medium. The cell suspensions were analysed by two measurements. The first was an absorbance measurement at 600 nm to determine the optical density and therefore the amount of cells present. The second measurement determined the amount of expressed GFP by exciting at 489 nm and measuring emission at 512 nm.      
<CENTER><A class="button" style="text-decoration: none; color:white; margin-bottom:30px;" href="https://2017.igem.org/Team:Groningen/FutureResearch">Next: Future research</A></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>