<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Groningen_Marttest skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Groningen/Marttest</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>
    Research
</P><P><B>Project Description:</B></P><P>
    Our IMPACT-system contains cells that are capable of detecting
    bacteriophage infections. If our cells are infected with a certain phage
    they will either up-regulate or downregulate a reporter gene, e.g. GFP,
    leading to an easily detectable signal. In this section we will explain how
    we want to get this system to work.
</P><P>
    In figure 1 (same figure as on poster) a general overview of the system is
    provided. In the first step, a Cas protein complex will take up spacers
    from the invading phage DNA. In the following step the CrRNA, corresponding
    to the new spacer, is used by dCas, which is not part of the spacer
    incorporation complex, to interfere with the expression of a reporter
    construct. Our dCas9 will target the region between the transcription start
    site and the RBS by inserting a sequence similar to the spacer that is
    taken up from the phage. To determine which spacer this would be, we have
    developed a model, which will be addressed on the next page.
</P><P>
    Finally, the entire system needs to be constructed in <I>L. lactis</I>,
    which is a Lactic Acid Bacteria. Working with <I>lactis </I>requires some
special attention, it is not as easily transformed as <I>E. coli </I>or    <I>B. subtilis </I>and <I>E. coli </I>or <I>B. subtilis </I>promotors tend
    to not work in <I>lactis.</I> Moreover <I>L. lactis </I>is an important
    organism in fermentation industries, but not a lot of informations or parts
    can be found in the iGEM database. Therefore we created a fourth
    sub-project, called lactis-toolbox, in which we share the problems we
    encountered, our protocols and in which we created three new <I>lactis </I>
    promotor parts.
</P><P>
    Since our entire project involves around CRISPR-Cas we will start by giving
    a short introduction into the most relevant aspects about CRISPR-Cas before
    going into more detail on our different sub-projects.
</P><P><B>--- CRISPR Cas-----</B></P><P>
In our project we will make use of the CRISPR-Cas system of    <I>Streptococcus pyogenes</I>. This system is a type II-A and consists of
    four proteins and two RNA molecules [Wei 2015)]. The natural CRISPR-system
    is a defence mechanism against invading bacteriophages. Often only one part
    of the system, namely Cas9, is used to target specific nucleotide
    sequences. First a quick overview of the mechanism behind CRISPR immunity
    will be provided followed by a more detailed description of all components.
</P><P>
    CRISPR immunity can be seen as a three stage process (Figure 2) [Doudna Lab
    (2012)]. The first stage begins with the bacteriophage inserting its DNA
    into the host's cell. A spacer is acquired, which is a small (20 nt)
    fragment originating from the foreign DNA . This spacer is incorporated
    into the CRISPR array, which is a collection of multiple spacers flanked by
    repeat regions. This region is also responsible for the CRISPR acronym,
which stands for <B>C</B>lustered<B> R</B>egularly<B> I</B>nterspaced    <B>S</B>hort<B> P</B>alindromic <B>R</B>epeats. In the second stage of
    defence, this array is transcribed and a pre-CrRNA is produced, which is
    subsequently processed into separate CrRNA molecules. Finally, Cas9 can
    utilize the processed Cr-RNA’s together with a TracrRNA to target a
    specific sequence encoded by the spacer, bind to it and use it endonuclease
    capability to cleave it. The CRISPR array offers the cell a molecular
    memory of infection enabling it to fend of bacteriophages that match the
    incorporated spacers.
</P><P>
    The four proteins (Cas1, Cas2, Csn2 and Cas9) are expressed together in one
    operon, which in the natural system also contains the TracrRNA. In our
    project we will use a plasmid, constructed by [Heler et al] containing only
    the four proteins coding genes. Therefore all references to the CRISPR
    operon will be referring to this plasmids CRISPR version and not the
    natural organization.
</P><P>
    The TracrRNA is one of the two RNA molecules in the natural CRISPR system.
    This part directs the CrRNA (transcribed and processes from spacer array)
    to the Cas9 protein. This constructs enables Cas9 proteins sequence
    specific binding as well as cleavage. Another RNA molecule that is often
    used in combination with CRISPR-Cas is a guide RNA (gRNA). The gRNA is not
    a component of the natural CRISPR-system, but it is a synthetic construct
    that resembles the CrRNA:TracrRNA dimer.
</P><P>
    By chemically synthesizing the part of the gRNA that corresponds to the
    spacer part, it can be used to target Cas9 to any arbitrary sequence.
    Variants of Cas9 in conjunction with DNA repair mechanisms allow for
    transcription inhibition (CRISPRi), Gene knock-ins or knock-outs and many
    more functions. One restriction on the targeted sequence is that it needs
    to be flanked by a short motive (NGG in the natural spCas9), which is
    called the PAM sequence. Recognition of this sequence is a property of the
    Cas9 protein and is not orchestrated by the RNA molecule [Sternberg
    (2014)]].
</P><P>
    --- <B>Spacer Acquisition--------</B></P><P>
    The first sub-project is concerned with the spacer acquisition for which we
use a slightly adapted version of the <I>S. pyogenes</I><U>CRISPR-Cas system</U>. Instead of using the native CRISPR system we will
    use one with an hyperactive Cas9 (hCas9). Heler et al. described a single
    amino acid mutation that turns Cas9 into hCas9, which resulted in an ~100
    fold increase of the spacer uptake rate [Heler (2017)].
</P><P>
    We have decided on this hCas9-variant due to the insufficient spacer uptake
    rate of the native Cas9 protein complex. The chance that the correct spacer
    (for which we pre-programmed the cells) is take up is would be too low. The
    use of the hCas9 has a much higher chance of incorporating the appropriate
    spacer.
</P><P>
    The downside of using an hyperactive Cas9 is the possibility of it
    incorporating chunks of genomic and DNA as well as infecting phage DNA .
    When hCas9 is directed to these interior genetic elements, this will result
    in clearing of the plasmid and the corresponding antibiotic marker, which
    results in cellular death. Therefore, the hyperactive CRISPR array might
    have a severe fitness disadvantage on the cells. This lead us to using a
    different variant, dCas9, for the CRISPR interference and signal output.
</P><P><B>--- CRISPR interference ---------</B></P><P>
    For the signal we decided on using a second specialised Cas9 variant,
    namely dCas9 with a changed PAM recognition site. Both Cas9s can be
    directed by the same spacers as their guides, so use the same crRNA and
    tracrRNA structure. However, four amino acid substitutions [table …] in
    Cas9 changed its PAM preference to NGCG <U>[Kleinsteiver et al.]</U>. Our
    plan is to combine the PAM-altering mutations with those of dCas9 to obtain
    a dCas9VRER. Using this dCas9 with an altered PAM preference afforded us a
    great number of benefits.
</P><P>
    First of all, spacers taken up by hCas9 need to be flanked by an NGG on the
    original sequence. In our target array the targeted spacers are flanked by
    NGCG and therefore our target array won’t be targeted by the hCas9. Thus,
    our plasmid won’t be cleaved and the cells will stay resistant against the
    antibiotic.
</P><P>
    Next, since the hCas9 is so much more efficient in taking up spacers than
    dCas9, it will be very unlikely that dCas9 will take up a spacer, which
    would be flanked by an NGCG sequence. As a result it is not likely that our
    detector will result in a signal after a spacer resulting from the plasmid
    that is taken up. If hCas9 would take up a spacer from the plasmid the
    cells will die, as debated in the previous section.
</P><P>
    --- <B>Reporter plasmid -------</B></P><P>
    To prove that our IMPACT system we will construct a reporter plasmid in
    which the expression of GFP is repressed after we have infected our cells
    with a phage. We have chosen for this approach since it is quite easy to
    construct and we are able to measure single cell fluorescence using a FACS
    (fluorescence activated cell sorting?) machine. However a FACS is an
    advanced and expensive machine, which is something what would be a problem
    for companies. Therefore we have thought of several other reporter
    constructs which are discussed on the outlook page.
</P><P><B>--- Lactis Toolbox ------</B></P><P>
    A fourth sub-project that is not involved with the project directly is the
    lactis-toolbox. Since we want to provide a suitable product for the dairy
    industry we require incorporation of all parts into <I>L. lactis. </I>
    However, parts that work in <I>E. coli </I>or <I>B. subtilis </I>do often
    not work in <I>L. lactis</I>, promotors for example often need to be
    interchangedwhich requires specific cloning protocols. Although
    several teams have tried working in <I>L. lactis </I>before us not a lot
were succesfull and limited information and parts are available. Since    <I>L. lactis </I>is an important food-grade bacterium we want to change
    this by submitting three promotors and uploading our protocols.
</P><P>
    The first promoter is pNisA, a nisin inducible promotor, which is is one of
    the most widely used expression systems in Gram-positive bacteria[Mierau
    (2005)]. It was first characterized <I>L. lactis</I> NZ-strains in 1996 and
    became a fundamental part of the NIsin Controlled Expression system[de
    Ruyter (1996), Kuipers (1998)].
</P><P>
    The second promoter is p32. This is a constitutive promoter first
    characterized in 1987 from<I> Streptococcus cremoris </I>[van der Vossen
    (1987)]. The final promoter is pUsp45, a promoter from <I>L. lactis</I>,
    first described in 1990 [van Asseldonk (1990)]. This promoter has been
    characterized in the past using sfGFP(Bs) in <I>L. lactis</I> [Overkamp
    (2016)].
</P><P>
    For this sub-project we would like to thank our supervisor Patricia Alvarez
    Sieiro in particular since she helped us a lot with the lactis work we
    performed and provided us with our protocols.
</P><P><B>----- References: -------------</B></P><P>
    van Asseldonk, M et al. Cloning of usp45, a gene encoding a secreted
    protein from Lactococcus lactis subsp. lactis MG1363. <I>Gene</I> 95 (1),
    155-160 (1990)
</P><P>
    Doudna Lab. CRISPR systems in prokaryotic immunity. Viewed at 24-10-2017
    13:14 via
    &lt;a href=&quot;<A rel="nofollow" class="external free" href="http://rna.berkeley.edu/crispr.html">http://rna.berkeley.edu/crispr.html</A>&quot;&gt;<A rel="nofollow" class="external free" href="http://rna.berkeley.edu/crispr.html">http://rna.berkeley.edu/crispr.html</A>&lt;/a&gt;
    (2012)
</P><P>
    Heler, R. et al. Mutations in Cas9 Enhance the Rate of Acquisition of Viral
    Spacer
</P><P>
    Sequences during the CRISPR-Cas Immune Response. Mol. Cell 65, 168–175
    (2017)
</P><P>
    Kleinstiver, B. P. et al. Engineered CRISPR-Cas9 nucleases with altered PAM
</P><P>
    specificities. Nature 523, 481–485 (2015).
</P><P>
    Kuipers, OP et al. Quorum sensing-controlled gene expression in lactic acid
    bacteria. <I>Journal of Biotechnology,</I> 64 (1), 15–21. (1998)
</P><P>
    Mierau, I et al. Optimization of the Lactococcus lactis nisin-controlled
gene expression system NICE for industrial applications.    <I>Microbial cell factories</I> 4 (16) (2005)
</P><P>
    Overkamp, W et al. Benchmarking various green fluorescent protein variants
    in Bacillus subtilis, Streptococcus pneumoniae, and Lactococcus lactis for
    live cell imaging. <I>Applied Environmental Microbiology</I> 79(20),
    6481-90.(2013)
</P><P>
    de Ruyter, Pg et al. Controlled gene expression systems for Lactococcus
    lactis with the food-grade inducer nisin. <I>Journal of Bacteriology</I>,
    178 (12), 3662–3667. (1996)
</P><P>
    Sternberg, H. et al. DNA interrogation by the CRISPR RNA-guided
    endonuclease Cas9. <I>Nature</I> 507, 62-67 (2014).
</P><P>
van der VOSSEN, JMBM et al. Isolation and characterization of<I>Streptococcus cremoris </I>Wg2-specific Promotors    <I>APPLIED AND ENVIRONMENTAL MICROBIOLOGY</I> 5 (10) 2452-2457. (1987)
</P><P>
    Wei, Y et al. Cas9 function and host genome sampling in Type II-A
    CRISPR–Cas adaptation. <I>Genes &amp; Development</I>, <I>29</I>(4),
    356–361. (2015)
</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>