Team:Groningen/Design
≡ Menu
Home
Project
Description
Model
Design
Results
Notebook
Future Research
Protocols
InterLab
Human Practices
Overview
Human Practices (Silver)
Integrated (Gold)
Public Engagement
Applied Design
Future Outlook
Safety
Parts
Collaborations
Team
Team Members
Achievements
Attributions
Fun Activities
Sponsors
Introduction
hCas9
dCas9
Reporter
Lactis Toolbox
References
Design
Introduction
The model results allow us to assume with reasonable certainty which spacers are likely to get incorporated. But more broadly, what is actually required for a CRISPR based bacteriophage detection system?  First off, the bacteriophage sequence has to get incorporated into a spacer array. This requires several different Cas proteins. We worried that the natural spacer adaptation rate would be too low, so we opted for an improved, hyperactive, hCas9 protein together with Cas1, Cas2 and Csn2. The altered version displays improved features, especially concerning the speed and the quantity of spacer incorporation. We expect the likelihood of the correct spacer to be adapted to increase proportionally with an increase in total quantity of adaptation. This subsystem would already allow for resistance and mimics the natural CRISPR immune system.  But then again, we do not wish to replicate the natural CRISPR mechanism but utilize its potential for a detection mechanism.  This leads to our other subprojects, the dCas9 and the reporter system. The crRNA that the hCas9 array produces can be used by a catalytically dead Cas9 protein as well. The gRNAs allow the dCas9 to target the reporter plasmid. This plasmid contains spacer target inserts upstream of a GFP protein. Binding of the dCas9 protein will sterically hinder transcription of the GFP gene causing a detectable decrease in fluorescence. Essentially, if the matching protospacer gets incorporated into the array and then leads to a decrease in GFP fluorescence, you can infer the presence of the respective bacteriophage.  In the following paragraphs, we would like to elaborate on the manner how we envision our design and system to look like in more detail and what was achieved during our project.
hCas9
Biobrick construction
As a basis for the biobrick compatible hCas9 operon we used the Cas9 CRISPR operon from Streptococcus pyogenes. To make this into a biobrick compatible hCas9 operon we had to introduce the I473F mutation [(Heler (2017)] , remove the prohibited restriction sites it contained and attach the biobrick prefix and suffix.
To achieve this, we used a combination of synthetic DNA (gBlocks) and DNA fragments that were PCR amplified from pWJ40 (containing the Cas9 operon, plasmid map ), which was a kind gift from the Marraffini group at the Rockefeller University. All fragments were finally put together using Gibson assembly. All cloning steps were performed in E. coli DH5α.  Figure 1 gives an overview of all the cloning steps that were used.
The first gblock was used to remove the XbaI site and introduce the iGEM prefix and suffix. This part was first placed in the pSB1A3 vector to form the backbone for the Gibson assembly. The plasmid had to be linearized using PCR before it could be put together with the other fragments.
The second gblock was used to introduce the I473F mutation and remove the EcoRI site.
The PCR fragments that were amplified from the pWJ40 plasmid were designed in such a way that they already contained the overhangs for the Gibson assembly.
After Gibson assembly and transformation of the product into E. coli DH5α and isolation of the plasmid, several PCR reactions were performed with different primers to confirm that it contained the intended product. After the gels seemed to be correct we sent the plasmid for sequencing and further verification. Unfortunately, it turned out that the amplified I473F gblock part of the gene contained the wrong sequence. Due to time constraints of the parts submission deadline, we were not able to construct a new biobrick compatible hCas9 operon. We encourage next iGEM teams to pick up where we left as hCas9 seems to be a valuable addition to the iGEM registry!
Besides the hCas9 operon, the tracrRNA and the CRISPR array are also required to get a successful CRISPR response. In this project we constructed biobrick compatible tracrRNA and several pre-programmed spacers arrays. We ordered the tracrRNA as a gblock in combination with the constitutive lactococcal promoter pUSP45. The sequence of the tracrRNA gene was taken from the S. Pyogenes genome. Since the tracrRNA is functional as RNA and has no start codon, we had to precisely position the tracrRNA gene relative to the promoter.  The construction of the crRNA will be discussed in the reporter part.
Validation construction
As mentioned earlier we were not able to construct the biobrick compatible hCas9 operon. So, the experiment described next was not executed but lays out how to validate that hCas9 can acquire spacers in both Lactococcus lactis and E.coli. We intended to perform an on-plate acquisition assay that would have measured the rate at which spacers would be acquired. For this assay, we first would have inserted the hCas9 operon into an inducible expression vector. We were planning to use an arabinose-inducible pBad vector for E. coli and a nisin-inducible pNZ8048 vector for L. lactis. For comparison, we would also have included the non-hyperactive Cas9 operon. The cells would then be exposed to bacteriophages and the surviving colonies isolated. Using PCR we could then have measured the increase in length of the spacer array. The amount of surviving colonies and the sizes of the spacer arrays would have given an indication of the efficiency/activity of the hCas9 operon in L. lactis and E.coli.
dCas9
Biobrick construction
For the next part for our project we created a biobrick compatible dCas9 protein that would recognize different PAM sites than hCas9. To achieve this, we had to introduce the VRER mutations into the dCas9 gene, that were described by [Kleinstiver (2015)] .
Name
Mutations
Description
dCas9
D10A, H840A
Inactive Cas9; contains two mutations in the nuclease domains.
Cas9VRER
D1135V, G1218R, R1335E, T1337R
Mutations in Cas9 that alter the PAM binding preference from 5'-NGG to 5'-NGCG
We started out by trying to transform the dCas9 from the biobrick dCas9-Ω ( BBa_K1723000 ), which was provided in the iGEM distribution plate, as a basis. However, we were not able to recover successful transformants. Therefore, we decided to pursue another strategy utilizing the addgene plasmid pJWV102-PL-dCas9 ( plasmid map ), which was supplied to us by our supervisor Chenxi.
To start off, the dCas9 was PCR amplified out of the plasmid using primers to incorporate XbaI as well as a PstI restriction sites in front of and behind dCas9, enabling dCas9 integration into an iGEM Vector.
Next, we removed the EcoRI site, since this was still present in the Addgene plasmid and interfering with Biobrick compatibility. In order to accomplish this,  two sets of quick-change primers were designed . This created pSB1C3:dCas9QC, which is a biobrick compatible dCas9.
Although the part contains the SpeI and the PstI part at the end, it does not have the suffix, since the sequence in between these sites is different. We fixed this by restriction ligation of this part and linearized pSB1C3 with EcoRI and SpeI. The resulting part has been submitted and can be seen as an improvement on ( BBa_K1026001 ).
dCasVRER
Since all mutations are positioned at the end of the gene, a gblock was designed containing the end of dCas9 with all four mutations.  To exchange the gblock with the original end of dCas9, the BamHI restriction site was used. The end of the gblock contained the biobrick suffix. To ensure that only mutated dCas9 would be transformed, the correct fragment was subjected to a gel-extraction after restriction of the pSB1C3 dCas9 plasmid.
From this sub-project both the biobrick-compatible dCas9 ( BBa_K23610000 ) and dCas9-VRER ( BBa_K2361001 ) were submitted to the iGEM HQ.
Validation
To validate that dCas9 worked we designed an experiment in which we would be able to determine if dCas9 was capable of interfering with the expression of GFP. For this validation experiment we need to express three parts in E. coli: a constitutively expressed GFP and gRNA and dCas9 or dCas9VRER behind an inducible promoter.
For dCas9 expression we chose to use the pBad-vector in which the dCas9 was cloned behind the inducible pAra promoter. For expression of the GFP we obtained a pSB3C5 plasmid which already contained a pLac-GFP construct ( BBa_K741002 ). We used a pSB3C5 plasmid instead of one of the biobrick construct in pSB1C3, because the origin of replication of pBad and pSB1C3 are not compatible.
For gRNA expression we designed a construct similar to the one of Qi et al. . We designed four gBlocks by taking the promotor-gRNA-terminator part from the Addgene plasmid and putting it in between two overhangs, which are compatible with pSB3C5. To put the gRNA into the pSB3C5 backbone we designed a set of primers to linearize it at the PstI restriction site. This way we could use Gibson and restriction ligation coloning with PstI to insert our gBlock. The parts were ordered for synthesis from IDT, see gRNA4 gblock as an example.
The last part left to design is the spacer part of the gRNA. This was done using an online tool called ChopChop in which the sequence of the pLac-GFP construct was loaded. The tool mapped all possible spacers flanked by a specific PAM and calculated the number of off-target hits on the gDNA of E. coli-K12. Two spacers flanked by NGG and two spacers flanked by NGCG were chosen in such a way that they did not have any off-target hits.
Spacer
Sequence
NGG 1
CATCTAATTCAACAAGAATT
NGG 2
CAAAATACTCCAATTGGCGA
NGCG 1
GGTTATGGTGTTCAATGCTT
NGCG 2
GACACAACATTGAAGATGGA
Reporter
Biobrick construction
The process of creating the dCas9 and tracRNA parts is described above and they could be used without further modification. The target array for the reporter plasmid and crRNA array were custom designed and synthesized for use in this sub-project.
The crRNA array design was based on the natural S. Pyogenes CRISPR locus. The array was reduced to a single spacer and the natural promoter was exchanged for the lactococcal pUsp45 promoter. Outward facing BsaI-sites were inserted to allow the crRNA array to be easily reprogrammable by insertion of a short oligo. Furthermore, a termination sequence proven to work in L. lactis was placed after the putative S. pyogenes terminator and the whole was flanked by biobrick adapters. The whole was ordered for synthesis from IDT, see here . We submitted:
completed crArrays( BBa_K2361004 )
empty ( BBa_K2361007 )
with spacer 21( BBa_K2361006 )
with spacer 20 ( BBa_K2361005 )
.
The reporter was designed to use a fluorescent protein (sfGFP) that has been shown to show robust expression in L. lactis. The sfGFP transcript is driven by the lactococcal p32 promoter and followed by two termination sequences. Because of the limited synthesis size available to us the reporter target array and sfGFP gene “unit” could not be ordered in its completed form. We shrunk our sequence by removing a chunk of the sfGFP sequence. We designed it in such a manner that this created a new HindIII restriction site that would be lost upon insertion of the missing sequence. The missing sequence was amplified from an entry from the 2014 Groningen iGEM team ( BBa_K1365020 ). As with the crRNA array, the sfGFP reporter was flanked by a biobrick prefix and suffix.
Between the transcription start site of pUsp45 and the RBS and start codon of sfGFP two Cas9 target sequences had to be placed. These 20nt target sites, “20” and “21”, were predicted by our bioinformatics approach to be good contenders for uptake by the CRISPR operon. They were flanked by a -NGCG- PAM to allow them to be recognized by the PAM targeting mutant dCas9VRER. Like the crRNA array, outward facing BsaI sites were added to allow the target sites to be exchanged with a short oligo. Unlike the BsaI sites in the crRNA these restriction sites in the reporter target array did not interfere with its intended function. Therefore, they could be left in for the initial test and called upon later if the target array needed to be exchanged. We were able to insert the first sfGFP from the gblock into the PSB1C3 backbone, but we were not able to insert the second part before the deadline.
Validation
To experimentally validate this part, several of our other subprojects have to be completed as well. The spacer acquisition subproject has to incorporate the appropriate spacer (20/21), transcribe the crRNA together with the tracRNA which should allow the dCas9VRER to target the reporter plasmid which leads to a measurable decrease in GFP fluorescence. This experiment can be performed in a standard 96-well platereader which is able to measure GFP fluorescence, maintain the required temperature as well as measure the optical density at 600nm. Due to time concerns we were not able to perform this feat experimentally.
Lactis toolbox
Biobrick construction
As our detection system is designed to ultimately by integrated into L. lactis,  we wanted to provide the registry with the desired promoters which were not available in a pSB1C3 backbone. For each promoter, a different plasmid and primer pair was used to amplify the sequences from their native backbones. The pNisA lactococcal promoter was amplified from the pNZ8048 plasmid using the G65 and G66 primers [Kuipers (1998)] . The p32 lactococcal promoter was amplified from the pMG36E plasmid using the G67 and G68 primers [van de Guchte (1989)] . The pUsp45 lactococcal promoter was amplified from the already cloned part BBa_K2361003 using the G63 and G64 primers.  This added the biobrick restriction sites combinations to the flanks of the promoter sequence. This allowed us to incorporate the promotor sequences into the biobrick-compatible format. In the end the sequencing data for p32 & pUsp45 did not match the expected sequence, so we resorted to only submitting ( BBa_K2361009 ) and testing PnisA activity.
Validation nisin inducible promoter (BBa_K2361009)
Since iGEM requires biobrick submissions to be contained in the pSB1C3 vector, the first step of validation was converting this shipment format to a genetic setup which is suitable for protein expression. Since the sequencing data for p32 & pUsp45 did not match the expected sequence we resorted to only testing pNisA activity. As the lactococcal expression vector already contains the pNisA promoter, this was used as the backbone for our measurement setup. The most convenient way to characterize the promoter was to let it express a fluorescent protein. Therefore, GFP was cloned behind this promoter within the pNZ8048 vector. The strength of the promoter was tested by inducing the L. lactis cells with various concentrations of nisin, ranging from 1 ng/μl to 80 ng/μl. This gave us valuable insight into the capabilities of this inducible promoter.
Validation of existing parts in L. lactis
Besides the submission of new parts for L. lactis we also tested the functionality of existing promoters pLac ( BBa_K1789012 ) and pTet ( BBa_R0040 ) in L. lactis. To construct reporter parts from these promoters we assembled them together with an RBS ( BBa_B0034 ),sfGFP ( BBa_K1365020 ) or RFP ( BBa_K1323009 ) and a terminator( BBa_B0015 ) using the 3A assembly protocol. These parts were ligated into the pIL252, which is a L. lactis cloning vector. Moreover, we fused GFP and RFP in pNZ8048,an expression vector containing the pNisA promotor. However, we made a mistake in the primer design causing the start codon of GFP being located too far from the RBS.
References
Heler, R. et al. Mutations in Cas9 Enhance the Rate of Acquisition of Viral Spacer Sequences during the CRISPR-Cas Immune Response. Mol. Cell 65, 168–175 (2017).
Kleinstiver, B. P. et al. Engineered CRISPR-Cas9 nucleases with altered PAM specificities. Nature 523, 481–485 (2015)
Qi, L.S. et al. Repurposing CRISPR as an RNA-Guided Platform for Sequence-Specific Control of Gene Expression. Cell. 152(5)
Kuipers OP, de Ruyter PG, Kleerebezem M, de Vos WM. Journal of Biotechnology. 1998;64:15–21.
van de Guchte, M., van der Vossen, J.M.B.M., Kok, J. and Venema, G. (1989) Appl. Environ. Microbiol. 55, 224-228
Cloning overviews were made with SnapGene software.
Next: Results
