"
Page
Discussion
View source
History
teams
Log in
 
Team:INSA Toulouse/contenu/project/e calculus design
From 2013.igem.org
Project
Overview
Logic-Based Devices
Biological Modules
E. calculus Design
Modelling
Novel Approach
Achievements
Team
Students
Supervisors
Collaborators
School
Sponsors
iGEM Buddies
Lab practice
Notebook
Parts
Results
Human practice
Ethical aspects
Conferences
Safety
iGEM Safety
Safety in the Lab
Extras
Attribution
References
Our City: Toulouse
Glossary
Gallery
Fun facts
Project
Overview
Logic-Based Devices
Biological Modules
Input
Logic Gates
Output
Carry
Full Adder
E. calculus Design
Modelling
Novel Approach
Achievements
Team
Students
Supervisors
Collaborators
School
Sponsors
iGEM Buddies
Lab Practice
Notebook
Calendar
Protocols
Parts
Used Parts
Submitted Parts
Other Parts
Results
Human Practice
Ethical aspects
Conferences
Safety
IGEM Safety
Safety in the lab
Extras
Attribution
References
Our City : Toulouse
Glossary
Gallery
Fun facts
E. calculus design
General idea
The general idea of the project was to have distinct colonies of bacteria, each being a one bit calculator. The E. calculus strain can then be streaked several times, in the form of a thin line or circle on an agar plate, at regular intervals. The input signal (blue or red light) illuminates only one group of E. calculus at a time, allowing the full one bit calculation and the synthesis of the carry (AHL) if needed. After a certain period of time (long enough for the carry to diffuse to the next colony), illumination of the second colony can start. As the system is genetically stable, a relatively small amount of cells can be used, leading to amplification of the signal during growth of the switched bacteria. The output signal (RFP) would then be produced, stating a clear difference between the 0 or 1 state.
We also imagined that the E. calculus strain should have two different states, one for living happy state, feeding on the agar plate and dividing, not bothering about blue or red lights and a second state, dedicated to calculation. We therefore implemented a general inducer based on TetR repression and activation that would switch the bacteria to the "calculation ready" state. TetR, constitutively expressed would repress the transcription of the three receptors (blue light, red light and AHL receptors). In the presence of aTc (anhydrotetracycline) TetR would not repress anymore the promoter controlling the receptors genes. Upon illumination or when AHL is bound, each receptor will activate the transcription of the different recombinases, PhiC31 for the carry, Bxb1 for the blue light and Tp901.1 for the red light. When the recombinases are produced, they will switch their specific gates. Depending on the switch performed, the output will (or will not!) be produced.
Implementing the whole calculator in one strain
The full DNA construct, general inducer, input system, logic gates, output and carry was clearly too large to fit on a single plasmid (22 kb). Furthermore, we also imagined that switching DNA elements on a multicopy plasmid may not be completely wise, especially when our riboregulators were supposed to exert a strong control over the recombinases expression (i.e. maintaining a very low level of recombinases). Integration of some modules in the E. calculus genome was clearly the solution. We imagined that two different integration sites, one bearing AND1 and AND2, the second bearing XOR1 and XOR2 would be ideal. With this design, recombination events between XOR1 and AND1 or XOR2 and AND2 would lead to chromosomal DNA defects and cell death. The system would then be forced to recombine only the two sites belonging to the same gate.
The general inducer, input and output system as well as the carry can then be assembled on a single plasmid. The general strain would then carry the logic gates and would be stable (no recombinases produced). Upon transformation with the plasmid containing the general inducer, input, output and carry modules, the E. calculus strain would be generated, but still not able to switch before the general inducer put in the medium.
For the final design, a great help was provided by François Cornet and Caroline Schiavon from the LMGM laboratory . They gave us the MG1655 strain that contains special integration sites with two plasmids that would allow these specific integrations.
iGEM Toulouse :
INSA Toulouse
135 Avenue de Rangueil
31400 Toulouse
05 61 55 95 13
e-mail: igem.toulouse@gmail.com
Agrandir le plan
Socialize :
This project is part of iGEM
Read more on the iGEM Main Page
Contact us :
Name
Subject
Message
Leave your message here
Retrieved from " http://2013.igem.org/Team:INSA_Toulouse/contenu/project/e_calculus_design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
