"
Page
Discussion
View source
History
teams
Log in
 
Team:INSA Toulouse/contenu/lab practice/results/logic gates
From 2013.igem.org
Project
Overview
Logic-Based Devices
Biological Modules
E. calculus Design
Modelling
Novel Approach
Achievements
Team
Students
Supervisors
Collaborators
School
Sponsors
iGEM Buddies
Lab practice
Notebook
Parts
Results
Human practice
Ethical aspects
Conferences
Safety
iGEM Safety
Safety in the Lab
Extras
Attribution
References
Our City: Toulouse
Glossary
Gallery
Fun facts
Project
Overview
Logic-Based Devices
Biological Modules
Input
Logic Gates
Output
Carry
Full Adder
E. calculus Design
Modelling
Novel Approach
Achievements
Team
Students
Supervisors
Collaborators
School
Sponsors
iGEM Buddies
Lab Practice
Notebook
Calendar
Protocols
Parts
Used Parts
Submitted Parts
Other Parts
Results
Human Practice
Ethical aspects
Conferences
Safety
IGEM Safety
Safety in the lab
Extras
Attribution
References
Our City : Toulouse
Glossary
Gallery
Fun facts
Results
Objective
We designed four gates, two XOR and two AND. (here)
Our objective was to characterize these gates with the recombinases as input and RFP as output, Tp901.1, Bxb1 for BBa_K1132031 and BBa_K1132032 and FimE and PhiC31 for BBa_K1132037 and BBa_K1132038 . With the RFP as output, it will be easy to see the switch of the logic gates.
Conception
The gates have been inserted into pSB1C3, and the RFP gene inserted after the gate or inside it (depending of the gate type). For XOR2 and AND2, the T7 promoter was used and the polymerase T7 was also inserted in the construct, with a promoter, a rbs and a terminator.
We succeeded in assembling Bxb1 with a ribosome binding site and a promoter and FimE with a ribosome binding site, a promoter and a terminator.
The gates and the recombinases parts have been submitted, here is the detailed description of the parts .
We did not manage to finish the cloning of PhiC31 and Tp901.1. But,thanks to Bonnet’s help, we were able to produce the Tp901.1 recombinases directly from the original plasmids ( view other parts )and use the produced recombinases for in vitro characterization.
Results
The following table presents our plan for testing the efficiency of the recombinase-based switch and the results we should expect.
The color of each cell of the table corresponds to the expected colors of the resulting colonies.
in vitro switch of the logic gates
The basic idea is to perform the switch with a cell lysis containing the recombinase and the plasmid containing the gate to be switched. After incubation, transformation of the plasmid containing the gate allows the quantification of switched versus non switched plasmids. Experiments have been performed as described in the protocol .
XOR1
In the absence of recombinase or in recombination assays with FimE, the resulting colonies are white. Therefore, the XOR1 gate does not present any transcriptional leakage and non-specific recombinases cannot switch the door. When XOR1 was put in the presence of Bxb1, the switch occurred as attested by the presence of numerous red colonies (RFP expression).
To test the time dependency of the reaction, the reaction mixture incubated was analysed at different times, from 15 min to 2.5 hours.
From the graph above, one can see that switching occurs after 15 min. After 2 hours reaction, over 90 % of XOR1 gates were switched and the percentage of XOR1 gates switched from 2 hours to 5 hours reaction is almost constant (not shown).
In the in vitro protocol, DNAses were not eliminated. Therefore, longer times reactions also means that plasmids containing the gates can be degraded. This could explain why, when incubation lasted longer than 2 hours, the number of colonies dropped as shown in the following graph.
We also tried to switch XOR1 with the two recombinases. The XOR1 gate was tested with the Dual Controller Plasmid provided by Bonnet (KC529324) and containing Tp901.1 and Bxb1. No switch was ever obtained. This negative result might be due to low expression levels of any of the two recombinases that would lead to partial switch in in vitro conditions. Further experiments will have to be performed to understand this phenomenon.
Finally, We analysed XOR1 switched plasmids by sequencing. The sequences correspond to the expected switches.
XOR2
In the absence of recombinase or in recombination assays with Bxb1, the resulting colonies are white. Therefore, non-specific recombinases cannot switch the XOR2 gate. When XOR2 was put in the presence of FimE, 5% of the colonies were red. And the switch of the red colonies were not validate by restriction and by sequencing.  Several explanations can be made: i) low expression of FimE, ii) low activity of FimE in the tested buffer; iii) none efficient FimE sites. The red colonies observed can be due to contamination by an RFP plasmid. At the end of the summer, the Michigan team which are working on FimE too, published result about FimE sites. They validate our assumption that the FimE sites, that we used, are not efficient. They advise to use the BBa_K1077001 and BBa_K1077000 sites.
AND1
In the absence of recombinase, the colonies are white. As a result, AND1 gate does not have transcriptional leakage and non-specific recombinases cannot switch the door.
We tested the AND1 gate with Bxb1. After incubation with Bxb1, the colonies are still white (this is normal!), therefore visual inspection of the switch was not possible. The only possibility was to sequence independent clones (see genotype section). We analysed five independent colonies by sequencing, all of them were switched.
AND2
In the absence of recombinase, the colonies are white. As a result, non-specific recombinases cannot switch the AND2 gate.
We tested the AND2 gate with FimE. After incubation with FimE, the colonies are still white (this is normal), therefore visual inspection of the switch was not possible. We analysed five independent colonies by sequencing, but none was switched. As said before, FimE sites that we used, do not work well, BBa_K137008 have to be replace by BBa_K107001 and BBa_K137010 by BBa_K1077000.
In vivo switch using cotransformation
The principle of this characterization is to cotransform the gate and the recombinase in the same strain. The gate construct ( BBa_K1132032 ) is on chloramphenicol and the Bxb1 construct ( BBa_K1132027 ) is on kanamycine, we used 20 ng of each plasmid to transform E. coli and let it growth on chloramphenicol and kanamycine. Some colonies were red after transformation. However, in these conditions, the genotype of the gate (presence of the switch) is hard to verify due to the presence of two plasmids in the cotransformed strain. We therefore resorted to deeper characterization with fluorescence measurements. The positive control is a plasmid expressing constitutively RFP and the negative control is the co-transformation of XOR1 with a plasmid not containing any recombinases but on the same antibiotics resistance.
On the above graph, there is a clear difference when XOR1 is cotransformed with Bxb1 contrarily to the negative control. Analysis of the fluorescence of the strain demonstrated that the level of RFP production of the negative control is probably due to light diffusion by the cells and not real fluorescence. Therefore the in vivo switch of XOR1 is confirmed. The lower fluorescence of our strains compared to the positive control could be due to the presence of a weak promoter in front of RFP and/or the longer than usual distance between the promoter and the coding sequence.
Summary
The following table summarizes our results with the different logic gates and the recombinases. Green cells are successful experiments while gray ones did not give interpretable results and the blue ones have not been done yet.
Discussion
Our main achievement here is the demonstration that some of our designed gates can be switched using the specific recombinases. All the Molecular Biology constructions for this characterization have been successfully realized. Our characterization of the different gates demonstrates that the gates do not have any transcriptional leakage and that non-specific recombinases cannot switch the gates. The single switches performed by one recombinase were demonstrated with a quick and dirty, but amazingly efficient in vitro protocol, for AND1 and XOR1 gates. This further demonstrates the successful gate design. The absence of result with the other recombinases has been caused by wrong molecular constructions or by not ideal conditions of the in vitro protocol. For the XOR2 and AND2 gates, the absence of results is due to deficient FimE sites, the design of these gates can be used by replacing the FimE sites used in this project by other one which are working well. Our results show that the best solution for in vivo characterization of these gates would be to integrate them on the chromosomal DNA. Our in vivo results are stochastic; probably because of the plasmid copy number that can vary in the cell while, in vitro, it is difficult to conclude with negative results, because of non ideal experimental conditions that can be specific for each recombinase. We started the construction of a designed strain for integration of our four logic gates, but, because of lack of time, we preferred to concentrate on the in vitro characterizations. The gates can now be implemented for other purposes. Specifically, the logic gates switches can be seen as an elegant way to detect some events that occurred only once in a cell. If the output of this event is the production of a recombinase, the switch will occur in the cells where the event took place and the red colonies (if the RFP is used as an output) represent the population in which the gates switched the gates, or more simply said, where the event took place. As the switch is permanent, it will be genetically transmitted to the offspring of the mother cell. Hence, in this system, many events, not necessarily occurring at the same time, can be followed. The logic gates can then serve as events detectors with logical operations. For example: if we need to detect two events that have occurred within the cell (and not necessarily at the same time), this is typically an AND gate.
iGEM Toulouse :
INSA Toulouse
135 Avenue de Rangueil
31400 Toulouse
05 61 55 95 13
e-mail: igem.toulouse@gmail.com
Agrandir le plan
Socialize :
This project is part of iGEM
Read more on the iGEM Main Page
Contact us :
Name
Subject
Message
Leave your message here
Retrieved from " http://2013.igem.org/Team:INSA_Toulouse/contenu/lab_practice/results/logic_gates "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
