"
Page
Discussion
View source
History
teams
Log in
 
Team:Leicester/September2012
From 2012.igem.org
Home
Team
Project
Chemistry
Parts Submitted to Registry
Modeling
Notebook
July
August
September
Safety
Human Practices
Attributions
Bibliography
    Saturday 1st September 2012
No entry for this date.
    Sunday 2nd September 2012
No entry for this date.
    Monday 3rd September 2012
(11:00 am) Some of the team calculated the concentration of 16s extracted DNA to be run on an agarose gel and later for sequencing. The calculations allow the most efficient use of the DNA stock.
(14:00 pm) 2μl of DNA were added for samples 3, 4, 5 and 6 whereas 4μl of sample 2 DNA were added due to the stock concentration being lower. The amount of marker added was varied, which allowed the concentration of samples to be determined and the accuracy of the nano-drop spectrophotometry to be checked.
(15:45 pm) The gel was loaded as:
X
X
Marker (4μl)
2 (4μl)
3 (2μl)
4 (2μl)
Marker (2μl)
5 (2μl)
6 (2μl)
Marker (1μl)
X
X
X
X
The gel was then run for 1 and a half hours.
(16:30 pm) A group meeting was held to inform the supervisors of how the project has progressed and discuss a detailed plan for the 4th September. Plans for the future of the project were also discussed.
(17:30 pm) The gel was then transilluminated (pictured below) and the concentration of the DNA was determined for use in sequencing tomorrow (4th September 2012).
Although quite faint, the bands from the 16S are present
(09:00 am) Although some colonies from the growth curve experiment could be counted, but in general there was too much growth.
(09:15 am) The group worked on the wiki for the rest of the day, writing up the past weeks work, making sure all the details were correct and in the right order, and then attributing the members as required.
    Tuesday 4th September 2012
(09:00 am) In preparation for a meeting later in the day, members of the lab discussed pathways involved in polystyrene degradation.
(11:00 am) A meeting was held with supervisors discussing the next steps of the project, with hopes to try and engineer a BioBrick. It is suspected that genes involved in toluene degradation such as TodX, TodC1&2, TobA&B and others present in the TOD operon may also be involved in the pathway for polystyrene degradation.
(12:00 pm) Databases containing genomic sequencing and BLAST were searched to find bacteria that used proteins similar to those discussed above for PCR. Although Pseudomonas aeruginosa didn’t have any of the genes, Pseudomonas putida F1 had them all.
(15:00 pm) With the help of a supervisor, one member of the team set up a PCR reaction for 16S DNA to increase the amount of DNA in the forward and reverse directions (in separate tubes) ready for sequencing tomorrow.
Yellow and Orange colonies were plated out and 01#502 was streaked again. This allows fresh colonies to be maintained, ready for storage at the end of the project.
(16:00 pm) The wiki was updated and BLAST searching for candidate genes continued.
(09:00 am) An early start with no set experiment today. Two of the colonies were plated out to see if they are two forms of the same bacteria or simply two separate species. This required plates being made, then individual colonies picked out to streak them. Once finished it was a day working on modelling, writing protocol and other computer work before checking the results of the plates in the late afternoon.
(16:00 pm) Some spec readings were taken of the MMP broth, with the blank just being MM broth. At 600nm wavelength the results came back for the mixed culture as 0.334, and the orange culture as 0.6.
    Wednesday 5th September 2012
(14:00 pm) A meeting was held to share the achievements of each individual section of the project (modelling, lab work and chemistry). Job roles were assigned to each person to complete in labs and on the computer modelling.
(16:30 pm) More polystyrene minimal media plates were prepared, ready for plating out some colonies from CSE kits, to see whether more positive results can be obtained from other samples.
(17:00 pm) Minimal media was made from minimal broth.
(17:05 pm) The Boilate was set up with a sample of our unknown bacteria to extract the DNA, ready to set an overnight PCR reaction so that we can sequence the 16S ribosome later on this week.
(17:20 pm) The master mix was prepared for 14 PCR reactions. 270μl of the master mix was made up, enough for aliquoting 18μl into 15 reactions.
The reaction mixture contained:
171μl PCR H20
60μl HF buffer
6μl dNTP's
15μl Primer A 28f AAGAGTTTGATCCTGGCTCAGA
15μl Primer B 519R GWATTACCGCGGCKGCTG
Template DNA to be added later as it is not PCR clean
3μl DNA Pol
DNApol is stored in glycerol and required mixing and spinning down for a second. The remainder of the dNTPs were disposed of as they should not be freeze/thawed many times.  In the PCR hood, the master mix was aliquotted into 9 separate PCR eppendorfs with 18μl in each and the negative control prepared using PCR clean H20. The tubes were then taken to the bench where the DNA and diluted bacteria were added. 2μl of all samples were added to make a final volume of 20μl, including the positive and negative controls (see gel lane organization).
PCR Cycle program used was iGEM16S which is as follows:
98 degrees C - 5 min
98 degrees C - 30 seconds
50 degrees C - 30 seconds
72 degrees C - 2 minutes
72 degrees C - 5 minutes
15 degrees C - Forever ( this is for the end of the reaction)
Also the teams organic chemists have been designing a number of mechanisms. The mechanism below shows the final synthesis after much improvements. It shows the conversion of the monomer styrene being converted to lactic acid. A company called styrofoam specialise in polymerising lactic acid for easy environental breakdown. We think this route may be a viable synthetic route however due to the nature of the reagents used and conditions, there are other routes to look in to.
    Thursday 6th September 2012
(09:00 am) A 2% agarose gel was prepared to run the PCR products on. One member of the team updated the project Wiki page. Attempts were made to track down the P. putida F1 strain which has possibly been located.
(09:40 am) The gel was poured and left to set. 5μl of loading dye was added to the PCR products ready to run on the gel.
(11:15 am) The gel was loaded and run at 120 volts. Lane organisation:
1) Positive control 10ng P. aeruginosa DNA from the Maxwell prep
2) Positive control 10ng P. aeruginosa DNA from the Maxwell prep
3) Neat orange culture DNA
4) x10-1 dilution Orange culture DNA
5) x10-2 dilution Orange culture DNA
6) x10-3 dilution Orange culture DNA
7) x10-4 dilution Orange culture DNA
8) Neat Yellow culture DNA
9) x10-1 dilution Yellow culture DNA
10) x10-2 dilution Yellow culture DNA
11) x10-3 dilution Yellow culture DNA
12) 5μl Marker 100bp thermo scientific
13) Negative Bench H20
14) Negative Hood PCR H20
The Sau3AI partial digest was prepared. The experiment was run for 30 minutes as from the last test the only lanes which were digested enough were the times 5-15 minutes. A "no enzyme" control was also run.
(12:30 pm) Following the results of sequencing the 16S ribosomal DNA, a BLAST Search was conducted to find the genus of the bacteria cultured from the 01#502 CSE kit, which turned out to be a Pseudomonas of unknown species.
(13:20 pm) The agarose gel was stopped and a photo obtained from the transilluminator:
The x10-2 and lower dilutions showed no bands, but the 0 dilution and x10-1 dilutions worked well
The PCR was repeated overnight with the x10-1 orange dilution and the 0 yellow dilution with 5μl of each as the amount recovered from this gel may not be enough for the sequencing. There were two different coloured bacteria growing and this was conducted to distinguish if these are two different species of bacteria or the same one with a different morphology of colony.
To see how much DNA can be recovered, a QIAGEN gel extraction was performed and more DNA amplified overnight.
A gel was prepared for the Sau3AI digest which has completed.
(15:10 pm) The gel was cut and the samples of DNA removed and weighed. The QG buffer was added at x6 the amount in μl of the amount of agarose there was. The samples were placed in the 50 degree incubator and vortexed every 2 minutes to dissolve the agarose.
(15:30 pm) Unfortunately, the Sau3AI digest was carried out at 37 degrees rather than the intended room temperature which might have produced better bands. Therefore, the experiment was repeated. However, the gel was still run to obtain results and confirm the need to reduce the activity of the enzyme. Spectrophotometric analysis was conducted for the experiment set up on the 31st, after 2 day's extra growth; mmb with 5% poly resulted in OD600 mixed = 0.053 at a 10x dilution, 0.53 at 1x dilution. Orange was 0.072 at a x10 dilution, 0.72 with no dilution. This was higher than previous demonstrating that the bacteria was growing in the broth with no other carbon other than sugar beads at 5% concentration. To confirm this the experiment, it will be repeated with more controls totalling at 8 tubes.
(16:00 pm) A gel was loaded using the Sau3AI digest from earlier, the DNA was extracted from the earlier electrophoresis using a QIAGEN QIAquick Gel Extraction Kit and the wiki protocol was updated.
(16:30 pm) The Boilate was prepared for PCR overnight using the 16S yellow and orange colonies. Only the 5 neat yellow cultures and 5 x10-1 orange cultures will undergo PCR as these seemed to be the best. Unfortunately, it seems that there will not be enough DNA to sequence.
(17:15 pm) The Boilate was finished: centrifuging and then removing the supernatant. Dilutions were prepared while the master mix for PCR was prepared in the PCR hood.
(17:30 pm) The Sau3A1 digest was then transilluminated:
Lane organisation is: x, x, 5 µl GeneRuler 1kb DNA ladder, No DNA, DNA without Sau3A1, DNA at 0 minutes, DNA at 5 minutes, DNA at 10 minutes, DNA at 15 minutes, DNA at 20 minutes, DNA at 25, DNA at 30 minutes, x, 10 µl GeneRuler 1kb DNA ladder, x, x. Notice the smooth downward trend. The expansions in the lane size we think are due to inproper mixing.
(17:45 pm) The DNA was added to the samples ready for PCR. The program used was the iGEM16S which progresses as follows:
98 degrees C - 5 min
98 degrees C - 30 seconds
50 degrees C - 30 seconds
72 degrees C - 2 minutes
72 degrees C - 5 minutes
15 degrees C - Forever (this is for the end of the reaction).
    Friday 7th September 2012
(09:00 am) A new 2% gel was prepared for the PCR samples which were being prepared. Prep for the Sau3A1 digest at room temperature began.
(10:30 am) One of the gels was remade, as well as another 2% gel ready for the "Gelception" gel, where the team is testing the extraction method.
(12:10 pm) The PCR products were then loaded onto a gel using as much of the sample as possible (22μl). The gel was run for 2.5 hours at 120 Volts, ensuring the bands are well separated for gel extraction. The Sau3A1 prep was finished and samples placed into the freezer.
(13:00 pm) Some lab members looked at the primer designs for BioBrick.
(15:35 pm) The primers were designed, ready for ordering Monday morning. Once the gel finished, it was taken down for transilluminating. Some plates were also taken as some Pseudomonas species' fluoresce under UV light, as can benzene rings. Below is the photo produced from the gel.
Lane organisation is: Chris, fill this in
(16:00 pm) None of the plates fluoresce under UV which may have been due to the excitation wavelength being different from those attempted. The PCR gel was then wrapped up and stored for gel extraction next week. As the Nanodrop results were low for yesterday's gel extraction from the PCR (at 4.8 for Orange, 5.1 for mixed and 3.2 for yellow when the two extracts were combined), PCR will be conducted using these samples to increase the amount of DNA. Time constraints meant that the "Gelception" gel is to be run on Monday.
(16:30 pm)The finalised primer sequences were sent to a supervisor to be checked and ordered. The master mix for PCR was set up in the PCR hood.There will be 13 PCR reactions, requiring 270µl of the master mix, enough for aliquots of 18µl in 15 reactions. Reaction mixture list:
171µl PCR H20
60µl HF buffer
6µl dNTPs
15µl Primer A 28f AAGAGTTTGATCCTGGCTCAGA
15µl Primer B 519R GWATTACCGCGGCKGCTG
2µl Template DNA to be added later to each PCR tube as it is not PCR clean
3µl DNA Polymerase
DNApol is stored in glycerol and required mixing and spinning down for a second. The remainder of the dNTPs were disposed of as it cannot be freeze thawed.  In the PCR hood, the master mix was aliquotted into 9 separate PCR eppendorfs with 18μl in each and the negative control prepared using PCR clean H20. The tubes were then taken to the bench where the DNA and the positive P.aeruginosa DNA and negative bench H2O controls. All samples were added at 2µl to make a final volume of 20µl.
PCR cycle program used was iGEM16S (same as before):
This time instead of loading bacteria from a boilate reaction, a 1/100 dilution was added in of the three different gel extractions at 2µl to each tube to amplify the amount of DNA in these samples. The 1/100 samples were prepared by taking 1µl of the DNA and adding in 99µl of TE buffer. Reaction organisation:
1 = Positive P. aeruginosa control
2 = Orange 100x dilution
3 = Orange 100x dilution
4 = Orange 100x dilution
5 = Yellow 100x dilution
6 = Yellow 100x dilution
7 = Yellow 100x dilution
8 = Yellow 100x dilution
9 = Mixed 100x dilution
10 = Mixed 100x dilution
11 = Mixed 100x dilution
12 = Negative control bench
13 = Negative control hood
Yellow had the lowest stock concentration so four reactions were prepared. Due to the difficulties with the dilutions in the boilate from Wednesday, multiple reactions were run for each of the extractions in case one doesn't work.
    Saturday 8th September 2012
No entry for this date.
    Sunday 9th September 2012
No entry for this date.
    Monday 10th September 2012
(09:00 am) A 0.7% gel was prepared and DNA markers found - ”HyperLadder 1".
(10:30 am) A 2% agarose gel was loaded with DNA recovered from a previous gel from last week. The lane order was as follows: 5µl DNA HyperLadder 1 x 4µl 100bp DNA ladder x 20µl of DNA recovered from PCRed mixed colonies x 2µl DNA HyperLadder 1 x 20µl of DNA recovered from PCRed Orange colonies x 20µl of DNA recovered from PCRed Yellow colonies x 1µl DNA HyperLadder 1 x
This gel was run at 120 volts.
(11:00 am) The 0.7% gel was poured ready for running the Sau3A1 digests.
(11:50 am) The gel containing recovered DNA from a previous gel was transilluminated. It showed little band movement and the gel was run for an additional hour.
(12:15 pm) The Sau3A1 digests were loaded into wells on the set 0.7% gel. The lane organisation was as follows: x DNA HyperLadder 1 x DNA digest without enzyme DNA digest after 0 minutes DNA digest after 5 minutes DNA digest after 10 minutes DNA digest after 15 minutes DNA digest after 20 minutes DNA digest after 25 minutes DNA digest after 30 minutes x DNA HyperLadder 1 x
This will be run at 100 volts
(13:00 pm) The recovered DNA gel was retransilluminated, and the bands are a lot more spread:
There are faint bands, indicating that little of the DNA has been recovered from the last gel.
(13:20 pm) The Sau3A1 digest was checked and it was decided to leave it in for longer, though the loading dye appeared to be running slightly quicker at one end of the gel.
(13:30 pm) The primers that were designed on Friday were looked at by one of the supervisors. The adjustments add degeneracy to make them more likely to PCR out the genes that the team want to extract. BLAST searches of the FASTA data, showed that the sequence in P. putida, regardless of strain, is remarkably conserved, with only one or two bases needing any degeneracy at primer binding sites. These modified primers were sent off to a supervisor in advance of the deadline and will hopefully arrive at the end of the week, allowing PCR to begin on the genes that will be made into Biobricks.
(14:45 pm) Once the Sau3A1 digest gel showed the loading dye was a couple of inches off the end, it was taken to the transilluminator. The results are depicted below.
Despite part of the gel appearing to run more quickly, it is a decent digestion curve showing a downward trend in fragment length as the time of the digest increases.
    Tuesday 11th September 2012
(09:00 am) P. putida was streaked out onto LA to isolate single colonies. The P. putida strains were also put into LB for Maxwell prep tomorrow to extract the DNA. The 1/100 dilution PCR was prepared for running on a gel that had been stored in the fridge. The wiki was updated. A 2% gel was prepared for running the PCR reaction products.
(11:00 am) Polystyrene 'sugar' was prepared for a polystyrene-propane experiment to see whether propane, which is an expanding agent in the expansion of polystyrene, could have caused the mystery Pseudomonas species to grow rather than the polystyrene. Propane is already in the sugar before expansion and requires ventilating it for several days to remove the majority of the propane. However, if the lid is left on for a time, propane will still build up. The 'sugar' was soaked in varying concentrations of propane, mixed with minimal media broth up to 0.4 ml, completely covering the 0.5g 'sugar' that will be put in each agar dish tomorrow. 0.4, 0.2, 0.1, 0.05 and 0.00ml of propane were used to soak 0.5g of polystyrene 'sugar' overnight on a shaker at 275RPM.
The Rockethub site for the project was edited in order to make it easier to read.
(14:30 pm) The gel was loaded with 1/100 dilution the PCR reaction. Unfortunately, with one of the gels, the comb was too long and the sample went straight through, wasting the majority of the first two samples.
Lane organisation:
1 = P. aeruginosa positive control
2 = Orange 100x dilution
3 = Orange 100x dilution
4 = Orange 100x dilution
5 = Yellow 100x dilution
6 = Yellow 100x dilution
7 = Yellow 100x dilution
8 = Yellow 100x dilution
9 = Mixed 100x dilution
10 = Mixed 100x dilution
11 = Mixed 100x dilution
DNA Hyperladder 5ul
13 = Negative control bench
14 = Negative control hood
This gel was run 2.5 hours at 120V . Each sample was 20μL, except samples 1 and 2 of which only 2μL and 5μL was left respectively.
The Light yellow colony plate
The Orange colony plate
(15:00 pm) Some Corning broth solutions were made using the pentane, 'sugar' and bacteria. They were made with varying amounts of pentane: 0.00, 0.05, 0.10, 0.20 or 0.40ml. There were 4 reaction conditions prepared: with the yellow Pseudomonas culture, 0.5g of polystyrene 'sugar', with the yellow bacteria only, with 0.5g polystyrene 'sugar' only, and with neither polystyrene or bacteria. The final volume for all solutions was made up to 10ml with the addition of minimal salts medium broth. Along with the 'sugar' soaking in pentane, this was put in the shaker at 240rpm overnight at room temperature to allow time to grow. It is planned to spec analyse the broth tomorrow and if the set with polystyrene and bacteria have the same growth as those without polystyrene, but with bacteria, then pentane has little if any effect on the growth of the yellow colony.
(15:40 pm) The bacteria are loaded into the Pentane Poly experiment, while the gel extraction of the PCR reaction of the orange and yellow single 16S was done. The experiment with the Yellow, Mixed and Orange colonies were ran through a spectrophotometer, however this experiment is going to be done again in triplicate to make sure that the results are not random, as well as the pentane with poly experiment to rule out this.
(16:00 pm) The yellow bacteria was inoculated in 10 corning tubes for the pentane test.
(17:00 pm) The gel extraction is completed at the same time a gel is being made ready for tomorrow. There were a total of 22 extractions going on:
1-5 were the orange bacteria.
6-10 were the yellow bacteria which have now been combined so there is only 3 tubes to Nanodrop tomorrow, which is equal to three lanes on the gel.
11-13 were orange PCR product gel extraction x100 dilution PCR product's.
14-17 were yellow PCR product gel extraction 100x dilution PCR product’s.
18-20 were the mixed PCR product gel extraction x100 dilution PCR product's.
The positive control was unable to be extracted as not enough was loaded.
(17:50 pm) The gel was poured ready for running tomorrow. The samples will need to be Nano dropped before the gel is ran to work out the amount of the marker to load.
    Wednesday 12th September 2012
(09:00 am) The contents of our swear box and late jar were counted to pay into our account when the bank opened. The class 2 lab was also decontaminated and all of the lab equipment brought back into our own lab. We're just sending/receiving emails at the moment.
(10:00 am) The pentane experiment from last night was ran through the spectrophotometer, as there has been some obvious growth. However after thinking about it, it may not be a very conclusive experiment. The corning tubes were combined and the gel extraction's from yesterday were nano dropped so we can work out how much to run on the gel. The shoes from the environment team were given to us as a Citizen Science experiment to be analysed.
The amounts of extract to use on the "Gelception" gel were worked out.
Lane organisation and volumes:
x
x
4μL of DNA Hyperladder ( 600bp band = 48ng 400bp band = 32ng)
2μL of DNA from extract labelled 4 ( according to the Nano 32.8ng)
2μL of DNA from extract labelled A ( according to the Nano 24.6ng)
2μL of DNA Hyperladder ( 600bp band = 24ng 400bp band = 16ng )
2μL of DNA from extract labelled 8 ( according to the Nano 22.4ng)
2μL of DNA from extract labelled 10 ( according to the Nano 22.8ng)
2μL of DNA Hyperladder ( 600bp band = 24ng 400bp band = 16ng )
2μL of DNA from extract labelled 5 ( according to the Nano 21.0ng)
2μL of DNA from extract labelled 1 ( according to the Nano 17.4ng)
1ul of DNA Hyperladder ( 600bp band = 12ng 400bp band = 8ng )
x
x
(11:00 am) We realised that we should have inoculated with a set amount of bacteria in solution, rather than just a swab. We will set up a rerun this afternoon so that we can analyse whether the bacteria use polystyrene or pentane in preference.
(11:20 am) The gel is loaded and it is running at 120V.
(13:30 pm) The pentane experiment is being reran. The protocol is almost the same, though this time, we used 0.1 ml less minimal media in the tubes needing the yellow bacteria, and replacing it with 0.1 ml of a 1ml yellow colony bacteria suspension in phosphate buffer solution (PBS) to make the amounts of bacteria in the broths more constant. The gel was stopped and taken to the transilluminator:
Yet again the bands of the gel extracts are very faint. This time incomparable to the marker DNA.
As poor amounts of DNA have been successfully extracted so far, the gel extraction kit is examined and needed testing to see if it worked. This meant we had to gather and PCR more DNA.
(15:00 pm)The pentane experiment was prepared and ready to be ran. The tubes used have been put on a shaker, and started off at 240rpm. It will be sampled tomorrow morning, and then Friday morning to determine the growth. The 1/100 dilutions of the last gel extraction which we amplified the DNA of were found, and the Boilate was ran to obtain more DNA and run this in the PCR as well.
The triplicate experiment was started.
(15:30 pm) To test the Gel extraction kit, our supervisor supplied a 1kb DNA ladder of known DNA concentration to pass through the coulomb to see what % recovery there is. This time new reagents from the box were used rather than the ones we have been using to see if this is the problem.
With the minimal media prepared it is set up with 10 ml amounts in 20 corning tubes, and 0.5g of polystyrene "sugar" is set up in the necessary tubes. In preparation for further testing after staying in the shaker for enough time.
(16:00 pm) One of the members worked on degenerate primers for the rest of the afternoon.
(16:30 pm) The gel extraction was finished and the tubes put in the fridge ready to run in the morning. The master mix for the PCR was created in the PCR hood. As we are doing 13 PCR reactions, we need to make up 270ul of the master mix which is enough for aliquots of 18µl in 15 reactions. Reaction mixture list:
171µl PCR H20
60µl HF buffer
6µl dNTPs
15µl Primer A 28f AAGAGTTTGATCCTGGCTCAGA
15µl Primer B 519R GWATTACCGCGGCKGCTG
2µl Template DNA to be added later to each PCR tube as it is not PCR clean
3µl DNA Polymerase
These reagents were added then as the DNApol was in glycerol the tube was mixed and spun for a second to remove bubbles. dNTPs cannot be freeze thawed so whatever was left was thrown. After this, 18µl aliquots of the master mix were placed into 9 separate PCR eppendorfs and prepared the negative control while in the PCR hood with the PCR clean H2O. The tubes were then taken to the bench where the DNA was added, and the positive P. aeruginosa DNA and negative bench H2O controls all samples were added at 2µl to make a final volume of 20µl.
PCR cycle program used was iGEM16S which is as follows:
98 degrees C - 5 min
98 degrees C - 30 seconds
50 degrees C - 30 seconds
72 degrees C - 2 minutes
72 degrees C - 5 minutes
15 degrees C - Forever ( this is for the end of the reaction)
DNA Samples: ( use this for loading of the gel)
1 = Positive control P. aeruginosa
2 = 100x dilution of orange gel extract ( from previous)
3 = 100x dilution of mixed gel extract ( from previous)
4 = 100x dilution of yellow gel extract ( from previous)
5 = Yellow boilate
6 = Yellow boilate
7 = Yellow boilate
8 = Yellow boilate
9 = Orange boilate
10 = Orange boilate
11 = Orange boilate
12 = Negative control hood PCR water
13 = Negative control bench H2O
    Thursday 13th September 2012
(9:00 am) A new gel is made ready to load the PCR reactions and the gel extraction test.
(10:00 am) 2 more gels are poured ready for electrophoresis. The tubes from the PCR machine are removed and loaded with dye ready for the gel electrophoresis.
(10:30 am) The samples are loaded into one of the gels.
Gel lane organisation:
1 = Positive control P. aeruginosa
2 = 100x dilution of orange gel extract (from previous)
3 = 100x dilution of mixed gel extract (from previous)
4 = 100x dilution of yellow gel extract (from previous)
5 = Yellow boilate
6 = Yellow boilate
7 = Yellow boilate
8 = Yellow boilate
9 = Orange boilate
10 = Orange boilate
11 = Orange boilate
12 = Marker Hyperladder 5μl
13 = Negative control hood PCR water
14 = Negative control bench H2O
Each lane was loaded with 20μl of sample as this was the most you could accurately load.
(11:00 am) The primers are produced , ready for PCRing tonight.
(13:00 pm) We loaded and ran the gel extraction test at 120v on a 2% gel. For this you need to make dilutions of the marker DNA to 100ng/50ng total amount of DNA. Load only 100ng of the samples, taking off 10% of the total volume of the elution from the column.
(14:30 pm) Delivery arrived from NewEngland Biolabs, a new sponsor donating some competent cells as well as more DNA ladders. The gel from earlier was transilluminated:
Only the mixed DNA extract failed to work
(14:40 pm) The Boilate reaction is prepared again to extract DNA for the PCR later.
(15:00 pm) The gel extraction test is redone, while the gel extraction test from earlier was transilluminated (no gel photo as it was an old machine). The test recovered too little DNA which is why we are re ran the experiment with new reagents from a new kit. As the PE was a new set it must be the QG at fault. 1ml aliquots are taken out of the pentane experiment corning tubes that was set up yesterday. Another 1ml aliquot will be taken tomorrow to compare the differences in growth of the bacteria in different concentrations of pentane, with or without polystyrene.
(15:40 pm) A new gel is made ready for the gel extraction to be run. The boilate for the PCR which is to be loaded later is completed, while a supervisor is setting up the Maxwell prep of the DNA for the PCR as well. The reaction mixture for the PCR of the different bacteria with the different primers was calculated.
(15:45 pm) The absorbances of the aliquots at 600nm of the pentane experiment were all measured: Key: 0.XX pent – volume of pentane added to the minimal broth Bac - 100µl of bacteria suspended in PBS added to the minimal broth PS - 0.5g polystyrene ‘sugar’ added to the minimal broth Abs - absorption at 600nm Absorbance readings: 0.00 pent, Bac, PS  Abs=0.292 0.00 pent, PS       Abs=0.014 0.00 pent, Bac      Abs=0.255 0.00 pent           Abs=-0.009 0.05 pent, Bac, PS  Abs=0.317 0.05 pent, PS       Abs=0.003 0.05 pent, Bac      Abs=0.259 0.05 pent           Abs=0.001 0.10 pent, Bac, PS  Abs=0.300 0.10 pent, Ps       Abs=-0.025 0.10 pent, Bac      Abs=0.250 0.10 pent           Abs=-0.005 0.20 pent, Bac, PS  Abs=0.182 0.20 pent, Bac      Abs=0.007 0.20 pent, PS       Abs=0.250 0.20 pent           Abs=0.001 0.40 pent, Bac, PS  Abs=0.203 0.40 pent, Bac      Abs=-0.018 0.40 pent, PS       Abs=0.260 0.40 pent           Abs=0.026
The corning tubes will have another aliquot taken out tomorrow at 15:00, and absorbance will be measured again to determine growth. Hopefully the absorbance will change most in the tubes with polystyrene and bacteria in. Seeing as this is just for the yellow colonies, we will be repeating this experiment twice next week with our orange and mixed colonies.
    Friday 14th September 2012
(11:00) The gel from last night was tranilluminated:
Both gel extraction kits we trialled extracted a similar, but small amount of DNA: another kit is needed to hopefully get an improved result.
(11:20) A gel was made ready for running the overnight PCR.
(12:00) The gel was loaded and started at 120 volts with the following lane organisation: 1 x 2 5µl DNA HyperLadder 3 P. aeruginosa 4 P. aeruginosa 5 P. putida strain A maxwell prep 6 P. putida strain B maxwell prep 7 Orange colony maxwell prep 8 Yellow colony maxwell prep 9 P. putida strain A boilate 10 P. putida strain B boilate 11 Orange colony boilate 12 Yellow colony boilate 13 -ve control bench 14 -ve control hood
(14:30)The gel was transilluminated. The extractions didn't work.
Both P. putida strains, as well as the orange and yellow unknown bacteria appear to have a TodX sized band.
(15:00) The pentane experiment was continued (notation is same as thursday): Absorbance readings: 0.00 pent, Bac, PS  Abs=0.292 0.00 pent, PS       Abs=0.014 0.00 pent, Bac      Abs=0.255 0.00 pent           Abs=-0.009 0.05 pent, Bac, PS  Abs=0.317 0.05 pent, PS       Abs=0.003 0.05 pent, Bac      Abs=0.259 0.05 pent           Abs=0.001 0.10 pent, Bac, PS  Abs=0.300 0.10 pent, Ps       Abs=-0.025 0.10 pent, Bac      Abs=0.250 0.10 pent           Abs=-0.005 0.20 pent, Bac, PS  Abs=0.182 0.20 pent, Bac      Abs=0.007 0.20 pent, PS       Abs=0.250 0.20 pent           Abs=0.001 0.40 pent, Bac, PS  Abs=0.203 0.40 pent, Bac      Abs=-0.018 0.40 pent, PS       Abs=0.260 0.40 pent           Abs=0.026
    Saturday 15th September 2012
No entry for this date.
    Sunday 16th September 2012
A new way to test if the bacteria are using residual pentane/removing residual pentane from the raw polystyrene sugar to try on Monday was thought up. If this works we can confirm that the bacteria are growing with the polystyrene alone.
    Monday 17th September 2012
(08:30 am) The master mix for the PCR was created, and the PCR tubes are put in a PCR machine.
(09:30 am) Several gel extractions are set up using a new kit with a view to extracting the TodX gene later today. 2 gels are also made.
(10:00 am) The PCR was started.
(10:50 am) An experiment on the aeration of polystyrene was created by putting 20g polystyrene in 500ml duran, putting it in the hybridiser at 60 degrees, to try and vaporise residue pentane in the sugar. Pentane boils at 37oC so the hybridiser is above the boiling point. The hybridiser has no exposed elements so should be safe for the small amount of pentane left to be vaporised. Two 10g amounts of poly have also been put into tubes and sealed to act as a time zero. These can then be used in a minimal broth poly orange and yellow growth experiment as we can be sure that there is no pentane left over.
(11:00 am) The pentane experiment was finished and the results analysed. There was a clear increase in absorbance by bacteria as pentane concentration also increases. The presence of polystyrene sugar also appears to increase the bacteria's absorbance, indicating an increase in bacteria concentration.
(13:00 pm) After nano dropping, and the PCR has finished, the gel extractions were ready to be ran. The PCR program for this run was changed as there was a lot of other interference bands of other amplified areas of DNA. The extension time has been reduced to one minute so only give enough time to amplify the shorter bands.
PCR cycle program used was iGEMTOL2 which is as follows:
98 degrees C - 5 min
98 degrees C - 30 seconds
65 degrees C - 30 seconds
72 degrees C - 1 minutes
72 degrees C - 5 minutes
15 degrees C - Forever (this is for the end of the reaction)
(12:00 pm) New primers for the genes we've managed to extract are designed, that incorporate the biobrick prefix/suffixes.
(13:15 pm) Nano drop of the gel extraction was done using a different kit- zymo.
(14:10 pm) The TobB and the TodF gels were completed and loaded onto gels.  They were then ran at 120V for 2.5 hours, the gel extraction nano drop results had a lot of salt contamination, so we looked into resolving this.
(16:20 pm) The final gel extraction was complete and the remaining samples from the TodX gel were put through 2x 200μL, 1x600μL, and 1x400μL. Although the protocol said 200μL twice, the bottle of the wash buffer said between 200μL and 600μL.
(17:30 pm) The gel extractions didn't work again:
Apart from the DNA markers, there are no obvious bands in our samples.
    Tuesday 18th September 2012
(09:00 am) Set up the PCR.
(11:00 am) Prepared and loaded the PCR gel of the 16S samples from last night, running at 120V on a 2% gel for 2.5 hours.
The lane organisation is as follows: Positive P aeruginosa Orange boilate Orange boilate Orange boilate Yellow boilate Yellow boilate Yellow boilate Marker 5μL (new England biolabs 100bp) Negative bench Negative hood
(12:00 pm) The gel extractions were prepared and loaded. Another 10μL of distilled water was run through the coulomb to see if there was any DNA left, so there are two lots of gel extractions being run. The gel was ran at 120V on a 2% gel.
The lane organisation is: 4μL marker (New england Biolabs 100bp) Orange boilate (9) extraction P. putida maxwell A(6B) extraction P. putida Maxwell A(6A) extraction 4μL marker (New england Biolabs 100bp) P. putida Maxwell B(5B) extraction P. putida Maxwell B(5A) extraction Orange boilate (9) extraction number 2 P. putida maxwell A(6B) extraction number 2 P. putida Maxwell A(6A) extraction number 2 2μL marker (New england Biolabs 100bp) P. putida Maxwell B(5A) extraction number 2 P. putida Maxwell B(5B) extraction number 2 1μL marker (New england Biolabs 100bp)
(13:40 pm) Stopped the PCR gel and the gel extraction gel:
The gel extraction worked! with bands for all but 3 of the extractions. Band number 9 is quite large at over 1.5kb.
After working out the concentrations it looks like we have: Number 9 = 1.8ng/μL Number 6A = 2.5 ng/μL Number 6B = 1.8ng/μL Number 5A = 5ng/μL Number 5B = 4ng/μL Number 9 = (extract 2) band just visible, but nothing of same intensity to compare Number 6B = (extract 2) band just visible, but nothing of same intensity to compare Number 6A = (extract 2) band just visible, but nothing of same intensity to compare
(14:00 pm) From this the samples for a sequencing reaction going to be run over night were made up. For the sequencing only the reverse primer was to be used. The primer was diluted three fold from the 10μM concentration, and 2μL of each of the 5A and 5B were sequenced as these were the best extracts.
(14:30 pm) After analysis of the 16S PCR from last night, it shows contamination in the negative controls, both bench and hood. Tis was done using new polymerase, DNTP's and buffers so the contamination was either in the primers or PCR water. As we ran out of PCR water this was going to be fresh for the next set of reactions, however the TodC1 used the same water so we can compare when this is run on the gel.
(15:00 pm) After preparing the TodX PCR samples from this morning with the old polymerase (had a slight problem so the TodC1 and TodG PCR reactions used fresh polymerase). We used this to test the polymerase for contamination from the 16s experiment. Next 5μL of loading dye was placed into each sample and two new gels are made ready for these to be run, with the 3rd gel in the gel tank.
(16:20 pm) The TodX PCR gel from this morning was loaded and ran at 120V.
The lane organisation is: 1 P. aeruginosa 2 P. aeruginosa 3 P. putida strain A maxwell prep 4 P. putida strain B maxwell prep 5 Orange colony maxwell prep 6 Yellow colony maxwell prep 7 P. putida strain A boilate 8 P. putida strain B boilate 9 Orange colony boilate 10 Yellow colony boilate 11 5μL marker (New england Biolabs 100bp) 12 Negative control bench 13 Negative control hood
(17:00 pm) The reamplification on the TodX Gel extraction was set up, and as there were only 5 samples the PCR was only set up for 10 to save on reagents, only doing each one once and the controls. The master mix for the PCR was made in the PCR hood. For the 9 PCR reactions, we needed to make up 180ul of the master mix which is enough for aliquots of 18ul in 10 reactions.
The reaction mixture list:
112ul PCR H20
40ul HF buffer
4ul dNTPs
10ul Primer A TODXF 5'-atgcccgccagtctgacgcttg-3'
10ul Primer B TODXR 5'-accagccagcaccatgcggc-3'
Template DNA to be added later as it is not PCR clean
2ul DNA Polymerase
As the DNApol was stored in glycerol the tube was mixed and spun for a second to remove bubbles. Then the reagents were added. dNTPs cannot be freeze thawed so whatever was left was thrown. After this the 18ul aliquots of the master mix were seperated into 9 PCR eppendorfs. The negative control was prepared in the PCR hood with the PCR clean H2O. The tubes were then taken to the bench where the DNA was added, and the positive P. aeruginosa DNA and negative bench H2O controls all samples were added at 2ul to make a final volume of 20ul.
The cycle program used was iGEMTOL which is as follows:
98 degrees C - 5 min
98 degrees C - 30 seconds
50 degrees C - 30 seconds
72 degrees C - 1 minutes
72 degrees C - 5 minutes
15 degrees C - Forever ( this is for the end of the reaction)
The italicised parts are the cycles which were repeated 30 times.
(17:40 pm) We loaded the PCR and prepared the samples to be ran over night on the gel at 20V. As there will be lots of gels in the morning, we're thinking this is the best use of time. Finally the orange and yellow colonies were restreaked.
The lane organisation was the same for both TodC1 and TodG: x 1 P. aeruginosa 2 P. aeruginosa 3 P. putida strain A maxwell prep 4 P. putida strain B maxwell prep 5 Orange colony maxwell prep 6 Yellow colony maxwell prep 7 P. putida strain A boilate 8 P. putida strain B boilate 9 Orange colony boilate 10 Yellow colony boilate 11 5µl DNA HyperLadder 12 Negative control bench 13 Negative control hood
(17:55 pm) The gel was stopped and transilluminated:
It appears that TodX could be present in both P. putida strains as both A and B have bands close to the target 544bp target fragment.
Before leaving the overnight gels was loaded and ran at 20 volts.
    Wednesday 19th September 2012
(09:00 am) All of the gels were stopped. After transilluminating, it was realised that the power pack was set too high (120v instead of 20v), so all of the DNA had run off the end of the gel:
This meant that once the PCR has been stopped we needed to re-run these primers to re extract the DNA from yesterday. As all of the DNA had run off the gel, it is remelted so we can reuse it. The gel is remade to run some of the TodX Gel extraction PCR amplification from last night's PCR. This time only single bands should have been amplified, so we did a PCR purification rather than run it all and gel extract due to the low yields.
(10:15 am) The first gel was made after a few problems with the melting and the PCR for the TodC1 and TodG samples was set up. When these were done we can ran 2µl of last nights PCR to see what bands are present.
(10:30 am) Next the master mix for the PCR of the TodG and TodC1 was set up in the PCR hood. As we did 12 PCR reactions, we needed to make up 270ul of the master mix which is enough for aliquots of 18ul in 15 reactions. Reaction mixture list:
171ul PCR H20
60ul HF buffer
6ul dNTPs
15ul Primer A
15ul Primer B
Template DNA to be added later as it is not PCR clean
3ul DNA Polymerase
As the DNApol was stored in glycerol the tube was mixed and spun for a second to remove bubbles. Then the reagents were added. dNTPs cannot be freeze thawed so whatever was left was thrown. After this the 18ul aliquots of the master mix were seperated into 9 PCR eppendorfs. The negative control was prepared in the PCR hood with the PCR clean H2O. The tubes were then taken to the bench where the DNA was added, and the positive P. aeruginosa DNA and negative bench H2O controls all samples were added at 2ul to make a final volume of 20ul.
PCR cycle program used was iGEM16S which is as follows:
98 degrees C - 5 min
98 degrees C - 30 seconds
50 degrees C - 30 seconds
72 degrees C - 1 minutes
72 degrees C - 5 minutes
15 degrees C - Forever (this is for the end of the reaction)
The italicised text was repeated 30 times.
(11:30 am) The DNA was placed into the PCR reactions ready to run the PCR. A gel is remade for running. Had a slight problem with the gel with a spillage so the lab had to be cleared up making sure there ws no ETBR on the surfaces. More gel had to be remade to replace the spilt stuff.
(12:00 pm) The TodX gel extraction re amplification gel samples were set up. As the team decided to do PCR purification rather than gel extraction, only 2μL of the 5 reactions were loaded so we needed 18 uL of DNA to purify. Samples were made by adding 4μL of loading dye to 14μL of TE buffer and 2μL of each of the sample. The P. aeruginosa and negative controls had 5μL of loading dye added and were loaded completly as they were more to test the PCR rather than for extraction of the DNA.
(12:40 pm) The gel was loaded and ran at 120V.
The lane organisation was: x Negative P. aeruginosa Negative P. aeruginosa 5μL marker (New england Biolabs 100bp) Number 6A Number 6B Number 5A Number 5b Number 9 5μL marker (New england Biolabs 100bp) Negative hood Negative bench x x
(14:00 pm) Gel has been transilluminated:
It appears 6B and 5B have fragments close to the TodX target length of 544bp.
The final PCR purification of the 18uL of the PCR left.
(15:00 pm) The PCR samples set up this morning are loaded and ready to be ran. We needed this for the insertion of the DNA into the biobrick. The purified PCR samples were Nano dropped, with maximum of 47ng/ul, minimum of 32ng/ul for the 5 samples so great results for the TodX PCR purification (QIAGEN).
(15:50 pm) All of the samples were loaded on the gels and set running at 120V until the end of the day
The lane organisation was the same for both gels: 1 P. aeruginosa 2 P. aeruginosa 3 P. putida strain A maxwell prep 4 P. putida strain B maxwell prep 5 Orange colony maxwell prep 6 Yellow colony maxwell prep 7 P. putida strain A boilate 8 P. putida strain B boilate 9 Orange colony boilate 10 Yellow colony boilate 11 5µl marker (New england Biolabs 100bp) 12 Negative control bench 13 Negative control hood
(17:15 pm) The aerated polystyrene experiment was continued, with spectrophotometry of all the samples.
(17:30 pm) The master mix for the PCR 16S orange and yellow was set up as we were donated more universal 16S primers. This was done in the PCR hood. As we had 9 PCR reactions, we need to make up 270ul of the master mix which is enough for aliquots of 18ul in 15 reactions.
The reaction mixture list was:
112µl PCR H20
40µl HF buffer
4µl dNTPs
10µl Primer A
10µl Primer B
Template DNA to be added later as it is not PCR clean
2µl DNA Polymerase
As the DNApol was stored in glycerol the tube was mixed and spun for a second to remove bubbles. Then the reagents were added. dNTPs cannot be freeze thawed so whatever was left was thrown. After this the 18ul aliquots of the master mix were seperated into 9 PCR eppendorfs. The negative control was prepared in the PCR hood with the PCR clean H2O. The tubes were then taken to the bench where the DNA was added, and the positive P. aeruginosa DNA and negative bench H2O controls all samples were added at 2µl to make a final volume of 20µl.
PCR cycle program used was iGEM16S which is as follows:
98 degrees C - 5 min
98 degrees C - 30 seconds
50 degrees C - 30 seconds
72 degrees C - 2 minutes
72 degrees C - 5 minutes
15 degrees C - Forever ( this is for the end of the reaction)
The italicised text was repeated 30 times
(17:50 pm) After the PCR was set up, the other gels were stopped and transilluminated. The results showed two bands at about the correct position.
    Thursday 20th September 2012
(09:00 am) Gels are made first thing this morning as lots needed to be ran.
(10:00 am) A new boilate was ran as last nights gels hasn't amplified any bands.
Below is the list of genes we have been trying to extract, with their respective sizes. Underneath the genes are all the bacteria we have used, and the size bands that have been produced from the PCR amplification. These show us which bacteria probably have the genes we have been looking for.
TodC1 - 1353bp > P. aeruginosa - Band at 800bp and 1kb > Maxwell reactions: > P. putida A and B - Band at 800bp, 1.2kb and one in-between 1.2 and 1.5kb > Orange bacteria - One band high up above the 1.5kb marker > Yellow bacteria - No bands > Boilate reactions: > Boilates failed to amplify
TodG - 807bp > P. aeruginosa - Band at 1kb and one high up > Maxwell reactions: > P. putida A and B - Band at 700bp and a fainter one at 800 and 600. Bands also present above the 1.5kb marker > Orange bacteroa - Good sized band at 800bp and a lower one at 700bp. Also has bands at 1.2 and 1.5 kb > Yellow bacteria - Faint band at 800bp and one at 1kb > Boilate reactions: > Boilate failed
TodX - 544bp > P. aeruginosa - None > Maxwell reactions: > P. putida A and B - Bands at <500bp, 600bp, 1200bp and >1500bp > Orange bacteria - 1 band at >1500bp > Yellow bacteria - Faint bands at 1200bp and >1500bp > Boilate reactions: > P. putida A and B - Faint bands at <500bp, 600bp, 1200bp > Orange and yellow boilates failed.
TodB - 324bp > P. aeruginosa - None > Maxwell reactions: > P. putida A and B - None > Orange bacteria - Several non distinct bands ranging from >300bp to 800bp, 2 are between 300bp-400bp > Yellow bacteria - Several non distinct bands ranging from >200bp to 1500bp, 1 faint band is between 300bp-400bp > Boilate reactions: > P. putida A and B - No distinct bands > Orange bacteroa - One faint band between 300bp-400bp and more faint bands between 400bp-1000bp > Yellow bacteria - None
> TodF - 460bp > P. aeruginosa - None > Maxwell reactions: > P. putida A and B - 1 band between 700bp-800bp > Orange bacteria - 1 band at 1500bp and 1 band between 1200-1500bp > Yellow bacteria - Faint band between 500bp-600bp, other bands are above > Boilate reactions: > P. putida A and B - Faint band between 500bp-600bp, other bands are above, 1 strong band between 700bp-800bp > Orange bacteria - Strong bands between 1000bp-1500bp > Yellow bacteria - Faint band between 500bp-600bp, and strong bands >1200bp
(11:00 am) The gel had set, so it was loaded for the 16s gel and the test of the PCR purification, loading (10%) 3µL of each of the samples with 4µL of loading dye and 13µL of TE buffer. The gel is run at 120V for 2 hours.
The lane organisation is: 5µL 100bp Marker, (New England Biolabs) 6a 6b 5a 5b 9 1µL 100bp marker (New England Biolabs) 10µL 100bp marker (New England Biolabs) The different amounts of DNA ladder were to see if the concentrations from the Nano Drop were correct.
(11:40 am) The 16s gel was stopped and transilluminated after being ran for a hour. Surprisingly there were bands for all of the reactions, however there were also bands in the negative bench and a lighter band in the negative hood controls. Because we used fresh primers, DNTp's, and there was no contamination last time we used the pol and reaction buffer the only logical explanation is the PCR H2O or technique. This meant we needed to obtain a fresh stock of PCR H2O for the next set of reactions. This sets the team back again.
The problematic 16S gel where the negatives are contaminated and appear on the gel
(13:30 pm) The gel purification gel was stopped which was transilluminated. There is DNA in the purification and it is high ng/ul. We have just got the sequence data back from the DNA and it looks like we haven't extracted the TodX gene itself. The sequence was from the reverse primer, we re did this with the forward primer to get the full set of data.
(14:30 pm) PCR was prepared for the 16s, TodX and TodF genes. At the same time the original PCR of the TodF gel was running at 150V, and stopped and transilluminated every 15 min.
(15:00 pm) A new master mix for the PCR 16S orange and yellow bacteria is created. This was done in the PCR hood. This time 9 PCR reactions are done, so we needed to make up 180µl of the master mix which is enough for aliquots of 18ul in 10 reactions. Reaction mixture list:
112µl PCR H20
40µl HF buffer
4µl dNTPs
10µl Primer A
10µl Primer B
Template DNA to be added later as it is not PCR clean
2µl DNA Polymerase
As the DNApol was stored in glycerol the tube was mixed and spun for a second to remove bubbles. Then the reagents were added. dNTPs cannot be freeze thawed so whatever was left was thrown. After this the 18ul aliquots of the master mix were seperated into 9 PCR eppendorfs. The negative control was prepared in the PCR hood with the PCR clean H2O. The tubes were then taken to the bench where the DNA was added, and the positive P. aeruginosa DNA and negative bench H2O controls all samples were added at 2ul to make a final volume of 20ul.
PCR cycle program used was iGEM16S which is as follows:
98 degrees C - 5 min
98 degrees C - 30 seconds
50 degrees C - 30 seconds
72 degrees C - 2 minutes
72 degrees C - 5 minutes
15 degrees C - Forever (this is for the end of the reaction)
The italicised text was repeated 30 times
The TodX and TodF which PCR under different conditions were set up and as there are 12 reactions, the mixture is for 15. Program used was iGEMTOL2 - used previously.
(15:50 pm) The spectrophotometry of the aerated polystyrene experiment was done.
(16:30 pm) Running Zymogen gel extraction for the TodF gel.
(17:30 pm) Prepared for tomorrow by making gels.
    Friday 21st September 2012
(08:50 am) The samples for the 16S PCR were made to be ran on a gel, while two new gels are made ready to run the Tod PCR's.
(09:30 am) The gel was loaded and ran at 120V.
The lanes were organised as: Positive P. aeruginosa Orange boilate Orange boilate Orange boilate Yellow boilate Yellow boilate Yellow boilate Marker 5μL (new England biolabs 100bp) Negative bench Negative hood
(10:20 am) The samples for the TodX and TodF PCR reactions were prepared and made ready to run on the gel.
(11:00 am) The gels were loaded and started at 120V, using the TodF and TodX PCRed out last night.
The lane organisation is: 1 x 2 P. aeruginosa 3 P. aeruginosa 4 P. putida strain A Maxwell prep 5 P. putida strain B Maxwell prep 6 Orange bacteria Maxwell prep 7 Yellow bacteria Maxwell prep 8 P. putida strain A boilate 9 P. putida strain B boilate 10 Orange boilate 11 Yellow boilate 12 5µl 100bp DNA ladder 13 bench H2O 14 hood H2O
(11:30 am) The 16s gel was transilluminated:
(13:30 pm) The TodX gel was transilluminated:
The TodF was on a longer gel, so it was not ready to transilluminate yet.
(14:00 pm) The TodF gel was transilluminated, however it didn't look like we had any of the gene we targeted in any of the DNA samples.
(15:00 pm) The gel extract experiments are started from the PCR gels ran today.
(17:00 pm) The gel extracts didn't go very well, so the team started preparing the PCR. The master mix is created for the PCR 16S orange and yellow. This is done in the PCR hood. As we were doing 9 PCR reactions, we needed to make up 180ul of the master mix which was enough for aliquots of 18µl in 10 reactions.
The reaction mixture list:
112µl PCR H20
40µl HF buffer
4µl dNTPs
10µl Primer A
10µl Primer B
Template DNA to be added later as it is not PCR clean
2µl DNA Polymerase
As the DNApol was stored in glycerol the tube was mixed and spun for a second to remove bubbles. Then the reagents were added. dNTPs cannot be freeze thawed so whatever was left was thrown. After this the 18ul aliquots of the master mix were seperated into 9 PCR eppendorfs. The negative control was prepared in the PCR hood with the PCR clean H2O. The tubes were then taken to the bench where the DNA was added, and the positive P. aeruginosa DNA and negative bench H2O controls all samples were added at 2µl to make a final volume of 20µl.
The tubes were placed into the PCR block and set going.
PCR cycle program used was iGEM16S which was as follows:
98 degrees C - 5 min
98 degrees C - 30 seconds
50 degrees C - 30 seconds
72 degrees C - 2 minutes
72 degrees C - 5 minutes
15 degrees C - Forever ( this is for the end of the reaction)
The italicised text was repeated 30 times .
    Saturday 22nd September 2012
University of Leicester open day today, the team had a stall outside our building to try and raise some money as well as helping out on the department tours.
    Sunday 23rd September 2012
No entry for this date.
    Monday 24th September 2012
(09:30 am) The group met in the computer room to start a day of editing the Wiki.
(10:00 am) The 16S PCR had finished so the samples were prepared to run on a gel. As there was 20μL of sample 2μL was loaded to make calculations easier, the remaining 18μL was to be put through the QIAquick QIAGEN PCR Purification (Cat. No. 28104). Protocol for the PCR purification can be found on the QIAGEN website, the only alterations to the protocol were step 4 and 7 leaving them for 5 minutes each time before the centrifugation step.
(12:00 pm) The gel was poured then loaded with the PCR products.
The lane organisation is as follows: P. aeruginosa Orange Orange Orange Yellow Yellow Yellow Marker 5μL (NEB) Negative hood Negative bench
(12:40 pm) A nano drop was performed on the PCR purification products to work out rough ng/μL for each of the samples. The 260/280 ratios were all around the 1.4 mark with concentrations being between 36 and 46ng/μL for the seven samples. Then the samples were prepared ready to be run on a gel. To work out the concentrations more accurately using 3μL ( 10%) of the 30μL total elution volume making each up to 20μL to be run on the gel. A new 2% gel was made ready for this to be run after lunch.
(13:00 pm) 3.3nM aliquots of the 16S primers were prepared for the sequencing reaction in the PCR hood. This was done by diluting 10nM primers 1:2 with PCR H2O. Once this had been done the samples were given to a supervisor ready to be run with forward and reverse primers.
(14:00 pm) We stopped and transilluminated the gel, the bands were in the correct place for the 16S sequences however there was contamination again in the negative controls. The gel was loaded with the PCR purification products. The lane organisation was: x 1μL 100bp Marker (NEB) 4 7 1A 2μL 100bp Marker (NEB) 5 6 4μL 100bp Marker (NEB) 2 3 8μL 100bp Marker ( NEB)
    Tuesday 25th September 2012
(08:50 am) A day of editing and completing the wiki for everyone before the freeze.
(10:00 am) Everyone continued working on the wiki and yesterdays gel was transilluminated.
    Wednesday 26th September 2012
(09:00 am) Another full day of editing for the team, continuing after 6pm in the library to make sure everything was completed.
    Thursday 27th September 2012
No entry for this date.
    Friday 28th September 2012
No entry for this date.
    Saturday 29th September 2012
No entry for this date.
    Sunday 30th September 2012
No entry for this date.
[edit]
Retrieved from " http://2012.igem.org/Team:Leicester/September2012 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
