"
Page
Discussion
View source
History
teams
Log in
 
Team:Leicester/August2012
From 2012.igem.org
Home
Team
Project
Chemistry
Parts Submitted to Registry
Modeling
Notebook
July
August
September
Safety
Human Practices
Attributions
Bibliography
    Wednesday 1st August 2012
(9:30 am) Several members of the team are given wrong start times, so only 1 student arrived in the lab at 9:30, however with the work needed to be done all the rest of the team are quickly on their way.
(10:30 am) With everyone now in the lab it turns out a lot of the work planned for today has to wait until tomorrow. This is due to the Pseudomonas aeruginosa strains needed to be grown in a rich luria broth before we can spin it into a pellet and then run the DNA extraction.
(11:40 am) With all the bacteria placed in the 15ml corning tubes, and placed in the orbital shaker all of today's lab work is complete. Now the team is going to finish the recording for the rockethub video as some scenes need to be retaken, then do some individual research/work.
(13:30 pm) After consulting with a supervisor and having a long meeting over lunch the team has decided several directions that is needed to go in. One member is testing the cooling times of a water bath that will be used in the hybridizing process of making our DNA libraries, and to start selecting out of the DNA genes that are in both our P. aeruginosa and the NB26 strain that definitely degrades polystyrene. A couple of members are looking through protocol and methods to use with the DNA extraction.
    Thursday 2nd August 2012
(9:30 am) An early start finishing all the protocol and work needed to start making the DNA libraries.
(10:00 am) ''E. coli'' is placed into 7ml of LB media and put into the warm room to grow overnight and be used as the test tomorrow to check the DNA extraction process before using the P. aeruginosa species.
(11:30 am) The protocol for tomorrow's practicals is determined, for the test of the DNA extraction, as well as dilutions to check the growth so there is a measurable amount that the P. aeruginosa can be compared to, so in future to measure the amount of P. aeruginosa the team has it can simply be spectroscopically analysed.
(4:00 pm) Made the E. coli into a 50ml broth to grow overnight for the dilutions and spectrophotometry.
    Friday 3rd August 2012
(9:00 am) Action stations. All lab work is a go. Today there are 2 experiments going: the DNA extraction test and the dilutions to test the method works. Once these are done, the team knows what works and doesn't work for the real experiments on P. aeruginosa early next week.
(9:30) moved all equipment necessary into the other lab as we cannot use P. aeruginosa, set up water baths for the incubation’s at 37 and 65 degrees that will be needed later in the day. spinning down our P. aeruginosa to get a pellet of cells, we removed the supernatant and re span to get the most concentrated pellet. After this we re suspended them in p1 buffer before adding some of the NaCl and lysozyme to lyse the cells. incubated at 37 degrees for 30 minutes the solution should go clear and viscous. solution has gone clear and viscous... success. Addition of 10% SDS and proteinase K to the solution, this will strip the DNA of bound proteins and remove secondary and tertiary structures with in the proteins. incubation for another 30 minutes to allow the reagents to act on the DNA in solution. The addition of 5M NaCl to precipitate out the DNA out of solution. Due to time restrictions our gel will be run tomorrow, the gel itself has been moved out of the tank and into the cold room and is covered. The nano drop will be run tomorrow as well, leaving time for our sample to re suspend by leaving over night at room temperature. Tomorrow Run nano drop, load and run gel.
(10:30 am) The half doing the dilutions to make the growth curve left our dedicated bit of lab space and headed into one of the supervisor's labs with a good spectrophotometer and space to work with the bacteria.
Protocol:
1. Take a reading of the overnight broth.
2. Innoculate a new conical flask of 49.9ml luria broth, with 0.1ml of overnight culture.
3. Put 1ml of culture into a spectrophotometer and take a reading at 600nm. Record this as time = 0. Start the timer.
4. At same time, take an aliquot of 100ul and put in eppendorf marked as time 0, 10^-1 dilution. Add 900ul of buffer and mix, then take 100ul from this solution and add to next eppendorf, marked time 0, 10^-2 dilution and so on to 10^-9.
5. Pipette 100ul of dilutions 10^-4 to 10^-9 onto similarly labelled pre-prepared agar plates and spread them.
6. Repeat step 2 every 40 minutes and step 4 every 2 hours (3 divisions of E. coli) until the end of time.
7. Leave plates to grow, then count the number of colonies growing on the most concentrated plate where colonies are distinguishable. Multiply up to the true concentration, then multiply by 10. This is colonies/ml.
8. Draw graph to see correlation and check against known correlation to see if it has worked.
(16:30 pm) The dilutions and CFU count part of the experiment is finished, there is 6 hours of readings for the CFU count as well as plates for 0, 2 and 4 hours.
A polystyrene plate filled with minimal media
    Saturday 4th August 2012
No entry for this date.
    Sunday 5th August 2012
The only thing needed to be done today is help the technician gave the team. This involved producing ''E. coli'' overnight cultures for the DNA extraction and digest.
    Monday 6th August 2012
(9:30 am) Today the team is going to extract the genome and start the DNA digest of the ''E. coli'' to test the DNA extraction kit and procedure that the team has identified to use.
This is a big step that if works, can then be applied to the P. aeruginosa strains that are growing. So that once the NB26 strain arrives it can be compared straight away to give an idea which genes could be involved in the degradation of polystyrene and to help start narrowing our search.
The overnight cultures of E. coli were then analysed using the spectrophotometer to give an initial bacterial cell count before the digest.
(12:30pm) The results for Friday's experiment is back. Unfortunately the data that was collected is not enough to create a graph to simply show us what the bacteria cell count is from reading an absorbance.
The new protocol for today's experiment that will tell the team this data is:
1. Take a reading of the overnight broth.
2. Inoculate 49.9ml of broth with 0.1ml of the overnight bacteria colony. Mix, start timer, and immediately do 3 and 4.
3. Put 1ml of culture into a spectrophotometer and take a reading at 600nm. Record this as time = 0. Start a timer.
4. At same time, take an aliquot of 100ul and put in eppendorf marked as time 0, 10^-1 dilution. Add 900ul of buffer and mix, then take 100ul from this solution and add to next eppendorf, marked time 0, 10^-2 dilution and so on to 10^-7.
5. Pipette 100ul of dilutions 10^-5 to 10^-7 onto similarly labelled pre-prepared agar plates and spread them.
6. Repeat steps 3-5 every 40min (1 doubling of E. coli) (labelling time appropriately) until the end of time.
7. Leave plates to grow, then count the number of colonies growing on the most concentrated plate where colonies are distinguishable. Multiply up to the true concentration, then multiply by 10. This is colonies/ml.
8. Draw graph to see correlation and check against known correlation to see if it has worked.
(13:30pm) After finally finishing all the reading up and gathering of ingredients, the DNA extraction process is finally started. Protocol is as mentioned in the QIAGEN Genomic DNA Handbook.
Almost as soon as it was started the first few sections were complete and the cells needed to be left in a 37oC water bath for a while so the team members had to wait again. And then had to wait again after adding different buffer while the cell extract was left at 50oC.
(15:00) The current step is using the genomic tip protocol to extract DNA. Unfortunately, the flow rate is very slow so this may take a while until the DNA can be washed and eluted.
(17:30) Today's refinement of the growth curve experiment worked. Well at least the protocol did, the actual experiment failed but after talking to the supervisor in a group meeting, the team realized what went wrong so it would work tomorrow. Just before finishing, 3ml of luria broth was inoculated with a single colony of E. coli as well as the plates being made up.
    Tuesday 7th August 2012
(8:00 am) A very early start for part of the group working on fixing the dilutions experiment. In for 8am, and started at 8:30 for a long day's practical.
(14:30 pm) The final set of plates and spectrophotometry readings have been taken. Now they can grow overnight to see if it works.
    Wednesday 8th August 2012
(9:30 am) The plates had grown, but unfortunately they had all grown a ridiculous amount. It turns out that the luria broth had been contaminated with some bacteria. The group didn't realize this as it was thought some of the nutrients had suspended like what had happened to our minimal media beforehand. This means another experiment must be done to see if it can finally be done right.
(11:30 am) The final dilutions experiment was started, but it needed time to grow out of lag phase. Just before lunch it still hadn't started to grow so the group was starting to despair, but after a long lunch break it had finally started to grow.
(14:30 pm) The final series of plating and spectrophotometry has started, only time will tell if it is finally right.
    Thursday 9th August 2012
(11:00 am) It worked! The final dilutions experiment came back with close to expected results so the plates could be counted and a graph made to show the cell count against absorbance.
    Friday 10th August 2012
While the lab leader was away the iGEM team carried on with business as usual.
However it was a different kind of business as the rest of the group decided to work on the Rockethub video, as it was getting late in the time the group had left to complete the video, so there is spare time to edit and so forth.
    Saturday 11th August 2012
No entry for this date.
    Sunday 12th August 2012
No entry for this date.
    Monday 13th August 2012
(11:00 am) Continuation of the DNA extraction. Finishing plating up all the CSE kits onto minimal media and polystyrene mix.
(12:00 pm) All the CSE kits are plated and incubating in the warm room ready to be analysed and hopefully come up with a positive result.
(16:00 pm) After the iGEM Weekly meeting, the overnight cultures were prepared ready for starting the next Genomic tip, while the supervisor and a couple of members prepared the lab ready for tomorrow after the leak.
    Tuesday 14th August 2012
(11:00 am) The group running the DNA extraction had a busy day ahead. First of all setting up to run the gel tomorrow, as well as finishing off the genomic tip number 2 prep. This included a team member and the supervisor trying to improvise a method to generate more pressure on the tip to produce a rate of 20-40 drips per min, as it was taking in excess of 24 hours for 10ml to flow through the column. This step was necessary as protocol says for slow rates to use a adaptor that would apply the pressure. As the team didn't have an adaptor, or syringe plunger that was the correct size, an improvisation was made, testing the used column in different ways to apply the pressure. Methods tried were adding a cut down syringe to the top of the column which then could accommodate a plunger, and using a oversized plunger rubber then pressing down with a thumb pushing into the top of the column to create the pressure needed. The second method was used this time to finish off the column as there was only about 2ml to run through. After this the samples were prepared to be loaded onto the gel tomorrow.
(13:00 pm) Once the column was finished, the eluted DNA was separated equally into 24 eppendorf tubes and span for 11 minutes at 13,000G. This was an alteration to the protocol by QIAGEN as it stated at least 5000G in the falcon tubes. However the centrifuge in the iGEM lab can't run this fast with the 50ml falcon tubes, so the separation was needed for the DNA to be span faster. 5000G was the minimum speed so they were span at the max for the centrifuge, making sure the hinge of the eppendorfs was pointing outwards so the pellet could easily be found.
(13:15 pm) After the centrifugation there was no large visible pellet, and but the group proceeded as if there was one on the orientated side of the eppendorf away from the hinge. After a second spin to remove more liquid, 500 micro litre of the 70% ethanol was added to wash the DNA of any salts using the same pippette tip transferring it between 5 of the tubes at a time as the total volume of ethanol to be used was 4ml. The supernatant was then removed and the eppendorf tubes were then allowed to air dry for 20 minutes.
(14:00 pm) The DNA was then re-suspended in 50 micro litre per eppendorf in TE buffer ( PH 8.0 ), placing all of the eppendorf tubes in the 55 degrees hot block for the next hour. This is the dissolving stage to re-suspend the DNA.
(15:00 pm) Once this was finished the DNA was recombined into a single falcon tube and the aliquots were prepared for the Gel electrophoresis.
For the samples, there is a 4x dilution of each aliquot, as well as the pure DNA, just in case there is a large amount of genomic DNA in the samples. These are stored in the 5 degrees room ready for the morning.
A genomic tip being used to extract DNA
(11:00 am) The group running the experiment for the rate of doubling within P. aeruginosa was started, the protocol is as follows.
1. Inoculate 49.5ml broth with 0.5ml of concentrated strain to give a 1% dilution (stock 1). Leave in warm room for 135min.
2. Take 5ml of stock 1 and inoculate into 35ml of broth. Start timer, take spectrophotometer reading at 600nm (zeroed with broth) of 1ml of both stock 1 and the further 8 fold dilution (stock 2).
3. After 5 mins, take a further reading of stock 2. Repeat for 135min, at which point the spec reading should equal the stock 1 reading at time zero (three doublings at 45minutes each).
(11:30 am) After about 30 minutes the readings where going all over the place, almost randomly. So the group decided to edit the protocol slightly. The reasoning for this is thought to be because the group is taking too much of the stock out so the bacteria doesn't have time to recuperate the numbers.
1. Take 5ml of stock 1 and inoculate into 35ml of broth. Start timer, take spectrophotometer reading at 600nm (zeroed with broth) of 1ml of both stock 1 and the further 8 fold dilution (stock 2).
2. After 20 mins, take a further reading of stock 2. Repeat for 135min, at which point the spec reading should equal the stock 1 reading at time zero (three doublings at 45minutes each).
(13:45 pm) This took the group nicely to lunch, with good results, so the growth curve experiment could be done tomorrow as the P. aeruginosa has a doubling time of about 35 minutes.
(15:00 pm) The afternoon was taken up by labelling plates and eppendorfs, as well as melting agar, then producing the plates. This took a few hours, by which point the group decided it was time to go home and start fresh tomorrow.
    Wednesday 15th August 2012
(9:00 am) The dye was loaded on the gel and the electrophoresis was run of the aliquots taken along the genome extraction of E. coli to make sure we have been doing it correctly. There seems to be very low yield from the maxi prep so the group are troubleshooting the problems with the tip and altering some of the steps from the prescribed QIAGEN genome extraction kit.
(11:15 am) The RNase is boiled for 10 minutes to make sure that it re-conforms to the RNase rather than DNase which is possible after a long period of time. This may have been the case with the previous 2 genomic tips, so trying this to see if we can gain a higher yield this time. The sample is now scanned using the nanodrop spectrophotometer to see if there is DNA in the aliquots and the pure DNA elution.
(12:00 pm) Genomic tip 2 gel results were blank so we're thinking it was the RNase that was the problem as there was no DNA in even the aliquot 1 from the gel. Markers and our mini prep showed the DNA ran as such. Now we are preparing the cultures made two nights ago for another Maxi prep with the E. coli, however with the boiled RNase and other alterations to the method.
The OD600 was 3.25 scaled from a 10x dilution, so there is 2.6x10^9 cells per ML. The max for the prep is 1x10^11 cells so we are using 28ML of our culture, working out at 7.28x10^10 cells which is within the limit. To start the bacteria have to be centrifuged into a pellet. The cultures have been balanced equally and are spun in the centrifuge at 2600G which is as high as the centrifuge will go with this size falcon tubes. As this is slightly under the 3000G recommended, the tubes are going to be spun for longer to maximize the size of the pellet. Once the supernatant has been removed, the tubes will be re-spun to make sure any remaining liquid has drained to the bottom, which can then be pipetted off.
(14:30 pm) The lysate is now in the 50 degree incubation stage to digest the proteins from the solution so that they can flow through the column.
(15:15 pm) This has now been going for 45 minutes and has yet to become as clear as before, which maybe due to having cells that are intact. Due to this the incubation will be for another 30 min then spin at 4 degrees for 20 minutes. The clear supernatant will then go into the tip.
In the periods of waiting more of the 0.5% TBE buffer solution was made for the gel tank and the agarose is made while preparing the gel tray. This was done by following the GENIE video which can be seen on youtube at ([http://www.youtube.com/watch?v=wXiiTW3pflM]).
Once the Ethidium Bromide was added it could be poured into the gel tray.
Results from the Nano Drop Spectrophotometer were 2.5ng per uL, at A260 = 0.051, at A280 = 0.021 making the ratio = 2.43 which is quite high as it should be between 1.7 and 1.9. This means that there was likely DNA with a mixture of RNA, however running the gel showed that there was no bands of DNA so it was most likely digested. Hopefully the boiling of the RNase to get it to refold should fix this problem as well as doing the other additional steps one of the supervisors suggested.
(16:15 am) The cell lysate had still yet to become clear after the further incubation, so the lysate was spun for 35 minutes at max speed for the 50ml falcon tube centrifuge. There then became a large fluffy mass of protein and cells that was at the bottom (whilst not forming a pellet) which made it hard to remove the supernatant that contains the DNA. To get around this the team is trying to remove the supernatant by passing the lysate through a 4 micron syringe filter.
(17:15 am) The protein mass was removed and the lysate was passed through the genomic tip. The flow rate was much faster than the other two preps, being within the 20-40 drops per min as stated by the kit, with a slight positive pressure from time to time. The team ended up getting to the precipitation stage before having to leave the lab so it was all put in the freezer for tomorrow.
(09:00 am) The other half of the team doing the growth curve experiment also started early. However after arriving and realizing that an overnight culture was not made, the experiment could not be carried out today. This meant that there wasn't much for the group to do. So the protocol was checked through to make sure it was correct and that the group knew what was happening.
(14:00 pm) An overnight culture was produced and the group headed home.
    Thursday 16th August 2012
(11:15 am) The eluted DNA from yesterday is now spinning down the centrifuge for 11 minutes at 13,000G after being separated equally into 24 eppendorf tubes. This was an alteration to the protocol by QIAGEN as it stated at least 5000G in the falcon tubes, however as the centrifuge in the lab can't do this with the 50ml falcon tubes it was separated it into eppendorf tubes. 5000G was the minimum so they were spun at the max for the centrifuge, making sure the hinge was pointing outwards so the pellet would form on the hinge side. After this there was no large visible pellet however there was at least something this time that looked like a pellet orientated on side of the eppendorf away from the hinge.
(12:00 pm) After a second spin to remove more liquid, 500uL of the 70% ethanol was added to wash the DNA of any salts using and the same 500 µl transferring it between 5 of the tubes at a time so the total volume of ethanol to be used was 4ml. The supernatant was then removed and the eppendorf tubes were then allowed to air dry for 20 minutes or until there is no liquid (Before the addition of the TE buffer).
(09:00 am) An early start for this half of the team. The overnight culture had grown so the 1st experiment with P. aeruginosa to create the growth curve could be done.
(14:00 pm) The experiment was complete, all the plates were put in the warm room and the lab bench cleaned up. This meant another fairly early finish for the group.
    Friday 17th August 2012
(11:30 am) Not much to do today with most of the group were at Google in London, giving a presentation on the iGEM project.
However that being said, the remainder of the group counted colonies of the previous days' work. There was an explosion of colonies, so the experiment would have to be repeated on Tuesday, allowing the overnight broth to be made on Monday.
(14:00 pm) As well as running the DNA prepared yesterday. This caused a bit of the problem as the gel was such a low percentage it broke very easily, but eventually it was set and it could be run. As it was early afternoon, the group decided to run the gel for 2 hours to make sure it separated properly (no electronic copy of gel photo).
    Saturday 18th August 2012
No entry for this date.
    Sunday 19th August 2012
No entry for this date.
    Monday 20th August 2012
(09:00 am) As the overnight culture could not be left over the weekend (grows too much - the Psuedomonas aeruginosa produces pigments which alter the absorbance values) the group doing the growth curve experiment spent most of the day doing the final edits and check throughs of the Rockethub video and website.
This process takes a while to check all spelling mistakes as well as editing the grammar ready for it to be published. The final parts of the video, namely the rewards were also stitched in, and the audio was smoothed out to produce a good quality sound from start to finish.
(16:30 pm) The iGEM weekly meeting took place, and the supervisors present looked through the Rockethub site, as well as watching the video before giving the go ahead for it to be published.
(17:30 pm) The overnight culture for tomorrow's growth curve experiment was produced, ready to start first thing in the morning.
    Tuesday 21st August 2012
(09:30 am) The preparations from yesterday for our mini prep allowed the team to get into full swing with lab work. As the group was working to extract the genome from P. aeruginosa the work was needed to be done in a different lab that was certified to handle opportunistic pathogenic organisms. So all the equipment necessary was moved, and the water baths set up for the incubation’s at 37 and 65 degrees that are needed later in the day. The P. aeruginosa was centrifuged to get a pellet of cells so they could be resuspended in P1 buffer before adding the NaCl and lysozyme to lyse the cells. The mix was then incubated at 37 degrees for 30 minutes,  this lets the solution should go clear and viscous.
(10:30 am) 10% SDS and proteinase K were added to the solution, this strips the DNA of bound proteins and removes any secondary and tertiary structures of the proteins. Anther 30 minute incubation to allow the reagants to act on the DNA in solution.
(12:00 am) The addition of preheated CTAB, as a cationic surfactant, acts as a buffer for the DNA extraction and later when running the gel it prevents the generation of fuzzy bands. Another incubation period is required for the reagents to react.
(15:30 pm) After this the addition chloroform:isoamylalcohol isolates nucleic acids (RNA and DNA) into separate layers. To do this they need to be centrifuged at 13000 rpm for 10 minutes. The nucleic acids are in the top layer dissolved in aqueous solution, and the lipids and proteins dissolved in phenol and chloroform in the bottom layer. The top layer is the desired extract so needs to be transfered to another tube. This process is tricky as it is important to not disturb the interphase layer to prevent the top and bottom layers remixing. As much as possible by hand, but being unable to remove all of the top layer it was decided to repeat the chloroform extraction step/remix, re add chloroform:isoamylalcohol and re-centrifuge.
(16:30 pm) After the recentrifugation the remaining top layer was removed. Next the addition of the isopropanol acts as as a way to precipitate the DNA out of solution into a pellet by centrifugation, as DNA is insoluble in the isopropanol. The isoproponal can then be removed by air drying the pellet. The pullet is then resuspensded in TE buffer to stabilize the DNA in solution, and the DNA is then ready to run on a gel. Before the gel is run though, a nano drop will be performed to ascertain the nucleic acid concentration and absorbance values at 260 and 280 nm plus an absorbance ratio calculated from these values.
(09:00 am) The protocol was changed slightly to compensate for the exuberant growth of the P. aeruginosa. The dilutions being used to plate out now go down to 10^-8 rather than -7. So the protocol is as follows:
1. Take a reading of the overnight broth.
2. Inoculate 49.9ml of broth with 0.1ml of the overnight bacteria colony. Mix, start timer, and immediately do 3 and 4.
3. Put 1ml of culture into a spectrophotometer and take a reading at 600nm. Record this as time = 0. Start a timer.
4. At same time, take an aliquot of 100ul and put in eppendorf marked as time 0, 10^-1 dilution. Add 900ul of buffer and mix, then take 100ul from this solution and add to next eppendorf, marked time 0, 10^-2 dilution and so on to 10^-8.
5. Pipette 100ul of dilutions 10^-6 to 10^-8 onto similarly labelled pre-prepared agar plates and spread them.
6. Repeat steps 3-5 every 40min (1 doubling of E. coli) (labelling time appropriately) until the end of time.
7. Leave plates to grow, then count the number of colonies growing on the most concentrated plate where colonies are distinguishable. Multiply up to the true concentration, then multiply by 10. This is colonies/ml.
8. Draw graph to see correlation and check against known correlation to see if it has worked.
(10:00 am) As this growth curve experiment gets through a lot of plates, some more agar had to be melted and plated up, while all the eppendorfs were marked ready to start the experiment.
(11:00 am) Finally ready to start the experiment again.
(15:00 pm) The experiment was completed with good results for the Absorbance values of the P. aeruginosa, hopefully it worked this time.
    Wednesday 22nd August 2012
(09:00 am) First thing of the day was to run the nano drop, and the results look very promising. The A260/A280 ratios are a little high averaging 2.10, perhaps due to RNA contamination. But the RNA can be removed later by an RNase treatment. A gel is going to be run to ascertain the amount of RNA contamination. The absorbency curves were text book examples of what you expect when looking for DNA which was very pleasing.
(10:30 am) The samples were prepared. Each sample was 15 µl in size with concentration of 200 ng of DNA in total. As the DNA concentrations in each of our 5 preps varied, ascertained from the nano drop, the amount of sample loaded into each well varied and as a result the distilled water added also varied. This was to ensure that equal volumes over all were added into each well to be run. The samples were loaded as well as the size marker being used, lambda Hind 111 (23kb). Due to the high molecular weight of DNA it isn’t expected for the sample to run down the gel very far. The presence of RNA should appear as a blob at the bottom of the gel, if there is a lot then the group will know there is very heavy contamination and the RNA can be removed and the gel rerun. Alternatively if there is contamination but it is very slight then we can remove the RNA when we perform our DNA digest. If there is no contamination from RNA then our high readings on A260/A280 ratio will have some other explanation.
(13:00 pm) The gel has finished running, and the bands are visible but faint on the photo:
There seems to be no contamination from RNA, therefore the next plan of action is to get a higher concentration genome extraction yield, using the following the CTAB protocol:
Take the DNA which is ~500 µl in TE, and add an equal volume of isopropanol. Mix thoroughly but gently. Spin for 10 min at 13,000 RMP. Pipette off the supernatant while ensuring the pellet remains undisturbed. Re-spin to get as much of the isopropanol to evaporate, then air dry. Resuspend the the pellet in the first tube in 100TE, when this has dissolved take the substance and add it to the next tube, each time making sure the pellet has dissolved, mix, tap, but don't pipette. This is because it is necessary to get the pellet into solution before it is placed within the next tube. Repeat for the 5 tubes, until you have all of the pellets resuspended in the same volume of liquid.
Spectrometry on the nano drop to check the concentration. The blank = TE then run on gel.
(15:20) Scraped a colony from C.S.E 01#502 onto a slide and examined it under a microscope and as it surrounded polystyrene beads it was intriguing. A plan was made to plate out onto 2x luria agar, 2x minimal media plates, and 2x 5% sprinkled poly plates. The plates will be inoculated, and one of the 5% sprinkled poly, minimal media, and luria agar placed in the 37o room. The other three inoculated plates left at room temperature to see in the morning which conditions it grows in better.
(16:40) A new 1% agarose gel was made, using the award winning GENIE vid available at [http://www.youtube.com/watch?v=wXiiTW3pflM]. Once this had set, it was then loaded into the gel tank and filled with 0.5X TBE buffer solution. The group's stock was getting low of this solution, so some more was diluted from the 10X TBE, into 2litres of 0.5X TBE and one litre of 1X TBE. The samples were then prepared with 3 µl of 5% loading die, and pipetted into the gel. The lane organization was:
1- x
2- 15 µl HindIII marker
3- x
4- 5 µl 1kb DNA ladder from ThermoFisher Scientific
5- x
6- 15 µl Sample DNA
7- x
8- 5 µl 1kb DNA ladder from ThermoFisher Scientific
9- x
10- 15 µl HindIII marker
11- x
12- x
13- x
14- x
(17:20 pm)The plates of the 01#502 were made. As well as that, the original plate is going to be photographed to see if a detailed photo can be recieved from the microscope. One was taken with a phone, however it would be much better resolution with a actual microscope camera. The gel needs more time to run, so will be wrapped for overnight storage, and will continue to run tomorrow.
(10:00 am) After a successful previous day of experiment the group running the growth curve experiment arrived to check for colonies on the plates, and it was clear there was error in the results. Most of the 10-7 had more cells than -6 as well as massive spreading problems that continued to make an uneven spread. This formed massive groups of colonies, that were unable to be counted easily.
(11:00 am) After discussion in the lab and with our supervisor the group headed down to the supervisors lab to retrieve a plate spinner that would give much better coverage with spreading.
(11:30 am) Another set of plates needed to be made, as well as more eppendorfs required to make the dilutions.
(14:00 pm) After everything was ready, and a quick bite for lunch the experiment was reran with the plate spinner, finishing just at 6. The protocol was slightly altered again for this experiment, as follows:
1. Take a reading of the overnight broth.
2. Inoculate 49.9ml of broth with 0.1ml of the overnight bacteria colony. Mix, start timer, and immediately do 3 and 4.
3. Put 1ml of culture into a spectrophotometer and take a reading at 600nm. Record this as time = 0. Start a timer.
4. At same time, take an aliquot of 100µl and put in eppendorf marked as time 0, 10-1 dilution. Add 900µl of buffer and mix, then take 100µl from this solution and add to next eppendorf, marked time 0, 10-2 dilution and so on to 10-8.
5. Create a further dilution for the 10-6 to 10-8 of 100µl into 200µl of luria broth.
6. Pipette the 300µl of dilutions 10,-6 to 10-8 onto similarly labelled pre-prepared agar plates and spread them.
7. Repeat steps 3-5 every 40min (1 doubling of E. coli) (labelling time appropriately) until the end of time.
8. Leave plates to grow, then count the number of colonies growing on the most concentrated plate where colonies are distinguishable. Multiply up to the true concentration, then multiply by 10. This is colonies/ml.
9. Draw graph to see correlation and check against known correlation to see if it has worked.
    Thursday 23rd August 2012
(09:00 am) The gel was replaced in the tank after being in the cold room wrapped all night and is set up and run at 100 volts. The next CTAB extraction is starting to get ready by gathering the reagents ready. A spectrophotometry reading is taken for the overnight culture of P. aeruginosa and then scaled up to the OD600 for 1ml of culture.
(09.40 am) The spectrophotometry reading was 5.7 which is quite high, however there won't be an exact number of cells until the CFU experiment works. But there is many more cells than before, so the genome extraction can be reran.
(10:00 am) For the past few weeks 1 of the biochemists has been looking at the enzyme modification inbetween lab work and in the evenings. In particular the modelling mutagenesis of toluene 2,3-dioxygenase. 4 base mutations have been identified that would theoretically allow polystyrene to fit into the enzyme's active site, one of which is found as a natural amino acid variation in other similar mono/dioxygenases. He's been in contact with Professor Emma Raven, a dioxygenase expert and will hopefully hear back about whether the mutations are likely to work.
(10:50 am) The P. aeruginosa cells were spun down and the supernatant removed, respun, and all liquid taken off. The cells were then resuspended in 250 µl of P1, as well as the 15 µl of 5M NaCl and 20 µl of 10mg/ml lysozyme. There were a large number of cells of a redish appearance when they were first centrifuged. The cells was left in the 37oC incubation to be lysed. The next stage is also 37oC for 30 minutes. After this stage the eppendorfs can be decontaminated and brought back into the lab.
The gel has finished running and the photograph taken using the machine downstairs:
As can be seen, there is a very faint band at around 23Kb (measured using the hindIII marker on the outer lanes). We need to get a better DNA yield
The bands are more spread out now, the gel has been wrapped back up in cling film and placed it in the 5 degree room in case it is needed again later.
(14:10 pm) The CTAB experiment isn't going to plan, the group thinks that there is far too many cells. The very viscous supernatant was removed and placed into fresh tubes and the cells that were left in the original tubes were resuspended in double quantities of the step 1 CTAB protocol reagents (500 µl P1 buffer, 30 µl of 5M NaCl, 40 uL of lysozyme). It was then realised that an extra 40 µl of lysozyme was put in, this meant the concentration was 100mg/ml rather than 10mg/ml, a 10 fold excess. Because of this the group used fresh lysozyme in the new CTAB prep and was also diluted so all 12 tubes are the same concentration.
(14:30 pm) The positive bacteria on the 01#502 CSE kit had grown well on the Luria agar. The colonies from the 37oC room are not co-joined, and the room temperature plate had more spread out on the plate in streaks and looks to have grown better. Now the plan was changed to have three different temperatures, including the cold room. The minimal media plates don't seem to have growth but they weren't expected to grow quick on it.
The CTAB experiment
PROTOCOL FOR TESTING THE 01#502 POSITIVE BACTERIA
Take a single colony of the bacteria from the LA plate, grown in the 37oC room as there are more single colonies. Streak it out onto another plate of LA so we have a fresh colony.
This single colony will then be used tomorrow to repeat the experiment with a definite single colony.
From the 37oC LA plate, take a single colony and streak it onto each of these plates: Luria Agar, Minimal Media, and Minimal Media with 5% sprinkled Polystyrene in triplicate for the three different temperatures.
One of each of these plates, (one LA, one MM, one MMPoly) then needs to be placed into the 37oC warm room, 5oC cold room and the remaining 3 placed in the lab at room temperature overnight.
(17:30 pm) By this time, the CTAB was almost at the isopropanol stage of the extraction, which ended up in the group being a little late out of the lab. A new gel was prepared ready for morning, while a supervisor prepared two plates of the 01#502 CSE Kit, streaking onto LA placing them one in 37oC one at the bench so there was definitely a fresh single colony plate of cells. There was a small problem with the minimal media agarose as it was a bit hot when the agarose was put in so it took a while to get this into the solution.
(09:00 am) The group running the growth curve experiment was now down to 1 member as the others had headed off home to see parents etc. The last member came into the lab expecting to find good results with the new spreading technique. However there was another problem. This time nothing grew, apart from 2 of the plates at 240 minutes, and these still only had double figures of colonies.
(10:30 am) After a long meeting with a supervisor, it was decided to run an experiment to see if the agar plates were broken somehow or if the broth was just too diluted. This is done by making a new plate, and taking one of the used plates and streaking 500µl of the overnight culture onto them. This would grow enough by the end of the day to prove which circumstance had happened.
(11:00 am) The tedious task of remaking all the plates and labelling the eppendorgs was undertaken again so the experiment could be rerun.
(14:00 pm) In the afternoon there was no tasks to be done so the member went to the computer lab and worked on the wiki, and looked at a couple of research papers to pass the time.
(17:30 pm) The plates had both grown indicating that the broth was just too dilute. This meant the experiment could be reran tomorrow making sure that it wasn't too dilute. This was done by realtering the protocol to make the lowest dilution at 10-7 again:
1. Take a reading of the overnight broth.
2. Inoculate 49.9ml of broth with 0.1ml of the overnight bacteria colony. Mix, start timer, and immediately do 3 and 4.
3. Put 1ml of culture into a spectrophotometer and take a reading at 600nm. Record this as time = 0. Start a timer.
4. At same time, take an aliquot of 100µl and put in eppendorf marked as time 0, 10-1 dilution. Add 900µl of buffer and mix, then take 100µl from this solution and add to next eppendorf, marked time 0, 10-2 dilution and so on to 1-7.
5. Create a further dilution for the 10-5 to 10-7 of 100µl into 200µl of luria broth.
6. Pipette the 300µl of dilutions 10-5 to 10-7 onto similarly labelled pre-prepared agar plates and spread them.
7. Repeat steps 3-5 every 40min (1 doubling of Pseudomonas) (labelling time appropriately) until the end of time.
8. Leave plates to grow, then count the number of colonies growing on the most concentrated plate where colonies are distinguishable. Multiply up to the true concentration, then multiply by 10. This is colonies/ml.
9. Draw graph to see correlation and check against known correlation to see if it has worked.
    Friday 24th August 2012
(08:30 am) An early start for the group running the CTAB with a lot to do today ready for the Bank holiday weekend. All of the isopropanol-supernatant mix from the DNA extractions were centrifuged at 13,000G for 10 minutes to spin down the precipitated DNA ready to air dry. After the air drying and resuspending in TE the slow gel needed to be run so a supervisor put them in the vacuum dryer to speed things up.
(09:00 am) 10 tubes of 1ml culture of P. aeruginosa was prepared. 4 for the robot (DNA extraction robot) and 6 for the CTAB. For the 4 robot tubes, 250 µl of TE buffer was used for the resuspension rather than the P1 buffer, then 2 µl of 100mg/ml lysozyme was added and the 5M of NaCl salts were left out and incubated for 30 min at 37 degrees. The robot takes about 40 minutes, and it will be ran on the fast gel once it is complete. Will The 6 tubes of CTAB extraction is being redone today to see if a lower OD600 culture will work better. Today's culture OD600 IS 2.48 scaled up from a 10X dilution.
(09:30 am) A new electrophoresis gel was prepared for the fast gel in the longer gel tray to run the robot DNA as well as the CTAB's from yesterday.
(10:20 am) The gel tray has been prepared. yesterday's extracted DNA was put on a vac line to dry, the CTAB method was being run in the other lab.
(11:00 am) The gel was poured and it was placed in the 5 degree room to set. The DNA was resuspended in 100 µl of TE buffer and the tubes placed in the 55 degree hot block to resuspend for a between an hour and two hours (from QIAGEN protocol). The robot is running and only has 44 minutes left, so the robot DNA can be placed on the fast gel. In the CTAB the RNase was added at the same time as the protease K. The SDS has also been put in and it needed to be incubated at 37 degrees for 30 minutes.
(13:15 pm) Unfortunately when the slow gel was poured last night there were a lot of bubbles, so it had to remelted, repoured and set.
After the samples were prepared they were loaded onto the gel for the fast electrophoresis, after the nanodrop spectrometer was used to find out the concentration of DNA in the tubes from extraction yesterday. One extraction had a DNA concentration (in ng/µl) of 65.2 and a 260nm/280nm ratio of 1.46. The other extraction had errors within the results, one had no DNA in it, and the one that gave results clocked in at an impressive concentration of 1755.2 ng/µl (which could easily just be down to RNA still present or other things) and a ratio of 1.94.
The loading order of both gels were:
1- x
2- x
3- 100bp Marker
4- 6
5- 5
6- 6a
7- marker
8- pa1
9- pa2
10- pa3
11- pa4
12- x
13- x
14- x
The volumes of DNA used in all but 6a were 12 µl, 6a was 1 µl after the high nanodrop reading. Each well was spotted with a small volume of loading dye to make each of the wells more visible.
(13:20 pm) The fast gel was started at 100 Volts.
(13:30 pm) The slow Gel was loaded, and is being run at 80 volts for 4 hours. .
The volumes of DNA used in all but 6a were 12 µl, 6a was 1 µl after the high nanodrop reading. As before each well was spotted with a small volume of loading dye to make each of the wells more visible.
There were problems removing the chloroform layer from the CTAB prep, so 250 µl of TE buffer were used to dilute what DNA is left. After centrifuging for 2 min, vortexing gently, then centrifuging for another 10 minutes. The group tried removing the layer once more.
(17:00 pm) The gel electrophoresis was finished and the DNA extraction worked this time, using both the CTAB method from yesterday, and Promega Maxwell DNA extractor robot. Fast gel:
Fast and slow gel runs. The fast gel is to the left, and the slow to the right. As can be seen, CTAB '6' appears to be heavily contaminated with RNA, '5' has a small amount of what appears to be genomic DNA present. '6a' appears to have some genomic DNA present, though it appears that the DNA wasn't loaded properly, or the gel wasn't evenly poured. The Maxwell prep DNA extracted seems to have worked, and be in quite high concentration.Nate that we loaded the wrong sized DNA ladder, using the 100bp rather than 1kb ones.
The PBS was needed to be boiled as it was cloudy which is bad, then cooled to room temperature. After this a single colony of our positive CSE kit was needed to be dispersed into a volume of PBS, to make sure an equal amount of cells is being put onto each plate. 3 of each media plates were made, 1 at each temperature. This was repeated with TE just in case the PBS is contaminated badly.
(12:30 pm) After agreeing to come in later so the last member could go into town, the growth curve experiment was ready to be reran again.
(13:00 pm) A prompt start at 1 meant the experiment would be finished at 5. With only packing up and clearing the lab space to go.
(17:30 pm) The experiment was finished, and hopefully this was the last time it was needed to be ran for P. aeruginosa. As next Monday is a bank holiday the team is not allowed in labs as no supervisors will be working. This meant there is a 3 day gap inbetween plating the bacteria and looking at the results. To slow the growth then and not just have an explosion on the plates, it was decided to leave them in room temperature over the weekend rather than take them to the warm room.
    Saturday 25th August 2012
No entry for this date.
    Sunday 26th August 2012
No entry for this date.
    Monday 27th August 2012
Bank holiday in the UK, so the university was closed and no supervisors were in labs. The remainder of the team (from both sides of the experiment) attended an event in the Aylstone Leisure Centre in Leicester to publicise the project and teach people a little more about polystyrene.
    Tuesday 28th August 2012
Photos have been taken of the replated CSE 01#502, and now have colonies growing well on LA. However nothing is growing on mm still, nor on the MMP plates. This was expected as they are meant to grow really slow on polystyrene. A Gram stain test has been done on the 01#502 which is inconclusive at the moment as there were too many cells as it has turned black.
The Gram test
(10:50 am) A 0.7% agarose electrophoresis gel was made then loaded as follows as a rerun. It was run at 40volts.
1- x
2- HindIII
3- 6
4- 5
5- 6a
6- x
7- x
8- HindIII
9- pa1
10- pa2
11- pa3
12- pa4
13- x
14- x
(17:00 pm) The gel has finished, and looks good!
We appear to have genomic DNA in every DNA extraction attempt, though 5's is a bit faint
Some cultures of 01#502 were prepared ready for tomorrow, as they are needed to be cultured on antibiotic plates, as well as start a CTAB and 16s to find out what the bacteria actually is.
Change the conc of enzyme rather than the time, need a ZERO control (Sal3A1 digest tomorrow).
(09:00 am) The morning was started by going to see the P. aeruginosa plates made on Friday, and the plates look great!
(09:45 am) The plates had been counted, the data collected and tabulated. Then taken to the computer room to provide a graph with excel so the growth curve can be made easily. The result of this showed a good curve that can now be used for all future P. aeruginosa experiments.
(11:00 am) With nothing particularly left to do, the rest of the day was filled with administration tasks. This included sorting out the finance spreadsheet, working on the wiki, as well as planning the next few days work with the 01#502 positive bacteria.
    Wednesday 29th August 2012
(09:00 am) The Gram stain was redone, with exactly the same results as yesterday. An oil immersion was needed to be conclusive. However down the microscope, the 40x looked exactly the same apart from the bacteria were more spread out. The biochemist is continuing to model with the toluene 2,3-dioxygenase enzyme.
(10:00 am) The team leader and the supervisors were discussing what to do next, while the gel from yesterday was replaced into the tank to run for a bit longer at 40 volts.
(11:30) New overnight cultures of the 01#502 of the single cell. However multi cell plate and cultures would also be needed. The PCR and antibiotic test plates were being organised as well.
(14:50 pm) The gel was stopped and photographed, and the bands had moved more. Now the H3 markers were much more separated. From this the team learnt that the H3 markers first should be heated first, as Lambda markers have tails on them which anneal if they are left and not heated before using. In the 3rd lane the CTAB 6 prep seemed to run very strangely, with very high molecular weight DNA in comparison to the other lanes.
(16:40 pm) The Sau3A1 preparation was started while the gel was made for the electrophoresis wanted to be ran over night at 20 volts. Three new overnight cultures of the orange culture 01#502 were made, which are now in the shaker to grow at room temp.
(17:45 pm) The Sau3A1 prep is now finished, and it is being loaded into the gel tank. The oil immersion of the second gram stain was completed and the positive bacteria is definitely gram positive.
The second Gram stain, result is positive
Sau3A1 preparation protocol:
One unit will digest 1 µg of DNA in one hour at 37 degrees in 50 µl. So use 14.24 µl of the teams DNA for one ug, which was worked out from 70ng/µl. Add in EDTA to chilate the Mg+ ion, and chill the reaction suddenly from 37 to iced water.
Take a 10 µl aliquot out and put into the EDTA and loading dye.
Use 100 µl of the robot prep = 700 ng of DNA. One uL of EDTA, 3 µl loading dye, 6 µl of TE buffer.
The team used 0.5M EDTA and 10mM MgCl in the 1x buffer, and enough EDTA was needed to chelate the MgCl to stop the reaction.
Take 107.4 µl of the DNA from the robot prep mixed with 15 µl of the NE buffer 1, 1.5 µl of BSA, 2 µl (8 units) of Sau3A1, then made up to a total volume of 150 µl with distilled water. This is the reaction mix with it being active as soon as the Sau3A1 has been added so this must be at the end of the reaction.
The reaction conditions are 37 degrees, taking out a 10 µl aliquot every 5 minutes for a hour to create a time course for the enzyme. Each of the aliquots is made up to 20 µl by adding in 1 uL of EDTA 0.5M, 3 µl of loading dye, and 6 µl of TE. Keep everything on ice until Sau3A1 is added, pipette up and down a few times, then take the time 0 recording.
1/5 Dilution of the robot DNA Nano Drop reading was 14.9ng/ml so at 1x it would be 74.5ng/ml
(10:00 am) As the overnight broths made by one of the other team hadn't been left in a shaker or the warm room there wasn't enough bacteria to run the doubling experiment, or the growth curve experiment for the 01#502. This meant that not much could be done today.
(11:30 am) A minimal broth was made for the other team so it could be used as they like for creating minimal plates with polystyrene, or for attempting to grow liquid cultures.
(13:00 pm) After an early lunch, the group decided to melt lots of agar (there were 2 bottles with about 4 plates worth left in each, as well a as a full bottle) to produce lots of plates to go in the cold room ready for experiments wanting to be ran tomorrow.
(15:00 pm) This was followed by producing several corning tubes filled with overnight broth, that were left in the warm room in the shaker. Then an early finish as there was nothing left to do.
    Thursday 30th August 2012
(09:00 am) The overnight gel was stopped 9, the gel was photographed as it was running for a bit long, however the DNA hadn't migrated very far. Because of this the gel back was put back into the tank and ran at 40 volts for 5 hours. The overnight culture, had the OD taken, reading at 0.807 at a x10 dilution as it is a very dense culture.
(09:30 am) The plates that were growing from the 01#502 as well as some other plates were photographed after looking down the microscope.
(10:00 am) After much deliberation, it was decided to run the PCR over night tonight rather than tomorrow so that the CTAB can be done sooner to find out what the positive bacteria is.
(12:00 pm) Preparing for the PCR, one of the supervisors made up some dNTP's for use in the reaction, while the team was working out the amounts to put in each PRC eppendorf tube. The team also learnt how to do the PCR and use the PCR hood. This took a fair amount of time making sure it was all ready.
(14:20 pm) The gel was stopped and exposed just in time as the bands were very close to the end of the gel:
The digest: seeing as the 10 minute lane was empty, and most of the lanes appeared to have almost and similarly completely digested, we need to rerun the experiment. This time the experiment is being ran with 4 units of Sau3A1 and 1µl more EDTA in case it was the stopping procedure that didn't work, as from 30-60 the DNA was almost all digested.
(16:00 pm) The Sau3A1 digest is almost ready to start, the Boilate is being run, ready for the serial dilutions of the 01#502.
The protocol of the boilate is:
Take 1000ul of PBS in a screw cap eppendorf with enough bacteria to make the solution cloudy, making sure to keep the bacteria and tubes next to the flame. Before this step, the tubes were boiled for 2 minutes to make sure that the outside was sterile.
Boil the tubes for 10 minutes, then centrifuge for 5 minutes at 13,000G.
The supernatant now has DNA in it ready for the serial dilutions.
For the PCR the bacterial DNA is to be diluted down so there is less in each as the amount is unknown in ng of DNA, or the amount of cells precisely. This means that the serial dilutions needed are from not diluted to 10-4.
Protocol for the serial dilutions:
Take 100ul of the supernatant into another eppendorph and dilute with 900µl of PBS, repeating this step 4 times to get a series of x10 dilutions down to x10-4.
(16:40 pm) The gel for running the Sau3A1 is prepared and made.
(17:30 pm) The supervisor made up the master mix for the PCR experiment in the PCR hood containing:
114µl PCR clean H2O
40µl HF buffer
4µl dNTPs
10µl Primer A 28f AAGAGTTTGATCCTGGCTCAGA
10µl Primer B 519R GWATTACCGCGGCKGCTG
Template DNA to be added after as it is not PCR clean
2µl DNA Pol
As the DNA Pol was in glycerol the tube was mixed and span for a second to remove bubbles. These reagents were then added. dNTPs cannot be freeze thawed so whatever was left was thrown. After this, in the hood 18ul aliquots of the master mix were taken into 9 separate PCR eppendorfs and the negative control was prepared, while in the PCR hood with the PCR clean H2O.
The tubes were then taken to the bench where the DNA and diluted bacteria were added, and the positive and negative controls (see gel lane organization). All samples were added at 2µl to make a final volume of 20µl.
The PCR block was programmed and named iGEM16S as follows:
98 degrees C - 5 min
98 degrees C - 30 seconds
50 degrees C - 30 seconds
72 degrees C - 2 minutes
72 degrees C - 5 minutes
15 degrees C - Forever (this is for the end of the reaction)
(16:40 pm)At the same time as the PCR and boilate the Sau3A1 digest was rerun, using the same protocol as Wednesday the 29th of August.
(09:00 am) An early start but with lots to do, and a very long day ahead. Preparations for the doubling experiment were undertaken so it could be started early.
(09:30 am) The doubling experiment was ran with the same protocol as the last doubling experiment, with spectrophotometer readings every 20 minutes. Inbetween the readings, work was done towards clearing the lab space and making sure all required materials were there for the growth curve experiment in the afternoon.
(11:30 am) The experiment was finished and the approximate doubling time for the 01#502 bacteria turned out to be just under 40 minutes. This was good as it meant the protocol for the growth curve experiment didn't have to be changed.
(12:00 pm) For the next hour all the plates made yesterday, as well as another load of eppendorfs were made ready and marked for the next experiment.
(13:30 pm) A quick lunch, then straight back into labs to start the growth curve experiment. As it takes 4 hours to complete with readings starting at this time lets the group finish and pack up just before lab closing at 6.
    Friday 31st August 2012
(09:00 am) The Sau3A1 samples were thawed, and made ready to be loaded and put into the pre made gel. The iGEM email account has been cleared and sorted out, putting the CSE kit data into the Google Map that can be seen on the CSE page of the Blog. A new 2% gel was made for the PCR electrophoresis.
(10:00 am) The Sau3A1 gel was set at 80Volts for 4 hours with the 1KB gene ruler from Thermofisher Scientific.
(10:30 am) More TBE buffer for the gel tanks is made, and then the PCR products are being loaded onto the gel. Some emails are being sent out to people who have bought CSE kits to get them returned to be cultured.
(11:15 am) Started the PCR Gel at 120V with the following lane organization: x Positive control 10ng P. aeruginosa DNA from the maxwell prep Positive control 10ng P. aeruginosa DNA from the maxwell prep Marker 100bp Thermo scientific Neat 0 dilution cells 10-1 dilution 10-2 dilution 10-3 dilution 10-4 dilution Negative control, No DNA PCR clean H2O Negative control, No DNA Distilled bench H2O Marker 100bp thermo scientific x x
each well we loaded contained 5µl of loading dye into each sample to make a total of 25µl. However upon trying to take out 25µl using the p200 pippette, it was very difficult to remove less than 25µl each time. Therefore 20µl of each of the samples was loaded into the gel, after spotting the wells.
(13:00 pm) The PCR gel was stopped and photo taken after running for an hour and a half.
As can be seen, all lanes with positive results have the same, or very similar length.
The gel was then put back in the gel tank and ran for another hour to make the large bands in lanes 4-7 separate more from the bacground, to allow the gel extraction to be easier.
(13:45 pm) The group started the job of racking tips ready to be autoclaved and giving the lab a good tidy.
(14:00 pm) The PCR Gel has been finished and photographed, and the bands are now much more separated. The gel was then iluminated in the dark room and the sections cut out where the DNA was at the highest concentration at the 500bp marker points, making sure not to get to much agarose. After the gel was cut, the samples were weighed to know how much was in each eppendorph to know how much QG buffer to add in the gel extraction.
(14:40 pm) The Sau3A1 gel was now finished so it too could be photographed and then the team realised it was running for a bit to long with the smallest fragment of the ladder not being visible. The run however was much better than last time:
This Sau3A1 digest clearly shows a downward trend in fragment length, despite the last few having parts of the band missing off the end of the gel.
Although after talking to one of the supervisors, this experiment needs to be run again at room temp (20 degrees) so that the fragments are larger still, making sure this time that there is a non digested DNA sample loaded onto the gel which has been incubated under the same conditions. As well as running the prep for a shorter time of 30 minutes, but taking readings every 5 minutes instead.
(16:00 pm) The gel extraction prep is almost done. The deadline tasks for the competition were being sorted out, and the preparation for the new Sau3A1 prep was started.
(17:00 pm) After a few calculation problems it was decided to leave the Sau3A1 prep to Monday when the lab leader is back and can help with the procedure. Photos of the 01#502 CSE MMP and MM plates were taken to look for signs of growth on the plates. Yet another 2% electrophoresis gel is made for running some of the gel extraction products on to make sure we have enough extracted DNA ready for sequencing of the 16S next week. A minimal media broth with 5% polystyrene as the carbon source is made, and a single colony of the 01#502 is placed into one of the tests and some of the multi colony original LA is placed under the same conditions.
Once the bacteria was suspended in the two falcon tubes, the lids were sealed with tape to make sure it doesn't come loose on the shaker and placed in the shaker to grow.
(17:45 pm) In the microscope room, there was what looked like bacteria colonies on the polystyrene plate left at room temperature of the 01#502. The bacteria on the plate seem to be at higher concentrations around the polystyrene beads. To rule out it being a flow problem with the PBS or H2O or an evaporation problem leaving behind crystals a new 5% polystyrene plate was set up under the same conditions as the 01#502 to be left over the weekend to look at these, and to see what the marks on the plate are like from just this and no bacteria to compare. The pltes were now made as split plates with a gap in the middle for the 01#502 conditions test. This was very useful when taking the photos as it showed no bacteria in the center of the plate meaning that the colonies were from the PBS or H2O 01#502 mixtures rather than contaminates with the PBS/H2O with no bacteria test, ruling out contamination from those.
(09:30 am) Came in expectant of a failed growth curve experiment, simply because nothing ever goes right first time. The group was not dissappointed. There was an incredible amount of growth over the plates, and much higher than even expected from the high absorbance values. It would be a pointless and time consuming cause to count them, and the data wouldn't be reliable anyway as no curve would fit it well.
(10:00 am) A few more plates were required to be made to complete the next growth curve experiment as more were used up yesterday than expected.
(11:00 am) After a quick discussion with a supervisor, it was determined that a higher dilution was required to start the experiment so it doesn't grow too much.
(11:30 am) The experiment was reran but with a higher starting dilution of 200x. This involved placing 0.25ml of the overnight culture into 49.75ml of luria broth. The readings were taken, and they looked suitable so the experiment was started. Another alteration to previous methods is to let the plates dry over a course of a couple of hours, as before the liquid took longer to dry and when moved caused streaks and colonies to form large groups where they can't be counted.
(15:30 pm) The experiment was finished and remaining plates left to dry for a short while. While waiting for them to dry the group worked on the wiki.
(16:30 pm) The plates were almost dry so they were flipped to prevent condensation on the agar, then left on the bench for the weekend. This is to try and stop an explosion of growth again as the group isn't allowed in labs at the weekend.
[edit]
Retrieved from " http://2012.igem.org/Team:Leicester/August2012 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
