"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Leicester/Parts
From 2012.igem.org
iGEM Leicester Test Page 2012
Home
Team
Project
Chemistry
Parts Submitted to Registry
Modeling
Notebook
July
August
September
Safety
Human Practices
Attributions
Bibliography
Parts Submitted to the Registry
An important aspect of the iGEM competition is the use and creation of standard biological parts. Each team will make new parts during iGEM and will place them in the Registry of Standard Biological Parts . The iGEM software provides an easy way to present the parts your team has created . The "groupparts" tag will generate a table with all of the parts that your team adds to your team sandbox. Note that if you want to document a part you need to document it on the Registry , not on your team wiki.
Remember that the goal of proper part documentation is to describe and define a part such that it can be used without a need to refer to the primary literature. The next iGEM team should be able to read your documentation and be able to use the part successfully. Also, you should provide proper references to acknowledge previous authors and to provide for users who wish to know more.
Primers and PCR
Having worked out how long screening a DNA library could take we realised that it would be quicker to try to isolate our genes of interest using PCR. We used BLAST on sequenced Pseudomonas genomes to locate the genes of the Tod operon, and found the whole operon was remarkably conserved. Only a couple of the genes varied at all between the sequenced strains: we took account of this by adding a few degenerate positions in the PCR primers (see below). We concentrated on genes in the Tod operon that didn't have restriction sites incompatible with the basic Biobrick standard:
TodX: (544bp) >TODXF 5'-ATGCCCGCCAGTCTGACGCTTG-3' >TODXR 5'-ACCAGCCAGCACCATGCGGC-3' TodX gene in all putida the same regardless of strain
TodX - 544bp > Aeruginosa - None > Promega Maxwell extracted DNA > Putida A and B - Bands at <500bp, 600bp, 1200bp and >1500bp > Orange bacteria - 1 band at >1500bp > Yellow bacteria - Faint bands at 1200bp and >1500bp > Boilate reaction > Putida A and B - Faint bands at <500bp, 600bp, 1200bp and >1500bp > Orange and yellow boilates failed. TodF: (460bp) >TODFF 5'-ATGGGTGCCGTTGGCGTGAG-3' >TODFR 5'-GTTTTTGCGATCAGTCCTCCGCG-3' All sequenced Pseudomonas putida strains appear to have the above sequences, although other species may amplify better with the degenerate primers: >TODFR 5'-GTTTTTGMGATCAGTCCTCCGYG-3'
> TodF - 460bp > Aeruginosa - None > Promega Maxwell extracted DNA > Putida A and B - 1 band between 700bp-800bp > Orange bacteria - 1 band at 1500bp and 1 band between 1200-1500bp > Yellow bacteria - Faint band between 500bp-600bp, other bands are above > Boilate reaction > Putida A and B - Faint band between 500bp-600bp, other bands are above, 1 strong band between 700bp-800bp > Orange - Strong bands between 1000bp-1500bp > Yellow - Faint band between 500bp-600bp, and strong bands >1200bp
TodC1: (1353bp) >TODC1F 5'-ATGAATCAGACCGACACATCACCTATC-3' >TODC1R 5'-TCAGCGTGTCGCCTTCAGCG-3' one strain has a C rather than a G base >TODC1R 5'-TCASCGTGTCGCCTTCAGCG-3'
TodC1 - 1353bp > Aeruginosa - band at 800bp and 1kb > Promega Maxwell extracted DNA > Putida A and b - Band at 800bp, 1.2kb and one in-between 1.2 and 1.5kb > Orange bacteria - one band high up above the 1.5kb marker > yellow bacteria - no bands > Boilate reaction > boilates failed to amplify TobB: (324bp) >TOBBF 5'-ATGACTTGGACATACATATTGCGGCAG-3' >TOBBR 5'-TCACTTCAACTCCCCGTTGTCGAG-3' ^ all sequenced Pseudomonas putida strains had the same gene sequence
TodB - 324bp > Aeruginosa - None > Promega Maxwell extracted DNA > Putida A and B - None > Orange bacteria - several indistinct bands ranging from >300bp to 800bp, 2 are between 300bp-400bp > Yellow bacteria - several indistinct bands ranging from >200bp to 1500bp, 1 faint band is between 300bp-400bp > Boilate reaction > Putida A and B - No distinct bands > Orange - one faint band between 300bp-400bp and more faint bands between 400bp-1000bp > Yellow - None TobG:(807bp) >TOBGF 5'-ATGAGCGAACTAGATACCGCGCG-3' >TOBGR 5'-TTATGCCTTTGCAAAAGCGGCGGTC-3' ^ all sequenced Pseudomonas putida strains had the same gene sequence
TodG - 807bp > Aeruginosa - band at 1kb and one high up > Promega Maxwell extracted DNA > putida A and b - band at 700bp and a fainter one at 800 and 600 ... bands also present above the 1.5kb marker > Orange good sized band at 800bp and a lower one at 700bp ... also has bands at 1.2 and 1.5 kb > yellow faint band at 800bp and one at 1kb > Boilate reaction > boilate failed
Unfortunately, with the 2 strains of Pseudomonas we had in the lab, none of the primers amplified a product of a size consistent with the target genes, meaning that the strains likely do not encode any of the Tod operon genes. As a result we couldn't make a biobrick, although the strategy should . However, these analyses and primers will be available for future iGEM Leicester teams. With the right strains of bacteria, or new bacteria isolated in the "Citizen Science Experiment", they may be able to extract the target genes relatively quickly, for biobrick construction and characterisation.
[edit]
Retrieved from " http://2012.igem.org/Team:Leicester/Parts "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
