"
Page
Discussion
View source
History
teams
Log in
 
Team:Leicester/Project
From 2012.igem.org
iGEM Leicester Test Page 2012
Home
Team
Project
Chemistry
Parts Submitted to Registry
Modeling
Notebook
July
August
September
Safety
Human Practices
Attributions
Bibliography
Overall Project
Our project is about trying to reduce the waste going to landfill by engineering a bacterium to degrade polystyrene. Some bacteria form biofilms on polystyrene, indicating that polystyrene may be being degraded. We have several different parts to our project, including a citizen science experiment (CSE)where members of the public are helping us.
It is not clear how the bacteria actually degrade the polystyrene, as no biochemical pathway for polystyrene degradation has been found, so far. It is possible that the reason why degradation appears to be so slow is because no existing degradation pathway has evolved specifically to tackle polystyrene. Enzymes in other pathways may be involved in degradation but as they are not adapted for polystyrene, the rate will be very slow. There are a couple of pathways we have been looking into that could be involved: the styrene degradation pathway (as styrene is the monomer of polystyrene), and the toluene degradation pathway (this could be involved in breaking open the bulky, unreactive phenyl side chain, making it easier for aliphatic degrading enzymes to degrade the main chain). We've done some modeling of the first enzyme in the toluene degrading pathway, and future Leicester iGEM teams may be able to artificially mutate the first enzyme to see whether our potential modification could work (see the modeling page).
We intend to extract the genes involved in pathways degrading Expanded Polystyrene (EPS), and / or develop a new pathway involving modified enzymes able to degrade aromatic and aliphatic hydrocarbons that can fit polystyrene and its derivatives into their active sites. The bacteria that we insert these new genes into should then be able to degrade polystyrene at a higher rate than natural bacteria. Future iGEM teams at the University of Leicester could harness this bacterial strain to produce useful products.
Since starting the project, as well as this wiki, we've created a blogspot blog uoleicesterigem2012 , a Twitter account @iGEM Leicester and a facebook page uoleicesterigem to increase our audience.
Project Details
Making Polystyrene selection media
The first stage involved making a selection media to isolate bacterial colonies that could survive with polystyrene as effectively the only carbon source available, making it difficult for bacteria unable to degrade polystyrene to survive. However, salts containing the other minerals needed for survival must be present. We found a minimal salts media recipe (Atiq et al.,2010) that avoided other carbon sources while still making other essential elements available. We spent several weeks attempting to dissolve polystyrene into the agar, using various solvents such as acetone (which just took the air out of expanded polystyrene, rather than dissolving the polymer, and dissolved petri dishes), toluene (which dissolved polystyrene 'sugar' (pre-expanded polystyrene beads donated to us by Styropack ltd.)but also dissolved the plates) and various alcohols (which didn't dissolve polystyrene). In the end, we decided not to use a solvent, and instead sprinkled the polystyrene 'sugar' on top of the minimal media once poured, so that the 'sugar' was available to bacteria spread on the surface.
The Protocol
Polystyrene selection media (or polystyrene minimal media) is made first by making up a minimal broth using the recipe from (Atiq et al.,2010) by dissolving 1g K2HPO4, 0.2g KH2PO4, 1g NaCl, 0.002g CaCl2.2H2O, 0.005g Boric Acid, 1g (NH4)2SO4, 0.5g MgSO4.7H2O, 0.001g CuSO4.5H2O, 0.001g ZnSO4.7H2O, 0.001 g MnSO4.H2O and 0.01g FeSO4.7H2O, per litre H2O. Melt 2% electrophoresis grade agarose to the broth to solidify it after cooling using a microwave, and heat until the agarose is completely melted/dissolved. Allow media to cool by keeping it at 60 degrees C until needed. To make a selection plate, use 95% media and 5% polystyrene 'sugar' (polystyrene beads before they've been expanded in the manufacturing process). We determined that 10 grams as a total mass of minimal media and solution per plate was optimal. We poured out the media, and before it solidified, sprinkled the polystyrene on top, to prevent the 'sugar' from sinking to the bottom so allowing bacteria colonies to access the polystyrene as a carbon source. The agarose is the only other carbon source in the plate (except obviously the plate itself), so any colonies that grow should be utilising polystyrene as a carbon source, or at least have evolved pathways to deal with the toxic effects that a long chain hydrocarbon could cause bacteria (such as interfering with the cell membrane, due to the hydrophobic nature of parts of both).
The Citizen Science Experiment
The CSE was used to get the public involved in trying to find bacteria that could live on, and degrade expanded polystyrene (EPS). We asked them to bury a strip of EPS in the ground and leave it for a length of time (around 2 months), to find out whether bacteria might establish colonies, thus indicating that a colony could at the very least bind to, and possibly degrade EPS. The kits were Risk Assessed, and a copy of the assessment was included in the kit, along with instructions, a strip of EPS, and a self seal bag to put the EPS into. This was all contained in a stamped, addressed envelope at a cost to the public of £2 (+50p postage if bought online via our blog to cover the cost of sending the kit to the person). We recently had our first batch of kits back, ready to analyse.
we made a YouTube video explaining what the public had to do with the polystyrene strip:
Since this experiment was launched, around 40 CSE kits have been returned, allowing us to plate them out and find bacteria able to degrade polystyrene. The best kit that has been returned so far has been kit 01#502, which has yielded 2 different types of bacteria (we're not sure whether they are the same species, but instead just different strains). Both types of bacteria were growing together on a polystyrene-minimal media plate, which could indicate a mutual relationship in polystyrene degradation. Later experiments (see polystyrene growth experiment) were conducted to confirm they were using polystyrene.
The Protocol
The first part involves halving the returned expanded polystyrene (EPS) lengthways with sterilized scissors in our returned citizen science experiment (CSE) kits so that we'll still have some of each sample left should we need it. First we need to do a practice with unused EPS strips (control and test). Pour/pipette 20ml LB agar into a petri dish and allow to set. Put line down middle in permanent marker USING A RULER! Then label one side swab (for the swabbing/scraping material) and the other side strip for the EPS strip (can use other names if you prefer). Add 50 micro litres of PBS (phosphate buffered saline) to each side of the half strip of EPS and scrape with spreader, before spreading onto the left half of the agar (or side labeled as 'swab'). Try to swab any mud off the strip if possible. Then place the half strip in the centre of the other half labeled 'strip'. Leave for a week at room temperature on the lab bench and examine agar for colonies. The other half of the strip should be put back into cold storage for future use.
After letting the organisms grow for a few weeks, select some vigourously growing colonies and spread them out onto a polystyrene minimal media plate divided into 6 sections, where each selected colony is allocated its own segment. Leave at room temperature or at 37oC, checking each day. The bacteria that grow are likely to only grow very slowly, so it could be a month or more until you see a visible colony, due to the low polystyrene degrading rates of bacterial enzymes. If a colony grows, extract some, add the sample to luria broth, and grow overnight on a shaker at room temperature or 37 oC. Boilate the colony by boiling the bacteria in distilled water for 10 minutes, before centrifuging at 13,000 RPM for 5 minutes to remove cell debris- the supernatent should contain DNA, which you can then add to a PCR mix and amplify overnight
Gram Stain
Prepare a light suspension of cells from very young cultures grown on appropriate agar medium. If the suspension prepared is too turbid, dilute with distilled water.  1.       Add one drop to a clean glass slide and spread the drop with a loop over the surface of the slide. Allow to air-dry.  2.       Carefully fixate the cells by moving the slide into a flame (bacteria upwards).  3.       Flood the slide for 1 minute with Hucker's reagent.  4.       Wash the slide by dipping the slide into slow running tap water.  5.       Flood the slide with iodine solution for 1 minute.  6.       Place slide diagonally in glass box and rinse of iodine solution with safranin.   7.       Add excess amount of fresh safranin and wait for 35 seconds.  8.       Rinse slide with water as described under (5).  9.       Allow the slide to air-dry.  10.   Examine the preparations with the oil immersion objective of the bright field microscope (do not use the phase-contrast objective!). A drop of oil can be placed on the slide directly.  11.   Gram-positive cells appear purple and Gram-negative cells pink. (The iris of the microscope condenser should be opened as wide as possible. With a closed condenser colours can hardly be discriminated.):
View of gram stain through microscope
DNA extraction
DNA extraction is of course vital when trying to make biobricks and to extract genes of interest, and we've had varying success with several methods. These have included QIAGEN's Blood & Cell Culture DNA Maxi kit, CTAB and Maxwell DNA extraction.
QIAGEN Blood & Cell Culture DNA Maxi kit
The protocol was as in instruction booklet supplied instruction booklet supplied (PDF reader required). Unfortunately, this method didn't really work, despite 4-5 retries with E. coli as a test.
CTAB
Eppendorfs used in DNA extration
This protocol was given to us by Miranda Johnson of the level 2 lab on our floor to use after the QUIGEN kit didn't work. It is designed for Staphylococcal chromosomal DNA extraction, but we had some success with the extraction regardless of the species of bacteria used:
Using an overnight culture, spin down 1.5ml of culture in an 1.5ml Eppendorf tube for 10 minutes at 13,000 RPM. remove the supernatant, then respin down so as to be able to remove any more supernatant. Resuspend the cells in 250µl P1 buffer (QIAGEN). Add 15µl 5M NaCl to the Eppendorf tube before adding 2µl 100mg/ml lysozyme and mix. Incubate the tube in a 37oC waterbath untill the solution become clear. This is when the cells have lysed. Add 2µl 25mg/ml proteinase K (QIAGEN) and the same amount of RNase. Add 27µl 10% SDS, invert the tube and incubate at 37oC for 20-30 minutes. If you are using a class 2 pathogen, but have your main work area in a class 1 lab, then after decontaminating the outside of the Eppendorf tube, you can now bring it into a level 1 lab. Add 97µl 5M NaCl and mix well. Wearing goggles, add 81µl CTAB (caution: corrosive, Toxic, Irritant so wear eye protection). Incubate at 65oC for 20 minutes. Add an equal volume (500µl) of 24:1 chloroform:isoamylalcohol mix (suspected carcinogen, toxic, irritant, so only use in a fume cupboard). Mix thoroughly but gently. Spin the tube at 13,000 RPM for 10 minutes, and transfer the upper layer formed to another tube, avoiding any precipitate. Add an equal
amount of isopropanol to this upper layer and mix thoroughly but gently. Spin for 10 minutes at 13,000 RPM. Remove all alcohol avoiding the pellet (use a vac line if possible), but don't allow to completely dry or else you may have difficulty resuspending the pellet. Resuspend the pellet in 100µl of TE buffer (more if the pellet is large).
Maxwell prep
To prepare your bacteria for this DNA extraction, WE followed the CTAB protocol until the first incubation step (when the solution goes clear after cell lysis). Because, as students, we were not allowed to use the machine, our lysed cells were put into the machine by Dr Badge. This got us our most reliable and best DNA extraction.
The extraction used the Maxwell Blood DNA extraction kit Maxwell Blood DNA extraction kit and the protocol used was that for Mouse tail genomic DNA extraction
PCR purification
We had problems with the gel extraction so we decieded to purify the DNA in the original PCR rather than running it on the gel. We did however run 10% of each aliquot on a gel to see if we had DNA and then after purification ran 10% of the purified PCR product on a gel to see if the purification was successful  by noticing that the bands were of correct size and were compatable with the nanodrop results of second purer gel QIAGEN PCR Purification Kit
Sau3A1 partial digest
A Sau3A1 partial digest was originally intended, after DNA extraction, to be a step towards making a DNA library, which we could then have grown on our polystyrene minimal media to find genome fragments that could degrade polystyrene. However, we ran out of time, and ended up attempting to PCR out Tod operon genes that we thought could be partially responsible for polystyrene degradation. We set up DNA digestion assays to find out the time taken to get fragments of a particular length (from 1Kbp to 10 Kbp). We started doing the digests at 37oC, but we realised that this caused Sau3A1 to digest DNA too quickly, even at very low concentrations.
The protocol
First off, a reaction mixture was prepared using: 4µg of DNA (in our case, we needed 53.7µl of DNA in buffer), 7.5µl 10x NE buffer 1, 0.75µl 100x BSA, 0.5µl Sau3A1 (to be added last), and topped up to a final total of 75µl with distilled H2O. 1µl Sau3A1 is 4 units of enzyme (1 unit can digest 1µg λ DNA in 1 hour @ 37oC in 50µl total reaction volume). The reaction we found, was easier to control and get better gel photos at room temperature. We made 8 stopping aliquots to stop samples of the reaction at time=0, 5, 10, 15, 20, 25, 30 and one taken before enzyme was added. Each stopping aliquot was made up of 4µl Loading dye, 2µl 0.5M EDTA and 6 µl H2O put in iced water. At each time, 8µl of reaction mixture was removed from the reaction mixture and added to a stop aliquot (EDTA binds to and removes Mg2+ ions from Sau3A1, inhibiting the enzyme, and the iced water stops the enzyme from working). These were then run on a 0.7% agarose gel and transilluminated to reveal a clear curve in DNA fragment length as time increased:
Although there are a few imperfections in the gel, making the DNA in each lane not run completely straight, a decrease in average sizes of DNA fragments is visible. The lanes are: X, 5µl DNA HyperLadder 1, X, No enzyme added, digest at time=0 minutes, digest at 5 minutes, digest at 10 minutes, digest at 15 minutes, digest at 20 minutes, digest at 25 minutes, digest at 30 minutes, X, 2µl DNA HyperLadder 1, X. At no enzyme added, the fragments are above 10Kbp, but by 30 minutes the fragments are averaging around 3Kb.
Colony counts
Photograph of an petri dish form the project
While we were attempting to use the QIAGEN Blood & Cell Culture DNA Maxi kit to extract DNA, we needed to know the number of cells in a broth so we could calculate the volume of overnight culture to use when using a genomic tip to extract chromosomal DNA from first E.coli then P. aeruginosa. However, the colony count was very unpredictable, with some dishes having so many colonies that they merged together, and others with no colonies at all, regardless of broth dilution.
The protocol
First prepare an overnight culture, and prepare 21 Luria agar petri dishes. The next day, take a reading of the overnight Luria broth. Inoculate 49.9ml of broth with 0.1ml of the overnight bacteria colony. Mix, start timer, and immediately put 1ml of culture into a spectrophotometer and take a reading at 600nm. Record this as time = 0, Start a timer, and take an aliquot of 100µl before putting in eppendorf marked as time 0, 10^-1 dilution. Add 900µl of Luria broth and mix, then take 100µl from this solution and add to next eppendorf, marked time 0, 10^-2 dilution and so on to 10^-7. Pipette the 300µl of dilutions 10^-5 to 10^-7 onto similarly labelled pre-prepared Luria agar plates and spread them under a Bunsen burner to prevent contamination. Repeat steps 3-5 every 40min (1 doubling of the bacterial  strains/species we used)until time = 240 minutes. Leave plates to grow overnight, then count the number of colonies growing on the most concentrated plate where colonies are distinguishable. Multiply up to the true concentration, then multiply by 10. This is colonies/ml. Draw graph to see correlation and check against known correlation to see if it has worked.
Doubling time
The doubling time experiment was designed to find out the approximate time taken for our bacteria to divide, so we could work out how often to sample the overnight broth in the colony count experiment. There were few problems with this expperiment, though the protocol had to be changed from sampling every 5 minutes to every 20 minutes as too little bacteria were left.
The protocol
Inoculate 49.5ml broth with 0.5ml of concentrated bacterial culture to give a 1% dilution (stock 1). Leave in warm room for 135min to move the culture out of lag phase and into log phase. Take 5ml of stock 1 and inoculate into 35ml of broth. Start timer, take spectrophotometer reading at 600nm (zeroed with broth) of 1ml of both stock 1 and the further 8 fold dilution (stock 2). After 20 mins, take a further reading of stock 2. Repeat for 120min, at which point the spectrophotometer reading should equal the stock 1 reading at time zero, if there have been 3 doublings in the time take
Polystyrene growth experiment
This experiment was to see whether the orange and yellow bacteria we isolated from CSE kit 01#502 (the only kit where bacteria grew on our minimal media- polystyrene sugar plate) were using the polystyrene as an energy source, or another one that we hadn't considered. After a brainstorm, we decided to aerate the polystyrene sugar further (as it had originally needed venting to remove hazardous levels of pentane when we were originally sent it (as pentane is used as an expanding agent in expanded polystyrene), as the bacteria could have been using pentane instead that was still present in small quantities within the sugar.
The protocol
Part 1
In an eppendorf add a swab of bacteria to 100ul PBS. Take 8 50ml corning tubes, and add 9.5ml minimal broth to each tube. To half of the tubes add 0.5g polystyrene sugar, and label them MMP (minimal media polystyrene). Add 0.5ml of further minimal broth to the other half and label them MM (minimal media only). Add 10ul PBS inoculated with yellow bacteria to one MMP and 1MM tube (labelling them yellow), and do the same again with 2 more tubes each time with orange and mixed colonies, labelling appropriately each time. The last 2 tubes should have 10ul PBS only (no bacteria) added and then labelled.
Part 2
The same protocol as above, only using 20 corning tubes, doing the experiment in triplicate (with 1 'no bacteria' for each of MM and MMP)
Part 3
A similar protocol to part one and 2, in triplicate but with the addition of 3 of each type of bacteria, and a blank with polystyrene sugar that has been aerated for 24 hours at 60oC to hopefully draw off any residual pentane still present in the sugar.
Results
16S sequencing
The 16S experiments were carried out using universal primers 28f and 519r to PCR amplify out the small subunit of the ribosome for sequencing and genus analysis. The experiment was carried out a multiple of times as there were many problems with the gel extraction, which lead to using PCR purification after size selection and re amplification to increase the yield making it possible to get sequence data
Sequences were then analysed on the NCBI database comparing the sequence to known 16S sequences.
The first set of sequencing which was carried out using what turned out to be a mixture of the bacteria we isolated, resulting in the following sequence caGCAAAGtATTAATTTaCTGCCCTTCCTCCCAACTTAAAGTGCTTTACAATCCGAAGACCTTCTTCACACACGCGGCATGGCTGGATCAGGCTTTCGCCCATTGTCCAATATTCCCCACTGCTGCCTCCCGTAGGAGTCTGGACCGTGTCTCAGTTCCAGTGTGACTGATCATCCTCTCAGACCAGTTACGGATCGTCGCCTTGGTGAGCCATTACCCCACCAACTAGCTAATCCGACCTAGGCTCATCTGATAGCGCAAGGCCCGAAGGTCCCCTGCTTTCTCCCGTAGGACGgATGCGGTATTAGCGTCCGTTTCCGAACGT
After running a BLAST search using the Tl/16S_ribosomal_RNA_Bacteria_and_Archaea database the top results were all Pseudomonas species
The second set of sequence data we retrieved was for the individual orange, yellow and the control P.aeruginosa
Results from P.aeruginosa 16S
>IG16S_1F TGCANTNGAGCGNNNGAAGGGGGCTTGGTCCTGANNTCTGAGTTGGACGGGGGAGTATNGCCTATGAATANGCTGGGTGGTGGGGGANGATNTCNGGTGNCAGGCNCNANTACTNCCTCCTTNCTGNNGGANANNGTGGANGATCTTNNGNCNNCANGCTATGCNATGAGCCTNGGNGNATNANNTTGTTGGNGGGGNAAAGGGCTANCAAGGCTACNATCCGTAACTGGNCTGANAGGATGATCAANCGTGATGGGNCTGANTGNCGCTGANAACTCCTNCAGAATGCTGCGGGGGGGAATATTGGAGAATGTTCGAAAGCCTGATCCAGCCCTTGCGAGNGTGTGCANAAGGGTTTCGAANGGTTTAGNATTTTAAATTTGTNNGAAGGGAAAAAAATTAATGNGTTGGTGTTTTGANNTTACTGTCANAATATCTTCCGNCCNAACTTANNGCTAACTNCNTGCNNAANNCCNNNNNAANANNNNANGTGNNCA
>IG16S_1R TACTGACCTTCCTTCaCtcTTAAAGtGcTTTACAaTaCgNANACCTTCTTCACACNCGCGGCATGGCTGGATCAGGATTTCCCCCNTTGTCNNNNNTTNCCCACNGCNGCCTCCCGNANNANTCTGGGNCNNNTNTNNNNTCCNGTGNNNNTGATNANCCTCTCNNACCNNTTANNGATCGTNNCCTTGGNNNNCTTTACCCCACCNACTANCTAATCCGAACTCAAGGTCATCTGATAGNGNGAGGTTTTAAAATCCCCCATTTTCTCCCTCGNGACATATGCTGTATTAGGGCCTGATTTCTGACGTTATCCCCCACTACCCGGCNAATTCCGTAGGCATTACTCACCCNTCCGCCCCTGAATCCANGAACCAGCACNATTCATCTCCTCCACTTGCNTGTGTTNTGCCTGCCGCNNGCNTTCANTCTGAGCCNNGANNGAACCTCAAANNNANNTCA
As you can see there are a lot of N’s in the sequence, with the top results being luckily all Pseudomonas species when searched on the NCBI BLAST database of 16S
Results from Orange 16S sequencing
>IG16S_3F GCAGTCGANCNNNNGANGCNNCTTGNCTNTNANGAGGGNNNGAACGGGTNNTAGTGCNTCGGCAGAGTNNACGGNNNGNNCNNGACCCAGGGNNTGGTGNNATTCCCACNAATCGAAGCTAATGCTGAGNNCTCCCTCAGACNGCANGGTNTGATGATGAANGGCCCGGCGGCNGCGCCTTTAGATGGNGTGGCAGGGGATTACTCACCTGGTGGGNNNATGGCCNCCCCACGGAACGAAGGATCCCNCNNGNTGNNNCTAAGNGACGNNACTGACTCANNGAGGNGACNTCNNNNANNTTNANAGATGGATGAAC
>IG16S_3R ACCTCTgaTacATGTTCTTCCCTTACAACAGAGaTTTACGATCCgAAAACCTTCGTCACTCACGCGGCGTgGCTCCATCAGACTTTCGTCCATTGTGGAAGATTCCCCACTGCTGCCTCCCGtAgGAGTCTGGGCCGTGTCTCAgTCCCAGGGTGGCCGATCACCCTCTCAgGTCGGcTATGcATCGTCGGCTTGGTGGGcCCTTACCCCACCAACTATCTAATGCGCCCCAAGGCCATCCCAAGGTGAAAGGGATTACCCTCTCATGCTATTACCATAAGGTCTTAAGAAGTATTACCtACTATCTCCGAgttctc
Sequence data after BLAST searching resulted in the top hits for both forward and reverse primers being Exiguobacterium . This however goes against the Gram Stain as the orange bacteria was gram –ve and Exiguobacterium are gram positive rod shaped bacteria http://www.ncbi.nlm.nih.gov/pubmed/19381755 . Another way we can conclude the bacteria was gram –ve is from the Maxwell preperations, with large amount of DNA being recovered from the orange without the addition of enzymes to break gram +ve outer cell walls. In contrast the yellow yielded little DNA as the enzymes used were for gram –ve bacteria and the gram stain also showed this was gram +ve
Results for yellow 16S sequencing
>IG16S_6R CGTCatatCCTAcgtataTCTagAtAgACCTTCTTCCatGatCAAcAGAgCTTTACGAtgCcGgaagacCTTCATCACTCACGCGGCGctGGCTgCATCAGGCTTTCGCCCATTGCCCAAGATTCCCCACTGCTGCCTCCCGNAGGAGTCTGGGCCGTGTCTCANTCCCAGTGTGGCTGNNCNTCCTCTC
>IG16S_6F TGCAGTCGACGgcagAGGGAtTGcTTGcgAcGTGAGtGGTGGAcGGGtGAGTAATGCGTacGAATCTGCCcCtTGCCCTCGGAACAGGCaTGgcaACCTGgAATgGTacCtAATACaaGATAATGttGTTTGCagGGtGtGCgaagAATgTtCGGNNNTGNANNCTNNNACTTGNNNNTGGGCCTGNNNGGGNTANNTNCTTGGTGGCCTAACGACNCNNNNTGGNNATGATGGGTAACCCACCTGNNAGGGTGTGCGGCCACNCTGGGACTGATACCCGGCGCAGACTCCTGGAAGTTGGANCANTGGGGAAACTTCCGAANCGGCCGATCCCCTGACNGAACAAAGGCC
Sequence data after BLAST searching resulted in various different bacteria after the low query coverage of the reverse sequence. Nevertheless the genus which had the best total score for forward and reverse was the genus Paenibacillus this could be correct as it agrees with the gram stain as a gram +ve http://www.ncbi.nlm.nih.gov/pmc/articles/PMC91948/ . However for both of the orange and yellow, better sequence analysis should be sort after for next year, but we have been able to give ball park genus and some of the characteristics of the two isolated bacteria
Polystyrene growth experiment
Tuesday 11th September
M.M (no bacteria)
0.002 abs
M.M (yellow)
0.03 abs
M.M (no bacteria)
0.021 abs
M.M (yellow)
0.057 abs
M.M.P (no bacteria)
0.009 abs
M.M.P (yellow)
0.072 abs
M.M.P (orange)
0.04 abs
M.M.P (mix)
0.046 abs
The significant results from this demonstrate that the yellow and the mix colonies grew better with polystyrene than without. Also the orange colonies have decreased growth demonstrating that this is not prehaps the ideal environment for their growth and therefore utilisation of polystyrene.
The controls containing no bacteria had very low readings in comparison indicating valid results. However further experimentation is needed to validate these results. Performing the experiment in triplicate is the next step.
Re-inoculated and repeated
 
Friday 14th September (abs)
Monday 17th September (abs)
 
 
Average Readings
 
Average Readings
Average Growth Over Days (abs)
M.M (no bacteria)
0
 
0
 
0
M.M.P (no bacteria)
0
 
0
 
0
M.M.P (yellow) X1
0.053
0.054333333
0.13
0.128666667
0.074333333
M.M.P (yellow) X2
0.052
 
0.126
 
 
M.M.P (yellow) X3
0.058
 
0.13
 
 
M.M.P (orange)X1
0.066
0.055666667
0.245
0.204
0.148333333
M.M.P (orange)X2
0.048
 
0.102
 
 
M.M.P (orange)X3
0.053
 
0.265
 
 
M.M.P (orange)X1
0.039
0.062666667
0.207
0.219666667
 
M.M.P (orange)X2
0.074
 
0.21
 
 
M.M.P (mix) X3
0.075
 
0.242
 
 
M.M (mix) X1
0.026
0.038333333
-0.03
-0.038333333
0.157
M.M (mix) X2
0.02
 
-0.043
 
 
M.M (mix) X3
0.069
 
-0.42
 
 
M.M (orange) X1
0.008
0.014
-0.023
-0.022333333
-0.076666667
M.M (orange) X2
0.019
 
-0.023
 
 
M.M (orange) X3
0.015
 
-0.021
 
 
M.M (yellow) X1
0.008
0.005333333
-0.02
-0.048
-0.053333333
M.M (yellow) X2
0.006
 
-0.009
 
 
M.M (yellow) X3
0.002
 
-0.115
 
 
After triplicating all our spectrophotometry readings we can see that mixed, yellow and orange colonies have increaded average growth in polystyrene, whereas in the absence of polystyrene their growth has declined. This indicates a possible need for a mixed, yellow and orange colonies to have access to polystyrene for theur growth using it as a sole carbon source. As in its absence it may indicate possible bacterial colony death, this assumption is based on the reduction in spectrophotometry readings between the two days. The spectrophotometer readings for each colony have at least two readings that are similar so this data can be classified as significant. However these results from the spectrophotometer lead to differing conclusions from what we can obtained on the 11/09/2012, so these results are still not conclusive.
Re-inoculated to test whether pentane is the energy source for growing bacteria.
The aerated treatment [DETAIL TREATMENT HERE!] is to hopefully eliminate pentane contamination by its removal to see that hopefully polystyrene is whats being used not pentane.
 
Friday 21st September (abs)
Average Readings (abs)
Monday 24th September (abs)
Average Readings (abs)
Average Growth Over Days (abs)
M.M (no bacteria)
0
0
0
0
0
M.M.P (aerated polystyrene no bacteria)
0
0
0
0
0
M.M.P (non-aerated polystyrene no   bacteria)
0
0
0
0
0
M.M.P (non-aerated,yellow) X1
0.012
0.013333333
-0.043
-0.033666667
-0.047
M.M.P (non-aerated,yellow) X2
0.013
 
-0.031
 
 
M.M.P (non-aerated,yellow) X3
0.015
 
-0.027
 
 
M.M.P (non aerated,orange)X1
0.01
0.015
-0.043
-0.033666667
-0.047
M.M.P (non aerated,orange)X2
0.009
 
-0.014
 
 
M.M.P (non aerated,orange)X3
0.026
 
-0.035
 
 
M.M.P (non aerated,mix) X1
0.002
0.014
-0.035
-0.016333333
-0.045666667
M.M.P (non aerated,mix) X2
0.038
 
0.003
 
 
M.M.P (non aerated,mix) X3
0.002
 
-0.017
 
 
M.M.P (aerated,mix) X1
0.005
0.013666667
-0.02
-0.015333333
-0.029
M.M.P (aerated,mix) X2
0.026
 
-0.012
 
 
M.M.P (aerated,mix) X3
0.01
 
-0.014
 
 
M.M.P (aerated,orange) X1
0.01
0.009333333
-0.027
-0.025666667
-0.035
M.M.P (aerated,orange) X2
0.009
 
-0.034
 
 
M.M.P (aerated,orange) X3
0.009
 
-0.016
 
 
M.M.P (aerated,yellow) X1
0.004
0.012333333
-0.008
-0.008333333
-0.020666667
M.M.P (aerated,yellow) X2
0.026
 
-0.005
 
 
M.M.P (aerated,yellow) X3
0.007
 
-0.012
 
 
M.M (yellow) X1
0.029
0.026
0.028
0.029666667
0.003666667
M.M (yellow) X2
0.025
 
0.027
 
 
M.M (yellow) X3
0.024
 
0.034
 
 
M.M (orange) X1
0.039
0.045
0.056
0.057
0.012
M.M (orange) X2
0.044
 
0.052
 
 
M.M (orange) X3
0.052
 
0.063
 
 
M.M (mix) X1
0.026
0.034
0.031
0.014666667
-0.019333333
M.M (mix) X2
0.056
 
0.006
 
 
M.M (mix) X3
0.02
 
0.007
 
 
These results are inconclusive so no firm conclusions can be drawn from this data. As a result we are not sure   if the pentane is the carbon source, or the polystyrene is.
Future research
Future iGEM Leicester team projects can continue looking for the elusive genes that can degrade polystyrene (as well as probably other substrates as well). Another approachis to concentrate on improving the enzymes within the pathway to accept polystyrene more readily via mutagenesis. Although we were unfortunately unable to amplify any genes present in the Tod operon from the isolated microbes, a future team, with the correct strain of P. putida, should be able to create a biobrick from genes of the Tod operon relatively easily. This that could potentially be partially responsible for polystyrene degradation. If future teams can find the elusive pathway(s) responsible, they may be able to couple this degradation (and the energy released) to synthesis of useful products (such as lactic acid for the synthesis of biofoam (a new product, very similar to polystyrene, but biodegradable in the right circumstances).
References
Atiq, N., Ahmed, S., Ali, M., Andleeb, S., Ahmad, B., Robson, G., 2010. Isolation and identification of polystyrene biodegrading bacteria from soil. African Journal of Microbiology Research. 4(14), 1537-1541.
[edit]
Retrieved from " http://2012.igem.org/Team:Leicester/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
