<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Fudan_China_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Fudan China/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_content_wrapper"><DIV id="nav"><DIV class="navList"><A href="https://2017.igem.org/Team:Fudan_China/Project"><H5>PROJECT</H5></A><UL style="display: none"><LI><A href="https://2017.igem.org/Team:Fudan_China/Description"><H6>DESCRIPTION</H6></A></LI><LI><A href="https://2017.igem.org/Team:Fudan_China/TSS"><H6>TWO-SIGNAL SYSTEM</H6></A></LI><LI><A href="https://2017.igem.org/Team:Fudan_China/MSS"><H6>MULTI-SIGNAL SYSTEM</H6></A></LI><LI><A href="https://2017.igem.org/Team:Fudan_China/Parts"><H6>PARTS</H6></A></LI></UL></DIV><DIV class="navList"><A href="https://2017.igem.org/Team:Fudan_China/Model"><H5>MODELING</H5></A><UL style="display: none"><LI><A href="https://2017.igem.org/Team:Fudan_China/IntModel"><H6>INTEGRASE MODEL</H6></A></LI><LI><A href="https://2017.igem.org/Team:Fudan_China/DifModel"><H6>DIFFUSION MODEL</H6></A></LI></UL></DIV><DIV class="navList"><A href="https://2017.igem.org/Team:Fudan_China/Notebook"><H5>NOTEBOOK</H5></A><UL style="display: none"><LI><A href="https://2017.igem.org/Team:Fudan_China/Timeline"><H6>TIMELINE</H6></A></LI><LI><A href="https://2017.igem.org/Team:Fudan_China/Protocol"><H6>PROTOCOL</H6></A></LI><LI><A href="https://2017.igem.org/Team:Fudan_China/InterLab"><H6>INTERLAB</H6></A></LI></UL></DIV><DIV class="navList"><A href="https://2017.igem.org/Team:Fudan_China/HP"><H5>HP</H5></A><UL style="display: none"><LI><A href="https://2017.igem.org/Team:Fudan_China/HP/Silver"><H6>GENERAL</H6></A></LI><LI><A href="https://2017.igem.org/Team:Fudan_China/HP/Gold_Integrated"><H6>INTEGRATED</H6></A></LI></UL></DIV><DIV class="navList"><A href="https://2017.igem.org/Team:Fudan_China/Member"><H5>TEAM</H5></A><UL style="display: none"><LI><A href="https://2017.igem.org/Team:Fudan_China/Member"><H6>TEAM MEMBER</H6></A></LI><LI><A href="https://2017.igem.org/Team:Fudan_China/Attributions"><H6>ATTRIBUTION</H6></A></LI><LI><A href="https://2017.igem.org/Team:Fudan_China/Collaborations"><H6>COLLABORATION</H6></A></LI></UL></DIV></DIV><DIV id="upButton"><H5><A href="#FudanChinaWikiBody" style="width:50px;color:inherit;">UP</A></H5></DIV><DIV id="head"><DIV id="logo"><H1>Description</H1></DIV><DIV id="FCcontent"><DIV id="contentLeft"><H2 style="top: 260px;">Overview</H2></DIV><DIV id="contentRight"><H2 style="top: -10px;">Background</H2></DIV></DIV></DIV><DIV id="main"><H2 id="back">Background - The era of cellular memory</H2><DIV class="row"><P>Biological memory has currently been defined as a sustained cellular response to a transient stimulus<SUP>1</SUP>. Cellular memory devices with cell or cell population have been developed in recent years. Some of them are based on transcriptional level, like toggle switch<SUP>2</SUP>, the others are based on DNA level by employing various recombinases<SUP>3, 4</SUP> or CRISPR<SUP>5, 6</SUP>. Although existing devices may have different functioning mechanisms, they all share the same ability to store the past and retrieve it whenever we need. Here, we will introduce some existing memory devices and the potential usage of cellular memory before introducing our project.</P></DIV><H3>Memory devices based on transcriptional level</H3><DIV class="row"><P>There are two types of devices in this category, toggle switches and positive-feedback loops. They all store the information by staying in a certain transcriptional state. Most of the devices here are mimics of circuits that have already existed in various organisms.</P></DIV><DIV class="row"><P>After the bacteriophage lambda switch was found in 2000, a genetic toggle switch was firstly constructed in <I>Escherichia coli</I><SUP>2</SUP> as a memory device(Fig.1). With this repression loop, the transcriptional system switches between two stable states. As shown, the repression loop mainly consists of two components, promoters and repressors(known as transcriptional factor). Repressor 1 inhibits Promoter 1 from expressing Represser 2, while Repressor 2 inhibits Promoter 2 from expressing Represser 1. In other words, Represser 1 and Represser 2 are mutual antagonists, which means only one of them can be highly expressed. Without inducers, both of the states are possible. However, after the induction, things may change. For example, when Repressor 1 is highly expressed, Inducer 1 can stop the inhibition of Repressor 1 and starts transcribing Repressor 2. After that, Repressor 2 represses Promoter 2, thus reducing the concentration of Represser 1 and switching to another state. Furthermore, even if Inducer 1 disappears after the transient induction, this state is stable. By adding a reporter following either of the repressors, we can retrieve the information whether the inducer has ever existed. When using this device, both inducers are allowed to be customized to detect and memorize various events, such as DNA damage and quorum sensing<SUP>7</SUP>.</P><DIV class="figure" style="width: 90%;"><B>Figure 1</B> | A representative memory device using toggle switch.
                </DIV></DIV><DIV class="row"><P>Another type of transcriptional-level-based memory device was designed later in <I>Saccharomyces cerevisiae</I>, which involves positive-feedback loops<SUP>8, 9</SUP>(Fig.2). This system is bistable as well. In one condition, Inducer 1 does not exist. As a result, Reporter 1 and Activator 2 cannot be transcribed by Promoter 1, making the positive-feedback loop untriggered. In another condition, Inducer 1 induces the sensor and transcribes Activator 2. Activator 2 then binds to Promoter 2, triggering the positive-feedback loop. After the trigger, Reporter 2 will be expressed sustainedly in the absence of Inducer 1. The information is thus stored. Similar to toggle switches, Inducer 1 can be customized. For example, positive-feedback loop was designed to memorize the existence of high pheromone levels in budding-yeast<SUP>10</SUP>.</P><DIV class="figure" style="width: 70%;"><B>Figure 2</B> | A representative memory device using positive-feedback loop.
    </DIV></DIV><H3>Memory devices based on DNA level</H3><DIV class="row"><P>Memory devices based on transcriptional level have two obvious disadvantages that information stored transmitting between two generations is not easy and that information will be lost after cell death. As a result, memory devices based on DNA level was developed, giving memory devices the ability to store information directly on DNA of chasis organisms. There are many types of devices in this category, respectively based on integrases(also known as recombinases) and CRISPR. </P></DIV><DIV class="row"><P>Memory devices using integrases was described in 2008<SUP>3</SUP>(Fig.3). Integrases recombination systems can catalyze reactions including inversion(Fig.3 a), deletion(Fig.3 b) and integration(Fig.3 c) of DNA sequence at specific sites. When two recognition sites(attB/P for serine integrases) are in opposite direction, integrases inverses the DNA sequence between two sites, functioning like a switch. To make it become a true memory device, one inducer controls the expression of the integrase. After the transient induction, integrase is expressed and completes the inversion. We can acquire the stored information in multiple ways, such as PCR or sequencing. Due to the unidirectional catalyzation manner of serine integrases, the storage of the information is not rewritable in common circumstances. However, with directional factors, the recombination reaction can be reversed, thus making rewritable memory devices possible<SUP>11</SUP>. Integrase-based memory device has been adopted to build more complicated system like state machine<SUP>3, 12</SUP> and counter<SUP>13</SUP>. Memory device using several orthogonal serine integrases to record multiple events was also developed in 2014<SUP>4</SUP>.</P><DIV class="figure" style="width: 90%;"><B>Figure 3</B> | Three recombination reactions catalyzed by integrases (serine integrases). a, Inversion of DNA sequence; b, Deletion of DNA sequence; c, Integration of DNA sequence.
                </DIV></DIV><DIV class="row"><P>Memory device that genomically encodes analog signal using beta-recombinase in cell population was successfully built in 2014<SUP>14</SUP>(Fig.4). Beta-recombinase recombines ssDNA to its homologous sites on the genome. After building a retron with the ssDNA homologous to the desired site and making it inducible by certain inducer, we can retrieve the information of the inducer by simply detecting this homologous site. More inducer means more ssDNA. And more ssDNA means higher chance that recombination will take place in single cell of the population. So it can store not only the existence of the induction, but also strength and duration. If the site is on CDS of a reporter which the recombination of the ssDNA can mutate or recover form mutation, we can easily measure the information stored in DNA by measuring the reporter. </P><DIV class="figure" style="width: 70%;"><B>Figure 4</B> | Cellular memory device based on Beta-recombinase which can be induced by two signals.
                </DIV></DIV><DIV class="row"><P>Some interesting works have been done with CRISPR, which creates new memory devices. Self-targeting guide RNA(stgRNA) directs Cas9 to where stgRNA itself is encoded on the genome<SUP>5</SUP>. Similar to other memory devices based on DNA level, Cas9 mutates DNA, which changes the sequence of future stgRNA. Newly-transcribed stgRNA will still target itself. By controlling the expression of Cas9 using customized inducer, events such as inflammation in human cells can be continuously recorded and stored. Much more excitingly, memory device using Cas1-Cas2 complex is also created<SUP>6</SUP>.</P></DIV><H3>The potential usage of cellular memory devices</H3><DIV class="row"><P> The ideal cellular memory devices are able to record stimulus exposure in the DNA circuits of the cells and express desired gene. This feature gives much potential to the cells.  </P><P> The cellular memory devices can record transient conditions and embed them in the circuits permanently to express genes. Once activated and stimulated, the cells are capable of storing the information to be accessed later. Some tough environmental detection tasks can be carried out in this way where the subject or target environment is harmful to humans or impracticable to place detecting machines. Also, some biological pathways and feedbacks deserve to be analyzed in vivo to better understand the human body, such as cell differentiation, tissue formation and malfunction of some organs. </P><P> In addition, information stored stably in the DNA circuits enable the cells carry out certain functions. In the biotechnical industry, cellular memory significantly reduces the consumption of inducers by a large scale, and maintain producing proteins for a longer time. Moreover, we can use the cells with stable memory to treat some diseases. The cells will be triggered by biomarkers of the disease and produce functional proteins chronologically.  </P></DIV><H2 id="over">Project overview - Developing the concept of cellular memory</H2><DIV class="row"><P>It seems that a new era of highly developed cellular information processing methods is approaching. It is so exciting that existing memory devices are diverse and useful in various conditions, but we find that they can only record the static state at the instant the recording action happens. Therefore, they are unable to monitor the dynamic changing process of one signal.</P><P>This year, we want to develop the concept of cellular memory, and to realize the monitoring of one changing signal.Namely, we want to record the different concentrations of one single inducer at different time periods.  So, we designed a unique memory device, <I> MemOrderY </I>, with sequential memory structure using integrases. After measuring the orthogonality and efficiency of our recombinases, we try to engineer our <I>E. coli</I> population to record several static states of the target signals at different time points (Fig.5 a).</P><DIV class="figure" style="width: 90%;"><B>Figure 5</B> | Project overview. a, Target and Clock signals are processed and interpreted into three types of information; b, Recording one changing signal in different time periods(shown in grey). Data recorded is shown; c, overview of our two-signal system circuit, details shown on the Two-signal system page.
                </DIV><P>Inspired by the work of making cellular counter<SUP>13</SUP>, a more sophisticated circuit is designed to achieve our goal (Fig.5 b, c). This circuit contains three parts: logic gates, sequential memory structures and basic orthogonal memory modules. The logic gates sense two signals. One is the signal we want to record, called Target. The other is the signal used to provide the circuit with information about time, called Clock. Actually, Clock signal can be any signal, such as sunlight and other commonly-used inducers, as long as it can oscillate automatically or artificially.The logic gates process the given signal and transform it into one of the three basic output patterns listed in the table above, making the signal ‘sensible’.The sequential memory structures sense these output patterns to determine whether to trigger recording and in which order the recording action happens. In each time period determined by the Clock signal, different orthogonal integrases are adopted to write the information directly on DNA at specific sites. We can investigate these sites on DNA to get the information of the inducer in various past time periods.</P><P>Thus, by arranging the results in the right time order, we can finally know how the signal changes with time. </P></DIV><H3>References</H3><OL><LI>Burrill, D.R., and Silver, P.A. (2010) Making Cellular Memories. <I>Cell.</I> 140(1):13-8. </LI><LI>Gardner, T.S., Cantor, C.R., and Collins, J.J. (2000) Construction of a genetic toggle switch in <I>Escherichia coli</I>. <I>Nature</I>. 403(6767):339-42. </LI><LI>Ham, T.S., Lee, S.K., Keasling, J.D., and Arkin, A.P. (2008) Design and construction of a double inversion recombination switch for heritable sequential genetic memory. <I>PLoS One</I>. 3(7):e2815.</LI><LI>Yang, L., Nielsen, A.A., Fernandez-Rodriguez, J., McClune, C.J., Laub, M.T., Lu, T.K., and Voigt, C.A. (2014) Permanent genetic memory with &gt;1-byte capacity. <I>Nat Methods</I>. 11(12):1261-6. </LI><LI>Perli, S.D., Cui, C.H., and Lu, T.K. (2016) Continuous genetic recording with self-targeting CRISPR-Cas in human cells. <I>Science</I>. 353(6304). pii: aag0511. </LI><LI>Shipman, S.L., Nivala, J., Macklis, J.D., and Church, G.M. (2016) Molecular recordings by directed CRISPR spacer acquisition. <I>Science</I>. 353(6298): aaf1175. </LI><LI>Kobayashi, H., Kaern, M., Araki, M., Chung, K., Gardner, T.S., Cantor, C.R., Collins, J.J. (2004) Programmable cells: interfacing natural and engineered gene networks. <I>Proc Natl Acad Sci U S A</I>. 101(22):8414-9.</LI><LI>Becskei, A., Séraphin, B., Serrano, L. (2001) Positive feedback in eukaryotic gene networks: cell differentiation by graded to binary response conversion. <I>EMBO J</I>. 20(10):2528-35.</LI><LI>Ingolia, N.T., Murray, A.W. (2007) Positive-feedback loops as a flexible biological module. <I>Curr Biol</I>. 17(8):668-77.</LI><LI>Ajo-Franklin, C.M., Drubin, D.A., Eskin, J.A., Gee, E.P., Landgraf, D., Phillips, I., Silver, P.A. (2007) Rational design of memory in eukaryotic cells. <I>Genes Dev</I>. 21(18):2271-6.</LI><LI>Bonnet, J., Subsoontorn, P., Endy, D. (2012) Rewritable digital data storage in live cells via engineered control of recombination directionality. <I>Proc Natl Acad Sci U S A</I>. 109(23):8884-9.</LI><LI>Roquet, N., Soleimany, A.P., Ferris, A.C., Aaronson, S., Lu, T.K. (2016) Synthetic recombinase-based state machines in living cells. <I>Science</I>. 353(6297):aad8559.</LI><LI>Friedland, A.E., Lu, T.K., Wang, X., Shi, D., Church, G. and Collins, J.J. (2009) Synthetic gene networks that count. <I>Science</I>. 324(5931):1199-202. </LI><LI>Farzadfard, F., Lu, T.K. (2014) Genomically encoded analog memory with precise in vivo DNA writing in living cell populations. <I>Science</I>. 346(6211):1256272.</LI></OL></DIV><DIV id="footer"><DIV id="acknowledgement"><DIV style="position:absolute;left: 100px;">复旦大学Fudan University</DIV><DIV style="position:absolute;top:110px;left: 140px;">复旦大学生命科学学院School of Life Science, Fudan University</DIV><DIV style="position:absolute;top:220px;left: 180px;">云锋基金Yunfeng Capital</DIV></DIV><DIV id="contact"><H2>Contact us</H2>
        E-mail:  igem@fudan.edu.cn
        WeChat:</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>