<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Duke_Notebook_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Duke/Notebook/July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Duke/Notebook/July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Notebook/July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Notebook/July&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Duke/Notebook/July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Duke/Notebook/July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><UL id="navbar"><LI><A href="https://2014.igem.org/Team:Duke">Home</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team">About Us</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Duke">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team/Gallery">Photo Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team/Attributions">Attributions</A></LI></UL><LI><A href="https://2014.igem.org/Team:Duke/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#summary">Summary</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#future">Future Directions</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Overview">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Months">Notebook by Month</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy">Policy and Practice</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#threed">3D Printing</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#hoco">House Course</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#smath">NCSSM Team</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Safety">Safety</A></LI></P><DIV class="day"><A id="jul1"><H2> July 1 </H2></A><DIV class="lab"><P class="obj">Objective: make LB+Cm plates </P><UL><LI>Four sleeves of plates with 4x500mL medium</LI><LI>34mg/mL chloramphenicol</LI></UL><P class="obj">Objective: insert crRNAs into pdCas9 and scaffold</P><P>Plate results</P><UL><LI>Experimental had lots of colonies, but backbone-only control did as well</LI></UL><UL><LI>reasoning: similarity of BsaI overhangs may lead to re-ligation</LI><LI>solution: treat with CIP before ligation</LI></UL><P>Agarose gel of BsaI cuts of pdCas9 and Repeat scaffold </P><UL><LI>plasmids used in 6/30/14 transformation</LI><LI>to make sure BsaI is cutting properly</LI><LI>Results: single band in both appears to be cut DNA</LI></UL><P>CIP treatment of pdCas9 and Repeat scaffold (both BsaI cut)</P><UL><LI>pdCas9: 10uL plasmid in 20 uL reaction with 1.5uL CIP</LI><LI>Repeat scaffold: 22uL plasmid in 30 uL reaction with 2 uL CIP</LI><LI>2 hours at 37C (longer than standard protocol)</LI></UL><P>PCR cleanup of pdCas9 and Repeat scaffold CIP treatments</P><UL><LI>Final concentrations too low to be of use</LI></UL><UL><LI>Particularly negligible in pdCas9</LI><LI>Need to culture and cut more plasmid to try ligation again</LI></UL><P>Spread plates from pdCas9 and Repeat scaffold frozen stocks</P><UL><LI>Used tube “1” for repeat scaffold</LI><LI>Plated on fresh LB+Cm plates</LI></UL><UL><LI>Plates still slightly wet, difficult to spread evenly</LI></UL><P class="obj">Objective: make CCEC stock of DH5alpha-ZI strain</P><P>Diluted back 5mL culture in morning</P><UL><LI>1mL into 5mL LB+Spec</LI></UL><UL><LI>note for future: Spec not necessary for most procedures, chromosomally integrated genes are more stable</LI></UL><P>Inoculated 1 mL culture into each of two 250mL flasks</P><UL><LI>Flasks contain SOC</LI><LI>30C shaker for overnight growing</LI></UL></DIV></DIV><DIV class="day"><A id="jul2"><H2> July 2 </H2></A><DIV class="lab"><P class="obj">Objective: Make CCEC stock of DH5alpha-ZI strain</P><P>Made fresh CCMB-80 buffer stock</P><UL><LI>precipitate formed during pH adjusting and may have filtered out</LI></UL><UL><LI>could be detrimental to final product</LI></UL><P>Followed iGEM online protocol to make CCEC from Z1 strain</P><UL><LI>Overnight culture was much more concentrated than recommended</LI></UL><UL><LI>OD &gt;1</LI><LI>Did not have enough SOC to dilute back fully, so diluted back only a bit</LI></UL><UL><LI>Had to vortex to resuspend cells at two stages of procedure</LI><LI>Final OD after procedure ~1.6 for 100mL of cells</LI><LI>Aliquoted 1mL into tubes, froze at -80C</LI></UL><P>Next steps: </P><UL><LI>Test competence with useful strains containing Lac, Tet promoters</LI></UL><P class="obj">Objective: insert crRNAs into pdCas9 and Repeat-scaffold</P><P>Culture colonies from plates of pdCas9 and pSB1C3-Repeat-scaffold</P><UL><LI>3 colonies per plate in LB+Cm medium at 37C overnight</LI><LI>Plates were messy--hard to get individual colonies </LI></UL></DIV></DIV><DIV class="day"><A id="jul3"><H2> July 3 </H2></A><DIV class="lab"><P class="obj">Objective: insert crRNAs into pdCas9 and Repeat-scaffold</P><P>Miniprep cultures from 7/2</P><UL><LI>Three tubes of pdCas9 with concentrations 143.4, 145.7, 151.4 ng/uL</LI><LI>Three tubes of Repeat-scaffold with concentrations 166.6, 145.8, 159.6 ng/uL</LI></UL><P>Digest tubes 1 and 2 of pdCas9 and tubes 1 and 2 of Repeat-scaffold</P><UL><LI>to insert crRNAs</LI><LI>Digest with BsaI in cutsmart buffer</LI><LI>50 uL DNA in 100 uL reactions with 7.5 uL BsaI</LI><LI>3 hours 20 min at 37C</LI></UL><P>PCR cleanup of BsaI digest</P><UL><LI>final concentrations (in 30 uL each):</LI></UL><UL><LI>pdCas9: 65.4, 70.5 ng/uL</LI><LI>Repeat-scaffold: 122.3, 114.8 ng/uL</LI></UL><P>CIP treatment of pdCas9 and Repeat-scaffold BsaI digests</P><UL><LI>Two digest tubes pooled into one CIP tube for each backbone</LI><LI>60 uL DNA in 100 uL reactions with 5 uL CIP in cutsmart buffer</LI><LI>1.5 hours at 37C</LI></UL><P>PCR cleanup of CIP treatment</P><UL><LI>final concentrations (in 30 uL):</LI></UL><UL><LI>pdCas9: 32.5 ng/uL</LI><LI>Repeat scaffold 129.9 ng/uL</LI></UL><P>Annealed oligos of crRNA inserts for ligation</P><UL><LI>3 crRNAs (GFP1, GFP2, GFP3) with 2 oligos each</LI><LI>5uL each oligo heated to ~95C, slowly brought to rt in water bath</LI></UL><P>Ligation of crRNA inserts into pdCas9 and Repeat-scaffold</P><UL><LI>6 working tubes + 5 controls</LI></UL><UL><LI>pdCas9 + each crRNA (GFP1, GFP2, or GFP3)</LI><LI>Repeat-scaffold + each crRNA (GFP1, GFP2, or GFP3)</LI><LI>pdCas9 Backbone only control</LI><LI>Repeat-scaffold Backbone only control</LI><LI>Insert only controls for each insert</LI></UL><UL><LI>100ng = 3uL pdCas9, BsaI digested and CIP treated and cleaned</LI><LI>100ng = 0.8uL Repeat-scaffold, BsaI digested and CIP treated and cleaned</LI><LI>1uL annealed oligo insert</LI><LI>10 uL reactions with 0.5 uL T4 Ligase</LI><LI>Transformed into DH5alpha and plated on LB+Cm</LI></UL><P class="obj">Objective: Test competency and effectiveness of DH5alpha-Z1 strain</P><P>CCEC Transformation efficiency test</P><UL><LI>Transformed 5 tubes of pSB1C3-K741002 into DH5alpha-Z1 CCEC</LI></UL><UL><LI>Total amount of DNA in 10 uL: 130ng, 13ng, 1.3ng, 0.13ng, and 0.013ng</LI></UL><UL><LI>Also for use in testing pLac regulation in Z1 strain</LI></UL><P>Transformed iGEM kit plate 3-6G into DH5alpha-Z1</P><UL><LI>contains pSB1C3-BBa_I13521: pTet-RBS-mRFP</LI><LI>in order to test regulation of pTet in Z1 strain</LI></UL></DIV></DIV><DIV class="day"><A id="jul4"><H2> July 4 </H2></A><DIV class="lab"><P class="obj">Objective: ligate crRNAs into pdCas9 and Repeat-scaffold</P><P>Plate results for ligation of crRNA GFP1, GFP2, and GFP3 into two backbones:</P><UL><LI>~1000 colonies on all experimental plates</LI><LI>BO control for pdCas9 has ~500 colonies</LI><LI>BO control for Repeat scaffold has ~1000 colonies</LI><LI>IO controls all have 0 colonies</LI></UL><P>Cultured 4 copies of each crRNA-GFP1,2,3 in pdCas9 backbone</P><UL><LI>12 tubes total in LB+Cm</LI><LI>None from Repeat-scaffold attempts</LI></UL><P class="obj">Objective: Test competency and effectiveness of DH5alpha-Z1 strain</P><P>CCEC Test plate results:</P><UL><LI>130 ng: 100s of colonies</LI><LI>13 ng:~100 colonies</LI><LI>1.3 ng: ~20 colonies</LI><LI>0.13 ng: 0 colonies</LI><LI>0.013 ng: 0 colonies</LI><LI>Competency ~10 colonies/ng (low but working)</LI></UL><P>Results of transformation of pSB1C3-BBa_I13521 into DH5alpha-Z1</P><UL><LI>Plate had ~5 colonies</LI></UL><P>Cultured colonies for Tet and Lac testing in DH5alpha-Z1</P><UL><LI>One colony of pSB1C3-K741002</LI><LI>One colony of pSB1C3-BBa_I13521</LI><LI>In LB+Cm</LI></UL></DIV></DIV><DIV class="day"><A id="jul5"><H2> July 5 </H2></A><DIV class="lab"><P class="obj">Objective: Insert crRNAs into pdCas9 backbone</P><P>Saved stocks of 12 strains in 15% glycerol at -80C</P><UL><LI>4 tubes each of potential GFP1,2,3 crRNA inserts in pdCas9</LI></UL><P>Miniprepped 12 tubes of pdCas9 with potential GFP-1,2,3 inserts</P><UL><LI>4 tubes each </LI><LI>Eluted in 50 uL EB, all concentrations 100-200 ng/uL</LI></UL><P class="obj">Objective: Test effectiveness of DH5alpha-Z1 strain</P><P>Saved stock of pSB1C3-K741002 in DH5alpha (15% glycerol at -80C)</P><UL><LI>Culture of I13521 did not grow</LI></UL><P>Cultured another colony of pSB1C3-BBa_I13521</P><UL><LI>from 7/3 plate</LI></UL></DIV></DIV><DIV class="day"><A id="jul6"><H2> July 6 </H2></A><DIV class="lab"><P class="obj">Objective: Test effectiveness of DH5alpha-Z1 strain</P><P>Saved stock of pSB1C3-I13521 in DH5alpha (15% glycerol at -80C)</P></DIV></DIV><DIV class="day"><A id="jul7"><H2> July 7 </H2></A><DIV class="lab"><P class="obj">Objective: insert crRNAs GFP1,2,3 into pdCas9</P><P>Analytical restriction digest </P><UL><LI>12 tubes with potential inserts (4 per crRNA)</LI><LI>pdCas9 as a control</LI><LI>BsaI and XbaI in cutsmart for 1.5 hrs</LI></UL><P>Agarose gel of restriction digest</P><UL><LI>Gel 1:</LI></UL><UL><LI>1-4: pdCas9-crRNA-GFP1</LI><LI>5-8: pdCas9-crRNA-GFP2</LI><LI>9-12: pdCas9-crRNA-GFP3</LI></UL><UL><LI>Gel 2:</LI></UL><UL><LI>pdCas9 (control)</LI></UL><UL><LI>Expected single band ~9.3 kb if BsaI cut and ligation worked</LI><LI>Control and uncut failures expect two bands, 5.0 and 4.2 kb</LI><LI>Results: all 12 samples have single band</LI></UL><UL><LI>No uncut plasmids</LI><LI>Could still be recircularized with no insert</LI></UL><P>gel pictures:</P><P>Anneal oligos of 3 crRNA-GFP inserts</P><UL><LI>Following Dewran’s crRNA protocol</LI><LI>2 uL each oligo diluted in PBS to 20 uL</LI><LI>Placed in 98C water bath, cooled gradually to rt</LI></UL><P>Phosphorylate oligo inserts of crRNAs</P><UL><LI>5 uL annealed oligo mix in 20uL 1x ligase buffer with 1uL PNK</LI><LI>30 mins at 37C, then deactivated for 20 mins at 65C</LI><LI>Diluted 1/50</LI></UL><P>Ligation </P><UL><LI>75 ng backbone = 0.6 uL Repeat scaffold or 2.5 uL pdCas9</LI></UL><UL><LI>Both BsaI cut and CIP treated 7/3/14</LI></UL><UL><LI>1.5 uL diluted, phosphorylated, annealed oligo insert</LI><LI>6 experimental tubes + 2 BO controls and 3 IO controls</LI><LI>Transformed into iGEM CCEC (DH5alpha) and plated on LB+Cm</LI></UL><P>Cultured 2 colonies from pdCas9 plate</P><UL><LI>For Charlie to try his hands at ligation</LI></UL><P class="obj">Objective: Prepare various backbones for multiple-plasmid use</P><P>Transformed two distribution kit parts</P><UL><LI>Plate 4-6H: pSB4K5-J04450</LI></UL><UL><LI>backbone contains pSC101 origin with Kan resistance</LI></UL><UL><LI>Plate 4-18A: pSB3K3-I20260</LI></UL><UL><LI>backbone contains p15A origin with Kan resistance</LI><LI>On LB+G418 plates </LI></UL><P>Streaked plate from frozen stock</P><UL><LI>pSB6A1-J04450</LI></UL><UL><LI>backbone contains pMB1 origin with Amp resistance</LI><LI>on LB+Amp</LI></UL><P class="obj">Objective: Test pTet and pLac in DH5alphaZ1 strain</P><P>Streaked plates from frozen stock</P><UL><LI>pSB1C3-K741002 in DH5alpha-Z1</LI></UL><UL><LI>contains pLac-GFP</LI></UL><UL><LI>pSB1C3-I13521 in DH5alpha-ZI</LI></UL><UL><LI>contains pTet-RFP</LI></UL><UL><LI>On LB+Cm plates</LI></UL></DIV></DIV><DIV class="day"><A id="jul8"><H2> July 8 </H2></A><DIV class="lab"><P class="obj">Objective: insert crRNAs into pdCas9</P><P>Plate results from 7/7 transformation</P><UL><LI>pdCas9-crRNAs: ~50 colonies each</LI><LI>pdCas9 BO: 100s of colonies</LI><LI>Repeat-crRNAs: 100s of colonies</LI><LI>Repeat BO: ~1000 colonies</LI><LI>IO controls: 0 colonies, ~15 for crRNA-GFP3</LI><LI>Notes: unsure why BO controls have more colonies than experimental</LI></UL><P>Restriction digest of 12 samples from 7/5 prep + pdCas9 control</P><UL><LI>With SacI/NheI</LI></UL><UL><LI>Should show difference between insert and recircularized backbone</LI></UL><P>Made and ran 1.5% agarose gel in TBE</P><UL><LI>for better resolution of small band differences in pdCas9 with/without insert</LI></UL><UL><LI>1: pdCas9 (control)</LI><LI>2-5: pdCas9-crRNA-GFP1 (1-4)</LI><LI>6-9: pdCas9-crRNA-GFP2 (1-4)</LI><LI>10-13: pdCas9-crRNA-GFP3 (1-4)</LI><LI>14: pdCas9 (control)</LI></UL><UL><LI>expected: </LI></UL><UL><LI>3.1, 2.8, and 2.7 kb bands</LI><LI>585 bp band with no insert, 620 bp band with insert (pdCas9 620bp)</LI></UL><UL><LI>Results: One sample, crRNA-GFP3-3, has larger band matching control</LI></UL><UL><LI>This colony may be correct, should be sequenced</LI><LI>Others have smaller band, are probably incorrect</LI></UL><P>Culture 40 colonies from existing plates to screen for crRNA inserts</P><UL><LI>10 each from crRNA-GFP1 and crRNA-GFP2 transformed 7/3/14</LI><LI>10 each from crRNA-GFP1 and crRNA-GFP2 transformed 7/7/14</LI><LI>in LB+Cm</LI></UL><P class="obj">Objective: Prepare various backbones for multiple-plasmid use</P><P>Prepared LB+Kan medium</P><UL><LI>50 ug/mL final concentration Kanamycin</LI></UL><P>Cultured colonies from transformations and stock streaking</P><UL><LI>3 colonies each from 3 plasmids:</LI></UL><UL><LI>pSB6A1-J04450 in LB+Amp</LI><LI>pSB3K3-I20260 in LB+Kan</LI><LI>pSB4K5-J04450 in LB+Kan</LI></UL><P class="obj">Objective: Test pLac and pTet in DH5alpha-Z1</P><P>Notes:</P><UL><LI>Need to obtain aTc for pTet testing</LI></UL><UL><LI>Tried to use Doxycycline, but had trouble with solubility in ethanol (?)</LI></UL><UL><LI>pLac is not the final promoter to be used, a different variant will be used</LI></UL><UL><LI>with no glucose dependency</LI></UL><UL><LI>Flow testing of pLac and pTet was suspended until these changes are made</LI></UL></DIV></DIV><DIV class="day"><A id="jul9"><H2>July 9</H2></A><DIV class="obj">Objective: Insert crRNAs into pdCas9</DIV><DIV class="lab"><P>Miniprep 40 cultures of potential crRNA inserts in pdCas9 </P><UL><LI>Labeled with three numbers</LI><UL><LI> Date of transformation - crRNA-GFPx - sample number</LI></UL></UL><P>700uL of each culture saved to freeze in glycerol if colonies appear successful </P><UL><LI> Stored at 4C for the day </LI></UL><P> Analytical restriction digest of potential crRNA inserts</P><UL><LI>All 40 tubes + 2 pdCas9 tubes as control</LI><LI>1 uL DNA in 10 uL digest for 2 hrs at 37C </LI><LI>SacI-HF and NheI-HF in cutsmart </LI></UL><P> Made and ran 1.5% agarose/TBE gels</P><UL><LI> To differentiate between small band differences</LI><LI> Gel 1:</LI><UL><LI>1. 2-log ladder</LI><LI>2. pdCas9 </LI><LI>3-12. pdCas9-crRNA-GFP1 from 7/3 transformation</LI><LI> 13-22. pdCas9 -crRNA-GFP2 from 7/3 transformation</LI><LI>23. pdCas9 </LI><LI>24. 2-log ladder </LI></UL><LI> Gel 2:</LI><UL><LI>1. 2-log ladder</LI><LI>2. pdCas9 </LI><LI>3-12. pdCas9-crRNA-GFP1 from 7/7 transformation</LI><LI> 13-22. pdCas9 -crRNA-GFP2 from 7/7 transformation</LI><LI>23. pdCas9 </LI><LI>24. 2-log ladder </LI></UL><LI> Expected results:</LI><UL><LI> pdCas9 and successful inserts:</LI><UL><LI> 3.1, 2.8, and 2.7 kb bands, plus 620 bp insert band</LI></UL><LI> unsuccessful recircularizations: </LI><UL><LI> 3.1, 2.8, and 2.7 kb bands, plus 585 bp insert band</LI></UL></UL><LI>Results:</LI><UL><LI>Gel 1 appears to only be recircularized </LI><LI> Gel 2 has some promising inserts:</LI><UL><LI> GFP1 samples 1,2, and 3 from 7/7 transformation</LI><LI> GFP2 samples 7,8, and 9 from 7/7 transformation</LI></UL></UL><LI> Glycerol stocks frozen for these six samples</LI><LI>Next Steps: </LI><UL><LI> Run digest with BsaI to confirm that plasmids are not uncut pdCas9</LI><LI> Sequence to confirm presence of insert (using oligos ordered 7/8/14)</LI></UL></UL></DIV><DIV class="obj">Objective: Obtain new backbones and switch inserts into new backbones</DIV><DIV class="lab"><P>Miniprep 3 new backbones </P><UL><LI>pSB6A1-J04450 (3 copies)</LI><UL><LI> concentrations 437.6, 483.9, and 450.1 ng/uL</LI></UL><LI>pSB3K3-I20260 (3 copies)</LI><UL><LI> concentrations 114.6, 120.0, and 97.3 ng/uL</LI></UL><LI>pSB4K5-J04450 (3 copies)</LI><UL><LI> concentrations 129.3, 165.9, and 155.1 ng/uL</LI></UL></UL><P>Prep-scale digest of all 9 backbone tubes</P><UL><LI> 50 uL DNA in 60 uL total reaction</LI><LI>EcoRI-HF/Spe-HF in cutsmart </LI><LI> 3 hours at 37C</LI></UL><P> Agarose gel extraction of 3 backbones</P><UL><LI> Gel 1, Left: pSB6A1</LI><LI> Gel 1, Right: pSB3K3</LI><LI> Gel 2, Left: pSB4K5</LI><LI> All 3 lanes had 2 bands as expected</LI><UL><LI> upper band (backbone) in 3K3 had messy trail behind it (not extracted)</LI></UL><LI> All 180 uL from three tubes combined with 20 uL loading dye in x-large gel well</LI><LI> top band extracted from all 3 plasmids, split into 2 tubes each, stored at -20C</LI></UL><P> Streak plate from frozen stock</P><UL><LI> pSB1C3-K608012</LI><LI> In order to switch into pSB6A1</LI></UL></DIV><DIV class="obj">Objective: Remake new version of pDGC3</DIV><DIV class="lab"><P>Transformation from iGEM distribution kit plates </P><UL><LI>Plate 2-6D: pSB1C3-BBa_R0011 </LI><UL><LI> Engineered Lac promoter (not affected by glucose)</LI></UL><LI> Plate 3-15O: pSB1C3-BBa_I13500</LI><UL><LI> Strong RBS-GFP</LI></UL><LI> Plated on LB+Cm</LI></UL></DIV></DIV><DIV class="day"><A id="jul10"><H2>July 10</H2></A><DIV class="obj">Objective: Obtain backbones and switch parts into new backbones</DIV><DIV class="lab"><P> Gel cleanup of 3 backbones with Zymoclean prep kit</P><UL><LI>Some tubes had to be divided into two samples for cleanup </LI><LI> Final elution was 20 uL each into 2-3 tubes per backbone</LI><LI> Nanodrops looked unclean: Most had high peak around 220-240 nm</LI><UL><LI> pSB6A1: 300mg + 120 mg gel = 230.4 ng/uL </LI><UL><LI> Peak ~220, then hump ~260</LI></UL><LI> pSB6A1: 270 mg gel = 220 ng/uL</LI><UL><LI> Peak, then hump as before</LI></UL><LI> pSB3K3: 320 mg gel = 53.9 ng/uL</LI><UL><LI> No clear sign of DNA in graph</LI></UL><LI> pSB3K3: 230 mg gel = 20 ng/uL</LI><UL><LI> No DNA</LI></UL><LI> pSB3K3: 130 mg gel = 72 ng/uL</LI><UL><LI> Possibly DNA, best option but may not be useful</LI></UL><LI> pSB4K5: 290 mg gel = 43.4 ng/uL</LI><UL><LI> No clear sign of DNA in graph</LI></UL><LI> pSB4K5: 320 mg gel = 117.3 ng/uL</LI><UL><LI> Possibly DNA, best option but may not be useful</LI></UL></UL></UL><P> Culture 3 colonies of pSB1C3-K608012</P><UL><LI> In order to switch into pSB6A1</LI></UL></DIV><DIV class="obj">Objective: Insert crRNAs into pdCas9</DIV><DIV class="lab"><P> Analytical Digest of promising crRNA inserts </P><UL><LI> XbaI/BsaI digest</LI><LI> In order to confirm that plasmids are not uncut pdCas9</LI><LI> Samples 7-1-1,2,3 and 7-2-7,8,9 (6 tubes) plus pdCas9 as control</LI></UL><P> Agarose gel of digest results:</P><UL><LI> 1. pdCas9</LI><LI> 2-4. pdCas9-crRNA-GFP1, samples 1,2,3</LI><LI> 5-7. pdCas9-crRNA-GFP2, samples 7,8,9</LI><LI> Expected: </LI><UL><LI> 2 bands at 4.2 and 5.1 kb in pdCas9</LI><LI> 1 band at 9.3 kb in successful plasmids with inserts</LI></UL><LI> Results: </LI><UL><LI> pdCas9 appears to be an incomplete digest, showing 2 small and 1 large band</LI><LI> Lack of any trace of smaller bands in 6 experimental samples indicates successful inserts.</LI></UL></UL></DIV><DIV class="obj">Objective: Make pDGC3</DIV><DIV class="lab"><P> Culture 3 colonies each from pSB1C3-R0011 and pSB1C3-I13500</P></DIV><DIV class="obj">Objective: Test effectiveness of DH5alpha-Z1 strain</DIV><DIV class="ppl">Mitch and Sargis</DIV><DIV class="lab"><P> Culture pSB1C3-I13521 (pTet-RFP)</P><UL><LI> For flow </LI><LI> In aTc and non-aTc </LI></UL></DIV></DIV><DIV class="day"><A id="jul11"><H2>July 11</H2></A><DIV class="obj">Objective: Switch K608012 into pSB6A1</DIV><DIV class="lab"><P> Miniprep 3 copies of pSB1C3-K608012</P><UL><LI> concentrations 361, 324, 362 ng/uL</LI></UL><P> Prep-scale digest of pSB1C3-K608012</P><UL><LI> 2 miniprep tubes, 50 uL DNA in 100 uL each</LI><LI> With EcoRI and SpeI</LI><LI> 2+ hrs at 37C</LI></UL><P> Gel extraction and cleanup to isolate K608012</P><UL><LI> Cut lower band</LI><LI> Zymoclean prep kit</LI><LI> Each of 2 bands divided into two tubes during cleanup</LI><UL><LI> 1A: 280 mg = </LI><LI> 1B: 240 mg = </LI><LI> 2A: 200 mg = </LI><LI> 2B: 200 mg = 62.5 ng/uL (20 uL)</LI></UL><LI> Graphs on nanodrop did not look ideal: peak lower than usual</LI></UL></DIV><DIV class="obj">Objective: Create pSB1C3-R0011-I13500 as destination of mCherry G-block</DIV><DIV class="lab"><P> Miniprep pSB1C3-R0011 and pSB1C3-I13500</P><UL><LI> 3 copies each</LI><LI> Concentrations</LI><UL><LI> pSB1C3-R0011: 181, 229, 194 ng/uL</LI><LI> pSB1C3-I13500: 321.5, 228, 300.4 ng/uL</LI></UL></UL><P> Prep scale digests of two tubes from each miniprep</P><UL><LI> pSB1C3-R0011 with SpeI/PstI</LI><UL><LI> no extraction necessary</LI></UL><LI> pSB1C3-I13521 with XbaI/PstI</LI><UL><LI> to extract I13521 insert</LI></UL><LI> 100 uL total reaction per tube</LI><LI> 2-3 hrs at 37C</LI></UL><P> Agarose gel of pSB1C3-I13500 XbaI/PstI digests</P><UL><LI> Intended for gel extraction</LI><LI> Expected 2 bands, but only one present</LI><LI> No extraction performed</LI></UL><P> Analytical agarose gel of pSB1C3-R0011 SpeI/PstI digests</P><UL><LI> Single band appears correct</LI><LI> see below for gel picture</LI></UL><P> PCR cleanup of pSB1C3-R0011 SpeI/PstI digests</P><UL><LI> concentrations 124.8, 157.3 ng/uL (30 uL each)</LI></UL></DIV><DIV class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</DIV><DIV class="lab"><P> Miniprep pdCas9</P><UL><LI> concentration 167.4 ng/uL</LI></UL><P> Prep-scale digest of pSB1C3-R0011</P><UL><LI> 1 miniprep with XbaI/PstI</LI><LI> To extract pSB1C3 backbone</LI></UL><P> Agarose gel of pSB1C3-R0011 XbaI/PstI digest</P><UL><LI> Intended for gel extraction</LI><LI> Expected 2 bands, but only one present</LI><LI> No extraction performed</LI></UL><P> PCR of dCas9-tracrRNA and anti-tracrRNA</P><UL><LI> 4 tubes each, with pdCas9 as template in 0, 0.4, 0.7, and 1.0 uL quantities</LI><LI> oligos dCas9tracr-up and dCas9tracr-dn for dCas9-tracrRNA PCR</LI><LI> oligos dCas9tracr-up and tracrRNA-dn for anti-tracrRNA PCR</LI><LI> Q5 polymerase, with 64C annealing and 2 min extension</LI></UL><P> Agarose gel of PCR (and digest) results:</P><UL><LI> Lanes 1-4: dCas9-tracrRNA PCR (0, 0.4, 0.7, 1.0 uL template)</LI><LI> Lanes 5-8: tracrRNA PCR (0, 0.4, 0.7, 1.0 uL template)</LI><LI> Lanes 9-10: pSB1C3-R0011 SpeI/PstI digests</LI><LI> Results: </LI><UL><LI> dCas9-tracr PCR appears correct, band ~4 kb in 3 lanes</LI><LI> tracrRNA PCR unclear: strong primer-sized band may be correct, faint bands ~4 kb indicate off-target extension. </LI><UL><LI> tracrRNA PCR should be done with shorter extension time</LI></UL></UL></UL><P> PCR cleanup of dCas9-tracrRNA PCR</P><UL><LI> Concentration &gt;200 ng/uL</LI></UL></DIV><DIV class="obj">Objective: Test BsaI and XbaI enzymes</DIV><DIV class="lab"><P> Analytical digest of dCas9</P><UL><LI> One tube with BsaI/EcoRI</LI><LI> One tube with XbaI/EcoRI</LI><UL><LI> XbaI from tube with exp. 5/14</LI></UL><LI> One tube with XbaI/EcoRI</LI><UL><LI> XbaI from tube with exp. 2/15</LI></UL></UL><P> Agarose gel of digest:</P><UL><LI> 1. BsaI/EcoRI: expected 3 bands at 2.7, 2.9, and 3.5 kb</LI><LI> 2. XbaI (5/14) / EcoRI: expected 3 bands at 5.7, 2.1, 1.4 kb</LI><LI> 3. XbaI (2/15) / EcoRI: expected 3 bands at 5.7, 2.1, 1.4 kb</LI><LI> Results: Lanes 1 and 3 look as expected. Lane two has one extra band at 3.5 kb corresponding to partial digestion with XbaI. 5/14 XbaI tube is ineffective and has been thrown out</LI></UL></DIV></DIV><DIV class="day"><A id="jul14"><H2>July 14</H2></A><DIV class="obj">Objective: insert crRNAs into pdCas9</DIV><DIV class="lab"><P> Sequencing Order #1608</P><UL><UL><LI> BigDye reaction with BD buffer 1.1</LI></UL><UL><LI> pdCas9-crRNA_GFP1 sample 7-1 w/ pdCas9up1</LI></UL><UL><LI> pdCas9-crRNA_GFP1 sample 7-2 w/ pdCas9up1</LI></UL><UL><LI> pdCas9-crRNA_GFP1 sample 7-3 w/ pdCas9up1</LI></UL><UL><LI> pdCas9-crRNA_GFP2 sample 7-7 w/ pdCas9up1</LI></UL><UL><LI> pdCas9-crRNA_GFP2 sample 7-8 w/ pdCas9up1</LI></UL><UL><LI> pdCas9-crRNA_GFP2 sample 7-9 w/ pdCas9up1</LI></UL><UL><LI> pdCas9-crRNA_GFP3 sample 3 w/ pdCas9up1</LI></UL><UL><LI> pdCas9-crRNA_GFP1 sample 7-1 w/ pdCas9dn1</LI></UL><UL><LI> pdCas9-crRNA_GFP1 sample 7-2 w/ pdCas9dn1</LI></UL><UL><LI> pdCas9-crRNA_GFP1 sample 7-3 w/ pdCas9dn1</LI></UL><UL><LI> pdCas9-crRNA_GFP2 sample 7-7 w/ pdCas9dn1</LI></UL><UL><LI> pdCas9-crRNA_GFP2 sample 7-8 w/ pdCas9dn1</LI></UL><UL><LI> pdCas9-crRNA_GFP2 sample 7-9 w/ pdCas9dn1</LI></UL><UL><LI> pdCas9-crRNA_GFP3 sample 3 w/ pdCas9dn1</LI></UL></UL><P>Results: </P><UL><UL><LI> pdCas9-crRNA_GFP1 use sample 7-1</LI></UL><UL><LI> pdCas9-crRNA_GFP2 use sample 7-9</LI></UL><UL><LI> pdCas9-crRNA_GFP3 need to find more colonies</LI></UL></UL></DIV><DIV class="obj">Objective: Create pSB1C3-R0011-I13500 as destination of mCherry G-block</DIV><DIV class="lab"><P> Prep scale digest of pSB1C3-I13500 with XbaI/PstI</P><UL><LI> Tube 3 from 7/11/14 miniprep</LI><LI> To extract both I13500 and pSB1C3 backbone</LI><LI> 100 uL reaction with 50 uL DNA in cutsmart</LI></UL><P> Agarose gel extraction and cleanup of pSB1C3 and I13500</P><UL><LI> Top band (~2kb): pSB1C3</LI><LI> Bottom band (~1kb): I13500</LI><LI> Zymoclean prep kit</LI><LI> Concentrations: </LI><UL><LI> pSB1C3: 319 ng/uL in 20 uL (dirty)</LI><LI> I13500: 91.2 ng/uL in 20 uL (dirty)</LI></UL></UL><P> Ligation of pSB1C3-R0011 and I13500</P><UL><LI> 100ng = 0.7 uL pSB1C3-R0011 (#2)  cut with SpeI/PstI on 7/11</LI><LI> 150ng = 1.8 uL I13500 cut with XbaI/PstI on 7/14</LI><LI> Experimental, BO control, and IO control</LI><LI> rt for 1 hr</LI><LI> Heat shock transformation into DH5alpha and plated on LB+Cm</LI></UL></DIV><DIV class="obj">Objective: Switch K608012 into pSB6A1</DIV><DIV class="lab"><P> Ligation of pSB6A1 and K608012</P><UL><LI> 100 ng = 0.5 uL pSB6A1 (“Yellow” sample) cut with EcoRI/SpeI on 7/9</LI><LI> 150 ng = 2.5 uL K608012 (#2B) cut with EcoRI/SpeI on 7/11</LI><LI> Experimental, BO control, and IO control</LI><LI> rt for 1 hr</LI><LI> Heat shock transformation into DH5alpha and plated on LB+Amp</LI></UL></DIV><DIV class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</DIV><DIV class="lab"><P> Prep scale digests</P><UL><LI> dCas9-tracr PCR with XbaI/PstI (no extraction)</LI><LI> pSB1C3-K608012 (tube 3) with XbaI/PstI (to obtain pSB1C3)</LI><LI> 100 uL reaction each with 50 uL DNA in cutsmart</LI></UL><P> PCR cleanup of dCas9-tracr digest</P><UL><LI> Qiagen miniprep kit</LI><LI> Concentration 38.1 ng/uL in 30 uL</LI></UL><P> Agarose gel extraction and cleanup of pSB1C3</P><UL><LI> gel order: left K608012, right I13500</LI><LI> extracted pSB1C3, top band of K608012 (~2 kb)</LI><LI> Zymoclean kit</LI><LI> Concentration 127.9 ng/uL in 20 uL (dirty)</LI></UL><P> Ligation of pSB1C3 and dCas9-tracrRNA PCR</P><UL><LI> 100 ng = 3 uL dCas9-tracrRNA PCR cut with XbaI/PstI on 7/14</LI><LI> 150 ng = 0.6 uL pSB1C3 (from I13500) cut with XbaI/PstI on 7/14</LI><LI> Experimental, BO control, and IO control</LI><LI> rt for 1 hr</LI><LI> Heat shock transformation into DH5alpha and plated on LB+Cm</LI></UL></DIV><DIV class="obj">Objective: Create pTet-anti-tracrRNA</DIV><DIV class="lab"><P> Prep-scale digest of pSB1C3-R0040</P><UL><LI> With SpeI/PstI (no extraction necessary)</LI><LI> 100 uL reaction with 50 uL DNA in cutsmart</LI><LI> 4 hrs at 37C</LI></UL><P> PCR anti-tracrRNA from pdCas9</P><UL><LI> Oligos dCas9tracr-up and tracrRNA-dn</LI><LI> 0, 0.4, 0.7, and 1.0 uL pdCas9 template in 50 uL reactions</LI><LI> Q5 polymerase</LI><LI> annealed at 64C, 20 sec extension</LI></UL><P> Agarose gel of anti-tracrRNA PCR</P><UL><LI> Results: only 1.0 uL template lane appeared to amplify</LI><UL><LI> Need to make more using this template concentration</LI></UL></UL><P> PCR anti-tracrRNA from pdCas9</P><UL><LI> Oligos dCas9tracr-up and tracrRNA-dn</LI><LI> 1.0 uL pdCas9 template in each 50 uL reaction</LI><LI> 8 tubes with Q5 polymerase</LI><LI> annealed at 64C, 20 sec extension</LI></UL><P> Agarose gel of anti-tracrRNA PCR and pSB1C3-R0040 digest</P><UL><LI> Lanes 1-8: anti-tracrRNA PCR</LI><LI> Lane 9: R0040 digest</LI><LI> Results: </LI><UL><LI> PCR appears to work in all lanes (band size ~0.1 kb)</LI><LI> R0040 digest single band appears correct</LI></UL></UL><P> PCR cleanup of anti-tracrRNA PCR and pSB1C3-R0040</P><UL><LI> Concentrations: </LI><UL><LI> anti-tracrRNA: 83.3 ng/uL in 30 uL</LI><LI> pSB1C3-R0040: 25 ng/uL in 30 uL</LI></UL></UL></DIV></DIV><DIV class="day"><A id="jul15"><H2>July 15</H2></A><DIV class="obj">Objective: Create pTet-anti-tracrRNA</DIV><DIV class="lab"><P> Digest of anti-tracrRNA PCR</P><UL><LI> with XbaI, PstI-HF, and DpnI</LI><LI> 3 hrs at 37C</LI><LI> 60 uL DNA in 100 uL total reaction</LI></UL><P> PCR Cleanup of anti-tracrRNA PCR digest and pSB1C3-R0040 digest</P><UL><LI> Concentrations</LI><UL><LI> anti-tracrRNA: 100 ng/uL</LI><LI> pSB1C3-R0040: 35 ng/uL</LI></UL></UL><P> Ligation of anti-tracrRNA and pSB1C3-R0040</P><UL><LI> 100 ng = 3 uL pSB1C3-R0040 cut with SpeI/PstI</LI><LI> 15 ng = 0.15 uL anti-tracrRNA PCR cut with XbaI/PstI/DpnI</LI><LI> Experimental, Backbone only, and Insert only</LI><LI> Transformed via heat shock into DH5alphas and plated on LB+Cm</LI></UL></DIV><DIV class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</DIV><DIV class="lab"><P> Plate results:</P><UL><LI> Experimental: 8 colonies</LI><LI> BO Control: 4 colonies</LI><LI> IO Control: 7 colonies</LI></UL><P> Culture colonies</P><UL><LI> 4 copies of potential pSB1C3-dCas9-tracrRNA construct</LI></UL><P> Digest of dCas9-tracrRNA PCR and heat inactivation</P><UL><LI> Digest with DpnI</LI><LI> 22 uL DNA in 30 uL reaction</LI><LI> 1 hr at 37C, then 30 mins at 80C to heat-inactivate</LI><LI> assumed concentration 20 ng/uL</LI></UL><P> PCR Cleanup of pSB1C3 digest</P><UL><LI> 2 tubes of gel-extracted and cleaned pSB1C3 cut with XbaI and PstI</LI><LI> 11 ug original yields 309.6 ng/uL in 20 uL “superclean”</LI></UL><P> Ligation of dCas9-tracrRNA PCR and pSB1C3</P><UL><LI> 150 ng = 0.5 uL pSB1C3 cut with XbaI and PstI</LI><LI> 100 ng = 5 uL dCas9-tracrRNA PCR cut with Xbal/PstI/DpnI</LI><LI> Experimental, Backbone only, and Insert only</LI><LI> Transformed via heat shock into DH5alphas and plated on LB+Cm</LI></UL></DIV><DIV class="obj">Objective: Create pSB1C3-R0011-I13500 as destination for mCherry G-block</DIV><DIV class="lab"><P> Plate results:</P><UL><LI> Lawn of colonies on Experimental and BO control</LI><LI> No colonies on IO control</LI></UL><P> Digest of pSB1C3-R0011</P><UL><LI> With SpeI, PstI, and CIP</LI><LI> 50 uL DNA in 100 uL reaction</LI><LI> 2.5 hrs at 37C</LI></UL><P> PCR Cleanup of pSB1C3-R0011 digest and I13500 digest</P><UL><LI> pSB1C3-R0011 concentration 175 ng/uL</LI><LI> I13500 (from previous digestion and extraction)</LI><UL><LI> 2.3 ug original yields 124 ng/uL in 20 uL “superclean”</LI></UL></UL><P> Ligation of I13500 and pSB1C3-R0011</P><UL><LI> 100 ng = 0.6 uL pSB1C3-R0011 cut with SpeI/PstI/CIP</LI><LI> 150 ng = 1.2 uL I13500 cut with XbaI/PstI</LI><LI> Experimental, Backbone only, and Insert only</LI><LI> Transformed via heat shock into DH5alphas and plated on LB+Cm</LI></UL></DIV><DIV class="obj">Objective: Switch K608012 into pSB6A1</DIV><DIV class="lab"><P> Plate results:</P><UL><LI> 2 colonies on experimental plate</LI><LI> 1 colony on BO, no colonies on IO</LI></UL><P> Cultured colonies</P><UL><LI> 2 colonies of potential pSB6A1-K608012 construct</LI></UL><P> PCR Cleanup of pSB6A1 digest and K608012 digest</P><UL><LI> pSB6A1, 2 tubes gel extracted and cleaned</LI><UL><LI> 10 ug original yields 203 ng/uL in 30 uL</LI></UL><LI> K608012, 3 tubes gel extracted and cleaned</LI><UL><LI> 4 ug original yields 73.0 ng/uL in 30 uL</LI></UL></UL><P> Ligation of K608012 and pSB6A1</P><UL><LI> 100 ng = 0.5 uL pSB6A1 cut with EcoRI and SpeI</LI><LI> 150 ng = 2 uL K608012 cut with EcoRI and SpeI</LI><LI> Experimental, Backbone only, and Insert only</LI><LI> Transformed via heat shock into DH5alphas and plated on LB+Amp</LI></UL></DIV></DIV><DIV class="day"><A id="jul16"><H2>July 16</H2></A><DIV class="obj">Objective: Create pTet-anti-tracrRNA</DIV><DIV class="lab"><P> Plate results:</P><UL><LI> Experimental: over 1000 colonies</LI><LI> BO control had ~500 colonies, IO control had ~50 colonies</LI></UL><P> Cultured colonies</P><UL><LI> 4 copies of potential pSB1C3-R0040-anti-tracrRNA</LI></UL></DIV><DIV class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</DIV><DIV class="lab"><P> Plate results: </P><UL><LI> Experimental: 12-15 colonies</LI><LI> BO control had 3 colonies, IO control had ~30 colonies</LI></UL><P> Minipreps</P><UL><LI> 4 copies of potential pSB1C3-dCas9-tracrRNA construct</LI><LI> From 7/14 ligation</LI><LI> concentrations: 537.2, 45.4, 79.1, and 71.5 ng/uL</LI><UL><LI> Only one has normal concentration, others very low</LI></UL></UL><P> Analytical digest of pSB1C3-dCas9-tracrRNA constructs:</P><UL><LI> 4 copies with NheI/XbaI</LI></UL><P> Agarose gel of digests:</P><UL><LI> Lanes 1-2: pSB6A1-K608012 with MfeI/XbaI (expected 4.1, 0.6 kb bands)</LI><LI> Lanes 3-6: pSB1C3-dCas9-tracrRNA with NheI/XbaI (expected 5.0, 1.5 kb bands)</LI><LI> Results: gel was fuzzy and unclear</LI><UL><LI> lanes 2,3,5,6 clearly incorrect: 1 band ~2.0 kb</LI><LI> lane 1 had one fuzzy band around 4-5 kb (could be incomplete digest)</LI><LI> lane 4 had band around 5 kb with smear above (could be incomplete digest)</LI></UL></UL><P> New analytical digest of pSB1C3-dCas9-tracrRNA construct</P><UL><LI> Only copy #2 (showed potential in previous digest</LI><LI> With KpnI/SpeI for 2 hours at 37C</LI><LI> Also put previous digest back in bath for 2 more hours</LI></UL><P> Agarose gel of digests</P><UL><LI> DETAILS</LI></UL></DIV><DIV class="obj">Objective: Create pSB1C3-R0011-I13500 as destination for mCherry G-block</DIV><DIV class="lab"><P> Plate results:</P><UL><LI> ~500 colonies on experimental plate</LI><LI> BO control had over 1000 colonies</LI><LI> IO control had no colonies</LI></UL><P> Cultured colonies</P><UL><LI> 4 copies of potential pSB1C3-R0011-I13500 construct</LI></UL></DIV><DIV class="obj">Objective: Switch K608012 into pSB6A1</DIV><DIV class="lab"><P> Plate results:</P><UL><LI> 0 colonies on experimental plate</LI><LI> 4 colonies on BO control, 1 colony on IO control</LI></UL></DIV><DIV class="obj">Objective: Insert crRNAs into pdCas9</DIV><DIV class="lab"><P> Colony PCR of pdCas9-GFP3 plate</P><UL><LI> Plate from 7/7/14</LI><LI> 15 colonies plus pdCas9 miniprep as control</LI><LI> double dipped in 50 uL Taq poly mix and 50 uL SOC</LI><LI> Oligos pdCas9-up1 and pdCas9-dn1</LI><LI> TALCOLONYPCR protocol with 64C anneal</LI></UL><P> Agarose gel of colony PCR</P><UL><LI> Expected to see higher band matching control in successful colonies</LI><LI> lanes 1 and 17 are pdCas9 control</LI><LI> assay looks helpful, successes in samples 3,5,6,8,9,12,13,15</LI></UL><P> Cultured colonies</P><UL><LI> Copies 3,5,6, and 8 from colony PCR in LB+Cm</LI></UL></DIV></DIV><DIV class="day"><A id="jul17"><H2>July 17</H2></A><DIV class="obj">Objective: Create pTet-anti-tracrRNA and pSB1C3-R0011-I13500</DIV><DIV class="lab"><P>Minipreps</P><UL><LI>4 copies of pSB1C3-R0040-anti-tracrRNA from 7/15 ligation</LI></UL><UL><LI>4 copies of pSB1C3-R0011-I13500 from 7/15 ligation</LI></UL><P>Analytical digest of minipreps</P><UL><LI>pSB1C3-R0040-anti-tracrRNA with EcoRI/PstI</LI><LI>pSB1C3-R0011-I13500 with MfeI/NdeI/SacI</LI></UL><P>Agarose gel of digest results:</P><UL><LI>Lanes 1-4: pSB1C3-R0040-anti-tracrRNA with EcoRI/PstI (expected 2.0, 0.2 kb bands) (incomplete expected 2.0, &lt;0.1 kb bands)</LI><LI>Lanes 5-8: pSB1C3-R0011-I13500 with MfeI/NdeI/SacI (expected 1.3, 0.9, 0.33,0.29 kb bands)</LI><LI>Results: </LI></UL><UL><LI>R0040-antitracrRNA looks correct for all 4 copies--small band is hard to tell exactly</LI><LI>R0011-I13500 does not look correct (only one band at 2.0 kb)</LI></UL><DIV class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</DIV><P>Colony PCR on pSB1C3-dCas9-tracrRNA plate</P><UL><LI>Ligation plate from 7/15/14</LI><LI>15 colonies doubled into 50 uL Taq poly mix and 50 uL SOC</LI><LI>oligos SB1C3-up1 and SB1C3-dn1</LI><LI>annealed at 65C, extension 2 mins</LI></UL><P>Agarose gel of colony PCR</P><UL><LI>results: primer-bands only (no successes)</LI></UL><DIV class="obj">Objective: Switch K608012 into pSB6A1</DIV><P>Culture colonies</P><UL><LI>2 copies of pSB6A1-J04450 (from 6-9-14 plate)</LI><LI>2 copies of pSB1C3-K608012
<DIV class="day"><A id="jul18"><H2>July 18</H2></A><DIV class="lab"><P class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</P><P>PCR of dCas9-tracrRNA from pdCas9</P></DIV></DIV></LI><LI>8 tubes, each with 1 uL pdCas9 template at 1 ng/uL</LI><LI>oligos dCas9tracr-up and dCas9-tracr-dn</LI><LI>2 min extension, 64C anneal</LI></UL><P>Agarose gel of PCR product</P><UL><LI>All 8 tubes look successful (band ~4 kb)</LI></UL><P>PCR cleanup of dCas9-tracrRNA</P><UL><LI>Qiagen kit</LI><LI>Into 2 tubes, 30 uL each</LI></UL><P>Prep-scale digest of PCR product</P><UL><LI>with PstI/XbaI/DpnI</LI><LI>in 100 uL each for 3 hrs at 37C</LI></UL><P>Treat pSB1C3 digest with CIP</P><UL><LI>1 hr at 37C</LI></UL><P class="obj">Objective: Switch K608012 into pSB6A1</P><P>Miniprep pSB1C3-K608012 and pSB6A1-J04450</P><UL><LI>1 copy each (the second copy of each did not grow in culture)</LI><LI>concentrations:</LI></UL><UL><LI>pSB1C3-K608012: 344 ng/uL</LI><LI>pSB6A1-J04450: 421.5 ng/uL</LI></UL><P>Prep-scale digest of K608012 and pSB6A1</P><UL><LI>50 uL DNA in 100 uL reaction</LI><LI>Both with EcoRI and PstI</LI><LI>3 hrs at 37C</LI></UL><P>CIP treatment of pSB6A1 and heat-inactivation</P><UL><LI>2.5 uL CIP added to digest tube</LI><LI>Both digests returned to 37C for 1 hr</LI><LI>Then 30 mins at 80C to heat-inactivate</LI></UL><P>Gel extraction of pSB6A1 and K608012</P><UL><LI>Gel had no EtBr added, and was discarded (product was lost)</LI></UL></DIV></DIV><DIV class="day"><A id="jul20"><H2>July 20</H2></A><DIV class="lab"><P class="obj">Objective: culture colonies for various purposes</P><P>Cultured colonies</P><UL><LI>4 copies pSB6A1-J04450</LI></UL><UL><LI>2 from older plate, 2 from newer</LI></UL><UL><LI>4 copies pSB1C3-K608012</LI></UL><UL><LI>2 from older plate, 2 from newer</LI></UL><UL><LI>4 copies pSB1C3-B0030</LI></UL><UL><LI>To obtain pSB1C3 without gel extraction</LI></UL><UL><LI>4 copies pSB1C3-R0011</LI><LI>4 copies pSB1C3-I13500</LI></UL></DIV></DIV><DIV class="day"><A id="jul21"><H2>July 21</H2></A><DIV class="lab"><P class="obj">Objective: Sequence to confirm constructs</P><P>Sequencing reaction and Order #1634</P><UL><LI>1-4: pdCas9-GFP3 #3,5,6,8 with primer pdCas9-up1</LI><LI>5-8: pdCas9-GFP3 #3,5,6,8 with primer pdCas9-dn1</LI><LI>9-10: pSB1C3-R0040-anti-tracrRNA #1,2 with primer SB1C3-up1</LI><LI>11-12: pSB1C3-R0040-anti-tracrRNA #1,2 with primer SB1C3-dn1</LI><LI>Big Dye, buffer version 1.1</LI></UL><P>Order 1634 Results:</P><UL><LI>pdCas9-GFP3 copies 3, 6, and 8 are correct</LI></UL><UL><LI>copy 5 has a single mutation</LI></UL><UL><LI>anti-tracrRNA: both copies are correct</LI></UL><UL><LI>copy 1 was a cleaner, more reliable </LI></UL><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Minipreps</P><UL><LI>Qiagen kit</LI><LI>4 tubes of pSB1C3-R0011</LI></UL><UL><LI>concentrations 255.7, 169.2, 204.8, 151.2 ng/uL</LI></UL><UL><LI>4 tubes of pSB1C3-I13500</LI></UL><UL><LI>concentrations 148.9, 214.2, 260.7, 432.5</LI></UL><P>Prep scale digests of pSB1C3-R0011 and I13500</P><UL><LI>pSB1C3-R0011 with SpeI/PstI, samples 1,3</LI><LI>pSB1C3-I13500 with XbaI/PstI, samples 1,3</LI><LI>50 uL DNA in 100 uL each reaction, 2 tubes per construct</LI><LI>3 hrs at 37C</LI><LI>Added CIP (2.5 uL) to pSB1C3-R0011 reaction</LI><LI>1 more hr for both at 37C</LI><LI>Heat inactivation of both reactions: 30 mins at 80C</LI></UL><P>Gel extraction of I13500</P><UL><LI>extraction gel 1, lanes 1 and 2</LI><LI>Zymoclean prep kit</LI><LI>sample 1 (lane 1): 300 mg gel yields</LI><LI>sample 3 (lane 2): 550 mg gel yields</LI></UL><P>PCR cleanup of pSB1C3-R0011 </P><UL><LI>Qiagen kit</LI><LI>Two tubes of digest combined into one tube product</LI><LI>Concentration </LI></UL><P>Ligation of pSB1C3-R0011 and I13500</P><UL><LI>Old backbone (pSB1C3-R0011 digested 7/15): 100 ng = 0.6 uL</LI><LI>New backbone (pSB1C3-R0011 digested 7/21): 100 ng = 2.2 uL</LI><LI>Old insert (I13500 extracted 7/15): 300 ng = 2.4 uL</LI><LI>New insert (I13500 extracted 7/21): 300 ng = 3 uL</LI><LI>Ligations numbered 1-8:</LI></UL><UL><LI>1. OB+OI</LI><LI>2. OB+NI</LI><LI>3. NB+OI</LI><LI>4. NB+NI</LI><LI>5. OB control</LI><LI>6. OI control</LI><LI>7. NB control</LI><LI>8. NI control</LI></UL><UL><LI>1 hr at rt</LI><LI>Transformed via heat shock into DH5alphas, plated on LB+Cm</LI></UL><P class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</P><P>Minipreps</P><UL><LI>Qiagen kit</LI><LI>4 tubes of pSB1C3-B0030</LI></UL><UL><LI>to obtain pSB1C3 without gel extraction</LI><LI>concentrations 116.8, 99.9, 115.3, 145.1 ng/uL</LI></UL><P>PCR cleanup of dCas9-tracrRNA PCR</P><UL><LI>Product digested 7/18 with XbaI/PstI/DpnI</LI><LI>Concentration 80.5 ng/uL in 30 uL</LI><LI>Heat inactivation of cleaned product, 30 mins at 80C</LI></UL><P>Prep scale digest of pSB1C3-B0030</P><UL><LI>With XbaI/PstI, samples 1,3</LI><LI>50 uL DNA in 100 uL each reaction, 2 tubes </LI><LI>3 hrs at 37C</LI><LI>Added CIP (2.5 uL) </LI><LI>1 more hr for both at 37C</LI><LI>Heat inactivation: 30 mins at 80C</LI></UL><P>PCR cleanup of pSB1C3 </P><UL><LI>B0030 should not be retained</LI><LI>Qiagen kit</LI><LI>Two tubes of digest combined into one tube product</LI><LI>Concentration </LI></UL><P>Ligation of pSB1C3 and dCas9-tracrRNA</P><UL><LI>Backbone (pSB1C3 digested 7/21): 300 ng = 3.5 uL</LI><LI>Old insert (dCas9-tracrRNA digested 7/15): 100 ng = 3 uL</LI><LI>New insert (dCas9-tracrRNA digested 7/18): 100 ng = 1.2 uL</LI><LI>Ligations numbered 9-13:</LI></UL><UL><LI>9. B+OI</LI><LI>10. B+NI</LI><LI>11. B control</LI><LI>12. OI control</LI><LI>13. NI control</LI></UL><UL><LI>1 hr at rt</LI><LI>Transformed via heat shock into DH5alphas, plated on LB+Cm</LI></UL><P class="obj">Objective: Switch K608012 into pSB6A1 </P><P>Minipreps</P><UL><LI>Qiagen kit</LI><LI>4 tubes of pSB6A1-J04450</LI></UL><UL><LI>concentrations 189.3, 432.5, 688.7, 487.3 ng/uL</LI></UL><UL><LI>4 tubes of pSB1C3-K608012</LI></UL><UL><LI>concentrations 68.8, 91.6, 346.1, ng/uL</LI></UL><P>Prep scale digests of pSB6A1-J04450 and pSB1C3-K608012</P><UL><LI>pSB6A1-J04450 with XbaI/SpeI, samples 1,3</LI><LI>pSB1C3-K608012 with EcoRI/PstI, samples 1,3</LI><LI>50 uL DNA in 100 uL each reaction, 2 tubes per construct</LI><LI>3 hrs at 37C</LI><LI>Added CIP (2.5 uL) to pSB6A1-J04450 reaction</LI><LI>1 more hr for both at 37C</LI><LI>Heat inactivation of both reactions: 30 mins at 80C</LI></UL><P>Gel extraction of pSB6A1 and K608012</P><UL><LI>K608012: gel 1, lane 3 and gel 2, lane 1</LI><LI>pSB6A1: gel 2, lanes 2 and 3</LI><LI>Zymoclean prep kit</LI><LI>K608012 sample 1 (lane 1-3): 300 mg gel yields</LI><LI>K608012 sample 3 (lane 2-1): 550 mg gel yields</LI><LI>pSB6A1 sample 1 (lane 2-2):</LI><LI>pSB6A1 sample 3 (lane 2-3):</LI></UL><P>Ligation of pSB6A1 and K608012</P><UL><LI>Old backbone (pSB6A1 extracted 7/15): 100 ng = 0.5 uL</LI><LI>Old insert (K608012 extracted 7/15): 300 ng = 4 uL</LI><LI>Ligations numbered 14-16:</LI></UL><UL><LI>14. B+I</LI><LI>15. B control</LI><LI>16. I control</LI></UL><UL><LI>1 hr at rt</LI><LI>Transformed via heat shock into DH5alphas, plated on LB+Amp</LI></UL><P>Gibson of pSB6A1 and K608012</P><UL><LI>New backbone (pSB6A1 extracted 7/21): 100 ng = 0.3 uL</LI><LI>New insert (K608012 extracted 7/21): 300 ng = 1.7 uL</LI><LI>Numbered 17-19:</LI></UL><UL><LI>17. B+I</LI><LI>18. B control</LI><LI>19. I control</LI></UL><UL><LI>15 uL Mert’s DIY Gibson mix</LI><LI>1 hr at 60C</LI><LI>Transformed via heat shock into DH5alphas, plated on LB+Amp</LI></UL><P>SLIC of pSB6A1 and K608012</P><UL><LI>New backbone (pSB6A1 extracted 7/21): 100 ng = 0.3 uL</LI><LI>New insert (K608012 extracted 7/21): 300 ng = 1.7 uL</LI><LI>Numbered 20-22:</LI></UL><UL><LI>20. B+I</LI><LI>21. B control</LI><LI>22. I control</LI></UL><UL><LI>2.5 mins at rt, then 10 mins at 4C</LI><LI>Transformed via heat shock into DH5alphas, plated on LB+Amp</LI></UL></DIV></DIV><DIV class="day"><A id="jul22"><H2>July 22</H2></A><DIV class="lab"><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Plate results:</P><UL><LI>All four experimental plates had lots of colonies</LI><LI>both Backbone-only controls had similar colony numbers to experimentals</LI><LI>Insert-only controls had no colonies</LI></UL><P>Colony PCR</P><UL><LI>4 copies each of 4 experimental plates = 16 colonies</LI><LI>Oligos SB1C3-up and SB1C3-dn with Taq polymerase</LI><LI>66C anneal temp, 2 min extension time</LI></UL><P>Agarose gel of colony PCR results</P><UL><LI>1-4: OB+OI pSB1C3-R0011-I13500</LI><LI>5-8: OB+NI pSB1C3-R0011-I13500</LI><LI>9-12: NB+OI pSB1C3-R0011-I13500</LI><LI>13-16: NB+NI pSB1C3-R0011-I13500</LI><LI>17-20: B+OI pSB1C3-dCas9-tracrRNA</LI><LI>21-24: B+NI pSB1C3-dCas9-tracrRNA</LI></UL><UL><LI>Results: no successful bands on any colonies</LI></UL><UL><LI>Just primer-dimers or small amplified bands?</LI></UL><P class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</P><P>Plate results:</P><UL><LI>Both experimental plates had lots of colonies</LI><LI>Backbone-only controls had similar colony numbers to experimentals</LI><LI>Insert-only controls had no colonies</LI></UL><P>Colony PCR</P><UL><LI>4 copies each of 2 experimental plates = 8 colonies</LI><LI>Oligos SB1C3-up and SB1C3-dn with Taq polymerase</LI><LI>66C anneal temp, 2 min extension time</LI></UL><P>Agarose gel of colony PCR results</P><UL><LI>see above for gel order</LI></UL><UL><LI>Results: no successful bands on any colonies</LI></UL><UL><LI>Just primer-dimers or small amplified bands?</LI></UL><P class="obj">Objective: Switch K608012 into pSB6A1</P><P>Plate results:</P><UL><LI>No colonies on Gibson or SLIC plates</LI><LI>3 colonies on Ligation plate</LI><LI>A few colonies on controls, but not many</LI></UL><P>Cultured colonies</P><UL><LI>3 colonies of potential pSB6A1-K608012 from Ligation plate</LI><LI>in LB+Amp</LI></UL><P>Ligation of K608012 and pSB6A1</P><UL><LI>Same constructs as 7/21 ligation</LI><LI>This time let recover in SOC for 2 hrs instead of 1 hr</LI><LI>Plated on LB+Amp</LI></UL><P class="obj">Objective: Try putting K608012 into pSB1A2 instead</P><P>Transformation from kit plate</P><UL><LI>Plate 4-1N</LI></UL><UL><LI>pSB1A2-B0034</LI></UL><UL><LI>Plated on LB+Amp</LI></UL></DIV></DIV><DIV class="day"><A id="jul23"><H2>July 23</H2></A><DIV class="lab"><P class="obj">Objective: Switch K608012 into pSB6A1</P><P>Miniprep 3 copies of potential pSB6A1-K608012</P><UL><LI>Qiagen kit</LI><LI>Saved stocks in 15% glycerol</LI></UL><P>Analytical digest of minipreps</P><UL><LI>With NdeI in Cutsmart, 10 uL final volume</LI></UL><P>Agarose gel of analytical digest</P><UL><LI>Results:</LI></UL><UL><LI>Copies 1 and 2 are strange and incorrect</LI><LI>Copy 3 looks correct</LI><LI>Need to check with more digests</LI></UL><P>Analytical digest to check promising sample</P><UL><LI>Copy 3 in 3 different reactions</LI></UL><UL><LI>EcoRI/SpeI</LI><LI>EcoRI/MfeI</LI><LI>NcoI/PvuI</LI></UL><UL><LI>Control: pSB6A1-J04450 with EcoRI/SpeI/PvuI</LI><LI>Control: pSB1C3-K608012 with MfeI/NcoI</LI></UL><P>Agarose gel of analytical digest</P><UL><LI>1. Ligation 3 with EcoRI/SpeI</LI><LI>2. Ligation 3 with EcoRI/MfeI</LI><LI>3. Ligation 3 with NcoI/PvuI</LI><LI>4. pSB6A1-J04450 with EcoRI/SpeI/PvuI</LI><LI>5. pSB1C3-K608012 with MfeI/NcoI</LI></UL><UL><LI>Results:</LI></UL><UL><LI>Sample looks good</LI><LI>Incorrect samples 1 and 2 glycerol stocks thrown out</LI></UL><P class="obj">Objective: Switch R0040-anti-tracrRNA into pSB4K5</P><P>Treat pSB4K5 with CIP</P><UL><LI>1.25 uL CIP in 25 uL reaction</LI><LI>pSB4K5 was cut with EcoRI/SpeI and extracted on 7/9</LI></UL><P>Prep-scale digest to obtain R0040-anti-tracrRNA</P><UL><LI>pSB1C3-R0040-anti-tracrRNA </LI><LI>with EcoRI/SpeI</LI></UL><P>Gel extraction and cleanup of R0040-anti-tracrRNA</P><UL><LI>Extracted smaller band</LI><LI>Attempted to clean up using Qiagen prep kit</LI></UL><UL><LI>dissolved with Zymoclean ADB</LI><LI>Then followed Qiagen PCR cleanup protocol</LI><LI>510 mg gel yielded little or no DNA </LI></UL><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Colony PCR</P><UL><LI>8 colonies from 7/21 ligation plates</LI><LI>Control colonies from plates of pSB1C3-I13500 and pSB1C3-R0011</LI></UL><P>Agarose gel of colony PCR results</P><UL><LI>No successes</LI><LI>Controls did not work properly either</LI><LI>colony PCR does not seem to be effective--need to just pick colonies</LI></UL><P>Culture colonies</P><UL><LI>8 colonies of potential pSB1C3-R0011-I13500</LI><LI>From 7/21 ligation plates (colonies from today’s colony PCR)</LI></UL><P class="obj">Objective: Measurement interlab study</P><P>Transformations from kit plates</P><UL><LI>Plate 1-20K</LI></UL><UL><LI>pSB1C3-K823005</LI></UL><UL><LI>Plate 1-22I</LI></UL><UL><LI>pSB1C3-K823012</LI></UL><UL><LI>Plate 2-24B</LI></UL><UL><LI>pSB1C3-E0240</LI></UL></DIV></DIV><DIV class="day"><A id="jul24"><H2>July 24</H2></A><DIV class="lab"><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Miniprep 8 colonies of potential pSB1C3-R0011-I13500</P><UL><LI>Qiagen kit</LI></UL><P>Analytical digest of minipreps</P><UL><LI>8 copies with NcoI</LI><LI>8 copies with EcoRI/SpeI</LI></UL><P>Agarose gel of digest results</P><UL><LI>1-8: copies 1-8 with NcoI</LI><LI>9-16: copies 1-8 with EcoRI/SpeI</LI><LI>Results: no successes, insert appears to be missing</LI></UL><UL><LI>Unsure whether incorrect colonies are pSB1C3 or pSB1C3-R0011</LI></UL><P>Agarose gel of digest results</P><UL><LI>Thicker gel, longer run time, more DNA</LI><LI>from same digest tubes</LI><LI>Results: </LI></UL><UL><LI>no small band on EcoRI/SpeI cut and no difference in larger band for the two cuts, indicating that R0011 is not present</LI></UL><P>Ligation of pSB1C3-R0011 and I13500</P><UL><LI>Used lab stock of T4 Ligase and buffer rather than iGEM stock</LI><LI>100 ng pSB1C3-R0011 = 2.5 uL</LI><LI>300 ng I13500 = 3 uL</LI><LI>4 tubes of ligation, run at rt with Ligase for 15, 30, 45, and 60 mins</LI><LI>BO and IO controls for each time</LI><LI>Plated on LB+Cm, two tubes per plate (used wooden streaker rods to spread on half)</LI></UL><P class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</P><P>Ligation of pSB1C3 and dCas9-tracrRNA </P><UL><LI>Used lab stock of T4 Ligase and buffer rather than iGEM stock</LI><LI>300 ng pSB1C3 = 3 uL</LI><LI>100 ng dCas9-tracrRNA PCR = 1 uL</LI><LI>4 tubes of ligation, run at rt with Ligase for 15, 30, 45, and 60 mins</LI><LI>BO and IO controls for each time</LI><LI>Plated on LB+Cm, two tubes per plate (used wooden streaker rods to spread on half)</LI></UL><P class="obj">Objective: Test GFP repression by crRNAs </P><P>Transform pSB6A1-K608012 into DH5alpha-ZI</P><UL><LI>Plated on LB+Amp</LI></UL><P class="obj">Objective: Obtain constructs for various uses</P><P>Streaked plate from frozen stock</P><UL><LI>One LB+Cm plate in thirds:</LI></UL><UL><LI>pSB1C3-R0040-antitracrRNA</LI><LI>pSB1C3-Repeat-seq scaffold</LI><LI>pSB1C3-Csy4-gRNA scaffold</LI></UL><P class="obj">Objective: Measurement interlab study</P><P>Cultured colonies (2 each)</P><UL><LI>pSB1C3-E0240</LI><LI>pSB1C3-K823005</LI><LI>pSB1C3-K823012</LI><LI>pSB3K3-I20260</LI></UL></DIV></DIV><DIV class="day"><A id="jul25"><H2>July 25</H2></A><DIV class="lab"><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Plate results:</P><UL><LI>no apparent difference between 15, 30, 45, or 60 minute ligations</LI><LI>many colonies on all Experimental and BO control plates</LI><LI>no colonies on IO control</LI></UL><P>Colony PCR:</P><UL><LI>16 colonies from pSB1C3-R0011-I13500 plates</LI></UL><UL><LI>4 from each ligation time</LI></UL><UL><LI>Taq polymerase with oligos SB1C3-up and SB1C3-dn</LI><LI>66C anneal temp, 45 sec extension</LI></UL><P>Agarose gel of colony PCR</P><UL><LI>No successes</LI><LI>It seems as though colony PCRs are not working--try control with a gradient</LI></UL><P>Gradient Colony PCR</P><UL><LI>to optimize with SB1C3-up and SB1C3-dn primers</LI><LI>template colonies of pSB1C3-K608011 from plate</LI><LI>anneal temps 62-72 C, extension time 45 sec</LI></UL><P>Agarose gel of gradient colony PCR</P><UL><LI>left to right: 62C to 72C anneal temps</LI><LI>Results: 62C was the best anneal temp--worked well</LI></UL><UL><LI>gradual decrease as temp increased, no amplification above 65C</LI></UL><P class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</P><P>Plate results:</P><UL><LI>no apparent difference between 15, 30, 45, or 60 minute ligations</LI><LI>many colonies on all Experimental and BO control plates</LI><LI>no colonies on IO control</LI></UL><P>Colony PCR:</P><UL><LI>16 colonies from pSB1C3-dCas9-tracrRNA plates</LI></UL><UL><LI>4 from each ligation time</LI></UL><UL><LI>Taq polymerase with oligos SB1C3-up and SB1C3-dn</LI><LI>66C anneal temp, 45 sec extension</LI></UL><P>Agarose gel of colony PCR</P><UL><LI>No successes</LI><LI>It seems as though colony PCRs are not working--try control with a gradient</LI></UL><UL><LI>see gradient results above</LI></UL><P class="obj">Objective: Various ligations</P><P>Prep-scale digests</P><UL><LI>pSB1C3-Csy4-scaffold (2 copies) with XbaI/PstI</LI></UL><UL><LI>to ligate into R0040</LI></UL><UL><LI>pSB1C3-Repeat-seq scaffold (2 copies) with XbaI/PstI</LI></UL><UL><LI>to ligate into R0040</LI></UL><UL><LI>pSB1C3-R0040-anti-tracrRNA (2 copies) with XbaI/PstI</LI><LI>100 uL reactions each tube</LI></UL><P>PCR cleanups on all digests</P><UL><LI>Qiagen kit</LI><LI>Concentrations (in 50 uL each):</LI></UL><UL><LI>pSB1C3-Csy4-scaffold: 55.9, 105.5 ng/uL</LI><LI>pSB1C3-Repeat-seq scaffold: 15.2, 55.2 ng/uL</LI><LI>pSB1C3-R0040-anti-tracrRNA: 90.6 ng/uL</LI></UL><UL><LI>Need to extract from gels before using, only cleaned up for safer storage </LI></UL><P class="obj">Objective: Measurement interlab study</P><P>Minipreps</P><UL><LI>Qiagen kit</LI></UL><UL><LI>pSB1C3-K823005 (2 copies) </LI></UL><UL><LI>concentrations 256.3, 314.8 ng/uL</LI></UL><UL><LI>pSB1C3-K823012 (2 copies)</LI></UL><UL><LI>concentrations 204.4, 192.5 ng/uL</LI></UL><UL><LI>pSB1C3-E0240 (2 copies)</LI></UL><UL><LI>concentrations 266.3, 241.5 ng/uL</LI></UL><P>Prep-scale digests</P><UL><LI>pSB1C3-K823005 (2 copies) with SpeI/PstI</LI><LI>pSB1C3-K823012 (2 copies) with SpeI/PstI</LI><LI>pSB1C3-E0240 (2 copies) with XbaI/PstI</LI><LI>100 uL each tube</LI></UL><P>PCR cleanups on all digests</P><UL><LI>Qiagen kit</LI><LI>Concentrations (in 50 uL each):</LI></UL><UL><LI>pSB1C3-K823005: 156.3, 181.7 ng/uL</LI><LI>pSB1C3-K823012: 110.0, 98.0 ng/uL</LI><LI>pSB1C3-E0240: 110.0, 114.6 ng/uL</LI></UL><P class="obj">Objective: Test crRNA repression of GFP reporter</P><P>Cultured colony</P><UL><LI>To make CCEC of reporter strain</LI><LI>One colony of pSB6A1-K608012 in DH5alpha-ZI</LI><LI>Grew for 8 hrs during the day in 2 mL LB+Amp at 37C</LI><LI>Inoculated into 100 mL flask of LB+Amp for overnight shaker at rt</LI></UL><P>Flow cytometry of reporter strain</P><UL><LI>quick check to make sure reporter still works with new backbone</LI><LI>Used parameters from previous GFP flow data</LI><LI>Levels slightly under what was seen before, but still strong</LI></UL></DIV></DIV><DIV class="day"><A id="jul26"><H2>July 26</H2></A><DIV class="lab"><P class="obj">Objective: Test crRNA repression of GFP reporter</P><P>Prepared Chemically competent E. coli </P><UL><LI>Using DH5alpha-ZI + pSB6A1-K608012</LI><LI>iGEM CCEC protocol with smaller volumes (made ~20 1-mL aliquots)</LI></UL><P>Double-Transformation of pdCas9 variants into reporter cells</P><P class="”ppl”">Charlie Cooper</P><UL><LI>pdCas9 (with BsaI insert as crRNA sequence)</LI><LI>pdCas9-crRNA_GFP1</LI><LI>pdCas9-crRNA_GFP2</LI><LI>pdCas9-crRNA_GFP3</LI></UL><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Colony PCRs</P><UL><LI>16 colonies from 7/24 ligation plates</LI><LI>oligos SB1C3-up and SB1C3-dn</LI><LI>62C anneal (as determined by gradient), 45 sec extension</LI></UL><P>Agarose gel of Colony PCRs</P><UL><LI>Results: colonies 5 and 14 amplified at the correct size</LI></UL><P>Culture colonies</P><UL><LI>Colonies 5 and 14 from colony PCR of potential pSB1C3-R0011-I13500</LI></UL><P class="obj">Objective: Create pSB1C3-dCas9-tracrRNA</P><P>Colony PCRs</P><UL><LI>16 colonies from 7/24 ligation plates</LI><LI>oligos SB1C3-up and SB1C3-dn</LI><LI>62C anneal (as determined by gradient), 45 sec extension</LI><LI>PCRs left in machine without starting reaction for ~1 hr, could have kill reaction</LI></UL><P>Agarose gel of Colony PCRs</P><UL><LI>No successes</LI></UL></DIV></DIV><DIV class="day"><A id="jul27"><H2>July 27</H2></A><DIV class="lab"><P class="obj">Objective: Test crRNA repression of GFP reporter</P><P>Cultured colonies for flow</P><UL><LI>3 copies of each</LI><LI>DH5alpha-Z1 (background)</LI><LI>DH5-alpha-Z1 + pSB6A1-K608012 (reporter positive control)</LI><LI>DH5-alpha-Z1 + pSB6A1-K608012 + pdCas9 (non-target control)</LI><LI>DH5-alpha-Z1 + pSB6A1-K608012 + pdCas9-crRNA_GFP1</LI><LI>DH5-alpha-Z1 + pSB6A1-K608012 + pdCas9-crRNA_GFP2</LI><LI>DH5-alpha-Z1 + pSB6A1-K608012 + pdCas9-crRNA_GFP3</LI></UL><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Minipreps</P><UL><LI>Colonies 5 and 14 from 7/26 colony PCRs</LI><LI>Qiagen kit</LI></UL></DIV></DIV><DIV class="day"><A id="jul28"><H2>July 28</H2></A><DIV class="lab"><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Analytical digest of minipreps</P><UL><LI>Colonies 5 and 14 of potential pSB1C3-R0011-I13500</LI><LI>with MfeI</LI><LI>with NcoI</LI><LI>with EcoRI/PstI</LI></UL><P>Agarose gel of analytical digest results</P><UL><LI>Gel Order:</LI></UL><UL><LI>#14 w/ MfeI</LI><LI>#14 w/ EcoRI/PstI</LI><LI>#14 w/ NcoI</LI><LI>#5 w/ MfeI</LI><LI>#5 w/ EcoRI/PstI</LI><LI>#5 w/ NcoI</LI></UL><UL><LI>Results: All looks correct, except that MfeI cut once instead of twice</LI></UL><UL><LI>could be pSB1C3-I13500 without R0011</LI></UL><P class="obj">Objective: Insert mCherry G-block into pSB1C3-R0011-I13500</P><P>PCR of mCherry G-block</P><UL><LI>Q5 polymerase with Oligos SB1C3-dn and pDGC3-up</LI><LI>4 tubes with 0, 0.4, 0.7, 1.0 uL template at 1 ng/uL</LI><LI>45 sec extension, 66C anneal temp</LI></UL><P>Agarose gel of PCR</P><UL><LI>3 tubes look successful</LI></UL><P>PCR cleanup of mCherry G-block PCR</P><UL><LI>Qiagen kit</LI><LI>40 uL at 27.6 ng/uL</LI></UL><P class="obj">Objective: Various ligations</P><P>Gel extraction and cleanup </P><UL><LI>Each extraction included 2 cleaned digests combined into one lane (~100 uL)</LI><LI>Cut and cleaned with Zymoclean prep kit</LI><LI>pSB1C3-Csy4-scaffold cut with XbaI/PstI on 7/25</LI></UL><UL><LI>pSB1C3 backbone 47.5 ng/uL in 30 uL</LI><LI>Csy4-scaffold insert 15.4 ng/uL in 30 uL</LI></UL><UL><LI>pSB1C3-Repeat-seq scaffold cut with XbaI/PstI on 7/25</LI></UL><UL><LI>pSB1C3 backbone 74.8 ng/uL in 30 uL</LI><LI>Repeat-seq scaffold insert 13.1 ng/uL in 30 uL</LI></UL><UL><LI>pSB1C3-R0040-anti-tracrRNA cut with XbaI/PstI on 7/25</LI></UL><UL><LI>pSB1C3 backbone 164.0 ng/uL in 30 uL</LI><LI>R0040-anti-tracrRNA 176.0 ng/uL in 30 uL</LI></UL><P>PCR Cleanup on gel extraction cleanups</P><UL><LI>to “superclean”</LI><LI>pSB1C3 4 tubes combined into 1: 469.6 ng/uL in 20 uL</LI><LI>Repeat-seq scaffold: 23.9 ng/uL</LI></UL><UL><LI>rough graph, probably not useful</LI></UL><UL><LI>Csy4 scaffold: 16 ng/uL</LI></UL><UL><LI>rough graph, probably not useful</LI></UL><UL><LI>R0040-anti-tracrRNA: 7.4 ng/uL</LI></UL><UL><LI>rough graph, probably not useful</LI></UL><P class="obj">Objective: Measurement interlab study</P><P>CIP treatment of backbones</P><UL><LI>Using Charlie’s CIP stock</LI><LI>pSB1C3-K823005 cut with SpeI/PstI on 7/25 (2 tubes)</LI><LI>pSB1C3-K823012 cut with SpeI/PstI on 7/25 (2 tubes)</LI></UL><P>Gel extraction and cleanup</P><UL><LI>2 cleaned digests combined into one lane (~100 uL)</LI><LI>Cut and cleaned with Zymoclean prep kit</LI><LI>pSB1C3-E0240 cut with XbaI/PstI on 7/25</LI></UL><UL><LI>pSB1C3 backbone 116.4 ng/uL in 30 uL</LI><LI>E0240 86.5 ng/uL in 30 uL</LI></UL><P>PCR Cleanup of CIP treatment and gel extraction cleanup</P><UL><LI>Qiagen kit</LI><LI>pSB1C3-K823005 SpeI/PstI CIP treated: 119.8, 119.7 ng/uL</LI><LI>pSB1C3-K823012 SpeI/PstI CIP treated: 60.9, 62.3 ng/uL</LI><LI>E0240 XbaI/PstI extracted: 56.5 ng/uL in 20 uL (“superclean”)</LI></UL><P>Ligations</P><UL><LI>E0240 into two backbones: pSB1C3-K823005 and pSB1C3-K823012</LI><LI>pSB1C3-K823005 SpeI/PstI/CIP: 100 ng = 0.87 uL</LI><LI>pSB1C3-K823012 SpeI/PstI/CIP: 100 ng = 1.67 uL</LI><LI>E0240 XbaI/PstI: 300 ng = 6 uL</LI><LI>2 BO controls and 1 IO control</LI><LI>20 mins at rt</LI><LI>Heat shock transformation and plated on LB+Cm</LI></UL><P class="obj">Objective: Test crRNA repression of GFP reporter</P><P>Dilute cultures to grow in log phase for flow</P><UL><LI>1/1000 dilution into 50 mL</LI><LI>18 samples: 3 copies each of 6 strains (labelled 2-19)</LI><LI>Samples 2-4: DH5alpha-Z1 (background) </LI></UL><UL><LI>grown in LB+Spec</LI></UL><UL><LI>Samples 5-7: DH5alpha-Z1 + pSB6A1-K608012 (reporter positive control)</LI></UL><UL><LI>grown in LB+Amp</LI></UL><UL><LI>Samples 8-10: DH5alpha-Z1 + pSB6A1-K608012 +pdCas9 (non-target control)</LI></UL><UL><LI>grown in LB+Amp+Cm</LI></UL><UL><LI>Samples 11-13: DH5alpha-Z1 + pSB6A1-K608012 +pdCas9_GFP1</LI></UL><UL><LI>grown in LB+Amp+Cm</LI></UL><UL><LI>Samples 14-16: DH5alpha-Z1 + pSB6A1-K608012 +pdCas9_GFP2</LI></UL><UL><LI>grown in LB+Amp+Cm</LI></UL><UL><LI>Samples 17-19: DH5alpha-Z1 + pSB6A1-K608012 +pdCas9_GFP3</LI></UL><UL><LI>grown in LB+Amp+Cm</LI></UL><P>Flow cytometry </P><UL><LI>Samples grown for 4.5 hrs after initial dilution</LI><LI>1/50 final dilution of sample into PBS, placed on ice until run</LI><LI>parameters (same as previous GFP runs):</LI></UL><UL><LI>FSC 350V log3</LI><LI>SSC 350V log3</LI><LI>B1 350V log5</LI><LI>Trigger 10</LI><LI>10,000 event limit (100,000 events used in the past)</LI></UL><UL><LI>Results</LI></UL><UL><LI>~3-fold repression of reporter with crRNA_GFP1 or crRNA_GFP3</LI><LI>crRNA_GFP2 did not repress</LI><LI>non-target control repressed little or not at all</LI><LI>See document “<A href="https://docs.google.com/file/d/0B8nzM6b_weNTaDZwLWtqc0FPcG8/edit">Flow stats 7-28-14</A>” for data details</LI></UL></DIV></DIV><DIV class="day"><A id="jul29"><H2>July 29</H2></A><DIV class="lab"><P class="obj">Objective: Switch R0040-anti-tracrRNA into pSB4K5</P><P>Minipreps</P><UL><LI>pSB4K5-J04450 (4 copies)</LI><LI>pSB1C3-R0040-anti-tracrRNA (4 copies)</LI></UL><P>Prep-scale digest</P><UL><LI>pSB4K5-J04450 (4 copies) with EcoRI/SpeI</LI><LI>pSB1C3-R0040-anti-tracrRNA (4 copies) with EcoRI/SpeI</LI></UL><P>Gel extraction and cleanup of digests</P><UL><LI>Cut pSB4K5 backbone from 4 copies of pSB4K5-J04450</LI></UL><UL><LI>2 digests per gel well</LI><LI>2.04 g total gel eluted into two tubes:</LI></UL><UL><LI>51.4 ng/uL and 94.5 ng/uL, 30 uL each</LI></UL><UL><LI>Cut R0040-anti-tracrRNA insert from pSB1C3-R0040-anti-tracrRNA</LI></UL><UL><LI>2 digests per gel well</LI><LI>1.61 g total gel eluted into two tubes:</LI></UL><UL><LI>99.4 ng/uL and 82.3 ng/uL, 30 uL each</LI></UL><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Colony PCR</P><UL><LI>24 colonies of potential pSB1C3-R0011-I13500</LI></UL><UL><LI>8 colonies from 7/15 ligation</LI><LI>4x4=16 colonies from 4 different 7/21 ligation plates</LI></UL><UL><LI>Taq polymerase with oligos SB1C3-up and SB1C3-dn</LI><LI>extension time 45 sec, anneal temp 62C</LI></UL><P>Agarose gel of colony PCR results</P><UL><LI>colony #19 is only promising result</LI></UL><P>Analytical digest</P><UL><LI>pSB1C3-R0011 copies 2, 4</LI><LI>pSB1C3-I13500 copies 2, 4</LI><LI>with MfeI/NcoI</LI><LI>To test whether enzymes or original parts are the source of ligation difficulties</LI></UL><P>Agarose gel of digest results</P><UL><LI>Order:</LI></UL><UL><LI>1. pSB1C3-R0011 #2 cut with MfeI/NcoI (expected 2 bands)</LI><LI>2. pSB1C3-R0011 #4 cut with MfeI/NcoI (expected 2 bands)</LI><LI>3. pSB1C3-I13500 #2 cut with MfeI/NcoI (expected 3 bands)</LI><LI>4. pSB1C3-I13500 #4 cut with MfeI/NcoI (expected 3 bands)</LI></UL><UL><LI>Results:</LI></UL><UL><LI>R0011 both copies only show one band</LI><LI>I13500 both copies appear correct</LI></UL><P>Cultured colonies</P><UL><LI>Potential pSB1C3-R0011-I13500 #19 from colony PCR</LI><LI>pSB1C3-R0011 (3 copies) from original transformation </LI></UL><UL><LI>suspicion that our R0011 is incorrect--could be just bad copies</LI></UL><P class="obj">Objective: Put R0040 promoter in front of crRNA and gRNA scaffolds</P><P>Culture colonies</P><UL><LI>pSB1C3-R0040 (4 copies)</LI></UL><P class="obj">Objective: Measurement interlab study</P><P>Plate results</P><UL><LI>both experimental and both backbone-only plates all had hundreds of colonies</LI><LI>insert-only controls had no colonies</LI></UL><P>Colony PCR</P><UL><LI>8 colonies of potential pSB1C3-K823005-E0240</LI><LI>8 colonies of potential pSB1C3-K823012-E0240</LI><LI>Taq polymerase with oligos SB1C3-up and SB1C3-dn</LI><LI>extension time 45 sec, anneal temp 62C</LI></UL><P>Agarose gel of colony PCR results</P><UL><LI>1-8: pSB1C3-K823005-E0240</LI><LI>9-16: pSB1C3-K823012-E0240</LI><LI>Results: no successes</LI></UL></DIV></DIV><DIV class="day"><A id="jul30"><H2>July 30</H2></A><DIV class="lab"><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Minipreps</P><UL><LI>pSB1C3-R0011-I13500 #19</LI></UL><UL><LI>concentration 246.4 ng/uL</LI></UL><UL><LI>pSB1C3-R0011 (3 copies)</LI></UL><UL><LI>concentrations 187.6, 178.8, 212.8 ng/uL</LI></UL><P>Analytical digest</P><UL><LI>pSB1C3-R0011-I13500 #19</LI></UL><UL><LI>three digests in cutsmart, 10 uL final volume each</LI></UL><UL><LI>NcoI</LI><LI>MfeI</LI><LI>XbaI/PstI</LI></UL><UL><LI>pSB1C3-R0011 (3 copies)</LI></UL><UL><LI>with NcoI/MfeI in cutsmart, 10 uL final volume</LI></UL><P>Agarose gel of digest results</P><UL><LI>1. pSB1C3-R0011-I13500 #19 with NcoI</LI><LI>2. pSB1C3-R0011-I13500 #19 with MfeI</LI><LI>3. pSB1C3-R0011-I13500 #19 with EcoRI/PstI</LI><LI>4. pSB1C3-R0011 #1 with NcoI/MfeI</LI><LI>5. pSB1C3-R0011 #2 with NcoI/MfeI</LI><LI>6. pSB1C3-R0011 #3 with NcoI/MfeI</LI><LI>Results: hard to explain</LI></UL><UL><LI>R0011-I13500 not correct</LI><LI>R0011 all three copies only cut once </LI></UL><P class="obj">Objective: Switch R0040-anti-tracrRNA into pSB4K5 </P><P>Treat pSB4K5 with Antarctic Phosphatase</P><UL><LI>2 copies digested on 7/29, combined into one tube with 100 uL final volume</LI><LI>2 hrs at 37C</LI></UL><P>PCR cleanup of AP-treated pSB4K5</P><UL><LI>Qiagen kit</LI><LI>concentration 45.2 ng/uL in 50 uL</LI></UL><P>Ligation of pSB4K5 and R0040-anti-tracrRNA</P><UL><LI>2 experimental plates with 3x and 6x molar insert</LI><LI>pSB4K5 backbone (EcoRI/SpeI/AP)100 ng = 2 uL</LI><LI>R0040-anti-tracrRNA insert (EcoRI/SpeI) 30 ng = 0.3 uL, or 60 ng = 0.6 uL</LI><LI>Backbone-only and insert-only controls</LI><LI>Transformed via heat shock and plated on LB+G418</LI></UL><P class="obj">Objective: Put R0040 promoter in front of crRNA and gRNA scaffolds</P><P>Minipreps</P><UL><LI>pSB1C3-R0040 (4 copies)</LI></UL><UL><LI>concentrations 47.4, 80.7, 109.7, 193.5</LI></UL><P class="obj">Objective: Switch dCas9-tracrRNA into pSB1C3</P><P>Treat pSB1C3 with Antarctic Phosphatase</P><UL><LI>cleaned copy from 7/28 with 100 uL final volume</LI><LI>2 hrs at 37C</LI></UL><P>PCR cleanup of AP-treated pSB1C3</P><UL><LI>Qiagen kit</LI><LI>concentration 104.5 ng/uL in 50 uL</LI></UL><P>Ligation of pSB1C3 and dCas9-tracrRNA</P><UL><LI>2 experimental plates</LI></UL><UL><LI>3x molar dCas9-tracrRNA “insert”</LI></UL><UL><LI>pSB1C3 100 ng = 1 uL</LI><LI>dCas9-tracrRNA 600 ng = 7.5 uL</LI></UL><UL><LI>3x molar pSB1C3 “backbone”</LI></UL><UL><LI>pSB1C3 150 ng = 1.5 uL</LI><LI>dCas9-tracrRNA 100 ng = 1.2 uL</LI></UL><UL><LI>Backbone-only and insert-only controls</LI><LI>Transformed via heat shock and plated on LB+Cm</LI></UL></DIV></DIV><DIV class="day"><A id="jul31"><H2>July 31 </H2></A><DIV class="lab"><P class="obj">Objective: Switch R0040-anti-tracrRNA into pSB4K5</P><P>Plate results:</P><UL><LI>Hundreds of colonies on both experimental plates</LI><LI>20-30 colonies on backbone-only control</LI><LI>0 colonies on insert-only control</LI></UL><P> Cultured colonies</P><UL><LI>4 copies of pSB4K5-R0040-anti-tracrRNA</LI></UL><P class="”obj”">Objective: Switch dCas9-tracrRNA and dCas9-tracrRNA-crRNA into pSB1C3</P><P>Plate results:</P><UL><LI>Experimental plate 2 (higher pSB1C3 concentration) had 2 colonies</LI><LI>Experimental plate 1 had no colonies</LI><LI>Backbone-only control had 1 colony, insert-only control had 0</LI></UL><P>Cultured colonies</P><UL><LI>2 copies of potential pSB1C3-dCas9-tracrRNA</LI></UL><P>PCRs of tracrRNA-dCas9 and tracrRNA-dCas9-crRNA</P><UL><LI>8 tubes of each, with 1 uL 1 ng/uL pdCas9 template per tube</LI><LI>tracrRNA-dCas9 oligos: dCas9tracr-up and dCas9tracr-dn</LI><LI>tracrRNA-dCas9-crRNA oligos: dCas9tracr-up and pdCas9-dn</LI><LI>Q5 polymerase</LI><LI>64C anneal temp, 2:30 extension time</LI></UL><P>Agarose gel of PCRs</P><UL><LI>1-8: tracrRNA-dCas9-crRNA</LI><LI>9-16: tracrRNA-dCas9</LI><LI>Results: all 16 lanes look good</LI></UL><P>PCR cleanup of PCRs</P><UL><LI>8 tubes into 50 uL for each of two PCRs</LI><LI>Concentrations</LI></UL><UL><LI>tracrRNA-dCas9-crRNA: 350 ng/uL</LI><LI>tracrRNA-dCas9: 335 ng/uL</LI></UL><P>Prep-scale digest of 2 PCRs</P><UL><LI>tracrRNA-dCas9-crRNA and tracrRNA-dCas9</LI><LI>Each in 100 uL final volume with 2 uL each of XbaI/PstI/DpnI</LI></UL><P>PCR cleanup of digests</P><UL><LI>Into 50 uL final volume </LI></UL><P class="obj">Objective: Put R0040 in front of Csy4 scaffold and Repeat-seq scaffold</P><P>Prep-scale digest</P><UL><LI>pSB1C3-R0040 (2 copies) with 2.5 uL each of SpeI/PstI</LI><LI>100 uL final volume each copy</LI></UL><P>PCR cleanup of digests</P><UL><LI>Into 50 uL final volume</LI></UL><P class="obj">Objective: Triple-transformation of R0040-anti-tracrRNA into Z1-reporter-crRNA_GFP cells</P><P>Cultured cells in morning</P><UL><LI>straight from frozen stocks into 2 mL LB+Cm+Amp</LI><LI>DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP1</LI><LI>DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP3</LI></UL><P>Cultured cells in evening</P><UL><LI>1 mL from morning culture into 50 mL flask for overnight growth</LI><LI>To make chemically competent cells</LI></UL><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Cultured colonies</P><UL><LI>2 copies of pSB1C3-R0011 from streak plate of original frozen stock</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Duke/Notebook/July">http://2014.igem.org/Team:Duke/Notebook/July</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Duke/Notebook/July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Duke/Notebook/July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Duke/Notebook/July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Duke/Notebook/July&amp;oldid=276501" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>