"
Page
Discussion
View source
History
teams
Log in
 
Team:Duke/Notebook/Overview
From 2014.igem.org
Home
Team
About Us
Official Team Profile
Photo Gallery
Attributions
Project
Background
Summary
Modeling
Results
Future Directions
Parts
Notebook
Overview
Notebook by Month
Protocols
Policy and Practice
3D Printing
House Course
NCSSM Team
Safety
April Overview
Click here for the April Lab Notebook
The objective in April, while classes were still in session, was to make something called the Z4EV-dCas9-MXI1 construct. The model organism used was yeast, and this was a continuation of last year’s project.
The general idea was to fuse a repressor domain onto the dCas9 gene, to make an even stronger repressor. MXI1 is the repressor domain, and Z4EV is the promoter that was used. It’s originally a zinc-finger induced promoter for yeast.
This approach was abandoned because the general consensus was that repression is harder to achieve in yeast. Instead of trying to continue last year’s project, we decided to start afresh, with a new model organism, Escherichia coli.
May, June Overview
Click here for the May and June Lab Notebook
Once we revamped our project and decided to use E. coli as our model organism, we had to make some preparations. We had to prepare chemically competent cell stocks for the numerous transformations in our future. The E. coli strains we used were DH5alpha and DH5-alpha-Z1. The primary difference between the two is that the Z1 strain has necessary Lac and Tet repressors in the genome, but the previous strain is more competent to transform. As a result, both strains have their pros and cons, and we primarily used DH5alpha this month.
We also had obtained the natural parts we needed in plasmid form: pdCas9 and pCsy4, as well as BioBricks we needed from the iGEM kit. We also ordered oligonucleotides, or oligos. These short pieces of single-stranded DNA were designed and ordered so that we could use them as primers to amplify inserts and turn them into standard BioBricks for submission to the registry.
Next, we had to create scaffolds: a BioBrick that we could use to facilitate easy cloning of multiple crRNAs and gRNAs. We needed two types of scaffolds for our two approaches:
Repeat-sequence-repeat scaffold
This scaffold was successfully created for our CRISPR/Cas9 system approach. It consists of several repeat sequences between segments of spacer DNA. Later on, we will replace the spacer DNA with the correct crRNA-coding sequences so that we can put several crRNA templates in a row and generate crRNAs for repression.
Csy4 scaffold
This was for our Csy4 approach, in order to put multiple gRNAs in a row. We decided to order it as a G-block, or a double-stranded piece of DNA synthesized by Integrated DNA Technologies™, because we knew that it would be really hard to make ourselves.
We also collected preliminary flow cytometry data to optimize viewing of our fluorescent proteins in E. coli.
July Overview
Click here for the July Notebook
We now have the correct scaffolds for both the crRNA and gRNA approaches, so now we have to insert the crRNA templates into the scaffolds to complete them. We also tried to insert the crRNA templates into pdCas9, and this was successful.
We ordered a G-block for the two-reporter construct and eventual Golden-Gate assembly because the large construct involved a difficult, multi-step cloning process.
During this time, we also began considering molecular titration as a new possible approach for ultrasensitivity, so we designed and built an anti-tracrRNA construct and are currently working on transferring the construct to the appropriate plasmid. Flow cytometry showed that the dCas9 system successfully repressed a GFP reporter. Future steps are testing the anti-tracrRNA production and tracrRNA sequestration in this system.
August, September Overview
Objectives
Click here for the August Lab Notebook
Click here for the September Lab Notebook
Our first tests at molecular titration with anti-tracrRNA showed no derepression, so we swapped plasmid backbones around to increase the copy number of anti-tracr relative to tracr. We tested our new anti-tracrRNA constructs with flow cytometry at single time points and with a time course, seeing negative results.   We also began construction of parts for a new molecular titration approach, which uses decoy binding sites to titrate dCas9 away from its site of repression. We successfully built arrays with 1, 6, and 12 tandem binding sites.
October Overview
Objectives
Click here for the October Lab Notebook
To help determine why our anti-tracrRNA approach is failing, we extracted RNA and performed RT-PCR to check if antitracr is being properly expressed.  We measured the response of dCas9-repressed GFP fluorescence to different numbers of decoy binding sites and demonstrated successful derepression!! We submitted our decoy arrays to the iGEM headquarters to add to the registry.
Protocols
Protocols can be found here
Retrieved from " http://2014.igem.org/Team:Duke/Notebook/Overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
