<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Duke_Notebook_September"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Duke/Notebook/September">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Duke/Notebook/September&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Notebook/September&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Notebook/September&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Duke/Notebook/September" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Duke/Notebook/September</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><UL id="navbar"><LI><A href="https://2014.igem.org/Team:Duke">Home</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team">About Us</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Duke">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team/Gallery">Photo Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team/Attributions">Attributions</A></LI></UL><LI><A href="https://2014.igem.org/Team:Duke/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#summary">Summary</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#future">Future Directions</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Overview">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Months">Notebook by Month</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy">Policy and Practice</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#threed">3D Printing</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#hoco">House Course</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#smath">NCSSM Team</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Safety">Safety</A></LI></P><DIV class="day"><A id="sept4"><H2>September 4</H2></A><P class="obj">Objective: test backbone shuffled antitracr bacteria</P><UL><LI>Started colonies from plates and diluted in to +/- aTc this morning. Let shake for 5(?) hours before collecting 500uL and spin/resusp in PBS for flow.</LI><LI>Took data to flowjo and plotted histograms below. This is ungated fluorescence</LI><LI>table with mean GFPAs</LI></UL><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TD><P>ungated GFPA</P></TD><TD><P>+/- aTc</P></TD></TR><TR><TD><P>1: if142014-09-04.001.fcs</P></TD><TD><P>AC-1</P></TD><TD><P>+ aTc</P></TD><TD><P>0.402</P></TD><TD><P>1.185840708</P></TD></TR><TR><TD><P>2: if142014-09-04.002.fcs</P></TD><TD><P>AC-2</P></TD><TD><P>+ aTc</P></TD><TD><P>0.455</P></TD><TD><P>1.197368421</P></TD></TR><TR><TD><P>3: if142014-09-04.003.fcs</P></TD><TD><P>G-1</P></TD><TD><P>+ aTc</P></TD><TD><P>14.9</P></TD><TD><P>0.949044586</P></TD></TR><TR><TD><P>4: if142014-09-04.004.fcs</P></TD><TD><P>G-2</P></TD><TD><P>+ aTc</P></TD><TD><P>11.7</P></TD><TD><P>0.573529412</P></TD></TR><TR><TD><P>5: if142014-09-04.005.fcs</P></TD><TD><P>G-3</P></TD><TD><P>+ aTc</P></TD><TD><P>8.91</P></TD><TD><P>0.530357143</P></TD></TR><TR><TD><P>6: if142014-09-04.006.fcs</P></TD><TD><P>G-4</P></TD><TD><P>+ aTc</P></TD><TD><P>5.92</P></TD><TD><P>0.34619883</P></TD></TR><TR><TD><P>7: if142014-09-04.007.fcs</P></TD><TD><P>C-1</P></TD><TD><P>+ aTc</P></TD><TD><P>186</P></TD><TD><P>1.016393443</P></TD></TR><TR><TD><P>8: if142014-09-04.008.fcs</P></TD><TD><P>C-2</P></TD><TD><P>+ aTc</P></TD><TD><P>224</P></TD><TD><P>1.230769231</P></TD></TR><TR><TD><P>9: if142014-09-04.009.fcs</P></TD><TD><P>C-3</P></TD><TD><P>+ aTc</P></TD><TD><P>214</P></TD><TD><P>0.922413793</P></TD></TR><TR><TD><P>10: if142014-09-04.010.fcs</P></TD><TD><P>C-4</P></TD><TD><P>+ aTc</P></TD><TD><P>222</P></TD><TD><P>1.193548387</P></TD></TR><TR><TD><P>11: if142014-09-04.011.fcs</P></TD><TD><P>AC-1</P></TD><TD><P>- aTc</P></TD><TD><P>0.339</P></TD></TR><TR><TD><P>12: if142014-09-04.012.fcs</P></TD><TD><P>AC-2</P></TD><TD><P>- aTc</P></TD><TD><P>0.38</P></TD></TR><TR><TD><P>13: if142014-09-04.013.fcs</P></TD><TD><P>G-1</P></TD><TD><P>- aTc</P></TD><TD><P>15.7</P></TD></TR><TR><TD><P>14: if142014-09-04.014.fcs</P></TD><TD><P>G-2</P></TD><TD><P>- aTc</P></TD><TD><P>20.4</P></TD></TR><TR><TD><P>15: if142014-09-04.015.fcs</P></TD><TD><P>G-3</P></TD><TD><P>- aTc</P></TD><TD><P>16.8</P></TD></TR><TR><TD><P>16: if142014-09-04.016.fcs</P></TD><TD><P>G-4</P></TD><TD><P>- aTc</P></TD><TD><P>17.1</P></TD></TR><TR><TD><P>17: if142014-09-04.017.fcs</P></TD><TD><P>C-1</P></TD><TD><P>- aTc</P></TD><TD><P>183</P></TD></TR><TR><TD><P>18: if142014-09-04.018.fcs</P></TD><TD><P>C-2</P></TD><TD><P>- aTc</P></TD><TD><P>182</P></TD></TR><TR><TD><P>19: if142014-09-04.019.fcs</P></TD><TD><P>C-3</P></TD><TD><P>- aTc</P></TD><TD><P>232</P></TD></TR><TR><TD><P>20: if142014-09-04.020.fcs</P></TD><TD><P>C-4</P></TD><TD><P>- aTc</P></TD><TD><P>186</P></TD></TR></TBODY></TABLE><UL><LI>aTc treatment seems to be moving fluor in the opposite direction of what we’d expect of antitracr was doing its job. The degree is inconsistent, reflow is called for. </LI></UL><H2>September 8</H2><P>Objective: retest backbone shuffled antitracr bacteria</P><UL><LI>Yesterday picked 4 new colonies from plates and 4 from 9/4/14 frozen stocks and grew in LB + antibiotics overnight. Diluted in to +/- aTc this morning and let shake for 5(?) hours before collecting 500uL and spin/resusp in PBS</LI><LI>G1-4 are old frozen coli with K608012, pdCas9-GFP1, and 1A2-R0040-antitracr, C1-4 are old frozen with pdCas9. clones 5-8 are new from plates. AC are no-fluor (1A2 and pdCas9) but they were diluted in to triple antibiotic medium instead of double so they did not grow and are not reliable indicators of bg fluorescence on this run.</LI><LI>After flow, isolated singlets with flowjo (smalls) or left alone (bugs) and got gfpa unmodified or normalized by fsca or ssca. </LI></UL><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TD><P>smalls</P></TD><TD><P>bugs</P></TD></TR><TR><TD><P>aTc</P></TD><TD><P>gfpa</P></TD><TD><P>gfpa/fsca*1000</P></TD><TD><P>gfpa/ssca*1000</P></TD><TD><P>gfpa</P></TD><TD><P>gfpa/fsca*1000</P></TD><TD><P>gfpa/ssca*1000</P></TD></TR><TR><TD><P>1: if142014-09-08.001.fcs</P></TD><TD><P>G1</P></TD><TD><P>+</P></TD><TD><P>5.25</P></TD><TD><P>243</P></TD><TD><P>112</P></TD><TD><P>7.79</P></TD><TD><P>247</P></TD><TD><P>114</P></TD></TR><TR><TD><P>2: if142014-09-08.002.fcs</P></TD><TD><P>G2</P></TD><TD><P>+</P></TD><TD><P>4.41</P></TD><TD><P>226</P></TD><TD><P>78.1</P></TD><TD><P>7.12</P></TD><TD><P>228</P></TD><TD><P>76.2</P></TD></TR><TR><TD><P>3: if142014-09-08.003.fcs</P></TD><TD><P>G3</P></TD><TD><P>+</P></TD><TD><P>4.66</P></TD><TD><P>231</P></TD><TD><P>84.8</P></TD><TD><P>7.8</P></TD><TD><P>231</P></TD><TD><P>83.4</P></TD></TR><TR><TD><P>4: if142014-09-08.004.fcs</P></TD><TD><P>G4</P></TD><TD><P>+</P></TD><TD><P>3.86</P></TD><TD><P>191</P></TD><TD><P>70.1</P></TD><TD><P>5.99</P></TD><TD><P>192</P></TD><TD><P>67.9</P></TD></TR><TR><TD><P>5: if142014-09-08.005.fcs</P></TD><TD><P>G1</P></TD><TD><P>-</P></TD><TD><P>6.81</P></TD><TD><P>311</P></TD><TD><P>145</P></TD><TD><P>8.56</P></TD><TD><P>310</P></TD><TD><P>143</P></TD></TR><TR><TD><P>6: if142014-09-08.006.fcs</P></TD><TD><P>G2</P></TD><TD><P>-</P></TD><TD><P>6.37</P></TD><TD><P>337</P></TD><TD><P>153</P></TD><TD><P>7.25</P></TD><TD><P>323</P></TD><TD><P>145</P></TD></TR><TR><TD><P>7: if142014-09-08.007.fcs</P></TD><TD><P>G3</P></TD><TD><P>-</P></TD><TD><P>6.5</P></TD><TD><P>318</P></TD><TD><P>143</P></TD><TD><P>8.24</P></TD><TD><P>305</P></TD><TD><P>136</P></TD></TR><TR><TD><P>8: if142014-09-08.008.fcs</P></TD><TD><P>G4</P></TD><TD><P>-</P></TD><TD><P>6.14</P></TD><TD><P>298</P></TD><TD><P>140</P></TD><TD><P>7.66</P></TD><TD><P>288</P></TD><TD><P>131</P></TD></TR><TR><TD><P>9: if142014-09-08.009.fcs</P></TD><TD><P>C1</P></TD><TD><P>+</P></TD><TD><P>88.9</P></TD><TD><P>4071</P></TD><TD><P>1777</P></TD><TD><P>117</P></TD><TD><P>3978</P></TD><TD><P>1706</P></TD></TR><TR><TD><P>10: if142014-09-08.010.fcs</P></TD><TD><P>C2</P></TD><TD><P>+</P></TD><TD><P>83.5</P></TD><TD><P>4073</P></TD><TD><P>1547</P></TD><TD><P>120</P></TD><TD><P>3969</P></TD><TD><P>1422</P></TD></TR><TR><TD><P>11: if142014-09-08.011.fcs</P></TD><TD><P>C3</P></TD><TD><P>+</P></TD><TD><P>81.7</P></TD><TD><P>3957</P></TD><TD><P>1551</P></TD><TD><P>115</P></TD><TD><P>3840</P></TD><TD><P>1437</P></TD></TR><TR><TD><P>12: if142014-09-08.012.fcs</P></TD><TD><P>C4</P></TD><TD><P>+</P></TD><TD><P>77.2</P></TD><TD><P>3915</P></TD><TD><P>1577</P></TD><TD><P>95.7</P></TD><TD><P>3883</P></TD><TD><P>1503</P></TD></TR><TR><TD><P>13: if142014-09-08.013.fcs</P></TD><TD><P>C1</P></TD><TD><P>-</P></TD><TD><P>77.7</P></TD><TD><P>3686</P></TD><TD><P>1471</P></TD><TD><P>100</P></TD><TD><P>3545</P></TD><TD><P>1411</P></TD></TR><TR><TD><P>14: if142014-09-08.014.fcs</P></TD><TD><P>C2</P></TD><TD><P>-</P></TD><TD><P>64.5</P></TD><TD><P>3093</P></TD><TD><P>1445</P></TD><TD><P>88.3</P></TD><TD><P>2962</P></TD><TD><P>1373</P></TD></TR><TR><TD><P>15: if142014-09-08.015.fcs</P></TD><TD><P>C3</P></TD><TD><P>-</P></TD><TD><P>80.9</P></TD><TD><P>3971</P></TD><TD><P>1541</P></TD><TD><P>106</P></TD><TD><P>3771</P></TD><TD><P>1432</P></TD></TR><TR><TD><P>16: if142014-09-08.016.fcs</P></TD><TD><P>C4</P></TD><TD><P>-</P></TD><TD><P>58.9</P></TD><TD><P>2899</P></TD><TD><P>1299</P></TD><TD><P>79.7</P></TD><TD><P>2727</P></TD><TD><P>1220</P></TD></TR><TR><TD><P>17: if142014-09-08.017.fcs</P></TD><TD><P>G5</P></TD><TD><P>+</P></TD><TD><P>4.1</P></TD><TD><P>210</P></TD><TD><P>79.1</P></TD><TD><P>5.4</P></TD><TD><P>209</P></TD><TD><P>76.2</P></TD></TR><TR><TD><P>18: if142014-09-08.018.fcs</P></TD><TD><P>G6</P></TD><TD><P>+</P></TD><TD><P>4.9</P></TD><TD><P>225</P></TD><TD><P>98.2</P></TD><TD><P>7.07</P></TD><TD><P>226</P></TD><TD><P>97.6</P></TD></TR><TR><TD><P>19: if142014-09-08.019.fcs</P></TD><TD><P>G7</P></TD><TD><P>+</P></TD><TD><P>4.42</P></TD><TD><P>220</P></TD><TD><P>84.7</P></TD><TD><P>5.97</P></TD><TD><P>218</P></TD><TD><P>81.7</P></TD></TR><TR><TD><P>20: if142014-09-08.020.fcs</P></TD><TD><P>G8</P></TD><TD><P>+</P></TD><TD><P>4.19</P></TD><TD><P>211</P></TD><TD><P>81.3</P></TD><TD><P>5.68</P></TD><TD><P>213</P></TD><TD><P>79.5</P></TD></TR><TR><TD><P>21: if142014-09-08.021.fcs</P></TD><TD><P>G5</P></TD><TD><P>-</P></TD><TD><P>9.73</P></TD><TD><P>477</P></TD><TD><P>201</P></TD><TD><P>12.5</P></TD><TD><P>455</P></TD><TD><P>190</P></TD></TR><TR><TD><P>22: if142014-09-08.022.fcs</P></TD><TD><P>G6</P></TD><TD><P>-</P></TD><TD><P>9.68</P></TD><TD><P>450</P></TD><TD><P>197</P></TD><TD><P>12.9</P></TD><TD><P>449</P></TD><TD><P>196</P></TD></TR><TR><TD><P>23: if142014-09-08.023.fcs</P></TD><TD><P>G7</P></TD><TD><P>-</P></TD><TD><P>10.1</P></TD><TD><P>482</P></TD><TD><P>212</P></TD><TD><P>13.3</P></TD><TD><P>464</P></TD><TD><P>198</P></TD></TR><TR><TD><P>24: if142014-09-08.024.fcs</P></TD><TD><P>G8</P></TD><TD><P>-</P></TD><TD><P>10.1</P></TD><TD><P>489</P></TD><TD><P>216</P></TD><TD><P>13.3</P></TD><TD><P>473</P></TD><TD><P>204</P></TD></TR><TR><TD><P>25: if142014-09-08.025.fcs</P></TD><TD><P>C5</P></TD><TD><P>+</P></TD><TD><P>79.6</P></TD><TD><P>3958</P></TD><TD><P>1486</P></TD><TD><P>113</P></TD><TD><P>3813</P></TD><TD><P>1374</P></TD></TR><TR><TD><P>26: if142014-09-08.026.fcs</P></TD><TD><P>C6</P></TD><TD><P>+</P></TD><TD><P>80.8</P></TD><TD><P>4045</P></TD><TD><P>1514</P></TD><TD><P>113</P></TD><TD><P>4043</P></TD><TD><P>1425</P></TD></TR><TR><TD><P>27: if142014-09-08.027.fcs</P></TD><TD><P>C7</P></TD><TD><P>+</P></TD><TD><P>80.7</P></TD><TD><P>4053</P></TD><TD><P>1491</P></TD><TD><P>116</P></TD><TD><P>3921</P></TD><TD><P>1375</P></TD></TR><TR><TD><P>28: if142014-09-08.028.fcs</P></TD><TD><P>C8</P></TD><TD><P>+</P></TD><TD><P>79.8</P></TD><TD><P>3804</P></TD><TD><P>1555</P></TD><TD><P>103</P></TD><TD><P>3634</P></TD><TD><P>1508</P></TD></TR><TR><TD><P>29: if142014-09-08.029.fcs</P></TD><TD><P>C5</P></TD><TD><P>-</P></TD><TD><P>68.9</P></TD><TD><P>3268</P></TD><TD><P>1505</P></TD><TD><P>95.7</P></TD><TD><P>3128</P></TD><TD><P>1419</P></TD></TR><TR><TD><P>30: if142014-09-08.030.fcs</P></TD><TD><P>C6</P></TD><TD><P>-</P></TD><TD><P>80.2</P></TD><TD><P>3806</P></TD><TD><P>1639</P></TD><TD><P>110</P></TD><TD><P>3630</P></TD><TD><P>1541</P></TD></TR><TR><TD><P>31: if142014-09-08.031.fcs</P></TD><TD><P>C7</P></TD><TD><P>-</P></TD><TD><P>67.8</P></TD><TD><P>3240</P></TD><TD><P>1498</P></TD><TD><P>91.7</P></TD><TD><P>3041</P></TD><TD><P>1412</P></TD></TR><TR><TD><P>32: if142014-09-08.032.fcs</P></TD><TD><P>C8</P></TD><TD><P>-</P></TD><TD><P>83.7</P></TD><TD><P>3810</P></TD><TD><P>1675</P></TD><TD><P>107</P></TD><TD><P>3618</P></TD><TD><P>1621</P></TD></TR><TR><TD><P>33: if142014-09-08.033.fcs</P></TD><TD><P>AC1</P></TD><TD><P>+</P></TD><TD><P>0.895</P></TD><TD><P>40.2</P></TD><TD><P>16.8</P></TD><TD><P>0.854</P></TD><TD><P>34.9</P></TD><TD><P>12.9</P></TD></TR><TR><TD><P>34: if142014-09-08.034.fcs</P></TD><TD><P>AC3</P></TD><TD><P>+</P></TD><TD><P>0.539</P></TD><TD><P>28.4</P></TD><TD><P>11</P></TD><TD><P>0.785</P></TD><TD><P>27.4</P></TD><TD><P>10.7</P></TD></TR><TR><TD><P>35: if142014-09-08.035.fcs</P></TD><TD><P>AC1</P></TD><TD><P>-</P></TD><TD><P>0.464</P></TD><TD><P>25.8</P></TD><TD><P>9.39</P></TD><TD><P>0.605</P></TD><TD><P>25.9</P></TD><TD><P>8.65</P></TD></TR><TR><TD><P>36: if142014-09-08.036.fcs</P></TD><TD><P>AC3</P></TD><TD><P>-</P></TD><TD><P>0.532</P></TD><TD><P>28.3</P></TD><TD><P>10.4</P></TD><TD><P>0.661</P></TD><TD><P>26.2</P></TD><TD><P>9.23</P></TD></TR></TBODY></TABLE><UL><LI>Got ratios of fluorescence in +/- aTc and am reporting them in the table below</LI></UL><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TD><P>n/a</P></TD><TD><P>n/a</P></TD><TD><P>+/-</P></TD><TD><P>smalls</P></TD><TD><P>bugs</P></TD></TR><TR><TD><P>n/a</P></TD><TD><P>n/a</P></TD><TD><P>gfpa</P></TD><TD><P>gfpa/fsca*1000</P></TD><TD><P>gfpa/ssca*1000</P></TD><TD><P>gfpa</P></TD><TD><P>gfpa/fsca*1000</P></TD><TD><P>gfpa/ssca*1000</P></TD></TR><TR><TD><P>1: if142014-09-08.001.fcs</P></TD><TD><P>G1</P></TD><TD><P>0.771</P></TD><TD><P>0.781</P></TD><TD><P>0.772</P></TD><TD><P>0.910</P></TD><TD><P>0.797</P></TD><TD><P>0.797</P></TD></TR><TR><TD><P>2: if142014-09-08.002.fcs</P></TD><TD><P>G2</P></TD><TD><P>0.692</P></TD><TD><P>0.671</P></TD><TD><P>0.510</P></TD><TD><P>0.982</P></TD><TD><P>0.706</P></TD><TD><P>0.526</P></TD></TR><TR><TD><P>3: if142014-09-08.003.fcs</P></TD><TD><P>G3</P></TD><TD><P>0.717</P></TD><TD><P>0.726</P></TD><TD><P>0.593</P></TD><TD><P>0.947</P></TD><TD><P>0.757</P></TD><TD><P>0.613</P></TD></TR><TR><TD><P>4: if142014-09-08.004.fcs</P></TD><TD><P>G4</P></TD><TD><P>0.629</P></TD><TD><P>0.641</P></TD><TD><P>0.501</P></TD><TD><P>0.782</P></TD><TD><P>0.667</P></TD><TD><P>0.518</P></TD></TR><TR><TD><P>9: if142014-09-08.009.fcs</P></TD><TD><P>C1</P></TD><TD><P>1.144</P></TD><TD><P>1.104</P></TD><TD><P>1.208</P></TD><TD><P>1.170</P></TD><TD><P>1.122</P></TD><TD><P>1.209</P></TD></TR><TR><TD><P>10: if142014-09-08.010.fcs</P></TD><TD><P>C2</P></TD><TD><P>1.295</P></TD><TD><P>1.317</P></TD><TD><P>1.071</P></TD><TD><P>1.359</P></TD><TD><P>1.340</P></TD><TD><P>1.036</P></TD></TR><TR><TD><P>11: if142014-09-08.011.fcs</P></TD><TD><P>C3</P></TD><TD><P>1.010</P></TD><TD><P>0.996</P></TD><TD><P>1.006</P></TD><TD><P>1.085</P></TD><TD><P>1.018</P></TD><TD><P>1.003</P></TD></TR><TR><TD><P>12: if142014-09-08.012.fcs</P></TD><TD><P>C4</P></TD><TD><P>1.311</P></TD><TD><P>1.350</P></TD><TD><P>1.214</P></TD><TD><P>1.201</P></TD><TD><P>1.424</P></TD><TD><P>1.232</P></TD></TR><TR><TD><P>17: if142014-09-08.017.fcs</P></TD><TD><P>G5</P></TD><TD><P>0.421</P></TD><TD><P>0.440</P></TD><TD><P>0.394</P></TD><TD><P>0.432</P></TD><TD><P>0.459</P></TD><TD><P>0.401</P></TD></TR><TR><TD><P>18: if142014-09-08.018.fcs</P></TD><TD><P>G6</P></TD><TD><P>0.506</P></TD><TD><P>0.500</P></TD><TD><P>0.498</P></TD><TD><P>0.548</P></TD><TD><P>0.503</P></TD><TD><P>0.498</P></TD></TR><TR><TD><P>19: if142014-09-08.019.fcs</P></TD><TD><P>G7</P></TD><TD><P>0.438</P></TD><TD><P>0.456</P></TD><TD><P>0.400</P></TD><TD><P>0.449</P></TD><TD><P>0.470</P></TD><TD><P>0.413</P></TD></TR><TR><TD><P>20: if142014-09-08.020.fcs</P></TD><TD><P>G8</P></TD><TD><P>0.415</P></TD><TD><P>0.431</P></TD><TD><P>0.376</P></TD><TD><P>0.427</P></TD><TD><P>0.450</P></TD><TD><P>0.390</P></TD></TR><TR><TD><P>25: if142014-09-08.025.fcs</P></TD><TD><P>C5</P></TD><TD><P>1.155</P></TD><TD><P>1.211</P></TD><TD><P>0.987</P></TD><TD><P>1.181</P></TD><TD><P>1.219</P></TD><TD><P>0.968</P></TD></TR><TR><TD><P>26: if142014-09-08.026.fcs</P></TD><TD><P>C6</P></TD><TD><P>1.007</P></TD><TD><P>1.063</P></TD><TD><P>0.924</P></TD><TD><P>1.027</P></TD><TD><P>1.114</P></TD><TD><P>0.925</P></TD></TR><TR><TD><P>27: if142014-09-08.027.fcs</P></TD><TD><P>C7</P></TD><TD><P>1.190</P></TD><TD><P>1.251</P></TD><TD><P>0.995</P></TD><TD><P>1.265</P></TD><TD><P>1.289</P></TD><TD><P>0.974</P></TD></TR><TR><TD><P>28: if142014-09-08.028.fcs</P></TD><TD><P>C8</P></TD><TD><P>0.953</P></TD><TD><P>0.998</P></TD><TD><P>0.928</P></TD><TD><P>0.963</P></TD><TD><P>1.004</P></TD><TD><P>0.930</P></TD></TR><TR><TD><P>33: if142014-09-08.033.fcs</P></TD><TD><P>AC1</P></TD><TD><P>1.929</P></TD><TD><P>1.558</P></TD><TD><P>1.789</P></TD><TD><P>1.412</P></TD><TD><P>1.347</P></TD><TD><P>1.491</P></TD></TR><TR><TD><P>34: if142014-09-08.034.fcs</P></TD><TD><P>AC3</P></TD><TD><P>1.013</P></TD><TD><P>1.004</P></TD><TD><P>1.058</P></TD><TD><P>1.188</P></TD><TD><P>1.046</P></TD><TD><P>1.159</P></TD></TR></TBODY></TABLE><P>Box plot of just G1-4 and C1-4. Note that the G clones from plates (clones 5-8) have even smaller ratios. </P><H2>September 9</H2><P>Objective: assembly decoy binding site arrays</P><UL><LI>Garima sets up a PCR and is asked to fill in notes about it.</LI><LI>Results</LI><LI>Charlie sets up a new set of tubes for PCR with a 5 rx master mix containing</LI></UL><UL><LI>190uL H2O</LI><LI>50 uL Q5 buffer</LI><LI>2.0 uL oligo 1 (gfp1-spacer-gfp1)</LI><LI>2.5 uL Q5 poly</LI><LI>5 uL dNTPs</LI></UL><UL><LI>And then divides that in to four tubes before adding in the following volumes of oligo 2 (spacer-gfp1-space)</LI></UL><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TD><P>tube number</P></TD><TD><P>1</P></TD><TD><P>2</P></TD><TD><P>3</P></TD><TD><P>4</P></TD></TR><TR><TD><P>uL oligo 2</P></TD><TD><P>0</P></TD><TD><P>0.5</P></TD><TD><P>1.0</P></TD><TD><P>2.0</P></TD></TR></TBODY></TABLE><UL><LI>Streakiness on the row with 0.5uL of oligo 2 looks promising? IGEM protocol on the thermal cycler modified</LI></UL><H2>September 10</H2><P>Objective: assemble decoy binding site arrays</P><UL><LI>Setting up two PCRs with same cycling program as last time: </LI></UL><UL><LI>one with 4x 50uL reactions using the same concs of everything as the 0.5 uL rx from 9/9/14 (blue labels)</LI><LI>one with equimolar oligo1 and oligo2 and varying amounts of reaction 0.5 from 9/9/14 (green labels)</LI></UL><UL><LI>So something’s happening. PCR cleanup’d reactions 1-4 (green) and 5-8 (blue) in to one tube each labeled Decoy array (1-4 or 5-8) PCR1. Tomorrow will PCR with the end oligos and try gibson, dig/lig in to pSB1C3 and/or pSB1A2</LI></UL><H2>9/11/14</H2><P>Objective: assemble decoy binding site arrays. Step 2: add prefix &amp; suffix to arrays</P><UL><LI>Faw setting up two series of PCRs with 1c337prfGfp1Space and 1c337sufGfp1Space as oligos on 0.5-2.0 ng of PCR cleanup’d green and blue rx from 9/10/14 and running with the same cycling conditions.</LI><LI>CBC did not take a picture and doesn’t remember why. PCR looked bad for some reason.</LI></UL><H2><A id="sept12">September 12</A></H2><P>Objective: assemble decoy binding site arrays. Step 2: add prefix &amp; suffix to arrays</P><UL><LI>Garima setting up two series of PCRs with 1c337prfGfp1Space and 1c337sufGfp1Space as oligos on higher concs than yesterday ofPCR cleanup’d green and blue rx from 9/10/14 and running with the same cycling conditions.</LI><LI>PCR cleanup to pool tubes 2-4 for both series and labeled Decoy Array PCR2</LI></UL><H2><A id="sept13">September 13</A></H2><P>Objective: assemble decoy binding site arrays. Step 3: insert putative arrays in to pSB backbone</P><UL><LI>CBC attempts a SLIC with 0.5 uL of the “Super Clean” XbaI/PstI pSB1C3 backbone. Note that puts 15 bp of nonhomology in the way of successful SLIC. I did this because sometimes it works despite mismatches and there was no EcoRI/PstI dig’d pSB1C3 for me to work with. Green series looks like there is a bigger difference between no-template and +template tubes, so I will try that</LI></UL><UL><LI>set up tubes for backbone-only, backbone + 0.5, 1, 2, 3uL of insert from Green tube at 88.8 ng/uL, and 3uL insert-only.</LI></UL><UL><LI>Didn’t work.</LI></UL><P>Objective: observe fluorescence changes with antitracr time course over time</P><UL><LI>Last night antitracr strains started in LB+AmpKanCm and let shake overnight. Diluted this morning in to +/- aTc and began collection at time indicated. Replaced LB with PBS before flowing</LI><LI>Isolated “singlets” which are probably not singlets (smalls) or left alone (bugs) and got mean GFPA or gfpa/fsca or /ssca to normalize roughly to size.</LI></UL><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TD><P>time point</P></TD><TD><P>n/a</P></TD><TD><P>n/a</P></TD><TD><P>1</P></TD><TD><P>2</P></TD><TD><P>3</P></TD><TD><P>4</P></TD><TD><P>5</P></TD></TR><TR><TD><P>time (min)</P></TD><TD><P>n/a</P></TD><TD><P>n/a</P></TD><TD><P>120</P></TD><TD><P>180</P></TD><TD><P>240</P></TD><TD><P>300</P></TD><TD><P>360</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>1.52</P></TD><TD><P>2.74</P></TD><TD><P>3.17</P></TD><TD><P>3.16</P></TD><TD><P>2.81</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>1.39</P></TD><TD><P>2.8</P></TD><TD><P>3.18</P></TD><TD><P>3.42</P></TD><TD><P>2.88</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>1.45</P></TD><TD><P>2.92</P></TD><TD><P>3.31</P></TD><TD><P>3.53</P></TD><TD><P>2.88</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>1.53</P></TD><TD><P>3.13</P></TD><TD><P>3.72</P></TD><TD><P>3.6</P></TD><TD><P>2.89</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>45.6</P></TD><TD><P>56</P></TD><TD><P>68.4</P></TD><TD><P>76.8</P></TD><TD><P>74.9</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>46.9</P></TD><TD><P>54</P></TD><TD><P>67.8</P></TD><TD><P>73.8</P></TD><TD><P>74.7</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>48.4</P></TD><TD><P>53.1</P></TD><TD><P>65.8</P></TD><TD><P>73</P></TD><TD><P>76.2</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>45.6</P></TD><TD><P>54.2</P></TD><TD><P>68.2</P></TD><TD><P>74.3</P></TD><TD><P>78.4</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>0.109</P></TD><TD><P>0.16</P></TD><TD><P>0.185</P></TD><TD><P>0.139</P></TD><TD><P>0.143</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>1.43</P></TD><TD><P>3.08</P></TD><TD><P>3.65</P></TD><TD><P>3.67</P></TD><TD><P>3.07</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>1.32</P></TD><TD><P>3.17</P></TD><TD><P>3.67</P></TD><TD><P>3.88</P></TD><TD><P>3.07</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>1.73</P></TD><TD><P>3.3</P></TD><TD><P>4.15</P></TD><TD><P>3.99</P></TD><TD><P>3.18</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>1.57</P></TD><TD><P>3.47</P></TD><TD><P>4.16</P></TD><TD><P>3.83</P></TD><TD><P>3.29</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>41.7</P></TD><TD><P>53.4</P></TD><TD><P>67</P></TD><TD><P>75.8</P></TD><TD><P>75.6</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>41.4</P></TD><TD><P>51.3</P></TD><TD><P>67.4</P></TD><TD><P>74.3</P></TD><TD><P>71.7</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>39.4</P></TD><TD><P>52.4</P></TD><TD><P>66</P></TD><TD><P>73.9</P></TD><TD><P>60.8</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>40</P></TD><TD><P>52.7</P></TD><TD><P>67</P></TD><TD><P>76</P></TD><TD><P>73.2</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (smalls)</P></TD><TD><P>0.0949</P></TD><TD><P>0.156</P></TD><TD><P>0.162</P></TD><TD><P>0.129</P></TD><TD><P>0.122</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>83.9</P></TD><TD><P>157</P></TD><TD><P>188</P></TD><TD><P>189</P></TD><TD><P>172</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>70.6</P></TD><TD><P>159</P></TD><TD><P>186</P></TD><TD><P>205</P></TD><TD><P>177</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>72.8</P></TD><TD><P>169</P></TD><TD><P>193</P></TD><TD><P>208</P></TD><TD><P>175</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>80.1</P></TD><TD><P>172</P></TD><TD><P>206</P></TD><TD><P>199</P></TD><TD><P>166</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>2526</P></TD><TD><P>3227</P></TD><TD><P>3986</P></TD><TD><P>4592</P></TD><TD><P>4520</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>2700</P></TD><TD><P>3078</P></TD><TD><P>4044</P></TD><TD><P>4486</P></TD><TD><P>4543</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>2892</P></TD><TD><P>3028</P></TD><TD><P>3795</P></TD><TD><P>4291</P></TD><TD><P>4553</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>2641</P></TD><TD><P>3131</P></TD><TD><P>4038</P></TD><TD><P>4531</P></TD><TD><P>4614</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>6.26</P></TD><TD><P>9.36</P></TD><TD><P>11.1</P></TD><TD><P>8.92</P></TD><TD><P>9.11</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>72.8</P></TD><TD><P>177</P></TD><TD><P>221</P></TD><TD><P>228</P></TD><TD><P>194</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>60.5</P></TD><TD><P>187</P></TD><TD><P>220</P></TD><TD><P>241</P></TD><TD><P>193</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>94.8</P></TD><TD><P>192</P></TD><TD><P>245</P></TD><TD><P>245</P></TD><TD><P>199</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>79.6</P></TD><TD><P>193</P></TD><TD><P>235</P></TD><TD><P>222</P></TD><TD><P>193</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>2184</P></TD><TD><P>3100</P></TD><TD><P>4012</P></TD><TD><P>4663</P></TD><TD><P>4666</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>2188</P></TD><TD><P>2957</P></TD><TD><P>4160</P></TD><TD><P>4554</P></TD><TD><P>4386</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>2004</P></TD><TD><P>3047</P></TD><TD><P>4035</P></TD><TD><P>4565</P></TD><TD><P>3842</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>2039</P></TD><TD><P>3051</P></TD><TD><P>4107</P></TD><TD><P>4630</P></TD><TD><P>4512</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (smalls)</P></TD><TD><P>5.64</P></TD><TD><P>9.13</P></TD><TD><P>9.66</P></TD><TD><P>8.24</P></TD><TD><P>7.79</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>47.7</P></TD><TD><P>75.1</P></TD><TD><P>78.8</P></TD><TD><P>75.1</P></TD><TD><P>64.8</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>46.3</P></TD><TD><P>76.2</P></TD><TD><P>79.1</P></TD><TD><P>81</P></TD><TD><P>66</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>48</P></TD><TD><P>78.7</P></TD><TD><P>82.6</P></TD><TD><P>83.3</P></TD><TD><P>66.4</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>51.5</P></TD><TD><P>85.9</P></TD><TD><P>96.7</P></TD><TD><P>93.4</P></TD><TD><P>71.1</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>1273</P></TD><TD><P>1469</P></TD><TD><P>1666</P></TD><TD><P>1779</P></TD><TD><P>1718</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>1226</P></TD><TD><P>1413</P></TD><TD><P>1636</P></TD><TD><P>1719</P></TD><TD><P>1688</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>1182</P></TD><TD><P>1382</P></TD><TD><P>1605</P></TD><TD><P>1704</P></TD><TD><P>1739</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>1182</P></TD><TD><P>1391</P></TD><TD><P>1645</P></TD><TD><P>1714</P></TD><TD><P>1790</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>3.43</P></TD><TD><P>4.38</P></TD><TD><P>4.82</P></TD><TD><P>3.89</P></TD><TD><P>3.69</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>46.3</P></TD><TD><P>80.9</P></TD><TD><P>89.5</P></TD><TD><P>87.5</P></TD><TD><P>71</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>45.6</P></TD><TD><P>80.6</P></TD><TD><P>89.6</P></TD><TD><P>91.3</P></TD><TD><P>71.4</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>50.4</P></TD><TD><P>84.4</P></TD><TD><P>101</P></TD><TD><P>95</P></TD><TD><P>74</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>55.3</P></TD><TD><P>93.8</P></TD><TD><P>108</P></TD><TD><P>100</P></TD><TD><P>83.8</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>1250</P></TD><TD><P>1375</P></TD><TD><P>1606</P></TD><TD><P>1758</P></TD><TD><P>1715</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>1215</P></TD><TD><P>1320</P></TD><TD><P>1606</P></TD><TD><P>1734</P></TD><TD><P>1635</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>1203</P></TD><TD><P>1319</P></TD><TD><P>1563</P></TD><TD><P>1718</P></TD><TD><P>1425</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>1230</P></TD><TD><P>1338</P></TD><TD><P>1595</P></TD><TD><P>1754</P></TD><TD><P>1661</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (smalls)</P></TD><TD><P>3.05</P></TD><TD><P>4.55</P></TD><TD><P>4.63</P></TD><TD><P>3.67</P></TD><TD><P>3.35</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>2.17</P></TD><TD><P>3.7</P></TD><TD><P>5.12</P></TD><TD><P>5.27</P></TD><TD><P>5.68</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>2.1</P></TD><TD><P>3.82</P></TD><TD><P>5.06</P></TD><TD><P>5.77</P></TD><TD><P>5.85</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>2.17</P></TD><TD><P>3.92</P></TD><TD><P>5.48</P></TD><TD><P>6.04</P></TD><TD><P>5.89</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>2.07</P></TD><TD><P>4.2</P></TD><TD><P>5.49</P></TD><TD><P>5.95</P></TD><TD><P>5.85</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>70.3</P></TD><TD><P>82.2</P></TD><TD><P>105</P></TD><TD><P>120</P></TD><TD><P>133</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>74.4</P></TD><TD><P>78.2</P></TD><TD><P>102</P></TD><TD><P>122</P></TD><TD><P>137</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>75.6</P></TD><TD><P>76.9</P></TD><TD><P>102</P></TD><TD><P>114</P></TD><TD><P>130</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>75.3</P></TD><TD><P>79.9</P></TD><TD><P>103</P></TD><TD><P>120</P></TD><TD><P>136</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>0.169</P></TD><TD><P>0.26</P></TD><TD><P>0.396</P></TD><TD><P>0.319</P></TD><TD><P>0.34</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>2.26</P></TD><TD><P>4.28</P></TD><TD><P>6.12</P></TD><TD><P>6.62</P></TD><TD><P>6.7</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>2.01</P></TD><TD><P>4.53</P></TD><TD><P>6.07</P></TD><TD><P>7.13</P></TD><TD><P>6.9</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>2.72</P></TD><TD><P>4.59</P></TD><TD><P>6.58</P></TD><TD><P>7.36</P></TD><TD><P>7.43</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>2.16</P></TD><TD><P>4.63</P></TD><TD><P>6.39</P></TD><TD><P>7.15</P></TD><TD><P>7.22</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>63.4</P></TD><TD><P>76.5</P></TD><TD><P>107</P></TD><TD><P>127</P></TD><TD><P>141</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>64.1</P></TD><TD><P>74.2</P></TD><TD><P>112</P></TD><TD><P>127</P></TD><TD><P>134</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>61.2</P></TD><TD><P>77.2</P></TD><TD><P>110</P></TD><TD><P>127</P></TD><TD><P>126</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>63.3</P></TD><TD><P>78.4</P></TD><TD><P>110</P></TD><TD><P>128</P></TD><TD><P>138</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa (bugs)</P></TD><TD><P>0.157</P></TD><TD><P>0.26</P></TD><TD><P>0.357</P></TD><TD><P>0.331</P></TD><TD><P>0.287</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>88.5</P></TD><TD><P>159</P></TD><TD><P>205</P></TD><TD><P>206</P></TD><TD><P>216</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>75.6</P></TD><TD><P>163</P></TD><TD><P>197</P></TD><TD><P>228</P></TD><TD><P>220</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>77.3</P></TD><TD><P>172</P></TD><TD><P>210</P></TD><TD><P>232</P></TD><TD><P>217</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>83.4</P></TD><TD><P>174</P></TD><TD><P>215</P></TD><TD><P>227</P></TD><TD><P>223</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>2559</P></TD><TD><P>3229</P></TD><TD><P>4012</P></TD><TD><P>4818</P></TD><TD><P>5003</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>2751</P></TD><TD><P>3078</P></TD><TD><P>4107</P></TD><TD><P>4768</P></TD><TD><P>5130</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>3006</P></TD><TD><P>3014</P></TD><TD><P>3787</P></TD><TD><P>4456</P></TD><TD><P>4927</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>2695</P></TD><TD><P>3129</P></TD><TD><P>4076</P></TD><TD><P>4738</P></TD><TD><P>5032</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>6.33</P></TD><TD><P>10.1</P></TD><TD><P>15.8</P></TD><TD><P>12.9</P></TD><TD><P>14.1</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>78.4</P></TD><TD><P>182</P></TD><TD><P>242</P></TD><TD><P>263</P></TD><TD><P>257</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>64.6</P></TD><TD><P>194</P></TD><TD><P>236</P></TD><TD><P>278</P></TD><TD><P>259</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>101</P></TD><TD><P>195</P></TD><TD><P>263</P></TD><TD><P>287</P></TD><TD><P>277</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>82.5</P></TD><TD><P>195</P></TD><TD><P>250</P></TD><TD><P>270</P></TD><TD><P>274</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>2213</P></TD><TD><P>3106</P></TD><TD><P>4099</P></TD><TD><P>5045</P></TD><TD><P>5411</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>2212</P></TD><TD><P>2976</P></TD><TD><P>4289</P></TD><TD><P>5022</P></TD><TD><P>5084</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>2022</P></TD><TD><P>3060</P></TD><TD><P>4164</P></TD><TD><P>4964</P></TD><TD><P>4574</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>2061</P></TD><TD><P>3053</P></TD><TD><P>4158</P></TD><TD><P>4990</P></TD><TD><P>5205</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/fsca*1000 (bugs)</P></TD><TD><P>5.59</P></TD><TD><P>10</P></TD><TD><P>14.4</P></TD><TD><P>13.3</P></TD><TD><P>12.3</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>49</P></TD><TD><P>73.4</P></TD><TD><P>74.8</P></TD><TD><P>71.9</P></TD><TD><P>69</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>48.4</P></TD><TD><P>74.5</P></TD><TD><P>74.4</P></TD><TD><P>77.2</P></TD><TD><P>68.9</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>49.8</P></TD><TD><P>77.1</P></TD><TD><P>79.2</P></TD><TD><P>80.5</P></TD><TD><P>69.6</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>53</P></TD><TD><P>84.8</P></TD><TD><P>90.9</P></TD><TD><P>88.6</P></TD><TD><P>74.1</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>1267</P></TD><TD><P>1433</P></TD><TD><P>1538</P></TD><TD><P>1591</P></TD><TD><P>1583</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>1210</P></TD><TD><P>1380</P></TD><TD><P>1489</P></TD><TD><P>1535</P></TD><TD><P>1541</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>1170</P></TD><TD><P>1359</P></TD><TD><P>1497</P></TD><TD><P>1514</P></TD><TD><P>1583</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>1174</P></TD><TD><P>1356</P></TD><TD><P>1512</P></TD><TD><P>1531</P></TD><TD><P>1634</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>+ aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>3.25</P></TD><TD><P>4.27</P></TD><TD><P>5.57</P></TD><TD><P>4.17</P></TD><TD><P>4.13</P></TD></TR><TR><TD><P>GFP1-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>48.5</P></TD><TD><P>77.5</P></TD><TD><P>84</P></TD><TD><P>84.1</P></TD><TD><P>77.8</P></TD></TR><TR><TD><P>GFP1-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>47.8</P></TD><TD><P>77.4</P></TD><TD><P>83.9</P></TD><TD><P>88.1</P></TD><TD><P>76.9</P></TD></TR><TR><TD><P>GFP1-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>51.9</P></TD><TD><P>80.9</P></TD><TD><P>94.2</P></TD><TD><P>92.9</P></TD><TD><P>82.8</P></TD></TR><TR><TD><P>GFP1-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>56.8</P></TD><TD><P>90.5</P></TD><TD><P>101</P></TD><TD><P>94.2</P></TD><TD><P>90.5</P></TD></TR><TR><TD><P>blankpdCas9-9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>1246</P></TD><TD><P>1323</P></TD><TD><P>1446</P></TD><TD><P>1575</P></TD><TD><P>1613</P></TD></TR><TR><TD><P>blankpdCas9-10</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>1208</P></TD><TD><P>1269</P></TD><TD><P>1413</P></TD><TD><P>1553</P></TD><TD><P>1529</P></TD></TR><TR><TD><P>blankpdCas9-11</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>1196</P></TD><TD><P>1255</P></TD><TD><P>1395</P></TD><TD><P>1529</P></TD><TD><P>1366</P></TD></TR><TR><TD><P>blankpdCas9-12</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>1231</P></TD><TD><P>1282</P></TD><TD><P>1420</P></TD><TD><P>1561</P></TD><TD><P>1553</P></TD></TR><TR><TD><P>pdCas9</P></TD><TD><P>- aTc</P></TD><TD><P>gfpa/ssca*1000 (bugs)</P></TD><TD><P>2.82</P></TD><TD><P>4.23</P></TD><TD><P>4.92</P></TD><TD><P>4.26</P></TD><TD><P>3.62</P></TD></TR></TBODY></TABLE><UL><LI>It looks like the decrease in fluor for pdCas9-GFP1 coli manifests fairly early and persists for 6 hours. Lutz &amp; Bujard’s paper use slightly less aTc for full induction of pTet, but if the decrease we see here were due to aTc inhibiting translation then we would expect to see a decrease in +aTc cultures with pdCas9. If pTet expression of aTc was removing trasncription/translation resources away from GFP production then you would also expect that to manifest in pdCas9. However it might be possible that the pdCas9 strains are experiencing the same decrease from aTc or growht burden but the strong GFP expression is masking the effect.</LI><LI>The other possibility is that something scientifically interesting, but not useful for iGEM purposes, is going on.</LI></UL><H2><A id="sept15">September 15</A></H2><P>Objective: assemble decoy binding site arrays. Step 2: add prefix &amp; suffix to arrays (retry)</P><UL><LI>CBC sets up PCR with 0.5 ng/uL green or blue PCR1 template with 0, 0.4, 0.7, 1.0 uL in 50uL reactions with same cycling conditions and oligo concs etc as before. Maybe with less template the dominant species in the tubes will be that with prefix &amp; suffix added. Maybe</LI><LI>No difference between + template and notemplate tubes. No picture taken.</LI></UL><H2><A id="sept16">September 16</A></H2><P>Objective: assemble decoy binding site arrays. Step 3: insert putative arrays in to pSB backbone</P><UL><LI>CBC prays, puts PCR2 products used on 9/13/14 in a digestion reaction with EcoRI/PstI and incubates 37C for 4h. PCR cleanup and CIP (buffer 2) for 20m then cleanup again. Both tubes have ~35 ng/uL and Dusseldorf ligation calculator calls for 12 ng of 100 (ish) bp insert  for 86 ng 2000 bp bb. Assuming that the large majority of the inserts lack the prefix and suffix, I will try 1 and 2 uL of insert for 1 of the bb. Also bb only and insert only, of course. Ligating 90m at RT</LI><LI>Shocking in to Z1; 30m on ice with 10uL of the ligation mix, 80m recovery before plating on LB+Cm</LI><LI>next morning: no colonies on any plates. Cm works</LI></UL><H2><A id="sept17">September 17</A></H2><P>Objective: PCR to assemble decoy binding site array</P><UL><LI>MZ does PCR on 4 tubes: 0 um, 0.5 um, 1 um, and 2 um of ~200 ng/uL prefix oligo respectively + 50 uL standard mix (Q5 buffer and polymerase, repeat-spacer-repeat and spacer-repeat-spacer oligos, dNTPS, H2O) </LI><LI>PCR for 75 min</LI><LI>Gel to confirm (picture)</LI><LI>From left to right, 0, 0.5, 1, 2 uL of prefix oligo. Shortening of schmear is consistent with intended effect but not conclusive evidence thereof.</LI><LI>CBC takes 0.3 uL out of wells 0.5, 1, 2 to serve as template for new PCR2 with prefix &amp; suffix oligos. MCF runs the gel</LI></UL><H2><A id="sept18">September 18</A></H2><P>Objective: PCR to assemble decoy binding site array</P><UL><LI>CBC sets up 3 new PCR series, each with 0.3, 0.6, 0.9, 1.2 uL of tubes 2, 3, 4 from previous reaction as template. </LI><LI>Left to right is series from reactions templated with tube 2, 3, and 4. Gels look more like small pieces are dominating (primer dimer??) but we will try gibson anyway</LI></UL><P>Objective: PCR to assemble decoy binding site array</P><UL><LI>MZ with CBC get Mert’s homemade 1.33x gibson assembly mix and make reactions with backbone only, then backbone + insert</LI><LI>1uL of (conc) backbone (=EcoRI/PstI/CIP pSB1C3-R0040 made by CBC on [date] conc = [conc]) and 0.3uL of PCR products from series 2, 3, 4 at concs [conc conc conc] respectively.</LI><LI>Next morning: 100s of colonies.</LI></UL><P>Objective: PCR to assemble decoy binding site array</P><UL><LI>Master mix X13:  (of a standard 50 uL reaction)</LI></UL><UL><LI>494 uL dH2O</LI><LI>130 uL buffer (Q5)</LI><LI>13 uL dNTPs</LI><LI>3.25 uL each oligo (prefix &amp; suffix oligos)</LI><LI>6.25 uL enzyme (Q5)</LI><LI>0.3 uL of each template (0.5, 1, 2)</LI></UL><UL><LI>Ran PCR with iGEM protocol (previously saved)</LI><LI>Some uneven loading in gel or at addition of template to tube step; either way it’s fine. Pool with PCR cleanup protocol and then can use this for GA. 0.5 and 1.0 series look the most promising for this. </LI></UL><H2><A id="sept19">September 19</A></H2><P>Objective: confirm insert presence &amp; size of decoy array inserts from gibson assembly yesterday</P><UL><LI>100s of colonies on bb+insert plates 2, 3, 4. None on insert-only plates or backbone-only. </LI><LI>Colony PCR with pSB1C3-up and -dn with 62C annealing temp (see 7/25/2014 entry, Farnitano figured this temp out for these oligos)</LI><LI>1ng pSB1C3-R0040, no template control, then 4 colonies from plate 2, 4 from 3, 6 from 4. </LI><LI>No amplification on anything. No picture</LI></UL><P>Objective: assemble decoy arrays. Insert putative arrays in to pSB1C3</P><UL><LI>Delta pools reactions from Garima’s PCR/gel produced yesterday in to three tubes. CBC sets up gibson assembly. 1 uL backbone and 0.3 insert as appropriate, H2O to 5 uL, vortexed briefly then added 2.5uL of this to 7.5 uL of master mix. Only enough master mix was in the freezer for 6 of these half-sized reactions so there is no insert-only control for reactions labeled 2 in Garima’s gel.</LI><LI>Transforming all 10 uL in to 80uL of competent Z1s. </LI><LI>Next morning, lots of colonies on all the places there should be colonies. Huzzah</LI></UL><H2><A id="sept21">September 21</A></H2><P>Objective: confirm insert presence &amp; size of decoy array inserts from gibson assembly </P><UL><LI>Picked 18 colonies from gibson/transformation performed on 9/19/14 yesterday and miniprepped. EcoRI/PstI digest for 3h then run 15m on 0.8% gel</LI></UL><UL><LI>Unadorned pSB1C3 is 41bp between EcoRI and PstI, 107 for minimal intended length of the four assembly oligos. It looks like there might be some incomplete digestion happening on these maybe, or just overloaded. </LI><LI>taaggatgatttctgg aattcgcggccgcttctagagCCATCTAATTCAACAAGAATTGGtgatgttaatCCATCTAATTCAACAAGAATTGGtgatgttaattactagtagcggccgctgca gtccggcaaaaaagggc</LI></UL><UL><LI>Left to right is 1-18. Two probably aberrant clones but they’re getting sequenced anyway. Sent to bio sequencing core seqd from pSB1C3-up-1 and -dn-1. Clones 1-4, 15-18. </LI></UL><H2><A id="sept22">September 22</A></H2><P>Objective: confirm insert presence &amp; size of decoy array inserts from gibson assembly </P><UL><LI>Mike picks 12 more colonies for miniprepping tomorrow</LI></UL><H2><A id="sept23">September 23</A></H2><P>Objective: confirm insert presence &amp; size of decoy array inserts from gibson assembly </P><UL><LI>Mike starts miniprep for 12 new colonies, charlie takes over after N3 addition &amp; spin. Mike sets up EcoRI-HF/PstI-HF digest (cutsmart) for each clone. </LI><LI>Garima loaded this 1.3% TAE gel. Lane 7 (ie clone 25) has one big band at ~600 but also looks to have a small band down at the 1x size. Contamination? Unstable repeats? Another gel 0.8% has the same thing, but smearier. </LI><LI>Sequencing results come back!! As expected and consistent with the gel above, almost all colonies have a single insert, but clone 17 has 5! It works, kind of!!! The chromatograms taper off in quality around the repeats which makes it difficult for seqman to put it assemble properly, but manually getting called bases and finding repeats makes it clear. Clone 2 from yesterday was trash. </LI></UL><UL><LI>ctrl-F in text editor highlights each decoy site. To the left and right are prefix &amp; suffixes</LI></UL><P>Objective: transfer decoy arrays to pSB1A2</P><UL><LI>CBC sets up prep scale digest of EcoRI/PstI of clones 17 &amp; 18 &amp; pSB1A2-R0040-antitracr. PCR cleanup and CIP treatment of clones 17 &amp; 18 followed by a second PCR cleanup. Gel extraction of pSB1A2 backbone leaves only 22 ng/uL of dirty DNA by nanodrop. Tomorrow will try ligating, and also will set up another EcoRI/PstI digest of more pSB1A2-R0040-antitracr</LI></UL><P>Objective: optimize colony PCR conditions for testing pSB1C3-decoy arrays</P><UL><LI>Garima sets up master mix with 9x50uL rx. pSB1C3-up-1 and -dn-1 included at 0.25*9uL from 100uM stock. </LI></UL><UL><LI>45 uL Taq buffer</LI><LI>2.25 uL Taq polymerase</LI><LI>9 uL dNTPs</LI><LI>9 uL of each pSB1C3-up-1 and -dn-1</LI><LI>375.75 uL dH2O</LI><LI>9 uL template in positive tube/ 0 uL template in negative tubes</LI></UL><UL><LI>Ran colony pcr using saved protocol:</LI></UL><UL><LI>edited annealing step of protocol from 62 degrees to a gradient : 58-65 degrees</LI><LI>extension time of 15s (was 30)</LI><LI>Fairly crappy snapshot (crapshot?) but you can see that the primer dimers on the - gel are all at ~100bp and the product on +template are at 2-300 which is consistent with R0040-antitracr size. the best amplification took place at higher temps. New protocol: 65C with 15s extension for short pieces, next try with some actual colonies.</LI></UL><H2><A id="sept24">September 24</A></H2><P>Objective: transfer decoy arrays to pSB1A2</P><UL><LI>Setting up ligation reaction with EcoRI/PstI/CIP treated clone 17 and 18 from yesterday (5x and 1x decoys respectively) and EcoRI/PstI/gel-purified pSB1A2 bb. Also including controls below</LI><LI>used ligation calculator from dusseldorf </LI></UL><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TD><P>17 (5x)</P></TD><TD><P>18 (1x)</P></TD><TD><P>1A2 bb</P></TD></TR><TR><TD><P>length</P></TD><TD><P>~500</P></TD><TD><P>~100</P></TD><TD><P>2252</P></TD></TR><TR><TD><P>conc</P></TD><TD><P>84.1</P></TD><TD><P>63.8</P></TD><TD><P>22.8</P></TD></TR><TR><TD><P>ng needed</P></TD><TD><P>66</P></TD><TD><P>13</P></TD><TD><P>100</P></TD></TR><TR><TD><P>using uL</P></TD><TD><P>1</P></TD><TD><P>0.5</P></TD><TD><P>5</P></TD></TR></TBODY></TABLE><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TD><P>thing</P></TD><TD><P>bb+ins 17</P></TD><TD><P>insert only 17</P></TD><TD><P>bb+ins 17</P></TD><TD><P>insert only 17</P></TD><TD><P>bb only</P></TD></TR><TR><TD><P>backbone</P></TD><TD><P>5uL</P></TD><TD><P>0</P></TD><TD><P>5</P></TD><TD><P>0</P></TD><TD><P>5</P></TD></TR><TR><TD><P>insert</P></TD><TD><P>1uL</P></TD><TD><P>1</P></TD><TD><P>0.5</P></TD><TD><P>0.5</P></TD><TD><P>0</P></TD></TR><TR><TD><P>T4 buffer</P></TD><TD><P>2uL</P></TD><TD><P>2</P></TD><TD><P>2</P></TD><TD><P>2</P></TD><TD><P>2</P></TD></TR><TR><TD><P>enzyme</P></TD><TD><P>1uL</P></TD><TD><P>1</P></TD><TD><P>1</P></TD><TD><P>1</P></TD><TD><P>1</P></TD></TR><TR><TD><P>h2o</P></TD><TD><P>11uL</P></TD><TD><P>16</P></TD><TD><P>11.5</P></TD><TD><P>16.5</P></TD><TD><P>12</P></TD></TR></TBODY></TABLE><UL><LI>Mixed andl et sit for 3h at room temp on bench. Took 10uL and transformed with 80uL of frozen competent Z1s</LI><LI>next day: lots of colonies on bb+insert plates, none on bb-only or insert-only. huzzah</LI></UL><P>Objective: find more long decoy arrays</P><UL><LI>miniprepping 12 more colonies from plate 1.0, middle one from 9/18/14</LI><LI>EcoRI/PstI digest for 3h gives all small bands at 1x size. No picture taken, all tubes discarded. </LI></UL><H2><A id="sept25">September 25</A></H2><P>Objective: find more long decoy arrays</P><UL><LI>colony PCR on 4 colonies from plate 1.0 next to no-template, 1ng of pSB1C3-R0040-antitracr, pSB1C3-5xGFP1 decoy, and pSB1C3-1xGFP1 decoy. 10s extension at 64C</LI><LI><SPAN style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 468.00px;"> No amplification on any of the colonies?</SPAN></LI></UL><UL><LI>no template, pSB1C3-R0040-antitracr, 5x, 1x, then 4 colonies. At first I thought this was no amplification but actually I think it worked but all of the colonies are 1x or maybe a 2x for colony 4. Threw them all away in any case and before I had the realization that they amplified I already set up the next PCR with 8 colonies on a 58-65C gradient. </LI></UL><UL><LI>left to right is 65 to 58. My strong suspicion is that the long band in lane 1 is from a long template rather than 65. Picking colonies 1 and 3 for further investigation. Labeled 43 and 44 on the tubes, because that’s where we are in the screening.</LI></UL><P>Objective: confirm insert presence &amp; size of decoy array inserts from gibson assembly </P><UL><LI>sequencing clone 25 with pSB1C3-up and -dn, dugsim job 1827</LI></UL><P>It might be better to take the 5x repeat and then expand it with some other scheme. </P><UL><LI>It is possible to do this by stepwise dig/lig/confirm cycles with XbaI/PstI dig’d insert and SpeI/PstI dig’d backbone (gel purified). </LI><LI>Another possible trick to speed up the expansion would be to cycle from 16C to 37C a couple of times with XbaI/SpeI dig’d pSB1C3-5xGFP1 and also add XbaI/SpeI &amp; T4 ligase and a third enzyme that cuts somewhere else in the backbone. </LI></UL><UL><LI>When XbaI and SpeI ligate it will destroy the recognition site. Any sites re-ligating to self will be cleaved in the next 37C cycle. The last step in the cycling reaction will be a long digest.</LI><LI>Follow that with a ligation to XbaI/SpeI/CIP treated backbone. </LI><LI>Circularization is possible with this scheme</LI></UL><UL><LI>Third possibility: cycle XbaI/PstI/CIPd insert with backbone, SpeI, PstI, ligase and cycle</LI></UL><UL><LI>XbaI ligs to SpeI and destroys site. XbaI ligs to XbaI and gets re-cut, same with PstI. On re-cutting this would restore the phosphate removed by CIP so that doesn’t help much. Hmm</LI></UL><UL><LI>…..or just the stepwise dig/lig/shock will work. I think the problems of circularization will be there no matter what I do with CIP &amp; including restriction enzymes the ligase.</LI></UL><H2><A id="sept26">September 26</A></H2><P>Objective: Confirm pSB1A2-GFP1 decoy arrays</P><UL><LI>Yesterday picked 4 colonies from transformation of 5x and 1x repeat arrays and set up with EcoRI/PstI digest, alongside colonies 43 and 44 from yesterday’s colony PCR and pSB1C3 5x and 1x parents. </LI><LI>Slightly hard to read on the picture, but they’re loaded in the order mentioned above. 43 might be longer than 5x, 44 looks like 1x so discarding. </LI><LI>Zoom in on the 5x on 1A2 and Colony 1 might be slightly shorter than colony 4? Making bacteria with both of these just in case. Should probably sequence as well. pSB1C3-up-1 binds to 1A2, but not pSB1C3-dn</LI><LI>Shocking Z1 to make a couple of strains:</LI></UL><UL><LI>pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-5xGFPdecoy-1</LI><LI>pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-5xGFPdecoy-4</LI><LI>pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-1xGFPdecoy-1</LI><LI>pdCas9, pSB4K5-K(gfp), pSB1A2-5xGFPdecoy-1</LI><LI>pdCas9, pSB4K5-K(gfp), pSB1A2-5xGFPdecoy-4</LI><LI>pdCas9, pSB4K5-K(gfp), pSB1A2-1xGFPdecoy-1</LI></UL><H2><A id="sept27">September 27</A></H2><P>Objective: extend decoy arrays</P><UL><LI>XbaI/PstI digest of 1C3-5xGFP1decoy followed by CIP, then SpeI/PstI of 1C3-5xGFP1 (same plasmid) with NO GEL PURIFICATION makes the backbone. I am hoping that since SpeI to PstI is only 18bp that the PCR cleanup kit gets rid of the fragment. Ligating with 3-fold molar excess of CIP-treated insert for 20m then shocking 10uL in to 70uL Z1. This is rushing the protocol; the other 10uL of the ligation mix will sit at RT for 90m in case the rushed version fails. </LI></UL><UL><LI>Update: it looks like it did not fail; there are some colonies on bb-only and ins-only but ~10x as many on b+i.</LI></UL><H2><A id="sept28">September 28</A></H2><P>Objective: test 1x and 5x decoy arrays</P><UL><LI>4 colonies each picked from plates from</LI></UL><UL><LI>pdCas9-GFP1, pSB4K5-K608012, pSB1A2-5xGFPdecoy-1 G51</LI><LI>pdCas9-GFP1, pSB4K5-K608012, pSB1A2-5xGFPdecoy-4 G54</LI><LI>pdCas9-GFP1, pSB4K5-K608012, pSB1A2-1xGFPdecoy-1 G11</LI><LI>pdCas9, pSB4K5-K608012, pSB1A2-5xGFPdecoy-1 C51</LI><LI>pdCas9, pSB4K5-K608012, pSB1A2-5xGFPdecoy-4 C54</LI><LI>pdCas9, pSB4K5-K608012, pSB1A2-1xGFPdecoy-1 C11</LI></UL><UL><LI>….and grown overnight (started 9/27/14). 9:15am today diluted 1/200 in 3 mL fresh LB+AmpKanCm. 3h45m in 37C shaker. 1mL collected, spun down, and resuspended in PBS. Flowed 75000 events and got something that looks super promising. Means here are written down from stats view on MQ during acquisition and might change after full processing. There is a lot of variability in G51</LI></UL><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TD><P>2014-09-28 flow data, not fully processed</P></TD></TR><TR><TD><P>(constitutive GFP reporter)</P></TD><TD><P>pdCas9 with or without GFP-targeting crRNA</P></TD><TD><P>decoy binding site count &amp; colony</P></TD><TD><P>expected GFP if working</P></TD><TD><P>tube label</P></TD><TD><P>unprocessed mean gfp</P></TD><TD><P>avg</P></TD><TD><P>std</P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-5xGFP1decoy-1</P></TD><TD><P>derepressed</P></TD><TD><P>G51-1</P></TD><TD><P>42</P></TD><TD><P>69.25</P></TD><TD><P>28.14</P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-5xGFP1decoy-1</P></TD><TD><P>derepressed</P></TD><TD><P>G51-2</P></TD><TD><P>50</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-5xGFP1decoy-1</P></TD><TD><P>derepressed</P></TD><TD><P>G51-3</P></TD><TD><P>83</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-5xGFP1decoy-1</P></TD><TD><P>derepressed</P></TD><TD><P>G51-4</P></TD><TD><P>102</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-5xGFP1decoy-4</P></TD><TD><P>derepressed</P></TD><TD><P>G54-1</P></TD><TD><P>64</P></TD><TD><P>68.33</P></TD><TD><P>5.13</P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-5xGFP1decoy-4</P></TD><TD><P>derepressed</P></TD><TD><P>G54-2</P></TD><TD><P>67</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-5xGFP1decoy-4</P></TD><TD><P>derepressed</P></TD><TD><P>G54-3</P></TD><TD><P>nan</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-5xGFP1decoy-4</P></TD><TD><P>derepressed</P></TD><TD><P>G54-4</P></TD><TD><P>74</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-1xGFP1decoy-1</P></TD><TD><P>near repressed levels</P></TD><TD><P>G11-1</P></TD><TD><P>16</P></TD><TD><P>16.00</P></TD><TD><P>0.82</P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-1xGFP1decoy-1</P></TD><TD><P>near repressed levels</P></TD><TD><P>G11-2</P></TD><TD><P>15</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-1xGFP1decoy-1</P></TD><TD><P>near repressed levels</P></TD><TD><P>G11-3</P></TD><TD><P>17</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9-GFP1</P></TD><TD><P>pSB1A2-1xGFP1decoy-1</P></TD><TD><P>near repressed levels</P></TD><TD><P>G11-4</P></TD><TD><P>16</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-5xGFP1decoy-1</P></TD><TD><P>unrepressed</P></TD><TD><P>C51-1</P></TD><TD><P>131</P></TD><TD><P>141.75</P></TD><TD><P>7.46</P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-5xGFP1decoy-1</P></TD><TD><P>unrepressed</P></TD><TD><P>C51-2</P></TD><TD><P>143</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-5xGFP1decoy-1</P></TD><TD><P>unrepressed</P></TD><TD><P>C51-3</P></TD><TD><P>145</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-5xGFP1decoy-1</P></TD><TD><P>unrepressed</P></TD><TD><P>C51-4</P></TD><TD><P>148</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-5xGFP1decoy-4</P></TD><TD><P>unrepressed</P></TD><TD><P>C54-1</P></TD><TD><P>153</P></TD><TD><P>149.00</P></TD><TD><P>3.16</P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-5xGFP1decoy-4</P></TD><TD><P>unrepressed</P></TD><TD><P>C54-2</P></TD><TD><P>150</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-5xGFP1decoy-4</P></TD><TD><P>unrepressed</P></TD><TD><P>C54-3</P></TD><TD><P>146</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-5xGFP1decoy-4</P></TD><TD><P>unrepressed</P></TD><TD><P>C54-4</P></TD><TD><P>147</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-1xGFP1decoy-1</P></TD><TD><P>unrepressed</P></TD><TD><P>C11-1</P></TD><TD><P>150</P></TD><TD><P>153.25</P></TD><TD><P>3.95</P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-1xGFP1decoy-1</P></TD><TD><P>unrepressed</P></TD><TD><P>C11-2</P></TD><TD><P>152</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-1xGFP1decoy-1</P></TD><TD><P>unrepressed</P></TD><TD><P>C11-3</P></TD><TD><P>159</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR><TR><TD><P>pSB4K5-K608012</P></TD><TD><P>pdCas9</P></TD><TD><P>pSB1A2-1xGFP1decoy-1</P></TD><TD><P>unrepressed</P></TD><TD><P>C11-4</P></TD><TD><P>152</P></TD><TD><P> </P></TD><TD><P> </P></TD></TR></TBODY></TABLE><UL><LI>Saved G51-1 through -4, then colony 1 for one of each different strain as glycerol stocks. Labeled very minimally with the bolded labels in first sub-bullet list. </LI><LI>Next steps:</LI></UL><UL><LI>repeat</LI><LI>try with more decoy site numbers: 10x is on the way, need to also make 0x</LI></UL><H2><A id="”sept30”">September 30 </A></H2><P>Objective: sequence confirm pSB1A2-5xGFP1</P><UL><LI>Sequenced with pSB1C3-up-1. They might actually have 6 repeats; it’s hard to say for sure but it looks more like 6 than 5. The start of the well-defined seq results is pretty much right where the repeat array butts up against the prefix. The same problem applies when sequencing 1C3 backbones too.</LI><LI>Sequenced decoy array clones 2 and 43. 25 looks most like a 1x, 43 like another 6x.</LI></UL><UL><LI>Toss 25, transfer 43 to 1A2 just in case, and design oligos for 1C3 that bind farther out from prefix/suffix</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Duke/Notebook/September">http://2014.igem.org/Team:Duke/Notebook/September</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Duke/Notebook/September" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Duke/Notebook/September" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Duke/Notebook/September&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Duke/Notebook/September&amp;oldid=303960" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>