"
Page
Discussion
View source
History
teams
Log in
 
Team:Duke/Notebook/July
From 2014.igem.org
Home
Team
About Us
Official Team Profile
Photo Gallery
Attributions
Project
Background
Summary
Modeling
Results
Future Directions
Parts
Notebook
Overview
Notebook by Month
Protocols
Policy and Practice
3D Printing
House Course
NCSSM Team
Safety
July 1
Objective: make LB+Cm plates
Four sleeves of plates with 4x500mL medium
34mg/mL chloramphenicol
Objective: insert crRNAs into pdCas9 and scaffold
Plate results
Experimental had lots of colonies, but backbone-only control did as well
reasoning: similarity of BsaI overhangs may lead to re-ligation
solution: treat with CIP before ligation
Agarose gel of BsaI cuts of pdCas9 and Repeat scaffold
plasmids used in 6/30/14 transformation
to make sure BsaI is cutting properly
Results: single band in both appears to be cut DNA
CIP treatment of pdCas9 and Repeat scaffold (both BsaI cut)
pdCas9: 10uL plasmid in 20 uL reaction with 1.5uL CIP
Repeat scaffold: 22uL plasmid in 30 uL reaction with 2 uL CIP
2 hours at 37C (longer than standard protocol)
PCR cleanup of pdCas9 and Repeat scaffold CIP treatments
Final concentrations too low to be of use
Particularly negligible in pdCas9
Need to culture and cut more plasmid to try ligation again
Spread plates from pdCas9 and Repeat scaffold frozen stocks
Used tube “1” for repeat scaffold
Plated on fresh LB+Cm plates
Plates still slightly wet, difficult to spread evenly
Objective: make CCEC stock of DH5alpha-ZI strain
Diluted back 5mL culture in morning
1mL into 5mL LB+Spec
note for future: Spec not necessary for most procedures, chromosomally integrated genes are more stable
Inoculated 1 mL culture into each of two 250mL flasks
Flasks contain SOC
30C shaker for overnight growing
July 2
Objective: Make CCEC stock of DH5alpha-ZI strain
Made fresh CCMB-80 buffer stock
precipitate formed during pH adjusting and may have filtered out
could be detrimental to final product
Followed iGEM online protocol to make CCEC from Z1 strain
Overnight culture was much more concentrated than recommended
OD >1
Did not have enough SOC to dilute back fully, so diluted back only a bit
Had to vortex to resuspend cells at two stages of procedure
Final OD after procedure ~1.6 for 100mL of cells
Aliquoted 1mL into tubes, froze at -80C
Next steps:
Test competence with useful strains containing Lac, Tet promoters
Objective: insert crRNAs into pdCas9 and Repeat-scaffold
Culture colonies from plates of pdCas9 and pSB1C3-Repeat-scaffold
3 colonies per plate in LB+Cm medium at 37C overnight
Plates were messy--hard to get individual colonies
July 3
Objective: insert crRNAs into pdCas9 and Repeat-scaffold
Miniprep cultures from 7/2
Three tubes of pdCas9 with concentrations 143.4, 145.7, 151.4 ng/uL
Three tubes of Repeat-scaffold with concentrations 166.6, 145.8, 159.6 ng/uL
Digest tubes 1 and 2 of pdCas9 and tubes 1 and 2 of Repeat-scaffold
to insert crRNAs
Digest with BsaI in cutsmart buffer
50 uL DNA in 100 uL reactions with 7.5 uL BsaI
3 hours 20 min at 37C
PCR cleanup of BsaI digest
final concentrations (in 30 uL each):
pdCas9: 65.4, 70.5 ng/uL
Repeat-scaffold: 122.3, 114.8 ng/uL
CIP treatment of pdCas9 and Repeat-scaffold BsaI digests
Two digest tubes pooled into one CIP tube for each backbone
60 uL DNA in 100 uL reactions with 5 uL CIP in cutsmart buffer
1.5 hours at 37C
PCR cleanup of CIP treatment
final concentrations (in 30 uL):
pdCas9: 32.5 ng/uL
Repeat scaffold 129.9 ng/uL
Annealed oligos of crRNA inserts for ligation
3 crRNAs (GFP1, GFP2, GFP3) with 2 oligos each
5uL each oligo heated to ~95C, slowly brought to rt in water bath
Ligation of crRNA inserts into pdCas9 and Repeat-scaffold
6 working tubes + 5 controls
pdCas9 + each crRNA (GFP1, GFP2, or GFP3)
Repeat-scaffold + each crRNA (GFP1, GFP2, or GFP3)
pdCas9 Backbone only control
Repeat-scaffold Backbone only control
Insert only controls for each insert
100ng = 3uL pdCas9, BsaI digested and CIP treated and cleaned
100ng = 0.8uL Repeat-scaffold, BsaI digested and CIP treated and cleaned
1uL annealed oligo insert
10 uL reactions with 0.5 uL T4 Ligase
Transformed into DH5alpha and plated on LB+Cm
Objective: Test competency and effectiveness of DH5alpha-Z1 strain
CCEC Transformation efficiency test
Transformed 5 tubes of pSB1C3-K741002 into DH5alpha-Z1 CCEC
Total amount of DNA in 10 uL: 130ng, 13ng, 1.3ng, 0.13ng, and 0.013ng
Also for use in testing pLac regulation in Z1 strain
Transformed iGEM kit plate 3-6G into DH5alpha-Z1
contains pSB1C3-BBa_I13521: pTet-RBS-mRFP
in order to test regulation of pTet in Z1 strain
July 4
Objective: ligate crRNAs into pdCas9 and Repeat-scaffold
Plate results for ligation of crRNA GFP1, GFP2, and GFP3 into two backbones:
~1000 colonies on all experimental plates
BO control for pdCas9 has ~500 colonies
BO control for Repeat scaffold has ~1000 colonies
IO controls all have 0 colonies
Cultured 4 copies of each crRNA-GFP1,2,3 in pdCas9 backbone
12 tubes total in LB+Cm
None from Repeat-scaffold attempts
Objective: Test competency and effectiveness of DH5alpha-Z1 strain
CCEC Test plate results:
130 ng: 100s of colonies
13 ng:~100 colonies
1.3 ng: ~20 colonies
0.13 ng: 0 colonies
0.013 ng: 0 colonies
Competency ~10 colonies/ng (low but working)
Results of transformation of pSB1C3-BBa_I13521 into DH5alpha-Z1
Plate had ~5 colonies
Cultured colonies for Tet and Lac testing in DH5alpha-Z1
One colony of pSB1C3-K741002
One colony of pSB1C3-BBa_I13521
In LB+Cm
July 5
Objective: Insert crRNAs into pdCas9 backbone
Saved stocks of 12 strains in 15% glycerol at -80C
4 tubes each of potential GFP1,2,3 crRNA inserts in pdCas9
Miniprepped 12 tubes of pdCas9 with potential GFP-1,2,3 inserts
4 tubes each
Eluted in 50 uL EB, all concentrations 100-200 ng/uL
Objective: Test effectiveness of DH5alpha-Z1 strain
Saved stock of pSB1C3-K741002 in DH5alpha (15% glycerol at -80C)
Culture of I13521 did not grow
Cultured another colony of pSB1C3-BBa_I13521
from 7/3 plate
July 6
Objective: Test effectiveness of DH5alpha-Z1 strain
Saved stock of pSB1C3-I13521 in DH5alpha (15% glycerol at -80C)
July 7
Objective: insert crRNAs GFP1,2,3 into pdCas9
Analytical restriction digest
12 tubes with potential inserts (4 per crRNA)
pdCas9 as a control
BsaI and XbaI in cutsmart for 1.5 hrs
Agarose gel of restriction digest
Gel 1:
1-4: pdCas9-crRNA-GFP1
5-8: pdCas9-crRNA-GFP2
9-12: pdCas9-crRNA-GFP3
Gel 2:
pdCas9 (control)
Expected single band ~9.3 kb if BsaI cut and ligation worked
Control and uncut failures expect two bands, 5.0 and 4.2 kb
Results: all 12 samples have single band
No uncut plasmids
Could still be recircularized with no insert
gel pictures:
Anneal oligos of 3 crRNA-GFP inserts
Following Dewran’s crRNA protocol
2 uL each oligo diluted in PBS to 20 uL
Placed in 98C water bath, cooled gradually to rt
Phosphorylate oligo inserts of crRNAs
5 uL annealed oligo mix in 20uL 1x ligase buffer with 1uL PNK
30 mins at 37C, then deactivated for 20 mins at 65C
Diluted 1/50
Ligation
75 ng backbone = 0.6 uL Repeat scaffold or 2.5 uL pdCas9
Both BsaI cut and CIP treated 7/3/14
1.5 uL diluted, phosphorylated, annealed oligo insert
6 experimental tubes + 2 BO controls and 3 IO controls
Transformed into iGEM CCEC (DH5alpha) and plated on LB+Cm
Cultured 2 colonies from pdCas9 plate
For Charlie to try his hands at ligation
Objective: Prepare various backbones for multiple-plasmid use
Transformed two distribution kit parts
Plate 4-6H: pSB4K5-J04450
backbone contains pSC101 origin with Kan resistance
Plate 4-18A: pSB3K3-I20260
backbone contains p15A origin with Kan resistance
On LB+G418 plates
Streaked plate from frozen stock
pSB6A1-J04450
backbone contains pMB1 origin with Amp resistance
on LB+Amp
Objective: Test pTet and pLac in DH5alphaZ1 strain
Streaked plates from frozen stock
pSB1C3-K741002 in DH5alpha-Z1
contains pLac-GFP
pSB1C3-I13521 in DH5alpha-ZI
contains pTet-RFP
On LB+Cm plates
July 8
Objective: insert crRNAs into pdCas9
Plate results from 7/7 transformation
pdCas9-crRNAs: ~50 colonies each
pdCas9 BO: 100s of colonies
Repeat-crRNAs: 100s of colonies
Repeat BO: ~1000 colonies
IO controls: 0 colonies, ~15 for crRNA-GFP3
Notes: unsure why BO controls have more colonies than experimental
Restriction digest of 12 samples from 7/5 prep + pdCas9 control
With SacI/NheI
Should show difference between insert and recircularized backbone
Made and ran 1.5% agarose gel in TBE
for better resolution of small band differences in pdCas9 with/without insert
1: pdCas9 (control)
2-5: pdCas9-crRNA-GFP1 (1-4)
6-9: pdCas9-crRNA-GFP2 (1-4)
10-13: pdCas9-crRNA-GFP3 (1-4)
14: pdCas9 (control)
expected:
3.1, 2.8, and 2.7 kb bands
585 bp band with no insert, 620 bp band with insert (pdCas9 620bp)
Results: One sample, crRNA-GFP3-3, has larger band matching control
This colony may be correct, should be sequenced
Others have smaller band, are probably incorrect
Culture 40 colonies from existing plates to screen for crRNA inserts
10 each from crRNA-GFP1 and crRNA-GFP2 transformed 7/3/14
10 each from crRNA-GFP1 and crRNA-GFP2 transformed 7/7/14
in LB+Cm
Objective: Prepare various backbones for multiple-plasmid use
Prepared LB+Kan medium
50 ug/mL final concentration Kanamycin
Cultured colonies from transformations and stock streaking
3 colonies each from 3 plasmids:
pSB6A1-J04450 in LB+Amp
pSB3K3-I20260 in LB+Kan
pSB4K5-J04450 in LB+Kan
Objective: Test pLac and pTet in DH5alpha-Z1
Notes:
Need to obtain aTc for pTet testing
Tried to use Doxycycline, but had trouble with solubility in ethanol (?)
pLac is not the final promoter to be used, a different variant will be used
with no glucose dependency
Flow testing of pLac and pTet was suspended until these changes are made
July 9
Objective: Insert crRNAs into pdCas9
Miniprep 40 cultures of potential crRNA inserts in pdCas9
Labeled with three numbers
Date of transformation - crRNA-GFPx - sample number
700uL of each culture saved to freeze in glycerol if colonies appear successful
Stored at 4C for the day
Analytical restriction digest of potential crRNA inserts
All 40 tubes + 2 pdCas9 tubes as control
1 uL DNA in 10 uL digest for 2 hrs at 37C
SacI-HF and NheI-HF in cutsmart
Made and ran 1.5% agarose/TBE gels
To differentiate between small band differences
Gel 1:
1. 2-log ladder
2. pdCas9
3-12. pdCas9-crRNA-GFP1 from 7/3 transformation
13-22. pdCas9 -crRNA-GFP2 from 7/3 transformation
23. pdCas9
24. 2-log ladder
Gel 2:
1. 2-log ladder
2. pdCas9
3-12. pdCas9-crRNA-GFP1 from 7/7 transformation
13-22. pdCas9 -crRNA-GFP2 from 7/7 transformation
23. pdCas9
24. 2-log ladder
Expected results:
pdCas9 and successful inserts:
3.1, 2.8, and 2.7 kb bands, plus 620 bp insert band
unsuccessful recircularizations:
3.1, 2.8, and 2.7 kb bands, plus 585 bp insert band
Results:
Gel 1 appears to only be recircularized
Gel 2 has some promising inserts:
GFP1 samples 1,2, and 3 from 7/7 transformation
GFP2 samples 7,8, and 9 from 7/7 transformation
Glycerol stocks frozen for these six samples
Next Steps:
Run digest with BsaI to confirm that plasmids are not uncut pdCas9
Sequence to confirm presence of insert (using oligos ordered 7/8/14)
Objective: Obtain new backbones and switch inserts into new backbones
Miniprep 3 new backbones
pSB6A1-J04450 (3 copies)
concentrations 437.6, 483.9, and 450.1 ng/uL
pSB3K3-I20260 (3 copies)
concentrations 114.6, 120.0, and 97.3 ng/uL
pSB4K5-J04450 (3 copies)
concentrations 129.3, 165.9, and 155.1 ng/uL
Prep-scale digest of all 9 backbone tubes
50 uL DNA in 60 uL total reaction
EcoRI-HF/Spe-HF in cutsmart
3 hours at 37C
Agarose gel extraction of 3 backbones
Gel 1, Left: pSB6A1
Gel 1, Right: pSB3K3
Gel 2, Left: pSB4K5
All 3 lanes had 2 bands as expected
upper band (backbone) in 3K3 had messy trail behind it (not extracted)
All 180 uL from three tubes combined with 20 uL loading dye in x-large gel well
top band extracted from all 3 plasmids, split into 2 tubes each, stored at -20C
Streak plate from frozen stock
pSB1C3-K608012
In order to switch into pSB6A1
Objective: Remake new version of pDGC3
Transformation from iGEM distribution kit plates
Plate 2-6D: pSB1C3-BBa_R0011
Engineered Lac promoter (not affected by glucose)
Plate 3-15O: pSB1C3-BBa_I13500
Strong RBS-GFP
Plated on LB+Cm
July 10
Objective: Obtain backbones and switch parts into new backbones
Gel cleanup of 3 backbones with Zymoclean prep kit
Some tubes had to be divided into two samples for cleanup
Final elution was 20 uL each into 2-3 tubes per backbone
Nanodrops looked unclean: Most had high peak around 220-240 nm
pSB6A1: 300mg + 120 mg gel = 230.4 ng/uL
Peak ~220, then hump ~260
pSB6A1: 270 mg gel = 220 ng/uL
Peak, then hump as before
pSB3K3: 320 mg gel = 53.9 ng/uL
No clear sign of DNA in graph
pSB3K3: 230 mg gel = 20 ng/uL
No DNA
pSB3K3: 130 mg gel = 72 ng/uL
Possibly DNA, best option but may not be useful
pSB4K5: 290 mg gel = 43.4 ng/uL
No clear sign of DNA in graph
pSB4K5: 320 mg gel = 117.3 ng/uL
Possibly DNA, best option but may not be useful
Culture 3 colonies of pSB1C3-K608012
In order to switch into pSB6A1
Objective: Insert crRNAs into pdCas9
Analytical Digest of promising crRNA inserts
XbaI/BsaI digest
In order to confirm that plasmids are not uncut pdCas9
Samples 7-1-1,2,3 and 7-2-7,8,9 (6 tubes) plus pdCas9 as control
Agarose gel of digest results:
1. pdCas9
2-4. pdCas9-crRNA-GFP1, samples 1,2,3
5-7. pdCas9-crRNA-GFP2, samples 7,8,9
Expected:
2 bands at 4.2 and 5.1 kb in pdCas9
1 band at 9.3 kb in successful plasmids with inserts
Results:
pdCas9 appears to be an incomplete digest, showing 2 small and 1 large band
Lack of any trace of smaller bands in 6 experimental samples indicates successful inserts.
Objective: Make pDGC3
Culture 3 colonies each from pSB1C3-R0011 and pSB1C3-I13500
Objective: Test effectiveness of DH5alpha-Z1 strain
Mitch and Sargis
Culture pSB1C3-I13521 (pTet-RFP)
For flow
In aTc and non-aTc
July 11
Objective: Switch K608012 into pSB6A1
Miniprep 3 copies of pSB1C3-K608012
concentrations 361, 324, 362 ng/uL
Prep-scale digest of pSB1C3-K608012
2 miniprep tubes, 50 uL DNA in 100 uL each
With EcoRI and SpeI
2+ hrs at 37C
Gel extraction and cleanup to isolate K608012
Cut lower band
Zymoclean prep kit
Each of 2 bands divided into two tubes during cleanup
1A: 280 mg =
1B: 240 mg =
2A: 200 mg =
2B: 200 mg = 62.5 ng/uL (20 uL)
Graphs on nanodrop did not look ideal: peak lower than usual
Objective: Create pSB1C3-R0011-I13500 as destination of mCherry G-block
Miniprep pSB1C3-R0011 and pSB1C3-I13500
3 copies each
Concentrations
pSB1C3-R0011: 181, 229, 194 ng/uL
pSB1C3-I13500: 321.5, 228, 300.4 ng/uL
Prep scale digests of two tubes from each miniprep
pSB1C3-R0011 with SpeI/PstI
no extraction necessary
pSB1C3-I13521 with XbaI/PstI
to extract I13521 insert
100 uL total reaction per tube
2-3 hrs at 37C
Agarose gel of pSB1C3-I13500 XbaI/PstI digests
Intended for gel extraction
Expected 2 bands, but only one present
No extraction performed
Analytical agarose gel of pSB1C3-R0011 SpeI/PstI digests
Single band appears correct
see below for gel picture
PCR cleanup of pSB1C3-R0011 SpeI/PstI digests
concentrations 124.8, 157.3 ng/uL (30 uL each)
Objective: Switch dCas9-tracrRNA into pSB1C3
Miniprep pdCas9
concentration 167.4 ng/uL
Prep-scale digest of pSB1C3-R0011
1 miniprep with XbaI/PstI
To extract pSB1C3 backbone
Agarose gel of pSB1C3-R0011 XbaI/PstI digest
Intended for gel extraction
Expected 2 bands, but only one present
No extraction performed
PCR of dCas9-tracrRNA and anti-tracrRNA
4 tubes each, with pdCas9 as template in 0, 0.4, 0.7, and 1.0 uL quantities
oligos dCas9tracr-up and dCas9tracr-dn for dCas9-tracrRNA PCR
oligos dCas9tracr-up and tracrRNA-dn for anti-tracrRNA PCR
Q5 polymerase, with 64C annealing and 2 min extension
Agarose gel of PCR (and digest) results:
Lanes 1-4: dCas9-tracrRNA PCR (0, 0.4, 0.7, 1.0 uL template)
Lanes 5-8: tracrRNA PCR (0, 0.4, 0.7, 1.0 uL template)
Lanes 9-10: pSB1C3-R0011 SpeI/PstI digests
Results:
dCas9-tracr PCR appears correct, band ~4 kb in 3 lanes
tracrRNA PCR unclear: strong primer-sized band may be correct, faint bands ~4 kb indicate off-target extension.
tracrRNA PCR should be done with shorter extension time
PCR cleanup of dCas9-tracrRNA PCR
Concentration >200 ng/uL
Objective: Test BsaI and XbaI enzymes
Analytical digest of dCas9
One tube with BsaI/EcoRI
One tube with XbaI/EcoRI
XbaI from tube with exp. 5/14
One tube with XbaI/EcoRI
XbaI from tube with exp. 2/15
Agarose gel of digest:
1. BsaI/EcoRI: expected 3 bands at 2.7, 2.9, and 3.5 kb
2. XbaI (5/14) / EcoRI: expected 3 bands at 5.7, 2.1, 1.4 kb
3. XbaI (2/15) / EcoRI: expected 3 bands at 5.7, 2.1, 1.4 kb
Results: Lanes 1 and 3 look as expected. Lane two has one extra band at 3.5 kb corresponding to partial digestion with XbaI. 5/14 XbaI tube is ineffective and has been thrown out
July 14
Objective: insert crRNAs into pdCas9
Sequencing Order #1608
BigDye reaction with BD buffer 1.1
pdCas9-crRNA_GFP1 sample 7-1 w/ pdCas9up1
pdCas9-crRNA_GFP1 sample 7-2 w/ pdCas9up1
pdCas9-crRNA_GFP1 sample 7-3 w/ pdCas9up1
pdCas9-crRNA_GFP2 sample 7-7 w/ pdCas9up1
pdCas9-crRNA_GFP2 sample 7-8 w/ pdCas9up1
pdCas9-crRNA_GFP2 sample 7-9 w/ pdCas9up1
pdCas9-crRNA_GFP3 sample 3 w/ pdCas9up1
pdCas9-crRNA_GFP1 sample 7-1 w/ pdCas9dn1
pdCas9-crRNA_GFP1 sample 7-2 w/ pdCas9dn1
pdCas9-crRNA_GFP1 sample 7-3 w/ pdCas9dn1
pdCas9-crRNA_GFP2 sample 7-7 w/ pdCas9dn1
pdCas9-crRNA_GFP2 sample 7-8 w/ pdCas9dn1
pdCas9-crRNA_GFP2 sample 7-9 w/ pdCas9dn1
pdCas9-crRNA_GFP3 sample 3 w/ pdCas9dn1
Results:
pdCas9-crRNA_GFP1 use sample 7-1
pdCas9-crRNA_GFP2 use sample 7-9
pdCas9-crRNA_GFP3 need to find more colonies
Objective: Create pSB1C3-R0011-I13500 as destination of mCherry G-block
Prep scale digest of pSB1C3-I13500 with XbaI/PstI
Tube 3 from 7/11/14 miniprep
To extract both I13500 and pSB1C3 backbone
100 uL reaction with 50 uL DNA in cutsmart
Agarose gel extraction and cleanup of pSB1C3 and I13500
Top band (~2kb): pSB1C3
Bottom band (~1kb): I13500
Zymoclean prep kit
Concentrations:
pSB1C3: 319 ng/uL in 20 uL (dirty)
I13500: 91.2 ng/uL in 20 uL (dirty)
Ligation of pSB1C3-R0011 and I13500
100ng = 0.7 uL pSB1C3-R0011 (#2)  cut with SpeI/PstI on 7/11
150ng = 1.8 uL I13500 cut with XbaI/PstI on 7/14
Experimental, BO control, and IO control
rt for 1 hr
Heat shock transformation into DH5alpha and plated on LB+Cm
Objective: Switch K608012 into pSB6A1
Ligation of pSB6A1 and K608012
100 ng = 0.5 uL pSB6A1 (“Yellow” sample) cut with EcoRI/SpeI on 7/9
150 ng = 2.5 uL K608012 (#2B) cut with EcoRI/SpeI on 7/11
Experimental, BO control, and IO control
rt for 1 hr
Heat shock transformation into DH5alpha and plated on LB+Amp
Objective: Switch dCas9-tracrRNA into pSB1C3
Prep scale digests
dCas9-tracr PCR with XbaI/PstI (no extraction)
pSB1C3-K608012 (tube 3) with XbaI/PstI (to obtain pSB1C3)
100 uL reaction each with 50 uL DNA in cutsmart
PCR cleanup of dCas9-tracr digest
Qiagen miniprep kit
Concentration 38.1 ng/uL in 30 uL
Agarose gel extraction and cleanup of pSB1C3
gel order: left K608012, right I13500
extracted pSB1C3, top band of K608012 (~2 kb)
Zymoclean kit
Concentration 127.9 ng/uL in 20 uL (dirty)
Ligation of pSB1C3 and dCas9-tracrRNA PCR
100 ng = 3 uL dCas9-tracrRNA PCR cut with XbaI/PstI on 7/14
150 ng = 0.6 uL pSB1C3 (from I13500) cut with XbaI/PstI on 7/14
Experimental, BO control, and IO control
rt for 1 hr
Heat shock transformation into DH5alpha and plated on LB+Cm
Objective: Create pTet-anti-tracrRNA
Prep-scale digest of pSB1C3-R0040
With SpeI/PstI (no extraction necessary)
100 uL reaction with 50 uL DNA in cutsmart
4 hrs at 37C
PCR anti-tracrRNA from pdCas9
Oligos dCas9tracr-up and tracrRNA-dn
0, 0.4, 0.7, and 1.0 uL pdCas9 template in 50 uL reactions
Q5 polymerase
annealed at 64C, 20 sec extension
Agarose gel of anti-tracrRNA PCR
Results: only 1.0 uL template lane appeared to amplify
Need to make more using this template concentration
PCR anti-tracrRNA from pdCas9
Oligos dCas9tracr-up and tracrRNA-dn
1.0 uL pdCas9 template in each 50 uL reaction
8 tubes with Q5 polymerase
annealed at 64C, 20 sec extension
Agarose gel of anti-tracrRNA PCR and pSB1C3-R0040 digest
Lanes 1-8: anti-tracrRNA PCR
Lane 9: R0040 digest
Results:
PCR appears to work in all lanes (band size ~0.1 kb)
R0040 digest single band appears correct
PCR cleanup of anti-tracrRNA PCR and pSB1C3-R0040
Concentrations:
anti-tracrRNA: 83.3 ng/uL in 30 uL
pSB1C3-R0040: 25 ng/uL in 30 uL
July 15
Objective: Create pTet-anti-tracrRNA
Digest of anti-tracrRNA PCR
with XbaI, PstI-HF, and DpnI
3 hrs at 37C
60 uL DNA in 100 uL total reaction
PCR Cleanup of anti-tracrRNA PCR digest and pSB1C3-R0040 digest
Concentrations
anti-tracrRNA: 100 ng/uL
pSB1C3-R0040: 35 ng/uL
Ligation of anti-tracrRNA and pSB1C3-R0040
100 ng = 3 uL pSB1C3-R0040 cut with SpeI/PstI
15 ng = 0.15 uL anti-tracrRNA PCR cut with XbaI/PstI/DpnI
Experimental, Backbone only, and Insert only
Transformed via heat shock into DH5alphas and plated on LB+Cm
Objective: Switch dCas9-tracrRNA into pSB1C3
Plate results:
Experimental: 8 colonies
BO Control: 4 colonies
IO Control: 7 colonies
Culture colonies
4 copies of potential pSB1C3-dCas9-tracrRNA construct
Digest of dCas9-tracrRNA PCR and heat inactivation
Digest with DpnI
22 uL DNA in 30 uL reaction
1 hr at 37C, then 30 mins at 80C to heat-inactivate
assumed concentration 20 ng/uL
PCR Cleanup of pSB1C3 digest
2 tubes of gel-extracted and cleaned pSB1C3 cut with XbaI and PstI
11 ug original yields 309.6 ng/uL in 20 uL “superclean”
Ligation of dCas9-tracrRNA PCR and pSB1C3
150 ng = 0.5 uL pSB1C3 cut with XbaI and PstI
100 ng = 5 uL dCas9-tracrRNA PCR cut with Xbal/PstI/DpnI
Experimental, Backbone only, and Insert only
Transformed via heat shock into DH5alphas and plated on LB+Cm
Objective: Create pSB1C3-R0011-I13500 as destination for mCherry G-block
Plate results:
Lawn of colonies on Experimental and BO control
No colonies on IO control
Digest of pSB1C3-R0011
With SpeI, PstI, and CIP
50 uL DNA in 100 uL reaction
2.5 hrs at 37C
PCR Cleanup of pSB1C3-R0011 digest and I13500 digest
pSB1C3-R0011 concentration 175 ng/uL
I13500 (from previous digestion and extraction)
2.3 ug original yields 124 ng/uL in 20 uL “superclean”
Ligation of I13500 and pSB1C3-R0011
100 ng = 0.6 uL pSB1C3-R0011 cut with SpeI/PstI/CIP
150 ng = 1.2 uL I13500 cut with XbaI/PstI
Experimental, Backbone only, and Insert only
Transformed via heat shock into DH5alphas and plated on LB+Cm
Objective: Switch K608012 into pSB6A1
Plate results:
2 colonies on experimental plate
1 colony on BO, no colonies on IO
Cultured colonies
2 colonies of potential pSB6A1-K608012 construct
PCR Cleanup of pSB6A1 digest and K608012 digest
pSB6A1, 2 tubes gel extracted and cleaned
10 ug original yields 203 ng/uL in 30 uL
K608012, 3 tubes gel extracted and cleaned
4 ug original yields 73.0 ng/uL in 30 uL
Ligation of K608012 and pSB6A1
100 ng = 0.5 uL pSB6A1 cut with EcoRI and SpeI
150 ng = 2 uL K608012 cut with EcoRI and SpeI
Experimental, Backbone only, and Insert only
Transformed via heat shock into DH5alphas and plated on LB+Amp
July 16
Objective: Create pTet-anti-tracrRNA
Plate results:
Experimental: over 1000 colonies
BO control had ~500 colonies, IO control had ~50 colonies
Cultured colonies
4 copies of potential pSB1C3-R0040-anti-tracrRNA
Objective: Switch dCas9-tracrRNA into pSB1C3
Plate results:
Experimental: 12-15 colonies
BO control had 3 colonies, IO control had ~30 colonies
Minipreps
4 copies of potential pSB1C3-dCas9-tracrRNA construct
From 7/14 ligation
concentrations: 537.2, 45.4, 79.1, and 71.5 ng/uL
Only one has normal concentration, others very low
Analytical digest of pSB1C3-dCas9-tracrRNA constructs:
4 copies with NheI/XbaI
Agarose gel of digests:
Lanes 1-2: pSB6A1-K608012 with MfeI/XbaI (expected 4.1, 0.6 kb bands)
Lanes 3-6: pSB1C3-dCas9-tracrRNA with NheI/XbaI (expected 5.0, 1.5 kb bands)
Results: gel was fuzzy and unclear
lanes 2,3,5,6 clearly incorrect: 1 band ~2.0 kb
lane 1 had one fuzzy band around 4-5 kb (could be incomplete digest)
lane 4 had band around 5 kb with smear above (could be incomplete digest)
New analytical digest of pSB1C3-dCas9-tracrRNA construct
Only copy #2 (showed potential in previous digest
With KpnI/SpeI for 2 hours at 37C
Also put previous digest back in bath for 2 more hours
Agarose gel of digests
DETAILS
Objective: Create pSB1C3-R0011-I13500 as destination for mCherry G-block
Plate results:
~500 colonies on experimental plate
BO control had over 1000 colonies
IO control had no colonies
Cultured colonies
4 copies of potential pSB1C3-R0011-I13500 construct
Objective: Switch K608012 into pSB6A1
Plate results:
0 colonies on experimental plate
4 colonies on BO control, 1 colony on IO control
Objective: Insert crRNAs into pdCas9
Colony PCR of pdCas9-GFP3 plate
Plate from 7/7/14
15 colonies plus pdCas9 miniprep as control
double dipped in 50 uL Taq poly mix and 50 uL SOC
Oligos pdCas9-up1 and pdCas9-dn1
TALCOLONYPCR protocol with 64C anneal
Agarose gel of colony PCR
Expected to see higher band matching control in successful colonies
lanes 1 and 17 are pdCas9 control
assay looks helpful, successes in samples 3,5,6,8,9,12,13,15
Cultured colonies
Copies 3,5,6, and 8 from colony PCR in LB+Cm
July 17
Objective: Create pTet-anti-tracrRNA and pSB1C3-R0011-I13500
Minipreps
4 copies of pSB1C3-R0040-anti-tracrRNA from 7/15 ligation
4 copies of pSB1C3-R0011-I13500 from 7/15 ligation
Analytical digest of minipreps
pSB1C3-R0040-anti-tracrRNA with EcoRI/PstI
pSB1C3-R0011-I13500 with MfeI/NdeI/SacI
Agarose gel of digest results:
Lanes 1-4: pSB1C3-R0040-anti-tracrRNA with EcoRI/PstI (expected 2.0, 0.2 kb bands) (incomplete expected 2.0, <0.1 kb bands)
Lanes 5-8: pSB1C3-R0011-I13500 with MfeI/NdeI/SacI (expected 1.3, 0.9, 0.33,0.29 kb bands)
Results:
R0040-antitracrRNA looks correct for all 4 copies--small band is hard to tell exactly
R0011-I13500 does not look correct (only one band at 2.0 kb)
Objective: Switch dCas9-tracrRNA into pSB1C3
Colony PCR on pSB1C3-dCas9-tracrRNA plate
Ligation plate from 7/15/14
15 colonies doubled into 50 uL Taq poly mix and 50 uL SOC
oligos SB1C3-up1 and SB1C3-dn1
annealed at 65C, extension 2 mins
Agarose gel of colony PCR
results: primer-bands only (no successes)
Objective: Switch K608012 into pSB6A1
Culture colonies
2 copies of pSB6A1-J04450 (from 6-9-14 plate)
2 copies of pSB1C3-K608012
July 18
Objective: Switch dCas9-tracrRNA into pSB1C3
PCR of dCas9-tracrRNA from pdCas9
8 tubes, each with 1 uL pdCas9 template at 1 ng/uL
oligos dCas9tracr-up and dCas9-tracr-dn
2 min extension, 64C anneal
Agarose gel of PCR product
All 8 tubes look successful (band ~4 kb)
PCR cleanup of dCas9-tracrRNA
Qiagen kit
Into 2 tubes, 30 uL each
Prep-scale digest of PCR product
with PstI/XbaI/DpnI
in 100 uL each for 3 hrs at 37C
Treat pSB1C3 digest with CIP
1 hr at 37C
Objective: Switch K608012 into pSB6A1
Miniprep pSB1C3-K608012 and pSB6A1-J04450
1 copy each (the second copy of each did not grow in culture)
concentrations:
pSB1C3-K608012: 344 ng/uL
pSB6A1-J04450: 421.5 ng/uL
Prep-scale digest of K608012 and pSB6A1
50 uL DNA in 100 uL reaction
Both with EcoRI and PstI
3 hrs at 37C
CIP treatment of pSB6A1 and heat-inactivation
2.5 uL CIP added to digest tube
Both digests returned to 37C for 1 hr
Then 30 mins at 80C to heat-inactivate
Gel extraction of pSB6A1 and K608012
Gel had no EtBr added, and was discarded (product was lost)
July 20
Objective: culture colonies for various purposes
Cultured colonies
4 copies pSB6A1-J04450
2 from older plate, 2 from newer
4 copies pSB1C3-K608012
2 from older plate, 2 from newer
4 copies pSB1C3-B0030
To obtain pSB1C3 without gel extraction
4 copies pSB1C3-R0011
4 copies pSB1C3-I13500
July 21
Objective: Sequence to confirm constructs
Sequencing reaction and Order #1634
1-4: pdCas9-GFP3 #3,5,6,8 with primer pdCas9-up1
5-8: pdCas9-GFP3 #3,5,6,8 with primer pdCas9-dn1
9-10: pSB1C3-R0040-anti-tracrRNA #1,2 with primer SB1C3-up1
11-12: pSB1C3-R0040-anti-tracrRNA #1,2 with primer SB1C3-dn1
Big Dye, buffer version 1.1
Order 1634 Results:
pdCas9-GFP3 copies 3, 6, and 8 are correct
copy 5 has a single mutation
anti-tracrRNA: both copies are correct
copy 1 was a cleaner, more reliable
Objective: Create pSB1C3-R0011-I13500
Minipreps
Qiagen kit
4 tubes of pSB1C3-R0011
concentrations 255.7, 169.2, 204.8, 151.2 ng/uL
4 tubes of pSB1C3-I13500
concentrations 148.9, 214.2, 260.7, 432.5
Prep scale digests of pSB1C3-R0011 and I13500
pSB1C3-R0011 with SpeI/PstI, samples 1,3
pSB1C3-I13500 with XbaI/PstI, samples 1,3
50 uL DNA in 100 uL each reaction, 2 tubes per construct
3 hrs at 37C
Added CIP (2.5 uL) to pSB1C3-R0011 reaction
1 more hr for both at 37C
Heat inactivation of both reactions: 30 mins at 80C
Gel extraction of I13500
extraction gel 1, lanes 1 and 2
Zymoclean prep kit
sample 1 (lane 1): 300 mg gel yields
sample 3 (lane 2): 550 mg gel yields
PCR cleanup of pSB1C3-R0011
Qiagen kit
Two tubes of digest combined into one tube product
Concentration
Ligation of pSB1C3-R0011 and I13500
Old backbone (pSB1C3-R0011 digested 7/15): 100 ng = 0.6 uL
New backbone (pSB1C3-R0011 digested 7/21): 100 ng = 2.2 uL
Old insert (I13500 extracted 7/15): 300 ng = 2.4 uL
New insert (I13500 extracted 7/21): 300 ng = 3 uL
Ligations numbered 1-8:
1. OB+OI
2. OB+NI
3. NB+OI
4. NB+NI
5. OB control
6. OI control
7. NB control
8. NI control
1 hr at rt
Transformed via heat shock into DH5alphas, plated on LB+Cm
Objective: Switch dCas9-tracrRNA into pSB1C3
Minipreps
Qiagen kit
4 tubes of pSB1C3-B0030
to obtain pSB1C3 without gel extraction
concentrations 116.8, 99.9, 115.3, 145.1 ng/uL
PCR cleanup of dCas9-tracrRNA PCR
Product digested 7/18 with XbaI/PstI/DpnI
Concentration 80.5 ng/uL in 30 uL
Heat inactivation of cleaned product, 30 mins at 80C
Prep scale digest of pSB1C3-B0030
With XbaI/PstI, samples 1,3
50 uL DNA in 100 uL each reaction, 2 tubes
3 hrs at 37C
Added CIP (2.5 uL)
1 more hr for both at 37C
Heat inactivation: 30 mins at 80C
PCR cleanup of pSB1C3
B0030 should not be retained
Qiagen kit
Two tubes of digest combined into one tube product
Concentration
Ligation of pSB1C3 and dCas9-tracrRNA
Backbone (pSB1C3 digested 7/21): 300 ng = 3.5 uL
Old insert (dCas9-tracrRNA digested 7/15): 100 ng = 3 uL
New insert (dCas9-tracrRNA digested 7/18): 100 ng = 1.2 uL
Ligations numbered 9-13:
9. B+OI
10. B+NI
11. B control
12. OI control
13. NI control
1 hr at rt
Transformed via heat shock into DH5alphas, plated on LB+Cm
Objective: Switch K608012 into pSB6A1
Minipreps
Qiagen kit
4 tubes of pSB6A1-J04450
concentrations 189.3, 432.5, 688.7, 487.3 ng/uL
4 tubes of pSB1C3-K608012
concentrations 68.8, 91.6, 346.1, ng/uL
Prep scale digests of pSB6A1-J04450 and pSB1C3-K608012
pSB6A1-J04450 with XbaI/SpeI, samples 1,3
pSB1C3-K608012 with EcoRI/PstI, samples 1,3
50 uL DNA in 100 uL each reaction, 2 tubes per construct
3 hrs at 37C
Added CIP (2.5 uL) to pSB6A1-J04450 reaction
1 more hr for both at 37C
Heat inactivation of both reactions: 30 mins at 80C
Gel extraction of pSB6A1 and K608012
K608012: gel 1, lane 3 and gel 2, lane 1
pSB6A1: gel 2, lanes 2 and 3
Zymoclean prep kit
K608012 sample 1 (lane 1-3): 300 mg gel yields
K608012 sample 3 (lane 2-1): 550 mg gel yields
pSB6A1 sample 1 (lane 2-2):
pSB6A1 sample 3 (lane 2-3):
Ligation of pSB6A1 and K608012
Old backbone (pSB6A1 extracted 7/15): 100 ng = 0.5 uL
Old insert (K608012 extracted 7/15): 300 ng = 4 uL
Ligations numbered 14-16:
14. B+I
15. B control
16. I control
1 hr at rt
Transformed via heat shock into DH5alphas, plated on LB+Amp
Gibson of pSB6A1 and K608012
New backbone (pSB6A1 extracted 7/21): 100 ng = 0.3 uL
New insert (K608012 extracted 7/21): 300 ng = 1.7 uL
Numbered 17-19:
17. B+I
18. B control
19. I control
15 uL Mert’s DIY Gibson mix
1 hr at 60C
Transformed via heat shock into DH5alphas, plated on LB+Amp
SLIC of pSB6A1 and K608012
New backbone (pSB6A1 extracted 7/21): 100 ng = 0.3 uL
New insert (K608012 extracted 7/21): 300 ng = 1.7 uL
Numbered 20-22:
20. B+I
21. B control
22. I control
2.5 mins at rt, then 10 mins at 4C
Transformed via heat shock into DH5alphas, plated on LB+Amp
July 22
Objective: Create pSB1C3-R0011-I13500
Plate results:
All four experimental plates had lots of colonies
both Backbone-only controls had similar colony numbers to experimentals
Insert-only controls had no colonies
Colony PCR
4 copies each of 4 experimental plates = 16 colonies
Oligos SB1C3-up and SB1C3-dn with Taq polymerase
66C anneal temp, 2 min extension time
Agarose gel of colony PCR results
1-4: OB+OI pSB1C3-R0011-I13500
5-8: OB+NI pSB1C3-R0011-I13500
9-12: NB+OI pSB1C3-R0011-I13500
13-16: NB+NI pSB1C3-R0011-I13500
17-20: B+OI pSB1C3-dCas9-tracrRNA
21-24: B+NI pSB1C3-dCas9-tracrRNA
Results: no successful bands on any colonies
Just primer-dimers or small amplified bands?
Objective: Switch dCas9-tracrRNA into pSB1C3
Plate results:
Both experimental plates had lots of colonies
Backbone-only controls had similar colony numbers to experimentals
Insert-only controls had no colonies
Colony PCR
4 copies each of 2 experimental plates = 8 colonies
Oligos SB1C3-up and SB1C3-dn with Taq polymerase
66C anneal temp, 2 min extension time
Agarose gel of colony PCR results
see above for gel order
Results: no successful bands on any colonies
Just primer-dimers or small amplified bands?
Objective: Switch K608012 into pSB6A1
Plate results:
No colonies on Gibson or SLIC plates
3 colonies on Ligation plate
A few colonies on controls, but not many
Cultured colonies
3 colonies of potential pSB6A1-K608012 from Ligation plate
in LB+Amp
Ligation of K608012 and pSB6A1
Same constructs as 7/21 ligation
This time let recover in SOC for 2 hrs instead of 1 hr
Plated on LB+Amp
Objective: Try putting K608012 into pSB1A2 instead
Transformation from kit plate
Plate 4-1N
pSB1A2-B0034
Plated on LB+Amp
July 23
Objective: Switch K608012 into pSB6A1
Miniprep 3 copies of potential pSB6A1-K608012
Qiagen kit
Saved stocks in 15% glycerol
Analytical digest of minipreps
With NdeI in Cutsmart, 10 uL final volume
Agarose gel of analytical digest
Results:
Copies 1 and 2 are strange and incorrect
Copy 3 looks correct
Need to check with more digests
Analytical digest to check promising sample
Copy 3 in 3 different reactions
EcoRI/SpeI
EcoRI/MfeI
NcoI/PvuI
Control: pSB6A1-J04450 with EcoRI/SpeI/PvuI
Control: pSB1C3-K608012 with MfeI/NcoI
Agarose gel of analytical digest
1. Ligation 3 with EcoRI/SpeI
2. Ligation 3 with EcoRI/MfeI
3. Ligation 3 with NcoI/PvuI
4. pSB6A1-J04450 with EcoRI/SpeI/PvuI
5. pSB1C3-K608012 with MfeI/NcoI
Results:
Sample looks good
Incorrect samples 1 and 2 glycerol stocks thrown out
Objective: Switch R0040-anti-tracrRNA into pSB4K5
Treat pSB4K5 with CIP
1.25 uL CIP in 25 uL reaction
pSB4K5 was cut with EcoRI/SpeI and extracted on 7/9
Prep-scale digest to obtain R0040-anti-tracrRNA
pSB1C3-R0040-anti-tracrRNA
with EcoRI/SpeI
Gel extraction and cleanup of R0040-anti-tracrRNA
Extracted smaller band
Attempted to clean up using Qiagen prep kit
dissolved with Zymoclean ADB
Then followed Qiagen PCR cleanup protocol
510 mg gel yielded little or no DNA
Objective: Create pSB1C3-R0011-I13500
Colony PCR
8 colonies from 7/21 ligation plates
Control colonies from plates of pSB1C3-I13500 and pSB1C3-R0011
Agarose gel of colony PCR results
No successes
Controls did not work properly either
colony PCR does not seem to be effective--need to just pick colonies
Culture colonies
8 colonies of potential pSB1C3-R0011-I13500
From 7/21 ligation plates (colonies from today’s colony PCR)
Objective: Measurement interlab study
Transformations from kit plates
Plate 1-20K
pSB1C3-K823005
Plate 1-22I
pSB1C3-K823012
Plate 2-24B
pSB1C3-E0240
July 24
Objective: Create pSB1C3-R0011-I13500
Miniprep 8 colonies of potential pSB1C3-R0011-I13500
Qiagen kit
Analytical digest of minipreps
8 copies with NcoI
8 copies with EcoRI/SpeI
Agarose gel of digest results
1-8: copies 1-8 with NcoI
9-16: copies 1-8 with EcoRI/SpeI
Results: no successes, insert appears to be missing
Unsure whether incorrect colonies are pSB1C3 or pSB1C3-R0011
Agarose gel of digest results
Thicker gel, longer run time, more DNA
from same digest tubes
Results:
no small band on EcoRI/SpeI cut and no difference in larger band for the two cuts, indicating that R0011 is not present
Ligation of pSB1C3-R0011 and I13500
Used lab stock of T4 Ligase and buffer rather than iGEM stock
100 ng pSB1C3-R0011 = 2.5 uL
300 ng I13500 = 3 uL
4 tubes of ligation, run at rt with Ligase for 15, 30, 45, and 60 mins
BO and IO controls for each time
Plated on LB+Cm, two tubes per plate (used wooden streaker rods to spread on half)
Objective: Switch dCas9-tracrRNA into pSB1C3
Ligation of pSB1C3 and dCas9-tracrRNA
Used lab stock of T4 Ligase and buffer rather than iGEM stock
300 ng pSB1C3 = 3 uL
100 ng dCas9-tracrRNA PCR = 1 uL
4 tubes of ligation, run at rt with Ligase for 15, 30, 45, and 60 mins
BO and IO controls for each time
Plated on LB+Cm, two tubes per plate (used wooden streaker rods to spread on half)
Objective: Test GFP repression by crRNAs
Transform pSB6A1-K608012 into DH5alpha-ZI
Plated on LB+Amp
Objective: Obtain constructs for various uses
Streaked plate from frozen stock
One LB+Cm plate in thirds:
pSB1C3-R0040-antitracrRNA
pSB1C3-Repeat-seq scaffold
pSB1C3-Csy4-gRNA scaffold
Objective: Measurement interlab study
Cultured colonies (2 each)
pSB1C3-E0240
pSB1C3-K823005
pSB1C3-K823012
pSB3K3-I20260
July 25
Objective: Create pSB1C3-R0011-I13500
Plate results:
no apparent difference between 15, 30, 45, or 60 minute ligations
many colonies on all Experimental and BO control plates
no colonies on IO control
Colony PCR:
16 colonies from pSB1C3-R0011-I13500 plates
4 from each ligation time
Taq polymerase with oligos SB1C3-up and SB1C3-dn
66C anneal temp, 45 sec extension
Agarose gel of colony PCR
No successes
It seems as though colony PCRs are not working--try control with a gradient
Gradient Colony PCR
to optimize with SB1C3-up and SB1C3-dn primers
template colonies of pSB1C3-K608011 from plate
anneal temps 62-72 C, extension time 45 sec
Agarose gel of gradient colony PCR
left to right: 62C to 72C anneal temps
Results: 62C was the best anneal temp--worked well
gradual decrease as temp increased, no amplification above 65C
Objective: Switch dCas9-tracrRNA into pSB1C3
Plate results:
no apparent difference between 15, 30, 45, or 60 minute ligations
many colonies on all Experimental and BO control plates
no colonies on IO control
Colony PCR:
16 colonies from pSB1C3-dCas9-tracrRNA plates
4 from each ligation time
Taq polymerase with oligos SB1C3-up and SB1C3-dn
66C anneal temp, 45 sec extension
Agarose gel of colony PCR
No successes
It seems as though colony PCRs are not working--try control with a gradient
see gradient results above
Objective: Various ligations
Prep-scale digests
pSB1C3-Csy4-scaffold (2 copies) with XbaI/PstI
to ligate into R0040
pSB1C3-Repeat-seq scaffold (2 copies) with XbaI/PstI
to ligate into R0040
pSB1C3-R0040-anti-tracrRNA (2 copies) with XbaI/PstI
100 uL reactions each tube
PCR cleanups on all digests
Qiagen kit
Concentrations (in 50 uL each):
pSB1C3-Csy4-scaffold: 55.9, 105.5 ng/uL
pSB1C3-Repeat-seq scaffold: 15.2, 55.2 ng/uL
pSB1C3-R0040-anti-tracrRNA: 90.6 ng/uL
Need to extract from gels before using, only cleaned up for safer storage
Objective: Measurement interlab study
Minipreps
Qiagen kit
pSB1C3-K823005 (2 copies)
concentrations 256.3, 314.8 ng/uL
pSB1C3-K823012 (2 copies)
concentrations 204.4, 192.5 ng/uL
pSB1C3-E0240 (2 copies)
concentrations 266.3, 241.5 ng/uL
Prep-scale digests
pSB1C3-K823005 (2 copies) with SpeI/PstI
pSB1C3-K823012 (2 copies) with SpeI/PstI
pSB1C3-E0240 (2 copies) with XbaI/PstI
100 uL each tube
PCR cleanups on all digests
Qiagen kit
Concentrations (in 50 uL each):
pSB1C3-K823005: 156.3, 181.7 ng/uL
pSB1C3-K823012: 110.0, 98.0 ng/uL
pSB1C3-E0240: 110.0, 114.6 ng/uL
Objective: Test crRNA repression of GFP reporter
Cultured colony
To make CCEC of reporter strain
One colony of pSB6A1-K608012 in DH5alpha-ZI
Grew for 8 hrs during the day in 2 mL LB+Amp at 37C
Inoculated into 100 mL flask of LB+Amp for overnight shaker at rt
Flow cytometry of reporter strain
quick check to make sure reporter still works with new backbone
Used parameters from previous GFP flow data
Levels slightly under what was seen before, but still strong
July 26
Objective: Test crRNA repression of GFP reporter
Prepared Chemically competent E. coli
Using DH5alpha-ZI + pSB6A1-K608012
iGEM CCEC protocol with smaller volumes (made ~20 1-mL aliquots)
Double-Transformation of pdCas9 variants into reporter cells
Charlie Cooper
pdCas9 (with BsaI insert as crRNA sequence)
pdCas9-crRNA_GFP1
pdCas9-crRNA_GFP2
pdCas9-crRNA_GFP3
Objective: Create pSB1C3-R0011-I13500
Colony PCRs
16 colonies from 7/24 ligation plates
oligos SB1C3-up and SB1C3-dn
62C anneal (as determined by gradient), 45 sec extension
Agarose gel of Colony PCRs
Results: colonies 5 and 14 amplified at the correct size
Culture colonies
Colonies 5 and 14 from colony PCR of potential pSB1C3-R0011-I13500
Objective: Create pSB1C3-dCas9-tracrRNA
Colony PCRs
16 colonies from 7/24 ligation plates
oligos SB1C3-up and SB1C3-dn
62C anneal (as determined by gradient), 45 sec extension
PCRs left in machine without starting reaction for ~1 hr, could have kill reaction
Agarose gel of Colony PCRs
No successes
July 27
Objective: Test crRNA repression of GFP reporter
Cultured colonies for flow
3 copies of each
DH5alpha-Z1 (background)
DH5-alpha-Z1 + pSB6A1-K608012 (reporter positive control)
DH5-alpha-Z1 + pSB6A1-K608012 + pdCas9 (non-target control)
DH5-alpha-Z1 + pSB6A1-K608012 + pdCas9-crRNA_GFP1
DH5-alpha-Z1 + pSB6A1-K608012 + pdCas9-crRNA_GFP2
DH5-alpha-Z1 + pSB6A1-K608012 + pdCas9-crRNA_GFP3
Objective: Create pSB1C3-R0011-I13500
Minipreps
Colonies 5 and 14 from 7/26 colony PCRs
Qiagen kit
July 28
Objective: Create pSB1C3-R0011-I13500
Analytical digest of minipreps
Colonies 5 and 14 of potential pSB1C3-R0011-I13500
with MfeI
with NcoI
with EcoRI/PstI
Agarose gel of analytical digest results
Gel Order:
#14 w/ MfeI
#14 w/ EcoRI/PstI
#14 w/ NcoI
#5 w/ MfeI
#5 w/ EcoRI/PstI
#5 w/ NcoI
Results: All looks correct, except that MfeI cut once instead of twice
could be pSB1C3-I13500 without R0011
Objective: Insert mCherry G-block into pSB1C3-R0011-I13500
PCR of mCherry G-block
Q5 polymerase with Oligos SB1C3-dn and pDGC3-up
4 tubes with 0, 0.4, 0.7, 1.0 uL template at 1 ng/uL
45 sec extension, 66C anneal temp
Agarose gel of PCR
3 tubes look successful
PCR cleanup of mCherry G-block PCR
Qiagen kit
40 uL at 27.6 ng/uL
Objective: Various ligations
Gel extraction and cleanup
Each extraction included 2 cleaned digests combined into one lane (~100 uL)
Cut and cleaned with Zymoclean prep kit
pSB1C3-Csy4-scaffold cut with XbaI/PstI on 7/25
pSB1C3 backbone 47.5 ng/uL in 30 uL
Csy4-scaffold insert 15.4 ng/uL in 30 uL
pSB1C3-Repeat-seq scaffold cut with XbaI/PstI on 7/25
pSB1C3 backbone 74.8 ng/uL in 30 uL
Repeat-seq scaffold insert 13.1 ng/uL in 30 uL
pSB1C3-R0040-anti-tracrRNA cut with XbaI/PstI on 7/25
pSB1C3 backbone 164.0 ng/uL in 30 uL
R0040-anti-tracrRNA 176.0 ng/uL in 30 uL
PCR Cleanup on gel extraction cleanups
to “superclean”
pSB1C3 4 tubes combined into 1: 469.6 ng/uL in 20 uL
Repeat-seq scaffold: 23.9 ng/uL
rough graph, probably not useful
Csy4 scaffold: 16 ng/uL
rough graph, probably not useful
R0040-anti-tracrRNA: 7.4 ng/uL
rough graph, probably not useful
Objective: Measurement interlab study
CIP treatment of backbones
Using Charlie’s CIP stock
pSB1C3-K823005 cut with SpeI/PstI on 7/25 (2 tubes)
pSB1C3-K823012 cut with SpeI/PstI on 7/25 (2 tubes)
Gel extraction and cleanup
2 cleaned digests combined into one lane (~100 uL)
Cut and cleaned with Zymoclean prep kit
pSB1C3-E0240 cut with XbaI/PstI on 7/25
pSB1C3 backbone 116.4 ng/uL in 30 uL
E0240 86.5 ng/uL in 30 uL
PCR Cleanup of CIP treatment and gel extraction cleanup
Qiagen kit
pSB1C3-K823005 SpeI/PstI CIP treated: 119.8, 119.7 ng/uL
pSB1C3-K823012 SpeI/PstI CIP treated: 60.9, 62.3 ng/uL
E0240 XbaI/PstI extracted: 56.5 ng/uL in 20 uL (“superclean”)
Ligations
E0240 into two backbones: pSB1C3-K823005 and pSB1C3-K823012
pSB1C3-K823005 SpeI/PstI/CIP: 100 ng = 0.87 uL
pSB1C3-K823012 SpeI/PstI/CIP: 100 ng = 1.67 uL
E0240 XbaI/PstI: 300 ng = 6 uL
2 BO controls and 1 IO control
20 mins at rt
Heat shock transformation and plated on LB+Cm
Objective: Test crRNA repression of GFP reporter
Dilute cultures to grow in log phase for flow
1/1000 dilution into 50 mL
18 samples: 3 copies each of 6 strains (labelled 2-19)
Samples 2-4: DH5alpha-Z1 (background)
grown in LB+Spec
Samples 5-7: DH5alpha-Z1 + pSB6A1-K608012 (reporter positive control)
grown in LB+Amp
Samples 8-10: DH5alpha-Z1 + pSB6A1-K608012 +pdCas9 (non-target control)
grown in LB+Amp+Cm
Samples 11-13: DH5alpha-Z1 + pSB6A1-K608012 +pdCas9_GFP1
grown in LB+Amp+Cm
Samples 14-16: DH5alpha-Z1 + pSB6A1-K608012 +pdCas9_GFP2
grown in LB+Amp+Cm
Samples 17-19: DH5alpha-Z1 + pSB6A1-K608012 +pdCas9_GFP3
grown in LB+Amp+Cm
Flow cytometry
Samples grown for 4.5 hrs after initial dilution
1/50 final dilution of sample into PBS, placed on ice until run
parameters (same as previous GFP runs):
FSC 350V log3
SSC 350V log3
B1 350V log5
Trigger 10
10,000 event limit (100,000 events used in the past)
Results
~3-fold repression of reporter with crRNA_GFP1 or crRNA_GFP3
crRNA_GFP2 did not repress
non-target control repressed little or not at all
See document “ Flow stats 7-28-14 ” for data details
July 29
Objective: Switch R0040-anti-tracrRNA into pSB4K5
Minipreps
pSB4K5-J04450 (4 copies)
pSB1C3-R0040-anti-tracrRNA (4 copies)
Prep-scale digest
pSB4K5-J04450 (4 copies) with EcoRI/SpeI
pSB1C3-R0040-anti-tracrRNA (4 copies) with EcoRI/SpeI
Gel extraction and cleanup of digests
Cut pSB4K5 backbone from 4 copies of pSB4K5-J04450
2 digests per gel well
2.04 g total gel eluted into two tubes:
51.4 ng/uL and 94.5 ng/uL, 30 uL each
Cut R0040-anti-tracrRNA insert from pSB1C3-R0040-anti-tracrRNA
2 digests per gel well
1.61 g total gel eluted into two tubes:
99.4 ng/uL and 82.3 ng/uL, 30 uL each
Objective: Create pSB1C3-R0011-I13500
Colony PCR
24 colonies of potential pSB1C3-R0011-I13500
8 colonies from 7/15 ligation
4x4=16 colonies from 4 different 7/21 ligation plates
Taq polymerase with oligos SB1C3-up and SB1C3-dn
extension time 45 sec, anneal temp 62C
Agarose gel of colony PCR results
colony #19 is only promising result
Analytical digest
pSB1C3-R0011 copies 2, 4
pSB1C3-I13500 copies 2, 4
with MfeI/NcoI
To test whether enzymes or original parts are the source of ligation difficulties
Agarose gel of digest results
Order:
1. pSB1C3-R0011 #2 cut with MfeI/NcoI (expected 2 bands)
2. pSB1C3-R0011 #4 cut with MfeI/NcoI (expected 2 bands)
3. pSB1C3-I13500 #2 cut with MfeI/NcoI (expected 3 bands)
4. pSB1C3-I13500 #4 cut with MfeI/NcoI (expected 3 bands)
Results:
R0011 both copies only show one band
I13500 both copies appear correct
Cultured colonies
Potential pSB1C3-R0011-I13500 #19 from colony PCR
pSB1C3-R0011 (3 copies) from original transformation
suspicion that our R0011 is incorrect--could be just bad copies
Objective: Put R0040 promoter in front of crRNA and gRNA scaffolds
Culture colonies
pSB1C3-R0040 (4 copies)
Objective: Measurement interlab study
Plate results
both experimental and both backbone-only plates all had hundreds of colonies
insert-only controls had no colonies
Colony PCR
8 colonies of potential pSB1C3-K823005-E0240
8 colonies of potential pSB1C3-K823012-E0240
Taq polymerase with oligos SB1C3-up and SB1C3-dn
extension time 45 sec, anneal temp 62C
Agarose gel of colony PCR results
1-8: pSB1C3-K823005-E0240
9-16: pSB1C3-K823012-E0240
Results: no successes
July 30
Objective: Create pSB1C3-R0011-I13500
Minipreps
pSB1C3-R0011-I13500 #19
concentration 246.4 ng/uL
pSB1C3-R0011 (3 copies)
concentrations 187.6, 178.8, 212.8 ng/uL
Analytical digest
pSB1C3-R0011-I13500 #19
three digests in cutsmart, 10 uL final volume each
NcoI
MfeI
XbaI/PstI
pSB1C3-R0011 (3 copies)
with NcoI/MfeI in cutsmart, 10 uL final volume
Agarose gel of digest results
1. pSB1C3-R0011-I13500 #19 with NcoI
2. pSB1C3-R0011-I13500 #19 with MfeI
3. pSB1C3-R0011-I13500 #19 with EcoRI/PstI
4. pSB1C3-R0011 #1 with NcoI/MfeI
5. pSB1C3-R0011 #2 with NcoI/MfeI
6. pSB1C3-R0011 #3 with NcoI/MfeI
Results: hard to explain
R0011-I13500 not correct
R0011 all three copies only cut once
Objective: Switch R0040-anti-tracrRNA into pSB4K5
Treat pSB4K5 with Antarctic Phosphatase
2 copies digested on 7/29, combined into one tube with 100 uL final volume
2 hrs at 37C
PCR cleanup of AP-treated pSB4K5
Qiagen kit
concentration 45.2 ng/uL in 50 uL
Ligation of pSB4K5 and R0040-anti-tracrRNA
2 experimental plates with 3x and 6x molar insert
pSB4K5 backbone (EcoRI/SpeI/AP)100 ng = 2 uL
R0040-anti-tracrRNA insert (EcoRI/SpeI) 30 ng = 0.3 uL, or 60 ng = 0.6 uL
Backbone-only and insert-only controls
Transformed via heat shock and plated on LB+G418
Objective: Put R0040 promoter in front of crRNA and gRNA scaffolds
Minipreps
pSB1C3-R0040 (4 copies)
concentrations 47.4, 80.7, 109.7, 193.5
Objective: Switch dCas9-tracrRNA into pSB1C3
Treat pSB1C3 with Antarctic Phosphatase
cleaned copy from 7/28 with 100 uL final volume
2 hrs at 37C
PCR cleanup of AP-treated pSB1C3
Qiagen kit
concentration 104.5 ng/uL in 50 uL
Ligation of pSB1C3 and dCas9-tracrRNA
2 experimental plates
3x molar dCas9-tracrRNA “insert”
pSB1C3 100 ng = 1 uL
dCas9-tracrRNA 600 ng = 7.5 uL
3x molar pSB1C3 “backbone”
pSB1C3 150 ng = 1.5 uL
dCas9-tracrRNA 100 ng = 1.2 uL
Backbone-only and insert-only controls
Transformed via heat shock and plated on LB+Cm
July 31
Objective: Switch R0040-anti-tracrRNA into pSB4K5
Plate results:
Hundreds of colonies on both experimental plates
20-30 colonies on backbone-only control
0 colonies on insert-only control
Cultured colonies
4 copies of pSB4K5-R0040-anti-tracrRNA
Objective: Switch dCas9-tracrRNA and dCas9-tracrRNA-crRNA into pSB1C3
Plate results:
Experimental plate 2 (higher pSB1C3 concentration) had 2 colonies
Experimental plate 1 had no colonies
Backbone-only control had 1 colony, insert-only control had 0
Cultured colonies
2 copies of potential pSB1C3-dCas9-tracrRNA
PCRs of tracrRNA-dCas9 and tracrRNA-dCas9-crRNA
8 tubes of each, with 1 uL 1 ng/uL pdCas9 template per tube
tracrRNA-dCas9 oligos: dCas9tracr-up and dCas9tracr-dn
tracrRNA-dCas9-crRNA oligos: dCas9tracr-up and pdCas9-dn
Q5 polymerase
64C anneal temp, 2:30 extension time
Agarose gel of PCRs
1-8: tracrRNA-dCas9-crRNA
9-16: tracrRNA-dCas9
Results: all 16 lanes look good
PCR cleanup of PCRs
8 tubes into 50 uL for each of two PCRs
Concentrations
tracrRNA-dCas9-crRNA: 350 ng/uL
tracrRNA-dCas9: 335 ng/uL
Prep-scale digest of 2 PCRs
tracrRNA-dCas9-crRNA and tracrRNA-dCas9
Each in 100 uL final volume with 2 uL each of XbaI/PstI/DpnI
PCR cleanup of digests
Into 50 uL final volume
Objective: Put R0040 in front of Csy4 scaffold and Repeat-seq scaffold
Prep-scale digest
pSB1C3-R0040 (2 copies) with 2.5 uL each of SpeI/PstI
100 uL final volume each copy
PCR cleanup of digests
Into 50 uL final volume
Objective: Triple-transformation of R0040-anti-tracrRNA into Z1-reporter-crRNA_GFP cells
Cultured cells in morning
straight from frozen stocks into 2 mL LB+Cm+Amp
DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP1
DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP3
Cultured cells in evening
1 mL from morning culture into 50 mL flask for overnight growth
To make chemically competent cells
Objective: Create pSB1C3-R0011-I13500
Cultured colonies
2 copies of pSB1C3-R0011 from streak plate of original frozen stock
Retrieved from " http://2014.igem.org/Team:Duke/Notebook/July "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
