<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Duke_Notebook_August"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Duke/Notebook/August">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Duke/Notebook/August&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Notebook/August&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Notebook/August&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Duke/Notebook/August" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Duke/Notebook/August</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><UL id="navbar"><LI><A href="https://2014.igem.org/Team:Duke">Home</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team">About Us</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Duke">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team/Gallery">Photo Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team/Attributions">Attributions</A></LI></UL><LI><A href="https://2014.igem.org/Team:Duke/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#summary">Summary</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#future">Future Directions</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Overview">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Months">Notebook by Month</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy">Policy and Practice</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#threed">3D Printing</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#hoco">House Course</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#smath">NCSSM Team</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Safety">Safety</A></LI></P><DIV class="day"><A id="aug1"><H2>August 1</H2></A><DIV class="lab"><P class="obj">Objective: Create pSB1C3-R0011-I13500</P><P>Minipreps</P><UL><LI>2 copies of pSB1C3-R0011 from streak plate of original frozen stock</LI><LI>Concentrations 187.0, 197.1 ng/uL</LI></UL><P>Analytical digest</P><UL><LI>To help determine what is causing the difficulties in this construct</LI><LI>2 copies of pSB1C3-R0011 from frozen stock</LI></UL><UL><LI>with MfeI and XbaI/PstI</LI></UL><UL><LI>pSB1C3-R0011-I13500 #19 from 7/29 colony PCR</LI></UL><UL><LI>with MfeI and XbaI/PstI</LI></UL><UL><LI>pSB1C3-K608012 as a control for MfeI activity</LI></UL><UL><LI>with MfeI</LI></UL><UL><LI>pSB1C3-I13500 as a control for MfeI activity</LI></UL><UL><LI>with MfeI</LI></UL><P>Agarose gel of digest results</P><UL><LI>Gel one: (see later entries for details on lanes 1-8)</LI></UL><UL><LI>Lanes 1-4: pSB4K5-R0040-anti-tracrRNA cut with XbaI/PstI</LI><LI>Lanes 5-6: pSB1C3-dCas9-tracrRNA cut with XbaI/PstI</LI><LI>Lanes 7-8: pSB1C3-dCas9-tracrRNA cut with XbaI/NheI</LI><LI>Lanes 9-10: pSB1C3-R0011 (2 copies) cut with XbaI/PstI</LI><LI>Lanes 11-12: pSB1C3-R0011 (2 copies) cut with MfeI</LI><LI>Lane 13: pSB1C3-R0011-I13500 #19 cut with XbaI/PstI</LI><LI>Lane 14: pSB1C3-R0011-I13500 #19 cut with MfeI</LI><LI>Lane 15: sample for Tony Burnetti</LI></UL><UL><LI>Gel two:</LI></UL><UL><LI>1: pSB1C3-K608012 cut with MfeI</LI><LI>2. pSB1C3-I13500 cut with MfeI</LI><LI>3. pSB1C3-K608012 uncut</LI><LI>4. pSB1C3-I13500 uncut</LI></UL><P class="obj">Objective: Switch R0040-anti-tracrRNA into pSB4K5</P><P>Minipreps</P><UL><LI>pSB4K5-R0040-anti-tracrRNA (4 copies)</LI></UL><UL><LI>concentrations 204, 154.2, 206.7, and 189.3 ng/uL</LI></UL><P>Analytical digests</P><UL><LI>pSB4K5-R0040-anti-tracrRNA (4 copies)</LI></UL><UL><LI>with XbaI/PstI</LI></UL><UL><LI>See gel above for results</LI></UL><UL><LI>All 4 copies appear successful</LI></UL><P class="obj">Objective: Triple-transformation of R0040-anti-tracrRNA into Z1-reporter-crRNA_GFP cells</P><P>Prepared chemically competent cells</P><UL><LI>DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP1</LI><LI>DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP3</LI><LI>From 50 mL overnight culture each</LI><LI>Made about 10 tubes each of 750 uL</LI></UL><P>Triple Transformation</P><UL><LI>pSB4K5-R0040-anti-tracrRNA into two strains</LI></UL><UL><LI>DH5alpha-ZI + pSB6A1-K608012 + pdCas9-GFP1</LI><LI>DH5alpha-ZI + pSB6A1-K608012 + pdCas9-GFP3</LI></UL><UL><LI>Transformed via heat shock, plated on LB+Cm+Amp+Kan</LI></UL><P class="obj">Objective: Switch tracrRNA-dCas9 and tracrRNA-dCas9-crRNA into pSB1C3</P><P>Minipreps</P><UL><LI>2 copies of potential pSB1C3-tracrRNA-dCas9</LI></UL><UL><LI>concentrations 185.9, 150.7 ng/uL</LI></UL><P>Analytical digests</P><UL><LI>pSB1C3-tracrRNA-dCas9 (2 copies)</LI></UL><UL><LI>with XbaI/PstI and with XbaI/NheI</LI></UL><UL><LI>See gel above for results</LI></UL><UL><LI>neither copy appears successful</LI></UL><P>Ligations of tracrRNA-dCas9-crRNA and tracrRNA-dCas9 into pSB1C3</P><UL><LI>Two ligations of different inserts into the same backbone</LI><LI>pSB1C3 backbone 100ng = 1 uL</LI></UL><UL><LI>gel extract of XbaI/PstI digest treated with Antarctic Phosphatase</LI></UL><UL><LI>tracrRNA-dCas9-crRNA 3x molar insert 600 ng = 2uL</LI></UL><UL><LI>PCR digested with XbaI/PstI/DpnI</LI></UL><UL><LI>tracrRNA-dCas9 3x molar insert 600 ng = 2 uL</LI></UL><UL><LI>PCR digested with XbaI/PstI/DpnI</LI></UL><UL><LI>Backbone-only and two insert-only controls</LI><LI>Transformed via heat shock into DH5alpha and plated on LB+Cm</LI></UL><P class="obj">Objective: Put R0040 in front of Csy4 and Repeat scaffolds</P><P>Antarctic Phosphatase treatment</P><UL><LI>pSB1C3-R0040 cut with SpeI/PstI</LI><LI>2 copies combined into one 100 uL reaction</LI><LI>1 hr at 37C</LI></UL><P>PCR cleanup of AP treatment</P><UL><LI>Qiagen kit</LI><LI>Concentration 130 ng/uL</LI></UL><P>Ligation of Repeat-scaffold and Csy4-scaffold into pSB1C3-R0040</P><UL><LI>Two ligations of different inserts into the same backbone</LI><LI>pSB1C3-R0040 backbone 100 ng = 0.75 uL</LI></UL><UL><LI>SpeI/PstI digest treated with Antarctic Phosphatase</LI></UL><UL><LI>Repeat-seq scaffold excess insert 7.75 uL</LI></UL><UL><LI>gel extract of XbaI/PstI digest</LI></UL><UL><LI>Csy4-scaffold excess insert 7.75 uL</LI></UL><UL><LI>gel extract of XbaI/PstI digest</LI></UL><UL><LI>Backbone-only and two insert-only controls</LI><LI>Transformed via heat shock into DH5alpha and plated on LB+Cm</LI></UL><P class="obj">Objective: Obtain new backbone for tracrRNA-dCas9-crRNAs</P><P>Transformation from kit plate</P><UL><LI>Plate 4-4D: pSB3C5-J04450</LI><LI>Transformed into DH5alpha and plated on LB+Cm</LI></UL><P class="obj">Objective: Measurement interlab study</P><P>Antarctic Phosphatase treatment</P><UL><LI>pSB1C3-K823005 cut with SpeI/PstI</LI></UL><UL><LI>2 copies combined into one 100 uL reaction</LI></UL><UL><LI>pSB1C3-K823012 cut with SpeI/PstI</LI></UL><UL><LI>2 copies combined into one 100 uL reaction</LI></UL><UL><LI>&gt;1 hr at 37C</LI></UL><P>PCR cleanup of AP treatment</P><UL><LI>Qiagen kit</LI></UL></DIV></DIV><DIV class="day"><A id="aug2"><H2>August 2</H2></A><DIV class="lab"><P class="obj">Objective: Test anti-tracrRNA induction effect on reporter</P><P>Plate results:</P><UL><LI>Triple-transformation seems to have worked</LI><LI>Many colonies on both plates</LI></UL><P>Culture colonies to prepare for flow</P><UL><LI>All strains cultured in +aTc and -aTc</LI></UL><UL><LI>+aTc = 5 uL of 100 ug/mL aTc/EtOH into 5 mL media</LI></UL><UL><LI>final concentration of aTc 100 ng/mL (full induction)</LI></UL><UL><LI>-aTc = 5 uL of 100% EtOH into 5 mL media</LI></UL><UL><LI>to control for effect of solvent on cells</LI></UL><UL><LI>3 colonies each from triple transformation plates</LI></UL><UL><LI>DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP1 + R0040-anti-tracrRNA</LI><LI>DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP1 + R0040-anti-tracrRNA</LI></UL><UL><LI>DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP1 </LI></UL><UL><LI>straight from frozen stock #1</LI></UL><UL><LI>DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP3</LI></UL><UL><LI>straight from frozen stock #1</LI></UL><UL><LI>DH5alpha-ZI + pSB6A1-K608012</LI></UL><UL><LI>straight from frozen stock</LI></UL><UL><LI>DH5alpha-ZI background strain</LI></UL><UL><LI>separate colonies from streak plate</LI></UL><UL><LI>All frozen stock and transformation colonies were inoculated into 50 uL SOC, which was divided between the +aTc and -aTc samples</LI></UL><UL><LI>this ensures that +aTc and -aTc samples are from the same parent copy</LI></UL><UL><LI>All samples grown to stationary phase (~18 hrs) in LB+(appropriate antibiotics)</LI></UL><P>Streak plates</P><UL><LI>3 strains for which frozen stocks were used directly above</LI></UL><P class="obj">Objective: Put R0040 in front of Csy4 and Repeat scaffolds</P><P>Plate results:</P><UL><LI>Lots of colonies on Experimental and Backbone-only controls</LI></UL><UL><LI>Possibly more on experimentals?</LI><LI>None on insert-only control</LI></UL><P>Colony PCR</P><UL><LI>8 colonies each:</LI></UL><UL><LI>pSB1C3-R0040-Csy4-scaffold</LI><LI>pSB1C3-R0040-Repeat-scaffold</LI></UL><UL><LI>Taq polymerase, oligos SB1C3-up and SB1C3-dn</LI><LI>20 sec extension, 62C anneal temp</LI></UL><P>

Agarose gel of colony PCR results</P><UL><LI>Csy4 scaffold colonies showed clear distinction between higher and lower</LI></UL><UL><LI>copies 3,4, and 5 are promising samples</LI></UL><UL><LI>Repeat scaffold bands are harder to differentiate</LI></UL><UL><LI>copies 1,3, and 4 may be promising</LI></UL><P>Cultured colonies</P><UL><LI>pSB1C3-R0040-Csy4-scaffold #3,4,5</LI><LI>pSB1C3-R0040-Repeat-seq scaffold #1,3,4</LI></UL><P class="obj">Objective: Obtain new backbone for tracrRNA-dCas9-crRNAs</P><P>Plate results</P><UL><LI>Many colonies on plate</LI></UL><P>Cultured colonies</P><UL><LI>4 copies of pSB3C5-J04450</LI></UL><P class="obj">Objective: Measurement interlab study</P><P>Cultured colonies</P><UL><LI>4 copies of pSB1C3-E0240</LI></UL></DIV></DIV><DIV class="day"><A id="aug3"><H2>August 3</H2></A><DIV class="lab"><P class="obj">Objective: Obtain new backbone for tracrRNA-dCas9-crRNAs</P><P>Minipreps</P><UL><LI>4 copies of pSB3C5-J04450</LI><LI>Saved glycerol stock of 1 copy</LI></UL><P class="obj">Objective: Measurement interlab study</P><P>Minipreps</P><UL><LI>4 copies of pSB1C3-E0240</LI></UL><P class="obj">Objective: Put R0040 in front of Csy4 and repeat scaffolds</P><P>Minipreps</P><UL><LI>pSB1C3-R0040-Csy4-scaffold #3,4,5</LI></UL><UL><LI>saved glycerol stocks</LI></UL><UL><LI>pSB1C3-R0040-Repeat-seq scaffold #1,3,4</LI></UL><UL><LI>saved glycerol stocks</LI></UL><P class="obj"> Objective: Test anti-tracrRNA induction effect on reporter</P><P>Dilute cultures and flow cytometry</P><UL><LI>1/1000 initial dilution: 5 uL overnight culture into 5 uL media</LI><LI>media same as overnight culture, including +aTc concentrations and noninduced (NI) EtOH concentrations</LI><LI>Initial dilution was grown for 4.5 hrs at 37C</LI><LI>1/50 Final dilution: 20 uL initial dilution into 1 mL PBS, placed on ice until run</LI><LI>3 replicates of each strain</LI></UL><UL><LI>All controls were replicates taken from a single overnight culture</LI><LI>Full triple-transformation strains were 3 independent colonies</LI></UL><UL><LI>1-3: DH5alpha-ZI (background) NI</LI><LI>4-6: DH5alpha-ZI (background) +aTc</LI><LI>7-9: DH5alpha-ZI + pSB6A1-K608012 NI</LI><LI>10-12: DH5alpha-ZI + pSB6A1-K608012 +aTc</LI><LI>13-15: DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP1 NI</LI><LI>16-18: DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP1 +aTc</LI><LI>19-21: DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP3 NI</LI><LI>22-24: DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP3 +aTc</LI><LI>25-27: DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP1 + pSB1C3-R0040-anti-tracrRNA NI</LI><LI>28-30: DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP1 + pSB1C3-R0040-anti-tracrRNA +aTc</LI><LI>31-33: DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP3 + pSB1C3-R0040-anti-tracrRNA NI</LI><LI>34-36: DH5alpha-ZI + pSB6A1-K608012 + pdCas9_GFP3 + pSB1C3-R0040-anti-tracrRNA +aTc</LI></UL></DIV></DIV><DIV class="day"><A id="aug4"><H2>August 4</H2></A><DIV class="lab"><P class="obj">Objective: Put R0040 promoter in front of Csy4 and repeat scaffolds</P><P>Analytical digests</P><UL><LI>Plasmids from 8/3 minipreps</LI><LI>pSB1C3-R0040-Csy4-scaffold (3 copies)</LI><LI>pSB1C3-R0040-Repeat-scaffold (3 copies)</LI><LI>All digested with BsaI/EcoRV, as well as with XbaI/PstI</LI><LI>1.5 hrs at 37C</LI></UL><P>Agarose gel of digest results</P><UL><LI>Gel I:</LI></UL><UL><LI>1-3. pSB1C3-R0040-Repeat-scaffold, copies 1,3,4, BsaI/EcoRV</LI><LI>4-6. pSB1C3-R0040-Csy4-scaffold, copies 3,4,5, BsaI/EcoRV</LI><LI>7. Sample for Tony Burnetti</LI></UL><UL><LI>Gel 2: </LI></UL><UL><LI>1-3. pSB1C3-R0040-Repeat-scaffold, copies 1,3,4, XbaI/PstI</LI><LI>4-6. pSB1C3-R0040-Csy4-scaffold, copies 3,4,5, XbaI/PstI</LI></UL><UL><LI>Results: all samples look correct</LI></UL><P class="obj">Objective: Switch R0040-anti-tracrRNA into pSB3C5</P><P>Prep-scale digests</P><UL><LI>Plasmids from 8/3 minipreps</LI><LI>pSB3C5-J04450 (4 copies) with SpeI/EcoRI</LI><LI>Each miniprep tube used in a 100 uL reaction </LI></UL><P>Gel extraction and cleanup of digests</P><UL><LI>Loaded 200 uL (two reaction tubes) per lane</LI><LI>Gel 4: </LI></UL><UL><LI>ladder in between two lanes of pSB3C5-J04450</LI></UL><UL><LI>Extracted top band (pSB3C5) from both lanes</LI><LI>Zymoclean prep kit</LI><LI>Final concentrations: 325 ng/uL both tubes in 30 uL each</LI></UL><P class="obj">Objective: Measurement interlab study</P><P>Prep-scale digests</P><UL><LI>Plasmids from 8/3 minipreps</LI><LI>pSB1C3-E0240 (4 copies) with XbaI/PstI</LI><LI>Each miniprep tube used in a 100 uL reaction </LI></UL><P>Gel extraction and cleanup of digests</P><UL><LI>Loaded 200 uL (two reaction tubes) per lane</LI><LI>Gel 3: </LI></UL><UL><LI>lanes 1 and 3: pSB1C3-E0240</LI><LI>middle lane blank</LI></UL><UL><LI>Extracted lower band (E0240) from both lanes</LI><LI>Zymoclean prep kit</LI><LI>Final concentrations: 75 ng/uL both tubes in 30 uL each</LI></UL></DIV></DIV><DIV class="day"><A id="aug5"><H2>August 5</H2></A><DIV class="lab"><P>DUGSIM Order 1699</P><UL><LI>BigDye Version 1.1</LI><LI>1-2: pSB1C3-R0011 (from frozen stock) with SB1C3-up, SB1C3-dn</LI><LI>3-4: pSB1C3-R0011 #1 with SB1C3-up, SB1C3-dn</LI><LI>5-6: pSB1C3-R0040-Csy4-scaffold #3 with SB1C3-up, SB1C3-dn</LI><LI>7-8: pSB1C3-R0040-Csy4-scaffold #4 with SB1C3-up, SB1C3-dn</LI><LI>9-10: pSB1C3-R0040-Repeat-scaffold #1 with SB1C3-up, SB1C3-dn</LI><LI>11-12: pSB1C3-R0040-Repeat-scaffold #3 with SB1C3-up, SB1C3-dn</LI><LI>Results:</LI></UL><UL><LI>R0011 stocks are not correct</LI><LI>Both scaffolds are correct with Tet promoters (all copies)</LI></UL><P class="obj">Objective: Switch R0040-anti-tracrRNA into pSB3C5</P><P>Antarctic Phosphatase treatment</P><UL><LI>pSB3C5 gel extracts from 8/4 </LI><LI>Two tubes combined into one reaction of 100 uL</LI><LI>2 hrs at 37C</LI></UL><P>PCR Cleanup of AP treatment</P><UL><LI>Qiagen kit</LI><LI>Final concentration 323.8 ng/uL in 30 uL</LI></UL><P>Ligation of R0040-anti-tracrRNA into pSB3C5</P><UL><LI>pSB3C5 backbone SpeI/EcoRI/AP: 100 ng = 0.3 uL</LI><LI>R0040-anti-tracrRNA insert SpeI/EcoRI: 30 ng = 0.3 uL</LI><LI>Backbone-only and insert-only controls</LI><LI>T4 Ligase for 45 minutes at rt, then transformed via heat shock</LI><LI>plated on LB+Cm</LI></UL><P class="obj">Objective: Switch tracrRNA-dCas9 and tracrRNA-dCas9-crRNA into pSB1C3</P><P>Ligation of tracrRNA-dCas9 and tracrRNA-dCas9-crRNA into pSB1C3</P><UL><LI>2 “inserts” into the same “backbone”</LI><LI>Used 3x molar “backbone” pSB1C3 because insert is larger</LI><LI>pSB1C3 backbone XbaI/PstI: 150 ng = 1.5 uL</LI><LI>tracrRNA-dCas9 insert XbaI/PstI/DpnI: 100 ng = 0.3 uL</LI><LI>tracrRNA-dCas9-crRNA insert XbaI/PstI/DpnI: 100 ng = 0.3 uL</LI><LI>Backbone-only and 2 insert-only controls</LI><LI>T4 Ligase for 45 minutes at rt, then transformed via heat shock</LI><LI>plated on LB+Cm</LI></UL><P class="obj">Objective: Measurement interlab study</P><P>Ligation of E0240 into pSB1C3-K823005 and into pSB1C3-K823012</P><UL><LI>2 backbones with the same insert</LI><LI>pSB1C3-K823005 backbone SpeI/PstI/AP: 100 ng = 0.33 uL</LI><LI>pSB1C3-K823012 backbone SpeI/PstI/AP: 100 ng = 0.66 uL</LI><LI>E0240 insert XbaI/PstI: 150 ng = 2 uL</LI><LI>2 Backbone-only and 1 insert-only controls</LI><LI>T4 Ligase for 45 minutes at rt, then transformed via heat shock</LI><LI>plated on LB+Cm</LI></UL><P class="obj">Objective: Swapping various backbones</P><P>Cultured colonies</P><UL><LI>6 colonies of pSB1A2-R0011 (transformed from 2013 kit plate)</LI><LI>4 copies of pSB4K5-J04450</LI><LI>4 copies of pSB1C3-R0040-anti-tracrRNA</LI></UL></DIV></DIV><DIV class="day"><A id="aug6"><H2>August 6</H2></A><DIV class="lab"><P class="obj">Objective: Swapping various backbones</P><P>Minipreps</P><UL><LI>6 colonies of pSB1A2-R0011 (transformed from 2013 kit plate)</LI><LI>4 copies of pSB4K5-J04450</LI><LI>4 copies of pSB1C3-R0040-anti-tracrRNA</LI></UL><P class="obj">Objective: Switching R0040-anti-tracrRNA into pSB3C5</P><P>Plate results:</P><UL><LI>1 colony on the experimental plate</LI><LI>0 colonies on backbone-only control</LI><LI>~25 colonies on insert-only control</LI></UL><UL><LI>Is is possible that naming got switched? This result is abnormal</LI></UL><P>Cultured colonies:</P><UL><LI>One colony from ligation plate of pSB3C5-R0040-anti-tracrRNA</LI><LI>3 colonies from insert-only plate</LI></UL><P class="obj">Objective: Switch tracrRNA-dCas9 and tracrRNA-dCas9-crRNA into pSB1C3</P><P>Plate results: </P><UL><LI>No colonies on pSB1C3-tracrRNA-dCas9-crRNA ligation plate</LI><LI>1 colony on pSB1C3-tracrRNA-dCas9 ligation plate</LI><LI>2 colonies on backbone-only control, none on insert-only controls</LI></UL><P>Cultured colony</P><UL><LI>One colony from pSB1C3-tracrRNA-dCas9 ligation</LI></UL><P class="obj">Objective: Measurement interlab study</P><P>Plate results:</P><UL><LI>Hundreds of colonies on both experimental and both backbone-only plates</LI><LI>No colonies on insert-only controls</LI></UL><P>Colony PCR</P><UL><LI>8 colonies each from pSB1C3-K823005-E0240 and pSB1C3-K823012-E0240</LI><LI>Taq polymerase with oligos SB1C3-up and SB1C3-dn</LI><LI>Anneal temp 62C, extension time 45 sec</LI></UL><P>Agarose gel of Colony PCR results</P><UL><LI>1-8: pSB1C3-K823005-E0240</LI><LI>9-16: pSB1C3-K823012-E0240</LI><LI>Results: no successes, looks like clear amplification of promoter-only</LI></UL></DIV></DIV><DIV class="day"><A id="aug7"><H2>August 7</H2></A><DIV class="lab"><P class="obj">Objective: Backbone switching</P><P>Minipreps abandoned</P><UL><LI>Accidentally poured off the DNA-containing liquid into the waste container</LI><LI>Frozen stocks were saved--those can be grown to miniprep another day</LI></UL><UL><LI>pSB1C3-tracrRNA-dCas9 ligation (1 copy, need to test)</LI><LI>pSB3C5-R0040-anti-tracrRNA (4 copies, need to test)</LI></UL><UL><LI>More LB+Cm was added to empty culture tubes and grown at 37C</LI></UL><UL><LI>An attempt to salvage the minipreps--probably not useful</LI></UL><P class="obj">Objective: Sequencing</P><P>Order 1708</P><UL><LI>Big Dye version 1.1</LI><LI>1. pSB1C3-R0040-anti-tracrRNA-1 with SB1C3-up</LI><LI>2. pSB1C3-R0040-anti-tracrRNA-1 with SB1C3-dn</LI><LI>3. pSB1C3-R0040-anti-tracrRNA-2 with SB1C3-up</LI><LI>4. pSB1C3-R0040-anti-tracrRNA-2 with SB1C3-dn</LI></UL></DIV></DIV><DIV class="day"><A id="aug8"><H2>August 8</H2></A><DIV class="obj">Objective: Backbone switching</DIV><DIV class="ppl">Delta Ghoshal, Matthew Faw</DIV><DIV class="lab"><P>Redo minipreps from yesterday - culture tubes grew overnight from salvaged cells</P><UL><LI>pSB1C3-dCas9-tracrRNA    - 171.3 ng/ul</LI><LI>pSB3C5-R0040-a-tracr 1    - 257.5 ng/ul</LI><LI>pSB3C5-R0040-a-tracr 2    - 202.7 ng/ul</LI><LI>pSB3C5-R0040-a-tracr 3    - 200.5 ng/ul</LI><LI>pSB3C5-R0040-a-tracr 4    - 198.3 ng/ul</LI></UL><P>Analytical digest of Tube 1 (pSB1C3-dCas9-tracrRNA) using XhoI and BamHI</P><UL><LI>2 hour digest at 37 degrees C</LI><LI>3X Master Mix:</LI><UL><LI>3 ul Cutsmart</LI><LI>22.5 ul water</LI><LI>0.75 ul XhoI</LI><LI>0.75 ul BamHI</LI></UL><LI>9 ul Master Mix added to 1 ul of pdCas9 in one tube and 1 ul of pSB1C3-dCas9-tracrRNA in another tube</LI><LI>Gel results were consistent with an unsuccessful ligation. 3 bands in the experimental column (4kb, 1.8 kb, and 0.9 kb) were expected. The results were more consistent with pSB1C3 and no insert. The band size in the pdCas9 control lane also looked wrong because it was too big. The pdCas9 tube may have been faulty or the BamHI might have been faulty.</LI></UL></DIV></DIV><DIV class="day"><A id="aug11"><H2>August 11</H2></A><DIV class="obj">Objective: Prepare chemically competent cells</DIV><DIV class="ppl">Delta Ghoshal</DIV><DIV class="lab"><P>The two types of cells to make competent were DH5-aZ1 cells containing the reporter gene and cells with the reporter and GFP1 site. Charlie started cultures on Saturday evening from a plate into LB + antibiotics. The antibiotics used were ampicillin in the reporter-only cells and ampicillin plus chloramphenicol in the reporter/GFP1 cells. On Sunday morning, he put them in the cold room and then took the OD that evening by adding 100 uL of culture to 900 uL broth plus antibiotics:</P><UL><LI>R: 0.276</LI><LI>R+G1: 0.268</LI></UL><P>He then started four cultures by adding 120 mL LB/appropriate antibiotic(s) to the following and growing them at 24 C:</P><UL><LI>70 uL R</LI><LI>0.7 mL R (denser culture) </LI><LI>70 uL R+G1</LI><LI>0.7 mL R+G1 (denser culture)</LI></UL><P> Today, at 10:25 AM, OD was measured again, but undiluted.</P><UL><LI> dense R: 0.023 </LI><LI> R: -0.001 </LI><LI> dense R+G1: 0.0037 </LI><LI> R+G1: -0.000 </LI></UL><P> Measurements were repeated at 12:34 PM, but only on the denser cultures. </P><UL><LI> dense R: 0.038 </LI><LI> dense R+G1: 0.061 </LI></UL><P> The temperature was increased to improve doubling time. </P></DIV></DIV><DIV class="day"><A id="aug12"><H2>August 12 </H2></A><DIV class="ppl">Delta Ghoshal, Charlie Cooper </DIV><DIV class="lab"><P class="obj"> Objective: Analyze gel from 8/11/14's analytical digest </P><UL><LI>Lanes 1-4 (pSB1C3-R0040-antitracr) identical and consistent with expected results (2000 and 200 bp bands) </LI><LI> Lanes 5-8 (pSB3C5-R0040-antitracr) not identical but should be. Expected was 2.7 kb, 36 bp, and 163 bp bands. 163 and 36 bp bands may blur together </LI><UL><LI> Lanes 5 and 7 had a ~3 kb band and a ~200 kb band, which is unexpected </LI><LI> Lanes 6 and 8 may have worked. </LI></UL><LI> Next steps: A second analytical digest to confirm these ambiguous results </LI></UL><P class="obj"> Objective: Perform a second analytical digest using AgeI and XhoI</P><UL><LI> Gel Design:
</LI><LI> Lane 1: pSB1C3-R0040-antitracr, tube #1 (from 8/5/14); Expected results: 892 bp and 1353 bp bands </LI><LI> Lane 2: pSB3C5-R0040-antitracr, tube #1 (from 8/7/14); Expected results: 323 bp and 2590 bp bands </LI><LI> Lane 3: pSB3C5-R0040-antitracr, tube #2 (from 8/7/14); Expected results: 323 bp and 2590 bp bands </LI><LI> Lane 4: pSB3C5-R0040-antitracr, tube #3 (from 8/7/14); Expected results: 323 bp and 2590 bp bands </LI><LI> Lane 5: pSB3C5-R0040-antitracr, tube #4 (from 8/7/14); Expected results: 323 bp and 2590 bp bands </LI></UL><LI> 6X Master Mix:
</LI><LI> 6 uL Cutsmart </LI><LI>1.5 uL AgeI </LI><LI>1.5 uL XhoI </LI><LI> 45 uL dH2O </LI><LI> 9 uL MM added to 1 uL DNA at 37 C </LI><LI> Results: Tube #1 of pSB3C5-R0040-antitracr had the expected results, so the remaining tubes 2-4 were discarded </LI><P class="obj"> Objective: Transform 8/11/14's CCEC with pSB4K5-J04450 </P><UL><LI> Used tube #4 of pSB4K5-J04450 from 8/6/14 : 135.8 ng/uL concentration </LI><LI> 10X serial dilution of DNA: 5 uL previous tube + 45 uL EB </LI><UL><LI> 6 transformants were 10^-1, 10^-2, 10^-3, 10^-4, 10^-5, and EB only </LI><LI> Cells were plated using appropriate antibiotics (Amp + Kan for Reporter only, Amp + Kan + Cm for Reporter+GFP1 cells) </LI></UL></UL></DIV></DIV><DIV class="day"><A id="aug13"><H2>August 13</H2></A><DIV class="ppl">Delta Ghoshal, Charlie Cooper </DIV><DIV class="lab"><P class="obj">Objective: Analyze yesterday's transformation results </P><UL><LI> pink lawn on 10^-1 R and G plates, no colonies on other plates; unexpected </LI><LI> expected: 10-fold reduction in colonies on each successive dilution plate </LI><LI> could be explained by pipetting errors </LI><LI> Next steps: repeat transformation with following modifications: </LI><UL><LI> Increase recovery time in SOC from 1 hour to 2 hours </LI><LI> Introduce antibiotics to the SOC media (Amp for R cells, Amp + Cm for G cells) </LI><LI> Use a 5-fold dilution instead of a 10-fold dilution </LI><LI> Instead of adding cells to DNA, add DNA to cells </LI><LI> Vortex the dilutions before adding them to the cell tubes </LI></UL></UL><P class="obj">Objective: Retry transformation </P><UL><LI> Tube #4 of pSB4K5-J04450 was used again.</LI><LI> The 6 transformants were:
</LI><LI>10X dilution : 3 uL DNA + 27 uL EB </LI><LI> 5X dilution of 10X: 5 uL previous tube + 20 uL EB </LI><LI> 5X dilution of (10^-1*5^-1): 5 uL previous tube + 20 uL EB </LI><LI> 5X dilution of (10^-1*5^-2): 5 uL previous tube + 20 uL EB </LI><LI> 5X dilution of (10^-1*5^-3): 5 uL previous tube + 20 uL EB </LI><LI> EB only </LI></UL></DIV></DIV><DIV class="day"><A id="aug14"><H2>August 14</H2></A><DIV class="ppl">Delta Ghoshal, Charlie Cooper </DIV><DIV class="lab"><P class="obj"> Objective: Analyze yesterday's transformation </P><UL><LI> Success: All plates except EB had colonies that were faintly pink </LI></UL><TABLE><TBODY><TR><TD>Dilution</TD><TD> Reporter Only </TD><TD> Reporter+GFP1</TD></TR><TR><TD>10^-1</TD><TD> lawn </TD><TD> lawn</TD></TR><TR><TD>10^-1*5^-1</TD><TD>lawn</TD><TD>lawn</TD></TR><TR><TD>10^-1*5^-2</TD><TD>lawn</TD><TD>lawn</TD></TR><TR><TD>10^-1*5^-3</TD><TD>dense colonies</TD><TD>dense colonies</TD></TR><TR><TD>10^-1*5^-4</TD><TD> 1530 colonies </TD><TD>2402 colonies</TD></TR><TR><TD>EB only</TD><TD> none </TD><TD> none </TD></TR></TBODY></TABLE>
Next steps: transforming again with more dilutions, to eliminate possibility of pipetting errors skewing results. Starting from the lowest concentration from above (10^-1*5^-4), we will perform 10X dilutions.
<P class="obj"> Objective: Transform cells with highly dilute DNA </P><UL><LI> Accidentally left tube #4 of pSB4K5-J04450 on benchtop overnight, so used tube #1 instead, just in case. Concentration: 162.1 ng/uL </LI><LI> Dilutions: </LI><UL><LI> 10^-1: 5 uL tube #1 DNA + 45 uL EB </LI><LI>10^-1*5^-4: 5 uL previous tube to 625 uL EB </LI><LI>10^-2*5^-4: 5 uL previous tube to 45 uL EB </LI><LI>10^-3*5^-4: 5 uL previous tube to 45 uL EB </LI><LI>10^-4*5^-4: 5 uL previous tube to 45 uL EB </LI></UL><LI> Transformation was done using the lowest 4 concentrations of DNA (not 10^-1). No control was tested, and yesterday's (8/13/14) modified transformation protocol was used. </LI></UL></DIV></DIV><DIV class="day"><A id="aug15"><H2>August 15</H2></A><DIV class="ppl">Delta Ghoshal, Charlie Cooper </DIV><DIV class="lab"><P class="obj"> Objective: Analyze results of diluted transformation </P><UL><LI> The transformation was successful; each plate had fewer colonies than the previous.</LI></UL><TABLE><TBODY><TR><TD>Dilution:</TD><TD>10^-1*5^-4</TD><TD>10^-2*5^-4  </TD><TD>10^-3*5^-4 </TD><TD>10^-4*5^-4</TD></TR><TR><TD>Reporter Only:   </TD><TD> dense colonies</TD><TD>905 colonies  </TD><TD>134 colonies</TD><TD>20 colonies</TD></TR><TR><TD>Reporter+GFP1: </TD><TD> dense colonies</TD><TD>1280 colonies</TD><TD>108 colonies</TD><TD>24 colonies</TD></TR></TBODY></TABLE><P class="obj"> Objective: Test validity of molecular titration approach using anti-tracr oligos and flow cytometry</P>
Oligos ordered on Wednesday arrived today: 2 tubes of both antitracrDNA and not-antitracrDNA.

Calculations:
<P>antitracrDNA	aaaaagcaccgactcggtgccactttttcaagttgataacggactagccttattttaacttgctatgctgttttgaatggttccaac </P><P>notantitracr	acaattttaggcttttgatgtgttgtcgtaatctgtaccacaattgcatatattgagaaccaagctactgaagcatcctttcccggc </P><P>

%%

cells_per_tube_R = .086 * 5 * 8E8 / 200; % http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp?_requestid=290172

cells_per_tube_RG = .0859 * 5 * 8E8 / 200;

 

d = 162.1; % ng/uL original tube

d1 = 16.21;

d2 = d1/5^4;

 

ngDNA = [d2 d2/1E1 d2/1E2 d2/1E3]; % ng in each transformation tube

pmolDNA = ngDNA * 1E-3 / 660 * 1E6 / 5578; % http://www.promega.com/a/apps/biomath/index.html?calc=ugpmols

</P>
One tube of each was resuspended in dH2O: 82 uL in antitracr (A) and 71.4 uL in not-antitracr (NA). Then, 2 uL of each oligo were transformed into both R and G cells, using 8/14/14's protocol without recovery time. Starting at t=0 minutes, samples were taken and placed on ice in cold room every 30 minutes, from t=0 to t=240 (9 time points). Flow cytometry was used to analyze the expression of GFP.

<TABLE><TBODY><TR align="center" colspan="3"><TD>Expected results:</TD></TR><TR><TD> Reporter Only Cells </TD><TD> Reporter + GFP1 Cells </TD></TR><TR><TD>A</TD><TD> GFP expression </TD><TD> GFP expression </TD></TR><TR><TD>NA</TD><TD> GFP expression</TD><TD>No GFP expression </TD></TR></TBODY></TABLE>

The results differed from the expected, which is a preliminary indicator that this approach may not work. The flow results showed that G cells were in fact repressing some of their GFP expression, but both strains showed an overall increase in fluorescence over time. Also, there was no difference in results between the cells transformed with A and with NA. The experiment will be repeated again in the future, possibly with the following modifications:
<UL><LI> Samples taken every hour, not half hour </LI><LI> Samples collected all day so there are more data points </LI><LI> Samples analyzed using flow cytometry immediately, instead of after being placed in cold room </LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Duke/Notebook/August">http://2014.igem.org/Team:Duke/Notebook/August</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Duke/Notebook/August" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Duke/Notebook/August" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Duke/Notebook/August&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Duke/Notebook/August&amp;oldid=302677" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>