<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Duke_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Duke/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Duke/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Duke/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Duke/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><UL id="navbar"><LI><A href="https://2014.igem.org/Team:Duke">Home</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team">About Us</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Duke">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team/Gallery">Photo Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team/Attributions">Attributions</A></LI></UL><LI><A href="https://2014.igem.org/Team:Duke/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#summary">Summary</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#future">Future Directions</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Overview">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Months">Notebook by Month</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy">Policy and Practice</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#threed">3D Printing</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#hoco">House Course</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#smath">NCSSM Team</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Safety">Safety</A></LI></P><DIV class="sec" id="background"><H2> Background Information </H2><H3> CRISPR/Cas9 System </H3><P>Clustered Regularly Interspersed Short Palindromic Repeats (CRISPRs) are a form of bacterial defense against foreign invaders. When foreign (viral) DNA enters the cell, its DNA is incorporated into the spaces between the CRISPRs.  This DNA is transcribed into a long RNA piece. &quot;CRISPR-associated&quot; (Cas) riboendonucleases cleave (cut) the RNA at the repeat sites to form CRISPR RNAs, or (crRNA). The crRNAs guide the Cas protein to to the target foreign DNA (aka the protospacer), which then uses its endonuclease activity to cleave the DNA and make it inactive. </P><P>The type of Cas protein we use is called Cas9. It cleaves both strands of DNA, but it requires two conditions for it to work: </P><P><OL><LI>The guide crRNA must match the sequence of the protospacer </LI><LI>presence of a protospacer-adjacent motif (PAM) downstream of the protospacer </LI></OL></P><P> dCas9 is a form of the Cas9 protein that lacks endonuclease capability. Instead, it just sits on the target site of the DNA. This blocks transcription of the DNA by RNA polymerase, thereby inhibiting gene expression. This is a way of modifying gene expression without modifying the genome by utilizing machinery that is already present in the bacteria. </P></DIV><DIV class="sec" id="summary"><H2> Project Summary </H2><P> The goal of this year’s project is to produce an ultrasensitive response. Ultrasensitive responses are defined as such because the level of response is highly dependent on the amount and concentration of the inducer, rather than other variables. Ultrasensitivity has implications for several biological applications, including bistable biological “toggle switches” within the genome. </P><P> There are two approaches we are taking to achieve ultrasensitivity. </P><OL><LI> We are designing a 2-reporter construct, where induction of one reporter triggers repression of the other. We built mathematical models of multiple repressors of a single gene and showed that it can result in ultrasensitivity. We have devised a construction scheme to implement this strategy, and realization of these constructs in underway.</LI><LI> Molecular titration: It has been shown by (Buchler and Louis 2008) and others that competitive binding can result in ultrasensitivity in a phenomenon known as molecular titration. Recall that the dCas9 protein requires two pieces of RNA to work properly: the sequence-specific crRNA that leads the protein to the intended target, as well as the tracrRNA that helps stabilize the structure and crRNA. We propose that by adding in RNA that is complementary to the tracrRNA, which we dub &quot;anti-tracrRNA&quot;, we can repress the repression carried out by dCas9.  It is also possible to implement this using RNAs that titrate sequence specific crRNAs or the dCas9 protein. Titrating out any of the necessary components for repression would inactivate the system and cause the expression of previously-repressed GFP, causing a quantifiable increase in GFP expression. We tested two approaches to take advantage of this phenomenon: to titrate tracrRNA with complementary RNA, or to titrate out the fully assembled pdCas9:tracrRNA:crRNA complex with decoy binding sites. We created mathematical models demonstrating the feasibility of both approaches and tested them in vivo. Our work shows that Anti-tracrRNA is does not derepress dCas9-mediated repression of transcription, but decoy binding sites are working as expected and shows promise for a means of modulating transcriptional control.
 </LI></OL><P>
The second approach uses the concept of &quot;molecular titration&quot;. This experiment involves having GFP repressed by dCas9 using complementary crRNAs, similar to Approach 1 above. Then, if the cell also has a plasmid containing the sequence for the anti-tracrRNA, this RNA will bind to the tracrRNA and &quot;sequester&quot; it, or make it unavailable to bind with dCas9. This would inactivate the system and cause the expression of previously-repressed GFP, causing a quantifiable increase in GFP expression. The ultrasensitivity aspect of this approach involves the concentration of anti-tracrRNA, especially in comparison to the concentration of tracrRNA. This is another reason why this approach is known as &quot;molecular titration&quot;, because it's conceptually very similar to the technique used in chemistry.
</P></DIV><DIV class="sec" id="results"><H2> Results </H2><P>We have designed a cloning approach to create plasmids that enable cooperative repression by dCas9. We have several intermediate constructs in hand and assembly of the final system is underway. This will allow us to test the predictions of ultrasensitive repression made by our mathematical modeling.</P><H3>Molecular titration with anti-tracrRNA. </H3><P>Modeling indicated that titrating out RNA could generate ultrasensitivity. To implement this, we constructed pTet-controlled antitracrRNA, which encodes the perfect reverse complement to tracrRNA. We predict that this would base pair with tracrRNA, thereby preventing its association with dCas9 and formation of an active repressive complex. </P><P>We assembled pSB1A2-R0040-antitracrRNA and tested it by transforming bacteria with this plasmid, a reporter (pSB4K5-K608012) and either blank pdCas9 or pdCas9-GFP1 which we confirmed earlier to successfully repress GFP production. We grew these bacteria in the presence or absence of aTc, a small molecule inducer of pTet, and measured fluorescence by flow cytometry after 5 hours. We gated for singlets and normalized fluorescence by forward scatter, an approximation of cell size, and compared the fluorescence levels in the plot below. </P><P>Contrary to expectations, it appears that cells with GFP-targeting crRNA have slightly lower fluorescence when antitracr is induced than when uninduced. We also performed a time course to observe how fluorescence changes over time in these strains.</P><P>Again, we see lower fluorescence in aTc in the GFP-repressing strains. The increase in fluorescence over time is likely from the bacteria recovering from stationary phase from the overnight pre-growth before diluting in to +/- aTc. One possible explanation for this is that aTc is inhibiting protein production generally, as Tetracycline antibiotics work by blocking translation elongation, but this seems unlikely because that effect should also be apparent in the nonrepressive pdCas9 strains. Another possibility is that pTet is not expressing strongly when induced, or that aTc is changing expression of tracrRNA from pdCas9. The latter is a possibility because there is a Tet promoter on pdCas9 upstream and on the opposite strand as tracrRNA. </P><P>To test these hypotheses, we extracted RNA from induced and uninduced bacteria with no plasmid, pdCas9, or pSB1A2-R0040-antitracr, and performed RT-PCR with oligos specific to tracr/antitracrRNA.</P><P>Left to right: </P><UL><LI>0 ladder</LI><LI>1 no template + RT</LI><LI>2 ng pdCas9 template (DNA) + RT</LI><LI>Z1 +aTc + RT</LI><LI>Z1 -aTc + RT</LI><LI>Z1+1A2-R0040-antitracr +aTc + RT</LI><LI>Z1+1A2-R0040-antitracr -aTc + RT</LI><LI>Z1+pdCas9 +aTc, Z1+pdCas9 -aTc + RT</LI><LI>Z1 +aTc - RT</LI><LI>Z1 -aTc - RT</LI><LI>Z1+1A2-R0040-antitracr +aTc - RT</LI><LI>Z1+1A2-R0040-antitracr -aTc - RT</LI><LI>Z1+pdCas9 +aTc, Z1+pdCas9 -aTc - RT</LI></UL><P>Rough quantification by densitometry yields the following plot: </P><P>We can see that there is some DNA contamination in our preps from the weak band in the no-RT lanes, but the +RT bands with +/- aTc indicate that indeed pTet is producing antitracrRNA and that tracrRNA originating from pdCas9 is not changing with the addition of aTc. </P><P>
We also tested whether we could achieve derepression by titrating tracrRNA by ordering DNA oligonucleotides perfectly complementary to tracr (antitracrDNA) or with the same base composition but randomized. A similar approach was used successfully to alter regulation by microRNAs in <A href="http://www.ncbi.nlm.nih.gov/pubmed/21857679">Mukherji et al</A>. When heat shocked with antitracrDNA oligos or scrambled controls, competent cells with pSB4K5-K608012 and pdCas9-GFP1 showed no difference in fluorescence between the strains [data not shown]. 
</P><P>From this we conclude that either antitracrRNA is not present at sufficient levels to titrate away tracrRNA, or that it is not interacting with tracrRNA at all.</P><H3>Molecular titration with decoy binding sites</H3><P>We built arrays of decoy binding sites designed to bind to dCas9 proteins guided by GFP1 (targeting sequence CCATCTAATTCAACAAGAATTGG from BBa_K608012). Using a PCR-based approach we successfully assembled 1x and 6x tandem GFP1 decoys, and doubled the 6x construct by taking advantage of SpeI/XbaI compatibility. When placed on pSB1A2 and introduced in to bacteria with pSB4K5-K608012 and pdCas9-GFP1 or blank pdCas9 lacking the GFP1 crRNA targeting sequence, we see a decrease in fluorescence in proportion to the number of binding sites on the plasmid.</P><P>The 1x, 6x, and 12x GFP1 decoy binding site arrays were submitted as parts BBa_K1545000, BBa_K1545001, and BBa_K1545002. </P></DIV><DIV class="sec" id="future"><H2> Future Directions </H2><P>We have DNA in hand for the assembly of dCas9 targeting multiple binding sites and construction is currently underway. We hope to test these in the near future. </P><P>For molecular titration, with antitracrRNA, multiple lines of evidence suggest that antitracr is failing to derepress GFP expression. This suggests that it may be better to try titrating other components of dCas9 repression. Expressing a gRNA, a fusion of tracrRNA and crRNA, might work to titrate out the protein component of dCas9-mediated repression rather than an RNA. Titrating with anti-crRNA is another alternative that has the advantage of having specificity to the crRNA rather than targeting tracrRNA, which is needed regardless of the sequence being targeted for repression. This would be better for building scalable gene circuits. </P><P>Our work with decoy binding sites is very promising. We have demonstrated successful derepression of gene expression, so in the future we will explore this further. Immediate next steps include expanding the decoy arrays to achieve full derepression and to look for the ultrasensitive regime. Additionally, modeling in (Buchler and Louis, 2008) demonstrated that ultrasensitivity is strongest when binding to the negative element (decoy binding sites in this case) is strong. We plan to engineer this factor in to system by introducing intentional mismatches in the crRNA with its bona fide binding site, but maintaining a perfect match with the decoy array. </P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Duke/Project">http://2014.igem.org/Team:Duke/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Duke/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Duke/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Duke/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Duke/Project&amp;oldid=392388" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>