"
Page
Discussion
View source
History
teams
Log in
 
Team:Duke/Notebook/October
From 2014.igem.org
Home
Team
About Us
Official Team Profile
Photo Gallery
Attributions
Project
Background
Summary
Modeling
Results
Future Directions
Parts
Notebook
Overview
Notebook by Month
Protocols
Policy and Practice
3D Printing
House Course
NCSSM Team
Safety
October 1
Objective: sequence confirm 1C3-2x5xGFP
Sequenced with pSB1C3-up-1 and -dn-1, and both clones look like they have a duplication of 6x, rather than 5x. That means that clone 43 from yesterday is not useful so it will be thrown away.
Objective: test RT-PCR primers
MZ setting up two test PCRs
4 samples of tracr amplification on pdcas9 - 0, 0.3, 0.6, 0.9 ng/uL tubes, 4 samples of anti-tracr amplification on pdcas9
results: no good
We can guess at what’s happening here, but probably the first thing to do is just to try it again.
October 2
Objective: confirm 1A2-2x5x
Picked 6 colonies and subjecting to EcoRI/PstI digestion
Left to right are clones 1-6 of 2x5x, rightmost = 1A2-5x. Looks perfect, freezing clones 1 and 2 and will sequence this weekend.
Objective: extract RNA and test for expression & aTc response of tracr, antitracr
http://openwetware.org/wiki/Sauer:RNA_Purification_from_E._coli
TYler of Gersbach lab recommends RNAzap, lab coat, holding breath, filter tips. Random hexamers are better if you want to RT everything which can be more generically useful than specific primers for this step. He uses hexamers for RT step then does specific amplification following that.         
Last night started from frozen some Z1 with pdCas9 and 1A2-R0040-antitracr in LB + antibioic. This morning measured 1/10 diluted OD600 and calculated amount to add to 1) balance cultures ODs and 2) dilute back to a sufficient degree that after several generations in +/- aTc the cultures would not be at too high a density to easily harvest the max number of cells called for by the agilent RNA purification kit. This means diluting to the equivalent of 1/1000 from a culture of OD600=3.0
http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp  for OD -> cell count calculations
http://www.chem-agilent.com/pdf/strata/400800.pdf
manual
http://cp.literature.agilent.com/litweb/pdf/5990-5152EN.pdf
Has details for lysozyme treatment of coli before following the rest of the lysis/extraction protocol. This one might be just for the microprep kit, rather than the miniprep kit which we have.
1 mg/mL lysozyme, ethanol dilutions were prepared with filtered EtOH and/or water. Not DEPC treated or certified RNAse free.
Kind of a big fucking mess. Centrifuge at CBC’s bench is broken so rna back and forth between Mert’s bench to do centrifugations. Lids broke off so the real ID of some samples isn’t actually known, and accidentally made too little lysis buffer so one sample is just gone. Ended up with low 20s to 50s of some nucleic acid as measured by ND but all was pretty contaminated with ethanol, or whatever it is that gives a low 260/230 ratio. Will try this again.
Also the pdCas9 dilutions grew less quickly than did the 1A2-R0040antitracr ones
~.065 after 3h in new medium pdCas9
~.112 after 3h in new medium 1A2-R0040antitracr
Tomorrow AM: retry test PCR on DNA with the anti/tracr oligos.
Objective: make more decoy strains for testing
Shocking Z1 with
pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-2x5xGFPdecoy-1
pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-B0011 (0x)
pdCas9, pSB4K5-K(gfp), pSB1A2-2x5xGFPdecoy-1
pdCas9, pSB4K5-K(gfp), pSB1A2-B0011 (0x)
October 3
Objective: test PCR of tracr & antitracr oligos on pdCas9
tracr oligos were designed wrong and will not work. Also the antitracr ones should amplify both tracr and antitracr as long as they’re part of a one-mix RTPCR kit. And also also it might not be possible to make one that amplifies tracr without also amplifying antitracr for the same reason.
Ran 5s extension and 62C annealing.
left to right: antitracr-up and -dn with 1ng/uL pdCas9 at 0.3, 0.6, 0.9 uL, then the same with tracr-up and -dn. The bigger bands are partial amplification of the entire backbone. There is more of a small product in the +_template lanes for antitracr than there is in the no-template. It might be possible to improve this with different cycling conditions.
October 4
Objective: test more decoy arrays for repression & cooperativity
Last night started from plates
pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-2x5xGFPdecoy-1 G251
pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-B0011 (0x) C251
pdCas9, pSB4K5-K(gfp), pSB1A2-2x5xGFPdecoy-1 G01
pdCas9, pSB4K5-K(gfp), pSB1A2-B0011 (0x) C01
...and also from frozen, three cultures started from same frozen stock
pdCas9-GFP1, pSB4K5-K608012, pSB1A2-5xGFPdecoy-4 G54
pdCas9-GFP1, pSB4K5-K608012, pSB1A2-1xGFPdecoy-1 G11
pdCas9, pSB4K5-K608012, pSB1A2-5xGFPdecoy-4 C54
pdCas9, pSB4K5-K608012, pSB1A2-1xGFPdecoy-1 C11
….then diluted 1/200 in fresh LB + AmpKanCm. Let roll for 3h30m before spinning and resuspending in PBS for flow. Collected 75000 events, 350 350 350V on fsc ssc b1
This is after guessing on some of the sample IDs. This flow should be redone since I somehow came up one sample short during acquisition (possibly lost a 0x?)
Name
Count/ml
GFP/FITC-A Mean
putative decoy count
if142014-10-04.001
7032348.63
65.24
6
if142014-10-04.002
7693097.17
58.46
6
if142014-10-04.003
13638844.73
60.12
6
if142014-10-04.004
6406423.83
4.64
1
if142014-10-04.005
6806425.29
9.83
1
if142014-10-04.006
8183306.15
9.88
1
if142014-10-04.007
30012003.91
7.32
0
if142014-10-04.008
7628153.32
7.33
0
if142014-10-04.009
4245683.59
77.01
12
if142014-10-04.010
32425421.88
88.33
12
if142014-10-04.011
19679875
81.53
12
if142014-10-04.012
9475678.71
155.04
6
if142014-10-04.013
8393017.58
145.65
6
if142014-10-04.014
8781172.85
148.24
6
if142014-10-04.015
9550490.23
150.54
1
if142014-10-04.016
10143360.35
148.22
1
if142014-10-04.017
9304056.64
142.56
1
if142014-10-04.018
7726383.3
150.92
0
if142014-10-04.019
8277232.42
148.32
0
if142014-10-04.020
8675535.16
150.66
0
if142014-10-04.021
8471705.08
148.45
12
if142014-10-04.022
7073470.21
145.62
12
live (if142014-10-04.023)
6844938.96
148.57
12
Derepression happens with even a single copy?
There is more derepression with 12 than with 6, but not tremendously more. Still not derepressing all the way to pdCas9-only levels. Eyeballing, it doesn’t look frightfully cooperative but we probably need more than 4 points to be sure.
Conclusions and next steps
We need more arrays between 1 and 6x
We need more arrays longer than 12x
Possible explanations for mild increase in derepression
Plasmid copy number makes the difference between 0x and 1x decoys on the plasmid translate to no copies in the cell to 100s. The difference between 1x and 6x is 6-fold, 6x to 12x is just 2-fold. Bigger decoy arrays are needed to test this. Copies of decoys vs copies of site of repression might be important
Instability: there is published work where long arrays of TF binding sites can lead to DNA breakage, and CBC has seen this in his own work with LacI arrays. It might be that 12x repeats of the decoy get broken, recombined, and shrunk down to something smaller. This is testable by extracting plasmid and looking at size of EcoRI/PstI fragments, but keep in mind that there are other plasmids in these bugs.
October 7
Objective: optimize tracr/antitracr PCR
Set up test PCR with 5s extension and a gradient from 58 - 68C for annealing temp. 30s at annealing, 0.5 uL/rx for + template, 0uL/rx for -template
Left to right is (+ template) 58 -> 68 and then (-template) 58-68 again there is more product with template, but right around the same size as primer dimer
Objective: try again RNA extraction, aim for more RNA
Last night started cultures in LB+amp or +Cm of Z1 + pdCas9 or 1A2-R0040-antitracr. Diluted 1/400 in new fresh medium + or 100nM aTc/50% EtOH. Let shake for 5h and collected 1 mL
Final ODs………...RNA concs (nanodropped)
1A2-R0040-antitracr + aTc     0.000 ………..
1A2-R0040-antitracr - aTc    0.000 ………..
pdCas9 + aTc     0.000 ………..
pdCas9 - aTc     0.000 ………..
Ran ~100ng out on a gel for 16m. They all look like 2 bands of something smeary. Maybe not plasmid? pdCas9 is ~10000 bp. tRNA is less than 100bp. Do this again but with DH5alphaZ1 in LB+Spec
October 8
Objective: another RNA isolation
This time with empty Z1 as well as 1A2-R0040-antitracr and pdCas9 strains alls +/- aTc
One of the pdCas9 culture’s pre-filter spin filtered very, very slowly and incompletely so it was discarded.
Left to right, empty Z1 +aTc, -aTc, then Z1+R0040-antitracr +aTc -aTc
Asked Tony Burnetti of the Buchler lab what RNA should look like and he said it would be mainly two bands which correspond to the big and small ribosomal subunits. RNA extraction successful!
October 9
Objective: send parts for iGEM competition
Submitted 1x, 6x, and 12x GFP1 decoy arrays in pSB1C3 and sent 10uL 25ng/uL plasmids in dH2O via FedEx priority overnight in accordance with the iGEM instructions.
Tracking number 771440542338
Shipment number 02826 (written on tube holding PCR tube strip)
iGEM website lists as received on 10/10/14. Huzzah!
October 14
Objective: Final reflow of decoy array strains
Started last night from frozen 3 replicate cultures each of pdCas9-GFP1, reporter, and 1A2-0x 1x 6x and 12x, and the same but with pdCas9 sans GFP1 crRNA. Diluted 1/400 in new LB+AmpKanCm and let shake for 4h before collecting 900uL and replacing medium with PBS. Vortex and flow
Gated for “singlets,” which are probably just smaller bacteria but did it for consistency with other plots and normalized by FSCA for the same area. Made ugly box plots with matlab: bam
legend indicates pdCas9 variant. All strians have reporter and some number of pSB1A2-decoy
If you plot it on a log scale it 1) wonks up the box widths but 2) shows that the derepression looks pretty smooth. The kind of ultrasensitivity we are looking for manifests on the log scale so this might be something to be concerned about. One way to improve this is to make the binding to the decoy sites stronger than it is to the actual site of repression. You could do this by intentionally having a mismatch between crRNA and the site of repression, but having the decoys be a perfect match.
Objective: RTPCR of antitracr strains to check for aTc-responsive antitracr RNA levels
RNA extracted DH5alphaZ1 and Z1+R0040, and Z1+pdCas9 in +/- aTc used as a template for one-step RT-PCR kit with antitracr-up and -dn according to bioline’s instructions. 1.5uL of 1 ng/uL dilution of RNA used as template, also ran no RT controls for each strain as well as no template and 1.5ng pdCas9 (DNA)
Left to right:
0 ladder
1 no template + RT
2 ng pdCas9 template (DNA) + RT
Z1 +aTc + RT
Z1 -aTc + RT
Z1+1A2-R0040-antitracr +aTc + RT
Z1+1A2-R0040-antitracr -aTc + RT
Z1+pdCas9 +aTc, Z1+pdCas9 -aTc + RT
Z1 +aTc - RT
Z1 -aTc - RT
Z1+1A2-R0040-antitracr +aTc - RT
Z1+1A2-R0040-antitracr -aTc - RT
Z1+pdCas9 +aTc, Z1+pdCas9 -aTc - RT
Quantifying with heavy caveats by drawing boxes in fiji.
Draw little boxes outside of the bands but near to get background
Draw little boxes inside each band
Measure gives mean pixel intensity. Subtract band mean intensity - bg mean intensity and you get this:
HOWEVER you have to take these with a fairly big grain of salt because this gel did not have a loading control and this is end-point RT-PCR so we don’t know whether the PCR was stopped in the linear range.
Product on -RT indicates some DNA contamination. +RT bands are stronger where they’re supposed to be stronger, though, and Z1 with RT having no bands might mean that there is less than the bands would indicate. The test of this is to take some, treat with RNAse, and then PCR the product of that
On +RT lanes we have no bands as expected for Z1, stronger on 1A2-R0040-antitracr +aTc than -aTc as expected, and levels unresponsive to aTc for pdCas9 lanes also as expected. Everything is as expected, except maybe the leakiness of pTet. That could be either DNA contamination or genuine leak.
The question motivating this RT PCR was whether pTet is actually expressing antitracr, and this gel indicates that it is. Possibilities for why the fluorescence changes aren’t what you’d expect when doing flow are
Insufficient expression of antitracr
In vivo binding might not be as strong as you’d expect from RNAfold etc predictions. If this is the case then you’d need a lot of antitracr to see derepression
tracr is co-cistronic with dCas9 on pdCas9 - one could imagine that maybe each tracrRNA that gets made is in close proximity to dCas9 proteins and is quickly bound to dCas9 before antitracr has access. More antitracr may also overcome this
maybe in vivo hairpinning prevents antitracr from grabbing tracr.
It’s worth noting that antitracr has a little extra sequence on 5’ from PCR/oligo design realities during construction and also the terminators on 3’. antitracr is probably not getting processed in the same way that tracr is so everything on the plasmid after TSS and including terminators are probably present.
If this is the case then expressing a full-on gRNA with non-targeting targeting sequence (20bp determining binding specificity) might be a better choice for universal dCas9 derepression.
You could also think about doing anti-crRNAs for specific derepression.
October 15
Objective: look for (in)stability of decoy arrays
Is the mediocre increase in derepression of 12x decoy GFP1 relative to 6x due to the long repeats getting broken down from repeated collisions of dCas9 with DNAP? Charlie has seen this before with the Lac repressor
DNA was miniprepped from the overnight cultures that made yesterdays’ flow data and EcoRI/PstI digested.
Left ot right
ladder
G0
G1
G6
G12
C0
C1
C6
C12
C6’s band is faint and smeary, but C12 looks nice? Reason for that is unclear, but all of the G bands’ clear bandiness and C12’s suggests that, for this round at least, the arrays are stable.
Retrieved from " http://2014.igem.org/Team:Duke/Notebook/October "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
