<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Duke_Notebook_October"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Duke/Notebook/October">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Duke/Notebook/October&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Notebook/October&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Duke/Notebook/October&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Duke/Notebook/October" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Duke/Notebook/October</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><UL id="navbar"><LI><A href="https://2014.igem.org/Team:Duke">Home</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team">About Us</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Duke">Official Team Profile</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team/Gallery">Photo Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Team/Attributions">Attributions</A></LI></UL><LI><A href="https://2014.igem.org/Team:Duke/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#background">Background</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#summary">Summary</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Modeling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#results">Results</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Project#future">Future Directions</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Overview">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Months">Notebook by Month</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy">Policy and Practice</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#threed">3D Printing</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#hoco">House Course</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Policy#smath">NCSSM Team</A></LI><LI><A href="https://2014.igem.org/Team:Duke/Safety">Safety</A></LI></P><DIV class="day"><H2><A id="oct1">October 1</A></H2><P>Objective: sequence confirm 1C3-2x5xGFP</P><UL><LI>Sequenced with pSB1C3-up-1 and -dn-1, and both clones look like they have a duplication of 6x, rather than 5x. That means that clone 43 from yesterday is not useful so it will be thrown away.</LI></UL><P>Objective: test RT-PCR primers</P><UL><LI>MZ setting up two test PCRs</LI><LI>4 samples of tracr amplification on pdcas9 - 0, 0.3, 0.6, 0.9 ng/uL tubes, 4 samples of anti-tracr amplification on pdcas9</LI><LI>results: no good</LI><LI>We can guess at what’s happening here, but probably the first thing to do is just to try it again.</LI></UL><H2><A id="oct2">October 2</A></H2><P>Objective: confirm 1A2-2x5x</P><UL><LI>Picked 6 colonies and subjecting to EcoRI/PstI digestion</LI></UL><UL><LI>Left to right are clones 1-6 of 2x5x, rightmost = 1A2-5x. Looks perfect, freezing clones 1 and 2 and will sequence this weekend. </LI></UL><P>Objective: extract RNA and test for expression &amp; aTc response of tracr, antitracr</P><UL><LI><A href="http://www.google.com/url?q=http%3A%2F%2Fopenwetware.org%2Fwiki%2FSauer%3ARNA_Purification_from_E._coli&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNEUMG4dvztCAaiAneGoVZIW0PKINA">http://openwetware.org/wiki/Sauer:RNA_Purification_from_E._coli</A></LI><LI>TYler of Gersbach lab recommends RNAzap, lab coat, holding breath, filter tips. Random hexamers are better if you want to RT everything which can be more generically useful than specific primers for this step. He uses hexamers for RT step then does specific amplification following that.         </LI></UL><UL><LI>Last night started from frozen some Z1 with pdCas9 and 1A2-R0040-antitracr in LB + antibioic. This morning measured 1/10 diluted OD600 and calculated amount to add to 1) balance cultures ODs and 2) dilute back to a sufficient degree that after several generations in +/- aTc the cultures would not be at too high a density to easily harvest the max number of cells called for by the agilent RNA purification kit. This means diluting to the equivalent of 1/1000 from a culture of OD600=3.0</LI></UL><UL><LI><A href="http://www.google.com/url?q=http%3A%2F%2Fwww.genomics.agilent.com%2Fbiocalculators%2FcalcODBacterial.jsp&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNHJj1DKIVUO326ebS70jsxbDBDHNw">http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp</A> for OD -&gt; cell count calculations</LI><LI><A href="http://www.google.com/url?q=http%3A%2F%2Fwww.chem-agilent.com%2Fpdf%2Fstrata%2F400800.pdf&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNGRsizNL201WP4khmvIUEEuqwIKKA">http://www.chem-agilent.com/pdf/strata/400800.pdf</A></LI></UL><UL><LI>manual</LI></UL><UL><LI><A href="http://www.google.com/url?q=http%3A%2F%2Fcp.literature.agilent.com%2Flitweb%2Fpdf%2F5990-5152EN.pdf&amp;sa=D&amp;sntz=1&amp;usg=AFQjCNGvbTOiNbW_-5dNRECt1JbqQC9CQA">http://cp.literature.agilent.com/litweb/pdf/5990-5152EN.pdf</A></LI></UL><UL><LI>Has details for lysozyme treatment of coli before following the rest of the lysis/extraction protocol. This one might be just for the microprep kit, rather than the miniprep kit which we have.</LI></UL><UL><LI>1 mg/mL lysozyme, ethanol dilutions were prepared with filtered EtOH and/or water. Not DEPC treated or certified RNAse free. </LI></UL><UL><LI>Kind of a big fucking mess. Centrifuge at CBC’s bench is broken so rna back and forth between Mert’s bench to do centrifugations. Lids broke off so the real ID of some samples isn’t actually known, and accidentally made too little lysis buffer so one sample is just gone. Ended up with low 20s to 50s of some nucleic acid as measured by ND but all was pretty contaminated with ethanol, or whatever it is that gives a low 260/230 ratio. Will try this again. </LI><LI>Also the pdCas9 dilutions grew less quickly than did the 1A2-R0040antitracr ones</LI></UL><UL><LI>~.065 after 3h in new medium pdCas9</LI><LI>~.112 after 3h in new medium 1A2-R0040antitracr</LI></UL><UL><LI>Tomorrow AM: retry test PCR on DNA with the anti/tracr oligos.</LI></UL><P>Objective: make more decoy strains for testing</P><UL><LI>Shocking Z1 with</LI></UL><UL><LI>pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-2x5xGFPdecoy-1</LI><LI>pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-B0011 (0x)</LI><LI>pdCas9, pSB4K5-K(gfp), pSB1A2-2x5xGFPdecoy-1</LI><LI>pdCas9, pSB4K5-K(gfp), pSB1A2-B0011 (0x)</LI></UL><H2><A id="oct3">October 3</A></H2><P>Objective: test PCR of tracr &amp; antitracr oligos on pdCas9</P><UL><LI>tracr oligos were designed wrong and will not work. Also the antitracr ones should amplify both tracr and antitracr as long as they’re part of a one-mix RTPCR kit. And also also it might not be possible to make one that amplifies tracr without also amplifying antitracr for the same reason. </LI><LI>Ran 5s extension and 62C annealing. </LI><LI>left to right: antitracr-up and -dn with 1ng/uL pdCas9 at 0.3, 0.6, 0.9 uL, then the same with tracr-up and -dn. The bigger bands are partial amplification of the entire backbone. There is more of a small product in the +_template lanes for antitracr than there is in the no-template. It might be possible to improve this with different cycling conditions.</LI></UL><H2><A id="oct4">October 4</A></H2><P>Objective: test more decoy arrays for repression &amp; cooperativity</P><UL><LI>Last night started from plates</LI></UL><UL><LI>pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-2x5xGFPdecoy-1 G251</LI><LI>pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-B0011 (0x) C251</LI><LI>pdCas9, pSB4K5-K(gfp), pSB1A2-2x5xGFPdecoy-1 G01</LI><LI>pdCas9, pSB4K5-K(gfp), pSB1A2-B0011 (0x) C01</LI></UL><UL><LI>...and also from frozen, three cultures started from same frozen stock</LI></UL><UL><LI>pdCas9-GFP1, pSB4K5-K608012, pSB1A2-5xGFPdecoy-4 G54</LI><LI>pdCas9-GFP1, pSB4K5-K608012, pSB1A2-1xGFPdecoy-1 G11</LI><LI>pdCas9, pSB4K5-K608012, pSB1A2-5xGFPdecoy-4 C54</LI><LI>pdCas9, pSB4K5-K608012, pSB1A2-1xGFPdecoy-1 C11</LI></UL><UL><LI>….then diluted 1/200 in fresh LB + AmpKanCm. Let roll for 3h30m before spinning and resuspending in PBS for flow. Collected 75000 events, 350 350 350V on fsc ssc b1</LI></UL><UL><LI>This is after guessing on some of the sample IDs. This flow should be redone since I somehow came up one sample short during acquisition (possibly lost a 0x?)</LI></UL><TABLE cellpadding="0" cellspacing="0"><TBODY><TR><TD><P>Name</P></TD><TD><P>Count/ml</P></TD><TD><P>GFP/FITC-A Mean</P></TD><TD><P>putative decoy count</P></TD></TR><TR><TD><P>if142014-10-04.001</P></TD><TD><P>7032348.63</P></TD><TD><P>65.24</P></TD><TD><P>6</P></TD></TR><TR><TD><P>if142014-10-04.002</P></TD><TD><P>7693097.17</P></TD><TD><P>58.46</P></TD><TD><P>6</P></TD></TR><TR><TD><P>if142014-10-04.003</P></TD><TD><P>13638844.73</P></TD><TD><P>60.12</P></TD><TD><P>6</P></TD></TR><TR><TD><P>if142014-10-04.004</P></TD><TD><P>6406423.83</P></TD><TD><P>4.64</P></TD><TD><P>1</P></TD></TR><TR><TD><P>if142014-10-04.005</P></TD><TD><P>6806425.29</P></TD><TD><P>9.83</P></TD><TD><P>1</P></TD></TR><TR><TD><P>if142014-10-04.006</P></TD><TD><P>8183306.15</P></TD><TD><P>9.88</P></TD><TD><P>1</P></TD></TR><TR><TD><P>if142014-10-04.007</P></TD><TD><P>30012003.91</P></TD><TD><P>7.32</P></TD><TD><P>0</P></TD></TR><TR><TD><P>if142014-10-04.008</P></TD><TD><P>7628153.32</P></TD><TD><P>7.33</P></TD><TD><P>0</P></TD></TR><TR><TD><P>if142014-10-04.009</P></TD><TD><P>4245683.59</P></TD><TD><P>77.01</P></TD><TD><P>12</P></TD></TR><TR><TD><P>if142014-10-04.010</P></TD><TD><P>32425421.88</P></TD><TD><P>88.33</P></TD><TD><P>12</P></TD></TR><TR><TD><P>if142014-10-04.011</P></TD><TD><P>19679875</P></TD><TD><P>81.53</P></TD><TD><P>12</P></TD></TR><TR><TD><P>if142014-10-04.012</P></TD><TD><P>9475678.71</P></TD><TD><P>155.04</P></TD><TD><P>6</P></TD></TR><TR><TD><P>if142014-10-04.013</P></TD><TD><P>8393017.58</P></TD><TD><P>145.65</P></TD><TD><P>6</P></TD></TR><TR><TD><P>if142014-10-04.014</P></TD><TD><P>8781172.85</P></TD><TD><P>148.24</P></TD><TD><P>6</P></TD></TR><TR><TD><P>if142014-10-04.015</P></TD><TD><P>9550490.23</P></TD><TD><P>150.54</P></TD><TD><P>1</P></TD></TR><TR><TD><P>if142014-10-04.016</P></TD><TD><P>10143360.35</P></TD><TD><P>148.22</P></TD><TD><P>1</P></TD></TR><TR><TD><P>if142014-10-04.017</P></TD><TD><P>9304056.64</P></TD><TD><P>142.56</P></TD><TD><P>1</P></TD></TR><TR><TD><P>if142014-10-04.018</P></TD><TD><P>7726383.3</P></TD><TD><P>150.92</P></TD><TD><P>0</P></TD></TR><TR><TD><P>if142014-10-04.019</P></TD><TD><P>8277232.42</P></TD><TD><P>148.32</P></TD><TD><P>0</P></TD></TR><TR><TD><P>if142014-10-04.020</P></TD><TD><P>8675535.16</P></TD><TD><P>150.66</P></TD><TD><P>0</P></TD></TR><TR><TD><P>if142014-10-04.021</P></TD><TD><P>8471705.08</P></TD><TD><P>148.45</P></TD><TD><P>12</P></TD></TR><TR><TD><P>if142014-10-04.022</P></TD><TD><P>7073470.21</P></TD><TD><P>145.62</P></TD><TD><P>12</P></TD></TR><TR><TD><P>live (if142014-10-04.023)</P></TD><TD><P>6844938.96</P></TD><TD><P>148.57</P></TD><TD><P>12</P></TD></TR></TBODY></TABLE><UL><LI>Derepression happens with even a single copy? </LI><LI>There is more derepression with 12 than with 6, but not tremendously more. Still not derepressing all the way to pdCas9-only levels. Eyeballing, it doesn’t look frightfully cooperative but we probably need more than 4 points to be sure. </LI><LI>Conclusions and next steps</LI></UL><UL><LI>We need more arrays between 1 and 6x</LI><LI>We need more arrays longer than 12x</LI></UL><UL><LI>Possible explanations for mild increase in derepression</LI></UL><UL><LI>Plasmid copy number makes the difference between 0x and 1x decoys on the plasmid translate to no copies in the cell to 100s. The difference between 1x and 6x is 6-fold, 6x to 12x is just 2-fold. Bigger decoy arrays are needed to test this. Copies of decoys vs copies of site of repression might be important</LI><LI>Instability: there is published work where long arrays of TF binding sites can lead to DNA breakage, and CBC has seen this in his own work with LacI arrays. It might be that 12x repeats of the decoy get broken, recombined, and shrunk down to something smaller. This is testable by extracting plasmid and looking at size of EcoRI/PstI fragments, but keep in mind that there are other plasmids in these bugs. </LI></UL><H2><A id="oct7">October 7</A></H2><P>Objective: optimize tracr/antitracr PCR</P><UL><LI>Set up test PCR with 5s extension and a gradient from 58 - 68C for annealing temp. 30s at annealing, 0.5 uL/rx for + template, 0uL/rx for -template</LI><LI>Left to right is (+ template) 58 -&gt; 68 and then (-template) 58-68 again there is more product with template, but right around the same size as primer dimer</LI></UL><P>Objective: try again RNA extraction, aim for more RNA</P><UL><LI>Last night started cultures in LB+amp or +Cm of Z1 + pdCas9 or 1A2-R0040-antitracr. Diluted 1/400 in new fresh medium + or 100nM aTc/50% EtOH. Let shake for 5h and collected 1 mL</LI><LI>Final ODs………...RNA concs (nanodropped)</LI></UL><UL><LI>1A2-R0040-antitracr + aTc    0.000 ………..</LI><LI>1A2-R0040-antitracr - aTc   0.000 ………..</LI><LI>pdCas9 + aTc    0.000 ………..</LI><LI>pdCas9 - aTc    0.000 ………..</LI></UL><UL><LI>Ran ~100ng out on a gel for 16m. They all look like 2 bands of something smeary. Maybe not plasmid? pdCas9 is ~10000 bp. tRNA is less than 100bp. Do this again but with DH5alphaZ1 in LB+Spec </LI></UL><H2><A id="oct8">October 8</A></H2><P>Objective: another RNA isolation</P><UL><LI>This time with empty Z1 as well as 1A2-R0040-antitracr and pdCas9 strains alls +/- aTc</LI><LI>One of the pdCas9 culture’s pre-filter spin filtered very, very slowly and incompletely so it was discarded. </LI></UL><UL><LI>Left to right, empty Z1 +aTc, -aTc, then Z1+R0040-antitracr +aTc -aTc</LI></UL><UL><LI>Asked Tony Burnetti of the Buchler lab what RNA should look like and he said it would be mainly two bands which correspond to the big and small ribosomal subunits. RNA extraction successful!</LI></UL><H2><A id="oct9">October 9</A></H2><P>Objective: send parts for iGEM competition</P><UL><LI>Submitted 1x, 6x, and 12x GFP1 decoy arrays in pSB1C3 and sent 10uL 25ng/uL plasmids in dH2O via FedEx priority overnight in accordance with the iGEM instructions. </LI></UL><UL><LI>Tracking number 771440542338</LI><LI>Shipment number 02826 (written on tube holding PCR tube strip)</LI><LI>iGEM website lists as received on 10/10/14. Huzzah!</LI></UL><H2><A id="oct14">October 14</A></H2><P>Objective: Final reflow of decoy array strains</P><UL><LI>Started last night from frozen 3 replicate cultures each of pdCas9-GFP1, reporter, and 1A2-0x 1x 6x and 12x, and the same but with pdCas9 sans GFP1 crRNA. Diluted 1/400 in new LB+AmpKanCm and let shake for 4h before collecting 900uL and replacing medium with PBS. Vortex and flow</LI><LI>Gated for “singlets,” which are probably just smaller bacteria but did it for consistency with other plots and normalized by FSCA for the same area. Made ugly box plots with matlab: bam</LI><LI>legend indicates pdCas9 variant. All strians have reporter and some number of pSB1A2-decoy</LI><LI>If you plot it on a log scale it 1) wonks up the box widths but 2) shows that the derepression looks pretty smooth. The kind of ultrasensitivity we are looking for manifests on the log scale so this might be something to be concerned about. One way to improve this is to make the binding to the decoy sites stronger than it is to the actual site of repression. You could do this by intentionally having a mismatch between crRNA and the site of repression, but having the decoys be a perfect match. </LI></UL><P>Objective: RTPCR of antitracr strains to check for aTc-responsive antitracr RNA levels</P><UL><LI>RNA extracted DH5alphaZ1 and Z1+R0040, and Z1+pdCas9 in +/- aTc used as a template for one-step RT-PCR kit with antitracr-up and -dn according to bioline’s instructions. 1.5uL of 1 ng/uL dilution of RNA used as template, also ran no RT controls for each strain as well as no template and 1.5ng pdCas9 (DNA)</LI><LI>Left to right: </LI></UL><UL><LI>0 ladder</LI><LI>1 no template + RT</LI><LI>2 ng pdCas9 template (DNA) + RT</LI><LI>Z1 +aTc + RT</LI><LI>Z1 -aTc + RT</LI><LI>Z1+1A2-R0040-antitracr +aTc + RT</LI><LI>Z1+1A2-R0040-antitracr -aTc + RT</LI><LI>Z1+pdCas9 +aTc, Z1+pdCas9 -aTc + RT</LI><LI>Z1 +aTc - RT</LI><LI>Z1 -aTc - RT</LI><LI>Z1+1A2-R0040-antitracr +aTc - RT</LI><LI>Z1+1A2-R0040-antitracr -aTc - RT</LI><LI>Z1+pdCas9 +aTc, Z1+pdCas9 -aTc - RT</LI></UL><UL><LI>Quantifying with heavy caveats by drawing boxes in fiji.</LI></UL><UL><LI>Draw little boxes outside of the bands but near to get background </LI><LI>Draw little boxes inside each band</LI><LI>Measure gives mean pixel intensity. Subtract band mean intensity - bg mean intensity and you get this:</LI><LI>HOWEVER you have to take these with a fairly big grain of salt because this gel did not have a loading control and this is end-point RT-PCR so we don’t know whether the PCR was stopped in the linear range. </LI></UL><UL><LI>Product on -RT indicates some DNA contamination. +RT bands are stronger where they’re supposed to be stronger, though, and Z1 with RT having no bands might mean that there is less than the bands would indicate. The test of this is to take some, treat with RNAse, and then PCR the product of that</LI><LI>On +RT lanes we have no bands as expected for Z1, stronger on 1A2-R0040-antitracr +aTc than -aTc as expected, and levels unresponsive to aTc for pdCas9 lanes also as expected. Everything is as expected, except maybe the leakiness of pTet. That could be either DNA contamination or genuine leak. </LI><LI>The question motivating this RT PCR was whether pTet is actually expressing antitracr, and this gel indicates that it is. Possibilities for why the fluorescence changes aren’t what you’d expect when doing flow are</LI></UL><UL><LI>Insufficient expression of antitracr</LI></UL><UL><LI>In vivo binding might not be as strong as you’d expect from RNAfold etc predictions. If this is the case then you’d need a lot of antitracr to see derepression</LI><LI>tracr is co-cistronic with dCas9 on pdCas9 - one could imagine that maybe each tracrRNA that gets made is in close proximity to dCas9 proteins and is quickly bound to dCas9 before antitracr has access. More antitracr may also overcome this</LI></UL><UL><LI>maybe in vivo hairpinning prevents antitracr from grabbing tracr. </LI></UL><UL><LI>It’s worth noting that antitracr has a little extra sequence on 5’ from PCR/oligo design realities during construction and also the terminators on 3’. antitracr is probably not getting processed in the same way that tracr is so everything on the plasmid after TSS and including terminators are probably present.</LI><LI>If this is the case then expressing a full-on gRNA with non-targeting targeting sequence (20bp determining binding specificity) might be a better choice for universal dCas9 derepression.</LI><LI>You could also think about doing anti-crRNAs for specific derepression. </LI></UL><H2><A id="oct15">October 15</A></H2><P>Objective: look for (in)stability of decoy arrays</P><UL><LI>Is the mediocre increase in derepression of 12x decoy GFP1 relative to 6x due to the long repeats getting broken down from repeated collisions of dCas9 with DNAP? Charlie has seen this before with the Lac repressor</LI><LI>DNA was miniprepped from the overnight cultures that made yesterdays’ flow data and EcoRI/PstI digested.</LI><LI>Left ot right</LI></UL><UL><LI>ladder</LI><LI>G0</LI><LI>G1</LI><LI>G6</LI><LI>G12</LI><LI>C0</LI><LI>C1</LI><LI>C6</LI><LI>C12</LI></UL><UL><LI>C6’s band is faint and smeary, but C12 looks nice? Reason for that is unclear, but all of the G bands’ clear bandiness and C12’s suggests that, for this round at least, the arrays are stable.</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Duke/Notebook/October">http://2014.igem.org/Team:Duke/Notebook/October</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Duke/Notebook/October" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Duke/Notebook/October" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Duke/Notebook/October&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Duke/Notebook/October&amp;oldid=304495" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>