"
Page
Discussion
View source
History
teams
Log in
 
Team:Duke/Notebook/September
From 2014.igem.org
Home
Team
About Us
Official Team Profile
Photo Gallery
Attributions
Project
Background
Summary
Modeling
Results
Future Directions
Parts
Notebook
Overview
Notebook by Month
Protocols
Policy and Practice
3D Printing
House Course
NCSSM Team
Safety
September 4
Objective: test backbone shuffled antitracr bacteria
Started colonies from plates and diluted in to +/- aTc this morning. Let shake for 5(?) hours before collecting 500uL and spin/resusp in PBS for flow.
Took data to flowjo and plotted histograms below. This is ungated fluorescence
table with mean GFPAs
ungated GFPA
+/- aTc
1: if142014-09-04.001.fcs
AC-1
+ aTc
0.402
1.185840708
2: if142014-09-04.002.fcs
AC-2
+ aTc
0.455
1.197368421
3: if142014-09-04.003.fcs
G-1
+ aTc
14.9
0.949044586
4: if142014-09-04.004.fcs
G-2
+ aTc
11.7
0.573529412
5: if142014-09-04.005.fcs
G-3
+ aTc
8.91
0.530357143
6: if142014-09-04.006.fcs
G-4
+ aTc
5.92
0.34619883
7: if142014-09-04.007.fcs
C-1
+ aTc
186
1.016393443
8: if142014-09-04.008.fcs
C-2
+ aTc
224
1.230769231
9: if142014-09-04.009.fcs
C-3
+ aTc
214
0.922413793
10: if142014-09-04.010.fcs
C-4
+ aTc
222
1.193548387
11: if142014-09-04.011.fcs
AC-1
- aTc
0.339
12: if142014-09-04.012.fcs
AC-2
- aTc
0.38
13: if142014-09-04.013.fcs
G-1
- aTc
15.7
14: if142014-09-04.014.fcs
G-2
- aTc
20.4
15: if142014-09-04.015.fcs
G-3
- aTc
16.8
16: if142014-09-04.016.fcs
G-4
- aTc
17.1
17: if142014-09-04.017.fcs
C-1
- aTc
183
18: if142014-09-04.018.fcs
C-2
- aTc
182
19: if142014-09-04.019.fcs
C-3
- aTc
232
20: if142014-09-04.020.fcs
C-4
- aTc
186
aTc treatment seems to be moving fluor in the opposite direction of what we’d expect of antitracr was doing its job. The degree is inconsistent, reflow is called for.
September 8
Objective: retest backbone shuffled antitracr bacteria
Yesterday picked 4 new colonies from plates and 4 from 9/4/14 frozen stocks and grew in LB + antibiotics overnight. Diluted in to +/- aTc this morning and let shake for 5(?) hours before collecting 500uL and spin/resusp in PBS
G1-4 are old frozen coli with K608012, pdCas9-GFP1, and 1A2-R0040-antitracr, C1-4 are old frozen with pdCas9. clones 5-8 are new from plates. AC are no-fluor (1A2 and pdCas9) but they were diluted in to triple antibiotic medium instead of double so they did not grow and are not reliable indicators of bg fluorescence on this run.
After flow, isolated singlets with flowjo (smalls) or left alone (bugs) and got gfpa unmodified or normalized by fsca or ssca.
smalls
bugs
aTc
gfpa
gfpa/fsca*1000
gfpa/ssca*1000
gfpa
gfpa/fsca*1000
gfpa/ssca*1000
1: if142014-09-08.001.fcs
G1
+
5.25
243
112
7.79
247
114
2: if142014-09-08.002.fcs
G2
+
4.41
226
78.1
7.12
228
76.2
3: if142014-09-08.003.fcs
G3
+
4.66
231
84.8
7.8
231
83.4
4: if142014-09-08.004.fcs
G4
+
3.86
191
70.1
5.99
192
67.9
5: if142014-09-08.005.fcs
G1
-
6.81
311
145
8.56
310
143
6: if142014-09-08.006.fcs
G2
-
6.37
337
153
7.25
323
145
7: if142014-09-08.007.fcs
G3
-
6.5
318
143
8.24
305
136
8: if142014-09-08.008.fcs
G4
-
6.14
298
140
7.66
288
131
9: if142014-09-08.009.fcs
C1
+
88.9
4071
1777
117
3978
1706
10: if142014-09-08.010.fcs
C2
+
83.5
4073
1547
120
3969
1422
11: if142014-09-08.011.fcs
C3
+
81.7
3957
1551
115
3840
1437
12: if142014-09-08.012.fcs
C4
+
77.2
3915
1577
95.7
3883
1503
13: if142014-09-08.013.fcs
C1
-
77.7
3686
1471
100
3545
1411
14: if142014-09-08.014.fcs
C2
-
64.5
3093
1445
88.3
2962
1373
15: if142014-09-08.015.fcs
C3
-
80.9
3971
1541
106
3771
1432
16: if142014-09-08.016.fcs
C4
-
58.9
2899
1299
79.7
2727
1220
17: if142014-09-08.017.fcs
G5
+
4.1
210
79.1
5.4
209
76.2
18: if142014-09-08.018.fcs
G6
+
4.9
225
98.2
7.07
226
97.6
19: if142014-09-08.019.fcs
G7
+
4.42
220
84.7
5.97
218
81.7
20: if142014-09-08.020.fcs
G8
+
4.19
211
81.3
5.68
213
79.5
21: if142014-09-08.021.fcs
G5
-
9.73
477
201
12.5
455
190
22: if142014-09-08.022.fcs
G6
-
9.68
450
197
12.9
449
196
23: if142014-09-08.023.fcs
G7
-
10.1
482
212
13.3
464
198
24: if142014-09-08.024.fcs
G8
-
10.1
489
216
13.3
473
204
25: if142014-09-08.025.fcs
C5
+
79.6
3958
1486
113
3813
1374
26: if142014-09-08.026.fcs
C6
+
80.8
4045
1514
113
4043
1425
27: if142014-09-08.027.fcs
C7
+
80.7
4053
1491
116
3921
1375
28: if142014-09-08.028.fcs
C8
+
79.8
3804
1555
103
3634
1508
29: if142014-09-08.029.fcs
C5
-
68.9
3268
1505
95.7
3128
1419
30: if142014-09-08.030.fcs
C6
-
80.2
3806
1639
110
3630
1541
31: if142014-09-08.031.fcs
C7
-
67.8
3240
1498
91.7
3041
1412
32: if142014-09-08.032.fcs
C8
-
83.7
3810
1675
107
3618
1621
33: if142014-09-08.033.fcs
AC1
+
0.895
40.2
16.8
0.854
34.9
12.9
34: if142014-09-08.034.fcs
AC3
+
0.539
28.4
11
0.785
27.4
10.7
35: if142014-09-08.035.fcs
AC1
-
0.464
25.8
9.39
0.605
25.9
8.65
36: if142014-09-08.036.fcs
AC3
-
0.532
28.3
10.4
0.661
26.2
9.23
Got ratios of fluorescence in +/- aTc and am reporting them in the table below
n/a
n/a
+/-
smalls
bugs
n/a
n/a
gfpa
gfpa/fsca*1000
gfpa/ssca*1000
gfpa
gfpa/fsca*1000
gfpa/ssca*1000
1: if142014-09-08.001.fcs
G1
0.771
0.781
0.772
0.910
0.797
0.797
2: if142014-09-08.002.fcs
G2
0.692
0.671
0.510
0.982
0.706
0.526
3: if142014-09-08.003.fcs
G3
0.717
0.726
0.593
0.947
0.757
0.613
4: if142014-09-08.004.fcs
G4
0.629
0.641
0.501
0.782
0.667
0.518
9: if142014-09-08.009.fcs
C1
1.144
1.104
1.208
1.170
1.122
1.209
10: if142014-09-08.010.fcs
C2
1.295
1.317
1.071
1.359
1.340
1.036
11: if142014-09-08.011.fcs
C3
1.010
0.996
1.006
1.085
1.018
1.003
12: if142014-09-08.012.fcs
C4
1.311
1.350
1.214
1.201
1.424
1.232
17: if142014-09-08.017.fcs
G5
0.421
0.440
0.394
0.432
0.459
0.401
18: if142014-09-08.018.fcs
G6
0.506
0.500
0.498
0.548
0.503
0.498
19: if142014-09-08.019.fcs
G7
0.438
0.456
0.400
0.449
0.470
0.413
20: if142014-09-08.020.fcs
G8
0.415
0.431
0.376
0.427
0.450
0.390
25: if142014-09-08.025.fcs
C5
1.155
1.211
0.987
1.181
1.219
0.968
26: if142014-09-08.026.fcs
C6
1.007
1.063
0.924
1.027
1.114
0.925
27: if142014-09-08.027.fcs
C7
1.190
1.251
0.995
1.265
1.289
0.974
28: if142014-09-08.028.fcs
C8
0.953
0.998
0.928
0.963
1.004
0.930
33: if142014-09-08.033.fcs
AC1
1.929
1.558
1.789
1.412
1.347
1.491
34: if142014-09-08.034.fcs
AC3
1.013
1.004
1.058
1.188
1.046
1.159
Box plot of just G1-4 and C1-4. Note that the G clones from plates (clones 5-8) have even smaller ratios.
September 9
Objective: assembly decoy binding site arrays
Garima sets up a PCR and is asked to fill in notes about it.
Results
Charlie sets up a new set of tubes for PCR with a 5 rx master mix containing
190uL H2O
50 uL Q5 buffer
2.0 uL oligo 1 (gfp1-spacer-gfp1)
2.5 uL Q5 poly
5 uL dNTPs
And then divides that in to four tubes before adding in the following volumes of oligo 2 (spacer-gfp1-space)
tube number
1
2
3
4
uL oligo 2
0
0.5
1.0
2.0
Streakiness on the row with 0.5uL of oligo 2 looks promising? IGEM protocol on the thermal cycler modified
September 10
Objective: assemble decoy binding site arrays
Setting up two PCRs with same cycling program as last time:
one with 4x 50uL reactions using the same concs of everything as the 0.5 uL rx from 9/9/14 (blue labels)
one with equimolar oligo1 and oligo2 and varying amounts of reaction 0.5 from 9/9/14 (green labels)
So something’s happening. PCR cleanup’d reactions 1-4 (green) and 5-8 (blue) in to one tube each labeled Decoy array (1-4 or 5-8) PCR1. Tomorrow will PCR with the end oligos and try gibson, dig/lig in to pSB1C3 and/or pSB1A2
9/11/14
Objective: assemble decoy binding site arrays. Step 2: add prefix & suffix to arrays
Faw setting up two series of PCRs with 1c337prfGfp1Space and 1c337sufGfp1Space as oligos on 0.5-2.0 ng of PCR cleanup’d green and blue rx from 9/10/14 and running with the same cycling conditions.
CBC did not take a picture and doesn’t remember why. PCR looked bad for some reason.
September 12
Objective: assemble decoy binding site arrays. Step 2: add prefix & suffix to arrays
Garima setting up two series of PCRs with 1c337prfGfp1Space and 1c337sufGfp1Space as oligos on higher concs than yesterday ofPCR cleanup’d green and blue rx from 9/10/14 and running with the same cycling conditions.
PCR cleanup to pool tubes 2-4 for both series and labeled Decoy Array PCR2
September 13
Objective: assemble decoy binding site arrays. Step 3: insert putative arrays in to pSB backbone
CBC attempts a SLIC with 0.5 uL of the “Super Clean” XbaI/PstI pSB1C3 backbone. Note that puts 15 bp of nonhomology in the way of successful SLIC. I did this because sometimes it works despite mismatches and there was no EcoRI/PstI dig’d pSB1C3 for me to work with. Green series looks like there is a bigger difference between no-template and +template tubes, so I will try that
set up tubes for backbone-only, backbone + 0.5, 1, 2, 3uL of insert from Green tube at 88.8 ng/uL, and 3uL insert-only.
Didn’t work.
Objective: observe fluorescence changes with antitracr time course over time
Last night antitracr strains started in LB+AmpKanCm and let shake overnight. Diluted this morning in to +/- aTc and began collection at time indicated. Replaced LB with PBS before flowing
Isolated “singlets” which are probably not singlets (smalls) or left alone (bugs) and got mean GFPA or gfpa/fsca or /ssca to normalize roughly to size.
time point
n/a
n/a
1
2
3
4
5
time (min)
n/a
n/a
120
180
240
300
360
GFP1-9
+ aTc
gfpa (smalls)
1.52
2.74
3.17
3.16
2.81
GFP1-10
+ aTc
gfpa (smalls)
1.39
2.8
3.18
3.42
2.88
GFP1-11
+ aTc
gfpa (smalls)
1.45
2.92
3.31
3.53
2.88
GFP1-12
+ aTc
gfpa (smalls)
1.53
3.13
3.72
3.6
2.89
blankpdCas9-9
+ aTc
gfpa (smalls)
45.6
56
68.4
76.8
74.9
blankpdCas9-10
+ aTc
gfpa (smalls)
46.9
54
67.8
73.8
74.7
blankpdCas9-11
+ aTc
gfpa (smalls)
48.4
53.1
65.8
73
76.2
blankpdCas9-12
+ aTc
gfpa (smalls)
45.6
54.2
68.2
74.3
78.4
pdCas9
+ aTc
gfpa (smalls)
0.109
0.16
0.185
0.139
0.143
GFP1-9
- aTc
gfpa (smalls)
1.43
3.08
3.65
3.67
3.07
GFP1-10
- aTc
gfpa (smalls)
1.32
3.17
3.67
3.88
3.07
GFP1-11
- aTc
gfpa (smalls)
1.73
3.3
4.15
3.99
3.18
GFP1-12
- aTc
gfpa (smalls)
1.57
3.47
4.16
3.83
3.29
blankpdCas9-9
- aTc
gfpa (smalls)
41.7
53.4
67
75.8
75.6
blankpdCas9-10
- aTc
gfpa (smalls)
41.4
51.3
67.4
74.3
71.7
blankpdCas9-11
- aTc
gfpa (smalls)
39.4
52.4
66
73.9
60.8
blankpdCas9-12
- aTc
gfpa (smalls)
40
52.7
67
76
73.2
pdCas9
- aTc
gfpa (smalls)
0.0949
0.156
0.162
0.129
0.122
GFP1-9
+ aTc
gfpa/fsca*1000 (smalls)
83.9
157
188
189
172
GFP1-10
+ aTc
gfpa/fsca*1000 (smalls)
70.6
159
186
205
177
GFP1-11
+ aTc
gfpa/fsca*1000 (smalls)
72.8
169
193
208
175
GFP1-12
+ aTc
gfpa/fsca*1000 (smalls)
80.1
172
206
199
166
blankpdCas9-9
+ aTc
gfpa/fsca*1000 (smalls)
2526
3227
3986
4592
4520
blankpdCas9-10
+ aTc
gfpa/fsca*1000 (smalls)
2700
3078
4044
4486
4543
blankpdCas9-11
+ aTc
gfpa/fsca*1000 (smalls)
2892
3028
3795
4291
4553
blankpdCas9-12
+ aTc
gfpa/fsca*1000 (smalls)
2641
3131
4038
4531
4614
pdCas9
+ aTc
gfpa/fsca*1000 (smalls)
6.26
9.36
11.1
8.92
9.11
GFP1-9
- aTc
gfpa/fsca*1000 (smalls)
72.8
177
221
228
194
GFP1-10
- aTc
gfpa/fsca*1000 (smalls)
60.5
187
220
241
193
GFP1-11
- aTc
gfpa/fsca*1000 (smalls)
94.8
192
245
245
199
GFP1-12
- aTc
gfpa/fsca*1000 (smalls)
79.6
193
235
222
193
blankpdCas9-9
- aTc
gfpa/fsca*1000 (smalls)
2184
3100
4012
4663
4666
blankpdCas9-10
- aTc
gfpa/fsca*1000 (smalls)
2188
2957
4160
4554
4386
blankpdCas9-11
- aTc
gfpa/fsca*1000 (smalls)
2004
3047
4035
4565
3842
blankpdCas9-12
- aTc
gfpa/fsca*1000 (smalls)
2039
3051
4107
4630
4512
pdCas9
- aTc
gfpa/fsca*1000 (smalls)
5.64
9.13
9.66
8.24
7.79
GFP1-9
+ aTc
gfpa/ssca*1000 (smalls)
47.7
75.1
78.8
75.1
64.8
GFP1-10
+ aTc
gfpa/ssca*1000 (smalls)
46.3
76.2
79.1
81
66
GFP1-11
+ aTc
gfpa/ssca*1000 (smalls)
48
78.7
82.6
83.3
66.4
GFP1-12
+ aTc
gfpa/ssca*1000 (smalls)
51.5
85.9
96.7
93.4
71.1
blankpdCas9-9
+ aTc
gfpa/ssca*1000 (smalls)
1273
1469
1666
1779
1718
blankpdCas9-10
+ aTc
gfpa/ssca*1000 (smalls)
1226
1413
1636
1719
1688
blankpdCas9-11
+ aTc
gfpa/ssca*1000 (smalls)
1182
1382
1605
1704
1739
blankpdCas9-12
+ aTc
gfpa/ssca*1000 (smalls)
1182
1391
1645
1714
1790
pdCas9
+ aTc
gfpa/ssca*1000 (smalls)
3.43
4.38
4.82
3.89
3.69
GFP1-9
- aTc
gfpa/ssca*1000 (smalls)
46.3
80.9
89.5
87.5
71
GFP1-10
- aTc
gfpa/ssca*1000 (smalls)
45.6
80.6
89.6
91.3
71.4
GFP1-11
- aTc
gfpa/ssca*1000 (smalls)
50.4
84.4
101
95
74
GFP1-12
- aTc
gfpa/ssca*1000 (smalls)
55.3
93.8
108
100
83.8
blankpdCas9-9
- aTc
gfpa/ssca*1000 (smalls)
1250
1375
1606
1758
1715
blankpdCas9-10
- aTc
gfpa/ssca*1000 (smalls)
1215
1320
1606
1734
1635
blankpdCas9-11
- aTc
gfpa/ssca*1000 (smalls)
1203
1319
1563
1718
1425
blankpdCas9-12
- aTc
gfpa/ssca*1000 (smalls)
1230
1338
1595
1754
1661
pdCas9
- aTc
gfpa/ssca*1000 (smalls)
3.05
4.55
4.63
3.67
3.35
GFP1-9
+ aTc
gfpa (bugs)
2.17
3.7
5.12
5.27
5.68
GFP1-10
+ aTc
gfpa (bugs)
2.1
3.82
5.06
5.77
5.85
GFP1-11
+ aTc
gfpa (bugs)
2.17
3.92
5.48
6.04
5.89
GFP1-12
+ aTc
gfpa (bugs)
2.07
4.2
5.49
5.95
5.85
blankpdCas9-9
+ aTc
gfpa (bugs)
70.3
82.2
105
120
133
blankpdCas9-10
+ aTc
gfpa (bugs)
74.4
78.2
102
122
137
blankpdCas9-11
+ aTc
gfpa (bugs)
75.6
76.9
102
114
130
blankpdCas9-12
+ aTc
gfpa (bugs)
75.3
79.9
103
120
136
pdCas9
+ aTc
gfpa (bugs)
0.169
0.26
0.396
0.319
0.34
GFP1-9
- aTc
gfpa (bugs)
2.26
4.28
6.12
6.62
6.7
GFP1-10
- aTc
gfpa (bugs)
2.01
4.53
6.07
7.13
6.9
GFP1-11
- aTc
gfpa (bugs)
2.72
4.59
6.58
7.36
7.43
GFP1-12
- aTc
gfpa (bugs)
2.16
4.63
6.39
7.15
7.22
blankpdCas9-9
- aTc
gfpa (bugs)
63.4
76.5
107
127
141
blankpdCas9-10
- aTc
gfpa (bugs)
64.1
74.2
112
127
134
blankpdCas9-11
- aTc
gfpa (bugs)
61.2
77.2
110
127
126
blankpdCas9-12
- aTc
gfpa (bugs)
63.3
78.4
110
128
138
pdCas9
- aTc
gfpa (bugs)
0.157
0.26
0.357
0.331
0.287
GFP1-9
+ aTc
gfpa/fsca*1000 (bugs)
88.5
159
205
206
216
GFP1-10
+ aTc
gfpa/fsca*1000 (bugs)
75.6
163
197
228
220
GFP1-11
+ aTc
gfpa/fsca*1000 (bugs)
77.3
172
210
232
217
GFP1-12
+ aTc
gfpa/fsca*1000 (bugs)
83.4
174
215
227
223
blankpdCas9-9
+ aTc
gfpa/fsca*1000 (bugs)
2559
3229
4012
4818
5003
blankpdCas9-10
+ aTc
gfpa/fsca*1000 (bugs)
2751
3078
4107
4768
5130
blankpdCas9-11
+ aTc
gfpa/fsca*1000 (bugs)
3006
3014
3787
4456
4927
blankpdCas9-12
+ aTc
gfpa/fsca*1000 (bugs)
2695
3129
4076
4738
5032
pdCas9
+ aTc
gfpa/fsca*1000 (bugs)
6.33
10.1
15.8
12.9
14.1
GFP1-9
- aTc
gfpa/fsca*1000 (bugs)
78.4
182
242
263
257
GFP1-10
- aTc
gfpa/fsca*1000 (bugs)
64.6
194
236
278
259
GFP1-11
- aTc
gfpa/fsca*1000 (bugs)
101
195
263
287
277
GFP1-12
- aTc
gfpa/fsca*1000 (bugs)
82.5
195
250
270
274
blankpdCas9-9
- aTc
gfpa/fsca*1000 (bugs)
2213
3106
4099
5045
5411
blankpdCas9-10
- aTc
gfpa/fsca*1000 (bugs)
2212
2976
4289
5022
5084
blankpdCas9-11
- aTc
gfpa/fsca*1000 (bugs)
2022
3060
4164
4964
4574
blankpdCas9-12
- aTc
gfpa/fsca*1000 (bugs)
2061
3053
4158
4990
5205
pdCas9
- aTc
gfpa/fsca*1000 (bugs)
5.59
10
14.4
13.3
12.3
GFP1-9
+ aTc
gfpa/ssca*1000 (bugs)
49
73.4
74.8
71.9
69
GFP1-10
+ aTc
gfpa/ssca*1000 (bugs)
48.4
74.5
74.4
77.2
68.9
GFP1-11
+ aTc
gfpa/ssca*1000 (bugs)
49.8
77.1
79.2
80.5
69.6
GFP1-12
+ aTc
gfpa/ssca*1000 (bugs)
53
84.8
90.9
88.6
74.1
blankpdCas9-9
+ aTc
gfpa/ssca*1000 (bugs)
1267
1433
1538
1591
1583
blankpdCas9-10
+ aTc
gfpa/ssca*1000 (bugs)
1210
1380
1489
1535
1541
blankpdCas9-11
+ aTc
gfpa/ssca*1000 (bugs)
1170
1359
1497
1514
1583
blankpdCas9-12
+ aTc
gfpa/ssca*1000 (bugs)
1174
1356
1512
1531
1634
pdCas9
+ aTc
gfpa/ssca*1000 (bugs)
3.25
4.27
5.57
4.17
4.13
GFP1-9
- aTc
gfpa/ssca*1000 (bugs)
48.5
77.5
84
84.1
77.8
GFP1-10
- aTc
gfpa/ssca*1000 (bugs)
47.8
77.4
83.9
88.1
76.9
GFP1-11
- aTc
gfpa/ssca*1000 (bugs)
51.9
80.9
94.2
92.9
82.8
GFP1-12
- aTc
gfpa/ssca*1000 (bugs)
56.8
90.5
101
94.2
90.5
blankpdCas9-9
- aTc
gfpa/ssca*1000 (bugs)
1246
1323
1446
1575
1613
blankpdCas9-10
- aTc
gfpa/ssca*1000 (bugs)
1208
1269
1413
1553
1529
blankpdCas9-11
- aTc
gfpa/ssca*1000 (bugs)
1196
1255
1395
1529
1366
blankpdCas9-12
- aTc
gfpa/ssca*1000 (bugs)
1231
1282
1420
1561
1553
pdCas9
- aTc
gfpa/ssca*1000 (bugs)
2.82
4.23
4.92
4.26
3.62
It looks like the decrease in fluor for pdCas9-GFP1 coli manifests fairly early and persists for 6 hours. Lutz & Bujard’s paper use slightly less aTc for full induction of pTet, but if the decrease we see here were due to aTc inhibiting translation then we would expect to see a decrease in +aTc cultures with pdCas9. If pTet expression of aTc was removing trasncription/translation resources away from GFP production then you would also expect that to manifest in pdCas9. However it might be possible that the pdCas9 strains are experiencing the same decrease from aTc or growht burden but the strong GFP expression is masking the effect.
The other possibility is that something scientifically interesting, but not useful for iGEM purposes, is going on.
September 15
Objective: assemble decoy binding site arrays. Step 2: add prefix & suffix to arrays (retry)
CBC sets up PCR with 0.5 ng/uL green or blue PCR1 template with 0, 0.4, 0.7, 1.0 uL in 50uL reactions with same cycling conditions and oligo concs etc as before. Maybe with less template the dominant species in the tubes will be that with prefix & suffix added. Maybe
No difference between + template and notemplate tubes. No picture taken.
September 16
Objective: assemble decoy binding site arrays. Step 3: insert putative arrays in to pSB backbone
CBC prays, puts PCR2 products used on 9/13/14 in a digestion reaction with EcoRI/PstI and incubates 37C for 4h. PCR cleanup and CIP (buffer 2) for 20m then cleanup again. Both tubes have ~35 ng/uL and Dusseldorf ligation calculator calls for 12 ng of 100 (ish) bp insert  for 86 ng 2000 bp bb. Assuming that the large majority of the inserts lack the prefix and suffix, I will try 1 and 2 uL of insert for 1 of the bb. Also bb only and insert only, of course. Ligating 90m at RT
Shocking in to Z1; 30m on ice with 10uL of the ligation mix, 80m recovery before plating on LB+Cm
next morning: no colonies on any plates. Cm works
September 17
Objective: PCR to assemble decoy binding site array
MZ does PCR on 4 tubes: 0 um, 0.5 um, 1 um, and 2 um of ~200 ng/uL prefix oligo respectively + 50 uL standard mix (Q5 buffer and polymerase, repeat-spacer-repeat and spacer-repeat-spacer oligos, dNTPS, H2O)
PCR for 75 min
Gel to confirm (picture)
From left to right, 0, 0.5, 1, 2 uL of prefix oligo. Shortening of schmear is consistent with intended effect but not conclusive evidence thereof.
CBC takes 0.3 uL out of wells 0.5, 1, 2 to serve as template for new PCR2 with prefix & suffix oligos. MCF runs the gel
September 18
Objective: PCR to assemble decoy binding site array
CBC sets up 3 new PCR series, each with 0.3, 0.6, 0.9, 1.2 uL of tubes 2, 3, 4 from previous reaction as template.
Left to right is series from reactions templated with tube 2, 3, and 4. Gels look more like small pieces are dominating (primer dimer??) but we will try gibson anyway
Objective: PCR to assemble decoy binding site array
MZ with CBC get Mert’s homemade 1.33x gibson assembly mix and make reactions with backbone only, then backbone + insert
1uL of (conc) backbone (=EcoRI/PstI/CIP pSB1C3-R0040 made by CBC on [date] conc = [conc]) and 0.3uL of PCR products from series 2, 3, 4 at concs [conc conc conc] respectively.
Next morning: 100s of colonies.
Objective: PCR to assemble decoy binding site array
Master mix X13:  (of a standard 50 uL reaction)
494 uL dH2O
130 uL buffer (Q5)
13 uL dNTPs
3.25 uL each oligo (prefix & suffix oligos)
6.25 uL enzyme (Q5)
0.3 uL of each template (0.5, 1, 2)
Ran PCR with iGEM protocol (previously saved)
Some uneven loading in gel or at addition of template to tube step; either way it’s fine. Pool with PCR cleanup protocol and then can use this for GA. 0.5 and 1.0 series look the most promising for this.
September 19
Objective: confirm insert presence & size of decoy array inserts from gibson assembly yesterday
100s of colonies on bb+insert plates 2, 3, 4. None on insert-only plates or backbone-only.
Colony PCR with pSB1C3-up and -dn with 62C annealing temp (see 7/25/2014 entry, Farnitano figured this temp out for these oligos)
1ng pSB1C3-R0040, no template control, then 4 colonies from plate 2, 4 from 3, 6 from 4.
No amplification on anything. No picture
Objective: assemble decoy arrays. Insert putative arrays in to pSB1C3
Delta pools reactions from Garima’s PCR/gel produced yesterday in to three tubes. CBC sets up gibson assembly. 1 uL backbone and 0.3 insert as appropriate, H2O to 5 uL, vortexed briefly then added 2.5uL of this to 7.5 uL of master mix. Only enough master mix was in the freezer for 6 of these half-sized reactions so there is no insert-only control for reactions labeled 2 in Garima’s gel.
Transforming all 10 uL in to 80uL of competent Z1s.
Next morning, lots of colonies on all the places there should be colonies. Huzzah
September 21
Objective: confirm insert presence & size of decoy array inserts from gibson assembly
Picked 18 colonies from gibson/transformation performed on 9/19/14 yesterday and miniprepped. EcoRI/PstI digest for 3h then run 15m on 0.8% gel
Unadorned pSB1C3 is 41bp between EcoRI and PstI, 107 for minimal intended length of the four assembly oligos. It looks like there might be some incomplete digestion happening on these maybe, or just overloaded.
taaggatgatttctgg aattcgcggccgcttctagagCCATCTAATTCAACAAGAATTGGtgatgttaatCCATCTAATTCAACAAGAATTGGtgatgttaattactagtagcggccgctgca gtccggcaaaaaagggc
Left to right is 1-18. Two probably aberrant clones but they’re getting sequenced anyway. Sent to bio sequencing core seqd from pSB1C3-up-1 and -dn-1. Clones 1-4, 15-18.
September 22
Objective: confirm insert presence & size of decoy array inserts from gibson assembly
Mike picks 12 more colonies for miniprepping tomorrow
September 23
Objective: confirm insert presence & size of decoy array inserts from gibson assembly
Mike starts miniprep for 12 new colonies, charlie takes over after N3 addition & spin. Mike sets up EcoRI-HF/PstI-HF digest (cutsmart) for each clone.
Garima loaded this 1.3% TAE gel. Lane 7 (ie clone 25) has one big band at ~600 but also looks to have a small band down at the 1x size. Contamination? Unstable repeats? Another gel 0.8% has the same thing, but smearier.
Sequencing results come back!! As expected and consistent with the gel above, almost all colonies have a single insert, but clone 17 has 5! It works, kind of!!! The chromatograms taper off in quality around the repeats which makes it difficult for seqman to put it assemble properly, but manually getting called bases and finding repeats makes it clear. Clone 2 from yesterday was trash.
ctrl-F in text editor highlights each decoy site. To the left and right are prefix & suffixes
Objective: transfer decoy arrays to pSB1A2
CBC sets up prep scale digest of EcoRI/PstI of clones 17 & 18 & pSB1A2-R0040-antitracr. PCR cleanup and CIP treatment of clones 17 & 18 followed by a second PCR cleanup. Gel extraction of pSB1A2 backbone leaves only 22 ng/uL of dirty DNA by nanodrop. Tomorrow will try ligating, and also will set up another EcoRI/PstI digest of more pSB1A2-R0040-antitracr
Objective: optimize colony PCR conditions for testing pSB1C3-decoy arrays
Garima sets up master mix with 9x50uL rx. pSB1C3-up-1 and -dn-1 included at 0.25*9uL from 100uM stock.
45 uL Taq buffer
2.25 uL Taq polymerase
9 uL dNTPs
9 uL of each pSB1C3-up-1 and -dn-1
375.75 uL dH2O
9 uL template in positive tube/ 0 uL template in negative tubes
Ran colony pcr using saved protocol:
edited annealing step of protocol from 62 degrees to a gradient : 58-65 degrees
extension time of 15s (was 30)
Fairly crappy snapshot (crapshot?) but you can see that the primer dimers on the - gel are all at ~100bp and the product on +template are at 2-300 which is consistent with R0040-antitracr size. the best amplification took place at higher temps. New protocol: 65C with 15s extension for short pieces, next try with some actual colonies.
September 24
Objective: transfer decoy arrays to pSB1A2
Setting up ligation reaction with EcoRI/PstI/CIP treated clone 17 and 18 from yesterday (5x and 1x decoys respectively) and EcoRI/PstI/gel-purified pSB1A2 bb. Also including controls below
used ligation calculator from dusseldorf
17 (5x)
18 (1x)
1A2 bb
length
~500
~100
2252
conc
84.1
63.8
22.8
ng needed
66
13
100
using uL
1
0.5
5
thing
bb+ins 17
insert only 17
bb+ins 17
insert only 17
bb only
backbone
5uL
0
5
0
5
insert
1uL
1
0.5
0.5
0
T4 buffer
2uL
2
2
2
2
enzyme
1uL
1
1
1
1
h2o
11uL
16
11.5
16.5
12
Mixed andl et sit for 3h at room temp on bench. Took 10uL and transformed with 80uL of frozen competent Z1s
next day: lots of colonies on bb+insert plates, none on bb-only or insert-only. huzzah
Objective: find more long decoy arrays
miniprepping 12 more colonies from plate 1.0, middle one from 9/18/14
EcoRI/PstI digest for 3h gives all small bands at 1x size. No picture taken, all tubes discarded.
September 25
Objective: find more long decoy arrays
colony PCR on 4 colonies from plate 1.0 next to no-template, 1ng of pSB1C3-R0040-antitracr, pSB1C3-5xGFP1 decoy, and pSB1C3-1xGFP1 decoy. 10s extension at 64C
 No amplification on any of the colonies?
no template, pSB1C3-R0040-antitracr, 5x, 1x, then 4 colonies. At first I thought this was no amplification but actually I think it worked but all of the colonies are 1x or maybe a 2x for colony 4. Threw them all away in any case and before I had the realization that they amplified I already set up the next PCR with 8 colonies on a 58-65C gradient.
left to right is 65 to 58. My strong suspicion is that the long band in lane 1 is from a long template rather than 65. Picking colonies 1 and 3 for further investigation. Labeled 43 and 44 on the tubes, because that’s where we are in the screening.
Objective: confirm insert presence & size of decoy array inserts from gibson assembly
sequencing clone 25 with pSB1C3-up and -dn, dugsim job 1827
It might be better to take the 5x repeat and then expand it with some other scheme.
It is possible to do this by stepwise dig/lig/confirm cycles with XbaI/PstI dig’d insert and SpeI/PstI dig’d backbone (gel purified).
Another possible trick to speed up the expansion would be to cycle from 16C to 37C a couple of times with XbaI/SpeI dig’d pSB1C3-5xGFP1 and also add XbaI/SpeI & T4 ligase and a third enzyme that cuts somewhere else in the backbone.
When XbaI and SpeI ligate it will destroy the recognition site. Any sites re-ligating to self will be cleaved in the next 37C cycle. The last step in the cycling reaction will be a long digest.
Follow that with a ligation to XbaI/SpeI/CIP treated backbone.
Circularization is possible with this scheme
Third possibility: cycle XbaI/PstI/CIPd insert with backbone, SpeI, PstI, ligase and cycle
XbaI ligs to SpeI and destroys site. XbaI ligs to XbaI and gets re-cut, same with PstI. On re-cutting this would restore the phosphate removed by CIP so that doesn’t help much. Hmm
…..or just the stepwise dig/lig/shock will work. I think the problems of circularization will be there no matter what I do with CIP & including restriction enzymes the ligase.
September 26
Objective: Confirm pSB1A2-GFP1 decoy arrays
Yesterday picked 4 colonies from transformation of 5x and 1x repeat arrays and set up with EcoRI/PstI digest, alongside colonies 43 and 44 from yesterday’s colony PCR and pSB1C3 5x and 1x parents.
Slightly hard to read on the picture, but they’re loaded in the order mentioned above. 43 might be longer than 5x, 44 looks like 1x so discarding.
Zoom in on the 5x on 1A2 and Colony 1 might be slightly shorter than colony 4? Making bacteria with both of these just in case. Should probably sequence as well. pSB1C3-up-1 binds to 1A2, but not pSB1C3-dn
Shocking Z1 to make a couple of strains:
pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-5xGFPdecoy-1
pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-5xGFPdecoy-4
pdCas9-GFP1, pSB4K5-K(gfp), pSB1A2-1xGFPdecoy-1
pdCas9, pSB4K5-K(gfp), pSB1A2-5xGFPdecoy-1
pdCas9, pSB4K5-K(gfp), pSB1A2-5xGFPdecoy-4
pdCas9, pSB4K5-K(gfp), pSB1A2-1xGFPdecoy-1
September 27
Objective: extend decoy arrays
XbaI/PstI digest of 1C3-5xGFP1decoy followed by CIP, then SpeI/PstI of 1C3-5xGFP1 (same plasmid) with NO GEL PURIFICATION makes the backbone. I am hoping that since SpeI to PstI is only 18bp that the PCR cleanup kit gets rid of the fragment. Ligating with 3-fold molar excess of CIP-treated insert for 20m then shocking 10uL in to 70uL Z1. This is rushing the protocol; the other 10uL of the ligation mix will sit at RT for 90m in case the rushed version fails.
Update: it looks like it did not fail; there are some colonies on bb-only and ins-only but ~10x as many on b+i.
September 28
Objective: test 1x and 5x decoy arrays
4 colonies each picked from plates from
pdCas9-GFP1, pSB4K5-K608012, pSB1A2-5xGFPdecoy-1 G51
pdCas9-GFP1, pSB4K5-K608012, pSB1A2-5xGFPdecoy-4 G54
pdCas9-GFP1, pSB4K5-K608012, pSB1A2-1xGFPdecoy-1 G11
pdCas9, pSB4K5-K608012, pSB1A2-5xGFPdecoy-1 C51
pdCas9, pSB4K5-K608012, pSB1A2-5xGFPdecoy-4 C54
pdCas9, pSB4K5-K608012, pSB1A2-1xGFPdecoy-1 C11
….and grown overnight (started 9/27/14). 9:15am today diluted 1/200 in 3 mL fresh LB+AmpKanCm. 3h45m in 37C shaker. 1mL collected, spun down, and resuspended in PBS. Flowed 75000 events and got something that looks super promising. Means here are written down from stats view on MQ during acquisition and might change after full processing. There is a lot of variability in G51
2014-09-28 flow data, not fully processed
(constitutive GFP reporter)
pdCas9 with or without GFP-targeting crRNA
decoy binding site count & colony
expected GFP if working
tube label
unprocessed mean gfp
avg
std
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-5xGFP1decoy-1
derepressed
G51-1
42
69.25
28.14
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-5xGFP1decoy-1
derepressed
G51-2
50
 
 
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-5xGFP1decoy-1
derepressed
G51-3
83
 
 
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-5xGFP1decoy-1
derepressed
G51-4
102
 
 
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-5xGFP1decoy-4
derepressed
G54-1
64
68.33
5.13
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-5xGFP1decoy-4
derepressed
G54-2
67
 
 
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-5xGFP1decoy-4
derepressed
G54-3
nan
 
 
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-5xGFP1decoy-4
derepressed
G54-4
74
 
 
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-1xGFP1decoy-1
near repressed levels
G11-1
16
16.00
0.82
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-1xGFP1decoy-1
near repressed levels
G11-2
15
 
 
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-1xGFP1decoy-1
near repressed levels
G11-3
17
 
 
pSB4K5-K608012
pdCas9-GFP1
pSB1A2-1xGFP1decoy-1
near repressed levels
G11-4
16
 
 
pSB4K5-K608012
pdCas9
pSB1A2-5xGFP1decoy-1
unrepressed
C51-1
131
141.75
7.46
pSB4K5-K608012
pdCas9
pSB1A2-5xGFP1decoy-1
unrepressed
C51-2
143
 
 
pSB4K5-K608012
pdCas9
pSB1A2-5xGFP1decoy-1
unrepressed
C51-3
145
 
 
pSB4K5-K608012
pdCas9
pSB1A2-5xGFP1decoy-1
unrepressed
C51-4
148
 
 
pSB4K5-K608012
pdCas9
pSB1A2-5xGFP1decoy-4
unrepressed
C54-1
153
149.00
3.16
pSB4K5-K608012
pdCas9
pSB1A2-5xGFP1decoy-4
unrepressed
C54-2
150
 
 
pSB4K5-K608012
pdCas9
pSB1A2-5xGFP1decoy-4
unrepressed
C54-3
146
 
 
pSB4K5-K608012
pdCas9
pSB1A2-5xGFP1decoy-4
unrepressed
C54-4
147
 
 
pSB4K5-K608012
pdCas9
pSB1A2-1xGFP1decoy-1
unrepressed
C11-1
150
153.25
3.95
pSB4K5-K608012
pdCas9
pSB1A2-1xGFP1decoy-1
unrepressed
C11-2
152
 
 
pSB4K5-K608012
pdCas9
pSB1A2-1xGFP1decoy-1
unrepressed
C11-3
159
 
 
pSB4K5-K608012
pdCas9
pSB1A2-1xGFP1decoy-1
unrepressed
C11-4
152
 
 
Saved G51-1 through -4, then colony 1 for one of each different strain as glycerol stocks. Labeled very minimally with the bolded labels in first sub-bullet list.
Next steps:
repeat
try with more decoy site numbers: 10x is on the way, need to also make 0x
September 30
Objective: sequence confirm pSB1A2-5xGFP1
Sequenced with pSB1C3-up-1. They might actually have 6 repeats; it’s hard to say for sure but it looks more like 6 than 5. The start of the well-defined seq results is pretty much right where the repeat array butts up against the prefix. The same problem applies when sequencing 1C3 backbones too.
Sequenced decoy array clones 2 and 43. 25 looks most like a 1x, 43 like another 6x.
Toss 25, transfer 43 to 1A2 just in case, and design oligos for 1C3 that bind farther out from prefix/suffix
Retrieved from " http://2014.igem.org/Team:Duke/Notebook/September "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
