"
Page
Discussion
View source
History
teams
Log in
 
Team:ITESM-CEM/Project/Data
From 2014.igem.org
TEC-CEM | Project
ITESM-CEM | Enzy7-K me
Project
3014
 
Description Details Materials & Methods Experiments Results & Discussion Human Practices Conclusions
PCR's
Digestions
Mammalian Cells Transfection
Protein Expression
NeoR characterization
Enzymes
PCR's for sequence isolation
Gel 1.High Fidelity PCRs Electrophoresis Gel . Well content: 1) NeoR PCR(786pb) 2)CMV Promoter PCR(588bp) 3) f1ori PCR 4) BGHPA PCR (228bp).
Biobricks in plasmid psB1C3
BGHPA
The ligation (view Material and Methods) of BGHPA was made with J04450 (psB1C3 with RFP protein). The BB_J04450 by itself produces red colonies and grows in the antibiotic Cloramphenicol. After the ligation and transformation, only the white colonies were selected. An extraction from a white colony growing on 50ml LB Cam+ was made in order to perform gel electrophoresis as shown in figure B.
Figure A. Isolated BGHPA transformed colony.
Figure B. BGHPA in plasmid psB1C3 gel electrophoresis. Lane 2.
CMV
The ligation (view material and methods) of CMV was made with J04450 (psB1C3 with RFP protein). The BB_J04450 by itself produces red colonies and grows in the antibiotic Cloramphenicol. After the ligation and transformation, only the white colonies were selected. An extraction from a white colony growing on 50ml LB Cam+ was made in order to perform gel electrophoresis as shown in figure B.
Figure A. Isolated CMV transformed colony.
Figure B. CMV in plasmid psB1C3 gel electrophoresis. Lanes 1-3.
NeoR
The ligation (view material and methods) of NeoR was made with J04450 (psB1C3 with RFP protein). The BB_J04450 by itself produces red colonies and grows in the antibiotic Cloramphenicol. After the ligation and transformation 8 (Figure A), only the white colonies were selected. An extraction from a white colony growing on 50ml LB Cam+ was made in order to perform gel electrophoresis as shown in Figure B. On lane 8, NeoR extraction is shown with the three bands isoforms, they are barely visible because the plasmid extraction was diluted 5 fold.
Figure A. Isolated NeoR transformed colony.
Figure B. NeoR in plasmid psB1C3 gel electrophoresis.
Lanes:
2-4. NeoR in psB1C3 digestion with XhoI.
6-8. NeoR in psB1C3.
10-12. CMV in psB1C3.
Back to top ↑
Gene isolation testing via digestion
CMV
A digestion proof of the CMV construction in pSB1C3 is shown on lane 1 (Figure A); when compared to the analysis performed in silico (Figure B), both band patterns coincided.
Figure A. Lane 5. CMV digestion with XhoI.
Figure B. In silico CMV-psB1C3 digestion.
NeoR
On the second, third and fourth lanes we can see the NeoR in pSB1C3 digestion with XhoI (Figure A). Even though dim bands can be seen because of the dilution of the original extraction, the banding pattern complies with what was expected from the in silico digestion of this construct (Figure B). The longest DNA fragment is undigested plasmid.
Figure A. Lanes 2-4. NeoR digestion with XhoI.
Figure B. In silico NeoR-psB1C3 digestion.
BGHPA
A digestion proof of the BGHPA construction in pSB1C3 is shown on lane 5 (Figure A). When compared to the analysis performed in silico (Figure B), both band patterns coincided.
Figure A. Lane 5. BGHPA digestion with XhoI.
Figure B. In silico BGHPA-psB1C3 digestion.
Back to top ↑
Mammalian Cells Transfection
7-Dehydratase in mammalian expression plasmid
Figure 1. Gel electrophoresis 0.8% agarose. 7-dehydratase plasmid extraction. Lanes: 1, Invitrogen 1Kb Plus Ladder (5uL). 2, unknown sample. 3, 7-dehydratase in pcDNA 3.1 Myc-His A plasmid extraction. (5 uL) 4, unknown sample.
The image below shows the presence of the correct ligation of 7-dehydratase in pcDNA 3.1 Myc-His A. No RNA or protein contamination is to be seen.
Figure 2. In silico restriction analysis of sense insertion of 7-dehydratase in pcDNA 3.1 Myc-His A. Three fragments are generated with XhoI and ScaI.
Figure 3. In silico restriction analysis of anti-sense insertion of 7-dehydratase in pcDNA 3.1 Myc-His A. Three fragments are generated with XhoI and ScaI.
Figure 4.7-dehydratase plasmid restriction. Lanes: 1, Invitrogen 1Kb Plus Ladder. 2, unknown sample. 3, unknown sample. 4, 7-dehydratase in pcDNA 3.1 Myc-His A plasmid extraction.
Comparing the analysis of restriction in silico of 7- dehydratase in pcDNA 3.1 Myc-His A (Figure 2) against the electrophoresis gel shown in the image above (Figure 4), the three expected bands can be seen (4072 bp, 1462 bp, 526 bp. Shown with arrows.) This shows that the enzyme was correctly inserted into the plasmid.
CMV, F1 origin of replication and BHGPA characterization
Once a construction which included the biobricks CMV promoter, and BGHPA polyadenylation signal was built, using GFP (BBa_E0240) as a marker of gene expression; the plasmid was transfected into monkey kidney cells (MARC-145 cell line) using the protocol previously described (see Materials and Methods) in order to assess its functionality in an eukaryotic environment. A photograph of the cells before transfection, taken with an inverted phase microscope, is shown in figure 1.
Figure 1. MARC-145 cells before transfection as seen with an inverted phase microscope.
Another photograph was taken using the inverted phase microscope 48 hours after transfection. Two control experiments were run: one without lipofectamine, and another one with lipofectamine and no plasmid. Figure 2 shows a comparison between both controls and the experiment. Cell growth exists for all samples because Geneticin selection cannot yet be measured.
Figure 2. From left to right, control cells treated with no lipofectamine, control cells treated with lipofectamine and no plasmid and cells exposed to lipofectamine and plasmid. Photographs were taken using an inverted phase microscope. Cell growth is seen for all the experiments.
Since cell growth exists for all cultures, the experiment was treated with UV light at 395nm in order to asses for fluorescence, which would be a proof of gene expression. The results of this analysis are shown in figure 3, where fluorescence can be seen. However, it can be easily seen that transfection efficiency is not as high as predicted.
Figure 3. Lipofected cells containing the GFP-expressing device exposed to UV radiation. Lipofection efficiency is low but construction funtionality is proven by the fluorescence observed.
Because gene expression exists, as seen in figure 3, we can correctly infer that both biobricks: CMV promoter, and BGHPA polyadenylation signal are properly working. If this was not the case, transcription would either not be possible or be unable to cease, because of a lack of termination signals. The existence of fluorescence reveals the presence of a functional GFP protein, which in turn proves that both bioparts are functional.
Finally, figure 4 shows the results for transfection of a construction including CMV promoter, BGHPA, and one of the enzymes cloned by the team (7-dehidratase). However, due to a lack of time, characterization of gene expression for this construction was not performed.
Figure 4. Lipofected cells containing the 7-dehydratase expressing plasmid construction as seen after transfection.
Back to top ↑
Recombinant Protein Expression
The samples were loaded in a 15% acrylamide gel, using Precision Plus Protein TM Dual Color Standards, for 20 minutes/90 V for the stacking gel and 60 minutes/150V for the resolving gel. The results are now presented:
Only the samples shown in the image before were the ones that presented notable bands that represent our protein of interest. As expected, the most remarked band is the one of the time 3, which means that inductions was taken correctly and more protein was produced, in other words, the protein was overexpressing. The band marked with the arrow represents a protein that weights approximately 34 kDa, which corresponds to the molecular weight of oxoacyl reductase according to ExPASy’s Compute pI/MW tool.
7-dehydratase was analyzed by SDS-PAGE in a 15% acrylamide gel using Precision Plus Protein TM Unstained Standards, for 20 minutes/90 V for the stacking gel and 90 minutes/110V for the resolving gel. Four samples were taken, including one before and after induction with IPTG, one from the soluble phase and one from the inclusion bodies; all prepared with Laemmli buffer. The results are shown in the image below.
No analysis of solubility was realized due to the quantity of protein. It was supposed to be done exactly the same than oxoacyl reductase, as the protein was found in a notable way in the inclusion bodies as shown in the lane 5.
For both enzymes no further work was done. After the identification of each of them, and after the analysis of solubility, the proteins have to be purified by affinity chromatography with a Invitrogen Ni-NTA Agarose column, taking the advantage of the histidine tag added to the protein. After the purification, enzymatic parameters would be determined by the interaction of the enzymes with the substrate; 7β-Hydroxycholesterol for cholesterol oxidase, and 5-Cholesten-3β-ol-7-one for 7-dehydratase and oxoacyl reductase.
Back to top ↑
NeoR Results
Results obtained from experimentation.
Image 1.Comparative graph showing cell number per ml in both groups.
Table 1.CFU count
Based on the CFU count, the neomycin concentration expected to properly work as a selective antibiotic is 100 ug/ml and above.  Any concentration below this point showed colonies in forms of clusters which were sometimes impossible to quantify and therefore it could be the cause of experimental difficulties to obtain isolated colonies.
Back to top ↑
Enzymes
BBa_K1313000
Figure 1. Growth of isolated white colony showing a successful transformation with cholesterol oxidase enzyme in LB plate with chloramphenicol (35 mg/ul).
Figure 2.Gel electrophoresis 0.8% agarose. Extraction of cholesterol oxidase enzyme ligated in psB1C3 (September 5th, 2014) Lane M: Marker Invitrogen DNA Ladder 1kb Plus (5 uL), Lane 5: Cholesterol oxidase in pSB1C3 plasmid (5uL).
The image below shows the presence of the correct ligation of Cholesterol oxidase in pSB1C3.
Figure 3. In silico restriction analysis of psB1C3 with cholesterol oxidase. Fragments generated with PstI and EcoRI.
Comparing the analysis of restriction in silico of cholesterol oxidase in pSB1C3 (Figure 3) against the electrophoresis gel shown in the image below, the two expected bands can be seen (2033 bp, 956 bp. Shown in boxes.) This shows that the enzyme was correctly inserted into the plasmid.
Image 4.Gel electrophoresis 0.8% agarose. Cholesterol oxidase restriction with EcoRI and PstI (September 23th, 2014) Lane M: Marker Invitrogen DNA Ladder 1kb Plus (5 uL), Lane 1: Undigested cholesterol oxidase (5 uL), Lane 2 and 3: empty, Lane 4: Cholesterol oxidase enzyme digestion with EcoRI and PstI (10 uL).
BBa_K1313001
Figure 1. Growth of isolated white colony showing a successful transformation with Oxoacyl-reductase in psB1C3; LB plate with cloramphenicol (35 ug/ml).
The sequence coding for oxoacyl-reductase was subcloned into the  BioBrick BBa_J04450, a psB1C3 plasmid containing a coding region for RFP and chloramphenicol resistance as a selection marker.  This ligation was further transformed into E. coli DH5α and grown both on LB plates, and liquid LB media with 1% v/v of antibiotic (35 ug/ml).  A photo of the colonies obtained after replatting the bacteria is shown. This is a valid proof of construction structure, since oxoacyl was previously contained in an ampicillin resistance-containing plasmid and colonies are white. This can only be explained by a functional chloramphenicol resistance (provided by psB1C3 plasmid), and an insertion that occurred in the middle of the RFP-coding region of the backbone.
BBa_K1313002
Figure 1. Growth of white colonies of 7-dehydratase enzyme in psB1C3; LB plate with cloramphenicol (35 ug/ml).
The ligation of 7-dehydratase was made with J04450 (psB1C3 with RFP protein). The BB_J04450 by itself, produces red colonies and grows in the antibiotic Cloramphenicol. After the ligation and transformation, only the white colonies were selected. 7-dehydratase was previously in pUC57 (Ampicillin Resistance),  this gives us more certainty that the white colony is in fact, the correct ligation of 7-dehydratase in pSB1C3, otherwise, colonies wouldn’t have grown due to cloramphenicol, or red colonies would have been seen.
Figure 2.Gel electrophoresis 0.8% agarose. 7-Dehydratase in pSB1C3 extraction. Lane M: Marker Invitrogen DNA Ladder 1kb Plus (5 uL), Lane 2: Plasmid extraction (5 uL).
The image above shows the presence of the correct ligation of 7-dehydratase in pSB1C3.No RNA or protein contamination is to be seen. Nevertheless, the sample concentration is high and less sample should be analyzed in the electrophoresis.
Figure 3. In silico restriction analysis of psB1C3 with 7-dehydratase. Three fragments are generated with XhoI.
Figure 4.Gel electrophoresis 0.8% agarose. Comprobation of 7- dehydratase in pSB1C3 digestion with Xho I.  Lane M: Marker Invitrogen DNA Ladder 1kb Plus (5 uL), Lane 1: Digestion of 7- dehydratase in pSB1C3.
Comparing the restriction in silico analysis of 7- dehydratase in pSB1C3 (Figure 3) with the electrophoresis gel shown in Figure 4, the three expected bands (1650 bp, 1000 bp, 250 bp. Shown with arrows.) were obtained. An additional band of approximately 3500 bp is seen at the top of the gel, this band corresponds to undigested plasmid. This shows that less plasmid has to be digested and less sample has to be loaded.
Back to top ↑
BBa_K1313003
Figure 1. Growth of red and white colonies of B0034 + Oxoacyl-reductase ligation in pSB1C3; LB plate with cloramphenicol (35 ug/ml).
The ligation of this enzyme and the RBS B0034 was in J00450, which has the RFP protein encoded. The left half of the Petri dish shows undesired colonies, because they are red-colored.  Nonetheles, the right half shows white colonies, representing a succesful ligation of B0034 and Oxoacyl Reductase given the fact that the RFP protein sequence was cut out during this restrction/ligation.
Figure 2.In silico restriction analysis of psB1C3 with B0034 + Oxoacyl-Reductase. Two fragments are generated with EcoRI and PstI.
Figure 3. 0.8% agarose gel electrophoresis. B0034 +Oxoacyl in psB1C3 extraction and digestion. Lane 1: 1 Kb plus DNA Ladder Invitrogen (10 ul). Lane 2: B0034 + Oxoacyl Reductase in psB1C3 extraction. Lane 3: B0034 + Oxoacyl Reductase in psB1C3 digestion with EcoRI and PstI
Results observed in Lane 2 are consistent with the expected ones, given that the band is of approximately 3000 bp and the expected one was of 3030. At the same time, lane 3 shows expected results from the in silico digestion. However, the 997 bp band is too thin to observe, the team believes this is due to low concentration of digested plasmid. However, the 2033 band can be clearly identified. These results were the beginning of the expected mammalian expression plasmid construction, however due to time issues, we were not able to ligate other Biobricks necessary.
Back to top ↑
 
Retrieved from " http://2014.igem.org/Team:ITESM-CEM/Project/Data "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
