"
Page
Discussion
View source
History
teams
Log in
 
Team:ITESM-CEM/Modeling
From 2014.igem.org
TEC-CEM | Modeling
ITESM-CEM | Enzy7-K me
Modeling
3325
 
Modeling Proteins Cholesterol Oxidase Oxoacyl Reductase 7-Dehydratase Enzymatic Kinetics
Modeling Proteins
When studying a protein function it is insufficient to analyze its aminoacidic sequence. Although it provides very useful information most protein functions, if not all, relay on their tridimensional structure which requires a much more complex set of tools both in silico and experimental. For this project it is important to have tridimensional models of the enzymes mainly because they are not well known proteins with a detailed characterization nor do they have experimental tridimensional structures proposed in the available international databases (in fact one of them is classified as an hypothetical protein). Knowing the possible structure of these enzymes is very useful to compare them to other enzymes found to have a sequence and function homology and to further extent this homology with their structure, therefore assuring to a certain degree that the proteins expressed will work the way it is needed.
Bioinformatic Tools
RCSB/PDB Database
This database provides a lot of useful information regarding the tridimensional structures of proteins as well as the relationships between the sequence, structure and function of proteins. For the project this database was mostly used to obtain the tridimensional models of comparable enzymes and to perform the sequence alignments as well as the homology analysis.
I-Tasser
This is an online platform that performs protein structure predictions based on the aminoacidic sequence. The end result of this platform are several tridimensional models each with a different probability of existance. The algorithims used by this platform allow it to deliver results in a matter of days. The 3D models of the project enzymes were obtained through I-Tasser.
GOR VI & XTalPred-RF
Both of these sites are used toperform secondary structure predictions based on the aminoacidic sequence of a protein. XTalPred-RF in particular offers some other parameters regarding the stability and hydrophobicity of the resulting protein.
Sequence Manipulation Suite
This site is a collection of programs for generating, formmating and analysing  DNA and protein sequences as their own website says. For the project it was used to translate DNA sequences to amino acidic sequences in fasta format.
Pymol 2.7.7
Pymol is a molecular visualization system which is free to use. This software allows the user to view and manipulate a tridimensional protein model .  The user can select certain parts of the structure manually or using the aminoacidic sequence. Interactions between different molecules are also an object of great interest for molecular biology and with Pymol it is possible to analyse them and observe them.  The 3D alignment and model analysis was made using Pymol as well as all the 3D images in this wiki.
 
Retrieved from " http://2014.igem.org/Team:ITESM-CEM/Modeling "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
