<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ITESM-CEM_Project_Materials"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ITESM-CEM/Project/Materials">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ITESM-CEM/Project/Materials&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ITESM-CEM/Project/Materials&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ITESM-CEM/Project/Materials&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ITESM-CEM/Project/Materials" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ITESM-CEM/Project/Materials</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>TEC-CEM | Project</TITLE></P><H3>ITESM-CEM | Enzy7-K me</H3><H1>Project 
        <H1N> 3014</H1N></H1> <UL><SUB><A href="https://2014.igem.org/Team:ITESM-CEM/Project">Description</A></SUB><SUB><A href="https://2014.igem.org/Team:ITESM-CEM/Project/Details">Details</A></SUB><SUB><A href="https://2014.igem.org/Team:ITESM-CEM/Project/Materials" style="color: #FFF;">Materials &amp; Methods</A></SUB><SUB><A href="https://2014.igem.org/Team:ITESM-CEM/Project/Experiments">Experiments</A></SUB><SUB><A href="https://2014.igem.org/Team:ITESM-CEM/Project/Data">Results &amp; Discussion</A></SUB><SUB><A href="https://2014.igem.org/Team:ITESM-CEM/Project/HP">Human Practices</A></SUB><SUB><A href="https://2014.igem.org/Team:ITESM-CEM/Project/Conclusions">Conclusions</A></SUB></UL><UL><SUB2><A href="#One" style="color: #FFF;">Transformation</A></SUB2><SUB2><A href="#Two" style="color: #FFF;">Miniprep</A></SUB2><SUB2><A href="#Three" style="color: #FFF;">Assembly</A></SUB2><SUB2><A href="#Four" style="color: #FFF;">PCR's</A></SUB2><SUB2><A href="#Five" style="color: #FFF;">Digestion</A></SUB2><SUB2><A href="#Six" style="color: #FFF;">Solubility Analysis</A></SUB2><SUB2><A href="#Seven" style="color: #FFF;">SDS</A></SUB2><SUB2><A href="#Eight" style="color: #FFF;">Protein Expression</A></SUB2></UL><TABLE width="100%" border="0" id="ContenidoSecciones"><DIV style="background-color: #f3f3e2; style=" width:95=""><A name="One"><H2>Transformation Protocol</H2></A><P style="text-align: justify; text-justify: inter-word;">All the previously assembled BioBricks were transformed into DH5α competent cells acquired from New England Biolabs (NEB ®). In order to do so, NEB ®’s transformation protocol (3) was used: 50 μl of competent cells are added to microtubes; then, 5 μl of each previously assembled device (DNA concentration between 200 and 300 pg/ml, as determined by spectrophotometry Abs600nm) are pipetted into the tube, which is then placed on ice for 30 minutes. Afterwards, the samples are submitted to 30 seconds of a 42°C heat shock; after which they are placed on ice for another 5 minutes. After incubation on ice, 950 μl of SOC medium is added. The tubes are placed at 37°C and 250 rpm for 60 minutes. Finally, 200 μl of each sample is plated into warm, solid LB media with 0.1% v/v of antibiotic (kanamycin 15 mg/ml, chloramphenicol 35 mg/ml or ampicillin 100mg/ml).
After 12 hours of incubation at 37°C, a single colony is isolated from each plate and cultured overnight in liquid LB media with the previously stated concentration of antibiotic. These liquid cultures are used for plasmid extraction and isolation.
</P><GOTOP><A href="#top">Back to top ↑</A></GOTOP><A name="Two"><H2>Miniprep Protocol</H2></A><P style="text-align: justify; text-justify: inter-word;">Common miniprep plasmid-DNA extraction was performed. To do so, an isolated colony obtained from the transformation step is transferred from the Petri dish into an Erlenmeyer flask containing LB medium with the selection antibiotic (0.1% v/v). The flask is incubated overnight at 37°C and 250 rpm; the resulting culture is centrifuged at 13500 rpm for 30 seconds, so that biomass can be recovered. The supernatant is discarded and cells are resuspended in 350 μl of STET buffer. The mixture is then transferred to a 1.5 ml microtube, where 5 μl of lysozyme (10 mg/ml) are added. The mixture is incubated during 3 minutes, after which the tube is transferred to a boiling water bath for 2 minutes in order to inactivate the enzyme. 
Afterwards, the sample is centrifuged at 13500 rpm for 10 minutes. The bacterial pellet is taken out of the liquid using a sterile micropipette, and 10 μl of RNase A (200ug/ml) are added. The mixture is incubated for 10 minutes at room temperature. Then, 40 μl of sodium acetate (3M), and 250 μl of isopropanol are added. The mixture is gently stirred and incubated for 10 minutes at room temperature. Afterwards, it is centrifuged for 10 minutes at 12400 rpm; the supernatant is discarded, and the pellet is washed 2 times with 1 ml of ethanol 70% v/v. Finally, the DNA is resuspended in 100 μl of nuclease-free water and quantified by spectrophotometry.</P><GOTOP><A href="#top">Back to top ↑</A></GOTOP><A name="Three"><H2>Assembly Protocol</H2></A><P style="text-align: justify; text-justify: inter-word;"> U: Upstream BB. D: Downstream BB. P: Plasmid. U is digested with SpeI and EcoRI. Reagents are added to a 0.5 ml PCR tube in the following order: 12.5 μl of nuclease free water, 4 μl of NEB® Buffer 2, 0.5 μl of BSA, 20 μl of Upstream BB, 1.5 μl of SpeI enzyme, and 1.5 μl of EcoRI enzyme. The content of the tube is gently mixed, and placed at a Thermoblock at 37°C for 75 minutes. After incubation, the tube is placed at a water bath at 80°C for 20 minutes so that the enzymes are inactivated. Finally, the digestion product is stored at -20°C.</P><P style="text-align: justify; text-justify: inter-word;">
Downstream BB is also obtained by digestion, now using XbaI, and PstI. The same procedure is used, but now 20 μl of Downstream BB, 1.5 μl of XbaI enzyme, and 1.5 μl of PstI enzyme were added to the tube.  The digestion product is also stored at -20°C.

This same protocol was followed to obtain the desired backbone by digesting a RFP-containing psB1C3 plasmid, using the restriction enzymes EcoRI and PstI; the product is stored at -20°C.

Then, ligation of the 3 previously obtained DNA fragments is then performed; since only one possible combination for BioBrick ligation exists (apart from relegation of two backbones) given the sequence of the cohesive ends generated by the restriction enzymes, there was no need to previously purify the DNA.
To ligate, 2 μl of water for molecular biology, 2 μl of NEB® T4 Ligase buffer, 8 μl of the first digestion product (U), 4 μl of the second digestion product (D), 5 μl of the third digestion product (P), and 1 μl of NEB® T4 Ligase were added to a 0.5 ml PCR tube. The mixture was incubated at 16°C for 16 hours, then inactivated for 10 minutes at 65°C and finally stored at -20°C.
</P><GOTOP><A href="#top">Back to top ↑</A></GOTOP><A name="Four"><H2>PCR for gene isolation.</H2></A><H4>Obtention of Biobricks by PCR</H4><P style="text-align: justify; text-justify: inter-word;">Every part was obtained from plasmid pcDNA 3.1 (-)/Myc-His A.</P><H4>Primers Design</H4><P><PIE><B>Table 1 annotations.</B>
A) Extra bases to allow cuts (salmon). B) Extra base to give space (light blue). C) Extra base to avoid methylation sites (royal blue). D) Extra base to avoid methylation site or start codon (green). E) EcoRI Site (light blue background). F) NotI Site (green background). G) XbaI Site (red background). H) Pst1 Site (yellow background). I) SpeI Site (purple background). J) When it comes to coding sequences like NeoR, extra bases need to be added to ensure there is a correct space between ribosome and ATG (yellow font color with black background). K) TTA TTA (salmon font color with black background) Works as a double stop codon.</PIE></P><H4>PCR non specific protocol</H4><P><PIE><B>Table 1.</B>This protocol is the New England Biolabs Taq2X Master Mix protocol.</PIE></P><P style="text-align: justify; text-justify: inter-word;">The reaction must be gently mixed, if necessary a quick spin can collect all liquid ti the bottom.</P><P style="text-align: justify; text-justify: inter-word;">Routine conditions for an average PCR are reported as follows:</P><GOTOP><A href="#top">Back to top ↑</A></GOTOP><A name="Five"><H2>General Digestion Protocol</H2></A><P style="text-align: justify; text-justify: inter-word;">All of the parts were digested with XhoI.</P><P style="text-align: justify; text-justify: inter-word;">Incubate at 37°C for 1:15 hours.</P><P style="text-align: justify; text-justify: inter-word;">Inactivate at 80°C for 20 minutes.</P><GOTOP><A href="#top">Back to top ↑</A></GOTOP><A name="Six"><H2>Recombinant Protein Expression Protocol</H2></A><H2>Materials</H2><H4>Reagents</H4><P style="text-align: justify; text-justify: inter-word;">
•  20 mM KH2PO4 and 10 mM KCl (pH 8) solution
•  300 mM KCl (pH 8), 20 mM Imidazole (pH 8) and 20 mM KH2PO4 solution
•  Biomass recovered from the 40 mL medium
•  Fresh Laemmli Buffer
•  Lysozyme (20 mg/mL)</P><H4>Equipment</H4><P style="text-align: justify; text-justify: inter-word;">
•  Centrifuge
•  Micropipettes
•  Conical tubes
•  Centrifugal tubes</P><H4>Procedure</H4><P style="text-align: justify; text-justify: inter-word;">
1. Centrifuge 40 mL medium 5000 rpm, 30 min. Remove supernatant.
2. Resuspend  pellet in 4 mL of a cold 300 mM KCl (pH 8), 20 mM Imidazole (pH 8) and 20 mM KH2PO4 solution.
3. Add 70uL of Lysozyme.
4. Incubate in shaker 200 rpm the biomass-lysozyme solution at 37° C for 60 minutes.
5. Centrifuge cell lysate at 14,000 rpm for 10 minutes. If the supernatant is not crystalline, the sample has to be centrifuged again. Store soluble phase at 4°C in a conical tube, covered with aluminum for chromatography.
6. Take 20 µL of the supernatant and mix with 20 µL of Laemmli buffer for SDS PAGE analysis.
7. The insoluble phase obtained from the previous centrifugation has to be washed twice with a 1% SDS solution and 20 mM KH2PO4 and 10 mM KCl (pH 8) solution. Then, mix insoluble phase with 20ul Laemmli buffer for SDS PAGE analysis.
8. Run a SDS PAGE with iGEM ITESM CEM’s protocol.</P><P style="text-align: justify; text-justify: inter-word;">
*Do not cease to contemplate that all samples should be perfectly mixed with the indicated reagents (Laemmli Buffer) before they are boiled for 5 minutes. In the case of the inclusion bodies, the boiling should be carried out for 10 minutes.
</P><GOTOP><A href="#top">Back to top ↑</A></GOTOP><A name="Seven"><H2>SDS-Page Protocol</H2></A><H4>Materials</H4><P style="text-align: justify; text-justify: inter-word;">

•  2 micro centrifuge tubes 
•  1 micropipettes set
•  2 beakers (50 ml )
•  1 Mini-Protean System
•  B-mercaptoethanol 
•  Running Buffer (1.5M Tris-HCl, pH 8.8) 
•  Stacking Buffer (0.5 M Tris-HCl pH 6.8) 
•  Running buffer 1X  (Tris-Glycine)
•  Laemmli buffer 
•  Blue Coomasie Solution
•  Deionized water 
•  SDS 10% 
•  PSA 10%
•  Molecular weight protein marker (Precision Plus Protein Dual Color Standard BIORAD)</P><P style="text-align: justify; text-justify: inter-word;">Experimental Procedure: 
•  Use gloves at all times. Acrylamide and bis-acrylamide are extremely toxic. 

1.  Check that the glasses (with spacers 1.0 mm), comb and module assembly are clean and dry. 
2.  Place the glasses on the module assembly. 
3.  Prepare the running gel solution as follows by combining the following components in the order they are shown. (mix gently to avoid bubble formation)</P><P><PIE><B>Table 1.</B>Preparation of  separating gel 15%</PIE></P><P style="text-align: justify; text-justify: inter-word;">
* The TEMED has to be added until the end, and immediately proceed to the next step, because polymerization will be initiated.</P><P style="text-align: justify; text-justify: inter-word;">
4.	With a micropipette, slowly add the solution to the space between the panes to the mark previously made over the glass.   
5.	Cover the separator gel solution with a little water and/or ethanol solution and add it slowly. It should avoid the presence of oxygen in the separating gel solution because not blocking free radicals inhibits polymerization.
6.	Wait until the gel solidifies (10-20 min).
7.	Dry the water added in Step 5 with filter paper. Be careful not to damage the gel.
8.	Prepare the stacking gel monomer solution by combining the following components (mix gently to avoid bubble formation).</P><P><PIE><B>Table 2.</B>Preparation of 1 stacking gel.</PIE></P><P style="text-align: justify; text-justify: inter-word;">
9.	With a micropipette, slowly add the solution into the space between the panes, until getting to the edge. 
10.	Install the comb (10 teeth, thickness 1.0 mm), taking care not to trap any bubbles, because they can cause distortion in the surface of the gel. 
11.	Wait until the gel solidifies (10-20 min). 
12.	Place the gel in the electrophoresis chamber. 
13.	Fill the upper chamber with running buffer 1X until covering the gel (3 mm above the gel) and the lower chamber to the marked signal. 
14.	Using a micropipette, slowly charge the molecular weight marker in the corresponding first well of the gel (10 ul). 
15.	Then, load 15 uL of each of the protein samples previously mixed with Laemmli buffer. 
16.	Close the electrophoresis chamber and connect it to the power source. The program would be, first; 20minutes at 90Volts and then 90 minutes at 130 Volts.
17.	When the power source program finishes; check that the front of glycine elecrophoretic front is at the end of the gel. Turn power source off and remove the gel from the camera. 
18.	Stain the gel with Coomassie solution overnight. Use the volume enough to cover the gel and cover with aluminium.
19.	Remove the Coomassie solution and now cover the gel with fading solution. After 30 min stirring, remove the solution and then recover it with more solution. Let stirr for another 30 min, until the protein bands have been clearly defined. 
20.	Visualize the proteins in the gel equipment Quantity One 1-D Analysis Software Bio-Rad.</P><GOTOP><A href="#top">Back to top ↑</A></GOTOP><A name="Seven"><H2>Expression of Recombinant Protein Protocol</H2></A><H4>Materials and Reagents</H4><P style="text-align: justify; text-justify: inter-word;">
-LB broth
-IPTG solution (1 M)
-Sterile microtubes (1.5 ml)
-Sterile test tubes (15 ml)
-Sterile Erlenmeyer Flasks (250 ml)
-Micopipettes (200 ul and 1000 ul)
-Sterile pipette tips
-Laminar flow hood
-Incubation chamber (or Shaker) at 37ºC
-Centrifuge</P><H4>Experimental Procedure</H4><P style="text-align: justify; text-justify: inter-word;">Firstly, 40ml of LB broth is poured into an Erlenmeyer flask; then a defined volume of ampicillin solution (100 mg/ml) is added until concentration of 100 ug/ml is reached. 
Once the antibiotic concentration is adjusted, the required bacterial strain is inoculated into the LB broth-containing flask. The Erlenmeyer flasks are incubated for 16 hours at 37°C and 150 rpm. 
With this 16h culture, another LB broth/Amp+ containing flask is inoculated until it reaches OD 0.5-0.6 in aproximately 3-4 hours. </P><P style="text-align: justify; text-justify: inter-word;">
Then, a defined volume of IPTG solution (1 M) is added to the 40 ml Erlenmeyer flask until the liquid reaches a concentration of 1 mM IPTG. Immediately, a 1 ml sample is taken from the flask in order to perform an SDS-PAGE analysis. The flask is then returned to the incubating conditions: 37°C and 150 rpm, where it remains for 6 hours.
Each hour, the OD has to be measured and a sample has to be taken for an SDS-analysis of expression induction.
After taking all samples, each one has to be centrifuged and the biomass has to be recovered. According to the different OD of each sample, the volume of laemmli Buffer for the SDS-analysis is defined. The SDS PAGE analysis is then performed. 
</P></DIV></TABLE><P> </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:ITESM-CEM/Project/Materials">http://2014.igem.org/Team:ITESM-CEM/Project/Materials</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ITESM-CEM/Project/Materials" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ITESM-CEM/Project/Materials" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ITESM-CEM/Project/Materials&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ITESM-CEM/Project/Materials&amp;oldid=399970" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>