"
Page
Discussion
View source
History
teams
Log in
 
Team:ITESM-CEM/Interlab
From 2014.igem.org
TEC-CEM | Interlab
ITESM-CEM | Enzy7-K me
Interlab
4040
 
Overview Protocols Assembly verification Controls Device analysis Device 1 Device 2 & 3 Conclusions Annexes
Overview
During 2014 iGEM competition, teams were requested to analyse the efficiency of 3 different genetic devices (BioBricks) using GFP as a marker of gene expression. Here, iGEM ITESM CEM team presents the results of this interlab fluorescence measurement study.
The three devices analysed are composed by a variable promoter, a gene encoding a mutant Green Fluorescence Protein (GFP) used as a marker of expression, and a plasmid backbone. Two promoters (BBa_J23101 and BBa_J23115, recently renamed BBa_K823005 and BBa_K823012 at iGEM’s catalogue) are used, both of them being members of a family of constitutive promoters described by Chris Anderson, member of iGEM Berkley Team, in 2006 (1). This family of parts is registered at the catalogue under the alphanumeric codes BBa_J23100 – BBa_J23119.
Two different plasmid backbones are used: a low-copy (psB3K3) and a high-copy plasmid (psB1C3). The GFP-expressing BioBrick remains the same for all devices (registered at the catalog as BBa_E0240), and is composed of a ribosome binding site (RBS), a mutant GFP gene, and two termination sequences.
The aim of this study is to report the relative efficiency of the following genetic devices:
1) Promoter BBa_K823005 in low-copy plasmid psB3K3
2) Promoter BBa_K823005 in high-copy plasmig psB1C3
3) Promoter BBa_K823012 in high-copy plasmid psB1C3
In order to do so, GFP (BBa_E0240) is used as a marker of gene expression or reporter gene, because of the ease of fluorescence measurement experiments.
GFP has long been used as a reporter of patterns of gene expression in both prokaryotes, were it is useful for characterization of promoters, enhancers and terminators; and in eukaryotes, were tissue-specific or time-specific gene expression can be traced (2). The basis of this procedure is the usage of GFP’s fluorescence as a reporter of activity of promoters and enhancers; the relative fluorescence of cells at different experimental conditions can be compared with statistical techniques, and so the efficiency of the parts can be tested.
Back to top ↑
Protocols
Transformation Protocol
All the previously assembled BioBricks were transformed into DH5α competent cells acquired from New England Biolabs (NEB ®). In order to do so, NEB ®’s transformation protocol (3) was used: 50 μl of competent cells were added to microtubes; then, 5 μl of each previously assembled device (DNA concentration between 200 and 300 pg/ml, as determined by spectrophotometry) were pipetted into the tube, which was placed on ice for 30 minutes. Afterwards, the samples were submitted to 30 seconds of a 42°C heat shock; after which they were placed on ice for another 5 minutes. After incubation on ice, 950 μl of SOC medium were added to each mixture.
The tubes were placed at 37°C and 250 rpm for 60 minutes. Finally, 200 μl of each sample were plated into warm, solid LB media with 0.1% v/v of antibiotic (kanamycin 15 mg/ml for device 1, and chloramphenicol 35 mg/ml for devices 2, and 3).
After 12 hours of incubation at 37°C, a single colony was isolated from each plate and cultured overnight in liquid LB media with the previously stated concentration of antibiotic. A part of these liquid cultures was used for plasmid extraction and isolation; while the rest of them was used to perform relative fluorescence measurements. Both extraction and fluorescence measurement protocols are described later.
MiniPrep Protocol
Common miniprep plasmid-DNA extraction was performed. To do so, an isolated colony obtained from the transformation step was transferred from the Petri dish into an Erlenmeyer flask containing LB medium with the selection antibiotic (0.1% v/v). The flask was incubated overnight at 37°C and 250 rpm; 10 ml of the resulting culture were centrifuged at 13500 rpm for 30 seconds, so that biomass could be separated. The supernatant was discarded and cells were resuspended in 350 μl of STET buffer. The mixture was then transferred to a 1.5 ml microtube, where 5 μl of lysozyme (10 mg/ml) were added. The mixture was incubated during 3 minutes, after which the tube was transferred to a boiling water bath for 2 minutes in order to inactivate the enzyme.
Afterwards, the sample was centrifuged at 13500 rpm for 10 minutes. The bacterial pellet was taken out of the liquid using a sterile micropipette, and 10 μl of RNase A were added. The mixture was incubated for 10 minutes at room temperature. Then, 20 μl of sodium acetate (3M), and 250 μl of isopropanol were added. The mixture was gently stirred and incubated for 10 minutes at room temperature. Afterwards, it was centrifuged for 10 minutes at 12400 rpm; the supernatant was discarded, and the pellet washed 2 times with 1 ml of ethanol 70% v/v. Finally, the DNA was resuspended in 100 μl of distilled water and quantified by spectrophotometry.
Assembly Protocol
Promoters from New device 1 and 2 (contained in a psB1C3 plasmid),were digested using the restriction enzymes SpeI and EcoRI. Reagents were added to a 0.5 ml PCR tube in the following order: 12.5 μl of water for molecular biology, 4 μl of NEB® Buffer 2.1, 0.5 μl of BSA, 20 μl of DNA (BioBrick BBa_K823005 in psB3K3 backbone), 1.5 μl of SpeI enzyme, and 1.5 μl of EcoRI enzyme. The content of the tube was gently mixed, and placed at a Thermoblock at 37°C for 75 minutes. After incubation, the tube was placed at a water bath at 80°C for 20 minutes so that the enzyme could be inactivated. Finally, the digestion product was stored at -20°C.
The GFP cassette, BBa_E0240 (contained in a psB1C3 plasmid) was also obtained by digestion, now using XbaI, and PstI. The same procedure was used, but now 20 μl of DNA (BioBrick BBa_K823005), 1.5 μl of XbaI enzyme, and 1.5 μl of PstI enzyme were added to the tube.  The digestion product was also stored at -20°C.
This same protocol was followed to obtain the desired backbone (psB1C3) by digesting a RFP-containing psB1C3 plasmid, using the restriction enzymes EcoRI and PstI; the product was stored at -20°C.
Then, ligation of the 3 previously obtained DNA fragments was then performed; since only one possible combination for BioBrick ligation existed (apart from relegation of two backbones) given the sequence of the cohesive ends generated by the restriction enzymes, there was no need to previously purify the DNA.
To ligate, 2 μl of water for molecular biology, 2 μl of NEB® T4 Ligase buffer, 8 μl of the first digestion product (BBa_K823005), 4 μl of the second digestion product (BBa_E0240), 5 μl of the third digestion product (psB1C3 backbone), and 1 μl of NEB® T4 Ligase were added to a 0.5 ml PCR tube.  The mixture was incubated at 16°C for 24 hours, then inactivated for 10 minutes at 65°C and finally stored at -20°C.
Fluorescence Measurement Protocol
In order to measure the relative fluorescence of the three different GFP-expressing genetic devices, a fluorometer was used. This was done at the Research Center CINVESTAV (Center of Advanced Studies and Investigations) of National Polytechnic Institute, at Mexico City. Firstly, isolated colonies of bacteria expressing BioBricks BBa_I20260 (Device 1), BBa_K823005 + BBa_E0240 (Device 2), and BBa_K823012 + BBa_E0240 (Device 3); as well as common Top10, and DH5α strains (negative controls) were cultured overnight at 37°C in a Shaker using 5 ml of LB media with 0.1% v/v of antibiotic (15 mg/ml kanamycin for device 1, and 35 mg/ml chloramphenicol for devices 2 and 3). Then, 1 ml of the resulting culture was further subcultured 2 hours at 37°C and 250 rpm in another 5 ml of LB media, with no antibiotic; while the rest of it was kept at 4°C.
After two hours, both sets of samples (stored at 37°C and 4°C respectively) were centrifuged at 13000 rpm for 1 minute to separate the biomass. As suggested in previous fluorometric studies (4), the supernatant was discarded, and the samples washed with 1 ml of PBS 1X three times using the same centrifuge conditions, so that all the remnants of LB liquid media could be removed (because LB is capable of emitting fluorescence). Finally, the sample was resuspended in 1 ml of PBS and diluted, the absorbance of 100 μl of the sample was determined at a wavelength of 600 nm using a spectrophotometer. Using the optical density data obtained, the original samples were serially diluted to create a standard curve with values of OD600 ranging from 0 to 1, with increasing intervals of 0.2 units. This data was later translated to cell number.
After verifying the OD600 of each dilution, samples of 150 μl for each biomass concentration were transferred to a fluorometer plaque, and their relative fluorescence was measured by triplicate using the appropriate filters (excitation 485 nm, emission 510 nm), while stirring. A standard curve was obtained for each bacterial strain (DH5α, Top10, and devices 1 to 3) using the statistical mean of the 3 measurements.
Back to top ↑
Assembly verification
In order to prove that BioBrick assembly and transformation had been properly done, a restriction map was generated. This was done by extracting, and digesting (as explained previously) each BioBrick, as well as each fully assembled device, with a unique combination of restriction enzymes. The resulting DNA fragments were analysed by DNA electrophoresis in 0.8% agarose gel, at a voltage of 110 V.
The results of this restriction analysis are shown in figure 1, where all electrophoresis results photographs are superposed in a single image, so that fragment sizes can be compared.
Device 1 (BBa_I20260) was digested using XbaI, PstI, and the combination of both. Device 2, using enzyme NcoI, and Device 3, with enzymes EcoRI and PstI acting together. Control samples with no enzyme are also included.
In order to compare fragment sizes, individual BioBricks were also digested. Promoter BBa_K823012, contained in backbone psB1C3 was digested using EcoRI and SpeI; and mutant GFP gene, BBa_E0240 (also contained in psB1C3 plasmid backbone) was digested with XbaI and PstI.
Figure 1. Results of DNA electrophoresis in 0.8% agarose gel for each BioBrick, and assembled device. Four 1 kb plus markers are included, and each lane is labelled with the BioBrick or device’s name. Device 1 is represented by the set of lanes labelled BBa_I20260.
Restriction analyses for each individual BioBrick (figure 1) appear to be correct, excepting a low number of upper bands generated by ligation of infrequent plasmid isoforms. However, electrophoresis results for the assembled devices appear blurred in most cases; since this result is generalized for all DNA samples obtained via alkaline extraction (including lanes for BBa_I20260), it is thought that an excessive contamination was obtained when performing plasmid extraction, as revealed by wide and dark RNA bands at the lower area of the gels, as well as widespread DNA degradation. Even though it is not possible to analyse the structure of the assembled devices as efficiently as was needed, it is clear that this is not a prove of the existence of errors in transformation protocols, since device 1 fluoresces, and still cannot be seen at the restriction assay. Device 3 is the only exception to these observations, since its control lane presents a band of the appropriate size; however, the restriction lane does not correspond with the predicted fragment sizes.
Given the previous analysis, the only possible explanation for the lack of fluorescence of devices 2 and 3 is an error in either the sequence or the cloning procedures for promoters in plasmid psB1C3; or an unexpected pattern of BioBrick ligation. It is then concluded that no efficient activity can be characterized for Promoter BBa_K823012 or for any of the BioBricks assembled with psB1C3 plasmid backbone.
Back to top ↑
Control measurements
Relative Fluorescence Units (RFU) for each bacterial strain, as determined by triplicate with the fluorometer for each overnight culture and 2-hour subculture sample are reported at Appendix A. Further, a statistical mean, as well as a value of standard deviation, was calculated for each triplicate of measurements; this data is also summarized in Appendix A. The relative fluorescence of both negative controls (DH5α and Top 10 strains) with respect to the optical density is presented in dispersion plots (figure 2). A smooth negative association was found for both sets of samples. The negative slope of the curve indicates that the biomass may absorb some of the basal fluorescence.
Figure 2. Mean Relative Fluorescence plot for negative control samples taken from overnight cultures at increasing optical densities. All correlation coefficients are shown.
Back to top ↑
Device analysis
The values of relative fluorescence for the DH5α negative control were then subtracted from the lectures obtained when analysing the modified bacterial strains. Dispersion plots for each genetic device built this way are presented in figure 3. An analysis of correlation reveals a strong positive association between relative fluorescence and OD600 in E. coli transformed with BioBrick BBa_I20260 (device 1) for both samples (overnight culture and 2-hour subculture). However, there appears to be no increase in fluorescence on the plots for BioBrick BBa_K823005 + BBa_E0240 (device 2) and BBa_K823012 + BBa_E0240 (device3). Finally, the curves of relative fluorescence for overnight culture and 2-hour subculture samples of bacteria expressing BioBrick BBa_I20260 are superposed in a single plot (figure 4). Correlation (R2) is kept almost constant; but there is a clear increase in the slope of the line when comparing the 37°C, 2-hour subculture with the regular overnight culture stored at 4°C.
Figure 3. Mean Relative Fluorescence plot for E. coli DH5α transformed with each of the three genetic devices, samples taken from overnight cultures at increasing optical densities. The basal levels of fluorescence were subtracted. All correlation coefficients are shown.
Figure 4. Comparative plot of Mean Relative Fluorescence for genetically modified E. coli DH5α strain, transformed with BioBrick BBa_I20260, at increasing optical densities. The upper curve belongs to a sample taken of the 2-hour subculture at 37°C in LB media; while the lower one was obtained directly from an overnight culture stored at 4°C. The basal levels of fluorescence emission were subtracted. All correlation coefficients are shown.
Back to top ↑
Device 1 (BBa_I20260)
Device 1 (BBa_I20260) shows a very strong positive linear relationship between biomass concentration (cell number) and relative fluorescence units (RFU) measured by the fluorometer, this result strongly suggests that promoter BBa_K23005 is properly working, and GFP is being expressed.  Furthermore, figure 4 shows a comparative plot where the levels of GFP expression for this device were tested at two different experimental conditions, but with a fixed number of cells: the lower curve belongs to a sample taken from bacteria stored on ice (4°C) during two hours, while the upper one belongs to a sample taken from bacteria incubated at 37°C and constant stirring during the same time. Since the cell number for each data point is the same for both measurements, this experiment was designed to assess the effect of metabolic activity in promoter function. A 2-fold (200%) increase was found for the slope of the curve of 37 °C samples. This strongly suggests that more molecules of mutant GFP exist per cell, and that promoter BBa_K823005 increases its level of gene expression as metabolic activity within the cell increases. According to these observations, an appropriate efficiency has been described for the levels of gene expression of the studied promoter.
Back to top ↑
Device 2 & 3 (BBa_K823005+BBa_E0240, BBa_K823012+BBa_E0240)
It was expected that a significant increase in fluorescence level would exist for device 2, since it is formed by the exact same arrangement of promoter BBa_K823005 and mutant GFP BBA_E0240, now contained in a higher copy-number plasmid (psB1C3); however, no fluorescence at all was found when compared with the negative control strains. This same result was found when analysing device 3 (composed of promoter BBa_K823012, combined with the same mutant GFP in the same high copy-number plasmid backbone). Since both promoter BBa_K823005 and mutant GFP had already been proved to work efficiently, as suggested by the curve obtained for device 1, it was hypothesized that an error could exist in the remaining structure.
Back to top ↑
Conclusions
Efficiency of promoter BBa_K823005 has been using mutant-GFP (BBa_E0240) and a psB3K3 plasmid backbone. The promoter is working and GFP is being expressed at a level that is positively linked with the rate of metabolic activity within the cell. This characterization could not be repeated when the promoter was assembled in a psB1C3 high copy-number plasmid, since no fluorescence was found.
No characterization was possible for promoter BBa_K823012 in plasmid psB1C3, because no fluorescence was found for any of the samples.
It is hypothesized that either an error existed in the cloning procedures used for BioBrick assembly in plasmid psB1C3, or a wrong ligation product was obtained when ITESM CEM team performed the assembly of BioBricks in psB1C3 plasmid.
Back to top ↑
Annexes
Measurements taken for 150 μl samples of overnight bacterial cultures after 2 hours of incubation at 4°C
Measurements taken for 150 μl samples of overnight cultures of negative controls, and E. coli DH5α transformed with BioBrick BBa_I20260. Samples were subcultured for 2 hours at 37°C and constant stirring.
Statistical means and standard deviations calculated for each triplicate of measurements of relative fluorescence, using samples taken from overnight cultures stored for 2 hours at 4°C.
Statistical means and standard deviations calculated for each triplicate of measurements of relative fluorescence, using samples taken from E. coli DH5α transformed with BioBrick BBa_I20260, subcultured for 2 hours at 37°C and constant stirring.
Back to top ↑
References
1. iGEM Berkeley Team 2006. Registry of Standard Biological Parts, Anderson Promoter Collection. Obtained on September 18th, 2014 from http://parts.igem.org/Promoters/Catalog/Anderson
2. Li J. Farmer A, Lindquist IE, Dukowic-Schulze S, Mudge J, Li T, Retzel E, Chen C. Characterization of a set o novel meiotically-active promoters in Arabidopsis. BMC Plant Biology. 2012; 12:104.
3. New England Biolabs. High Efficiency Transformation Protocol (C2987H/C2987I). Obtained on September 17th, 2014 from https://www.neb.com/protocols/1/01/01/high-efficiency-transformation-protocol-c2987?device=pdf
4. Bongaerts R, Hautefort I, Sidebotham JM, Hinton JC.  Green Fluorescence Protein as a Marker for Conditional Expression in Bacterial Cells. Methods Enzymol. 2002; 358:43-66.
 
Retrieved from " http://2014.igem.org/Team:ITESM-CEM/Interlab "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
