<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SYSU-China_Notebook_Labnotes"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SYSU-China/Notebook/Labnotes">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SYSU-China/Notebook/Labnotes&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-China/Notebook/Labnotes&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-China/Notebook/Labnotes&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SYSU-China/Notebook/Labnotes" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SYSU-China/Notebook/Labnotes</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="animat"><SPAN><P>ipsc</P></SPAN></DIV><DIV class="content_body" align="center"><DIV id="cont_column"><DIV class="chapter"><SPAN>Notebook/Labnotes</SPAN><H1>Introduction </H1><P>
Our experiments can be clearly separated as molecular experiments and cellular experiments. However, considering the amount of work,our experimenters are divided into 3 groups, one for construction of upstream elements, one for downstream elements, and one for cellular tests. Thus the labnote of each group is recorded by different writers. These three parts of labnotes have different style and expression, but they are all precious memories in this iGEM summer.
</P><H1> Molecule labnotes </H1><H2> Construction of upstream elements of tet system and suicide genes(noted by Dawei He) </H2><P>
Author:Dawei He
</P><P>
Members in charge:Dawei He,Yiming Fang,Haoqi Chen and Mingxuan Zhang
</P><H3> Introduction</H3><P>
Despite a good comprehension of the knowledge of Molecular biology and its relating experimental technologies，We never came to and stay in a Molecular biology lab and were completely lack of any experience. To achieve the designated tasks in extremely insufficient time(less than 3 months),We had to learn fast and start our work independently as soon as possible.
</P><H3> 2013.5.20-5.23
</H3><P>
Contents: Contact and learning of Molecular construction
</P><P>
1.	DNA extraction techniques: Plasmid extraction, DNA clean up(extraction from PCR or restriction endonuclease digestion systems,extraction from Ag gel, etc)  
</P><P>
2.	PCR techniques:
</P><P>
Primer(for molecular construction or sequencing) designing, PCR for molecular construction, colony PCR for verification.
</P><P>
3.	Endonuclease digestion
</P><H3> 2013.5.24 to 5.31
</H3><P>
Contents: First plasmid construction：pCDNA3.0-PGK-BSD
</P><P>
Annotation：pCDNA3.0 is a backbone plasmid ideal for transient expression in our designated cell lines including HEK293T,HepG2 and Bocs. All elements of our project are first to be incised into pCDNA3.0 and tested during transient expression in cell lines. Driven by promoter PGK,BSD functions as a resistantgene specially for our up-stream tet-system during stable transfection by lentivirus)
</P>Figure 1.Backbone plasmid pCDNA3.0(completed featured) align=&quot;center&quot; Figure 2.pCDNA3.0-PGK-BSD<P>
construction details:
</P><H4>1.PCR of PGK-BSD(915bp in length) from donor plasmid(as PCR template)
</H4><P>
Primers：Forward primer with restriction site BamHI and necessary base pairs from 5’ end
</P><P>
         Reverse primer with restriction site XhoI and necessary base pairs from 5’ end
</P><P>
Bases pairs should appear at both flanks of restriction sites to provide anchoring sites for endonuclease,3’ end of the restriction sites in a primer is already covered by base pairs, so some base pairs must be added to its 5’ end. Without such manually added base pairs from 5’ end, efficiency of endonuclease digestion of PCR product will be significantly reduced.
</P><H4>
1.1	PCR system set up
</H4><TABLE width="536"><TBODY><TR class="first-table-line"><TD>5×fastpfu PCR buffer </TD><TD>5μl </TD></TR><TR><TD>2.5mM dNTP：</TD><TD>2.5μl </TD></TR><TR><TD>10uM  PB  F primer </TD><TD>0.8μl </TD></TR><TR><TD>10uM  PB  R primer </TD><TD>0.8μl
</TD></TR><TR><TD> Template(plasmid)</TD><TD>4μl(5.36ng/ul,21.4ng in total. Acceptable quantity from 5-30ng)
</TD></TR><TR><TD> Fastpfu DNA polymerase </TD><TD>0.5μl </TD></TR><TR><TD> ddH20</TD><TD>11.4μl(up to 25ul)</TD></TR></TBODY></TABLE><H4>
1.2 temperature program set up
</H4><P>
Annealing temperature in main PCR cycles =Primer Tm-5℃
</P><P>
Pre-cycles with lower annealing temperature in PCR process of primers with restriction sites(or any primers that include bases pairs not complementary to their original templates ) are necessary to achieve both specificity and quantity demand of PCR product.  
</P><P>
During the first PCR cycle, primer binds only and incompletely to its original templates, so tm is lower compared to the one when primer binds total- complementarily to its product generated since the first cycle, which was why the idea of adding pre-cycles before main cycles came up. Although annealing temperature in pre-cycles can be calculated to details, the temperature 15℃ below annealing temperature in main cycles will be quite universal.
</P><P>
                                              Primer binding to its original template
</P><P>
                                                 Primer binding to its PCR product
</P><H4>
1.3	Endonuclease digestion of backbone plasmid and insert fragment（PCR product）
</H4><P>
PCR product PGK-BSD was purified using PCR clean up kit.
</P><TABLE width="536"><TBODY><TR class="first-table-line"><TD>10xFermentas Buffer G </TD><TD>3μL </TD></TR><TR><TD> pCDNA3.0</TD><TD>2μL（3500ng）</TD></TR><TR><TD> BamHI(Fer) （100% activity）</TD><TD>1μL </TD></TR><TR><TD> XhoI（Fer）（50-100% activity）</TD><TD>	2μL</TD></TR><TR><TD> ddH20 </TD><TD> 22μL（up to 30μL）</TD></TR></TBODY></TABLE><TABLE width="536"><TBODY><TR class="first-table-line"><TD>10xFermentas Buffer G </TD><TD>3μL </TD></TR><TR><TD> PGK-BSD </TD><TD>9μL（900ng）</TD></TR><TR><TD> BamHI(Fer) （100% activity）</TD><TD>1μL </TD></TR><TR><TD> XhoI（Fer）（50-100% activity）</TD><TD>	2μL</TD></TR><TR><TD> ddH20 </TD><TD> 15μL（up to 30μL）</TD></TR></TBODY></TABLE><P>
Both digestion systems were incubated in 37℃ for 2h 
</P>Figure 3. <P>
Result：backbone plasmid pCDNA3.0 was fully digested，but as the fragment excised from pCDNA3.0 was too short(which means too small in quality because it has the same amount of substance as the backbone’s) in length(less than 50bps),it cannot be identified in agarose gel-electrophoresis. When digested in a least one restriction site, plasmid turns from supercoil status to linearized and has lower mobility in gel-electrophoresis, so it can only be judged that backbone was definitely complete digested in at least one restriction site. Such assessment applies also to digestion of PCR product from whom the excised parts are almost never more than 50bps. 
</P><H4>
1.4	Ligation
</H4><P>
Ligation type：1 vector+1 insert fragment
</P><P>
Desired ratio of the amount of substance：
</P><P>
            Vector：insert fragment =1:5
</P><TABLE width="536"><TBODY><TR class="first-table-line"><TD>10xFermentas T4 ligation buffer </TD><TD>2μL </TD></TR><TR><TD> BamHI-(pCDNA3.0)-XhoI	</TD><TD>1μL（60ng）</TD></TR><TR><TD> BamHI-（PGK-BSD）-XhoI	</TD><TD>2μL（60ng）</TD></TR><TR><TD> Fermentas T4 ligase</TD><TD>	1μL </TD></TR><TR><TD> ddH20 </TD><TD>12μLup to 20μL）</TD></TR></TBODY></TABLE><P>
Ligation for 2h at room temperature(25℃)
</P><H4>
1.5	Transformation of ligation product 
</H4><P>
Transformation system
</P><TABLE width="536"><TBODY><TR class="first-table-line"><TD> DNA（ligation system）</TD><TD>2μL </TD></TR><TR><TD> Competent E.coli（Top10 strain）</TD><TD>	50μL </TD></TR></TBODY></TABLE><P>
After 2h ligation，ligation system,as well as Competent E.coli taken from -80℃ storage， was first incubated in ice for 5min before mixing together. 
</P><P>
Transformation system continued to be incubated in ice for 30mins,42℃heat shock for 60s，then again ice incubation for 5mins.
</P><P>
Transformation system was added 600μL antibiotic-free LB medium and put in 37℃ thermostat shaker for 30mins-resuscitation（expression of Amp resistance protein）
</P><P>
Transformation system was then centrifuged at 130,000g for 3mins.All but little supernatant was kept to suspend the E.coli，which was later transferred to the LB agar plate. The plate was put in 37℃ thermostat incubator overnight for colonies growth.
</P><H4>
1.6	Verification of the colonies（each as a single clone）
</H4>Figure 4. <P>
colony PCR set up 
</P><TABLE width="536"><TBODY><TR class="first-table-line"><TD>5×fastpfu PCR buffer </TD><TD>5μl </TD></TR><TR><TD>2.5mM dNTP</TD><TD>2.5μl </TD></TR><TR><TD>10uM  PGK-BSD  F primer  F primer </TD><TD>0.8μl </TD></TR><TR><TD>10uM  PGK-BSD  R primer </TD><TD>0.8μl
</TD></TR><TR><TD> Template （single-clone colony）</TD><TD>1μL（taken from the single-clone Colony suspended with 5μL ddH20
</TD></TR><TR><TD> Fastpfu DNA polymerase </TD><TD>0.5μl </TD></TR><TR><TD> ddH20</TD><TD>11.4μl(up to 25ul)</TD></TR></TBODY></TABLE>Figure 5<P>
Result：Clear band around 1000bp appeared for every 12 clone.
</P><H4>
1.7 Plasmid extraction and verification
</H4><P>
6 of the single-clone colonies were selected and added to 5ml Amp-LB medium and put to 37℃ thermostat shaker.
</P><P>
The bacteria were centrifuge after 16h-growth in the thermostat shaker.Plasmids were extracted using endotoxin-free kit.
</P><P>
The plasmids were later digested by BamHI and XhoI for verification.
</P><TABLE width="536"><TBODY><TR class="first-table-line"><TD>10xFermentas Buffer G </TD><TD>2μL </TD></TR><TR><TD> pCDNA3.0</TD><TD>0.5μL（400ng）</TD></TR><TR><TD> BamHI(Fer) （100% activity）</TD><TD>0.3μL </TD></TR><TR><TD> XhoI（Fer）（50-100% activity）</TD><TD>0.6μL</TD></TR><TR><TD> ddH20 </TD><TD> 16.6μL（up to 30μL）</TD></TR></TBODY></TABLE>Figure 6<P>
Result: Clear band around 1000bp appeared. Plasmids from every 4 clones were verified.
</P><P>
Plasmids were later sent for sequencing 
</P><H3> 
2013.6.1-6.14
</H3><P>
Construction of pCDNA3.0-pEF1a-tTA-PGK-BSD
</P><P>
            pCDNA3.0-pEF1a-rtTA-PGK-BSD
</P><P>
pCDNA3.0-pEF1a-tTA advanced-PGK-BSD
</P><P>
pCDNA3.0-pEF1a-eGFP-PGK-BSD
</P>Figure.7<P>
PCR system set up
</P><TABLE width="536"><TBODY><TR class="first-table-line"><TD>5×fastpfu PCR buffer </TD><TD>5μl </TD></TR><TR><TD>2.5mM dNTP：</TD><TD>2.5μl </TD></TR><TR><TD>10uM  PB  F primer </TD><TD>1.6μl </TD></TR><TR><TD>10uM  PB  R primer </TD><TD>1.6μl
</TD></TR><TR><TD> Template(plasmid)</TD><TD>2ul(50ng)</TD></TR><TR><TD> Fastpfu DNA polymerase </TD><TD>1.0μl </TD></TR><TR><TD> ddH20</TD><TD>28.8μl(up to 25ul)</TD></TR></TBODY></TABLE><P>
Temperature program set up
</P><TABLE width="536"><TBODY><TR class="first-table-line"><TD>（1×） </TD><TD>95℃ </TD><TD>5min </TD><TD> initial denaturation </TD></TR><TR><TD>（5×）</TD><TD>95℃ </TD><TD>20s </TD><TD> denaturation </TD></TR><TR><TD>  52℃ </TD><TD>20s </TD><TD> annealing </TD></TR><TR><TD>  72℃ &lt; /td&gt;
        </TD><TD> 35s  </TD><TD>elongation </TD></TR><TR><TD>（25×）
</TD><TD>95℃ </TD><TD> 20s   </TD><TD>denaturation </TD></TR><TR><TD> 63℃ </TD><TD> 20s e </TD><TD>annealing </TD></TR><TR><TD>72℃ </TD><TD> 35s  </TD><TD> elongation </TD></TR><TR><TD>  72℃   </TD><TD>（1×）</TD><TD> 5min </TD></TR><TR><TD>（1×）  
 </TD><TD> 4℃  </TD><TD> 10min </TD></TR></TBODY></TABLE>Figure.8<P>
Digestions
</P><P>
Incubated in 37℃ for 90mins
</P><P>
Ligation
</P><P>
Ligation type：1 vector+2 insert fragments
</P><P>
Desired ratio of the amount of substance：
</P><P>
            Vector：insert fragment 1：insert fragment 2=1：3：3
</P><P>
Verification of single-clone colonies 
</P><P>
PCR verification of Result All chosen colonies are positive, so were the sequencing results.
</P><H3> 
2013.6.18-7.1
</H3><P>
construction of pCDNA3.0-pCMV-RIP1
</P><P>
             pCDNA3.0-pCMV-RIP1-1X miR122 complete target
</P><P>
	pCDNA3.0-pCMV-RIP1-2X miR122 complete target
</P><H3> 
2013.7.5-7.31
</H3><P>
construction of  p199-pEF1a-tTA-PGK-BSD
</P><P>
            p199-pEF1a-rtTA-PGK-BSD
</P><P>
p199-pEF1a-tTA advanced-PGK-BSD
</P><P>
p199-pEF1a-RIP1-2XmiR122 targets
</P><P>
p199-pEF1a-RIP3-2XmiR122 targets
</P><P>
p199-pEF1a-eGFP-PGK-BSD
</P>Figure.9<H3> 

2013.8.1-8.22   A major problem: DNA Recombination of pEF1a
</H3><P>
Promoter EF1a we used to drive regulationg parts of tet-on or tet-off system are 537bp in length(provided by professor Lei xiao, Zhejiang University).By NCBI blast search，first 349bp of the 537bp-pEF1a was found to be CPPT，one of the essential elements of inserting backbone plasmid of lentivirus. While the function of CPPT cassette inside pEF1a is yet to figure out, the result of molecular construction suggested that CPPT, which exists both inside 537bp-pEF1a and p199 backbone plasmid, has high ratio of recombination that can interfere the normal inserting process of lentivirus vector and the expression of downstream genes.
</P>Figure 10. The recombination zone<P>
High-ratio recombination between the 2 copies CPPT in plasmid p199-pEF1a(537bp)-gene was first suspected as the forward sequencing primer often present 3 kinds of sequencing results: 
1.No signal(most)     2.Double signal(few)      3.single signal(very few)
</P>Figure.11<P>
The recombination mechanism has it proof that the existence of type c plasmid was confirmed by reverse sequencing result (The sequence of some expressed genes driven by pEF1a are short enough to be covered and passed through by reverse sequencing),and that of type B plasmid was confirmed by double sequencing signal. Such recombination mechanism can also be supported by digestion of XhoI restriction site which exists right between CPPT and the 188-bp key region of pEF1a,and in the downstream of expressed genes. </P><P>
Figure 12. (A).Hypothetical calculation of the result of digestion of XhoI restriction sites.(B).Agarose gel image of the result of digestion of XhoI restriction sites,and the verification of recombination
</P><P>
According to the sequencing and digestion result, the recombination stopped in the downstream of CPPT cassette，unlike a dsDNA whose recombination will terminate at the end of the strain. Due to variable terminating position of recombination on our plasmid p199-537bp pEF1a, the impact on pEF1a and the expression of the genes driven can be different. However,as the first XhoI restriction site appears right between CPPT cassette and 170bps-key region( The 170bps region ,damage of which may has devastating impact on the expression of downstream genes,is found in almost every type of pEF1a), any recombination mode whose terminating point passes through the end of CPPT cassette will destroy XhoI site, therefore the XhoI digestion test(figure.11 B) is acute of any damaging recombination. The variable termination of recombination on our plasmid has its proof that some of plasmids with a single signal of forward sequencing, turned out to be recombined in XhoI digestion test. 
</P>Figure 13. Multiple recombination modes<H3> 

2013.8.1-8.22   Solutions
</H3><P>
As the function of CPPT cassette inside pEF1a remains unclear,Several strategies were adopted to deal with the recombination of p199-537bp plasmid and keep such 537bps-pEF1a intact,but none of them has worked out ideally so far.
</P><P>
Strategy A
</P><P>
Reconstruct all p199-537bp pEF1a and transport them into competent bacteria MDS42 strain whose genes expressing recombinases are completed knocked off, instead of Top10 strain.
</P><P>
Result: Failure in collecting single-clone colony, as the competence level of MDS42 strain is extreme low.
</P><P>
Stategy B
</P><P>
Pick up massive single-clone colonies(Top10) and conduct plasmids extraction 
</P><P>
     Result: All plasmid in XhoI digestion test is positive in recombination.
</P><P>
Then we cutted the growth time in 37℃ thermostat shaker down to 8h in,as we hypothesized that the recombination process mainly occurred during bacteria growth in LB medium. But results were still the same.
</P><P>
Strategy C
</P><P>
Pick up all colonies in the plate and concentrate them for direct plasmids extraction. The plasmids are then retransported  to MDS42 strain, whose single-clone colonies will be selected again for plasmids extraction.
</P><P>
Status: currently still underway.
</P><P>
In such case , with the problem of recombination of 537bps-pEF1a unsolved, we chose to replace it with 188bps-pEF1a which was reported still possessing at least 30% of driving intensity of a full-length version pEF1a.And theoretically, the 188bps-pEF1a won’t be recombination capable.
</P><P>
Currently we have successfully constructed p199-188bps pEF1a-tTA-PGK-BSD.
</P><H2> 
Construction of downstream elements of tet system, miR-122 target construction and quantitive measurement(By Chen Haoqi and Luo Yingjie)
</H2><STRONG> 
Status: currently still underway.
</STRONG><H3> 
May
</H3><P>
May 1-3rd, International Labour’s Day, amplification of the very original plasmids we collected. This is the first time I tried transfection, this is the beginning of all our laboratory work!
</P><P>
The first 20 days of May, we worked on construction of the first plasmids pcDNA3.0-pCMV-emGFP-IRES-Suicide Gene. However, due to lack of experience and difficulty of manipulation of IRES, a sequence abundant in GC base, we kept failing for many days!
</P><P>
May 20th, Monday, we had a very important meeting with our advisor, Jiang Shuai. We made big changes of our plans and divided our teammates responsible for molecular cloning into 2 groups, each handles with a plasmid. And the cloning scheme is designed step by step so that we can test and characterize each parts in cells as soon as we successfully cloned it. The plan proved much more efficient and vital to our work.
</P><P>
May 28th, we get the first successful constructed plasmids: pcDNA3.0-pCMV-eGFP. This plasmid served as a control to indicate the transfection efficiency. Although it was indeed a very simple work and a tiny step for our schedule, it was a big step for newcomer and the growing SYSU-China team!
</P><H3> 
June
</H3><P>
Early June, we received the purchased plasmids Bax\Bax S184A\Caspase 3(P17 and P12 fragment) from Addgene. And the next few days we successfully cloned Bax and Bax S184A into pcDNA3.0, same position as eGFP.
</P><P>
The whole June was an epic full of failure、puzzle and pain. We carelessly mess up the plasmids and used the wrong pcDNA3.0-eGFP to do transfection . Also, the sequenced verified Bax and Bax S184A did’t work as expected. We had to solve some possible problems like removing tags or checking up the backbone of plasmid, but all that efforts ended in vain. What’s more, since the construction of caspase 3 was kind of complicated, it also took us a lot of time but paid us with disappointment.
</P><H3> 
July
</H3><P>
Early July, we finally found out the problem and learnt to store our plasmids in good order and proper method. At the same time, eGFP transfection worked as expected, which greatly inspired us. Also we started the miR122 target site construction and 
</P><P>
July 4th, after finishing the end-term examination, we decided to go camping for a good rest and have some fun! The stars were really bright that night!
</P><P>
Late July, miR122 target site construction and trial testing were completed. Thanks to the courses we’d taken from the previous obstacles, this time it was smooth and targets showed a good knock-down performance. We also did the first point-mutation PCR to stop translation of the emGFP from miR122 expression plasmids to prevent fluorescence interference. This prepared us for brick construction later.
</P><P>
July, while the target site construction was moving forward, we also started construction work on pPRIME plasmids. pTRE-Tight promoter was replaced with pTRE-2 and pTRE-3G, whose performance in on-off switching would be tested and compared later.
</P><H3> 
August
</H3><P>
August, while Bax and Bax S184, after many times of experiments, still did not function well, our advisor Jiang Shuai introduced us the new suicide gene, RIP1 and RIP3. Together with Apoptin(VP3) whose performance was unstable, we started the final construction of response plasmids: pPRIME-pTRE-2/Tight/3G-eGFP/RIP1/RIP3/VP3-target sites-PGK-Puro. The construction was a huge project and since we were already in a tight schedule, we had to work on it desperately. Fortunately, all the work paid off and by June 25th, we got all the response plasmids. 
</P><H3> 
September
</H3><P>
September, for a better characterization of our devices, Western-Blot and RT-qPCR, two powerful quantitation techniques, became our new challenge. 
</P><P>
September 7th, after two failure, we adjusted our program for miRNA and successfully got the first reliable data. Later we successfully quantitated the miR122 level that was expressed by a gradient concentration of expression plasmids and that was in hepatocyte. We confirmed that miR122 level in hepatocyte was undoubtedly sufficient to knock down suicide gene with our target-sites.
</P><H1> 
Cellular labnotes(noted by Kaixuan Lin)
</H1><P>
Author:Kaixuan Lin
</P><P>
Members in charge: Kaixuan Lin,Yuchen Zhao and Runwen Yao
</P><P>
13-6-4 TRE test Dox introduction does not affect the basal expression level of p199-TRE; Tet-on works well with dose increasement. 10ug/ml is enough for observable level of GFP; Tet-off works well, but it is very sensitive as the minimon level of Dox we added is enough for a full knockdown
</P><P>
4BaxS184A and Bax failed
</P><P>
13-6-7 Apoptin works
</P><P>
13-6-30 Bax, apoptin BaxS184A does not work
</P><P>
13-7-12 Isolating primary liver cell; Culturing hiPSCs from prof. Xiang
</P><P>
13-7-15 MiR-122 2 complete target with old pmiR-122 works well
</P><P>
13-7-26 Puro test fails in hepG2
</P><P>
13-8-02 Repeat target test using old pmir122 works
</P><P>
13-8-10 HepG2 test with TRE
</P><P>
13-8-12 Test for toxity for transfection reaget
</P><P>
13-8-15 P tight+regulatory test , rtTa+response test
</P><P>
13-8-17 Mutated Target test without GFP; 2-com target dilute 
</P><P>
13-8-22 Apoptin test in HepG2
</P><P>
13-8-23Reviving miPSCs
</P><P>
13-8-24 target test
</P><P>
13-8-26 little clone of hiPSCs was observed 
</P><P>
13-08-24 Rescule 2om-GFP
</P><P>
13-9-6 Enriching virus 
</P><P>
13-9-12 Different targets test
</P><P>
13-9-13 Retests
</P><P>
13-9-15 Downstrem ratio test
</P><P>
13-9-16 Cross-combination tests in different cell lines
</P></DIV></DIV></DIV><DIV id="address"><P>Sun Yat-Sen University, Guangzhou, China</P><P>Address: 135# Xingang Rd.(W.), Haizhu Guangzhou, P.R.China</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:SYSU-China/Notebook/Labnotes">http://2013.igem.org/Team:SYSU-China/Notebook/Labnotes</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SYSU-China/Notebook/Labnotes" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SYSU-China/Notebook/Labnotes" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SYSU-China/Notebook/Labnotes&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SYSU-China/Notebook/Labnotes&amp;oldid=243556" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>