<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SYSU-China_Project_result_stable_assay"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SYSU-China/Project/result/stable_assay">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SYSU-China/Project/result/stable_assay&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-China/Project/result/stable_assay&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-China/Project/result/stable_assay&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SYSU-China/Project/result/stable_assay" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SYSU-China/Project/result/stable assay</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="animat"><SPAN><P>ipsc</P></SPAN></DIV><DIV class="content_body" align="center"><DIV id="cont_column"><DIV class="chapter"><SPAN>Project/result/Test in different cells</SPAN><H1>iPSCs stable assay</H1><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 1.</STRONG> The fate of an unfortunate iPSC.  </P></DIV><H2>Mouse iPSCs</H2><H3>Generating transgenic Oct4-GFP mouse iPSCs</H3><P>Among numerous genes which the mammals expressed, Oct4 is one of the most ancient and essential transcription factors for embryonic stem cells maintenance and embryo development. Oct4 has been longwise recognized as a gatekeeper and marker for pluripotency. </P><P>Transgenic Oct4-GFP mice from <EM>Stem cell and functional genomics laboratory SYSU</EM> were harbored with an IRES-GFP fusion cassette downstream of the stop codon of endogenous Oct4 gene. When infected with retrovirus which expressed specific transcription factors(Oct4,Sox2,Klf4,cMyc), some Oct4-GFP murine embryonic fibroblasts(MEFs) have the properties of induced pluripotent stem cells(iPSCs). These Oct4-GFP iPSCs whose endogenous Oct4 have been activated will trigger the expression of GFP(green fluorescence protein) thus to be very convenient to iPSCs verification as well as FCM(flow cytometer) analysis. </P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 2.</STRONG>     Flow chart for generation of mouse iPSCs.
Wild type C57 BL/6 female was injected with 5IU PMSG,and 5IU hCG 48h later, then mated with transgenic C57 BL/6 Oct4-GFP(OG) male.OG MEF was isolated and generated from 13.5 dpc embryo back tissue.GFP specific primer was used to perform genome PCR and essured F0 homozygote and F0 heterzygote.Retrovirus pMXs-Oct4/Sox2/Klf4/cMyc(OSKM) was packaged in HEK293T and ultalcentrifuged to enriching viral titer.OG MEF was infected with OSKM viral cocktail and replated on feeder layer 48h later.10 days after viral infection could we observe iPS clone formation.     </P></DIV><P>We first breeded transgenic Oct4-GFP C57 BL/6 homozygote F0 male with wild type C57 BL/6 F0 female, and isolated and generated Oct4-GFP primary MEF cell lines from 13.5dpc embryo. Genome typing was performed to determining F0 homozygote and F1 heterzygote. We later packaged retrovirus pMXs-Oct4/Sox2/cMyc/Klf4(Constructed by Shinya Yamanaka) in HEK293T. Oct4-GFP MEF cell line, C57-4-A P2, was infected ultracentrifuge enriched viral cocktail. We Replated reprogramming MEFs on mitomycin C treated pMEF-NL feeder layer two days later, and supplied with mouse iPSCs medium from this day on. iPSCs monoclone formed two weeks after infection(Figure 3)，and this cell lines refer as C57-4-A iPS later.</P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 3.</STRONG> Morphology during reprogramming. </P></DIV><H3>Verification of mouse iPSCs:</H3><P>As C57-4-A iPS was reprogrammed from Oct4-GFP MEFs, we can observe robust GFP expression under inverted fluorescence microscope. Immuno-Fluorescence was also performed to detect some classical pluripotency marker such as Oct4/Nanog(nuclear colocalized) and SSEA1(cytomembrane colocalized).Karyotype analysis demonstrated 53.3% C57-4-A iPS possessing correct chromosome complement(Figure 4). </P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 4.</STRONG>   Pluripotency marker identification.
iPSCs GFP expression was detected through FITC filter.Karyotype analysis revealed 2n=40 intact mouse chromosome complement.Immuno-Fluorescence stainning proved Oct4/Nanog/SSEA-1 expression in C57-4-A iPS.     </P></DIV><P>Three million C57-4-A iPS cells were injected to three 8 weeks BALBc nu/nu(Nude mice) left shoulder subcutaneously, and same amount of C57-4-A MEFs to the right as control. Four weeks later, significant tumors formed under left(but not the right) shoulder skin of each BALBc nu/nu mouse. Tumors were fixated in 10% formalin, dehydrated, paraffin embedded and HE(hematoxylin and eosin)stainned at last. Various type of tissue developing from entoderm, mesoderm and ectoderm (Figure 5) demonstrated our C57-4-A iPS cell lines not only expressed dogma iPS marker but also can form teratoma in vivo which combined together proved it possessing qualified pluripotency and stemness.</P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 5.</STRONG>    iPSCs_Teratoma HE stainning.
teratoma was formed in BALBc nu/nu(nude mice) developing from three million iPSCs. Hematoxylin and eosin(HE) stainning brought out intestines ,lung ,bladder tissue  indicated that iPSCs prossess qualified pluripotency. </P></DIV><H3>Stable mouse iPSC line assay:</H3><P>iPSCs was infected with lentivirus just three days before the wiki freezing. Because the time was limited, we have to finish the stable line test within the number of a 12-well plate of iPSCs. Among them, 7 wells were injected with tTA and Ptight-suicide gene, 4 were added Dox (5 ug/ml), one was injected with nothing, 4 are injected with TRE3G-suicide/GFP. The rate of tTA lentivirus and Ptight-suicide gene lentivirus was 1:2, which was the best rate having been proved by the transient experiments we did in Bosc before. </P><P>With routine medium refresh, 72 hours later, we ran FACS Calibur flow cytometer to identify the iPSCs(with robust GFP signal) and differentiated cells. Here are the outputs. </P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 6. </STRONG> From top to bottom, the test groups are: neg ctrl, TRE3G-GFP, TRE3G-rip1, TRE3G-RIP3, mock+tTA, mock+tTA+Dox, pTight-RIP1+tTA, pTight-RIP1+tTA+Dox, pTight-RIP3+tTA, pTight-RIP3 +tTA+Dox, pTight-VP3+tTA, and pTight-VP3 +tTA+Dox. From left to right, A group represents the density of iPSCs’ GFP(X) and cell size(Y); B group represents the density of iPSCs’ GFP(X) and whole cell numbers(Y); C group represents the density of iPSCs’ GFP(X) and cell size in another tunnel(Y).</P></DIV><P>From the figure above, what we can observe is as follows: </P><P>1. The number of cells infected with both tTA and Ptight-suicide gene (RIP1, RIP3, VIP3) decreases significantly, while that of cells only infected with TRE3G, which has been proven before to have a higher expression than Ptight, is relatively close to the negative control group. And in the survived cells infected with both tTA and Ptight-suicide gene, most cells have differentiated. This result proved that <STRONG>each of these three suicide genes not only have significant killing effects in mouse iPSCs, but also destroy the features of iPSCs, causing them easier to differentiate. </STRONG></P><P>2. The cells infected with TRE3G live quite normally. This indicates that the leaky expression of <STRONG>TRE 3G in stable cell lines is tolerable</STRONG>, which conflicts with the result of transient transfection with TRE3G in Bosc.</P><P>3. The cells infected with tTA and Dox live normally. This indicates that <STRONG>Dox has little or no side effect for iPSC features,</STRONG> which eliminates the biggest worry of us before. </P><P>4. The cells infected with tTA, Ptight-suicide gene and Dox(5 ug/ml) don’t show a rescue effect compared with the cells without Dox. This result indicates that <STRONG>something is wrong with tet-off system,</STRONG> probably in the combination of tTA and TRE, given that the number of cells only infected with tTA and Dox is much more than the cells infected with tTA, Ptight-suicide gene and Dox. Here are the statistic table showing the number and rate of survived cells and survived iPSCs.</P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 7. </STRONG>A shows the survival rate of all the cells, including iPSCs and differentiated cells, in every group with different infections; B shows the rates of iPSCs in all survival cells; C shows the number of survived cells in every group with different infections; D shows the number of iPSCs in all survived cells. </P></DIV><P>From the discussion above, we can make some conclusions here. The good news is that our suicide genes really work in mouse iPSCs, while the bad one is that our tet-off system does not work well enough in stable transfection in mouse iPSCs. But where does the problem come from? We come up with several proposals: </P><P>1. First is the <STRONG>tTA protein</STRONG>. Since the tTA protein, when Dox is added, is supposed to immediately combines with Dox, changes its comformation and thus separates from the TRE promoter and inactivates it, we propose that there is something wrong in this microcosmic process. If this is the case, maybe it’s better for us to search into the sequence and 3D structure of tTA protein, to verify whether there is some mutant sites or changes in 3D structure.</P><P>2. Second is the <STRONG>TRE element</STRONG>. Since there are two plasmids composing of tet-off system, there is possibility that TRE element has mutated in some binding sites and therefore once binding to tTA protein, it will never let it go, no matter what state the protein’s conformation is at.  </P><P>3. Last is the <STRONG>dose of Dox</STRONG>. Since the expression level in stable transfection differs from that in transient transfection, we are wondering maybe the dose of Dox(5 ug/ml), which is enough in transient transfection, is enough here. So what happens in cells may be like this: After Dox are added, some of the tTA proteins immediately combine with all of Dox and thus separate from TRE and lose the ability to bind it again. However, the rest of tTA proteins, which have no Dox to combine, continually bind to the TRE promoters and thus keep activating them. To verify this, the next experiment we should do is to raise the dose of Dox and see what will happen. However, because the time limited (damn it!), by far we haven’t time to finish this step of trouble shooting. ( Maybe you can hear good news from us on Oct. 5~)</P><P>Besides, we also plan to do the following tests:</P><P>1. <STRONG>Try to adjust the MOI of lentivirus.</STRONG> Because the MOI used in this test is recommended by our Advisor Jiang Shuai, who are very complished in mouse iPSC’s culture and infection, we have not set experiments to test the properest MOI before. Since lentivirus has a tremendous effect on infected cells, it’s necessary to regulate the MOI of lentivirus to the best level.</P><P>2. <STRONG>Try to use a new tet-off protein called tTA-advanced.</STRONG> Actually we’ve also got another advanced tet-off system in hand. If the old one does not work as expected, maybe we’d better to drop it and move to try the new one. </P><H2>Human iPSCs:</H2><P>In addition to test our device in mouse iPSCs, we even tried to test it human iPSCs. We were honored to receive the iPSCs from the laboratory in Medical School in Sun Yat-sen University. However, compared with mouse iPSCs, human iPSCs were much more difficult to culture. The steps and operations are more complicated and require exquisite skills. Struggling for more than 2 mouths, what we have achieved by far is successfully culture human iPSCs on feeder. However, when transferred from feeder on metrigel, iPSCs always went something wrong. Sometimes they died, sometimes they differentiated, sometimes they just stopped grow, which was the case worse than dying &gt;.&lt;. We tried to shoot the potential troubles for several times, but it seemed that we were not hitting the point. Finally, out of driven crazy, we decided to have fun with them. ╮(╯▽╰)╭</P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 8. </STRONG>There were drawn by the injector on human iPSCs metrigel. ：D </P></DIV><H1>Hepatocytes:</H1><H2>Isolation:</H2><P>In order to test the efficiency of miRNA-122 target in mice Hepatocytes, the first thing we need do is to isolate the primary hepatocytes from mice and culture it.  Literally there are several different methods to isolating primary hepatocytes, what we chose to do was the most complicated but most guaranteed one: in situ perfusion.</P><P>Tough and challenging as the isolation technology is, we devoured many scientific manuals and finally successfully achieved doing it.Here are the general steps: After inject proper dose of heparin into the mice femoral vein, we opened the abdomen and cannulated the liver via the portal vein and flush with HBS for 15min, then perfused with collagenase for 15min, then we excised the liver and dispersed the cells in L-15/BSA. The next step was filtering the supernatant cell suspension through gauze and washed the cells several times and finally checked cell yield and viability with trypan blue in a hemocytometer counting chamber. </P><H2> Culture:</H2><P>When the hepatocytes we isolated are observed under the microscope, we can see that most of them have the obvious characters of normal hepatocytes: several (2~3 ) nucleus in single cell. Here are some pictures.</P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 9. </STRONG>The pictures of normal primary hepatocytes. </P></DIV><P>After the isolation of hepatocytes, it turns out that the culture of them are also challenging for us. Hepatocyte has a very fast growth rate, several days later, some of the cells’ morphology become more angulargrained. The photo is as follows. After several times of trial and error, we finally succeeded in culture hepatocytes.</P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 10. </STRONG>the angulargrained morphology of Hepatocytes</P></DIV><H2>Transient and stable infection:</H2><P>By far, we have established the platform of primary mice hepatocyte isolation and culture. The next step should be moved to transient and stable infection in it to test two things: </P><P>1. Whether the suicide genes we’ve chosen have the ability to suicide hepatocytes; </P><P>2. Whether the high expression level of miRNA122 in hepatocytes has the ability to silence the expression of suicide genes. </P><P>However, when it comes to the test our system in Hepatocytes, we met a huge problem. With the knowledge that hepatocyte is a type of cell which could hardly passage and has a very low transient infection efficiency , it seems impossible or meaningless (if possible) for us to do the transient transfection and stable infection in hepatocytes. After trying several times, we failed to succeed transfect them. We have reached out to many other laboratories and professors for help, but only pitifully found out that there still remained the technological difficulties in transfecting hepatocytes. </P><P>In fact, to some degree, the test in Hepatocytes is not necessary. Theoretically the suicide genes only work when miRNA122 level is low, so even the suicide part are delivered into hepatocytes, the extremely high expression level of miRNA122 will bind to the target and silence the genes immediately. Our result for quantitative characterization of the complete-target-site series and the linear-regression model further proves our expectation.</P><H2>qPCR:</H2><P>In our iPSC-Safeguard pathway, we make use of the high miR122 expression level specific for Hepatocyte to set up a security device. To confirm our design, we performed RT-qPCR for detection of miR122 level in Hepatocyte together with the level expressed exogenously by our plasmids for positive control. Since the positive controls were done with HEK293, 0ug of expression plasmids in the cell represents the nature miRNA level in that cell line. Protocols can been seen in our Method module and U6 snRNA was used as the internal reference.</P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure11.</STRONG>MiR122 level in hepatocyte quantitated by RT-qPCR</P></DIV><P>From the result we can see that the miRNA level in hepatocyte was significantly higher than normal HEK293, and even higher than that was expressed by 1ug of plasmids. Since under such a concentration the target-site we used in our pathway already showed an outstanding knockdown performance on GOI, it’s reasonable that we expect a good protection from Suicide Gene by our MicroRNA-Target system for hepatocyte.</P><H1>Stable HepG2 cells </H1><P>As for the stable transfection in hepatoma cells, we chose Heg G2, which is the most common hepatoma cell line, to be our chassis. Hep G2 is a perpetual cell line which was derived from the liver tissue of a 15-year-old Caucasian American male with a well-differentiated hepatocellular carcinoma. These cells are epithelial in morphology, have a model chromosome number of 55, and are not tumorigenic in nude mice. There are two plasmids which are supposed to be delivered into genome. </P><P>Considering the high resistance pressure from double-stable infection which cells can hardly burden, we finally decided to singly infect each plasmid one by one. The steps of establishing the Heg G2 cell lines stably expressing tTA and TRE-suicide genes are as follows. </P><H2>First stable infection and selection:</H2><P>After 4 days’ resistance(1 ug/ml) selection, only 10% cells survived. Two days later, we have got a batch of stable Hep G2 cell lines which contain the TRE-suicide gene-miRNA122 targets in cell genome. Here are some photos during the selection. </P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 12. </STRONG>Stable HepG2 line with lentivirus of TRE-GFP-target. All CM are added puro. A: Day 0. initial density before infection; B: Day 2 after puro selection; C: Day 3 after puro selection; D: Day 7 after selection.</P></DIV><P>We figured out that the stable Hep G2 cell line, integrated with TRE-suicide gene-target, grow much slower than before. But after several passaging, their growing rates returned to normal. </P><H2>Second stable infection and selection:</H2><P>Up to Sep. 25, we have successfully got the single stable Hep G2 cell line, and the second stable cell line containing both the tTA and suicide gene, are still under selection. After selection, what we plan to do is try to remove CM with Dox and see what will happen for these cells. </P><P>Our experiment design is like this:</P><P>Discussion:</P><P>Even though still under selecting the second stable Hep G2 cell lines, we have noticed many detailed phenomena,which indicated us something during the selection:</P><P>1. In the steps of lentivirus packaging, when collected after 48 hours, the different colors of the CM of different lentivirus showed that different suicide genes have different suicidal effects. (go back to see lentivirus) </P><P>2. After infecting Hep G2, the number of cells with suicide genes are pretty close to the number of cells with GFP, which means that the leak expression of pTight is low enough to be safe. Besides, the fact that very little fluorescence has been observed in cells injected with pTight-GFP-target under fluorescence microscope further proved the conclusion.</P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 13. </STRONG>The photos of fluorescence expression.Left is the lentivirus-pTight-GFP, right is the lentivirus-pMXs GFP, which is the positive control.</P></DIV><H1>Achievements: (Up to Oct. 27)
</H1><P>
√  Via transient transfection experiments, we successfully proved that each parts of our design (suicide gene, tet-off system and miR122 target) work as expected in both hepatma cell lines and 293T cell lines.
</P><P>
√  Via stable transfection with lentivirus, we successfully established several cell lines (Hep G2, HTC and mouse iPSC) stably expressing our device. 
</P><P>
√  Successfully generated Oct4-GFP iPSCs, which harbor an IRES-EGFP fusion cassette downstream of the stop codon of the Oct4 (Pou5f1) gene.
</P><P>
√  Successfully proved that our suicide genes part worked in stable expression mouse iPSC lines. 
</P><P>
√  Successfully isolated and cultured the primary hepatocytes from mice. Because hepatocyte is very hard to both transiently and stably tranfect (at only 10% efficiency), we are still trying to improve the transfection efficiency.
</P><P>
√  Trying to inject the mouse stable iPSCs containing our device into nude mice and wait for the in vivo result. 
</P></DIV></DIV></DIV><DIV id="address"><P>Sun Yat-Sen University, Guangzhou, China</P><P>Address: 135# Xingang Rd.(W.), Haizhu Guangzhou, P.R.China</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:SYSU-China/Project/result/stable_assay">http://2013.igem.org/Team:SYSU-China/Project/result/stable_assay</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SYSU-China/Project/result/stable_assay" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SYSU-China/Project/result/stable_assay" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SYSU-China/Project/result/stable_assay&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SYSU-China/Project/result/stable_assay&amp;oldid=359879" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>