<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SYSU-China_Project_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SYSU-China/Project/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SYSU-China/Project/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-China/Project/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SYSU-China/Project/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SYSU-China/Project/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SYSU-China/Project/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="animat"><SPAN><P>ipsc</P></SPAN></DIV><DIV class="content_body" align="center"><DIV id="cont_column"><DIV class="chapter"><SPAN>Project/Modeling</SPAN><H1> 1.Overview </H1><P>
Modeling is a powerful tool in synthetic biology and engineering. In the iPSCs Safeguard 
project, modeling has provided us with an important engineering approach to characterize our 
pathways and predict their performance, thus helped us with modifying and testing our 
designing.
</P><P>
Basically, the models built by us can be divided into two levels.On gene level, we hope to gain insight of the gene expression dynamics of our 

whole circuit. And also we tried to better characterize our parts, analyze our experimental data, for all of the sensor, switch, and the killer. 

Several tools including ODEs and interpolation are employed. 
</P><P>
On cell level, we proposed 
the multi-compartment model to trace the change of the iPS cells in different time nodes, 
thus we are able to describe the growth and decay of iPSCs. The number of cells at the 
initial stage, growth rate and death rate of cells caused by suicide gene in our Safe-guard 
pathway were all taken into account.
</P><H1>2.Gene expression dynamics of iPSC Safeguard</H1><H2>2.1 analysis of the problem</H2><P>
Firstly we built a model to help ourselves better understand the gene expression dynamics of the whole circuit. In our device, tTA protein is 

firstly expressed by a EF1-α promoter and will then bind to TRE element, activating transcription of gene of interest, in our case suicide gene or 

GFP. This process is affected by the Dox suppression effect. And the TRE element alone without tTA protein has a leaky expression. After production 

and functioning, the protein of interest will undergo degradation. And to simplify the model, we consider the miRNA degradation effect as part of 

this process.
</P><P>
Then we wrote down the four chemical reactions in represent of each process. 
</P><P>
The symbol declaration is:
</P><P>
X1:tTA protein
</P><P>
D: the TRE promoter
</P><P>
X1D:the tTA-TRE complex
</P><P>
X2: the protein of interest
</P><P>
And we assigned each process with a reaction rate constant.
</P><P>
The first process is about binding and dissociation of molecules and is a fast reaction. Time unit for that is second. Since EF1-αis a 

constitutive promoter, and we define the initial state as when we remove the Dox from medium, we can regard the concentration of X1, or tTA 

protein, as a given value, determined by intensity of EF1-α. Reaction rate constant k1,k-1 are affected by Dox. In the tolerable range descried in 

part3, the more Dox we apply to D, the smaller value of k1 for the positive direction and the greater value of k-1 for the negative direction.
</P><P>
In the second and third process, k2 and k3 are affected by what kind of TRE promoter we use. There are several available TRE promoters, with which 

we can assemble different version of Tet-off system. These promoters differ in leaky expression and switching performance.
</P><P>
MiRNA-mediated mRNA degradation lead to a decrease in X2 production, which can be thought as identical with natural degradation of X2, the protein 

of interest. So K4 will be affected by the knockdown efficiency of miRNA binding site. We have constructed several binding sites to fine tune this 

process.
</P><H2>2.2 solutions and implication</H2><P>
According the analysis, we then wrote down the ODEs:
</P><P>
Here [D0] is the initial concentration of promoter Pcmv，c1 is the given concentration of X1 at the beginning. Next we will do some algebra to 

simplify the equations and solve the differential equations. 
</P><P>
We use Mathematica’s Dynamic Interactive Function Manipulation to control the variation of the parameters α,β, and thus get curves with 

different dynamics and steady state.
</P><DIV class="figure"><P class="des" style="margin-top:0px;width:700px"><STRONG>Figure 1. Different values of α,βgenerate curves with different dynamics and steady 

state. </STRONG></P></DIV><P>
As analyzed in 2.1, choosing different candidates with variant characteristic will affect the parameters in the chemical reactions, and change 

values of α and β. We’ve got 4 tet-off systems, a series of miR122 binding sites and experimental characterize them. And also we have two EF1-α 

promoters with different enhancers(not mentioned in Result section). This implicate that we can assemble different circuits and fine-tune the 

dynamics and steady state. We believe that this is important to practical engineering, just as basic logic is to conceptual designing.
</P><H1>3. Dosage effect of DOX in turning off the Tet-off system</H1><P>
DOX ,as is discussed above, hinders the binding of tTA to pTRE in Tet-Off system and 
knockdown expression of suicide gene. In our experiment, we employ fluorescence technique to 
manifest the amount of protein product by detecting the strength of the fluorescence.
</P>
Table 1. Stimulating data of GFP-Dox

Figure 4. GFP-Dox line chart
<P>
Our task is to find the proper curve to fit the sample data. First of all we plot the 
scatter diagram, and according to its tendency, we use type curve to fit the relation of 
GFP-DOX. We use MATLAB to aid our fitting, i.e. to determine the parameter a, b and k.
</P><STRONG><P>
%expun.m
</P><P>
function y=expun(s,t) %coefficient and variable
</P><P>
y=s(1)+s(2)*exp(-s(3)*t)
</P><P>
%curvefit.m
</P><P>
treal=[0 0.125 0.25 0.5 1 2]; %experimental data
</P><P>
yreal=[25 13 10 8 6 5.7];
</P><P>
s0=[0.2 0.05 0.05];  %iteration initial value
</P><P>
sfit=lsqcurvefit('expun',s0,treal,yreal); %least square curve fit
</P><P>
f=expun(sfit,treal);
</P><P>
disp(sfit);
</P></STRONG><P>
The result :
</P>
Figure 5. Running result
<P>
So a=6.4147,b=18.3999,k=7.3173.
</P><P>
Then we program the diagram file GFP-DOX.m
</P><STRONG><P>
%GFP-DOX curve
</P><P>
treal=[0 0.125 0.25 0.5 1 2]; %experimental data
</P><P>
yreal=[25 13 10 8 6 5.7];
</P><P>
t=0:0.1:2.5;
</P><P>
a=6.4147;b=18.3999;k=7.3173;
</P><P>
y=a+b*exp(-k*t);
</P><P>
plot(treal,yreal,'rx',t,y,'g');</P><P>
xlabel('Dosage of DOX');
</P><P>
ylabel('GFP');
</P></STRONG>
Figure 6. GFP-Dox curve
<P>
As is shown in the figure above, we can conclude that the amount of GFP tend to be steadily 
over 1.5 ug, the higher concentration of DOX we set, the lower GFP we expect. However, under 
the real experimental conditions, over 2.2 ug DOX will lead to the undesired necrosis of the 
cells. This is a trial-experiment which proved that such a balance point for good turning-
off effect and cell tolerance does exist in a certain interval concentration. More accurate 
experiment should be conducted on stable-transfected iPSCs to find the best cultivating 
condition.
</P><H1>4. Knockdown efficiency interpolation</H1><P>
According to the experimental data, here we use interpolation technique to find the 
relationship between miRNA-122 concentration, the number of miR122 target sites and cell 
knockdown efficiency, which leads to a function with two variables. The knockdown efficiency 
is represented by GFP expression level which is actually the ratio of the amount of GFP and 
that of the parameter GAPDH. The knockdown efficiency then is
</P>
Figure 7. Two target sites, gradient miRNA concentration

Table 2. Experimental data of 2 target sites, gradient miRNA concentration

Table 3. Experimental data of 0.75ug miRNA plasmid with gradient target sites
<P>
We use the data above to do the interpolation. We use the griddata function to implement the 
interpolation.
</P><P>
MATLAB codes:
</P><P>
clear
</P><P>
miRNA=[0 0.025 0.05 0.1 0.25 0.75 0.75 0.75];
</P><P>
site=[2 2 2 2 2 1 2 4];
</P><P>
KD=[0 29 43 55 64 55 39 32];
 </P><P>
cx=0:0.01:0.75;
</P><P>
cy=0:0.05:4;
</P><P>
cz=griddata(miRNA,site,KD,cx,cy','cubic');
</P><P>
meshz(cx,cy,cz),rotate3d
</P><P>
%shading flat
</P><P>
xlabel('miRNA(plasmid ug)'),ylabel('Target Site'),zlabel('knockdown efficiency(%)');
</P>
Figure 8. Knockdown efficiency-mRNA-target site surface chart
<H1> 5. The excursion of little mathematician--Data analysis of the FACS data
</H1><P>
When our project was proceeding, we found out an interesting problem, that is, how to calculate the killing efficiency of each suicide gene?  And 

this quantity is also an important part of our modeling. Trying to solve this problem,one of our mathematician,  young Yang ZiYi, excursed a little 

bit from modeling to data analysis. And he analyzed our data of FACS of our transient transfection experiment of Hep G2.
</P><P>
One important point in mathematical analysis about the complicated biological system is, not to draw arbitrary assumption, arbitrary assumption 

just lead to disaster. Another important point is not to draw complicated assumption, which is hard to calculate. Base on these rules, Young  ZiYi 

draw some simple and reasonable assumptions:
</P><P>
1.	The initial condition of each parallel well is the same, that means, every well before transient transfection should have the same cell 

density, and the cells' state should be approximately the same.
</P><P>
2.	In the GFP control group, the cells should be regarded as the same, whether they are transfected with GFP or not, since GFP do not harm the 

cells. But lipo-2000 will harm the cells, and may have some long term effect. So the GFP control group would be relatively weaker compare to 

negative control, which without any transfection manipulation;
</P><P>
3.	The cells transfected with GFP should have an innate death rate r1 after 3 days cultivation. Besides, the ratio of “the initial number of 

cells” to “the number of cells harvested via FACS after 3 days” Should be the same, since if  you sample any part of the wells you will observe 

the same distribution of cells, this means the cell experiment is scalable. Hence, we can define a special “state” to represent GFP cells in 

certain well, (r1,n1), n1 is the number of the cells we harvest in 30s, depends on r1 and the initial cell number.
</P><P>
4.	In the well we transfect suicide genes, the cells which are transfected with suicide gene will be different from the cells which are not, 

due to the killing effect of suicide gene, and the cells without transfection will be at the same condition as GFP control and have the same r1. 

Cells transfected with suicide gene will have a different r2, we denote this part of cells by (r2,n2). If the transfection efficiency is a, then 

for any initial cell number N0, there will be N0×a cells transfected with suicide gene, the remaining N0×（1-a）are not. Here &quot;a&quot; in an unknown 

factor.
</P><P>
Then Young ZiYi built a model:
</P><P>
In the wells that had been tranfected with suicide genes, the final observed “state” (X, C) of the cells is an combination of two kinds of cells, 

one kind with state (r1,n1), the other kind with state  (r2,n2). The effect of these two kinds will be summed up together. Then we can write down a 

equation:
</P><P>
Because we can not easily get the transfection efficiency, we select a reasonable range from experience:[30%,60%], and solve the equation with the 

data from FACS, and get the following results:
</P><P>
It turns out that, the death rate of cells transfected with suicide gene will be greater than 60%, significantly higher than normal cells and GFP 

control cells(29.07%, according to our data).
</P><P>
Then, We pick up one suicide gene, VP3, and take an “a” value 35% which we believe is fairly closed to the real tranfection efficiency, solve the 

equation and get the result r2 and n2. And then, we substitute the calculated r2 and n2 to the left side of the equation, and raise the value of “

a” to predict what the death rate and the “number per 30 seconds” would be when the transfection efficiency is raised. The result is shown in a 

chart below
</P><P>
That makes our model become a real theory, because it predicts something that can be disproved. In the following day, we will try to raise our 

transfection efficiency, and try to comparing the result with the prediction.
</P><P>
And, let’s wait for the result.
</P><H1>  6. Multi-compartment model
</H1><H2>  6.1 Analysis of the problem
</H2><P>
We firstly focus on factors that regulate the performance of the whole pathway. Protein tTA 
expressed by a EF1α promoter binds to the promoter pTRE to drive the transcription of 
target gene( in this case, eGFP or suicide gene) while Dox acts as a co-repressor 
prohibiting the transcription. MiR122 isa downstream part in the pathway after transcription 
of target mRNA, and mediated degradation of the mRNA, thus rescue the cell or knockdown its 
GFP expression. However, the miR122 level in iPSC was low and insufficient to exert obvious 
effect on the expression.
</P><P>
Apart from Dox concentration,we also monitored other parameters, including cell number after 
the stable infection and number of cell that survived the Suicide Gene. Moreover, we also 
kept track of fluoresence intensity of the control group who has been transfected with GFP, 
which can be employed to indicate the GOI expression level driven by Tet-Off system.
</P><P>
In pratical, we planned to monitor the cell group scale every 5 hours and technically, we 
counted the total clone area instead of cell number.
</P><H2> 6.2 Symbols declaration and assumption
</H2><P>
X1: initial number of iPS cells with Suicide Gene
</P><P>
X2: number of the iPS cells whose TRE have been combined with tTA
</P><P>
X3: number of iPS cells which have died from expressing Suicide Gene
</P><P>
k1: converting rate of the number of cells from phase X1 to phase X2
</P><P>
k2: converting rate of the number of cells from phase X2 to phase X3
</P><P>
The unit of ki(i=1,2) is hour-1.We measured it by dividing the absolute value of the cell 
number difference between former phase and latter phase, with the time period length.
</P><P>
Two cases are taken into account. In case (a), self-renewal and replication of cels are 
ingored while in case (b), we take that into consideration. To further simplify the model, 
we also assumed that every single cell in phase X1 turns into n1 state before phase X2, and 
every single cell in phase X2 turns into n2 state before phase X3. We simulated the kinetic 
process of gene expression and assumed an even distribution of cell content in the 
medium,after which the phase can be regarded as a compartment.
</P><H2> 6.3 Solution
</H2><P>
For each compartment, we construct unsteady state equilibrium equation, hence we obtain the 
ordinary equations
</P><P>
For case (b), we just need to modify the scalar coefficients of the equations above, and we 

obtain
</P>
Figure 1. dynamic process
<P>
We are going to solve X1(t), X2(t),X3(t), then we will plot the time course curve.
</P><P>
The initial conditions of the differential equations are as follows:
</P><P>
X1(0)= 5000 cells, X2(0)=0 cell, X3(0)=0 cell
</P><P>
k1=1day<SUP>-1</SUP>,k2=1 day<SUP>-1</SUP></P><P>
As for case b, the cell replicates every 26 hours, to simplify we consider one cell turns into 2 cells before next phase. Therefore, n1=n2=2. 
</P><P>
Source code
</P><P><STRONG>
%igem_test1.m-Solution of the IPS cell differentiation model
</STRONG></P><P>
%using MATLAB function ode45.m to integrate the differential equations
</P><P>
%that are contained in the file cell_diff_eq.m
</P><P>
clc; clear all;
</P><P>
%set the initial conditions, constants and time span
</P><P>
xzero=[5000,0,0];tmax=4;
</P><P>
k1=1; k2=1;
</P><P>
tspan=0:0.1: tmax;
</P><P>
N=3;
</P><P>
%Integrate the equations
</P><P>
[t X]=ode45(@cell_diff_eq,tspan,xzero,[ ],k1,k2);
</P><P>
last=X(length(X),N);
 </P><P>
%Plot time curve
</P><P>
plot(t,X(:,1),'-',t, X(:,2),'-',t, X(:,3),'-.');
</P><P>
legend('X1','X2','X3');
</P><P>
xlabel('time,days');
</P><P>
ylabel('number of cells');
</P><P>
function dx= cell_diff_eq(t,x,k1,k2)
</P><P>
%cell expression kinetic procedure
</P><P>
dx=[-k1*x(1);
    k1*x(1)-k2*x(2);

    k2*x(2); ];
</P> Figure 2. The result of case (a)

 Figure 3. The result of case (b)



7.reference
<P>
[1] Systems biology in practice concepts, implementation and application / (德) E. Klipp等著 

; 主译:贺福初, 杨

芃原, 朱云平 ,上海 : 复旦大学出版社, 2007
</P><P>
[2]Numerical methods in biomedical engineering / (美) Stanley M. Dunn, Alkis Constantinides, 

Prabhas V. Moghe著 

; 封洲燕译,北京 : 机械工业出版社, 2009
</P><P>
[3]miRNA regulatory circuits in ES cells differentiation: chemical kinetics modeling 

approach , Luo Z, Xu X, Gu 

P, Lonard D, Gunaratne PH, et al. (2011)
</P><P>
[4]kinetic signatures of microRNA modes of action, N Morozova, A Zinovyev, N Nonne, LL 

Pritchard - RNA, 2012
</P><P>
[5]On the inhibition of the mitochondrial inner membrane anion uniporter by cationic amphiphiles and other drugs. Beavis AD,J Biol Chem. 1989 Jan 

25;264(3):1508-15.
</P></DIV></DIV><DIV id="address"><P>Sun Yat-Sen University, Guangzhou, China</P><P>Address: 135# Xingang 

Rd.(W.), Haizhu 

Guangzhou, P.R.China</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:SYSU-China/Project/Modeling">http://2013.igem.org/Team:SYSU-China/Project/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SYSU-China/Project/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SYSU-China/Project/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SYSU-China/Project/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SYSU-China/Project/Modeling&amp;oldid=349454" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>