<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Lab_Reporter skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Lab/Reporter</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="js-message abstract">
    If you can see this message, you do not use Javascript. This Website is best to use with Javascript enabled. Without Javascript enabled, many features including the mobile version are not usable.
  </DIV><DIV class="vviki" id="vviki"><DIV id="head"><DIV class="navbar {{{extra-class}}}"><UL id="slick-menu"><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Hardware">Robotics</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Team">Team</A></LI></UL></DIV><DIV id="mainHeader"><H1>REPORTING LOW nnAA LEVELS</H1></DIV></DIV><DIV class="page"><DIV class="abstract"><P>ABSTRACT</P><P>Glow before you go - What does this actually mean? The aim of our project is to make biology safer by introducing a suicide system to <I>E. coli</I>. Before the suicide is triggered, a <B>reporter protein</B> is expressed to indicate the release of <I>E. coli</I> or to show a deficiency of the non-natural amino acid in the surrounding medium which is necessary for the bacteria to survive. As a reporter protein, we chose <B>mVenus</B> which is a mutant of eYFP. mVenus is located downstream of a promoter which is repressed by a dimeric protein, the <B>Zif23-GCN4 repressor</B>. This repressor carries an <I>amber</I> mutation at position 4 (F4OMT). As a result, the non-natural amino acid <B><I>O</I>-methyl-<SPAN style="font-variant:small-caps">l</SPAN>-tyrosine (OMT)</B> is integrated into the protein sequence as long as there is enough OMT in the medium. With decreasing OMT concentration, the translation of the repressor stops due to the early <I>amber</I> stop codon and the repressor cannot bind to the promoter. This leads to expression of the reporter protein mVenus which can be detected by fluorescence measurements.</P></DIV><DIV class="content" id="lab2c"><H5>Reporter System</H5><P>For the detection of a low non-natural amino acid concentration, in this case <I>O</I>-methyl-<SPAN style="font-variant:small-caps">l</SPAN>-tyrosine, we designed a reporter system that includes the reporter protein <B>mVenus</B>. In order to make sure that the expression of mVenus does only start at a low OMT concentration, we use a dimeric repressor in which an <I>amber</I> mutation was introduced to the encoding DNA sequence. This  <A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair"><I>amber</I> mutation</A> leads to OMT being integrated in the dimeric repressor protein. However, the repression of the mVenus promoter can only be executed if there is a sufficient concentration of OMT in the medium. If the OMT concentration drops below a threshold, the expression of mVenus is induced. As a result, we can detect a yellow fluorescence signal. We utilize a dimeric repressor because this kind of repressor binds strongly to the respective promotor. Moreover, this dimeric repressor creates a sigmoidal repression curve (x‑axis = concentration of OMT; y‑axis = repressor molecule concentration). If the concentration of OMT drops, we quickly obtain a signal.</P><P>To make sure that the repression does not take place when the concentration drops below the treshold, an LVA degradation tag is expressed with the dimeric repressor. In addition, to ensure that no permanent fluorescent signal is caused by mVenus, it is fused to a LVA degradation tag as well. In consequence, both proteins shall be short-living after their translation [1]. To fit this system to the expression of colicin E2, we can use different Anderson promoters (<A href="http://parts.igem.org/Part:BBa_J23100">BBa_J23100</A>, <A href="http://parts.igem.org/Part:BBa_J23104">BBa_J23104</A>,  <A href="http://parts.igem.org/Part:BBa_J23107">BBa_J23107</A>, <A href="http://parts.igem.org/Part:BBa_J23113">BBa_J23113</A>, and <A href="http://parts.igem.org/Part:BBa_J23114">BBa_J23114</A>). We aimed to find an expression rate ensuring that mVenus is translated before the DNase degrades mVenus encoding DNA and therefore inhibits the protein biosynthesis of mVenus.</P><H5>mVenus</H5><P>The fluorescent reporter protein mVenus is a mutant of the green fluorescent protein GFP which is often used for fluorescence assays. Due to mutagenesis (F46L/F64L/M153T/V163A/S175G), the maturation time is decreased compared to GFP. In general, the maturation process can be divided in the folding step and formation of the chromophore. During the maturation process, the chromophore formation is the rate-limiting step. After the folding, a torsional rearrangement effects the formation of the chromophore, triggered by the close proximity of involved residues. After cyclization of two amino acids has taken place, oxidation is the final step. Molecular oxygen is necessary for the reaction that generates the delocalized π−electron system, resulting in the fluorophore being maturated and fluorescent. It is protected by the surrounding β−barrel from interfering influences. All the processes are influenced by the general cell- and cell-cycle processes and can be delayed or accelerated. In vitro, the maturation time of mVenus is in average 40 minutes. 
Another effect of the mutation F46L is the lowered sensitivity to the pH and the chloride ion concentration, one of the drawbacks of wild‑type GFP [2]. 
</P><P>mVenus is expressed with a LVA degradation tag to decrease the protein half‑life. Moreover, the reporter is not regulated by any proteins, cofactors or substrates. The lack of disulfide bonds supports the choice of mVenus in our model microorganism <I>E. coli</I>. Its absorption maximum is at 512 nm and its emission maximum at 528 nm. The atomic mass is approximately 27 kDa. [3, 4]   </P><CENTER><DIV class="bild" style="width:35%;"><B>Figure 1:</B> Cyrstal structure of the mVenus reporter protein without the LVA degradation tag. Yellow: β−barrel fold enclosing the fluorophore. Created with the <A href="https://2016.igem.org/Team:TU_Darmstadt/Model">PyMOL Molecular Graphic System</A>. <A href="http://www.rcsb.org/pdb/explore.do?structureId=1MYW">PDB entry</A>, <A href="http://www.jbc.org/content/277/52/50573">Rekas <I>et al.</I> 2002</A> .</DIV></CENTER><H5>Rational Design of an <I>amber</I> mutant of the Zif23-GCN4 repressor</H5><P>The regulation of the reporter protein mVenus is carried out by a dimeric zinc finger protein binding cooperatively to a specific promoter region by interfering with its major groove of the DNA. The dimeric Cys2His2 zinc finger protein is the DNA binding domain and attached to a leucine zipper dimerization domain. Hence, the transcription of the targeted gene is repressed by the specific DNA binding. The monomers bind the DNA specifically and dimerization happens upon binding.[5]</P><P>In order to control expression of the repressor on a translational level, an <I>amber</I> stop codon is introduced into the sequence of the repressor. First, the mutation site had to be determined. A position was chosen in which the non-natural amino acid should not interfere with the protein structure. A localization close to the N-terminus was selected to stop protein expression early, if the non-natural amino acid concentration drops below a treshold. Phenylalanine was replaced by <I>O</I>-methyl-<SPAN style="font-variant:small-caps">l</SPAN>-tyrosine (F4OMT) in order to retain stacking interactions. All nearby side chains as well as the helix (starting from R15) were considered and destabilizing mutations were avoided. Additionally, it is important to choose a residue that is not involved in DNA binding. Otherwise, the repressor may lose its function. The residue of the <I>amber</I> mutation is highlighted in figure 2.</P><CENTER><DIV class="bild" style="width:60%;"><B>Figure 2:</B> Crystall structure of Zif23-GCN4 repressor protein (blue) bound to DNA (black) with the <I>amber</I> mutation site highlighted (yellow). The residue is located close to the N-terminus of the repressor protein in order to interrupt protein expression early in case of low non-natural amino acid concentration. Created with <A href="https://2016.igem.org/Team:TU_Darmstadt/Model">PyMOL Molecular Graphic System</A>, <A href="http://www.rcsb.org/pdb/explore.do?structureId=1llm">PDB entry</A>, <A href="http://pubs.acs.org/doi/abs/10.1021/bi034830b">Wolfe <I>et al.</I> 2003</A></DIV></CENTER></DIV><DIV class="references"><H6>References</H6><UL><LI>[1] Purcell Oliver,
Grierson Claire S,
Bernardo Mario,
Savery Nigel J, Temperature dependence of ssrA-tag mediated protein degradation, Journal of Biological Engineering, vol. 6, pp.13-15, 2012
</LI><LI>[2] Agata Rekas, Jean-René Alattia, Takeharu Nagai, Atsushi Miyawaki and Mitsuhiko Ikura, Crystal Structure of Venus, a Yellow Fluorescent Protein with Improved Maturation and Reduced Environmental Sensitivity, 
J. Biol. Chem., vol 277, pp. 50573-50578, 2002
</LI><LI>[3] Nagai Takeharu, Ibata Keiji,Park Eun Sun, Kubota Mie, Mikoshiba Katsuhiko, A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications, Nature Biotechnology, vol. 20, pp. 1585-1588, 2001</LI><LI>[4] Sarkar, Pabak
Koushik, Srinagesh V
Vogel, Steven S
Gryczynski, Ignacy
Gryczynski, Zygmunt,
Photophysical Properties of Cerulean and Venus Fluorescent Proteins, Journal of Biomedical Optics, vol 14, pp. 1-25, 2009</LI><LI>[5] Wolfe, Scot A
Grant, Robert A
Pabo, Carl O, Structure of a Designed Dimeric Zinc Finger Protein Bound to DNA, Biochemistry, vol. 42, pp. 13401-13409, 2003</LI></UL></DIV></DIV><DIV class="rechts"><DIV class="scrollbox"><BUTTON class="see_other_full"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Back to the Lab</A></BUTTON><DIV class="highlights"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair">Incorporation of a nnAA</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/Reporter">Reporter</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/KILLswitch">KILL(switch)</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/MetabolicBurden">Metabolic Burden</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/ChemicalSynthesis">Chemical Synthesis</A></DIV><BUTTON class="back_top_full"><A href="#mainHeader">Back to the Top</A></BUTTON></DIV></DIV><DIV class="footer" style="text-decoration: none;"><CENTER><DIV id="igemTUDA"><A href="https://2016.igem.org/Team:TU_Darmstadt" style="color:white">iGEM Technische Universität Darmstadt</A></DIV></CENTER><CENTER><DIV id="footer_linear">
We thank our <A href="https://2016.igem.org/Team:TU_Darmstadt/Team/Acknowledgements" style="color:#019ac8">sponsors</A>, especially:

Email: igem@bio.tu-darmstadt.de | 
Design: Viktoria Schuster</DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>