<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="js-message abstract">
    If you can see this message, you do not use Javascript. This Website is best to use with Javascript enabled. Without Javascript enabled, many features including the mobile version are not usable.
  </DIV><DIV class="vviki" id="vviki"><DIV id="head"><DIV class="navbar {{{extra-class}}}"><UL id="slick-menu"><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Hardware">Robotics</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Team">Team</A></LI></UL></DIV><DIV id="mainHeader"><H1>METABOLIC BURDEN</H1></DIV></DIV><DIV class="page"><DIV class="abstract"><P><B>ABSTRACT</B></P><P>Artificial plasmids are a significant burden to the host. The design of our pathways, for example the combination of a promoter and RBS, results in different amounts of product. The measurement of the metabolic burden is the key for a quantitative optimization 
in metabolic engineering. We want to establish a new approach to iGEM by providing a measurement strain to the community. As described by F. Ceroni et al., we genomically integrated one copy of GFP into <I>E. coli</I>, which offers us a highly accurate and instantaneous measurement of the impact of our plasmids on the host. This is of economical interest because it enables academic and industrial researchers to test a lot of different pathways at once in a short time just by using a microplate reader. For the integration we used the λ‑Integrase site‑specific recombination pathway, described by A. Landy in 2015 [9]. Therefore, we designed two plasmids (BBa_K1976000 and BBa_K1976001) and measured them using single cell measurement and via microplate reader.

</P></DIV><DIV class="content" id="lab4c"><H5>Metabolic burden</H5><P>In synthetic biology, the term &quot;metabolic burden&quot; describes the influence of heterologously expressed genes on the distribution availability of resources in the host‑cell. Thereby the extent of metabolic burden is not only dependent on the consumption of energy and reaction equivalents, but is also influenced by a number of different factors. Next to general factors, like for example size and copy number of the used plasmid [1], especially the specific properties of the heterologous proteins play an important role. Proteins that interfere with the host's metabolism, influence the proton gradient or are toxic in a different way can already at low levels of expression pose a strong burden for the cell [1, 2]. I. Shachrai et al. could furthermore show that ribosome availability, as a limiting factor, represents a big portion of metabolic burden [3].
If the metabolic burden on the host cell is too high, the physiology and biochemistry of the cell is drastically altered. In this case, for example the viability or proliferation of the cell could be disturbed. In addition there is a higher error rate in the translation which heightens the immunogenicity of the incorporated proteins and can lead to a reduced protein activity and stability [1]. Exactly for this reason the metabolic burden poses a big problem, especially in the industrial sector [4]. That is because it has a big impact on expression efficiency and thus affects the amount of obtained product. 
Examples for often used methods to reduce the metabolic burden include the separation of the biosynthetic pathway into multiple organisms through co‑cultivating them or the identification and deletion of expendable genes as a part of strain optimization [6]. Another approach is the regulation of protein expression with <I>Dynamic Sensor and Regulatory Systems</I> (DSRSs). Basis of such systems is the usage of transcription factors that can detect certain key‑metabolites and regulate the transcription simultaneously [7]. 
One possibility to quantify the metabolic burden <I>in vivo</I> was described by F. Ceroni et al. In this method, a GFP reporter gene is integrated in the genome of <I> E. coli </I> using the <I>λ</I>‑integrase [8]. Through measuring the fluorescence it could be shown that the constitutive expression of GFP after transformation with expression‑plasmids drastically lowers in comparison to not transformed cells. </P><H6>Measurement of the metabolic burden via microplate reader</H6><P> The total amount of formed GFP is, next to the stress forced onto the cell, also significantly influenced by the cell amount. Because of that reason the measurement of GFP with a microplate reader is done under continuous observation of the cell density. This kind of measurement seems feasible and with a lot of samples, like in <I>96‑well‑plates</I>, easily doable. It is possible, with the measurement of different combinations of plasmid and cell‑type, to determine the artificially caused stress on the cells proportional to the decrease in GFP expression.  Due to this reason this kind of measurement is a more economical approach than the  single cell measurement described in the next paragraph.</P><H6>Single-cell-measurement</H6><P>While measuring every single cell individually, the cell density can be neglected, which concludes in a smaller error in the analysis of the fluorescence. This method shows the direct influence of the metabolic burden on the GFP production and is therefore a more exact method than the previously described method using a microplate reader.</P><H5>Genomic integration</H5><P>The <I>λ</I>‑integrase, originally derived from the <I>λ</I>‑Phage, catalyzes in combination with several assisting proteins the excessive and integrative recombination of the phage's genome with the chromosomal genome of a host. For this, two attachment sites are needed: one located on the bacterial genome (<I>attB</I>) and the other located on the <I>λ</I>‑genome (<I>attP</I>), which also contains several binding sites for regulatory proteins. The attachment sites contain homologous recognition sequences, called BOB' Region (<I>attB</I>) and COC' Region (<I>attP</I>). These can be connected by the <I>λ</I>‑integrase and the bacterial <I>integration host factor</I> (IHF) via <I>Holliday junction</I> forming an intasome, a DNA‑protein‑complex, producing hybrid attachment sites <I>attL</I> and <I>attR</I>. </P><H6>Integration plasmid and helper plasmid</H6><P>For the integration of a gene of interest (GOI) into the chromosomal genome of <I>E. coli</I> there are two plasmids needed. 
The integration plasmid contains the constitutively expressed GOI GFP, which, as previously mentioned, is also the reporter that is necessary for the measurement of the metabolic burden and should be integrated into the <I>E. coli</I> genome. To measure only the temporary fluorescence a LVA degradation tag is added to the GFP. The plasmid also contains the <I>attP </I>site that enables the integration. Additionally, two bidirectional terminators are located on each side of the <I>attP</I> to protect the GFP operon from the transcription of the other neighbouring genes. 
To create the integration plasmid E0240 (RBS(BB0032+GFP)) was put on J61002 to locate the GFP behind the promoter J23101. The construct J23101+E0240 was then transformed on the high copy vector pSB1C3 to increase the yield of the prep after a Quick Change PCR, which was necessary for the optimization of  BBa_I11023, mutating attp2 to <I>λ</I>‑attP. The final construct was then transformed on the backbone pSB4A5, which possesses a <I>low copy ori</I> and eases the later performed plasmid curing.
The second plasmid is a helper plasmid, that is necessary for transposing the GFP into the chromosomal genome as it contains the synthesized protein <I>λ</I>‑integrase with a ribosomal binding site (RBS). For the registry the construct was transformed on a pSB1C3, then we again chose a <I>low copy vector</I> pSB4K5 to ease the later transformed plasmid curing via sustained lack of selection pressure.
To verify whether the recombination was successful one can perform a PCR with primers binding to the <I>attB</I> site of the <I>E. coli</I> and the VR Primer, which binds on every BioBrick compliant plasmid. As the one primer binds on the genome and the other on a plasmid, there can only be a PCR amplicon if the integration has succeeded. </P><H5>Integration strains</H5><P>A suitable genomic integration strain needs to carry the <I>attB</I> sequence needed for <I>λ</I>‑integrase mediated recombination, which can be troublesome because many commonly used <I>E. Coli</I> strains already have the <I>λ</I>‑phage integrated into their genome. Also, the <I>attB site</I> needed for the integration is blocked in <I>λ</I> (DE3) phages.
For our integration strain we chose the <I>E. Coli</I> JM109 strain because it matched all our demands and  was also freely and easily available to us.</P></DIV><DIV class="references"><H6>References</H6><UL><LI>[1] B. Glick, Metabolic load and heterologous gene expression, Biotechnology Advances,
vol. 13, pp. 247261, 1995.</LI><LI>[2] M. Eames and T. Kortemme, Cost-benet tradeos in engineered lac operons, Science,
vol. 336, pp. 911915, 2012.</LI><LI>[3] I. Shachrai, A. Zaslaver, U. Alon, and E. Dekel, Cost of unneeded proteins in E. coli
is reduced after several generations in exponential growth, Molecular Cell, vol. 38,
pp. 758767, 2010.</LI><LI>[4] G. Wu, Q. Yan, J. Jones, Y. Trang, S. Fong, and M. Koas, Metabolic burden: Cornerstones
in synthetic biology and metabolic engineering applications, Trends in Bio-
technology, 2016.</LI><LI>[5] H. Zhang, B. Peireira, Z. Li, and G. Stephanopoulos, Cost of unneeded proteins in E.
coli is reduced after several generations in exponential growth, Proc Natl Acad Sci, vol. 112, pp. 82668271, 2015. </LI><LI>[6] H. Westers, R. Dorenbos, J. M. van Dijl, J. Kabel, T. Flanagan, K. M. Devine, F. Jude,
S. J. Seror, A. C. Beekman, E. Darmon, C. Eschevins, A. de Jong, S. Bron, O. P.
Kuipers, A. M. Albertini, H. Antelmann, M. Hecker, N. Zamboni, U. Sauer, C. Bruand,
D. S. Ehrlich, J. C. Alonso, M. Salas, and W. J. Quax, Genome engineering reveals
large dispensable regions in Bacillus subtilis, Proc Nat Acad Sci, vol. 20, pp. 20762090,
2003. </LI><LI>[7] F. Zhang, J. Carothers, and J. Keasling, Design of a dynamic sensor-regulatory system
for production of chemicals and fuels derived from fatty acids, Nature Biotechnology,
vol. 30, pp. 354359, 2012. </LI><LI>[8] F. Ceroni, R. Algar, G.-B. Stan, and T. Ellis, Quantifying cellular capacity identies
gene expression designs with reduced burden, Nature Methods, vol. 12, pp. 415418,
2015.</LI><LI> [9] A. Landy, Dynamic, structural, and regulatory aspects of lambda site-specific recombination, Annual Review of Biochemisty, vol. 58, pp. 913-949, 1989.</LI></UL></DIV></DIV><DIV class="rechts"><DIV class="scrollbox"><BUTTON class="see_other_full"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Back to the Lab</A></BUTTON><DIV class="highlights"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair">Incorporation of OMT</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/Reporter">Reporter</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/KILLswitch">KILL(switch)</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/MetabolicBurden">Metabolic Burden</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/ChemicalSynthesis">Chemical Synthesis</A></DIV><BUTTON class="back_top_full"><A href="#mainHeader">Back to the Top</A></BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>