<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Lab_MetabolicBurden skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Lab/MetabolicBurden</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="js-message abstract">
    If you can see this message, you do not use Javascript. This Website is best to use with Javascript enabled. Without Javascript enabled, many features including the mobile version are not usable.
  </DIV><DIV class="vviki" id="vviki"><DIV id="head"><DIV class="navbar {{{extra-class}}}"><UL id="slick-menu"><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Hardware">Robotics</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Team">Team</A></LI></UL></DIV><DIV id="mainHeader"><H1>METABOLIC BURDEN</H1></DIV></DIV><DIV class="page"><DIV class="abstract"><P><B>ABSTRACT</B></P><P>Artificial plasmids are a significant burden to the host. The design of pathways, e.g. the combination of different promoter and RBS systems, results in different amounts of product. Measurement of the metabolic burden is a key for quantitative optimization of metabolic engineering approaches. We want to establish a new approach to iGEM by providing a measurement strain to the community. As described by F. Ceroni <I>et al.</I>, we integrated one copy of the <I>gfp</I> gene into the genome of <I>E. coli</I>, which offers us a highly accurate and instantaneous measurement of the impact of our plasmids on the host. Metabolic burden measurement is of economical interest, because it enables academic and industrial research testing several different pathways at once in a short period of time by using microplate reader.
For the integration we used the λ‑Integrase site‑specific recombination pathway, described by A. Landy in 2015 [9]. Therefore, we designed two plasmids (<A href="http://parts.igem.org/Part:BBa_K1976000">BBa_K1976000</A> and <A href="http://parts.igem.org/Part:BBa_K1976001">BBa_K1976001</A>) and measured them using single cell measurement via microplate reader.

</P></DIV><DIV class="content" id="lab4c"><H5>Metabolic Burden</H5><P>In synthetic biology, the term &quot;metabolic burden&quot; describes the influence of heterologously expressed genes on the distribution availability of resources in the host cell. Thereby the extent of metabolic burden is not only dependent on the consumption of energy and reaction equivalents but is also influenced by a number of different factors. In addition to size and copy number of the used plasmid [1], the specific properties of the heterologous proteins play an important role. Proteins that interfere with the host's metabolism, influence the proton gradient or are toxic in a different way can pose a strong burden for the cell already at low expression levels [1, 2]. I. Shachrai <I>et al</I>. could show that ribosome availability represents a highly limiting factor for metabolic performance [3].
If the metabolic burden on the host cell is too high, the physiology and biochemistry of the cell will be drastically altered, <I>e.g.</I> the viability or proliferation of the cell could be disturbed. In addition, the error rate in translation increases which heightens the immunogenicity of the incorporated proteins and can lead to a reduced protein activity and stability [1]. Metabolic burden is a big issue especially in the industrial sector [4], because lowered expression efficiency decreases product yield. 
Often used methods to reduce the metabolic burden include for example the separation of the biosynthetic pathway into multiple organisms through co‑cultivating them or the identification and deletion of expendable genes as a part of strain optimization [6]. Another approach is the regulation of protein expression with <I>Dynamic Sensor and Regulatory Systems</I> (DSRSs). Such systems are based on the usage of transcription factors which detect certain key‑metabolites and regulate the transcription simultaneously [7]. 
One possibility to quantify the metabolic burden <I>in vivo</I> was described by F. Ceroni <I>et al.</I> In this method a GFP reporter gene is integrated in the genome of <I> E. coli </I> using the <I>λ</I>‑integrase [8]. Through measuring the fluorescence it could be shown that the constitutive expression of GFP after transformation with expression‑plasmids drastically decreases in comparison to not transformed cells. </P><H6>Measurement of the Metabolic Burden via Microplate Reader</H6><P> In addition to the stress forced onto the cell, the total amount of formed GFP is significantly influenced by the cell amount. For this reason the measurement of GFP with a microplate reader was performed under continuous observation of the cell density. With this kind of measurement many samples, like in <I>96‑well‑plates</I>, can be analyzed. Since the measurement of different combinations of plasmid and cell‑type, would enable to determine the artificially caused stress on the cells proportional to the decrease in GFP expression, this kind of measurement is assumed to be a more economical approach than the single cell measurement described in the next paragraph.</P><H6>Single-cell Measurement</H6><P>While measuring every single cell individually, the cell density can be neglected which leads to a smaller error in the  fluorescence measurement. This method enables the detection of the influence of the metabolic burden based on the GFP production. Therefore, it is assumed to be a more exact method than the previously described method using a microplate reader.</P><H5>Genomic Integration</H5><P>The <I>λ</I>‑integrase, originally derived from the <I>λ</I>‑phage, catalyzes the recombination of the phage genome with the chromosomal genome of its host in combination with several assisting proteins. Therefore, two attachment sites are necessary: one located on the bacterial genome (<I>attB</I>) and the other located on the <I>λ</I>‑genome (<I>attP</I>), which also contains several binding sites for regulatory proteins. The attachment sites contain homologous recognition sequences, called BOB' region (<I>attB</I>) and COC' region (<I>attP</I>). These regions can be connected by the <I>λ</I>‑integrase and the bacterial <I>integration host factor</I> (IHF) via <I>Holliday junction</I>, forming an intasome, a DNA‑protein‑complex, producing hybrid attachment sites <I>attL</I> and <I>attR</I>. </P><H6>Integration Plasmid and Helper Plasmid</H6><P>For the integration of the gene of interest (GOI) into the chromosomal genome of <I>E. coli</I> two plasmids are needed. The integration plasmid contains the constitutive generator of GFP, which is also the necessary reporter for the measurement of the metabolic burden and should be integrated into the genome of <I>E. coli</I>. To ensure that only temporary fluorescence are measured, a LVA degradation tag was added to the GFP. The plasmid also contains the <I>attP</I> that enables the integration. Additionally, two bidirectional terminators are located on each side of the <I>attP</I> to protect the GFP operon from the transcription during biosynthesis of neighbouring genes. To create the integration plasmid E0240 (RBS(BB0032+GFP)) was cloned in a J61002 backbone to locate the GFP gene under the control of the promoter J23101. The construct J23101+E0240 was mutated via mutagenesis PCR for optimizing BBa_I11023 and afterwards cloned in the high‑copy vector pSB1C3 to increase the yield of the plasmid preparation. The mutagenesis PCR aimed to mutate <I>attp2</I> to <I>&amp;lambda&amp;;‑attP</I>. To finalize the construct, we changed the backbone to pSB4A5 which possesses a <I>low-copy ori</I> to facilitate the later performed plasmid curing.
The second plasmid is a helper plasmid which is necessary for transposing the <I>gfp</I> gene into the chromosomal genome as it contains the <I>λ</I>‑integrase generator. For inscription in the registry the construct was cloned in a pSB1C3 backbone and , additionally, to facilitate additional performed plasmid curing via sustained lack of selection pressure, we again chose the <I>low‑copy vector</I> pSB4K5 .
To verify the success of the recombination, PCR was performed, using specific primers to the <I>attB</I> site of the <I>E. coli</I> and the VR primer binding on every BioBrick standard vectors. Since the first primer binds to the genome and the other is complimentary to the plasmid, a PCR amplicon could be optained only if the integration has succeeded. </P><CENTER><DIV><B>Figure 1: </B> Overall scheme of the genomic integration of the integration plasmid by the helper plasmid.</DIV></CENTER><H5>Integration Strains</H5><P>A suitable genomic integration strain needs to carry the <I>attB</I> sequence which is necessary for the <I>λ</I>‑integrase mediated homologous recombination. Many commonly used <I>E. coli</I> strains carry the <I>λ</I>‑phage integrated in their genome. But since the <I>attB</I> site was often used to integrate genes via <I>λ</I> recombination, <I>e.g.</I> T7 polymerase in BL21 (DE3), in many strains the attachment site is not accessible anymore. For our integration we chose <I>E. coli</I> JM109 because it matches all our demands and is also freely and easily available to our lab.</P><H5>Results</H5><H6>1. Integration Plasmid - GFP‑construct</H6><P>
The first major step in assembling the integration plasmid was to choose a useful promoter for an expression of GFP, which should be strong enough to be measured but low enough to keep the metabolic burden as low as possible.</P><CENTER><DIV><B>Figure 2: </B><I>E.coli</I> transformed with GFP behind three different promoters. A: GFP and JM23109, B: GFP and JM23115, C: GFP and JM23101</DIV></CENTER><P>
Three different promoters and GFP were transformed into E. Coli to test their strength.
In dish A the GFP was combined with the JM23109 promoter, in dish B with the JM23115 promoter and in dish C with the JM23101 promoter. As it can be seen above in the comparison, GFP is best transcripted with the JM23101 promoter, so we decided to use it in our integration plasmid. With the other two promoters the fluorescence would not be strong enough to be measured after plasmid curing, as there would be only one GFP copy left in the cells.
</P><P>
The next major step was to mutate the synthesized <I>attp2</I>‑site to the needed <I>attp</I>‑site of the λ‑integrase.

This sequence was <B>improved</B> to the following sequence:</P><DIV><B>Figure 3:</B> Compared alignment of the <I>attp2</I> gene and the <I>attp</I> gene. The red marked spots show the mutations that were corrected
</DIV><P>
The third major step in the assembly of the integration plasmid was the adding of a LVA‑Tag to the GFP sequence to ensure a fast degradation. (This was done to further decrease the metabolic burden caused by the GFP and to make a fast answer in the fluorescence possible so accurate measurements can be made.)
Click the following link to get a .zip file with further informations on our <A href="https://2016.igem.org/File:T--TU_Darmstadt--Sequencing_genomic_integration_TUD.zip">sequences</A>. The sequence shows that the mutagenesis PCR for adding the LVA‑Tag was succesful.

</P><P>
The last major step of the assembly was to ligate the λ‑<I>attp</I>+GFP‑LVA construct into a pSB1C3 vector to submit the part to the iGEM registry.
To determine if the ligation was successful a gelelectrophoresis was made using VR‑ and VF2‑primers.</P><DIV><B>Figure 4:</B> Gelelectrophoresis of the λ‑<I>attp</I>+GFP‑LVA construct in pSB1C3. The marked band shows the wanted product</DIV>
The band coming from the sixth batch of the gelelectrophoresis of the cPCR shows that the ligation was succesful. 
<H6>2. Helping Plasmid - Integrase </H6><P>
Due to a mistake in ordering the integrase there was a LVA‑tag at the end of the sequence. 
Therefore the LVA‑tag was deleted with a mutagenesis PCR.
The sequencing shows that the deletion of the LVA‑Tag was succesful.
Click the following link to get a .zip file with further informations on our <A href="https://2016.igem.org/File:T--TU_Darmstadt--Sequencing_genomic_integration_TUD.zip">sequences</A>.

</P><P>
Next was the ligation of the integrase on a pSB1C3 for part submission and to measure if the integrase is expressed.</P><DIV><B>Figure 5: </B> Gelelectrophoresis of the λ‑integrase in pSB1C3. The marked band shows the wanted product</DIV>
The lanes three and five show a succesful ligation in pSB1C3 and are therefore used for further experiments. To determine if the integrase is expressed a SDS‑Page was done.<CENTER><DIV><B>Figure 6</B>: SDS‑Page of the λ‑integrase
The marked spot on the SDS‑Page shows a sufficient expression of the wanted integrase<DIV><P>
To verify whether the integrase works as expected and therefore shows a proof of concept, we tried to genomically integrate our integration plasmid into the genome of <I>E. coli</I> with the enzyme integrase. But before doing that we put both constructs on midi‑copy vectors (pSB1K3 and JM23101). To test if this was succesful a cPCR was performed with attb_fw and VR.</P><DIV class="bild" style="width: 30%"><DIV><DIV><B>Figure 7:</B> Proof for the genomic integration
The gelelectrophoresis shows that the genomic integration in colonies three and four was successful. Hereby it is proven that <B> the integrase works as expected</B>.
</DIV><H6>3. Genomic Integration and Measurement</H6><P>
	As previously described the genomic integration of the integration plasmid was succesful.
	To test if our measurement system works as expected we transformed the cells which carry the GFP with the naringenin biosensor BBa_K1497021 of the 2014 TU Darmstdt iGEM Team.
The measurement showed that the metabolic burden caused by the naringenin biosensor on a high copy vector, the integrase on a high copy vector and the integrating plasmid on a midi copy vector has been way to high to measure any difference in cell activity. There was neither a measured difference in optical density, decrease of GFP expression nor in the increase of mKate expression (Data not shown). Especially the constant optical density in 8 hours of measurement shows that the metabolic burden is way to high, so there is no measurable cell proliferation.  
</P><A name="improvedpart"><H6>4. Improved Part </H6></A><P> For genomic integration via λ-phage (<I>attP/attB</I>) recombination, a functional phage-attachment site (<I>attP</I>) is essential. Here, we initally used the attachment sequence from <A href="http://parts.igem.org/Part:BBa_I11023"> BBa_I11023 </A> that we combined with two bidirectional terminators (<A href=""> BB1001</A>) and had it synthesized by iDT. But the attachment sequence used in BBa_I111023 is not the λ-<I>attP</I>-site, it corresponds to the attachment site <I>attp2</I>, that is used in context of the Gateway ® cloning system. Compared to the λ-phage <I>attP</I>-sequence it bears three significant mutations within its for the recombination process highly relevant O-site. We corrected these mutations by mutagenesis PCR. The λ-<I>attP</I>-sequence, the sequence of BBa_1001 and of our corrected construct are shown in Figure 2. The correct λ‑<I>attp</I>‑sequence is part of our integration plasmid <A href="http://parts.igem.org/Part:BBa_K1976000">BBa_K1976000</A>.
</P><H5>Outlook</H5><P>
	Due to a lack of time the measurement of the metabolic burden could not be executed properly and should therefore be continued in the next year. 
	The biggest problem in the measurement was the high metabolic burden caused by additional plasmids left in the cells. Because of this the first step should be to transpose the integrase and the integration construct on low copy vectors. The second step would be plasmid curing to get rid of the additional plasmids in the cells. Then a new measurement with only one kind of plasmid in the cells, after transformation, should be possible. A quantitative measurement with the desired methods will only be possible after successful plasmid curing.
</P></DIV><DIV class="references"><H6>References</H6><UL><LI>[1] B. Glick, Metabolic load and heterologous gene expression, Biotechnology Advances,
vol. 13, pp. 247261, 1995.</LI><LI>[2] M. Eames and T. Kortemme, Cost-benet tradeos in engineered lac operons, Science,
vol. 336, pp. 911915, 2012.</LI><LI>[3] I. Shachrai, A. Zaslaver, U. Alon, and E. Dekel, Cost of unneeded proteins in E. coli
is reduced after several generations in exponential growth, Molecular Cell, vol. 38,
pp. 758767, 2010.</LI><LI>[4] G. Wu, Q. Yan, J. Jones, Y. Trang, S. Fong, and M. Koas, Metabolic burden: Cornerstones
in synthetic biology and metabolic engineering applications, Trends in Bio-
technology, 2016.</LI><LI>[5] H. Zhang, B. Peireira, Z. Li, and G. Stephanopoulos, Cost of unneeded proteins in E.
coli is reduced after several generations in exponential growth, Proc Natl Acad Sci, vol. 112, pp. 82668271, 2015. </LI><LI>[6] H. Westers, R. Dorenbos, J. M. van Dijl, J. Kabel, T. Flanagan, K. M. Devine, F. Jude,
S. J. Seror, A. C. Beekman, E. Darmon, C. Eschevins, A. de Jong, S. Bron, O. P.
Kuipers, A. M. Albertini, H. Antelmann, M. Hecker, N. Zamboni, U. Sauer, C. Bruand,
D. S. Ehrlich, J. C. Alonso, M. Salas, and W. J. Quax, Genome engineering reveals
large dispensable regions in Bacillus subtilis, Proc Nat Acad Sci, vol. 20, pp. 20762090,
2003. </LI><LI>[7] F. Zhang, J. Carothers, and J. Keasling, Design of a dynamic sensor-regulatory system
for production of chemicals and fuels derived from fatty acids, Nature Biotechnology,
vol. 30, pp. 354359, 2012. </LI><LI>[8] F. Ceroni, R. Algar, G.-B. Stan, and T. Ellis, Quantifying cellular capacity identies
gene expression designs with reduced burden, Nature Methods, vol. 12, pp. 415418,
2015.</LI><LI> [9] A. Landy, Dynamic, structural, and regulatory aspects of lambda site-specific recombination, Annual Review of Biochemisty, vol. 58, pp. 913-949, 1989.</LI></UL></DIV></DIV><DIV class="rechts"><DIV class="scrollbox"><BUTTON class="see_other_full"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Back to the Lab</A></BUTTON><DIV class="highlights"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair">Incorporation of a nnAA</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/Reporter">Reporter</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/KILLswitch">KILL(switch)</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/MetabolicBurden">Metabolic Burden</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/ChemicalSynthesis">Chemical Synthesis</A></DIV><BUTTON class="back_top_full"><A href="#mainHeader">Back to the Top</A></BUTTON></DIV></DIV><DIV class="footer" style="text-decoration: none;"><CENTER><DIV id="igemTUDA"><A href="https://2016.igem.org/Team:TU_Darmstadt" style="color:white">iGEM Technische Universität Darmstadt</A></DIV></CENTER><CENTER><DIV id="footer_linear">
We thank our <A href="https://2016.igem.org/Team:TU_Darmstadt/Team/Acknowledgements" style="color:#019ac8">sponsors</A>, especially:

Email: igem@bio.tu-darmstadt.de | 
Design: Viktoria Schuster</DIV></CENTER></DIV></DIV></DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>