Team:TU Darmstadt/Lab
If you can see this message, you do not use Javascript. This Website is best to use with Javascript enabled. Without Javascript enabled, many features including the mobile version are not usable.
Achievements
Wet Lab
Parts
Robotics
Modeling
Human Practices
Collaborations
Notebook
Results
Team
WET LAB
THE PROJECT
The Colicide Squad is a biosafety system which makes bacterial strains survive only in medium containing a specific non-natural amino acid (nnAA). The genetic circuit requires four parts:
Orthogonal Pair for the incorporation of the nnAA
mVenus as a reporter for low levels of this nnAA
Colicin E2 as a DNA degrading toxin, activated by the absence of the nnAA
λ-Phage Integration for genomic integration of GFP as readout for metabolic burden
image/svg+xml
aaRS
tRNA
mVenus
Zif23-GCN4
mColE2
Im2
Chemical Synthesis
Incorporation
of a nnAA
Reporting
low nnAA
levels
KILL(switch)
Metabolic Burden
λ-Integrase
λ-attP
GFP
Figure 1: The entire genetic circuit of this year's project
INCORPORATION OF A NON-NATURAL AMINO ACID
ABSTRACT
In order to detect the presence of a specific non-natural amino acid (nnAA) in vivo the concept of amber suppression is used. This means the occurrence of the amber stop codon (UAG) in an open reading frame does not cancel protein translation but codes for a specific nnAA, in our case O-methyl-l-tyrosine (OMT). However, for the incorporation the nnAA needs to be present in the medium, otherwise the translation stops. The mechanism requires a tRNA with an anticodon complementary to the amber stop codon and an aminoacyl RNA synthetase (aaRS) loading the tRNA with the specific nnAA. The tRNA and aaRS combination is called an 'orthogonal pair'.
Interested? Read more
REPORTING LOW nnAA LEVELS
ABSTRACT
Glow before you go - What does this actually mean? The aim of our project is to make biology safer by introducing a suicide system into E. coli. Before the suicide is triggered, a reporter protein is expressed to indicate the release of E. coli or to show a deficiency of the non-natural amino acid in the surrounding medium which is necessary for the bacteria to survive. As a reporter protein, we chose mVenus which is a mutant of eYFP. mVenus is located downstream of a promoter which is repressed by a dimeric protein, the Zif23-GCN4 repressor. This repressor carries an amber mutation at position 4 (F4OMT). As a result, the non-natural amino acid O-methyl-l-tyrosine (OMT) is integrated into the protein sequence as long as there is enough OMT in the medium. With decreasing OMT concentration, the translation of the repressor stops due to the early amber stop codon and the repressor cannot bind to the promoter. This leads to the expression of the reporter protein mVenus which can be detected by fluorescence measurements.
Interested? Read more
KILL(SWITCH)
ABSTRACT
Synthetic suicide systems have been chosen safeguards in synthetic biology since the field exists. There are different kinds of designs, often based on a regulating mechanism and a host killing toxin or different kinds of inhibition of metabolic pathways. However, these mechanisms most often do not tackle the problem of synthetic DNA surviving the death of host cells. Here we show a possible design for a simple synthetic killswitch based on the endonuclease Colicin E2 and its corresponding suppressing protein Im2. Endonuclease and suppression protein expression is regulated by amber suppression. Therefore, an amber stop codon coding for the non-natural amino acid O-methyl-l-tyrosine is implemented into the gene. The aim of the system is not to simply kill the host, but also to degrade all DNA within the cell and its surroundings, preventing the escape of transgenic DNA.
Interested? Read more
METABOLIC BURDEN
ABSTRACT
Artificial plasmids are a significant burden to the host. The design of pathways, e.g. the combination of different promoter and RBS systems, results in different amounts of product. Measurement of the metabolic burden is a key for quantitative optimization of metabolic engineering approaches. We want to establish a new approach to iGEM by providing a measurement strain to the community. As described by F. Ceroni et al., we integrated one copy of the GFP gene into the genome of E. coli, which offers us a highly accurate and instantaneous measurement of the impact of our plasmids on the host. Metabolic burden measurement is of economical interest, because it enables academic and industrial research testing several different pathways at once in a short period of time by using microplate reader. For the integration, we used the λ‑Integrase site‑specific recombination pathway, described by A. Landy in 2015. Therefore, we designed two plasmids ( BBa_K1976000 and BBa_K1976001 ) and measured them using single cell measurement via microplate reader.
Interested? Read more
CHEMICAL SYNTHESIS
ABSTRACT
Since non‑natural amino acids are expensive in comparison to natural amino acids, we searched for a high yielding synthesis of O-methyl-l-tyrosine. When chemically altering an amino acid to a non‑natural derivate the higher reactivity of the amino and carboxyl groups in comparison to the desired reactive group has to be considered. For this reason, amino and carboxyl groups need to be protected before carrying out the desired synthesis.
For the protection of the amino group an acetylation reaction was performed to form N‑acetyl‑L‑tyrosine, which was then methylated at the carboxyl group and the hydroxyl group using dimethyl sulfate in a Williamson ether synthesis. In order to finally form the non‑natural amino acid, an acidic hydrolysis using hydrochloric acid was performed.
Interested? Read more
Incorporation of a nnAA
Reporter
KILL(switch)
Metabolic Burden
Chemical Synthesis
Back to the Top
iGEM Technische Universität Darmstadt
We thank our sponsors , especially:
Email: igem@bio.tu-darmstadt.de |  Design: Viktoria Schuster
