<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="js-message abstract">
    If you can see this message, you do not use Javascript. This Website is best to use with Javascript enabled. Without Javascript enabled, many features including the mobile version are not usable.
  </DIV><DIV class="vviki" id="vviki"><DIV class="navbar {{{extra-class}}}"><UL id="slick-menu"><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Hardware">Robotics</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Team">Team</A></LI></UL></DIV><CENTER><DIV class="page_full"><DIV id="mainHeader"><H1>RESULTS</H1></DIV><DIV class="abstract"><P><B>In our iGEM project we managed to create a basic killswitch circuit as shown in Figure 1. In combination with an <A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair">orthogonal pair</A> for non-natural amino acid (nnAA) incorporation, the transformed strain should only survive in presence of the respective nnAA. The killswitch is based on the interaction of the <A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/KILLswitch">Colicin E2 DNase domain</A> (miniColicin) with its corresponding immunity protein, whose functional translation is activated upon presence of the nnAA.</B></P><DIV><B>Figure 1:</B> Design of our killswitch.</DIV></DIV><DIV class="content"><H5>nnAA Synthesis and Incorporation</H5><P>
					For our project we chose the nnAA <I>O</I>‑methyl‑<SPAN style="font-variant:small-caps">l</SPAN>‑tyrosine (OMT). Since this amino acid is relatively expensive in comparison to canonical amino acids, we tried to synthesize OMT ourselves. The synthesis protocol can be found on the dedicated <A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/ChemicalSynthesis">page</A>. The thin layer chromatography of the synthesis delivered promising results since the R<SUB>f</SUB> value of the reaction product is similar to the one corresponding to tyrosine. Further the melting temperature of the product was determined to be 266 °C - 268 °C which is near the literature melting point of OMT (342 °C). For confirmation of the results further analyses are required like nuclear magnetic resonance or mass spectrometry.
					
					The activation of the killswitch operates through translational incorporation of OMT into the immunity protein. For this task the concept of <I>amber</I> supression is used, which requires an <A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair">orthogonal pair</A>. Therefore we designed an <I>E. coli</I> codon optimized version of the OMT tRNA synthetase (aaRS) from <I>Wang et al</I>. We were able to successfully produce the aaRS under the control of the constitutive <I>Anderson</I> promoter as shown in Figure 3. There is an additional band at ~35 kDa, which is not visible in the control sample. Since the orthogonal pair is not completely assembled yet, optimisation of this part is required.
					</P><DIV class="bild" style="width:35%;"><B>Figure 2:</B> SDS-PAGE of <I>E. coli</I> TOP10 culture lysate after 6 hours of constitutive expression of OMT-RS. Left: Cell lysate from <I>E. coli</I> TOP10 not transformed with any plasmid. Right: Cell lysate from <I>E. coli</I> TOP10 transformed with the constitutive OMT generator <A href="http://parts.igem.org/Part:BBa_K1976022">J23101-B0034-OMT-RS (BBa_K1976022)</A>. The OMT-RS holds a molar mass of ~35 kDa.</DIV><H5>Functionality of the Killswitch</H5><P>
					In order to make the killswitch work, it is required that the immunity protein including the <I>amber</I> stop codon is only translated in presence of OMT. Consequently, in absence of OMT we expected no translation of the complete immunity protein. To validate this assumption we induced the immunity protein expression under control of the T7 promoter. We tested two variants of the protein, one including the <I>amber</I> stop codon and one wild-type version including the natural tyrosine codon (Figure 3).
					</P><DIV class="bild" style="width:40%;" id="figure_3"><B>Figure 3:</B> SDS-PAGE of the Im2 immunity protein, the blue arrow marks the band of the expression. From left to right: Im2 without the <I>Amber</I>-mutation and induced with IPTG, with the <I>Amber</I>-mutation and induced with IPTG, with and without the <I>Amber</I>-mutation uninduced.</DIV>
					The functionality of miniColicin was confirmed in an <I>in vivo</I> an an <I>in vitro</I> assay: In our <I>in vivo</I> assay we tested the influence of miniColicin on cell growth of <I>E. coli</I>. Upon transformation of <I>E. coli</I> BL21 with miniColicin (<A href="http://parts.igem.org/Part:BBa_K1976048">(BBa_K1976048)</A>) controlled by the T7 promoter, the amount of obtained colonies after cell plating was significantly lower than the amount after conducting the same experiment with usage of miniColicin (<A href="http://parts.igem.org/Part:BBa_K1976049">(BBa_K1976049)</A>) with a (C266A) mutation (Figure 3). This indicates that the mentioned mutation inactivates miniColicin and therefore the cytotoxic activity causing a low cell count is negated. The transformation was conducted with the same amount of the respective part on pSB1C3. The miniColicin expression was not induced in this experiment, this indicates that the low basal expression the T7 promoter is known for is sufficient to achieve a significant cytotoxic activity.
					<CENTER><DIV class="bild" style="width:40%;" align="right"><B>Figure 4:</B> Growth of BL21 <I>E. coli</I> transformed with a) <A href="http://parts.igem.org/Part:BBa_K1976054"> 
BBa_K1976054</A> and b) <A href="http://parts.igem.org/Part:BBa_K1976055"> 
BBa_K1976055</A> with (C266A) mutation.</DIV></CENTER>
					
					Additionally, the activity was detected via an <I>in vitro</I> assay. At first the expression of miniColicin was confirmed via expression under control of the weak constitutive <I>Anderson</I> promoter <A href="http://parts.igem.org/Part:BBa_J23104">BBa_J23104</A>. Both miniColicin variants, <A href="http://parts.igem.org/Part:BBa_K1976048"> 
BBa_K1976048</A> and <A href="http://parts.igem.org/Part:BBa_K1976049"> 
BBa_K1976049</A>, were detected via SDS_PAGE (Figure 5). Secondly, the endonucleic activity was tested via incubation of purified pSB1C3 with <I>E. coli</I> cell lysate containing miniColicin (<A href="http://parts.igem.org/Part:BBa_K1976048"> 
BBa_K1976048</A> or <A href="http://parts.igem.org/Part:BBa_K1976049"> 
BBa_K1976049</A> respectively) (Figure 6). The 'smear' at ~0.2 kbp and ~3.0 kbp that is visible after incubation with both miniColicin variants indicate a successful DNA degradation. However, the 'smear' could also be caused by DNA in the cell lysate itself. Further investigations are needed to confirm the activity.
					<CENTER><DIV class="bild" style="width:40%;"><B>Figure 5:</B> SDS-PAGE of miniColicin, the blue arrow marks the band of the expression. From left to right: Top10 <I>E. coli</I>, Top10 <I>E. coli</I> transformed with <A href="http://parts.igem.org/Part:BBa_K1976052"> 
BBa_K1976052</A> and Top10 <I>E. coli</I> transformed with <A href="http://parts.igem.org/Part:BBa_K1976053"> 
BBa_K1976053</A>.</DIV></CENTER><CENTER><DIV class="bild" style="width:60%"><B>Figure 6:</B> 1% agarose gel electrophoresis, stained with HD-green. From left to right: pSB1C3 alone, pSB1C3 in the lysate buffer 1x, pSB1C3 in the lysate buffer 3x, pSB1C3 incubated with lysate of Top10 transformed with <A href="http://parts.igem.org/Part:BBa_K1976052"> 
BBa_K1976052</A> 1x, pSB1C3 incubated with lysate of Top10 transformed with <A href="http://parts.igem.org/Part:BBa_K1976052"> 
BBa_K1976052</A> 3x, pSB1C3 incubated with lysate of Top10 transformed with <A href="http://parts.igem.org/Part:BBa_K1976053"> 
BBa_K1976053</A> 1x, pSB1C3 incubated with lysate of Top10 transformed with <A href="http://parts.igem.org/Part:BBa_K1976053"> 
BBa_K1976053</A> 3x.</DIV></CENTER><H5>Detection of a Low Non-Natural Amino Acid Concentration</H5><P>
					In order to detect a low unnatural amino acid concentration, we implemented a <A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/Reporter">reporter system</A> based on the fluorescent protein mVenus. We could create the basic bricks (verified by sequencing) that are necessary for this genetic circuit, but further experiments are to be performed.
				</P><H5>Measurement of Metabolic Burden</H5><P>
					Our biosaftey system is designed to be used in standard bacterial organisms like they are used in academic and industrial context. Therefore it is favorable, if our system causes a minimal metabolic burden to its host cell. In order to <A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/MetabolicBurden">quantify the metabolic burden</A> we genomically integrated GFP like it was previously done by <I>Ceroni et al</I>.</P><H6>1. Integration Plasmid - GFP‑construct</H6><P>
The first major step in assembling the integration plasmid was to choose a useful promoter for an expression of GFP, which should be strong enough to be measured but low enough to keep the metabolic burden as low as possible.</P><CENTER><DIV><B>Figure 2: </B><I>E.coli</I> transformed with GFP behind three different promoters. A: GFP and JM23109, B: GFP and JM23115, C: GFP and JM23101</DIV></CENTER><P>
Three different promoters and GFP were transformed into E. Coli to test their strength.
In dish A the GFP was combined with the JM23109 promoter, in dish B with the JM23115 promoter and in dish C with the JM23101 promoter. As it can be seen above in the comparison, GFP is best transcripted with the JM23101 promoter, so we decided to use it in our integration plasmid. With the other two promoters the fluorescence would not be strong enough to be measured after plasmid curing, as there would be only one GFP copy left in the cells.
</P><P>
The next major step was to mutate the synthesized <I>attp2</I>‑site to the needed <I>attp</I>‑site of the λ‑integrase.

This sequence was <B>improved</B> to the following sequence:</P><DIV><B>Figure 3:</B> Compared alignment of the <I>attp2</I> gene and the <I>attp</I> gene. The red marked spots show the mutations that were corrected
</DIV><P>
The third major step in the assembly of the integration plasmid was the adding of a LVA‑Tag to the GFP sequence to ensure a fast degradation. (This was done to further decrease the metabolic burden caused by the GFP and to make a fast answer in the fluorescence possible so accurate measurements can be made.)
Click the following link to get a .zip file with further informations on our <A href="https://2016.igem.org/File:T--TU_Darmstadt--Sequencing_genomic_integration_TUD.zip">sequences</A>. The sequence shows that the mutagenesis PCR for adding the LVA‑Tag was succesful.

</P><P>
The last major step of the assembly was to ligate the λ‑<I>attp</I>+GFP‑LVA construct into a pSB1C3 vector to submit the part to the iGEM registry.
To determine if the ligation was successful a gelelectrophoresis was made using VR‑ and VF2‑primers.</P><DIV><B>Figure 4:</B> Gelelectrophoresis of the λ‑<I>attp</I>+GFP‑LVA construct in pSB1C3. The marked band shows the wanted product</DIV>
The band coming from the sixth batch of the gelelectrophoresis of the cPCR shows that the ligation was succesful. 
<H6>2. Helping Plasmid - Integrase </H6><P>
Due to a mistake in ordering the integrase there was a LVA‑tag at the end of the sequence. 
Therefore the LVA‑tag was deleted with a mutagenesis PCR.
The sequencing shows that the deletion of the LVA‑Tag was succesful.
Click the following link to get a .zip file with further informations on our <A href="https://2016.igem.org/File:T--TU_Darmstadt--Sequencing_genomic_integration_TUD.zip">sequences</A>.

</P><P>
Next was the ligation of the integrase on a pSB1C3 for part submission and to measure if the integrase is expressed.</P><DIV class="bild" style="width:30%"><B>Figure 5: </B> Gelelectrophoresis of the λ‑integrase in pSB1C3. The marked band shows the wanted product</DIV>
The lanes three and five show a succesful ligation in pSB1C3 and are therefore used for further experiments. To determine if the integrase is expressed a SDS‑Page was done.<CENTER><DIV class="bild" style="width:30%"><B>Figure 6</B>: SDS‑Page of the λ‑integrase
The marked spot on the SDS‑Page shows a sufficient expression of the wanted integrase</DIV></CENTER><P>
To verify whether the integrase works as expected and therefore shows a proof of concept, we tried to genomically integrate our integration plasmid into the genome of <I>E. coli</I> with the enzyme integrase. But before doing that we put both constructs on midi‑copy vectors (pSB1K3 and JM23101). To test if this was succesful a cPCR was performed with attb_fw and VR.</P><DIV class="bild" style="width: 30%"><B>Figure 7:</B> Proof for the genomic integration
The gelelectrophoresis shows that the genomic integration in colonies three and four was successful. Hereby it is proven that <B> the integrase works as expected</B>.
</DIV><H6>3. Genomic Integration and Measurement</H6><P>
	As previously described the genomic integration of the integration plasmid was succesful.
	To test if our measurement system works as expected we transformed the cells which carry the GFP with the naringenin biosensor BBa_K1497021 of the 2014 TU Darmstdt iGEM Team.
The measurement showed that the metabolic burden caused by the naringenin biosensor on a high copy vector, the integrase on a high copy vector and the integrating plasmid on a midi copy vector has been way to high to measure any difference in cell activity. There was neither a measured difference in optical density, decrease of GFP expression nor in the increase of mKate expression (Data not shown). Especially the constant optical density in 8 hours of measurement shows that the metabolic burden is way to high, so there is no measurable cell proliferation.  
</P><A name="improvedpart"><H6>4. Improved Part </H6></A><P> For genomic integration via λ-phage (<I>attP/attB</I>) recombination, a functional phage-attachment site (<I>attP</I>) is essential. Here, we initally used the attachment sequence from <A href="http://parts.igem.org/Part:BBa_I11023"> BBa_I11023 </A> that we combined with two bidirectional terminators (<A href=""> BB1001</A>) and had it synthesized by iDT. But the attachment sequence used in BBa_I111023 is not the λ-<I>attP</I>-site, it corresponds to the attachment site <I>attp2</I>, that is used in context of the Gateway ® cloning system. Compared to the λ-phage <I>attP</I>-sequence it bears three significant mutations within its for the recombination process highly relevant O-site. We corrected these mutations by mutagenesis PCR. The λ-<I>attP</I>-sequence, the sequence of BBa_1001 and of our corrected construct are shown in Figure 2. The correct λ‑<I>attp</I>‑sequence is part of our integration plasmid <A href="http://parts.igem.org/Part:BBa_K1976000">BBa_K1976000</A>.

				</P><H5>Modeling of the Interaction of MiniColicin with the Immunity Protein</H5><P>Three amino acid positions were chosen for <I>O</I>-methyl-<SPAN style="font-variant: small-caps;">l</SPAN>-tyrosine (OMT) exchange evaluation (tyrosine 8 (Y8), phenylalanine 13 (F13) and phenylalanine 16 (F16)). These positions were selected because of their small deviation in regard to OMT and were therefore expected to cause the smallest structural differences. All molecular dynamics steps were performed on these mutation variants, as well as on the wildtype protein for comparison. All simulations were performed over 100 ns, leading to 10001 conformations each. These conformations were evaluated using RMSD, RMSF, SASA and the amount of secondary structures over time.Figure 7 displays the RMSD in regard to the initial conformation. It can be observed that the mutational variant Y8O exhibits similar curve characteristics as the wildtype variant. The mutational variant F16O on the contrary shows a more severe deviation from the wildtype.
					</P><DIV class="bild" style="width:70%;"><B>Figure 13:</B> RMSD of the mutation variants Y8O (green), F16O (blue) and the wildtype (red).</DIV>
					The solvent accessible surface area (SASA) was calculated for every simulation step using DSSP and is displayed in Figure 8. The little to no fluctuation in the SASA of all simulated variants is an argument for the high structural stability of the wildtype and the mutational variants. The disparity between the mutational variants and the wildtype can be traced back to the DSSP algorithm since it is based solely on natural amino acids. Therefore it cannot evaluate a non-natural amino acid and would cause a discrepancy between surface areas if non-natural amino acids like OMT are involved.
					<CENTER><DIV class="bild" style="width:70%;"><B>Figure 14:</B> SASA of the mutation variants Y8O (green), F16O (blue) and the wildtype (red).</DIV></CENTER>
					Additionally we evaluated the RMSF and the amount of secondary structures over time (data not shown) which exhibited no differences between the mutational variants and the wildtype. Therefore the results of these evaluation methods are not shown on this wiki.Closing up we can conclude that the mutational variant Y8O does fit our demands best, since its behavior exhibits the least disparity in our applied evaluation methods towards the wildtype. Therefore Y8O was chosen for further analyses.
					Furthermore we evaluated the stability of our designed miniColicin by performing 100 ns of MD simulations with different force fields (CHARMm27, AMBER03, GROMOS56a7). The RMSD of these simulations is displayed in Figure 15. It can be observed that some deviations between the different force fields occur. The simulation run with the GROMOS56a7 shows the biggest discrepancy towards the AMBER and CHARMm simulations which display little to no fluctuations over time. This behavior can be traced to the different parametrization approach in the force fields as well as to the fact that the GROMOS56a7 force field is a united atom force field, in which all CH<SUB>3</SUB> and CH<SUB>2</SUB> groups are described as one group.
					<CENTER><DIV class="bild" style="width:76%;"><B>Figure 15:</B> RMSD of miniColicin simulated using the the force fields CHARMm27 (red), AMBER03 (blue) and GROMOS56a7 (green).</DIV></CENTER><H5>Construction of a Pipetting Robot</H5><P>
					In order to facilitate the work with our biosafety system, we constructed a <A href="https://2016.igem.org/Team:TU_Darmstadt/Hardware">pipetting robot</A> based on an 'Ultimaker 2' 3D printer. This robot should be able to detect cell samples expressing mVenus, due to a low nnAA level, and automatically refill the respective samples with the nnAA.
The robot can move a probe in x, y, and z direction and the accuracy is as good as a normal 3D printer. It has an infrared table illuminating the samples homogeneously from underneath which makes a sample detection within a rack possible. Our syringe pump is working and can dispense single drops of liquids into the samples. Our tracking system is capable of detecting samples and also is capable of track them, if they are moved. The high power LEDs can excite mVenus and a long pass filter ensures to filter the high power LED light out for measuring only the light, emitted from mVenus. This makes it possible to take long time exposure photos to record more data. The camera has an auto focus routine to improve the data acquisition.
We started to program a GUI uniting the developed functionality.
Our robot can be controlled via a network and further improvements are already in pipeline.
We deliver a complete bill of materials, a construction video and all needed CADs. Furthermore, we deliver a full image for a 'Raspberry Pi' computer.
				</P></DIV></DIV></CENTER><DIV class="footer" style="text-decoration: none;"><CENTER><DIV id="igemTUDA"><A href="https://2016.igem.org/Team:TU_Darmstadt" style="color:white">iGEM Technische Universität Darmstadt</A></DIV></CENTER><CENTER><DIV id="footer_linear">
We thank our <A href="https://2016.igem.org/Team:TU_Darmstadt/Team/Acknowledgements" style="color:#019ac8">sponsors</A>, especially:

Email: igem@bio.tu-darmstadt.de | 
Design: Viktoria Schuster</DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>