<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Lab_KILLswitch skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Lab/KILLswitch</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="js-message abstract">
    If you can see this message, you do not use Javascript. This Website is best to use with Javascript enabled. Without Javascript enabled, many features including the mobile version are not usable.
  </DIV><DIV class="vviki" id="vviki"><DIV id="head"><DIV class="navbar {{{extra-class}}}"><UL id="slick-menu"><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Hardware">Robotics</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Team">Team</A></LI></UL></DIV><DIV id="mainHeader"><H1>KILL(switch)</H1></DIV></DIV><DIV class="page"><DIV class="abstract"><P><B>ABSTRACT</B><B>Synthetic suicide systems have been safeguards in synthetic biology for as long as the field exists. There are different kinds of designs, often based on a regulating mechanism and a host killing toxin or different kinds of inhibition of metabolic pathways. However, these mechanisms most often do not tackle the problem of synthetic DNA surviving the death of host cells.
Here we show a possible design for a simple synthetic killswitch based on the endonuclease Colicin E2 and its corresponding suppressing protein Im2. Endonuclease and suppression protein expression is regulated by amber suppression. Therefore, an amber stop codon coding for the non-natural amino acid <I>O</I>-methyl-<SPAN style="font-variant:small-caps">l</SPAN>-tyrosine is implemented into the gene. The aim of the system is not to simply kill the host, but also to degrade all DNA within the cell and its surroundings, preventing the escape of transgenic DNA.
</B></P></DIV><DIV class="content" id="maintext_mico"><H5>Killswitch</H5><P>
There are a lot of fears in the general public's mind when it comes to genetics and genetically modified organisms (GMOs). 
That represent a major burden synthetic biology has to overcome before it will gain widespread acceptance for more applications in daily life.

Since the advent of synthetic biology, scientists have been working hard to implement safeguards in their genetically modified organisms. They were making use of many different designs such as controlled essential gene regulations [1], engineered auxotrophies [2] and inducible toxin switches [3].

One problem that arises with most of these systems is not tackling the problem of bioorthogonal DNA escaping into the environment after cell death. That DNA cannot only be used for industrial espionage, but  bacteria in the environment could also get transformed with synthetic plasmids and thus propagate synthetic pathways throughout ecosystems.

The general aim of a killswitch is to kill its host, most often bearer of plasmids or other forms of synthetic DNA. They are generally designed with some sort of regulatory mechanism connected to a host-killing element. There is a high variability of host-killing mechanisms. The toxin we chose for the design of our killswitch is colicin E2, a nicking endonuclease [4]. Colicins occur in some wild-type coliform bacteria and are used to kill rivaling strains of <I>Escherichia coli</I>. 

An essential part of the design is the regulation of the toxin activity. Colicin E2 has a natural counterpart, the immunity protein Im2, which inhibits colicin E2 by tightly binding to it. It can be used as a suppression mechanism for our toxin to protect the bearer of the colicin E2 from its martial effects.

We chose to use the <I>amber</I> suppression [5] for regulating the expression of the immunity protein. By introducing a bioorthogonal pair of a <A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair">tRNA and an aminoacyl RNA synthetase</A> into the cells, we can insert a non-natural amino acid into the sequence of the immunity protein. The rational design approach was validated by <A href="https://2016.igem.org/Team:TU_Darmstadt/Model">molecular dynamic simulation</A> to ensure a functional suppression system.

In the design of our killswitch, we aimed for enabling the user to not simply create genetically modified organisms safely, but also to minimize the risk of their vectors to propagate freely throughout ecosystems and to protect the users intellectual property from being stolen or recreated. The activity of our toxin after cell death assures DNA degradation in the vicinity of the host.
</P><DIV class="content" id="#design_text"><H5>Design</H5><H6>Killswitch</H6><P>Our killswitch approach uses constitutive promoters for expressing the DNase and the immunity protein (see Figure 1).  Since the immunity protein concentration needs to be at least equimolar to the colicin E2 concentration to obtain a tight suppression, a higher  expression than an equimolar expression is desirable. The killswitch is activated as soon as the biosynthesis of the immunity protein is inhibited by the lack of the non-natural amino acid OMT, causing incomplete protein translation of the repressor and in consequence biosynthesis of the DNase. Functional expression of the DNase causes cell death.</P></DIV><CENTER><DIV><B>Figure 1:</B> Design of our killswitch, including constitutive anderson promoter systems for the DNase miniColicin and the Im2 immunity protein.</DIV></CENTER><DIV class="content"><H6>MiniColicin</H6><P>Colicin E2 is comprised of three domains (see Figure 2), resulting in a considerable metabolic burden. Two of the three domains are import and export signals and thus not necessary for its DNase activity. To minimalize the metabolic burden, caused by incorporating our killswitch into the cells, we designed a rational approach to express only the DNase domain of colicin E2. By removing the import and export signals on DNA level, we shortened the sequence by more than two-thirds. In addition, we modelled its stability to validate the approach <I>in silico</I>. 
</P></DIV></DIV><DIV class="content"><CENTER><DIV class="bild" style="width:70%" id="figure_2"><B>Figure 2:</B> Complex of colicin E2 (black: export domain; blue: import domain; yellow: DNase) and Im2 (green), <A href="https://2016.igem.org/Team:TU_Darmstadt/Model#Mod_m">rendered with PyMol v1.7.2.1</A></DIV></CENTER></DIV><DIV class="content" id="results_mico"><H5>Results</H5><H6>Expression</H6><P>For the constructs we assembled containing corresponding Anderson promoters, we were able to show expression, as you can see in figure three and four.</P><CENTER><DIV><DIV><DIV class="bild" style="width:40%;" id="figure_3"><B>Figure 3:</B> SDS-PAGE of the Im2 immunity protein, the blue arrow marks the band of the expression. From left to right: Im2 without the <I>Amber</I>-mutation and induced with IPTG, with the <I>Amber</I>-mutation and induced with IPTG, with and without the <I>Amber</I>-mutation uninduced.</DIV></DIV><DIV><DIV class="bild" style="width:40%;" id="figure_4"><B>Figure 4:</B> SDS-PAGE of miniColicin, the blue arrow marks the band of the expression. From left to right: Top10 <I>E. coli</I>, Top10 <I>E. coli</I> transformed with <A href="http://parts.igem.org/Part:BBa_K1976052"> 
BBa_K1976052</A> and Top10 <I>E. coli</I> transformed with <A href="http://parts.igem.org/Part:BBa_K1976053"> 
BBa_K1976053</A>.</DIV></DIV></DIV></CENTER><H6>Functionality of DNase</H6><P>A first hint given at the functionality of our proteins was by transforming Top10 <I>E. coli</I> with<A href="http://parts.igem.org/Part:BBa_K1976055"> 
BBa_K1976055</A> and <A href="http://parts.igem.org/Part:BBa_K1976056"> 
BBa_K1976056</A> to show expression by inducing with IPTG. As you can see in Figure 5, the leakage of the T7 promoter alone seemed to show significant growth inhibition, even without induction.</P><CENTER><DIV class="bild" style="width:40%;" align="right" id="figure_5"><B>Figure 5:</B> growth of BL21 <I>E. coli</I> transformed with a): <A href="http://parts.igem.org/Part:BBa_K1976054"> 
BBa_K1976054</A> and b): <A href="http://parts.igem.org/Part:BBa_K1976055"> 
BBa_K1976055</A>.</DIV></CENTER></DIV><P>
To further characterize the functionality of our protein, we we cleaned up a lysate of Top10 to the point we desired, which were transformed with our generators, as reported in our <A href="https://2016.igem.org/Team:TU_Darmstadt/Notebook/Methods">methods</A> under the point.
We incubated pSB1C3 together with the lysate for an hour at 37°C and ran an agarose gel with it to show its functionality, as you can see in figure 6.
</P><CENTER><DIV class="bild" style="width:60%" id="figure_6"><B>Figure 6:</B> 1% agarose gel electrophoresis, stained with HD-green. From left to right: pSB1C3 alone, pSB1C3 in the lysate buffer 1x, pSB1C3 in the lysate buffer 3x, pSB1C3 incubated with lysate of Top10 transformed with <A href="http://parts.igem.org/Part:BBa_K1976052"> 
BBa_K1976052</A> 1x, pSB1C3 incubated with lysate of Top10 transformed with <A href="http://parts.igem.org/Part:BBa_K1976052"> 
BBa_K1976052</A> 3x, pSB1C3 incubated with lysate of Top10 transformed with <A href="http://parts.igem.org/Part:BBa_K1976053"> 
BBa_K1976053</A> 1x, pSB1C3 incubated with lysate of Top10 transformed with <A href="http://parts.igem.org/Part:BBa_K1976053"> 
BBa_K1976053</A> 3x.</DIV></CENTER><DIV class="references"><H6>References</H6><UL><LI>[1]<SPAN id="#source_one"> Kong, W. et al. Regulated programmed lysis of recombinant Salmonella in host tissues to release protective antigens and confer biological containment. Proc. Natl Acad. Sci. USA105, 9361-9366 (2008)</SPAN></LI><LI id="#source_two">[2]<SPAN> Steidler, L. et al. Biological containment of genetically modified Lactococcus lactis for intestinal delivery of human interleukin 10. Nature Biotechnol. 21, 785-789 (2003)</SPAN></LI><LI>[3]<SPAN id="#source_three"> Szafranski, P. et al. A new approach for containment of microorganisms: dual control of streptavidin expression by antisense RNA and the T7 transcription system. Proc. Natl Acad. Sci. USA 94, 1059-1063 (1997)</SPAN></LI><LI>[4]<SPAN id="#source_four"> Schaller, K. and Nomura, M. Colicin E2 is DNA endonuclease. Proc. Natl Acad. Sci. USA 73, No. 11, pp. 3989-S993 November (1976) Biochemistry</SPAN></LI><LI>[5]<SPAN id="#source_five"> Lang, K. and Chin, J. W., Cellular Incorporation of Unnatural Amino Acids and Bioorthogonal Labeling of Proteins, 114, 4764-4806 Chem. Rev. (2014)</SPAN></LI></UL></DIV></DIV><DIV class="rechts"><DIV class="scrollbox"><BUTTON class="see_other_full"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Back to the Lab</A></BUTTON><DIV class="highlights"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair">Incorporation of OMT</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/Reporter">Reporter</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/KILLswitch">KILL(switch)</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/MetabolicBurden">Metabolic Burden</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/ChemicalSynthesis">Chemical Synthesis</A></DIV><BUTTON class="back_top_full"><A href="#mainHeader">Back to the Top</A></BUTTON></DIV></DIV><DIV class="footer" style="text-decoration: none;"><CENTER><DIV id="igemTUDA"><A href="https://2016.igem.org/Team:TU_Darmstadt" style="color:white">iGEM Technische Universität Darmstadt</A></DIV></CENTER><CENTER><DIV id="footer_linear">
We thank our <A href="https://2016.igem.org/Team:TU_Darmstadt/Team/Acknowledgements" style="color:#019ac8">sponsors</A>, especially:

Email: igem@bio.tu-darmstadt.de | 
Design: Viktoria Schuster</DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>