<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Lab_Collaboration skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Lab/Collaboration</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="js-message abstract">
    If you can see this message, you do not use Javascript. This Website is best to use with Javascript enabled. Without Javascript enabled, many features including the mobile version are not usable.
  </DIV><DIV class="vviki" id="vviki"><DIV id="head"><DIV class="navbar"><DIV class="tablet" style="display: none"><DIV id="menu"><DIV id="tabletmenu"><UL><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A class="current" href="https://2016.igem.org/Team:TU_Darmstadt/Lab">In the Lab</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Parts">Parts</A></LI><LI><A href="">Results</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Hardware">Robotics</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Notebook">Labbook</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Team">Team</A></LI></UL></DIV></DIV></DIV><DIV class="computer" id="computer" style="display: none"><UL><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A class="current" href="https://2016.igem.org/Team:TU_Darmstadt/Lab">In the Lab</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Parts">Parts</A></LI><LI><A href="">Results</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Hardware">Robotics</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Team">Team</A></LI></UL></DIV></DIV><DIV id="mainHeader"><H1>INCORPORATION of OMT</H1></DIV></DIV><DIV class="page"><DIV class="abstract"><P>ABSTRACT</P><P>In order to detect the presence of the specific non-natural amino acid (nnAA) <I>in vivo</I>, the concept of <B><I>amber</I> suppression</B> is used [1]. This means the occurrence of the <I>amber</I> stop codon (UAG) in an <I>open reading frame</I> ORF does not cancel the protein translation but codes for a specific nnAA, in our case <I>O</I>-methyl-<SPAN style="font-variant:small-caps">l</SPAN>-tyrosine (OMT). However, without the nnAA in the medium the incorporation is not possible, the translation stops at the position. The mechanism requires a <B>tRNA</B> with an anticodon complementary to the <I>amber</I> stop codon as well was an aminoacyl RNA synthetase (aaRS), which loads the tRNA with the specific nnAA. The tRNA and aaRS combination is called an 'orthogonal pair'.</P></DIV><DIV class="content" id="lab1c"><H5>Orthogonal Pair</H5><P>The recognition of the <I>amber</I> stop codon requires a tRNA with an anticodon complementary to the <I>amber</I> stop codon and an aaRS specifically loading the tRNA with the nnAA. In order to ensure the nnAA is not incorporated for other codons except the <I>amber</I> stop codon, the tRNA and the aaRS have to be orthogonal to the natural aaRS's and tRNAs. This means the aaRS must not load any other tRNA and the tRNA must not be loaded by any other aaRS. Therefore, <I>Wang et. al</I> originally used the tyrosyl-tRNA and tyrosyl-RS from the methanogenic archaeon <I>Methanocaldococcus jannaschii</I> : The anticodon of the tRNA was replaced by the <I>amber</I> anticodon and the aaRS was optimized for the recognition of OMT (Figure 1) in place of tyrosine via directed evolution. Introduced into <I>Escherichia coli</I>, this pair is orthogonal to every natural pair due to the genetic distance between <I>E. coli</I> and <I>M. jannaschii</I>. Nowadays, over 70 different aaRS <B>[3]</B> have been designed, each one capable of incorporating a specific amino acid, many of them with special chemical characteristics, allowing e.g. 'click' chemistry or photoactivation.</P><P> In our project, we use an orthogonal pair from the <A href="https://2014.igem.org/Team:Austin_Texas/kit">&quot;Expanded Genetic Code Measurement Kit&quot;</A> as template, specifically the one used for incorporation of ONBY <A>(BBa_SomeBrick)</A>, and replaced the ORF with an <I>E. coli</I> codon optimized ORF for OMT-RS. Furthermore we placed the OMT-RS coding region behind a RBS <A href="http://partsregistry.org/Part:BBa_B0034">(BBa_B0034)</A> and a strong constitutive Anderson promotor <A href="http://parts.igem.org/Part:BBa_J23101">(BBa_J23101)</A>.</P><DIV class="bild"><B>Figure 1:</B> Dimer of the <I>Methanocaldococcus jannaschii</I> tyrosyl-tRNA synthetase specific for <I>O</I>-methyl-tyrosine (RCSB PDB entrance <A href="http://www.rcsb.org/pdb/explore.do?structureId=1u7x">1U7X</A>)</DIV><H5>Usage of <I>amber</I> codon</H5><P>The incorporation of an <I>amber</I> codon causes the complete translation of the respective protein in presence of the nnAA and cancels the translation in absence. In our implementation the <I>amber</I> codon is replacing a codon in the beginning of the ORFs of the <A href="bla">Colicin E2 Immunity protein</A> (Y8OMT) and the <A href="bla">Zif23-GCN4 repressor</A> (F4OMT). In consequence, both proteins are functionally produced only if the nnAA is available in sufficient concentration in the medium.</P><H5>The non-natural amino acid</H5><P>We decided to use <I>O</I>-methyl-<SPAN style="font-variant:small-caps">l</SPAN>-tyrosine for our nnAA due to its multiple advantageous properties:</P><UL style="list-style-type:disc"><LI>Low costs</LI><LI>Nontoxic</LI><LI>Unproblematic import into cells</LI><LI>No further biochemical activity</LI><LI>Feasible chemical synthesis</LI><LI>Stable in water</LI><LI>Unavailable in nature</LI><LI>Well documented</LI><LI>Low interference with protein activity</LI></UL><P>An institute or company could choose its own specific nnAA with the corresponding orthogonal pair. This enables a reliable protection against corporate espionage or bioterrorism, since the opposing party does normally not know which nnAA is used in the respective application. However, using the same nnAA like OMT in every application should prevent the biological and genetic spread of the respective microorganism in the environment.</P><H5>Results</H5><P>The OMT-RS was successfully expressed under control of the strong constitutive Anderson promoter <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A> with the RBS  <A href="http://partsregistry.org/Part:BBa_B0034">BBa_B0034</A> as showed in Figure 2. The expression was conducted in TOP10 cells. Furthermore the OMT-RS was epxressed in control of the combined T7 promoter and RBS brick <A href="http://parts.igem.org/Part:BBa_K525998">BBa_K525998</A> in BL21. In this case no expression was detected via SDS_PAGE up to 6 h after induction with 10 m<SPAN style="font-variant:small-caps">m</SPAN> IPTG. This negative result might be caused by <A href="http://parts.igem.org/Part:BBa_K525998">BBa_K525998</A> itself, since the brick does not contain a spacer sequence between the T7 promoter and the RBS. Similar negative results were also observed in the expression of <A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/Reporter">mVenus</A>.</P><CENTER><DIV class="bild" style="width:35%;"><B>Figure 2:</B> SDS-PAGE of TOP10 culture lysate with and without OMT tRNA synthatase. The left column shows a lysate from a TOP10 control culture with no plasmid, the right column shows the lysate from a TOP10 culture containing the J23101-B0034-Synthetase (BBa_blabla). The blue arrow indicates the additional band corresponding the OMT tRNA synthatase at ~35 kDa.</DIV></CENTER></DIV><DIV class="references"><H6>References</H6><UL><LI>[1]</LI><LI>[2]</LI><LI>[3]</LI></UL></DIV></DIV><DIV class="rechts"><DIV class="scrollbox"><BUTTON class="see_other_full"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Back to the Lab</A></BUTTON><DIV class="highlights"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair">Incorporation of OMT</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/Reporter">Reporter</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/KILLswitch">KILL(switch)</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/MetabolicBurden">Metabolic Burden</A><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/ChemicalSynthesis">Chemical Synthesis</A></DIV><BUTTON class="back_top_full"><A href="#mainHeader">Back to the Top</A></BUTTON></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>