<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Lab skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Lab</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="js-message abstract">
    If you can see this message, you do not use Javascript. This Website is best to use with Javascript enabled. Without Javascript enabled, many features including the mobile version are not usable.
  </DIV><DIV class="vviki" id="vviki"><DIV id="head"><DIV class="navbar {{{extra-class}}}"><UL id="slick-menu"><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Achievements">Achievements</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab">Wet Lab</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Hardware">Robotics</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Team">Team</A></LI></UL></DIV><DIV id="mainHeader"><H1>WET LAB</H1></DIV></DIV><DIV class="page"><DIV class="abstract"><P><B>THE PROJECT</B></P><P>The Colicide Squad is a biosafety system which makes bacterial strains survive only in medium containing a specific non-natural amino acid (nnAA). The genetic circuit requires four parts:<UL style="list-style-type:disc"><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair">Orthogonal Pair</A> for the incorporation of the nnAA</LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/Reporter">mVenus</A> as a reporter for low levels of this nnAA</LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/KILLswitch">Colicin E2</A> as a DNA degrading toxin, activated by the absence of the nnAA</LI><LI><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/MetabolicBurden">λ-Phage Integration</A> for genomic integration of GFP as readout for metabolic burden</LI></UL></P></DIV><CENTER><DIV class="bild" style="width:80%;"><SVG xmlns:DC="http://purl.org/dc/elements/1.1/" xmlns:CC="http://creativecommons.org/ns#" xmlns:RDF="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:SVG="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg" xmlns:XLINK="http://www.w3.org/1999/xlink" xmlns:SODIPODI="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd" xmlns:INKSCAPE="http://www.inkscape.org/namespaces/inkscape" inkscape:export-ydpi="100" inkscape:export-xdpi="100" inkscape:export-filename="C:\Users\Viktoria\Documents\University\IGEM\Corporate Design\wiki-grafiken\SumItUp.png" sodipodi:docname="T--TU_Darmstadt--WetLab_chris.svg" inkscape:version="0.91 r" version="1.1" id="svg2" viewbox="0 0 1771.6535 1771.6535"><METADATA id="metadata7"><RDF:RDF><CC:WORK rdf:about=""><DC:FORMAT>image/svg+xml</DC:FORMAT></CC:WORK></RDF:RDF></METADATA><G transform="translate(0,719.29134)" id="layer1" inkscape:groupmode="layer" inkscape:label="Ebene 1"><G transform="matrix(0.78137935,0.00137306,-0.00137535,0.78007542,-336.38397,-148.59182)" id="g5492" style="display:inline;stroke:#fe2d81;stroke-width:53.79592133;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"><G transform="matrix(0.68577902,0,0,0.68577902,802.82136,329.36708)" id="g4184"><G id="aaRS" inkscape:label="#g4309"><TEXT transform="matrix(0.86093697,0.50792138,-0.50706877,0.86237316,0,0)" sodipodi:linespacing="125%" id="text4387" y="-1580.0018" x="678.1972" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;display:inline;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" xml:space="preserve"><TSPAN style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:74.70949554px;font-family:Calibri;-inkscape-font-specification:'Calibri Bold';fill:#ffffff" y="-1580.0018" x="678.1972" id="tspan4389" sodipodi:role="line">aaRS</TSPAN></TEXT></G><G id="tRNA" inkscape:label="#g4345"><TEXT xml:space="preserve" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;display:inline;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" x="1190.1185" y="-1587.6243" id="text4391" sodipodi:linespacing="125%" transform="matrix(0.86093697,0.50792138,-0.50706877,0.86237316,0,0)"><TSPAN sodipodi:role="line" id="tspan4393" x="1190.1185" y="-1587.6243" style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:74.70949554px;font-family:Calibri;-inkscape-font-specification:'Calibri Bold';fill:#ffffff">tRNA</TSPAN></TEXT></G><G id="mVenus" inkscape:label="#g4394"><TEXT transform="matrix(0.00176162,1.0008339,-0.99916371,0.00175869,0,0)" sodipodi:linespacing="125%" id="text4395" y="-2246.1797" x="228.1183" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;display:inline;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" xml:space="preserve"><TSPAN style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:74.70949554px;font-family:Calibri;-inkscape-font-specification:'Calibri Bold'" y="-2246.1797" x="228.1183" id="tspan4397" sodipodi:role="line">mVenus</TSPAN></TEXT></G><G id="Zif23-GCN4" inkscape:label="#g4376"><TEXT xml:space="preserve" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;display:inline;fill:#ffffff;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" x="-365.8222" y="-2238.7087" id="text4425" sodipodi:linespacing="125%" transform="matrix(0.00176162,1.0008339,-0.99916371,0.00175869,0,0)"><TSPAN sodipodi:role="line" id="tspan4427" x="-365.8222" y="-2238.7087" style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:74.70949554px;font-family:Calibri;-inkscape-font-specification:'Calibri Bold';fill:#ffffff">Zif23-GCN4</TSPAN></TEXT></G><G id="mColE2" inkscape:label="#g4408"><TEXT transform="matrix(0.86334121,-0.5037999,0.50296435,0.86478741,0,0)" sodipodi:linespacing="125%" id="text4421" y="1857.6792" x="1036.5864" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" xml:space="preserve"><TSPAN style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:74.70949554px;font-family:Calibri;-inkscape-font-specification:'Calibri Bold';fill:#ffffff" y="1857.6792" x="1036.5864" id="tspan4423" sodipodi:role="line">mColE2</TSPAN></TEXT></G><G id="g4963"><G id="Im2" inkscape:label="#g4432"><TEXT transform="matrix(0.86334121,-0.5037999,0.50296437,0.8647874,0,0)" sodipodi:linespacing="125%" id="text4437" y="1848.1503" x="542.97485" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" xml:space="preserve"><TSPAN style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:74.70949554px;font-family:Calibri;-inkscape-font-specification:'Calibri Bold';fill:#ffffff" y="1848.1503" x="542.97485" id="tspan4439" sodipodi:role="line">Im2</TSPAN></TEXT></G></G><G id="TextsInTheRing" inkscape:label="#g5056"><G transform="matrix(1.2483785,0,0,1.2483785,233.51564,-451.53447)" id="g4270"><TEXT inkscape:label="#text4351" transform="matrix(0.86278464,-0.50475576,0.50391854,0.86422982,0,0)" sodipodi:linespacing="125%" id="circuit_ChemSyn" y="32.798939" x="62.542736" style="font-style:normal;font-weight:normal;font-size:37.40326691px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" xml:space="preserve"><TSPAN style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:78.54685974px;font-family:Calibri;-inkscape-font-specification:Calibri" y="32.798939" x="62.542736" sodipodi:role="line" id="tspan4355">Chemical Synthesis</TSPAN></TEXT></G><TEXT transform="matrix(0.87099617,0.490412,-0.48958856,0.87244918,0,0)" sodipodi:linespacing="125%" id="text4357" y="-1322.2103" x="1026.9" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;text-align:center;letter-spacing:0px;word-spacing:0px;text-anchor:middle;display:inline;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" xml:space="preserve"><TSPAN style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:98.05621338px;font-family:Calibri;-inkscape-font-specification:Calibri;text-align:center;text-anchor:middle" y="-1322.2103" x="1026.9" sodipodi:role="line" id="tspan4361">Incorporation</TSPAN><TSPAN id="tspan4365" style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:98.05621338px;font-family:Calibri;-inkscape-font-specification:Calibri;text-align:center;text-anchor:middle" y="-1224.0267" x="1026.9" sodipodi:role="line">of a nnAA</TSPAN></TEXT><TEXT transform="matrix(0.99916372,-0.00175869,0.00176162,1.0008339,0,0)" sodipodi:linespacing="125%" id="text4325" y="36.760098" x="1929.9784" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;text-align:end;letter-spacing:0px;word-spacing:0px;text-anchor:end;display:inline;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" xml:space="preserve"><TSPAN style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:98.05621338px;font-family:Calibri;-inkscape-font-specification:Calibri;text-align:end;text-anchor:end" y="36.760098" x="1947.7742" id="tspan4327" sodipodi:role="line">Reporting </TSPAN><TSPAN id="tspan4341" style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:98.05621338px;font-family:Calibri;-inkscape-font-specification:Calibri;text-align:end;text-anchor:end" y="134.94366" x="1947.7742" sodipodi:role="line">low nnAA </TSPAN><TSPAN id="tspan4345" style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:98.05621338px;font-family:Calibri;-inkscape-font-specification:Calibri;text-align:end;text-anchor:end" y="233.12724" x="1929.9784" sodipodi:role="line">levels</TSPAN></TEXT><TEXT transform="matrix(0.86203872,-0.50603284,0.50519349,0.86348267,0,0)" sodipodi:linespacing="125%" id="text4367" y="1625.8903" x="656.29315" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;display:inline;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" xml:space="preserve"><TSPAN style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:98.05621338px;font-family:Calibri;-inkscape-font-specification:Calibri" y="1625.8903" x="656.29315" sodipodi:role="line" id="tspan4371">KILL(switch)</TSPAN></TEXT><TEXT xml:space="preserve" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;display:inline;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1" x="701.47534" y="515.95685" id="text4333" sodipodi:linespacing="125%" transform="matrix(0.86672333,0.49794958,-0.49711357,0.86816921,0,0)"><TSPAN id="tspan4337" sodipodi:role="line" x="701.47534" y="515.95685" style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:98.05621338px;font-family:Calibri;-inkscape-font-specification:Calibri">Metabolic Burden</TSPAN></TEXT></G><G id="Integrase" inkscape:label="#g4452"><TEXT transform="matrix(0.87077486,0.49080614,-0.48998203,0.87222754,0,0)" sodipodi:linespacing="125%" id="text4475" y="752.57031" x="898.41321" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#ffffff;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" xml:space="preserve"><TSPAN style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:74.70949554px;font-family:Calibri;-inkscape-font-specification:'Calibri Bold';fill:#ffffff" y="752.57031" x="898.41321" id="tspan4477" sodipodi:role="line">λ-Integrase</TSPAN></TEXT></G><G id="attP" inkscape:label="#g4470"><TEXT transform="matrix(-0.00176162,-1.0008339,0.99916371,-0.00175869,0,0)" sodipodi:linespacing="125%" id="text4479" y="-37.975777" x="-553.19403" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#ffffff;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" xml:space="preserve"><TSPAN style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:74.70949554px;font-family:Calibri;-inkscape-font-specification:'Calibri Bold';fill:#ffffff" y="-37.975777" x="-553.19403" id="tspan4481" sodipodi:role="line">λ-attP</TSPAN></TEXT></G><G id="GFP" style="opacity:1" inkscape:label="#g4315"><TEXT xml:space="preserve" style="font-style:normal;font-weight:normal;font-size:46.69343567px;line-height:125%;font-family:sans-serif;letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1" x="24.391235" y="-40.994553" id="text4483" sodipodi:linespacing="125%" transform="matrix(-0.00176162,-1.0008338,0.9991638,-0.00175869,0,0)"><TSPAN sodipodi:role="line" id="tspan4485" x="24.391235" y="-40.994553" style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:74.70949554px;font-family:Calibri;-inkscape-font-specification:'Calibri Bold'">GFP</TSPAN></TEXT></G></G></G></G></SVG><P><B>Figure 1:</B> The entire genetic circuit of this year's project</P></DIV></CENTER><DIV class="verlinked" id="ortho"><H5>INCORPORATION OF A NON-NATURAL AMINO ACID</H5></DIV><DIV class="abstract"><P>ABSTRACT</P><P>In order to detect the presence of a specific non-natural amino acid (nnAA) <I>in vivo</I> the concept of <B><I>amber</I> suppression</B> is used. This means the occurrence of the <I>amber</I> stop codon (UAG) in an <I>open reading frame</I> does not cancel protein translation but codes for a specific nnAA, in our case <I>O</I>-methyl-<SPAN style="font-variant:small-caps">l</SPAN>-tyrosine (OMT). However, for the incorporation the nnAA needs to be present in the medium, otherwise the translation stops. The mechanism requires a <B>tRNA</B> with an anticodon complementary to the <I>amber</I> stop codon and an aminoacyl RNA synthetase (aaRS) loading the tRNA with the specific nnAA. The tRNA and aaRS combination is called an 'orthogonal pair'.</P><BUTTON class="read_more" id="lab1b"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/OrthogonalPair">Interested? Read more</A></BUTTON></DIV><DIV class="verlinked" id="repo"><H5>REPORTING LOW nnAA LEVELS</H5></DIV><DIV class="abstract"><P>ABSTRACT</P><P>Glow before you go - What does this actually mean? The aim of our project is to make biology safer by introducing a suicide system into <I>E. coli</I>. Before the suicide is triggered, a reporter protein is expressed to indicate the release of <I>E. coli</I> or to show a deficiency of the non-natural amino acid in the surrounding medium which is necessary for the bacteria to survive. As a reporter protein, we chose mVenus which is a mutant of eYFP. mVenus is located downstream of a promoter which is repressed by a dimeric protein, the <I>Zif23-GCN4</I> repressor. This repressor carries an <I>amber</I> mutation at position 4 (F4OMT). As a result, the non-natural amino acid <I>O</I>-methyl-<SPAN style="font-variant:small-caps">l</SPAN>-tyrosine (OMT) is integrated into the protein sequence as long as there is enough OMT in the medium. With decreasing OMT concentration, the translation of the repressor stops due to the early <I>amber</I> stop codon and the repressor cannot bind to the promoter. This leads to the expression of the reporter protein mVenus which can be detected by fluorescence measurements.</P><BUTTON class="read_more" id="lab2b"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/Reporter">Interested? Read more</A></BUTTON></DIV><DIV class="verlinked" id="kill"><H5>KILL(SWITCH)</H5></DIV><DIV class="abstract"><P>ABSTRACT</P><P>Synthetic suicide systems have been chosen safeguards in synthetic biology since the field exists. There are different kinds of designs, often based on a regulating mechanism and a host killing toxin or different kinds of inhibition of metabolic pathways. However, these mechanisms most often do not tackle the problem of synthetic DNA surviving the death of host cells.
Here we show a possible design for a simple synthetic killswitch based on the endonuclease <I>Colicin E2</I> and its corresponding suppressing protein <I>Im2</I>. Endonuclease and suppression protein expression is regulated by <I>amber</I> suppression. Therefore, an <I>amber</I> stop codon coding for the non-natural amino acid  <I>O</I>-methyl-<SPAN style="font-variant:small-caps">l</SPAN>-tyrosine is implemented into the gene. The aim of the system is not to simply kill the host, but also to degrade all DNA within the cell and its surroundings, preventing the escape of transgenic DNA.</P><BUTTON class="read_more" id="lab3b"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/KILLswitch">Interested? Read more</A></BUTTON></DIV><DIV class="verlinked" id="GI"><H5>METABOLIC BURDEN</H5></DIV><DIV class="abstract"><P>ABSTRACT</P><P>Artificial plasmids are a significant burden to the host. The design of pathways, e.g. the combination of different promoter and RBS systems, results in different amounts of product. Measurement of the metabolic burden is a key for quantitative optimization of metabolic engineering approaches. We want to establish a new approach to iGEM by providing a measurement strain to the community. As described by F. Ceroni <I>et al.</I>, we integrated one copy of the GFP gene into the genome of <I>E. coli</I>, which offers us a highly accurate and instantaneous measurement of the impact of our plasmids on the host. Metabolic burden measurement is of economical interest, because it enables academic and industrial research testing several different pathways at once in a short period of time by using microplate reader. For the integration, we used the λ‑Integrase site‑specific recombination pathway, described by A. Landy in 2015. Therefore, we designed two plasmids (<A href="http://parts.igem.org/Part:BBa_K1976000">BBa_K1976000</A> and <A href="http://parts.igem.org/Part:BBa_K1976001">BBa_K1976001</A>) and measured them using single cell measurement via microplate reader.

</P><BUTTON class="read_more" id="lab4b"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/MetabolicBurden">Interested? Read more</A></BUTTON></DIV><DIV class="verlinked" id="chem"><H5>CHEMICAL SYNTHESIS</H5></DIV><DIV class="abstract"><P>ABSTRACT</P><P>Since non‑natural amino acids are expensive in comparison to natural amino acids, we searched for a high yielding synthesis of <I>O</I>-methyl-<SPAN style="font-variant:small-caps">l</SPAN>-tyrosine. When chemically altering an amino acid to a non‑natural derivate the higher reactivity of the amino and carboxyl groups in comparison to the desired reactive group has to be considered. For this reason, amino and carboxyl groups need to be protected before carrying out the desired synthesis.
For the protection of the amino group an acetylation reaction was performed to form <I>N</I>‑acetyl‑L‑tyrosine, which was then methylated at the carboxyl group and the hydroxyl group using dimethyl sulfate in a Williamson ether synthesis. In order to finally form the non‑natural amino acid, an acidic hydrolysis using hydrochloric acid was performed.</P><BUTTON class="read_more" id="lab5b"><A href="https://2016.igem.org/Team:TU_Darmstadt/Lab/ChemicalSynthesis">Interested? Read more</A></BUTTON></DIV></DIV><DIV class="rechts"><DIV id="box_right" class="scrollbox"><DIV class="highlights"><A href="#ortho">Incorporation of a nnAA</A><A href="#repo">Reporter</A><A href="#kill">KILL(switch)</A><A href="#GI">Metabolic Burden</A><A href="#chem">Chemical Synthesis</A></DIV><BUTTON class="back_top_full"><A href="#mainHeader">Back to the Top</A></BUTTON></DIV></DIV><DIV class="footer" style="text-decoration: none;"><CENTER><DIV id="igemTUDA"><A href="https://2016.igem.org/Team:TU_Darmstadt" style="color:white">iGEM Technische Universität Darmstadt</A></DIV></CENTER><CENTER><DIV id="footer_linear">
We thank our <A href="https://2016.igem.org/Team:TU_Darmstadt/Team/Acknowledgements" style="color:#019ac8">sponsors</A>, especially:

Email: igem@bio.tu-darmstadt.de | 
Design: Viktoria Schuster</DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>