Team:Duke/Experiments
Toggle navigation
Home
Team
Team
Collaborations
Project
Description
Experiments
Results
Notebook
Parts
Safety
Attributions
Human Practices
Human Practices
Silver
Integrated Practices
Engagement
Our project consisted of two main sections, the cloning and the kinetic assays.
Cloning
gBlock Design & Cloning
Genes associated with the production of known Taxol biosynthesis enzymes were ordered as gBlock DNA sequences. These sequences were derived from the amino acid sequences of the enzyme strains used in the above studies. These deduced amino acid sequences were then used to create a DNA sequence using IDT’s codon optimization tool. At this point, the gene sequences are screened for restriction sites used later in the cloning experiment or for iGEM use: EcoRI, XbaI, SpeI, PstI and BsaI. When these sites are found, they are removed by designing synonymous mutations within the restriction domain to break the site. This was done in PAM, TycA and DBTNBT.
From this, the gBlock is designed for additional sequences for later modification and function. A yibDp phosphate-repressed bacterial promoter with ribosome binding site is attached upstream and two stop codons and an intrinsic terminator are included downstream to enable the promotion of each gene in bacteria. The promoter is a strong promoter and each part is created using the same promoter. Future designed may include weaker promoter for faster enzymes to better balance the flux of the system.
Each gBlock also has sequences to be used for the facilitation of the cloning project. The outermost edge of each gBlock includes homology, both 3’ and 5’, to the pSMART (Lucigen), which is the vector we used for characterizing the enzymes. The homology was used to put the each enzyme into pSMART via Gibson Assembly.  Adjacent to the pSMART homology is homology to the iGEM standard backbones in the form of the iGEM “prefix” and “suffix” sequences. This sequence is present in the pSB1C3 submission backbone as well as the entire iGEM plasmid backbone suite. These will be of use for the addition of the gBlocks to backbones for the production in E. coli.
XbaI sites are located at the end of the 5’ iGEM “prefix” and between the promoter and gene for excision. SpeI sites are located between the gene and terminator and at the beginning of the 3’ iGEM “suffix” to remove the terminator. This method works both while on pSMART and while on pSB1C3 backbone and prepares a plasmid with only the unexpressed gene.
Finally, the gBlocks have a series of BsaI sites in order to perform a Golden Gate Assembly of the many parts. A series of orthogonal overhangs were arranged for a one-step ligation of the many genes. The overhangs are arranged in to put the genes in the following order, each with including a promoter and terminator: BadA, PAM, TycA S563A, TAT, mTBT, DBAT, BAPT, DBTNBT. This reaction could occur using either pSMART or the iGEM backbones.
Kinetic Assays
Protocols
Preparation of Metabolic Cell Line
To begin the characterization of the enzymes, each recombinant plasmid was transformed into DLF-00286 cells (optimized for metabolic expression). This was accomplished using the transformation protocol used during cloning.
Phosphate Limited Expression
Recombinant genes for the taxol pathway enzymes were expressed using limited-phosphate media to active the phosphate induced promoter. In order to build up biomass for enzymatic analysis, growth of the cells in SM10++ media—a high density (10-15 grams of biomass per liter) growth media—was used. Starter cultures of cells expressing each enzyme were grown overnight at 37℃ and 220 rpm shaking and then transferred into 50 ml of SM10++ in a flat cell culture flasks, which were incubated until sufficient growth. Sufficient growth for harvest was set at an optical density (OD 600) of between 7 and 10. After centrifugation, the cells were then washed with FGM-No Phosphate media prior to resuspension. This was to ensure the removal of residual phosphate remained in the solution. Expression for our genes was inhibited by the presence of phosphate from the growth media. After final resuspension in FGM-No Phosphate, cultures were allowed to express at 37℃ and 220 rpm between 16 and 28 hours. Experimentally, the incubation time was positively correlated with greater protein expression.
Sonication
Once cells had been grown and the enzyme expressed, cells were stored at -80℃ until they were to be used in SDS-PAGE and Kinetic Assays. To begin the sonication process, cells were unfrozen and then mixed with 2-5 mL of Lysate buffer, a mixture of phosphate buffer and protease inhibitors. EDTA was not added to our lysate buffer because our enzymes would react with the EDTA. Once samples were prepared, they were sonicated in a sonicator. Samples were sonicated at max amplitude of the machine using a pulse system with an on cycle of 10 s and and off cycle of 30 s. Total sonication time was between 3 and 5 min. After sonication, the samples were spun at 10,000 rpm for 20 min at 4℃. This was to remove the cell debris from the sample. The supernatant was then placed into a 30 kDa filter and filtered. Each spin was for 2-5 min at max speed permitted by the filter. When half of the sample had been filtered, more lysate buffer was added and mixed to the sample. The samples were washed three times in this manner. Once the samples had been filtered, they were kept on ice until an SDS-PAGE and/or kinetic assays could be attempted.
SDS-PAGE Electrophoresis and Visualization
Electrophoresis using a 10% Polyacrylamide tris-glycine gels (Thermo Fisher) was run to identify the presence of recombinant proteins in the lysates prepared from expression induced E. coli cultures. Prior to loading the gel, a Bradford Protein Quantification Assay was used to quantify the total protein in a sample by comparing it to the bovine serum albumin (BSA) standards. Using the results of the Bradford assay, 20 ng of total protein diluted to 10 μl total volume was mixed with 10 μl 2x Laemmli Blue SDS Sample Buffer in order to denature the proteins. Samples were heated at 95℃ for 3-5 min to further denature proteins and then allowed to cool on ice. The Fisher gel box was assembled with 10% Polyacrylamide tris-glycine gel and filled with 1x Tris Glycine Running Buffer pH 8.0. In each well, 18 μl of each sample mix was loaded along with 10 μl of NEB 10-250 kDa Protein Ladder. The gel was allowed to run for 1 hour and 45 minutes at 180V. Gels were visualized by overnight staining in Coomassie Brilliant Blue. The gel was incubated in destain solution until the appearance of visible bands and the disappearance of background blue color. Bands on the gel were compared to the protein ladder.
Enzyme Kinetics Assays
Enzyme Kinetics Assays were conducted to characterize the rate of each reaction in order to determine efficacy and predict rate for final taxol synthesis. Reactions buffers and substrates were added to wells of a 96-well plate and were incubated to 37℃. The reactions were initiated by addition of sonicated lysate (with the desired enzyme). Using the spectrophotometer, the consumption of the reactants was measured by the disappearance of Ellman’s reagent which approximates the amount of coenzyme A (CoA) by absorbing at 412 nm in the presence of CoA.
Experiments
When characterizing the BadA cells were grown according to the protocols described above. The lysate was then sonicated and prepared for a kinetic assay. For the BadA kinetic assays, key ingredients to the assay included Coenzyme-A (CoA, adenosine-triphosphate (ATP), benzoic acid, and ellman’s reagent. The variable that was varied was the amount of benzoic acid. A 96-well plate was used in conjunction with a plate reader to measure the assay over time. The final concentration of ATP in well (unless otherwise indicated) was 50 mM; 250 mM for CoA. A total of 25 uL of lysate and a corresponding 25 uL of Ellman’s reagent was added to each well. The buffer used for this assay was a combination of 50 mM sodium phosphate buffer and 10 mM magnesium chloride. These buffers were used to balance the high pH of the Ellman’s reagent and to provide magnesium for the enzyme to function properly.
The BadA enzyme adds CoA to benzoate; thus, it was possible to use a reduction in absorbance of the Ellman’s reagent to indicate activity of the BadA enzyme. The reduction in absorbance of Ellman’s reagent would correspond to the Ellman’s reagent being unbound from the CoA indicating that the CoA had been used by BadA to attach to the benzoate.
Controls used for the experiment included an empty vector control (286 cells with just a pSMART backbone with no insert were grown and expressed) for every concentration of benzoic acid tried on the BadA lysate, and BadA lysates with missing ingredients (i.e. one with water added instead of CoA). Below is the diagram of our plate for this experiment. The varying shades of red in the first column indicate varying concentrations of benzoic acid. The hot pink in the second column corresponds to the empty vector control with a benzoic acid concentration of  0.5 mM. Orange corresponds to a lack of ATP, blue corresponds to lack of CoA, and green corresponds to a lack of benzoic acid in solution. The purple box represents BADA enzyme while the Green box represents EV.
