<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_XMU_Software_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:XMU_Software/Project">Page               </A></LI><LI><A href="/Talk:Team:XMU_Software/Project">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU_Software/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU_Software/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:XMU_Software/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:XMU Software/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="xmu_body"><DIV id="xmu_software"><DIV id="project"><DIV class="navigation"><DIV><UL id="jsddm"><LI><A href="https://2013.igem.org/Team:XMU_Software">Home</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Requirements">Requirements</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Outreach">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Notebook">Notebook</A></LI></UL></DIV></DIV><DIV id="project_s1"><DIV class="project_title">PROJECT</DIV><DIV class="project_introduce">Our project includes 2 independent software tools-the brick worker and E' NOTE. The former is a software suit for the evaluation and optimization of biobricks, i.e., promoter, RBS, protein coding sequences and terminator. E' NOTE is a web application serving as an assistant for experiments. Its useful functions such as experiments recording and experimental template customization make experimental process easier and more enjoyable.</DIV></DIV><DIV class="navi_text"><A id="project_navi0" class="navi0" href="#_promoter" onclick="project_promoter();" onmouseover="navi0();" onmouseout="navi01();" style="color:white">Promoter-decoder</A><A id="project_navi1" class="navi1" href="#_rbs" onclick="project_rbs();" onmouseover="navi1();" onmouseout="navi11();" style="color:white">RBS-decoder</A><A id="project_navi2" class="navi2" href="#_terminator" onclick="project_terminator();" onmouseover="navi2();" onmouseout="navi21();" style="color:white">Terminator</A><A id="project_navi3" class="navi3" href="#_protein" onclick="project_protein();" onmouseover="navi3();" onmouseout="navi31();" style="color:white">SynoProteiner</A><A id="project_navi4" class="navi4" href="#_enote" onclick="project_enote();" onmouseover="navi4();" onmouseout="navi41();" style="color:white">E' NOTE</A><A id="project_navi5" class="navi5" href="#_achieve" onclick="project_achieve();" onmouseover="navi5();" onmouseout="navi51();" style="color:white">Achievement</A><A id="project_navi6" class="navi6" href="javascript:void(0)" onmouseover="navi6();" onmouseout="navi61();" style="color:white">Back To Top</A></DIV><DIV id="promoter"><DIV id="promoter_m1"><DIV class="sep_title">Promoter-decoder</DIV></DIV><DIV id="promoter_m2"><H1>Abstract</H1><P>In a promoter sequences, the sigma factor binding site and other transcription factor binding site affect the strength of binding significantly.
							For annotating promoters, some software was developed which mostly focused on the prediction of other transcription factors or one particular type of sigma factors but failed to analyze the promoter with both sigma factors and other transcription factors. <SUP>1-2</SUP>
							To solute this problem, a module of our software was designed which can analyze and evaluate promoters.</P><P>Our software use PWM method to calculate the similarity between promoter sequences and the position frequency matrix of transcription factor binding sites (TFBS) to locate the TFBS as well as to predict the relative strength of the promoter.							
							Promoter-Decoder overshadows its counterparts with all-round analysis and the prediction of promoter strength.
							It enables users to figure out promoter types, predict promoter strength, changeit by mutating the key sites and even change the property of certain promoter by adding new TFBS to the promoter sequences. </P><H1>Background</H1><H4>Sigma Factors</H4><P>Bacteria encode several thousands of different proteins, which are necessary for normal cell functions or for adaptation to environmental changes.<SUP>3</SUP> These proteins are not required at the same time or in the same amount. Regulation of gene expression therefore enables the cell to control the production of proteins needed for its life cycle or for adaptation to extracellular changes. The various steps during transcription and translation are therefore subject to different regulatory mechanisms.<SUP>4</SUP></P><P>The most prominent step in gene regulation is the initiation of transcription in which the DNA-dependent RNA polymerase (RNAP) is the key enzyme. The RNAP or the RNAP core enzyme is the catalytic machinery for the synthesis of RNA from a DNA template. However, RNAP cannot initiate transcription by itself. Initiation of transcription requires an additional polypeptide known as a sigma factor.<SUP>5</SUP> sigma factors are a family of relatively small proteins that can associate in a reversible way with the RNAP core enzyme. Together, the sigma factor and the RNAP core enzyme form an initiation-specific enzyme, the RNAP holoenzyme.</P><DIV class="body_photo">Figure 1 The initiation of transcription</DIV><P>The sigma factor directs RNA polymerase to a specific class of promoter sequences. Most bacterial species synthesize several different sigma factors that recognize different consensus sequences.<SUP>6</SUP></P><P> This variety in sigma factors provides bacteria with the opportunity to maintain basal gene expression as well as for regulation of gene expression in response to altered environmental or developmental signals.</P><P>The frequency at which the RNAP holoenzyme initiates transcription, also known as the strength of a promoter, is influenced by the promoter sequences and the conformation of the DNA in the promoter region. The sigma factors recognize two conserved sequences in the promoter region, known as the promoter consensus sequences. sigma factors or fragments of sigma factors bind specifically to promoter DNA sequences and by specific base pair and amino acid substitutions in the promoter consensus sequences or sigma factors. Most bacterial species synthesize several different sigma factors which direct the RNAP holoenzyme to distinct classes of promoters with a different consensus sequences. This variety in sigma factors provides the bacterium with the opportunity to maintain basal gene expression as well as for regulation of gene expression in response to specific environmental stimuli.</P><P>The identification of bacterial promoters is an essential step in the elucidation of gene regulation.<SUP>7</SUP></P><P>As a general rule, the more complex the life-cycle and environmental niche of a bacterium, the greater the number of sigma factors with corresponding promoter types. Typically however, the most common promoter type is that which regulates the housekeeping genes and the corresponding major sigma factor is shared by all bacteria (sigma 70 in the well studied <I>E. coli</I>, and its homologues in other species). The binding site for the sigma70-family of promoters is defined by two consensus hexamers, TTGACA and TATAAT, located at approximately −35 and −10, respectively relative to the transcript start site (TSS) and spaced 15–21 base pairs (bp) apart2. RNA polymerase core enzyme associates with the major sigma factor to form the holoenzyme which in turn binds to its cognate promoters to initiate transcription.</P><DIV class="body_photo">Figure 2 The RNA polymerase Figure 3 Consensus sequences of sigma 70 factor</DIV><P>In prokaryotes, the minimum requirement for RNA polymerase binding is recognition of the promoter by the sigma factor. In general, prokaryotic RNA polymerases can interchange a number of sigma factors which bind and initiate different groups of genes.<SUP>3</SUP></P><H4>Transcription Factors</H4><DIV class="body_photo">Figure 4 Transcription factor binding site</DIV><P>Sigma factors are essential for the transcription initiation in <I>E. coli</I>.<SUP>10</SUP></P><P> In addition, promoter strengths are not determined purely by the binding of the sigma factor. Other transcription factors can bind specific sequences surrounding or overlapping the promoter to either activate or repress transcription.<SUP>4</SUP> The mechanism is transcriptional activators and repressors contribute to and detract from the accessibility of DNA by the RNA polymerase. <SUP>12</SUP></P><P>These transcription-regulating nuclear proteins bind to specific binding sites in the regulatory regions (e.g. promoters, enhancers) of the genes thus providing their activation or repression. </P><DIV class="body_photo">Figure 5 Transcription factor binding site</DIV><P>Computational methods of predicting TF binding sites in DNA are very important for understanding the molecular mechanisms of gene regulation.</P><P>The binding sites of the same transcription factor show a significant sequences conservation, which is often summarized as a short (5–20 bases long) common pattern called a transcription factor binding site (TFBS) or binding consensus. Our software aims to figure out the possible TFBS in promoters and precisely locate the TFBS so that the user may know the exact sites that play a role in regulating the transcription.</P><P>In prokaryotes (lower organisms without nuclei), there are fewer TFs, their motifs tend to be relatively long and the strength of regulation for a particular gene often depends on how closely a particular site matches the consensus for the motif. The more mismatches to the consensus in a binding site, the less often the TF will bind and therefore the less control it will exert on the target gene. So our software will calculate the similarity between the possible TFBS in the promoter and the standard motifs so the user will know to which extent the transcription factor will control the promoter transcription</P><H4>Primer Design</H4><P>To facilitate the design of PCR primers of various promoters, we've developed an additional function, namely, primer design in this part of our program. After inputting the promoter sequences, the software will figure out the most suitable primers based on the theory of Thomas Kämpke<SUP>1</SUP>, Markus Kieninger, and Michael Mecklenburg.<SUP>13</SUP></P><H1>Data Source</H1><H4>RegulonDB</H4><P>Genes and operons that are under control of the same TF are members of that TF's regulon.  Although methods for the prediction of regulons have been substantially improved, they are still far from perfect. </P><P>Comparative genomics tools can be used to predict regulons in bacterial genomes but the procedure can lead to incorrect regulon calling. Despite this drawback, several regulon databases are available that are based on comparative genomics methods and lack experimental evidence.</P><P>Probably the extended and accurate databases of regulons for <I>E. coli</I> are RegulonDB which provides the data source for our program.</P><H1>Algorithm</H1><P>Experimental results show that these are the strongest promoters that have been characterized in vitro so far and confirm the hypothesis that the consensus promoter sequences is &quot;best&quot;. To calculate the similarity between the promoter sequences and the best sequences, we implement the PWM method<SUP>6</SUP> in our program.</P><H4>PWM (Position Weight Matrix)</H4><P>Molecular techniques for the identification of promoters are both costly and time consuming, hence in silico methods are an attractive and well explored alternative. The most common in silico method to identify sigma 70 promoters uses position weight matrices (PWMs) and depends on the relative conservation of the transcription factor binding site (TFBS, or motifs ). </P><P>The algorithm can be divided into two parts regarding to the difference between the motifs of sigma factors and other transcription factors.  </P><DIV class="body_photo">Figure 6 The consensus sequences, the position frequency matrix and the frequency logo</DIV><H4>Part 1: the recognition of other transcription factors.<SUP>7</SUP></H4><P>Other transcription factors are proteins that can bind to a specific DNA sequences (motifs) and regulate the promoter's transcription. To recognize these possible motifs in a given promoter sequences, we calculate the Matrix Similarity Score (MSS) of every possible sites in the promoter sequences using the position frequency matrix of 86 transcription factors published by RegulonDB. The algorithm reports only those matches of a matrix that have got MSS higher than the settled threshold. And MSS for a subsequences <I>x</I> of the length <I>L</I> is calculated in following steps:</P><P><I>f<SUB>i,Bi</SUB></I> , frequency of nucleotide <I>B</I> to occur at the position <I>i</I> of the matrix (<I>B</I> ∈{A, T, G, C})</P><P><I>f <SUB>i</SUB><SUP>min</SUP></I> , frequency of the nucleotide which is rarest in position <I>i</I> in the matrix</P><P><I>f <SUB>i</SUB><SUP>max</SUP></I> , highest frequency in position <I>i</I>.</P><P>The information vector</P><P>describes the conservation of the positions <I>i</I> in a matrix. Multiplication of the frequencies with the information vector leads to a higher acceptance of mismatches in less conserved regions, whereas mismatches in highly conserved regions are very much discouraged. This leads to a better performance in recognition of TF binding sites if compared with methods that do not use the information vector.</P><P>To determine the best threshold of the motif finding algorithm, we test various threshold values and analyze the true negative and false positive rate of each threshold value. The ideal threshold is supposed to have both the least true negative and false positive rates.</P><DIV><TABLE width="600" border="0" cellspacing="0" class="promoter_table1"><CAPTION>
									Table 1 The threshold setting data
								  </CAPTION><TBODY><TR border="1px"><TD align="center" valign="middle"><STRONG>Threshold</STRONG></TD><TD align="center" valign="middle" bgcolor="#FEA801">0.5977</TD><TD align="center" valign="middle">0.598</TD><TD align="center" valign="middle">0.69</TD><TD align="center" valign="middle">0.7</TD><TD align="center" valign="middle">0.73</TD><TD align="center" valign="middle">0.76</TD><TD align="center" valign="middle" bgcolor="#FEA801">0.0778</TD><TD align="center" valign="middle">0.84</TD><TD align="center" valign="middle" bgcolor="#FEA801">0.85</TD><TD align="center" valign="middle">0.86</TD><TD align="center" valign="middle">0.9</TD></TR><TR><TD align="center" valign="middle" nowrap="nowrap"><STRONG>True megtive</STRONG></TD><TD align="center" valign="middle" bgcolor="#FEA801">0.1</TD><TD align="center" valign="middle">0.11</TD><TD align="center" valign="middle">0.21</TD><TD align="center" valign="middle">0.23</TD><TD align="center" valign="middle">0.3</TD><TD align="center" valign="middle">0.45</TD><TD align="center" valign="middle" bgcolor="#FEA801">0.5</TD><TD align="center" valign="middle">0.63</TD><TD align="center" valign="middle" bgcolor="#FEA801">0.7</TD><TD align="center" valign="middle">0.72</TD><TD align="center" valign="middle">0.77</TD></TR><TR><TD align="center" valign="middle" nowrap="nowrap"><STRONG>False positive</STRONG></TD><TD align="center" valign="middle" bgcolor="#FEA801">56.4778</TD><TD align="center" valign="middle">57.01124</TD><TD align="center" valign="middle">31.56962</TD><TD align="center" valign="middle">29.15584</TD><TD align="center" valign="middle">23.58571</TD><TD align="center" valign="middle">20.92727</TD><TD align="center" valign="middle" bgcolor="#FEA801">17.90796</TD><TD align="center" valign="middle">9.945946</TD><TD align="center" valign="middle" bgcolor="#FEA801">10.66667</TD><TD align="center" valign="middle">10.17857</TD><TD align="center" valign="middle">6.608696</TD></TR></TBODY></TABLE></DIV><P>The picture above shows part of our test results and to keep both the true negative and false positive rates at a reasonable level, we adopt 3 threshold values, namely, low (0.5977) , median (0.0778) and high (0.85), with a true negative rate at 0.1, 0.5, and 0.7 respectively. And for more flexibility, we also allow the users to set their own thresholds.</P><H4>Part 2: the recognition of sigma factor motif and the evaluation of relative promoter strength.</H4><P>In the case of sigma 70 factors, the motifs are the −35 and −10 hexamers. Enclosing a spacer of length 15–19 bp.</P><P>Given a promoter sequence, the -10 and -35 hexamers are located by the total MSS of the two hexamers calculated by the position frequency matrices of the sigma factor binding sites, which are derived from Regulon DB. And the calculating process is subject to two constraints:</P><P>1. That the spacer length (the number of base pairs between the −35 hexamer and the −10 hexamer) should lie in the range (15–20);</P><DIV class="body_photo">Figure 7 The consensus sequences of sigma 70 factor binding site</DIV><P>2. The total MSS  (our results are the sum of the scores for the −10 and −35 hexamers and therefore lie in the interval [0,2], with a score of 2 corresponding to the joint consensus TTGACA (−35) and TATAAT (−10).</P><P>Score(Promoter)=score(-10 box)+score(-35 box)+score(spacer between -10 &amp; -35 boxes)</P><P>And the score of spacer length is calculated by algorithms propozed by Ryan K. Shultzaberger.el. in <ANATOMY of="">E. coli sigma70 promoters.<SUP>8</SUP> But due to a lack of experimental data of promoter strength with both different motifs and spacer length, the weight of the total MSS and the spacer score is very roughly determined with few experimental data available. Currently our weight is determined with the promoter strength data in a literature<SUP>16</SUP> to best fit the the promoter score with promoter strength. Now the relative weight between the total MSS of the two motifs and the spacer score is 0.29:0.71.</ANATOMY></P><P>In prokaryotes, the strength of sigma factor regulation for a particular gene often depends on how closely a particular site matches the consensus for the motif. The more mismatches to the consensus in a binding site, the less often the sigma factor will bind and therefore the less strength the promoter will have. Experiments have confirmed the hypothesis that the consensus promoter sequences is &quot;best&quot;. We set the best promoter strength to 100% and calculate the relative strength of a given promoter by the Score (promoter). </P><H4>Primer design</H4><P> A primer pair (<I>p</I>, <I>q</I>) is assigned the scoring vector</P><P><I>sc</I> (<I>p</I>, <I>q</I>) = (|<I>p</I>|, |<I>q</I>|, <I>GC</I> (<I>p</I>), <I>GC</I> (<I>q</I>), <I>Tm</I> (<I>p</I>), <I>Tm</I> (<I>q</I>), <I>sa</I> (<I>p</I>), <I>sa</I> (<I>q</I>), <I>sea</I> (<I>p</I>), <I>sea</I> (<I>q</I>), <I>pa</I> (<I>p</I>, <I>q</I>), <I>pea</I> (<I>p</I>, <I>q</I>))<SUP>T</SUP> ∈ <I>R</I><SUP><I>12</I></SUP>.</P><P>All primers are designed to have ideal values of length, GC content, and melting temperature which are specified externally by the designer of the hybridization experiment. These ideal values are to be specified for forward and reverse primers. The ideal score vector or reference vector for the primer pair is </P><P><I>sc<SUB>ideal</SUB></I> =(<I>length<SUB>f</SUB></I> ,  <I>length<SUB>r</SUB></I> ,  <I>GC<SUB>f</SUB></I> ,  <I>GC<SUB>r</SUB></I> ,  <I>T<SUB>m,f</SUB></I> ,  <I>T<SUB>m,r</SUB></I> , 0, 0, 0, 0, 0, 0)<SUP>T</SUP>. </P><P>All ideal annealing values are set to zero and typically</P><P><I>T<SUB>mf</SUB></I> = <I>T<SUB>m,r</SUB></I> as well as <I>GC<SUB>f</SUB></I> = <I>GC<SUB>r</SUB></I> . The final assessment of a primer pair (<I>p</I>, <I>q</I>) can be its deviation from the reference in terms of the <I>l1</I>-distance</P><P>Here, we employ a weighted distance</P><P>with weights given in the following table.</P><P>The formulas for calculating the variations above are provided in Efficient primer design algorithms.<SUP>13</SUP></P><H1>Results</H1><H4>Sigma factors recognition </H4><P> Our program has a correctness rate of 56% in recognition various types of sigma factors. We ran our program with 100 various promoters sequences whose types have already been confirmed experimentally. And we've recognized 56 of them correctly. Specifically, as for sigma 70 promoters, which are the most prevalent, the recogtion correctness rate has reached 92%. The results are showed below.</P><P><A href="https://2013.igem.org/Team:XMU_Software/result1" target="_blank">Link to the page of results</A></P><H4>TFBS Location</H4><P>We then tested the reliability of our software regarding TFBS location and results show that the correct site prediction rate is 64%. We used the sigma70 promoter sequences with annotated -35 and -10 region provided by RegulonDB to test the correct prediction rate of the binding site of a specific transcription. We input 89 sigma70 promoter sequences and ran our program to precisely locate the sigma factor binding site. </P><P>The test results are as follows. The numbers represent the site of actual-35 motif, the actual spacer length, the predicted site and predicted spacer length respectively.</P><P><A href="https://2013.igem.org/Team:XMU_Software/promoter" target="_blank">Link to the page of results</A></P><H4>Promoter strength correlation &amp; experiments</H4><P>To testify our prediction of promoter strength, our team has done a considerable amount of lab work. First, we located the -10 region of the pBAD promoter (BBa _K206000) and accordingly mutated the -10 region into BBa _K1070002, BBa _K1070003, the sequences of these promoters are given bellow (-10 regions are highlighted):</P><P>pBAD Strong (BBa _K206000)：a c a t t g a t t a t t t g c a c g g c g t c a c a c t t t g c t a t g c c a t a g c a a <SPAN class="light">g a t a g t</SPAN> c c a t a a g a t t a g c g g a t c c t a c c t g a c g c t t t t t a t c g c a a c t c t c t a c t g t t t c t c c a t a c c g t t t t t t t g g g c t a g c</P><P>BBa _K1070002：a c a t t g a t t a t t t g c a c g g c g t c a c a c t t t g c t a t g c c a t a g c a a <SPAN class="light">t a t a g t </SPAN>c c a t a a g a t t a g c g g a t c c t a c c t g a c g c t t t t t a t c g c a a c t c t c t a c t g t t t c t c c a t a c c g t t t t t t t g g g c t a g c</P><P>BBa _K1070003：a c a t t g a t t a t t t g c a c g g c g t c a c a c t t t g c t a t g c c a t a g c a a <SPAN class="light">g a t a a t</SPAN> c c a t a a g a t t a g c g g a t c c t a c c t g a c g c t t t t t a t c g c a a c t c t c t a c t g t t t c t c c a t a c c g t t t t t t t g g g c t a g c
							</P><P>Subsequently, we measured the fluorescence intensity of these promoters and related it to the actual promoter strength. The experimental results are shown in Figure 8.</P><DIV class="body_photo">Figure 8 The fluorescence intensity reached a stable stage after 60 min. The fluorescence intensity is measured with the inducer, L-arabinose at 1 mM, 37℃, 200 rpm
. The promoter strength is related to the relative fluorescence intensity compared to the control group (K206000 without the inducing of L-arabinose).</DIV><P>Then we fitted the actual strength and the predicted strength, as can be seen in figure 9, the coefficient of determination is 0.8924.</P><DIV class="body_photo">Figure 9 The correlation between experimental data and the strength predicted by our program.</DIV><H1>Future work</H1><P>Apply our algorithms to more species. Now Pro-decoder is designed expressly for prediction and evaluation of <I>E. coli</I> promoters, in the near future we'll study the transcription regulation mechanism of other species and try to apply our algorithms to an extended range of species.</P><P>Enhance promoter strength prediction accuracy. Because our experimental data is so limited, the weight of the spacer length and the motif similarity is roughly determined, which lead to a weak correlation between the promoter strength. In the future we hope to obtain more experimental data with regard to the effect of spacer length and motif similarity having on promoter strength so we can revise the weight coefficients  of the two factors and get more reliable results.</P><P>The next version of this part of our program will be able to analyze not only the promoters of <I>E. coli</I>, but other species such as <I>Bacillus subtilis</I>, we'll integrate the transcription factor binding site data of more species into our database and use PWM algorithm to predict the TFBS in the promoters.</P><H1>References</H1><DIV class="body_references">[1] Wösten, M., Eubacterial sigma‐factors. <I>FEMS microbiology reviews</I><B>1998</B>,<I> 22</I> (3), 127-150.</DIV><DIV class="body_references">[2] Shultzaberger, R. K.; Chen, Z.; Lewis, K. A.; Schneider, T. D., Anatomy of <I>E. coli</I> σ70 promoters. <I>Nucleic acids research</I><B>2007</B>, <I>35</I>  (3),771-788.</DIV><DIV class="body_references">[3] Paget, M.; Helmann, J. D., The sigma70 family of sigma factors.<I> Genome Biol</I><B>2003</B>,<I> 4 </I> (1),203.</DIV><DIV class="body_references">[4] Jensen, S. T.; Liu, X. S.; Zhou, Q.; Liu, J. S., Computational discovery of gene regulatory binding motifs: a Bayesian perspective. <I>Statistical Science</I><B>2004</B>, <I>19 </I>(1), 188-204.</DIV><DIV class="body_references">[5] Kämpke, T.; Kieninger, M.; Mecklenburg, M., Efficient primer design algorithms.<I> Bioinformatics</I><B>2001</B>,<I> 17 </I> (3),214-225.</DIV><DIV class="body_references">[6] (a) Rhodius, V. A.; Mutalik, V. K., Predicting strength and function for promoters of the <I>E. coli</I> alternative sigma factor, σE. <I> Proceedings of the National Academy of Sciences </I><B>2010</B>,  <I>107 </I> (7), 2854-2859; (b) Mulligan, M. E.; Brosius, J.; McClure, W. R., Characterization in vitro of the effect of spacer length on the activity of <I>E. coli</I> RNA polymerase at the TAC promoter.  <I>Journal of Biological Chemistry </I><B>1985</B>,  <I>260 </I> (6), 3529-3538; (c) Qureshi, S. A.; Jackson, S. P., sequences-Specific DNA Binding by the S. shibatae TFIIB Homolog, TFB, and Its Effect on Promoter Strength. <I>Molecular cell</I><B>1998</B>,<I> 1</I> (3), 389-400.</DIV><DIV class="body_references">[7] Kel, A. E.; Gößling, E.; Reuter, I.; Cheremushkin, E.; Kel-Margoulis, O. V.; Wingender, E., MATCHTM: a tool for searching transcription factor binding sites in DNA sequences. <I>Nucleic acids research</I><B>2003</B>, <I>31</I> (13), 3576-3579.</DIV><DIV class="body_references">[8] Deuschle, U.; Kammerer, W.; Gentz, R.; Bujard, H., Promoters of <I>E. coli</I>: a hierarchy of in vivo strength indicates alternate structures. <I>The EMBO journal</I><B> 1986</B>,<I> 5 </I>(11), 2987.</DIV></DIV></DIV><DIV id="RBS"><DIV id="RBS_m1"><DIV class="sep_title">RBS-decoder</DIV></DIV><DIV id="RBS_m2"><H1>Abstract</H1><P>The efficiency of translation in bacteria is greatly influenced by the binding affinity between the ribosome and RBS, which can be measured by RBS strength. Experimental work to determine a RBS sequences can be awfully laborious while our software program can solve this problem easily. RBS-decoder is a software tool for evaluating RBS strength and locating SD sequences. This program uses the same method-PWM to calculate the similarity between the RBS sequences and the position frequency matrix of SD sequences and transform the similarity to the relative strength of a RBS sequences.</P><H1>Background</H1><P>Translational efficiency in <I>Escherichia coli</I> is generally determined at the stage of initiation. There are several principal mRNA sequences elements which can affect the kinetics of ternary initiation complex formation (30 S-mRNA-fMet-tRNA): SD sequences and the start codon (ATG). The SD sequences base-pairs with a RNA molecule that forms part of the bacterial ribosome (the 16s rRNA), while the start codon base-pairs with the initiator tRNA which is bound to the ribosome. In addition the SD sequences and the start codon being important, the spacer between them also influences the RBS strength, these two sequences need to be positioned approximately 6-7 nucleotides apart so they can both make contact with the appropriate parts of the ribosome complex<SUP>1</SUP>.</P><H1>Introduction</H1><P>How do baterial Ribosome Binding Sites work?</P><P>The bacterial ribosome binds to particular sequences on an mRNA, primarily the SD sequences and the start codon (ATG). The SD sequences base-pairs with an RNA molecule that forms part of the bacterial ribosome (the 16s rRNA), while the start codon base-pairs with the initiator tRNA which is bound to the ribosome. In addition to SD sequences and the start codon being important, these two sequences need to be positioned approximately 6-7 nucleotides apart so they can both make contact with the appropriate parts of the ribosome complex.<SUP>1</SUP></P><H4>The Shine-Dalgarno sequences</H4><DIV class="body_photo">Figure 1 The RBS sequences logo representing the sequences of 149 RBS from <I>E. coli</I>. The height of each letter represents the frequency of the base at that location. From Tom Schneider, &quot;A Gallery of sequences Logos&quot;.</DIV><P>The end of the 16s rRNA that is free to bind with the mRNA includes the sequences 5′–ACCUCC–3′. The complementary sequences, 5′–GGAGGU–3′, named the Shine-Dalgarno sequences, can be found in whole or in part in many bacterial mRNA. Very roughly speaking, ribosome binding sites with purine-rich sequences (A's and G's close to the Shine-Dalgarno sequences will lead to high rates of translation initiation whereas sequences that are very different from the Shine-Dalgarno sequences will lead to low or negligible translation rates. You can see how common the sequences is by looking at the RBS sequences logo on the right (where the height of a letter indicates the frequency of the letter at that location).</P><H1>Algorithms</H1><P>As we know, the RBS strength is greatly influenced by the SD sequences, where the 16s RNA of the ribosome binds to, so the strength can be determined by the binding free energy between the SD sequences and the 16s RNA. So we designed a program calculating the binding free energy but the results show that the correlation between the free energy and the strength of RBS is rather weak (R<SUP>2</SUP>=0.5517). So we decide to find other algorithms for better accuracy.</P><P>Inspired by the strength prediction algorithms used in promoter part, in which the similarity to the sigma factors' PWM is interlocking with the binding affinity between the protein and DNA sequences. We obtained the Position Frequency Matrix of SD sequences of <I>E. coli</I> and use the PMW method (illustrated in detail in the promoter part) to calculate the similarity between the RBS sequences and the Position frequency sequences, what is different from the promoter is that, the spacer length between the SD sequences and the startcodon and the start codon itself both act as constraints in locating the SD sequences, which is confined to 3-16 bp and ATG/TTG/GTG. And similar to the prediction of promoter strength, the spacer length between the SD sequence also contributes to the RBS strength, the optimal spacer length is 7 bp, and the spacer score is calculated using the same algorithm applied in the promoter part.<SUP>2</SUP> The weight of the influence of the spacer on the strength isderived from the algorithm to predict the promoter strength, in which the weight of the total MSS and the spacer is 0.29:0.71, and since in promoter the total MSS is the sum of two motifs while the SD sequences is only one motif, the weight between the MSS(SD sequences) and the spacer is 0.29:0.355.</P><TABLE border="1" cellspacing="0" class="RBS_table1"><CAPTION><STRONG>Nucleotide frequencies for the RBS model</STRONG></CAPTION><TBODY><TR><TD>1</TD><TD>2</TD><TD>3</TD><TD>4</TD><TD>5</TD></TR><TR><TD>T</TD><TD>0.161</TD><TD>0.050</TD><TD>0.012</TD><TD>0.071</TD><TD>0.115</TD></TR><TR><TD>C</TD><TD>0.077</TD><TD>0.037</TD><TD>0.012</TD><TD>0.025</TD><TD>0.046</TD></TR><TR><TD>A</TD><TD>0.681</TD><TD>0.105</TD><TD>0.105</TD><TD>0.861</TD><TD>0.164</TD></TR><TR><TD>G</TD><TD>0.077</TD><TD>0.808</TD><TD>0.960</TD><TD>0.043</TD><TD>0.659</TD></TR></TBODY></TABLE><DIV class="body_photo">Figure 2 The RBS nucleotide position frequency matrix.<SUP>3</SUP></DIV><H1>Results</H1><P>We used the RBS sequences listed on the iGEM registry with experimentally determined relative strength,<SUP>4</SUP> and the correlation between the RBS strength predicted by our software and the actual relative strength is strong, with a determination coefficient value 0.8039.</P><DIV class="body_photo">Figure 3 The correlation between actual RBS strength and predicted strength</DIV><H1>Future work</H1><P>Due to scarcity of experimental data, the relative weight of the SD sequences and the spacer length used currently is roughly determined which may undermine the accuracy of RBS strength prediction. For further improvement of our program, we'll try to obtain more reliable experimental data to accurately determine the weight used in our algorithm and hopefully elevate the accuracy of RBS-decoder.</P><P>In the next version of RBS-decoder, the secondary structure of the RBS sequences will be shown on the software and we'll also include the other species' SD sequence data in order to predict the RBS strength of a larger range of species.</P><H1>Reference</H1><DIV class="body_references">[1] Ma, J.; Campbell, A.; Karlin, S., Correlations between Shine-Dalgarno sequences and gene features such as predicted expression levels and operon structures. <I>Journal of bacteriology</I><B> 2002</B>,<I> 184 </I>(20), 5733-5745.</DIV><DIV class="body_references">[2] Noguchi, H.; Taniguchi, T.; Itoh, T., MetaGeneAnnotator: detecting species-specific patterns of ribosomal binding site for precise gene prediction in anonymous prokaryotic and phage genomes.<I> DNA research </I><B>2008</B>,<I> 15</I> (6), 387-396.</DIV><DIV class="body_references">[3] Alexander V. Lukashin, Mark B, GeneMark.hmm: new solutions for gene finding, Nucleic Acids Research, <B>1998</B>, 1107–11153.</DIV><DIV class="body_references">[4] http://parts.igem.org/Ribosome_Binding_Sites/Prokaryotic/Constitutive/Community_Collection.</DIV></DIV></DIV><DIV id="terminator"><DIV id="terminator_m1"><DIV class="sep_title">Terminator</DIV></DIV><DIV id="terminator_m2"><H1>Background</H1><P>Gene  expression  in  both  prokaryotes  and  eukaryotes  is  frequently  controlled  at  the  level  of  transcription.  This  process  can  be represented  as a cycle  consisting  of  four  major  steps:  (1)  promoter binding;  (2)  RNA  chain  initiation;  (3)  RNA  chain elongation;  and  (4) termination. Since regulatory controls are exerted at each step, an understanding of the mechanism of each step is of general importance in understanding gene expression.</P><P>In the promoter part of our program, we've discussed the mechanism of promoter binding step and how it affects the transcription level.  To complete our biobrick evaluation program and to better understanding of transcription process, we integrated the software developed by 2012 iGEM team SUSTC-Shenzhen-B to realize the prediction of transcription termination efficiency.</P><H1>Introduction</H1><P>Termination, the last step of the transcription cycle, occurs when the RNA polymerase releases the RNA transcript and dissociates from the DNA template. It is important that transcription is imperfectly terminated at some terminator so that the ratio of the amount of the mRNA transcribed from upstream and that from downstream of the terminator is controlled. This regulation is quantified by the termination efficiency (%T).</P><P>Two mechanisms of transcription termination and two classes of termination signals have been described in bacteria: rho-dependent and rho-independent. </P><P>Rho-independent (also known as intrinsic) terminators are sequences motifs found in many prokaryotes that cause the transcription of DNA to RNA to stop. These termination signals typically consist of a short, often GC-rich hairpin followed by a sequences enriched in thymine residues.</P><P>The conventional model of transcriptional termination is that the stem loop causes RNA polymerase to pause and transcription of the poly-A tail causes the RNA: DNA duplex to unwind and dissociate from RNA polymerase.</P><H1>Algorithm</H1><P>In 2011, iGEM team SUSTC-Shenzhen-B developed a software tool TTEC to predict terminator efficiency. It takes DNA sequences as input and returns the terminator efficiency value.</P><P>In the algorithm, it takes 3 steps to calculate the terminator efficiency:</P><P>1. Use RNA folding algorithm to predict the secondary structure of terminator and and recognize A tail, stemloop and T tail.</P><P>2. From the secondary structure, we calculate the free energy of stem loop, and generate a score by considering stem loop free energy and T tail</P><P>3. From the score, we predict the terminator efficiency based on the score-terminator equation.</P><P>The prediction of secondary and recognition of A tail, stemloop and T tail are achieved by Kingsford scoring system. </P><H1>Kingsford Scoring System</H1><P>In 2007, Carleton L. Kingsford et al. described TransTermHP1, a new computational method to rapidly and accurately detect Rho-independent transcription terminators. </P><P>They put forward an algorithm to predict Rho-independent terminators. The first 15 bases of the potential tail sequences are scored using a function:</P><P>where</P><P>for <I>n</I>=1,2,..,15 and =1.</P><P>The energy of potential hairpin configurations adjacent to a reference position can be found efficiently with a dynamic programming algorithm. The table entry hairpin_score[i,j] gives the cost of the best hairpin structure for which the base of the 5' stem is at nucleotide position i and the base of the 3' stem is at position j. The entry hairpin_score[i,j] can be computed recursively as follows:</P><P>The function <I>energy</I>(<I>i</I>,<I>j</I>) gives the cost of pairing the nucleotide at <I>i</I> with that at <I>j</I>, and <I>loop_pen</I>(<I>n</I>) gives the cost of a hairpin loop of length n. The hairpin's loop is forced to have a length between 3 and 13 nt, inclusive, by setting <I>loop_pen</I>(<I>n</I>) to a large constant for any <I>n</I> outside that range. The constant 'gap' gives the cost of not pairing a base with some base on the opposite stem and thus introducing a gap on one side of the hairpin stem.</P><P>Table 1</P><P>Parameters used to evaluate hairpins</P><P>Pairing Energy</P><P>G-C -2.3</P><P>A-T -0.9</P><P>G-T 1.3</P><P>Mismatch 3.5</P><P>Gap 6.0</P><P>Loop_pen(n) 1•(n - 2)</P><P>Parameters used to evaluate the energy of a potential hairpin where <I>n</I> is the length of the hairpin loop</P><P>The D score is calculated by Carafa Scoring System.</P><H1>Carafa Scoring System</H1><P>Scoring System 2 is based on the model created by d'Aubenton Carafa 2. The score of terminator consists of two parts, the free energy of stemloop and the score of 15 nt poly T tail. The free energy of stemloop is calculated using Loop Dependent Energy Rules 3. The minimization of the free energy also determined the secondary structure of the stemloop. T tail score is calculated by the formula given by d' Aubenton Carafa. </P><P>Detailed Calculation of Score</P><H3>1. Some definitions3</H3><P>i. Closing Base Pair</P><P>For an RNA sequences, we number it from 5' to 3' . If i &lt; j and nucleotides ri and rj form a base pair,we denote it by i.j. We call base ri' or base pair i'.j' is accessible from i.j if i &lt;i' ( &lt;j' ) &lt;j and if there is no other base pair k.l so that i &lt;k &lt;i' ( &lt;j' ) &lt;l &lt;j. We denote the collection of base and base pair accessible from i.j by L(i,j). Then i.j is the closing base pair. Here “L” means loop.</P><P>ii. n-loop</P><P>If the loop contain n – 1 base pairs, we denote it by n-loop. (Because there is a closing base pair, so we denote it by n-loop even though the closing base pair is not included in the loop.)</P><P>Here we can divide loops which may be formed in the terminator secondary structure into two kinds.</P><P>1-loop : Hairpin loop(size of loop shouldn't be smaller than 3) </P><P>2-loop : Interior Loop(right strand size and left strand size are both bigger than 0.)</P><P>Buldge(Size of one strand is bigger than 0 and that of another strand is 0.)Stack(size of the loop is 0.)</P><H3>2. Calculation of the Minimum Free Energy Change of Stemloop Formation4 Assume i.j is the closing base pair of the loop </H3><P><I>G（i,j）= min { GH ( i , j ) , GS( i , j ) + G ( i + 1 , j – 1 ) , GBI( i , j ) } </I>;</P><P><I>GBI ( i , j ) = min{ gbi( i , j , k , l ) + G( k , l ) } for all 0 &lt; k – i + l – j - 2 &lt; max_size</I></P><P>G(i,j) is the minimum free energy change of stemloop formation. GH is the free energy change to form a hairpin loop. GS is the free energy change to form a stack. GBI is to calculate the minimum free energy change of structure containing 2-loop. gbi(i,j,k,l) is the free energy change to form 2-loop.</P><H3>3.Calculation of T Tail Score</H3><P>Here we consider 15 nucleotide in the downstream of stemloop. T tail score nT is calculated as follows :</P><P>In our program, if the length of the T tail( n ) is less than 15, we will only consider n nucleotides. If TL is more than 15, we will only consider 15 nucleotides.</P><H3>4.Calculation of Score</H3><P>Score = nT * 18.16 + deltaG / LH * 96.59 – 116.87</P><P>Here nT is T tail score. deltaG is the minimum free energy change of stemloop formation. LH is the length of stemloop.<SUP>5,6</SUP></P><H1>References</H1><DIV class="body_references">[1] Kingsford, C. L.; Ayanbule, K.; Salzberg, S. L., Rapid, accurate, computational discovery of Rho-independent transcription terminators illuminates their relationship to DNA uptake. <I>Genome biology  </I><B>2007</B>,<I> 8</I> (2), R22.</DIV><DIV class="body_references">[2] Carafa, Y. d. A.; Brody, E.; Thermes, C., Prediction of rho-independent <I>E. coli</I> transcription terminators: A statistical analysis of their RNA stem-loop structures. <I>Journal of molecular biology</I><B>1990</B>,<I>216</I> (4), 835-858.</DIV><DIV class="body_references">[3] Manual of Mfold Version 3.5.</DIV><DIV class="body_references">[4] http://unafold.math.rpi.edu/lectures/old_RNAfold/node2.html.</DIV><DIV class="body_references">[5] Lesnik, E. A.; Sampath, R.; Levene, H. B.; Henderson, T. J.; McNeil, J. A.; Ecker, D. J., Prediction of rho-independent transcriptional terminators in E. coli.<I>Nucleic acids research</I> 3583-3594.</DIV><DIV class="body_references">[6] Sugimoto, N.; Nakano, S.-i.; Katoh, M.; Matsumura, A.; Nakamuta, H.; Ohmichi, T.; Yoneyama, M.; Sasaki, M., Thermodynamic parameters to predict stability of RNA/DNA hybrid duplexes. <I>Biochemistry</I><B>1995</B>,<I> 34</I>  (35),11211-11216.</DIV></DIV></DIV><DIV id="protein"><DIV id="protein_m1"><DIV class="sep_title">SynoProteiner</DIV></DIV><DIV id="protein_m2"><H1>Abstract</H1><P>Our team mainly focuses on programming the software by genetic algorithm, evaluating both optimization of single codon and codon pair and hence determining the fittest optimized sequences for expression in heterologous host cell.</P><P>In addition, Synoproteiner protects enzyme cutting site. Users can choose corresponding cutting sites, not substituted during the optimization, so that the power of chosen enzyme cutting site won’t fail.</P><P>Apart from the optimization, we have two additional functions. One is the statistics analysis, which provides the numbers and the proportion of the codon in the original and optimized sequences, making the optimization easier to understand. The other is the prediction of the protein folding rate. The purpose of the prediction is to seek the law of the folding rate in general, computing a relatively accurate folding rate value of the optimized sequences for the users.</P><H1>Background</H1><H4>Synonymous codons and the efficiency</H4><P>Except methionine and tryptophan, all amino acids can be encoded by two to six synonymous codons, resulting from the degeneracy of the genetic code.<SUP>1</SUP>  However, unequal utilization of the synonymous condons leads to the phenomenon of codon usage bias, which is mainly due to natural selection, mutation and genetic drift.<SUP>2</SUP> According to related studies, codon usage bias has certain connection with gene expression level.<SUP>3</SUP> The larger the value of codon usage bias is, the higher gene expression will be. So the problem, how to substitute the synonymous codons aimed at raising the efficiency of gene expression and thus increasing the production of recombination protein in heterologous host cell, is expected to be addressed.</P><H4>Protein folding rate</H4><P>Protein is an important class of biological macromolecules. It is the main bearer of life activities and occupies a special position in vivo. Each protein has its own unique amino acid composition and sequences. Only when the amino acid chain is folded into the correct three-dimensional structure, will the protein have normal biological functions. Misfolded ones will not only lose its biological function but also even cause diseases such as mad cow disease, Alzheimer's syndrome, etc. The protein folding problem, an important biological question that the central dogma of molecular biology has not solved yet, has been listed as an important topic in twenty-first century. The folding mechanism of the protein is a challenging task, one of which is to determine factor influencing the folding rate. Although the answer can be found in a variety of biological experiments, such as various spectroscopy, mass spectrometry and nuclear magnetic resonance, these methods are time-consuming and costly. With the development of physics, mathematics, especially the progress of computer technology, how to apply a fast and accurate calculation method to predict protein folding rate attracts more and more attention.<SUP>4</SUP></P><H1>Introduction</H1><H4>Balance with single codon and codon pair</H4><P>Individual codon usage optimization has been attached importance to, taking Codon optimizer,<SUP>5</SUP> Gene Designer,<SUP>6</SUP> OPTIMIZER<SUP>7</SUP> for example. Subsequently, people found the effect of gene expression optimization cannot be perfect just by single codon optimization. Codon pair, namely the pair of k-th and (k+1)-th codons from the 5’ to 3’ end, is another crucial factor. Due to potential tRNA-tRNA steric interaction within the ribosomes,<SUP>8</SUP> the usage of rare condon pairs, which correlate with translation elongation, decrease protein translation rates.<SUP>9</SUP> Optimization of individual codon has an influence on the corresponding codon pair resulting in maybe-not-the-best codon pair optimization. In the same way, optimizing codon pair merely contributes to maybe-not-the-best single codon optimization. Therefore, it is a challenging way for us to apply a method considering and weighing the effects of single codon and codon pair optimization and thus make the whole best.</P><P>Our team focuses on evaluating both optimization of single codon and codon pair and thus selecting the best sequences for expression in heterologous host cell.</P><H4>Host Cell</H4><P>Considering <I>E. coli</I> and <I>S. cerevisiae</I> are the ideal hosts for recombinant proteinexpression, and Gram-positive bacterium <I>L. lactis</I> and methylotrophic yeast <I>P. pastoris</I> are also promising candidates for expressing recombinant proteins,<SUP>10</SUP> we attached importance to selecting these four kinds of bacterium as host cell to optimize the sequences.</P><H4>Enzyme cutting site protection</H4><P>To avoid deconstruction of enzyme cutting site and therefore incapability during substitution of the base, Synoproteiner succeeds in exchanging <A href="https://2013.igem.org/Team:XMU_Software/Project_protein/cutting_site" target="_blank">14 kinds of commonly used cutting sites </A>such as EcoRI, XbaI and so on.</P><P>Users can multi-select the enzyme cutting site that they want to stay unaltered and hence get an optimized sequence without changing selected region.</P><H4>Method of prediction</H4><P>In recent years, many researchers have made great efforts to explore the determinants of the folding rate, and various forecasting methods have been proposed. The existed prediction methods can be roughly divided into three categories.<SUP>11-12</SUP> The first one is based on the tertiary structure.<SUP>13-19</SUP> However, it takes lots of molecular experiments, expensive and in long period, to acquire the information of the tertiary structure, which fails to meet the demand of rapid prediction. The second category is based on the secondary structure.<SUP>20-24</SUP> This kind of method requires information of the secondary structure, similarly obtained by molecular experiments, or from the primary sequences prediction, but it will be limited by accuracy of the secondary structure prediction method. The last one is based on the primary structure,<SUP>25-34</SUP> which predicts the folding rate from amino acid sequences without most structure information.<SUP>4</SUP> And our prediction of the protein folding rate focuses on the last method.</P><H1>Algorithm</H1><H4>Part I—Optimization</H4><H3>Basic Table</H3><P>Based on the synonymous codons table, we calculate, we calculate function of single codon, function of codon pair  and the function of multi-objective codon optimization. The method aims at make the optimization of whole best by calculating the relative effect of single codon and codon pair.</P><H4>Fitness calculation<SUP>36</SUP></H4><H3>Fitness function：</H3><P>In the function,</P><P><I>cpi</I> is a value larger than zero, ranging from 10<SUP>-4</SUP> to 0.5,<I>fit<SUB>cp</SUB></I> (<I>g</I>) is the fitness function of the codon pair，<I>fit<SUB>sc</SUB></I> (<I>g</I>) is the fitness function of the single codon，<I>w</I> ( (<I>c</I> (<I>k</I>),<I>c</I> (<I>k+1</I>)) is the weight of codon pairs in sequences <I>g</I>,|<I>g</I>| is the length of encoding sequences, <I>c</I> (<I>k</I>) is k-th codon in the sequences, is the target ratio of k-th codon,  is the actual ratio of k-th codon in the sequences，the best value of <I>cpi</I> is 0.2 in the software.</P><P>In the function, the target ratio of k-th codon can be approximated by the equation below:</P><P>In the function, weight can be calculated by the equation below:</P><P>stands for the ratio of single codon c<SUB>k</SUB> in the complete genome'is the number of pair ( <I>c<SUB>i</SUB>,c<SUB>j</SUB></I> )  in high-expression genes, and high-expression genes are genes whose copy numbers of mRNA can be detected at least 20 per cell. </P><P>syn (c<SUB>k</SUB>) stands for the synonymous codon set related to <I>c<SUB>k</SUB></I>,equals to the number of amino acid encoded by <I>c<SUB>i</SUB></I> in the whole protein set. </P><H3>Calculation of Fitness</H3><P>NSGA-II algorithm applied<SUP>35</SUP>:</P><P>1.	Randomly initialize a population of coding sequences for target protein.</P><P>2.	Evaluate <I>fit<SUB>sc</SUB></I> (<I>g</I>) and <I>fit<SUB>sp</SUB></I> (<I>g</I>) fitness of each sequences in the population.</P><P>3.	Group the sequences into nondominated sets and rank the sets.</P><P>4.	Check termination criterion.</P><P>5.	If termination criterion is not satisfied, select the “fittest” sequences (top 50% of the population) as the parents for creation of offsprings via recombination and mutation.</P><P>6.	Combine the parents and offsprings to form a new population.</P><P>7.	Repeat steps 2 to 5 until termination criterion is satisfied.</P><P>The identification and ranking of nondominated sets in step 3 is performed via pair-wise comparison of the sequences' <I>fit<SUB>sc</SUB></I> (<I>g</I>) and <I>fit<SUB>sp</SUB></I> (<I>g</I>) fitness. For a given pair of sequences with fitness values expressed as (<I>fit<SUP>1</SUP><SUB>cp</SUB></I> (<I>g</I>),<I>fit<SUP>1</SUP><SUB>sc</SUB></I> (<I>g</I>)) and (<I>fit<SUP>2</SUP><SUB>cp</SUB></I> (<I>g</I>),<I>fit<SUP>2</SUP><SUB>sc</SUB></I> (<I>g</I>)), the domination status can be evaluated as follows:</P><P>•	If (<I>fit<SUP>1</SUP><SUB>sc</SUB></I> (<I>g</I>)&gt;<I>fit<SUP>2</SUP><SUB>sc</SUB></I> (<I>g</I>))and (<I>fit<SUP>1</SUP><SUB>cp</SUB></I> (<I>g</I>)&gt;=<I>fit<SUP>2</SUP><SUB>cp</SUB></I> (<I>g</I>)), sequences 1 dominates sequences 2.</P><P>•	If (<I>fit<SUP>1</SUP><SUB>sc</SUB></I> (<I>g</I>)&gt;=<I>fit<SUP>2</SUP><SUB>sc</SUB></I> (<I>g</I>)) and (<I>fit<SUP>1</SUP><SUB>cp</SUB></I> (<I>g</I>)&gt;<I>fit<SUP>2</SUP><SUB>cp</SUB></I> (<I>g</I>)), sequences 1 dominates sequences 2.</P><P>•	If  (<I>fit<SUP>1</SUP><SUB>sc</SUB></I> (<I>g</I>)&lt;<I>fit<SUP>2</SUP><SUB>sc</SUB></I> (<I>g</I>))and (<I>fit<SUP>1</SUP><SUB>cp</SUB></I> (<I>g</I>)&lt;=<I>fit<SUP>2</SUP><SUB>cp</SUB></I> (<I>g</I>)), sequences 2 dominates sequences 1.</P><P>•	If  (<I>fit<SUP>1</SUP><SUB>sc</SUB></I> (<I>g</I>)&lt;=<I>fit<SUP>2</SUP><SUB>sc</SUB></I> (<I>g</I>))and (<I>fit<SUP>1</SUP><SUB>cp</SUB></I> (<I>g</I>)&lt;<I>fit<SUP>2</SUP><SUB>cp</SUB></I> (<I>g</I>)), sequences 2 dominates sequences 1.</P><P>The process is showed in the figure below:</P><DIV class="body_photo">Figure 1</DIV><H3>Result:</H3><P>By this method, there are enough experimental data to prove the sequences optimized works. Xylose isomerase in <I>Bacillus stearothermophilus</I>, Xylose isomerase in <I>Streptomyces olivochromogenes</I> and L-arabinose isomerase in <I>Thermoanaerobacter mathranii</I>  all, the optimized ones, were highly expressed  in <I>Bacillus subtilis</I>. In addition, the activity of the optimized Aspergillusniger fungal amylase was enhanced to 400% compared with the original sequences in <I>A. niger</I>.<SUP>36</SUP></P><P><A href="https://2013.igem.org/Team:XMU_Software/Project_protein/Example_of_optimization" target="_blank">Link to the page of Example of optimization</A></P><H4>Part II—Prediction of protein folding rate</H4><P>In order to illustrate protein folding rate quantitatively, we determine the folding rate of 60 kinds of proteins as an experimental data set from literature and database<SUP>37</SUP>,  and information of the sequences comes from PBD and NCBI.</P><TABLE border="1" cellspacing="0" class="protein_table2"><TBODY><TR><TH>   protein   </TH><TH>Logarithm of the folding rate Ln(k<SUB>f</SUB>)</TH><TH>   protein   </TH><TH>Logarithm of the folding rate  Ln(k<SUB>f</SUB>)</TH><TH>   protein   </TH><TH>Logarithm of the folding rate  Ln(k<SUB>f</SUB>)</TH></TR><TR><TD>2PDD</TD><TD>9.8</TD><TD>1FKB</TD><TD>1.5</TD><TD>1RA9</TD><TD>-2.5</TD></TR><TR><TD>2ABD</TD><TD>6.6</TD><TD>2CI2</TD><TD>3.9</TD><TD>1QOP</TD><TD>-6.9</TD></TR><TR><TD>256B</TD><TD>12.2</TD><TD>1URN</TD><TD>5.8</TD><TD>1PHP</TD><TD>2.3</TD></TR><TR><TD>1IMQ</TD><TD>7.3</TD><TD>1APS</TD><TD>-1.5</TD><TD>1PHP</TD><TD>-3.5</TD></TR><TR><TD>1LMB</TD><TD>8.5</TD><TD>1RIS</TD><TD>5.9</TD><TD>1BNI</TD><TD>2.6</TD></TR><TR><TD>1WIT</TD><TD>0.4</TD><TD>1POH</TD><TD>2.7</TD><TD>2LZM</TD><TD>4.1</TD></TR><TR><TD>1TEN</TD><TD>1.1</TD><TD>1DIV</TD><TD>6.1</TD><TD>1UBQ</TD><TD>5.9</TD></TR><TR><TD>1SHG</TD><TD>1.4</TD><TD>2VIK</TD><TD>6.8</TD><TD>1SCE</TD><TD>4.2</TD></TR><TR><TD>1SRL</TD><TD>4</TD><TD>1A6N</TD><TD>1.1</TD><TD>1YCC</TD><TD>9.62</TD></TR><TR><TD>1PNJ</TD><TD>-1.1</TD><TD>1CEI</TD><TD>5.8</TD><TD>1VII</TD><TD>11.52</TD></TR><TR><TD>1SHF</TD><TD>4.5</TD><TD>2CRO</TD><TD>3.7</TD><TD>1NYF</TD><TD>4.54</TD></TR><TR><TD>1PSF</TD><TD>3.2</TD><TD>2A5E</TD><TD>3.5</TD><TD>2AIT</TD><TD>4.2</TD></TR><TR><TD>1CSP</TD><TD>7</TD><TD>1IFC</TD><TD>3.4</TD><TD>1PIN</TD><TD>9.44</TD></TR><TR><TD>1C9O</TD><TD>7.2</TD><TD>1EAL</TD><TD>1.3</TD><TD>1C8C</TD><TD>6.91</TD></TR><TR><TD>1G6P</TD><TD>6.3</TD><TD>1OPA</TD><TD>1.4</TD><TD>1BRS</TD><TD>3.4</TD></TR><TR><TD>1MJC</TD><TD>5.3</TD><TD>1CBI</TD><TD>-3.2</TD><TD>1UBQ</TD><TD>5.9</TD></TR><TR><TD>1LOP</TD><TD>6.6</TD><TD>1QOP</TD><TD>-2.5</TD><TD>3CHY</TD><TD>1</TD></TR><TR><TD>1C8C</TD><TD>7</TD><TD>1BRS</TD><TD>3.4</TD><TD>1BIN</TD><TD>2.6</TD></TR><TR><TD>1HZ6</TD><TD>4.1</TD><TD>3CHY</TD><TD>1</TD><TD>1SCE</TD><TD>4.2</TD></TR><TR><TD>1PGB</TD><TD>6</TD><TD>2RN2</TD><TD>0.1</TD><TD>1GXT</TD><TD>4.38</TD></TR></TBODY></TABLE><P>In order that the characteristic factors of the folding rate can be extracted from protein sequences, we introduced the Chou's pseudo amino acid composition concept.<SUP>38</SUP> According to the pseudo amino acid composition principle, the position information of protein sequences can be, to some extent, reflected by a group of serial correlation factors <I>θ<SUB>1</SUB>，θ<SUB>2</SUB> ，θ<SUB>3</SUB>……，θ<SUB>n</SUB></I> ,which is defined as follows:</P><P>in the function, <I>θ<SUB>1</SUB></I> is called the first-tier correlation factor that reflects the sequences order correlation between all the most contiguous residues along a protein chain (Fig. 2a), <I>θ<SUB>2</SUB></I> the second-tier correlation factor that reflects the sequences order correlation between all the second most contiguous residues (Fig.2b), <I>θ<SUB>3</SUB></I> the third-tier correlation factor that reflects the sequences order correlation between all the 3rd most contiguous residues (Fig.2c), and so forth.<SUP>38</SUP></P><DIV class="body_photo">Figure 2</DIV><P>the correlation function is given by<SUP>4</SUP>:</P><DIV class="body_photo">Θ(R<SUB>i</SUB>,R<SUB>j</SUB>)=|<I>H</I>(<I>R<SUB>j</SUB></I>)-<I>H</I>(<I>R<SUB>i</SUB></I>)|</DIV><P>where <I>H1(R<SUB>i</SUB>)), H2(R<SUB>i</SUB>), and M(R<SUB>i</SUB>)</I> are, respectively, the hydrophobicity value. Studies have shown that λ=10 will be the best predictor.<SUP>39</SUP> But there will be a large amount of calculation considering all possible situations—the 30 factors. We should select factors that can obtain the best prediction accuracy in least calculation. For that reason, we drew lessons from the literature<SUP>4</SUP> by using the method of Monte Carlo simulation and then 14 optimal characteristic factor were obtained. Other studies have indicated that the logarithm of the sequences length has a good correlation with folding rate, so Ln (L) will be the fifteenth factors. We apply SPSS software to calculate the coefficient of 15 factor by multivariate linear regression, and this will be the forecast formula of the rate of protein folding. We compared the experimental data and the predicted data and the results are as follows:</P><DIV class="body_photo">Figure 3 The comparision of the experimental and the predicted results</DIV><P>Through the test, our software succeeded in showing a relatively accurate folding rate value.</P><H1>Future work</H1><P>First of all, we will modify our software by advancing the program and the framework to improve its ability of concurrent computation and shorten the computing time.</P><P>Secondly, to accelerate the calculation, we may simplify the function of calculation by neglecting some term in our equations. However, considering the time spent on running program was extremely little, we will pay more attention on how to modify the equations for increasing the accuracy which maybe dramatically progress optimization result.</P><P>Thirdly, enriching the database is other way to improve our software. According to time-space tradeoff law, we could pre-process a bunch of sequences in common use to optimized one and save the result into our database. By assessing our data, investigators could select the optimized sequences for their synthesis. Then, users are required to feedback their result. When it collects enough information, our app will learn users’ bias therefore modify our optimizing function by some methods, like NSGA-II algorithm.</P><P>The specific points are listed as following:</P><P>1.	Shortening the computing time of the software.</P><P>2.	Expanding the range of the host cells.</P><P>3.	Improving bacterium's resistance to toxic molecule.</P><P>4.	Advancing existed paths of synthetic biology by the method.</P><P>5.	Designing new paths of synthetic biology by the method.</P><P>6.	Increasing the output of recombinant protein.</P><P>7.	Predicting the expression of heterologous gene in a new host cell.</P><P>8.	Considering more factors such as spiral structure in folding which influence the folding rate and thereby obtaining more accurate prediction rate.</P><P>9.	Providing a set of software tools for protein folding, especially in molecular dynamics simulation of protein folding.</P><H1>References</H1><DIV class="body_references">[1] Grantham, R.; Gautier, C.; Gouy, M.; Mercier, R.; Pave, A., Codon catalog usage and the genome hypothesis. <I>Nucleic acids research  </I><B>1980</B>, <I>8</I> (1),  197-197.</DIV><DIV class="body_references">[2] Hershberg, R.; Petrov, D. A., Selection on codon bias. <I> Annual review of genetics </I><B>2008</B>, <I> 42,  </I>287-299.</DIV><DIV class="body_references">[3] Gouy, M.; Gautier, C., Codon usage in bacteria: correlation with gene expressivity. <I> Nucleic acids research </I><B>1982</B>,  <I>10</I> (22), 7055-7074.</DIV><DIV class="body_references">[4] 郭建秀，饶妮妮, 刘广雄, 李杰, &amp; 王云鹤. 从氨基酸序列预测蛋白质折叠速率. 生物化学与生物物理进展 <I>Progress in Biochemistry and Biophysics</I><B>2010</B>, <I>37</I>(12): 1331~1338</DIV><DIV class="body_references">[5] Fuglsang, A., Codon optimizer: a freeware tool for codon optimization.  <I>Protein expression and purification  </I><B>2003</B>,  <I>31</I> (2),  247-249.</DIV><DIV class="body_references">[6] Villalobos, A.; Ness, J. E.; Gustafsson, C.; Minshull, J.; Govindarajan, S., Gene Designer: a synthetic biology tool for constructing artificial DNA segments. <I> Bmc Bioinformatics </I><B>2006</B>,  <I>7</I> (1),  285.</DIV><DIV class="body_references">[7]	Puigbò, P.; Guzmán, E.; Romeu, A.; Garcia-Vallvé, S., OPTIMIZER: a web server for optimizing the codon usage of DNA sequences.  <I>Nucleic acids research </I><B>2007</B>, <I> 35</I> (suppl 2),  W126-W131.</DIV><DIV class="body_references">[8] Smith, D.; Yarus, M., tRNA-tRNA interactions within cellular ribosomes.  <I>Proceedings of the National Academy of Sciences  </I><B>1989</B>,  <I>86 </I>  (12),4397-4401.</DIV><DIV class="body_references">[9] Coleman, J. R.; Papamichail, D.; Skiena, S.; Futcher, B.; Wimmer, E.; Mueller, S., Virus attenuation by genome-scale changes in codon pair bias.  <I>Science </I><B>2008</B>, <I> 320</I> (5884),  1784-1787.</DIV><DIV class="body_references">[10] (a) Wildt, S.; Gerngross, T. U., The humanization of N-glycosylation pathways in yeast. Nature Reviews Microbiology <B>2005</B>, <I>3</I> (2), 119-128; (b) Morello, E.; Bermudez-Humaran, L.; Llull, D.; Sole, V.; Miraglio, N.; Langella, P.; Poquet, I., Lactococcus lactis, an efficient cell factory for recombinant protein production and secretion.  <I>Journal of molecular microbiology and biotechnology  </I><B>2007</B>,  <I>14</I> (1-3),  48-58.</DIV><DIV class="body_references">[11] 郭建秀, 马彬广, 张红雨. 蛋白质折叠速率预测研究进展. 生物物理学报, <B>2006</B>, <I>22</I>(2):89-95 Guo J X, Ma B G, Zhang H Y. <I>Acta Biophys Sin</I>, <B>2006</B>, <I>22</I> (2):89-95.</DIV><DIV class="body_references">[12] Gromiha M M, Selvaraj S. Bioinformatics approaches for understanding and predicting protein folding rates. <I>Current Bioinformatics</I>, <B>2008</B>, <I>3</I>(1): 1-9</DIV><DIV class="body_references">[13] Plaxco K W, Simons K T, Baker D. Contact order, transition state placement and the refolding rates of single domain proteins. <I>J MolBiol</I>, <B>1998</B>, <I>277</I>(4): 985-994</DIV><DIV class="body_references">[14] Gromiha M M, Selvaraj S. Comparison between long-range interactions and contact order in determining the folding rate of two-state proteins: application of long-range order to folding rate prediction. <I>J Mol Biol</I>, <B>2001</B>, <I>310</I>(1): 27-32</DIV><DIV class="body_references">[15] Zhou H, Zhou Y. Folding rate prediction using total contact distance. <I>Biophys J</I>, <B>2002</B>, <I>82</I>(1): 458-463</DIV><DIV class="body_references"> [16] Nölting B, Schälike W, Hampel P, <I>et al</I>. Structural determinants of the rate of protein folding.<I> J Theor Biol</I>, <B>2003</B>, <I>223</I>(3): 299-307</DIV><DIV class="body_references"> [17] Weikl T R, Dill K A. Folding kinetics of two-state proteins: Effect of circularization, permutation, and crosslinks. <I>J Mol Biol</I>, <B>2003</B>,<I>332</I>(4): 953-963</DIV><DIV class="body_references"> [18] Ivankov D N, Garbuzynskiy S O, Alm E, et al. Contact order revisited: influence of protein size on the folding rate. <I>Protein Sci</I>,<B>2003</B>, <I>12</I>(9): 2057-2062</DIV><DIV class="body_references"> [19] Mirny L, Shakhnovich E. Protein folding theory: from lattice to all-atom models. <I>Annu Rev Biophys Biomol Struct</I>, <B>2001</B>, <I>30</I> (1):361-396</DIV><DIV class="body_references"> [20] Gong H, Isom D G, Srinivasan R, et al. Local secondary structure content predicts folding rates for simple two-state proteins. <I>J MolBiol</I>, <B>2003</B>, <I>327</I>(5): 1149-1154</DIV><DIV class="body_references"> [21] Ivankov D N, Finkelstein A V. Prediction of protein folding rates from the amino acid sequences-predicted secondary structure. <I>Proc Nat Acad Sci USA</I>, <B>2004</B>, <I>101</I>(24): 8942-8944</DIV><DIV class="body_references"> [22] Fleming P J, Gong H P, Rose G D. Secondary structure determines protein topology. <I>Protein Sci</I>, <B>2006</B>, <I>15</I>(8): 1829-1834</DIV><DIV class="body_references"> [23] Huang J T, Cheng J P, Chen H. Secondary structure length as a determinant of folding rate of proteins with two- and three-state kinetics. <I>Proteins</I>, <B>2007</B>, <I>67</I>(1): 12-17</DIV><DIV class="body_references"> [24] Prabhu N P, Bhuyan A K. Prediction of folding rates of small proteins: empirical relations based on length, secondary structure content, residue type, and stability. <I>Biochemistry</I>,<B> 2006</B>, <I>45</I> (11):3805-3812</DIV><DIV class="body_references"> [25] Shao H, Peng Y, Zeng Z H. A simple parameter relating sequenceswith folding rates of small helical proteins. <I>Protein Pept Lett</I>, <B>2003</B>,<I>10</I>(3): 277-280</DIV><DIV class="body_references"> [26] Galzitskaya O V, Garbuzynskiy S O, Ivankov D N, et al. Chainlength is the main determinant of the folding rate for proteins withthree-state folding kinetics. <I>Proteins</I>, <B>2003</B>, <I>51</I>(2): 162-166</DIV><DIV class="body_references"> [27] Huang J T, Jing T. Amino acid sequences predicts folding rate for middle-size two-state proteins. <I>Proteins</I>, <B>2006</B>, <I>63</I>(3): 551-554</DIV><DIV class="body_references"> [28] Gromiha M M. A statistical model for predicting protein folding rates from amino acid sequences with structural class information.<I>J Chem Inf Model</I>, <B>2005</B>, <I>45</I>(2): 494-501</DIV><DIV class="body_references"> [29] Ma B G, Guo J X, Zhang H Y. Direct correlation between proteins'folding rates and their amino acid compositions: an ab initio foldingrate prediction. <I>Proteins</I>, <B>2006</B>, <I>65</I>(2): 362-372</DIV><DIV class="body_references"> [30] Gromiha M M, Thangakani A M, Selvaraj S. FOLD-RATE:prediction of protein folding rates from amino acid sequences.<I>Nucleic Acids Res</I>, <B>2006</B>, <I>34</I>(suppl_2): 70-74.</DIV><DIV class="body_references"> [31] OuYang Z, Liang J. Predicting protein folding rates from geometric contact and amino acid sequences. <I>Protein Sci</I>, <B>2008</B>, <I>17</I>(7): 1256-1263</DIV><DIV class="body_references"> [32] Huang L T, Gromiha M M. Analysis and prediction of proteinfolding rates using quadratic responde surface models. <I>J ComputChem</I>, <B>2008</B>, <I>29</I>(10): 1675-1683</DIV><DIV class="body_references"> [33] Shen H B, Song J N, Chou K C. Prediction of protein folding ratesfrom primary sequences by fusing multiple sequential features.<I>J Biomedical Science and Engineering</I>, <B>2009</B>,<I> 2</I>(3): 136-143</DIV><DIV class="body_references"> [34] Jiang Y, Iglinski P, Kurgan L. Prediction of protein folding ratesfrom primary sequences using hybrid sequences representation.<I>J Comput Chem</I>, <B>2009</B>, <I>30</I>(5): 772-783</DIV><DIV class="body_references">[35] Chung, B.; Lee, D.-Y., Computational codon optimization of synthetic gene for protein expression.<I> BMC systems biology </I><B>2012</B>,  <I>6</I> (1),  134.</DIV><DIV class="body_references">[36] 	帝斯曼知识产权资产管理有限. 公司实现改进的多肽表达的方法: 中国, 200780024670.5[P]. 2009-07-22</DIV><DIV class="body_references"> [37]	Gromiha, M. M.; Thangakani, A. M.; Selvaraj, S., FOLD-RATE: prediction of protein folding rates from amino acid sequences.<I> Nucleic acids research</I><B>2006</B>, <I>34 </I>(suppl 2), W70-W74. </DIV><DIV class="body_references"> [38] Chou, K. C., Prediction of protein cellular attributes using pseudo‐amino acid composition. <I>Proteins: Structure, Function, and Bioinformatics</I><B>2001</B>, <I>43</I> (3), 246-255. </DIV><DIV class="body_references"> [39]	Galzitskaya, O. V.; Garbuzynskiy, S. O.; Ivankov, D. N.; Finkelstein, A. V., Chain length is the main determinant of the folding rate for proteins with three‐state folding kinetics. <I>Proteins: Structure, Function, and Bioinformatics</I><B>2003</B>, <I>51 </I>(2), 162-166. </DIV></DIV></DIV><DIV id="enote"><DIV id="enote_m1"><DIV class="sep_title">E' NOTE</DIV></DIV><DIV id="enote_m2"><H1>Abstract</H1><P><A href="http://1.trysomething.sinaapp.com" target="_blank">E' NOTE</A> is an experiment recording tool specifically designed for iGEMers. It allows users to take notes, draw tables, upload images, and provides a series of templates for them. The templates are capable of some basic calculations such as enzyme digestion system and ligation systems setting, which significantly ease the burden of experiment recording. Besides, <A href="http://1.trysomething.sinaapp.com" target="_blank">E' NOTE</A> also provides a plasmid library which can be further constructed by users, in addition to basic information, the library can offer the data of the time of purification and breed conservation to avoid the death of strains caused by freezing for too long. The library is linked to the templates, hence, after recording the data in the library, the templates will present the data which the users recorded in the library.<A href="http://1.trysomething.sinaapp.com" target="_blank">E' NOTE</A> also offers a calculation board for some basic calculations during solution configuration and a built-in E-mail sender to facilitate communication among team members. Of course, the tools provided by <A href="http://1.trysomething.sinaapp.com" target="_blank">E' NOTE</A> is far from complete, so <A href="http://1.trysomething.sinaapp.com" target="_blank">E' NOTE</A> contains a software integration board. It can integrate the software tools useful for synthetic biology experiment process on the internet. Users can easily find their desired software in this part.The broad majority of iGEMers may often get entangled with uploading the experiment notes to the wiki by either PDF file or constructing a webpage. But it’s a pity that simple experiment notes cannot show the experimental method and thus barricade communication. If you use <A href="http://1.trysomething.sinaapp.com" target="_blank">E' NOTE</A> to record your experiments, simply input your data according to the specification and the experiments data will be transferred to wiki conveniently, just let <A href="http://1.trysomething.sinaapp.com" target="_blank">E' NOTE</A> to present your experiment note perfectly.
							</P><H1>Demo</H1><H1>Introduction for using</H1><P>1.Click here to learn how to use E' NOTE: <A href="https://static.igem.org/mediawiki/2013/a/ac/Enote2.8xxxxxxxxxxxxxxxxxxxxxxxxxxx.pdf" target="_blank">Tutorial of E' note</A>.</P><P>2.Output the record to create a wiki page: <A href="https://static.igem.org/mediawiki/2013/0/06/Xmuxmuxmuxmu%E6%95%B0%E6%8D%AE%E5%AF%BC%E5%87%BA%E8%AF%B4%E6%98%8E%E6%96%87%E6%A1%A3.ppt" target="_blank">Data output</A>. ( <A href="https://2013.igem.org/Team:XMU_Software/Project/enote/xmlcss" target="_blank">xml.css</A> )</P><P>
								( Click me to use E' NOTE: <A href="http://1.trysomething.sinaapp.com" target="_blank">Link to E' NOTE</A> )
							</P><H1>Future work</H1><P>
	In the future, <A href="http://1.trysomething.sinaapp.com" target="_blank">E' NOTE</A> will integrate more online tools for synthetic biology experiments and classify them to facilitate the users’ searching process(Figure 1). In addition, it will provide a port for the users to contribute the software they think might help with synthetic biology experiments. And the software will appear on the software integration board after verification(Figure 2). Furthwer nore, <A href="http://1.trysomething.sinaapp.com" target="_blank">E' NOTE</A> also make an attempt to integrate the software tools offline. We have known that we can do it through Python and even provide a draft for that (Figure 3). <A href="http://1.trysomething.sinaapp.com" target="_blank">E' NOTE</A> is far more than a experiment recording tool for iGEMers, it is a platform for iGEMers to start can develop their projects. So, experiments recording is, just a start.						
							</P></DIV></DIV><DIV id="achieve"><DIV id="achieve_m1"><DIV class="sep_title">Achievement</DIV></DIV><DIV id="achieve_m2"><DIV id="brick_demo"><H1>Brick Worker</H1><P>Reading the Brick-Worker's source code,
								please click here:   <A href="https://static.igem.org/mediawiki/2013/d/d5/Xmu_software_ttt_Brick_Worker.pdf" target="_blank">Tutorial of Brick Worker</A></P><P>Download the Brick-Worker's source code,
								please click here:       <A href="https://static.igem.org/mediawiki/2013/3/34/Software_xmu_xmu_xmu_Brick_Worker.zip" target="_blank">Brick-Worker</A></P></DIV><DIV id="enote_demo1"><H1>E' NOTE</H1><P>Website:<A href="http://1.trysomething.sinaapp.com" target="_blank">http://1.trysomething.sinaapp.com</A></P></DIV><DIV id="team_video"><H1>TEAM</H1></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:XMU_Software/Project">http://2013.igem.org/Team:XMU_Software/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:XMU_Software/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:XMU_Software/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:XMU_Software/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:XMU_Software/Project&amp;oldid=358981" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>