<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_XMU_Software_Project_promoter"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:XMU_Software/Project/promoter">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:XMU_Software/Project/promoter&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU_Software/Project/promoter&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU_Software/Project/promoter&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:XMU_Software/Project/promoter" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:XMU Software/Project/promoter</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="xmu_body"><DIV id="xmu_software"><DIV id="project"><DIV class="navigation"><DIV><UL id="jsddm"><LI><A href="https://2013.igem.org/Team:XMU_Software">Home</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Requirements">Requirements</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Outreach">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Notebook">Notebook</A></LI></UL></DIV></DIV><DIV id="project_s1"><DIV class="project_title">PROJECT</DIV><DIV class="project_introduce">Our project includes 2 independent software tools-the brick worker and E' NOTE. The former is a software suit for the evaluation and optimization of biobricks, i.e., promoter, RBS, protein coding sequences and terminator. E' NOTE is a web application serving as an assistant for experiments. Its useful functions such as experiments recording and experimental template customization make experimental process easier and more enjoyable.</DIV></DIV><DIV class="navi_text"><A id="project_navi0" class="navi0" href="https://2013.igem.org/Team:XMU_Software/Project/promoter" onmouseover="navi0();" onmouseout="navi01();" style="color:white">Promoter-decoder</A><A id="project_navi1" class="navi1" href="https://2013.igem.org/Team:XMU_Software/Project/rbs" onmouseover="navi1();" onmouseout="navi11();" style="color:white">RBS-decoder</A><A id="project_navi2" class="navi2" href="https://2013.igem.org/Team:XMU_Software/Project/terminator" onmouseover="navi2();" onmouseout="navi21();" style="color:white">Terminator</A><A id="project_navi3" class="navi3" href="https://2013.igem.org/Team:XMU_Software/Project/protein" onmouseover="navi3();" onmouseout="navi31();" style="color:white">SynoProteiner</A><A id="project_navi4" class="navi4" href="https://2013.igem.org/Team:XMU_Software/Project/enote" onmouseover="navi4();" onmouseout="navi41();" style="color:white">E' NOTE</A><A id="project_navi5" class="navi5" href="https://2013.igem.org/Team:XMU_Software/Project/achieve" onmouseover="navi5();" onmouseout="navi51();" style="color:white">Achievement</A><A id="project_navi6" class="navi6" href="javascript:void(0)" onmouseover="navi6();" onmouseout="navi61();" style="color:white">Back To Top</A></DIV><DIV id="promoter"><DIV id="promoter_m1"><DIV class="sep_title">Promoter-decoder</DIV></DIV><DIV id="promoter_m2"><H1>Abstract</H1><P>In a promoter sequences, the sigma factor binding site and other transcription factor binding site affect the strength of binding significantly.
							For annotating promoters, some software was developed which mostly focused on the prediction of other transcription factors or one particular type of sigma factors but failed to analyze the promoter with both sigma factors and other transcription factors. <SUP>1-2</SUP>
							To solute this problem, a module of our software was designed which can analyze and evaluate promoters.</P><P>Our software use PWM method to calculate the similarity between promoter sequences and the position frequency matrix of transcription factor binding sites (TFBS) to locate the TFBS as well as to predict the relative strength of the promoter.							
							Promoter-Decoder overshadows its counterparts with all-round analysis and the prediction of promoter strength.
							It enables users to figure out promoter types, predict promoter strength, changeit by mutating the key sites and even change the property of certain promoter by adding new TFBS to the promoter sequences. </P><H1>Background</H1><H4>Sigma Factors</H4><P>Bacteria encode several thousands of different proteins, which are necessary for normal cell functions or for adaptation to environmental changes.<SUP>3</SUP> These proteins are not required at the same time or in the same amount. Regulation of gene expression therefore enables the cell to control the production of proteins needed for its life cycle or for adaptation to extracellular changes. The various steps during transcription and translation are therefore subject to different regulatory mechanisms.<SUP>4</SUP></P><P>The most prominent step in gene regulation is the initiation of transcription in which the DNA-dependent RNA polymerase (RNAP) is the key enzyme. The RNAP or the RNAP core enzyme is the catalytic machinery for the synthesis of RNA from a DNA template. However, RNAP cannot initiate transcription by itself. Initiation of transcription requires an additional polypeptide known as a sigma factor.<SUP>5</SUP> sigma factors are a family of relatively small proteins that can associate in a reversible way with the RNAP core enzyme. Together, the sigma factor and the RNAP core enzyme form an initiation-specific enzyme, the RNAP holoenzyme.</P><DIV class="body_photo">Figure 1 The initiation of transcription</DIV><P>The sigma factor directs RNA polymerase to a specific class of promoter sequences. Most bacterial species synthesize several different sigma factors that recognize different consensus sequences.<SUP>6</SUP></P><P> This variety in sigma factors provides bacteria with the opportunity to maintain basal gene expression as well as for regulation of gene expression in response to altered environmental or developmental signals.</P><P>The frequency at which the RNAP holoenzyme initiates transcription, also known as the strength of a promoter, is influenced by the promoter sequences and the conformation of the DNA in the promoter region. The sigma factors recognize two conserved sequences in the promoter region, known as the promoter consensus sequences. sigma factors or fragments of sigma factors bind specifically to promoter DNA sequences and by specific base pair and amino acid substitutions in the promoter consensus sequences or sigma factors. Most bacterial species synthesize several different sigma factors which direct the RNAP holoenzyme to distinct classes of promoters with a different consensus sequences. This variety in sigma factors provides the bacterium with the opportunity to maintain basal gene expression as well as for regulation of gene expression in response to specific environmental stimuli.</P><P>The identification of bacterial promoters is an essential step in the elucidation of gene regulation.<SUP>7</SUP></P><P>As a general rule, the more complex the life-cycle and environmental niche of a bacterium, the greater the number of sigma factors with corresponding promoter types. Typically however, the most common promoter type is that which regulates the housekeeping genes and the corresponding major sigma factor is shared by all bacteria (sigma 70 in the well studied <I>E. coli</I>, and its homologues in other species). The binding site for the sigma70-family of promoters is defined by two consensus hexamers, TTGACA and TATAAT, located at approximately −35 and −10, respectively relative to the transcript start site (TSS) and spaced 15–21 base pairs (bp) apart2. RNA polymerase core enzyme associates with the major sigma factor to form the holoenzyme which in turn binds to its cognate promoters to initiate transcription.</P><DIV class="body_photo">Figure 2 The RNA polymerase Figure 3 Consensus sequences of sigma 70 factor</DIV><P>In prokaryotes, the minimum requirement for RNA polymerase binding is recognition of the promoter by the sigma factor. In general, prokaryotic RNA polymerases can interchange a number of sigma factors which bind and initiate different groups of genes.<SUP>3</SUP></P><H4>Transcription Factors</H4><DIV class="body_photo">Figure 4 Transcription factor binding site</DIV><P>Sigma factors are essential for the transcription initiation in <I>E. coli</I>.<SUP>10</SUP></P><P> In addition, promoter strengths are not determined purely by the binding of the sigma factor. Other transcription factors can bind specific sequences surrounding or overlapping the promoter to either activate or repress transcription.<SUP>4</SUP> The mechanism is transcriptional activators and repressors contribute to and detract from the accessibility of DNA by the RNA polymerase. <SUP>12</SUP></P><P>These transcription-regulating nuclear proteins bind to specific binding sites in the regulatory regions (e.g. promoters, enhancers) of the genes thus providing their activation or repression. </P><DIV class="body_photo">Figure 5 Transcription factor binding site</DIV><P>Computational methods of predicting TF binding sites in DNA are very important for understanding the molecular mechanisms of gene regulation.</P><P>The binding sites of the same transcription factor show a significant sequences conservation, which is often summarized as a short (5–20 bases long) common pattern called a transcription factor binding site (TFBS) or binding consensus. Our software aims to figure out the possible TFBS in promoters and precisely locate the TFBS so that the user may know the exact sites that play a role in regulating the transcription.</P><P>In prokaryotes (lower organisms without nuclei), there are fewer TFs, their motifs tend to be relatively long and the strength of regulation for a particular gene often depends on how closely a particular site matches the consensus for the motif. The more mismatches to the consensus in a binding site, the less often the TF will bind and therefore the less control it will exert on the target gene. So our software will calculate the similarity between the possible TFBS in the promoter and the standard motifs so the user will know to which extent the transcription factor will control the promoter transcription</P><H4>Primer Design</H4><P>To facilitate the design of PCR primers of various promoters, we've developed an additional function, namely, primer design in this part of our program. After inputting the promoter sequences, the software will figure out the most suitable primers based on the theory of Thomas Kämpke<SUP>1</SUP>, Markus Kieninger, and Michael Mecklenburg.<SUP>13</SUP></P><H1>Data Source</H1><H4>RegulonDB</H4><P>Genes and operons that are under control of the same TF are members of that TF's regulon.  Although methods for the prediction of regulons have been substantially improved, they are still far from perfect. </P><P>Comparative genomics tools can be used to predict regulons in bacterial genomes but the procedure can lead to incorrect regulon calling. Despite this drawback, several regulon databases are available that are based on comparative genomics methods and lack experimental evidence.</P><P>Probably the extended and accurate databases of regulons for <I>E. coli</I> are RegulonDB which provides the data source for our program.</P><H1>Algorithm</H1><P>Experimental results show that these are the strongest promoters that have been characterized in vitro so far and confirm the hypothesis that the consensus promoter sequences is &quot;best&quot;. To calculate the similarity between the promoter sequences and the best sequences, we implement the PWM method<SUP>6</SUP> in our program.</P><H4>PWM (Position Weight Matrix)</H4><P>Molecular techniques for the identification of promoters are both costly and time consuming, hence in silico methods are an attractive and well explored alternative. The most common in silico method to identify sigma 70 promoters uses position weight matrices (PWMs) and depends on the relative conservation of the transcription factor binding site (TFBS, or motifs ). </P><P>The algorithm can be divided into two parts regarding to the difference between the motifs of sigma factors and other transcription factors.  </P><DIV class="body_photo">Figure 6 The consensus sequences, the position frequency matrix and the frequency logo</DIV><H4>Part 1: the recognition of other transcription factors.<SUP>7</SUP></H4><P>Other transcription factors are proteins that can bind to a specific DNA sequences (motifs) and regulate the promoter's transcription. To recognize these possible motifs in a given promoter sequences, we calculate the Matrix Similarity Score (MSS) of every possible sites in the promoter sequences using the position frequency matrix of 86 transcription factors published by RegulonDB. The algorithm reports only those matches of a matrix that have got MSS higher than the settled threshold. And MSS for a subsequences <I>x</I> of the length <I>L</I> is calculated in following steps:</P><P><I>f<SUB>i,Bi</SUB></I> , frequency of nucleotide <I>B</I> to occur at the position <I>i</I> of the matrix (<I>B</I> ∈{A, T, G, C})</P><P><I>f <SUB>i</SUB><SUP>min</SUP></I> , frequency of the nucleotide which is rarest in position <I>i</I> in the matrix</P><P><I>f <SUB>i</SUB><SUP>max</SUP></I> , highest frequency in position <I>i</I>.</P><P>The information vector</P><P>describes the conservation of the positions <I>i</I> in a matrix. Multiplication of the frequencies with the information vector leads to a higher acceptance of mismatches in less conserved regions, whereas mismatches in highly conserved regions are very much discouraged. This leads to a better performance in recognition of TF binding sites if compared with methods that do not use the information vector.</P><P>To determine the best threshold of the motif finding algorithm, we test various threshold values and analyze the true negative and false positive rate of each threshold value. The ideal threshold is supposed to have both the least true negative and false positive rates.</P><DIV><TABLE width="600" border="0" cellspacing="0" class="promoter_table1"><CAPTION>
									Table 1 The threshold setting data
								  </CAPTION><TBODY><TR border="1px"><TD align="center" valign="middle"><STRONG>Threshold</STRONG></TD><TD align="center" valign="middle" bgcolor="#FEA801">0.5977</TD><TD align="center" valign="middle">0.598</TD><TD align="center" valign="middle">0.69</TD><TD align="center" valign="middle">0.7</TD><TD align="center" valign="middle">0.73</TD><TD align="center" valign="middle">0.76</TD><TD align="center" valign="middle" bgcolor="#FEA801">0.0778</TD><TD align="center" valign="middle">0.84</TD><TD align="center" valign="middle" bgcolor="#FEA801">0.85</TD><TD align="center" valign="middle">0.86</TD><TD align="center" valign="middle">0.9</TD></TR><TR><TD align="center" valign="middle" nowrap="nowrap"><STRONG>True megtive</STRONG></TD><TD align="center" valign="middle" bgcolor="#FEA801">0.1</TD><TD align="center" valign="middle">0.11</TD><TD align="center" valign="middle">0.21</TD><TD align="center" valign="middle">0.23</TD><TD align="center" valign="middle">0.3</TD><TD align="center" valign="middle">0.45</TD><TD align="center" valign="middle" bgcolor="#FEA801">0.5</TD><TD align="center" valign="middle">0.63</TD><TD align="center" valign="middle" bgcolor="#FEA801">0.7</TD><TD align="center" valign="middle">0.72</TD><TD align="center" valign="middle">0.77</TD></TR><TR><TD align="center" valign="middle" nowrap="nowrap"><STRONG>False positive</STRONG></TD><TD align="center" valign="middle" bgcolor="#FEA801">56.4778</TD><TD align="center" valign="middle">57.01124</TD><TD align="center" valign="middle">31.56962</TD><TD align="center" valign="middle">29.15584</TD><TD align="center" valign="middle">23.58571</TD><TD align="center" valign="middle">20.92727</TD><TD align="center" valign="middle" bgcolor="#FEA801">17.90796</TD><TD align="center" valign="middle">9.945946</TD><TD align="center" valign="middle" bgcolor="#FEA801">10.66667</TD><TD align="center" valign="middle">10.17857</TD><TD align="center" valign="middle">6.608696</TD></TR></TBODY></TABLE></DIV><P>The picture above shows part of our test results and to keep both the true negative and false positive rates at a reasonable level, we adopt 3 threshold values, namely, low (0.5977) , median (0.0778) and high (0.85), with a true negative rate at 0.1, 0.5, and 0.7 respectively. And for more flexibility, we also allow the users to set their own thresholds.</P><H4>Part 2: the recognition of sigma factor motif and the evaluation of relative promoter strength.</H4><P>In the case of sigma 70 factors, the motifs are the −35 and −10 hexamers. Enclosing a spacer of length 15–19 bp.</P><P>Given a promoter sequence, the -10 and -35 hexamers are located by the total MSS of the two hexamers calculated by the position frequency matrices of the sigma factor binding sites, which are derived from Regulon DB. And the calculating process is subject to two constraints:</P><P>1. That the spacer length (the number of base pairs between the −35 hexamer and the −10 hexamer) should lie in the range (15–20);</P><DIV class="body_photo">Figure 7 The consensus sequences of sigma 70 factor binding site</DIV><P>2. The total MSS  (our results are the sum of the scores for the −10 and −35 hexamers and therefore lie in the interval [0,2], with a score of 2 corresponding to the joint consensus TTGACA (−35) and TATAAT (−10).</P><P>Score(Promoter)=score(-10 box)+score(-35 box)+score(spacer between -10 &amp; -35 boxes)</P><P>And the score of spacer length is calculated by algorithms propozed by Ryan K. Shultzaberger.el. in <ANATOMY of="">E. coli sigma70 promoters.<SUP>8</SUP> But due to a lack of experimental data of promoter strength with both different motifs and spacer length, the weight of the total MSS and the spacer score is very roughly determined with few experimental data available. Currently our weight is determined with the promoter strength data in a literature<SUP>16</SUP> to best fit the the promoter score with promoter strength. Now the relative weight between the total MSS of the two motifs and the spacer score is 0.29:0.71.</ANATOMY></P><P>In prokaryotes, the strength of sigma factor regulation for a particular gene often depends on how closely a particular site matches the consensus for the motif. The more mismatches to the consensus in a binding site, the less often the sigma factor will bind and therefore the less strength the promoter will have. Experiments have confirmed the hypothesis that the consensus promoter sequences is &quot;best&quot;. We set the best promoter strength to 100% and calculate the relative strength of a given promoter by the Score (promoter). </P><H4>Primer design</H4><P> A primer pair (<I>p</I>, <I>q</I>) is assigned the scoring vector</P><P><I>sc</I> (<I>p</I>, <I>q</I>) = (|<I>p</I>|, |<I>q</I>|, <I>GC</I> (<I>p</I>), <I>GC</I> (<I>q</I>), <I>Tm</I> (<I>p</I>), <I>Tm</I> (<I>q</I>), <I>sa</I> (<I>p</I>), <I>sa</I> (<I>q</I>), <I>sea</I> (<I>p</I>), <I>sea</I> (<I>q</I>), <I>pa</I> (<I>p</I>, <I>q</I>), <I>pea</I> (<I>p</I>, <I>q</I>))<SUP>T</SUP> ∈ <I>R</I><SUP><I>12</I></SUP>.</P><P>All primers are designed to have ideal values of length, GC content, and melting temperature which are specified externally by the designer of the hybridization experiment. These ideal values are to be specified for forward and reverse primers. The ideal score vector or reference vector for the primer pair is </P><P><I>sc<SUB>ideal</SUB></I> =(<I>length<SUB>f</SUB></I> ,  <I>length<SUB>r</SUB></I> ,  <I>GC<SUB>f</SUB></I> ,  <I>GC<SUB>r</SUB></I> ,  <I>T<SUB>m,f</SUB></I> ,  <I>T<SUB>m,r</SUB></I> , 0, 0, 0, 0, 0, 0)<SUP>T</SUP>. </P><P>All ideal annealing values are set to zero and typically</P><P><I>T<SUB>mf</SUB></I> = <I>T<SUB>m,r</SUB></I> as well as <I>GC<SUB>f</SUB></I> = <I>GC<SUB>r</SUB></I> . The final assessment of a primer pair (<I>p</I>, <I>q</I>) can be its deviation from the reference in terms of the <I>l1</I>-distance</P><P>Here, we employ a weighted distance</P><P>with weights given in the following table.</P><P>The formulas for calculating the variations above are provided in Efficient primer design algorithms.<SUP>13</SUP></P><H1>Results</H1><H4>Sigma factors recognition </H4><P> Our program has a correctness rate of 56% in recognition various types of sigma factors. We ran our program with 100 various promoters sequences whose types have already been confirmed experimentally. And we've recognized 56 of them correctly. Specifically, as for sigma 70 promoters, which are the most prevalent, the recogtion correctness rate has reached 92%. The results are showed below.</P><P><A href="https://2013.igem.org/Team:XMU_Software/result1" target="_blank">Link to the page of results</A></P><H4>TFBS Location</H4><P>We then tested the reliability of our software regarding TFBS location and results show that the correct site prediction rate is 64%. We used the sigma70 promoter sequences with annotated -35 and -10 region provided by RegulonDB to test the correct prediction rate of the binding site of a specific transcription. We input 89 sigma70 promoter sequences and ran our program to precisely locate the sigma factor binding site. </P><P>The test results are as follows. The numbers represent the site of actual-35 motif, the actual spacer length, the predicted site and predicted spacer length respectively.</P><P><A href="https://2013.igem.org/Team:XMU_Software/promoter" target="_blank">Link to the page of results</A></P><H4>Promoter strength correlation &amp; experiments</H4><P>To testify our prediction of promoter strength, our team has done a considerable amount of lab work. First, we located the -10 region of the pBAD promoter (BBa _K206000) and accordingly mutated the -10 region into BBa _K1070002, BBa _K1070003, the sequences of these promoters are given bellow (-10 regions are highlighted):</P><P>pBAD Strong (BBa _K206000)：a c a t t g a t t a t t t g c a c g g c g t c a c a c t t t g c t a t g c c a t a g c a a <SPAN class="light">g a t a g t</SPAN> c c a t a a g a t t a g c g g a t c c t a c c t g a c g c t t t t t a t c g c a a c t c t c t a c t g t t t c t c c a t a c c g t t t t t t t g g g c t a g c</P><P>BBa _K1070002：a c a t t g a t t a t t t g c a c g g c g t c a c a c t t t g c t a t g c c a t a g c a a <SPAN class="light">t a t a g t </SPAN>c c a t a a g a t t a g c g g a t c c t a c c t g a c g c t t t t t a t c g c a a c t c t c t a c t g t t t c t c c a t a c c g t t t t t t t g g g c t a g c</P><P>BBa _K1070003：a c a t t g a t t a t t t g c a c g g c g t c a c a c t t t g c t a t g c c a t a g c a a <SPAN class="light">g a t a a t</SPAN> c c a t a a g a t t a g c g g a t c c t a c c t g a c g c t t t t t a t c g c a a c t c t c t a c t g t t t c t c c a t a c c g t t t t t t t g g g c t a g c
							</P><P>Subsequently, we measured the fluorescence intensity of these promoters and related it to the actual promoter strength. The experimental results are shown in Figure 8.</P><DIV class="body_photo">Figure 8 The fluorescence intensity reached a stable stage after 60 min. The fluorescence intensity is measured with the inducer, L-arabinose at 1 mM, 37℃, 200 rpm
. The promoter strength is related to the relative fluorescence intensity compared to the control group (K206000 without the inducing of L-arabinose).</DIV><P>Then we fitted the actual strength and the predicted strength, as can be seen in figure 9, the coefficient of determination is 0.8924.</P><DIV class="body_photo">Figure 9 The correlation between experimental data and the strength predicted by our program.</DIV><H1>Future work</H1><P>Apply our algorithms to more species. Now Pro-decoder is designed expressly for prediction and evaluation of <I>E. coli</I> promoters, in the near future we'll study the transcription regulation mechanism of other species and try to apply our algorithms to an extended range of species.</P><P>Enhance promoter strength prediction accuracy. Because our experimental data is so limited, the weight of the spacer length and the motif similarity is roughly determined, which lead to a weak correlation between the promoter strength. In the future we hope to obtain more experimental data with regard to the effect of spacer length and motif similarity having on promoter strength so we can revise the weight coefficients  of the two factors and get more reliable results.</P><P>The next version of this part of our program will be able to analyze not only the promoters of <I>E. coli</I>, but other species such as <I>Bacillus subtilis</I>, we'll integrate the transcription factor binding site data of more species into our database and use PWM algorithm to predict the TFBS in the promoters.</P><H1>References</H1><DIV class="body_references">[1] Wösten, M., Eubacterial sigma‐factors. <I>FEMS microbiology reviews</I><B>1998</B>,<I> 22</I> (3), 127-150.</DIV><DIV class="body_references">[2] Shultzaberger, R. K.; Chen, Z.; Lewis, K. A.; Schneider, T. D., Anatomy of <I>E. coli</I> σ70 promoters. <I>Nucleic acids research</I><B>2007</B>, <I>35</I>  (3),771-788.</DIV><DIV class="body_references">[3] Paget, M.; Helmann, J. D., The sigma70 family of sigma factors.<I> Genome Biol</I><B>2003</B>,<I> 4 </I> (1),203.</DIV><DIV class="body_references">[4] Jensen, S. T.; Liu, X. S.; Zhou, Q.; Liu, J. S., Computational discovery of gene regulatory binding motifs: a Bayesian perspective. <I>Statistical Science</I><B>2004</B>, <I>19 </I>(1), 188-204.</DIV><DIV class="body_references">[5] Kämpke, T.; Kieninger, M.; Mecklenburg, M., Efficient primer design algorithms.<I> Bioinformatics</I><B>2001</B>,<I> 17 </I> (3),214-225.</DIV><DIV class="body_references">[6] (a) Rhodius, V. A.; Mutalik, V. K., Predicting strength and function for promoters of the <I>E. coli</I> alternative sigma factor, σE. <I> Proceedings of the National Academy of Sciences </I><B>2010</B>,  <I>107 </I> (7), 2854-2859; (b) Mulligan, M. E.; Brosius, J.; McClure, W. R., Characterization in vitro of the effect of spacer length on the activity of <I>E. coli</I> RNA polymerase at the TAC promoter.  <I>Journal of Biological Chemistry </I><B>1985</B>,  <I>260 </I> (6), 3529-3538; (c) Qureshi, S. A.; Jackson, S. P., sequences-Specific DNA Binding by the S. shibatae TFIIB Homolog, TFB, and Its Effect on Promoter Strength. <I>Molecular cell</I><B>1998</B>,<I> 1</I> (3), 389-400.</DIV><DIV class="body_references">[7] Kel, A. E.; Gößling, E.; Reuter, I.; Cheremushkin, E.; Kel-Margoulis, O. V.; Wingender, E., MATCHTM: a tool for searching transcription factor binding sites in DNA sequences. <I>Nucleic acids research</I><B>2003</B>, <I>31</I> (13), 3576-3579.</DIV><DIV class="body_references">[8] Deuschle, U.; Kammerer, W.; Gentz, R.; Bujard, H., Promoters of <I>E. coli</I>: a hierarchy of in vivo strength indicates alternate structures. <I>The EMBO journal</I><B> 1986</B>,<I> 5 </I>(11), 2987.</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:XMU_Software/Project/promoter">http://2013.igem.org/Team:XMU_Software/Project/promoter</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:XMU_Software/Project/promoter" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:XMU_Software/Project/promoter" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:XMU_Software/Project/promoter&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:XMU_Software/Project/promoter&amp;oldid=359507" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>