<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_XMU_Software_Project_protein"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:XMU_Software/Project/protein">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:XMU_Software/Project/protein&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU_Software/Project/protein&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU_Software/Project/protein&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:XMU_Software/Project/protein" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:XMU Software/Project/protein</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="xmu_body"><DIV id="xmu_software"><DIV id="project"><DIV class="navigation"><DIV><UL id="jsddm"><LI><A href="https://2013.igem.org/Team:XMU_Software">Home</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Project/promoter">Project</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Requirements">Requirements</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Outreach">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Notebook">Notebook</A></LI></UL></DIV></DIV><DIV id="project_s1"><DIV class="project_title">PROJECT</DIV><DIV class="project_introduce">Our project includes 2 independent software tools-the brick worker and E' NOTE. The former is a software suit for the evaluation and optimization of biobricks, i.e., promoter, RBS, protein coding sequences and terminator. E' NOTE is a web application serving as an assistant for experiments. Its useful functions such as experiments recording and experimental template customization make experimental process easier and more enjoyable.</DIV></DIV><DIV class="navi_text"><A id="project_navi0" class="navi0" href="https://2013.igem.org/Team:XMU_Software/Project/promoter" onmouseover="navi0();" onmouseout="navi01();" style="color:white">Promoter-decoder</A><A id="project_navi1" class="navi1" href="https://2013.igem.org/Team:XMU_Software/Project/rbs" onmouseover="navi1();" onmouseout="navi11();" style="color:white">RBS-decoder</A><A id="project_navi2" class="navi2" href="https://2013.igem.org/Team:XMU_Software/Project/terminator" onmouseover="navi2();" onmouseout="navi21();" style="color:white">Terminator</A><A id="project_navi3" class="navi3" href="https://2013.igem.org/Team:XMU_Software/Project/protein" onmouseover="navi3();" onmouseout="navi31();" style="color:white">SynoProteiner</A><A id="project_navi4" class="navi4" href="https://2013.igem.org/Team:XMU_Software/Project/enote" onmouseover="navi4();" onmouseout="navi41();" style="color:white">E' NOTE</A><A id="project_navi5" class="navi5" href="https://2013.igem.org/Team:XMU_Software/Project/achieve" onmouseover="navi5();" onmouseout="navi51();" style="color:white">Achievement</A><A id="project_navi6" class="navi6" href="javascript:void(0)" onmouseover="navi6();" onmouseout="navi61();" style="color:white">Back To Top</A></DIV><DIV id="protein"><DIV id="protein_m1"><DIV class="sep_title">SynoProteiner</DIV></DIV><DIV id="protein_m2"><H1>Abstract</H1><P>Our team mainly focuses on programming the software by genetic algorithm, evaluating both optimization of single codon and codon pair and hence determining the fittest optimized sequences for expression in heterologous host cell.</P><P>In addition, Synoproteiner protects enzyme cutting site. Users can choose corresponding cutting sites, not substituted during the optimization, so that the power of chosen enzyme cutting site won’t fail.</P><P>Apart from the optimization, we have two additional functions. One is the statistics analysis, which provides the numbers and the proportion of the codon in the original and optimized sequences, making the optimization easier to understand. The other is the prediction of the protein folding rate. The purpose of the prediction is to seek the law of the folding rate in general, computing a relatively accurate folding rate value of the optimized sequences for the users.</P><H1>Background</H1><H4>Synonymous codons and the efficiency</H4><P>Except methionine and tryptophan, all amino acids can be encoded by two to six synonymous codons, resulting from the degeneracy of the genetic code.<SUP>1</SUP>  However, unequal utilization of the synonymous condons leads to the phenomenon of codon usage bias, which is mainly due to natural selection, mutation and genetic drift.<SUP>2</SUP> According to related studies, codon usage bias has certain connection with gene expression level.<SUP>3</SUP> The larger the value of codon usage bias is, the higher gene expression will be. So the problem, how to substitute the synonymous codons aimed at raising the efficiency of gene expression and thus increasing the production of recombination protein in heterologous host cell, is expected to be addressed.</P><H4>Protein folding rate</H4><P>Protein is an important class of biological macromolecules. It is the main bearer of life activities and occupies a special position in vivo. Each protein has its own unique amino acid composition and sequences. Only when the amino acid chain is folded into the correct three-dimensional structure, will the protein have normal biological functions. Misfolded ones will not only lose its biological function but also even cause diseases such as mad cow disease, Alzheimer's syndrome, etc. The protein folding problem, an important biological question that the central dogma of molecular biology has not solved yet, has been listed as an important topic in twenty-first century. The folding mechanism of the protein is a challenging task, one of which is to determine factor influencing the folding rate. Although the answer can be found in a variety of biological experiments, such as various spectroscopy, mass spectrometry and nuclear magnetic resonance, these methods are time-consuming and costly. With the development of physics, mathematics, especially the progress of computer technology, how to apply a fast and accurate calculation method to predict protein folding rate attracts more and more attention.<SUP>4</SUP></P><H1>Introduction</H1><H4>Balance with single codon and codon pair</H4><P>Individual codon usage optimization has been attached importance to, taking Codon optimizer,<SUP>5</SUP> Gene Designer,<SUP>6</SUP> OPTIMIZER<SUP>7</SUP> for example. Subsequently, people found the effect of gene expression optimization cannot be perfect just by single codon optimization. Codon pair, namely the pair of k-th and (k+1)-th codons from the 5’ to 3’ end, is another crucial factor. Due to potential tRNA-tRNA steric interaction within the ribosomes,<SUP>8</SUP> the usage of rare condon pairs, which correlate with translation elongation, decrease protein translation rates.<SUP>9</SUP> Optimization of individual codon has an influence on the corresponding codon pair resulting in maybe-not-the-best codon pair optimization. In the same way, optimizing codon pair merely contributes to maybe-not-the-best single codon optimization. Therefore, it is a challenging way for us to apply a method considering and weighing the effects of single codon and codon pair optimization and thus make the whole best.</P><P>Our team focuses on evaluating both optimization of single codon and codon pair and thus selecting the best sequences for expression in heterologous host cell.</P><H4>Host Cell</H4><P>Considering <I>E. coli</I> and <I>S. cerevisiae</I> are the ideal hosts for recombinant proteinexpression, and Gram-positive bacterium <I>L. lactis</I> and methylotrophic yeast <I>P. pastoris</I> are also promising candidates for expressing recombinant proteins,<SUP>10</SUP> we attached importance to selecting these four kinds of bacterium as host cell to optimize the sequences.</P><H4>Enzyme cutting site protection</H4><P>To avoid deconstruction of enzyme cutting site and therefore incapability during substitution of the base, Synoproteiner succeeds in exchanging <A href="https://2013.igem.org/Team:XMU_Software/Project_protein/cutting_site" target="_blank">14 kinds of commonly used cutting sites </A>such as EcoRI, XbaI and so on.</P><P>Users can multi-select the enzyme cutting site that they want to stay unaltered and hence get an optimized sequence without changing selected region.</P><H4>Method of prediction</H4><P>In recent years, many researchers have made great efforts to explore the determinants of the folding rate, and various forecasting methods have been proposed. The existed prediction methods can be roughly divided into three categories.<SUP>11-12</SUP> The first one is based on the tertiary structure.<SUP>13-19</SUP> However, it takes lots of molecular experiments, expensive and in long period, to acquire the information of the tertiary structure, which fails to meet the demand of rapid prediction. The second category is based on the secondary structure.<SUP>20-24</SUP> This kind of method requires information of the secondary structure, similarly obtained by molecular experiments, or from the primary sequences prediction, but it will be limited by accuracy of the secondary structure prediction method. The last one is based on the primary structure,<SUP>25-34</SUP> which predicts the folding rate from amino acid sequences without most structure information.<SUP>4</SUP> And our prediction of the protein folding rate focuses on the last method.</P><H1>Algorithm</H1><H4>Part I—Optimization</H4><H3>Basic Table</H3><P>Based on the synonymous codons table, we calculate, we calculate function of single codon, function of codon pair  and the function of multi-objective codon optimization. The method aims at make the optimization of whole best by calculating the relative effect of single codon and codon pair.</P><H4>Fitness calculation<SUP>36</SUP></H4><H3>Fitness function：</H3><P>In the function,</P><P><I>cpi</I> is a value larger than zero, ranging from 10<SUP>-4</SUP> to 0.5,<I>fit<SUB>cp</SUB></I> (<I>g</I>) is the fitness function of the codon pair，<I>fit<SUB>sc</SUB></I> (<I>g</I>) is the fitness function of the single codon，<I>w</I> ( (<I>c</I> (<I>k</I>),<I>c</I> (<I>k+1</I>)) is the weight of codon pairs in sequences <I>g</I>,|<I>g</I>| is the length of encoding sequences, <I>c</I> (<I>k</I>) is k-th codon in the sequences, is the target ratio of k-th codon,  is the actual ratio of k-th codon in the sequences，the best value of <I>cpi</I> is 0.2 in the software.</P><P>In the function, the target ratio of k-th codon can be approximated by the equation below:</P><P>In the function, weight can be calculated by the equation below:</P><P>stands for the ratio of single codon c<SUB>k</SUB> in the complete genome'is the number of pair ( <I>c<SUB>i</SUB>,c<SUB>j</SUB></I> )  in high-expression genes, and high-expression genes are genes whose copy numbers of mRNA can be detected at least 20 per cell. </P><P>syn (c<SUB>k</SUB>) stands for the synonymous codon set related to <I>c<SUB>k</SUB></I>,equals to the number of amino acid encoded by <I>c<SUB>i</SUB></I> in the whole protein set. </P><H3>Calculation of Fitness</H3><P>NSGA-II algorithm applied<SUP>35</SUP>:</P><P>1.	Randomly initialize a population of coding sequences for target protein.</P><P>2.	Evaluate <I>fit<SUB>sc</SUB></I> (<I>g</I>) and <I>fit<SUB>sp</SUB></I> (<I>g</I>) fitness of each sequences in the population.</P><P>3.	Group the sequences into nondominated sets and rank the sets.</P><P>4.	Check termination criterion.</P><P>5.	If termination criterion is not satisfied, select the “fittest” sequences (top 50% of the population) as the parents for creation of offsprings via recombination and mutation.</P><P>6.	Combine the parents and offsprings to form a new population.</P><P>7.	Repeat steps 2 to 5 until termination criterion is satisfied.</P><P>The identification and ranking of nondominated sets in step 3 is performed via pair-wise comparison of the sequences' <I>fit<SUB>sc</SUB></I> (<I>g</I>) and <I>fit<SUB>sp</SUB></I> (<I>g</I>) fitness. For a given pair of sequences with fitness values expressed as (<I>fit<SUP>1</SUP><SUB>cp</SUB></I> (<I>g</I>),<I>fit<SUP>1</SUP><SUB>sc</SUB></I> (<I>g</I>)) and (<I>fit<SUP>2</SUP><SUB>cp</SUB></I> (<I>g</I>),<I>fit<SUP>2</SUP><SUB>sc</SUB></I> (<I>g</I>)), the domination status can be evaluated as follows:</P><P>•	If (<I>fit<SUP>1</SUP><SUB>sc</SUB></I> (<I>g</I>)&gt;<I>fit<SUP>2</SUP><SUB>sc</SUB></I> (<I>g</I>))and (<I>fit<SUP>1</SUP><SUB>cp</SUB></I> (<I>g</I>)&gt;=<I>fit<SUP>2</SUP><SUB>cp</SUB></I> (<I>g</I>)), sequences 1 dominates sequences 2.</P><P>•	If (<I>fit<SUP>1</SUP><SUB>sc</SUB></I> (<I>g</I>)&gt;=<I>fit<SUP>2</SUP><SUB>sc</SUB></I> (<I>g</I>)) and (<I>fit<SUP>1</SUP><SUB>cp</SUB></I> (<I>g</I>)&gt;<I>fit<SUP>2</SUP><SUB>cp</SUB></I> (<I>g</I>)), sequences 1 dominates sequences 2.</P><P>•	If  (<I>fit<SUP>1</SUP><SUB>sc</SUB></I> (<I>g</I>)&lt;<I>fit<SUP>2</SUP><SUB>sc</SUB></I> (<I>g</I>))and (<I>fit<SUP>1</SUP><SUB>cp</SUB></I> (<I>g</I>)&lt;=<I>fit<SUP>2</SUP><SUB>cp</SUB></I> (<I>g</I>)), sequences 2 dominates sequences 1.</P><P>•	If  (<I>fit<SUP>1</SUP><SUB>sc</SUB></I> (<I>g</I>)&lt;=<I>fit<SUP>2</SUP><SUB>sc</SUB></I> (<I>g</I>))and (<I>fit<SUP>1</SUP><SUB>cp</SUB></I> (<I>g</I>)&lt;<I>fit<SUP>2</SUP><SUB>cp</SUB></I> (<I>g</I>)), sequences 2 dominates sequences 1.</P><P>The process is showed in the figure below:</P><DIV class="body_photo">Figure 1</DIV><H3>Result:</H3><P>By this method, there are enough experimental data to prove the sequences optimized works. Xylose isomerase in <I>Bacillus stearothermophilus</I>, Xylose isomerase in <I>Streptomyces olivochromogenes</I> and L-arabinose isomerase in <I>Thermoanaerobacter mathranii</I>  all, the optimized ones, were highly expressed  in <I>Bacillus subtilis</I>. In addition, the activity of the optimized Aspergillusniger fungal amylase was enhanced to 400% compared with the original sequences in <I>A. niger</I>.<SUP>36</SUP></P><P><A href="https://2013.igem.org/Team:XMU_Software/Project_protein/Example_of_optimization" target="_blank">Link to the page of Example of optimization</A></P><H4>Part II—Prediction of protein folding rate</H4><P>In order to illustrate protein folding rate quantitatively, we determine the folding rate of 60 kinds of proteins as an experimental data set from literature and database<SUP>37</SUP>,  and information of the sequences comes from PBD and NCBI.</P><TABLE border="1" cellspacing="0" class="protein_table2"><TBODY><TR><TH>   protein   </TH><TH>Logarithm of the folding rate Ln(k<SUB>f</SUB>)</TH><TH>   protein   </TH><TH>Logarithm of the folding rate  Ln(k<SUB>f</SUB>)</TH><TH>   protein   </TH><TH>Logarithm of the folding rate  Ln(k<SUB>f</SUB>)</TH></TR><TR><TD>2PDD</TD><TD>9.8</TD><TD>1FKB</TD><TD>1.5</TD><TD>1RA9</TD><TD>-2.5</TD></TR><TR><TD>2ABD</TD><TD>6.6</TD><TD>2CI2</TD><TD>3.9</TD><TD>1QOP</TD><TD>-6.9</TD></TR><TR><TD>256B</TD><TD>12.2</TD><TD>1URN</TD><TD>5.8</TD><TD>1PHP</TD><TD>2.3</TD></TR><TR><TD>1IMQ</TD><TD>7.3</TD><TD>1APS</TD><TD>-1.5</TD><TD>1PHP</TD><TD>-3.5</TD></TR><TR><TD>1LMB</TD><TD>8.5</TD><TD>1RIS</TD><TD>5.9</TD><TD>1BNI</TD><TD>2.6</TD></TR><TR><TD>1WIT</TD><TD>0.4</TD><TD>1POH</TD><TD>2.7</TD><TD>2LZM</TD><TD>4.1</TD></TR><TR><TD>1TEN</TD><TD>1.1</TD><TD>1DIV</TD><TD>6.1</TD><TD>1UBQ</TD><TD>5.9</TD></TR><TR><TD>1SHG</TD><TD>1.4</TD><TD>2VIK</TD><TD>6.8</TD><TD>1SCE</TD><TD>4.2</TD></TR><TR><TD>1SRL</TD><TD>4</TD><TD>1A6N</TD><TD>1.1</TD><TD>1YCC</TD><TD>9.62</TD></TR><TR><TD>1PNJ</TD><TD>-1.1</TD><TD>1CEI</TD><TD>5.8</TD><TD>1VII</TD><TD>11.52</TD></TR><TR><TD>1SHF</TD><TD>4.5</TD><TD>2CRO</TD><TD>3.7</TD><TD>1NYF</TD><TD>4.54</TD></TR><TR><TD>1PSF</TD><TD>3.2</TD><TD>2A5E</TD><TD>3.5</TD><TD>2AIT</TD><TD>4.2</TD></TR><TR><TD>1CSP</TD><TD>7</TD><TD>1IFC</TD><TD>3.4</TD><TD>1PIN</TD><TD>9.44</TD></TR><TR><TD>1C9O</TD><TD>7.2</TD><TD>1EAL</TD><TD>1.3</TD><TD>1C8C</TD><TD>6.91</TD></TR><TR><TD>1G6P</TD><TD>6.3</TD><TD>1OPA</TD><TD>1.4</TD><TD>1BRS</TD><TD>3.4</TD></TR><TR><TD>1MJC</TD><TD>5.3</TD><TD>1CBI</TD><TD>-3.2</TD><TD>1UBQ</TD><TD>5.9</TD></TR><TR><TD>1LOP</TD><TD>6.6</TD><TD>1QOP</TD><TD>-2.5</TD><TD>3CHY</TD><TD>1</TD></TR><TR><TD>1C8C</TD><TD>7</TD><TD>1BRS</TD><TD>3.4</TD><TD>1BIN</TD><TD>2.6</TD></TR><TR><TD>1HZ6</TD><TD>4.1</TD><TD>3CHY</TD><TD>1</TD><TD>1SCE</TD><TD>4.2</TD></TR><TR><TD>1PGB</TD><TD>6</TD><TD>2RN2</TD><TD>0.1</TD><TD>1GXT</TD><TD>4.38</TD></TR></TBODY></TABLE><P>In order that the characteristic factors of the folding rate can be extracted from protein sequences, we introduced the Chou's pseudo amino acid composition concept.<SUP>38</SUP> According to the pseudo amino acid composition principle, the position information of protein sequences can be, to some extent, reflected by a group of serial correlation factors <I>θ<SUB>1</SUB>，θ<SUB>2</SUB> ，θ<SUB>3</SUB>……，θ<SUB>n</SUB></I> ,which is defined as follows:</P><P>in the function, <I>θ<SUB>1</SUB></I> is called the first-tier correlation factor that reflects the sequences order correlation between all the most contiguous residues along a protein chain (Fig. 2a), <I>θ<SUB>2</SUB></I> the second-tier correlation factor that reflects the sequences order correlation between all the second most contiguous residues (Fig.2b), <I>θ<SUB>3</SUB></I> the third-tier correlation factor that reflects the sequences order correlation between all the 3rd most contiguous residues (Fig.2c), and so forth.<SUP>38</SUP></P><DIV class="body_photo">Figure 2</DIV><P>the correlation function is given by<SUP>4</SUP>:</P><DIV class="body_photo">Θ(R<SUB>i</SUB>,R<SUB>j</SUB>)=|<I>H</I>(<I>R<SUB>j</SUB></I>)-<I>H</I>(<I>R<SUB>i</SUB></I>)|</DIV><P>where <I>H1(R<SUB>i</SUB>)), H2(R<SUB>i</SUB>), and M(R<SUB>i</SUB>)</I> are, respectively, the hydrophobicity value. Studies have shown that λ=10 will be the best predictor.<SUP>39</SUP> But there will be a large amount of calculation considering all possible situations—the 30 factors. We should select factors that can obtain the best prediction accuracy in least calculation. For that reason, we drew lessons from the literature<SUP>4</SUP> by using the method of Monte Carlo simulation and then 14 optimal characteristic factor were obtained. Other studies have indicated that the logarithm of the sequences length has a good correlation with folding rate, so Ln (L) will be the fifteenth factors. We apply SPSS software to calculate the coefficient of 15 factor by multivariate linear regression, and this will be the forecast formula of the rate of protein folding. We compared the experimental data and the predicted data and the results are as follows:</P><DIV class="body_photo">Figure 3 The comparision of the experimental and the predicted results</DIV><P>Through the test, our software succeeded in showing a relatively accurate folding rate value.</P><H1>Future work</H1><P>First of all, we will modify our software by advancing the program and the framework to improve its ability of concurrent computation and shorten the computing time.</P><P>Secondly, to accelerate the calculation, we may simplify the function of calculation by neglecting some term in our equations. However, considering the time spent on running program was extremely little, we will pay more attention on how to modify the equations for increasing the accuracy which maybe dramatically progress optimization result.</P><P>Thirdly, enriching the database is other way to improve our software. According to time-space tradeoff law, we could pre-process a bunch of sequences in common use to optimized one and save the result into our database. By assessing our data, investigators could select the optimized sequences for their synthesis. Then, users are required to feedback their result. When it collects enough information, our app will learn users’ bias therefore modify our optimizing function by some methods, like NSGA-II algorithm.</P><P>The specific points are listed as following:</P><P>1.	Shortening the computing time of the software.</P><P>2.	Expanding the range of the host cells.</P><P>3.	Improving bacterium's resistance to toxic molecule.</P><P>4.	Advancing existed paths of synthetic biology by the method.</P><P>5.	Designing new paths of synthetic biology by the method.</P><P>6.	Increasing the output of recombinant protein.</P><P>7.	Predicting the expression of heterologous gene in a new host cell.</P><P>8.	Considering more factors such as spiral structure in folding which influence the folding rate and thereby obtaining more accurate prediction rate.</P><P>9.	Providing a set of software tools for protein folding, especially in molecular dynamics simulation of protein folding.</P><H1>References</H1><DIV class="body_references">[1] Grantham, R.; Gautier, C.; Gouy, M.; Mercier, R.; Pave, A., Codon catalog usage and the genome hypothesis. <I>Nucleic acids research  </I><B>1980</B>, <I>8</I> (1),  197-197.</DIV><DIV class="body_references">[2] Hershberg, R.; Petrov, D. A., Selection on codon bias. <I> Annual review of genetics </I><B>2008</B>, <I> 42,  </I>287-299.</DIV><DIV class="body_references">[3] Gouy, M.; Gautier, C., Codon usage in bacteria: correlation with gene expressivity. <I> Nucleic acids research </I><B>1982</B>,  <I>10</I> (22), 7055-7074.</DIV><DIV class="body_references">[4] 郭建秀，饶妮妮, 刘广雄, 李杰, &amp; 王云鹤. 从氨基酸序列预测蛋白质折叠速率. 生物化学与生物物理进展 <I>Progress in Biochemistry and Biophysics</I><B>2010</B>, <I>37</I>(12): 1331~1338</DIV><DIV class="body_references">[5] Fuglsang, A., Codon optimizer: a freeware tool for codon optimization.  <I>Protein expression and purification  </I><B>2003</B>,  <I>31</I> (2),  247-249.</DIV><DIV class="body_references">[6] Villalobos, A.; Ness, J. E.; Gustafsson, C.; Minshull, J.; Govindarajan, S., Gene Designer: a synthetic biology tool for constructing artificial DNA segments. <I> Bmc Bioinformatics </I><B>2006</B>,  <I>7</I> (1),  285.</DIV><DIV class="body_references">[7]	Puigbò, P.; Guzmán, E.; Romeu, A.; Garcia-Vallvé, S., OPTIMIZER: a web server for optimizing the codon usage of DNA sequences.  <I>Nucleic acids research </I><B>2007</B>, <I> 35</I> (suppl 2),  W126-W131.</DIV><DIV class="body_references">[8] Smith, D.; Yarus, M., tRNA-tRNA interactions within cellular ribosomes.  <I>Proceedings of the National Academy of Sciences  </I><B>1989</B>,  <I>86 </I>  (12),4397-4401.</DIV><DIV class="body_references">[9] Coleman, J. R.; Papamichail, D.; Skiena, S.; Futcher, B.; Wimmer, E.; Mueller, S., Virus attenuation by genome-scale changes in codon pair bias.  <I>Science </I><B>2008</B>, <I> 320</I> (5884),  1784-1787.</DIV><DIV class="body_references">[10] (a) Wildt, S.; Gerngross, T. U., The humanization of N-glycosylation pathways in yeast. Nature Reviews Microbiology <B>2005</B>, <I>3</I> (2), 119-128; (b) Morello, E.; Bermudez-Humaran, L.; Llull, D.; Sole, V.; Miraglio, N.; Langella, P.; Poquet, I., Lactococcus lactis, an efficient cell factory for recombinant protein production and secretion.  <I>Journal of molecular microbiology and biotechnology  </I><B>2007</B>,  <I>14</I> (1-3),  48-58.</DIV><DIV class="body_references">[11] 郭建秀, 马彬广, 张红雨. 蛋白质折叠速率预测研究进展. 生物物理学报, <B>2006</B>, <I>22</I>(2):89-95 Guo J X, Ma B G, Zhang H Y. <I>Acta Biophys Sin</I>, <B>2006</B>, <I>22</I> (2):89-95.</DIV><DIV class="body_references">[12] Gromiha M M, Selvaraj S. Bioinformatics approaches for understanding and predicting protein folding rates. <I>Current Bioinformatics</I>, <B>2008</B>, <I>3</I>(1): 1-9</DIV><DIV class="body_references">[13] Plaxco K W, Simons K T, Baker D. Contact order, transition state placement and the refolding rates of single domain proteins. <I>J MolBiol</I>, <B>1998</B>, <I>277</I>(4): 985-994</DIV><DIV class="body_references">[14] Gromiha M M, Selvaraj S. Comparison between long-range interactions and contact order in determining the folding rate of two-state proteins: application of long-range order to folding rate prediction. <I>J Mol Biol</I>, <B>2001</B>, <I>310</I>(1): 27-32</DIV><DIV class="body_references">[15] Zhou H, Zhou Y. Folding rate prediction using total contact distance. <I>Biophys J</I>, <B>2002</B>, <I>82</I>(1): 458-463</DIV><DIV class="body_references"> [16] Nölting B, Schälike W, Hampel P, <I>et al</I>. Structural determinants of the rate of protein folding.<I> J Theor Biol</I>, <B>2003</B>, <I>223</I>(3): 299-307</DIV><DIV class="body_references"> [17] Weikl T R, Dill K A. Folding kinetics of two-state proteins: Effect of circularization, permutation, and crosslinks. <I>J Mol Biol</I>, <B>2003</B>,<I>332</I>(4): 953-963</DIV><DIV class="body_references"> [18] Ivankov D N, Garbuzynskiy S O, Alm E, et al. Contact order revisited: influence of protein size on the folding rate. <I>Protein Sci</I>,<B>2003</B>, <I>12</I>(9): 2057-2062</DIV><DIV class="body_references"> [19] Mirny L, Shakhnovich E. Protein folding theory: from lattice to all-atom models. <I>Annu Rev Biophys Biomol Struct</I>, <B>2001</B>, <I>30</I> (1):361-396</DIV><DIV class="body_references"> [20] Gong H, Isom D G, Srinivasan R, et al. Local secondary structure content predicts folding rates for simple two-state proteins. <I>J MolBiol</I>, <B>2003</B>, <I>327</I>(5): 1149-1154</DIV><DIV class="body_references"> [21] Ivankov D N, Finkelstein A V. Prediction of protein folding rates from the amino acid sequences-predicted secondary structure. <I>Proc Nat Acad Sci USA</I>, <B>2004</B>, <I>101</I>(24): 8942-8944</DIV><DIV class="body_references"> [22] Fleming P J, Gong H P, Rose G D. Secondary structure determines protein topology. <I>Protein Sci</I>, <B>2006</B>, <I>15</I>(8): 1829-1834</DIV><DIV class="body_references"> [23] Huang J T, Cheng J P, Chen H. Secondary structure length as a determinant of folding rate of proteins with two- and three-state kinetics. <I>Proteins</I>, <B>2007</B>, <I>67</I>(1): 12-17</DIV><DIV class="body_references"> [24] Prabhu N P, Bhuyan A K. Prediction of folding rates of small proteins: empirical relations based on length, secondary structure content, residue type, and stability. <I>Biochemistry</I>,<B> 2006</B>, <I>45</I> (11):3805-3812</DIV><DIV class="body_references"> [25] Shao H, Peng Y, Zeng Z H. A simple parameter relating sequenceswith folding rates of small helical proteins. <I>Protein Pept Lett</I>, <B>2003</B>,<I>10</I>(3): 277-280</DIV><DIV class="body_references"> [26] Galzitskaya O V, Garbuzynskiy S O, Ivankov D N, et al. Chainlength is the main determinant of the folding rate for proteins withthree-state folding kinetics. <I>Proteins</I>, <B>2003</B>, <I>51</I>(2): 162-166</DIV><DIV class="body_references"> [27] Huang J T, Jing T. Amino acid sequences predicts folding rate for middle-size two-state proteins. <I>Proteins</I>, <B>2006</B>, <I>63</I>(3): 551-554</DIV><DIV class="body_references"> [28] Gromiha M M. A statistical model for predicting protein folding rates from amino acid sequences with structural class information.<I>J Chem Inf Model</I>, <B>2005</B>, <I>45</I>(2): 494-501</DIV><DIV class="body_references"> [29] Ma B G, Guo J X, Zhang H Y. Direct correlation between proteins'folding rates and their amino acid compositions: an ab initio foldingrate prediction. <I>Proteins</I>, <B>2006</B>, <I>65</I>(2): 362-372</DIV><DIV class="body_references"> [30] Gromiha M M, Thangakani A M, Selvaraj S. FOLD-RATE:prediction of protein folding rates from amino acid sequences.<I>Nucleic Acids Res</I>, <B>2006</B>, <I>34</I>(suppl_2): 70-74.</DIV><DIV class="body_references"> [31] OuYang Z, Liang J. Predicting protein folding rates from geometric contact and amino acid sequences. <I>Protein Sci</I>, <B>2008</B>, <I>17</I>(7): 1256-1263</DIV><DIV class="body_references"> [32] Huang L T, Gromiha M M. Analysis and prediction of proteinfolding rates using quadratic responde surface models. <I>J ComputChem</I>, <B>2008</B>, <I>29</I>(10): 1675-1683</DIV><DIV class="body_references"> [33] Shen H B, Song J N, Chou K C. Prediction of protein folding ratesfrom primary sequences by fusing multiple sequential features.<I>J Biomedical Science and Engineering</I>, <B>2009</B>,<I> 2</I>(3): 136-143</DIV><DIV class="body_references"> [34] Jiang Y, Iglinski P, Kurgan L. Prediction of protein folding ratesfrom primary sequences using hybrid sequences representation.<I>J Comput Chem</I>, <B>2009</B>, <I>30</I>(5): 772-783</DIV><DIV class="body_references">[35] Chung, B.; Lee, D.-Y., Computational codon optimization of synthetic gene for protein expression.<I> BMC systems biology </I><B>2012</B>,  <I>6</I> (1),  134.</DIV><DIV class="body_references">[36] 	帝斯曼知识产权资产管理有限. 公司实现改进的多肽表达的方法: 中国, 200780024670.5[P]. 2009-07-22</DIV><DIV class="body_references"> [37]	Gromiha, M. M.; Thangakani, A. M.; Selvaraj, S., FOLD-RATE: prediction of protein folding rates from amino acid sequences.<I> Nucleic acids research</I><B>2006</B>, <I>34 </I>(suppl 2), W70-W74. </DIV><DIV class="body_references"> [38] Chou, K. C., Prediction of protein cellular attributes using pseudo‐amino acid composition. <I>Proteins: Structure, Function, and Bioinformatics</I><B>2001</B>, <I>43</I> (3), 246-255. </DIV><DIV class="body_references"> [39]	Galzitskaya, O. V.; Garbuzynskiy, S. O.; Ivankov, D. N.; Finkelstein, A. V., Chain length is the main determinant of the folding rate for proteins with three‐state folding kinetics. <I>Proteins: Structure, Function, and Bioinformatics</I><B>2003</B>, <I>51 </I>(2), 162-166. </DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:XMU_Software/Project/protein">http://2013.igem.org/Team:XMU_Software/Project/protein</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:XMU_Software/Project/protein" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:XMU_Software/Project/protein" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:XMU_Software/Project/protein&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:XMU_Software/Project/protein&amp;oldid=359557" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>