<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_XMU_Software_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:XMU_Software/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:XMU_Software/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU_Software/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:XMU_Software/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:XMU_Software/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:XMU Software/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="xmu_body"><DIV id="xmu_software"><DIV id="notebook"><DIV class="navigation"><DIV><UL id="jsddm"><LI><A href="https://2013.igem.org/Team:XMU_Software">Home</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Project/promoter">Project</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Requirements">Requirements</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Outreach">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:XMU_Software/Notebook">Notebook</A></LI></UL></DIV></DIV><DIV id="notebook_s1"><DIV class="notebook_title">NOTEBOOK</DIV></DIV><DIV class="navi_text"><A id="notebook_navi1" class="navi2" href="#_tool" onmouseover="navi1();" onmouseout="navi11();" style="color:white">Tools</A><A id="notebook_navi2" class="navi3" href="#_journal" onmouseover="navi2();" onmouseout="navi21();" style="color:white">Journal</A><A id="notebook_navi3" class="navi4" href="#_protocols" onmouseover="navi3();" onmouseout="navi31();" style="color:white">Protocols</A><A id="notebook_navi4" class="navi5" href="#_experiment" onmouseover="navi4();" onmouseout="navi41();" style="color:white">Lab Record</A><A id="notebook_navi5" class="navi6" href="javascript:void(0)" onmouseover="navi5();" onmouseout="navi51();" style="color:white">Back To Top</A></DIV><DIV id="notebook_s2"><DIV id="notebook_s2m1"><DIV class="sep_title">Tools</DIV></DIV><DIV id="notebook_s2m2"><DIV id="notebook_s2m2n2"><DIV id="notebook_s2m2t1" class="notebook_s2t3">XMind</DIV><DIV id="notebook_s2m2t2" class="notebook_s2t3">PowerPoint</DIV><DIV id="notebook_s2m2t3" class="notebook_s2t3">Robert's Rules of Order</DIV></DIV><DIV id="notebook_s2m2n4"><DIV id="notebook_s2m2a1" class="notebook_s2t4">XMind</DIV><DIV id="notebook_s2m2a2" class="notebook_s2t5">The software was used for record our brainstorm and make new ideas. It is free! And it has so many models for new users to build up their own models.</DIV><DIV id="notebook_s2m2a3" class="notebook_s2t4">PowerPoint</DIV><DIV id="notebook_s2m2a4" class="notebook_s2t5">That Powerpoint is used for reporting our progress and giving presentations. Thus we practice it and study how to tell a funny story.</DIV><DIV id="notebook_s2m2a5" class="notebook_s2t4">Robert's Rules of Order</DIV><DIV id="notebook_s2m2a6" class="notebook_s2t5">This book guided us on how to make a meeting effective. And it really worth the hornor.</DIV></DIV><DIV id="notebook_s2m2n5">TOOLS</DIV></DIV></DIV><DIV id="notebook_s3"><DIV id="notebook_s3m1"><DIV class="sep_title">Journal</DIV></DIV><DIV id="notebook_s3m2"><DIV id="notebook_s3m2n1" onclick="march();"><DIV class="notebook_s3t1">MARCH</DIV><DIV class="notebook_s3t2">2013.3</DIV></DIV><DIV id="notebook_s3m2n2"><DIV class="notebook_s3t3"><DIV id="note1"><H3>3.30.2013</H3><P>Chen: Experimental workspace can automatically decide whether it is a personal account or a business account of networking and the workspace affords the download link of those relevant software. Can we get those software embedded? How to help our customers find out the software they want quickly? (By searching them?) It is also important to process data and generate lab reports and logs in the workspace. </P><P>Future plan: We plan to add software in the software library, optimize the software's functions and landscaping interface. Besides, we want to afford real time data while monitoring on different platforms. And how about syncing to the cloud? </P><P>Qiu: Easy Express. The software is designed to save time for users. Then they just need to stay on creating new ideas and putting a possible idea into practice. In addition, unified expressions get researchers to understand other's work easily. Its system's function: drawing, display sequences, dynamic display structure of the plasmid and parts of the experiments, shows the meaning of each graph and table, guiding and help. The challenge for us is that whether we can dynamic display the data? If we standardization of the presentation, the software may be rejected by other teams for its boredom standardization separate kinds of efforts may be impossible. This software told you that you don't need to learn MAYA and other magic software, it can include their functions. Will it be possible? (Perhaps unattainable for now) The idea may be realized by attaching the database to the software and affording individuation information search for our users. </P><P>Wang: Document management and real time monitoring can help you on scientific literature management, take notes and share your ideas. It can also automatic detection and data recording of streamlined repetition, design shortcut keys to improved operational efficiency. Thus, users can use graph visualization to understand the data easily. </P><P>Zhang: She had done a lot of work on directional optimization of BioBricks. </P><P>Purpose: Inputing environmental parameters include special loci genes, the expected expression level and chooses candidate BioBricks. Optimize and find out the best sequences under given conditions and setting number of evolutionary (n) chooses the BioBricks, and the system will guide users until the given conditions are met, variation to generate different sequencess. Copy, retain or reject BioBricks according to the matched-degree, this step is parallel to natural selection. Circulation stops after n times circulations, output the best sequences. The problems are how to make sure that the genotype equals its phenotype? How to define the f(x) that we want? And now we need to read some books about the genetic algorithm to get the way. </P><P>Summary: Now we have two ideas. One is to build a platform for researchers that focusing on synthetic biology (aim to increase their efficiency). Former researchers have already done some researches about it, so it might be a rat race. </P><P>Solve synthetic biology problems with GA. We need to supplement relevant basic knowledge of GA. So, studying CLOTHO developed by Berkeley and other universities and try to find out some flesh ideas and make a research to find out what front-line researchers need. We also need to study pertinent software and GA, protein quaternary structure and knowledge about directional optimization, read more books and literature to make sure the possibility of the idea. GA is too difficult for some of us. We still need to get come up with some novel ideas. </P><P>OK, for so many tasks, work in groups and share additional ideas~! </P></DIV><DIV class="notebook_s3m2t1"><DIV class="notebook_s3t5">Meeting Record</DIV><DIV class="notebook_s3t4">3.30</DIV></DIV></DIV></DIV><DIV id="notebook_s3m2n3" onclick="april();"><DIV class="notebook_s3t1">APRIL</DIV><DIV class="notebook_s3t2">2013.4</DIV></DIV><DIV id="notebook_s3m2n4"><DIV class="notebook_s3t3"><DIV id="note2"><H3>2013.4.7</H3><P>Zhang had learnt GA. Also, we read the news about Calgary and Newcastle, SUSTC（calculate terminator efficiency by a score system）. Code is linked up with fitness, which influences the secondary structure of protein. For example, the changing of a-tail, stem loop and T-tail, we can also find it in the project of SUSTC. Additionally, she also read papers on how to calculate f(x) &amp; take notes when learn new things. SUSTC' fault: Just one function, it may not catch everybody's eyes. She plans to change promoter so as to confirm the system efficiency. Jianxing schedule to learn neural network, after that, he will change the source code to be used on today's system. Huang Xin and Huang Jian now know just a little about Matlab, but they think the code can be changed easily. Wang learned Matlab as well. Tang Chun read some books about system efficiency and downloaded papers to share with others. From N-terminal domain rule, he noticed that choose the parameters is a key point for our project. </P><P>Next week's task had been written on the wall. And we suggest everybody to use PowerPoint to explain their ideas. </P></DIV><DIV id="note3"><H3>2013.4.14</H3><P>Specific our goals: For most of us aren't familiar with GA. We misunderstand the idea at the beginning. We determined to find the best fit BioBricks through genetic recombination and variation inside the BioBricks, and we evaluating them by our algorithm. Compare with other team' work: EbolGENTS (2008, Calgary) and Evolutionary Algorithm (2008, Newcastle University). Both of them had used GA, however, they construct a system with given BioBricks just like build blocks, after which they find the best system in those &quot;structures&quot;. As for our group, we focus on genetic recombination and variation inside the BioBricks to make further ones, and search for the best with given conditions. </P><P>Reduce the degree of difficulty: it may be a huge project to ensure phenotype and fitness result from mutations of the base sequences in the whole BioBricks. So we decided to narrow down and aim at optimizing the decision of particular bases, promoter, terminator and RBS locus. Yijuan had mentioned that SUSTC had done research on computational efficiency of the Terminator's gene sequences. </P><P>Core problem: How to ensure phenotype from genotype? How to find the f(x) meet our goal? Fitness function: three ways had been chosen to solve it: first, search for relevant literatures; read the project offered by Professor Fang named “microorganism’s choice of the optimal conditions&quot;; if possible, our wet lab should give us a hand on affording real time data and experimental simulation to ensure f(x). </P><P>Current task: Check the project of iGEM &quot;calculate terminators efficiency by the sequences of the terminators”; all of us need to take a look at SUSTC' work and discuss the next time. Learning Matlab and GA! </P><P>Annotations: We had set sail now! All of us need to learn SUSTC's work on ensuring genotype by phenotype and discuss the next time. We are badly in need of professor's help! </P><P>Try to find more idea! Stay hungry! </P></DIV><DIV id="note4"><H3>2013.4.21</H3><P>This week we had a summary of the work we had done before and nowadays we need to fix it. GA as an algorithm can work on several biological questions. And several new ways and improvement had been found out to solve the problem. Take an example, GASCO is a new algorithm that improves the answer of GA. </P><P>We start the preparation for the human practice. That may be of great importance. We had to distribute the job and explicit the goal so as to lift efficiency. </P><P>Neural network can be used in it. </P></DIV><DIV id="note5"><H3>2013.4.28</H3><P>Zhang: After reading several papers, she got down to SBOL. She got some new ideas on our program. </P><P>Han Tao &amp; Huang Jiang &amp; Huang Xin: They had successly got a software that can simulate the situation. However, software remains some problem to be solved. We had better make a new one, do not use this one, for we can use Matlab. That will be a good idea. </P><P>Platform and relevant work went well. </P></DIV><DIV class="notebook_s3m2t2"><DIV class="notebook_s3t5">Meeting Record</DIV><DIV class="notebook_s3t4">4.7</DIV><DIV class="notebook_s3t4">4.14</DIV><DIV class="notebook_s3t4">4.21</DIV><DIV class="notebook_s3t4">4.28</DIV></DIV></DIV></DIV><DIV id="notebook_s3m2n5" onclick="may();"><DIV class="notebook_s3t1">MAY</DIV><DIV class="notebook_s3t2">2013.5</DIV></DIV><DIV id="notebook_s3m2n6"><DIV class="notebook_s3t3"><DIV id="note6"><H3>2013.5.5</H3><P>Most of us take a rest after long hours.</P></DIV><DIV id="note7"><H3>2013.5.12</H3><P>This week, we will be divided into two groups, one named design group and another one Matlab group. For the first group, they need to finish most of the work on the design. Our logo, clothes and slogan will be their works. What’s more, the platform and the wiki to show our achievement require them to try their best to do it. For the second group, they must continue working on the software and try to improve existing algorithms. </P><P>We had got some ideas on human practice and the next step was to check if they can be carried. </P></DIV><DIV id="note8"><H3>2013.5.19</H3><P>Design group:the wiki has been designed and we are now considering combining wet lab's </P>Design group: The wiki has been designed and we are now considering combining wet lab's work with ours. Han Chen had done a lot work in the wet lab and found out what problems they met. We will finish the Wiki in the next few days and afford some tools for users. <P>Matlab group: They had already written part of our software and challenging it. We test it and find that its interface is not friendly and difficult for users to master it at the first sight. And we will call for help from the design group. Also, for the semifinals is nearby. We need to take a break. </P></DIV><DIV id="note9"><H3>2013.5.26</H3><P>Huang Jiang &amp;Huang Xin: They learn C++ and try to make a program. They learn different kinds of arithmetic and compare them to find what we need. GASCO and some additional algorithms are inconsideration. </P><P>Zhang: She had already found software that can help us to solve those problems. In additon, she also focuses on SBOL and finds out the way to solve some easy problems we put forward before. </P><P>Zhang Shiqi &amp; Wang Chuyue: They had already used GA to make a program and test it. However, we think they need to make more progress so as to up to standard. </P><P>Qiu Likai &amp; Chen Yuezeng: The software had been finished and they are designing the wiki and writing codons. That might be finished in two months.
		</P><P>Huang Jianxing: He read and found some papers of Codon usage bias. He should have been better. NCBI is of great importance if we write a software ourselves. And we need to know how to make use of the data in NCBI. </P></DIV><DIV class="notebook_s3m2t2"><DIV class="notebook_s3t5">Meeting Record</DIV><DIV class="notebook_s3t4">5.5</DIV><DIV class="notebook_s3t4">5.12</DIV><DIV class="notebook_s3t4">5.19</DIV><DIV class="notebook_s3t4">5.26</DIV></DIV></DIV></DIV><DIV id="notebook_s3m2n7" onclick="june();"><DIV class="notebook_s3t1">JUNE</DIV><DIV class="notebook_s3t2">2013.6</DIV></DIV><DIV id="notebook_s3m2n8"><DIV class="notebook_s3t3"><DIV id="note10"><H3>2013.6.2</H3><P>Lin shen: Codon bias database is important for our works. We need to use Latin to search for the correct sequences for the organisms. NCBI had been downloaded and can be imported. </P><P>Chen Han: Everything goes well. He starts to prepare for the human practice. </P><P>Zhang Yijuan: She had already explained the meaning of RBS. And figure out the way that how to evaluate it. That idea had been written and be used in our software. </P><P>Hon Yuming: He drew something and had got the idea of doing so. The logo design is the handle. And the logo design influences the next step and we shall finish it ASA possible.</P></DIV><DIV id="note11"><H3>2013.6.9</H3><P>Exam week. So call off.</P></DIV><DIV id="note12"><H3>2013.6.16</H3><P>Exam week. So call off.</P></DIV><DIV id="note13"><H3>2013.6.23</H3><P>Zhang Yijuan had distinguish general transcription factors, recognition of inductivity transcription. Improving by similar software. Then she will import the database and check them one by one. So we need to know how to call PWM from the database. On sigma factor, statistic PSFM model by using a neural network will be a good way. We know that the best length requires a certain distance. No ready-made matrix, we had to write one. She will read papers about RBS following. Huang Jianxing &amp; Huang Jiang &amp; Lin shen &amp; Han Tao had read to know the basic knowledge. Cheng Han finds papers on Sigma 70 &amp; do some work on how to build an experience platform that suits user’s need. He improves and redoes the platform to make a new model. </P><P>Team member Yijuan Zhang reported our software project (mainly for evaluating of promoters) to Mrs. Wu. We received several suggestions from Mrs. Wu. To enhance the accuracy of our software work, we are supposed to pay attention to the false positives raised during recognition of promoters. And to further improve our promoter-evaluation program, Mrs. Wu suggested that we do some research on some distinct promoters besides <I>E. coli</I> promoters. On the entire Wu applauded progress we had achieved and encouraged our software team to do our wonderful jobs as always. </P><P>Summer camp, garden party, Cathedra of synthetic biology, card game, Foldit and lantern riddles will help us to introduce iGEM to people. Monetary reimbursement includes the registration fee. Numberous matters need attention: like travel expense, laboratory items, office supplies, car fare. Printing and copying cost should label the amount of money. Qiu &amp; Chen made the concept map and interface of our software. Huang Xin makes use of the software that we wrote before. Then he will look for more papers of sigma 70. </P></DIV><DIV id="note14"><H3>2013.6.30</H3><P>Arithmetic software team on evaluating and optimizing BioBricks held a meeting and report our progress and current problems to former iGEMer Youbin Mo. Mo listened to our program with extreme concern and appreciation and taught one of our team members to use an artificial neural network to build weight position matrix for Sigma motifs. We discussed the difficulties in front of us, which is mainly about the refinement of our algorithm and how to store and use data from database to RDB. We decide to concentrate on improving the algorithm. In addition, Mo suggested that we start to present our slides in English and make a summary of our theories so as to post them on our wiki. </P><P>Easy Note is the software based on Web, which is designed to convenient biosynthetic researchers to do some experimental record and query information. After registration and login, users can do some rudimentary records, related database and background program and so on. </P><P>Because the interface is too monotonous, and the function is too simple, we plan to design a succinctly and naturally interface and add query information, timing function, set, import and export document in the resulting month. </P></DIV><DIV class="notebook_s3m2t2"><DIV class="notebook_s3t5">Meeting Record</DIV><DIV class="notebook_s3t4">6.2</DIV><DIV class="notebook_s3t4">6.9</DIV><DIV class="notebook_s3t4">6.16</DIV><DIV class="notebook_s3t4">6.23</DIV><DIV class="notebook_s3t4">6.30</DIV></DIV></DIV></DIV><DIV id="notebook_s3m2n9" onclick="july();"><DIV class="notebook_s3t1">JULY</DIV><DIV class="notebook_s3t2">2013.7</DIV></DIV><DIV id="notebook_s3m2n10"><DIV class="notebook_s3t3"><DIV id="note15"><H3>2013.7.7</H3><P>Han designed small icon so that the interface will be more beautiful. Small icon included addition, delete, document, tool, and set, text, picture and so on. </P><P>Qiu had successfully realized login in and registry database. At the same time, he coded the data board. Chen had written code for the interface including addition, delete, document, tool function. Nonetheless, it is still not enough perfect because of various reasons. What to do next will be discussed at tonight’s meeting. </P><P>Huang Xin learned that we can deal with the data from the database using Matlab. </P><P>Han Tao refined our algorithm to evaluate promoter strength on Java and try to run some existing software designed to improve protein coding sequencess. </P><P>Zhang Yijuan started to learn theories on RBS and its prediction. By looking up to literatures, we find several factors which can influence the RBS strength including the SD sequences, spacer length, initiation codon and secondary structure. And we come up with algorithms similar to the algorithms used in the promoter part which take 3 factors, namely, SD sequences, spacer length, start codon into account. And we’ll study the impact of RBS structure on its strength next week. </P><P>Rao has built a database in access containing all the transcription factors’ binding sites and we are now discussing how to use this database in our software.</P></DIV><DIV id="note16"><H3>2013.7.14</H3><P>To improve the login interface and the registry interface, Chen designed a fresh style’s interface which had divided into four sections and had used an irregular design. When clicking one section, this section enlarged so that the interaction effect was better. Kai made the numerous efforts to improve the database. Because the database is problematic, he spent a lot of time for it so he stayed up late sometimes. Next week, we will improve the function and interface. </P><P>Zhang figured out the algorithm to calculate the binding free energy between RBS mRNA and 16s rRNA and Han Tao started to write code for it at once. Now we have a simple program for calculating the binding free energy. So far we have worked out two algorithms to evaluate RBS strength. </P></DIV><DIV id="note17"><H3>2013.7.21</H3><P>Zhang put forward 2 methods to evaluate RBS strength. The basic method is to use the PWM (position weight matrix) of the SD sequences and the spacer length between the SD sequences and start codon to calculate the relative strength. The second is to calculate the binding free energy between the RBS mRNA and the 16s rRNA in the ribosome. </P><P>Han Tao, Huang Xin and Rao are working together on the program code. </P><P>Qiu had finished the record module that we can add text, photo, table and so on. Because the record interface isn’t too concise, we should improve in the next week. At the same time, import and export function had accomplished. In the process of programming, we met with numerous difficulties. For completing the task better in the subsequences stage, Chen studied the jQuery. Next week, we will continue to work hard. </P></DIV><DIV id="note18"><H3>2013.7.28</H3><P>HanTao began to deal with the GUI of the promoter evaluation part of our project. </P><P>Rao decides to inlay the BioBrick sequences data in our software as a text file.</P><P>ZhangYijuan had put forward the problem of how to determine the weight of motif similarity and spacer length score in the strength evaluation algorithm .After consulting with QiuRuosang, Zhang and QiuRuosang began to design experiments for the data needed to modify as well as validate our algorithm to evaluate both the promoter and RBS strength.</P><P>After the end of the third semester in this week, we invest a lot of time for the software production. Qiu had finished and modified the function of adding a photograph, table, step and the text. Difficulties in the database exceed we imagine. With increasing a lot of novel effect, login interface had been modified by Chen. Next week, we will tidy up the code for easier to read and improve the existing functionality.s</P></DIV><DIV class="notebook_s3m2t2"><DIV class="notebook_s3t5">Meeting Record</DIV><DIV class="notebook_s3t4">7.7</DIV><DIV class="notebook_s3t4">7.14</DIV><DIV class="notebook_s3t4">7.21</DIV><DIV class="notebook_s3t4">7.28</DIV></DIV></DIV></DIV><DIV id="notebook_s3m2n11" onclick="august();"><DIV class="notebook_s3t1">AUGUST</DIV><DIV class="notebook_s3t2">2013.8</DIV></DIV><DIV id="notebook_s3m2n12"><DIV class="notebook_s3t3"><DIV id="note19"><P>Before, we have been entangled with the problem that how to get the position frequency matrix of Sigma promoter motifs (we have only obtained the data of sigma 70 and sigma E promoters from literatures). This week, after deliberate discussion, we come up with an idea that we can use the consensus sequences provided in literatures and promoter sequencess in RDB to calculate the matrix by ourselves. Han Tao and Zhang are in charge with this little program and now we have worked out the frame of the program and have tested it using the data of sigma 32. There is more work to be done to refine the program and make our position frequency matrix more reliable. </P><P>Members of our software team are doing experiments in the lab this week. </P><P>Group of Protein experienced a tough time these weeks. In 21th, July, we decided to apply the algorithm of patent 200780024670.5 in further work. However, concern on sort of using the patent delayed our pace forward. </P><P>After communication with Prof. Fang this week, we stick to the direction and all neglected tasks are being undertaken. Unfortunately, things don’t go smoothly as expected. </P><P>Jianxing Huang employed data from NCBI (database) and paid enormous efforts on changing sequences files to FASTE files. He exerted himself in making the algorithm programmable by understanding the patient in detail. </P><P>Tao Han went home, but he was still making attempts to program the available parts. </P><P>Zach Lin went home as well, in which the documents cannot be downloaded, only doing his endeavor to read limited documentary sources online. </P><P>Next week: </P><P>Jianxing Huang will find out and download all those parameters mentioned in the algorithm from the database. </P><P>Examples will be made by Jiang Huang to explain the algorithm and make the main idea of the program possible. </P><P>Tao Han tends to begin compiling the computer program. </P><P>Zach Lin wants to blend an innovative point into the algorithm by scanning lots of papers. </P></DIV><DIV id="note20"><P>The week we had finished the elementary function so that user can use our software by registering. We named the software for “E’Note”. User can add and delete an experiment, add step, add text, table, photo and so on. </P><P>Qiu had put the software on the line. Chen had finished one of the experiment formwork that the user can utilize directly. However, the tool hadn’t finished which we will work hard next week. </P></DIV><DIV id="note21"><P> Frist, we had uploaded the wiki of team project descriptions to the iGEM server. It is beautiful but half-baked, so we decide to make the wiki after the platform is done, maybe in mid-August. Second, we designed a series of models of experimental record, and we are adding them to the platform as a different function. Users can add a model for recording experimental data efficiently. Third, the innovative design of platform’s interface is made. We will improve the interface according to it. </P></DIV><DIV id="note22"><P> Most of the software function had been completed and the users can use it. We will improve some function that not good enough next week. Before the ensuring Wednesday, we will give instruction and the users can use the software more efficient. We plan to make a community post at iGEM official page and received responses from other iGEM teams who use our software.</P></DIV><DIV id="note23"><P>That was a funny day. We discussed how we can make full use of things to make a mascot. Additionally, we sat around and have a brainstorm, after which we got the idea of how to record the introduction movie. Together we put forward a lot of helpful ideas and funny tips. Hong planned to draw more than 1000 pictures! That was amazing! </P><P>And for distinct parts of our wiki, the job was distributed to five members and each one had to draw the scheme quickly. After that, they need to discuss with our designer Hong to know whether the idea can be achieved or not. </P></DIV><DIV class="notebook_s3m2t2"><DIV class="notebook_s3t5">Meeting Record</DIV><DIV class="notebook_s3t4">8.1</DIV><DIV class="notebook_s3t4">8.8</DIV><DIV class="notebook_s3t4">8.15</DIV><DIV class="notebook_s3t4">8.22</DIV><DIV class="notebook_s3t4">8.29</DIV></DIV></DIV></DIV><DIV id="notebook_s3m2n13" onclick="september();"><DIV class="notebook_s3t1">SEPTEMBER</DIV><DIV class="notebook_s3t2">2013.9</DIV></DIV><DIV id="notebook_s3m2n14"><DIV class="notebook_s3t3"><DIV id="note24"><P> For this is the last month for us to prepare for the iGEM. We are now ready to start preparing for our speaking. That was really a hard work. Thus we need to know what other teams had done before. So we downloaded their videos and carefully watching them. That may help us on make a good presentation.</P></DIV><DIV id="note25"><P> We have got the idea on how to make the PPT for our presentation. Besides, this week, we also made some instruction books for users to use our software. We also worked on the wiki to make sure that we will be able to show it to teams from all over the world. Our designer, Hong, had drew so many graphs to beautify our wiki. There is still so many problems need to be solved in the last two weeks. We need to hurry up! Keep on!</P></DIV><DIV id="note26"><P> This week, we tried to give a presentation to our guiders. They offered us some good suggestions. We need to beautify our PPT. That’s a hard but interesting work. That’s a challenge for us. And we believe that we can finish the job by our hard working!</P></DIV><DIV id="note27"><P> This is the last week for us to finish the wiki. We took turns to have a rest and keep on practicing our presentation. On the other hand, all of us were devoted in the work so as to get the gold medal someday in the future. Just like what Jobs had done to the world, we will to change the world with our software. We are too hurry to hear the news that we are given the gold medal in MIT. We are the best and we had change people’s opinion on iGEM. Keep on going, my guys!</P></DIV><DIV class="notebook_s3m2t2"><DIV class="notebook_s3t5">Meeting Record</DIV><DIV class="notebook_s3t4">9.5</DIV><DIV class="notebook_s3t4">9.12</DIV><DIV class="notebook_s3t4">9.19</DIV><DIV class="notebook_s3t4">9.26</DIV></DIV></DIV></DIV></DIV><DIV id="notebook_s4"><DIV id="notebook_s4m1"><DIV class="sep_title">General Protocols</DIV></DIV><DIV id="notebook_s4m2"><H1>General Protocols</H1><H4>1 Stock solution</H4><P>
								50 mg/mL Ampicillin
								-0.5 g Amp, 10 mL water,  filter sterilize with millipore express membrane, freeze in aliquots 
								50 mmol/L Arabinose
								-  0.1876 g Arabinose, 25 mL water, filter sterilize with millipore express  membrane.</P><H4>2 Preparation of Competent Cell</H4><P>
								Thaw an aliquot of cells (without any  plasmid in them) on ice 
								- To 20 mL of sterile LB,  add 100 μL aliquot of the thawed cells: remember, this LB does not have any  antibiotic in it, so work as aseptically as possible (i.e. autoclave all  solutions and use sterile pipettes).
								- Grow cells in the shaker at 37 ℃ and 200 rpm, until they reach an OD<SUB>600</SUB>= 0.3 - 0.4. This usually takes 1.5 - 2 hours.
								- Ice down the LB with  growing cells for 30 min.
								- Aliquot into sterile  1.5 mL tubes and spin down at 1500 xg for 5 min at 4 ℃; discard supernatant.
								- Ice  down Solution A and Solution B of TaKaRa Competent Cell Preparation Kit during centrifugation. 
								- Gently resuspend each  pellet with 100 μL Solution A.
								- Centrifuge 1500 xg for 5 min and discard supernatant. 
								- Resuspend each pellet on  ice in 100 μL Solution B.
							</P><H4>3 Transformation</H4><P>
							  - Add 10 μL of DNA.  Swirl gently with pipette. 
							  - Incubate tubes on ice for 30 min 
							  - Heat pulse tubes in 42 ℃ water bath for 90 s. 
							  - Incubate on ice for 5 - 10 min 
							  - Add  400 μL of LB broth to each tube and incubate for an hour at 37 ℃ with shaking. 
							  - Spread 100 μL of each culture on an LB agar plate containing the appropriate antibiotics and incubate overnight at 37 ℃ (spread using beads). 
							</P><H4>4 Plasmid Purification</H4><P><STRONG>- </STRONG>Centrifuge  sample in eppendorf tube approximately 1.5 mL at a time, draining off  supernatant after each spin and adding more cell solution. <STRONG>- </STRONG>Resuspend the pelleted cells in  250 μL of the resuspension Solution (mixture with Solution I and RNasa A). The  bacteria should be resuspended completely by vortexing or pipetting up and down  until no cell clumps remain.<STRONG>- </STRONG>Add 250 μL of the Lysis Solution  (Solution II) and mix thoroughly and gently by inverting the tube 5-6 times,  letting it stand for 1-2 min at room temperature until the solution becomes  viscous and slightly clear.<STRONG>- </STRONG>Add 350 μL of the Neutralization  Solution (Solution III) and mix immediately and thoroughly by inverting the  tube 5 - 6 times.
							  - Centrifuge  for 10 min at 12,000 rpm to pellet cell debris.
							  - Apply the  supernatant to the supplied spin column by decanting. Avoid disturbing or  applying the white precipitate.
							  - Centrifuge  for 1 min at 12,000 rpm. Discard  flow-through and place the column back into the same collection tube.
							  - Add 500 μL of the  Wash Buffer PB to the spin column. Centrifuge for 1min at 12,000 rpm and discard  flow-through. Place the column back into the same collection tube.	  - Add 500 μL of the  Wash Buffer W to the spin column. Centrifuge for 1min at 12,000 rpm and discard  the flow-through. Place the column back into the same collection tube.
							  - Repeat the step 9  again.
							  - Discard flow-through  and centrifuge for an additional 3 min to remove residual Wash Solution. 
							  - Place the spin  column in a clean 1.5 mL centrifuge tube, and pipet 20 μL Elution Buffer TE  (prewarm to 60 ℃)  directly to the center of the column without touching the membrane. Let it  stand for 2 min at room temperature and centrifuge for 1 min at 12,000 rpm.
							  - Discard the column and  store the purified plasmid DNA at -20 °C.
							</P><H4>5 Standard BioBrick Assembly</H4><P>
							 - Digestion of insert: 2 - 5 μg DNA / 100 μL  volume, 10<STRONG>×</STRONG> H buffer, <EM>Eco</EM>R I, <EM>Spe </EM>I. Digestion and inactivation. Clean  up the insert via gel electrophoresis. When cutting the insert out of the gel,  try avoiding staining or exposure to ultraviolet light of the insert. 
							 - Digestion of vector: 2 - 5 μg DNA / 100 μL  volume, 10<STRONG>×</STRONG> M buffer, <EM>Eco</EM>R I, <EM>Xba </EM>I. Digestion and inactivation. Clean  up the insert via gel electrophoresis. When cutting the insert out of the gel,  try to avoid staining or exposure to ultraviolet light of the insert.
							</P><H4>6 Suffix Insertion</H4><P>
								- Digestion of insert: 2 μg~5 μg DNA / 100 μL  volume, 10<STRONG>×</STRONG> M buffer, <EM>Xba </EM>I, <EM>Pst </EM>I. Digestion and inactivation. Clean  up the insert.
								- Digestion of vector : 2 μg~5 μg DNA  / 100 μL volume, 10<STRONG>×</STRONG> H buffer, <EM>Spe </EM>I,<EM> Pst </EM>I. Digestion and inactivation.  Clean up the vector.
							</P><H4>7 Ligation</H4><P>
								- After digestion and clean-up, the next step  is ligation. 
								According to the ligation equation of Insert/Vector=3 mol/mol, combine the vector, insert, 10<STRONG>×</STRONG> T4 ligation buffer and 1 μL of T4 ligase in a 10 μL reaction overnight at 4 ℃.
							</P><H4>8 Gel Extraction</H4><P>
							  - Weigh a 1.5 mL centrifuge tube  for each DNA fragment to be isolated and record the weight.
							  - Excise gel slice containing the  DNA fragment using a clean scalpel or razor blade. Cut as close as possible to  the DNA to minimize the gel volume. Place the gel slice into a pre-weighed 1.5 mL  tube and weigh. Record the weight of the gel slice.
							  - Add Bing Buffer BD at a ratio  of 100 μL of solution per 100 mg of agarose gel slices.
							  - Incubate the gel mixture at 55-65 ℃ for 7-10 min or until the gel slice is completely  dissolved. Mix the tube by inversion every few minutes to facilitate the  melting process. Ensure that the gel is completely dissolved.
							  - After the dissolved gel mixture  cool down, transfer it to the Spin Columns assembly and incubate for 2 min at  room temperature.
							  - Centrifuge the Spin Columns  assembly in a microcentrifuge at 12,000 rpm for 1 min, and discard the  flow-through.
							  - Wash the columns by adding 500 μL of Wash Buffer PE to the Columns. Centrifuge the columns assembly for 1 min  at 12,000 rpm, and discard the flow-through.
							  - Repeat step 7 again.
							  - Centrifuge the Columns for an  additional 3 min to completely remove residual wash buffer.
							  - Empty the Collection Tube and  recentrifuge the column assembly for 1 min with the microcentrifuge lid open  (or off) to allow evaporation of any residual ethanol.
							  - Place the spin column in a  clean 1.5 mL microcentrifuge tube, and pipet 20 μL deionized water (pH is  8.0 - 8.5 and prewarm to 60 ℃)directly to  the center of the column without touching the membrane. Incubate at room  temperature for 2 min.
							  - Centrifuge for 1 min at 12,000 rpm. Discard the columns and store the microcentrifuge  tube containing the eluted DNA at–20 ℃ .
							</P><H4>9 Restriction analysis</H4><P>
							  - Combine H<SUB>2</SUB>0, 4μL of dNTP Mixture, 10<STRONG>×</STRONG>Ex Taq Buffer, 0.25μL of Ex Taq, 1 μL of VR primer, 1 μL 
							  of VF2 primer and pick the colony in a 25 μL reaction for PCR.
							  - Gel electrophoresis.
							</P><H4>10 Genetic mutations</H4><P>
												 1. Design the primers with mutation.
												 2. PCR.
												  - 0.25 μL of Pyrobest DNA Polymerase (5U/μL).
												  - 5 μL of 10 x <I>Pyrobest</I> buffer II.
												  - 4 μL of dNTP Mixture (2.5 mM each).
												  - 1 μL of Primer 1 (20 uM).
												  - 1 μL of Primer 2 (20 uM).
												  - 0.01 – 1 ng Template.
												  - Add ddH<SUB>2</SUB>O up to 50 μL.
												 3. PCR condition.
												  - 94 ℃ for 30 s, 55 ℃ for 30 s, 72 ℃ for 5 min . circulate for 24 times.
												 4. Run the gel and do the gel extraction.
												 5. Blunting Kination.
												  - 1 pmol of DNA Fragment.
												  - 2 μL of 10 x Blunting Kination Buffer.
												  - 1 μL of Blunting Kination Enzyme Mix.
												  - Add ddH<SUB>2</SUB>O up to 20 μL.
												  - Reaction under 37 ℃ for 10 min.
												  - Reaction under 70 ℃ for 10 min.
												 6. Ligation.
												  - Take 5 μL reacting solution into a new PCR tube.
												  - Add 5 μL Ligation Solution I.
												  - Reaction under 16 ℃ for 1 h.
												  - Transfor the reacting solution into 100 μL competent cell.</P><H1>Characterization</H1><H4>Fluorescence Measurements</H4><P>
							  - The samples to be  tested are cultured from plates in 20 mL of the Basal Minimal Medium with  appropriate antibiotics and incubated overnight at 37 ℃ at 200 rpm. 
							  - The culture is  checked for OD<SUB>600</SUB> next day and then subculture by the same medium  with antibiotics at 37 ℃ shaking  for 2 h. 
							  - Add corresponding  inducer at concentration gradients into the above-mentioned culture and keep on  incubating. During the time incubating, every 15 min, take 1 mL bacteria  liquid, then centrifuge the cells( 6000 rpm, 10 min ) and resuspend them in 1 mL PBS. At last, pipette to a 96 well plate.
							  - The plate reader made  by Molecular Device then read. 
							  - The program does the  following: 
							  - In endpoint reads,  following measurements are taken in a time interval of 15 min: absorbance (600 nm filter) and fluorescence (485 nm and 520 nm for GFP). 
							  - The results then  transfer to excel sheet and interpret.</P></DIV></DIV><DIV id="notebook_s5"><DIV id="notebook_s5m1"><DIV class="sep_title">Lab Record</DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:XMU_Software/Notebook">http://2013.igem.org/Team:XMU_Software/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:XMU_Software/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:XMU_Software/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:XMU_Software/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:XMU_Software/Notebook&amp;oldid=359814" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>