"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF-Lausanne/Protocols/Site-specific mutagenesis
From 2011.igem.org
Introduction
Home
The Team
Photo Gallery
Our Project
Results Summary
Selection System
In-Vitro Characterization
In-Vivo Characterization
Microfluidics
Data
Attributions
Tools
Gibson Assembly
MITOMI
Notebook
Protocols
May
June
July
August
September
October
Considerations
Human Practices
Safety
Acknowledgements
Sponsors
Partners
Site-specific mutagenesis of tetR
Back to protocols .
Purpose: induce site-specific mutations in a gene, contained on a plasmid.
Site-specific mutagenesis or site-directed mutagenesis is a method to induce site-specific substitutions, deletions, or insertions in a plasmid. It is carried out using a PCR-like reaction, with two primers: one for the sense strand, one for the antisense strand. Both primers overlap the same region, and contain the desired mutations. At each cycle of the reaction, a mutated copy of the entire plasmid is created. In the end, the original (template) plasmid is digested, leaving only the mutants.
The principle of site-directed mutagenesis is explained in more detail in the manual for their mutagenesis kits.
Once all components (primers, buffers, culture media...) are available, the procedure should take approximately one half-day, followed by an overnight culture, then a miniprep, that should take an hour or so. The procedure is outlined as follows:
Reagent preparation
Primer design and ordering
Culture media preparation
Thermal cycling: extension reaction to copy the template and induce mutations (PCR-like reaction) (1 hour).
Digestion of template: enzymes digest the template (unmutated by definition) to leave only the mutants (5 minutes).
Transformation into competent cells: to repair nicks left in the plasmid by the extension reaction (1.5 hours).
Overnight culture on ampicillin: to select for transformed cells and amplify the mutated DNA (1 night).
Miniprep: to recover the mutated DNA (1 hour).
Contents
1 Primer Design
2 Culture media preparation
3 Mutation reaction
3.1 Control Reaction
3.2 Thermal Cycles
Primer Design
The primers are designed using Agilent's online tool: Primer Design Program (requires login; use the iGEM EPFL gmail address. Ask Doug for the password).
According to the Agilent manual for the kit we use, the primers are designed with the following constraints:
Length between 25 and 45 bases, ideally.
Melting temperature above 78° C.
Mutation near the middle of the primers, with 10 to 15 bases on each side.
Ideally, have a GC content of at least 40%, and terminate with one or more G or C bases.
The online tool ensures the constraints are met, all that is needed is to choose the site and type of mutation.
The easiest way to design primers is to copy the wild-type sequence into the sequence, then "upload and translate" it. A little clarification to the provided documentation: when the sequence is uploaded, check boxes appear below a set of radio-buttons. They list the amino acids, translated from the provided sequence. Check up to seven positions, then choose what substitutions to operate using the drop-down menus above.
our primers are then ordered from Invitrogen. Upon arrival, they must be diluted ideally to 100 ng/µl. The primers we order contain too much DNA for the volume of the tube at that concentration, so we dilute them instead to 1 µg/µl.
Culture media preparation
The culture media must be prepared in advance. The required components are, once again, listed in the manual (p.15). For quick reference, three solutions are needed:
LB-Ampicillin Agar, prepared from autoclaved LB Agar. The LB Agar is not needed alone.
NZY+ Broth (autoclaved)
TE Buffer which can just be taken from a miniprep or gel extraction kit.
The agar broth is used to pour plates, as described in the agar plates protocol .
Mutation reaction
The mutation reaction creates the mutant strand. It is carried out like a PCR, in a thermal cycler. Mix the following reagents, for each 50 µl reaction volume:
Reagents for the mutation reactions
Qty
Reagent
5 µl
10x reaction buffer
0.5 µl
Template DNA (or adjust to obtain 10-100 ng)
1.25 µl
Sense primer at 100 ng / µl (or adjust to obtain 125 ng)
1.25 µl
Antisense primer at 100 ng / µl (or adjust to obtain 125 ng)
1 µl
dNTP
1.5 µl
Quicksolution
1 µl
Quickchange Enzyme
38.5 µl
ddH20 (or adjust to obtain 50 µl total)
All reagents (but the water) are included in the mutagenesis kit.
Control Reaction
The control reaction reveals the efficacy of plasmid mutations. It mutates a pWhitescript plasmid into a pBluescript plasmid. Competent cells subsequently transformed with the pBluescript plasmid express the beta-galactosidase gene, which turns them blue in presence of IPTG and X-gal.
Mix the following reagents:
Reagents for 50 µl control reaction
Qty
Reagent
5 µl
Reaction buffer
5 µl
pWhitescript
1.25 µl
Control primer #1
1.25 µl
Control primer #2
1 µl
dNTP
1.5 µl
Quicksolution
34 µl
ddH20
Thermal Cycles
Run both reactions with the heat cycles listed below. The control reaction needs a 2'30" extension step.
Heat cycles for mutagenesis plasmid copy
Segment
Cycles
Temp [°C]
Time
1
1
95
2 min
2
18
95
20 s
60
10 s
68
30 s / kb plasmid length
3
1
68
5 min
Retrieved from " http://2011.igem.org/Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
