<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF-Lausanne_Notebook_October2011"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF-Lausanne/Notebook/October2011">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF-Lausanne/Notebook/October2011&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/October2011&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/October2011&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF-Lausanne/Notebook/October2011" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF-Lausanne/Notebook/October2011</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#">Introduction</A></LI><LI><A href="/Team:EPF-Lausanne">Home</A></LI><LI><A href="/Team:EPF-Lausanne/TheTeam">The Team</A></LI><LI><A href="/Team:EPF-Lausanne/Photos">Photo Gallery</A></LI></UL><LI><A class="head" href="#">Our Project</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Summary">Results Summary</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/T7_promoter_variants">Selection System</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/TetR_mutants"><I>In-Vitro</I> Characterization</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Reporter_Systems"><I>In-Vivo</I> Characterization</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/Microfluidics">Microfluidics</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Data">Data</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Attributions">Attributions</A></LI><LI><A class="head" href="#">Tools</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/Gibson_assembly">Gibson Assembly</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/MITOMI">MITOMI</A></LI><LI><A class="head" href="/Team:EPF-Lausanne/Notebook/September2011">Notebook</A></LI><LI><A href="/Team:EPF-Lausanne/Protocols">Protocols</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/May2011">May</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/June2011">June</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/July2011">July</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/August2011">August</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/September2011">September</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/October2011">October</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:EPF-Lausanne/Human_Practices">Human Practices</A></LI><LI><A href="/Team:EPF-Lausanne/Safety">Safety</A></LI><LI><A href="/Team:EPF-Lausanne/Sponsors" class="head">Acknowledgements</A></LI><LI><A href="/Team:EPF-Lausanne/Sponsors">Sponsors</A></LI><LI><A href="/Team:EPF-Lausanne/Partners">Partners</A></LI></DIV><DIV id="right"><H1>Notebook: October 2011</H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Friday.2C_October_7_2011"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Friday, October 7 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Monday.2C_October_10_2011"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Monday, October 10 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Wednesday.2C_October_12_2011"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Wednesday, October 12 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Thursday.2C_October_13_2011"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Thursday, October 13 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Sunday.2C_October_16_2011"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Sunday, October 16 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Sunday.2C_October_23_2011"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Sunday, October 23 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Monday.2C_October_24_2011"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Monday, October 24 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Tuesday.2C_October_25_2011"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Tuesday, October 25 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Wednesday.2C_October_25_2011"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Wednesday, October 25 2011</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Friday.2C_October_7_2011"> Friday, October 7 2011 </SPAN></H2><P>Matt did gene-specific PCR -- touchdown-style -- on the lysis cassette. The gel results look fantastic. Pictures coming soon. Next up, adding the T7 promoters (all twelve of them) upstream of lysis. 
</P><H2><SPAN class="mw-headline" id="Monday.2C_October_10_2011"> Monday, October 10 2011 </SPAN></H2><P>Over the weekend, Matt and Vincent figured out how the PCRs ought to work out to get the different T7 promoters upstream of the lysis cassette, which had been successfully amplified earlier last week. A helpful website indicated what the right melting temperatures should be. From there, Matt kindly took the initiative of making all 12 T7-lysis combinations via PCR. He then ran a gel and took a bunch of sweet pictures, and the results look great: no additional PCR products and some very clean bands of the right size. 
</P><H2><SPAN class="mw-headline" id="Wednesday.2C_October_12_2011"> Wednesday, October 12 2011 </SPAN></H2><P>Vincent ran a PCR to amplify pSB3K1, which we plan on using as a backbone for the T7 + Lysis inserts. He used Expand High Fidelity Polymerase and its associated protocol for an expected band of 2774 bp. The gel to check it will be run on Thursday.
</P><H2><SPAN class="mw-headline" id="Thursday.2C_October_13_2011"> Thursday, October 13 2011 </SPAN></H2><P>The gel for the K1 PCR amplification was run, and the results look good: a nice thick band right below the 3000 bp mark. Now we have to put the T7 + Lysis together with the backbone, and for that there will have to be some blunt-ends with phosphorylation before we can ligate the two pieces together. Stay tuned.  
</P><H2><SPAN class="mw-headline" id="Sunday.2C_October_16_2011"> Sunday, October 16 2011 </SPAN></H2><P>Vincent came in to work on the blunt-end cloning for the inserts (T7 promoters + Lysis) and the psB3K1 vector. The idea is to use the T4 DNA polymerase, which has a 3' to 5' exonuclease, to fill-in and chew away the ends till we have blunt ends. I added the protocol to the appropriate protocol section. After running through this protocol, the T7 inserts had an average concentration of 40 ng/uL while the backbone vector K1 had a concentration of 90 ng/uL (to within Nanodrop measurement error). Those concentrations are high enough to allow us to move on to the next step which is phosphorylating the T7 inserts (not the backbone!) for ligation. 
</P><P>The phosphorylation step is needed because the ligation (via ligase enzyme) requires that one of the 5' termini have a phosphate residue that will induce phosphodiester bond. If both the insert and the backbone have the phosphate residue, then the backbone may bind to itself, leaving the insert out -- which would defeat the purpose of ligation. 
</P><P>The specifics of the blunt-end protocol were as follows:
</P><P>For the T7 + Lysis inserts,
</P><UL><LI> H20 - 11.25 uL
</LI><LI> NEBuffer2 (10X, found in enzyme ice box) - 3 uL (needs to be at a final concentration of 1X)
</LI><LI> dNTP (10 mM) - 0.3 (needs to be at a final concentration of 100 uM)
</LI><LI> template - 15 uL (for 1.5 ug of DNA, assuming the T7 + Lysis was at a concentration of 100 ng/uL)
</LI><LI> T4 Polymerase - 0.45 uL (it comes at a &quot;concentration&quot; of 3000 units/mL and we want 1.5 units)
</LI></UL><P>I made a master mix of everything except the template DNA, which then went into separate PCR tubes. The total is a 30 uL reaction. The tubes were placed in the Bio-Rad machine were we setup a protocol (called BLUNTEND) that keeps the tubes at a constant 12 C for 15 minutes. If you want, you can add EDTA to a final concentration of 10 mM (as advised by New England Biolabs) and heat activate the samples at 75 C for 20 minutes. We decided that it was not necessary in our case. 
</P><P>For the K1 backbone, the recipe was slightly different due to the higher concentration of blunted DNA we wanted to achieve:
</P><UL><LI> H20 - 4.4 uL
</LI><LI> NEBuffer2 (10X, found in enzyme ice box) - 4 uL (needs to be at a final concentration of 1X)
</LI><LI> dNTP (10 mM) - 0.4 (needs to be at a final concentration of 100 uM)
</LI><LI> template - 30 uL (for 1.5 ug of DNA, assuming the T7 + Lysis was at a concentration of 100 ng/uL)
</LI><LI> T4 Polymerase - 1.2 uL (it comes at a &quot;concentration&quot; of 3000 units/mL and we want 1.5 units)
</LI></UL><P>for a total of 40 uL. 
</P><H2><SPAN class="mw-headline" id="Sunday.2C_October_23_2011"> Sunday, October 23 2011 </SPAN></H2><P>Vincent ran a colony PCR on the 1 and 13 T7 promoter + Lysis + psB3K1 constructs, since they were the ones that yielded the highest colony count (compared to the negative control). He chose ten colonies from each plate. The protocol was as followed:
</P><UL><LI> TPB 2.5 uL
</LI><LI> EPFL Taq 0.25 uL
</LI><LI> primer FW (10 uM) 0.5 uL (1289_F, originally 100 uM)
</LI><LI> primer RV (10 uM) 0.5 uL (pSB-Pcon-TetR-r, originally 20 uM)
</LI><LI> template 2 uL of DNA (boiled)
</LI><LI> dNTP 0.5 uL
</LI><LI> dH20 18.75 uL
</LI></UL><P>_________________________
25 uL total
</P><P>The cycle (now called EPFLCOP for EPFL Colony PCR) is as follows:
</P><UL><LI> 94 Â°C, 2 minutes
</LI></UL><P>30 times:
</P><UL><LI> 94 Â°C, 15 sec.
</LI><LI> 58 Â°C, 15 sec.
</LI><LI> 68 Â°C, 1 minute (1 min / kb, and we expect 629 bp)
</LI></UL><P>End of 30
</P><UL><LI> 68 Â°C, 5 min
</LI><LI> 10 Â°C, infinitely
</LI></UL><P>
The gel showed a strong signal at 630 bp for two colonies of T7-1 but none for T7-13. Unfortunately, this is the same promoter as the one amplified by Alina two months ago, so we can't use it to run an interesting experiment.
</P><H2><SPAN class="mw-headline" id="Monday.2C_October_24_2011"> Monday, October 24 2011 </SPAN></H2><P>Vincent mini-prepped the liquid culture from the successful T71-Lysis-K1 plasimd identified by PCR the previous day. He then transformed that DNA into BL21 cells and plated the transformant. 
</P><P>Since the primers used on Sunday would miss half of the inserts (the blunt-end strategy implies that using primers on the insert and on the vector will miss half of the potentially good plasmids), we chose to switch to different primers: the 816_R and 30_F which amplify on the lysis cassette. 
</P><P>With these primers, Vincent ran a colony PCR on 10 colonies per plate for the 2, 3, 4, and 6 plates (2,3,4, and 6 refers to the T7 promoter type). He used the same protocol as on Sunday. The resulting gel shows that 3 out of the 4 T7 variants show an amplification on some colony:
</P><H2><SPAN class="mw-headline" id="Tuesday.2C_October_25_2011"> Tuesday, October 25 2011 </SPAN></H2><P>Nadine and Douglas came in and mini-prepped liquid cultures prepared the previous day by Nadine. 
</P><P>Vincent transformed the resulting DNA into BL21 cells. 
</P><H2><SPAN class="mw-headline" id="Wednesday.2C_October_25_2011"> Wednesday, October 25 2011 </SPAN></H2><P>Late on Tuesday, Henrike noticed that the colonies had grown sufficiently to make liquid cultures for a platereader. Nadine ran the platereader on Wednesday with the following results:
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:EPF-Lausanne/Notebook/October2011">http://2011.igem.org/Team:EPF-Lausanne/Notebook/October2011</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF-Lausanne/Notebook/October2011" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF-Lausanne/Notebook/October2011" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/October2011&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/October2011&amp;oldid=250345" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>