<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF-Lausanne_Protocols_Site-specific_mutagenesis"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF-Lausanne/Protocols/Site-specific mutagenesis</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#">Introduction</A></LI><LI><A href="/Team:EPF-Lausanne">Home</A></LI><LI><A href="/Team:EPF-Lausanne/TheTeam">The Team</A></LI><LI><A href="/Team:EPF-Lausanne/Photos">Photo Gallery</A></LI></UL><LI><A class="head" href="#">Our Project</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Summary">Results Summary</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/T7_promoter_variants">Selection System</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/TetR_mutants"><I>In-Vitro</I> Characterization</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Reporter_Systems"><I>In-Vivo</I> Characterization</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/Microfluidics">Microfluidics</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Data">Data</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Attributions">Attributions</A></LI><LI><A class="head" href="#">Tools</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/Gibson_assembly">Gibson Assembly</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/MITOMI">MITOMI</A></LI><LI><A class="head" href="/Team:EPF-Lausanne/Notebook/September2011">Notebook</A></LI><LI><A href="/Team:EPF-Lausanne/Protocols">Protocols</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/May2011">May</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/June2011">June</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/July2011">July</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/August2011">August</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/September2011">September</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/October2011">October</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:EPF-Lausanne/Human_Practices">Human Practices</A></LI><LI><A href="/Team:EPF-Lausanne/Safety">Safety</A></LI><LI><A href="/Team:EPF-Lausanne/Sponsors" class="head">Acknowledgements</A></LI><LI><A href="/Team:EPF-Lausanne/Sponsors">Sponsors</A></LI><LI><A href="/Team:EPF-Lausanne/Partners">Partners</A></LI></DIV><DIV id="right"><H1>Site-specific mutagenesis of tetR</H1><SPAN class="backlink">Back to <A href="/Team:EPF-Lausanne/Protocols" title="Team:EPF-Lausanne/Protocols">protocols</A>.</SPAN><P><B>Purpose</B>: induce site-specific mutations in a gene, contained on a plasmid.
</P><P><I>Site-specific mutagenesis</I> or <I>site-directed mutagenesis</I> is a method to induce site-specific substitutions, deletions, or insertions in a plasmid. It is carried out using a PCR-like reaction, with two primers: one for the sense strand, one for the antisense strand. Both primers overlap the same region, and contain the desired mutations. At each cycle of the reaction, a mutated copy of the entire plasmid is created. In the end, the original (template) plasmid is digested, leaving only the mutants.
</P><P>The principle of site-directed mutagenesis is explained in more detail in the <A href="https://www.genomics.agilent.com/files/manual/210518.pdf|Agilent" class="external text" rel="nofollow">manual</A> for their mutagenesis kits.
</P><P>Once all components (primers, buffers, culture media...) are available, the procedure should take approximately one half-day, followed by an overnight culture, then a miniprep, that should take an hour or so. The procedure is outlined as follows:
</P><UL><LI><B>Reagent preparation</B></LI><LI> Primer design and ordering
</LI><LI> Culture media preparation
</LI></UL><LI><B>Thermal cycling</B>: extension reaction to copy the template and induce mutations (PCR-like reaction) (1 hour).
</LI><LI><B>Digestion of template</B>: enzymes digest the template (unmutated by definition) to leave only the mutants (5 minutes).
</LI><LI><B>Transformation</B> into competent cells: to repair nicks left in the plasmid by the extension reaction (1.5 hours).
</LI><LI><B>Overnight culture</B> on ampicillin: to select for transformed cells and amplify the mutated DNA (1 night).
</LI><LI><B>Miniprep</B>: to recover the mutated DNA (1 hour).
</LI><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Primer_Design"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Primer Design</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Culture_media_preparation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Culture media preparation</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Mutation_reaction"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Mutation reaction</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Control_Reaction"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Control Reaction</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Thermal_Cycles"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Thermal Cycles</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Primer_Design"> Primer Design </SPAN></H2><P>The primers are designed using Agilent's online tool: <A href="https://www.genomics.agilent.com/CollectionOverview.aspx?PageType=Application&amp;SubPageType=ApplicationOverview&amp;PageID=111|QuickChange" class="external text" rel="nofollow">Primer Design Program</A> (requires login; use the iGEM EPFL gmail address. Ask Doug for the password).
</P><P>According to the <A href="https://www.genomics.agilent.com/files/manual/210518.pdf" class="external text" rel="nofollow">Agilent manual</A> for the kit we use, the primers are designed with the following constraints:
</P><UL><LI> Length between 25 and 45 bases, ideally.
</LI><LI> Melting temperature above 78° C.
</LI><LI> Mutation near the middle of the primers, with 10 to 15 bases on each side.
</LI><LI> Ideally, have a GC content of at least 40%, and terminate with one or more G or C bases.
</LI></UL><P>The online tool ensures the constraints are met, all that is needed is to choose the site and type of mutation.
</P><P>The easiest way to design primers is to copy the wild-type sequence into the sequence, then &quot;upload and translate&quot; it. A little clarification to the provided documentation: when the sequence is uploaded, check boxes appear below a set of radio-buttons. They list the amino acids, translated from the provided sequence. Check up to seven positions, then choose what substitutions to operate using the drop-down menus <I>above</I>.
</P><P>our primers are then ordered from Invitrogen. Upon arrival, they must be diluted ideally to 100 ng/µl. The primers we order contain too much DNA for the volume of the tube at that concentration, so we dilute them instead to <B>1 µg/µl</B>.
</P><H2><SPAN class="mw-headline" id="Culture_media_preparation"> Culture media preparation </SPAN></H2><P>The culture media must be prepared in advance. The required components are, once again, listed in the manual (p.15). For quick reference, three solutions are needed:
</P><UL><LI><B>LB-Ampicillin Agar</B>, prepared from autoclaved LB Agar. The LB Agar is not needed alone.
</LI><LI><B>NZY+ Broth</B> (autoclaved)
</LI><LI><B>TE Buffer</B> which can just be taken from a miniprep or gel extraction kit.
</LI></UL><P>The agar broth is used to pour plates, as described in the <A href="/Team:EPF-Lausanne/Protocols/Agar_Plates" title="Team:EPF-Lausanne/Protocols/Agar Plates">agar plates protocol</A>.
</P><H2><SPAN class="mw-headline" id="Mutation_reaction"> Mutation reaction </SPAN></H2><P>The mutation reaction creates the mutant strand. It is carried out like a PCR, in a thermal cycler. Mix the following reagents, for each <B>50 µl</B> reaction volume:
</P><TABLE><CAPTION> Reagents for the mutation reactions
</CAPTION><TBODY><TR><TH> Qty </TH><TH> Reagent
</TH></TR><TR><TD> 5 µl </TD><TD> 10x reaction buffer
</TD></TR><TR><TD> 0.5 µl </TD><TD> Template DNA <I>(or adjust to obtain 10-100 ng)</I></TD></TR><TR><TD> 1.25 µl </TD><TD> Sense primer at 100 ng / µl <I>(or adjust to obtain 125 ng)</I></TD></TR><TR><TD> 1.25 µl </TD><TD> Antisense primer at 100 ng / µl <I>(or adjust to obtain 125 ng)</I></TD></TR><TR><TD> 1 µl </TD><TD> dNTP
</TD></TR><TR><TD> 1.5 µl </TD><TD> Quicksolution
</TD></TR><TR><TD> 1 µl </TD><TD> Quickchange Enzyme
</TD></TR><TR><TD> 38.5 µl </TD><TD> ddH20 <I>(or adjust to obtain 50 µl total)</I></TD></TR></TBODY></TABLE><P>All reagents (but the water) are included in the mutagenesis kit.
</P><H3><SPAN class="mw-headline" id="Control_Reaction"> Control Reaction </SPAN></H3><P>The control reaction reveals the efficacy of plasmid mutations. It mutates a pWhitescript plasmid into a pBluescript plasmid. Competent cells subsequently transformed with the pBluescript plasmid express the beta-galactosidase gene, which turns them blue in presence of IPTG and X-gal.
</P><P>Mix the following reagents:
</P><TABLE><CAPTION> Reagents for 50 µl control reaction
</CAPTION><TBODY><TR><TH> Qty </TH><TH> Reagent
</TH></TR><TR><TD> 5 µl </TD><TD> Reaction buffer
</TD></TR><TR><TD> 5 µl </TD><TD> pWhitescript
</TD></TR><TR><TD> 1.25 µl </TD><TD> Control primer #1
</TD></TR><TR><TD> 1.25 µl </TD><TD> Control primer #2
</TD></TR><TR><TD> 1 µl </TD><TD> dNTP
</TD></TR><TR><TD> 1.5 µl </TD><TD> Quicksolution
</TD></TR><TR><TD> 34 µl </TD><TD> ddH20
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Thermal_Cycles"> Thermal Cycles </SPAN></H3><P>Run both reactions with the heat cycles listed below. The control reaction needs a 2'30&quot; extension step.
</P><TABLE><CAPTION> Heat cycles for mutagenesis plasmid copy
</CAPTION><TBODY><TR><TH> Segment </TH><TH> Cycles </TH><TH> Temp [°C] </TH><TH> Time
</TH></TR><TR><TD> 1 </TD><TD> 1 </TD><TD> 95 </TD><TD> 2 min
</TD></TR><TR><TD> 2 </TD><TD> 18 </TD><TD> 95 </TD><TD> 20 s
</TD></TR><TR><TD> 60 </TD><TD> 10 s
</TD></TR><TR><TD> 68 </TD><TD> 30 s / kb plasmid length
</TD></TR><TR><TD> 3 </TD><TD> 1 </TD><TD> 68 </TD><TD> 5 min
</TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis">http://2011.igem.org/Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis&amp;oldid=54910" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>