<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF-Lausanne_Notebook_September2011"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF-Lausanne/Notebook/September2011">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF-Lausanne/Notebook/September2011&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/September2011&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/September2011&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF-Lausanne/Notebook/September2011" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF-Lausanne/Notebook/September2011</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#">Introduction</A></LI><LI><A href="/Team:EPF-Lausanne">Home</A></LI><LI><A href="/Team:EPF-Lausanne/TheTeam">The Team</A></LI><LI><A href="/Team:EPF-Lausanne/Photos">Photo Gallery</A></LI></UL><LI><A class="head" href="#">Our Project</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Summary">Results Summary</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/T7_promoter_variants">Selection System</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/TetR_mutants"><I>In-Vitro</I> Characterization</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Reporter_Systems"><I>In-Vivo</I> Characterization</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/Microfluidics">Microfluidics</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Data">Data</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Attributions">Attributions</A></LI><LI><A class="head" href="#">Tools</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/Gibson_assembly">Gibson Assembly</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/MITOMI">MITOMI</A></LI><LI><A class="head" href="/Team:EPF-Lausanne/Notebook/September2011">Notebook</A></LI><LI><A href="/Team:EPF-Lausanne/Protocols">Protocols</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/May2011">May</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/June2011">June</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/July2011">July</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/August2011">August</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/September2011">September</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/October2011">October</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:EPF-Lausanne/Human_Practices">Human Practices</A></LI><LI><A href="/Team:EPF-Lausanne/Safety">Safety</A></LI><LI><A href="/Team:EPF-Lausanne/Sponsors" class="head">Acknowledgements</A></LI><LI><A href="/Team:EPF-Lausanne/Sponsors">Sponsors</A></LI><LI><A href="/Team:EPF-Lausanne/Partners">Partners</A></LI></DIV><DIV id="right"><H1>Notebook: September 2011</H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Thursday.2C_September_1_2011"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Thursday, September 1 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Friday.2C_September_2_2011"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Friday, September 2 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Monday.2C_September_5_2011"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Monday, September 5 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Tuesday.2C_6_September_2011"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Tuesday, 6 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Wednesday.2C_7_September_2011"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Wednesday, 7 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Thursday.2C_8_September_2011"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Thursday, 8 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Friday.2C_09_September_2011"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Friday, 09 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Saturday.2C_10_September_2011"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Saturday, 10 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Sunday.2C_11_September_2011"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Sunday, 11 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Someday.2C_12_September_2011"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Someday, 12 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Tuesday.2C_13_September_2011"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Tuesday, 13 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Wednesday.2C_14_September_2011"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Wednesday, 14 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Thursday.2C_15_September_2011"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Thursday, 15 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Friday.2C_16_September_2011"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Friday, 16 September 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Saturday.2C_17_September_2011"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Saturday, 17 September 2011</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Thursday.2C_September_1_2011"> Thursday, September 1 2011 </SPAN></H2><P>Vincent PCR purified the pSB3K1-extended (no TetR) for use in the Gibsons and negative controls. 
</P><P>The Gibson assembly of J6-Plac-Lysis had produced a single colony. Vincent colony-PCRed it using two different sets of primers to amplify subportions of the lysis cassette. The first set of primers (30-f &amp; 816-r) amplify an 800 bp piece whereas the second set (Seq-J6-RFP-f &amp; Seq-J6-RFPlysis-r) amplifies a 1100 bp piece. 
</P><P>The gel seems to confirm that lysis is in the plasmid. We will send the plasmid for sequencing on Friday. 
</P><P>After taking a closer look at the plate from the platereader, it seems that none of the T7 variants did any lysing. This realization made it urgent that we discover whether or not the full T4-lysis cassette was really in these Gibson-assembled plasmids. The samples that had been sent in for sequencing had primers that seemed to amplify the backbone, instead of the lysis cassette. While this does not necessarily mean that the lysis is not there, it does suggest that all the PCRS and gels used to verify the presence of lysis were only showing the existence of the backbone. 
</P><P>Alina ran a PCR with different primers from Douglas' first attempts at putting together the 2700 bp cassette that amplify select regions of the cassette.
</P><P>Lilia tested the expression of the TetR mutants with the T7 promoter. The expression mix contained the Green Lysine and the gel fluorescence was imaged and is shown on the next picture. 
</P><P>The BP2col2 does not express, what explains the failure of the protein pool down at the MITOMI made on Wednesday.
</P><H2><SPAN class="mw-headline" id="Friday.2C_September_2_2011"> Friday, September 2 2011 </SPAN></H2><P>Alina wanted to try a new extension PCR of the T4-lysis and a Gibson of the exteneded-lysis into the K1 backbone. The transformation was successful with tons of colonies. A colony PCR will be run on Monday.
</P><P>
Nadine came back in the lab and cotransformed pSB3K1 Pconst-TetR with J61002 Plac-RFP since the sequencing results were concluent, in order to perform IPTG and ATC platereader experiments.
We had a lot of contaminated bottles, so she also made new SOC bottles.
</P><H2><SPAN class="mw-headline" id="Monday.2C_September_5_2011"> Monday, September 5 2011 </SPAN></H2><P>Alina ran a colony PCR on the K1-T7-lysis Gibson assembly. 
</P><P>
After having set liquid cultures on Saturday, Nadine set up the platereader experiment as follows:
</P><UL><LI> J61002 Plac-RFP with increasing concentrations of IPTG
</LI><LI> pSB3K1 TetR-LacI + J61002 Plac-RFP with increasing concentrations of IPTG
</LI><LI> pSB3K1 TetR-LacI + J61002 Plac-RFP with increasing concentrations of ATC
</LI></UL><P>She had also prepared co-transformations with pSB3K1 TetR-LacI + J61002 Plac-lysis, but the sequencing results indicated that the lysis cassette was wrong.
</P><H2><SPAN class="mw-headline" id="Tuesday.2C_6_September_2011"> Tuesday, 6 September 2011 </SPAN></H2><P>The results from the platereader experiment came back and they look good!
</P><UL><LI> J6 Plac-RFP: The level of RFP expression doesn't rise much with high doses of IPTG, showing that DH5alpha cells don't express much LacI and that Plac is a quite weak  promoter.
</LI></UL><UL><LI> J6 Plac-RFP + pSB TetR-LacI, IPTG: you can clearly see an increase of RFP when IPTG is added. This means that even with a constitutive promoter in front of TetR, LacI is still expressed. In fact, Pconst is weaker than Ptet. The &quot;2.5&quot; data seems strange, I perhaps missed to put IPTG in this well.
</LI></UL><UL><LI> J6 Plac-RFP + pSB TetR-LacI, ATC: Without ATC, we have a similar expression curve as in the IPTG experiment. When ATC is added, we see a diminution of RFP expression because LacI is not repressed anymore.
</LI></UL><P>Vincent transformed the &quot;randomer&quot; Gibson assemblies (from nearly a month ago) into BL21 cells. If you cross-check with the numerical scheme, those would be variants 7, 8, 9, 16, 17, 18.
</P><P>We also went after the remaining non-randomer variants that were left (6, 13, and 14) using a colony PCR on six colonies of each variant. The PCR used 64 C for annealing and 1 minute for extension.
</P><H2><SPAN class="mw-headline" id="Wednesday.2C_7_September_2011"> Wednesday, 7 September 2011 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">first part of the colony PCR on T7 promoter variants biobricks</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">second part of the colony PCR on T7 promoter variants biobricks</DIV></DIV></DIV><P>
Nadine made colony PCRs for the T7 promoter variants that Henrike cloned into the biobrick vector. The right insert should be ~450 bp long, but all the colonies show a fragment at 300 bp. Nevertheless, we have 5 colonies with the 450 insert. The low number of positive colonies correlates with the observation that the ligation negative control plate had about half as much colonies as the different ligations.
She also prepared some liquid cultures for tomorrow's platereader experiment, and miniprepped the pSB TetR-LacI plasmid before sending it for 2 new sequencing reactions.
</P><P>Vdog made glycerol stocks and minipreps of the non-randomer RFP variants 6, 13, and 14 as well as Alina's T7-const-C2/C11, T7 lac 1, and T7 lac2 3. 
</P><P>He also sent all of Alina's constructs (aforementioned) for sequencing using two different primers: 947_f and 816_r. He then ran a control PCR on all the T7 constructs using Alina's EPFL TAQ with the following combination of primers:
</P><UL><LI> 440-f and 1333-r 
</LI></UL><UL><LI> 947-f and pSB3K1-ext-r
</LI></UL><P>The protocol asked for a 25 uL aliquot with 
</P><UL><LI> 2.5 uL Thermo Pol buffer (TPB)
</LI><LI> 0.5 uL dNTP
</LI><LI> 0.5 uL primer 1
</LI><LI> 0.5 uL primer 2
</LI><LI> 1 uL plasmid
</LI><LI> 0.25 uL TAQ
</LI></UL><P>He went on to transform the non-randomer RFP minipreps and the T7-const-c2/c11, lac1, and lac2 3 constructs into BL21 cells
This was followed by the production of a 96 round-bottom well plate with 100 uL of LB with 1 uL of Kan. Twelve colonies from each plate were chosen and swirled into the individual wells. This plate was put on a shaker overnight (after having been sealed with the Easy-Breathe adhesive paper).
</P><H2><SPAN class="mw-headline" id="Thursday.2C_8_September_2011"> Thursday, 8 September 2011 </SPAN></H2><P>Sequencing of T7-const-C2 returned the correct sequence, as did the Control PCR, so it seems that the lysis cassette has found its way into the T7-const-C2! 
</P><P>
On the other hand, sequencing results for pSB TetR-LacI show that the Ptet promoter before LacI is mutated: we have 3 deletions and one mismatch. The deletions are in the two binding sites of TetR, probably preventing TetR action on the promoter. This can then explain why we had a low RFP expression in the paltereader for the pSB TetR-LacI + J6 Plac-RFP cotransformation: we thought Pconst was too weak, leading to a too big LacI expression, but actually the problem was that TetR couldn't repress well LacI.
Nadine made liquid cultures of J61002 Ptet-RFP because we were running out of DNA. We want to send it for sequencing, to be sure that in this plasmid Ptet is OK - although the experiments suggest that yes. She also made liquid cultures of the successful ligations (see yesterday's results) in order to sequence-verify them afterwards.
</P><P>Vincent transformed T7-const-C2-Lysis and T7-const-C11-Lysis (as a negative control) into BL21 cells and plated them. He also co-transformed:
</P><UL><LI> T7-const-C2-Lysis + Ptet-GFP 
</LI></UL><UL><LI> T7-const-C2-Lysis + Ptet-RFP
</LI></UL><UL><LI> T7-const-C11-Lysis + Ptet-GFP
</LI></UL><UL><LI> T7-const-C11-Lysis + Ptet-RFP
</LI></UL><P>into BL21 using Amp + Kan plates. He used 1 uL of each plasmid. 
</P><P>Vince also prepared a 96 well plate for testing all the randomers (7, 8, 9, 16, 17, 18) of T7-RFP and grew it overnight (shaking, with Easy-Breathe paper to let the cells get air).
</P><H2><SPAN class="mw-headline" id="Friday.2C_09_September_2011"> Friday, 09 September 2011 </SPAN></H2><P>Nadine's platereader experiment gave disappointing results: the cells didn't grow much, reaching only 0.1-0.2 OD instead of 1 normally. This will be repeated next Tuesday.
The minipreps of J6 Ptet-RFP and the T7 biobricks yielded high concentrations, and therefore were sent for sequencing.
</P><P>Vince made 2 new plates from his &quot;main&quot; plate that had grown overnight, and he grew those plates to make them into glycerol stocks. He took the &quot;main&quot; plate to the Bioscreening Facility to use their platereader. Meanwhile, he set up a time-lapse, qualitative test of the lysing ability of the C2 culture. We set up two sets of three test tubes, one with C2 induced with 250 uM IPTG and the other un-induced. Here are the photos, and, as you can perhaps see, the last one shows some lysis. 
</P><P>The randomer plate-reader experiment went well and results should be appearing shortly. We used 250 uM IPTG.
Vincent made liquid cultures of all 12 of the T7 and T7-lac2 RFP variants for a big plate-reader experiment on Saturday.
</P><H2><SPAN class="mw-headline" id="Saturday.2C_10_September_2011"> Saturday, 10 September 2011 </SPAN></H2><P>Vincent ran the plate-reader experiment for the 12 T7, T7-lac2 RFP variants, using 250 uM IPTG and 0 uM as a negative control. The induction curves look great, results should be coming in shortly. 
</P><P>Vdog made liquid cultures of T7-C2, T7-C11 and T7-RFP-1 for the lysis platereader experiment. 
</P><H2><SPAN class="mw-headline" id="Sunday.2C_11_September_2011"> Sunday, 11 September 2011 </SPAN></H2><P>Vincent ran another plate-reader experiment, this time for lysis. We tested three different IPTG concentrations: 0 uM, 100 uM, 250 uM, and 500 uM. Two different types of dilution were used: 1:10 and 1:20 (to see how the initial dilution affected the lysis curves). Finally, we checked to see how C2 behaved versus C11 (which does not have the lysis cassette but has T7-const) and versus T7-RFP (as far as induction time is concerned). Results should be appearing shortly.  
</P><H2><SPAN class="mw-headline" id="Someday.2C_12_September_2011"> Someday, 12 September 2011 </SPAN></H2><P>Lilia prepared minipreps of the 5 muTetRs in Biobrick standart (pSB3K1), they will be send for sequencing. Here is the updated list of available mutants that are being characterized:
</P><P>
Nadine started liquid cultures for tomorrow's platereader experiment. She also made the PCRs for putting TetR mutants into the pSB3K1 Pconst-TetR plasmid by Gibson. We can see the expected band at around 700 bp! Unfortunately, amplification of the pSB3K1 backbone didn't work...
</P><P>The mutants used were:
</P><OL><LI> BP1col5 (V36FW43S)
</LI><LI> BP2col6 (E37AW43SA141
</LI><LI> BP3col1 (P39K)
</LI><LI> BP4col1 (Y42F)
</LI><LI> BP4col6 (Y42FK108E)
</LI><LI> V36F
</LI><LI> P39QY42M
</LI></OL><H2><SPAN class="mw-headline" id="Tuesday.2C_13_September_2011"> Tuesday, 13 September 2011 </SPAN></H2><P>Nadine re-did the PCR to amplify pSB3K1 backbone, this time using the right template DNA and it worked! She had strange results on the platereader experiment again, but it's because she diluted cells in water instead of LB medium. The experiment will be re-ran a 3rd time!
</P><P>About the T7 biobricks, the sequencing results show that we already have T7 const (1) and T7 Lac2 030 (12). The other most interesting promoter variants are: T7 030 (3), T7 054 (4), T7 111 (6), T7 lac2 054 (13) and T7 lac2 111 (15). We will focus on them and try to build biobrick formats before the deadline.
</P><H2><SPAN class="mw-headline" id="Wednesday.2C_14_September_2011"> Wednesday, 14 September 2011 </SPAN></H2><P>Nadine tried to purify the PCR products of the TetR mutants, but lost them during the process... So she re-ran the PCR and used it directly do do Gibson assemblies that she later transformed into DH5alpha cells.
</P><P>She looked at the plates containing the biobrick variants and made colony PCRs on five colonies of variats 3,4 and 15. The resuklts show that we have 2 good colonies for 3 and one for 5.
</P><P>Finally, the Plac promoter being correctly inserted into J61002 Plac-RFP plasmid, Nadine also ran a PCR to add biobrick extensions to the promoter.
</P><P>
Vdog ran a ten-hour experiment with the help of Henrike and Matt. We made two 100 mL batches LB with Kan and Amp in 1:1000 dilutions. We then took the two sets of four tubes of 5 mL liquid cultures (of C2-Lys-Ptet-RFP &amp; C11-Lys-Ptet-RFP) and spun them down, threw away the supernatant, resuspended them in 5 mL of PBS buffer, spun them down again, resuspended once more in 5 mL of PBS buffer, and then spun them down again before resuspending one last time in 1 mL of PBS buffer. The final tube of resuspended and clean culture contained no more than 4 mL. We then took cuvettes and measured the OD of each culture in a 1:100 dilution. We then calculated the amount of liquid culture we would need to put in each flask to produce an equal starting OD for both flasks of around .4
</P><P>Every hour, we took 1 mL samples from both flasks, spun them down, sterile filtered the supernatant, and stored the resulting filtered supernatant for transformations the next day. We also took OD measurements of samples from the flask. The flasks were incubated for an hour at 37 C before being taken out for measurements. 
</P><P>The results of the experiment indicate that not only do we have successful lysing of the relevant cells, but we also are able to recover the relevant DNA, not only as is found in the supernatant by qPCR but also by looking at the number of colonies produced by each hourly sample when transformed. 
</P><H2><SPAN class="mw-headline" id="Thursday.2C_15_September_2011"> Thursday, 15 September 2011 </SPAN></H2><P>Lilia prepared pSB1C3 backbone from distribution KitPlate in linearised form and also transformed it DH5alpha. If the transformation works, minipreps on pSB1C3 will be done tomorrow. 
</P><P>4x1L of LB-agar were prepared and should be autoclaved by tomorrow in AI. LB-kanamycin plates are ready and will be brought to BM. 
</P><P>
Nadine tested 5 colonies on each Gibson assembly plate, but with no results... So Gibson need to be done again, this time purifying the TetRs before. She also digested the Plac biobrick fragment with EcoRI and PstI.
</P><H2><SPAN class="mw-headline" id="Friday.2C_16_September_2011"> Friday, 16 September 2011 </SPAN></H2><P>Lilia made colony PCR on muTetR variants from the 2nd batch of biobricking: TetR_Biobrick_f &amp;_n primers and EPFL taq polymerase were used. 
</P><P>
The same PCR on plasmids from the first batch of biobricked mutants gave an unexpected band at higher MW than what was expected. Same for each of five mutants.   
</P><P>The pSB1C3 biobrick backbone was linearized by PCR with the pSB_prep_2Eb and pSB_prep_3P-1 primers, but the PCR failed - there is no plasmid band in the second well on the gel. It was also transformed into DH5alpha on CM LB-agar plate, but that failed because pSB1C3 in incompatible with DH5alpha cells. 
</P><P>
Nadine ligated the Plac biobrick into pSB1C3 and transformed. She also purified the TetR genes with Gibson extensions, sending them for sequencing after having done a new Gibson reaction. The Gibson reactions were also transformed, hoping that this time it will work.
</P><P>Second attempt to PCR tetR mutants with BioBrick primers yield in PCR product for all the mutants we have:
</P><P>We will now digest and ligate it to the pSB1C3 backbone!
</P><H2><SPAN class="mw-headline" id="Saturday.2C_17_September_2011"> Saturday, 17 September 2011 </SPAN></H2><P>Nadine made colony PCRs on the Gibson assemblies with TetR mutants and on Plac-biobrick ligations. For the Plac-biobrick, all the 5 colonies tested seem to have the Plac insert.
</P><P>Doug made all the graphs of Nadine's platereader experiments. We still have some platereaders left to do: J61002 Ptet-RFP wil be done tomorrow and hopefully the TetR mutants will be ready and co-transformed on Tuesday.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:EPF-Lausanne/Notebook/September2011">http://2011.igem.org/Team:EPF-Lausanne/Notebook/September2011</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF-Lausanne/Notebook/September2011" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF-Lausanne/Notebook/September2011" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/September2011&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/September2011&amp;oldid=150114" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>