"
Page
Discussion
View source
History
teams
Log in
 
Team:EPF-Lausanne/Our Project
From 2011.igem.org
Introduction
Home
The Team
Photo Gallery
Our Project
Results Summary
Selection System
In-Vitro Characterization
In-Vivo Characterization
Microfluidics
Data
Attributions
Tools
Gibson Assembly
MITOMI
Notebook
Protocols
May
June
July
August
September
October
Considerations
Human Practices
Safety
Acknowledgements
Sponsors
Partners
Our Project
For this year’s iGEM competition our team has set out to find a method for producing new regulatory parts used in the construction of complex biological devices.
Background & Motivations
Transcription is the first step of gene expression that can be regulated. Key elements of this regulation are proteins recognising and binding specific DNA sequences that either recruit the full  transcription machinery or prevent it from assembling on the DNA. It is this class of proteins, called transcription factors (TFs), that we intend to engineer.
Existing DNA binding proteins can be a starting point for developing these new TFs. By changing their residues it is possible to modify the affinity of these proteins for a specific binding site and alter how they interact with the activation or repression mechanisms of the downstream gene transcription. Investigating these effects can improve our understanding of how the residue composition affects the TF's ability to recognise a given DNA consensus sequence.
Engineering new transcription factors will not only give us a better idea of how they work but also extend the tool kit of the synthetic biologist. Within synthetic biology, TFs are used to design biological circuits where they act as building blocks for reporters, inverters, logic functions and switches. Extending the tool kit with new TFs will enable combinations of devices that currently share the same regulatory elements and thus cannot be used in the same system.
Goal
Our project’s outcome will be a method to derive pairs of TFs and DNA binding sites meeting selection criteria for orthogonality, affinity and specificity. It will include setting a selection scheme that allows for high-throughput identification of functional TF-consensus sequence pairs in vitro and in vivo. We plan to use bacterial cultures in parallel with chemostat platforms and MITOMI to screen a library of TFs against a set of consensus sequences.
Our research will begin with the tetracycline repressor (TetR), a transcription factor which down-regulates (represses) the transcription of the adjacent gene. This protein and its mutants are often used by synthetic biologists “as control elements to regulate gene expression”   due to their detailed characterisation, but also because of the limited number of proteins available for use in the repressor-TF class.
We want to enhance the diversity of the repressor-TF class by engineering new transcription factors that can bind unique promoter sequences different from the wild-type DNA binding site normally associated with TetR. Our proposed research plan for this summer is as follows:
1) Introduce mutations in the TetR sequence and its binding site
2) Identify promising TF-binding site pairs (using in vivo and in vitro techniques)
3) Conduct multiple rounds of selection to improve affinity and specificity to optimize new pairs of repressor-TFs.
We are also developing a new negative-selection method: lysing the cells with interesting TF-binding site pairs, in order to recover their DNA. Coupled to a microfluidics device such as a chemostat chamber, our method would allow high-throughput in vivo selection.
Strategy
During the summer, we worked on 3 main sections needed to complete the whole project:
1) TetR mutants: Building mutations of TetR, either random or defined (i.e. already characterized in the litterature). This part also comprised in vitro characterization, using MITOMI setup.
2) Reporter plasmids: Assembling different plasmids allowing an in vivo characterization of TetR binding to its recognition sequence. We used RFP and a lysis cassette as a readout - either for characterization or for recovering DNA
3) T7 promoter variants: Coupling T7 promoters of different strength with our 2 reporter genes, to have a better regulation of our Reporter plasmids system. This part includes both T7 characterization with RFP and a proof-of-concept that we can efficiently lyse cells and recover their DNA.
Section 1 presents the different TetR mutants and characterizes them in vitro; sections 2 and 3 demonstrate in vivo TetR-Ptet interaction as well as recovery of DNA from the successful interactions. Details of each section can be found in the "Our project" tab.
Retrieved from " http://2011.igem.org/Team:EPF-Lausanne/Our_Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
