<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_EPF-Lausanne_Notebook_July2011"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:EPF-Lausanne/Notebook/July2011">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:EPF-Lausanne/Notebook/July2011&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/July2011&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/July2011&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:EPF-Lausanne/Notebook/July2011" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:EPF-Lausanne/Notebook/July2011</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="left"><UL id="accmenu"><LI><A class="head" href="#">Introduction</A></LI><LI><A href="/Team:EPF-Lausanne">Home</A></LI><LI><A href="/Team:EPF-Lausanne/TheTeam">The Team</A></LI><LI><A href="/Team:EPF-Lausanne/Photos">Photo Gallery</A></LI></UL><LI><A class="head" href="#">Our Project</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Summary">Results Summary</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/T7_promoter_variants">Selection System</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/TetR_mutants"><I>In-Vitro</I> Characterization</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Reporter_Systems"><I>In-Vivo</I> Characterization</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/Microfluidics">Microfluidics</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Data">Data</A></LI><LI><A href="/Team:EPF-Lausanne/Our_Project/Attributions">Attributions</A></LI><LI><A class="head" href="#">Tools</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/Gibson_assembly">Gibson Assembly</A></LI><LI><A href="/Team:EPF-Lausanne/Tools/MITOMI">MITOMI</A></LI><LI><A class="head" href="/Team:EPF-Lausanne/Notebook/September2011">Notebook</A></LI><LI><A href="/Team:EPF-Lausanne/Protocols">Protocols</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/May2011">May</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/June2011">June</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/July2011">July</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/August2011">August</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/September2011">September</A></LI><LI><A href="/Team:EPF-Lausanne/Notebook/October2011">October</A></LI><LI><A class="head" href="#">Considerations</A></LI><LI><A href="/Team:EPF-Lausanne/Human_Practices">Human Practices</A></LI><LI><A href="/Team:EPF-Lausanne/Safety">Safety</A></LI><LI><A href="/Team:EPF-Lausanne/Sponsors" class="head">Acknowledgements</A></LI><LI><A href="/Team:EPF-Lausanne/Sponsors">Sponsors</A></LI><LI><A href="/Team:EPF-Lausanne/Partners">Partners</A></LI></DIV><DIV id="right"><H1>Notebook: July 2011</H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Friday.2C_1st_of_July_2011"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Friday, 1st of July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Friday.2C_1st_of_July_2011_2"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Friday, 1st of July 2011</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Colonies_from_Gibson"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Colonies from Gibson</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Tween_experiment"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Tween experiment</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Monday.2C_4th_of_July_2011"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Monday, 4th of July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Tuesday.2C_5th_of_July_2011"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Tuesday, 5th of July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Wednesday.2C_6th_of_July_2011"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Wednesday, 6th of July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Thursday.2C_7th_of_July_2011"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Thursday, 7th of July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Friday.2C_8th_of_July_2011"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Friday, 8th of July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Saturday.2C_9th_of_July_2011"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Saturday, 9th of July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Monday.2C_11th_of_July_2011"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Monday, 11th of July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Tuesday.2C_12_July_2011"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Tuesday, 12 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Wednesday.2C_13_July_2011"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Wednesday, 13 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Thursday.2C_14_July_2011"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Thursday, 14 July 2011</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#J61002_plasmids"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">J61002 plasmids</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#pSB3C5_plasmid"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">pSB3C5 plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#tetR_mutants"><SPAN class="tocnumber">12.3</SPAN><SPAN class="toctext">tetR mutants</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Friday.2C_15_July_2011"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Friday, 15 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Monday.2C_18_July_2011"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Monday, 18 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Tuesday.2C_19_July_2011"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">Tuesday, 19 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Wednesday.2C_20_July_2011"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Wednesday, 20 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Thursday.2C_21_July_2011"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Thursday, 21 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Friday.2C_22_July_2011"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">Friday, 22 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Monday.2C_25_July_2011"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">Monday, 25 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#Tuesday.2C_26_July_2011"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">Tuesday, 26 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Wednesday.2C_27_July_2011"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">Wednesday, 27 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Thursday.2C_28_July_2011"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">Thursday, 28 July 2011</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#Friday.2C_29_July_2011"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">Friday, 29 July 2011</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Friday.2C_1st_of_July_2011"> Friday, 1st of July 2011</SPAN></H2><P>Alina and Lilia did MITOMI on His-tetR expressed from linear template, it was loaded on chip in ITT expression mix. DNA was spotted on June 29 in different concentrations for both: consensus (tetO1) sequence and a random sequence ((-)control).
</P><P>Images: Green fluorescence comes from green-lysine and red fluorescence comes from DNA (cy-5 labeled). 
</P><P>Although we don't observe much protein bound to the anti His-tag antibody, we can still see that it does bound TetO1 sequence and did not bind any (-)control sequence.  
Some possible reasons for low protein fluorescence under the buttons: the expression yield from the linear template with T7 promoter might be too low, the His-tag to antibody binding is not strong enough or the concentration of antibody is too low.
</P><H2><SPAN class="mw-headline" id="Friday.2C_1st_of_July_2011_2"> Friday, 1st of July 2011</SPAN></H2><P>Alina and Lilia amplified the His-tetR linear template. After PCR sample was loaded on 1% agarose gel with Rad Safe, “1Kb Plus DNA Ladder” was used.  
</P><P>
We also prepared about thirty LB-agar plates with ampicilin, they are in the BM fridge. 
</P><H3><SPAN class="mw-headline" id="Colonies_from_Gibson"> Colonies from Gibson </SPAN></H3><P>After an overnight incubation, we found four colonies on the Ampicillin plates that had cells containing the plasmids from the Gibson assembly. The low number of colonies is attributed to the low competence of the cells. Of the four colonies, two were pink and two were yellow.
</P><H3><SPAN class="mw-headline" id="Tween_experiment"> Tween experiment </SPAN></H3><P>Clara and Henrike tested the right dose of Tween 20 that can be used for the &quot;chemostat&quot; experiment without affecting the cell growth. Tween is a detergent that would prevent the cells from sticking and thus clogging the microfluidic channels. We tested 12 different dilutions of Tween 12 both for GFP and RFP e-coli which can be potentially used for the &quot;chemostat&quot; experiment.
We read the OD(absorption @600nm) and fluorescence every 10 minutes, for 18 hours at 37°C.
</P><P>Growth curves for all the different concentrations vs time 
</P><P>
The legend shows the percentage of tween 20 that was added to the medium.
</P><UL><LI>We can observe that the growth factor is smaller for Chloramphenicol resistant e-coli due to the fact that Chloramphenicol is stronger than Kanamycin.
</LI></UL><P><B>Calculated growth factor for the previous curves</B>(Lilia)
</P><P>Growth factor
</P><P>
We can observe that tween did not have a major effect on the cells' growth. 
I suggest we use 0.075% tween in media for the experiments
</P><H2><SPAN class="mw-headline" id="Monday.2C_4th_of_July_2011"> Monday, 4th of July 2011</SPAN></H2><P>Alina and Vincent made plates (large and small) with chloramphenicol (concentration of 34 mg/mL). We used .74 mL of chloramphenicol for a liter of LB agar and plated them directly under the flame. 
</P><P>
To figure out what was going wrong with the P23019 plasmid, we used the Nanodrop to determine that the sample (Plate 4, well 14E) was at a concentration of 97.4 ng/microL. This result seemed to indicate that there was DNA in the sample of the plate (as opposed to just red dye without DNA). 
</P><H2><SPAN class="mw-headline" id="Tuesday.2C_5th_of_July_2011"> Tuesday, 5th of July 2011</SPAN></H2><P>Since  we were running out of the P23019 plasmid DNA to try new PCRs, we thought it would be a good idea to transform it. We used both Lilia's and Alina's cells, adding a negative control. The incubation for chloramphenicol required a two hour recovery period in the incubator, since the antibiotic is stronger. After Tuesday's meeting, we went ahead and plated the result and let the plates incubate overnight. 
</P><P>Meanwhile, moving on with the Gibson assembly of the J61002 plasmid, Vincent tried to obtain the final Gibson sequence. This sequence would be used to see if PstI and SpeI would cut differently on the assembled plasmid versus the normal J61002 plasmid: we hoped this would allow us to check that Gibson had worked successfully if we ran the ligated plasmids on a gel. Ideally, the J61002 original plasmid would cut at two locations while the Gibson plasmid would only cut in a single location (the biobrick scar would make it impossible to cut at the previous cut location). On a gel, we would see a single piece of about 3000 bp for the Gibson plasmid and two bands of approximately 2200 and 700 bp respectively for the normal plasmid.
</P><P>Using the plates with the four Gibson colonies, Vincent made cultures which he put in the incubator overnight. 
</P><H2><SPAN class="mw-headline" id="Wednesday.2C_6th_of_July_2011"> Wednesday, 6th of July 2011</SPAN></H2><P>With the Gibson cultures in hand, Vincent made glycerol stocks (500 microL glycerol, 500 microL cells) and then mini-prepped the four cultures using the standard protocol. The resulting DNA plasmids were ready for enzyme ligase.  
</P><P>Vincent also looked at the results of the transformation of the P23019 using two different sets of competent cells and negative controls. No colonies grew, leaving us quite perplexed as to what was in the original P23019 well. Determined to find a substitute for the P23019 plasmid that would serve as a vector for the TetR sequence, we searched through the database looking for a plasmid with resistance to something other than ampicillin as well as a P15a origin of replication. We found four distinct possibilities, only two of which were available in the plates from the standard registry: pSB3K1 which has kanamycin resistance, and pSB3C5 which has chloramphenicol resistance (very similar to p23019).  
</P><H2><SPAN class="mw-headline" id="Thursday.2C_7th_of_July_2011"> Thursday, 7th of July 2011</SPAN></H2><P>Vincent and Nadine made a digestion on the minipreps from the 4 colonies that were transformed with the Gibson-extended J61002 plasmid. If Gibson was successful, only Pst1 will cut and we should have one band of 3094 bp. If Gibson didn't work, then Pst1 and Spe1 will cut and we should have 2 bands of 890 and 2060 bp.
The digestion was made according to last year's protocol, with 1.5 hour incubation. The ladder on the gel is 1kb.
</P><P>Colonies 1 and 4 have the expected band over 3kb, which is coherent with the observation made that they did express RFP. But colonies 2 and 3 have only one band... It is likely that in these cells the plasmid recombined in a shorter way, without RFP but with the resistance; otherwise the plasmid sequence is not what we think. We'll check tomorrow on the J61002 plasmid if we do get 2 bands on the starting plasmid.
Also, we have some light smaller bands in colonies 1 and 4, indicating that the miniprep is perhaps not so pure.
</P><P>Vincent and Alessandro transformed the J61002 original plasmid, since we were running out of DNA with which to compare to the Gibson assembly. In addition, they transformed the results of the previous day's registry search for convenient substitute plasmids (for p23019) using Alina's cells using 50 ng/microL concentration (Chloramphenicol Chl-SB3C5 70.6 ng/microL, K-SB3K1 67.7 ng/microL). These were plated appropriately and put in the incubator overnight. 
</P><P>
Lilia and Alessandro did a MITOMI experiment on deBrujin sequences. Non of the sequences showed up to be bound nicely. But the protein pull down was not so robust due to the surface chemistry problems... Experiment needs to be repeated.
</P><H2><SPAN class="mw-headline" id="Friday.2C_8th_of_July_2011"> Friday, 8th of July 2011 </SPAN></H2><P>Thanks to Henrike's expert eyes, we were able to harvest many colonies from the chloramphenicol pSB3C5 plasmid as well as many for the J61002 plasmid and the kanamycin plasmid. Since the chloramphenicol plasmid seemed closest to the p23019 plasmid for which we already had primers, it made sense to pursue the Gibson strategy for that plasmid. Alessandro and Vincent made culture tubes and put those in the incubator overnight. In the meanwhile, primers were designed for the next phase of Gibson assembly (i.e. the new pSB3C5 plasmid with the old TetR piece). 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel from the tetR mutants PCR. 1000kb ladder.</DIV></DIV></DIV><P>We ran the PCR on the tetR linear template, using Clara's primers for PCR mutagenesis.
</P><P>Six PCRs were run. The first amplified the common sequence of the mutants: everything up to the mutated sites. The six other reactions amplified the second half of the gene, inducing specific mutations in tetR.
</P><P>The fluorescence traces are insufficiently clear for accurate conclusions, therefore the PCR and gel will be repeated with different concentrations on Monday.
</P><H2><SPAN class="mw-headline" id="Saturday.2C_9th_of_July_2011"> Saturday, 9th of July 2011 </SPAN></H2><P>Vincent came in to pick up the J61002, pSB3C5, and pSB3K1 cultures from the incubator. He found that all the J61002 cultures were pink (as expected since they have the RFP gene). Both kanamycin cultures were pink, and two of the three chloramphenicol cultures were pink (though not as strong a pink as that of the J61002 plasmid). He went ahead and mini-prepped all the cultures, using hot water (55 C) instead of warm TE buffer during the elution step. He then made glycerol stocks (50% cell, 50% glycerol) and placed them in the -80 freezer. Finally, he measured the concentration of the resulting DNA after the mini-prep:
</P><P>pSB3C5, colony 1: 35.2 ng/microL (accidentally used 2 microL, vs. 1 microL so more diluted)
</P><P>
pSB3C5, colony 2: 78.4 ng/microL
</P><P>
pSB3C5, colony 3: 70.2 ng/microL
</P><P>
pSB3K1, colony 1: 15.5 ng/microL
</P><P>
pSB3K1, colony 2: 14.3 ng/microL
</P><P>
J61002, colony 1: 54.2 ng/microL
</P><P>
J61002, colony 2: 74.3 ng/microL
</P><P>
J61002, colony 3: 45.9 ng/microL
</P><H2><SPAN class="mw-headline" id="Monday.2C_11th_of_July_2011"> Monday, 11th of July 2011 </SPAN></H2><P>Nadine made a PCR to prepare next Gibson assembly, ran it on a gel and recovered the according fragments from the gel. Details of the 4 PCRs:
</P><UL><LI> ColE1 backbone with Ptet (from newly assembled J61002 plasmid), final concentration: 15.1 ng/ul
</LI><LI> LacI ssrA rrnB T on repressilator plasmid, final concentration: 46.7 ng/ul
</LI><LI> T4 lysis cassette on registry plasmid, final concentration: 29.7 ng/ul
</LI><LI> RFP amplification with Plac from Gibson J61002 plasmid, final concentration: 61,2 ng/ul
</LI></UL><P>
Alessandro made again a digestion on the 4 colonies recovered from last Gibson assembly, with this time also the original J61002 plasmid. See Thursday 7th July for more details. This confirms that colony 3 has somehow reassembled without RFP because it is smaller than colonies 1 and 4. The bands are bigger after digestion because the plasmids have become linear and they run slower than the coiled plasmids.
</P><H2><SPAN class="mw-headline" id="Tuesday.2C_12_July_2011"> Tuesday, 12 July 2011 </SPAN></H2><P>Douglas obtained satisfactory amounts of product from the mutation PCR on the tetR linear template. The most successful one required 1 uM of primers, mixed to the 100x diluted template (0.327 ng/uL concentration).
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel from the first successful variants PCR. Codes indicate the point mutations.</DIV></DIV></DIV><P>Again, seven PCRs were run. The first copies the common sequence, up to the mutated bases in the tetR coding region, with some padding (illustrated above). The next six copy the remaining sequence, and induce these specific mutations:
</P><OL><LI> VF 36
</LI><LI> EA37
</LI><LI> PK39
</LI><LI> YF42
</LI><LI> PQ39YM42
</LI><LI> PQ39LV41YM42
</LI></OL><P>The gel fluorescence traces show what appears to be PCR product in large amounts for the Common and EA37 reactions, and smaller amounts for the YF36 and YF42 reactions. The remaining three show almost no activity.
</P><P>The PCR products were subsequently separated using gel electrophoresis then cut out. They will be purified tomorrow.
</P><P>Nadine made 2 Gibson assemblies with the PCR products form yesterday, transformed cells and plated them on ampicillin dishes. The newly assembled vectors will have ampicillin resistance, LacI under Ptet plus either lysis under Plac or RFP under Plac.
</P><H2><SPAN class="mw-headline" id="Wednesday.2C_13_July_2011"> Wednesday, 13 July 2011 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">First result from the fusion PCR: tetR mutant EA37 appears in &quot;detectable&quot; amounts</DIV></DIV></DIV><P>Douglas purified the gels containing yesterday's mutation-PCR products, following the <A href="/Team:EPF-Lausanne/Protocols/Gel_purification" title="Team:EPF-Lausanne/Protocols/Gel purification"> gel purification protocol</A>. Isopropanol was added to the &quot;Common PCR&quot; product, as its expected length is inferior to 500bp. The final DNA concentrations, measured by photospectrometry, are listed in the following table. Even the very low concentration of YF36 should be sufficient for a PCR template, and the weird &quot;260/280&quot; ratio apparently results from agarose gel residues.
</P><TABLE><CAPTION> DNA concentrations of the extension PCR products, after gel purification.
</CAPTION><TBODY><TR><TH><B>PCR #</B></TH><TH><B>Ref.</B></TH><TH><B>Conc. [ng/ul]</B></TH><TH><B>&quot;260/280&quot;</B></TH></TR><TR><TD> 1 </TD><TD> Common </TD><TD> 17.8 </TD><TD> 4.82
</TD></TR><TR><TD> 2 </TD><TD> YF36 </TD><TD> 1.9 </TD><TD> -0.26
</TD></TR><TR><TD> 2 (repeat) </TD><TD> YF36 </TD><TD> -4.4 </TD><TD> 0.36
</TD></TR><TR><TD> 3 </TD><TD> EA37 </TD><TD> 26.3 </TD><TD> 96.23
</TD></TR><TR><TD> 5 </TD><TD> YF42 </TD><TD> 39.4 </TD><TD> -5.24
</TD></TR><TR><TD> 5 (repeat) </TD><TD> YF42 </TD><TD> 40.1 </TD><TD> -2.41
</TD></TR></TBODY></TABLE><P>
These sequences were then used for a fusion PCR, to stitch mutants YF36, EA37, and YF42 to the common sequence, according to the <A href="/Team:EPF-Lausanne/Protocols/TetR_Extension_PCR" title="Team:EPF-Lausanne/Protocols/TetR Extension PCR">overlap extension PCR protocol</A>. Only the EA37 mutant, which was also the brightest on the PCR gels, yielded a significant amount of product. The next step is to check the product is the desired mutant, in order to confirm the success of this method. We also have to refine the PCR to get the other mutants to work; it seems the problem comes from the primers' melting temperature.
</P><P>
Alessandro and Nadine autoclaved beads and prepared 4 bottles of SOC medium. We took the plated cells (J61002 Ptet-LacI Plac-lysis &amp; Ptet-LacI Plac-RFP) from incubator and put them in suspension (LB medium + ampicillin) to grow overnight. We also plated the rest of these transformed cells.
We did a Klenow reaction on the 1off library to have dsDNA. The ordered plasmids arrived,so we made a PCR on pSB3C5 (to amplify the backbone) and on TetR (adding pConst) to prepare the parts needed for Gibson.
</P><H2><SPAN class="mw-headline" id="Thursday.2C_14_July_2011"> Thursday, 14 July 2011 </SPAN></H2><H3><SPAN class="mw-headline" id="J61002_plasmids"> J61002 plasmids </SPAN></H3><P>Alessandro made minipreps on cells grown overnight and made glycerol stocks of them (500 microL).
Because the cells that we assume to have RFP (under the control of Plac) are not a little red, we also kept an aliquot of these cells to make a test with IPTG (this is a molecular mimic of allocaltose therefore it triggers the transcription on Plac): if these cells become red with IPTG we can be sure that RFP is controlled by Plac in our plasmid (from Gibson assembly).
</P><P>
Alessandro digested the plasmind (from miniprep) with PstI because Nadine found that if the Gibson went well, they should have 2 restricition sites for this enzyme - instead of 1 for J61002 Ptet-RFP plasmid. Unfortunately, the results show that GIbson didn't work... For every colony we have a band at 2.5 kb, which corresponds exactly to the size of the backbone.
</P><P>The backbone must have closed without the inserts... We can redo Gibson with less backbone. On the plates we prepared yesterday one colony is red! We suspended it, we'll check tomorrow if this one has the correct plasmid.
</P><H3><SPAN class="mw-headline" id="pSB3C5_plasmid"> pSB3C5 plasmid </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">pSB3C5 backbone purification, both bands contain the same product.</DIV></DIV></DIV><P>The PCR to add Pconst to TetR didn't work, but we already have it in the freezer. Nadine purified the pSB3C5 backbone, but the bands on the gel were really weak even with all the PCR products loaded onto the gel and the concentration is only 6 ng/ul. We will anyway try Gibson tomorrow...
</P><H3><SPAN class="mw-headline" id="tetR_mutants"> tetR mutants </SPAN></H3><P>Douglas designed primers for site-specific mutagenesis of tetR, following the <A href="/Team:EPF-Lausanne/Protocols/Site-specific_mutagenesis" title="Team:EPF-Lausanne/Protocols/Site-specific mutagenesis">protocol</A>. We are testing this as another approach to inducing mutations. The primers were designed for the same mutations as attempted by extension PCR, i.e.:
</P><OL><LI> V36F
</LI><LI> E37A
</LI><LI> P39K
</LI><LI> Y42F
</LI><LI> P39Q_Y42M
</LI><LI> P39Q_L41V <I>([26 July 2011] The Y42M mutation was left out here)</I></LI></OL><P><S>By laziness</S> For convenience, the primer codes were changed to match those used by Agilent's primer design software. For example, V36F indicates a substitution of amino acid F for V in position 36. The primers are listed in this Google spreadsheet: <A href="https://spreadsheets.google.com/spreadsheet/ccc?key=0AvMKvyibLKKxdENDWkNLRV9ZSHhvVU45RHQ0bnNCWFE&amp;hl=en_US" class="external free" rel="nofollow">https://spreadsheets.google.com/spreadsheet/ccc?key=0AvMKvyibLKKxdENDWkNLRV9ZSHhvVU45RHQ0bnNCWFE&amp;hl=en_US</A></P><H2><SPAN class="mw-headline" id="Friday.2C_15_July_2011"> Friday, 15 July 2011 </SPAN></H2><P>The IPTG test gave us bad results (as expected due to data came from digestion): no increase in RFP and no decrease for the T4 lysis cassette.
Alessandro made miniprep and the digestion for the only red colony that he inoculated the day before.
The digestion gave a band of the expected size of the Gibson assembly (4428 bp) but we were expecting to see 2 bands instead we can see just 1; maybe one restriction site has been removed by the PCR or by the assembly. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:122px;"><DIV class="thumbcaption">Gibson assembly from the red colony doesn't show 2 bands but just one band between 4 and 5 kb. Maybe one restriction site had been lost during the assembly.</DIV></DIV></DIV><P>
Nadine made Gibson reactions for pSB3C5-pConst-TetR and for J61002-Ptet-LacI-Plac-RFP/lysis:
</P><UL><LI> pSB3C5:
</LI><LI> pSB3C5 backbone (purified from gel yesterday)
</LI><LI> pConst-TetR (the one Vincent had made for J23019 plasmid)
</LI></UL><LI> J61002 Ptet-LacI Plac-RFP
</LI><LI> J61002-Ptet backbone (3 ul instead of 4 ul last time)
</LI><LI> LacI ssrA rrnB T (2x more than last time)
</LI><LI> Plac-RFP (2x more than last time)
</LI><LI> J61002 Ptet-LacI Plac-lysis
</LI><LI> J61002-Ptet backbone (3 ul instead of 4 ul last time)
</LI><LI> LacI ssrA rrnB T (2x more than last time)
</LI><LI> Plac-T4 lysis cassette (2x more than last time)
</LI><P>We hope that with less backbone we will have a smaller proportion of plasmids with only the J61002 backbone.
</P><P>
Douglas and Nadine ran a PCR to prepare another Gibson assembly: adding pConst-TetR on J61002 Ptet-RFP plasmid.
</P><UL><LI> backbone:
</LI><LI> primers: J61002-plasmid-f + J61002-Plasmid-r
</LI><LI> template: J61002 Ptet-RFP (we took colony 1)
</LI></UL><LI> TetR:
</LI><LI> primers: J61002 TetR-f + J61002 TetR-r
</LI><LI> template: repressilator plasmid
</LI><P>Unfotunately, none of these 2 reactions worked because there was no band to see in the UV light...
</P><H2><SPAN class="mw-headline" id="Monday.2C_18_July_2011"> Monday, 18 July 2011 </SPAN></H2><P>Alessandro made IPTG stocks and inoculated cells of the red colony (from glycerol stock, plasmid J61002-LacI-RFP) in order to use them the next day to perform the IPTG test.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Nothing to see on the gel</DIV></DIV></DIV><P>Nadine transformed cells with:
</P><UL><LI> pSB3C5-pConst-TetR
</LI><LI> J61002 ptet-LacI pLac-RFP
</LI><LI> J61002 ptet-LacI pLac-lysis
</LI></UL><P>and plated them on ampicillin or chloraphenicol dishes. We need to wait until Wednesday to be able to test these plasmids.
</P><P>
To prepare next Gibson, Nadine aslo re-did the PCR on J61002-Ptet backbone and on TetR. Details are as follows:
</P><UL><LI> backbone:
</LI><LI> primers: J61002-plasmid-f + J61002-Plasmid-r
</LI><LI> template: J61002 Ptet-RFP (this time colony 4)
</LI></UL><LI> TetR:
</LI><LI> primers: J61002 TetR-f + J61002 TetR-r
</LI><LI> template: repressilator plasmid
</LI><P>We took 62°C for annealing step and 1min for extension step. But it didn't work... 
I checked the primers features and no hairpin or dimers formation seems to be problematic (delta G's were -14 kcal/mol for the worse cases). Based on Henrike's advice, I checked the melting temperatures for only the parts of the primers that actually bind to template DNA and they are significantly lower... So Nadine ran a PCR with 42°C annealing step instead of 62°C, let's see tomorrow if it's better.
</P><H2><SPAN class="mw-headline" id="Tuesday.2C_19_July_2011"> Tuesday, 19 July 2011</SPAN></H2><P>Alessandro started the IPTG test and the results will be available the next day.
</P><P>The PCR with 42°C for annealing step didn't work either... We'll put this assembly aside, hoping that we will be able to insert TetR in others plasmids.
</P><P>The Gibson assemblies made last Friday and transformed yesterday show disappointing results:
</P><UL><LI> J61002 Ptet-LacI Plac-RFP: only two colonies and they both are white
</LI><LI> J61002 Ptet-LacI Plac-lysis: no colonies
</LI><LI> pSB3C5 Pconst-TetR: a lot of red colonies, although they should not have RFP
</LI></UL><P>For the J61002 assemblies, we will do the PCRs again to have fresh parts, and then try again. For pSB3C5 the results are really strange, as we don't think RFP should be included in the original plasmid. Also, Vincent's results for the miniprep of the original are suspiciously high (60 ng/ul is more than expected)so perhaps it's not the plasmid we think it is. We will then try to use pSB3K1 plasmid.
</P><P>Nadine and Alessandro designed the plasmid for the new Gibson assembly to put P-const-TetR on a new backbone (pSB3K1) because the pSB3C5 didn't give the expected results (high DNA concentration while expected low copy number).
</P><H2><SPAN class="mw-headline" id="Wednesday.2C_20_July_2011"> Wednesday, 20 July 2011 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">The first image is the plot of the OD whereas the second is the plot of fluorescence of the RFP</DIV></DIV></DIV><P>The results from the IPTG test (picture below) show a slight difference in RFP fluorescence between the sample (w/ IPTG) and the negative control (w/ water) but this difference isn't significant therefore we must consider this as a bad result: we know our plasmid is not <A href="/wiki/images/5/54/EPFL_Plasmids%28RFP_instead_of_Lysis%29.png" class="internal" title="EPFL Plasmids(RFP instead of Lysis).png">as expected (the plasmid on the right)</A>, but we don't know what we have.
</P><P>Alessandro made the transformation of the Repressilator plasmid because we haven't a sufficient amount to start PCRs.
Once we have that plasmid we can re-do PCRs to get the part for the Gibson assembly (RFP/Lysis plasmids).
Alessandro also designed the primers to do the colony-PCR wich we'll help us in a faster fashion to screen colony with the correct plasmid from the gibson assembly (instead of making minipreps and the following DNA digestion).
</P><P>Douglas prepared agar-ampicillin plates for the mutagenesis, and transformed a pF3A plasmid with GFP (and ampicillin resistance). It can be used for a trial run of mutagenesis if the gibson assembled tetR plasmid is not yet available. He also transformed some repressilator, so we should have plenty to keep us going for a while. Also, Nadine debriefed him on her work for the past few weeks, so he can take over while she is on holiday.
</P><H2><SPAN class="mw-headline" id="Thursday.2C_21_July_2011"> Thursday, 21 July 2011 </SPAN></H2><P>Douglas and Alessandro inoculated the colonies transformed with the repressilator plasmid to make minipreps the following day. Alessandro also made the LB broth (because one bottle got contaminated) and distilled water.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gel from our first attempt to replicate Nadine's PCR, to create fragments for the J61002-based reporter plasmid. None of the expected products show up: they should all be between 700 b and 2.4 kb</DIV></DIV></DIV><P>Douglas and Alessandro repeated Nadine's unsuccessful PCR, to make the <B>fragments for Gibson assembly of the 'reporter plasmid' </B> (pTet-lacI-pLac-RFP/Lysis). The PCR was just as <B>unsuccessful</B>, showing no trace of the expected products, as shown on the figure to the right. The fragments being assembled are:
</P><UL><LI> A J61002 <B>backbone</B> amplified from the J61002-pTet-RFP plasmid
</LI><LI><B>LacI</B> amplified from the Repressilator plasmid
</LI><LI><B>pLac-RFP</B> from the T4 Lysis plasmid
</LI><LI> OR <B>pLac-T4Lysis</B> also from the J61002-pTet-RFP plasmid
</LI></UL><P>We found two similarly labelled tubes containing J61002-pTet-RFP, so we used both for our PCRs. The 'bis' reactions used the plasmid miniprepped by Allesandro, labelled in green writing. The 'non-bis' reactions used the other plasmid, of mysterious origin (miniprepped by Vincent perhaps?). 
</P><P>New reagents were prepared to repeat the PCR, using this time the optimal concentrations recommended by Bio-Rad for the iProof polymerase, i.e. primer concentrations of 0.5 uM and DNA template quantities between 1 pg and 10 ng for 50 ul. Specifically, we prepared a set with 1:1000 diluted template (resulting in a mass between 25 and 100 pg) and a set with 1:100 diluted template (resulting in a mass between 0.25 and 1 ng). They have been frozen with no polymerase, to attempt a touchdown PCR tomorrow.
</P><H2><SPAN class="mw-headline" id="Friday.2C_22_July_2011"> Friday, 22 July 2011 </SPAN></H2><P>Alessandro made minipreps for the repressilator plasmid.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">*EPIC FAIL* - Touchdown PCR to amplify the fragments for Gibson assembly of the two reporter plasmids</DIV></DIV></DIV><P><B>Touchdown PCR</B> was run on yesterday's samples, to amplify the <B>fragments for Gibson assembly of the reporter plasmids</B>, using the Bio-Rad thermal cycler. The thermal cycles were designed according to the iProof datasheet, using touchdown PCR recommendations from Korbie &amp; Mattick (2008) [1]. Since the primer melting temperatures range from 47° C to 60° C, the cycle step the annealing temperature down from 65° C to 45° C over 20 cycles in constant decrements. It then repeats another 20 cycles at the 45° C floor temperature. Elongation lasts 40 s, following Bio-Rad's recommendation of 15 s/kb, for a 2400 kb product. The other parameters are as usual for iProof polymerase.
</P><P>The PCR yielded absolutely <B>no products</B>. To further investigate the cause of failure, the PCR has been <B>repeated</B> using the Roche <B>High Fidelity PLUS</B> PCR kit, adapting the thermal cycles to the new polymerase. In a desparate move, the old PCR products were re-used for an almost-identical run on the Eppendorf cycler, with slightly longer elongation and denaturation steps (50 s and 7 s, respectively).
</P><P>﻿1. Korbie, D.J. &amp; Mattick, J.S. Touchdown PCR for increased specificity and sensitivity in PCR amplification. Nature protocols 3, 1452-6(2008).
</P><H2><SPAN class="mw-headline" id="Monday.2C_25_July_2011"> Monday, 25 July 2011 </SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Gibson fragments for the reporter plasmid, PCR'd with HifiPLUS enzyme. Good yield, but some unexpected product sizes, especially for the backbone.</DIV></DIV></DIV><P>Alessandro ran the Gel on Friday's PCR - that made Gibson fragments for the reporter plasmid, using the HifiPLUS enzyme. There is definitely some product (whereas iProof yields nothing at all). LacI and RFP seem to have worked properly. The lysis fragment is possibly a little short, and the backbone band is way too small. Overall, except for the backbone, it worked well.
</P><P>In conclusion: our <B>iProof enzyme seems not to work </B>, either because it degraded, or it is just not suitable for this type of experiment. We should now use another enzyme. We ordered some High Fidelity PLUS for the next PCRs.
</P><P>To investigate the lack of backbone product, Douglas ran a gradient PCR for the backbone fragment and the T4Lysis fragment. Five annealing temperatures were tested for each. The row temperatures are listed in the table below. To straddle their primers' expected annealing temperature, the backbone was placed in rows D--H and the Lysis fragment in rows A--E. Please note this PCR was run with the <B>standard High Fidelity (not PLUS)</B> enzyme!
</P><TABLE><CAPTION> Temperature settings for the gradient PCR
</CAPTION><TBODY><TR><TH> Row </TH><TH> Temperature [°C]
</TH></TR><TR><TD> A </TD><TD> 65.0
</TD></TR><TR><TD> B </TD><TD> 63.8
</TD></TR><TR><TD> C </TD><TD> 61.6
</TD></TR><TR><TD> D </TD><TD> 57.6
</TD></TR><TR><TD> E </TD><TD> 52.7
</TD></TR><TR><TD> F </TD><TD> 48.9
</TD></TR><TR><TD> G </TD><TD> 46.2
</TD></TR><TR><TD> H </TD><TD> 45.0
</TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Tuesday.2C_26_July_2011"> Tuesday, 26 July 2011 </SPAN></H2><P>Alessandro made PCR to amplify pSB3K1 backbone to use for a gibson assembly to make the Pconst-TetR plasmid. We can't do the assembly now since we are waiting for primers for TetR.
The PCR gave good result for one colony (on two) as we can see a big band of the expected size (3.1 kb).
This time the purification will be performed not from the gel but directly from PCR with a commercial kit.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">pSB3K1 successfully amplified by PCR</DIV></DIV></DIV><P>Alessandro send the Vincent's plasmid (J6001-Ptet-RFP) to be sequenced (as the primer arrived).
</P><P>All the mutagenesis primers have been delivered, so Douglas diluted them. They are now all at 1 ug/ul concentrations, and need a further 1:10 or sodilution before being used for mutagenesis. He then ran a first mutagenesis on Alina's pF3A-tetR-GFP plasmid, introducing the following five mutations, in addition to a control reaction:
</P><OL><LI> V36F
</LI><LI> E37A
</LI><LI> P39K
</LI><LI> P39Q_Y42M
</LI><LI> P39Q_L41V
</LI></OL><P>The mutagenesis kit (remains from last year) was almost empty, so he made 10 ul instead of 50 ul reactions. There should be enough remaining reagents for one more batch, possibly with control, but hardly any more.
</P><P>The mutated plasmids were frozen over night, in order to digest and transform the following day.
</P><P>A gel was run for the gradient PCR, that used HiFi enzyme. It gave no product! In conclusion, we shall wait for the High Fidelity PLUS Enzyme, which should get here tomorrow.
</P><P>In the mean time, Doug ran an identical gradient PCR with Matt's HiFiPLUS enzyme, changing the buffer concentration to 10 ul per 50 ul and lengthening the extension step to 2'30&quot;, as recommended by the datasheet.
</P><H2><SPAN class="mw-headline" id="Wednesday.2C_27_July_2011"> Wednesday, 27 July 2011 </SPAN></H2><P>Alessandro purified the pSB3K1 backbone (to use for Gibson assembly and make the Pconst-TetR plasmid) from the PCR of the day before getting 60.6 ng/uL.
The sequencing reaction failed and checking the conditions of primers and template with NanoDrop, it seems like reverse primer has no DNA (but this doesn't explain why the reaction failed also with the forward primer); there's also a weird thing to annotate: checking the template on NanoDrop we always have an error of calibration.
Alessandro made new sample to send again sequencing.
</P><P>We <B>FINALLY HAVE A WORKING PCR!!!!1111one'</B>.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Backbone and Lysis fragments for the reporter plasmids, PCR'd with HifiPLUS. Expected product lengths: 2.4 kb backbone, 1.9 kb Lysis cassette.</DIV></DIV></DIV><P>More seriously, a gel was run on yesterday's gradient PCR products --- the PCR was ran using the High Fidelity PLUS enzyme, on the J61002-pTet backbone and pLac-T4Lysis fragments, for the Gibson assembly of the reporter plasmids. For both fragments, high quantities of specific product were yielded, though the backbone also contains large amounts of non-specific product.
</P><P>For the backbone, lower annealing temperatures yield more product, as expected. The non-specific product seems relatively unaffected by annealing temperature (or it may decrease slightly at lower temperatures). The unspecific product present in large amounts is of similar length to the RFP gene, which is also present on the backbone template --- but determining its origin is unnecessary: it doesn't contain an antibiotic resistance gene, so plasmids assembled from it would not allow culture growth in the amplification step.
</P><P>The T4 Lysis cassette fragment is a little too small. It could be due to GelRed affecting migration speed (according to Alina, it happens with large amounts of DNA), or it could indeed be an undesired product of shorter length. Please note the product quantities are barely affected by temperature (to the extent we can eyeball from the gel), whereas the expected primer melting temperatures are between 52° and 57°C, so we do not expect any product with 65°C annealing steps.
</P><P>The assembly step was started. Transformation can be started tomorrow morning.
</P><P><B>For the following PCRs</B>: Use Roche High Fidelity PLUS enzyme, and adapt any protocols to the appropriate specifications by Roche. Also, use either bottled water or the newer jar with fresher water (the one with autoclave tape on the lid).
</P><H2><SPAN class="mw-headline" id="Thursday.2C_28_July_2011"> Thursday, 28 July 2011 </SPAN></H2><P>The sequencing reaction this time went good therefore the plasmid is definitely the one expected.
Alessandro transformed and plated the plasmid to perform the day after the colony PCR just to optimize the protocol (this will be our positive control) for this technique that will be used to screen the gibson assembly.
</P><P>Since seems to be some debris in the SOC medium bottle, we also plated it (under the same conditions of the cells) to test if something grows. Alessandro also tested the OD of the medium after 1 hour of incubation and no significant difference was found.
</P><P>Douglas transformed and plated the Gibson-assembled reporter plasmids at the same time as Alessandro's plasmids. He later transformed it in a different <I>E. Coli</I> colony that over-expressed LacI, and therefore has a reduced probability of expressing the holin (from the lysis cassette) before they synthesise enough LacI to repress it.
</P><P>Douglas also repeated the <B>mutation PCR</B> using the new High Fidelity PLUS enzymes. In a bold and dareful move, he programmed the thermal cycler to carry out both a touchdown PCR AND progressively extended elongation steps (during the second set of cycles, those at the floor annealing temperature, the elongation step is extended of five seconds every cycle).
</P><H2><SPAN class="mw-headline" id="Friday.2C_29_July_2011"> Friday, 29 July 2011 </SPAN></H2><P>Douglas and Alessandro's transformed cells didn't grow (maybe problems with competent cells?).
Alessandro transformed the cells again (with the J61002-Ptet-RFP Gibson plasmid) but this time he trasformed also the sample labeled Col1-RFP and Col4-RFP (supposing that these are the same as J61002-Ptet-RFP Gibson plasmid).
Alessandro transformed the Gibson assembly made by Douglas on Lilia's (or Alina's?) competent cells and also on Henrike's competent cells. Due to an error only the lysis cassette has been transformed. On Tuesday the Gibson assembly has to be repeated since we ran out of the previous one.
</P><P>The Mutation PCR yielded no product, probably because the primers were too diluted. It has been repeated with a higher primer concentration.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:EPF-Lausanne/Notebook/July2011">http://2011.igem.org/Team:EPF-Lausanne/Notebook/July2011</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:EPF-Lausanne/Notebook/July2011" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:EPF-Lausanne/Notebook/July2011" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/July2011&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:EPF-Lausanne/Notebook/July2011&amp;oldid=62071" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>