"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Munich/Project/Vector Design
From 2012.igem.org
Project
Results
Safety
Vector Design
Limonene
Thaumatin
Caffeine
Xanthohumol
Constitutive Promoter
Light-Switchable Promoter
Ethanol-Inducible Promoter
Genome Integration
Brewing
BioBricks
Data Page
RFC
Modeling
Methods
Priors
Gal1 Promoter
Yeast Growth
Human Practice
Overview
School Visit
Public Relations
Comic
Action Day
Politics
Movie
Radio Interview
Survey
Notebook
Methods
Labjournal
Meetings
Attributions
Team
Members
Activities
Collaborations
Contents
1Vector Design
1.1Design
1.2Results
1.2.1BioBrick backbones
1.2.1.1Induction assays with galactose
1.2.1.2Outlook and conclusions
1.3References
Vector Design
Responsible: Ingmar Polte
What is the use of DNA sequences coding for valuable enzymes without the possibility to express them and analyze the protein activity?
As we planned to detect enzymes from our biosynthetic pathways via Strep-tag II, a RFC25 compatible backbone was necessary. As no such backbone was available for yeast in the Registry of Standard Biological Parts, an important task at the beginning of our project was the design of an expression vector for yeast compatible to the iGEM cloning principles and standards.  Based on the commercially available pYES2 vector, manufactured by Invitrogen, we created a vector containing:
a RFC 25 compatible multiple cloning site (MCS)
a Strep-tag II
a 2µ origin for high copy number replication in Saccharomyces cerevisiae
the URA3 gene to use the uracil prototrophy of S. cerevisiae INVSc as a selection marker for   successful transfection
a pUC origin for high copy number replication in E.coli
the ß-lactamase coding gene to use ampicillin as a selection marker for cloning applications in E.coli
a galactose inducible PGal1 promoter
a CYC1 transcription terminator
Furthermore we designed several vectors containing our constitutive promoters Tef1, Tef2 and ADH and different additional terminators. The diverse versions of the vector have been applied and tested in all other subprojects  successfully.
Design
The project work began with the replacement of the original MCS for a completely new designed MCS containing the typical RFC10/25 pre- and suffixes. Furthermore we integrated a Strep-tag II coding gene sequence previous to the suffix. This facilitates purification and detection of expressed enzymes via Western Blot  dramatically. Construction of the new MCS by four desoxyribooligonucleotides via oligonucleotide hybridization and ligation into the original pYES vector was restricted with the outermost restriction enzymes of the old MCS.
Per side directed mutagenesis all forbidden restriction sides of enzymes used in RFC10/25 standard in the vector backbone were deleted. Different vector samples of this successive process are shown in the picture below. Their have been digested by NgoMIV, PstI and SpeI: The resulting fragments were decreased with each step leading to the pure vector backbone linearized by cutting in MCS. Hence our first original galactose inducible pGal1 promoter containing expression vector was ready for cloning with iGEM standards!
A second step was the exclusion of f1 origin of replication for phage λ and of pGAL1 promoter. The resulting vector pTUM100 can be used as powerful basis for integration of a wide variety of user defined promoters, genes and terminators. In our case we integrated the constitutive promoters Tef1, Tef2 and ADH in order to characterize them.
Results
Fig. 1(A) shows the new MCS containing RFC 10 /25 restriction sides and DNA sequence coding for the Strep-tag II.
Fig. 2(A) shows the new MCS containing RFC 10 /25 restriction sides and DNA sequence coding for the Strep-tag II. (B) gives an overview of all important functional elements located on the vector backbone. A T7 promoter primer binding site allowing easy forward sequencing of integrated gene constructs using standard T7 primer lies upstream to the new MCS. The URA3 gene is a prototrophy marker used for selection of transfected cells. (C) to (E) present the successful designed BioBricks:  * pTUM100 simply contains the new MCS, the transcription terminator and further elements required for cloning and transfection.  * In addition pTUM101, pTUM102 and pTUM103 contain the constitutive promoters pTef1, pTef2 and pADH.  * pTUM104 keeps the same constitutive promoters and the galactose inducible promoter pGAL1.
The figure on the right gives an impression of all important elements located on vector backbone:
The T7 promoter primer binding site allows easy forward sequencing of integrated gene constructs using standard T7 primer.
The new RFC 10 /25 compatible MCS enables straightforward cloning operations according to iGEM standards.
The Strep-tag II serves as a powerful tool purifying and detecting expressed proteins
The CYC1 transcription terminator adjusted for S. cerevisiae INVSc transcription efficiently.
The pUC origin of replication permits high copy number replication in E.coli and therefore simplifies cloning.
The ß-lactamase coding gene allows the use of ampicillin as selection marker for cloning applications in E.coli
The URA3 gene permit the use of uracil prototrophy of the stem S. cerevisiae INVSc as a selection marker for transfection.
The 2µ origin of replication offers the opportunity of high copy number replication in S. cerevisiae INVSc
Based on the yeast shuttle vector without promoters, called pTUM100, we created several vector versions containing constitutive promoters. This is represented in figures C to E:
pTUM100 simply contains the new MCS, the transcription terminator and further elements required for cloning and transfection.
In addition pTUM101, pTUM102 and pTUM103 contain the constitutive promoters pTef1, pTef2 and pADH.
pTUM104 keeps the same constitutive promoters and the galactose inducible promoter pGAL1.
All of these versions were submitted as BioBricks.
BioBrick backbones
BBa_K801000 pTUM100 yeast shuttle vector without promoter
BBa_K801001 pTUM101 yeast shuttle vector with pTeF1 promoter
BBa_K801002 pTUM102 yeast shuttle vector with pTeF2 promoter
BBa_K801003 pTUM103 yeast shuttle vector with pADH1 promoter
BBa_K801004 pTUM104 yeast shuttle vector with pGal1 promoter
Induction assays with galactose
Fig. 3(A) and (B) show the normalized fluorescence signal in dependence of time after induction.
The pGaL1 promoter system is repressed in presence to glucose (West et al., 1984). Therefore maintaining cells in glucose containing medium showed the lowest basal transcription activity. Changing carbon source to galactose derepresses promoter activity with followed increasing transcription activity (Giniger et al., 1985). Alternatively cells can be maintained in raffinose containing medium neither repressing nor inducing transcriptional activity. However the induction of transcriptional activity by galactose is accelerated by maintaining cells in raffinose. For characterizing the pGal1 promoter we cloned eGFP in pTUM104 vector and transfected S. cerevisiae INVSc cells. A single colony of 2-3 mm diameter was inoculated in 15 ml SC-U medium containing glucose as overnight culture. OD at 600 nm was measured and an appropriated volume of cells was sedimented and resuspended in galactose containing SC-U medium creating an initial OD600 of 0.4. In further processes samples of 1 ml volume were taken hourly and the OD600 as well as fluorescence at 488 nm were determined. The fluorescence signal was normalized per division by the adapted OD600 signal.
The figures show the necessity of cells needing 6-8 h to reach a stable level of promoter activity. However the obtained data may vary in dependence of the protein folding time.
Outlook and conclusions
The data shown above as well as the numerous successful expressions of desired enzymes based on the pGaL1 promoter proved its functionality. Nevertheless we will characterize this as well as all of the constitutive promoters via luciferase assays to consider different protein folding times. Furthermore we will design an additional vector carrying a His-tag instead of a Strep-tag II in order to extend the application range.
References
[ West et al., 1984 ] West, R. W. J., Yocum, R. R., and Ptashne, M. (1984). Saccharomyces cerevisiae GAL1-GAL10 Divergent Promoter Region: Location and Function of the Upstream Activator Sequence UASG. Mol. Cell. Biol.4, 2467-2478.
[ Giniger et al., 1985 ] Giniger, E., Barnum, S. M., and Ptashne, M. (1985). Specific DNA Binding of GAL4, a Positive Regulatory Protein of Yeast. Cell 40, 767-774.
Retrieved from " http://2012.igem.org/Team:TU_Munich/Project/Vector_Design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
