<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Munich_Project_Overview"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Munich/Project/Overview">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Munich/Project/Overview&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Project/Overview&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Project/Overview&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Munich/Project/Overview" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Munich/Project/Overview</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navigation"><UL><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Project/Overview">Project</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Overview">Results</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Vector_Design">Vector Design</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Limonene">Limonene</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Thaumatin">Thaumatin</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Caffeine">Caffeine</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Xanthohumol">Xanthohumol</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Constitutive_Promoter">Constitutive Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Light_Switchable_Promoter">Light-Switchable Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Ethanol_Inducible_Promoter">Ethanol-Inducible Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Genome_Integration">Genome Integration</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Brewing">Brewing</A></LI></UL><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Results/Biobricks">BioBricks</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Results/Biobricks">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Results/RFC">RFC</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Methods">Modeling</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Methods">Methods</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Priors">Priors</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Gal1_Promoter">Gal1 Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Yeast_Growth">Yeast Growth</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Overview">Human Practice</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/School_Visit">School Visit</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Public">Public Relations</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Comic_children">Comic</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Information_Day">Action Day</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Politics">Politics</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Movie">Movie</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Radio">Radio Interview</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Survey">Survey</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Labjournal">Notebook</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Protocols">Methods</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Meetings">Meetings</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Attribution">Attributions</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Team/Members">Team</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Activities">Activities</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Collaborations">Collaborations</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Vision"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Vision</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Biosynthesis_pathways"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Biosynthesis pathways</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Limonene"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Limonene</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Thaumatin"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Thaumatin</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Caffeine"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Caffeine</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Xanthohumol"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Xanthohumol</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-8"><A href="#Vector_Design"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Vector Design</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#pTUM100"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">pTUM100</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Regulation_of_Genexpression"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Regulation of Genexpression</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Ethanol-inducible_promoter"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Ethanol-inducible promoter</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Light-switchable_promoter"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Light-switchable promoter</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Genome_Integration"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Genome Integration</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Brewing_our_SynBio_Beer"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Brewing our SynBio Beer</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#References"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Overview"> Overview </SPAN></H1><H2><SPAN class="mw-headline" id="Vision"> Vision </SPAN></H2><P>We, TU Munich’s 2012 iGEM team, strive to catalyze the diffusion process of knowledge about genetic engineering and synthetic biology among the general public. Using the example of iGEM’s first and finest SynBio Beer we involve, interest and inspire people to reconsider preconceived ideas and encourage them to openly engage in a broad discussion weighing pros and cons of genetic engineering in foodstuff. We sketch a future where new technology can be applied in a meaningful way to complement traditional foods or beverages.
</P><H2><SPAN class="mw-headline" id="Biosynthesis_pathways">Biosynthesis pathways</SPAN></H2><DIV class="bezel mfull"><H3><SPAN class="mw-headline" id="Limonene">Limonene</SPAN></H3><P>Limonene is a cyclic terpene and a major constituent of several citrus oils. D-limonene has been used as a component of flavorings and fragrances. It is formed from geranyl pyrophosphate by limonene synthase.
</P><P>We successfully demonstrated the production of the flavoring substance limonene by expressing limonene synthase in <I>S. cerevisiae</I>, which naturally synthesizes the educt geranyl pyrophosphate.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Main results from our limonene subproject: reaction mechanism (A), constructed BioBricks (B) and proof of principle for the in vivo production of limonene</DIV></DIV></DIV><CENTER><B>Experimental results:</B></CENTER><P>(+)-limonene synthase 1 (&lt;partinfo&gt;BBa_K801065&lt;/partinfo&gt;) and (+)-limonene synthase 1 with yeast consensus sequence (&lt;partinfo&gt;BBa_K801060&lt;/partinfo&gt;) were successfully cloned into our new yeast expression vector pTUM104. Expression of recombinant limonene synthase in <I>Saccharomyces cerevisiae</I> was proven by western blotting. Subsequently the protein was purified using SA-chromatography and size exclusion chromatography. The functionality of the enzyme was verified by <I>in vivo</I> and <I>in vitro</I> detection of limonene via GC-MS. 
</P><P>Furthermore, we established gene constructs of the limonene synthase coding sequence with different yeast specific promoters and terminators (&lt;partinfo&gt;BBa_K801062&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K801063&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K801064&lt;/partinfo&gt;). 
</P><P>Last but not least, we have brewed iGEM's first SynBio beer containing limonene.
</P><CENTER><B>Conclusion and outlook:</B></CENTER><P>We have achieved functional expression of <I>Citrus limon</I> limonene synthase and production of limonene in yeast. Furthermore, we have been able to proof the production of limonene in the beers we brewed. Further experiments regarding the concentrations of limonene produced will be carried out in the future.
</P></DIV><DIV class="bezel mfull"><H3><SPAN class="mw-headline" id="Thaumatin">Thaumatin</SPAN></H3><P>Thaumatin is a natural protein which is synthesized by the katamfe plant (<I>Thaumatococcus daniellii</I>). It is said to be 2,000 to 100,000 times sweeter than sucrose on molar basis, but the sweetness builds up slow and lasts long. It has been approved as a sweetener by the European Union (E957).
</P><P>Our aim is to have <I>S. cerevisiae</I> secrete functional thaumatin by expressing preprothaumatin – a principle which has been proven by [<A href="http://www.ncbi.nlm.nih.gov/pubmed/6327079" class="external text" rel="nofollow">Edens et al., 1984</A>].
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Main results from the Thaumatin subproject: Structure of Thaumatin (A), constructed BioBricks (B), profile of an ion exchange chromatography (IEC) used to detect our recombinant Thaumatin and a SDS-PAGE gel showing IEC elution fractions containing Thaumatin</DIV></DIV></DIV><CENTER><B>Experimental results:</B></CENTER><P>The BioBrick for preprothaumatin <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K801080" class="external text" rel="nofollow">BBa_K801080</A> as well as an expression cassette <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K801080" class="external text" rel="nofollow">K801080</A> were successfully cloned, expressed in yeast, purified using an ion exchange chromatography (see figure C) and detected in the SDS-PAGE. Therefore, the expression of thaumatin in yeast could be demonstrated and functionality of the BioBrick is confirmed.
</P><CENTER><B>Conclusion and outlook:</B></CENTER><P>A proof of principle for the expression of thaumatin was achieved. Further goals are the increase of the expression of  thaumatin and the investigation of the secretion.
</P></DIV><DIV class="bezel mfull"><H3><SPAN class="mw-headline" id="Caffeine">Caffeine</SPAN></H3><P>Caffeine is a purine-alkaloid and its biosynthesis is known from coffee and tea plants. The molecule acts as a competitive antagonist of adenosine receptors and, therefore, increases indirectly neurotransmitter concentrations resulting in warding off drowsiness and restoring alertness. 
</P><P>The idea is to perform a heterologous gene expression of the three enzymes 7-methylxanthosine synthase (CaXMT1), N-methyl nucleosidase (CaMXMT1) and caffeine synthase (CaDXMT1) required for caffeine biosynthesis in <I>Saccharomyces cerevisiae</I>. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Figure showing a schematic overview of the reaction in (A), a western blot against the Strep-tag II for GFP (lane 1), [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K801070" class="external text" rel="nofollow">BBa_K801070</A>] (lane 2) and [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K801071" class="external text" rel="nofollow">BBa_K801071</A>] (lane 3) in (B) and finally the same western blot development for [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K801072" class="external text" rel="nofollow">BBa_K801072</A>] (C) LC/MS spectra of <I>in vitro</I> caffeine synthesis enzyme assay, showing produced theobromine</DIV></DIV></DIV><CENTER><B>Experimental results:</B></CENTER><UL><LI> Successful cloning of the three enzymes <A href="http://partsregistry.org/Part:BBa_K801070" class="external text" rel="nofollow">7-methylxanthosine synthase (CaXMT1)</A>, <A href="http://partsregistry.org/Part:BBa_K801071" class="external text" rel="nofollow">theobromine synthase (CaMXMT1)</A> and  the <A href="http://partsregistry.org/Part:BBa_K801072" class="external text" rel="nofollow">caffeine synthase (CaDXMT1)</A> into the shuttle vector pTUM104 and pSB1C3 each. 
</LI><LI> Successful assembly of the BioBricks to form expression cassettes consisting of promoter, gene and terminator: <A href="http://partsregistry.org/Part:BBa_K801073" class="external text" rel="nofollow">pTEF2-CaXMT1-tADH1</A>, <A href="http://partsregistry.org/Part:BBa_K801074" class="external text" rel="nofollow">pTEF1-CaMXMT1-tADH1</A> and <A href="http://partsregistry.org/Part:BBa_K801075" class="external text" rel="nofollow">pTEF2-CaDXMT1-tADH1</A>) into pSB1C3.
</LI><LI> Successful assembly of the expression cassettes of the three relevant enzymes forming a composite part of 6.4 kb capable of caffeine production in yeast (<A href="http://partsregistry.org/Part:BBa_K801077" class="external text" rel="nofollow">Caffeine Synthesis Pathway</A>) into pSB1C3.
</LI><LI> Successful expression of CaXMT1, CaMXMT1 and CaDXMT1 in <I>Saccharomyces cerevisiae</I> INVSc1 in selective Sc minimal induction medium lacking uracil with 2 % galactose.
</LI></UL><CENTER><B>Conclusion and outlook:</B></CENTER><P>The homologue expression of the three required enzymes for caffeine synthesis in <I>Saccharomyces cerevisiae</I> INVSc1 transformed with pTUM102_CaXMT1, pTUM102_CaMXMT1 and pTUM102_CaDXMT1 was successful. Further testing of caffeine production using crude extracts from lysed yeast cells which had previously been transformed with our caffeine synthesis expression cartridge has been done and we were successful in producing  theobromine, the immediate precursor of caffein, which we detected by the use of LC/MS with multiple reaction monitoring (MRM).  
</P></DIV><DIV class="bezel mfull"><H3><SPAN class="mw-headline" id="Xanthohumol">Xanthohumol</SPAN></H3><P>Xanthohumol is known as a putative cancer chemopreventive agent due to its antioxidant activities [<A href="http://www.ncbi.nlm.nih.gov/pubmed/10995285" class="external text" rel="nofollow">Miranda et al., 2000</A>]. Our goal is a heterologous gene expression of all enzymes required for xanthohumol biosynthesis in <I>S. cerevisiae</I>.
</P><P>The pathway for the production of this plant secondary metabolite is composed of five steps, starting with the conversion of tyrosine and followed by four further enzymatic reactions.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Main results from the Xanthohumol subproject: Successful reconstruction of the metabolic pathway for Xanthohumol</DIV></DIV></DIV><CENTER><B>Experimental results:</B></CENTER><P>The whole biosynthetic pathway for the production of xanthohumol was converted into BioBricks. Except for APT each of the enzymes were cloned in two versions one having the proposed consensus sequence for more efficient expression in yeast chassis and another for usage of these BioBricks in other chassis. All BioBricks were sequenced. Sequences can be found in the registry of standard biological parts:
PAL (&lt;partinfo&gt;BBa_K801090&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K801091&lt;/partinfo&gt;), 4CL (&lt;partinfo&gt;BBa_K801092&lt;/partinfo&gt;,  &lt;partinfo&gt;BBa_K801093&lt;/partinfo&gt;), CHS (&lt;partinfo&gt;BBa_K801094&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K801095&lt;/partinfo&gt;), APT (&lt;partinfo&gt;BBa_K801096&lt;/partinfo&gt; and OMT (&lt;partinfo&gt;BBa_K801097&lt;/partinfo&gt;,  &lt;partinfo&gt;BBa_K801098&lt;/partinfo&gt;). 
</P><CENTER><B>Conclusion and outlook:</B></CENTER><P>The construction of the xanthohumol pathway was achieved, whereas the expression and characterization might be an interesting task for iGEM teams in the future.
</P></DIV><H2><SPAN class="mw-headline" id="Vector_Design">Vector Design</SPAN></H2><DIV class="bezel mfull"><H3><SPAN class="mw-headline" id="pTUM100">pTUM100</SPAN></H3><P>Designing an expression vector for yeast which is compatible to the iGEM cloning principles and standards was the main aim of this subproject. Based on the commercially available pYES2 vector we created vectors containing inducible and constitutive promoters in order to establish efficient possibilities to clone and express our enzymes.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Explanations on the figure:  Figure A shows the new multiple cloning site (MCS) containing the <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A>/25 restriction sides and the DNA sequence coding for the <I>Strep</I>-tag II.  Figure B gives an overview of all important functional elements located on the vector backbone. Upstream to the new MCS lies a T7 promoter primer binding site allowing easy forward sequencing of integrated gene constructs using the standard T7 primer. The URA 3 gene is a prototrophy marker used for the selection of transfected cells.  Figure C to E present the successfully designed BioBricks: pTUM100 simply contains the new MCS, the transcription terminator and further elements required for cloning and transfection. pTUM102 to pTUM104 contain in addition the constitutive promoters pTef1, pTef2 and ADH. On pTUM104 the galactose inducible promoter pGAL1 is located.</DIV></DIV></DIV><CENTER><B>Experimental results:</B></CENTER><P>Using the pYES vector from Invitrogen we first deleted five forbidden restriction sites in the vector backbone via side directed mutagenesis. Furthermore, the original multiple cloning site was replaced by a multiple cloning site compatible with the <A href="http://tools.ietf.org/html/rfc10" class="external mw-magiclink-rfc">RFC 10</A>/25 cloning standards. To allow easy extraction and purification of proteins for <I>in vitro</I> applications the new multiple cloning site allows to express proteins with a <I>Strep</I>-tag II. 
Exclusion of the galactose inducible promoter provided a powerful basis vector for the integration of user-defined promoters. This way the pTUM100 vector gives a valuable contribution to our and to further protein expression and promoter characterization experiments in <I>Saccharomyces cerevisiae</I>.
Moreover, we used the pTUM100 to integrate the three constitutive promoters Tef1, Tef2 and ADH which come all with different promoter intensities.
</P><CENTER><B>Outlook and conclusion:</B></CENTER><P>The galactose inducible expression system was a great aid for the majority of all subprojects. Especially the opportunity to purify and detect (via Western blot) proteins using the <I>Strep</I>-tag II did facilitate our laboratory practice and accelerated our work progress.
To cover even more demands we are planning to design a second vector template containing a His-tag.
</P><P>All BioBricks were sequenced. Sequences can be found in the registry of standard biological parts under the following entries:
</P><P>pTUM100 (&lt;partinfo&gt;BBa_K801000&lt;/partinfo&gt;), pTUM101 (&lt;partinfo&gt;BBa_K801001&lt;/partinfo&gt;), pTUM102 (&lt;partinfo&gt;BBa_K801002&lt;/partinfo&gt;), pTUM103 (&lt;partinfo&gt;BBa_K801003&lt;/partinfo&gt;) and pTUM104 (&lt;partinfo&gt;BBa_K801004&lt;/partinfo&gt;). 
</P></DIV><H2><SPAN class="mw-headline" id="Regulation_of_Genexpression">Regulation of Genexpression</SPAN></H2><P>By developing inducible promoters and placing them upstream of our biosynthetic pathways we create the possibility to make <I>S. cerevisiae</I> dynamically respond to concentration changes in its medium as well as to external stimuli. 
</P><P>An optimal inducing substance needs to be inexpensive, nontoxic and fully controllable in its application. Only substances with these characteristics allow to precisely regulate a system temporally, spatially and quantitatively.  
</P><DIV class="bezel mfull"><H3><SPAN class="mw-headline" id="Ethanol-inducible_promoter">Ethanol-inducible promoter</SPAN></H3>
The KlADH4-promoter from the yeast <I>Kluyveromyces lactis</I> regulates the expression of a mitochondrial alcohol dehydrogenase in an ethanol-dependent way. An alcohol-inducible promoter would be incredibly useful for anyone planning to brew a beer with a transgenic yeast - it would allow for the induction of the target genes after the main fermentation has finished and this way, the metabolic burden for the yeast cells could be lowered. All the transcription factors known to be involved in the regulation of the KlADH4-promoter in <I>K. lactis</I> also occur in <I>S. cerevisiae</I> [<A href="http://www.ncbi.nlm.nih.gov/pubmed/10724480" class="external text" rel="nofollow">Mazzoni et al., 2000</A>]. This is why we are confident that this promoter maintains its unique characteristics when transformed into <I>S. cerevisiae</I>.<DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>A</B>: The KlADH4-promoter was cloned from genomic DNA of <I>Kluyveromyces lactis</I>. The new BioBrick BBa_K801020 was inserted into our pTUM100 vector. eGFP served as a reporter gene for characterization in <I>S. cerevisiae</I> (plasmid name: pTUM100_KLADH4_eGFP). <B>B</B> Emission spectra of eGFP obtained during cultivation of <I>S. cerevisiae</I> transformed with pTUM100_KLADH4_eGFP using different carbon sources. Blue: Galactose was used as carbon source. The measured ethanol concentration was 1.7 % (v/v). The peak at 509 nm indicates that eGFP is expressed. Red: Glycerol was used as carbon source. The measured ethanol concentration was 0.2 % (v/v). No eGFP fluorescence could be detected.</DIV></DIV></DIV><CENTER><B>Experimental results:</B></CENTER><P>At this time our results concerning the KlADH4-promoter (originally from the yeast <I>Kluyveromyces lactis</I>) suggest that this promoter is ethanol inducible in <I>S. cerevisiae</I>. Further experiments are still being done to abolish residual ambiguities. The lowest ethanol concentration at which eGFP-expression was detected is 0.9 Vol.-%.
</P><CENTER><B>Conclusion and outlook:</B></CENTER><P>Because <I>S. cerevisiae</I> is such a good brewer, it was difficult to produce a stringent negative control in our characterization experiments. However, we finally figured out some experiments that allowed us to keep the ethanol concentration below 0.5 % v/v, which is a concentration at which induction is observed in <I>K. lactis</I>. We are working hard on providing additional data, but we are confident that we will be able to provide clear evidence that this promoter is ethanol-inducible not only in <I>K. lactis</I>, but also in <I>S. cerevisiae</I>.
</P></DIV><DIV class="bezel mfull"><H3><SPAN class="mw-headline" id="Light-switchable_promoter">Light-switchable promoter</SPAN></H3><P>The idea behind a light-switchable system (LSPS) is to create a gene expression system which can be induced and deactivated by light of a certain wavelength.
</P><P>This system is extremely attractive, as induction does not require the addition of a specific substance. This makes induction <B>cheap, fast, precise</B> and also compatible with the Bavarian purity law.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Principle of light-dependent switching of gene-expression.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Data for the GAL4 based light-switchable promoter system.</DIV></DIV></DIV><CENTER><B>Experimental results:</B></CENTER><P>All fusion proteins for the two types of a light-switchable promoter system has been finished (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K801039" class="external text" rel="nofollow">BBa_K801039</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K801040" class="external text" rel="nofollow">BBa_K801040</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K801041" class="external text" rel="nofollow">BBa_K801041</A>), also pTEF1 driven gene expression batteries coding for all components of each type of our light-switchable promoter system has been done (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K801042" class="external text" rel="nofollow">BBa_K801042</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K801043" class="external text" rel="nofollow">BBa_K801043</A>). GAL4 based light-switchable promoter system is working fine! LexA based light-switchable promoter system even works more efficient but reporter construct (<A href="http://partsregistry.org/Part:BBa_K165031" class="external text" rel="nofollow">BBa_K165031</A>, iGEM08_BrownTwo) is extremly leaky due to suboptimal LexA binding elements, which contains the TATATAA sequence of the typical yeast TATA-Box. The solution to prevent high basal gene expression is to create a minimal promoter preceded by DNA motifs which are recognized by LexA but do not contain the TATA sequence (e. g.: <A href="http://partsregistry.org/Part:BBa_K079039" class="external text" rel="nofollow">BBa_K079039</A> or <A href="http://partsregistry.org/Part:BBa_K079040" class="external text" rel="nofollow">BBa_K079040</A>). A general improvement to lower the basal activity and to get an higher S/N-ratio in both systems is to use low-copy yeast plasmids instead of high-copy plasmids.
</P><CENTER><B>Conclusion and outlook:</B></CENTER><P>GAL4 based light-switchable promoter is working! LexA based light-swichable promoter is also working but reporter system has to be improved. So the logical next step would be to design of a minimal promoter preceded by DNA motifs which is recognized by LexA but not by TATA binding proteins of the yeast transcription machinery (e. g.: <A href="http://partsregistry.org/Part:BBa_K079039" class="external text" rel="nofollow">BBa_K079039</A> or <A href="http://partsregistry.org/Part:BBa_K079040" class="external text" rel="nofollow">BBa_K079040</A>).
</P><P>A general improvement to lower the basal activity and to get an higher S/N-ratio in both systems is to use low-copy yeast plasmids instead of high-copy plasmids.
</P></DIV><H2><SPAN class="mw-headline" id="Genome_Integration">Genome Integration</SPAN></H2><DIV class="bezel mfull"><P>Working with food, it is unacceptable to use antibiotics to keep up the selective pressure during the brewing process. Since we cannot work with auxotrophies in beer either, we have to make sure the yeast cells do not lose the plasmids harboring our BioBricks. The most promising way to accomplish a long lasting presence of our constructs is to achieve <B>genome integration</B>.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:432px;"><DIV class="thumbcaption">Plasmid backbone used for integration of our expression cassettes</DIV></DIV></DIV><CENTER><B>Experimental results:</B></CENTER><P>First experiments to characterize the function of the yeast integration system were performed and the used selection marker was maintained in the yeast culture, although the selection pressure was switched off. This indicates that first integrations were achieved. 
</P><CENTER><B>Conclusion and outlook:</B></CENTER><P>Maintaining the plasmids harboring our expression cassettes in the yeast cells during the brewing process is best possible using genome integration. This becomes increasingly interesting, when a yeast strain with different expression cassettes is to be created. Because this is intended for the next step of our project the integration of our expression cassettes becomes increasingly important.
</P></DIV><H2><SPAN class="mw-headline" id="Brewing_our_SynBio_Beer">Brewing our SynBio Beer</SPAN></H2><DIV class="bezel mfull"><P>Contrary to popular opinion the chief ingredient of beer is not <B>YPD</B> but <B>gyle</B>, a carefully prepared mixture of malt, hop and water. Although the name of the yeast strain commonly used in the lab, <B>S. cerevisiae</B>, suggests that it is used in the beer brewing process. The yeast strains generally employed in brewing have <B>strongly adapted to gyle</B>, as they are reutilized in every succeeding brewing cycle.
Hence some investigation on how our yeast <B>performs</B> in gyle and experiments on the toxicity of the substances produced by our biosynthetic pathways were necessary.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Picture of the first SynBio Beer brewed during the iGEM competition in 2012</DIV></DIV></DIV><CENTER><B>Experimental results:</B></CENTER><P>Our experiments on the different yeast strains show that the growth of several different yeast strains is <B>not impaired in gyle</B>! 
The toxicity test with the substances caffeine and limonene showed a toxicity for yeast cells at higher concentrations in the cultur media. </P><P>Expression assays proved the necessity of <A href="/Team:TU_Munich/Project/Genome_Integration" title="Team:TU Munich/Project/Genome Integration"><B>genome integration</B></A> for a proper <B>SynBio Beer</B>.
</P><CENTER><B>Conclusion and outlook:</B></CENTER><P>At the day of the final Wiki-Freeze we could finish the brewing with three different ingredients (limonene, thaumatin and caffeine) whose biosynthesis we have engineered during this summer. This was done using yeast culture expressing BioBricks from a plasmid as well as using cultures in which the expression cassette was integrated into the genome.
This gives us the great honor to present <B>iGEM's first and finest SynBio Beer: TUM-Brew</B>.
</P></DIV><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><UL><LI> [<A href="http://www.ncbi.nlm.nih.gov/pubmed/6327079" class="external text" rel="nofollow">Edens et al., 1984</A>] Edens, L., Bom, I., Ledeboer, A. M., Maat, J., Toonen, M. Y., Visser, C., and Verrips, C. T. (1984). Synthesis and processing of the plant protein thaumatin in yeast. <I>Cell</I>, 37(2):629–33.
</LI><LI>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/10995285" class="external text" rel="nofollow">Miranda et al., 2000</A>] Miranda, C. L., Stevens, J. F., Ivanov, V., McCall, M., Frei, B., Deinzer, M. L., and Buhler, D. R. (2000). Antioxidant and prooxidant actions of prenylated and nonprenylated chalcones and flavanones in vitro. <I>J Agric Food Chem</I>, 48(9):3876–84.
</LI><LI>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/10724480" class="external text" rel="nofollow">Mazzoni et al., 2000</A>] Mazzoni, C., Santori, F., Saliola, M., and Falcone, C. (2000). Molecular analysis of uas(e), a cis element containing stress response elements responsible for ethanol induction of the kladh4 gene of <I>kluyveromyces lactis</I>. <I>Res Microbiol</I>, 151(1):19–28.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:TU_Munich/Project/Overview">http://2012.igem.org/Team:TU_Munich/Project/Overview</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Munich/Project/Overview" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Munich/Project/Overview" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Munich/Project/Overview&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Munich/Project/Overview&amp;oldid=298740" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>