<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Munich_Notebook_Labjournal"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Munich/Notebook/Labjournal">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Munich/Notebook/Labjournal&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Notebook/Labjournal&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Notebook/Labjournal&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Munich/Notebook/Labjournal" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Munich/Notebook/Labjournal</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navigation"><UL><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Project/Overview">Project</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Overview">Results</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Vector_Design">Vector Design</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Limonene">Limonene</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Thaumatin">Thaumatin</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Caffeine">Caffeine</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Xanthohumol">Xanthohumol</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Constitutive_Promoter">Constitutive Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Light_Switchable_Promoter">Light-Switchable Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Ethanol_Inducible_Promoter">Ethanol-Inducible Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Genome_Integration">Genome Integration</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Brewing">Brewing</A></LI></UL><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Results/Biobricks">BioBricks</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Results/Biobricks">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Results/RFC">RFC</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Methods">Modeling</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Methods">Methods</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Priors">Priors</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Gal1_Promoter">Gal1 Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Yeast_Growth">Yeast Growth</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Overview">Human Practice</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/School_Visit">School Visit</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Public">Public Relations</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Comic_children">Comic</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Information_Day">Action Day</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Politics">Politics</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Movie">Movie</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Radio">Radio Interview</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Survey">Survey</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Labjournal">Notebook</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Protocols">Methods</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Meetings">Meetings</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Attribution">Attributions</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Team/Members">Team</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Activities">Activities</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Collaborations">Collaborations</A></LI></DIV><FORM id="labselect"><FIELDSET class="ui-widget ui-widget-content ui-corner-left"><B>Display:</B><B style="color: rgb(166, 126, 166)">Vector Design</B><B style="color: rgb(116, 183, 112)">Limonene</B><B style="color: rgb(255, 122, 97)">Xanthohumol</B><B style="color: rgb(202, 85, 85)">Thaumatin</B><B style="color:rgb(192, 167, 4)">Caffeine</B><B style="color: rgb(115, 208, 255)">Constitutive promoter</B><B style="color: rgb(0, 32, 96)">Light-switchable promoter</B><B style="color: rgb(0, 102, 0)">Ethanol-inducible promoter</B><B style="color: rgb(222, 77, 185)">Genome integration</B><A href="#" id="ExAll">Expand All ...</A><A href="#" id="ColAll">Collapse All ...</A><I>You can also click on individual experiments to show/hide them</I><B>Jump to:</B><A href="#Week_1">Week 1</A> 13.6-17.6<A href="#Week_2">Week 2</A> 18.6-24.6<A href="#Week_3">Week 3</A> 25.6-1.7<A href="#Week_4">Week 4</A> 2.7-8.7<A href="#Week_5">Week 5</A> 9.7-15.7<A href="#Week_6">Week 6</A> 16.7-22.7<A href="#Week_7">Week 7</A> 23.7-29.7<A href="#Week_8">Week 8</A> 30.7-5.8<A href="#Week_9">Week 9</A> 6.8-12.8<A href="#Week_10">Week 10</A> 13.8-19.8<A href="#Week_11">Week 11</A> 20.8-26.8<A href="#Week_12">Week 12</A> 27.8-2.9<A href="#Week_13">Week 13</A> 3.9-9.9<A href="#Week_14">Week 14</A> 10.9-16.9<A href="#Week_15">Week 15</A> 17.9-23.9<A href="#Week_16">Week 16</A> 24.9-27.9
        </FIELDSET></FORM><DIV id="ladder" class="ui-widget ui-widget-content ui-corner-right"><B><U>1 kbp GeneRuler:</U></B><B><U>100 bp GeneRuler:</U></B><B><U>PageRuler Plus:</U></B></DIV><H1><SPAN class="mw-headline" id="Labjournal"> Labjournal </SPAN></H1><P>P1-923 and PCR1-73 are the tube numbers for plasmids/PCR products from our <A href="/Team:TU_Munich/Notebook/Inventory" title="Team:TU Munich/Notebook/Inventory">inventory list (most of the descriptions are in german)</A></P><P>For a shorter summary of what happened each week, see our <A href="/Team:TU_Munich/Notebook/Meetings" title="Team:TU Munich/Notebook/Meetings">meeting protocols</A>.
</P><DIV class="labbook"><H1><SPAN class="mw-headline" id="Week_1">Week 1</SPAN></H1><DIV class="week" id="WWeek_1"><H2><SPAN class="mw-headline" id="Wednesday.2C_June_13th"><B>Wednesday, June 13th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Exchange_of_the_Multiple_Cloning_Site_of_pTUM104">Exchange of the Multiple Cloning Site of pTUM104 </SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B> Exchange of the Multiple Cloning Site of pTUM104
</P><P><B>Hybridisation of the primers O1 with O2 and O3 with O4</B></P><P>Primer preparation:
</P><UL><LI>centrifugation
</LI><LI>dilution in the denoted quantity in bidest. water (concentration = 100 pmol/µl)
</LI><LI>centrifugation
</LI></UL><P>Hybridisation:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>34 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>5 µl
</TD><TD>PNK-buffer
</TD></TR><TR><TD>2.5 µl
</TD><TD>Primer O1
</TD></TR><TR><TD>2.5 µl
</TD><TD>Primer O2
</TD></TR><TR><TD>1 µl
</TD><TD>PNK (10mM)
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>34 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>5 µl
</TD><TD>PNK-buffer
</TD></TR><TR><TD>2.5 µl
</TD><TD>Primer O3
</TD></TR><TR><TD>2.5 µl
</TD><TD>Primer O4
</TD></TR><TR><TD>1 µl
</TD><TD>PNK (10mM)
</TD></TR></TBODY></TABLE><UL><LI>30 min 37 °C
</LI><LI>10 min 80 °C
</LI><LI>put the Thermo Block with the mixture in a styrofoam box and let it cool down over night
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_June_14th"><B>Thursday, June 14th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Exchange_of_the_Multiple_Cloning_Site_of_pTUM104_2">Exchange of the Multiple Cloning Site of pTUM104 </SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B> Exchange of the Multiple Cloning Site of pTUM104
</P><P><B>Digestion of pTUM104 with HindIII and XbaI</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>Miniprep (pYES2 SH 1.7.3 with a concentration of 87.8 ng/µl)
</TD></TR><TR><TD>5 µl
</TD><TD>NEB2
</TD></TR><TR><TD>5 µl
</TD><TD>10x BSA
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI (10 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>HinIII (10U/µl)
</TD></TR><TR><TD>24 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C, 1.75 h
</P><P><B>DNA preparative gel electrophoresis</B></P><UL><LI>gel: 1% with LMP-agarose
</LI><LI>band 1: 10 µl DNA ladder (1kb)
</LI><LI>band 2: 50 µl probe + 5 µl loading dye
</LI><LI>70 V, 90 min
</LI></UL><P><B>Gelextration</B></P><UL><LI>cut the bands (5.7-5.8 kb) and split it in two eppis 
</LI><LI> m1=165.3 mg
</LI><LI> m2=211.1 mg
</LI></UL><LI>QIAquick Gel Extractrion Kit
</LI><LI> eppi1: 495.9 µl QG-buffer
</LI><LI> eppi2: 633.3 µl QG-buffer
</LI><LI> step 6 was left out
</LI><LI> step 9: 30µl buffer, 4 min incubation
</LI><UL><LI>the product was named P5
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_plasmids_from_Prof._Schwab_in_E.coli_XL-1_Blue">Transformation of plasmids from Prof. Schwab in <I>E.coli</I> XL-1 Blue</SPAN></H3><P><B>Investigator:</B> Lara, Andrea
</P><P><B>Aim of the experiment:</B> Preparation of the plasmids for transformation
</P><P><B>Overnight cultures of cells with limonenesynthase-plasmid from Prof. Schwab</B></P><UL><LI>resuspend 50 µl / 200 µl of competent cells with 50 ml LB medium
</LI><LI>add 0,1 ml Ampicillin (100 µg/ml) and 0,28 ml Chloramphenicol (170 µ/ml) for strain 108
</LI><LI>add 0,07 ml Kanamycin (50 µg/ml) and 0,28 ml Chloramphenicol (170 µ/ml) for strain 106
</LI><LI>incubate at 37 °C
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_15th"><B>Friday, June 15th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Exchange_of_the_Multiple_Cloning_Site_of_pTUM104_3">Exchange of the Multiple Cloning Site of pTUM104</SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B> Exchange of the Multiple Cloning Site of pTUM104
</P><P><B>Analytical DNA gel electrophoresis</B></P><UL><LI>gel: 1.2 %
</LI><LI>band 1: 10 µl DNA ladder (1kb)
</LI><LI>band 2: 3 µl pYES2 digested + 7 µl TAE-buffer + 1 µl loading dye
</LI><LI>band 3: 3 µl O5 + 7 µl TAE-buffer + 1 µl loading dye
</LI><LI>band 4: 3 µl O6 + 7 µl TAE-buffer + 1 µl loading dye
</LI><LI>band 5: 10 µl DNA ladder (100 bp)
</LI></UL><P><B>Ligation of Plasmid P5 (pYES2 digested) with the hybridized primers O5 and O6</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>pYES2 digested (P5)
</TD></TR><TR><TD>1 µl
</TD><TD>O5
</TD></TR><TR><TD>1 µl
</TD><TD>O6
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>0,5 µl
</TD><TD>T4 DNA-ligase
</TD></TR><TR><TD>11.5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Negative control
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>pYES2 digested (P5)
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>0,5 µl
</TD><TD>T4 DNA-ligase
</TD></TR><TR><TD>13.5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>water bath 16 °C 
</LI><LI>after 3 h a probe for the transformation was taken
</LI><LI>the rest was ligated over the weekend
</LI></UL><P><B>Transformation of E. coli with ligated products (P6)</B></P><UL><LI>competent cells: SHXL1 Blue (by Simon)
</LI><LI>Transformation with ligation product (P6) and negative control
</LI></UL><P>results:
</P><UL><LI>P6 (100 µl): 1 clone
</LI><LI>P6 (concentrated): 30 clones
</LI><LI>negative control (100 µl): 0 clones
</LI><LI>negative control (concentrated): 6 clones
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_plasmids_from_Prof._Schwab_into_E.coli_XL-1_Blue">Transformation of plasmids from Prof. Schwab into <I>E.coli</I> XL-1 Blue</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B> Preparation of the plasmids for transformation
</P><P><B>Determination of the concentration with Nano Drop</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>concentration [ng/µl]</B></TD></TR><TR><TD>P3
</TD><TD>1353
</TD></TR><TR><TD>P4
</TD><TD>no result
</TD></TR></TBODY></TABLE><UL><LI>the strain 106 culture was not grown satisfying and were incubated for 2 more days
</LI></UL><P><B>Miniprep of pGex-4T-1 of strain 108 from Prof. Schwab</B></P><UL><LI>see QIAprep Spin Miniprep Kit
</LI><LI>stored as P3 (-20 °C)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_June_17th"><B>Sunday, June 17th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Exchange_of_the_Multiple_Cloning_Site_of_pTUM104_4">Exchange of the Multiple Cloning Site of pTUM104</SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B> Exchange of the Multiple Cloning Site of pTUM104
</P><P><B>Picking clones for Miniprep</B></P><UL><LI>10 clones of transformed E.coli with P6 were picked
</LI><LI>medium: 5ml LB with Amp
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_2">Week 2</SPAN></H1><DIV class="week" id="WWeek_2"><H2><SPAN class="mw-headline" id="Monday.2C_June_18th"><B>Monday, June 18th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Exchange_of_the_Multiple_Cloning_Site_of_pTUM104_5">Exchange of the Multiple Cloning Site of pTUM104</SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B> Exchange of the Multiple Cloning Site of pTUM104
</P><P><B>Miniprep of transformed E.coli with P6</B></P><UL><LI>QIAprepS Spin Miniprep Kit
</LI><LI>step 3: invert 2-3 times (don't shake to avoid destruction of genomic DNA)
</LI><LI>the 10 Minipreps were named: P7 - P16
</LI></UL><P><B>Determination of the concentration with Nano Drop</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>concentration [ng/µl]</B></TD><TD><B>260/280</B></TD></TR><TR><TD>P7
</TD><TD>157.6
</TD><TD>2.32
</TD></TR><TR><TD>P8
</TD><TD>207.3
</TD><TD>1.63
</TD></TR><TR><TD>P9
</TD><TD>171.2
</TD><TD>2.02
</TD></TR><TR><TD> P10
</TD><TD>183.1
</TD><TD>1.57
</TD></TR><TR><TD>P11
</TD><TD>160.4
</TD><TD>2.2
</TD></TR><TR><TD>P12
</TD><TD>179.9
</TD><TD>1.75
</TD></TR><TR><TD>P13
</TD><TD>179.2
</TD><TD>2.07
</TD></TR><TR><TD> P14
</TD><TD>188.3
</TD><TD>1.6
</TD></TR><TR><TD> P15
</TD><TD>166.7
</TD><TD>2.05
</TD></TR><TR><TD>P16
</TD><TD>174.6
</TD><TD>2.08
</TD></TR></TBODY></TABLE><P><B>Controll digestion with HindIII XbaI and NgoMIV</B></P><UL><LI> Samples P7-P16: 2.5 µl
</LI><LI> Negative controll pYES SH 1.7.3: 2.5 µl
</LI></UL><UL><LI>Master Mix HindIII and XbaI: 17,5 µl for a 20 µl preparation
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>HindIII
</TD></TR><TR><TD>3 µl
</TD><TD>XbaI
</TD></TR><TR><TD>24 µl
</TD><TD>NEB2
</TD></TR><TR><TD>24 µl
</TD><TD>10x BSA
</TD></TR><TR><TD>156 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>Master Mix NgoMIV: 17,5 µl for a 20 µl preparation
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>24 µl
</TD><TD>NEB4
</TD></TR><TR><TD>180 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C, 1.5 h
</P><P><B>Analytical gel electrophoresis of P7-P16</B></P><UL><LI>gel: 1.5 %
</LI></UL><P>gel 1: 
</P><UL><LI>band 1: 10 µl DNA ladder (1 kb)
</LI><LI>band 2: 3 µl pTUM104 SH 1.7.3 digested with HindIII and XbaI + 7 µl TAE buffer + 1 µl loading dye
</LI><LI>band 3 - 12: 3 µl P7-P16 digested with HindIII and XbaI + 7 µl TAE buffer + 1 µl loading dye
</LI></UL><P>gel 2:
</P><UL><LI>band 1: 10 µl DNA ladder (1 kb)
</LI><LI>band 2: 3 µl pTUM104 SH 1.7.3 digested with NgoMIV + 7 µl TAE buffer + 1 µl loading dye
</LI><LI>band 3 - 12: 3 µl P7-P16 digested with NgoMIV + 7 µl TAE buffer + 1 µl loading dye
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_19th"><B>Tuesday, June 19th</B></SPAN></H2><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_BBa_I742111_.28Limonenesynthase.29_into_E.coli_XL-1_Blue">Transformation of BBa_I742111 (Limonenesynthase) into <I>E.coli</I> XL-1 Blue</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B> Transformation
</P><UL><LI> for each Biobrick 100 µl cells were used and pooled together with 2 µl of plasmid DNA
</LI></UL><UL><LI> Incubation on ice for 30 min
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> cells were prefilled with 1 ml of LB-medium and incubated in a cell-culture shaker at 37 °C for 45 min
</LI></UL><UL><LI> 100 µl of these cell suspension were plated on antibiotic selection plates (Ampicillin)
</LI></UL><UL><LI> cell suspension was centrifuged at 13000 rpm for 60 s for resuspending the pellet with 100 µl LB and plating also
</LI></UL><UL><LI> incubation at 37 °C overnight
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_20th"><B>Wednesday, June 20th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Exchange_of_the_Multiple_Cloning_Site_of_pTUM104_6">Exchange of the Multiple Cloning Site of pTUM104</SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B> Exchange of the Multiple Cloning Site of pTUM104
</P><P><B>Sequencing of P13 and P14: pTUM104 with new MCS</B>
sequencing primer:
</P><UL><LI>1.6 µM forward primer O9
</LI><LI>DNA P13 and P14
</LI></UL><P>The Multiple Cloning Site was exchanged successfully!!!
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_BBa_I742111_.28Limonenesynthase.29_into_E.coli_XL-1_Blue_2">Transformation of BBa_I742111 (Limonenesynthase) into <I>E.coli</I> XL-1 Blue</SPAN></H3><P><B>Investigator:</B> Daniela
</P><P><B>Aim of the experiment:</B> Transformation
</P><P><B>Picking of Clones</B></P><UL><LI>6 clones were picked
</LI><LI>Incubation at 37 °C in LB + Amp
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_June_21st"><B>Thursday, June 21st</B></SPAN></H2><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_BBa_I742111_and_plasmids_from_Prof._Schwab_into_E.coli_XL-1_Blue">Transformation of BBa_I742111 and plasmids from Prof. Schwab into <I>E.coli</I> XL-1 Blue</SPAN></H3><P><B>Investigator: Lara, Andrea</B></P><P><B>Aim of the experiment:</B> Controll of Transformation
</P><P><B>Controll digestion</B></P><UL><LI> Sample P3
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>14 µl
</TD><TD>Plasmid-DNA
</TD></TR><TR><TD>0,25 µl
</TD><TD>NcoI
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer Tango (Fermentas)
</TD></TR><TR><TD>0,25 µl
</TD><TD>HindIII
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer Red (Fermentas)
</TD></TR><TR><TD>1,5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI> Sample P4
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>Plasmid-DNA
</TD></TR><TR><TD>0,25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer EcoRI (Fermentas)
</TD></TR><TR><TD>0,25 µl
</TD><TD>NotI
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer Orange (Fermentas)
</TD></TR><TR><TD>9,5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI> Sample Biobrick-clones
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>Plasmid-DNA
</TD></TR><TR><TD>0,25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer EcoRI (Fermentas)
</TD></TR><TR><TD>0,25 µl
</TD><TD>PstI
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer Orange (Fermentas)
</TD></TR><TR><TD>10,5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Analytic Gelelectrophoresis</B></P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_22nd"><B>Friday, June 22nd</B></SPAN></H2><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_XL1-Blue_with_pKS2.C2.B5Hyg-PAL-4Cl-CHS"> Transformation of <I>E.coli</I> XL1-Blue with pKS2µHyg-PAL-4Cl-CHS</SPAN></H3><P><B>Investigator:</B> Ingmar, Volker
</P><P><B>Aim of the experiment:</B> Plasmid amplification
</P><P>Operation Sequence:
</P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells
</LI><LI> addition of 1 µl of the Plasmid pKS2µHyg-PAL-4Cl-CHS
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_June_23rd"><B>Saturday, June 23rd</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_remove_NgoMIV_from_pTUM104"> Quick Change mutagenis to remove NgoMIV from pTUM104 </SPAN></H3><P><B>Investigator:</B> Ingmar, Volker
</P><P><B>Aim of the experiment:</B> Generation of an <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible version of the pTUM104 Vector.
</P><P><B>PCR</B><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P7 pYes2_RFC25 MCS 1.1 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O38 (10 pmol/µL)
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O39 (10 pmol/µL)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>15
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>The vector resulting from the PCR-product was named pYes2_RFC25 MCS 1.2.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells
</LI><LI> addition of 1 µl of the Plasmid P7 pYes2_RFC25 MCS 1.2
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_June_24th"><B>Sunday, June 24th</B></SPAN></H2><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_of_E.coli_XL1-Blue_with_pKS2.C2.B5Hyg-PAL-4Cl-CHS"> Miniprep of <I>E.coli</I> XL1-Blue with pKS2µHyg-PAL-4Cl-CHS </SPAN></H3><P><B>Investigator:</B> Ingmar, Volker
</P><P><B>Aim of the experiment:</B> Plasmid purification
</P><P>Operation Sequence:
</P><UL><LI> A single clone of E.coli XL1-Blue with pKS2µHyg-PAL-4Cl-CHS was picked an transferred to 5 ml LB Amp on saturday evening. Incubation overnight at 37°C 180 rpm.
</LI><LI> Using a Quiagen kit a miniprep of the overnight culture was done.
</LI></UL></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_remove_NgoMIV_from_pTUM104_2"> Quick Change mutagenis to remove NgoMIV from pTUM104 </SPAN></H3><P><B>Investigator:</B> Ingmar, Volker
</P><P><B>Aim of the experiment:</B> Removal of a NgoMIV restriction site in the backbone of pTUM104.
</P><P>Operational sequence:
</P><UL><LI> A single clone of <I>E. coli</I> pYes2_RFC25 MCS 1.2 was picked an transferred to 5 ml LB Amp. Incubation overnight at 37°C 180 rpm.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_2_Biobricks_into_E._coli_XL1-Blue"> Transformation of 2 Biobricks into <I>E. coli</I> XL1-Blue </SPAN></H3><P><B>Investigator:</B> Jeffery Truong
</P><P><B>Aim of the experiment:</B> Transformation of Biobricks into <I>E. coli</I> for plasmid propagation for PCR with new RFC pre- and suffix primer in order to do protein fusions.
</P><UL><LI> 2 Biobricks from the distribution kit were used: First, LexA (BBa_K105005, Plate 3 Well 9E) in the pSB1A2 plasmid with ampicillin resistance and second, the heme oxygenase (BBa_I15008, Plate 2 Well 13J) in the pSB2K3 plasmid with kanamycin resistance.
</LI></UL><UL><LI> 10 µL of autoclaved H2O were added to each well on the distribution kit. The well immediately turned red which means that one does it right.
</LI></UL><UL><LI> The now resuspended DNA liquids were transferred into a new ERG on ice.
</LI></UL><UL><LI> CaCL2 competent E. coli XL1-Blue cells from the stock were gently defrezed on ice.
</LI></UL><UL><LI> For each Biobrick 100 µL cells were used and pooled together with 2 µL of plasmid DNA in a ERG on ice.
</LI></UL><UL><LI> Incubation on ice for 30 min.
</LI></UL><UL><LI> 5 min heat shock at 37 °C.
</LI></UL><UL><LI> Each ERG now is transferred in a new ERG prefilled with 1 mL of LB-medium and incubated in a cell-culture shaker at 37 °C for 45 min.
</LI></UL><UL><LI> 100 µL of these cell suspension were plated on antibiotic selection plates (Ampicillin for LexA and Kanamycin for heme oxygenase).
</LI></UL><UL><LI> The rest of the cell suspension is centrifuged at 13000 rpm for 60 s and the supernatant is discarded.
</LI></UL><UL><LI> The pellet is resuspended with 100 µL for each ERG and is plated on another antibiotic selection plate
</LI></UL><UL><LI> These 4 plates were put at 37 °C overnight
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_3">Week 3</SPAN></H1><DIV class="week" id="WWeek_3"><H2><SPAN class="mw-headline" id="Monday.2C_June_25th"><B>Monday, June 25th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Miniprep_of_E._coli_XL1-Blue_with_pTUM104_RFC25_MCS_1.2"> Miniprep of <I>E. coli</I> XL1-Blue with pTUM104_RFC25 MCS 1.2 </SPAN></H3><P><B>Investigator:</B> Alois, Martin
</P><P><B>Aim of the experiment:</B> proof of successful removal of NgoMIV in the backbone of pTUM104
</P><P>Operation Sequence:
</P><UL><LI> Mini prep of pTUM104 1.2. The resulting purified DNA is P33.
</LI><LI> Control digest of pTUM104_RFC25 MCS 1.2 and p13 (+ analytical gel electrophoresis: 90 V, 1 h:
</LI></UL><PRE>*  15 µl ddH20
*   2 µl NEBuffer4
* 0,5 µl NgoMIV
* 2,5 µl pTUM104 1.2/p13
* 37°C, 1 h.
</PRE></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_remove_SpeI_from_pTUM104_RFC25_MCS_1.2"> Quick Change mutagenis to remove SpeI from pTUM104_RFC25 MCS 1.2 </SPAN></H3><P><B>Investigator:</B> Ingmar, Volker
</P><P><B>Aim of the experiment:</B> Generation of an <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible version of the pTUM104 Vector.
</P><P><B>PCR</B><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P33 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O44 (10 pmol/µL)
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O45 ((10 pmol/µL)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>15
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>The procedure was furthermore applied to P13 and P14.
</LI><LI>The vector resulting from the <B>PCR-product was named pTUM104_RFC25 MCS 1.3</B>.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> This operation sequence was applied to the PCR prducts of P33, P13 and P14 respectively.
</LI><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_cells_on_antibiotic_selection_plates:_pSB1A2_plasmid_with_BBa_K105005_.28LexA.29_and_pSB2K3_plasmid_BBa_I15008_.28heme_oxygenase.29"> Picking of E. coli cells on antibiotic selection plates: pSB1A2 plasmid with BBa_K105005 (LexA) and pSB2K3 plasmid BBa_I15008 (heme oxygenase) </SPAN></H3><P><B>Investigator:</B> Jeffery Truong
</P><P><B>Aim of the experiment:</B> Picking colonies from transformed E. coli XL1-Blue, 4x picked for each Biobrick.
</P><UL><LI> pSB1A2 plasmid with BBa_K105005 (LexA): Colonies were on both ampicillin selection plates, the one with diluted cell suspension and the one with concentrated E. coli cell suspension. Typical E. coli colony morphology. Picking was performed on the plate with diluted cell suspension.
</LI></UL><UL><LI> pSB2K3 plasmid BBa_I15008 (heme oxygenase): Colonies were only on the kanamycin selection plate with concentrated cell suspension. The one with diluted susepension was empty. Typical but very small E. coli colonies. Picking was performed from the first plate.
</LI></UL><UL><LI> Picked pipette tips was transferred into a special cell-culture tubes with air-permeable, but sterile cover. In each tube 4 mL of LB-medium + ampicillin (???)(for pSB1A2) or kanamycin (35 mg/mL) (for pSB2K3).
</LI></UL><UL><LI> 4 colonies for each Biobrick was picked; total: 8 tubes overnight culture.
</LI></UL><UL><LI> These tubes were transferred in a cell culture shaker at 37 °C and were incubated overnight
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_June_26th"><B>Tuesday, June 26th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_remove_SpeI_from_pTUM104_RFC25_MCS"> Quick Change mutagenis to remove SpeI from  pTUM104_RFC25 MCS </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Removal of a SpeI restriction site in the backbone of pTUM104.
</P><P>Operational sequence:
</P><UL><LI> For each transformation of the PCR-products of P14 and P33 a single clone was picked an transferred to 6 ml LB Amp. Incubation overday at 37°C 180 rpm. The transfomation with the PCR product of P13 was not successfull. Therfore no clone could be picked.
</LI><LI> Using a Quiagen kit a miniprep of the overnight culture was done.
</LI><LI> The resulting purified DNA was aliquoted in new tubes labeled as follows:
</LI></UL><P>PCR product of P33(transformation done by Ingmar): P29
</P><P>PCR product of P33(transformation done by Saskia&amp;Jara): P30
</P><P>PCR product of P14(transformation done by Ingmar): P31
</P><P>PCR product of P14(transformation done by Saskia&amp;Jara): P32
</P><UL><LI>Afterwards a control digestion of P29-P32 was done.
</LI></UL><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Plasmid
</TD><TD>P29
</TD><TD>P30
</TD><TD>P31
</TD><TD>P32
</TD></TR><TR><TD>NEB4 buffer
</TD><TD>2 µl
</TD><TD>2 µl
</TD><TD>2 µl
</TD><TD>2 µl
</TD></TR><TR><TD>DNA
</TD><TD>2,5 µl
</TD><TD>2,5 µl
</TD><TD>5 µl
</TD><TD>5 µl
</TD></TR><TR><TD>SpeI-HF
</TD><TD>0.25 µl
</TD><TD>0.25 µl
</TD><TD>0.25 µl
</TD><TD>0.25 µl
</TD></TR><TR><TD>NgoMIV
</TD><TD>0.25 µl
</TD><TD>0.25 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>15 µl
</TD><TD>15 µl
</TD><TD>12.75 µl
</TD><TD>12.75 µl
</TD></TR><TR><TD>Sum
</TD><TD>20 µl
</TD><TD>20 µl
</TD><TD>20 µl
</TD><TD>20 µl
</TD></TR></TBODY></TABLE><UL><LI>Incubation at 37 °C for 1h.
</LI><LI>Verification of control digest by agarose gel electrophoresis: 
</LI></UL><P>20 µl of each digest product was mixed with 2 µl DNA loading buffer and loaded into the gel. The separation process lasted 1h at 90 V.
</P></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Verification_of_the_PCR_products_P30.2C_P31_and_P33"> Verification of the PCR products P30, P31 and P33</SPAN></H3><P><B>Investigator:</B> Saskia, Jara
</P><P><B>Aim of the experiment:</B> Verification of the PCR produts P30, P31 and P33
</P><P><B>Nano Drop</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Sample</B></TD><TD><B>concentration [ng/µl]</B></TD><TD><B>260/280</B></TD></TR><TR><TD>P33
</TD><TD>1072.6
</TD><TD>1.01
</TD></TR><TR><TD>P30
</TD><TD>1588
</TD><TD>1.28
</TD></TR><TR><TD>P31
</TD><TD>926.2
</TD><TD>0.82
</TD></TR></TBODY></TABLE><P><B>Analytical gel electrophoresis</B></P><UL><LI> gel: 1 %
</LI><LI> band 1: 10 µl DNA ladder (1kb)
</LI><LI> band 2: P30
</LI><LI> band 3: P33
</LI><LI> band 4: P31
</LI></UL></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Control_of_the_competent_cells_and_transformation_with_P20"> Control of the competent cells and transformation with P20</SPAN></H3><P><B>Investigator: Saskia, Jara</B></P><P>Aim of the experiment: Control of the competent cells and transformation with P20
<B>Transformation</B></P><UL><LI>competent cells: by Simon and Ingmar
</LI><LI>plasmid: P20
</LI></UL><P>result:
</P><UL><LI>successful transformation: red colonies
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="PCR_of_PAL.2C_4CL.2C_CHS.2C_OMT_.28Xanthohumol-CoA.29"> PCR of PAL, 4CL, CHS, OMT (Xanthohumol-CoA) </SPAN></H3><P><B>Investigator: Daniela, Mary</B></P><P>Determination of concentration of plasmids (Nanodrop): 
c(pKS2µHyg-PAL-4CL-CHS) = 500 ng/µl
c (pOMT) = 20 ng/µl
</P><P>PCR:
</P><TABLE class="wikitable" cellpadding="10" border="1px"><TBODY><TR><TD> Name of tube </TD><TD> Enzyme </TD><TD> consensus (+)/ consensless (-) </TD><TD> used Oligos
</TD></TR><TR><TD> CHS - </TD><TD> CHS </TD><TD> -</TD><TD> O13 and O24
</TD></TR><TR><TD> CHS + </TD><TD> CHS </TD><TD> +</TD><TD> O23 and O24
</TD></TR><TR><TD> PAL - </TD><TD> PAL </TD><TD> -</TD><TD> O15 and O16
</TD></TR><TR><TD> PAL + </TD><TD> PAL </TD><TD> +</TD><TD> O22 and O16
</TD></TR><TR><TD> OMT - </TD><TD> OMT </TD><TD> -</TD><TD> O17 and O26
</TD></TR><TR><TD> OMT + </TD><TD> OMT </TD><TD> +</TD><TD> O25 and O26
</TD></TR><TR><TD> 4CL - </TD><TD> 4CL </TD><TD> -</TD><TD> O19 and O20
</TD></TR><TR><TD> 4CL + </TD><TD> 4CL </TD><TD> +</TD><TD> O21 and O20
</TD></TR></TBODY></TABLE><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>dNTP's (each 2.5 mM)
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II (2.5 U/µL)
</TD></TR><TR><TD>5 µl
</TD><TD>1:10 dilution of used forward primers (10µM)
</TD></TR><TR><TD>5 µl
</TD><TD>1:10 dilution of used reversed primers (10µM)
</TD></TR><TR><TD>1 µl
</TD><TD>DNA (pKS2µHyg-PAL-4CL-CHS 50 ng/µL or pOMT 20 ng/µL)
</TD></TR><TR><TD>29.5 µL
</TD><TD>bidest. sterile Water
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 5 min (and adding Pfu Ultra after 3 min)
</TD></TR><TR><TD>2
</TD><TD>30
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>46°C
</TD><TD> 2.5 min
</TD></TR><TR><TD>72°C
</TD><TD> 1.5 min
</TD></TR><TR><TD>3
</TD><TD>72°C
</TD><TD> 5 min
</TD></TR></TBODY></TABLE><P>PCR purification
</P><UL><LI>Purification was done using QIAquick PCR Purification Kit (250)
</LI></UL><P>Analytical Gel Electrophoresis:
</P><P>-&gt; going on with CHS, 4CL and OMT; the PCR of PAL will be repeated
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_and_analytical_gel_of_picked_transformed_overnight_culture_with_pSB1A2_plasmid_with_BBa_K105005_.28LexA.29_pSB2K3_plasmid_BBa_I15008_.28heme_oxygenase.29"> Miniprep and analytical gel of picked transformed overnight culture with pSB1A2 plasmid with BBa_K105005 (LexA) pSB2K3 plasmid BBa_I15008 (heme oxygenase) </SPAN></H3><P><B>Investigator: Jeffery Truong, Georg Schützinger</B></P><P>Aim of the experiment: Plasmid isolation from the picked transformed overnight E. coli cells with pSB1A2 plasmid with BBa_K105005 (LexA) pSB2K3 plasmid BBa_I15008 (heme oxygenase).
</P><UL><LI> The LB-medium with antibiotics of every tube was opaque which means that the picked cells were successfully inoculated.
</LI></UL><UL><LI> Centrifugation step at 5000 rpm for 10 min at 16 °C.
</LI></UL><UL><LI> Every single step now was performed on ice. 
</LI></UL><UL><LI> Miniprep (Qiagen Qiaprep spin) after manufacturer's protocoll.
</LI></UL><UL><LI> Analytical restriction master mix was prepared after following scheme (Using XbaI and PstI):
</LI><LI> 4.4 µL XbaI
</LI><LI> 4.4 µL PstI
</LI><LI> 17.6 µL Tango-buffer (10x)
</LI><LI> 132 µL ELGA H2O
</LI></UL><UL><LI> 17.5 µL from the master mix was poooled together with 2.5 µL of plasmid DNA. That corresponds to 2.5 µL of plasmid DNA, 0.25 µL XbaI, 0.25 µL PstI, 2 µL Tango-buffer (10x), 15 µL ELGA H2O.
</LI></UL><UL><LI> Incubation at 37 °C for 120 min on a ERG heating unit.
</LI></UL><UL><LI> BUT: error was performed during preparing the digested plasmid DNA for analytical gelelectrophoresis in the dilution step. 1:10 dilution of analyctical probe with DNA loading buffer:
</LI><LI> 3.3 µL sample (contains already 1x loading buffer!)
</LI><LI> 0.7 µL loading buffer (?x)
</LI><LI> 6 µL TAE-buffer
</LI></UL><UL><LI> Should have done: 3 µL sample + 1 µL loading buffer (10x) + 6 µL TAE-buffer (1x)
</LI></UL><UL><LI> DNA-ladder preperation: 10 µL ladder stock solution + 10 µL DNA loading buffer + 80 µL TAE-buffer. 10 µL of this solution was pipetted in one gel pocket of the prepared 1% agarose gel including ehtiudiumbromid.
</LI></UL><UL><LI> 20 µL of each samples were also pipetted into the gel pockets.
</LI></UL><UL><LI> The gel pockets were pipetted after following scheme:
</LI></UL><TABLE class="wikitable" cellpadding="10" border="1px"><TBODY><TR><TD> Heme oxygenase (colony 1) </TD><TD> Heme oxygenase (colony 2) </TD><TD> Heme oxygenase (colony 3) </TD><TD> Heme oxygenase (colony 4) </TD><TD> DNA-ladder </TD><TD> LexA (colony 1) </TD><TD> LexA (colony 2) </TD><TD> LexA (colony 3) </TD><TD> LexA (colony 4)
</TD></TR></TBODY></TABLE><UL><LI> Gel electrophoresis at 90 V
</LI></UL><UL><LI> After 20 min the resolution was still poor; 20 min longer.
</LI></UL><UL><LI> Analytical Gel okay, but samples interchanged
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_June_27th"><B>Wednesday, June 27th</B></SPAN></H2><DIV class="limonene"><H3><SPAN class="mw-headline" id="Repetition_of_analytic_gel_of_21st_June"> Repetition of analytic gel of 21st June</SPAN></H3><P><B>Investigator: Andrea, Lara</B></P><P>Buffer systems were adjusted. -&gt; only use of one buffer per reaction.
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_June_29th"><B>Friday, June 29th</B></SPAN></H2><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparative_digest_of_PCR-products_of_4CL.2C_CHS_and_OMT"> Preparative digest of PCR-products of 4CL, CHS and OMT </SPAN></H3><P><B>Investigator: Katrin, Mary</B></P><P>each digestion will dure 2.5h at 37°C
</P><UL><LI>CHS: digestion with Xba1 and HF-Age1 (both NEB)
</LI></UL><TABLE class="wikitable" cellpadding="10" border="1px"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD> 25µl </TD><TD> PCR-product
</TD></TR><TR><TD> 5µl </TD><TD> Buffer NEB4
</TD></TR><TR><TD> 0.5µl </TD><TD> BSA
</TD></TR><TR><TD> 1µl </TD><TD> Xba1 (NEB; 20u/µl)
</TD></TR><TR><TD> 1µl </TD><TD> HF-Age1 (NEB; 20u/µl)
</TD></TR><TR><TD> 17.5µl </TD><TD> bidest. sterile H2O
</TD></TR></TBODY></TABLE><UL><LI>OMT: digestion with Xba1 and HF-Age1 (both NEB)
</LI></UL><TABLE class="wikitable" cellpadding="10" border="1px"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD> 25µl </TD><TD> PCR-product
</TD></TR><TR><TD> 5µl </TD><TD> Buffer NEB4
</TD></TR><TR><TD> 0.5µl </TD><TD> BSA
</TD></TR><TR><TD> 1µl </TD><TD> Xba1 (NEB; 20u/µl)
</TD></TR><TR><TD> 1µl </TD><TD> HF-Age1 (NEB; 20u/µl)
</TD></TR><TR><TD> 17.5µl </TD><TD> bidest. sterile H2O
</TD></TR></TBODY></TABLE><UL><LI>4CL: digestion with Xba1 and Pst1 (both Fermentas)
</LI></UL><TABLE class="wikitable" cellpadding="10" border="1px"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD> 25µl </TD><TD> PCR-product
</TD></TR><TR><TD> 5µl </TD><TD> Buffer Tango
</TD></TR><TR><TD> 2µl </TD><TD> Xba1 (Fermentas; 10u/µl)
</TD></TR><TR><TD> 3µl </TD><TD> Pst1 (Fermentas; 10u/µl)
</TD></TR><TR><TD> 15µl </TD><TD> bidest. sterile H2O
</TD></TR></TBODY></TABLE></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparative_Gelelectrophoresis_of_PCR-products_of_4CL.2C_CHS"> Preparative Gelelectrophoresis of PCR-products of 4CL, CHS </SPAN></H3><P><B>Investigator: Katrin, Mary</B></P><P>Gelextraction of 4CL+, 4CL-, CHS+, CHS- (bands are as expected; +=with consensus-sequence, -=without consensus-sequence)
</P><P>DNA-purification with Kit from Quiagen 
</P></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_PstI_in_URA3_from_pTUM104_RFC25_MCS_1.2">  Quick Change mutagenesis to remove PstI in URA3 from pTUM104_RFC25 MCS 1.2 </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Generation of an <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible version of the pTUM104 Vector.
</P><P><B>PCR</B><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P29 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O40 (10 pmol/µL)
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O41 ((10 pmol/µL)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Turbo DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>15
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6.5 min
</TD></TR></TBODY></TABLE><UL><LI>The procedure was furthermore applied to P31.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> This operation sequence was applied to the PCR prducts of P29 and P31 respectively.
</LI><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_June_30th"><B>Saturday, June 30th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_remove_PstI_in_the_URA_3_gene_from_pTUM104_RFC25_MCS"> Quick Change mutagenis to remove PstI in the URA 3 gene from  pTUM104_RFC25 MCS </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Removal of a PstI restriction site in the backbone of pTUM104.
</P><P>Operational sequence:
</P><UL><LI> For each transformation of the PCR-products of P29 and P30 a single clone was picked an transferred to 6 ml LB Amp. Incubation overnight at 37°C 180 rpm. The transfomation with the PCR product of P31, P32 and P33 was not successfull. Therfore no clone could be picked from these plates and four instead of one clone was picked from the plates containing the  transformations of P29.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_July_1st"><B>Sunday, July 1st</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_remove_PstI_in_the_URA_3_gene_from_pTUM104_RFC25_MCS_2"> Quick Change mutagenis to remove PstI in the URA 3 gene from  pTUM104_RFC25 MCS </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Removal of a PstI restriction site in the backbone of pTUM104.
</P><P>Operational sequence:
</P><UL><LI> Using a Quiagen kit a miniprep of the overnight culture was done.
</LI><LI> The resulting purified DNA was aliquoted in new tubes labeled as follows:
</LI></UL><P>1st PCR product of P29: P34
</P><P>2nd PCR product of P29: P35
</P><P>3rd PCR product of P29: P36
</P><P>4th PCR product of P29: P37
</P><P>PCR product of P30: P38
</P><UL><LI>Afterwards a control digestion of P34-P38 was done.
</LI></UL><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Plasmid
</TD><TD>P34
</TD><TD>P35
</TD><TD>P36
</TD><TD>P37
</TD><TD>P38
</TD></TR><TR><TD>Fermentas 10x R buffer
</TD><TD>0.5 µl
</TD><TD>0.5 µl
</TD><TD>0.5 µl
</TD><TD>0.5 µl
</TD><TD>0.5 µl
</TD></TR><TR><TD>DNA
</TD><TD>2 µl
</TD><TD>2 µl
</TD><TD>2 µl
</TD><TD>2 µl
</TD><TD>5 µl
</TD></TR><TR><TD>PstI
</TD><TD>0.25 µl
</TD><TD>0.25 µl
</TD><TD>0.25 µl
</TD><TD>0.25 µl
</TD><TD>0.25 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>17.25 µl
</TD><TD>17.25 µl
</TD><TD>17.25 µl</TD><TD>17.25 µl
</TD><TD>14.25 µl
</TD></TR><TR><TD>Sum
</TD><TD>20 µl
</TD><TD>20 µl
</TD><TD>20 µl
</TD><TD>20 µl
</TD><TD>20 µl
</TD></TR></TBODY></TABLE><UL><LI>Incubation at 37 °C for 1h.
</LI><LI>Verification of control digest by agarose gel electrophoresis: 
</LI></UL><P>20 µl of each digest product was mixed with 4 µl 6x DNA loading buffer and loaded into the gel. The separation process lasted 1h at 90 V.
</P><UL><LI>All digest products show the expected two bonds at 3526 bp and at 2332 bp. The Miniprep product P35 was chosen to be used for the further Quickchange Mutagenesis.
</LI></UL></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_PstI_in_the_2.C2.B5_ori_from_pTUM104_RFC25_MCS">  Quick Change mutagenesis to remove PstI in the 2µ ori from pTUM104_RFC25 MCS  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Generation of an <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible version of the pTUM104 Vector.
</P><P><B>PCR</B><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P35 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O42 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P35 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O43 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_4">Week 4</SPAN></H1><DIV class="week" id="WWeek_4"><H2><SPAN class="mw-headline" id="Monday.2C_July_2nd"><B>Monday, July 2nd</B></SPAN></H2><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Repetition_of_PCR_of_PAL"> Repetition of PCR of PAL</SPAN></H3><P><B>Investigator: Mary</B></P><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>dNTP's (each 2.5 mM)
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II (2.5 U/µL)
</TD></TR><TR><TD>5 µl
</TD><TD>1:10 dilution of used forward primers (10µM)
</TD></TR><TR><TD>5 µl
</TD><TD>1:10 dilution of used reversed primers (10µM)
</TD></TR><TR><TD>1 µl
</TD><TD>DNA (pKS2µHyg-PAL-4CL-CHS 50 ng/µL)
</TD></TR><TR><TD>29.5 µL
</TD><TD>bidest. sterile Water
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 2 min (and adding Pfu Ultra after 2 min)
</TD></TR><TR><TD>2
</TD><TD>30
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>72°C
</TD><TD> 2.5 min
</TD></TR><TR><TD>3
</TD><TD>72°C
</TD><TD> 5 min
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_of_LexA_with_primers_including_the_RFC25_pre-_and_suffix"> PCR of LexA with primers including the RFC25 pre- and suffix </SPAN></H3><P><B>Investigator: Jeffery Truong, Georg Schützinger</B></P><P>Aim of the experiment: The Biobrick BBa_K105005 (LexA) has a RFC10 pre- and suffix, but we need RFC25 pre- and suffix for protein fusion, so we have to do a PCR with primer containing the RFC25 pre- and suffix.
</P><UL><LI> The received forward and reverse primer TUM12-LexA-fw and TUM12-LexA-rv are resuspended in 204 µL and 221 µL ELGA water as described in the data sheet to get a final primer concentration of 100 pmol/µL=100 µM. For the PCR reaction mixture we took 0.5 µL of these resuspended primer and add 4.5 µL of ELGA water to get a final primer concentration of 10 µM.
</LI></UL><UL><LI> Clone 3 of BBa_K105005 (LexA) has beed choosen for the PCR (ERG No. P23).
</LI></UL><P><B>PCR reaction mixture</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (BBa_K105005) from P23 (Clone 3)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ELGA Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>55 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>60 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>overnight
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_3rd"><B>Tuesday, July 3rd</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytic_gelelectrophoresis_of_cleaned_up_PCR_product_from_LexA_with_primer_containing_RFC25_pre-_and_suffix"> Analytic gelelectrophoresis of cleaned up PCR product from LexA with primer containing RFC25 pre- and suffix </SPAN></H3><P><B>Investigator: Jeffery Truong</B></P><P>Aim of the experiment: Analytical gelelectrophoresis of cleaned up PCR product from LexA (BBa_K105005) with primer containing the RFC25 pre- and suffix (TUM12-LexA-fw and TUM12-LexA-rv).
</P><UL><LI> The clean-up of the PCR product LexA (BBa_K105005) with primer containing the RFC25 pre- and suffix (TUM12-LexA-fw and TUM12-LexA-rv) was performed with the QIAquick PCR Purification Kit from Qiagen after manufacturer's protocoll.
</LI></UL><UL><LI> 5 µL of the purificated PCR product was taken to perform a analytical gelelectrophoresis to verify the success of the PCR.
</LI></UL><UL><LI> 1% agarose gel containing ethidium bromide was used for the analytical gelelectrophoresis.
</LI></UL><UL><LI> The analytical gelelectrophoresis was performed for 60 min at 90 V.
</LI></UL><UL><LI> Scheme of the gel:
</LI></UL><TABLE class="wikitable" cellpadding="10" border="1px"><TBODY><TR><TD> 100 bp ruler </TD><TD> PCR product of BBa_105005 </TD><TD> 1000 bp ruler
</TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="LS:_Plating_of_Schwab_expression_stains_.23106_.26_.23108_which_contain_lavendula_limonene_synthase">LS: Plating of Schwab expression stains #106 &amp; #108 which contain lavendula limonene synthase</SPAN></H3><P><B>Investigator: Lara</B></P><P>Aim of the experiment: To get colonies of BL21 strains containing lavendula LS for amplification and subsequent plasmid extraction.
</P><UL><LI> Schwab strain #106 was plated on a chloramphenicol containing LB plate. #108 was plated on a amp+chlp containing LB plate. The plates were incubated at 37°C over night.
</LI></UL></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_remove_PstI_in_the_2.C2.B5_Ori_from_pTUM104_RFC25_MCS"> Quick Change mutagenis to remove PstI in the 2µ Ori from  pTUM104_RFC25 MCS </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Removal of a PstI restriction site in the backbone of pTUM104.
</P><P>Operational sequence:
</P><UL><LI> From the transformation of the PCR-product of P35 two  single clones were picked an transferred to 6 ml LB Amp. Incubation overnight at 37°C 180 rpm. 
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="PCR_of_BBa_I742111_.28Limonenesynthase.29_Clone_3_.28Transformation_of_19.06.12.29"> PCR of BBa_I742111 (Limonenesynthase) Clone 3 (Transformation of 19.06.12) </SPAN></H3><P><B>Investigator: Andrea</B></P><P>PCR used forward primer with consensus sequence;
<B>PCR reaction mixture</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer O27
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer O30
</TD></TR><TR><TD>0.25 µl
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (BBa_I742111) Clone 3
</TD></TR><TR><TD>35.75 µl
</TD><TD>dd water
</TD></TR><TR><TD>50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>PCR used forward primer without consensus sequence;
<B>PCR reaction mixture</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer O28
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer O30
</TD></TR><TR><TD>0.25 µl
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (BBa_I742111) Clone 3
</TD></TR><TR><TD>35.75 µl
</TD><TD>dd water
</TD></TR><TR><TD>50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>47 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>1,75 min
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>1 h
</TD></TR></TBODY></TABLE><P><B>Analytical Gelelectrophoresis</B></P><UL><LI> 5 µl DNA + 1 µl loading buffer
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_4th"><B>Wednesday, July 4th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_remove_PstI_in_the_2.C2.B5_Ori_from_pTUM104_RFC25_MCS_2"> Quick Change mutagenis to remove PstI in the 2µ Ori from  pTUM104_RFC25 MCS </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Removal of a PstI restriction site in the backbone of pTUM104.
</P><P>Operational sequence:
</P><UL><LI> Using a Quiagen kit a miniprep of the overnight culture was done.
</LI><LI> The resulting purified DNA was aliquoted in new tubes labeled as follows:
</LI></UL><P>1st Transformation of P35: P43
</P><P>2nd Transformation of P35: P44
</P><UL><LI>Afterwards a control digestion of P43 and P44 was done.
</LI></UL><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Plasmid
</TD><TD>P43
</TD><TD>P44
</TD></TR><TR><TD>Fermentas 10x R buffer
</TD><TD>0.5 µl
</TD><TD>0.5 µl
</TD></TR><TR><TD>DNA
</TD><TD>5 µl
</TD><TD>5 µl
</TD></TR><TR><TD>PstI
</TD><TD>0.25 µl
</TD><TD>0.25 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>14.25 µl
</TD><TD>14.25 µl
</TD></TR><TR><TD>Sum
</TD><TD>20 µl
</TD><TD>20 µl
</TD></TR></TBODY></TABLE><UL><LI>Incubation at 37 °C for 1h.
</LI><LI>Verification of control digest by agarose gel electrophoresis: 
</LI></UL><P>20 µl of each digest product was mixed with 4 µl 6x DNA loading buffer and loaded into the gel. The separation process lasted 1h at 90 V.
</P><UL><LI>All digest products show the expected bond at 5858 bp. The Miniprep product P40 was chosen to be used for the further Quickchange Mutagenesis.
</LI></UL></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_insert_Ala_in_front_of_the_Strep_-_tag_II_in_pTUM104_RFC25_MCS"> Quick Change mutagenis to  insert Ala in front of the Strep - tag II in pTUM104_RFC25 MCS  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Generation of an <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible version of the pTUM104 Vector; operating purfication possibility via Strep-tag II.
</P><P><B>PCR</B><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P44
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O54 (10 pmol/µL)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O55 ((10 pmol/µL)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Turbo DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>16
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Analytical_Gelelektrophoresis_of_PCR-Products_of_PAL">Analytical Gelelektrophoresis of PCR-Products of PAL</SPAN></H3><P><B>Investigator: Mary</B></P><P>Aim of the experiment: purification and testing if PCR was successful
</P><P>Purification of PCR-Products with purification kit from quiagen
analytical gelelectrophoresis of PAL+, PAL-; expected band at 2,1 kb
</P><P>Analytical Gel Electrophoresis:
<A href="/File:TUM12_20120704_PAL-PCR_v2.tiff" title="File:TUM12 20120704 PAL-PCR v2.tiff">File:TUM12 20120704 PAL-PCR v2.tiff</A></P><P>-&gt; PCR was not successful, no band at 2,1 kb
</P><P>-&gt; next steps: new Design of Primer and repetition of PCR with new primers 
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_clones_of_Schwab_expression_stains_.23106_.26_.23108">Picking of clones of Schwab expression stains #106 &amp; #108</SPAN></H3><P><B>Investigator: Andrea</B></P><P>Aim of the experiment: Getting clones of cells with plasmids containing the gene for limonenesynthase and amplification of these clones for further plasmid preparation.
</P><P><B>Picking of Clones</B></P><UL><LI>2 clones of every stain were picked
</LI><LI>Incubation at 37 °C in LB + Amp (#108) / LB + Kan (#106)
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparative_Gelelectrophoresis_of_PCR-products_of_OMT"> Preparative Gelelectrophoresis of PCR-products of OMT </SPAN></H3><P><B>Investigator: Mary, Katrin</B></P><P>Aim of the experiment: Purification of the previously digested DNA, test if digestion was successful
</P><P>picture follows!
</P><P>Gelextraction of OMT- and OMT+  (bands are as expected; +=with consensus-sequence, -=without consensus-sequence)
</P><P>DNA-purification with Kit from Quiagen 
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_5th"><B>Thursday, July 5th</B></SPAN></H2><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_plasmids_containing_lavendula_LS"> Preparation of plasmids containing lavendula LS</SPAN></H3><P><B>Investigator: Lara </B></P><P><B>Aim:</B> Purify Schwab plasmids containing lavendula limonene synthase
</P><P>Experiment was conducted using Qiagen Plasmid Miniprep Kit. 
</P><P>P39: Plasmid from Schwab strain #106 (1); 35 ng/µl
</P><P>P40: Plasmid from Schwab strain #106 (2); 72 ng/µl
</P><P>P41: Plasmid from Schwab strain #108 (1); 240 ng/µl
</P><P>P42: Plasmid from Schwab strain #108 (2); 120 ng/µl
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Restriction_digest_of_Schwab_plasmids"> Restriction digest of Schwab plasmids</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> To check whether extracted plasmids from Schwab expression strains #106 &amp; #108 are OK.
</P><P>Digest of plasmid from strain #106 was conducted with the following protocoll:
</P><P>500 ng Plasmid DNA
</P><P>0,25 µl Nco1
</P><P>0,25 µl Hind 3
</P><P>2 µl Buffer Tango
</P><P>dd H2O to a total volume of 20 µl.
</P><P>Digest of plasmid from strain #108 was conducted with the following protocoll:
</P><P>500 ng Plasmid DNA
</P><P>0,25 µl EcoR1
</P><P>0,25 µl Not1
</P><P>2 µl Buffer Orange 
</P><P>dd H2O to a total volume of 20 µl.
</P><P>(All enzymes and buffers were from Fermentas).
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_gel_electrophoresis_of_digested_Schwab_plasmids"> Analytical gel electrophoresis of digested Schwab plasmids</SPAN></H3><P><B>Investigator:</B> Lara
<B>Aim</B>: Check plasmids for insert.
</P><P>Plasmids were digested, 2 µl loading dye (10x) was added to each sample. Gel was loaded in the following order:
</P><P>1. 6 µl gene ruler 1 kb, 2.-6. 11 µl of: P39, P40, P41, P42, Xanthohumol-Plasmid (Katrin)
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Analytical_Gelelektrophoresis_of_plasmid_pKS2.C2.B5Hyg-PAL-4Cl-CHS">Analytical Gelelektrophoresis of plasmid pKS2µHyg-PAL-4Cl-CHS</SPAN></H3><P><B>Investigator: Katrin</B></P><P>Aim of the experiment: testing if PAL is part of the plasmid that was sent to us (troubleshooting because PCR of PAL was not successful)
</P><P>digestion took 1 h at 37°C
</P><UL><LI>digestion with ApaI (Fermentas)
</LI></UL><TABLE class="wikitable" cellpadding="10" border="1px"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD> 9.3 µl </TD><TD> plasmid pKS2µHyg-PAL-4Cl-CHS (miniprep)
</TD></TR><TR><TD> 2 µl </TD><TD> Buffer B
</TD></TR><TR><TD> 0.5 µl </TD><TD> ApaI (Fermentas; 10u/µl)
</TD></TR><TR><TD> 8.2 µl </TD><TD> bidest. sterile H2O
</TD></TR></TBODY></TABLE><P>analytical gelelectrophoresis: expected band at 3,14 kb (Gal-PAL-XK)
</P><P>-&gt; experiment was successful, PAL is part of the plasmid pKS2µHyg-PAL-4Cl-CHS
</P></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_insert_Ala_in_front_of_the_Strep_-_tag_II_in_pTUM104_RFC25_MCS_2"> Quick Change mutagenis to insert Ala in front of the Strep - tag II in pTUM104_RFC25 MCS </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Generation of an <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible version of the pTUM104 Vector; operating purfication possibility via Strep-tag II. 
</P><P>Operational sequence:
</P><UL><LI> From the transformation of the PCR-product of P44 two  single clones were picked an transferred to 6 ml LB Amp. Incubation overday at 37°C 180 rpm. 
</LI><LI> Using a Quiagen kit a miniprep of the overnight culture was done.
</LI><LI> The resulting purified DNA was aliquoted in new tubes labeled as follows:
</LI></UL><P>1st picked clone of P44: P45
</P><P>2nd picked clone of P44: P46
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Minipreparation_of_biobricks_BBa_J52028.2C_BBa_E2030.2C_BBa_E2020"> Minipreparation of biobricks BBa_J52028, BBa_E2030, BBa_E2020</SPAN></H3><P><B>Investigator: Martin, Alois </B></P><P><B>Aim:</B> Getting &quot;reporter proteins&quot;
</P><UL><LI> 10 µl water war added to the well of the distribution kit (=&gt; red)
</LI><LI> BBa_J52028: GFP with PEST191 tag =&gt; p51
</LI><LI> BBa_E2030: EYFP, yeast optimized =&gt; p52
</LI><LI> BBa_E2020: ECFP, yeast optimized =&gt; p53
</LI><LI> Transformation + Minipreparation (Qiagen Plasmid Miniprep Kit)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_6th"><B>Friday, July 6th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenis_to_insert_Ala_in_front_of_the_Strep_-_tag_II_in_pTUM104_RFC25_MCS_3">Quick Change mutagenis to insert Ala in front of the Strep - tag II in pTUM104_RFC25 MCS</SPAN></H3><UL><LI>Afterwards a control digestion of P45 and P46 was done.
</LI></UL><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Plasmid
</TD><TD>P45
</TD><TD>P46
</TD></TR><TR><TD>Fermentas 10x O buffer
</TD><TD>2 µl
</TD><TD>2 µl
</TD></TR><TR><TD>DNA
</TD><TD>3 µl
</TD><TD>3 µl
</TD></TR><TR><TD>PstI
</TD><TD>0.25 µl
</TD><TD>0.25 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>14.75 µl
</TD><TD>14.75 µl
</TD></TR><TR><TD>Sum
</TD><TD>20 µl
</TD><TD>20 µl
</TD></TR></TBODY></TABLE><UL><LI>Incubation at 37 °C for 1h.
</LI><LI>Verification of control digest by agarose gel electrophoresis: 
</LI></UL><P>20 µl of each digest product was mixed with 4 µl 6x DNA loading buffer and loaded into the gel. The separation process lasted 1h at 90 V.
</P><UL><LI>All digest products show the expected bonds at 5858 bp. The Miniprep product P45 was chosen to check the insertion of Ala in front of the strep-tag II via DNA sequencing.
</LI></UL><P>The results of the sequencing are shown below:
</P><P>
The sequencing results show that the insertion of Alanin in front of the Strep-tag II was not successful.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="PCR_of_Schwab_plasmid_DNA_to_amplify_gene_for_lavendula_LS"> PCR of Schwab plasmid DNA to amplify gene for lavendula LS</SPAN></H3><P><B>Instructor:</B> Lara 
</P><P><B>Aim:</B> PCR of Schwab plasmids with primers which were designed to amplify the lavendula LS gene and to add RFC25 restriction sites.
</P><P>3 different primer combinations were used:
</P><P>1. O33/O37
</P><P>2. O34/O37
</P><P>3. O35/O37
</P><P>Each primer combination was used for plasmid DNA amplification of P40 and P41.
</P><P><B>PCR reaction mixture</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer
</TD></TR><TR><TD>0.25 µl
</TD><TD>OneTaq Hot Start DNA Polymerase (Final: 1.25 units/50 µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (BBa_I742111) Clone 3
</TD></TR><TR><TD>35.75 µl
</TD><TD>dd water
</TD></TR><TR><TD>50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>50 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>1,75 min
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>1 h
</TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytic_Gel_of_PCR_of_Schwab_plasmid_DNA"> Analytic Gel of PCR of Schwab plasmid DNA</SPAN></H3><P><B>Instructor: Andrea</B></P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Minipreparation_of_biobricks_BBa_J52028.2C_BBa_E2030.2C_BBa_E2020_2"> Minipreparation of biobricks BBa_J52028, BBa_E2030, BBa_E2020</SPAN></H3><P><B>Investigator: Martin, Alois </B></P><P><B>Aim:</B> Getting &quot;reporter proteins&quot;
</P><UL><LI> 10 µl water war added to the well of the distribution kit (=&gt; red)
</LI><LI> BBa_J52028: GFP with PEST191 tag =&gt; p51
</LI><LI> BBa_E2030: EYFP, yeast optimized =&gt; p52
</LI><LI> BBa_E2020: ECFP, yeast optimized =&gt; p53
</LI><LI> Transformation + Minipreparation (Qiagen Plasmid Miniprep Kit)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_July_7th"><B>Saturday, July 7th</B></SPAN></H2><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_insert_Ala_in_front_of_the_Strep-tag_II_in_pTUM104_RFC25_MCS">  Quick Change mutagenesis to insert Ala in front of the Strep-tag II in pTUM104_RFC25 MCS  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Generation of an <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible version of the pTUM104 Vector.
</P><UL><LI>As the sequencing of the first attempt to introduce Ala in front of the Strep-tag II did not show a successfull insertion of Ala, the quickchange mutagense was performed once again with a modified setup. Presumably a formation of primer dimers was responsable for the experiment's results. Therefore the second PCR was operated in two steps as shown below.
</LI></UL><P><B>PCR</B><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P44 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O54 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Turbo DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P44 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O55 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Turbo DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_XL1_blue_with_ADH1_promoter_.28BBa_J63005.29.2C_ADH1_terminator.28BBa_J63002.29_and_TEF2_promoter_from_Igem_Distribution_kit">  Transformation of <I>E.coli</I> XL1 blue with ADH1 promoter (BBa_J63005), ADH1 terminator(BBa_J63002) and TEF2 promoter from Igem Distribution kit</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of DNA was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on ampicillin plates for ADH1-P, and ADH1-T and Kanamycin plates for TEF2-Promoter.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin- and kanamycin plates.
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_5">Week 5</SPAN></H1><DIV class="week" id="WWeek_5"><H2><SPAN class="mw-headline" id="Monday.2C_July_9th"><B>Monday, July 9th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_of_colonies_of_TEF2-P.2C_ADH1-P.2C_ADH1-T_.28iGEM.29"> Picking of colonies of TEF2-P, ADH1-P, ADH1-T (iGEM)</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> To 5 µl LB-Medium,5 µl of 1000x stock of ampicillin (ADH-P, ADH1-T) and kanamycin (TEF2-P) were added
</LI><LI> 4 colonies from the plate with TEF2-P and 5 colonies of ADH1-P and ADH1-T were picked
</LI><LI> Each colony was transferred into 5 ml LB-Medium with 1x ampicillin or kanamycin
</LI><LI> Incubation over night at 37°C in the 180rpm cell-culture shaker.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Repetition_of_picking_of_colonies_of_ADH1-P.2C_ADH1-T.2C_TEF2-P_from_iGEM_distribution_kit">Repetition of picking of colonies of ADH1-P, ADH1-T, TEF2-P from iGEM distribution kit</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI> Analytical Gel was loaded with 9 µl digestion and 1 µl 10x buffer
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Repetition_of_picking_of_colonies_of_TEF2-P.2C_ADH1-P.2C_ADH1-T_.28iGEM.29"> Repetition of picking of colonies of TEF2-P, ADH1-P, ADH1-T (iGEM)</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI>Only ADH1 promoter showed cell proliferation. The rest showed no proliferation.The reason was a 10x too high amount of antibiotics.
</LI></UL><UL><LI>Colonies from  transformation from ADH1-promoter and terminator  were picked again and incubated in LB medium with   
</LI></UL><P>ampicillin at a dilution of 1:1000 and for TEF2 –Promoter with Kanamycin at a dilution of 1:1000
</P><UL><LI>From the grown colonies from the transformation with ADH1-Promoter then were the plasmids extracted, using the 
</LI></UL><P>Quiaprep Spin Miniprep kit from Quiagen
</P><UL><LI>The extracted ADH1-P DNA was then analytically digested  with XbaI and PstI from Fermentas
</LI></UL><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>PSTI (Fermentas)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (Fermentas)
</TD></TR><TR><TD>8 µl
</TD><TD>Tango buffer
</TD></TR><TR><TD>60µl
</TD><TD>dd H20
</TD></TR><TR><TD>=70µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 17,5 µl mastermix was 2,5 µl of plasmid DNA added
</LI></UL><UL><LI> to 2,5 ml of DNA 17,5 µl reaction batch were added
</LI><LI> Digestion took place at 37°C for 1 h
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Phycocyanobilin_.28PCB.29_extraction_from_dried_Spirulina_platensis_powder_.28part_1.2F4.29"> Phycocyanobilin (PCB) extraction from dried <I>Spirulina platensis</I> powder (part 1/4) </SPAN></H3><P><B>Investigator: Jeff, Alois, Martin</B></P><P><B>Aim of the experiment:</B> Phycocyanobilin (PCB) is a cofactor neeeded for the funtion of phytochrome B. Phycocyanobilin is covalently bound to Cys457 of phytochrome B. <I>Saccharomyces cerevisiae</I> does not contain endogenous PCB. For proof of concept PCB should be added to the medium. In the following experiment, PCB is extracted from dried <I>Spirulina platensis</I> powder.
</P><P><B>Operational sequence:</B></P><UL><LI> 50 g of <I>Spirulina platensis</I> powder was (from concept-vitalprodukte.de) resuspended in 1.5 l of H2O (30 mg/l) in a beaker covered with aluminium foil.
</LI></UL><UL><LI> Stirring for 10 min at RT.
</LI></UL><UL><LI> Green <I>Spirulina</I> suspension was divided in 6 centrifuge bottles, covered in aluminium foil.
</LI></UL><UL><LI> Centrifation at 10500 rpm at 4 °C (SLA-3000 rotor, Thermo Scientific) for 1 h.
</LI></UL><UL><LI> Supernatant was discarded
</LI></UL><UL><LI> 25 ml of MeOH added to each bottle and was heavily shaked to resuspend the pellet for the next cleaning step with MeOH.
</LI></UL><UL><LI> Each bottle with the resuspended pellet were filled with MeOH to a final volume of 250 ml and shaked again to fully resuspend the pellet.
</LI></UL><UL><LI> Centrifugation at 10500 rpm at 4 °C (SLA-3000 rotor, Thermo Scientific) for 10 min.
</LI></UL><UL><LI> Supernatant was discarded.
</LI></UL><UL><LI> The last 4 steps were repeated until the supernatant of the washed pellet was colorless or cyanblue but not green anymore (Regulary, it takes 3 or 4 times). Pellet should be cyanblue now.
</LI></UL><UL><LI> The pellet of the 6 centrifuge tubes was collected in a sole centrifugation tube, covered in aluminium foil tube, by scratching it out with a small spoon.
</LI></UL><UL><LI> The remaining rest of the pellet which cannot be scratched out were resuspended in a small amount of MeOH and were transformed from tube to tube with a interim shaking step.
</LI></UL><UL><LI> This suspension was transferred into the tube with the scratched-out pellet.
</LI></UL><UL><LI> Centrifugation at 10500 rpm at 4 °C (SLA-3000 rotor, Thermo Scientific) for 10 min.
</LI></UL><UL><LI> Supernatant was discarded.
</LI></UL><UL><LI> Finally washed pellet was stored, wrapped in foil overnight for the methanolysis next day.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_10th"><B>Tuesday, July 10th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_ADH1-P_and_gelelectrophoresis">Analytical digestion of ADH1-P and gelelectrophoresis</SPAN></H3><P><B> Investigator:</B> Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>8 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,8 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>59,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=70µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Extraction_of_ADH1-P.2C_TEF2-T_with_plasmid">Extraction of ADH1-P, TEF2-T with plasmid</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Plasmid-DNA was extracted according to Quiaprep Plasmid extraction kit
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Inoculation_of_TEF1-P.2C_PGK-P.2C_Cyc-T">Inoculation of TEF1-P, PGK-P, Cyc-T</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Transfomed E. coli cells from iGEM were inoculated onto Amp-LB-Plates in case of PGK1-P and Cyc1-T
</LI><LI> E. coli with TEF1-T were inoculated onto psb1c3
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Phycocyanobilin_.28PCB.29_extraction_from_dried_Spirulina_platensis_powder_.28part_2.2F4.29"> Phycocyanobilin (PCB) extraction from dried <I>Spirulina platensis</I> powder (part 2/4) </SPAN></H3><P><B>Investigator: Jeff, Alois, Martin</B></P><P><B>Aim of the experiment:</B> Phycocyanobilin (PCB) is a cofactor neeeded for the funtion of phytochrome B. Phycocyanobilin is covalently bound to Cys457 of phytochrome B. <I>Saccharomyces cerevisiae</I> does not contain endogenous PCB. For proof of concept PCB should be added to the medium. In the following experiment, PCB is extracted from dried <I>Spirulina platensis</I> powder.
</P><P><B>Operational sequence:</B></P><UL><LI> Washed pellet from the day before was resuspended in 500 ml MeOH.
</LI></UL><UL><LI> Heat suspension in a 500 ml flask in a water bath at 70 – 75 ºC with a condensing coil cooled with water for 5 – 8 hrs.
</LI></UL><UL><LI> After this, the suspension was transferred into a new centrifuge tube and centrifuged at 10500 rpm at 4 °C (SLA-3000 rotor, Thermo Scientific) for 20 min.
</LI></UL><UL><LI> The supernatant was decanted trough a filter paper into new centrifugation tube and stored, wrapped in a aluminium foil, at -20 °C.
</LI></UL><UL><LI> The pellets also was stored, wrapped in a aluminium foil, at -20 °C.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Plating_of_received_E..C2.A0coli_containing_biobricks"> Plating of received <I>E. coli</I> containing biobricks </SPAN></H3><P><B>Investigator: Jeff</B></P><P><B>Aim of the experiment:</B> The received biobricks were already transformed in <I>E. coli</I> and were in an agar stabs. These <I>E. coli</I> cells were transferred with an inoculation loop on antibiotic selection plates and were incubated over night.
</P><P><B>Operational sequence:</B></P><UL><LI> Bacterias containing plasmids with biobricks were transferred with a sterile inoculation loop on antibiotic plates and were incubated at 37 °C overnight.
</LI></UL><UL><LI> The biobricks were: <B>BBa_K207001</B> in pSB1A2, <B>BBa_K243006</B> in BBa_K157000, <B>BBa_K300001</B> in K300007 (part for other subproject), <B>BBa_K268000</B> in pSB6A0 (part for other subproject), <B>BBa_K365005</B> RFC25 in pSB1C3, <B>BBa_K365000</B> in pSB1C3, <B>BBa_K207000</B> in pSB3K3, <B>BBa_K165031</B> in pSB1AK3
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_11th"><B>Wednesday, July 11th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_ADH1-Terminator_and_TEF2-P">Analytical digestion and gelelectrophoresis of ADH1-Terminator and TEF2-P</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Transformants of ADH1-T showed no Plasmid
</LI><LI> TEF2 Promoter-colonies were all positive
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_of_inoculated_CYC1-T.2C_TEF1-P.2C_PGK1-P_for_overnight_cultures"> Picking of inoculated CYC1-T, TEF1-P, PGK1-P for overnight cultures</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> To 5 µl LB-Medium,5 µl of 1000x Chloramphenicol were added
</LI><LI> 4 colonies from the plate with TEF1-P, 4 colonies of Cyc-T were picked and 4 colonies of PGK1-P were picked
</LI><LI> Each colony was transferred into 5 ml LB-Medium with 1x Chloramphenicol (TEF1-P)or Amp (PGK1-P, Cyc1-T)
</LI><LI> Incubation over night at 37°C in the 180rpm cell-culture shaker.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Phycocyanobilin_.28PCB.29_extraction_from_dried_Spirulina_platensis_powder_.28part_3.2F4.29"> Phycocyanobilin (PCB) extraction from dried <I>Spirulina platensis</I> powder (part 3/4) </SPAN></H3><P><B>Investigator: Jeff, Alois, Martin</B></P><P><B>Aim of the experiment:</B> Phycocyanobilin (PCB) is a cofactor neeeded for the funtion of phytochrome B. Phycocyanobilin is covalently bound to Cys457 of phytochrome B. <I>Saccharomyces cerevisiae</I> does not contain endogenous PCB. For proof of concept PCB should be added to the medium. In the following experiment, PCB is extracted from dried <I>Spirulina platensis</I> powder.
</P><P><B>Operational sequence:</B></P><UL><LI> The pellet after the first methanolysis of the day before was undergone a second methanolysis step to ensure high efficiacy of PCB extraction. Operational sequence was performed like the first methanolysis including the centrifugation and filtering step. The pellet after the second methanolysis should be more colorless and the filtered supernatant was freezed, like the one from the first methanolysis, at -20 °C, wrapped in aluminium foil
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_pGADT7_and_pGBKT7_plasmid_for_Yeast-two-hybrid_.28Y2H.29"> Transformation of <I>E. coli</I> XL1-Blue with pGADT7 and pGBKT7 plasmid for Yeast-two-hybrid (Y2H) </SPAN></H3><P><B>Investigator: Jeff</B></P><P><B>Aim of the experiment:</B> pGADT7 and pGBKT7 are plasmids containing the transcriptional activation domain or the DNA binding domain of the transcription activator Gal4. pGADT7 contains the transcriptional activation domain of gal4; we want to clone the the first 100 residues of Pif3 (BBa_K365000) into this plasmid. As a result we have a fusion contruct of Gal4 and Pif3, which is nescassary for the light-switchable promoter system. pGBKT7 is for backup, if the ordered biobricks are not working.
</P><P><B>Operational sequence:</B></P><UL><LI><I>E. coli</I> XL1-Blue are transformed with pGADT7 and pGBKT7 seperately after standard protocol of our lab.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Introducing_new_Saccharomyces_cerevisiae_strain_.28Y190_strain.29_from_Schwab_lab_for_Y2H"> Introducing new <I>Saccharomyces cerevisiae</I> strain (Y190 strain) from Schwab lab for Y2H </SPAN></H3><P><B>Investigator: Jeff</B></P><P><B>Aim of the experiment:</B> The Y190 <I>Saccharomyces cerevisiae</I> is a special strain for Yeast-two-hybrid. This strain carries a Gal4 and Gal80 deletion to higher the signal/noise-ratio of protein-protein interactions. Reporter for protein-protein interactions are HIS3, lacZ and MEL1 and are encoded in the genomic DNA. Transformation markers are trp1, leu2 and cyhR2 and are encoded on the transformation plasmids.
</P><P><B>Operational sequence:</B></P><UL><LI> The freshly plated Y190 cells are transferred with a inoculation loop from the original plate on a new YPD agar plate.
</LI></UL><UL><LI> After 2 days the the plate was put in 4 °C.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Repetition_of_PCR_of_PAL_2"> Repetition of PCR of PAL</SPAN></H3><P><B>Investigator: Katrin, Daniela</B></P><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>dNTP's (each 2.5 mM)
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II (2.5 U/µL)
</TD></TR><TR><TD>5 µl
</TD><TD>1:10 dilution of used forward primers (10µM)
</TD></TR><TR><TD>5 µl
</TD><TD>1:10 dilution of used reversed primers (10µM)
</TD></TR><TR><TD>1 µl
</TD><TD>DNA (pKS2µHyg-PAL-4CL-CHS 50 ng/µL)
</TD></TR><TR><TD>29.5 µL
</TD><TD>bidest. sterile Water
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 2 min (and adding Pfu Ultra after 2 min)
</TD></TR><TR><TD>2
</TD><TD>30
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>52°C
</TD><TD> 1 min
</TD></TR><TR><TD>72°C
</TD><TD> 2.5 min
</TD></TR><TR><TD>3
</TD><TD>72°C
</TD><TD> 5 min
</TD></TR></TBODY></TABLE><P><B>PCR purification</B></P><UL><LI>Purification was done using QIAquick PCR Purification Kit (250)
</LI></UL><P><B>Analytical gelelektrophoresis of PCR-Products of PAL</B></P><P>-&gt; PCR was not successful, no band at 2,1 kb
(picture follows)
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparative_Digest_of_pYES2_iGEM"> Preparative Digest of pYES2_iGEM</SPAN></H3><P><B>Investigator: Katrin, Daniela</B></P><P><B>Digestion of P50 with Xbal and NgOMIV</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>P50 (concentration: 264.5 ng/µl)
</TD></TR><TR><TD>2.5 µl
</TD><TD>NEB4
</TD></TR><TR><TD>2.5 µl
</TD><TD>10x BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NgOMIV (10U/µl)
</TD></TR><TR><TD>7 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C, 3 h
</P><P><B>Digestion of P50 with Xbal and PstI</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>P50 (concentration: 264.5 ng/µl)
</TD></TR><TR><TD>5 µl
</TD><TD>Tango buffer
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI
</TD></TR><TR><TD>3 µl
</TD><TD>PstI
</TD></TR><TR><TD>7.5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C, 3 h
</P><P><B>DNA preparative gel electrophoresis</B></P><UL><LI>gel: 1% with LMP-agarose
</LI><LI>band 1: P50 digested with XbaI and PstI
</LI><LI>band 2: P50 digested with XbaI and NgOMIV
</LI><LI>70 V, 90 min
</LI></UL><P><B>Gelextration</B></P><UL><LI>cut the bands 
</LI><LI>QIAquick Gel Extractrion Kit was used
</LI><LI> step 6 was left out
</LI><LI> step 9: 30µl buffer, 4 min incubation
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Ligation_of_digested_P50_with_digested_PCR-products_of_PCR_15-20_.284CL.2C_CHS_and_OMT.29"> Ligation of digested P50 with digested PCR-products of PCR 15-20 (4CL, CHS and OMT)</SPAN></H3><P><B>Investigator: Katrin, Daniela</B></P><P>Concentration (Nano Drop: 
4CL+ (PCR 15) = 40.3 ng/µl
4CL- (PCR 16) = 37.3 ng/µl
CHS+ (PCR 17) = 46.1 ng/µl
CHS- (PCR 18) = 51.2 ng/µl
OMT+ (PCR 19) = 22.3 ng/µl
OMT- (PCR 20) = 16.7 ng/µl
P50 digested with XbaI and PstI = 23.9 ng/µl (the digested Plasmid has the number P71)
P50 digested with XbaI and NgOMIV = 28.4 ng/µl (the digested Plasmid has the number P72)</P><UL><LI>required volumes were calculated using Lab Tools
</LI></UL><P>4CL+ (PCR 15) with P50 digested with XbaI and PstI
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5.27 µl
</TD><TD>P50 digested
</TD></TR><TR><TD>2.73 µl
</TD><TD>4CL+ (PCR 15)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>4CL- (PCR 16) with P50 digested with XbaI and PstI
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5.13 µl
</TD><TD>P50 digested
</TD></TR><TR><TD>2.87 µl
</TD><TD>4CL- (PCR 16)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>CHS+ (PCR 17) with P50 digested with XbaI and NgOMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5.84 µl
</TD><TD>P50 digested
</TD></TR><TR><TD>2.16 µl
</TD><TD>CHS+ (PCR 17)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>CHS- (PCR 18) with P50 digested with XbaI and NgOMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>P50 digested
</TD></TR><TR><TD>2 µl
</TD><TD>CHS- (PCR 18)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>OMT+ (PCR 19) with P50 digested with XbaI and NgOMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.45 µl
</TD><TD>P50 digested
</TD></TR><TR><TD>3.55 µl
</TD><TD>OMT+ (PCR 19)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>OMT- (PCR 20) with P50 digested with XbaI and NgOMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.87 µl
</TD><TD>P50 digested
</TD></TR><TR><TD>4,13 µl
</TD><TD>OMT- (PCR 20)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>Negative control
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>P50 digested with XbaI and PstI or NgOMIV
</TD></TR><TR><TD>3 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA-ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><UL><LI>water bath 16 °C 
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_12th"><B>Thursday, July 12th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_and_analytical_digestion_of_CYC1-terminator.2C_TEF1-promoter.2C_and_PGK1-Promoter_from_pSB1C3_and_pSB1A2"> Gelextraction and analytical digestion of CYC1-terminator, TEF1-promoter, and PGK1-Promoter from pSB1C3 and pSB1A2</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI>Genes from overnight cultures were extracted, using the Quiaprep gene extraction Kit. Extracted Plasmids then were analytically digested with XbaI and PstI (Fermentas).
</LI></UL><P><B>Analytical digestion XbaI, PstI</B></P><TABLE border="1"><TBODY><TR><TD><B>Chemical</B></TD><TD><B>Volume</B></TD></TR><TR><TD>XbaI
</TD><TD>2,5 µl
</TD></TR><TR><TD>PstI
</TD><TD>2,5 µl
</TD></TR><TR><TD>10xBuffer Tango
</TD><TD>2 µl
</TD></TR><TR><TD>Plasmid-DNA
</TD><TD>2,5 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>165,5 µl
</TD></TR><TR><TD>=175µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Analytical gel was run (1% Agarose).
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Phycocyanobilin_.28PCB.29_extraction_from_dried_Spirulina_platensis_powder_.28part_4.2F4.29"> Phycocyanobilin (PCB) extraction from dried <I>Spirulina platensis</I> powder (part 4/4) </SPAN></H3><P><B>Investigator: Martin, Jeff, Alois</B></P><P><B>Aim of the experiment:</B> Phycocyanobilin (PCB) is a cofactor neeeded for the funtion of phytochrome B. Phycocyanobilin is covalently bound to Cys457 of phytochrome B. <I>Saccharomyces cerevisiae</I> does not contain endogenous PCB. For proof of concept PCB should be added to the medium. In the following experiment, PCB is extracted from dried <I>Spirulina platensis</I> powder.
</P><P><B>Operational sequence:</B></P><UL><LI> The supernatants (~1l in total) of the two previous experiments were pooled and put into a rotary evaporater in order to acquire a concentrate of approximately 100ml.
</LI><LI> The settings of the rotary evaporater: ~160 millibars, waterbath temperature of 25-30°C
</LI><LI> Furthermore, there were measures to be taken to protect the solution from direct sunlight: blinds down, aluminum foil wrapped around the waterbath 
</LI></UL><UL><LI> Afterwards we transfered the Methanol solution into a separating funnel and added another 100ml of aqua dest..
</LI><LI> By means of adding chloroform we created two phases. The lower (chloroform) phase with the solved Phycocyanobilin was separated and put into another rotary evaporater flask. This step was repeated three times in order to get all the Phycocanobilin into the next step
</LI></UL><UL><LI> Then the chloroform was completely removed in the rotary evaporater (same settings as before), the pure (?) Phycocyanobilin in the flask was solved in 60 ml DMSO, transfered into a new flask and frozen at -20 °C.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_transformated_.28pGADT7_and_pGBKT7.29_E.coli_cells_on_antibiotic_selection_plates"> Picking of transformated (pGADT7 and pGBKT7) <I>E.coli cells</I> on antibiotic selection plates  </SPAN></H3><P><B>Investigator: Jeff</B></P><P><B>Aim of the experiment:</B> pGADT7 and pGBKT7 are plasmids containing the transcriptional activation domain or the DNA binding domain of the transcription activator Gal4. pGADT7 contains the transcriptional activation domain of gal4; we want to clone the the first 100 residues of Pif3 (BBa_K365000) into this plasmid. As a result we have a fusion contruct of Gal4 and Pif3, which is nescassary for the light-switchable promoter system. pGBKT7 is for backup, if the ordered biobricks are not working. 
</P><P><B>Operational sequence:</B></P><UL><LI> A <I>E. coli</I> colony was picked for each plasmid and was transferred in a tube containing 4 ml of LB-medium containing antibiotics (Amp for pGADT7 and Kan for pGBKT7).
</LI></UL><UL><LI> Overnight culture at 37 °C in a cell-culture shaker.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_transformated_E.coli_from_overnight_culture_.288_plasmids_containing_biobricks.29"> Miniprep of transformated <I>E.coli</I> from overnight culture (8 plasmids containing biobricks)  </SPAN></H3><P><B>Investigator: Andrea</B></P><P><B>Aim of the experiment: Miniprep of transformated E. coli from overnight culture to get the plasmids with biobricks</B></P><P><B>NanoDrop Measure</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>Concentration</B></TD></TR><TR><TD>P73
</TD><TD>41.6 ng/µl
</TD></TR><TR><TD>P74
</TD><TD>85.2 ng/µl
</TD></TR><TR><TD>P75
</TD><TD>72.4 ng/µl
</TD></TR><TR><TD>P76
</TD><TD>254.3 ng/µl
</TD></TR><TR><TD>P77
</TD><TD>74.4 ng/µl
</TD></TR><TR><TD>P78
</TD><TD>50.0 ng/µl
</TD></TR><TR><TD>P79
</TD><TD>46.2 ng/µl
</TD></TR><TR><TD>P80
</TD><TD>116.7 ng/µl
</TD></TR><TR><TD>P81
</TD><TD>77.1 ng/µl
</TD></TR><TR><TD>P82
</TD><TD>65.5 ng/µl
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_biobricks_.288_plasmids_containing_biobricks.29"> Analytical digestion and gelelectrophoresis of biobricks (8 plasmids containing biobricks) </SPAN></H3><P><B>Investigator: Jeff</B></P><P><B>Aim of the experiment</B>: Checking whether the biobricks are part of the plasmid-backbone.
</P><P><B>Operational sequence:</B></P><UL><LI> Reaction batch for each plasmid:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD>Tango Buffer 10x
</TD><TD>4 µl
</TD></TR><TR><TD>XbaI (Fermentas)
</TD><TD>0.25 µl
</TD></TR><TR><TD>PstI (Fermentas)
</TD><TD>0.5 µl
</TD></TR><TR><TD>Plasmid DNA
</TD><TD>2.5 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>12.75 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD><B>20 µl</B></TD></TR></TBODY></TABLE><UL><LI> Incubation at 37 °C for 1 h 45 min.
</LI></UL><UL><LI> Analytical gelelectrophoresis at 90 V for 1 h.
</LI></UL><UL><LI> Order of gel-pockets:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder
</TD><TD>P82
</TD><TD>P81
</TD><TD>P80
</TD><TD>P79
</TD><TD>P76
</TD><TD>P75
</TD><TD>P74
</TD><TD>P73
</TD><TD>1 kbp ladder
</TD></TR><TR><TD>OK
</TD><TD>BAD
</TD><TD>OK
</TD><TD>OK
</TD><TD>OK
</TD><TD>OK
</TD><TD>OK
</TD><TD>BAD
</TD></TR></TBODY></TABLE><UL><LI> P73 and P81, both parts from Havard university are bad. To exclude errors, one should pick another colony for each plasmid and do again the analytical digestion and gelelectrophoresis after the miniprep.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Transformation_of_Ligationproducts_of_pTUM104_.2B_OMT.2C_4Cl_and_CHS_in_E.coli"> Transformation of Ligationproducts of pTUM104 + OMT, 4Cl and CHS in <I>E.coli</I></SPAN></H3><P><B>Investigator: Mary</B></P><UL><LI> adding 5µl ligation product in 100µl competent XL blue <I>E.coli</I> cells
</LI><LI> incubation 30 min on ice
</LI><LI> 5 min at 37°C
</LI><LI> adding cells in 1 ml LB (Withouth antibiotica) and incubate at 37°C, 30 min, 180 rpm
</LI><LI> plate on agar with Ampicillin over night
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Repetition_of_PCR_of_PAL_3"> Repetition of PCR of PAL</SPAN></H3><P><B>Investigator: Daniela</B></P><P>Aim of the experiment: Repetition of PCR of PAL (so far not successful) with the use of different polymerase and try of 3 temperatures
</P><P>only with PAL+, 3 different temperatures (see cycling parameters) and one batch at 52.5 °C with DMSO
</P><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x Herculase II reaction buffer
</TD></TR><TR><TD>0,5 µl
</TD><TD>dNTP Mix (each dNTP 2.5 mM)
</TD></TR><TR><TD>1 µl
</TD><TD>Herculase II fusion DNA Polymerase
</TD></TR><TR><TD>1,25 µl
</TD><TD>1:10 dilution of used forward primers (O22)
</TD></TR><TR><TD>1,25 µl
</TD><TD>1:10 dilution of used reversed primers (O59)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of DNA (P19=pKS2µHyg-PAL-4CL-CHS 50 ng/µL) -&gt; 5 ng/µL
</TD></TR><TR><TD>35 µL
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Reaction batch with DMSO</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x Herculase II reaction buffer
</TD></TR><TR><TD>0,5 µl
</TD><TD>dNTP Mix (each dNTP 2.5 mM)
</TD></TR><TR><TD>1 µl
</TD><TD>Herculase II fusion DNA Polymerase
</TD></TR><TR><TD>1,25 µl
</TD><TD>1:10 dilution of used forward primers (O22)
</TD></TR><TR><TD>1,25 µl
</TD><TD>1:10 dilution of used reversed primers (O59)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of DNA (pKS2µHyg-PAL-4CL-CHS 50 ng/µL) -&gt; 5 ng/µL
</TD></TR><TR><TD>1,5 µl
</TD><TD>DMSO (3% des Ansatzes)
</TD></TR><TR><TD>33.5 µL
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 2 min
</TD></TR><TR><TD>2
</TD><TD>30
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>52.5°C
</TD><TD> 30 sec
</TD></TR><TR><TD>72°C
</TD><TD> 4 min
</TD></TR><TR><TD>3
</TD><TD>72°C
</TD><TD> 3 min
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 2 min
</TD></TR><TR><TD>2
</TD><TD>30
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>45°C
</TD><TD> 30 sec
</TD></TR><TR><TD>72°C
</TD><TD> 4 min
</TD></TR><TR><TD>3
</TD><TD>72°C
</TD><TD> 3 min
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 2 min
</TD></TR><TR><TD>2
</TD><TD>30
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>56.2°C
</TD><TD> 30 sec
</TD></TR><TR><TD>72°C
</TD><TD> 4 min
</TD></TR><TR><TD>3
</TD><TD>72°C
</TD><TD> 3 min
</TD></TR></TBODY></TABLE><P>Analytical gel electrophoresis
</P><UL><LI>no bands at all (picture follows) 
</LI><LI>possibly due to low concentration of P19 (5 ng/µl)
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Ligation_of_P93_.28pSB1C3_digested_with_XbaI_and_AgeI.29_with_digested_PCR-products_of_PCR_17-20_.28CHS_and_OMT.29"> Ligation of P93 (pSB1C3 digested with XbaI and AgeI) with digested PCR-products of PCR 17-20 (CHS and OMT)</SPAN></H3><P><B>Investigator: Daniela</B></P><P>Concentration (Nano Drop: 
CHS+ (PCR 17) = 46.1 ng/µl
CHS- (PCR 18) = 51.2 ng/µl
OMT+ (PCR 19) = 22.3 ng/µl
OMT- (PCR 20) = 16.7 ng/µl
P93 (digested with XbaI and AgeI) = 4.8 ng/µl</P><P>
CHS+ (PCR17) with P93 digested with XbaI and AgeI
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>7.04 µl
</TD><TD>P93
</TD></TR><TR><TD>0.96 µl
</TD><TD>CHS+ (PCR 17)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>CHS- (PCR 18) with P93 digested with XbaI and AgeI
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>7.13 µl
</TD><TD>P93
</TD></TR><TR><TD>0.87 µl
</TD><TD>CHS- (PCR18)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>OMT+ (PCR19) with P93 digested with XbaI and AgeI
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6.19 µl
</TD><TD>P93
</TD></TR><TR><TD>1.81 µl
</TD><TD>OMT+ (PCR19)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>OMT- (PCR20) with P50 digested with XbaI and AgeI
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5.75 µl
</TD><TD>P93
</TD></TR><TR><TD>2.25 µl
</TD><TD>OMT- (PCR20)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>Negative control
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>P93
</TD></TR><TR><TD>3 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA-ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><UL><LI>water bath 16 °C 
</LI><LI>products were named P94-P97
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_digest_of_PCR1-PCR7_and_P50">Preparative digest of PCR1-PCR7 and P50</SPAN></H3><P><B>Investigator: Andrea</B></P><P><B>Digestion of P50 with XbaI and NgoMIV</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P50
</TD></TR><TR><TD>4 µl
</TD><TD>NEB Buffer
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (10 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>27.4 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Digestion of PCR1-PCR7 with XbaI and AgeI</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR-Product
</TD></TR><TR><TD>5 µl
</TD><TD>NEB Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (10 U/µl)
</TD></TR><TR><TD>32.5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C, 3 h
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_gel_electrophoresis_of_digested_plasmids_PCR1-PCR7_and_PCR9"> Preparative gel electrophoresis of digested plasmids PCR1-PCR7 and PCR9 </SPAN></H3><P><B>Investigator: Andrea</B></P><P><B>Aim of the experiment:</B> Analytical gel electrophoresis of products from restriction digest of plasmids PCR1 - PCR7 and PCR9
</P><P>Limonensynthase: 1600 bp
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_digested_PCR1-PCR7_.28digested_with_XbaI_and_AgeI.29_with_pSB1C3_.28digested_with_XbaI_and_AgeI.29_and_with_pYESnew_.28digested_with_XbaI_and_NgoMIV.29"> Ligation of digested PCR1-PCR7 (digested with XbaI and AgeI) with pSB1C3 (digested with XbaI and AgeI) and with pYESnew (digested with XbaI and NgoMIV)</SPAN></H3><P><B>Investigator: Andrea</B></P><P>Concentration (Nano Drop: 
LIMS Citrus (PCR 1) = 13.4 ng/µl
LIMS Citrus (PCR 2) = 10.3 ng/µl
LIMS Lavendula (PCR 3) = 2.2 ng/µl
LIMS Lavendula (PCR 3) = 9.9 ng/µl
LIMS Lavendula (PCR 3) = 9.6 ng/µl
LIMS Lavendula (PCR 3) = 9.8 ng/µl
LIMS Lavendula (PCR 3) = 12.3 ng/µl
P50 (digested with XbaI and NgoMIV) = 10.3 ng/µl
P93 (digested with XbaI and AgeI) = 4 ng/µl</P><P>
PCR1 with P50 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.89µl
</TD><TD>P50
</TD></TR><TR><TD>3.11 µl
</TD><TD>PCR 1
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR2 with P50 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.38 µl
</TD><TD>P50
</TD></TR><TR><TD>3.62 µl
</TD><TD>PCR2
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR3 with P50 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.64 µl
</TD><TD>P50
</TD></TR><TR><TD>6.36 µl
</TD><TD>PCR3
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR4 with P50 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.3 µl
</TD><TD>P50
</TD></TR><TR><TD>3.7 µl
</TD><TD>PCR4
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR5 with P50 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.26 µl
</TD><TD>P50
</TD></TR><TR><TD>3.74 µl
</TD><TD>PCR5
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR6 with P50 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.26 µl
</TD><TD>P50
</TD></TR><TR><TD>3.74 µl
</TD><TD>PCR6
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR7 with P50 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>7.83 µl
</TD><TD>P50
</TD></TR><TR><TD>3.27 µl
</TD><TD>PCR7
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>Negative control
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>P50
</TD></TR><TR><TD>3 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA-ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>
PCR1 with P93 digested with XbaI and AgeI
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.89 µl
</TD><TD>P93
</TD></TR><TR><TD>3.11 µl
</TD><TD>PCR1
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR2 with P93 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.38 µl
</TD><TD>P93
</TD></TR><TR><TD>3.62 µl
</TD><TD>PCR2
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR3 with P93 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.52 µl
</TD><TD>P93
</TD></TR><TR><TD>6.48 µl
</TD><TD>PCR3
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR4 with P93 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.3 µl
</TD><TD>P93
</TD></TR><TR><TD>3.7 µl
</TD><TD>PCR4
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR5 with P93 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.26 µl
</TD><TD>P93
</TD></TR><TR><TD>3.74 µl
</TD><TD>PCR5
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR6 with P93 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.26 µl
</TD><TD>P93
</TD></TR><TR><TD>3.74 µl
</TD><TD>PCR6
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR7 with P93 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.72 µl
</TD><TD>P93
</TD></TR><TR><TD>3.28 µl
</TD><TD>PCR7
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><UL><LI>water bath 16 °C 
</LI><LI>products were named P100-P115
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_13th"><B>Friday, July 13th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_ADH1-T"> Transformation of ADH1-T </SPAN></H3><P><B>Procedure:</B></P><P><B>Investigator</B>: Georg
</P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of ADH-T plasmid was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on ampicillin plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin plates.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_of_P73.2C_P80_and_P81_to_add_RFC25_pre-_and_suffix"> PCR of P73, P80 and P81 to add RFC25 pre- and suffix </SPAN></H3><P><B>Investigator:</B> Jeff, Saskia
</P><P><B>Aim of the experiment:</B> Introducing RFC25 pre- and suffix into parts of P73, P80 and P81.
</P><P><B>Operational sequence:</B></P><UL><LI><B>PCR reaction mixture</B></LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer (For P73: O46 (TUM12-PhyBGal4bd-fw); for P80: O31 (TUM12-LSPS-fw); for P81: O50 (TUM12-Phyb-fw))
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer (For P73: O47 (TUM12-PhyBGal4bd-rv); for P80: O32 (TUM12-LSPS-rv ); for P81: O51 (TUM12-Phyb-rv))
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (P80 or P73 or P81)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ELGA Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The <B>gradient</B> PCR program was performed after following scheme with following conditions (Tm=58 ΔG=5 °C; P73 in row 9(=61.1 °C); P80 in row 1 (=53.0 °C); P81 in row 7(=58.4 °C):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>Tm=58 °C; ΔG=5 °C
</TD><TD>150 s
</TD></TR><TR><TD>68 °C
</TD><TD>60 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>overnight
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep.2C_analcytical_digestion_and_gelelectrophoresis_of_pGADT7_and_pGBKT7"> Miniprep, analcytical digestion and gelelectrophoresis of pGADT7 and pGBKT7 </SPAN></H3><P><B>Investigator:</B> Jeff, Saskia
</P><P><B>Aim of the experiment:</B>Miniprep, analcytical digestion and gelelectrophoresis of pGADT7 (tube P98, EcoRI and BamHI)and pGBKT7 (tube P99, BamHI and PstI)
</P><P><B>Operational sequence:</B></P><UL><LI> Operated after standard protocol of the lab for analytical digestion and gelelectrophoresis.
</LI></UL><UL><LI> Gel <B>OK</B>, like expected
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophoresis_of_P79_and_O56"> Preperative digestion and gelelectrophoresis of P79 and O56 </SPAN></H3><P><B>Investigator:</B> Jeff, Saskia, Georg
</P><P><B>Aim of the experiment:</B> Preperativ gelelectrophoresis and gel of digested PCR product of LexA (NgoMIV+PstI) and pSB1C3 containing the RFC25 compatible 20aa linker with RFC25 pre- and suffix (AgeI and PstI).
</P><P><B>Operational sequence:</B></P><UL><LI> Operated after standard protocol of the lab for preperative digestion and gelelectrophoresis.
</LI></UL><UL><LI> Gel <B>OK</B>, like expected.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparative_digest_of_P19">Preparative digest of P19</SPAN></H3><P><B>Investigator:Daniela, Ingmar</B></P><P><B>Digestion of P19 with ApaI</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P19 (concentration: 53.8 ng/µl)
</TD></TR><TR><TD>4 µl
</TD><TD>Buffer B
</TD></TR><TR><TD>2 µl
</TD><TD>ApaI (10 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C, 3 h
</P><P><B>DNA preparative gel electrophoresis</B></P><UL><LI>gel: 1% with LMP-agarose
</LI><LI>band 1: P50 digested with XbaI and PstI
</LI><LI>band 2: P50 digested with XbaI and NgOMIV
</LI><LI>70 V, 90 min
</LI></UL><P><B>Gelextration</B></P><UL><LI>cut the bands 
</LI><LI>QIAquick Gel Extractrion Kit was used
</LI><LI> step 6 was left out
</LI><LI> step 9: 30µl buffer, 4 min incubation
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Gradient_PCR_of_PAL_to_optimize_the_primer_annealing_temperature">  Gradient PCR of PAL to optimize the primer annealing temperature  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: As all PCR experiments to amplify the PAL gene contained in P19 failed, we decided to run a gradient PCR to optimize the primer annealing temperature.
</P><P><B>PCR</B><B>Reaction batch </B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>Plasmid P19 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O15 (10 pmol/µL)
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O59 (10 pmol/µL)
</TD></TR><TR><TD>0.25 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.25 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>29
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>gadient 33-47.5 °C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 3 min
</TD></TR><TR><TD>3
</TD><TD>1
</TD><TD>68°C
</TD><TD>5 min
</TD></TR><TR><TD>4
</TD><TD>1
</TD><TD>4°C
</TD><TD>infinity
</TD></TR></TBODY></TABLE><UL><LI>Verification of the PCR by agarose gel electrophoresis: 
</LI></UL><P>10 µl of each PCR tube was mixed with 2 µl 6x DNA loading buffer and loaded into the gel. The separation process lasted 1h at 90 V in a 1% agarose gel.
</P><UL><LI>A bond at the expected length of 2160 bp appears at 47 °C. Therefore a PCR using this primer annealing temperature will done on sunday.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Transformation_of_P94_-_P97_.28ligationproducts_of_pSB1C3_with_CHS_and_OMT_respectively.29_in_E.coli">Transformation of P94 - P97 (ligationproducts of pSB1C3 with CHS and OMT respectively) in <I>E.coli</I></SPAN></H3><P>Investigator: Daniela
</P><UL><LI> adding 5µl ligation product (P94-P97) in 100µl competent XL blue <I>E.coli</I> cells
</LI><LI> incubation 30 min on ice
</LI><LI> 5 min at 37°C
</LI><LI> adding cells in 1 ml LB (Withouth antibiotica) and incubate at 37°C, 30 min, 180 rpm
</LI><LI> plate on agar with Ampicillin over night
</LI></UL><P>wrong antibiotica was used!!! Repetition follows!!!
Nevertheless, some colonies could be observed.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_LIMS_in_pYESnew_and_pSB1C3">Ligation of LIMS in pYESnew and pSB1C3</SPAN></H3><P><B>Investigator:Andrea</B></P><P><B>Transformation</B></P><UL><LI> adding 5µl ligation product (PCR1-PCR7) in 100µl competent XL blue <I>E.coli</I> cells
</LI><LI> incubation 30 min on ice
</LI><LI> 5 min at 37°C
</LI><LI> adding cells in 1 ml LB (without antibiotica) and incubate at 37°C, 30 min, 180 rpm
</LI><LI> plate on agar with Ampicillin (pYESnew) or Chloramphenicol (pSB1C3) over night
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Preparation_of_YPD_medium"> Preparation of YPD medium </SPAN></H3><P><B>Investigator: A lot of Alois, Martin</B></P><P><B>Manual for YPD production</B>
Yeast Extract Peptone Dextrose Medium (1 liter)
</P><UL><LI>1% yeast extract
</LI><LI>2% peptone
</LI><LI>2% dextrose (D-glucose)
</LI></UL><P>1. Dissolve the following in 1000 ml of water:
</P><UL><LI>10 g yeast extract
</LI><LI>20 g peptone
</LI><LI>20 g dextrose (see note below if making plates)
</LI></UL><P>2. Autoclave for 20 minutes on liquid cycle.
</P><P>3. Store medium at room temperature. The shelf life is approximately one to two months.
</P></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_July_15th"><B>Sunday, July 15th </B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_purification_of_PCR_products_of_P73.2C_P80_and_P81_and_analytical_gelelectrophoresis"> PCR purification of PCR products of P73, P80 and P81 and analytical gelelectrophoresis </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> PCR was performed to introduce restriction sites to the gene of interest. MfeI and BamHI for P73 and RFC25 pre- and suffix for P80 and P81. The aim is to contruct fusion proteins with the aid of these restriction sites.
</P><P><B>Operational sequence:</B></P><UL><LI> PCR cleaning with Qiagen PCR purification kit after manufacturer's protocol.
</LI></UL><UL><LI> Analytical gelelectrophoresis (1% agarose) at 90  for 60&amp;min.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="PCR_of_PAL_consless_using_the_optimized_primer_annealing_temperature_of_47_.C2.B0C">  PCR of PAL consless using the optimized primer annealing temperature of 47 °C </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Amplification of the PAL gene contained in P19.
</P><P><B>PCR</B><B>Reaction batch </B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid P19 template
</TD></TR><TR><TD>2.5 µl
</TD><TD>1:10 dilution of O15 (10 pmol/µL)
</TD></TR><TR><TD>2.5 µl
</TD><TD>1:10 dilution of O59 (10 pmol/µL)
</TD></TR><TR><TD>1.25 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>35 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1.25 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>29
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>47 °C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 3 min
</TD></TR><TR><TD>3
</TD><TD>1
</TD><TD>68°C
</TD><TD>5 min
</TD></TR><TR><TD>4
</TD><TD>1
</TD><TD>4°C
</TD><TD>infinity
</TD></TR></TBODY></TABLE><P>PCR successful - Band at about 2,1 kb :
</P></DIV></DIV><H1><SPAN class="mw-headline" id="Week_6">Week 6</SPAN></H1><DIV class="week" id="WWeek_6"><H2><SPAN class="mw-headline" id="Monday.2C_July_16th"><B>Monday, July 16th </B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Inoculation_of_ADH1-T_.28iGEM.29">Inoculation of ADH1-T (iGEM)</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Transformantion with ADH1-T didn't go well
</LI><LI> By iGEM transformed cells with ADH1-T were inoculated onto Amp-LB-Plates
</LI><LI> Cells were to grow at room-temperature over the weekend
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P86_and_P55"> Preparative digestion and gelelectrophoresis of P86 and P55 </SPAN></H3><P><B>Investigator:</B>Georg
<B>Aim of the experiment:</B> Digestion of CYC1-Terminator, ADH1-Promoter for ligation 
</P><UL><LI> Preparative digestion after manufacturer's advice (NEB) with 20 u PstI and 20 u XbaI in 1x Tango buffer with 25 µl DNA and 4 µl NEB4 10x buffer and 0,4 µl 100x BSA. Water was added to a volume of 40 µl. Restriction time was 3 hours at 37 °C; 3 hours.
</LI></UL><UL><LI> Preparative gelelectrophoresis after laboratory's standart protocol. (70 V, 90 min)
</LI></UL><UL><LI> Gel was stored at -20 °C.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophoresis_of_P98_and_PCR26"> Preperative digestion and gelelectrophoresis of P98 and PCR26 </SPAN></H3><P><B>Investigator:</B> Jeff, Georg
</P><P><B>Aim of the experiment:</B> Construction of Gal4AD-Pif3, a part of the light-switchable promoter system.
</P><P><B>Operational sequence:</B></P><UL><LI> Preperative digestion after manufacturer's (Fermentas) advise for double-digestion for EcoRI+BamHI and MfeI(MunI)+BamHI; EcoRI+BamHI: Buffer 2X Tango™, 2-fold excess of BamHI; MfeI(MunI)+BamHI: Buffer G.
</LI></UL><UL><LI> Preperative gelelectrohphoresis after laboratory's standard protocol. (70 V, 90 min)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Plating_of_received_E..C2.A0coli_containing_biobrick_BBa_K165055"> Plating of received <I>E. coli</I> containing biobrick BBa_K165055 </SPAN></H3><P><B>Investigator: Jeff</B></P><P><B>Aim of the experiment:</B> The received biobrick BBa_K165055 (LexA binding sites + mCYC + Kozak + YFPx2 + ADH1 terminator) were already transformed in <I>E. coli</I> and were in an agar stabs. These <I>E. coli</I> cells were transferred with an inoculation loop on antibiotic selection plates and were incubated over night.
</P><P><B>Operational sequence:</B></P><UL><LI> Bacterias containing plasmids with biobricks were transferred with a sterile inoculation loop on antibiotic plates and were incubated at 37 °C overnight.
</LI></UL><UL><LI> The biobricks was BBa_K165055 in BBa_J63009 plasmid (AmpR, low copy plasmid)
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Preparatory_culture_of_S..C2.A0cerevisiae"> Preparatory culture of <I>S. cerevisiae</I></SPAN></H3><P><B>Investigator:</B> Alois, Martin
</P><P><B>Aim of the experiment:</B> In order to have sufficient viable cells to inoculate an experimental culture in which we want to compare the growth rate of <I>s. cerevisiae</I> with a strain of brewing yeast (34/70 <I>s. pastorianus weihenstephan</I>) we aim to establish a preparatory over night culture.
</P><P><B>Operational sequence:</B>
A sample of -80°C stored <I>s. cerevisiae</I> (glycerol stock) is used to inoculate 20ml of YPD-medium. This is shaken overnight at 30°C.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_plasmids_containing_lavendula_limonene_synthase.2Fcitrus_limonene_synthase"> Miniprep of plasmids containing lavendula limonene synthase/citrus limonene synthase </SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim of the experiment:</B> Extract plasmids (pYESnew and pBS1C3) that contain limonene synthase/citrus limonene synthase.
</P><P>Miniprep with Qiagen Kit; Plasmids p116-p122. Restriction digest with Xba1 and Pst1. 
</P><P>Plasmid DNA concentrations:
</P><P>p116: 285 ng/µl
</P><P>p117: 337 ng/µl
</P><P>p118: 520 ng/µl
</P><P>p119: 370 ng/µl
</P><P>p120: 320 ng/µl
</P><P>p121: 320 ng/µl
</P><P>p122: 335 ng/µl
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Restriction_digest_of_p116-p122"> Restriction digest of p116-p122 </SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim of the experiment:</B> Analytical restriction digest of plasmids p116, p117, p118, p119, p120, p121, p122.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>DNA
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer 4
</TD></TR><TR><TD>0,25 µl
</TD><TD>Xba1
</TD></TR><TR><TD>0,25 µl
</TD><TD>Pst1-HF
</TD></TR><TR><TD>15,3 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C; 1,5 h
</P><P>Restriction digest with Xba1 and Pst1-HF. 
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_gel_electrophoresis_of_p116-p122"> Analytical gel electrophoresis of p116-p122 </SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim of the experiment:</B> Analytical gel electrophoresis of products from restriction digest of plasmids p116, p117, p118, p119, p120, p121, p122.
</P><P>p116: PCR1/pYES
</P><P>p117: PCR2/pYES
</P><P>p118: PCR4/pYES
</P><P>p119: PCR4/pSB1C3
</P><P>p120: PCR5/pYES
</P><P>p121: PCR6/pYES
</P><P>p122: PCR7/pYES
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Midiprep_of_pSB1C3_.28RFC25-compatible.29"> Midiprep of pSB1C3 (RFC25-compatible) </SPAN></H3><P><B>Investigator:</B>Mary
</P><P><B>Aim of the experiment:</B> Get a lot of pSB1C3 for further experiments
</P><P>Midipreparation of the pellet from over night cultures (<I>E.coli</I>, Transformed with pSB1C3 - RFC25; name in registry: K365005)
</P><P>was done with the Midiprep Kit from Qiagen
</P><P>Result was named as P123 and had the concentration: 469,5 ng/µl (100µl at all)
</P></DIV><DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="PCR-Products_of_PAL_.28consless.29:_digestion.2C_extraction_and_ligation"> PCR-Products of PAL (consless): digestion, extraction and ligation </SPAN></H3><P><B>investigator: </B> Daniela, Mary
</P><P><B>Aim of the experiment:</B> digestion of PCR-Products to gain the correct cutting sites for ligation in pYES2 and pSB1C3 afterwards
</P><P>Purification of PCR-Products with PCR-Purification Kit
</P><UL><LI>digestion with Xba1 and HF-Age1 (both NEB)
</LI></UL><TABLE class="wikitable" cellpadding="10" border="1px"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD> 25µl </TD><TD> PCR-product
</TD></TR><TR><TD> 5µl </TD><TD> Buffer NEB4
</TD></TR><TR><TD> 0.5µl </TD><TD> BSA
</TD></TR><TR><TD> 1µl </TD><TD> Xba1 (NEB; 20u/µl)
</TD></TR><TR><TD> 1µl </TD><TD> HF-Age1 (NEB; 20u/µl)
</TD></TR><TR><TD> 17.5µl </TD><TD> bidest. sterile H2O
</TD></TR></TBODY></TABLE><P>digestion took 3h at 37°C
</P><UL><LI>extraction from preparative gel:
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="PCR_of_PAL.2Bcons_using_the_optimized_primer_annealing_temperature_of_47_.C2.B0C">  PCR of PAL+cons using the optimized primer annealing temperature of 47 °C </SPAN></H3><P><B>Investigator: Daniela</B></P><P>Aim of the experiment: Amplification of the PAL cons gene contained in P19.
</P><P><B>PCR</B><B>Reaction batch </B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid P19 template
</TD></TR><TR><TD>2.5 µl
</TD><TD>1:10 dilution of O22 (10 pmol/µL)
</TD></TR><TR><TD>2.5 µl
</TD><TD>1:10 dilution of O59 (10 pmol/µL)
</TD></TR><TR><TD>1.25 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>35 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1.25 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>29
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>47 °C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 3 min
</TD></TR><TR><TD>3
</TD><TD>1
</TD><TD>68°C
</TD><TD>5 min
</TD></TR><TR><TD>4
</TD><TD>1
</TD><TD>4°C
</TD><TD>infinity
</TD></TR></TBODY></TABLE><P>PCR was succesful, band at 2,1kb:
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Transformation_of_P94_-_P97_.28ligationproducts_of_pSB1C3_with_CHS_and_OMT_respectively.29_in_E.coli_2">Transformation of P94 - P97 (ligationproducts of pSB1C3 with CHS and OMT respectively) in <I>E.coli</I></SPAN></H3><P>Investigator: Daniela
</P><UL><LI> adding 5µl ligation product (P94-P97) in 100µl competent XL blue E.coli cells
</LI><LI> incubation 30 min on ice
</LI><LI> 5 min at 37°C
</LI><LI> adding cells in 1 ml LB (without antibiotica) and incubate at 37°C, 45 min, 180 rpm
</LI><LI> plate on agar with chloramphenicol over night
</LI></UL><P>-&gt; were succesfull: some colonies were grown
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_17th"><B>Tuesday, July 17th </B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_of_ADH1-T_colonies">Picking of ADH1-T colonies</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> To 5 µl LB-Medium,5 µl of 1000x Amp were added
</LI><LI> 5 colonies from the plate with ADH1-P were picked
</LI><LI> Each colony was transferred into 5 ml LB-Medium with Chloramphenicol
</LI><LI> Incubation over night at 37°C in the 180rpm cell-culture shaker.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_transformed_E..C2.A0coli_containing_biobrick_BBa_K165055"> Picking of transformed <I>E. coli</I> containing biobrick BBa_K165055 </SPAN></H3></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparative_digest_of_P123_.28pSB1C3_RFC25.29">Preparative digest of P123 (pSB1C3 RFC25)</SPAN></H3><P><B>Investigator: Mary, Daniela</B></P><P><B>Digestion of P123 with Xbal/PstI and XbaI/AgeI</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>23.1 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>4 µl
</TD><TD>NEB4
</TD></TR><TR><TD>0.4 µl
</TD><TD>10x BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1.5 µl
</TD><TD>PstI (10U/µl)
</TD></TR><TR><TD>10 µl
</TD><TD>P123
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>23.35 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>4 µl
</TD><TD>NEB4
</TD></TR><TR><TD>0.4 µl
</TD><TD>10x BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1.25 µl
</TD><TD>AgeI (10U/µl)
</TD></TR><TR><TD>10 µl
</TD><TD>P123
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C, 3 h
</P><P><B>DNA preparative gel electrophoresis</B></P><UL><LI>gel: 1% with LMP-agarose
</LI><LI>band 1: P123 digested with XbaI and PstI 
</LI><LI>band 2: P123 digested with XbaI and AgeI
</LI><LI>70 V, 90 min
</LI></UL><P><B>Gelextration</B></P><UL><LI>cut the bands 
</LI><LI>QIAquick Gel Extractrion Kit was used
</LI><LI> step 6 was left out
</LI><LI> step 9: 30µl buffer, 4 min incubation
</LI></UL><P>Products were names as follows:
</P><UL><LI> P123 digested with XbaI and PstI: P132
</LI><LI> P123 digested with XbaI and AgeI: P133
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="PCR-Products_of_PAL_.28with_consensussequence.29:_digestion_and_extraction"> PCR-Products of PAL (with consensussequence): digestion and extraction </SPAN></H3><P><B>investigator: </B> Daniela, Mary
</P><P><B>Aim of the experiment:</B> digestion of PCR-Products to gain the correct cutting sites for ligation in pTUM104 and pSB1C3 afterwards
</P><P>Purification of PCR-Products with PCR-Purification Kit
</P><UL><LI>digestion with Xba1 and HF-Age1 (both NEB)
</LI></UL><TABLE class="wikitable" cellpadding="10" border="1px"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD> 25µl </TD><TD> PCR-product
</TD></TR><TR><TD> 5µl </TD><TD> Buffer NEB4
</TD></TR><TR><TD> 0.5µl </TD><TD> BSA
</TD></TR><TR><TD> 1µl </TD><TD> Xba1 (NEB; 20u/µl)
</TD></TR><TR><TD> 1µl </TD><TD> HF-Age1 (NEB; 20u/µl)
</TD></TR><TR><TD> 17.5µl </TD><TD> bidest. sterile H2O
</TD></TR></TBODY></TABLE><P>digestion took 3h at 37°C
</P><UL><LI>extraction from preparative gel:
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Picking_Clones_of_CHS_and_OMT_in_pSB1C3-RFC25"> Picking Clones of CHS and OMT in pSB1C3-RFC25 </SPAN></H3><P><B>investigator: </B> Daniela, Mary
</P><P><B>Aim of the experiment:</B> preculture over night for the miniprep next day
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Repetition_of_ligation_of_PCR_15-20_.284CL.2C_CHS_and_OMT.29_with_pTUM104_.28P71_and_P72_respectively.29"> Repetition of ligation of PCR 15-20 (4CL, CHS and OMT) with pTUM104 (P71 and P72 respectively)</SPAN></H3><P><B>Investigator: Mary, Daniela</B></P><P>Concentration (Nano Drop: 
4CL+ (PCR 15) = 40.3 ng/µl
4CL- (PCR 16) = 37.3 ng/µl
CHS+ (PCR 17) = 46.1 ng/µl
CHS- (PCR 18) = 51.2 ng/µl
OMT+ (PCR 19) = 22.3 ng/µl
OMT- (PCR 20) = 16.7 ng/µl
P50 digested with XbaI and PstI = 23.9 ng/µl (the digested Plasmid has the number P71)
P50 digested with XbaI and NgOMIV = 28.4 ng/µl (the digested Plasmid has the number P72)</P><UL><LI>required volumes were calculated using Lab Tools
</LI></UL><P>4CL+ (PCR 15) with P71
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5.27 µl
</TD><TD>P71
</TD></TR><TR><TD>2.73 µl
</TD><TD>4CL+ (PCR 15)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>4CL- (PCR 16) with P71
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5.13 µl
</TD><TD>P71
</TD></TR><TR><TD>2.87 µl
</TD><TD>4CL- (PCR 16)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>CHS+ (PCR 17) with P72
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5.84 µl
</TD><TD>P72
</TD></TR><TR><TD>2.16 µl
</TD><TD>CHS+ (PCR 17)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>CHS- (PCR 18) with P72
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>P72
</TD></TR><TR><TD>2 µl
</TD><TD>CHS- (PCR 18)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>OMT+ (PCR 19) with P72
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.45 µl
</TD><TD>P72
</TD></TR><TR><TD>3.55 µl
</TD><TD>OMT+ (PCR 19)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>OMT- (PCR 20) with P72
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.87 µl
</TD><TD>P72
</TD></TR><TR><TD>4,13 µl
</TD><TD>OMT- (PCR 20)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>Negative control
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>P71 or P72
</TD></TR><TR><TD>3 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA-ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><UL><LI>water bath 16 °C 
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Transformation_of_ligationproducts_of_4CL.2C_OMT_and_CHS_respectively_in_pTUM104_in_E.coli">Transformation of ligationproducts of 4CL, OMT and CHS respectively in pTUM104 in <I>E.coli</I></SPAN></H3><P>Investigator: Mary,Daniela
</P><UL><LI> adding 5µl ligation product  in 100µl competent XL blue E.coli cells
</LI><LI> incubation 30 min on ice
</LI><LI> 5 min at 37°C
</LI><LI> adding cells in 1 ml LB (without antibiotica) and incubate at 37°C, 45 min, 180 rpm
</LI><LI> plate on agar with ampicillin over night
</LI></UL><P>results:
</P><UL><LI>CHS+ in pTUM104 (P72) 100 µl: 7 clones
</LI><LI>CHS- in pTUM104 (P72) 100 µl: 9 clones
</LI><LI>CHS+ in pTUM104 (P72) Pellet: 111 clones
</LI><LI>CHS- in pTUM104 (P72) Pellet: 77 clones
</LI></UL><UL><LI>4CL+ in pTUM104 (P71) 100 µl: 9 clones
</LI><LI>4CL- in pTUM104 (P71) 100 µl: 3 clones
</LI><LI>4CL+ in pTUM104 (P71) Pellet: 47 clones
</LI><LI>4CL- in pTUM104 (P71) Pellet: 45 clones
</LI></UL><UL><LI>OMT+ in pTUM104 (P72) 100 µl: 7 clones
</LI><LI>OMT- in pTUM104 (P72) 100 µl: 5 clones
</LI><LI>OMT+ in pTUM104 (P72) Pellet: 30 clones
</LI><LI>OMT- in pTUM104 (P72) Pellet: 53 clones
</LI></UL><UL><LI>negative control P71 100 µl: 14
</LI><LI>negative control P71 Pellet: 71
</LI><LI>negative control P72 100 µl: 7
</LI><LI>negative control P72 Pellet: 64
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Inoculation_of_YPD_medium_with_S._cerevisiae_and_brewing_yeast_strain_34.2F70_.28part_1.2F3.29"> Inoculation of YPD medium with <I>S. cerevisiae</I> and brewing yeast strain 34/70 (part 1/3) </SPAN></H3><P><B>investigator: </B> Maddin, Aloisius
</P><P><B>Aim of the experiment:</B> Discrimination of growing ability in wort medium
</P><P>4 different worts:
</P><UL><LI>21%, without hop
</LI><LI>21%, without hop, autoclaved
</LI><LI>16%, with hop
</LI><LI>16%, with hop, autoclaved
</LI></UL><P>were diluted with ELGA to a solution of 12%. These 4 different worts were inoculated with 5ml of preparatory culture of <I>S. cerevisiae</I> and 1ml of brewing yeast strain 34/70 (provided by the Forschungsbrauerei, Weihenstephan). Start OD was measured at 550nm (for table see next experiment).
Those 8 flasks were incubated over night at room temperature and 130 rpm.
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_18th"><B>Wednesday, July 18th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Minipreparation_of_ADH1-T_Plasmids"> Minipreparation of ADH1-T Plasmids</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> ADH1-T Plasmids were extracted according to protocoll of Quiaprep gelextraction protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_ADH1-T_and_gelelectrophoresis"> Analytical digestion of ADH1-T and gelelectrophoresis</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1,25 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>1,25 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>5 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,5 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>79,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>= 87,5µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17,5 µl Mastermix were added to 2,5 µl plasid DNA
</LI></UL><UL><LI> They have sent the wrong insert
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_plasmid_pSB1C3_containing_20aaLinker_with_LexA_and_plasmid_containing_Gal4AD_with_Pif3_for_fusion_protein_construction"> Ligation of plasmid pSB1C3 containing 20aaLinker with LexA and plasmid containing Gal4AD with Pif3 for fusion protein construction</SPAN></H3></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_from_picked_transformed_E._coli_containing_biobrick_BBa_K165055"> Miniprep of overnight culture from picked transformed <I>E. coli</I> containing biobrick BBa_K165055 </SPAN></H3></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Inoculation_of_YPD_medium_with_S._cerevisiae_and_brewing_yeast_strain_34.2F70_.28part_2.2F3.29"> Inoculation of YPD medium with <I>S. cerevisiae</I> and brewing yeast strain 34/70 (part 2/3) </SPAN></H3><P><B>Investigators: </B> Martin, Alois
</P><P><B>Aim of the experiment:</B> Discrimination of growing ability in wort medium.
</P><P>After 12h of cultivating the optical density (OD) at 550nm was determined:
</P><TABLE cellspacing="0" border="2"><TBODY><TR><TD><B>Assay</B></TD><TD><B>Yeast</B></TD><TD><B>Autoclaved</B></TD><TD><B>Hop</B></TD><TD><B>OD 0h</B></TD><TD><B>OD 12h</B></TD><TD><B>delta OD</B></TD></TR><TR><TD>A1
</TD><TD><I>S. cerevisae</I></TD><TD>No
</TD><TD>No
</TD><TD>1.2
</TD><TD>2.2
</TD><TD> + 1.0
</TD></TR><TR><TD>A2
</TD><TD><I>S. cerevisae</I></TD><TD><B>Yes</B></TD><TD>No
</TD><TD>2.0
</TD><TD>2.6
</TD><TD> + 0.6
</TD></TR><TR><TD>A3
</TD><TD><I>S. cerevisae</I></TD><TD>No
</TD><TD><B>Yes</B></TD><TD>1.8
</TD><TD>2.4
</TD><TD> + 0.6
</TD></TR><TR><TD>A4
</TD><TD><I>S. cerevisae</I></TD><TD><B>Yes</B></TD><TD><B>Yes</B></TD><TD>2.4
</TD><TD>2.7
</TD><TD> + 0.3
</TD></TR><TR><TD>B1
</TD><TD>34/70
</TD><TD>No
</TD><TD>No
</TD><TD>1.6
</TD><TD>2.5
</TD><TD> + 0.9
</TD></TR><TR><TD>B2
</TD><TD>34/70
</TD><TD><B>Yes</B></TD><TD>No
</TD><TD>3.0
</TD><TD>2.9
</TD><TD><B>- 0.1</B></TD></TR><TR><TD>B3
</TD><TD>34/70
</TD><TD>No
</TD><TD><B>Yes</B></TD><TD>2.2
</TD><TD>2.5
</TD><TD> + 0.3
</TD></TR><TR><TD>B4
</TD><TD>34/70
</TD><TD><B>Yes</B></TD><TD><B>Yes</B></TD><TD>2.7
</TD><TD>2.8
</TD><TD> + 0.1
</TD></TR></TBODY></TABLE><P>After 12h of cultivating the <I>S. cerevisiae</I> is growing quite well. There is a tendency that the brewing yeast strain 34/70 is not capable to grow sufficiently in autoclaved (e.g. proteinfree) medium. Since the 34/70 was stored at 4°C over 5 days, directly used to inoculate the medium (no preparatory culture) and we only used 1ml to inoculate with (compared to 5ml of the preparatory culture of <I>S. cerevisiae</I> - because of obvious differences in opacity of the inoculation media) there is not yet a conclusion to be drawn. We decided to incubate for another 24h at least.
</P></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="New_Miniprep_of_P50_pTUM104"> New Miniprep of P50 pTUM104 </SPAN></H3><P><B>Investigators: Andrea</B></P><P>NanoDrop concentration: 303.6 ng/µl (260/280: 1.89)
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Transformation_of_P50_.28pTUM104.29_in_E.coli">Transformation of P50 (pTUM104) in <I>E.coli</I></SPAN></H3><P>Investigator: Katrin
</P><P><B>Aim of the experiment:</B> Get more P50(pTUM104) for further experiments
</P><UL><LI> adding 2-3 µl of plasmid P50 in 100µl competent XL blue E.coli cells
</LI><LI> incubation 30 min on ice
</LI><LI> 5 min at 37°C
</LI><LI> adding cells in 1 ml LB (withouth antibiotica) and incubate at 37°C, 30 min, 180 rpm
</LI><LI> plate on agar with Ampicillin over night
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Ligation_of_4CL_and_PAL_in_pSB1C3-RFC25_and_ligation_of_PAL_in_pTUM104">Ligation of 4CL and PAL in pSB1C3-RFC25 and ligation of PAL in pTUM104</SPAN></H3><P>Investigator: Katrin, Daniela
</P><P>Concentration (Nano Drop: 
4CL+ (PCR 15) = 40.3 ng/µl
4CL- (PCR 16) = 37.3 ng/µl
PAL+ (PCR 32) = 25 ng/µl
PAL- (PCR 33) = 16.8 ng/µl
P132 (pSB1C3-RFC25 digested with Xbal and Pstl) = 33.5 ng/µl
P133 (pSB1C3-RFC25 digested with Xbal and AgeI) = 41.8 ng/µl</P><UL><LI>required volumes were calculated using Lab Tools
</LI></UL><P>4CL+ (PCR 15) with P132 (pSB1C3-RFC25)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.64 µl
</TD><TD>P132
</TD></TR><TR><TD>5.36 µl
</TD><TD>4CL+ (PCR 15)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>4CL- (PCR 16) with P132 (pSB1C3-RFC25)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.51 µl
</TD><TD>P132
</TD></TR><TR><TD>5.49 µl
</TD><TD>4CL- (PCR 16)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>PAL+ (PCR 32) with P133 (pSB1C3-RFC25)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.29 µl
</TD><TD>P133
</TD></TR><TR><TD>6.71 µl
</TD><TD>PAL+ (PCR 32)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>PAL- (PCR 33) with P133 (pSB1C3-RFC25)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.91 µl
</TD><TD>P133
</TD></TR><TR><TD>7.09 µl
</TD><TD>PAL- (PCR 33)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>PAL+ (PCR 32) with P72 (pTUM104)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.55 µl
</TD><TD>P72
</TD></TR><TR><TD>4.45 µl
</TD><TD>PAL+ (PCR 32)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>PAL- (PCR 33) with P72 (pTUM104)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.79 µl
</TD><TD>P72
</TD></TR><TR><TD>5.21 µl
</TD><TD>PAL- (PCR 33)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Negative control:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>P72, P132 or P133
</TD></TR><TR><TD>12 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><UL><LI>water bath 16 °C
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_of_CHS_and_OMT_in_pSB1C3-RFC25">Miniprep of CHS and OMT in pSB1C3-RFC25</SPAN></H3><P>Investigator: Daniela
</P><P><B>Aim of the experiment</B>: Extract plasmids (pSB1C3-RFC25) that contain CHS and OMT
</P><P>QIAprepS Spin Miniprep Kit
</P><UL><LI>step 3: invert 2-3 times (don't shake to avoid destruction of genomic DNA)
</LI></UL><P>the Minipreps were named as follows:
</P><UL><LI>P135: CHS+ in pSB1C3-RFC25 (c = 150.9 ng/µl)
</LI><LI>P136: CHS- in pSB1C3-RFC25 (c = 204 ng/µl)
</LI><LI>P137: OMT+ in pSB1C3-RFC25 (c = 179.3 ng/µl)
</LI><LI>P138: OMT- in pSB1C3-RFC25 (c = 146.7 ng/µl)
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Control_digest_of_CHS_and_OMT_in_pSB1C3-RFC25">Control digest of CHS and OMT in pSB1C3-RFC25</SPAN></H3><P>Investigator: Katrin, Daniela
</P><P><B>Aim of the experiment:</B> Test whether the ligation of CHS and OMT in pSB1C3 was successful
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>P135 / P136 / P137 / P138
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>pSB1C3: 2070 bp
</LI><LI>CHS: 1173 bp
</LI><LI>OMT: 1059 bp
</LI></UL><P>It seems as if the ligation was successful. However it is strange that the band of OMT is lower that the one of CHS. 
we had the wrong information about the length of OMT -&gt; the right length of OMT is 1059 bp, so everything is fine :)
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Picking_clones_of_4CL.2C_CHS_and_OMT_in_pTUM104"> Picking clones of 4CL, CHS and OMT in pTUM104 </SPAN></H3><P><B>investigator: </B> Katrin, Daniela
</P><P><B>Aim of the experiment:</B> preculture over night for the miniprep next day
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_19th"><B>Thursday, July 19th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_P124.2BPCR29_and_P126.2BPCR31">Transformation of P124+PCR29 and P126+PCR31</SPAN></H3><P><B>Investigator:</B> Daniela
</P><P><B>Aim of the experiment:</B> Transformation of the ligation of P124+PCR29 and P126+PCR31 overnight, to see if ligation has been successful.
</P><P><B>Operational sequence:</B></P><UL><LI>P124+PCR29 -&gt;Chlp
</LI><LI>negative control: NK1 -&gt;Chlp
</LI><LI>P126+PCR31 -&gt;Amp
</LI><LI>negative control: NK2 -&gt;Amp
</LI></UL><UL><LI> adding 5µl ligation product in 100 µl competent XL blue E.coli cells
</LI><LI> incubation 30 min on ice
</LI><LI> 5 min at 37°C
</LI><LI> adding cells in 1 ml LB (without antibiotica) and incubate at 37°C, 30 min for ampicillin and 45 min for chloramphenicol, 180 rpm
</LI><LI> plate on agar with ampicillin or chloramphenicol over night
</LI></UL><P><B>EDIT from 20.07.2012:</B> Unfortunately, no colonies for P124+PCR29 can be identified but colonies on from the concentrated pellet of P126+PCR31 ligation, BUT more but very small colonies on the control plate from the pellet. For further identification the colonies of the ligation has been picked on 21.07.2012 for miniprep and analytical digestion and gelelectrophoresis.
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Restriction_digest_of_P79">Restriction digest of P79</SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> Double digest of p79 with Xba1 and Age1 to prepair it for ligation with an N- terminal nuclear localization signal sequence
</P><P>40 µl composition
</P><UL><LI> 18 µl plasmid DNA
</LI><LI> 1 µl Age1 
</LI><LI> 1 µl Xba1
</LI><LI> 4 µl NEBuffer 4 (10x)
</LI><LI> 15,6 µl ddH2O 
</LI><LI> 0,4 µl BSA (100x)
</LI></UL><P>The mixture was incubated over night at 37 °C
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Oligohybridization_of_single-stranded_DNA_for_creating_SV40_nuclear_localization_signal_for_fusion_proteins_for_translocating_them_into_the_nucleus"> Oligohybridization of single-stranded DNA for creating SV40 nuclear localization signal for fusion proteins for translocating them into the nucleus </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Oligohybridization of single-stranded DNA (TUM12-SV40-fw and TUM12-SV40-rv)for creating SV40 nuclear localization signal for fusion proteins for translocating them into the nucleus
</P><P><B>Operational sequence:</B></P><UL><LI> 25 µL of 100 pM of TUM12-SV40-fw and 25 µL of 100 pM TUM12-SV40-rv in one ERG
</LI></UL><UL><LI> Heating up to 95 °C for 30 min
</LI></UL><UL><LI> Cooling at RT in a styropor box overnight.
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Inoculation_of_YPD_medium_with_S._cerevisiae_and_brewing_yeast_strain_34.2F70_.28part_3.2F3.29"> Inoculation of YPD medium with <I>S. cerevisiae</I> and brewing yeast strain 34/70 (part 3/3) </SPAN></H3><P><B>Investigators: </B> Andrea, (Martin, Alois)
</P><P><B>Aim of the experiment:</B> Discrimination of growing ability in wort medium.
</P><P>After 40h of cultivating the optical density (OD) at 550nm was determined:
</P><TABLE cellspacing="0" border="2"><TBODY><TR><TD><B>Assay</B></TD><TD><B>Yeast</B></TD><TD><B>Autoclaved</B></TD><TD><B>Hop</B></TD><TD><B>OD 0h</B></TD><TD><B>OD 12h</B></TD><TD><B>OD 40h</B></TD><TD><B>delta OD</B></TD></TR><TR><TD>A1
</TD><TD><I>S. cerevisae</I></TD><TD>No
</TD><TD>No
</TD><TD>1.2
</TD><TD>2.2
</TD><TD>2.5
</TD><TD> + 1.3
</TD></TR><TR><TD>A2
</TD><TD><I>S. cerevisae</I></TD><TD><B>Yes</B></TD><TD>No
</TD><TD>2.0
</TD><TD>2.6
</TD><TD>2.7
</TD><TD> + 0.7
</TD></TR><TR><TD>A3
</TD><TD><I>S. cerevisae</I></TD><TD>No
</TD><TD><B>Yes</B></TD><TD>1.8
</TD><TD>2.4
</TD><TD>2.6
</TD><TD> + 0.8
</TD></TR><TR><TD>A4
</TD><TD><I>S. cerevisae</I></TD><TD><B>Yes</B></TD><TD><B>Yes</B></TD><TD>2.4
</TD><TD>2.7
</TD><TD>2.8
</TD><TD> + 0.4
</TD></TR><TR><TD>B1
</TD><TD>34/70
</TD><TD>No
</TD><TD>No
</TD><TD>1.6
</TD><TD>2.5
</TD><TD>2.8
</TD><TD> + 1.2
</TD></TR><TR><TD>B2
</TD><TD>34/70
</TD><TD><B>Yes</B></TD><TD>No
</TD><TD>3.0
</TD><TD>2.9
</TD><TD>3.0
</TD><TD><B>0</B></TD></TR><TR><TD>B3
</TD><TD>34/70
</TD><TD>No
</TD><TD><B>Yes</B></TD><TD>2.2
</TD><TD>2.5
</TD><TD>2.8
</TD><TD> + 0.6
</TD></TR><TR><TD>B4
</TD><TD>34/70
</TD><TD><B>Yes</B></TD><TD><B>Yes</B></TD><TD>2.7
</TD><TD>2.8
</TD><TD>2.9
</TD><TD> + 0.2
</TD></TR></TBODY></TABLE><P>After 40h of cultivating the <I>S. cerevisiae</I> is growing comparably to the brewing yeast strain, though we seem to have entered a phase of substrate limitation.
Autoclaved media lack solved protein and is thus for neither of the yeasts a satisfying medium; added hop also represses their growth (<I>S. cerevisiae</I> shows a higher susceptability - which seems logic, since the brewing yeast is selected to survive and prosper in beer brewing environment).
</P><P>The next step would be to brew with our &quot;laboratory strain&quot; <I>S. cerevisiae</I>.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_digest_of_PCR1_and_PCR2_and_P50_and_P123">Preparative digest of PCR1 and PCR2 and P50 and P123</SPAN></H3><P><B>Investigator: Andrea</B></P><P><B>Digestion of PCR1 and PCR2 with XbaI and AgeI</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>PCR-Product
</TD></TR><TR><TD>2 µl
</TD><TD>NEB Buffer
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (10 U/µl)
</TD></TR><TR><TD>7 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Digestion of P50 with XbaI and NgoMIV</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>P50
</TD></TR><TR><TD>4 µl
</TD><TD>NEB Buffer
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (10 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>22.6 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Digestion of P123 with XbaI and AgeI</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P123
</TD></TR><TR><TD>4 µl
</TD><TD>NEB Buffer
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (10 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C, 3 h
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_gel_of_pSB1C3_and_pTUM104">Preparative gel of pSB1C3 and pTUM104</SPAN></H3><P><B>Investigator: Andrea</B></P><UL><LI>40 µl pSB1C3 + 4 µl loading dye
</LI><LI>40 µl pTUM104 + 4 µl loading dye
</LI><LI>20 µl PCR1 + 2 µl loading dye
</LI></UL><UL><LI>Limonensynthase: 1600 bp
</LI><LI>pTUM104: 5800 bp
</LI><LI>pSB1C3: 2000bp
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_digested_PCR1_.28digested_with_XbaI_and_AgeI.29_with_pSB1C3_.28digested_with_XbaI_and_AgeI.29"> Ligation of digested PCR1 (digested with XbaI and AgeI) with pSB1C3 (digested with XbaI and AgeI)</SPAN></H3><P><B>Investigator: Andrea</B></P><P>Concentration (Nano Drop: 
LIMS Citrus (PCR 1) = 18.6 ng/µl
P50 (digested with XbaI and NgoMIV) = 24.9 ng/µl
P123 (digested with XbaI and AgeI) = 44.5 ng/µl</P><P>
PCR1 with P123 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.19µl
</TD><TD>P123
</TD></TR><TR><TD>6.81 µl
</TD><TD>PCR 1
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><UL><LI>water bath 16 °C over night
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_of_4CL.2C_CHS_and_OMT_in_pTUM104_.28P71_and_P72.29">Miniprep of 4CL, CHS and OMT in pTUM104 (P71 and P72)</SPAN></H3><P>Investigator: Daniela
</P><P><B>Aim of the experiment</B>: Extract plasmids (pTUM104) that contain 4CL, CHS and OMT
</P><P>The day before 2 clones were picked for each enzyme.
</P><P>QIAprepS Spin Miniprep Kit
</P><UL><LI>step 3: invert 2-3 times (don't shake to avoid destruction of genomic DNA)
</LI></UL><P>Concentration:
</P><UL><LI>4CL+ in pTUM104 clone 1: c = 172.5 ng/µl
</LI><LI>4CL- in pTUM104 clone 1: c = 192.1 ng/µl
</LI><LI>CHS+ in pTUM104 clone 1: c = 136.6 ng/µl
</LI><LI>CHS- in pTUM104 clone 1: c = 85.7 ng/µl
</LI><LI>OMT+ in pTUM104 clone 1: c = 116.0 ng/µl
</LI><LI>OMT- in pTUM104 clone 1: c = 129.7 ng/µl
</LI></UL><UL><LI>4CL+ in pTUM104 clone 2: c = 160.0 ng/µl
</LI><LI>4CL- in pTUM104 clone 2: c = 144.5 ng/µl
</LI><LI>CHS+ in pTUM104 clone 2: c = 128.5 ng/µl
</LI><LI>CHS- in pTUM104 clone 2: c = 116.2 ng/µl
</LI><LI>OMT+ in pTUM104 clone 2: c = 142.8 ng/µl
</LI><LI>OMT- in pTUM104 clone 2: c = 134.5 ng/µl
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Control_digest_of_4CL.2C_CHS_and_OMT_in_pTUM104">Control digest of 4CL, CHS and OMT in pTUM104</SPAN></H3><P>Investigator: Daniela
</P><P><B>Aim of the experiment:</B> Test whether the ligation of 4CL, CHS and OMT in pTUM104 was successful
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>4CL+ in pTUM104 clone 1 / 4CL- in pTUM104 clone 1
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>4CL+ in pTUM104 clone 2
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>14.3 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.5 µl
</TD><TD>4CL- in pTUM104 clone 2
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>13.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3.5 µl
</TD><TD>CHS+ in pTUM104 clone 1 / OMT+ in pTUM104 clone 1 / OMT- in pTUM104 clone 1 / CHS+ in pTUM104 clone 2 / OMT+ in pTUM104 clone 2 / OMT- in pTUM104 clone 2
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>13.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>CHS- in pTUM104 clone 2
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI(20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>13.3 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>CHS- in pTUM104 clone 1
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI(20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>12.3 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>expected bands:
</P><UL><LI>pTUM104: about 5800bp
</LI><LI>4CL: 1685bp
</LI><LI>CHS: 1173bp
</LI><LI>OMT: 1222bp
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Transformation_of_ligationsproducts_of_4CL_and_PAL_in_pSB1C3-RFC25_and_of_PAL_in_pTUM104_in_E.coli_.28see_July.2C_18th.29">Transformation of ligationsproducts of 4CL and PAL in pSB1C3-RFC25 and of PAL in pTUM104 in <I>E.coli</I> (see July, 18th)</SPAN></H3><P>Investigator: Daniela
</P><UL><LI>4CL+ (PCR15) in pSB1C3-RFC25 (P132) -&gt;Chlp
</LI><LI>4CL- (PCR16) in pSB1C3-RFC25 (P132) -&gt;Chlp
</LI><LI>PAL+ (PCR32) in pSB1C3-RFC25 (P133) -&gt;Chlp
</LI><LI>PAL- (PCR33) in pSB1C3-RFC25 (P133) -&gt;Chlp
</LI><LI>PAL+ (PCR32) in pTUM104(P72) -&gt;Amp
</LI><LI>PAL- (PCR33) in pTUM104(P72) -&gt;Amp
</LI><LI>negative control: P72 -&gt;Amp
</LI><LI>negative control: P132 -&gt;Chlp
</LI><LI>negative control: P133 -&gt;Chlp
</LI></UL><UL><LI> adding 5µl ligation product in 100 µl competent XL blue E.coli cells
</LI><LI> incubation 30 min on ice
</LI><LI> 5 min at 37°C
</LI><LI> adding cells in 1 ml LB (without antibiotica) and incubate at 37°C, 30 min for ampicillin and 45 min for chloramphenicol, 180 rpm
</LI><LI> plate on agar with ampicillin or chloramphenicol over night
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_20th"><B>Friday, July 20th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_Digestion_of_PGK1-P.2C_TEF2-T.2C_TEF1-P_and_gelelectrophoresis">Preparative Digestion of PGK1-P, TEF2-T, TEF1-P and gelelectrophoresis</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3,3 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>3,3 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>13,2µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>1,32 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>44,88 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=72µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 20 µl preparative mastermix were mixed with 20 µl of plasmid-DNA
</LI><LI> Preparative gelelectrophoresis was processed for 90 min at 70 V
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Gel_purification_of_Xba1.2F_Age1_digested_P79"> Gel purification of Xba1/ Age1 digested P79  </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the Experiment:</B> Aim of the experiment is the purification of the double digested vector p79 (pSB1C3_RFC25_Linker), in which the N- terminal SS is to be cloned.
</P><P><B>Operational sequence:</B></P><UL><LI> The mixture was seperated by gel electrophoresis (LMP agarose)
</LI><LI> The DNA was cut out of the gel (2 pieces) and weight ==&gt; 200 mg each piece.
</LI><LI> Afterwards the plasmid DNA was extracted as described in the Quiagen Gel Purification protocol.Elution was performed in two steps (à 25 µl) with elution buffer (heated up to 40°). To improve the yeald of plasmid DNA during the elution, the column was incubated at RT for about 4 minutes before centrifugation 
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_transformed_E..C2.A0coli_XL1-Blue_with_pTUM104_new_from_P50_tube_.284x.29.2C_BBa_K365005_.284x.29.2C_BBa_K207000_.281x.29and_BBa_K207001_.281x.29"> Miniprep of transformed <I>E. coli</I> XL1-Blue with pTUM104 new from P50 tube (4x), BBa_K365005 (4x), BBa_K207000 (1x)and BBa_K207001 (1x)  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the study:</B>: Miniprep
</P><P><B>Operational sequence:</B></P><UL><LI> Miniprep has been performed after manufacturer's protocol. QIAGEN - QIAprep Spin Miniprep Kit.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P152.2C_P155.2C_P156.2C_P157.2C_P158.2C_P159.2C_P160.2C_P161"> Analytical digestion and gelelectrophoresis of P152, P155, P156, P157, P158, P159, P160, P161  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the study:</B> Analytical digestion and gelelectrophoresis of P152, P155, P156, P157, P158, P159, P160, P161, to prove the insert and backbone size.
</P><P><B>Operational sequence:</B></P><UL><LI> Analytical digestion and gelelectrophoresis were performed after lab's standard protocol.
</LI></UL><UL><LI> Digestion with XbaI and PstI-HF in Buffer 4.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_from_the_overnight_transformation_of_the_ligated_P126.2BPCR31"> Picking from the overnight transformation of the ligated P126+PCR31  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the study:</B> Picking from the overnight transformation of the ligated P126+PCR31, to see whether ligation is successful or not.
</P><P><B>Operational sequence:</B></P><UL><LI> Picking and overnight culture after standard laboratory's protocol. (AmpR LB-medium)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_of_P23_.28LexA.2C_BBa_K105005.29_to_indroduce_RFC25_pre-_and_suffix_because_last_ligation_was_not_successful"> PCR of P23 (LexA, BBa_K105005) to indroduce RFC25 pre- and suffix because last ligation was not successful </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Last ligation with LexA was not successful and the tube with the PCR product was empty from the ligation, so one has to do another PCR for another ligation.
</P><P><B>Operational sequence:</B></P><UL><LI> Clone 3 (tube P23) of BBa_K105005 (LexA) has beed choosen for the PCR.
</LI></UL><P><B>PCR reaction mixture</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (BBa_K105005) from P23 (Clone 3)
</TD></TR><TR><TD>35.75 µL
</TD><TD>ELGA Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>55 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>60 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>overnight
</TD></TR></TBODY></TABLE><UL><LI> Freezed at -20 °C. Stil has to be purified with PCR purification kit on the next day
</LI></UL></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Control_digest_of_pTUM104_.28P153_and_P154.29">Control digest of pTUM104 (P153 and P154)</SPAN></H3><P>Investigator: Mary, Ingmar, Daniela
</P><P><B>Aim of the experiment:</B> Test whether the vector pTUM104 is right.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P153
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>16.25 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P153
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>14.25 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P153
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P153
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P153
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>16.25 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P153
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>13.75 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.8 µl
</TD><TD>P154
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>14.95 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.8 µl
</TD><TD>P154
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>12.95 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.8 µl
</TD><TD>P154
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>12.7 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.8 µl
</TD><TD>P154
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>12.7 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.8 µl
</TD><TD>P154
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>14.95 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.8 µl
</TD><TD>P154
</TD></TR><TR><TD>0.25 µl
</TD><TD>NgoMIV (10U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>12.45 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>P153
</LI></UL><UL><LI>P154
</LI></UL><P>The vector seems to be correct. However NgoMIV did not digest the plasmid properly.
</P></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_July_21st"><B>Saturday, July 21st</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_transformed_E._coli_XL1-Blue_with_ligation_product_of_P126.2BPCR31"> Miniprep of transformed E. coli XL1-Blue with ligation product of P126+PCR31 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the study:</B> Miniprep
</P><P><B>Operational sequence:</B></P><UL><LI> Miniprep has been performed after manufacturer's protocol. QIAGEN - QIAprep Spin Miniprep Kit. 
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_of_transformed_E._coli_XL1-Blue_with_ligation_product_of_P126.2BPCR31"> Analytical digestion and gelelectrophoresis of the minipreps of transformed E. coli XL1-Blue with ligation product of P126+PCR31 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> To see whether ligation ist successful.
</P><UL><LI> Reaction batch for each plasmid:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD>Tango Buffer 10x
</TD><TD>4 µl
</TD></TR><TR><TD>NdeI (Fermentas)
</TD><TD>0.5 µl
</TD></TR><TR><TD>BamHI (Fermentas)
</TD><TD>0.5 µl
</TD></TR><TR><TD>Plasmid DNA
</TD><TD>2.5 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>12.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD><B>20 µl</B></TD></TR></TBODY></TABLE><UL><LI> Incubation at 37 °C for 1 h 30 min.
</LI></UL><UL><LI> Analytical gelelectrophoresis at 90 V for 1 h.
</LI></UL><UL><LI> Order of gel-pockets:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder
</TD><TD>P172
</TD><TD>P173
</TD><TD>P174
</TD><TD>1000 bp ladder
</TD></TR><TR><TD><B>corrupt</B></TD><TD><B>corrupt</B></TD><TD><B>corrupt</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was not successful!
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_purification_of_PCR_products_from_P23"> PCR purification of PCR products from P23 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment</B> Purification of PCR products from P23.
</P><P><B>Operational sequence:</B></P><UL><LI> The 4 tubes with PCR products of P23 were purificated with the PCR purification kit from Qiagen after manufacturer'S protocol.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_July_22nd"><B>Sunday, July 22nd</B></SPAN></H2></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparative_digest_of_pTUM104_RFC25_and_ligation_with_PCR_products_PCR_15_-_20_and_32.2B33"> Preparative digest of pTUM104 RFC25 and ligation with PCR products PCR 15 - 20 and 32+33 </SPAN></H3><P><B>Investigator: Ingmar</B></P><P><B>Aim of the experiment:</B> Intsert the genes of PAL, 4CL, CHS and OMT in pTUM104 <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> in order to test their expression in yeast.
</P><P><B>Preparative digest of P 154 with XbaI and NgoMIV and of P 153 with XbaI and PstI</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>2.5 µl
</TD><TD>NEB 4 buffer
</TD></TR><TR><TD>2.5 µl
</TD><TD>Tango buffer
</TD></TR><TR><TD>4 µl
</TD><TD>XbaI (10 U/µl)
</TD></TR><TR><TD>4 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>30 µl
</TD><TD>P 153
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>4 µl
</TD><TD>Tango buffer
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI (10 U/µl)
</TD></TR><TR><TD>4 µl
</TD><TD>PstI (10 U/µl)
</TD></TR><TR><TD>20 µl
</TD><TD>P 154
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C, 3 h
</P><P><B>DNA preparative gel electrophoresis</B></P><UL><LI>gel: 1% with LMP-agarose
</LI><LI>Each digest product was mixed with an adequate volume of DNA loading buffer and loaded into the gel
</LI><LI>The separation process lasted 90 min at 70 V.
</LI></UL><P><A href="/File:Xxx.jpg" title="File:Xxx.jpg">Gel picture of control digest with PstI</A></P><UL><LI>All digest products show the expected bonds at 5849 bp (digest with XbaI and NgoMIV) and 5785 bp (digest with XbaI and PstI) respectively.
</LI></UL><P><B>Gelextration</B></P><UL><LI>cut the bands 
</LI><LI>QIAquick Gel Extractrion Kit was used
</LI><LI> step 6 was left out
</LI><LI> step 9: 30µl buffer, 4 min incubation
</LI></UL><P>Products were named as follows:
</P><UL><LI> P154 digested with XbaI and NgoMIV: P175
</LI><LI> P153 digested with XbaI and PstI: P176
</LI></UL><P><B>Ligation</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>PCR product</B></TD><TD><B>Volume</B></TD><TD><B>Vector</B></TD><TD><B>Volume</B></TD><TD><B>Volume T4 DNA Ligase</B></TD><TD><B>Volume T4 DNA Ligase buffer</B></TD><TD><B>Volume ddH20</B></TD><TD><B>New Plasimd Number</B></TD></TR><TR><TD>PCR15
</TD><TD>5.25
</TD><TD>P 176
</TD><TD>2.75
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD><TD>P 177
</TD></TR><TR><TD>PCR16
</TD><TD>5.25
</TD><TD>P 176
</TD><TD>2.75
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD><TD>P 178
</TD></TR><TR><TD>PCR 17
</TD><TD>7.2
</TD><TD>P 175
</TD><TD>0.8
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD><TD>P 179
</TD></TR><TR><TD>PCR 18
</TD><TD>7.2
</TD><TD>P 175
</TD><TD>0.8
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD><TD>P 180
</TD></TR><TR><TD>PCR 19
</TD><TD>7.1
</TD><TD>P 175
</TD><TD>0.9
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD><TD>P 181
</TD></TR><TR><TD>PCR 20
</TD><TD>7.1
</TD><TD>P 175
</TD><TD>0.9
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD><TD>P 182
</TD></TR><TR><TD>PCR 32
</TD><TD>7.25
</TD><TD>P 175
</TD><TD>0.75
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD><TD>P 183
</TD></TR><TR><TD>PCR 33
</TD><TD>7.5
</TD><TD>P 175
</TD><TD>0.5
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD><TD>P 184
</TD></TR></TBODY></TABLE></DIV></DIV><H1><SPAN class="mw-headline" id="Week_7">Week 7</SPAN></H1><DIV class="week" id="WWeek_7"><H2><SPAN class="mw-headline" id="Monday.2C_July_23rd"><B>Monday, July 23rd</B></SPAN></H2><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_of_PAL.2C_4CL_in_pSB1C3_and_PAL_in_pTUM104">Miniprep of PAL, 4CL in pSB1C3 and PAL in pTUM104</SPAN></H3><P>Investigator: Mary
</P><P><B>Aim of the experiment</B>: Extract plasmids (pTUM104 and pSB1C3-RFC25) that contain 4CL and PAL´
</P><P>The day before one clone were picked for each enzyme from plates from 19th July 2012.
</P><P>QIAprepS Spin Miniprep Kit
</P><UL><LI>step 3: invert 2-3 times (don't shake to avoid destruction of genomic DNA)
</LI></UL><P>Concentration:
</P><UL><LI>4CL+ (PCR15) in pSB1C3-RF25 (P132): c = 340 ng/µl -&gt; new name of Ligationproduct after Miniprep: P185
</LI><LI>4CL- (PCR16) in pSB1C3-RF25 (P132): c = 385 ng/µl -&gt; new name of Ligationproduct after Miniprep: P186
</LI><LI>PAL+ (PCR32) in pSB1C3-RF25 (P133): c = 395 ng/µl -&gt; new name of Ligationproduct after Miniprep: P187
</LI><LI>PAL- (PCR33) in pSB1C3-RF25 (P133): c = 255 ng/µl -&gt; new name of Ligationproduct after Miniprep: P188
</LI></UL><UL><LI>PAL+ (PCR32) in pTUM104 (P72): c = 190 ng/µl
</LI><LI>PAL- (PCR33) in pTUM104 (P72): c = 238 ng/µl
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Control_digest_of_4CL.2C_PAL_in_pSB1C3-RFC25_and_PAL_in_pTUM104">Control digest of 4CL, PAL in pSB1C3-RFC25 and PAL in pTUM104</SPAN></H3><P>Investigator: Mary
</P><P><B>Aim of the experiment:</B> Test whether the ligation of 4CL, PAL in pSB1C3-RFC25 and PAL in pTUM104 was successful
</P><P>Plasmids are taken from miniprep from 23.07.2012
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>4CL+ in pSB1C3-RFC25 / 4CL- in pSB1C3-RFC25
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>PAL+ in pSB1C3-RFC25 / PAL- in pSB1C3-RFC25
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>PAL+ in pSB1C3-RFC25 / PAL- in pSB1C3-RFC25
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>expected bands:
</P><UL><LI>pTUM104: about 5800bp
</LI><LI>pSB1C3-RFC25: 2070bp
</LI><LI>4CL: 1685bp
</LI><LI>PAL: 2151bp
</LI></UL><P>Ligation of PAL+/- in pTUM104 was not succesful
Ligation of PAL+/- and 4CL+/- in pSB1C3-RFC25 were succesful! (pSB1C3 is overlapping with PAL)
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="APT_Solubilisation_and_Transformation">APT Solubilisation and Transformation</SPAN></H3><P>Investigator: Mary
</P><P><B>Aim of the experiment:</B> solubilisation of the synthetic gene of APT and transformation of sent plasmid (including APT) in <I>E.coli</I> to copy it if necessary
</P><P>short zentrifugation of sent product and adding 50µl bidest. H2O to it (5µg in 50µl = concentration of 0.1 µg/µl)
the dissolved product was named as G1 (as Geneproduct number 1) and stored at -20°C 
</P><P>2µl of solved product used for transformation in Ecoli (Kit of Quiagen) and Amp-resistance (said GeneArt)
Plating cells on Agar with Amp, 37°C over night
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparative_digest_of_APT">Preparative digest of APT</SPAN></H3><P>Investigator: Mary
</P><P><B>Aim of the experiment:</B> Extraction of the sequence of APT out of the sent plasmid. 
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD> 10µl
</TD><TD>ddH2O
</TD></TR><TR><TD>4 µl
</TD><TD>NEB 4 buffer
</TD></TR><TR><TD>4 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (20 U/µl)
</TD></TR><TR><TD>20 µl
</TD><TD>solubilised APT-product (see Solubilisation APT)
</TD></TR></TBODY></TABLE><P>Incubation: 37°C, 2.5h
</P><P>Bond at 1244bp as expected:
</P><P>The bond was cutted out of the gel and stored at -20°C (P189)
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Transformation_of_ligationsproducts_of_4CL.2C_CHS.2C_OMT_and_PAL_in_pTUM104_in_E.Coli_.28Ligation_see_22th_of_July.29">Transformation of  ligationsproducts of 4CL, CHS, OMT  and PAL in pTUM104 in <I>E.Coli</I> (Ligation see 22th of July)</SPAN></H3><P>Investigator: Mary
</P><P><B>Aim of the experiment:</B> ligation of enzymes in pTUM104 to transform it into yeast if this was successful.
</P><UL><LI> 4CL+ (PCR15) in pYES (P176) -&gt; new name: P177
</LI><LI> 4CL- (PCR16) in pYES (P176) -&gt; new name: P178
</LI><LI> CHS+ (PCR17) in pYES (P175) -&gt; new name: P179
</LI><LI> CHS- (PCR18) in pYES (P175) -&gt; new name: P180
</LI><LI> OMT+ (PCR19) in pYES (P175) -&gt; new name: P181
</LI><LI> OMT- (PCR20) in pYES (P175) -&gt; new name: P182
</LI><LI> PAL+ (PCR32) in pYES (P175) -&gt; new name: P183
</LI><LI> PAL- (PCR33) in pYES (P175) -&gt; new name: P184
</LI></UL><UL><LI> adding 5µl ligation product in 100 µl competent XL blue <I>E.coli</I> cells 
</LI><LI> incubation 30 min on ice 
</LI><LI> 5 min at 37°C 
</LI><LI> adding cells in 1 ml LB (without antibiotica) and incubate at 37°C, 30 min 180 rpm 
</LI><LI> plate on agar with ampicillin and incubate over night at 37°C
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Gel-_purification_of_hybridized_oligos:_SV40_SS_.28Tube_PCR34.29">Gel- purification of hybridized oligos: SV40 SS (Tube PCR34)</SPAN></H3><P><B>Investigator: Roman</B><B>Aim of the experiment:</B> Purification of the hybridized oligos in order to make them ready for ligation in pSB1C3
<B>Operational Sequence:</B></P><UL><LI> oligos were seperated by gel electrophoresis (LMP agarose)
</LI><LI> picture:
</LI></UL><UL><LI> afterwards the DNA was cut out and extracted with the Quiagen Gel- purification kit as described in the manufacturer's protocol. Elution was performed in two steps á 25 µl elution buffer. DNA was collected in one tube, which was annotated with PCR34 purif.
</LI><LI> determined concentration (NanoDrop): ca. 385 ng/µl
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_24nd"><B>Tuesday, July 24nd</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_preparative_digested_ADH1-P.2C_CYC1-T.2C_TEF2-P.2C_TEF1-P.2CPGK1-P"> Gelextraction of preparative digested ADH1-P, CYC1-T, TEF2-P, TEF1-P,PGK1-P</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI>Plasmid-DNA was extracted according to Quiaquick gel extraction kit
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop:_measuring_concentration_of_P128-P131.2C_P168-P169">Nanodrop: measuring concentration of P128-P131, P168-P169</SPAN></H3><P><B> Investigator:</B> Georg
</P><UL><LI>P128: 28,3 ng/µl
</LI><LI>P129: 21   ng/µl
</LI><LI>P130: 4,2 ng/µl
</LI><LI>P131: 4,7 ng/µl
</LI><LI>P168: 3,5 ng/µl
</LI><LI>P169: 4,2 ng/µl
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Gelextraction_of_APT_digested_with_Xbal_and_AgeI_.28P189.29">Gelextraction of APT digested with Xbal and AgeI (P189)</SPAN></H3><P>Investigator: Daniela
</P><P><B>Gelextration</B></P><UL><LI>QIAquick Gel Extractrion Kit was used
</LI><LI>the product was named P189
</LI><LI>concentration: c = 9.1 ng/µl
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Ligation_of_APT_.28P189.29_in_pSB1C3-RFC25_.28P133.29">Ligation of APT (P189) in pSB1C3-RFC25 (P133)</SPAN></H3><P>Investigator: Daniela
</P><P>APT (P189) in pSB1C3-RFC25 (P133)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.86 µl
</TD><TD>P133
</TD></TR><TR><TD>7.14 µl
</TD><TD>APT (P189)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>9 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Negative control:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.86 µl
</TD><TD>P133
</TD></TR><TR><TD>16.14 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><UL><LI>water bath 16 °C
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="NanoDrop_determination_of_PCR34-_PCR38">NanoDrop determination of PCR34- PCR38</SPAN></H3><P><B>Investigator:</B>Roman
</P><P><B>Aim:</B> Determination of the concentration of the samples previously to the ligation
</P><P>Determined values:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>PCR Product</B></TD><TD><B>Concentration in ng/µl</B></TD></TR><TR><TD>PCR34
</TD><TD>385 (hybridized oligos)
</TD></TR><TR><TD>PCR35
</TD><TD>41,2
</TD></TR><TR><TD>PCR36
</TD><TD>39,5
</TD></TR><TR><TD>PCR37
</TD><TD>41,9
</TD></TR><TR><TD>PCR38
</TD><TD>57,4
</TD></TR></TBODY></TABLE><UL><LI> PCR38 will be used in following restriction digest
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Restriction_digest_of_p123_and_PCR38_with_NgoMIV_and_Pst1">Restriction digest of p123 and PCR38 with NgoMIV and Pst1</SPAN></H3><P><B>Investigator:</B> Roman
<B>Aim of the experiment:</B> To prepare samples of p123 (pSB1C3 RFC25) and PCR38 for a following ligation
<B>Operational sequence:</B> 
p123 was digested in a 40 µl preparation (miniprep product):
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Plasmid- DNA
</TD><TD>20 µl
</TD></TR><TR><TD>Enzyme NgoMIV
</TD><TD>2 µl
</TD></TR><TR><TD>Enzyme Pst1
</TD><TD>2 µl
</TD></TR><TR><TD>Buffer 4
</TD><TD>4 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>12 µl
</TD></TR></TBODY></TABLE><P>PCR38 was digested in a 50 µl preparation (PCR product)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Purified PCR product
</TD><TD>25
</TD></TR><TR><TD>Enzyme NgoMIV
</TD><TD>2 µl
</TD></TR><TR><TD>Enzyme Pst1
</TD><TD>2 µl
</TD></TR><TR><TD>Buffer 4
</TD><TD>5 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>16 µl
</TD></TR></TBODY></TABLE><P>The preparations were incubated at 37°C for 3h and then stored at -20°C in box  The tubes were annotated with &quot;p123_doubledigest_NgoMIV+Pst1_unpurified_20120724&quot; and &quot;PCR38_doubledigest_NgoMIV+Pst1_unpurified_20120724&quot;
Afterwards, the samples were load on a 1% universal agarose gel and separated at 100 V for ca. 45 min. Corresponding gel- bands were cut out and stored at -20°C over night until extraction.
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_p151_and_PCR34">Ligation of p151 and PCR34</SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> Ligation of fragments previously to a transformation
</P><P><B>Operational Sequence:</B>
Length of fragments:
</P><UL><LI> p151: 2070 bp; c = 9,5 ng/µl
</LI><LI> PCR34: 67 bp; c = 385 ng/µl
</LI></UL><P>20 µl preparation:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>p151- DNA
</TD><TD>11 µl
</TD></TR><TR><TD>PCR34- DNA
</TD><TD>3 µl (out of a 1/100 dilution)
</TD></TR><TR><TD>Ligase
</TD><TD>0,5
</TD></TR><TR><TD>T4 Ligase Buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>3,5 µl
</TD></TR></TBODY></TABLE><P>Negative controls were prepared the same way, using 3 µl of ddH2O instead of PCR34- DNA.
The preparation was incubated 30 min at room temperature and then over night at 16°C (water bath). Afterwards, the samples were stored at 4°C until the transformation.
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_p126_and_PCR31">Ligation of p126 and PCR31</SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B>Ligation of fragments previously to a transformation
</P><P><B>Operational Sequence:</B>
Length of fragments:
</P><UL><LI> p126: 7961 bp; c = 46 ng/µl
</LI><LI> PCR34: 306 bp; c = 11,3 ng/µl
</LI></UL><P>10 µl preparation:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>p126- DNA
</TD><TD>2 µl
</TD></TR><TR><TD>PCR31- DNA
</TD><TD>1 µl
</TD></TR><TR><TD>Ligase
</TD><TD>0,5
</TD></TR><TR><TD>T4 Ligase Buffer
</TD><TD>1 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>5,5 µl
</TD></TR></TBODY></TABLE><P>Negative controls were prepared the same way, using 1 µl of ddH2O instead of PCR31- DNA.
The preparation was incubated 30 min at room temperature and then over night at 16°C (water bath).
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_p175_with_.22PCR1.2C_19.7.2012.22_and_p144">Ligation of p175 with &quot;PCR1, 19.7.2012&quot; and p144</SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> Ligation of fragments previously to a transformation
</P><P><B>Operational Sequence:</B>
Length of fragments:
</P><UL><LI> p175: 5900 bp; c = 107,3 ng/µl
</LI><LI> PCR1: 1680 bp; c = 18 ng/µl
</LI><LI> p144: ??? bp; c = ???
</LI></UL><P>10 µl preparation:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>p175- DNA
</TD><TD>1 µl
</TD></TR><TR><TD>PCR1/ p144- DNA
</TD><TD>3 µl
</TD></TR><TR><TD>Ligase
</TD><TD>0,5
</TD></TR><TR><TD>T4 Ligase Buffer
</TD><TD>1 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>4,5 µl
</TD></TR></TBODY></TABLE><P><B>Note:</B> This preparation was not made optimal, due to use of a wrong fragment lengths (PCR1 and p144, respectively) during the calculation of the volumes.
Negative controls were prepared the same way, using 3 µl of ddH2O instead of PCR fragment.
The preparation was incubated 30 min at room temperature and then over night at 16°C (water bath).
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparation_of_yeast_SCU_Minimal_Medium_for_Plates">Preparation of yeast SCU Minimal Medium for Plates</SPAN></H3><P>Investigator: Katrin, Daniela
</P><UL><LI>The recipe was taken from the pYES2 manual.
</LI><LI>The ingredients were dissolved in 900 ml dest. water (ELGA) corresponding to the recipe. Lysine was only available as lysine-dihydrochloride, therefore 0.149 g instead of 0.1 g were used. Uracil was omited.
</LI><LI>The medium was divided in 2 x 450 ml. One will later be used as the induction medium through the addition of galactose the other one will be used as the non-induction medium (addition of glucose). Sugar solutions will be added after autoclaving to prevent maillard-reaction.
</LI><LI>10 g agar and a magnetic stir bar were added to each preparation
</LI><LI>Autoclaving.
</LI><LI>Glucose solution: 10 g glucose were dissolved in 50 ml ELGA.
</LI><LI>Galactose solution: 10 g galactose were dissolved in 50 ml ELGA.
</LI><LI>No raffinose will be used.
</LI><LI>Autoclaving
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Transformation_of_APT_in_pSB1C3-RFC25_.28P190.29">Transformation of APT in pSB1C3-RFC25 (P190)</SPAN></H3><P>Investigator: Daniela
</P><UL><LI> APT (P189) in pSB1C3 (P133) -&gt; new name: P190
</LI><LI> Negative control P133
</LI></UL><UL><LI> adding 5µl ligation product in 100 µl competent XL blue <I>E.coli</I> cells 
</LI><LI> incubation 30 min on ice 
</LI><LI> 5 min at 37°C 
</LI><LI> adding cells in 1 ml LB (without antibiotica) and incubate at 37°C, 45 min 180 rpm 
</LI><LI> plate on agar with chloramphenicol and incubate over night at 37°C
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Picking_Clones_of_4CL.2C_CHS.2C_OMT_and_PAL_in_pTUM104_and_APT_in_original_plasmid"> Picking Clones of 4CL, CHS, OMT and PAL in pTUM104 and APT in original plasmid</SPAN></H3><P><B>investigator: </B> Daniela
</P><P><B>Aim of the experiment:</B> preculture over night for the miniprep next day
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_July_25th"><B>Wednesday, July 25th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Restriction_digest_of_p123_with_Xba1_and_Age1"> Restriction digest of p123 with Xba1 and Age1 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> 
Aim of the experiment is the preparation of the vector pSB1C3 RFC25 (p123) for a ligation with the PCR fragment PCR34 (as repetition for the ligation of p151 with PCR34, which has probably not worked, due to low vector concentration. 
</P><P><B>Operational sequence:</B>
30 µl preparation for restriction digest of p123
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>p123- DNA
</TD><TD>10 µl (results in ca. 5µg DNA)
</TD></TR><TR><TD>Xba1- RE
</TD><TD>1 µl
</TD></TR><TR><TD>Age1- RE
</TD><TD>1 µl
</TD></TR><TR><TD>NEBuffer 4
</TD><TD>3 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>15 µl
</TD></TR></TBODY></TABLE><P>The preparation was incubated at 37°C for 3,5 h. Afterwards the fragments were purificated by means of a preparative gel electrophoresis (see below).
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Restriction_digest_of_p123_with_Age1_and_Pst1"> Restriction digest of p123 with Age1 and Pst1 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> 
Aim of the experiment is the preparation of the pSB1C3- Vector (p123) for a ligation with PCR38 (which has been digested with NgoMIV and Pst1).
</P><P><B>Operational sequence:</B>
30 µl preparation for restriction digest of p123
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>p123- DNA
</TD><TD>10 µl (results in ca. 5µg DNA)
</TD></TR><TR><TD>Pst1- RE
</TD><TD>1 µl
</TD></TR><TR><TD>Age1- RE
</TD><TD>1 µl
</TD></TR><TR><TD>NEBuffer 4
</TD><TD>3 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>15 µl
</TD></TR></TBODY></TABLE><P>The preparation was incubated at 37°C for 3,5 h. Afterwards the fragments were purificated by means of a preparative gel electrophoresis (see below).
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Gel_extraction_of_PCR38_and_p123_.28both_digested_with_NgoMIV_and_Pst1.29"> Gel extraction of PCR38 and p123 (both digested with NgoMIV and Pst1) </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> 
The separated PCR38 and p123 (pSB1C3 RFC25) DNA- fragments from yesterday's restriction digest (both NgoMIV and Pst1) are to be extracted from agarose- gel (which has been stored at -20°C over night), to make them ready for further usage.
</P><P><B>Operational sequence:</B>
The extraction was performed as described in the manual of the Quiagen Gel- extraction kit. Elution was performed with 30 µl of elution- buffer EB in two steps (à 15 µl), temperated at 50°C. Concentration was determined with NanoDrop:
</P><UL><LI> PCR38: 17,3 ng/µl
</LI><LI> p123:  37,2 ng/µl
</LI></UL><P>Tubes were annotated with p191 (p123 NgoMIV/Pst1) and p192 (PCR38 NgoMIV/Pst1).
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Gel_purification_of_p123_.28Xba1.2FAge1_double_digest.29_and_p123_.28Age1.2FPst1_double_digest.29"> Gel purification of p123 (Xba1/Age1 double digest) and p123 (Age1/Pst1 double digest) </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> 
Aim of the experiment is the purification of two different double digested p123 samples, to make them ready for ligation with the inserts PCR38 and PCR34, respectively.
</P><P><B>Operational sequence:</B>
The samples were loaded on a 1% universal agarose gel and separated at 100 V for about 45 min. Afterwards, the corresponding gel- bands were cut out and weight. Extraction from gel was performed as described in the manufacturers protocoll (Quiagen gel extraction kit). Changes: Elution was performed in one step with 30µl ddH2O (autoclaved) temperated at 50°C.
Concentrations were determined with NanoDrop:
</P><UL><LI> p123 (Xba1/Age1): 70,8 ng/µl
</LI><LI> p123 (Pst1/Age1): 59,1 ng/µl
</LI></UL><P>The tubes were annotated with p206 (p123 Age1/Pst1) and p207 (p123 Age1/Xba1).
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_p123_.28Age1.2FPst1.29_and_PCR38"> Ligation of p123 (Age1/Pst1) and PCR38 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> Ligation of the fragments p123 and PCR38
</P><P><B>Operational sequence:</B>
Length of fragments:
</P><UL><LI> p123: 2070 bp; c = 59,1 ng/µl
</LI><LI> PCR38: 617 bp; c = 17,3 ng/µl
</LI></UL><P>20 µl preparation:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>p123- DNA
</TD><TD>3 µl (ca. 180 ng vector- dna)
</TD></TR><TR><TD>PCR38- DNA
</TD><TD>10 µl
</TD></TR><TR><TD>Ligase
</TD><TD>0,5
</TD></TR><TR><TD>T4 Ligase Buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>4,5 µl
</TD></TR></TBODY></TABLE><P>Negative controls were prepared the same way, using 10 µl of ddH2O instead of PCR38- DNA.
The preparation was incubated 30 min at room temperature and then over night at 16°C (water bath). Afterwards, the samples were stored at 4°C in the fridge until transformation.
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_p123_.28Age1.2FXba1.29_and_PCR34"> Ligation of p123 (Age1/Xba1) and PCR34 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> 
Ligation of fragments p123 and PCR34
<B>Operational sequence:</B>
Length of fragments:
</P><UL><LI> p123: 2070 bp; c = 70,8 ng/µl
</LI><LI> PCR34: 67 bp; c = 3,85 ng/µl (1:100 dilution of original sample)
</LI></UL><P>20 µl preparation:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>p123- DNA
</TD><TD>3 µl (ca. 210 ng vector- dna)
</TD></TR><TR><TD>PCR34- DNA
</TD><TD>6 µl (out of 1:100 dilution)
</TD></TR><TR><TD>Ligase
</TD><TD>0,5
</TD></TR><TR><TD>T4 Ligase Buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>8,5 µl
</TD></TR></TBODY></TABLE><P>Negative controls were prepared the same way, using 6 µl of ddH2O instead of PCR34- DNA.
The preparation was incubated 30 min at room temperature and then over night at 16°C (water bath). Afterwards, the samples were stored at 4°C in the fridge until transformation.
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_of_4CL.2C_CHS.2C_OMT.2C_PAL_in_pTUM104_and_APT_in_original_plasmid_from_GeneArt">Miniprep of 4CL, CHS, OMT, PAL in pTUM104 and APT in original plasmid from GeneArt</SPAN></H3><P>Investigator: Katrin
</P><P><B>Aim of the experiment</B>: extraction of plasmids (pTUM104) that contain 4CL, CHS, OMT, PAL; extraction of original plasmid from GeneArt with APT
</P><P>QIAprepS Spin Miniprep Kit
</P><UL><LI>step 3: invert 2-3 times (don't shake to avoid destruction of genomic DNA)
</LI></UL><P>the Minipreps were named as follows:
</P><UL><LI>P193: 4CHL+ in pSB1C3-RFC25 (c = 110,7 ng/µl)
</LI><LI>P194: 4CL- in pYEs (c = 342,8 ng/µl)
</LI><LI>P195: CHS+ in pYEs (c = 423,6 ng/µl)
</LI><LI>P196: CHS- in pYEs (c = 92,1 ng/µl)
</LI><LI>P197: OMT+ in pYEs (c = 394,6 ng/µl)
</LI><LI>P198: OMT- in pYEs (c = 183,0 ng/µl)
</LI><LI>P199: PAL+ in pYEs (c = 138,4 ng/µl)
</LI><LI>P200: PAL- in pYEs (c = 464,5 ng/µl)
</LI><LI>P201: APT in original plasmid (c = 260,0 ng/µl)
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_Ligation-products_into_E.coli_XL-1_Blue">Transformation of Ligation-products into <I>E.coli</I> XL-1 Blue</SPAN></H3><P><B>Investigator: Andrea</B></P><UL><LI> for each Biobrick 100 µl cells were used and pooled together with 5 µl of ligation product from the 24th of july 
</LI></UL><P>(3 hours at 16°C and over night at 4 °C) and from the 19th of july (5 days at 16°C)
</P><UL><LI> Incubation on ice for 30 min
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> cells were prefilled with 1 ml of LB-medium and incubated in a cell-culture shaker at 37 °C for 40 min
</LI></UL><UL><LI> 100 µl of these cell suspension were plated on antibiotic selection plates (ligations in pTUM104: Ampicillin; ligations in pSB1C3: Chloramphenicol)
</LI></UL><UL><LI> cell suspension was centrifuged at 13000 rpm for 90 s for resuspending the pellet with 100 µl LB and plating also
</LI></UL><UL><LI> incubation at 37 °C overnight
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_July_26th"><B>Thursday, July 26th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_ligation_products_of_P126.2BPCR31.2C_P151.2BPCR34.2C_P123.2BPCR38_and_P123.2BPCR34_in_E..C2.A0coli_XL1-Blue"> Transformation of ligation products of P126+PCR31, P151+PCR34, P123+PCR38 and P123+PCR34 in <I>E. coli</I> XL1-Blue </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B>Transformation of ligation products of P126+PCR31 (AmpR), P151+PCR34 (CamR), P123+PCR38 (CamR) and P123+PCR34 (CamR) in <I>E. coli</I> XL1-Blue
</P><P><B>Operational sequence:</B></P><UL><LI> Performed after lab's standard protocol.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_of_APT_in_pSB1C3-RFC25_and_control_digestion_with_Xba1_and_Pst1"> Miniprep of APT in pSB1C3-RFC25 and control digestion with Xba1 and Pst1</SPAN></H3><P>Investigator: Mary
</P><P><B>Aim of the experiment</B>: extraction of plasmid (pSB1C3-RFC25) that contains APT and testing if ligation was succesful; 
three clones were picked the day before
</P><P><B>Miniprep</B></P><P>QIAprepS Spin Miniprep Kit
</P><UL><LI>step 3: invert 2-3 times (don't shake to avoid destruction of genomic DNA)
</LI></UL><P>the Minipreps were named as follows:
</P><UL><LI>APT in pSB1C3-RFC25 clone 1: c=96 ng/µl
</LI><LI>APT in pSB1C3-RFC25 clone 2: c=108 ng/µl
</LI><LI>APT in pSB1C3-RFC25 clone 3: c=97 ng/µl
</LI></UL><P><B>Control digest</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>4CL+/-, PAL+/-, CHS+/-, OMT+/-, APT+/-
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>expected bands:
</P><UL><LI>pSB1C3-RF25: 2070bp
</LI><LI>APT: 1244bp
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Control_digest_of_4CL.2C_PAL.2C_CHS.2C_OMT.2C_APT_in_pTUM104">Control digest of 4CL, PAL, CHS, OMT, APT in pTUM104</SPAN></H3><P>Investigator: Mary
</P><P><B>Aim of the experiment:</B> Test whether the ligation of 4CL, PAL, CHS, OMT, APT in pTUM104 was successful
</P><P>Plasmids are taken from miniprep from 25.07.2012
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>4CL+/-, PAL+/-, CHS+/-, OMT+/-, APT+/-
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>expected bands:
</P><UL><LI>pYES: about 5800bp
</LI><LI>4CL: 1685bp
</LI><LI>PAL: 2151bp
</LI><LI>CHS: 1173bp
</LI><LI>OMT: 1222bp
</LI><LI>APT: 1244bp
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Pour_on_yeast_agarplates_for_selection_of_yeast_cells"> Pour on yeast agarplates for selection of yeast cells </SPAN></H3><P>Investigator: Mary
</P><P><B>Aim of the experiment:</B> prepare selection-agarplates without Uracil (Some plates including glucose, some including galactose)
</P><P>it was strange that the agar was still pretty liquid after waiting for one hour.
</P><P>see pYES-manual 
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_clones_of_transformation_from_25.07.">Picking of clones of transformation from 25.07.</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Amplification of clones for extraction of plasmids and subsequent proof of functional ligation.
</P><P>6 clones were picked for each ligation (PCR1 in pYESnew, PCR2 (p144) in pYESnew and PCR in pSB1C3. Clones containing pYESnew were put into 4 ml LB with 4 µl of ampillicin stock solution, clones containing pSB1C3 were put into chloramphenicol-containing media. Incubation at 37 °C over night.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_plasmids_P40.26P42_into_E.coli_XL1_blue">Transformation of plasmids P40&amp;P42 into <I>E.coli</I> XL1 blue</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B></P><P>Transformation of plasmids P40 and P42 into<I>E.coli</I> XL1 blue to get plasmids of higher quality. P40&amp;P42 are directly extracted from the expression strain BL21(DE). The quality of the plasmids might not be sufficient for Quickchange site-directed mutagenesis. Therefore transformation into <I>E.coli</I> XL1 blue and subsequent plasmid preparation to get plasmids with sufficient quality for SDM.
</P><P><B>Procedure:</B></P><P>100 μl of competent XL1 blue cells were thawed on ice. 1 µl plasmid DNA (P40 or P42) was added. Incubation for 30 min on ice. 5 min heatshock	at 37°C. 1 ml of LB medium without antibiotic was added, incubation for 45 min at 180 rpm/37°C.
After incubation, 100 µl of the cell suspension were plated on antibiotic containing plates (P40: Kan, P42: Amp). The remaining solution was centrifuged for 60 sec, resuspended in 100 µl LB and plated, as well.
Incubation at 37 °C over night.
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_July_27th"><B>Friday, July 27th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_from_the_overnight_transformation_of_ligated_P126.2BPCR31.2C_P151.2BPCR34.2C_P123.2BPCR38_and_P123.2BPCR34_in_E..C2.A0coli_XL1-Blue"> Picking from the overnight transformation of ligated P126+PCR31, P151+PCR34, P123+PCR38 and P123+PCR34 in <I>E. coli</I> XL1-Blue </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment</B> Picking of transformated <I>E. coli</I>XL1-Blue with ligation product of P126+PCR31 (AmpR), P151+PCR34 (CamR), P123+PCR38 (CamR) and P123+PCR34 (CamR).
</P><P><B>Procedure</B></P><UL><LI> Colonies were picked with pipette tips and transformed into a new cell-culture tube with 4 ml of LB-medium and antibiotics and were put overnight in a 180 rpm cell culture shaker at 37 °C. P126+PCR31 (AmpR), P151+PCR34 (CamR), P123+PCR38 (CamR) and P123+PCR34 (CamR)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transferring_E..C2.A0coli_XL1-Blue_colonies.2C_transformed_with_BBa_I15008_.28heme_oxygenase.29_and_BBa_K105005_.28LexA.29.2C_from_old_antibiotic_plates_on_new_plates"> Transferring <I>E. coli</I> XL1-Blue colonies, transformed with BBa_I15008 (heme oxygenase) and BBa_K105005 (LexA), from old antibiotic plates on new plates </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment</B> With a sterile incolation loop the colonies were transferred on a new antibiotic plate, because the plates are already 4 weeks old.
</P><P><B>Procedure</B></P><UL><LI> Incolation loop was put for few second into the flame of a Bunsen burner
</LI></UL><UL><LI> Colonies from plate with BBa_I15008 (heme oxygenase, KanR) and BBa_K105005 (LexA, AmpR) were transferred on a new antibiotic plate
</LI></UL><UL><LI> Overnight-culture at 37 °C
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_and_control_digest_of_PAL_in_pYES_picked_on_thursday_26th_July">Miniprep and control digest of PAL in pYES picked on thursday 26th July</SPAN></H3><P>Investigator: Ingmar
</P><P><B>Aim of the experiment:</B> Test whether the ligation of PAL in pTUM104 was successful
</P><P><B>Miniprep</B>
Using a Quiagen Miniprep kit a plasmid isolation of two picked clones of the PAL+ in pYes2 RFC25 ligation was done. The miniprep product of the first clone was named P219, the one of the second clone P220. Both were used for the following control digest.
</P><P><B>control digest</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>PAL+
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>expected bands:
</P><UL><LI>pYES:5819bp
</LI><LI>PAL: 2151bp
</LI></UL><P><A href="/File:TUM12_Xanthohumol_P219_und_P220_27.07.2012.jpg" title="File:TUM12 Xanthohumol P219 und P220 27.07.2012.jpg">500px</A></P><P>As the picture only shows one band at the expected length of the vecotr backbone the ligation was not successfull and will be repeated with a reduced quotient of insert to vector of 3.
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Ligation_of_PAL.2B_.28PCR32.29_and_APT_.28P189.29_with_pYes2_RFC25_.28P175.29"> Ligation of PAL+ (PCR32) and APT (P189) with pYes2 RFC25 (P175) </SPAN></H3><P><B>Investigator: Ingmar</B></P><P><B>Aim of the experiment:</B> Insert the genes of PAL and APT in pYEs2 <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> in order to test their expression in yeast.
</P><P><B>Ligation</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>PCR product</B></TD><TD><B>Volume</B></TD><TD><B>Vector</B></TD><TD><B>Volume</B></TD><TD><B>Volume T4 DNA Ligase</B></TD><TD><B>Volume T4 DNA Ligase buffer</B></TD><TD><B>Volume ddH20</B></TD><TD><B>New Plasimd Number</B></TD></TR><TR><TD>PAL+ (PCR32)
</TD><TD>6.6
</TD><TD>P 175
</TD><TD>1.4
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD><TD>P 221
</TD></TR><TR><TD>APT (P 189)
</TD><TD>7.06
</TD><TD>P 175
</TD><TD>0.94
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD><TD>P 222
</TD></TR><TR><TD>control: ddH2O
</TD><TD>7
</TD><TD>P 175
</TD><TD>1
</TD><TD>1 µl
</TD><TD>2 µl
</TD><TD>9 µl
</TD></TR></TBODY></TABLE><P>The Ligation product of PAL+ and pYes was labeled P233, the one of APT and pYes P234.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Plasmid_extraction_of_plasmids_PCR1.2FpTUM104.2C_P144.2FpTUM104.2C_PCR1.2FpSB1C3"> Plasmid extraction of plasmids PCR1/pTUM104, P144/pTUM104, PCR1/pSB1C3 </SPAN></H3><P><B>Investigator: Lara</B></P><P><B>Aim of the experiment:</B> Extract plasmids from ligation of PCR1 in pYES, P144 in pTUM104 and PCR1 in pSB1C3 for further analytical restriction digest. (6 clones were picked for each ligation and plasmids subsequently extracted.)
</P><P><B>Prodedure:</B> Plasmids were extraced by using Qiagen plasmid miniprep kit.
</P><P>Following concentrations were obtained:
</P><P>1_1 - 1_6
</P><UL><LI>PCR1 in pYESnew clone 1: c=66 ng/µl
</LI><LI>PCR1 in pYESnew clone 2: c=182 ng/µl
</LI><LI>PCR1 in pYESnew clone 3: c=98 ng/µl
</LI><LI>PCR1 in pYESnew clone 4: c=326 ng/µl
</LI><LI>PCR1 in pYESnew clone 5: c=87 ng/µl
</LI><LI>PCR1 in pYESnew clone 6: c=156 ng/µl
</LI></UL><P>2_1 - 2_6
</P><UL><LI>P144(PCR2) in pYESnew clone 1: c=195 ng/µl
</LI><LI>P144(PCR2) in pYESnew clone 2: c=202 ng/µl
</LI><LI>P144(PCR2) in pYESnew clone 3: c=198 ng/µl
</LI><LI>P144(PCR2) in pYESnew clone 4: c=77 ng/µl
</LI><LI>P144(PCR2) in pYESnew clone 5: c=95 ng/µl
</LI><LI>P144(PCR2) in pYESnew clone 6: c=200 ng/µl
</LI></UL><P>3_1 -3_6
</P><UL><LI>PCR1 in pSB1C3-RFC25 clone 1: c=116 ng/µl
</LI><LI>PCR1 in pSB1C3-RFC25 clone 2: c=110 ng/µl
</LI><LI>PCR1 in pSB1C3-RFC25 clone 3: c=116 ng/µl
</LI><LI>PCR1 in pSB1C3-RFC25 clone 4: c=112 ng/µl
</LI><LI>PCR1 in pSB1C3-RFC25 clone 5: c=139 ng/µl
</LI><LI>PCR1 in pSB1C3-RFC25 clone 6: c=80 ng/µl
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Restriction_digest_of_plasmids_from_6_clones_each_of_PCR1.2FpTUM104new.2C_P144.28PCR2.29.2FpTUM104new.2C_PCR1.2FpSB1C3"> Restriction digest of plasmids from 6 clones each of PCR1/pTUM104new, P144(PCR2)/pTUM104new, PCR1/pSB1C3 </SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim of the experiment:</B> Analytical restriction digest of plasmids to check for insert.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>DNA
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer 4
</TD></TR><TR><TD>0,25 µl
</TD><TD>Xba1
</TD></TR><TR><TD>0,25 µl
</TD><TD>Spe1-HF
</TD></TR><TR><TD>0,2 µl
</TD><TD>BSA(100x)
</TD></TR><TR><TD>14,3 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation: 37 °C; 1,5 h
</P><P>A mastermix for 19 samples was made.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_gel_electrophoresis_of_PCR1.2FpTUM104new.2C_P144.28PCR2.29.2FpTUM104new.2C_PCR1.2FpSB1C3"> Analytical gel electrophoresis of PCR1/pTUM104new, P144(PCR2)/pTUM104new, PCR1/pSB1C3 </SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim of the experiment:</B> Analytical gel electrophoresis of products from restriction digest.
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Small-Scale_Yeast_Transformation_of_4CL.2C_CHS.2C_OMT_and_PAL-_in_pTUM104">Small-Scale Yeast Transformation of 4CL, CHS, OMT and PAL- in pTUM104</SPAN></H3><P><B>Investigator:</B> Daniela
</P><P><B>Aim of the experiment:</B>Transformation of P193 - P198 and P200 in Yeast 
</P><P>The protocol on page 13 of the pYES2 manual was used.
</P><P>Only modifications are noted:
</P><P>step 1: inoculate in 4 ml YPD medium
</P><P>step 2: OD600 = 10 -&gt; to determine a OD600 of 0.4 in 50 ml, 2.5 ml yeast suspension and 47.5 ml YPD medium were used (1:20 dilution)
</P><P>steps 3 and 4: centrifugation for 5 min, 4 °C
</P><P>step 5: room temperature was about 35 °C
</P><P>step 6: 
</P><P>1 µg plasmid DNA
</P><UL><LI>P193: c = 110.7 ng/µl -&gt; 9 µl 
</LI><LI>P194: c = 342.8 ng/µl -&gt; 3 µl
</LI><LI>P195: c = 423.6 ng/µl -&gt; 2.4 µl  
</LI><LI>P196: c = 92.1 ng/µl -&gt; 11 µl 
</LI><LI>P197: c = 394.6 ng/µl -&gt; 2.5 µl
</LI><LI>P198: c = 183.4 ng/µl -&gt; 5.5 µl  
</LI><LI>P200: c = 464.5 ng/µl -&gt; 2.2 µl 
</LI></UL><P>100 µg denatured sheared salmon sperm: the one from Simon was used -&gt; 10 µl
</P><P>step 8: incubation was carried out at 35 °C because the thermoblock did not achieve a lower temperature
</P><P>The SCU- plates are incubated at 30°C over the weekend.
</P><P>Colonies were grown on 2 plates (31.07.2012): 
CHS+ : 1 colony
OMT+ : 3 colonies
</P><P>-&gt; repetition of transformation on plates with glucose!
</P></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_July_28th"><B>Saturday, July 28th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_picked_E..C2.A0coli_XL1-Blue_transformated_with_ligation_products_of_P126.2BPCR31.2C_P151.2BPCR34.2C_P123.2BPCR38_and_P123.2BPCR34"> Miniprep of picked <I>E. coli</I> XL1-Blue transformated with ligation products of P126+PCR31, P151+PCR34, P123+PCR38 and P123+PCR34 </SPAN></H3></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_July_29th"><B>Sunday, July 29th</B></SPAN></H2><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Picking_of_clones_from_PAL.2B_for_a_overnight_culture">Picking of clones from PAL+ for a overnight culture</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B>Inoculation of LB medium with clones picked from the transfomation of P183 (PAL+ (PCR32) in pYES (P175)) done on Monday, 23rd July in order to test on monday wheather the Ligation was successfull. 
<B>Operationale sequence:</B></P><UL><LI>Two clones were picked an transferred to 6 ml LB medium containing 1:1000 Ampicillin. 
</LI><LI>Incubation overnight at 37°C, 180 rpm.
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_8">Week 8</SPAN></H1><DIV class="week" id="WWeek_8"><H2><SPAN class="mw-headline" id="Monday.2C_July_30th"><B>Monday, July 30th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Extraction_of_Yeast_genomic_DNA_for_amplify_TEF2-P.2C_TEF1-T.2CADH1-T">Extraction of Yeast genomic DNA for amplify TEF2-P, TEF1-T,ADH1-T</SPAN></H3><P><B>Investigator</B>: Georg
</P><P><B>Aim of Experiment:</B> To get genomic DNA of yeast for amplify TEF2-P, TEF1-T,ADH1-T by PCR
</P><UL><LI> 100 µl of overnight culture of S. cerevisiae is centrifuged 5 min (13400 rpm)
</LI><LI> Remove supernatant and resuspend pellet in 100 µl 200 mM LiAC 1% SDS
</LI><LI> Incubate for 5 min at 70 C
</LI><LI> Precipitate DNA by adding 300µl 96% Ethanol and vortexing
</LI><LI> Sedimentation for 3 min (13400 rpm)
</LI><LI> Removal of supernatant and washing with 500 µl 70&amp; EtOH + centrifuge
</LI><LI> remove supernatant and solubilisate pellet in 100 ml 1x TE-buffer
</LI><LI> Remove cell-parts by 1 min of centrifugation and move supernatant in new Eppi
</LI><LI> Add 50 ml 7 M NH4AC-solution, let incubate at room-temperature for 5 min 
</LI><LI> Centrifuge for 10 min
</LI><LI> Move supernatant to a new eppi
</LI><LI> ADD 70 µl 7,5 M NH4OAc and 280 µl Isopropanol
</LI><LI> Incubate 10 min at room-temperature and centrifuge 10 min
</LI><LI> remove supernatant
</LI><LI> wash pellet with 100µl 80% Ethanol, centrifuge for 10 min
</LI><LI> remove supernatant
</LI><LI> dry pelleted DNA, and resolubilsate in 1x TE-buffer
</LI></UL><P>( Extraction had to be done twice)
</P></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_of_yeast_genomic_DNA_to_measuring_the_concentration">Nanodrop of yeast genomic DNA to measuring the concentration</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI>S.C1: 51,2 ng/µl
</LI><LI>S.C2: 24,6 ng/µl
</LI><LI>S.C3: 41,3 ng/µl
</LI><LI>S.C4: 45,9 ng/µl
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_plasmids_P221-P232"> Analytical digestion and gelelectrophoresis of the plasmids P221-P232 </SPAN></H3><P><B>Investigator: Saskia</B></P><P><B>Aim of the experiment:</B> Analytical digest of P221-P223 with NdeI and BamHI and of P224-P232 with XbaI and AgeI-HF and analytical gelelectrophoresis .
</P><P><B>Procedure:</B></P><UL><LI><B>Analytical restriction digest and gelelectrophoresis of P221-P223 with NdeI and BamHI</B></LI></UL><UL><LI> Reaction batch for each plasmid:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD>Tango Buffer 10x
</TD><TD>4 µl
</TD></TR><TR><TD>BamHI (NEB)
</TD><TD>0.5 µl
</TD></TR><TR><TD>NdeI (NEB)
</TD><TD>0.5 µl
</TD></TR><TR><TD>Plasmid DNA
</TD><TD>2.5 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>12.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD><B>20 µl</B></TD></TR></TBODY></TABLE><UL><LI> Incubation at 37 °C for 1 h 20 min.
</LI></UL><UL><LI> Analytical gelelectrophoresis  (1%) at 90 V for 40-45 min.
</LI></UL><UL><LI><B>Analytical restriction digest and gelelectrophoresis of P224-P232 with XbaI and AgeI-HF</B></LI><LI> Reaction batch for each plasmid:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD>NEBuffer4 10x
</TD><TD>2 µl
</TD></TR><TR><TD>BSA 100x
</TD><TD>0.2 µl
</TD></TR><TR><TD>XbaI (Fermentas)
</TD><TD>0.25 µl
</TD></TR><TR><TD>AgeI-HF (Fermentas)
</TD><TD>0.25 µl
</TD></TR><TR><TD>Plasmid DNA
</TD><TD>2.5 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>14.8 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD><B>20 µl</B></TD></TR></TBODY></TABLE><UL><LI> Incubation at 37 °C for 1 h 20 min.
</LI></UL><UL><LI> Analytical gelelectrophoresis  (1%) at 90 V for 40-45 min.
</LI></UL><UL><LI> Order of gel-pockets:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>P221
</TD><TD>P222
</TD><TD>P223
</TD><TD>100 bp ladder
</TD></TR><TR><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD></TR></TBODY></TABLE><UL><LI> Order of gel-pockets:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>P224
</TD><TD>P225
</TD><TD>P226
</TD><TD>P227
</TD><TD>P228
</TD><TD>P229
</TD><TD>P230
</TD><TD>P231
</TD><TD>P232
</TD><TD>100 bp ladder
</TD></TR><TR><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>?Questionable?
</TD><TD>?Questionable?
</TD><TD>?Questionable?
</TD></TR></TBODY></TABLE></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Transformation_of_ligation_products_of_PAL.2B_.28PCR32.29_and_APT.28P189.29_in_pTUM104_RFC25_.28P175.29into_E.coli_Xl1-Blue"> Transformation of ligation products of PAL+ (PCR32) and APT(P189) in pTUM104 RFC25 (P175)into <I>E.coli</I> Xl1-Blue</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B>Transformation of the ligation products of PAL+ (PCR32), APT (P189) and the  negative control in pTUM104 in XL1 Blue.
<B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 5 µl of the ligation products
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_and_control_digest_of_PAL.2B_in_pTUM104_picked_on_sunday_29th_July">Miniprep and control digest of PAL+ in pTUM104 picked on sunday 29th July</SPAN></H3><P>Investigator: Ingmar
</P><P><B>Aim of the experiment:</B> Test whether the ligation of PAL in pTUM104 was successful
</P><P><B>Miniprep</B></P><P>Using a Quiagen Miniprep kit a plasmid isolation of two picked clones of the PAL+ in pTUM104 RFC25 ligation was done. The miniprep product of the first clone was named P235, the one of the second clone P236. Both were used for the following control digest.
</P><P><B>control digest</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>PAL+
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (10U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (10U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>Tango (10x)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>expected bands:
</P><UL><LI>pYES:5795bp
</LI><LI>PAL: 2201bp
</LI></UL><P>The picture shows in both cases the two expected bands at 5795 bp an 2201 bp. Therefore the ligation was successfull.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_Schwab_plasmids_P40_.26_P42_from_E.coli_XL1_blue">Miniprep of Schwab plasmids P40 &amp; P42 from <I>E.coli</I> XL1 blue</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim of experiment:</B> Get plasmids carrying lavendula limonene synthase gene for subsequent site-directed mutagenesis.
</P><P>Plasmid concentrations:
</P><UL><LI> P40 clone 1: 155 ng/µl
</LI><LI> P40 clone 2: 114 ng/µl
</LI><LI> P42 clone 1: 40 ng/µl
</LI><LI> P42 clone 2: 32 ng/µl
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_extracted_Schwab_plasmids_carrying_Lavendula_LS">Analytical restriction digest of extracted Schwab plasmids carrying Lavendula LS</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim of experiment:</B> Check whether insert is OK.
</P><UL><LI>Digest of P40 clone 1 and clone 2
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>NcoI
</TD></TR><TR><TD>0.25 µl
</TD><TD>HindIII
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer Tango (10x)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>Digest of P42 clone 1 and clone 2
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.5 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.25 µl
</TD><TD>NotI
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer Orange (10x)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation for 1,5 h at 37°C.
</P><P>Clones renamed:
</P><UL><LI> P40 clone 1: now P241
</LI><LI> P40 clone 2: now P242
</LI><LI> P42 clone 1: now P243
</LI><LI> P42 clone 2: now P244
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_plasmids_containing_citrus_LS_.28repetition_of_digest_from_July.2C_27th.29">Analytical restriction digest of plasmids containing citrus LS (repetition of digest from July, 27th)</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim of experiment:</B> Check whether ligations PCR1/pTUM104new, P144(PCR2)/pTUM104new and PCR1/pSB1C3 have worked. This time, a standard protocol with 2,5 µl of DNA was used. Furthermore, Pst-1 HF was used instead of Spe1 (as on Friday, July 27th).
</P><UL><LI>Digest of plasmids PCR1/pTUM104new clone 1,3,5 and 6; P144(PCR2)/pTUM104 new clone 1,3,5 and 6; PCR1/pSB1C3 clone 1,5 and 6.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>Pst1-HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xba1
</TD></TR><TR><TD>2 µl
</TD><TD>NEB Buffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Incubation for 1,5 h at 37 °C.
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_July_31st"><B>Tuesday, July 31st</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_for_measuring_Plasmidconcentrations_of_P54.2CP55.2CP57.2CP87.2CP86.2CP88.2CP90.2CP91.2CP89">Nanodrop for measuring Plasmidconcentrations of P54,P55,P57,P87,P86,P88,P90,P91,P89</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> P54: 235 ng/µl
</LI><LI> P55: 645 ng/µl
</LI><LI> P57: 491,1 ng/µl
</LI></UL><UL><LI> P87: 259 ng/µl
</LI><LI> P86: 99 ng/µl
</LI><LI> P88: 86 ng/µl
</LI></UL><UL><LI> P90: 1161 ng/µl
</LI><LI> P91: 261 ng/µl
</LI><LI> P89: 630 ng/µl
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Prep._digestion_of_P90.2C_P87.2C_P55_and_prep._gelelectrophoresis">Prep. digestion of P90, P87, P55 and prep. gelelectrophoresis</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>3 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>12 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>1,2 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>40,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=60µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 20 µl digestive mastermix 20 µl of DNA was added and mixed
</LI><LI> Digestion was processed for 3 hours at 37 C
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_pTUM104_with_PvuII_and_SwaI"> Analytical digestion of pTUM104 with PvuII and SwaI</SPAN></H3><P><B> Investigator:</B> Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,25 µl
</TD><TD>SWAI (NEB)
</TD></TR><TR><TD>0,25 µl
</TD><TD>PvuII (NEB)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB-3 buffer
</TD></TR><TR><TD>0,2 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>14,85 µl
</TD><TD>dd H20
</TD></TR><TR><TD>= 17,5µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> After 1 h digestion with SwaI at room temperature, PvuII was added and digested at 37 C, 1 h
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_from_the_overnight_transformation_of_ligated_P123.2BPCR38_in_E._coli_XL1-Blue_from_July.2C_27th"> Picking from the overnight transformation of ligated P123+PCR38 in E. coli XL1-Blue from July, 27th  </SPAN></H3><P><B>Investigator:</B> Jeff, Georg
</P><P><B>Aim of the experiment:</B> Repeat picking from the overnight transformation of ligated P123+PCR38 in E. coli XL1-Blue because the last 3 picked colonies are negative.
</P><P><B>Procedure:</B></P><UL><LI> Colonies were picked with pipette tips and transformed into a new cell-culture tube with 4 ml of LB-medium and antibiotics and were put overnight in a 180 rpm cell culture shaker at 37 °C. P123+PCR38 (CamR).
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Quickchange_of_Schwab_plasmid_P40_clone_1">Quickchange of Schwab plasmid P40 clone 1</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim of experiment:</B> Remove Age1 restriction site in gene coding for lavendula limonene synthase.
</P><P><B>PCR</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>2 µl
</TD><TD>Plasmid P241
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O60 (10 pmol/µL)
</TD></TR><TR><TD>1 µl
</TD><TD>1:10 dilution of O61 (10 pmol/µL)
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>12
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 7 min
</TD></TR></TBODY></TABLE><UL><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B></P><UL><LI> thawing of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the SDM PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Kan-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Kan-LB-plate
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_digest_of_PCR1.2C_PCR2_with_XbaI_and_AgeI_.28additional_positive_control_P155.29">Preparative digest of PCR1, PCR2 with XbaI and AgeI (additional positive control P155)</SPAN></H3><P><B>Investigator: Andrea</B></P><P><B>Digestion of PCR1, PCR2 and P155 (pTUM104) with XbaI and AgeI</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>8 µl
</TD><TD>PCR1
</TD></TR><TR><TD>4 µl
</TD><TD>NEB Buffer
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (20 U/µl)
</TD></TR><TR><TD>25,6 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>PCR2
</TD></TR><TR><TD>4 µl
</TD><TD>NEB Buffer
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (20 U/µl)
</TD></TR><TR><TD>23,6 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>P155
</TD></TR><TR><TD>4 µl
</TD><TD>NEB Buffer
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (20 U/µl)
</TD></TR><TR><TD>23,6 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Mastermix of Buffer, BSA and Enzymes was added to the DNA and water. 
</P><P>Incubation: 37 °C, 3 h
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_gel_electrophoresis_of_PCR1.2FPCR2.2C_P155">Preparative gel electrophoresis of PCR1/PCR2, P155</SPAN></H3><P><B>Investigator:</B> Andrea
</P><UL><LI>PCR1 / PCR2: ca. 1600 bp
</LI><LI>P155: ca. 5800 bp
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_digested_PCR1_with_P133_.28pSB1C3.29_and_P175_.28pTUM104.29_.28XbaI_and_AgeI.29"> Ligation of digested PCR1 with P133 (pSB1C3) and P175 (pTUM104) (XbaI and AgeI)</SPAN></H3><P><B>Investigator: Andrea</B></P><P>Concentration (Nano Drop: 
LIMS Citrus (PCR 1) = ? ng/µl
LIMS Citrus (PCR 2) = ? ng/µl
P175 = 107,3 ng/µl
P133 = 41.8 ng/µl</P><P>
PCR1 and P175
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>P175
</TD></TR><TR><TD>14,27 µl
</TD><TD>PCR 1
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>1,73 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>
PCR1 and P133
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,4 µl
</TD><TD>P133
</TD></TR><TR><TD>14,27 µl
</TD><TD>PCR 1
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>0,33 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>
PCR2 and P175
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>P175
</TD></TR><TR><TD>11,99 µl
</TD><TD>PCR 2
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>4,01 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>
PCR2 and P133
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,4 µl
</TD><TD>P133
</TD></TR><TR><TD>11,99 µl
</TD><TD>PCR 2
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>2,61 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>
Negativ control P175
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>P175
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>
Negativ control P133
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,4 µl
</TD><TD>P175
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>2 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR><TR><TD>14,4 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>water bath 16 °C over night
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Media_and_plates_for_yeast_transformation"> Media and plates for yeast transformation  </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim of the experiment:</B> Producing media and plates for yeast transformation (according to pYes_manual_kommentiert)
</P><UL><LI> YPD-Medium, YPD-plates, 10X TE, 10X LiAc, 1x LiAc/0.5x TE
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_1st"><B>Wednesday, August 1st</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_picked_E._coli_XL1-Blue_transformated_with_ligation_products_of_P123.2BPCR38"> Miniprep of picked E. coli XL1-Blue transformated with ligation products of P123+PCR38 </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Miniprep of picked E. coli XL1-Blue transformated with ligation products of P123+PCR38.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol. (P251-P260)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_Miniprep_E._coli_XL1-Blue_transformated_with_ligation_products_of_P123.2BPCR38_and_of_pGBKT7_plasmid"> Analytical digestion and gelelectrophoresis of Miniprep E. coli XL1-Blue transformated with ligation products of P123+PCR38 and of pGBKT7 plasmid </SPAN></H3><P><B>Investigator:</B> Dennis, Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of Miniprep E. coli XL1-Blue transformated with ligation products of P123+PCR38 to see whether the ligation was successful and of pGBKT7 plasmid to see if the MfeI and BamHI restriction enzymes work.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for digestion with XbaI+AgeI-Hf of ligation products of P123+P38:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD>Buffer 4 (10x)
</TD><TD>22 µl
</TD></TR><TR><TD>BSA (100x) (NEB)
</TD><TD>2.2 µl
</TD></TR><TR><TD>XbaI (NEB)
</TD><TD>1.75 µl
</TD></TR><TR><TD>AgeI-Hf (NEB)
</TD><TD>1.75 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>162.8 µl
</TD></TR><TR><TD><B>TOTAL MASTERMIX</B></TD><TD><B>192.5 µl</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of Mastermix + 2.5 µl of the plasmid DNA (P251-P260)
</LI></UL><UL><LI> Composition for analytical restriction digestion of P99 with MfeI+BamHI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD>Buffer G (10x)
</TD><TD>2 µl
</TD></TR><TR><TD>MfeI (NEB)
</TD><TD>0.25 µl
</TD></TR><TR><TD>BamHI (NEB)
</TD><TD>0.25 µl
</TD></TR><TR><TD>Plasmid DNA (P99; pGBKT7)
</TD><TD>2.5 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>15 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD><B>20 µl</B></TD></TR></TBODY></TABLE><UL><LI> Analytic Gelelectrophoresis for 1 h at 90 V.
</LI></UL><UL><LI> Order of gel-pockets:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>P251
</TD><TD>P252
</TD><TD>P253
</TD><TD>P254
</TD><TD>P255
</TD><TD>P256
</TD><TD>P257
</TD><TD>P258
</TD><TD>P259
</TD><TD>P260
</TD><TD>100 bp ladder
</TD></TR><TR><TD>100 ladder
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>Corrupt
</TD><TD>1 kb ladder
</TD></TR></TBODY></TABLE><UL><LI> Order of gel-pockets:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder
</TD><TD>P99 control digestion with MfeI+BamHI
</TD><TD>P99 control undigested
</TD><TD>P123 control undigested
</TD><TD>100 bp ladder (accidently)
</TD></TR><TR><TD>Okay
</TD><TD>Okay
</TD><TD>Okay
</TD></TR></TBODY></TABLE><P>→MfeI and BamHI are working! (Expected DNA strands for P99 digestion at 731 bp, 2443 bp and 4129 bp; that's what we got.)
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophoresis_of_PCR37_with_XbaI.2BAgeI-HF"> Preperative digestion and gelelectrophoresis of PCR37 with XbaI+AgeI-HF </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR37
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>17.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Gel_extraction_of_preperative_gelelectrohphoresis_of_PCR37"> Gel extraction of preperative gelelectrohphoresis of PCR37 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B></P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_PCR39.2BP207_for_cloning_LexA_with_new_RFC25_pre-_and_suffix_into_a_plasmid"> Ligation of PCR39+P207 for cloning LexA with new RFC25 pre- and suffix into a plasmid </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P207 (digested plasmid DNA (P123) with XbaI and AgeI-HF)
</TD><TD>1.42 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR39 (digested insert DNA (PCR37) with XbaI and AgeI-HF)
</TD><TD>10.17 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>5.91 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_of_P80_to_introduce_RFC25_pre-_and_suffix_to_Pif3"> PCR of P80 to introduce RFC25 pre- and suffix to Pif3 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI><B>PCR reaction mixture</B></LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer O31 1:10 dilution (TUM12-LSPS-fw)
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer O32 1:10 dilution (TUM12-LSPS-rv )
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA P80
</TD></TR><TR><TD>35.75 µL
</TD><TD>ELGA Water
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme with following conditions
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>55 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>30 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_purification_of_PCR_of_P80"> PCR purification of PCR of P80 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B></P></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_pTUM104_with_SwaI_and_PvuII"> Preparative digestion of pTUM104 with SwaI and PvuII </SPAN></H3><UL><LI>Preparative digestion of pYesII
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume in µl</B></TD></TR><TR><TD>SwaI
</TD><TD>2 µl
</TD></TR><TR><TD>PvuII
</TD><TD>2 µl
</TD></TR><TR><TD>Neb3 buffer
</TD><TD>4 µl
</TD></TR><TR><TD>BSA
</TD><TD>0,4 µl
</TD></TR><TR><TD>H20
</TD><TD>31,6 µl
</TD></TR><TR><TD>Volume
</TD><TD>40 µl
</TD></TR></TBODY></TABLE><P>Geldigestion was performed with SwaI and BSA at room temperature for 2,5 hours. Afterwards PvuII was added and digested another 2,5  hours at 37 C
</P></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_Gel_of_pTUM104_digestion">Preparative Gel of pTUM104 digestion</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Preparative Digestion was loaded onto a 1% preparative agarose gel
</LI><LI> Gel was run for 150 min at 70 V
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_measurement_of_P245-P250"> Nanodrop measurement of P245-P250</SPAN></H3><P>Gelextraction was performed according to gelextraction protocol
</P><P>Probe:     Concentration:
P245       4,5 ng/µl 
P246       7,5 ng/µl
P247       22,7 ng/µl
P248       21,1 ng/µl
P249       3,3 ng/µl
P250       3,8 ng/µl
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_ligation_products_and_quickchange_products">Transformation of ligation products and quickchange products</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Transformation of ligation products (ligation July 31st) and quickchange products (repetition, quickchange of July 31st).
</P><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B></P><UL><LI> thawing of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the SDM PCR product or 5 µl of ligation product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Kan-LB-plate (Quickchange product), Amp-plate (pYES) or Chloramphenicol (pSB1C3)
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an antibiotic containing LB-plate
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_and_control_digest_of_APT_in_pTUM104">Miniprep and control digest of APT in pTUM104 </SPAN></H3><P>Investigator: Katrin
</P><P><B>Aim of the experiment:</B> Test whether the ligation of APT in pYES was successful
</P><P><B>Miniprep</B></P><P>Using a Quiagen Miniprep kit, a plasmid isolation of three picked clones of a APT in pYes2 RFC25 ligation was done. The miniprep products were named P261 (1st clone), P262 (2nd clone) and  P263 (3rd clone).All three of them were used for the following control digest.
</P><P>The resulting plamid-concentrations (measured with nanodrop) were
clone 1 (P261): 149,0 ng/µl
clone 2 (P262): 153,5 ng/µl
clone 3 (P263): 158,4 ng/µl
</P><P><B>control digest</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>APT (clones 1,2,3)
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal (10U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI (10U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>expected bands (pYes has 2 AgeI restriction site):
</P><UL><LI>pYES fragment 1:5398 bp
</LI><LI>pYes fragment 2: 451
</LI><LI>APT: 1244 bp 
</LI></UL><P>
The picture shows the expected bands for clones 1 and 3
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Media_and_plates_for_yeast_transformation_2"> Media and plates for yeast transformation  </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim of the experiment:</B> Producing media and plates for yeast transformation (according to pYes_manual_kommentiert)
</P><UL><LI> 20% glucose, 20x SC stock (without uracil and without yeast nitrogen base), SC plates with glucose
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_2nd"><B>Thursday, August 2nd</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_ligation_product_P207.2FPCR39">Transformation of ligation product P207/PCR39</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Transformation of ligation products P207/PCR39 and P207/NK.
</P><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B></P><P>Unfortunately the lab assistents went home so that the incubation times had to be shortened.
</P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 5 µl of the ligation products
</LI><LI> incubation on ice for 25 min
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubation at 37°C and 180 rpm for 25 min
</LI><LI> plating of 100 µl on an Chlp-LB-plate
</LI><LI> sedimenting the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspension of the sediment in 100 µl LB-medium and plating it as well on an Chlp-LB-plate
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_pTUM104_-f1.2C-Gal4"> Gelextraction of pTUM104 -f1,-Gal4 </SPAN></H3><P>pTUM104 extraction was performed according to manufacturers protocoll
</P><P>Concentration was measured with nanodrop
</P><P>Probe:    Concentration
P265       12,8 ng/µl
P264       12,9 ng/µl
</P></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Circularization_of_linearized_plasmid_backbone_pTUM104"> Circularization of linearized plasmid backbone pTUM104</SPAN></H3><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3,9 µl
</TD><TD>pYes2
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase
</TD></TR><TR><TD>5 µl
</TD><TD>T4 ligase buffer
</TD></TR><TR><TD>2,5 µl
</TD><TD>PEG 4000
</TD></TR><TR><TD>35,1 µl
</TD><TD>H20
</TD></TR><TR><TD>50 µl
</TD><TD>Volume
</TD></TR></TBODY></TABLE><P>Ligation was performed over night at 16 C
Negative control was done with H20 in exchange to pYES2
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Repetition_of_Quickchange_mutagenesis_of_lavendula_limonene_synthase">Repetition of Quickchange mutagenesis of lavendula limonene synthase</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> No colonies were observable after second transformation of quickchange PCR products. Therefore the quickchange pcr reaction was repeated with a modified protocol:
</P><P><B>PCR</B></P><P><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P242
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O60 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Turbo DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P242
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O61 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Turbo DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>68°C
</TD><TD> 7 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were once more applied.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B>! Unfortunately someone turned off the heat block, so that the reaction tube was below 37 °C (approx. 25 °C) for about 45 min. After recognition (after one hour of incubation), the heat block was turned on again and the reaction tube was incubated for another 30 min.</B></P><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation on ice for 25 min
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubation at 37°C and 180 rpm for 25 min
</LI><LI> plating of 100 µl on an Kan-LB-plate
</LI><LI> sedimenting the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspension of the sediment in 100 µl LB-medium and plating it as well on an Kan-LB-plate
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="PCR_of_C-LS3_.28BBa_I742111.2C_Trafo_16.6.29"> PCR of C-LS3 (BBa_I742111, Trafo 16.6) </SPAN></H3><P><B>Investigator: Lara</B></P><P><B>Aim:</B> To get more PCR product in case another preparative digest is needed.
</P><P><B>Primer with consensus sequence</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer O27
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer O30
</TD></TR><TR><TD>0.25 µl
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (BBa_I742111) Clone 3
</TD></TR><TR><TD>35.75 µl
</TD><TD>dd water
</TD></TR><TR><TD>50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Primer without consensus sequence</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer O28
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer O30
</TD></TR><TR><TD>0.25 µl
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (BBa_I742111) Clone 3
</TD></TR><TR><TD>35.75 µl
</TD><TD>dd water
</TD></TR><TR><TD>50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>47 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>1,75 min
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>1 h
</TD></TR></TBODY></TABLE><P><B>Analytical gel electrophoresis</B></P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparation_of_YPD_Medium_and_Picking_of_S._cerevisiae_clones">Preparation of YPD Medium and Picking of <I>S. cerevisiae</I> clones</SPAN></H3><P><B>Investigator:</B> Lara, Katrin
</P><P>YPD Yeast Extract Peptone Dextrose Medium (1 liter)
the following substances were dissolved in 1000 ml of water and then autoclaved
</P><UL><LI> 1% yeast extract
</LI><LI> 2% peptone
</LI><LI> 2% dextrose (D-glucose)
</LI></UL><P>Picking of two clones of S. cerevisiae DD (17.17.2012): The clones were put into YPD medium and incubated at 30 °C over night. Additional 0,8 g Glucose were added to one of the cell culture tubes.
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_3rd"><B>Friday, August 3rd</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_Xl1-blue_with_Vector_pGADT7-AD"> Transformation of <I>E.coli</I> Xl1-blue with Vector pGADT7-AD</SPAN></H3><P><B>Investigator:</B>Georg
</P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1 µl of pGADT7-AD (P98) was added to 100 µl of competent cells and mixed gently.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on ampicillin plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_Xl1-blue_with_pTUM104_overnight_ligation"> Transformation of <I>E.coli</I> Xl1-blue with pTUM104 overnight ligation</SPAN></H3><P><B>Investigator:</B>Georg
</P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1 µl of pGADT7-AD (P98) was added to 100 µl of competent cells and mixed gently.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on ampicillin plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_TEF1-P_and_ADH1-P_with_XbaI_and_PstI">Preparative digestion of TEF1-P and ADH1-P with XbaI and PstI</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>3 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>12 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>1,2 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>40,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=60µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 20 µl Mastermix and 20 µl DNA were mixed and digestion processed at 37 C for 3 hours.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="PCR_of_ADH1-T.2C_TEF2-P.2C_TEF1-T_from_Saccharomyces_cerv._genome"> PCR of ADH1-T, TEF2-P, TEF1-T from Saccharomyces cerv. genome</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Reaction batch for PCR-ADH1-T:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10µl
</TD><TD>5x One taq buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM DNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>O66/10
</TD></TR><TR><TD>1µl
</TD><TD>O67/10
</TD></TR><TR><TD>0,25
</TD><TD>dd H20
</TD></TR><TR><TD>=35,75µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for PCR-TEF2-P:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1,23 ml
</TD><TD>P237
</TD></TR><TR><TD>10µl
</TD><TD>5x One taq buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM DNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>O64/10
</TD></TR><TR><TD>1µl
</TD><TD>O65/10
</TD></TR><TR><TD>0,25
</TD><TD>dd H20
</TD></TR><TR><TD>=35,52µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for PCR-TEF2-P:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1,23 ml
</TD><TD>P237
</TD></TR><TR><TD>10µl
</TD><TD>5x One taq buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM DNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>O64/10
</TD></TR><TR><TD>1µl
</TD><TD>O65/10
</TD></TR><TR><TD>0,25
</TD><TD>dd H20
</TD></TR><TR><TD>=35,75µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for PCR-TEF1-T:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1,23 ml
</TD><TD>P237
</TD></TR><TR><TD>10µl
</TD><TD>5x One taq buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM DNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>O62/10
</TD></TR><TR><TD>1µl
</TD><TD>O63/10
</TD></TR><TR><TD>0,25
</TD><TD>dd H20
</TD></TR><TR><TD>=35,75µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Cycling condition were used as follows:
</LI><LI> 94 C 30 sek
</LI><LI> (30 Cycles) 94 C 30 sek
</LI><LI> (30 Cycles) 56 C 60 sek
</LI><LI> (30 Cycles) 68 C 2 min
</LI><LI> Final extension: 68 C
</LI><LI> Hold: 4 C
</LI><LI> PCR was successful
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_XL1_blue_with_ligation_product_P207.2FPCR39">Transformation of <I>E.coli</I> XL1 blue with ligation product P207/PCR39</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Incubation times of transformation of August 2nd had to be shortened because the lab employees left. Therefore the transformation didn't work properly. --&gt; Repetition of the transformation of <I>E.coli</I> XL1 blue with ligation products P207/PCR39 and P207/NK.
</P><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 5 µl of the ligation products
</LI><LI> incubation on ice for 30 min
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubation at 37°C and 180 rpm for 45 min
</LI><LI> plating of 100 µl on an Chlp-LB-plate
</LI><LI> sedimenting the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspension of the sediment in 100 µl LB-medium and plating it as well on an Chlp-LB-plate
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_quickchange_products">Analytical restriction digest of quickchange products</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Check whether site directed mutagenesis to eliminate Age1 restriction site has worked.
</P><P><B>Quickchange PCR product purification</B>:
The Quickchange PCR products were purified with Qiagen PCR Purification Kit before analytical restriction digest:
</P><UL><LI> P241 SDM: 7 ng/µl
</LI><LI> P242 SDM: 63 ng/µl
</LI></UL><P><B>Restriction digest of p241 SDM (31.7.), p242 SDM (2.8.) and p242 (as positive control)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>Age1-HF
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>2 µl
</TD><TD>BSA (10x)
</TD></TR><TR><TD>13.25 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Analytical Gel</B></P><P>1. Standard (1 kb Gene Ruler)
</P><P>2. p242, digested with Age1 (positive control)
</P><P>3. p241 SDM
</P><P>4. p241 SDM, digested with Age1 
</P><P>5. p242 SDM
</P><P>6. p242 SDM, digested with Age1
</P><P>7. PCR 42
</P><P>8. PCR 43
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_XL1_blue_with_P242_SDM">Transformation of <I>E.coli</I> XL1 blue with P242 SDM</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Incubation times of transformation of August 2nd had to be shortened because the lab employees left. Therefore the transformation didn't work properly. --&gt; Repetition of the transformation with P242 SDM.
</P><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> two transformations: 1. addition of 1 µl of the purified Quickchange PCR product; 2. addition of 4 µl of the purified Quickchange PCR product
</LI><LI> incubation on ice for 30 min
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubation at 37°C and 180 rpm for 45 min
</LI><LI> plating of 100 µl on an Kan-LB-plate
</LI><LI> sedimenting the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspension of the sediment in 100 µl LB-medium and plating it as well on an Kan-LB-plate
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Repetition_of_small-Scale_Yeast_Transformation_of_4CL.2C_CHS.2C_OMT.2C_PAL_and_APT_in_pTUM104">Repetition of small-Scale Yeast Transformation of 4CL, CHS, OMT, PAL and APT in pTUM104</SPAN></H3><P><B>Investigator:</B> Mary
</P><P><B>Aim of the experiment:</B>Transformation of P193 - P198, P200, P236, P263 and GFP (as a positive control - from simon) in Yeast 
</P><P>The protocol on page 13 of the pYES2 manual was used.
</P><P>Only modifications are noted:
</P><P>step 1: inoculate in 4 ml YPD medium
</P><P>step 2: OD600 = 9 -&gt; to determine a OD600 of 0.4 in 50 ml, 2.2 ml yeast suspension and 42.5 ml YPD medium were used, 5ml 20% Glucose was added (wrong information about the YPD-medium: we suggested that Glucose caramelizes when autoclaved together with medium - but it is not the case! Only agar and glucose will caramelize). 
</P><P>steps 3 and 4: centrifugation for 5 min, 4 °C
</P><P>step 6: 
</P><P>1 µg plasmid DNA
</P><UL><LI>P193: c = 110 ng/µl -&gt; 9.1 µl 
</LI><LI>P194: c = 342 ng/µl -&gt; 2.9 µl
</LI><LI>P195: c = 423 ng/µl -&gt; 2.4 µl  
</LI><LI>P196: c = 92 ng/µl -&gt; 10.8 µl 
</LI><LI>P197: c = 394 ng/µl -&gt; 2.5 µl
</LI><LI>P198: c = 183 ng/µl -&gt; 5.5 µl  
</LI><LI>P200: c = 464 ng/µl -&gt; 2.2 µl
</LI><LI>P236: c = 291 ng/µl -&gt; 3.4 µl
</LI><LI>P263: c = 158 ng/µl -&gt; 6.3 µl
</LI><LI>Positive control: 2.19/4 (pYES2+EGFP) c = 200 ng/µl -&gt; 5µl
</LI></UL><P>100 µg denatured sheared salmon sperm: the one from Simon was used -&gt; 10 µl
</P><P>step 7: 
</P><P>7 ml 50% PEG3350 was synthesized (3.5 g in 7 ml bidest.water)
6.4 ml 50% PEG3350 + 800 µl 10x LiAc + 800 µl 10x TE were mixed
</P><P>The cells were plated out on SC-U plates with Glucose and incubated the weekend at 30°C
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_XL1_blue_with_ligation_products">Transformation of <I>E.coli</I> XL1 blue with ligation products</SPAN></H3><P>Investigator: Katrin
</P><P>Aim: Transformation of ligation products PCR2/P133, PCR1/P133, MK/P133, PCR2/P175, MK/P175
</P><P>Transformation into <I>E.coli</I> Xl1-Blue
</P><UL><LI> thawing of 100 µl Ca-competent E.coli XL1-Blue cells on ice
</LI><LI> of 1 µl of the SDM PCR product or 5 µl of ligation product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 45 min
</LI><LI> plate 100 µl on an Amp-plate (pYES (P175) or Chloramphenicol-plate (pSB1C3, P133)
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an antibiotic containing LB-plate 
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_August_5th"><B>Sunday, August 5th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_of_pGBKT7_.28P99.29_to_introduce_RFC_pre-_and_suffix_to_Gal4_DNA_binding_domain_.28Gal4DBD.2FGal4BD.29"> PCR of pGBKT7 (P99) to introduce RFC pre- and suffix to Gal4 DNA binding domain (Gal4DBD/Gal4BD) </SPAN></H3><P><B>Investigator</B>: Jeff
</P><P><B>Aim of the experiment:</B>PCR of pGBKT7 (P99) to introduce RFC pre- and suffix to Gal4 DNA binding domain (Gal4DBD/Gal4BD) for fusion protein construction of N'-PhyB(N-Terminus)-20aaLinker-GalDBD-C'. (If there is time one can direcly clone PhyB in the C-Terminus of Gal4DBD directly.)
</P><P><B>Procedure:</B></P><UL><LI> PCR with primer: TUM12-Gal4BD-fw and TUM12-Gal4BD-rv
</LI></UL><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>100 µM Forward Primer O73 (TUM12-Gal4BD-fw)
</TD></TR><TR><TD>1 µl
</TD><TD>100 µM Reverse Primer O74 (TUM12-Gal4BD-rv)
</TD></TR><TR><TD>0.25 µL
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µL)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA P99
</TD></TR><TR><TD>35.75 µL
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme with following conditions
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>57 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>30 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transfer_of_old_transformed_E..C2.A0coli_XL-1_blue_from_old_antibiotic_plates_to_new_antibiotic_plates_and_of_old_Y190_S..C2.A0cerevisiae_strain_to_a_new_YPD_plates"> Transfer of old transformed <I>E. coli</I> XL-1 blue from old antibiotic plates to new antibiotic plates and of old Y190 <I>S. cerevisiae</I> strain to a new YPD plates </SPAN></H3><P><B>Investigator</B>: Jeff
</P><P><B>Aim of the experiment:</B> Transfer of old transformed <I>E. coli</I> XL-1 blue from old antibiotic plates to new antibiotic plates and of old Y190 <I>S. cerevisiae</I> strain to a new YPD plates.
</P><P><B>Procedure:</B></P><UL><LI> Transformed <I>E. coli</I> XL1-Blue: With an incolution loop under sterile conditions the bacteria colonies from the plates were transferred by dilution plating on new antibiotic plates. Attempts were made to take all the colonies to have a multiclonal plate.
</LI></UL><UL><LI> The <I>E. coli</I> XL1-Blue plates from these transformations were taken:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TH colspan="4"><B>Parts for Phycocyanobilin synthesis:</B></TH></TR><TR><TD><I><B>BBa_I15008</B></I> in pSB2K3 (KanR): Heme oxygenase
</TD></TR><TR><TH colspan="4"><B>Parts for N'-SV40NLS-PhyB(NT)-20aaLinker-LexA-C' and N'-PhyB(NT)-20aaLinker-Gal4DBD-C'/N'-SV40NLS-PhyB(NT)-20aaLinker-Gal4DBD-C' construction:</B></TH></TR><TR><TD><I><B>BBa_K207000</B></I> in pSB3K3 (KanR): PhyB(621NT)-GalDBD
</TD><TD><I><B>BBa_K207001</B></I> in pSB1A2 (AmpR): PhyB(621NT)
</TD><TD><I><B>BBa_K105005</B></I> in pSB2K3 (KanR): LexA DNA binding protein
</TD><TD><I><B>pGBKT7</B></I> (KanR): Yeast-two-hybrid vector with C-terminal MCS after Gal4 DNA binding domain (Gal4DBD/Gal4BD)
</TD></TR><TR><TH colspan="4"><B>Parts for N'-Gal4AD-linker-Pif3 construction:</B></TH></TR><TR><TD><I><B>BBa_K365000</B></I> in pSB1C3 (CamR): Pif3(100NT)
</TD><TD><I><B>pGADT7 AD</B></I> (AmpR): Yeast-two-hybrid vector with C-terminal MCS after Gal4 transcription activation domain (Gal4AD)
</TD></TR><TR><TH colspan="4"><B>Oligopeptide linkers for linkage of two functional independent protein domains:</B></TH></TR><TR><TD><I><B>BBa_K365005</B></I> in pSB1C3 (CamR): 20 amino acids long linker with RFC25 pre- and suffix
</TD><TD><I><B>BBa_K243006</B></I> in BBa_K157000 (AmpR): 12 amino acids long linker (Gly-Gly-Ser-Gly)x3
</TD></TR><TR><TH colspan="4"><B>Synthetical promoter constructs for LexA based Y2H:</B></TH></TR><TR><TD><I><B>BBa_K165055</B></I> in BBa_J63009 (AmpR): LexA binding sites + mCYC + Kozak + YFPx2 + ADH1 terminator
</TD><TD><I><B>BBa_K165031</B></I> in pSB1AK3 (AmpR, KanR): LexA binding sites + mCYC
</TD></TR></TBODY></TABLE><UL><LI> Y190 <I>S. cerevisiae</I>: With an incolution loop under sterile conditions the yeast colonies from the plates were transferred by dilution plating on new YPD plates. Attempts were made to take all the colonies to have a multiclonal plate.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TH colspan="13"><B>Genotype:</B></TH></TR><TR><TD>MATa
</TD><TD>ura3-52
</TD><TD>his3-200
</TD><TD>ade2-101
</TD><TD>lys2-801
</TD><TD>trp1-901
</TD><TD>leu2-3, 112
</TD><TD>gal4Δ</TD><TD>gal80Δ</TD><TD>cyhR2
</TD><TD>LYS2 : : GAL1(UAS)-HIS3(TATA)-HIS3
</TD><TD>MEL1
</TD><TD>URA3 : : GAL1UAS-GAL1TATA-lacZ
</TD></TR><TR><TH colspan="13"><B>Reporters:</B></TH></TR><TR><TD>HIS3
</TD><TD>lacZ
</TD><TD>MEL1
</TD></TR><TR><TH colspan="13"><B>Transformation markers</B></TH></TR><TR><TD>trp1
</TD><TD>leu2
</TD><TD>cyhR2
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_from_the_overnight_transformation_of_ligated_P207.2BPCR39_product_in_E..C2.A0coli_XL1-Blue"> Picking from the overnight transformation of ligated P207+PCR39 product in <I>E. coli</I> XL1-Blue </SPAN></H3><P><B>Investigator</B>: Jeff
</P><P><B>Aim of the experiment:</B> Picking from the overnight transformation of ligated P207+PCR39 product in <I>E. coli</I> XL1-Blue from the August, 2nd and August, 3rd.
</P><P><B>Procedure:</B></P><UL><LI> 4 colonies from August, 2nd and 4 colonies from August, 3rd were taken. Total: 8 colonies.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and antibiotics. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C. P207+PCR39 (CamR).
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_with_BBa_K165055"> Picking of <I>E. coli</I> XL1-Blue transformed with BBa_K165055 </SPAN></H3><P><B>Investigator</B>: Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformed with BBa_K165055.
</P><P><B>Procedure:</B></P><UL><LI> 2 colonies were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and antibiotics. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C. BBa_K165055 in BBa_J63009 (AmpR).
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformed_with_pGADT7_AD"> Picking of <I>E. coli</I> XL1-Blue transformed with pGADT7 AD </SPAN></H3><P><B>Investigator</B>: Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformed with pGADT7 AD.
</P><P><B>Procedure:</B></P><UL><LI> 2 colonies were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and antibiotics. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C. pGADT7 AD (AmpR).
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_clones_of_transformation_August.2C_1st.2C_2nd_and_3rd">Picking of clones of transformation August, 1st, 2nd and 3rd</SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim:</B> Picking.
</P><P><B>Procedure:</B></P><UL><LI> 3 or 4 clones were picked for each ligation. Clones containing pYES2-new were put into 4 ml LB with 4 µl of ampillicin stock solution, clones containing pSB1C3 were put into chloramphenicol containing media. Incubation at 37 °C overnight.
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_9">Week 9</SPAN></H1><DIV class="week" id="WWeek_9"><H2><SPAN class="mw-headline" id="Monday.2C_August_6th"><B>Monday, August 6th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_transformated_E._coli_XL1-Blue_with_with_ligated_P207.2BPCR39"> Miniprep of overnight culture of transformated <I>E. coli</I> XL1-Blue with with ligated P207+PCR39  </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of transformated E. coli XL1-Blue with with ligated P207+PCR39 
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done after manufacturer's protocol for 8 picked colonies. (P285-P292)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_miniprep_P285-P292"> Analytical digestion and gelelectrophoresis of miniprep P285-P292  </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Analytical digestion with NgoMIV+PstI-HF and gelelectrophoresis of miniprep P285-P292.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for NgoMIV+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>22 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2.75 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>2.75 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>165 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=192.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,5 µl
</TD><TD>Plasmid-DNA (P285-P292)
</TD></TR><TR><TD>17,5 µl
</TD><TD>Mastermix for NgoMIV+PstI-HF
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P285</B></TD><TD><B>P286</B></TD><TD><B>P287</B></TD><TD><B>P288</B></TD><TD><B>P289</B></TD><TD><B>P290</B></TD><TD><B>P291</B></TD><TD><B>P292</B></TD><TD>P295
</TD><TD>P296
</TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Corrupt
</TD><TD><B>Ligation successful</B></TD><TD><B>Ligation successful</B></TD><TD><B>Ligation successful</B></TD><TD><B>Ligation successful</B></TD><TD><B>Ligation successful</B></TD><TD><B>Ligation successful</B></TD><TD><B>Ligation successful</B></TD><TD>please see next subsection
</TD><TD>plase see next subsection
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_miniprep_P295.2C_P296"> Analytical digestion and gelelectrophoresis of miniprep P295, P296  </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Analytical digestion with XbaI+PstI-HF and gelelectrophoresis of miniprep P295 and P296.
</P><P><B>Procedure:</B></P><UL><LI> Digestion reaction batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid-DNA (P295, P296)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD>P285
</TD><TD>P286
</TD><TD>P287
</TD><TD>P288
</TD><TD>P289
</TD><TD>P290
</TD><TD>P291
</TD><TD>P292
</TD><TD><B>P295</B></TD><TD><B>P296</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Please see last subsection
</TD><TD>Please see last subsection
</TD><TD>Please see last subsection
</TD><TD>Please see last subsection
</TD><TD>Please see last subsection
</TD><TD>Please see last subsection
</TD><TD>Please see last subsection
</TD><TD>Please see last subsection
</TD><TD><B>corrupt</B></TD><TD><B>corrupt</B></TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_miniprep_P293.2C_P294"> Analytical digestion and gelelectrophoresis of miniprep P293, P294  </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of miniprep P2P93, P294 to see whether the enzymes EcoRI and BamHI are working. Because the size of the insert is too short, the backbone is additionally cut with Eco91I (cuts only 1x in the backbone). So digestion were done with EcoRI+Eco91I and BamHI+Eco91I.
</P><P><B>Procedure</B></P><UL><LI> Reaction batch for P293:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid-DNA (P293)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI (Fermentas)
</TD></TR><TR><TD>0.25 µl
</TD><TD>Eco91I (Fermentas)
</TD></TR><TR><TD>2 µl
</TD><TD>Buffer 0 (Fermentas)
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for P294:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid-DNA (P293)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BamHI (Fermentas)
</TD></TR><TR><TD>0.25 µl
</TD><TD>Eco91I (Fermentas)
</TD></TR><TR><TD>4 µl
</TD><TD>Tango-Buffer (10x) (Fermentas)
</TD></TR><TR><TD>12.75 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P293 (digested with EcoRI+Eco91I)</B></TD><TD><B>P294 (digested with BamHI+Eco91I)</B></TD><TD>PCR48
</TD><TD>PCR49
</TD><TD>P230
</TD><TD>P231
</TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD><B>2 DNA strands are visible and have expected length</B></TD><TD><B>2 DNA strands are visible and have expected length</B></TD><TD>Please see next subsection
</TD><TD>Please see next subsection
</TD><TD>Please see next subsection
</TD><TD>Please see next subsection
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_miniprep_P230_and_P232"> Analytical digestion and gelelectrophoresis of miniprep P230 and P232 </SPAN></H3><P><B>Investigator: Jeff, Dennis</B></P><P><B>Aim of the experiment</B>: Analytical digestion and gelelectrophoresis of miniprep P230 and P232 to check whether the oligohybrdization product PCR34 has been successful ligated to pSB1C3 with RFC25 pre- and suffix.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for P230 and P231:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,5 µl
</TD><TD>Plasmid-DNA (P285-P292)
</TD></TR><TR><TD>17,5 µl
</TD><TD>Mastermix for NgoMIV+PstI-HF (please see subsection: &quot;Analytical digestion and gelelectrophoresis of miniprep P285-P292 &quot;)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD>P293
</TD><TD>P294
</TD><TD>PCR48
</TD><TD>PCR49
</TD><TD><B>P230</B></TD><TD><B>P231</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Please see next subsection
</TD><TD>Please see next subsection
</TD><TD>Please see next subsection
</TD><TD>Please see next subsection
</TD><TD><B>91 bp long insert seems to be visible after manual gain of signal</B></TD><TD><B>91 bp long insert seems to be visible after manual gain of signal</B></TD></TR></TBODY></TABLE><UL><LI> To be sure, one has to sequence this with the iGEM primer VF2 (O68) and VR (O69).
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_PCR48_and_PCR49"> Analytical gelelectrophoresis of PCR48 and PCR49 </SPAN></H3><P><B>Investigator: Jeff, Dennis</B></P><P><B>Aim of the experiment</B>: Analytical gelelectrophoresis of PCR48 and PCR49 to check whether the PCR has been successful.
</P><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD>P293
</TD><TD>P294
</TD><TD><B>PCR48</B></TD><TD><B>PCR49</B></TD><TD>P230
</TD><TD>P231
</TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Please see next subsection
</TD><TD>Please see next subsection
</TD><TD><B>PCR was successful</B></TD><TD><B>PCR was successful</B></TD><TD>Please see next subsection
</TD><TD>Please see next subsection
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_P289"> Preperative digestion of P289 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preperative digestion and gelelectrophoresis of P289 in order to fuse insert (LexA with RFC25 pre- and suffix) into the N-terminus of 20aa linker in pSB1C3 (P206). Intermediate construction: N'-20aaLinker-LexA-C'. Final construction should be N'-SV40NLS-PhyB(642NT/908NT)-20aaLinker-LexA-C'.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid 289
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>11 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Preperative gelelectrophoresis was done the next day
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Extraction_of_ADH1-Promoter.2C_CYC1-Terminator.2C_TEF1-Promoter"> Extraction of ADH1-Promoter, CYC1-Terminator, TEF1-Promoter </SPAN></H3><P><B>Investigator:</B> Georg
</P><P><B>Aim of the experiment:</B> Miniprep of overnight cultures of transformed <I>E. coli</I> XL1-Blue
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done after manufactuer's protocol for 3x3 colonies.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="PCR_of_TEF1-Terminator.2C_TEF2-Promoter_and_ADH1-Terminator_with_PCR-purification"> PCR of TEF1-Terminator, TEF2-Promoter and ADH1-Terminator with PCR-purification </SPAN></H3><P><B>Investigator:</B> Georg
</P><P><B>Aim of the experiment:</B> Amplification of TEF1-Terminator, TEF2-Promoter from genomic DNA of <I>Saccharomyces cerevisiae</I> and ADH1-Terminator from yeast Vector PGADT7 AD
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer
</TD></TR><TR><TD>0.25 µl
</TD><TD>OneTaq Hot Start DNA Polymerase (Final: 1.25 units/50 µl)
</TD></TR><TR><TD>1,3µl P237
</TD><TD>1µl P98
</TD></TR><TR><TD>33.75 µl
</TD><TD>dd water
</TD></TR><TR><TD>50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>56 °C
</TD><TD>1 min
</TD></TR><TR><TD>68 °C
</TD><TD>2 min
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>1 h
</TD></TR></TBODY></TABLE><UL><LI>Afterwards, the PCR-product was purified by Quiaquick PCR purification KIt
</LI><LI>The success of the PCR was tested by analytical gelelectrophoresis
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_PCR_products_and_TEF1-.2CADH1-Promoter_with_XbaI_and_PstI"> Preparative digestion of PCR products and TEF1-,ADH1-Promoter with XbaI and PstI </SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Mastermix for XbaI+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>3 µl
</TD><TD>XbaI
</TD></TR><TR><TD>3 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>1,2 µl
</TD><TD>BSA 100x
</TD></TR><TR><TD>20,8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mastermix for XbaI+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>3 µl
</TD><TD>XbaI
</TD></TR><TR><TD>3 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>1,5 µl
</TD><TD>BSA 100x
</TD></TR><TR><TD>27,5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_lavendula_LS_and_citrus_LS_with_pSB1C3_and_pTUM104">Ligation of lavendula LS and citrus LS with pSB1C3 and pTUM104</SPAN></H3><P>Investigator: Andrea
</P><P><B>Procedure</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>1 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 42 RL
</TD><TD>0.8 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>15.7 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_Ligation_of_PCR1_and_PCR2_in_P133_or_P175">Miniprep of Ligation of PCR1 and PCR2 in P133 or P175 </SPAN></H3><P>Investigator: Andrea
</P><P>Using a Quiagen Miniprep kit. The miniprep products were named P266-P284. 
</P><P>The resulting plamid-concentrations (measured with nanodrop) were
</P><UL><LI>P 266: 1,3 ng/µl
</LI><LI>P 267: 0,2 ng/µl
</LI><LI>P 268: 1,0 ng/µl
</LI><LI>P 269: 13,5 ng/µl
</LI><LI>P 270: 10,3 ng/µl
</LI><LI>P 271: 12,1 ng/µl
</LI><LI>P 272: 14,7 ng/µl
</LI><LI>P 273: 12,8 ng/µl
</LI><LI>P 274: 13,2 ng/µl
</LI><LI>P 275: 9,4 ng/µl
</LI><LI>P 276: 8,5 ng/µl
</LI><LI>P 277: 12,4 ng/µl
</LI><LI>P 278: 10,2 ng/µl
</LI><LI>P 279: 7,1 ng/µl
</LI><LI>P 280: 5,7 ng/µl
</LI><LI>P 281: 8,0 ng/µl
</LI><LI>P 282: 8,6 ng/µl
</LI><LI>P 283: 31,4 ng/µl
</LI><LI>P 284: 12,6 ng/µl
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="PCR_of_Schwab_plasmid_Quickchange-DNA_to_amplify_gene_for_lavendula_LS"> PCR of Schwab plasmid Quickchange-DNA to amplify gene for lavendula LS</SPAN></H3><P><B>Instructor:</B> Andrea 
</P><P><B>Aim:</B> PCR of Schwab plasmids after Quickchange-Mutagenesis with primers which were designed to amplify the lavendula LS gene and to add RFC25 restriction sites.
</P><P>3 different primer combinations were used:
</P><P>1. O33/O37
</P><P>2. O34/O37
</P><P>3. O35/O37
</P><P>Each primer combination was used for plasmid DNA amplification of P283.
</P><P><B>PCR reaction mixture</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer
</TD></TR><TR><TD>0.25 µl
</TD><TD>OneTaq Hot Start DNA Polymerase (Final: 1.25 units/50 µl)
</TD></TR><TR><TD>3 µl
</TD><TD>Plasmid DNA P283
</TD></TR><TR><TD>33.75 µl
</TD><TD>dd water
</TD></TR><TR><TD>50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>50 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>1,75 min
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>1 h
</TD></TR></TBODY></TABLE></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparation_of_SC-U_medium_with_2.25_glucose_.28300ml.29_or_galactose_.28700_ml.29_for_expression_in_Saccharomyces_cerevisiae">Preparation of SC-U medium with 2% glucose (300ml) or galactose (700 ml) for expression in <I>Saccharomyces cerevisiae</I></SPAN></H3><P><B>Investigator:</B> Daniela
</P><P>1 l medium:
</P><UL><LI>One alliquot (50 ml) of amino acis prepared by Alois Bräuer was used
</LI><LI>6.7 g yeast nitrogen base
</LI><LI>850 ml ELGA water
</LI><LI>after dissolving, the solution was divided into 270 ml and 630 ml
</LI><LI>autoclaving
</LI></UL><P>Sugar solution were prepared separately:
</P><UL><LI>6 g glucose were dissolved in 30 ml ELGA water
</LI><LI>14 g galactose were dissolved in 70 ml ELGA water
</LI><LI>autoclaving
</LI></UL><P>After autoclaving:
</P><UL><LI>30 ml of glucose solution were added to 270 ml SC-U medium
</LI><LI>70 ml of galactose solution were added to 630 ml SC-U medium
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Inoculation_of_Saccharomyces_cerevisiae_colonies_of_transformation_products_from_August.2C_3rd">Inoculation of <I>Saccharomyces cerevisiae</I> colonies of transformation products from August, 3rd</SPAN></H3><P><B>Investigator:</B> Daniela
</P><P><B>Aim of the experiment:</B> First step of the expression protocol in <I>Saccharomyces cerevisiae</I></P><P>Inoculation of a single colony of <I>Saccaromyces cerevisiae</I> transformed with enzymes PAL+/-, 4CL+/-, CHS+/-, OMT+/-, APT and the positive control GFP in pTUM104, respectively in 15 ml of SC-U medium with 2% glucose.
Incubation at 30°C over night.
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_August_7th"><B>Tuesday, August 7th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Plating_of_received_E..C2.A0coli_containing_biobricks_BBa_K181000.2C_BBa_K365002_and_BBa_K365003"> Plating of received <I>E. coli</I> containing biobricks BBa_K181000, BBa_K365002 and BBa_K365003 </SPAN></H3><P><B>Investigator: Jeff</B></P><P><B>Aim of the experiment:</B> The received biobricks were already transformed in <I>E. coli</I> and were in an agar stabs. These <I>E. coli</I> cells were transferred with an inoculation loop on antibiotic selection plates and were incubated over night. The received biobricks were: BBa_K181000 (This is the PcyA gene, known to convert biliverdin to phycocyanobilin, codon optimized for S. cerevisiae (baker's yeast). ), BBa_K365002 (This part consists in the first 908 amino-acids of Phytochrome B from A. thaliana.) and BBa_K365003 (This part consists in the first 642 amino-acids of Phytochrome B from A. thaliana.).
</P><P><B>Procedure:</B></P><UL><LI> Bacterias containing plasmids with biobricks were transferred with a sterile inoculation loop on antibiotic plates and were incubated at 37 °C overnight.
</LI></UL><UL><LI> The biobricks were: BBa_K181000 in BBa_J63010 (AmpR), BBa_K365002 in pSB1C3 (CamR) and BBa_K365003 in pSB1C3.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_gelelectrophrosis_of_digested_P289"> Preperative gelelectrophrosis of digested P289 </SPAN></H3><P><B>Investigator</B>: Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Preperative digestion and gelelectrophoresis of P289 in order to fuse insert (LexA with RFC25 pre- and suffix) into the N-terminus of 20aa linker in pSB1C3 (P206). Intermediate construction: N'-20aaLinker-LexA-C'. Final construction should be N'-SV40NLS-PhyB(642NT/908NT)-20aaLinker-LexA-C'.
</P><P><B>Procedure:</B></P><UL><LI> Gelelectorphoresis was performed on a 1% low-melting agarose gel at 70  for 1.5 h.
</LI></UL><UL><LI> DNA strand at about 600-700 bp was cut out
</LI></UL><UL><LI> Cut out band was purified with gel extraction kit from Qiagen after manufacturer's protocol.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P206.2BP310"> Ligation of P206+P310 </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Ligation of P206+P310
</P><P><B>Procuedure:</B>
Reaction batch
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P206
</TD><TD>1.7 µl (~100 ng vector DNA)
</TD></TR><TR><TD>P310
</TD><TD>8.5 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>7.3 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophrosis_of_P123_and_P80"> Preperative digestion and gelelectrophrosis of P123 and P80 </SPAN></H3><P><B>Investigator</B>: Jeff
</P><P><B>Aim of the experiment:</B> Preperative digestion and gelelectrophoresis of P80 and P123 in order to fuse insert (Pif3 with RFC25 pre- and suffix) of P80 into the C-terminus of 20aa linker in pSB1C3 (P123). Intermediate construction: N'-Pif3-20aaLinker-C'. Final construction should be N'-Pif3-20aaLinker-Gal4AD-C'.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for P123:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid-DNA P123
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for P80:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid-DNA P80
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.44 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Gelelectorphoresis was performed on a 1% low-melting agarose gel at 70  for 1.5 h.
</LI></UL><UL><LI> DNA strand at about 300 bp from P80 was cut out. And the band from P123 at about 2.1 kbp was cut out.
</LI></UL><UL><LI> Cut out bands were purified with gel extraction kit from Qiagen after manufacturer's protocol.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P80 digested with EcoRI-HF+AgeI-HF
</TD><TD>100 bp ladder DNA ladder
</TD><TD>P123 digested with EcoRI-HF+NgoMIV
</TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Gel OK
</TD><TD>Gel OK
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P311.2BP312"> Ligation of P311+P312 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of P311+P312
</P><P><B>Procuedure:</B>
Reaction batch
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P312
</TD><TD>1.92 µl (~100 ng vector DNA)
</TD></TR><TR><TD>P311
</TD><TD>6.38 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>9.2 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_received_E..C2.A0coli_containing_biobricks_BBa_K181000.2C_BBa_K365002_and_BBa_K365003"> Picking of received <I>E. coli</I> containing biobricks BBa_K181000, BBa_K365002 and BBa_K365003 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of received <I>E. coli</I> containing biobricks BBa_K181000, BBa_K365002 and BBa_K365003.
</P><P><B>Procedure:</B></P><UL><LI> For each biobrick 1 coloniy was taken with a pipette tip and transferred into a cell-culture tube with 4  of LB-medium and antibiotics (CamR for BBa_K365002 and BBa_K365003, AmpR for BBa_K181000). 
</LI></UL><UL><LI> These 3 tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_of_PCR_56.2C57.2C58_.28lavendula_LS.29_and_PCR_42.2C43_.28citrus_LS.29">Preparative restriction digest of PCR 56,57,58 (lavendula LS) and PCR 42,43 (citrus LS)</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Prepare digested PCR fragments for further ligation into pTUM104 and pSB1C3.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR products
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>17.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> A mastermix for 16 reactions was produced. 
</LI><LI> PCR products were digested at 37 °C for 3 hours.
</LI></UL><P>P187 was used as positive control for restriction digest:
</P><UL><LI>3 µl p187 + 15 µl of Mastermix (see above)
</LI></UL><P><B>Gelelectrophoresis:</B></P><P>Gel 1:
</P><P>1. Gene ruler 1 kb
</P><P>2. PCR 42
</P><P>3. PCR 43
</P><P>5. P187
</P><P>Gel 2:
</P><P>1. Gene ruler 1 kb
</P><P>2. PCR 56
</P><P>3. PCR 57
</P><P>4. PCR 58
</P><P><B>Digested PCR products (PCR 42 RL, PCR 43 RL, PCR 56 RL, PCR 57 RL, PCR 58 RL) are stored in the iGEM PCR product box at -20°C. </B></P><UL><LI> RL = ready for ligation
</LI></UL><UL><LI>concentrations:
</LI></UL><P>PCR 42: 102 ng/µl
</P><P>PCR 43: 60 ng/µl
</P><P>PCR 56: 35 ng/µl
</P><P>PCR 57: 34 ng/µl (consumed)
</P><P>PCR 58: 40 ng/µl
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_ligation_products_after_miniprep">Analytical restriction digest of ligation products after miniprep</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Check ligation of PCR 1 and PCR 2 into pTUM104 and pSB1C3.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>miniprep products
</TD></TR><TR><TD>2.3 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>Pst1-HF (20 U/µl)
</TD></TR><TR><TD>=23 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation at 37°C for 2 h.
</LI></UL><P><B>Gelelectrophoresis:</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_lavendula_LS_and_citrus_LS_with_pSB1C3_and_pTUM104_2">Ligation of lavendula LS and citrus LS with pSB1C3 and pTUM104</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Ligate both lavendula and citrus limonene synthase (with all primer combinations -&gt; with or without consensus sequence) with pSB1C3 and pTUM104.
</P><P><B>Procedure</B></P><P><B>PCR 42 RL in pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>1 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 42 RL
</TD><TD>0.8 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>15.7 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>PCR 42 RL in pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>2.5 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 42 RL
</TD><TD>2.5 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>12.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>PCR 43 RL in pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>1 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 43 RL
</TD><TD>1.5 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>15.0 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>PCR 43 RL in pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>2.5 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 43 RL
</TD><TD>4 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>11 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>PCR 56 RL in pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>1 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 56 RL
</TD><TD>2 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>14.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>PCR 56 RL in pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>2.5 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 56 RL
</TD><TD>6 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>9 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>PCR 57 RL in pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>1 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 57 RL
</TD><TD>2 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>14.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>PCR 57 RL in pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>2.5 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 57 RL
</TD><TD>6 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>9 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>PCR 58 RL in pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>1 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 58 RL
</TD><TD>2 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>14.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>PCR 58 RL in pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>2.5 µl (~100 ng vector dna)
</TD></TR><TR><TD>PCR 58 RL
</TD><TD>6 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>9 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>Negative control/pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>1 µl (~100 ng vector dna)
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>16.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><P><B>Negative control/pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>2.5 µl (~100 ng vector dna)
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>15 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20 µl
</TD></TR></TBODY></TABLE><UL><LI>ligation over night at 16 °C (waterbath).
</LI></UL><P><B>Eppis with Ligation products are stored in two 50 ml Falcon tubes at -20°C (lowest drawer)! </B></P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Induction_of_PAL.2C4CL.2CCHS.2COMT.2CAPT_and_GFP_in_pTUM104_in_S._cerevisiae_in_SC-U_medium_with_2.25_galactose">Induction of PAL,4CL,CHS,OMT,APT and GFP in pTUM104 in <I>S. cerevisiae</I> in SC-U medium with 2% galactose</SPAN></H3><P><B>Investigator:</B> Mary, Daniela
</P><P><B>Aim of the experiment:</B> Expression of all enzymes in <I>Saccharomyces cerevisiae</I>.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>enzyme</B></TD><TD><B>OD600 </B></TD><TD><B>volume to be pelleted [ml]</B></TD></TR><TR><TD>PAL+
</TD><TD>3.246
</TD><TD>6.2
</TD></TR><TR><TD>PAL-
</TD><TD>3.604
</TD><TD>5.5
</TD></TR><TR><TD>4CL+
</TD><TD>3.875
</TD><TD>5.2
</TD></TR><TR><TD>4CL-
</TD><TD>4.0
</TD><TD>5.0
</TD></TR><TR><TD>CHS+
</TD><TD>3.685
</TD><TD>5.43
</TD></TR><TR><TD>CHS-
</TD><TD>6.68
</TD><TD>5.43
</TD></TR><TR><TD>OMT+
</TD><TD>3.795
</TD><TD>5.27
</TD></TR><TR><TD>OMT-
</TD><TD>4.085
</TD><TD>4.896
</TD></TR><TR><TD>APT
</TD><TD>3.415
</TD><TD>5.86
</TD></TR><TR><TD>GFP
</TD><TD>3.855
</TD><TD>5.188
</TD></TR></TBODY></TABLE><P>Incubation at 30 °C over night.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_solutions_for_generating_chemo_competent_E.Coli-_cells"> Preparation of solutions for generating chemo competent <I>E.Coli-</I> cells </SPAN></H3><P><B>Investigator: Roman</B></P><P><B>Procedure:</B></P><P>Required solutions (for a 24ml suspension of competent cells)
</P><UL><LI> 0,1 M MgCl2
</LI><LI> 0,05 M CaCl2
</LI><LI> 0,05 M CaCl2, 15% v/v glycerine
</LI></UL><P>Prepared volumes:
</P><UL><LI> 30 ml CaCl2, 15% v/v glycerine; 0,22g CaCl2 were mixed with 4,5 ml glycerine and 25,5 ml bidestillated water and autoclaved
</LI><LI> 500 ml MgCl2; 10,17g MgCl2 were mixed with 500 ml bidestillated water and autoclaved
</LI><LI> 250 ml CaCl2; 1,84g CaCl2 were mixed with 250 ml bidestillated water and autoclaved
</LI></UL><P>Preparation of competent cells see next day.
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_8th"><B>Wednesday, August 8th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_E..C2.A0coli_containing_biobricks_BBa_K181000.2C_BBa_K365002_and_BBa_K365003"> Miniprep of overnight culture of <I>E. coli</I> containing biobricks BBa_K181000, BBa_K365002 and BBa_K365003 </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of <I>E. coli</I> containing biobricks BBa_K181000, BBa_K365002 and BBa_K365003.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep with Kit from Qiagen after manufacturer's protocol
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophoresis_of_P27_and_P315"> Preperative digestion and gelelectrophoresis of P27 and P315 </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI> Reaction batch for P27:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>Plasmid-DNA P27
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>8.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for P315:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>Plasmid-DNA P27
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>8.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P27 digested with XbaI+PstI-HF BBa_I15008: Heme oxygenase 1 (HO1)
</TD><TD>100 bp DNA ladder
</TD><TD>P315 digested with XbaI+PstI-HF BBa_K181000: PcyA
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Backbone and insert correct
</TD><TD>Backbone and insert correct
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P313_BBa_K365002:_PhyB.28908NT.29and_P314_BBa_K365003_:_PhyB.28642NT.29"> Analytical digestion and gelelectrophoresis of P313 BBa_K365002: PhyB(908NT)and P314 BBa_K365003 : PhyB(642NT) </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI> Reaction batch for P313 (BBa_K365002):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid-DNA P313
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for P314 (BBa_K365003):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid-DNA P313
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P313 digested with XbaI+PstI-HF (BBa_K365002: PhyB(908NT)
</TD><TD>P314 digested with XbaI+PstI-HF (BBa_K365003: PhyB(642NT)
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Backbone and insert correct
</TD><TD>Backbone and insert correct
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_P206.2BP310_and_P312.2BP311"> Transformation of <I>E. coli</I> XL1-Blue with ligation products P206+P310 and P312+P311 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P206+P310 (N'-20aaLinker-LexA-C' construction) and P312+P311 (N'-Pif3-20aaLinker-C')
</P><P><B>Procedure:</B></P><UL><LI> Transformation was performed after standard transformation protocol from the lab.
</LI></UL><UL><LI> Transformated <I>E. coli</I> XL1-Blue cells were plated on antibiotic plates and were put at 37 °C overnight.
</LI></UL><UL><LI> P206+P310 (CamR), P312+P311 (CamR)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_colonies_containing_BBa_K365000"> Picking of <I>E. coli</I> XL1-Blue colonies containing BBa_K365000 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue colonies containing BBa_K365000 to a miniprep on the nextday from the overnight culture.
</P><P><B>Procedure:</B></P><UL><LI> 3 colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chloramphenicol. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C. 
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_products_of_transformation_from_August_3rd">Miniprep of ligation products of transformation from August 3rd</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Get transformed plasmids for further analytical restriction digest to check whether ligation worked.
</P><P><B>Procedure:</B> Miniprep was done using Qiagen miniprep kit.
</P><UL><LI> PCR1/p175 clone 1: 200 ng/µl
</LI><LI> PCR1/p175 clone 2: 260 ng/µl
</LI><LI> PCR1/p175 clone 3: 460 ng/µl
</LI><LI> PCR1/p175 clone 4: 450 ng/µl
</LI><LI> PCR2/p175: 222 ng/µl
</LI><LI> PCR2/p133 clone 1: 145 ng/µl
</LI><LI> PCR2/p133 clone 2: 150 ng/µl
</LI><LI> PCR2/p133 clone 3: 140 ng/µl
</LI><LI> PCR2/p133 clone 4: 510 ng/µl
</LI></UL><P><B>Eppis containing miniprep products were stored in 50 ml Falcon tubes at the lowest drawer of -20°C.</B>
--&gt; will be thrown away in case ligation hasn't worked.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_miniprep_products">Analytical restriction digest of miniprep products</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Analytical restriction digest to check whether ligation of Tuesday, July 31st has worked.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xba1
</TD></TR><TR><TD>0.25 µl
</TD><TD>Spe1
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0,2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI> Incubation for 1.5 h at 37°C.
</LI></UL><P><B>Analytical Gel</B></P><P>1. Standard (1 kb Gene Ruler)
</P><P>2. PCR1/p175 clone 1
</P><P>3. PCR1/p175 clone 2
</P><P>4. PCR1/p175 clone 4
</P><P>5. PCR1/p175 clone 3
</P><P>6. PCR2/p175
</P><P>7. PCR2/p133 clone 1
</P><P>8. PCR2/p133 clone 2
</P><P>9. PCR2/p133 clone 3
</P><P>10.PCR2/p133 clone 4 --&gt; successful!
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_ligation_products_from_ligation_of_August_7th">Transformation of ligation products from ligation of August 7th</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Transform <I>E.coli</I> XL 1 blue with ligation products to produce colonies containing plasmid DNA.
</P><P><B>Transformation into <I>E.coli</I> Xl1-Blue</B></P><UL><LI> thawing of 100 µl Ca-competent E.coli XL1-Blue cells on ice
</LI><LI> adding of 5 µl of ligation products
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> adding of 1 ml LB-medium without antibiotics to the cells and incubation at 37°C and 180 rpm for 60 min
</LI><LI> plating of 100 µl of the cells on an Amp-plate (pYES (P175) or Chloramphenicol-plate (pSB1C3, P133)
</LI><LI> sedimentation of the leftover in a centrifuge (30 - 60 sec, 13 000 rpm), resuspension of the sediment in 100 µl LB-medium and plating it as well on an antibiotic containing LB-plate 
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Cell_lysis_of_S._cerevisiae">Cell lysis of <I>S. cerevisiae</I></SPAN></H3><P><B>Investigator:</B> Mary,Daniela
</P><P><B>Aim of the experiment:</B>Receive (hopefully) expressed enzymes.
</P><P>The protocol 'Expression Hefe' (Dropbox) was used. If you want to repeat the experiment please read comments before you start.
</P><P>You should make a glycerol stock of clones picked. You should as well take a sample of the uninduced yeast and do a cell lysis to be able to compare the induced and uninduced status.
</P><P>Preparation of sodium phosphate buffer (50 mM):
</P><UL><LI>1 M Na2HPO4x2H2O (M = 177.99 g/mol):  1.7799 g dissolved in 10 ml ELGA water
</LI><LI>1 M NaH2PO4x2H2O (M = 156.01 g/mol):  0.468 g dissolved in 3 ml ELGA water
</LI><LI>8.095 ml Na2HPO4 and 1.905 ml NaH2PO4 and 190 ml ELGA water were mixed -&gt; pH should be 7.5 but in our case it wasn't. Therefore we used all of the Na2HPO4, but could only achieve an pH of 7.47.
</LI></UL><P>Preparation of PMSF (100 mM)-&gt;solution is in the fridge:
</P><UL><LI>0.0174 g were dissolved in 1 ml isopropanol (techn.)
</LI></UL><P>Preparation of breaking buffer:
</P><UL><LI>11.25 ml sodium phosphate buffer (50 mM)
</LI><LI>750 µl Glycerol (80%)
</LI><LI>24 µl EDTA
</LI></UL><P>Preparation of breaking buffer with PMSF:
</P><UL><LI> 6 ml of breaking buffer
</LI><LI> 60 µl of PMSF (100 mM)
</LI></UL><P><B>Preparations for SDS gel the next day:</B></P><P>The absorption A280 was measured with Nano Drop. The SDS gel should be loaded with 30 µg protein, each assay was filled up with buffer to a total volume of 10 µl. 2.5 µl 5xLaemmlired were added and all assays were denatured at 95 °C for 5 min.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>enzyme</B></TD><TD><B>concentration [mg/ml] (A280)</B></TD><TD><B>concentration of 1:10 dilution [mg/ml] (A280)</B></TD><TD><B>volume of 1:10 dilution with 30 µg protein [µl]</B></TD></TR><TR><TD>PAL+
</TD><TD>64.024
</TD><TD>7.946
</TD><TD>3.78
</TD></TR><TR><TD>PAL-
</TD><TD>61.140
</TD><TD>8.605
</TD><TD>3.486
</TD></TR><TR><TD>4CL+
</TD><TD>85.65
</TD><TD>9.459
</TD><TD>3.17
</TD></TR><TR><TD>4CL-
</TD><TD>54.166
</TD><TD>7.443
</TD><TD>not applied on the gel
</TD></TR><TR><TD>CHS+
</TD><TD>65.338
</TD><TD>6.045
</TD><TD>4.96
</TD></TR><TR><TD>CHS-
</TD><TD>51.167
</TD><TD>6.135
</TD><TD>4.89
</TD></TR><TR><TD>OMT+
</TD><TD>49.128
</TD><TD>5.453
</TD><TD>5.5
</TD></TR><TR><TD>OMT-
</TD><TD>62.074
</TD><TD>4.193
</TD><TD>7.15
</TD></TR><TR><TD>APT
</TD><TD>42.554
</TD><TD>4.787
</TD><TD>6.27
</TD></TR><TR><TD>GFP
</TD><TD>36.443
</TD><TD>4.247
</TD><TD>7.06
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_chemocompetent_E.Coli-_cells"> Preparation chemocompetent <I>E.Coli-</I> cells </SPAN></H3><P><B>Investigator:</B> Roman, Simon
</P><P><B>Operational sequence:</B></P><P>The competent cells were prepared as described in the protocoll:
</P><UL><LI> 600ml LB medium were inoculated with 6ml of an <I>E.Coli</I> XL1 blue overnight culture and grown to an OD550 of 0,67
</LI><LI> centrifugation and washing of the cells as described in the protocol
</LI><LI> cells were aliquoted as 100µl suspensions in the cold room
</LI><LI> storage at -80°C
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_9th"><B>Thursday, August 9th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_PCR_50-55_and_P297-P302">Gelextraction of PCR 50-55 and P297-P302</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI>Gelextraction was done according to Quiaquick gelextraktion protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_measurement_of_PCR_50-55_and_P297-302">Nanodrop measurement of PCR 50-55 and P297-302</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI> PCR products 50-55 (ADH1-T, TEf1-T, TEF2-P) and P297-302 (TEF1-P, ADH1-P) were measured with Nanodrop in ng/µl
</LI><LI> ADH1-T: 22,4
</LI><LI> TEF2-P: 7,4
</LI><LI> TEF1-P: 16,8
</LI><LI> ADH1-P: 52,6
</LI></UL></DIV><DIV class="constitutive_promoter"><P><B>Investigator</B>: Georg
</P><H3><SPAN class="mw-headline" id="Ligation_of_TEF1-P.2C_TEF2-P_ADH1-T.2C_TEF1-T_and_ADH1-T_and_pTUM104">Ligation of TEF1-P, TEF2-P ADH1-T, TEF1-T and ADH1-T and pTUM104</SPAN></H3><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,48 µl
</TD><TD>P132 (psb1c3 33,5 ng/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>4,52 µl
</TD><TD>TEF1-P (16,8 ng/µl)
</TD></TR><TR><TD>9,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,46 µl
</TD><TD>P132 (psb1c3 33,5 ng/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>6,54 µl
</TD><TD>TEF2-P (7,4 ng/µl)
</TD></TR><TR><TD>9,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,42 µl
</TD><TD>P132 (psb1c3 33,5 ng/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>4,52 µl
</TD><TD>ADH1-P (52,6 ng/µl)
</TD></TR><TR><TD>9,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>0,94 µl
</TD><TD>P132 (psb1c3 33,5 ng/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>7,06 µl
</TD><TD>TEF1-T (3,1 ng/µl)
</TD></TR><TR><TD>9,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>4,44 µl
</TD><TD>P132 (psb1c3 33,5 ng/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>4,44 µl
</TD><TD>ADH1-T (22,4 ng/µl)
</TD></TR><TR><TD>9,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>8 (=100 ng)
</TD><TD>pYESII (12,8 ng/µl)
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>2 µl
</TD><TD>50% PEG 4000
</TD></TR><TR><TD>7,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=100µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> From these ligation batches, overnight reactions were generated with negative control.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_amplification_of_a_fragment_of_P313"> PCR amplification of a fragment of P313 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of experiment:</B> Amplification of wanted fragment
</P><P><B>Operational Sequence:</B></P><P>PCR- reaction batch was prepared as follows:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>substrate</B></TD><TD><B>volume in µl</B></TD></TR><TR><TD>ddH2O
</TD><TD>35,75
</TD></TR><TR><TD>One Taq Standard Reaction Buffer (5x)
</TD><TD>10
</TD></TR><TR><TD>Plasmid template (p313)
</TD><TD>1
</TD></TR><TR><TD>Primer O70 (10 µM)
</TD><TD>1
</TD></TR><TR><TD>Primer O72 (10 µM)
</TD><TD>1
</TD></TR><TR><TD>dNTP (10 mM)
</TD><TD>1
</TD></TR><TR><TD>Polymerase One Taq
</TD><TD>0,25
</TD></TR></TBODY></TABLE><P>Negative control was prepared the same way, with 1 µl ddH2O instead of plasmid template.
</P><P>PCR temperature program:
</P><UL><LI>initial heating: 94°C for 30s
</LI><LI>30 cycles
</LI><LI>denaturation: 94°C for 30s
</LI><LI>annealing: 67°C for 60s
</LI><LI>elongation: 68°C for 3 min
</LI><LI>final elongation: 68°C for 5 min
</LI><LI>4°C
</LI></UL><P>Afterwards, the PCR reaction tubes were stored over night in the fridge at 4 °C
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_picked_E..C2.A0coli_XL1-Blue_containing_BBa_K365000"> Miniprep of picked <I>E. coli</I> XL1-Blue containing BBa_K365000 </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> Miniprep of picked <I>E. coli</I> XL1-Blue containing BBa_K365000. From those minipreps PCRs should be done to introduce RFC25 pre- and suffixes and MfeI and BamHI restriction sites.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed with Qiaprep Spin miniprep kit from Qiagen.
</LI></UL><UL><LI> Miniprep was performed after manufacturer's protocol.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophoresis_of_P324"> Preperative digestion and gelelectrophoresis of P324 </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> Preperative digestion and gelelectrophoresis of P324 for fusion protein construction the plasmid containing the phytochrome interacting factor 3 including the RFC25 pre- and suffix was digested with NgoMIV and PstI-HF.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid-DNA Pxxx
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> After 2-3 h of incubation time at 37 °C the reaction mix was mixed with 4.44 µl DNA running buffer (10x) and was pipetted into a gelpocket of an 1% low-melting agarose gel.
</LI></UL><UL><LI> The gel was running for about 1  at 80 V
</LI></UL><UL><LI> Then the gel extraction kit from Qiagen was used for extracting the linearized insert from the agarose gel.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_from_the_overnight_transformation_of_ligated_P206.2BP310_product_and_P312.2BP311_product_in_E..C2.A0coli_XL1-Blue"> Picking from the overnight transformation of ligated P206+P310 product and P312+P311 product in <I>E. coli</I> XL1-Blue </SPAN></H3><P><B>Investigator</B>: Dennis
</P><P><B>Aim of the experiment:</B> Picking from the overnight transformation of ligated P206+P310 product and P312+P311 product in <I>E. coli</I> XL1-Blue from August, the 8th.
</P><P><B>Procedure:</B></P><UL><LI> 5 colonies from P206+P310 transformated cells and 5 colonies from P312+P311 transformated cells were picked.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chloramphenicol. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_colonies_of_transformation_.28August_8th.29">Picking of colonies of transformation (August 8th)</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Get cells for miniprep for further analytical restriction digest to check whether ligation has worked.
</P><P><B>Procedure:</B> Picking of 3 clones each of PCR42/p175, PCR43/p175, PCR56/p175, PCR57/p175, PCR58/p175 into 4 ml LB with Amp and PCR42/p133, PCR43/p133, PCR56/p133, PCR57/p133, PCR58/p133 into 4 ml with Chlp. Inbucation at 37°C (shaker) over night.
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="SDS-PAGE_of_crude_protein_extract_after_induced_expression_of_PAL.2C_4CL.2C_CHS.2C_OMT.2C_APT_and_GFP_in_yeast">SDS-PAGE of crude protein extract after induced expression of PAL, 4CL, CHS, OMT, APT and GFP in yeast</SPAN></H3><P><B>Investigator:</B> Daniela
</P><P><B>Aim of the experiment</B>: Getting the right concentration of protein for the western blot.
</P><P>The preparation of the samples was already described on August, 8th.
</P><UL><LI>12 % gel
</LI><LI>6 µl Unstained Protein MW Marker (Thermo Scientific)
</LI><LI>12.5 µl sample
</LI></UL><P>Mass of enzymes:
</P><UL><LI>PAL = 76.880 kDa
</LI><LI>4CL = 61.053 kDa
</LI><LI>CHS = 42.713 kDa
</LI><LI>OMT = 39.265 kDa
</LI><LI>APT = 45.481 kDa
</LI></UL><P>The enzymes of the coumaroyl pathway were not detectable within the other proteins. However the gel looks good, therefore the same concentrations were used for the western blot.
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Western_Blot_of_PAL.2C_4CL.2C_CHS.2C_OMT.2C_APT_and_GFP_expressed_in_yeast">Western Blot of PAL, 4CL, CHS, OMT, APT and GFP expressed in yeast</SPAN></H3><P><B>Investigator:</B> Daniela
</P><P><B>Aim of the experiment:</B> Detecion of enzymes via Strep-tag
</P><P><B>SDS-PAGE:</B></P><P>The supernatant of the lysis reaction (see August, 8th) was diluted 1:10 with ELGA water. The absorption A280 was measured with Nano Drop. The SDS gel should be loaded with 30 µg protein, each assay was filled up with buffer to a total volume of 10 µl. 2.5 µl 5xLaemmlired were added and all assays were denatured at 95 °C for 5 min.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>enzyme</B></TD><TD><B>concentration of 1:10 dilution [mg/ml] (A280)</B></TD><TD><B>volume of 1:10 dilution with 30 µg protein [µl]</B></TD></TR><TR><TD>PAL+
</TD><TD>8.585
</TD><TD>3.49
</TD></TR><TR><TD>PAL-
</TD><TD>7.824
</TD><TD>3.83
</TD></TR><TR><TD>4CL+
</TD><TD>10.218
</TD><TD>2.94
</TD></TR><TR><TD>CHS+
</TD><TD>12.875
</TD><TD>2.33
</TD></TR><TR><TD>CHS-
</TD><TD>7.245
</TD><TD>4.14
</TD></TR><TR><TD>OMT+
</TD><TD>8.158
</TD><TD>3.68
</TD></TR><TR><TD>OMT-
</TD><TD>5.002
</TD><TD>6
</TD></TR><TR><TD>APT
</TD><TD>5.15
</TD><TD>5.82
</TD></TR><TR><TD>GFP
</TD><TD>4.805
</TD><TD>6.24
</TD></TR></TBODY></TABLE><UL><LI>12 % gel
</LI><LI>120 V, 1.5 h
</LI></UL><P><B>Blotting:</B></P><UL><LI>blotting of proteins on a nitrocellulose membrane (Whatman)
</LI><LI>50mA, 1 h
</LI></UL><P><B>Blocking:</B></P><UL><LI>wash 3x5min with PBS-T0.1
</LI><LI>the membrane is blocked over night at 4 °C on a shaking device
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Retransformations_with_pSB1C3">Retransformations with pSB1C3</SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of experiment:</B> Aim of the experiment was on the one hand to fill up the stock of usable pSB1C3 for both, the caffeine and the whole team. On the other hand, the experiment was used as a test for previously prepared competent <I>E.Coli</I> cells
</P><P><B>Operational sequence:</B> 
The used vectors for the transformations were pSB1C3 RFC25 (p123), and pSB1C3 containing CHS- from the coumaroyl group (p136). 
</P><P><B>Transformations:</B></P><UL><LI> Transformation of (old) chemo competent <I>E.coli</I> cells, used up to now, and the newly prepared with p123 
</LI><LI> Transformation of the new chemo competent <I>E.coli</I> cells with p136 with two slightly different methods:
</LI><LI> as described in previous protocolls, with an heat shock at 37°C for 5 minutes
</LI><LI> with an heat shock at 42°C for 30 seconds, followed by an incubation on ice for 30 minutes (before the cells being incubated with LB- medium)
</LI></UL><P>The reaction batches were plated out on chloramphenicol containing LB- agar plates (both, unconcentrated and concentrated, i.e. resuspended in ca. 100 µl LB medium). Plates were incubated over night at 37°C.
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_10th"><B>Friday, August 10th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_overnight_ligation_with_TEF1-P.2C_TEF2-P.2C_TEF1-T.2C_ADH1-P.2C_ADH1-T_and_pTUM104"> Transformation of overnight ligation with TEF1-P, TEF2-P, TEF1-T, ADH1-P, ADH1-T and pTUM104</SPAN></H3><P><B>Investigator</B>: Georg
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of ligation product was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on chloramphenicol, or ampicillin plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="PCR_of_TEF1-T_and_TEF2-P_from_Saccharomyces_cerevisiae_genome">PCR of TEF1-T and TEF2-P from <I>Saccharomyces cerevisiae</I> genome</SPAN></H3><P><B>Investigator</B>: Georg
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for PCR:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4,3 µl
</TD><TD>P237 (Yeast genome)
</TD></TR><TR><TD>35 µl
</TD><TD>5x One Taq St. Reaction buffer (NEB)
</TD></TR><TR><TD>3,5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>3,5 µl
</TD><TD>O62/10
</TD></TR><TR><TD>3,5 µl
</TD><TD>O63/10
</TD></TR><TR><TD>0.875 µl
</TD><TD>Taq Polym. (NEB)
</TD></TR><TR><TD>124,32 µl
</TD><TD>H20
</TD></TR><TR><TD>=175 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Reaction batch for PCR:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4,3 µl
</TD><TD>P237 (Yeast genome)
</TD></TR><TR><TD>35 µl
</TD><TD>5x One Taq St. Reaction buffer (NEB)
</TD></TR><TR><TD>3,5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>3,5 µl
</TD><TD>O64/10
</TD></TR><TR><TD>3,5 µl
</TD><TD>O65/10
</TD></TR><TR><TD>0.875 µl
</TD><TD>Taq Polym. (NEB)
</TD></TR><TR><TD>124,32 µl
</TD><TD>H20
</TD></TR><TR><TD>=175 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 2x3 PCR reactions were mixed and PCR was run at the following conditions:
</LI><LI> 30 sec, 94 C Initial Denaturation
</LI><LI> (30 Cycles) 94 C, 30 sec.
</LI><LI> (30 Cycles) 57,5, C 30 sec.
</LI><LI> (30 Cycles) 68 C, 1 min
</LI><LI> Final extension 68 C, 10 min
</LI><LI> Hold 4 C
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="PCR-Purification_of_TEF1-T_and_TEF2-P">PCR-Purification of TEF1-T and TEF2-P </SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI> PCR-Purification was accomplished according to Quiaquick PCR-purification protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_gel_of_purified_PCR-Products_.28TEF1-T_and_TEF2-P.29">Analytical gel of purified PCR-Products (TEF1-T and TEF2-P) </SPAN></H3><UL><LI> PCR was succesfull
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_PCR_product_of_P313_to_introduce_RFC25_pre-_and_suffix"> Analytical gelelectrophoresis of PCR product of P313 to introduce RFC25 pre- and suffix </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of PCR product of P313 to introduce RFC25 pre- and suffix to see wether the PCR was successful.
</P><P><B>Procedure:</B></P><UL><LI> The PCR product was first purificated with PCR purification kit from Qiagen
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for ~1 h.
</LI></UL><UL><LI> PCR was successful!
</LI></UL><P>
(3 µl of PCR solution were analyzed on gel)
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_transformated_E._coli_XL1-Blue_with_with_ligated_P206.2BP310_and_ligated_P312.2BP311"> Miniprep of overnight culture of transformated E. coli XL1-Blue with with ligated P206+P310 and ligated P312+P311 </SPAN></H3><P><B>Investigator:</B> Dennis, Roman
</P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of transformated E. coli XL1-Blue with with ligated P206+P310 and ligated P312+P311.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done with the Qiagen Qiaprep spin miniprep kit and was performed after manufacturer's protocol.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_analysis_and_preparation_of_over-_night_cultures"> Transformation analysis and preparation of over- night cultures </SPAN></H3><P><B>Investigator:</B>Roman
</P><P><B>Aim of the experiment:</B>
Analysis of the transformation with old and new chemocompetent <I>E.Coli-</I> cells and preparation of over night (over weekend, respectively) cultures, to amplificate the vector pSB1C3
</P><P><B>Results:</B></P><UL><LI> All plates showed a lot of colonies 
</LI><LI> The concentrated plates were quite overgrown, the unconcentrated ones showed single colonies
</LI><LI> no difference could be detected between transformation with 42°C heat shock (followed by 30 min incubation on ice, see previous day) and 37°C heat shock
</LI></UL><P>Newly prepared competent cells seem to work, further investigations (e.g. plating out with different antibiotics), see below.
</P><P><B>Operational sequence:</B>
Liquid cultures of the following transformants were prepared:
</P><UL><LI> XL1-blue with p123 (old competent cells used)
</LI><LI> XL1-blue with p123 (new competent cells used)
</LI><LI> XL1- blue with p136 
</LI></UL><P>To prepare the over night cultures, 4ml of LB medium containing chloramphenicol (4µl) were inoculated with a pipet- tip and incubated at room temperature over the weekend with 180 rpm.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Plating_of_newly_generated_chemocompetent_cells_with_different_antibiotics"> Plating of newly generated chemocompetent cells with different antibiotics </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> Test, if prepared competent cells (untransformed) are sensitive to different antibiotics 
</P><P><B>Operational sequence:</B></P><P>Used antibiotics:
</P><UL><LI> ampicillin
</LI><LI> kanamycin
</LI><LI> tetracyclin
</LI><LI> chloramphenicol
</LI></UL><P>100µl of untransformed, melted <I>E.Coli-</I> cells were plated and incubated over the weekend at room temperature. No growth should occure.
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Western_Blot_of_PAL.2C_4CL.2C_CHS.2C_OMT.2C_APT_and_GFP_expressed_in_yeast_-_continued">Western Blot of PAL, 4CL, CHS, OMT, APT and GFP expressed in yeast - continued</SPAN></H3><P><B>Investigator:</B> Daniela
</P><P><B>Aim of the experiment:</B> Detecion of enzymes via Strep-tag
</P><P><B>Detection:</B></P><UL><LI>via Streptavidin-AP (1:4000) -&gt;7ml, Streptavidin-AP diluted in PBST0.1
</LI><LI>1h incubation, be sure that the whole membrane is covered with Streptavidin-AP solution.
</LI></UL><P><B>Developing:</B></P><UL><LI>15 ml AP-buffer mixed with 45 µl BCIP (50 mg/ml in DMF) and 7.5 µl NBT (75 mg/ml in 70 % DMF)
</LI><LI>incubation: about 30 min
</LI></UL><P>Expected bands:
</P><P>PAL:  length: 716, mass =  76,880 kDa
</P><P>4CL:  no strep-tag -&gt;negative control (length: 561, mass =  61.053 kDa)
</P><P>CHS:  length: 390, mass =  42.713 kDa
</P><P>OMT:  length: 352, mass = 39.265 kDa
</P><P>APT:  length: 411, mass =  45.481 kDa
</P><P>GFP: mass: 26.9 kDa -&gt;positive control
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_transformation_august_8th">Miniprep of transformation august 8th</SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> Miniprep of transformation August 8th
</P><P>The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 40 µl ddH2O at room temperature, with the columns having been incubated 2 minutes at room temperature before the final centrifugation. 
<B>Miniprep products are stored in a 50 ml Falcon tube in the lowest drawer at -20°C </B></P></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_August_11th"><B>Saturday, August 11th</B></SPAN></H2><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Repetition_of_SDS-PAGE_and_Western_Blot_of_crude_protein_extract_after_induced_expression_of_PAL.2C_4CL.2C_CHS.2C_OMT.2C_APT_and_GFP_in_yeast">Repetition of SDS-PAGE and Western Blot of crude protein extract after induced expression of PAL, 4CL, CHS, OMT, APT and GFP in yeast</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment</B>: Execute the Western Blot using the StrepMab Classic antibody in order to test whether the immunospecifity is better. Furthermore one of the first SDS-PAGEs was done with an increased protein concentration.
</P><P><B>Operational sequence</B></P><UL><LI> To test wheather the increased protein concentrations lead to overloaded bonds, two SDS-PAGE gels which have been stained with Coomassie brilliant blue afterwards were run parallely to the ones for the Western blots.
</LI><LI> The preparation of the samples was executed as describes on August, 8th.
</LI><LI> 12 % SDS-PAGE gel, width of the collection gel: 2 mmm
</LI><LI> 6 µl Unstained Protein MW Marker (Thermo Scientific) for the staining with Coomassie brilliant blue and 6 µl Prestained Protein MW Marker (Thermo Scientific)for the western blots.
</LI><LI>15 µl sample
</LI><LI>SDS gel with 30 µg protein of each sample:
</LI></UL><P>Mass of enzymes:
</P><UL><LI>PAL = 76.880 kDa
</LI><LI>4CL = 61.053 kDa
</LI><LI>CHS = 42.713 kDa
</LI><LI>OMT = 39.265 kDa
</LI><LI>APT = 45.481 kDa
</LI></UL><UL><LI>SDS gel with maximal protein concentration using 15 µl of the crude cell extract:
</LI></UL><P>Mass of enzymes:
</P><UL><LI>PAL = 76.880 kDa
</LI><LI>4CL = 61.053 kDa
</LI><LI>CHS = 42.713 kDa
</LI><LI>OMT = 39.265 kDa
</LI><LI>APT = 45.481 kDa
</LI></UL><P>The enzymes of the coumaroyl pathway were not detectable within the other proteins. However the gels look good, therefore the same concentrations were used for the western blot.
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Western_Blot_of_PAL.2C_4CL.2C_CHS.2C_OMT.2C_APT_and_GFP_expressed_in_yeast_2">Western Blot of PAL, 4CL, CHS, OMT, APT and GFP expressed in yeast</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Detecion of enzymes via Strep-tag using Strep MAB classic
</P><P><B>SDS-PAGE:</B></P><P>Two separate approaches were executed: First, one Wester Blot was prepared using 30 µg protein of each sample as described by Daniela Dichtler on 09.08.2012. Second, a maximal protein concentration was used by utilizing undiluted crude cell extract. 10 µl of diluted an undiluted cell extract supernatant  were mixed with 2.5 µl 5xLaemmlired and incubated at 95 °C for 5 min.
</P><UL><LI>12 % SDS-PAGE gel, width of the collection gel: 2 mmm
</LI><LI> 6 µl Prestained Protein MW Marker (Thermo Scientific)
</LI><LI>12.5 µl sample
</LI><LI>90 V as long as the samples cross the collecting gel, afterwards 110 V until the dye is close to the lower end of the gel.
</LI></UL><P><B>Blotting:</B></P><UL><LI>blotting of proteins on a nitrocellulose membrane (Whatman) according to the protocol of Nadine Kallweit
</LI><LI>50mA per gel, 1 h
</LI></UL><P><B>Blocking:</B></P><UL><LI>wash 3x7.5min with PBS-T0.1
</LI><LI>the membrane is blocked over night at 4 °C on a shaking device with 3 % BSA in PBS-T0.1
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_August_12th"><B>Sunday, August 12th</B></SPAN></H2><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Western_Blot_of_PAL.2C_4CL.2C_CHS.2C_OMT.2C_APT_and_GFP_expressed_in_yeast_-_continued_2">Western Blot of PAL, 4CL, CHS, OMT, APT and GFP expressed in yeast - continued</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Detecion of enzymes via Strep-tag
</P><P><B>Detection:</B></P><UL><LI>wash 3x 7,5 moin with PBS-T0.1
</LI><LI>1h incubation with Strep MAB-classic primary antibody (10 ml per gel)
</LI><LI>wash 3x 7,5 moin with PBS-T0.1
</LI><LI>1h incubation with DAKO rabbit anti-mouse secondary antibody wich is conjugated to alkaline peroxidase(10 ml per gel)
</LI></UL><P><B>Developing:</B></P><UL><LI>wash 2x 5 min with PBS-T 0.1 and 2x 5 min with PBS. Brandish the membranes afterwards for 2 min in AP-buffer.
</LI><LI>Add 15 ml AP-buffer mixed with 45 µl BCIP (50 mg/ml in DMF) and 7.5 µl NBT (75 mg/ml in 70 % DMF)
</LI><LI>Incubation until the bonds become sufficiently visible, here 10-15 min.
</LI><LI>Western Blot using diluted samples:
</LI></UL><P>Expected bands:
</P><P>PAL:  length: 716, mass =  76,880 kDa
</P><P>4CL:  no strep-tag -&gt;negative control (length: 561, mass =  61.053 kDa)
</P><P>CHS:  length: 390, mass =  42.713 kDa
</P><P>OMT:  length: 352, mass = 39.265 kDa
</P><P>APT:  length: 411, mass =  45.481 kDa
</P><P>GFP: mass: 26.9 kDa -&gt;positive control
</P><UL><LI>Western Blot using concentrated samples:
</LI></UL><P>PAL:  length: 716, mass =  76,880 kDa
</P><P>4CL:  no strep-tag -&gt;negative control (length: 561, mass =  61.053 kDa)
</P><P>CHS:  length: 390, mass =  42.713 kDa
</P><P>OMT:  length: 352, mass = 39.265 kDa
</P><P>APT:  length: 411, mass =  45.481 kDa
</P><P>GFP: mass: 26.9 kDa -&gt;positive control
</P><P>On both gels the desired proteins can not be detected doubtless. Therefore several taks have to be done:
</P><UL><LI>Sequencing of the used genconstructs
</LI><LI>Sample taking after variable time after inducton in order to optimizie the amount of desired protein.
</LI><LI>Test different blocking techniques for the Western Blot to reduce the background signal.
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_10">Week 10</SPAN></H1><DIV class="week" id="WWeek_10"><H2><SPAN class="mw-headline" id="Monday.2C_August_13th"><B>Monday, August 13th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_PCR-Products_TEF2-P_and_TEF1-T">Preparative digestion of PCR-Products TEF2-P and TEF1-T</SPAN></H3><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6 µl
</TD><TD>PstI (NEB)
</TD></TR><TR><TD>6 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>30 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>3 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>105 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=150 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 25 µl mastermix were 25 µl of PCR-product added
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_of_colonies_of_psb1c3_with_TEF1-t.2C_TEF2-P.2C_ADH1-P_und_ADH1-t.2C_pTUM104">Picking of colonies of psb1c3 with TEF1-t, TEF2-P, ADH1-P und ADH1-t, pTUM104</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> no colonies of pYES2-TUM
</LI><LI> 10 colonies of TEF1-T, TEF2-P, 8 from ADH1-P and ADH1-T
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_pTUM104">Preparative digestion of pTUM104</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>pYESII
</TD></TR><TR><TD>2 µl
</TD><TD>SwaI
</TD></TR><TR><TD>2 µl
</TD><TD>PvuII
</TD></TR><TR><TD>4 µl
</TD><TD>NEB-3 buffer
</TD></TR><TR><TD>0,4 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>13,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=40µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestion was started with SwaI and BSA, for 3 hours at room temperature. Afterwards PvuII was added and digestion processed at 37 C
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_of_transformated_E..C2.A0coli_XL1-Blue_with_ligated_P206.2BP310_products_.28P327-P331.29"> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P206+P310 products (P327-P331) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P206+P310. P327-P331 (N'-Pif3-20aaLinker-C').
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with NgoMIV+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>NgoMIV
</TD></TR><TR><TD>1.5 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>90 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=105 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix for NgoMIV+PstI-HF was added to 2.5 µl of plasmid DNA of P327, P328, P329, P330, P331.
</LI></UL><UL><LI> Incubation for 2 h at 37 °C.
</LI></UL><P>Order of gel-pockets:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P327 (digested with NgoMIV+PstI-HF)</B></TD><TD><B>P328 (digested with NgoMIV+PstI-HF)</B></TD><TD><B>P329 (digested with NgoMIV+PstI-HF)</B></TD><TD><B>P330 (digested with NgoMIV+PstI-HF)</B></TD><TD><B>P331 (digested with NgoMIV+PstI-HF)</B></TD><TD><B>P310 control (before ligation and already digested with NgoMIV+PstI-HF)</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Ligation/Fusion was successful!
</TD><TD>Ligation/Fusion was successful!
</TD><TD>Ligation/Fusion was successful!
</TD><TD>Ligation/Fusion was successful!
</TD><TD>Ligation/Fusion was successful!
</TD><TD>Control OK!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_of_transformated_E..C2.A0coli_XL1-Blue_with_ligated_P312.2BP311_products_.28P332-P336.29"> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P312+P311 products (P332-P336) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P206+P310. P327-P331 (N'-20aaLinker-LexA-C').
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with EcoRI-HF+AgeI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>EcoRI-HF
</TD></TR><TR><TD>1.5 µl
</TD><TD>AgeI-HF
</TD></TR><TR><TD>88.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=105 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix for EcoRI-HF+AgeI-HF was added to 2.5 µl of plasmid DNA of P332, P333, P334, P335, P336.
</LI></UL><UL><LI> Incubation for 2 h at 37 °C.
</LI></UL><P>Order of gel-pockets:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P332 (digested with EcoRI-HF+AgeI-HF)</B></TD><TD><B>P333 (digested with EcoRI-HF+AgeI-HF)</B></TD><TD><B>P334 (digested with EcoRI-HF+AgeI-HF)</B></TD><TD><B>P335 (digested with EcoRI-HF+AgeI-HF)</B></TD><TD><B>P336 (digested with EcoRI-HF+AgeI-HF)</B></TD><TD><B>P311 control (before ligation and already digested with EcoRI-HF+AgeI-HF)</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Ligation/Fusion was successful!
</TD><TD>Ligation/Fusion was successful!
</TD><TD>Ligation/Fusion was successful!
</TD><TD>Ligation/Fusion was successful!
</TD><TD>Ligation/Fusion was successful!
</TD><TD>Control OK!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_PCR41_.28Pif3_100NT.29"> Preperative digestion of PCR41 (Pif3 100NT) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Contruction of N'-SV40NLS-Gal4AD-pGADT7 AD linker-Pif3(100NT)-C'.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR41
</TD></TR><TR><TD>5 µl
</TD><TD>Buffer G (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>BamHI (10 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>MfeI (MunI) (10 U/µl)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_PCR48_.28Gal4DBD.29"> Preperative digestion of PCR48 (Gal4DBD) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> To backup amplificated Gal4DBD with RFC25 pre- and suffix in pSB1C3.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR48
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>17.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI><B>WRONG DIGESTION, THERE IS NO PstI-HF SITE IN THE PCR PRODUCT! IT SHOULD HAVE BEEN XbaI+AgeI-HF</B></LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_PCR65_.28PhyB_908NT.29"> Preperative digestion of PCR65 (PhyB 908NT) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> To backup amplificated PhyB (908NT) with RFC25 pre- and suffix in pSB1C3.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR65
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>17.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI><B>WRONG DIGESTION, THERE IS NO PstI-HF SITE IN THE PCR PRODUCT! IT SHOULD HAVE BEEN XbaI+AgeI-HF</B></LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_P292"> Preperative digestion of P292 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Only the backbone of this part is needed; this part including LexA is used because to see if the double digest work. If it doesn't work, there won't be 2 DNA strands. Backbone was cut out for ligation with SV40NLS (PCR34).
</P><P><B>Procudure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P292
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI><B>WRONG DIGESTION! IT SHOULD HAS BEEN XbaI+AgeI-HF! BUT IT CAN BE USED FOR OTHER PURPOSES</B></LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_P293_.28Gal4AD.29"> Preperative digestion of P293 (Gal4AD) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Contruction of N'-SV40NLS-Gal4AD-pGADT7 AD linker-Pif3(100NT)-C'.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P293
</TD></TR><TR><TD>8 µl
</TD><TD>Buffer Tango (10x)
</TD></TR><TR><TD>4 µl
</TD><TD>EcoRI (10 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>BamHI (10 U/µl)
</TD></TR><TR><TD>6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_gelelectrophoresis_of_digested_PCR41.2C_PCR48.2C_PCR65"> Preperative gelelectrophoresis of digested PCR41, PCR48, PCR65 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preperative gelelectrophoresis of digested PCR41, PCR48, PCR65.
</P><P><B>Procedure:</B></P><UL><LI> Preperative gelelectrophoresis was performed at 70 V for 90 min.
</LI></UL><UL><LI> 5.55 µl of DNA loading buffer (10x) was added to each of the digested PCR products.
</LI></UL><P>Order of gel-pockets:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>PCR41 digested MfeI+BamHI
</TD><TD>100 bp ladder DNA ladder
</TD><TD>PCR48 digested with XbaI+PstI-HF
</TD><TD>1 kbp ladder DNA ladder
</TD><TD>PCR65 digested with XbaI+PstI-HF
</TD></TR><TR><TD>Okay!
</TD><TD>Length OK but wrong digestion, should have been XbaI+AgeI-HF
</TD><TD>Length OK but wrong digestion, should have been XbaI+AgeI-HF
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_gelelectrophoresis_of_digested_P292_and_P293"> Preperative gelelectrophoresis of digested P292 and P293 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preperative gelelectrophoresis of digested P292 and P293
</P><P><B>Procedure:</B></P><UL><LI> Preperative gelelectrophoresis was performed at 70 V for 90 min.
</LI></UL><UL><LI> 4.44 µl of DNA loading buffer (10x) was added to each of the digested plasmid products.
</LI></UL><P>Order of gel-pockets:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P292 digested XbaI+PstI-HF
</TD><TD>100 bp ladder DNA ladder
</TD><TD>P293 digested with EcoRII+BamHI
</TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Length OK but wrong digestion, should have been XbaI+AgeI-HF
</TD><TD>Okay!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P71.2BP322"> Ligation of P71+P322 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of P71+P322.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.18 µl
</TD><TD>P71
</TD></TR><TR><TD>3.63 µl
</TD><TD>P322
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>T4 ligase (5 U/µl)
</TD></TR><TR><TD>9.69 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> No negative control because there was no sufficient amount of P71 anymore.
</LI></UL><UL><LI> The ligation was performed at room temperature for 1 h after manufacturer's protocol.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P71.2BP323"> Ligation of P71+P323 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of P71+P323.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4.18 µl
</TD><TD>P71
</TD></TR><TR><TD>5.01 µl
</TD><TD>P323
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>T4 ligase (5 U/µl)
</TD></TR><TR><TD>8.31 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> No negative control because there was no sufficient amount of P71 anymore.
</LI></UL><UL><LI> The ligation was performed at room temperature for 1 h after manufacturer's protocol.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P206.2BP337"> Ligation of P206+P337 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of P206+P337.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.69 µl
</TD><TD>P206
</TD></TR><TR><TD>7.62 µl
</TD><TD>P337
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>T4 ligase (5 U/µl)
</TD></TR><TR><TD>8.19 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was performed with P206 only wihtout insert, i. e. the only difference is that ddH2O was used instead of P337 in the reaction batch.
</LI></UL><UL><LI> The ligation was performed at room temperature for 1 h after manufacturer's protocol.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P342.2BPCR66"> Ligation of P342+PCR66 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of P342+PCR66.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.04 µl
</TD><TD>P342
</TD></TR><TR><TD>1.92 µl
</TD><TD>PCR66
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>T4 ligase (5 U/µl)
</TD></TR><TR><TD>14.54 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was performed with P342 only wihtout insert, i. e. the only difference is that ddH2O was used instead of PCR66 in the reaction batch.
</LI></UL><UL><LI> The ligation was performed at room temperature for 1 h after manufacturer's protocol.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_of_P71.2BP322.2C_P71.2BP323.2C_P206.2BP337_and_P342.2BPCR66"> Transformation of <I>E. coli</I> XL1-Blue with ligation products of P71+P322, P71+P323, P206+P337 and P342+PCR66 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products of P71+P322, P71+P323, P206+P337 and P342+PCR66.
</P><P><B>Procedure:</B></P><UL><LI> 5 µl of the ligation products (P71+P322, P71+P323, P206+P337 and P342+PCR66) and their negative controls (P206 NK, P324 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on suitable antiotic plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on suitable antbiotic plates.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_SpeI.28at_684_bp.29_in_the_4CL_and_SpeI_.28at_470_bp.29_in_CHS">  Quick Change mutagenesis to remove SpeI(at 684 bp) in the 4CL and SpeI (at 470 bp) in CHS  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Generation of <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible versions of the 4CL and CHS genes.
<B>Used plasmids</B></P><UL><LI>pYes
</LI><LI>4Cl+: P193
</LI><LI>4CL-: P194
</LI><LI>CHS+: P195
</LI><LI>CHS-: P196
</LI></UL><LI>PSB1C3
</LI><LI>4CL+ P185
</LI><LI>4CL-: P186
</LI><LI>CHS+: P135
</LI><LI>CHS-: P136
</LI><P><B>PCR</B><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O81 for 4 CL and O83 for CHS (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O82 for 4 CL and O84 for CHS (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analysis_of_plated.2C_untransformed_chemocompetent_E.Coli-_cells"> Analysis of plated, untransformed chemocompetent <I>E.Coli-</I> cells </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> 
To test the prepared chemocompetent <I>E.Coli-</I> cells.
</P><P><B>Results:</B></P><UL><LI> Plate with kanamycin: no colonys (as expected)
</LI><LI> Plate with ampicillin: no colonys (as expected)
</LI><LI> Plate with tetracyclin: full (as expected, due to natural resistance)
</LI><LI> Plate with chloramphenicol: 12 colonies (not expected, should be empty)
</LI></UL><P>The plating on a chloramphenicol containing plate will be repeated, at 37°C over night instead of room temperature over the weekend, because perhaps chloramphenicol is degraded by the bacteria after two days
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Small-Scale_Yeast_Transformation_of_pTUM104_without_insert">Small-Scale Yeast Transformation of pTUM104 without insert</SPAN></H3><P><B>Investigator:</B> Katrin, Ingmar
</P><P><B>Aim of the experiment:</B>Transformation of P152 (pTUM104 without insert) in Yeast 
</P><P>The protocol on page 13 of the pYES2 manual was used. To ensure a positive result, two transformations were made.
</P><P>Only modifications are noted:
</P><P>step 1: inoculate in 4 ml YPD medium
</P><P>step 2: OD600 = 5,6 -&gt; to determine a OD600 of 0.4 in 50 ml, 3 ml yeast suspension and 50 ml YPD medium were used. 
</P><P>steps 3 and 4: centrifugation for 5 min, 4 °C
</P><P>step 6: 
</P><P>1 µg plasmid DNA
</P><UL><LI>P152: c = 156.8 ng/µl -&gt; 6,4 µl 
</LI></UL><P>100 µg denatured sheared salmon sperm: the one from Simon was used -&gt; 10 µl
</P><P>The cells were plated out on SC-U plates with Glucose and incubated at 30°C
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Plasmid_isolation_of_p123_and_p136"> Plasmid isolation of p123 and p136 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> 
Aim of the experiment was the isolation of p123 out of transformed (see friday) XL-1 blue chemo competent <I>e.coli-</I> cells. Old and newly prepared competent cells were used (see friday). Besides, p136 was isolated out of transformed cells, which is pSB1C3 with CHS- inserted. The whole experiment should provide an adequate amount of pSB1C3 backbone for the caffeine team, because we want to insert our three genes in this vector.
</P><P><B>Operational Sequence:</B></P><P>The plasmid preparation was performed as described in the manufacturers protocoll (Quiagen Miniprep Kit). Elution was done with 50 µl Elution Buffer in one step.
Due to the incubation of the liquid cultures at room temperature over the weekend instead of 37°C over night, the pellet was a bit smaller than usual, resulting in concentrations of only:
</P><UL><LI> 51,9 ng/µl (p123, old competent cells used)
</LI><LI> 57,8 ng/µl (p123, new competent cells used)
</LI><LI> 59,2 ng/µl (p136)
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Restriction_digest_of_p123_and_p136_and_preparative_gel_electrophoresis"> Restriction digest of p123 and p136 and preparative gel electrophoresis</SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B></P><P>Preparation of Xba1 and Pst1 digested pSB1C3 backbone, to be able to ligate the three caffeine genes into it.
</P><P><B>Operational sequence</B></P><P>Reaction batch: 
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>50 µl (ca. 2-3 µg)
</TD><TD>p123/p136
</TD></TR><TR><TD>6 µl
</TD><TD>10x Buffer NEB 4
</TD></TR><TR><TD>0,6 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>Xba1 RE
</TD></TR><TR><TD>2 µl</TD><TD>Pst1- HF RE
</TD></TR><TR><TD>=60 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Afterwards, about 6 µl of 10x loading dye were added and the fragments were separated by preparative gel electrophoresis, with the CHS- fragment being about 1300 bp in length.
</P><P><B>From left to right:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>DNA- Ladder
</TD><TD>pSB1C3_Xba1Pst1
</TD><TD>pSB1C3_CHS-_Xba1Pst1
</TD><TD>DNA- Ladder
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Dilution_of_gen-_synthesis_products_and_transformation"> Dilution of gen- synthesis products and transformation </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B></P><P>Dilution of dried pUCIDT- Amp vector, which contains the gene synthesis products. Afterwards, transformation of pUCIDT_CaXMT1, pUCIDT_CaMXMT1 and pUCIDT_CaDXMT1 in chemo competent XL-1 blue <I>e.coli-</I> cells, to make the genes available for cloning in pSB1C3 and pTUM104 
</P><P><B>Operational sequence</B></P><P><B>Basical vector data of pUCIDT</B></P><UL><LI> length: 2752 bp
</LI><LI> resistance: ampicillin
</LI></UL><P>About 2 µg of dried vector (containing our inserts) were delivered. By dilution with 20 µl bidestillated water (after centrifugation), a final concentration of about 100 ng/µl was adjusted.
</P><P><B>Transformation</B></P><P>Transformation in chemo competent cells was performed as previously described. Unconcentrated plating (100 µl) was performed on ampicillin containing LB agar plates, whereas the rest of the transformation- batch was stored in the fridge at 4°C (previous experiments showed, that unconcentrated plating led to enough colonies). The plates were incubated at room temperature over the next two days.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Repetition_of_plating_of_newly_prepared_chemo_competent_cells"> Repetition of plating of newly prepared chemo competent cells </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B></P><P>Test of newly prepared chemo competet cells, due to present chloramphenicol resistance
</P><P><B>Operational sequence</B></P><P>Because 12 colonies were grown on chloramphenicol containing agar plates after incubation at room temperature over the weekend (although &quot;empty&quot; competent XL1- blue cells were used), this plating was repeated and incubation was carried out at 37°C over night.
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_and_gel_electrophoresis_of_the_preprothaumatin_plasmid_and_pTUM104"> Preparative restriction digest and gel electrophoresis of the preprothaumatin plasmid and pTUM104 </SPAN></H3><P><B>Investigator:</B> Maddin, Aloisius
</P><P><B>Aim:</B> Cloning of preprothaumatin on pTUM104
</P><P><B>Operational sequence</B></P><P>Preparative restriction digest:
</P><UL><LI> 5 µl pYes2; 1 µl <I>Xba</I>I; 1 µl <I>Spe</I>I; 5 µl 10x NEBuffer 4;0,5 µl BSA; 37.5 µl ddH2O; 37 °C, 1 h.
</LI></UL><UL><LI> 10 µl	Preprothaumatinplasmid (gene synthesis!); 1 µl <I>Xba</I>I; 1 µl <I>Spe</I>I; 5 µl 10x NEBuffer 4; 0,5 µl BSA; 32.5 µl ddH2O; 37 °C, 1 h.
</LI></UL><UL><LI> Preparative gel electrophoresis: expected lengths for pYes2: backbone: 5.8 kb, (insert: 62 bp); expected length for preprothaumtin plasmid: backbone: 2.2 kb, insert: 721 bp.
</LI></UL><UL><LI> Gel extraction according to QIAqick®  Gel  Extraction Kit. Not successful, EtOH was forgotten.
</LI><LI> Transformation of <I>E. coli</I> XL1-Blue with the preprothaumatin plasmid (selection with amp).
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_August_14th"><B>Tuesday, August 14th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_picked_colony-DNA_with_XbaI-_.2B_PSTI-HF">Analytical digestion of picked colony-DNA with XbaI- + PSTI-HF</SPAN></H3><P><B> Investigator</B>: Georg
</P><P><B>Aim of experiment:</B> Find positive clones from transformation with ligation products ADH1-P+psb1c3, ADH1-T+psb1c3,TEF1-t+psb1c3 and TEF1-P+psb1c3
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5,5 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>5,5 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>44 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>4,4 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>325,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=385µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> to 2,5 ml of DNA 17,5 µl reaction batch were added
</LI><LI> Digestion took place at 37°C for 1 h
</LI></UL><UL><LI> NO TEF1-T as insert but TEF1-P, you probably has mismatched tubes
</LI></UL><UL><LI> Ligation was succesful
</LI><LI> wrong insert
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_from_the_overnight_transformation_of_ligated_P71.2BP322.2C_P71.2BP323.2C_P206.2BP337_and_P342.2BPCR66_product_in_E._coli_XL1-Blue"> Picking from the overnight transformation of ligated P71+P322, P71+P323, P206+P337 and P342+PCR66 product in <I>E. coli</I> XL1-Blue </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment</B>: Picking from the overnight transformation of ligated P71+P322, P71+P323, P206+P337 and P342+PCR66 product in <I>E. coli</I> XL1-Blue.
</P><P><B>Procedure:</B></P><UL><LI> 5 colonies of each transformation from P71+P322 (AmpR), P71+P323 (AmpR), P206+P337 (CamR) and P342+PCR66 (AmpR) transformated cells were picked.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and antibiotics (P71+P322 (AmpR), P71+P323 (AmpR), P206+P337 (CamR) and P342+PCR66 (AmpR)). These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Reapeat_of_transformation_of_E._coli_XL1-Blue_with_ligation_products_of_P71.2BP322.2C_P71.2BP323.2C_P206.2BP337_and_P342.2BPCR66_from_August.2C_the_13th"> Reapeat of transformation of <I>E. coli</I> XL1-Blue with ligation products of P71+P322, P71+P323, P206+P337 and P342+PCR66 from August, the 13th </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment</B>: Reapeat of transformation of <I>E. coli</I> XL1-Blue with ligation products of P71+P322, P71+P323, P206+P337 and P342+PCR66 from August, the 13th. The reason is that the freshly prepared competent cells are contaminated with a foreign plasmid.
</P><P><B>Procedure:</B></P><UL><LI> 5 µl of the ligation products (P71+P322, P71+P323, P206+P337 and P342+PCR66) and their negative controls (P206 NK, P324 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on suitable antiotic plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on suitable antbiotic plates. Accidently, only the pellet of P342+PCR66 and P342 NK was plated.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analysis_of_plated_XL1-blue_chemo_competent_E.coli_.28untransformed.29"> Analysis of plated XL1-blue chemo competent <I>E.coli</I> (untransformed) </SPAN></H3><P><B>Investigator:</B> Volker
</P><P><B>Aim:</B></P><P>Test of prepared chemo competent cells (empty) due to present resistance for chloramphenicol
</P><P><B>Results</B></P><P>Again 14 colonies have grown on the plate, after incubating at 37°C over night. One can presume, that the newly prepared competent cells have taken up any plasmid providing chloramphenicol resistance (e.g. pSB1C3), makin them unuseful for our experiments. The uptake probably took place when inoculating the LB medium, one day befor the cells were prepared, because a sample of our old competent cells was used for the inoculation instead of a plated colony of <I>e.coli-</I> XL1-blue. 
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_ligation_products_of_10.08._after_miniprep">Analytical restriction digest of ligation products of 10.08. after miniprep</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Check ligation of PCR 42 and PCR43 (Citrus) &amp; PCR 56, PCR 57 and PCR 58 (Lavendula) into pTUM104 and pSB1C3.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>miniprep products
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>Pst1-HF (20 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Master-Mix: 7 µl Enzyme, 56 µl NEB-Buffer, 5,6 µl BSA, 414,4 µl ddH2O
</LI><LI> 2,5 µl DNA added to 17,5 µl Master-Mix
</LI><LI> Incubation at 37°C for 1,5 h.
</LI></UL><P><B>Gelelectrophoresis:</B></P><UL><LI> ligation failed
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparation_of_material_for_protein_expression_in_yeast">Preparation of material for protein expression in yeast </SPAN></H3><P><B>Investigator:</B> Ingmar, Katrin
</P><P><B>Aim of the experiment:</B>Preparation of media for protein expression in yeast (expression will be done on thursday, august 16th) 
</P><UL><LI>The following media were made:
</LI></UL><P>SC-U medium (synthetic complete medium, without uracil)
</P><UL><LI> 0.67% 		yeast nitrogen base (without amino acids)
</LI><LI> 2% 		carbon source (i.e. glucose or galactose)
</LI><LI> 0.01% 		(adenine, arginine, cysteine, leuine, lysine, threonine, tryptophan)
</LI><LI> 0.005% 		(aspartic acid, histidine, isoleucine, methionine, phenylalanine, proline, serine, tyrosine, valine)
</LI></UL><P>--&gt; Production:We produced 1l of induction media(containig 2% galactose and 1 l of growth media(containing 2% glucose). A 50ml AS-aliquot and 6.7g Yeast Nitrogen Base were dissolved in 850ml water and autoklaved. Parallely 100 ml solutions with 20 % galactose and galactose respectively were autoclaved and mixed with 900 ml of the other media afterwards. 
</P><P>Breaking buffer
</P><UL><LI> 50 mM		sodium phosphate buffer, pH 7,5
</LI><LI> 1 mM		EDTA
</LI><LI> 5% (v/v)	glycerol
</LI><LI> 1 mM		PMSF
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Minipreparation_of_the_preprothaumatin_plasmid_.28gene_synthesis.29"> Minipreparation of the preprothaumatin plasmid (gene synthesis) </SPAN></H3><P><B>Investigator:</B> Martianus, Aloisius
</P><UL><LI> p343, p344, p345.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_15th"><B>Wednesday, August 15th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_transformated_E._coli_XL1-Blue_with_with_ligated_P71.2BP322.2C_P71.2BP323.2C_P206.2BP337_and_P342.2BPCR66"> Miniprep of overnight culture of transformated <I>E. coli</I> XL1-Blue with with ligated P71+P322, P71+P323, P206+P337 and P342+PCR66 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of transformated <I>E. coli</I> XL1-Blue with with ligated P71+P292, P71+P293, P206+P337 and P342+PCR66.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done with the Qiagen Qiaprep spin miniprep kit and was performed after manufacturer's protocol.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_of_transformated_E..C2.A0coli_XL1-Blue_with_ligated_P71.2BP322_and_P71.2BP323_products"> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P71+P322 and P71+P323 products </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P71+P322 (P352-P356) and P71+P323 products (P357-P361).
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with XbaI+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>22 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2.75 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>2.75 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>162.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=192.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix for XbaI+PstI-HF was added to 2.5 µl of plasmid DNA of P352-P356.
</LI></UL><UL><LI> Incubation for 2 h at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P352</B></TD><TD><B>P353</B></TD><TD><B>P354</B></TD><TD><B>P355</B></TD><TD><B>P356</B></TD><TD><B>P357</B></TD><TD><B>P358</B></TD><TD><B>P359</B></TD><TD><B>P360</B></TD><TD><B>P361</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD></TR></TBODY></TABLE><UL><LI> Only  the linearized backbone (pYES2 new with ~6 kbp) is visible but no inserts were visible.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_of_transformated_E..C2.A0coli_XL1-Blue_with_ligated_P206.2BP337_products"> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P206+P337 products </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P206+P337 products (P352-P366).
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with NgoMIV+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1.5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>90 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=105 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix for NgoMIV+PstI-HF was added to 2.5 µl of plasmid DNA of P362-P366.
</LI></UL><UL><LI> Incubation for 2 h at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P362</B></TD><TD><B>P363</B></TD><TD><B>P364</B></TD><TD><B>P365</B></TD><TD><B>P366</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Digestion not worked properly? Seems like it was successful but digestion does not work properly.
</TD><TD>Ligation failed
</TD><TD>Digestion not worked properly? Seems like it was successful but digestion does not work properly.
</TD><TD>Ligation failed
</TD><TD>Digestion not worked properly? Seems like it was successful but digestion does not work properly.
</TD></TR></TBODY></TABLE><UL><LI> For P362, P364 and P366, it seems that the digestion does not work properly, the expected band around 2-2.1 kbp and ~360 bp were only weak.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_of_transformated_E..C2.A0coli_XL1-Blue_with_ligated_P342.2BPCR66_products"> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P342+PCR66 products </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P342+PCR66 products (P367-P371).
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with NdeI+BamHI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>24 µl
</TD><TD>Tango (10x)
</TD></TR><TR><TD>6 µl
</TD><TD>NdeI (10 U/µl)
</TD></TR><TR><TD>6 µl
</TD><TD>BamHI (10 U/µl)
</TD></TR><TR><TD>69 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=105 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix for NdeI+BamHI was added to 2.5 µl of plasmid DNA of P367-P371.
</LI></UL><UL><LI> Incubation for 2 h at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P367</B></TD><TD><B>P368</B></TD><TD><B>P369</B></TD><TD><B>P370</B></TD><TD><B>P371</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD></TR></TBODY></TABLE><UL><LI> Ligation failed, even no Backbone was visible (~8 kbp)!
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_XL1_blue_with_ligation_products_2">Transformation of <I>E.coli</I> XL1 blue with ligation products</SPAN></H3><P>Investigator: Andrea
</P><P>Aim: Transformation of ligation products PCR42/P133, PCR43/P133, P56/P133, P57/P133, P58/P133, PCR42/P175, P43/P175, P56/P175, P57/P175, P58/P175
</P><P>Transformation into E.coli Xl1-Blue
</P><UL><LI> thawing of 100 µl Ca-competent E.coli XL1-Blue cells on ice
</LI><LI> add 5 µl of ligation product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 45 min
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an antibiotic containing LB-plate (Amp-plate (pYES (P175) or Chloramphenicol-plate (pSB1C3, P133)) 
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Repetition_of_analytical_restriction_digest_of_ligation_products_of_10.08._after_miniprep">Repetition of analytical restriction digest of ligation products of 10.08. after miniprep</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Check ligation of PCR 42 and PCR43 (Citrus) &amp; PCR 56, PCR 57 and PCR 58 (Lavendula) into pTUM104 and pSB1C3.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>miniprep products
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Master-Mix: 7 µl Enzyme, 56 µl NEB-Buffer, 5,6 µl BSA, 414,4 µl ddH2O
</LI><LI> 2,5 µl DNA added to 17,5 µl Master-Mix
</LI><LI> Incubation at 37°C for 2 h.
</LI></UL><P><B>Gelelectrophoresis:</B></P><UL><LI>ligation failed
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_colonies_of_transformation_.28August_3rd_and_August_8th.29">Picking of colonies of transformation (August 3rd and August 8th)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim:</B> Get cells for miniprep for further analytical restriction digest to check whether ligation has worked.
</P><P><B>Procedure:</B></P><UL><LI> Picking of 10 clones each of PCR42/p175 and PCR56/p175 (Trafo 08.08.)
</LI><LI> Picking of 3 clones of PCR2/P133 (Trafo 03.08.)
</LI><LI> Picking of 1 clone of PCR2/P175 (Trafo 03.08.)
</LI><LI> clones were put into 4 ml LB with Amp (P175) or Chlp (P133) 
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Preparation_of_cells_for_protein_expression_in_yeast">Preparation of cells for protein expression in yeast </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B>Picking transformed <I>Saccharomyces cerevisiae</I> cells for protein expression in yeast (expression will be done on Thursday, August 16th)
</P><UL><LI> Inocultion of overnight cultures: one clone from each plates with PAL+/-, CHS+/-, 4CL+/-, OMT1+/-, APT, GFP was picked,resuspended in 15 ml SCU medium with glucose and incubated at 30 °C overnight
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_16th"><B>Thursday, August 16th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P307.2BP309">Analytical digestion of P307+P309</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>0,5 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>4µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,4 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>29,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=35 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 17,5 µl reaction mix, 2,5 µl DNA were added
</LI><LI> Digestion took 1 hour at 37 C
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_linear_pTUM104">Gelextraction of linear pTUM104</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Gelextraction was done according to Quiaquick gel extraction kit
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_for_measuring_the_concentration_of_P265.2C264">Nanodrop for measuring the concentration of P265,264</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI>P265 :11,2 ng/µl
</LI><LI>P264 :12,7 ng/µl
</LI><LI>P264B:13,8 ng/µl
</LI><LI>P265B:14,9 ng/µl
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_pTUM104">Ligation of pTUM104</SPAN></H3><P><B>Investigator</B>: Georg
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,36 µl (=50 ng)
</TD><TD>P265B (PYesII digested)
</TD></TR><TR><TD>5 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>2,5 %
</TD><TD>50% PEG 4000
</TD></TR><TR><TD>24,14 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=50µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation took place at 16 C overnight
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P_402_.2B_P91">Preparative digestion of P 402 + P91</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>2 µl
</TD><TD>SpeI-HF (NEB)
</TD></TR><TR><TD>8 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,8 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>27,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=40µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestion was done at 37 C for 3 hours
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_TEF2-P_.28PCR59.29_and_Cyc1-T_P131_with_P132">Ligation of TEF2-P (PCR59) and Cyc1-T P131 with P132</SPAN></H3><P><B>Investigator</B>: Georg
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>2,99 µl (=100 ng)
</TD><TD>P132
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>8,01 µl
</TD><TD>PCR59
</TD></TR><TR><TD>6,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>2,99 µl (=100 ng)
</TD><TD>P132
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>9,31 µl
</TD><TD>P131</TD></TR><TR><TD>5,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was done at 16 C overnight
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytic_digestion_and_gelelectrophoresis_of_P362.2C_P364_and_P366"> Analytic digestion and gelelectrophoresis of P362, P364 and P366 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytic digestion and gelelectrophoresis of P362, P364 and P366 because the analytical digestion from the day before does not work properly.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with NgoMIV+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>8 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>59 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=70 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix for NgoMIV+PstI-HF was added to 2.5 µl of plasmid DNA of P362, P364 and P366.
</LI></UL><UL><LI> Incubation for 90 min at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P362</B></TD><TD><B>P364</B></TD><TD><B>P366</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation succesfull, but to be sure, sequencing has to be performed!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Sequencing_of_P366"> Sequencing of P366 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Succesful fusion of 20aa Linker + Pif3 has to be proven. Sequencing was performed by Eurofins.
</P><P><B>Procedure:</B></P><UL><LI> Preparing sequencing primer VF2 for Eurofins (c=2 pmol/µl)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.4 µl
</TD><TD>O68 (100 pmol/µl)
</TD></TR><TR><TD>19.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl (2 pmol/µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 15 µl of P366 was pippetted in a new tube for sequencing.
</LI></UL><UL><LI> Both tubes were sent to Eurofins
</LI></UL><UL><LI> Sequencing results:
</LI></UL><PRE>TATAAATAGGCGTATCACGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTAA
GGATGATTTCTGGAATTCGCGGCCGCTTCTAGATGGCCGGCGCGAGCGGCGCGGGCGGCA
GCGAAGGCGGCGGCAGCGAAGGCGGCACCAGCGGCGCGACCACCGGCATGCCTCTGTTTG
AACTTTTCAGGCTCACCAAAGCTAAGCTTGAATCTGCTCAAGACAGGAACCCTTCTCCAC
CTGTAGATGAAGTTGTGGAGCTGGTGTGGGAAAATGGTCAGATATCAACTCAAAGTCAGT
CAAGTAGATCGAGGAACATTCCTCCACCACAAGCAAACTCTTCAAGAGCTAGAGAGATTG
GAAATGGCTCAAAGACGACTATGGTGGACGAGATCCCTATGTCAGTGCCATCACTAATGA
CGGGTTTGAGTCAAGACGATGACTTTGTTCCATGGTTGAATCATCATACCGGTTAATACT
AGTAGCGGCCGCTGCAGTCCGGCAAAAAAACGGCAAGGTGTCACCACCCTGCCCTTTTTC
TTTAAAACCGAAAAGATTACTTCGCGTTATGCAGGCTTCCTCGCTCACTGACTCGCTGCG
CTCGGTCGTTCGGCTGCGGCGAGCGGTATCAGCTCACTCAAAGGCGGTAATACGGTTATC
CACAGAATCAGGGGATAACGCAGGAAAGAACATGTGAGCAAAAGGCCAGCAAAAGGCCAG
GAACCGTAAAAAGGCCGCGTTGCTGGCGTTTTTCCACAGGCTCCGCCCCCCTGACGAGCA
TCACAAAAATCGACGCTCAAGTCAGAGGTGGCGAAACCCGACAGGACTATAAAGATACCA
GGCGTTTCCCCCTGGAAGCTCCCTCGTGCGCTCTCCTGTTCCGACCCTGTCGCTTACCGG
ATACCTGTCCGCCTTTCTCCCTTCGGGAAGCGTGGCGCTTTCTCATAGCTCACGCTGTAG
GTATCTCAGTTCGGTGTAGGTCG
</PRE><UL><LI> Sequencing is fine!
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_PCR48_.28Gal4DBD.29_2"> Preperative digestion of PCR48 (Gal4DBD) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> To backup amplificated Gal4DBD with RFC25 pre- and suffix in pSB1C3.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>17 µl
</TD><TD>PCR48
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>23.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_PCR65_.28PhyB_908NT.29_2"> Preperative digestion of PCR65 (PhyB 908NT) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> To backup amplificated PhyB (908NT) with RFC25 pre- and suffix in pSB1C3.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>7 µl
</TD><TD>PCR65
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>35.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_P231"> Preperative digestion of P231 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preperative digestion of P231 for ligation to N-terminal SV40 nuclear localization signal. But sequencing result has first to be awaited.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P231
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_P235"> Preperative digestion of P235 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preperative digestion of P235 for get the pYES2newMCS backbone for cloning heme oxygenase (HO1) and phycocyanobilin:ferredoxin oxidoreductase (PcyA) into pYES2newMCS.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P235
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_gelelectrophoresis_of_digested_PCR48_and_PCR65"> Preperative gelelectrophoresis of digested PCR48 and PCR65 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preperative gelelectrophoresis of digested PCR48 and PCR65.
</P><P><B>Procedure:</B></P><UL><LI> Preperative gelelectrophoresis was performed at 70 V for 90 min.
</LI></UL><UL><LI> 5.55 µl of DNA loading buffer (10x) was added to each of the digested plasmid products.
</LI></UL><P>Order of gel-pockets:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD>PCR48 digested XbaI+AgeI-HF
</TD><TD>PCR65 digested with XbaI+AgeI-HF
</TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Length OK!
</TD><TD>Length OK!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_gelelectrophoresis_of_digested_P231_and_P235"> Preperative gelelectrophoresis of digested P231 and P235 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preperative gelelectrophoresis of digested P231 and P235
</P><P><B>Procedure:</B></P><UL><LI> Preperative gelelectrophoresis was performed at 70 V for 90 min.
</LI></UL><UL><LI> 4.44 µl of DNA loading buffer (10x) was added to each of the digested plasmid products.
</LI></UL><P>Order of gel-pockets:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P231 digested XbaI+AgeI-HF
</TD><TD>100 bp ladder DNA ladder
</TD><TD>P235 digested with XbaI+AgeI-HF
</TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Length OK!
</TD><TD>Length OK!
</TD></TR></TBODY></TABLE><UL><LI> From digested P235 only the backbone was cut out for further ligations.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_amplification_of_a_fragment_of_P313_to_introduce_RFC25_pre-_and_suffix"> PCR amplification of a fragment of P313 to introduce RFC25 pre- and suffix</SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of experiment:</B> Amplification of wanted fragment of P313 (PhyB 908-NT) to introduce RFC25 pre- and suffix.
</P><P><B>Operational Sequence:</B></P><UL><LI> 2 tubes were prepared
</LI></UL><UL><LI> PCR reaction batch was prepared as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>substrate</B></TD><TD><B>volume in µl</B></TD></TR><TR><TD>ddH2O
</TD><TD>35.75
</TD></TR><TR><TD>One Taq Standard Reaction Buffer (5x)
</TD><TD>10
</TD></TR><TR><TD>Plasmid template (P313)
</TD><TD>1
</TD></TR><TR><TD>Primer O70 (10 µM)
</TD><TD>1
</TD></TR><TR><TD>Primer O72 (10 µM)
</TD><TD>1
</TD></TR><TR><TD>dNTP (10 mM)
</TD><TD>1
</TD></TR><TR><TD>Polymerase One Taq
</TD><TD>0.25
</TD></TR></TBODY></TABLE><P>PCR temperature program:
</P><UL><LI>initial heating: 94°C for 30s
</LI><LI>30 cycles
</LI><LI>denaturation: 94°C for 30s
</LI><LI>annealing: 67°C for 60s
</LI><LI>elongation: 68°C for 3 min
</LI><LI>final elongation: 68°C for 5 min
</LI><LI>4°C
</LI></UL><P>Afterwards, the PCR reaction products were cleaned with the Qiagen's PCR purification kit after manufacturer's protocol and were stored over night in the fridge at 4 °C
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_the_10.C2.A0h_culture_of_transformated_E._coli_XL1-Blue_with_with_ligated_P71.2BP322.2C_P71.2BP323.2C_P206.2BP337_and_P342.2BPCR66"> Miniprep of the 10 h culture of transformated <I>E. coli</I> XL1-Blue with with ligated P71+P322, P71+P323, P206+P337 and P342+PCR66 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of the 10 h culture of transformated <I>E. coli</I> XL1-Blue with with ligated P71+P292, P71+P293, P206+P337 and P342+PCR66.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done with the Qiagen Qiaprep spin miniprep kit and was performed after manufacturer's protocol.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P207.2BPCR69"> Ligation of P207+PCR69 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of P207+PCR69 to backup Gal4DBD with RFC25 pre- ans suffix on pSB1C3.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.41 µl
</TD><TD>P207
</TD></TR><TR><TD>7.19 µl
</TD><TD>PCR69
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>T4 ligase (5 U/µl)
</TD></TR><TR><TD>9.15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was performed with P207 only wihtout insert, i. e. the only difference is that ddH2O was used instead of PCR69 in the reaction batch.
</LI></UL><UL><LI> The ligation was performed at 16 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P207.2BPCR70"> Ligation of P207+PCR70 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of P207+PCR70 to backup PhyB(908 NT) with RFC25 pre- ans suffix on pSB1C3.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.12 µl
</TD><TD>P207
</TD></TR><TR><TD>16.38 µl
</TD><TD>PCR70
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>T4 ligase (5 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was performed with P207 only wihtout insert, i. e. the only difference is that ddH2O was used instead of PCR70 in the reaction batch.
</LI></UL><UL><LI> The ligation was performed at 16 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_from_the_transformation_of_ligated_P71.2BP322.2C_P71.2BP323.2C_P206.2BP337_and_P342.2BPCR66_product_in_E._coli_XL1-Blue_from_Tuesday.2C_the_14th"> Picking from the transformation of ligated P71+P322, P71+P323, P206+P337 and P342+PCR66 product in <I>E. coli</I>XL1-Blue from Tuesday, the 14th </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment</B>: Picking from the transformation of ligated P71+P322, P71+P323, P206+P337 and P342+PCR66 product in E. coli XL1-Blue from Tuesday, the 14th because the same transformation the day before failed because the competent cells were contaminated, e. g. they already carries a plasmid.
</P><P><B>Procedure:</B></P><UL><LI> 5 colonies of each transformation from P71+P322 (AmpR), P71+P323 (AmpR), P206+P337 (CamR) and P342+PCR66 (AmpR) transformated cells were picked from the transformation from Tuesday, the 14th.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and antibiotics (P71+P322 (AmpR), P71+P323 (AmpR), P206+P337 (CamR) and P342+PCR66 (AmpR)). These tubes were put 10 h in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Induction.2C_Incubation_and_cell_disruption_of_S.c._containing_PAL.2B.2F-.2C_4Cl.2B.2C_CHS.2B.2F-.2C_OMT.2B.2F-.2C_APT_and_GFP"> Induction, Incubation and cell disruption of <I>S.c.</I> containing PAL+/-, 4Cl+, CHS+/-, OMT+/-, APT and GFP </SPAN></H3><P><B>Investigator:</B> Ingmar, Katrin
</P><P><B>Aim of the experiment:</B> Take samples at  variable times after induction and disrupt the cells in order to optimize the amount of desired protein.
<B>Procedure:</B></P><UL><LI>at 13:30h the overnight culture was diluted in 50 ml SC-U medium conataining 2 % galactose until an OD600 of 0.4.
</LI><LI> beginning 4h after induction every 3 h samples were taken and cells were disrupted according to &quot;Expression in Saccharomyces cerevisiae&quot; of Huang FC, Studart-Witkowski C &amp; Schwab W 2010. The supernatant of the disrupted cells were used for protein concentration appreciation using the nanodrop photometer. Afterwards they were stored at -80°C.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Gel_extraction_of_p123_and_p136"> Gel extraction of p123 and p136 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B></P><P>Extraction of pSB1C3 (digested with Xba1 and Pst1, = p123 and p136; digest was made on Monday) out of agarose gel. 
</P><P><B>Operational Sequence</B></P><P>Gel extraction was performed as described the manufacturers protocol (Quiagen). However, the determined concentrations (NanoDrop) were quite low:
</P><UL><LI> 8 ng/µl for p123 (Xba/Pst1)
</LI><LI> 11 ng/µl for p136 (Xba1/Pst1)
</LI></UL><P>This results in a yield of only about 20% (original concentrations were between 50 and 60 ng/µl)
Thus, a new over night culture was prepared (see below) to gain a higher initial concentration of plasmid DNA. The next gel extraction will be performed with bidestillated water (instead of elution buffer EB), temperated at 50°C.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Restriction_digest_of_p373_.28pTUM104_new_MCS_RFC25.29"> Restriction digest of p373 (pTUM104 new MCS RFC25) </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B></P><P>Restriction digest of pTUM104 with Xba1 and Pst1-HF to be able to clone the caffeine genes into. 
</P><P><B>Operational Sequence</B></P><P>Reaction batch:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0,4µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>2µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>11,6µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>The reaction batch was incubated for 2.5h at 37°C.
</P><P>Picture:
</P><P><B>From left to right:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>DNA- Ladder
</TD><TD>pTUM104_Xba1Pst1
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Picking_of_transformed_E.coli-_XL1-blue_cells_and_preparation_of_liquid_cultures"> Picking of transformed <I>E.coli-</I> XL1-blue cells and preparation of liquid cultures </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B></P><P>Preparation of liquid over night cultures of the following transformants:
</P><P><B>Operational sequence:</B></P><UL><LI> pUCIDT_CaXMT1 
</LI><LI> pUCIDT_CaMXMT1 
</LI><LI> pUCIDT_CaDXMT1 
</LI><LI> pSB1C3_CHS- (=p136)
</LI></UL><P>Two colonies of each plate were picked and used for inoculation of 6ml LB medium, containing the respective antibiotic (ampicillin or chloramphenicol).
The tubes were incubated at 37°C over night.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Gel_extraction_of_p373_.28.3DpTUM104.29"> Gel extraction of p373 (=pTUM104) </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B></P><P>Extraction of Xba1 and Pst1 digested pTUM104 out of agarose gel, to make it ready for ligation
</P><P><B>Operational sequence:</B></P><P>Gel extraction was performed as described in the manufacturers protocoll. However, bidestillated water was used for elution (30µl). Besides, the water was heated up to 37°C and also the 5 minutes of incubation (before final centrifugation) was done on the heating block at 37°C.
</P><P><B>NanoDrop</B></P><P>Concentration of the extracted plasmid was determined with NanoDrop and resulted in a concentration of 75,9 ng/µl.
As of now, all elution steps in gel extraction procedures will be performed at these conditions.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="PCR_of_C-LS3_.28BBa_I742111.2C_Trafo_19.6.29"> PCR of C-LS3 (BBa_I742111, Trafo 19.6) </SPAN></H3><P><B>Investigator: Andrea</B></P><P><B>Aim:</B> To get more PCR product in case another preparative digest is needed.
</P><P><B>Primer with consensus sequence</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer O27
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer O30
</TD></TR><TR><TD>0.25 µl
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (BBa_I742111) Clone 3
</TD></TR><TR><TD>35.75 µl
</TD><TD>dd water
</TD></TR><TR><TD>50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Primer without consensus sequence</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer O28
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer O30
</TD></TR><TR><TD>0.25 µl
</TD><TD>OneTaq Hot Start DNA Polymerase (Finally: 1.25 units/50 µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (BBa_I742111) Clone 3
</TD></TR><TR><TD>35.75 µl
</TD><TD>dd water
</TD></TR><TR><TD>50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>47 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>1,75 min
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>1 h
</TD></TR></TBODY></TABLE><UL><LI>1: Primer O27 / O30
</LI><LI>2: Primer O28 / O30
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="PCR_of_Schwab_plasmid_Quickchange-DNA_to_amplify_gene_for_lavendula_LS_2"> PCR of Schwab plasmid Quickchange-DNA to amplify gene for lavendula LS</SPAN></H3><P><B>Instructor:</B> Andrea 
</P><P><B>Aim:</B> PCR of Schwab plasmids after Quickchange-Mutagenesis with primers which were designed to amplify the lavendula LS gene and to add RFC25 restriction sites.
</P><P>3 different primer combinations were used:
</P><P>1. O33/O37
</P><P>2. O34/O37
</P><P>3. O35/O37
</P><P>Each primer combination was used for plasmid DNA amplification of P283.
</P><P><B>PCR reaction mixture</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Forward Primer
</TD></TR><TR><TD>1 µl
</TD><TD>10 µM Reverse Primer
</TD></TR><TR><TD>0.25 µl
</TD><TD>OneTaq Hot Start DNA Polymerase (Final: 1.25 units/50 µl)
</TD></TR><TR><TD>3 µl
</TD><TD>Plasmid DNA P283
</TD></TR><TR><TD>33.75 µl
</TD><TD>dd water
</TD></TR><TR><TD>50 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>50 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>1,75 min
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>1 h
</TD></TR></TBODY></TABLE><UL><LI>3: Primer O33/O37
</LI><LI>4: Primer O34/O37
</LI><LI>5: Primer O35/O37
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_pTUM104_clones_picked_on_August_15th">Miniprep of pTUM104 clones picked on August 15th</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Get transformed plasmids for further ligations.
</P><P><B>Procedure:</B> Miniprep was done using Qiagen miniprep kit.
</P><UL><LI> concentrations: see &quot;Inventarliste&quot;
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_products_of_transformation_from_August_3rd_and_August_8th">Miniprep of ligation products of transformation from August 3rd and August 8th</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Get transformed plasmids for further analytical restriction digest to check whether ligation worked.
</P><P><B>Procedure:</B> Miniprep was done using Qiagen miniprep kit.
</P><UL><LI> 1: 84 ng/µl
</LI><LI> 2: 75  ng/µl
</LI><LI> 3: 115  ng/µl
</LI><LI> 4: 107  ng/µl
</LI><LI> 5: 97  ng/µl
</LI><LI> 6: 112  ng/µl
</LI><LI> 7: 63  ng/µl
</LI><LI> 8: 53  ng/µl
</LI><LI> 9: 87  ng/µl
</LI><LI> 10: 107  ng/µl
</LI><LI> 11: 92  ng/µl
</LI><LI> 12: 48  ng/µl
</LI><LI> 13: 107  ng/µl
</LI><LI> 14: 105  ng/µl
</LI><LI> 15: 118  ng/µl
</LI><LI> 16: 106  ng/µl
</LI><LI> 17: 80  ng/µl
</LI><LI> 18: 84  ng/µl
</LI><LI> 19: 90  ng/µl
</LI><LI> 20: 97  ng/µl
</LI><LI> 21: 250  ng/µl
</LI><LI> 22: 60  ng/µl
</LI><LI> 23: 41  ng/µl
</LI><LI> 24: 275  ng/µl
</LI></UL><P><B>Eppis containing miniprep products were stored in an disposal bag at the upper drawer of -20°C.</B>
--&gt; will be thrown away in case ligation hasn't worked.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_of_PCR_56.2C57.2C58_.28lavendula_LS.29_and_PCR_42.2C43_.28citrus_LS.29_2">Preparative restriction digest of PCR 56,57,58 (lavendula LS) and PCR 42,43 (citrus LS)</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Prepare digested PCR fragments for further ligation into pTUM104 and pSB1C3.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR products
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>17.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> A mastermix for 6 reactions was produced (30µl NEB-Buffer, 3µl BSA, 6µl per enzyme, 105µl ddH2O). 
</LI><LI> PCR products were digested at 37 °C for 3 hours.
</LI></UL><P>P187 was used as positive control for restriction digest (XbaI and AgeI).
</P><P>Before Gelelectrophoresis Dephosphorylation of several fragments was done (see next part) !
</P><P><B>Gelelectrophoresis:</B></P><P>Limonensynthase: 1600 bp
</P><P><B>Digested PCR products (still in gel) are stored in an disposal bag at the upper drawer of -20°C. </B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Dephosphorylation_of_digested_PCR_42.2C_PCR_43.2C_PCR_56.2C_PCR_57_and_PCR_58">Dephosphorylation of digested PCR 42, PCR 43, PCR 56, PCR 57 and PCR 58</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Dephosphorylation of digested PCR products to avoid religation of the insert and enhance ligation rate.
</P><UL><LI> Befor Dephosphorylation procedure can be performed, the PCR Purification Kit of Qiagen needs to be used for purifying the digest solution (for removing enymes and buffer) !!!
</LI></UL><P><B>Procedure:</B> Use of SAP from Fermentas
</P><UL><LI> fill up the 50 µl digest solution to 60 µl with ddH20 (10 µl ddH2O were added)
</LI><LI> add each 6µl SAP-Buffer and 1 µl SAP
</LI><LI> mix 
</LI><LI> incubate at 37°C for 30 min
</LI><LI> inactivation at 65°C for 15 min
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_of_P343.2C_P344.2C_P345.2C_P152_.28pTUM104.29_and_P286_.28pSB1C3.29">Preparative restriction digest of P343, P344, P345, P152 (pTUM104) and P286 (pSB1C3)</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Prepare digested Preprothaumatin plasmids for further ligation into pTUM104 and pSB1C3.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>PCR products
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> A mastermix for 6 reactions was produced (24µl NEB-Buffer, 2.4µl BSA, 6µl per enzyme, 81.6µl ddH2O). 
</LI><LI> plasmids were digested at 37 °C for 3 hours.
</LI></UL><P>P187 was used as positive control for restriction digest (XbaI and AgeI).
</P><P>Before Gelelectrophoresis Dephosphorylation of vectors (pTUM104 and pSB1C3) was done (see next part) !
</P><P><B>Gelelectrophoresis:</B></P><P>pSB1C3 digested with XbaI and AgeI was lost during gelelectrophoresis 
</P><P><B>Digested PCR products (still in gel) are stored in an disposal bag at the upper drawer of -20°C. </B></P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Dephosphorylation_of_digested_pSB1C3_and_pTUM104_.28with_XbaI_and_SpeI.29">Dephosphorylation of digested pSB1C3 and pTUM104 (with XbaI and SpeI)</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Dephosphorylation of digested vectors to avoid religation and enhance ligation rate.
</P><UL><LI> Befor Dephosphorylation procedure can be performed, the PCR Purification Kit of Qiagen needs to be used for purifying the digest solution (for removing enymes and buffer) !!!
</LI></UL><P><B>Procedure:</B> Use of SAP from Fermentas
</P><UL><LI> fill up the 50 µl digest solution to 60 µl with ddH20 (10 µl ddH2O were added)
</LI><LI> add each 6µl SAP-Buffer and 3 µl SAP
</LI><LI> mix 
</LI><LI> incubate at 37°C for 30 min
</LI><LI> inactivation at 65°C for 15 min
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_17th"><B>Friday, August 17th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_of_transformated_E..C2.A0coli_XL1-Blue_with_ligated_P71.2BP322_and_P71.2BP323_products_2"> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P71+P322 and P71+P323 products </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P71+P322 (P379-P383) and P71+P323 products (P384-P388).
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with XbaI+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>22 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2.75 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>2.75 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>162.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=192.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix for XbaI+PstI-HF was added to 2.5 µl of plasmid DNA of P379-P388 in an ERG.
</LI></UL><UL><LI> Incubation for 2 h at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P379</B></TD><TD><B>P380</B></TD><TD><B>P381</B></TD><TD><B>P382</B></TD><TD><B>P383</B></TD><TD><B>P384</B></TD><TD><B>P385</B></TD><TD><B>P386</B></TD><TD><B>P387</B></TD><TD><B>P388</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD></TR></TBODY></TABLE><UL><LI> Only  the linearized backbone (pYES2 new with ~6 kbp) is visible but no inserts were visible.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_of_transformated_E..C2.A0coli_XL1-Blue_with_ligated_P206.2BP337_products_2"> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P206+P337 products </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P206+P337 products (P389-P393).
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with NgoMIV+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1.5 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1.5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>90 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=105 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix for NgoMIV+PstI-HF was added to 2.5 µl of plasmid DNA of P389-P393 in an ERG.
</LI></UL><UL><LI> Incubation for 2 h at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P389</B></TD><TD><B>P390</B></TD><TD><B>P391</B></TD><TD><B>P392</B></TD><TD><B>P393</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Ligation failed
</TD><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD></TR></TBODY></TABLE><UL><LI> Succesful ligation for P390 and P391. 20aaLinker-PIF3 ready for further fusion!
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_of_transformated_E..C2.A0coli_XL1-Blue_with_ligated_P342.2BPCR66_products_2"> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P342+PCR66 products </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligated P342+PCR66 products (P394-P398).
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with NdeI+BamHI
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>24 µl
</TD><TD>Tango (10x)
</TD></TR><TR><TD>6 µl
</TD><TD>NdeI (10 U/µl)
</TD></TR><TR><TD>6 µl
</TD><TD>BamHI (10 U/µl)
</TD></TR><TR><TD>69 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=105 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix for NdeI+BamHI was added to 2.5 µl of plasmid DNA of P394-P398.
</LI></UL><UL><LI> Incubation for 2 h at 37 °C.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P394</B></TD><TD><B>P395</B></TD><TD><B>P396</B></TD><TD><B>P397</B></TD><TD><B>P398</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Ligation failed
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD></TR></TBODY></TABLE><UL><LI> Ligation for P395-P398! These construct is finished (N'SV40NLS-Gal4AD-Linker-Pif3-C')! Only the RFC10 pre- and suffixes are missing. Next step for this fusion construct is to introduce RFC10 pre- and suffix and ligate it to pSB1C3 and pYES2new for expression. And <B>sequencing</B> has to be done!
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_of_P207.2BPCR69_and_P207.2BPCR70"> Transformation of <I>E. coli</I> XL1-Blue with ligation products of P207+PCR69 and P207+PCR70 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products of P207+PCR69 and P207+PCR70.
</P><P><B>Procedure:</B></P><UL><LI> 5 µl of the ligation products (P207+PCR69 and P207+PCR70) and their negative controls (P206 NK (PCR69), P206 NK (PCR70)) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_Xl1-Blue_with_ligation_of_TEF2-p.2C_CYC1-t_with_psb1c3_and_pTUM104_selfligation"> Transformation of <I>E.coli</I> Xl1-Blue with ligation of TEF2-p, CYC1-t with psb1c3 and pTUM104 selfligation </SPAN></H3><P><B> Investigator</B>: Georg
</P><UL><LI>5 µl of each ligation reaction were transferred to 100 µl of competent cells. 
</LI><LI>Cells were kept on ice for 30 min.
</LI><LI>Cells were heatshocked for 5 min. at 37 C
</LI><LI>Cells were incubated for 45 min. at 37 C
</LI><LI>Cells that were transformed with psb1c3 ligation product were transferred on Agarplates with Chla. Cells with *pYesII ligation product were palste on Amp Agar plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparation_of_probes_for_sequencing"> Preparation of probes for sequencing </SPAN></H3><P><B>Investigator</B>: Georg
</P><P>For sequencing of ligation product of Tef1-p, ADH1-p, Tef1-t and ADH1-t, with vector psb1c3 ligation products were diluted on a concentration between 50 ng/µl and 100 ng/µl at a min. volume of 15 µl. 
Sequencing primers were diluted to 2 µM at an min. volume of 15 µl
</P><UL><LI>ADH1-P (188 ng/µl):2 = 7,5 µl (P401) + 7,5 µl H20
</LI><LI>ADH-T (153,5 ng :2 = 7,5 µl + 7,5 µl H20
</LI><LI>TEF1-P (207,8 ng/µl) :3 = 5 µl + 10 µl H20
</LI><LI>Tef1-T (202,2 ng/µl) :3 = 5 µl + 10 µl H20
</LI><LI>TEF2 (PCR-Product)  (12,3 ng/µl)= 6,1 µl (TEF2-p) + 8,9 µl H20 = 5 ng/µl
</LI></UL><P>Sequencing primer: VF2 for ADH1-p, ADH1-t, TEF1-p, TEF1-t. VR for ADH1-p rv. TEF2p_fw for TEF2-p
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_EcoRI.28at_310_bp.29_in_the_4CL">  Quick Change mutagenesis to remove EcoRI(at 310 bp) in the 4CL  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P>Aim of the experiment: Generation of an <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible version of the 4CL gene.
</P><P><B>PCR</B><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O79 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O80 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Gel_extraction_of_digested_PCR_products">Gel extraction of digested PCR products</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Extract digested PCR products for further ligation.
</P><P>Due to previous phosphatase treatment (and presumably application of only 20 µl on the preparative gel), concentrations of products were very low:
</P><UL><LI> PCR 42: 10 ng/µl
</LI><LI> PCR 43: 10 ng/µl
</LI><LI> PCR 56: 10 ng/µl
</LI><LI> PCR 57: 13 ng/µl
</LI><LI> PCR 58: 14 ng/µl
</LI></UL><P><B>Digested products are stored in a disposal bag at the upper drawer of -20°C (label: Andrea, miniprep 16.8.)</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_PCR_42.2C_43.2C_56.2C_57_and_58_with_pTUM104_.28p175.29_and_pSB1C3_.28p133.29">Ligation of PCR 42, 43, 56, 57 and 58 with pTUM104 (p175) and pSB1C3 (p133)</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Ligate both lavendula and citrus limonene synthase (with all primer combinations -&gt; with or without consensus sequence) with pSB1C3 and pTUM104.
</P><P><B>Procedure</B></P><P><B>PCR 42 in pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>0.5 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 42
</TD><TD>8 µl (~6x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.25 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>0.25 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>PCR 42 in pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>1.2 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 42
</TD><TD>20 µl (~6x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=24.2 µl
</TD></TR></TBODY></TABLE><P><B>PCR 43 in pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>0.5 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 43
</TD><TD>8 µl (~6x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.25 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>0.25 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>PCR 43 in pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>1.2 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 43
</TD><TD>20 µl (~6x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=24.2 µl
</TD></TR></TBODY></TABLE><P><B>PCR 56 in pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>0.5 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 56
</TD><TD>8 µl (~6x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.25 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>0.25 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>PCR 42 in pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>1.2 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 56
</TD><TD>20 µl (~6x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=24.2 µl
</TD></TR></TBODY></TABLE><P><B>PCR 57 in pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>0.5 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 57
</TD><TD>6.5 µl (~6x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.25 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>1.75 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>PCR 57 in pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>1.2 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 57
</TD><TD>18 µl (~6x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=22.2 µl
</TD></TR></TBODY></TABLE><P><B>PCR 58 in pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>0.5 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 58
</TD><TD>6.0 µl (~6x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.25 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>2.25 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>PCR 58 in pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>1.2 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 58
</TD><TD>16.5 µl (~6x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=20.7 µl
</TD></TR></TBODY></TABLE><P><B>Negative control/pYES (p175)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P175
</TD><TD>0,5 µl (~50 ng vector dna)
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.25 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>8.25 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>Negative control/pSB1C3 (p133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>1.2 µl (~50 ng vector dna)
</TD></TR><TR><TD>T4 ligase
</TD><TD>0.5 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>2.5 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=25 µl
</TD></TR></TBODY></TABLE><UL><LI>ligation for 2 h at room temperature.
</LI><LI> 5 µl of ligation product was transformed. The remaining products were incubated at 16 °C (waterbath) for the weekend.
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Gel_extraction_of_digested_product">Gel extraction of digested product</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Extract digested prepro-thaumatin for further ligation.
</P><P>Due to previous phosphatase treatment (and presumably application of only 20 µl on the preparative gel), concentrations of products were very low:
</P><UL><LI> P343: 5 ng/µl
</LI><LI> P344: 5 ng/µl
</LI><LI> P345: 5 ng/µl
</LI><LI> pSB1C3: 9 ng/µl
</LI><LI> pYES: 7 ng/µl
</LI></UL><P><B>Digested products are stored in a disposal bag at the upper drawer of -20°C (label: Andrea, miniprep 16.8.)</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_miniprep_.28miniprep_16.8..2C_ligations_31.7._and_7.8..29">Analytical restriction digest of miniprep (miniprep 16.8., ligations 31.7. and 7.8.)</SPAN></H3><P><B>Investigators:</B> Katrin (digest), Lara (gelelectrophoresis)
</P><P><B>Aim:</B> Check whether ligations have worked (transformation products of 3.8. (ligation 31.7.) &amp; 8.8. (ligation 7.8.))
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>Not 1
</TD></TR><TR><TD>2 µl
</TD><TD>Puffer O (Fermentas)
</TD></TR><TR><TD>14.5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Gelelectrophoresis:</B></P><P>Gel1: Gene Ruler 1kb, 1, 2, 3, 4, 5, 6, 7, 8, 9
</P><P>Gel2: Gene Ruler 1kb, 10, 11, 12, 13, 14, 15, 16, 17
</P><P>Gel3: Gene Ruler 1kb, 18, 19, 20, 21, 22, 23, 24
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_ligation_products">Transformation of ligation products</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Transform <I>E.coli</I> XL 1 blue with ligation products (PCR42/p133, PCR42/p175, PCR43/p133, PCR43/p175, PCR56/p133, PCR56/p175, PCR57/p133, PCR57/p175, PCR58/p133, PCR58/p175)  to produce colonies containing plasmid DNA.
</P><P><B>Transformation into <I>E.coli</I> Xl1-Blue</B></P><UL><LI> thawing of 100 µl Ca-competent E.coli XL1-Blue cells on ice
</LI><LI> adding of 5 µl of ligation products
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> adding of 1 ml LB-medium without antibiotics to the cells and incubation at 37°C and 180 rpm for 45 min
</LI><LI> plating of 100 µl of the cells on an Amp-plate (pYES (P175) or Chloramphenicol-plate (pSB1C3, P133)
</LI><LI> sedimentation of the leftover in a centrifuge (60 sec, 13 000 rpm), resuspension of the sediment in 100 µl LB-medium and plating it as well on an antibiotic containing LB-plate 
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Isolation_of_of_plasmids_pUCIDT_Insert_and_pSB1C3"> Isolation of of plasmids pUCIDT_Insert and pSB1C3 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B>  Isolation of plasmids pUCIDt_CaXMT1, pUCIDT_CaMXMt1, pUCIDT_CaDXMT1 and pSB1C3_CHS- out of 6 ml of an over night culture (2-fold batch for each plasmid)
</P><P><B>Operational sequence:</B></P><P>Plasmids were isolated as described in the manufacturers protocoll (Quiagen). Elution was performed with 50µl buffer EB in one step.
</P><P><B>Determined concentrations:</B></P><UL><LI> pUCIDT_CaXMT1 A: 170 ng/µl
</LI><LI> pUCIDT_CaXMT1 B: 156 ng/µl
</LI><LI> pUCIDT_CaXMT1 A: 144 ng/µl
</LI><LI> pUCIDT_CaXMT1 B: 143 ng/µl
</LI><LI> pUCIDT_CaXMT1 A: 128 ng/µl
</LI><LI> pUCIDT_CaXMT1 B: 100 ng/µl
</LI><LI> pSB1C3_CHS-   A: 73 ng/µl
</LI><LI> psb1C3_CHS-   B: 108 ng/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Restriction_digest_of_isolated_plasmids"> Restriction digest of isolated plasmids </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> Digest of plasmids with Xba1 and Pst1 to gain the caffeine- synthesis genes and pSB1C3 backbone, respectively.
</P><P><B>Operational Sequence:</B></P><P>For restriction digest, the isolated plasmids pUCIDT_CaXMT1A, pUCIDT_CaMXMT1A, pUCIDT_CaDMT1A and pSB1C3_CHS-B were used (highest concentrations).
</P><P>Reaction batch:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume</B></TD></TR><TR><TD>ddH2O
</TD><TD>13,6 µl
</TD></TR><TR><TD>Plasmid-DNA
</TD><TD>20 µl
</TD></TR><TR><TD>Enzyme Pst1HF
</TD><TD>1 µl
</TD></TR><TR><TD>Enzyme Xba1
</TD><TD>1 µl
</TD></TR><TR><TD>10x NEB Buffer 4
</TD><TD>4 µl
</TD></TR><TR><TD>BSA 100x
</TD><TD>0,4 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>40 µl
</TD></TR></TBODY></TABLE><P>The reaction batch was incubated at 37°C für 1.5 h. Afterwards, the fragments were seperated by preparative agarose gel electrophoresis (see picture below).
</P><P><B>Gel- picture 1:</B></P><P><B>From left to right:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>DNA- Ladder
</TD><TD>pUCIDT_CaXMT1_Xba1Pst1
</TD><TD>pUCIDT_CaMXMT1_Xba1Pst1
</TD><TD>pUCIDT_CaXMT1_uncut
</TD></TR></TBODY></TABLE><P><B>Gel- picture 2:</B></P><P><B>From left to right:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>DNA- Ladder
</TD><TD>pUCIDT_CaDXMT1_Xba1Pst1
</TD><TD>pSB1C3_CHS-_Xba1Pst1
</TD><TD>pSB1C3_CHS-_uncut
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Gel_extraction_and_NanoDrop_determination_of_separated_DNA_fragments"> Gel extraction and NanoDrop determination of separated DNA fragments </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> Extraction of DNA fragments and determination of concentration previously to the ligation
</P><P><B>Operational sequence:</B>
The extraction was performed as described in the manufacturers protocoll (Quiagen).
Elution was performed with 30µl water (bidestillated) instead of elution buffer. Furthermore, the water has been heated up to 50°C and also the incubation before final centrifugation was performed on the heating block at 50°C.
</P><P><B>NanoDrop determination</B></P><UL><LI> CaXMT1: 21,4 ng/µl
</LI><LI> CaMXMT1: 15,1 ng/µl
</LI><LI> CaDMT1: 20,6 ng/µl
</LI><LI> pSB1C3: 20 ng/µl
</LI></UL><P>(all digested with Xba1 and Pst1)
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Ligation_of_genes_envolved_in_caffeine_synthesis_into_pTUM104"> Ligation of genes envolved in caffeine synthesis into pTUM104 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> Ligation of the genes, which are responsible for caffeine synthesis, into pTUM104 (for further expression studies) and pSB1C3 (to create BioBricks).
</P><P><B>Operational Sequence:</B></P><P>The following reaction batch refers to a molar ratio of 1:3 between vector and insert.
</P><P><B>Reaction batch:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume</B></TD></TR><TR><TD>pTUM104
</TD><TD>1,5 µl (ca. 120 ng)
</TD></TR><TR><TD>InsertA: CaXMT1
</TD><TD>3,5 µl (ca. 70 ng)
</TD></TR><TR><TD>InsertB: CaMXMT1
</TD><TD>5 µl (ca. 70 ng)
</TD></TR><TR><TD>InsertC: CaDXMT1
</TD><TD>3,5 µl (ca. 70 ng)
</TD></TR><TR><TD>Ligase
</TD><TD>1 µl
</TD></TR><TR><TD>10x Ligase Buffer
</TD><TD>2 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>ad 20 µl
</TD></TR></TBODY></TABLE><P>The batch was incubated over the weekend at 16°C in the water bath.
</P></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_August_18th"><B>Saturday, August 18th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_plated_transformated_E..C2.A0coli_XL1-Blue_colonies_with_ligated_P207.2BPCR69_and_P207.2BPCR70"> Picking of plated transformated <I>E. coli</I> XL1-Blue colonies with ligated P207+PCR69 and P207+PCR70 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment</B>: Picking of plated <I>E. coli</I> XL1-Blue colonies, transformed with ligated P207+PCR69 and P207+PCR70 on August, the 17th.
</P><P><B>Procedure:</B></P><UL><LI> 10 colonies of each transformation from P207+PCR69 (CamR) and P207+PCR70 (CamR) transformated cells were picked.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chloramphenicol. These tubes were put 10 h in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Colony_PCR_of_clones_of_PCR58.2Fp133_.28transformation_17.8..29">Colony PCR of clones of PCR58/p133 (transformation 17.8.)</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Introduce new method to allow for higher throughput of clone screenings.
</P><UL><LI> make sure you have PCR tubes and cell culture tubes labeled sufficiently. you need to have one pcr tube and one cell culture tube for each colony.
</LI><LI> pick a colony with a sterile toothpick or pipet tip, smear some of the cells onto the wall of the pcr tube (a bit of pressure and smearing) and subsequently put the toothpick into a  cell culture tube with LB-medium and antibiotic.
</LI></UL><P><B>19 Clones of PCR58/p133 (transformation of August 17th) were picked + 1 clone of 4CL+(PCR15)/p132 as positive control</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>1 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.15 µl
</TD><TD>40 µM O68
</TD></TR><TR><TD>0.15 µl
</TD><TD>40 µM O69
</TD></TR><TR><TD>0.1 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>1 µl
</TD><TD>Plasmid DNA (BBa_I742111) Clone 3
</TD></TR><TR><TD>5.6 µl
</TD><TD>dd water
</TD></TR><TR><TD>10 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>95 °C
</TD><TD>6 min
</TD></TR><TR><TD>35 cycles
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>53 °C
</TD><TD>30 s
</TD></TR><TR><TD>72 °C
</TD><TD>1.75 min
</TD></TR><TR><TD>Final extension
</TD><TD>72 °C
</TD><TD>10 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>1 h
</TD></TR></TBODY></TABLE><UL><LI> Colony PCR failed (and/or failed ligation, in case &quot;negative&quot; positive control was picked)
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_different_clones">Analytical restriction digest of different clones</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Repetition of analytical digestion of products of miniprep on August 16th (clone 6,11,24) and August 10th (PCR56/p175 clone 1; PCR56/p175 clone 3; PCR57/p175 clone 2, PCR58/p175 clone 1)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.3 µl
</TD><TD>Xba1
</TD></TR><TR><TD>0.3 µl
</TD><TD>Spe1
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>12.2 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI> Incubation for 1 h at 37°C.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_August_19th"><B>Sunday, August 19th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_the_overnight_culture_of_transformated_E._coli_XL1-Blue_with_with_ligated_P207.2BPCR69_and_P207.2BPCR70"> Miniprep of the overnight culture of transformated <I>E. coli</I> XL1-Blue with with ligated P207+PCR69 and P207+PCR70 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of the overnight culture of transformated <I>E. coli</I> XL1-Blue with with ligated P207+PCR69 and P207+PCR70.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done with the Qiagen Qiaprep spin miniprep kit and was performed after manufacturer's protocol.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_of_transformated_E..C2.A0coli_XL1-Blue_with_ligation_products_of_P207.2BPCR69_and_P207.2BPCR70"> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligation products of P207+PCR69 and P207+PCR70 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps of transformated <I>E. coli</I> XL1-Blue with ligation products of P207+PCR69 and P207+PCR70.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with XbaI+AgeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>42 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>4.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>5.25 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>5.25 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>310.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=367.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix for XbaI+AgeI-HF was added to 2.5 µl of prepped plasmid DNA of P408-P427.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P408</B></TD><TD><B>P409</B></TD><TD><B>P410</B></TD><TD><B>P411</B></TD><TD><B>P412</B></TD><TD><B>P413</B></TD><TD><B>P414</B></TD><TD><B>P415</B></TD><TD><B>P416</B></TD><TD><B>P417</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD><B>Ligation successful!</B> Gal4DBD ready for protein fusion!
</TD><TD><B>Ligation successful!</B> Gal4DBD ready for protein fusion!
</TD><TD><B>Ligation successful!</B> Gal4DBD ready for protein fusion!
</TD><TD><B>Ligation successful!</B> Gal4DBD ready for protein fusion!
</TD><TD><B>Ligation successful!</B> Gal4DBD ready for protein fusion!
</TD><TD><B>Ligation successful!</B> Gal4DBD ready for protein fusion!
</TD><TD><B>Ligation successful!</B> Gal4DBD ready for protein fusion!
</TD><TD><B>Ligation successful!</B> Gal4DBD ready for protein fusion!
</TD><TD><B>Ligation successful!</B> Gal4DBD ready for protein fusion!
</TD><TD><B>Ligation successful!</B> Gal4DBD ready for protein fusion!
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P418</B></TD><TD><B>P419</B></TD><TD><B>P420</B></TD><TD><B>P421</B></TD><TD><B>P422</B></TD><TD><B>P423</B></TD><TD><B>P424</B></TD><TD><B>P425</B></TD><TD><B>P426</B></TD><TD><B>P427</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD><B>Ligation successful!</B> PhyB(908-NT) ready for protein fusion!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_Clones_for_Ingmar_and_Roman"> Picking Clones for Ingmar and Roman </SPAN></H3><P><B>Investigator:</B> Volker
</P><P>4 Clones for each plate were picked in apropiate media at 1.40 AM =&gt; please do not prep before 12 AM!
Eight of them are in the 30° incubator on the right side. If you read this at the right time please put them into the 37°C incubator if there is not some space and prep them in the second round. Thanks!
</P><UL><LI> From Ingmas plates p187, p196 and p193 no clones could be picked because there were no single colonies</LI><LI> on all positive plates there was a sufficient number of clones (50-300)</LI><LI> Romans negative controle plates had no (Bneg) and 6 (Yneg) clones which is good / acceptable</LI><LI> on all positive plates there was a sufficient number of clons (30-200)</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_11">Week 11</SPAN></H1><DIV class="week" id="WWeek_11"><H2><SPAN class="mw-headline" id="Monday.2C_August_20th"><B>Monday, August 20th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophoresis_of_P422.2C_P331.2C_P417.2C_P290"> Preperative digestion and gelelectrophoresis of P422, P331, P417, P290 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preperative digestion and gelelectrophoresis of P422 (AgeI+PstI-HF), P331 (NgoMIV+PstI-HF), P417 (NgoMIV+PstI-HF), P290 (XbaI+PstI).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for P422:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P422
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for P331:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P331
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV(10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for P417:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P417
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV(10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for P290:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P290
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI(20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> These reaction batches were incubated for 3 h at 37 °C.
</LI></UL><UL><LI> Following gelelectrophoresis was performed with whole digestion batch after adding 4.44 µl of DNA loading buffer (10x) to each digestion batch at 70 V for 1.5 h on 1% low-melting agarose gel.
</LI></UL><UL><LI> Pocket order:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P422 (AgeI-HF+PstI-HF)
</TD><TD>100 bp DNA ladder
</TD><TD>P331 (NgoMIV+PstI-HF)
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Backbone incl. PhyB was cut out
</TD><TD>Insert was cut out
</TD></TR></TBODY></TABLE><UL><LI> Pocket order:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P417 (NgoMIV+PstI-HF)
</TD><TD>100 bp DNA ladder
</TD><TD>P290 (XbaI+PstI-HF)
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Insert was cut out
</TD><TD>Backbone was cut out
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Sequencing_of_P395.2C_P398.2C_P422.2C_P331.2C_PCR2.2FP133_Klon_4"> Sequencing of P395, P398, P422, P331, PCR2/P133 Klon 4 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Sequencing of P395, P398, P422, P331, PCR2/P133 Klon 4.
</P><P><B>Procedure:</B></P><UL><LI> Plasmid concentrations were measured by nanodrop to calculate the dilution factor to have a concentration between 50 ng/µl and 100 ng/µl.
</LI></UL><UL><LI> Plasmid concentration, determinated by nanodropping.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>Concentration in ng/µl</B></TD></TR><TR><TD>P395
</TD><TD>465.6
</TD></TR><TR><TD>P398
</TD><TD>391.8
</TD></TR><TR><TD>P422
</TD><TD>375.8
</TD></TR><TR><TD>P331
</TD><TD>210.9
</TD></TR><TR><TD>PCR2/P133 Klon 4
</TD><TD>469.1
</TD></TR></TBODY></TABLE><UL><LI> To get an end concentration between 50-100 ng/µl and an end volume of 15 µl, the dilution was performed as following:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>Concentration in ng/µl</B></TD></TR><TR><TD>P395
</TD><TD>1:5 dilution: 3 µl plasmid + 12 µl ddH2O
</TD></TR><TR><TD>P398 dilution:
</TD><TD>1:4 dilution: 3.75 µl plasmid + 11.25 µl ddH2O
</TD></TR><TR><TD>P422
</TD><TD>1:4 dilution: 3.75 µl plasmid + 11.25 µl ddH2O
</TD></TR><TR><TD>P331
</TD><TD>1:3 dilution: 5 µl plasmid + 10 µl ddH2O
</TD></TR><TR><TD>PCR2/P133 Klon 4
</TD><TD>1:5 dilution: 3 µl plasmid + 12 µl ddH2O
</TD></TR></TBODY></TABLE><UL><LI> Also the VF2 forward primer must have a end concentration of 2 ng/µl. So a mastermix for everybody was done with O68 (100 ng/µl)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>O68 (VF2, 100 pmol/µl)
</TD><TD>20 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>980 µl
</TD></TR><TR><TD><B>TOTAL (2 pmol/µl)</B></TD><TD><B>1000 µl</B></TD></TR></TBODY></TABLE><UL><LI> 15 µl was sent with each of P422, P331 and PCR2/P133 Klon 4 in a seperate ERG.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P436.2BP437_and_P206.2BP438"> Ligation of P436+P437 and P206+P438 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of P436+P437 (PhyB(908NT) + 20aaLinker-LexA)and P206+P438 (20aaLinker + Gal4DBD.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for P436+P437:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.11 µl
</TD><TD>P436
</TD></TR><TR><TD>4.21 µl
</TD><TD>P437
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12.18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for P206+P438:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.69 µl
</TD><TD>P207
</TD></TR><TR><TD>4.92 µl
</TD><TD>P438
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>10.89 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> These ligation batches including their negative controls, which only contain ddH2O instead of the insert, were put on a 16 °C waterbath and were incubated overnight.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_pTUM100_without_f1-origin_and_Gal-promoter"> Ligation of pTUM100 without f1-origin and Gal-promoter </SPAN></H3><UL><LI>Reaction batch for pYes2 circulation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10,5 µl
</TD><TD>T4-Ligase
</TD></TR><TR><TD>6,72 µl (100 ng)
</TD><TD>Dig. pYes 2
</TD></TR><TR><TD>10 µl
</TD><TD>PEG 4000
</TD></TR><TR><TD>10 µl
</TD><TD>10,5 x T4 ligase buffer
</TD></TR><TR><TD>68,28 µl
</TD><TD>H20
</TD></TR><TR><TD>= 106 µl
</TD><TD>total
</TD></TR></TBODY></TABLE><P>The mastermix was heated up to 72 C for 5 min bevore T4 ligase was added. It was separated in one reaction at room temperature and one reaction over night at 16 C.
</P></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_Xl1-blue"> Transformation of <I>E.coli</I> Xl1-blue </SPAN></H3><P>To 100 µl competent cells of E. coli Xl1-blue 10 µl of the pYes II Ligation reaction were added. 
Cells were kept on ice for 30 minutes. 
Heatshock was done at 37 C for 5 minutes
Cells were incubated for 45 minutes in 1 ml LB-medium at 37 C
Transformed cells were transferred on Agar plates with ampicillin
</P></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_gelrun_of_TEF1-p.2C_ADH1-p_psb1c3_after_digestion_with_PstI-HF_and_SpeI-HF"> Preparative gelrun of TEF1-p, ADH1-p psb1c3 after digestion with PstI-HF and SpeI-HF </SPAN></H3><P>After preparative digestion with PstI and SpeI, 44,4 µl of each reaction were transferred onto 
an agarose gel. Gelrun was performed at 70 Volt for 90 min. 
</P></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_colonies_of_E.coli_Xl1-blue.2C_transforemed_with_Tef2-p_.2B_psb1c3_and_Cyc-t_.2B_psb1c3"> Picking colonies of <I>E.coli</I> Xl1-blue, transforemed with Tef2-p + psb1c3 and  Cyc-t + psb1c3 </SPAN></H3><P>15 colonies from Tef2-p and 6 from Cyc-t were picked and transferred into LB-medium with 1x Chloramphenicole.
Incubation was performed over night at 37 C.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Colony_PCR_of_PCR43.2Fp175_and_PCR58.2Fp133"> Colony PCR of PCR43/p175 and PCR58/p133</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Establish colony PCR to increase speed of clone screening.
</P><UL><LI> make sure you have PCR tubes and cell culture tubes labeled sufficiently. you need to have one pcr tube and one cell culture tube for each colony.
</LI><LI> pick a colony with a sterile toothpick or pipet tip, smear some of the cells onto the wall of the pcr tube (a bit of pressure and smearing) and subsequently put the toothpick into a  cell culture tube with LB-medium and antibiotic.
</LI></UL><UL><LI><B>Citrus</B>: 8 clones of PCR43/p175 were picked.
</LI><LI><B>Lavendula</B>: 8 clones of PCR58/p133 (5 clones of transformation of August 17th, 3 clones of August 8th) and PCR58/p175 (8 of transformation of August 17th) were picked
</LI><LI> plasmid DNA of PCR2/p133 clone 4 as positive control
</LI></UL><P><B>Citrus</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O28
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O30
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Lavendula</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O35
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O37
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> PCR program:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>10 min
</TD></TR><TR><TD>30 cycles
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>59 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>1 min 55 sec
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><P>Labels of the cell culture tubes/PCR tubes:
</P><P>1-8: PCR43/p175
</P><P>9: PCR2/p133 clone 4 (positive control)
</P><P>10-17: PCR58/p175
</P><P>18-22: PCR58/p133, transformation of August 17th
</P><P>23-25: PCR58/p133, transformation of August 8th
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_PCR58.2Fp133"> Miniprep of PCR58/p133 </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> Miniprep of over night cultures with number 1,2,4,6,7,9,12,13,15,16,18 (clones that were picked for colony PCR on saturday, PCR58/p133).
</P><P>The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 40 µl of elution buffer (Kit) at room temperature, with the columns having been incubated 5 minutes at room temperature before the final centrifugation. 
<B>Miniprep products are stored in a 50 ml Falcon tube in the lowest drawer at -20°C </B></P><P>Miniprep does not work.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_PCR58.2Fp133_2"> Miniprep of PCR58/p133 </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim:</B> Miniprep of over night cultures with number 3,5,8,10,11,14,17 (clones that were picked for colony PCR on saturday, PCR58/p133).
</P><P>The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 40 µl of elution buffer (Kit) at room temperature, with the columns having been incubated 5 minutes at room temperature before the final centrifugation. 
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Number</B></TD><TD><B>Concentration in ng/µl</B></TD></TR><TR><TD>3
</TD><TD>254.7
</TD></TR><TR><TD>5
</TD><TD>223.4
</TD></TR><TR><TD>8
</TD><TD>339.4
</TD></TR><TR><TD>10
</TD><TD>303.5
</TD></TR><TR><TD>11
</TD><TD>162.4
</TD></TR><TR><TD>14
</TD><TD>142.1
</TD></TR><TR><TD>17
</TD><TD>151.8
</TD></TR></TBODY></TABLE><P><B>Miniprep products are stored in a 50 ml Falcon tube in the lowest drawer at -20°C. (Falcon tube label: Miniprep Saskia 20.8.)</B></P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Ligation_of_insert_and_pTUM104">Ligation of insert and pTUM104</SPAN></H3><P>Investigator: Martin y Aloís
</P><P>Aim: Ligate the extracted plasmids of the experiment before
</P><P>Due to the small yield we only were able to run 4 ligations:
</P><PRE>  1: pSB1C3 and P343
  I: pSB1C3 and P343 OVER NIGHT at 16 °C
  4: pYes and P343
 IV: pYes and P344 OVER NIGHT at 16 °C
</PRE><P>1 and 4 were directly transformed in <I>E. Coli</I> - to be picked the next day.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_over-weekend_ligation_of_caffeine_synthesis_genes_in_pSB1C3_and_pTUM104"> Transformation of over-weekend ligation of caffeine synthesis genes in pSB1C3 and pTUM104 </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim:</B> Transformation of the ligation
</P><P><B>Operational Sequence</B></P><P>The transformation procedure was carried out as previously described, whereas the entire ligation batch was used for transformation in XL1 blue chemo competent <I>e.coli-</I>cells. The 30 minutes on ice incubation was followed by the heat shock at 37°C for 5 minutes and an incubation at 180 rpm at 37°C for about 45 minutes. Afterwords, the suspension was plated on appropriate plates (pSB1C3 transformants with chloramphenicol, pTUM104 transformants with ampicillin) and incubated at 37°C over night. Besides, the pellet was also plated out (resuspension in about ca. 100 µl LB medium).
</P><P><B>Note:</B> This transformation was performed without knowing, that a transformation with 5 µl of every ligation batch was already done on saturday. Furthermore, clone picking and preparing of over night liquid cultures was made, too (on sunday).
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_picked_clones_of_possible_positive_pSB1C3_Caffeine_involved_gene_and_pTUM104_caffeine_involved_gene_transformants"> Miniprep of picked clones of possible positive pSB1C3_Caffeine_involved_gene and pTUM104_caffeine_involved_gene transformants </SPAN></H3><P><B>Investigator:</B> Roman, Saskia
</P><P><B>Aim:</B></P><P>Isolation of plasmids pSB1C3 and pTUM104, to check for uptaken inserts by an analytical restriction digest.
</P><P><B>Operational sequence:</B></P><P>The putative positive clones were picked and used for inoculation on monday, 1 am. Thus, the plasmid preparation took place at 3 pm. Four clones of each plate had been picked. The plates were stored in the fridge, at 4°C.
The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 40 µl of elution buffer (Kit) at room temperature, with the columns having been incubated 5 minutes at room temperature before the final centrifugation. 
</P><P><B>Plasmidconcentration:</B> Nanodroplet
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>pSB1C3_Caffeine_involved_gene</B></TD><TD><B>Concentration in ng/µl</B></TD></TR><TR><TD>B1A
</TD><TD>196.5
</TD></TR><TR><TD>B1B
</TD><TD>220.6
</TD></TR><TR><TD>B1C
</TD><TD>203.8
</TD></TR><TR><TD>B1D
</TD><TD>190.6
</TD></TR><TR><TD>B2A
</TD><TD>273.1
</TD></TR><TR><TD>B2B
</TD><TD>225.2
</TD></TR><TR><TD>B2C
</TD><TD>209.2
</TD></TR><TR><TD>B2D
</TD><TD>161.7
</TD></TR><TR><TD>B3A
</TD><TD>235.3
</TD></TR><TR><TD>B3B
</TD><TD>228.5
</TD></TR><TR><TD>B3C
</TD><TD>228.1
</TD></TR><TR><TD>B3D
</TD><TD>226.4
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>pTUM104_caffeine_involved_gene</B></TD><TD><B>Concentration in ng/µl</B></TD></TR><TR><TD>Y1A
</TD><TD>117.5
</TD></TR><TR><TD>Y1B
</TD><TD>200.9
</TD></TR><TR><TD>Y1C
</TD><TD>175.8
</TD></TR><TR><TD>Y1D
</TD><TD>206.7
</TD></TR><TR><TD>Y2A
</TD><TD>266.1
</TD></TR><TR><TD>Y2B
</TD><TD>306.2
</TD></TR><TR><TD>Y2C
</TD><TD>290.6
</TD></TR><TR><TD>Y2D
</TD><TD>302.7
</TD></TR><TR><TD>Y3A
</TD><TD>278.4
</TD></TR><TR><TD>Y3B
</TD><TD>162.7
</TD></TR><TR><TD>Y3C
</TD><TD>117.6
</TD></TR><TR><TD>Y3D
</TD><TD>211.3
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytical_digest_of_pSB1C3_Caffeine_involved_gene_and_pTUM104_caffeine_involved_gene_plasmids"> Analytical digest of pSB1C3_Caffeine_involved_gene and pTUM104_caffeine_involved_gene plasmids </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim:</B></P><P>Analytical digest of B1A-D, B2A-D, B3A-D, Y1A-D, Y2A-D, Y3A-D with PstI ans XbaI to controll the ligation of the inserts in pSB1C3 and pTUM104.
</P><P><B>Procedure:</B></P><UL><LI> Digestion reaction batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid-DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Digestion over night at 37 °C. 
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_August_21st"><B>Tuesday, August 21st</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_overnight_ligation_with_P436.2BP437_and_P206.2BP438_in_E..C2.A0coli_XL1-Blue"> Transformation of overnight ligation with P436+P437 and P206+P438 in <I>E. coli</I> XL1-Blue </SPAN></H3><P><B>Investigator:</B> Jeff 
</P><P><B>Aim of the experiment:</B> Transformation of overnight ligation with P436+P437 and P206+P438 in <I>E. coli</I> XL1-Blue.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation products (P436+P437 and P206+P438) and their negative controls (P436 NK, P206 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on new chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_cultures_of_TEF2-P_in_psb1c3_and_CYC-T_in_psb1c3"> Miniprep of overnight cultures of TEF2-P in psb1c3 and CYC-T in psb1c3</SPAN></H3><P><B>Investigator:</B> Georg
</P><P><B>Procedure:</B></P><UL><LI> Plasmid extraction from over-night grown colonies was done according to the protocoll of Quiaprep Spin genextraction kit.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_TEF2-P_and_Cyc-T_in_psb1c3">Analytical digestion of TEF2-P and Cyc-T in psb1c3</SPAN></H3><P><B> Investigator:</B> Georg
</P><P><B>Aim of the experiment:</B> To find positive clones of TEF2-P and Cyc-T
</P><P><B>Procedure</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5,5 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>5,5 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>44 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>4,4 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>325,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=385µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> to 2,5 ml of DNA 17,5 µl reaction batch were added
</LI><LI> Digestion took place at 37°C for 1 h
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_gel_with_Transformants_of_TEF2-P_and_CYC-T_in_psb1c3-Vector"> Analytical gel with Transformants of TEF2-P and CYC-T in psb1c3-Vector</SPAN></H3><P><B> Investigator</B>: Georg
</P><P><B> Aim of the experiment</B>: Find positive clones
</P><UL><LI> 9 µl of analytical digestion was applicated onto a 1% analytical agarose gel
</LI><LI> Run was performed at 90 Volt for 1h
</LI><LI>Result: Gelrun showed only two positive TEF2 clones and no CYC-T clone
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_overnight_ligation_with_P265B_.28pTUM100.29"> Transformation of overnight ligation with P265B (pTUM100)</SPAN></H3><P><B>Investigator:</B> Georg
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on ampicillin plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P451_.28TEF2-P_in_psb1c3.29_with_SpeI-HF_and_PstI-HF"> Preparative digestion of P451 (TEF2-P in psb1c3) with SpeI-HF and PstI-HF</SPAN></H3><P><B>Investigator</B>:Georg, Dennis
</P><P><B>Aim of the experiment</B>: Prepare TEF2-P for cloning 
</P><P><B>Procedure</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>1 µl
</TD><TD>Spe-HF (NEB)
</TD></TR><TR><TD>0,4 µl
</TD><TD>100xBSA (NEB)
</TD></TR><TR><TD>4 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>20 µl
</TD><TD>P 451
</TD></TR><TR><TD>13,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Digestion took place at 37 C for 3 hours
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_.22positive.22_colonies_of_colony_PCR">Miniprep of &quot;positive&quot; colonies of colony PCR</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Get plasmid DNA of the plasmids that were identified as insert-positive clones.
</P><P>Clones 2, 8, 11, 13, 18 &amp; 20 of colony PCR (20.8.) were miniprepped.
</P><UL><LI> Clone 2: 214 ng/µl
</LI><LI> Clone 8: 142 ng/µl
</LI><LI> Clone 11: 152 ng/µl
</LI><LI> Clone 13: 252 ng/µl
</LI><LI> Clone 18: 132 ng/µl
</LI><LI> Clone 20: 210 ng/µl
</LI></UL><P><B>Plasmid DNA is stored in a 50 ml Falcon tube at the lowest drawer of -20°C. (label: Miniprep 21.8. Lara)</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_miniprep_products_21.8._.28.22positive.22_clones_of_colony_PCR.29_and_miniprep_products_20.8.">Analytical restriction digest of miniprep products 21.8. (&quot;positive&quot; clones of colony PCR) and miniprep products 20.8.</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Check whether clones contain insert. 
Gel 1: Some clones that were picked for colony PCR of Saturday, August 18th.
Gel 2: Clones that were identified as positive in colony PCR of Monday, August 20th.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xba1
</TD></TR><TR><TD>0.25 µl
</TD><TD>Spe1
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>12.3 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI> digest at 37°C for 2.5h.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Colony_PCR_of_PCR42.2Fp133_and_PCR56.2Fp133">Colony PCR of PCR42/p133 and PCR56/p133</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> To decrease number of false positive PCR products, a different primer combination was used: Instead of forward + reverse primers for the insert, the forward primer of the vector backbone and the reverse primer of the insert were used.
</P><UL><LI> extension time of PCR reaction was adjusted to the longer PCR product (insert+primers+backbone from primer binding site)
</LI><LI> annealing temperature was adjusted to lowest primer melting temperature
</LI></UL><UL><LI><B>Citrus</B>: 5 clones of PCR42/p133 (transformation of 8.8.) and PCR42/p175 (trafo 17.8.) were picked.
</LI><LI><B>Lavendula</B>: 5 clones of PCR56/p133 (trafo 17.8.) and PCR56/p175 (transformation of August 17th) were picked.
</LI><LI> positive control: plasmid DNA of PCR2/p133 clone 4 as positive control
</LI><LI> negative controls: 1. clone containing pSB1C3 with proteinlinker (Jeff); 2. ddH2O; 3. clone containing pTUM104 (P50, no insert)
</LI></UL><P><B>Citrus in pSB1C3</B> (PCR42/p133)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O68
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O30
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Citrus in pYES</B> (PCR42/p175)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O8
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O30
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Lavendula in pSB1C3</B> (PCR56/p133)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O68
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O37
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Lavendula in pYES</B> (PCR56/p175)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O8
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O37
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> PCR program:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>10 min
</TD></TR><TR><TD>30 cycles
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>50 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>1 min 55 sec
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><P><B>Labels:</B></P><P>1-5: PCR42/p133 (Trafo 8.8)
6: PCR2/p133 clone 4 (positive control, trafo 8.8.)
7: pSB1C3 neg. control (proteinlinker)
8-12: PCR56/p133
13-17: PCR42/p175
18: neg. control: ddH2O
19: p50 clone
20-24: PCR56/p175
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_of_17th_August"> Miniprep of ligation of 17th August</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P>The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 50 µl of elution buffer (Kit) at room temperature, with the columns having been incubated 5 minutes at room temperature before the final centrifugation. 
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Number</B></TD><TD><B>Concentration in ng/µl</B></TD></TR><TR><TD>PCR56 in P133 (clone 1)
</TD><TD>63.7
</TD></TR><TR><TD>PCR56 in P133 (clone 2)
</TD><TD>50.6
</TD></TR><TR><TD>PCR56 in P133 (clone 3)
</TD><TD>63.3
</TD></TR><TR><TD>PCR56 in P133 (clone 4)
</TD><TD>77.9
</TD></TR><TR><TD>PCR56 in P133 (clone 5)
</TD><TD>82.1
</TD></TR><TR><TD>PCR56 in P175 (clone 1)
</TD><TD>94.3
</TD></TR><TR><TD>PCR56 in P175 (clone 2)
</TD><TD>50.6
</TD></TR><TR><TD>PCR56 in P175 (clone 3)
</TD><TD>55.9
</TD></TR><TR><TD>PCR56 in P175 (clone 4)
</TD><TD>120.9
</TD></TR><TR><TD>PCR56 in P175 (clone 5)
</TD><TD>119.4
</TD></TR><TR><TD>PCR 57 in P133 (clone 1)
</TD><TD>47.6
</TD></TR><TR><TD>PCR 57 in P133 (clone 2)
</TD><TD>185.1
</TD></TR><TR><TD>PCR 57 in P133 (clone 3)
</TD><TD>60.9
</TD></TR><TR><TD>PCR 57 in P133 (clone 4)
</TD><TD>61.3
</TD></TR><TR><TD>PCR 57 in P133 (clone 5)
</TD><TD>57.0
</TD></TR><TR><TD>PCR57 in P175 (clone 1)
</TD><TD>113.4
</TD></TR><TR><TD>PCR57 in P175 (clone 2)
</TD><TD>67.7
</TD></TR><TR><TD>PCR57 in P175 (clone 3)
</TD><TD>116.1
</TD></TR><TR><TD>PCR57 in P175 (clone 4)
</TD><TD>66.2
</TD></TR><TR><TD>PCR57 in P175 (clone 5)
</TD><TD>47.5
</TD></TR><TR><TD>PCR42 in P133 (clone 1)
</TD><TD>66.2
</TD></TR><TR><TD>PCR42 in P133 (clone 2)
</TD><TD>131.9
</TD></TR><TR><TD>PCR42 in P175 (clone 1)
</TD><TD>154.4
</TD></TR><TR><TD>PCR42 in P175 (clone 2)
</TD><TD>72.2
</TD></TR><TR><TD>PCR42 in P175 (clone 3)
</TD><TD>105.8
</TD></TR><TR><TD>PCR42 in P175 (clone 4)
</TD><TD>109.0
</TD></TR><TR><TD>PCR42 in P175 (clone 5)
</TD><TD>116.4
</TD></TR><TR><TD>PCR1 in P175 (clone 1)
</TD><TD>121.6
</TD></TR><TR><TD>PCR1 in P175 (clone 2)
</TD><TD>94.8
</TD></TR><TR><TD>PCR1 in P175 (clone 3)
</TD><TD>123.5
</TD></TR><TR><TD>PCR1 in P175 (clone 4)
</TD><TD>103.7
</TD></TR><TR><TD>PCR1 in P175 (clone 5)
</TD><TD>106.9
</TD></TR></TBODY></TABLE><P><B>Miniprep products are stored in a disposal bag in the middle drawer at -20°C. (label: Miniprep 21.08. Andrea)</B></P><UL><LI>numbering:
</LI></UL><P>1: PCR 56 in P133 clone 1
</P><P>2: PCR 56 in P133 clone 2
</P><P>3: PCR 56 in P133 clone 3
</P><P>4: PCR 56 in P133 clone 4
</P><P>5: PCR 56 in P133 clone 5
</P><P>6: PCR 56 in P175 clone 1
</P><P>7: PCR 56 in P175 clone 2
</P><P>8: PCR 56 in P175 clone 3
</P><P>9: PCR 56 in P175 clone 4
</P><P>10: PCR 56 in P175 clone 5
</P><P>11: PCR 57 in P133 clone 1
</P><P>12: PCR 57 in P133 clone 2
</P><P>13: PCR 57 in P133 clone 3
</P><P>14: PCR 57 in P133 clone 4
</P><P>15: PCR 57 in P133 clone 5
</P><P>16: PCR 57 in P175 clone 1
</P><P>17: PCR 57 in P175 clone 2
</P><P>18: PCR 57 in P175 clone 3
</P><P>19: PCR 57 in P175 clone 4
</P><P>20: PCR 57 in P175 clone 5
</P><P>21: PCR 42 in P133 clone 1
</P><P>22: PCR 42 in P133 clone 2
</P><P>23: PCR 42 in P175 clone 1
</P><P>24: PCR 42 in P175 clone 2
</P><P>25: PCR 42 in P175 clone 3
</P><P>26: PCR 42 in P175 clone 4
</P><P>27: PCR 42 in P175 clone 5
</P><P>28: PCR 1 in P175 clone 1
</P><P>29: PCR 1 in P175 clone 2
</P><P>30: PCR 1 in P175 clone 3
</P><P>31: PCR 1 in P175 clone 4
</P><P>32: PCR 1 in P175 clone 5
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_miniprep_.28miniprep_21.08..29">Analytical restriction digest of miniprep (miniprep 21.08.)</SPAN></H3><P><B>Investigators:</B> Andrea
</P><P><B>Aim:</B> Check whether ligations have worked (ligation of 17th August)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>Xba 1
</TD></TR><TR><TD>0.5 µl
</TD><TD>Spe-HF 1
</TD></TR><TR><TD>2 µl
</TD><TD>NEB
</TD></TR><TR><TD>0,2 µl
</TD><TD>BSA
</TD></TR><TR><TD>12.2 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>Mastermix for 33 reactions (9.9 µl XbaI, 9.9 µl SpeI, 66 µl NEB, 6.6 µl BSA, 402.6 ddH2O)
</LI><LI>5 µl DNA were added to 15 µl Mastermix
</LI></UL><P><B>Gelelectrophoresis:</B></P><UL><LI>expected: 
</LI></UL><P>Limonensynthase 1600 bp
</P><P>pYES2 (P175)       5800 bp
</P><P>pSB1C3 (P133)     2000 bp
</P><UL><LI>positive ligations: 
</LI></UL><P>Gel 2, PCR57 in P133 (pSB1C3), Klon 2
</P><P>Gel 3, PCR1 in P175 (pYES2), Klon 2
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_of_4CL_in_psB1C3_and_pTUM104_after_Quick_Change_from_August_17th">Miniprep of 4CL in psB1C3 and pTUM104 after Quick Change from August 17th</SPAN></H3><P><B>Investigator: Saskia, Daniela</B></P><P>Overnight culutres were grown too long. To obtain bacteria with plasmids we exchanged the media and let them grow for another 6 hours. A Miniprep was done afterwards.
</P><P>Using a Quiagen Miniprep kit, a plasmid isolation of three picked clones of a APT in pTUM104 RFC25 ligation was done. 
</P><P>The resulting plamid-concentrations (measured with nanodrop) were:
</P><P>4CL- pYES clone 1: 181.8 ng/µl
</P><P>4CL- pYES clone 2: 167.5 ng/µl
</P><P>4CL- pYES clone 3: 158.3 ng/µl
</P><P>4CL- pYES clone 4: 136.2 ng/µl
</P><P>4CL+ pYES clone 1: 88.4 ng/µl
</P><P>4CL+ pYES clone 2: 143.5 ng/µl
</P><P>4CL+ pYES clone 3: 144.4 ng/µl
</P><P>4CL+ pYES clone 4: 104.9 ng/µl
</P><P>4CL- pSB1C3 clone 1: 170.4 ng/µl
</P><P>4CL- pSB1C3 clone 2: 248.8 ng/µl
</P><P>4CL- pSB1C3 clone 3: 231.1 ng/µl
</P><P>4CL- pSB1C3 clone 4: 254.2 ng/µl
</P><P>4CL+ pSB1C3 clone 1: 272.8 ng/µl
</P><P>4CL+ pSB1C3 clone 2: 220.5 ng/µl
</P><P>4CL+ pSB1C3 clone 3: 220.1 ng/µl
</P><P>4CL+ pSB1C3 clone 4: 170.1 ng/µl
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytical_DNA-gelelectrophoresis_of_pSB1C3_Caffeine_involved_gene_and_pTUM104_caffeine_involved_gene_plasmids_after_digestion_with_PstI_and_XbaI"> Analytical DNA-gelelectrophoresis of pSB1C3_Caffeine_involved_gene and pTUM104_caffeine_involved_gene plasmids after digestion with PstI and XbaI</SPAN></H3><P><B>Investigator:</B> Roman, Saskia
</P><P><B>Aim:</B></P><P>Analytical DNA-gelelectrophoresis of B1A-D, B2A-D, B3A-D, Y1A-D, Y2A-D, Y3A-D with PstI ans XbaI to controll the ligation of the inserts in pSB1C3 and pTUM104.
</P><P><B>Procedure:</B>
Analytical gelelectrophoresis was performed at 90 V for 1 h.
</P><P><B>Gel 1 up:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kp ladder DNA ladder
</TD><TD><B>B1A</B></TD><TD><B>B1B</B></TD><TD><B>B1C</B></TD><TD><B>B1D</B></TD><TD><B>B2A</B></TD><TD><B>B2B</B></TD><TD><B>B2C</B></TD><TD><B>B2D</B></TD><TD><B>B3A</B></TD><TD><B>B3B</B></TD><TD><B>B3C</B></TD></TR></TBODY></TABLE><P><B>Gel 1 below:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kp ladder DNA ladder
</TD><TD><B>B3D</B></TD><TD><B>Y1A</B></TD><TD><B>Y1B</B></TD><TD><B>Y1C</B></TD><TD><B>Y1D</B></TD></TR></TBODY></TABLE><P><B>gel 2:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kp ladder DNA ladder
</TD><TD><B>Y2A</B></TD><TD><B>Y2B</B></TD><TD><B>Y2C</B></TD><TD><B>Y2D</B></TD><TD><B>Y3A</B></TD><TD><B>Y3B</B></TD><TD><B>Y3C</B></TD><TD><B>Y3D</B></TD></TR></TBODY></TABLE><P><B>Result:</B> All ligations were positive!
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_for_transformation_of_pTUM104_with_caffeine_genes_in_yeast"> Preparation for transformation of pTUM104 with caffeine genes in yeast</SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim:</B></P><P>Transformation of <I>S. cerevisiae</I> INVSc1 with pTUM104 with CaXMT1, CaMXMT1 and CaDXMT1.
</P><P><B>Procedure:</B></P><P>production of YPD medium as described in the pYES2 manual
</P><P>Incubation of <I>S. cerevisiae</I> INVSc1 in 4 ml YPD as described in the pYES manual
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Minipreparation_of_pTUM104_Preprothaumatin_RFC10_and_pSB1C3_Preprothaumatin_RFC10"> Minipreparation of pTUM104_Preprothaumatin_RFC10 and pSB1C3_Preprothaumatin_RFC10</SPAN></H3><P><B>Investigator:</B> Martianus Capella, Albertus Magnus
</P><UL><LI> Inoculation.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_22nd"><B>Wednesday, August 22nd</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="PCR_of_P395_to_introduce_RFC10_pre-_and_suffix_to_finished_SV40-Gal4AD-Linker-Pif3_construct"> PCR of P395 to introduce RFC10 pre- and suffix to finished SV40-Gal4AD-Linker-Pif3 construct </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> PCR of P395 to introduce RFC10 pre- and suffix to finished SV40-Gal4AD-Linker-Pif3 construct for further cloning into pSB1C3 for partsregistry.
</P><P><B>Procedure:</B></P><UL><LI> PCR reaction batch was prepared as follows:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>substrate</B></TD><TD><B>volume in µl</B></TD></TR><TR><TD>One Taq Standard Reaction Buffer (5x)
</TD><TD>10
</TD></TR><TR><TD>Plasmid template (P395)
</TD><TD>1
</TD></TR><TR><TD>Primer O75 (10 µM)
</TD><TD>1
</TD></TR><TR><TD>Primer O76 (10 µM)
</TD><TD>1
</TD></TR><TR><TD>dNTP (10 mM)
</TD><TD>1
</TD></TR><TR><TD>Polymerase One Taq
</TD><TD>0,25
</TD></TR><TR><TD>ddH2O
</TD><TD>35.75
</TD></TR></TBODY></TABLE><UL><LI> The PCR program was performed after following scheme with following conditions
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>57 °C
</TD><TD>60 s
</TD></TR><TR><TD>68 °C
</TD><TD>60 s
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><UL><LI> After PCR, PCR purification kit from Qiagen was used.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Sequencing_results_of_P395.2C_P398.2C_P422.2C_P331"> Sequencing results of P395, P398, P422, P331 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Results:</B></P><UL><LI> P395:
</LI></UL><PRE>CCATACGACGTACCAGATTACGCTCATATGGCCATGGAGGCCAGTGAATTGCCTCTGTTT
GAACTTTTCAGGCTCACCAAAGCTAAGCTTGAATCTGCTCAAGACAGGAACCCTTCTCCA
CCTGTAGATGAAGTTGTGGAGCTGGTGTGGGAAAATGGTCAGATATCAACTCAAAGTCAG
TCAAGTAGATCGAGGAACATTCCTCCACCACAAGCAAACTCTTCAAGAGCTAGAGAGATT
GGAAATGGCTCAAAGACGACTATGGTGGACGAGATCCCTATGTCAGTGCCATCACTAATG
ACGGGTTTGAGTCAAGACGATGACTTTGTTCCATGGTTGAATCATCATTAAGGATCCATC
GAGCTCGAGCTGCAGATGAATCGTAGATACTGAAAAACCCCGCAAGTTCACTTCAACTGT
GCATCGTGCACCATCTCAATTTCTTTCATTTATACATCGTTTTGCCTTCTTTTATGTAAC
TATACTCCTCTAAGTTTCAATCTTGGCCATGTAACCTCTGATCTATAGAATTTTTTAAAT
GACTAGAATTAATGCCCATCTTTTTTTTGGACCTAAATTCTTCATGAAAATATATTACGA
GGGCTTATTCAGAAGCTTTGGACTTCTTCGCCAGAGGTTTGGTCAAGTCTCCAATCAAGG
TTGTCGGCTTGTCTACCTTGCCAGAAATTTACGAAAAGATGGAAAAGGGTCAAATCGTTG
GTAGATACGTTGTTGACACTTCTAAATAAGCGAATTTCTTATGATTTATGATTTTTATTA
TTAAATAAGTTATAAAAAAAATAAGTGTATACAAATTTTAAAGTGACTCTTAGTTTTAAA
ACGAAAATTCTTATTCTTGAGTAACTCTTTCCTGTAGGTCAGTT
</PRE><P>Sequence of interest matches the expectation
</P><UL><LI> P398:
</LI></UL><PRE>CATACGACGTACCAGATTACGCTCATATGGCCATGGAGGCCAGTGAATTGCCTCTGTTTG
AACTTTTCAGGCTCACCAAAGCTAAGCTTGAATCTGCTCAAGACAGGAACCCTTCTCCAC
CTGTAGATGAAGTTGTGGAGCTGGTGTGGGAAAATGGTCAGATATCAACTCAAAGTCAGT
CAAGTAGATCGAGGAACATTCCTCCACCACAAGCAAACTCTTCAAGAGCTAGAGAGATTG
GAAATGGCTCAAAGACGACTATGGTGGACGAGATCCCTATGTCAGTGCCATCACTAATGA
CGGGTTTGAGTCAAGACGATGACTTTGTTCCATGGTTGAATCATCATTAAGGATCCATCG
AGCTCGAGCTGCAGATGAATCGTAGATACTGAAAAACCCCGCAAGTTCACTTCAACTGTG
CATCGTGCACCATCTCAATTTCTTTCATTTATACATCGTTTTGCCTTCTTTTATGTAACT
ATACTCCTCTAAGTTTCAATCTTGGCCATGTAACCTCTGATCTATAGAATTTTTTAAATG
ACTAGAATTAATGCCCATCTTTTTTTTGGACCTAAATTCTTCATGAAAATATATTACGAG
GGCTTATTCAGAAGCTTTGGACTTCTTCGCCAGAGGTTTGGTCAAGTCTCCAATCAAGGT
TGTCGGCTTGTCTACCTTGCCAGAAATTTACGAAAAGATGGAAAAGGGTCAAATCGTTGG
TAGATACGTTGTTGACACTTCTAAATAAGCGAATTTCTTATGATTTATGATTTTTATTAT
TAAATAAGTTATAAAAAAAATAAGTGTATACAAATTTTAAAGTGACTCTTAGTTTTAAAA
CGAAAATTCTTATTCTTGAGTAACTCTTTCCTGTAGTCAGTTGCTTTCTCAGGTATAGCA
TGAGGTCGCTCTTATTGACCACACCTCTACCGGCCGGTCGAAATTCCCCTACCCTATGAA
CATATTCCATTTTGTAATTTCGTGTCGTTTCTATTATGATTTCATTTATAAAGTTTATGT
ACAATATCAT
</PRE><P>Sequence of interest matches the expectation
</P><UL><LI> P422:
</LI></UL><PRE>TATAAAATAGGCGTATCACGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTA
AGGATGATTTCTGGAATTCGCGGCCGCTTCTAGATGGCCGGCGTTTCCGGAGTCGGGGGT
AGTGGCGGTGGCCGTGGCGGTGGCCGTGGCGGAGAAGAAGAACCGTCGTCAAGTCACACT
CCTAATAACCGAAGAGGAGGAGAACAAGCTCAATCGTCGGGAACGAAATCTCTCAGACCA
AGAAGCAACACTGAATCAATGAGCAAAGCAATTCAACAGTACACCGTCGACGCAAGACTC
CACGCCGTTTTCGAACAATCCGGCGAATCAGGGAAATCATTCGACTACTCACAATCACTC
AAAACGACGACGTACGGTTCCTCTGTACCTGAGCAACAGATCACAGCTTATCTCTCTCGA
ATCCAGCGAGGTGGTTACATTCAGCCTTTCGGATGTATGATCGCCGTCGATGAATCCAGT
TTCCGGATCATCGGTTACAGTGAAAACGCCAGAGAAATGTTAGGGATTATGCCTCAATCT
GTTCCTACTCTTGAGAAACCTGAGATTCTAGCTATGGGAACTGATGTGAGATCTTTGTTC
ACTTCTTCGAGCTCGATTCTACTCGAGCGTGCTTTCGTTGCTCGAGAGATTACCTTGTTA
AATCCGGTTTGGATCCATTCCAAGAATACTGGTAAACCGTTTTACGCCATTCTTCATAGG
ATTGATGTTGGTGTTGTTATTGATTTAGAGCCAGCTAGAACTGAAGATCCTGCGCTTTCT
ATTGCTGGTGCTGTTCAATCGCAGAAACTCGCGGTTCGTGCGATTTCTCAGTTACAGGCT
CTTCCTGGTGGAGATATTAAGCTTTTGTGTGACACTGTCGTGGAAAGTGTGAGGGACTTG
ACTGGTTATGATCGTGTTATGGTTTATAAGTTTCATGAAGATGAGCATGGAGAAGTTGTA
GCTGAGAGTAAACGAGACGATTTAGAGCCTTATA
</PRE><P>Partial sequence match the part (part is too long for one read)
</P><UL><LI> P331:
</LI></UL><PRE>TATAAAATAGGCGTATCACGAGGCAGAATTTCAGATAAAAAAAATCCTTAGCTTTCGCTA
AGGATGATTTCTGGAATTCGCGGCCGCTTCTAGATGGCCGGCGCGAGCGGCGCGGGCGGC
AGCGAAGGCGGCGGCAGCGAAGGCGGCACCAGCGGCGCGACCACCGGCAAAGCGTTAACG
GCCAGGCAACAAGAGGTGTTTGATCTCATCCGTGATCACATCAGCCAGACAGGTATGCCG
CCGACGCGTGCGGAAATCGCGCAGCGTTTGGGGTTCCGTTCCCCAAACGCGGCTGAAGAA
CATCTGAAGGCGCTGGCACGCAAAGGCGTTATTGAAATTGTTTCCGGCGCATCACGCGGG
ATTCGTCTGTTGCAGGAAGAGGAAGAAGGGTTGCCGCTGGTAGGTCGTGTGGCTGCCGGT
GAACCACTTCTGGCGCAACAGCATATTGAAGGTCATTATCAGGTCGATCCTTCCTTATTC
AAGCCGAATGCTGATTTCCTGCTGCGCGTCAGCGGGATGTCGATGAAAGATATCGGCATT
ATGGATGGTGACTTGCTGGCAGTGCATAAAACTCAGGATGTACGTAACGGTCAGGTCGTT
GTCGCACGTATTGATGACGAGGTTACCGTTAAGCGCCTGAAAAAACAGGGCAATAAAGTC
GAACTGTTGCCAGAAAATAGCGAGTTTAAACCAATTGTCGTAGATCTTCGTCAGCAGAGC
TTCACCATTGAAGGGCTGGCGGTTGGGGTTATTCGCAACGGCGACTGGCTGACCGGTTAA
TACTAGTAGCGGCCGCTGCAGTCCGGCAAAAAAACGGCAAGGTGTCACCACCCTGCCCTT
TTTCTTTAAAACCGAAAAGATTACTTCGCGTTATGCAGCTTCCTCGCTCACTGACTCGCT
GCGCTCGGTCGTTCGGCTGCGGCGAGCGGTATCAGCTCACTCAAAGGCGGTAATACGGTT
ATCCACAGAATCAGGGGATAACGCAGGAAGACAT
</PRE><P>3 synonymous well-known point-mutation, the rest matches expectation.
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_of_P436.2BP437_and_P206.2BP437"> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P436+P437 and P206+P437 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P436+P437 and P206+P437.
</P><P><B>Procedure:</B></P><UL><LI> 6 colonies of each transformation from P207+PCR69 (CamR) and P207+PCR70 (CamR) transformated cells were picked.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chloramphenicol. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_P422"> Transformation of <I>E. coli</I> XL1-Blue with P422 </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B> Transform <I>E. coli</I> XL 1 blue with P422 to backup it.
</P><P><B>Procedure:</B></P><UL><LI> thawing of 100 µl Ca-competent E.coli XL1-Blue cells on ice
</LI><LI> adding of 1 µl of DNA
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> adding of 1 ml LB-medium without antibiotics to the cells and incubation at 37°C and 180 rpm for 45 min
</LI><LI> plating of 100 µl of the cells on an Chloramphenicol-plate
</LI><LI> sedimentation of the leftover in a centrifuge (2 min, 13 000 rpm), resuspension of the sediment in 100 µl LB-medium and plating it as well on an antibiotic containing LB-plate 
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P451_.28TEF2-P_in_psb1c3.29">Preparative digestion of P451 (TEF2-P in psb1c3)</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (NEB)
</TD></TR><TR><TD>4 µl
</TD><TD>10xNEB-4 buffer
</TD></TR><TR><TD>0,4 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>20 µl
</TD><TD>P 451
</TD></TR><TR><TD>13,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=40µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> to 20 of DNA 20 µl reaction batch were added
</LI><LI> Digestion took place at 37°C for 3 h
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_measurement_of_ADH1-P">Nanodrop measurement of ADH1-P</SPAN></H3><P><B> Investigator</B>: Georg
</P><UL><LI> ADH1-P (720 bp) Kol1: 24,9 ng/µl
</LI><LI> ADH1-P (720 bp) Kol2: 38,6 ng/µl
</LI><LI> ADH1-P (720 bp) Kol3: 42,1 ng/µl
</LI><LI> ADH1-P (720 bp) Kol4: 42,5 ng/µl
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_gel_of_TEF1-P_.28P309.29.2C_TEF2-P_.28P451.29_in_psb1c3_and_pTUM104_without_GAL-P_and_f1_.28pTUM100.29">Preparative gel of TEF1-P (P309), TEF2-P (P451) in psb1c3 and pTUM104 without GAL-P and f1 (pTUM100)</SPAN></H3><UL><LI> Preparative digests were loaded onto 1% preparative agarose gels
</LI><LI> Gel run at 70 V for 3 hours
</LI><LI> DNA bands were cut out 
</LI><LI> DNA was extracted according to Quiaquick gel purification Kit (salt removed)
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_Xl1-blue_competent_cells_with_Preprothaumatin.2C_TEF2-P_.28P351.29.2C_and_ADH1-P_.28720_bp.29in_psb1c3"> Transformation of E.coli Xl1-blue competent cells with Preprothaumatin, TEF2-P (P351), and ADH1-P (720 bp)in psb1c3</SPAN></H3><P><B>Investigator</B>: Georg
</P><P><B>Aim of the experiment</B>: To get a stock of TEF2p, Thaumatin and ADH1-P (720 bp)
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 1 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on Chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new Chloramphenicol plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_Digestion_of_P375_.28pTUM104.29">Preparative Digestion of P375 (pTUM104)</SPAN></H3><P><B>Investigator:</B> Georg
</P><P><B>Aim of the experiment</B>: Digestion of pYesII with SwaI and PvuII for new ligation reaction to generat pYesII-TUM
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>SwaI(NEB)
</TD></TR><TR><TD>2 µl
</TD><TD>PvuII (NEB)
</TD></TR><TR><TD>4 µl
</TD><TD>NEB-3 buffer
</TD></TR><TR><TD>0,4 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>11,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=40µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> pTUM104 was first digested with SwaI and BSA at room temperature. Afterwards PvuII was added and digested for another 3 hours at 37°C
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Control_digest_of_pTUM104_Preprothaumatin_and_pSB1C3_Preprothaumatin"> Control digest of pTUM104_Preprothaumatin and pSB1C3_Preprothaumatin</SPAN></H3><P><B>Investigator:</B> Cicero, Platon
</P><P><B>Aim:</B> Is the insert integrated?
</P><UL><LI> Miniprep of p453-p462.
</LI></UL><P>Analytical digest: 
</P><UL><LI> pSB1C3_Preprothaumatin: 6.25 µl (p453-457), 2 µl NEBuffer 4, 0.2 µl BSA, 0.25 µl XbaI. 0.25 µl SpeI, 11 µl ddH2O.
</LI><LI> pYes2_Preprothaumatin: 6.25 µl (p458-462), 2 µl NEBuffer 4, 0.2 µl BSA, 0.25 µl XbaI. 0.25 µl SpeI, 11 µl ddH2O.
</LI><LI> 37°C, 1h.
</LI></UL><P>Analytical gel electrophoresis: 1 kb marker, p453, p454, p455, p456, p457, p458, p459, p459, p460, p461, p462
</P><UL><LI> Expected: pSB1C3_Preprothaumatin: backbone: 2.1 kb, insert 721 bp; pYes2_Preprothaumatin: backbone: 4.8 kb, insert: 721 bp.
</LI></UL><P><B>Result:</B> p456 and p457 were successful, the insert preprothamatin is integrated into pSB1C3.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Small_scale_yeast_transformation_with_pTUM104_CaXMT1.2C_pTUM104_CaMXMT1_and_pTUM104_CaDXMT1"> Small scale yeast transformation with pTUM104_CaXMT1, pTUM104_CaMXMT1 and pTUM104_CaDXMT1 </SPAN></H3><P><B>Investigator:</B> Saskia, Roman
</P><P><B>Aim:</B> To transform our yeast strain <I>saccharomyces cerevisiae</I> INVSC1 -URA with the created pTUM104 plasmids, to be able to start expression of our genes.
</P><P><B>Operational sequence:</B></P><UL><LI> The transformation of the yeast cells was performed as described in the pYES2 Manual of Invitrogen (small scale  yeast transformation). 
</LI></UL><UL><LI> Used plasmids pTUM104_Insert were previously isolated out of clones: Y1B, Y2D, Y3A
</LI></UL><UL><LI> As positive control, we used a pTUM104_eGFP plasmid (provided by Simon) and as negative control, no plasmid (water) was &quot;transformed&quot;.
</LI></UL><UL><LI> 1xTE and 1xLiAc/40%PEG-3350/1xTE solution were created immediately before use and sterilized by steril- filtration. The other solutions had already been available (and had also been sterilized by steril- filtration).
</LI></UL><UL><LI> The resuspended pellets were plated on adequate selective SC -URA plates (minimal, defined medium for yeast, without uracil, due to selection for pTUM104 vector).
</LI></UL><UL><LI> Plates were incubated at 30°C over night
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_SC_Minimal_Medium"> Preparation of SC Minimal Medium </SPAN></H3><P><B>Investigator:</B> Saskia, Roman
</P><P>We prepared 3 l of SC Minimal Medium as described in the pYES manual on page 10.
We mixed and autoclaved 50 ml of the already prepared SC-aliquots with 850 ml ELGA and 6.7 g Yeast Nitrogen Base.
Additionally we prepared a 20 % glucose and two 20 % galactose solutions. Therefore we mixed 20 g galactose or glucose with 100 ml ELGA and autoclaved it.
Note that you need more induction media with galactose than SC medium with glucose. And we don't use raffinose for the induction media.
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Western_Blot_of_APT_at_different_times_of_expression_in_yeast">Western Blot of APT at different times of expression in yeast</SPAN></H3><P><B>Investigator:</B> Daniela
</P><P><B>Aim of the experiment:</B> Find out optimum time for expression.
</P><P><B>SDS-PAGE:</B></P><P>Samples:
</P><UL><LI>APT after 0 h
</LI><LI>APT after 2 h -&gt; not loaded correctly
</LI><LI>APT after 4 h
</LI><LI>APT after 6 h
</LI><LI>APT after 8 h
</LI><LI>APT after 11 h
</LI><LI>APT after 21 h
</LI><LI>GFP after 8 h (positive control)
</LI></UL><P>10 µl of the samples (supernatant of the lysis reaction -&gt; see August, 15th) were mixed with 2.5 µl 5xLaemmlired and denatured at 95 °C for 5 min.
</P><UL><LI>12 % gel
</LI><LI>120 V, 1.5 h
</LI></UL><P><B>Blotting:</B></P><UL><LI>blotting of proteins on a nitrocellulose membrane (Whatman)
</LI><LI>50mA, 1 h
</LI></UL><P><B>Blocking:</B></P><UL><LI>wash 3x10min with PBS-T0.1
</LI><LI>the membrane is blocked over night at 4 °C on a shaking device
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Control_digest_of_4CL_in_pTUM104_and_pSB1C3-RFC25_after_Quick_Change_from_August_17th">Control digest of 4CL in pTUM104 and pSB1C3-RFC25 after Quick Change from August 17th</SPAN></H3><P>Investigator: Daniela
</P><P><B>Aim of the experiment:</B> Test whether the Quick Change was successful
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>miniprep products
</TD></TR><TR><TD>0.25 µl
</TD><TD>Eco RI HF (20U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI HF (20U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4
</TD></TR><TR><TD>15 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Gel 1: 4CL in pYES</B></P><P>lane 1: 4CL- in pYES clone 1
</P><P>lane 2: 4CL- in pYES clone 2
</P><P>lane 3: 4CL- in pYES clone 3
</P><P>lane 4: 4CL- in pYES clone 4
</P><P>lane 5: 4CL+ in pYES clone 1
</P><P>lane 6: 4CL+ in pYES clone 2
</P><P>lane 7: 4CL+ in pYES clone 3
</P><P>lane 8: 4CL+ in pYES clone 4
</P><P><B>Gel 2: 4CL in pSB1C3-RFC25</B></P><P>lane 1: 4CL- in pSB1C3 clone 1
</P><P>lane 2: 4CL- in pSB1C3 clone 2
</P><P>lane 3: 4CL- in pSB1C3 clone 3
</P><P>lane 4: 4CL- in pSB1C3 clone 4
</P><P>lane 5: 4CL+ in pSB1C3 clone 1
</P><P>lane 6: 4CL+ in pSB1C3 clone 2
</P><P>lane 7: 4CL+ in pSB1C3 clone 3
</P><P>lane 8: 4CL+ in pSB1C3 clone 4
</P><P>The results are not unambiguous. Nevertheless the last Quick Change will be done on Friday 24th.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Repitition_of_analytical_restriction_digest_and_gel_electrophoresis_of_miniprep_.28miniprep_21.08..29">Repitition of analytical restriction digest and gel electrophoresis of miniprep (miniprep 21.08.)</SPAN></H3><P><B>Investigators:</B> Andrea
</P><P><B>Aim:</B> Check whether ligations have worked (ligation of 17th August)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>Xba 1
</TD></TR><TR><TD>0.5 µl
</TD><TD>Spe-HF 1
</TD></TR><TR><TD>2 µl
</TD><TD>NEB
</TD></TR><TR><TD>0,2 µl
</TD><TD>BSA
</TD></TR><TR><TD>12.2 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Gelelectrophoresis:</B></P><UL><LI>expected: 
</LI></UL><P>Limonensynthase 1600 bp
</P><P>pYES2 (P175)       5800 bp
</P><P>pSB1C3 (P133)     2000 bp
</P><UL><LI>positive ligations: 
</LI></UL><P>PCR1 in P175 (pYES2), Klon 2
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_positive_ligation_products">Transformation of positive ligation products</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Transform E.coli XL 1 blue with ligation products (PCR58/p133 (Klon20) (Miniprep: 21rst August), PCR2/p133 (Klon 4) (Miniprep: 8th August))  to produce colonies containing plasmid DNA.
</P><P><B>Transformation into E.coli Xl1-Blue</B></P><UL><LI> thawing of 100 µl Ca-competent E.coli XL1-Blue cells on ice
</LI><LI> adding of 1 µl of DNA
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> adding of 1 ml LB-medium without antibiotics to the cells and incubation at 37°C and 180 rpm for 45 min
</LI><LI> plating of 100 µl of the cells on an Chloramphenicol-plate (pSB1C3, P133)
</LI><LI> sedimentation of the leftover in a centrifuge (2 min, 13 000 rpm), resuspension of the sediment in 100 µl LB-medium and plating it as well on an antibiotic containing LB-plate 
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Breeding_of_positive_clones_of_colony_PCR_of_21rst_August">Breeding of positive clones of colony PCR of 21rst August</SPAN></H3><P><B>Investigator:</B> Andrea
</P><UL><LI>clone 1 / 3 / 5 / 11 were stored at 4°C in 2 ml LB medium
</LI><LI>the clone suspension were added with the pipet tip to 2 ml fresh LB medium including antibiotics (Chlp) into culture tubes
</LI><LI>tube incubate at 37°C and 180 rpm
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Preparative_digest_of_pSB1C3_Preprothaumatin_.28p456.29_and_ligation_with_p400_.28pTUM104_digested_with_XbaI_and_PstI.29"> Preparative digest of pSB1C3_Preprothaumatin (p456) and ligation with p400 (pTUM104 digested with XbaI and PstI)</SPAN></H3><P><B>Investigator:</B> Martin, Dennis, Alois
</P><P><B>Aim:</B> Getting pYes2_Preprothaumatin
</P><P>Preparative digest of p456:
</P><UL><LI> pSB1C3_Preprothaumatin: 25 µl p456, 5 µl NEBuffer 4, 0.5 µl BSA, 1 µl XbaI, 1 µl PstI, 12.5 µl ddH2O.
</LI><LI> 37°C, 2h.
</LI></UL><P>Preparative gel electrophoresis: p456 (digested with XbaI and PstI), 1 kb marker
</P><UL><LI> Expected: pSB1C3_Preprothaumatin: backbone: 2.1 kb, insert 721 bp; successful.
</LI></UL><UL><LI> Gel extraction, NanoDrop 2000.
</LI></UL><P>Ligation:
</P><UL><LI> 1.7 µl p400 (digested with XbaI and PstI), 5 µl p456 (digested with XbaI and PstI), 1 µl 10x T4-DNA ligase, 1 µl T4-DNA ligase buffer, 10.3 µl ddH2O. Additional: negative control without insert.
</LI><LI> 16°C over night.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_23rd"><B>Thursday, August 23rd</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_for_ADH1-P_.28720_bp.29"> Analytical gelelectrophoresis for ADH1-P (720 bp)</SPAN></H3><UL><LI> Undigested ADH1-P and PCR-Products of TEF2-P were loaded
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparation_of_Probes_for_sequencing_2"> Preparation of Probes for sequencing</SPAN></H3><P><B>Investigator</B>: Georg
</P><P><B> Aim of this work</B>: Concentration of DNA of 50-100 ng/µl 
</P><UL><LI>ADH1-P (186,1 ng/µl) : 5 µl DNA + 10 µl H2O   (Primer Vf2 and VR)
</LI><LI>ADH1-T (P403) (153,5 ng/µl) : 10 µl DNA + 10 µl H20 (VF2)
</LI><LI>TEF1T (P465): 216,5 ng/µl : 7,5 µl DNA + 12,5 µl H20 (VF2) (Ist doch TEF1-P)
</LI><LI>P(231) (207,8ng/µl) : 5 µl DNA+ 10 µl H20
</LI><LI>TEF2-P (P451) 390 ng/µl: 3 µl DNA + 12 µl H20
</LI><LI>TEF2-P (P450) 116 ng/µl : 7,5 µl + 7,5 µl H20
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_preparative_digested_TEF1-P.2C_TEF2-P_in_psb1c3.2C_and_digested_pTUM104"> Gelextraction of preparative digested TEF1-P, TEF2-P in psb1c3, and digested pTUM104</SPAN></H3><P><B>Investigator</B>: Georg
</P><P><B>Aim of experiment</B>: Extraction of preparative digested TEF1-P, TEF2-P in psb1c3, and digested pYES2-TUM
</P><P><B>Investigator</B>: Georg 
</P><UL><LI>  Extraction was done according to Quiaquick gel extraction kit ( with elimination of salt)
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_Measurement_of_TEF1-P_and_TEF2-P"> Nanodrop Measurement of TEF1-P and TEF2-P</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI>TEF1-P= 57,5 ng/µl
</LI><LI>TEF2-P= P 91,6 ng/ µl
</LI><LI>Pyes2-Vector: 34,4 ng /µl (salt removed)
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Self-ligation_.28self-circularization.29_of_pTUM100"> Self-ligation (self-circularization) of pTUM100 </SPAN></H3><P><B> Aim of the experiment:</B> Ligation of pYESII-TUM for transformation in E.coli XL1-blue
Since no colonies were seen from the last transformation, a new reaction batch for ligase reaction was made.
</P><UL><LI>Two reaction batches with 100 and 50 ng linearized Vector were made
</LI></UL><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation (100 ng):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10,0 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>5,8 (=200 ng)
</TD><TD>PYesII digested (salt removed)
</TD></TR><TR><TD>10,0
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>10 µl
</TD><TD>50% PEG 4000
</TD></TR><TR><TD>64,1 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=100µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation (50 ng):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10,0 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>2,9 (=100 ng)
</TD><TD>PYesII digested (salt removed)
</TD></TR><TR><TD>10,0
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>10 µl
</TD><TD>50% PEG 4000
</TD></TR><TR><TD>67,1 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=100µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Both batches were divided into a reaction at room temperature for 1h and 16 C over night.
</P></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_ADH1-P_.28720_bp.29in_psb1c3_with_SpeI_and_PSTI">Preparative digestion of ADH1-P (720 bp)in psb1c3 with SpeI and PSTI</SPAN></H3><P><B>Aim of experiment</B>: Promoter ready for Ligation with proteine
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>4 µl
</TD><TD>SpeI (NEB)
</TD></TR><TR><TD>16 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>1,6 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>54,4 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=80µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>20 µl of DNA were mixed with 20 µl of digestion batch. Digestion was performed for 3 hours at 37 C. 
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_gel_with_ADH1-P_.28720_bp.29">Preparative gel with ADH1-P (720 bp)</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI> 40 µl of each preparative digestion was loaded onto a preparative 1% Agarose gel.
</LI><LI> Gel war run at 70 V for 1 h
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_EYFP_.28Dist._Kit.29_ECFP_.28Distr._Kit.29.2C_Lig._pTUM100_1h.2C_50ng_and_100_ng_of_E.coli_Xl-1_blue">Transformation EYFP (Dist. Kit) ECFP (Distr. Kit), Lig. pTUM100 1h, 50ng and 100 ng of E.coli Xl-1 blue</SPAN></H3><P><B>Investigator</B>: Georg
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product pYESII, and 2 µl of EYFP and ECFP were added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on ampicillin plates (pYESII and ECFP)and Kanamycin plates EYFP.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new  antibiotic plates.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_Xl1-blue_competent_cells_with_ligation_batches_at_room_temperature">Transformation of E.coli Xl1-blue competent cells with ligation batches at room temperature</SPAN></H3><P>Investigator: Georg
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on ampicillin plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin plates.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_culture_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_of_P436.2BP437_and_P206.2BP437"> Miniprep of overnight culture of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P436+P437 and P206+P437  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of overnight culture of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P436+P437 and P206+P437.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep of P436+P437 (6 clones picked) and P206+P437 (6 clones picked) was performed with Qiagen Qiaprep spin miniprep kit after manufacturer's protocol.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P468-P473"> Analytical digestion and gelelectrophoresis of P468-P473 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P468-P473 (miniprep of <I>E. coli</I>, transformed with P436+P437 ligation product) to prove whether ligation was succesful.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with NgoMIV+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>26 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>3.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>3.25 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>195 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=227.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of samples P468-P473 in a new ERG and was mixed gently by pipetting up and down.
</LI></UL><UL><LI> 90 min incubation at 37 °C.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P468</B></TD><TD><B>P469</B></TD><TD><B>P470</B></TD><TD><B>P471</B></TD><TD><B>P472</B></TD><TD><B>P473</B></TD><TD>1 kbp ladder DNA ladder
</TD><TD><B>PCR71</B></TD></TR><TR><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD><B>Please see subsection for analytical gelelectrophoresis for PCR71</B></TD></TR></TBODY></TABLE><UL><LI> Ligation/Fusion of P436+P437 has been successful for all clones (P468-P473). N'-PhyB(908NT)-20aaLinker-LexA-C' is ready for last fusion to the SV40 nuclear localization signal sequence.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P474-P479"> Analytical digestion and gelelectrophoresis of P474-P479 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P474-P479 (miniprep of <I>E. coli</I>, transformed with P206+P438 ligation product) to prove whether ligation was succesful.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with NgoMIV+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>26 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>3.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>3.25 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>195 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=227.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of samples P474-P479 in a new ERG and was mixed gently by pipetting up and down.
</LI></UL><UL><LI> 90 min incubation at 37 °C.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P474</B></TD><TD><B>P475</B></TD><TD><B>P476</B></TD><TD><B>P477</B></TD><TD><B>P478</B></TD><TD><B>P479</B></TD><TD><B>P438</B></TD><TD>1 kbp ladder DNA ladder
</TD></TR><TR><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD><B>Ligation successful!</B></TD><TD>Control
</TD></TR></TBODY></TABLE><UL><LI> Ligation/Fusion of P206+P438 has been successful for all clones (P474-P479). N'-20aaLinker-Gal4DBD-C' is ready for (last?) fusion to the PhyB(908NT).
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_PCR71"> Analytical gelelectrophoresis of PCR71 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis of PCR71, to check whether PCR was successful.
</P><P><B>Procedure:</B></P><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp ladder DNA ladder
</TD><TD><B>P468</B></TD><TD><B>P469</B></TD><TD><B>P470</B></TD><TD><B>P471</B></TD><TD><B>P472</B></TD><TD><B>P473</B></TD><TD>1 kbp ladder DNA ladder
</TD><TD><B>PCR71</B></TD></TR><TR><TD>Please see second last subsection
</TD><TD>Please see second last subsection
</TD><TD>Please see second last subsection
</TD><TD>Please see second last subsection
</TD><TD>Please see second last subsection
</TD><TD>Please see second last subsection
</TD><TD><B>PCR was successful!</B></TD></TR></TBODY></TABLE><UL><LI> PCR was successful to introduce RFC10 pre- and suffix, N'-SV40NLS-Gal4AD-Linker-Pif3-C' is ready for ligation to pSB1C3 as a new part in the partsregistry.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophoresis_of_P475"> Preperative digestion and gelelectrophoresis of P475 </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Preperative digestion and gelelectrophoresis of P475 with NgoMIV+PstI-HF for further fusion to C-terminus of PhyB(908NT).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for preperative digestion of P475 with NgoMIV+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid P475
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> This reaction batch has been incubated for 90 min at 37 °C.
</LI></UL><UL><LI> Preperative gelelectrophoresis has been performed at 70 V for 90 min.
</LI></UL><UL><LI> Insert has been cut out and extracted with Qiagen gel extraction kit.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P436.2BP480"> Ligation of P436+P480 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of P436+P480. Final construct: N'-PhyB-20aaLinker-Gal4DBD-C'. Maybe also fusion to SV40NLS is nescessary (normally Gal4DBD itself contains a own nuclear localization signal), if nuclear import is not sufficient enough because NLS of Gal4DBD is hidden by N-terminal linker and PhyB(908NT).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for ligation of P436+P480:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.11 µl
</TD><TD>P436
</TD></TR><TR><TD>6.43 µl
</TD><TD>P480
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>9.46 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Also negative control was prepared with the same reagents and volume but instead of the insert (P480) ddH2O was used.
</LI></UL><UL><LI> Ligation was performed at 16 °C overnight.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_of_PCR58_.2F_P133_Klon20_.28lavendula_LS_in_pSB1C3.2C_Miniprep_21rst_August.29_and_PCR2_.2F_P133_Klon4_.28citrus_LS_in_pSB1C3.2C_Miniprep_8th_August.29_and_P373_.28pTUM104.29">Preparative restriction digest of PCR58 / P133 Klon20 (lavendula LS in pSB1C3, Miniprep 21rst August) and PCR2 / P133 Klon4 (citrus LS in pSB1C3, Miniprep 8th August) and P373 (pTUM104)</SPAN></H3><P><B>Investigator:</B> Lara (digest), Andrea (gel electrophoresis)
</P><P><B>Aim:</B> Prepare digested PCR fragments for further ligation into pYES.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>PCR2 / P133 clone 4 (8.8.)
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>23.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>17 µl
</TD><TD>PCR58 / P133 (clone 20)
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>16.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P375
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>4 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>9.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Incubation at 37°C for 3 h.
</LI></UL><P>expected:
</P><P>Limonensynthase: 1600 bp
</P><P>pYES2: 5800 bp
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_PCR2.2FP133_.28Miniprep_08.08..29_and_PCR58.2F133_.28Miniprep_21.08..29_in_pTUM104_.28P375.29">Ligation of PCR2/P133 (Miniprep 08.08.) and PCR58/133 (Miniprep 21.08.) in pTUM104 (P375)</SPAN></H3><P><B>Investigator:</B> Andrea
</P><UL><LI>concentrations after gel extraction:
</LI></UL><P>PCR2: 15.0 ng/µl
</P><P>PCR58: 9.1 ng/µl
</P><P>pYES2 (P375): 17.1 ng/µl
</P><UL><LI>stored in 50ml Falcon (label &quot;Ligierung 23.08. Andrea&quot;)
</LI></UL><P>PCR2 in P375 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.78µl
</TD><TD>P375
</TD></TR><TR><TD>5.22 µl
</TD><TD>PCR 2
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>PCR58 in P375 digested with XbaI and AgeI/NgoMIV
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.95 µl
</TD><TD>P375
</TD></TR><TR><TD>6.05 µl
</TD><TD>PCR58
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><P>Negative control
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>P375
</TD></TR><TR><TD>6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>1 µl
</TD><TD>T4 DNA Ligase
</TD></TR><TR><TD>1 µl
</TD><TD>T4-ligase buffer (10x)
</TD></TR></TBODY></TABLE><UL><LI>ligation ration 1:6
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_colonies_of_transformation_.28August_22nd.29">Picking of colonies of transformation (August 22nd)</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Get cells for miniprep for amplifying ligation products.
</P><P><B>Procedure:</B></P><UL><LI> Picking of 5 clones each of PCR2/p133 clone 20 (Miniprep 21.08.) and PCR58/P133 clone 4 (mini prep 08.08.)
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_colony_PCR_clones_of_21th_August"> Miniprep of colony PCR clones of 21th August</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Get plasmid DNA of the plasmids that were identified as insert-positive clones.
</P><P>Clones 1, 3, 5, 11 of colony PCR (21.08.) were miniprepped.
</P><UL><LI> Clone 1: 71.3 ng/µl
</LI><LI> Clone 3: 55.6 ng/µl
</LI><LI> Clone 5: 122.1 ng/µl
</LI><LI> Clone 11: 118.2 ng/µl
</LI></UL><P><B>Plasmid DNA is stored in a 50 ml Falcon tube at the lowest drawer of -20°C. (label: Miniprep 23.8. Andrea)</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_and_gel_electrophoresis_of_miniprep_of_colony_PCR_products_.28miniprep_23.08..29">Analytical restriction digest and gel electrophoresis of miniprep of colony PCR products (miniprep 23.08.)</SPAN></H3><P><B>Investigators:</B> Andrea
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>Xba 1
</TD></TR><TR><TD>0.5 µl
</TD><TD>Spe-HF 1
</TD></TR><TR><TD>2 µl
</TD><TD>NEB
</TD></TR><TR><TD>0,2 µl
</TD><TD>BSA
</TD></TR><TR><TD>12.2 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Gelelectrophoresis:</B></P><UL><LI>expected: 
</LI></UL><P>Limonensynthase 1600 bp
</P><P>pSB1C3 (P133)     2000 bp
</P><UL><LI>positive ligations: 
</LI></UL><P>clone 11 = PCR56/p133
</P><UL><LI>only 10 µl were used for gel electrophoresis (rest is stored in the same tube with appendant miniprep products)
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Transformation_of_pTUM104_Preprothaumatin_and_inoculation"> Transformation of pTUM104_Preprothaumatin and inoculation</SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim:</B> Getting pYes2_Preprothaumatin
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Small_Scale_Yeast_Transformation"> Small Scale Yeast Transformation </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim:</B> Preculture of <I>S. cerevisiae</I> according to pYes2_manuel_kommentiert
</P><P>1. Inoculate 4 ml of YPD medium with a colony of S. cerevisiae and shake overnight at 30°C.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Colony_PCR_of_PCR43.2Fp175_and_PCR57.2Fp133">Colony PCR of PCR43/p175 and PCR57/p133</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Forward primer of vector backbone and the reverse primer of the insert were used.
</P><UL><LI> extension time of PCR reaction was adjusted to the longer PCR product (insert+primers+backbone from primer binding site)
</LI><LI> annealing temperature was adjusted to lowest primer melting temperature
</LI></UL><UL><LI><B>Citrus</B>: 8 clones of PCR43/p175 (transformation of 17.8.) were picked.
</LI><LI><B>Lavendula</B>: 8 clones of PCR57/p133 (trafo 17.8.), 8 clones PCR57/p175 (transformation of August 17th) and 8 clones of PCR56/p175 (trafo 17.8.) were picked.
</LI><LI> positive control: plasmid DNA of PCR1/p175 (clone 20) as positive control
</LI><LI> negative controls: ddH2O
</LI></UL><P><B>Citrus in pYES</B> (PCR43/p175)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O8
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O30
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Lavendula in pSB1C3</B> (PCR57/p133)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O68
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O37
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Lavendula in pYES</B> (PCR56/p175 and PCR57/p175)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O8
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O37
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> PCR program:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>10 min
</TD></TR><TR><TD>30 cycles
</TD><TD>94 °C
</TD><TD>30 s
</TD></TR><TR><TD>50 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>2 min
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><P><B>Labels:</B></P><P>1-7: PCR43/p175
</P><P>8: Pos. control: PCR1/p175 (20)
</P><P>9-16: PCR57/p133
</P><P>17-24: PCR57/p175
</P><P>25-32: PCR56/p175
</P><P>33: Negative control: ddH2O
</P><P><B>Gelelectrophoresis still needs to be done</B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_aliquots_for_SC_Minimal_Medium"> Preparation of aliquots for SC Minimal Medium </SPAN></H3><P><B>Investigator:</B> Saskia, Roman
</P><P>Preparation of ten 50 ml aliquots of SC Minimal Medium (URA-) as described in the commented pYES manual on page 10. To solve the substances you need to mix and heat it.
For 1 l SC Minimal Medium: 
50 ml Aliquot + 6.7 g Yeast Nitrogen Base + 850 ml water.
Autoclave seperately 20 g glucose or galactose in 100 ml water. 
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_200_ml_flasks_with_SC_Minimal_Medium_.28URA-.29"> Preparation of 200 ml flasks with SC Minimal Medium (URA-)</SPAN></H3><P><B>Investigator:</B> Saskia, Volker
</P><P>Preparation of 200 ml flasks with SC Minimal Medium (URA-) and galactose for expression.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_Selective_SC_Minimal_Medium_Plates"> Preparation of Selective SC Minimal Medium Plates </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P>Preparation of selective SC Minimal Medium Plates without Uracil and without Uracil and Leucin as described in the pYES manual on page 10. 3 % Agar were used instead of 2 % (the last time the agar didn't jell with 2 % Agar).
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Western_Blot_of_APT_at_different_times_of_expression_in_yeast_-_continued">Western Blot of APT at different times of expression in yeast - continued</SPAN></H3><P><B>Investigator:</B> Daniela
</P><P><B>Aim of the experiment:</B> Find out optimum time for expression.
</P><P><B>Detection:</B></P><UL><LI>via Streptavidin-AP (1:4000) -&gt;10ml, Streptavidin-AP diluted in PBST0.1+3%BSA
</LI><LI>1h incubation, be sure that the whole membrane is covered with Streptavidin-AP solution.
</LI></UL><P><B>Developing:</B></P><UL><LI>15 ml AP-buffer mixed with 45 µl BCIP (50 mg/ml in DMF) and 7.5 µl NBT (75 mg/ml in 70 % DMF)
</LI><LI>incubation: about 30 min
</LI></UL><P>Expected bands:
</P><P>APT:  length: 411, mass =  45.481 kDa
</P><P>GFP: mass: 26.9 kDa -&gt;positive control
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Sequencing_of_plasmids_pSB1C3_Insert_and_pTUM104_Insert"> Sequencing of plasmids pSB1C3_Insert and pTUM104_Insert </SPAN></H3><P><B>Investigator:</B> Roman, Saskia
</P><P><B>Aim:</B> Proof of inserted sequence
</P><P>Sequenced plasmids:
</P><UL><LI> pSB1C3_CaXMT1 (clone B1B): sequence ok
</LI><LI> pSB1C3_CaMXMT1 (clone B2A): sequence ok
</LI><LI> pSB1C3_CaDXMT1 (clone B3D): sequence ok
</LI></UL><UL><LI> pTUM104CaXMT1 (clone Y1B): sequencing failed
</LI><LI> pTUM104_CaMXMT1 (clone Y2D): sequencing failed
</LI><LI> pTUM104_CaDXMT1 (clone Y3A): sequencing failed
</LI></UL><P>Note: Sequencing of pTUM104 plasmids will be repeated next week with newly picked clones. Sequencing results of all three pSB1C3 plasmids (containing inserts) were positive.
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_24th"><B>Friday, August 24th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_gelelectrophoresis_of_digested_ADH1-P_.28720_bp.29">Preparative gelelectrophoresis of digested ADH1-P (720 bp)</SPAN></H3><UL><LI> Digestion with PstI-HF and Spe-HF was succesful
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_P465_and_P_309">Analytical digestion of P465 and P 309</SPAN></H3><P><B>Investigator</B>: Georg
</P><P><B>Aim of experiment</B>: Since the sequence in P465 was from TEF1-P instead of TEF1-T you have to verify the insert
</P><P><B>Procedure</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,75 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>0,75 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>6 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,6 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>44,4 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=60µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical gel showed that everywhere was TEF1-P
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_ADH1-P_.28720_bp.29_P299.2CP300_.28TEF1-P.29.2C_P432_and_P433_.28TEF1-P_.2Bpsb1c3.29.2C_P434_and_P435_.28ADP-P_lang_.2B_psb1c3.29">Gelextraction of ADH1-P (720 bp) P299,P300 (TEF1-P), P432 and P433 (TEF1-P +psb1c3), P434 and P435 (ADP-P lang + psb1c3)</SPAN></H3><P><B>Investigator</B>:Georg
</P><UL><LI> Gelextraction was performed according to Quiaquick gel extraction protocol (salt removed)
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_for_concentration_measuring_of_ADH1-P.2C_P434.2C_P299.2C_P432"> Nanodrop for concentration measuring of ADH1-P, P434, P299, P432</SPAN></H3><P><B>Investigator</B>:Georg
</P><UL><LI>ADH1-P (720 bp)= 22,1 ng/µl
</LI><LI>P434 = 64,1 ng/ml
</LI><LI>P299 = 12,6 ng/µl
</LI><LI>P432 = 49,8 ng/µl
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_of_colonies_of_pTUM104.2C_GFP_and_ADH1-P_.28720_bp.29">Picking of colonies of pTUM104, GFP and ADH1-P (720 bp)</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI> 10 colonies from pYES2-TUM transformation with 50 ng/µl (1h), 4 Colonies GFP and 2 Colonies from ADH1-P were Picked 
</LI><LI> Colonies from pYES2 and GFP were transferred to 5 ml LB-Medium with 1x Ampicillin, ADH1-P colonies were transferred to LB-Medium with 1x Chloramphenicol
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophoresis_of_P470_and_PCR71"> Preperative digestion and gelelectrophoresis of P470 and PCR71 </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI> Reaction batch for preperative digestion of P470 with NgoMIV+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid P470
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for preperative digestion of PCR71 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>25 µl
</TD><TD>PCR71
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>17.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> These reaction batches were incubated for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4.44 µl of DNA loading buffer (10x) was added to the digestion of PCR71 and 5.55 µl of DNA loading buffer (10x) was added to the digestion of P470.
</LI></UL><UL><LI> Preperative gelelectrophoresis was performed at 70 V for 90 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P470 (NgoMIV+PstI-HF)
</TD><TD>PCR71 (XbaI+PstI-HF)
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Digested Insert (upper band!) was cut out
</TD><TD>Digested PCR product was cut out
</TD></TR></TBODY></TABLE><UL><LI> Insert very difficult to cut out because insert and backbone are near and has a very broad width.
</LI></UL><UL><LI> Analytical gelelectrophoresis should be performed to verify the insert length and the purity of the insert by checking if there is a pSB1C3 contamination at about 2.1 kbp.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_P436.2BP480"> Transformation of <I>E. coli</I> XL1-Blue with ligation product P436+P480 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product P436+P480 for the (pre-)finale contruction of N'-PhyB(908NT)-20aaLinker-Gal4DBD-C'.
</P><P><B>Procedure:</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation product (P436+480) and it's negative control (P436 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on new chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_P491.2C_PCR53.2C_PCR62.2C_P492.2C_P493"> Analytical gelelectrophoresis of P491, PCR53, PCR62, P492, P493 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical gelelectrophoresis was done for P491 because the length of the part was not exactly determinable in the preperative gelelectrophoresis because the width of the DNA band was very strong. PCR53, PCR62, P492, P493 was performed for Georg because there was place for other samples.
</P><P><B>Procedure:</B></P><UL><LI> 5 µl of the samples were mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1.5 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P491
</TD><TD>PCR53
</TD><TD>PCR62
</TD><TD>P492
</TD><TD>P493
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Length is okay, but contaminated with small amount of pSB1C3 backbone
</TD><TD>Please see subsection for constitutive promoters
</TD><TD>Please see subsection for constitutive promoters
</TD><TD>Please see subsection for constitutive promoters
</TD><TD>Please see subsection for constitutive promoters
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P399.2BP491_and_P439.2BPCR72"> Cycled ligation of P399+P491 and P439+PCR72 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Etablishing a new method for ligation to combine high ligase activity and optimal annealing temperature of the sticky ends. Cycled ligation was performed for P399+P491 and P439+PCR72 and their negative controls P399 NK and P439 NK
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P399+P491
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.97 µl
</TD><TD>P399
</TD></TR><TR><TD>7.08 µl
</TD><TD>P491
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>7.95 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P499+P491
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.24 µl
</TD><TD>P439
</TD></TR><TR><TD>5.19 µl
</TD><TD>PCR72
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>9.57 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Minipreparation_of_pSB1C3_Preprothaumatin_and_pTUM104_Preprothaumatin"> Minipreparation of pSB1C3_Preprothaumatin and pTUM104_Preprothaumatin </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><UL><LI> pYes2_Preprothaumatin: p481-p484
</LI></UL><UL><LI> pSB1C3_Preprothaumatin: p485-490.
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Control_digest_of_pTUM104_Preprothaumatin"> Control digest of pTUM104_Preprothaumatin </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim:</B> Is the insert (preprothaumatin) integrated into pYes2?
</P><P>Analytical digest:
</P><UL><LI> 12 µl (p481-p485), 2 µl NEBuffer 4, 0.2 µl BSA, 0.25 µl XbaI, 0.25 µl SpeI, 5.3 µl ddH2O; 37°C, 1h.
</LI></UL><P>Analytical gel electrophoresis: 1 kb marker, p481, p482, p483. p484, p485
</P><UL><LI> Expected: backbone: 5.8 kbp, insert: 721 bp. 
</LI></UL><P><B>Result:</B> All clones are positive, preprothaumtain is integrated in pYes2.
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Small_scale_yeast_transformation_of_S._cerevisiae_with_pTUM104_Preprothaumatin"> Small scale yeast transformation of <I>S. cerevisiae</I> with pTUM104_Preprothaumatin </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P>Procedure according to pYes2_manual_kommentiert:
</P><UL><LI> 2. Determine the OD600 of your overnight culture. Dilute culture to an OD600 of 0.4 in 50 ml of YPD medium, add 5ml 20% glucose and grow an additional 2–4 hours. 3. Pellet (5 min, 4°C) the cells at 2500 rpm and resuspend the pellet in 40 ml 1X TE. 4. Pellet (5 min, 4°C) the cells at 2500 rpm and resuspend the pellet in 2 ml of 1X LiAc/0.5X TE. 5. Incubate the cells at room temperature for 10 minutes. 6. For each transformation, mix together 1 μg plasmid DNA and 100 μg (10 μl) denatured sheared salmon sperm DNA with 100 μl of the yeast suspension from Step 5. 7. Add 700 μl of 1X LiAc/40% PEG-3350/1X TE (7 ml 50% PEG3350 was synthesized (3.5 g in 7 ml bidest.water); 6.4 ml 50% PEG3350 + 800 μl 10x LiAc + 800 μl 10x TE were mixed) and mix well. 8. Incubate solution at 30°C for 30 minutes. 9. Add 88 μl DMSO, mix well, and heat shock at 42°C for 7 minutes. 10. Centrifuge in a microcentrifuge for 10 seconds and remove supernatant. 11. Resuspend the cell pellet in 1 ml 1X TE and re-pellet. 12. Resuspend the cell pellet in 50–100 μl 1X TE and plate on a selective plate (SCU-plates + Glucose). Incubate the SCU-plates at 30°C over the weekend.
</LI></UL><P><B>Name:</B> pYes2_Preprothaumatin, S. cerevisiae, Schappert, Bräuer. 
</P><P><B>Results:</B> Didn't work. Have to redo it.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_positive_clones">Miniprep of positive clones</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Get more plasmid DNA of the positive clones.
</P><UL><LI> PCR58/p133 (tube 1): 440 ng/µl
</LI><LI> PCR58/p133 (tube 2): 224 ng/µl
</LI><LI> PCR58/p133 (tube 3): 356 ng/µl
</LI><LI> PCR2/p133 (tube 1): 224 ng/µl
</LI><LI> PCR2/p133 (tube 2): 282 ng/µl
</LI><LI> PCR2/p133 (tube 3): 227 ng/µl
</LI></UL><P><B>Eppis are stored in a disposal bag (label: Positive Klone, Lara, 24.8.) in the lowest drawer of -20°C.</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_transformed_positive_clones">Analytical restriction digest of transformed positive clones</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Check that transformed clones are OK.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xba 1
</TD></TR><TR><TD>0.25 µl
</TD><TD>Spe-HF 1
</TD></TR><TR><TD>2 µl
</TD><TD>NEB
</TD></TR><TR><TD>0,2 µl
</TD><TD>BSA
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Gelelectrophoresis:</B></P><P>Gel: 
</P><P>1. Gene Ruler 1kb
</P><P>2. PCR58/p133 (1)
</P><P>3. PCR58/p133 (2)
</P><P>4. PCR58/p133 (3)
</P><P>5. PCR2/p133 (1)
</P><P>6. PCR2/p133 (2)
</P><P>7. PCR2/p133 (3)
</P><UL><LI>expected: 
</LI></UL><P>Limonene synthase 1600 bp
</P><P>pSB1C3 (P133)     2000 bp
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_with_insert-positive_plasmids_and_ligation_products_.28ligation_23.8..29">Transformation with insert-positive plasmids and ligation products (ligation 23.8.)</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Transform E.coli XL 1 blue with plasmids that were identified as being positive for insert. Furthermore transformation with ligation products of ligation 23.8.
</P><P>Transformations:
</P><P>1. PCR1/pYES clone 2 (no. 29, disposal bag, middle drawer of -20°C)
</P><P>2. PCR56/p133 (miniprep 23.8., clone 11, Falcon, lowest drawer of -20°C)
</P><P>3. Ligation product PCR2/pYES (ligation 23.8.)
</P><P>4. Ligation product PCR58/pYES (ligation 23.8.)
</P><P>5. Ligation product negative control (ligation 23.8.)
</P><P><B>Transformation into E.coli Xl1-Blue</B></P><UL><LI> thawing of 100 µl Ca-competent E.coli XL1-Blue cells on ice
</LI><LI> adding of 1 µl of DNA (positive plasmids) or of 5 µl of ligation products
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> adding of 1 ml LB-medium without antibiotics to the cells and incubation at 37°C and 180 rpm for 45 min
</LI><LI> plating of 100 µl of the cells on an Chloramphenicol-plate (pSB1C3, P133), Ampicillin-plate (pYES)
</LI><LI> sedimentation of the leftover in a centrifuge (2 min, 13 000 rpm), resuspension of the sediment in 100 µl LB-medium and plating it as well on an antibiotic containing LB-plate 
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_EcoRI.28at_150_bp.29_in_the_4CL">  Quick Change mutagenesis to remove EcoRI(at 150 bp) in the 4CL  </SPAN></H3><P><B>Investigator: Saskia, Daniela</B></P><P>Aim of the experiment: Generation of an <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible version of the 4CL gene.
</P><P>Samples:
</P><UL><LI>4CL- in pYES clone 2
</LI><LI>4CL+ in pYES clone 2
</LI><LI>4CL- in pSB1C3 clone 1
</LI><LI>4CL+ in pSB1C3 clone 2
</LI></UL><P><B>PCR</B><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O77 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O78 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> addition of 1 ml LB-medium without antibiotics to the cells and incubation at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate (pYES) or an Chlp-LB-plate (pSB1C3)
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate (pYES) or an Chlp-LB-plate (pSB1C3)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_August_25th"><B>Saturday, August 25th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_P399.2B491_and_P439.2BPCR72"> Transformation of <I>E. coli</I> XL1-Blue with ligation product P399+491 and P439+PCR72 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P399+491 and P439+PCR72 for the constructs of N'-SV40NLS-PhyB(908NT)-20aaLinker-LexA-C' and N'-SV40-Gal4AD-Linker-Pif3-C'.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation products (P399+491 and P439+PCR72) and it's negative control (P499 NK and P439 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and these concentrated cell suspensions was plated again on new chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_of_P436.2B480"> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P436+480 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation product of P436+480 (P495).
</P><P><B>Procedure:</B></P><UL><LI> 6 colonies of P436+480 (P495) (CamR) transformated cells were picked.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chloramphenicol. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Miniprep_pTUM100.2C_GFP_and_ADH1-P_.28720_bp.29"> Miniprep pTUM100, GFP and ADH1-P (720 bp) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of pTUM100, GFP and ADH1-P (720 bp)
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAGEN - QIAprep Spin Miniprep Kit).
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Picking_of_E.coli_XL1-Blue_transformed_with_4CL_after_Quick_Change_from_August_24th"> Picking of <I>E.coli</I> XL1-Blue transformed with 4CL after Quick Change from August 24th</SPAN></H3><P><B>Investigator</B>: Daniela
</P><P><B>Aim of the experiment:</B> Picking of <I>E.coli</I> XL1-Blue transformed with 4CL after Quick Change from August 24th.
</P><P><B>Procedure:</B></P><P>4 colonies were taken of each transformation.
</P><UL><LI>4CL- in pYES clone 2
</LI><LI>4CL+ in pYES clone 2
</LI><LI>4CL- in pSB1C3 clone 1
</LI><LI>4CL+ in pSB1C3 clone 2
</LI></UL><UL><LI>Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4-5 ml of LB-medium and antibiotics (Amp -&gt; pYES, Chlp -&gt; pSB1C3. These tubes were put overnight in a 180rpm cell culture shaker at 37°C. 
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_August_26th"><B>Sunday, August 26th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_E._coli_XL-Blue_cells.2C_transformated_with_ligation_products_of_P436.2B480"> Miniprep of E. coli XL-Blue cells, transformated with ligation products of P436+480   </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI> Miniprep was performed after manufacturer's protocol (QIAGEN - QIAprep Spin Miniprep Kit). Tubes were labeled with P501-P506.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P501-P506"> Analytical digestion and gelelectrophoresis of P501-P506 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P501-P506 with NgoMIV+PstI-HF.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with NgoMIV+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>14 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1.75 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1.75 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>105 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=122.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of samples P501-P506 in a new ERG and was mixed gently by pipetting up and down.
</LI></UL><UL><LI> 90 min incubation at 37 °C.
</LI></UL><UL><LI> 9 µl of the samples were mixed with 1 µl of DNA loading buffer (10x). 
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1.5 h. 
</LI></UL><UL><LI> All picked clones are positive! Finished construct: N'-Phyb(908NT)-20aaLinker-Gal4DBD-C'. Possible last fusion to SV40NLS, if N-terminal NLS of Gal4DBD is strongly concealed by N-terminal fusion by PhyB-20aaLinker.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_of_P399.2B491_and_P439.2BPCR72"> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P399+491 and P439+PCR72  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P399+491 and P439+PCR72.
</P><P><B>Procedure:</B></P><UL><LI> 6 colonies of the transformation with P399+491 (CamR) and 10 colonies of the transformation with P439+PCR72 (CamR) were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chloramphenicol. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_Y1B.2C_Y2D_and_Y3A"> Miniprep of Y1B, Y2D and Y3A </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim:</B></P><P>Preparation of pTUM104 Y1B, Y2D and Y3A for sequencing.
</P><P><B>Plasmidconcentration:</B> Nanodroplet
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>pTUM104_caffeine_involved_gene</B></TD><TD><B>Concentration in ng/µl</B></TD></TR><TR><TD>Y1B
</TD><TD>66.4
</TD></TR><TR><TD>Y2D
</TD><TD>47.6
</TD></TR><TR><TD>Y3A
</TD><TD>49.7
</TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_colonies_of_PCR2_pTUM104_and_PCR58_pTUM104_and_of_PCR1_pTUM104_and_PCR56_pSBC3">Picking of colonies of PCR2 pTUM104 and PCR58 pTUM104 and of PCR1 pTUM104 and PCR56 pSBC3 </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Procedure:</B></P><UL><LI> Picking of 5 clones each of PCR2 pTUM104 and PCR58 pTUM104 and picking of 2 clone each of PCR1 pTUM104 and PCR56 pSBC3
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_and_analytical_gel_electrophoresis">Analytical restriction digest and analytical gel electrophoresis</SPAN></H3><P><B>Investigator:</B> Saskia and Katrin
</P><P><B>Aim of the experiment:</B> Controll of success of the Quickchange mutagenesis from 24.08.2012 to remove EcoRI from 4Cl
</P><P><B>Analytical digest:</B> of Cl- pYES, Cl+ PYES, Cl- pSB1C3 and Cl+ pSB1C3
</P><P><B>Procedure:</B></P><UL><LI> Digestion reaction batch
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid-DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI HF
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI HF
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>Digestion over night at 37 °C. 
</P><P><B>Analytical DNA-gelelectrophoresis:</B></P><P><B>Procedure:</B>
Analytical gelelectrophoresis was performed at 90 V for 1 h.
</P><P><B>Gel 1:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kp ladder DNA ladder
</TD><TD><B>Cl+ pYES1</B></TD><TD><B>Cl- pSB1C3</B></TD><TD><B>Cl+ pYES3</B></TD><TD><B>Cl+ pYES4</B></TD><TD><B>Cl- pYES1</B></TD><TD><B>Cl- pYES2</B></TD><TD><B>Cl- pYES3</B></TD><TD><B>Cl- pYES4</B></TD><TD>100 bp ladder DNA ladder
</TD></TR></TBODY></TABLE><P><B>Gel 2:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kp ladder DNA ladder
</TD><TD><B>Cl+ pSB1C31</B></TD><TD><B>Cl+ pSB1C3</B></TD><TD><B>Cl+ pSB1C33</B></TD><TD><B>Cl+ pSB1C34</B></TD><TD><B>Cl- pSB1C31</B></TD><TD><B>Cl+ pYES</B></TD><TD><B>Cl- pSB1C33</B></TD><TD><B>Cl- pSB1C34</B></TD><TD>100 bp ladder DNA ladder
</TD></TR></TBODY></TABLE></DIV></DIV><H1><SPAN class="mw-headline" id="Week_12">Week 12</SPAN></H1><DIV class="week" id="WWeek_12"><H2><SPAN class="mw-headline" id="Monday.2C_August_27th"><B>Monday, August 27th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_of_P399.2BP491_and_P439.2BPCR72"> Miniprep of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P399+P491 and P439+PCR72 </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P399+P491 and P439+PCR72.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done after manufacturer's protocol. (QIAGEN - QIAprep Spin Miniprep Kit)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_P524-P539"> Analytical digestion and gelelectrophoresis of the minipreps P524-P539 </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps P524-P539 with NgoMIV+PstI-HF.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion for P507-P522 with NgoMIV+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>34 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>4.25 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>4.25 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>255 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=297.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of plasmid DNA (P507-P522).
</LI></UL><UL><LI> The reaction mixes were incubated at 37 °C for 90 min.
</LI></UL><UL><LI> 9 µl of the digested DNA was mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for about 60 min.
</LI></UL><P>P524-P533:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P524
</TD><TD>P525
</TD><TD>P526
</TD><TD>P527
</TD><TD>P528
</TD><TD>P529
</TD><TD>P530
</TD><TD>P531
</TD><TD>P532
</TD><TD>P533
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD><TD>Ligation failed
</TD></TR></TBODY></TABLE><P>P534-P539:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P533
</TD><TD>P534
</TD><TD>P535
</TD><TD>P536
</TD><TD>P537
</TD><TD>P538
</TD><TD>P539
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Ligation maybe succesful, SEQUENCING has to be done!!
</TD><TD>Ligation maybe succesful, SEQUENCING has to be done!!
</TD><TD>Ligation maybe succesful, SEQUENCING has to be done!!
</TD><TD>Ligation maybe succesful, SEQUENCING has to be done!!
</TD><TD>Ligation maybe succesful, SEQUENCING has to be done!!
</TD><TD>Ligation maybe succesful, SEQUENCING has to be done!!
</TD><TD>Ligation maybe succesful, SEQUENCING has to be done!!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophoresis_of_P506"> Preperative digestion and gelelectrophoresis of P506 </SPAN></H3><P><B>Investigator:</B> Jeff, Dennis
</P><P><B>Aim of the experiment:</B> Preperative digestion and gelelectrophoresis of P506 with NgoMIV+PstI-HF.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for P506 with NgoMIV+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P506
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2 µl
</TD><TD>NgoMIV (10 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Preperative digestion was carried out at 37 °C for 2 h.
</LI></UL><UL><LI> Preperative gelelectrophoresis was performed at 70 V for 2 h.
</LI></UL><UL><LI> Upper band (Insert) was cut out and DNA was extracted with gel extraction kit from Qiagen after manufacturer's protocol.
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Small_Scale_Yeast_Transformation_-_Preparatory_culture"> Small Scale Yeast Transformation - Preparatory culture </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim:</B> Preculture of <I>S. cerevisiae</I> according to pYes2_manuel_kommentiert.
</P><P>1. Inoculate 4 ml of YPD medium with a colony of S. cerevisiae and shake well overnight at <B>30 °C</B>.
</P><P><B>Results:</B> 30 °C and not 37 °C. Redo!
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_positive_clones_2">Miniprep of positive clones</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Get more plasmid DNA of the positive clones (PCR1/pYES &amp; PCR56/p133). Check ligation products (PCR58/pYES, PCR2/pYES). Check colony PCR clones.
</P><UL><LI> PCR58/pYES (tube 1):  ng/µl
</LI><LI> PCR58/pYES (tube 2): 83 ng/µl
</LI><LI> PCR58/pYES (tube 3): 92 ng/µl
</LI><LI> PCR58/pYES (tube 4): 136 ng/µl
</LI><LI> PCR58/pYES (tube 5): 110 ng/µl
</LI></UL><UL><LI> PCR2/pYES (tube 1): 212 ng/µl
</LI><LI> PCR2/pYES (tube 2): 170 ng/µl
</LI><LI> PCR2/pYES (tube 3): 173 ng/µl
</LI><LI> PCR2/pYES (tube 4): 162 ng/µl
</LI><LI> PCR2/pYES (tube 5): 132 ng/µl
</LI></UL><UL><LI> PCR1/pYES (tube 1): 112 ng/µl
</LI><LI> PCR1/pYES (tube 2): 104 ng/µl
</LI></UL><UL><LI> PCR56/p133 (tube 1): 207 ng/µl
</LI><LI> PCR56/p133 (tube 2): 80 ng/µl
</LI></UL><UL><LI>ColPCR Clone2: 121 ng/µl
</LI><LI>ColPCR Clone9: 82 ng/µl
</LI><LI>ColPCR Clone16: 81 ng/µl
</LI><LI>ColPCR Clone26: 96 ng/µl
</LI><LI>ColPCR Clone28: 280 ng/µl
</LI><LI>ColPCR Clone32: 146 ng/µl
</LI></UL><P><B>Eppis are stored in a disposal bag (label: Positive Klone, Lara, 27.8.) in the lowest drawer of -20°C.</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest">Analytical restriction digest</SPAN></H3><P><B>Investigator:</B> Lara (digest), Andrea (gel)
</P><P><B>Aim:</B> Check clones for insert.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xba 1
</TD></TR><TR><TD>0.25 µl
</TD><TD>Spe-HF 1
</TD></TR><TR><TD>2 µl
</TD><TD>NEB
</TD></TR><TR><TD>0,2 µl
</TD><TD>BSA
</TD></TR><TR><TD>14.3 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Gelelectrophoresis:</B></P><P>Gel: 
</P><UL><LI>expected: 
</LI></UL><P>Limonene synthase 1600 bp
</P><P>pYES2 5800 bp
</P><P>pSB1C3 (P133)     2000 bp
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_cells_for_protein_expression_in_yeast_2">Preparation of cells for protein expression in yeast </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B>Picking transformed Saccharomyces cerevisiae cells for protein expression in yeast
</P><UL><LI> Inoculation of overnight cultures: one clone from plate with PCR 1 in P175 clone 2 (Miniprep 21.08.) (29) was picked,resuspended in 15 ml SCU medium with glucose and incubated at 30 °C overnight
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Picking_of_colonies_for_miniprep"> Picking of colonies for miniprep </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim:</B> New miniprep for new transformation of S. cerevisiae with Y1, Y2 and Y3 and new sequencing (the last one wasn't successful)
</P><UL><LI> Picking of 3 clones each of Y1, Y2 and Y3. There are named Y1E, Y1F, Y1G and Y2E, Y2F, Y2G and Y3E, Y3F, Y3G
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Picking_of_colonies_for_yeast_transformation">Picking of colonies for yeast transformation </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim:</B> yeast transformation for expression of caffeine-genes 
</P><P><B>Procedure:</B></P><UL><LI> Picking of 1 clone of S. cerevisiae INVScN1 in 4 ml YPD</LI><LI> Inbucation at 30°C (shaker) over night.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_August_28th"><B>Tuesday, August 28th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_pTUM100_undigested">Analytical gelelectrophoresis of pTUM100 undigested</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Plasmid-DNA of undigested pYES-TUM colonies was applicated on analytical 0,5% Agarosegels
</LI><LI> pYES2 served as control plasmid
</LI><LI> colonies 1-4 and 6-10 showed positive pYES-TUM2 variants
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_pTUM100"> Analytical digestion of pTUM100</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,75 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>22 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>167,75 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=192,5µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> to 2,5 ml of DNA 17,5 µl reaction batch were added
</LI><LI> Digestion took place at 37°C for 1 h
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_pTUM100"> Analytical gelelectrophoresis of pTUM100</SPAN></H3><P><B>Investigator:</B></P><UL><LI> Digested pYES2-Tum clones were analyzed on 0,5% Agarose gels
</LI><LI> Clones 1-4 and 6-10 were positive
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_for_concentration_measuring_of_ADH-P.2CpTUM100.2CGFP">Nanodrop for concentration measuring of ADH-P,pTUM100,GFP</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> ADH-P Kol1: 112,5 ng/µl
</LI><LI> ADH-P Kol2: 105,4 ng/µl
</LI><LI> pYES2-TUM Kol1: 155,7 ng/µl
</LI><LI> pYES2-TUM Kol2: 67,3 ng/µl
</LI><LI> pYES2-TUM Kol3: 124,5 ng/µl
</LI><LI> pYES2-TUM Kol4: 91,5 ng/µl
</LI><LI> pYES2-TUM Kol5: 91,7 ng/µl
</LI><LI> pYES2-TUM Kol6: 122,8 ng/µl
</LI><LI> pYES2-TUM Kol7: 105,9 ng/µl
</LI><LI> pYES2-TUM Kol8: 123,9 ng/µl
</LI><LI> pYES2-TUM Kol9: 129,8 ng/µl
</LI><LI> pYES2-TUM Kol10:136,9 ng/µl
</LI><LI> GFP Kol1: 152,1 ng/µl
</LI><LI> GFP Kol2: 107,5 ng/µl
</LI><LI> GFP Kol3: 53,9 ng/µl
</LI><LI> GFP Kol4: 146,9 ng/µl
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P399.2BP552"> Cycled ligation of P399+P552 </SPAN></H3><P><B>Investigator:</B> Dennis, Jeff
</P><P><B>Aim of the experiment:</B> Cycled ligation was performed for P399+P552 and it's negative control P399 NK.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P399+P552
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.97 µl
</TD><TD>P399
</TD></TR><TR><TD>12 µl
</TD><TD>P491
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>3.03 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_PCR57_in_pTUM104_and_pSB1C3.2C_of_PCR_42_in_pSB1C3.2C_of_PCR_56_in_pTUM104">Ligation of PCR57 in pTUM104 and pSB1C3, of PCR 42 in pSB1C3, of PCR 56 in pTUM104</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Get successful ligated clones of PCR57 in pYES and pSB1C3, of PCR 42 in pSB1C3, of PCR 56 in pYES
</P><P><B>Procedure</B></P><P><B>PCR 42 in pSB1C3 (P133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>6.7 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 42
</TD><TD>6.3 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=15 µl
</TD></TR></TBODY></TABLE><P><B>PCR 56 in pYES (P375 verdaut)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P375 verdaut
</TD><TD>9.3 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 56
</TD><TD>3.7 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=15 µl
</TD></TR></TBODY></TABLE><P><B>PCR 57 in pYES (P375 verdaut)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P375 verdaut
</TD><TD>9.2 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 57
</TD><TD>3.8 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=15 µl
</TD></TR></TBODY></TABLE><P><B>PCR 57 in pSB1C3 (P133)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P133
</TD><TD>1.6 µl (~50 ng vector dna)
</TD></TR><TR><TD>PCR 57
</TD><TD>11.45 µl (~3x of n(vector))
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 </TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=15 µl
</TD></TR></TBODY></TABLE><UL><LI>incubation at RT for 1 hour
</LI><LI>afterwards transformation
</LI><LI>rest of ligation incubated over night at 16 °C
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_into_E.coli_.28ligation_28.8..29">Transformation into <I>E.coli</I> (ligation 28.8.)</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Transform E.coli XL 1 blue with ligation products of ligation 28.8.
</P><P>Transformations:
</P><P>1. PCR57 (digested 7th August) in pYES (P375, digested 23th August)
</P><P>2. PCR57 (digested 7th August) in pSB1C3 (P133)
</P><P>3. PCR 42 (digested 7th August) in pSB1C3 (P133)
</P><P>4. PCR 56 (digested 7th August) in pYES (P375, digested 23th August)
</P><P><B>Transformation into E.coli Xl1-Blue</B></P><UL><LI> thawing of 100 µl Ca-competent E.coli XL1-Blue cells on ice
</LI><LI> adding of 1 µl of DNA (positive plasmids) or of 5 µl of ligation products
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> adding of 1 ml LB-medium without antibiotics to the cells and incubation at 37°C and 180 rpm for 45 min
</LI><LI> plating of 100 µl of the cells on an Chloramphenicol-plate (pSB1C3, P133), Ampicillin-plate (pYES, P375) 
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Forward primer of vector backbone and the reverse primer of the insert were used.
</P><UL><LI> extension time of PCR reaction was adjusted to the longer PCR product (insert+primers+backbone from primer binding site)
</LI><LI> annealing temperature was adjusted to lowest primer melting temperature
</LI></UL><UL><LI><B>Citrus</B>: 8 clones of PCR42/p133 (4 clones transformation of 8.8., 4 clones of 27.7.) were picked.
</LI><LI><B>Lavendula</B>: 8 clones of PCR57/p133 (trafo 8.8.), 8 clones PCR57/p175 (transformation of August 17th) and 8 clones of PCR56/p175 (trafo 8.8.) were picked.
</LI><LI> positive control: clones PCR2/p133, PCR58/p133, PCR58/pYES
</LI><LI> negative controls: ddH2O
</LI></UL><P><B>Citrus in pYES</B> (PCR43/p133)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O68
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O30
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Lavendula in pSB1C3</B> (PCR57/p133)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O68
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O37
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Lavendula in pYES</B> (PCR56/p175 and PCR57/p175)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>5x OneTaq Standard Reaction Buffer
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 mM dNTPs
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O8
</TD></TR><TR><TD>0.5 µl
</TD><TD>10 µM O37
</TD></TR><TR><TD>0.13 µl
</TD><TD>OneTaq Hot Start DNA Polymerase
</TD></TR><TR><TD>18.37 µl
</TD><TD>dd water
</TD></TR><TR><TD>25 µL
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> PCR program:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>95 °C
</TD><TD>10 min
</TD></TR><TR><TD>30 cycles
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>50 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>2 min
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><P><B>Labels:</B></P><P>1-8: PCR42/pSB1C3 (1-4: 8.8.; 5-8: 27.7.)
</P><P>9-16: PCR57/pSB1C3
</P><P>17-24: PCR57/p175
</P><P>25-32: PCR56/p175
</P><P>33: Negative control: ddH2O
34: PCR2/p133
35: PCR58/p133
36: PCR58/pYES
</P><P><B>Gelelectrophoresis still needs to be done</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_cells_for_protein_expression_in_yeast_3">Preparation of cells for protein expression in yeast </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B>Transfer of a certain amount of transformed Saccharomyces cerevisiae cells from yesterday to new medium for protein expression in yeast
</P><UL><LI> Resuspended Saccharomyces cerevisiae cells in 15 ml SCU medium with glucose growed successful
</LI><LI> Measurement: OD600 = 1.8 (used OD600 = 0.4)
</LI><LI> 10 ml of resuspended S. cerevisiae were added to 40 ml SCU medium with glucose for reaching OD600 = 0.4
</LI><LI> Incubation at 30 °C overnight
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Plasmid_isolation_of_pTUM104_Insert_containing_over_night_cultures"> Plasmid isolation of pTUM104_Insert containing over night cultures </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> 
Isolation of plasmids pTUM104_CaXMT1, pTUM104_CaMXMT1 and pTUM104_CaDXMT1 to be able to repeat yeast transformation
</P><P><B>Operational sequence:</B>
Plasmid isolation was performed with the Quiagen Miniprep Kit as described in the manufacturers protocoll. Three over night cultures of each plasmid had been prepared on the day before. They had been established of three colonies of each plate, named Y1E-G, Y2E-G and Y3E-G.
Elution was made with 40µl elution buffer, heated up to 37°C and 1 minute incubation before the final centrifugation step.
</P><P><B>Concentrations:</B>
Concentration was measured with NanoDrop:
</P><UL><LI> Y1E: 230,7 ng/µl
</LI><LI> Y1F: 164,5 ng/µl
</LI><LI> Y1G: 290,6 ng/µl
</LI></UL><UL><LI> Y2E: 181,2 ng/µl
</LI><LI> Y2F: 91,8 ng/µl
</LI><LI> Y2G: 191,0 ng/µl
</LI></UL><UL><LI> Y3E: 98,9 ng/µl
</LI><LI> Y3F: 181,5 ng/µl
</LI><LI> Y3G: 157,1 ng/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Control_digest_of_isolated_pTUM104_Insert_plasmids_with_Xba1_and_Pst1-HF"> Control digest of isolated pTUM104_Insert plasmids with Xba1 and Pst1-HF </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> 
Test of isolated plasmids, wether they are containing the right inserts (CaXMT1, CaMXMT1 and CaDXMT1).
</P><P><B>Operational sequence:</B></P><P>Reaction batch:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>reagent</B></TD><TD><B>volume</B></TD></TR><TR><TD>Plasmid template
</TD><TD>5µl
</TD></TR><TR><TD>Buffer NEB 4
</TD><TD>2µl
</TD></TR><TR><TD>BSA 100x
</TD><TD>0,2µl
</TD></TR><TR><TD>RE Xba1
</TD><TD>0,25µl
</TD></TR><TR><TD>RE Pst1-HF
</TD><TD>0,25µl
</TD></TR><TR><TD>ddH2O
</TD><TD>12,3
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>20µl
</TD></TR></TBODY></TABLE><P>The reaction batch was incubated at 37°C for about 1,5 h, beeing followed by an analytical gel electrophoresis.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_digested_plasmids"> Analytical gelelectrophoresis of digested plasmids </SPAN></H3><P><B>Investigator: Roman</B></P><P><B>Aim of the experiment:</B>
Proof of the expected inserts
</P><P><B>Operational sequence:</B>
The whole amount of the reaction batch (restriction digest, 20µl, see above) was load on gel after having mixed the samples with 2 µl of 10x loading dye.
</P><P>Expected lengths of the inserts: ca. 1200bp (all three genes have almost the same length).
</P><P><B>From left to right:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>DNA Ladder
</TD><TD>Y1E Xba1/Pst1
</TD><TD>Y1F Xba1/Pst1
</TD><TD>Y1G Xba1/Pst1
</TD><TD>Y2E Xba1/Pst1
</TD><TD>Y2F Xba1/Pst1
</TD><TD>Y2G Xba1/Pst1
</TD><TD>Y3E Xba1/Pst1
</TD><TD>Y3F Xba1/Pst1
</TD><TD>Y3G Xba1/Pst1
</TD><TD>DNA Ladder
</TD></TR></TBODY></TABLE><P><B>Result:</B>
All the newly picked colonies show the right, expected insert. Clones Y1G, Y2G and Y3G will be used for the repetition of the yeast transformation.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Repetition_of_yeast_transformation"> Repetition of yeast transformation </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B>
Transformation of yeast cells (INVSCN1) with the plasmids pTUM104_CaXMT1, pTUM104_CaMXMT1 and pTUM104_CaDXMT1, as well as pTUM104_eGFP (positive controll).
</P><P><B>Operational sequence:</B></P><P>The yeast transformation was performed as described in the pYES2 manual of Invitrogen (see small scale yeast transformation), using an over night culture of INVSC1- yeast cells. The solutions 1xTE and 1xLiAc/40% PEG 3350/1xTE were made immediately before use. 
</P><UL><LI> 1xTE: 
</LI></UL><P>5µl of 10x TE were mixed with 45µl ddH2O and sterile filtrated.
</P><UL><LI> 1xLiAc/40% PEG 3350/1xTE
</LI></UL><P>At first, 3,5g PEG 3350 were dissolved in 7ml of bidestillated water, resulting in a 50% PEG solution. 6,4ml of this 50% PEG solution were mixed with 800µl 10xTE and 800µl 10x LiAc and sterile filtrated.
</P><P>Used amount of vector DNA:
</P><UL><LI> Y1G (p554): 4µl (ca. 1,1µg)
</LI><LI> Y2G (p555): 6µl (ca. 1,1µg)
</LI><LI> Y3G (p556): 6,5µl (ca. 1µg)
</LI><LI> eGFP: 5 µl (ca. 1µg)
</LI></UL><P>Negative controll: water used
</P><P>At last, the cells were plated on appropriate agar- plates with SC -Ura medium and incubated at 30°C for the next three days. Colonies should be visible on friday.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Yeast_transformation_with_p542_and_p549"> Yeast transformation with p542 and p549 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B>
Transformation of yeast cells (INVSC1) with the plasmids p542 and p549
</P><P><B>Operational sequence:</B></P><P>The yeast transformation was performed as described in the pYES2 manual of Invitrogen (see small scale yeast transformation), using an over night culture of INVSC1- yeast cells. The solutions 1xTE and 1xLiAc/40% PEG 3350/1xTE were made immediately before use. 
</P><UL><LI> 1xTE: 
</LI></UL><P>5µl of 10x TE were mixed with 45µl ddH2O and sterile filtrated.
</P><UL><LI> 1xLiAc/40% PEG 3350/1xTE
</LI></UL><P>At first, 3,5g PEG 3350 were dissolved in 7ml of bidestillated water, resulting in a 50% PEG solution. 6,4ml of this 50% PEG solution were mixed with 800µl 10xTE and 800µl 10x LiAc and sterile filtrated.
</P><P>Used amount of vector DNA:
</P><UL><LI> p542: 5µl 
</LI><LI> p549: 13µl
</LI></UL><P>Positive controll: eGFP (see caffeine group)
Negative controll: water used
</P><P>At last, the cells were plated on appropriate agar- plates with SC -Ura medium and incubated at 30°C for the next three days. Colonies should be visible on friday.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_over_night_cultures_for_yeast_expression"> Preparation of over night cultures for yeast expression </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B>
Expression of enzymes CaXMT1, CaMXMT1 and Protein eGFP in yeast.
</P><P><B>Operational sequence:</B></P><P>To express the gene products, a single yeast colonie (transformation was performed on last wednesday) of each plate was used for inoculation of 15ml  SC -URA 2% glucose medium. However, there was no colony of a pTUM104_CaDXMT1 transformant (repetition of transformation is already in progress). Incubation was performed at 30°C over night and 180 rpm.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_over_night_cultures_of_positive_sequenced_clones_B1B.2C_B2A_and_B3D_for_plasmid_isolation_and_stock_generation"> Preparation of over night cultures of positive sequenced clones B1B, B2A and B3D for plasmid isolation and stock generation </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B>
The sequencing of the named clones was positive. Thus, a small amount (has been stored in the fridge for a few days, i.e. a rest of the last over night culture) of each clone was used for inoculation of 8ml LB medium containing chloramphenicol (8µl). Incubation was performed at 37°C over night at 180 rpm.
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_August_29th"><B>Wednesday, August 29th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_GFP_.28P567.29.2C_Limonesynthase_.28P509.29.2C_ADH-P_.28720_bp.29_.28P565.29"> Preparative digestion of GFP (P567), Limonesynthase (P509), ADH-P (720 bp) (P565)</SPAN></H3><P><B>Investigator</B>: Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>3 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>6 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,6 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>47,4 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=60 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 20 µl reaction batch 20 µl of plasmid with insert were added
</LI><LI> Digestion was performed at 37C for 3 hours
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_TEF2-P_in_psb1c3_with_Thaumatin_P467"> Ligation of TEF2-P in psb1c3 with Thaumatin P467 </SPAN></H3><P><B>Investigator:</B> Georg
<B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>0,54 µl (=100 ng)
</TD><TD>P492
</TD></TR><TR><TD>4,46 µl
</TD><TD>P467
</TD></TR><TR><TD>1
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>3,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=10µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed for 1h at room temperature
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_TEF1-P_in_psb1c3_with_Thaumatin_P467"> Ligation of TEF1-P in psb1c3 with Thaumatin P467 </SPAN></H3><P><B>Investigator:</B> Georg
<B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>0,87 µl (=100 ng)
</TD><TD>P493
</TD></TR><TR><TD>4,93 µl
</TD><TD>P467
</TD></TR><TR><TD>1
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>2,7 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=10µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed for 1h at room temperature
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_ADH1-P_in_psb1c3_with_Thaumatin_P467"> Ligation of ADH1-P in psb1c3 with Thaumatin P467 </SPAN></H3><P><B>Investigator:</B> Georg
<B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>2,25 µl (=100 ng)
</TD><TD>P494
</TD></TR><TR><TD>4,45 µl
</TD><TD>P467
</TD></TR><TR><TD>1
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>1,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=10µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed for 1h at room temperature
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_psb1c3_P132_with_Thaumatin_P467"> Ligation of psb1c3 P132 with Thaumatin P467 </SPAN></H3><P><B>Investigator:</B> Georg
<B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,49 µl
</TD><TD>P132
</TD></TR><TR><TD>6,51 µl
</TD><TD>P467
</TD></TR><TR><TD>1
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>1,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=10µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed for 1h at room temperature
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_Cyc1-T_.28P130.29_in_psb1c3_.28P132.29"> Ligation of Cyc1-T (P130) in psb1c3 (P132)</SPAN></H3><P><B>Investigator:</B> Georg
<B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>6,51 µl
</TD><TD>P130
</TD></TR><TR><TD>1,49 µl
</TD><TD>P132
</TD></TR><TR><TD>1
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>0,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=10µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed for 1h at room temperature
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_TEF1-T_in_psb1c3_P132"> Ligation of TEF1-T in psb1c3 P132 </SPAN></H3><P><B>Investigator:</B> Georg
<B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,5 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,49 µl
</TD><TD>P132
</TD></TR><TR><TD>4,51 µl
</TD><TD>PCR62
</TD></TR><TR><TD>1
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>2,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=10µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed for 1h at room temperature
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_digested_.28Xba.2BPST-HF.29_ADH-P.2C_GFP.2C_Limonens._and_psb1c3_.28P577.29"> Gelextraction of digested (Xba+PST-HF) ADH-P, GFP, Limonens. and psb1c3 (P577)</SPAN></H3><P><B>Investigator:</B> Georg
<B>Procedure:</B></P><UL><LI> Gelextraction was performed according to Quiaquick extraction protocoll 
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_Xl1-blue_with_ligation_batch_.281h.29_psb1c3_with_Thaumatin.2C_Cyc1-T.2C_TEF1-T_and_Thaumatin_with_P492.2CP493_and_P494"> Transformation of <I>E.coli</I> Xl1-blue with ligation batch (1h) psb1c3 with Thaumatin, Cyc1-T, TEF1-T and Thaumatin with P492,P493 and P494 </SPAN></H3><P><B>Investigator:</B> Georg
<B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on Chloramphenicole plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin plates.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_P399.2BP552"> Transformation of <I>E. coli</I> XL1-Blue with ligation product P399+P552 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P399+P552 for the construction of N'-SV40NLS-PhyB(908NT)-20aaLinker-Gal4DBD-C'.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation products (P399+P552) and it's negative control (P399 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and these concentrated cell suspensions was plated again on new chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_gelelectrophoresis_of_P473_and_p556"> Preperative digestion and gelelectrophoresis of P473 and p556 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preperative digestion and gelelectrophoresis of P473 and p556
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for preperative digestion of P473 with XbaI+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P473
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for preperative digestion of P556 with XbaI+PstI-HF
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P556
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Preperative digestion was performed for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4.44 µl of DNA loading buffer (10x) was added to the digestion mix after incubation time.
</LI></UL><UL><LI> Preperative gelelectrophoresis was perforemd at 70 V for 1.5 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P473 (XbaI+PstI-HF)
</TD><TD>P556 (XbaI+PstI-HF)
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Digested Backbone (lower band!) was cut out
</TD><TD>Digested Backbone pYESnew without promoter was cut out
</TD></TR></TBODY></TABLE><UL><LI> The gel oucuts were undergone a gel extraction with the gel extraction kit from Qiagen.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P571.2BPCR72"> Cycled ligation of P571+PCR72 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Cycled ligation was performed for P571+PCR72 and it's negative control P571 NK.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P571+PCR72
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.54 µl
</TD><TD>P571 (39.3 ng/µl, 2048 bp)
</TD></TR><TR><TD>5.19 µl
</TD><TD>PCR72 (23.2 ng/µl, 825 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>9.27 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_E._Coli_with_ligation_products">Transformation of <I>E. Coli</I> with ligation products </SPAN></H3><P><B>Investigator:</B> Martin
</P><P><B>Aim of the experiment:</B>Transform E.coli XL 1 blue with ligation products of ligation 28.8.
</P><P>Transformations:
</P><P>1. PCR57 (digested 7th August) in pSB1C3 (P133)
</P><P>3. PCR 42 (digested 7th August) in pSB1C3 (P133)
</P><P>Transformation into E.coli Xl1-Blue
</P><PRE>   thawing of 100 µl Ca-competent E.coli XL1-Blue cells on ice
   adding of 1 µl of DNA (positive plasmids) or of 5 µl of ligation products
   incubation for 30 min on ice
   heat shock for 5 min at 37 °C
   adding of 1 ml LB-medium without antibiotics to the cells and incubation at 37°C and 180 rpm for 45 min
   plating of 100 µl of the cells on an Chloramphenicol-plate (pSB1C3, P133)
</PRE></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Small_Scale_Yeast_Transformation_-_Preparatory_culture_2"> Small Scale Yeast Transformation - Preparatory culture </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim:</B> Preculture of <I>S. cerevisiae</I> according to pYes2_manuel_kommentiert.
</P><P>1. Inoculate 4 ml of YPD medium with a colony of S. cerevisiae and shake well overnight at <B>30 °C</B>.
</P><P><B>Results:</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_cells_for_protein_expression_in_yeast_4">Preparation of cells for protein expression in yeast </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B>Transfer of a certain amount of transformed Saccharomyces cerevisiae cells from yesterday to new medium with galactose for protein expression in yeast
</P><UL><LI> Measurement: OD600 = 1.9 (used OD600 = 0.4)
</LI><LI> 10.5 ml of resuspended S. cerevisiae were added to 39.5 ml SCU medium with galactose for reaching OD600 = 0.4
</LI><LI> Incubation at 30 °C overnight
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_colonies">Picking of colonies </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Procedure:</B></P><UL><LI> Picking of 6 clones each of PCR56 pYES and PCR57 pYES (transformation 28th August)
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Picking_of_Integrationvector_mOrange">Picking of Integrationvector_mOrange</SPAN></H3><P><B>Investigator:</B> Martin
</P><P><B>Aim:</B>To gain a stock of plasmids in order to integrate biobricks into yeast genome
</P><P><B>Procedure:</B> Pick a clone from the plate, put it into 7 ml LB-medium (7 µl Ampicillin added) and incubate over night.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Repetition of sequencing to check whether insert is wrong. This time, p546 (PCR56/p133)and p512 (PCR58/p133) were sequenced. Furthermore, p550 (PCR58/pYES) was sequenced.
</P><UL><LI>p133: Primer VF2
</LI><LI>pYES: Primer T7
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Yeast_expression_Part_1"> Yeast expression Part 1 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B>
Induction of protein expression of CaXMT1, CaMXMT1 and eGFP
</P><P><B>Operational sequence:</B></P><P>At first, the OD600 of the over night cultures were measured:
</P><UL><LI> CaXMT1: 4,4/ml
</LI><LI> CaMXMT1: 3,9/ml
</LI><LI> eGFP: 4,5/ml
</LI></UL><P>(1/10 dilutions have been prepared for measurment)
</P><P>To obtain an OD of 0,4/ml in 50ml of SC -URA 2% gal medium, 4,5ml, 3,9ml and 4,5ml of the over night cultures were centrifuged at 1500xg for minutes at 4°C. Afterwards, the pellet was resuspended in 2ml SC -URA 2% gal. The suspension was then given to 48ml SC -URA 2% gal medium, followed by incubation at 180rpm and 30°C.
</P><P>Afterwards, 5ml samples were taken of each culture after incubation times of 20min, 140 min and 320 min. The samples were first stored in the fridge and then centrifuged at 1500xg and 4°C for 5 minutes. After washing with 500µl sterile water, the pellet was stored at -80°C until further usage (cell lysis). Furthermore, the OD600 was measured: 
</P><P>After 20 min:
</P><UL><LI> CaXMT1: 0,41
</LI><LI> CaMXMT1: 0,40
</LI><LI> eGFP: 0,42
</LI></UL><P>After 140 min:
</P><UL><LI> CaXMT1: 0,43
</LI><LI> CaMXMT1: 0,42
</LI><LI> eGFP: 0,45
</LI></UL><P>After 320 min:
</P><UL><LI> CaXMT1: 0,5
</LI><LI> CaMXMT1: 0,45
</LI><LI> eGFP: 0,5
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_glycerol_stocks_of_positive_sequenced_pSB1C3_Insert_clones"> Preparation of glycerol stocks of positive sequenced pSB1C3_Insert clones </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Operational sequence:</B>
750µl of each of the three over night cultures with pSB1C3_CaXMT1 (clone B1B), pSB1C3_CaMXMT1 (clone B2A) and pSB1C3 (clone B3D) were mixed with 250µl glycerol and stored at -80°C (in the box of the competent XL1-blue cells).
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Sequencing_of_newly_prepared_pTUM104_RFC25_Insert_plasmids_.28clones:_Y1G.2C_Y2G.2C_Y3G.29"> Sequencing of newly prepared pTUM104_RFC25_Insert plasmids (clones: Y1G, Y2G, Y3G) </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> Proof of inserted sequence (former sequencing failed)
</P><P><B>Results:</B></P><UL><LI> pTUM104_CaXMT1 with T7 primer
</LI></UL><P>GCCGCTTCTAGAGTACACAATGTCTTTACAAGAAGTCTTGAGAATGAACGGTGGTGAAGGTGATACTTCTTACGCTAAAA
ACTCCGCCTACAATCAATTGGTTTTGGCTAAAGTTAAGCCAGTCTTGGAACAATGCGTCAGAGAATTATTGAGAGCTAAC
TTGCCAAACATCAACAAGTGCATTAAGGTTGCTGATTTGGGTTGTGCTTCTGGTCCAAATACTTTGTTGACTGTTAGAGA
CATCGTCCAATCCATTGATAAGGTTGGTCAAGAAAAGAAGAACGAATTGGAAAGACCAACCATCCAAATTTTCTTGAACG
ACTTGTTCCCAAACGACTTCAACTCTGTTTTTAAGTTGTTGCCATCCTTCTACAGAAAGTTGGAAAAAGAAAACGGTAGA
AAGATCGGTTCCTGTTTGATTGGTGCTATGCCAGGTTCTTTCTACTCCAGATTATTTCCTGAAGAATCCATGCATTTCTT
GCACTCTTGTTATTGCTTGCAATGGTTGTCTCAAGTTCCATCTGGTTTGGTTACTGAATTGGGTATTTCTACCAACAAGG
GTTCCATCTACTCTTCTAAAGCTTCAAGATTGCCAGTTCAAAAGGCCTACTTGGATCAATTCACTAAGGATTTCACCACC
TTTTTGAGAATCCACTCCGAAGAATTATTCTCCCACGGTAGAATGTTGTTGACCTGTATATGTAAGGGTGTTGAATTGGA
TGCTAGAAACGCCATTGATTTGTTGGAAATGGCTATCAACGATTTGGTTGTTGAAGGTCACTTAGAAGAAGAAAAGTTGG
ACTCTTTCAACTTGCCAGTTTACATTCCATCTGCCGAAGAAGTTAAGTGCATCGTTGAAGAAGAAGGTTCCTTCGAAATC
TTGTACTTGGAAACTTTCAAGGTCTTGTACGATGCCGGTTTCTCTATTGATGATGAACATATTAAGGCCGAATACGTTGC
CTCTTCTGTTAGAGCTGTTTACGAACCTATTTTGGCTTCTCATTTCGGTGAAGCC
</P><UL><LI> pTUM104_CaXMT1 with reversed primer
</LI></UL><P>GCGTGATGTAGCGTGACATAACTAATTACATGATGCGGCCCTCTAGTCTGCAGCGGCCGCTACTAGTATCATCACTTTTC
GAATTGTGGATGTGACCAAGCAGAAACATCAGACTTTTCTGGCTTTTTGGCCAAGGAGATGATCAAGTTGTTGTAAAAAC
CTTTACCCAATGGCAAAACCTTAGCAGCGTGCTTAGCAAATCTATGAAAGATATCTGGGATAATGGCTTCACCGAAATGA
GAAGCCAAAATAGGTTCGTAAACAGCTCTAACAGAAGAGGCAACGTATTCGGCCTTAATATGTTCATCATCAATAGAGAA
ACCGGCATCGTACAAGACCTTGAAAGTTTCCAAGTACAAGATTTCGAAGGAACCTTCTTCTTCAACGATGCACTTAACTT
CTTCGGCAGATGGAATGTAAACTGGCAAGTTGAAAGAGTCCAACTTTTCTTCTTCTAAGTGACCTTCAACAACCAAATCG
TTGATAGCCATTTCCAACAAATCAATGGCGTTTCTAGCATCCAATTCAACACCCTTACATATACAGGTCAACAACATTCT
ACCGTGGGAGAATAATTCTTCGGAGTGGATTCTCAAAAAGGTGGTGAAATCCTTAGTGAATTGATCCAAGTAGGCCTTTT
GAACTGGCAATCTTGAAGCTTTAGAAGAGTAGATGGAACCCTTGTTGGTAGAAATACCCAATTCAGTAACCAAACCAGAT
GGAACTTGAGACAACCATTGCAAGCAATAACAAGAGTGCAAGAAATGCATGGATTCTTCAGGAAATAATCTGGAGTAGAA
AGAACCTGGCATAGCACCAATCAAACAGGAACCGATCTTTCTACCGTTTTCTTTTTCCAACTTTCTGTAGAAGAT
</P><UL><LI> pTUM104_CaMXMT1 with T7 primer
</LI></UL><P>GCGCTTCTAGAGTACACAATGTCTTTACAAGAAGTCTTGCACATGAACGAAGGTGAAGGTGATACTTCTTACGCTAAGAA
TGCTTCTTACAACTTGGCTTTGGCTAAGGTTAAGCCATTCTTGGAACAATGCATCAGAGAATTATTGAGAGCCAACTTGC
CAAACATCAACAAGTGTATTAAGGTTGCTGATTTGGGTTGTGCTTCTGGTCCAAATACTTTGTTGACTGTTAGAGACATC
GTCCAATCCATTGATAAGGTTGGTCAAGAAGAAAAGAACGAATTGGAAAGACCAACCATCCAAATTTTCTTGAACGACTT
GTTCCAAAACGACTTCAACTCCGTTTTTAAGTTGTTGCCATCCTTCTACAGAAAGTTGGAAAAAGAAAACGGTAGAAAGA
TCGGTTCCTGCTTGATTTCTGCTATGCCAGGTTCTTTTTACGGTAGATTATTCCCTGAAGAATCCATGCATTTCTTGCAC
TCTTGTTACTCTGTTCACTGGTTGTCTCAAGTTCCATCTGGTTTGGTTATTGAATTGGGTATTGGTGCTAACAAGGGTTC
CATCTATTCTTCTAAAGGTTGTAGACCACCAGTTCAAAAGGCTTACTTGGATCAATTCACTAAGGACTTCACCACTTTCT
TGAGAATCCACTCCAAAGAATTATTCTCCAGAGGTAGAATGTTGTTGACCTGTATCTGTAAGGTTGACGAATTTGATGAA
CCTAACCCATTGGATTTGTTGGATATGGCCATTAACGATTTGATCGTCGAAGGTTTGTTGGAAGAAGAAAAGTTGGACTC
CTTCAACATTCCATTCTTTACTCCATCTGCCGAAGAAGTTAAGTGCATCGTTGAAGAAGACGTTCTTGCGAAATCTTGTA
CTTGGAAACTTTCAAGGCTCATTACGATGCTGCCTTCTCTATTGATGATGATTACCCAGTTAGATCCCACGAACAAATCA
AGCTGAT
</P><UL><LI> pTUM104_CaMXMT1 with reversed primer
</LI></UL><P>CGTGATGTAGCGTGACATAACTAATTACATGATGCGGCCCTCTAGTCTGCAGCGGCCGCTACTAGTATCATCACTTTTCG
AATTGTGGATGTGACCAAGCAGAAACATCAGACTTTTCTGGCTTTTTGGCCAAGGAGATAATCAAGTTGTTGTAACAACC
CTTACCCATGTGTAAAACCTTAGCAGCATGTTTAGCCAATCTGTGAAACAAATCTGGCATAATAGCTTCACCGAAATGAG
AGGCCAAAATAGGTTCGTAAACAGATCTGATCAAGGAGGCAACGTATTCAGCTTTGATTTGTTCGTGGGATCTAACTGGG
TAATCATCATCAATAGAGAAGGCAGCATCGTAATGAGCCTTGAAAGTTTCCAAGTACAAGATTTCGCAAGAACCTTCTTC
TTCAACGATGCACTTAACTTCTTCGGCAGATGGAGTAAAGAATGGAATGTTGAAGGAGTCCAACTTTTCTTCTTCCAACA
AACCTTCGACGATCAAATCGTTAATGGCCATATCCAACAAATCCAATGGGTTAGGTTCATCAAATTCGTCAACCTTACAG
ATACAGGTCAACAACATTCTACCTCTGGAGAATAATTCTTTGGAGTGGATTCTCAAGAAAGTGGTGAAGTCCTTAGTGAA
TTGATCCAAGTAAGCCTTTTGAACTGGTGGTCTACAACCTTTAGAAGAATAGATGGAACCCTTGTTAGCACCAATACCCA
ATTCAATAACCAAACCAGATGGAACTTGAGACAACCAGTGAACAGAGTAACAAGAGTGCAAGAAATGCATGGATTCTTCA
GGGAATAATCTACCGTAAAAAGAACCTGGCATAGCAGAATCAAGCAGGAACCGATCTTTCTACCGTTTTCTTTTTCCAAC
TTTCTGTAGAAGGATGGCAACAACTTAAAAACGGAGTTGAAGTCGTTTTGGACAAGTCGTTCAGAAAATTTGGATGGTTG
GTCTTTCCATTCGTTCTTTCTTCTTGACCAACCTTA
</P><UL><LI> pTUM104_CaDXMT1 with T7 primer
</LI></UL><P>CGGCCGCTTCTAGAGTACACAATGTCTTTACAAGAAGTCTTGCATATGAACGGTGGTGAAGGTGATACTTCTTACGCTAA
GAACTCTTTCTACAACTTGTTCTTGATCAGAGTCAAGCCAATCTTGGAACAATGCATCCAAGAATTATTGAGAGCCAACT
TGCCAAACATCAACAAGTGTATTAAGGTTGCTGATTTGGGTTGTGCTTCTGGTCCAAATACTTTGTTGACTGTTAGAGAC
ATCGTCCAATCCATTGATAAGGTTGGTCAAGAAAAGAAGAACGAATTGGAAAGACCAACCATCCAAATTTTCTTGAACGA
CTTGTTCCAAAACGACTTCAACTCCGTTTTTAAGTCCTTGCCATCCTTCTACAGAAAGTTGGAAAAAGAAAACGGTAGAA
AGATCGGTTCCTGTTTGATTGGTGCTATGCCAGGTTCTTTTTACGGTAGATTATTCCCTGAAGAATCCATGCATTTCTTG
CATTCTTGTTACTGCTTGCACTGGTTGTCTCAAGTTCCATCTGGTTTGGTTACTGAATTGGGTATTTCTGCTAACAAGGG
TTGCATCTACTCTTCTAAAGCTTCAAGACCACCAATTCAAAAGGCCTACTTGGATCAATTCACTAAGGATTTCACCACTT
TCTTGAGAATCCACTCCGAAGAATTGATCAGTAGAGGTAGAATGTTGTTGACCTGGATCTGCAAAGAAGATGAATTTGAA
AACCCAAACTCCATCGATTTGTTGGAAATGTCCATCAACGATTTGGTTATCGAAGGTCACTTAGAAGAAGAAAAGTTGGA
CTCTTTCAACGTTCCAATCTATGCTCCATCTACCGAAGAAGTTAAGTGCATCGTTGAAGAAGAAGGTTCCTTCGAAATCT
TGTACTTGGAAACCTTTAAAGTTCCATACGATGCCGGTTTCTCTATCGATGATGATTATCAAGGTAGATCCCACTCTCCA
GTTTCTTGTGATGAACATGCTAGAGCTGCTCATGTTGCTTCAGTTGTAGAT
</P><UL><LI> pTUM104_CaDXMT1 with reversed primer
</LI></UL><P>CGTGATGTAGCGTGACATAACTAATTACATGATGCGGCCCTCTAGTCTGCAGCGGCCGCTACTAGTATCATCACTTTTCG
AATTGTGGATGTGACCAAGCAGAAACATCGGACTTTTCTGGCTTTTTGGCCAAGGAAATAATCAAGGAGTCGTAAAAACC
CTTACCAGATCTCAAAACCTTGGCAGCATTCTTAGCAATTCTATGGGACAAATCTGGCATAATAGCTTCACCGAAATGAG
AGGCAACGATAGGTTCGAAAATAGATCTAACAACTGAAGCAACATGAGCAGCTCTAGCATGTTCATCACAAGAAACTGGA
GAGTGGGATCTACCTTGATAATCATCATCGATAGAGAAACCGGCATCGTATGGAACCTTAAAGGTTTCCAAGTACAAGAT
TTCGAAGGAACCTTCTTCTTCAACGATGCACTTAACTTCTTCGGTAGATGGAGCATAGATTGGAACGTTGAAAGAGTCCA
ACTTTTCTTCTTCTAAGTGACCTTCGATAACCAAATCGTTGATGGACATTTCCAACAAATCGATGGAGTTTGGGTTTTCA
AATTCATCTTCTTTGCAGATCCAGGTCAACAACATTCTACCTCTACTGATCAATTCTTCGGAGTGGATTCTCAAGAAAGT
GGTGAAATCCTTAGTGAATTGATCCAAGTAGGCCTTTTGAATTGGTGGTCTTGAAGCTTTAGAAGAGTAGATGCAACCCT
TGTTAGCAGAAATACCCAATTCAGTAACCAAACCAGATGGAACTTGAGACAACCAGTGCAAGCAGTAACAAGAATGCAAG
AAATGCATGGATTCTTCAGGGAATAATCTACCGTAAAAAGAACCTGGCATAGCACCAATCAAACAGGAACCGATCTTTCT
ACCGTTTTCTTTTTCCAACTTTCTGTAGAAGGATGGCAAGGACTTAAAAACGGAGTTGAAGTCGTTTTGGAACAGTCGTT
CAGAAAATTTGGATGGTTGGTCTTTCCAATTCGTTCTTCTTTCTTGACCACCTTATCAATGGATTGGACGATGTCTCTAA
C
</P><P><B>Conclusion:</B></P><P>The sequencing has worked and the sequences are right. These clones will be used for the further experiments (yeast transformation, expression, etc.)
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_B2A.2CB1B.2CB3D"> Miniprep of B2A,B1B,B3D</SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> Miniprep of over night cultures with number B2A, B1B, B3D (clones were isolated of overnightculture, 5ml LB-medium with CHLP).
</P><P>The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 40 µl of elution buffer (Kit) at room temperature, with the columns having been incubated 5 minutes at room temperature before the final centrifugation. 
<B>Miniprep products are stored in a 50 ml Falcon tube in the lowest drawer at -20°C </B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_B2A.2C_B1B.2C_B3D"> Preperative digestion of B2A, B1B, B3D </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> digestion of pSB1C3_CaXMT1 (clone B1B), pSB1C3_CaMXMT1 (clone B2A) and pSB1C3 (clone B3D)
<B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>40 µl
</TD><TD>B1B, B2A, B3D
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>2,5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preperative_Gelelectrophoresis_of_pSB1C3_CaMXMT1">Preperative Gelelectrophoresis of pSB1C3_CaMXMT1</SPAN></H3><P><B>Investigator:</B> Dennis
</P><UL><LI> 4.44 µl of DNA loading buffer (10x) was added to the digestion mix after incubation time.
</LI></UL><UL><LI> Preperative gelelectrophoresis was perforemd at 90 V for 1 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pSB1C3_CaMXMT1 (clone B2A)
</TD><TD>pSB1C3_CaMXMT1 (clone B1B)
</TD></TR></TBODY></TABLE><UL><LI> Digested Inserts B1B, B2A was cut out (lower band!- ca1200bp)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pSB1C3_CaMXMT1 (clone B3D)
</TD></TR></TBODY></TABLE><UL><LI> Digested Insert B2D was cut out (lower band!- ca1200bp)
</LI></UL></DIV><DIV class="caffein"><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Gelextraction">Gelextraction </SPAN></H3><P><B>Investigator</B>:Dennis
</P><UL><LI> Gelextraction of B2A, B1B, B3D was performed according to Quiaquick gel extraction protocol 
</LI><LI> following concentrations were detected by Nanodrop: 
</LI><LI> B2A ==&gt; 13,6 ng/µl
</LI><LI> B1B ==&gt; 12,8 ng/µl
</LI><LI> B2D ==&gt; 8,4 ng/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P492.2BB2A.2C_P4492.2BB1B.2C_P493.2BB3D"> Cycled ligation of P492+B2A, P4492+B1B, P493+B3D </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> Cycled ligation was performed for P492 (Konz: 91 ng/µl), P493 (Konz: 57 ng/µl)
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P492+B2A
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2  µl
</TD><TD>P492 (91 ng/µl, 2048 bp)
</TD></TR><TR><TD>16,5  µl
</TD><TD>B2A (13,5  ng/µl, 825 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=21 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P492+B1B
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2  µl
</TD><TD>P492 (91 ng/µl, 2048 bp)
</TD></TR><TR><TD>16,5  µl
</TD><TD>B1B (12,4  ng/µl, 825 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=21 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P493+B3D
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1,25  µl
</TD><TD>P493 (91 ng/µl, 2048 bp)
</TD></TR><TR><TD>17,5  µl
</TD><TD>B3D (8,4  ng/µl, 825 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Gelelectrophoresis_of_colony_pcr_products">Gelelectrophoresis of colony pcr products</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Check products of colony pcr whether there are positive clones.
</P><P><B>Results:</B> See pictures that were uploaded into lab journal entry of colony pcr (august 28th).
</P><P>--&gt; Positive clones were transferred into 4 ml of LB medium with antibiotic (Chlp -&gt; pSB1C3)
</P><P>--&gt; please prep plasmid DNA and do an analytical restriction digest
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_August_30th"><B>Thursday, August 30th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_picked_colonies_from_E.coli_transformed_with_TEF2-P_in_psb1c3_from_P451"> Miniprep of picked colonies from <I>E.coli</I> transformed with TEF2-P in psb1c3 from P451</SPAN></H3><P><B> Investigator:</B> Georg
</P><UL><LI> Miniprep was performed according Quiaprep plasmid extraction kit
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_TEF2-psb1c3_clones_P588_and_P591"> Preparative digestion of TEF2-psb1c3 clones P588 and P591</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>8 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,8 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>29,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=40µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 20 µl reaction batch 20 µl plasmid-DNA was added
</LI><LI> Digestion was performed for 3 h at 37C
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_digested_TEF2-P">Gelextraction of digested TEF2-P</SPAN></H3><P><B>Investigator:</B>Georg
</P><UL><LI> Gelextraction was performed according to Quiaquick gel extraction protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_measurement_of_digested_limonensynthases.2C_GFP.2C_ADH1-P.2C_TEF2-P">Nanodrop measurement of digested limonensynthases, GFP, ADH1-P, TEF2-P</SPAN></H3><P><B>Investigator:</B> Georg
</P><UL><LI>P578 Limonens.: 36,7 ng/µl
</LI><LI>P576 GFP: 15,7 ng/µl
</LI><LI>P575 ADH-P: 11,2 ng/µl
</LI><LI>P588 TEF2-P: 289,7 ng/µl
</LI><LI>P589 TEF2-P: 225,1 ng/µl
</LI><LI>P590 TEF2-P: 260,5 ng/µl
</LI><LI>P591 TEF2-P: 433,2 ng/µl
</LI><LI>P592 TEF2-P (Preparative digestion (XBAI+PSTI-HF)): 50 ng/µl
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P494_and_P567">Ligation of P494 and P567</SPAN></H3><P><B>Investigator:</B> Georg
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>4,52 µl (=100 ng)
</TD><TD>P494
</TD></TR><TR><TD>4,88 µl (=100 ng)
</TD><TD>P567
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>7,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed 1h at room temperature
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P492_and_P567">Ligation of P492 and P567</SPAN></H3><P><B>Investigator:</B> Georg
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,08 µl (=100 ng)
</TD><TD>P492
</TD></TR><TR><TD>4,92 µl (=100 ng)
</TD><TD>P567
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>11 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed 1h at room temperature
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P493_and_P567">Ligation of P493 and P567</SPAN></H3><P><B>Investigator:</B> Georg
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,73 µl (=100 ng)
</TD><TD>P493
</TD></TR><TR><TD>5,37 µl (=100 ng)
</TD><TD>P567
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>9,9 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was performed 1h at room temperature
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_digested_TEF2-P_.28XbaI.2BPstI.29">Gelextraction of digested TEF2-P (XbaI+PstI)</SPAN></H3><P><B>Investigator:</B>Georg
</P><UL><LI> Gelextraction was performed according to Quiaquick gelextraction-kit
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparation_of_new_Chloramphenicole_plates">Preparation of new Chloramphenicole plates</SPAN></H3><P><B> Investigator:</B> Georg
</P><UL><LI> To 2 L of LB-Medium 15 g Agar was added
</LI><LI> The mixture was then autoclaved 
</LI><LI> 2 ml 1000x Chloramphenicole was given when temperature has cooled down properly
</LI><LI> Agar was distributed to plates and then stored in the cooler
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_P571.2BPCR72"> Transformation of <I>E. coli</I> XL1-Blue with ligation product P571+PCR72 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P571+PCR72 for bricking the fusion protein of N'-SV40NLS-Gal4AD-Linker-Pif3-C'.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation products (P571+PCR72) and it's negative control (P571) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and these concentrated cell suspensions was plated again on new chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_of_P399.2BP552"> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P399+P552 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P399+P552.
</P><P><B>Procedure:</B></P><UL><LI> 6 colonies of the transformation with P399+491 (CamR) and 10 colonies of the transformation with P399+P552 (CamR) were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chloramphenicol. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_P20_.28BBa_J04450_in_pSB1C3.29"> Picking of <I>E. coli</I> XL-Blue cells, transformated with P20 (BBa_J04450 in pSB1C3) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with P20 (BBa_J04450 in pSB1C3), in order to introduce RFC25 pre- and suffix by PCR to the RFP generator device as a helper construct for cloning with RFC25/RFC10 parts.
</P><P><B>Procedure:</B></P><UL><LI> 4 colonies of the transformation with P20 (CamR) and were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and 4 µl of chloramphenicol (1000x). These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Design_of_an_minimal_multiple_cloning_site_to_insert_protein_coding_part_between_promoter_and_terminator_for_pTUM104new-without-promoter_and_pSB1C3"> Design of an minimal multiple cloning site to insert protein coding part between promoter and terminator for pTUM104new-without-promoter and pSB1C3 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><UL><LI><B>Ask me if you want to use constitutive promoter and terminator for express your target enzyme or protein or you want to integrate your coding device into the genome!!</B> (Jeff)
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P572_with_P299.2C_P575.2C_P592.2C_PCR59"> Cycled ligation of P572 with P299, P575, P592, PCR59 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Ligation of TEF1-P, TEF2-P, ADH1-P in pYES2-TUM
</P><P><B>Procedure:</B></P><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,12 µl (=100 ng)
</TD><TD>P572
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>2,08 µl
</TD><TD>P299
</TD></TR><TR><TD>12,3 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,16 µl
</TD><TD>P572
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>3,84µl
</TD><TD>P575
</TD></TR><TR><TD>10 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,13 µl
</TD><TD>P572
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>3,27µl
</TD><TD>PCR59
</TD></TR><TR><TD>10,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,18 µl
</TD><TD>P572
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>0,82 µl
</TD><TD>P592
</TD></TR><TR><TD>13 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Yeast_expression_Part_2"> Yeast expression Part 2 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B> Expression of gen products CaXMT1, CaMXMT1 and eGFP
</P><P>Analogous to yesterday, one more sample of each yeast culture growing in SC -URA 2%gal medium was taken (5ml), 20h and 24h after induction. Furthermore, the OD600 was determined:
</P><P>20h:
</P><UL><LI> CaXMT1: 4,9
</LI><LI> CaMXMT1: 4,6
</LI><LI> eGFP: 5,8
</LI></UL><P>24h:
</P><UL><LI> CaXMT1: 5,6
</LI><LI> CaMXMT1: 5,4
</LI><LI> eGFP: 6,8
</LI></UL><P>The samples were handled as described yesterday and stored at -80°C
</P><P><B>Note:</B> At the eGFP pellet is green ==&gt; expression of positive control works
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_with_P492.2BB1B.2C_P492.2BB2A.2C_P493.2BB3D"> Transformation of <I>E. coli</I> with P492+B1B, P492+B2A, P493+B3D </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products of P492+B1B, P492+B2A, P493+B3D.
</P><P><B>Procedure:</B></P><UL><LI> 20 µl of each ligation products were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Small_scale_yeast_transformation_of_S._cerevisiae_with_pTUM104_Preprothaumatin.2C_Integration-Vector_mOrange_and_pTUM104_GFP"> Small scale yeast transformation of <I>S. cerevisiae</I> with pTUM104_Preprothaumatin, Integration-Vector_mOrange and pTUM104_GFP </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P>Procedure according to pYes2_manual_kommentiert:
</P><UL><LI> 2. Determine the OD600 of your overnight culture. Dilute culture to an OD600 of 0.4 in 50 ml of YPD medium, add 5ml 20% glucose and grow an additional 2–4 hours. 
</LI></UL><P>3. Pellet (5 min, 4°C) the cells at 2500 rpm and resuspend the pellet in 40 ml 1X TE. 
4. Pellet (5 min, 4°C) the cells at 2500 rpm and resuspend the pellet in 2 ml of 1X LiAc/0.5X TE. 
5. Incubate the cells at room temperature for 10 minutes. 
6. For each transformation, mix together 1 μg plasmid DNA and 100 μg (10 μl) denatured sheared salmon sperm DNA with 100 μl of the yeast suspension from Step 5. 
7. Add 700 μl of 1X LiAc/40% PEG-3350/1X TE (7 ml 50% PEG3350 was synthesized (3.5 g in 7 ml bidest.water); 6.4 ml 50% PEG3350 + 800 μl 10x LiAc + 800 μl 10x TE were mixed) and mix well. 
8. Incubate solution at 30°C for 30 minutes. 
9. Add 88 μl DMSO, mix well, and heat shock at 42°C for 7 minutes. 
10. Centrifuge in a microcentrifuge for 10 minutes and remove supernatant. 
11. Resuspend the cell pellet in 1 ml 1X TE and re-pellet. 
12. Resuspend the cell pellet in 50–100 μl 1X TE and plate on a selective plate (SCU-plates + Glucose, respectively Kanamycin-YPD plates for the Integration-Vector_mOrange). Incubate the plates at 30°C over the weekend.
</P><P><B>Name:</B> pYes2_Thaumatin/Integ-Vek/GFP, S. cerevisiae, Schappert, Bräuer. 
</P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Miniprep_of_Integvek_mOrange"> Miniprep of Integvek_mOrange </SPAN></H3><P><B>Investigator</B>: Martin, Alois
</P><P><B>Aim:</B> To gain enough plasmid to digest with restriction enzymes (and ligate with other inserts than the mOrange) and to transform yeast.
</P><P><B>Procedure:</B> Miniprepkit - Qiagen, Procedure according to the manual
</P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Gelextraction_of_Thaumatin.2C_Limonene-Synthase_and_Integration-Vector-Backbone"> Gelextraction of Thaumatin, Limonene-Synthase and Integration-Vector-Backbone </SPAN></H3><P><B>Investigator</B>: Martin, Alois
</P><P><B>Aim:</B> P583 (Thaumatin), 584 (Limonene-Synthase), 585 (Integration-vector-backbone)
</P><P><B>Procedure:</B> Gelextraction with a 1%-LMP-Agarose-Gel - Qiagen Gelextraction Kit
</P><P>Picture of Integration-Vector (Backbone at 3650 bp)
</P><P>Picture of Thaumatin (721 bp, upper) and Limonene (1650bp, lower)
</P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Ligation_of_Thaumatin.2FLimonene_and_Integration-Vector"> Ligation of Thaumatin/Limonene and Integration-Vector </SPAN></H3><P><B>Investigator</B>: Martin, Alois
</P><P><B>Aim:</B> Ligate the Thaumatin/Limonene as inserts into the Integration-Vector.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P583+P585 and P584+P585
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5.5 µl
</TD><TD>P585 (18,4 ng/µl, 3600 bp)
</TD></TR><TR><TD>4.6 µl
</TD><TD>P583 (13.1 ng/µl, 721 bp)
</TD><TD>P584 (28.5 ng/µl, 1650 bp)
</TD></TR><TR><TD>0,5 µl
</TD><TD>BSA
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>5.9 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=21 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Transformation_of_E._Coli_XL1blu_with_Integ-Vec_Thaumatin_and_Integ-Vec_Limonene-Synthase"> Transformation of <I>E. Coli</I> XL1blu with Integ-Vec_Thaumatin and Integ-Vec_Limonene-Synthase </SPAN></H3><P><B>Investigator</B>: Martin, Alois
</P><P><B>Aim:</B> To establish a <I>E. Coli</I>-Strain carrying the Integration-Vector backbone with a Thaumatin, respectively Limonene-Synthase, insert.
</P><P><B>Procedure:</B> The products of the Ligation before were used to transform <I>E. Coli</I> XL1blu. For the procedure have a close look at the quadrizillion transformations before. I'm outta here.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_products_and_colony_PCR_products">Miniprep of ligation products and colony PCR products</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Check ligation products (PCR56/P375 (pYES), PCR57/P375 (pYES)). Check colony PCR clones (3, 5, 6, 7, 8) .
</P><UL><LI> PCR56/pYES (tube 1): 242 ng/µl
</LI><LI> PCR56/pYES (tube 2): 71 ng/µl
</LI><LI> PCR56/pYES (tube 3): 63 ng/µl
</LI><LI> PCR56/pYES (tube 4): 36 ng/µl
</LI><LI> PCR56/pYES (tube 5): 137 ng/µl
</LI><LI> PCR56/pYES (tube 6): 63 ng/µl
</LI></UL><UL><LI> PCR57/pYES (tube 1): 74 ng/µl
</LI><LI> PCR57/pYES (tube 2): 98 ng/µl
</LI><LI> PCR57/pYES (tube 3): 124 ng/µl
</LI><LI> PCR57/pYES (tube 4): 100 ng/µl
</LI><LI> PCR57/pYES (tube 5): 152 ng/µl
</LI><LI> PCR57/pYES (tube 6): 85 ng/µl
</LI></UL><UL><LI> Colony PCR (clone 3): 63.5 ng/µl
</LI><LI> Colony PCR (clone 5): 101 ng/µl
</LI><LI> Colony PCR (clone 6): 161.5 ng/µl
</LI><LI> Colony PCR (clone 7): 61 ng/µl
</LI><LI> Colony PCR (clone 8): 59 ng/µl
</LI></UL><P><B>Eppis are stored in a disposal bag (label: Miniprep, Andrea, 30.8.) in the middle drawer of -20°C.</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_transformed_ligation_products_and_colony_PCR_products">Analytical restriction digest of transformed ligation products and colony PCR products</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Check that transformed clones are positive.
</P><UL><LI>PCR56/pSB1C3 (transformation 28th August)
</LI><LI>PCR57/pSB1C3 (transformation 28th August)
</LI><LI>Colony PCR (clone 3, 5, 6, 7, 8) 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xba 1
</TD></TR><TR><TD>0.25 µl
</TD><TD>Spe-HF 1
</TD></TR><TR><TD>2 µl
</TD><TD>NEB
</TD></TR><TR><TD>0,2 µl
</TD><TD>BSA
</TD></TR><TR><TD>14.3 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P><B>Gelelectrophoresis:</B></P><UL><LI>expected: 
</LI></UL><P>Limonene synthase 1600 bp
</P><P>pSB1C3 (P133)     2000 bp
</P><P>ligations negative
colony PCR products negative
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_cells_of_protein_expressed_yeasts">Preparation of cells of protein expressed yeasts</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B> Get yeast cell pellets for further cell disruption
</P><UL><LI> measure OD600 at different time points (after 15 hours, 20 hours, 24 hours)
</LI><LI> centrifugation of each 5 ml (4°C, 5 min, 5000x g)
</LI><LI> resuspend pellet in 1ml ddH2O 
</LI><LI> centrifugation of each resuspended pellet (4°C, 5 min, 5000x g)
</LI><LI> stored pellet at -20°C
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_PMSF_solution">Preparation of PMSF solution</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B> Get PMSF solution for breaking buffer for yeast expression procedure
</P><UL><LI> 0.0174 g in 1ml Isopropanol (techn.)
</LI><LI> stored Eppi at 4°C (great iGEM fridge) 
</LI><LI> added 60 µl to breaking buffer --&gt; breaking buffer + PMSF ready for yeast cell disruption
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Friday.2C_August_31st"><B>Friday, August 31st</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_with_ligation_product_pTUM100_with_PCR59_.28TEF1-T.29.2C_TEF2-P.2C_TEF1-P_and_ADH1-P"> Transformation of <I>E.coli</I> with ligation product pTUM100 with PCR59 (TEF1-T), TEF2-P, TEF1-P and ADH1-P</SPAN></H3><P><B>Investigator: Georg</B><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on ampicillin plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_of_colonies_with_ligation_products_CYC-T.2Bpsb1c3.2C_TEF1-T-psb1c3.2C_Thaumatin.2B_ADH1-TEF2-TEF1-P">Picking of colonies with ligation products CYC-T+psb1c3, TEF1-T-psb1c3, Thaumatin+ ADH1-TEF2-TEF1-P</SPAN></H3><P><B>Investigator:Georg</B></P><UL><LI> 42 colonies were picked and transferred to 5 ml medium with 5 µl 1000x CAM in case of ligation and CYC-T and TEF1-T
</LI></UL><UL><LI> Ampicillin in case of ligation products of Thaumatin
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_product_of_P399.2BP552"> Miniprep of <I>E. coli</I> XL-Blue cells, transformated with ligation product of P399+P552 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P399+P552 (P573).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done after manufacturer's protocol. (QIAGEN - QIAprep Spin Miniprep Kit)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_P601-P606"> Analytical digestion and gelelectrophoresis of the minipreps P601-P606 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps P601-P606 with XbaI+PstI-HF.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion for P601-P606 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>14 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1.75 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1.75 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>103.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=122.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of plasmid DNA (P601-P606).
</LI></UL><UL><LI> The reaction mixes were incubated at 37 °C for 90 min.
</LI></UL><UL><LI> 9 µl of the digested DNA was mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for about 60 min.
</LI></UL><P>P601-P606:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P601
</TD><TD>P602
</TD><TD>P603
</TD><TD>P604
</TD><TD>P605
</TD><TD>P606
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Ligation seems to  be successful. Sequencing has to be done!
</TD><TD>Ligation seems to  be successful. Sequencing has to be done!
</TD><TD>Ligation seems to  be successful. Sequencing has to be done!
</TD><TD>Ligation seems to  be successful. Sequencing has to be done!
</TD><TD>Ligation seems to  be successful. Sequencing has to be done!
</TD><TD>Ligation seems to  be successful. Sequencing has to be done!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_of_P571.2BPCR72"> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P571+PCR72 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P571+PCR72.
</P><P><B>Procedure:</B></P><UL><LI> 6 colonies of the transformation with P571+PCR72 (CamR) were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chloramphenicol. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_clones:_PSB1C3_P57_and_PSB1C3_P42"> Picking of clones: PSB1C3_P57 and PSB1C3_P42 </SPAN></H3><P><B>Investigator:</B> Martin, Alois <B>(???)</B></P><P><B>Aim:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P113 and P57/P42.
</P><P><B>Procedure:</B></P><UL><LI> 2 colonies of the transformation (29th August) from each plate were taken, 4 in total.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 7 ml of LB-medium and chloramphenicol. These tubes were put in a 180 rpm cell culture shaker at 37 °C overnicht.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Harvest_proteins_expressed_in_yeasts">Harvest proteins expressed in yeasts</SPAN></H3><P><B>Aim:</B> Cell disruption for harvesting proteins expressed in yeasts
</P><P><B>Procedure:</B></P><UL><LI>resuspending stored cell pellets in defined volume of braking buffer + PMSF
</LI><LI>adding glass beads to eppis
</LI><LI>vortex for 30 min and incubate on ice for 30 min (repetition for 25 times)
</LI><LI>centrifuge at 4 °C, full speed, 10 min
</LI><LI>take supernatant and centrifuge at 4 °C, full speed, 10 min again
</LI><LI>store supernatant at -80°C (second box)
</LI></UL><UL><LI>concentrations
</LI></UL><P>LS (15 hours after galactose induction): 4.053 mg/ml
</P><P>LS (20 hours after galactose induction): 1.932 mg/ml
</P><P>LS (24 hours after galactose induction): 4.828 mg/ml
</P><P>Caffein CaXMT1 (20 hours after galactose induction): 10.142 mg/ml
</P><P>Caffein CaXMT1 (24 hours after galactose induction): 2.433 mg/ml
</P><P>Caffein CaMXMT1 (20 hours after galactose induction): 8.508 mg/ml
</P><P>Caffein CaMXMT1 (24 hours after galactose induction): 4.712 mg/ml
</P><P>Caffein eGFP (20 hours after galactose induction): 6.661 mg/ml
</P><P>Caffein eGFP (24 hours after galactose induction): 4.244 mg/ml
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Repitition_of_analytical_restriction_digest_of_transformed_ligation_products_and_colony_PCR_products">Repitition of analytical restriction digest of transformed ligation products and colony PCR products</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Check that transformed clones are positive.
</P><UL><LI>PCR56/pSB1C3 (transformation 28th August)
</LI><LI>PCR57/pSB1C3 (transformation 28th August)
</LI><LI>Colony PCR (clone 3, 5, 6, 7, 8) 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>Xba 1
</TD></TR><TR><TD>0.5 µl
</TD><TD>Spe-HF 1
</TD></TR><TR><TD>2 µl
</TD><TD>NEB
</TD></TR><TR><TD>0,2 µl
</TD><TD>BSA
</TD></TR><TR><TD>12.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>incubation: 2 hours, 37°C
</LI></UL><P><B>Gelelectrophoresis:</B></P><P>needs to be done
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_schwab_vectors">Analytical restriction digest of schwab vectors</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Check that schwab plasmids and insert are right.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>P40
</TD></TR><TR><TD>0.3 µl
</TD><TD>Nco1
</TD></TR><TR><TD>0.3 µl
</TD><TD>Hind3
</TD></TR><TR><TD>2 µl
</TD><TD>Tango Buffer
</TD></TR><TR><TD>11.4 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>P42
</TD></TR><TR><TD>0.3 µl
</TD><TD>Not1
</TD></TR><TR><TD>0.3 µl
</TD><TD>EcoR1
</TD></TR><TR><TD>2 µl
</TD><TD>Orange Buffer
</TD></TR><TR><TD>11.4 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>incubation: 2 hours, 37°C
</LI><LI>digested probes were stored at -20°C (Ingmar) 
</LI></UL><P><B>Gelelectrophoresis:</B></P><P>needs to be done
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Preparation_of_SC-U_medium_with_2.25_glucose_.282_l.29_or_galactose_.282_l.29_for_expression_in_Saccharomyces_cerevisiae">Preparation of SC-U medium with 2% glucose (2 l) or galactose (2 l) for expression in <I>Saccharomyces cerevisiae</I></SPAN></H3><P><B>Investigator:</B> Maddin
</P><P>4 x 1 l medium:
</P><UL><LI>One aliquot (50 ml) of amino acids prepared by <B>Alois Bräuer</B> was used
</LI><LI>6.7 g yeast nitrogen base
</LI><LI>850 ml ELGA water
</LI><LI>autoclaving
</LI></UL><P>Sugar solution were prepared separately:
</P><UL><LI>40 g glucose were dissolved in 200 ml ELGA water
</LI><LI>40 g galactose were dissolved in 200 ml ELGA water
</LI><LI>autoclaving
</LI></UL><P>After autoclaving:
</P><UL><LI>100 ml of glucose solution were added to 1 l SC-U medium, 2x50 ml were added to 2x500 ml SC-U medium.
</LI><LI>100 ml of galactose solution were added to 1 l SC-U medium, 2x50 ml were added to 2x500 ml SC-U medium.
</LI></UL><P><B>Notes:</B> The media are stored in the shelf above the bench in the iGEM laboratory.
</P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Picking_of_clones:_Integvec_Thaumatin_.28P585_P583.29_and_Integvec_Limonene-Synthase_.28P585_P584.29"> Picking of clones: Integvec_Thaumatin (P585_P583) and Integvec_Limonene-Synthase (P585_P584) </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P585 and P583/P584.
</P><P><B>Procedure:</B></P><UL><LI> 2 colonies of the transformation from each plate were taken, 4 in total.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 7 ml of LB-medium and Ampicillin. These tubes were put in a 180 rpm cell culture shaker at 37 °C overnight.
</LI></UL></DIV></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_September_1st"><B>Saturday, September 1st</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_colonies_with_Cyc-T-TEF1-T.2Bpsb1c3_and_Thaumatin_with_ADH1P-TEF2P-TEF1P_in_pTUM104"> Gelextraction of colonies with Cyc-T-TEF1-T+psb1c3 and Thaumatin with ADH1P-TEF2P-TEF1P in pTUM104</SPAN></H3><P><B>Investigator:Georg</B></P><UL><LI>Gelextraction was done according to Quiaprep-protocol
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_colonies_with_Cyc-T-TEF1-T.2Bpsb1c3_and_Thaumatin_with_ADH1P-TEF2P-TEF1P_in_pTUM100">Analytical digestion of colonies with Cyc-T-TEF1-T+psb1c3 and Thaumatin with ADH1P-TEF2P-TEF1P in pTUM100 </SPAN></H3><P><B>Investigator:Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>11,25 µl
</TD><TD>EcorI-HF (NEB)
</TD></TR><TR><TD>11,25 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>90 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>9 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>778,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=900 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> to 2,5 ml of DNA 17,5 µl reaction batch were added
</LI><LI> Digestion took place at 37°C for 1 h
</LI></UL><UL><LI> TEF1-T, CYC1-T, ADH1-P+Thaum and Tef2-P+Thaum were succesfully transformated into E.coli
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P571.28psb1c3.29_with_PCR_62_.28TEF1-T.29"> Ligation of P571(psb1c3) with PCR 62 (TEF1-T)</SPAN></H3><P><B>Investigator:Georg</B><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>8,16 µl
</TD><TD>PCR62
</TD></TR><TR><TD>2,54 µl (=100 ng)
</TD><TD>P571
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>6,3 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation was cycled at 12 and 22 C
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_product_of_P571.2BPCR72"> Miniprep of <I>E. coli</I> XL-Blue cells, transformated with ligation product of P571+PCR72 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL-Blue cells, transformated with ligation products of P571+PCR72 (P586).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done after manufacturer's protocol (P611-P616). (QIAGEN - QIAprep Spin Miniprep Kit)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_P611-P616"> Analytical digestion and gelelectrophoresis of the minipreps P611-P616 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of the minipreps P611-P616 with XbaI+PstI-HF.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion for P601-P606 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>14 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>1.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1.75 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1.75 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>103.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=122.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of plasmid DNA (P611-P616).
</LI></UL><UL><LI> The reaction mixes were incubated at 37 °C for 90 min.
</LI></UL><UL><LI> 9 µl of the digested DNA was mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for about 60 min.
</LI></UL><P>P611-P616:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P611
</TD><TD>P612
</TD><TD>P613
</TD><TD>P614
</TD><TD>P615
</TD><TD>P616
</TD><TD>PCR72 control
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD></TR></TBODY></TABLE><UL><LI> N'-SV40-Gal4AD-Linker-Pif3-C' has been successfully cloned with RFC10 pre- and suffix into pSB1C3.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_schwab_vector_and_repetition_of_ligation_product_and_colony_PCR_products_.28continue.29">Analytical restriction digest of schwab vector and repetition of ligation product and colony PCR products (continue)</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Check that schwab plasmids and insert are right.
</P><P><B>Gelelectrophoresis:</B></P><P>
expected:
</P><P>Limonensynthase: 1600 bp
</P><P>pYES: 5800 bp
</P><P>pSB1C3: 2000 bp
</P><P>pGEX: 5000 bp
</P><P>pET: 5700 bp
</P><P>ligations: failed
</P><P>colony PCR: negative
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_products_of_P57_and_P42">Miniprep of ligation products of P57 and P42</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Check ligation products (PCR57/pSB1C3, PCR42/pSB1C3).
</P><UL><LI> PCR57/pSB1C3 (1): 155.8 ng/µl
</LI><LI> PCR57/pSB1C3 (2): 65.1 ng/µl
</LI></UL><UL><LI> PCR42/pSB1C3 (1): 217.7 ng/µl
</LI><LI> PCR42/pSB1C3 (2): 52.4 ng/µl
</LI></UL><P><B>ligation products (P57/P42) are stored in a 50 ml Falcon tube (label: Miniprep, Andrea, 01.09.) in the upper drawer of -20°C.</B></P></DIV><DIV class="genome integration"><H3><SPAN class="mw-headline" id="Miniprep_of_ligated_integration_vector">Miniprep of ligated integration vector</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Check integration vector with thaumatin (P607, P608) and limonensynthase (P609, P610).
</P><UL><LI> Integration vector + Thaumatin (1): 580 ng/µl
</LI><LI> Integration vector + Thaumatin (2): 383.8 ng/µl
</LI></UL><UL><LI> Integration vector + Limonensynthase (1): 484.2 ng/µl
</LI><LI> Integration vector + Limonensynthase (2): 714.7 ng/µl
</LI></UL><P><B>ligation products (integration vector) are stored in plasmid box 6 (P607 - P610)</B></P></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_September_2nd"><B>Sunday, September 2nd</B></SPAN></H2><DIV class="integration"><H3><SPAN class="mw-headline" id="Picking_of_S._cerevisiae_clones_from_SC-U_plates_.28pTUM104_preprothaumatin_and_pTUM104_limonene-synthase.29_and_YPD-Kan_plates_.28integrational_vector.29"> Picking of <I>S. cerevisiae</I> clones from SC-U plates (pTUM104_preprothaumatin and pTUM104_limonene-synthase) and YPD-Kan plates (integrational vector) </SPAN></H3><P><B>Investigator:</B> Martin
</P><P><B>Aim:</B> to establish a strain of <I>S. cerevisiae</I> with desired genotype
</P><P><B>Procedure:</B> Same procedure as every year.
</P></DIV><H1><SPAN class="mw-headline" id="Week_13">Week 13</SPAN></H1><DIV class="week" id="WWeek_13"><H2><SPAN class="mw-headline" id="Monday.2C_September_3rd"><B>Monday, September 3rd</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Plasmidextraction_of_P451.2CP450.2C_P51_and_P403"> Plasmidextraction of P451,P450, P51 and P403</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI>Plasmid-DNA was extracted according to Quiaprep genextraction kit
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_Digestion_of_ADH1-P_and_TEF2-P.28P451.29_with_XbaI.2BPstI-HF.2C_as_well_as_Spei-HF_and_PstI-HF"> Preparative Digestion of ADH1-P and TEF2-P(P451) with XbaI+PstI-HF, as well as Spei-HF and PstI-HF</SPAN></H3><P><B> Investigator: Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,5 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>2,5 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>10 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>1 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>34 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=50µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,5 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>2,5 µl
</TD><TD>SpeI-HF (NEB)
</TD></TR><TR><TD>10 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>1 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>34 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=50µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 20 µl reaction batch, 20 µl of DNA were added and digested for 3 h at 37 C
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_measurement_of_ADH-T_and_TEF2-P"> Nanodrop measurement of ADH-T and TEF2-P</SPAN></H3><P><B>Investigator:Georg</B></P><UL><LI>ADH1-T(1) in psb1c3: 113,2 ng/µl
</LI><LI>ADH1-T(2) in psb1cr: 158,2 ng/µl
</LI><LI> &quot; &quot;(3): 159,2 ng/µl
</LI><LI>TEF2-P in psb1c3:214,3 ng/µl
</LI><LI>TEF2-P in psb1c3:108,2 ng/µl
</LI><LI>TEF2-P in psb1c3:200 ng/µl
</LI></UL><UL><LI> P678: 90,2 ng/µl
</LI><LI> P679: 79,6 ng/µl
</LI><LI> P680 : 52,2 ng/µl
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P578_with_P680_and_P679"> Ligation of P578 with P680 and P679</SPAN></H3><P><B>Investigator: Georg</B><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>2,45 µl (=100 ng)
</TD><TD>P578
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>1,91 µl
</TD><TD>P680
</TD></TR><TR><TD>12,64 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><B>Investigator: Georg</B><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>2,55 µl (=100 ng)
</TD><TD>P578
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>1,25 µl
</TD><TD>P680
</TD></TR><TR><TD>13,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative controls were mixed with ddH20 instead of the insert
</LI><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparation_of_P588_.28TEF2-P_.28450.29_for_sequencing"> Preparation of P588 (TEF2-P (450) for sequencing</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> 5 µl DNA (289,7 ng/µl) and 12 ml H20 with seq. Number 025
</LI></UL></DIV><DIV class="constitutive¬_promoter"><DIV class="constitutive¬_promoter"><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_TEF1-T-psb1c3">Transformation of TEF1-T-psb1c3</SPAN></H3><P><B>Investigator:Georg</B></P><UL><LI> E.coli was transformed with with P638 for amplification
</LI></UL><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on CAM plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new CAM plates.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_the_minipreps_P534-P539.2C_P611-P616"> Analytical digestion and gelelectrophoresis of the minipreps P534-P539, P611-P616 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> To prove whether SV40 NLS (contains a KpnI resstriction site) are successfully fused to in the minipreps (P534-P539 and P601-P606).
</P><P><B>Procedure:</B></P><P>Mastermix for KpnI-HF(NEB)+BamHI(Fermentas)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>26 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2.6 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>3.25 µl
</TD><TD>KpnI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>6.5 µl
</TD><TD>BamHI (10 U/µl) (Fermentas)
</TD></TR><TR><TD>189.15 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=227.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of plasmid DNA (P534-P539, P601-P606).
</LI></UL><UL><LI> The reaction mixes were incubated at 37 °C for 90 min.
</LI></UL><UL><LI> 9 µl of the digested DNA was mixed with 1 µl of DNA loading buffer (10x).
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for about 60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P534
</TD><TD>P535
</TD><TD>P536
</TD><TD>P537
</TD><TD>P538
</TD><TD>P539
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>SV40 NLS successfully fused!
</TD><TD>SV40 NLS successfully fused!
</TD><TD>SV40 NLS successfully fused!
</TD><TD>SV40 NLS successfully fused!
</TD><TD>SV40 NLS successfully fused!
</TD><TD>SV40 NLS successfully fused!
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P601
</TD><TD>P602
</TD><TD>P603
</TD><TD>P604
</TD><TD>P605
</TD><TD>P606
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>SV40 NLS successfully fused!
</TD><TD>SV40 NLS successfully fused!
</TD><TD><B>No SV40 NLS fused!</B></TD><TD>SV40 NLS successfully fused!
</TD><TD>SV40 NLS successfully fused!
</TD><TD>SV40 NLS successfully fused!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preperative_digestion_and_preperative_gelelectrophoresis_of_P616.2C_P606.2C_P539.2C_P638"> Preperative digestion and preperative gelelectrophoresis of P616, P606, P539, P638 </SPAN></H3><P><B>Investigator:</B> Jeff, Georg
</P><P><B>Aim of the experiment:</B> Preperative digestion and preperative gelelectrophoresis of:
</P><UL><LI> P616 (SV40NLS-Gal4AD-Linker-Pif3(100NT)) (XbaI + PstI-HF)
</LI></UL><UL><LI> P606 (SV40NLS-PhyB(908NT)-20aaLinker-Gal4DBD) (SpeI-HF + PstI-HF)
</LI></UL><UL><LI> P539 (SV40NLS-PhyB(908NT)-20aaLinker-LexA) (SpeI-HF + PstI-HF)
</LI></UL><UL><LI> P638 (TEF1 terminator) (SpeI-HF + PstI-HF and SpeI-HF + PstI-HF ).
</LI></UL><P><B>Procedure:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P616
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P606
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P539
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>P638
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>18.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15 µl
</TD><TD>P638
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>18.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Preperative digestion was performed at 37 °C for 2.5 h.
</LI></UL><UL><LI> Preperative gelelectrophoresis was performed at 70 V for 3 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P616 XbaI + PstI-HF
</TD><TD>1 kbp DNA ladder
</TD><TD>P606 SpeI-HF + PstI-HF
</TD><TD>P606 SpeI-HF + PstI-HF
</TD></TR><TR><TD>Lower band was cut out
</TD><TD>Band was cut out
</TD><TD>Band was cut out
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 bp DNA ladder
</TD><TD>P638 XbaI + PstI-HF
</TD><TD>1 kbp DNA ladder
</TD><TD>P638 SpeI-HF + PstI-HF
</TD></TR><TR><TD>Band was cut out
</TD><TD>Band was cut out
</TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_digest_and_gel_electrophoresis"> Analytical digest and gel electrophoresis </SPAN></H3><P><B>Investigator:</B> Andrea
<B>Aim:</B> Check that transformed clones are positive.
</P><UL><LI>PCR42/pSB1C3 (transformation 29th August)
</LI><LI>PCR57/pSB1C3 (transformation 29th August)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.5 µl
</TD><TD>Xba 1
</TD></TR><TR><TD>0.5 µl
</TD><TD>Spe-HF 1
</TD></TR><TR><TD>2 µl
</TD><TD>NEB
</TD></TR><TR><TD>0,2 µl
</TD><TD>BSA
</TD></TR><TR><TD>12.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>incubation: 2 hours, 37°C
</LI></UL><P><B>Gelelectrophoresis:</B></P><P>all ligations: negative
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_cells_for_protein_expression_in_yeast_5">Preparation of cells for protein expression in yeast </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B>Picking transformed Saccharomyces cerevisiae cells for protein expression in yeast
</P><UL><LI> Inoculation of overnight cultures: one clone from plate with PCR 1 in P175 clone 2 (Miniprep 21.08.) (29) &amp; one clone from plate with P542 (PCR2) (Miniprep 28.08.) was picked,resuspended in 15 ml SCU medium with glucose and incubated at 30 °C overnight
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="SDS-_Page_of_crude_extract_.28CaXMT1.2C_CaMXMT1_and_eGFP_expression.2C_as_well_as_three_samples_of_the_limonene_group.29"> SDS- Page of crude extract (CaXMT1, CaMXMT1 and eGFP expression, as well as three samples of the limonene group) </SPAN></H3><P><B>Investigator:</B> Roman, Andrea
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment is the proof of successful protein expression of CaXMT1, CaMXMT1 and eGFP (used as positive controll), as well as xxx (limonene).
</P><P><B>Operational sequence:</B></P><P>The SDS gel was prepared as previously described (12%). 10µl of cell lysate (20h after induction of expression) was mixed with 2,5 µl Laemmli- RED SDS loading buffer, resulting in an amount of protein 100µg (CaXMT1- crude extract), 80µg (CaMXMT1- crude extract) and 60µg (eGFP- crude extract).
</P><P>The proteins were separated at 120V for about 1,5 hours. Afterwards, we did a western blot and used the same gel for a staining with coomassie brilliant blue.
</P><P>Picture of gel (after having washed it with washing solution 1 (15 min) and 2 (4 hours):
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Prestained Proteine Ladder
</TD><TD>LimX
</TD><TD>Limy
</TD><TD>Limz
</TD><TD>CaXMT1
</TD><TD>CaMXMT1
</TD><TD>eGFP
</TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_cells_for_transformation_in_yeast">Preparation of cells for transformation in yeast </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B>Picking Saccharomyces cerevisiae colony (INVSc1) for frther transformation in yeast
</P><UL><LI> Inoculation of overnight cultures: one clone from plate with INVSc1 from 23.08. was picked,resuspended in 4 ml YPD medium and incubated at 30 °C overnight
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Western_blot_of_20_h_samples_of_the_expression_of_CaXMT1.2C_CaMXMT1_and_eGFP">Western blot of 20 h samples of the expression of CaXMT1, CaMXMT1 and eGFP</SPAN></H3><P><B>Investigator:</B> Saskia, Andrea
</P><P><B>Aim:</B> Analysis of the expression of CaXMT1, CaMXMT1 and eGFP (positive controll)after SDS-PAGE (120V, 1.5 h)
</P><P><B>Procedure:</B><B>preparation of solutions:</B> as described in the materials
</P><UL><LI>PBS-T0.1
</LI><LI>PBS-T0.1 with 3% BSA
</LI></UL><P><B>Blotting:</B></P><UL><LI>blotting of proteins on ImmobilonP Membrane (activating in methanol for 10 min)
</LI><LI>gel and membrane in transferbuffer for 20 min
</LI><LI>50mA, 1 h
</LI></UL><P><B>Blocking:</B></P><UL><LI>wash 3x7min with PBS-T0.1
</LI><LI>the membrane is blocked over night in PBS-T0.1 with 3% BSA at 4 °C on a shaking device
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_B2A.2C_B1B.2C_B3D"> Miniprep of B2A, B1B, B3D </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> Miniprep of 9 over night cultures with number B2A (3 times), B1B (3 times), B3D (3 times) (clones were isolated of over night culture, 5ml LB-medium with CHLP).
</P><P>The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 40 µl ddH2O at room temperature, with the columns having been incubated 2 minutes at room temperature before the final centrifugation. 
<B>Miniprep products are stored in a 50 ml Falcon tube in the lowest drawer at -20°C </B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytic_digestion_of_B2A.2C_B1B.2C_B3D"> Analytic digestion of B2A, B1B, B3D </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> digestion of pSB1C3_CaXMT1 (clone B1B), pSB1C3_CaMXMT1 (clone B2A) and pSB1C3 (clone B3D)
<B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,5 µl
</TD><TD>B1B, B2A, B3D
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0,5 µl
</TD><TD>Ecor1 (20 U/µl)
</TD></TR><TR><TD>0,5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>14,5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytic_Gelelectrophoresis_of_B2A.2C_B1B.2C_B3D">Analytic Gelelectrophoresis of B2A, B1B, B3D</SPAN></H3><P><B>Investigator:</B> Dennis
</P><UL><LI> 4.44 µl of DNA loading buffer (10x) was added to the digestion mix after incubation time.
</LI></UL><UL><LI> analytic gelelectrophoresis was perforemd at 80 V for 1 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pSB1C3 CaMXMT1 (clone B2A 3)
</TD><TD>pSB1C3 CaMXMT1 (clone B1B 2)
</TD><TD>pSB1C3 CaMXMT1 (clone B3D 2)
</TD><TD>pSB1C3 CaMXMT1 (clone B1B 1)
</TD><TD>pSB1C3 CaMXMT1 (clone B3D 3)
</TD><TD>pSB1C3 CaMXMT1 (clone B3D 2)
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pSB1C3 (P463)
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_E.Coli_with_pTUM104_CaXMT1.2C_pTUM104_CaMXMT1_and_pTUM104_CaDXMT1_as_backup">Transformation of <I>E.Coli</I> with pTUM104_CaXMT1, pTUM104_CaMXMT1 and pTUM104_CaDXMT1 as backup</SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> To fill up the amount of available pTUM104 for further yeast transformations
</P><P><B>Operational sequence:</B></P><P>The transformation was done as previously described. 100µl of unconcentrated <I>e.coli-</I> cells were plated on ampicillin plates and incubated over night at 37°C.
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_September_4th"><B>Tuesday, September 4th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_P403_.28ADH1-T_in_psb1c3.29_in_E.coli_Xl1-blue">Transformation of P403 (ADH1-T in psb1c3) in <I>E.coli</I> Xl1-blue</SPAN></H3><P><B>Investigator:Georg</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on CAM plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new CAM plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_Cyc1-T-Kol2.2C_TEF1-P-Kol1.2C_ADH1-P_and_TEF2-P_Kol6_in_psb1c3_with_XbaI_and_PstI-HF">Preparative digestion of Cyc1-T-Kol2, TEF1-P-Kol1, ADH1-P and TEF2-P Kol6 in psb1c3 with XbaI and PstI-HF</SPAN></H3><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>4 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>24 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>2,4 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>63,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=80µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17,8 µl were taken from each batch for digestion and added to 20 ml of the mastermix
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparation_of_P450_TEF2-P_in_psb1c3_for_sequencing"> Preparation of P450 TEF2-P in psb1c3 for sequencing</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> 10 µl P450 were mixed with 5 µl ddH20 for 50-100 ng/µl DNA
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_digested_TEF1-Thaum.2CTEF2-Thaum.2C_ADH1-P-Thaum_and_Cyc-T"> Gelextraction of digested TEF1-Thaum,TEF2-Thaum, ADH1-P-Thaum and Cyc-T</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Gelextraction was performed according to Quiaquick gelextraction protocol
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P572_.28dig._pTUM100.29with_P684.2C_P682.2C_P683"> Ligation of P572 (dig. pTUM100)with P684, P682, P683</SPAN></H3><P><B>Investigator: Georg</B></P><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,24 µl (=100 ng)
</TD><TD>P684 (PYesII digested)
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>10,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3,16 µl
</TD><TD>P572
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,24 µl
</TD><TD>P684
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>10,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3,14 µl
</TD><TD>P572
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,16 µl
</TD><TD>P683
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>10,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3,17 µl
</TD><TD>P572
</TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,11 µl
</TD><TD>P682
</TD></TR><TR><TD>2 µl
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>12,72 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative controls were also prepared, instead of the insert the same amount of ddH20 was added
</LI><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_measurement_of_digested_CYC1-T.2C_P684.2C_P682.2C_P683"> Nanodrop measurement of digested CYC1-T, P684, P682, P683 </SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Cyc1-T (xbaI+PstI)P681 :19,4 ng/µl
</LI><LI> ADH-P with Thaumatin (P684): 28 ng/µl
</LI><LI> TEF1-P with Thaumatin (P682): 61,9 ng/µl
</LI><LI> TEF2-P with Thaumatin (P683):27 ng/µl
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P493.2BP639.2C_P643.2BP299.2C_P640.2BP642.2C_P641.2BP642"> Cycled ligation of P493+P639, P643+P299, P640+P642, P641+P642 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Cycled ligation of P493+P639 (TEF1 promoter + SV40NLS-Gal4AD-30aaLinker-Pif3(100NT)), P643+P299 (TEF1 terminator + TEF1 promoter), P640+P642 (SV40NLS-PhyB(908NT)-20aaLinker-Gal4DBD), P641+P642 (SV40NLS-PhyB(908NT)-20aaLinker-LexA).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P493+P639
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.74 µl
</TD><TD>P493 (57.5 ng/µl, 2506 bp)
</TD></TR><TR><TD>6.31 µl
</TD><TD>P639 (15.6 ng/µl, 824 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>8.95 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P643+P299
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.89 µl
</TD><TD>P643 (52.9 ng/µl, 2564 bp)
</TD></TR><TR><TD>4.37 µl
</TD><TD>P299 (12.6 ng/µl, 472 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>10.74 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P640+P642
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.03 µl
</TD><TD>P640 (97.3 ng/µl, 5366 bp)
</TD></TR><TR><TD>2.53 µl
</TD><TD>P642 (11.7 ng/µl, 530 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>13.44 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P641+P642
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.89 µl
</TD><TD>P641 (112.9 ng/µl, 5531 bp)
</TD></TR><TR><TD>2.45 µl
</TD><TD>P642 (11.7 ng/µl, 530 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>13.66 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative controls were als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batches and were called: P493 NK, P643 NK, P640 NK, P641 NK.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_B2A.2C_B1B.2C_B3D_2"> Preperative digestion of B2A, B1B, B3D </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> preperative digestion of pSB1C3_CaXMT1 (clone B1B 2), pSB1C3_CaMXMT1 (clone B2A 3) and pSB1C3 (clone B3D 2) and the ADH1 terminator
<B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>30 µl
</TD><TD>B1B, B2A, B3D, ADH1 Terminator
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>10,5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>30 µl
</TD><TD>ADH1 Terminator
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>10,5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="preperative_Gelelectrophoresis">preperative Gelelectrophoresis</SPAN></H3><P><B>Investigator:</B> Dennis
</P><UL><LI> 4.44 µl of DNA loading buffer (10x) was added to the digestion mix after incubation time.
</LI></UL><UL><LI> analytic gelelectrophoresis was perforemd at 120 V for 1 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pSB1C3 CaMXMT1 (ADH1 Terminator)
</TD><TD>pSB1C3 CaMXMT1 (clone B2A 3)
</TD><TD>pSB1C3 CaMXMT1 (clone B1B 2)
</TD><TD>pSB1C3 CaMXMT1 (clone B3D 2)
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Gelextraction_2">Gelextraction </SPAN></H3><P><B>Investigator</B>:Dennis
</P><UL><LI> Gelextraction of ADH1 terminator, B2A, B1B, B3D was performed according to Quiaquick gel extraction protocol 
</LI><LI> following concentrations were detected by Nanodrop: 
</LI><LI> ADH1==&gt; 16,6 ng/µl
</LI><LI> B2A_3 ==&gt; 54,6 ng/µl
</LI><LI> B1B_2 ==&gt; 91,8 ng/µl
</LI><LI> B3D_2 ==&gt; 28,4 ng/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_ADH1.2BB2A_3.2C_ADH1.2BB1B_2.2C_ADH1.2BB3D_2"> Cycled ligation of ADH1+B2A_3, ADH1+B1B_2, ADH1+B3D_2 </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> Cycled ligation 
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for ADH1+B2A_3
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1,6  µl
</TD><TD>ADH1 (16,6 ng/µl, 2048 bp)
</TD></TR><TR><TD>2  µl
</TD><TD>B2A_3 (13,5  ng/µl, 825 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>13 ddH20
</TD><TD>=21 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for ADH1+B1B_2
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,66  µl
</TD><TD>ADH1 (16,6 ng/µl, 2048 bp)
</TD></TR><TR><TD>1,8  µl
</TD><TD>B1B_2 (91,4  ng/µl, 825 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12,5 ddH2O
</TD><TD>=21 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for ADH1+B3D_2
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,8  µl
</TD><TD>ADH1 (16,6 ng/µl, 2048 bp)
</TD></TR><TR><TD>1,9  µl
</TD><TD>B3D_2 (28,7  ng/µl, 825 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD> 14 ddH20
</TD><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_of_p647_and_ligation_with_p642">Preparative restriction digest of p647 and ligation with p642</SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim:</B> Getting pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term
</P><P>Preparative restriction digest:
</P><UL><LI> 20 µl p647 (pSB1C3_Ter1Pro_Preprothaumatin), 4 µl NEBufer 4, 0.4 µl BSA, 1 µl SpeI, 1 µl PstI, 13.6 µl ddH2O; 37°C, 2 h.
</LI></UL><P>Preparative gel electrophoresis: 
</P><UL><LI> Expected: ca. 3 kb.
</LI></UL><P>Ligation of p648 (= p647 digested with SpeI and PstI) and ligation with p642 (&quot;TER1Term digested with XbaI, PstI&quot;):
</P><UL><LI> 1 µl T4 DNA ligase, 2 µl T4 DNA ligase buffer, 2 µl p648, 5 µl p642, 10 µl ddH2O; over night, 16°C.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Western_blot_analysis_of_protein_crude_extract_of_CaXMT1.2C_CaMXMT1_and_eGFP_transformants.2C_as_well_as_three_samples_of_limonene_synthase"> Western blot analysis of protein crude extract of CaXMT1, CaMXMT1 and eGFP transformants, as well as three samples of limonene synthase </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Detection of expressed proteins CaXMT1, CaMXMT1, eGFP and limonene synthase (citrus).
</P><P><B>Operational sequence:</B></P><P>The western blot membran (having been blocked over the night at 4°C) was washed with PBS- T0.1 four times (4x 15min). Afterwards, the membran was incubated for 1h with detection reagent.
Detection reagent:
</P><UL><LI> 10ml 1xPBS
</LI><LI> 0,2% BSA (0,02g have been weighted in)
</LI><LI> 2µl anti body MABclassic
</LI></UL><P>After incubating with the first antibody, the membran was washed again with PBS-T0.1 three times (3x 5min), followed by the incubation with the second antibody (anti mouse, fused with alk. phosphatase) for another our.
Second detection reagent:
</P><UL><LI> 10ml 1xPBS
</LI><LI> 0,2% BSA (see above)
</LI><LI> 5µl AntiMouse-alk. Phosphatase conjugate (1/2000 dilution)
</LI></UL><P>The development was performed after washing the membran with PBS- T0.1 for 10 min (two times) and with 1xPBS for 10 min (two times).
For this, the membran was shortly washed with AP buffer and then incubated with a solution containing 15ml AP buffer, 45µl BCIP (50mg/ml in DMF) and 7,5µl NBT (75mg/ml in 70% DMF) for a few minutes, until clear bands appeared.
</P><P>Western blot membran:
</P><P>From left to right:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Prestained protein marker (Page Ruler Plus)
</TD><TD>Limonene synthase (15h)
</TD><TD>Limonene synthase (20h)
</TD><TD>Limonene synthase (24h)
</TD><TD>CaXMT1 crude extract (20h)
</TD><TD>CaMXMT1 crude extract (20h)
</TD><TD>eGFP crude extract (20h)
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Picking_of_colonies_2">Picking of colonies </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Procedure:</B></P><UL><LI> Picking of 3 clones for miniprep: Y2G, Y3G, Y1G (4 ml LB, Amp)
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Picking_of_colony_for_yeast_transformation">Picking of colony for yeast transformation </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Procedure:</B></P><UL><LI> Picking of 1 clone of S. crevisiae INVSc1 for yeast transformation (4 ml YPD)
</LI><LI> Inbucation at 30°C (shaker) over night.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_cells_for_protein_expression_in_yeast_6">Preparation of cells for protein expression in yeast </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B>Transfer of a certain amount of transformed Saccharomyces cerevisiae cells from yesterday to new medium with galactose for protein expression in yeast
</P><UL><LI> Measurement: OD600
</LI><LI> defined amount of resuspended S. cerevisiae were added to SCU medium with galactose for reaching OD600 = 0.4
</LI><LI> Incubation at 30 °C overnight
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Yeast_transformation_with_p542_and_p551"> Yeast transformation with p542 and p551 </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B>
Transformation of yeast cells (INVSC1) with the plasmids p542 (PCR2, Citrus) and p551 (PCR58, Lavendel) and pYES (for Coumaroyl group) 
</P><P><B>Operational sequence:</B></P><P>The yeast transformation was performed as described in the pYES2 manual of Invitrogen (see small scale yeast transformation), using an over night culture of INVSC1- yeast cells. The solutions 1xTE and 1xLiAc/40% PEG 3350/1xTE were made a few days before use. 
</P><P>Used amount of vector DNA:
</P><UL><LI> p542: 4.8 µl 
</LI><LI> p551: 9 µl
</LI><LI>pYES: 8.2 µl
</LI></UL><P>At last, the cells were plated on appropriate agar- plates with SC -Ura medium and incubated at 30°C for the next three days. Colonies should be visible on friday.
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_September_5th"><B>Wednesday, September 5th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_P451_.28TEF2-P_in_psb1c3.29_in_E.coli_Xl1-blue.29"> Transformation of P451 (TEF2-P in psb1c3) in <I>E.coli</I> Xl1-blue)</SPAN></H3><P><B>Investigator: Georg</B><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on CAM plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new CAM plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_TEF1-P_and_ADH1-P_in_pTUM104"> Preparative digestion of TEF1-P and ADH1-P in pTUM104</SPAN></H3><P><B> Investigator: Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3 µl
</TD><TD>SpeI-HF
</TD></TR><TR><TD>3 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>12 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>1,2 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>40,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=60 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 20 µl of reaction batch 20 µl of DNA were added and digested for at 37 C over night
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_gelelectrophoresis_of_ADH1-P.2C_TEF1-P_and_TEF2-P_in_pTUM100"> Preparative gelelectrophoresis of ADH1-P, TEF1-P and TEF2-P in pTUM100</SPAN></H3><P><B>Investigator:Georg</B></P><UL><LI> Digestions were loaded onto 1% ultra pure Agarose gels
</LI><LI> gelrun took 90 min at 70 V
</LI></UL><UL><LI>Digestion was succesfull
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Transformation_and_Picking_of_tonight.27s_ligation_products_.28p648_.2B_p642.29">Transformation and Picking of tonight's ligation products (p648 + p642)</SPAN></H3><P><B>Investigator:</B> Alois the HONK
</P><P><B>Aim:</B> Getting pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term.
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Transformation_of_p647">Transformation of p647</SPAN></H3><P><B>Investigator:</B> Martianus Capella, Nikolaus von Kues
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Yeast_transformation_with_pTUM104_RFC25_CaDXMT1"> Yeast transformation with pTUM104_RFC25_CaDXMT1 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>The aim of the experiment was the transformation of chemo competent yeast cells with the plasmid pTUM104_CaDXMT1, which contains the enzyme &quot;caffeine synthase&quot;, being the last enzyme of the caffein biosynthesis pathway.
</P><P><B>Operational sequence:</B></P><P>Contrary to previous experiments, this transformation was done with the <I>S.C.</I> EasyComp(TM) Transformation Kit (Invitrogen). The chemocompetent yeast cells have already been prepared (thanks to the chair of Prof. Schwab, Biotechnology of natural compounds) as 50µl aliquots.
</P><P>Ca. 2 µg of plasmid DNA (Clone Y3G) were added to the competent cells together with &quot;Solution III&quot; (Kit) as described in the manufacturers protocoll.
</P><P>The reaction batch was plated on SC-U agar plates and incubated at 30°C for 3 - 4 days.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_E.Coli_over_night_cultures.2C_transformed_with_pTUM104_RFC25_constructs"> Miniprep of <I>E.Coli</I> over night cultures, transformed with pTUM104_RFC25 constructs </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of experiment:</B></P><P>Isolation of plasmids: pTUM104_CaXMT1, pTUM104_CaMXMT1 and pTUM104_CaDXMT1 for further usage.
</P><P><B>Operational sequence:</B></P><P>The plasmid preparation was done with the Quiagen Plasmid Miniprep Kit as previously described.
Elution was performed with 40µl of elution buffer, heated up to 37°C. Before final centrifugation, the column was incubated for 2 minutes at 37°C.
</P><P><B>Determined concentrations (NanoDrop)</B></P><UL><LI> pTUM104_CaXMT1 (p553): 179,6 ng/µl
</LI><LI> pTUM104_CaMXMT1 (p554): 340,7 ng/µl
</LI><LI> pTUM104_CaDXMT1 (p555): 182,9 ng/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_with_ADH1.2BB1B_2.2C_ADH1.2BB2A_3.2C_ADH1.2BB3D_2"> Transformation of <I>E. coli</I> with ADH1+B1B_2, ADH1+B2A_3, ADH1+B3D_2</SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products of ADH1+B1B_2, ADH1+B2A_3, ADH1+B3D_2.
</P><P><B>Procedure:</B></P><UL><LI> 20 µl of each ligation products were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_ADH1.2BB1B_1.2C_ADH1.2BB3D_3"> Cycled ligation of ADH1+B1B_1, ADH1+B3D_3 </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> Cycled ligation was performed for B1B_1 and B3D_3
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for ADH1+B1B_1
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,66  µl
</TD><TD>ADH1 (16,6 ng/µl, 2048 bp)
</TD></TR><TR><TD>1,8  µl
</TD><TD>B1B_1 (91,4  ng/µl, 825 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12,5 ddH2O
</TD><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for ADH1+B3D_3
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,8  µl
</TD><TD>ADH1 (16,6 ng/µl, 2048 bp)
</TD></TR><TR><TD>1,9  µl
</TD><TD>B3D_3 (28,7  ng/µl, 825 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD> 14 ddH20
</TD><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_P493.2BP639.2C_P643.2BPP299.2C_P640.2BP642.2C_P641.2BP642"> Transformation of <I>E. coli</I> XL1-Blue with ligation products P493+P639, P643+PP299, P640+P642, P641+P642 </SPAN></H3><P><B>Investigator:</B> Mary
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P493+P639, P643+PP299, P640+P642, P641+P642.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation products (P493+P639, P643+PP299, P640+P642, P641+P642) and their's negative controls (P493 NK, P643 NK, P640 NK, P641 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and these concentrated cell suspensions was plated again on new chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P82"> Preparative digestion and gelelectrophoresis of P82 </SPAN></H3><P><B>Investigator:</B> Mary
</P><P><B>Aim of the experiment:</B> Digest P82 for further cloning experiments
</P><UL><LI> Reaction batch for digestion of P82 with SpeI-HF+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P82
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>Spe-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch was incubated at 37 °C for 2.5 h
</LI></UL><UL><LI> 4.44 µl of DNA loading buffer (10x) was added to the digested product and preperative gelelectrophoresis was performed at 70 V for 1.5 h.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_P664.2C_P51.2C_P52.2C_P53"> Transformation of <I>E. coli</I> XL1-Blue with P664, P51, P52, P53 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P664 (KanR, AmpR), P51 (KanR, AmpR), P52 (KanR), P53 (KanR).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 2 µl of the biobricks (P664, P51, P52, P53) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on kanamycin plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and these concentrated cell suspensions was plated again on new kanamycin plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_BBa_K268000_.28pSB6A0.29"> Picking of <I>E. coli</I> XL-Blue cells, transformated with BBa_K268000 (pSB6A0) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with BBa_K268000 (pSB6A0), a empty hybrid yeast bacteria vector without promoter and terminator. For future cloning of expression cassettes.
</P><P><B>Procedure:</B></P><UL><LI> 4 colonies of the transformation with BBa_K268000 (pSB6A0) were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and ampicillin. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_SCU-medium_containing_galactose_.28induction_medium.29">Preparation of SCU-medium containing galactose (induction medium)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B>Preparation of induction medium for a 2 l expression experiment
</P><P>for 1 liter:
</P><UL><LI> dissolve 50 ml AS-Aliquot + 6.7g Yeast Nitrogen Base in 850 ml of water, then autoclave
</LI><LI> dissolve 20 g galactose in 100 ml water
</LI><LI> autoclave both solutions separately
</LI></UL><P>4 liter of induction medium were made
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Harvest_proteins_expressed_in_yeasts_2">Harvest proteins expressed in yeasts</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim:</B> Cell disruption for harvesting proteins expressed in yeasts (crude protein extract)
</P><P><B>Procedure:</B></P><UL><LI> resuspending cell pellets in defined volume of breaking buffer + PMSF
</LI><LI> adding glass beads
</LI><LI> vortex for 30 seconds and incubate on ice for 30 seconds (repetition for 40 times)
</LI><LI> centrifuge at 4 °C, full speed, 10 min
</LI><LI> take supernatant and centrifuge at 4 °C, full speed, 10 min again
</LI><LI> store supernatant at -80°C (second box)
</LI></UL><P>a 5 ml aliquot of cells was harvested at 9:30 and a 10 ml aliquot was harvested at 12:30
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_PCR_42_.28citrus_limonene_synthase_with_consensus_sequence.29_in_plasmid_P133_.28pSB1C3.29_and_PCR_57_.28lavendula_limonene_synthase_from_Prof._Schwab.29_in_plasmid_P133_.28pSB1C3.29_.286_clones_each.29">Miniprep of PCR 42 (citrus limonene synthase with consensus sequence) in plasmid P133 (pSB1C3) and PCR 57 (lavendula limonene synthase from Prof. Schwab) in plasmid P133 (pSB1C3) (6 clones each)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim:</B> Get more plasmid DNA (PCR42/pSB1C3, PCR 57/pSB1C3)
</P><P>concentrations:
</P><UL><LI> PCR57/P133 clone 1: 75,4 ng/µl
</LI><LI> PCR57/P133 clone 2: 244,6 ng/µl
</LI><LI> PCR57/P133 clone 3: 232,9 ng/µl
</LI><LI> PCR57/P133 clone 4: 63 ng/µl
</LI><LI> PCR57/P133 clone 5: 77,2 ng/µl
</LI><LI> PCR57/P133 clone 6: 100,5 ng/µl
</LI><LI> PCR42/P133 clone 1: 57 ng/µl
</LI><LI> PCR42/P133 clone 2: 73,3 ng/µl
</LI><LI> PCR42/P133 clone 3: 57,4 ng/µl
</LI><LI> PCR42/P133 clone 4: 74,4 ng/µl
</LI><LI> PCR42/P133 clone 5: 75,8 ng/µl
</LI><LI> PCR42/P133 clone 6: 178,4 ng/µl
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="picking_of_clones_for_a_2_l_expression_experiment">picking of clones for a 2 l expression experiment</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim:</B> picking of clones for a 2 l expression experiment
2 clones of S. cerevisiae containing citrus limonene synthase (mit consensus sequence) were picked from the plate Andrea 29 (22.08.) and resuspended in 100 ml SCU medium + glucose
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Vectorfragmentation_of_structural_genes_of_Xanthohumol">Vectorfragmentation of structural genes of Xanthohumol</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> The detection of our desired proteins via western blot failed three times. To rule out that recombinations of vector elements did take place, the vector backbone of our gene constructs and of a pYes vector from the limonengroup containing a gene insert whose protein could be detected via western blot was fragmented and the fragmentation patterns were compared. 
</P><P><B>Operational sequence:</B></P><UL><LI> General reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>x µl
</TD><TD>DNA Template
</TD></TR><TR><TD>1.2 µl
</TD><TD>NEB buffer 3 (10x)
</TD></TR><TR><TD>0.8 µl
</TD><TD>Tango buffer (10x)
</TD></TR><TR><TD>0.2µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>NotI (20 U/µl)
</TD></TR><TR><TD>0,25 µl
</TD><TD>PvuI (20 U/µl)
</TD></TR><TR><TD>0.25 µl
</TD><TD>AflIII (20 U/µl)
</TD></TR><TR><TD>y µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> The volume of DNA was calculated to have 500 ng of DNA. A suitable volume water was added to result in a total volume of 20 µl.
</LI><LI>Incubation at 37 °C for 2h.
</LI><LI>Verification of control digest by agarose gel electrophoresis: 
</LI></UL><P>20 µl of each digest product was mixed with 4 µl 6x DNA loading buffer and loaded into the gel. The separation process lasted 1h at 90 V.
</P><P><A href="/File:TUM12_Xanthohumol_Vektorfragmentierung_1.jpg" title="File:TUM12 Xanthohumol Vektorfragmentierung 1.jpg">Gel picture of vectorfragmentation 1</A></P><P><A href="/File:TUM12_Xanthohumol_Vektorfragmentierung_2.jpg" title="File:TUM12 Xanthohumol Vektorfragmentierung 2.jpg">Gel picture of vectorfragmentation 2</A></P><P>The fragmentation pattern does not show unexpeted bonds. Therefore the gene constructs might have some point mutations which are not visible via Agarose gel electrophoresis. As the transformation of yeast with our constructs resulted in a reasonable number of cfu, it is likely that the URA3 gene and the 2µOri are not affected by possible mutations. Thats why we will continue our work with sequencing our gene constructs including their respective promoters.
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_September_6th"><B>Thursday, September 6th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_P678.2C_P679.2C_P680"> Gelextraction of P678, P679, P680</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Gelextraction was done according to the Quiaquick extraction kit protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="analytical_gelelectrophoresis_with_P678.2C_P680_and_P679">analytical gelelectrophoresis with P678, P680 and P679</SPAN></H3><P><B> Investigator: Georg</B></P><UL><LI> Gelelectrophoresis was done on 0,5% analytical agarose gels with 90 V for 60 min
</LI></UL><UL><LI> Experiment done in order to differentiate tubes
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E._coli_Xl1-blue_with_ligation_products_P572_and_TEF2-P-Thaum.2C_TEF1-P-Thaum_and_ADH1-P-Thaum">Transformation of <I>E. coli</I> Xl1-blue with ligation products P572 and TEF2-P-Thaum, TEF1-P-Thaum and ADH1-P-Thaum</SPAN></H3><P><B>Investigator:Georg</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on ampicillin plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_of_colonies_from_transformants_with_P403_.28ADH1-T.29.2C_P451_.28TEF2-P.29.2C_P450_.28TEF2-P.29_and_P51_.28eGFP.29"> Picking of colonies from transformants with P403 (ADH1-T), P451 (TEF2-P), P450 (TEF2-P) and P51 (eGFP)</SPAN></H3><UL><LI> 5 µl of 1000x Amp-solution were added to 5 ml LB-medium
</LI><LI> colonies were picked to AMP-LB medium
</LI><LI> Colonies grew over night at 37 C
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_biobrick_BBa_K268000_.28emtpy_yeast_vector.29"> Miniprep of <I>E. coli</I> XL-Blue cells, transformated with biobrick BBa_K268000 (emtpy yeast vector) </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL-Blue cells, transformated with biobrick BBa_K268000 (emtpy yeast vector=without promoter and terminator).
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done after manufacturer's protocol (P674-P677). (QIAGEN - QIAprep Spin Miniprep Kit)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P674-P677"> Analytical digestion and gelelectrophoresis of P674-P677 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P674-P677 with XbaI+PstI-HF.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for P674-P677
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P674-P677)
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI (20 U/µl) (NEB)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI (20 U/µl) (NEB)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion reaction mix was incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 2 µl of DNA loading buffer (10x) was added to the 20 µl of reaction mix after digestion.
</LI></UL><UL><LI> 10 µl of these were loaded in the gel pockets.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for ~60 min.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P674
</TD><TD>P675
</TD><TD>P676
</TD><TD>P677
</TD></TR><TR><TD>Biobricks still contain forbidden restriction sites!
</TD><TD>Biobricks still contain forbidden restriction sites!
</TD><TD>Biobricks still contain forbidden restriction sites!
</TD><TD>Biobricks still contain forbidden restriction sites!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_P664"> Transformation of <I>E. coli</I> XL1-Blue with  P664 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with P664 (KanR and AmpR).
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 6 µl (because 2 µl on KanR plates didn't work)of the biobrick BBa_I712019 in pSB1AK8 (P664) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspension were plated on ampicillin plate.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and these concentrated cell suspensions was plated again on new ampicillin plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_P493.2BP639.2C_P643.2BP299.2C_P640.2BP642.2C_P641.2BP642"> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products P493+P639, P643+P299, P640+P642, P641+P642 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products P493+P639, P643+P299, P640+P642, P641+P642.
</P><P><B>Procedure:</B></P><UL><LI> 3 colonies of each transformation with were taken (4x3 colonies = 16 colonies).
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chloramphenicol. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Picking_of_pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term_pSB1C3_TEF1Prom_Preprothaumatin_.28p647.29_and_pTUM104_Preprothaumatin">Picking of pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term pSB1C3_TEF1Prom_Preprothaumatin (p647) and pTUM104_Preprothaumatin</SPAN></H3><P><B>Investigator:</B> Eriugena, Nietzsche
</P><P><B>Aim:</B> Getting pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_SC-U_medium_with_2.25_glucose_.284_l.29_and_transfer_of_colonies_into_medium">Preparation of SC-U medium with 2% glucose (4 l) and transfer of colonies into medium</SPAN></H3><P><B>Investigator:</B> Andrea, Lara
</P><P>Amino acid solution was prepared seperately (see pYES manual):
</P><UL><LI>1 g Adenine, Arginine, Leucine, Threonine, Tryptophan
</LI><LI>0.5 g Aspartic acid, Histidine, Isoleucine, Methionine, Phenylalanine, Proline, Serine, Tyrosine, Valine
</LI><LI>1,49 g Cysteiniumchlorid, Lysindihydrochlorid
</LI><LI><B>six aliquots were stored at the lowest drawer of -20°C.</B></LI></UL><P>
Yeast nitrogen base was prepared separately (by Katrin):
</P><UL><LI>6.7g Yeast nitrogen base in 850 ml ELGA water
</LI></UL><P>
4 l medium (in two 2 l flasks):
</P><UL><LI>4 aliquots (50 ml) of amino acis prepared before were used
</LI><LI>each 6.7 g yeast nitrogen base
</LI><LI>each 850 ml ELGA water
</LI><LI>after dissolving, the solution was autoclaved
</LI></UL><P>
Sugar solution was prepared separately (by Martin):
</P><UL><LI>200g of glucose were dissolved in 1000 ml ELGA water
</LI><LI>autoclaving
</LI></UL><P>
After autoclaving:
</P><UL><LI>200 ml of glucose solution was added to a total of 2 l SC-U medium
</LI></UL><P>--&gt; transfer of pre-culture (100 ml, OD600=5) each into 2 L of SC-U medium
</P><P>--&gt; 4 µl of pre-culture was applied to a SC-U plate to keep clones for further use (incubation at 30°C, <B>check tomorrow</B>!)
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Minipreparation_of_pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term_pSB1C3_TEF1Prom_Preprothaumatin_.28p647.29_and_pTUM104_Preprothaumatin">Minipreparation of pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term pSB1C3_TEF1Prom_Preprothaumatin (p647) and pTUM104_Preprothaumatin</SPAN></H3><P><B>Investigator:</B> Eriugena, Nietzsche, masterblaster
</P><P><B>Aim:</B> Getting pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term.
</P><UL><LI> pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term: p665-p669.
</LI><LI> pSB1C3_TEF1Prom_Preprothaumatin: p670-p671.
</LI><LI> pYes2_Preprothaumatin: p672-p673.
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Analytical_digest_and_analytical_gel_electrophoresis_of_pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term">Analytical digest and analytical gel electrophoresis of pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term</SPAN></H3><P><B>Investigator:</B> Eriugena, Nietzsche, Dennis
</P><P><B>Aim:</B> Getting pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term.
</P><P>Digest 1: 
</P><UL><LI> 5 µl p665-p669, 0.25 µl EcoR1, 0.25 µl Pst1, 0.2 µl BSA, 2 µl NEBuffer EcoR1, 12 µl ddH2O; 37°C, 1h.
</LI></UL><P>Digest 2: 
</P><UL><LI> 5 µl p665-p669, 0.5 µl Not1, 0.2 µl BSA, 2 µl NEBuffer 3, 12 µl ddH2O; 37°C, 1h.
</LI></UL><P>Analytical gel electrophoresis: 1 kb marker, p665_1, p666_1, p667_1, p668_1, p669_1, p665_2, p666_2, p667_3, p668_4, p669_5.
</P><UL><LI> Bild
</LI></UL><UL><LI> Expected: backbone: 2.2 kb, insert: 1.7 kb.
</LI></UL><P><B>Result:</B></P><P>On the left you see the digest with EcoRI and PstI. The bands show the desired length - the ligation was a historic triumph. The bands on the right show the same plasmids digested with NotI - as indicated by sequence data obtained bei Schorsch, the TefI-Terminator lost the NotI restriction site due to a punctual mutation.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Picking_of_colonies_for_miniprep_2"> Picking of colonies for miniprep </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> New miniprep for new transformation of E. coli with ligation batch B2A_3+ADH1 terminator and B1B_2+ ADH1 terminator and B3D_3+ ADH1 terminator
</P><UL><LI> Picking of 3 clones each of  ligation batch
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="preperative_Gelelectrophoresis_2">preperative Gelelectrophoresis</SPAN></H3><P><B>Investigator:</B> Dennis
</P><UL><LI> 4.44 µl of DNA loading buffer (10x) was added to the digestion mix after incubation time.
</LI></UL><UL><LI> Preperative gelelectrophoresis was perforemd at 90 V for 1 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pSB1C3_CaMXMT1 (clone B1B_1)
</TD><TD>pSB1C3_CaMXMT1 (clone B3D_3)
</TD></TR></TBODY></TABLE><UL><LI> all bands was cut out and purified via Gelextraction
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Gel_extraction_of_separated_fragments">Gel extraction of separated fragments</SPAN></H3><P><B>Investigator</B>:Dennis
</P><UL><LI> Gelextraction of B1B_1, B3D_3 was performed according to Quiaquick gel extraction protocol 
</LI><LI> following concentrations were detected by Nanodrop: 
</LI><LI> B1B_1 ==&gt; 8,8 ng/µl
</LI><LI> B3D_3 ==&gt; 10,4 ng/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli"> Transformation of <I>E. coli</I></SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products of ADH1 Terminator+B1B_1, ADH1 terminator+ B3D_3.
</P><P><B>Procedure:</B></P><UL><LI> 20 µl of each ligation products were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_plasmid_prep_of_september_5th">Analytical restriction digest of plasmid prep of september 5th</SPAN></H3><P><B>Instructor:</B> Katrin
</P><P><B>PCR57/p133</B></P><P><A href="/File:TUM12_LS_analytgel0609.jpg" title="File:TUM12 LS analytgel0609.jpg">File:TUM12 LS analytgel0609.jpg</A></P><P><B>PCR42/p133</B></P><P><A href="/File:TUM12_LS_analytgel20609.jpg" title="File:TUM12 LS analytgel20609.jpg">File:TUM12 LS analytgel20609.jpg</A></P><P>--&gt; ligations failed
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Cell_lysis_of_5ml_samples_taken_20min_after_expression_induction"> Cell lysis of 5ml samples taken 20min after expression induction</SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Cell lysis of yeast cell- samples taken 20 min after expression. The idea is to compare the gene expression 20min after induction and 20h after induction. The signals on the western blott should show, that the two bands of our expressed enzymes are only visible after 20h.
</P><P><B>Operational sequence:</B></P><UL><LI> an adequate amount of breaking buffer (containing PMSF) was given to the frozen cell pellet, as well as an equal amount of glass beads. <B>Note:</B> because the supernatant could not be taken after 10 min centrifugation, another 30µl of breaking buffer were added.
</LI></UL><UL><LI> cell- walls were disrupted by vortexing for 30 sec, followed by 30 sek on ice. This procedure was repeated 4 times.
</LI></UL><UL><LI> protein concentration was determined by NanoDrop (blank: breaking buffer)
</LI></UL><UL><LI> CaXMT1 crude extract: 1 mg/ml
</LI><LI> CaMXMT1 crude extract: 2,23 mg/ml
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="SDS-_Page_of_protein_crude_extracts.2C_taken_at_different_times_after_induction_of_expression"> SDS- Page of protein crude extracts, taken at different times after induction of expression </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment is the preparation of a SDS- page, to make a subsequent western blot and to compare the samples, taken at different expression times.
</P><P><B>Operational sequence:</B></P><P>The SDS- gel was prepared as previously described (12%). 20µg protein crude extract of each sample were loaded on the gel. Running- time: ca. 2h at 120V.
Additionally to the prestained protein marker, we used an un- prestained marker, which contained a 45 kDa protein.
Afterwards, the gel was immediately used for the western blot.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Western_blot_of_samples_of_the_expression_of_CaXMT1.2C_CaMXMT1_and_eGFP">Western blot of samples of the expression of CaXMT1, CaMXMT1 and eGFP</SPAN></H3><P><B>Investigator:</B> Saskia, Roman
</P><P><B>Aim:</B> Analysis of the expression of CaXMT1, CaMXMT1 and eGFP (positive controll)after SDS-PAGE (120V, 1.5 h)
</P><P><B>Procedure:</B><B>preparation of solutions:</B> as described in the materials
</P><UL><LI>PBS-T0.1
</LI><LI>PBS-T0.1 with 3% BSA
</LI></UL><P><B>Blotting:</B></P><UL><LI>blotting of proteins on ImmobilonP Membrane (activating in methanol for 10 min)
</LI><LI>gel and membrane in transferbuffer for 20 min
</LI><LI>50mA, 1 h
</LI></UL><P><B>Blocking:</B></P><UL><LI>wash 3x7min with PBS-T0.1
</LI><LI>the membrane is blocked over night in PBS-T0.1 with 3% BSA at 4 °C on a shaking device
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Sequencing_of_structural_genes_of_Xanthohumol_including_the_promoter">Sequencing of structural genes of Xanthohumol including the promoter</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Check wheather the gene constructs have mutations in the promoters 
</P><P><B>Results:</B>
The following geneconstructs were sequenced including the promoter:  
P194 4Cl- in pYes
P263 APT in pYes
P197 OMT+ in pYes
P198 OMT- in pYes
P200 PAL- in pYes
P236 PAL+ in pYes
All of these sequences did not show any mutation in the promoter. Therefore the fact that our proteins were not detectable might be a result of errors during the transformation. Hence the transformation will be repeated. Parallely our gene constructs will be cloned into a pYes vector backbone of the Limonen group who could successfully express and detect their proteins. 
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_September_7th"><B>Friday, September 7th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Plasmidextraction_of_P451.2CP450.2C_P51_and_P403_2"> Plasmidextraction of P451,P450, P51 and P403</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI>Plasmid-DNA was extracted according to Quiaprep genextraction kit
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_P493.2BP639.2C_P643.2BP299.2C_P640.2BP642.2C_P641.2BP642"> Miniprep of <I>E. coli</I> XL-Blue cells, transformated with ligation products P493+P639, P643+P299, P640+P642, P641+P642 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of <I>E. coli</I> XL-Blue cells, transformated with ligation products P493+P639, P643+P299, P640+P642, P641+P642.
</P><P><B>Procedure:</B></P><P>`* Miniprep was done after manufacturer's protocol (P686-P697). (QIAGEN - QIAprep Spin Miniprep Kit)
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P686-P697"> Analytical digestion and gelelectrophoresis of P686-P697 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P686-P697 with XbaI+PstI-HF.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for digestion of P686-P697 with XbaI and PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>26 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2.6 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>3.25 µl
</TD><TD>XbaI (20 U/µl) (NEB)
</TD></TR><TR><TD>3.25 µl
</TD><TD>PstI (20 U/µl) (NEB)
</TD></TR><TR><TD>192.4 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=227.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix was added to 2.5 µl of plasmid DNA (P686-P697).
</LI></UL><UL><LI> Analytical digestion reaction mix was incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 2 µl of DNA loading buffer (10x) was added to the 20 µl of reaction mix after digestion.
</LI></UL><UL><LI> 10 µl of these were loaded in the gel pockets.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min on a 0.5% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P686
</TD><TD>P687
</TD><TD>P688
</TD><TD>P689
</TD><TD>P690
</TD><TD>P691
</TD></TR><TR><TD>Ligation was successful!
</TD><TD>Ligation was successful!
</TD><TD>Ligation was successful!
</TD><TD>Ligation was successful!
</TD><TD>Ligation was successful!
</TD><TD>Ligation was successful!
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P692
</TD><TD>P693
</TD><TD>P694
</TD><TD>P695
</TD><TD>P696
</TD><TD>P697
</TD></TR><TR><TD>Ligation was successful!
</TD><TD>Ligation was successful!
</TD><TD>Ligation was successful!
</TD><TD>Ligation was successful!
</TD><TD>Ligation was successful!
</TD><TD>Ligation was successful!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P687.2C_P690.2C_P693.2C_P697"> Preparative digestion of P687, P690, P693, P697 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P687 (SpeI-HF+PstI-HF), P690 (XbaI+PstI-HF), P693(XbaI+PstI-HF), P697(XbaI+PstI-HF).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for digestion of P687 with SpeI-HF+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P687
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>Spe-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P690 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P690
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P693 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P693
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P697 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P697
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch was incubated at 37 °C overnight.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_of_PCR1.2FpTUM104_.28P515neu.29_and_PCR58.2FpSB1C3_.28P510.29">Preparative restriction digest of PCR1/pTUM104 (P515neu) and PCR58/pSB1C3 (P510)</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Prepare PCR1 for ligation into pSB1C3
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>Plasmid
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>AgeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> plasmids were digested at 37 °C for 3 hours.
</LI></UL><P><B>Gelelectrophoresis:</B></P><P>Limonensynthase: 1600 bp
</P><P>pSB1C3: 2000 bp
</P><P>pYES: 5800 bp
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Single_colony_streak_of_PCR2_in_yeast">Single colony streak of PCR2 in yeast </SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Since we only had one colony of PCR2/pYES in yeast, we wanted to get more yeast colonies by picking a bit of this clone and smearing it onto a new SC-U plate.
The plate was incubated at 30°C over night.
</P><P><B>check plate tomorrow and store in fridge!</B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_E.Coli_over_night_cultures"> Miniprep of <I>E.Coli</I> over night cultures </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of experiment:</B></P><P>Isolation of plasmids: B2A_3+ADH1 terminator (1), B1B_2+ADH1 terminator (2), B3D_3+ADH1 (4) terminator for further usage.
</P><P><B>Operational sequence:</B></P><P>The plasmid preparation was done with the Quiagen Plasmid Miniprep Kit as previously described.
Elution was performed with 40µl of elution buffer, heated up to 37°C. Before final centrifugation, the column was incubated for 2 minutes at 37°C.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="analytic_digestion_of_B2A_3.2BADH1_terminator.2C_B1B_2.2BADH1_terminator.2C_B3D_3.2BADH1_terminator"> analytic digestion of B2A_3+ADH1 terminator, B1B_2+ADH1 terminator, B3D_3+ADH1 terminator </SPAN></H3><P><B>experimenter:</B> Dennis
</P><P><B>Aim of the experiment:</B> analytic digestion of B2A_3+ADH1 terminator, B1B_2+ADH1 terminator, B3D_3+ADH1 terminator
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>B2A_3+ADH1 terminator, B1B_2+ADH1 terminator, B3D_3+ADH1 terminator
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0,5  µl
</TD><TD>EcorI (20 U/µl)
</TD></TR><TR><TD>0,5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> gelelectrophoresis: B2A_3+ADH1 terminator (3 times), B1B_2+ADH1 terminator (3 times), B3D_3+ADH1 terminator (3 times)
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Staining_with_Ponceau.27s_reagent_and_western_blot_analysis_of_protein_crude_extract_of_CaXMT1.2C_CaMXMT1_and_eGFP_transformants_after_20_minutes_and_20_hours_of_expression"> Staining with Ponceau's reagent and western blot analysis of protein crude extract of CaXMT1, CaMXMT1 and eGFP transformants after 20 minutes and 20 hours of expression </SPAN></H3><P><B>Investigator:</B> Saskia, Roman
</P><P><B>Aim of the experiment:</B></P><P>Detection of expressed proteins CaXMT1, CaMXMT1, eGFP.
</P><P><B>Operational sequence:</B></P><P>The western blot membran (having been blocked over the night at 4°C) was washed with PBS- T0.1 four times (4x 15min). 
Because of the use of un- prestained protein marker (see Thursday, 6.9.), we stained the western blot membran with Ponceau's reagent and assigned the bands, followed by washing the membran with PBS- T0.1 again (15 minutes, 2x).
Afterwards, the membran was incubated for 1h with detection reagent.
Detection reagent:
</P><UL><LI> 10ml 1xPBS
</LI><LI> 0,2% BSA (0,02g have been weighted in)
</LI><LI> 2µl anti body MABclassic
</LI></UL><P>After incubating with the first antibody, the membran was washed again with PBS-T0.1 three times (3x 5min), followed by the incubation with the second antibody (anti mouse, fused with alk. phosphatase) for another our.
Second detection reagent:
</P><UL><LI> 10ml 1xPBS
</LI><LI> 0,2% BSA (see above)
</LI><LI> 5µl AntiMouse-alk. Phosphatase conjugate (1/2000 dilution)
</LI></UL><P>The development was performed after washing the membran with PBS- T0.1 for 10 min (two times) and with 1xPBS for 10 min (two times).
For this, the membran was shortly washed with AP buffer and then incubated with a solution containing 15ml AP buffer, 45µl BCIP (50mg/ml in DMF) and 7,5µl NBT (75mg/ml in 70% DMF) for a few minutes, until clear bands appeared.
</P><P>Western blot membran:
</P><P>From left to right (with expression time in brackets):
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Prestained protein marker (Page Ruler Plus)
</TD><TD> CaXMT1 crude extract (20min)
</TD><TD> CaMXMT1 crude extract (20min)
</TD><TD> eGFP(20h)
</TD><TD>CaMXMT1 crude extract (20h)
</TD><TD>CaXMT1 crude extract (20h)
</TD><TD>Unstained protein marker (pen marking)
</TD></TR></TBODY></TABLE><P><B>Picture with annotations:</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparing_of_controls_for_large-scale_expression">Preparing of controls for large-scale expression </SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Prepare controls for analyzing background protein expression in yeast.
</P><UL><LI>Control 1: S. cerevisiae without plasmid containing limonene synthase. 
</LI></UL><P>-&gt; growth in 50 ml of SC 
(--&gt; 0.005 g of uracil were put into 50 ml of SC-U (+Glucose) and filtered with a steril-filter)
</P><UL><LI>Control 2: Transformed yeast (containing PCR1/pYES) without induction of expression 
</LI></UL><P>-&gt; culture was transferred into fresh medium.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transfer_of_yeast_containing_PCR.2FpTUM104_into_induction_medium_.28.2BGal.29">Transfer of yeast containing PCR/pTUM104 into induction medium (+Gal) </SPAN></H3><P><B>Investigator:</B> Andrea, Lara
</P><P><B>Aim:</B> Cultures that were grown in 2 L of SC-U (+Glu) were transferred into 2 L of SC-U (+Gal). 
</P><UL><LI> First, the OD was measured: OD=4
</LI><LI> Approximately 300 ml of the culture was centrifuged and resuspended in induction medium
</LI><LI> transfer into 2 L of induction medium
</LI></UL></DIV><DIV class="Xanthohumol"><H3><SPAN class="mw-headline" id="Cloning_of_Xanthohumol_gene_constructs_in_pTUM104_vector_backbone_of_the_Limonen_group">Cloning of Xanthohumol gene constructs in pTUM104 vector backbone of the Limonen group</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Rule out negative effects of possible mutations in the vector backbone. 
</P><P><B>Operational sequence:</B></P><UL><LI> Reaction batch for digestion of P515(Limonen vector) and of Xanthohumol gene construts with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>DNA template
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 3 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>11.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch was incubated at 37 °C for 3 h
</LI></UL><UL><LI> 4.5 µl of DNA loading buffer (10x) was added to the digested product and preperative gelelectrophoresis was performed at 70 V for 1.5 h.
</LI></UL><P><A href="/File:TUM12_Xanthohumol_Prepgel_07.09.2012_P193_P194_P197_.jpg" title="File:TUM12 Xanthohumol Prepgel 07.09.2012 P193 P194 P197 .jpg">500px</A></P><P><A href="/File:TUM12_Xanthohumol_Prepgel_07.09.2012_P200_P236_.jpg" title="File:TUM12 Xanthohumol Prepgel 07.09.2012 P200 P236 .jpg">500px</A></P><P><A href="/File:TUM12_Xanthohumol_Prepgel_07.09.2012_P261_P428_P429.jpg" title="File:TUM12 Xanthohumol Prepgel 07.09.2012 P261 P428 P429.jpg">500px</A></P><UL><LI>The desired bonds of the genes were cut out of the gel and the DNA was extracted using a Quiagen gel extraction kit. Resulting DNA samples: 
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Label</B></TD><TD><B>Content</B></TD><TD><B>Concentration [ng/µl]</B></TD></TR><TR><TD>P755
</TD><TD>PAL+(P236)
</TD><TD>7.4
</TD></TR><TR><TD>P756
</TD><TD>PAL-(P200)
</TD><TD>6.9
</TD></TR><TR><TD>P757
</TD><TD>4CL+(P193)
</TD><TD>5.7
</TD></TR><TR><TD>P758
</TD><TD>4Cl-(P194)
</TD><TD>7.5
</TD></TR><TR><TD>P759
</TD><TD>CHS+(P428)
</TD><TD>53.5
</TD></TR><TR><TD>P760
</TD><TD>CHS-(P429)
</TD><TD>3.3
</TD></TR><TR><TD>P761
</TD><TD>OMT+(P197)
</TD><TD>3.9
</TD></TR><TR><TD>P762
</TD><TD>APT(P263)
</TD><TD>6.4
</TD></TR><TR><TD>P763
</TD><TD>pYes backbone of P515
</TD><TD>116.5
</TD></TR></TBODY></TABLE><UL><LI>Ligation batches:
</LI></UL><P>The component volumina were calculated to result in a ratio 1:6 vector to insert. The ligation process lasted overnight with a repeated sequence of 1 min at 16°C and 1 min at 22°C
</P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Insert</B></TD><TD align="center" style="background:#f0f0f0;"><B>Insertvolume [µl]</B></TD><TD align="center" style="background:#f0f0f0;"><B>Vectorvolume (P763) [µl]</B></TD><TD align="center" style="background:#f0f0f0;"><B>T4 DNA Ligase</B></TD><TD align="center" style="background:#f0f0f0;"><B>T4 DNA Ligase Buffer</B></TD><TD align="center" style="background:#f0f0f0;"><B>Water</B></TD><TD align="center" style="background:#f0f0f0;"><B>Sum</B></TD></TR><TR><TD> PAL+ (P755)</TD><TD>7,78</TD><TD>0,22</TD><TD>1</TD><TD>2</TD><TD>9</TD><TD>20
</TD></TR><TR><TD> PAL-(P756)</TD><TD>7,8</TD><TD>0,2</TD><TD>1</TD><TD>2</TD><TD>9</TD><TD>20
</TD></TR><TR><TD> 4Cl+(P757)</TD><TD>7,79</TD><TD>0,21</TD><TD>1</TD><TD>2</TD><TD>9</TD><TD>20
</TD></TR><TR><TD> 4Cl-(P758)</TD><TD>7,72</TD><TD>0,28</TD><TD>1</TD><TD>2</TD><TD>9</TD><TD>20
</TD></TR><TR><TD> CHS+(P759)</TD><TD>5,84</TD><TD>2,16</TD><TD>1</TD><TD>2</TD><TD>9</TD><TD>20
</TD></TR><TR><TD> CHS-(P760)</TD><TD>7,82</TD><TD>0,18</TD><TD>1</TD><TD>2</TD><TD>9</TD><TD>20
</TD></TR><TR><TD> APT(P761)</TD><TD>7,67</TD><TD>0,33</TD><TD>1</TD><TD>2</TD><TD>9</TD><TD>20
</TD></TR><TR><TD> OMT-(P762)</TD><TD>7,77</TD><TD>0,23</TD><TD>1</TD><TD>2</TD><TD>9</TD><TD>20
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="September.2C_Saturday_8th"><B>September, Saturday 8th</B></SPAN></H2><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Cloning_of_Xanthohumol_gene_constructs_in_pTUM104_vector_backbone_of_the_Limonen_group_2">Cloning of Xanthohumol gene constructs in pTUM104 vector backbone of the Limonen group</SPAN></H3><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 5 µl of the ligation batch from 07.09.2012
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 45 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_gelelectrophoresis_of_preparative_digestion_of_P687.2C_P690.2C_P693.2C_P697"> Preparative gelelectrophoresis of preparative digestion of P687, P690, P693, P697 </SPAN></H3><P><B>Investigator:</B> Jeff, Saskia
</P><P><B>Aim of the experiment:</B> Preparative gelelectrophoresis was performed from the samples from the preparative digestion overnight of P687 (SpeI-HF+PstI-HF), P690 (XbaI+PstI-HF), P693(XbaI+PstI-HF), P697(XbaI+PstI-HF).
</P><P><B>Procedure:</B></P><UL><LI> 4.44 µl of DNA loading buffer was added to the digestion mixture.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 70 V for 3 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P687 SpeI + PstI-HF
</TD><TD>1 kbp DNA ladder
</TD><TD>P690 XbaI-HF + PstI-HF
</TD></TR><TR><TD>Band was cut out (3316 bp, 75.5 ng/µl)
</TD><TD>Lower band was cut out (988 bp, 30.4 ng/µl)
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P693 XbaI-HF + PstI-HF
</TD><TD>1 kbp DNA ladder
</TD><TD>P697 XbaI-HF + PstI-HF
</TD></TR><TR><TD>Upper band was cut out (3848 bp, 68.7 ng/µl)
</TD><TD>Upper band was cut out (4013 bp, 69.8 ng/µl)
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_BBa_I712019.2C_BBa_E2020.2C_BBa_E2030"> Transformation of <I>E. coli</I> XL1-Blue with BBa_I712019, BBa_E2020, BBa_E2030 </SPAN></H3><P><B>Investigator:</B> Saskia 
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with BBa_I712019, BBa_E2020, BBa_E2030 (H10, 12D, 12B)
</P><P><B>Procedure:</B></P><UL><LI> 2µl of each ligation product were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cells were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets were resuspended in 100 µl of LB-medium and plated on: H10 Ampicillin plates, 12D Kanamycin plates, 12B Kanamycin plates
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P698.2BP699"> Cycled ligation of P698+P699 </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B> Cycled ligation of P698+P699 (TEF1 promoter - SV40NLS-Gal4AD-30aaLinker-Pif3(100NT) + TEF1 terminator - TEF1 promoter).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P698+P699
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.32 µl
</TD><TD>P698 (75.5 ng/µl, 3316 bp)
</TD></TR><TR><TD>1.19 µl
</TD><TD>P699 (30.4 ng/µl, 988 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>14.49 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batch and was labeled as P698 NK.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P676.2C_P73.2C_P81.2C_P134"> Analytical digestion and gelelectrophoresis of P676, P73, P81, P134 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P676 (pSB6A0 (BBa_K268000)), P73 (BBa_K207001 in pSB1A2), P81 (BBa_K207000 in pSB3K3), P134 (BBa_K165055 in BBa_J63009).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for digestion of P676 with EcoRI-HF and SpeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P676)
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=60 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Mastermix for digestion of P676 (pSB6A0 (BBa_K268000)), P73 (BBa_K207001 in pSB1A2), P81 (BBa_K207000 in pSB3K3), P134 (BBa_K165055 in BBa_J63009):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>8 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.8 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl) (NEB)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI (20 U/µl) (NEB)
</TD></TR><TR><TD>49.2 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=60 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> For digestion of P73, P81, P134 15 µl of the mastermix for XbaI+PstI-HF was added to 5 µl of plasmid DNA.
</LI></UL><UL><LI> Analytical digestion mixes were incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 2 µl of DNA loading buffer (10x) was added to each reaction batch.
</LI></UL><UL><LI> 10 µl of each sample was loaded into the gel pocket.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 1 h
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P676(pSB6A0 (BBa_K268000))
</TD><TD>P73 (BBa_K207001 in pSB1A2)
</TD><TD>P81 (BBa_K207000 in pSB3K3)
</TD><TD>P134 (BBa_K165055 in BBa_J63009)
</TD></TR><TR><TD>Bands like expected!
</TD><TD>Corrupt!
</TD><TD>Corrupt!
</TD><TD>Corrupt!
</TD></TR></TBODY></TABLE><UL><LI> Conclusion for P676 (=pSB6A0 (BBa_K268000)): This part contains at least one forbidden restriction site (XbaI or PstI, please see analytical gel from September, the 6th. But here, one can see that EcoRI and SpeI can nevertheless be used for cloning parts into pSB6A0
</LI></UL><UL><LI> The other parts (P73 (BBa_K207001 in pSB1A2), P81 (BBa_K207000 in pSB3K3), P134 (BBa_K165055 in BBa_J63009)) are all completely wrong! Wrong size of bands and partially also the vector!
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_citrus_limonene_synthase_into_pSB1C3_containing_TEF1.2FTEF2_promoter">Ligation of citrus limonene synthase into pSB1C3 containing TEF1/TEF2 promoter </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Ligate citrus limonene synthase into pSB1C3 behind TEF2 and TEF1 promoter.
</P><P><B>Procedure</B></P><P><B>Citrus LS (p578) in pSB1C3 with TEF2 (p492)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P492 (TEF2/pSB1C3; digested with Spe+Pst)
</TD><TD>1.52 µl
</TD></TR><TR><TD>P578 (LS, digested with Xba+Pst)
</TD><TD>6.48 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>Negative control water in P492</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P492 (digested with Xba1+Age1)
</TD><TD>1.52 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>6.488 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>Citrus LS (p578) in pSB1C3 with TEF1 (p493)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P493 (TEF2/pSB1C3; digested with Spe+Pst)
</TD><TD>2.05 µl
</TD></TR><TR><TD>P578 (LS, digested with Xba+Pst)
</TD><TD>5.95 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>Negative control water in P493</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P493 (digested with Xba1+Age1)
</TD><TD>2.05 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>5.958 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>Citrus LS (PCR1, restriction digest on 7th September) in pSB1C3 (restriction digest 7th September)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>pSB1C3 (digested with Xba1+Age1)
</TD><TD>0.71 µl
</TD></TR><TR><TD>LS (PCR1 digested with Xba1+Age1)
</TD><TD>7.298 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>Negative control water in pSB1C3 (restriction digest 7th September)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>pSB1C3 (digested with Xba1+Age1)
</TD><TD>0.71 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>7.298 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>ligation reactions are stored in a 50 ml falcon at the lowest drawer of -20°C.</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Large-scale_cell_lysation">Large-scale cell lysation </SPAN></H3><P><B>Investigator:</B> Volker, Lara
</P><P><B>Aim:</B> Production of cell lysate (large-scale).
</P><P>
1. OD of cell cultures were measured:
</P><UL><LI> clone 1: OD600=4.2
</LI><LI> clone 2: OD600=2.8
</LI><LI> control: clone 2 (uninduced) OD600=5,6 (medium: SC+Glu)
</LI><LI> control: S. cer (not transformed) OD600=4,6 (medium:SC+uracil)
</LI></UL><P>
2. Cell cultures were centrifuged (large scale:25 min at 3000 rpm; controls: 10 min at 3000rpm) and the pellet was washed in breaking buffer (without PMSF).
</P><UL><LI>200 ml BB for large-scale cultures
</LI><LI>50 ml BB for controls
</LI></UL><P>
3. After centrifugation, pellets were resuspended in breaking buffer (with PMSF) to reach an OD of 70-100.
</P><UL><LI> large-scale cultures(2L): 80 ml BB+PMSF
</LI><LI> controls (50ml): 2,5 ml BB+PMSF
</LI></UL><P>
4. 1 volume of glass beads (0,5 mm) was added. In cool room: 20-40 cycles of vortexing (30 sec) and incubation (30 sec).
</P><P>
5. the supernatant was transferred to fresh tubes.
</P><P>Samples were stored at the lowest drawer of -20°C in a 50 ml falcon tube.
</P><P><B>Suggestion for next time:</B> Take a transformed yeast (induced) in small-scale as third control.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Production_of_Breaking_buffer">Production of Breaking buffer </SPAN></H3><P><B>Investigator:</B> Lara, Andrea
</P><P><B>For 1 L of 1 M sodium phosphate buffer:</B></P><P>prepare:
</P><UL><LI> 127,8 g Na2HPO4; add 900 ml of distilled water -&gt; solution 1
</LI><LI> 24 g of NaH2PO4; add 200 ml of distilled water -&gt; solution 2
</LI></UL><UL><LI> mix 845 ml of solution 1 with 155 ml of solution 2 (you can measure the amount by weight scale)
</LI><LI> adjust pH to pH=7.5 by adding more of solution 1 or solution 2 
</LI><LI> autoclave and store at room temperature
</LI></UL><P><B>For PMSF solution:</B></P><UL><LI> has to be made each time, can not be stored due to short half time
</LI><LI> 0,0522 g PMSF; add 3 ml isopropanol
</LI></UL><P><B>For breaking buffer with PMSF:</B></P><P>50 mM sodium phosphate buffer
1 mM EDTA
5 % glycerol
1 mM PMSF
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_2"> Transformation of <I>E. coli</I></SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with the ligation products Lara 1,2,3 and NK492, NK493, NKpSB1C3
</P><P><B>Procedure:</B></P><UL><LI> 5µl of each ligation product were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest was centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on chloramphenicol plates.
</LI></UL><UL><LI>incubation over night (37°C)
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Cloning_of_Xanthohumol_gene_constructs_in_pTUM104_vector_backbone_of_the_Limonen_group_continued">Cloning of Xanthohumol gene constructs in pTUM104 vector backbone of the Limonen group continued</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><UL><LI>From each gene construct transformed in XL1 blue cells on saturday 08th september 2012 a single clone was picked an transferred into 7 ml fresh LB-media containing 1:1000 ampicilin(as the number of colonies of CHS+/- was low a few clones were picked). Incubation overnight at 37°C and 180 rpm.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_September_9th"><B>Sunday, September 9th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_P698.2BP699_and_biobricks_BBa_J52008.2C_BBa_E0020.2C_BBa_E2060.2C_BBa_E0026.2C_BBa_E0036.2C_BBa_E0040.2C_BBa_E1010"> Transformation of <I>E. coli</I> XL1-Blue with ligation products P698+P699 and biobricks BBa_J52008, BBa_E0020, BBa_E2060, BBa_E0026, BBa_E0036, BBa_E0040, BBa_E1010 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P698+P699 and biobricks BBa_J52008 in pSB1AK3, BBa_E0020 in pSB1A2, BBa_E2060 in pSB2K3, BBa_E0026 in pSB1A2, BBa_E0036 in pSB1A2, BBa_E0040 in pSB1A2, BBa_E1010 in pSB2K3.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation product (P698+P699) and it's negative control (P698 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice. 2 µl of the biobricks BBa_J52008 in pSB1AK3, BBa_E0020 in pSB1A2, BBa_E2060 in pSB2K3, BBa_E0026 in pSB1A2, BBa_E0036 in pSB1A2, BBa_E0040 in pSB1A2, BBa_E1010 in pSB2K3 were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension (only ligation products!) were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest (ligation products and biobricks) were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and these concentrated cell suspensions was plated on suitable antibiotic plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P698.2BP699_2"> Cycled ligation of P698+P699 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Cycled ligation of P698+P699 (TEF1 promoter - SV40NLS-Gal4AD-30aaLinker-Pif3(100NT) + TEF1 terminator - TEF1 promoter).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P698+P699
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.32 µl
</TD><TD>P698 (75.5 ng/µl, 3316 bp)
</TD></TR><TR><TD>2.92 µl
</TD><TD>P699 (30.4 ng/µl, 988 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12.76 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batch and was labeled as P698 NK.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Cloning_of_Xanthohumol_gene_constructs_in_pTUM104_vector_backbone_of_the_Limonen_group_continued_2"> Cloning of Xanthohumol gene constructs in pTUM104 vector backbone of the Limonen group continued </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Get purified DNA for transformation in yeast and check wheather the inserts are correct.
</P><P><B>Procedure:</B></P><UL><LI> Using a Quiagen kit a miniprep of the overnight culture was done.
</LI><LI> The resulting purified DNA was aliquoted in new tubes labeled as follows:
</LI></UL><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Plasmid number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>Concentration</B></TD></TR><TR><TD> P739</TD><TD>PAL+ (P236) in Limonen pYes(P515)</TD><TD>220,9 ng/µl
</TD></TR><TR><TD> P740</TD><TD>PAL- (P200) in Limonen pYes(P515)</TD><TD>133,6 ng/µl
</TD></TR><TR><TD> P741</TD><TD>4Cl+ (P193)in Limonen pYes (P515)</TD><TD>312,7 ng/µl
</TD></TR><TR><TD> P742</TD><TD>4Cl- (P194)in Limonen pYes(P515)</TD><TD>89,4 ng/µl
</TD></TR><TR><TD> P743</TD><TD>CHS+ (P428)in Limonen pYes(P515)</TD><TD>204,7 ng/µl
</TD></TR><TR><TD> P744</TD><TD>CHS in Limonen pYes</TD><TD>53,1 ng/µl
</TD></TR><TR><TD> P745</TD><TD>CHS- (P429)in Limonen pYes(P515)</TD><TD>65,8 ng/µl
</TD></TR><TR><TD> P746</TD><TD>CHS- (P429)in Limonen pYes(P515)</TD><TD>531,8 ng/µl
</TD></TR><TR><TD> P747</TD><TD>CHS- (P429)in Limonen pYes(P515)</TD><TD>53,6 ng/µl
</TD></TR><TR><TD> P748</TD><TD>OMT- (P198)in Limonen pYes(P515)</TD><TD>250,4 ng/µl
</TD></TR><TR><TD> P749</TD><TD>APT (P263)in Limonen pYes(P515)</TD><TD>97,1 ng/µl
</TD></TR></TBODY></TABLE><UL><LI>Afterwards a control digestion of P739-P749 was done.
</LI></UL><P><B>Reaction batch</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Plasmid DNA
</TD><TD>2.5 µl
</TD></TR><TR><TD>NEB3 buffer
</TD><TD>2 µl
</TD></TR><TR><TD>PstI-HF
</TD><TD>0.25 µl
</TD></TR><TR><TD>XbaI
</TD><TD>0.25 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>15 µl
</TD></TR><TR><TD>Sum
</TD><TD>20 µl
</TD></TR></TBODY></TABLE><UL><LI>Incubation at 37 °C for 1h.
</LI><LI>Verification of control digest by agarose gel electrophoresis: 
</LI></UL><P>20 µl of each digest product was mixed with 2 µl 10x DNA loading buffer and loaded into the gel. The separation process lasted 1h at 90 V.
</P><P><A href="/File:TUM12_Xanthohumol_Strukturenzyme_in_pYes_Limonen_Kontrollverdau_beschriftet_1.jpg" title="File:TUM12 Xanthohumol Strukturenzyme in pYes Limonen Kontrollverdau beschriftet 1.jpg">Gel picture of control digest genes in limonene pYes 1</A></P><P>
The ligation of all batches except P 743 was successfull as the bonds appear at the expected length. 
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_mutations_found_by_sequencing_in_our_gene_constructs">  Quick Change mutagenesis to remove mutations found by sequencing in our gene constructs  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P><B>Aim of the experiment: </B>Deletion of found mutations.
</P><P><B>Operational sequence</B></P><P><B>1. PCR general setup</B><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of appropriate Oligo (= 5 pmol)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of appropriate Oligo (= 5 pmol)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI></UL><P><B>2. Used constructs and primers</B></P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Quickchange number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Plasmid number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Geneconstruct</B></TD><TD align="center" style="background:#f0f0f0;"><B>Primer number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Primer</B></TD></TR><TR><TD> QC 1</TD><TD>P428</TD><TD>CHS+ pYes</TD><TD>O104 &amp; O105</TD><TD>c614t
</TD></TR><TR><TD> QC 2</TD><TD>P429</TD><TD>CHS- pYes</TD><TD>O104 &amp; O105</TD><TD>c614t
</TD></TR><TR><TD> QC 3</TD><TD>P430</TD><TD>CHS+ pSB1C3</TD><TD>O104 &amp; O105</TD><TD>c614t
</TD></TR><TR><TD> QC 4</TD><TD>P431</TD><TD>CHS- pSB1C3</TD><TD>O104 &amp; O105</TD><TD>c614t
</TD></TR><TR><TD> QC 5</TD><TD>P197</TD><TD>OMT+ pYes</TD><TD>O96 &amp; O97</TD><TD>a1055g
</TD></TR><TR><TD> QC 6</TD><TD>P198</TD><TD>OMT- pYes</TD><TD>O96 &amp; O97</TD><TD>a1055g
</TD></TR><TR><TD> QC 7</TD><TD>P236</TD><TD>PAL+ pYes</TD><TD>O108 &amp; O109</TD><TD>a770g
</TD></TR><TR><TD> QC 8</TD><TD>P200</TD><TD>PAL- pYes</TD><TD>O108 &amp; O109</TD><TD>a770g
</TD></TR><TR><TD> QC 9</TD><TD>P188</TD><TD>PAL- pSB1C3</TD><TD>O108 &amp; O109</TD><TD>a770g
</TD></TR><TR><TD> QC 10</TD><TD>P187</TD><TD>PAL+ pSB1C3</TD><TD>O108 &amp; O109</TD><TD>a770g
</TD></TR></TBODY></TABLE><UL><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_14">Week 14</SPAN></H1><DIV class="week" id="WWeek_14"><H2><SPAN class="mw-headline" id="Monday.2C_September_10th"><B>Monday, September 10th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_Xl1-blue_with_ligation_of_P578_and_P679.2CP680">Transformation of <I>E.coli</I> Xl1-blue with ligation of P578 and P679,P680</SPAN></H3><P><B>Investigator: Georg</B><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on ampicillin plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new ampicillin plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_TEF2-P_clone_P451_.28xbaI_and_PSTI-HF.2C_SpeI-HF_PstI-HF.29_and_ADH1-T_.28XbaI_PstI-HF.2C_SpeI-HF_and_PstI-HF"> Gelextraction of TEF2-P clone P451 (xbaI and PSTI-HF, SpeI-HF PstI-HF) and ADH1-T (XbaI PstI-HF, SpeI-HF and PstI-HF</SPAN></H3><P><B>Investigator:Georg</B></P><UL><LI> Gelextraction was done according to Quiaquick gelextraction protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_measurement_of_P707.2C_P708.2C_P709_and_P710">Nanodrop measurement of P707, P708, P709 and P710</SPAN></H3><P><B> Investigator:Georg</B></P><UL><LI> P707: 9,9 ng/µl
</LI><LI> P708: 19,6 ng/µl
</LI><LI> P709: 55,7 ng/µl
</LI><LI> P710: 61,6 ng/µl
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_TEF2-P_.28clone_P451.29_with_pTUM100_and_of_P578_with_P710">Ligation of TEF2-P (clone P451) with pTUM100 and of P578 with P710</SPAN></H3><P><B>Investigator: Georg</B></P><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>2,48 µl
</TD><TD>P578
</TD></TR><TR><TD>1,62 µl
</TD><TD>P710
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>12,9 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,17 µl
</TD><TD>P572
</TD></TR><TR><TD>2,08 µl
</TD><TD>P708
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>11,75 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> From each ligation, negative controls were made with ddH20 instead of the insert
</LI><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_colonies_from_P638.2C_BBa_J52008.2C_BBa_E2060.2C_BBa_E0036.2C_BBa_E1010"> Picking colonies from P638, BBa_J52008, BBa_E2060, BBa_E0036, BBa_E1010</SPAN></H3><P><B> Investigator: Georg</B></P><UL><LI> Colonies were picked and transferred into 5 ml LB-medium with 1x CAM (P638), Kan (BBa_J52008,BBa_E2060), Amp (BBa_E0036)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_P698.2BP699"> Transformation of <I>E. coli</I> XL1-Blue with ligation product P698+P699 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation product (P698+P699) and it's negative control (P698 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest of the suspension was centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and was plated on chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Electroporation_of_electrocompetent_E..C2.A0coli_with_P698.2B699"> Electroporation of electrocompetent <I>E. coli</I> with P698+699 </SPAN></H3><P><B>Investigator:</B> Jeff, Adam
</P><P><B>Aim of the experiment:</B> Electroporation of electrocompetent <I>E. coli</I> with P698+699
</P><P><B>Procedure:</B></P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P676"> Preparative digestion and gelelectrophoresis of P676 </SPAN></H3><P><B>Investigator:</B> Alois, Martin
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P676.
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for digestion of P687 with SpeI-HF+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P676
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Preparative digestion mix was incubated for 2 h at 37 °C.
</LI></UL><UL><LI> After incubation ~4-5 µl of DNA loading buffer (10x) was added to reaction mix.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed for xxx h at 70 V
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Part from other subproject
</TD><TD>1 kbp DNA ladder
</TD><TD>Part from other subproject
</TD><TD>P676 EcoRI-HF + SpeI-HF
</TD></TR><TR><TD>Band was cut out (4791 bp, 76.1 ng/µl)
</TD></TR></TBODY></TABLE><UL><LI> Gel purification after manufacturer's protocol (Qiagen) after cutting out the DNA band.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL1-Blue_transformated_with_biobricks_BBa_J52008.2C_BBa_E2060.2C_BBa_E0036.2C_BBa_E1010"> Picking of <I>E. coli</I> XL1-Blue transformated with biobricks BBa_J52008, BBa_E2060, BBa_E0036, BBa_E1010 </SPAN></H3><P><B>Investigator:</B> Georg
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL1-Blue transformated with biobricks BBa_J52008 in pSB1AK3, BBa_E2060 in pSB2K3, BBa_E0036 in pSB1A2, BBa_E1010 in pSB2K3.
</P><P><B>Procedure:</B></P><UL><LI> 4 colonies of each transformation were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and suitable antibiotics. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P686.2C_P691.2C_P692.2C_P696"> Preparative digestion of P686, P691, P692, P696 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P686 (SpeI-HF+PstI-HF), P691 (XbaI+PstI-HF), P692(XbaI+PstI-HF), P696(XbaI+PstI-HF).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for digestion of P686 with SpeI-HF+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P686
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>Spe-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P691 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P691
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P692 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P692
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P696 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P696
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch was incubated at 37 °C overnight.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Induction_of_expression_of_CaDXMT1"> Induction of expression of CaDXMT1 </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the start of the expression of CaDXMT1. This is the third and last gene, neccessary for caffeine- synthesis (previous transformation experiments did not work).
</P><P><B>Operational sequence:</B></P><P>First of all, the OD600 of the over night culture was determined: 4,42 (0,442 at 1/10 dilution). Next, 200ml of SC-U 2% galactose medium were inoculated with an adequate amount of the over night culture (has grown in SC-U 2% glucose). 
<B>Note:</B> Not the whole amount of the calculated over night culture was centrifuged down. Thus, about 12 ml of glucose medium were given to the galactose medium. In theory, the expression will start a little bit later, due to the repression of the Gal1 promoter (glucose mediated).
</P><P><B>Sampling:</B></P><UL><LI> 15 min after induction (12:00 am) (OD600: 0,442)
</LI><LI> xx min after induction
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Picking_of_colonies_for_miniprep_3"> Picking of colonies for miniprep </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> New miniprep for new transformation of E. coli with ligation batch B2A_3+ADH1 terminator and B1B_2+ ADH1 terminator and B3D_3+ ADH1 terminator
</P><UL><LI> Picking of 3 clones each of  ligation batch
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_3"> Transformation of <I>E. coli</I></SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products of ADH1 Terminator+B1B_1, ADH1 terminator+ B3D_3.
</P><P><B>Procedure:</B></P><UL><LI> 20 µl of each ligation products were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Ligation_of_ADH1.2BB1B_1.2C_ADH1.2BB3D_3"> Ligation of ADH1+B1B_1, ADH1+B3D_3 </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> ligation was performed for B1B_1 and B3D_3
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for ADH1+B1B_1
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,66  µl
</TD><TD>ADH1 (16,6 ng/µl, 2048 bp)
</TD></TR><TR><TD>1,8  µl
</TD><TD>B1B_1 (91,4  ng/µl, 825 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12,5 ddH2O
</TD><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> the ligation has been performed at 16 °C
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Plating_of_transformed_yeast_colony_.28with_pTUM104_CaDXMT1.29"> Plating of transformed yeast colony (with pTUM104_CaDXMT1) </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim:</B> Backup of the transformant for further usage
</P><P><B>Operational sequence:</B></P><P>150 µl were plated on SC-U agar plates (2% glucose) and incubated at 30°C for a few days.
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Preparative_digest_of_p667_.28pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term.29_and_ligation_with_p649_.28integration_vector_digested_with_EcoRI_and_SpeI.29_.2B_Trafo"> Preparative digest of p667 (pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term) and ligation with p649 (integration vector digested with EcoRI and SpeI) + Trafo </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim of the experiment:</B> Getting integration_vector_TEF1Prom_Preprothaumatin_TEF1Term
</P><P>Preparative digest:
</P><UL><LI> 30 µl p667, 1 µl EcoRI, 1 µl SpeI, 0.5 µl BSA, 5 µl NEBuffer 4, 12.5 µl ddH2O; 37°C, 2 h.
</LI></UL><P>Preparative gel electrophoresis: 1 kb, p667, (p468), (Jeff).
</P><UL><LI> Expected: backbone: 2.2 kb, insert: 1.7 kb. <B>Successful</B>.
</LI></UL><P>Gel extraction: 
</P><UL><LI> pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term digested with EcoRI and SpeI = p735.
</LI></UL><P>Ligation of p735 and p649 (=&gt; p737)
</P><UL><LI> 1 µl T4 DNA ligase, 2 µl T4 DNA ligase buffer, 10 µl p649, 8.5 µl p735; 45 min RT.
</LI></UL><P>Transformation of p737 integration_vector_TEF1Prom_Preprothaumatin_TEF1Term (=&gt; Amp).
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Preparative_digest_of_p468_.28.22pSB1C3_backbone.22.29_and_ligation_with_p735_.28.22TEF1Prom_Preprothaumatin_TEF1Term.22.29_.2B_Trafo"> Preparative digest of p468 (&quot;pSB1C3_backbone&quot;) and ligation with p735 (&quot;TEF1Prom_Preprothaumatin_TEF1Term&quot;) + Trafo </SPAN></H3><P><B>Investigator:</B> Martin, Alois, Fabiano Celentano
</P><P><B>Aim of the experiment:</B> Getting pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term with functional NotI
</P><P>Preparative digest:
</P><UL><LI> 20 µl p468, 1 µl EcoRI, 1 µl SpeI, 0.5 µl BSA, 5 µl NEBuffer 4, 22.5 µl ddH2O; 37°C, 2 h.
</LI></UL><P>Preparative gel electrophoresis: 1 kb, (p667), p468, (Jeff).
</P><UL><LI> Expected: backbone: &gt; 2.2, insert: 2.2 kb (&quot;pSB1C3_backbone&quot;). <B>Successful</B>.
</LI></UL><P>Gel extraction: 
</P><UL><LI> pSB1C3_backbone digested with EcoRI and SpeI = p736.
</LI></UL><P>Ligation of p736 and p735 (=&gt; p738)
</P><UL><LI> 1 µl T4 DNA ligase, 2 µl T4 DNA ligase buffer, 1.3 µl p736, 13.5 µl p735, 2 µl ddH2O; 45 min RT.
</LI></UL><P>Transformation of p738 = pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term with functional NotI (Cam).
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Western_blot_of_samples_of_the_expressed_Limonenesynthase_.28harvested_on_5th_September_and_on_8th_September.29">Western blot of samples of the expressed Limonenesynthase (harvested on 5th September and on 8th September)</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Analysis of the expression of Limonenesynthase
</P><P><B>Procedure:</B></P><P><B>SDS-PAGE</B></P><UL><LI>order: I am not sure about the order, but it is noted (sheet in labjournal) --&gt; the marker is on the right side !!
</LI><LI>120V, 2h
</LI></UL><P><B>preparation of solutions:</B> as described in the materials
</P><UL><LI>PBS-T0.1
</LI><LI>PBS-T0.1 with 3% BSA
</LI></UL><P><B>Blotting:</B></P><UL><LI>blotting of proteins on Nitrocellulose Membrane
</LI><LI>gel and membrane in transferbuffer for 10 min
</LI><LI>50mA, 1.25 h
</LI></UL><P><B>Blocking:</B></P><UL><LI>wash 3x5min with PBS-T0.1
</LI><LI>the membrane is blocked over night in PBS-T0.1 with 3% BSA at 4 °C on a shaking device
</LI></UL></DIV><DIV class="Xanthohumol"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue"> Transformation of <I>E. coli</I> XL1-Blue  </SPAN></H3><P><B>Investigator:</B> Ingmar, Daniela
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with quickchange products and plasmids for stemm preservation.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI><LI> 5 µl of the quickchange products and 1 µl of the miniprep products  were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI><LI> Used DNA:
</LI><LI> P430 CHS+ in pSb1C3
</LI><LI> P431 CHS- in pSB1C3
</LI><LI> P593 4Cl+ in pSB1C3 after sdm 24.08.2012
</LI><LI> P594 4Cl- in pSB1C3 after sdm 24.08.2012
</LI><LI> P595 4Cl+ in pYes after sdm 24.08.2012
</LI><LI> P596 4Cl- in pYes after sdm 24.08.2012
</LI><LI> P209 APT in pSb1C3
</LI><LI> P137 OMT+ in pSB1C3
</LI><LI> Quickchange products: 
</LI><LI> P428 CHS+ in pYes sdm with c614t
</LI><LI> P429 CHS- in pYes sdm with c614t
</LI><LI> P430 CHS+ in pSb1C3 sdm with c614t
</LI><LI> P431 CHS- in pSb1C3 sdm with c614t
</LI><LI> P197 OMT+ in pYes sdm with a1055g
</LI><LI> P198 OMT- in pYes sdm with a1055g
</LI><LI> P236 PAL+ in pYes sdm with a770g
</LI><LI> P200 PAL- in pYes sdm with a770g
</LI><LI> P188 PAL- in pSb1C3 sdm with a770g
</LI><LI> P187 PAL+ in pSb1C3 sdm with a770g
</LI></UL><LI> 30 min incubation on ice.
</LI><LI> 5 min heat shock at 37 °C
</LI><LI> Adding of 1 ml LB-medium to each tube.
</LI><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI><LI> 100 µl of those cell suspension were plated on LB plates containing the appropriate antibiotics.
</LI><LI> The rest of the suspension was centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI><LI> The pellet was resuspended in 100 µl of LB-medium and was plated on LB plates containing the appropriate antibiotics.
</LI><LI> Incubation at 37 °C overnight.
</LI></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_September_11th"><B>Tuesday, September 11th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Tranformation_of_E.coli_Xl1-blue_with_ligation_product_from_07.9.2012">Tranformation of <I>E.coli</I> Xl1-blue with ligation product from 07.9.2012</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Last time, heat shock was forgotten and plates didn't show any transformants
</LI><LI> Therefore the experiment was repeated
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Plasmid-extraction_of_picked_colonies_from_10.9.2012"> Plasmid-extraction of picked colonies from 10.9.2012</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Plasmid-DNA was extracted according to Quiaprep-genextraction protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_minipreped_BBa_J52008.2C_BBa_E2060.2C_BBa_E1010.2C_BBa_E0036"> Analytical digestion of minipreped BBa_J52008, BBa_E2060, BBa_E1010, BBa_E0036</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2,75 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>2,75 µl
</TD><TD>EcorI (NEB)
</TD></TR><TR><TD>22 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>2,2 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>166 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=192,5µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> to 2,5 ml of DNA 17,5 µl reaction batch were added
</LI><LI> Digestion took place at 37°C for 1 h
</LI></UL><UL><LI> only BBa_J52008 and BBa_E1010 showed positive inserts
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_pTUM100_.283.29.2C_P638_and_psb1c3-TEF1-P">Preparative digestion of pTUM100 (3), P638 and psb1c3-TEF1-P</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>4 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,4 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>13,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 20 µl pTUM104 were added
</LI></UL><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF(NEB)
</TD></TR><TR><TD>1 µl
</TD><TD>EcorI-HF (NEB)
</TD></TR><TR><TD>4 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,4 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>13,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 20 µl of pTUM104 (3) were added
</LI><LI> Digestion took place at 37°C for 3 h
</LI></UL><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>EcorI-HF (NEB)
</TD></TR><TR><TD>5,5 µl
</TD><TD>SpeI-HF (NEB)
</TD></TR><TR><TD>4 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,4 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>13,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> to 20 µl of reaction batch 20 µl of psb1c3-TEF1-P were added
</LI><LI> Digestion took place at 37°C for 3 h
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_gelelectrophoresis_of_picked_BBa_J52008.2C_BBa_E2060.2C_BBa_E1010.2C_BBa_E0036"> Analytical gelelectrophoresis of picked BBa_J52008, BBa_E2060, BBa_E1010, BBa_E0036 </SPAN></H3><P><B> Investigator: Georg</B></P><UL><LI> DNA was applicated onto a 1% analytical agarose Gel
</LI><LI> Analytical gelelectrophoresis was performed 1 h at 90 V
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_BBa_J52008.2C_BBa_E1010_and_psb1c3-Limonensynth."> Preparative digestion of BBa_J52008, BBa_E1010 and psb1c3-Limonensynth.</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12 µl
</TD><TD>PstI-HF (Ferm)
</TD></TR><TR><TD>6 µl
</TD><TD>XbaI (Ferm)
</TD></TR><TR><TD>12 µl
</TD><TD>Tango-buffer
</TD></TR><TR><TD>40 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=80µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 20 µl reaction batch, 20 ml of BBa_J52008,TEF2-pTUM104 or BBa_E1010 were added
</LI><LI> Digestion was performed over night at 37 C
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_gelelectrophoresis_of_preparative_digestion_of_P686.2C_P691.2C_P692.2C_P696"> Preparative gelelectrophoresis of preparative digestion of P686, P691, P692, P696 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative gelelectrophoresis of preparative digestion of P686, P691, P692, P696.
</P><P><B>Procedure:</B></P><UL><LI> 4.44 µl of DNA loading buffer was added to the digestion mixture.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 70 V for 2-3 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P686 SpeI + PstI-HF
</TD><TD>1 kbp DNA ladder
</TD><TD>P691 XbaI-HF + PstI-HF
</TD></TR><TR><TD>Band was cut out (3316 bp, 80.3 ng/µl)
</TD><TD>Lower band was cut out (988 bp, 34.3 ng/µl)
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P692 XbaI-HF + PstI-HF
</TD><TD>1 kbp DNA ladder
</TD><TD>P696 XbaI-HF + PstI-HF
</TD></TR><TR><TD>Upper band was cut out (3848 bp, 68.7 ng/µl)
</TD><TD>Upper band was cut out (4013 bp, 69.8 ng/µl)
</TD></TR></TBODY></TABLE><UL><LI> Gel extraction was performed with gel extraction kit from Qiagen after manufacturer's protoco..
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P751.2BP752"> Cycled ligation of P751+P752 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Cycled ligation of P751+P752.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P751+P752
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.25 µl
</TD><TD>P751 (80.3 ng/µl, 3316 bp)
</TD></TR><TR><TD>2.61 µl
</TD><TD>P752 (34.3 ng/µl, 988 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>13.14 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batch and was labeled as P751 NK.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_product_P751.2BP752"> Transformation of <I>E. coli</I> XL1-Blue with ligation product P751+P752 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation product P751+P752.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation product (P751+P752) and it's negative control (P751 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest of the suspension was centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and was plated on chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Picking_of_clones_of_tranformations_done_on_monday_10th_september">Picking of clones of tranformations done on monday 10th september</SPAN></H3><P><B>Investigator:</B> Ingmar
</P><UL><LI>From each gene construct transformed in XL1 blue cells on monday1 8th september 2012 a single clone was picked an transferred into 7 ml fresh LB-media containing 1:1000 of the apropriate antibiotics. Incubation overday at 37°C and 180 rpm.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_of_clones_of_tranformations_done_on_monday_10th_september"> Miniprep of clones of tranformations done on monday 10th september </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Get purified DNA for transformation in yeast and stem preservation.
</P><P><B>Procedure:</B></P><UL><LI> Using a Quiagen kit a miniprep of the overday culture was done.
</LI><LI> The resulting purified DNA was aliquoted in new tubes labeled as follows:
</LI></UL><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Plasmid number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>Concentration</B></TD></TR><TR><TD> P766</TD><TD>P428 CHS+ in pYes nach sdm c614t</TD><TD>364,7 ng/µl
</TD></TR><TR><TD> P767</TD><TD>P429 CHS- in pYes nach sdm c614t</TD><TD>42,7 ng/µl
</TD></TR><TR><TD> P768</TD><TD>P430 CHS+ in pSB1C3 nach sdm c614t</TD><TD>158,5ng/µl
</TD></TR><TR><TD> P769</TD><TD>P431 CHS- in pSB1C3 nach sdm c614t</TD><TD>336,2 ng/µl
</TD></TR><TR><TD> P770</TD><TD>P197 OMT+ in pYes sdm with a1055g</TD><TD>146 ng/µl
</TD></TR><TR><TD> P771</TD><TD>P198 OMT- in pYes sdm with a1055g</TD><TD>114,3 ng/µl
</TD></TR><TR><TD> P772</TD><TD>P236 PAL+ in pYes sdm with a770g</TD><TD>395,6 ng/µl
</TD></TR><TR><TD> P773</TD><TD>P200 PAL- in pYes sdm with a770g</TD></TR><TR><TD> P774</TD><TD>P188 PAL- in pSb1C3 sdm with a770g</TD><TD>227,4 ng/µl
</TD></TR><TR><TD> P775</TD><TD>P187 PAL+ in pSb1C3 sdm with a770g</TD><TD>124,8 ng/µl
</TD></TR><TR><TD> P776</TD><TD>P430 CHS+ in pSb1C3</TD><TD>71,2 ng/µl
</TD></TR><TR><TD> P777</TD><TD>P431 CHS- in pSB1C3</TD><TD>18,1 ng/µl
</TD></TR><TR><TD> P778</TD><TD>P593 4Cl+ in pSB1C3 after sdm 24.08.2012</TD><TD>205,3 ng/µl
</TD></TR><TR><TD> P779</TD><TD>P594 4Cl- in pSB1C3 after sdm 24.08.2012</TD><TD>207 ng/µl
</TD></TR><TR><TD> P780</TD><TD>P595 4Cl+ in pYes after sdm 24.08.2012</TD><TD>174,5 ng/µl
</TD></TR><TR><TD> P781</TD><TD>P596 4Cl- in pYes after sdm 24.08.2012</TD><TD>240,5 ng/µl
</TD></TR><TR><TD> P782</TD><TD>P209 APT in pSb1C3</TD><TD>148,7 ng/µl
</TD></TR><TR><TD> P783</TD><TD>P137 OMT+ in pSB1C3</TD><TD>117,5 ng/µl
</TD></TR><TR><TD> P784</TD><TD>P431 CHS- in pSB1C3 nach sdm c614t Klon 2</TD><TD>115,8 ng/µl
</TD></TR></TBODY></TABLE><P>The side directed mutagenesis leading to P773 (P200 PAL- in pYes sdm with a770g) was not successfull and will be repeated. As the yield of P777 (P431 CHS- in pSB1C3, C = 18,1 ng/µl) is very low a new clone will be picked for an overnight culture.
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Picking_and_Miniprep_of_p737_.28pIntVector_TEF1Prom_Preprothaumatin_TEF1Term.29_and_p738_.28pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term_Not1.29"> Picking and Miniprep of p737 (pIntVector_TEF1Prom_Preprothaumatin_TEF1Term) and p738 (pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term_Not1) </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim of the experiment:</B> Getting pSB1C3_TEF1Prom_Preprothaumatin_TEF1Term with functional NotI
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Expression_of_pTUM104_Preprothaumatin_in_S._cerevisiae"> Expression of pTUM104_Preprothaumatin in <I>S. cerevisiae</I></SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P>Expression according to pYes2_manuel:
</P><UL><LI> Preparation of SC-U medium with 2% glucose/2% galactose: 50 ml-amino acid aliquot + 6.7tg Yeast Nitrogen Base + 850 ml Elga; devide solution in 270 ml and 630 ml; autoclave, add 30 ml 20% glucose to 270 ml and 70 ml 20% galactose to 630 ml. 
</LI></UL><UL><LI> 1. Inoculate a single colony of <I>S. cerevisiae</I> containing your pYES2 construct (pYes2_Preprothaumatin) into 5 ml of SC-U medium containing 2% glucose. Grow overnight at 30°C with shaking. 
</LI></UL><UL><LI> 2. Determine the OD600 of your overnight culture. Calculate the amount of overnight culture necessary to obtain an OD600 of 0.4 in 50 ml of induction medium (0.4*50 ml = OD600*X ml). 3. Remove the amount of overnight culture as determined in Step 2 (X ml) and pellet the cells at 1,500 g for 5 minutes at 4°C. 4. Resuspend the cells in 1–2 ml of induction medium (SC-U medium containing 2% galactose) and inoculate into 50 ml of induction medium. Grow at 30°C with shaking. 5. Harvest an aliquot of cells at 0, 2, 4, 6, 8, and 10 hours (+ over night) after addition of cells to the induction medium. For each time point, remove <B>&quot;1 ml &lt;=&gt; OD = 1&quot;</B> of culture from the flask. 6. Centrifuge the cells at 1,500 g for 5 minutes at 4°C. 7. Decant the <B>supernatant</B>; we only need the supernatant.
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Glycerol_stock_of_pTUM104_Preprothaumatin_in_S._cerevisiae"> Glycerol stock of pTUM104_Preprothaumatin in <I>S. cerevisiae</I></SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><UL><LI> Colony + 5 ml YPD (50 ml-Erlenmeyer flask), 30°C over night; addition of 1 ml sterile 80% glycerol; 1 ml aliquots were stored at -80°C.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Western_blot_analysis_of_limonene_synthase"> Western blot analysis of limonene synthase </SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim of the experiment:</B></P><P>Detection of expressed limonene synthase (citrus).
</P><P><B>Operational sequence:</B></P><P>The western blot membran (having been blocked over the night at 4°C) was washed with PBS- T0.1 three times (3x 10-15min). Afterwards, the membran was incubated for 1h with detection reagent: The membrane was put into a 50 ml falcon tube containing detection reagent and the falcon was incubated in the cold room on the rollator.
Detection reagent:
</P><UL><LI> 10ml 1xPBS+0.1%Tween
</LI><LI> 0,2% BSA (0,02g have been weighted in)
</LI><LI> 2µl anti body MABclassic
</LI></UL><P>After incubating with the first antibody, the membran was washed again with PBS-T0.1 three times (3x 5min), followed by the incubation with the second antibody (anti mouse, fused with alk. phosphatase) for another hour.
Second detection reagent:
</P><UL><LI> 10ml 1xPBS
</LI><LI> 0,2% BSA (see above)
</LI><LI> 5µl AntiMouse-alk. Phosphatase conjugate (1/2000 dilution)
</LI></UL><P>The development was performed after washing the membran with PBS- T0.1 for 10 min (two times) and with 1xPBS for 10 min (two times).
For this, the membran was shortly washed with AP buffer and then incubated with a solution containing 15ml AP buffer, 45µl BCIP (50mg/ml in DMF) and 7,5µl NBT (75mg/ml in 70% DMF) for a few minutes, until clear bands appeared.
</P><P>Western blot membrane:
</P><P>
From left to right:
</P><UL><LI>1. Clone 1 (after dialysis), large-scale (LS with consensus sequence, PCR1)
</LI><LI>2. Clone 2 (after dialysis), large-scale (LS with consensus sequence, PCR1)
</LI><LI>3. LS without consensus sequence (PCR2) (cell lysis 5.9., Katrin, 17 h after induction of expression)
</LI><LI>4. LS without consensus sequence (PCR2) (cell lysis 5.9., Katrin, 20 h after induction of expression)
</LI><LI>5. Clone 1 (before dialysis), large-scale cell lysis on 8.9 (PCR1).
</LI><LI>6. Clone 2 (before dialysis), large-scale cell lysis on 8.9 (PCR1).
</LI><LI>7. Clone 2 (not induced), cell lysis on 8.9 (PCR1).
</LI><LI>8. Yeast, not transformed, cell lysis on 8.9.
</LI><LI>9. page ruler plus
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_transformation_September_10th"> Miniprep of transformation September 10th</SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> Miniprep of 9 over night cultures with number B2A + terminator ADH1 (3 times, colony 1,2,3), B1B + terminator ADH1(3 times, colony 1,2,3), B3D + ADH1 terminator (3 times, colony 1,2,3). Clones were isolated of over night culture, 5ml LB-medium with CHLP.
</P><P>The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 40 µl ddH2O at room temperature, with the columns having been incubated 2 minutes at room temperature before the final centrifugation. 
<B>Miniprep products are stored in a 50 ml Falcon tube in the lowest drawer at -20°C </B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytic_digestion_of_B2A_3.2BADH1_terminator.2C_B1B_2.2BADH1_terminator.2C_B3D_3.2BADH1_terminator_2"> Analytic digestion of B2A_3+ADH1 terminator, B1B_2+ADH1 terminator, B3D_3+ADH1 terminator </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> analytic digestion of B2A_3+ADH1 terminator, B1B_2+ADH1 terminator, B3D_3+ADH1 terminator, B3D_2+ADH1 terminator (plate 5)
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>B2A_3+ADH1 terminator, B1B_2+ADH1 terminator, B3D_3+ADH1 terminator
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0,5  µl
</TD><TD>EcorI (20 U/µl)
</TD></TR><TR><TD>0,5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>12 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytic_gel_electrophoresis">Analytic gel electrophoresis</SPAN></H3><P><B>experimenter:</B> Dennis
</P><UL><LI> 1 µl of DNA loading buffer (10x) was added to the digestion mix after incubation time.
</LI></UL><UL><LI> analytic gelelectrophoresis was perforemd at 90 V for 1 h.
</LI></UL><TABLE cellspacing="0" border="1" style="font-size:75%"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pSB1C3 CaMXMT1 (clone B2A colony 1)
</TD><TD>pSB1C3 CaMXMT1 (clone B2A colony 2)
</TD><TD>pSB1C3 CaMXMT1 (clone B2A colony 3)
</TD><TD>pSB1C3 CaMXMT1 (clone B1B colony 1)
</TD><TD>pSB1C3 CaMXMT1 (clone B1B colony 2)
</TD><TD>pSB1C3 CaMXMT1 (clone B1B colony 3)
</TD><TD>pSB1C3 CaMXMT1 (clone B3D colony 1)
</TD><TD>pSB1C3 CaMXMT1 (clone B3D colony 2)
</TD><TD>pSB1C3 CaMXMT1 (clone B3D colony 3)
</TD><TD>pSB1C3 CaMXMT1 (clone B3D 5 colony 1)
</TD><TD>pSB1C3 CaMXMT1 (clone B3D 5 colony 2)
</TD><TD>pSB1C3 CaMXMT1 (clone B3D 5 colony 3)
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD></TR></TBODY></TABLE><P> + </P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Sampling_of_expressed_proteins"> Sampling of expressed proteins </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was to take a 5 ml sample of the cell suspension 20h after expression induction.
</P><P><B>Operational sequence:</B></P><P>At first, the OD600 was measured (1/10 dilution) and determined as 4,0. Afterwards, 5 ml of the cell suspension were taken out and pelleted by centrifugation for 5 min at 4°C and 1500xg. The pellet was washed with 500µl of sterile water, followed by a second centrifugation step. The pellet was then stored at -80°C until cell lysis.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Cell_lysis_of_stored_pellets_and_SDS_page"> Cell lysis of stored pellets and SDS page </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was to isolate the protein crude extract out of the stored cell pellet. 
Afterwards, we wanted to separate the proteins by SDS gel electrophoresis.
</P><P><B>Operational sequence:</B></P><P>At first, we resuspended the frozen cell pellet in the adequate volume of breaking buffer (containing 1mM PMSF) to get an OD of 50.
</P><UL><LI> CaDXMT1 (15 min after induction of expression): 42 µl breaking buffer (OD600: 0,42)
</LI><LI> CaDXMT1 (20h after induction of expression): 400µl breaking buffer (OD600: 4,0)
</LI></UL><P>The equal amount of glass beads was added, followed by 20- 30 steps of vortexing (30sek) and ice incubation (30sek). 
Afterwards, the protein concentration was determined with NanoDrop (absorption at 280nm)
</P><UL><LI> CaDXMT1 (15min): 17,2 mg/ml
</LI><LI> CaDXMT1 (20h): 7,4 mg/ml
</LI></UL><P>The protein crude extracts were diluted with breaking buffer, to get 10µl samples containing 30µg of protein crude extracts. The samples were mixed with 2,5µl Laemmli (non reducing) and boiled for 5 min. Afterwards, the samples were loaded on the gel, together with 6µl of prestained protein marker (PageRuler Plus) and 6µl unprestained protein marker.
</P><P>The gel was performed at 120V, ca. 2h.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Western_blot_analysis_of_isolated_protein_crude_extract"> Western blot analysis of isolated protein crude extract </SPAN></H3><P><B>Investigator:</B> Roman, Saskia
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the detection of the expressed enzyme CaDXMT1 in the isolated protein crude extract, having been separated on SDS page before (see above). 
</P><P>The proteins were blotted on a PVDF membran for 1h with 50mA. The membran was activated before blocking by 10min incubation in 100% methanol. 
</P><P>After blotting the proteins, we performed a reversible staining of the membran with Ponceau's reagent, to make the unprestained marker visible and to be able to assign it with a pen.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_clones_from_transformation_of_september_8th"> Miniprep of clones from transformation of september 8th </SPAN></H3><P><B>Investigator:</B> Lara, David
</P><P><B>Aim:</B> Get plasmid DNA of clones that contain pSB1C3/TEF1/LS, pSB1C3/TEFF2/LS, pSB1C3/PCR1(LS).
</P><P>LS/TEF2/pSB1C3
</P><UL><LI>1: 263 ng/µl
</LI></UL><P>LS/TEF1/pSB1C3
</P><UL><LI>2(1): 144 ng/µl
</LI><LI>2(2): 123,5 ng/µl
</LI><LI>2(4): 214,6 ng/µl
</LI><LI>2(5): 202,3 ng/µl
</LI><LI>2(6): 290,2 ng/µl
</LI><LI>2(7): 257,8 ng/µl
</LI></UL><P>LS(PCR1)/pSB1C3
</P><UL><LI>3(1): 181,4 ng/µl
</LI><LI>3(2): 168,8 ng/µl
</LI><LI>3(3): 208,1 ng/µl
</LI><LI>3(4): 179,4 ng/µl
</LI><LI>3(5): 186,2 ng/µl
</LI><LI>3(6): 224,8 ng/µl
</LI><LI>3(7): 186,0 ng/µl
</LI><LI>3(8): 280,0 ng/µl
</LI><LI>3(9): 120,3 ng/µl
</LI><LI>3(10): 206,1 ng/µl
</LI></UL><P><B>miniprep products are stored in a disposal bag in the middle drawer of -20°C. label: lara miniprep 11.9.</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_digest_and_gel_electrophoresis_2"> Analytical digest and gel electrophoresis </SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Check whether ligations worked (ligation with TEF promoters and ligation of PCR1 with pSB1C3)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoR1
</TD></TR><TR><TD>0.25 µl
</TD><TD>Spe-HF1
</TD></TR><TR><TD>2 µl
</TD><TD>NEB
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>incubation: 1.5 hours, 37°C
</LI></UL><P><B>Gelelectrophoresis:</B></P><P>gel 1: marker, 1, 2(1), 2(2), 2(3), 2(4), 2(5), 2(6), 2(7)
gel 2: marker, 3(1), 3(2), 3(3), 3(4), 3(5), 3(6), 3(7), 3(8), 3(9), 3(10)
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_September_12th"><B>Wednesday, September 12th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_BBa_E1010.2C_BBa_J52008.2C_pTUM101.2C_pTUM103.2C_P508_with_EcorI-HF_and_SpeI-HF"> Preparative digestion of BBa_E1010, BBa_J52008, pTUM101, pTUM103, P508 with EcorI-HF and SpeI-HF</SPAN></H3><P><B> Investigator:Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>7 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>7 µl
</TD><TD>XbaI (NEB)
</TD></TR><TR><TD>28 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>2,8 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>95,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=140µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 20 µl digestion batch 20 µl DNA were added digestion was done for 3 hours at 37 C
</LI><LI> Experiment was stopped since it didn't made sense
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_Gelexlectrophoris_and_gelextraction_of_preparative_digested_probes"> Preparative Gelexlectrophoris and gelextraction of preparative digested probes </SPAN></H3><P><B> Investigator: Georg</B></P><UL><LI> Gelelectrophoresis was done on 1% preparative agarose at 70 V for 90 min
</LI><LI> Gelextraction was performed according to Quiaquick Gel extraction protocoll
</LI></UL><P><A href="/File:20120912_pYES2_verdaut_mit_EcorI_und_SpeI.png" title="File:20120912 pYES2 verdaut mit EcorI und SpeI.png">File:20120912 pYES2 verdaut mit EcorI und SpeI.png</A></P></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_measurement_of_digested_BBa_E1010.2C_pTUM_101.2CBBa_J52008.2C_TEF2-P-Limonens..2C_psb1c3_.28P.28811.29.2C_TEF1-T_.28P812.29.2C_Midiprep"> Nanodrop measurement of digested BBa_E1010, pTUM 101,BBa_J52008, TEF2-P-Limonens., psb1c3 (P(811), TEF1-T (P812), Midiprep</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> BBa_E1010: 7,5 ng/µl
</LI><LI> pTUM101(22,2 ng/µl)
</LI><LI> BBa_J52008 (P791): 27,1 ng/µl
</LI><LI> TEF2-P-Limonens.: 18 ng/µl
</LI><LI> psb1c3 (P811): 47,8 ng/µl
</LI><LI> TEF1-T (P812): 10,5 ng/µl
</LI><LI> Midiprep: 339 ng/µl
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligations_of_P680.2B_P791.2C_P679.2B_P791.2C_P885.2B_P886.2C_P679.2B_P578.2C_P680.2B_P578"> Ligations of P680+ P791, P679+ P791, P885+ P886, P679+ P578, P680+ P578</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI><B>Procedure:</B></LI><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,92 µl
</TD><TD>P680
</TD></TR><TR><TD>1,88 µl
</TD><TD>P791
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>13,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>Reaction batch for Ligation:
</LI></UL><TABLE border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,25 µl
</TD><TD>P679
</TD></TR><TR><TD>1,95 µl
</TD><TD>P791
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>16,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>2,08 µl
</TD><TD>P885
</TD></TR><TR><TD>7,22 µl
</TD><TD>P886
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>7,7 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>4,51 µl
</TD><TD>pYES2
</TD></TR><TR><TD>2,29 µl
</TD><TD>TEF2-P-Limonens.
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>10,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,26
</TD><TD>P679
</TD></TR><TR><TD>P5,14 µl
</TD><TD>P578
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>10,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,91 µl
</TD><TD>P680
</TD></TR><TR><TD>4,89 µl
</TD><TD>P578
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>10,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI>To each ligation batch a negative control was mixed by adding the same volum of ddH20 instead of insert-DNA
</LI><LI>Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_product_of_P751.2BP752"> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation product of P751+P752 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation product of P751+P752.
</P><P><B>Procedure:</B></P><UL><LI> 4 colonies of the transformation with P751+P752 (CamR) and were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chloramphenicol. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P687.2BP690"> Preparative digestion and gelelectrophoresis of P687+P690 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P687 (EcoRI-HF+SpeI-HF), P690 (EcoRI-HF+XbaI).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for digestion of P687 with EcoRI-HF+SpeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P687
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P690 with EcoRI-HF+XbaI:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P690
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch was incubated at 37 °C for 2.5 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P687 EcoRI-HF + SpeI-HF
</TD><TD>1 kbp DNA ladder
</TD><TD>P690 EcoRI-HF + XbaI
</TD></TR><TR><TD>Lower band was cut out (1283 bp, 37.6 ng/µl)
</TD><TD>Band was cut out (3021 bp, 105.6 ng/µl)
</TD></TR></TBODY></TABLE><UL><LI> Gel extraction was carried out after manufacturer's protocol (Qiagen).
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P765.2BP764_and_P751.2BP752"> Cycled ligation of P765+P764 and P751+P752 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Cycled ligation of P765+P764 (1:3) and P751+P752 (1:6).
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P765+P764 (1:3):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.94 µl
</TD><TD>P765 (105.6 ng/µl, 3021 bp)
</TD></TR><TR><TD>3.35 µl
</TD><TD>P764 (37.6 ng/µl, 1283 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12.71 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P751+P752 (1:6):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.25 µl
</TD><TD>P751 (80.3 ng/µl, 3316 bp)
</TD></TR><TR><TD>5.21 µl
</TD><TD>P752 (34.3 ng/µl, 988 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>10.54 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative controls were als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batches and were called: P765 NK and P751 NK.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler overnight:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>infinite cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Oligohybridization_of_oligo-nucleotides_for_creating_mini_MCS_for_cloning_RFC10_compatible_parts_between_promoter_and_terminator"> Oligohybridization of oligo-nucleotides for creating mini MCS for cloning RFC10 compatible parts between promoter and terminator </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Oligohybridization of oligo-nucleotides for creating mini MCS for cloning RFC10 compatible parts between promoter and terminator.
</P><P><B>Operational sequence:</B></P><UL><LI> 25 µL of 100 µM of Mini MCS fw pSB oligos (O92) and 25 µL of 100 µM MCS pSB1 rv neu oligos (O128) was pooled together in one ERG
</LI></UL><UL><LI> Heating up to 90 °C for 5 min
</LI></UL><UL><LI> Cooling at RT in a styropor box overnight.
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="SDS-PAGE_of_Thaumatin_.28pTUM104_Preprothaumatin_in_S._cerevisiae.29"> SDS-PAGE of Thaumatin (pTUM104_Preprothaumatin in <I>S. cerevisiae</I>) </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><UL><LI> Separation gel: 1.75 ml H20, 2 ml Acrylamid-Bis (30 % w/v), 1.25 ml 4x Lower Tris, 4 μl TEMED und 40 μl APS (10 % w/v). 
</LI><LI> Collection gel: 1.75 ml H20, 0.5 ml Acrylamid-Bis (30 % w/v), 0.75 ml 4x Upper Tris, 4 μl TEMED und 40 μl APS (10 % w/v).
</LI><LI> Marker mix: 6 µl marker, 6 μl H2O and 4 μl 5x loading dye (non reducing) =&gt; 15 µl.
</LI><LI> Sample mix: 16 μl sample (pot. diluted), 4 µl 5x Auftragspuffer (non reducing) =&gt; 15 µl.
</LI><LI> Electrophoresis: Ca. 10 min, 80 V; then 120 V; about 2 h until bromphenol blue marker reaches the bottom of the gel.
</LI><LI> The separatory gel was swivelled 20 min in Coomassie, 15 min in Entfärbelösung 1 and then in Entfärbelösung 2.
</LI></UL><UL><LI> SDS-PAGE1 (non reducing): M/S1/S2/0h/2h/4h/6h/8h/10h/24h.
</LI><LI> SDS-PAGE2 (non reducing): M/S1/S2/0h/2h/4h/6h/8h/10h/24h.
</LI></UL><P>Thaumatin: <B>22,2 kDa</B>.
</P><P><B>Result</B>: 
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_clone_1_into_E..C2.A0coli"> Transformation of clone 1 into <I>E. coli</I></SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B> Transformation of clone 1 (Miniprep 11.09.) into <I>E. coli</I> XL1-Blue
</P><P><B>Procedure:</B></P><UL><LI> 1µl of each miniprep product were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates (two plates).
</LI></UL><UL><LI> incubation over night (37°C)
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Protein_Purification_of_Limonensynthase"> Protein Purification of Limonensynthase </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiment:</B> Purification of Limonensynthase via Streptavidin column (1.6)
</P><P><B>Procedure:</B></P><UL><LI> reaching baseline with 1x SA buffer
</LI><LI> binding of proteins 
</LI><LI> elution of limonenesynthase with 1x SA buffer + 5 mM Biotin
</LI><LI> washing of pipes and column with 1x SA buffer (eight fold of column volume) and ddH2O 
</LI></UL><P><B>Concentration:</B></P><UL><LI> made by Alois
</LI><LI> LS: 0.539 mg/ml
</LI><LI> concentrated protein is stored at -20°C (upper drawer) 
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Staining_with_Ponceau.27s_reagent_and_western_blot_analysis_of_protein_crude_extract_of_CaXMT1.2C_CaMXMT1_and_eGFP_transformants_after_20_minutes_and_20_hours_of_expression_2"> Staining with Ponceau's reagent and western blot analysis of protein crude extract of CaXMT1, CaMXMT1 and eGFP transformants after 20 minutes and 20 hours of expression </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B></P><P>Detection of expressed proteins CaDXMT1, eGFP.
</P><P><B>Operational sequence:</B></P><P>The western blot membran (having been blocked over the night at 4°C) was washed with PBS- T0.1 four times (4x 15min). 
Because of the use of un- prestained protein marker (see Thursday, 6.9.), we stained the western blot membran with Ponceau's reagent and assigned the bands, followed by washing the membran with PBS- T0.1 again (15 minutes, 2x).
Afterwards, the membran was incubated for 1h with detection reagent.
Detection reagent:
</P><UL><LI> 10ml 1xPBS
</LI><LI> 0,2% BSA (0,02g have been weighted in)
</LI><LI> 2µl anti body MABclassic
</LI></UL><P>After incubating with the first antibody, the membran was washed again with PBS-T0.1 three times (3x 5min), followed by the incubation with the second antibody (anti mouse, fused with alk. phosphatase) for another our.
Second detection reagent:
</P><UL><LI> 10ml 1xPBS
</LI><LI> 0,2% BSA (see above)
</LI><LI> 5µl AntiMouse-alk. Phosphatase conjugate (1/2000 dilution)
</LI></UL><P>The development was performed after washing the membran with PBS- T0.1 for 10 min (two times) and with 1xPBS for 10 min (two times).
For this, the membran was shortly washed with AP buffer and then incubated with a solution containing 15ml AP buffer, 45µl BCIP (50mg/ml in DMF) and 7,5µl NBT (75mg/ml in 70% DMF) for a few minutes, until clear bands appeared.
</P><P>Western blot membran:
</P><P>From left to right:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Prestained protein marker (Page Ruler Plus)
</TD><TD> eGFP crude extract
</TD><TD> CaDXMT1 crude extract
</TD><TD>Unstained protein marker (pen marking)
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytic_digestion_of_B2A_3.2BADH1_terminator.2C_B1B_2.2BADH1_terminator.2C_B3D_3.2BADH1_terminator_3"> Analytic digestion of B2A_3+ADH1 terminator, B1B_2+ADH1 terminator, B3D_3+ADH1 terminator </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> analytic triple digestion of B2A_3+ADH1 terminator, B1B_2+ADH1 terminator, B3D_3+ADH1 terminator, B3D_2+ADH1 terminator (plate 5)
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>B2A_3+ADH1 terminator, B1B_2+ADH1 terminator, B3D_3+ADH1 terminator
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0,5  µl
</TD><TD>EcorI (20 U/µl)
</TD></TR><TR><TD>0,5 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoR53kI (10 U/µl)
</TD></TR><TR><TD>11 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Cell_lysis_of_200ml_culture_of_CaDXMT1_after_24h_after_expression_induction"> Cell lysis of 200ml culture of CaDXMT1 after 24h after expression induction</SPAN></H3><P><B>Investigator:</B> Daniela, Saskia
</P><P><B>Aim of the experiment:</B></P><P>Cell lysis of yeast cells- taken 24h after expression. 
</P><P><B>Operational sequence:</B></P><UL><LI> centrifugation for 10 min at 14000 rpm (4x 50 ml Falcons)
</LI></UL><UL><LI> discard supernatant
</LI></UL><UL><LI> the pellets were resuspended in breaking buffer without PMSF and centrifugated at  14000 rpm for 5 min; discard supernatant
</LI></UL><UL><LI> an adequate amount of breaking buffer containing PMSF (OD:13.6) was given to the cell pellet, as well as an equal amount of glass beads. 
</LI></UL><UL><LI> cell- walls were disrupted by vortexing for 30 sec, followed by 30 sek on ice. This procedure was repeated 30 times.
</LI></UL><UL><LI> centrifugation and storage of the supernatant at -20°C
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytic_gel_electrophoresis_2">Analytic gel electrophoresis</SPAN></H3><P><B>experimenter:</B> Dennis
</P><UL><LI> 1 µl of DNA loading buffer (10x) was added to the digestion mix after incubation time.
</LI></UL><UL><LI> analytic gelelectrophoresis was perforemd at 90 V for 1 h.
</LI></UL><TABLE cellspacing="0" border="1" style="font-size:75%"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pSB1C3 CaMXMT1 ( B2A 1, new)
</TD><TD>pSB1C3 CaMXMT1 ( B2A 2, new)
</TD><TD>pSB1C3 CaMXMT1 ( B2A 3, new)
</TD><TD>pSB1C3 CaMXMT1 ( B2A 4, new)
</TD><TD>pSB1C3 CaMXMT1 ( B2A )
</TD><TD>pSB1C3 CaMXMT1 ( B1B )
</TD><TD>pSB1C3 CaMXMT1 ( B3D )
</TD><TD>pSB1C3 CaMXMT1 ( negativ: B3D without ADH1 )
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD></TR></TBODY></TABLE></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_mutations_found_by_sequencing_in_our_gene_constructs_2">  Quick Change mutagenesis to remove mutations found by sequencing in our gene constructs  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P><B>Aim of the experiment: </B>Deletion of found mutations.
</P><P><B>Operational sequence</B></P><P><B>1. PCR general setup</B><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of appropriate Oligo (= 5 pmol)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of appropriate Oligo (= 5 pmol)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI></UL><P><B>2. Used constructs and primers</B></P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Quickchange number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Plasmid number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Geneconstruct</B></TD><TD align="center" style="background:#f0f0f0;"><B>Primer number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Primer</B></TD></TR><TR><TD> QC 1</TD><TD>P769</TD><TD>CHS- pSB1C3 sdm c614t</TD><TD>O98 &amp; O99</TD><TD>t375c
</TD></TR><TR><TD> QC 2</TD><TD>P774</TD><TD>PAL- pSB1C3</TD><TD>O110 &amp; O111</TD><TD>c869t
</TD></TR><TR><TD> QC 3</TD><TD>P775</TD><TD>PAL+ pSB1C3</TD><TD>O110 &amp; O111</TD><TD>c869t
</TD></TR><TR><TD> QC 4</TD><TD>P772</TD><TD>PAL+ pYes</TD><TD>O110 &amp; O111</TD><TD>c869t
</TD></TR><TR><TD> QC 5</TD><TD>P745</TD><TD>CHS- pYes Limonen</TD><TD>O104 &amp; O105</TD><TD>c614t
</TD></TR><TR><TD> QC 6</TD><TD>P748</TD><TD>OMT- pYes Limonen</TD><TD>O96 &amp; O97</TD><TD>a1055g
</TD></TR><TR><TD> QC 7</TD><TD>P137</TD><TD>OMT+pSB1C3</TD><TD>O96 &amp; O97</TD><TD>a1055g
</TD></TR><TR><TD> QC 8</TD><TD>P739</TD><TD>PAL+ pYes Limonen</TD><TD>O110 &amp; O111</TD><TD>c869t
</TD></TR><TR><TD> QC 9</TD><TD>P740</TD><TD>PAL- pYes Limonen</TD><TD>O110 &amp; O111</TD><TD>c869t
</TD></TR></TBODY></TABLE><UL><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_September_13th"><B>Thursday, September 13th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_XL1-blue_with_ligation_batches_from_12.9">Transformation of <I>E.coli</I> XL1-blue with ligation batches from 12.9</SPAN></H3><P><B>Investigator:Georg</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on chloramphenicole plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new CAM plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_pTUM104_with_TEF2-P">Miniprep of pTUM104 with TEF2-P </SPAN></H3><UL><LI> plasmid-extraction was done according to quiaprep plasmid extraction protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_pTUM102-P.2CpTUM101-P.2C_pTUM103-P.2C_Limonensynthases"> Preparative digestion of pTUM102-P,pTUM101-P, pTUM103-P, Limonensynthases </SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>PstI-HF (Ferm)
</TD></TR><TR><TD>2 µl
</TD><TD>SpeI (NEB)
</TD></TR><TR><TD>4 µl
</TD><TD>10x Tango-buffer
</TD></TR><TR><TD>4 µl
</TD><TD>10xNEB 4 buffer
</TD></TR><TR><TD>0,4 µl
</TD><TD>100 BSA
</TD></TR><TR><TD>27,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 40 µl reaction batch 40 µl of pTUM104-TEF2-P were added.
</LI></UL><P><I>Investigator: Georg'</I></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>PstI-HF (Ferm)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (NEB)
</TD></TR><TR><TD>2 µl
</TD><TD>10x Tango-buffer
</TD></TR><TR><TD>2 µl
</TD><TD>10xNEB 4 buffer
</TD></TR><TR><TD>0,2 µl
</TD><TD>100 BSA
</TD></TR><TR><TD>11,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 20 µl reaction batch 20 µl of pTUM104-TEF1-P were added
</LI></UL><P><I>Investigator: Georg'</I></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>PstI-HF (Ferm)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (NEB)
</TD></TR><TR><TD>2 µl
</TD><TD>10x Tango-buffer
</TD></TR><TR><TD>2 µl
</TD><TD>10xNEB 4 buffer
</TD></TR><TR><TD>0,2 µl
</TD><TD>100 BSA
</TD></TR><TR><TD>10,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P><I>Investigator: Georg'</I></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>PstI-HF (Ferm)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI (NEB)
</TD></TR><TR><TD>2 µl
</TD><TD>10x Tango-buffer
</TD></TR><TR><TD>2 µl
</TD><TD>10xNEB 4 buffer
</TD></TR><TR><TD>0,2 µl
</TD><TD>100 BSA
</TD></TR><TR><TD>10,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 20 µl reaction batch 20 µl of pTUM104-ADH1-P were added
</LI></UL><P><I>Investigator: Georg'</I></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>4 µl
</TD><TD>PstI-HF (Ferm)
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI (Ferm)
</TD></TR><TR><TD>4 µl
</TD><TD>10x Tango-buffer
</TD></TR><TR><TD>2 µl
</TD><TD>10xNEB 4 buffer
</TD></TR><TR><TD>0,2 µl
</TD><TD>100 BSA
</TD></TR><TR><TD>10,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 20 µl reaction batch 20 µl of Limonensynthases in psb1c3
</LI><LI> Digestions took place for 3 hours at 37 C
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="gelextraction_of_digested_pTUM102_.28P806.29"> gelextraction of digested pTUM102 (P806)</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Gelextraction was done according to quiaquick gel extraction kit protocoll
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_expansion_E..C2.A0coli_XL-Blue_cells_culture.2C_transformated_with_ligation_product_P751.2BP752"> Miniprep of overnight expansion <I>E. coli</I> XL-Blue cells culture, transformated with ligation product P751+P752  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of overnight expansion <I>E. coli</I> XL-Blue cells culture, transformated with ligation product P751+P752.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done after manufacturer's protocol (P685-P688). (QIAGEN - QIAprep Spin Miniprep Kit)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P785-P788"> Analytical digestion and gelelectrophoresis of P785-P788 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P785-P788 with EcoRI-HF+SpeI-HF+PvuII.
</P><P><B>Procedure:</B></P><UL><LI> Digestion of P785-P788 with EcoRI-HF+SpeI-HF+PvuII:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P785-P788)
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>0.50 µl
</TD><TD>PvuII (10 U/µl) (NEB)
</TD></TR><TR><TD>14.3 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion reaction mix was incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 2 µl of DNA loading buffer (10x) was added to the 20 µl of reaction mix after digestion.
</LI></UL><UL><LI> 10 µl of these mixes were loaded in the gel pockets.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 60 min on an 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P785
</TD><TD>P786
</TD><TD>P787
</TD><TD>P788
</TD><TD>P751
</TD><TD>P752
</TD></TR><TR><TD>Too much bands visible. Furhter analysis required.
</TD><TD>Too much bands visible. Furhter analysis required.
</TD><TD>Too much bands visible. Furhter analysis required.
</TD><TD>Too much bands visible. Furhter analysis required.
</TD><TD>Only for control of prep. digestion
</TD><TD>Only for control of prep. digestion
</TD></TR></TBODY></TABLE><UL><LI> Further analytical digestion and gelelectrophoresis with only EcoRI-HF+SpeI-HF have to be carried out to make sure that ligation was successful.
</LI></UL><UL><LI> Please see next analytical digestion and gelelectrophoresis section with only EcoRI-HF and SpeI-HF for end results.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P785-P788_II"> Analytical digestion and gelelectrophoresis of P785-P788 II </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P785-P788 with EcoRI-HF+SpeI-HF.
</P><P><B>Procedure:</B></P><UL><LI> Digestion of P785-P788 with EcoRI-HF+SpeI-HF+PvuII:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>Plasmid DNA (P785-P788)
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Analytical digestion reaction mix was incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 2 µl of DNA loading buffer (10x) was added to the 20 µl of reaction mix after digestion.
</LI></UL><UL><LI> 10 µl of these mixes were loaded in the gel pockets.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 90 min on an 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P785
</TD><TD>P786
</TD><TD>P787
</TD><TD>P788
</TD></TR><TR><TD>Ligation NOT successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD></TR></TBODY></TABLE><UL><LI> Last ligation step to SV40NLS-PhyB(908NT)-20aaLinker-Gal4DBD/LexA+TEF1-terminator nescessary and cloning whole expression casette battery into pSB6A0 with EcoRI-HF and SpeI-HF because pSB6A0 still contains at least one of the forbidden restriction sites XbaI or PstI.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P786.2C_P693.2C_P697"> Preparative digestion of P786, P693, P697 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion of P786 (EcoRI-HF+SpeI-HF), P693 (EcoRI-HF+XbaI), P697 (EcoRI-HF+XbaI).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for digestion of P786 with EcoRI-HF+SpeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P786
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P693 with EcoRI-HF+XbaI:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>19 µl
</TD><TD>P693
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>14.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P697 with EcoRI-HF+XbaI:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P697
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Sequencing_2">Sequencing</SPAN></H3><P><B>Investigator:</B> Lara, Andrea
</P><P><B>Aim:</B> Sequencing to check whether insert is wrong. Miniprep products of 11.09. (2: pSB1C3/TEF1/LS, 3: pSB1C3/PCR1(LS) ) were sequenced. Additionally probes of Dennis (B2A, B1B, B3D, B3D5) were sequenced. 
</P><UL><LI>3 (8): Primer VF2, VR
</LI><LI>2 (5): Primer VF2
</LI><LI>2 (7): Primer VF2
</LI></UL><UL><LI>B2A: Primer VF2
</LI><LI>B2A: Primer VR
</LI><LI>B1B: Primer VF2
</LI><LI>B1B: Primer VR
</LI><LI>B3D: Primer VF2
</LI><LI>B3D: Primer VR
</LI><LI>B3D5: Primer VF2
</LI><LI>B3D5: Primer VR
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_colonies_of_clone_1_.28miniprep_11.09..29">Picking of colonies of clone 1 (miniprep 11.09.) </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Procedure:</B></P><UL><LI> Picking of 1 clone of pSB1C3/TEF2/LS (transformation 8th August)
</LI><LI> add 4 µl Chloramphenicol
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_of_pSB1C3.2FTEF2.2FLS_.28Miniprep_11.09..29_and_pSB1C3.2FTEF1.2FLS">Preparative restriction digest of pSB1C3/TEF2/LS (Miniprep 11.09.) and pSB1C3/TEF1/LS</SPAN></H3><P><B>Investigator:</B> Andrea, Lara
</P><P><B>Aim:</B> Prepare backbone for Promotor-LS-Terminator construct in pSB1C3.
</P><P>pSB1C3/TEF2/LS
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>15  µl
</TD><TD>Plasmid (clone 1, miniprep 11.09.)
</TD></TR><TR><TD>4  µl
</TD><TD>Buffer Tango, Fermentas (10x)
</TD></TR><TR><TD>1  µl
</TD><TD>Spe-HF (20 U/µl), NEB
</TD></TR><TR><TD>2.5  µl
</TD><TD>PstI (10 U/µl), Fermentas
</TD></TR><TR><TD>18 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><P>pSB1C3/TEF1/LS
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10  µl
</TD><TD>Clone 2(5) DNA (miniprep 11.09.)
</TD></TR><TR><TD>4  µl
</TD><TD>Buffer Tango, Fermentas (10x)
</TD></TR><TR><TD>1  µl
</TD><TD>SpeI (20 U/µl), NEB
</TD></TR><TR><TD>2.5  µl
</TD><TD>PstI (10 U/µl), Fermentas
</TD></TR><TR><TD>23  µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> plasmids were digested at 37 °C for 3 hours. THe samples were stored at the lowest drawer of -20°C over night (50 ml Falcon, label: &quot;Präp Verdau, Lara, 13.9.&quot;).
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Miniprep_of_clones_with_Psb1C3_TEF1P_Thaumatin_TEF1T_and_Integ-Vec_TEF1P_Thaumatin_TEF1T"> Miniprep of clones with Psb1C3_TEF1P_Thaumatin_TEF1T and Integ-Vec_TEF1P_Thaumatin_TEF1T </SPAN></H3><P><B>Investigat0r:</B> Martin
<B>Results:</B>: P792-P801
</P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Midiprep_of_clone_with_Integ-Vec_mOrange-cassette"> Midiprep of clone with  Integ-Vec_mOrange-cassette </SPAN></H3><P><B>Investigat0r:</B> Martin
<B>Results:</B>: P802
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_September_14th"><B>Friday, September 14th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Gelextraction_of_digested_pTUM104-TEF1-P.2C_pTUM104-ADH1-P.2C_Limonensynthases_from_23.9"> Gelextraction of digested pTUM104-TEF1-P, pTUM104-ADH1-P, Limonensynthases from 23.9</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Gelextraction was performed according to quiaquick gelextraction kit protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Nanodrop_measurement"> Nanodrop measurement</SPAN></H3><P><B>Investigator: Georg,Mary</B></P><UL><LI> pTUM104-TEF1 (P809): 22.2 ng/µl
</LI><LI> pTUM104-ADH1 (P807): 27.2 ng/µl
</LI><LI> Limonensynthases (P810): 13.3 ng/µl
</LI><LI> New Multiple Cloning Site (P808): 150 ng/µl
</LI><LI> pTUM104-TEF2-P(P806): 122 ng/µl 
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_reaction_of_P809.2BP810.2C_P806.2BP810.2C_P807.2BP810.2C_P571_.28psb1c3-vector.29.2B_P808.2C_P571.2B_P812.2C_P809.2B_P791.2C_P807.2B_P791.2C_P809.2B_P791">Ligation reaction of P809+P810, P806+P810, P807+P810, P571 (psb1c3-vector)+ P808, P571+ P812, P809+ P791, P807+ P791, P809+ P791 </SPAN></H3><P><B> Investigator: Georg</B></P><P><B>Procedure:</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>4,04 µl
</TD><TD>P809
</TD></TR><TR><TD>6,26 µl
</TD><TD>P810
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>6,7 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>0,82 µl
</TD><TD>P806
</TD></TR><TR><TD>6,88 µl
</TD><TD>P810
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>9,3 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,67 µl
</TD><TD>P807
</TD></TR><TR><TD>6,63 µl
</TD><TD>P810
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>6,7 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>2,54 µl
</TD><TD>P571
</TD></TR><TR><TD>2,08 µl
</TD><TD>P810
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>14,05 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>2,08 µl
</TD><TD>P571
</TD></TR><TR><TD>7,22 µl
</TD><TD>P812
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>7,7 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>4,53 µl
</TD><TD>P809
</TD></TR><TR><TD>1,97 µl
</TD><TD>P791
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>10,5 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>0,82 µl
</TD><TD>P806
</TD></TR><TR><TD>1,88 µl
</TD><TD>P791
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>14,3 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>3,67 µl
</TD><TD>P807
</TD></TR><TR><TD>1,89µl
</TD><TD>P791
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>11,44 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> For each Vector, a negative control was made
</LI><LI> Cycled ligation was performed as follows
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Transformation_of_with_ligation_batches_clone1_.2B_P681.2C_clone2.2B_p681.2C_clone2_.2B_P642"> Transformation of with ligation batches clone1 + P681, clone2+ p681, clone2 + P642</SPAN></H3><P><B> Investigator:Georg</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on chloramphenicole plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new CAM plates.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Miniprep.2C_of_TEF2-P-Thaum.2C_TEF1-P-Thaum.2C_ADH1-P-Thaum_in_pTUM104">Miniprep, of TEF2-P-Thaum, TEF1-P-Thaum, ADH1-P-Thaum in pTUM104</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Miniprep was performed according to quiaprep miniprep kit protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelctrophoresis_of_extracted_DNA_from_transformants_with_Limonens.-pTUM102.2C_Thaum-pTUM102_.2C_Thaum-pTUM101.2CThaum-pTUM103"> Analytical digestion and gelelctrophoresis of extracted DNA from transformants with Limonens.-pTUM102, Thaum-pTUM102 , Thaum-pTUM101,Thaum-pTUM103</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>3,25 µl
</TD><TD>EcorI-HF (NEB)
</TD></TR><TR><TD>3,25 µl
</TD><TD>PstI
</TD></TR><TR><TD>26 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>195 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=227,5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> To 17,5 µl reaction batch 2,5 µl DNA was added and digested for 1 h at 37 C
</LI><LI> Gelelectrophoresis was run on 1% analytical agarose gels at 90 V for 1 hour
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_gelelectrophoresis_of_preparative_digestion_of_P786.2C_P693.2C_P697"> Preparative gelelectrophoresis of preparative digestion of P786, P693, P697 </SPAN></H3><P><B>Investigator:</B> Mary, Jeff
</P><P><B>Aim of the experiment:</B></P><P><B>Procedure:</B></P><UL><LI> 4.44 µl of DNA loading buffer (10x) were added to reaction mixes after incubation time and loaded on an 1% preparative agarose gel.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed 1.5 h at 70 V.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P786 EcoRI-HF + SpeI-HF
</TD><TD>1 kbp DNA ladder
</TD><TD>P693 EcoRI-HF + XbaI
</TD><TD>1 kbp DNA ladder
</TD><TD>P697 EcoRI-HF + XbaI
</TD></TR></TBODY></TABLE><UL><LI> No  bands were cut out because it seems that SpeI-HF doesn't work (It was the last rest in the tube, so it seems that it was only condensed water.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P786.2C_P692.2C_P696"> Preparative digestion and gelelectrophoresis of P786, P692, P696 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P786 (SpeI-HF+PstI-HF), P692 (XbaI-HF+PstI-HF), P696 (XbaI-HF+PstI-HF).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for digestion of P786 with SpeI-HF+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P786
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P692 with XbaI-HF+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P692
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P696 with XbaI-HF+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P696
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch was incubated at 37 °C for 2.5 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P786 SpeI-HF + PstI-HF
</TD><TD>P692 XbaI + PstI-HF
</TD><TD>P696 XbaI + PstI-HF
</TD></TR><TR><TD>Band was cut out (4290 bp, 68.4 ng/µl)
</TD><TD>Upper band was cut out (3848 bp,82.0 ng/µl)
</TD><TD>Upper band was cut out (4013 bp, 77.5 ng/µl)
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P803.2BP804.2C_P803.2BP805.2C_P663.2BP791"> Cycled ligation of P803+P804, P803+P805, P663+P791  </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Cycled ligation of P803+P804, P803+P805, P663+P791.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P803+P804 (1:3):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.46 µl
</TD><TD>P803 (68.4 ng/µl, 4290 bp)
</TD></TR><TR><TD>3.27 µl
</TD><TD>P804 (82.0 ng/µl, 3848 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>12.27 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P803+P805 (1:3):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.46 µl
</TD><TD>P803 (68.4 ng/µl, 4290 bp)
</TD></TR><TR><TD>3.6 µl
</TD><TD>P805 (77.5 ng/µl, 4013 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>11.94 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P663+P791 (1:3):
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12.82 µl
</TD><TD>P663 (7.8 ng/µl, 3578 bp)
</TD></TR><TR><TD>2.96 µl
</TD><TD>P791 (27.1 ng/µl, 956 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>1.22 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative controls were als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batches and were called: P803 NK and P663 NK.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler overnight:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>400x
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparing_plates_with_minimal_media_.28URA-.2C_TRP-_and_URA-.2C_TRP-.2C_HIS_-.29_for_LiAc_transfection_of_S..C2.A0cerevisiae_Y190_and_INVSc1_strain"> Preparing plates with minimal media (<I>URA<SUP>-</SUP></I>, <I>TRP<SUP>-</SUP></I> and <I>URA<SUP>-</SUP></I>, <I>TRP<SUP>-</SUP></I>, <I>HIS</I><SUP>-</SUP>) for LiAc transfection of <I>S. cerevisiae</I> Y190 and INVSc1 strain </SPAN></H3><P><B>Investigator:</B> Mary, Jeff, Katrin
</P><P><B>Aim of the experiment:</B> Preparing plates with minimal media (<I>URA<SUP>-</SUP></I>, <I>TRP<SUP>-</SUP></I> and <I>URA<SUP>-</SUP></I>, <I>TRP<SUP>-</SUP></I>, <I>HIS</I><SUP>-</SUP>) for LiAc transfection of <I>S. cerevisiae</I> Y190 and INVSc1 strain.
</P><P><B>Procedure:</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_clones_.2813.09..29">Miniprep of clones (13.09.)</SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim:</B> Get more plasmid DNA of positive clone 1 
</P><P>concentrations:
</P><UL><LI> clone 1_1: 254,8 ng/µl
</LI><LI> clone 1_2: 241,5 ng/µl
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_gelelectrophoresis_of_pSB1C3.2FLS.2FProm_and_Terminator_digested_on_13th_September">Preparative gelelectrophoresis of pSB1C3/LS/Prom and Terminator digested on 13th September</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Gelelectrophoresis:</B></P><P>LS/pSB1C3/Prom: 4500 bp
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_pSB1C3.2FLS.2FTEF1_and_pSB1C3.2FLS.2FTEF2_with_CYC1-Terminator_and.2For_TEF1-Terminator">Ligation of pSB1C3/LS/TEF1 and pSB1C3/LS/TEF2 with CYC1-Terminator and/or TEF1-Terminator </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Procedure</B></P><P><B>clone 1 with P681 (CYC1)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>clone
</TD><TD>6.61 µl
</TD></TR><TR><TD>P681
</TD><TD>1.398 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>Negative control water in clone 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>clone
</TD><TD>6.61 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>1.398 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>clone 2 with CYC1 (P681)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>clone 2
</TD><TD>5.89 µl
</TD></TR><TR><TD>P681
</TD><TD>2.11 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>Negative control water in clone 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>clone 2
</TD><TD>5.89 µl
</TD></TR><TR><TD>ddH2O
</TD><TD>2.118 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>clone 2 with TEF1 (P642)</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>clone 2
</TD><TD>4.11 µl
</TD></TR><TR><TD>TEF1
</TD><TD>3.898 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><UL><LI>Trafo made by Georg
</LI><LI>rest of ligation solution incubated over the weekend at RT --&gt; now stored at -20°C in a 50ml Falcon tube in the middle drawer (label: Ligierung 14.09. Andrea)
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="SDS_Page_of_pTUM104_thaumatin_expression_experiment.27s_cell_lysates"> SDS Page of pTUM104_thaumatin expression experiment's cell lysates </SPAN></H3><P><B>Investigator:</B> Katrin, Martin
</P><P><B>Aim:</B> To find that frigging Thaumatin
</P><P><B>Results:</B></P><UL><LI> Thaumatin: 22,2 kD
</LI></UL><P>
SDS Page:
</P><P>
Nowhere to be found.
</P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Preparation_of_competent_SCV-I_cells"> Preparation of competent SCV-I cells </SPAN></H3><P><B>Investigator:</B> Martin
</P><P><B>Procedure:</B> Competent cells were prepared with the <B><I>S.c.</I> EasyComp Transformation Kit</B> by invitrogen.
</P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Preparative_digest_of_P793_and_P802_with_Sbf1_.28for_integrative_transformation_in_yeast.29"> Preparative digest of P793 and P802 with Sbf1 (for integrative transformation in yeast)</SPAN></H3><P><B>Investigator:</B> Martin
<B>Aim:</B> Integrational vectors with thaumatin (P793) and mOrange (P802) inserts were preparatively digested with Sbf1 to achieve linearization.
<B>Procedure:</B></P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Transformation_in_yeast_with_P793_and_P802_.28digested_with_Sbf1.29"> Transformation in yeast with P793 and P802 (digested with Sbf1)</SPAN></H3><P><B>Investigator:</B> Martin, Alois van de Uckermark
</P><P><B>Procedure:</B> According to the manual with the <B><I>S.c.</I> EasyComp Yeast Transformation Kit</B> by invitrogen.
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Inoculation_of_2_l_SC-U_.282_.25_galactose.29_with_overnight_culture_of_yeast_strain_carrying_pTUM104_thaumatin"> Inoculation of 2 l SC-U (2 % galactose) with overnight culture of yeast strain carrying pTUM104_thaumatin</SPAN></H3><P><B>Investigator:</B> Martin
</P><P><B>Aim:</B> Expression of thaumatin
</P><P><B>Procedure:</B> 2 l SC-U medium with 2 % galactose were inoculated with the cells harvested in 100 ml preparatory culture transformed with pYES_thaumatin.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_E.Coli_over_night_cultures_2"> Miniprep of <I>E.Coli</I> over night cultures </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of experiment:</B></P><P>Isolation of plasmids: B2A_3+ADH1 terminator (1), B1B_2+ADH1 terminator (2), B3D_3+ADH1 (4) terminator for further usage.
</P><P><B>Operational sequence:</B></P><P>The plasmid preparation was done with the Quiagen Plasmid Miniprep Kit as previously described.
Elution was performed with 40µl of elution buffer, heated up to 37°C. Before final centrifugation, the column was incubated for 2 minutes at 37°C.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Sequencing_analysis">Sequencing analysis</SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> Sequence analysis to check the right sequence of the constructs. Miniprep products of B2A, B1B, B3D, B3D5 were sequenced and with the programm genoius pro analysed
</P><UL><LI>B2A: Primer VF2
</LI><LI>B2A: Primer VR
</LI><LI>B1B: Primer VF2
</LI><LI>B1B: Primer VR
</LI><LI>B3D: Primer VF2
</LI><LI>B3D: Primer VR
</LI><LI>B3D5: Primer VF2
</LI><LI>B3D5: Primer VR
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_September_15th"><B>Saturday, September 15th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_P803.2BP804.2C_P803.2BP805.2C_P663.2BP791"> Transformation of <I>E. coli</I> XL1-Blue with ligation products P803+P804, P803+P805, P663+P791 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P803+P804, P803+P805, P663+P791
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation products (P803+P804, P803+P805, P663+P791) and it's negative controls (P803 NK, P663 NK) were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on suitable antibiotic plates (P803+X: CamR, P663+X: AmpR, KanR).
</LI></UL><UL><LI> The rest of the suspension was centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and was plated again on antibiotic plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_of_pTUM104-TEF1-Limonensynthases"> Picking of pTUM104-TEF1-Limonensynthases </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of pTUM104-TEF1-Limonensynthases for brewing experiments and promoter characterization.
</P><P><B>Procedure:</B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preperative_digestion_of_B2A.2C_B1B.2C_B3D_3"> Preperative digestion of B2A, B1B, B3D </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> after the sequences of the constructs were succsesful, we started a preperative digestion of the constructs to fuse them together. Therefor the constructs B2A and B3D will be cut out, by XbaI and PstI and finally purified of the plasmid backbone with gelelectrophoresis
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>30 µl
</TD><TD>B2A, B3D
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>10,5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>30 µl
</TD><TD>B1B
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2 µl
</TD><TD>SpeI (20 U/µl)
</TD></TR><TR><TD>2 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>10,5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_4"> Transformation of <I>E. coli</I></SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with the sequenced constructs CaXMT1, CaMXMT1
</P><P><B>Procedure:</B></P><UL><LI> 20 µl of each ligation products were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_mutations_found_by_sequencing_in_our_gene_constructs_3">  Quick Change mutagenesis to remove mutations found by sequencing in our gene constructs  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P><B>Aim of the experiment: </B>Deletion of found mutations.
</P><P><B>Operational sequence</B></P><P><B>1. PCR general setup</B><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of appropriate Oligo (= 5 pmol)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of appropriate Oligo (= 5 pmol)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI></UL><P><B>2. Used constructs and primers</B></P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Quickchange number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Plasmid number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Geneconstruct</B></TD><TD align="center" style="background:#f0f0f0;"><B>Primer number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Primer</B></TD></TR><TR><TD> QC 1</TD><TD>P783</TD><TD>OMT+ pSB1C3</TD><TD>O96 &amp; O97</TD><TD>a1055g
</TD></TR><TR><TD> QC 2</TD><TD>P138</TD><TD>OMT- pSB1C3</TD><TD>O96 &amp; O97</TD><TD>a1055g
</TD></TR><TR><TD> QC 3</TD><TD>P813</TD><TD>CHS+ pYes Limonen</TD><TD>O104 &amp; O105</TD><TD>c614t
</TD></TR><TR><TD> QC 4</TD><TD>P814</TD><TD>OMT+ pYes Limonen</TD><TD>O96 &amp; O97</TD><TD>a1055g
</TD></TR><TR><TD> QC 5</TD><TD>P822</TD><TD>PAL+ pYes Limonen</TD><TD>O108 &amp; O109</TD><TD>a770g
</TD></TR><TR><TD> QC 6</TD><TD>P823</TD><TD>PAL- pYes Limonen</TD><TD>O108 &amp; O109</TD><TD>a770g
</TD></TR><TR><TD> QC 7</TD><TD>P719</TD><TD>PAL- pYes</TD><TD>O108 &amp; O109</TD><TD>a770g
</TD></TR><TR><TD> QC 8</TD><TD>P818</TD><TD>PAL+ pYes</TD><TD>O106 &amp; O107</TD><TD>t667g
</TD></TR></TBODY></TABLE><UL><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_September_16th"><B>Sunday, September 16th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_P803.2BP804.2C_P803.2BP805.2C_P663.2BP791"> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products P803+P804, P803+P805, P663+P791 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products P803+P804, P803+P805, P663+P791.
</P><P><B>Procedure:</B></P><UL><LI> 4 colonies of each transformation with P803+P804 (CamR), P803+P805 (CamR), P663+P791 (CamR, AmpR) were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and antibiotics. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_pTUM104-TEF1-P-Limonensynthases"> Analytical digestion and gelelectrophoresis of pTUM104-TEF1-P-Limonensynthases </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of Limonens.-pTUM101 to check whether Limonensythase was successfully cloned downstream of the pTUM101 for constitutive expression in yeast.
</P></DIV></DIV><H1><SPAN class="mw-headline" id="Week_15"> Week 15 </SPAN></H1><DIV class="WWeek_15"><H2><SPAN class="mw-headline" id="Monday.2C_September_17th"><B>Monday, September 17th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Picking_of_colonies_with_ligations_806_.2B_810_.2C_P571_.2B_808.2C_P571_.2B_P812.2C_P809_.2B_P791.2C_P807_.2B_P791"> Picking of colonies with ligations 806 + 810 , P571 + 808, P571 + P812, P809 + P791, P807 + P791</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> colonies were picked and transferred to 5 µl LB-medium with 1x Ampicillin ( 806 + 810,P809 + P791, P807 + P791)
</LI></UL><P>and CAM (P571 + 808, P571 + P812)
</P><UL><LI> picked colonies were incubated over night at 37 C
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P492_.2B_P808.2C_P493_.2B_P808.2C_P494_.2B_P808.2C_P806_.2B_P791"> Ligation of P492 + P808, P493 + P808, P494 + P808, P806 + P791</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,09 µl (=100 ng)
</TD><TD>P492
</TD></TR><TR><TD>0,31 µl
</TD><TD>P808/10
</TD></TR><TR><TD>1
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>16,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,74 µl (=100 ng)
</TD><TD>P493
</TD></TR><TR><TD>0,34 µl
</TD><TD>P808/10
</TD></TR><TR><TD>1
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>14,92 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>4,51 µl (=100 ng)
</TD><TD>P494
</TD></TR><TR><TD>0,31 µl
</TD><TD>P808/10
</TD></TR><TR><TD>1
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>12,18 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>0,82 µl (=100 ng)
</TD><TD>P806
</TD></TR><TR><TD>0,31 µl
</TD><TD>P791
</TD></TR><TR><TD>1
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>14,3 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> DNA was heated for 5 min at 60 C and cooled on ice before T4-ligase buffer was added
</LI><LI> Cycled ligation was done at following conditions
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_Xl1-blue_competent_cells_with_ligation_batches"> Transformation of <I>E.coli</I> Xl1-blue competent cells with ligation batches</SPAN></H3><P><B>Investigator: Georg</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on chloramphenicole plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new CAM plates.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_expansion_E..C2.A0coli_XL-Blue_cells_culture.2C_transformated_with_ligation_products_P803.2BP804.2C_P803.2BP805.2C_P663.2BP791"> Miniprep of overnight expansion <I>E. coli</I> XL-Blue cells culture, transformated with ligation products P803+P804, P803+P805, P663+P791 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of overnight expansion <I>E. coli</I> XL-Blue cells culture, transformated with ligation products P803+P804, P803+P805, P663+P791.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done after manufacturer's protocol (P840-P843: P803+P804, P844-P847: P803+P805, P848-P851: P663+P791). (QIAGEN - QIAprep Spin Miniprep Kit)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P840-P851"> Analytical digestion and gelelectrophoresis of P840-P851 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P840-P851.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with EcoRI-HF+SpeI-HF+PvuII:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>26 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2.6 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>3.25 µl
</TD><TD>EcoRI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>3.25 µl
</TD><TD>SpeI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>192.4 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=227.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix were added to 2.5 µl of plasmid DNA of P840-P851.
</LI></UL><UL><LI> Analytical digestion reaction mix was incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 2 µl of DNA loading buffer (10x) was added to the 20 µl of reaction mix after digestion.
</LI></UL><UL><LI> 10 µl of these mixes were loaded in the gel pockets.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 90 min on an 0.5% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P840
</TD><TD>P841
</TD><TD>P842
</TD><TD>P843
</TD><TD>P844
</TD><TD>P845
</TD><TD>P846
</TD><TD>P847
</TD></TR><TR><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P840
</TD><TD>P841
</TD><TD>P842
</TD><TD>P843
</TD></TR><TR><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P304.2C_P564.2C_P515"> Preparative digestion of P304, P564, P515 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P304 (EcoRI-HF+XbaI), P564 (EcoRI-HF+SpeI-HF), P515 (XbaI+PstI-HF).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for digestion of P304 with EcoRI-HF+XbaI:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P304
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P515 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P515
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P564 with EcoRI-HF+SpeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P564
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Preparative digestion was performed for 2.5 h at 37 °C.
</LI></UL><UL><LI> Digested samples were freezed at -20 °C for preparative gelelectrophoresis on the next days.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Second_preparative_gelelectrophoresis_of_P804.2C_P805"> Second preparative gelelectrophoresis of P804, P805 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Second preparative gelelectrophoresis of P804, P805, to remove vector contamination because first gelelectrophoresis was not efficient enough to purificate the insert.
</P><P><B>Procedure:</B></P><UL><LI> The samples was loaded with 4 µl of DNA loading buffer (10x) and preparative gelelectrophoresis was performed on an 1% agarose gel at 70 V for 2 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp ladder
</TD><TD>P804
</TD><TD>P805
</TD></TR><TR><TD>Upper band was cut out
</TD><TD>Upper band was cut out
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_P803.2BP804.2C_P803.2BP805"> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products P803+P804, P803+P805 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products P803+P804, P803+P805.
</P><P><B>Procedure:</B></P><UL><LI> 11 colonies of each transformation with P803+P804 (CamR), P803+P805 (CamR) were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and chlorampenicol. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P803.2BP852.2C_P803.2BP853"> Cycled ligation of P803+P852, P803+P853 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Cycled ligation of P803+P852, P803+P853, in order to transform the next day, if the newly picked clones are again negative.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P803+P852
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.46 µl
</TD><TD>P803 (68.4 ng/µl, 4290 bp)
</TD></TR><TR><TD>9.44 µl
</TD><TD>P852 (28.5 ng/µl, 3848 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>6.1 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P803+P853
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.46 µl
</TD><TD>P803 (68.4 ng/µl, 4290 bp)
</TD></TR><TR><TD>4.37 µl
</TD><TD>P853 (46.9 ng/µl, 4013 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>9.59 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batches and was called P803 NK.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Over_night_cultures_of_INVSC-1_yeast_cultures_transformed_with_caffeine_synthesis_gene-_constructs"> Over night cultures of INVSC-1 yeast cultures transformed with caffeine synthesis gene- constructs </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the prepare of over night cultures previously to an expression of the caffeine synthesis genes CaXMT1, CaMXMT1 and CaDXMT1. eGFP was used as positive control. 
</P><P><B>Operational sequence:</B></P><P>If possible, one single colony of plated transformants was used for inoculation of 20ml SC-U 2% glucose medium. Otherwise, a pipet dip was taken (if colony density was to high). Afterwards, the flasks were incubated at 30°C for about 12h.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Over_night_cultures_of_INVSC-1_yeast_cultures_transformed_with_caffeine_synthesis_gene-_constructs_2"> Over night cultures of INVSC-1 yeast cultures transformed with caffeine synthesis gene- constructs </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the prepare of over night cultures previously to an expression of the caffeine synthesis genes CaXMT1, CaMXMT1 and CaDXMT1. eGFP was used as positive control. 
</P><P><B>Operational sequence:</B></P><P>If possible, one single colony of plated transformants was used for inoculation of 20ml SC-U 2% glucose medium. Otherwise, a pipet dip was taken (if colony density was to high). Afterwards, the flasks were incubated at 30°C for about 12h.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Over_night_cultures_of_INVSC-1_yeast_cultures_transformed_with_caffeine_synthesis_gene-_constructs_3"> Over night cultures of INVSC-1 yeast cultures transformed with caffeine synthesis gene- constructs </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the prepare of over night cultures previously to an expression of the caffeine synthesis genes CaXMT1, CaMXMT1 and CaDXMT1. eGFP was used as positive control. 
</P><P><B>Operational sequence:</B></P><P>If possible, one single colony of plated transformants was used for inoculation of 20ml SC-U 2% glucose medium. Otherwise, a pipet dip was taken (if colony density was to high). Afterwards, the flasks were incubated at 30°C for about 12h.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Yeast_Expression:_S._cerevisiae_INVSc1_CaXMT1.2C_CaMXMT1.2C_CaDXMT1_and_eGFP_.28200_ml.29"> Yeast Expression: <I>S. cerevisiae</I> INVSc1 CaXMT1, CaMXMT1, CaDXMT1 and eGFP (200 ml) </SPAN></H3><P><B>Investigator:</B> Roman, Saskia
</P><P><B>Aim:</B>
Induction of protein expression of CaXMT1, CaMXMT1, CaDXMT1 and eGFP in a larger scale (200ml) to have enough protein for enzymatic analysis
</P><P><B>Operational sequence:</B></P><P>At first, the OD600 of the over night cultures from today (17.09.2012) were measured:
</P><UL><LI> CaXMT1: 7.6/ml
</LI><LI> CaMXMT1: 7.2/ml
</LI><LI> CaMXMT1: 8.9/ml
</LI><LI> eGFP: 8.8/ml
</LI></UL><P>(1/10 dilutions have been prepared for measurment)
</P><P>To obtain an OD of 0,4/ml in 200 ml of SC -URA 2% gal medium for CaXMT1, CaMXMT1, CaDXMT1 and 50ml of SC-URA 2% gal medium for eGFP, 10 ml, 11 ml, 9 ml and 2.27 ml of the over night cultures were centrifuged at 1500xg for 5 minutes at 4°C. Afterwards, the pellet was resuspended in 1ml SC -URA 2% gal. The suspension was then given to 49ml SC-URA 2% gal medium, followed by incubation at 180rpm and 30°C.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_ligation_products_.28ligation_14th_September.29_and_of_clone_2_.285.29_.28Trafo_8th_September.29"> Miniprep of ligation products (ligation 14th September) and of clone 2 (5) (Trafo 8th September) </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Concentrations</B></P><UL><LI>TEF2-P/LS/CYC-T (clone 1) 1: 113 ng/µl
</LI><LI>TEF2-P/LS/CYC-T (clone 1) 2: 133.5 ng/µl
</LI><LI>TEF2-P/LS/CYC-T (clone 1) 3: 121 ng/µl
</LI><LI>TEF2-P/LS/CYC-T (clone 1) 4: 148 ng/µl
</LI><LI>TEF1-P/LS/CYC-T (clone 2) 1: 68 ng/µl
</LI><LI>TEF1-P/LS/CYC-T (clone 2) 2: 57 ng/µl
</LI><LI>TEF1-P/LS/CYC-T (clone 2) 3: 143.5 ng/µl
</LI><LI>TEF1-P/LS/CYC-T (clone 2) 4: 88 ng/µl
</LI><LI>TEF1-P/LS/TEF1-T (clone 2) 1: 148.5 ng/µl
</LI><LI>TEF1-P/LS/TEF1-T (clone 2) 2: 153 ng/µl
</LI><LI>TEF1-P/LS/TEF1-T (clone 2) 3: 138 ng/µl
</LI><LI>TEF1-P/LS/TEF1-T (clone 2) 4: 84 ng/µl
</LI></UL><UL><LI>clone 2(5) 1: 115 ng/µl
</LI><LI>clone 2(5) 2: 147 ng/µl
</LI></UL><P><B>probes are stored in an disposal bag in the lower drawer (label: Miniprep 17.09. Andrea)</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_miniprep_products_2"> Analytical restriction digest of miniprep products </SPAN></H3><P><B>Investigator:</B> Lara
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoR1
</TD></TR><TR><TD>0.25 µl
</TD><TD>Pst1- HF
</TD></TR><TR><TD>2 µl
</TD><TD>NEB 4
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>incubation: 3 hours, 37°C
</LI></UL><P><B>Gelelectrophoresis:</B></P><P>gel 1: marker, 1(1), 1(2), 1(3), 1(4), 2(1), 2(2), 2(3)
</P><P>gel 2: marker, 2(4), 2-TEF(1), 2-TEF(2), 2-TEF(3), 2-TEF(4)
</P><P><B>lengths:</B></P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Miniprep_of_clones_of_tranformations_done_on_saturday_15th_september"> Miniprep of clones of tranformations done on saturday 15th september </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Get purified DNA for transformation in yeast and stem preservation.
</P><P><B>Procedure:</B></P><UL><LI> Using a Quiagen kit a miniprep of the overday culture was done.
</LI><LI> The resulting purified DNA was aliquoted in new tubes labeled as follows:
</LI></UL><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Plasmid number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Content</B></TD><TD align="center" style="background:#f0f0f0;"><B>Concentration</B></TD></TR><TR><TD> P831</TD><TD>QC1 P137 OMT+pSB1C3 sdm a1055g</TD><TD>64,4  ng/µl
</TD></TR><TR><TD> P832</TD><TD>QC2 P138 OMT-pSB1C3 sdm a1055g</TD><TD>70,1 ng/µl
</TD></TR><TR><TD> P833</TD><TD>QC3 P813 CHS+pYes Limonen c614t</TD><TD>164,5 ng/µl
</TD></TR><TR><TD> P834</TD><TD>QC4 P814 OMT+ pYes Limonen a1055g</TD><TD>105,4 ng/µl
</TD></TR><TR><TD> P835</TD><TD>QC5 P822 PAL+pYes Limonen a770g</TD><TD>69,8 ng/µl
</TD></TR><TR><TD> P836</TD><TD>QC6 P823 PAL-pYes Limonen a770g</TD><TD>58,4 ng/µl
</TD></TR><TR><TD> P837</TD><TD>QC6 P823 PAL-pYes Limonen a770g Klon2</TD><TD>75,3 ng/µl
</TD></TR><TR><TD> P838</TD><TD>QC7 P719 PAL-pYes a770g</TD><TD>68,9 ng/µl
</TD></TR><TR><TD> P839</TD><TD>QC8 P818 PAL+pYes t667g</TD><TD>75,4 ng/µl
</TD></TR></TBODY></TABLE><P>As the side directed mutagenesis of P823 lead to a low number of colonies it will be repeated.
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Repeated_quick_change_mutagenesis_to_remove_mutations_found_by_sequencing_in_our_gene_constructs">  Repeated quick change mutagenesis to remove mutations found by sequencing in our gene constructs  </SPAN></H3><P><B>Investigator: Ingmar</B></P><P><B>Aim of the experiment: </B>Deletion of found mutations.
</P><P><B>Operational sequence</B></P><P><B>1. PCR general setup</B><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of appropriate Oligo (= 5 pmol)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>DNA template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of appropriate Oligo (= 5 pmol)
</TD></TR><TR><TD>17 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 6 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI></UL><P><B>2. Used constructs and primers</B></P><TABLE><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>Quickchange number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Plasmid number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Geneconstruct</B></TD><TD align="center" style="background:#f0f0f0;"><B>Primer number</B></TD><TD align="center" style="background:#f0f0f0;"><B>Primer</B></TD></TR><TR><TD> QC 1</TD><TD>P823</TD><TD>PAL- pYes Limonen c869t </TD><TD>O108 &amp; O109</TD><TD>a770g
</TD></TR><TR><TD> QC 2</TD><TD>P838</TD><TD>PAL- pYes sdm a770g </TD><TD>O110 &amp; O111</TD><TD>c869t
</TD></TR></TBODY></TABLE><UL><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Amp-LB-plate
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_September_18th"><B>Tuesday, September 18th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_picked_colonies_from_17.9">Miniprep of picked colonies from 17.9 </SPAN></H3><P><B>Investigator:Georg</B></P><UL><LI> Miniprep was performed according to quiaprep spin miniprep kit protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_picked_colonies">Analytical digestion and gelelectrophoresis of picked colonies</SPAN></H3><P><B>Investigator:Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>6,5 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>6,5 µl
</TD><TD>Xba
</TD></TR><TR><TD>52 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>5,2 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>384,8 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=455 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17,5 µl of reaction batch were mixed with 2,5 µl of DNA
</LI><LI> Digestion took 1 h at 37 C
</LI></UL><UL><LI> Gelelectrophoresis was run 1 h at 90 V
</LI></UL><UL><LI> ligation of pTUM101 and pTUM103 with luciferase and limonensynthase was succesful
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_expansion_E..C2.A0coli_XL-Blue_cells_culture.2C_transformated_with_ligation_products_P803.2BP804.2C_P803.2BP805"> Miniprep of overnight expansion <I>E. coli</I> XL-Blue cells culture, transformated with ligation products P803+P804, P803+P805 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of overnight expansion <I>E. coli</I> XL-Blue cells culture, transformated with ligation products P803+P804, P803+P805.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done after manufacturer's protocol. (QIAGEN - QIAprep Spin Miniprep Kit)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_ligation_products_P803.2BP804.2C_P803.2BP805_from_overnight_expansion_culture"> Analytical digestion and gelelectrophoresis of ligation products P803+P804, P803+P805 from overnight expansion culture </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of ligation products P803+P804, P803+P805 from overnight expansion culture.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>48 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>4.8 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>6 µl
</TD><TD>XbaI (20 U/µl) (NEB)
</TD></TR><TR><TD>6 µl
</TD><TD>PstI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>355.2 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=420 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix were added to 2.5 µl of plasmid DNA.
</LI></UL><UL><LI> Analytical digestion reaction mix was incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 2 µl of DNA loading buffer (10x) was added to the 20 µl of reaction mix after digestion.
</LI></UL><UL><LI> 10 µl of these mixes were loaded in the gel pockets.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 90 min on an 0.5% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P858
</TD><TD>P859
</TD><TD>P860
</TD><TD>P861
</TD><TD>P862
</TD></TR><TR><TD>Ligation successful!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation failed!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P863
</TD><TD>P864
</TD><TD>P865
</TD><TD>P866
</TD><TD>P867
</TD><TD>P868
</TD><TD>P869
</TD><TD>P870
</TD><TD>P871
</TD><TD>P872
</TD></TR><TR><TD>Ligation successful!
</TD><TD>Ligation failed!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_gelelectrophoresis_of_preparative_digested_P304.2C_P564.2C_P515"> Preparative gelelectrophoresis of preparative digested P304, P564, P515 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative gelelectrophoresis of preparative digested P304, P564, P515.
</P><P><B>Procedure:</B></P><UL><LI> The stored digestion batches were gently thawed on ice.
</LI></UL><UL><LI> 4.44 µl of DNA loading buffer (10x) were added to each digestion batch after incubation.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 70 V on a 0.5% agarose gel for 2 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P304 (EcoRI-HF+XbaI)
</TD><TD>1 kbp ladder
</TD><TD>P515 (XbaI+PstI-HF)
</TD><TD>P654 (EcoRI-HF+SpeI-HF)
</TD></TR><TR><TD>Band was cut out
</TD><TD>Upper band was cut out
</TD><TD>Band was cut out
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_P850.2C_P860.2C_P870"> Preparative digestion of P850, P860, P870 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P850 (EcoRI-HF+SpeI-HF), P860 (EcoRI-HF+SpeI-HF), P870 (EcoRI-HF+SpeI-HF).
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch for digestion of P850 with EcoRI-HF+SpeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P850
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P860 with EcoRI-HF+SpeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P860
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for digestion of P870 with EcoRI-HF+SpeI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>P870
</TD></TR><TR><TD>4 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA  (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>EcoRI-HF (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI-HF (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Preparative digestion was performed for 2.5 h at 37 °C.
</LI></UL><UL><LI> 4.44 µl of DNA loading buffer (10x) were added to each digestion batch after incubation.
</LI></UL><UL><LI> Preparative gelelectrophoresis was performed at 70 V on a 0.5% agarose gel for 2 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P850 (EcoRI-HF+SpeI-HF)
</TD><TD>1 kbp ladder
</TD><TD>P860 (EcoRI-HF+SpeI-HF)
</TD><TD>P870 (EcoRI-HF+SpeI-HF)
</TD></TR><TR><TD>Lower was cut out
</TD><TD>Upper band was cut out
</TD><TD>Upper band was cut out
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_over_night_cultures"> Miniprep of over night cultures </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the isolation of pSB1C3 plasmids containing our three expression cartridges [Tef2PROM-CaXMT1-Adh1TER], [Tef1PROM-CaMXMT1-Adh1TER] and [Tef2PROM-CaDXMT1-Adh1TER]. After the isolation of these plasmids, we will perform a preparative digest with suitable enzymes, to generate the &quot;caffeine synthesis device&quot;.
</P><P><B>Operational sequence:</B></P><P>The plasmid isolation was done as previously described. Elution was carried out by using 40µl of elution buffer heated up to 37°C and incubating for 1 minute before the final centrifugation.
</P><P>Afterwards, the concentration was determined with NanoDrop:
</P><UL><LI> pSB1C3_[Tef2PROM-CaXMT1-Adh1TER]:  222,6 ng/µl  
</LI><LI> pSB1C3_[Tef1PROM-CaMXMT1-Adh1TER]: 180,3 ng/µl
</LI><LI> pSB1C3_[Tef2PROM-CaDXMT1-Adh1TER]: 168   ng/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparative_digest_of_the_three_expression_cartridges_.5BTef2PROM-CaXMT1-Adh1TER.5D.2C_.5BTef1PROM-CaMXMT1-Adh1TER.5D_and_.5BTef2PROM-CaDXMT1-Adh1TER.5D"> Preparative digest of the three expression cartridges [Tef2PROM-CaXMT1-Adh1TER], [Tef1PROM-CaMXMT1-Adh1TER] and [Tef2PROM-CaDXMT1-Adh1TER] </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Preparative digest previously to ligation (production of the &quot;caffeine synthesis device&quot;). To do this, we want to ligate the second and third expression cartridges consecutively on the first one. Since the length of the expression cassetts is almost the same as the vector backbone (pSB1C3), we make also use of the restriction enzyme Eco53KI which cuts that backbone, thereby enabling a satisfying separation on the agarose gel.
</P><P><B>Operational sequence:</B></P><P>Reaction batches (for pSB1C3 plasmids with CaMXMT1 and CaDXMT1 cartridges)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Plasmid DNA
</TD><TD>20µl (3,6 and 3,4 µg)
</TD></TR><TR><TD>Xba1 RE
</TD><TD>1µl
</TD></TR><TR><TD>Pst1-HF RE
</TD><TD>1µl
</TD></TR><TR><TD>Eco52KI RE
</TD><TD>2µl
</TD></TR><TR><TD>10x NEBuffer 4
</TD><TD>4µl
</TD></TR><TR><TD>100x BSA
</TD><TD>0,4µl
</TD></TR><TR><TD>ddH2O
</TD><TD>11,6µl
</TD></TR></TBODY></TABLE><P>Reaction batch (for pSB1C3 containing CaXMT1 expression cartridge)
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Reagent</B></TD><TD><B>Volume</B></TD></TR><TR><TD>Plasmid DNA
</TD><TD>20µl (4,4 µg)
</TD></TR><TR><TD>Spe1-HF RE
</TD><TD>1µl
</TD></TR><TR><TD>Pst1-HF Re
</TD><TD>1µl
</TD></TR><TR><TD>10x NEBuffer 4
</TD><TD>4µl
</TD></TR><TR><TD>100x BSA
</TD><TD>0,4µl
</TD></TR><TR><TD>ddH2O
</TD><TD>13,6µl
</TD></TR></TBODY></TABLE><P>The reaction batches were incubated at 37°C for 2,5 hours. Afterwards, the fragments were separated by gel electrophoresis (90V, 1,5h).
</P><P><B>Gel pictures:</B></P><P><B>From left to right:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>DNA Ladder
</TD><TD>pSB1C3 with expression cartridge Tef2PROM-CaXMT1-Adh1TER digested with Spe1-HF and Pst1-HF
</TD><TD>pSB1C3 with expression cartridge Tef1PROM-CaMXMT1-Adh1TER digested with Xba1, Pst1-HF and Eco53KI
</TD><TD>DNA Ladder
</TD></TR></TBODY></TABLE><P><B>From left to right:</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>pSB1C3 with expression cartridge Tef2PROM-CaDXMT1-Adh1TER digested with Xba1, Pst1-HF and Eco53KI
</TD></TR></TBODY></TABLE><P><B>Note:</B>
Expected lenghts are:
</P><UL><LI> pSB1C3 with CaXMT1 expression cartridge: ca. 4200bp
</LI><LI> pSB1C3 with CaMXMT1 expression cartridge (triple digest):
</LI><LI> ca. 2030bp (expression cartridge)
</LI><LI> ca. 900bp (pSB1C3 backbone fragment 1)
</LI><LI> ca. 1100bp (pSB1C3 backbone fragment 2)
</LI></UL><LI> pSB1C3 with CaDXMT1 expression cartridge (triple digest):
</LI><LI> ca. 2050bp (expression cartridge)
</LI><LI> ca. 900bp (pSB1C3 backbone fragment 1)
</LI><LI> ca. 1100bp (pSB1C3 backbone fragment 2)
</LI><P>The required bands were cut out and purified by gel extraction.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Gel-_extraction_of_separated_fragments"> Gel- extraction of separated fragments </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B>Extraction of DNA from agarose gel
</P><UL><LI> Gel extraction was done using the Qiagen gel extraction kit 
</LI></UL><P><B>Determined concentrations (NanoDrop):</B></P><UL><LI> CaXMT1 with pSB1C3 backbone: 48 ng/µl
</LI><LI> CaMXMT1- expression cartridge: 21 ng/µl
</LI><LI> CaDXMT1- expression cartridge: 24ng/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Ligation_of_preparative_digestion"> Ligation of preparative digestion </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> ligation was performed for CaXMT1+CaMXMT1, incubated at 16 °C over night
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Yeast_expression:_Pelletizing_16_h_after_expression_induction"> Yeast expression: Pelletizing 16 h after expression induction </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was to pellet the 200ml cell suspension 16h after expression induction, as well as to wash the pellet with water, followed by storage at -20°C until further usage.
</P><P><B>Operational sequence:</B></P><P>At first, the OD600 of the suspensions was determined:
</P><UL><LI> CaXMT1: n.a.
</LI><LI> CaMXMT1: 2,9/ml
</LI><LI> CaDXMT1: n.a.
</LI><LI> eGFP: 8,9/ml
</LI></UL><P><B>Note:</B></P><P>Unfortunately, (almost) no colonies have grown of CaXMT1 and CaDXMT1. This is a bit strange, because the same medium was used at the inoculation and all colonies had grown in the SC-U 2% glucose medium.
The induction of the expression will be repeated with these transformants.
</P><P>The taken samples were centrifuged at 1500xg for 5 minutes at 4°C (200ml in several steps, eGFP 50ml in one step) and washed with 10 and 5 ml water, respectively. Afterwards, the suspension was centrifuged again and resuspended in 2ml water. The suspension was repelleted again and stored at -80°C until further usage.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_over_night_cultures"> Preparation of over night cultures</SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the preparation of over night cultures of <I>E.Coli</I>- cells, having been transformed with the CaXMT1-, CaMXMT1- and CaDXMT1- expression cartridges, in order to have a sufficient amount of the plasmids (i.e. composite parts).
</P><P><B>Operational sequence:</B></P><P>A single colony grown on LB agar plates with chloramphenicol was picked and used for inoculation of 6ml liquid LB medium containing 6µl of a chloramphenicol solution (final concentration: 30µg/ml). Incubation was done at 37°C over night with 180 rpm.
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Harvesting_of_cells_.28S._cervisiae_.2B_pTUM104_Preprothaumatin.29"> Harvesting of cells (<I>S. cervisiae</I> + pTUM104_Preprothaumatin) </SPAN></H3><P><B>Investigator:</B> Martin, Alois
</P><P><B>Aim of the experiment:</B></P><UL><LI> 5. Determine the OD600 =&gt; 6.7. 6. Centrifuge the cells at 1,500 g for 5 minutes at 4°C. 7. Decant the <B>supernatant</B> (stored at 4°C; secreted thaumatin?). Resuspend cells in 8 ml of sterile water. 8. Transfer cells to a sterile centrifuge tube. Centrifuge samples for 30 seconds at top speed in the microcentrifuge. 9. Remove the supernatant. 10. Store the cell pellets at –80°C until ready to use. Proceed to the next section to prepare cell lysates to detect your recombinant protein (see next page).
</LI></UL><P>Cell lysates:
</P><UL><LI> 2. Resuspend fresh or frozen cell pellets in 10 ml of breaking buffer. Centrifuge at 1,500 g for 5 minutes at 4°C to pellet cells. 3. Remove supernatant and resuspend the cells in a volume (180 ml) of breaking buffer to obtain an OD600 of 50–100. Use the OD600 determined in Step 5, previous page, to calculate the appropriate volume of breaking buffer to use. 4. Add an equal volume of acid-washed glass beads. 5. Vortex mixture for 30 seconds, followed by 30 seconds on ice. Repeat eight times. 6. Centrifuge in a centrifuge for 10 minutes at maximum speed. 7. Remove supernatant and transfer to a fresh microcentrifuge tube.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preperative_restriction_digest_of_miniprep_products"> Preperative restriction digest of miniprep products </SPAN></H3><P><B>Investigator:</B> Katrin, Andrea, Lara
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>1 µl
</TD><TD>EcoR1
</TD></TR><TR><TD>1 µl
</TD><TD>Spe1- HF
</TD></TR><TR><TD>4 µl
</TD><TD>NEB 4
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>incubation: 4 hours, 37°C
</LI></UL><P><B>Gelelectrophoresis:</B></P><P><B>lengths:</B></P><UL><LI> 1(4): 2800 bp
</LI><LI> 2(3): 2400 bp
</LI><LI> 2TEF(2): 2550 bp
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Enzyme_assay_and_limonene_extraction">Enzyme assay and limonene extraction</SPAN></H3><P><B>Investigators: Andrea, Lara</B></P><P><B>Aim:</B> Test functionality of purified enzyme.
</P><P><B>Enzyme assay</B></P><P><B>All reactions and extractions were done in glass containers because limonene is very non-polar.</B></P><P>
Enzyme assay was carried out in a total volume of 500 µl containing buffer (25 mM Tris-Cl, pH 7.5, 5 % glycerol, 1 mM DTT) supplemented with cofactors (10 mM MgCl2, 1 mg/ml BSA).
</P><UL><LI> stock solutions were prepared:
</LI></UL><P>1. 50 mM Tris-HCL pH 7.5 + 10 % glycerol
</P><P>2. 1 M MgCl2
</P><P>
50 µM substrate (geranyl pyrophosphate, dissolved in DMSO) and 10 µg purified recombinant enzyme (extracted limonene synthase, after dialysis) were added.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>250 µl
</TD><TD>2 x Buffer (Tris-HCl, Glycerol)
</TD></TR><TR><TD>5 µl
</TD><TD>1 M MgCl2
</TD></TR><TR><TD>0.5 µl
</TD><TD>1 M DTT
</TD></TR><TR><TD>5 µl
</TD><TD>100 mg/ml BSA
</TD></TR><TR><TD>2.5 µl
</TD><TD>GPP (10 mM)
</TD></TR><TR><TD>18.5 µl
</TD><TD>Enzyme (10 µg)
</TD></TR><TR><TD>218.5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>The mixture was overlaid with 500 µl pentane (carefully, phases should not mix) and incubated at room temperature for 15 minutes. The reaction was stopped by vigorous mixing and centrifugation (5 min) to separate phases. The upper phase (solvent phase) was taken and put into a pasteur pipette which contained glass wool with Na2SO4 (to dry the solvent phase). Since the amount of pentane seemed to be insufficient (no solvent reached the GC glass container), the next two extractions were done with 1 ml of pentane each. Afterwards the combined extracts were reduced to approximately 300 µl under a stream of nitrogen.
</P><P>3 replicates of negative control (reaction batch without enzyme) and enzymatic reaction batch were done.
</P><P>
The pentane extracts were analyzed with gas chromatography- mass spectrometry to identify the enzymatically formed products. An aliquot of each sample (0.5 µl) was injected into &quot;5890 Series II GC&quot; coupled to a &quot;Finnigan Mat 55 S MS&quot;.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_yeast_cultures_for_large_scale_expression_.281l.29_1"> Preparation of yeast cultures for large scale expression (1l) 1</SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the preparation of three 30ml yeast cultures (in selective SC-U 2% glucose medium), based on yeast clones transformed with pTUM104 vectors, each containing one of the three genes CaXMT1, CaMXMT1 and CaDXMT1 for having enough crude extract for further analysis. 
</P><P><B>Operational sequence:</B></P><P>If possible, a single colony which had grown on selective SC-U 2%glucose agar plates was used for inoculation of the 30ml liquid medium. If the density of the colonies was to high, a small amount (pipett dip) was taken. 
The generated cell suspensions were incubated at 30°C over night.
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_September_19th"><B>Wednesday, September 19th</B></SPAN></H2><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_picked_colonies_.28P806.2B_P791.29"> Miniprep of picked colonies (P806+ P791)</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Miniprep was performed according to quiaprep Spin Miniprep kit protocoll
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_of_.28P806.2B_P791.29"> Analytical digestion of (P806+ P791)</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0,75
</TD><TD>XbaI
</TD></TR><TR><TD>0,75 µl
</TD><TD>PstI-HF (NEB)
</TD></TR><TR><TD>6µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0,6 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>44,4 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=52,5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> to 2,5 ml of DNA 17,5 µl reaction batch were added
</LI><LI> Digestion took place at 37°C for 1 h
</LI></UL><UL><LI>ligation of P806 (digested pTUM102 ) and P791 (Renilla luciferase) failed
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_of_TEF1-T_in_psb1c3_and_ADH1-T_in_psb1c3_with_EcorI_and_SpeI"> Preparative digestion of TEF1-T in psb1c3 and ADH1-T in psb1c3 with EcorI and SpeI</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Reaction batch for digestion:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>EcorI-HF (NEB)
</TD></TR><TR><TD>2 µl
</TD><TD>SpeI-HF (Neb)
</TD></TR><TR><TD>8 µl
</TD><TD>NEB-4 buffer
</TD></TR><TR><TD>0.8 µl
</TD><TD>100 x BSA (NEB)
</TD></TR><TR><TD>29,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 20 µl reaction batch were added to 20 µl of DNA
</LI><LI> Digestion then was performed for 3 hours at 37 C
</LI></UL><P><A href="/File:20120919_Prep._Dig_TEF1-T_und_Psb1c3_mit_EcorI_und_SpeI.png" title="File:20120919 Prep. Dig TEF1-T und Psb1c3 mit EcorI und SpeI.png">File:20120919 Prep. Dig TEF1-T und Psb1c3 mit EcorI und SpeI.png</A></P></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P806_.2B_P791_and_P492_.2B_P791">Ligation of P806 + P791 and P492 + P791</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,89 µl
</TD><TD>P791
</TD></TR><TR><TD>0,41 µl (=100 ng)
</TD><TD>P806
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>6,2 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>1,09 µl
</TD><TD>P492
</TD></TR><TR><TD>7,81 µl (=100 ng)
</TD><TD>P791
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>4,05 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_gelelectrophoresis_of_P874.2C_P877.2C_P878"> Preparative gelelectrophoresis of P874, P877, P878 </SPAN></H3><P><B>Investigator:</B> Georg
</P><P><B>Aim of the experiment:</B> Preparative gelelectrophoresis of P874, P877, P878 (2nd time) in order to completely get rid of the backbone contamination.
</P><P><B>Procedure:</B></P><UL><LI> Second preparative gelelectrophoresis of P874, P877, P878 was performed on an 0.5% agarose gel at 70 V for 3 h.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>P874
</TD><TD>1 kbp ladder
</TD><TD>P877
</TD><TD>P878
</TD></TR><TR><TD>Band was cut out
</TD><TD>Band was cut out
</TD><TD>Band was cut out
</TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Agarose_gel_extraction"> Agarose gel extraction </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Extraction of DNA from agarose gel
</P><P>concentrations:
</P><UL><LI> 1 (4): 5,3 ng/µl
</LI><LI> 2 (3): 14,7 ng/µl
</LI><LI> 2 TEF 2: 14,9 ng/µl
</LI></UL><P>Gel extraction was done using the Qiagen gel extraction kit
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparing_of_cultures_for_toxicity_test"> Preparing of cultures for toxicity test</SPAN></H3><P><B>Investigators:</B> Andrea, Lara
</P><UL><LI> three cultures were prepared (in 200 ml YPD medium): S. cerevisae (lab strain), brewing yeast, supermarket yeast (&quot;wieninger hefe&quot;)
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_promotor.2FLS.2Fterminator_constructs_with_integration_vector">Ligation of promotor/LS/terminator constructs with integration vector </SPAN></H3><P><B>Investigators:</B> Katrin, Andrea, Lara
</P><P><B>1(4))</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P649 (integration vector; digested with EcoR1+Spe)
</TD><TD>0.23 µl
</TD></TR><TR><TD>1(4) (LS construct, digested with EcoR1+Spe)
</TD><TD>7.77 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>2(3))</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P649 (integration vector; digested with EcoR1+Spe)
</TD><TD>0.7 µl
</TD></TR><TR><TD>2(3) (LS construct, digested with EcoR1+Spe)
</TD><TD>7.3 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><P><B>2TEF(2))</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P649 (integration vector; digested with EcoR1+Spe)
</TD><TD>0.63 µl
</TD></TR><TR><TD>2TEF(2) (LS construct, digested with EcoR1+Spe)
</TD><TD>7.33 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Sniffing_test_of_Limonene"> Sniffing test of Limonene </SPAN></H3><P><B>Investigator:</B> Lara, Andrea
</P><P><B>Aim of the experiment:</B> 8 persons (Lara, Katrin, Georg, Alois, Martin, Volker, Ingmar, Prof. Skerra) described and differed the smell of 6 probes
</P><P><B>probes:</B></P><UL><LI> 1: <B>positive</B>, SC medium + Galactose, PCR1 (Citrus) (Klon 29) with consensus sequence
</LI><LI> 2: <B>Positive control</B>, 3 µl Limonene in 10 ml SC medium + Galactose 
</LI><LI> 3: <B>negative</B>, SC medium + Uracile + Glucose, not transformed
</LI><LI> 4: <B>negative</B>, SC medium + Glucose, PCR1 (Citrus) (Klon 29), not induced
</LI><LI> 5: <B>positive</B>, SC medium + Galactose, PCR2 (Citrus) (P542) without consensus sequence
</LI><LI> 6: <B>positive</B>, SC medium + Galactose + GPP (10mM, 25µl in 50ml), PCR1 (Citrus) (Klon 29) with consensus sequence
</LI></UL><P><B>results, smells like ... :</B></P><UL><LI> 1) yeast, alcoholic, fruity, sweet, medium, wine
</LI><LI> 2) wood, resin, condiment, limonene
</LI><LI> 3) fruity, wine, carbonic acid, malic acid, similar to 1
</LI><LI> 4) fruity, wine, malic acid, similar to 3
</LI><LI> 5) yeast, vinegar
</LI><LI> 6) very fruity, wine, a bit resin
</LI></UL><P><B>conclusion:</B></P><UL><LI> positive and negative probes could not differed clearly
</LI><LI> positive control was identified often
</LI><LI> addition of GPP led to a stronger limonene recognition than the other positive probes
</LI></UL><P><B>cell solution were centrifuged and the pellet were stored at -20°C</B></P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Picking_of_36_clones"> Picking of 36 clones </SPAN></H3><P><B>Investigators:</B> Martin, Alois
</P><P><B>Aim:</B>  To study integration efficiency, we kept track of 36 single clones, which were studied over a time course of 48, respectively 96 hours.
</P><P><B>Procedure:</B> Two 96s wells were covered in aluminum foil and autoclaved. 24 (respectively 12) chambers of one well were each filled with 150µl YPD medium and inoculated with a single clone that was picked from the transformation plate. After 12h the cells were passaged into a new well with new medium.
</P></DIV></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Dialysis_against_20_mM_MES.2C_pH_6.0_.28buffer_1.29_for_cation_exchange_chromatography"> Dialysis against 20 mM MES, pH 6.0 (buffer 1) for cation exchange chromatography </SPAN></H3><P><B>Investigators:</B> Martin, Alois
</P><P><B>Aim:</B>  Purification of thaumatin (standard, lysate, supernatant)
</P><UL><LI> pI of thaumatin: 8.3.
</LI><LI> Buffer 2: 20 mM MES, 0.5 M NaCl, pH 6.0.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_yeast_cultures_for_large_scale_expression_.281l.29_2"> Preparation of yeast cultures for large scale expression (1l) 2</SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the preparation of three 80 ml yeast cultures (in selective SC-U 2% glucose medium), based on the 30 ml yeast cultures from 18.09., each containing one of the three genes CaXMT1, CaMXMT1 and CaDXMT1 for having enough crude extract for further analysis. 
</P><P><B>Operational sequence:</B></P><P>The OD600nm was measured. All had an OD600nm of about 8/ml. Therefore 4 ml of each culture was transferred into 80 ml Sc-Uracil + glucose medium to reach an OD600nm of 0.4.
The generated cell suspensions were incubated at 30°C over night.
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_September_20th"><B>Thursday, September 20th</B></SPAN></H2><DIV class="constitutive_promoter"><P><B>Investigator:</B> Kathrin Fischer
</P><H3><SPAN class="mw-headline" id="Transformation_of_competent_invitrogen_YEAST_cells_with_pTUM101-Luciferase.2C_pTUM102-Luciferase.2C_pTUM101-Limonensynth..2C_pTUM102-Limonensynth..2C_pTUM103-Limonensynth.">Transformation of competent invitrogen YEAST cells with pTUM101-Luciferase, pTUM102-Luciferase, pTUM101-Limonensynth., pTUM102-Limonensynth., pTUM103-Limonensynth.</SPAN></H3><UL><LI>Transformation was done according to the manual of the S.c. EasyComp Yeast Transformation Kit from invitrogen
</LI></UL></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P885_and_P886"> Ligation of P885 and P886</SPAN></H3><P><B>Investigator: Georg</B></P><UL><LI> Reaction batch for Ligation:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1 µl
</TD><TD>T4-Ligase (1u/µl)
</TD></TR><TR><TD>7 µl
</TD><TD>P886
</TD></TR><TR><TD>3,4 µl (=100 ng)
</TD><TD>P885
</TD></TR><TR><TD>2
</TD><TD>10x T4-ligase buffer
</TD></TR><TR><TD>6,6 µl
</TD><TD>dd H20
</TD></TR><TR><TD>=20µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE></DIV><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E.coli_XL1-blue_competent_cells_with_ligation_of_P885_.2B_P886"> Transformation of <I>E.coli</I> XL1-blue competent cells with ligation of P885 + P886</SPAN></H3><P><B> Investigator: Georg</B></P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 10 µl of ligation product (P265B) was added to 100 µl of competent cells and gently mixed.
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min. heat shock at 37°C
</LI></UL><UL><LI> Adding of 1ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45min at 37°C in the 180rpm cell-culture shaker.
</LI></UL><UL><LI> 100µl of those cell suspension were plated on chloramphenicole plates.
</LI></UL><UL><LI> The rest were centrifuged for 1min at 13000rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100µl of LB-medium and this concentrated cell suspension was plated again on new CAM plates.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P750.2BP882.2C_P750.2BP884.2C_P885.2BP882.2C_P885.2BP884.2C_P881.2BP791.2C_P875.2BP876.2C_P873.2BP876"> Cycled ligation of P750+P882, P750+P884, P885+P882, P885+P884, P881+P791, P875+P876, P873+P876 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Cycled ligation of P750+P882 (cloning Gal4 based expression casette battery into pSB6A0), P750+P884 (cloning LexA based expression casette battery into pSB6A0), P885+P882 (cloning Gal4 based expression casette battery in another pSB1C3 to correct the lost NotI site), P885+P884 (cloning LexA based expression casette battery in another pSB1C3 to correct the lost NotI site), P881+P791 (cloning of Renilla luciferase into pYES2new), P875+P876 (cloning of TEF1-promoter+Renilla luciferase into pYES2new(pGAL<SUP>-</SUP>)), P873+P876 (in the front of CYC1 terminator in pSB1C3.
</P><P><B>Procedure:</B></P><UL><LI> Ligation batch for P750+P882 (1:3)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.31 µl
</TD><TD>P750 (76.1 ng/µl, 4791 bp)
</TD></TR><TR><TD>7.23 µl
</TD><TD>P882 (52.4 ng/µl, 6091 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>8.46 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P750+P884 (1:3)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.31 µl
</TD><TD>P750 (76.1 ng/µl, 4791 bp)
</TD></TR><TR><TD>8.76 µl
</TD><TD>P884 (44.4 ng/µl, 6256 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>6.94 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P885+P882 (1:3)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.8 µl
</TD><TD>P885 (29.8 ng/µl, 2047 bp)
</TD></TR><TR><TD>14.2 µl
</TD><TD>P882 (52.4 ng/µl, 6091 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P885+P884 (1:3)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.38 µl
</TD><TD>P885 (29.8 ng/µl, 2047 bp)
</TD></TR><TR><TD>14.62 µl
</TD><TD>P884 (44.4 ng/µl, 6256 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P881+P791(1:3)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.56 µl
</TD><TD>P881 (63.9 ng/µl, 5789 bp)
</TD></TR><TR><TD>1.82 µl
</TD><TD>P791 (27.1 ng/µl, 956 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>13.62 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P875+P876(1:3)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.72 µl
</TD><TD>P875 (36.7 ng/µl, 4844 bp)
</TD></TR><TR><TD>2.35 µl
</TD><TD>P876 (35.7 ng/µl, 1362 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>11.93 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P873+P876(1:3)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>12.7 µl
</TD><TD>P875 (6.9 ng/µl, 2336 bp)
</TD></TR><TR><TD>4.3 µl
</TD><TD>P876 (35.7 ng/µl, 1362 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batches.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Measurement_of_absorption_spectrum_of_extracted_phycocyanobilin"> Measurement of absorption spectrum of extracted phycocyanobilin </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Measurement of absorption spectrum of extracted phycocyanobilin.
</P><P><B>Procedure:</B></P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_P750.2BP882.2C_P750.2BP884.2C_P885.2BP882.2C_P885.2BP884.2C_P881.2BP791.2C_P875.2BP876.2C_P873.2BP876"> Transformation of <I>E. coli</I> XL1-Blue with ligation products P750+P882, P750+P884, P885+P882, P885+P884, P881+P791, P875+P876, P873+P876 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P750+P882, P750+P884, P885+P882, P885+P884, P881+P791, P875+P876, P873+P876.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation products (P750+P882 (AmpR), P750+P884 (AmpR), P885+P882 (CamR), P885+P884 (CamR), P881+P791 (AmpR), P875+P876 (AmpR), P873+P876 (CamR)) and the negative controls were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets was resuspended in 100 µl of LB-medium and were plated on antibiotic plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Sequencing_of_promotor.2Flimonensynthase.2Fterminator-constructs">Sequencing of promotor/limonensynthase/terminator-constructs</SPAN></H3><P><B>Investigators:</B> Andrea, Lara
</P><P>clone 1(2): TEF2/PCR2/Cyc-T/pSB1C3
</P><P>clone 2(4): TEF1/PCR2/Cyc-T/pSB1C3
</P><P>clone 2TEF(1): TEF1/PCR2/TEF1/pSB1C3
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Cell_lysis_for_expression_comparison_test">Cell lysis for expression comparison test</SPAN></H3><P><B>Investigator:</B> Lara, Andrea
</P><P><B>Aim:</B> Cell disruption for harvesting proteins expressed in yeasts (crude protein extract)
</P><P><B>Procedure:</B></P><UL><LI> resuspending cell pellets in defined volume of breaking buffer + PMSF
</LI><LI> adding glass beads
</LI><LI> vortex for 30 seconds and incubate on ice for 30 seconds (repetition for 20 times)
</LI><LI> centrifuge at 4 °C, full speed, 10 min
</LI><LI> take supernatant and centrifuge at 4 °C, full speed, 10 min again
</LI><LI> store supernatant at -20°C 
</LI></UL><P>
1: PCR1 (Clone 29), cons - induced (SC-U + Gal)
</P><P>3: S. cerevisiae - not transformed, SC + Uracil + Glu
</P><P>4: PCR1 (Clone 29), cons - not induced (SC-U + Glu)
</P><P>5: PCR2 (p542), consless - induced (SC-U + Gal)
</P><P><B>please apply this order on SDS-gel: 5, 1, 4, 3</B></P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Cation_exchange_chromatography_of_thaumatin_.28standard.2C_lysate.2C_supernatant.29">Cation exchange chromatography of thaumatin (standard, lysate, supernatant)</SPAN></H3><P><B>Investigator</B>: Volkic, Martinic, Aloisic
</P><UL><LI> ÄKTA purifier 3, resource S; superloop: programm: Volker Morath_iGEM.
</LI><LI> pI of thaumatin: 8.3.
</LI><LI> Buffer 1: 20 mM MES, pH 6.0.
</LI><LI> Buffer 2: 20 mM MES, 0.5 M NaCl, pH 6.0.
</LI></UL><UL><LI> Standard: ca. 8 ml, 1 mg/ml.
</LI><LI> Lysate: ca. 10 ml.
</LI><LI> Supernatant: ca. X ml.
</LI></UL></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id=".22Continuation:_Picking_of_36_clones.22"> &quot;Continuation: Picking of 36 clones&quot; </SPAN></H3><P><B>Investigators:</B> Martin, Alois
</P><P><B>Aim:</B>  To study integration efficiency, we kept track of 36 single clones, which were studied over a time course of 48, respectively 96 hours.
</P><P><B>Procedure:</B> Two 96s wells were covered in aluminum foil and autoclaved. 24 (respectively 12) chambers of one well were each filled with 150µl YPD medium and inoculated with a single clone that was picked from the transformation plate. After 12h the cells (5 µl) were passaged into a new well with new medium (140 µl YPD).
</P><UL><LI><I>Continuation</I>: After further 12h (10:00) the cells (5 µl) were passaged into a new well with new medium (140 µl YPD).
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="SDS-PAGE_of_cation_exchange_chromatography_probes_of_thaumatin_.28standard.2C_lysate.29">SDS-PAGE of cation exchange chromatography probes of thaumatin (standard, lysate)</SPAN></H3><P><B>Investigator</B>: Aloisic
</P><UL><LI> S (= standard): M/S0(without iEX)/S12/S13/<B>S14</B>/S15/S16/S17.
</LI></UL><UL><LI> The bought thaumatin (MedHerb) is quite clean; the second protein bands in fraction S15, S16, S17 could be thaumatin dimers.
</LI></UL><UL><LI> L1 (lysate): M/S14/6/7/8/9/10/11/12/13.
</LI></UL><UL><LI> L2 (lysate): M/S14/<B>14</B>/<B>15</B>/16/17/18/19/20/21.
</LI></UL><UL><LI><B>Result</B>: Fractions 14 and 15 show a slight protein band of thaumatin (22.2 kDa). The expression was successful, but the yield was low.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_transformation"> Miniprep of transformation</SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> Miniprep of 9 over night cultures with number cassette CaXMT1 (biobrick1),cassette CaMXMT1 (biobrick2), cassette CaDXMT1 (biobrick3). Clones were isolated of over night culture, 5ml LB-medium with CHLP.
</P><P>The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 40 µl ddH2O at room temperature, with the columns having been incubated 2 minutes at room temperature before the final centrifugation. 
<B>Miniprep products are stored in a 50 ml Falcon tube in the lowest drawer at -20°C </B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_transformation_2"> Miniprep of transformation </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> Miniprep of 3 over night cultures with number B2A + terminator ADH1, B1B + terminator ADH1, B3D + ADH1 terminator. Clones were isolated of over night culture, 5ml LB-medium with CHLP.
</P><P>The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 40 µl ddH2O at room temperature, with the columns having been incubated 2 minutes at room temperature before the final centrifugation. 
<B>Miniprep products are stored in a 50 ml Falcon tube in the lowest drawer at -20°C </B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preperative_digestion"> Preperative digestion </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> after the sequences of the constructs were succsesful, we started a preperative digestion of the constructs to ligate them together. All three: cassette CaXMT1 (biobrick1),cassette CaMXMT1 (biobrick2), cassette CaDXMT1 (biobrick3)
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Ligation_of_preparative_digestion_2"> Ligation of preparative digestion</SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> ligation was performed for cassette CaXMT1 (biobrick1)+ cassette CaMXMT1 (biobrick2)+ cassette CaDXMT1 (biobrick3)
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Start_of_the_large_scale_expression_.281l.29_3"> Start of the large scale expression (1l) 3</SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the expression of the three caffein genes CaDXMT1, CaMXMT1 and CaXMT1 in selective SC-U 2% galactose expression medium), based on the 70 ml yeast cultures from 19.09., each containing one of the three genes CaXMT1, CaMXMT1 and CaDXMT1 for having enough crude extract for further analysis. 
</P><P><B>Operational sequence:</B></P><P>The OD600nm was measured:
</P><UL><LI>CaDXMT1: 7
</LI><LI>CaMXMT1: 6.52
</LI><LI>CaXMT1: 7.1
</LI></UL><P>therefore 57 ml, 61 ml and 56 ml yeast culture were centrifuged, the supernatant discarded, the pellet resuspended and transferred into selective SC-U 2% galactose expression medium and incubated at 30°C for 16 h.                                          
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_September_21st"><B>Friday, September 21st</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_P885.2BP882.2C_P885.2BP884.2C_P881.2BP791.2C_P875.2BP876.2C_P873.2BP876"> Transformation of <I>E. coli</I> XL1-Blue with ligation products P885+P882, P885+P884, P881+P791, P875+P876, P873+P876 </SPAN></H3><P><B>Investigator:</B> Simon
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P885+P882, P885+P884, P881+P791, P875+P876, P873+P876.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation products (P885+P882 (CamR), P885+P884 (CamR), P881+P791 (AmpR), P875+P876 (AmpR), P873+P876 (CamR)) and the negative controls were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspensions were plated on suitable antibiotic plates.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets was resuspended in 100 µl of LB-medium and were plated on antibiotic plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_P676_2"> Preparative digestion and gelelectrophoresis of P676 </SPAN></H3><P><B>Investigator:</B> Ingmar, Simon
</P><P><B>Aim of the experiment:</B> Preparative digestion and gelelectrophoresis of P676 with EcoRI-HF and SpeI-HF.
</P><P><B>Procedure:</B></P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Cycled_ligation_of_P898.2BP882.2C_P898.2BP884"> Cycled ligation of P898+P882, P898+P884 </SPAN></H3><P><B>Investigator:</B></P><P><B>Aim of the experiment:</B> Cycled ligation of P898+P882, P898+P884 in order to clone the expression casette battery in pSB6A0, an yeast/bacterial hybrid plasmid.
</P><P><B>Procedure:</B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytic_digestion_of_CaXMT1_and_CaMXMT1"> Analytic digestion of CaXMT1 and CaMXMT1  </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> the constructs CaXMT1 and CaMXMT1 together will be cut out, by XbaI and SpeI, and analysed
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>probes
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI HF (20 U/µl)
</TD></TR><TR><TD>10,5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytic_gel_electrophoresis_3">Analytic gel electrophoresis</SPAN></H3><P><B>experimenter:</B> Dennis
</P><UL><LI> 1 µl of DNA loading buffer (10x) was added to the digestion mix after incubation time.
</LI></UL><UL><LI> analytic gelelectrophoresis was perforemd at 90 V for 1 h.
</LI></UL><TABLE cellspacing="0" border="1" style="font-size:75%"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pSB1C3 CaXMT1+CaMXMT1 ( miniprep1)
</TD><TD>pSB1C3 CaXMT1+CaMXMT1 ( miniprep2)
</TD><TD>pSB1C3 CaXMT1+CaMXMT1 ( miniprep5)
</TD><TD>pSB1C3 CaXMT1+CaMXMT1 ( miniprep6)
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_5"> Transformation of <I>E. coli</I></SPAN></H3><P><B>Investigator:</B> Katrin, Dennis
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with the sequenced constructs CaXMT1, CaMXMT1, CaDXMT1
</P><P><B>Procedure:</B></P><UL><LI> 20 µl of each ligation products were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on chloramphenicol plates.
</LI></UL><UL><LI> The rest were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellet was resuspended in 100 µl of LB-medium and this concentrated cell suspension was plated again on chloramphenicol plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Ligation_of_the_three_cassettes"> Ligation of the three cassettes</SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> ligation was performed for cassette CaXMT1 (biobrick1)+ cassette CaMXMT1 (biobrick2)+ cassette CaDXMT1 (biobrick3)
</P></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="SDS-PAGE_of_cation_exchange_chromatography_of_thaumatin_of_lysate_.282nd_run.29_and_supernatant">SDS-PAGE of cation exchange chromatography of thaumatin of lysate (2nd run) and supernatant</SPAN></H3><P><B>Experimenter:</B> Marin, Alois
</P><UL><LI> L3: M/S14/X/12/13/<B>14</B>/15/16/17.
</LI></UL><UL><LI><B>Result</B>: Fraction 14 shows again a slight protein band of thaumatin (22.2 kDa). The expression was successful, but the yield was low. 
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="SDS-PAGE_.2B_silver_staining_of_cation_exchange_chromatography_of_supernatant">SDS-PAGE + silver staining of cation exchange chromatography of supernatant</SPAN></H3><P><B>Experimenter:</B> Marko Marin, Alois
</P><UL><LI> U1: M/S14/X/4/5/6/7/8/9/10.
</LI><LI> U2: M/S14/X/11/12/13/14/15/16/17.
</LI></UL><P><B>Results:</B></P><P>If there is any thaumatin in the supernatant, the silver staining may not be sensitive enough.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Large_scale_expression_.281l.29_4"> Large scale expression (1l) 4</SPAN></H3><P><B>Investigator:</B> Saskia, Kathrin
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the expression of the three caffein genes CaDXMT1, CaMXMT1 and CaXMT1 in selective SC-U 2% galactose expression medium for having enough crude extract for further analysis. 
</P><P><B>Operational sequence:</B></P><P>The cells didn't grow, because the OD600nm was still 0.4 as we started the day before. Thus we centrifuged the cells, discarded the supernatant and resuspended the cells in selective SC-U 2% glucose medium and grew the cells again in 200 ml of SC-U 2% glucose medium to get enough cells for repetition of the expression.
The cells were incubated at 30°C over night.                                          
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Toxicity_Assay:_effect_of_caffeine.2C_thaumatin_and_limonene_on_yeast_growth"> Toxicity Assay: effect of caffeine, thaumatin and limonene on yeast growth </SPAN></H3><P><B>Investigator:</B> Mary
</P><P><B>Aim of the experiment:</B> To check whether caffeine, thaumatin and limonene have a toxic effect on different yeast strains. 
</P><P>One over-night culture of INVSc were inoculated and the OD(600) measured at the next day. 
</P><P>The OD(600) was adjusted to 0,5 in different media and the growth rate were measured after about 2-3 hours. 
</P><UL><LI>Caffeine: concentrations of 100mM to 1µM were used, but caffeine did not resolve completely, so the experiment did not show any effect and was repeated the next day. 
</LI></UL><UL><LI>Thaumatin: a concentration of 1mg/ml was used, by adding 10mg of thaumatin to 10ml YPD medium. The growth rate was measured four times.
</LI></UL><P>Results:
</P><CENTER><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Evaluation of the Toxicity Assay for Thaumatin.</DIV></DIV></DIV></DIV></CENTER><UL><LI>Limonene: following concentrations were used: 100mM, 10mM, 1mM, 100µM, 10µM, 1µM and the growth rate was measured five times. 
</LI></UL><P>Results:
</P><CENTER><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Evaluation of the Toxicity Assay for Limonene.</DIV></DIV></DIV></DIV></CENTER><P>As the negative control (without any substance in the medium) the INVSc strain of the growth experiment was used. 
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Growth_experiment_of_three_different_yeast_strains_in_different_conditions">Growth experiment of three different yeast strains in different conditions</SPAN></H3><P><B>Investigator:</B> Mary
</P><P><B>Aim of the experiment:</B> We want to establish the effect of gyle with and without hop on the growth rate of the INVSc strain, which was used in our laboratory research compared to two different strains. 
</P><P>Three over-night cultures were inoculated: one clone of INVSc was picked, a crumb of a yeast typically for brewing and a crumb of a yeast which can be purchased in the supermarket were added to the YPD medium.
</P><P>The growth rate was measured four times (every 2-3h) 
</P><P>Results: 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">Evaluation of measured OD600 of several yeast strains under different conditions.</DIV></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_September_22nd"><B>Saturday, September 22nd</B></SPAN></H2><DIV class="constitutive¬_promoter"><DIV class="constitutive_promoter"><H3><SPAN class="mw-headline" id="Inoculation_of_yeast_overnight_culture_with_yeast_transformated_with_pTUM101-P791_and_pTUM103-P791"> Inoculation of yeast overnight culture with yeast transformated with pTUM101-P791 and pTUM103-P791</SPAN></H3><P><B>Aim of the project:</B> Make an overnight culture of yeast transformants for luciferase assay
</P><UL><LI> one colony was transferred into 30 ml sterile filtrated USC-medium (Uracil missing) and incubated at 30 min over night.
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_E..C2.A0coli_XL1-Blue_with_ligation_products_P898.2BP882.2C_P898.2BP884"> Transformation of <I>E. coli</I> XL1-Blue with ligation products P898+P882, P898+P884 </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B> Transformation of <I>E. coli</I> XL1-Blue with ligation products P898+P882, P898+P884.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation products (P898+P882 (AmpR), P898+P884 (AmpR)) and the negative controls were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspensions were plated on suitable antibiotic plates.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 1 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets was resuspended in 100 µl of LB-medium and were plated on antibiotic plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Protein_purification">Protein purification</SPAN></H3><P><B>Investigators:</B> Volker, Lara
</P><P><B>Aim:</B> Get purified limonene synthase for enzyme assays.
</P><P>
Dialysed and steril-filtered protein samples from large scale cell lysis were applied to a streptavidin solid support.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_integration_vector_constructs">Miniprep of integration vector constructs</SPAN></H3><P><B>Investigators:</B> Lara
</P><P><B>Aim:</B> Get plasmid DNA for integration of limonene synthase gene into yeast.
</P><P><B>1(4):</B> TEF2-P/LS consless/CYC-T: 
</P><UL><LI> 80 ng/µl
</LI></UL><P><B>2(3):</B> TEF1-P/LS consless/CYC-T:
</P><UL><LI> 1: 155 ng/µl
</LI></UL><UL><LI> 2: 750 ng/µl
</LI></UL><UL><LI> 3: 175 ng/µl
</LI></UL><P><B>2TEF2</B> TEF1-P/LS consless/TEF1-T
</P><UL><LI> 1: 205 ng/µl
</LI></UL><UL><LI> 2: 192 ng/µl
</LI></UL><UL><LI> 3: 208 ng/µl
</LI></UL><P><B>Analytical restriction digest (or/and) preparative gel electrophoresis still have to be done!</B><B>Miniprep products are stored in a 50 ml falcon in the lowest drawer of -20°C. Label: Integrationsvektor, Limonen, 22.9.</B></P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparation_of_cultures_for_expression">Preparation of cultures for expression</SPAN></H3><P><B>Investigators:</B> Lara
</P><P><B>Aim:</B> Making of reparatory cultures in glucose medium for transfer into induction medium the next day.
</P><P>
Picking of clones:
</P><UL><LI> Clone carrying PCR1-LS: SC-U + Glu (50 ml)
</LI><LI> Clone carrying PCR2-LS: SC-U + Glu (50 ml)
</LI><LI> S. cerevisiae (not transformed): SC-U + Glu + uracil (50 ml)
</LI></UL><P>
Preparation of medium for induction:
</P><P>1. SC-U + Gal: PCR1
</P><P>2. SC-U + Gal: PCR2
</P><P>3. SC-U + Gal: PCR1 + GPP
</P><P>4. SC-U + Gal: PCR2 + GPP 
</P><P>5. SC-U + Glu: PCR1 
</P><P>6. SC + Glu + uracil: S. cerevisiae (not transformed)
</P><P>7. SC + Glu + uracil + limonene: S. cerevisiae (not transformed)
</P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Plating_of_31_clones_originally_transfected_with_integ-vec_mOrange_.28see_preparative_experiment.29_.2B_negative_control_.28not_transfected.29"> Plating of 31 clones originally transfected with integ-vec_mOrange (see preparative experiment) + negative control (not transfected) </SPAN></H3><P><B>Investigator:</B> Martin the Weird
</P><P><B>Procedure:</B> The last culture of the preparative experiment was used to inoculate an overnight culture. The OD 600 of all candidates was measured. 
</P><TABLE cellspacing="2" border="4"><TBODY><TR><TD><B>Clone</B></TD><TD><B>OD 600</B></TD><TD><B>Clone</B></TD><TD><B>OD 600</B></TD><TD><B>Clone</B></TD><TD><B>OD 600</B></TD></TR><TR><TD><B>1</B></TD><TD>1.73
</TD><TD><B>13</B></TD><TD>1.05
</TD><TD><B>25</B></TD><TD>2.02
</TD></TR><TR><TD><B>2</B></TD><TD>1.88
</TD><TD><B>14</B></TD><TD>0.79
</TD><TD><B>26</B></TD><TD>1.98
</TD></TR><TR><TD><B>3</B></TD><TD>1.96
</TD><TD><B>15</B></TD><TD>1.98
</TD><TD><B>27</B></TD><TD>fail
</TD></TR><TR><TD><B>4</B></TD><TD>fail
</TD><TD><B>16</B></TD><TD>1.96
</TD><TD><B>28</B></TD><TD>fail
</TD></TR><TR><TD><B>5</B></TD><TD>2.06
</TD><TD><B>17</B></TD><TD>1.99
</TD><TD><B>29</B></TD><TD>1.96
</TD></TR><TR><TD><B>6</B></TD><TD>2.02
</TD><TD><B>18</B></TD><TD>1.97
</TD><TD><B>30</B></TD><TD>2.00
</TD></TR><TR><TD><B>7</B></TD><TD>2.01
</TD><TD><B>19</B></TD><TD>1.99
</TD><TD><B>31</B></TD><TD>1.94
</TD></TR><TR><TD><B>8</B></TD><TD>1.64
</TD><TD><B>20</B></TD><TD>1.98
</TD><TD><B>32</B></TD><TD>1.96
</TD></TR><TR><TD><B>9</B></TD><TD>1.89
</TD><TD><B>21</B></TD><TD>fail
</TD><TD><B>33</B></TD><TD>1.95
</TD></TR><TR><TD><B>10</B></TD><TD>1.99
</TD><TD><B>22</B></TD><TD>1.87
</TD><TD><B>34</B></TD><TD>2.02
</TD></TR><TR><TD><B>11</B></TD><TD>1.97
</TD><TD><B>23</B></TD><TD>1.96
</TD><TD><B>35</B></TD><TD>1.97
</TD></TR><TR><TD><B>12</B></TD><TD>fail
</TD><TD><B>24</B></TD><TD>1.96
</TD><TD><B>36</B></TD><TD>2.03
</TD></TR></TBODY></TABLE><P>We had 5 negatives, probably due to mistakes in pipeting. The rest of the cultures were diluted to OD 0.4 and 100 ml of it were used to plate on selective (meaning G418+) petri dishes.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Start_of_the_large_scale_expression_.281.5_l.29_5"> Start of the large scale expression (1.5 l) 5</SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the expression of the three caffein genes CaDXMT1, CaMXMT1 and CaXMT1 in selective SC-U 2% galactose expression medium), based on the 200 ml yeast cultures from 21.09., each containing one of the three genes CaXMT1, CaMXMT1 and CaDXMT1 for having enough crude extract for further analysis. 
</P><P><B>Operational sequence:</B></P><P>3x 1.5 l selective SC-U 2% medium were produced and 167 ml 20% galactose were added each to gain 2% galactose.
</P><P>The OD600nm of the over night cultures grown in selective SC-U 2% glucose medium was measured:
</P><UL><LI>CaDXMT1: 6
</LI><LI>CaMXMT1: 6.5
</LI><LI>CaXMT1: 7
</LI></UL><P>therefore 100 ml, 92 ml and 86 ml yeast culture were centrifuged, the supernatant discarded, the pellet resuspended in selective SC-U 2% galactose expression medium and transferred into 1.67 l selective SC-U 2% galactose expression medium and incubated at 30°C for 16 h.                                          
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_0.1_M_sodium_phosphate_.28pH_7.4.29_and_breaking_buffer_with_and_without_PMSF"> Preparation of 0.1 M sodium phosphate (pH 7.4) and breaking buffer with and without PMSF </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>0.1 M sodium phosphate (pH 7.4): 1 liter</B></P><UL><LI> solution of 15.601 g NaH2PO4x2H20 in 100 ml ddH2O: 1 M NaH2PO4x2H20 
</LI><LI> solution of 17.799 g Na2HPO4x2H20 in 100 ml ddH2O: 1 M Na2HPO4x2H20  
</LI><LI> 22.6 ml 1 M NaH2PO4x2H20 + 77.4 ml 1 M Na2HPO4x2H20. Before bringing up the volume to 1 liter controll of pH (7.4) and if needed correction of pH. 
</LI></UL><P><B>Breaking buffer: 1 liter</B></P><UL><LI> 50 mM sodiumphosphate (pH 7.4): 500ml of 0.1 M sodiumphosphate 
</LI><LI> 1 mM EDTA: 2 ml of 500 mM EDTA
</LI><LI> 5 % glycerol: 50 ml of 100% glycerol
</LI><LI> 348 ml ddH2O
</LI></UL><P>when preparing breaking buffer with PMSF 100 ml of 10 mM PMSF solved in isopropanol were added to get 1 mM PMSF 
</P><UL><LI> (10 mM PMSF: 0.1741 g PMSF + 100 ml isopropanol)
</LI></UL></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Repetition_of_Toxicity_Assay:_effect_of_caffeine_on_yeast_growth"> Repetition of Toxicity Assay: effect of caffeine on yeast growth </SPAN></H3><P><B>Investigator:</B> Mary
</P><P><B>Aim of the experiment:</B> To check whether caffeinehas a toxic effect on different yeast strains. 
</P><P>One over-night culture of INVSc were inoculated and the OD(600) measured at the next day. 
</P><P>The OD(600) was adjusted to 0,5 in different media and the growth rate were measured after about 2-3 hours. 
</P><UL><LI>Caffeine: concentrations of 100mM to 1µM were used. To resolve the caffeine completely incubation in ultrasonic bath was needed. Afterwards, caffeine was resolved completely.
</LI></UL><P>As the negative control (without any substance in the medium) the INVSc strain of the growth experiment was used. 
</P><P>Results: 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Evaluation of the Toxicity Assay for Caffeine.</DIV></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Sunday.2CSeptember_23rd"><B>Sunday,September 23rd</B></SPAN></H2><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Yeast_expression:_Pelletizing_18_h_after_expression_induction_and_cell_lysis"> Yeast expression: Pelletizing 18 h after expression induction and cell lysis </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was to pellet the 1.67 l cell suspension 18 h after expression induction and to wash the pellet with breaking buffer. Then the cells were lysed using glass beads.
</P><P><B>Operational sequence:</B></P><P>At first, the OD600 of the suspensions was determined:
</P><UL><LI> CaXMT1: 0.424
</LI><LI> CaMXMT1: 0.384
</LI><LI> CaDXMT1: 0.516
</LI></UL><P>The cell suspensions were centrifuged at 4000 rpm for 25 minutes at 4°C. Afterwards the supernatant was discarded and the pellet was resuspended in 20 ml breaking buffer without PMSF. The cells were centrifuged again at 4000 rpm for 25 minutes at 4 °C. The supernatant was discarded and the cells were resuspended in 13-14 ml breaking buffer with PMSF for the cell lysis. An equal amount of glass beads solved in breaking buffer were added and the cell supsensions were votexed and stored on ice each for 30 seconds. This two steps were repeated 30 times. Then the suspensions were centrifuged at 4000 rpm for 20 min at 4 °C. The supernatant was stored and the beads were washed again in 5 ml breaking buffer with PMSF and centrifuged at 4000 rpm for 20 min at 4 °C. 1 ml was stored at - 80 °C for enzymatic analysis. And the rest was used for dialysis and purification for detection of all three enzymes via LCMS.
</P></DIV><H1><SPAN class="mw-headline" id="Week_16">Week 16</SPAN></H1><DIV class="WWeek_16"><H2><SPAN class="mw-headline" id="Monday.2C_September_24th"><B>Monday, September 24th</B></SPAN></H2><DIV class="limonene"><H3><SPAN class="mw-headline" id="Headspace_GC-MS_of_limonene">Headspace GC-MS of limonene</SPAN></H3><P><B>Investigator:</B> Lara
</P><P><B>Aim:</B> Check whether limonene is detectable in the cell culture supernatant. Since extraction is time-consuming, SPME was done.
</P><UL><LI> The SPME needle was injected into the headspace. Incubation for 30 min at 45 °C.
</LI><LI> Injection into the GC (starting temperature: 40 °C)
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_transformation_3"> Miniprep of transformation </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim:</B> Miniprep of 8 over night cultures with number all cassette CaXMT1, CaMXMT1, CaDXMT1. Clones were isolated of over night culture, 5ml LB-medium with CHLP.
</P><P>The plasmid preparation was performed as described in the manufactueres protocol (Quiagen). Elution was done with 40 µl ddH2O at room temperature, with the columns having been incubated 2 minutes at room temperature before the final centrifugation. 
</P><P><B>Miniprep products are stored in a 50 ml Falcon tube in the lowest drawer at -20°C </B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytic_digestion_of_miniprep"> Analytic digestion of miniprep </SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B> the constructs CaXMT1, CaMXMT1 and CaDXMT1 together will be cut out, by XbaI and SpeI, and analysed
</P><P><B>Procedure:</B></P><UL><LI> Reaction batch:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>5 µl
</TD><TD>probes
</TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1 µl
</TD><TD>SpeI HF (20 U/µl)
</TD></TR><TR><TD>10,5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytic_gel_electrophoresis_4"> Analytic gel electrophoresis</SPAN></H3><P><B>experimenter:</B> Dennis
</P><UL><LI> 1 µl of DNA loading buffer (10x) was added to the digestion mix (pSB1C3 CaXMT1+CaMXMT1+CaDXMT1, clone 1-8) after incubation time.
</LI></UL><UL><LI> analytic gelelectrophoresis was perforemd at 90 V for 1 h.
</LI></UL><TABLE cellspacing="0" border="1" style="font-size:60%"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD> ( miniprep1)
</TD><TD> ( miniprep2)
</TD><TD> ( miniprep3)
</TD><TD> ( miniprep4)
</TD><TD> ( miniprep5)
</TD><TD> ( miniprep6)
</TD><TD> ( miniprep7)
</TD><TD> ( miniprep8)
</TD><TD>1 kbp DNA ladder
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="High_scale_yeast_expression:_Induction"> High scale yeast expression: Induction </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the induction of the expression of the following three genes: CaXMT1, CMXMT1 and CaDXMT1. Previous attempts to do a high scale expression with these genes had failled, because the transformed yeast cells showed no growth after transferring them into 2% galactose containing induction medium.
</P><P><B>Operational sequence:</B></P><P>As previously described, the OD600 of the over night cultures (in SC-U medium with 2% glucose) was determined. 1l of SC-U 2% galactose medium were inoculated with a calculated amount of pelleted over night cultures (to get an OD600 of ca. 0,4). The cultures were incubated at 30°C for 20 hours and then stored in the cooling room (4°C) until cell lysis.
</P><P>Determined OD600 20h after induction:
</P><UL><LI> CaXMT1: 5,5
</LI><LI> CaMXMT1: 5,0
</LI><LI> CaDXMT1: 4,9
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vitro_reconstruction_of_the_caffeine_biosynthesis_pathway:_Enzyme_assay"> In vitro reconstruction of the caffeine biosynthesis pathway: Enzyme assay </SPAN></H3><P><B>Investigator:</B> Roman, Saskia
</P><P><B>Aim of the experiments:</B></P><P>Aim of the experiment was the proof of enzyme function by reconstruction of the caffeine biosynthesis pathway.
</P><P><B>Operational sequence:</B></P><P>The enzyme assay was performed as described in the paper of Hiroshi Sano et al., 2003. 100µl reaction volumes were prepared as follows:
</P><UL><LI> 50 mM Tris/HCl (pH 8.0)
</LI><LI> 500 µM Xanthosine (not used in two negative controlls, see below)
</LI><LI> 1,5 mM SAM (S- adenosyl methionine)
</LI><LI> 200 µM MgCl<SUB>2</SUB></LI><LI> 200 µg of each used (see below) protein (crude extract)
</LI></UL><P>(each reaction batch was filled up to 100µl with water)
</P><P>We prepared the following reaction batches (differing enzyme- and substrate-compositions):
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Batch</B></TD><TD><B>Enzyme content</B></TD></TR><TR><TD>1
</TD><TD>200µg of all three enzymes, isolated on 31.08. and 11.09., with xanthosine
</TD></TR><TR><TD>2
</TD><TD>200µg of all three enzymes, isolated on 31.08. and 11.09., with xanthosine
</TD></TR><TR><TD>3
</TD><TD>200µg of all three enzymes, isolated on 31.08. and 11.09., without xanthosine
</TD></TR><TR><TD>4
</TD><TD>no enzymes (water instead), with xanthosine
</TD></TR><TR><TD>5
</TD><TD>200µg of all three enzymes, isolated on 22.09., with xanthosine
</TD></TR><TR><TD>6
</TD><TD>200µg of all three enzymes, isolated on 22.09., with xanthosine
</TD></TR><TR><TD>7
</TD><TD>200µg of all three enzymes, isolated on 22.09., without xanthosine
</TD></TR><TR><TD>A
</TD><TD>200µg of CaXMT1, isolated on 31.08. and 11.09., with xanthosine
</TD></TR><TR><TD>B
</TD><TD>200µg of CaXMT1 and CaMXMT1, isolated on 31.08., with xanthosine
</TD></TR><TR><TD>C
</TD><TD>200µg of CaXMT1, isolated on 22.09., with xanthosine
</TD></TR><TR><TD>D
</TD><TD>200µg of CaXMT1 and CaMXMT1, isolated on 31.08., with xanthosine
</TD></TR></TBODY></TABLE><UL><LI> Samples 2 and 6 were incubated for 16 hours at 16°C (highest fermenting temperature)
</LI><LI> All other samples were incubated for 16 hours at 27°C (enzyme optima after Hiroshi Sano et al., 2003)
</LI></UL><P>To end the reaction, the batches were frozen at -80°C until further usage (chloroform extraction).
</P></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_September_25th"><B>Tuesday, September 25th</B></SPAN></H2><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vitro_reconstruction_of_caffeine_biosynthesis_pathway:_chloroform_extraction"> In vitro reconstruction of caffeine biosynthesis pathway: chloroform extraction </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the extraction of potential reaction products, to be able to start a LCMS analysis in order to detect the synthesized caffeine.
</P><P><B>Operational sequence:</B></P><P>We added 1 ml of chloroform to the 100µl reaction batches, mixed a few seconds and stored the tubes for about 30 min at room temperature, to get a proper phase- separation. Afterwards, the lower phase (chloroform) was pipetted in tubes (glass tubes would have been better than plastic tubes) (1ml) and incubated for about 60 min at 60°C (=drying) until total chloroform removal.
</P><P>The tubes were then stored at -80°C over night until further usage (LCMS)
</P></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_September_26th"><B>Wednesday, September 26th</B></SPAN></H2><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vitro_reconstruction_of_caffeine_biosynthesis_pathway:_LCMS_analysis_of_extracted_compounds"> In vitro reconstruction of caffeine biosynthesis pathway: LCMS analysis of extracted compounds </SPAN></H3><P><B>Investigator:</B> Roman
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the detection of the chemical compounds, having been extracted of the reaction batch of our enzyme assay. We hoped to detect the metabolits 7- methylxanthine, theobromine (3,7- dimethylxanthine) and caffeine (1,3,7- trimethylxanthine).
</P><P><B>Operational sequence:</B></P><P>First of all, the dried extract was solved in 200 µl of 70% methanol, mixed and than transferred into small glass tubes (LCMS tubes) and applied to the LCMS machine.
</P><P><B>Results:</B></P><P>There were indeed two signals for theobromine and caffeine, howerver, these signals appeared in all samples, suggesting that it is not specific and belongs to the &quot;background&quot;. A reason could be that the applied substrate xanthosine was not totally clean and contained small amounts of caffeine. If the enzymes had worked probably, we would have expected more significant signals. A reason could be that we had stored the enzymes for mor than two weeks at -80°C after the cell lyses and before using them for the enzyme assay. Maybe the experiment should be repeated with fresh isolated enzymes.
</P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Analysis_of_yeasts.27_genome_integration"> Analysis of yeasts' genome integration </SPAN></H3><P><B>Investigators:</B> Martin
</P><P><B>Aim:</B> Study of genome integration
<B>Procedure:</B> The clones of the experiment before were incubated in an over night culture, diluted to OD600 = 0.4 and put on a plate with G418 again to study the perseverance of antibiotics tolerance
</P><P><B>Results:</B> Of an overall number of 36 clones the experiment started with, 5 were already lost after the over night incubation (probably because of pipetting mistakes during the inoculation procedure on the 96-well plate). Of the 31 remaining plates, 21 were empty. The cfu of the positive plates were:
</P><UL><LI> 3
</LI><LI> 8
</LI><LI> 49
</LI><LI> 68
</LI><LI> 282
</LI><LI>&gt;1000
</LI><LI>&gt;10 000
</LI><LI>&gt;10 000
</LI></UL><P>The plan is to PCR at least two of the clones to gain knowledge why there's been such drastic differences in the integration efficiency and endurance.
</P></DIV><H1><SPAN class="mw-headline" id="Week_17">Week 17</SPAN></H1><DIV class="WWeek_17"><H2><SPAN class="mw-headline" id="Friday.2C_October_12th"><B>Friday, October 12th</B></SPAN></H2><DIV class="ethanol_inducible_promoter"><H3><SPAN class="mw-headline" id="Plating_of_yeast_for_future_experiments">Plating of yeast for future experiments</SPAN></H3><UL><LI><B>Preparation of working stocks</B></LI></UL><P>Untransformed <I>S. cerevisiae</I> INVSc1 from a glycerol stock were thawed on ice and spread on YDP-Plates. Incubation for 2.5 days at 30 °C.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plate</B></TD><TD><B>Description</B></TD></TR><TR><TD>1
</TD><TD>streaking of a small amout to obtain single colonies (inoculation loop)
</TD></TR><TR><TD>2
</TD><TD>20 µl spread on YPD-plate
</TD></TR><TR><TD>3
</TD><TD>50 µl spread on YPD-plate
</TD></TR><TR><TD>4
</TD><TD>100 µl spread on YPD-plate
</TD></TR></TBODY></TABLE><UL><LI><B>Checking of phenotype of glycerol stock</B></LI></UL><P>Untransformed <I>S. cerevisiae</I> INVSc1 from a glycerol stock were thawed on ice and spread on SC-Plates. Incubation for 2.5 days at 30 °C.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plate</B></TD><TD><B>Description</B></TD></TR><TR><TD>5
</TD><TD>streaking of a small amout to obtain single colonies (inoculation loop)
</TD></TR><TR><TD>6
</TD><TD>100 µl spread on SC-plate
</TD></TR></TBODY></TABLE><UL><LI><B>Checking of phenotype of glyerol stock and of SC-U selective plates</B></LI></UL><P>Untransformed <I>S. cerevisiae</I> INVSc1 from a glycerol stock were thawed on ice and spread on SC-U-plates. Incubation for 2.5 days at 30 °C.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plate</B></TD><TD><B>Description</B></TD></TR><TR><TD>7
</TD><TD>streaking of a small amout to obtain single colonies (inoculation loop)
</TD></TR><TR><TD>8
</TD><TD>100 µl spread on SC-plate
</TD></TR></TBODY></TABLE><UL><LI><B>Streaking of one colony of <I>S. cerevisiae</I> INVSc1 transformed with pTUM100-KlADH4-eGFP</B></LI></UL><P>One colony of <I>S. cerevisiae</I> INVSc1 transformed with pTUM100-KlADH4-eGFP was picked from plate &quot;SH 5.5, Klon 4, date 24.06.2012&quot;, resuspended in 200 µl of SC-U liquid Medium and spread on SC-U-plates. Incubation for 2.5 days at 30 °C. The aim of this experiment is to obtain a sufficient ammount of monoclonal yeast cells carrying the plasmid for future experiments.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plate</B></TD><TD><B>Description</B></TD></TR><TR><TD>9
</TD><TD>streaking of a small amout to obtain single colonies (inoculation loop)
</TD></TR><TR><TD>10
</TD><TD>50 µl spread on SC-plate
</TD></TR><TR><TD>11
</TD><TD>150 µl spread on SC-plate
</TD></TR></TBODY></TABLE></DIV></DIV><H1><SPAN class="mw-headline" id="Week_18">Week 18</SPAN></H1><DIV class="WWeek_18"><H2><SPAN class="mw-headline" id="Monday.2C_October_15th"><B>Monday, October 15th</B></SPAN></H2><DIV class="ethanol_inducible_promoter"><H3><SPAN class="mw-headline" id="Checking_of_plates_from_Friday.2C_October_12th">Checking of plates from Friday, October 12th</SPAN></H3><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Plate</B></TD><TD><B>Description</B></TD><TD><B>Result</B></TD></TR><TR><TD>1
</TD><TD>&quot;empty&quot; <I>S. cerevisiae</I> INVSc1 (inoculation loop), YPD-Agar
</TD><TD>single colonies of about 1-2 mm diameter
</TD></TR><TR><TD>2
</TD><TD>&quot;empty&quot; <I>S. cerevisiae</I> INVSc1 (20 µl), YPD-Agar
</TD><TD>thin lawn, plate discarded
</TD></TR><TR><TD>3
</TD><TD>&quot;empty&quot; <I>S. cerevisiae</I> INVSc1 (50 µl), YPD-Agar
</TD><TD>lawn, plate discarded
</TD></TR><TR><TD>4
</TD><TD>&quot;empty&quot; <I>S. cerevisiae</I> INVSc1 (100 µl), YPD-Agar
</TD><TD>thick lawn, plate discarded
</TD></TR><TR><TD>5
</TD><TD>&quot;empty&quot; <I>S. cerevisiae</I> INVSc1 (inoculation loop), SC-Agar
</TD><TD>single colonies of about 1-2 mm diameter, phenotype ok, plate discarded
</TD></TR><TR><TD>6
</TD><TD>&quot;empty&quot; <I>S. cerevisiae</I> INVSc1 (100 µl), SC-Agar
</TD><TD>thick lawn, phenotype ok, plate discarded
</TD></TR><TR><TD>7
</TD><TD>&quot;empty&quot; <I>S. cerevisiae</I> INVSc1 (inoculation loop), SC-U-Agar
</TD><TD>small single colonies!, <B>phenotype not ok!</B>, contradiction to result of plate 8!
</TD></TR><TR><TD>8
</TD><TD>&quot;empty&quot; <I>S. cerevisiae</I> INVSc1 (100 µl), SC-U-Agar
</TD><TD>no growth, <B>phenotype ok</B>, contradiction to result of plate 7!
</TD></TR><TR><TD>9
</TD><TD><I>S. cerevisiae</I> INVSc1, pTUM100-KlADH4-eGFP, from plate SH 5.5 (inoculation loop), SC-U-Agar
</TD><TD>single colonies of about 1-2 mm diameter
</TD></TR><TR><TD>10
</TD><TD><I>S. cerevisiae</I> INVSc1, pTUM100-KlADH4-eGFP, from plate SH 5.5 (50 µl), SC-U-Agar
</TD><TD>lawn
</TD></TR><TR><TD>11
</TD><TD><I>S. cerevisiae</I> INVSc1, pTUM100-KlADH4-eGFP, from plate SH 5.5 (150 µl), SC-U-Agar
</TD><TD>lawn
</TD></TR></TBODY></TABLE><P>The results from plate 7 &amp; 8 contradict each other. Why do untransformed yeast cells grow on plate 7 but not on plate 9? Possible explanation: Plates bad.
</P></DIV><DIV class="coumaryl"><H3><SPAN class="mw-headline" id="Analytical_digest_with_XbaI.2C_SpeI.2C_EcoRI_and_PstI">Analytical digest with XbaI, SpeI, EcoRI and PstI</SPAN></H3><P>Investigator: Daniela
</P><P><B>Aim of the experiment:</B> Test whether the enzymes are functional.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P718
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI / PstI
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>16.75 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P718
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI / XbaI
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>16.55 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P718
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.25 µl
</TD><TD>SpeI
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>15.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P718
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>15.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>1.5 µl
</TD><TD>P718
</TD></TR><TR><TD>0.25 µl
</TD><TD>EcoRI
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4 (10x)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>EcoRI was used from fermentas. Enzymes of fermentas seem not be compatible with NEB4 buffer. PstI did not work proberly. SpeI was frozen in the tube.
</P></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Quick_Change_mutagenesis_to_remove_AgeI_in_the_RFP-Generator_.28RFC25-compatible.29">  Quick Change mutagenesis to remove AgeI in the RFP-Generator (RFC25-compatible)  </SPAN></H3><P><B>Investigator: Mary</B></P><P>Aim of the experiment: Generation of an <A href="http://tools.ietf.org/html/rfc25" class="external mw-magiclink-rfc">RFC 25</A> compatible version of the RFP-Generator (pSB1C3).
</P><P><B>PCR</B><B>Reaction batch 1</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P899 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O102 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>Reaction batch 2</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>10x Pfu Ultra II buffer
</TD></TR><TR><TD>4 µl
</TD><TD>Plasmid P899 template
</TD></TR><TR><TD>0.5 µl
</TD><TD>1:10 dilution of O103 (10 pmol/µL)
</TD></TR><TR><TD>16.5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>0.5 µl
</TD><TD>dNTP mix
</TD></TR><TR><TD>0.5 µl
</TD><TD>Pfu Ultra II DNA polymerase (2.5 U / µl)
</TD></TR></TBODY></TABLE><P><B>PCR cycling parameters</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Segment</B></TD><TD><B>Cycles</B></TD><TD><B>Temperature</B></TD><TD><B>Time</B></TD></TR><TR><TD>1
</TD><TD>1
</TD><TD>95 °C
</TD><TD> 30 sec
</TD></TR><TR><TD>2
</TD><TD>10
</TD><TD>95°C
</TD><TD> 30 sec
</TD></TR><TR><TD>55°C
</TD><TD> 1 min
</TD></TR><TR><TD>67°C
</TD><TD> 4 min
</TD></TR></TBODY></TABLE><UL><LI>Having completed the PCR cycling parameters listed above both PCR reaction batches were mixed together and the cycling parameters listed above were one time more applied.
</LI><LI>Digestion of the parental DNA with DpnI: Addition of 1 µl DpnI to the PCR batch and incubate for 1 h at 37 °C.
</LI></UL><P><B> Transformation into <I>E.coli</I> Xl1-Blue</B><B>Operation Sequence</B></P><UL><LI> melting of 100 µl Ca-competent  <I>E.coli</I> XL1-Blue cells on ice
</LI><LI> addition of 1 µl of the PCR product
</LI><LI> incubation for 30 min on ice
</LI><LI> heat shock for 5 min at 37 °C
</LI><LI> transfer of cells to 1 ml LB-medium without antibiotics and incubate at 37°C and 180 rpm for 30 min
</LI><LI> plate 100 µl on an Amp-LB-plate
</LI><LI> sediment the leftover in a centrifuge (30 - 60 sec, 13 000 rpm) and resuspend the sediment in 100 µl LB-medium and plate it as well on an Chloramphenicol-LB-plate
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparative_digestion_and_gelelectrophoresis_of_pTUM100_and_expression_cassette_of_the_three_caffeine_enzymes_in_pSB1C3"> Preparative digestion and gelelectrophoresis of pTUM100 and expression cassette of the three caffeine enzymes in pSB1C3 </SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B> Cloning of the expression cassette containing CaXMT1, CaMXMT1 and CaDXMT1 into pTUM100
</P><P><B>Procedure:</B></P><UL><LI> preparative digestion with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20 µl
</TD><TD>pTUM100 or expression cassette
</TD></TR><TR><TD>5 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.5 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>1 µl
</TD><TD>XbaI (20 U/µl) (NEB)
</TD></TR><TR><TD>1 µl
</TD><TD>PstI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>23 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>50 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> preparative digestion reaction mix was incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 5 µl of DNA loading buffer (10x) was added to the 50 µl of reaction mix after digestion and loaded in the gel pockets.
</LI></UL><UL><LI> preparative gelelectrophoresis was performed at 70 V for 90 min on an 1% LMP-agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pTUM100
</TD><TD>expression cassette
</TD></TR><TR><TD>cut bond: 5000 bp
</TD><TD>cut bond: 6000 bp
</TD></TR></TBODY></TABLE><UL><LI> gel extraction
</LI><LI> concentration: 
</LI><LI> pTUM100: 17.4 ng/µl
</LI><LI> expression cassette: 28.2 ng/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Ligation_of_pTUM100_and_caffeine_expression_cassette"> Ligation of pTUM100 and caffeine expression cassette </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B> Cloning of the expression cassette containing CaXMT1, CaMXMT1 and CaDXMT1 into pTUM100
</P><P><B>Procedure:</B></P><UL><LI> Ligation (1:2)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.88 µl
</TD><TD>pTUM100 (17.4 ng/µl, 5000 bp)
</TD></TR><TR><TD>4.42 µl
</TD><TD>expression cassette (28.2 ng/µl, 6000 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>9.7 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation (1:1)
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.89 µl
</TD><TD>pTUM100 (17.4 ng/µl, 5000 bp)
</TD></TR><TR><TD>2.21 µl
</TD><TD>expression cassette (28.2 ng/µl, 6000 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>11.9 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batches.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_pTUM100_into_E..C2.A0coli"> Transformation of pTUM100 into <I>E. coli</I></SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B>There is no more pTUM100 left.
</P><P><B>Procedure:</B></P><UL><LI> 1µl of the miniprep product was added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on ampicilin plates.
</LI></UL><UL><LI> centrifugation at 14000 rpm, discard supernatant, resuspend in little LB medium, plate on ampicilin plate
</LI></UL><UL><LI> incubation over night (37°C)
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_October_16th"><B>Tuesday, October 16th</B></SPAN></H2><DIV class="integration"><H3><SPAN class="mw-headline" id="Transformation_of_Integration_vectors_containing_Thaumatin_and_Limonene_synthase_into_E.coli"> Transformation of Integration vectors containing Thaumatin and Limonene synthase into <I>E.coli</I></SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Getting enough DNA for preparative gel electrophoresis followed by gel extraction an transformation in yeast (genome integration)
</P><P><B>Procedure:</B> the following constructs were transformed:
</P><P>Thaumatin: Integ_Tef1P-Thaumatin-Tef1T
</P><P>Limonene: Integ_TEF1P-LSconsless-CYC-T and Integ_TEF1P-LS consless-TEF1T 
</P><UL><LI> 1 µl of the miniprep product was added to 100 µl of CaCl2 competent <I>E.coli</I> XL1-Blue cells on ice
</LI></UL><UL><LI> 30 min incubation on ice
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on ampicilin plates.
</LI></UL><UL><LI> centrifugation at 14000 rpm, discard supernatant, resuspend in little LB medium, plate on ampicilin plate
</LI></UL><UL><LI> incubation over night (37°C)
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_yeast_cultures_for_large_scale_expression_.281l.29_1_2"> Preparation of yeast cultures for large scale expression (1l) 1</SPAN></H3><P><B>Investigator:</B> Dennis, Saskia
</P><P><B>Aim of the experiment:</B></P><P>Detection of caffeine via LCMS
</P><P><B>Operational sequence:</B></P><P>Preparation of three 20ml yeast cultures (in selective SC-U 2% glucose medium), based on yeast clones transformed with pTUM104 vectors, each containing one of the three genes CaXMT1, CaMXMT1 and CaDXMT1 for having enough crude extract for further analysis. 
</P><P>A single colony which had grown on selective SC-U 2%glucose agar plates was used for inoculation of the 20ml liquid medium. The generated cell suspensions were incubated at 30°C over night.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_pTUM_expressioncassette_caffeine_into_E..C2.A0coli"> Transformation of pTUM_expressioncassette_caffeine into <I>E. coli</I></SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B>Cloning of the expression cassette containing CaXMT1, CaMXMT1 and CaDXMT1 into pTUM100 .
</P><P><B>Procedure:</B></P><UL><LI> 1µl of the miniprep product was added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of those cell suspension were plated on ampicilin plates.
</LI></UL><UL><LI> centrifugation at 14000 rpm, discard supernatant, resuspend in little LB medium, plate on ampicilin plate
</LI></UL><UL><LI> incubation over night (37°C)
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_over_night_cultures_of_E.coli_transformed_with_pTUM100">Preparation of over night cultures of <I>E.coli</I> transformed with pTUM100 </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Procedure:</B></P><UL><LI> Picking of 1 clone of <I>E.coli</I> transformed with pTUM_expressioncassette_caffeine (transformation 16th october)
</LI><LI> 4 ml LB medium + 4 µl ampicillin
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_October_17th"><B>Wednesday, October 17th</B></SPAN></H2><DIV class="integration"><H3><SPAN class="mw-headline" id="Picking_of_clones_of_transformed_E.coli_clones_containing_integration_vector_.2B_Thaumatin_or_Limonene_synthase"> Picking of clones of transformed <I>E.coli</I> clones containing integration vector + Thaumatin or Limonene synthase </SPAN></H3><P><B>Investigator:</B> Katrin
</P><P><B>Aim of the experiment:</B> Getting enough DNA for preparative gel electrophoresis followed by gel extraction an transformation in yeast (genome integration)
</P><P><B>Procedure:</B>
From each 100  µl plate, 7 clones were picked for a mini prep
</P></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Miniprep_of_E.coli_XL1-Blue_with_pTUM100"> Miniprep of <I>E.coli</I> XL1-Blue with pTUM100</SPAN></H3><P><B>Investigator:</B> Daniela
</P><P><B>Aim of the experiment:</B> Plasmid purification
</P><P>Operation Sequence:
</P><UL><LI> A miniprep of 4 clones <I>E.coli</I> XL1-Blue with pTUM100U was done using a Quiagen kit.
</LI></UL><P>
Concentration was measured using Nano Drop: 
</P><UL><LI> clone 1: 111.7 ng/µl
</LI><LI> clone 2: 90.4 ng/µl
</LI><LI> clone 3: 120.8 ng/µl
</LI><LI> clone 4: 89.9. ng/µl
</LI></UL></DIV><DIV class="vector_design"><H3><SPAN class="mw-headline" id="Analytical_digest_of_pTUM100_.28P924-927.29">Analytical digest of pTUM100 (P924-927)</SPAN></H3><P>Investigator: Daniela
</P><P><B>Aim of the experiment:</B> Test whether pTUM100 after new miniprep is still correct.
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>P924 / P925 / P926 / P927
</TD></TR><TR><TD>0.25 µl
</TD><TD>PvuI
</TD></TR><TR><TD>0.25 µl
</TD><TD>AflIII
</TD></TR><TR><TD>1.2 µl
</TD><TD>NEB3
</TD></TR><TR><TD>0.8 µl
</TD><TD>Tango
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>P924 / P925 / P926 / P927
</TD></TR><TR><TD>0.25 µl
</TD><TD>Xbal
</TD></TR><TR><TD>0.5 µl
</TD><TD>BglI
</TD></TR><TR><TD>2 µl
</TD><TD>NEB4
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA
</TD></TR><TR><TD>14.55 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>
1% gel
</P><UL><LI>lane 1: RFP QC Age 1
</LI><LI>lane 2: RFP QC Age 2
</LI><LI>lane 3: P924 (pTUM100 clone 1) digested with PvuI and AflIII
</LI><LI>lane 4: P925 (pTUM100 clone 2) digested with PvuI and AflIII
</LI><LI>lane 5: P926 (pTUM100 clone 3) digested with PvuI and AflIII
</LI><LI>lane 6: P927 (pTUM100 clone 4) digested with PvuI and AflIII
</LI><LI>lane 7: P924 (pTUM100 clone 1) digested with BglI and XbaI
</LI><LI>lane 8: P925 (pTUM100 clone 2) digested with BglI and XbaI
</LI><LI>lane 9: P926 (pTUM100 clone 3) digested with BglI and XbaI
</LI><LI>lane 10: P927 (pTUM100 clone 4) digested with BglI and XbaI
</LI><LI>lane 11: 1 kb DNA ladder
</LI></UL><P>Expected bands for digestion with PvuI and AflIII:
</P><UL><LI>1507
</LI><LI>1260
</LI><LI>1135
</LI><LI>766
</LI><LI>255
</LI></UL><P>Expected bands for digestion with BglI and XbaI:
</P><UL><LI>3391
</LI><LI>1532
</LI></UL><P>Result: All expected bands can be seen on the gel.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_of_P548_.28PCR58_.28consless.29_in_pYES.29_and_P899_.28pSB1C3.29">Preparative restriction digest of P548 (PCR58 (consless) in pYES) and P899 (pSB1C3)</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Prepare backbone of pSB1C3 and PCR 58 (LS without consensus sequence).
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20  µl
</TD><TD>Plasmid (P548 or P899)
</TD></TR><TR><TD>4  µl
</TD><TD>Buffer NEB4 (10x)
</TD></TR><TR><TD>0.48 µl
</TD><TD>BSA
</TD></TR><TR><TD>1  µl
</TD><TD>XbaI (10 U/µl)
</TD></TR><TR><TD>1  µl
</TD><TD>AgeI (10 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> plasmids were digested at 37 °C for 2.5 hours.
</LI></UL><P><B>Gelelectrophoresis</B></P><P>expected: 
</P><P>PCR 58: 1600 bp
</P><P>pSB1C3: 2000 bp
</P><P><B>Gel Extraction</B></P></DIV><DIV class="ethanol_inducible_promoter"><H3><SPAN class="mw-headline" id="Preculture_of_S._cerevisiae_transformed_with_pTUM100-KlADH4-eGFP">Preculture of <I>S. cerevisiae</I> transformed with pTUM100-KlADH4-eGFP</SPAN></H3><P>A large amount of cell material from plate 11 (see october 15th, = <I>S. cerevisiae</I> INVSc1 transformed with pTUM100-KlADH4-eGFP, monoclonal culture on plate) was used to inoculate 3 baffled 250 ml flasks containing 50 mL SC-U Medium (carbon source: 3 % glycerol) each. These pre-cultures were incubated at 30 °C, 180 rpm
</P></DIV><DIV class="ethanol_inducible_promoter"><H3><SPAN class="mw-headline" id="Simple_induction_experiment_in_SC-U_.28C-source:_3_.25_Glycerol.29">Simple induction experiment in SC-U (C-source: 3 % Glycerol)</SPAN></H3><P>6 culture tubes (13 ml) were used in this experiment:
</P><UL><LI> 3 containing 2 ml SC-U Medium (C-source: 3 % Glycerol), 2 % EtOH: <B>inducing conditions</B></LI><LI> 3 containing 2 ml SC-U Medium (C-source: 3 % Glycerol), 0 % EtOH: <B>uninducing conditions</B></LI></UL><P>cell material from plate 11 (see october 15th, <I>S. cerevisiae</I> INVSc1 transformed with pTUM100-KlADH4-eGFP, monoclonal culture on plate) was used to inoculate the tubes. The tubes were covered in aluminum foil (to avoid bleaching of expressed eGFP) and incubated at 30°C, 180 rpm.
</P></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparation_of_SC-medium_without_leucine_and_without_uracil">Preparation of SC-medium without leucine and without uracil</SPAN></H3><P>Investigator: Daniela
</P><P>2 l medium were prepared using the recipe of the pYES manual. 
</P><UL><LI> Leucine and uracil were omitted. All other amino acids and yeast nitrogen  base were dissolved in 1.8 l ddH2O and subsequently autoclaved.
</LI><LI> 40 g glucose were dissolved in 200 ml ddH2O and autoclaved separately.
</LI></UL></DIV><DIV class="thaumatin"><H3><SPAN class="mw-headline" id="Making_YPD_.2B_G418_plates"> Making YPD + G418 plates </SPAN></H3><P>Stored in cold room.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_over_night_cultures_of_E.coli_transformed_with_pTUM_expressioncassette_caffeine">Preparation of over night cultures of <I>E.coli</I> transformed with pTUM_expressioncassette_caffeine </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Procedure:</B></P><UL><LI> Picking of 1 clone of <I>E.coli</I> transformed with pTUM_expressioncassette_caffeine (transformation 16th october)
</LI><LI> 4 ml LB medium + 4 µl amicillin
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Preparation_of_yeast_cultures_for_large_scale_expression_.281l.29_2_2"> Preparation of yeast cultures for large scale expression (1l) 2</SPAN></H3><P><B>Investigator:</B> Dennis
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the preparation of three 200 ml yeast cultures (in selective SC-U 2% glucose medium), based on the 20 ml yeast cultures from 16.10., each containing one of the three genes CaXMT1, CaMXMT1 and CaDXMT1. 
</P><P><B>Operational sequence:</B></P><P>The OD600nm was measured and as much yeast culture was transferred to gain an OD600 of 0.4 in 200 ml.
The generated cell suspensions were incubated at 30°C over night.
</P></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_October_18th"><B>Thursday, October 18th</B></SPAN></H2><DIV class="integration"><H3><SPAN class="mw-headline" id="Midiprep_of_Thaumatin.2C_2Tef2_2_and_2_.283.293">Midiprep of Thaumatin, 2Tef2 2 and 2 (3)3</SPAN></H3><P>Investigator: Ingmar, Daniela
</P><P>Aim of the experiment: Plasmid purification.
</P><P>A Quiagen midiprep kit was used.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Ligation_of_pSB1C3_.28P899.29_with_P548_.28PCR58.29_and_PCR42_RL">Ligation of pSB1C3 (P899) with P548 (PCR58) and PCR42 RL</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Procedure</B></P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>P548
</TD><TD>6.77 µl
</TD></TR><TR><TD>P899
</TD><TD>1.238 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Substance</B></TD><TD><B>Volume</B></TD></TR><TR><TD>PCR42RL
</TD><TD>4.95 µl
</TD></TR><TR><TD>P899
</TD><TD>3.058 µl
</TD></TR><TR><TD>T4 ligase
</TD><TD>1 µl
</TD></TR><TR><TD>T4 ligase buffer
</TD><TD>1 µl
</TD></TR><TR><TD><B>TOTAL</B></TD><TD>=10 µl
</TD></TR></TBODY></TABLE><UL><LI>Incubation for 2 h at RT
</LI></UL><UL><LI> afterwards Trafo in <I>E.coli</I></LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_of_P850.2C_P372_.28pTUM104.29_and_P926_.28pTUM100.29">Preparative restriction digest of P850, P372 (pTUM104) and P926 (pTUM100)</SPAN></H3><P><B>Investigator:</B> Andrea, Daniela
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20  µl
</TD><TD>Plasmid (P850, P372, P926)
</TD></TR><TR><TD>4  µl
</TD><TD>Buffer NEB4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA
</TD></TR><TR><TD>1  µl
</TD><TD>XbaI (20 U/µl)
</TD></TR><TR><TD>1  µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>13.6 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> plasmids were digested at 37 °C for 3 hours.
</LI></UL><P><B>Gelelectrophoresis</B></P><UL><LI> 1 % gel (low melting point agarose)
</LI><LI> 70 V, 2.5 h
</LI></UL><P>expected: 
</P><P>P850 insert: 1361 bp
</P><P>P372 backbone = pTUM104: 5789 bp
</P><P>P926 backbone = pTUM100: 4845 bp
</P><P><B>Gel Extraction</B></P><UL><LI> Gel extraction was done using Qiagen Gel Extraction Kit.
</LI></UL><P>
Concentration was measured using Nano Drop:
</P><UL><LI> P850 insert was named P935 and had a concentration of 154.5 ng/µl
</LI></UL><UL><LI> pTUM104 digested with Xbal and PstI was named P936 and had a concentration of 153.9 ng/µl
</LI></UL><UL><LI> pTUM100 digested with Xbal and PstI was named P937 and had a concentration of 61.3 ng/µl
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Preparative_restriction_digest_of_P861.2C_P872_and_P294_.28pGADT7_AD.29">Preparative restriction digest of P861, P872 and P294 (pGADT7 AD)</SPAN></H3><P><B>Investigator:</B> Andrea, Daniela
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20  µl
</TD><TD>Plasmid (P861, P872)
</TD></TR><TR><TD>4  µl
</TD><TD>Buffer NEB4 (10x)
</TD></TR><TR><TD>0.4 µl
</TD><TD>BSA
</TD></TR><TR><TD>1  µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1  µl
</TD><TD>PstI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40.4 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>20  µl
</TD><TD>Plasmid (P294)
</TD></TR><TR><TD>4  µl
</TD><TD>Buffer NEB4 (10x)
</TD></TR><TR><TD>0.48 µl
</TD><TD>BSA
</TD></TR><TR><TD>1  µl
</TD><TD>EcoRI (20 U/µl)
</TD></TR><TR><TD>1  µl
</TD><TD>SbfI (20 U/µl)
</TD></TR><TR><TD>14 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=40.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> plasmids were digested at 37 °C for 3 hours.
</LI></UL><P><B>Gelelectrophoresis</B></P><UL><LI> 0.5 % gel
</LI><LI> 70 V, 2.5 h
</LI></UL><P>expected: 
</P><P>P861 insert: 6099 bp
</P><P>P872 insert: 6264 bp
</P><P>P294 yeast-backbone: 6006 bp
</P><P><B>Gel Extraction</B></P><UL><LI> Gel extraction was done using Qiagen Gel Extraction Kit.
</LI></UL><P>
Concentration was measured using Nano Drop:
</P><UL><LI> P861 insert was named P938 and had a concentration of 168.9 ng/µl
</LI></UL><UL><LI> P872 insert was named P939 and had a concentration of 307.7 ng/µl
</LI></UL><UL><LI> P294 yeast-backbone was named P940 and had a concentration of 133.4 ng/µl
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Ligation_of_P940.2BP938.2C_P940.2BP939.2C_P936.2BP891.2C_P937.2BP935"> Ligation of P940+P938, P940+P939, P936+P891, P937+P935 </SPAN></H3><P><B>Investigator:</B> Many thanks to Daniela and Andrea
</P><P><B>Aim of the experiment:</B> Ligation of P940+P938, P940+P939, P936+P891, P937+P935 for clone the Gal4 and LexA based light-switchable promoter system (LSPS) and luciferase reporters for the LSPSs.
</P><P><B>Procedure:</B></P><P>Ligation batch for P940+P938, P940+P939, P936+P891, P937+P935:
</P><UL><LI> Ligation batch for P940+P938
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.37 µl
</TD><TD>P940 (133.4 ng/µl, 6006 bp)
</TD></TR><TR><TD>0.6 µl
</TD><TD>P938 (168.9 ng/µl, 6099 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>16.03 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P940+P939
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.37 µl
</TD><TD>P940 (133.4 ng/µl, 6006 bp)
</TD></TR><TR><TD>0.34 µl
</TD><TD>P939 (307.7 ng/µl, 6264 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>16.29 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P936+P891
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.32 µl
</TD><TD>P936 (153.9 ng/µl, 5789 bp)
</TD></TR><TR><TD>0.68 µl
</TD><TD>P891 (36.7 ng/µl, 956 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>16 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Ligation batch for P937+P935
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>0.82 µl
</TD><TD>P937 (61.3 ng/µl, 4845 bp)
</TD></TR><TR><TD>0.28 µl
</TD><TD>P935 (154.5 ng/µl, 1361 bp)
</TD></TR><TR><TD>2 µl
</TD><TD>T4 ligase buffer (10x)
</TD></TR><TR><TD>1 µl
</TD><TD>T4 ligase (1 U/µl)
</TD></TR><TR><TD>15.9 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=20 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> Negative control was als prepared. Instead of the insert, the same volume of ddH2O was added to the reaction batches.
</LI></UL><UL><LI> Cycled ligation has been performed after following protocol in a thermocycler:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>100 cycles
</TD><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>12 °C
</TD><TD>60 s
</TD></TR><TR><TD>22 °C
</TD><TD>60 s
</TD></TR><TR><TD>Hold
</TD><TD>16 °C
</TD><TD>infinite
</TD></TR></TBODY></TABLE><P>Lid temperature = 37 °C
</P></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="Preparative_digest_of_integration_vector_containing_thaumatin_and_limonene_synthase">Preparative digest of integration vector containing thaumatin and limonene synthase</SPAN></H3><P>Investigator: Ingmar, Katrin
</P><P>Aim of the experiment: linearize integration vectors to increase integration efficiency
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>350 µl
</TD><TD>Plasmid (Integr.vector_Tef1P-Thaumatin-Tef1T, Integr.vector_Tef1P-LS-CYCT)
</TD></TR><TR><TD>40  µl
</TD><TD>Buffer NEB4 (10x)
</TD></TR><TR><TD>5 µl
</TD><TD>Sbf1 (20 U/µl)
</TD></TR><TR><TD>5 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=400 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> plasmids were digested at 37 °C for 3 hours.
</LI></UL></DIV><DIV class="integration"><H3><SPAN class="mw-headline" id="preparative_gel_electrophoresis_and_gel_extraction_of_cut_integrationvectors_containing_thaumatin_and_limonene_synthase">preparative gel electrophoresis and gel extraction of cut integrationvectors containing thaumatin and limonene synthase</SPAN></H3><P>Investigator: Ingmar, Katrin
</P><P>Aim of the experiment: 
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Miniprep_of_over_night_cultures_of_E._coli_transformed_with_pTUM_expressioncassette_caffeine"> Miniprep of over night cultures of <I>E. coli</I> transformed with pTUM_expressioncassette_caffeine </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Operational sequence:</B></P><P>The plasmid isolation was done as previously described. Elution was carried out by using 50µl of elution buffer.
</P><P>Afterwards, the concentration was determined with NanoDrop:
</P><UL><LI> pTUM_expressioncassette_caffeine clone 1: 192.9 ng/µl 
</LI><LI> pTUM_expressioncassette_caffeine clone 2: 148.6 ng/µl
</LI><LI> pTUM_expressioncassette_caffeine clone 3: 142.0 ng/µl
</LI><LI> pTUM_expressioncassette_caffeine clone 4: 174.1 ng/µl
</LI><LI> pTUM_expressioncassette_caffeine clone 5: 169.8 ng/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_pTUM_expressioncassette_caffeine"> Analytical digestion and gelelectrophoresis of pTUM_expressioncassette_caffeine </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of pTUM_expressioncassette_caffeine.
</P><P><B>Procedure:</B></P><UL><LI> Analytical digestion with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>0.25 ;µl
</TD><TD>XbaI (20 U/µl) (NEB)
</TD></TR><TR><TD>0.25 µl
</TD><TD>PstI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>17.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix were added to 2.5 µl of plasmid DNA.
</LI></UL><UL><LI> Analytical digestion reaction mix was incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 2 µl of DNA loading buffer (10x) was added to the 20 µl of reaction mix after digestion.
</LI></UL><UL><LI> 15 µl of these mixes were loaded in the gel pockets.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 90 min on an 1 % agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>pTUM_expressioncassette_caffeine 1
</TD><TD>pTUM_expressioncassette_caffeine 2
</TD><TD>pTUM_expressioncassette_caffeine 3
</TD><TD>pTUM_expressioncassette_caffeine 4
</TD><TD>pTUM_expressioncassette_caffeine 5
</TD></TR><TR><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Transformation_of_S._cerevisiae_INVSc1_with_pTUM_expressioncassette_caffeine"> Transformation of <I>S. cerevisiae</I> INVSc1 with pTUM_expressioncassette_caffeine </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B> Transformation of <I>S. cerevisiae</I> INVSc1 with pTUM_expressioncassette_caffeine for the constitutive expression of the caffeine expression cassette and further analysis.
</P><P><B>Procedure:</B></P><UL><LI> the yeast transformation was prepared as described in the kit manual of invitrogen.
</LI><LI> already prepared competent yeast cells were used
</LI><LI> 5 µl of each plasmid were used
</LI><LI> T1: pTUMcaffeine_expressioncassette 2 (P931)
</LI><LI> T2: pTUMcaffeine_expressioncassette 2 (P931)
</LI><LI> T3: pTUMcaffeine_expressioncassette 2 (P931)
</LI><LI> T4: NC --&gt; no plasmid --&gt; water
</LI><LI> T5: PC --&gt; pTUM100 (P925)
</LI></UL><LI> the cells were incubated at 30 °C for 2 days
</LI></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Start_of_the_large_scale_expression_.281l.29_3_2"> Start of the large scale expression (1l) 3</SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the expression of the three caffein genes CaDXMT1, CaMXMT1 and CaXMT1 in selective SC-U 2% galactose expression medium), based on the 200 ml yeast cultures from 17.10., each containing one of the three genes CaXMT1, CaMXMT1 and CaDXMT1. 
</P><P><B>Operational sequence:</B></P><P>The OD600nm was measured:
</P><UL><LI>CaDXMT1: 6.2
</LI><LI>CaMXMT1: 5
</LI><LI>CaXMT1: 4
</LI></UL><P>therefore 64.5 ml, 80 ml and 100 ml yeast culture were centrifuged, the supernatant discarded, the pellet resuspended in sc-U + Gal and transferred into selective SC-U 2% galactose expression medium and incubated at 30°C for 16 h.                                          
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_October_19th"><B>Friday, October 19th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_CaCl2_competent_E..C2.A0coli_XL1-Blue_with_ligation_products_P940.2BP938.2C_P940.2BP939.2C_P936.2BP891.2C_P937.2BP935"> Transformation of CaCl2 competent <I>E. coli</I> XL1-Blue with ligation products P940+P938, P940+P939, P936+P891, P937+P935 </SPAN></H3><P><B>Investigator:</B> Your mom!
</P><P><B>Aim of the experiment:</B> Transformation of CaCl2 competent <I>E. coli</I> XL1-Blue with ligation products P940+P938, P940+P939, P936+P891, P937+P935.
</P><P><B>Procedure:</B></P><UL><LI> CaCl2 competent <I>E. coli</I> XL1-Blue cells were put out from the stock in -80 °C freezer and were gently thawed on ice.
</LI></UL><UL><LI> 5 µl of the ligation products (P940+P938 (AmpR), P940+P939 (AmpR), P936+P891 (AmpR), P937+P935 (AmpR)) and their negative controls were added to 100 µl of CaCl2 competent <I>E. coli</I> XL1-Blue cells on ice.
</LI></UL><UL><LI> 30 min incubation on ice.
</LI></UL><UL><LI> 5 min heat shock at 37 °C
</LI></UL><UL><LI> Adding of 1 ml LB-medium to each tube.
</LI></UL><UL><LI> Incubation for 45 min at 37 °C in the 180 rpm cell-culture shaker.
</LI></UL><UL><LI> 100 µl of the cell suspensions were plated on suitable ampicillin plates.
</LI></UL><UL><LI> The cell suspensions were centrifuged for 2 min at 13000 rpm and the supernatant was dicarded.
</LI></UL><UL><LI> The pellets was resuspended in 100 µl of LB-medium and were plated on ampicillin plates.
</LI></UL><UL><LI> Incubation at 37 °C overnight.
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Picking_of_colonies_of_PCR42_and_P548">Picking of colonies of PCR42 and P548</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Procedure:</B></P><UL><LI> Picking of 6 clones of PCR42 (ligation 18th August) and 6 clones of P548 (ligation 18th August)
</LI><LI> add 4 µl Chloramphenicol to 4 ml LB medium
</LI><LI> Inbucation at 37°C (shaker) over night.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Large_scale_expression_.281l.29_4_2"> Large scale expression (1l) 4</SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the expression of the three caffein genes CaDXMT1, CaMXMT1 and CaXMT1 in selective SC-U 2% galactose expression medium. 
</P><P><B>Operational sequence:</B></P><P>The cells didn't grow, because the OD600nm was still around 0.5 as we started the day before. Therefore the cells weren't centrifuged, washed and lysed as planned before.
</P></DIV><H2><SPAN class="mw-headline" id="Saturday.2C_October_20th"><B>Saturday, October 20th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Picking_of_E..C2.A0coli_XL-Blue_cells.2C_transformated_with_ligation_products_P940.2BP938.2C_P940.2BP939.2C_P936.2BP891.2C_P937.2BP935"> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products P940+P938, P940+P939, P936+P891, P937+P935 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Picking of <I>E. coli</I> XL-Blue cells, transformated with ligation products P940+P938, P940+P939, P936+P891, P937+P935.
</P><P><B>Procedure:</B></P><UL><LI> 5 colonies of each transformation with P940+P938 (AmpR), P940+P939 (AmpR), P936+P891 (AmpR), P937+P935 (AmpR) were taken.
</LI></UL><UL><LI> Colonies were picked with pipette tips and transferred into a new cell-culture tube with 4 ml of LB-medium and ampicillin. These tubes were put overnight in a 180 rpm cell culture shaker at 37 °C.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="200_ml_expression_of_caffeine_expression_cassette_in_S._cerevisiae_.281.29"> 200 ml expression of caffeine expression cassette in <I>S. cerevisiae</I> (1)  </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B> Detection and functional analysis of three enzymes of the caffeine pathway.
</P><P><B>Procedure:</B>
Preparatory cultures:
</P><UL><LI><I>S. cerevisiae</I> INVSc1 transformed with pTUMcaffeine_expression_cassette
</LI><LI> picking of one clone
</LI><LI> 20 ml Sc-U + glucose
</LI><LI> 30 °C; 20 h
</LI></UL><UL><LI><I>S. cerevisiae</I> INVSc1 transformed with pTUM100
</LI><LI> picking of one clone
</LI><LI> 8 ml Sc-U + glucose
</LI><LI> 30 °C; 20 h
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Sunday.2C_October_21th"><B>Sunday, October 21th</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Miniprep_of_overnight_expansion_E..C2.A0coli_XL-Blue_cells_culture.2C_transformated_with_ligation_products_P940.2BP938.2C_P940.2BP939.2C_P936.2BP891.2C_P937.2BP935"> Miniprep of overnight expansion <I>E. coli</I> XL-Blue cells culture, transformated with ligation products P940+P938, P940+P939, P936+P891, P937+P935 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Miniprep of overnight expansion <I>E. coli</I> XL-Blue cells culture, transformated with ligation products P940+P938, P940+P939, P936+P891, P937+P935.
</P><P><B>Procedure:</B></P><UL><LI> Miniprep was done after manufacturer's protocol. (QIAGEN - QIAprep Spin Miniprep Kit)
</LI></UL></DIV><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Analytical_digestion_and_gelelectrophoresis_of_P951-P970"> Analytical digestion and gelelectrophoresis of P951-P970 </SPAN></H3><P><B>Investigator:</B> Jeff
</P><P><B>Aim of the experiment:</B> Analytical digestion and gelelectrophoresis of P951-P970 to see whether the light-switchable promoter systems and reporter constructs for the light-switchable promoter systems are successfully cloned into yeast vectors.
</P><P><B>Procedure:</B></P><UL><LI> Mastermix for analytical digestion of P951-P960 with EcoRI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>22 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2.75 µl
</TD><TD>EcoRI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>167.75 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=192.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix were added to 2.5 µl of plasmid DNA.
</LI></UL><UL><LI> Analytical digestion reaction mix was incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 2 µl of DNA loading buffer (10x) was added to the 20 µl of reaction mix after digestion.
</LI></UL><UL><LI> 10 µl of these mixes were loaded in the gel pockets.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 90 min on an 0.5% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P951 (P940+P938)
</TD><TD>P952 (P940+P938)
</TD><TD>P953 (P940+P938)
</TD><TD>P954 (P940+P938)
</TD><TD>P955 (P940+P938)
</TD><TD>P956 (P940+P939)
</TD><TD>P957 (P940+P939)
</TD><TD>P958 (P940+P939)
</TD><TD>P959 (P940+P939)
</TD><TD>P960 (P940+P939)
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation failed!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD></TR></TBODY></TABLE><UL><LI> Mastermix for analytical digestion of P961-P970 with XbaI+PstI-HF:
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>22 µl
</TD><TD>NEBuffer 4 (10x)
</TD></TR><TR><TD>2.2 µl
</TD><TD>BSA (100x)
</TD></TR><TR><TD>2.75 µl
</TD><TD>XbaI (20 U/µl) (NEB)
</TD></TR><TR><TD>2.75 µl
</TD><TD>PstI-HF (20 U/µl) (NEB)
</TD></TR><TR><TD>162.8 µl
</TD><TD>ddH2O
</TD></TR><TR><TD>=192.5 µl
</TD><TD><B>TOTAL</B></TD></TR></TBODY></TABLE><UL><LI> 17.5 µl of the mastermix were added to 2.5 µl of plasmid DNA.
</LI></UL><UL><LI> Analytical digestion reaction mix was incubated at 37 °C for 1.5 h.
</LI></UL><UL><LI> 2 µl of DNA loading buffer (10x) was added to the 20 µl of reaction mix after digestion.
</LI></UL><UL><LI> 10 µl of these mixes were loaded in the gel pockets.
</LI></UL><UL><LI> Analytical gelelectrophoresis was performed at 90 V for 90 min on an 1% agarose gel.
</LI></UL><TABLE cellspacing="0" border="1"><TBODY><TR><TD>1 kbp DNA ladder
</TD><TD>P961 (P936+P891)
</TD><TD>P962 (P936+P891)
</TD><TD>P963 (P936+P891)
</TD><TD>P964 (P936+P891)
</TD><TD>P965 (P936+P891)
</TD><TD>P966 (P937+P935)
</TD><TD>P967 (P937+P935)
</TD><TD>P968 (P937+P935)
</TD><TD>P969 (P937+P935)
</TD><TD>P970 (P937+P935)
</TD><TD>1 kbp DNA ladder
</TD></TR><TR><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation failed!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD><TD>Ligation successful!
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="200_ml_expression_of_caffeine_expression_cassette_in_S._cerevisiae_.282.29"> 200 ml expression of caffeine expression cassette in <I>S. cerevisiae</I> (2)  </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B> Detection and functional analysis of three enzymes of the caffeine pathway.
</P><P><B>Procedure:</B></P><UL><LI> OD600nm of preparatory culture of <I>S. cerevisiae</I> INVSc1 transformed with pTUMcaffeine_expression_cassette from october 20th: 4.51/ml 
</LI><LI> therefore 17.7 ml were transferred into 200 ml Sc-U + glucose (to reach an OD600nm of 0.4/ml)
</LI><LI> incubation at 30 °C for 20 h
</LI></UL><UL><LI> OD600nm of preparatory culture of <I>S. cerevisiae</I> INVSc1 transformed with pTUM100 from october 20th: 3.24/ml 
</LI><LI> therefore 6.2 ml were transferred into 50 ml Sc-U + glucose (to reach an OD600nm of 0.4/ml)
</LI><LI> incubation at 30 °C for 20 h
</LI></UL></DIV></DIV><H1><SPAN class="mw-headline" id="Week_18_2">Week 18</SPAN></H1><DIV class="WWeek_18"><H2><SPAN class="mw-headline" id="Monday.2C_October_22st"><B>Monday, October 22st</B></SPAN></H2><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Yeast_expression:_Pelletizing_20_h_after_expression_and_cell_lysis"> Yeast expression: Pelletizing 20 h after expression and cell lysis </SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B></P><P>Detection and functional analysis of three enzymes of the caffeine pathway. 
</P><P><B>Operational sequence:</B></P><P>At first, the OD600 of the suspensions was determined:
</P><UL><LI> S. cerevisiae INVSc1 transformed with pTUMcaffeine_expression_cassette: 4.95
</LI><LI> S. cerevisiae INVSc1 transformed with pTUM100: 4.85
</LI></UL><UL><LI> centrifugation at 4000 rpm for 25 minutes at 4°C and discarding of the supernatant
</LI><LI> resuspension of the pellet in 5 ml (pTUM100) or 20 ml (pTUMcaffeine_expression_cassette) breaking buffer without PMSF
</LI><LI> centrifugation at 4000 rpm for 25 minutes at 4°C and discarding of the supernatant
</LI><LI> resuspension of the pellet in 4.85 ml (pTUM100) or 19.8 ml (pTUMcaffeine_expression_cassette) breaking buffer with PMSF for the cell lysis
</LI><LI> addition of an equal amount of glass beads solved in breaking buffer 
</LI><LI> vortexing of the cell supsensions and storage on ice each for 30 seconds. This two steps were repeated 30 times. 
</LI><LI> centrifugation at 4000 rpm for 20 min at 4 °C
</LI><LI> storage of the supernatant 
</LI><LI> centrifugation of the beads again at 4000 rpm for 20 min at 4 °C. 
</LI><LI> 2 ml were stored at - 80 °C for enzymatic analysis and detection of caffeine via LCMS 
</LI><LI> the rest was used for dialysis, enzymatic analysis and detection of caffeine via LCMS 
</LI></UL><UL><LI> protein concentration in crude extracts: 
</LI><LI> S. cerevisiae INVSc1 transformed with pTUMcaffeine_expression_cassette: 11.1 µg/µl
</LI><LI> S. cerevisiae INVSc1 transformed with pTUM100: 13.5 µg/µl
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="SDS-_Page_of_protein_crude_extracts_from_the_cultivation_of_the_caffeine_expression_cassette"> SDS- Page of protein crude extracts from the cultivation of the caffeine expression cassette </SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiment:</B>  Detection and functional analysis of three enzymes of the caffeine pathway.
</P><P><B>Operational sequence:</B></P><P>The SDS- gel was prepared as previously described (12%). 20µg protein crude extract of each sample were loaded on the gel. Running- time about 2h at 120V.
Additionally to the prestained protein marker, an unstained marker was used.
Afterwards, the gel was immediately used for the western blot.
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Western_blot_of_samples_from_the_cultivation_of_the_caffeine_expression_cassette_after_SDS-PAGE">Western blot of samples from the cultivation of the caffeine expression cassette after SDS-PAGE</SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim:</B> Detection and functional analysis of three enzymes of the caffeine pathway
</P><P><B>Procedure:</B><B>preparation of solutions:</B> as described in the materials
</P><UL><LI>PBS-T0.1
</LI><LI>PBS-T0.1 with 3% BSA
</LI></UL><P><B>Blotting:</B></P><UL><LI>blotting of proteins on ImmobilonP Membrane (activating in methanol for 10 min)
</LI><LI>gel and membrane in transferbuffer for 20 min
</LI><LI>50mA, 1 h
</LI></UL><P><B>Blocking:</B></P><UL><LI>wash 3x7min with PBS-T0.1
</LI><LI>the membrane is blocked over night in PBS-T0.1 with 3% BSA at 4 °C on a shaking device
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vitro_reconstruction_of_the_caffeine_biosynthesis_pathway:_Enzyme_assay_2"> In vitro reconstruction of the caffeine biosynthesis pathway: Enzyme assay </SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiments:</B></P><P>Proof of enzyme function by reconstruction of the caffeine biosynthesis pathway.
</P><P><B>Operational sequence:</B></P><P>The enzyme assay was performed as described in the paper of Hiroshi Sano et al., 2003. 100µl reaction volumes were prepared as follows:
</P><UL><LI> 50 mM Tris/HCl (pH 8.0)
</LI><LI> 500 µM Xanthosine (not used in two negative controlls, see below)
</LI><LI> 1,5 mM SAM (S- adenosyl methionine)
</LI><LI> 200 µM MgCl<SUB>2</SUB></LI><LI> 500 µg protein (crude extract)
</LI></UL><P>(each reaction batch was filled up to 100µl with water)
</P><P>We prepared the following reaction batches (differing enzyme- and substrate-compositions):
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Batch</B></TD><TD><B>Enzyme content</B></TD></TR><TR><TD>NC 1
</TD><TD>no protein
</TD></TR><TR><TD>NC 2
</TD><TD>no substrates (SAM and xanthosine)
</TD></TR><TR><TD>pTUM100
</TD><TD>also a NC (45.05 µl)
</TD></TR><TR><TD>cassette 1
</TD><TD>500 µg crude extract isolated on october 22th (37.04 µl)
</TD></TR><TR><TD>cassette 2
</TD><TD>500 µg crude extract isolated on october 22th (37.04 µl)
</TD></TR><TR><TD>cassette 3
</TD><TD>500 µg crude extract isolated on october 22th (37.04 µl)
</TD></TR><TR><TD>Ü1
</TD><TD>supernatant with substrates
</TD></TR><TR><TD>Ü2
</TD><TD>supernatant without substrates
</TD></TR></TBODY></TABLE><UL><LI> All samples were incubated for 16 hours at 27°C (enzyme optima after Hiroshi Sano et al., 2003)
</LI></UL><UL><LI> After the reaction a chloroform extraction was performed.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vivo_reconstruction_of_the_caffeine_biosynthesis_pathway:_Enzyme_assay"> In vivo reconstruction of the caffeine biosynthesis pathway: Enzyme assay </SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiments:</B></P><P>Proof of enzyme function by adding Xanthosine and SAM to S. cerevisiae culture transformed with pTUMcaffeine_expression_cassette.
</P><P><B>Operational sequence:</B></P><UL><LI> 4 ml Sc-U + glucose
</LI><LI> + one clone S. cerevisiae culture transformed with pTUMcaffeine_expression_cassette
</LI><LI> + 500 µM Xanthosine
</LI><LI> + 1 µM SAM (S-adenosylmethionine)
</LI></UL><UL><LI> 48 h; 30 °C; shaking
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Miniprep_of_PCR42_in_pSB1C3_and_P548_in_pSB1C3_.28ligation_18.10.12.29"> Miniprep of PCR42 in pSB1C3 and P548 in pSB1C3 (ligation 18.10.12) </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Concentrations:</B></P><UL><LI> PCR42 (LS cons) in pSB1C3 (1): 464.2 ng/µl
</LI><LI> PCR42 (LS cons) in pSB1C3 (2): 96.4 ng/µl
</LI><LI> PCR42 (LS cons) in pSB1C3 (3): 232.1 ng/µl
</LI><LI> PCR42 (LS cons) in pSB1C3 (4): 259.7 ng/µl
</LI><LI> PCR42 (LS cons) in pSB1C3 (5): 175.7 ng/µl
</LI><LI> PCR42 (LS cons) in pSB1C3 (6): 201.0 ng/µl
</LI></UL><UL><LI> P548 (LS consless) in pSB1C3 (1): 672.2 ng/µl
</LI><LI> P548 (LS consless) in pSB1C3 (2): 167.6 ng/µl
</LI><LI> P548 (LS consless) in pSB1C3 (3): 289.5 ng/µl
</LI><LI> P548 (LS consless) in pSB1C3 (4): 193.9 ng/µl
</LI><LI> P548 (LS consless) in pSB1C3 (5): 493.6 ng/µl
</LI><LI> P548 (LS consless) in pSB1C3 (6): 620.0 ng/µl
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Tuesday.2C_October_23st"><B>Tuesday, October 23st</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Transformation_of_P955.2BP963_.28GAL4_based_LSPS_.2B_reporter.29_and_P959.2BP968_.28LexA_based_LSPS_.2B_reporter.29_in_Y190_S..C2.A0cerevisiae_with_S..C2.A0c._EasyComp_Transformation_Kit"> Transformation of P955+P963 (GAL4 based LSPS + reporter) and P959+P968 (LexA based LSPS + reporter) in Y190 <I>S. cerevisiae</I> with <I>S. c.</I> EasyComp Transformation Kit </SPAN></H3><P><B>Investigator:</B> Jeff, Ingmar
</P><P><B>Aim of the experiment:</B> Transformation of P955+P963 (GAL4 based LSPS + reporter) and P959+P968 (LexA based LSPS + reporter) in Y190 <I>S. cerevisiae</I> with <I>S. c.</I> EasyComp Transformation Kit.
</P><P><B>Procedure:</B></P><UL><LI> Co-transformation of P955+P963 (GAL4 based LSPS + reporter) and P959+P968 (LexA based LSPS + reporter) into Y190 S. cerevisiae cells was performed with the <I>S. c.</I> EasyComp Transformation Kit from Invitrogen after manufacturer's protocoll.
</LI></UL><UL><LI> For each transformation we used 5 µg of DNA in total.
</LI></UL><UL><LI> Transformation was performed after protocol with the only difference in the last step that we did't plated the transformated cell-suspension, but in liquid SC medium without LEU and URA (auxotrophic markers of plasmid with the light-switchable promoter systems and plasmid with the reporter constructs).
</LI></UL><UL><LI> We did not add phycocyanobilin in the preculture here.
</LI></UL><UL><LI> Also a negativ control (transformation without plasmids) was prepared.
</LI></UL><UL><LI> The transformated cell suspension were incubated for 2 days in a 30 °C shaker at 200 rpm.
</LI></UL><UL><LI> Addendum from October, 25th: only in the negativ control the cell suspension was still clear, the rest of the transformated cells were blurred. --&gt; Succesfull co-transformation of both light-switchable systems and reporter constructs
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Staining_with_Ponceau.27s_reagent_and_western_blot_analysis_of_protein_crude_extract"> Staining with Ponceau's reagent and western blot analysis of protein crude extract </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B>
Detection of the three enzymes of the caffeine pathway.
</P><P><B>Operational sequence:</B></P><UL><LI> the blocked western blot membrane was washed with PBS- T0.1 three times (3x 7min). 
</LI><LI> because of the use of unstained protein marker , we stained the western blot membrane with Ponceau's reagent (Ponceau's reagent; incubation for 5 min; washing with ddH2O; marking of unstained bonds) 
</LI><LI> washing the membrane with PBS - T0.1 again (7 minutes, 2x).
</LI><LI> incubation for 1h with detection reagent:
</LI><LI> 10ml 1xPBS
</LI><LI> 0,2% BSA (0,02g have been weighted in)
</LI><LI> 2µl anti body MABclassic
</LI></UL><LI> washing with PBS-T0.1 three times (3x 5min)
</LI><LI> incubation with the second antibody (anti mouse, fused with alk. phosphatase) for 1 h:
</LI><LI> 10ml 1xPBS
</LI><LI> 0,2% BSA (see above)
</LI><LI> 5µl AntiMouse-alk. Phosphatase conjugate (1/2000 dilution)
</LI><UL><LI> washing the membrane with PBS- T0.1 for (7 min; 3x)
</LI><LI> washing 1xPBS for 10 min (2x).
</LI><LI> shortly washing with AP buffer 
</LI><LI> incubation with a solution containing 15ml AP buffer, 45µl BCIP (50mg/ml in DMF) and 7,5µl NBT (75mg/ml in 70% DMF) for a few minutes, until clear bonds appear
</LI></UL><P>Western blot membran:
</P><P>From left to right:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Prestained protein marker (Page Ruler Plus)
</TD><TD> caffeine_expression_cassette
</TD><TD> pTUM100
</TD><TD>Unstained protein marker (pen marking)
</TD></TR></TBODY></TABLE></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vitro_reconstruction_of_caffeine_biosynthesis_pathway:_chloroform_extraction_2"> In vitro reconstruction of caffeine biosynthesis pathway: chloroform extraction </SPAN></H3><P><B>Investigator:</B> Simon
<B>Aim of the experiment:</B></P><P>Aim of the experiment was the extraction of potential reaction products, to be able to start a LCMS analysis in order to detect the synthesized caffeine.
</P><P><B>Operational sequence:</B></P><P>We added 1 ml of chloroform to the 100µl reaction batches, mixed a few seconds and stored the tubes for about 30 min at room temperature, to get a proper phase- separation. Afterwards, the lower phase (chloroform) was pipetted in tubes (glass tubes would have been better than plastic tubes) (1ml) and incubated for about 60 min at 60°C (=drying) until total chloroform removal.
</P><P>The tubes were then stored at -80°C over night until further usage (LCMS)
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Dialysis_of_protein_crude_extract_of_lysed_S._cerevisiae_transformed_with_caffeine_expression_cassette_from_22th_october"> Dialysis of protein crude extract of lysed <I>S. cerevisiae</I> transformed with caffeine_expression_cassette from 22th october </SPAN></H3><P><B>Investigator:</B> Saskia, Volker
<B>Aim of the experiment:</B></P><P>purification of the crude extract
</P><P><B>Operational sequence:</B></P><UL><LI> Tris
</LI><LI> 24 h 
</LI></UL></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Analytical_restriction_digest_of_miniprep_products_.2822.10.12.29"> Analytical restriction digest of miniprep products (22.10.12) </SPAN></H3><P><B>Investigator:</B> Andrea, Lara
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>2.5 µl
</TD><TD>plasmid DNA
</TD></TR><TR><TD>0.25 µl
</TD><TD>XbaI
</TD></TR><TR><TD>0.25 µl
</TD><TD>AgeI
</TD></TR><TR><TD>2 µl
</TD><TD>NEB 4
</TD></TR><TR><TD>0.2 µl
</TD><TD>BSA
</TD></TR><TR><TD>14.8 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI>incubation: 2 hours, 37°C
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Wednesday.2C_October_24st"><B>Wednesday, October 24st</B></SPAN></H2><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vivo_reconstruction_of_the_caffeine_biosynthesis_pathway:_chloroform_extraction"> In vivo reconstruction of the caffeine biosynthesis pathway: chloroform extraction </SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiments:</B></P><P>Proof of enzyme function by adding Xanthosine and SAM to S. cerevisiae culture transformed with pTUMcaffeine_expression_cassette.
</P><P><B>Operational sequence:</B></P><UL><LI> centrifugation: 4200 rpm; 4 °C; 15 min
</LI><LI> 100 µl supernatant + 1 ml of chloroform
</LI><LI> vortexing a few seconds
</LI><LI> storage of the tubes for about 30 min at room temperature
</LI><LI> discarding of the upper phase and remaining of the lower one (chloroform) in tubes 
</LI><LI> incubation for about 60 min at 60°C (=drying) until total chloroform removal
</LI></UL><P>The tubes were then stored at -80°C over night until further usage (LCMS) 
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vitro_reconstruction_of_caffeine_biosynthesis_pathway:_LCMS_analysis_of_extracted_compounds_2"> In vitro reconstruction of caffeine biosynthesis pathway: LCMS analysis of extracted compounds </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the detection of the chemical compounds, having been extracted of the reaction batch of our enzyme assay. We hoped to detect the metabolits 7- methylxanthine, theobromine (3,7- dimethylxanthine) and caffeine (1,3,7- trimethylxanthine).
</P><P><B>Operational sequence:</B></P><P>First of all, the dried extract was solved in 200 µl of 70% methanol, mixed and than transferred into small glass tubes (LCMS tubes) and applied to the LCMS machine.
</P><P><B>Results:</B>
theobromine was detected :)
See <A href="/wiki/images/9/9c/TUM12_LCMS_No1.pdf" class="internal" title="TUM12 LCMS No1.pdf"> LC/MS spectra</A>. Note that nu = nc (negative control) and u = &quot;ueberstand&quot; (supernatant)
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vitro_reconstruction_of_the_caffeine_biosynthesis_pathway_after_dialysis:_Enzyme_assay"> In vitro reconstruction of the caffeine biosynthesis pathway after dialysis: Enzyme assay </SPAN></H3><P><B>Investigator:</B> Saskia, Daniela
</P><P><B>Aim of the experiments:</B></P><P>Proof of enzyme function by reconstruction of the caffeine biosynthesis pathway.
</P><P><B>Operational sequence:</B></P><P>The enzyme assay was performed as described in the paper of Hiroshi Sano et al., 2003. 100µl reaction volumes were prepared as follows:
</P><UL><LI> 50 mM Tris/HCl (pH 8.0)
</LI><LI> 500 µM Xanthosine (not used in two negative controlls, see below)
</LI><LI> 1,5 mM SAM (S- adenosyl methionine)
</LI><LI> 200 µM MgCl<SUB>2</SUB></LI><LI> 500 µg protein (crude extract)
</LI></UL><P>(each reaction batch was filled up to 100µl with water)
</P><P>We prepared the following reaction batches (differing enzyme- and substrate-compositions):
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>Batch</B></TD><TD><B>Enzyme content</B></TD></TR><TR><TD>NC 1
</TD><TD>no protein
</TD></TR><TR><TD>NC 2
</TD><TD>no substrates (SAM and xanthosine)
</TD></TR><TR><TD>cassette after dialysis
</TD><TD>37.04 µl dialyzed protein extract
</TD></TR></TBODY></TABLE><UL><LI> All samples were incubated for 16 hours at 27°C (enzyme optima after Hiroshi Sano et al., 2003)
</LI></UL><UL><LI> After the reaction a chloroform extraction was performed.
</LI></UL></DIV><H2><SPAN class="mw-headline" id="Thursday.2C_October_25st"><B>Thursday, October 25st</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Induction_of_the_light-switchable_promoter_systems_in_Y190_S..C2.A0cerevisiae_with_red_light"> Induction of the light-switchable promoter systems in Y190 <I>S. cerevisiae</I> with red light </SPAN></H3><P><B>Investigator:</B> Jeff, Ingmar
</P><P><B>Aim of the experiment:</B> Induction of the light-switchable promoter systems in Y190 <I>S. cerevisiae</I> with red light. The induced light-switchable promoter system will hopefully switch on the gene expresesion of the reporter <I>Renilla</I> luciferase.
</P><P><B>Procedure:</B></P><UL><LI> The 2 days old transformated Y190 yeast cell suspension were diluted until it reach an OD<SUB>600</SUB> of ~0.5-0.6.
</LI></UL><UL><LI> Phycocyanobilin (PCB), which we extracted moths ago, was added after steril-filtration into the medium (c<SUB>end</SUB>(PCB)=25 µM).
</LI></UL><UL><LI> 3 samples of each system were induced with full red light intensity, 2 samples of each system with 1/10 intensity and 2 samples of each system remain non-induced.
</LI></UL><UL><LI> Every sample contains 3 ml cell suspension and was pipetted in a 4 ml cuvette.
</LI></UL><UL><LI> Before start of the experiment, we took one sample for luciferase assay and OD<SUB>600</SUB> measurement for getting the start value. (PCB was added, but we defined this sample as <I>without PCB</I> because PCB needs about 30 min to get into the cells)
</LI></UL><UL><LI> The induction red-light-box was set up as following: Every 5 min there was a 10 s lasting red-light pulse with intensities as described.
</LI></UL><UL><LI> Induction was performed overnight (15 h) at 30 °C in a 200 rpm shaker.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Theobromine_Assay"> Theobromine Assay </SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim of the experiments:</B></P><P>Proof of Coffeine Synthase function by adding Theobromine and SAM to <I>S. cerevisiae</I> culture transformed with pTUMcaffeine_expression_cassette.
</P><P><B>Operational sequence:</B></P><UL><LI> 50 mM Tris/HCl (pH 8.0)
</LI><LI> Theobromine Solution 500 µM: 0.009 g in 100 µl ddH2O (dissolved per ultrasound for 15 min)
</LI><LI> 1,5 mM SAM (S- adenosyl methionine)
</LI><LI> 200 µM MgCl<SUB>2</SUB></LI><LI> 500 µg protein (crude extract)
</LI></UL><P>(each reaction batch was filled up to 100µl with water)
</P><P>complete mixture
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>Theobromine
</TD></TR><TR><TD>37 µl
</TD><TD>Crude Extract
</TD></TR><TR><TD>0.8 µl
</TD><TD>MgCl2
</TD></TR><TR><TD>4.69 µl
</TD><TD>SAM
</TD></TR><TR><TD>47.5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Negative Control without substrate
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>37 µl
</TD><TD>Crude Extract
</TD></TR><TR><TD>0.8 µl
</TD><TD>MgCl2
</TD></TR><TR><TD>62.2 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><P>Negative Control without protein
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD><B>volume</B></TD><TD><B>reagent</B></TD></TR><TR><TD>10 µl
</TD><TD>Theobromine
</TD></TR><TR><TD>0.8 µl
</TD><TD>MgCl2
</TD></TR><TR><TD>4.69 µl
</TD><TD>SAM
</TD></TR><TR><TD>84.5 µl
</TD><TD>ddH2O
</TD></TR></TBODY></TABLE><UL><LI> Incubation at 27°C for ... h
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vitro_reconstruction_of_caffeine_biosynthesis_pathway_after_dialysis:_chloroform_extraction"> In vitro reconstruction of caffeine biosynthesis pathway after dialysis: chloroform extraction </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the extraction of potential reaction products, to be able to start a LCMS analysis in order to detect the synthesized caffeine.
</P><P><B>Operational sequence:</B></P><P>We added 1 ml of chloroform to the 100µl reaction batches, mixed a few seconds and stored the tubes for about 30 min at room temperature, to get a proper phase- separation. Afterwards, the lower phase (chloroform) was pipetted in tubes (glass tubes would have been better than plastic tubes) (1ml) and incubated for about 60 min at 60°C (=drying) until total chloroform removal.
</P><P>The tubes were then stored at -80°C over night until further usage (LCMS)
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vitro_reconstruction_of_caffeine_biosynthesis_pathway_after_dialysis:_LCMS_analysis_of_extracted_compounds"> In vitro reconstruction of caffeine biosynthesis pathway after dialysis: LCMS analysis of extracted compounds </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the detection of the chemical compounds, having been extracted of the reaction batch of our enzyme assay. We hoped to detect the metabolits 7- methylxanthine, theobromine (3,7- dimethylxanthine) and caffeine (1,3,7- trimethylxanthine).
</P><P><B>Operational sequence:</B></P><P>First of all, the dried extract was solved in 200 µl of 70% methanol, mixed and than transferred into small glass tubes (LCMS tubes) and applied to the LCMS machine.
</P><P><B>Results:</B></P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="In_vivo_reconstruction_of_caffeine_biosynthesis_pathway:_LCMS_analysis_of_extracted_compounds"> In vivo reconstruction of caffeine biosynthesis pathway: LCMS analysis of extracted compounds </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B></P><P>Aim of the experiment was the detection of the chemical compounds, having been extracted of the reaction batch of our enzyme assay. We hoped to detect the metabolits 7- methylxanthine, theobromine (3,7- dimethylxanthine) and caffeine (1,3,7- trimethylxanthine).
</P><P><B>Operational sequence:</B></P><P>First of all, the dried extract was solved in 200 µl of 70% methanol, mixed and than transferred into small glass tubes (LCMS tubes) and applied to the LCMS machine.
</P></DIV><H2><SPAN class="mw-headline" id="Friday.2C_October_26st"><B>Friday, October 26st</B></SPAN></H2><DIV class="light_switchable_promoter"><H3><SPAN class="mw-headline" id="Measurement_of_Renilla_luciferase_reporter_activity_of_GAL4_and_LexA_based_light-switchable_promoter_systems"> Measurement of <I>Renilla</I> luciferase reporter activity of GAL4 and LexA based light-switchable promoter systems </SPAN></H3><P><B>Investigator:</B> Jeff, Daniela
</P><P><B>Aim of the experiment:</B>: Measurement of <I>Renilla</I> luciferase reporter activity of GAL4 and LexA based light-switchable promoter systems.
</P><P><B>Procedure:</B></P><UL><LI><I>Renilla</I> luciferase reporter activity assay was performed with the Dual-Luciferase Reporter Assay System from Promega after manufacturer's protocol.
</LI></UL><UL><LI> All values has been normalized to OD<SUB>600</SUB> and dilution factors.
</LI></UL></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Theobromine_Assay:_chloroform_extraction"> Theobromine Assay: chloroform extraction </SPAN></H3><P><B>Investigator:</B> Saskia
</P><P><B>Aim of the experiment:</B></P><P>Proof of caffeine synthase function by adding theobromine and SAM to S. cerevisiae culture transformed with pTUMcaffeine_expression_cassette. 
</P><P><B>Operational sequence:</B></P><P>We added 1 ml of chloroform to the 100µl reaction batches, mixed a few seconds and stored the tubes for about 30 min at room temperature, to get a proper phase- separation. Afterwards, the lower phase (chloroform) was pipetted in tubes (glass tubes would have been better than plastic tubes) (1ml) and incubated for about 60 min at 60°C (=drying) until total chloroform removal.
</P><P>The tubes were then stored at -80°C over night until further usage (LCMS)
</P></DIV><DIV class="caffeine"><H3><SPAN class="mw-headline" id="Theobromine_Assay:_LCMS_analysis_of_extracted_compounds"> Theobromine Assay: LCMS analysis of extracted compounds </SPAN></H3><P><B>Investigator:</B> Ingmar
</P><P><B>Aim of the experiment:</B></P><P>Proof of caffeine synthase function by adding theobromine and SAM to S. cerevisiae culture transformed with pTUMcaffeine_expression_cassette. 
</P><P><B>Operational sequence:</B></P><P>First of all, the dried extract was solved in 200 µl of 70% methanol, mixed and than transferred into small glass tubes (LCMS tubes) and applied to the LCMS machine.
</P></DIV><DIV class="limonene"><H3><SPAN class="mw-headline" id="Headspace_GC-MS_of_limonene_2">Headspace GC-MS of limonene</SPAN></H3><P><B>Investigator:</B> Andrea
</P><P><B>Aim:</B> Check whether limonene is detectable in the brewed beer. Since extraction is time-consuming, SPME was done.
</P><UL><LI> The SPME needle was injected into the headspace. Incubation for 30 min at 45 °C.
</LI><LI> Injection into the GC (starting temperature: 40 °C)
</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:TU_Munich/Notebook/Labjournal">http://2012.igem.org/Team:TU_Munich/Notebook/Labjournal</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Munich/Notebook/Labjournal" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Munich/Notebook/Labjournal" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Munich/Notebook/Labjournal&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Munich/Notebook/Labjournal&amp;oldid=300636" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>