<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Munich_Modeling_Priors"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Munich/Modeling/Priors">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Munich/Modeling/Priors&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Modeling/Priors&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Modeling/Priors&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Munich/Modeling/Priors" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Munich/Modeling/Priors</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navigation"><UL><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Project/Overview">Project</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Overview">Results</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Vector_Design">Vector Design</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Limonene">Limonene</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Thaumatin">Thaumatin</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Caffeine">Caffeine</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Xanthohumol">Xanthohumol</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Constitutive_Promoter">Constitutive Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Light_Switchable_Promoter">Light-Switchable Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Ethanol_Inducible_Promoter">Ethanol-Inducible Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Genome_Integration">Genome Integration</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Brewing">Brewing</A></LI></UL><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Results/Biobricks">BioBricks</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Results/Biobricks">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Results/RFC">RFC</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Methods">Modeling</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Methods">Methods</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Priors">Priors</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Gal1_Promoter">Gal1 Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Yeast_Growth">Yeast Growth</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Overview">Human Practice</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/School_Visit">School Visit</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Public">Public Relations</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Comic_children">Comic</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Information_Day">Action Day</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Politics">Politics</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Movie">Movie</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Radio">Radio Interview</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Survey">Survey</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Labjournal">Notebook</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Protocols">Methods</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Meetings">Meetings</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Attribution">Attributions</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Team/Members">Team</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Activities">Activities</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Collaborations">Collaborations</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Prior_Data"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Prior Data</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Yeast_mRNA_Degradation_Rate"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Yeast mRNA Degradation Rate</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Yeast_Protein_Degradation_Rate"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Yeast Protein Degradation Rate</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Yeast_Transcription_Rate"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Yeast Transcription Rate</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Reference"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Reference</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Prior_Data"> Prior Data </SPAN></H1><H2><SPAN class="mw-headline" id="Yeast_mRNA_Degradation_Rate"> Yeast mRNA Degradation Rate </SPAN></H2><DIV class="show"><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Fig. 1</B> Genome-wide mRNA decay rate. Picture taken from Wang et. al. 2002</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Fig. 2</B> Approximated probability density for the genome-wide mRNA decay rate.</DIV></DIV></DIV><P>Data was obtained from the Paper [<A href="http://www.ncbi.nlm.nih.gov/pubmed/11972065" class="external text" rel="nofollow">Wang et al., 2002</A>] and processed by [<A href="http://arohatgi.info/webplotdigitizer/app/" class="external text" rel="nofollow">Rohatgi, 2012</A>] to obtain raw data.
Using a least-squared error approximation the distribution of the half life time in was approximated as <B>noncentral <I>t</I>-distribution</B> with parameters μ = 1.769 and ν = 20.59.
</P><PRE>
dataGraph = [
0.0018691649126431735,0.0016851538590669062
0.05978099456360327,0.01885629059542104
0.11548146330755026,0.21910551258377348
0.17122389948476902,0.396902157771723
0.2253457470848775,0.4417136917136917
0.2815821076690642,0.3552607791738227
0.3359848142456839,0.249812760682326
0.39216629434020744,0.19272091011221448
0.4465173486912618,0.11490683229813668
0.5026600896166115,0.07854043723608946
0.5569239808370243,0.04735863431515607
0.6111394480959699,0.04208365077930302
0.667233765059852,0.031624075102336016
0.7233280820237343,0.021164499425369035
0.777540321018582,0.017616637181854626
0.8373665112795547,0.010604847561369285
0.8897063570976615,0.008787334874291503
0.9420462029157682,0.006969822187213598
0.9999935434718044,0.005142624707842157
];
%scale the data
X = round(dataGraph(:,1)*90);
y = round(dataGraph(:,2)*2000);

k(1) = 1.769292045467269;
k(2) = 20.589996419308118;
k(3) = 24852.48237036381;

k=fminunc(@(z) sum((y-z(3)*nctpdf(X,z(1),z(2))).^2),k);
k=fminunc(@(z) sum((y-z(3)*nctpdf(X,z(1),z(2))).^2),k);
k=fminunc(@(z) sum((y-z(3)*nctpdf(X,z(1),z(2))).^2),k);
k=fminunc(@(z) sum((y-z(3)*nctpdf(X,z(1),z(2))).^2),k);
k=fminunc(@(z) sum((y-z(3)*nctpdf(X,z(1),z(2))).^2),k);
</PRE><P>The matlab nctpdf.m script needs to compute the limit of an infinite series to calculate the probabilities for the noncentral <I>t</I>-distribution. As this the function will be called several million times during the generation of samples, the computation was quite time consuming and the funtion was approximated using chebyshev interpolation [<A href="http://www2.maths.ox.ac.uk/chebfun/" class="external text" rel="nofollow">Trefethen, 2012</A>].
</P></DIV><H2><SPAN class="mw-headline" id="Yeast_Protein_Degradation_Rate"> Yeast Protein Degradation Rate </SPAN></H2><DIV><P>For the Degradation Rate the N-end rule [<A href="http://www.ncbi.nlm.nih.gov/pubmed/9112437" class="external text" rel="nofollow">Varshavsky, 1997</A>] served as approximation for the half life time.
It states that the half life time in <I>S. cerevisiae</I> can be approximated based on the amino acid after the initial start codon.
</P><TABLE class="wikitable"><TBODY><TR><TH> Residue ! Half-life
</TH></TR><TR><TD> Arg </TD><TD> 2 min
</TD></TR><TR><TD> Lys, Phe, Leu, Trp, His, Asp, Asn </TD><TD> 3 min
</TD></TR><TR><TD> Tyr, Gln </TD><TD> 10 min
</TD></TR><TR><TD> Ile, Glu </TD><TD> 30 min
</TD></TR><TR><TD> Pro </TD><TD>&gt; 5 h
</TD></TR><TR><TD> Cys, Ala, Ser, Thr, Gly, Val, Met </TD><TD>&gt; 30 h
</TD></TR></TBODY></TABLE><P>As these values do not give enough information to infer a proper distribution, only the two lower bounds 5 h and 30 h will serve as approximate lower bounds for the optimization routines.
</P></DIV><H2><SPAN class="mw-headline" id="Yeast_Transcription_Rate"> Yeast Transcription Rate </SPAN></H2><DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Fig. 3</B> Genome-wide transcription rate. Picture taken from Pelechano et al. 2010</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Fig. 4</B> Approximated probability density for the genome-wide transcription rate.</DIV></DIV></DIV><P>Data was obtained from the Paper [<A href="http://www.ncbi.nlm.nih.gov/pubmed/21103382" class="external text" rel="nofollow">Pelechano et al., 2010</A>] and processed by [<A href="http://arohatgi.info/webplotdigitizer/app/" class="external text" rel="nofollow">Rohatgi, 2012</A>] to obtain raw data.
Using a least-squared error approximation the distribution of the transcription rate was approximated as <B>log-normal distribution</B> with parameters μ = -1.492 and σ = 0.661;.
</P><PRE>
dataGraph = [
-1.8,0.3442950751957339
-1.6,1.3525375039897853
-1.4,3.5492668181220783
-1.2,11.28874786429094
-1.0,23.213749272450762
-0.8,26.31522126884587
-0.6,18.273455248681024
-0.4,7.913623476840467
-0.2,3.7755111620134825
0,1.9559339854677913
0.2,0.6458759692833385
0.4,0.12767315671880167
];

x = 10.^dataGraph(:,1);
y = dataGraph(:,2);

k(1) = -0.8;
k(2) = 0.2;
k(3) = 25;

k=fminunc(@(z) sum((y-z(3)*lognpdf(x,z(1),z(2))).^2),k);
k=fminunc(@(z) sum((y-z(3)*lognpdf(x,z(1),z(2))).^2),k);
k=fminunc(@(z) sum((y-z(3)*lognpdf(x,z(1),z(2))).^2),k);
k=fminunc(@(z) sum((y-z(3)*lognpdf(x,z(1),z(2))).^2),k);
</PRE></DIV><H2><SPAN class="mw-headline" id="Reference">Reference</SPAN></H2><UL><LI>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/21103382" class="external text" rel="nofollow">Pelechano et al., 2010</A>] Pelechano, V., Chávez, S., and Pérez-Ortín, J. E. (2010). A complete set of nascent transcription rates for yeast genes. <I>PLoS One</I>, 5(11):e15442.
</LI><LI>[<A href="http://arohatgi.info/webplotdigitizer/app/" class="external text" rel="nofollow">Rohatgi, 2012</A>] Rohatgi, A. (2012). <A href="http://arohatgi.info/webplotdigitizer/app/" class="external free" rel="nofollow">http://arohatgi.info/webplotdigitizer/app/</A>.
</LI><LI>[<A href="http://www2.maths.ox.ac.uk/chebfun/" class="external text" rel="nofollow">Trefethen, 2012</A>] Trefethen, N. (2012). <A href="http://www2.maths.ox.ac.uk/chebfun/" class="external free" rel="nofollow">http://www2.maths.ox.ac.uk/chebfun/</A>.
</LI><LI>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/9112437" class="external text" rel="nofollow">Varshavsky, 1997</A>] Varshavsky, A. (1997). The n-end rule pathway of protein degradation. <I>Genes Cells</I>, 2(1):13–28.
</LI><LI>[<A href="http://www.ncbi.nlm.nih.gov/pubmed/11972065" class="external text" rel="nofollow">Wang et al., 2002</A>] Wang, Y., Liu, C. L., Storey, J. D., Tibshirani, R. J., Herschlag, D., and Brown, P. O. (2002). Precision and functional specificity in mrna decay. <I>Proc Natl Acad Sci U S A</I>, 99(9):5860–5.
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:TU_Munich/Modeling/Priors">http://2012.igem.org/Team:TU_Munich/Modeling/Priors</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Munich/Modeling/Priors" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Munich/Modeling/Priors" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Munich/Modeling/Priors&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Munich/Modeling/Priors&amp;oldid=288633" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>