<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_TU_Munich_Notebook_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:TU_Munich/Notebook/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:TU_Munich/Notebook/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Notebook/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:TU_Munich/Notebook/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:TU_Munich/Notebook/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:TU Munich/Notebook/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="navigation"><UL><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Project/Overview">Project</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Overview">Results</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Vector_Design">Vector Design</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Limonene">Limonene</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Thaumatin">Thaumatin</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Caffeine">Caffeine</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Xanthohumol">Xanthohumol</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Constitutive_Promoter">Constitutive Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Light_Switchable_Promoter">Light-Switchable Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Ethanol_Inducible_Promoter">Ethanol-Inducible Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Genome_Integration">Genome Integration</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Project/Brewing">Brewing</A></LI></UL><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Results/Biobricks">BioBricks</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Results/Biobricks">Data Page</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Results/RFC">RFC</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Methods">Modeling</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Methods">Methods</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Priors">Priors</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Gal1_Promoter">Gal1 Promoter</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Modeling/Yeast_Growth">Yeast Growth</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Overview">Human Practice</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/School_Visit">School Visit</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Public">Public Relations</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Comic_children">Comic</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Information_Day">Action Day</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Politics">Politics</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Movie">Movie</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Radio">Radio Interview</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Human_Practice/Survey">Survey</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Labjournal">Notebook</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Protocols">Methods</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Labjournal">Labjournal</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Meetings">Meetings</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Notebook/Attribution">Attributions</A></LI><LI><B><A href="https://2012.igem.org/Team:TU_Munich/Team/Members">Team</A></B></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Members">Members</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Activities">Activities</A></LI><LI><A href="https://2012.igem.org/Team:TU_Munich/Team/Collaborations">Collaborations</A></LI></DIV><H1><SPAN class="mw-headline" id="Methods"> Methods </SPAN></H1><H2><SPAN class="mw-headline" id="1._Molecular_Biology_Methods">1. Molecular Biology Methods</SPAN></H2><H4><SPAN class="mw-headline" id="Isolation_of_Plasmid_DNA_from_E.coli_.28miniprep.29">Isolation of Plasmid DNA from <I>E.coli</I> (miniprep)</SPAN></H4><P>Plasmid DNA from E. coli was isolated from overnight cultures using the DNA extraction mini-prep kit (Qiagen). The principle of this method is alkaline lysis of bacterial cells followed by a selective immobilization of the plasmid DNA on a column, subsequent washing steps to remove impurities and the elution of plasmid DNA.
</P><H4><SPAN class="mw-headline" id="Isolation_of_Genomic_DNA_from_S.cerevisiae">Isolation of Genomic DNA from <I>S.cerevisiae</I></SPAN></H4><P>Extraction of genomic DNA from <I>S.cerevisiae</I> was done using a protocol by Lõoke et. al (2011). This method is based on yeast cell lysis with a LiOAc-SDS-solution followed by DNA precipitation with ethanol.
For DNA isolation, 100 µl of a stationary overnight yeast culture was pelleted and resuspended in 100 µl of 200 mM LiOAc 1 % SDS. Cells were incubated for 5 minutes at 70 °C. DNA was precipitated by adding 300 µl of 96 % ethanol and subsequent vortexing. After centrifugation for 3 minutes at 13400 rpm and discarding the supernatant, the DNA was washed with 500 µl of 70 % Ethanol an centrifuged once more. The supernatant was discarded and DNA was resuspended in 100 µl 1x TE buffer. Remaining cell debris was centrifuged down and the supernatant was transferred into a new tube. Another precipitation step with isopropanol was done to purify the genomic DNA for following PCR reactions.
</P><H4><SPAN class="mw-headline" id="Determination_of_DNA_Concentration">Determination of DNA Concentration</SPAN></H4><P>DNA concentration was measured using a NanoDrop Spectrophotometer by Thermo Scientific. The concentration was calculated after determination of DNA specific absorbance at 260 nm. Furthermore, the ratio of sample absorbance at 260 and 280 nm as well as at 260 and 230 nm were measured to specify the purity of the samples. A ratio of 260/280 of ~1.8 is generally accepted as “pure” for DNA. If the ratio is appreciably lower, it may indicate contamination with proteins. The 260/230 nm ratio indicates contamination with thiocyanates and phenolate ions since these absorb at 230 nm. The value is expected to be in the range of 1.8-2.2 in case DNA is relatively pure.
</P><H4><SPAN class="mw-headline" id="Agarose_Gel-Electrophoresis">Agarose Gel-Electrophoresis</SPAN></H4><P>Agarose gel-electrophoresis was used to separate double-stranded DNA fragments by length. Ethidium bromide was applied as a nucleic acid stain (Sambrook et al., 1989). This method was used for the restriction analysis of plasmids (analytical gel-electrophoresis) as well as for the isolation of DNA fragments (preparative gel-electrophoresis). After preparative gel-electrophoresis, the bands were cut out and purified using a Qiagen Gel extraction kit.
</P><H4><SPAN class="mw-headline" id="Polymerase_Chain_Reaction_.28PCR.29">Polymerase Chain Reaction (PCR)</SPAN></H4><P>Polymerase chain reaction (PCR) was used for the selective amplification of desired DNA fragments (for example from a plasmid). Primers were designed for the desired target sequences. 
The PCR reaction was divided in to three steps which were repeated up to 30 times. Firstly, the DNA template strand was heat-denatured at 95 °C to produce single-stranded DNA. Secondly, the temperature of the reaction batch was lowered to 55 – 60 °C to allow the primers to bind. Thirdly, the temperature was raised to 72 °C. This enabled the DNA polymerase to synthesize the other DNA strand. Special PCR methods that were used include colony and genomic PCR.
</P><H5><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H5><P>A method to allow for higher throughput of clone screenings. Colonies were picked with a sterile toothpick or pipet tip. Some of the cells were smeared onto the wall of the PCR tube. Subsequently the toothpick was put into a cell culture tube with LB-medium and suitable antibiotic. Colony PCR was performed using OneTaq Hot Start DNA Polymerase (Qiagen) following this temperature scheme:
</P><TABLE cellspacing="0" border="1"><TBODY><TR><TD>Initial denaturation
</TD><TD>94 °C
</TD><TD>10 min
</TD></TR><TR><TD>30 cycles
</TD><TD>95 °C
</TD><TD>30 s
</TD></TR><TR><TD>59 °C
</TD><TD>30 s
</TD></TR><TR><TD>68 °C
</TD><TD>1 min 55 sec
</TD></TR><TR><TD>Final extension
</TD><TD>68 °C
</TD><TD>5 min
</TD></TR><TR><TD>Hold
</TD><TD>4 °C
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="Purification_of_PCR_products">Purification of PCR products</SPAN></H5><P>PCR products were purified using the PCR purification kit by Qiagen.
</P><H4><SPAN class="mw-headline" id="Dephosphorylation_of_DNA">Dephosphorylation of DNA</SPAN></H4><P>Dephosphorylation of digested PCR products via Shrimp Alkaline Phosphatase (Fermaentas) was done to avoid religation of the insert and enhance ligation rate.
Before dephosphorylation was performed, digest solution of restriction enzymes and buffer were purified with PCR Purification Kit (Qiagen). Afterwards 1 µg sample DNA was mixed with 10 % of 10x SAP Buffer and 1 unit SAP. The mixture was incubated at 37 °C for 30 min. Inactivation occurred at 65 °C for 15 min.
</P><H4><SPAN class="mw-headline" id="DNA_Restriction_Enzyme_Digest">DNA Restriction Enzyme Digest</SPAN></H4><P>For the preparation of DNA fragments and the restriction analysis of plasmid DNA, DNA was cut using restriction endonucleases. Buffers and DNA concentrations were used according to the manufacturer's suggestions.
</P><H4><SPAN class="mw-headline" id="Ligation_.2F_Cycled_Ligation">Ligation / Cycled Ligation</SPAN></H4><P>After digestion with an restriction enzyme, plasmid fragments were inserted into vectors (which were cut with matching restriction enzymes) by ligation. The enzyme T4 ligase connected complementary overhangs of fragments by catalyzing the formation of the bond between the 5'phosphoryl group and the 3' hydroxyl group.
</P><H4><SPAN class="mw-headline" id="Oligohybridization_of_Single-Stranded_DNA">Oligohybridization of Single-Stranded DNA</SPAN></H4><P>Oligohybridization of oligo-nucleotides was used to create a mini MCS for cloning RFC10 compatible parts between promoter and terminator. This was achieved by using complementary oligo-nucleotides that contained the desired sequence with specific overhangs for cloning. For oligohybridization, 25 ml of 100 mM of forward and reverse oligos were put together in one tube and heated to 90 °C for 5 min. The samples were slowly cooled to room temperature in a styrofoam box overnight.
</P><H4><SPAN class="mw-headline" id="Site-Directed_Mutagenesis">Site-Directed Mutagenesis</SPAN></H4><P>Site-Directed Mutagenesis was used to mutate specific bases of DNA sequences. Therefore, specific primers, which bind at the same site and contain a mismatch at the specific base, were required. The original base pair that had to be replaced was replaced by the mismatch. The method works just as PCR by amplifying the desired product that contains the mismatch. Afterwards, the product was digested with the restriction enzyme DpnI to destroy the plasmids strands which do not contain the desired base pair exchange. The QuikChange Site-Directed Mutagenesis Kit by Agilent Technologies was used.
</P><H4><SPAN class="mw-headline" id="Genome_Integration">Genome Integration</SPAN></H4><P>Genome integration of DNA constructs was done using an existing biobrick as an integration vector (BBa K300001 (Pavia '10)). Prior to transformation of yeast cells, the integration vector was linearized with the restriction enzyme Sbf1 (because the frequency of integration is much higher for linearized plasmids). After linearization and preparative gel extraction of the vector, it was transformed into yeast cells using the S. c. EasyComp Transformation Kit by Invitrogen <I>S. cerevisiae</I> and G418 antibiotic plates.
</P><H4><SPAN class="mw-headline" id="Sequencing_of_Plasmid_DNA">Sequencing of Plasmid DNA</SPAN></H4><P>DNA constructs were sequenced by <A href="http://eurofinsdna.com/" class="external text" rel="nofollow">Eurofins mwg operon</A> using our own sequencing primers.
</P><H4><SPAN class="mw-headline" id="Gene_Synthesis">Gene Synthesis</SPAN></H4><P>Some of our genetic constructs were synthesized by GeneArt and iDT. For all our syntheses, the GeneArt® GeneOptimizer® was used for codon optimization in <I>S.cerevisiae</I>.
</P><H2><SPAN class="mw-headline" id="2._Protein_Biochemical_Methods">2. Protein Biochemical Methods</SPAN></H2><H4><SPAN class="mw-headline" id="Protein_Expression_in_S._cerevisiae">Protein Expression in <I>S. cerevisiae</I></SPAN></H4><P>Expression was induced by transferring overnight cultures of the INVSc1 strain from glucose medium into a medium that contained galactose. Transcription in INVSc1 strains (GAL1 promoter) is repressed in the presence of glucose. Transcription may be induced by removing glucose and adding galactose as a carbon source. Transferring cells from glucose- to galactose-containing medium causes the GAL1 promoter to become de-repressed and allows transcription to be induced.
</P><H4><SPAN class="mw-headline" id="Crude_Protein_Extraction_from_S.cerevisiae">Crude Protein Extraction from <I>S.cerevisiae</I></SPAN></H4><P>To detect the recombinant protein (for example by western blot) cell lysates from our yeast transformants were prepared. Cell lysates were produced using PMSF-containing breaking buffer and acid-washed glass beads to break the cell wall.
</P><H4><SPAN class="mw-headline" id="SDS_Polyacrylamide_Gelelectrophoresis_.28SDS-PAGE.29">SDS Polyacrylamide Gelelectrophoresis (SDS-PAGE)</SPAN></H4><P>Firstly, glass plates were wiped with 70 % ethanol and then assembled onto a setting rig. No rubber spacers were added as they were already fixed to the glass plates. 
A 15 % resolving gel was made as follows:
</P><UL><LI> 5 ml protogel
</LI><LI> 2.5 ml 4 x Lower Tris (pH 8.8)
</LI><LI> 2,5 ml H2O
</LI><LI> 50 μl ammonium persulphate (APS) (10 %)
</LI><LI> 2,5 μl N,N,N’,N’-tetramethylethylenediamine (TEMED) 
</LI></UL><P>The tube was mixed thoroughly and added to the setting rig between the glass plates, covered with water and left to set for about 30 minutes. Then the water was poured off and a stacking gel was prepared as follows:
</P><UL><LI> 1 ml protogel
</LI><LI> 1.5 ml 4 x Upper Tris
</LI><LI> 3,5 ml H2O
</LI><LI> 36 μl ammonium persulphate (APS) (10 %) 
</LI><LI> 3 μl N,N,N’,N’-tetramethylethylenediamine (TEMED) 
</LI></UL><P>All substances were mixed by inversion. 1 ml was applied on top of the already set resolving gel and a comb was put in. Once the gel was set, the comb was taken out and the wells were cleaned out with sterile water. The set gels were removed from the setting rig and placed in the running rig. 1x running buffer was poured into the rig ensuring the plates were covered.
For protein preparation, 30 µg protein in 10 µl H20 and 2,5 µl 5 x Laemmli buffer were denatured for 5 minutes at 95 °C. Then 6 µl marker (unstained marker for coomassie-staining or prestained marker for Western Blot) was put into a well. The remaining wells were filled with 12,5 µl of the protein that was to be analyzed. The SDS-PAGE was performed at 120 V for about 1,5 h. For coomassie staining, the gel was incubated in coomassie staining dye for 20 minutes, then put into the first decolorizing solution for 20 minutes. Finally. the gel was put into a second decolorizing solution until the background color was gone. Another method that was used for protein staining was silver staining.
</P><H4><SPAN class="mw-headline" id="Western_Blot">Western Blot</SPAN></H4><P>After SDS-PAGE, gels were transferred to a nitrocellulose membrane in transfer buffer (20 ml 5x SDS buffer, 20 ml methanol, 60 ml H2O) at 500 mA for 1h. Membranes were washed 3x 15 min in PBS-T0.1 (PBS + 0,1 % v/v Tween 20) and subsequently blocked with 3 % BSA for one hour. For antibody detection, membranes were washed 3x 15 min in PBS-T0.1 and then incubated in detection solution containing the antibody straptavidin-AP (1:4000 in PBS-T0.1) for one hour. The Western blot was washed 2x 10 min in PBS-T0.1 and then 2x 10 min in PBS. Afterwards, the developing solution (15 ml alkaline phosphatase buffer, 45 µl BCIP (50 mg/ml in DMF) and 7,5 µl NBT (75 mg/ml in 70 % DMF) was added. After the appearance of bands, the blot was washed and stored in water.
</P><H4><SPAN class="mw-headline" id="Enzyme_Assay_and_Extraction_of_Limonene_Synthase">Enzyme Assay and Extraction of Limonene Synthase</SPAN></H4><P>We used an optimized protocol of <A href="http://www.ncbi.nlm.nih.gov/pubmed/17662687" class="external text" rel="nofollow">Landmann et al, 2007</A> to test functionality of purified limonene synthase. The enzyme assay was carried out in a total volume of 500 µl containing buffer (25 mM Tris-Cl, pH 7.5, 5 % glycerol, 1 mM DTT) supplemented with cofactors (10 mM MgCl2, 1 mg/ml BSA) with successive addition of 50 µM substrate (geranyl pyrophosphate, dissolved in DMSO) and 10 µg purified recombinant enzyme (extracted limonene synthase, after purification). The mixture was gently overlaid with 1 ml pentane and incubated at room temperature for 15 minutes. The reaction was stopped by vigorous mixing and centrifugation (5 min, 5000g) to separate phases. The solvent phase (upper phase) was attached to a pasteur pipette containing glass wool with sodium sulfate for drying the solvent phase. Afterwards, the combined extracts were reduced to approximately 300 µl under a stream of nitrogen.
The pentane extracts were analyzed by gas chromatography-mass spectrometry to identify the enzymatically synthesized products. An aliquot of each sample (0.5 µl) was injected into &quot;5890 Series II GC&quot; coupled to a &quot;Finnigan Mat 55 S MS&quot;.
</P><H4><SPAN class="mw-headline" id="Headspace_GC-MS_of_Limonene">Headspace GC-MS of Limonene</SPAN></H4><P>Because Limonene is an secretory protein <A href="http://www.ncbi.nlm.nih.gov/pubmed/21310602" class="external text" rel="nofollow">[Misawa, 2011</A>] we expected an arbitrarily amount of Limonene outside the cells. To check this predication we detected Limonene via Headspace GC-MS in the yeast cell culture supernatant. Therefore the preparatory cell culture was induced with galactose after 24 hours (see '<I>Protein expression in S.cerevisiae</I>). For further 24 hours the protein expression proceeded. Afterwards the SPME needle was injected into the headspace and incubated for 30 min at 45 °C before injection into GC.
</P><H2><SPAN class="mw-headline" id="3._Microbiological_Methods">3. Microbiological Methods</SPAN></H2><H4><SPAN class="mw-headline" id="Cultivation_of_E.coli">Cultivation of <I>E.coli</I></SPAN></H4><P><I>E.coli</I> strain XL1-Blue was cultivated in LB-medium (lysogeny broth) and LB plates. 
for the preparation of 1 liter of LB, dissolve the following and autoclave:
</P><UL><LI> 10 g tryptone
</LI><LI> 5 g yeast extract
</LI><LI> 10 g NaCl
</LI></UL><P>For making plates, add 15 g bacto-agar before autoclaving.
</P><H4><SPAN class="mw-headline" id="Cultivation_of_S.cerevisiae">Cultivation of <I>S.cerevisiae</I></SPAN></H4><P><I>S.cerevisiae</I> strain INVSc1 was cultivated using the following media and plates:
</P><P>SC-Uracil minimal Medium and plates
for 10x 50 ml amino acid aliquots: Dissolve the following reagents in 500 ml deionized water:
</P><UL><LI> 1 g of adenine, arginine, cysteine, leucine, lysine, threonine, tryptophan and
</LI><LI> 0.5 g of aspartic acid, histidine, isoleucine, methionine, phenylalanine, proline, serine, tyrosine, valine.
</LI><LI> Autoclave at 15 psi, 121 °C for 20 minutes.
</LI><LI> Make 50 ml aliquots, store at -22 °C.
</LI></UL><P>SC-U medium + Glucose (minimal medium) or Galactose (induction medium):
</P><UL><LI> Add 6.7 g Yeast Nitrogen Base and 850 ml deionized water to the 50 ml amino acid aliquot. Autoclave. Add 100 ml autoclaved/filter sterilized 20 % glucose (minimal medium) or 20% galactose (induction medium), store at room temperature.
</LI></UL><P>SC-plates + Glucose:
</P><UL><LI> Add 6.7 g Yeast Nitrogen Base and 850 ml ELGA and 20 g agar to the 50 ml aliquot. Autoclave.Add 100 ml autoclaved/filter sterilized 20 % glucose.5. Pour plates and allow to harden. Invert the plates and store at 4 °C. Plates are stable for 6 months.
</LI></UL><H4><SPAN class="mw-headline" id="Heat_Shock_Transformation_of_E.coli_with_Plasmid_DNA">Heat Shock Transformation of <I>E.coli</I> with Plasmid DNA</SPAN></H4><P>Before transformation, CaCl<SUB>2</SUB> competent cells were produced after Cohen et al., 1972. For the production of competent cells, 50 ml LB medium were inoculated with an overnight culture of the used ‘’E.coli’’ strain and incubated at 37 °C, 180 rpm. After an OD <SUB>550</SUB> of 0,5 was reached, the culture was centrifuged for 4 minutes at 5000 g for 10 minutes. The pellet was then resuspended in 40 ml pre-chilled in 0,1 M MgCl<SUB>2</SUB> solution, centrifuged again and resuspended in 20 ml of pre-chilled 0,05 M CaCl<SUB>2</SUB> solution. After 30 minutes of incubation on ice, the cells were centrifuged and resuspended in 2 ml 0,05 M CaCl<SUB>2</SUB> solution, 15 % v/v glycerol. The competent cells were aliquoted and stored an – 80 °C.
For the transformation, 100 µl competent cells and 1 ng plasmid or 5 µg of a ligation product were mixed and incubated for 30 minutes on ice. Afterwards, the cells were heat shocked at 37 °C for 5 minutes, then mixed with 2 ml LB medium and incubated at 180 rpm and 37 °C for 30-45 minutes. The transformed cells were then plated on LB medium containing an antibiotic.
</P><H4><SPAN class="mw-headline" id="Transformation_of_S.cerevisiae">Transformation of <I>S.cerevisiae</I></SPAN></H4><P>Yeast cells were transformed using the S. c. EasyComp Transformation Kit by Invitrogen. Firstly, cells were grown in YPD medium to mid-log phase. Then, cells were pelleted and washed with a washing solution. Afterwards, cells were pelleted again and washed with Lithium cation solution in order to make the cells competent. Finally, the cells were aliquoted and stored at -80 °C. 
For yeast transformation, frozen competent cells were mixed with up to 5 mg of plasmid DNA and transformation solution. The transformation batch was mixed by vortexing and incubated at 30 °C to induce uptake of DNA for one hour. The transformed cells were plated on selective plates and grown for 2-4 days at 30 °C.
</P><H4><SPAN class="mw-headline" id="Genome_Integration_2">Genome Integration</SPAN></H4><P>Genome integration of DNA constructs was done using an existing biobrick as an integration vector (BBa K300001 (Pavia '10)). Prior to transformation of yeast cells, the integration vector was linearized with the restriction enzyme Sbf1 because the frequency of integration is much higher for linearized plasmids. After linearization and preparative gel extraction of the vector it was transformed into yeast cells using the S. c. EasyComp Transformation Kit by Invitrogen <I>S. cerevisiae</I> and G418 antibiotic plates.
</P><H2><SPAN class="mw-headline" id="4._Chemical_Methods">4. Chemical Methods</SPAN></H2><H4><SPAN class="mw-headline" id="Phycocyanobilin_.28PCB.29_Extraction_from_Dried_Spirulina_Platensis_Powder">Phycocyanobilin (PCB) Extraction from Dried Spirulina Platensis Powder</SPAN></H4><P>Phycocyanobilin (PCB) extraction from dried Spirulina platensis powder by methanolysis was done according to a protocol by Jim Tepperman (<A href="http://openwetware.org/images/2/27/Y2H_documentation.pdf" class="external free" rel="nofollow">http://openwetware.org/images/2/27/Y2H_documentation.pdf</A>). 50 g Spirulina powder were suspended in 1.5 liter H2O (30 ml/g) in a flask covered with aluminum foil. The mixture was stirred for 10 minutes and then centrifuged (GS3) at 8000 RPM at 4 °C for 1 hr. The supernatant was decanted and 15 g TCA (final concentration of 1 % (w/v)) was added. The mixture was stirred for 1 hr in the dark (aluminum foil cover) at 4 °C and then centrifuged at 8000 RPM at 4 °C for 10 min. The supernatant was discarded. The pellets were washed on ice several times with MeOH until the formerly green supernatant turned blue. The pellets were stored at 20 °C overnight, wrapped in aluminum foil. Since the free chromophore is very susceptible to photobleaching, the following steps were performed under green ‘safelight’ conditions in a darkroom. The next day, all pellets were resuspended in a final volume of 500 ml MeOH. Then the suspension was heated in a water bath at 70– 75 °C with a condensing coil cooled with tap water for 5-8 hrs. The suspension was then centrifuged at 8000 RPM, GS3 rotor, 4 °C for 20 min and the supernatant was filtered through miracloth. The filtered suspension and the pellets were stored at 20 °C overnight in the dark. The next day, a second methanolysis of the pellets from the first methanolysis was done. The two supernatants were pooled, the volume of MeOH was reduced to 50 ml using a rotary evaporator and then transferred separatory funnel where it was diluted with 100 ml water. The solution was mixed with 50 ml chloroform and shook to obtain the PCB through solvent extraction. The solvent extraction with chloroform was repeated until the aqueous phase was alsmost colorless. The chloroform was evaporated with a rotary evaporator and a stream of nitrogen gas was blown over the remaining PCB to remove residues of chloroform. The dried PCB as dissolved in DMSO, aliquoted and stored at -80 °C.
</P><H2><SPAN class="mw-headline" id="References:">References:</SPAN></H2><UL><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/20010584" class="external text" rel="nofollow"><B>Pubmed:</B> Prasher, 1995: Using GFP to see the light.(Review)</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/17662687" class="external text" rel="nofollow"><B>Pubmed:</B> Landmann et al, 2007: Cloning and functional characterization of three terpene synthases from lavender (Lavandula angustifolia).</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/21310602" class="external text" rel="nofollow"><B>Pubmed:</B> Misawa, 2011: Pathway engineering for functional isoprenoids.</A></LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:TU_Munich/Notebook/Protocols">http://2012.igem.org/Team:TU_Munich/Notebook/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:TU_Munich/Notebook/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:TU_Munich/Notebook/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:TU_Munich/Notebook/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:TU_Munich/Notebook/Protocols&amp;oldid=283916" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>