"
Page
Discussion
View source
History
teams
Log in
 
Team:TU Munich/Project/Caffeine
From 2012.igem.org
Project
Results
Safety
Vector Design
Limonene
Thaumatin
Caffeine
Xanthohumol
Constitutive Promoter
Light-Switchable Promoter
Ethanol-Inducible Promoter
Genome Integration
Brewing
BioBricks
Data Page
RFC
Modeling
Methods
Priors
Gal1 Promoter
Yeast Growth
Human Practice
Overview
School Visit
Public Relations
Comic
Action Day
Politics
Movie
Radio Interview
Survey
Notebook
Methods
Labjournal
Meetings
Attributions
Team
Members
Activities
Collaborations
Contents
1Caffeine
1.1Background and Principles
1.1.1Biosynthesis and Metabolic Engineering
1.1.2The Molecular and Physiological Effects of Caffeine
1.2Results
1.2.1BioBricks
1.2.1.1BBa_K801070 RFC10 compatible BioBrick encoding the enzyme CaXMT1
1.2.1.2BBa_K801071 RFC10 compatible BioBrick encoding the enzyme CaMXMT1
1.2.1.3BBa_K801072 RFC10 compatible BioBrick encoding the enzyme CaDXMT1
1.2.1.4BBa_K801073 Generator BioBrick for CaXMT1
1.2.1.5BBa_K801074 Generator BioBrick for CaMXMT1
1.2.1.6BBa_K801075 Generator BioBrick for CaDXMT1
1.2.1.7BBa_K801076 Generator BioBrick for CaXMT1 and CaMXMT1
1.2.1.8BBa_K801077 Generator BioBrick for CaXMT1, CaMXMT1 and CaDXMT1
1.2.2Characterization
1.2.2.1Western Blot Detection of Expressed Enzymes CaXMT1, CaMXMT1 and CaDXMT1 (BBa_K801070, BBa_K801071 and BBa_K801072)
1.2.2.2Composition of Expression Cassettes
1.2.2.3Caffeine Synthesis Pathway Composite Part BBa_K801077
1.2.2.4Toxicity Assay
1.3References
Caffeine
Responsible: Saskia König, Roman Prechtl and Dennis Hell
Hops is not only one of the main ingredients of beer, it is also responsible for the sedative effect of beer [ Franco et al. 2012 ].
In contrast, caffeine wards off drowsiness and already enjoys great popularity as additive in a multitude of beverages. As hops is essential to the brewing process, omitting hops is not an option. This makes caffeine a desirable agent to counteract the soporific effect of beer.
We were successful in expressing (SDS-PAGE and western blot analysis) all three genes which are necessary for caffeine biosynthesis (in plants) in yeast strain INVSc1 after having cloned the genes in the new yeast expression vector pTUM104. The proof of enzyme functionality by detection of synthesized caffeine via LC-MS is already in progress. The three single genes were submitted as BioBricks BBa_K801870 , BBa_K801071 and BBa_K801072 .
Furthermore, we generated the caffeine synthesis BioBrick BBa_K801077 , which contains all the genes necessary for caffeine synthesis - each gene with a promoter and terminator. The strength of the promoters has been chosen individually.
In order to investigate the growth of our yeast cells under the influence of caffeine at different concentrations, we also performed a toxicity assay.
Caffeine production in tobacco plants [ Uefuji et al., 2005 , Kim and Sano, 2008 ] and in an in vitro assay has already been shown, but has never been performed in yeast.
Although the detection of caffeine in our performed enzyme assay turned out to be difficult, we were able to show by LC/MS analysis, that theobromine, the immediate precursor of caffeine, can in fact be synthesized out of provided Xanthosine by a crude extract derived of Saccharomyces cerevisiae cells having previously been transformed with our "caffeine synthesis Biobrick" BBa_K801077 .
Background and Principles
Biosynthesis and Metabolic Engineering
The biosynthetic pathway of caffeine (1,3,7 Trimethylxanthine) starts with xanthosine, which is a natural component of the purine metabolism of all organisms. Necessary for its production are three distinct N-methyl transferases and one nucleosidase.  However, it remains to be elucidated whether the nucleosidase reaction is catalyzed by an unspecific purine nucleoside phosphorylase (PNP) or by the first N-methyl transferase of the reaction cascade shown in the picture (Fig. 1). Being involved in the catabolism of all purine nucleosides, PNP is an essential enzyme for organisms and was shown to catalyze the removal of the ribose moiety of 7-methylxanthine in in vitro caffeine biosynthesis [ Uefuji et al., 2003 ]. The in vivo synthesis of caffeine has also been shown by expression of the three N-methyl transferases in tobacco plants. Therefore, the assumption that the first methyl transferase is bifunctional and catalyzes the nucleosidase reaction is favored [ Ashihara et al., 2008 ], [ McCarthy and McCarthy, 2007 ]. The presence of PNP or equal enzymes in plants has not yet been proven and a catalytic mechanism of the nucleosidase reaction is rather thinkable and has been explained by [ McCarthy and McCarthy, 2007 ].  As a matter of fact, a single N-methyl nucleosidase, as it is shown on the depicted pathway below, has indeed been partially purified out of tea leaves [ Negishi et al. (1988) ], but neither the native enzyme nor its DNA have ever been isolated [ Ashihara et al., 2008 ].
The enzyme caffeine synthase (last reaction step) can catalyze both the conversion of 7-methyl xanthine to theobromine and the methylation of theobromine to caffeine. Unfortunately, the affinity of this enzyme to 7-methyl xanthosine is less than one sixth of that of the other isoform CaMXMT1 [ Uefuji et al., 2003 ]. To obtain the best results, we decided to express both enzymes. One can also see the Km values for the required enzymes in this paper - it shows that the substrate affinity decreases continuously towards the endpoint (caffeine), "making the reaction proceed irreversibly and stepwise" [ Uefuji et al., 2003 ].
Fig. 1: Xanthosine providing metabolic pathways
The chemical compound xanthosine is produced via at least four different routes, shown in the picture "xanthosine routes". To improve caffeine production, these pathways could be a possible target for metabolic engineering in the future.
Fig. 2: Pathway for in vivo caffeine biosynthesis in coffea arabica
During the degradation of caffeine, it is demethylated to theophylline by 7N-demethylase (main pathway). The decreased rate of this reaction is the reason for the accumulation of caffeine in the plant. Afterwards, theophylline is degraded to xanthine via 3-methylxanthine and xanthine enters the conventional purine catabolism pathway (degradation to CO2 and NH3) [ Ashihara et al., 2008 ]. This catabolistic pathway is another possible target for metabolic engineering to increase the amount of caffeine.
The Molecular and Physiological Effects of Caffeine
Fig. 3: Similar structures of caffeine and adenosine
Caffeine is a purine alkaloid and its biosynthesis occurs in coffee plants and tea plants. Its chemical structure is similar to that of the ribonucleoside adenosine. Hence, it can block specific receptors in the hypothalamus. Adenosine binding leads to decreased neurotransmitter release and therefore decreased neuron activity. This induces sleep and thus avoids overexertion of the brain. Since caffeine antagonizes adenosine and increases neuronal activity, it is used as a means to stay awake. On average, one cup (150 ml) of coffee contains about 50 - 130 mg caffeine and one cup of tea about 25 - 90 mg. At higher doses (1 g), caffeine leads to higher pulse rates and hyperactivity, but until that, the alcohol will already have done its work...
Moreover, caffeine was shown to decrease the growth of E. Coli and yeast reversibly as of a concentration of 0.1 % by acting as a mutagen (Putrament et al., On the Specificity of Caffeine Effects, MGG, 1972), but previous caffeine synthesis experiments (see below) have only led to a concentration of about 5 µg/g (per g fresh weight of tobacco leaves). Therefore, we do not expect to reach critical concentrations and the amounts of caffeine in coffee or tea (leading to physiological effects) is usually a little bit lower.
Results
BioBricks
Fig. 4: Biobricks created for caffeine synthesis
All the generated BioBricks are based on mRNA sequences that have been isolated from Coffea arabica by Hiroshi Sano et al., 2003, and registered at NCBI (see numbers below). However, these sequences were modified in several ways, to make them iGEM compatible and improve the usage in general.
Modifications:
the 5' UTR and 3' UTR of the original sequences were removed
the yeast consensus sequence for improved ribosome binding (TACACA) was added 5' of the start codon ATG
according to the N- end rule and the yeast consensus sequence for improved ribosome binding, the first triplet after ATG (GAG) was exchanged with TCT (serine) to optimize protein stability and mRNA translation. This decision was made after analysis of the 3D- structure of the enzyme CaDXMT1. Because the first two residues of the amino acid sequence are not shown in the crystallized structure (probably because of high flexibility) we chose to exchange this amino acid. Further study (uniprot entry) showed that the first two residues of the sequence are not immediately involved in ligand binding in one of the three enzymes. Due to high similarity of the sequences of the three enzymes, we also changed this amino acid in the enzymes CaXMT1 and CaMXMT1.
we added a C- terminal Strep-tag for purification and detection
the remaining coding sequence was extended with the standard RFC10 prefix and suffix
we made an optimization of the coding sequences with respect to the codon usage in yeast and mRNA structures (online tool of gene-synthesis company)
we exchanged all critical restriction sites (RFC10 and RFC25)
All mentioned methyltransferases use SAM as methyl donor and are located in the cytoplasm of the plants. Furthermore, they exist as homodimers, being also able to form heterodimers with each other (see Brenda , also for further characteristics). The temperature- and pH-optima of all three enzymes are quite similar (between 20°C - 37°C and 7,5 - 8,5). The pH-optimum is compatible with the slightly acidic environment during beer brewing.
BBa_K801070 RFC10 compatible BioBrick encoding the enzyme CaXMT1
NCBI access number of original gene sequence: AB048793
This RFC10 compatible BioBrick encodes the enzyme CaXMT1 (xanthosine N-methyltransferase 1 of coffea arabica). It catalyzes the first reaction step of the caffeine biosynthesis pathway.
Further information:
UniProt entry: Q9AVK0
E.C. Number: 2.1.1.158
PDB: 3D- Structure: C. canephora CaXMT1
Part length: 1158 bp
BBa_K801071 RFC10 compatible BioBrick encoding the enzyme CaMXMT1
NCBI access number of original gene sequence: AB048794
This RFC10 compatible BioBrick encodes the enzyme CaMXMT1 (7-methylxanthine N-methyltransferase of coffea arabica). It catalyzes the third reaction step of caffeine biosynthesis pathway.
Further information:
UniProt entry: Q9AVJ9
E.C. Number: 2.1.1.159
Part length: 1176 bp
BBa_K801072 RFC10 compatible BioBrick encoding the enzyme CaDXMT1
NCBI access number of original gene sequence: AB084125
This RFC10 compatible BioBrick encodes the enzyme CaDXMT1 (3,7-dimethylxanthine N-methyltransferase of coffea arabica). It catalyzes the fourth reaction step of the caffeine biosynthesis, leading to caffeine.
Further information:
UniProt entry: Q8H0D2
2.1.1.160
PDB: 3D- Structure: C. canephora CaDXMT1
Part length: 1194 bp
BBa_K801073 Generator BioBrick for CaXMT1
This BioBrick generates the enzyme xanthosine N-methyltransferase 1 (CaXMT1). It is regulated by the constitutive promoter Tef2, which is a strong yeast promoter. The used terminator is Adh1, a widely used yeast terminator.
This BioBrick is also a part of the "caffeine synthesis device" (see below) and the accuracy of the sequence has been proven by sequencing.
BBa_K801074 Generator BioBrick for CaMXMT1
This BioBrick generates the enzyme 7-methylxanthine N-methyltransferase 1(CaMXMT1) (= theobromine synthase). It is regulated by the constitutive promoter Tef1, which is one of the strongest yeast promoters. The used terminator is Adh1, as it is among the other expression cassettes.
This BioBrick is also a part of the "caffeine synthesis device" (see below) and the accuracy of the sequence has been proven by sequencing.
BBa_K801075 Generator BioBrick for CaDXMT1
This BioBrick generates the enzyme 3,7-dimethylxanthine N-methyltransferase 1 (CaDXMT1) (= caffeine synthase), i.e. the last enzyme involved in caffeine biosynthesis. It is regulated by the constitutive promoter Tef2, which is also used at the expression cassette of CaXMT1. The used terminator is Adh1, as it is among the other expression cassettes.
This BioBrick is also a part of the "caffeine synthesis device" (see below) and the accuracy of the sequence has been proven by sequencing.
BBa_K801076 Generator BioBrick for CaXMT1 and CaMXMT1
This BioBrick generates the first two enzymes envolved in caffeine biosynthesis: methylxanthosine N-methyltransferase 1 and 7- methylxanthine N-methyltransferase 1. It is made up of the two single generators (see above), which means that CaXMT1 is regulated by the Tef2 promoter and CaMXMT1 is regulated by the Tef1 promoter. In both cases, the Adh1 terminator was used.
BBa_K801077 Generator BioBrick for CaXMT1, CaMXMT1 and CaDXMT1
This BioBrick is the final "caffeine synthesis device". It contains all three necessary enzymes: CaXMT1, CaMXMT1 and CaDXMT1, i.e. it is made of the three single expression cassettes for each enzyme (see also BBa_K801073, BBa_K801074 and BBa_K801075). It can be transformed directly into competent yeast cells or cloned into an adequate yeast genome integration vector.
The accuracy of this BioBrick has not yet been proven by sequencing.
Characterization
Western Blot Detection of Expressed Enzymes CaXMT1, CaMXMT1 and CaDXMT1 (BBa_K801070, BBa_K801071 and BBa_K801072)
We were successful in expressing all three enzymes which are necessary for the caffeine biosynthesis pathway in yeast by using yeast expression vector pTUM104. The results can be seen on the right picture. Detection was performed by using a specific antibody against Strep-tag II. The Strep-tag coding sequence has been fused to each of the enzyme coding sequences.
Fig. 5: Western Blot of protein crude extracts: Left: (1) CaXMT1 (uninduced);(2) CaMXMT1 (uninduced);(3) eGFP (20h);(4) CaMXMT1 (20h); Right: (1) CaDXMT1 (uninduced); (2) CaDXMT1 (20h)
The theoretical weights of the enzymes (predicted with the online tool "ProtParam") are as follows:
CaXMT1: 42998,4 Da
CaMXMT1: 43903,3 Da
CaDXMT1: 44473,7 Da
The real weights of the expressed enzymes are likely to be influenced by several posttranslational modifications after expression in yeast, and that is probably the reason why the detected proteins showed a higher apparent mass on the western blot. Amongst others, ExPASy Proteomics tools predicts the following modifications:
acetylation at serine (second amino acid)
O-GlcNAc modifikation at several positions
phosphorylation at several positions
Composition of Expression Cassettes
In order to be able to perform a continuous expression of the three enzymes, we created several expression cassettes with individual promoters and the widely used Adh1 terminator. The BioBricks BBa_K801073, BBa_K801074 and BBa_K801075 are enzyme generators, which provide continuous expression of the particular enzymes and are all part of the final caffeine synthesis pathway composite part. The choice for the respective promoters was made to support the irreversible production of caffeine, as it is in vivo in coffee plants, being assured by continuously increasing Km values of the three enzymes (H. Sano et al., 2003). Besides, we focused on prohibiting metabolic stress reactions. Thus, we used the Tef2 promoter for the first enzyme CaXMT1 and the stronger Tef1 promoter for the next enzyme, which is CaMXMT1, to establish high concentrations of the caffeine precursor theobromine. As soon as a certain amount of theobromine is available, the caffeine synthesis can go on. The last enzyme CaDXMT1 is then again regulated by the Tef2 promoter.
The sequences of all three expression cassettes have been checked by sequencing, but expression has not yet been investigated.
Caffeine Synthesis Pathway Composite Part BBa_K801077
Fig. 6: Gel electrophoresis of BBa_K801077 digested with Xba1 and Pst1
This part is our final product, made up of all three expression cassettes mentioned above. As previously said, it provides constitutive expression of all three neccessary enzymes for caffein biosynthesis. It can be cloned in a yeast expression vector or be used for genome integration by the use of appropriate vectors. An analytical restriction digest can be seen on the left.
We have been successful in the expression of the three single enzymes making up the caffeine biosynthesis metabolic pathway. Therefore, we used this composite part BBa_K801077 for expression of the enzymes, followed by enzymatic analyses, in order to prove the functionality of our enzymes. After having cloned it into our vector pTUM100, we performed an enzyme assay as described in [ Uefuji et al., 2003 ], with the enzymes having been harvested after 20 hours. The reaction conditions were 27 °C at a pH of 8.0.
Detection of single metabolites of the caffeine biosynthesis pathway was performed by LC/MS analysis, using multiple reaction monitoring (MRM). The picture below shows that we were in fact able to perform an in vitro synthesis of theobromine, which is the immediate precursor of caffeine. However, the detection of caffeine was quite difficult, because the negative control also showed the corresponding signal with an identical retention time and it is hard to say if caffeine has really been produced by our expression cassette. We assumed, that a possible reason for the failed detection could be the fact, that the concentration of available theobromine in the reaction mixture was too less. Thus, we started another experiment, in which we added theobromine directly to the isolated crude extract. However, it turned out that the used theobromine was too impure and already contained caffeine, which made it hard to draw any reliable conclusions.
Fig. 7: LC/MS chromatograms of the first enzyme assay. Used substrate: Xanthosine; A: Reference chromatogram of theobromine; B: Chromatogram of a solution obtained from the enzyme assay: reaction batch was prepared as described below, with the crude extract having been isolated of yeast cells transformed with the caffeine synthesis pathway composite part; C: Chromatogram of a solution obtained from the enzyme assay: reaction batch was prepared as described below, whereas the crude extract had been isolated of yeast cells transformed with the empty vector pTUM100; Reacton conditions: T= 27°C, pH= 8.0; Reaction time: 16h; Reaction batch: 100µl reaction batch consisted of 50mM Tris/HCl (pH 8.0), 500 µM Xanthosine, 1,5 mM SAM, 200 µM MgCl2, 500 µg protein (crude extract); Reaction products were extracted with 1 ml chloroform, dried at 60°C and subjected to LC/MS analysis after having been resolved in 100µl 70% methanol; For all results, check our online labjournal on October, 24th.
Toxicity Assay
Since high doses of caffeine (> 10 mM) have mutagenic effects on yeast ( Kuranda et al., 2006 ) we investigated the effect of different caffeine concentrations on different yeast strains.  The used yeast strains were the laboratory strain INVSc1, a strain which is used for brewing beer and a strain which can be purchased in a supermarket. Caffeine was added to the YPD medium in concentrations from 1 µM up to 100 mM and the growth rate was measured after a defined period of time.
We confirmed the toxic effect of caffeine on yeast at the concentration of 100 mM and the growth inhibition at the concentration of 10 mM.  Furthermore we showed the correlation between decreasing caffeine concentration and growth rate. Cells incubated with caffeine concentrations in the range of micro molar showed a similar growth rate than the negative control (incubation without caffeine).
During our brewing experiments, we won’t exceed the level of toxicity (>10 mM) of caffeine.
Fig. 7: Evaluation of the Toxicity Assay for Caffeine
References
[ Ashihara et al., 2008 ] Ashihara, H., Sano, H., and Crozier, A. (2008). Caffeine and related purine alkaloids: biosynthesis, catabolism, function and genetic engineering. Phytochemistry, 69(4):841–56.
[ Franco et al., 2012 ] Franco, L., Sánchez, C., Bravo, R., Rodriguez, A., Barriga, C., and Juánez, J. C. (2012). The sedative effects of hops (humulus lupulus), a component of beer, on the activity/rest rhythm. Acta Physiol Hung, 99(2):133–9.
[ Kim and Sano, 2008 ] Kim, Y.-S. and Sano, H. (2008). Pathogen resistance of transgenic tobacco plants producing caffeine. Phytochemistry, 69(4):882–8.
[ Kuranda et al., 2006 ] Kuranda, K., Leberre, V., Sokol, S., Palamarczyk, G., and François, J. (2006). Investigating the caffeine effects in the yeast Saccharomyces cerevisiae brings new insights into the connection between TOR, PKC and Ras/cAMP signalling pathways. Mol Microbiol, 61(5):1147–66.
[ McCarthy and McCarthy, 2007 ] McCarthy, A.A., McCarthy, J.G. (2007). The Structure of Two N-Methyltransferases from the Caffeine Biosynthetic Pathway. Plant Physiology, 144(2):879-889.
[ Negishi et al. (1988) ] Negishi O, Ozawa T and Imagawa H (1988). N-Methyl nucleosidase from tea leaves. Agric. Biol. Chem. 52: 169–175.
[ Uefuji et al., 2003 ] Uefuji, H., Ogita, S., Yamaguchi, Y., Koizumi, N., and Sano, H. (2003). Molecular cloning and functional characterization of three distinct n-methyltransferases involved in the caffeine biosynthetic pathway in coffee plants. Plant Physiol, 132(1):372–80.
[ Uefuji et al., 2005 ] Uefuji, H., Tatsumi, Y., Morimoto, M., Kaothien-Nakayama, P., Ogita, S., and Sano, H. (2005). Caffeine production in tobacco plants by simultaneous expression of three coffee n-methyltrasferases and its potential as a pest repellant. Plant Mol Biol, 59(2):221–7.
Retrieved from " http://2012.igem.org/Team:TU_Munich/Project/Caffeine "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
