"
Page
Discussion
View source
History
teams
Log in
 
Team:HKU HongKong/Data/Mol Protocols.html
From 2012.igem.org
Team:HKU Hong Kong - 2012
"
Team:HKU HK
From 2011.igem.org
 
  Home  
  Team  
About Us
Profiles
  Project  
Background
Future Implications
  Data  
Weekly Notebook
Bio Bricks
Results
  Protocols  
 Molecular Cloning 
 pvdQ Expression Analysis 
  Safety  
  Human Practice  
 
 
Molecular Cloning Protocols
Contents
1.1 Transforming Competent E.coli with the Plasmid
1.2 Miniprep to Extract the Plasmid from E.coli
1.3 Midiprep for Large Volumes of Culture
1.4 PCR Amplification of Gene from Bacterial Genome
1.5 PCR Cleanup Qiagen QIAquick PCR Purification
    1.5.1 Colony PCR
    1.5.2 Agarose Gel Electrophoresis
1.6 Gel Purification of DNA (Qiagen QIAquick Gel Extraction Kit)
1.7 Determining DNA Concentration using NanoDrop Spectrophotometry
1.8 DNA digestion
1.9 Dephosphorylation of 5' Ends of Vector Backbone
2.0 Vector-Insert Litigation
2.1 PCR   Deletion Reaction
 
Transforming Competent E.coli with the            Plasmid:
We chose the biobrick            K137076 to ligate to the pvdQ gene for the following reasons:
Take out the competent E-coli              cells from the -80 freezer. (Keep all tubes on ice).
Add 1uL              of the plasmid DNA in 100uL competent cells (if from Kit). For              transformation of ligated product, add 10uL of the ligated plasmid DNA in              100uL competent cells.
Incubate on ice for 10 min.
Place              in water bath at 42°C for 90s.
Place              immediately back on ice for at least 2 min.
Add              800uL LB broth. Incubate for 1hr              at 37°C shaker.
Centrifuge at 130rpm for 5min              to remove supernatant. Re-suspend the pellet in about 10uL of supernatant.
Spread the entire re-suspended              pellet on ampicillin agar dishes.
Incubate for 12-16 hours at 37 °C.
Notes:
- Colonies grown on plate were selected for colony PCR screening.
-    Correct colonies were then cultured in 5 mL LB Broth with 0.5uL    ampicillin for subsequent screening or mass culturing.
-          Mass culturing involved adding 200uL of the culture into 35mL LB Broth         with 35uL ampicillin.
Miniprep to Extract the Plasmid from E.coli (QIAprep Spin Miniprep Kit):
Transfer some of the 5mL bacterial culture into a microcentrifuge           tube. Pellet by centrifugation at 13,500 rpm for 1 min. Repeat till           all the culture has been pelleted.
Resuspend the pellet in 250uL Buffer P1.
Add 250uL Buffer P2 (Lysis Buffer). Mix thoroughly by inverting the           tube 4-6 times. Do not allow prolonged lysis.
Immediately add 350uL Buffer N3 (Neutralization Buffer). Mix           immediately by inverting the tube.
Centrifuge for 10 minutes at 13,500 rpm.
Apply the resulting supernatant to the QIAprep spin column by           pipetting. Centrifuge for 1 minute at 13,500 rpm.
Wash the QIAprep spin column by applying 0.75mL Buffer PE.           Centrifuge for 1 minute at 13,500 rpm.
Centrifuge for an additional 1 minute to remove residual ethanol.
Place the QIAprep spin column in microcentrifuge tube. Elute DNA by           adding 50uL warm H2O.
Centrifuge for 1 minute at 13,500 rpm.
Midiprep for Large Volumes of Culture (QIAGEN Plasmid Midi Kit):
Refer to the QIAGEN website for the protocol.
PCR Amplification of Gene from Bacterial Genome/Standard Biobrick Parts
37.5μL   ddH2O
5.0μL     10x PCR Buffer
2.5μL     dNTPs
1.0μL     Forward Primer (Prefix)
1.0μL     Reverse Primer (Suffix)
1.0μL     Template DNA
1.0        μL    RTaq DNA Polymerase
50.0μL Total
Note:
- For sequential PCR reactions: After the first PCR is complete, perform    PCR clean-up. Then use the product to conduct the second PCR reaction,    after which gel purification needs to be carried out.
- The            volume of DNA used in the second PCR reaction depends on the            concentration of the DNA after the PCR Clean-up.
PCR Clean-Up –    Qiagen QIAquick PCR Purification:
Add 5 volumes of Buffer PB to 1 volume of PCR mix.
Place this mix within the QIAquick column. Centrifuge at 13,500 rpm           for 1 minute.
Discard flow through and centrifuge again to allow all the sample to           pass through.
Wash with 750uL Buffer PE. Centrifuge at 13,500 rpm for 1 minute.
Discard flow through and centrifuge again to remove all residual           buffer.
Place the column in a micro centrifuge tube.
Apply 50uL of pre-warmed distilled H2O to the column. Stand for at           least 2 minutes.
Centrifuge at 13,500 rpm for 1 minute.
Colony PCR
14.52μL ddH2O
2.0μL     10x PCR Buffer
1.6μL     dNTPs
0.4μL     Forward Primer (Prefix)
0.4μL     Reverse Primer (Suffix)
1.0μL     Template DNA
0.08μL   RTaq DNA Polymerase
20.0μL Total
 Notes:
- The            mixture was pipetted into PCR tubes
- All    materials were kept on ice
- Colonies will inoculated into 5uL broth prior to PCR
- Prefix and Suffix were used as Forward and Reverse Primers respectively            while amplifying standard biobrick parts
- PCR         Reaction: 95°C - 10 min
                           95°C - 30 sec
                           57°C - 30 sec (appropriate annealing temperature for prefix and    suffix)
                           72°C - 30 sec
                          (28 cycles all together)
                           72°C - 5 min
Agarose Gel Electrophoresis
Prepare a 2% or 1% Agarose Gel (amount in grams depending on volume           of TAE buffer used). Add 0.1% Ethidium Bromide of the total volume.
Place the gel in the Electrophoresis Apparatus with the wells facing           the Negative Electrode.
Fill the apparatus with 1% TAE Buffer to fully submerge the wells.
Load 5µL of 1kb Ladder for each Run
Add 0.1% of 10% Loading Dye to the respective volume of sample. Mix           well and spin down.
Pipette the samples into the wells and run at 106 Volts.
Gel            Purification of DNA (Qiagen QIAquick Gel Extraction Kit)
Exercise the DNA fragment from the Agarose Gel using a scalpel.           Minimize the extra peripheral gel slice.
Weigh the gel slice (0.1g = 100uL) and add 3 Volumes of Buffer QG to every 1           Volume of Gel.
Incubate in 50°C water bath for 10 minutes to completely dissolve           the gel slice.
Add 1 Volume of           Isopropanol to the sample. Mix well
Apply the sample            to the QIAquick column. Centrifuge at 13,500 rpm for 1 minute.            [Repeat till the total volume of the sample has sieved through the           column]. Discard the flow through
Apply 0.5mL Buffer QG to the QIAquick column. Centrifuge at 13,500 rpm for              1 minute.
Discard the flow           through
Wash the column           with 0.75mL Buffer PE. Centrifuge at 13,500 rpm for 1 minute.
Discard the flow            through and Centrifuge at 13,500 rpm for an additional 1 minute to           eliminate any residual ethanol.
Place the QIAquick column into a 1.5mL Eppendorf tube.
Apply the 30uL warm H2O to the column. Let the column stand for at              least 1 minute.
Centrifuge at            13,500 rpm for 1 minute.
 
Determining DNA Concentration Using NanoDrop Spectrophotometry
Choose the Nucleic Acid Measurement option in the programme.
Initialize the NanoDrop by adding 1uL clean H2O. Clean           the sensor gently with tissue.
Set Blank by adding an additional 1uL of clean H2O. Wipe           off.
Add 1uL of the DNA sample to be measured. Wipe off after each run.
DNA Digestion
__μL   ddH2O (to a total of 40uL)
4μL     10X NEBuffer
0.4μL  100X BSA
1μg     DNA Sample
__μL   1st Restriction Enzyme
__μL   2nd Restriction Enzyme (optional)
40μL Total
Notes:
-        The NEB official website should be checked for buffers suitable for each        restriction enzyme. Results can vary depending on double or single         digestion.
- Incubate the digestion sample at 37°C for 3 hours (digestion time can            also vary depending on enzyme). Prolonged digestion may lead to Star            Activity otherwise.
- The            volume of DNA must be calculated from its concentration. In restriction            digestion test, the minimum volume that is equals 1ug DNA can be            utilized. However, for purification, a much greater volume of DNA should            be used.   
- Appropriate amount of enzyme is derived from its concentration and the            fact that 5 units of enzyme digest 1ug DNA.
 Dephosphorylation of 5' Ends of Vector Backbone
- Add          0.5µL (0.5 units per 1 ug DNA) of Calf Intestinal Alkaline Phosphatase           (CIAP) to the digested sample immediately after digestion.
- Incubate at 37°C for 30 minutes 
Vector-Insert Ligation
__μL  Autoclaved ddH2O (to a total of 20uL)
2μL    T4 Ligase Buffer
1μL    T4 DNA Ligase
__μL  Vector DNA
__μL   Insert DNA
20μL  Total
 Notes:
- Insert and Vector must be in a 3:1 ratio. The amount of each depends on            their concentration (ng/uL) and legnth (bp).
- Incubate    at room temperature for 1 hour. 
PCR Deletion (Site-Directed Mutagenesis) Reaction
Note:            Keep everything on ice and add all volumes in a PCR tube.
?    µL ddH2O (? = whatever volume needed to bring the total volume up to    50µL)
5.0μL 10x PfuUltra buffer
1.0μL dNTPs
?         uL forward primer = 125ng fwd primer ÷ fwd primer concentration (ng/µL)
?         uL reverse primer = 125ng rvs primer ÷ rvs primer concentration (ng/µL)
?         µL dsDNA= 20ng insert ÷ insert concentration (ng/µL)
1.0μL PfuUltra high-fidelity DNA polymerase
50.0μL Total
 
- Volumes of diluted primer based on calculations for our ng/µL            concentrations
95°C for 2min
95°C for 30sec (18 times)
55°C for 30sec
72°C for 1 min/kb
1min/kb corresponds to: 3.20min (RFP), 3.50min (VioA), 5.40min (VioB),         3.00min (VioE)
 
 
 
 
Our kind Sponsors:
   
  
Retrieved from " http://2012.igem.org/Team:HKU_HongKong/Data/Mol_Protocols.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
