<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_HKU_HongKong_Project_Future_Implications_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:HKU_HongKong/Project/Future_Implications.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:HKU_HongKong/Project/Future_Implications.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:HKU_HongKong/Project/Future_Implications.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:HKU_HongKong/Project/Future_Implications.html&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:HKU_HongKong/Project/Future_Implications.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HKU HongKong/Project/Future Implications.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Team:HKU Hong Kong - 2012</TITLE></P><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:HKU HK</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="main"><DIV id="header"><DIV id="buttons"><P> </P><UL id="css3menu2" class="topmenu"><LI class="topfirst"><A href="https://2012.igem.org/Team:HKU_HongKong" style="height:26px;line-height:26px;">  Home  </A></LI><LI class="topmenu"><A href="#" style="height:26px;line-height:26px;">  Team  </A></LI><LI><FONT size="10"><A href="https://2012.igem.org/Team:HKU_HongKong/Team/About_Us.html">About Us</A></FONT></LI><LI><FONT size="10"><A href="https://2012.igem.org/Team:HKU_HongKong/Team/Profiles.html">Profiles</A></FONT></LI></UL><LI class="topmenu"><A href="#" style="height:26px;line-height:26px;">  Project  </A></LI><LI><FONT size="10"><A href="https://2012.igem.org/Team:HKU_HongKong/Project/Background.html">Background</A></FONT></LI><LI><FONT size="10"><A href="https://2012.igem.org/Team:HKU_HongKong/Project/Future_Implications.html">Future Implications</A></FONT></LI><LI class="topmenu"><A href="#" style="height:26px;line-height:26px;">  Data  </A></LI><LI><FONT size="10"><A href="https://2012.igem.org/Team:HKU_HongKong/Data/Weekly_Notebook.html">Weekly Notebook</A></FONT></LI><LI><FONT size="10"><A href="https://2012.igem.org/Team:HKU_HongKong/Data/Bio_Bricks.html">Bio Bricks</A></FONT></LI><LI><FONT size="10"><A href="https://2012.igem.org/Team:HKU_HongKong/Data/Results.html">Results</A></FONT></LI><LI class="topmenu"><A href="#" style="height:26px;line-height:26px;">  Protocols  </A></LI><LI><FONT size="10"><A href="https://2012.igem.org/Team:HKU_HongKong/Data/Mol_Protocols.html"> Molecular Cloning </A></FONT></LI><LI><FONT size="10"><A href="https://2012.igem.org/Team:HKU_HongKong/Data/pvdQ_Protocols.html"> pvdQ Expression Analysis </A></FONT></LI><LI class="topmenu"><A href="https://2012.igem.org/Team:HKU_HongKong/Safety.html" style="height:26px;line-height:26px;">  Safety  </A></LI><LI class="toplast"><A href="https://2012.igem.org/Team:HKU_HongKong/Human_Practice.html" style="height:26px;line-height:26px;">  Human Practice  </A></LI><P> </P><H2 style="font-variant: normal; vertical-align: baseline; clear: left; color: #232323; font-family: Gentium Basic; letter-spacing: normal; line-height: 19.5px; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; border: 0px none; margin: 0.7em 0px; padding: 0px"><SPAN style="font-weight: 400; text-decoration:underline"><FONT face="Trebuchet MS" size="6">Future implications</FONT></SPAN></H2><P style="text-align: left; font-style: normal; font-variant: normal; font-weight: normal; font-size: 13px; vertical-align: baseline; color: rgb(85, 85, 85); font-family: Lato, Tahoma, Arial, sans-serif; letter-spacing: normal; line-height: 19.5px; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; border: 0px none; margin-left: 0px; margin-right: 0px; margin-top: 0px; margin-bottom: 20px; padding: 0px"> </P><P align="left"><STRONG>Future  Biobrick 1</STRONG></P><P align="left" style="text-align: left; font-style: normal; font-variant: normal; font-weight: normal; font-size: 13px; vertical-align: baseline; color: #232323; font-family: Lato, Tahoma, Arial, sans-serif; letter-spacing: normal; line-height: 19.5px; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; border: 0px none; margin-left: 0px; margin-right: 0px; margin-top: 0px; margin-bottom: 20px; padding: 0px"><STRONG><U>Construction  of J23119-B0034-pvdQ-B0015: </U></STRONG></P><P align="left"> </P><P align="left">This biobrick was constructed to test the expression of the pvdQ  gene amplified by PCR from genomic DNA of <EM>Pseudomonas  aeruginosa</EM>. It is an uncontrollable, standard biobrick that can later be  used as a baseline reference to test whether our future biobricks result in an  improvement in pvdQ expression level. </P><P align="left">The biobrick is a composite part consisting of the already  prevalent Constitutive Promoter [2006 Berkley], along with the newly inserted  pvdQ gene. It is also composed of a strong RBS and a transcription double  terminator. </P><DIV align="left"><UL><LI>Constitutive Promoter (J23119): This promoter biobrick is the  consensus sequence and therefore results in the idealized transcription  scenario. Since it is a strong promoter and is constitutively expressed, it  complies with our preliminary goal of testing transcription and translation of  the pvdQ gene in an AHL-independent manner. </LI></UL></DIV><P align="left">The  constitutive promoter directs expression in virtually all tissues. Also, it is  independent of environmental and developmental (endogenous) factors. This is  why it can be utilized across species. The promoter will ensure that pvdQ will  be expressed ubiquitously, and pvdQ expression will increase with increase in  cell density of <EM>E.coli. </EM>As the <EM>E.coli </EM>growth curve is exponential, so  will be the curve of pvdQ expression. 
  We tried to construct this biobrick during our wet lab sessions. However,  several problems were encountered, resulting in its unsuccessful creation: </P><DIV align="left"><UL><LI>The J23119 promoter could be  ligated to B0034 RBS by digesting with EcoRI and PvuI. This digestion was used  instead of the conventional EcoRI + SpeI and EcoRI + XbaI of the insert and  vector respectively because the fragment size of the EcoRI+ SpeI digested  insert is only a little more than a 100 bp. This produces a band of very low  concentration as most of the DNA makes up the unwanted backbone, making the  insert difficult to gel purify.   Digestion with EcoRI and PvuI, conversely, yields a larger fragment that  can be subsequently ligated. </LI><LI>Ligating the J23119-B0034 to  the pvdQ-B0015 was a problem, however. EcoRI + PvuI digestion could no longer  be used because the backbone of the two biobricks are different. </LI><LI>Forward primers were designed  to carry out a two-step PCR. The first reaction involved adding the RBS to the  pvdQ-B0015 biobrick. The second reaction would add the Prefix and J23119  promoter to the RBS-pvdQ-B0015 biobrick from the first reaction. The standard  suffix was used as the reverse primer. The second PCR, however, was  unsuccessful perhaps due to the large difference in Tm between the forward and  reverse primers. </LI></UL></DIV><P align="left"> </P><P align="left"> </P><P align="left"><STRONG>Future  Biobricks 2 &amp; 3</STRONG></P><P align="left" style="text-align: left; font-style: normal; font-variant: normal; font-weight: normal; font-size: 13px; vertical-align: baseline; color: #232323; font-family: Lato, Tahoma, Arial, sans-serif; letter-spacing: normal; line-height: 19.5px; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; border: 0px none; margin-left: 0px; margin-right: 0px; margin-top: 0px; margin-bottom: 20px; padding: 0px"> </P><P align="left">This biobrick will allow constitutive expression of pvdQ at the  baseline level. However, pvdQ expression can be enhanced if it is also  dependent on AHL-luxR binding to the pLuxR. This system allows a baseline pvdQ  expression even in the absence of AHL. Hence, it may confer an advantage over  the BBa_K855001 biobrick we constructed by keeping the AHL from accumulating  and further minimizing the oscillatory effect. </P><P align="left">Another system would be one that is dependent on another factor of  cell density but independent of AHL. If pvdQ is dependent on AHL, then when AHL  levels decreases, the pvdQ expression level will decrease accordingly. Such a negative  feedback system is not desirable to our project because it may lead to the  re-accumulation of AHL, re-establishing biofilm formation through quorum  sensing.  Thus, a system in which pvdQ  expression is dependent on P. aeruginosa cell density may alleviate this  problem.  </P><P align="left" style="text-align: left; font-style: normal; font-variant: normal; font-weight: normal; font-size: 13px; vertical-align: baseline; color: rgb(85, 85, 85); font-family: Lato, Tahoma, Arial, sans-serif; letter-spacing: normal; line-height: 19.5px; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; border: 0px none; margin-left: 0px; margin-right: 0px; margin-top: 0px; margin-bottom: 20px; padding: 0px"> </P><P align="left" style="text-align: left; font-style: normal; font-variant: normal; font-weight: normal; font-size: 13px; vertical-align: baseline; color: rgb(85, 85, 85); font-family: Lato, Tahoma, Arial, sans-serif; letter-spacing: normal; line-height: 19.5px; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; border: 0px none; margin-left: 0px; margin-right: 0px; margin-top: 0px; margin-bottom: 20px; padding: 0px"> </P><P align="left"><STRONG>Alginate  Encapsulation</STRONG></P><P align="left"> </P><P align="left" style="text-align: left; font-style: normal; font-variant: normal; font-weight: normal; font-size: 13px; vertical-align: baseline; color: #232323; font-family: Lato, Tahoma, Arial, sans-serif; letter-spacing: normal; line-height: 19.5px; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; border: 0px none; margin-left: 0px; margin-right: 0px; margin-top: 0px; margin-bottom: 20px; padding: 0px">Immobilizing living cells or other biomolecules in polymeric,  alginate gels is a technique of broadening use in biochemical or industrial  applications. These entrapped cells can be manipulated to excrete biomolecules of  therapeutic use. The alginate network not only keeps cells viable and  catalytically operating, but also protects them from biotic and abiotic stress as  well as toxic compounds. Alginate is a copious marine biopolymer that can form  heat stable, strong gels. These gels polymerize under moderate conditions at  room temperature. Additionally, in the process of cross-linking to enclose the  bacterial cell, the gel does not swell or shrink greatly. Instead, it maintains  a constant shape and boundary.  Thereby,  this method of encapsulation supports a steady bacterial cell density even  after lengthy durations of storage. 
  Encapsulation efficiency is evaluated by the ability of the system  to keep the cells physiologically and metabolically competent, maintain a high  bacterial cell density, and maximum encapsulation time. This is why the making  of the gel must include appropriate additives that stabilize and protect the  cells whilst their storage and transport. These supplements provide nutrition  to the bacterial cells without causing cell death. The technology must also  take into consideration the coordinated release of the biomolecule, and the  system’s biodegradability and cost-effectiveness.  
This method can be applied to HKU’s iGEM project as the whole cell <EM>E.coli </EM>can be enclosed in alginate  beads. The E.coli contain synthetic mechanisms for the production of pvdQ. A  signal peptide on the 5’-end of the pvdQ gene sequence, which secretes the  enzyme from the periplasmic space to the external environment, can be added to  directly transport the acylase into the environment where it can interact with  and degrade C-12 AHL molecules. Even though the bacterial cell remains trapped  within the alginate network, the pvdQ protein is small enough to diffuse from  the beads.   This system allows E.coli to  actively produce pvdQ, which can be utilized without being subject to lengthy  procedures of protein purification.  Pseudomonas  aeuroginosa is a soil-inhabiting bacterium. Thereby, E.coli-alginate beads can  be inoculated into the soil, and the subsequent release of pvdQ from the  immobilized gel matrix can be used to impede P.aeuroinosa biofilm formation.  Additionally, enzymes within the human body do  not hydrolyze alginate, unlike cellulose and other polymers that may be  alternatively used in encapsulation. Hence, these beads can also be directed to  sites of P.aeuroginosa colonization in vivo, mimicking the drug delivery  mechanisms. </P><P align="left">Suggested Protocol for pvdQ Excretion into the Soil: </P><P align="left"><STRONG><EM>Preparing the Alginate Solution: -</EM></STRONG></P><P align="left">1). A 2% concentration of the medium  viscosity sodium alginate solution is prepared by dissolving the alginate  powder in distilled water. 
  2). Subject the solution to agitation using  a magnetic stirrer at room temperature. 
  3). Autoclave the solution at 121°C  for 20 minutes. </P><P align="left"><STRONG><EM>Preparing the Additive Nutrient, Humic Acid: - </EM></STRONG></P><P align="left">1). Air dry 200g of peat soil. 
  2). Mix the air dried soil with 2,000nL  solution of 0.1 M NaOH and 0.1 M Ca(OH)2 (1:1) by shaking at 270 rpm for 4  hours. 
  3). Centrifuge the solution at 12,000xg for 30 minutes. 
  4). Collect the supernatant and acidify with 6M HCl to pH 1 overnight. 
  5). Filter the pH-adjusted supernatant through a sintered glass funnel. Dry  filterate at 80°C. 
  6). Prepare 10% stock solution by dissolving humic acid in deionized water.  Store in dark till subsequent use.<STRONG><EM>Preparing the Alginate Beads: -</EM></STRONG>
  1). Sterilize all the glassware and  solutions at 121°C for 20 minutes.
  2). Mix 2.5mL of 10% humic acid with 750uL of 30% glycerol. 
  3). Add the mixture to the 2% sodium alginate to obtain a final volume of 25mL. 
  4). Centrifuge 250mL of bacterial culture. Discard the  supernatant. 
  5). Wash the cell pellet with saline, 0.85% NaCl. 
  6). Suspend the saline-containing pellet in 25mL of the alginate-humic  acid mixture. Mix thoroughly. 
  7). The suspension should be added drop  b drop using a 26-gauze needle into pre-chilled sterile 1.5% aqueous CaCl2.  Should be performed under mild agitation. 
  8). Allow the beads to harden for 3-6 hours  at room temperature. </P><P align="left">The beads formed have a spherical shape with a diameter of 2-3mm and  a weight of about 7mg.  While selecting  the additive, several factors must be considered. Firstly, the nutritional  supplement must be able to be mixed homogenously with the alginate polymer.  Since alginate is polyanionic, the additive must be of a similar charge. Moreover,  the molecule should be of a desirable size such that the bacteria can utilize  it for survival in its immobilized state. It should also be able to provide the  carbon and nitrogen nutrients required by the entrapped cells. Humic acid is an  ideal organic compound for bacterial-beads added into the soil because it  satisfies these criteria. It is colloidal, producing particles of 5uM as well  as those that range between 0.04-0.5uM. </P><P align="left"> (a). Surface  of the intact bead. </P><P align="left"> </P><P align="left"> </P><P align="left"> </P><P align="left" style="text-align: left; font-style: normal; font-variant: normal; font-weight: normal; font-size: 13px; vertical-align: baseline; color: #232323; font-family: Lato, Tahoma, Arial, sans-serif; letter-spacing: normal; line-height: 19.5px; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; border: 0px none; margin-left: 0px; margin-right: 0px; margin-top: 0px; margin-bottom: 20px; padding: 0px"> </P><P align="left" style="text-align: left; font-style: normal; font-variant: normal; font-weight: normal; font-size: 13px; vertical-align: baseline; color: #232323; font-family: Lato, Tahoma, Arial, sans-serif; letter-spacing: normal; line-height: 19.5px; orphans: 2; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; border: 0px none; margin-left: 0px; margin-right: 0px; margin-top: 0px; margin-bottom: 20px; padding: 0px"> </P><P align="left">(d).  Bacteria distributed in the gel matrix of the beads.  A indicates the bacteria, and B indicates the  humic acid particles. </P><P align="left"> </P><P align="left"> </P><P align="left"> </P><P align="left"> </P><P align="left"> </P><DIV align="left"><OL><LI>Additional cleavage step is required for obtaining intact native protein  because the signal peptide remains attached to the secreted peptide.</LI><LI>Co-expression of component proteins in the system are required to  increase transport capacity but at the same time lead to the competition with  the target protein for the transport machinery.</LI></OL></DIV><P align="left"> </P><P align="left">Type II Secretion Mechanism
  SecB – Dependent Pathway
  Advantages:</P><P align="left">Drawbacks:</P><P align="left"><STRONG>Secretction  Protein system</STRONG></P><P align="left"> </P><P align="left">For the future development of PseudoKill, we need  to develop an extracellular secretion system for pvdQ protein for the following  reasons. Firstly, presenting system of PseudoKill can only transport the pvdQ  protein into the periplasm which means it can only digest the diffused C12-HSL  and its efficiency is limited. Secondly, protein aggregation is likely to  happen when gene expression is performed at high levels, threatening the  physiological condition of PseudoKill itself.</P><P align="left">Type I secretion mechanism are used commly for  recombinant protein secretion in E.coli B strains and it would be a suitable  choice for PseudoKill.</P><P align="left">Type I Secretion Mechanism
  Advantages:</P><DIV align="left"><OL><LI>Can transport target protein in one step across the two cellular  membranes without a periplasmic intermediate step.</LI><LI>Capable of transporting high-molecular weight components.</LI></OL><P align="left">Drawbacks:</P><P align="left">1. Additional cleavage step is required for obtaining intact native protein because the signal peptide remains attached to the secreted peptide.:</P><P align="left&gt;2. Co-expression of component proteins in the system are required to increase transport capacity but at the same time lead to the competition with the target protein for the transport machinery. &lt;/p&gt;

&lt;/body&gt;

&lt;/p&gt;
&lt;!-- 
NewPP limit report
Preprocessor node count: 4/1000000
Post-expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Expensive parser function count: 0/100
--&gt;

&lt;!-- Saved in parser cache with key 2012_igem_org:pcache:idhash:19974-0!1!0!!en!2!edit=0 and timestamp 20220829173816 --&gt;
&lt;div class=" printfooter="">
Retrieved from &quot;<A href="http://2012.igem.org/Team:HKU_HongKong/Project/Future_Implications.html">http://2012.igem.org/Team:HKU_HongKong/Project/Future_Implications.html</A>&quot;</P></DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:HKU_HongKong/Project/Future_Implications.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:HKU_HongKong/Project/Future_Implications.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:HKU_HongKong/Project/Future_Implications.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:HKU_HongKong/Project/Future_Implications.html&amp;oldid=229355" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>