<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LMU-Munich_Data_Vectors"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:LMU-Munich/Data/Vectors">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LMU-Munich/Data/Vectors&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Data/Vectors&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Data/Vectors&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LMU-Munich/Data/Vectors" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LMU-Munich/Data/Vectors</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3></DIV><DIV id="nav"><UL><LI><A href="/Team:LMU-Munich" title="Team:LMU-Munich">Home</A></LI><LI><A href="/Team:LMU-Munich/Team" title="Team:LMU-Munich/Team">Team</A></LI><LI><A href="/Team:LMU-Munich/Project" title="Team:LMU-Munich/Project">Project</A></LI><LI><A href="/Team:LMU-Munich/Why_Beadzillus" title="Team:LMU-Munich/Why Beadzillus">Why <B>Bead</B>zillus?</A></LI><LI><A href="/Team:LMU-Munich/Bacillus_Introduction" title="Team:LMU-Munich/Bacillus Introduction"><I>Bacillus</I> Introduction</A></LI><LI><A href="/Team:LMU-Munich/Bacillus_BioBricks" title="Team:LMU-Munich/Bacillus BioBricks"><B><I>Bacillus </I>B</B>io<B>B</B>rick<B>B</B>ox</A></LI><LI><A href="/Team:LMU-Munich/Germination_Stop" title="Team:LMU-Munich/Germination Stop"><B>Germination</B>STOP</A></LI><LI><A href="/Team:LMU-Munich/Spore_Coat_Proteins" title="Team:LMU-Munich/Spore Coat Proteins"><B>Sporo</B>beads</A></LI><LI><A href="/Team:LMU-Munich/Data" title="Team:LMU-Munich/Data">Data</A></LI><LI><A href="/Team:LMU-Munich/Application" title="Team:LMU-Munich/Application">Applications</A></LI></UL><LI><A href="/Team:LMU-Munich/Extras" title="Team:LMU-Munich/Extras">Extras</A></LI><LI><A href="/Team:LMU-Munich/Attributions" title="Team:LMU-Munich/Attributions">Attributions</A></LI><LI><A href="/Team:LMU-Munich/Collaborations" title="Team:LMU-Munich/Collaborations">Collaborations</A></LI><LI><A href="/Team:LMU-Munich/Parts" title="Team:LMU-Munich/Parts">Registry</A></LI><LI><A href="/Team:LMU-Munich/Strains" title="Team:LMU-Munich/Strains"><I>Bacillus</I> Strains</A></LI><LI><A href="/Team:LMU-Munich/Inverter" title="Team:LMU-Munich/Inverter">Special: Inverter</A></LI><LI><A href="/Team:LMU-Munich/Achievements" title="Team:LMU-Munich/Achievements">Achievements</A></LI><LI><A href="/Team:LMU-Munich/Safety" title="Team:LMU-Munich/Safety">Safety</A></LI><LI><A href="/Team:LMU-Munich/Laboratory_Safety" title="Team:LMU-Munich/Laboratory Safety">Project Safety</A></LI><LI><A href="/Team:LMU-Munich/Project_Safety" title="Team:LMU-Munich/Project Safety">Release for application</A></LI><LI><A href="/Team:LMU-Munich/Lab_Notebook" title="Team:LMU-Munich/Lab Notebook">Lab Notebook</A></LI><LI><A href="/Team:LMU-Munich/Weekly_Journal" title="Team:LMU-Munich/Weekly Journal">Weekly Journal</A></LI><LI><A href="/Team:LMU-Munich/Lab_Notebook/Protocols" title="Team:LMU-Munich/Lab Notebook/Protocols">Protocols</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice" title="Team:LMU-Munich/Human Practice">Human Practice</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/SynBio_Day" title="Team:LMU-Munich/Human Practice/SynBio Day">SynBio Day</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/Students_Practical_Course" title="Team:LMU-Munich/Human Practice/Students Practical Course">Students Practical Course</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/CAS_Conference" title="Team:LMU-Munich/Human Practice/CAS Conference">CAS SynBio Conference</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/Berlin" title="Team:LMU-Munich/Human Practice/Berlin">Berlin Conference</A></LI><LI><A href="/Team:LMU-Munich/Sponsors" title="Team:LMU-Munich/Sponsors">Sponsors</A></LI></DIV><DIV id="left"><DIV id="news"><P align="justify">
The LMU-Munich team is exuberantly happy about the great success at the World Championship Jamboree in Boston. Our project <B>Bead</B>zillus <B>finished 4th</B> and won the prize for the &quot;<B>Best Wiki</B>&quot; (with Slovenia) and &quot;<B>Best New Application Project</B>&quot;.</P><P align="right">
[ <A href="/Team:LMU-Munich/news" title="Team:LMU-Munich/news">more news</A> ]
</P></DIV></DIV><DIV id="right"><H2><SPAN class="mw-headline" id="Bacillus_subtilis_vectors"><I>Bacillus subtilis</I> vectors</SPAN></H2><P>For the detailed protocols, please see our <A href="https://2012.igem.org/Team:LMU-Munich/Lab_Notebook/Protocols" class="external text" rel="nofollow">protocol section</A>.
</P><H3><SPAN class="mw-headline" id="Overview">Overview</SPAN></H3><TABLE class="colored"><TBODY><TR><TH>Name BioBrick
</TH><TH><I>E. coli</I> res.
</TH><TH><I>B. subt.</I> res.
</TH><TH>Insertion locus
</TH><TH>Description
</TH><TH>Cloning status
</TH><TH>Works?
</TH></TR><TR><TH>pSB<SUB>Bs</SUB>1C <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823023" class="external text" rel="nofollow"><FONT color="#EBFCE4" size="2">(BBa_K823023)</FONT></A></TH><TD>Amp
</TD><TD>Cm
</TD><TD><I>amyE</I></TD><TD>empty
</TD></TR><TR><TH>pSB<SUB>Bs</SUB>4S <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823022" class="external text" rel="nofollow"><FONT color="#EBFCE4" size="2">(BBa_K823022)</FONT></A></TH><TD>Amp
</TD><TD>Spec
</TD><TD><I>thrC</I></TD><TD>empty
</TD></TR><TR><TH>pSB<SUB>Bs</SUB>2E <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823027" class="external text" rel="nofollow"><FONT color="#EBFCE4" size="2">(BBa_K823027)</FONT></A></TH><TD>Amp
</TD><TD>MLS
</TD><TD><I>lacA</I></TD><TD>empty
</TD><TD>in progress
</TD><TD>not tested
</TD></TR><TR><TH>pSB<SUB>Bs</SUB>1C-<I>lac</I>Z <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823021" class="external text" rel="nofollow"><FONT color="#EBFCE4" size="2">(BBa_K823021) </FONT></A></TH><TD>Amp
</TD><TD>Cm
</TD><TD><I>amyE</I></TD><TD><I>lacZ</I> reporter
</TD></TR><TR><TH>pSB<SUB>Bs</SUB>3C-<I>lux</I>ABCDE <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823025" class="external text" rel="nofollow"><FONT color="#EBFCE4" size="2">(BBa_K823025)</FONT></A></TH><TD>Amp
</TD><TD>Cm
</TD><TD><I>sacA</I></TD><TD><I>luxABCDE</I> reporter
</TD></TR><TR><TH>pSB<SUB>Bs</SUB>4S-P<SUB><I>Xyl</I></SUB><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823024" class="external text" rel="nofollow"><FONT color="#EBFCE4" size="2">(BBa_K823024)</FONT></A></TH><TD>Amp
</TD><TD>Spec
</TD><TD><I>thrC</I></TD><TD>Xylose-promoter
</TD><TD>refuses transformation
</TD></TR><TR><TH>pSB<SUB>Bs</SUB>0K-P<SUB><I>spac</I></SUB><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823026" class="external text" rel="nofollow"><FONT color="#EBFCE4" size="2">(BBa_K823026)</FONT></A></TH><TD>Amp
</TD><TD>Kan
</TD><TD>replicative
</TD><TD>IPTG-promoter
</TD><TD>unexpected
</TD></TR><TR><TH><B>Sporo</B>vector <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823026" class="external text" rel="nofollow"><FONT color="#EBFCE4" size="2">(BBa_K823054)</FONT></A></TH><TD>Amp
</TD><TD>Spec
</TD><TD><I>thrC</I></TD><TD>to make <A href="/Team:LMU-Munich/Spore_Coat_Proteins" title="Team:LMU-Munich/Spore Coat Proteins"><B>Sporo</B>beads</A></TD><TD>not tested
</TD></TR></TBODY></TABLE><DIV class="box"><H3><SPAN class="mw-headline" id="pSBBs1C">pSB<SUB><I>Bs</I></SUB>1C</SPAN></H3><P align="justify"> 
pSB<SUB>Bs</SUB>1C is one of our empty vectors. It integrates into the <I>amy</I>E locus of <I>Bacillus subtilis</I> and carries a chloramphenicol resistance as well as an ampicillin resistance for <I>E. coli</I>. In between the two recombination sites, there is the constitutively expressed Cm resistance and the multiple cloning site which contains RFP to simplify the insertion of your BioBrick.</P><P align="justify"> 
The plasmid is then transformed into <I>B. subtilis</I> and checked via the resistance for &quot;integration at all&quot; and via a starch test for &quot;integration into the right locus&quot;. Correct integration results in loss of halo due to disruption of the amylase gene.
</P><P align="justify"> 
We tested this vector with various inserts, for example the final construct of our Sporobeads with GFP. The starch test demonstrated that the plasmid inserted correctly in the <I>amy</I>E locus and fluorescence microscopy revealed the functionality of the construct.

</P></DIV><DIV class="box"><H3><SPAN class="mw-headline" id="pSBBs4S">pSB<SUB><I>Bs</I></SUB>4S</SPAN></H3><P align="justify"> 
pSB<SUB>Bs</SUB>4S is also an empty vector with only the multiple cloning site and the spectinomycin resistance gene in between the homologous regions. This vector integrates into the <I>thr</I>C locus of <I>B. subtilis</I>. </P><P align="justify"> 
We tested the integration of this plasmid with the threonine test and a construct from the <B>Suicide</B>switch as an insert.
</P></DIV><DIV class="box"><H3><SPAN class="mw-headline" id="pSBBs2E">pSB<SUB><I>Bs</I></SUB>2E</SPAN></H3><P align="justify"> 
pSB<SUB>Bs</SUB>2E is our third empty vector, aiming to provide a set of three empty vectors with three different integration loci and three different compatible resistance casseettes for full flexibility in their combination.
</P><P align="justify"> 
This vector is still a work in progress, but currently only the multiple cloning site is missing. Of course, it is not tested yet.

</P></DIV><DIV class="box"><H3><SPAN class="mw-headline" id="pSBBs1C-lacZ">pSB<SUB><I>Bs</I></SUB>1C-<I>lac</I>Z</SPAN></H3><P align="justify"> pSB<SUB>Bs</SUB>1C-<I>lac</I>Z is a reporter vector, which contains the multiple cloning site upstream of a functional reporter device, consisting of ribosome binding site, Î²-galactosidase gene <I>lac</I>Z and terminator. Also, a chloramphenicol resistance is in between the two flanking homolgy regions. Upon chloramphenicol selection, this vector integrates into the <I>amy</I>E locus, which can be verified by the starch test.</P><P align="justify"> 
This vector was used for several <A href="/Team:LMU-Munich/Data#Anderson_Promoter_Evaluation" title="Team:LMU-Munich/Data">promoter evaluations</A>.
</P></DIV><DIV class="box"><H3><SPAN class="mw-headline" id="pSBBs3C-luxABCDE">pSB<SUB><I>Bs</I></SUB>3C-<I>luxABCDE</I></SPAN></H3><P align="justify"> pSB<SUB>Bs</SUB>3C-<I>luxABCDE</I> is our second reporter vector. It contains the multiple cloning site upstream of a functional reporter device, consisting of the <I>lux</I>-operon and a downstream terminator. Upon chloramphenicol selection, this vector integrates, this vector integrates into the <I>sac</I>A locus of <I>B. subtilis</I> by double homologous recombination. </P> Correct integration is verified by colony PCR. One primer is located outside the integration site and its respective partner is located on the inegrated part of the vector, facing outwards. gDNA from W168 as a negative control should not give a PCR product.<P align="justify">
A pre-version of this vector with one forbidden <I>Pst</I>I site was used for most promoter evaluations (All graphs that you can find in the Data section of the promoters derive from this pre-version). To prove that our final version also works, we did another experiment where we compared the luminescence of P<SUB><I>lepA</I></SUB> in the pre- and final version of pSB<SUB>Bs</SUB>3C-<I>luxABCDE</I>. Here you can see the results of this experiment:</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><P align="justify">Luminescence measurements of the promoter P<SUB><I>lepA</I></SUB> in the pre-version of the vector pSB<SUB>Bs</SUB>3C-<I>luxABCDE</I> in comparison to the final vector.</P></DIV></DIV></DIV></DIV><P align="justify">
In this experiment, the finished vector showed about half of the activity throughout the measurement, hence while the absolute luminescence values are lower, the overall dynamics are unaffected.</P></DIV><DIV class="box"><H3><SPAN class="mw-headline" id="pSBBs4S-PXyl">pSB<SUB><I>Bs</I></SUB>4S-P<SUB><I>Xyl</I></SUB></SPAN></H3><P align="justify"><FONT size="4" color="FF0000">CAUTION: DOES NOT WORK! CANNOT BE TRANSFORMED INTO <I>B. SUBTILIS</I>!</FONT></P><P align="justify">
pSB<SUB><I>Bs</I></SUB>4S-P<SUB><I>Xyl</I></SUB> is an integrative expression vector. Upstream of the multiple cloning site, there is a xylose-inducible promoter, P<SUB><I>xyl</I></SUB>, followed by a terminator downstream. In the absence of xylose,  P<SUB><I>xyl</I></SUB> is inhibited by the repressor XylR, which is expressed in <I>B. subtilis</I> and not part of this vector. For selection, a spectinomycin resistance cassette is located in between the recombination sites. This vector integrates into the <I>thrC</I> locus, which can be checked by the threonine test. </P><P align="justify">
However, we could not manage to transform this vector into <I>B. subtilis</I>, although we tried three times. Moreover, the team from <A href="https://2012.igem.org/Team:Groningen" class="external text" rel="nofollow">Groningen</A> also tested this vector, but again was not able to transform it into <I>B. subtilis</I>, either.
</P><P align="justify">
There are two possible reasons for the failure: First, a defective resistance cassette or second, the presence of a forbidden restriction site preventing correct linearization prior to transformation.</P></DIV><DIV class="box"><H3><SPAN class="mw-headline" id="pSBBs0K-Pspac">pSB<SUB><I>Bs</I></SUB>0K-P<SUB><I>spac</I></SUB></SPAN></H3><P><FONT size="4" color="FF0000">CAUTION: THIS VECTOR DOES NOT WORK AS EXPECTED! THE PROMOTER IS STRONG CONSTITUTIVE INSTEAD OF IPTG-INDUCIBLE!</FONT></P><P align="justify">
pSB<SUB><I>Bs</I></SUB>0K-P<SUB><I>spac</I></SUB> is a replicative expression vector with a kanamycin resistance. The IPTG -inducible promoter P<SUB><I>spac</I></SUB> is followed by the multiple cloning site and a terminator. Moreover, it contains <I>lac</I>Y, encoding aÎ²-Galactosid-Permease, and <I>lac</I>I, encoding the Lac-repressor. In the presence of IPTG, LacI is released from the promoter and the gene of interest is expressed. 
</P><P align="justify">
The <I>ble</I> gene encodes the bleomycin resisitance protein (BRP), which can be used for by bleomycin selection in <I>E. coli</I> and <I>B. subtilis</I>. We did not use this resistance.
</P><P align="justify">
The presence of the vector can be checked by colony PCR with the primers: CTACATCCAGAACAACCTCTGC and TTCGGAAGGAAATGATGACCTC. We did not perfom the colony PCR because we selected for functionality of our lacZ-construct (see figure below) on plates with X-Gal and IPTG, and got blue colonies. 
</P><TABLE style="text-align:left;" cellpadding="3" width="70%" cellspacing="0" border="0" align="center"><TBODY><TR><TD style="width: 70%;background-color: #EBFCE4;"><P><FONT size="2"><I>lacZ</I>-activity in Miller units in pSB<SUB>Bs</SUB>0K-P<SUB><I>spac</I></SUB> depending on <I>B. subtilis</I> growth phase.</FONT></P></TD></TR></TBODY></TABLE><P align="justify">
This expression vector was tested by insertion of <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823016" class="external text" rel="nofollow"><I>lac</I>Z</A> into the multiple cloning site, transformation into <I>W168</I> and ONPG assays. Overnight cultures were diluted 1:100 in LB-medium and incubated at 37Â°C, 230 rpm. Cells were harvested in 2 ml reaction tubes and frozen. For the induction assay, a culture of about OD<SUB>600</SUB>=0.9 was split into 3 ml aliquots in test tubes with different IPTG-concentrations and incubated for another hour. (Same results were obtained for OD<SUB>600</SUB>=0.3, data not shown.)
</P><TABLE style="text-align:left;" cellpadding="3" width="70%" cellspacing="0" border="0" align="center"><TBODY><TR><TD style="width: 70%;background-color: #EBFCE4;"><P><FONT size="2"><I>lacZ</I>-activity in Miller units in pSB<SUB>Bs</SUB>0K-P<SUB><I>spac</I></SUB> depending on IPTG concentrations.</FONT></P></TD></TR></TBODY></TABLE><P align="justify">
In summary, the figures show that the expression is uniformly very strong [in comparison to the native <I>B. subtilis</I> promoters], even without addition of IPTG. It does change depending on the growth phase, with a maximum strength in the stationary phase.
</P><P align="justify">
This vector was designed for the inducible overexpression of proteins, but is constitutively active.
</P><P align="justify">
Possible reason for these results are: a dysfunctional Lac-repressor, or point mutations in the operator. This needs to be checked by sequencing.</P></DIV><DIV class="box"><H3><SPAN class="mw-headline" id="Sporovector"><B>Sporo</B>vector</SPAN></H3><P align="justify">
Our <B>Sporo</B>vector is specially designed to give you the opportunity to easily create <B>Sporo</B>beads with any fusion protein you can think of. The part between the <I>EcoR</I>I and <I>Pst</I>I is available as BioBrick in pSB1C3 and can be cloned in any of our empty <I>B. subtilis</I> vectors. We also offer it âpreclonedâ in pSB<SUB>Bs</SUB>4S, which in this version lacks the <I>NgoM</I>IV restriction sites.
</P><P align="justify">
The <B>Sporo</B>vector was created by ligation of three cut PCR-products with special overhangs (P<SUB><I>cotYZ</I></SUB>, <I>mRFP</I> and <I>cgeA</I>-B0014). It is designed to allow N-terminal fusions of genes in Freiburg standard (<A href="http://dspace.mit.edu/bitstream/handle/1721.1/45140/BBF_RFC%2025.pdf?sequence=1" class="external text" rel="nofollow">RFC25</A>). Towards that end, the gene to be inserted must be cut with <I>Xba</I>I and <I>Age</I>I and the vector must be digested with <I>Xba</I>I and <I>NgoM</I>IV. This removes the RFP cassette and allows easy selection for the integration of the new insert. After ligation, there is a scar of six nucleotides between both genes but no restriction site left. </P><P align="justify">
Expression of the gene fusion is controlled by the P<SUB><I>cot</I>YZ</SUB>-promoter, which was the strongest in our promoter evaluations. The C-terminal part of the resulting fusion gene encodes CgeA, one of the two known spore crust proteins.
</P><P align="justify">
The vector is finished but has not yet been tested with an insert.
We are working on three additional <B>Sporo</B>vectors to also offer C and N-terminal fusions with both <I>cot</I>Z and <I>cge</I>A.
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:LMU-Munich/Data/Vectors">http://2012.igem.org/Team:LMU-Munich/Data/Vectors</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LMU-Munich/Data/Vectors" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LMU-Munich/Data/Vectors" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LMU-Munich/Data/Vectors&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LMU-Munich/Data/Vectors&amp;oldid=284179" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>