"
Page
Discussion
View source
History
teams
Log in
 
Team:LMU-Munich/Lab Notebook/Protocols
From 2012.igem.org
Home
Team
Project
Why Beadzillus?
Bacillus Introduction
Bacillus BioBrickBox
GerminationSTOP
Sporobeads
Data
Applications
Extras
Attributions
Collaborations
Registry
Bacillus Strains
Special: Inverter
Achievements
Safety
Project Safety
Release for application
Lab Notebook
Weekly Journal
Protocols
Human Practice
SynBio Day
Students Practical Course
CAS SynBio Conference
Berlin Conference
Sponsors
The LMU-Munich team is exuberantly happy about the great success at the World Championship Jamboree in Boston. Our project Beadzillus finished 4th and won the prize for the "Best Wiki" (with Slovenia) and "Best New Application Project".
[ more news ]
On this page, we offer our unique protocols to the public.
Because Bacillus subtilis is our choice organism, we have focused on protocols specifically for Bacillus.
However below , you will also find other standard protocols we use.
Protocols for the work with B. subtilis
1. Growth, storage, media and components for Bacillus subtilis
This protocol gives the basic growth- and storage conditions, and the recipes for various media and the antibiotic concentrations used for B. subtilis.
2. Transformation of Bacillus subtilis
Protocol for the transformation of B. subtilis.
3. Isolation of chromosomal DNA from Bacillus subtilis for transformation
A quick protocol for how to isolate genomic DNA from B. subtilis to transform it to a different strain. By this means, you can very easily combine different B. subtilis genotypes. Transformation of genomic DNA is much more efficient than plasmids. Isolated DNA with this protocol is not clean!
4. Bacillus subtilis vectors
How to work with integrative B. subtilis vectors and how to verify the integration.
5. Isolation of chromosomal DNA from Bacillus subtilis (for PCR....)
Isolation of pure genomic DNA from B. subtilis.
6. Long Flanking Homology PCR
PCR protocol for long flanking homology PCR. You might need those PCR products for interrupting genes with resistance cassettes.
Example: Using LFH PCR to replace germination genes with resistance cassettes
An explanation of how we used this protocol for how we knocked out our germination genes using LFH.
7. Clean deletion of genomic fragments in Bacillus subtilis
Protocol for the production of markerless gene deletions. You will need the pMAD plasmid. This method is more time consuming than inserting resistance cassettes but offers better cell perfomance.
8. Spore germination assay
Protocol to count efficiency of sporulation and germination.
9. Enhancement of mature spore numbers
Protocol to enrich the spore numbers and purification from vegetative cells.
10. ß-Galactosidase Assay B. subtilis
Protocol how to measure lacZ gene activity in B. subtilis
11. Luminescence measurement
This protocol explains how cultures were treated in advance of a plate reader assay.
Other protocols we used
Competent Cells
E.coli competent cells and transformation. We used the XL1 blue strain .
Alkaline Lysis Plasmid Preparation
Alkaline Lysis Plasmid Preparation for E. coli
Restriction digest
Ligation
Standard restriction digest and ligation. We often used longer incubation times (up to over night) and did not dephosphorylate the backbones, if they had incompatible sticky ends.
β-Galactosidase Assay in E. coli
Quantitative lacZ-assay in E.coli.
Retrieved from " http://2012.igem.org/Team:LMU-Munich/Lab_Notebook/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
