<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LMU-Munich_Bacillus_BioBricks_vector_use"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:LMU-Munich/Bacillus_BioBricks/vector_use">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LMU-Munich/Bacillus_BioBricks/vector_use&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Bacillus_BioBricks/vector_use&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Bacillus_BioBricks/vector_use&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LMU-Munich/Bacillus_BioBricks/vector_use" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LMU-Munich/Bacillus BioBricks/vector use</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3></DIV><DIV id="nav"><UL><LI><A href="/Team:LMU-Munich" title="Team:LMU-Munich">Home</A></LI><LI><A href="/Team:LMU-Munich/Team" title="Team:LMU-Munich/Team">Team</A></LI><LI><A href="/Team:LMU-Munich/Project" title="Team:LMU-Munich/Project">Project</A></LI><LI><A href="/Team:LMU-Munich/Why_Beadzillus" title="Team:LMU-Munich/Why Beadzillus">Why <B>Bead</B>zillus?</A></LI><LI><A href="/Team:LMU-Munich/Bacillus_Introduction" title="Team:LMU-Munich/Bacillus Introduction"><I>Bacillus</I> Introduction</A></LI><LI><A href="/Team:LMU-Munich/Bacillus_BioBricks" title="Team:LMU-Munich/Bacillus BioBricks"><B><I>Bacillus </I>B</B>io<B>B</B>rick<B>B</B>ox</A></LI><LI><A href="/Team:LMU-Munich/Germination_Stop" title="Team:LMU-Munich/Germination Stop"><B>Germination</B>STOP</A></LI><LI><A href="/Team:LMU-Munich/Spore_Coat_Proteins" title="Team:LMU-Munich/Spore Coat Proteins"><B>Sporo</B>beads</A></LI><LI><A href="/Team:LMU-Munich/Data" title="Team:LMU-Munich/Data">Data</A></LI><LI><A href="/Team:LMU-Munich/Application" title="Team:LMU-Munich/Application">Applications</A></LI></UL><LI><A href="/Team:LMU-Munich/Extras" title="Team:LMU-Munich/Extras">Extras</A></LI><LI><A href="/Team:LMU-Munich/Attributions" title="Team:LMU-Munich/Attributions">Attributions</A></LI><LI><A href="/Team:LMU-Munich/Collaborations" title="Team:LMU-Munich/Collaborations">Collaborations</A></LI><LI><A href="/Team:LMU-Munich/Parts" title="Team:LMU-Munich/Parts">Registry</A></LI><LI><A href="/Team:LMU-Munich/Strains" title="Team:LMU-Munich/Strains"><I>Bacillus</I> Strains</A></LI><LI><A href="/Team:LMU-Munich/Inverter" title="Team:LMU-Munich/Inverter">Special: Inverter</A></LI><LI><A href="/Team:LMU-Munich/Achievements" title="Team:LMU-Munich/Achievements">Achievements</A></LI><LI><A href="/Team:LMU-Munich/Safety" title="Team:LMU-Munich/Safety">Safety</A></LI><LI><A href="/Team:LMU-Munich/Laboratory_Safety" title="Team:LMU-Munich/Laboratory Safety">Project Safety</A></LI><LI><A href="/Team:LMU-Munich/Project_Safety" title="Team:LMU-Munich/Project Safety">Release for application</A></LI><LI><A href="/Team:LMU-Munich/Lab_Notebook" title="Team:LMU-Munich/Lab Notebook">Lab Notebook</A></LI><LI><A href="/Team:LMU-Munich/Weekly_Journal" title="Team:LMU-Munich/Weekly Journal">Weekly Journal</A></LI><LI><A href="/Team:LMU-Munich/Lab_Notebook/Protocols" title="Team:LMU-Munich/Lab Notebook/Protocols">Protocols</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice" title="Team:LMU-Munich/Human Practice">Human Practice</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/SynBio_Day" title="Team:LMU-Munich/Human Practice/SynBio Day">SynBio Day</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/Students_Practical_Course" title="Team:LMU-Munich/Human Practice/Students Practical Course">Students Practical Course</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/CAS_Conference" title="Team:LMU-Munich/Human Practice/CAS Conference">CAS SynBio Conference</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/Berlin" title="Team:LMU-Munich/Human Practice/Berlin">Berlin Conference</A></LI><LI><A href="/Team:LMU-Munich/Sponsors" title="Team:LMU-Munich/Sponsors">Sponsors</A></LI></DIV><DIV id="left"><DIV id="news"><P align="justify">
The LMU-Munich team is exuberantly happy about the great success at the World Championship Jamboree in Boston. Our project <B>Bead</B>zillus <B>finished 4th</B> and won the prize for the &quot;<B>Best Wiki</B>&quot; (with Slovenia) and &quot;<B>Best New Application Project</B>&quot;.</P><P align="right">
[ <A href="/Team:LMU-Munich/news" title="Team:LMU-Munich/news">more news</A> ]
</P></DIV></DIV><DIV id="right"><H2><SPAN class="mw-headline" id="How_to_work_with_Bacillus_subtilis_vectors">How to work with Bacillus subtilis vectors</SPAN></H2><P align="justify">
Many vectors for <I>Bacillus subtilis</I> are integrating into the genome, so do all but one that we provide to the registry.
</P><P>There are some features in <I>B. subtilis</I> vectors that have to be taken into account, while working with them.
</P><UL><LI>	Cloning in <I>E.coli</I></LI><LI>	Linearisation before <I>B. subtilis</I> transformation
</LI><LI>	Verification of integration
</LI></UL><H3><SPAN class="mw-headline" id="Pre-Cloning_in_E._coli">Pre-Cloning in <I>E. coli</I></SPAN></H3><P align="justify">
The cloning, that means the insertion of your part into the multiple cloning site of a vector, is done by normal ligations and <I>E. coli</I>. However, since the <I>B. subtilis</I> vectors are quite large, the cloning works best if only one insert is inserted. You could try to finish your construct in e.g. pSB1C3 and then clone it into the <I>B. subtilis</I> vector. For convenience, all vectors carry an RFP with promoter and terminator which is substituted by your insert during the ligation.
</P><P align="justify">
All our vectors carry the <I>bla</I> gene that mediates Ampicillin resistance (100 µg/ml) in <I>E. coli</I>. Also, they all have two recombination sites for integration into the <I>B. subtilis</I> genome. In between those recombination sites, there is the multiple cloning site and a resistance marker for <I>B. subtilis</I>.
</P><P align="justify">
(Some vectors from other working groups do not carry an extra <I>E. coli</I> resistance, so the <I>B. subtilis</I> resistance is also used in <I>E. coli</I> but with lower antibiotic concentrations. There is also the possibility of single-crossover plasmids which do also work fine, but the vector can then easily cross-out again, so it is not stably integrated.)
</P><H3><SPAN class="mw-headline" id="Linearisation_before_transformation_in_B._subtilis">Linearisation before transformation in <I>B. subtilis</I></SPAN></H3><P align="justify">
For integration ofs by double crossover, plasmids (if it is not replicative) have to be linearized before transformation. <I>B. subtilis</I> is naturally competent, takes up DNA fragments and integrates preferably linear fragment into its genome via double cross-over. The linearization of our plasmids can all be performed with <I>Sca</I>I which cuts only inside the <I>bla</I> gene. If that enzyme also cuts in your insert, please check for other single cutters outside of the area that is integrated into the <I>B. subtilis</I> genome.
For the actual transformation of <I>B. subtilis</I>, please linearize 1-2 µg of your plasmid and then proceed with our <A href="https://static.igem.org/mediawiki/2012/4/41/LMU-Munich_2012_Transformation_of_Bacillus_subtilis.pdf" class="external text" rel="nofollow">transformation protocol</A>.
</P><H3><SPAN class="mw-headline" id="Verification_of_correct_integration">Verification of correct integration</SPAN></H3><P align="justify">
Transformants are plated on selective media containing the appropriate antibiotic (resistance gene in between recombination sites).  The obtained colonies then are tested for their insertion into the correct locus. Usually it is sufficient to test 4-8 colonies.
</P><P>To test the insertion into the
</P><UL><LI><P align="justify"><I>amyE</I>-locus:  <I>amyE</I> codes for an α-Amylase which degrades starch. Disruption of <I>amyE</I> disables <I>B. subtilis</I> of degrading starch. Starch is usually visualized by the starch-iodine reaction with Lugol's iodine that reveals a dark blue colour.
</P></LI></UL>
To test your transformants, streak the obtained colonies (and the WT as control) on a replica plate (with antibiotic) and on a starch plate and incubate overnight at 37°C. The next day, pour Lugol’s iodine  on the plate so that it is covered with a thin film. Around colonies which can degrade starch (WT and wrong colonies), there should be a bright zone around the colony. Correct clones do not show this bright zone. (see also Figure 1)  <TABLE style="text-align:left;" cellpadding="3" width="70%" cellspacing="0" border="0" align="center"><TBODY><TR><TD style="width: 70%;background-color: #EBFCE4;"><P><FONT size="2">Figure 1: Starch plate with <I>B. subtilis</I> streaked out colonies, covered with Lugol’s iodine. 3 strains can still hydrolyze starch, which can be seen by the bright surrounding area. The other clones have the insertion in the correct locus.</FONT></P></TD></TR></TBODY></TABLE><UL><LI><P align="justify"><I>thrC</I>-locus: <I>thrC</I> codes for the threonine synthase which performs an essential reaction for the production of the amino acid threonine. Disruption of that gene leads to threonine auxotrophy which can be tested for with minimal medium.Totest the transformants, streak the obtained colonies (and the WT as control) on a replica plate (with antibiotic) and on minimal medium without threonine and minimal medium with threonine (use the MNGE media, recipe see transformation protocol). Correct colonies should grow only on LB and minimal medium with threonine (see Figure 2).</P></LI></UL><TABLE style="text-align:left;" cellpadding="3" width="70%" cellspacing="0" border="0" align="center"><TBODY><TR><TD style="width: 70%;background-color: #EBFCE4;"><FONT size="2"><P align="justify">
Figure 2: Agar-plates with MNGE-medium; left plate with threonine added, right plate without threonine. All colonies grow well on the left plate but not at all on the right plate. Those colonies all have the insertion in the correct locus. The colony in the lower right corner (4) is an exception. Is grows on both plates, indicating that the plasmid is not in the correct locus.
</P></FONT></TD></TR></TBODY></TABLE><UL><LI><P align="justify"><I>sacA</I>-locus and <I>lacA</I>-locus: for those two loci, a colony PCR should be performed. The protocol can be found on our website. As shown below with an examplary locus, one of the primers of each pair is located on the genome, facing inwards, and the other one is located in between the recombination sites, facing outwards.</P></LI><LI><P align="justify">For sacA: you can use the following primers: up: TM2505: CTGATTGGCATGGCGATTGC together with TM2506: ACAGCTCCAGATCCTCTACG as well as down: TM2507: GTCGCTACCATTACCAGTTG together with TM2508: TCCAAACATTCCGGTGTTATC.
</P></LI></UL><TABLE style="text-align:left;" cellpadding="3" width="70%" cellspacing="0" border="0" align="center"><TBODY><TR><TD style="width: 70%;background-color: #EBFCE4;"><FONT size="2"><P align="justify"> 
Figure 3: Colony PCR of pSB<SUB>Bs</SUB>3C-<I>luxABCDE</I>-P<SUB>lepA</SUB> integrated into the <I>sacA</I>-locus. The expected bands are: up: 946 bp, down: 930 bp, none in WT and none with water as negative control. So all of the checked colonies have the insertion in the right locus.</P></FONT></TD></TR></TBODY></TABLE><UL><LI><P align="justify"> for <I>lacA</I>, you can use the primers: TM2624: GAACGAAGGGCTAAGAGAAC
</P></LI></UL><P>and TM2625:AAGCAGAAGGCCATCCTGAC (result: 650 bp)as well as TM2624 and TM2627: AAGAATCCGCCCATATCGAG (result: 3000 bp+ length of construct)
</P><P>With colony PCR you can check any other locus.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:LMU-Munich/Bacillus_BioBricks/vector_use">http://2012.igem.org/Team:LMU-Munich/Bacillus_BioBricks/vector_use</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LMU-Munich/Bacillus_BioBricks/vector_use" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LMU-Munich/Bacillus_BioBricks/vector_use" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LMU-Munich/Bacillus_BioBricks/vector_use&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LMU-Munich/Bacillus_BioBricks/vector_use&amp;oldid=301231" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>