<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_LMU-Munich_Bacillus_BioBricks_Reporters"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:LMU-Munich/Bacillus_BioBricks/Reporters">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:LMU-Munich/Bacillus_BioBricks/Reporters&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Bacillus_BioBricks/Reporters&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:LMU-Munich/Bacillus_BioBricks/Reporters&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:LMU-Munich/Bacillus_BioBricks/Reporters" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:LMU-Munich/Bacillus BioBricks/Reporters</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3></DIV><DIV id="nav"><UL><LI><A href="/Team:LMU-Munich" title="Team:LMU-Munich">Home</A></LI><LI><A href="/Team:LMU-Munich/Team" title="Team:LMU-Munich/Team">Team</A></LI><LI><A href="/Team:LMU-Munich/Project" title="Team:LMU-Munich/Project">Project</A></LI><LI><A href="/Team:LMU-Munich/Why_Beadzillus" title="Team:LMU-Munich/Why Beadzillus">Why <B>Bead</B>zillus?</A></LI><LI><A href="/Team:LMU-Munich/Bacillus_Introduction" title="Team:LMU-Munich/Bacillus Introduction"><I>Bacillus</I> Introduction</A></LI><LI><A href="/Team:LMU-Munich/Bacillus_BioBricks" title="Team:LMU-Munich/Bacillus BioBricks"><B><I>Bacillus </I>B</B>io<B>B</B>rick<B>B</B>ox</A></LI><LI><A href="/Team:LMU-Munich/Germination_Stop" title="Team:LMU-Munich/Germination Stop"><B>Germination</B>STOP</A></LI><LI><A href="/Team:LMU-Munich/Spore_Coat_Proteins" title="Team:LMU-Munich/Spore Coat Proteins"><B>Sporo</B>beads</A></LI><LI><A href="/Team:LMU-Munich/Data" title="Team:LMU-Munich/Data">Data</A></LI><LI><A href="/Team:LMU-Munich/Application" title="Team:LMU-Munich/Application">Applications</A></LI></UL><LI><A href="/Team:LMU-Munich/Extras" title="Team:LMU-Munich/Extras">Extras</A></LI><LI><A href="/Team:LMU-Munich/Attributions" title="Team:LMU-Munich/Attributions">Attributions</A></LI><LI><A href="/Team:LMU-Munich/Collaborations" title="Team:LMU-Munich/Collaborations">Collaborations</A></LI><LI><A href="/Team:LMU-Munich/Parts" title="Team:LMU-Munich/Parts">Registry</A></LI><LI><A href="/Team:LMU-Munich/Strains" title="Team:LMU-Munich/Strains"><I>Bacillus</I> Strains</A></LI><LI><A href="/Team:LMU-Munich/Inverter" title="Team:LMU-Munich/Inverter">Special: Inverter</A></LI><LI><A href="/Team:LMU-Munich/Achievements" title="Team:LMU-Munich/Achievements">Achievements</A></LI><LI><A href="/Team:LMU-Munich/Safety" title="Team:LMU-Munich/Safety">Safety</A></LI><LI><A href="/Team:LMU-Munich/Laboratory_Safety" title="Team:LMU-Munich/Laboratory Safety">Project Safety</A></LI><LI><A href="/Team:LMU-Munich/Project_Safety" title="Team:LMU-Munich/Project Safety">Release for application</A></LI><LI><A href="/Team:LMU-Munich/Lab_Notebook" title="Team:LMU-Munich/Lab Notebook">Lab Notebook</A></LI><LI><A href="/Team:LMU-Munich/Weekly_Journal" title="Team:LMU-Munich/Weekly Journal">Weekly Journal</A></LI><LI><A href="/Team:LMU-Munich/Lab_Notebook/Protocols" title="Team:LMU-Munich/Lab Notebook/Protocols">Protocols</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice" title="Team:LMU-Munich/Human Practice">Human Practice</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/SynBio_Day" title="Team:LMU-Munich/Human Practice/SynBio Day">SynBio Day</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/Students_Practical_Course" title="Team:LMU-Munich/Human Practice/Students Practical Course">Students Practical Course</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/CAS_Conference" title="Team:LMU-Munich/Human Practice/CAS Conference">CAS SynBio Conference</A></LI><LI><A href="/Team:LMU-Munich/Human_Practice/Berlin" title="Team:LMU-Munich/Human Practice/Berlin">Berlin Conference</A></LI><LI><A href="/Team:LMU-Munich/Sponsors" title="Team:LMU-Munich/Sponsors">Sponsors</A></LI></DIV><DIV id="left"><DIV id="news"><P align="justify">
The LMU-Munich team is exuberantly happy about the great success at the World Championship Jamboree in Boston. Our project <B>Bead</B>zillus <B>finished 4th</B> and won the prize for the &quot;<B>Best Wiki</B>&quot; (with Slovenia) and &quot;<B>Best New Application Project</B>&quot;.</P><P align="right">
[ <A href="/Team:LMU-Munich/news" title="Team:LMU-Munich/news">more news</A> ]
</P></DIV></DIV><DIV id="right"><H2><SPAN class="mw-headline" id="Bacillus_Reporters"><I>Bacillus</I> Reporters   </SPAN></H2><P align="justify">
We designed and codon-optimized a set of reporters that are commonly used in <I>B. subtilis</I>. All reporters have a modified iGEM Freiburg standard (<A href="http://partsregistry.org/Help:Assembly_standard_25" class="external text" rel="nofollow">RFC 25</A>) pre- and suffix for assembly of in-frame fusion proteins. Our prefix also includes the <I>B. subitlis</I> optimized ribosome binding site. </P><P>Find out more about the design of our <A href="https://static.igem.org/mediawiki/2012/b/b9/LMU-Munich_2012_Our_Freiburg_standard_FusionPrefix.pdf" class="external text" rel="nofollow">prefix with ribosome binding site</A>.
</P><P>prefix: GAATTCCGCGGCCGCTTCTAGATAAGGAGGAACTACTATGGCCGGC
</P><P>suffix: ACCGGTTAATACTAGTAGCGGCCGCTGCAGT
</P><UL><LI><B>GFP</B>        (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823039" class="external text" rel="nofollow">BioBrick:BBa_K823039</A>)
</LI></UL><TABLE style="text-align:left;" cellpadding="3" width="70%" cellspacing="0" border="0" align="center"><TBODY><TR><TD style="width: 70%;background-color: #EBFCE4;"><FONT size="2"><P align="justify">Fig. 5: Fluorescence of wild type spores and B53-<B>Sporo</B>beads (containing the P<SUB><I>cotYZ</I></SUB>-<I>cotZ</I>-<I>gfp</I>-terminator construct)</P></FONT></TD></TR></TBODY></TABLE><P align="justify">We took a <I>gfp</I> derivate of the <I>Bacillus subtilis</I> plasmid pGFPamy and added the BioBrick compatible pre- and suffix of the Freiburg standard (<A href="http://partsregistry.org/Help:Assembly_standard_25" class="external text" rel="nofollow">RFC 25</A>). The functionality of this BioBrick verified by creat e.g. our <A href="https://2012.igem.org/Team:LMU-Munich/Spore_Coat_Proteins" class="external text" rel="nofollow"><B>Sporo</B>beads</A>.</P><UL><LI><B>mKate2</B>      (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823029" class="external text" rel="nofollow">BioBrick:BBa_K823029</A>)
</LI></UL><TABLE style="text-align:left;" cellpadding="3" width="70%" cellspacing="0" border="0" align="center"><TBODY><TR><TD style="width: 70%;background-color: #EBFCE4;"><FONT size="2"><P align="justify"><B><I>mKate2</I> fused to the terminator B0014 under the control of the Anderson promoter J23101 (up), P<SUB><I>liaI</I></SUB> (middle) and P<SUB><I>lepA</I></SUB> (down) in pSB<SUB><I>Bs</I></SUB>1C.</B> Pellets are <I>Escherichia coli</I> cells which contain the plasmid with the insert labelled to the right.</P></FONT></TD></TR></TBODY></TABLE><P align="justify">We codon-optimized the sequence of this monomeric far-red fluorescence protein for the use in <I>B. subtilis</I> and synthesized it (by <A href="http://de-de.invitrogen.com/site/de/de/home/Products-and-Services/Applications/Cloning/gene-synthesis.html" class="external text" rel="nofollow">GeneArt</A>) with pre- and suffix of the Freiburg standard. We cloned this reporter in front of the terminator B0014. For the evaluation, this reporter was successfully combined with the promoters P<SUB><I>liaI</I></SUB>, P<SUB><I>lepA</I></SUB> and the Anderson promoter J23101 in the empty <I>Bacillus</I> vector pSB<SUB><I>Bs</I></SUB>1C from our <B><I>Bacillus</I>B</B>io<B>B</B>rick<B>B</B>ox. At the moment we already have the right construct integrated into the chromosome of <I>B. subtilis</I>. Unfortunately, we have not yet measured this reporter.</P><P>Reference: <A href="http://www.evrogen.com/products/mKate2/mKate2.shtml" class="external text" rel="nofollow">mkate2</A></P><UL><LI><B>LacZ</B>       (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823019" class="external text" rel="nofollow">BioBrick:BBa_K823019</A>)
</LI></UL><TABLE style="text-align:left;" cellpadding="3" width="70%" cellspacing="0" border="0" align="center"><TBODY><TR><TD style="width: 70%;background-color: #EBFCE4;"><FONT size="2"><P align="justify"><I>lacZ</I> fused to the terminator B0014 under the control of P<SUB><I>spac</I></SUB> in the expression vector pSB<SUB>Bs</SUB>0K-P<SUB><I>spac</I></SUB>. P<SUB><I>spac</I></SUB> induced with IPTG</P></FONT></TD></TR></TBODY></TABLE><P align="justify">This evaluated <I>lacZ</I> gene is derived from the <I>Bacillus</I> reporter vector pAC6. It is constructed in the Freiburg Standard (<A href="http://partsregistry.org/Help:Assembly_standard_25" class="external text" rel="nofollow">RFC 25</A>) for in-frame fusion proteins. It also includes a ribosome binding site optimized for <I>Bacillus subtilis</I> translation. This <I>lacZ</I> BioBrick was tested in the expression vector pSB<SUB><I>Bs</I></SUB>0K-P<SUB><I>spac</I></SUB>. This construct gave output, so the reporter gene is functional qualitatively. See <A href="https://2012.igem.org/Team:LMU-Munich/Data/Vectors" class="external text" rel="nofollow">Data</A> in the vector evaluation section of pSB<SUB><I>Bs</I></SUB>0K-P<SUB><I>spac</I></SUB>. A more qualitatively measurement will follow soon.</P><P>Reference for the vector: <A href="http://www.ncbi.nlm.nih.gov/pubmed/11902727" class="external text" rel="nofollow">pAC6</A></P><UL><LI><B>luc</B>    (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K823028" class="external text" rel="nofollow">BioBrick:BBa_K823028</A>)
</LI></UL><P align="justify">
The <I>luc</I> gene encodes the monomeric firefly luciferase, which requires a special substrate to produce luminescence. We codon-optimized it for <I>Bacillus subtilis</I>, added a ribosome binding site and synthesized by <A href="http://de-de.invitrogen.com/site/de/de/home/Products-and-Services/Applications/Cloning/gene-synthesis.html" class="external text" rel="nofollow">GeneArt</A>. 
</P><P align="justify">
Firefly luciferase is by far the most commonly used bioluminescent reporter. This monomeric enzyme of 61kDa catalyzes a two-step oxidation reaction to yield light, usually in the green to yellow region, typically 550â570nm . The first step is activation of the luciferyl carboxylate by ATP to yield a reactive mixed anhydride. In the second step, this activated intermediate reacts with oxygen to create a transient dioxetane that breaks down to the oxidized products, oxyluciferin and CO<SUB>2</SUB>. Upon mixing with substrates, firefly luciferase produces an initial burst of light that decays over about 15 seconds to a low level of sustained luminescence. This kinetic profile reflects the slow release of the enzymatic product, thus limiting catalytic turnover after the initial reaction.
</P><P align="justify">
The popularity of native firefly luciferase as a genetic reporter is due to the sensitivity and convenience of the enzyme assay and tight coupling of protein synthesis with enzyme activity. Firefly luciferase, which is encoded by the <I>luc</I> gene, is a monomer that does not require any post-translational modifications; it is available as a mature enzyme directly upon translation of its mRNA. Catalytic competence is attained immediately after release from the ribosome. Also, luciferase has a very short half-life in cells (approximately 3 hours). Combined, these properties make luciferase an extremely responsive reporter, far more so than other commonly used reporters.
</P><TABLE style="text-align:left;" cellpadding="3" width="70%" cellspacing="0" border="0" align="center"><TBODY><TR><TD style="width: 70%;background-color: #EBFCE4;"><P><FONT size="2">Reference:<A href="http://www.promega.com/resources/product-guides-and-selectors/protocols-and-applications-guide/bioluminescent-reporters/" class="external text" rel="nofollow">Promega</A></FONT></P></TD></TR></TBODY></TABLE><P align="justify">It was used in <I>B. subtilis</I> before (<A href="http://www.ncbi.nlm.nih.gov/pubmed/21552330" class="external text" rel="nofollow">Mirouze et al. 2011</A>).
The non-optimized version of this reporter gene was already successfully used in B. subtilis. While we have not found the time to evaluate this BioBrick we expect that it will work at least as well.</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:LMU-Munich/Bacillus_BioBricks/Reporters">http://2012.igem.org/Team:LMU-Munich/Bacillus_BioBricks/Reporters</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:LMU-Munich/Bacillus_BioBricks/Reporters" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:LMU-Munich/Bacillus_BioBricks/Reporters" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:LMU-Munich/Bacillus_BioBricks/Reporters&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:LMU-Munich/Bacillus_BioBricks/Reporters&amp;oldid=284042" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>