"
Page
Discussion
View source
History
teams
Log in
 
Team:SCUT-China/Project/PPS
From 2014.igem.org
Home
Team
Members
Attributions
Sponsors
Collaborators
Notebook
Protocol
Project
Background
Overview
Polyketide Synthesis
Chassis
PPS
Docking Domain
Have A Try
Modeling
Overview
Simulation
Results
Data Page
Parts
Judging Criteria
Policy&Practices
Overview
Campus Activity
Public Outreach
Social Media
Life Science Community
Survey
Evaluation
Safety
Background
Overview
Polyketide Synthesis
Background Design Operation References
Chassis
Overview Design Result References
PPS
Part1 Part2 Part3 References
Docking Domain
Overview Design Result Conclusion
Have A Try
Background Design Operation
PPS-Programmed Polyketide Synthesis
The domain in type I polyketide synthase(PKS) can work independently without losing their function[1]. We want to standardize the domain and assemble them in correct order and finally we can rational design and produce new-to-nature polyketide as we want.
1.What kind of polyketide synthase do we choose as our candidate?
Polyketide almost exist in every organisms but many of their synthesis processes are unclear[2]. First thing we did is to search suitable PKS. Erythromycin PKS modular 6-deoxyerythronolide B synthase(DEBS) from Saccharopolyspora erythraea is a good candidate[3]. It has served as a module system for researching to engineer PKSs and to explore their structural plasticity and substrate tolerance[4] for long. DEBS, encoded by the gene eryA, are divided into three part including totally six module. DEBS1 which include three module, is the first part of DEBS[5]. (figure.1)
2.How to search the boundary between the domains and linker in a module?
The domains in PKS have independent functions[6]. We tried to separate different domains in the sequence of erythromycin type I polyketide synthase. To make sure that the integrity of domains are unbroken, it’s important that searching for the boundary between domain precisely. We find out the boundary between two domain, each domain are connected by the short adhesion.We search two database ASMPKS (Analysis System for Modular Polyketide Synthases: http://gate.smallsoft.co.kr:8008/~hstae/asmpks/pks_prediction.pl ) and MAPSI (Management and Analysis for Polyketide Synthase type I: http://gate.smallsoft.co.kr:8008/pks/ ), both of which provide the information for researching type I polyketide synthase[4]. According to the information provided on two websites, we identify and seperate all the domains from DEBS1.
3. Generating new polyketide by “programming” domain in module.
After identifying and obtaining the sequence of each domains, we assemble the domain in the order of what we want. Then we can test whether the PKS are expressed correctly and the polyketide synthesized as we want.(figure.2) GC content of PKS DEBS1 sequence is very high, so we have the codon optimized to make it suitable for E.coli[7].
3.1 Standardzing the domain.
We first assemble the domain as original DEBS1+TE, to find whether our reconstructed domain works.(figure1)
Figure1: Standardzing of domains
At the same time, we try to exchange position of domain with the same function. We exchange the domain KR and ACP and insert the didomain DH+ER into different module to know whether our standardized domains works[8].(figure2)
Figure2: Repositioning and insert of domain inside module.
Thirdly, we obtain the loading module from diffenent organisms.
Loading module is the first module in polyketide synthase that recognize the starter unit of the polyketide chain and initiates the polyketide synthesis[9]. It include at least two domains that AT and ACP. Some loading modules may contain KS. The loading module influences the efficience of polyketide synthesis as it start the synthesis process.
We choose loading module of PKS of pyoluteorin[10] and amphotericin[11]. Erythromycin’s starter unit is propionyl-CoA and The starter unit of Amphotericin are acetyl-CoA. (Figure3)
Figure3: Exchange of loading module
Starter unit of Pyoluteorin in two PKS database is different that malonyl-CoA in ASMPKS and acetyl-CoA in MAPSI. We compare the efficience of that loading module between two concentration of substrates.
The loading module of Amphotericin contain an redundant domain DH that can’t modify the starter unit or any other polyketide unit[11]. We eliminate the DH domain of Amphotericin to optimize loading module.(figure.4)
Figure4: Elimation of redundant dh of amphotericin
Finally, we will test the seletivity of diffenent AT so that we can determine the structure by selecting the substrates. AT domain is responsible for selecting CoA linked extender as building blocks for constructing the polyketide chain[12].
3.2 "programming" the PKS
By all these work we do, we can control the structure of polyketide in three aspect: by choosing or engineering the appropriate host, the supplement of building blocks will be enough[13][14]; then, by choosing the suitable loading module and KS-AT domain, the PKS can select the building blocks of polyketide synthesis. Finally, according to the structure of polyketide, we can insert different modified domains into specific module, so that the building blocks can be modified correctly.
We will try to establish a database that can provide the information about the utilization of  standardized domain. According to the structure of polyketide that user need, the database can provide the information about the assemble of standardization domains and host needed for synthsis. Achieving the programmed synthesis of polyketide.(figure5)
Figure5: how do we produce new polyketide. a) approriate host for producing substrate; b)choose suitable didomain KS-AT to select correct building blocks; c)arrange the suitable proccessing domains to modify the polyketide chain.
Reference
[1]Cane, David E. "Programming of erythromycin biosynthesis by a modular polyketide synthase." Journal of Biological Chemistry 285.36 (2010): 27517-27523.
[2]Komaki, Hisayuki, et al. "Genome based analysis of type-I polyketide synthase and nonribosomal peptide synthetase gene clusters in seven strains of five representative Nocardia species." BMC genomics 15.1 (2014): 323.
[3]Pfeifer, Blaine A., et al. "Biosynthesis of complex polyketides in a metabolically engineered strain of E. coli." Science 291.5509 (2001): 1790-1792.
[4]Tae, Hongseok, Jae Kyung Sohng, and Kiejung Park. "Development of an analysis program of type I polyketide synthase gene clusters using homology search and profile hidden Markov model." Journal of microbiology and biotechnology 19.2 (2009): 140-146.
[5]Cortes, Jesus, et al. "An unusually large multifunctional polypeptide in the erythromycin-producing polyketide synthase of Saccharopolyspora erythraea." (1990): 176-178.
[6]Khosla, Chaitan, Shiven Kapur, and David E. Cane. "Revisiting the modularity of modular polyketide synthases." Current opinion in chemical biology 13.2 (2009): 135-143.
[7]Menzella, Hugo G., et al. "Redesign, synthesis and functional expression of the 6-deoxyerythronolide B polyketide synthase gene cluster." Journal of Industrial Microbiology and Biotechnology 33.1 (2006): 22-28.
[8]Oliynyk, Markiyan, et al. "A hybrid modular polyketide synthase obtained by domain swapping." Chemistry & biology 3.10 (1996): 833-839.
[9]Lau, Janice, David E. Cane, and Chaitan Khosla. "Substrate specificity of the loading didomain of the erythromycin polyketide synthase." Biochemistry39.34 (2000): 10514-10520.
[10]Nowak-Thompson, Brian, et al. "Characterization of the pyoluteorin biosynthetic gene cluster of Pseudomonas fluorescens Pf-5." Journal of bacteriology 181.7 (1999): 2166-2174.
[11]Caffrey, Patrick, et al. "Amphotericin biosynthesis in Streptomyces nodosus deductions from analysis of polyketide synthase and late genes." Chemistry & biology 8.7 (2001): 713-723. [12]Dunn, Briana J., et al. "Comparative analysis of the substrate specificity of trans-versus cis-acyltransferases of assembly line polyketide synthases."Biochemistry (2014).
[13]Jiang, Ming, and Blaine A. Pfeifer. "Metabolic and pathway engineering to influence native and altered erythromycin production through E. Coli ."Metabolic engineering 19 (2013): 42-49. [14]Chen, Xianzhong, et al. "Metabolic engineering of Escherichia coli : A sustainable industrial platform for bio-based chemical production."Biotechnology advances 31.8 (2013): 1200-1223.
Retrieved from " http://2014.igem.org/Team:SCUT-China/Project/PPS "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
