"
Page
Discussion
View source
History
teams
Log in
 
Team:SCUT-China/Project/Chassis
From 2014.igem.org
Home
Team
Members
Attributions
Sponsors
Collaborators
Notebook
Protocol
Project
Background
Overview
Polyketide Synthesis
Chassis
PPS
Docking Domain
Have A Try
Modeling
Overview
Simulation
Results
Data Page
Parts
Judging Criteria
Policy&Practices
Overview
Campus Activity
Public Outreach
Social Media
Life Science Community
Survey
Evaluation
Safety
Background
Overview
Polyketide Synthesis
Background Design Operation References
Chassis
BAP1 Construction Substrate supply
PPS
Part1 Design Result Conclusion
Docking Domain
Overview Design Result Conclusion
Have A Try
Background Design Operation
Chassis
BAP1 Construction
Introduction
Since the system we design is used for synthesis of polyketones, it is necessary to ensure the supply of various substrates. As a typical substrate of diversified polyketones, propionyl-CoA is needed to reach the level of enough concentration. Thus, we knock out some pathways in E.coli to increase the concentration of propionyl-CoA. Moreover, we enhance the pathways that convert propionate into propionyl-CoA. By the way, sfp gene is also integrated into BL21 (DE3), and BAP1 is achieved after all of these modification.
Design
In E.coli, propionate is converted into propionyl-CoA and then utilized for metabolism with the steps catalyzed by proteins of the prp locus. Specifically, PrpE is in charge of the converting part, and PrpRBCD utilize propionyl-CoA as a metabolic substrate. Therefore, we delete prpRBCD genes in order that the ability of E.coli to utilize propionate as a carbon and energy source was eliminated. However, propionyl-CoA can only be produced through the converting reaction, so the prpE gene was placed under control of an IPTG-inducible T7 promoter.
By the way, using homologous recombination, a single copy of the sfp gene under control of the T7 RNA polymerase promoter was integrated in the prp operon of BL21(DE3), yielding E. coli BAP1.
The sfp gene product, which is part of the surfactin biosynthetic gene cluster in Bacillus subtilis, can effectively modify ACPs from all PKS subclasses as well as related peptidyl carrier protein and aryl carrier protein domains from nonribosomal peptide synthetases (NRPSs). Only with the function of Sfp can propionyl-CoA be linked to ACPs, which is the basement of the reactions.
In the preparatory phase, we transformed pKD46 into BL21, as preparation for gene knock-out. pKD46 contains Red system which is composed from three genes (gam, bet, exo ). The Gam protein prevents an E. coli nuclease of RecBCD from degrading linear DNA fragments, thus allowing preservation of transformed linear DNA in vivo. The bet gene product, Beta, is an ssDNA-binding protein that promotes annealing of two complementary DNA molecules, and the exo gene product, Exo, has a 5´ to 3´ dsDNA exonuclease activity.
After the preparation, we achieved the target gene as above by fusion PCR, which was then transformed into BL21(DE3), yielding E. coli BAP1. Using prpE and the fragment of prpR as homologous arm, we can ensure the gene replacement happens in targeted location. And Sh Ble gene product is zeocin-resistant, helping us to screen E.coli.
Results
After gene knock-out, we achieved BAP1 as the preparation for those following experiments. According to PCR results, prpRBCD gene had been knocked out while sfp gene and prpE gene is still kept on. Obviously, the gene had been transformed into the targeted location.
Conclusion
Although frameshift mutation has taken place in sfp gene, we transformed plasmids with sfp gene into BAP1 to make up for it, which has been proved to work as the following experiments showed. Therefore, by the gene knock-out, we have successfully achieved workable BAP1.
Reference
[1] Pfeifer B A, Admiraal S J, Gramajo H, et al. Biosynthesis of complex polyketides in a metabolically engineered strain of E. coli ［J］. Science,2001,291:1790
[2] M. Madyagol, H. Al-Alami, Z. Levarski, H. Drahovská, J. Tur˜na, S. Stuchlik, Gene replacement techniques for Escherichia coli genome modification, Folia Microbiol. 56 (2011) 253–263.
[3] A.R. Horswill, J.C. Escalante-Semerena, The prpE gene of Salmonella typhimurium LT2 encodes propionyl-CoA synthetase, Microbiology 145 (1999) 1381–1388
[4] Pfeifer, B.A., Khosla, C., 2001. Biosynthesis of polyketides in heterologous hosts. Microbiol. Mol. Biol. Rev. 65, 106.
Substrate supply
Background
There are three enzymes needed in our chassis. Propionyl-CoA carboxylase(pcc), an enzyme needed in the biosynthesis of polyketides, is an enzyme complex containing two components playing the core role in our chassis, of which accA is the α biotinylated subunit functioning as the acetyl-CoA or propionyl-CoA carboxylase while pccB，the β subunit，acts as the carboxyl transferase【1】. BirA is a biotin ligase that improves the activity of accA, acts as an enhancer like sfp, the phosphopantetheinyl transferase facilitating posttranslational modification of the DEBS proteins in E.coli BL21(DE3) [2]. With the pcc and sfp genes transformed into E.coli BL21(DE3), it will express DEBS genes at the comparable level to S. erythraea or S.coelicolor [2].
Design
As the three enzymes are suitable to produce the substrate for all of our polyketide synthases(PKS), we decided to co-express them in one plasmid—the low copy plasmid pSB4A5 and make it the chassis for our expression of PKSs.
Figure 2: Our design of the desired genes on pSB4A5. Sfp was originally desigened to be knocked into the prp operon of E.coli BL21(DE3) by homologous recombination but was frame shifted, so we put it on pSB4A5 with individual PT7 and T7T. (A) The desired gene of chassis 1. (B) The desired gene of chassis 2. PT7, T7 promotor; RBS, ribosome binding site; 2TM, double T7terminator; birA, the biotin ligase gene; accA, the α biotinylated subunit gene of Propionyl-CoA carboxylase; pccB, the β subunit gene of Propionyl-CoA carboxylase; sfp, the phosphopantetheinyl transferase gene.
Here, we use T7 promoter and terminator to stay the same with our plasmid pSB1C3 carrying the genes encoding DEBS1 PKS so that they will be co-expressed under the same condition. RBS is placed between every two genes in order to ensure the transcription of every gene. Since the sfp gene knocked into the prp operon was frame shifted, we worried about whether it works as expected. Therefore，we constructed two chassis, of which one carries sfp(chassis 1) while another(chassis 2) doesn’t, and we will verify the activity of the sfp by transforming them into the E.coli BL21(DE3) separately with pSB1C3 carrying DEBS1 at the same time and see if there will be any difference between these two strain of E.coli BL21(DE3) under the same condition.
Construction
Since we didn’t have much time to buy Streptomyces coelicolor A3(2) to get gene accA and pccB from it by pcr. Besides, the Saccharopolyspora erythraea NRRL23338 we already had is not suitable to provide the two genes we needed[3], we chose to synthesize them. Meanwhile, we gained sfp and birA by pcr from E.coli BL21(DE3) and optimized their sequence by site-specific mutagenesis. Then, we constructed the pSB4A5 chassis by digest and assemble them step by step.
Result
1. Construction
Figure3: Analysis of the chassis we constructed. The 8.5kb plasmid pSB4A5 was digest by EcoRⅠand PstⅠinto two pieces, the 3.4kb backbone and the 5.2kb desired gene. M: 1kb marker; lane1、2 are parallel sample.
Figure 4:Analysis of the two devices transformed into E.coli BL21(DE3), the 15kb device 1 pSB1C3 and the 8.5kb device 2 pSB4A5. M1: 1kb marker; M2 DL15,000 marker; lane1~8: sample1~8 of E.coli BL21(DE3) transformed with both devices.
2. Transcription level
To test the transcription level of our desired genes, we did a series of RT PCR. Besides, as birA is an endogenous gene of E.coli BL21(DE3), we didn’t test it.  From figure 5 we could draw a conclusion that accA, pccB and sfp were transcripted as expected.
Figure 5: Transcription level of accA、pccB and sfp, with 16S rRNA2 as the reference gene. ACC: accA; PCC: pccB; SFP: sfp.
3. SDS-PAGE
We detected target protein of the metabolically engineered strain of E. coli BL21(DE3) by SDS-PAGE.
Figure 6: Analysis of the cell disruption product, from which we can see accA、birA and sfp were produced. PccB was not found, but the result of mass spectrometry analysis shows that the expected product triketide lactone was produced, meaning that pccB was translated successfully in our engineered E.coli, too.
Conclusion
We constructed the chassis during August and detected whether it works as expected in the early September. From result 1 and result 2, we can safely draw a conclusion that our chassis was successfully constructed. The later MS analysis shows that it works as expected. As the first desired product triketide lactone was detected and proved our chassis does work, we will see whether it works with other polyketide synthase we transformed into E.coli BL21(DE3).
Reference
1. Rodrıguez E, Gramajo H. Genetic and biochemical characterization of the α and β components of a propionyl-CoA carboxylase complex of Streptomyces coelicolor A3 (2)[J]. Microbiology, 1999, 145(11): 3109-3119. 2. Biosynthesis of Complex
2. Pfeifer B A, Admiraal S J, Gramajo H, et al. Biosynthesis of complex polyketides in a metabolically engineered strain of E. coli[J]. Science, 2001, 291(5509): 1790-1792.
3. Oliynyk M, Samborskyy M, Lester J B, et al. Complete genome sequence of the erythromycin-producing bacterium Saccharopolyspora erythraea NRRL23338[J]. Nature biotechnology, 2007, 25(4): 447-453.
Retrieved from " http://2014.igem.org/Team:SCUT-China/Project/Chassis "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
