Team:CIEI-BJ/Results
Top
Home
Achievements
Medals
Parts
Project
Background
Design
Results
Experiments
Modeling
InterLab
Human_Practices
Notebook
Safety
Lab Book
Protocol
Team
Gallery
Attributions
Collaborations
Results
Preparation and parts
Operation of the Activation Module
Operation of the Detection Module
Operation of the Degradation Module
Conclusion
Future Direction
Results
From our design, this project includes experiments in 4 sections: the preparation work to construct all the vectors and required parts; the demonstration of the activation by AFT-B1 in yeast, the detection of AFT-B1 in yeast, and the degradation analysis in both bacterium and yeast.
Preparation and parts
1. Enzyme genes
Because the genes for enzymes and antibody fragments used in this projects were not convenient to isolate from their original organisms in the lab, we simply synthesized them. For convenient of further use, we directly cloned the open reading frame (ORF) of each candidate gene in the pMAL-c5x vector (New England BioLabs) downstream the sequence for the maltose binding protein (MBP) fusion tag (Fig. 1).
Figure 1. Each of the genes for candidate enzymes were cloned into pMAL-c5x.
(A) The vector structure of pMAL-c5x.
(B) Colony PCR result of positive colonies harboring each of the genes. M, DNA size marker; lanes 1-4 are amplicons for thioredoxin-MSMEG 5998 fusion protein (F420), aflatoxin-detoxifizyme (ADTZ), Manganese peroxidase (MNP) and BacC from Bacillus, respectively.
2. Activation module
In this module, we fused the ORF for ScFv1 downstream of AD in pGAD-T7 vector (Clontech), and fused the ORF for ScFV2 downstream of BD in pGBK-T7 vector (Clontech). Digested vectors and amplified inserts are demonstrated in Fig. 2.
Figure 2. Fragments for the construction of the two vectors with AD and BD fusions.
M, DNA size marker; 1, pGAD-T7 digested by EcoRI; 2, AD-ScFv1 fusion gene; 3, pGBK-T7 digested by EcoRI; 4, BD-ScFv2 fusion gene.
After transformation into bacterial cells, positive colonies were picked and confirmed by PCR (Fig. 3).
Figure 3. Colonies PCR screening for positive colonies with correct AD-ScFv1 and BD-ScFv2 fusion genes. M, DNA size marker.
3. Detection module
For this module, we used the inducible Gal1 promoter and the gene for enhanced yellow fluorescent protein (EYFP). This module was cloned into pGBK-T7 vector together with the BD-ScFv2 fusion (Fig. 4, 5).
Figure 4. The Gal1::EYFP fusion gene (A) and the pGBK-T7/BD-ScFv2 vector digested by XhoI (B).
Figure 5. Colonies PCR screening for positive colonies with correct Gal1::EYFP fusion gene in pGBK-T7/BD-ScFv2. M, DNA size marker.
4. Degradation module
Similar to the detection module, we arranged the degradation module in pGAD-T7 that contains the BD-ScFv2 fusion. ORF of each of the candidate gene was cloned after the same Gal1 promoter to enable a simultaneous induction by the presence of AFT-B1. Figure 6 showed the digested vector of pGAD-T7/AD-ScFv1.
Figure 6. pGAD-T7/AD-ScFv1 vector digested by XhoI. M, DNA size marker.
Figure 7. Colonies PCR screening for positive colonies with correct Gal1::ADTZ fusion gene in pGAD-T7/AD-ScFv1. M, DNA size marker.
Operation of the Activation Module
First, we compared if AFT-B1 affects the growth of the yeast without any plasmids in a full medium. The growth curves were very similar (Fig. 8). This means that AFT-B1 itself does not affect the growth of the yeast. Then we compared the growth of the transformed yeast in the presence of AFT-B1 in different drop out media. Fig. 9 showed that the growth of yeast cells was significantly slower in the three drop-out medium (TDO, SD/-His/-Leu/-Trp), compared with that in the double drop-out medium (DDO, SD/-Leu/-Trp).
Figure 8. The growth of wild-type yeast in YPDA medium with (black line) or without (blue line) AFT-B1.
Figure 9. The growth rate of yeast cells with both pGAD-T7 and pGBK-T7 constructs in three drop-out medium (TDO, SD/-His/-Leu/-Trp) (black line) and in double drop-out medium (DDO, SD/-Leu/-Trp) (blue line) with the presence of AFT-B1.
This result demonstrated that, first, AFT-B1 was able to activate the expression of the HIS3 gene in the chromosome to facilitate the synthesis of histidine, however, second, the synthesis of histidine was a serious bottleneck that affected the growth of yeast cells, and would make our system not practical for future utilization. This was the reason that we only used HIS3 gene in early selection of correct transformants, but designed to use the EYFP gene for detection.
Operation of the Detection Module
According to our design, AFT-B1 is also able to activate the expression of EYFP. We observed the yeast cells growing in DDO medium with AFT-B1 under a fluorescent microscope first (Fig. 10).
Figure 10. The expression of EYFP was induced in yeast cells growing in DDO medium supplied with AFT-B1 at 100 μg/L.
We further cultivate yeast cells in DDO media supplied with different concentration of AFT-B1, harvested cells in parallel and performed western blot analysis using the antibody against EYFP (Fig. 11). Our results demonstrated that the expression of EYFP was induced by enhanced concentration of AFT-B1 in the medium in a dose-related manner. Moreover, our western blot also showed a drastic induction of EYFP at ATF-B1 concentrations above 500 μg/L.
Figure11. Western blot analysis showing the induction of EYFP in yeast cells by different concentration of AFT-B1 in the medium.
Operation of the Degradation Module
Because of the limited information of ATF-B1-degrading enzymes and their corresponding gens, we first tested their in vitro activities by expressing these enzymes in bacterial cells. When Escherichia coli BL21(DE3) cells were transformed, induced by IPTG at 1 mM at 30°C, all enzymes were successfully expressed (Fig. 12).
Figure 12. SDS-PAGE showing the expression of all enzyme proteins in E. coli cells after IPTG induction. Lanes 1-4 are aflatoxin-detoxifizyme (ADTZ), Manganese peroxidase (MNP) and BacC from Bacillus and thioredoxin-MSMEG 5998 fusion protein (F420), respectively. M, protein size marker.
In vitro degradation reaction of AFT-B1 was performed in a final volume of 700 μL composed of 150 μL H2O, 350 μL buffer (100 mM Na2HPO4, 50 mM citric acid, 0.4 mg/L AFT-B1, pH 6.0) and 200 μL crude bacterial extract after sonication. The mixture was incubated in the dark at 30°C without shaking for 0, 3, 6, and 12 h. After incubation, AFT-B1 was extracted three times with 700 μL chloroform. After the chloroform was evaporated under nitrogen gas, the samples were dissolved in 140 μL acetonitrile and analyzed by high performance liquid chromatography (HPLC) using a Diamonsil C18 column (250×4.6 mm). The mobile phase was acetonitrile/water (45:55, v/v) at a flow rate of 1 mL/min, and the sample temperature was 28°C. The detection wavelength was 360 nm. From our assay, only ADTZ was found to be able to degrade AFT-B1 (Fig. 13).
Figure 13. HPLC analysis of AFT-B1 degradation by ADTZ.
We also tested the in vivo degradation of AFT-B1 by different enzymes in yeast cells. However, we have not detected a significant degradation yet. The system is under optimization.
For a robust screening of novel enzymes that are able to degrade AFT-B1, we isolated total RNA from Arabidopsis and reverse transcribed into a cDNA pool with adaptors on both ends (Fig. 14). The design of these two adaptors fits the site for the cloning of individual enzyme genes in our degradation module, and enables our future incorporation of each member of the cDNA pool into the same position by homologous recombination (In-Fusion) technology.
Figure 14. I need an agarose gel for the cDNA library, and the sequence of the adaptors that are good for the cloning work, and also a map showing the structure.
Figure 14. The strategy for cloning a cDNA pool into the enzyme gene cassette driven by the Gal1 promoter in pGBK-T7 for screening novel enzymes that can degrade AFT-B1. (A) A cDNA pool prepared from the model plant Arabidopsis thaliana. (B) The cloning strategy and primers used. Variable number of nucleotides were added after the start codon (ATG) to enable the translation from a correct reading frame.
Conclusion
In this project, we finished the cloning of enzyme genes for the expression in both bacterial and yeast cells, constructed the fusion genes for AD-ScFv1 and BD-ScFv2 for gene activation, developed the Gal1::EYFP for online detection without affecting yeast growing, confirmed one of the AFT-B1-degrading enzymes and established an inducible detection-degradation system that can be used for screening novel AFT-B1-degrading enzymes.
We submitted parts for Gal1::EYFP (No. BBa_K2631001) and for the ADTZ gene (No. BBa_K2631000).
Future Direction
Therefore, from our work, we have both detection and degradation modules ready in two plasmids that can be easily used for yeast applications.
First, for industry, this system provide a real-time detection and degradation of AFT-B1 that can be easily transplanted for fermentation process.
Second, for scientific research, we are able to carry out a robust screening by using this system for novel enzymes.In our system, we can directly cloned genes from a library from any organism after the inducible promoter, instead of the signal enzyme gene by homologous recombination. Thus future work will not rely on those published enzymes.
Third，such a system, actually, can be revised to screen enzymes for the degradation of other chemicals, such as environmental pollutants, a long as the fragments of corresponding antibody can be generated. We established such a strategy from our work.
Sponsors
