<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_CIEI-BJ_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:CIEI-BJ/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="backtop-wrapper"><A href="#" class="sf-back-to-top">Top</A></DIV><HEADER class="header-navigation" id="header"><DIV class="navi_body"><DIV class="navi_head"><DIV style="width:100%; margin-left:auto; margin-right:auto;"><SPAN><DIV class="navi_title"><A href="https://2018.igem.org/Team:CIEI-BJ">
								Home</A></DIV></SPAN><SPAN><DIV class="navi_title">Achievements</DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Medals">Medals</A></DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Parts">Parts</A></DIV></SPAN><SPAN><DIV class="navi_title">Project</DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Background">Background</A></DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Design">Design</A></DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Results">Results</A></DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Experiments">Experiments</A></DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Model">Modeling</A></DIV></SPAN><SPAN><DIV class="navi_title"><A href="https://2018.igem.org/Team:CIEI-BJ/InterLab">InterLab</A></DIV></SPAN><SPAN><DIV class="navi_title"><A href="https://2018.igem.org/Team:CIEI-BJ/Human_Practices">Human_Practices</A></DIV></SPAN><SPAN><DIV class="navi_title">Notebook</DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Safety">Safety</A></DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Notebook">Lab Book</A></DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Protocol">Protocol</A></DIV></SPAN><SPAN><DIV class="navi_title">Team</DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Team">Gallery</A></DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Attributions">Attributions</A></DIV><DIV><A href="https://2018.igem.org/Team:CIEI-BJ/Collaborations">Collaborations</A></DIV></SPAN></DIV></DIV></DIV></HEADER><DIV id="my-container"><DIV id="my-sidebar"><DIV class="sidebar__inner"><UL class="page-anchors"><LI><A href="#a1">Experiments </A></LI><LI><A href="#a2">Design of the experiment </A></LI><LI><A href="#a3">Construction of the vectors </A></LI><LI><A href="#a4">Transform</A></LI></UL><LI><A href="#a5">  OD-Mock    OD-AFB1</A></LI><LI><A href="#a6">Soluble expression of ADTZ in <I>E．coli</I> and the degradation of aflatoxin B1 by ADTZ</A></LI><LI><A href="#a7">construction of pMAL-c5x- ADTZ</A></LI><LI><A href="#a8">expression of MBP-ADTZ</A></LI><LI><A href="#a9">degradation of aflatoxin B1 by MBP-ADTZ</A></LI><LI><A href="#a10">Soluble expression of ADTZ in Saccharomyces cerevisiae and the degradation of aflatoxin B1 by ADTZ</A></LI><LI><A href="#a11">ADTZ-SC sequence was synthesized as following:</A></LI><LI><A href="#a12">full length of ADTZ-SC with BamHI and XbaI was cloned by PCR</A></LI><LI><A href="#a13">BamHI and XbaI digestion of ADTZ-SC and pYES2 empty plasmid</A></LI><LI><A href="#a14">transformation of competent cells in <I>Escherichia coli</I></A></LI><LI><A href="#a15">plasmid extraction</A></LI><LI><A href="#a16">transformation of ADTZ-pYES2 plasmid in yeast</A></LI><LI><A href="#a17">yeast colony pcr</A></LI><LI><A href="#a18">Yeast cell culture</A></LI><LI><A href="#a19">protein extraction from yeast</A></LI><LI><A href="#a20">degradation of aflatoxin B1 by ADTZ-pYES2</A></LI><LI><A href="#a21">in vitro pYES2-ADTZ total protein enzyme activity analysis</A></LI><LI><A href="#a22">References</A></LI></DIV></DIV><DIV id="my-adjust-content"><DIV id="adjust-padder"><DIV class="first-level" id="a1">Verification of the degradation of aflatoxin by the interaction of ScFv1 and ScFv2 which induce the expression of BacC, ADTZ, MNP, MSMEG5998 in yeast</DIV><DIV class="second-level" id="a2">The design of the experiment：</DIV><DIV class="third-level">Characterization</DIV><P class="my-content">Figure 1. The design of our eYFP reporter in Aflatoxin-sensing system</P><P class="my-content">This scheme shows the working principle of our aflatoxin (AFT) biosensor system. Two anti-AFB1 single chain variable fragments (ScFv) targeting different affinity sites of AFT were fused with Gal4 transcription activation domain (AD) and DNA binding domain (BD) respectively. In the presence of AFB1, the two anti-AFB1 ScFvs could be drawn near, enabling the association of Gal4 AD and BD, and thus driving the expression of genes under the Gal promoter. At the same time eYFP protein was also expressed under this condition.</P><DIV class="second-level" id="a3">Construction of the vectors:</DIV><TABLE><TBODY><TR><TD>pGADT7-ScFv1-pGal1-BacC</TD></TR><TR><TD>pGADT7-ScFv1-pGal1-ADTZ</TD></TR><TR><TD>pGADT7-ScFv1-pGal1-MNP</TD></TR><TR><TD>pGADT7-ScFv1-pGal1-MSMEG5998</TD></TR><TR><TD>pGBKT7-ScFv2-pGal1-EYFP</TD></TR></TBODY></TABLE><P class="my-content">The primer for pGADT7-pGal1-BacC:</P><P class="my-content">Note:</P><TABLE><TBODY><TR><TD>bacc1-1F/1R for cloning the AD transcription terminator in yeast</TD></TR><TR><TD>bacc1-2F/2R for cloning the yeast reporter gene GAL4 and the promoter GAL1</TD></TR><TR><TD>bacc1-3F/3R for cloning the protein BacC to degradate AFB1</TD></TR></TBODY></TABLE><P class="my-content">primers:</P><TABLE><TBODY><TR><TD>bacc1&gt;ScFv1AD-1F</TD><TD>TCGATACGGGATCCATCGAGGCGAATTTCTTATGATTTATGATTTTTATTATTAAATAA</TD></TR><TR><TD>bacc1&gt;ScFv1AD-1R</TD><TD>TTCTAATCCGCCGGTAGAGGTGTGGTCA</TD></TR><TR><TD>bacc1&gt;ScFv1AD-2F</TD><TD>CCTCTACCGGCGGATTAGAAGCCGCCGAG</TD></TR><TR><TD>bacc1&gt;ScFv1AD-2R</TD><TD>GTTGACCCATCTCCTTGACGTTAAAGTATAGAGGTATATTAACAATTTTTTGT</TD></TR><TR><TD>bacc1&gt;ScFv1AD-3F</TD><TD>CGTCAAGGAGATGGGTCAACAAATGATCATGAACCT</TD></TR><TR><TD>bacc1&gt;ScFv1AD-3R</TD><TD>TATCTACGATTCATCTGCAGTTACTGAGCGGTGTAACCACCGT</TD></TR></TBODY></TABLE><P class="my-content">The primer 1F/1R,2F/2R,3F/3R amplify ADTer（188bp）,the promoter GAL1（442bp） and bacc（780bp）:</P><P class="my-content">M: takara DL2000; 1 ADTer；2 GAL1；3 Bacc</P><P class="my-content">The three fragments were connected by using the multi-clone kit (Vazyme Biotech Co.Ltd), after reaction, transformed <I>E.coli</I> TOP10 competent cells.</P><P class="my-content">Screen colonies using carboxybenzyl plates and verify the colonies through PCR with the 2F/2R primer. The gene length is 442bp. Then choos positive colonies to sequencing using primer t7-pro / 3’AD, and all results are right.</P><P class="my-content">M: takara DL2000; 1,2,3, amplification of positive colonies</P><P class="my-content">The vector map:</P><P class="my-content">Design the primer ADpGAL1-F/R and BDpGAL1-F/R to amplify ADter and promoter GAL1 which will construct the downstream of ScFv1/2 in order to insert more candidate genes.</P><P class="my-content">The primer:</P><TABLE><TBODY><TR><TD>ADpGAL1-F</TD><TD>TCGATACGGGATCCATCGAGGCGAATTTCTTATGATTTATGATTTTTAT</TD></TR><TR><TD>ADpGAL1-R</TD><TD>ATCTACGATTCATCTGCAGCTCGAGCTCCTTGACGTTAAAGTATAGAG</TD></TR><TR><TD>BDpGAL1-F</TD><TD>CAAACGGTGAGAATTCCCGGGCGAATTTCTTATGATTTATGAT</TD></TR><TR><TD>BDpGAL1-R</TD><TD>GCGGCCGCTGCAGGTCGACGGATCCCTCCTTGACGTTAAAGTATAG</TD></TR><TR><TD>BDeYFP-F</TD><TD>TATACTTTAACGTCAAGGAGATGGTGAGCAAGGGCGAGGAGC</TD></TR><TR><TD>BDeYFP-R</TD><TD>GCGGCCGCTGCAGGTCGACGGATCCTTACTTGTACAGCTCGTCCAT</TD></TR><TR><TD>ADADTZ-F</TD><TD>TATACTTTAACGTCAAGGAGATGGCTACTACTACTGTTCAC</TD></TR><TR><TD>ADADTZ-R</TD><TD>ATCTACGATTCATCTGCAGCTCGAGTTACAATCTTCTTTCGATGA</TD></TR><TR><TD>ADMEG5998-F</TD><TD>TATACTTTAACGTCAAGGAGATGGCCGACACTTCCCGTCCCCTCAAC</TD></TR><TR><TD>ADMEG5998-R</TD><TD>ATCTACGATTCATCTGCAGCTCGAGTTAAGCCGGGTCGCAGATGAC</TD></TR><TR><TD>ADMNP-F</TD><TD>TATACTTTAACGTCAAGGAGATGGCTTTCAGCACCCTCC</TD></TR><TR><TD>ADMNP-R</TD><TD>ATCTACGATTCATCTGCAGCTCGAGTTACTTGTCATCGTCGTCCTTGTAATCAGC</TD></TR></TBODY></TABLE><P class="my-content">The enzyme site xhoi in ADT7-ScFv1 and BamHi in BDT7-ScFv2 are used to construct target map with ADter and GAL1.</P><P class="my-content">The vector pADT7-ScFv1-BacC as template amplify adter&amp;gal1 and deposit linear vector by alcohol.</P><P class="my-content">M: takara DL2000; 1,ADter+gal1 for pAD,2 ,ADter+gal1 for pBK</P><P class="my-content">Connect the genes using the multi-clone test kit (Vazyme Biotech Co.Ltd), and transform <I>E.coli</I> TOP10.</P><P class="my-content">Screen colonies using carboxybenzyl plates and verify the colonies through PCR with the t7-pro / 3’AD primer. The gene length is 1700bp. The sequencing results of positive clones are right.</P><P class="my-content">M: takara DL2000; 1,pAD+ter+gal1,2 ,Pbk+ter+gal1;</P><P class="my-content">ADTZ/MNP/ MSMEG5998 are cloned from AD/BK vector with promoter GAL1, then construct into AD-ScFv1, and eYFP to BK-ScFv2. The result is:</P><P class="my-content">M: takara DL2000; A, ADTZ B,MNP C,MSMEG5998 D,Eyfp</P><P class="my-content">The positive clones are sequenced for future. Vector map is:</P><DIV class="second-level" id="a4">Transform</DIV><TABLE><TBODY><TR><TD>pADT7-ScFv1;</TD></TR><TR><TD>pADT7-ScFv1-BacC;</TD></TR><TR><TD>pADT7-ScFv1-MNP;</TD></TR><TR><TD>pADT7-ScFv1-ADTZ;;</TD></TR><TR><TD>pADT7-ScFv1-MSMEG5998</TD></TR></TBODY></TABLE><P class="my-content">to yeast strain AH109 with pBKT7-ScFv2 and pBKT7-ScFv2-eYFP, separately. After transformed three days (8.24 to 8.27), we got the result of pADT7-ScFv1&amp;pBKT7-ScFv2 and pADT7-ScFv1-BacC &amp; pBKT7-ScFv2:</P><P class="my-content">After ten days (8.24</P><P class="my-content">to 9.3):</P><P class="my-content">The three missing flat with or without AFB1 all have grown yeast.</P><P class="my-content">The clones in two missing flat are suspended on 100ul 0.9%NACl solution, and inoculated into three missing culture medium with or without AFB1 in Aug 31. After three days, check the OD600 :</P><TABLE><TBODY><TR><TD>   OD-Mock  </TD><TD>   OD-AFB1</TD></TR><TR><TD>Sv1+Sv2    </TD><TD>    0.250   </TD><TD>    0.161</TD></TR><TR><TD>Sv1+Sv2    </TD><TD>    0.119   </TD><TD>    0.102</TD></TR><TR><TD>Sv1+Sv2     </TD><TD>   0.079   </TD><TD>    0.063</TD></TR><TR><TD>Sv1bacc+Sv2  </TD><TD>  0.047   </TD><TD>    0.034</TD></TR><TR><TD>Sv1bacc+Sv2  </TD><TD>  0.041   </TD><TD>    0.035</TD></TR><TR><TD>Sv1bacc+Sv2  </TD><TD>  0.249    </TD><TD>   0.119</TD></TR></TBODY></TABLE><P class="my-content">The result show that the three clones in AFB1 flat are not stranger than in flat without AFB1; on the contrary, Sv1+Sv2 clones in AFB1 flat is stranger. And in the chart, the yeast with BacC gene is bad than the control. For next, we will check more OD value in different time and take 3-AT to inhibit self-activation in the control, then continue the work. And other vectors have transformed yeast for analysis.</P><P class="my-content"><B>Fig. 2 Expression from pGal promoter can be induced in the presence of AFT.</B></P><P class="my-content">The yeast cells transformed with two plasmids were cultured in SD-His media with no Aflatoxin B1 (control) or 100 ug/L (100 ppb) AFT B1. According to our design, the presence of AFT B1 should facilitate the association of Gal4 AD and BD, thus driving the expression of His3 protein under the endogenous Gal1 promoter, and enabling yeast growth in SD-His media. The yeast growth (OD 600) was measured every 2 hour for 14 hours on end, and the growth curve clearly showed that, compared to control, the presence of AFT B1 significantly promoted the growth of the yeast. Therefore, the results shows the similar changes with AD-ScFv1 (BBa_K2247008) and BD-ScFv2 (BBa_K2247009). Besides this, the candidate genes which may degrade AFB1 shows very hard to growth in SD-His media. Whether these genes are really working, should be further confirmed by more experiments.</P><P class="my-content"><B>Fig. 3 Experimental validation of eYFP Aflatoxin-sensing system</B></P><P class="my-content">This diagram shows the experiment the eYFP is working under the AFB1 existing conditions in the yeast cells. According to our design, the presence of AFB1 should lead to the association of Gal4 AD and BD domains, and drive the expression of eYFP enabling the yeast show yellow fluorescence under excitation light.</P><P class="my-content"><B>Fig. 4  Western bolt to confirm the EYFP protein can be induced in the presence of AFT.</B></P><P class="my-content">The yeast cells were cultured in SD-His media with 100 ug/L (100 ppb), 5mg/L (5000ppb) Aflatoxin B1. And cells cultured in SD-Trp/-Leu with no AFB1 (control). According to our design, the presence of AFB1 should facilitate the association of Gal4 AD and BD, thus driving the expression of His3 protein under the endogenous Gal1 promoter, and enabling yeast growth in SD-His media. Before loading samples all proteins were normalized by Bradford method. Results showed that, compared to control, the presence of AFB1 can induce the protein content of eYFP. Therefore, we believe the two anti-AFB1 antibody single-chain variable fragments (ScFv1, BBa_K2247006, and ScFv2, BBa_K2247007), as well as the fusion proteins AD-ScFv1 (BBa_K2247008) and BD-ScFv2 (BBa_K2247009) function well as expected.</P><DIV class="first-level" id="a5">Soluble expression of ADTZ in <I>E．coli</I> and the degradation of aflatoxin B1 by ADTZ</DIV><P class="my-content">In order to express soluble ADTZ in <I>E. coli</I>，the DNA coding ADTZ was inserted into the vector pMAL-c5x to produce the fusion protein of malte-binding protein (MBP) and ADTZ.</P><DIV class="second-level" id="a6">construction of pMAL-c5x- ADTZ</DIV><P class="my-content">Primers used for pMAL-c5x- ADTZ construction are as follows.</P><TABLE><TBODY><TR><TD>MAL1: </TD><TD>   GATTGTAAGGATCCGAATTCCCTGCA</TD></TR><TR><TD>MAL4: </TD><TD>   GTAGCCATCCTTCCCTCGATCCCGA</TD></TR><TR><TD>AD2:  </TD><TD>    AGGGAAGGATGGCTACTACTACTGTTCACAG</TD></TR><TR><TD>AD3: </TD><TD>     TTCGGATCCTTACAATCTTCTTTCGATGAA</TD></TR><TR><TD>MAL：</TD><TD> TCGTCAGACTGTCGATGAAG</TD></TR><TR><TD>AD： </TD><TD>  AGCGTAGATGTGTTGCTTCA</TD></TR></TBODY></TABLE><P class="my-content">A 2.1kb fragment was obtained by PCR from synthesized ADTZ template with primer AD2 and AD3. A 5.6kb linearized vector fragment was obtained by PCR from pMAL-c5x template with primer MAL1 and MAL4 (Fig.5). Then In-Fusion reaction for the two fragment was carried out. The colony was identified by PCR with primer MAL and AD, and the product of PCR was about 560bp(Fig.6). The positive colonies were sequenced further.</P><P class="my-content">Fig.5 PCR fragment for pMAL-c5x- ADTZ construction</P><P class="my-content">1: ADTZ fragment; 2: linearized pMAL-c5x fragment; M: λ-EcoT14I digest Marker</P><P class="my-content">Fig.6 colony PCR for pMAL-c5x- ADTZ</P><P class="my-content">M: DL2000 DNA Marker； 1, 2, 3, 4: PCR results of different colonies</P><DIV class="second-level" id="a7">expression of MBP-ADTZ</DIV><P class="my-content">Firstly the vector pMAL-c5x-ADTZ was transformed into <I>E.coli</I> BL21 Gold DE(3). Then an overnight culture of the bacteria was inoculated into LB broth containing 2g/L glucose and 100mg/L ampicillin. After 2h culture at 37 oC and 200rpm, recombinant protein production was induced by addition of 0.3 mM IPTG at 20oC and 200rpm for 24h. The cell was collected by centrifugation. The pellet was resuspended in H2O about an OD600 of 30. Then the cell was disrupted by sonication. The results showed that most of highly expressed MBP-ADTZ(119kDa) was soluble(Fig. 7).</P><P class="my-content">Fig.7  SDS-PAGE of expressed MBP-ADTZ fusion protein in <I>E.coli</I></P><P class="my-content">1: total protein before IPTG induction; 2: total protein after IPTG induction; 3: supernatant after sonication; 4: deposition after sonication; M: PAGE-MASTER Protein Standard plus</P><DIV class="second-level" id="a8">degradation of aflatoxin B1 by MBP-ADTZ</DIV><P class="my-content">The degradation reaction of aflatoxin B1 was performed in a final volume of 700 ul composed of 150ul H2O, 350 ul buffer(100mM Na2HPO4, 50mM citric acid, 0.4mg/L aflatoxin B1, pH6.0) and 200ul crude MBP-ADTZ obtained after sonication. The mixture was incubated in the dark at 30 oC without shaking for 0, 3, 6, and 12 h. After incubation, aflatoxin B1 was extracted three times with 700ul chloroform. After the chloroform was evaporated under nitrogen gas, the samples were dissolved in 140ul acetonitrile and analyzed by HPLC. HPLC analysis was performed using Diamonsil C18 column (250×4.6 mm). The mobile phase was acetonitrile/water (45:55, v/v) at a flow rate of 1 ml/min and the sample temperature was 28oC. The detection wavelength was 360 nm. The result showed that MBP-ADTZ was able to degrade AFT B1 obviously(Fig.8).</P><P class="my-content">Fig.8 HPLC analysis of AFB1 degraded by MBP-ADTZ</P><DIV class="first-level" id="a9">Soluble expression of ADTZ in Saccharomyces cerevisiae and the degradation of aflatoxin B1 by ADTZ</DIV><DIV class="second-level" id="a10">ADTZ-SC sequence was synthesized as following:</DIV><P class="my-content">&gt; ADTZ-SC-XhoI+SphI 5’XbaI 3’SpeI</P><P class="my-content">CCGCTCGAGCGGTGCTCTAGAATGGCTACTACTACTGTTCACAGAGAAAGATTCTTGGCTGACAAGTCTGCTCCATTGTGTGGTATGGACATCAGAAAGTCTTTCGACCAATTGTCTTCTAAGGAAAAGTTGTACACTCACTACGTTACTGAAGCTTCTTGGGCTGGTGCTAGAATCATCCAAGCTCAATGGACTCCACAAGCTACTGACTTGTACGACTTGTTGATCTTGACTTTCTCTGTTAACGGTAAGTTGGCTGACTTGAACGCTTTGAAGACTTCTTCTGGTTTGTCTGAAGACGACTGGGAAGCTTTGATCCAATACACTGTTCAAGTTTTGTCTAACTTGGTTAACTACAAGACTTTCGGTTTCACTAAGATCATCCCAAGAGTTGACGCTGAAAAGTTCGAATCTGTTGTTAAGGCTTCTTCTAACGCTGACCAAGGTTCTGCTTTGTTCACTAAGTTGAAGCAACACATCTACGCTTTGTCTCCAGAATCTGCTTTGTTCATCGGTAAGAGAAAGGACGGTCACGTTTCTAACTACTACTTGGGTGAACCAGTTGGTGACGCTGAAGTTGACGCTATCCAAAACGTTGCTGAAAAGTTGGGTGTTGACATCTTGAACACTAGAGTTAAGAAGAACGGTGCTGGTGACTACACTTTGTTGGTTGCTTCTGCTAAGACTTCTCCACCATCTGTTCACGACTTCCAAATCGACTCTACTCCAGCTAAGTTGACTATCGAATACGGTGACTACGCTTCTTCTTTGACTAAGGTTGTTGCTGCTTTGCAAGAAGCTAAGCAATACACTGCTAACGACCACCAATCTGCTATGATCGAAGGTTACGTTAAGTCTTTCAACTCTGGTTCTATCCCAGAACACAAGGCTGCTTCTACTGAATGGGTTAAGGACATCGGTCCAGTTGTTGAATCTTACATCGGTTTCGTTGAAACTTACGTTGACCCATACGGTGGTAGAGCTGAATGGGAAGGTTTCACTGCTATCGTTGACAAGCAATTGTCTGCTAAGTACGAAGCTTTGGTTAACGGTGCTCCAAAGTTGATCAAGTCTTTGCCATGGGGTACTGACTTCGAAGTTGACGTTTTCAGAAAGCCAGACTTCACTGCTTTGGAAGTTGTTTCTTTCGCTACTGGTGGTATCCCAGCTGGTATCAACATCCCAAACTACTACGAAGTTAGAGAATCTACTGGTTTCAAGAACGTTTCTTTGGCTAACATCTTGGCTGCTAAGGTTCCAAACGAAGAATTGACTTTCATCCACCCAGACGACGTTGAATTGTACAACGCTTGGGACTCTCGTGCTTTCGAATTGCAAGTTGCTAACCACGAATTGTTGGGTCACGGTTCTGGTAAGTTGTTCCAAGAAGGTGCTGACGGTAAGTTGAACTTCGACCCAGAAAAGGTTATCAACCCATTGACTGGTAAGCCAATCACTTCTTGGTACAAGCCAGGTCAAACTCCAGACTCTGTTTTGGGTGAAGTTTCTTCTTCTATGGAAGAATGTAGAGCTGAAACTGTTGCTTTGTACTTGGTTTCTAACTTGGACATCTTGAAGATCTTCAACTACGTTGACAAGCAAGACATCGAAGACATCCAATACATCACTTTCTTGTTGATGGCTAGAGCTGGTTTGAGAGCACTAGAGTTCTACGACCCAGCTACTAAGAAGCACGGTCAAGCTCACATGCAAGCTAGAATGGGTATCACTCAATACTTGATCCAAGCTGGTATCGCTAGATTGGAATTGATCCAAGACGCTAACGGTGAATTGGAAAACTTGTACGTTAGAGTTGACAGAGAAAAGGTTTTGTCTAAGGGTAAGGAAGTTGTTGGTCAATTGTTGATCGAATTGCAAGTTAGAAAGTCTACTGCTGACGGTACTGGTTCTCGTGACTTCTACACTACTTTGACTGAACCAATCTCTGGTTGGGAAGGTAAGATCAGAGACATCGTTTTGAAGAAGAAGTTGCCAAGAAAGATCTTCGTTCAACCAAACACTTTCGTTGTTAACGGTGAAGTTCAATTGAAGGAATACCCATTGACTGCTGCTGGTGTTATCGAATCTTTCATCGAAAGAAGATTGTAAACTAGTCCGACATGCATGCATGT</P><P class="my-content">The synthesized gene was integrated into vector pUC57 through EcoRV restriction sites, the vector map is shown as below:</P><P class="my-content">Fig.9 synthesized ADTZ in pUC57</P><DIV class="second-level" id="a11">full length of ADTZ-SC with BamHI and XbaI was cloned by PCR</DIV><P class="my-content">The gene was added with a 5’ BamHI site and a 3’ XbaI site through PCR, using the integrated plasmid as template and NEB Q5 as polymerase for amplification, and then the PCR product was purified. The purified PCR product is shown below (DNA marker: D2000)</P><P class="my-content">Fig.10 PCR fragment of ADTZ</P><P class="my-content">1: ADTZ fragment; M: D2000 Marker</P><DIV class="second-level" id="a12">BamHI and XbaI digestion of ADTZ-SC and pYES2 empty plasmid</DIV><P class="my-content">BamHI and XbaI was used to cut the PCR product (described above) and vector pYES2. The vector map of pYES2 is shown below:</P><P class="my-content">Fig.11 pYES2 vector</P><P class="my-content">The restriction enzyme digestion reactionconsists of:</P><TABLE><TBODY><TR><TD>10×Kbuffer(Takara)</TD><TD>    2.5 μL</TD></TR><TR><TD>XbaⅠ</TD><TD>    2 μL</TD></TR><TR><TD>BamHⅠ       </TD><TD>2 μL</TD></TR><TR><TD>plasmid     </TD><TD>30 μL</TD></TR><TR><TD>ddH2O       </TD><TD>13.5 μL</TD></TR><TR><TD>Total       </TD><TD>50 μL</TD></TR></TBODY></TABLE><P class="my-content">Enzyme digestion for 6 hours at 37℃</P><P class="my-content">Gel purification of BamHI and XbaI digested vector pYES2 and ADTZ-SC gene, the purified product after electrophoresis is shown below:</P><P class="my-content">Fig.12 BamHI and XbaI fragment of ADTZ and pYES2</P><P class="my-content">1: pYES2 fragment; 2: ADTZ fragment; M: D2000 Marker</P><P class="my-content">##ligation reaction</P><P class="my-content">The digested product of ADTZ-SC was integrated into vector pYES2, with the ligation system as below:</P><TABLE><TBODY><TR><TD>purified product of pYES2 digested by BamHI and XbaI    </TD><TD>    1 μL</TD></TR><TR><TD>purified product of ADTZ-SC digested by BamHI and XbaI   </TD><TD>   5 μL</TD></TR><TR><TD>T4 DNA ligase                                     </TD><TD>     1 μL</TD></TR><TR><TD>T4 DNA ligase buffer                            </TD><TD>       1 μL</TD></TR><TR><TD>H2O                                         </TD><TD>        2 μL</TD></TR></TBODY></TABLE><P class="my-content">Ligation at 4℃ overnight.</P><DIV class="second-level" id="a13">transformation of competent cells in Escherichia coli</DIV><P class="my-content">The ligation product was used to transform competent cells of <I>E. coli</I>, and the steps are:</P><P class="my-content">1.Thaw the competent <I>E. coli</I> cells on ice, add in the ligation product and mix by gently flick, then incubate on ice for 30 min;</P><P class="my-content">2.Heat shock at 42℃ for 90 s, and then immediately transfer to ice for another 2 min.</P><P class="my-content">3.Add 800 μL SOC liquid media, incubate at 37℃, 180 rpm for 1 h;</P><P class="my-content">4.Plate the transformed cells to LB agar with 100 mg/mL Amp, incubate at 37℃ overnight.</P><P class="my-content">Pick single colony and check by colony PCR, isolate the plasmid from positive colony. The colony PCR result is shown below:</P><P class="my-content">Fig.13 <I>E.coli</I> colony pcr</P><P class="my-content">1: negative control; 2-6: positive clones; M: D2000 Marker</P><DIV class="second-level" id="a14">plasmid extraction</DIV><P class="my-content">GenStar plasmid miniprep isolation kit was used, and the protocol is:</P><P class="my-content">1.Collect overnight liquid culture in 2mL tube, centrifuge at 12,000 rpm for 1min at room temperature, discard the supernatant.</P><P class="my-content">2.Add 250 μL RNaseA containing suspension buffter (S1) towell resuspend cells.</P><P class="my-content">3.Add 250 μL cell lysis buffer (S2), mix by inverting the tube 5 times, settle for 1-5 min at room temperature to fully lyse the cell.</P><P class="my-content">4.Add 350 μL neutralization buffer (S3), mix by inverting the tube 5 times, centrifuge at 12,000 rpm for 10 min at room temperature.</P><P class="my-content">5.Carefully transfer the supernatant to a column in a collection tube, centrifuge at 12,000 rpm for 1min at room temperature, discard the flow through in collection tube, put the column back into the collection tube.</P><P class="my-content">6.Add 500 μL wash buffter, centrifuge at 12,000 rpm for 1min at room temperature, discard the flow through in collection tube.</P><P class="my-content">7.Repeat the above step.</P><P class="my-content">8.Centrifuge at 12,000 rpm for 2 min at room temperature to completely remove wash buffer.</P><P class="my-content">9.  Carefully transfer the column into a new 1.5mL tube, add 50μL elution buffer in the center of silica-membrane, settle for 2 min at room temperature, and thencentrifuge at 12,000 rpm for 2 min to collect the plasmid DNA, then check by electrophoresis. Sequence the plasmid isolated from positive colony, confirm it is correct before applying to the following experiments. Keep the strain of correct ADTZ-SC-pYES2.</P><DIV class="second-level" id="a15">transformation of ADTZ-pYES2 plasmid in yeast</DIV><P class="my-content">ADTZ-SC-pYES2 plasmid was used to transform yeast strain INVSc1 (the empty vector pYES2 was also used as a control), the protocol is:</P><P class="my-content">1.Streak out the INVSc1 yeast on YPD agar, culture for 2-3 days at 30℃.</P><P class="my-content">2.Pick single colony and incubate in YPD liquid media at 30℃, 200 rpm for overnight.</P><P class="my-content">3.Check OD600 of the overnight cell culture, use 50mL YPD liquid media to dilute the cell culture until the OD reaches 0.4, then incubateat 30℃, 200 rpm for 2-4h;</P><P class="my-content">4.Transfer the cell culture to 50mL tubes, centrifuge at 2,500 g for 10 min, collect the pellet and discard the supernatant, resuspend the cell with 40 mL 1× TE buffer.</P><P class="my-content">5.Centrifuge at 2,500 g for 10 min, collect the pellet and discard the supernatant, resuspend the cell with 2 mL1× LiAc/0.5× TE buffer.</P><P class="my-content">6.Settle for 10 min at room temperature.</P><P class="my-content">7.Take 100 μl yeast competent cell obtained from above, add 1 μg target gene containing pYES2 plasmid, and 100 μg denatured salmon sperm DNA, mixwell.</P><P class="my-content">8.Add 700 μl 1× LiAc/40% PEG-3350/1× TE buffer, mix well;</P><P class="my-content">9.Incubate at 30℃ for 30 min;</P><P class="my-content">10.Add 88 μl DMSO, mix well, then heat shock at 42℃ for 7 min;</P><P class="my-content">11.Centrifuge at 2,500 g for 10 sec, discard the supernatant, resuspend cells with 1 mL1× TE buffer.</P><P class="my-content">12.Centrifuge at 2,500 g for 10 sec, resuspend cells with 50-100 μl 1× TE, then plate onto Ura- media (SC-U) with 5 mg/mL Tween 80, incubate for 2-3 days at 30℃.</P><DIV class="second-level" id="a16">yeast colony pcr</DIV><P class="my-content">Recombinant yeast colony PCR, the protocol is:</P><P class="my-content">1.  Pick single colony to PCR tubes with 10 μL NaOH (0.02 M), resuspend the cells.</P><P class="my-content">2.  Incubate the tubes in PCR machine at 99℃ for 10 min, then transfer to ice;</P><P class="my-content">3.  Perform colony PCR as below:</P><TABLE><TBODY><TR><TD>10×Ex Taq buffer (Takara)   </TD><TD>1 μL</TD></TR><TR><TD>forward primer(10μΜ)        </TD><TD>2 μL</TD></TR><TR><TD>reverse primer(10μΜ)</TD><TD>    2 μL</TD></TR><TR><TD>Betaine (5 M，sigma)        </TD><TD>0.6 μL</TD></TR><TR><TD>dNTP (2.5 μΜ)       </TD><TD>1 μL</TD></TR><TR><TD>ExTaq       </TD><TD>0.1 μL</TD></TR><TR><TD>liquid culture      </TD><TD>1 μL</TD></TR><TR><TD>single colony solution      </TD></TR><TR><TD>ddH2O       </TD><TD>2.3 μL</TD></TR><TR><TD>Total       </TD><TD>10 μL</TD></TR></TBODY></TABLE><P class="my-content">The PCR program is: denaturingat 94℃for 5 min；94℃ 30 s，65℃ 30 s，72℃ 30 s，for a total 35 cycles；72℃ 10 min. Then examine by electrophore/p&gt;
</P><P class="my-content">Note: Yeast colony PCR has a primer concentration 10 times more than regular PCR, the PCR product should be 300-500 bp for best results.</P><P class="my-content">Yeast colony PCR result is shown below (first lane is negative control; all colonies were positive):</P><P class="my-content">Fig.14 yeast colony pcr</P><P class="my-content">1: negative control; 2-6: positive clones; M: D2000 Marker</P><DIV class="second-level" id="a17">Yeast cell culture</DIV><P class="my-content">Yeast cell culture, the protocol is:</P><P class="my-content">1.Pick single colony to 20 mL YPD liquid media with 5 mg/mL Tween 80（T）, incubate at 30℃，200 rpm for 2 days.</P><P class="my-content">2.Centrifuge at 3,000 rpm for 5 min, discard the supernatant, resuspend in non-glucose SC-U media (SC-U+T+2% galactose).</P><P class="my-content">3. Centrifuge at 3,000 rpm for 5 min, discard the supernatant, resuspend in 0.1M potassium phosphate buffer (3% glucose, 5 mg/mL Tween 80，p 7.0), incubate at 30℃，200 rpm for 24 h.</P><DIV class="second-level" id="a18">protein extraction from yeast</DIV><P class="my-content">Extract proteins from the yeast, SDS gel electrophoresis didn’t reveal band of the target protein. The result was not saved.</P><DIV class="second-level" id="a19">degradation of aflatoxin B1 by ADTZ-pYES2</DIV><P class="my-content">ADTZ-SC gene was inserted into vector pYES2 and transformed into yeast strain INVSc1. In order to examine if yeast can degrade aflatoxin, this chemical was added into the media when protein expression was induced. The protocol is:</P><P class="my-content">1、Incubate yeast, transformed with either pYES2-empty vector or pYES2-SC, in SC-U+Glc media for 2 days.</P><P class="my-content">2、Incubate in SC-U+Glc+Aflatoxin (5mL) for 24 h, different amount of aflatoxin was used as:</P><TABLE><TBODY><TR><TD>pYES2 no aflatoxin</TD></TR><TR><TD>pYES2+0.05 mg/L Aflatoxin（5 μL50 mg/L Aflatoxin）</TD></TR><TR><TD>pYES2+0.5 mg/L Aflatoxin（50μL50 mg/L Aflatoxin）</TD></TR><TR><TD>pYES2+2 mg/L Aflatoxin（200μL50 mg/L Aflatoxin）</TD></TR><TR><TD>pYES2+5 mg/L Aflatoxin（500μL50 mg/L Aflatoxin）</TD></TR><TR><TD>pYES2-SCno aflatoxin</TD></TR><TR><TD>pYES2-SC+0.05 mg/L Aflatoxin（5 μL 50 mg/L Aflatoxin）</TD></TR><TR><TD>pYES2-SC +0.5 mg/L Aflatoxin（50μL 50 mg/L Aflatoxin）</TD></TR><TR><TD>pYES2-SC +2 mg/L Aflatoxin（200μL 50 mg/L Aflatoxin）</TD></TR><TR><TD>pYES2-SC +5 mg/L Aflatoxin（500μL 50 mg/L Aflatoxin）</TD></TR></TBODY></TABLE><P class="my-content">Note: Aflatoxin is dissolved in ethanol, in case the yeast growth could be suppressedby ethanol, evaporate half amount of the ethanol then refill with same amount of water.</P><P class="my-content">3、24 hours after induced by galactose, 1 mL cell culture was taken and mixed with 1 mL chloroform to isolate aflatoxin, repeat isolation fo 3 times. After drying up by nitrogen, samples were dissolved in 100 μL methanol and apply 15 μL for HPLC analysis. The result shows chromatographic peaks of aflatoxin can be detected after aflatoxin, with a final concentration of 2 mg/L and 5 mg/L, was added to yeast transformed with both pYES-empty vector and pYES2-ADTZ. However, yeast transformed with pYES2-ADTZ has no significant difference in the aflatoxin peak area comparing to yeast transformed with empty pYES2 vector.</P><P class="my-content">Fig15 HPLC results of degradation of aflatoxin B1 by pYES2-ADTZ</P><P class="my-content">From the top: 1. Standard Aflatoxin; 2. pYES2 yeast without aflatoxin; 3. pYES2 yeast with 2mg/L aflatoxin; 4. pYES2 yeast with 5mg/L aflatoxin; 5. pYES2-ADTZ yeast without aflatoxin;6. pYES2-ADTZ yeast with 2mg/L aflatoxin; 7. pYES2-ADTZ yeast with 5mg/L aflatoxin;</P><DIV class="second-level" id="a20">in vitro pYES2-ADTZ total protein enzyme activity analysis</DIV><P class="my-content">50mL ADTZ yeast (use yeast transformed with empty vector as control) was induced, the induction is described in (9). Cell was collected by centrifuge and resuspended in protein isolation buffer (50 mM sodium phosphate, pH 7.4，1 mM PMSF，1 mM EDTA，5% glycerol). Equal volume of acid-washed glass beads  added and vortexed for 30s, then incubate on ice for 30s, repeat for 7 times. Then samples were centrifuged at 4℃ for 5-10 min, 10000 g, and the supernatant was transferred to a clean tube.</P><P class="my-content">The in vitro enzyme activity was analyzed in 500uL system with a final concentration of aflatoxin at either 0, 0.05, 0.5, or 5 mg/L. 50uL total yeast protein was added, and the reaction buffer was 0.2M disodium hydrogen phosphate-0.1M citric acid, pH 6.0, incubate at 30℃ for 24 hours. Chloroform was used for isolation, and after nitrogen air drying and dissolved in methanol, samples were examined by LC-MS. The result is shown below, yeast transformed with pYES2-ADTZ has no significant difference in the aflatoxin peak area comparing to yeast transformed with empty pYES2 vector.</P><P class="my-content">Fig.16 LC-MS results of degradation of aflatoxin B1 by pYES2-ADTZ</P><P class="my-content">From the top: 1. Standard Aflatoxin; 2. pYES2 yeast without aflatoxin; 3. pYES2 yeast with 0.05mg/L aflatoxin; 4. pYES2 yeast with 0.5mg/L aflatoxin; 5. pYES2 yeast with 5mg/L aflatoxin; 6. pYES2-ADTZ yeast without aflatoxin;7. pYES2-ADTZ yeast with 0.05mg/L aflatoxin; 8. pYES2-ADTZ yeast with 0.5mg/L aflatoxin; 9.pYES2-ADTZ yeast with 5mg/L aflatoxin.</P><DIV class="first-level" id="a22">Reference</DIV><P class="my-content">1.Sang-Woo Lee, Min-Kyu Oh. “A synthetic suicide riboswitch for the high-throughput screening of metabolite production in Saccharomyces cerevisiae.” <I>Metabolic Engineering</I>, Vol28 (2015): 143–150</P><P class="my-content">2.Mohsen Farzaneh, Zhi-Qi Shi, Alireza, Narges Sedaghat, Masoud Ahmadzadeh, Mansoureh Mirabolfathy, Mohammad Javan-Nikkhah. “Aflatoxin B1 degradation by Bacillus subtilis UTBSP1 isolated from pistachio nuts of Iran.” <I>Food Control</I>, Vol 23 (2012): 100-106</P><P class="my-content">3.Hamideh Afsharmanesh, Alejandro Perez Garcia, Houda Zeriouh, Masoud Ahmadzadeh, Diego Romero. &quot;Aflatoxin degradation by Bacillus subtilis UTB1 is based on production of an oxidoreductase involved in bacilysin biosynthesis.&quot; <I>Food Control</I>, Vol 94(2018): 48-55</P><P class="my-content">4.Floudas D, Binder M, Riley R, et al. “The Paleozoic origin of enzymatic lignin decomposition reconstructed from 31 fungal genomes.” <I>Science</I>, Vol 336(2012): 1715-1719</P><P class="my-content">5.R. D. Smiley and F. A. Draughon. “Preliminary Evidence that Degradation of Aflatoxin B1 by Flavobacterium aurantiacum is Enzymatic.” <I>Journal of Food Protection</I>, Vol 63, No. 3(2000): 415–418</P><P class="my-content">6.O. A. Adebo, P. B. Njobeh, S. Gbashi, O. C. Nwinyi &amp; V. Mavumengwana. “Review on microbial degradation of aflatoxins.” <I>Critical Reviews in Food Science and Nutrition</I>, Vol 57, No. 15(2017): 3208-3217</P><P class="my-content">7.Yan Ma, Donglian Zhang, Xiaoqin Su, Shuangmei Duan, Ming Zhao. “Overview of research on mycotoxin contamination in tea.” <I>Chinese Journal of Food Hygiene</I>, Vol 26, No 6 (2014): 627- 631</P><P class="my-content">8.CHEN Jian-ling, LI Wen-xue, YANG Guang-yu, ZHOU Zhi-tao, CHEN Wen, ZHU Wei, LIU Hua-zhang. “Bioligical contamination of Pu'er tea in a Guangzhou tea market.” <I>CARCINOGENESIS, TERATOGENESIS &amp; MUTAGENESIS</I>, Vol 23, No 1 (2011): 68-71</P><P class="my-content">9.Hamideh Afsharmanesh, Alejandro Perez-Garcia, Houda Zeriouh, Masoud Ahmadzadeh, Diego Romero. “Aflatoxin degradation by Bacillus subtilis UTB1 is based on production of an oxidoreductase involved in bacilysin biosynthesis.” <I>Food Control</I>, Vol 94(2018): 48-55</P><P class="my-content">10.Wengui Li, Kunlong Xu, Rong Xiao, Gefen Yin &amp; Wenwen Liu (2015) “Development of an HPLC-Based Method for the Detection of Aflatoxins in Pu-erh Tea.” <I>International Journal of Food Properties</I>, 18:4, 842-848, https://doi.org/10.1080/10942912.2014.885043</P><P class="my-content">11.Martina Loi, Francesca Fanelli, Maria Teresa Cimmarusti, Valentina Mirabelli, Miriam Haidukowski, Antonio F. Logrieco, Rocco Caliandro, Giuseppina Mule. “In vitro single and combined mycotoxins degradation by Ery4 laccase from Pleurotus eryngii and redox mediators.” <I>Food Control</I>, Vol 90 (2018): 401-406</P><P class="my-content">12.L.H. Zhao, S. Guan, X. Gao, Q.G. Ma, Y.P. Lei, X.M. Bai and C. Ji. “Preparation, purification and characteristics of an aflatoxin degradation enzyme from Myxococcus fulvus ANSM068.” <I>Journal of Applied Microbiology</I>, 110, 147–155. 2010.</P><P class="my-content">13.Marisa Motomura, Tetsuo Toyomasu, Keiko Mizuno, Takao Shinozawa. “Purification and characterization of an aflatoxin degradation enzyme from Pleurotus ostreatus.” <I>Microbiological Research</I>, (2003) 158, 237–242</P><P class="my-content">14.A.J. De Lucca, C.H. Carter-Wientjes, S. Boue, and D. Bhatnagar. “Volatile Trans-2-Hexenal, a Soybean Aldehyde, Inhibits Aspergillus flavus Growth and Aflatoxin Production in Corn.” <I>Journal of Food Science</I>, Vol. 76, No 6(2011): 381-386.</P></DIV></DIV></DIV><FOOTER><DIV class="sponsor"><DIV class="sponsor-txt">Sponsors</DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>