<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ColumbiaNYC_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ColumbiaNYC/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-id" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" data-hover="dropdown" data-animations="pulse" id="navbar-id"><UL class="nav navbar-nav"><LI id="navbar-brand"><A href="https://2017.igem.org/Team:ColumbiaNYC" style="text-decoration: none; background-color:transparent !important; color:#fff;">
										COLUMBIA UNIVERSITY
									</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:ColumbiaNYC/Team">TEAM</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Team">ABOUT US</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Attributions">ATTRIBUTIONS</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:ColumbiaNYC/Description">PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Description">DESCRIPTION</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Design">PROOF OF CONCEPT</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Results">RESULTS</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/HP/Silver">SAFETY</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/InterLab">INTERLAB</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Applied_Design">APPLIED DESIGN</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Improve">FUTURE WORK</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:ColumbiaNYC/Part_Collection">PARTS</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Basic_Part">BASIC PARTS</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Composite_Part">COMPOSITE PARTS</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Part_Collection">PARTS COLLECTION</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:ColumbiaNYC/Model">MODELLING</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:ColumbiaNYC/Notebook">NOTEBOOK</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Brainstorm">BRAINSTORM</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Notebook">LAB NOTEBOOK</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:ColumbiaNYC/Engagement">HUMAN PRACTICES</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/HP/Gold_Integrated">HUMAN PRACTICES</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Engagement">ENGAGEMENT</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Entrepreneurship">ENTREPRENEURSHIP</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Collaborations">COLLABORATIONS</A></LI><LI><A href="https://2017.igem.org/Team:ColumbiaNYC/Medals">MEDAL CRITERION</A></LI></DIV></DIV><DIV class="jumbotron"><DIV class="container"><H1>Results</H1><P>Assessment of shRNA knockdown</P></DIV></DIV><DIV class="container"><H3><STRONG> shRNA Design </STRONG></H3><P> We chose an shRNA sequenced published on Addgene to knockdown our eGFP, and we compared the shRNA sequence to the eGFP
      DNA sequence to make sure the knockdown will theoretically occur. To construct the plasmid, we assembled the shRNA-encoding
      DNA from two segments that made up the sequence because the natural hairpin structure in the shRNA made the full sequence
      difficult to synthesize in one piece. Because our project design relies on the shRNA to inhibit gene expression (ultimately
      oncogene expression) and shRNA is not extremely stable (but not unstable), the bacteria must produce large quantities
      of shRNA for this gene therapy approach to be effective. Thus, we chose to put the gene in a pUC plasmid, which is
      a very high-copy plasmid (500-600 copies per cell). To increase the production of shRNA further, we placed the shRNA
      production under the control of a T7 promoter, which causes very strong gene expression compared to other promoters.
      Our final shRNA plasmid design is the following: </P><H6><STRONG>Figure 1: </STRONG> The SilenshR plasmid. This is an IPTG-inducible circuit with a T7 promoter that produces an shRNA
      that knocks down commercialized eGFP. </H6><H3> Transformation of Plasmid into BL21 (DE3)
      <EM> E.coli </EM></H3><P> We transformed this plasmid into BL21 (DE3) E. coli cells and got three colonies. To confirm that cloning was successful,
      we first performed colony PCR to determine if the insert was successfully cloned into the backbone. As can be seen,
      the colony PCR results show that the insert was successfully cloned in each of the three colonies: </P><H6><STRONG>Figure 2: </STRONG> Lane 1 is the Quick-LoadÂ® 2-Log DNA Ladder(0.1-10.0 kb) from NEB, lane 2 is the negative control,
      and lane 3 through 5 are from the three transformed colonies. The bands in lanes 3 through 5 are about 500bp, which
      is what we expect, and they are different in size from the negative control. Thus, this shows that cloning was successful.
    </H6><H3> Sequencing our Results</H3><P>We then performed Sanger sequencing on each colony to ensure that the shRNA sequence in the plasmid is correct, and analysis
      of the results showed that it is correct.
    </P><H6><STRONG>Figure 3: </STRONG> Sanger sequencing results for the insert for the 2 colonies</H6><H3> GFP Expression Results</H3><P>We then produced the shRNA in vitro using the New England Biolabs T7 expression kit to use in the lipofectamine assay.
      After the in vitro shRNA is successful, we reran the lipofectamine assay with shRNA that was produced in vivo using
      IPTG induction. The shRNA produced in vivo was isolated using the miRNeasy kit. We extracted a significant quantity
      of shRNA using both in vitro expression and in vivo expression. In in vitro expression, the concentration of shRNA
      was about 2230 ng/uL. In in vivo expression, concentrations ranged from 850 ng/uL to 1070 ng/uL. The lipofectamine
      assay would test whether the shRNA successfully knocks down the eGFP in mammalian cells. In this assay, the shRNA would
      be introduced into liposomes, and then the liposomes would be introduced into the mammalian cell line. The liposomes
      would fuse with the cell membrane of the mammalian cells and then the shRNA would enter the cells. Upon entry, the
      shRNA would knock down the expression of eGFP. The effectiveness of the knockdown of eGFP from the shRNA is measured
      using a flow cytometer. The flow cytometer measures the GFP florescence in every cell and gives a statistical distribution
      of the florescence. The results from the synthesized shRNA is compared to a negative control, where none of the shRNA
      is introduced to the mammalian cells during the lipofectamine assay. The results will also be compared to a positive
      control, where we used siRNA designed and proven to knock down eGFP from Thermo Fisher. We analyzed the decrease in
      florescence to determine the effectiveness of the shRNA-mediated knockdown of GFP. (For shRNA synthesized to knockdown
      oncogenes, we plan to use RT-qPCR to detect knockdown of the target gene. For the RT-qPCR, we will use the reverse
      transcriptase to transcribe the mRNA produced by the bacteria into cDNA. Then in qPCR, we can determine the amount
      of mRNA remaining in the mammalian cells from the rate at which qPCR proceeds.)
    </P><H6><STRONG>Figure 4: </STRONG> The x-axis shows the negative control (PBS), the positive controls (siRNA), and the experimental
      shRNA designed to inhibit eGFP (shRNA). The y-axis shows the percentage of mammalian cells that were fluorescing eGFP
      (defined as eGFP positive). The error bars are 2 standard error, or a 95% confidence interval. As can be seen, the
      confidence intervals for the negative control and the experimental shRNA do not overlap, which shows that this difference
      is statistically significant. This indicates that our shRNA successfully knocks down eGFP expression in HeLa cells.
      Hard data used to generate this graph can be found
      <A href="https://docs.google.com/a/columbia.edu/spreadsheets/d/18TKvTeAUxyy6XsE-lLb64fjqCos5Lhl2_f7p8wQLM4I/edit?usp=sharing">here</A>.
    </H6><P>These results show that our proof-of-concept works successfully, where shRNA successfully knocks down expression of the
      targeted gene.
    </P></DIV><DIV class="footer-right"><P> Email: cuigem2017team@googlegroups.com </P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>