"
Page
Discussion
View source
History
teams
Log in
 
Team:LA Biohackers/Project
From 2014.igem.org
Click here  to edit this page!
Home
Team
Official Team Profile
Project
Parts
Safety
Practice and Policy
Goal
Utility
Create a platform for assembling and “booting up” a genome.
Some microbial species like E coli and B. subtilis are easy to edit – add, delete, rearrange genes.  Most are not, either because they don’t take up DNA from the environment willingly or they grow so slow that even simply maintaining them in vitro is difficult.
1. We currently have the methods to assemble large fragments of chemically synthesized DNA but no method for turning a piece of DNA into a functioning cell, aside from the method performed by the Venter Institute  which only works on the smallest of bacterial chromosomes.  With our strategy one could build the synthetic chromosome piecewise inside of Bacillus subtilis as part of the B subtilis chromosome.  When the synthetic portion of the chromosome is complete it is segregated from the B subtilis chromosome and is now contained within a functioning cytoplasm without the need for transferring it to a competent recipient cell.  This would allow for the construction of large chromosomes using modular parts or chemically synthesized gBlocks or larger assemblies.
2. Modifying or engineering a select few species of bacteria (E. coli, B. subtilis and a few others) is relatively simple and well-documented.  But most species cannot be easily modified either due to the lack of transformation tools such as recombination plasmids or due to the complexities of their growth requirements.  For example, the human pathogen Mycoplasm genetalium is very against transformation and even when transformation is possible, it grows very slowly so the time in between successive transformations is at least a week.  If the Mycoplasm chromosome were to be hybridized within B subtilis, a scientist studying Mycoplasm could maintain a stock of these hybrid spores.  They could rapidly make genetic changes to the Mycoplasm chromosome with B subtilis’s simple recombination protocols, then “boot” the Mycoplasm genome when ready.
Project Description
The Plan
We are using Bacillus subtilis as a backbone to incorporate the entire         genome of Streptococcus thermophilus. Once incorporated, we will use the     cre/lox recombination system to remove the Bacillus subtilis genome so     only the Streptococcus thermophilus genome remains. This will demonstrate    the use of Bacillus subtilis as a useful chassis to boot up an artificial     genome.
As you may have guessed from the sly shift in tense from present to future, we    did not actually succeed in transferring the entire genome of Streptococcus thermophilus into Bacillus subtilis -- yet. We have developed, and will describe, a method which     we believe will enable us to do so, and the results we have obtained for the early     steps in that method have been encouraging. We still believe it may be possible to    accomplish our original goal, and will continue to pursue it in the weeks and months    to come.
And so it begins...
Strategy
Technical Challenges
For an initial proof of concept we are integrating the chromosome of Streptococcus thermophilus (the “guest”) into the chromosome of Bacillus subtilis (the “host”) creating a hybrid circular chromosome.  S. thermophilus was chosen because it’s not human pathogenic, has a similar overall structure to B. subtilis, and much of the housekeeping machinery is very similar at the amino acid level while its DNA sequence is different enough from B. subtilis to avoid erroneous recombination during hybridization.
The guest chromosome will be inserted into a location within the host chromosome that we surrounded with lambda recombination sites (attR and attL).  After confirming the successful hybridization of the chromosomes through sequencing, lambda recombinase will be expressed to homologously recombine the two att sites at the hybrid boundary to re-segregate the host and guest chromosomes.  After cell division we assume some percent of cells will only have the (formerly) guest chromosome.  Any cells containing the host chromosome will then be killed using an inducible toxin on the host chromosome.
Hybridizing guest and host chromosomes
Previous work has shown that B. subtilis is capable of carrying a large amount of extra DNA (3.5Mb) without any growth restrictions in laboratory conditions .  The only caveat is that the resulting hybrid chromosome must be approximately symmetric with respect to the DNA origin of replication and terminus.  But other work has shown  and we have confirmed that the terminus can be removed without ill effects on DNA replication.  Therefore, inserting the guest chromosome into the location of the deleted host terminus and orienting the guest such that the guest terminus sequence is located 180° from the host origin should eliminate replication-symmetry roadblocks.
Another technical challenge involved in hybridizing the chromosomes is that of handling large DNA outside of a cell.  Bacillus transformation doesn’t require DNA to be completely intact for success, since Bacillus cuts the incoming DNA into ~18kb fragments, but shearing the DNA decreases the transformation efficiency.  Therefore we are using specialized electrophoresis techniques (CHEF) to minimize shearing.
Incompatibilities between host and guest genome
There is a possibility that some regions of the guest genome will be transcribed and those transcription products will be toxic to the host cell.  We have inserted fragments of S thermophilus into B subtilis and those genes are not transcribed at a detectable level.  But the possibility remains that some loci will be transcribed and cause an incompatibility.  If this occurs we will use RNA-Seq to diagnose the problem and devise a solution.
Initiating transcription in the booted genome
Once the guest and host genomes are separated into different cells, transcription of the formerly guest genome must be initiated (“booted”) in the remnants of the host cytoplasm.  Initial tests show that the host RNA polymerases don’t initiate any detectable transcription on guest genes, which ostensibly would imply that the guest genome could not be booted in the host cytoplasm.  Although, when there is no more host promoter DNA to bind the polymerases, the free polymerases may begin transcription on less favorable promoters.  If this does not happen we may need to induce expression of guest polymerase components using a host promoter, which has been done before with T7 polymerase  and is commercially used in recombinant enzyme production.
Why S. Thermophilus and B. Subtilis?
The bacteria
We chose these species for this project because they're both gram positive Firmicutes.  S. thermophilus has a small genome (1.8Mb), fully sequenced, and can grow at higher temperatures than bacillus, so selecting for S. thermo in a sea of B subtilis is simple. Since we're attempting a genome transplant, the fact that they're evolutionary cousins should help. As for B. subtilis, it has mechanisms for integrating large fragments of DNA into its genome and it is known that it can handle a double in it's genome size.
Designing the plasmids
Putting it together
We decided to use E. coli to replicate our plasmids, which meant that the plasmids would not only need an E. coli origin of replication, but two ampicillin resistance genes we could use for selection -- one that would work in E. coli, and one that would work in Bacillus subtilis. We used a 6-piece Gibson assembly designed on SnapGene to create our initial plasmids with these features and the Streptococcus thermophilus homologous sequences that would become the landing pad for the S. thermophilus genome.
Keoni explains the next step in the process, something he would do again, and again, and again...
Confirming the transfer
Something sweet...with poison in it
After our E. coli had obliged by growing us a bunch of baby plasmids, we transformed Bacillus by stuffing it full of those plasmids. But how could we be sure?
Easy. We cultured the Bacillus on media which contained ampicillin. Unless the Bacillus had acquired ampicillin resistance by accepting the plasmid which contained our landing pad, it wouldn't grow. We'd used a similar tactic previously, selecting for the E. coli which had accepted our plasmid. Our landing pad had been inserted successfully -- so far, so good!
Tony prepares petri dishes with agar and ampicillin.
But will it blend?
If you build it they will come
So now we've confirmed that our landing pad has been built, but can our genome actually land there? At this point we went off-plan a little bit. Our goal was still to insert S. thermophilis, but while we were working out exactly how we would do that with the whole genome, we occupied ourselves with "proof of concept" experiments. This involved using yeast as a stunt double while our star thermophilus was still in make-up.
For this, we relied on positive selection, with a toxin that would be expressed in the presence of lactose. The gene containing the toxin would be replaced by the yeast sequence if all went well. We successfully inserted 5 kilobase, 20 kilobase, and 100 kilobase strands of yeast DNA using homologous sequences on our landing site, which was confirmed both by our B. Subtilis surviving in the presence of lactose and by gel electrophoresis.
Sequencing confirmed that the transformation of yeast DNA into the landing pad was successful and that the subsequent removal of the insert (via xylose induction driving the expression of Cre recombinase) was successful.  So the overall strategy is do-able.  Next we need to actually insert 1.8Mb of DNA from S. thermophilus.
More petri dishes, as performed by the Blue Hand Group.
Tomorrow is another day
Back to the future
Unfortunately, at this point, someone found a calendar and informed us we were out of time.
We still believe at some point soon we will have a Streptococcus thermophilus genome fused to a Bacilus subtilis genome, and we'll be able to use loxP sites to cut the DNA into two loops, a Bacilus genome and a Streptococcus genome. We hope some of these dual-genome bacteria reproduce, and that at some point divisions occur in which the daughter bacteria contain only the Streptococcus genome.
And that's when you came up with the idea for the flux capacitor.
Retrieved from " http://2014.igem.org/Team:LA_Biohackers/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
