"
Page
Discussion
View source
History
teams
Log in
 
Team:KULeuven/Lab/Blue Light Receptor
From 2009.igem.org
K.U.Leuven presents:
Essencia coli,
The fragrance factory
Project
Overview
Main Components
Blue Light Receptor
Design
Modeling
Wetlab Planning
Notebook
Vanillin Production
Design
Modeling
Wetlab Planning
Notebook
Vanillin Receptor
Design
Modeling
Wetlab Planning
Notebook
Key Lock AntiKey
Design
Modeling
Wetlab Planning
Notebook
Integrated Model
Design
Modeling
BioBricks
Used Parts
Submitted Parts
Notebook
Safety
Ethics
General
Our Project
Survey
Accomplishments
Dear Judges
Project Results
Medal Checklist
About Us
Team
University
Acknowledgments
Pictures
Our Blog
Our Sponsors
Contact Us
Hidden Tools
Main Components
Integrated Model
BioBricks
Notebook
Safety
Ethics
Contents
1 Planning
1.1 Goal
1.2 Required
1.3 Where from
1.4 Steps
1.5 Important
Planning
Goal
Purifying the promoter region of the blue light receptor from E. Coli. This region needs to be ‘cleaned’ and possible restriction sites need to be mutated out. Afterwards a biobrick can be made.
Required
E.Coli strain ( MC4100)
Primers (1) for PCR: already ordered (nummers: 2171 (FP) - 2172 (RP))
Primers (2) for ‘cleaning’ region (under construction)
GFP with RBS and Terminator sequence
Where from
Strain: from lab
Primers: self made and ordered
For PCR
Forward:  CATCAT GAATTCGCGGCCGCTTCTAGAG  TTT GAC AGG TTC GTC GTC
Reverse: CTGCAGCGGCCGCTACTAGTA   CCT CTG TTA AAA ATG TTA ATC AAT GTT AAG
For ‘cleaning’
Under construction
The GFP () came from the freezer (-80) at the lab
Steps
PCR reaction to purify suspected promoter region. Probably includes a promoter, RBS and SpeI restriction site.
Partial digestion with SpeI. Afterwards, run an agarose gel to identify the correctly cut sequence and then perform a gel extraction. Cut with EcoRI.
Cut GFP with EcoRI and XbaI. Couple PCR fragment to GFP. Measure reactivity of the promoter.
Cut vector with SpeI and use klenow to create blunt ends. Then ligate the fragments together. Use initial primers to recreate restriction sites and to select correctly ligated sequences. Link to GFP to check the activity of the sequence.
“Cleaning” region to strictly promoter. Cutting in different pieces and measuring the GFP activity:
a. Same reverse primer, shortening through forward primer.    b. Once activity with forward primer ceased, keep it constant and shorten reverse     c. Once promoter found: update those primers with a pre en suffix to make a biobrick out of the promoter
standard promotor test
Important
Following conditions need to be taken into account:
For growth of the bacteria: 37°C
For expression of the genes regulated by ycgF/E system: 16°C
At 16°C: expression will start after 50h and a very slow reversion to the ground state of ycgF
Working with a colony in the dark and one in light so that the effects of cold temperature on the gene expression pattern can be calculated out.
Blue light does NOT induce stress and cell death in E. Coli
To monitor growth of the cells, measurement at OD 578 can be used
Retrieved from " http://2009.igem.org/Team:KULeuven/Lab/Blue_Light_Receptor "
Categories : Team:KULeuven/BioBricks | Team:KULeuven/BioBricks/BBa E0240
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
