"
Page
Discussion
View source
History
teams
Log in
 
Team:KULeuven/Lab/Vanillin Receptor
From 2009.igem.org
K.U.Leuven presents:
Essencia coli,
The fragrance factory
Project
Overview
Main Components
Blue Light Receptor
Design
Modeling
Wetlab Planning
Notebook
Vanillin Production
Design
Modeling
Wetlab Planning
Notebook
Vanillin Receptor
Design
Modeling
Wetlab Planning
Notebook
Key Lock AntiKey
Design
Modeling
Wetlab Planning
Notebook
Integrated Model
Design
Modeling
BioBricks
Used Parts
Submitted Parts
Notebook
Safety
Ethics
General
Our Project
Survey
Accomplishments
Dear Judges
Project Results
Medal Checklist
About Us
Team
University
Acknowledgments
Pictures
Our Blog
Our Sponsors
Contact Us
Hidden Tools
Main Components
Integrated Model
BioBricks
Notebook
Safety
Ethics
Contents
1 Planning
1.1 Goal
1.2 Required
1.3 Where from
1.4 Steps
Planning
Goal
Besides producing vanillin, we would also like to control the concentration of vanillin outside of the bacteria. To do this, we need a way to detect the vanillin extracellular and create a proper response intracellular. The vanillin sensing system receptor we propose, consists of the VirA/G system normally found in the Agrobacterium Tumefaciens. VirA is a receptor that responses to monosaccharides and phenols. Hence, it is able to sense vanillin.  When binding vanillin, VirA is activated and will activate in its turn VirG. VirG is a transcription activator that will bind to a Vir box sequence in order to activate gene expression. This Vir box sequence can be found in the VirB promoter region. RpoA is a alfa subunit polymerase that is needed to help VirG work in E. coli. For extra literature see: Y.W Lee, S.Jin, W.S.Sim and E.W.Nester, Genetic evidence for direct sensing of phenolic compounds by the vira protein of agrobacterium tumefaciens. Proc. Natl. Acad. Sci. U.S.A, 1995 december 19; 92(26): 12245–12249.
Required
Strains/isolated DNA of Agrobacterium tumefaciens: pTiBo542 and pTiA6NC/A1011
Biobricks:
VirA
VirG
RpoA
VirB promoter region
Primers:
For VirA, VirG, RpoA, VirB promoter region and for the site directed mutagenesis to remove unwanted restriction sites in VirA (PstI 2x) and RpoA (EcoRI 2x)
TOPO-Vector or iGEM vector
Where from
The primers were ordered
The plasmids: from the lab
One strain was ordered and another was available in the lab
Steps
The Agrobacterium strain is plated out on agar from -80°C. From the other strain, DNA is expected to be sent.
The primers are used to perform a PCR on the bacteria or on the isolated DNA to amplify the different genes or fragments (VirA, VirG,  RpoA and virB promoter region). An agarose is performed as a control.
The PCR fragments are ligated in a TOPO vector and electroporated into E. coli.
The plasmids are isolated via miniprep and again verified through restriction digest and agarose gel.
Site directed mutagenesis (according to stratagene) is performed on RpoA and VirA to get the EcoRI and PstI restriction sites out.
All parts are sequenced.
The different plasmids (consideration: put in registry as separate part) are combined into one standard iGEM plasmid, with the appropriate promoter, RBS and terminators.
The system’s function is checked by linking a GFP to the virB-promoter region and subsequent induction with Vanillin and Ferrulic Acid
Retrieved from " http://2009.igem.org/Team:KULeuven/Lab/Vanillin_Receptor "
Categories : Team:KULeuven/BioBricks | Team:KULeuven/BioBricks/BBa K238008 | Team:KULeuven/BioBricks/BBa K238009 | Team:KULeuven/BioBricks/BBa K238010 | Team:KULeuven/BioBricks/BBa K238011
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
