"
Page
Discussion
View source
History
teams
Log in
 
Team:KULeuven/Modeling/Vanillin Production
From 2009.igem.org
K.U.Leuven presents:
Essencia coli,
The fragrance factory
Project
Overview
Main Components
Blue Light Receptor
Design
Modeling
Wetlab Planning
Notebook
Vanillin Production
Design
Modeling
Wetlab Planning
Notebook
Vanillin Receptor
Design
Modeling
Wetlab Planning
Notebook
Key Lock AntiKey
Design
Modeling
Wetlab Planning
Notebook
Integrated Model
Design
Modeling
BioBricks
Used Parts
Submitted Parts
Notebook
Safety
Ethics
General
Our Project
Survey
Accomplishments
Dear Judges
Project Results
Medal Checklist
About Us
Team
University
Acknowledgments
Pictures
Our Blog
Our Sponsors
Contact Us
Hidden Tools
Main Components
Design
Modeling
Wetlab Planning
Notebook
Vanillin Production: Modeling
Biological model
Sequences used to test the vanillin production, the pathway from ferulic acid to vanillin in E. Coli is better investigated than the first. [7]
Mathematical model
Parameter values (Vanillin Production)
Name
Value
Comments
Reference
Degradation Rates
dmRNA
2.3105E-3 s-1
[1]
dSam8
2.8881E-4 s-1
Fast degradation through to LVA tag
[2]
dSam5
1.9254E-5 s-1
[3]
dCOMT
2.8881E-4 s-1
Fast degradation through to LVA tag
[2]
dFcs
1.9254E-5 s-1
[3]
dEch
1.9254E-5 s-1
[3]
dVanillin
8.6643E-6 s-1
Equivalent degradation dilution rate of vanillin
[4]
Transcription Rates
ktranscription
0.00848 s-1
estimate
[6]
ktranslation
0.167 s-1
estimate
[6]
Key Lock Parameters
kunlock
0.00237 s-1
Rate of unlocking the RIBOLOCK through key
[5]
klock
0.00416 s-1
Rate of locking of unlocked RIBOLOCK-Key complex.
[5]
kopen
7.5 s-1
Rate of unlocking RIBOLOCK when no key is present (LEAK).
[5]
kclose
500 s-1
Rate of locking of leaked RIBOLOCK.
[5]
Sam8 catalysis
kcat
0.015 s-1
[8]
Km
15.5 µM
[8]
Sam5 catalysis
kcat
4.2385 s-1
[9]
Km
432.0 µM
[9]
COMT catalysis
kcat
0.03319 s-1
[10]
Km
43.0 µM
[10]
Fcs catalysis
kcat
0.074068 s-1
[11]
Km
43.0 µM
[11]
Ech catalysis
kcat
2.3 s-1
[12]
Km
2.4 µM
[12]
Tyrosine parameters
ntyrosine
300000 s-1
Because tyrosine is an amino acid we consider its level regulated by the cell
[13]
Simulation
Because COMT and SAM8 are regulated by the amount of mRNA key its worthy to investigate the change of COMT and SAM8 production in function of the input amount of mRNA key. Because Sam8 and COMT are located behind the same lock they are both translated at the same rate, therefore only the concentration of COMT is shown, the translation rate of both RNA strands is considered equal.
The time scale on which COMT and Sam8 reaches equilibrium conditions is about 2 magnitudes faster than the time scale on which vanillin reaches equilibrium. Therefore only the steady state levels of COMT and Sam8 are relevant for further investigation regulatory role of both enzymes in the production of vanillin.
Because we expect only to encounter small amount of free RIBOKEY (because of the fast decay times of untranslated RNA), the steady state levels of COMT and Sam8 are relatively linearly controlled by the RIBOKEY.
Concentration of COMT in function of time and input key
.
Due to the complicated non linear (Michaelis-Menten kinetics) pathway from tyrosine to vanillin, the steady state level of vanillin well not be linearly depend on the concentration of RIBOKEY, also it takes a relatively large amount of time (hours) to reach steady state conditions.
The following figure shows that the steady state level of vanillin concentration is (non linearly) regulated by the amount of mRNA input key. Thought not linearly, the most important fact remains that the concentration of vanillin is efficiently regulated by the amount of RIBOKEY, and the leak of the RIBOLOCK is sufficiently low to obtain relatively low levels of vanillin.
Molecules of Vanillin in function of time and input key
References
[1] J.A. Bernstein et al., “Global analysis of mRNA decay and abundance in Escherichia coli at single-gene resolution using two-color fluorescent DNA microarrays,” Proceedings of the National Academy of Sciences of the United States of America, vol. 99, Jul. 2002, pp. 9697–9702
[2] J.B. Andersen et al., “New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria,” Applied and Environmental Microbiology, vol. 64, Jun. 1998, pp. 2240–2246
[3] K. Nath et al., “Protein degradation in Escherichia Coli,” The Journal of Biological Chemistry, vol. 246, Nov. 1971, pp. 6956-6967
[4] https://2009.igem.org/Team:KULeuven/Modeling/Vanillin_Receptor
[5] https://2008.igem.org/Team:KULeuven/Model/Filter
[6] S.L. Gotta et al., “rRNA Transcription Rate in Escherichia Coli,” Journal of Bacteriology, vol. 173, Oct. 1991, pp. 6647-6649
[7] E.G. LEE et al., “Directing Vanillin Production From Ferulic Acid by Increased Acetyl-CoA Consumption in Recombinant Escherichia coli,” Biotechnology and Bioengineering, Jul. 2008
[8] http://jb.asm.org/cgi/content/abstract/188/7/2666
[9] B. Hamberger et al., “The 4-coumarate:CoA ligase gene family in Arabidopsis thaliana comprises one rare, sinapate activating and three commonly occurring isoenzymes,” Max Planck Institute for Plant Breeding Research, Dec. 2003
[10] C. Zubieta et al., “Structural Basis for the Modulation of Lignin Monomer Methylation by Caffeic Acid/5-Hydroxyferulic Acid 3/5-O Methyltransferase,” American Society of Plant Biologists, Jun. 2002
[11] J. Overhage et al., “Biochemical and genetic analyses of ferulic acid catabolism in Pseudomoas sp. Strain HR199,” Applied and Environmental Microbiology, Aug. 1999
[12] A. Mitra et al., “4-Hydroxycinnamoyl-CoA Hydratase/lyase (HCHL) An Enzyme of Phenylpropanoid Chain Cleavage from Pseudomonas,” Department of Genetics and Microbiology, Apr. 2002
[13] http://gchelpdesk.ualberta.ca/CCDB/cgi-bin/STAT_NEW.cgi
© K.U.Leuven iGEM 2009 Team
Retrieved from " http://2009.igem.org/Team:KULeuven/Modeling/Vanillin_Production "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
