"
Page
Discussion
View source
History
teams
Log in
 
Team:Tokyo Tech/Project/M13 Shuriken
From 2013.igem.org
Home
Project
Ninja State Switching
M13 "Shuriken"
Farming
Experiment
Crosstalk Confirmation Assay
Crosstalk Circumvention Assay
pSB-M13 Plasmid Assay
Inducible Plaque Forming Assay
phoA Promoter Assay
Quantitative Analysis for Cytokinin
RM/lac Hybrid Promoter Assay
Modeling
Crosstalk Circumvention
Incoherent Feed Forward Loop
Submitted Parts
Human Practice
More
Safety
Attribution and Contribution
Team
Collaboration
Sponsors
Origami
Official Team Profile
Judging form
 
M13 "Shuriken"
Contents
1 1. Introduction
2 2. Phage life cycle dependent on g2p expression
3 3. Inducible M13 phage release in the whole circuit design
4 4. Model system for inducible phage release
5 5. Application
6 6. References
1. Introduction
Fig. 2-2-1. M13 phage play the role of "shuriken".
E. ninja throws "shuriken" (ninja stars) to attack E. samurai in response to 3OC12HSL, an intercellular molecule. In our story, M13 phage are the shuriken.  We confirmed that the system of AHL-dependent release and infection of M13 phage shuriken properly worked, using a plaque forming assay.
Fig. 2-2-2. Design of phage DNA for inducible phage release
We constructed the plasmid for the system of inducible release of M13 phage (Fig. 2-2-2). In this system, E. coli releases the phage in response to induction. We inserted the sequence amplified from M13mp18 vector to pSB3K3 backbone. Next, we replaced the promoter upstream of g2p, which is concerned with the release of M13 phage.
Our assay revealed that the plasmid we designed still had plaque forming activity ( pSB-M13 Plasmid Assay ).
Moreover, you can utilize various promoters with our part. If you insert an inducible promoter you like into BBa_K1139022 , phage are released triggered by the induction corresponding to the inducible promoter. In fact, we confirmed that we could use two kinds of promoters with our part. (DNA messaging Assay)
Fig. 2-2-3. The analysis of the distribution of the plaques by using ImageJ
Fig. 2-2-4. The distribution histogram of the plaques
We confirmed whether the release of M13 phage particles depends on the induction of g2p expression. Fig. 2-2-3 and Fig. 2-2-4 are the results of the assay. We analyzed the distribution of plaques by using ImageJ (Fig. 2-2-3). This histogram shows distance from the piece of paper in a horizontal axis, and shows number of plaques over distance squared, which means the rate of plaque formation, in a vertical axis (Fig. 2-2-4). This histogram shows that the inducible release of M13 phage is realized ( inducible Plaque Forming Assay ).
2. Phage life cycle dependent on g2p expression
In natural M13 phage life cycle, g2p expression is required for production of a single stranded DNA (ssDNA) and thus required for phage release.  Nicking on a double stranded DNA by G2p nickase triggers a series of reactions to amplify the phage ssDNA.  In the presence of the ssDNA, the coat proteins are assembled around it ( See more about phage assembly ). [1][2]
3. Inducible M13 phage release
in the whole circuit design
In our story, E. ninja releases the phage particles, only when E. ninja is in the Attack state induced by 3OC12HSL.  In the whole circuit (Fig. 2-2-5), g2p expression is indirectly regulated by 3OC12HSL via cI expression.
Fig. 2-2-5. 3OC12HSL-dependent phage release in the whole circuit, Model system for inducible phage release
4. Model system for inducible phage release
We constructed a model system for inducible phage release by regulation of g2p expression. The genome DNA of this engineered phage, shown in Fig. 2-2-6, needs two functions.  One is inducible expression of g2p protein.  We thus designed to replace the promoter for g2p with lux promoter.  Note that we used, not 3OC12HSL, but 3OC6HSL in this model experiment.  The other is maintenance of the genome DNA in the absence of g2p expression.  We combined M13 genome double stranded DNA with pSB3K3 backbone.
By replacement of the promoter upstream of g2p with various promoters, we have made a part collection of M13 phage. We have not only constructed the part collection, but also assayed the function of parts.
Fig. 2-2-6. Design of phage DNA for inducible phage release, pSB3K3 backbone is required for maintenance of the DNA without inducer.
Firstly, we confirmed that M13 genome with two modifications related to our design still had plaque forming activity.  One is replacement of the promoter for g2p protein with a constitutive promoter, PlacIq ( BBa_I14032 ).  The other is accommodation of pSB3K3 backbone.  Even though the plasmid has two different types of replication origins, M13 origin and pSB3 origin, this plasmid ( BBa_K1139020 ) formed plaques (Fig. 2-2-7).  In contrast, construction intermediates without a promoter upstream of g2p coding sequence (Promoterless-M13 + Plac: BBa_K1139018 , Promoterless-M13 + Plac-GFP: BBa_K1139022 ) could form no plaque ( See more about experiment: pSB-M13 Plasmid Assay ).
Fig. 2-2-7. M13 genome with two modifications, replacement of the promoter and accommodation of pSB3K3 backbone
We then confirmed that replacement of the g2p promoter with lux promoter, using as a model inducible promoter, accomplished inducible phage release. For a plaque forming assay, we used a plasmid with lux promoter upstream of g2p coding sequence (Plux-M13-Plac-GFP: BBa_K1139021 ) ( See more about DNA construction ).  First, we cultured E. coli which is introduced our new part ( BBa_K1139021 ), and added AHL which induces the phage release. We then centrifuged the culture and obtained phage particles. Finally, we added the phage particles and lawn culture (E. coli containing pSB6A1-Ptet-luxR: BBa_S03119 ) to the soft agar, and poured it on an agar plate.  Also, to make a concentration gradient of the inducer (AHL), we put a piece of filter paper on which dropped the AHL solution.
Fig. 2-2-8. A new BioBrick part ( BBa_K1139021 ) for the assay
The expression of g2p is activated by the induction, resulting in release of phage particles.  We used the phage particles for the plaque forming assay (To make a concentration gradient, we put a piece of filter paper on the plate and dripped AHL solution on the piece of filter paper).
The result shows that the plaques are formed only when the inducer exists in the medium at appropriate concentrations.  On the lawn, we firstly put two pieces of filter paper with or without AHL (Fig. 2-2-9 A).  Plaque formation was dependent on distance from the paper with AHL.  For further analysis, we thus put only a piece of filter paper with AHL in the center of the plate (Fig. 2-2-9 B).  The distribution of the plaques has an optimum value, depending on the distance from the piece of filter paper with AHL (Fig. 2-2-10).
In the vicinity of the piece of filter paper with AHL, the expression of g2p is so activated that the phage particles cannot be produced efficiently.  The reason why no plaques were formed near the piece of the filter paper is the production of the coat proteins largely exceeds that of the single stranded DNA. In contrast, far from the piece of filter paper, the expression of g2p is so poor that the phage particles cannot be produced efficiently, because the production of the single stranded DNA isn’t enough to produce phage particles.
These results show that the inducible release of M13 phage is realized; our plan was achieved ( See more about experiment : inducible Plaque Forming Assay ).
Fig. 2-2-9. Conclusion that AHL included the piece of filter paper induced phage release
Fig. 2-2-10. The distribution histogram of the plaques(Fig. 2-2-9 B)
In the same way as lux promoter, we replaced the promoter upstream of g2p with arabinose-inducible BAD promoter. First, we cultured E. coli which is introduced our new part ( BBa_K1139026 , and added arabinose, which induces the phage release. We then added the culture of E. coli introduced our new part and lawn culture (E. coli containing pSB6A1-Pcon-araC) to the soft agar, and poured it on an agar plate. Also, to make a concentration gradient of the inducer (arabinose), we put a piece of filter paper on which dropped the arabinose solution. The similar result to lux promoter was acquired (Fig. 2-2-11). The result shows that the inducible release of M13 phage is also realized with BAD promoter.
Fig. 2-2-11. Arabinose induces M13 phage to form plaque
Fig. 2-2-12. No plaques with arabinose and without araC+ receiver
5. Application
We achieved the DNA messaging system using M13 phage which is released by promoter induction.  You can apply our DNA messaging system to following two ideas because you utilize various stimuli for induction.
Drew Endy's group recently reported that they achieved the cell-cell communication system in which DNA is regarded as message [3] (Fig. 2-2-13). In this system, phages, which transmit DNA to receiver cell, were released by a limited stimulation: addition induction of a helper phage.  On the other hand, if you apply our DNA messaging system, you can make DNA messaging system more versatile and complex because you utilize various stimuli, not limiting the inducer of phage release to a helper phage [4] (Fig. 2-2-14).
Fig. 2-2-13. The system Endy's group achieved
Fig. 2-2-14. The system our team achieved
It is also important to research infectious spectrum, what kinds of bacteria phage can infect, before we use phage for bioremediation.  In future, we could use a phage as carrier of DNA transfection for bioremediation.  M13 phage can be a model system for such case.  If you use M13 phage for transfecting to bacteria living in the environment, we need to research what kinds of bacteria can be the host of M13 phage using a sample of the soil.  The gene coding fluorescence protein in the engineered phage we designed is useful when you research the host of M13 phage.  When M13 phage including the genetic circuit cording fluorescence protein infects bacteria, they shine fluorescence.  Because the fluorescence intensity depends on the kind of infected cell, you can separate these kinds of bacteria by cell sorter.  Also, you can analyze the genome of infected bacteria by single cell genomics.  As a similar study, Nojiri’s group and Shintani’s group reported that they can transfect plasmid DNA to uncultivable bacteria in soil. [5][6] They use conjugation for transfection of DNA in this research.  You can make the idea more versatile by applying our idea because you can also transfect DNA to bacteria which can be the host of M13 phage (Fig. 2-2-15).  Moreover, you could identify the strain which would be the host of M13 phage only on a specific condition because you can control the phage release with knowledge of synthetic biology.  For example, if you could find the strain which fluoresces only when AHL exists, the strain would be the host on the condition that AHL exists (Fig. 2-2-16, 2-2-17).
Fig. 2-2-15. The method applied our idea to the method which Nojiri's group and Shintani's group reported.  If you use this method, you will find the new host of M13 phage out of uncultivable bacteria.
Fig. 2-2-16. M13 phage is not released when there are not induction (i.e. AHL).
Fig. 2-2-17. M13 phage is released when there are induction (i.e. AHL).
6. References
Atsushi Higashitani. Nahoko Higashitani. Kensuke Horiuchi, DNA Replication in Filamentous Bacteriophage, Protein, Nucleic Acid and Enzyme(1994), vol39:2189-2197.
Kensuke Horiuchi, Origin of DNA replication of filamentous coliphages, Jpn. J. Genet(1990), vol65:225-241.
ME Ortiz. D Endy., Engineered cell-cell communication via DNA messaging, J. Biol. Engineering(2012), 6-16.
A Goñi-Moreno, M Amos, F de la Cruz, Multicellular Computing Using Conjugation for Wiring, PLOS ONE(2013), Vol8.:Issue6:e65986.
M Shintani. N Fukushima. M Tezuka. H Yamane. H Nojiri, Conjugative transfer of the IncP-7 carbazole degradative plasmid, pCAR1, in river water samples, Biotechnol Lett (2008). vol30:117-122
H Nojiri Structural and Molecular Genetic Analyses of the Bacterial Carbazole Degradation System Biosci, Biotechnol, Biochem, 76(1), 1-18, 2012
Retrieved from " http://2013.igem.org/Team:Tokyo_Tech/Project/M13_Shuriken "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
