<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tokyo_Tech_Project_M13_Shuriken"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tokyo_Tech/Project/M13_Shuriken">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tokyo_Tech/Project/M13_Shuriken&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo_Tech/Project/M13_Shuriken&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tokyo_Tech/Project/M13_Shuriken&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tokyo_Tech/Project/M13_Shuriken" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tokyo Tech/Project/M13 Shuriken</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="text-area"><DIV id="pagemenu"><UL id="dropmenu" class="qmmc"><LI><A class="qmparent2" href="https://2013.igem.org/Team:Tokyo_Tech">Home</A></LI><LI><A class="qmparent" href="javascript:void(0)">Project</A></LI><LI class="qmswitch"><A href="https://2013.igem.org/Team:Tokyo_Tech/Project/Ninja_State_Switching">Ninja State Switching</A></LI><LI class="qmm13"><A href="https://2013.igem.org/Team:Tokyo_Tech/Project/M13_Shuriken">M13 &quot;Shuriken&quot;</A></LI><LI class="qmfarming"><A href="https://2013.igem.org/Team:Tokyo_Tech/Project/Farming">Farming</A></LI></UL><LI><A class="qmparent" href="javascript:void(0)">Experiment</A></LI><LI class="qmswitch"><A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/Crosstalk_Confirmation_Assay">Crosstalk Confirmation Assay</A></LI><LI class="qmswitch"><A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/Crosstalk_Circumvention_Assay">Crosstalk Circumvention Assay</A></LI><LI class="qmm13"><A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/pSB-M13_Plasmid_Assay">pSB-M13 Plasmid Assay</A></LI><LI class="qmm13"><A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/Inducible_Plaque_Forming_Assay">Inducible Plaque Forming Assay</A></LI><LI class="qmfarming"><A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/phoA_Promoter_Assay"><I>phoA</I> Promoter Assay</A></LI><LI class="qmfarming"><A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/Quantitative_Analysis_of_Cytokinin">Quantitative Analysis for Cytokinin</A></LI><LI class="qmfarming"><A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/RM-lac_Hybrid_Promoter_Assay"><I>RM/lac</I> Hybrid Promoter Assay</A></LI><LI><A class="qmparent" href="javascript:void(0)">Modeling</A></LI><LI class="qmswitch"><A href="https://2013.igem.org/Team:Tokyo_Tech/Modeling/Crosstalk_Circumvention">Crosstalk Circumvention</A></LI><LI class="qmfarming"><A href="https://2013.igem.org/Team:Tokyo_Tech/Modeling/Incoherent_Feed_Forward_Loop">Incoherent Feed Forward Loop</A></LI><LI><A class="qmparent2" href="https://2013.igem.org/Team:Tokyo_Tech/Submitted_Parts">Submitted Parts</A></LI><LI><A class="qmparent2" href="https://2013.igem.org/Team:Tokyo_Tech/Human_Practice">Human Practice</A></LI><LI><A class="qmparent" href="javascript:void(0)">More</A></LI><LI><A href="https://2013.igem.org/Team:Tokyo_Tech/More/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Tokyo_Tech/More/Attribution_and_Contribution">Attribution and Contribution</A></LI><LI><A href="https://2013.igem.org/Team:Tokyo_Tech/More/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Tokyo_Tech/More/Collaboration">Collaboration</A></LI><LI><A href="https://2013.igem.org/Team:Tokyo_Tech/More/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Tokyo_Tech/More/Origami">Origami</A></LI><LI><A href="https://igem.org/Team.cgi?id=1139">Official Team Profile</A></LI><LI><A href="https://2013.igem.org/Team:Tokyo_Tech/More/Judging_form">Judging form</A></LI><LI class="qmclear"> </LI></DIV></DIV><DIV id="text-area"><DIV class="box" id="title"><P style="line-height:0em; text-indent:0em;">M13 &quot;Shuriken&quot;</P></DIV><DIV class="box"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#1._Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">1. Introduction</SPAN></A></LI><LI class="toclevel-1"><A href="#2._Phage_life_cycle_dependent_on_g2p_expression"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">2. Phage life cycle dependent on g2p expression</SPAN></A></LI><LI class="toclevel-1"><A href="#3._Inducible_M13_phage_release_in_the_whole_circuit_design"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">3. Inducible M13 phage release in the whole circuit design</SPAN></A></LI><LI class="toclevel-1"><A href="#4._Model_system_for_inducible_phage_release"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">4. Model system for inducible phage release</SPAN></A></LI><LI class="toclevel-1"><A href="#5._Application"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">5. Application</SPAN></A></LI><LI class="toclevel-1"><A href="#6._References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">6. References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="1._Introduction">1. Introduction</SPAN></H1><H2><DIV align="right"><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Fig. 2-2-1. M13 phage play the role of &quot;shuriken&quot;.</DIV></DIV></DIV></DIV><P><I>E. ninja</I> throws &quot;shuriken&quot; (ninja stars) to attack <I>E. samurai</I> in response to 3OC12HSL, an intercellular molecule. In our story, M13 phage are the shuriken.  We confirmed that the system of AHL-dependent release and infection of M13 phage shuriken properly worked, using a plaque forming assay.</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:277px;"><DIV class="thumbcaption">Fig. 2-2-2. Design of phage DNA for inducible phage release</DIV></DIV></DIV><P>
We constructed the plasmid for the system of inducible release of M13 phage (Fig. 2-2-2). In this system, <I>E. coli</I> releases the phage in response to induction. We inserted the sequence amplified from M13mp18 vector to pSB3K3 backbone. Next, we replaced the promoter upstream of <I>g2p</I>, which is concerned with the release of M13 phage.
Our assay revealed that the plasmid we designed still had plaque forming activity (<A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/pSB-M13_Plasmid_Assay" class="external text" rel="nofollow">pSB-M13 Plasmid Assay</A>).

Moreover, you can utilize various promoters with our part. If you insert an inducible promoter you like into 
<A href="http://parts.igem.org/Part:BBa_K1139022" class="external text" rel="nofollow">BBa_K1139022</A>
, phage are released triggered by the induction corresponding to the inducible promoter. In fact, we confirmed that we could use two kinds of promoters with our part. (DNA messaging Assay)
</P></H2><TABLE class="gallery" cellspacing="0" cellpadding="0" style="margin-left:auto; margin-right:auto; text-align: center;"><TBODY><TR><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Fig. 2-2-3. The analysis of the distribution of the plaques by using ImageJ
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 385px;"><DIV class="gallerytext"><P>Fig. 2-2-4. The distribution histogram of the plaques
</P></DIV></DIV></TD></TR></TBODY></TABLE><H2><P>
We confirmed whether the release of M13 phage particles depends on the induction of <I>g2p</I> expression. Fig. 2-2-3 and Fig. 2-2-4 are the results of the assay. We analyzed the distribution of plaques by using ImageJ (Fig. 2-2-3). This histogram shows distance from the piece of paper in a horizontal axis, and shows number of plaques over distance squared, which means the rate of plaque formation, in a vertical axis (Fig. 2-2-4). This histogram shows that the inducible release of M13 phage is realized (<A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/Inducible_Plaque_Forming_Assay" class="external text" rel="nofollow">inducible Plaque Forming Assay</A>).
</P></H2></DIV><DIV class="box"><H1><SPAN class="mw-headline" id="2._Phage_life_cycle_dependent_on_g2p_expression">2. Phage life cycle dependent on <I>g2p</I> expression</SPAN></H1><H2><P>
  In natural M13 phage life cycle, <I>g2p</I> expression is required for production of a single stranded DNA (ssDNA) and thus required for phage release.  Nicking on a double stranded DNA by G2p nickase triggers a series of reactions to amplify the phage ssDNA.  In the presence of the ssDNA, the coat proteins are assembled around it (<A href="https://2013.igem.org/Team:Tokyo_Tech/Project/M13_supplement#4._Phage_assembly" class="external text" rel="nofollow">See more about phage assembly</A>). [1][2]
</P></H2></DIV><DIV class="box"><H1><SPAN class="mw-headline" id="3._Inducible_M13_phage_release_in_the_whole_circuit_design">3. Inducible M13 phage release <DIV align="right">in the whole circuit design</DIV></SPAN></H1><H2><P>
In our story, <I>E. ninja</I> releases the phage particles, only when <I>E. ninja</I> is in the Attack state induced by 3OC12HSL.  In the whole circuit (Fig. 2-2-5), <I>g2p</I> expression is indirectly regulated by 3OC12HSL via <I>cI</I> expression.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Fig. 2-2-5. 3OC12HSL-dependent phage release in the whole circuit, Model system for inducible phage release</DIV></DIV></DIV></DIV></H2></DIV><DIV class="box"><H1><SPAN class="mw-headline" id="4._Model_system_for_inducible_phage_release">4. Model system for inducible phage release</SPAN></H1><H2><P>We constructed a model system for inducible phage release by regulation of <I>g2p</I> expression. The genome DNA of this engineered phage, shown in Fig. 2-2-6, needs two functions.  One is inducible expression of <I>g2p</I> protein.  We thus designed to replace the promoter for <I>g2p</I> with <I>lux</I> promoter.  Note that we used, not 3OC12HSL, but 3OC6HSL in this model experiment.  The other is maintenance of the genome DNA in the absence of <I>g2p</I> expression.  We combined M13 genome double stranded DNA with pSB3K3 backbone.
By replacement of the promoter upstream of g2p with various promoters, we have made a part collection of M13 phage. We have not only constructed the part collection, but also assayed the function of parts. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig. 2-2-6. Design of phage DNA for inducible phage release, pSB3K3 backbone is required for maintenance of the DNA without inducer.</DIV></DIV></DIV></DIV><P>
Firstly, we confirmed that M13 genome with two modifications related to our design still had plaque forming activity.  One is replacement of the promoter for <I>g2p</I> protein with a constitutive promoter, PlacI<SUP>q</SUP> (<A href="http://parts.igem.org/Part:BBa_I14032" class="external text" rel="nofollow">BBa_I14032</A>).  The other is accommodation of pSB3K3 backbone.  Even though the plasmid has two different types of replication origins, M13 origin and pSB3 origin, this plasmid (<A href="http://parts.igem.org/Part:BBa_K1139020" class="external text" rel="nofollow">BBa_K1139020</A>) formed plaques (Fig. 2-2-7).  In contrast, construction intermediates without a promoter upstream of <I>g2p</I> coding sequence (Promoterless-M13 + Plac: <A href="http://parts.igem.org/Part:BBa_K1139018" class="external text" rel="nofollow">BBa_K1139018</A>, Promoterless-M13 + Plac-<I>GFP</I>: <A href="http://parts.igem.org/Part:BBa_K1139022" class="external text" rel="nofollow">BBa_K1139022</A>) could form no plaque (<A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/pSB-M13_Plasmid_Assay#1._Introduction" class="external text" rel="nofollow">See more about experiment: pSB-M13 Plasmid Assay</A>).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:555px;"><DIV class="thumbcaption">Fig. 2-2-7. M13 genome with two modifications, replacement of the promoter and accommodation of pSB3K3 backbone</DIV></DIV></DIV></DIV><P>
We then confirmed that replacement of the <I>g2p</I> promoter with <I>lux</I> promoter, using as a model inducible promoter, accomplished inducible phage release. For a plaque forming assay, we used a plasmid with <I>lux</I> promoter upstream of <I>g2p</I> coding sequence (Plux-M13-Plac-<I>GFP</I>: <A href="http://parts.igem.org/Part:BBa_K1139021" class="external text" rel="nofollow">BBa_K1139021</A>) (<A href="https://2013.igem.org/Team:Tokyo_Tech/Project/M13_supplement#1._M13_plasmid_construction" class="external text" rel="nofollow">See more about DNA construction</A>).  First, we cultured <I>E. coli</I> which is introduced our new part (<A href="http://parts.igem.org/Part:BBa_K1139021" class="external text" rel="nofollow">BBa_K1139021</A>), and added AHL which induces the phage release. We then centrifuged the culture and obtained phage particles. Finally, we added the phage particles and lawn culture (<I>E. coli</I> containing pSB6A1-Ptet-<I>luxR</I>:<A href="http://parts.igem.org/Part:BBa_S03119" class="external text" rel="nofollow">BBa_S03119</A>) to the soft agar, and poured it on an agar plate.  Also, to make a concentration gradient of the inducer (AHL), we put a piece of filter paper on which dropped the AHL solution. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Fig. 2-2-8. A new BioBrick part (<A href="http://parts.igem.org/Part:BBa_K1139021" class="external text" rel="nofollow">BBa_K1139021</A>) for the assay  The expression of <I>g2p</I> is activated by the induction, resulting in release of phage particles.  We used the phage particles for the plaque forming assay (To make a concentration gradient, we put a piece of filter paper on the plate and dripped AHL solution on the piece of filter paper).</DIV></DIV></DIV></DIV><P>
The result shows that the plaques are formed only when the inducer exists in the medium at appropriate concentrations.  On the lawn, we firstly put two pieces of filter paper with or without AHL (Fig. 2-2-9 A).  Plaque formation was dependent on distance from the paper with AHL.  For further analysis, we thus put only a piece of filter paper with AHL in the center of the plate (Fig. 2-2-9 B).  The distribution of the plaques has an optimum value, depending on the distance from the piece of filter paper with AHL (Fig. 2-2-10). 
</P><P>
In the vicinity of the piece of filter paper with AHL, the expression of <I>g2p</I> is so activated that the phage particles cannot be produced efficiently.  The reason why no plaques were formed near the piece of the filter paper is the production of the coat proteins largely exceeds that of the single stranded DNA. In contrast, far from the piece of filter paper, the expression of <I>g2p</I> is so poor that the phage particles cannot be produced efficiently, because the production of the single stranded DNA isn’t enough to produce phage particles. 
</P><P>
These results show that the inducible release of M13 phage is realized; our plan was achieved (<A href="https://2013.igem.org/Team:Tokyo_Tech/Experiment/Inducible_Plaque_Forming_Assay#1._Introduction" class="external text" rel="nofollow">See more about experiment : inducible Plaque Forming Assay</A>).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:677px;"><DIV class="thumbcaption">Fig. 2-2-9. Conclusion that AHL included the piece of filter paper induced phage release</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig. 2-2-10. The distribution histogram of the plaques(Fig. 2-2-9 B)</DIV></DIV></DIV></DIV><P>
In the same way as lux promoter, we replaced the promoter upstream of g2p with arabinose-inducible BAD promoter. First, we cultured E. coli which is introduced our new part (<A href="http://parts.igem.org/Part:BBa_K1139026" class="external text" rel="nofollow">BBa_K1139026</A>, and added arabinose, which induces the phage release. We then added the culture of E. coli introduced our new part and lawn culture (E. coli containing pSB6A1-Pcon-<I>araC</I>) to the soft agar, and poured it on an agar plate. Also, to make a concentration gradient of the inducer (arabinose), we put a piece of filter paper on which dropped the arabinose solution.
The similar result to lux promoter was acquired (Fig. 2-2-11). The result shows that the inducible release of M13 phage is also realized with BAD promoter.
</P></H2><TABLE class="gallery" cellspacing="0" cellpadding="0" style="margin-left:auto; margin-right:auto; text-align: center;"><TBODY><TR><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>Fig. 2-2-11. Arabinose induces M13 phage to form plaque
</P></DIV></DIV></TD><TD><DIV class="gallerybox" style="width: 335px;"><DIV class="gallerytext"><P>Fig. 2-2-12. No plaques with arabinose and without araC+ receiver
</P></DIV></DIV></TD></TR></TBODY></TABLE></DIV><DIV class="box"><H1><SPAN class="mw-headline" id="5._Application">5. Application</SPAN></H1><H2><P>
We achieved the DNA messaging system using M13 phage which is released by promoter induction.  You can apply our DNA messaging system to following two ideas because you utilize various stimuli for induction. 
</P><P>
Drew Endy's group recently reported that they achieved the cell-cell communication system in which DNA is regarded as message [3] (Fig. 2-2-13). In this system, phages, which transmit DNA to receiver cell, were released by a limited stimulation: addition induction of a helper phage.  On the other hand, if you apply our DNA messaging system, you can make DNA messaging system more versatile and complex because you utilize various stimuli, not limiting the inducer of phage release to a helper phage [4] (Fig. 2-2-14).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig. 2-2-13. The system Endy's group achieved</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig. 2-2-14. The system our team achieved</DIV></DIV></DIV></DIV><P>It is also important to research infectious spectrum, what kinds of bacteria phage can infect, before we use phage for bioremediation.  In future, we could use a phage as carrier of DNA transfection for bioremediation.  M13 phage can be a model system for such case.  If you use M13 phage for transfecting to bacteria living in the environment, we need to research what kinds of bacteria can be the host of M13 phage using a sample of the soil.  The gene coding fluorescence protein in the engineered phage we designed is useful when you research the host of M13 phage.  When M13 phage including the genetic circuit cording fluorescence protein infects bacteria, they shine fluorescence.  Because the fluorescence intensity depends on the kind of infected cell, you can separate these kinds of bacteria by cell sorter.  Also, you can analyze the genome of infected bacteria by single cell genomics.  As a similar study, Nojiri’s group and Shintani’s group reported that they can transfect plasmid DNA to uncultivable bacteria in soil. [5][6] They use conjugation for transfection of DNA in this research.  You can make the idea more versatile by applying our idea because you can also transfect DNA to bacteria which can be the host of M13 phage (Fig. 2-2-15).  Moreover, you could identify the strain which would be the host of M13 phage only on a specific condition because you can control the phage release with knowledge of synthetic biology.  For example, if you could find the strain which fluoresces only when AHL exists, the strain would be the host on the condition that AHL exists (Fig. 2-2-16, 2-2-17).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig. 2-2-15. The method applied our idea to the method which Nojiri's group and Shintani's group reported.  If you use this method, you will find the new host of M13 phage out of uncultivable bacteria.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig. 2-2-16. M13 phage is not released when there are not induction (i.e. AHL).</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Fig. 2-2-17. M13 phage is released when there are induction (i.e. AHL).</DIV></DIV></DIV></DIV></H2></DIV><DIV class="box"><H1><SPAN class="mw-headline" id="6._References">6. References</SPAN></H1><H2><OL><LI>Atsushi Higashitani. Nahoko Higashitani. Kensuke Horiuchi, DNA Replication in Filamentous Bacteriophage, Protein, Nucleic Acid and Enzyme(1994), vol39:2189-2197.
</LI><LI>Kensuke Horiuchi, Origin of DNA replication of filamentous coliphages, Jpn. J. Genet(1990), vol65:225-241.
</LI><LI>ME Ortiz. D Endy., Engineered cell-cell communication via DNA messaging, J. Biol. Engineering(2012), 6-16.
</LI><LI>A Goñi-Moreno, M Amos, F de la Cruz, Multicellular Computing Using Conjugation for Wiring, PLOS ONE(2013), Vol8.:Issue6:e65986.
</LI><LI>M Shintani. N Fukushima. M Tezuka. H Yamane. H Nojiri, Conjugative transfer of the IncP-7 carbazole degradative plasmid, pCAR1, in river water samples, Biotechnol Lett (2008). vol30:117-122 
</LI><LI>H Nojiri Structural and Molecular Genetic Analyses of the Bacterial Carbazole Degradation System Biosci, Biotechnol, Biochem, 76(1), 1-18, 2012
</LI></OL></H2></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Tokyo_Tech/Project/M13_Shuriken">http://2013.igem.org/Team:Tokyo_Tech/Project/M13_Shuriken</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tokyo_Tech/Project/M13_Shuriken" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tokyo_Tech/Project/M13_Shuriken" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tokyo_Tech/Project/M13_Shuriken&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tokyo_Tech/Project/M13_Shuriken&amp;oldid=348205" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>