Team:Nanjing-China/Model
Nanjing-China2018
Modeling
Introduction
Method
Refinement:
Document
Reference
For_judges
N2 CHASER
Team
Members
Attributions
For_Judges
PROJECT
Background
Design
Results
Demonstrate
Hardware
InterLab
PARTS
Basic_Part
Composite
Improve
MODELING
PRACTICES
Human_Practices
Safety
Collaboration
NOTEBOOK
This year our team created a mathematical model to optimize the arrangement of the nif gene  cluster. This model helped we refined our design and provided some new  perspectives of our nitrogen-fixation system attranscriptional level.
We developed this model  with two goals in mind:
1. We want to  achieve the putative best stoichiometric proportion of each nif gene, which is nifB: nifH: nifD: nifK: nifE: nifN: nifX: nifV=1: 3: 4: 4: 1: 1: 1: 1.
2. We want our system as simple as possible, that means minimizing numbers of promoters and each nif gene.
We made the following assumptions:
1. There are two kinds of promoters, both of which can successfully launch the expression of every nitrogen fixation gene involved in our system.
2. One promoter is stronger(called H) while the other is relatively weak(called L). Under promoter H, each gene’s transcription level is double that of under promoter L.
3. The order of genes has little influence on their transcription level.
We conducted Real-time Quantitative PCR to detect the transcription level of nif gene cluster and the experimental data we received became an important reference for our modeling.
gene
Average value of Cq
Relative expression level
16S DNA
6.33
 
nifB
19.97
7.80E-05
nifH
17.37
4.74E-04
nifD
18.34
2.42E-04
nifK
20.77
4.48E-05
nifE
22.20
1.66E-05
nifN
22.24
1.62E-05
nifX
22.92
1.01E-05
nifV
21.25
3.22E-05
Table1  The result of qPCR
Method:
To start with, we put all genes into two groups. One group is under the strong promoter while the other is under the  weak one. We constructed two arrays, weak[i] and expected[i].
Parameters(i=1,2,3,4,5,6,7,8)
Meanings
weak[i]
the relative expression level of each nif gene under the weak promoter
weak[i]*
the relative expression level of each nif gene under the weak promoter after normalization
expected[i]
the ideal stoichiometric proportion
expected[i]*
the ideal stoichiometric proportion after normalization
strong[i]
the relative expression level of each nif gene under the strong promoter after normalization
ei
the ideal stoichiometric proportion of  the ith gene after all preprocessings
ai
the relative expression level of the ith gene under the weak promoter after all preprocessings
mi
the number of the ith gene under the strong promoter
ni
the number of the ith gene under the weak promoter
Table 2 The table of parameters in our model
Then we did some  necessary preprocessings. Firstly, we found the smallest data in weak[i] and  called it “min”. We normalized all the other data accordingly by doing:
We constructed  strong[i]:
strong[i]=2×weak[i]*   
                                              
Secondly, to guarantee the existence of a solution, we adjusted expected[i]* by examining whether it is no less than the corresponding weak[i]*, if not, we did:
expected[i]*=weak[i]*    
                                             
     
After that, we  began the organization. In order to minimize the total numbers of genes, we  arranged the strong promoter group first, and considered the weak group later.  Because each gene can be considered separately, here we only describe the  organization of the ith gene as an example.
For the ith gene, we tried adding one copy of it under the strong promoter. If
|ei-2×ai|<ei,    
we actually added  it. Until we have added (mi+1) ith genes, and got
|ei-2(mi+1)×ai|>=|ei-2mii×ai|  
Then we stopped  adding it and recorded that we have added mi ith genes  under the strong promoter.
For the weak  promoter group, we applied a similar method. For the ith gene, we  tried adding one copy of it under the weak promoter. If
|ei-2×mi×ai-ai|<|ei-2×mi×ai|,     
we actually added  it. Until we have added (ni+1) ith genes, and got
|ei-2×mi×ai-(ni+1)×ai|>=|ei-2×mii×ai-ni×ai|  
Then we stopped  adding it and recorded that we have added ni ith genes  under the weak promoter.
In that way, we  were able to determine numbers of the ith gene under the two  promoters with which the deviation was the smallest.
Fig 1. A flow diagram describing the idea of our modeling process
According to this flow diagram, we programmed with Python and got the following results:
Fig 2. The best arrangement of nif genes according to our calculation
With this arrangement, the proportion of nifB: nifH: nifD: nifK: nifE: nifN: nifX: nifV= 15.44: 46.93: 71.88: 62.10: 16.44: 16.04: 16.0: 15.94, which is close enough to the ideal proportion among all the solutions.
Refinement of  our model:
We modified the  putative best expression level of nifB:nifH:nifD:nifK:nifE:nifN:nifX:nifV to 5:3:4:4:1:1:1:1.  We believed in this way, we could better simulate the expression of nitrogenase  in our engineered E.coli strains. We made this change because of three  reasons.
Firstly, nifB is  indispensable for nitrogenase assembly no matter in diazotrophs or engineered E. coli strains. Apart from the minimal nitrogen fixation gene cluster, the genomic DNA  of wide type Paenibacillus  polymyxa includes analogues of nifM, nifU, nifS and other genes which exist in other nitrogen-fixing microorganisms and  are essential for the correct folding of nitrogenase iron protein. However, the E. coli genome doesn’t have such analogues. Nevertheless, it has been reported that the excessive expression of nifB can compensate for the absence  of nifU and nifS. That is, if nifB is overexpressed in E. coli, these auxiliaries are not necessary. Therefore, the expression level  of nifB should be the highest 5.
Secondly, compared with  nitrogen-fixing microorganisms, E. coli also lacks some genes that provide electron  transfer function, such as nifF and nifJ. So the intracellular reductive power of E. coli is insufficient to accomplish nitrogen fixation. Thus it is necessary to overexpress nifH(nitrogenase reductase) and the value  is set to 3 instead of 5 because our semiconductor, the CdS part, can provide additional electrons.
Thirdly, we set the expression  level of nifD and nifK to be 4 because molybdenum iron protein is an ɑ2β2 allotetramer and is the core of  nitrogenase.
Based on the new ideal stoichiometric proportion, we adjusted the code and received a more accurate result.
Fig 3 The best arrangement of nif genes version 2.0.
The achieved stoichiometric proportion of nifB: nifH: nifD: nifK: nifE: nifN: nifX: nifV=77.23: 46.93: 71.88: 62.10: 16.44: 16.04: 16.0: 15.94, which  is close enough to the ideal 5:3:4:4:1:1:1:1.
This model provided a potential strategy for the improvement of biological activity of nitrogenase expressed in our engineered E. coli strain and offered a great help to our further experiments.
Here is the code we taped and used.
TXT download: https://static.igem.org/mediawiki/2018/f/fe/T--Nanjing-China--model.txt
The number we typed in:
findSequence([7.8,47.4,24.2,4.48,1.66,1.62,1.01,3.22],[1,3,4,4,1,1,1,1],['nifB','nifH','nifD','nifK','nifE','nifN','nifX','nifV'])
findSequence([7.8,47.4,24.2,4.48,1.66,1.62,1.01,3.22],[5,3,4,4,1,1,1,1],['nifB','nifH','nifD','nifK','nifE','nifN','nifX','nifV'])
References
Wang, X.,  et al., Using  synthetic biology to distinguish and overcome regulatory and functional  barriers related to nitrogen fixation. PLoS One,2013. 8(7):p.e68677.
Yang, J.,  et al., Modular  electron-transport chains from eukaryotic organelles function to support  nitrogenase activity. Proc Natl Acad Sci U S A, 2017. 114(12):p.E2460-E2465.
Yang, J.,  et al., Polyprotein  strategy for stoichiometric assembly of nitrogen fixation components for  synthetic biology. Proc Natl  Acad Sci U S A, 2018. 115(36):p.E8509-E8517.
Yang, J.G.,  et al., Reconstruction  and minimal gene requirements for the alternative iron-only nitrogenase in  Escherichia coli. Proceedings  of the National Academy of Sciences of the United States of America, 2014. 111(35):p.E3718-E3725.
Address
Life Science Department
#163 Xianlin Blvd, Qixia District
Nanjing University
Nanjing, Jiangsu Province
P.R. of China
Zip: 210046
Email
nanjing_china@163.com
Social media
Twitter button:
iGEM Nanjing-China@nanjing_china
