<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Nanjing-China_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Nanjing-China/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Nanjing-China2018</TITLE></P><DIV id="d"><DIV align="center"><DIV id="HOME"><DIV class="sub"><UL><LI><A href="https://2018.igem.org/Team:Nanjing-China/Results">Results</A></LI></UL></DIV><UL><LI><A href="#overview">Overview</A></LI><LI><A href="#cds">CdS</A></LI><LI><A href="#nit"><FONT size="-1">Nitrogen fixation</FONT></A></LI></UL></DIV></DIV></DIV><DIV id="for_judge" align="center"><DIV class="i"><UL><A href="https://2018.igem.org/Team:Nanjing-China/For_Judges"><STRONG>For_judges</STRONG></A></UL></DIV></DIV><DIV class="container" align="center"><DIV id="menu"><UL><LI><A href="https://2018.igem.org/Team:Nanjing-China">N<FONT size="-1"><SUB>2</SUB></FONT> CHASER</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/For_Judges">For_Judges</A></LI></UL><LI><A href="https://2018.igem.org/Team:Nanjing-China/Background">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Demonstrate"><FONT size="-0.1">Demonstrate</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Basic_Part">Basic_Part</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Composite_Part">Composite</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Model">MODELING</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Human_Practices">PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Human_Practices"><FONT size="-1">Human_Practices</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Collaborations"><FONT size="-0.1">Collaboration</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Notebook">NOTEBOOK</A></LI></DIV><DIV class="contain"><DIV class="word" id="overview"><H2>Overview</H2><P>This year we aim to achieve light-driven nitrogen fixation based on nitrogenase and OmpA-PbrR. So we divided our experiments into 2 sections: Verification and characterization of CdS semiconductor and Authentication of nitrogenase in <EM>E. coli</EM> cells.</P><P>To demonstrate that our design,  we set the <EM>E. coli</EM> JM109 strain containing both <EM>nif</EM> gene cluster and <EM>OmpA-PbrR</EM> gene (abbreviated as EJNC in  the following passage) as the experimental group. while <EM>E. coli</EM> JM109 (abbreviated as EJ) that doesn’t contain any of those genes was set as a control group.</P></DIV><DIV class="word" id="cds"><H2>Verification and characterization of CdS semiconductor</H2><DIV class="word-1"><DIV class="word-2"><DIV class="word-note" align="center"><P><FONT size="-1">Figure 1. Cd<SUP>2+</SUP> toxicity test. Cadmium ions shows no significant toxic effects on both strains.</FONT></P></DIV></DIV><DIV class="word-2"><P>In order to  determine the optimum Cd<SUP>2+</SUP> concentration of the system, we conducted  a Cd<SUP>2+</SUP> toxicity test. The existence of cadmium ions shows no  significant toxic effects on both strains which is evidenced by the virtually equal colony forming units, though a short term growth readjustment brought by  heterologous gene expression does occur in EJNC. Considering that, we select 150 μM as the Cd<SUP>2+</SUP> concentration for our follow-up experiments.</P></DIV></DIV><DIV class="word-1"><DIV class="word-2"><DIV class="word-note" align="center"><P><FONT size="-1">Figure 2. Cd<SUP>2+</SUP> absorption test. The introduction of OmpA-PbrR confers the host cell with Cd<SUP>2+</SUP> absorption capacity.</FONT></P></DIV></DIV><DIV class="word-2"><P>The amount of  biosynthesized CdS semiconductor on the <EM>E. coli</EM> cell surface was measured using inductively coupled plasma mass spectrometry (ICP-MS). It confirms the surface-displayed PbrR-mediated  biological precipitation of CdS semiconductor on the outer membranes of cells.</P></DIV></DIV><P> </P><DIV class="word-note" align="center"><P><FONT size="-1">Figure 3. (a) TEM images of biosynthesized CdS nanoparticles on the surface of a EJNC cell.(b) EDX confirmation of randomly chosen CdS nanoparticle. The absorbed Cd<SUP>2+</SUP> precipitates on the outer membrane of EJNC in the form of CdS nanoparticles. </FONT></P></DIV><P>To acquire a fuller  understanding of CdS nanoparticles’ characteristics, we performed transmission electron microscopy (TEM) analysis of CdS nanoparticles.  The morphology and particle size of CdS nanoparticles are shown in TEM images. Next, we conducted energy-dispersive x-ray spectroscopy (EDX) to analyze its elemental composition. The result demonstrates that the semiconductor on cell surface is mainly composed of cadmium and sulfide.</P><P> </P><DIV class="word-note" align="center"><P><FONT size="-1">Figure 4. Characterization of biologically precipitated CdS nanoparticles on the outer  membranes of <EM>E. coli</EM> cells. The UV-Vis Spectrum of <EM>E. coli</EM>/CdS hybrids in solution demonstrating a band gap at 424 nm.</FONT></P></DIV><P>We performed ultraviolet-visible (UV-vis) spectral measurements to directly determine the optical band gap and photocatalytic capability of these CdS semiconductor. The lowest-energy transition of the biosynthesized CdS nanoparticles  was detected in the visible region of the solar spectrum (Eg = 2.92  eV, λ<SUB>absorption</SUB> = 424 nm), verifying  its photocatalytic ability.</P><P> </P><DIV class="word-note" align="center"><P><FONT size="-1">Figure 5. Quantitative comparison of the photoelectrical capacity of in situ  biosynthesized CdS nanoparticles. The concentrations of reduced methylviologen (MV) in various experimental groups confirm that the CdS nanoparticles  precipitate on the EJNC cells adsorb a photon and transfer an electron to MV<SUP>2+</SUP>.</FONT></P></DIV><P>The redox dye methylviologen  (MV<SUP>2+</SUP>) is a well-established electron mediator. In combination with MV<SUP>2+</SUP>, the <EM>E. coli</EM>/CdS hybrids system easily serve as a biocatalyst for photosynthesis. The concentrations of reduced MV in various experimental groups were measured under anaerobic conditions, confirming that CdS semiconductor transfers an electron to MV<SUP>2+</SUP> for every photon it absorbs.</P><P> </P></DIV><DIV class="word" id="nit"><H2>Authentication of nitrogenase system in <EM>E. coli</EM> cells</H2><DIV class="word-note" align="center"><P><FONT size="-1">Figure 6. Expression profiles of each structure gene in the <EM>nif</EM> cluster that overexpressed in EJNC. Relative expression compared to the housekeeping gene 16S rRNA is shown. qRT-PCR analysis demonstrates that all the component genes of the <EM>nif</EM> cluster are significantly over expressed in EJNC whereas the transcription of these genes are no detected (N.D.) in nondiazotrophic <EM>E. coli</EM> JM109. </FONT></P></DIV><P>To verify the  expression of <EM>nif</EM> gene cluster, we conducted Real-time Quantitative PCR(qRT-PCR) to detect the transcriptional  level of each <EM>nif</EM> gene in engineered <EM>E. coli</EM>, using 16S DNA as an internal  reference. The result provide the relative expression level of each <EM>nif</EM> gene in our constructed <EM>E. coli </EM>strain. After comparing the  result with the nif gene cluster expression in <EM>Paenibacillus polymyxa CR1</EM>, we modeled the transcription and planned to optimize the structure of <EM>nif</EM> gene cluster to achieve the best stoichiometric proportion of each <EM>nif</EM> gene. <A href="https://2018.igem.org/Team:Nanjing-China/Model">(see modeling for more  details)</A></P><DIV class="word-note"><P><FONT size="-1">Figure 7-13. Acetylene reduction assay.</FONT></P></DIV><P>Nitrogenase can not only  reduce dinitrogen to ammonia, but also reduce acetylene to ethylene. Therefore, we performed acetylene reduction assay to indirectly detected its nitrogen fixation activity. First, we plugged in 2ml of engineered bacteria (OD<SUB>600</SUB>≈0.3) in a 20ml thermo gas injection bottle, and injected Ar gas into the bottle to remove the air as much as possible. Since the removal of air is definitely not complete, it was necessary to cultivate for 6h to exhaust the oxygen through bacterial respiration. We then took 2ml of acetylene and  injected it into a gas collecting bottle after oxygen removal in culture. The  culture then reacted in our device for 16h. After that, we took 1ml of the gas  from the post-reaction gas collection bottle. In order not to exceed the gas chromatography’ s range, 1ml sample gas was injected into a gas-collecting bag  filled with nitrogen to dilute the sample gas. Finally, we used gas chromatography to measure ethylene content in sample. Our sample has been sent for testing by the time Wiki freeze, and we anticipate to receive the result within 5 days. We hope to share our results at the Giant Jamboree. </P></DIV></DIV><DIV class="footer"><DIV align="center"><DIV class="f"><DIV class="f-word"><P><STRONG><U>Address</U></STRONG></P><P>Life Science Department
              #163 Xianlin Blvd, Qixia District
              Nanjing University
              Nanjing, Jiangsu Province
              P.R. of China
              Zip: 210046
            </P></DIV><DIV class="f-word"><P><STRONG><U>Email</U></STRONG></P><P>nanjing_china@163.com</P><P><STRONG><U>Social media</U></STRONG></P><P>Twitter button: <A href="https://twitter.com/Guja1501">iGEM Nanjing-China@nanjing_china</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>