<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Nanjing-China_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Nanjing-China/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Nanjing-China2018</TITLE></P><DIV id="d"><DIV align="center"><DIV id="HOME"><DIV class="sub"><UL><LI><A href="https://2018.igem.org/Team:Nanjing-China/Model">Modeling</A></LI></UL></DIV><UL><LI><A href="#intro">Introduction</A></LI><LI><A href="#method">Method</A></LI><LI><A href="#r">Refinement:</A></LI><LI><A href="#document">Document</A></LI><LI><A href="#reference">Reference</A></LI></UL></DIV></DIV></DIV><DIV id="for_judge" align="center"><DIV class="i"><UL><A href="https://2018.igem.org/Team:Nanjing-China/For_Judges"><STRONG>For_judges</STRONG></A></UL></DIV></DIV><DIV class="container" align="center"><DIV id="menu"><UL><LI><A href="https://2018.igem.org/Team:Nanjing-China">N<FONT size="-1"><SUB>2</SUB></FONT> CHASER</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/For_Judges">For_Judges</A></LI></UL><LI><A href="https://2018.igem.org/Team:Nanjing-China/Background">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Demonstrate"><FONT size="-0.1">Demonstrate</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Parts">PARTS</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Basic_Part">Basic_Part</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Composite_Part">Composite</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Model">MODELING</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Human_Practices">PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Human_Practices"><FONT size="-1">Human_Practices</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Collaborations"><FONT size="-0.1">Collaboration</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Nanjing-China/Notebook">NOTEBOOK</A></LI></DIV><DIV class="contain"><DIV class="word" id="intro"><P>This year our team created a mathematical model to optimize the arrangement of the <EM>nif</EM> gene  cluster. This model helped we refined our design and provided some new  perspectives of our nitrogen-fixation system attranscriptional level.</P><P>We developed this model  with two goals in mind:
      1. We want to  achieve the putative best stoichiometric proportion of each nif gene, which is  <EM>nifB: nifH: nifD: nifK: nifE: nifN: nifX: nifV</EM>=1: 3: 4: 4: 1: 1: 1: 1.
      2. We want our system as simple as possible, that means minimizing numbers of promoters and each <EM>nif</EM> gene.</P><P>We made the following assumptions:
      1. There are two kinds of promoters, both of which can successfully launch the expression of every nitrogen fixation gene involved in our system. 
      2. One promoter is stronger(called H) while the other is relatively weak(called L). Under promoter H, each gene’s transcription level is double that of under promoter L.
      3. The order of genes has little influence on their transcription level.</P><P>We conducted Real-time Quantitative PCR to detect the transcription level of nif gene cluster and the experimental data we received became an important reference for our modeling.</P><DIV class="word-note"><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD valign="top"><P>gene</P></TD><TD valign="top">Average value of Cq</TD><TD valign="top">Relative expression level</TD></TR><TR><TD width="191" valign="top">16S DNA</TD><TD width="191" valign="top">6.33</TD><TD width="191" valign="top"> </TD></TR><TR><TD width="191" valign="top"><EM>nifB</EM></TD><TD width="191" valign="top">19.97</TD><TD width="191" valign="top">7.80E<SUP>-05</SUP></TD></TR><TR><TD width="191" valign="top"><EM>nifH</EM></TD><TD width="191" valign="top">17.37</TD><TD width="191" valign="top">4.74E<SUP>-04</SUP></TD></TR><TR><TD width="191" valign="top"><EM>nifD</EM></TD><TD width="191" valign="top">18.34</TD><TD width="191" valign="top">2.42E<SUP>-04</SUP></TD></TR><TR><TD width="191" valign="top"><EM>nifK</EM></TD><TD width="191" valign="top">20.77</TD><TD width="191" valign="top">4.48E<SUP>-05</SUP></TD></TR><TR><TD width="191" valign="top"><EM>nifE</EM></TD><TD width="191" valign="top">22.20</TD><TD width="191" valign="top">1.66E<SUP>-05</SUP></TD></TR><TR><TD width="191" valign="top"><EM>nifN</EM></TD><TD width="191" valign="top">22.24</TD><TD width="191" valign="top">1.62E<SUP>-05</SUP></TD></TR><TR><TD width="191" valign="top"><EM>nifX</EM></TD><TD width="191" valign="top">22.92</TD><TD width="191" valign="top">1.01E<SUP>-05</SUP></TD></TR><TR><TD width="191" valign="top"><EM>nifV</EM></TD><TD width="191" valign="top">21.25</TD><TD width="191" valign="top">3.22E<SUP>-05</SUP></TD></TR></TBODY></TABLE><P><FONT size="-1">Table1  The result of qPCR </FONT></P></DIV></DIV><DIV class="word" id="method"><H3> Method:</H3><P>To start with, we put all genes into two groups. One group is under the strong promoter while the other is under the  weak one. We constructed two arrays, weak[i] and expected[i].</P><DIV class="word-note"><TABLE border="1" cellspacing="0" cellpadding="0" width="90%"><TBODY><TR><TD width="40%" valign="top">Parameters(i=1,2,3,4,5,6,7,8)</TD><TD width="60%" valign="top">Meanings</TD></TR><TR><TD valign="top">weak[i]</TD><TD valign="top">the relative expression level of each <EM>nif</EM> gene under the weak promoter</TD></TR><TR><TD valign="top">weak[i]*</TD><TD valign="top">the relative expression level of each <EM>nif</EM> gene under the weak promoter after normalization</TD></TR><TR><TD valign="top">expected[i]</TD><TD valign="top">the ideal stoichiometric proportion</TD></TR><TR><TD valign="top">expected[i]*</TD><TD valign="top">the ideal stoichiometric proportion after normalization</TD></TR><TR><TD valign="top">strong[i]</TD><TD valign="top">the relative expression level of each <EM>nif</EM> gene under the strong promoter after normalization</TD></TR><TR><TD valign="top">e<SUB>i</SUB></TD><TD valign="top">the ideal stoichiometric proportion of  the i<SUP>th</SUP> gene after all preprocessings </TD></TR><TR><TD valign="top">a<SUB>i</SUB></TD><TD valign="top">the relative expression level of the i<SUP>th</SUP> gene under the weak promoter after all preprocessings</TD></TR><TR><TD valign="top">m<SUB>i</SUB></TD><TD valign="top">the number of the i<SUP>th</SUP> gene under the strong promoter</TD></TR><TR><TD valign="top">n<SUB>i</SUB></TD><TD valign="top">the number of the i<SUP>th</SUP> gene under the weak promoter</TD></TR></TBODY></TABLE><P align="center"><FONT size="-1">Table 2 The table of parameters in our model</FONT></P></DIV><P>Then we did some  necessary preprocessings. Firstly, we found the smallest data in weak[i] and  called it “min”. We normalized all the other data accordingly by doing:</P><P>We constructed  strong[i]:</P><DIV align="center"><EM>strong[i]=2×weak[i]*</EM>   </DIV>                                                <P>Secondly, to guarantee the existence of a solution, we adjusted expected[i]* by examining whether it is no less than the corresponding weak[i]*, if not, we did:</P><DIV align="center"><EM>expected[i]*=weak[i]* </EM>   </DIV>                                              
      <P>
After that, we  began the organization. In order to minimize the total numbers of genes, we  arranged the strong promoter group first, and considered the weak group later.  Because each gene can be considered separately, here we only describe the  organization of the i<SUP>th</SUP> gene as an example.
For the i<SUP>th</SUP>  gene, we tried adding one copy of it under the strong promoter. If </P><DIV align="center"><EM>|e<SUB>i</SUB>-2×a<SUB>i</SUB>|&lt;e<SUB>i</SUB>,     </EM></DIV><P>we actually added  it. Until we have added (m<SUB>i</SUB>+1) i<SUP>th</SUP> genes, and got</P><DIV align="center"><EM>|e<SUB>i</SUB>-2(m<SUB>i</SUB>+1)×a<SUB>i</SUB>|&gt;=|e<SUB>i</SUB>-2mi<SUB>i</SUB>×a<SUB>i</SUB>| </EM> </DIV><P>Then we stopped  adding it and recorded that we have added m<SUB>i</SUB> i<SUP>th</SUP> genes  under the strong promoter.</P><P>For the weak  promoter group, we applied a similar method. For the i<SUP>th</SUP> gene, we  tried adding one copy of it under the weak promoter. If</P><DIV align="center"><EM>|e<SUB>i</SUB>-2×m<SUB>i</SUB>×a<SUB>i</SUB>-a<SUB>i</SUB>|&lt;|e<SUB>i</SUB>-2×m<SUB>i</SUB>×a<SUB>i</SUB>|, </EM>    </DIV><P>we actually added  it. Until we have added (n<SUB>i</SUB>+1) i<SUP>th</SUP> genes, and got </P><DIV align="center"><EM>|e<SUB>i</SUB>-2×m<SUB>i</SUB>×a<SUB>i</SUB>-(n<SUB>i</SUB>+1)×a<SUB>i</SUB>|&gt;=|e<SUB>i</SUB>-2×mi<SUB>i</SUB>×a<SUB>i</SUB>-n<SUB>i</SUB>×a<SUB>i</SUB>|</EM>  </DIV><P>Then we stopped  adding it and recorded that we have added n<SUB>i</SUB> i<SUP>th</SUP> genes  under the weak promoter.</P><P>In that way, we  were able to determine numbers of the i<SUP>th</SUP> gene under the two  promoters with which the deviation was the smallest.</P><DIV class="word-note" align="center"><P><FONT size="-1">Fig 1. A flow diagram describing the idea of our modeling process</FONT></P></DIV><P>According to this flow diagram, we programmed with Python and got the following results:</P><DIV class="word-note" align="center"><P><FONT size="-1">Fig 2. The best arrangement of <EM>nif</EM> genes according to our calculation</FONT></P></DIV><P>With this arrangement, the proportion of <EM>nifB: nifH: nifD: nifK: nifE: nifN: nifX: nifV</EM>= 15.44: 46.93: 71.88: 62.10: 16.44: 16.04: 16.0: 15.94, which is close enough to the ideal proportion among all the solutions.</P></DIV><DIV class="word" id="r"><H2>Refinement of  our model:</H2><P>We modified the  putative best expression level of <EM>nifB:nifH:nifD:nifK:nifE:nifN:nifX:nifV</EM> to 5:3:4:4:1:1:1:1.  We believed in this way, we could better simulate the expression of nitrogenase  in our engineered <EM>E.coli</EM> strains. We made this change because of three  reasons.</P><P>Firstly, <EM>nifB</EM> is  indispensable for nitrogenase assembly no matter in diazotrophs or engineered <EM>E. coli</EM> strains. Apart from the minimal nitrogen fixation gene cluster, the genomic DNA  of wide type <EM>Paenibacillus  polymyxa </EM>includes analogues of <EM>nifM</EM>, <EM>nifU</EM>, <EM>nifS</EM> and other genes which exist in other nitrogen-fixing microorganisms and  are essential for the correct folding of nitrogenase iron protein. However, the <EM>E. coli </EM>genome doesn’t have such analogues. Nevertheless, it has been reported that the excessive expression of <EM>nifB</EM> can compensate for the absence  of <EM>nifU</EM> and <EM>nifS</EM>. That is, if <EM>nifB</EM> is overexpressed in <EM>E. coli</EM>, these auxiliaries are not necessary. Therefore, the expression level  of <EM>nifB</EM> should be the highest 5.</P><P>Secondly, compared with  nitrogen-fixing microorganisms, <EM>E. coli</EM> also lacks some genes that provide electron  transfer function, such as <EM>nifF</EM> and <EM>nifJ</EM>. So the intracellular reductive power of <EM>E. coli</EM> is insufficient to accomplish nitrogen fixation. Thus it is necessary to overexpress <EM>nifH</EM>(nitrogenase reductase) and the value  is set to 3 instead of 5 because our semiconductor, the CdS part, can provide additional electrons.</P><P>Thirdly, we set the expression  level of <EM>nifD</EM> and <EM>nifK</EM> to be 4 because molybdenum iron protein is an ɑ2β2 allotetramer and is the core of  nitrogenase.</P><P>Based on the new ideal stoichiometric proportion, we adjusted the code and received a more accurate result.</P><DIV class="word-note" align="center"><P><FONT size="-1">Fig 3 The best arrangement of nif genes version 2.0.</FONT></P></DIV><P>The achieved stoichiometric proportion of <EM>nifB: nifH: nifD: nifK: nifE: nifN: nifX: nifV</EM>=77.23: 46.93: 71.88: 62.10: 16.44: 16.04: 16.0: 15.94, which  is close enough to the ideal 5:3:4:4:1:1:1:1.</P><P>This model provided a potential strategy for the improvement of biological activity of nitrogenase expressed in our engineered <EM>E. coli</EM> strain and offered a great help to our further experiments.</P></DIV><DIV class="word" id="document">
    Here is the code we taped and used.
    
    TXT download:<A href="https://static.igem.org/mediawiki/2018/f/fe/T--Nanjing-China--model.txt">https://static.igem.org/mediawiki/2018/f/fe/T--Nanjing-China--model.txt</A><DIV id="code" align="left"><P>The number we typed in:</P><OL><LI>findSequence([7.8,47.4,24.2,4.48,1.66,1.62,1.01,3.22],[1,3,4,4,1,1,1,1],['nifB','nifH','nifD','nifK','nifE','nifN','nifX','nifV'])</LI><LI>findSequence([7.8,47.4,24.2,4.48,1.66,1.62,1.01,3.22],[5,3,4,4,1,1,1,1],['nifB','nifH','nifD','nifK','nifE','nifN','nifX','nifV'])</LI></OL></DIV></DIV><DIV class="word" id="reference" align="left"><H2>References</H2><OL><LI>Wang, X.,  et al., <EM>Using  synthetic biology to distinguish and overcome regulatory and functional  barriers related to nitrogen fixation. </EM>PLoS One,2013. <STRONG>8</STRONG>(7):p.e68677.</LI><LI>Yang, J.,  et al., <EM>Modular  electron-transport chains from eukaryotic organelles function to support  nitrogenase activity.</EM> Proc Natl Acad Sci U S A, 2017. <STRONG>114</STRONG>(12):p.E2460-E2465.</LI><LI>Yang, J.,  et al., <EM>Polyprotein  strategy for stoichiometric assembly of nitrogen fixation components for  synthetic biology. </EM>Proc Natl  Acad Sci U S A, 2018. <STRONG>115</STRONG>(36):p.E8509-E8517.</LI><LI>Yang, J.G.,  et al., <EM>Reconstruction  and minimal gene requirements for the alternative iron-only nitrogenase in  Escherichia coli. </EM>Proceedings  of the National Academy of Sciences of the United States of America, 2014. <STRONG>111</STRONG>(35):p.E3718-E3725.</LI></OL></DIV></DIV><DIV class="footer"><DIV align="center"><DIV class="f"><DIV class="f-word"><P><STRONG><U>Address</U></STRONG></P><P>Life Science Department
              #163 Xianlin Blvd, Qixia District
              Nanjing University
              Nanjing, Jiangsu Province
              P.R. of China
              Zip: 210046
            </P></DIV><DIV class="f-word"><P><STRONG><U>Email</U></STRONG></P><P>nanjing_china@163.com</P><P><STRONG><U>Social media</U></STRONG></P><P>Twitter button: <A href="https://twitter.com/Guja1501">iGEM Nanjing-China@nanjing_china</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>