Team:Nanjing-China/Notebook
Nanjing-China2018
Notebook
Journal
Protocol
Reference
For_judges
N2 CHASER
Team
Members
Attributions
For_Judges
PROJECT
Background
Design
Results
Demonstrate
Hardware
InterLab
PARTS
Basic_Part
Composite
Improve
MODELING
PRACTICES
Human_Practices
Safety
Collaboration
NOTEBOOK
March
3
Our team was founded this week! We met and communicated with each other. Through our team leader’s presentation, we knew iGEM a lot. Each member was assigned his/her mission in the team.
We broadly read paper and brainstormed this year’s project. At first we came up with several different ideas, later we reached an agreement that the most interesting and meaningful one was about nitrogen fixation.
April
4
Human  Practices, Collaboration&Society:
We learned about the advancement of nitrogenous fertilizer production in China. In order to better understand the actual demand of nitrogenous fertilizer in agriculture, we decided to visit farmers in Xiaohe Bei Village.
We distributed brochures about our project at NJU.
Having learned of the dearth of efficient and affordable fertilizer, we spared no effort to seek a cost effective nitrogen fixation method. Inspired by our previous work(Nanjing-China 2016), we creatively proposed an idea of “whole-cell photocatalytic nitrogen fixation”.
We helped Nanjing Forestry University build their team.
We held conferences with Nanjing Agricultural University and China Pharmaceutical University to share experiences of being iGEMers.
Technical works Wet& Dry labs:
Wet Lab: Having confirmed the theme of our project, we began to work on our design. We read latest papers about biological nitrogen fixation, focused on the method sections and discussed what we didn’t understand in details. During the last week of this month, we worked out the first version of our design.
Dry lab: We communicated and exchanged ideas frequently in order to identify possible modeling directions which could provide useful guidance to our wet experiments. Later we proposed a few directions. The idea of developing homologous modeling of nitrogenase didn’t work successfully because we couldn’t get access to relevant software.
May
5
Human  Practices, Collaboration& Society:
We planned to investigate the current production of nitrogenous fertilizer so we prepared interview questions and contacted Yantai Wuzhou Feng Fertilizer Plant. Then we went there, met the manager and were shown around the factories. We communicated with the technical R&D personnel and realized the big challenge we had to overcome before putting our project into practical application.
Technical works Wet& Dry labs:
Wet lab: We measured the transcriptional  activity of nif promoter. Then we transformed the plasmid pUC57 containing the nif cluster and the fusion protein  expression plasmid including E. coli outer membrane protein OmpA and the PbrR protein into E. coli strain JM109. Besides, based on our Human Practice, we modified our design by  adding Cd2+ toxicity test to it.
June
6
Exam Break
July
7
Human  Practices, Collaboration& Society:
We invited Professor Haoqian Zhang and held  a meet up with iGEM teams in Nanjing.
We were interviewed by Nanjing University  Student Career Guidance Center. The Wechat Push introducing our team was issued  on the public account “ NJU Employment”.
We borrowed the No.5 plasmid in the  InterLab kit to team AHUT.
Technical works Wet& Dry labs:
Wet  lab: We conducted Real-time Quantitative PCR(qPCR). Meanwhile, we conducted Cd2+ toxicity test and ICP-MS measurement of Cd2+ adsorption.
We improved our part to make it easier to operate.
Dry lab: Enlightened by the different  relative transcriptional levels of each nitrogenase component which was shown in the result of qPCR, we turned our attention to the  complexity of nitrogenase system. We perused literature on the stoichiometry of  nitrogenase components.
August
8
Human  Practices, Collaboration& Society:
We attended the 5th Conference  of China iGEMer Community at Shanghai Tech University to demonstrate our project  to all teams in China and learn from each other.
We helped Central South University found  team.
We communicated with two members of iGEM  USTC.
Technical works Wet& Dry labs:
Wet  lab: We conducted TEM-EDX analysis and UV-vis scanning. Then we used methyl  viologen to verify the generation of electron. Finally, we successfully biosynthesized  CdS semiconductor which could be excited by visible light to generate  electrons.
Dry lab: We finally decided to model on the  best stoichiometry of nif gene cluster. We looked through many common algorithms  and figured out two modeling methods. After further comparison, we finally  chose a method similar to greedy algorithm. We drew a flow diagram to describe  the core idea of our method and as a reference for programming. Then we  programmed with python, debugged our code and received the result.
September
9
Human  Practices, Collaboration& Society:
We performed Language project with IIT  Madras and received the finished video a few days later.
We helped another iGEM team, CSU-China to  establish their team. We issued our Emoji chanllenge on the  official website and received some interesting feedback shortly after that.
Technical works Wet& Dry labs:
Wet  lab: Inspired by our Human Practice, we raised an idea of designing a device  for the growth of engineered E.coli strain. First we drew a draft on  paper and then used the software Solid Works to draw a 3D version draft.  Eventually, a real device came out. The device provided a great help to our  further experiments because it provided a suitable place for the engineered  strain to grow.
Near the end of this month, we performed gas chromatography to detect the amount of acetylene reduced to indirectly test the nitrogen fixation activity of our system. Up to wiki freeze, samples have been sent out and we will receive the result about five days later.
Dry lab: We refined our model by further reading literature and verifying one of our assumptions. In that way, we got a  more accurate result. This result provided useful guidance to our further  experiments.
Protocol
Plasmids and Bacterial Strains.
The bacterial strains, plasmids and primers used in this study are all listed in  Table 1. Escherichia  coliJM109 was purchased from Takara and  designated EJ. A high-copy plasmid, pUC57-nif (pMB1 ori), harboring the minimal nitrogen fixation gene  cluster (nif) of Paenibacillus polymyxa CR1 was chemically synthesized and then  transformed into E. coli JM109, and the resulting recombinant was designated EJN. For  construction of the second plasmid, pJQ200SK OmpA/PbrR (with  a compatible p15A ori), a lab store  plasmid pBAD24-OmpA/PbrR was used as  the template to PCR-amplify OmpA/PbrR with P200F and P200R primers. After confirmation by sequencing, the PCR product was digested with Kpn I and Hind III and then insert into  pJQ200SK to yield pJQ200SK-OmpA/PbrR. EJN transformed with pJQ200SK-OmpA/PbrR was selected from LB agar plates containing appropriate antibiotics, and the resulting strain was designated EJNC.
Culture  Conditions.
LB broth for E. coli JM109 growth contained 10g/L tryptone, 10 g/L NaCl,  and 5 g/L yeast extract. KPM minimal medium was adopted for all nitrogen  fixation assays and contained per liter 1040 mg Na2HPO4, 3400 mg KH2PO4, 26 mg CaCl2·2H2O, 30 mg MgSO4, 7.5 mg Na2MoO4·2H2O, 0.3mg MnSO4, 8000 mg glucose, 500 mg casein hydrolysate, 36  mg ferric citrate, 10 mg para-aminobenzoic acid, 5 mg biotin, and 1 mg vitamin  B1, supplied with 10 mM (NH4)2SO4 (KPM-HN) for  pregrowth or 10 mM glutamate (KPM-LN) for nitrogenase activity assays.  Antibiotics were supplemented as required at the following concentrations: 100 μg/mL of ampicillin, and 20 μg/mL of gentamycin.
Quantitative  Real-time PCR.
After harvesting bacteria from  LB medium, purification of total RNA was performed using RNAiso Plus reagent (TaKaRa,  Japan) following the protocol described by the manufacturer. One microgram of  qualified total RNA was subjected to reverse transcription with a PrimeScript  RT reagent Kit with gDNA Eraser per the manufacturer’s  instructions (TaKaRa, Japan). qRT-PCR of the  resulting cDNA was performed with gene-specific primers (Table 1) on a CFX  Connect Real-Time PCR Detection System (Bio-Rad, USA) with a SYBR Premix Ex Taq  (Tli RNaseH Plus) Kit (TaKaRa, Japan). Standard curves of cDNA dilutions were used to determine the PCR efficiency. An expression data analysis was performed  by the Pfaffl method of relative quantification using CFX Manager 3.1 software (Bio-Rad, USA).
Nitrogenase  Activity Assay.
The C2H2 reduction method was used to assay nitrogenase activity. EJNC was initially  grown overnight in KPM-HN medium and then diluted in 2 mL KPM-LN medium in 20  mL sealed tube to a final OD600 of about 0.3.  Air in the tubes was repeatedly evacuated and replaced with argon. After  incubation at 37 °C for 6 to 8  h, 2 mL C2H2 was injected. 1 mL of gas  was sampled from the gas phase 16 h later and analyzed with a GC-7890B  (Agilent, USA) gas chromatograph after appropriate 10-fold serially dilution  with nitrogen. Both EJ and EJN severed as controls.
ICP-MS(Inductively Coupled Plasma Mass  Spectrometry) measurement of Cd2+ adsorption.
Escherichia coli BL21 containing OmpA-PbrR-PJQ200SK (pBAD33) plasmid was cultured in LB medium to an OD600 of 0.4-0.6.  Arabinose and CdCl2 were added to the medium to a final arabinose  concentration of 40 μM and a final Cd2+ concentration of 100 μM, to  induce the formation of CdS nano semiconductors.From  the start of the induction, 5 ml of the bacterial solution was taken from the culture  every 6 hours (sampling to 24 hours), centrifuged at 4000 rpm for 2 minutes,  and washed three times with water to remove the medium involved in the  bacterial surface.The washed bacteria were resuspended in 5 ml of water. OD600 was measured, and the bacteria were collected by centrifugation.3 ml of  concentrated nitric acid was added and the mixture was digested overnight at 90  °C.The Cd2+ content in the sample was measured using ICP-MS.
Cd2+ toxicity test.
Multiple groups of LB medium were prepared, and  arabinose with a final concentration of 40 μM and different amounts of CdCl2 were added to the medium to form a Cd2+ gradient of 0,150 μM, 300  μM, 600 μM, and 1000 μM.E. coli BL21 containing the OmpA-PbrR-PJQ200SK  (pBAD33) plasmid and plasmid-free E. coli BL21 (control) were cultured  in different media.The OD600 value was measured every 2 hours and measured for  12 hours.
Transmission electron  microscopy with energy-dispersive x-ray spectroscopy (TEM-EDX).
After the Cd2+ adsorption induction was completed, the bacteria were collected by  centrifugation and resuspended in ultrapure water. Samples were sent for TEM  image acquisition.The thick carbon film (20 to 30 nm) on the copper grid was  immersed in the bacteria solution for 1 second before imaging, dried under atmospheric conditions, and then imaged using TEM. At the same time, the EDX  system (EDAX, AMETEK) was attached to the microscope for elemental analysis.  All TEM images were imaged using a JEOL JEM-2100 electron microscope at an acceleration bias of 200 kV.
Characterization of  biologically precipitated CdS nanoparticles
 
The photocatalytic MV2+ reduction assay was performed using a 10-mm quartz cuvette with a cap and a  light source(350-W Xe lamp). E.coli cells containing biosynthesized CdS  nanoparticles were harvested from LB medium by centrifugation (4000 rpm for 10  min). The reaction system consisted of the same amounts of different  semiconductors [TiO2 anatase (10) and synthesized free CdS nanoparticles  (29)] and 3ml of 100 mM tris-HCl(PH 7), 150mM NaCl, 5% glycerol, 100mM  ascorbic acid, and 5mM MV2+ in the quartz cuvette. O2 was  removed by bubbling N2 into the solution for 30 min. The reaction  was initiated by light irradiation and stopped by centrifugation and separation  of E.coli-CdS nanoparticles from the MV buffer. The absorption spectra  were immediately measured after centrifugation (1000g for 1 min). The  amount of reduced MV2+(MV+) that formed was calculated by  monitoring the OD605 using the molar conversion coefficient ɛ=1.3 × 104 M-1 cm-1.
Strains
E.    coli
Source
EJ
E. coli JM109
TaKaRa
EJN
E. coli JM109 harboring plasmid pUC57-nif
This study
EJNC
E. coli JM109 harboring plasmids pUC57-nif and pJQ200SK-OmpA/PbrR
This study
Plasmids
characteristic
Source
pUC57
Cloning    vector; pMB1 ori; Ampr
Lab store
pUC57-nif
pUC57    with nif; pMB1 ori; Ampr
Chemically synthesized
pJQ200SK
Cloning    vector;p15A ori; Gmr
Lab store
pJQ200SK-OmpA/PbrR
pJQ200SK    with OmpA/PbrR; p15A ori; Gmr
This study
PCR Primers
sequence
Amplicon
P200F
5’-GCTCTAGACATGAAAAAGACAGCTATCGCGA
OmpA/PbrR
P200R
5’-TCCCCCGGGTCAGATCTTATCGTCGTCATC
qRT-PCR Primers
sequence
Amplicon
QnifBF
5’-TCGGCCGTGCCAAGGAATTT
nifBfor  qRT-PCR
QnifBR
5’-CCTATGCCGGACGACAGCAG
QnifHF
5’-TGCGCCGTATGACCGTTACC
nifH for  qRT-PCR
QnifHR
5’-CCGGACGCCTCAGCTTTGTT
QnifDF
5’-GCCCGACCAAGACGATGGAG
nifD for  qRT-PCR
QnifDR
5’-CCGCAGTCCGCCAATCAGAA
QnifKF
5’-ACCTGAAGTTCGCGGCCAAA
nifK for  qRT-PCR
QnifKR
5’-ATCCGGAGCCTGCTCTTCCA
QnifEF
5’-TGCGGCAGATGGCTTACCTG
nifE for  qRT-PCR
QnifER
5’-AGCACTGCCCGCTTTCCTTT
QnifNF
5’-TCGAGAGCCGATTGCCGTTC
nifN for  qRT-PCR
QnifNR
5’-ATCCAGCGCCTCCTCCAGAT
QnifXF
5’-CGACGGAAGACGGTGTGCAT
nifX for  qRT-PCR
QnifXR
5’-TCCAGGAACTGGACGCCTGA
QnifVF
5’-TGGGCGCTGACCATTCGTTT
nifV for  qRT-PCR
QnifVR
5’-ACTGCAGCCAGCGCCTTAAA
Q16SF
5’-ACTCCTACGGGAGGCAGCAG
16S    rRNAfor qRT-PCR
Q16SR
5’-ATTACCGCGGCTGCTGG
Reference
Wang, L., et  al., A minimal nitrogen fixation gene  cluster from Paenibacillus sp. WLY78 enables expression of active nitrogenase  in Escherichia coli. PLoS Genet, 2013. 9(10):  p. e1003865.
Fixen, K.R., et  al., Light-driven carbon dioxide  reduction to methane by nitrogenase in a photosynthetic bacterium. Proc  Natl Acad Sci U S A, 2016. 113(36):  p. 10163-7.
Brown, K.A., et  al., Light-driven dinitrogen reduction  catalyzed by a CdS:nitrogenase MoFe protein biohybrid. Science, 2016. 352(6284): p. 448-50.
Kuypers, M.M.M.,  H.K. Marchant, and B. Kartal, The  microbial nitrogen-cycling network. Nat Rev Microbiol, 2018. 16(5): p. 263-276.
Wei, W., et al., A surface-display biohybrid approach to  light-driven hydrogen production in air. Sci Adv, 2018. 4(2): p. eaap9253.
Wang, X., et  al., Using synthetic biology to  distinguish and overcome regulatory and functional barriers related to nitrogen  fixation. PLoS One, 2013. 8(7):  p. e68677.
Yang, J., et  al., Modular electron-transport chains  from eukaryotic organelles function to support nitrogenase activity. Proc  Natl Acad Sci U S A, 2017. 114(12):  p. E2460-E2465.
Yang, J., et  al., Polyprotein strategy for  stoichiometric assembly of nitrogen fixation components for synthetic biology. Proc Natl Acad Sci U S A, 2018. 115(36):  p. E8509-E8517.
Yang, J.G., et  al., Reconstruction and minimal gene  requirements for the alternative iron-only nitrogenase in Escherichia coli. Proceedings of the National Academy of Sciences of the United States of  America, 2014. 111(35): p.  E3718-E3725.
Howard, J.B. and D.C. Rees, Structural basis of biological nitrogen  fixation. Chemical Reviews, 1996. 96(7):  p. 2965-2982.
Address
Life Science Department
#163 Xianlin Blvd, Qixia District
Nanjing University
Nanjing, Jiangsu Province
P.R. of China
Zip: 210046
Email
nanjing_china@163.com
Social media
Twitter button:
iGEM Nanjing-China@nanjing_china
