<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Brasil-SP_Results_Sporulation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Brasil-SP/Results/Sporulation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Brasil-SP/Results/Sporulation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Brasil-SP/Results/Sporulation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Brasil-SP/Results/Sporulation&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Brasil-SP/Results/Sporulation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Brasil-SP/Results/Sporulation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV id="innercontent"><TABLE width="80%" align="center"><TBODY><TR><TABLE width="100%"><NAV id="nav_initial"><UL><LI><A href="https://2014.igem.org/Team:Brasil-SP/TheIssue">Read more</A></LI></UL><LI><A href="https://2014.igem.org/Team:Brasil-SP/Project">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Project/Cystatin">Cystatin C</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Project/DetectionModule">Detection Module</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Project/DiagnosisModule">Diagnosis Module</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Project/ResponseModule">Response Module</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Project/Device">The Device</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Results/MainAssemblyMap">Main Assembly Map</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Results/CharacterizationAssemblies">Characterization Assemblies</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Results/Sporulation">Sporulation and Germination</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Results/interlabstudy">Interlab study</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Parts">Biobricks</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Modeling/Detectionmodule">Detection, Diagnosis and Response Module</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Notebook">Lab Journal</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Safety">Lab Safety</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Outreach/BioEtics">Public Opinion on SynBio and Bioethics</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Outreach/Jambor%C3%A9">Jamboré - The Brazilian Jamboree</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Outreach/WorkingTogether">Working Together</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Outreach/Society_within_the_Academy">Society within the Academy </A></LI><LI><A href="https://2014.igem.org/Team:Brasil-sp/media_coverage">Media Coverage</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Team">Team Members</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Team/Sponsors">Sponsors and Support</A></LI><LI><A href="https://igem.org/Team.cgi?id=1521">Team Official Profile</A></LI><LI><A href="https://2014.igem.org/Team:Brasil-SP/Achievements">Achievements</A></LI></NAV></TABLE><DIV class="whitebox"><DIV class="centering"><H1 align="center">Sporulation and Germination</H1>
.
<P align="justify">     <I>Bacillus subtilis</I> is a Gram-positive bacterium, considered a model organism which has been studied over the last 50 years. In this study we provided the microbiological bases to the right working of our device. In addition, we summarized the information we got so other teams can use our page as a basis to their work.</P><H2>Who is it?</H2><P align="justify">     When we began to outline our project, we first decided to use <I>Escherichia coli</I> as a chassis. However, looking at <I>Bacillus subtilis</I> more closely we concluded it would be a much better choice to what we aimed.</P><P align="justify">     More than a thousand years ago humans started working with <I>B. subtilis</I>. Japaneses discovered that fermented soybeans could be delicious at breakfast, and for centuries <I>Bacillus subtilis</I> has been used to produce natto<SUP>1</SUP>. Nowadays, <I>Bacillus subtilis</I> is also used as a biological factory, to produce enzymes and fine biochemicals. It’s also an interesting host for heterologous protein expression. These Gram positive bacteria can produce and secrete large amounts of protein to the cell culture medium. In addition, the genre <I>Bacillus</I> is noticeable by its high capacity of adsorbing and uptaking exogenous DNA<SUP>2</SUP>. This is one of the main reasons it has been extensively used in applied and fundamental research in the past decades.</P><P align="justify">     As a model organism, there is available a lot of detailed data about it. <I>Bacillus subtilis</I> was the first Gram-positive bacteria to have its entire genome sequenced and annotated (strain 168)<SUP>3</SUP>. Following this project, all essential genes were identified<SUP>4</SUP>. Currently, large-scale data such as transcriptome<SUP>5</SUP>, proteome<SUP>6</SUP>, secretome<SUP>7</SUP> and metabolome<SUP>8</SUP> are available.</P><P align="justify">     Furthermore, its use is promoted mainly due to the non-pathogenic nature of this organism. It has been awarded the GRAS (Generally Recognized as Safe) status by the US Food and Drug Administration - FDA. In the opposite, <I>Bacillus subtilis</I> is close enough to other clinically relevant Gram-positive pathogens to be used as an organism relevant to research new targets to antimicrobials and anti-infectives<SUP>2</SUP>.</P><P align="justify">     <I>Bacillus subtilis</I> can easily uptake exogenous DNA, secrete a lot of proteins in the medium, has been used through centuries and is a safe chassis. Can it be better?</P><H2>Yes, it can.</H2><P align="justify">     We haven’t mentioned before what made us choose <I>Bacillus subtilis</I> as our favorite “pet”-chassis and why you should use it too: the endospores.</P><P align="justify">     In a suitable growth medium, cells double in length and then divide centrally to produce two identical daughter cells. However, bacteria's life can be stressful and complex, needing to face a lot of adversities and we can use this in our favor. In this situation, like starvation, many bacteria trigger a cellular response involving many genes. In <I>Bacillus subtilis</I> this process entails the activity of more than 500 genes and takes approximately ten hours.<SUP>9</SUP></P><P align="justify">     The sporulation process starts with an asymmetric division. The biggest part of the cell (called mother) engulfs the smallest (called prespore). When this process is completed, the membrane which surrounds the cytoplasm of the prespore (now called the forespore) gets a very amorphous appearance, probably due to the absence of peptidoglycan layer to define the cell shape. Next, a modified form of cell wall, known as cortex, is synthesized between the forespore membranes, providing it with an oblong shape. Simultaneously, the protein spore coat begins to be deposited on the outside surface of the forespore. The lysis of a mother cell releases the mature spore. The spore is a resistant structure, conceived to resist to environmental adversities like heat, radiation and toxic chemicals, remaining dormant until exposed to mild conditions when it gets germinated. As a commercial product, it’s perfect. Storing and switching the cell on are equally easy, bringing to an open-and-shut product.<SUP>10</SUP></P><P align="justify">     The primary question is: exposing our cells to a stressful medium will lead them all to form spores? Certainly no. Sporulation is the ultimate bacteria's decision. And when they finally decide to do it, there are always some of them who betray the movement.</P><P align="justify">     Many of other strategies are tried before sporulation. These include secretion of antibiotics and other chemical weapons to kill other microbes that are in the same niche, the activation of cellular motility by flagella to search for new sources of nutrients and the secretion of hydrolytic enzymes to remove extracellular polysaccharides and proteins. Furthermore, before sporulation starts the cells check chromosome integrity and the state of chromosomal replication, so if the process begins the bacterium is sure that it will be finished.<SUP>9</SUP></P><P align="justify">     Once situation is extreme, cells must choose between three ways: sporulation, competence or cannibalism. Most cells make the commitment to sporulation. However, the genetic material released by cell lysis isn’t wasted by the colony. Some cells opt for a different state of competence, triggered by ComK exceeding a certain threshold level. If a cell chooses this way, it can uptake exogenous DNA resulting from lysis and uses it to repair his own DNA or sometimes new genetic information. In the case of <I>Bacillus subtilis</I> approximately 10% of cells choose this path and after approximately twenty hours they switch back to the vegetative state. On the other hand, it has been showed that some bacteria further along sporulation to produce and secrete antibacterial factors that block sibling cells from sporulating. This makes them to lysis. Such cannibalistic cells feed on the nutrients released by the lysed cells. So, in the end, they prevent their own progression toward sporulation by satiating their feeding needs.<SUP>9</SUP></P><P align="justify">     But it's even more important to understand how spore's germination occurs. In our device, we provide nutrients to the bacteria. These nutrients act as germinants. The spores can recognize them as receptor proteins encoded by the gerA family of operons, which includes gerA, gerB and gerK. We can induce germination providing alanine (Ala) or a mixture of asparagine (Asn), glucose, fructose and potassium ions (AGFK) for example. The first one activate the alanine receptor GerA and the second, the asparagine receptor GerB. The interaction between nutrient germinants and receptors triggers the replace of spore core's huge depot of dipicolinic acid and cations by water. This effect triggers the hydrolysis of the spore's peptidoglycan cortex by one of two redundant enzymes. The completion of cortex hydrolysis and subsequent germ cell wall expansion allows spore core hydration. These results in resumption of spore metabolism and macromolecular synthesis.<SUP>11</SUP> Understanding this process we can choose when and how our product will start to work.</P><P align="justify">     However, it isn’t so simple. There are different individuals in the population, with different propensities to germinate. Together, the activation status of the spore, level of germinant used and the number of germinant receptors that are triggered by the germinant influence the intensity of the signal to germination. Spores that germinate slowly are known as superdormant.The use of different pathways (above mentioned)is a good approach to promote the superdormant germination as well. A big germinate percentage is important to the right work of our device<SUP>12</SUP>.</P><P align="justify">     Another important reason to explain our choice is that we use in our circuit membrane proteins from gram positive bacteria. Because of the different characteristics of the cell membrane of Gram-positive and Gram-negative bacteria, probably our circuit will not function in an organism such as E. coli. Thus, in addition to all the advantages presented, we were also forced to adopt a different chassis than we're used due to the circuit that we chose to build.</P><H2>What we did</H2><P align="justify">     <A href="https://2014.igem.org/Team:Brasil-SP/Notebook/Protocols">Here</A> you can see all our protocols, including the Microbiology ones. We acknowledge to <A href="https://2012.igem.org/Team:LMU-Munich">LMU-Munich</A> team 2012 for making available the protocols in which we based ours. In our project, we used the <I>Bacillus subtilis</I> PRD 66 [IFO 13722]  (ATCC 19659) strain.</P><P>note: we also received a <I>Bacillus subtilis</I> donation (strain PYT9) from the Phd. André Pulshen (Chemistry Institute/ University of São Paulo). In the near future, we are planning to test this strain as a chassi as well.</P><H3>1. Growth and sporulation</H3><P align="justify">First step was to grow our chassis. <I>Bacillus subtilis</I> grows quickly at Luria-Bertani medium (agar or broth) and it was used to propagate our cultures.</P><P align="center"><I>Bacillus subtilis</I> growing in LB medium.</P><P align="center">Gram’s test for detection of vegetative <I>Bacillus subtilis</I>.</P><P align="justify">Next, sporulation was induced through extensive culture (18 h) in <A href="https://static.igem.org/mediawiki/2014/3/34/DSM_Medium_Sporulation.pdf">DSM</A>. Then, it was used a proper <A href="https://static.igem.org/mediawiki/2014/0/07/Neubauer_chamber.pdf">spore count protocol</A>, using a Neubauer chamber.</P><P align="center">Use of Neubauer’s chamber to count the number of spores (in red) and vegetative form (in yellow).</P><P align="justify">     We could achieve a sporulation level of 48.3 %, which is conniving with literature data<SUP>9</SUP>. However, it’s important to know spore's viability. Our work showed the efficiency of germination. This is very important from industrial view. Once our circuit is finished, it will be simple and cheap to grow <I>Bacillus subtilis</I> cells and to sporulate them. Understanding both processes in theory and practice, we can achieve a high efficiency approach, which could bring our device to market.</P><H3>2. Germination</H3><H4>What we tried</H4><P align="justify">     Our original attempt was heat the Spores solution for 15 minutes at 60°C to activate spores and kill bacteria, which would be plated at LB medium. So, at the end, we only would have spores at solution and we could measure the germination by counting the amount of Colony Forming Unities (CFUs) at the plate. Comparing this number with the spore number would give us the viable spore number, showing the efficiency of our germination process.</P><P align="justify">      In our first attempt, <I>Bacillus subtilis</I> didn’t grow after plated. Experiments were performed in triplicate with three dilutions (10<SUP>-2</SUP>, 10<SUP>-4</SUP> and 10<SUP>-6</SUP>) and none of them show bacterial growth. Our first thought was that we made some technical mistake, so we redid the experiment under the same conditions. </P><P align="center">After plating, the spores did not grow.</P><P align="justify">     We tried again, and in the second attempt the bacteria grew. However, something was wrong. We diluted the sample to 10-6 and this plate, as all others, was completely full after only 18 hours. Our spores were growing too well to be true, so we started to think about the heating process. Are we sure that all bacteria get killed at 60°C for 15 minutes? Probably they didn’t. So the vertiginous growth was due to the presence of bacteria, not of spores. Thus, we couldn´t say that our germination process was working.</P><P align="justify">     In the third attempt, we used a different approach. The experiment was performed in duplicate. We exposed the spore solution to boiling water for 5 minutes to kill only the bacteria. After this, we plated at LB medium. After 24 hours of growth, we have the following result:</P><P align="center">Sample 1 diluted to 10<SUP>-6</SUP>. Growth in the non-heated plate, but not in the heated one.</P><P align="center">Sample 2 diluted to 10<SUP>-6</SUP>. Growth in the non-heated plate, but not in the heated one.</P><P align="center">Sample 1 diluted to 10<SUP>-4</SUP>. Growth in the non-heated plate, but not in the heated one.</P><P align="center">Sample 2 diluted to 10<SUP>-4</SUP>. Growth in the non-heated plate, but not in the heated one.</P><P align="center">Sample 1 diluted to 10<SUP>-2</SUP>. Growth in the non-heated plate, but not in the heated one.</P><P align="center">Sample 2 diluted to 10<SUP>-2</SUP>. Growth in the non-heated plate, but not in the heated one.</P><P align="justify">     In five individual experiments, our spores didn’t germinate. They only germinate once, in the only one that didn’t have a replica. We discussed about two potential reasons to explain the differences between the first and the second attempts: 1 - bacteria didn’t get killed at 60°C, and 2 - contamination.</P><P align="justify">      Our experiments showed that with this germination approach our strain can’t grow. Even with heat activation, the free amino acids able to trigger the molecular pathways showed above are in a very low concentration in LB medium, being unable to turn on the bacteria again. </P><H2>What we learned</H2><P align="justify">     Our first conclusion is that 60°C is too low to kill all bacteria, generating an erroneous result. We suggest the use of an autoclave, equipment available in almost all laboratory, to create a gradient of temperature making sure that all bacteria get killed and spores survive. We can also use a less intense heat-treatment to kill bacteria and also activate the spores, what enhances the germination process, as follow. In literature, we found information about variation in the temperatures and the time that spores are exposed to this temperature. One of the possible choices is 75°C for 30 minutes, followed by 15 minutes on ice. It gives a more consistent result between spores in the same sample and ensures higher germination efficiency. However, in our device, we can’t activate spores using heat. Without heat activation, germination didn’t work well, but can be triggered at lower levels. So, we concluded that the solution could be a concentrated LB medium (10X)<SUP>12</SUP>,without heat treatment. This medium has a low percentage of free amino acids. By concentrating it, we can increase this percentage, allowing the metabolic pathways described above. The advantages of a complex medium over a chemically defined medium are the easiest and cheapest fabrication process, which is important to turn our device accessible to the society. Furthermore, as LB medium has different free components, allowing also the activation of pathways that trigger the germination of superdormant spores <SUP>12</SUP>.</P><P align="justify">     Due to the time, we haven’t tested this suggestive <A href="https://static.igem.org/mediawiki/2014/a/a2/Germination_Sporulation.pdf">protocol</A> yet. However, we will do this as soon as possible. Right now, time is over!</P><H2>References</H2><P align="justify">1. SCHALLMEY M et al. Developments in the use of <I>Bacillus</I> species for industrial production. <B>Canadian Journal of Microbiology</B>, 2004, 50(1): 1-17.</P><P align="justify">2. ZWEERS JC et al. Towards the development of <I>Bacillus subtilis</I> as a cell factory for membrane proteins and protein complexes. <B>Microbial Cell Factories</B>, 2008, 7:10.</P><P align="justify">3. KUNST F et al. The complete genome sequence of the gram-positive bacterium <I>Bacillus subtilis</I>. <B>Nature</B>, 1997, 390:249-256.</P><P align="justify">4. KOBAYASHI K et al. Essential <I>Bacillus subtilis</I> genes. <B>Proceedings of the National Academy of Sciences USA</B>, 2003, 100:4678-4683.</P><P align="justify">5. SIERRO N et al. DBTBS: a database of transcriptional regulation in <I>Bacillus subtilis</I> containing upstream intergenic conservation information. <B>Nucleic Acids Research</B>, 2007, 36(Database issue):D93-6.</P><P align="justify">6. WOLFF S et al. Towards the entire proteome of the model bacterium <I>Bacillus subtilis</I> by gel-based and gel-free approaches. <B>Journal of Chromatography B</B>, 2007, 849:129-140.</P><P align="justify">7. TJALSMA H Proteomics of protein secretion by <I>Bacillus subtilis</I>: separating the &quot;secrets&quot; of the secretome. <B>Microbiology and Molecular Biology Reviews</B>, 2004, 68:207-233.</P><P align="justify">8. FISCHER E and SAUER U. Large-scale in vivo flux analysis shows rigidity and suboptimal performance of <I>Bacillus subtilis</I> metabolism. <B>Nature Genetics</B>, 2005, 37:636-640.</P><P align="justify">9. SCHULTZ D et al. Deciding fate in adverse times: Sporulation and competence in <I>Bacillus subtilis</I>. <B>Proceedings of the  National Academy of Sciences USA</B>, 15, 2009. 106.</P><P align="justify">10. ERRINGTON J. <I>Bacillus subtilis</I> Sporulation: Regulation of Gene Expression and Control of Morphogenesis. <B>Microbiological Reviews</B>, 1993, 57:1-33.</P><P align="justify">11. SETLOW P. Spore germination. <B>Current Opinion in Microbiology</B>, 2003, 6:550-556.</P></DIV></DIV><DIV class="footer"><DIV class="center"><DIV class="sponsors"><H3>Sponsors</H3></DIV><DIV class="info"><DIV class="leftInfo"><DIV class="igemCaption">
					This project is part of iGEM 2014
                                        </DIV></DIV><DIV class="rightInfo"><H3>Wiki Links</H3><DIV class="wikiLeaks"><A href="https://2014.igem.org/Special:RecentChanges">Recent Changes</A><A href="https://2014.igem.org/Special:RecentChangesLinked/Team:Brasil-SP">Related Changes</A><A href="https://2014.igem.org/Special:RecentChangesLinked/Team:Brasil-SP">Special Pages</A><A href="https://2014.igem.org/2014.igem.org:Privacy_policy">Privacy Policy</A><A href="https://2014.igem.org/Special:WhatLinksHere/Team:Brasil-SP">What Links Here</A><A href="https://2014.igem.org/Special:Upload">Upload File</A><A href="https://2014.igem.org/Special:Preferences">My Preferences</A><A href="https://2014.igem.org/2014.igem.org:General_disclaimer">Disclaimers</A></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Brasil-SP/Results/Sporulation">http://2014.igem.org/Team:Brasil-SP/Results/Sporulation</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Brasil-SP/Results/Sporulation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Brasil-SP/Results/Sporulation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Brasil-SP/Results/Sporulation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Brasil-SP/Results/Sporulation&amp;oldid=376816" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></BODY></HTML>