"
Page
Discussion
View source
History
teams
Log in
 
Team:ZJU-China/Modeling/Biobrick
From 2011.igem.org
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" " http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd ">
Home
Project Overview | Background | Modeling | Data Page | Parts | Future Application
Team Meet our Team | collaboration | Our University
Notebook Labnote | Protocol | Brainstorm
HumanPractice Overview Safety | App | Novel | and Contributions |
Modeling|biobrick
 Introduction  |  Model  |  Simulation  | Reference
This model is used for simulating the behavior of three genetic circuits we designed
Overview
Summary Background
Results
Future Application
Modeling
|Biofilm formation
|Biobrick
|Results
Results
Data page Parts
Introduction
abstract:
  We will model our gene regulatory networks using Michaelis-Menten enzymatic kinetics,together with the usual rules of reaction kinetics. The resulting models, when spatial effects are neglected, are given in terms of ordinary differential equations describing the rate of change of the concentrations of gene products and proteins. A key component of all these models is the Hill function, used to describe the transcription phase. The presence of this highly nonlinear function, whilst accurately modeling the network, inevitably leads to restrictions on the analytical tools available to understand and predict the dynamics.
Basic concepts and assumptions
The ODE formalism models the concentrations of RNAs, proteins, and other molecules by time-dependent variables with values contained in the set of nonnegative real numbers. Regulatory interactions take the form of functional and differential relations between the concentration variables. For a typical transcription-translation process, the ODEs modeling approach associates two ODEs with any given gene i; one modeling the rate of change of the concentration of the transcribed mRNA r_i, and the other describing the rate of change of the concentration of its corresponding translated protein p_i. Thus for our network with 3 genes we have:
Where (1) describes transcription, (2) describes translation, and i = 1,…,N. The functions R_i(p_j) describe the dependence of mRNA concentration on protein concentration p_j (If protein p_j has no effect on mRNA r_i, then correspond function is set to zero.) The functional F(·) in (1) is defined in terms of sums and products of functions R_i. Function P_i in (2) describes the translation of the mRNA r_i into a protein p_i. Parameters γ_i, δ_i (i = 1,…,N), represent the degradation parameters of the mRNAs and proteins produced by gene i. As is common, we shall assume that the degradation of proteins or mRNAs is not regulated, namely that it does not depend on the concentrations of other molecules in the cell. Function R_i is assumed to be in the form of Hill function as usual (since our cases are all inhibitors, we shall denote the Hill function h-(p,K,n)), and the function P_i is taken to be a linear term proportional to the concentration of mRNA r_i.
where K_i is the microscopic dissociation constant, and n_i is Hill coefficient, describing cooperativity.
 >>Model 
Equations&Parameters
Based on the above, our model could be founded with following ODEs:
Where
Parameters
Variables are defined in following table.
  We should note that α and β are two functions of the oxygen concentration, which are determined by the properties of corresponding promoter. For a certain depth of the biofilm, the concentration of oxygen is a constant in our model, so are  and  . Therefore we could solve these equations at different oxygen concentration and combine all the results to show how this system work.
  Because of the nonlinearity of the Hill functions, the solutions of a system of ordinary differential equations of a network of many genes cannot generally be determined by analytical means.
Equilibrium analysis
Notice that we care more about the final state of the bacteria in different depth of the biofilm(thus in different oxygen condition), we assume that the system has reached a steady state. We could calculus this steady state by setting all derivatives with respect to time to zero. That is, let
With (4)-(10), we have:
Clearly, the behavior of this system is determined by the two functions, thus determined by the behavior of promoter vgb & promoter fdhF. On the other hand, the solutions of equations The two functions αand β are hard to determine precisely, but their behavior could be illustrated in follow graph qualitatively.
First case, When the expression of promoter vgb and fdhF could be neglected, that is, we could setαandβto be both zero. The ODEs system become
The results are shown as follow.
 
 
Secondly,in the anaerobic condition, that is, when the expression of promoter vgb could be neglected, we setαto be zero, and setβto be bigger than the dissociation constant in (13). Then we have
The results are shown as follow
 
 
 
Thirdly, since our ODEs model are symmetric respect to YFP & CFP in some sense, therefore the case of microaerobic (where we could setβto be zero) would be the same(symmetric) as in second case.
and results:
 
 
 
Robustness analysis
  Robustness is very important to a genetic circuit. Scientists hope their artificial system have high tolerance to the variety of environment or system parameters. In our genetic circuit,   and   are directly represent the properties of Vgb and Fdhf, so we are interested in to what degree these two parameters could influence the behavior of Ptet.
  We could show the contribution of the two parameters   and   by fix one of them, and let the other one varies linearly. When fix   at 1e-9, and    changing from 0 to 1e-7 continuously, the production of each protein is shown in following graphs.
Cfp.figure
Yfp.figure
Rfp.figure
Also, we could let both   and   to vary linearly (from 0 to 1e-11) to see how protein CFP changes. The actual curve with respect to depth of biofilm for CFP production is a curve on this surface.
Results
combine Data acquired in part characterization and modeling of biofilm and biobrick, we can get the relationship between distribution of oxygen in the biofilm and the expression rate of CFP RFP YFP as follow:
YFP:
RFP:
From the fitting results we know that: The PoPS of YFP could be regard as zero since PPO of 10%, and approximately linearly increasing from 0% to 2.5%, and decreasing from 2.5% to 10%.  The PoPS of RFP could be regard as zero since PPO of 2%, and approximately linear from 0% to 2%. Together with the results of former modeling sections, we could show the stratified biofilm in following graph.
 
Reference
[1]
J.B. Andersen et al., “New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria,” Applied and Environmental Microbiology, vol. 64, Jun. 1998, pp. 2240–2246.
[2]
Goryachev, A.B., D.J. Toh and T. Lee. “System analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constant.” BioSystems 2006: 83, 178-187.
[3]
Alon, Uri. “An Introduction to Systems Biology Design Principles of Biological Circiuts.” London: Chapman & Hall/CRC, 2007.
[4]
David Braun et al. Parameter estimation for two synthetic gene networks: A case study. IEEE 2005.
[5]
Aberdeen_Scotland 2009. “Modeling Parameters” iGEM wiki.
[6]
Wilfried Weber, Markus Rimann, Manuela Spielmann, Bettina Keller, Marie Daoud-El Baba, Dominique Aubel, Cornelia C Weber & Martin Fussenegger, Gas-inducible transgene expression in mammalian cells and mice, Nature Biotechnology, volume 22, number 11, November 2004
iGem 2011 Home Page Upload Files Edit CSS Team Wikis Contact Us
Retrieved from " http://2011.igem.org/Team:ZJU-China/Modeling/Biobrick "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
