<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ZJU-China_Notebook_Brainstorm"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ZJU-China/Notebook/Brainstorm">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ZJU-China/Notebook/Brainstorm&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ZJU-China/Notebook/Brainstorm&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ZJU-China/Notebook/Brainstorm&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ZJU-China/Notebook/Brainstorm" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ZJU-China/Notebook/Brainstorm</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P>&lt;!DOCTYPE html PUBLIC &quot;-//W3C//DTD XHTML 1.0 Transitional//EN&quot; &quot;<A href="http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd" class="external free" rel="nofollow">http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd</A>&quot;&gt;</P><DIV class="header"><DIV class="nav"><UL id="topnav"><LI><A id="P_Home" href="https://2011.igem.org/Team:ZJU-China">Home</A></LI><LI><A id="P_Project">Project</A><SPAN id="P_Pr"><A id="pro" href="https://2011.igem.org/Team:ZJU-China/Project">Overview</A> |
            <A href="https://2011.igem.org/Team:ZJU-China/Project">Background</A>|
            <A href="https://2011.igem.org/Team:ZJU-China/Project">Modeling</A> |
            <A href="https://2011.igem.org/Team:ZJU-China/Project&gt;Result&lt;/a&gt;|
            &lt;a href=" https:="">Data Page</A>|
            <A href="https://2011.igem.org/Team:ZJU-China/Project">Parts</A>|
            <A href="https://2011.igem.org/Team:ZJU-China/Project">Future Application</A></SPAN></LI><LI><A id="P_Team">Team</A><SPAN id="P_Te"><A id="tea" href="https://2011.igem.org/Team:ZJU-China/Team">Meet our Team</A> |
            <A href="">collaboration</A>|
            <A href="https://2011.igem.org/Team:ZJU-China/Team/University">Our University</A></SPAN></LI><LI><A id="P_Notebook">Notebook</A><SPAN id="P_No"><A id="not" href="https://2011.igem.org/Team:ZJU-China/Notebook">Labnote</A> |
            <A href="https://2011.igem.org/Team:ZJU-China/Protocol">Protocol</A>  |
                <A href="https://2011.igem.org/Team:ZJU-China/Notebook/Brainstorm">Brainstorm</A></SPAN></LI><LI><A id="P_HumanPractice">HumanPractice</A><SPAN id="P_Hu"><A id="hum" href="https://2011.igem.org/Team:ZJU-China/Humanpractice">Overview</A><A href="https://2011.igem.org/Team:ZJU-China/Safety">Safety</A> |
            <A href="https://2011.igem.org/Team:ZJU-China/Humanpractice">App</A> |
            <A href="https://2011.igem.org/Team:ZJU-China/Humanpractice">Novel</A>|
            <A href="https://2011.igem.org/Team:ZJU-China/Humanpractice"> and Contributions</A> |
</SPAN></LI></UL></DIV></DIV><DIV id="contentwrapper"><DIV class="main"><DIV class="pagetitle" id="nbook"><TABLE style="background-color:transparent;" width="750" border="0" cellspacing="0" cellpadding="1"><TBODY><TR><TD><H3>Brainstorm</H3></TD></TR></TBODY></TABLE></DIV><DIV id="leftul"><DIV class="leftul"><DIV id="accordion"><H4 class="current">May&amp;June</H4><DIV class="pane" style="height:260px;display:block;"><A href="#rf">Rainbofilm</A><A href="#gf">Gut Flora</A><A href="#as">Alternative Splicing</A><A href="#Ew">E-waste</A><A href="#sm">Signal Modular by Protein Scaffold</A><A href="#Ch">Change the color by shaking</A><A href="#Sp">Streptococcus pneumonia</A><A href="#AS">ASTHMA</A></DIV><H4>July</H4><DIV class="pane"><A href="https://2011.igem.org/Team:ZJU-China/Notebook">In this month we......&gt;&gt;Click to see</A></DIV><H4>August</H4><DIV class="pane"><A href="https://2011.igem.org/Team:ZJU-China/Notebook/August">In this month we......&gt;&gt;Click to see</A></DIV><H4>September</H4><DIV class="pane"><A href="https://2011.igem.org/Team:ZJU-China/Notebook/September">In this month we......&gt;&gt;Click to see</A></DIV><H4>Protocol</H4><DIV class="pane"><A href="https://2011.igem.org/Team:ZJU-China/Protocol">&gt;&gt;Click to see our lab protocol about biofilm formation</A></DIV></DIV></DIV></DIV><DIV class="bigblock"><DIV class="inner" id="week2"><DIV class="block" id="n_brainstorm"><H3 id="rf">Rainbofilm</H3><P>  Biofilm has been our team’s long-standing interest. Though our motivation to study biofilm at first is on its effective elimination, a review paper on biofilm totally changed our view of it and led us into a new direction. 
When something as difficult to eliminate and natural as biofilm is, can we make use of it instead of destroying it? Following this strand of thought, we started to investigate the advantages brought by the advent of biofilm for the bacteria, such as its increased resistance to harsh conditions under which normal cells cannot survive. Another fascinating fact about biofilm is that cells are differentially expressed to form different layers within biofilm. 
We continued to develop our project from two ideas based on the background. The first is based on biofilm’s differentiated expression. We hope to construct a three-layer stratified expression system within biofilm in response to oxygen concentration gradient. This could serve as a prototype for future construction simply by replacing the reporter gene with user’s target genes. The second idea stems from the biofilm’s strong resistance to bad growth conditions, such as when surrounded by high concentrations of heavy metal. We want to engineer and develop our biofilm into the second generation of biosensor. As a natural concentration gradient of heavy metal is formed along the depth of biofilm, with the same threshold for detection constructed in all the cells, the whole biofilm system would be able to detect different levels of the heavy metal concentration in its surrounding environment.
</P></DIV><DIV class="block" id="n_brainstorm"><H3 id="gf">Gut Flora</H3><P>  We believe engineering gut bacteria is a promising field in the future, for bacteria are easy to be engineered compared with engineering human cell, and human gut is naturally inhabited by many different bacteria. We could construct a gut bacteria kit for either supplying nutrients such as vitamins or synthesizing and delivering drugs. We’re highly motivated to do this project, for it offers a good solution to alleviate the current famine and diseases brought by malnutrition in many developing countries around the world. The drug delivery, adapted to host’s circadium rhythm and physiological state, would be an ideal approach to cure some chronic diseases. However, we didn’t choose to further develop Gut Flora Kit as our team’s project this year, because we have limited knowledge and experience in gut flora and intestinal environment. </P></DIV></DIV><DIV class="inner" id="week3"><DIV class="block" id="n_brainstorm"><H3 id="as">Alternative Splicing</H3><P>  Alternative splicing is a type of post-transcription mRNA modification observed in most multi-cellular organisms. It happens where there is more than one intron in a gene, meaning that different splicing site choices generate different mature mRNA thus produce different protein. The choice of the splicing site is controlled by a very complex system, yet we have some interesting and convincing understanding about it, and may it become a new perspective in synthetic biology.
In a multi-exon pre-mRNA, some exons are constitutively cut off or maintained, while some exons are sometimes skipped and sometimes kept. The latter is called an alternative splicing. The pattern can be controlled by many mechanisms involving a good many RNA secondary structure formations, protein-protein and protein-RNA interactions. Different splicing patterns always have relatively constant expression pattern in certain cell condition.
We are inspired by this interesting phenomenon and hope to design a new synthetic biology tool which can use compiled gene to produce many different kinds of proteins under different conditions.
We are inspired by a consistent mechanism of splicing pattern determination, which is called the mutually exclusive splicing. Every time only one out of several exon cassettes(altering exons) are presented. Latest discoveries revealed this is affected by certain pre-RNA secondary structure. If we change the sequence of the regulatory region, we may change the splicing pattern, and some proteins are expressed in a regulated and constant way.
One big problem about this idea is the chassis. Multi-cellular life-form is not very suitable for synthetic biology, and there is no such system in bacteria. So yeast becomes the only solution we can come up with. Very few alternative splicing is reported in yeast, yet it can normally carry out the pre-RNA processing function. Another is that changing the gene sequence to control the expression pattern is too difficult to apply.
Finally, we suspended this idea and tried to find other doable and potential projects.
</P></DIV></DIV><DIV class="inner" id="week4"><DIV class="block" id="n_brainstorm"><H3 id="Ew">E-waste</H3><P>  E-waste now is becoming a worldwide environmental problem. Current treatment is like this: most countries transport the E-waste to the developing areas; then the workers there recycle the useful or worthy parts by hand; the useless parts are shredded and the powder undergoes further selection by different physical and chemical properties; eventually the remains are deeply buried. The whole process costs tremendously and still results in environmental pollution. So we tried to solve this problem in a synthetic biology approach. We checked the components of the E waste: plastic and metal. We continued to read related papers about bacteria degrading plastic, mobilizing metal elements, or absorbing metal elements. Unfortunately, the plastic substances in the E-waste like epoxy resins, and PCBs, PVC are very hard for bacteria to degrade. Secondly, though there’s some paper indicating the power of some bacteria of recycling the metal, the detailed mechanism is still a mystery. In order to first mobilize the metal elements from the solid E-waste, we had proposed some ideas like producing acid such as acetic acid, citric acid even cyanic acid, thus the bacteria we used must be tolerant to the high pH environment. Here the dilemma is: on one hand, if we use some special bacteria, it is difficult to engineer it; on the other hand, if we use E.coli, it is difficult for us to engineer it to adapt to the acid environments. Due to these problems, we stopped further discussion on this topic.4</P></DIV></DIV><DIV class="inner" id="week5"><DIV class="block" id="n_brainstorm"><H3 id="sm">Signal Modular by Protein Scaffold</H3><P>  This idea comes from the paper by Keasling in which he used protein scaffold to boost the metabolic flow in biosynthesis. We think we can adopt similar methosd in signal transduction. At first, we were all excited about the idea. However, the tragedy is that we found out UCSF had done a similar project in 2007 iGEM. : (</P></DIV></DIV><DIV class="inner" id="week6"><DIV class="block" id="n_brainstorm"><H3 id="Ch">Change the color by shaking</H3><P>  This is just an interesting idea that there’s no specific practical application. The key of this idea is to discover a sensing system based on the physical attachment between cells. So we looked up some paper, and found one paper's novel discovery of sensing the touching. But the fatal drawback is that it lacks detailed mechanism that we can harness.</P></DIV></DIV><DIV class="inner" id="week7"><DIV class="block" id="n_brainstorm"><H3 id="Sp">Streptococcus pneumonia</H3><P>  Childhood pneumonia is the leading single cause of mortality in children aged less than 5 years. It has been noted that Pathogenicity islands (PAIs) plays an important role of being sickness. PAIs are a distinct class of genomic islands and are incorporated in the genome of pathogenic microorganisms but are usually absent from those non-pathogenic organisms of the same or closely related species. We would like to concentrate on the special properties of PAIs with measures of synthetic biology. PAIs have correlation of other diseases and may become a fundamental therapy.</P></DIV></DIV><DIV class="inner" id="week8"><DIV class="block" id="n_brainstorm"><H3 id="AS">ASTHMA</H3><P>  Asthma， a chronic inflammatory disease of the lungs, has been increasingly observed in developed countries. Its remarkable increase attributes to changes in the environment due to improved hygiene and fewer childhood infections. Researchers have found that asthma has close correlation with increased IgE level. Those who are prone to asthma are able to express complete FcεRI in dendritic cell(DC).  We planned to use synthetic biology to focus on dendritic cell to lower the level of IgE expression. This may lead to a new-era of asthma therapy and bring hope to those suffering from asthma.</P></DIV></DIV></DIV><DIV class="footer" id="footer"><A href="https://2011.igem.org/"> iGem 2011 Home Page</A><A href="http://ung.igem.org/Special:Upload/"> Upload Files</A><A href="https://2011.igem.org/wiki/index.php?title=Template:Zjucss_main&amp;action=edit">Edit CSS</A><A href="http://ung.igem.org/Team_Wikis?year=2011"> Team Wikis</A><A href="mailto:zjuigem@gmail.com?"><STRONG>Contact Us</STRONG></A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:ZJU-China/Notebook/Brainstorm">http://2011.igem.org/Team:ZJU-China/Notebook/Brainstorm</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ZJU-China/Notebook/Brainstorm" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ZJU-China/Notebook/Brainstorm" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ZJU-China/Notebook/Brainstorm&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ZJU-China/Notebook/Brainstorm&amp;oldid=235241" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>