<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ZJU-China_Modeling_Biobrick"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ZJU-China/Modeling/Biobrick">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ZJU-China/Modeling/Biobrick&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ZJU-China/Modeling/Biobrick&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ZJU-China/Modeling/Biobrick&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ZJU-China/Modeling/Biobrick" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ZJU-China/Modeling/Biobrick</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P>&lt;!DOCTYPE html PUBLIC &quot;-//W3C//DTD XHTML 1.0 Transitional//EN&quot; &quot;<A href="http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd" class="external free" rel="nofollow">http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd</A>&quot;&gt;</P><DIV class="header"><DIV class="nav"><UL id="topnav"><LI><A id="P_Home" href="https://2011.igem.org/Team:ZJU-China">Home</A></LI><LI><A id="P_Project">Project</A><SPAN id="P_Pr"><A id="pro" href="https://2011.igem.org/Team:ZJU-China/Project">Overview</A> |
            <A href="https://2011.igem.org/Team:ZJU-China/Project">Background</A>|
            <A href="https://2011.igem.org/Team:ZJU-China/Project">Modeling</A> |
            <A href="https://2011.igem.org/Team:ZJU-China/Project&gt;Result&lt;/a&gt;|
            &lt;a href=" https:="">Data Page</A>|
            <A href="https://2011.igem.org/Team:ZJU-China/Project">Parts</A>|
            <A href="https://2011.igem.org/Team:ZJU-China/Project">Future Application</A></SPAN></LI><LI><A id="P_Team">Team</A><SPAN id="P_Te"><A id="tea" href="https://2011.igem.org/Team:ZJU-China/Team">Meet our Team</A> |
            <A href="">collaboration</A>|
            <A href="https://2011.igem.org/Team:ZJU-China/Team/University">Our University</A></SPAN></LI><LI><A id="P_Notebook">Notebook</A><SPAN id="P_No"><A id="not" href="https://2011.igem.org/Team:ZJU-China/Notebook">Labnote</A> |
            <A href="https://2011.igem.org/Team:ZJU-China/Protocol">Protocol</A>  |
                <A href="https://2011.igem.org/Team:ZJU-China/Notebook/Brainstorm">Brainstorm</A></SPAN></LI><LI><A id="P_HumanPractice">HumanPractice</A><SPAN id="P_Hu"><A id="hum" href="https://2011.igem.org/Team:ZJU-China/Humanpractice">Overview</A><A href="https://2011.igem.org/Team:ZJU-China/Safety">Safety</A> |
            <A href="https://2011.igem.org/Team:ZJU-China/Humanpractice">App</A> |
            <A href="https://2011.igem.org/Team:ZJU-China/Humanpractice">Novel</A>|
            <A href="https://2011.igem.org/Team:ZJU-China/Humanpractice"> and Contributions</A> |
</SPAN></LI></UL></DIV></DIV><DIV id="contentwrapper"><DIV class="main"><DIV class="pagetitle" id="nbook"><TABLE style="background-color:transparent;" width="750" border="0" cellspacing="0" cellpadding="1"><TBODY><TR><TD><H3>Modeling|biobrick</H3></TD><TD><DIV class="roundcorner"><P><A name="jumptop" id="p_intro"> Introduction </A>|<A id="p_model"> Model </A>|<A id="p_simul"> Simulation </A>|<A id="p_refer">Reference</A></P></DIV></TD></TR></TBODY></TABLE><P style="margin-left:10px;">This model is used for simulating the behavior of three genetic circuits we designed</P></DIV><DIV id="leftul"><DIV class="leftul"><DIV id="accordion"><H4>Overview</H4><DIV class="pane"><A href="https://2011.igem.org/Team:ZJU-China/Project">Summary</A><A href="https://2011.igem.org/Team:ZJU-China/Project/Stratifiedsystem">Background</A><A href="https://2011.igem.org/Team:ZJU-China/Project/Result">Results</A><A href="https://2011.igem.org/Team:ZJU-China/Project/Future">Future Application</A></DIV><H4 class="current">Modeling</H4><DIV class="pane" style="display:block;"><A href="https://2011.igem.org/Team:ZJU-China/Modeling">|Biofilm formation</A><A href="https://2011.igem.org/Team:ZJU-China/Modeling/Biobrick">|Biobrick</A><A href="#">|Results</A></DIV><H4>Results</H4><DIV class="pane"><A href="https://2011.igem.org/Team:ZJU-China/Datapage">Data page</A><A href="https://2011.igem.org/Team:ZJU-China/Parts">Parts</A></DIV></DIV></DIV></DIV><DIV class="bigblock" id="m_biofilm"><DIV class="block" id="introduction"><H3><FONT color="white">Introduction</FONT></H3><P id="intro"><STRONG>abstract:</STRONG>  We will model our gene regulatory networks using Michaelis-Menten enzymatic kinetics,together with the usual rules of reaction kinetics. The resulting models, when spatial
effects are neglected, are given in terms of ordinary differential equations describing the
rate of change of the concentrations of gene products and proteins. A key component of all
these models is the Hill function, used to describe the transcription phase. The presence of
this highly nonlinear function, whilst accurately modeling the network, inevitably leads to
restrictions on the analytical tools available to understand and predict the
dynamics.</P><DIV class="block" id="compartment"><H3>Basic concepts and assumptions</H3><P>The ODE formalism models the concentrations of RNAs, proteins, and other molecules by
time-dependent variables with values contained in the set of nonnegative real numbers.
Regulatory interactions take the form of functional and differential relations between the
concentration variables. For a typical transcription-translation process, the ODEs modeling
approach associates two ODEs with any given gene<STRONG> i</STRONG>; one modeling the rate of change of the
concentration of the transcribed mRNA<STRONG> r_i</STRONG>, and the other describing the rate of change of
the concentration of its corresponding translated protein<STRONG> p_i</STRONG>. Thus for our network with 3
genes we have:
</P><P>Where (1) describes transcription, (2) describes translation, and i = 1,…,N. The functions<STRONG> R_i</STRONG>(<STRONG>p_j</STRONG>)
describe the dependence of mRNA concentration on protein concentration<STRONG> p_j</STRONG> (If protein<STRONG> p_j </STRONG>has
no effect on mRNA<STRONG> r_i</STRONG>, then correspond function is set to zero.) The functional F(·) in (1) is
defined in terms of sums and products of functions <STRONG>R_i</STRONG>. Function <STRONG>P_i</STRONG> in (2) describes the
translation of the mRNA<STRONG> r_i </STRONG>into a protein<STRONG> p_i</STRONG>. Parameters<STRONG> γ_i</STRONG>, <STRONG>δ_i</STRONG> (i = 1,…,N), represent the
degradation parameters of the mRNAs and proteins produced by gene<STRONG> i</STRONG>. As is common, we shall
assume that the degradation of proteins or mRNAs is not regulated, namely that it does not
depend on the concentrations of other molecules in the cell. Function <STRONG>R_i</STRONG> is assumed to be in the
form of Hill function as usual (since our cases are all inhibitors, we shall denote the Hill function
<STRONG>h</STRONG>-(p,K,n)), and the function <STRONG>P_i</STRONG> is taken to be a linear term proportional to the concentration of
mRNA <STRONG>r_i</STRONG>.
</P><P>where<STRONG> K_i </STRONG>is the microscopic dissociation constant, and<STRONG> n_i </STRONG>is Hill coefficient, describing cooperativity.
</P><A style="float:right;" id="p_model"> &gt;&gt;Model </A></DIV></DIV><DIV class="block" id="mmodeling"><H3>Equations&amp;Parameters</H3><P>Based on the above, our model could be founded with following ODEs:
</P><P>Where</P><P><STRONG>Parameters</STRONG></P><P>Variables are defined in following table.</P><P>  We should note that<STRONG> α </STRONG>and<STRONG> β </STRONG>are two functions of the oxygen concentration, which are
determined by the properties of corresponding promoter. For a certain depth of the biofilm, the
concentration of oxygen is a constant in our model, so are  and  . Therefore we could solve
these equations at different oxygen concentration and combine all the results to show how this
system work.</P><P>  Because of the nonlinearity of the Hill functions, the solutions of a system of ordinary differential
equations of a network of many genes cannot generally be determined by analytical means.</P></DIV><DIV class="block" id="sr"><H3><FONT color="white">Equilibrium analysis</FONT></H3><DIV class="block"><P>Notice that we care more about the final state of the bacteria in different depth of the
biofilm(thus in different oxygen condition), we assume that the system has reached a steady
state. We could calculus this steady state by setting all derivatives with respect to time to zero.
That is, let
</P><P>With (4)-(10), we have:
</P><P>Clearly, the behavior of this system is determined by the two functions, thus determined by the
behavior of promoter vgb &amp; promoter fdhF. On the other hand, the solutions of equations
The two functions <STRONG>α</STRONG>and <STRONG>β</STRONG> are hard to determine precisely, but their behavior could be
illustrated in follow graph qualitatively.</P></DIV><DIV class="block"><P>First case, When the expression of promoter vgb and fdhF could be neglected, that is, we could
set<STRONG>α</STRONG>and<STRONG>β</STRONG>to be both zero. The ODEs system become
</P><P>The results are shown as follow.</P><P> </P><P> </P></DIV><DIV class="block"><P>Secondly,in the anaerobic condition, that is, when the expression of promoter vgb could be
neglected, we set<STRONG>α</STRONG>to be zero, and set<STRONG>β</STRONG>to be bigger than the dissociation constant in (13).
Then we have
</P><P>The results are shown as follow</P><P> </P><P> </P><P> </P></DIV><DIV class="block"><P>Thirdly, since our ODEs model are symmetric respect to YFP &amp; CFP in some sense, therefore the
case of microaerobic (where we could set<STRONG>β</STRONG>to be zero) would be the same(symmetric) as in
second case.
</P><P>and results:</P><P> </P><P> </P><P> </P></DIV><DIV class="block"><H3>Robustness analysis</H3><P>  Robustness is very important to a genetic circuit. Scientists hope their artificial system have high tolerance to the variety of environment or system parameters. In our genetic circuit,   and   are directly represent the properties of Vgb and Fdhf, so we are interested in to what degree these two parameters could influence the behavior of Ptet.</P><P>  We could show the contribution of the two parameters   and   by fix one of them, and let the other one varies linearly. When fix   at 1e-9, and    changing from 0 to 1e-7 continuously, the production of each protein is shown in following graphs.</P><P>Cfp.figure</P><P>Yfp.figure</P><P>Rfp.figure</P><P>Also, we could let both   and   to vary linearly (from 0 to 1e-11) to see how protein CFP changes. The actual curve with respect to depth of biofilm for CFP production is a curve on this surface.</P></DIV><DIV class="block"><H3>Results</H3><P>combine Data acquired in part characterization and modeling of biofilm and biobrick, we can get the relationship between distribution of oxygen in the biofilm and the expression rate of CFP RFP YFP as follow:</P><P>YFP:</P><P>RFP:</P><P>From the fitting results we know that:
The PoPS of YFP could be regard as zero since PPO of 10%, and approximately linearly increasing from 0% to 2.5%, and decreasing from 2.5% to 10%.

The PoPS of RFP could be regard as zero since PPO of 2%, and approximately linear from 0% to 2%.
Together with the results of former modeling sections, we could show the stratified biofilm in following graph.
</P><P> </P></DIV></DIV><DIV class="block" id="reference"><H3>Reference</H3><TABLE style="background-color:transparent;" width="0" border="0" cellspacing="1" cellpadding="1"><TBODY><TR><TD><P>[1]</P></TD><TD><P>J.B. Andersen et al., “New Unstable Variants of Green Fluorescent Protein for Studies of Transient Gene Expression in Bacteria,” Applied and Environmental Microbiology, vol. 64, Jun. 1998, pp. 2240–2246.</P></TD></TR><TR><TD><P>[2]</P></TD><TD><P>Goryachev, A.B., D.J. Toh and T. Lee. “System analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constant.” BioSystems 2006: 83, 178-187.</P></TD></TR><TR><TD><P>[3]</P></TD><TD><P>Alon, Uri. “An Introduction to Systems Biology Design Principles of Biological Circiuts.” London: Chapman &amp; Hall/CRC, 2007.</P></TD></TR><TR><TD><P>[4]</P></TD><TD><P>David Braun et al. Parameter estimation for two synthetic gene networks: A case study. IEEE 2005.</P></TD></TR><TR><TD><P>[5]</P></TD><TD><P>Aberdeen_Scotland 2009. “Modeling Parameters” iGEM wiki.</P></TD></TR><TR><TD><P>[6]</P></TD><TD><P>Wilfried Weber, Markus Rimann, Manuela Spielmann, Bettina Keller, Marie Daoud-El Baba, Dominique Aubel, Cornelia C Weber &amp; Martin Fussenegger, Gas-inducible transgene expression in mammalian cells and mice, Nature Biotechnology,
volume 22, number 11, November 2004</P></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="footer" id="footer"><A href="https://2011.igem.org/"> iGem 2011 Home Page</A><A href="http://ung.igem.org/Special:Upload/"> Upload Files</A><A href="https://2011.igem.org/wiki/index.php?title=Template:Zjucss_main&amp;action=edit">Edit CSS</A><A href="http://ung.igem.org/Team_Wikis?year=2011"> Team Wikis</A><A href="mailto:zjuigem@gmail.com?"><STRONG>Contact Us</STRONG></A></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:ZJU-China/Modeling/Biobrick">http://2011.igem.org/Team:ZJU-China/Modeling/Biobrick</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ZJU-China/Modeling/Biobrick" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ZJU-China/Modeling/Biobrick" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ZJU-China/Modeling/Biobrick&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ZJU-China/Modeling/Biobrick&amp;oldid=232793" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>