<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Kingsborough_NY_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Kingsborough NY/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="kbcc"><UL><LI><A href="https://2016.igem.org/Team:Kingsborough_NY">HOME</A></LI><LI><A href="https://2016.igem.org/Team:Kingsborough_NY/Problem">PROBLEM</A></LI><LI><A href="https://2016.igem.org/Team:Kingsborough_NY/Pathway">PATHWAY</A></LI><LI><A href="https://2016.igem.org/Team:Kingsborough_NY/Design">DESIGN</A></LI><LI><A href="https://2016.igem.org/Team:Kingsborough_NY/Results">RESULTS</A></LI><LI><A href="https://2016.igem.org/Team:Kingsborough_NY/Outreach">OUTREACH</A></LI><LI><A href="https://2016.igem.org/Team:Kingsborough_NY/Attributions">TEAM</A></LI></UL></DIV><H2><SPAN class="mw-headline" id="DNA_Design">DNA Design</SPAN></H2><P>DNA was designed from K. stuttgarteinsis protein sequences corresponding to the gamma, beta, and alpha subunits of Hydrazine Synthase (GenBank accessions CAJ73611.1<A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/protein/91200562">[1]</A> CAJ73612.1<A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/protein/91200563">[2]</A> and CAJ73613.1<A rel="nofollow" class="external autonumber" href="https://www.ncbi.nlm.nih.gov/protein/91200564">[3]</A>, respectively). This DNA was reverse translated first using the Sequence Manipulation Suite<A rel="nofollow" class="external autonumber" href="http://www.bioinformatics.org/sms2/">[4]</A>, and ultimately using software developed by our team.
</P><P>The program we developed takes FASTA formatted protein sequence(s) in a single file and reverse translates them to DNA, using codons preferred by E. coli. Furthermore, the program will eliminate any of the standard BioBrick restriction sites from the DNA sequence (i.e. EcoRI, XbaI, SpeI, and PstI), while preserving the encoded information. Finally, the sequence will be changed (again, while preserving the encoded information) so as to eliminate any repetitive blocks of 8 nucleotides; that is, any pattern of 8 nucleotides that occurs more than once in the sequence. This last step is performed to satisfy a requirement imposed by IDT during synthesis.
</P><P>The program, <A rel="nofollow" class="external text" href="https://2016.igem.org/File:T--Kingsborough_NY--protein2bioBrickZip.zip">protein2bioBrick</A>, is freely available in this zip package, that includes a stand-alone executable JAR file, the source code, and some example data. Your computer must have some Java package already installed in order to use the program.
</P><H2><SPAN class="mw-headline" id="Restriction_Enzyme_Cloning">Restriction Enzyme Cloning</SPAN></H2><P>Using the DNA designed above and ordered from our gracious sponsor, Integrated DNA Technologies, we proceeded to clone the three HZS genes into the standard BioBrick vector, pSB1C3, through routine restriction enzyme digestion and ligation.
</P><P>Our strategy for this initial step and subsequent cloning steps was to take advantage of the standard BioBrick restriction sites, EcoRI, XbaI, SpeI, and PstI. Our apparently unique strategy involved a triple digest to circumvent the need for gel purification of the desired fragment. This strategy is highlighted in the following diagram:
</P><P>The top panel depicts two standard BioBrick plasmids, both with the same antibiotic resistance. For example, this could be combining a promoter (like R0011) with one of our composite parts, K1952003 (RBS + HZS-gamma). The first plasmid is cut with EcoRI, SpeI, and PstI. The second plasmid is cut with EcoRI, XbaI, and PstI.
</P><P>A quick and routine purification of DNA from a restriction digest, using QIAgen spin columns for example, will eliminate the very small SpeI-PstI fragment in the first digest, or the small EcoRI-XbaI fragment in the 2nd digest).
</P><P>In both cases, empty vector is generated with EcoRI and PstI overhangs. However, the only viable product of a ligation and transformation will be the vector that contains both BioBricks ligated together. In summary, this strategy uses a triple digest and takes advantage of the fact that most commercial spin columns do not capture small fragments like the EcoRI-XbaI fragment, which would have interfered with the ligation if not removed (however, gel purification is not necessary here).
</P><H2><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H2><P>We also developed a strategy to use Gibson Assembly, both to clone individual HZS subunits fused to a reporter gene and to generate a three-protein operon that will produce the entire protein. You can view this strategy in <A rel="nofollow" class="external text" href="https://2016.igem.org/File:T--Kingsborough_NY--gibson_strategy.pdf">PDF</A> form, in which homologous DNA sequences are represented by blocks of the same color.
</P></DIV></DIV></DIV></DIV></DIV></BODY></HTML>