"
Page
Discussion
View source
History
teams
Log in
 
Team:UNIPV-Pavia/Project/IntYeast
From 2010.igem.org
Home
The Project
Materials & Methods
Parts
The Team
Notebook
Biological Safety
Pavia
Gallery
Sponsors
ProteInProgress: a cellular assembly line for protein manufacturing
Implementation and Results: Integrative standard vector for yeast
Self-inducible promoters
Integrative standard vector for E. coli
Integrative standard vector for yeast
Purification of proteins
Integrative standard vector for yeast
Materials and Methods
Strain: the S. cerevisiae S288C strain (<partinfo>BBa_K300979</partinfo>, genotype: MATα ρ° trp1-0) was purchased from Open Biosystems.
Construction of BBa_K300001:
<partinfo>BBa_K300980</partinfo>, provided by Mr Gene DNA synthesis service (www.mrgene.com), was excised from its original shipping vector (pMA) through digestion with MfeI (Fermentas) and NsiI (Fermentas) restriction enzymes. It was then isolated through a 1% agarose gel electrophoresis and gel-extracted (Macherey-Nagel NucleoSpin Extract II).
<partinfo>BBa_I763007</partinfo>, available in the Registry, was digested with EcoRI-PstI (Roche), run on agarose gel and its <partinfo>pSB1A2</partinfo> vector was gel-extracted as described above.
Digested <partinfo>BBa_K300980</partinfo> and <partinfo>pSB1A2</partinfo> have compatible ends (EcoRI-MfeI and PstI-NsiI). They were ligated with T4 Ligase (Roche) and transformed into competent TOP10 E. coli strain (<partinfo>BBa_V1009</partinfo>) which were then plated on Ampicillin (100 ug/ml) plates.
Positive transformants were identified by colony PCR with VF2 (<partinfo>BBa_G00100</partinfo>) and VR (<partinfo>BBa_G00101</partinfo>) standard primers and by restriction mapping with EcoRI (Roche) or NsiI (Fermentas). The yielded plasmid had <partinfo>BBa_B0033</partinfo> flanked by EcoRI and SpeI.
The yielded plasmid was then digested with EcoRI-SpeI (Roche) and <partinfo>BBa_K300007</partinfo> (digested with EcoRI-SpeI as well) was assembled in the vector, thus yielding <partinfo>BBa_K300001</partinfo>-<partinfo>BBa_K300007</partinfo> (i.e. there is the <partinfo>BBa_K300007</partinfo> part in the cloning site).
Yeast transformation:
S288C strain was inoculated in 5 ml of YPD from a long term 15% glycerol stock and grown for 24h (30°C, 200rpm).
The culture was diluted 1:10 in 50 ml of pre-warmed YPD in a 250 ml flask and was grown for additional 4 hours under the same conditions as before.
Cells were pelleted (4000 rpm, 5 min) and resuspended in 25 ml of deionized water.
Cells were pelleted (4000 rpm, 5 min), the supernatant was discarded and the pellet was resuspended in 1 ml of deionized water and transferred into a 1.5 ml tube.
Cells were pelleted (4000 rpm, 30 sec), the supernatant was discarded and the pellet was resuspended in deionized water to a final volume of 1 ml (vortex mix vigorously).
Three 100 ul aliquots were transferred into 1.5 ml tubes, while the remaining 600 ul of cells were not used in this protocol.
The three tubes were centrifuged (4000 rpm, 30 sec) and the supernatant discarded.
Each of the three pellets were resuspended (vortex mix vigorously) in 360 ul of transformation mix (240 ul of PEG 3350 50% w/v, 36 ul of LiAc 1.0 M, boiled salmon sperm DNA, 34 ul of linearized plasmid DNA plus water). The salmon sperm DNA was boiled for 5 min and pre-chilled before adding it in the transformation mix. The plasmid DNA was previously digested with SbfI (Fermentas), purified with the NucleoSpin Extract II kit (MN) and quantified with the NanoDrop in order to add 1 ug of DNA to the transformation mix.
The tubes were heated at 42°C for 40 min.
Cells were pelleted (4000 rpm, 30 sec), the supernatant was removed by pipetting and the pellet was gently resuspended in 1 ml of deionized water.
Cells were pelleted (4000 rpm, 30 sec), the supernatant was discarded, the pellet was resuspended in 1 ml of YPD and incubated at 30°C, 200 rpm for 3 hours.
Cells were pelleted (4000 rpm, 30 sec), resuspended in 200 ul of YPD and plated on a YPD agar plate with G418 antibiotic at 200 ug/ml.
The plates were incubated at 30°C for about 3 days until colonies appeared.
The integration efficiency was estimated as the colony forming units (CFUs) yielded for each ug of DNA.
Protocol references:
[1] http://openwetware.org/wiki/High_Efficiency_Transformation
[2] Guldener U, Heck S, Fiedler T, Beinhauer J, Hegemann JH (1996), A new efficient gene disruption cassette for repeated use in budding yeast. Nucleic Acids Research, Vol. 24, No. 13 2519–2524.
Results
The transformed inserts and their integration efficiency in S288C are listed here:
SbfI-digested plasmid
ug of transformed DNA
# of colonies
Estimated integration efficiency [CFU/ug]
<partinfo>BBa_K300001</partinfo>-<partinfo>BBa_K300006</partinfo>
1
6500
6.5*10^3
<partinfo>BBa_K300001</partinfo>-<partinfo>BBa_K300007</partinfo>
1
1700
1.7*10^3
no DNA
0
0
0
The colony count was quite difficult (see pictures), but it is useful to have an estimation of the integration efficiency.
S288C transformed with no DNA (negative control).
S288C transformed with linearized <partinfo>BBa_K300001</partinfo>-<partinfo>BBa_K300006</partinfo>.
S288C transformed with linearized <partinfo>BBa_K300001</partinfo>-<partinfo>BBa_K300007</partinfo>.
Discussion
The obtained results suggest that the integrative vector actually works and that the selection marker is highly specific (no colonies appeared on the "no DNA" plate).
Although this integrative vector has already given promising integration results, a lot of work still remains to do:
the correct phenotype of the S288C bearing these parts has to be validated (by mOrange fluorescence measurement for the <partinfo>BBa_K300007</partinfo> part);
the actual integration position has to be confirmed by PCR with ad hoc-designed primers;
the KanMX selection marker excision has to be validated.
The compatibility with the BioBrick standard (RFC10, RFC12 and RFC23) enables the engineering the chromosomes of S. cerevisiae by integrating the desired biological functions following a standard procedure. Moreover, the modularity of the designed integrative vector enables the modification of the integration target site, in fact the default Homologous Regions can only target the Gal genomic system and users may need to target different DNA sequences.
As reported for the integrative vector for E. coli, the main information about the usage of this integrative vector for yeast is shared in the Registry. We hope to have positively contributed to the diffusion of standard biological parts for yeast, a very interesting chassis for synthetic biology for which only a limited number of parts are available in the Registry and only a few of them has been exaustively characterized.
^top
Retrieved from " http://2010.igem.org/Team:UNIPV-Pavia/Project/IntYeast "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
