<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNIPV-Pavia_Parts_Characterization_NewParts"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNIPV-Pavia/Parts/Characterization/NewParts">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNIPV-Pavia/Parts/Characterization/NewParts&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Parts/Characterization/NewParts&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Parts/Characterization/NewParts&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNIPV-Pavia/Parts/Characterization/NewParts" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNIPV-Pavia/Parts/Characterization/NewParts</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><TABLE width="100%" border="0"><TBODY><TR align="left" valign="top"><TD height="20" valign="top"><A href="https://2010.igem.org/Team:UNIPV-Pavia"> Home </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Project"> The Project </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Material_Methods"> Materials &amp; Methods </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Parts"> Parts </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Team"> The Team </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Notebook"> Notebook </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Biological_Safety"> Biological Safety </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Pavia"> Pavia </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Gallery"> Gallery </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Sponsor"> Sponsors </A></TD></TR></TBODY></TABLE><TABLE border="0" align="center" width="100%"><TBODY><TR><TD align="justify" valign="top" style="padding:20px" width="100%"><P align="center"><FONT size="4"><B>NEW PARTS</B></FONT></P></TD></TR><TR><TD align="center" width="25%"><P><A href="https://2010.igem.org/Team:UNIPV-Pavia/Parts/Characterization" class="external text" rel="nofollow">Return to Characterization</A></P></TD><TH align="center" width="25%"><P><A href="https://2010.igem.org/Team:UNIPV-Pavia/Parts/Characterization/NewParts" class="external text" rel="nofollow">New Parts</A></P></TH><TH align="center" width="25%"><P><A href="https://2010.igem.org/Team:UNIPV-Pavia/Parts/Characterization/RebExistingParts" class="external text" rel="nofollow">Improved Parts</A></P></TH><TH align="center" width="25%"><P><A href="https://2010.igem.org/Team:UNIPV-Pavia/Parts/Characterization/ExistingPartsRegistry" class="external text" rel="nofollow">Existing Parts from the Registry</A></P></TH></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="New_Parts:_list">New Parts: list</SPAN></H1><UL><LI><A href="/Team:UNIPV-Pavia/Parts/Characterization/NewParts#BBa_K300000_-_BioBrick_integrative_base_vector_for_E._coli" title="Team:UNIPV-Pavia/Parts/Characterization/NewParts">BBa_K300000 - BioBrick integrative base vector for E. coli</A></LI><LI><A href="/Team:UNIPV-Pavia/Parts/Characterization/NewParts#BBa_K300001_-_BioBrick_integrative_base_vector_for_S._cerevisiae" title="Team:UNIPV-Pavia/Parts/Characterization/NewParts">BBa_K300001 - BioBrick integrative base vector for S. cerevisiae</A></LI><LI><A href="/Team:UNIPV-Pavia/Parts/Characterization/NewParts#BBa_K300010_-_PoPS-based_self-inducible_device" title="Team:UNIPV-Pavia/Parts/Characterization/NewParts">BBa_K300010 - PoPS-based self-inducible device</A></LI><LI><A href="/Team:UNIPV-Pavia/Parts/Characterization/NewParts#BBa_K300093.2C_BBa_K300094.2C_BBa_K300097.2C_BBa_K300095.2C_BBa_K300086_and_BBa_K300084_-_Phasin_and_Intein-based_tags_for_protein_purification" title="Team:UNIPV-Pavia/Parts/Characterization/NewParts">BBa_K300093, BBa_K300094, BBa_K300097, BBa_K300095, BBa_K300086 and BBa_K300084 - Phasin and Intein-based tags for protein purification</A></LI></UL><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EBBa_K300000.3C.2Fpartinfo.3E_-_BioBrick_integrative_base_vector_for_E._coli">&lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt; - BioBrick integrative base vector for <I>E. coli</I></SPAN></H1><H2><SPAN class="mw-headline" id="Materials_and_Methods">Materials and Methods</SPAN></H2><P><B>Plasmids and strains:</B> the &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; helper plasmid was kindly given by Prof. JC Anderson (UC Berkeley). BW23474 (&lt;partinfo&gt;BBa_K300985&lt;/partinfo&gt;), MC1061 (&lt;partinfo&gt;BBa_K300078&lt;/partinfo&gt;) and MG1655 (&lt;partinfo&gt;BBa_V1000&lt;/partinfo&gt;) E. coli strains and the pCP20 helper plasmid were purchased from the Coli Genetic Stock Center (Yale University).
</P><P><B>Verification primers:</B> all the oligonucleotides were purchased from Primm (San Raffaele Biomedical Science Park, Milan, Italy). The P1 (&lt;partinfo&gt;BBa_K300975&lt;/partinfo&gt;) and P4 (&lt;partinfo&gt;BBa_K300978&lt;/partinfo&gt;) primers had already been used in [Anderson JC et al., 2010]. The P2 (&lt;partinfo&gt;BBa_K300976&lt;/partinfo&gt;) and P3 (&lt;partinfo&gt;BBa_K300977&lt;/partinfo&gt;) primers have been newly designed using ApE and Amplify 3X. P2 and P3 have been designed also considering the previously used verification primers P2 and P3 in the pG80ko integrative plasmid, described in [DeLoache W, 2009].
</P><P>The relative position of the P1, P2, P3 and P4 primers is shown in Fig.1:
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 1: Relative position of the verification primers. a) no integrants; b) single integrant and c) integrant with multiple tandem copies. P1/P2 and P3/P4 pairs give an amplicon when at least one copy of the vector is integrated in the Phi80 locus. P2/P3 pair show an amplicon only when multiple tandem copies occur.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><B>Competent cells preparation:</B> all the <I>E. coli</I> strains were made competent following a slightly modified version of the protocol described in [Sambrook J et al., 1989]. Briefly, cells were grown to and OD600 of ~0.4-0.6, harvested (4000 rpm, 10 min, 4°C) and the supernatant discarded. Cells were resuspended in (30 ml for each 50 ml of initial culture) pre-chilled Mg-Ca buffer (80 mM MgCl2, 20 mM CaCl2), centrifuged as before and the supernatant discarded. Cells were resuspended in (2 ml for each 50 ml of initial culture) pre-chilled Ca buffer (100 mM CaCl2, 15% glycerol), aliquoted in 0.5 ml tubes and freezed immediately at -80°C. Test the transformation efficiency in Colony Forming Units (CFU)/ug of transformed DNA
</P><P>The Chloramphenicol concentration in plates was 34 ug/ml for the high copy plasmids, 12.5 ug/ml for the medium/low copy plasmids and 12.5 for the three control strains transformed with the R6K plasmid.
</P><P><B>Integration protocol:</B></P><OL><LI> Transform the &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; helper plasmid in the host strain (MC1061 or MG1655) and select transformants on Amp (50 ug/ml) plates under permissive conditions (30°C) overnight.
</LI><LI> Inoculate a single colony in selective LB and let the culture grow at 30°C, 220 rpm. When the culture reaches the OD600 of 0.4-0.6 prepare chemically competent cells.
</LI><LI> Transform the integrative vector with the desired insert in the BBa_J72008-containing strain and select co-transformants on Cm (34 ug/ml) plates under permissive conditions (30°C) overnight. At this temperature &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; can be replicated and so the pir protein product can be expressed in the cells. The pir product enables the propagation of the integrative vector by replicating the R6K origin.
</LI><LI> Inoculate a single colony in 5 ml of LB + Cm at 12.5 ug/ml and incubate the culture at 37°C, 220 rpm overnight. At this temperature the &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; helper cannot be replicated and the Phi80 integrase is expressed by the remaining copies of the helper. The bacteria that are able to grow in this selective medium should be correct integrants because the integrative vector cannot be replicated by the pir product anymore.
</LI><LI> Streak the culture on a Cm plate (at 12.5 ug/ml) and incubate it at 43°C overnight to ensure the loss of the helper plasmid. The bacteria that form colonies should be correct integrants without the &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; helper plasmid.
</LI></OL><P>Validate the loss of the helper plasmid by inoculating colonies in Cm (at 12.5 ug/ml) media and counterselecting them in Amp (at 50 ug/ml) media. Validate the correct integration position by performing colony PCR with primers P1/P2, P3/P4, P1/P4, P2,P3 and VF2/VR. Validate the phenotype (when possible).
</P><P>
Expected amplicon length [bp] when the vector is integrated into the Phi80 locus:
</P><TABLE border="1"><TBODY><TR><TD> </TD><TD><B>No integrant</B></TD><TD><B>Single integrant</B></TD><TD><B>Multiple tandem integrants (&gt;1)</B></TD></TR><TR><TD><B>VF2/VR</B></TD><TD>none
</TD><TD>280 + insert length
</TD><TD>280 + insert length
</TD></TR><TR><TD><B>P1/P4</B></TD><TD>546
</TD><TD>546 + insert length + 2171 (i.e. the BBa_K300000 length)
</TD><TD>546 + insert length + 2171 (i.e. the BBa_K300000 length)
</TD></TR><TR><TD><B>P1/P2</B></TD><TD>none
</TD><TD>452
</TD><TD>452
</TD></TR><TR><TD><B>P3/P4</B></TD><TD>none
</TD><TD>666
</TD><TD>666
</TD></TR><TR><TD><B>P2/P3</B></TD><TD>none
</TD><TD>none
</TD><TD>572
</TD></TR></TBODY></TABLE><P><B>Marker excision protocol:</B></P><OL><LI> Inoculate an integrant in selective LB medium and let it grow to OD600=0.4-0.6. Prepare chemically competent cells.
</LI><LI> Transform the pCP20 helper plasmid in the competent strain and select transformants on Amp (100 ug/ml) plates under permissive conditions (30°C) overnight. At this temperature the pCP20 can be replicated. The pCP20 plasmid contains Amp and Cm resistance markers, a thermoinducible Flp recombinase expression system and a heat-sensitive replication origin. The permissive temperatures for the pCP20 propagation are the same as &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt;.
</LI><LI> Inoculate a single colony in 5 ml of LB without antibiotic and incubate the culture at 37°C, 220 rpm overnight. At this temperature the pCP20 helper cannot be replicated and the Flp recombinase is expressed by the remaining copies of the helper. The bacteria should loose the R6K origin and the Cm resistance upon FRT sites recombination, mediated by Flp.
</LI><LI> Streak the culture on a LB plate and incubate it at 43°C overnight to ensure the loss of the helper plasmid. The bacteria that form colonies should be without the pCP20 helper plasmid.
</LI></OL><P>Validate the loss of the helper plasmid by inoculating colonies in Amp (at 100 ug/ml) media and validate the loss of the Cm resistance from the genome by inoculating colonies in Cm (at 12.5 ug/ml) media. Validate the correct length of the integrated part without Cm resistance and R6K origin by performing colony PCR with primers P1/P4 (which amplify the entire Phi80 locus) and VF2/VR (which amplify the integrated part). Validate the phenotype (when possible).
</P><P><B>Colony PCR:</B> a single colony or 1 ul of culture was added to the Invitrogen Platinum Taq reaction mix and was heated at 94°C for 10 min. Then it was assayed with this cycle (X 35): 94°C 30 sec, 60°C (for VF2/VR) or 63°C (for the other primers) 30 sec, 72°C according to the amplicon expected length (1Kb/min). Then the reaction was kept at 72°C for 10 min and it was run on a 1% agarose gel with the GeneRuler 1Kb Plus DNA ladder (Fermentas).
</P><P><B>Fluorescence assays:</B> integrants were inoculated in 1 ml of M9 + Cm (12.5 ug/ml) and grown at 37°C, 220 rpm overnight. The cultures were diluted 1:100 in 2 ml of selective M9 and let grow for about 4-6 hours under the same conditions as before. Three 200 ul aliquots for each culture were transferred to a 96-well microplate and assayed in the Infinite F200 microplate reader (Tecan) for about 20 hours with the following kinetic cycle: 37°C, 5 min sampling time, linear shaking 15 sec (amplitude=3), wait 5 sec, measure absorbance at 600nm, measure fluorescence with the proper filter (EX:nm/EM:540nm for GFP or EX:535nm/EM:620nm for RFP) with gain=70. The same protocol was followed for the MC1061 and the MG1655 non-integrant strains, which were grown in M9 without antibiotic.
</P><P><B>Data analysis:</B> the absorbance measurements were normalized by subtracting the absorbance of the M9, while the fluorescence measurements were normalized by subtracting the fluorescence of the non-integrant strains over time. For each well, the S<SUB>cell</SUB> signal (proportional to the reporter protein synthesis rate per cell) was computed as (1/OD600*dXFP/dt), where OD600 is the normalized absorbance and XFP is the normalized fluorescence. The S<SUB>cell</SUB> signal was then averaged over time to obtain a single value for each well. Results are presented as the average S<SUB>cell</SUB> with their 95% confidence intervals of the mean.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Integration_of_the_desired_BioBrick_part_into_the_Phi80_genome_locus">Integration of the desired BioBrick part into the Phi80 genome locus</SPAN></H3><P>MC1061 and MG1655 were chosen as host strains for integration. &lt;partinfo&gt;BBa_K173001&lt;/partinfo&gt; (constitutive strong promoter with GFPmut3) and the EcoRI-PstI fragment of &lt;partinfo&gt;BBa_J61002&lt;/partinfo&gt;-&lt;partinfo&gt;BBa_J23101&lt;/partinfo&gt; (here called PconRFP - constitutive strong promoter with RFP) were chosen as two proof of concept BioBrick parts to test the integration capability of the &lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt; vector in the Phi80 genome locus of these strains. For this reason, &lt;partinfo&gt;BBa_K173001&lt;/partinfo&gt; and PconRFP were ligated in &lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt; (digested with EcoRI-PstI) and propagated using BW23474.
The integration protocol was performed as described in the Materials and Methods section for 4 different combination:
</P><TABLE border="1"><TBODY><TR><TD><B>Integrant name</B></TD><TD><B>Strain</B></TD><TD><B>Insert of &lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt;</B></TD></TR><TR><TD>MC-GFP
</TD><TD>MC1061
</TD><TD>&lt;partinfo&gt;BBa_K173001&lt;/partinfo&gt;</TD></TR><TR><TD>MC-RFP
</TD><TD>MC1061
</TD><TD>PconRFP
</TD></TR><TR><TD>MG-GFP
</TD><TD>MG1655
</TD><TD>&lt;partinfo&gt;BBa_K173001&lt;/partinfo&gt;</TD></TR><TR><TD>MG-RFP
</TD><TD>MG1655
</TD><TD>PconRFP
</TD></TR></TBODY></TABLE><P>Three colonies grown after the overnight incubation at 43°C (step 5 of integration protocol) were analyzed for each plate. These 12 clones were called: MC-GFP-A,B,C , MC-RFP-A,B,C , MG-GFP-A,B,C and MG-RFP-A,B,C.
</P><P><B>Validation of the loss of BBa_J72008:</B> all the picked colonies did not grow in Amp (50 ug/ml) media, thus validating that &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; Amp-resistant helper had been actually cured from the cells. However, one of these 12 clones (MG-GFP-A) also failed to grow in Cm (12.5 ug/ml) liquid media, probably because of a mistake in its inoculation. We decided not to consider this clone and to continue with 11 clones.
</P><P><B>Validation of the actual integration site:</B> colony PCR was performed for all the 11 clones, using MC1061 and MG1655 without integrants as negative controls. Primer pairs P1/P2 and P3/P4 were used to validate the presence of the integrative vector in the Phi80 genomic locus, while the primer pair P2/P3 was used to validate the presence of multiple tandem integrants (see Fig.1 in Materials and Methods).
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 2: colony PCR with P1/P2 on all the 11 integrant clones. The blank is the reaction mix without bacteria. Expected amplicon for correct integrants: 452 bp.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 3: colony PCR with P3/P4 on all the 11 integrant clones. The blank is the reaction mix without bacteria. Expected amplicon for correct integrants: 666 bp.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 4: colony PCR with P2/P3 on all the 11 integrant clones. The blank is the reaction mix without bacteria. The lanes with the amplicon were expected to come from bacteria with multiple tandem integrants. Expected amplicon for multiple integrants: 572 bp.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
PCR results with primers P1/P2 and P3/P4 showed that each clone had the correct integrant in the correct genomic position (see Materials and Methods for a list of the expected amplicon lengths). Negative controls showed no amplicons with primers P1/P2 as expected, but showed an unexpected band with P3/P4. The reason of the presence of this band was not further investigated and the results with this primer pair cannot be a useful tool for future analysis. Anyway, the P1/P2 primer pair can be sufficient to successfully validate the presence of the DNA of interest in the Phi80 genomic locus.
</P><P>PCR results with primers P2/P3 showed that two clones (MC-GFP-B and MC-GFP-C) were single integrants, while all the other clones were multiple tandem integrants (i.e. the Phi80 locus contained more than one copy of the DNA of interest). Negative controls showed no amplicons, as expected.
</P><P><B>Validation of the integrants phenotype:</B> all the 11 clones were assayed as described in the Materials and Methods section. Unfortunately, the green fluorescent clones (MC-GFP-A,B,C and MG-GFP-B,C) did not show appreciable differences when compared to negative controls, most probably because the autofluorescence of the cells was too high and hid the GFP signal. For this reason, GFP clones were not considered for further analysis. Other instruments should be used to detect the GFP signal.
</P><P>On the other hand, RFP clones (MC-RFP-A,B,C and MG-RFP-A,B,C) all showed a higher fluorescence than the negative controls (see Fig.5). As Fig.5 show, the fluorescence of the three MG-RFP had a higher variability between clones when compared to the three MC-RFP. However, the clones were not necessarily expected to behave in the same way because all of them were multiple tandem integrants and the copy number of the PconRFP construct could be arbitrary.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 5: relative RFP synthesis rate for all the RFP expressing clones. Note: as a reference, typical values of the relative RFP synthesis rate measured for PconRFP in a low copy vector (~5 plasmids per cell) are about 6-7 fold higher (data not shown).</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Chloramphenicol_resistance_marker_excision">Chloramphenicol resistance marker excision</SPAN></H3><P>The marker excision was performed on two of the previously validated integrant strains: MC-RFP-A and MG-RFP-A (even if they were multiple tandem integrants).
</P><P>The marker excision protocol was performed as described in the Materials and Methods section for both strains, here named:
</P><TABLE border="1"><TBODY><TR><TD><B>Original name</B></TD><TD><B>Name after marker excision</B></TD></TR><TR><TD>MC-RFP
</TD><TD>MC-RFPflip
</TD></TR><TR><TD>MG-RFP
</TD><TD>MG-RFPflip
</TD></TR></TBODY></TABLE><P>
Three colonies grown after the overnight incubation at 43°C (step 4 of marker excision protocol) were analyzed for each plate. These 6 clones were called MC-RFPflip-A,B,C and MG-FRPflip-A,B,C.
</P><P><B>Validation of the loss of pCP20 and the resistance marker:</B>  all the 6 picked colonies failed to grow on both Amp (100 ug/ml) media and Cm (12.5 ug/ml) media. They could only grow in LB without antibiotics, thus validating that the pCP20 helper had been actually cured and the R6K-CmR DNA containing the Chloramphenicol selection marker had been actually eliminated.
</P><P><B>Validation of the length of the integrated part:</B> colony PCR was performed for all the 6 clones, using MC1061 and MG1655 without integrants as negative controls. Primer pairs VF2/VR and P1/P4 were used to validate if the <I>passenger</I> of interest was still present in the genome and the length of the entire Phi80 locus respectively after the marker excision.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 6: colony PCR with VF2/VR on all the 6 flipped clones. The blank is the reaction mix without bacteria. Expected amplicon for correct insert: 1.2 Kb.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 7: colony PCR with P1/P4 on all the 6 flipped clones. The blank is the reaction mix without bacteria. Expected amplicon for correct construct in the correct position: 2.3 Kb. Expected amplicon for the non-integrant strain MG1655: 546 bp.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
PCR results with primers VF2/VR showed that all the 6 clones still contain the <I>passenger</I> of interest, i.e. PconRFP, in the genome after the marker excision. The reaction blank, the MG1655 strain (neg control) and also the other samples showed some extra bands, but the ~1.2Kb amplicons of MC-RFP-A,B,C and MG-RFP-A,B,C had the correct length and was much brighter than the other bands.
</P><P>
PCR results with primers P1/P4 (Fig.7) showed that an amplicon of ~2.3Kb was present in all but one screened clones, while the MG1655 negative control showed the expected 546bp length for a non-integrant. MC-RFPflip-C did not show the P1-P4 amplicon because the reaction failed: the tube was damaged and the reaction mix was completely evaporated at the end of the PCR program. For this reason, a PCR was performed again on this clone (Fig.8).
</P><P>The ~2.3Kb amplicon was consistent with a single integrant of &lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt;-PconRFP without the R6K-CmR DNA fragment, thus validating the successful excision of the FRT-flanked DNA fragment containing R6K-CmR and confirming that PconRFP was still present in the correct locus in single copy.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 8: colony PCR with P1/P4 on MC-RFPflip-C clone. The blank is the reaction mix without bacteria. Expected amplicon for correct construct in the correct position: 2.3 Kb.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
These results showed that, even if the clones were multiple tandem integrants, they became single integrants after marker excision. This is because the Flp recombinase mediated the recombination of all the FRT sites of the multiple integrants until only a single FRT site was present in the Phi80 locus, thus leaving only the single integrant of interest without the selection marker in the genome.
</P><P><B>Validation of the marker-less phenotype:</B>all the 6 clones were assayed as described in the Materials and Methods section. They all showed a low variability and their fluorescence was lower than their two <I>parents</I>, i.e. MC-RFP-A for the MC1061 strains and MG-RFP-A for the MG1655 strains (see Fig.9). This result is consistent with the copy number of the PconRFP construct in the clones, in fact both MC-RFP-A and MG-RFP-A were multiple tandem integrants, while MC-RFPflip-A,B,C and MG-RFPflip-A,B,C were single integrants, as described above.
</P><P>All the MG-RFPflip showed a very low relative RFP synthesis rate when compared to the other strains, but the signal is systematically grater than the fluorescence of the negative control, thus validating the phenotype for the MG1655 strain. MC-RFPflip-A,B,C showed a higher fluorescence than MG-RFPflip-A,B,C.
</P><P>In conclusion, it has been demonstrated that, even after the marker excision process, the phenotype of the engineered cells is maintained.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 9: relative RFP synthesis rate for all the RFP-expressing clones after marker excision. In this figure, the bars corresponding to the fluorescence of the clones before marker excision is also reported to facilitate the comparison between them. Note that all the three <I>flip</I> clones are derived from MC-RFP-A for the MC1061 clones and from MG-RFP-A for the MG1655 clones.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>A novel integrative vector for <I>E. coli</I> has been successfully designed, constructed and used to integrate two proof of concept protein expression systems in two commonly used E. coli strains.
</P><P>The results showed that the vector is fully functional and can integrate into the correct targeted locus of the host chromosome through the Phi80 site-specific recombination system by using &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt;, an existing BioBrick helper plasmid from the Registry. In most cases, the integration occurs in tandem copies, probably because of the too high Chloramphenicol concentration used during the selection of integrants, which forces multiple integration of Cm-resistant constructs. This concentration was the same used during the pSC101 low copy plasmid (~5 copies per cell) selection. In some cases, it is desirable to have a single copy of the desired BioBrick in the genome, for example when the gene dosage is important. In [Haldimann A and Wanner BL, 2001] the usage of Chloramphenicol at 6 ug/ml yielded a very high percentage of single integrants. However, when tested in our lab, the MG1655 strain could survive on LB plates with Cm at 6 ug/ml and also at 8 ug/ml. For this reason a higher concentration of Cm was chosen for selection. Further studies should investigate the optimal antibiotic concentration to yield the highest single integrants percentage as possible.
</P><P>
The Flp/FRT mediated excision of the R6K and, most importantly, of the Cm resistance marker also worked by using the pCP20 helper plasmid. The estimated efficiency of this process was 100%. In addition, multiple tandem integrants became single integrants after the marker excision. This is because the Flp recombinase mediated the recombination of all the FRT sites of the multiple integrants until only a single FRT site was present in the Phi80 locus. The marker excision is a powerful tool to engineer microbial strains for industrial protein manufacturing because the engineered organism should not carry unsafe antibiotic resistances that may be diffused in the environment.
</P><P>
The fluorescence phenotype confirmed the correct integration into the <I>E. coli</I> chromosome. As expected, in general multiple integrants showed a higher fluorescence than the single integrants.
</P><P>
The BioBrick compatibility and the vector modularity give the possibility to the scientific community to stably engineer novel biological functions in <I>E. coli</I> with a very easy and user friendly methodology. A user’s handbook about the vector usage is shared in the Registry, as well as the users experiences and the compatibility information.
</P><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EBBa_K300001.3C.2Fpartinfo.3E_-_BioBrick_integrative_base_vector_for_S._cerevisiae">&lt;partinfo&gt;BBa_K300001&lt;/partinfo&gt; - BioBrick integrative base vector for <I>S. cerevisiae</I></SPAN></H1><P>The integration capability of this vector has been tested in S288C <I>S. cerevisiae</I> strain (&lt;partinfo&gt;BBa_K300979&lt;/partinfo&gt;). Here is reported the followed protocol and the obtained results.
</P><P><B>Protocol:</B></P><UL><LI>S288C strain (Open Biosystems) was inoculated in 5 ml of YPD from a long term 15% glycerol stock and grown for 24h (30°C, 200rpm).
</LI><LI>The culture was diluted 1:10 in 50 ml of pre-warmed YPD in a 250 ml flask and was grown for additional 4 hours under the same conditions as before.
</LI><LI>Cells were pelleted (4000 rpm, 5 min) and resuspended in 25 ml of deionized water.
</LI><LI>Cells were pelleted (4000 rpm, 5 min), the supernatant was discarded and the pellet was resuspended in 1 ml of deionized water and transferred into a 1.5 ml tube.
</LI><LI>Cells were pelleted (4000 rpm, 30 sec), the supernatant was discarded and the pellet was resuspended in deionized water to a final volume of 1 ml (vortex mix vigorously).
</LI><LI>Three 100 ul aliquots were transferred into 1.5 ml tubes, while the remaining 600 ul of cells were not used in this protocol.
</LI><LI>The three tubes were centrifuged (4000 rpm, 30 sec) and the supernatant discarded.
</LI><LI>Each of the three pellets were resuspended (vortex mix vigorously) in 360 ul of transformation mix (240 ul of PEG 3350 50% w/v, 36 ul of LiAc 1.0 M, boiled salmon sperm DNA, 34 ul of linearized plasmid DNA plus water). The salmon sperm DNA was boiled for 5 min and pre-chilled before adding it in the transformation mix. The plasmid DNA was previously digested with SbfI (Fermentas), purified with the NucleoSpin Extract II kit (MN) and quantified with the NanoDrop in order to add 1 ug of DNA to the transformation mix.
</LI><LI>The tubes were heated at 42°C for 40 min.
</LI><LI>Cells were pelleted (4000 rpm, 30 sec), the supernatant was removed by pipetting and the pellet was gently resuspended in 1 ml of deionized water.
</LI><LI>Cells were pelleted (4000 rpm, 30 sec), the supernatant was discarded, the pellet was resuspended in 1 ml of YPD and incubated at 30°C, 200 rpm for 3 hours.
</LI><LI>Cells were pelleted (4000 rpm, 30 sec), resuspended in 200 ul of YPD and plated on a YPD agar plate with G418 antibiotic at 200 ug/ml.
</LI><LI>The plates were incubated at 30°C for about 3 days until colonies appeared.
</LI></UL><P>
The integration efficiency was estimated as the colony forming units (CFUs) yielded for each ug of DNA.
</P><P>
Protocol references:
</P><P>[1] <A href="http://openwetware.org/wiki/High_Efficiency_Transformation" class="external free" rel="nofollow">http://openwetware.org/wiki/High_Efficiency_Transformation</A></P><P>[2] Guldener U, Heck S, Fiedler T, Beinhauer J, Hegemann JH (1996), A new efficient gene disruption cassette for repeated use in budding yeast. Nucleic Acids Research, Vol. 24, No. 13 2519–2524.
</P><P><B>Results:</B></P><P>The transformed inserts and their integration efficiency in S288C are listed here:
</P><TABLE border="1"><TBODY><TR><TD><B>SbfI-digested plasmid</B></TD><TD><B>ug of transformed DNA</B></TD><TD><B># of colonies</B></TD><TD><B>Estimated integration efficiency [CFU/ug]</B></TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300001&lt;/partinfo&gt;-&lt;partinfo&gt;BBa_K300006&lt;/partinfo&gt;</TD><TD>1
</TD><TD>1700
</TD><TD>1.7*10^3
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300001&lt;/partinfo&gt;-&lt;partinfo&gt;BBa_K300007&lt;/partinfo&gt;</TD><TD>1
</TD><TD>6500
</TD><TD>6.5*10^3
</TD></TR><TR><TD>no DNA
</TD><TD>0
</TD><TD>0
</TD><TD>0
</TD></TR></TBODY></TABLE><P>
These results suggest that the integrative vector actually works and that the selection marker is highly specific (no colonies appeared on the &quot;no DNA&quot; plate).
</P><P>The correct phenotype of the S288C bearing these parts has still to be validated (by mOrange fluorescence measurement for the &lt;partinfo&gt;BBa_K300007&lt;/partinfo&gt; part), as well as the actual integration position (by PCR).
</P><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EBBa_K300010.3C.2Fpartinfo.3E_-_PoPS-based_self-inducible_device">&lt;partinfo&gt;BBa_K300010&lt;/partinfo&gt; - PoPS-based self-inducible device</SPAN></H1><P>This is a PoPS-in/PoPS-out device.
</P><P>The luxR gene (&lt;partinfo&gt;BBa_C0062&lt;/partinfo&gt;) is constitutively produced by the &lt;partinfo&gt;BBa_R0040&lt;/partinfo&gt; promoter and it can activate the <I>lux pR</I> in presence of the autoinducer 3-oxo-C6-homoserine-lactone (3OC6HSL or simply HSL). The PoPS input regulates the production of luxI gene (&lt;partinfo&gt;BBa_C0061&lt;/partinfo&gt;). It encodes for the LuxI enzyme, which is able to produce HSL. The produced HSL can diffuse in the growth media of the cells that express LuxI. The <I>lux pR</I> produces a PoPS output when HSL reaches a critical concentration.
</P><P>This device can be specialized by assembling a promoter upstream and a promoterless expression system with the gene of interest downstream. When a cell population expresses LuxI, the concentration of HSL is an increasing function of cell culture density and so the induction of the <I>lux pR</I> promoter occurs only when the cells reach a threshold density.
</P><P>In this way, the upstream promoter autoinduces the production of the target protein at a critical culture density, depending on the HSL synthesis rate. The HSL synthesis rate can be tuned by assembling promoters of different strengths upstream of <I>luxI</I>.
</P><P>This enables the construction of a library of self-inducible devices capable of starting the target protein production at a predictable culture density.
</P><P>This device has been characterized in many different experimental conditions:
</P><UL><LI> varying the strength of the promoter controlling the production of the signal molecule (Sender Modulation);
</LI><LI> varying the copy number of vectors containing Sender and Receiver circuits;
</LI><LI> varying the growth medium (LB or M9).
</LI></UL><P>The results obtained are reported in the sections below.
</P><P>Notation:
</P><UL><LI>Sender: Pcon-RBS-luxI-TT
</LI><LI>Receiver: pTetR-RBS-luxR-TT- lux pR
</LI></UL><P>&lt;partinfo&gt;BBa_K300010&lt;/partinfo&gt; was assembled downstream of the constitutive promoters reported in the table, thus obtaining the following parts:
</P><TABLE border="1" align="center"><TBODY><TR><TD><B>BioBrick</B></TD><TD><B>Description</B></TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300017&lt;/partinfo&gt;</TD><TD>J23118
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300014&lt;/partinfo&gt;</TD><TD>J23110
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300015&lt;/partinfo&gt;</TD><TD>J23114
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300016&lt;/partinfo&gt;</TD><TD>J23116
</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300012&lt;/partinfo&gt;</TD><TD>J23105
</TD></TR></TBODY></TABLE><P>&lt;partinfo&gt;BBa_J23101&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_J23106&lt;/partinfo&gt; could not be cloned upstream of these devices because they produced amounts of LuxI protein that give a high metabolic burden for <I>E. coli</I>, so it was not possible to study all the combinations as transformants could not be obtained in some cases.
For each part, a measurement system was built, exploiting the production of the reporter gene GFP (Green Fluorescent Protein) to evaluate the &quot;switch on&quot; condition of every self-inducible promoter. Many different combinations were explored, in order to provide a library of promoters able to initiate transcription at the desired culture density.
</P><H3><SPAN class="mw-headline" id="Modulation_of_plasmid_copy_number"><B>Modulation of plasmid copy number</B></SPAN></H3><P>Sender and receiver devices were assembled in an unique part (such as &lt;partinfo&gt;BBa_K300017&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K300014&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K300015&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K300016&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K300012&lt;/partinfo&gt;) in high copy number plasmid &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt; or low copy number plasmid &lt;partinfo&gt;pSb4C5&lt;/partinfo&gt;. 
</P><H3><SPAN class="mw-headline" id="Results_2"><B>Results</B></SPAN></H3><P>The following measurement systems were realized assembling GFP downstream of each self-inducible device. The parts characterized are reported in this table:
</P><TABLE border="1" align="center" width="80%"><TBODY><TR><TD><B>Sender device</B></TD><TD><B>Sensor systems with GFP</B></TD><TD><B>Measurement Device</B></TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300030&lt;/partinfo&gt; in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;J23118
</TD><TD>&lt;partinfo&gt;BBa_T9002&lt;/partinfo&gt; in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K300024&lt;/partinfo&gt;in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300028&lt;/partinfo&gt; in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;J23110
</TD><TD>&lt;partinfo&gt;BBa_T9002&lt;/partinfo&gt; in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K300021&lt;/partinfo&gt;in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300029&lt;/partinfo&gt; in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;J23116
</TD><TD>&lt;partinfo&gt;BBa_T9002&lt;/partinfo&gt; in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K300022&lt;/partinfo&gt;in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300026&lt;/partinfo&gt; in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;J23105
</TD><TD>&lt;partinfo&gt;BBa_T9002&lt;/partinfo&gt; in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K300019&lt;/partinfo&gt;in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</TD></TR><TR><TD>xxxJ23114
</TD><TD>&lt;partinfo&gt;BBa_T9002&lt;/partinfo&gt; in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K300023&lt;/partinfo&gt;in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300030&lt;/partinfo&gt; in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;J23118
</TD><TD>&lt;partinfo&gt;BBa_T9002&lt;/partinfo&gt; in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K300024&lt;/partinfo&gt;in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300028&lt;/partinfo&gt; in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;J23110
</TD><TD>&lt;partinfo&gt;BBa_T9002&lt;/partinfo&gt; in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K300021&lt;/partinfo&gt;in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300029&lt;/partinfo&gt; in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;J23116
</TD><TD>&lt;partinfo&gt;BBa_T9002&lt;/partinfo&gt; in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K300022&lt;/partinfo&gt;in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;</TD></TR><TR><TD>&lt;partinfo&gt;BBa_K300026&lt;/partinfo&gt; in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;J23105
</TD><TD>&lt;partinfo&gt;BBa_T9002&lt;/partinfo&gt; in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;</TD><TD>&lt;partinfo&gt;BBa_K300019&lt;/partinfo&gt;in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;</TD></TR></TBODY></TABLE><P>Cultures of <I>E. coli</I> TOP10 bearing the plasmids containing the self-inducible devices expressing GFP were grown according to <A href="/Team:UNIPV-Pavia/Parts/Characterization#Microplate_reader_experiments_for_self-inducible_promoters_-_Protocol_.231" title="Team:UNIPV-Pavia/Parts/Characterization">this protocol</A> and all data collected were analyzed as explained in <A href="/Team:UNIPV-Pavia/Parts/Characterization#Data_analysis_for_self-inducible_promoters_.28initiation-treshold_determination.29" title="Team:UNIPV-Pavia/Parts/Characterization">this section</A>. An example of O.D.600 and fluorescence signals for a self-inducible device expressing GFP (&lt;partinfo&gt;BBa_K300026&lt;/partinfo&gt;), as well as its Scell signal and the estimated threshold value, is reported below.
</P><TABLE><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Growth curve of &lt;partinfo&gt;BBa_K300019&lt;/partinfo&gt; (O.D.600)</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fluorescence curve of &lt;partinfo&gt;BBa_K300019&lt;/partinfo&gt; (G.F.P.)</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fluorescence VS Optical density curve of &lt;partinfo&gt;BBa_K300019&lt;/partinfo&gt;</DIV></DIV></DIV></DIV></TD><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Scell=(dGFP/dt)/O.D.600 and threshold</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>For other examples for the threshold evaluation, you can see:
</P><UL><LI>&lt;partinfo&gt;BBa_K300012&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K300016&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K300028&lt;/partinfo&gt;</LI></UL><P>For every self-inducible device, several parameters were evaluated:
</P><UL><LI>O.D.start is the O.D.600 corresponding to the transcription initiation of the gene of interest; it was evaluated as reported <A href="/Team:UNIPV-Pavia/Parts/Characterization#Data_analysis_for_self-inducible_promoters_.28initiation-treshold_determination.29" title="Team:UNIPV-Pavia/Parts/Characterization">in this section</A>;
</LI><LI>K_HSL is the HSL synthesis rate per cell; it was estimated with the algorithm described <A href="/Team:UNIPV-Pavia/Parts/Characterization#Data_analysis_to_estimate_the_HSL_synthesis_rate_per_cell" title="Team:UNIPV-Pavia/Parts/Characterization">here</A></LI><LI>Doubling time is the period of time required for a cell population to double; it was evaluated as described in <A href="/Team:UNIPV-Pavia/Parts/Characterization#Doubling_time_evaluation" title="Team:UNIPV-Pavia/Parts/Characterization">Doubling time evaluation section</A></LI><LI>Scell_ratio was evaluated as (Scell_max_Phi)/(Scell_max_J101). Phi is the self-inducible device of ineterst, J101 is the reference standard &lt;partinfo&gt;BBa_J23101&lt;/partinfo&gt; contained in the same vector of the receiver device. Scell_max_phi was evaluated for times subsequent to the transcription initiation.
</LI></UL><P>Results are summarized in the following tables:
</P><P><B>Tab. 1 - Sender and Receiver on high copy plasmid &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;</B></P><TABLE border="1" width="80%"><TBODY><TR><TD rowspan="2" align="center"><B>Self-inducible device</B><FONT color="#50C878">working</FONT><FONT color="red">not working</FONT></TD><TD colspan="4" style="background: yellow" align="center">LB
</TD><TD colspan="4" style="background: cyan" align="center">M9
</TD></TR><TR><TD align="center"><B>O.D.start</B></TD><TD align="center"><B>K_HSL</B> [nmol/min]
</TD><TD align="center"><B>Doubling time</B> [min]
</TD><TD align="center"><B>Scell ratio</B></TD><TD align="center"><B>O.D.start</B></TD><TD align="center"><B>K_HSL</B> [nmol/min]
</TD><TD align="center"><B>Doubling time</B> [min]
</TD><TD align="center"><B>Scell ratio</B></TD></TR><TR><TD><FONT color="#50C878">&lt;partinfo&gt;BBa_K300017&lt;/partinfo&gt; (wiki name: I7) in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt; plasmid</FONT></TD><TD> Constitutive
</TD><TD> 3.11 10^-16  ±  2.23 10^-17
</TD><TD> 31.15  ±  2.52
</TD><TD> 0.95  ±  0.15
</TD><TD> 0.027  ±  0.002
</TD><TD> 1.52 10^-15  ±  9.76 10^-17
</TD><TD> 61.28  ±  2.67
</TD><TD> 1.68  ±  0.23
</TD></TR><TR><TD><FONT color="#50C878">&lt;partinfo&gt;BBa_K300014&lt;/partinfo&gt; (wiki name: I8) in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt; plasmid</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD> 27.86  ±  1.14
</TD><TD><FONT color="red" size="+2">X</FONT></TD><TD> 0.13  ±  0.023 **
</TD><TD> 3.59 10^-16  ±  1.22 10^-16 **
</TD><TD> 0.37  ±  0.13 **
</TD><TD> 53.09  ±  1.18 **
</TD></TR><TR><TD><FONT color="#50C878">&lt;partinfo&gt;BBa_K300015&lt;/partinfo&gt; (wiki name: I9) in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt; plasmid</FONT></TD><TD> 0.33  *
</TD><TD> 8.94 10^-17  *
</TD><TD> 33.81  ±  3.03
</TD><TD> 0.98  *
</TD><TD> 0.38  ±  0.02
</TD><TD> 3.78 10^-17  ±  4.65 10^-18
</TD><TD> 57.20 ±  1.32
</TD><TD> 0.07  ±  0.01
</TD></TR><TR><TD><FONT color="#50C878">&lt;partinfo&gt;BBa_K300016&lt;/partinfo&gt; (wiki name: I10) in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt; plasmid</FONT></TD><TD> 0.54  *
</TD><TD> 7.53 10^-18  *
</TD><TD> 39.55  ±  0.32
</TD><TD> 0.21  *
</TD><TD> 0.32  ±  0.02
</TD><TD> 5.70 10^-17  ±  7.75 10^-18
</TD><TD> 55.33  ±  5.19
</TD><TD> 0.13  ±  0.02
</TD></TR><TR><TD><FONT color="#50C878">&lt;partinfo&gt;BBa_K300012&lt;/partinfo&gt; (wiki name: I12) in &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt; plasmid</FONT></TD><TD> 0.50  ±  0.01
</TD><TD> 1.16 10^-17  ±  6.41 10^-19
</TD><TD> 36.66  ±  2.50
</TD><TD> 0.58  ±  0.02
</TD><TD> 0.30  ±  0.03
</TD><TD> 6.53 10^-17  ±  1.43 10^-17
</TD><TD> 50.92  ±  2.92
</TD><TD> 0.21  ±  0.06
</TD></TR></TBODY></TABLE><P><B>Tab. 2 - Sender and Receiver on low copy plasmid &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt;</B></P><TABLE border="1" width="80%"><TBODY><TR><TD rowspan="2" align="center"><B>Self-inducible device</B><FONT color="#50C878">working</FONT><FONT color="red">not working</FONT></TD><TD colspan="4" style="background: yellow" align="center">LB
</TD><TD colspan="4" style="background: cyan" align="center">M9
</TD></TR><TR><TD align="center"><B>O.D.start</B></TD><TD align="center"><B>K_HSL</B> [nmol/min]
</TD><TD align="center"><B>Doubling time</B>[min]
</TD><TD align="center"><B>Scell ratio</B></TD><TD align="center"><B>O.D.start</B></TD><TD align="center"><B>K_HSL</B> [nmol/min]
</TD><TD align="center"><B>Doubling time</B>[min]
</TD><TD align="center"><B>Scell ratio</B></TD></TR><TR><TD><FONT color="#50C878">&lt;partinfo&gt;BBa_K300017&lt;/partinfo&gt; (wiki name: I7) in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt; plasmid</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD> 0.24  ±  0.0004 **
</TD><TD> not computed
</TD><TD> 62.41  ±  1.56 **
</TD><TD> not computed
</TD></TR><TR><TD><FONT color="red">&lt;partinfo&gt;BBa_K300014&lt;/partinfo&gt; (wiki name: I8) in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt; plasmid</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD></TR><TR><TD><FONT color="red">&lt;partinfo&gt;BBa_K300015&lt;/partinfo&gt; (wiki name: I9) in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt; plasmid</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD></TR><TR><TD><FONT color="red">&lt;partinfo&gt;BBa_K300016&lt;/partinfo&gt; (wiki name: I10) in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt; plasmid</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD></TR><TR><TD><FONT color="red">&lt;partinfo&gt;BBa_K300012&lt;/partinfo&gt; (wiki name: I12) in &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt; plasmid</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD><TD><FONT color="red" size="+2">X</FONT></TD></TR></TBODY></TABLE><DIV id="box" style="width: 70%; margin-left: 137px; padding: 5px; border: 1px solid #000; background-color: #f6f6f6;"><DIV id="template" style="text-align: center; font-weight: bold; font-size: large; color: black; padding: 5px;"><P>LEGEND OF TABLES:
</P></DIV><DIV id="legend" style="font-weight: normal; font-size: small; color: black; padding: 5px;"><P><I>Constitutive</I>: the induction point, in term of O.D.600, is under the minimum detectable value calculated by the algorithm. This minimum value was estimated by running the algorithm on data acquired from a culture that constitutively produces GFP. For this reason, the devices labelled as <I>constitutive</I> can be considered as constitutive GFP producers.
</P><P>*: in two of three experiments the self-induction failed, thus having a non-induced culture for all the cell densities. The standard errors were not computed for these cultures.
</P><P>**: in one of three experiments the self-induction failed, thus having a non-induced culture for all the cell densities. The standard errors were computed on two independent experiments.
</P><P>&lt;partinfo&gt;BBa_K300016&lt;/partinfo&gt; is labelled with *, but probably induction failed in two of the three experiments because the culture didn't reach the O.D.start point (the experiment was stopped before the culture reached the O.D.600 critical value).
</P></DIV></DIV><P><B>Discussion</B>: a modular PoPS-based devices (&lt;partinfo&gt;BBa_K300010&lt;/partinfo&gt; was designed and used to realize a library of self-inducible devices, able to start the production of the heterologous protein at a defined culture density. Other members of this promoters' family are derived from &lt;partinfo&gt;BBa_K300009&lt;/partinfo&gt; (see the proper section for details).
</P><P>They were characterized in many different experimental conditions:
</P><UL><LI>varying the strength of the promoter controlling the production of the signal molecule (Sender Modulation)
</LI><LI>varying the copy number of vectors bearing both Sender and Receiver circuits
</LI><LI>varying the growth medium (LB or M9)
</LI></UL><P>A graphical summary is reported in the figures below:
</P><TABLE><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Typical <I>E. coli</I> growth curve with O.D.start evaluated by Threshold algorithm in LB</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Typical <I>E. coli</I> growth curve with O.D.start evaluated by Threshold algorithm in M9</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>A model-based approach was proposed to estimate many interesting parameters, such as the HSL synthesis rate per cell, and an algorithm was proposed in order to evaluate the O.D.start for every self-inducible device.
</P><P>In the figures below, the O.D.start and the HSL synthesis rate as a function of the strength of the promoter (RPU) controlling the signal molecule production are reported for &lt;partinfo&gt;BBa_K300010&lt;/partinfo&gt; (Sender&amp;Receiver device in HC plasmid) and for &lt;partinfo&gt;BBa_K300009&lt;/partinfo&gt;/&lt;partinfo&gt;BBa_F2620&lt;/partinfo&gt; (Sender device in LC plasmid in combination with Receiver device in HC plasmid).
</P><TABLE><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">O.D.start and K_HSL as a function of RPUs of the promoters controlling the signal molecule production for &lt;partinfo&gt;BBa_K300010&lt;/partinfo&gt; in high copy number plasmid in M9 medium</DIV></DIV></DIV></DIV></TD></TR><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">O.D.start and K_HSL as a function of RPUs of the promoters controlling the autoinducer production for &lt;partinfo&gt;BBa_K300010&lt;/partinfo&gt; in low copy number plasmid used in combination with &lt;partinfo&gt;BBa_F2620&lt;/partinfo&gt; in high copy plasmid in M9 medium</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>A strong correlation between the promoter strength, previously measured in RPU, and the O.D.start is depicted by the reported graphs. This is consistent with the expected behaviour of these parts, since the HSL synthesis rate is an increasing function of the upstream promoter's strength.
</P><P>For RPU values greater or equal to 1, the expected behaviour is not confirmed anymore. This is probably because too high luxI expression levels and/or synthesis rate of HSL are injurious for the cell.
</P><P>The combination of promoter strength variation and plasmid copy number modulation allows the creation of a library of self-inducible devices able to perform autoinduction at many different O.D.600 values, in any cellular growth phase.
</P><H1><SPAN class="mw-headline" id=".3Cpartinfo.3EBBa_K300093.3C.2Fpartinfo.3E.2C_.3Cpartinfo.3EBBa_K300094.3C.2Fpartinfo.3E.2C_.3Cpartinfo.3EBBa_K300097.3C.2Fpartinfo.3E.2C_.3Cpartinfo.3EBBa_K300095.3C.2Fpartinfo.3E.2C_.3Cpartinfo.3EBBa_K300086.3C.2Fpartinfo.3E_and_.3Cpartinfo.3EBBa_K300084.3C.2Fpartinfo.3E_-_Phasin_and_Intein-based_tags_for_protein_purification">&lt;partinfo&gt;BBa_K300093&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K300094&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K300097&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K300095&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K300086&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K300084&lt;/partinfo&gt; - Phasin and Intein-based tags for protein purification</SPAN></H1><P>These parts are built by assembling Phasins (&lt;partinfo&gt;BBa_K300002&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K300003&lt;/partinfo&gt;) that are able to bind to PolyHydroxyAlkanoates (PHA)
granules and Intein (&lt;partinfo&gt;BBa_K300004&lt;/partinfo&gt;), a sequence capable of self-exciding from a precursor protein through a process known as self-splicing. In addition a flexible linker sequence (&lt;partinfo&gt;BBa_K105012&lt;/partinfo&gt;) has been used to connect these parts in order to facilitate the binding and folding of the tag and the target protein of interest. Thanks to Phasin and Intein properties these parts can be used as high-specific TAGs for low-cost protein purification.
</P><P>At the moment were were not able to test Intein efficiency, but we could check if they affected the right folding of the target protein: we achieved this goal through the Silver Standard Assembly by using the GFP (&lt;partinfo&gt;BBa_K300005&lt;/partinfo&gt;).
</P><P>These parts has been characterized respectively through:
</P><UL><LI>pTet costitutive promoter devices:
</LI><LI>&lt;partinfo&gt;BBa_K300086&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K300088&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K300090&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K300099&lt;/partinfo&gt;</LI></UL><LI>3OC6HSL inducible devices:
</LI><LI>&lt;partinfo&gt;BBa_K300091&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K300092&lt;/partinfo&gt;</LI><P>and compared to a positive (&lt;partinfo&gt;BBa_K173000&lt;/partinfo&gt;) and negative (&lt;partinfo&gt;BBa_B0031&lt;/partinfo&gt;) control.
</P><H2><SPAN class="mw-headline" id="pTet_costitutive_promoter_devices">pTet costitutive promoter devices</SPAN></H2><H4><SPAN class="mw-headline" id="Methods">Methods</SPAN></H4><P>Inoculum (into 5 ml LB+Amp) from glycerol stock of:
</P><UL><LI>&lt;partinfo&gt;BBa_K300086&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K300088&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K300090&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K300099&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K173000&lt;/partinfo&gt; (positive control, J23100 constitutive promoter expressing GFP)
</LI><LI>&lt;partinfo&gt;BBa_B0031&lt;/partinfo&gt; (negative control, a non-fluorescent culture)
</LI></UL><P>Cultures were grown ON at 37°C, 220 rpm.
</P><P>The following day cultures were diluted 1:100 and let grow again for about five hours at 37°C, 220 rpm.
</P><P>The optical density (O.D.) of each culture was than measured with TECAN Infinte F200. Samples were diluted in order to obtain the same O.D. equal to 0.02.
</P><P>Then we performed a 21-hour experiment with measurements of absorbance and green fluorescence every five minutes with TECAN Infinite F200; cultures were shaken for 15 seconds every five minutes. Acquired data were blanked by subtracting the media absorbance (for absorbance measurements) and the &lt;partinfo&gt;BBa_B0031&lt;/partinfo&gt; fluorescence (for fluorescence measurements). Then, the relative GFP synthesis rate per cell was evaluated by computing (1/O.D.600)*dGFP/dt, where O.D.600 is the blanked absorbance of the culture of interest and GFP is its blanked fluorescence.
Each value shown below is the mean of three measurements in exponential phase and error bars represent the 95% confidence interval of the mean.
</P><H4><SPAN class="mw-headline" id="Results_3">Results</SPAN></H4><DIV align="center"><TABLE border="0"><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Raw growth curve</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Raw GFP curve</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Mean (dGFP/dt)/O.D. over the exponential phase (under the hypothesis that GFP half-life in fusion contructs is similar to the original one -&lt;partinfo&gt;BBa_E0040&lt;/partinfo&gt;)</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR align="center"><TH>Culture</TH><TH>Doubling time [min.] ± std error</TH></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_K173000&lt;/partinfo&gt;</TD><TD>76.3336 ± 1.4362</TD></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_K300086&lt;/partinfo&gt;</TD><TD>73.6685 ± 1.6245</TD></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_K300088&lt;/partinfo&gt;</TD><TD>74.8806 ± 2.7699</TD></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_K300090&lt;/partinfo&gt;</TD><TD>75.9433 ± 3.6808</TD></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_K300099&lt;/partinfo&gt;</TD><TD>78.4634 ± 2.5622</TD></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_B0031&lt;/partinfo&gt;</TD><TD>70.8421 ± 2.2181</TD></TR></TBODY></TABLE></DIV><H4><SPAN class="mw-headline" id="Discussion_2">Discussion</SPAN></H4><P>All the cultures showed a similar growth curve; doubling time was computed as described <A href="https://2010.igem.org/Team:UNIPV-Pavia/Parts/Characterization#Doubling_time_evaluation" class="external text" rel="nofollow">here</A> in order to obtain information about the metabolic burden due to the synthesis of the studied fusion proteins. It is possible to see that all doubling times are comparable; it is possible to assert that the expression of these BioBrick parts doesn't cause abnormal stress to the cells.
</P><P>From GFP curve it is possible to appreciate that in &lt;partinfo&gt;BBa_K300086&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K300088&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K300090&lt;/partinfo&gt;, &lt;partinfo&gt;BBa_K300099&lt;/partinfo&gt; GFP accumulation is very similar and it is significantly different from the one of the negative control &lt;partinfo&gt;BBa_B0031&lt;/partinfo&gt;. These results show that the green fluorescent protein assembled downstream of the genetic circuit is correctly folded.
</P><P>The mean protein synthesis rate was also computed over the exponential growth phase, showing again an appreciable GFP production rate that is about half of the positive control GFP.
</P><H2><SPAN class="mw-headline" id="3OC6HSL_inducible_devices">3OC6HSL inducible devices</SPAN></H2><H4><SPAN class="mw-headline" id="Methods_2">Methods</SPAN></H4><P>Inoculum (into 5 ml LB+Amp) from glycerol stock of:
</P><UL><LI>&lt;partinfo&gt;BBa_K300091&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K300092&lt;/partinfo&gt;</LI><LI>&lt;partinfo&gt;BBa_K173000&lt;/partinfo&gt; (positive control)
</LI><LI>&lt;partinfo&gt;BBa_B0031&lt;/partinfo&gt; (negative control)
</LI></UL><P>Cultures were grown ON at 37°C, 220 rpm.
</P><P>The following day cultures were diluted 1:100 and let grow again for about five hours at 37°C, 220 rpm.
</P><P>The optical density (O.D.) of each culture was than measured with TECAN Infinte F200. Samples were diluted in order to obtain the same O.D. equal to 0.02.
</P><P>Then we performed a 21-hour experiment with measurements of absorbance and green fluorescence every five minutes using TECAN Infinite F200; cultures were shaken for 15 seconds every five minutes. &lt;partinfo&gt;BBa_K300091&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K300092&lt;/partinfo&gt; constructs were induced with 100nM of HSL directly in the 96-well microplate.  Acquired data were blanked by subtracting the media absorbance (for absorbance measurements) and the &lt;partinfo&gt;BBa_B0031&lt;/partinfo&gt; fluorescence (for fluorescence measurements). Then, the relative GFP synthesis rate per cell was evaluated by computing (1/O.D.600)*dGFP/dt, where O.D.600 is the blanked absorbance of the culture of interest and GFP is its blanked fluorescence.
Each value shown below is the mean of three measurements in exponential phase and error bars represent the 95% confidence interval of the mean.
</P><H4><SPAN class="mw-headline" id="Results_4">Results</SPAN></H4><DIV align="center"><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Raw growth curve</DIV></DIV></DIV></TD><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Raw GFP curve</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Mean (dGFP/dt)/O.D. over the exponential phase (under the hypothesis that GFP half-life in fusion contructs is similar to the original one -&lt;partinfo&gt;BBa_E0040&lt;/partinfo&gt;)</DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR align="center"><TH>Culture</TH><TH>Doubling time [min.] ± std error</TH></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_K173000&lt;/partinfo&gt;</TD><TD>76.3336 ± 1.4362</TD></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_K300091&lt;/partinfo&gt;induced</TD><TD>121.1434 ± 7.0275</TD></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_K300091&lt;/partinfo&gt;not induced</TD><TD>74.4267 ± 1.3696</TD></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_K300092&lt;/partinfo&gt;induced</TD><TD>122.6088 ± 1.2785</TD></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_K300092&lt;/partinfo&gt;not induced</TD><TD>71.5105 ± 2.7113</TD></TR><TR align="center"><TD>&lt;partinfo&gt;BBa_B0031&lt;/partinfo&gt;</TD><TD>70.8421 ± 2.2181</TD></TR></TBODY></TABLE></DIV><H4><SPAN class="mw-headline" id="Discussion_3">Discussion</SPAN></H4><P>All the cultures showed a similar growth curve; doubling time was computed as described <A href="https://2010.igem.org/Team:UNIPV-Pavia/Parts/Characterization#Doubling_time_evaluation" class="external text" rel="nofollow">here</A> in order to obtain information about the burden due to the synthesis of such fusion proteins. It is possible to see that all doubling times are very similar except for induced cultures. In this case doubling time is much higher than both positive control and non-induced cultures; for this reason it is possible to assert that induction gives a high metabolic burden.
</P><P>From GFP curve and mean protein synthesis rate it is possible to appreciate that induced &lt;partinfo&gt;BBa_K300091&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K300092&lt;/partinfo&gt; GFP accumulation profiles are comparable and they significantly differ from the GFP raw time series of the negative control &lt;partinfo&gt;BBa_B0031&lt;/partinfo&gt;. On the other hand not induced &lt;partinfo&gt;BBa_K300091&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K300092&lt;/partinfo&gt; show a profile that is very similar to the negative control. These results show that the green fluorescent protein assembled downstream of the construct is correctly folded and that the inducible system works as expected.
</P><P>Not induced &lt;partinfo&gt;BBa_K300091&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K300092&lt;/partinfo&gt; show a low GFP synthesis rate maybe due to 3OC6HSL inducible 
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:UNIPV-Pavia/Parts/Characterization/NewParts">http://2010.igem.org/Team:UNIPV-Pavia/Parts/Characterization/NewParts</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNIPV-Pavia/Parts/Characterization/NewParts" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNIPV-Pavia/Parts/Characterization/NewParts" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Parts/Characterization/NewParts&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Parts/Characterization/NewParts&amp;oldid=206858" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>