"
Page
Discussion
View source
History
teams
Log in
 
Team:UNIPV-Pavia/Project/solution
From 2010.igem.org
Home
The Project
Materials & Methods
Parts
The Team
Notebook
Biological Safety
Pavia
Gallery
Sponsors
ProteInProgress: a cellular assembly line for protein manufacturing
Solutions
Several solutions were explored in this project to potentially improve the industrial production of recombinant proteins.
Self-inducible promoters were considered to avoid the usage of inducible systems especially in large-scale industrial bioprocesses, in which protein production has to be triggered by expensive inducer molecules.
Standard and user friendly integrative vectors for E. coli and S. cerevisiae were designed to stably integrate the expression systems of interest in the microbial host genome and to eliminate the need of expensive selection techniques, such as antibiotics or auxotrophic media.
Finally, an "in-cell" protein purification system was implemented using BioBrick parts: PolyHydroxyAlkanoate (PHA) granules were used as a substrate for PHA-binding peptides (Phasins) fused to the target protein, while a pH-based self-cleaving peptide (Intein) was used instead of a protease cleavage site. This solution can thus replace the usage of expensive affinity resins/columns and proteases.
Self-inducible promoters
Integrative standard vector for E. coli
Integrative standard vector for yeast
Purification of proteins
Self-inducible promoters
The aim of this section is the realization and characterization of a library of self-inducible promoters. These devices are promoters able to initiate the production of the target protein when the cell culture reaches the desired culture density.
Exploiting quorum sensing mechanism...
Different studies demonstrated that many bacteria can communicate through a mechanism called quorum sensing and can regulate gene expression relying on cell culture density. One of the most studied organisms is V. fischeri, for which the quorum sensing is regulated by two genes: luxI and luxR. The first one encodes a protein responsible for the synthesis of 3-oxo-C6-homoserine-lactone (3OC6HSL or simply HSL), a small molecule called autoinducer. The second one encodes a protein capable to bind the HSL. The lux pR promoter, which is normally off, can be activated by the LuxR-HSL complex when the autoinducer reaches a critical concentration.
When a cell population expresses luxI, the concentration of HSL is an increasing function of cell culture density and so the induction of the lux pR promoter occurs only when the cells reach a threshold density.
Taking inspiration from this natural regulation mechanism, a library of self-inducible devices was built by engineering quorum sensing circuits in E. coli. The critical cell density was modulated by changing the autoinducer molecule synthesis rate. In this way, the library members can initiate the lux pR gene expression at different cell densities of the host strain. In V. fischeri, the lux pR regulates a set of genes involved in the bioluminescence of the bacteria, but in synthetic circuits based on this regulatory mechanism users can regulate the expression of the desired genes (Fig.1).
Figure 1: Sender/receiver behaviour exploited to obtain self-inducible devices
Parts and system overview
Two BioBrick parts already present in the Registry were used in this module. The RBS-luxI part (<partinfo>BBa_K081008</partinfo>) was assembled upstream of the double terminator <partinfo>BBa_B0015</partinfo>, thus obtaining the fundamental part to build self-inducible circuits, <partinfo>BBa_K300009</partinfo> (Fig.2).
Figure 2: <partinfo>BBa_K300009</partinfo> PoPS->HSL sender device.
This part was used as signal generator, while the signal receiver part is <partinfo>BBa_F2620</partinfo> and is shown in Fig.3.
Figure 3: <partinfo>BBa_F2620</partinfo> receiver device.
In order to build a library of self-inducible devices, another foundamental device was obtained by assembling <partinfo>BBa_K300009</partinfo> upstream of <partinfo>BBa_F2620</partinfo>, thus obtaining the part <partinfo>BBa_K300010</partinfo> (Fig.4).
Figure 4: <partinfo>BBa_K300010</partinfo>, a PoPS-based self-inducible device.
These systems have the behaviour shown in Fig.5: luxR is constitutively produced under the control of the tetR promoter, while luxI is produced under the control of a different constitutive promoter.
The HSL synthesis rate was modulated by assembling constitutive promoters of different strength upstream of luxI gene. In this way, the lux pR can be activated when the HSL concentration in the growth media is greater than a threshold, which changes as a function of the HSL synthesis rate. The constitutive promoters were chosen from the Anderson Promoters Collection, available in the Registry.
Figure 5: Self-inducible devices behaviour. Pcon is a generic constitutive promoter.
Besides the use of constitutive promoters of different strength to regulate the production of the signal molecule, the plasmid copy number was taken into consideration as another important parameter. The studied combinations are summarized in Fig.6, 7 and 8.
Figure 6: Both sender and receiver are assembled on high copy number plasmid.
Figure 7: Both sender and receiver are assembled on low copy number plasmid.
Figure 8: Sender part in low copy number plasmid and receiver on high copy number plasmid.
Thus, these BioBrick parts can be used to express recombinant proteins without adding an inducer to trigger the transcription initiation of downstream genes; in large-scale production of such proteins this strategy can be cost saving and ease the entire process. Users can rationally choose the cell density at which the initiation has to occur by selecting a self-inducible device library member.
Figure 9: behaviour of self-inducible device library members. Each device is able to initiate the synthesis of the recombinant protein of interest at a specific cell density.
^top
Integrative standard vector for E. coli
The integration of the genetic circuits of interest into the microbial host genome can eliminate the need of expensive selection techniques, such as antibiotics or auxotrophic media, in cell cultures.
In order to simplify the engineering of the host genome, two standard and modular integrative vectors have been designed for Escherichia coli and Saccharomyces cerevisiae, two commonly used hosts for industrial protein production. Here, a detailed description of the integrative vector for E. coli is reported, while the following section deals with the integrative vector for yeast. The parts notation is reported in Fig.11.
The structure of the designed vector, here named <partinfo>BBa_K300000</partinfo>, is shown in Fig.10. Most of its features have been inspired by <partinfo>BBa_I51020</partinfo> (BioBrick base vector) and <partinfo>BBa_J72007</partinfo> (BamHI methyltransferase encoding CRIM plasmid), described by [Shetty RP et al., 2008] and [Anderson JC et al., 2010] respectively.
Figure 10: BioBrick integrative base vector <partinfo>BBa_K300000</partinfo>.
Figure 11: Parts notation.
This vector can be considered as a base vector, which can be specialized to target the desired integration site in the host genome. The default version of this backbone has the bacteriophage Phi80 attP (<partinfo>BBa_K300991</partinfo>) as integration site.
This vector enables multiple integrations in different positions of the same genome.
Glossary
The passenger is the desired DNA part to be integrated into the genome.
The guide is the DNA sequence that is used to target the passenger into a specific locus in the genome.
Design features
The main design features for vector engineering and for the genome integration of the vector are reported below.
Vector engineering features:
The cloning site is compatible with the original BioBrick standard (RFC10), i.e. it is composed by the BioBrick Prefix (<partinfo>BBa_G00000</partinfo>) and Suffix (<partinfo>BBa_G00001</partinfo>). The presence of illegal restriction sites (XbaI in <partinfo>BBa_J72001</partinfo> and SpeI in <partinfo>BBa_K300991</partinfo>) prevents the usage of this backbone in the classic BioBrick Standard Assembly process. However, the presence of unique EcoRI and PstI sites in Prefix and Suffix fully supports the assembly of the desired BioBrick parts in the cloning site upon EcoRI-PstI digestion and also supports the 3A Assembly.
The two NheI restriction sites flanking the default integration guide sequence <partinfo>BBa_K300991</partinfo> enable the engineering of this backbone by assembling new user-defined BioBrick integration guides upon XbaI-SpeI digestion, if the desired guide conforms to the RFC10 or a compatible standard.
The default insert <partinfo>BBa_I52002</partinfo> contains the positive selection marker <partinfo>BBa_P1016</partinfo> and the pUC19-derived replication origin <partinfo>BBa_I50022</partinfo>. <partinfo>BBa_P1016</partinfo> expresses the ccdB toxin gene, which is lethal for most E. coli strains and is useful to prevent the growth of transformants containing the uncut plasmid contaminant DNA. For this reason, the default vector must be propagated in ccdB-tolerant strains, such as DB3.1 (<partinfo>BBa_V1005</partinfo>). <partinfo>BBa_I50022</partinfo> enables the propagation of this vector at high copy in the used ccdB-tolerant strain.
Like in many other standard vector backbones (e.g. the pSB**5 series), the binding sites for standard primers VF2 (<partinfo>BBa_G00100</partinfo>) and VR (<partinfo>BBa_G00102</partinfo>) are present upstream and downstream of the BioBrick cloning site respectively. These two sequences are sufficiently distant from the cloning site to enable a good quality sequencing of the insert.
Genome integration features:
The four transcriptional terminators <partinfo>BBa_B0053</partinfo>, <partinfo>BBa_B0054</partinfo>, <partinfo>BBa_B0055</partinfo> and <partinfo>BBa_B0062</partinfo> ensure the transcriptional insulation of the integrated part from its flanking genome sequences.
The two FRT recombination sites (<partinfo>BBa_J72001</partinfo>) enable the excision of <partinfo>BBa_K300994</partinfo>-<partinfo>BBa_K300998</partinfo>-<partinfo>BBa_G0001</partinfo>-<partinfo>BBa_B0025</partinfo>-<partinfo>BBa_G0001</partinfo>-<partinfo>BBa_K300999</partinfo>-<partinfo>BBa_K300995</partinfo> (i.e. the R6K origin and the Chloramphenicol resistance marker) upon Flp recombinase activity. This marker excision allows users to make multiple integrations in the same strain, always using the same antibiotic resistance marker.
The engineering of the integration guide allows the integration of parts in user-defined genome positions and for this reason this vector supports the integration by exploiting bacteriophage attP-mediated integration as well as homologous recombination.
How to use it
<partinfo>BBa_K300000</partinfo> can be:
propagated in E. coli
engineered to change the passenger and/or the integration guide
integrated into the desired locus of the host genome
used to perform the desired number of serial integrations in the same genome
How to propagate it before performing genome integration
The default version of this vector contains the <partinfo>BBa_I52002</partinfo> insert, so it *must* be propagated in a ccdB-tolerant strain such as DB3.1 (<partinfo>BBa_V1005</partinfo>).
After the insertion of the desired BioBrick part in the cloning site, this vector does not contain a standard replication origin anymore, so it *must* be propagated in a pir+ or pir-116 strain such as BW25141 (<partinfo>BBa_K300984</partinfo>) or BW23474 (<partinfo>BBa_K300985</partinfo>) that can replicate the R6K conditional origin (<partinfo>BBa_J61001</partinfo>).
How to engineer it
The DNA guide can be changed as follows:
Figure 12: How to engineer the integrative base vector to assemble the desired DNA guide.
Be sure to have the desired guide in the RFC10 standard or a compatible one (Fig.12-a).
Digest the guide with XbaI-SpeI (Fig.12-b).
Digest the integrative base vector <partinfo>BBa_K300000</partinfo> with NheI (Fig.12-c) and dephosphorylate the linearized vector to prevent re-ligation.
Ligate the digestion products (Fig.12-d). XbaI, SpeI and NheI all have compatible protruding ends. Note that the ligation is not directional, but the guide can work in both directions.
Transform the ligation in a ccdB-tolerant strain and screen the clone.
The DNA passenger can be changed as follows:
Figure 13: How to engineer the integrative base vector to assemble the desired DNA passenger.
Be sure to have the desired passenger in the RFC10 standard or a compatible one (Fig.13-a).
Digest the passenger with EcoRI-PstI (Fig.13-b).
Digest the integrative base vector <partinfo>BBa_K300000</partinfo> with EcoRI-PstI (Fig.13-c).
Ligate the digestion products (Fig.13-d).
Transform the ligation in a pir+/pir-116 strain. Transformants with the uncut plasmid contaminant DNA do not grow because of the ccdB toxin in <partinfo>BBa_I52002</partinfo>. Screen the clone.
How to perform genome integration
The integration into the E. coli chromosome can exploit the bacteriophage attP-mediated integration or the homologous recombination.
Detailed protocols about attP-mediated integration can be found here:
Anderson JC et al., 2010
Haldimann A and Wanner BL, 2001
Detailed protocols about homologous recombination can be found here:
Martinez-Morales F et al., 1999
Posfai G et al., 1997
When using the default integration guide <partinfo>BBa_K300991</partinfo>, the integration method relies on the bacteriophage site-specific recombination (attP-mediated recombination) through the attP site on the integrative vector and the attB site in the host genome.
This integration method is applicable when the host strain does not have prophages in the att(Phi80) locus. TOP10 (<partinfo>BBa_V1009</partinfo>) and DH5alpha (<partinfo>BBa_V1001</partinfo>) strains have the Phi80 prophage and so their chromosome cannot be engineered with this procedure.
The genomic integration of the desired BioBrick part into the attP(Phi80) locus has to be mediated by co-transforming a helper plasmid, such as the Amp-resistant <partinfo>BBa_J72008</partinfo> plasmid, which carries the IntPhi80 site-specific integrase gene under the control of a thermoinducible promoter (see Fig.14). The helper plasmid also has a heat-sensitive replication origin, whose replication can be inhibited at temperatures of 37-42°C, while a permissive temperature for this vector is 30°C. For this reason, it can be cured at high temperatures, when the integrase expression is triggered at the same time. The Phi80 integrase mediates the site-specific recombination between the attP site in the integrative vector and the attB site in the bacterial genome (for a schematic description of this process, see Fig.15 and http://partsregistry.org/Recombination ).
Figure 14: Schematic description of the <partinfo>BBa_J72008</partinfo> plasmid. cI857 is the expression system for the thermoinducible cI repressor; int is the Phi80 integrase regulated by the lambda cI-repressible promoter; pir is the expression system for the pir-116 gene which is able to trigger the propagation of the R6K conditional replication origin; ori_ts is the heat-sensitive replication origin (low copy) of the vector; bla is the Ampicillin resistance marker.
Figure 15: Schematic description of site-specific recombination between a bacteriophage attP attachment site in the plasmid and an attB attachment site in the bacterial genome. In this way the sequence of interest (called "Part" in the figure) can be stably integrated into the attB genomic locus. This process is mediated by a specific integrase.
Thanks to its R6K conditional replication origin, the integrative vector cannot be replicated in common E. coli strains, so the Chloramphenicol resistant bacteria are actual integrants.
In the Materials and Methods section ( https://2010.igem.org/Team:UNIPV-Pavia/Project/results ), a detailed protocol to target the desired BioBrick part into the Phi80 locus is reported.
How to perform multiple integrations in the same genome
When this vector is integrated in the genome, the desired passenger should be maintained into the host, as well as the Chloramphenicol resistance marker and the R6K conditional replication origin. The CmR and the R6K can be excised from the genome by exploiting the two FRT recombination sites that flank them. The Flp recombinase protein mediates this recombination event (for a schematic description of this process, see Fig.16 and http://partsregistry.org/Recombination ), so it has to be expressed by a helper plasmid, such as pCP20 (CGSC#7629). This enables the sequential integration of several parts using the same antibiotic resistance marker, which can be each time eliminated.
Detailed protocols about homologous recombination can be found here:
Cherepanov PP and Wackernagel W, 1995
Datsenko KA and Wanner BL, 2000
Figure 16: Schematic description of direct repeat-recombination between two FRT sites which flank the R6K-CmR DNA. In this way, the R6K-CmR DNA is excised from the construct and a single FRT site remains in the molecule. This process is mediated by a specific recombinase, the Flp recombinase, which recognizes the FRT sites.
^top
Integrative standard vector for yeast
Here, a detailed description of the integrative vector for the yeast S. cerevisiae is reported.
The structure of the designed vector, here named <partinfo>BBa_K300001</partinfo>, is shown in Fig.17. Most of its features have been inspired by the pUG6 plasmid (GenBank: AF298793.1), constructed by [Guldener U et al., 1996]. The parts notation is reported in Fig.18.
Figure 17: BioBrick integrative base vector <partinfo>BBa_K300001</partinfo>.
Figure 18: Parts notation.
This is an integrative vector which can be used to insert the desired RFC10-compatible BioBrick parts/devices/systems into the genome of S. cerevisiae. This vector can also be specialized to target the desired integration site in the host genome.
The default version of this backbone targets the Gal system of the S288C strain (<partinfo>BBa_K300979</partinfo>) through the two homologous regions <partinfo>BBa_K300986</partinfo> and <partinfo>BBa_K300987</partinfo>. The Gal system is not essential for yeast survival if the strain is grown on carbon sources other than galactose.
This vector enables multiple integrations in different positions of the same genome. The usage of the KanMX dominant selection marker can avoid the usage of auxotrophic markers. In the industrial framework auxotrophies are usually deleterious for the process productivity because they affect the growth rate of cells. For this reason, this vector can be a concrete solution for the design of industrial yeast strains with novel user-defined functions.
Glossary
A HR (Homologous Region) is a sequence that can recombine with the host genome.
As explained for the integrative vector for E. coli, the passenger is the desired DNA part to be integrated into the genome.
Design features
This vector backbone was designed as a modular integrative vector for S. cerevisiae. In this section, the main design features for vector engineering and for the genome integration of the vector are reported.
Vector engineering features:
The cloning site is not the same as other RFC10 compatible vectors. It contains a RFC10 BioBrick Prefix (<partinfo>BBa_G00000</partinfo>) and a SpeI restriction site instead of the original BioBrick Suffix. However, the presence of unique EcoRI and SpeI sites in the cloning site fully supports the assembly of the desired BioBrick parts in the cloning site upon EcoRI-SpeI digestion. This design feature has been forced by the presence of illegal XbaI and PstI sites in the TEF promoter in the LoxP-KanMX-LoxP cassette (<partinfo>BBa_K300989</partinfo>). This vector does not support the 3A Assembly.
The two NheI sites and the two AvrII sites flanking the default HR integration sequences <partinfo>BBa_K300986</partinfo> and <partinfo>BBa_K300987</partinfo> enable the engineering of this backbone by assembling new user-defined BioBrick integration sequences upon XbaI-SpeI digestion.
This vector can be propagated in E. coli at high copy thanks to the pMB1 replication origin and the Ampicillin resistance marker present in <partinfo>pSB1A2</partinfo> (=<partinfo>BBa_K300988</partinfo>), that is one of the standard parts that compose this integrative vector.
Standard verification primer binding sites VF2 (<partinfo>BBa_G00100</partinfo>) and VR (<partinfo>BBa_G00102</partinfo>) are present in the <partinfo>pSB1A2</partinfo> (=<partinfo>BBa_K300988</partinfo>) backbone. They can be used to verify the vector length and sequence comprised between the two integration sites.
Genome integration features:
The LoxP-KanMX-LoxP cassette (<partinfo>BBa_K300989</partinfo>) enables the selection of positive yeast integrants on YPD agar plates supplemented with 200 ug/ml of G418 geneticin. Once integrated, this cassette can be excised upon Cre recombinase activity. This allows to perform multiple integrations in the same strain, always using the same dominant G418 resistance marker.
The heterologous modules in the LoxP-KanMX-LoxP cassette (<partinfo>BBa_K300989</partinfo>), i.e. the TEF promoter and the TEF transcriptional terminator from A. gossypii and the KanR from the Tn903 transposon of E. coli, show a very low homology with the S. cerevisiae genome. For this reason, the vector integration events in unwanted positions in the yeast genome are limited.
How to use it
<partinfo>BBa_K300001</partinfo> can be:
propagated in E. coli
engineered to change the passenger part and/or the HRs.
integrated into the desired locus of the host genome
used to perform the desired number of serial integrations in the same genome
How to propagate it before performing genome integration
This vector can be easily propagated in E. coli thanks to the high-copy replication origin and the Ampicillin resistance selection marker, both derived from the <partinfo>pSB1A2</partinfo> vector backbone.
How to integrate a BioBrick into the yeast genome
Digest <partinfo>BBa_K300001</partinfo> and the desired BioBrick part with EcoRI-SpeI and ligate them (Fig.19).
Propagate the resulting plasmid in E. coli and extract plasmid DNA from bacteria.
Digest the resulting plasmid with SbfI to linearize the DNA of interest (Fig.20). This is known to increase the integration efficiency from 10- to 50-fold when compared to a non-linearized DNA Reference 11 .
Transform the linearized plasmid into S. cerevisiae and select integrants on G418 antibiotic plates (Fig.21).
Figure 19: Assembly of the desired BioBrick part into the integrative vector.
Figure 20: The plasmid can be linearized upon SbfI digestion to separate the yeast DNA fragment (containing the part of interest and the LoxP-KanMX-LoxP cassette) from the E. coli DNA fragment (containing the replication origin and the Amp resistance).
Figure 21: Integration of the BioBrick of interest into the S. cerevisiae genome by homologous recombination. In this figure, the integration in the Gal system genomic region is shown.
Users can change the integration site by engineering the vector: <partinfo>BBa_K300986</partinfo> and <partinfo>BBa_K300987</partinfo> are flanked by two AvrII and two NheI respectively and for this reason the two Homologous Regions can be excided. New homologous sequences compatible with RFC10 can be digested with XbaI-SpeI and assembled because AvrII, NheI, XbaI and SpeI have compatible sticky ends. Note that this assembly is not directional and the correct orientation can be validated through sequencing with standard VF2 and VR primers.
How to perform the KanMX marker excision
The KanMX dominant selection marker is flanked by two loxP recombination sites and for this reason it can be excided upon Cre recombinase activity (Fig.22). The Cre recombinase has to be expressed by a helper plasmid.
Figure 22: KanMX excision with loxP sites recombination. For more details about this process, see http://partsregistry.org/Recombination
^top
Self-cleaving affinity tags to easily purify proteins
Conventional affinity-based protein purification methods rely on specific binding of the fusion tag to an immobilized ligand, but they are affected by severe limitations, as explained in the Motivation section. Briefly, they imply the use of expensive proteases for tag removal from the fusion protein, requiring the appropriate aminoacidic sequence to be included between the tag and the target protein. Moreover, the cost of the affinity resins used in the process is far from negligible, especially on industrial scale.
The use of self-cleaving protein elements coupled with innovative affinity tags has been recently proposed to overcome these limitations. For all these reasons, in this section a technique compliant to the BioBrick assembly standard and based on self-splicing affinity tags derived from the fusion of Phasins, Inteins and a short flexible linker is proposed.
Tag
Phasin
Phasins are proteins involved in formation and stabilization of PolyHydroxyAlkanoates (PHA), intracytoplasmic inclusions in microorganisms like Ralstonia eutropha which serve as carbon and energy storage. As such, Phasins exhibit highly specific binding to PHA granules and can be used to create tags for proteins, while PHB can be used as the affinity matrix.
In literature, it has also been shown that the Phasin-tagged fusion protein's affinity with PHA granules can be improved by increasing the number of Phasins in the fusion protein.
The Ralstonia eutropha‘s Phasin coding sequence, <partinfo>K208001</partinfo>, was already present in the Registry in Silver standard. However, at the end of this gene there was a TGA stop codon which prevents the usage of this Phasin as a head or internal domain.
In order to overcome this problem, <partinfo>K208001</partinfo> has been improved for fusion protein applications by mutagenesis through PCR using custom primers. Two new BioBrick parts were built and submitted to the Registry:
<partinfo>BBa_K300002</partinfo>: Phasin - head domain (removed stop codon; Assembly Standard 10 prefix; Silver Standard suffix)
<partinfo>BBa_K300003</partinfo>: Phasin - internal domain (removed stop codon; Silver Standard prefix; Silver Standard suffix)
These two new parts allow the construction of composite synthetic affinity tags, built by assembling an arbitrary number of Phasins (it has been shown that a better affinity is achieved by fusing of two or more Phasins). In this work a flexible linker sequence (<partinfo>BBa_K105012</partinfo>) that connects the Phasins has also been used in order to test if it can improve or facilitate the binding and folding of the tag.
PHA production
PHA production in R. eutropha is achieved by the PhaCAB operon, which contains three genes, phbC, phbA and phbB, each encoding for an enzyme essential for the formation of polyhydroxybutyrate (PHB, a kind of polyhydroxyalkanoate) inclusions. In literature, the production of PHB has already been achieved in E. coli by incorporating the PhaCAB operon. The PHB granules produced by engineered E. coli can be used as an affinity matrix for Phasin-based affinity tags and the binding of the Phasin-tagged protein of interest can occur in vivo.
Several parts for PHB production are present in the Registry: in the past iGEM editions, the Duke (2008 and 2009), Hawaii (2008), Tsinghua (2008), Virginia (2008) teams worked on it and submitted a set of BioBrick parts for PHB synthesis. However, the complete phaCAB operon is not available in the Registry. Because a lot of work about PHB has already been done and because our short term goal was not to optimize the PHB granules productions, an existing engineered E. coli strain able to produce PHB was used as a proof-of-concept affinity matrix producer. This strain comes from the DSMZ public collection and is named DMSZ15372 .
Taking advantage of the natural affinity of Phasins to PHB, it is possible to engineer strains in which PHB and the Phasin-tagged protein of interest are co-produced and the binding can occur. The overall yield and specificity of the process depends on the structure of the Phasin-based tags, which can be easily constructed or modified by using BioBrick parts compatible with the common RFC10 and RFC23 standards to perform in-frame assemblies.
For this reason these tags are modular and can be freely expanded; a generic one is shown in Fig.23.
Figure 23: Generic tag composed by two Phasin coding sequences separated by a flexible peptide linker coding sequence.
Protein purification system
Phasins and PHB granules can replace the common affinity tags and matrix respectively. However, expensive proteases are still essential to remove the tag after the Phasins-PHB binding. A novel engineered inducible Intein can be used to complete the purification process by removing the affinity tag, thus replacing proteases.
Intein
Inteins (Intervening Proteins) are sequences capable of self-exciding from a precursor protein through a process known as self-splicing, forming a peptide bond between the flanking proteins (exteins). Many so-called mini-Inteins have been engineered, whose key feature is the capability to completely release a flanking extein (the target protein) in response to a simple stimulus, either chemical or physical, with no need of expensive proteases.
In literature, one mini-Intein was obtained through mutagenesis of Mycobacterium tuberculosis Mtu RecA Intein. The sequence of this Intein, referred to as ΔI-CM, allows for pH/heat-controlled C-terminal cleavage.
Thanks to this feature, the ΔI-CM Intein can be fused downstream of an affinity tag and upstream of the protein coding sequence of interest in order to enable a cheap cleavage process to remove the N-terminal tag.
In this project, the ΔI-CM Intein sequence was designed according to [Wood DW et al., 1999] and codon-optimized for E. coli to yield <partinfo>BBa_K300004</partinfo>. This part was designed as an internal domain (prefix and suffix compatible with Silver standard) in order to enable protein coding sequence assemblies to generate the desired synthetic self-cleavable affinity tags for protein purification.
Thus, it is possible to create an engineered protein purification system that uses the tag construct and relies on Intein self-cleaving capabilities. Fig.24 shows a generic purification tag, composed by Phasins and Intein, as an example.
Figure 24: Generic purification system.
Protein purification takes place as follows:
1. Affinity tag (and consequently fused protein) binding to PolyHydroxyAlkanoates (Fig.25).
Figure 25: In vivo binding activity.
2. Cells lysis.
3. Recovery of the fused protein, bound with PHA granules, by centrifugation.
4. Elution in the supernatant of the target protein by Intein cleavage through a pH or heat stimulus (Fig.26).
Figure 26: Purification.
^top
Retrieved from " http://2010.igem.org/Team:UNIPV-Pavia/Project/solution "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
