<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNIPV-Pavia_Project_IntColi"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNIPV-Pavia/Project/IntColi">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNIPV-Pavia/Project/IntColi&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Project/IntColi&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Project/IntColi&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNIPV-Pavia/Project/IntColi" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNIPV-Pavia/Project/IntColi</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><TABLE width="100%" border="0" align="center"><TBODY><TR align="left" valign="top"><TD height="20" valign="top"><A href="https://2010.igem.org/Team:UNIPV-Pavia"> Home </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Project"> The Project </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Material_Methods"> Materials &amp; Methods </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Parts"> Parts </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Team"> The Team </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Notebook"> Notebook </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Biological_Safety"> Biological Safety </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Pavia"> Pavia </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Gallery"> Gallery </A></TD></TR><TR align="left"><TD height="20"><A href="https://2010.igem.org/Team:UNIPV-Pavia/Sponsor"> Sponsors </A></TD></TR></TBODY></TABLE><P align="center"><FONT size="5" face="Cataneo BT"><I><B>ProteInProgress: a cellular assembly line for protein manufacturing</B></I></FONT></P><P align="center"><FONT size="5"><B>Implementation and Results: Integrative standard vector for E. coli</B></FONT></P><TABLE align="center" border="0" width="80%"><TBODY><TR><TD align="center" valign="center" style="padding:20px" width="25%"><P><A href="/Team:UNIPV-Pavia/Project/PromotoriAuto" title="Team:UNIPV-Pavia/Project/PromotoriAuto">Self-inducible promoters</A></P></TD><TD align="center" valign="center" style="padding:20px" width="25%"><P><STRONG class="selflink">Integrative standard vector for E. coli</STRONG></P></TD><TD align="center" valign="center" style="padding:20px" width="25%"><P><A href="/Team:UNIPV-Pavia/Project/IntYeast" title="Team:UNIPV-Pavia/Project/IntYeast">Integrative standard vector for yeast</A></P></TD><TD align="center" valign="center" style="padding:20px" width="25%"><P><A href="/Team:UNIPV-Pavia/Project/FasinaInteina" title="Team:UNIPV-Pavia/Project/FasinaInteina">Purification of proteins</A></P></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Integrative_standard_vector_for_E._coli">Integrative standard vector for E. coli</SPAN></H1><H2><SPAN class="mw-headline" id="Materials_and_Methods">Materials and Methods</SPAN></H2><P><B>Plasmids and strains:</B> the &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; helper plasmid was kindly given by Prof. JC Anderson (UC Berkeley). MC1061 (&lt;partinfo&gt;BBa_K300078&lt;/partinfo&gt;) and MG1655 (&lt;partinfo&gt;BBa_V1000&lt;/partinfo&gt;) E. coli strains and the pCP20 helper plasmid were purchased from the Coli Genetic Stock Center (Yale University). DH5alpha (&lt;partinfo&gt;BBa_V1001&lt;/partinfo&gt;) strain was purchased from Invitrogen.
</P><P><B>Verification primers:</B> all the oligonucleotides were purchased from Primm (San Raffaele Biomedical Science Park, Milan, Italy). The P1 (&lt;partinfo&gt;BBa_K300975&lt;/partinfo&gt;) and P4 (&lt;partinfo&gt;BBa_K300978&lt;/partinfo&gt;) primers had already been used in [Anderson JC et al., 2010]. The P2 (&lt;partinfo&gt;BBa_K300976&lt;/partinfo&gt;) and P3 (&lt;partinfo&gt;BBa_K300977&lt;/partinfo&gt;) primers have been newly designed using ApE and Amplify 3X. P2 and P3 have been designed also considering the previously used verification primers P2 and P3 in the pG80ko integrative plasmid, described in [DeLoache W, 2009].
</P><P>The relative position of the P1, P2, P3 and P4 primers is shown in Fig.1:
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 1: Relative position of the verification primers. a) no integrants; b) single integrant and c) integrant with multiple tandem copies. P1/P2 and P3/P4 pairs give an amplicon when at least one copy of the vector is integrated in the Phi80 locus. P2/P3 pair show an amplicon only when multiple tandem copies occur.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P><B>Competent cells preparation:</B> all the <I>E. coli</I> strains were made competent following a slightly modified version of the protocol described in [Sambrook J et al., 1989]. Briefly, cells were grown to and OD600 of ~0.4-0.6, harvested (4000 rpm, 10 min, 4°C) and the supernatant discarded. Cells were resuspended in (30 ml for each 50 ml of initial culture) pre-chilled Mg-Ca buffer (80 mM MgCl2, 20 mM CaCl2), centrifuged as before and the supernatant discarded. Cells were resuspended in (2 ml for each 50 ml of initial culture) pre-chilled Ca buffer (100 mM CaCl2, 15% glycerol), aliquoted in 0.5 ml tubes and freezed immediately at -80°C. Test the transformation efficiency as:
</P><DIV align="center"><P><I>efficiency [CFU/ug of DNA]= # CFU * 1000 ng of DNA / amount of transformed DNA [ng]</I></P></DIV><P><B>Pir strains validation:</B> in order to test the capability of BW25141 (pir+) and BW23474 (pir-116) to propagate this conditional replication origin, they were made competent, as well as three control strains: MC1061, MG1655 and DH5alpha. Then, a vial of 100 ul of competent cells was transformed with 2-4 ng of:
</P><UL><LI>no DNA (negative control);
</LI><LI>a pSB*** series vector (positive control);
</LI><LI>self-ligated &lt;partinfo&gt;BBa_K300008&lt;/partinfo&gt; (a R6K plasmid with Cm resistance).
</LI></UL><P>Self-ligated &lt;partinfo&gt;BBa_K300008&lt;/partinfo&gt; was prepared by digesting &lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;-&lt;partinfo&gt;BBa_K300008&lt;/partinfo&gt; (yielded by BioBrick Standard Assembly) with XbaI-SpeI. The insert was isolated and purified from a 1% agarose gel. Then, it was self-ligated to generate a Cm-resistant R6K plasmid. The colonies were counted in each plate and the transformation efficiency was estimated as described before.
</P><P>The Chloramphenicol concentration in plates was 34 ug/ml for the high copy plasmids, 12.5 ug/ml for the medium/low copy plasmids and 12.5 for the three control strains transformed with the R6K plasmid.
</P><P><B>BBa_K300000 construction:</B> This section describes how this vector backbone was assembled using BioBrick parts.
</P><OL><LI><B>First step - &lt;partinfo&gt;BBa_K300982&lt;/partinfo&gt; intermediate part construction:</B></LI><LI>&lt;partinfo&gt;BBa_K300983&lt;/partinfo&gt;, provided by Mr Gene DNA synthesis service (www.mrgene.com), was excised from its original shipping vector (pMK-RQ) through digestion with AvrII restriction enzyme (Roche). It was then isolated through a 1% agarose gel electrophoresis and gel-extracted (Macherey-Nagel NucleoSpin Extract II).
</LI><LI>&lt;partinfo&gt;BBa_K300008&lt;/partinfo&gt; was assembled via BioBrick Standard Assembly from available parts and was excised from its vector (&lt;partinfo&gt;pSB1A2&lt;/partinfo&gt;) through digestion with XbaI-SpeI (Roche). It was then isolated using the same procedure described above and it was dephosphorylated by using Antarctic Phosphatase (NEB).
</LI><LI>Digested &lt;partinfo&gt;BBa_K300983&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K300008&lt;/partinfo&gt;, all having compatible sticky ends, were ligated with T4 Ligase (Roche) and transformed into competent BW23474 E. coli strain (&lt;partinfo&gt;BBa_K300985&lt;/partinfo&gt;) in order to allow the ligated plasmid propagation at high copy number. This strain was necessary because the replication origin of the resulting plasmid was the conditional R6K origin (&lt;partinfo&gt;BBa_J61001&lt;/partinfo&gt;).
</LI><LI>Positive transformants, grown on Chloramphenicol (34 ug/ml) plates, were identified by restriction mapping with EcoRI-HindIII (Roche). The yielded plasmid was &lt;partinfo&gt;BBa_K300982&lt;/partinfo&gt;.
</LI></OL><LI><B>Second step - &lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt; final vector construction:</B></LI><LI>&lt;partinfo&gt;BBa_K300982&lt;/partinfo&gt; DNA was miniprepped, digested with EcoRI-PstI (Roche), run on agarose gel and gel-extracted as described above, in order to cut out and eliminate the RBS (&lt;partinfo&gt;BBa_B0033&lt;/partinfo&gt;) between EcoRI and PstI.
</LI><LI>&lt;partinfo&gt;BBa_I52002&lt;/partinfo&gt; DNA was excised from &lt;partinfo&gt;pSB4C5&lt;/partinfo&gt; by EcoRI-PstI (Roche) digestion and isolated by gel run and gel extraction as described above.
</LI><LI>Digested &lt;partinfo&gt;BBa_K300982&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_I52002&lt;/partinfo&gt; were ligated and transformed into competent DB3.1 E. coli strain (&lt;partinfo&gt;BBa_V1005&lt;/partinfo&gt;), selected with Chloramphenicol at 34 ug/ml.
</LI><LI>Positive transformants were screened by restriction mapping with EcoRI-HindIII (Roche) and sequencing with VF2 (&lt;partinfo&gt;BBa_G00100&lt;/partinfo&gt;) and VR (&lt;partinfo&gt;BBa_G00101&lt;/partinfo&gt;) standard primers. The yielded plasmid was &lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt;.
</LI><P>All the DNA manipulations were performed according to manufacturer's protocols.
</P><P><B>Integration protocol:</B></P><OL><LI> Transform the &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; helper plasmid in the host strain (MC1061 or MG1655) and select transformants on Amp (50 ug/ml) plates under permissive conditions (30°C) overnight.
</LI><LI> Inoculate a single colony in selective LB and let the culture grow at 30°C, 220 rpm. When the culture reaches the OD600 of 0.4-0.6 prepare chemically competent cells.
</LI><LI> Transform the integrative vector with the desired insert in the BBa_J72008-containing strain and select co-transformants on Cm (34 ug/ml) plates under permissive conditions (30°C) overnight. At this temperature &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; can be replicated and so the pir protein product can be expressed in the cells. The pir product enables the propagation of the integrative vector by replicating the R6K origin.
</LI><LI> Inoculate a single colony in 5 ml of LB + Cm at 12.5 ug/ml and incubate the culture at 37°C, 220 rpm overnight. At this temperature the &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; helper cannot be replicated and the Phi80 integrase is expressed by the remaining copies of the helper. The bacteria that are able to grow in this selective medium should be correct integrants because the integrative vector cannot be replicated by the pir product anymore.
</LI><LI> Streak the culture on a Cm plate (at 12.5 ug/ml) and incubate it at 43°C overnight to ensure the loss of the helper plasmid. The bacteria that form colonies should be correct integrants without the &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; helper plasmid.
</LI></OL><P>Validate the loss of the helper plasmid by inoculating colonies in Cm (at 12.5 ug/ml) media and counterselecting them in Amp (at 50 ug/ml) media. Validate the correct integration position by performing colony PCR with primers P1/P2, P3/P4, P1/P4, P2,P3 and VF2/VR. Validate the phenotype (when possible).
</P><P>
Expected amplicon length [bp] when the vector is integrated into the Phi80 locus:
</P><TABLE border="1"><TBODY><TR><TD> </TD><TD><B>No integrant</B></TD><TD><B>Single integrant</B></TD><TD><B>Multiple tandem integrants (&gt;1)</B></TD></TR><TR><TD><B>VF2/VR</B></TD><TD>none
</TD><TD>280 + insert length
</TD><TD>280 + insert length
</TD></TR><TR><TD><B>P1/P4</B></TD><TD>546
</TD><TD>546 + insert length + 2171 (i.e. the BBa_K300000 length)
</TD><TD>546 + insert length + 2171 (i.e. the BBa_K300000 length)
</TD></TR><TR><TD><B>P1/P2</B></TD><TD>none
</TD><TD>452
</TD><TD>452
</TD></TR><TR><TD><B>P3/P4</B></TD><TD>none
</TD><TD>666
</TD><TD>666
</TD></TR><TR><TD><B>P2/P3</B></TD><TD>none
</TD><TD>none
</TD><TD>572
</TD></TR></TBODY></TABLE><P><B>Marker excision protocol:</B></P><OL><LI> Inoculate an integrant in selective LB medium and let it grow to OD600=0.4-0.6. Prepare chemically competent cells.
</LI><LI> Transform the pCP20 helper plasmid in the competent strain and select transformants on Amp (100 ug/ml) plates under permissive conditions (30°C) overnight. At this temperature the pCP20 can be replicated. The pCP20 plasmid contains Amp and Cm resistance markers, a thermoinducible Flp recombinase expression system and a heat-sensitive replication origin. The permissive temperatures for the pCP20 propagation are the same as &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt;.
</LI><LI> Inoculate a single colony in 5 ml of LB without antibiotic and incubate the culture at 37°C, 220 rpm overnight. At this temperature the pCP20 helper cannot be replicated and the Flp recombinase is expressed by the remaining copies of the helper. The bacteria should loose the R6K origin and the Cm resistance upon FRT sites recombination, mediated by Flp.
</LI><LI> Streak the culture on a LB plate and incubate it at 43°C overnight to ensure the loss of the helper plasmid. The bacteria that form colonies should be without the pCP20 helper plasmid.
</LI></OL><P>Validate the loss of the helper plasmid by inoculating colonies in Amp (at 100 ug/ml) media and validate the loss of the Cm resistance from the genome by inoculating colonies in Cm (at 12.5 ug/ml) media. Validate the correct length of the integrated part without Cm resistance and R6K origin by performing colony PCR with primers P1/P4 (which amplify the entire Phi80 locus) and VF2/VR (which amplify the integrated part). Validate the phenotype (when possible).
</P><P><B>Colony PCR:</B> a single colony or 1 ul of culture was added to the Invitrogen Platinum Taq reaction mix and was heated at 94°C for 10 min. Then it was assayed with this cycle (X 35): 94°C 30 sec, 60°C (for VF2/VR) or 63°C (for the other primers) 30 sec, 72°C according to the amplicon expected length (1Kb/min). Then the reaction was kept at 72°C for 10 min and it was run on a 1% agarose gel with the GeneRuler 1Kb Plus DNA ladder (Fermentas).
</P><P><B>Fluorescence assays:</B> integrants were inoculated in 1 ml of M9 + Cm (12.5 ug/ml) and grown at 37°C, 220 rpm overnight. The cultures were diluted 1:100 in 2 ml of selective M9 and let grow for about 4-6 hours under the same conditions as before. Three 200 ul aliquots for each culture were transferred to a 96-well microplate and assayed in the Infinite F200 microplate reader (Tecan) for about 20 hours with the following kinetic cycle: 37°C, 5 min sampling time, linear shaking 15 sec (amplitude=3), wait 5 sec, measure absorbance at 600nm, measure fluorescence with the proper filter (EX:nm/EM:540nm for GFP or EX:535nm/EM:620nm for RFP) with gain=70. The same protocol was followed for the MC1061 and the MG1655 non-integrant strains, which were grown in M9 without antibiotic.
</P><P><B>Data analysis:</B> the absorbance measurements were normalized by subtracting the absorbance of the M9, while the fluorescence measurements were normalized by subtracting the fluorescence of the non-integrant strains over time. For each well, the S<SUB>cell</SUB> signal (proportional to the reporter protein synthesis rate per cell) was computed as (1/OD600*dXFP/dt), where OD600 is the normalized absorbance and XFP is the normalized fluorescence. The S<SUB>cell</SUB> signal was then averaged over time to obtain a single value for each well. Results are presented as the average S<SUB>cell</SUB> with their 95% confidence intervals of the mean.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Validation_of_pir_strains_to_propagate_the_R6K_replication_origin">Validation of pir strains to propagate the R6K replication origin</SPAN></H3><P>The BW25141 (pir+) and BW23474 (pir-116) <I>E. coli</I> strains were chosen to propagate the vectors with the R6K replication origin at medium copy (~15 molecules per cell) and high copy (~250) respectively.
The results about their capability to propagate R6K plasmids are shown here:
</P><TABLE border="1"><TBODY><TR><TD><B>Strain</B></TD><TD><B>Efficiency with no DNA</B></TD><TD><B>Efficiency with pSB*** (positive control)</B></TD><TD><B>Efficiency with the self-ligated &lt;partinfo&gt;BBa_K300008&lt;/partinfo&gt; (R6K plasmid)</B></TD></TR><TR><TD>BW25141 (&lt;partinfo&gt;BBa_K300084&lt;/partinfo&gt;, pir+)
</TD><TD>0
</TD><TD>10^5
</TD><TD>10^5
</TD></TR><TR><TD>BW23474 (&lt;partinfo&gt;BBa_K300085&lt;/partinfo&gt;, pir-116)
</TD><TD>0
</TD><TD>10^6
</TD><TD>10^6
</TD></TR><TR><TD>DH5alpha (&lt;partinfo&gt;BBa_V1001&lt;/partinfo&gt;, neg control)
</TD><TD>0
</TD><TD>10^8
</TD><TD>0
</TD></TR><TR><TD>MC1061 (&lt;partinfo&gt;BBa_K300078&lt;/partinfo&gt;, neg control)
</TD><TD>0
</TD><TD>10^6
</TD><TD>0
</TD></TR><TR><TD>MG1655 (&lt;partinfo&gt;BBa_V1000&lt;/partinfo&gt;, neg control)
</TD><TD>0
</TD><TD>10^5
</TD><TD>0
</TD></TR></TBODY></TABLE><P>
These results show that the R6K conditional replication origin can be only propagated in pir+ and pir-116 strains (&lt;partinfo&gt;BBa_K300084&lt;/partinfo&gt; and &lt;partinfo&gt;BBa_K300085&lt;/partinfo&gt;), while the transformation of other strains with the R6K plasmid yielded no colonies after transformation. Plasmid DNA was purified/HindIII-digested/gel-run for at least one colony for BW25141-&lt;partinfo&gt;BBa_K300008&lt;/partinfo&gt; and BW23474-&lt;partinfo&gt;BBa_K300008&lt;/partinfo&gt;. The electrophoresis showed the expected length of the transformed DNA for all the clones.
</P><P>Miniprep of the BW25141 and BW23474 strains transformed with &lt;partinfo&gt;BBa_K300008&lt;/partinfo&gt; yielded a DNA concentration of ~20 ng/ul (qualitatively comparable with medium copy number plasmids) and ~90-100 ng/ul (qualitatively comparable with high copy number plasmids).
The results shown in the table above also show that the R6K plasmid in pir+ and pir-116 strains was transformed with the same efficiency as the pSB*** positive control plasmid, demonstrating that the R6K origin doesn't give any handicap in plasmid transformation.
So, the BW25141 and BW23474 strains can be successfully used to propagate the integrative vector after the excision of the pUC19-derived high copy replication origin, present in the default insert &lt;partinfo&gt;BBa_I52002&lt;/partinfo&gt;.
</P><H3><SPAN class="mw-headline" id="Integration_of_the_desired_BioBrick_part_into_the_Phi80_genome_locus">Integration of the desired BioBrick part into the Phi80 genome locus</SPAN></H3><P>MC1061 and MG1655 were chosen as host strains for integration. &lt;partinfo&gt;BBa_K173001&lt;/partinfo&gt; (constitutive strong promoter with GFPmut3) and the EcoRI-PstI fragment of &lt;partinfo&gt;BBa_J61002&lt;/partinfo&gt;-&lt;partinfo&gt;BBa_J23101&lt;/partinfo&gt; (here called PconRFP - constitutive strong promoter with RFP) were chosen as two proof of concept BioBrick parts to test the integration capability of the &lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt; vector in the Phi80 genome locus of these strains. For this reason, &lt;partinfo&gt;BBa_K173001&lt;/partinfo&gt; and PconRFP were ligated in &lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt; (digested with EcoRI-PstI) and propagated using BW23474.
The integration protocol was performed as described in the Materials and Methods section for 4 different combination:
</P><TABLE border="1"><TBODY><TR><TD><B>Integrant name</B></TD><TD><B>Strain</B></TD><TD><B>Insert of &lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt;</B></TD></TR><TR><TD>MC-GFP
</TD><TD>MC1061
</TD><TD>&lt;partinfo&gt;BBa_K173001&lt;/partinfo&gt;</TD></TR><TR><TD>MC-RFP
</TD><TD>MC1061
</TD><TD>PconRFP
</TD></TR><TR><TD>MG-GFP
</TD><TD>MG1655
</TD><TD>&lt;partinfo&gt;BBa_K173001&lt;/partinfo&gt;</TD></TR><TR><TD>MG-RFP
</TD><TD>MG1655
</TD><TD>PconRFP
</TD></TR></TBODY></TABLE><P>Three colonies grown after the overnight incubation at 43°C (step 5 of integration protocol) were analyzed for each plate. These 12 clones were called: MC-GFP-A,B,C , MC-RFP-A,B,C , MG-GFP-A,B,C and MG-RFP-A,B,C.
</P><P><B>Validation of the loss of BBa_J72008:</B> all the picked colonies did not grow in Amp (50 ug/ml) media, thus validating that &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt; Amp-resistant helper had been actually cured from the cells. However, one of these 12 clones (MG-GFP-A) also failed to grow in Cm (12.5 ug/ml) liquid media, probably because of a mistake in its inoculation. We decided not to consider this clone and to continue with 11 clones.
</P><P><B>Validation of the actual integration site:</B> colony PCR was performed for all the 11 clones, using MC1061 and MG1655 without integrants as negative controls. Primer pairs P1/P2 and P3/P4 were used to validate the presence of the integrative vector in the Phi80 genomic locus, while the primer pair P2/P3 was used to validate the presence of multiple tandem integrants (see Fig.1 in Materials and Methods).
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 2: colony PCR with P1/P2 on all the 11 integrant clones. The blank is the reaction mix without bacteria. Expected amplicon for correct integrants: 452 bp.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 3: colony PCR with P3/P4 on all the 11 integrant clones. The blank is the reaction mix without bacteria. Expected amplicon for correct integrants: 666 bp.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 4: colony PCR with P2/P3 on all the 11 integrant clones. The blank is the reaction mix without bacteria. The lanes with the amplicon were expected to come from bacteria with multiple tandem integrants. Expected amplicon for multiple integrants: 572 bp.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
PCR results with primers P1/P2 and P3/P4 showed that each clone had the correct integrant in the correct genomic position (see Materials and Methods for a list of the expected amplicon lengths). Negative controls showed no amplicons with primers P1/P2 as expected, but showed an unexpected band with P3/P4. The reason of the presence of this band was not further investigated and the results with this primer pair cannot be a useful tool for future analysis. Anyway, the P1/P2 primer pair can be sufficient to successfully validate the presence of the DNA of interest in the Phi80 genomic locus.
</P><P>PCR results with primers P2/P3 showed that two clones (MC-GFP-B and MC-GFP-C) were single integrants, while all the other clones were multiple tandem integrants (i.e. the Phi80 locus contained more than one copy of the DNA of interest). Negative controls showed no amplicons, as expected.
</P><P><B>Validation of the integrants phenotype:</B> all the 11 clones were assayed as described in the Materials and Methods section. Unfortunately, the green fluorescent clones (MC-GFP-A,B,C and MG-GFP-B,C) did not show appreciable differences when compared to negative controls, most probably because the autofluorescence of the cells was too high and hid the GFP signal. For this reason, GFP clones were not considered for further analysis. Other instruments should be used to detect the GFP signal.
</P><P>On the other hand, RFP clones (MC-RFP-A,B,C and MG-RFP-A,B,C) all showed a higher fluorescence than the negative controls (see Fig.5). As Fig.5 shows, the fluorescence of the three MG-RFP had a higher variability between clones when compared to the three MC-RFP. However, the clones were not necessarily expected to behave in the same way because all of them were multiple tandem integrants and the copy number of the PconRFP construct could be arbitrary.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 5: relative RFP synthesis rate for all the RFP expressing clones. Note: as a reference, typical values of the relative RFP synthesis rate measured for PconRFP in a low copy vector (~5 plasmids per cell) are about 6-7 fold higher (data not shown).</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Chloramphenicol_resistance_marker_excision">Chloramphenicol resistance marker excision</SPAN></H3><P>The marker excision was performed on two of the previously validated integrant strains: MC-RFP-A and MG-RFP-A (even if they were multiple tandem integrants).
</P><P>The marker excision protocol was performed as described in the Materials and Methods section for both strains, here named:
</P><TABLE border="1"><TBODY><TR><TD><B>Original name</B></TD><TD><B>Name after marker excision</B></TD></TR><TR><TD>MC-RFP
</TD><TD>MC-RFPflip
</TD></TR><TR><TD>MG-RFP
</TD><TD>MG-RFPflip
</TD></TR></TBODY></TABLE><P>
Three colonies grown after the overnight incubation at 43°C (step 4 of marker excision protocol) were analyzed for each plate. These 6 clones were called MC-RFPflip-A,B,C and MG-FRPflip-A,B,C.
</P><P><B>Validation of the loss of pCP20 and the resistance marker:</B>  all the 6 picked colonies failed to grow on both Amp (100 ug/ml) media and Cm (12.5 ug/ml) media. They could only grow in LB without antibiotics, thus validating that the pCP20 helper had been actually cured and the R6K-CmR DNA containing the Chloramphenicol selection marker had been actually eliminated.
</P><P><B>Validation of the length of the integrated part:</B> colony PCR was performed for all the 6 clones, using MC1061 and MG1655 without integrants as negative controls. Primer pairs VF2/VR and P1/P4 were used to validate if the <I>passenger</I> of interest was still present in the genome and the length of the entire Phi80 locus respectively after the marker excision.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 6: colony PCR with VF2/VR on all the 6 flipped clones. The blank is the reaction mix without bacteria. Expected amplicon for correct insert: 1.2 Kb.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 7: colony PCR with P1/P4 on all the 6 flipped clones. The blank is the reaction mix without bacteria. Expected amplicon for correct construct in the correct position: 2.3 Kb. Expected amplicon for the non-integrant strain MG1655: 546 bp.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
PCR results with primers VF2/VR showed that all the 6 clones still contain the <I>passenger</I> of interest, i.e. PconRFP, in the genome after the marker excision. The reaction blank, the MG1655 strain (neg control) and also the other samples showed some extra bands, but the ~1.2Kb amplicons of MC-RFP-A,B,C and MG-RFP-A,B,C had the correct length and was much brighter than the other bands.
</P><P>
PCR results with primers P1/P4 (Fig.7) showed that an amplicon of ~2.3Kb was present in all but one screened clones, while the MG1655 negative control showed the expected 546bp length for a non-integrant. MC-RFPflip-C did not show the P1-P4 amplicon because the reaction failed: the tube was damaged and the reaction mix was completely evaporated at the end of the PCR program. For this reason, a PCR was performed again on this clone (Fig.8).
</P><P>The ~2.3Kb amplicon was consistent with a single integrant of &lt;partinfo&gt;BBa_K300000&lt;/partinfo&gt;-PconRFP without the R6K-CmR DNA fragment, thus validating the successful excision of the FRT-flanked DNA fragment containing R6K-CmR and confirming that PconRFP was still present in the correct locus in single copy.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Figure 8: colony PCR with P1/P4 on MC-RFPflip-C clone. The blank is the reaction mix without bacteria. Expected amplicon for correct construct in the correct position: 2.3 Kb.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><P>
These results showed that, even if the clones were multiple tandem integrants, they became single integrants after marker excision. This is because the Flp recombinase mediated the recombination of all the FRT sites of the multiple integrants until only a single FRT site was present in the Phi80 locus, thus leaving only the single integrant of interest without the selection marker in the genome.
</P><P><B>Validation of the marker-less phenotype:</B>all the 6 clones were assayed as described in the Materials and Methods section. They all showed a low variability and their fluorescence was lower than their two <I>parents</I>, i.e. MC-RFP-A for the MC1061 strains and MG-RFP-A for the MG1655 strains (see Fig.9). This result is consistent with the copy number of the PconRFP construct in the clones, in fact both MC-RFP-A and MG-RFP-A were multiple tandem integrants, while MC-RFPflip-A,B,C and MG-RFPflip-A,B,C were single integrants, as described above.
</P><P>All the MG-RFPflip showed a very low relative RFP synthesis rate when compared to the other strains, but the signal is systematically grater than the fluorescence of the negative control, thus validating the phenotype for the MG1655 strain. MC-RFPflip-A,B,C showed a higher fluorescence than MG-RFPflip-A,B,C.
</P><P>In conclusion, it has been demonstrated that, even after the marker excision process, the phenotype of the engineered cells is maintained.
</P><TABLE align="center"><TBODY><TR><TD><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Figure 9: relative RFP synthesis rate for all the RFP-expressing clones after marker excision. In this figure, the bars corresponding to the fluorescence of the clones before marker excision is also reported to facilitate the comparison between them. Note that all the three <I>flip</I> clones are derived from MC-RFP-A for the MC1061 clones and from MG-RFP-A for the MG1655 clones.</DIV></DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>A novel integrative vector for <I>E. coli</I> has been successfully designed, constructed and used to integrate two proof of concept protein expression systems in two commonly used E. coli strains.
</P><P>The results showed that the vector is fully functional and can integrate into the correct targeted locus of the host chromosome through the Phi80 site-specific recombination system by using &lt;partinfo&gt;BBa_J72008&lt;/partinfo&gt;, an existing BioBrick helper plasmid from the Registry. In most cases, the integration occurs in tandem copies, probably because of the too high Chloramphenicol concentration used during the selection of integrants, which forces multiple integration of Cm-resistant constructs. This concentration was the same used during the pSC101 low copy plasmid (~5 copies per cell) selection. In some cases, it is desirable to have a single copy of the desired BioBrick in the genome, for example when the gene dosage is important. In [Haldimann A and Wanner BL, 2001] the usage of Chloramphenicol at 6 ug/ml yielded a very high percentage of single integrants. However, when tested in our lab, the MG1655 strain could survive on LB plates with Cm at 6 ug/ml and also at 8 ug/ml. For this reason a higher concentration of Cm was chosen for selection. Further studies should investigate the optimal antibiotic concentration to yield the highest single integrants percentage as possible.
</P><P>
The Flp/FRT mediated excision of the R6K and, most importantly, of the Cm resistance marker also worked by using the pCP20 helper plasmid. The estimated efficiency of this process was 100%. In addition, multiple tandem integrants became single integrants after the marker excision. This is because the Flp recombinase mediated the recombination of all the FRT sites of the multiple integrants until only a single FRT site was present in the Phi80 locus. The marker excision is a powerful tool to engineer microbial strains for industrial protein manufacturing because the engineered organism should not carry unsafe antibiotic resistances that may be diffused in the environment.
</P><P>
The fluorescence phenotype confirmed the correct integration into the <I>E. coli</I> chromosome. As expected, in general multiple integrants showed a higher fluorescence than the single integrants.
</P><P>
The BioBrick compatibility and the vector modularity give the possibility to the scientific community to stably engineer novel biological functions in <I>E. coli</I> with a very easy and user friendly methodology. A user’s handbook about the vector usage is shared in the Registry, as well as the users experiences and the compatibility information.
</P><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:UNIPV-Pavia/Project/IntColi">http://2010.igem.org/Team:UNIPV-Pavia/Project/IntColi</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNIPV-Pavia/Project/IntColi" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNIPV-Pavia/Project/IntColi" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Project/IntColi&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Project/IntColi&amp;oldid=182401" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>