"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Illinois/Project
From 2009.igem.org
Home
Team
Project
Biobricks
Home
Home
Sponsors
Donations
Team
Meet the Team
Pictures
Fun Stuff
Project
Introduction
Research
Modeling
Results
Experiments
Protocols
Vectors
Biobricks Notebook
Introduction
Recently, synthetic biologists have been focusing on creating logic gates in bacteria by controlling gene expression at various levels: transcription, translation, and post-translational processes.  Successfully installing logic gates in bacteria have large implications for engineering and could be used in applications involving drug development or biofuel fermentation.  Various challenges have presented themselves in regard to this objective, such as the tendencies of outputs to be expressed at intermediate levels rather than being "on" or "off".  Simple logic gates, such as AND, OR, NAND, and NOR gates, have been successfully created in bacteria, but more complex logic circuits are still waiting to be designed and implemented into bacterial species.
A 2-to-4 decoder and truth table.  Courtesy of Wikipedia
One such logic circuit is known as a n-to-2n binary decoder.  A binary decoder is a device that receives n binary inputs (each input is either "on" or "off") and decodes them to produce one of 2n unique outputs.  For example, a 2-to-4 binary decoder has two binary inputs and four different outputs.  Each output can be represented in terms of a combination of the two binary inputs.  For example, in a two-to-four decoder, the inputs are represented as 00, 01, 10, and 11, where each input can be 0 ("off") or 1 ("on"), the first digit corresponds to the state of the first input, and the second digit corresponds to the state of the second input.  Larger decoders such as 3-to-8 and 4-to-16 are also possible.  Binary decoders are widely used in computers for random-access memory and for converting microcode into control signals.
sRNAs can be either cis(A) or trans(B).  They can prevent protein synthesis from an mRNA by several methods, including binding to the RBS and preventing ribosome binding or binding to the 5' UTR and recruiting degradative enzymes.  Some sRNAs positively regulate a gene by binding to a folded mRNA and revealing the RBS.
Our team has chosen to create a 2-to-4 binary decoder in Escherichia coli.  Our novel bacteria will be able to sense concentrations of two inputs (such as two different sugars), identify these concentrations as "on" or "off", and produce one of four unique outputs (we chose to use fluorescent proteins) depending on the combination of inputs.  Thus, our four outputs can be represented in terms of our binary inputs as 00, 01, 10, and 11, where 0 corresponds to the "off" state (low concentration), 1 corresponds to the "on" state (high concentration), the first digit corresponds to the state of the first input, and the second digit corresponds to the state of the second input.
One feature of our bacterial decoder will be that it will be modular.  In other words, it should be trivial to change the two inputs that our decoder will respond to.  Our design uses only two promoters that are each sensitive to a specific input.  Swapping inputs simply involves using two different promoters sensitive to two different inputs in the proper locations.  This ensures that our bacterial decoder does not just work with one specific scenario but that it will be flexible and have a wide variety of applications.
Our bacterial decoder will also involve the usage of small non-coding RNAs, or sRNAs.  sRNAs are very short transcripts of RNA (usually ~100 base pairs in E. coli) that are used in bacteria to positively or negatively regulate genes at the translational level by binding to the mRNA transcript of the gene and either occluding the ribosomal binding site (RBS) and preventing ribosome binding or recruiting nucleases to degrade the mRNA.  The roles and mechanisms of sRNAs in bacteria currently comprise a very prominent topic in the field of biology, and many sRNAs have been extensively studied and characterized.  Using sRNAs in our bacterial decoder will help us achieve on/off behavior rather than intermediate levels of outputs, since sRNAs do little at low concentrations but demonstrate strong regulation at high concentrations.  They also offer other advantages over transcriptional regulation such as using fewer cell resources and the ability to turn genes on or off more quickly.  Currently, there are very few working sRNAs in the Parts Registry.  During the course of our project, we hope to test several sRNAs along with their respective target sequences and add them to the Parts Registry as working components for the use of other iGEM teams.
Sources
Barry, Patrick. "Bacteria That Do Logic". Science News, Vol. 174 #10, pg. 20. 8 November 2008. http://www.sciencenews.org/view/generic/id/37724/title/Bacteria_that_do_logic .
Garren, Emma. "Logic Gates". GcatWiki. http://gcat.davidson.edu/GcatWiki/index.php/Logic_Gates_-_Emma_Garren .
Storz, Gisela and Waters, Lauren. "Regulatory RNAs in Bacteria". Cell Vol. 136, Issue 4, pg. 615-28. 20 Feb 2009.
Wikipedia. "Decoder". http://en.wikipedia.org/wiki/Decoder .
Questions about our Wiki page?  Please email us at illinoisiGEM@gmail.com .
Home
Team
Calendar
Project
Experiments
Biobricks
Notebook
Retrieved from " http://2009.igem.org/Team:Illinois/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
