"
Page
Discussion
View source
History
teams
Log in
 
Team:Illinois/sRNA Library
From 2009.igem.org
Home
Team
Project
Biobricks
Home
Home
Sponsors
Donations
Team
Meet the Team
Pictures
Fun Stuff
Project
Introduction
Research
Modeling
Results
Experiments
Protocols
Vectors
Biobricks Notebook
Contents
1 sRNA Library Team
2 June 15
3 June 16
4 June 17
5 June 18
6 June 19
7 June 30
8 July 1
9 July 2
10 July 6
11 July 7
12 July 10
13 July 17
14 July 20
15 July 21
16 July 22
17 July 23
18 July 27
19 July 28
20 August 3
21 August 5
22 August 6
sRNA Library Team
Goals: Our team will be characterizing various small RNAs and submitting them to the Parts Registry as working biobricks.  We will insert the sRNA gene into a high-copy plasmid and its target sequence before the GFP gene in a low-copy plasmid.  We will then transform both plasmids into E. coli cells.  The sRNA should successfully repress expression of the GFP gene and result in a decrease in fluorescence.  We will be using a protocol adapted from one used by Urban and Vogel (please see the Protocols page).
Note: Our team split into four groups at the beginning of the summer and each team worked to get an sRNA working from the following: MicF, MicC, MicA, and SgrS.  After the first week of lab work, our team split to work on things other than sRNA characterization, and the work done was resumed by the new sRNA Library team.
Click on a link below to view information pertaining to a specific sRNA, including gene sequences and primers used (early notebook entries are also included).
MicF
MicC
MicA
SgrS
June 15
Each of the four sRNA groups ran two PCR reactions: one to synthesize their sRNA gene and one to synthesize the target sequence of the sRNA.
We ran a PCR for plasmid pJU-334 and ran a gel on the reaction.  For some reason, the gel had a band at ~1.9kbp, rather than at 3.1kbp, where we expected the plasmid to be.
June 16
All four groups ran gels on their PCR reactions to verify that the appropriate DNA fragments had been synthesized.  All PCR reactions except for the MicC gene were successful.
We ran a PCR on plasmid pJU-334 and ran a gel.  The gel indicated a band at ~11kbp rather than at 3.1kbp, where the plasmid is expected to fall.
June 17
The four groups successfully added restriction endonucleases to their synthesized DNA and purified it by gel electrophoresis.  The MicC gene was synthesized by PCR and verified to be successful by gel electrophoresis.
We performed a PCR on the pJU-334 plasmid and ran a gel, which indicated a faint band at ~3.1kbp (the expected size of the plasmid) and a strong band at ~1.9kbp.
June 18
We attempted another PCR of plasmid pJU-334 and ran a gel on it, using all 45 μL of the reaction.  We detected a faint band at ~3.1kbp (the expected size of the plasmid) and a strong band at ~1.9kbp.  We purified the ~3.1kbp band and ran another PCR on the purified DNA.
June 19
We found out that we ordered one of the wrong primers for plasmid pJU-334.  We will have to order the correct primer and PCR our plasmids before we can continue.
June 30
We started creating primers for 5 new sRNA's and each of their respective target sequences:
sRNA's / target sequences
dicF  /  ftsZ   dsrA  /  hns   rhyB  /  sodB   spot42/  galK   rseX  /  ompC and ompA   gcvB  /  dppa and oppa
We PCR'ed all of the genes listed above except for ompC and ompA, and we obtained bands for each of the target sequences.  However, for the sRNA's, we only obtained bands for gcvB and dicF.
July 1
We re-did the PCR's, of the four sRNA's that didn't work.  We also began a digestion of the two sRNA's and each of their target sequences.
The gel we ran after the PCR showed no bands.  The gel we ran after the digestions showed strong bands, and we performed a gel extraction to purify the products.  However the DNA concentrations we obtained from using the nanodrop were unusable.
July 2
We once again re-did the PCR's of dicF, gcvB, dppA, oppA, and ftsZ.  There was no more TBE buffer so the gel of the PCR products will be run after the weekend.
July 6
We performed another PCR of all of the new sRNA's, and we ran a gel with the 5 sRNA's from last week.
Lane 1    100bp ladder
2    ftsZ     3    ftsZ*     4    oppA     5    oppA*     6    dppa     7    dppa*     8    hns     9    galK     10   sodB
Lane 1    100bp ladder
2    dicF     3    dicF*     4    gcvB     5    gcvB*     6    rseX     7    spot42     8    ryhB     9    dsrA
The genes that have an asterisk are the samples from July 2st, and none of them seemed to show bands.  SodB, spot42, and dsrA also didn't work.
We ran a digestion with the PCR products that did work, however we did not do the PCR purification.
July 7
We ran two gels with all of our digestion products.
Lane 1    100bp ladder
2    ftsZ     3    ftsZ     4    oppA     5    oppA     6    dppA     7    dppA     8    hns     9    hns     10   galK
Lane 1    100bp ladder
2    galK     3    dicF     4    dicF     5    gcvB     6    gcvB     7    rseX     8    rseX     9    ryhB     10   ryhB
The samples were split up into two lanes just in case the wells on the gels didn't overflow.
After the digestion gels, we performed a gel purification and used the nanodrop to check the DNA concentrations for each of the digestion products.
Concentrations: ng/microliter
ftsZ:  21.6 oppA:  14.1 dppA:  15.0 hns:   11.8 galK:  19.0 dicF:  23.0 gcvB:  21.3 rseX:  6.8 ryhB:  7.5
July 10
Hopefully now the ligation will be working.  After many many attempts we got a decent looking gel!  The resolution isn't very good.  This was an attempt to fit both the insert and the vector on the same gel but that didn't work at all.  Its 1% with a 1kbp ladder.
The large, slow bands are hopefully the ligated plasmids.  The ones just below that don't look like they worked.  They look reasonably bright.  Each lane is a variant of Insert:Vector ratio, ranging from 3:1 to 15:1 which were all ligated at room temperature for 4 hrs.  (Protocol called for 1hr)
July 17
We were running out of the digestion products of micC, micA, micF, and sgrS.  So we re-pcr'ed the original 4 sRNAs.  We set up a 25 microliter reaction as described in the protocols.  We tried running a gel on the PCR products once they were purified but when we tried loading the gels, the samples floated out.  Rather than waste all of the PCR product, we assumed the gel would show the appropriate bands and went on with the digestion.
We ran the digestion with all 10 microliters of the remaining PCR purified product, and ran the digestions for 3 hours.  After the three hours, we stopped the digestion reaction by placing the samples in a 60 degrees celsius water bath for 20 minutes.
July 20
We ran a 2% gel of the digested products from July 17th at 145 volts for 45 minutes.
Lane 1    100bp ladder
3    micA     5    micC     7    micF     9    sgrS
We performed a gel extraction and obtained the following concentrations:
micA: 7 ng/microliter micC: 4  micF: 12 sgrS: 7.5
July 21
We PCR'd the 6 new sRNA's and this time we used double the volumes of everything.  We PCR purified everything as well and started an overnight digestion.
July 22
We ran a gel with the digested products and performed a gel extraction.  We obtained the following DNA concentrations:
gcvB: 21 ng/microliter
ryhB: 16.4
dicF: 3.8
dppA: 21.4
oppA: 3.0
galK: 12.3
hns:  6.5
ftsZ: 21.0
July 23
Today we re-PCR'ed the original eight sRNA's and their target sequences.  Once again, we used double volumes of everything to get a higher DNA concentrations.  14 microliters of the PCR'ed sample was digested for an hour and then PCR purified rather than gel extracted.
July 27
We digested the PCR samples from July 23rd step by step, rather than doing a double digestion.  After each digestion, we did a PCR purification.  After the digestions, we took the DNA concentrations of each.  All but 4 showed high levels of contamination when the Nanodrop readings were taken.
ompC: 22.5 ng/microliter
ompF: 27
ptsG: 18
micA: 40
July 28
We PCR'd the 4 sRNAs/target sequences that were found to have contamination on July 27th.  Double volumes were used.
August 3
We set up a ligation for micC and ompC.
August 5
We obtained colonies for micC and ompA, indicating that our ligations may have been successful.
August 6
We verified by PCR that our ligations successfully included the MicC gene and the OmpA target sequence.  The gel below indicates strong bands that correspond closely to 109bp for MicC and 178bp for OmpA.
Questions about our Wiki page?  Please email us at illinoisiGEM@gmail.com .
Home
Team
Calendar
Project
Experiments
Biobricks
Notebook
Retrieved from " http://2009.igem.org/Team:Illinois/sRNA_Library "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
