<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Glasgow_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Glasgow/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title-bar" data-responsive-toggle="main-menu" data-hide-for="medium"><DIV class="title-bar-title">Glasgow iGEM 2017</DIV></DIV><DIV class="top-bar" id="main-menu"><DIV><DIV class="top-bar-right"><UL class="vertical large-horizontal menu" id="main-nav" data-responsive-menu="medium-accordion large-dropdown"><LI><A class="headline" href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Description">Project Background</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/mtlR">Exploiting the Mannitol regulator MtlR</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/araC">Mutagenesis of AraC</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/QuorumSensing">Quorum Sensing </A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/ANDGate">AND Gate testing</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/XyluloseBiosynthesis">Xylulose Biosynthesis</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/InterLab">InterLab study</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Measurement">Measurement</A></LI></UL><LI><A href="https://2017.igem.org/Team:Glasgow/Parts">Parts List</A></LI><LI><A class="headline" href="#">Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Applied_Design">Designing the Biosensor Device</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Hardware">Hardware</A></LI><LI><A href="#">Additional Work</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HydrogelStrip">Hydrogel Strip Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/BacillusSubtilis"><I>Bacillus subtilis</I> Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Collaborations">Collaborations</A></LI><LI><A class="headline" href="#">Practices</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated">Gold and 
                                                 Integrated</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Engagement">Education and Public 
                                                Outreach</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated/PolicyBrief">Policy Brief</A></LI><LI><A class="headline" href="#">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/WeeklyJournal">Weekly Journal</A></LI><LI><A class="headline" href="#">People</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Attributions">Attributions</A></LI><LI><A class="headline" href="#">Accomplishments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Accomplishments">Accomplishments</A></LI></DIV></DIV></DIV><DIV class="photo-block" id="Interlab"><DIV class="text left"><DIV class="title">
			Measurement
		</DIV></DIV></DIV><DIV class="wiki-block"><DIV id="article"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Introduction"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Materials_and_Methods"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Transformations"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Transformations</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Calibration"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Calibration</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Dimerization"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Dimerization</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Measurements"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Measurements</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#NEB5.CE.B1"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">NEB5α</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Dimers"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Dimers</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#NEB5.CE.B1_2"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">NEB5α</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Dimers_2"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Dimers</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Concluding_Remarks"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Concluding Remarks</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>There are many contributing factors that could explain outlier data points for such studies as the 2017 InterLab study and so we decided to investigate two of these. They were the use of commercial derivatives of DH5α and the appearance of plasmid dimers. Commercial strains of <I>E. coli </I> - although closely related to DH5α - often have slightly differing genotypes. For the purposes of our investigation we used NEB5α and found that expression levels were in the most part higher in NEB5α than in DH5α, suggesting that commercial derivatives of DH5α should be avoided for studies such as the InterLab. Plasmid dimers are caused by two plasmids joining together to form one larger plasmid. They are often an issue seen in DH5α as they form spontaneously and are often not screened for. Our results showed the unpredictable nature of dimer behaviour and highlighted the importance of avoiding these in quantitative studies of gene expression as they can cause errors in results.
</P><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P>For studies in which a large data set will be collated from different research labs –such as the 2017 InterLab- it is imperative that all conditions are the same. We decided to look at two possible sources of error which are frequently overlooked. To do this all testing was carried out using the 2017 InterLab set of test devices and controls. These being <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J364000">J364000</A>, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J364001">J364001</A>, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J364002">J364002</A>, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J364003">J364003</A>, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J364004">J364004</A> and <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_J364005">J364005</A>. The positive control was <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I20270">I20270</A> – contains the J23151 promoter, B0032 ribosome binding site, E0040 gfp gene, B0010 terminator and the B0012 terminator- and the negative control was <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_R0040">R0040</A> –tetR repressible promotor with no gfp gene, which will not express GFP-. All of the above were in the pSB1C3 plasmid. Further details of the relative strengths of each part can be seen on the <A rel="nofollow" class="external text" href="https://2017.igem.org/Team:Glasgow/InterLab">InterLab</A> page.
</P><P>The first possible source of error we looked at was the use of commercially available DH5α<I>E. coli</I> derivatives. To do this we used NEB5α, as it is very closely related to DH5α it is thought that their expression levels should be extremely similar. However, there are some differences in their genotypes, both can be seen in table 1 below. These differences in genotype could be enough to see a significant difference in expression levels.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Table 1</B> Shows the genotypes of DH5α<SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP> and NEB5α<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP>.
		</DIV></DIV></CENTER><P>
In addition to the different strain of <I>E.coli</I>, we also looked at how dimerization caused differences in the expression levels. These are caused by two monomeric plasmids recombining together to create one big plasmid - therefore, they contain two copies of each gene contained on the plasmid a diagram of this can be seen in figure 1. Due to the way in which control mechanisms count plasmid origins dimers are maintained at about half the value of monomers and so there should be an equal number of copies of the gene. From this it would be expected that dimers will not have any effect on the expression of GFP, however other factors can make promoter strength and transcription levels sensitive to dimerization. A previous iGEM team found that one of their test devices for the 2016 InterLab study was not behaving as expected. When they checked it uncut on a gel they found that it was running twice as high as it should have been and came to the conclusion that the plasmid had dimerized, They then concluded this was the reason why their results were not as expected <SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP>.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 1</B> Shows J36400 in pSB1C3 in both the monomer and dimer form.
		</DIV></DIV></CENTER><P>
We hypothesised that there will be little to no difference between the NEB5α strain of <I>E.coli</I> compared to the DH5α and that there would be a difference in expression levels in the dimers compared to the monomer plasmids, with the dimers causing higher expression levels.
</P><H2><SPAN class="mw-headline" id="Materials_and_Methods">Materials and Methods</SPAN></H2><H4><SPAN class="mw-headline" id="Transformations">Transformations</SPAN></H4><P>These were carried out as per our <A rel="nofollow" class="external text" href="https://2017.igem.org/Team:Glasgow/Protocols">standard calcium chloride protocol.</A></P><H4><SPAN class="mw-headline" id="Calibration">Calibration</SPAN></H4><P>Both the OD600 reference point and the fluorescein fluorescence standard curve were carried out as per the <A rel="nofollow" class="external text" href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">InterLab 2017 Plate Reader Protocol</A><SUP id="cite_ref-name_4-0" class="reference"><A href="#cite_note-name-4">[4]</A></SUP>.
</P><H4><SPAN class="mw-headline" id="Dimerization">Dimerization</SPAN></H4><P>To generate dimer plasmids a strain of <I>E. coli</I> JC8679 <SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP> known to cause interplasmid recombination at a higher frequency and so cause multimers was used. Plasmid DNA of each InterLab device was transformed into JC8679 using standard transformation protocol and was then repurified from JC8679 by standard QIAGEN DNA miniprep protocol. The plasmid extracts were then separated by standard gel electrophoresis and the DNA band corresponding to the dimeric plasmid DNA was cut out from the gel and extracted by standard gel extraction protocol. A diagram of a gel showing the monomer plasmid in comparison to the plasmid products when multimerization has occurred can be seen in Figure 2. Multimerisation didn’t occur 100% of the time so this needed to be repeated until dimers were obtained for all of the test devices and the two controls. This extract was then transformed into DH5α<I>E. coli</I>. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 2</B> Lane 1 shows a 1Kb DNA Ladder, Lane 2 shows a monomer plasmid of I20270 (the InterLab positive control) and Lane 3 shows I20270 after multimerization had occurred. In both lanes 2 and 3 the stronger bands correspond to the supercoiled plasmid DNA with the fainter bands being the linear and relaxed circular plasmid DNA.
		</DIV></DIV></CENTER><H4><SPAN class="mw-headline" id="Measurements">Measurements</SPAN></H4><P>For the purpose of the measurements two biological duplicates (colonies) of all 6 test devices transformed in to DH5α, along with a positive and negative control, were grown up in LB broth overnight at 37°C with shaking at 220rpm. These were then diluted down to an OD600 of 0.02 and 100µl of the samples were placed into wells of a clear bottomed black sided 96 well plate as shown as per Figure 3. This was placed into the plate reader at 37°C with shaking at 200rpm and measurements of OD600 and fluorescence at an excitation of 485nm and emission of 530nm were taken at time points of 0hrs, 2hrs, 4hrs and 6hrs.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 3</B>  Layout of samples in 96 well plate. <SUP id="cite_ref-name_4-1" class="reference"><A href="#cite_note-name-4">[4]</A></SUP></DIV></DIV></CENTER><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="NEB5.CE.B1">NEB5α</SPAN></H3><P>To compare the expression levels in the NEB5α strain to the DH5α strain, the test devices were transformed into NEB5α following the same protocol as with the DH5α and measurements of fluorescence and OD600 taken the same as with the DH5α and compared to those results above. As there is a possibility the two strains can have different growth rates it is important to only look at the OD600 corrected results - these are shown in Figure 4 below. All of the test devices, except one, showed higher noise when in the NEB5α strain, the negative control being almost 5 times higher in NEB5α, the positive control, test device 1 and test device 2 all being between 5% and 10% higher than their DH5α counterparts, test device 4 being around 35% higher and finally test device 5 and 6 being over 100% higher. As for test device 3, this showed as only having less than 20% of the fluorescence of its DH5α pair. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 4</B> Shows the OD600 corrected fluorescence values for the two controls and the 6 test devices (average fluorescence/ average OD600) for the monomers in DH5α and the monomers in NEB5α at the time point of 4 hours. Error bars are standard deviation. AU= arbitrary units
		</DIV></DIV></CENTER><H3><SPAN class="mw-headline" id="Dimers">Dimers</SPAN></H3><P>To create dimers JC8679 was used, this is a hyper-recombinogenic strain of <I>E. coli</I><SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP>. Most lab strains of E. coli contain a mutation that prevents them from being able to recombine and so stops them from producing multimers –although they are still able to recombine at a low frequency- and JC8679 is no different to these <SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP>. The main difference is that JC8679 contains a mutation that recovers its ability to recombine. This is a sbcA23 mutation and acts to supress the knockdown of recBC. This sbcA23 mutation causes the expression of recE which is within an inactive bacteriophage in the E. coli genome <SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP>. 
</P><P>All the 2017 InterLab study test devices and the controls were transformed into JC8679. The plasmid DNA was extracted and only the dimeric DNA purified and transformed into DH5α<I>E.coli</I>. A gel verifying the presence of dimers can be seen in Figure 5 below, the top gel shows that in both monomeric and dimeric form all the test devices contain the same size of insert and vector and nothing has been inserted or deleted during dimerisation. The lower gel shows the test devices uncut, and shows that the dimer for each test device is running twice as high as its monomer counterpart - proving that they are in fact dimers. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 5</B> Shows a gel of all test devices in both monomeric and dimeric forms. The top gel shows all of the devices cut with EcoR1 and PstI and the lower gel shows all of the devices uncut.
		</DIV></DIV></CENTER><P>
The dimers were then tested in exactly the same way as the monomer DH5α to ensure the comparability of the results. Figure 6 shows the comparison of these at time point 4. The comparison of the dimer plasmids to the monomers were extremely variable, with some of the dimers showing lower fluorescence than the monomers and others showing much higher. When comparing the fluorescence in the dimer compared to the monomer the positive control fluorescence was over 200% higher, test device 2 doubled, test device 5 had an 800% increase in fluorescence and test device 6 doubled. As for the ones where a decrease was seen, test device 1 showed only 2% of the fluorescence of the monomer when in its dimeric form, with test device 3 showing half fluorescence and test device 4 showing 90% fluorescence.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 6</B> Shows the OD600 corrected fluorescence values for the two controls and the 6 test devices (average fluorescence/ average OD600) for the monomers in DH5α and the dimers in DH5α. Error bars are standard deviation. AU= arbitrary units
		</DIV></DIV></CENTER><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><H3><SPAN class="mw-headline" id="NEB5.CE.B1_2">NEB5α</SPAN></H3><P>From our results, it seems that the NEB5α and DH5α<I>E. coli</I> have different expression levels with some of the test devices showing double –or in some cases more than double- the expression when in NEB5α and so NEB5α isn’t suitable when attempting to compare results to DH5α. However, it isn’t clear where the differing in expression is likely to have come from and so more analysis of this strain needs to be carried out and we would advise that in future InterLab studies only the DH5α strain should be used. No commercial derivatives should be used as these cause alterations to the results.
</P><H3><SPAN class="mw-headline" id="Dimers_2">Dimers</SPAN></H3><P>The results for the dimers don’t follow a clear pattern and it is clear that their behaviours is extremely unpredictable. As test device 1 had an almost complete loss of fluorescence when in the dimeric form this was sent for sequencing to see if the dimerization had caused any alterations to the sequence. It was found that TD1 had a deletion in the promoter sequence. Whether this is due to a random mutation or if it is a survival technique by the <I> E.coli </I> to prevent over expression of GFP and cell death is still unclear. Sequencing of the remaining 7 dimer test devices should be carried out to indicate if it is alterations in the genes that have affected the expression levels in the other dimers or if it is other factors that are changing the gene expression. More work needs to be done to fully understand the affect dimers have on gene expression. It can be concluded, however, that dimers need to be avoided when undertaking such experiments such as the InterLab as it is clear that they do affect the expression levels and all results obtained. 
</P><H3><SPAN class="mw-headline" id="Concluding_Remarks">Concluding Remarks</SPAN></H3><P>It can be seen from the above results that it is extremely important that when data regarding expression levels in <I>E. coli</I> is going to be collated from many different research groups that all testing is carried out in genetically identical strains as even closely related derivatives can cause alterations to expression levels and cause outlier data points. Secondly, as dimers can cause drastic alterations to results when looking at gene expression levels it is extremely important to avoid them. We would recommend that all future InterLab teams (and in fact anyone carrying out gene expression studies using plasmids) should check all of their test devices uncut on a gel before testing is carried out as this is the quickest and simplest way of identifying dimers and avoid outlier results. 
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><SMALL><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Thermofisher.com. (2017). <I>DH5Alpha Genotypes | Thermo Fisher Scientific.</I> [online] Available at: <A rel="nofollow" class="external free" href="https://www.thermofisher.com/uk/en/home/life-science/cloning/competent-cells-for-transformation/chemically-competent/dh5alpha-genotypes.html">https://www.thermofisher.com/uk/en/home/life-science/cloning/competent-cells-for-transformation/chemically-competent/dh5alpha-genotypes.html</A> [Accessed 26 Sep. 2017].</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Biolabs, N. (2017). <I>NEB® 5-alpha Competent E. coli (High Efficiency)</I> | NEB. [online] Neb.com. Available at: <A rel="nofollow" class="external free" href="https://www.neb.com/products/c2987-neb-5-alpha-competent-e-coli-high-efficiency">https://www.neb.com/products/c2987-neb-5-alpha-competent-e-coli-high-efficiency</A> [Accessed 26 Sep. 2017].</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">iGEM Glasgow 2016 (2017). <I>Team:Glasgow/Interlab - 2016.igem.org</I>. [online] 2016.igem.org. Available at: <A rel="nofollow" class="external free" href="https://2016.igem.org/Team:Glasgow/Interlab">https://2016.igem.org/Team:Glasgow/Interlab</A> [Accessed 25 Sep. 2017].</SPAN></LI><LI id="cite_note-name-4"><SPAN class="mw-cite-backlink">↑ <SUP><A href="#cite_ref-name_4-0">4.0</A></SUP><SUP><A href="#cite_ref-name_4-1">4.1</A></SUP></SPAN><SPAN class="reference-text"> Anon, (2017). [online] Available at: <A rel="nofollow" class="external free" href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf</A> [Accessed 16 Jul. 2017]. </SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">↑</A></SPAN><SPAN class="reference-text"> Summers, D.K. &amp; Sherratt, D.J. Resolution of ColE1 dimers requires a DNA sequence implicated in the three-dimensional organization of the cer site. <I>EMBO J 7</I>, 851-8 (1988).</SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text"> Summers, D. and Sherratt, D. (1984). Multimerization of high copy number plasmids causes instability: Cole 1 encodes a determinant essential for plasmid monomerization and stability. <I>Cell</I>, 36(4), pp.1097-1103. </SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text"> Cgsc2.biology.yale.edu. (2017). <I>Strain - JC8679</I>. [online] Available at: <A rel="nofollow" class="external free" href="https://cgsc2.biology.yale.edu/Strain.php?ID=12925">https://cgsc2.biology.yale.edu/Strain.php?ID=12925</A> [Accessed 20 Oct. 2017]. </SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">↑</A></SPAN><SPAN class="reference-text">Ncbi.nlm.nih.gov. (2017). <I>recE Rac prophage; exonuclease VIII, 5' to 3' specific dsDNA exonuclease [Escherichia coli str. K-12 substr. MG1655] - Gene - NCBI.</I> [online] Available at: <A rel="nofollow" class="external free" href="https://www.ncbi.nlm.nih.gov/gene/945918">https://www.ncbi.nlm.nih.gov/gene/945918</A> [Accessed 20 Oct. 2017] </SPAN></LI></OL></SMALL></DIV></DIV><FOOTER class="marketing-site-footer"><DIV class="row medium-unstack"><DIV class="medium-4 columns"><H4 class="marketing-site-footer-name">iGEM 2017 Glasgow</H4><P>Using synthetic biology to create a biosensor for detection of <I>Campylobacter jejuni</I>, a bacteria linked to food poisoning.</P></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Contact Info</H4><DIV class="marketing-site-footer-block"><P>                    iGEM 2017 Glasgow 
Wilkins Teaching Laboratory
                    Bower Building 
                    University of Glasgow</P></DIV><DIV class="marketing-site-footer-block"><P>igem.glasgow2017@gmail.com</P></DIV></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Sponsors</H4></DIV></DIV><DIV class="marketing-site-footer-bottom"><DIV class="row large-unstack align-middle"><DIV class="column"><UL class="menu marketing-site-footer-bottom-links"><LI><A href="https://2017.igem.org/Team:Glasgow">Home</A></LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>