<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Glasgow_mtlR skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Glasgow/mtlR</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title-bar" data-responsive-toggle="main-menu" data-hide-for="medium"><DIV class="title-bar-title">Glasgow iGEM 2017</DIV></DIV><DIV class="top-bar" id="main-menu"><DIV><DIV class="top-bar-right"><UL class="vertical large-horizontal menu" id="main-nav" data-responsive-menu="medium-accordion large-dropdown"><LI><A class="headline" href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Description">Project Background</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/mtlR">Exploiting the Mannitol regulator MtlR</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/araC">Mutagenesis of AraC</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/QuorumSensing">Quorum Sensing </A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/ANDGate">AND Gate testing</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/XyluloseBiosynthesis">Xylulose Biosynthesis</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/InterLab">InterLab study</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Measurement">Measurement</A></LI></UL><LI><A href="https://2017.igem.org/Team:Glasgow/Parts">Parts List</A></LI><LI><A class="headline" href="#">Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Applied_Design">Designing the Biosensor Device</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Hardware">Hardware</A></LI><LI><A href="#">Additional Work</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HydrogelStrip">Hydrogel Strip Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/BacillusSubtilis"><I>Bacillus subtilis</I> Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Collaborations">Collaborations</A></LI><LI><A class="headline" href="#">Practices</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated">Gold and 
                                                 Integrated</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Engagement">Education and Public 
                                                Outreach</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated/PolicyBrief">Policy Brief</A></LI><LI><A class="headline" href="#">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/WeeklyJournal">Weekly Journal</A></LI><LI><A class="headline" href="#">People</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Attributions">Attributions</A></LI><LI><A class="headline" href="#">Accomplishments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Accomplishments">Accomplishments</A></LI></DIV></DIV></DIV><DIV class="photo-block" id="mtlR"><DIV class="text left"><DIV class="title">
			Exploiting the mannitol regulatory protein MtlR
		</DIV></DIV></DIV><DIV class="wiki-block"><DIV id="article"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Aims"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Aims</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Materials_and_Methods"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Results_and_Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Results and Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Plasmid_Assembly:"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Plasmid Assembly:</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Testing_Constructs:"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Testing Constructs:</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Discussion"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Outlook"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Outlook</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Parts_List"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Parts List</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#References"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>When designing our biosensor device, we wanted to use a dual-input system for the detection of <I>Campylobacter jejuni</I> in order to reduce false positive readings. One of these inputs was a genetic circuit with an inducible promoter - therefore, allowing transcription to be induced by the sugar xylulose. Xylulose is a 5-carbon ketopentose sugar which is present on the coat of <I>Campylobacter jejuni</I>, but on the coat of few other bacteria. This page describes the process we undertook to build the genetic circuit and also details the tests we carried out in order to characterise parts from the mannitol operon of <I>Pseudomonas fluorescens.</I></P><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><P>The <I>mtlR</I> subgroup project involved the exploitation of the mannitol operon from <I>P. fluorescens</I><SUP id="cite_ref-Liu_1-0" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP>combined with a reporter construct to detect the presence of <I>C. jejuni</I>.  
Mannitol is a readily available sugar alcohol which is used by many organisms in nature.
<I>P. fluorescens</I>, a gram-negative bacterium commonly found in water and soil, can utilize this sugar as its sole energy source as it has organised its mannitol utilization gene set into an operon to transport and convert mannitol (Figure 1). Hoffman and Atlenbuchner <SUP id="cite_ref-name_2-0" class="reference"><A href="#cite_note-name-2">[2]</A></SUP> described the MtlR regulator from <I>Pseudomonas</I> as a multi-sugar transcriptional regulator (Hoffmann and Altenbuchner, 2015). <SUP id="cite_ref-name_2-1" class="reference"><A href="#cite_note-name-2">[2]</A></SUP></P><P>Previous studies<SUP id="cite_ref-name_2-2" class="reference"><A href="#cite_note-name-2">[2]</A></SUP> showed that these sugars switch on transcription of a cluster of genes which are involved in the transportation and metabolism of mannitol.  
In particular, the mannitol operon &quot;repressor&quot; (MtlR) activates transcription at the <I>mtlE</I> promoter in the presence of polyol sugars, such as mannitol, arabitol or glucitol<SUP id="cite_ref-name_2-3" class="reference"><A href="#cite_note-name-2">[2]</A></SUP>. In the absence of the sugars, the protein acts in a repressive manner, ensuring no transcription from P<I>mtlE</I>. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 1: Structure of the mannitol operon, in the presence of the sugars tested in Liu et al (2015)<SUP id="cite_ref-Liu_1-1" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP> - Ribose, Mannitol, Fructose, Xylulose, Xylose, Sucrose, and Sorbitol.</B>
 After expression, MtlR protein binds to the sugars and then binds and activates transcription from P<I>mtlE</I>, allowing <I>mtlE-mtlZ</I> to be transcribed. Adapted from Hoffman and Altenbuchner Figure 1 <SUP id="cite_ref-name_2-4" class="reference"><A href="#cite_note-name-2">[2]</A></SUP></DIV></DIV></CENTER><P>
The above figure shows the layout of the mannitol operon. MtlR protein is expressed from the upstream <I>mtlR</I> gene and initiates transcription at the <I>mtlE</I> promoter in the presence of certain sugars. Downstream genes involved in sugar transport and conversion are then transcribed. 
</P><P>Liu et al (2015) <SUP id="cite_ref-Liu_1-2" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP> studied the induction of the operon through the sugar xylose, an aldopentose isolated from plants. However, their study found that xylulose (the first breakdown product of xylose) was in fact the sugar that induced the operon rather than xylose. This study expressed <I>lacZ</I> from the P<I>mtlE</I> promoter on the chromosome of <I>P. fluorescens </I> . Liu et al <SUP id="cite_ref-Liu_1-3" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP> then measured beta-galactosidase (encoded by <I>lacZ</I>) activity in the presence of different sugars. They used a substrate for beta galactosidase that became fluorescent when cleaved by LacZ, and measured fluorescence in a plate reader. The sugars tested included sorbitol, mannitol, xylose, ribose, fructose and arabinose. In the absence of sugar it was demonstrated that the MtlR protein functions as a repressor and not an activator. Only upon induction via the sugar xylulose, P<I>mtlE</I> became active and was able to induce <I>lacZ</I> expression.
</P><P>We decided to use this information in construction of our <I>Campylobacter</I> biosensor â€“ it was shown in a study by Gilbert et al (2015) <SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP> that xylulose is a component of the bacterial coat of <I>Campylobacter</I> specifically but very few other bacterial strains. Therefore, upon exploitation of the P<I>mtlE</I> promoter and the MtlR transcriptional activator and using xylulose as a trigger, we can infer the presence of <I>Campylobacter</I>. 
</P><P>Our biosensor used a two plasmid system to show the interaction between MtlR and xylulose. The first plasmid used the tetracycline regulated Ptet promoter <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_R0040">BBa_R0040</A> to selectively express the <I>mtlR</I> gene.  Upon transcription and translation, the MtlR protein is then available to interact with xylulose. The second (reporter) plasmid contained the mtlE promoter and a downstream GFP gene. In a finished biosensor device, a higher level of fluorescence of GFP would indicate the presence of xylulose in a sample through the interaction of the MtlR-xylulose complex with the mtlE promoter, driving transcription of the GFP gene.
</P><P>The mannitol regulatory system will be combined with the Autoinducer-2 <A rel="nofollow" class="external text" href="https://2017.igem.org/Team:Glasgow/QuorumSensing">quorum sensing regulatory system</A> to create a two part system that will provide a reliable result in the presence of xylulose. With the addition of an <A rel="nofollow" class="external text" href="https://2017.igem.org/Team:Glasgow/ANDGate">AND gate</A>, we created an all or none system, whereby both inputs - xylulose AND autoinducer-2 - must be present to provide a positive result - e.g. GFP fluorescence - after swabbing a potentially contaminated surface. 
</P><H2><SPAN class="mw-headline" id="Aims">Aims</SPAN></H2><UL><LI><B><U>Aim One</U></B></LI></UL><P>To utilize parts of the <I>P. fluorescens</I> mannitol operon to create a system that will identify the presence of xylulose from <I>campylobacter</I></P><UL><LI><B><U>Aim Two</U></B></LI></UL><P>To create a regulatory plasmid expressing mtlR and reporter plasmid expressing mtlE and GFP.
</P><UL><LI><B><U>Aim Three</U></B></LI></UL><P>To study the sugar induction of our constructs and test the viability of these in detecting different sugars 
</P><H2><SPAN class="mw-headline" id="Materials_and_Methods">Materials and Methods</SPAN></H2><P>A standard collection of <A rel="nofollow" class="external text" href="https://2017.igem.org/Team:Glasgow/Protocols">protocols</A> were followed by all subgroups in the lab to assemble BioBrick constructs and have not been described in detail here. 
</P><P>In order to test the activity of our mtlR regulatory system under different sugar induction conditions we studied the basal levels of GFP fluorescence using a 96 well plate in the FLUOstar omega plate reader (BMG). Each plasmid was grown in an overnight culture of LB with appropriate antibiotics for each plasmid (chloramphenicol and kanamycin) and then diluted 1:100 with fresh LB and placed into a black bottomed 96 well plate. All readings were done in the plate reader at 37Â°C shaking at 200 RPM. GFP excitation and emission levels were read, excitation at 485nm and emission at 530nm, every hour for 8 hours.
</P><H2><SPAN class="mw-headline" id="Results_and_Discussion">Results and Discussion</SPAN></H2><H3><SPAN class="mw-headline" id="Plasmid_Assembly:"><U>Plasmid Assembly</U>:</SPAN></H3><P>We constructed two plasmid constructs- the regulatory plasmid <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442202">BBa_K2442202</A> responsible for the transcription and translation of the MtlR protein, and the reporter plasmid which would allow for a measurable response to the MtlR-xylulose interaction. 
mtlR coding sequence was ordered as a G-block construct from IDT and was ligated into pSB1C3. As this was to be under the control of a tet promoter, the tet promoter and medium RBS <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0032">B0032</A> were ordered as oligos from IDT and were ligated into a separate pSB1C3 plasmid. In order for the mtlR sequence to be behind the promoter and RBS, mtlR was restriction digested from its plasmid and ligated into the promoter plasmid. This produced our final regulatory plasmid.
</P><P>The reporter plasmid was constructed by first ligating the mtlE promoter into pSB1C3. We ordered the sequence Liu et al (2015) <SUP id="cite_ref-Liu_1-4" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP> used, which contained the native promoter and RBS. We also decided to experiment with a variety of RBS in order to find the optimal one for our genetic circuit. We used the male promoter ligated to two different RBS, one stronger <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0034">(B0034)</A> and one weaker <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0032">(B0032)</A>. Overall we created three reporter constructs, the wild type reporter plasmid <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442203">(BBa_K2442203)</A>, and two variants, <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442204">BBa_K2442204</A> and <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442205">BBa_K2442205</A>.  These variants were ordered as single-stranded oligonucleotides, annealed together (to make double-stranded oligonucleotides) and then ligated into pSB1C3. We then used GFP from the kitplate <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_E0040">(E0040)</A>. This was restriction digested out of the plasmid and ligated behind Ptet and the various RBS. 
</P><P>In order to test in the presence of both plasmids, the gene construct present in the reporter plasmid was built in pSB1C3 and then digested and ligated into a new plasmid, pSB3K3. This ensured both plasmids had a different origin of replication and therefore could both be present in one cell. It also ensured we could select for cells which contained both the plasmids required to carry out testing.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 2: Diagrams of our plasmid constructs.</B> A: Regulatory plasmid construct. mtlR gene is under the regulation of Ptet and the RBS B0032. B: Reporter plasmid construct. GFP gene, E0040, under the regulation of PmtlE and a number of variant RBS, the native RBS from Liu et al (2015) <SUP id="cite_ref-Liu_1-5" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP> paper, B0032 and B0034.
		</DIV></DIV></CENTER><H3><SPAN class="mw-headline" id="Testing_Constructs:"><U>Testing Constructs</U>:</SPAN></H3><P>Our aim was to replicate the findings of Liu et al <SUP id="cite_ref-Liu_1-6" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP> and as such we studied a number of sugars and recorded GFP output in the presence of different inducers. Liu et al achieved these results by studying <I>P. fluorescens</I> - however, we are testing these constructs for the first time in <I>E. coli</I>. To replicate their results, each reporter variant was tested with each sugar and the reporter variants and regulatory plasmid were tested together with each sugar. These sugars were selected based on their paper <SUP id="cite_ref-Liu_1-7" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP> and other similar sugars available in the lab were used. We tested these combinations with 6 different sugars; mannitol, sorbitol, sucrose, xylose, ribose, and fructose. Our controls were DH5Î± empty cells, fresh LB, and each plasmid alone. 
The two constructs were individually tested with a variety of sugars in order to record the fluorescence in the absence of MtlR. Controls were set up without the constructs in order to measure the basal level of GFP fluorescence. Our results showed: 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 3: Construct testing.</B>The graph shows OD normalised fluorescence minus L-broth background for each sugar condition. The OD reflects the fluorescence of each construct tested in the presence of sugar. Ribose and sorbitol show the greatest fluorescence response across the plasmids with the regulatory plus reporter plasmid showing the overall greatest fluorescence levels. The regulatory constructs are in pSB1C3 whilst the reporter plasmids are in PSB3K3. DH5Î± was used as a control.
		</DIV></DIV></CENTER><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>Liu et al (2015) <SUP id="cite_ref-Liu_1-8" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP> carried out similar experiments to those conducted in this project utilizing B-galactosidase as a reporter to look at the levels of gene expression after inducing the mannitol regulator. They studied the induction of the reporter plasmid in a wild type system and in the absence of mtlR. Their results, figure 4, show around a 7 fold increase in B-galactosidase expression in the presence of mannitol and sorbitol in the wilt type, whilst the other sugars they tested with (xylose, fructose, ribose and xylulose) show little increase in response. Upon removal of mtlR, the Î²-galactosidase response in the presence of mannitol noticeably drops to the levels of the other sugars. In contrast, sorbitol continues to induce the system. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 4: Results from Liu et al<SUP id="cite_ref-name_2-5" class="reference"><A href="#cite_note-name-2">[2]</A></SUP></B> Shows the response of the mtlE promoter induced by six different sugars. The left hand side of the graph shows the wild type system, whilst the right hand side shows the same system upon removal of mtlR. Adapted from Figure S10 of Liu et al (2015) <SUP id="cite_ref-Liu_1-9" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP></DIV></DIV></CENTER><P>
The results obtained from our tests do not reflect the findings of Liu et al <SUP id="cite_ref-Liu_1-10" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP>. We got positive results for the working of the regulatory protein, mtlR and its regulated promoter, PmtlE in <I>E. coli</I> in response to ribose and sorbitol. Whilst Liu et al demonstrated a strong response to sorbitol and mannitol, our constructs showed ribose and sorbitol evoke the largest fluorescence response across all plasmids. A potential reason our results do not reflect those of Liu et al is the use of different organisms. Our findings could show that <I>E. coli</I> exhibits a different response to ribose and mannitol than <I>Pseudomonas fluorescens</I>. E. coli may not utilize these sugars in induction of this system and therefore downstream genes are not turned on as they are in <I>Pseudomonas</I>. 
</P><P>The use of the reporter plasmid alone with varying sugars was to show the requirement of mtlR in this construct. However, Figure 3 shows a relatively strong reporter response from the reporter plasmid alone. Although the GFP gene is present it lies downstream of the mtlE promoter, which is activated by the MtlR protein in the presence of sugar. In this experiment, the sugar is available however there is no MtlR protein to bind to and activate the promoter and therefore GFP should not be expressed (Fig 5). As the regulatory plasmid does produce a fluorescence response in the absence of the regulatory protein, this suggests that <I>E. coli</I> may have its own regulatory system that can induce the reporter of this construct. <I>E. coli</I> may produce its own protein that can regulate the induction of the mtlE promoter and therefore produce a GFP response. 
We expected that only when both the regulatory and reporter plasmids are present, then fluorescence should be shown (Figure 6). Although we get a response from the reporter plasmid alone, when the regulatory plasmid is introduced there is almost a 2 fold increase in GFP response. This indicated that MtlR does induce this system within <I>E. coli</I> and does impact the expression of downstream genes. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 5: Structure of the mannitol construct used in our experiments.  </B> In the absence of mtlR, the sugar cannot bind to the regulatory protein and therefore there is no induction at pmtlE, as such GFP transcription is repressed, no fluorescence is shown.
		</DIV></DIV></CENTER><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 6: Structure of the mannitol construct used in our experiments. </B>
Expressed MtlR protein binds to the sugars being tested and then initiates transcription through PmtlE, allowing downstream GFP to be transcribed, showing fluorescence.
		</DIV></DIV></CENTER><P>
Our testing however lacks a no inducer control. This control would allow for the understanding of our system and its induction. As the reporter plasmid alone shows fluorescence response, it may be that this system is constitutively on and therefore will always show some fluorescence response. As our results show an almost doubling of fluorescence with added sugar this also indicates that the sugar is needed to induce a meaningful response, however without this control it cannot be certain that the system isnâ€™t always on. 
</P><P>As the WT reporter shows fluorescence both in the presence and absence of MtlR it would be expected that the other reporter variants, WT mtlE with additional B0032 RBS and WT mtlE with additional B0034 RBS, would show similar results. However, these two variants show very little fluorescence activity. This suggests that upon designing these plasmids, there has been an issue in reducing the activity of mtlE. The mtlE promoter gene sequence was taken from the Liu et al paper <SUP id="cite_ref-Liu_1-11" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP> and adapted in order to add different ribosome binding sites. The lack of activity of these two variants suggests that removing the region we depicted as the WT RBS was an important promoter element, and its removal has compromised the working of the variants. In order to understand this further, future experiments using more variants of this promoter and recording their fluorescent response would need to be conducted. 
</P><H2><SPAN class="mw-headline" id="Outlook">Outlook</SPAN></H2><P>Overall, the testing of our constructs shows much variability to the results obtained by Liu et al <SUP id="cite_ref-Liu_1-12" class="reference"><A href="#cite_note-Liu-1">[1]</A></SUP>. Mannitol did not induce our system whilst ribose did, suggesting difference in sugar response between <I>Pseudomonas</I> and<I> E. coli</I> may have played a part. Upon testing all variants, our results show that wild type reporter produces the greatest fluorescence response, both alone as well as with the regulatory plasmid. As the other variables of the reporter plasmid do not fluoresce once induced with sugar, this suggests the modifications made to mtlE have impacted the function of the promoter and thus changed the ability of the sugar-MtlR complex to bind and induce. Unexpectedly, our reporter plasmid shows a fluorescence response in the absence of the regulatory plasmid, indicating that <I>E. coli</I> may have its own regulatory proteins that can control this system similarly. Future work would include finding and knocking out any potential regulatory molecules that are wild type to <I>E. coli.</I></P><H2><SPAN class="mw-headline" id="Parts_List">Parts List</SPAN></H2><UL><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442201">K2442201</A> - mtlR gene </LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442202">K2442202</A> - R0040 + B0032 + mtlR </LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442203">K2442203</A> - pmtlE WT + WT RBS </LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442204">K2442204</A> - pmtlE + B0032 </LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442205">K2442205</A> - PmtlE + B0034 </LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442206">K2442206</A> - PmtlE WT + WT RBS+ E0040 </LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442207">K2442207</A> - PmtlE + B0032 + E0040 </LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442208">K2442208</A> - PmtlE + B0034 + E0040 </LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><SMALL><OL class="references"><LI id="cite_note-Liu-1"><SPAN class="mw-cite-backlink">â†‘ <SUP><A href="#cite_ref-Liu_1-0">1.00</A></SUP><SUP><A href="#cite_ref-Liu_1-1">1.01</A></SUP><SUP><A href="#cite_ref-Liu_1-2">1.02</A></SUP><SUP><A href="#cite_ref-Liu_1-3">1.03</A></SUP><SUP><A href="#cite_ref-Liu_1-4">1.04</A></SUP><SUP><A href="#cite_ref-Liu_1-5">1.05</A></SUP><SUP><A href="#cite_ref-Liu_1-6">1.06</A></SUP><SUP><A href="#cite_ref-Liu_1-7">1.07</A></SUP><SUP><A href="#cite_ref-Liu_1-8">1.08</A></SUP><SUP><A href="#cite_ref-Liu_1-9">1.09</A></SUP><SUP><A href="#cite_ref-Liu_1-10">1.10</A></SUP><SUP><A href="#cite_ref-Liu_1-11">1.11</A></SUP><SUP><A href="#cite_ref-Liu_1-12">1.12</A></SUP></SPAN><SPAN class="reference-text"> Liu, Y., Rainey, P., and Zhang, X. (2015). Molecular mechanisms of xylose utilization by Pseudomonas fluorescens: overlapping genetic responses to xylose, xylulose, ribose and mannitol. </SPAN></LI><LI id="cite_note-name-2"><SPAN class="mw-cite-backlink">â†‘ <SUP><A href="#cite_ref-name_2-0">2.0</A></SUP><SUP><A href="#cite_ref-name_2-1">2.1</A></SUP><SUP><A href="#cite_ref-name_2-2">2.2</A></SUP><SUP><A href="#cite_ref-name_2-3">2.3</A></SUP><SUP><A href="#cite_ref-name_2-4">2.4</A></SUP><SUP><A href="#cite_ref-name_2-5">2.5</A></SUP></SPAN><SPAN class="reference-text"> Hoffmann, J., and Altenbuchner, J. (2015). Functional Characterization of the Mannitol Promoter of Pseudomonas fluorescens DSM 50106 and Its Application for a Mannitol-Inducible Expression System for Pseudomonas putida KT2440.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">â†‘</A></SPAN><SPAN class="reference-text"> Gilbert, M., Mandrell, R., Parker, C., Li, J., and Vinogradov, E. (2015). Structural Analysis of the Capsular Polysaccharide from Campylobacter jejuni RM1221. </SPAN></LI></OL></SMALL></DIV></DIV><FOOTER class="marketing-site-footer"><DIV class="row medium-unstack"><DIV class="medium-4 columns"><H4 class="marketing-site-footer-name">iGEM 2017 Glasgow</H4><P>Using synthetic biology to create a biosensor for detection of <I>Campylobacter jejuni</I>, a bacteria linked to food poisoning.</P></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Contact Info</H4><DIV class="marketing-site-footer-block"><P>                    iGEM 2017 Glasgow 
Wilkins Teaching Laboratory
                    Bower Building 
                    University of Glasgow</P></DIV><DIV class="marketing-site-footer-block"><P>igem.glasgow2017@gmail.com</P></DIV></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Sponsors</H4></DIV></DIV><DIV class="marketing-site-footer-bottom"><DIV class="row large-unstack align-middle"><DIV class="column"><UL class="menu marketing-site-footer-bottom-links"><LI><A href="https://2017.igem.org/Team:Glasgow">Home</A></LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>