<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Glasgow_QuorumSensing skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Glasgow/QuorumSensing</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title-bar" data-responsive-toggle="main-menu" data-hide-for="medium"><DIV class="title-bar-title">Glasgow iGEM 2017</DIV></DIV><DIV class="top-bar" id="main-menu"><DIV><DIV class="top-bar-right"><UL class="vertical large-horizontal menu" id="main-nav" data-responsive-menu="medium-accordion large-dropdown"><LI><A class="headline" href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Description">Project Background</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/mtlR">Exploiting the Mannitol regulator MtlR</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/araC">Mutagenesis of AraC</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/QuorumSensing">Quorum Sensing </A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/ANDGate">AND Gate testing</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/XyluloseBiosynthesis">Xylulose Biosynthesis</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/InterLab">InterLab study</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Measurement">Measurement</A></LI></UL><LI><A href="https://2017.igem.org/Team:Glasgow/Parts">Parts List</A></LI><LI><A class="headline" href="#">Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Applied_Design">Designing the Biosensor Device</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Hardware">Hardware</A></LI><LI><A href="#">Additional Work</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HydrogelStrip">Hydrogel Strip Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/BacillusSubtilis"><I>Bacillus subtilis</I> Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Collaborations">Collaborations</A></LI><LI><A class="headline" href="#">Practices</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated">Gold and 
                                                 Integrated</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Engagement">Education and Public 
                                                Outreach</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated/PolicyBrief">Policy Brief</A></LI><LI><A class="headline" href="#">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/WeeklyJournal">Weekly Journal</A></LI><LI><A class="headline" href="#">People</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Attributions">Attributions</A></LI><LI><A class="headline" href="#">Accomplishments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Accomplishments">Accomplishments</A></LI></DIV></DIV></DIV><DIV class="photo-block" id="QuorumSensing"><DIV class="text left"><DIV class="title">
			Quorum Sensing
		</DIV></DIV></DIV><DIV class="wiki-block"><DIV id="article"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Aims"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Aims</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Materials_and_Methods"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Ultramer_Oligo_Annealing"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Ultramer Oligo Annealing</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#PCR_Amplification_of_lsrR-lsrA_Intergenic_Region"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">PCR Amplification of <I>lsrR</I>-<I>lsrA</I> Intergenic Region</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Fluorescence_Readings"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Fluorescence Readings</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Results_and_Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Results and Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Assembly_of_PlsrA_and_PlsrA.2B14"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Assembly of P<SUB>lsrA</SUB> and P<SUB>lsrA</SUB>+14</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Assembly_of_lsrR-lsrA_Intergenic_Region"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Assembly of <I>lsrR</I>-<I>lsrA</I> Intergenic Region</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Final_Constructs"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Final Constructs</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Construct_Testing"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Construct Testing</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Discussion"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>Quorum sensing is used by many types of bacteria including <I>Escherichia coli (E. coli)</I> and <I>Campylobacter</I>. The system naturally used by <I>E. coli</I> which involves the LsrR protein binding to and repressing the LsrA promoter (P<SUB>lsrA</SUB>) was utilised in this subproject. I constructed a plasmid with <I>E. coli</I> P<SUB>lsrA</SUB> followed by green fluorescent protein (GFP). This promoter should be turned off in the presence of the chromosomal encoded LsrR protein. When Autoinducer-2 (AI-2) is present, it binds to the LsrR protein and stops it from repressing P<SUB>lsrA</SUB>, allowing P<SUB>lsrA</SUB> to start transcription of GFP. I expected the results to show an increase in fluorescence when the plasmid containing P<SUB>lsrA</SUB> was introduced into cells which make AI-2, such as DS941 <I>E. coli</I>, when compared to the same construct in cells that don’t make AI-2, such as DH5α <I>E. coli</I>. The results did not show an increase in fluorescence when comparing cells not exposed to AI-2 to cells exposed to AI-2. 
</P><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><P>We designed our Campylobacter biosensor in the form of an AND-gate, which requires two positive inputs to yield an output signal. In our design one of these input signals would be the presence of the rare sugar xylulose (see <A rel="nofollow" class="external text" href="https://2017.igem.org/Team:Glasgow/mtlR">mtlR</A> subpage) from the <I>Campylobacter</I> cellular capsule. For the second input signal we intended to exploit the secretion of quorum-sensing molecules, AI-2, by <I>Campylobacter</I>. Our final biosensor device would only report an output signal of GFP fluorescence if both inputs (Xylulose AND AI-2) were present, which would reduce the possibility of false-positive results.
</P><P><I>E. coli</I>, and many other bacteria including <I>Campylobacter</I>, use quorum sensing to sense population density and coordinate gene expression dependent on this. Signalling molecules, such as AI-2, are constitutively produced by bacteria. AI-2 is a furanosyl borate diester which can be recognised and produced by many gram-positive and gram-negative bacteria. AI-2 is a byproduct of the activated methyl cycle which is responsible for recycling S-Adenosyl-L-Methionine (SAM). During this cycle, the LuxS enzyme produces the precursor of AI-2, 4,5-Dihydroxy-2,3-Pentanedione (DPD).  DPD then spontaneously becomes cyclic forming AI-2 <SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>. Cyclic forms of DPD can be seen in figure 1. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 1. Structure of AI-2.</B> Conversion between the cyclisized molecules is spontaneous so isolation for testing each compound is very difficult. Thus, it is impossible to know which form of cylicised DPD is recognised by E. coli. <SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP></DIV></DIV></CENTER><P>
AI-2 is then imported into bacteria by the LsrABCD transporter and is subsequently phosphorylated by LsrK kinase. Transcription of the lsrABCD operon is repressed by the LsrR repressor, a DNA binding protein that binds to P<SUB>lsrA</SUB>. However, phosphorylated AI-2 is bound by LsrR changing its conformation so that LsrR no longer binds to DNA. Thus phosphorylated AI-2 induces expression of the lsrABCD operon <SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP>. 
</P><P>Previous iGEM teams have used a similar process to detect AI-2 before. The Singapore 2008 iGEM team used a sequence from <I>Rahnella aqualitis</I> which was characterised by researchers in 2000 <SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP>. This promoter was tested by the Tokyo 2011 iGEM team who discovered it did not work as intended. They then redesigned the promotor using a sequence of the <I>lsrA</I> promoter from <I>E. coli</I><SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP>. LsrR is encoded just upstream of the <I>lsrABCD</I> genes in <I>E. coli</I>, and the region between <I>lsrR</I> and <I>lsrA</I> contains the promoter and regulatory sequences for the <I>lsrABCD</I> operon.  However, further research found that the Tokyo team missed out the last 14 bases of the promoter sequence which are important for LsrR binding. Therefore, the same sequence that the Tokyo team used was ordered from IDT along with that sequence with the 14 based included on so that the whole LsrR binding site could be tested. 
</P><P>Three different promoters were ordered from IDT. The nucleotide sequence of the whole intergenic region between the <I>lsrR</I> gene and the <I>lsrA</I> gene can be seen in figure 2. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 2. <I>lsrR</I>-<I>lsrA</I> intergenic region. </B>The green and orange sections are where the LsrR repressor protein binds. The underlined sections are the -35 and -10 regions of the promoters.
		</DIV></DIV></CENTER><UL><LI> LsrR promoter - this region can be seen between the { } brackets. </LI><LI> LsrA promoter - this region can be seen between the [ ] brackets. This is exactly the same sequence that was ordered by the Tokyo 2011 team. </LI><LI> LsrA+14 promoter - this region can be seen between the second set of { } brackets. This includes the sequence ordered by the Tokyo 2011 team with the last section of the promoter is included. </LI><LI><I>lsrR</I>-<I>lsrA</I> intergenic region - this is the full sequence of nucleotides seen in figure 2. </LI></UL><H2><SPAN class="mw-headline" id="Aims">Aims</SPAN></H2><UL><LI> Transform P<SUB>lsrA</SUB>, P<SUB>lsrA</SUB>+14, and <I>lsrR</I>-<I>lsrA</I> intergenic region promoters into plasmids in <I>E. coli</I>. </LI></UL><UL><LI> Test these plasmids within two different cells; DH5α and DS941. DH5α does not make AI-2 because there is a frameshift mutation in the LuxS gene <SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP> and DS941 does make AI-2. This allows for comparison of the constructs when AI-2 is present and when it is not. </LI></UL><H2><SPAN class="mw-headline" id="Materials_and_Methods">Materials and Methods</SPAN></H2><H3><SPAN class="mw-headline" id="Ultramer_Oligo_Annealing">Ultramer Oligo Annealing</SPAN></H3><P>The ultramers, P<SUB>lsrA</SUB> and P<SUB>lsrA</SUB>+14, were 97 and 111 nucleotides long respectively. They were ordered from IDT as single stranded DNA. The annealed ultramers were designed to have sticky ends compatible with EcoRI and SpeI for ligation upstream of other biobrick parts.
</P><UL><LI> 40µl TE buffer added to the forward and reverse single stranded ultramer to resuspend the ultramers. These were vortexed and left for 10 minutes then vortexed again. </LI><LI> 10µl of the forward ultramer from step 1 and 10µl of the reverse ultramer from step 1 were added to 80µl of TE buffer. </LI><LI> This mixture was then left in an 80℃ water bath for 10 minutes for the strands to anneal.</LI><LI> The annealing reaction tube and roughly 200ml of water was scooped out of the water bath in a beaker and left to cool to room temperature overnight. </LI></UL><H3><SPAN class="mw-headline" id="PCR_Amplification_of_lsrR-lsrA_Intergenic_Region">PCR Amplification of <I>lsrR</I>-<I>lsrA</I> Intergenic Region</SPAN></H3><P>The primers ordered from IDT to amplify the <I>lsrR</I>-<I>lsrA</I> intergenic region can be seen in figure 3. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 3. Primers ordered from IDT. </B></DIV></DIV></CENTER><UL><LI> Black: random DNA</LI><LI> Green: prefix </LI><LI> Blue: reverse complement suffix </LI><LI> Pink: forward and reverse primers </LI></UL><P>The template for the PCR was colony PCR of DH5α genomic DNA. The PCR products were analysed by gel electrophoresis, gel extracted then digested at EcoRI and SpeI for ligation. This was then ran on a gel and gel extracted. 
</P><H3><SPAN class="mw-headline" id="Fluorescence_Readings">Fluorescence Readings</SPAN></H3><UL><LI> Overnights were set up from single colonies of strains containing P<SUB>lsrA</SUB> promoter construct. </LI><LI> These were cultured in L-broth with antibiotic at 37℃ shaking at 225rpm. </LI><LI> The following morning, 200µl of each culture was placed in a well of a 96-well plate. </LI><LI> GFP fluorescence was measured at excitation wavelength 485 nm and emission wavelength 530nm, along with cell density measured at 600nm, in a FLUOstar omega plate reader. </LI></UL><H2><SPAN class="mw-headline" id="Results_and_Discussion">Results and Discussion</SPAN></H2><H3><SPAN class="mw-headline" id="Assembly_of_PlsrA_and_PlsrA.2B14">Assembly of P<SUB>lsrA</SUB> and P<SUB>lsrA</SUB>+14</SPAN></H3><P>The P<SUB>lsrA</SUB> was tested so the Tokyo 2011 team’s results could be replicated. 
The P<SUB>lsrA</SUB>+14 was examined to see if our predictions of where the LsrR repressor protein binds were correct. 
</P><P>The ultramers containing P<SUB>lsrA</SUB> and P<SUB>lsrA</SUB>+14 promoter constructs were annealed as described in the methods section. 
They were then ligated into PSB1C3 in front of B0032 (RBS) and E0040 (GFP). 
To verify that the promotors had successfully ligated into the plasmids, a unique restriction site, BSU36I, was exploited. This restriction site was unique to the promoter sequences only, which allows insertion into the plasmid to be verified. The results of this restriction digest can be seen in figure 4. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 4. Gel image of P<SUB>lsrA</SUB> and P<SUB>lsrA</SUB>+14 cut as BSU36I.</B> Ladder used is the 1kb+ ladder. Lanes 1, 3, 5; P<SUB>lsrA</SUB> uncut. Lane 2, 4, 6; P<SUB>lsrA</SUB> cut with BSU36I. Lanes 7, 9, 11; P<SUB>lsrA</SUB>+14 uncut. Lanes P<SUB>lsrA</SUB>+14 cut with BSU36I.
		</DIV></DIV></CENTER><P>
It was expected that the P<SUB>lsrA</SUB> and P<SUB>lsrA</SUB>+14 in pSB1C3 in front of the RBS (B0032) and GFP (E0040) would be 2941bp and 2955, respectively. The results from the gel image show this to be the length of the plasmid in all lanes. The linearised plasmid runs slightly slower than the uncut plasmid because the uncut plasmid is supercoiled so can run through the gel slightly more quickly than the cut plasmid. 
</P><P>The insertion of the parts was verified in all transformants tested, so they were then verified by sequencing. 
</P><UL><LI> P<SUB>lsrA</SUB> followed by B0032 and E0040 were given the biobrick number <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442001">K2442001</A>.</LI><LI> P<SUB>lsrA</SUB>+14 followed by B0032 and E0040 were given the biobrick number <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442003">K2442003</A>.</LI></UL><H3><SPAN class="mw-headline" id="Assembly_of_lsrR-lsrA_Intergenic_Region">Assembly of <I>lsrR</I>-<I>lsrA</I> Intergenic Region</SPAN></H3><P>The <I>lsrR</I>-<I>lsrA</I> intergenic region was tested to investigate how the whole promoter region, including PlsrR, would act in a plasmid. 
</P><P>The PCR product was made as described in methods. 
The PCR product was then digested with EcoRi and SpeI so that it could be ligated into PSB1C3. 
Unfortunately, the PCR product could not be ligated successfully into the PSB1C3 plasmid. 
</P><H3><SPAN class="mw-headline" id="Final_Constructs">Final Constructs</SPAN></H3><P>Overall reporter plasmids made
</P><UL><LI> P<SUB>lsrA</SUB> ligated in front of B0032 (RBS) and E0040 (GFP).</LI><LI>P<SUB>lsrA</SUB>+14 promoter ligated in front of B0032 (RBS) and E0040 (GFP)</LI></UL><P>
The final constructs made can be seen in figure 5.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 5. Final constructs made.</B> The P<SUB>lsrA</SUB> and P<SUB>lsrA</SUB>+14 are followed by RBS and GFP. These are in pSB1C3.
		</DIV></DIV></CENTER><H3><SPAN class="mw-headline" id="Construct_Testing">Construct Testing</SPAN></H3><P>The different strains of <I>E. coli</I> were used to show the difference between fluorescence in cells exposed to AI-2 and cells not exposed to AI-2. DH5α cannot make the LuxS protein as there is a frameshift mutation in the luxS gene <SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP>. However, DS941 can make the LuxS protein so AI-2 is made in these bacterial cells. If AI-2 is present, the P<SUB>lsrA</SUB> should be active and transcribe GFP. If AI-2 is absent, the P<SUB>lsrA</SUB> should be off as it will be bound by the LsrR repressor protein. The results from the plate reader can be seen in figure 6.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 6. Comparison between DH5α cells and DS941 cells.</B> The pink columns are DH5α and the purple columns are DS941.
		</DIV></DIV></CENTER><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>We aimed to make a quorum sensing system to detect the signalling molecule, AI-2. To do this we ordered ultramers for the P<SUB>lsrA</SUB> and P<SUB>lsrA</SUB>+14 promoters and primers to synthesise the complete <I>lsrR</I>-<I>lsrA</I> intergenic region from <I>E. coli</I>. We then aimed to characterise three promoters; P<SUB>lsrA</SUB>, P<SUB>lsrA</SUB>+14, and <I>lsrR</I>-<I>lsrA</I> intergenic region using a FLUROstar plate reader and measuring fluorescence emitted from DS941 and DH5α <I>E. coli</I> cells. 
</P><P>There are no results for the <I>lsrR</I>-<I>lsrA</I> intergenic region because we did not succeed in ligating the PCR fragment into the plasmid. 
</P><P>The results were not as expected. It was hypothesised that the cells with P<SUB>lsrA</SUB> and P<SUB>lsrA</SUB>+14 in DS941 <I>E. coli</I> cells would result in higher fluorescence readings due to GFP being made when the lsrA promoter was activated. However, fluorescence did not increase when comparing a cell with no plasmid to a cell with P<SUB>lsrA</SUB> or P<SUB>lsrA</SUB>+14 promoters.
</P><P>Furthermore, there should have been higher fluorescence in DS941 cells compared to DH5α cells. DS941 makes AI-2 which can repress LsrR protein, meaning the LsrA promoter can be activated. However, the results do not show that there was higher fluorescence in these cells. Due to time constraints it was not possible to investigate further. To gain definitive data on the functions of the P<SUB>lsrA</SUB> promoter we would have required a larger set of repeated data. In addition, an important future control for this experiment would be using LsrR knockout strains in which P<SUB>lsrA</SUB> should be constitutively active. 
</P><P>In future, AI-2 could be bought and added to the overnight culture while <I>E. coli</I> cells are growing to ensure the bacterial cells are exposed to enough AI-2 to bind to the LsrR repressor protein. Another method would be to use one strain of <I>E. coli</I> which produces AI-2 in the culture with another <I>E. coli</I> strain which contains the biosensor plasmid. The <I>E. coli</I> with the biosensor plasmid could detect the level of AI-2 being made by the AI-2 producing <I>E. coli</I> cell. 
</P><P>To fully depress P<SUB>lsrA</SUB>, purified AI-2 would need to be added to the cells. However, AI-2 is highly expensive at $400 for 5mg from <A rel="nofollow" class="external free" href="http://www.ommscientific.com/products.html">http://www.ommscientific.com/products.html</A>. For iGEM, a low-cost purification method needs to be developed before an AI-2 based biosensor could be properly tested. 
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><SMALL><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Vendeville, A., Winzer, K., Heurlier, K., Tang, C. and Hardie, K. (2005). Making 'sense' of metabolism: autoinducer-2, LUXS and pathogenic bacteria. Nature Reviews Microbiology, 3(5), pp.383-396.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Federle, M. (2011). Autoinducer-2-Based Chemical Communication in Bacteria: Complexities of Interspecies Signaling. Contributions to Microbiology, 16, pp.18-32.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text"> Rutherford, S. and Bassler, B. (2012). Bacterial Quorum Sensing: Its Role in Virulence and Possibilities for Its Control. Cold Spring Harbor Perspectives in Medicine, 2(11), pp.a012427-a012427. </SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text"> Seo, J., Song, K., Jang, K., Kim, C., Jung, B. and Rhee, S. (2000). Molecular cloning of a gene encoding the thermoactive levansucrase from Rahnella aquatilis and its growth phase-dependent expression in Escherichia coli. Journal of Biotechnology, 81(1), pp.63-72. </SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">↑</A></SPAN><SPAN class="reference-text"> Xue, T., Zhao, L., Sun, H., Zhou, X. and Sun, B. (2009). LsrR-binding site recognition and regulatory characteristics in Escherichia coli AI-2 quorum sensing. Cell Research, 19(11), pp.1258-1268. </SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text"> Wang, L., Forsyth, M., Hullo, M., Merritt, J. and Sperandio, V. (2017). WikiGenes - Collaborative Publishing. [online] WikiGenes - Collaborative Publishing. Available at: <A rel="nofollow" class="external free" href="https://www.wikigenes.org/e/gene/e/947168.html">https://www.wikigenes.org/e/gene/e/947168.html</A> [Accessed 17 Oct. 2017]. </SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text"> Wang, L., Forsyth, M., Hullo, M., Merritt, J. and Sperandio, V. (2017). WikiGenes - Collaborative Publishing. [online] WikiGenes - Collaborative Publishing. Available at: <A rel="nofollow" class="external free" href="https://www.wikigenes.org/e/gene/e/947168.html">https://www.wikigenes.org/e/gene/e/947168.html</A> [Accessed 17 Oct. 2017].</SPAN></LI></OL></SMALL></DIV></DIV><FOOTER class="marketing-site-footer"><DIV class="row medium-unstack"><DIV class="medium-4 columns"><H4 class="marketing-site-footer-name">iGEM 2017 Glasgow</H4><P>Using synthetic biology to create a biosensor for detection of <I>Campylobacter jejuni</I>, a bacteria linked to food poisoning.</P></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Contact Info</H4><DIV class="marketing-site-footer-block"><P>                    iGEM 2017 Glasgow 
Wilkins Teaching Laboratory
                    Bower Building 
                    University of Glasgow</P></DIV><DIV class="marketing-site-footer-block"><P>igem.glasgow2017@gmail.com</P></DIV></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Sponsors</H4></DIV></DIV><DIV class="marketing-site-footer-bottom"><DIV class="row large-unstack align-middle"><DIV class="column"><UL class="menu marketing-site-footer-bottom-links"><LI><A href="https://2017.igem.org/Team:Glasgow">Home</A></LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>