<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Glasgow_XyluloseBiosynthesis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Glasgow/XyluloseBiosynthesis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title-bar" data-responsive-toggle="main-menu" data-hide-for="medium"><DIV class="title-bar-title">Glasgow iGEM 2017</DIV></DIV><DIV class="top-bar" id="main-menu"><DIV><DIV class="top-bar-right"><UL class="vertical large-horizontal menu" id="main-nav" data-responsive-menu="medium-accordion large-dropdown"><LI><A class="headline" href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Description">Project Background</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/mtlR">Exploiting the Mannitol regulator MtlR</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/araC">Mutagenesis of AraC</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/QuorumSensing">Quorum Sensing </A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/ANDGate">AND Gate testing</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/XyluloseBiosynthesis">Xylulose Biosynthesis</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/InterLab">InterLab study</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Measurement">Measurement</A></LI></UL><LI><A href="https://2017.igem.org/Team:Glasgow/Parts">Parts List</A></LI><LI><A class="headline" href="#">Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Applied_Design">Designing the Biosensor Device</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Hardware">Hardware</A></LI><LI><A href="#">Additional Work</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HydrogelStrip">Hydrogel Strip Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/BacillusSubtilis"><I>Bacillus subtilis</I> Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Collaborations">Collaborations</A></LI><LI><A class="headline" href="#">Practices</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated">Gold and 
                                                 Integrated</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Engagement">Education and Public 
                                                Outreach</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated/PolicyBrief">Policy Brief</A></LI><LI><A class="headline" href="#">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/WeeklyJournal">Weekly Journal</A></LI><LI><A class="headline" href="#">People</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Attributions">Attributions</A></LI><LI><A class="headline" href="#">Accomplishments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Accomplishments">Accomplishments</A></LI></DIV></DIV></DIV><DIV class="photo-block" id="Xylulose"><DIV class="text left"><DIV class="title">
			Xylulose Biosynthesis
		</DIV></DIV></DIV><DIV class="wiki-block"><DIV id="article"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Xylulose"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Xylulose</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#The_Problem"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">The Problem</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#LysS"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">LysS</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Methods"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#SDS-PAGE:"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">SDS-PAGE:</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Expression_Plasmid_Assembly:"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Expression Plasmid Assembly:</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#SDS-PAGE:_2"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">SDS-PAGE:</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Regulatory_Plasmid:"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Regulatory Plasmid:</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Discussion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Proposed_Application_of_Xylose_Isomerase:"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Proposed Application of Xylose Isomerase:</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#Conclusion:"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Conclusion:</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>We are working to create a biosensor to detect xylulose, a sugar found in the capsule of <I>Campylobacter jejuni</I>. Purified xylulose was required for our team’s experiments to develop a xylulose biosensor, but was prohibitively expensive to purchase. The aim of this subproject was therefore to synthesise xylulose from the much cheaper and more readily available sugar xylose. The enzyme xylose isomerase (XI) isomerises xylose to xylulose, and vice versa, in a single step. The <I>Escherichia coli</I> gene for xylose isomerase (<I>xylA</I>) was isolated by PCR and inserted into a BioBrick-compatible plasmid under the control of the highly-active T7 promoter. <I>xylA</I> was expressed at high levels in <I>E. coli</I> BL21(DE3), in which T7 RNA polymerase is IPTG inducible. Additionally, we explored strict regulation of <I>xylA</I> expression to avoid potential toxicity by creating a regulatory plasmid expressing T7 lysozyme. Both the <I>xylA</I> gene and the T7 lysozyme gene were made into biobricks and submitted to the parts registry for use by future iGEM teams.
</P><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><H3><SPAN class="mw-headline" id="Xylulose">Xylulose</SPAN></H3><P>Xylulose is an important metabolite in the pentose sugar metabolic pathway (PSMP), specifically in xylose catabolism. Xylulose's stereoisomer xylose, a ubiquitous sugar in flowering plants, is readily metabolised by bacteria. Xylulose and xylose share the chemical formula C<SUB>5</SUB>H<SUB>10</SUB>O<SUB>5</SUB>, and a molecular mass of 150.3g/mol. The sugars can be isomerised from one to another by the enzyme xylose isomerase (XI) – which is also known commercially as glucose isomerase for its role in the similar isomerisation reaction between glucose to fructose in the food industry, such as to produce high-fructose corn syrup <SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>.
</P><P>The thermodynamics of this reaction are detailed by Tewari, Steckler and Goldberg<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP>, who highlight an equilibrium constant (K) of 0.17 at near room temperature (298K), but a higher K value nearer equilibrium (K=0.42) as temperatures approached ~70°C (342K) (Figure 1). Furthermore, they noted the reaction requires magnesium as a co-factor (from MgCl<SUB>2</SUB>), and was carried out in a phosphate buffer at neutral pH.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 1 - Xylose-Xylulose Equilibrium:</B> Xylose and Xylulose exist in an equilibrium, where the isomerisation reaction is controlled by xylose isomerase (XI). Equilibrium favours xylose over xylulose significantly at room temperature, but becomes more balanced with increasing temperature<SUP id="cite_ref-3" class="reference"><A href="#cite_note-3">[3]</A></SUP></DIV></DIV></CENTER><P>
Xylose utilisation in <I>E. coli</I>, amongst other bacteria, is subject to carbon catabolite repression; it is only utilised after glucose, lactose and arabinose have been consumed <SUP id="cite_ref-4" class="reference"><A href="#cite_note-4">[4]</A></SUP>. In <I>E. coli</I>, xylose enters the cell through an ABC-type transport system, and is later modified by multiple enzymes – all of which are coded in a xylose utilisation (<I>xut</I>) locus. Upon entering the cell, xylose is quickly isomerised to xylulose by XI (coded by the <I>xylA</I> gene), and is then phosphorylated by xylulokinase (<I>xylB</I>) before entering a more complex sugar metabolism (Figure 2). This reaction chain is rapid, and so xylulose is not present for a long period of time in bacterial metabolism.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 2 - Xylose Metabolic pathways:</B> Xylose enters the cell by a transmembrane transport protein, and is converted to D-xylulose through multiple pathways. D-Xylulose must be phosphorylated before it enters the pentose phosphate pathway. This applies to most microbes, including <I>E. coli</I>, and <I>Saccharomyces cerevisiae</I> (Yeast)<SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP></DIV></DIV></CENTER><H3><SPAN class="mw-headline" id="The_Problem">The Problem</SPAN></H3><P>The molecular basis for our <I>C. jejuni</I> biosensor is the rare abundance of xylulose within the bacteria’s capsular polysaccharide (CPS), which could theoretically be exploited as an indicator of the presence of <I>C. jejuni</I> on the surface of raw poultry. We have designed genetic circuits based on a mannitol-sensing transcriptional regulator which has cross-activity with xylulose, or mutants of the araC arabinose regulatory system, which will respond to xylulose with a fluorescent output. The later stages of the project will involve testing these biosensor genetic circuits with xylulose in lab conditions. However, the price of xylulose (Sigma catalogue no. X4625) exceeds our financial capacity: 25mg of D-xylulose costs £194.50 ($258.41), and so to make a 25ml agar plate with 20mM/L concentration of xylulose, we would need 75mg of xylulose for each plate. This would cost £583.50 ($775.23) for a single xylulose screening experiment, and thus, we were unable to purchase the quantities needed to test our biosensors. Therefore, as a subproject, an expression plasmid was developed which utilises a T7 promoter system and the BL21(DE3) strain of <I>E. coli</I> (which encodes T7 RNA polymerase) to express high levels of the XI enzyme. This would allow for the overexpression and purification of the isomerase enzyme for use in the reaction to produce xylulose from xylose.
</P><H3><SPAN class="mw-headline" id="LysS">LysS</SPAN></H3><P>The inducible T7 expression system is a highly controlled system for overproduction of proteins in <I>E. coli</I><SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP>. BL21(DE3) carries a chromosomal copy of the enterobacteria phage T7 RNA polymerase gene expressed from an Isopropyl β-D-1-thiogalactopyranoside (IPTG) inducible lac promoter. T7 RNA polymerase initiates transcription exclusively at T7 promoters. Therefore, a plasmid-borne gene placed downstream of a T7 promoter and an efficient ribosome binding site (RBS) will be highly expressed in an IPTG-inducible fashion in this strain (Figure 3A). As well as IPTG induction of RNA polymerase, two other regulatory mechanisms can be used to fine-tune expression levels: Lac operator sites can be incorporated into the T7 promoter to repress transcription in the absence of IPTG, and the lysozyme protein of T7 (encoded by T7 gene 3.5, but referred to as <I>LysS</I>) inhibits transcription by T7 RNA Polymerase (Figure 3B). Expression of lysozyme has been used extensively to repress basal transcription of the gene in the absence of IPTG to avoid toxic effect of some proteins. Repressing basal expression in the absence of IPTG helps to stabilise expression plasmids, and guard against selections for mutations in the T7 promoter or gene coding region. Two T7 lysozyme expression plasmids are commonly used: pLysS expresses lysozyme at low levels, giving better induced expression levels but slightly higher leakage than pLysE which expresses lysozyme at higher levels <SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP>.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 3 - Genetic and Molecular summary of Xylulose Biosynthesis: A)</B> T7 RNA polymerase only has affinity for T7 promoters, and thus strongly promotes transcription of any downstream genes upon binding. <B>B)</B> The LysS protein inhibits the binding of T7 RNA polymerase, and so decreases promoter activity, thus strictly regulating gene transcription <B>C)</B> The Regulatory plasmid: Uses a constitutive promoter to promote constant expression of the LYSS protein to regulate T7 RNA polymerase activity. <B>D)</B> The expression plasmid: XylA (Xylose Isomerase) is highly expressed by the T7 promoter when a bacterial strain produces T7 RNA polymerase, such as BL21 (DE3).
		</DIV></DIV></CENTER><H2><SPAN class="mw-headline" id="Methods">Methods</SPAN></H2><P>A standard set of protocols were used by all in the lab, and are therefore described in our <A rel="nofollow" class="external text" href="https://2017.igem.org/Team:Glasgow/Protocols">Protocols page</A>. This includes:
</P><UL><LI>DNA miniprep</LI><LI>Restriction digestion</LI><LI>Gel electrophoresis</LI><LI>Gel Staining/Imaging</LI><LI>Gel DNA extraction</LI><LI>DNA ligation</LI><LI>Cell Transformation</LI><LI>PCR</LI></UL><H3><SPAN class="mw-headline" id="SDS-PAGE:">SDS-PAGE:</SPAN></H3><P><B>Sodium Dodecyl Sulfate – Polyacrylamide Gel Electrophoresis (SDS-PAGE)</B> was used to analyse the effect of the expression plasmid (designed to overexpress xylose isomerase) on the entire protein complement of the <I>E. coli</I> BL21(DE3) cells, and compare to normal cell protein expression to conclude whether the expression plasmid works as hypothesised.
</P><P>BL21(DE3) strain <I>E. coli</I> cells were first transformed with the expression plasmid. 3 separate fresh 3ml liquid cultures (labelled cultures A, B and C) were made using separate colonies, L-broth, and with the addition of chloramphenicol (Cm) (1,000X) and glucose (20mM/l). Glucose was added to the media because it precedes lactose in the <I>E. coli</I> sugar-consumption hierarchy, and the lactose-induced lac operon loci is known to transcribe RNA polymerase. As such, a source of glucose suppresses RNA polymerase transcription, and so gene expression only occurs after addition of the lac-operon inducer Isopropyl β-D-1-thiogalactopyranoside (IPTG). 
</P><P>Each culture was then diluted 100-fold (300µl added to 30ml fresh L-broth) in broth containing Cm, and then was incubated (37°C in a shaking rack at 225 rpm) for a period of 3 hours. During this period, cell growth was measured every hour by optical density (OD), using a spectrophotometer (600nm wavelength). Once each culture reached a pre-induction (Pre) OD of roughly 0.5, an aliquot was removed and kept for analysis during SDS-PAGE as samples showing pre-induction cell protein expression. IPTG was added to induce protein expression in the remaining culture. The cells were then incubated (37°C in a shaking rack at 225 rpm) for another 3 hours, recording the post induction (Post) OD of A, B and C.  
</P><P>Following protein expression, 1ml of cultures A, B and C were transferred to a 1.5ml eppendorf tube and spun down (13,000rpm for 5 minutes). The supernatant was then pipetted off, and the cells were resuspended in laemmli loading buffer (950µl 5X laemmli sample buffer + 50µl 0.5M dithriotheitol (DTT)). The proportion of buffer added to each culture was normalised to the culture with the highest OD, where that culture received 1ml of laemmli buffer (Table 1). 
</P><P>The gel used was a 10-well Mini-PROTEAN TGX Precast Gel. 25µl of each of the 6 samples (PreABC and PostABC) were loaded on the gel, along with 20µl of protein marker (Biorad Precision All-Blue Molecular Weight Marker ladder). 25µl of dye was also added to the remaining wells to avoid protein running at an angle. The gel was run until the bands reached the green reference line on the SDS-PAGE kit. The gel was then removed from the precast cassette, and stained with Coomassie blue.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Table 1 - Cell-culture preparation for SDS-PAGE:</B> The optical density was measured using a 600nm spectrophotometer, and represents the amount of bacterial mass present in the culture. The volume of dye added was normalised to the culture with the highest OD.
		</DIV></DIV></CENTER><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Expression_Plasmid_Assembly:">Expression Plasmid Assembly:</SPAN></H3><P>The expression plasmid (given the part name <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442371">BBa_K2442371</A>) was designed to have the <I>xylA</I> gene, from the <I>E. coli</I> genome, inserted behind a T7 promoter (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I719005">BBa_I719005</A>) and a strong RBS (<A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>), which already existed as a composite pSB1C3 biobrick called <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K1321338">BBa_K1321338</A>, which was readily available from the iGEM parts plate. It was also attempted to clone the <I>xylA</I> gene alone in to pSB1C3 to submit as a separate biobrick, but this failed.
</P><P>The <I>xylA</I> gene was first isolated from DH5α cells by colony PCR, where 3 colonies were re-suspended in TE buffer and used as template DNA for 3 separate 50µl PCR reactions. This reduced the chances of only selecting a mutant colony. Primers were designed to anneal flanking regions of the 1320-nucleotide long gene, and to add the biobrick prefix and suffix sequences on to either end of the amplified gene (Table 2). The Melting temperature (Tm) of the primers were within 1°C of one another (~60°C), and had a relatively low guanine/cytosine content (GC%) between 30-50. Furthermore, a 6-nucleotide random DNA sequence was also added to the primers which lets the enzymes cut efficiently.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Table 2 - <I>XylA</I> Colony PCR Primers:</B> The bold sequences are the designed primers, whilst the underlined sequences are the prefix and suffix in the forward and reverse primers respectively. The first six nucleotides of both primers are individual random DNA. The melting temperature (Tm) of the annealing primers (not the prefix, suffix or random DNA sequence) and the cytosine/guanine content (GC%) are also included.
		</DIV></DIV></CENTER><P>
The amplified DNA was then digested with XbaI and PstI restriction enzymes, and ligated with the K1321338 vector, which was digested with SpeI and PstI restriction enzymes, so that the <I>xylA</I> gene ligated behind the T7 promoter and RBS of the K1321338 vector. The ligated plasmids (now K2442371) were then transformed and cloned in DH5α cells. Successful transformation was assessed by initially digesting K2442371 with EcoRI and PstI, and observing that the insert band and vector band corresponded with the expected sizes of 1328bp and 2048bp respectively (Figure 4B). This was later confirmed by sequencing. Following this, the expression plasmid was then isolated and cloned in the desired BL21(DE3) cells, which coded for the T7 RNA polymerase needed for the plasmid to be expressed. 
</P><H4><SPAN class="mw-headline" id="SDS-PAGE:_2">SDS-PAGE:</SPAN></H4><P>To analyse plasmid expression, and specifically the effect on the cellular XI protein content, SDS-PAGE was used to show the protein content of BL21(DE3) cell lysate (lysed by boiling the cells at 95°C) and to compare protein expression of induced cells with non-induced cells (See Methods). We made 3 separate fresh liquid cultures, using colonies from BL21(DE3) cloning, and first calculated their optical densities, so that a proportional volume of buffer could be added to the cultures (Table 1). The cells were resuspended in laemmli loading buffer, and ran on an SDS polyacrylamide gel (Figure 4A). In all induced cultures, a stronger band of ~50kD appears when compared to uninduced cells. This band size corresponds to the predicted molecular weight of the XI protein (49.742kD, calculated by EMBOSS Pepstats) from <I>E. coli.</I> This clearly suggests that XI is being overexpressed by the expression plasmid.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 4 – Expression Plasmid Assembly and Testing: A)</B> Protein expression analysis compared three BL21 (DE3) colonies before (Pre) and after (Post) construct expression was induced with IPTG. Marker used was 1kb+ ladder (Invitrogen) <B>B)</B> Gel electrophoresis of the L18 (Ligation 18) expression plasmid after restriction digestion shows the insert and vector separate, and with expected band sizes.
		</DIV></DIV></CENTER><H3><SPAN class="mw-headline" id="Regulatory_Plasmid:">Regulatory Plasmid:</SPAN></H3><P>The T7 lysozyme gene was isolated from pLysS (where it is called T7 gene 3.5, but referred to here as <I>LysS</I>) using colony PCR, and was then ligated in to pSB1C3 initially (part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442372">K2442372</A>) , although the final plasmid would have used pSB3K3. This is because all biobricks are submitted in pSB1C3, and also to avoid potential competition between plasmids, which can arise if both the expression and regulatory plasmids have the same antibiotic resistance or the same origin of replication. 
</P><P>pLysS was used as template DNA in the 50µl colony PCR reaction, where 3 reactions were carried out using three different colonies, so as not to choose a single mutated colony. Primers were designed (containing biobrick prefix and suffix sequences) to flank the 456 nucleotide <I>lysS</I> gene for amplification (Table 3). Following PCR, the PCR product was digested with EcoRI and PstI to prepare for ligation in to pSB1C3 (digested with the same restriction enzymes). Following the ligation reaction, the plasmids were then transformed and cloned in DH5α cells. To ensure the plasmids were correct, the plasmid was isolated, digested and observed in a gel: a 3ml overnight culture was made using transformant colonies, which were left overnight to grow. The culture was then miniprepped and digested with EcoRI and PstI, before being ran on a 2% agarose gel electrophoresis. The band sizes correlated with the expected vector and insert sizes, indicating successful transformation, and this was later confirmed by sequencing (Figure 5).  
</P><P>Furthermore, a constitutive promoter would be added in front of the <I>lysS</I> gene using PCR: Primers were designed to add a variety of promoter with different strengths before the gene, and thus we could select the lysozyme transcription level with the greatest transformation success. However, <I>lysS</I> was only successfully ligated in to pSB1C3, with no promoter.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Table 3 – <I>LysS</I> colony PCR primers:</B> The bold sequences are the designed primers, whilst the underlined sequences are the prefix and suffix in the forward and reverse primers respectively. The first six nucleotides of both primers are individual random DNA. The melting temperature (Tm) and the cytosine/guanine content (GC% are also included.
		</DIV></DIV></CENTER><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 5 – Gel Image of Regulatory Plasmid:</B> Ligation 11 (L11) was ran against a standard Invitrogen 1kb+ ladder, and the band sizes were compared. After digestion with EcoRI and PstI, the pSB1C3 vector is separated from the <I>lysS</I> gene insert.
		</DIV></DIV></CENTER><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><H3><SPAN class="mw-headline" id="Proposed_Application_of_Xylose_Isomerase:">Proposed Application of Xylose Isomerase:</SPAN></H3><P>The protein expression analysis (SDS-PAGE) shows that when the expression plasmid is induced in <I>E. coli</I> strains containing T7 bacteriophage (such as BL21), there is a significant increase in expression of protein of similar size to XI (49Kd). Whilst this indicates that XI has been over-expressed, no conclusion could be drawn without an enzymatic activity assay. There are simple assays which have been described over the last few decades. Yamanka<SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP> describes an assay involving a D-arabitol Dehydrogenase reaction coupled with the xylose isomerisation reaction, and measures NADH oxidation spectrophotometrically, whilst Schellenberg et al.<SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP> used a similar assay, but using sorbitol dehydrogenase instead of arabitol dehydrogenase. Either of these methods would be sufficient to conclude that xylose isomerase expression is increased in <I>E. coli</I>.
</P><P>The next step would be to utilise the excess enzyme produced by <I>E. coli</I> cells to efficiently isomerise xylose to xylulose. There are two methods to doing this: Purifying the enzyme from the cell lysate to isomerize xylose in optimal conditions for the reaction, or using the entire cell lysate (containing overexpressed XI) to isomerize xylose, before quantifying the xylulose product. Of the two, the first option looked the most promising, as it avoided the potential interference of other proteins that in cell lysate. 
</P><P>XI has been successfully purified using His-tag in other bacteria<SUP id="cite_ref-10" class="reference"><A href="#cite_note-10">[10]</A></SUP>, and the method has been widely used for general protein expression. This involves the addition of a poly-histidine tag on the N-terminal of the XI protein. Following cell lysis, the lysate is then passed over a Nickel-column, which binds the his-tag and extracts the XI protein from the remaining lysate. The protein can then be washed from the column, and isolated. Following purification, the isomerisation reaction environment could then be optimised (i.e. pH, Mg2<SUP>+</SUP> co-factors, buffers, pH, temperature) to modify the equilibrium constant, and a large, pure yield of xylulose could theoretically be isolated. However, this is a lengthy process. It involves using PCR to add the His-tag to the gene, and then cloning the resulting plasmid before the purification process could be done. It was decided that this would not be feasible given the remaining time, but would have been possible and promising otherwise.
</P><H2><SPAN class="mw-headline" id="Conclusion:">Conclusion:</SPAN></H2><P>Xylulose, as a rare sugar and as a potential indicator of the presence of <I>C. jejuni</I>, is a valuable sugar that is an expensive product. Therefore, designing a genetic circuit that exploits <I>E. coli’s</I> natural xylulose biosynthesis reaction and that uses the cheap and ubiquitous xylose sugar to do so, makes the sugar more accessible both for further development of our Campylobacter biosensor, or for other future research involving the xylose metabolism, or the industrial advantages of commercial xylose isomerase. We therefore made the <I>xylA</I> (XI) expression plasmid (part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442371">K2442371</A>) and a simple biobrick containing the lysozyme <I>LysS</I> (part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442372">K2442372</A>) and submitted them for use in future iGEM projects. 
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><SMALL><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">↑</A></SPAN><SPAN class="reference-text">Bhosale, S., Rao, M. and Deshpande, V. (1996). Molecular and industrial aspects of glucose isomerase. Microbiol Rev, 60(2), pp.280-300.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">↑</A></SPAN><SPAN class="reference-text">Tewari, Y., Steckler, D. and Goldberg, R. (1985). Thermodynamics of the conversion of aqueous xylose to xylulose. Biophysical Chemistry, 22(3), pp.181-185.</SPAN></LI><LI id="cite_note-3"><SPAN class="mw-cite-backlink"><A href="#cite_ref-3">↑</A></SPAN><SPAN class="reference-text">Tewari, Y., Steckler, D. and Goldberg, R. (1985). Thermodynamics of the conversion of aqueous xylose to xylulose. Biophysical Chemistry, 22(3), pp.181-185</SPAN></LI><LI id="cite_note-4"><SPAN class="mw-cite-backlink"><A href="#cite_ref-4">↑</A></SPAN><SPAN class="reference-text">Desai, T. and Rao, C. (2009). Regulation of Arabinose and Xylose Metabolism in Escherichia coli. Applied and Environmental Microbiology, 76(5), pp.1524-1532.</SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">↑</A></SPAN><SPAN class="reference-text">Nieves, L., Panyon, L. and Wang, X. (2015). Engineering Sugar Utilization and Microbial Tolerance toward Lignocellulose Conversion. Frontiers in Bioengineering and Biotechnology, 3.</SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">↑</A></SPAN><SPAN class="reference-text">Studier, F. W. &amp; Moffatt, B. A. (1986). Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. J. Mol. Biol. 189, 113-130</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">↑</A></SPAN><SPAN class="reference-text">Studier, F. (1991). Use of bacteriophage T7 lysozyme to improve an inducible T7 expression system. Journal of Molecular Biology, 219(1), pp.37-44.</SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">↑</A></SPAN><SPAN class="reference-text">Yamanka, K. (1969). Enzymatic Assay of D-Xylose Isomerase Activity. Agricultural and Biological Chemistry, 33(6), pp.834-839.</SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">↑</A></SPAN><SPAN class="reference-text">Schellenberg, G., Sarthy, A., Larson, A., Backer, M., Crabb, J., Lidstrom, M., Hall, B. and Furlong, C. (1983). Xylose Isomerase from Escherichia coli. Biol. Chem., 259(11), pp.6826-6832.</SPAN></LI><LI id="cite_note-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-10">↑</A></SPAN><SPAN class="reference-text">Lajoie, C., Kitner, J., Potochnik, S., Townsend, J., Beatty, C. and Kelly, C. (2016). Cloning, expression and characterization of xylose isomerase from the marine bacterium Fulvimarina pelagiin in Escherichia coli. Biotechnology Progress, 32(5), pp.1230-1237.</SPAN></LI></OL></SMALL></DIV></DIV><FOOTER class="marketing-site-footer"><DIV class="row medium-unstack"><DIV class="medium-4 columns"><H4 class="marketing-site-footer-name">iGEM 2017 Glasgow</H4><P>Using synthetic biology to create a biosensor for detection of <I>Campylobacter jejuni</I>, a bacteria linked to food poisoning.</P></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Contact Info</H4><DIV class="marketing-site-footer-block"><P>                    iGEM 2017 Glasgow 
Wilkins Teaching Laboratory
                    Bower Building 
                    University of Glasgow</P></DIV><DIV class="marketing-site-footer-block"><P>igem.glasgow2017@gmail.com</P></DIV></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Sponsors</H4></DIV></DIV><DIV class="marketing-site-footer-bottom"><DIV class="row large-unstack align-middle"><DIV class="column"><UL class="menu marketing-site-footer-bottom-links"><LI><A href="https://2017.igem.org/Team:Glasgow">Home</A></LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>