Team:Glasgow/araC
Glasgow iGEM 2017
Project
Project Background
Demonstrate
Exploiting the Mannitol regulator MtlR
Mutagenesis of AraC
Quorum Sensing
AND Gate testing
Xylulose Biosynthesis
InterLab study
Measurement
Parts List
Engineering
Designing the Biosensor Device
Hardware
Additional Work
Hydrogel Strip Experiments
Bacillus subtilis Experiments
Collaborations
Practices
Silver
Gold and                                                   Integrated
Education and Public                                                  Outreach
Policy Brief
Notebook
Protocols
Weekly Journal
People
Team Members
Attributions
Accomplishments
Accomplishments
Mutagenesis of the L-arabinose regulator AraC
Contents
1 Overview
2 Background
3 Aims
4 Materials and Methods
4.1 Plasmid design
4.2 Library Construction
4.3 Fluorescence-based screening
4.4 Liquid culture fluorescence assay
5 Results
6 Discussion
7 Outlook
8 Parts Made
9 References
Overview
While doing initial research into building a biosensor against Campylobacter jejuni, we created this subproject to  focus on utilization of the L-arabinose sensing AraC regulatory protein and its regulated PBAD promoter in order to construct one of the parts of the final detection system. The regulatory protein AraC binds to operator sequences in the PBAD promoter and activates transcription of downstream genes only when bound to its substrate sugar (L-arabinose). Here we performed saturation mutagenesis on the araC gene to construct AraC mutant library with four randomized ligand-binding pocket amino acid residues corresponding to positions 8, 24, 80 and 82. The AraC mutant library was subjected to fluorescence-based screening using a GFP reporter. Due to unavailability of xylulose we tested for mutants responsive to a similar molecule - xylose. We isolated and sequenced 4 mutants which showed altered effector specificity. Our results demonstrate an approach that allows to engineer a regulatory protein to respond to an effector molecule of interest and represent steps towards developing a functional biosensor to detect C. jejuni on contaminated surfaces. Our protocol can be used by future teams to generate AraC variants responsive to any small molecule of choice, and provides a toolkit for developing biosensors.
Background
Our team aims to construct two alternative regulatory systems as potential components of the biosensor. One of them exploits the mannitol operon, naturally found in Pseudomonas fluorescens [1] . The regulatory protein MtlR activates transcription of downstream genes upon binding to xylulose [2] . However, this approach may have some disadvantages. Coming from a different organism, the MtlR protein may not function in E. coli cells as expected. Furthermore, the MtlR operon has not been previously exploited in biosensors and may not show sufficient specificity to xylulose. Hence, we aim to construct an alternative regulatory system which exploits the L-arabinose operon.
Figure 1: Genes required for catabolism of L-arabinose and the operon components. araC expression is regulated by the PC promoter and is transcribed in the opposite direction from araBAD genes, which are controlled by the PBAD promoter. AraC acts negatively (-) at the PC promoter and negatively or positively (±) at the PBAD promoter. Adapted from Shleif (2000) [3]
The L-arabinose operon is naturally found in Escherichia coli [3] . The regulatory protein, AraC, acts as a dimer. As depicted in Fig. 1, the operon contains two regulatory cis elements: the PC promoter for the transcription of araC, and the PBAD promoter for transcription of genes required for catabolism of L-arabinose. The mechanism of transcriptional regulation has been described by Schleif (2000) and is summarized in Fig. 2. The PBAD promoter is regulated by 3 half sites that each bind to one subunit of AraC - O2, I1, I2. The PC promoter is autoregulated by AraC binding to O1, which is composed of O1L and O1R half sites. The O2 half site required for proper regulation of PBAD is within the araC coding region. In absence of L-arabinose the AraC dimer binds to operator half-sites O2 and I1. This causes DNA looping upstream of the PBAD promoter which represses transcription by excluding RNA polymerase from binding to PBAD. Binding of L-arabinose causes a conformational change in the protein such that the DNA-binding domains of the dimer bind to adjacent I1 and I2 half-sites, giving access for RNA polymerase and cyclic AMP receptor protein (CRP) to bind PBAD. This results in transcriptional activation.
Figure 2: Regulation of the L-arabinose operon by arabinose. In the absence of L-arabinose, AraC dimer binds to O2 and I1 half sites causing DNA looping, which prevents RNA polymerase from accessing PBAD. Upon binding of L–arabinose, the AraC dimer binds I1 and I2 half sites instead, allowing transcription of the polycistronic araBAD mRNA. araC is transcribed in opposite direction from araBAD, and is under control of the PC promoter. RNA polymerase and AraC compete for binding at O1 and PC. Adapted from Shleif (2000) [3]
Several previous studies have shown that AraC protein can be engineered to activate transcription in response to non-native small molecules. Some of them include D-arabinose (enantiomer of L-arabinose) [4] , mevalonate [5] and triacetic acid lactone [6] . Site-saturation mutagenesis of residues positioned within the ligand-binding pocket of AraC, coupled with fluorescence-based cell sorting, allowed the groups to isolate AraC variants with altered effector specificity.  Based on these findings we aimed to use multiple site-saturation mutagenesis and fluorescence-based screening to generate mutant AraC responsive specifically to xylulose. Although xylulose comes in the form of two enantiomers, we chose to focus on D-xylulose, which is the isomer present in the capsule of C. jejuni [7] . Both L-arabinose and D-xylulose are monosaccharides with five carbons, with molecular formulas C5H10O5. As seen in Fig. 3a, arabinose is an aldopentose with an aldehyde functional group at carbon position 1, while xylulose is a ketopentose, possessing a ketone group instead. The 3-dimensional structure of D-xylulose differs from that of L-arabinose, with the most pronounced difference being the 5-membered vs 6-membered heterocyclic ring in its structure. Fig. 3b represents key residues of AraC protein that play a role in ligand binding. Previous studies have shown that mutagenesis of amino acids in positions 8, 24, 80 and 82 is sufficient to change effector specificity [4] .
Due to prohibitively high cost of purified xylulose, we aimed to synthesise our own . In the meantime, due to unavailability of the sugar, we screened for mutants responsive to a similar molecule - xylose.
Figure 3: (a) Structures of L-arabinose and D-xylulose. (b) Crystal strycture of AraC binding pocket with bound L-arabinose (LA), showing key residues that play a role  in ligand binding. Residues targeted for mutagenesis are underlined in red. Water molecules are shown as spheres (adapted from Tang et al, 2008 [4] )
Aims
Aim 1
To split the AraC regulatory system into two plasmids: regulatory plasmid expressing AraC and AraC mutants and reporter plasmid expressing GFP under control of PBAD promoter.
Aim 2
To perform multiple site-saturation mutagenesis on the AraC protein at residue positions 8, 24, 80 and 82.
Aim 3
To isolate AraC variants with altered effector specificity using fluorescence-based screening.
Materials and Methods
Plasmid design
Plasmid construction was as follows.
Oligos containing parts R0011 (LacI-regulated promoter) upstream of ribosome-binding site (RBS) B0032 were synthesised by Integrated DNA Technologies (IDT). Each strand had different, incomplete prefix and suffix sequences, such that when annealed the resulting dsDNA had BioBrick-compatible sticky ends for ligation into EcoRI and PstI digested BioBrick vectors. The resulting product was ligated into high copy number pSB1C3 vector digested with EcoRI and PstI. The ligation reaction was introduced into chemically competent E.coli strain DH5α, and sequence from a chosen colony was verified.
WT araC sequence as described by Miyada et al (1980) [8] was amplified from BBa_I0500 using primers araC_BBPre_F and araC_BBSuf_R (see Table 1) to introduce BioBrick prefix and suffix to both ends of WT araC. The PCR product was subsequently digested with XbaI and PstI. pSB1C3+R0011+B0032 was digested with SpeI and PstI. The two digested parts were ligated to generate K2442104 plasmid with WT araC. Sequence was verified.
PBADmin was synthesised by Integrated DNA Technologies (IDT) as a G-block and digested with EcoRI and PstI. The cut product was then ligated into pSB3K3 vector digested with EcoRI and PstI. The ligation product was introduced into E.coli strain DH5α by chemical transformation and the sequence was verified. Part BBa_I13500 was then digested with enzymes XbaI and PstI and ligated into pSB3K3+PBADmin, digested with SpeI and PstI. This resulted in the final reporter plasmid K2442102 , and the sequence was verified.
Library Construction
Primer sequences are listed in Table 1. araC gene was amplified from the BBa_I0500 part using primers araC_F and araC_R. The PCR product was then used in three PCR reactions with the following sets of primers: araC_Frag1_F and araC_Frag1_R; araC_Frag2_F and araC_Frag2_R; araC_Frag3_F and araC_Frag3_R.
The PCR conditions were:
98°C for 30s
35 cycles of 98°C for 10s, 67.5°C (for F1 and F2) or 59.8°C (for F3) for 30s and 72°C for 1min
Final step of 72°C for 10 mins
The reactions resulted in 3 araC fragments (F1, F2 and F3). Primers araC_Frag1_F, araC_Frag1_R and araC_Frag2_R contain the degenerate sequences NNS at specific triplets so that fragments F1 and F2 contain saturation mutagenesis at codon positions 8, 24, 80 and 82. PCR products were gel purified and DNA concentration of each was measured with a NanoDropTM spectrophotometer. Equimolar aliquots (0.15pmol each) of adjacent fragments were combined (F1+F2 and F2+F3) and PCR-assembled without primers under the following conditions:
98°C for 30s
15 cycles of 98°C for 10s, 60°C for 1min and 72°C for 40s
72C for 10 mins.
15 μl of each reaction product was combined and PCR-assembled without primers under the following conditions:
20 cycles of 98°C for 30s
72°C for 40s
Finally primers araC_BBPre_F and araC_BBSuf_R were added and a final round of PCR was ran with the following programme:
98°C for 30s
30 cycles of 98°C for 10s
60°C for 1 min
72°C for 40 s
72°C for 10 min.
QIAQuick PCR purification of the product was performed according to the Qiagen protocol.  The purified PCR product was then digested with XbaI and PstI and ligated into pSB1C3+R0011+B0032 digested with SpeI and PstI. Ligation reaction was ethanol-precipitated, and transformed into electrocompetent DH5α by electroporation. This resulted in the araC mutant library.  Colonies carrying the mutant library were washed off transformation plates with 1.5ml double distilled H2O, (ddH2O) then transferred to a 1.5ml tube. The cell pellet was centrifuged at room temperature, the supernatant was then discarded and pellet resuspended in 1.5ml ddH2O. This centrifugation-resuspension step was repeated 3 times to remove agar plate debris. Plasmid DNA was extracted using the Qiagen Plasmid MiniPrep Kit with 2 of each PB buffer and PE buffer wash steps to ensure maximum extract purity. Sequencing primer VF2 was then added and sample sent for sequencing.
Table 1: Primer sequences used in library construction. Degenerate sequences are highlighted in magenta. N, any base. S, strong base (G or C). Red, random 6bp sequence. Yellow, BioBrick prefix. Cyan, reverse compliment of BioBrick suffix.
Fluorescence-based screening
Plasmid DNA containing the araC mutant libraries were transformed into DS941 carrying the reporter plasmid K2442102 (in pSB3K3 vector). Transformants were plated on LB agar medium containing chloramphenicol plus kanamycin (to select for K2442102 and K2442104) plus one of the tested inducers: arabinose, xylose or decanal. Concentrations were 40mM for arabinose and xylose and 2mM for decanal. As a control test transformants were also plated on LB medium containing chloramphenicol and kanamycin only. Fluorescence images of the conditional transformation plates were obtained using a GE-Healthcare Typhoon FLA-9500 laser scanner. Excitation was recorded at 473nm, emission was recorded using 520-540nm filter. Colonies which exhibited fluorescence were observed as dark colonies on the scan (Fig. 8). 200 of those which appeared dark on xylose or decanal plates were re-streaked onto plates containing xylose, arabinose, decanal or no additive. Each colony of interest was picked and then immediately short-streaked onto each new condition plate using the same toothpick, into the same position using grids. This method of replica plating ensures the plates can later be aligned and short streaks of the same origin directly compared between the plates. After overnight incubation at 37°C, fluorescence scans of each plate were again obtained using the scan fluorescent scanning method.
Liquid culture fluorescence assay
Colony of interest was inoculated into L-broth containing chloramphenicol and kanamycin. Each liquid culture was grown overnight at 37°C, shaking at 225 rpm. The following day the culture was diluted 1:100 into fresh L-broth containing the above antibiotics plus one of the following inducer conditions:
40mM xylose
40mM arabinose
2mM decanal
200µl of each culture to be tested was placed in a well of a clear-based 6-well plate, then incubated at 37°C shaking at 300 rpm for 12 hours in a BMG Fluorostar Omega fluorescence plate reader. Fluorescence of each culture was measured at 1 hour time intervals for the duration of the culture experiment, using excitation wavelength 485nm and emission wavelength 530nm. Cell growth was simultaneously tracked by measuring optical density at 600nm.
Results
Regulatory plasmid K2442104 contains wild-type araC gene under control of LacI-regulated promoter R0011 linked to RBS B0032. This is to enhance control over production of the AraC, so that the growth of bacteria is not affected by overproduction of the protein. Production of AraC can be stimulated by addition of Isopropyl β-D-1-thiogalactopyranoside (IPTG) to the media. IPTG mimics allolactose and prevents LacI from binding to its operator sequences in the DNA, switching on the R0011 promoter [9] . The construct was ligated into pSB1C3 plasmid backbone (Fig. 4).
The reporter plasmid K2442102 contains GFP gene downstream of minimal PBAD promoter (PBADmin) (Fig. 5A). Minimal PBAD contains minimal DNA sequence from the native PBAD that is sufficient to retain its function (Figure 5B). The promoter retains all the operator sites O1, O2, I1 and I2 required for binding of AraC. As these sites lie within the PC promoter and overlap with AraC coding sequence, the minimal PBAD has the potential araC start codon ATG mutated to AGT to prevent the protein from being made (Fig. 5B). PBADmin was created and tested with the help of Steven Kane, a member of Colloms laboratory. To enable incorporation of PBADmin into a BioBrick-compatible plasmid, we added BioBrick prefix and suffix sequences to either side of the PBADmin sequence (Fig. 5B). Part K2442102 was expected to work as follows: when AraC is activated by its binding substrate, it binds to I1 and I2 half sites within PBADmin and drives expression of GFP, causing the host cell to produce green fluorescence. For the purposes of testing, the construct was ligated into pSB3K3 plasmid backbone. The plasmid has a different origin of replication from that of pSB1C3, therefore both plasmids can co-exist in the same E. coli cell.
Figure 4: Regulatory Plasmid K2442104. The part contains araC or mutant araC amplified from part I0500. Expression of araC is regulated by lacI-regulated promoter R0011 with a downstream RBS B0032. The promoter is IPTG-inducible, but gives some leaky expression even in the absence of IPTG. The construct is flanked by BioBrick prefix and suffix, containing restriction sites for EcoRI, XbaI and SpeI, PstI, respectively. pSB1C3 carries Cm, providing resistance to chloramphenicol. The replication origin is a pUC19-derived pMB1 (copy number of 100-300 per cell)
Figure 5: (A) Reporter plasmid K2442102. The part encodes GFP with the B0034 RBS downstream of minimal PBAD promoter, regulated by AraC. The part is flanked by BioBrick prefix and suffix. pSB3K3 plasmid backbone carries resistance to kanamycin. The vector has p15A pMR101-derived replication origin (copy number 20-30). (B) Annotated sequence of minimal PBAD promoter as ordered as a G-block. AraC potential start codon mutation ATG→AGT position is highlighted in magenta (in reverse compliment). AraC-binding half sites are underlined and annotated. -35 and -10 elements of the PBAD promoter are highlighted in cyan. AraC transcription start site (+1) highlighted in grey. PC promoter and its direction is indicated by an orange arrow. -35 and -10 elements of the PC are indicated. Yellow, BioBrick prefix. Green, BioBrick suffix. Red, random 6 bp sequences to allow enzyme digest.)
The plasmids were all BioBrick compatible and at every stage of cloning were checked with gel electrophoresis and sequencing.  K2442102 and K2442104 were then transformed into DS941 cells and plated onto arabinose conditions to investigate function (Fig. 6). The scan showed that the colonies produced GFP in the presence of arabinose, even without the addition of IPTG (an inducer for the lacI-regulated promoter). The promoter is therefore leaky – background transcription of araC from K2442104 is sufficient to activate PBADmin in the K2442102 construct.
Figure 6: Fluorescence scan of DS941 cells with both reporter and regulatory plasmids incorporated. Cells are plated on nutrient agar supplemented with 40mM arabinose.
Fig. 6 shows every colony to have high levels of fluorescence on arabinose. This confirms that the araC and PBADmin parts still function without being adjacent to one another.
The regulatory plasmid carrying the library of mutant araC was also constructed and transformed as discussed in the methods section. This gave us approximately 24,000 colonies, each containing a ligand-binding pocket variant of the AraC protein. This number however is not sufficient to cover all 160,000 possible variants. The mutant library DNA was confirmed by sequencing. Sequencing showed the mutations had been introduced at the correct positions (Fig. 7). The trace showed obvious NNS mutations at residues 8, 24, 80 and 82. At N positions 4 peaks are visible for any base (N), at S positions 2 peaks are visible for strong bases (S): G or C.
Figure 7: Sequencing trace from mutant library MiniPrep, shows NNS mutations at residues 8, 24, 80 and 82.
MiniPrep DNA of the mutant araC library plasmid and K2442102 (reporter plasmid) were then transformed into E. coli DS941. These cells were then plated onto LB agar containing kanamycin and chloramphenicol with: A) no additive, B) 40mM xylose, C) 40mM arabinose or D) 2mM decanal and then scanned for GFP fluorescence (Fig. 8). The mutants were screened against no additive to look for mutant AraC with constitutive activity. They were screened against arabinose to check for unmutated/ mutated AraC with retained wild-type function. Xylose was used as it has a similar chemical structure to xylulose, but was much more economically viable. Xylose does not naturally stimulate WT AraC [10] . Decanal is a simple aldehyde, with a 3D structure very different to arabinose. It was expected to have no effect on the binding pocket of wild type AraC, however responsive mutants may be generated. We chose to screen against decanal to check whether mutants obtained are responsive specifically to xylose, or are also inducible at other conditions.
Figure 8: Plated E. coli transformants carrying mutant library DNA. Each plate was supplemented with either 40mM arabinose, 40mM xylose, 2mM Decanal or nothing at all (as labelled). The plates have been fluorescence imaged.
The mutant library was screened in four different conditions. Highly fluorescent, non- fluorescent, and colonies with intermediate fluorescence were seen in all conditions (Fig. 8). Multiple plates of each condition were screened. The presence of non-fluorescent colonies on arabinose shows that AraC has been mutated so that it no longer responds to arabinose.  Some fluorescent colonies were observed on plates containing xylose or decanal, suggesting that mutants that responded to these substances might have been obtained. Some fluorescent colonies were even observed with no added sugar (or decanal) in the agar. Presumably these reflect AraC mutants that are in the activated state in the absence of any sugar. Together these data show that the AraC protein has been effectively mutated, as some variations are showing different behaviour to the wild-type protein.
Due to time and labour constraints only around 200 colonies which exhibited fluorescence on either decanal or xylose plates were re-streaked onto plates with all four original conditions (i.e. xylose, arabinose, decanal or no chemical). Colonies were picked by referencing a printed out copy of the fluorescence scan and dotting the colonies which appeared to have strong fluorescence. This allowed for room for error. The fluorescence scans of the restreaked mutants showed a variety of behaviours (Fig. 9).
Figure 9: Fluorescence scans of re-streaks of mutant DS941 colonies which appeared to show fluorescence under xylose or decanal conditions.  Mutants which appeared to have novel behaviour are labelled 1-4.
The restreaks (Fig. 9) were examined and four colonies were chosen for further examination. All four of these colonies originated from the decanal plate. Colony no.1, (circled in green) was picked as it appeared to be fluorescing under decanal, xylose or arabinose conditions and less so when there was no additive in the agar. Colonies 2 and 3 (circled red) appeared to be fluorescing under all four conditions. Colony 4 appeared to only be fluorescing when streaked onto the plate containing decanal.  These four colonies were then picked from the no additive (decanal mutants) plate to be grown in liquid broth (no additive) with antibiotic, overnight. Portions of these overnights were then grown in a plate reader under all four conditions to monitor fluorescence over time (Fig. 10). Biobrick numbers K2442105 - K2442108 were given to the plasmids which contained R0011+ B0032+ a mutant variant of araC. Relevant BioBricks/ composite parts were also monitored for fluorescence to show function when PBAD and araC are split, to reconfirm that our constructs work as intended and to act as controls against the mutant plasmids activity (Fig. 10).
Figure 10: Average relative fluorescence over optical density at hour 8. Shows fluorescence levels under no additive, 40mM Arabinose, 40mM Xylose and 2mM decanal.  The cells which the plasmids have been transformed into are DS941 or DH5α (specified). Biobricks (K244210..) and plasmids (pSB1C3, pSB3K3) specified.
Figure 10 shows that our split constituent parts of I0500 (PBADmin and araC) function correctly. Reading from left to right on Fig. 10: There was negligible fluorescence of DH5α cells when transformed with R0011, B0032 and WT araC in pSB1C3. DH5α cells (which have a chromosomal copy of araC) with  PBADmin and I13500 (GFP) in pSB1C3 backbone (second along) showed 2000x fold increase in fluorescence under arabinose conditions and negligible fluorescence under all other conditions. This shows that the chromosomal copy of araC was sufficient to activate PBADmin and initiated expression of GFP. This reconfirms the functionality of the promoter, as well as proves high efficiency of minimal PBAD and improvement over part BBa_I0500 . Third along in Fig. 10 shows this experiment again, except PBADmin+GFP are in pSB3K3 backbone instead of pSB1C3. This is a lower copy number plasmid than pSB1C3. Again, expression of GFP was only seen under arabinose conditions, however levels of fluorescence were much lower. This suggests that the change in fluorescence is due to the plasmid copy number and must be taken into account in further experiments. The next set of data shows DS941 cells containing PBADmin+GFP in pSB3K3. DS941 does not have a chromosomal copy of araC. There were negligible levels of fluorescence in this instance, showing that araC is necessary for our construct to express GFP.  5th along shows DS941 cells transformed with both the regulatory plasmid (R0011+B0032+WT araC in pSB1C3) and the reporter plasmid PBADmin+GFP in pSB3K3). As they are DS941 cells, the only araC protein present is the one supplied by the regulatory plasmid. The results were comparable to data set 3, when a chromosomal copy was used to activate PBADmin(in pSB3K3), with high levels of expression only when exposed to arabinose. This shows that the coding sequence used for expression of wild-type araC in our construct produces functional protein. Empty DS941 and DH5α cells were also scanned for fluorescence - the levels were very low and have been used as controls to determine basal level for negligible fluorescence.
DS941 cells with the mutant variations of araC in the regulatory plasmid and the reporter plasmid were tested in a separate experiment due to lack of space in the plate reader. Protocol was carried out in the same way.
Mutant 1 (K2442105) showed levels of fluorescence comparable to empty DS941 cells. There was no obvious difference between arabinose, decanal and no additive (Fig. 10) when liquid culture was examined.   Mutant 2 (K2442106) and Mutant 3 (K2442107) both gave an increased level of fluorescence (~100 fold) over no araC plasmid, with or without sugar or decanal additives. Addition of arabinose seemed to give a slight reduction in the level of fluorescence with these two mutants, suggesting that arabinose may be having an inhibitory effect on these variant of AraC. Mutant 4, K2442108, showed no significant fluorescence (Fig. 10) in any conditions, similar to mutant 1. the level of fluorescence observed was similar to that of empty cells and is likely due to natural fluorescence of E. coli.
None of the mutants shows an increase in GFP expression under arabinose conditions. This shows that the AraC molecule has been successfully mutated and is not behaving as the wild-type would do.
The mutant plasmids were sequenced. AraC naturally has proline at residue 8, threonine at residue 24, histidine at 80 and tyrosine at 82. Mutant 1 (K2442105) has the following mutations:
proline-8→glycine (P8G), histidine-80→proline (H80P) and tyrosine-82→tryptophan (Y82W)
Mutant 2 (K2442106) and 3 (K2442107) both had the same mutations as follows:
proline-8→serine (P8S), threonine-24→aspartic acid (T24D), histidine-80→isoleucine (H80I) and tyrosine-82→alanine (Y82A)
Sequencing result of mutant 4 (K2442108) came back inconclusive due to mispriming in the sequencing reaction. Sequencing remains to be repeated.
Discussion
We have successfully shown that constituent parts of BBa_I0500 can function when split into two seperate plasmids.  Figures 5 and 10 show that the wild-type variant of araC under control of R0011 promoter and RBS B0032 on a high copy number plasmid is sufficient to bind to our minimal PBAD promoter and drive transcription of GFP encoded on a low copy number plasmid. The DS941 strain of E. coli does not produce its own copy of the AraC protein, therefore the only AraC which may be binding to the PBADmin promoter is the one transcribed from our construct. We have improved BBa_I0500 by providing additional control over arabinose-inducible systems. AraC can be placed under inducible promoters of choice, taking translational load off growing cells.
Sequencing results confirmed that the protocol described generated desired mutations at amino acid residue positions 8, 20, 24, and 80. The four mutants showed different behaviour to the wild-type AraC. Substitution of these residues to change the effector specificty of AraC was previously done by Tang et al (2008) [4] to generate AraC responsive to the enantiomer D-Arabinose as opposed to L-Arabinose. Naturally AraC does not respond to D-arabinose. L-xylose was chosen by us for mutant screening as it has an identical chemical formula to L-arabinose but a different structural formula, just as L-arabinose and D-arabinose do. Decanal is an aldehyde molecule, just as arabinose, however the 3D structure of the molecule is very different, allowing for investigation of mutant AraC response to different types of molecules.
K2442105 showed no increase in fluorescence under all tested conditions. The mutant AraC has lost its responsiveness to arabinose. As the only mutations are present in the ligand-binding pocket, the protein most likely no longer binds to arabinose, or upon binding no longer undergoes required change in conformation to activate pBAD. The result shows that the mutations present in this mutant variant of AraC do not cause specific activation by any of the conditions tested. However, we tested a very limited number of possible inducers. The protein may be responsive to a molecule not yet identified.
The sequencing results of K2442106 and K2442107 show amino acid changes P8S, T24D, H80I and Y82A. These changes resulted in a decrease in levels of fluorescence under arabinose conditions relative to the no sugar control, and 100x fold increase in fluorescence relative no AraC. This shows that unlike WT AraC, the mutant version is active in absence of arabinose. Fluorescence on the plate with no additive may be explained by loss of the protein's repressive effect on PBAD, which results in activation the promoter even when not bound to its ligand. Alternatively, mutant AraC may be activated by a compound present in the cell or abundant in the LB broth/ LB agar. Interestingly, both colonies showed decrease in levels of fluorescence in presence of arabinose, suggesting that arabinose may be actually having an inhibitory effect on mutant AraC. The change in protein structure and activity would need to be examined further before we could infer how the substitutions in the ligand-binding pocket of AraC are causing the observed response. It is intriguing that both mutants show the same mutations, as the chances of us finding two mutants with identical DNA sequence (out of 1,048,576 possible combinations) are extremely low. However, this is possible because of the way the mutant library was amplifed prior to our screening, allowing us to pick two colonies that had been transformed with duplicates of the same mutant. However, this does reconfirm the activity of this mutant sequence, which was inadvertently replicated and showed the same behaviour.
K2442108, similarly to K2442105 showed negligible expression of GFP. Sequencing result of colony 4 (K2442108) came back inconclusive due to mispriming in the sequencing reaction. This colony, similarly to K2442105 might have a mutated AraC which no longer responds to arabinose, but is possibly induced by a molecule not yet identified. However, it is also possible that this colony has lost araC or a part of the sequence from the construct all together. Sequencing needs to be repeated in order to explain the behaviour of this mutant.
Any changes between behaviour on the original restreak plates (Fig. 9) and the fluorescence plate reader data (Fig. 10) were probably due to varying densities of E. coli on the restreak plates. Higher cell density made the colony appear darker and thus more fluorescent.
All the mutant colonies were only loaded into the plate reader once per colony per condition. These would need to be repeated at very least in triplicate before any more solid conclusions can be made. Also, if time permitted we would have gone on to use error prone PCR on our mutants to optimise their activity, as was done by Tang et al (2008) [4] . It must be noted that 200 is an extremely low number of colonies to do initial screening on. Our protocol was expected to have generated 160,000 different amino acid combinations at the 4 randomised residue positions. 3X library coverage of this number would give 95% confidence of every variant appearing once, which is 480,000 colonies. However, the initial transformation generated 24,000 colonies, out of which only 200 were picked for further screening. Expanding of our mutant library could allow identification of AraC variants responsive specifically to xylose, xylulose or other compounds.  Unfortunately, this was not possible in the time frame of the project. Screening with a FACS machine, as was done in a previous study [4] , would allow for screening of more mutants. It would also be useful to test different concentrations of each additive on the mutant colonies to investigate if the behaviour is concentration-dependent. As a further experiment mutant cells could be screened against many different molecules such as general sugars, aldehyde molecules or even molecules completely unlike arabinose (e.g. quorum sensing molecules).
Outlook
The production of the mutant library of AraC is useful for synthetic biology. A catalogue of many of these mutants could possibly make the construction of regulatory systems such as biosensors much easier, and could be applied as a component of a biosensor toolbox. This could allow the generation of biosensors responsive to molecules which do not bind to any receptors in nature.
The protocol undertaken in this investigation did have a reasonable degree of success and could be replicated by other iGEM teams. Time constraints were the main reason for the limited degree of testing. This could be overcome by other iGEM teams by following our protocol of ethanol precipitating the ligation DNA before transformation and recruiting more people to do screening work, or ideally using a FACS machine to sort the many variant mutant colonies.
Parts Made
K2442101 - minimal PBAD promoter
K2442102 - minimal PBAD promoter + GFP
K2442103 - WT AraC coding region
K2442104 - R0011 + B0032 + WT AraC
K2442105 - R0011 + B0032 + AraC variant 1
K2442106 - R0011 + B0032 + AraC variant 2
K2442107 - R0011 + B0032 + AraC variant 3
K2442108 - R0011 + B0032 + AraC variant 4
References
↑ Brünker, P., Hils, M., Altenbuchner, J., and Mattes, R. (1998). The mannitol utilization genes of Pseudomonas fluorescens are regulated by an activator: Cloning, nucleotide sequence and expression of the mtlR gene. Gene 215, 19-27.
↑ Hoffmann, J., and Altenbuchner, J. (2015). Functional Characterization of the Mannitol Promoter of Pseudomonas fluorescens DSM 50106 and Its Application for a Mannitol-Inducible Expression System for Pseudomonas putida KT2440. PLOS ONE 10, e0133248.
↑ 3.0 3.1 3.2 Schleif, R. (2000). Regulation of the L-arabinose operon of Escherichia coli. Trends In Genetics 16, 559-565..
↑ 4.0 4.1 4.2 4.3 4.4 4.5 Tang, S., Fazelinia, H., and Cirino, P. (2008). AraC Regulatory Protein Mutants with Altered Effector Specificity. Journal Of The American Chemical Society 130, 5267-5271.
↑ Tang, S., and Cirino, P. (2010). Design and Application of a Mevalonate-Responsive Regulatory Protein. Angewandte Chemie 123, 1116-1118.
↑ Frei, C., Wang, Z., Qian, S., Deutsch, S., Sutter, M., and Cirino, P. (2016). Analysis of amino acid substitutions in AraC variants that respond to triacetic acid lactone. Protein Science 25, 804-814.
↑ Gilbert, M., Mandrell, R., Parker, C., Li, J., and Vinogradov, E. (2007). Structural Analysis of the Capsular Polysaccharide from Campylobacter jejuni RM1221. Chembiochem 8, 625-631.
↑ Miyada, C., Horwitz, A., Cass, L., Timko, J., and Wilcox, G. (1980). DNA sequence of the araC regulatory gene from Escherichia coli B/r. Nucleic Acids Research 8, 5267-5274.
↑ Daber, R., Stayrook, S., Rosenberg, A., and Lewis, M. (2007). Structural Analysis of Lac Repressor Bound to Allosteric Effectors. Journal Of Molecular Biology 370, 609-619.
↑ Doyle, M. E., Brown, C., Hogg, R. W., & Helling, R. B. (1972). Induction of the ara Operon of Escherichia coli B/r. Journal of Bacteriology, 110(1), 56–65.
iGEM 2017 Glasgow
Using synthetic biology to create a biosensor for detection of Campylobacter jejuni, a bacteria linked to food poisoning.
Contact Info
iGEM 2017 Glasgow
Wilkins Teaching Laboratory
Bower Building
University of Glasgow
igem.glasgow2017@gmail.com
Sponsors
Home
