<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Glasgow_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Glasgow/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title-bar" data-responsive-toggle="main-menu" data-hide-for="medium"><DIV class="title-bar-title">Glasgow iGEM 2017</DIV></DIV><DIV class="top-bar" id="main-menu"><DIV><DIV class="top-bar-right"><UL class="vertical large-horizontal menu" id="main-nav" data-responsive-menu="medium-accordion large-dropdown"><LI><A class="headline" href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Description">Project Background</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/mtlR">Exploiting the Mannitol regulator MtlR</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/araC">Mutagenesis of AraC</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/QuorumSensing">Quorum Sensing </A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/ANDGate">AND Gate testing</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/XyluloseBiosynthesis">Xylulose Biosynthesis</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/InterLab">InterLab study</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Measurement">Measurement</A></LI></UL><LI><A href="https://2017.igem.org/Team:Glasgow/Parts">Parts List</A></LI><LI><A class="headline" href="#">Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Applied_Design">Designing the Biosensor Device</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Hardware">Hardware</A></LI><LI><A href="#">Additional Work</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HydrogelStrip">Hydrogel Strip Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/BacillusSubtilis"><I>Bacillus subtilis</I> Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Collaborations">Collaborations</A></LI><LI><A class="headline" href="#">Practices</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated">Gold and 
                                                 Integrated</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Engagement">Education and Public 
                                                Outreach</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated/PolicyBrief">Policy Brief</A></LI><LI><A class="headline" href="#">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/WeeklyJournal">Weekly Journal</A></LI><LI><A class="headline" href="#">People</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Attributions">Attributions</A></LI><LI><A class="headline" href="#">Accomplishments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Accomplishments">Accomplishments</A></LI></DIV></DIV></DIV><DIV class="photo-block" id="test1"><DIV class="text left"><DIV class="title">
			Protocols
		</DIV></DIV></DIV><DIV class="wiki-block"><DIV id="article"><P>Described here are the various techniques we used throughout our project so that any other team may reproduce our experiments. 
</P><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Preparation_of_CaCl2_competent_cells"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext"><B>Preparation of CaCl<SUB>2</SUB> competent cells</B></SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Transformation_of_CaCl2_competent_cells"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext"><B>Transformation of CaCl<SUB>2</SUB> competent cells</B></SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Restriction_digests"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext"><B>Restriction digests</B></SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Miniprep"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext"><B>Miniprep</B></SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Making_a_gel"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext"><B>Making a gel</B></SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Gel_electrophoresis"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext"><B>Gel electrophoresis</B></SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Gel_extraction_using_Qiagen_kit"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext"><B>Gel extraction using Qiagen kit</B></SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Ligation"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext"><B>Ligation</B></SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Annealing_oligos"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext"><B>Annealing oligos</B></SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Polyacrylamide_Hydrogel"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext"><B>Polyacrylamide Hydrogel</B></SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Sporulation_of_Bacillus_subtilis"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext"><B>Sporulation of <I>Bacillus subtilis</I></B></SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Difco_Sporulation_Medium"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext"><B>Difco Sporulation Medium</B></SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Colony_PCR"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext"><B>Colony PCR</B></SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#SDS-PAGE"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext"><B>SDS-PAGE</B></SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#Ethanol_precipitation_of_ligation"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext"><B>Ethanol precipitation of ligation</B></SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Transformation_of_Library_competent_DH5a_cells"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext"><B>Transformation of Library competent DH5a cells</B></SPAN></A></LI></UL></DIV><H4><SPAN class="mw-headline" id="Preparation_of_CaCl2_competent_cells"><B>Preparation of CaCl<SUB>2</SUB> competent cells</B></SPAN></H4><OL><LI>Dilute 500μl of overnight liquid culture into 20ml of broth with any necessary antibiotics to select for any plasmids already transformed into the cells.</LI><LI>Incubate at 37⁰C, shaking at 225rpm for 105 minutes.</LI><LI>Spin down for 2 minutes at 7000G at 4⁰C.</LI><LI>Discard supernatant, resuspend pellet in 10ml of 50 mM CaCl<SUB>2</SUB>, keep on ice.</LI><LI>Repeat centrifugation for 2 minutes at 7000G at 4⁰C.</LI><LI>Discard supernatant and the resuspend pellet in 1ml of 50 mM CaCl<SUB>2</SUB>, keep on ice.</LI><LI>CaCl<SUB>2</SUB> competent cells can be kept on ice in the fridge overnight.</LI></OL><H4><SPAN class="mw-headline" id="Transformation_of_CaCl2_competent_cells"><B>Transformation of CaCl<SUB>2</SUB> competent cells</B></SPAN></H4><OL><LI>Add 1μl of plasmid DNA to 100μl of competent cells.</LI><LI>Heat shock at 42°C for 45 seconds.</LI><LI>Add 200μl of L-broth of the sample.</LI><LI>Keep on ice for 2 minutes.</LI><LI>Incubate the cells at 37°C. The time varies depending on which antibiotic resistance the plasmid holds.</LI></OL><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 1:</B> Incubation time for transformed cells containing different antibiotic resistances
		</DIV></DIV></CENTER><H4><SPAN class="mw-headline" id="Restriction_digests"><B>Restriction digests</B></SPAN></H4><P>A 20μl reaction typically contained:
</P><OL><LI>2μl buffer</LI><LI>4μl Miniprep (or 8μl G-Block) dependent on concentration of Miniprep</LI><LI>Make up to 20μl with ddH20</LI><LI>Add 10-20 Units restriction enzyme(s)</LI><LI>Vortex briefly.</LI><LI>Incubate at 37°C for at least 60 minutes.</LI><LI>Heat inactivate restriction digests where appropriate.</LI></OL><H4><SPAN class="mw-headline" id="Miniprep"><B>Miniprep</B></SPAN></H4><P>We used the protocol from a standard Qiagen QIAprep Spin Miniprep Kit.
</P><OL><LI>Pipette 1mL of bacterial overnight culture into a microcentrifuge tube</LI><LI>Centrifuge at 13,000 rpm for 1 min</LI><LI>Discard the supernatant</LI><LI>Repeat steps 1-3 two more times</LI><LI>Add 250μl of P1 buffer and pipette up and down to resuspend the pellet</LI><LI>Add 250μl of P2 buffer and invert Note: don’t allow this lysis reaction to proceed for more than 5 minutes</LI><LI>Add 350μl of N3 buffer to neutralise the reaction and invert</LI><LI>Centrifuge at 13,000 rpm for 10 minutes</LI><LI>Transfer 800μl of supernatant into a column</LI><LI>Centrifuge for 1 minute and discard flow-through</LI><LI>Add 500μl of PB buffer and centrifuge for 1 minute</LI><LI>Discard flow-through</LI><LI>Add 750μl of PE buffer and centrifuge for 1 minute</LI><LI>Discard flow-through</LI><LI>Centrifuge again for 1 minute to get rid of any residual buffer</LI><LI>Transfer the column to a microcentrifuge tube</LI><LI>Add 50μl of EB buffer and let it stand for 1 minute</LI><LI>Centrifuge for 1 minute</LI></OL><H4><SPAN class="mw-headline" id="Making_a_gel"><B>Making a gel</B></SPAN></H4><P>To make a 1.5L buffer: 
</P><OL><LI>Measure 30mL of 50x TAE buffer</LI><LI>Transfer to a 2L measuring cylinder</LI><LI>Fill up to 1.5L with distilled water</LI></OL><P>To make a 1% agarose gel: 
</P><OL><LI>Weigh out 1g of agarose powder</LI><LI>Transfer to a microwave bottle</LI><LI>Pour 100mL of buffer into the bottle</LI><LI>Microwave until clear</LI><LI>Cool down to 55⁰C before pouring the gel</LI></OL><H4><SPAN class="mw-headline" id="Gel_electrophoresis"><B>Gel electrophoresis</B></SPAN></H4><P><U>To run the gel: </U></P><OL><LI>Place the gel in the tank and cover it with the buffer</LI><LI>Load 5μl of DNA ladder into the 2nd well (leave the first one empty)</LI><LI>Add 5μl of loading dye (30% glycerol, 1% bromophenol blue, 0.5% sodium dodecyl sulphate, diluted in TE buffer) into each restriction digest and pipette up and down a few times</LI><LI>Load 25μl of each restriction digest into a separate well</LI><LI>Run gel at 100V ( ~100 mA) for approximately 1 hour</LI></OL><P><U>To stain the gel: </U></P><UL><LI>SYBR Safe: stain in (concentration) for 40 minutes, destain for 40 minutes, image under UV light</LI><LI>Azure A (for gel extraction): stain in 0.04%/20% ethanol for 15 minutes, destain for 15 minutes (multiple rounds of destaining may be required), image under visible light</LI></UL><H4><SPAN class="mw-headline" id="Gel_extraction_using_Qiagen_kit"><B>Gel extraction using Qiagen kit</B></SPAN></H4><OL><LI>Weigh an empty eppendorf and record the weight</LI><LI>Cut out a the band from the gel and place it in the eppendorf</LI><LI>Weigh the eppendorf again and calculate the weight difference</LI><LI>Add 3 volumes of Buffer QG to 1 volume of gel</LI><LI>Incubate at 50⁰C for 10 minutes until the agarose has dissolved. Vortex every 2-3 minutes</LI><LI>Add 1 volume of isopropanol and mix</LI><LI>Transfer the sample from the eppendorf to a QIAprep spin column in a 2mL collection tube</LI><LI>Centrifuge for 1 minute at 13,000rpm and discard flowthrough</LI><LI>Add 500μl of Buffer QG to column</LI><LI>Centrifuge for 1 minute at 13,000rpm and discard flowthrough</LI><LI>Add 750μl of Buffer PE and let it stand for 2-5 minutes</LI><LI>Centrifuge for 1 minute at 13,000rpm and discard flowthrough. Repeat this step</LI><LI>Transfer the column to a new eppendorf and discard the collection tube</LI><LI>Add 30μl of Buffer EB to the sample and let it stand for 1 minute</LI><LI>Centrifuge for 1 minute at 13,000rpm and keep discard the column</LI></OL><H4><SPAN class="mw-headline" id="Ligation"><B>Ligation</B></SPAN></H4><P>For a 10μl ligation reaction (these volumes change be changed depending on the concentration of the gel extracted DNA): 
</P><UL><LI>3μl of vector</LI><LI>5μl of insert</LI><LI>1μl of 10x ligation buffer</LI><LI>0.5μl of ddH2O</LI><LI>0.5μl of ligase</LI></UL><P>Vortex
</P><P>For ligating oligo and vector: 
</P><UL><LI>1μl of oligo (annealed and diluted to 0.1 μM)</LI><LI>5μl of vector</LI><LI>1μl of 10x ligation buffer</LI><LI>3μl of ddH2O</LI><LI>0.5μl of ligase</LI></UL><P>Vortex
</P><P>Ligation reactions were left for a minimum of 1 hour or ideally overnight at room temperature or in a temperature-stable room at 16⁰C to ensure maximum ligation efficiency. 
</P><H4><SPAN class="mw-headline" id="Annealing_oligos"><B>Annealing oligos</B></SPAN></H4><OL><LI>Dissolve dried oligo in TE buffer to make a 100μM solution and leave for 10 minutes</LI></OL><P>To make a 100μl reaction:
</P><UL><LI>10μl of top strand 100μM oligo</LI><LI>10μl of bottom strand 100μM oligo</LI><LI>80μl of TE buffer</LI></UL><OL><LI>Heat at 86⁰C in a water bath for 5 minutes using a float</LI><LI>Scoop out some water and the float from the water bath using a large beaker</LI><LI>Let the water cool down slowly to approximately 30⁰C</LI><LI>Dilute annealed oligos 1/100 to 0.1 μM</LI></OL><H4><SPAN class="mw-headline" id="Polyacrylamide_Hydrogel"><B>Polyacrylamide Hydrogel</B></SPAN></H4><P>For 10ml of a 5% acrylamide hydrogel:
</P><UL><LI>8.3ml PBS Buffer</LI><LI>1.7ml 30% Acrylamide/Bis-Acrylamide Solution (37.5:1 weight ratio)</LI><LI>100ul APS</LI><LI>10ul TEMED</LI></UL><OL><LI>Add PBS, acrylamide solution and APS to a 15ml falcon tube and swirl to mix thoroughly.</LI><LI>Add TEMED and again swirl to mix and wait for polymerisation.</LI></OL><P>For different percentage acrylamide hydrogels, only the volumes of acrylamide solution and PBS buffer need altered.
</P><H4><SPAN class="mw-headline" id="Sporulation_of_Bacillus_subtilis"><B>Sporulation of <I>Bacillus subtilis</I></B></SPAN></H4><P>Followed the protocol developed by the collaboration between the Bonn and Freiburg iGEM teams from 2016. 
</P><UL><LI><B> Step 1: Overnight culture-</B> Inoculate a colony of <I>Bacillus subtilis</I> in 4ml LB-medium and let the cells grow overnight at 37°C, 200 rpm</LI></UL><UL><LI><B>Step 2: Exponential growth-</B> Measure the OD600 of the overnight culture and dilute by using LB-medium to an OD600 in the region of 0.1-0.2/ml in 10ml LB. Let the cells grow to an OD600 of 0.8/ml at 37°C, 200 rpm</LI></UL><UL><LI><B>Step 3: Sporulation-</B> Centrifuge 10ml of the cells at 13,000 x g for 1 minute. Wash the pellet five times with 1x PBS buffer by suspending the pellet in 1ml of 1x PBS and centrifuging. Re-suspend the pellet in 5ml DSM (Difco Sporulation Medium). Let the cells grow for 24 hours at 37°C, 200 rpm.</LI></UL><UL><LI><B>Step 4: Lysozyme Treatment-</B> Treat the sample with 416ul of 30mg/ml stock lysozyme, incubate for one hour at room temperature and wash six times with 1x PBS</LI></UL><H4><SPAN class="mw-headline" id="Difco_Sporulation_Medium"><B>Difco Sporulation Medium</B></SPAN></H4><P>Fresh DSM was prepared as the FeSO<SUB>4</SUB> rusts when in solution.
</P><P><U>For 200ml:</U></P><UL><LI>1.6g Nutrient Broth</LI><LI>0.2g KCl</LI><LI>0.2ml MgSO4 (1M)</LI><LI>0.2ml MnCl2 (10mM)</LI><LI>200ml ddH2O</LI></UL><P>Autoclave and add:
</P><UL><LI>0.5ml CaCl2 (1M)</LI><LI>1ml FeSO4 (1M)</LI></UL><H4><SPAN class="mw-headline" id="Colony_PCR"><B>Colony PCR</B></SPAN></H4><P>The target gene was first isolated from DH5α cells by colony PCR, where 3 colonies were resuspended in 200µl TE buffer and used as template DNA for 3 separate 50µl PCR reactions. This reduced the chances of only selecting a mutant colony. Conditions were adapted from the NEB PCR Protocol for Phusion High-Fidelity (HF) DNA Polymerase with Standard Taq Buffer<A rel="nofollow" class="external autonumber" href="https://www.neb.com/protocols/1/01/01/pcr-protocol-m0530">[1]</A>for a 50µl reaction, as were guidelines for primer design, annealing temperatures and extension times:
</P><UL><LI>5X Phusion HF Buffer		(10µl)</LI><LI>2mM dNTPs			        (5µl)</LI><LI>10µM Forward Primer		(2.5µl)</LI><LI>10µM Reverse Primer		(2.5µl)</LI><LI>Template DNA			(1µl)</LI><LI>Taq DNA Polymerase		(0.5µl)</LI><LI>Nuclease-Free Water		(31µl)	</LI></UL><UL><LI><B>Primers</B> were designed to anneal flanking regions of the gene, and to include the biobrick prefix and suffix sequences on either end of the amplified gene. The Melting temperature (Tm) of the primers were to be within 1<SUP>o</SUP>C of one another (~60<SUP>o</SUP>C), and to have a guanine/cytosine content (GC%) of between 30% and 50%. A 6-nucleotide random DNA sequence was also added to the primers which lets the enzymes cut efficiently. </LI><LI>A longer initial <B>denaturation time</B> of 5 minutes was used since the DNA was still contained within the cells. </LI><LI>The <B>annealing temperature</B> was taken to be a 2<SUP>o</SUP>C lower than the average melting temperature. </LI><LI>The <B>extension time</B> was suggested to be 15 seconds per kilobases of DNA being amplified. Following amplification, the PCR products were digested (using a restriction site known to be present in the amplicon) and gel electrophoresis was used to check for successful amplification.</LI></UL><H4><SPAN class="mw-headline" id="SDS-PAGE"><B>SDS-PAGE</B></SPAN></H4><P><B>Sodium Dodecyl Sulfate – Polyacrylamide Gel Electrophoresis (SDS-PAGE)</B> was used to analyse the effect of the expression plasmid (designed to overexpress xylose isomerase) on the entire protein complement of the <I>E. coli</I> BL21(DE3) cells, and compare to normal cell protein expression to conclude whether the expression plasmid (part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442371">K2442371</A>) works as hypothesised.
</P><UL><LI>BL21(DE3) strain <I>E. coli</I> cells were first transformed with the expression plasmid. 3 separate fresh 3ml liquid cultures (labelled cultures A, B and C) were made using separate colonies, L-broth, and with the addition of of chloramphenicol (Cm) (1,000X) and glucose (20mM/l). Glucose was added to the media because it precedes lactose in the <I>E. coli</I> sugar-consumption hierarchy, and the lactose-induced lac operon loci is known to transcribe RNA polymerase. As such, a source of glucose suppresses RNA polymerase transcription, and so gene expression only occurs after addition of the lac-operon inducer Isopropyl β-D-1-thiogalactopyranoside (IPTG). </LI><LI>Each culture was then diluted 100-fold (300µl added to 30ml fresh L-broth) in broth containing Cm, and then was incubated (37<SUP>o</SUP>C in a shaking rack at 225 rpm) for a period of 3 hours. During this period, cell growth was measured every hour by optical density (OD), using a spectrophotometer (600nm wavelength). Once each culture reached a pre-induction (Pre) OD of roughly 0.5, an aliquot was removed and kept for analysis during SDS-PAGE as samples showing pre-induction cell protein expression. IPTG was added to induce protein expression in the remaining culture. The cells were then incubated (37<SUP>o</SUP>C in a shaking rack at 225 rpm) for another 3 hours, recording the post induction (Post) OD of A, B and C.</LI><LI>Following protein expression, 1ml of cultures A, B and C were transferred to a 1.5ml eppendorf tube and spun down (13,000rpm for 5 minutes). The supernatant was then pipetted off, and the cells were resuspended in laemmli loading buffer (950µl 5X laemmli sample buffer + 50µl 0.5M dithriotheitol (DTT)). The proportion of buffer added to each culture was normalised to the culture with the highest OD, where that culture received 1ml of laemmli buffer (Table 1). </LI></UL><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Table 1 - Cell-culture preparation for SDS-PAGE:</B> The optical density was measured using a 600nm spectrophotometer, and represents the amount of bacterial mass present in the culture. The volume of dye added was normalised to the culture with the highest OD.
		</DIV></DIV></CENTER><UL><LI>The gel used was a 10-well Mini-PROTEAN TGX Precast Gel. 25µl of each of the 6 samples (PreABC and PostABC) were loaded on the gel, along with 20µl of protein marker (Biorad Precision All-Blue Molecular Weight Marker ladder). 25µl of dye was also added to the remaining wells to avoid protein running at an angle. The gel was run until the bands reached the green reference line on the SDS-PAGE kit. The gel was then removed from the precast cassette, and stained with Coomassie blue.</LI></UL><H4><SPAN class="mw-headline" id="Ethanol_precipitation_of_ligation"><B>Ethanol precipitation of ligation</B></SPAN></H4><OL><LI>To ligation reaction add 1/10 volume of 5M NaCl</LI><LI>Add 2 volumes of 100% ethanol</LI><LI>Vortex,place at -70<SUP>o</SUP>C for 20 mins</LI><LI>Spin in chilled micro-centrifuge for 45 mins at 3000rpm</LI><LI>Pour off supernatant, spin 1 min, pipette off remainder</LI><LI>Add 1 volume 70% ethanol</LI><LI>Spin again for 30 mins, discard supernatant</LI><LI>Air dry</LI><LI>Resuspend in 10 ul of nuclease free H20</LI></OL><H4><SPAN class="mw-headline" id="Transformation_of_Library_competent_DH5a_cells"><B>Transformation of Library competent DH5a cells</B></SPAN></H4><P>Protocol sourced from thermofisher scientific
</P><OL><LI>Thaw cells on competent wet ice. Place required number of 17 times 100nm polypropylene tubes on ice</LI><LI>Gently mix cells, then aliquot 100 ul of competent cells into chilled tubes</LI><LI>For DNA from ligation reactions, dilute the reactions 5-fold in 10mM TE buffer. Add 1ul of the dilution to the cells (1-10 ng DNA), moving the pipette through the cells whilst dispensing. Gently tap to mix</LI><LI>Incubate on ice for 30 mins</LI><LI>Heat shock cells for 45 seconds in a 42<SUP>o</SUP>C water bath; do not shake</LI><LI>Place on ice for 2 mins</LI><LI>Add 0.9ml of room temp S.O.C Medium</LI><LI>Shake at 225rpm for 1 hour</LI><LI>Dilute the reactions as necessary and spread 100-200 ul of this dilution into LB plates with requred antibiotic.</LI><LI>Incubate overnight</LI></OL></DIV></DIV><FOOTER class="marketing-site-footer"><DIV class="row medium-unstack"><DIV class="medium-4 columns"><H4 class="marketing-site-footer-name">iGEM 2017 Glasgow</H4><P>Using synthetic biology to create a biosensor for detection of <I>Campylobacter jejuni</I>, a bacteria linked to food poisoning.</P></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Contact Info</H4><DIV class="marketing-site-footer-block"><P>                    iGEM 2017 Glasgow 
Wilkins Teaching Laboratory
                    Bower Building 
                    University of Glasgow</P></DIV><DIV class="marketing-site-footer-block"><P>igem.glasgow2017@gmail.com</P></DIV></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Sponsors</H4></DIV></DIV><DIV class="marketing-site-footer-bottom"><DIV class="row large-unstack align-middle"><DIV class="column"><UL class="menu marketing-site-footer-bottom-links"><LI><A href="https://2017.igem.org/Team:Glasgow">Home</A></LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>