<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Glasgow_araC skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Glasgow/araC</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="title-bar" data-responsive-toggle="main-menu" data-hide-for="medium"><DIV class="title-bar-title">Glasgow iGEM 2017</DIV></DIV><DIV class="top-bar" id="main-menu"><DIV><DIV class="top-bar-right"><UL class="vertical large-horizontal menu" id="main-nav" data-responsive-menu="medium-accordion large-dropdown"><LI><A class="headline" href="#">Project</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Description">Project Background</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/mtlR">Exploiting the Mannitol regulator MtlR</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/araC">Mutagenesis of AraC</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/QuorumSensing">Quorum Sensing </A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/ANDGate">AND Gate testing</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/XyluloseBiosynthesis">Xylulose Biosynthesis</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/InterLab">InterLab study</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Measurement">Measurement</A></LI></UL><LI><A href="https://2017.igem.org/Team:Glasgow/Parts">Parts List</A></LI><LI><A class="headline" href="#">Engineering</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Applied_Design">Designing the Biosensor Device</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Hardware">Hardware</A></LI><LI><A href="#">Additional Work</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HydrogelStrip">Hydrogel Strip Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/BacillusSubtilis"><I>Bacillus subtilis</I> Experiments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Collaborations">Collaborations</A></LI><LI><A class="headline" href="#">Practices</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated">Gold and 
                                                 Integrated</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Engagement">Education and Public 
                                                Outreach</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/HP/Gold_Integrated/PolicyBrief">Policy Brief</A></LI><LI><A class="headline" href="#">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/WeeklyJournal">Weekly Journal</A></LI><LI><A class="headline" href="#">People</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Attributions">Attributions</A></LI><LI><A class="headline" href="#">Accomplishments</A></LI><LI><A href="https://2017.igem.org/Team:Glasgow/Accomplishments">Accomplishments</A></LI></DIV></DIV></DIV><DIV class="photo-block" id="AraC"><DIV class="text left"><DIV class="title">
			Mutagenesis of the L-arabinose regulator AraC
		</DIV></DIV></DIV><DIV class="wiki-block"><DIV id="article"><DIV id="toc" class="toc"><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Background"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Aims"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Aims</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Materials_and_Methods"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Materials and Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Plasmid_design"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Plasmid design</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Library_Construction"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Library Construction</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Fluorescence-based_screening"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Fluorescence-based screening</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Liquid_culture_fluorescence_assay"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Liquid culture fluorescence assay</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#Results"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Discussion"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Outlook"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Outlook</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Parts_Made"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Parts Made</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#References"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">References</SPAN></A></LI></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>While doing initial research into building a biosensor against <I>Campylobacter jejuni</I>, we created this subproject to  focus on utilization of the L-arabinose sensing AraC regulatory protein and its regulated P<SUB>BAD</SUB> promoter in order to construct one of the parts of the final detection system. The regulatory protein AraC binds to operator sequences in the P<SUB>BAD</SUB> promoter and activates transcription of downstream genes only when bound to its substrate sugar (L-arabinose). Here we performed saturation mutagenesis on the <I> araC </I> gene to construct AraC mutant library with four randomized ligand-binding pocket amino acid residues corresponding to positions 8, 24, 80 and 82. The AraC mutant library was subjected to fluorescence-based screening using a GFP reporter. Due to unavailability of xylulose we tested for mutants responsive to a similar molecule - xylose. We isolated and sequenced 4 mutants which showed altered effector specificity. Our results demonstrate an approach that allows to engineer a regulatory protein to respond to an effector molecule of interest and represent steps towards developing a functional biosensor to detect <I>C. jejuni</I> on contaminated surfaces. Our protocol can be used by future teams to generate AraC variants responsive to any small molecule of choice, and provides a toolkit for developing biosensors.
</P><H2><SPAN class="mw-headline" id="Background">Background</SPAN></H2><P>Our team aims to construct two alternative regulatory systems as potential components of the biosensor. One of them exploits the mannitol operon, naturally found in <I>Pseudomonas fluorescens</I><SUP id="cite_ref-1" class="reference"><A href="#cite_note-1">[1]</A></SUP>. The regulatory protein MtlR activates transcription of downstream genes upon binding to xylulose<SUP id="cite_ref-2" class="reference"><A href="#cite_note-2">[2]</A></SUP>. However, this approach may have some disadvantages. Coming from a different organism, the MtlR protein may not function in <I>E. coli</I> cells as expected. Furthermore, the MtlR operon has not been previously exploited in biosensors and may not show sufficient specificity to xylulose. Hence, we aim to construct an alternative regulatory system which exploits the L-arabinose operon. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 1:</B> Genes required for catabolism of L-arabinose and the operon components. <I>araC</I> expression is regulated by the PC promoter and is transcribed in the opposite direction from <I>araBAD</I> genes, which are controlled by the P<SUB>BAD</SUB> promoter. AraC acts negatively (-) at the PC promoter and negatively or positively (Â±) at the P<SUB>BAD</SUB> promoter. Adapted from Shleif (2000) <SUP id="cite_ref-Schleif_3-0" class="reference"><A href="#cite_note-Schleif-3">[3]</A></SUP></DIV></DIV></CENTER><P>
The L-arabinose operon is naturally found in <I>Escherichia coli</I><SUP id="cite_ref-Schleif_3-1" class="reference"><A href="#cite_note-Schleif-3">[3]</A></SUP>. The regulatory protein, AraC, acts as a dimer. As depicted in Fig. 1, the operon contains two regulatory <I>cis</I> elements: the P<SUB>C</SUB> promoter for the transcription of <I>araC</I>, and the P<SUB>BAD</SUB> promoter for transcription of genes required for catabolism of L-arabinose. The mechanism of transcriptional regulation has been described by Schleif (2000) and is summarized in Fig. 2. The P<SUB>BAD</SUB> promoter is regulated by 3 half sites that each bind to one subunit of AraC - O<SUB>2</SUB>, I<SUB>1</SUB>, I<SUB>2</SUB>. The P<SUB>C</SUB> promoter is autoregulated by AraC binding to O<SUB>1</SUB>, which is composed of O<SUB>1L</SUB> and O<SUB>1R</SUB> half sites. The O<SUB>2</SUB> half site required for proper regulation of P<SUB>BAD</SUB> is within the <I>araC</I> coding region. In absence of L-arabinose the AraC dimer binds to operator half-sites O<SUB>2</SUB> and I<SUB>1</SUB>. This causes DNA looping upstream of the P<SUB>BAD</SUB> promoter which represses transcription by excluding RNA polymerase from binding to P<SUB>BAD</SUB>. Binding of L-arabinose causes a conformational change in the protein such that the DNA-binding domains of the dimer bind to adjacent I<SUB>1</SUB> and I<SUB>2</SUB> half-sites, giving access for RNA polymerase and cyclic AMP receptor protein (CRP) to bind P<SUB>BAD</SUB>. This results in transcriptional activation.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 2:</B> Regulation of the L-arabinose operon by arabinose. In the absence of L-arabinose, AraC dimer binds to O<SUB>2</SUB> and I<SUB>1</SUB> half sites causing DNA looping, which prevents RNA polymerase from accessing P<SUB>BAD</SUB>. Upon binding of Lâarabinose, the AraC dimer binds I<SUB>1</SUB> and I<SUB>2</SUB> half sites instead, allowing transcription of the polycistronic <I>araBAD</I> mRNA. <I>araC</I> is transcribed in opposite direction from <I>araBAD</I>, and is under control of the PC promoter. RNA polymerase and AraC compete for binding at O<SUB>1</SUB> and P<SUB>C</SUB>. Adapted from Shleif (2000) <SUP id="cite_ref-Schleif_3-2" class="reference"><A href="#cite_note-Schleif-3">[3]</A></SUP></DIV></DIV></CENTER><P>
Several previous studies have shown that AraC protein can be engineered to activate transcription in response to non-native small molecules. Some of them include D-arabinose (enantiomer of L-arabinose)<SUP id="cite_ref-Tang_4-0" class="reference"><A href="#cite_note-Tang-4">[4]</A></SUP>, mevalonate<SUP id="cite_ref-5" class="reference"><A href="#cite_note-5">[5]</A></SUP> and triacetic acid lactone<SUP id="cite_ref-6" class="reference"><A href="#cite_note-6">[6]</A></SUP>. Site-saturation mutagenesis of residues positioned within the ligand-binding pocket of AraC, coupled with fluorescence-based cell sorting, allowed the groups to isolate AraC variants with altered effector specificity.  Based on these findings we aimed to use multiple site-saturation mutagenesis and fluorescence-based screening to generate mutant AraC responsive specifically to xylulose. Although xylulose comes in the form of two enantiomers, we chose to focus on D-xylulose, which is the isomer present in the capsule of <I>C. jejuni</I><SUP id="cite_ref-7" class="reference"><A href="#cite_note-7">[7]</A></SUP>. Both L-arabinose and D-xylulose are monosaccharides with five carbons, with molecular formulas C<SUB>5</SUB>H<SUB>10</SUB>O<SUB>5</SUB>. As seen in Fig. 3a, arabinose is an aldopentose with an aldehyde functional group at carbon position 1, while xylulose is a ketopentose, possessing a ketone group instead. The 3-dimensional structure of D-xylulose differs from that of L-arabinose, with the most pronounced difference being the 5-membered vs 6-membered heterocyclic ring in its structure. Fig. 3b represents key residues of AraC protein that play a role in ligand binding. Previous studies have shown that mutagenesis of amino acids in positions 8, 24, 80 and 82 is sufficient to change effector specificity<SUP id="cite_ref-Tang_4-1" class="reference"><A href="#cite_note-Tang-4">[4]</A></SUP>. 
</P><P>Due to prohibitively high cost of purified xylulose, we aimed to <A rel="nofollow" class="external text" href="https://2017.igem.org/Team:Glasgow/XyluloseBiosynthesis">synthesise our own</A>. In the meantime, due to unavailability of the sugar, we screened for mutants responsive to a similar molecule - xylose. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Figure 3:</B> (a) Structures of L-arabinose and D-xylulose. (b) Crystal strycture of AraC binding pocket with bound L-arabinose (LA), showing key residues that play a role  in ligand binding. Residues targeted for mutagenesis are underlined in red. Water molecules are shown as spheres (adapted from Tang <I>et al</I>, 2008<SUP id="cite_ref-Tang_4-2" class="reference"><A href="#cite_note-Tang-4">[4]</A></SUP>)
		</DIV></DIV></CENTER><H2><SPAN class="mw-headline" id="Aims">Aims</SPAN></H2><UL><LI>Aim 1</LI></UL><P>To split the AraC regulatory system into two plasmids: regulatory plasmid expressing AraC and AraC mutants and reporter plasmid expressing GFP under control of P<SUB>BAD</SUB> promoter.
</P><UL><LI>Aim 2</LI></UL><P>To perform multiple site-saturation mutagenesis on the AraC protein at residue positions 8, 24, 80 and 82.
</P><UL><LI>Aim 3</LI></UL><P>To isolate AraC variants with altered effector specificity using fluorescence-based screening.
</P><H2><SPAN class="mw-headline" id="Materials_and_Methods">Materials and Methods</SPAN></H2><H3><SPAN class="mw-headline" id="Plasmid_design">Plasmid design</SPAN></H3><P>Plasmid construction was as follows.
</P><P>Oligos containing parts <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_R0011">R0011</A> (LacI-regulated promoter) upstream of ribosome-binding site (RBS) <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0032">B0032</A> were synthesised by Integrated DNA Technologies (IDT). Each strand had different, incomplete prefix and suffix sequences, such that when annealed the resulting dsDNA had BioBrick-compatible sticky ends for ligation into EcoRI and PstI digested BioBrick vectors. The resulting product was ligated into high copy number pSB1C3 vector digested with EcoRI and PstI. The ligation reaction was introduced into chemically competent <I>E.coli</I> strain DH5Î±, and sequence from a chosen colony was verified.  
</P><P>WT <I>araC</I> sequence as described by Miyada <I>et al</I> (1980) <SUP id="cite_ref-8" class="reference"><A href="#cite_note-8">[8]</A></SUP> was amplified from <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A> using primers <I>araC_BBPre_F</I>  and <I>araC_BBSuf_R</I> (see Table 1) to introduce BioBrick prefix and suffix to both ends of WT <I>araC</I>. The PCR product was subsequently digested with XbaI and PstI. pSB1C3+R0011+B0032 was digested with SpeI and PstI. The two digested parts were ligated to generate <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442104">K2442104</A> plasmid with WT <I>araC</I>. Sequence was verified.
</P><P>P<SUB>BADmin</SUB> was synthesised by Integrated DNA Technologies (IDT) as a G-block and digested with EcoRI and PstI. The cut product was then ligated into pSB3K3 vector digested with EcoRI and PstI. The ligation product was introduced into <I>E.coli</I> strain DH5Î± by chemical transformation and the sequence was verified. Part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I13500">BBa_I13500</A> was then digested with enzymes XbaI and PstI and ligated into pSB3K3+P<SUB>BADmin</SUB>, digested with SpeI and PstI. This resulted in the final reporter plasmid <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442102">K2442102</A>, and the sequence was verified. 
</P><H3><SPAN class="mw-headline" id="Library_Construction">Library Construction</SPAN></H3><P>Primer sequences are listed in Table 1. <I>araC</I> gene was amplified from the <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A> part using primers <I>araC_F</I> and <I>araC_R</I>. The PCR product was then used in three PCR reactions with the following sets of primers: <I>araC_Frag1_F</I> and <I>araC_Frag1_R</I>;  <I>araC_Frag2_F</I> and <I>araC_Frag2_R</I>; <I>araC_Frag3_F</I> and <I>araC_Frag3_R</I>.
</P><P>The PCR conditions were: 
<OL><LI>98Â°C for 30s</LI><LI>35 cycles of 98Â°C for 10s, 67.5Â°C (for F1 and F2) or 59.8Â°C (for F3) for 30s and 72Â°C for 1min</LI><LI>Final step of 72Â°C for 10 mins </LI></OL></P><P>The reactions resulted in 3 <I>araC</I> fragments (F1, F2 and F3). Primers <I>araC_Frag1_F</I>, <I>araC_Frag1_R</I> and <I>araC_Frag2</I>_R contain the degenerate sequences NNS at specific triplets so that fragments F1 and F2 contain saturation mutagenesis at codon positions 8, 24, 80 and 82. PCR products were gel purified and DNA concentration of each was measured with a NanoDrop<SUP>TM</SUP> spectrophotometer. Equimolar aliquots (0.15pmol each) of adjacent fragments were combined (F1+F2 and F2+F3) and PCR-assembled without primers under the following conditions: 
<OL><LI>98Â°C for 30s</LI><LI>15 cycles of 98Â°C for 10s, 60Â°C for 1min and 72Â°C for 40s</LI><LI>72C for 10 mins. </LI></OL></P><P>15 Î¼l of each reaction product was combined and PCR-assembled without primers under the following conditions:
<OL><LI>20 cycles of 98Â°C for 30s</LI><LI>72Â°C for 40s</LI></OL></P><P>Finally primers <I>araC_BBPre_F</I> and <I>araC_BBSuf_R</I> were added and a final round of PCR was ran with the following programme: 
<OL><LI>98Â°C for 30s</LI><LI>30 cycles of 98Â°C for 10s</LI><LI>60Â°C for 1 min </LI><LI>72Â°C for 40 s</LI><LI>72Â°C for 10 min. </LI></OL></P>


QIAQuick PCR purification of the product was performed according to the Qiagen protocol. 
The purified PCR product was then digested with XbaI and PstI and ligated into pSB1C3+R0011+B0032 digested with SpeI and PstI. Ligation reaction was ethanol-precipitated, and transformed into electrocompetent DH5Î± by electroporation. This resulted in the <I>araC</I> mutant library.

Colonies carrying the mutant library were washed off transformation plates with 1.5ml double distilled H<SUB>2</SUB>O, (ddH<SUB>2</SUB>O) then transferred to a 1.5ml tube. The cell pellet was centrifuged at room temperature, the supernatant was then discarded and pellet resuspended in 1.5ml ddH<SUB>2</SUB>O. This centrifugation-resuspension step was repeated 3 times to remove agar plate debris. Plasmid DNA was extracted using the Qiagen Plasmid MiniPrep Kit with 2 of each PB buffer and PE buffer wash steps to ensure maximum extract purity. Sequencing primer VF2 was then added and sample sent for sequencing.

<CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B>Table 1:</B> Primer sequences used in library construction. Degenerate sequences are highlighted in <FONT color="magenta">magenta</FONT>. N, any base. S, strong base (G or C). <FONT color="red">Red</FONT>, random 6bp sequence. <FONT color="yellow">Yellow</FONT>, BioBrick prefix. <FONT color="cyan">Cyan</FONT>, reverse compliment of BioBrick suffix.
		</DIV></DIV></CENTER><H3><SPAN class="mw-headline" id="Fluorescence-based_screening">Fluorescence-based screening</SPAN></H3><P>Plasmid DNA containing the <I>araC</I> mutant libraries were transformed into DS941 carrying the reporter plasmid K2442102 (in pSB3K3 vector). Transformants were plated on LB agar medium containing chloramphenicol plus kanamycin (to select for K2442102 and K2442104) plus one of the tested inducers: arabinose, xylose or decanal. Concentrations were 40mM for arabinose and xylose and 2mM for decanal. As a control test transformants were also plated on LB medium containing chloramphenicol and kanamycin only. Fluorescence images of the conditional transformation plates were obtained using a GE-Healthcare Typhoon FLA-9500 laser scanner. Excitation was recorded at 473nm, emission was recorded using 520-540nm filter. Colonies which exhibited fluorescence were observed as dark colonies on the scan (Fig. 8). 200 of those which appeared dark on xylose or decanal plates were re-streaked onto plates containing xylose, arabinose, decanal or no additive. Each colony of interest was picked and then immediately short-streaked onto each new condition plate using the same toothpick, into the same position using grids. This method of replica plating ensures the plates can later be aligned and short streaks of the same origin directly compared between the plates. After overnight incubation at 37Â°C, fluorescence scans of each plate were again obtained using the scan fluorescent scanning method.
</P><H3><SPAN class="mw-headline" id="Liquid_culture_fluorescence_assay">Liquid culture fluorescence assay</SPAN></H3><P>Colony of interest was inoculated into L-broth containing chloramphenicol and kanamycin. Each liquid culture was grown overnight at 37Â°C, shaking at 225 rpm. The following day the culture was diluted 1:100 into fresh L-broth containing the above antibiotics plus one of the following inducer conditions: 
</P><UL><LI>40mM xylose</LI><LI>40mM arabinose</LI><LI>2mM decanal</LI></UL><P>
200Âµl of each culture to be tested was placed in a well of a clear-based 6-well plate, then incubated at 37Â°C shaking at 300 rpm for 12 hours in a BMG Fluorostar Omega fluorescence plate reader. Fluorescence of each culture was measured at 1 hour time intervals for the duration of the culture experiment, using excitation wavelength 485nm and emission wavelength 530nm. Cell growth was simultaneously tracked by measuring optical density at 600nm. 
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>Regulatory plasmid K2442104 contains wild-type  <I>araC</I> gene under control of LacI-regulated promoter R0011 linked to RBS B0032. This is to enhance control over production of the AraC, so that the growth of bacteria is not affected by overproduction of the protein. Production of AraC can be stimulated by addition of Isopropyl Î²-D-1-thiogalactopyranoside (IPTG) to the media. IPTG mimics allolactose and prevents LacI from binding to its operator sequences in the DNA, switching on the R0011 promoter<SUP id="cite_ref-9" class="reference"><A href="#cite_note-9">[9]</A></SUP>. The construct was ligated into pSB1C3 plasmid backbone (Fig. 4).
</P><P>The reporter plasmid K2442102 contains GFP gene downstream of minimal P<SUB>BAD</SUB> promoter (P<SUB>BADmin</SUB>) (Fig. 5A). Minimal P<SUB>BAD</SUB> contains minimal DNA sequence from the native P<SUB>BAD</SUB> that is sufficient to retain its function (Figure 5B). The promoter retains all the operator sites O<SUB>1</SUB>, O<SUB>2</SUB>, I<SUB>1</SUB> and I<SUB>2</SUB> required for binding of AraC. As these sites lie within the PC promoter and overlap with AraC coding sequence, the minimal P<SUB>BAD</SUB> has the potential <I>araC</I> start codon ATG mutated to AGT to prevent the protein from being made (Fig. 5B). P<SUB>BADmin</SUB> was created and tested with the help of Steven Kane, a member of Colloms laboratory. To enable incorporation of P<SUB>BADmin</SUB> into a BioBrick-compatible plasmid, we added BioBrick prefix and suffix sequences to either side of the P<SUB>BADmin</SUB> sequence (Fig. 5B). Part K2442102 was expected to work as follows: when AraC is activated by its binding substrate, it binds to I<SUB>1</SUB> and I<SUB>2</SUB> half sites within P<SUB>BADmin</SUB> and drives expression of GFP, causing the host cell to produce green fluorescence. For the purposes of testing, the construct was ligated into pSB3K3 plasmid backbone. The plasmid has a different origin of replication from that of pSB1C3, therefore both plasmids can co-exist in the same <I>E. coli</I> cell. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B> Figure 4:</B> Regulatory Plasmid K2442104. The part contains <I>araC</I> or mutant <I>araC</I> amplified from part I0500. Expression of <I>araC</I> is regulated by lacI-regulated promoter R0011 with a downstream RBS B0032. The promoter is IPTG-inducible, but gives some <I>leaky</I> expression even in the absence of IPTG. The construct is flanked by BioBrick prefix and suffix, containing restriction sites for EcoRI, XbaI and SpeI, PstI, respectively. pSB1C3 carries Cm, providing resistance to chloramphenicol. The replication origin is a pUC19-derived pMB1 (copy number of 100-300 per cell)
		</DIV></DIV></CENTER><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B> Figure 5:</B> (A) Reporter plasmid K2442102. The part encodes GFP with the B0034 RBS downstream of minimal P<SUB>BAD</SUB> promoter, regulated by AraC. The part is flanked by BioBrick prefix and suffix. pSB3K3 plasmid backbone carries resistance to kanamycin. The vector has p15A pMR101-derived replication origin (copy number 20-30). (B) Annotated sequence of minimal P<SUB>BAD</SUB> promoter as ordered as a G-block. AraC potential start codon mutation ATGâAGT position is highlighted in <FONT color="magenta">magenta</FONT> (in reverse compliment). AraC-binding half sites are underlined and annotated. -35 and -10 elements of the P<SUB>BAD</SUB> promoter are highlighted in <FONT color="cyan">cyan</FONT>. AraC transcription start site (+1) highlighted in <FONT color="grey">grey</FONT>. P<SUB>C</SUB> promoter and its direction is indicated by an <FONT color="orange">orange</FONT> arrow. -35 and -10 elements of the P<SUB>C</SUB> are indicated. <FONT color="yellow">Yellow</FONT>, BioBrick prefix. <FONT color="green">Green</FONT>, BioBrick suffix. <FONT color="red">Red</FONT>, random 6 bp sequences to allow enzyme digest.)
		</DIV></DIV></CENTER><P>
The plasmids were all BioBrick compatible and at every stage of cloning were checked with gel electrophoresis and sequencing.  K2442102 and K2442104 were then transformed into DS941 cells and plated onto arabinose conditions to investigate function (Fig. 6). The scan showed that the colonies produced GFP in the presence of arabinose, even without the addition of IPTG (an inducer for the lacI-regulated promoter). The promoter is therefore leaky â background transcription of <I>araC</I> from K2442104 is sufficient to activate P<SUB>BADmin</SUB> in the K2442102 construct. 
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B> Figure 6:</B> Fluorescence scan of DS941 cells with both reporter and regulatory plasmids incorporated. Cells are plated on nutrient agar supplemented with 40mM arabinose.
		</DIV></DIV></CENTER><P>
Fig. 6 shows every colony to have high levels of fluorescence on arabinose. This confirms that the <I>araC</I> and P<SUB>BADmin</SUB> parts still function without being adjacent to one another. 
</P><P>The regulatory plasmid carrying the library of mutant <I>araC</I> was also constructed and transformed as discussed in the methods section. This gave us approximately 24,000 colonies, each containing a ligand-binding pocket variant of the AraC protein. This number however is not sufficient to cover all 160,000 possible variants. The mutant library DNA was confirmed by sequencing. Sequencing showed the mutations had been introduced at the correct positions (Fig. 7). The trace showed obvious NNS mutations at residues 8, 24, 80 and 82. At N positions 4 peaks are visible for any base (N), at S positions 2 peaks are visible for strong bases (S): G or C.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B> Figure 7:</B> Sequencing trace from mutant library MiniPrep, shows NNS mutations at residues 8, 24, 80 and 82.
		</DIV></DIV></CENTER><P>
MiniPrep DNA of the mutant <I>araC</I> library plasmid and K2442102 (reporter plasmid) were then transformed into <I>E. coli</I> DS941. These cells were then plated onto LB agar containing kanamycin and chloramphenicol with: A) no additive, B) 40mM xylose, C) 40mM arabinose or D) 2mM decanal and then scanned for GFP fluorescence (Fig. 8). The mutants were screened against no additive to look for mutant AraC with constitutive activity. They were screened against arabinose to check for unmutated/ mutated AraC with retained wild-type function. Xylose was used as it has a similar chemical structure to xylulose, but was much more economically viable. Xylose does not naturally stimulate WT AraC<SUP id="cite_ref-10" class="reference"><A href="#cite_note-10">[10]</A></SUP>. Decanal is a simple aldehyde, with a 3D structure very different to arabinose. It was expected to have no effect on the binding pocket of wild type AraC, however responsive mutants may be generated. We chose to screen against decanal to check whether mutants obtained are responsive specifically to xylose, or are also inducible at other conditions.
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B> Figure 8:</B> Plated E. coli transformants carrying mutant library DNA. Each plate was supplemented with either 40mM arabinose, 40mM xylose, 2mM Decanal or nothing at all (as labelled). The plates have been fluorescence imaged.
		</DIV></DIV></CENTER><P>
The mutant library was screened in four different conditions. Highly fluorescent, non- fluorescent, and colonies with intermediate fluorescence were seen in all conditions (Fig. 8). Multiple plates of each condition were screened. The presence of non-fluorescent colonies on arabinose shows that AraC has been mutated so that it no longer responds to arabinose.  Some fluorescent colonies were observed on plates containing xylose or decanal, suggesting that mutants that responded to these substances might have been obtained. Some fluorescent colonies were even observed with no added sugar (or decanal) in the agar. Presumably these reflect AraC mutants that are in the activated state in the absence of any sugar. Together these data show that the AraC protein has been effectively mutated, as some variations are showing different behaviour to the wild-type protein. 
</P><P>Due to time and labour constraints only around 200 colonies which exhibited fluorescence on either decanal or xylose plates were re-streaked onto plates with all four original conditions (i.e. xylose, arabinose, decanal or no chemical). Colonies were picked by referencing a printed out copy of the fluorescence scan and dotting the colonies which appeared to have strong fluorescence. This allowed for room for error. The fluorescence scans of the restreaked mutants showed a variety of behaviours (Fig. 9).
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B> Figure 9:</B> Fluorescence scans of re-streaks of mutant DS941 colonies which appeared to show fluorescence under xylose or decanal conditions.  Mutants which appeared to have novel behaviour are labelled 1-4.
		</DIV></DIV></CENTER><P>
The restreaks (Fig. 9) were examined and four colonies were chosen for further examination. All four of these colonies originated from the decanal plate. Colony no.1, (circled in green) was picked as it appeared to be fluorescing under decanal, xylose or arabinose conditions and less so when there was no additive in the agar. Colonies 2 and 3 (circled red) appeared to be fluorescing under all four conditions. Colony 4 appeared to only be fluorescing when streaked onto the plate containing decanal.  These four colonies were then picked from the no additive (decanal mutants) plate to be grown in liquid broth (no additive) with antibiotic, overnight. Portions of these overnights were then grown in a plate reader under all four conditions to monitor fluorescence over time (Fig. 10). Biobrick numbers K2442105 - K2442108 were given to the plasmids which contained R0011+ B0032+ a mutant variant of <I>araC</I>. Relevant BioBricks/ composite parts were also monitored for fluorescence to show function when P<SUB>BAD</SUB> and <I>araC</I> are split, to reconfirm that our constructs work as intended and to act as controls against the mutant plasmids activity (Fig. 10).
</P><CENTER><DIV class="article-image-container"><DIV class="article-image-caption "><B> Figure 10:</B> Average relative fluorescence over optical density at hour 8. Shows fluorescence levels under no additive, 40mM Arabinose, 40mM Xylose and 2mM decanal.  The cells which the plasmids have been transformed into are DS941 or DH5Î± (specified). Biobricks (K244210..) and plasmids (pSB1C3, pSB3K3) specified.
		</DIV></DIV></CENTER><P>
Figure 10 shows that our split constituent parts of I0500 (P<SUB>BADmin</SUB> and <I>araC</I>) function correctly. Reading from left to right on Fig. 10: There was negligible fluorescence of DH5Î± cells when transformed with R0011, B0032 and WT <I>araC</I> in pSB1C3. DH5Î± cells (which have a chromosomal copy of <I>araC</I>) with  P<SUB>BADmin</SUB> and I13500 (GFP) in pSB1C3 backbone (second along) showed 2000x fold increase in fluorescence under arabinose conditions and negligible fluorescence under all other conditions. This shows that the chromosomal copy of <I>araC</I> was sufficient to activate P<SUB>BADmin</SUB> and initiated expression of GFP. This reconfirms the functionality of the promoter, as well as proves high efficiency of minimal P<SUB>BAD</SUB> and improvement over part <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I0500">BBa_I0500</A>. Third along in Fig. 10 shows this experiment again, except P<SUB>BADmin</SUB>+GFP are in pSB3K3 backbone instead of pSB1C3. This is a lower copy number plasmid than pSB1C3. Again, expression of GFP was only seen under arabinose conditions, however levels of fluorescence were much lower. This suggests that the change in fluorescence is due to the plasmid copy number and must be taken into account in further experiments. The next set of data shows DS941 cells containing P<SUB>BADmin</SUB>+GFP in pSB3K3. DS941 does not have a chromosomal copy of <I>araC</I>. There were negligible levels of fluorescence in this instance, showing that <I>araC</I> is necessary for our construct to express GFP.  5th along shows DS941 cells transformed with both the regulatory plasmid (R0011+B0032+WT <I>araC</I> in pSB1C3) and the reporter plasmid P<SUB>BADmin</SUB>+GFP in pSB3K3). As they are DS941 cells, the only <I>araC</I> protein present is the one supplied by the regulatory plasmid. The results were comparable to data set 3, when a chromosomal copy was used to activate P<SUB>BADmin</SUB>(in pSB3K3), with high levels of expression only when exposed to arabinose. This shows that the coding sequence used for expression of wild-type <I>araC</I> in our construct produces functional protein. Empty DS941 and DH5Î± cells were also scanned for fluorescence - the levels were very low and have been used as controls to determine basal level for negligible fluorescence. 
</P><P>DS941 cells with the mutant variations of <I>araC</I> in the regulatory plasmid and the reporter plasmid were tested in a separate experiment due to lack of space in the plate reader. Protocol was carried out in the same way. 
</P><P>Mutant 1 (K2442105) showed levels of fluorescence comparable to empty DS941 cells. There was no obvious difference between arabinose, decanal and no additive (Fig. 10) when liquid culture was examined.  
Mutant 2 (K2442106) and Mutant 3 (K2442107) both gave an increased level of fluorescence (~100 fold) over no araC plasmid, with or without sugar or decanal additives. Addition of arabinose seemed to give a slight reduction in the level of fluorescence with these two mutants, suggesting that arabinose may be having an inhibitory effect on these variant of AraC.
Mutant 4, K2442108, showed no significant fluorescence (Fig. 10) in any conditions, similar to mutant 1. the level of fluorescence observed was similar to that of <I>empty</I> cells and is likely due to natural fluorescence of <I>E. coli</I>.
</P><P>None of the mutants shows an increase in GFP expression under arabinose conditions. This shows that the AraC molecule has been successfully mutated and is not behaving as the wild-type would do. 
</P><P>The mutant plasmids were sequenced. AraC naturally has proline at residue 8, threonine at residue 24, histidine at 80 and tyrosine at 82. Mutant 1 (K2442105) has the following mutations: 
</P><CENTER><B>proline-8âglycine (P8G), histidine-80âproline (H80P) and tyrosine-82âtryptophan (Y82W) </B></CENTER><P>
Mutant 2 (K2442106) and 3 (K2442107) both had the same mutations as follows: 
</P><CENTER><B>proline-8âserine (P8S), threonine-24âaspartic acid (T24D), histidine-80âisoleucine (H80I) and tyrosine-82âalanine (Y82A) </B></CENTER><P>
Sequencing result of mutant 4 (K2442108) came back inconclusive due to mispriming in the sequencing reaction. Sequencing remains to be repeated.
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>We have successfully shown that constituent parts of BBa_I0500 can function when split into two seperate plasmids.  Figures 5 and 10 show that the wild-type variant of <I>araC</I> under control of R0011 promoter and RBS B0032 on a high copy number plasmid is sufficient to bind to our minimal P<SUB>BAD</SUB> promoter and drive transcription of GFP encoded on a low copy number plasmid. The DS941 strain of <I>E. coli</I> does not produce its own copy of the AraC protein, therefore the only AraC which may be binding to the P<SUB>BADmin</SUB> promoter is the one transcribed from our construct. We have improved BBa_I0500 by providing additional control over arabinose-inducible systems. AraC can be placed under inducible promoters of choice, taking translational load off growing cells.
</P><P>Sequencing results confirmed that the protocol described generated desired mutations at amino acid residue positions 8, 20, 24, and 80. The four mutants showed different behaviour to the wild-type AraC. Substitution of these residues to change the effector specificty of AraC was previously done by Tang <I>et al</I> (2008)<SUP id="cite_ref-Tang_4-3" class="reference"><A href="#cite_note-Tang-4">[4]</A></SUP> to generate AraC responsive to the enantiomer D-Arabinose as opposed to L-Arabinose. Naturally AraC does not respond to D-arabinose. L-xylose was chosen by us for mutant screening as it has an identical chemical formula to L-arabinose but a different structural formula, just as L-arabinose and D-arabinose do. Decanal is an aldehyde molecule, just as arabinose, however the 3D structure of the molecule is very different, allowing for investigation of mutant AraC response to different types of molecules. 
</P><P>K2442105 showed no increase in fluorescence under all tested conditions. The mutant AraC has lost its responsiveness to arabinose. As the only mutations are present in the ligand-binding pocket, the protein most likely no longer binds to arabinose, or upon binding no longer undergoes required change in conformation to activate pBAD. The result shows that the mutations present in this mutant variant of AraC do not cause specific activation by any of the conditions tested. However, we tested a very limited number of possible inducers. The protein may be responsive to a molecule not yet identified.
</P><P>The sequencing results of K2442106 and K2442107 show amino acid changes P8S, T24D, H80I and Y82A. These changes resulted in a decrease in levels of fluorescence under arabinose conditions relative to the no sugar control, and 100x fold increase in fluorescence relative no AraC. This shows that unlike WT AraC, the mutant version is active in absence of arabinose. Fluorescence on the plate with no additive may be explained by loss of the protein's repressive effect on P<SUB>BAD</SUB>, which results in activation the promoter even when not bound to its ligand. Alternatively, mutant AraC may be activated by a compound present in the cell or abundant in the LB broth/ LB agar. Interestingly, both colonies showed decrease in levels of fluorescence in presence of arabinose, suggesting that arabinose may be actually having an inhibitory effect on mutant AraC. The change in protein structure and activity would need to be examined further before we could infer how the substitutions in the ligand-binding pocket of AraC are causing the observed response. It is intriguing that both mutants show the same mutations, as the chances of us finding two mutants with identical DNA sequence (out of 1,048,576 possible combinations) are extremely low. However, this is possible because of the way the mutant library was amplifed prior to our screening, allowing us to pick two colonies that had been transformed with duplicates of the same mutant. However, this does reconfirm the activity of this mutant sequence, which was inadvertently replicated and showed the same behaviour. 
</P><P>K2442108, similarly to K2442105 showed negligible expression of GFP. Sequencing result of colony 4 (K2442108) came back inconclusive due to mispriming in the sequencing reaction. This colony, similarly to K2442105 might have a mutated AraC which no longer responds to arabinose, but is possibly induced by a molecule not yet identified. However, it is also possible that this colony has lost <I>araC</I> or a part of the sequence from the construct all together. Sequencing needs to be repeated in order to explain the behaviour of this mutant.
</P><P>Any changes between behaviour on the original restreak plates (Fig. 9) and the fluorescence plate reader data (Fig. 10) were probably due to varying densities of <I>E. coli</I> on the restreak plates. Higher cell density made the colony appear darker and thus more fluorescent. 
</P><P>All the mutant colonies were only loaded into the plate reader once per colony per condition. These would need to be repeated at very least in triplicate before any more solid conclusions can be made. Also, if time permitted we would have gone on to use error prone PCR on our mutants to optimise their activity, as was done by Tang <I>et al</I> (2008)<SUP id="cite_ref-Tang_4-4" class="reference"><A href="#cite_note-Tang-4">[4]</A></SUP>. It must be noted that 200 is an extremely low number of colonies to do initial screening on. Our protocol was expected to have generated 160,000 different amino acid combinations at the 4 randomised residue positions. 3X library coverage of this number would give 95% confidence of every variant appearing once, which is 480,000 colonies. However, the initial transformation generated 24,000 colonies, out of which only 200 were picked for further screening. Expanding of our mutant library could allow identification of AraC variants responsive specifically to xylose, xylulose or other compounds.  Unfortunately, this was not possible in the time frame of the project. Screening with a FACS machine, as was done in a previous study<SUP id="cite_ref-Tang_4-5" class="reference"><A href="#cite_note-Tang-4">[4]</A></SUP>, would allow for screening of more mutants. It would also be useful to test different concentrations of each additive on the mutant colonies to investigate if the behaviour is concentration-dependent. As a further experiment mutant cells could be screened against many different molecules such as general sugars, aldehyde molecules or even molecules completely unlike arabinose (e.g. quorum sensing molecules).
</P><H2><SPAN class="mw-headline" id="Outlook">Outlook</SPAN></H2><P>The production of the mutant library of AraC is useful for synthetic biology. A catalogue of many of these mutants could possibly make the construction of regulatory systems such as biosensors much easier, and could be applied as a component of a biosensor toolbox. This could allow the generation of biosensors responsive to molecules which do not bind to any receptors in nature.
</P><P>The protocol undertaken in this investigation did have a reasonable degree of success and could be replicated by other iGEM teams. Time constraints were the main reason for the limited degree of testing. This could be overcome by other iGEM teams by following our protocol of ethanol precipitating the ligation DNA before transformation and recruiting more people to do screening work, or ideally using a FACS machine to sort the many variant mutant colonies. 
</P><H2><SPAN class="mw-headline" id="Parts_Made">Parts Made</SPAN></H2><UL><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442101">K2442101</A> - minimal P<SUB>BAD</SUB> promoter </LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442102">K2442102</A> - minimal P<SUB>BAD</SUB> promoter + <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_I13500">GFP</A></LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442103">K2442103</A> - WT AraC coding region</LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442104">K2442104</A> - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_R0011">R0011</A> + <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0032">B0032</A> + WT AraC</LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442105">K2442105</A> - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_R0011">R0011</A> + <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0032">B0032</A> + AraC variant 1</LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442106">K2442106</A> - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_R0011">R0011</A> + <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0032">B0032</A> + AraC variant 2</LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442107">K2442107</A> - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_R0011">R0011</A> + <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0032">B0032</A> + AraC variant 3</LI><LI><A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_K2442108">K2442108</A> - <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_R0011">R0011</A> + <A rel="nofollow" class="external text" href="http://parts.igem.org/Part:BBa_B0032">B0032</A> + AraC variant 4</LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><SMALL><OL class="references"><LI id="cite_note-1"><SPAN class="mw-cite-backlink"><A href="#cite_ref-1">â</A></SPAN><SPAN class="reference-text">BrÃ¼nker, P., Hils, M., Altenbuchner, J., and Mattes, R. (1998). The mannitol utilization genes of Pseudomonas fluorescens are regulated by an activator: Cloning, nucleotide sequence and expression of the mtlR gene. Gene 215, 19-27.</SPAN></LI><LI id="cite_note-2"><SPAN class="mw-cite-backlink"><A href="#cite_ref-2">â</A></SPAN><SPAN class="reference-text">Hoffmann, J., and Altenbuchner, J. (2015). Functional Characterization of the Mannitol Promoter of Pseudomonas fluorescens DSM 50106 and Its Application for a Mannitol-Inducible Expression System for Pseudomonas putida KT2440. PLOS ONE 10, e0133248.</SPAN></LI><LI id="cite_note-Schleif-3"><SPAN class="mw-cite-backlink">â <SUP><A href="#cite_ref-Schleif_3-0">3.0</A></SUP><SUP><A href="#cite_ref-Schleif_3-1">3.1</A></SUP><SUP><A href="#cite_ref-Schleif_3-2">3.2</A></SUP></SPAN><SPAN class="reference-text"> Schleif, R. (2000). Regulation of the L-arabinose operon of <I>Escherichia coli</I>. Trends In GeneticsÂ <I>16</I>, 559-565..</SPAN></LI><LI id="cite_note-Tang-4"><SPAN class="mw-cite-backlink">â <SUP><A href="#cite_ref-Tang_4-0">4.0</A></SUP><SUP><A href="#cite_ref-Tang_4-1">4.1</A></SUP><SUP><A href="#cite_ref-Tang_4-2">4.2</A></SUP><SUP><A href="#cite_ref-Tang_4-3">4.3</A></SUP><SUP><A href="#cite_ref-Tang_4-4">4.4</A></SUP><SUP><A href="#cite_ref-Tang_4-5">4.5</A></SUP></SPAN><SPAN class="reference-text">Tang, S., Fazelinia, H., and Cirino, P. (2008). AraC Regulatory Protein Mutants with Altered Effector Specificity. Journal Of The American Chemical SocietyÂ <I>130</I>, 5267-5271.</SPAN></LI><LI id="cite_note-5"><SPAN class="mw-cite-backlink"><A href="#cite_ref-5">â</A></SPAN><SPAN class="reference-text">Tang, S., and Cirino, P. (2010). Design and Application of a Mevalonate-Responsive Regulatory Protein. Angewandte ChemieÂ <I>123</I>, 1116-1118.</SPAN></LI><LI id="cite_note-6"><SPAN class="mw-cite-backlink"><A href="#cite_ref-6">â</A></SPAN><SPAN class="reference-text">Frei, C., Wang, Z., Qian, S., Deutsch, S., Sutter, M., and Cirino, P. (2016). Analysis of amino acid substitutions in AraC variants that respond to triacetic acid lactone. Protein ScienceÂ <I>25</I>, 804-814.</SPAN></LI><LI id="cite_note-7"><SPAN class="mw-cite-backlink"><A href="#cite_ref-7">â</A></SPAN><SPAN class="reference-text">Gilbert, M., Mandrell, R., Parker, C., Li, J., and Vinogradov, E. (2007). Structural Analysis of the Capsular Polysaccharide from Campylobacter jejuni RM1221. ChembiochemÂ 8, 625-631.</SPAN></LI><LI id="cite_note-8"><SPAN class="mw-cite-backlink"><A href="#cite_ref-8">â</A></SPAN><SPAN class="reference-text"> Miyada, C., Horwitz, A., Cass, L., Timko, J., and Wilcox, G. (1980). DNA sequence of the araC regulatory gene from <I>Escherichia coli</I> B/r. Nucleic Acids ResearchÂ 8, 5267-5274.</SPAN></LI><LI id="cite_note-9"><SPAN class="mw-cite-backlink"><A href="#cite_ref-9">â</A></SPAN><SPAN class="reference-text">Daber, R., Stayrook, S., Rosenberg, A., and Lewis, M. (2007). Structural Analysis of Lac Repressor Bound to Allosteric Effectors. Journal Of Molecular Biology 370, 609-619.</SPAN></LI><LI id="cite_note-10"><SPAN class="mw-cite-backlink"><A href="#cite_ref-10">â</A></SPAN><SPAN class="reference-text">Doyle, M. E., Brown, C., Hogg, R. W., &amp; Helling, R. B. (1972). Induction of the ara Operon of Escherichia coli B/r. Journal of Bacteriology, 110(1), 56â65.</SPAN></LI></OL></SMALL></DIV></DIV><FOOTER class="marketing-site-footer"><DIV class="row medium-unstack"><DIV class="medium-4 columns"><H4 class="marketing-site-footer-name">iGEM 2017 Glasgow</H4><P>Using synthetic biology to create a biosensor for detection of <I>Campylobacter jejuni</I>, a bacteria linked to food poisoning.</P></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Contact Info</H4><DIV class="marketing-site-footer-block"><P>                    iGEM 2017 Glasgow 
Wilkins Teaching Laboratory
                    Bower Building 
                    University of Glasgow</P></DIV><DIV class="marketing-site-footer-block"><P>igem.glasgow2017@gmail.com</P></DIV></DIV><DIV class="medium-4 columns"><H4 class="marketing-site-footer-title">Sponsors</H4></DIV></DIV><DIV class="marketing-site-footer-bottom"><DIV class="row large-unstack align-middle"><DIV class="column"><UL class="menu marketing-site-footer-bottom-links"><LI><A href="https://2017.igem.org/Team:Glasgow">Home</A></LI></UL></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>