<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SupBiotech-Paris_Antitumor_action"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:SupBiotech-Paris/Antitumor_action">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SupBiotech-Paris/Antitumor_action&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SupBiotech-Paris/Antitumor_action&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SupBiotech-Paris/Antitumor_action&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SupBiotech-Paris/Antitumor_action" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SupBiotech-Paris/Antitumor action</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:SupBiotech-Paris/Antitumoral_action&amp;redirect=no" title="Team:SupBiotech-Paris/Antitumoral action">Team:SupBiotech-Paris/Antitumoral action</A>)</DIV><DIV id="menuhaut"><TABLE style="background-color: transparent;" cellpadding="3" cellspacing="20" align="center"><TBODY><TR><TH align="center"><A href="/Team:SupBiotech-Paris/HomeEn#drapeau" title="Team:SupBiotech-Paris/HomeEn">Home</A></TH><TH align="center"><A href="/Team:SupBiotech-Paris/Team#drapeau" title="Team:SupBiotech-Paris/Team">Team</A></TH><TH align="center"><A href="/Team:SupBiotech-Paris/Project_Management#drapeau" title="Team:SupBiotech-Paris/Project Management">Project Management</A></TH><TH align="center"><A href="/Team:SupBiotech-Paris/Gallery#drapeau" title="Team:SupBiotech-Paris/Gallery">Gallery</A></TH><TH align="center"><A href="/Team:SupBiotech-Paris/Acknowledgement#drapeau" title="Team:SupBiotech-Paris/Acknowledgement">Acknowledgement</A></TH></TR></TBODY></TABLE></DIV><DIV id="menu" style="margin-top: 315px; background: transparent;"><DIV id="Partie" style="color: white; font-weight: bold; margin-bottom: -7px;">Project</DIV><P><SPAN style="color: black;"><B>Introduction</B></SPAN><A href="/Team:SupBiotech-Paris/Current_therapies#drapeau" title="Team:SupBiotech-Paris/Current therapies">Current therapies</A><A href="/Team:SupBiotech-Paris/Gene_therapies#drapeau" title="Team:SupBiotech-Paris/Gene therapies">Gene therapies</A><SPAN style="color: black;"><B>DVS</B></SPAN><A href="/Team:SupBiotech-Paris/Concept#drapeau" title="Team:SupBiotech-Paris/Concept">Concept</A><A href="/Team:SupBiotech-Paris/Concept1#drapeau" title="Team:SupBiotech-Paris/Concept1">Tissue Vector</A><A href="/Team:SupBiotech-Paris/Concept2#drapeau" title="Team:SupBiotech-Paris/Concept2">Cell Vector</A><A href="/Team:SupBiotech-Paris/Concept3#drapeau" title="Team:SupBiotech-Paris/Concept3">Therapeutic plasmid</A><A href="/Team:SupBiotech-Paris/Conclusion1#drapeau" title="Team:SupBiotech-Paris/Conclusion1">Conclusion</A><SPAN style="color: black;"><B>Implementation</B></SPAN><A href="/Team:SupBiotech-Paris/Tissue_targeting#drapeau" title="Team:SupBiotech-Paris/Tissue targeting">Tissue targeting</A><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Antitumor action</A><A href="/Team:SupBiotech-Paris/Treatement_modeling#drapeau" title="Team:SupBiotech-Paris/Treatement modeling">Treatment modeling</A><A href="/Team:SupBiotech-Paris/Conclusion3#drapeau" title="Team:SupBiotech-Paris/Conclusion3">Conclusion</A></P><DIV id="Partie" style="color: white; font-weight: bold; margin-bottom: -7px;">Ressources</DIV><P><A href="/Team:SupBiotech-Paris/Biobricks#drapeau" title="Team:SupBiotech-Paris/Biobricks">BioBricks</A><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Material &amp; Method</A><A href="/Team:SupBiotech-Paris/Bibliography#drapeau" title="Team:SupBiotech-Paris/Bibliography">Bibliography</A></P><DIV id="Partie" style="color: white; font-weight: bold; margin-bottom: -7px;">Ethics &amp; Safety</DIV><P><A href="/Team:SupBiotech-Paris/Ethic#drapeau" title="Team:SupBiotech-Paris/Ethic">Ethics</A><A href="/Team:SupBiotech-Paris/Safety#drapeau" title="Team:SupBiotech-Paris/Safety">Safety</A></P></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Cell_targeting"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Cell targeting</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Context"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Context</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Objective"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Objective</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Experimental_approach"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Experimental approach</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Results"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Design_of_the_fusion_protein"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">Design of the fusion protein</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Transfection_of_eukaryotic_cells_by_the_Lambda_phage_recombined_with_the_penton_base_fused_to_the_D_protein_.28Stefania_Piersanti_et_al..2C_2004.29"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">Transfection of eukaryotic cells by the Lambda phage recombined with the penton base fused to the D protein (Stefania Piersanti et al., 2004)</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-8"><A href="#Discussion"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Conclusion"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Antitumoral_Plasmid"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Antitumoral Plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Context_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Context</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#The_objective"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">The objective</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Experimental_approach_2"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Experimental approach</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Cancer_cell_line_and_reported_gene"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Cancer cell line and reported gene</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#TP53_gene_incorporation"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">TP53 gene incorporation</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Apoptosis_detection"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">Apoptosis detection</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#The_running_of_the_study"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">The running of the study</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Results_.5B1.2C2.5D"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Results [1,2]</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Conclusion_.5B3.2C4.2C5.2C6.2C7.2C8.2C9.5D"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Conclusion [3,4,5,6,7,8,9]</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Cell_targeting"> Cell targeting </SPAN></H1><H2><SPAN class="mw-headline" id="Context"> Context </SPAN></H2><P>After the <A href="/Team:SupBiotech-Paris/Concept1#drapeau" title="Team:SupBiotech-Paris/Concept1"> tissue vector</A> action, comes the <A href="/Team:SupBiotech-Paris/Concept2#drapeau" title="Team:SupBiotech-Paris/Concept2"> cell  vector</A>, this one is a modified bacteriophage which has the ability to infect eukaryotic cells. Lambda phage, because of its high capacity of cloning and a capsid structure adapted to a concentrated presence of exogenous proteins, is a good candidate to design an eukaryotic <A href="/Team:SupBiotech-Paris/Concept1#drapeau" title="Team:SupBiotech-Paris/Concept1"> tissue vector</A>. The <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> originally from the adenovirus capsid appears as a promising candidate for Lambda phage targeting. Indeed, it is endowed of several functions like the cell receptors link, the viral particles internalisation and the release of the capsid by the endosome.</P><H2><SPAN class="mw-headline" id="Objective">Objective </SPAN></H2><P>Our objectives are to design a <A href="/Team:SupBiotech-Paris/Concept2#drapeau" title="Team:SupBiotech-Paris/Concept2"> cell vector</A> of Lambda phage type recombined with a <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> from the adenovirus 5 fused by its <A href="/Team:SupBiotech-Paris/Biobricks#drapeau" title="Team:SupBiotech-Paris/Biobricks"> D protein</A>. The <A href="/Team:SupBiotech-Paris/Concept1#drapeau" title="Team:SupBiotech-Paris/Concept1"> tissue vector</A> should be able to integer the cell, go out of the endosome, transport its DNA to the nucleus of the cell and finally to transcript its <A href="/Team:SupBiotech-Paris/Concept3#drapeau" title="Team:SupBiotech-Paris/Concept3"> therapeutic genes</A>. </P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Cell_targeting#drapeau" title="Team:SupBiotech-Paris/Cell targeting" class="mw-redirect">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Experimental_approach"> Experimental approach </SPAN></H2><P>In the framework of recombinant phage gene design we decided to fuse the adenovirus 5 <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> to the <A href="/Team:SupBiotech-Paris/Biobricks#drapeau" title="Team:SupBiotech-Paris/Biobricks"> D protein</A> of the Lambda phage. The protein D extraction from Lambda phage genome has been lead by Polymerase Chain Reaction (PCR) with several couple of primers. The same  strategy has been applied for the adenovirus 5 <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> extraction which has been extracted from a plasmid coding for the virus offered by Dr. Karim Benihoud (UMR8121, CNRS/IGR, Villejuif, France). </P><P>After the fusion protein formation, this one is introduced in a BioBrick plasmid. This plasmid contains a resistance against an antibiotic to confirm the transfection of the recombined phage into bacteria and a reporter gene, like GFP, with eukaryotic promoter, the CMV of the <I>Simian virus</I> 40 (SV40), to confirm the transfection in eukaryotic cells. This strategy permits us to prove that the bacteriophage is able to infect eukaryotic cells. </P><P>Unfortunately, we have not been able to build the fusion protein in time. However, scientific literature show that the <A href="/Team:SupBiotech-Paris/Concept2#drapeau" title="Team:SupBiotech-Paris/Concept2"> cell vector</A>, a Lambda phage type, confection is possible by fusion of the <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> with the <A href="/Team:SupBiotech-Paris/Biobricks#drapeau" title="Team:SupBiotech-Paris/Biobricks"> D protein</A> (Stefania Piersanti et al. 2004). The central sequence of the <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A>, amino -acids 1 to 571, fused with the bacteriophage offers a transfection in eukaryotic cells, like the use of the RGD fragment responsible for the entry of the virus and the exit of the endosome, fragment 286 to 393. </P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><H3><SPAN class="mw-headline" id="Design_of_the_fusion_protein"> Design of the fusion protein </SPAN></H3><P>For the fusion protein design, we decided to extract separately the adenovirus 5<A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> and the <A href="/Team:SupBiotech-Paris/Biobricks#drapeau" title="Team:SupBiotech-Paris/Biobricks"> D protein</A> of the Lambda phage thanks to primers containing a BalI restriction site on the <A href="/Team:SupBiotech-Paris/Biobricks#drapeau" title="Team:SupBiotech-Paris/Biobricks"> protein D</A> reverse primer and the <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> forward primer. Moreover the finale fusion protein contains specific BioBricks fragments to its prefix and suffix. 
For these 2 genes extraction we used the following primers: </P><P>
First and second pair for genes extraction: </P><P>
D protein of the Lambda phage: </P><P>ForwardÂ : 5' ATG-ACG-AGC-AAA-GAA-ACC-TT 3'; 
ReverseÂ : 5' AAA-AAA-ATC-CCG-TAA-AAA-AAG-C 3'. </P><P>Adenovirus 5 penton baseÂ : </P><P>ForwardÂ : 5' AAT-GGC-CAA-TGC-GGC-GCG-CGG-CGA-TG 3' 
ReverseÂ : 5' CTG-CAG-CGG-CCG-CTA-CTA-GTA-TCA-AAA-AGT-GCG-G 3' </P><P>
Third pair for extension of the BalI restriction site and the BioBrick prefix only for the <A href="/Team:SupBiotech-Paris/Biobricks#drapeau" title="Team:SupBiotech-Paris/Biobricks">D protein</A> (already done for the penton base). </P><P>
ForwardÂ : 5' CGA-AAA-AAA-TGC-CCT-AAA-AAA-AAC-CGG-T 3' 
ReverseÂ : 5' AAT-GGC-CAA-AAA-AAA-TCC-CGT-AAA-AAA-AGC 3'</P><P>
Fourth pair for the D protein fusion amplification after ligation of the two fragments. </P><P>
ForwardÂ : 5' CTT-AAG-CGC-CGG-CGA-AGA-TC 3' 
ReverseÂ : 5' CTG-CAG-CGG-CCG-CTA-CTA-GTA 3' </P><P>PCR results are presented in figure 1. We check that there is the right amplification size fragment 1715bp for the penton base (sample number 11) and 385bp for the D protein (samples 7 and 8). However there is lots of mismatching during amplification cycles. This can have a negative effect on the result of the final amplification. </P><P><I>Figure 1: PCR of D protein BioBrick (1, 2, 3) and the penton base (4, 5, 6), D protein (7 and 8) and penton base (9, 10, 11) with BalI sites </I></P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H3><SPAN class="mw-headline" id="Transfection_of_eukaryotic_cells_by_the_Lambda_phage_recombined_with_the_penton_base_fused_to_the_D_protein_.28Stefania_Piersanti_et_al..2C_2004.29"> Transfection of eukaryotic cells by the Lambda phage recombined with the penton base fused to the D protein (Stefania Piersanti et al., 2004) </SPAN></H3><P>A cytofluorimetric study has been done to analyze the transfection rate of recombined Lambda phages. Figure 2 shows cytofluorimetric results of COS-1 cells analyze after to have been exposed to a concentration of 10^6 PFU/cells of recombinants phages, Pb (1-571) or Pb (286-393).
</P><P><I> Figure 2Â : Analyze of the  GFP fluorescence on non recombined Lambda phages (Lambda), recombined Lambda phages with the fragment 286-393 of the penton base (LambdaPb286-393), recombined Lambda phages with the complete penton base (1-571), GFP tagged adenovirus (Ad10 and Ad100)</I></P><P>
Firstly, we observe that the recombined phage shows a tag difference independently of the fragment of the <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> used compared to the non transformed bacteriophage. Secondly, the recombined phage with the RGD fragment alone (286-393) has a higher fluorescence than the phage with a complete fragment and closer to the adenovirus one (figure 2). </P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Discussion"> Discussion </SPAN></H2><P>Even if the tissue vector has not been finished, scientific literature shows that a recombinant phage creation with a protein coding the adenovirus <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> is possible. It demonstrates as well that fragment coding for RGD sequence alone has a higher capacity to infect eukaryotic cells compared to the <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> complete fragment (figure 2). In the case of our application it is possible to use a recombined Lambda phage to insert our therapeutic gene. </P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Conclusion"> Conclusion </SPAN></H2><P>To conclude the RGD fragment of the <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> alone has a higher efficiency of interaction with integrines of eukaryotic cells. However for our project, it was more judicious to use the complete sequence of the <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> (fragment 1-571) because the use of the <A href="/Team:SupBiotech-Paris/Concept2#drapeau" title="Team:SupBiotech-Paris/Concept2"> cell vector</A> and the induction system by doxycycline give a very fast and target injection of bacteriophages. The use of a highly efficient transfection system is not advised because phages do not have the time to disperse properly and will infect several times the same cell. The use of the complete fragment of the <A href="/Team:SupBiotech-Paris/Concept2#PB" title="Team:SupBiotech-Paris/Concept2"> penton base</A> is sufficient for the phage to infect properly eukaryotic cells and let it time to have a bigger dispersion. </P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H1><SPAN class="mw-headline" id="Antitumoral_Plasmid"> Antitumoral Plasmid </SPAN></H1><H2><SPAN class="mw-headline" id="Context_2"> Context </SPAN></H2><P>In non-small cell lung cancer, or NSCLC, like in all other cancers, the loss of apoptotic capacity of tumor cells is due to the functional loss of various tumor suppressors incoming in the apoptotic pathway.</P><P>The <A href="/Team:SupBiotech-Paris/Introduction1#drapeau" title="Team:SupBiotech-Paris/Introduction1" class="mw-redirect">DVS</A> application in the anticancer fight is based on the reactivation of this apoptotic pathway by bringing into tumor cells the wild type genes coding for functional tumor suppressors.</P><P>The <A href="http://www.sanger.ac.uk/genetics/CGP/cosmic/" class="external text" rel="nofollow">COSMIC project</A> from <A href="http://www.sanger.ac.uk/" class="external text" rel="nofollow">Sanger institute</A> allowed us to determine which genes to bring to the <A href="/Team:SupBiotech-Paris/Concept3#drapeau" title="Team:SupBiotech-Paris/Concept3">therapeutic plasmid</A> in the non-small cell lung cancer case. This project sums up all detected mutations for each type of cancer in function of their appearance frequency. So, from their data, the loss of apoptotic capacity of tumor cells for lung cancer can be due to the functional loss of proteins from the following genesÂ :</P><P>
These different genes play a predominant role in the application of the apoptotic process and are the most susceptible to be mutated in the lung cancer case. They compose the <A href="/Team:SupBiotech-Paris/Concept3#drapeau" title="Team:SupBiotech-Paris/Concept3">therapeutic plasmid</A>.</P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="The_objective"> The objective </SPAN></H2><P>The objective of this study is to check if a wild type version of a tumor suppressor gene inside the tumor cell, for which the own version is mutated, induce or not the apoptotic phenomenon.</P><H2><SPAN class="mw-headline" id="Experimental_approach_2"> Experimental approach </SPAN></H2><H3><SPAN class="mw-headline" id="Cancer_cell_line_and_reported_gene"> Cancer cell line and reported gene </SPAN></H3><P>Cancer cell lines, which have a mutation of tumor suppressor gene has been selected from our database. We also possess a wild type version of the TP53 gene. Then we choose the prostatic cancer p53 mutated DU-145 in the goals to test if bringing the wild type version of the p53 protein (p53wt) in the DU-145 cell line allows to induce the apoptotic process. </P><P><I>Cell culture protocolÂ : </I></P><OL><LI>Take out  ampoule from liquid nitrogen</LI><LI>Place the ampoule in 37Â°C water bath for 5 minutes</LI><LI>In a 50 ml Falcon tube, put 9 ml of 10% MEM + 1 ml of ampoule</LI><LI>Harvest 5 min at 1200 rpm</LI><LI>Discard the supernatant without touching pellet cells (DMSO elimination) </LI><LI>Resuspend pellet in 1 ml of media</LI><LI>Put the suspension in a new T25 containing 5 ml of media</LI><LI>Incubate at 37Â°C</LI><LI>Do not forget to change the media the day after to eliminate all DMSO traces </LI><LI>One week later, cells are at 100% confluence</LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H3><SPAN class="mw-headline" id="TP53_gene_incorporation"> TP53 gene incorporation </SPAN></H3><P>Incorporation of the plasmid containing p53wt, pcDNA3 CMV+p53wt, inside DU-145 cells is done by electroporation. </P><P><I>MaterialÂ :</I></P><UL><LI> DU-145 cells </LI><LI>pcDNA3 CMV+p53wt plasmid</LI><LI> Electrocompetent culture media</LI><LI>Trypsin</LI><LI>PBS</LI><LI>Icebox
</LI><LI>Electrotransfer Cuvette 
</LI><LI>Centrifuge
</LI><LI>Incubator
</LI><LI>Electroporator (cliniporator)
</LI></UL><P><I>Protocol: </I></P><OL><LI>Discard the media of T25 containing DU-145</LI><LI>Rinse with PBS</LI><LI>Add 500 Âµl of trypsin and let it acts for 3 minutes at room temperature </LI><LI>Add 5 ml of 10% MEM to neutralize trypsin</LI><LI>Suspend cells</LI><LI>Recover media containing DU-145 in a tube and harvest at 1000rpm for 10 minutes</LI><LI> Discard the supernatant and resuspend the pellet in XÂµl (X= 90Âµl x Number of cuvettes) of electrocompetent media (around 5x105 cells per cuvettes) </LI><LI>Suspend your DNA solution in electrocompetent media (18x10-2g/L) </LI><LI>Add 10Âµl DNA solution per cuvette</LI><LI> Add 90Âµl of the cell suspension</LI><LI>Put cuvettes in ice</LI><LI>Pass cuvettes to the electroporator (cliniporator) and save each result </LI><LI>Incubate cuvettes at 37Â°C for 30 minutes</LI><LI>Put the content of each cuvette in a sterile tube, add 3ml of MEM 10% culture media, and incubate at 37Â°C for the necessitate time (until the annexin V assay) </LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H3><SPAN class="mw-headline" id="Apoptosis_detection"> Apoptosis detection </SPAN></H3><P>Detection of apoptotic cells is done by the annexin V assay: </P><P>In the early stage of the apoptosis, we observe the phosphatidyl-serine translocation outside the cell membrane. This is highlighted by the specific fixation of the annexin V coupled with a fluorophore and analyzed by flow cytometry. </P><P><I>MaterialÂ :</I></P><UL><LI> Propidium iodide 1 mg/ml Invitrogen stored cold in the fridge, diluted 10 times</LI><LI>Annexin V</LI><LI>Annexin buffer</LI></UL><P>
Work as much as possible in the dark (fluorophores are photolabile) </P><P><I>ProtocolÂ : </I></P><OL><LI>Recover culture media (3 ml), put it in a Falcon tube of 50 ml</LI><LI>Rinse the culture with 3 ml of PBS, and dispose it in the Falcon tube</LI><LI>Remove cells with trypsin, and dispose them in the Falcon tube</LI><LI>Harvest</LI><LI>Resuspend the pellet in 0.5 or 1 ml of cold PBS in function of the confluence level</LI><LI>Take 10 Âµl to count and harvest</LI><LI>Re-suspend the pellet in annexin buffer at a concentration of 1*106 cell/ml</LI><LI>Take 2 aliquots of 100 Âµl in 2 FACS tubes </LI><LI>Add in each tube 5 Âµl of annexin V and 1 Âµl of propidium iodide</LI><LI>Incubate 15 min at RT</LI><LI>Stop the reaction by put tubes in melting ice </LI><LI>Add 400 Âµl of annexin V buffer</LI><LI>Read in FACS as quick as possible and let tubes in the ice</LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="The_running_of_the_study"> The running of the study </SPAN></H2><P>To analyse the timing of plasmid expression in the DU-145 cell line, we realized a kinetic monitoring of the apoptosis induction by making an annexin V assay every 6 hours for 48 hours after the plasmid electroporation. By coupling apoptosis rate of the population control (blank electroporation) and the population assay (electroporation with plasmid) with their respective growth rate, we will be able to determine the p53wt impact on apoptosis induction. We use a control without the plasmid pcDNA3 CMV+p53wt to quatify the level of death cells causes by the electroporation and by the culture transfert. </P><P>Because we had not a continuous access to the cytometer, we grouped the all 48h analyses in 2 cytometry runs. Each time slot of the study is represented by a distinct cell population. So, we realized 14 electroporations corresponding to the 7 time slots: +6h, +12h, +18h, +24h, +30h, +36h and +48h (two by slot: population assay+ population control). </P><P>
Here is the allocation planning of electroporations: </P><P>
Three cell populations were respectively electropored 12h, 24h et 36h before the first cytometry run (in red, at 9 a.m, day 3), four others 6h, 18h, 30h and 48h before the second run (in green, at 4 p.m, day 3). </P><P>The first cytometric analyze allowed us to obtain data for the monitoring at +12h, +24h and +36h, while the second one, allowed us to obtain data for the monitoring at +6h, 18h, +30h and +48h. </P><P>By coupling all these data, we obtain a monitoring on 48h of the apoptosis induction after p53wt electroporation.</P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Results_.5B1.2C2.5D"> Results [1,2] </SPAN></H2><P>Each cell population, which represents different time range of the monitoring, has been subjected to an annexin V assay at the instant looked for. Unfortunately, a wrong dilution of the annexin buffer caused the death of each cell population during the test. Even if results were convincing for the monitoring at +24h, +30h and +48h by simple comparison between the control and the test population in the microscope (figure 1), we could not confirm it by cytometric analysis. </P><CENTER><P><FONT size="1"><I>Figure 1</I>Â : cells morphology with or without p53 wild-type incorporation </FONT></P></CENTER><P>
Because we could only start DU-145 culture at the beginning of October, the two weeks needed to reach the necessary confluence did not let us to try a second experiment. </P><P>
However, several studies showed that to bring p53 wild type into tumor mutated cells launch the apoptosis process. It is notably the case of the study leaded by Chunlin Yang in 1995, who was working, like us, on mutated p53 prostatic cancer cells (Tsu-pr1). The p53 wild type were not transfected by electroporation but by infecting tumor cells by non replicatives adenoviruses containing p53wt (AdCMV.p53). 48 hours after the infection of a tumor population with AdCMV.p53, a high expression of p53 is correlated with an important rate of cell death. If control populations (non-infected cells and cells infected with adenovirus containing lacZ gene, AdCMV.NLSÃgal) show a similar and healthy morphology, condensation and cell detachment are observed in p53 infected population. To check if the death process followed by cells correspond to the apoptotic way, a migration on agar gel of their genome has been realized.</P><P><FONT size="1"><I>Figure 2 </I>: Electrophoresis on agar gel of isolated non-infected DNA cells (a), infected by AdCMV.NLSÃgal (b) and AdCMV.p53 (c).</FONT></P><P>Infected by AdCMV.p53, cells show multiple bands (laddering pattern) while non-infected cells or AdCMV.NLSÃgal infected cells show a unique one at high molecular weight. These results indicate that the cell induced by p53 wild type is from apoptotic origin by the genome fragmentation observation, consequence to the CAD (Caspase Activated DNase) activity, a specific endonuclease of the apoptotic process. </P><P>A MTT test allowed to quantify the effect induced by the p53 wild type expression into infected cells.</P><P><FONT size="1"><I>Figure 3 </I>: AdCMV.p53 effect on cell survive. Control and AdCMV.p53 infected cells were incubated in serum-free media after 1h of infection.</FONT></P><P>In serum absence, non-infected and Ãgal infected cells continue to proliferate. In contrast, for p53 infected cells, proliferation is stopped and followed by an important fall of population. After 72h, nearly the totality of p53 infected cells were dead (figure 3). </P><P><U><I>According to this study, it appears clearly that to bring a p53 wild-type version into the p53 mutated cell population induces the apoptosis phenomenon and decreases significantly the tumor population.</I></U></P><P>Similar results were reported in the study leaded by Corrado Cirielli (in 1999) by using similar analyses on a U251 cancer strain from a glioma. </P><P><DL><DT> Morphologic analysis of  AdCMV.p53 infected cells (a), non-infected (b) or infected by AdCMV.NULL (c)Â : </DT></DL></P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><P><DL><DD><FONT size="1"><I>Figure 4</I>Â : morphology AdCMV.p53 infected cells (a), non-infected (b) or infected by AdCMV.NULL (c), after one week infection. </FONT></DD></DL></P><P>Control populations (b and c) proliferate and form a cell layer one week after the beginning of experiences while the test population (a) show very few adherent cells (important cell loss) and a consequent morphologic change: cells are spherical.</P><P><DL><DT> AdCMV.p53 effect on DNA fragmentationÂ :</DT></DL></P><DL><DD><P><FONT size="1"><I>Figure 5 </I>: electrophoresis on agar gel of isolated DNA of non-infected cells, infected by AdCMV.NULL and AdCMV.p53.</FONT></P><P>After AdCMV.p53 infection, U-251 cells show a fragmentation of their genome, characteristic of the apoptosis process (laddering pattern).</P></DD><DT>Monitoring of the non-infected cells and infected by AdCMV.p53 or AdCMV.NULL cells by a MTT test:</DT><DD><P><FONT size="1"><I>Figure 6</I>Â : Control population proliferation (non-infected or AdCMV.NULL infected) and the test population by monitoring of the optical density after a MTT test.</FONT></P></DD><DD> Non-infected cells and  AdCMV.NULL infected cells proliferate in a significant way during the week of analysis while  AdCMV.p53 infected cells present a total absence of proliferation and a continuous decrease of their population.</DD><DD><U><I>This study shows one more time that to bring a p53wild-type version into a mutated p53 cell population induces cell death by apoptosis.</I></U><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Conclusion_.5B3.2C4.2C5.2C6.2C7.2C8.2C9.5D"> Conclusion [3,4,5,6,7,8,9] </SPAN></H2><P>Even if we could not give the proof by our own experiments, many studies show that to bring a wild-type version of a tumor suppressor gene into a mutated tumor cell for this gene permits to launch the apoptosis. <I><I>In vivo</I></I> studies on Human in the framework of the prostate, ovary and lung cancers have already been hold and present convincing results. </P><P>The implementation of this study has been originally done to determine if the <A href="/Team:SupBiotech-Paris/Concept#drapeau" title="Team:SupBiotech-Paris/Concept">DVS</A>, application in the fight against non small cell lung cancer, is feasible or not. Because we have not been able to conclude, the implementation of the study has been done by analyzing several publications. According to these publications, the application is first confirmed in the framework of the chosen pathology but it can also be reached to others cancers like hepatocellular carcinoma, on which the fact to bring a gene suppressor of tumor launch the apoptosis process. The only limitation is set by the <A href="/Team:SupBiotech-Paris/Concept1#drapeau" title="Team:SupBiotech-Paris/Concept1"> tissue vector</A> tropism.</P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Back to top</A></SPAN></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:SupBiotech-Paris/Antitumor_action">http://2009.igem.org/Team:SupBiotech-Paris/Antitumor_action</A>&quot;</DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SupBiotech-Paris/Antitumor_action" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SupBiotech-Paris/Antitumor_action" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SupBiotech-Paris/Antitumor_action&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SupBiotech-Paris/Antitumor_action&amp;oldid=169410" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DD></DL></DIV></DIV></DIV></BODY></HTML>