<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SupBiotech-Paris_Material_Method"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:SupBiotech-Paris/Material_%26_Method">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SupBiotech-Paris/Material_%26_Method&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SupBiotech-Paris/Material_%26_Method&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SupBiotech-Paris/Material_%26_Method&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SupBiotech-Paris/Material_%26_Method" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SupBiotech-Paris/Material &amp;amp; Method</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="menuhaut"><TABLE style="background-color: transparent;" cellpadding="3" cellspacing="20" align="center"><TBODY><TR><TH align="center"><A href="/Team:SupBiotech-Paris/HomeEn#drapeau" title="Team:SupBiotech-Paris/HomeEn">Home</A></TH><TH align="center"><A href="/Team:SupBiotech-Paris/Team#drapeau" title="Team:SupBiotech-Paris/Team">Team</A></TH><TH align="center"><A href="/Team:SupBiotech-Paris/Project_Management#drapeau" title="Team:SupBiotech-Paris/Project Management">Project Management</A></TH><TH align="center"><A href="/Team:SupBiotech-Paris/Gallery#drapeau" title="Team:SupBiotech-Paris/Gallery">Gallery</A></TH><TH align="center"><A href="/Team:SupBiotech-Paris/Acknowledgement#drapeau" title="Team:SupBiotech-Paris/Acknowledgement">Acknowledgement</A></TH></TR></TBODY></TABLE></DIV><DIV id="menu" style="margin-top: 315px; background: transparent;"><DIV id="Partie" style="color: white; font-weight: bold; margin-bottom: -7px;">Project</DIV><P><SPAN style="color: black;"><B>Introduction</B></SPAN><A href="/Team:SupBiotech-Paris/Current_therapies#drapeau" title="Team:SupBiotech-Paris/Current therapies">Current therapies</A><A href="/Team:SupBiotech-Paris/Gene_therapies#drapeau" title="Team:SupBiotech-Paris/Gene therapies">Gene therapies</A><SPAN style="color: black;"><B>DVS</B></SPAN><A href="/Team:SupBiotech-Paris/Concept#drapeau" title="Team:SupBiotech-Paris/Concept">Concept</A><A href="/Team:SupBiotech-Paris/Concept1#drapeau" title="Team:SupBiotech-Paris/Concept1">Tissue Vector</A><A href="/Team:SupBiotech-Paris/Concept2#drapeau" title="Team:SupBiotech-Paris/Concept2">Cell Vector</A><A href="/Team:SupBiotech-Paris/Concept3#drapeau" title="Team:SupBiotech-Paris/Concept3">Therapeutic plasmid</A><A href="/Team:SupBiotech-Paris/Conclusion1#drapeau" title="Team:SupBiotech-Paris/Conclusion1">Conclusion</A><SPAN style="color: black;"><B>Implementation</B></SPAN><A href="/Team:SupBiotech-Paris/Tissue_targeting#drapeau" title="Team:SupBiotech-Paris/Tissue targeting">Tissue targeting</A><A href="/Team:SupBiotech-Paris/Antitumor_action#drapeau" title="Team:SupBiotech-Paris/Antitumor action">Antitumor action</A><A href="/Team:SupBiotech-Paris/Treatement_modeling#drapeau" title="Team:SupBiotech-Paris/Treatement modeling">Treatment modeling</A><A href="/Team:SupBiotech-Paris/Conclusion3#drapeau" title="Team:SupBiotech-Paris/Conclusion3">Conclusion</A></P><DIV id="Partie" style="color: white; font-weight: bold; margin-bottom: -7px;">Ressources</DIV><P><A href="/Team:SupBiotech-Paris/Biobricks#drapeau" title="Team:SupBiotech-Paris/Biobricks">BioBricks</A><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Material &amp; Method</A><A href="/Team:SupBiotech-Paris/Bibliography#drapeau" title="Team:SupBiotech-Paris/Bibliography">Bibliography</A></P><DIV id="Partie" style="color: white; font-weight: bold; margin-bottom: -7px;">Ethics &amp; Safety</DIV><P><A href="/Team:SupBiotech-Paris/Ethic#drapeau" title="Team:SupBiotech-Paris/Ethic">Ethics</A><A href="/Team:SupBiotech-Paris/Safety#drapeau" title="Team:SupBiotech-Paris/Safety">Safety</A></P></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Electrophoresis"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Material"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Method"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Extraction_of_DNA_from_an_agarose_gel_.28Nucleospin_Clontech.29"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Extraction of DNA from an agarose gel (Nucleospin Clontech)</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Material_2"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Method_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Preparation_of_electrocompetent_Escherichia_coli"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Preparation of electrocompetent Escherichia coli</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Material_3"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Method_3"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Electroporation_of_electrocompetent_Escherichia_coli"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Electroporation of electrocompetent Escherichia coli</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Material_4"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Method_4"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Preparation_of_electrocompetent_Mycobacterium"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Preparation of electrocompetent Mycobacterium</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Material_5"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Method_5"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Electroporation_of_electrocompetent_Mycobacterium"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Electroporation of electrocompetent Mycobacterium</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Material_6"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Method_6"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Restriction"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Restriction</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Material_7"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Method_7"><SPAN class="tocnumber">7.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Linearization"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Linearization</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Material_8"><SPAN class="tocnumber">8.1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Method_8"><SPAN class="tocnumber">8.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#Ligation"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Material_9"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Method_9"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#PCR_.28Promega_Kit.29"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">PCR (Promega Kit)</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Material_10"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">Material</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Method_10"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">Method</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Primers"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">Primers</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#D_protein:"><SPAN class="tocnumber">10.3.1</SPAN><SPAN class="toctext">D protein:</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#COS_sequences:"><SPAN class="tocnumber">10.3.2</SPAN><SPAN class="toctext">COS sequences:</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#CI_repressor:"><SPAN class="tocnumber">10.3.3</SPAN><SPAN class="toctext">CI repressor:</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#Adenovirus_penton_based:"><SPAN class="tocnumber">10.3.4</SPAN><SPAN class="toctext">Adenovirus penton based:</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#BioBrick"><SPAN class="tocnumber">10.3.5</SPAN><SPAN class="toctext">BioBrick</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Electrophoresis"> Electrophoresis </SPAN></H2><H3><SPAN class="mw-headline" id="Material"> Material </SPAN></H3><UL><LI>DNA Template</LI><LI>Loading buffer</LI><LI>Molecular size marker</LI><LI>TAE 1X Buffer</LI><LI>Ethidium Bromide (BET) </LI><LI>Agarose</LI><LI>Detection and migration materials for an agarose gel electrophoresis</LI></UL><H3><SPAN class="mw-headline" id="Method"> Method </SPAN></H3><OL><LI>Agarose gel preparation: </LI><LI>For BioBricks: 
Prepare a 2% agarose gel (1.5 grams of agarose in 100mL of bi distilled H2O + 15ÂµL of BET). </LI><LI>To isolate DNA from a lambda phage: 
Prepare a 0.7% agarose gel (0.4 grams of agarose in 100mL of bi distilled H2O + 15ÂµL of BET). </LI><LI>Heat agarose in a microwave 5 to 10 minutes. </LI><LI>Prepare the electrophoresis plate. 
Ensure water tightness of the plate. </LI><LI>Add the TAE 1X buffer until the agarose gel is covered. </LI><LI>Prepare the samples:	
Mix 5ÂµL of loading buffer for 15ÂµL of DNA sample. 
Prepare the molecular size marker if needed: 
High range: 0.5ÂµL of DNA + 1ÂµL of loading buffer of the marker + 4.5ÂµL of bi distilled H2O. </LI><LI>Vortex the sample then centrifuge it for 2 seconds. </LI><LI>Deposit delicately the samples and the molecular size marker. 
Do not use the wells located on the edges of the gel. 
Deposit the molecular size marker in the center of the gel so as to ease the size analysis of the samples. 
An 8 well plate has a capacity of about 45ÂµL of sample. 
An15 well plate can hold 15ÂµL of sample. </LI><LI>Place the electrodes in the electrophoresis tank in order for the DNA to migrate from the anode to the cathode. </LI><LI>Activate the migration of the gel from 85 volts to 130 volts during 45 to 80 minutes. 
Check for the appearance of bubbles due to the electric current and that the DNA is migrating the right way. </LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Extraction_of_DNA_from_an_agarose_gel_.28Nucleospin_Clontech.29"> Extraction of DNA from an agarose gel (Nucleospin Clontech) </SPAN></H2><H3><SPAN class="mw-headline" id="Material_2"> Material </SPAN></H3><UL><LI>30mL of NT buffer</LI><LI>14mL of NT3 buffer</LI><LI>15mL of NE buffer</LI><LI>50 Nucleospin Extract II columns</LI><LI>50 Nucleospin collector tubes (2mL) </LI><LI>Sterile scalpel</LI><LI>Scales</LI><LI>Vortex</LI><LI>Incubator (50Â°C/122Â°F) </LI><LI>A centrifuge (11000g) </LI><LI>Sterile 1.5mL tubes</LI></UL><H3><SPAN class="mw-headline" id="Method_2"> Method </SPAN></H3><OL><LI>Excise the fragment of agarose gel that contains the DNA fragment, with a scalpel, whilst being careful to remove as little gel as possible. </LI><LI>Determine the weight of the slice of gel and place it in a sterile tube. </LI><LI>For every 100mg of agarose gel add 200Î¼L of NT buffer. 
For gels containing more than 2% of agarose, double the volume of NT buffer. 
The maximum weight for a slice of gel, produced by NucleoSpinÂ® Extract II, is 400mg or 200mg for high percentage gels (&gt;2%), in this case, 2 steps are necessary (DNA linking steps: 6 and 7). </LI><LI>Place the samples at 50Â°C (122Â°F) in the incubator until the gel has dissolved (5-10min). </LI><LI>Vortex briefly the sample every 2 to 3 min until the gel has completely dissolved. </LI><LI>Place a NucleoSpinÂ® Extract II column in a 2mL NucleoSpin collector tube and insert the sample, then centrifuge for 1 min at 11000g. </LI><LI>Throw away the solution contained in the NucleoSpin collector tube and replace the NucleoSpinÂ® Extract II column in the same NucleoSpin collector tube. </LI><LI>Add 600Î¼L of NT3 buffer and centrifuge for 1 min at 11000g. </LI><LI>Throw away the solution contained in the NucleoSpin collector tube and replace the NucleoSpinÂ® Extract II column in the same NucleoSpin collector tube. </LI><LI>Centrifuge for 2 min at 11000g to get rid of all the NT3 buffer. 
Make sure that the NucleoSpinÂ® Extract II column is not in contact with the eluted solution during centrifugation (in the NucleoSpin collector tube). 
The next reactions are inhibited by the residual ethanol from the NT3 buffer, so it must be entirely removed at this step. In addition to the centrifugation, placing the NucleoSpinÂ® Extract II column in an incubator at 70Â°C (158Â°F) for 2-5 min before elution will ensure total retrieval of the ethanol. </LI><LI>Place the NucleoSpinÂ® Extract II column in a fresh 1.5mL tube. </LI><LI>Add 15-50Î¼L of NE elution buffer, then incubate at room temperature for 1 min to enhance the yield of the eluted DNA. </LI><LI>Centrifuge for 1 min at 11000g. 
The yield of the large fragments (&gt;5-10 kb) can be enhanced by using pre-heated elution buffer (70Â°C/158Â°F): for the elution, add the pre-heated elution buffers and incubate at room temperature for 1 min before collecting the elute by centrifugation. </LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Preparation_of_electrocompetent_Escherichia_coli"> Preparation of electrocompetent Escherichia coli </SPAN></H2><H3><SPAN class="mw-headline" id="Material_3"> Material </SPAN></H3><UL><LI>500ÂµL of E.Coli culture in an LB culture medium</LI><LI>LB culture medium</LI><LI>60mL of glycerol at 10%</LI><LI>Ethanol</LI><LI>Incubator with rotor</LI><LI>Tank of ice</LI><LI>1.5mL tubes</LI><LI>50mL Falcon tubes</LI><LI>300mL tubes for the centrifuge</LI><LI>A refrigerated centrifuge at 4Â°C (39.2Â°F) </LI></UL><H3><SPAN class="mw-headline" id="Method_3"> Method </SPAN></H3><OL><LI>Prepare a pre-culture of E.Coli in 5mL of LB culture medium (all night long at 37Â°C/39.2Â°F with rotation). 
Prepare the centrifuge at 4Â°C (39.2Â°F). </LI><LI>Place the pre-culture of E.Coli in 450mL of LB culture medium and incubate with agitation for 3 hours at 37Â°C (98.6Â°F). </LI><LI>Place 225mL of the solution in 2 centrifuge tubes. </LI><LI>Place both tubes in the ice for 15 minutes. </LI><LI>Centrifuge at 2000g for 10 minutes at 4Â°C (39.2Â°F). </LI><LI>Throw away the supernatant and re-suspend delicately the pellet with 200mL of cold sterile water.
Firstly add 10 to 20mL of cold sterile water and re-suspend by pipeting, and then add the other 200mL. </LI><LI>Centrifuge at 2000g for 10 minutes at 4Â°C (39.2Â°F). </LI><LI>Throw away the supernatant and re-suspend delicately the pellet with 200mL of cold sterile water.
Firstly add 10 to 20mL of cold sterile water and re-suspend by pipeting, and then add the other 200mL. </LI><LI>Place the tubes in the ice for 30 minutes. </LI><LI>Centrifuge at 2000g for 15 minutes at 4Â°C (39.2Â°F). </LI><LI>Throw away the supernatant and re-suspend delicately the pellet with 25mL of glycerol at 10%.
You may also transfer the solution into a 50mL Falcon tube. </LI><LI>Place the tubes in the ice for 30 minutes. </LI><LI>Centrifuge at 1500g for 15 minutes at 4Â°C (39.2Â°F). </LI><LI>Throw away the supernatant and add 500ÂµL of glycerol at 10%.</LI><LI>Re-suspend the cells in a total volume of 1mL. </LI><LI>Aliquote 50ÂµL per tube (tubes already in the ice). </LI><LI>Store at -80Â°C (-112Â°F). </LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Electroporation_of_electrocompetent_Escherichia_coli"> Electroporation of electrocompetent Escherichia coli </SPAN></H2><H3><SPAN class="mw-headline" id="Material_4"> Material </SPAN></H3><UL><LI>Liquid LB medium</LI><LI>Solid LB medium (LB culture medium + agarose + specific antibiotics) </LI><LI>Frozen electro competent Escherichia coli bacteria (40ÂµL in an eppendorf tube) 
DNA</LI><LI>Tank full of ice</LI><LI>Electroporation basin (2mm) </LI><LI>Electroporator</LI><LI>Incubator with agitation incorporated</LI></UL><H3><SPAN class="mw-headline" id="Method_4"> Method </SPAN></H3><OL>
0. Place the electroporation basin in the ice. <LI>Leave the tube containing the bacteria (about 40ÂµL) at room temperature, so as to unfreeze the medium. Then place it in the ice. </LI><LI>Set the eclectroporator according to the following parameters: <DIV style="margin-left: 100px;"><P>	Voltage 2500 V
	Capacitance 25 ÂµF
	Resistance 200 Î©
	Basin 2 mm</P></DIV></LI><LI>Add 1ÂµL of DNA solution in the tube containing the bacteria. </LI><LI>Place the tube in the ice for 1 minute. </LI><LI>Transfer the contents of the tube into the electroporation basin. </LI><LI>Dry the outside of the basin with paper towel. Place it in the specific compartment of the electroporator. </LI><LI>Start the pulse. </LI><LI>Add immediately 1mL of culture medium in the basin. 
Beware: a 3 minute delay reduces by 90% the effectiveness of the transformation. </LI><LI>Transfer the contents of the basin into a tube. Incubate the tube at 37Â°C (98.6Â°F) with agitation for an hour. 
Soak the basins in bleach for a few minutes. Rinse them out thoroughly with tap water, then with bi distilled water and 70% ethanol (no acetone), before letting them dry. Thus they can be re-used. </LI><LI>Place the bacteria on a solid LB medium for cultivation. </LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Preparation_of_electrocompetent_Mycobacterium"> Preparation of electrocompetent Mycobacterium </SPAN></H2><H3><SPAN class="mw-headline" id="Material_5"> Material </SPAN></H3><UL><LI>500Î¼L of Mycobacterium culture at -80Â°C (-112Â°F) </LI><LI>Glycerol 10%</LI><LI>1.5mL tubes</LI><LI>Centrifuge</LI></UL><H3><SPAN class="mw-headline" id="Method_5"> Method </SPAN></H3><OL>
0. All the mediums and the whole manipulation are done at room temperature. <LI>Transfer the cells from your frozen sample into the centrifuge tubes. </LI><LI>Centrifuge the cells at 3600 rpm for 10 min, then throw away the supernatant. </LI><LI>Wash the cells with Â½ a volume (25mL) of sterile 10% glycerol by pipetting slowly until the cell mass has been dissolved. </LI><LI>Wash and precipitate the cells as done in steps 2 and 3; with Â¼ vol (12.5mL) of sterile 10% glycerol. </LI><LI>Wash and precipitate the cells as done in steps 2 and 3; with â vol (6.75mL) of sterile 10% glycerol. </LI><LI>Wash and precipitate the cells as done in steps 2 and 3; with â10 vol (5mL) of sterile 10% glycerol. </LI><LI>Wash and precipitate the cells as done in steps 2 and 3; with â25 vol (2mL) of sterile 10% glycerol. </LI><LI>Aliquot 100ÂµL of cells into the 1.5mL tubes. </LI><LI>Leave to cool quickly in an ice tank. </LI><LI>Store at -80Â°C (-112Â°F) or use immediately (the effectiveness of the electro-transfer is improved). </LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Electroporation_of_electrocompetent_Mycobacterium"> Electroporation of electrocompetent Mycobacterium </SPAN></H2><H3><SPAN class="mw-headline" id="Material_6"> Material </SPAN></H3><UL><LI>Liquid Loewenstein medium</LI><LI>Solid Loewenstein medium (congealed Loewenstein culture medium + adapted antibiotics) </LI><LI>Frozen electro competent mycobacterium (40ÂµL in a tube) 
DNA</LI><LI>Tank full of ice</LI><LI>Electroporation basins (2mm) </LI><LI>Electroporator</LI><LI>Incubator with agitation incorporated</LI></UL><H3><SPAN class="mw-headline" id="Method_6"> Method </SPAN></H3><OL>
0. Place the electroporation basin in the ice. <LI>Leave the tube containing the bacteria (about 40ÂµL) at room temperature, so as to unfreeze the medium. Then place it in the ice. </LI><LI>Set the eclectroporator according to the following parameters: <DIV style="margin-left: 100px;"><P>	Voltage 2500 V
	Capacitance 25 ÂµF
	Resistance 1000 Î©
	Basin 2 mm</P></DIV></LI><LI>Add 1ÂµL of DNA solution in the tube containing the bacteria. </LI><LI>Place the tube in the ice for 1 minute. </LI><LI>Transfer the contents of the tube into the electroporation basin. </LI><LI>Dry the outside of the basin with paper towel. Place it in the specific compartment of the eclectroporator. </LI><LI>Start the pulse. </LI><LI>Add immediately 1mL of culture medium in the basin. 
Beware: a 3 minute delay reduces by 90% the effectiveness of the transformation. </LI><LI>Transfer the contents of the basin into a tube. Incubate the tube at 37Â°C (98.6Â°F) with agitation for an hour. 
Soak the basins in bleach for a few minutes. Rinse them out thoroughly with tap water, then with bi distilled water and 70% ethanol (no acetone), before letting them dry. Thus they can be re-used. </LI><LI>Place the bacteria on a solid Loewenstein medium for cultivation. </LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Restriction"> Restriction </SPAN></H2><H3><SPAN class="mw-headline" id="Material_7"> Material </SPAN></H3><UL><LI>10ÂµL of DNA (1ÂµL for 0.5g of DNA) </LI><LI>3ÂµL of restriction buffer (1X of the total volume) </LI><LI>1ÂµL of restriction enzyme</LI><LI>3ÂµL of BSA (1X the total volume) </LI><LI>15ÂµL of sterile distilled water</LI><LI>Total volume 30ÂµL</LI><LI>1.5mL tubes</LI></UL><H3><SPAN class="mw-headline" id="Method_7"> Method </SPAN></H3><OL><LI>In a sterile eppendorf tube place 3ÂµL of buffer. </LI><LI>Add DNA. </LI><LI>Add 3ÂµL of BSA. </LI><LI>Then add 1ÂµL of restriction enzyme. </LI><LI>Complete with sterile distilled water. 
TOTAL VOLUME = 30ÂµL</LI><LI>Set the thermocycler according to the following parameters and place the tube inside: <DIV style="margin-left: 100px;"><P>	a. 1-2 hours at 37Â°C (98.6Â°F) 
	b. 20 min at 80Â°C to inactivate the enzymes
	c. leave at 4Â°C until opening of the thermocycler. </P></DIV><P>NOTE: if you can leave the tube in the incubator for more than 6 hours for the restriction, leave it longer to obtain a better quality restriction! </P></LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Linearization"> Linearization </SPAN></H2><H3><SPAN class="mw-headline" id="Material_8"> Material </SPAN></H3><UL><LI>DNA template</LI><LI>1ÂµL of alkaline phosphatase</LI><LI>3ÂµL of alkaline phosphatase buffer</LI><LI>Pure water for a total volume of 30ÂµL</LI><LI>A hotplate</LI><LI>1.5mL tubes</LI></UL><H3><SPAN class="mw-headline" id="Method_8"> Method </SPAN></H3><OL>
0. Prepare a hot tub at 37Â°C (98.6Â°F). <LI>Add the alkaline phosphatase concentration 1X buffer in the DNA template, meaning 3ÂµL of buffer for a final volume of 30ÂµL. </LI><LI>Add 1ÂµL of alkaline phosphatase. </LI><LI>Incubate for 1 hour at 37Â°C (98.6Â°F). </LI><LI>Deactivate the alkaline phosphatase by heating for 5 minutes at 65Â°C (149Â°F). 
Proceed to the ligation just after the linearization. </LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="Ligation"> Ligation </SPAN></H2><H3><SPAN class="mw-headline" id="Material_9"> Material </SPAN></H3><UL><LI>3ÂµL of 10X ligation buffer</LI><LI>1ÂµL of T4 DNA Ligase</LI><LI>100 to 1000Âµg of DNA insert solution</LI><LI>1ÂµL of 0.05g/L DNA vector</LI><LI>Sterile water to obtain Ã  final volume of 30ÂµL</LI><LI>1.5mL tubes</LI></UL><H3><SPAN class="mw-headline" id="Method_9"> Method </SPAN></H3><OL><LI>In a 1.5mL tube, place 2ÂµL of ligation buffer. </LI><LI>Add the DNA insert as well as 1ÂµL of vector. </LI><LI>Add 1ÂµL of T4 DNA Ligase. </LI><LI>Complete with sterile water to obtain a final volume of 30ÂµL. </LI><LI>Let the reaction take place for 35 minutes at 22.5Â°C (72.5Â°F). </LI><LI>Denaturize the T4 DNA Ligase at 65Â°C (149Â°F) for 10 minutes. </LI></OL><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Back to top</A></SPAN></P><H2><SPAN class="mw-headline" id="PCR_.28Promega_Kit.29"> PCR (Promega Kit) </SPAN></H2><H3><SPAN class="mw-headline" id="Material_10"> Material </SPAN></H3><UL><LI>DNA template</LI><LI>5â to 3â Primer</LI><LI>3â to 5â Primer</LI><LI>GoTaq Hot Start Green Mix (Promega) </LI><LI>Thermocycler</LI></UL><H3><SPAN class="mw-headline" id="Method_10"> Method </SPAN></H3><P>1. Place the GoTaq Hot Start Green Mix kit at room temperature, vortex then centrifuge it briefly to concentrate the mix at the bottom of the tube. </P><P>2. Prepare the following PCR mix:</P><P>
a. For 25ÂµL:</P><P>
b. For 50ÂµLÂ :</P><P>
c. For 100ÂµLÂ :</P><P>
3. Add the components in the order listed in the table. </P><P>Be careful to insert the volume of primers and of DNA template correctly on the edge of the tube so as to visualize all the volumes that have been deposited. </P><P>4. Vortex the total volume. </P><P>5. Â«Â Start-StopÂ Â» Centrifuge (2 to 5 seconds).</P><P>If the thermocycler does not have a heating top, add 1 to 2 drops of sterile mineral oil. </P><P>6. Insert the preparation into the thermocycler, then launch the cycles: </P><P>Cycles:</P><P>
7. Check your PCR products by electrophoresis. </P><P><SPAN style="float: right"><A href="/Team:SupBiotech-Paris/Material_%26_Method#drapeau" title="Team:SupBiotech-Paris/Material &amp; Method">Back to top</A></SPAN></P><H3><SPAN class="mw-headline" id="Primers"> Primers </SPAN></H3><H4><SPAN class="mw-headline" id="D_protein:"> D protein: </SPAN></H4><P>To extract the sequences from the full genome of the Lambda phage: 
ForwardÂ : 5'-ATG-ACG-AGC-AAA-GAA-ACC-TT 
ReverseÂ : 5'-AAA-AAA-ATC-CCG-TAA-AAA-AAG-C </P><P>To add BioBrick prefix and suffix: 
Forward: 5'-GAA-TTC-GCG-GCC-GCT-TCT-AGA-TGA-CGA-GCA-AAG-AAA-CCT-TTA-CCC
Reverse: 5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA-AAA-AAA-ATC-CCG-TAA-AAA-AAG-CCG-C</P><P>To add BalI sequence: 
Reverse:  5'-AAT-GGC-CAA-AAA-AAA-TCC-CGT-AAA-AAA-AGC</P><H4><SPAN class="mw-headline" id="COS_sequences:"> COS sequences: </SPAN></H4><P>To extract the sequences from the full genome of the Lambda phage: 
ForwardÂ : 5'-TGG-CCT-CCA-CGC-ACG-TTG
ReverseÂ : 5'-AGA-GGC-CAA-AAA-GCC-TCG-CTT-TC</P><P>To add BioBrick prefix and suffix: 
Forward:  5'-GAA-TTC-GCG-GCC-GCT-TCT-AGA-GGA-TAT-GTA-GAT-GAT-AAT-CAT-TAT-C
Reverse:  5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA-AGA-GGC-CAA-AAA-GCC-TCG-CTT-T</P><H4><SPAN class="mw-headline" id="CI_repressor:"> CI repressor: </SPAN></H4><P>To extract the sequences from the full genome of the Lambda phage: 
ForwardÂ : 5'-AAT-TCA-ATC-CAT-TTA-CTA-TGT-TAT-GT
ReverseÂ : 5'-ATG-AGC-ACA-AAA-AAG-AAA-CC</P><P>To add BioBrick prefix and suffix: 
Forward:  5'-GAA-TTC-GCG-GCC-GCT-TCT-AGA-GAA-TTC-AAT-CCA-TTT-ACT-ATG-TTA-TG
Reverse:  5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA-ATG-AGC-ACA-AAA-AAG-AAA-CCA-TTA-A</P><H4><SPAN class="mw-headline" id="Adenovirus_penton_based:"> Adenovirus penton based: </SPAN></H4><P>To extract the sequences from the full genome of the Lambda phage: 
ForwardÂ : 5'-ATG-CGG-CGC-GCG-GCG-ATG
ReverseÂ : 5'-TCA-AAA-AGT-GCG-GCT-CGA-TAG-GAC-G</P><P>To add BioBrick prefix and suffix: 
Forward:  5'-GAA-TTC-GCG-GCC-GCT-TCT-AGA-TGC-GGC-GCG-CGG-CGA-TGT-A
Reverse:  5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA-TCA-AAA-AGT-GCG-GCT-CGA-TAG-G</P><P>To add BalI sequence: 
Forward: 5'-AAT-GGC-CAA-TGC-GGC-GCG-CGG-CGA-TG
Reverse: 5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA-TCA-AAA-AGT-GCG-G</P><H4><SPAN class="mw-headline" id="BioBrick"> BioBrick </SPAN></H4><P>To amplify BioBrick prefix and suffix: 
Prefix for sequences which begin by ATG 
5'-GAA-TTC-GCG-GCC-GCT-TCT-AG
Prefix for sequences which not begin by ATG 
5'-GAA-TTC-GCG-GCC-GCT-TCT-AGA-G
Suffix: 
5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:SupBiotech-Paris/Material_%26_Method">http://2009.igem.org/Team:SupBiotech-Paris/Material_%26_Method</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SupBiotech-Paris/Material_%26_Method" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SupBiotech-Paris/Material_%26_Method" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SupBiotech-Paris/Material_%26_Method&amp;_Method=&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SupBiotech-Paris/Material_%26_Method&amp;oldid=166683" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>