"
Page
Discussion
View source
History
teams
Log in
 
Team:SupBiotech-Paris/Material &amp; Method
From 2009.igem.org
Home
Team
Project Management
Gallery
Acknowledgement
Project
Introduction Current therapies Gene therapies DVS Concept Tissue Vector Cell Vector Therapeutic plasmid Conclusion Implementation Tissue targeting Antitumor action Treatment modeling Conclusion
Ressources
BioBricks Material & Method Bibliography
Ethics & Safety
Ethics Safety
Contents
1 Electrophoresis
1.1 Material
1.2 Method
2 Extraction of DNA from an agarose gel (Nucleospin Clontech)
2.1 Material
2.2 Method
3 Preparation of electrocompetent Escherichia coli
3.1 Material
3.2 Method
4 Electroporation of electrocompetent Escherichia coli
4.1 Material
4.2 Method
5 Preparation of electrocompetent Mycobacterium
5.1 Material
5.2 Method
6 Electroporation of electrocompetent Mycobacterium
6.1 Material
6.2 Method
7 Restriction
7.1 Material
7.2 Method
8 Linearization
8.1 Material
8.2 Method
9 Ligation
9.1 Material
9.2 Method
10 PCR (Promega Kit)
10.1 Material
10.2 Method
10.3 Primers
10.3.1 D protein:
10.3.2 COS sequences:
10.3.3 CI repressor:
10.3.4 Adenovirus penton based:
10.3.5 BioBrick
Electrophoresis
Material
DNA Template
Loading buffer
Molecular size marker
TAE 1X Buffer
Ethidium Bromide (BET)
Agarose
Detection and migration materials for an agarose gel electrophoresis
Method
Agarose gel preparation:
For BioBricks:
Prepare a 2% agarose gel (1.5 grams of agarose in 100mL of bi distilled H2O + 15µL of BET).
To isolate DNA from a lambda phage:
Prepare a 0.7% agarose gel (0.4 grams of agarose in 100mL of bi distilled H2O + 15µL of BET).
Heat agarose in a microwave 5 to 10 minutes.
Prepare the electrophoresis plate.
Ensure water tightness of the plate.
Add the TAE 1X buffer until the agarose gel is covered.
Prepare the samples:
Mix 5µL of loading buffer for 15µL of DNA sample.
Prepare the molecular size marker if needed:
High range: 0.5µL of DNA + 1µL of loading buffer of the marker + 4.5µL of bi distilled H2O.
Vortex the sample then centrifuge it for 2 seconds.
Deposit delicately the samples and the molecular size marker.
Do not use the wells located on the edges of the gel.
Deposit the molecular size marker in the center of the gel so as to ease the size analysis of the samples.
An 8 well plate has a capacity of about 45µL of sample.
An15 well plate can hold 15µL of sample.
Place the electrodes in the electrophoresis tank in order for the DNA to migrate from the anode to the cathode.
Activate the migration of the gel from 85 volts to 130 volts during 45 to 80 minutes.
Check for the appearance of bubbles due to the electric current and that the DNA is migrating the right way.
Back to top
Extraction of DNA from an agarose gel (Nucleospin Clontech)
Material
30mL of NT buffer
14mL of NT3 buffer
15mL of NE buffer
50 Nucleospin Extract II columns
50 Nucleospin collector tubes (2mL)
Sterile scalpel
Scales
Vortex
Incubator (50°C/122°F)
A centrifuge (11000g)
Sterile 1.5mL tubes
Method
Excise the fragment of agarose gel that contains the DNA fragment, with a scalpel, whilst being careful to remove as little gel as possible.
Determine the weight of the slice of gel and place it in a sterile tube.
For every 100mg of agarose gel add 200μL of NT buffer.
For gels containing more than 2% of agarose, double the volume of NT buffer.
The maximum weight for a slice of gel, produced by NucleoSpin® Extract II, is 400mg or 200mg for high percentage gels (>2%), in this case, 2 steps are necessary (DNA linking steps: 6 and 7).
Place the samples at 50°C (122°F) in the incubator until the gel has dissolved (5-10min).
Vortex briefly the sample every 2 to 3 min until the gel has completely dissolved.
Place a NucleoSpin® Extract II column in a 2mL NucleoSpin collector tube and insert the sample, then centrifuge for 1 min at 11000g.
Throw away the solution contained in the NucleoSpin collector tube and replace the NucleoSpin® Extract II column in the same NucleoSpin collector tube.
Add 600μL of NT3 buffer and centrifuge for 1 min at 11000g.
Throw away the solution contained in the NucleoSpin collector tube and replace the NucleoSpin® Extract II column in the same NucleoSpin collector tube.
Centrifuge for 2 min at 11000g to get rid of all the NT3 buffer.
Make sure that the NucleoSpin® Extract II column is not in contact with the eluted solution during centrifugation (in the NucleoSpin collector tube).
The next reactions are inhibited by the residual ethanol from the NT3 buffer, so it must be entirely removed at this step. In addition to the centrifugation, placing the NucleoSpin® Extract II column in an incubator at 70°C (158°F) for 2-5 min before elution will ensure total retrieval of the ethanol.
Place the NucleoSpin® Extract II column in a fresh 1.5mL tube.
Add 15-50μL of NE elution buffer, then incubate at room temperature for 1 min to enhance the yield of the eluted DNA.
Centrifuge for 1 min at 11000g.
The yield of the large fragments (>5-10 kb) can be enhanced by using pre-heated elution buffer (70°C/158°F): for the elution, add the pre-heated elution buffers and incubate at room temperature for 1 min before collecting the elute by centrifugation.
Back to top
Preparation of electrocompetent Escherichia coli
Material
500µL of E.Coli culture in an LB culture medium
LB culture medium
60mL of glycerol at 10%
Ethanol
Incubator with rotor
Tank of ice
1.5mL tubes
50mL Falcon tubes
300mL tubes for the centrifuge
A refrigerated centrifuge at 4°C (39.2°F)
Method
Prepare a pre-culture of E.Coli in 5mL of LB culture medium (all night long at 37°C/39.2°F with rotation).
Prepare the centrifuge at 4°C (39.2°F).
Place the pre-culture of E.Coli in 450mL of LB culture medium and incubate with agitation for 3 hours at 37°C (98.6°F).
Place 225mL of the solution in 2 centrifuge tubes.
Place both tubes in the ice for 15 minutes.
Centrifuge at 2000g for 10 minutes at 4°C (39.2°F).
Throw away the supernatant and re-suspend delicately the pellet with 200mL of cold sterile water. Firstly add 10 to 20mL of cold sterile water and re-suspend by pipeting, and then add the other 200mL.
Centrifuge at 2000g for 10 minutes at 4°C (39.2°F).
Throw away the supernatant and re-suspend delicately the pellet with 200mL of cold sterile water. Firstly add 10 to 20mL of cold sterile water and re-suspend by pipeting, and then add the other 200mL.
Place the tubes in the ice for 30 minutes.
Centrifuge at 2000g for 15 minutes at 4°C (39.2°F).
Throw away the supernatant and re-suspend delicately the pellet with 25mL of glycerol at 10%.
You may also transfer the solution into a 50mL Falcon tube.
Place the tubes in the ice for 30 minutes.
Centrifuge at 1500g for 15 minutes at 4°C (39.2°F).
Throw away the supernatant and add 500µL of glycerol at 10%.
Re-suspend the cells in a total volume of 1mL.
Aliquote 50µL per tube (tubes already in the ice).
Store at -80°C (-112°F).
Back to top
Electroporation of electrocompetent Escherichia coli
Material
Liquid LB medium
Solid LB medium (LB culture medium + agarose + specific antibiotics)
Frozen electro competent Escherichia coli bacteria (40µL in an eppendorf tube)
DNA
Tank full of ice
Electroporation basin (2mm)
Electroporator
Incubator with agitation incorporated
Method
0. Place the electroporation basin in the ice.
Leave the tube containing the bacteria (about 40µL) at room temperature, so as to unfreeze the medium. Then place it in the ice.
Set the eclectroporator according to the following parameters:
Voltage 2500 V
Capacitance 25 µF
Resistance 200 Ω
Basin 2 mm
Add 1µL of DNA solution in the tube containing the bacteria.
Place the tube in the ice for 1 minute.
Transfer the contents of the tube into the electroporation basin.
Dry the outside of the basin with paper towel. Place it in the specific compartment of the electroporator.
Start the pulse.
Add immediately 1mL of culture medium in the basin.
Beware: a 3 minute delay reduces by 90% the effectiveness of the transformation.
Transfer the contents of the basin into a tube. Incubate the tube at 37°C (98.6°F) with agitation for an hour.
Soak the basins in bleach for a few minutes. Rinse them out thoroughly with tap water, then with bi distilled water and 70% ethanol (no acetone), before letting them dry. Thus they can be re-used.
Place the bacteria on a solid LB medium for cultivation.
Back to top
Preparation of electrocompetent Mycobacterium
Material
500μL of Mycobacterium culture at -80°C (-112°F)
Glycerol 10%
1.5mL tubes
Centrifuge
Method
0. All the mediums and the whole manipulation are done at room temperature.
Transfer the cells from your frozen sample into the centrifuge tubes.
Centrifuge the cells at 3600 rpm for 10 min, then throw away the supernatant.
Wash the cells with ½ a volume (25mL) of sterile 10% glycerol by pipetting slowly until the cell mass has been dissolved.
Wash and precipitate the cells as done in steps 2 and 3; with ¼ vol (12.5mL) of sterile 10% glycerol.
Wash and precipitate the cells as done in steps 2 and 3; with ⅛ vol (6.75mL) of sterile 10% glycerol.
Wash and precipitate the cells as done in steps 2 and 3; with ⅟10 vol (5mL) of sterile 10% glycerol.
Wash and precipitate the cells as done in steps 2 and 3; with ⅟25 vol (2mL) of sterile 10% glycerol.
Aliquot 100µL of cells into the 1.5mL tubes.
Leave to cool quickly in an ice tank.
Store at -80°C (-112°F) or use immediately (the effectiveness of the electro-transfer is improved).
Back to top
Electroporation of electrocompetent Mycobacterium
Material
Liquid Loewenstein medium
Solid Loewenstein medium (congealed Loewenstein culture medium + adapted antibiotics)
Frozen electro competent mycobacterium (40µL in a tube)
DNA
Tank full of ice
Electroporation basins (2mm)
Electroporator
Incubator with agitation incorporated
Method
0. Place the electroporation basin in the ice.
Leave the tube containing the bacteria (about 40µL) at room temperature, so as to unfreeze the medium. Then place it in the ice.
Set the eclectroporator according to the following parameters:
Voltage 2500 V
Capacitance 25 µF
Resistance 1000 Ω
Basin 2 mm
Add 1µL of DNA solution in the tube containing the bacteria.
Place the tube in the ice for 1 minute.
Transfer the contents of the tube into the electroporation basin.
Dry the outside of the basin with paper towel. Place it in the specific compartment of the eclectroporator.
Start the pulse.
Add immediately 1mL of culture medium in the basin.
Beware: a 3 minute delay reduces by 90% the effectiveness of the transformation.
Transfer the contents of the basin into a tube. Incubate the tube at 37°C (98.6°F) with agitation for an hour.
Soak the basins in bleach for a few minutes. Rinse them out thoroughly with tap water, then with bi distilled water and 70% ethanol (no acetone), before letting them dry. Thus they can be re-used.
Place the bacteria on a solid Loewenstein medium for cultivation.
Back to top
Restriction
Material
10µL of DNA (1µL for 0.5g of DNA)
3µL of restriction buffer (1X of the total volume)
1µL of restriction enzyme
3µL of BSA (1X the total volume)
15µL of sterile distilled water
Total volume 30µL
1.5mL tubes
Method
In a sterile eppendorf tube place 3µL of buffer.
Add DNA.
Add 3µL of BSA.
Then add 1µL of restriction enzyme.
Complete with sterile distilled water.
TOTAL VOLUME = 30µL
Set the thermocycler according to the following parameters and place the tube inside:
a. 1-2 hours at 37°C (98.6°F)
b. 20 min at 80°C to inactivate the enzymes
c. leave at 4°C until opening of the thermocycler.
NOTE: if you can leave the tube in the incubator for more than 6 hours for the restriction, leave it longer to obtain a better quality restriction!
Back to top
Linearization
Material
DNA template
1µL of alkaline phosphatase
3µL of alkaline phosphatase buffer
Pure water for a total volume of 30µL
A hotplate
1.5mL tubes
Method
0. Prepare a hot tub at 37°C (98.6°F).
Add the alkaline phosphatase concentration 1X buffer in the DNA template, meaning 3µL of buffer for a final volume of 30µL.
Add 1µL of alkaline phosphatase.
Incubate for 1 hour at 37°C (98.6°F).
Deactivate the alkaline phosphatase by heating for 5 minutes at 65°C (149°F).
Proceed to the ligation just after the linearization.
Back to top
Ligation
Material
3µL of 10X ligation buffer
1µL of T4 DNA Ligase
100 to 1000µg of DNA insert solution
1µL of 0.05g/L DNA vector
Sterile water to obtain à final volume of 30µL
1.5mL tubes
Method
In a 1.5mL tube, place 2µL of ligation buffer.
Add the DNA insert as well as 1µL of vector.
Add 1µL of T4 DNA Ligase.
Complete with sterile water to obtain a final volume of 30µL.
Let the reaction take place for 35 minutes at 22.5°C (72.5°F).
Denaturize the T4 DNA Ligase at 65°C (149°F) for 10 minutes.
Back to top
PCR (Promega Kit)
Material
DNA template
5’ to 3’ Primer
3’ to 5’ Primer
GoTaq Hot Start Green Mix (Promega)
Thermocycler
Method
1. Place the GoTaq Hot Start Green Mix kit at room temperature, vortex then centrifuge it briefly to concentrate the mix at the bottom of the tube.
2. Prepare the following PCR mix:
a. For 25µL:
b. For 50µL :
c. For 100µL :
3. Add the components in the order listed in the table.
Be careful to insert the volume of primers and of DNA template correctly on the edge of the tube so as to visualize all the volumes that have been deposited.
4. Vortex the total volume.
5. « Start-Stop » Centrifuge (2 to 5 seconds).
If the thermocycler does not have a heating top, add 1 to 2 drops of sterile mineral oil.
6. Insert the preparation into the thermocycler, then launch the cycles:
Cycles:
7. Check your PCR products by electrophoresis.
Back to top
Primers
D protein:
To extract the sequences from the full genome of the Lambda phage:
Forward : 5'-ATG-ACG-AGC-AAA-GAA-ACC-TT
Reverse : 5'-AAA-AAA-ATC-CCG-TAA-AAA-AAG-C
To add BioBrick prefix and suffix:
Forward: 5'-GAA-TTC-GCG-GCC-GCT-TCT-AGA-TGA-CGA-GCA-AAG-AAA-CCT-TTA-CCC
Reverse: 5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA-AAA-AAA-ATC-CCG-TAA-AAA-AAG-CCG-C
To add BalI sequence:
Reverse:  5'-AAT-GGC-CAA-AAA-AAA-TCC-CGT-AAA-AAA-AGC
COS sequences:
To extract the sequences from the full genome of the Lambda phage:
Forward : 5'-TGG-CCT-CCA-CGC-ACG-TTG
Reverse : 5'-AGA-GGC-CAA-AAA-GCC-TCG-CTT-TC
To add BioBrick prefix and suffix:
Forward:  5'-GAA-TTC-GCG-GCC-GCT-TCT-AGA-GGA-TAT-GTA-GAT-GAT-AAT-CAT-TAT-C
Reverse:  5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA-AGA-GGC-CAA-AAA-GCC-TCG-CTT-T
CI repressor:
To extract the sequences from the full genome of the Lambda phage:
Forward : 5'-AAT-TCA-ATC-CAT-TTA-CTA-TGT-TAT-GT
Reverse : 5'-ATG-AGC-ACA-AAA-AAG-AAA-CC
To add BioBrick prefix and suffix:
Forward:  5'-GAA-TTC-GCG-GCC-GCT-TCT-AGA-GAA-TTC-AAT-CCA-TTT-ACT-ATG-TTA-TG
Reverse:  5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA-ATG-AGC-ACA-AAA-AAG-AAA-CCA-TTA-A
Adenovirus penton based:
To extract the sequences from the full genome of the Lambda phage:
Forward : 5'-ATG-CGG-CGC-GCG-GCG-ATG
Reverse : 5'-TCA-AAA-AGT-GCG-GCT-CGA-TAG-GAC-G
To add BioBrick prefix and suffix:
Forward:  5'-GAA-TTC-GCG-GCC-GCT-TCT-AGA-TGC-GGC-GCG-CGG-CGA-TGT-A
Reverse:  5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA-TCA-AAA-AGT-GCG-GCT-CGA-TAG-G
To add BalI sequence:
Forward: 5'-AAT-GGC-CAA-TGC-GGC-GCG-CGG-CGA-TG
Reverse: 5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA-TCA-AAA-AGT-GCG-G
BioBrick
To amplify BioBrick prefix and suffix:
Prefix for sequences which begin by ATG
5'-GAA-TTC-GCG-GCC-GCT-TCT-AG
Prefix for sequences which not begin by ATG
5'-GAA-TTC-GCG-GCC-GCT-TCT-AGA-G
Suffix:
5'-CTG-CAG-CGG-CCG-CTA-CTA-GTA
Retrieved from " http://2009.igem.org/Team:SupBiotech-Paris/Material_%26_Method "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
