Team:TEC-Costa Rica/Project/System/Detection
Toggle navigation
Home
Team
Members
Instructors & Investigators
Sponsors
Attributions
Project
System
Modelling & Simulations
Parts
Collaborations
Safety
Judging
Golden Medals
Silver Medals
Bronze Medals
Human Practice
Synbiothon
Integrated Human Practice
Giant Jamboree
Notebook
Laboratory
Protocols
Interlab
System
Detection Module
We found out that conformational changes of the dCas9 during protein activation and action are an useful approach to create a detection system. Even though Cas9 naturally targets only DNA molecules, we found a PAMmer strategy to detect single stranded RNA, ideal for targeting PCA3 Prostate Cancer biomarker. To create a diagnose system we select a dCas9 chassis and analyzed hotspots, defining a two protein insertion strategy.
Advantageous Detection
Our modified dCas9 detection approach has the advantage of being an useful system for detecting any nucleic acid molecule. By creating a new sgRNA the system could detect an specific DNA molecule of interest and we could also target RNA based on the PAMmer strategy, where a small DNA fragments complement the single stranded RNA. Finally, the system is ideal to set a detection output because of the hotspots found in the dCas9's sequence that could accept protein insertions.
Background
1. Cas9 conformational changes
We decided to base our detection module in the CRISPR/Cas9 system due to its versatility to detect different targets. The CRISPR system is based on two molecules, one Cas9 protein with nuclease activity and one sgRNA that activates the protein; this sgRNA is the key component also due to that it complements the target thanks to a 20 nucleotide variable region (Sternberg et al, 2015). Our system is adaptable then to target different nucleic acids just designing new sgRNA but using the same Cas9 protein.
Noticing this useful system we started to study the Cas9 protein by structure and activity. We focused in how the sgRNA activates the Cas9 and found out that the protein has three states depending on the presence of sgRNA and target, as detailed in Figure 1. The Cas9 has a defined conformation in an inactive state and it changes its conformation when the sgRNA is bounded due to interaction between the recognition lobe (REC) and the sgRNA; upon activation the nuclease domain HNH moves to cleave the nucleic acid as a final conformational change mencioned Sternberg et al (2015).
Figure 1. Conformational changes of Cas9 upon sgRNA binding and sgRNA associated to the target.
x
We thought that the HNH domain could be a potential site to engineer the Cas9 protein. The HNH domain is activated by the sgRNA conformational change and moves along to the target, if the sgRNA and the target are complementary the HNH would cleave; upon HNH activity the RuvC nuclease is activated commented Sternberg et al (2015) . Looking to establish a detection system we selected this conformational change as a profitable approach.
2. Adapting dCas9 to RNA
As we needed to detect an RNA biomarker for prostate cancer we next focused on the detection of nucleic acids by the Cas9. The HNH domain of the Cas9 needs to detect a PAM sequence next to the target sequence (but in the opposite strand) for being activated. The PAM sequence is only present in DNA sequences (O’Connell et al, 2014) and in our case, our target molecule is single stranded RNA.
Due to this problem we search for RNA detection mediated by Cas9 and found out that it is possible. There’s an approach to target single stranded RNA using a Streptococcus pyogenes dCas9. To target RNA the dCAs9 needs to recognize the PAM sequence in a DNA strand, a short DNA sequence containing the PAM sequence, called PAMmer, is needed the to target RNA upon hybridization between the PAMmer and the target argued O’Connell et al (2014).
We wanted to design a system that target specific RNA’s of interest, as PCA3 a biomarker for prostate cancer. Considering that the dCas9 needs a PAMmer DNA sequence to target the RNA, we started thinking of our system as a laboratory test in the PAMmer could be add as a reagent.
3. dCas9 insertion hotspots
After selecting the dCas9 as our chassis for the detection and noticing the conformational changes of the protein in the different states; we focused on how to design a synthetic system to generate an output, after targeting the RNA of interest using these conformational changes. Due to the potencial of targeting the genome there’s a lot of research in Cas9 applications. One approach elucidates that the Cas9 protein tolerates protein domain insertions at specific sites of the sequence without affecting the binding nor the activity (Oakes et al, 2016).
Thinking of an interesting approach of modifying the detection system and nor the response part we started to look up points of interest for insertions. Cas9 insertion were validated inserting random PDZ, a 86 amino acid domain, into the DNA sequence of the protein using transposons; finding a lot of hotspots in the sequence but specially in six clusters, including the HNH domain defined Oakes et al (2016). As we saw the HNH domain is a key component in the dCas9 activity, so we define this hotspots as a posible insertion site.
Inserting various domain into the Cas9 protein it’s possible but it has some considerations. Cas9 or dCAs9 could tolerate multiple domain insertions while retaining its activity; however three or more insertions could lead into a considerable decrease of the activity, in the experiment case repressing GFP expression argued Oakes et al (2016). We decided to use a multiple insertion strategy to create the detection chassis as shown in Figure 2. We chose pairs of hotspots based on our structural modelling and finally define two events: version A with Leucine L390 - Glutamic acid E802 and version B with Asparagine N588-Asparagine N888 as hotspots.
Figure 2. Screening hotspots in the dCas9.
x
Design
1. dCas9
We were able to design two dCas9 that allow protein insertions in two sets of hotspots. For testing these spots, we created an A and B version of the protein: A-dCas9 BBa_K1903001 accepts protein insertions in amino acids L390 and E802; these hotspots come close when dCas9 binds to the target molecule, which could trigger a reaction between the proteins inserted in the sites. On the other hand, B-dCas9 BBa_K1903002 allows insertions in amino acids N588 and N888 and these sites come close since the sgRNA binds to the dCas9.
For designing these proteins we used Streptococcus pyogenes’s Cas9 (Uniprot accession Q99ZW2) optimized for E.coli and mutated for being catalytically inactive (mutations D10A and H840A). We added a Double Terminator (BBa_B0015) at the end of each protein’s sequence and then we divided each protein in 4 gBlocks of approximately 1000 bp each.
Since each Cas9’s hotspots where on gBlocks 2 and 3, we planned an assembly strategy in which we could exchange gBlocks 1 and 4 for assembling two complete dCas9 protein with different insertion frames at the studied hotspots. We based in Golden Gate Assembly Method for designing our gBlocks and we used Weber et.al (2011) fusion sites stragety.
Each of our gBlocks has BsaI recognition sites at its start and end; the “start” recognition sites from each gBlock are inverted so that the recognition sites can be lost after the Golden Gate reaction. As its shown in Figure 3, each of the 6 gBlocks has different fusion sites, which where added next to each BsaI recognition site.
In our assembly strategy, by performing a Golden Gate reaction with BsaI enzyme and the gBlocks 1, 2A, 3A and 4, we would obtain our A version dCas9 that includes L 390 and E802 hotspots; and by assembling gBlocks 1, 2B, 3B and 4, we would obtain our B version dCas9, that has hotspots N588 and N888.
Figure 3. dCas9’s gBlock Golden Gate assemble strategy.
x
Because we wanted to insert proteins in each of dCas9’s hotspots, we designed two insertion windows inside each dCas9 protein sequence. A-dCas9 has two insertion windows: one from amino acid 1 to 60 that contains L390 hotspot and other from 248 to 266, which contains E802 site. On the other hand, B-dCas9 has its insertion frames from amino acid 583 to 601, that includes N588 hotspot and from 882 to 908, which includes the hotspot N888.
These insertion frames are flanked by BbsI recognition sites; the “start” recognition sites of each frame are inverted so that the recognition sequences can be lost after this second Golden Gate reaction, leaving a functional dCas9 with two protein insertions. Figures 4 and 5 show A and B version of complete dCas9 with their insertion frames assemble strategy.
Figure 4. Golden Gate assemble strategy for adding protein insertions to A-dCas9.
x
Figure 5. Golden Gate assemble strategy for adding protein insertions to B-dCas9.
x
For this assemble strategy we also used Weber et.al (2011) fusion sites and as its shown below, every frame has different versions of sites, which are part of the Cas9 protein sequence and are going to be kept after the insertions's assembly. We designed two dCas9 version A protein insertions which can be found as BioBricks BBa_K1903020 and BBa_K1903021 and two dCas9 version B insertions: BBa_K1903022 and BBa_K1903023 . Using these BioBricks sequences as a temple, new protein insertions can be designed for assembling version A and B dCas9.
2. Golden Gate Adaptor
We design a Golden Gate Adapter as the Biobrick BBa_K1903003 , enabling iGEMS's standard assembly plasmid to be a Golden Gate assembly vector. This part is composed of the prefix lacking the XbaI however instead there’s a BsaI fusion site follow by the BsaI recognition sequence, a 100 random nucleotides spacer and finally the suffix having also a BsaI fusion site. Digesting this BioBrick with EcoRI and PstI and ligating it to the plasmid, would make any standard assembly vector adaptable to Golden Gate upon the reaction takes place. The BsaI fusion site is inverted and only when the Golden Gate reaction is made the XbaI site will apear having a normal prefix for the part.
3.PAMmer
For the PAMmer design we follow O’Connell et al (2014) findings:
PAMmer must be a short deoxyribonucleotide single stranded sequence.
PAMmer need a 19 nucleotide extension downstream the base paring region of the RNA target.
PAMmer should have an 2 to 8 nucleotide extension upstream the NGG to gain binding specificity, however it shouldn’t be to long to avoid losing binding affinity.
PAM region in the PAMmer must not be the same as in the gene to avoid gene silencing.
4. sgRNA
To design the sgRNA we used a two steps strategy to create the sequence. First we searched the sequence for the sgRNA for Streptococcus pyogenes already modified as a chimeric sgRNA just with the variable base-paring region; we used the sequence detailed in Larson et al. (2013). Next we used Deskgen tool to find possible targets sites in the PCA3 variant 2 gene.
Results
Assembly
First we adapted a pSB1C3 plasmid to Golden Gate using the adapter that we previously designed. We ordered the adapter and digested and ligated it to the plasmid; then we did a transformation and obtained positive results validated by PCR of the miniprep as shown in Figure 6. This miniprep was used as the vector of the dCas9’s Golden Gate reaction.
Figure 6. Golden gate adapter in pSB1C3 validated by PCR and electrophoresis agarose gel.
x
We used the NEB Golden Gate Reaction for multiple fragments (5-20) despite of having a 4 fragment reaction due to negative results with the 1-4 fragment NEB reaction. For this reaction we defined the thermocycler program as the general based on a 37oC digest for 5 minutes followed by a 16oC ligation for 5 minutes, both steps repeated for 30 cycles, and a final inactivation at 55oC for 5 minutes. This strategy ensured efficiency thanks to the fact that upon digestion, BsaI recognition sites will be lost, and was selected due to the fact that we were working with the dCas9 open reading frame.
We performed a transformation with 4 uL of the reaction and obtained 66 white colonies with only 4 red colonies as a result of the dCas9 assembly. Because we didn't have a selection method, the efficiency of the reaction was 6%, with a total of 4 events confirmed by minipreps’s PCR from 66 colonies processed.
As shown in Figure 7, the fragments of both versions of the dCas9, have a size between 4000 and 5000 bp based on the Mass Ruler ladder, which matches the theoretical size of 4525 bp for both version A-dCas9 and B-dCas9, respectively. Based on this result we could argue that the fusion sites TCTA, GCAA, GGGT, AATG, and TACT can be used in a same Golden Gate assembly reaction.
Figure 7. dCas9 assembly validated by PCR and an agarose gel electrophoresis.
x
Under construction
For further work we will assembly the insertions into the dCas9 with a second Golden Gate reaction with BbsI enzyme. Upon assembly we’ll need to characterized the dCas9 with its insertions by sequencing; to finally validated our system with a fluorescent essay mediated by the release of this insertions.
References
Larson, M. H., Gilbert, L. A., Wang, X., Lim, W. A., Weissman, J. S., & Qi, L. S. (2013). CRISPR interference (CRISPRi) for sequence-specific control of gene expression. Nature protocols, 8(11), 2180-2196.
Oakes, B. L., Nadler, D. C., Flamholz, A., Fellmann, C., Staahl, B. T., Doudna, J. A., & Savage, D. F. (2016). Profiling of engineering hotspots identifies an allosteric CRISPR-Cas9 switch. Nature biotechnology, 34(6), 646-651.
O’Connell, M. R., Oakes, B. L., Sternberg, S. H., East-Seletsky, A., Kaplan, M., & Doudna, J. A. (2014). Programmable RNA recognition and cleavage by CRISPR/Cas9. Nature, 516(7530), 263-266.
Sternberg, S. H., LaFrance, B., Kaplan, M., & Doudna, J. A. (2015). Conformational control of DNA target cleavage by CRISPR–Cas9. Nature, 527(7576), 110-113.
Weber, E., Engler, C., Gruetzner, R., Werner, S., & Marillonnet, S. (2011). A modular cloning system for standardized assembly of multigene constructs. PloS one, 6(2), e16765.
Signaling
Response
iGEM TEC Costa Rica
Tecnológico de Costa Rica
Biotechnology Investigation Center
Cartago, Costa Rica
prostaltec@gmail.com
Follow Us
