<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-inverse navbar-fixed-top" style="margin-top:15px;"><DIV class="container"><DIV class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Team">Team </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Team">About Us</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Design">Design </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Design">Project Design</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Patent_Law">Patent Law</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Experiments">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Demonstrate">Demonstrate</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Model">Model </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Model">Our Modelling</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Human_Practices">Communications </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Interlab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Rate_My_Gel">Rate My Gel</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Parts">Our Parts</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Composite_Part">Our Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Basic_Part">Our Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Improve">Part Improvement</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Medals">Medals </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Medals"> Medals</A></LI></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" align="center"><DIV class="content" id="please"><CENTER><H1>Evidence of Protease Knockout in <I>Bacillus subtilis</I> WB800</H1></CENTER><H6><B>Figure 1:</B> Wild type (WT) B.subtilis 168 and WB800 B.subtilis grown on an LB milk plate. 
 WB800 has 8 proteases knocked out of it and this test was used to determine whether in fact this was the correct strain with knocked out proteases. With 8 proteases removed,  we did not expect to see clearing around the colony as a result of the proteolytic breakdown of the milk in the plate as you see in the 2 WT168 colonies on the plate. This result was evident on the plate as there was no clear zone around the WB800 strains, whereas substantial clearing is seen around the WT168 colonies.</H6></DIV></DIV><DIV class="row" align="center"><DIV class="content" id="please"><CENTER><H1>Evidence of Chromosome Integration in <I>Bacillus subtilis</I> WB800</H1></CENTER><H6><B>Figure 2:</B> pUS258::YncM insulin transformed into <I>B.subtilis</I> WB800 patched on LB Agar + 1% starch + Spectinomycin 100. </H6><H4>pUS258 is an integrative plasmid, meaning when transformation occurs, the insert will integrate into the amylase genes of the chromosome of <I>Bacillus</I>, knocking them out. Therefore, by patching these transformants on starch plates, cells which have had integration of the YncM insulin gene into their amylase genes will be unable to digest starch on the plate. When flooded with iodine, the colonies in which integration have occurred are identified as showing no visible clearing. The colonies without clearing can therefore can be expected to contain the integrated plasmid, and may be induced to produce insulin. </H4></DIV></DIV><DIV class="row" align="center"><DIV class="content" id="please"><CENTER><H1>Confirmation of Inserts in pSB1C3</H1></CENTER><H6>Figure 3: Double digest of constructs in pSB1C3 using EcoRI and PstI. Products were run on a 1.5% agarose gel in 0.5X TBE at 80V for 2 hours and post stained with GelGreen (Bio-rad) according to manufacturers protocol.</H6><H4>
Our constructs were ligated into the submission vector, pSB1C3 to submit as composite parts to the iGEM registry. PCR was performed on the plasmid preps across the insert site using high fidelity polymerase and those PCR products were sequenced with the Australian Genomic Research Facility to ensure we would be submitting the correct sequence. The sequencing data showed some disparity in the YncM insulin construct compared to the G-block that was ligated. This is reflected in the gel in figure 3 that shows multiple digested bands. This could potentially be due to multiple partial inserts but we cannot confirm this. Based on this data, we made the decision not to submit this part to the registry and unfortunately, despite our attempts, we couldn’t redo the process due to a limited supply of our construct and time. On the other hand, our other constructs that can be seen in lanes 2-6 show clean single bands that correspond to their respective construct sizes (Cytoplasmic Winsulin: 306bp, Ecotin Winsulin: 867bp, YncM Winsulin: 522bp, Cytoplasmic insulin: 392, Ecotin insulin: 917bp, YncM insulin: 583). These were additionally confirmed with accurate sequencing data and so were able to be submitted.</H4></DIV></DIV><DIV class="row" align="center"><DIV class="content" id="please"><CENTER><H1>Evidence for ELISA Results</H1></CENTER><H6>Figure 4: ELISA confirms expression of constructs, and the correct folding of Insulin and Winsulin constructs. ELISA was performed on cell lysates for all constructs and on the cell culture media of YncM Winsulin from B.subtilis, according to the manufacturer's protocol. </H6><H4> This showed the presence of expressed insulin or Winsulin in cells expressing Cytoplasmic expressed insulin, Cytoplasmic Winsulin, Ecotin expressed insulin and YncM Winsulin, as well as proving their ability to bind anti-insulin antibodies. 5µL of cell lysates were tested at multiple dilutions, shown here are the 1:1 dilutions. The inclusion of a whole-cell lysate control for YncM Winsulin proves the YncM secretion tag was effectively causing Bacillus secretion of Winsulin into the surrounding media, as it was not found in the cell lysate fraction.

We performed an ELISA assay using capture antibodies specific to insulin (Figure 4). The purpose of performing this assay was:<DIV style="width:70%">
To identify if our insulin constructs had formed folded proteins of a shape similar enough to native human insulin that they would be bound by an anti-insulin antibody.
Determine if we produced these constructs in our cells.
Identify if the translocation tags used on some constructs (namely YncM Winsulin) transported the protein where we had predicted.</DIV>
From this assay we were able to confirm that our insulin analogues bound insulin antibodies. The parts tested, Cytoplasmic insulin, Cytoplasmic Winsulin, Ecotin insulin and YncM Winsulin, therefore all have a structure very similar to that of native insulin. The fact that we detected them in the ELISA assay also confirmed expression of our constructs.
We also used this assay to identify the location of proteins with tags inducing translocation to different parts of the cell. Whilst we were unable to test the periplasmic fraction of Ecotin insulin due to time constraints, we did test our YncM Winsulin construct. YncM is an N-terminal tag which, when fused to a protein of interest and expressed in Bacillus subtilis, directs its secretion out of the cell and into the surrounding media. We proved that YncM does induce secretion by showing that whilst YncM Winsulin was present in the surrounding media, it was completely absent in the cell lysate fractions, as expected.
</H4></DIV></DIV><DIV class="row" align="center"><DIV class="content" id="please"><CENTER><H1>Cell Culture Assay Results</H1></CENTER><H6>Figure 5: Expressed recombinant insulins stimulate glycogen synthesis. 4A and 4B: Comparison of mean glycogen synthesis rate in a glucose uptake assay using human HepG2 (4A) and murine AML12 cell lines (4B). 4C: Comparison of mean rate of glucose oxidation in HepG2 cell culture from glucose uptake assay. Conditions indicated on the graphs are: Basal (no insulin added); Ecotin Proinsulin (from whole cell lysate with trypsin treatment); Cytoplasmic Proinsulin (from whole cell lysate with trypsin treatment); YncM Winsulin (cell culture media only). Refer to protocols page for experimental details. Error bars represent SEM, horizontal bars indicate statistical significance (* = p&lt;0.05, ** = p&lt;0.0001) calculated with GraphPad Prism using unpaired 1-tailed T-test, n=3. </H6><H4>
While the ELISA provided evidence for whether or not our cells were producing insulin, and gave an indication that the protein fold was correct, the glucose uptake assay was essential for determining the functional capacity of our constructs. It is testing if our expressed human insulin and expressed Winsulin can stimulate an increase in the absorption of radioactive glucose by binding to the insulin receptor in vitro using human HepG2 and mice AML12 liver cells. The change is detected by measuring glycogen synthesis and glucose oxidation and the protocols for the experiment are available on our protocols page. We only had the materials and time available to use the samples that gave positive readings in the ELISA and were also restricted to using those that had high enough concentrations to fit the protocol, however we produced excellent preliminary data suggesting bioactivity of multiple insulin and Winsulin constructs.</H4><H2>Ecotin Insulin</H2><H4>Because we were simply trying to show functionality, we used the fraction that would have overall the most Ecotin Insulin in it according to the ELISA experiment, so we used whole cell lysate in this assay that had been treated with trypsin protease to cleave insulin to the active form of human insulin. The results indicate that both the human (Figure 4) and murine (Figure 5) hepatocytes significantly increased their glycogen synthesis upon treatment with Ecotin Insulin, strongly indicating that there was successful expression and folding of the insulin to its correctly folded and functional form.</H4><H2>Cytoplasmic Insulin</H2><H4>As with Ecotin Insulin, we used whole cell lysate to test Cytoplasmic Insulin function, however this was because of its intended expression in the cytoplasm. In the human hepatocyte cell line (HepG2), Cytoplasmic insulin significantly increased glycogen synthesis (Figure 4), and in the murine cell line (Figure 5), the result was highly significant (p&lt;0.0001) compared to basal glycogen synthesis. Additionally, Cytoplasmic insulin showed a promising increase in glucose oxidation although the result was statistically insignificant. This data together is a very strong indication that this construct was successfully expressed and processed.</H4><H2>YncM Winsulin</H2><H4>Because the ELISA showed successful secretion of YncM Winsulin into the cell culture media, we used that fraction in the glucose uptake assay. Unfortunately we only had a very small volume remaining and were only able to test on HepG2, and due to time constraints were unable to produce more for testing. However, it did appear to show an increase in glycogen synthesis, although not statistically significant, as illustrated in Figure 4.</H4><H4>Overall, these results are an incredibly exciting outcome. We have produced strong evidence that proinsulin and ecotin proinsulin isolated from the cytoplasm are functional following trypsin cleavage to active insulin. More importantly, we have a promising indication that our novel single chain insulin analogue, Winsulin, is functional. As  we were limited by both supplies and time, we recognise that the data is of limited reliability, however we have certainly produced some promising preliminary results that warrant pursuing further investigation.</H4></DIV></DIV><DIV class="row" align="center"><DIV class="content" id="please"><CENTER><H1>Future Directions</H1></CENTER><H2>Improve reliability of preliminary data</H2><H4>We were limited in time and supplies to test all of our constructs rigorously and although we have good preliminary data, it would be insufficient to justify Winsulin for clinical trials at this point. Re-doing the expression process and assays with more samples would gather the data necessary to put it in the pipeline for large scale experimentation and eventually clinical trials.</H4><H2>Optimise recombinant Insulin/Winsulin Yield</H2><H4>Increasing yield would be the most important aspect of improving our product to make large scale production viable and ultimately reduce the cost per dose. Here we have outlined a few ways in which we believe are realistic methods to approach this:
Part of increasing yield would be to increase the amount of protein moved out of the cell in our Bacillus secretion method and into the periplasm for E. coli expression. This could potentially be achieved by increasing the efficiency of the respective targeting tags, YncM and Ecotin using random or targeted mutagenesis.While pET15-b and pUS258 are designed for high expression of recombinant protein, it may be the case that other vector/expression systems are better suited to expressing our particular constructs, and it would be naive to proceed to large scale production without trialling other options. 
Another option that some of the other big insulin producers have adopted is to modify their cell lines. While we already took the first steps towards this by using the existing protease knockout strain WB800, further optimisations could be made using CRISPR gene editing. We could experimentally target a number of genes including folding chaperones and proteins related to the secretory pathway. Another viable method would be to use whole genome CRISPR screening as a more random method of identifying knockouts that may enhance expression where we wouldn’t think of targeting.</H4><H2>Characterise Winsulin</H2><H4>Because the Winsulin analogue is completely novel, it will need in depth characterisation, particularly if it continues to show functional capacity. Proteomic analysis to provide information about structure, charge, binding mechanism and stability are just some of the properties that are essential if it was to be considered as a therapeutic. </H4><H2>Additional functional analysis</H2><H4>While the glucose uptake assay using human hepatocytes is an excellent in vitro experiment to determine functionality of our recombinant protein, it simply won’t be sufficient evidence to prove that complies with the overall physiology of humans. We will need to perform a number of in vivo experiments using a diabetic mouse model to show that it can be viable as a hormone replacement therapy that won’t elicit an immune response.</H4><H2>Scale-up Production</H2><H4>Once the expression efficiency has been optimised and the recombinant proteins have been well characterised, we will need to consider methods to scale up production to produce enough insulin to compete in the market. Our vision involves the use of large vats with bacteria secreting our insulin into the media, and figuring out a way to efficiently separate the protein directly giving a somewhat continuous production, rather than emptying the vats and starting from scratch.</H4></DIV></DIV><DIV class="row" align="center"><DIV class="content" id="please"><CENTER><H1>Considerations for replication of experiments</H1></CENTER><H2>Alternate vector system</H2><H4>pUS258 was a sufficient vector to use in our preliminary experiments to determine if we could express proinsulin and Winsulin in B. subtilis WB800. However, as an integrative plasmid with a limited multiple cloning site it added a number of steps to the process. Additionally, because it integrates into the chromosome, there is only a single copy of the recombinant gene in each cell which means less potential protein product produced. On our project design page, we outline an additional solution to this problem in the development of a novel replicative vector pUS270. Following construction and testing, we would opt for using this plasmid as an alternative for the reasons outlined on Project <A href="https://2017.igem.org/Team:Sydney_Australia/Design">Design.</A></H4><H2>Modify Purification Protocol</H2><H4>We encountered some trouble while trying to purify our proteins using the His column. Our SDS-PAGE gels were showing significant protein carry over that may be attributed to insufficient washing or possibly a large number of proteins binding to the column. To improve our purity, we could add in a dialysis step to remove larger proteins leaving the relatively smaller insulin behind. It also appeared as though we may have lost some insulin during purification. This may be attributed to insulins limited solubility in water. One option may be to elute the protein off the His column in HCl with Tris-HCL sitting in the collection tube to immediately neutralise the acidity and minimise damage to the protein with dialysis to dilute the HCl. Realistically, there are a number of small modification like this that could be made to optimise purification that we would repeat the experiment with. </H4><H2>Assays</H2><H4>Due to the complications with purification, we were unable to use purified Proinsulin/Winsulin in either the ELISA or glucose uptake assay. We also didn’t have the opportunity to use test all of our expressed constructs in the glucose uptake assay either because of the low concentrations that resulted in from using whole cell lysates in the ELISA. In repeating the ELISA, we would ideally use a higher concentration of purified samples from different extractions of the appropriate cellular compartments as a comparison like originally planned, rather than the un-purified samples. This would provide a more accurate representation of the culture yield. In the glucose uptake assay we would use higher concentrations of a purified sample rather than just the unpurified extracts to prevent interference of other cellular proteins to give a better assessment of function.</H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>