<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sydney_Australia_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sydney Australia/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar navbar-inverse navbar-fixed-top" style="margin-top:15px;"><DIV class="container"><DIV class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Team">Team </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Team">About Us</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Design">Design </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Design">Project Design</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/HP/Gold_Integrated">Integrated Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Patent_Law">Patent Law</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Description">Project </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Experiments">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Demonstrate">Demonstrate</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Model">Model </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Model">Our Modelling</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Human_Practices">Communications </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/HP/Silver">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Interlab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Rate_My_Gel">Rate My Gel</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Parts">Parts </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Parts">Our Parts</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Composite_Part">Our Composite Parts</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Basic_Part">Our Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Improve">Part Improvement</A></LI><LI class="dropdown"><A class="dropdown-toggle" href="https://2017.igem.org/Team:Sydney_Australia/Medals">Medals </A></LI><LI><A href="https://2017.igem.org/Team:Sydney_Australia/Medals"> Medals</A></LI></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row" align="center"><DIV class="para_container"><H2>Our Key Goals</H2><H4>The aim of the USYD iGEM 2017 team was to address the problem of insulin inaccessibility. The design of our insulin, and its means of expression, needed to look at five key characteristics:</H4></DIV></DIV><DIV class="row" id="boxes"><DIV class="box_container"><DIV class="box1"><H3>Stability</H3><H4>For our project to work effectively, we must have a supply chain that’s not a cold chain, so that costs can be reduced. This will ultimately mean that the cost of these cold chains will not be passed onto the consumer. To achieve this, we hope to design an insulin that will not lose efficacy after being exposed to room temperature for long periods of time.</H4></DIV><DIV class="box2"><H3>Single Chained</H3><H4>As a result of the difficult purification methods, Single Chain Insulins, or SCIs for short, have been developed with a small, C-peptide chain linker. This linker connects the A and B chains in such a way that the di-sulfide bonds form more favorably. We aim to develop our own single chain insulin to compare it’s simplicity.
</H4></DIV><DIV class="box3"><H3>Ease of Purification and Affordaility</H3><H4>We must also consider the impact of a difficult, costly manufacturing process on small scale manufacturing companies. This impact is too great to impose on this grass-roots organisations, so we have pursued to find a cheap, simple purification method which is able to produce high yields from a recombinant system.  
</H4></DIV><DIV class="box4"><H3>Intellectual Property Issues</H3><H4>As a result of the way drugs are currently developed, all new inventions for therapies are protected by Intellectual Property Law through patents. These patents surrounding all currently prescribed and newly invented insulins has inspired our team to pursue a completely open source project. 
</H4></DIV><DIV class="box5"><H3>Safety and Efficacy</H3><H4>Our insulin products must be of certifiable medical grade such that it can be approved for human use after stage IV clinical trials, or biosimilar clinical trials. Furthermore, it must also be at least as effective as the other insulins on the market. 
</H4></DIV></DIV></DIV><DIV class="row" id="analogue-intro" align="center"><DIV class="para_container"><H2>Our Insulin Analogue</H2><CENTER><H10>Proinsulin</H10></CENTER><H4>We began working on expressing proinsulin as it is a reliable, unpatented, and well characterised therapeutic. Proinsulin itself is inactive, and to become active it requires additional processing to form insulin. This processing is performed by the protease trypsin, which cleaves the C chain from proinsulin. After trypsin activity, the A and B chains are left over to form active insulin. Proinsulin processing is demonstrated in Figure 1. with the cleavage of the C chain.</H4><DIV><H7><CENTER> Figure 1. Cleavage of proinsulin results in active insulin</CENTER></H7></DIV><H4>Due to the additional processing steps required to produce purified active insulin from proinsulin, increasing the cost of insulin synthesis, we looked for way to diminish this cost by reducing the steps required to obtain active insulin.</H4><CENTER><H10>Winsulin</H10></CENTER><H4>Our team decided to design a single-chain insulin, as single-chain insulins have been demonstrated to have <I>higher stability</I> and <I>activity</I> than human proinsulin. Stability in particular was important to our design due to the need to synthesise an insulin that could be transported over long distances without requiring cold temperatures. Crucially, single-chain insulins do <I>not</I> need to be cleaved to be active, requiring less processing than proinsulin.</H4><H7><CENTER>Figure 2. Sequence comparison of proinsulin, active insulin and Winsulin.</CENTER></H7><H4> We based the linker peptide of Winsulin (shown above) on sequences and principles tested in Rajpal et al., 2009. This paper found that peptides with a length of 5 to 12 amino acids had the highest receptor binding activity. Importantly, the composition of the sequence is identified as integral in determining whether the single chain construct has activity.</H4><H4>With the above considerations in mind, we designed the linker peptide sequence for Winsulin with the following characteristics:</H4><H4>A further consideration was in ensuring that this single chain construct woudl not infringe any potential <A href="https://2017.igem.org/Team:Sydney_Australia/Patent_Law">patents</A>. With this in mind - we designed our construct to also have 12 amino acids.</H4><H4>Additionally, the C-terminal residue of proinsulin is an asparagine. In Winsulin we substituted this for a glycine to increase the pI of Winsulin to approximately 7.8. This pI is somewhat high, but it leads to the aggregation of Winsulin monomers to form hexamers. This substitution is present in many long-acting insulins currently on the market. Yet, <I>interestingly</I>, our <A href="https://2017.igem.org/Team:Sydney_Australia/Model">modelling</A> indicates our Winsulin is still a rapid-acting insulin, and dissociates from hexamers more readily than human insulin.</H4></DIV></DIV><DIV class="row" id="constructs-intro" align="center"><DIV class="para_container"><H2>Our Constructs</H2><H4>We designed our expression constructs in order to test winsulin and proinsulin with multiple expression systems. Click on each element of the construct to learn more about <B>why</B> we chose it:</H4></DIV></DIV><DIV class="row" id="constructs" align="center"><DIV class="col-xs-8"><DIV class="construct_container"><DIV class="construct_1_caption"><H5> Winsulin secreted by B. subtilis </H5></DIV><DIV class="construct_1"><DIV class="BB-prefix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;"> BB prefix</P></DIV><DIV class="rbs element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">RBS</P></DIV><DIV class="yncm element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">YNCM Tag</P></DIV><DIV class="his element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">His Tag</P></DIV><DIV class="tev element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">TEV</P></DIV><DIV class="winsulin element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">Winsulin</P></DIV><DIV class="BB-suffix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">BB suffix</P></DIV></DIV><DIV class="construct_2_caption"><H5> Proinsulin secreted by B. subtilis </H5></DIV><DIV class="construct_2"><DIV class="BB-prefix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;"> BB prefix</P></DIV><DIV class="rbs element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">RBS</P></DIV><DIV class="yncm element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">YNCM Tag</P></DIV><DIV class="his element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">His Tag</P></DIV><DIV class="arg element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">R</P></DIV><DIV class="proinsulin element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">Proinsulin</P></DIV><DIV class="BB-suffix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">BB suffix</P></DIV></DIV><DIV class="construct_3_caption"><H5> Winsulin targeted to the periplasm of E. coli </H5></DIV><DIV class="construct_3"><DIV class="BB-prefix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;"> BB prefix</P></DIV><DIV class="rbs element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">RBS</P></DIV><DIV class="ecotin element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">Ecotin Tag</P></DIV><DIV class="his element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">His Tag</P></DIV><DIV class="tev element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">TEV</P></DIV><DIV class="winsulin element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">Winsulin</P></DIV><DIV class="BB-suffix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">BB suffix</P></DIV></DIV><DIV class="construct_4_caption"><H5> Proinsulin targeted to the periplasm of E. coli </H5></DIV><DIV class="construct_4"><DIV class="BB-prefix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;"> BB prefix</P></DIV><DIV class="rbs element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">RBS</P></DIV><DIV class="ecotin element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">Ecotin Tag</P></DIV><DIV class="his element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">His Tag</P></DIV><DIV class="arg element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">R</P></DIV><DIV class="proinsulin element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">Proinsulin</P></DIV><DIV class="BB-suffix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">BB suffix</P></DIV></DIV><DIV class="construct_5_caption"><H5> Winsulin targeted to the cytoplasm of E. coli </H5></DIV><DIV class="construct_5"><DIV class="BB-prefix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;"> BB prefix</P></DIV><DIV class="rbs element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">RBS</P></DIV><DIV class="his element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">His Tag</P></DIV><DIV class="tev element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">TEV</P></DIV><DIV class="winsulin element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">Winsulin</P></DIV><DIV class="BB-suffix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">BB suffix</P></DIV></DIV><DIV class="construct_6_caption"><H5> Proinsulin targeted to the cytoplasm of E. coli </H5></DIV><DIV class="construct_6"><DIV class="BB-prefix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;"> BB prefix</P></DIV><DIV class="rbs element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">RBS</P></DIV><DIV class="his element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">His Tag</P></DIV><DIV class="arg element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">R</P></DIV><DIV class="proinsulin element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">Proinsulin</P></DIV><DIV class="BB-suffix element"><P class="element_text" style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">BB suffix</P></DIV></DIV></DIV></DIV><DIV class="col-xs-4"><DIV class="element_info_box_container"><DIV class="info_box BBprefix_info"><H6>iGEM BioBrick Prefix</H6><H4>Contains the restriction sites that are necessary for BioBrick compatibility including EcoRI, NotI &amp; XbaI.</H4></DIV><DIV class="info_box RBS_info"><H6>E. coli Extended Ribosome Binding Site</H6><H4>A derivative of the RBS found in gene 10 of the T7 bacteriophage, this 23 base pair sequence rich in A’s &amp; T’s enhances ribosome binding to boost expression.
</H4></DIV><DIV class="info_box YNCM_info"><H6>YncM Tag</H6><H4>The YNCM tag is a 12 amino acid sequence whose presence on the N-terminus of the protein targets it for secretion out of the cell into the surrounding media via the Sec pathway in Bacillus subtilis. YNCM was chosen because it was recently shown to be massively successful in targeting recombinant protein for secretion compared to a library of other signal peptides. Additionally, this was shown in B. subtilis strain WB600, which is the bacteria that our WB800 strain was derived from. So we expect that it should give us similar success in secretion of our constructs. (Guan et. al. 2016)</H4></DIV><DIV class="info_box HIS_info"><H6>His Tag</H6><H4>We have included a tag comprised of 6 sequential histidines that form a vital aspect of our purification technique using affinity chromatography. Histidine’s high attraction to metal ions will cause the entire protein, insulin and all, to bind to a nickel column and separate it from the other proteins of the cell.</H4></DIV><DIV class="info_box TEV_info"><H6>TEV Protease Cleavage Site</H6><H4>TEV is a sequence-specific cysteine protease derived from Tobacco Etch Virus. Because of its high specificity, it is commonly used for deliberate protein cleavage. In our project, we will use it to exclusively detach Winsulin from the nickel column, leaving the his tag and Ecotin/YNCM tags behind. This should provide us with a pure elution of Winsulin.</H4></DIV><DIV class="info_box ARG_info"><H6>“R” Arginine Cleavage Site</H6><H4>Arginine acts as a recognition site for Trypsin Protease which we will use to specifically remove Proinsulin from the his tag and YNCM/Ecotin tag in a similar way to TEV. We have chosen to use Trypsin in these constructs because it allows us to further simplify the processing of proinsulin. Trypsin naturally cleaves the C-peptide from proinsulin which, following disulfide bond formation, leaves the active form of insulin. This is the way it works in our body, so we are confident that it will work here too. </H4></DIV><DIV class="info_box BBsuffix_info"><H6>iGEM BioBrick Suffix</H6><H4>Contains the restriction sites that are necessary for BioBrick compatibility including SpeI, NotI &amp; PstI. We have also added an additional BamHI site at the terminus of our E. coli expressed constructs for ligation into pET-15b.</H4></DIV><DIV class="info_box ecotin_info"><H6>Ecotin Tag</H6><H4>Ecotin acts as a signal sequence to target the translated protein to the periplasm of the cell. There are a number of advantages that make it a good choice over other tags.
</H4><UL><LI style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">Relatively low metabolic burden due to its small size
	</LI><LI style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">No interaction with other proteins within the periplasm</LI><LI style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">Is native in E. coli and contains a disulfide bond meaning it undergoes through an oxidative compartment that may assist in the formation of the disulfides in Proinsulin and Winsulin.</LI><LI style="font-family:'Quicksand';font-size:20px;color:#3e3f3f;text-align:center;">It has already been shown to successfully target proinsulin to the periplasm (Malik et. al. 2007)
	</LI></UL></DIV><DIV class="info_box proinsulin_info"><H6>Proinsulin</H6><H4>See our section above!</H4></DIV><DIV class="info_box winsulin_info"><H6>Winsulin</H6><H4>See our section above!</H4></DIV></DIV></DIV></DIV><DIV class="row" id="vector-intro" align="center"><DIV class="para_container"><H2>Our Vectors</H2></DIV></DIV><DIV class="row" id="vector_box" align="center"><DIV class="col-xs-10"><SECTION><LABEL for="tab-one1"><H3 class="tabheading">E. Coli</H3></LABEL><DIV class="bish"><H4>We have chosen to use the pET-15b expression system paired with BL21 strain E. coli to express our periplasmic and cytoplasmic constructs. It is a well characterised vector that uses the T7 induction system to prevent expression without the presence of IPTG, providing a convenient level of control over production.</H4><DIV class="row"><H7>Figure 3. pET-15b was used for cytoplasmic and periplasmic expression of proinsulin and winsulin.</H7></DIV></DIV><LABEL for="tab-two1"><H3 class="tabheading">B. Subtilis</H3></LABEL><DIV class="bish"><H4>
pUS258 is an integrative expression vector which undergoes homologous recombination to insert into the amyE gene of the Bacillus subtilis chromosome. It’s a relatively simple plasmid using the same IPTG induction system that we use across all of our expression systems. 
</H4><DIV class="row" align="center"><FIGCAPTION>Figure 4. pus258 was used for cytoplasmic and periplasmic expression of proinsulin and winsulin.</FIGCAPTION></DIV></DIV></SECTION></DIV></DIV><DIV class="row" id="expression-systems-intro" align="center"><DIV class="para_container"><H2> Our Expression Systems</H2><H4> In order to maximise the yield of our insulin, while also reducing the post-expression methods currently undertaken by manufacturers, we tested both of these constructs in three different expression systems. Two of these expression systems were in BL21 E. coli, and the other was in Bacillus subtilis.</H4></DIV></DIV><DIV class="row" id="expression_box" align="center"><DIV class="col-xs-10"><SECTION><LABEL for="tab-zero" class="exp_tab"><H3 class="tabheading">E. Coli- Cytoplasm</H3></LABEL><DIV class="bish"><DIV class="row"><H3> Cytoplasmic Expression</H3><H4>Cytoplasmic expression is our recombinant protein production in its raw and simple form. With no tags attached to the fusion protein, both Winsulin and Proinsulin will accumulate within the cytoplasm to form aggregated inclusion bodies. Extraction will involve complete lysis of the cell and purification will require separating them from DNA,  membranes and other proteins.

The reason to express in the cytoplasm is mainly to compare our other expression systems to where we will be able to see whether they are a viable means of production.

We are using the E. Coli shuffle strain as alternative to BL21 because it is associated with a higher capacity to fold recombinant protein within the cytoplasm This should promote the formation of disulfide bonds and reduce the formation of inclusion bodies
</H4></DIV><DIV class="row" align="center"><FIGCAPTION>Figure 5. Details of E. Coli cytoplasmic expression of pro/winsulin</FIGCAPTION></DIV></DIV><LABEL for="tab-one" class="exp_tab"><H3 class="tabheading">E. Coli- Periplasm</H3></LABEL><DIV class="bish"><DIV class="row"><H3> Periplasmic Expression</H3><H4>Potentially the biggest hurdle to overcome in efficiently producing recombinant insulin is having the three disulfide bonds in proinsulin and active insulin form correctly. This is because they require an oxidative environment that isn’t usually found in the cytoplasm of common protein factories like E. coli where proteins are naturally expressed.

Fortunately, the periplasmic compartment of gram negative bacteria, including E. coli is more oxidative than the cytoplasm and will therefore improve disulfide bond formation. So we are using the Ecotin tag to target proinsulin and Winsulin to the periplasm in the hope that they will fold correctly and form the correct disulfides.

This method also has the additional advantage that the periplasmic fraction can be extracted without lysing the entire cell, separating our insulin from cells DNA and the rest of the cytoplasmic “junk”.</H4></DIV><DIV class="row" align="center"><FIGCAPTION>Figure 6. Details of E. Coli periplasmic expression of pro/winsulin</FIGCAPTION></DIV></DIV><LABEL for="tab-two" class="exp_tab"><H3 class="tabheading">B. Subtilis</H3></LABEL><DIV class="bish"><H3> Secretory Expression</H3><H4> Current methods of insulin production grow bacteria in huge vats. But bacteria like E. coli that only express the protein in the cytoplasm need to be lysed to extract it. Which means they need to drain hundreds of thousands of liters of culture to separate the cells from the media before lysing the cells and then purifying insulin from the DNA and other cellular proteins. Although we will be testing cytoplasmic E. coli expression similar to these methods, we wanted to come up with a way to efficiently separate our insulin from the cells and decided the best way would be have it secreted directly into the media. 
<CENTER>This is where Bacillus subtilis comes in!</CENTER>
Bacillus has been a common tool for recombinant protein production for years and has been proven again and again that they are highly efficient secretion factories. The hope is that having the insulin separated from the cells in the media would mean that scaled up production could use a system where the media is constantly cycled out and the cells can continuously produce our insulin. As can be seen in Figure 7, expression using B. Subtilis can be very simple contributing to our goal to create an easily purifiable product.
 </H4><FIGCAPTION>Figure 7. Details of expression of pro/winsulin in B. Subtilis</FIGCAPTION></DIV></SECTION></DIV></DIV><DIV class="row" id="purification-intro" align="center"><DIV class="para_container"><H2> Our Purification Process</H2><H4> Our insulins were designed with a 6x His sequence at the N-terminus of the insulin. This sequence, comprised of 6 histidine residues, binds to nickel. When cell lysates or media containing insulin are added to nickel beads, all of the cellular components should wash off from the beads. This separates them from insulin which remains bound to the nickel beads. As such, the 6x His tag sequence's affinity to nickel can be used to purify insulin from cells or media.
 We considered two methods for the purification of our insulins: nickel columns, and nickel agarose resin. The nickel column method is simpler, as insulin can be purified in a single step, yet it is more expensive. On the other hand, the nickel agarose resin method involves more steps but is more cost-effective.
 An additional factor in the purification of our insulins is the need to remove the 6x His sequence and the expression tags from the insulin. We designed our Winsulin constructs so that both could be cleaved off in a single step by TEV protease. Our Proinsulin constructs take this further - in a single step using the protease trypsin, the His-tag, expression tag and C chain are removed, simultaneously removing the tags and allowing the proinsulin to fold into active insulin. 
</H4></DIV></DIV><DIV class="row" id="purification_box_row" align="center"><DIV class="purification_box"><FIGURE><FIGCAPTION>Figure 8. Schematic demonstrating the purification methods</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row" align="center"><DIV class="para_container"><H4>In the nickel column method, the protease step also releases the insulin from the nickel column for collection. However, it also reduces the amount of insulin released from the nickel column. Taking this into consideration, along with the increased expense of the nickel column method, our team decided to use the nickel agarose resin method for insulin purification.</H4></DIV></DIV><DIV class="row" id="Assays Intro-intro" align="center"><DIV class="para_container"><H2> Assays</H2><CENTER><H10>ELISA Assay</H10></CENTER><H4> As an initial test to determine protein folding and accurately quantify expression yield, we will be using an ELISA. This test uses antibodies that bind to the folded structure of insulin with extremely high specificity. In this regard, it gives an initial indication as to whether the protein is in a conformation that could potentially bind to the insulin receptor. Additionally, the antibodies are tagged with fluorescent protein, so the relative fluorescence will correlate to the amount of insulin in the sample, also with great accuracy.<FIGURE><FIGCAPTION>Figure 9. Schematic for ELISA insulin assay</FIGCAPTION></FIGURE><CENTER><H10>Glucose Uptake Assay</H10></CENTER>Following the ELISA, we will test the function of our expressed recombinant Winsulin, and Proinsulin (with trypsin removal of the C-peptide) using an in vitro glucose uptake assay. Here, we measure the changes in glycogen synthesis and glucose oxidation in response to activation of the insulin receptor in cell cultures. C14 radioalabelled glucose will be taken up into the cell at a faster rate upon stimulation and result in incorporation of the carbon into glycogen and CO2, which is measured by a scintillation counter. This assay is a great alternative to an in vivo assay that uses involves assessing the convulsions of mice in response to injections of increasing concentrations of insulin.</H4><FIGURE><FIGCAPTION>Figure 10. Schematic for glucose uptake assays for insulin assay</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row" id="additional-intro" align="center"><DIV class="para_container"><H2> Additional Design Considerations</H2><H3> WB800 Bacillus Strain</H3><H4>
One of the challenges that we found in the literature of producing recombinant insulin was its susceptibility to proteolytic decay. So to continue with the theme of maximising expression, it would make sense to reduce the effect of proteolytic decay. 

Our idea: knockout proteases from an existing strain such as B. subtilis 168. 

Our actual solution: find one that already exists. 

And hence we discovered B. subtilis WB800. It is one of the more recent of a long line of protease deficient strains of Bacillus subtilis that have been optimised for recombinant protein secretion (Figure 11).

WB800 specifically has 8 proteases knocked that include both intracellular and membrane bound. The strain was kindly provided to us for experimental use by Professor Sui-Lam Wong of the University of Calgary, Canada.
</H4><FIGURE><FIGCAPTION>Figure 11. WB800 protease knockout strain</FIGCAPTION></FIGURE></DIV></DIV><DIV class="row" id="pus270" align="center"><DIV class="para_container"><H3> pUS270 </H3><H4>Although pUS258 appeared to work quite well as an expression vector, we noticed that it wasn't the most streamlined process of cloning in our constructs nor transforming into Bacillus subtilis because of an MCS	that was incompatible with the BioBrick cloning system, and the extra steps needed to ensure the correct integration of the plasmid into the chromosome.

So we decided to design a new compact, highly efficient replicative expression vector that would serve not just our own future experiments, but the whole iGEM community. Thus, pUS270 was born!

While we were too short on time to construct it ourselves, we are holding onto it for a future project that can be tested and eventually submitted to the registry.

Select each dropdown for more information
</H4></DIV></DIV><DIV class="row" style="padding:20px;" align="center"><DIV class="col-xs-7"><DIV class="pus_270_info element_info_box_container" style="padding:5%;background:white;border:3px solid #e2e2e2;"><DIV class="panel-group" id="anti_info"><DIV class="panel panel-default"><DIV class="panel-heading" style="background-color:#f2cbf1"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse4" style="font-size:30px;">Antibiotic Resistance</A></H4></DIV><DIV id="collapse4" class="panel-collapse collapse"><DIV class="panel-body"><H4>Codes for Neomycin Phosphotransferase II that gives transformants resistance to both Neomycin and Kanamycin. Allows selective screening of transformants</H4></DIV></DIV></DIV></DIV><DIV class="panel-group" id="pip_info"><DIV class="panel panel-default"><DIV class="panel-heading" style="background-color:#b1d7dd"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse1" style="font-size:30px;">pIP501 Rep</A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse"><DIV class="panel-body"><H4>This gene codes for the Replicator protein that pairs with the Bacillus OriV and is essential to promote the independent replication of pUS270 in Bacillus. </H4></DIV></DIV></DIV></DIV><DIV class="panel-group" id="boriv_info"><DIV class="panel panel-default"><DIV class="panel-heading" style="background-color:#eca0a7"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse5" style="font-size:30px;">B. Subtilis origin of replication</A></H4></DIV><DIV id="collapse5" class="panel-collapse collapse"><DIV class="panel-body"><H4>This region allows independent replication of the vector within Bacillus at a high copy number to provide more copies of the recombinant gene to maximise expression.</H4></DIV></DIV></DIV></DIV><DIV class="panel-group" id="eoriv_info"><DIV class="panel panel-default"><DIV class="panel-heading" style="background-color:#fbde9e"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse2" style="font-size:30px;">E. coli OriV</A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse"><DIV class="panel-body"><H4>Taken from pSB1C3, having this origin of replication will make it easier to construct the plasmid using E. coli as well as improve cloning efficiency downstream. Chosen because of the high copy number of pSB1C3.</H4></DIV></DIV></DIV></DIV><DIV class="panel-group" id="laco_info"><DIV class="panel panel-default"><DIV class="panel-heading" style="background-color:#cdf8ff"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse6" style="font-size:30px;">Lac Operator</A></H4></DIV><DIV id="collapse6" class="panel-collapse collapse"><DIV class="panel-body"><H4>Consists of the target sequence for the lac repressor protein to contribute to the IPTG induction system.</H4></DIV></DIV></DIV></DIV><DIV class="panel-group" id="lacr_infolacr_info"><DIV class="panel panel-default"><DIV class="panel-heading" style="background-color:#d0cece"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse3" style="font-size:30px;">Multiple Cloning Site</A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse"><DIV class="panel-body"><H4>The multiple cloning site of pUS258 and other expression plasmids that we considered using either had too few restriction sites or sites that were incompatible for the efficient cloning with the BB Prefix and Suffix. So we have designed two highly diverse MCS’ that have 27 unique sites amongst them on either side of SF AmylCP-6 to promote simple cloning of parts from the Bio Brick Registry.</H4></DIV></DIV></DIV></DIV><DIV class="panel-group" id="super_info"><DIV class="panel panel-default"><DIV class="panel-heading" style="background-color:#a395c8"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse7" style="font-size:30px;">SuperFold AmilCP-6</A></H4></DIV><DIV id="collapse7" class="panel-collapse collapse"><DIV class="panel-body"><H4>SuperFold AmilCP-6 is a variant of amilCP which was used by the 2016 Sydney IGEM team and originally submitted to the registry by Team Uppsala Sweden in 2011. This variant has undergone site directed mutagenesis to increase the folding rate of the chromoprotein and enhance the blue colour. We are taking advantage of this protein to enable blue-white screening without the need for adding X-gal to the plates. Cloning with the two multiple cloning sites that flank SF AmilCP-6 essentially swaps it with the insert to leave the recombinant bacteria white rather than blue. </H4></DIV></DIV></DIV></DIV><DIV class="panel-group" id="lacr_infolacr_info"><DIV class="panel panel-default"><DIV class="panel-heading" style="background-color:#afe0b2"><H4 class="panel-title"><A data-toggle="collapse" href="#collapse0" style="font-size:30px;">Lac Operon Repressor</A></H4></DIV><DIV id="collapse0" class="panel-collapse collapse"><DIV class="panel-body"><H4>Codes for the production of the repressor protein that binds to the Lac Operator to prevent expression until induction of the media with IPTG. We have designed pUS270 to have the LacI under the pVeg promoter for strong constitutive expression in Bacillus.</H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row" align="center"><H4>References</H4><H4><OL><LI style="font-family:'Quicksand';color:#3e3f3f;font-size:16px;">Rajpal, G., Liu, M., Zhang, Yi, Arvan, P. (2009). Single-Chain Insulins as Receptor Agonists. Mol Endocrinol 23(5):679-688</LI></OL></H4></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>