Team:HZAU-China/Description
Members
Attributions
Collaborations
Silver
Gold
Integrated Practices
Engagement
Basic Parts
Composite Parts
Safety
Model
Software
Hardware
Measurement
Chassis
Light control
Motility
Riboswitch
Interlab
Description
Design
Proof
Demonstrate
description
Overview
What is AR?
Bio-Pattern Formation in Augmented Reality (BioPaFiAR)
Background
Bio-pattern formation
Why and what is bacteria motility
Light-switchable TCS
Riboswitch
Hardware
Light Plate Apparatus (LPA)
The BioPaFiAR hardware device
Software
BioBrick parts
References
Description
1. Overview
1.1 What is AR?
With the unprecedented rapid development in computer technology and the significant elevation in computing capacity, it is an undeniable truth that computers and similar devices such as smartphones have played a great role in our lives. People spend their spare time using Twitter and Facebook and analyze data through computer applications; they cannot live without computer products. Similar to the real world that we actually live in, computers can create virtual reality. Though a computer itself is tangible, what emerges on the screen is an intangible product of a series of logical combinations from electronic circuits. We are already in a mixed-reality age [1].  In 2015, our iGEM team  proposed the concept of “Mixed-Reality Cell” , constructed by the interaction between the real genetic oscillator and the simulated virtual oscillator, eventually leading to coupling and synchronization. However, existing in a world based on reality rather than mixed reality, computers are utilized as a tool to enhance one’s perception on reality and meet substantial requirements. This is a reality-oriented mixed-reality, which can also be called augmented reality (AR).
Figure 1. Augmented reality.
1.2 Bio-Pattern Formation in Augmented Reality (BioPaFiAR)
The growth of bacteria complies with certain laws, among which the easiest would be the law of reproduction-diffusion, under whose control the shape of colony would appears to be round. If a global regulation on bacteria growth is added, for example, enhancing the bacteria motility, the size of colony will become larger; if a local regulation is imposed on bacteria growth, the round shape of the colony will disappear; if the regulation is time-dependent, the regulation will change according to certain circumstances of the bacteria colony at specific times. Therefore, the shape of the bacterial lawn can be adjusted to match our present pattern. In contrary to the isolated state in Nature, in our project, the bacterial culture medium and the computer together form one system, which is composed of both reality and virtual world, and reality occupies the major part of the augmented reality system. In the system, the pattern formation of the bacteria population will be regulated as required. We design this project to propose a notion that biochemistry reactions in different kinds of biological stimulation can be mimicked by placing cells in an AR system, such as synthesis of enzymes, signal transduction, and bio-pattern formation. In addition, in the near future, this AR system can be applied to medical fields or in our daily lives. For example, it can be used in 3D printing of biological materials and in the in vitro generation of tissues or organs.
Welcome to our project: BioPaFiAR!
Figure 2. Welcome to BioPaFiAR!
2. Background
2.1 Bio-pattern formation
Bio-pattern formation is the establishment of spatial patterns in morphogenesis. The process of morphogenesis is of great importance to our perception and creation of life. However, it is a complicated process that has long been studied but still not totally understood. Models have been built to illustrate the principles of pattern formation throughout the century, among those are Reaction-Diffusion Model (Turing model) [2], kinetic model of density-suppressed motility, and Clock-Wavefront Model [3]. Recently, synthetic biology increasingly plays a significant role in unveiling the mystery of bio-pattern formation, with the concept of “build life to understand it”. By constructing genetically engineered circuits, scientists revealed the relationship between density and motility, and further the mechanism of pattern formation in bacterial community.
Figure 3. Bio-pattern formation in Nature.
Take a tour to the research history of bio-pattern formation with us↓
Bio-pattern formation is a significant field of research in synthetic biology. The process of morphogenesis is of great importance to our perception and creation of life. However, bio-pattern formation is a complicated process that has been studying since the 1950s. Alan Turing, father of computer as well as a prestigious mathematician, first proposed of the Reaction-Diffusion Model [2], which is also known as Turing model, to explain the basic chemical principles of bio-pattern formation. In this self-recovery system, interaction between two diffusible substances, short-range positive feedback and long-range negative feedback, leads to pattern formation. And the pattern is tuned according to different initial conditions. This classical model has been emphasized, tested and improved incessantly. Meanwhile, other models have appeared, for instance, the Clock-Wavefront Model and the kinetic model of density-suppressed motility. The Clock-Wavefront Model aims to elucidate the mechanism of pattern formation of somite in animal morphogenesis. Assuming that there’s a clock which is a coupled cellular oscillation, and a wavefront which in fact is rapid cell change moving along axis in cell growth, somite pattern formation is driven by alteration in oscillation when wavefront reaches the boundary of the cell community. As for model of density-suppressed motility, whose purpose is to demonstrate the pattern formation of a community of unicellular organisms such as bacteria, unique pattern can be formed relating the motility of bacteria to local cell density. These models are all confirmed by experiments.
With the rapid development in synthetic biology and systems biology in the 21st century, scientists increasingly tend to build life to understand bio-pattern formation. In 2005, a synthetic multicellular system was built by Subhayu Basu et al [4]. Through ‘band-detect’ gene networks, genetically engineered receiver cells respond to user-defined ranges of chemical molecule AHL concentrations produced by sender cells by producing different concentration of GFP. If sender cells are placed in the center, receiver cells will form a ring-like configuration due to the gradient of AHL.
Bio-pattern formation is the process of moulding specific spatial structure. In reality, to obtain particular shape of substance, we would sculpt a delicately designed mould in a traditional way or adopt modern 3D-printing technology. Method developed by Subhayu Basu is equivalent to mould sculpture, since the growth of bacteria is not considered. Distinguishingly, Chenli liu et al [5] constructed a genetic circuit that works in a single cell rather than a cell community. The circuit is consist of two parts, Density-sensing and Motility-control. Density-sensing part is responsible for producing AHL, while motility-controlling part act as receptor of AHL and further inhibits basic expression of the motility-related protein cheZ. In this self-recovery system, motility of cell is connected with cell density. If a suspension of bacteria is inoculated at the center of the dish, the spatialtemporal pattern of bacterial lawn will appear as a ring-like configuration, as suggested in B&C.
2.2 Why and what is bacteria motility?
Bio-pattern formation is determined by two major factors, growth and diffusion. The most crucial and controllable one is the pattern cells disperse, which is a reflection of bacterial motility. Hence, in order to study the principle of bio-pattern formation, motility-related gene cheZ is introduced into our project. CheZ is a positive regulating protein that modulates flagella rotation. Expression of CheZ leads bacteria to swim in semisolid agar while its deletion causes cells to tumble incessantly, resulting in a nonmotile phenotype.
Figure 4. The mechanisms of bacteria motility and the role of protein cheZ.
2.3 Light-switchable TCS
2.3.1 Why light-switch?
When bacteria establish bio-pattern, they receive different kind of signals in the ambient environment they live in. In our project, computer output light signal has been given as a motility-control module, which was devised to modify bacterial motility by regulating the transcription of cheZ, aiming at imitating the environment. The genetically engineered bacteria is able to perceive and distinguish different wavelength as an input signal, and express motility-related gene as an output. Therefore, the motility of the bacteria can be controlled responding to input light signal.
2.3.2 What is light-switchable TCS?
Optogenetics is a prevalent technology wherein light is utilized to mediate molecular biological processes via light-switchable proteins. In our project, a green/red two-component signal transduction system Ccas-CcaR [6] is used to control the gene expression of cheZ according to our need. We also call it the Traffic light model. When green light is on, light-switchable cascade is amplified and output protein PleD [7] is activated, resulting in the expression of cheZ and enhancement in motility. Conversely, when red light is on, the signal pathway is impaired. In a word, green light drives the bacteria to move and swim, while red light makes them stop.
Figure 5. The working mechanisms of the TCS circuits.
read more
2.4 Riboswitch
2.4.1 Why riboswitch?
In order to enhance the robustness of the AR system, to be specific, to filter the leakiness of light-switchable output promoter, riboswitch is incorporated in our project. Riboswitch possesses the characteristic that the aptamer is unlikely to respond to the upstream signal and activate downstream expression until it reaches its threshold. Thus, riboswitch can be applied to filter the noise resulting from leakiness.
2.4.2 What is riboswitch?
Riboswitches are cis-regulatory RNA elements residing in the 5’-untranslated region (5’-UTR) of bacterial mRNAs that modulate downstream gene expression upon binding of small specific molecules. A typical riboswitch usually contains two domains, an evolutionarily conserved aptamer domain for binding a target metabolite, and an expression platform for regulating downstream gene expression via a ligand-dependent conformational switch [8].
Figure 6. The working mechanisms of Riboswitches.
To take control of the specific bio-pattern formation of bacteria, relevant software and hardware has been devised.
3. Hardware
3.1 Light Plate Apparatus (LPA)
Two versions of devices are designed to quantify the effect of light-switchable TCS. V1.0 is a rough handmade device using paper cups which can achieve the goal to provide different wavelength of light and fixation of centrifugal tube where bacteria are cultured. In v2.0, based on the platform exploited by Gerhardt(Light Plate Apparatus ) [9], we constructed the hardware device that is able to precisely regulate the optogenetic circuit of the engineered bacteria utilizing 3D printing technology. This device successfully averts the interference of external light from the outside environment. Meanwhile, with the equipment of Single Chip Microcomputer (SCM), precise regulation on the intensity and wavelength of light can be accomplished. In summary, v2.0 eliminates the interference from ambient environment to the best extent when doing quantitative experiment of light-switchable system, thus improving the reliability and repeatablility of experimental result.
Figure 7. The hardware devices to quantify light-switchable TCS.
3.2 The BioPaFiAR hardware device
To control the bio-pattern formation of bacteria, we designed a device of industrial level. This device is capable of providing constant temperature and sterile environment that is suitable for bacterial growth. Besides, this device is also responsible for real-time monitoring of bacterial growth, light control over pattern formation, and projecting function.
Figure 8. Illustration for the BioPaFiAR hardware device.
4. Software
we developed a set of sofetware kits related to bacterial motility.
We designed two visualization software based on the principles of reaction-diffusion model and cellular automata model, so that synthetic biologists will have better understanding on the process of bacterial motility and bio-pattern formation. In the software, operators are free to simulate the process If bio-pattern formation by regulating the factors that affect bacterial growth and diffusion.
In order to better support our hardware，We programmed the software that is in line with the industrial level hardware device. When operators enter the pattern they want to form, the software controls the bacterial lawn to grow as preset pattern through real-time monitoring of bacterial growing status, predict future growing tendency and export the optical pattern to the bacteria as feedback.
Semi-automatic bacterial colony radius calculating software. Error from machinary judgement reduced by semi-manual operation. Value displayed directly at the same time.
Figure 9. The software GUIs. a. GUI for the simulation software of bio-pattern formation (platform: MATLAB).  b. GUI for the simulation software of bio-pattern formation (platform: Python). c. GUI for the software of bacteria colony size calculation.
5. BioBrick parts
1. BBa_K592003 PcpcG2 promoter, ccaR-regulated. PcpcG2 is a 238bp green-light activated promoter. We did quantitative measurement on this promoter to prove the CcaS/CcaR light-switchable TCS works efficiently. Moreover, to reduce the leakiness, we refactored the promoter by truncating the constitutive promoter region. See PcpcG2-172 ( BBa_K2012015 )
2. BBa_K819010 Medium cheZ generator. CheZ gene under medium promoter BBa_J23113 . We did quantitative measurement on this generator and made precise data analysis on the distance bacteria swim.
3. BBa_K1631018 , BBa_K1631019 , BBa_K1631020 . These three generators are CheZ gene under promoter BBa_J23101 , BBa_J23110 and BBa_J23102 , respectively. We made quantitative analysis on diffusion rate of E.coli, and debugged the data about promoter strength.
References：
1. Bin-Guang Ma, About Mechanics of Virtual Reality, written in 2005, published in: The General Science Journal, August 25, 2006.
2. A. M. Turing, The chemical basis of morphogenesis. 1953. Bull Math Biol 52, 153-197; discussion 119-152. (1990)
3. O. A. Igoshin, D. Kaiser, G. Oster, Breaking symmetry in myxobacteria. Curr Biol 14, R459-462. (2004)
4. S. Basu, Y. Gerchman, C. H. Collins, F. H. Arnold, R. Weiss, A synthetic multicellular system for programmed pattern formation. Nature 434, 1130-1134. (2005)
5. Chenli liu, et al. Sequential Establishment of Stripe Patterns in an Expanding Cell Population. Science 334, 238. (2011)
6. E. J. Olson, L. A. Hartsough, B. P. Landry, R. Shroff, J. J. Tabor, Characterizing bacterial gene circuit dynamics with optically programmed gene expression signals. Nature Methods 11, 449-455. (2014)
7. P. Aldridge, R. Paul, P. Goymer, P. Rainey, U. Jenal, Role of the GGDEF regulator PleD in polar development of Caulobacter crescentus. Mol Microbiol 47, 1695-1708. (2003)
8. A. Serganov, E. Nudler, A decade of riboswitches. Cell 152, 17-24. (2013)
9. K. Gerhardt et al., An open-hardware platform for optogenetics and photobiology. bioRxiv, 055053. (2016)
