<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HZAU-China_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HZAU-China/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><NAV><UL id="accordion" class="accordion"><LI><A href="https://2016.igem.org/Team:HZAU-China/Members">Members</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Attributions">Attributions</A></LI></UL><LI><A href="https://2016.igem.org/Team:HZAU-China/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Composite_Part">Composite Parts</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Safty">Safety</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Hardware">Hardware</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Measurement">Measurement</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Experiments-chassis_integration">Chassis</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Experiments-Light_control">Light control</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Experiments-Motility">Motility</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Experiments-Riboswitch">Riboswitch</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Interlab">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:HZAU-China/Demonstrate">Demonstrate</A></LI></NAV></DIV><TITLE>measurments</TITLE><DIV class="content_wrapper"><DIV class="content_part" id="masterdiv"><A href="#A1">1.Light-switchable System</A><UL class="my_submenu" id="sub1" style="display: block"><LI><A href="#A11">1.1 PCB Optimization in CcaS-CcaR System</A></LI><LI><A href="#A12">1.2 Optimization on Chassis Strain</A></LI><LI><A href="#A12">1.3 Optimization on PcpcG2</A></LI></UL><A href="#A2">2.Riboswitch System</A><UL class="my_submenu" id="sub2" style="display: block"><LI><A href="#A21">2.1 Phenotype of pleD Expression</A></LI><LI><A href="#A22">2.2 Characterization of Riboswitch</A></LI></UL><A href="#A3">3.Motility Switch System</A><UL class="my_submenu" id="sub3" style="display: block"><LI><A href="#A31">3.1 CheZ Protein Causes Differences in Motility of E.coli</A></LI><LI><A href="#A32">3.2 Relationship between Promoter Strength and Swarming Ability</A></LI><LI><A href="#A33">3.3 Normalization of Swarming Rate</A></LI></UL><A href="#A4">Reference</A></DIV><DIV class="main_part"><DIV class="article"><H2>Measurments</H2><P>We performed quantitative studies on different combinations of riboswitches and characterized the strength of a riboswitch by the rate of reading-through, which provided important reference values for quantifying the component of the riboswitches. Meanwhile, we performed a detailed and systematic characterization of the motion of bacteria. We made specialized equipment by 3D-printing technology to systematically quantify the photo-control system. Moreover, we measured motility of E. coli and characterized the quantitative relationship between the expression level of cheZ gene and the motility of bacteria.</P><H5>Highlight</H5><P>1. Fusion of cheZ with GFP as a reporter allows us to directly observe the expression level of cheZ in a visualized way.</P><P>2. An optimized inoculation method and condition has been created in swarming assay (Online Method can be found in <A href="https://2016.igem.org/Team:HZAU-China/Experiments-Motility">Wetlab-Experiment-Motility</A> )</P><P>3. Fluorescence assay method has been optimized in the experiment of light-switch and riboswitch part.
                (Online Method can be found in <A href="https://2016.igem.org/Team:HZAU-China/Experiments-Light_control">Wetlab-Experiment-Light-switchable TCS</A>&amp;<A href="https://2016.igem.org/Team:HZAU-China/Experiments-Riboswitch">Riboswitch</A> )
            </P><P>4. Fluorescence quantification measurement result in light-switch and riboswitch part shows accurate value with little variance, manifesting the reliability of the drawn conclusion about the validity of the devices.</P><P>5. Results in swarming assay give evidence to the effect of CheZ on bacterial motility, relationship between promoter strength and swarming ability, normalization on swarming rate.</P><H3>1.Light-switchable System</H3><P>CcaS-CcaR light-switchable system is a green-light activated signal transduction system. We constructed the primitive CcaS-CcaR system(pJT119b/pSR43.6, presented by Laboratory of Synthetic Biology, HZAU), and optimization on PCB synthetic plasmid, PcpcG2 promoter and chromosomal gene in different strains has been conducted. To testify the validity of the light-switchable TCS, fluorescence arrays were applied with the aid of the measurement instrument we projected. </P><P>Online Method can be found in <A href="https://2016.igem.org/Team:HZAU-China/Experiments-Light_control">Wetlab-Experiment-Light-switchable TCS</A></P><H4>1.1 PCB Optimization in CcaS-CcaR System</H4><P>As previously described, we removed the redundant ccaS in pSR43.6. Quantitative measurement on sfGFP fluorescence was operated in the purpose of certifying the feasibility of the removal, a premise for constructing plasmid PCB-J23117-BC345-cheZ and PCB-J23117-BC345-cheZ in our ultimate genetic circuit. Under green-light illumination, JT2 with PCB (ΔccaS) and CcaS-CcaR encoding plasmid pJTb119 occured to be green which can be recognized by naked eye after centrifugation (Figure 1.a). This qualitatively demonstrates that the light-switchable system is available without the redundant ccaS in pSR43.6. Moreover, quantitative measurement result on sfGFP fluorescence depicts that in E.coli strain JT2 CcaS-CcaR system produces 266.0 ± 19.3 and 509.0 ± 55.4 au of sfGFP in red and green light, corresponding to 1.91 ± 0.34-fold activation, illustrating that PcpcG2 is functionally regulated by light, green-activated and red-repressed (Figure 1.b). And bacteria without redundant ccaS shows significantly higher dynamic, indicating that the optimization is effective. We suspect that the exceeding expression level of CcaS has great impact on the metabolic pathway of bacteria. </P><P style="text-align:center">Figure 1.Quantification on sfGFP fluorescence in PCB optimization. </P><P style="text-align:center">a. Green-light activated JT2 with PCB (ΔccaS) and pJTb119 centrifugation. </P><P style="text-align:center"> b. Fluorescence assay of primitive PCB and PCB (△CcaS) in JT2. </P><H4>1.2 Optimization on Chassis Strain</H4><P>To better integrate the three devices, light-switchable TCS, riboswitch device, and motility device, it is inevitable to achieve co-expression in CL1 E.coli strain, cheZ deleted mutant strain. Optimization on chassis strain has been accomplished as we created SBSP strain (CL1 △EnvZ)(see <A href="https://2016.igem.org/Team:HZAU-China/Experiments-chassis_integration">chassis-integration</A>). Quantitative fluorescence measurement on CcaS-CcaR system in different strains JT2, CL1 and SBSP allude that the differences in fluorescence value under green and red light are approximately the same in three strains, although JT2 has higher expression efficiency (Figure 2). So it is plausible to transform the CcaS-CcaR system into CL1 (△EnvZ) for experiment on motility. </P><P style="text-align:center">Figure 2. Fluorescence assay in different chassis strain B,C,D. </P><P style="text-align:center">All three strains have co-expression of both two plasmids, PCB (△ccaS) and pJTb119.</P><H4>1.3 Optimization on PcpcG2</H4><P>Although CcaS-CcaR light-switchable system has shown excellent difference responding to green and red light, leakiness still exist and will be a huge problem when this device is applied for motility control. Therefore, optimization on green-light activated promoter PcpcG2 was executed (1). In bacteria strain D and E, containing PcpcG2 before and after truncation respectively, E seems to be less green than D when cultured in dark condition (Figure 3a). This is an indication of reduction in leakiness when constitutive promoter is eliminated. Fluorescence assays also infer that leaked expression in darkness significantly reduced after truncation of the constitutive promoter. (Figure 3b) By the way, the difference of fluorescence under green and red light is narrowed unexpectedly. We suspect it is due to the limited precision in cultural environment that the simple constructed detecting device v1.0 provides. For example, wavelength, light intensity, and uniformity in each centrifugal tube. These factors could be the reason to measurement mistakes.</P><P style="text-align:center">Figure 3. Characterization in PcpcG2 optimization. </P><P>a. Bacteria cultured in darkness. Strain D and E are both CL1 (△EnvZ) strains, contain the same plasmid PCB (△ccaS), but contain different PcpcG2, PcpcG2-238 and PcpcG2-172. b. Fluorescence assay of CcaS-CcaR system with PcpcG2-238 and PcpcG2-172 in CL1 (△EnvZ), and PCB (△CcaS) as chromophore.</P><H3>2. Riboswitch System</H3><P>Riboswitches are versatile devices for synthetic biology applications. They are RNA-based gene expression regulatory elements, composed of a ligand-sensing aptamer domain followed by an overlapping expression platform. Mechanisms of modulation of gene expression are highly divergent in prokaryotes and involve control of transcription, translation, splicing, and mRNA stability (2). Design on riboswitch device PT7+lacO+PleD (plasmid pET-28b) and J231XX+Bcxx+B0034+sfGFP(plasmid pSB4A5) has been done, making further quantification on single riboswitch (Bc3, Bc4, Bc5) and their tandem version Bc3-5 possible.
                Online Method can be found in <A href="https://2016.igem.org/Team:HZAU-China/Experiments-Riboswitch">Riboswitch</A></P><H4>2.1 Phenotype of pleD Expression</H4><P style="text-align:center">Figure 4. Phenotype of pleD expression </P><P>(a). Negative control: DE3 contain pET28b plasmid; experimental group: DE3 contain pET28b-pleD plasmid. The level of aggregation increased with the increase of IPTG’s concentration.</P><P>(b).For demonstrating expression of pleD, we used Congo red staining assay. As previous mentioned, high concentration of c-di-GMP could induce E. coli synthesize exopolysaccharides, and Congo red binding is a complex phenotype that reflects various outer membrane and surface properties including the presence of adhesive structures such as curli fimbria which are involved in biofilm formation. Wild type: DE3 contain pET28b plasmid, colony which was stained red color containing pET-pleD plasmid; Concentration of IPTG: 0.5mM.</P><H4>2.2 Characterization of Riboswitch</H4><P>We use “read-through rate” (RTR) of the downstream gene to measure the terminator forming efficiency of a riboswitch. </P><P>The read-through rate of each riboswitch in liquid medium was assessed by relative fluorescence intensity, which is the ratio of specific activity of a test strain to specific activity of the control strain (pET-28-pleD/J23117+sfGFP) with the same promoter of test circuits (such as J23117+Bc3-5+sfGFP).</P><H3>3.Motility Switch System</H3><P>CheZ is a positive regulating protein that modulates flagella rotation. Expression of CheZ leads bacteria to swim in semisolid agar while its deletion causes cells to tumble incessantly, resulting in a non-motile phenotype. We established a whole set of method to measure bacterial motility both qualitatively and quantitatively. Quantification has been done on the impact of CheZ on bacterial motility, relationship between promoter strength and swarming ability, normalization on swarming rate.</P><P><A href="https://2016.igem.org/Team:HZAU-China/Experiments-Motility">Online Method can be found in Wetlab-Experiment-Motility</A></P><H4>3.1 CheZ Protein Causes Differences in Motility of E.coli</H4><P>We constructed an overlapping extension fragment and purified it (Figure 6A).
                Its partial function as a reporter is certified by the observation under the fluorescence microscope (Figure 6B). As you can see, no matter the vector with a GFP reporter we constructed or the official CheZ generator (BBa_K819010) can complement the swarming ability of CL1, CheZ lacking strain of MG1655, at a certain degree. This experiment proves the function of CheZ on swarming assay on the one hand. On the other hand, the swarming ability of complementary strain can’t fully recover to the degree of wild type or the wild type introduced with our constructed vector. We suspect that it is due to the poor survival ability of the knock-out strain .By the way it will support the future work of quantifying. （Figure 7）
            </P><P style="text-align:center">Figure 6. cheZ-GFP overlapping extension fragment. </P><P>(A)Lane 1、2、3、5 are fusion cheZ-GFP fragment (including Biobrick) ,1377bp.(B)The observation of our strain under the fluorescence microscope proves the expression of certain protein.  </P><P style="text-align:center">Figure 7. Quantification on function of CheZ in swarming assay. </P><P>CL1 means the E.coli K12 mg1655 cheZ lacking strain as the negative control. CK means CL1 strain transformed with BBa_K819010. CZ means CL1 transformed with our constructed Biobrick, BBa_K2012007. MG means motile wild type strain, the E.coli K12 MG1666 strain as a positive control. MK means MG1655 strain transformed with BBa_K819010. MZ means MG1655 transformed with our constructed Biobrick, BBa_K2012007.</P><H4>3.2 Relationship between Promoter Strength and Swarming Ability</H4><P>Because the swarming ability of neither CheZ mutant nor CheA mutant can reach up to the level of the wild type, and there is a balance between swimming and tumbling which controls the size of swarming rings, so, to better proof the practicality of our project, we tried to use different concentration of CheZ to investigate the best condition of swarming ability. Therefore, we constructed a series of vectors containing different strength promoter followed by the standard RBS(B0034) and CheZ. According to our result of swarming assays, we found that the state of tumbling and swimming reaches its balance point adjacent to fairly weak promoter strength. Finally we decided to choose promoter J23117 as our final promoter choice. However, we have limited time to give details about the correlation between the concentration of cheZ and the size of swarm rings. We will confirm this conclusion by constructing IPTG inducing promoter-GFP fragment in single copy plasmid in further research.</P><P style="text-align:center">Figure 8. Quantification on relationship between promoter strength and swarming ability. </P><P>(A) Diameter of the colonies under different promoters at 17 hours, 36 hours and 48 hours separately. (B) The purple line means the diameters of CL1, the cheZ lacking strain, transformed with BBa_K20120xx. The green line means the diameters of CL1 transformed with BBa_K20120xx. The red line means the diameters of CL1 transformed with BBa_K20120xx. The blue line means the diameters of CL1 transformed with BBa_K20120xx. The sapphire line means the diameters of CL1 with no plasmids transformed as a negative control. (C)Different motility of CL1 transformed with cheZ under different strength of promoters. (D) Swarming ability of CL1 transformed with cheZ under different strength of promoters cultured for 17h, 36h, 48h after inoculation. </P><H4>3.3 Normalization of Swarming Rate</H4><P>Due to different strain and cultivate condition will influence the swarming diameter. But because of the limitation of experimental condition, like constant temperature and moisture, we need to demonstrate the swarming ability of wild type strain MG1655 under our experimental condition.</P><P style="text-align:center">Figure 9. Normalization of swarming rate.  </P><P>(A)The swam diameter at 8h,12h,28h,32h,38h,48h; (B)Visualization of data in (A). </P><H3>Reference</H3><P>1. S. R. Schmidl, R. U. Sheth, A. Wu, J. J. Tabor, Refactoring and optimization of light-switchable Escherichia coli two-component systems. ACS synthetic biology 3, 820-831 (2014); published online EpubNov 21 (10.1021/sb500273n).</P><P>2. M. Wachsmuth, S. Findeiß, N. Weissheimer, P. F. Stadler, M. Mörl, De novo design of a synthetic riboswitch that regulates transcription termination. Nucleic Acids Research 41, 2541-2551 (2012).</P><P>3. D. G. Ha, S. L. Kuchma, G. A. O'Toole, Plate-based assay for swimming motility in Pseudomonas aeruginosa. Methods in molecular biology 1149, 59-65 (2014)10.1007/978-1-4939-0473-0_7).</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>