Team:HZAU-China/Experiments-chassis integration
Members
Attributions
Collaborations
Silver
Gold
Integrated Practices
Engagement
Basic Parts
Composite Parts
Safety
Model
Software
Hardware
Measurement
Chassis
Light control
Motility
Riboswitch
Interlab
Description
Design
Proof
Demonstrate
chassis
1.Introduction
2.Materials and Methods
2.1 Material
2.2 Method
3.Result
3.1EnvZ gene disruption
3.2 Resistance Gene Elimination
Reference
Experiments-chassis integration
1.Introduction
To better integrate the three devices, light-switchable TCS, riboswitch device, and motility device, co-expression in a demanding E.coli strain is inevitable. However, it is a tough decision on which strain to choose as the chassis of our project. At first we have two options, one is to choose CL1, a strain lacks motility-related gene cheZ but has envZ which is functionally equivalent to CcaS. The other is to choose JT2, optogenetically optimized but possesses cheZ. Therefore, to better achieve our goal on making the whole project work, we are determined to enforce gene knockout with λ Red recombination system, either to knockout envZ in CL1, or to knockout cheZ in JT2. However, swarming assay shows that been JT2 has been traumatized in previous research and show no motility (swarming ability even lower than CL1, data not shown). Consequently, deletion of envZ in CL1 strain is the optimized design.
Figure 1. Mechanism of λ Red recombination
EnvZ- mutants were isolated as kanamycin-resistant colonies. The kanamycin resistance gene was then eliminated by using a helper plasmid encoding the CRE recombinase (plasmid pSC101-BAD-Cre-tet), based on the cre/lox recombination system (Figure 2)[2].
Figure 2. Mechanism of Cre/Lox recombination
With this instructive recombination system, we constructed a new strain, SBSP (CL1△envZ::lox).
2.Materials and Methods
2.1 Material
Chemical reagents: L- arabinose,
Kits: Plasmid Mini prep Kit (TIANGEN), Gel Extraction Kit (Axygen)
Enzyme: DraI, EcoRV(New England Biolabs), Primestar HS (TAKARA), PrimestarMax (TAKARA), TaqMix (TAKARA)
Bacteria: CL1 (MG1655△cheZ) (shared by Dr. Chenli Liu, SIAT CSynBER)
Plasmid: pSC101-BAD-gbaA-tet, pR6K-lox-kan-lox-new, pSC101-BAD-Cre-tet (all sources from Dr. Youming Zhang’s lab, Shandong University) (Figure3)
Figure3. Plasmid: pSC101-BAD-gbaA-tet, pR6K-lox-kan-lox-new, pSC101-BAD-Cre-tet
2.2 Method
2.2.1Replacing EnvZ with KanR
1.PCR cloning of insertion fragment. Primers included 80-bp sequence homology for the desired insertion location in the chromosome, and 20-bp sequence binding to template. Linearized plasmid pR6K-lox-kan-lox-new is used as template, digested with DraI. (Try not to use supercoiled DNA as template for PCR amplification [3]). PCR product is then gel-purified.
2.Preparation of transformants with pSC101-BAD-gbaA-tet. Transform plasmid pSC101-BAD-gbaA-tet into CL1 strain by electroporation. Select the transformants on a Tet-LB plate after incubation overnight at 30°C [4]. Plasmid extraction and digestion verification should be done as a confirmation of desired transformants, detailed in molecular cloning manual[4].
3.Integration of PCR product. After 10% L- arabinose induction at 37°C 950rpm for 40 min (puncture hole on top of centrifugal tube), cells are electroporated with 300-400 ng purified PCR fragments. Recover at 37°C 950rpm for 1h and plated on LB agar plates containing 30ug/ml kanamycin for selection, grown overnight at 37°C[5].
4.PCR verification of potential colony are done with forward primer binding to ompR while reverse primer binding to kanamycin resistance sequence. Positive clones are then sequenced with primers binding to ompR and pck.
2.2.2 Elimination of Kanamycin-resistant Gene
1.Transform plasmid pSC101-BAD-Cre-tet into positive clones by electroporation. Cells were cultured on LB dish with tetracycline at 30°C. Tetracycline-resistant (TetR) colonies are candidates for resistance gene elimination.
2.Three candidate colonies are picked and inoculated in LB media, grown at 30°C 200rpm for 2h. After induced by L- arabinose as previously mentioned, 1000-fold dilution is made and plated on LB without antibiotic at 37°C.
3.Resistance test on resistant gene elimination. Pick 16 colonies of each plate and double streak them on a KAN and a TET plate. Cultured at 30°C.
4. PCR verification of resistant gene elimination. PCR is enforced for those candidates grown on NEITHER plates. Primers bind to ompR and pck. Candidates are sequenced for ultimate confirmation on the elimination of KanR gene.
3.Result
3.1 EnvZ gene disruption
Both PCR verification and sequencing result (not shown here) indicates that chromosomal gene envZ is replaced with kanamycin resistance gene flanked by lox sites (Figure 4).
Figure 4. EnvZ gene disruption.
a. Recombination colonies on plate with kanamycin. Shows integration of KanR.
b. PCR verification on replacement of envZ with kanR. Primers bind to ompR and kanR sequence respectively..
3.2 Resistance Gene Elimination
To test the elimination of resistance genes, resistance test on Tet and Kan are operated. Most of the inoculation streaks did not grow (Figure 5), testifying that in most candidates, chromosomal integrated kanR is erradicated by Cre/Lox system and plasmid gene TetR is also eliminated at 37°C because of the temperature sensitive replicon PSC101.
Figure 5. Resistance test on resistance gene elimination, Kan (upper plates),Tet (lower plates).
Both PCR verification and sequencing result (not shown here) certify that KanR gene is eliminated, scarred by a lox site (Figure 6).
Figure 6. PCR verification of resistant gene elimination. Primers bind to upstream ompR and downstream pck sequence respectively. Lane1-5 refers to mutant candidates, Lane6 refers to CL1 as positive control.
In addition, verification on a phenotype level is completed as well. For more information, please check results here
In conclusion, envZ deleted strain of CL1 is constructed. We named it SBSP (for the spongebobmania in our team^____________^)
Reference
1.Datsenko, K. A. and B. L. Wanner (2000). "One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products." Proc Natl Acad Sci U S A 97(12): 6640-6645.
2.Tuntufye, H. N. and B. M. Goddeeris (2011). "Use of lambda Red-mediated recombineering and Cre/lox for generation of markerless chromosomal deletions in avian pathogenic Escherichia coli." FEMS Microbiol Lett 325(2): 140-147.
3.Sharan, S. K., et al. (2009). "Recombineering: a homologous recombination-based method of genetic engineering." Nat Protoc 4(2): 206-223.
4.J. Sambrook, D.W. . (1989). "Molecular Cloning: A Laboratory Manual".
5.Kuhlman, T. E. and E. C. Cox (2010). "Site-specific chromosomal integration of large synthetic constructs." Nucleic Acids Res 38(6): e92.
Acknowledgement
Thanks to Dr. Youming Zhang’s laboratory, Shandong University, for sharing the material and instruction on method.
