<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Kent_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Kent/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Kent/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Kent/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Kent/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Kent/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Kent/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><TABLE style="color:#BBF7EE;background-color:#CCBEB8;" cellpadding="1" cellspacing="1" border="0" width="100%" align="center"><TBODY><TR><TH align="center"><A href="/Team:Kent" title="Team:Kent">Home</A></TH><TH align="center"><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Kent" class="external text" rel="nofollow">Official Team Profile</A></TH><TH align="center"><A href="/Team:Kent/Team" title="Team:Kent/Team">Team</A></TH><TH align="center"><STRONG class="selflink">Project</STRONG></TH><TH align="center"><A href="/Team:Kent/Parts" title="Team:Kent/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:Kent/Notebook" title="Team:Kent/Notebook">Notebook</A></TH><TH align="center"><A href="/Team:Kent/Attributions" title="Team:Kent/Attributions"> Attributions</A></TH></TR></TBODY></TABLE><P><INS><BIG><B>No to NO: A novel approach to reduce greenhouse gas</B></BIG></INS></P><P><INS><BIG><B>Abstract</B></BIG></INS></P><P>In today’s rapidly changing environment greenhouse gases such as NO are an issue that need to be addressed. NO has been proven to have a detrimental impact on the environment and iGem Team Kent 2013 will provide a solution that focuses on reducing the amount of NO formed in waste water. Our system will utilise an engineered strain of <I>E. coli</I> which will be capable of converting this excess NO into ammonia. Our Biobricks have been designed to enable the detection of NO using the <I>norV</I> promoter. The NO can then be converted into ammonia via the nitrite reductase enzyme encoded by the <I>E. coli</I> gene <I>nrfA</I>. Our solution will have many advantages over the current approaches to waste water treatment such as reduced cost and risk of contamination. Our system will provide a source of recycled ammonia and could be a greener alternative to the Haber Bosch process.
</P><P><INS><BIG><B>Introduction</B></BIG></INS></P><P><B>Background of our problem</B></P><P>Nitric oxide is a greenhouse gas that is evolved during the processing of wastewater. Oxides of nitrogen (NOX) are key oxidants that have a role in the photochemical production of ozone, whilst also being linked with an increase in the oxidising capacity of the atmosphere2. Too much NOX within the atmosphere has a detrimental effect on these processes, adding to the greenhouse effect and causing acid rain2.
</P><P><B>Strategy</B></P><P>Team Kent aims to tackle the production of nitric oxide from waste water treatment processes by exploiting systems already present in laboratory strains of <I>E. coli</I>: <I>norR</I>, <I>norVW</I> and NrfA. Our idea is to engineer a plasmid to that will express the nitrite reductase enzyme <I>nrfA</I> in an NO-dependent manner under the control of the NorV promoter: in the presence of NO, NrfA will be expressed and will convert the NO to ammonia.
</P><P><B>NrfA</B></P><P>The nitrite reductase NrfA is encoded within the <I>nrfA</I> operon that is found in enteric bacteria such as <I>E. coli</I>. The primary function of NrfA is to enable the use of nitrite as a respiratory electron acceptor, forming ammonia as a final product. In addition, NO is a catalytic intermediate in this reaction, and can be can then also be used as a substrate reduction to ammonia by NrfA 3.
</P><P><B>NorR</B></P><P><I>norR</I> is the transcriptional regulator of the <I>norVW</I> operon encoding NO-detoxifying flavorubredoxin and associated oxidoreductase in E. coli. It is a nitric oxide (NO) sensing bEBP that is σ54 – dependent, and forms as an oligomer wrapping the promoter DNA around itself. Additionally, it is made of 3 domains, GAF, AAA+ and HTH. The regulatory GAF domain represses AAA+ activity, but NO binding to GAF relieves this repression. AAA+ contacts σ54 and induces open promoter complex formation powered by
</P><P>N-terminal regulatory GAF domain contains non-haem iron centre that binds NO. It represses AAA+ domain ATPase activity in the absence of NO, but NO binding relieves this repression. AAA+ domain is the active component of transcriptional activation. When ATP is bound, it contacts σ54 via a loop containing a conserved GAFTGA motif. On phosphate release following an ATP hydrolysis cycle, it relocates the σ54 RNA polymerase holoenzyme to induce open promoter complex formation
</P><P>HTH (helix-turn-helix) domain binds to 3 enhancer DNA sequences. All 3 are required for formation of <I>norR</I> oligomer at promoter, and thus AAA+ domain ATPase activity. DNA wrapping at this type of promoter is assisted by integration host factor (IHF) in E. coli. In contrast to many other bEBPs, in which the regulatory signal stimulates oligomerization, <I>NorR</I> oligomerization precedes NO signal sensing. By preforming at the <I>norVW</I> promoter, <I>NorR</I> is ‘primed’ to rapidly initiate <I>norVW</I> transcription in response to an NO signal
</P><P><B>Existing technology: limitations and scope for improvement</B></P><P>One currently existing method process used for the conversion of nitric oxides is selective catalytic reduction (SCR), in which a reductant gas, such as anhydrous ammonium, is added to a flue gas stream and is adsorbed onto a catalyst. Common catalysts used include base metals, zeolites or precious metals. This method may achieve 70-95% NOx reduction. However, it has a number of limitations. One such limitation is the occurrence of plugging of porous catalysts, often used due to their greater surface area, by compounds such as ammonium bisulphate. SCR catalysts are also sensitive to destruction by poisons such as halogens and alkaline metals that may disrupt the catalytic efficiency or cause unwanted ammonia oxidation. Tuning of essential reaction conditions, including temperature, catalyst functionality and ammonia distribution in the injected gas, is also required. Unreacted ammonia may pass through SCR in heightened quantity if the temperature is too low or if too much catalyst is depleted.
</P><P>Another related process is wet scrubbing, in which pollutant gas is scrubbed by liquid, either by spraying the gas with liquid or by forcing the gas through a liquid pool, in order to remove pollutants such as NOx from the gas. Wet scrubbers are compact, and can efficiently handle high temperatures and humidity, remove both gases and particles and neutralise corrosive gases. They provide a low initial cost and can give 60-70% pollutant removal. However, they suffer a great deal from corrosion, and also incur high operating costs when attempting to increase collection efficiency, due primarily to higher power requirements.
</P><P>Our proposed biological NOx elimination system in E. coli will have significant benefits over current removal methods. Firstly, it would be substantially
</P><P>cheaper to use, as it would not need the high temperatures required in the other methods, nor would it need the costly catalysts used in SCR. The absence of SCR catalysts from our system would also mean that it would lack the potential toxicity that can arise from metal by-products of catalyst degradation. In addition, being operated by genetically modified bacteria, our system could be more flexibly implemented, since provided optimal growth conditions, they may be operated on a variety of production levels and in a variety of mechanical systems. For example, many waste water purification systems already use batches of bacteria that degrade a significant proportion of pollutants present in waste water, and so our modified bacteria could be added into this batch to improve NOx clearance.
</P><P><INS><BIG><B>Materials and Methods</B></BIG></INS></P><P>Key methods such as ligation, transformations and digests were performed as suggested in the protocols laid out by iGEM modifying them when necessary, whilst other methods such as PCR, minipreps and gel purifications were carried out under protocols determined by the manufacturers of those materials. Colony PCRs were optimised for our primers. Below is a PDF containing the detailed description of the experiments performed.
</P><P><A href="https://2013.igem.org/File:Teamkentigemmethods.pdf" class="external text" rel="nofollow">PDF showing the materials and methods used in the project.</A></P><P><INS><BIG><B>Results</B></BIG></INS></P><P>In summary, two biobrick constructs were made. The sequences corresponding to <I>nrfA</I> and <I>norV</I> were put into pSB1C3 plasmid. Below is a link to the PDF containing detailed results of our project, including the diagnostic gels of the PCRs and ligated products of the genes.
</P><P><A href="https://2013.igem.org/File:Teamkentigemresults.pdf" class="external text" rel="nofollow">PDF showing the results from the methods we used in the project.</A></P><P><INS><BIG><B>Discussion</B></BIG></INS></P><P>Throughout our 18-week summer project we collected a number of different results that enabled us to collate and draw conclusions with. From a number of different transformation plates, gels and ligations; we were successfully able to produce our BioBricks <I>norV</I> and <I>nrfA</I>.
</P><P><B>Colony PCR</B></P><P>Through the use of colony PCR we were able to amplify our required BioBricks out of the genomic DNA of <I>E. coli</I>. As can be seen in experiment 3, with the use of our primers we designed through using various software, we were successful in producing the correct size fragments for our BioBricks. As we used colony PCR there is a chance that what was amplified had other sections of DNA elongated with the use of Taq polymerase through non-specific binding or other sequences of DNA being complementary to our oligonucleotide. However the likelihood of this was so small that we decided to continue as we had repeated the experiment numerous times. 
</P><P><B>Agarose gels</B></P><P>After designing our primers and knowing the base pair lengths of our insert, we were able to use agarose gels as seen in experiment 4 after PCR purification in order to see if they were the right length (Figure 6). In order to estimate if they were the right length, we used DNA marker bands that were standardised, showing us fragment lengths along the gel. This allowed us to visualise and estimate whether or not we had our insert. The use of agarose gels was quick and efficient, allowing us to carry on with other experiments when we got positive results. There are however margins of error with agarose gels, as they do not show differences in small amounts of fragments, so we sometimes had to go on the assumption that the enzymes had cut what they should have, with fragments being only 6 base pairs sometimes. We did however do positive controls on each gel to ensure the enzymes were working correctly in order to reduce this margin of error. 
</P><P><B>Ligations</B></P><P>The majority of our work over the summer period was spent on trying to get ligations to work. We had troubles using both rapid ligation kits and normal ligation kits. Through using a number of diagnostic gels, we were able to deduce that the rapid ligation kit was unsuitable for use and we switched to using a normal ligation kit, and with varying amounts of plasmid : insert ratios (<I>norV</I> and <I>nrfA</I>) (1:0, 1:1, 1:3 and 1:5) and a number of contaminated suspensions, we were able to achieve correct ligations of our single BioBricks at the last minute! We ran a ligation trying to stick together both BioBricks into the plasmid, however that ligation did not work so we were unable to test the products in the way which we initially wanted to (Figures 5 and 7).
</P><P><B>Future Experiments</B></P><P>As we had ran out of time due to having problems with ligations, we were unable to complete the joining of both of our BioBricks together, and unable to utilise the GFP we had prepared for ligations. If we had more time, we would have ligated both <I>norV</I> and <I>nrfA</I> together to complete our original function of using nitric oxide to produce ammonia. We would also have joined GFP to <I>norV</I> in order to receive visual information that our promotor was working efficiently. The activation of NorV would have increased the production of GFP allowing us to image it in real-time. 
</P><P>We ideally would have liked to use western blotting in order to ensure that the protein made by our product was correct. This would have allowed us to double check that the insert was working correctly and ensure reliability and validity. We would have first had to have transformed the plasmid in to bacteria to have been able to do this however, and we did not have enough time. As stated earlier, we were only able to take an educated guess that the inserts were at about the right length with 0.75% agarose gels.
</P><P>We would have also liked to gain some quantitative data on the amount of nitric oxide that was able to be converted into ammonia with this system, and the precise biochemical reactions involved. Knowing how many moles of cofactors and enzymes are required in each individual <I>E. coli</I> before they become detrimental would have been useful. This would have allowed us to know the maximum yield without being wasting resources. This would also have told us whether increasing this innate process in <I>E. coli</I> would kill it. If this were the issue we would try to resolve it by balancing out other systems already in place in order to ensure the smooth running of the process, possibly by creating other BioBricks. We would have been able to perform these experiments if the ligation of the two BioBricks together had worked.
</P><P>As our project has been very focussed on its applications, we are well aware that there are different microenvironments in different waste water treatment processes. For more future experiments, we were planning on taking samples from different waste water treatment plants to see the conditions and find an organism that perhaps would be more suited to it than our model <I>E. coli</I> organism is. We then would have transformed the plasmid into said organism to see if the plasmid was compatible with the organism and how efficient it was.
</P><P>GFP = green fluorescence protein. 
PCR = polymerase chain reaction.
</P><P><INS><BIG><B>References</B></BIG></INS></P><P>1. Cristina Muñoz, C., Paulino, L., Monreal, C., Zagal, E., Chilean Journal of Agricultural Research 70, 3: 485-497
</P><P>2. Fowler, D., Coyle, M. et al, The Global Nitrogen Cycle in the twenty-first century (2013) Philosophical Transactions of the Royal Biological Society 368 no. 1621
</P><P>3. Clarke, TA., Mills, PC. et al (2008). Escherichia coli cytochrome c nitrite reductase N<I>Italic text</I>rfA.. Methods in Enzymology. 437: 63-77. * Bush M, Ghosh T, Tucker N, Zhang X, Dixon R: Transcriptional regulation by the dedicated NO sensor, NorR: a route towards NO detoxification; Biochem. Soc. Trans. (2011) 39: 289–293; doi:10.1042/BST0390289
</P><P>4. Bush M, Ghosh T, Tucker N, Zhang X, Dixon R: Transcriptional regulation by the dedicated NO sensor, NorR: a route towards NO detoxification; Biochem. Soc. Trans. (2011) 39: 289–293; doi:10.1042/BST0390289
</P><P>5. Tucker N, Ghosh T, Bush M, Zhang X, Dixon R: Essential roles of here enhancer sites in σ54 – dependent transcription by the nitric oxide sensing regulatory protein NorR; Nucl. Acids Res. (2010) 38(4): 1182 – 1194; doi: 10.1093/nar/gkp1065
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Kent/Project">http://2013.igem.org/Team:Kent/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Kent/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Kent/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Kent/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Kent/Project&amp;oldid=298683" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>