"
Page
Discussion
View source
History
teams
Log in
 
Team:UNAM-Genomics Mexico/Notebook/pBBR1MCS-5
From 2011.igem.org
UNAM-Genomics_Mexico
Home
Project
Hydrogen Production
RhizoKit
Notebook
System Assembly
pBBR1MCS-5
repC
nifH
HydA
Optimization
HydA Optimization
Parts
Human Practices
BioSintetizarte
Hydrobium etli goes Cuatzo
Safety
Modeling
Flux Balance Analysis
Game Theory
Markov Model
Extreme Pathways
Celular Automaton
Team
Attributions
Individuals
Wiki Requirements
Attributions
Notebook
Parts
Team
Safety
Lab Logbook for pBBR1MCS.
Contents
1Lab Logbook for pBBR1MCS.
1.1Week: june 13 to 18
1.2Week: june 20 to 25
1.3Week: june 27 to july 1
1.4Week July 4 to 9
1.5Week July 11 to 16
1.6Week July 18 to 23
1.7Week July 25 to 30
1.8Week August 1 to 5
1.9Week August 7 to 12
1.10Week September 5 to 10
1.11Week September 12 to 17
1.12Week September 19 to 24
Week: june 13 to 18
At beginning of the week was decided the best way to insert a prefix-RFP-suffix fragment to thepBBR1MCS-5 plasmid, which is a plasmid with broad host-range and is capable to conjugate to R. etli through E. coli S17, being the most efficient way to introduce genes into R. eltli.  The pBBR1MCS-5 has a multiple cloning site (MCS) with the following restriction sites:
KpnI
ApaI
XhoI
Sal I
Bsp106 I
ClaI
HindIII
EcoR1
PstI
SmaI
Bam H1
SpeI
XbaI
BstXI
SacI
So we want to eliminate the most of the restriction sites and obviously we need to eliminate the prefix and suffix sites (italic). To do it we are going to cut in the ApaI and SacI site, on the other hand we're going to amplify a region which contains the suffix, an RFP and the prefix of the plasmid pSB1T3, this PCR amplification will be done with primers containing ApaI and SacI respectively, thereafter we'll can insert this amplification in our pBBRMCS-5.
Once decided the way to add the prefix and suffix to pBBR1MCS-5, we spent the rest of the week isolating the plasmid and trying to digest the plasmid by apaI, knowing that only works with strains dcm-.  Knowing that E. coli DH5α is dsc+ we tried unsuccessfully  to cut with ApaI, the following gel shows it:
Lanes:1.Ladder 10 kb  2.pBBR1MCS-5 digested by apaI 3.pBBRMCS-5
Week: june 20 to 25
It was made competent BL21 E. coli cells, because that strain is dmc-. After done, the transformation of the pBBR1MCS-5 using the previous competent cell was done and further plasmid isolations were also done. We tested if apaI works well in this strain, the results were successful and are shown in the following electrophoresis gel, however our plasmid isolation were too dirty to obtain a single band
Lanes:1.Ladder 10kb 2 pBBR1MCS-5 digested by ApaI 3.PBBR1MCS-5
Week: june 27 to july 1
In this week we did plasmid isolations again, but rather using mini-preps we used a Roche kit, this in order to obtain plasmid clean enough to further digestion. Once the plasmid isolations were done it was proceeded to do a double digestion using apaI and sacI(see week: june 13 to 18). A PCR amplifying a prefix-RFP-suffix fragment also was done; to that fragment were added ApaI and SacI at the ends through the primers:
Lanes:1. Ladder 10 Kb 2. pBBRMCS-5 3. pBBRMCS-5 digested by sacI 4. pBBRMCS-5 5. pBBRMCS-5 digested by apaI and sacI 6. Digested PCR(RFP).
We spent the rest of the week doing the ligation between the pBBR1MCS-5 and the prefix-RFP-suffix fragment and doing the further transformation. But we failed to obtain any colony with our insert probably because of the dirty PCR, so we did the PCR(prefix-RFP-suffix) again.
Week July 4 to 9
In this week the  ligation and transformation of the digested clean PCR(prefix-RFP-suffix) and pBBR1MCS-5 was repeated. Of 18 colonies that grew only one became red, indicating that the assembly was done:
Petri dish showing  the only  re-plated  red colony
In order to see if the new plasmid properly contains the prefix and suffix a digestion with EcoR1 was made, this was successful. Digestions with other enzymes were done in the following week.
Week July 11 to 16
Following testing restriction enzymes on our new plasmid, a transformation in sp. BL21 was done in order to digest with ApaI both, the transformation and the digestion were successful. A digestion with sacI was also and successful made:
Electrophoresis gel showing the insert (prefix-RFP-suffix) and plasmid (pBBR1MCS-5) with their correct sizes. Lanes: 1.Ladder 10 kb 2. Double digestion
Digestions with XbaI and SpeI were also successful. We did not do a digestion with PsteI because we didn’t have it.  Once we checked the plasmid integrity we proceeded to try to conjugate the plasmid into R. etli in order to see if it conserves that capability. But the most efficient conjugation protocol require a S17 E. coli strain which didn’t have, so we tried to conjugate with an alternative protocol. The results are in the next week.
Week July 18 to 23
On this week negative results about the conjugation were obtain, attempts to fix and improve the alternative conjugation protocol were made, but they were unsuccessful. We ask to other lab for E. coli s17 in order to do the original protocol, and they were placed to give it to us.
Week July 25 to 30
Competent S17 cells were made and transformation of the new plasmid (pBBR1MCS-5) into them was done.  After, the conjugation with the original protocol was made and we waited for the results until next week. Simultaneously the plasmid stability characterization was started. It consists to watch the percentage of bacteria without any selection that loose the plasmid. To do it we started with a 100 ml culture at O.D of 0.01 without selection, and every 12 hours we re-plated an aliquot in both antibiotic medium and antibiotic-free medium and observed the percentage of bacteria that loose the plasmid. This experiment was long, because we made observations for 72 hrs. The results are shown the next week.
Week August 1 to 5
The conjugation made the previous week had a problem, so it was carefully repeated These are the results of the plasmid stability
Hour
Colonies without antibiotic
Colonies with antibiotic
Ratio
0
100
100
1
12
100
100
1
24
97
96
0.989691
36
68
65
0.955882
48
100
98
0.98
60
100
99
0.99
72
100
99
0.99
We conclude that the plasmid has high stability
Week August 7 to 12
The conjugation made the previous week was successful. We spent the whole week doing antibiotics selections in order to get only R. eli with our plasmid, although the colonies did not get red in this week, they did after one week, so there’s something in R. elti  that reduces the replication and/or the transcription.
Week September 5 to 10
We realized that when we amplify the RFP from the pSB1T3 we designed wrong our primers, because we also wanted to get the sequence previous the prefix and next the suffix where verification primers bind, but we did not get the sequence next de suffix, so a new primer was re-designed. In these wee a second attempt to ligate with this PCR product was made, but it was unsuccessful
Week September 12 to 17
Problems with the digestion of the pBBR1MCS-5 by apaI and sacI arise, the digestion was made in order to ligate by sticky ends with the PCR product (prefix-RFP-suffix).  Because of the unknown problems the ligation could not be made.
Electrophoresis gel showing that the digestion if the pBBR1MCS-5 failed. Lanes:1.Empty 2.Digested PCR (apaI and sacI) 3.Digested pBBR1MCS-5 (apaI and sacI) 4.Digested pBBR1MCS-5 (apaI and sacI) 5.Digested pBBR1MCS-5(apaI and sacI) 6.Ladder 500pb
Week September 19 to 24
Isolations and double digestions were made in the pBBR1MCS as well as in the PCR product. The ligation was tried for the last time. The digestions were clean:
Back to the Rhizobial kit Overview
Retrieved from " http://2011.igem.org/Team:UNAM-Genomics_Mexico/Notebook/pBBR1MCS-5 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
