"
Page
Discussion
View source
History
teams
Log in
 
Team:UNAM-Genomics Mexico/Notebook/repC
From 2011.igem.org
UNAM-Genomics_Mexico
Home
Project
Hydrogen Production
RhizoKit
Notebook
System Assembly
pBBR1MCS-5
repC
nifH
HydA
Optimization
HydA Optimization
Parts
Human Practices
BioSintetizarte
Hydrobium etli goes Cuatzo
Safety
Modeling
Flux Balance Analysis
Game Theory
Markov Model
Extreme Pathways
Celular Automaton
Team
Attributions
Individuals
Wiki Requirements
Attributions
Notebook
Parts
Team
Safety
Lab Logbook - Rhizobium etli kit repC
This section explains the generation of:
the repC part and its use for generating
a plasmid capable of replication in Rhizobials.
A section of plasmid pSB1C3 between XhoI restriction sites would be excised. It would be replaced with a construction containing the repC locus and an antibiotic resistance with proper terminators on its borders. This would give the plasmid means to replicate in Rhizobiaceas, while maintaining its standard assembly properties. At the same time the repC locus would be standardized as a biobrick part.
The activities are organized by weeks (monday to sunday). We mention the general remarks for the activities that we performed.
Contents
1Lab Logbook - Rhizobium etli kit repC
1.1Week: July 4 – July 10
1.2Week: July 11 – July 17
1.3Week: July 18 – July 24
1.4Week: July 25 – July 31
1.5Week: August 1 – August 7
1.6Week: August 8 – August 14
1.7Week: August 15 - August 21
1.8Week: August 22 - August 28
1.9Week: August 29 – September 4
1.10Week: September 5 - September 11
1.11Week: September 12 - September 18
1.12Week: September 19 - September 25
Week: July 4 – July 10
We designed oligonucleotides to amplify the repC region. This oligos have codes 5207 for the forward amplifications and 5208 for the reverse amplifications, and they include the standard prefix and suffix. Plasmid pUX19-XRK5 was transformed in E. coli cells. The plasmid was then extracted and digested with SpeI to verify its identity.
Week: July 11 – July 17
During this week efforts were focused on other project areas, while the oligos were synthesized.
Week: July 18 – July 24
During this week efforts were focused on other project areas, while the oligos were synthesized.
Week: July 25 – July 31
We extracted plasmids from the transformation of pUX19-XRK5. We transformed with the registry part B0015, a double terminator, in pSB1AK3.
Week: August 1 – August 7
We extracted and digested B0015 in pSB1AK3 with EcoRI and XbaI for further biobrick assembly. We amplified P1003, the kanamycin resitance, with a PCR. This amplification was then cleaned and digested with EcoRI and SpeI.
Week: August 8 – August 14
During this week efforts were focused on other project areas.
Week: August 15 - August 21
We worked on PCR of the parts we would assemble to generate the plasmid (B0015 and pSB1T3). We ligated P1003 with B0015, but the ligation reaction was unsuccessful.
Week: August 22 - August 28
We performed a PCR reaction to amplify, with oligos that add both the prefix and the suffix, the repC region including the regulatory antisense RNA gene, and both the RNA and the RepC ORF promoters. This PCR was cleaned and digested with EcoRI and SpeI. We attempted the ligation of the B0015 double terminator, which is sent in pSB1AK3, to the plasmid pSB1T3. This was unsuccessful, so we double digested B0015 in pSB1AK3 with EcoRI and XbaI, to ligate the repC region and fuse repC with the terminator. We transformed competent bacteria with this ligation.
Week: August 29 – September 4
Six colonies from the transformation of the ligation of B0015 and repC grew in ampicillin. From these, only one grew in kanamycin. This colony was thought to harbor the proper ligation product. For the proper assembly of our plasmid we required a bidirectional transcriptional terminator and an antibiotic resistance gene. We transformed bacteria with five different parts: three transcriptional terminators (B0014, B1007, B1001) and two resistances (tetracycline P1001, chloramphenicol P1004). The tetracycline resistance was unsuccessful. We extracted plasmid from these transformations and double digested with EcoRI and either SpeI (for the chloramphenicol resistance) or XbaI (terminators in plasmid). The goal was to fuse P1004 with the resistance plasmids.
Week: September 5 - September 11
Band isolation of EcoRI, SpeI double digested P1004 was unsuccessful. We extracted plasmid from three strains. The first one, the presumed ligation of repC with B0015 turned to be a false positive not containing repC. The second one was more B0015 in pSB1AK3 for repeating the ligation. The third one was P1004 in pSB1AK3, to attempt the isolation of P1004 again. These plasmids were double digested as the week before (E,X for the terminators, E,S for P1004). P1004 was isolated successfully this time. We prepared ligations of P1004 with B0014 and of repC with B0015.
Week: September 12 - September 18
We transformed cells with last week ligations. repC showed some bacterial colonies, but P1004 didn’t. We replated and extracted the plasmid of five repC bacterial colonies (1, 2, 5, 6, 9).We performed more ligation reactions with different insert:vector ratios from those used before for P1004 with B0014. We transformed these ligations. To verify the repC ligation we digested the five extracted plasmids with SpeI and we ran these plasmids in an agarose gel. All plasmids showed a size similar to pSB1AK3, meaning false positives, except strain two which had a band at the size of the sum of the lengths of pSB1AK3 and repC.
Lane:
1 250bp Ladder
2-6 Digestions with SpeI of plasmid extractions of strains 1, 2, 5, 6, 9.
7-11 Plasmid extractions of strains 1, 2, 5, 6, 9.
Preventing the possibility that these transformed cells would be contaminated, we prepared a new ligation reaction, but this time pSB1AK3 would be dephosphorylated. We extracted plasmid from the latter ligations of P1004, and more strains from the repC ligation of the previous week (4, 7, 8 , 10-13).
Week: September 19 - September 25
We transformed cells with the ligation of repC and B0015. We tested both the quality of the extraction and the size of each plasmid, expecting to see the right size. Plasmid size was observed after digestion with XbaI:
Gel A
Sample
Gel B
Sample
1
Ladder 500bp
1
Ladder 500bp
2
P1004ter 1 SD XbaI
2
P1004ter 1
3
P1004ter 2 SD XbaI
3
P1004ter 2
4
P1004ter 3 SD XbaI
4
P1004ter 3
5
repCter 4 SD XbaI
5
repCter 4
6
repCter 7 SD XbaI
6
repCter 7
7
repCter 8 SD XbaI
7
repCter 8
8
repCter 10 SD XbaI
8
repCter 10
9
repCter 11 SD XbaI
9
repCter 11
10
repCter 12 SD XbaI
10
repCter 12
11
repCter 13 SD XbaI
11
repCter 13
12
10μL Gibson HydG 15/Sep Pae
12
PBB53 2A Melissa
13
 
13
PBB53 2B Melissa
14
 
14
PBB53 3A Melissa
15
 
15
PBB53 3B Melissa
We decided that PCR was the only technique that would tell us definitely which plasmids had repC. So we performed them and realized only strains 2 and 10 harbored repC. We tried using restriction enzymes patterning but it was not possible because the sample would be diluted too much.
Week: September 26 - October 2
We cleaned the repC PCR reaction, digested with EcoRI and PstI and ligated it into pSB1C3 to send the part to the registry.
Back to the Rhizobial kit Overview .
Retrieved from " http://2011.igem.org/Team:UNAM-Genomics_Mexico/Notebook/repC "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
