<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_KU_Leuven_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:KU Leuven/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right"><LI id="TeamKU_Leuven_page" class="active"><A href="https://2017.igem.org/Team:KU_Leuven">HOME</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:KU_Leuven/Team" class="dropdown-toggle " data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">TEAM</A></LI><LI id="Team_page"><A href="https://2017.igem.org/Team:KU_Leuven/Team">Team</A></LI><LI id="Collaborations_page"><A href="https://2017.igem.org/Team:KU_Leuven/Collaborations">Collaborations</A></LI><LI id="Sponsors_page"><A href="https://2017.igem.org/Team:KU_Leuven/Sponsors">Sponsors</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:KU_Leuven/Project" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">PROJECT</A></LI><LI id="Description_page"><A href="https://2017.igem.org/Team:KU_Leuven/Description">Description</A></LI><LI id="Design_page"><A href="https://2017.igem.org/Team:KU_Leuven/Design">Design</A></LI><LI id="Experiments_page"><A href="https://2017.igem.org/Team:KU_Leuven/Protocols">Protocols</A></LI><LI id="Notebook_page"><A href="https://2017.igem.org/Team:KU_Leuven/Notebook">Notebook</A></LI><LI id="Logbook_page"><A href="https://2017.igem.org/Team:KU_Leuven/Logbook">Logbook</A></LI><LI id="Interlab_page"><A href="https://2017.igem.org/Team:KU_Leuven/InterLab">InterLab</A></LI><LI id="Model_page"><A href="https://2017.igem.org/Team:KU_Leuven/Model">Model</A></LI><LI id="Demonstrate_page"><A href="https://2017.igem.org/Team:KU_Leuven/Demonstrate">Demonstrate</A></LI><LI id="Attributions_page"><A href="https://2017.igem.org/Team:KU_Leuven/Attributions">Attributions</A></LI><LI class="dropdown active"><A href="https://2017.igem.org/Team:KU_Leuven/Parts" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">PARTS</A></LI><LI id="Parts_page"><A href="https://2017.igem.org/Team:KU_Leuven/Parts">Parts</A></LI><LI id="Basic_Part_page"><A href="https://2017.igem.org/Team:KU_Leuven/Basic_Part">Basic Parts</A></LI><LI id="Safety_page"><A href="https://2017.igem.org/Team:KU_Leuven/Safety">SAFETY</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:KU_Leuven/HP/HP_Main" class="dropdown-toggle active" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">HUMAN PRACTICES</A></LI><LI id="HP_Main_page"><A href="https://2017.igem.org/Team:KU_Leuven/HP/HP_Main">Human Practices</A></LI><LI id="Silver_page"><A href="https://2017.igem.org/Team:KU_Leuven/HP/Silver">Silver</A></LI><LI id="Gold_Integrated_page"><A href="https://2017.igem.org/Team:KU_Leuven/HP/Gold_Integrated">Integrated and Gold</A></LI><LI id="Education"><A href="https://2017.igem.org/Team:KU_Leuven/HP/Education">Education</A></LI><LI id="Engagement_page"><A href="https://2017.igem.org/Team:KU_Leuven/Engagement">Public Engagement</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:KU_Leuven/Awards" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">AWARDS</A></LI><LI id="Applied_Design_page"><A href="https://2017.igem.org/Team:KU_Leuven/Applied_Design">Applied Design</A></LI><LI id="Entrepreneurship_page"><A href="https://2017.igem.org/Team:KU_Leuven/Entrepreneurship">Entrepreneurship</A></LI><LI id="Prizes"><A href="https://2017.igem.org/Team:KU_Leuven/Prizes">Prizes</A></LI><LI id="Model_page"><A href="https://2017.igem.org/Team:KU_Leuven/Model">Model</A></LI><LI><A href="https://igem.org/2017_Judging_Form?team=KU_Leuven">JUDGING FORM</A></LI></DIV></DIV><DIV class="igem_2017_content_wrapper"><DIV id="grad"><DIV class="background"><H1 style="text-align:center; padding: 30px; font-size:50px; color: white;">Demonstrate</H1><P class="head" style="text-align:center;color: white; padding: 10px 10px 60px 10px; font-size:15px;">Our team gathered a lot of results during the course of three months. The main source of the data was derived by patch clamp, an instrument to measure membrane potentials and currents in cells. The goal of our HEKcite project was to design a device that can detect changes in drug concentration by giving a change in overall rhythm as output.
								Below you will find a summary of our results. 
								</P></DIV><DIV class="container"><DIV class="border2"><H3><B>Key Achievements</B></H3><UL style="font-size:18px;"><LI> • Creating a mathematical model of an oscillating biological system </LI><LI> • Creating electrically oscillating HEK293 cells </LI><LI> • Adapting the rhythm with different types of medications</LI></UL></DIV><H2>The mathematical model</H2><P style="text-align:justify; padding: 0px 50px 0px 50px;">Our team developed a mathematical model describing the oscillation of the membrane potential in  a transfected HEK cell. The model is based on the (h)existing model of Kharche (2011), which describes the ionic currents through ion channels in sinoatrial node cells in the heart. We created a model containing three ion channels of interest, which are HCN, hERG and α1G. Furthermore, the model also contains equations for Sodium-Potassium exchangers, Sodium-Calcium exchangers, and several background currents. By adjusting the time kinetics of these channels, we fitted the model to our own experimental values. Other parameters that were adjusted to match experimental values were ion channel conductance, membrane capacitance and cell volume. 
                        This electrophysiological model allowed us to perform experiments in silico to learn more about optimal ion channel ratios, rhythm modulation and specific ionic currents.

						</P><H2>Creating oscillating HEK cells</H2><H3>Calcium Imagining</H3><P style="text-align:justify; padding: 0px 50px 0px 50px;">We performed calcium imaging to screen for intracellular changes in calcium in the transfected HEK cells. The calcium imaging setup allowed us to visualize more than 100 cells at the same time, which is ideal for an optimization process. The experiments consisted of several transfection ratios and extracellular potassium concentrations. We found an optimal transfection ratio of 2:1 α1G to hERG at an extracellular potassium level of 2-5 mMol. The transfected cells were already stably expressing HCN2, which is ideal since our mathematical model showed that HCN2 is the most important ion channel responsible for a steady rhythm. A stable HCN2 expression across different experiments is beneficial for replicating results with a similar rhythm. Our method allowed to measure the intracellular calcium every two seconds, which was too slow to measure the expected oscillations, but enough to see a subtle change in intensity across different images. 
                        </P><H3>Patch Clamp</H3><P style="text-align:justify; padding: 0px 50px 0px 50px;">Whole-cell patch clamp is a technique where you can measure oscillation at a greater temporal resolution, with a sampling rate up to 10 000 Hz. It measures the electrical current or voltage difference across the membrane of a single cell. After optimizing the experimental setup with calcium imaging, we started using patch clamp with the parameters derived from the experiment.    HEK cells transfected with α1G, hERG and HCN2 oscillated when stimulated with 40-150 pA. We did not observe oscillations in cells without stimulation.
                        </P><H2>Patch clamp results</H2><H3>1. Ethosuximide</H3><P style="text-align:center; padding: 0px 50px 0px 50px;"><I>Figure 1:</I> A 30 seconds measurement of a transfected HEK cell with α1G, hERG and HCN2 is shown. The measurement was performed in the physiological Krebs buffer, while stimulating with 100pA.
                                                </P><P style="text-align:center; padding: 0px 50px 0px 50px;"><I>Figure 2:</I> A 30 seconds measurement of the same cell as in Figure 1 is shown at a different timeframe. The measurement was performed in the physiological Krebs buffer with ethosuximide, while stimulating with 100pA.
						</P><P style="text-align:center; padding: 0px 50px 0px 50px;"><I>Figure 3:</I> Comparison of the peak intervals of control vs. ethosuximide.

Using MATLAB, we analyzed the  the amount of peaks per 10 seconds for both control and ethosuximide conditions. The analysis showed a 20% decrease in frequency of rhythm when ethosuximide was added.
A paired t-test (p = 0.0004) was performed to prove that there is a change in overall rhythm. This means that the decrease in rhythm is significantly different from the control rhythm. The error bars on the figure visualize the 95% confidence interval.

						</P><H4>Conclusion</H4><P style="text-align:justify; padding: 0px 50px 0px 50px;">
						Ethosuximide slowed down the rhythm of the transfected cells significantly. Moreover, this measurement was shown to be continuously and reversibly (not shown in figures). The drug works by directly blocking α1G, which has a considerable impact on the rhythm.

						</P><H3>2. Ivabradine</H3><P style="text-align:center; padding: 0px 50px 0px 50px;"><I>Figure 4:</I> A 30 seconds measurement of a transfected HEK cell with α1G, hERG and HCN2 is shown. The measurement was performed in the physiological Krebs buffer, while stimulating with 40pA.

                                                </P><P style="text-align:center; padding: 0px 50px 0px 50px;"><I>Figure 5:</I> A 30 seconds measurement of the same cell as in Figure 4 is shown at a different timeframe. The measurement was performed in the physiological Krebs buffer with Ivabradine, while stimulating with 40pA.

						</P><P style="text-align:center; padding: 0px 50px 0px 50px;"><I>Figure 6:</I> Comparison of peak intervals of control vs. Ivabradine
As previously explained, we analyzed the intervals using Matlab. A paired t-test was performed (p &gt; 0.05), which did not show a significant difference. The figure demonstrates that the rhythm is not significantly changed.

						</P><P style="text-align:center; padding: 0px 50px 0px 50px;"><I>Figure 7:</I> Comparison of peak amplitudes of control vs. Ivabradine
Using Matlab, we measured the amplitudes of the peaks in the same way as we measured the intervals, comparing periods of 10 seconds. A paired t-test was performed (p &lt; 0.0001). The analysis showed an increase in amplitude when adding Ivabradine compared to control settings.  The error bars on the figure visualize the 95% confidence interval.

						</P><H4>Conclusion</H4><P style="text-align:justify; padding: 0px 50px 0px 50px;">
						Ivabradine does not significantly alter the rhythm in our system, but it increases the amplitude of oscillations. By blocking HCN, we believe that the cell hyperpolarizes more then compared to control, which causes more activation of α1G which in turn causes a bigger depolarizing current. This unexpected result clearly shows that there is a complex interaction within currents that cannot always be predicted.
						</P><H3>2. cAMP</H3><P style="text-align:center; padding: 0px 50px 0px 50px;"><I>Figure 8:</I>
						A 30 seconds measurement of a transfected HEK cell with α1G, hERG and HCN2 is shown. The measurement was performed in the physiological Krebs buffer, while stimulating with 40pA.

                                                </P><P style="text-align:center; padding: 0px 50px 0px 50px;"><I>Figure 9:</I>
						A 30 seconds measurement of the same cell as in Figure 8 at a different timeframe. The measurement was performed in the physiological Krebs buffer after addition of cAMP, while stimulating with 40pA.

						</P><P style="text-align:center; padding: 0px 50px 0px 50px;"><I>Figure 10:</I> As previously explained, we analyzed the intervals using Matlab.  A paired t-test was performed (p = 0.0023). The analysis showed a 30% increase in frequency when adding cAMP compared to control settings.  The error bars on the figure visualize the 95% confidence interval.
						</P><H4>Conclusion</H4><P style="text-align:justify; padding: 0px 50px 0px 50px;">
						cAMP has been shown to stimulate HCN in cardiac cells, which causes a faster depolarization phase and subsequently a faster rhythm. Our statistical analysis demonstrated that this hypothesis also holds true for the HEKciting cells.
						</P><H2>Future directions</H2><H3>Stable cell line</H3><P style="text-align:justify; padding: 0px 50px 0px 50px;">
The biggest drawback of our current project is the variability of gene expression in our manipulated cell line. Since we only transfect in a transient manner, the exact concentration of the DNA containing our ion channels differs slightly with every transfection. In the future, a stable cell line containing the three ion channels of interest could reduce this variability. The cells would all oscillate at the same frequency, causing a more coherent rhythm. 

</P><H3>Extra ion channel modulation</H3><P style="text-align:justify; padding: 0px 50px 0px 50px;">A logical next step could be to introduce a fourth ion channel to the system. For example the temperature-sensitive ion channel TRPV1, which activates at a lower temperature. Influencing the rhythm by a fourth ion channel would increase the amount of applications by not having to pick a drug that interacts directly with the three original ion channels. 

By introducing Designer Receptors Exclusively Activated by Designer Drugs (DREADDs) for example, we can influence the depolarizing rate by a designer drug that has no effect on the human body, such as clozapine-N-oxide (CNO). This biological sensor could be used to control drug adherence in patients by giving CNO on top of their medication, followed by quantification of the rhythm of the cells. Other channels such as KATP channels could be used to indirectly measure extracellular glucose concentratios. These examples provide only a few possibilities that an extra ion channel could provide.</P><H3>Receptor modulation</H3><P style="text-align:justify; padding: 0px 50px 0px 50px;">We created a biosensor that enables us to sense real-time changes of drug concentration in the blood, by measuring the frequency of our manipulated HEK cells. As we have demonstrated, cAMP clearly affects the rhythms. We could influence cAMP in the cells indirectly by coupling the cascade to any GPCR protein, such as the dopamine receptor.
Dopamine-agonists, often used in Parkinson’s disease, would benefit greatly from rapid and continuous quantification in the blood.

						
						Thus, our therapeutic drug monitoring device could be generalized for numerous types of drugs that need a close follow-up. 
						</P><H3>Genetic link</H3><P style="text-align:justify; padding: 0px 50px 0px 50px;">A genetic link could infinitely expand the amount of applications. By coupling ion channels to an inducible promoter and having a short half-life, we could influence the rhythm by adjusting the expression level of the ion channel of interest. Coupling one, two or more ion channels to promoters would provide us with endless options to modulate the frequency of the oscillating cells.</P><H3>Considerations for replicating the experiments</H3><P style="text-align:justify; padding: 0px 50px 0px 50px;">
						We made an inquiry about the type of cells we would use. HEK 293 cells, like the ones we created a sinus rhythm in, are known to have a high growth rate. This would result in the cells quickly filling the capsule, which would cause them to starve and die. Furthermore, they contain adenoviral DNA. The possible risks of an infection with an adenovirus and its effect on cells with adenoviral DNA present in the body are not yet known. This is why cells containing foreign DNA are not allowed to be used in humans. These two factors reduce the applicability of our HEK cells strictly to this proof of concept study. For this reason, we started looking for other cells.

						
						As noted by professor Cosson, the key decision criteria for choosing new cells are: (1) the ease with which they are engineered, (2) the ability to store them in vitro in large batches, (3) the ease to be cultured in cheap media, and finally (4), the cells need to be in biosafety category 1 to comply with regulation. Two cell types that meet these requirements, human myoblasts and ARPE-19 cells, were investigated to be used in the finalized capsule. Different studies already confirm that using human myoblasts in capsules is possible. However additional research is still needed to further improve the viability of these cells. Secondly, the adult human retinal pigment epithelium (ARPE-19) cells have also been tested in capsules. These cells appeared to be sturdy, have a long life and have a good viability within the capsule. However, human myoblasts and ARPE-19 cells are only examples of multiple other options, such as non-stem cells: fibroblasts, chondrocytes, osteoblasts, adipocytes and skeletal muscle progenitors. Also, different stem cells such as human mesenchymal stem cells (hMSCs), human embryonic stem cells (hESCs) and hESC-derived mesoderm progenitors could be interesting to transfect.

						</P><H4>References</H4><P style="text-align:justify; padding: 0px 50px 0px 50px;">
						Li, A., Bourgeois, J., Potter, M. and Chang, P. (2007). Isolation of human foetal myoblasts and its application for microencapsulation. Journal of Cellular and Molecular Medicine, 12(1), pp.271-280.
                                                
						Tao, W., Rein, D., Dean, B., Stabila, P. and Goddard, M. (2002). ARPE-19 AS A PLATFORM CELL LINE FOR ENCAPSULATED CELL BASED DELIVERY. US 6,361,771B1.
						
						Wen, J., Xu, N., Li, A., Bourgeois, J., Ofosu, F. and Hortelano, G. (2007). Encapsulated human primary myoblasts deliver functional hFIX in hemophilic mice. The Journal of Gene Medicine, 9(11), pp.1002-1010.
						</P></DIV></DIV><FOOTER><CENTER><DIV class="container" style="margin-bottom:20px;"><UL class="row" id="sponsors"><H1>Sponsors and Partners:</H1></UL></DIV></CENTER><DIV class="container-fluid social" style="padding: 0px;"><DIV class="col-md-3" id="facebook"><A href="https://www.facebook.com/KULeuveniGEM" style="padding-right: 0px;"><H2 style="color:white;">Facebook</H2></A></DIV><DIV class="col-md-3" id="twitter"><A href="https://twitter.com/kuleuvenigem" style="padding-right: 0px;"><H2 style="color:white;">Twitter</H2></A></DIV><DIV class="col-md-3" id="youtube"><A href="https://www.youtube.com/channel/UCvIKudANB5NvCR5S8XJn78g" style="padding-right: 0px;"><H2 style="color:white;">Youtube</H2></A></DIV><DIV class="col-md-3" id="instagram"><A href="https://www.instagram.com/kuleuvenigem/" style="padding-right: 0px;"><H2 style="color:white;">Instagram</H2></A></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>