Team:KU Leuven/Parts
Toggle navigation
HOME
TEAM
Team
Collaborations
Sponsors
PROJECT
Description
Design
Protocols
Notebook
Logbook
InterLab
Model
Demonstrate
Attributions
PARTS
Parts
Basic Parts
SAFETY
HUMAN PRACTICES
Human Practices
Silver
Integrated and Gold
Education
Public Engagement
AWARDS
Applied Design
Entrepreneurship
Prizes
Model
JUDGING FORM
Parts
We have designed several new biobricks related to electrophysiology, which will enable future teams to develop novel projects in this field. They are ion channels, adapted to conform to the biobrick standard.
For our project, ion channels were crucial. This is why we added four new ion channel-biobricks to the registry. You can read more about them on this page, or on the registry pages themselves.
On the left, we show the system we developed: A combination of HCN, hERG and α1G channels lead to oscillating cells. Based on our model, we suspected that the characteristics of these ion channels were crucial for our system. Therefore, we made an alternative system, where the hERG channel was replaced by another potassium channel, Kir2.1. As expected, the latter system did not show oscillations.
Biobrick 1: mHCN2
Hyperpolarization-activated cyclic nucleotide–gated (HCN) channels are a family of ion channels responsible for the generation of rhythmic activity in heart and nerve cells. Consequently, they are often nicknamed pacemaker channels. They show several unique characteristics: first of all, the channels are opened in response to hyperpolarisation instead of depolarisation. Furthermore, they are not completely selective for K+-ions, and as a result, there is a small Na+ inward current which causes a slow depolarisation of the membrane, the so-called funny current or pacemaker depolarisation. These attributes cause the particular rhythmic behaviour of cells containing HCN channels, and as a result, an HCN channel is vital for our project.
In our project, we have created a biobrick based on the HCN2 isoform found in mice. The gene has been adapted to conform to the biobrick standard, meaning that several prohibited restriction sequences have been deleted by silent mutations. Furthermore, we have lowered the GC content of the protein, to enable production of the gene as an IDT gBlock for use in our and in future projects.
More information can be found on the registry page.
Biobrick 2: hERG
Another gene that has played a major role in our project is the human Ether-à-go-go-Related Gene, also known as hERG and, alternatively, as KCNH2. Like HCN, it is associated with the rhythm generated in natural sinus node cells. It encodes the K+-channel responsible for the repolarisation during the cardiac action potential. Furthermore, the hERG-encoded protein Kv11.1 is of great medical and pharmaceutical importance, as inhibition of this protein is associated with the dangerous long QT syndrome, which may result in fainting, seizures and even sudden death. Therefore, hERG inhibition is a major antitarget in drug development, and all drugs have to be tested for hERG inhibition in early development.
We have created a biobrick based on the hERG gene, which has allowed us to replicate the sinus rhythm in HEK-293 cells. The biobrick has been developed based on the human mRNA isoform 1 sequence, which we have adapted to conform to the biobrick standard. In order to achieve this standard, multiple illegal restriction sites have been removed by silent mutations. We think this biobrick might be useful for future projects in the field of electrophysiology, but also for projects with a pharmaceutical aspect, as testing for hERG inhibition is an important aspect of drug development.
More information can be found on the registry page.
Biobrick 3: Kir2.1
Kir2.1 is an inwardly rectifying potassium channel, encoded by the kcnj2 gene. This ion channel is an interesting target for many interesting studies, as it has already been shown that Kir2.1 influences the electrophysiological properties when expressed in HEK-293 cells, resulting in a stable membrane potential of approximately -70mV, instead of the -30 to -40 mV typically found in non-transfected cells.
In our project, we used a model to indicate which ion channels could result in the oscillating system we present in the HEKcite project. To verify that we could not just use any ion channel, we made an analog system using Kir2.1. As expected, this system did not result in oscillation, which demonstrated the importance of our model.
This biobrick has been designed based on the mRNA sequence of the kcnj2 gene found in mice. In our design, we have removed several forbidden restriction sites using silent mutations.
More information can be found on the registry page.
Biobrick 4: α1G
α1G, which is also known as CACNA1G or Cav3.1, is the α1G subunit of the calcium voltage-gated channel. In our system, it is functional as a depolarising channel which allows the electrophysiological oscillations to occur. Furthermore, its functionality as a calcium channel allowed us to visualise a rhythm using calcium imaging in one of our earliest experiments.
Even though we were excited to deliver an α1G biobrick to the depository, we have unfortunately been unable to adapt and clone this ion channel to follow the biobrick standards. We suspect these troubles are largely the result of the large size of this gene, as it is almost 7000 nucleotides in length.
More information can be found on the registry page.
During our project, we used all ion channels in pIRES vectors. These are mammalian expression vectors, followed by an internal ribosome entry site (IRES) and a fluorescent marker. In such vectors, the part and the marker are transcribed at the same time, but translated independently. In such a system, the marker indicates that the part is transcribed in the cell, but they are not physically linked. This is beneficial, as a linked marker protein may affect or hinder the protein function.
All information is summarised in the following table:
<groupparts>iGEM17 KU_Leuven</groupparts>
Sponsors and Partners:
Facebook
Twitter
Youtube
Instagram
