"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins-Software/AutoPlasmid
From 2012.igem.org
AutoPlasmid
AutoPlasmid was developed to take the process of annotating a plasmid to a whole new level. Firstly, we wanted to create a centralized depot of many thousands of well-known features and their associated sequences. We chose what we thought were some of the more popular features, and relied on databases such as SGD, PlasMapper, and the University of Wisconsin Madison's E. Coli Database.
But we knew this database would grow as large as it did. To address this concern, we implemented Cloud Computing , what we feel is one of the most impressive aspects of our software application. A plasmid such as puc18 normally takes 39 minutes on a slow computer to annotate and search through all our features. But in the cloud, with 30 parallel processes running at once, we can split up the algorithm and divvy up the work, resulting in a run of just 68 seconds, a dramatic decrease in time by 34-fold!
But we added many, many more features on top of this architectural backend. In particular, we gave users the ability to annotate their plasmid using imperfect matches selecting any threshold they want (e.g. 95% match). While this significantly increases the time a particular annotation takes, it still results in rapidly fast annotations when the Cloud is used. In addition, users may select particular features they want to search for, such as Genes, Promoters, Terminators, and more. Lastly, users may translate DNA into amino acid in all 6 reading frames.
Once a plasmid is fully annotated, the user can do even more. They can look inside the plasmid, view details about each feature that was annotated, and manipulate the plasmid even more. They can add custom annotations, view where a particular segment of DNA is, and even isolate out features to later design with. Users can view oligo matches, and amino acid translation of particular DNA segments. One of the most useful features of the AutoPlasmid's plasmid view window is that a user can find exactly why a annotation was identified as an imperfect match. In puc18, it is well known that the origin has a 1 base pair mutation, for example. This can be easily located by viewing the details of the alignment.
Lastly, a user may export their annotated plasmid in any format they'd like: genbank, fasta, or SBOL . These standardized file formats are useful when using AutoGene in collaboration with other standard softwares, such as Ape. Our genbank format, in particular, preserves characteristics of annotations that may be crossed over to Ape, such as the color of each annotation.
Database
One of the key issues we recognized with the Autogene was the organization of the database. The structure would have to be robust to help encompass the different types of feature type annotations (terminator, promoter, coding sequence, etc.), organism taxonomy, and custom annotations made by the user. Therefore, there was a need to:
     - standardize the different feature type names
     - standardize organism taxonomy
     - keep the database flexible and capable of managing new types of features
To this end, we looked for current standards for feature type annotations and organism taxonomic identification, and found the sequence ontology project as a reliable source for feature annotations and NCBI’s taxonomy database as a source of organism taxon identification.
We obtained gene sequences from the University of Wisconsin Madison's E. Coli Database , the Saccharomyces Genome Database , Virulence Factors of Pathogenic Bacteria , and the Registry of Standard Biological Parts .
Implementation
PlasmidView: this is the AutoPlasmid window. On the left is a picture of the plasmid. On the right is the DNA and the list of features contained inside that plasmid. Using the many buttons in the toolbar on the right side, or by clicking annotations in the picture of the plasmid, the user may interact with AutoPlasmid in many ways described above.
Plasmid List: this is where AutoGene keeps track of the Plasmids that the user is annotation and/or designing with throughout their use of AutoGene. Anytime a plasmid is imported, it pops into the Plasmid List so that a user may easily open it at any time.
Private Registry of Features: this is where AutoGene keeps track of features that users would like to design with. A user drags features from the PlasmidView into the Private Registry as a type of Bookmarking utility. It is a way to quickly access features to later use in designing plasmids.
Log: this is AutoGene's log, which gives warnings or error messages to the user. For example, if the user tries to import a DNA sequence with incorrect characters. It also will notify users of tasks that are running, such as annotations (and features that have been found) or DNA inverting.
Using AutoPlasmid
First, hand AutoPlasmid a sequence of DNA, however big or small you'd like. Import a file or copy and paste the sequence.
Then get the annotation loaded and started. AutoPlasmid will search through a database containing 40,000 features and look for matches. Not only does it find perfect matches, but if you specify, you can search for imperfect matches with any threshold you'd like (ex. 90% match).
After searching for annotations, open your plasmid to look inside.
Interact with it by selecting features, manipulating the DNA, and adding any custom annotations you'd like. You can even view amino acid translations or oligo matches.
Lastly, you can view the details of an imperfect match.
AutoPlasmid is also compatible with a variety of standard biology tools, such as ApE. A plasmid can be imported as a fasta, gb, and SBOL file, as well as being exported as a fasta, gb, or SBOL file.
Team
Members
Attributions
Sponsors
Software
Overview
AutoPlasmid
AutoDesign
Cloud
Compatibility
Screencast
Download
Requirements
Medals
Safety
Collaboration
Human Practices
Retrieved from " http://2012.igem.org/Team:Johns_Hopkins-Software/AutoPlasmid "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
