"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins-Software/theSoftware overview
From 2012.igem.org
Overview
AutoGene is an innovative plasmid design suite meant to streamline the processes of annotating and building sequences. Written in Java and Python and consisting of two modules -- AutoPlasmid and AutoDesign -- it first uses a highly curated database of features to search an imported plasmid, scanning the sequence for both perfect and imperfect alignments, and then generating an interactive visualization of the annotated plasmid. Once a plasmid is annotated, a user is then able to alter its contents, using the AutoGene feature database, the biobrick database, as well as custom components. Still in progress, the design module enables structural optimization by maintaining sets of sequence rules and taking an algorithmic approach to minimizing structural violations. Additionally the program reduces the problems caused by restriction sites during application of designs, and proposes the most suitable enzyme selections through an analysis of standard restriction sites libraries.
The processes will be run on the cloud as a standard web server gateway interface service to dramatically increasing its speed and accessibility while maintaining the same level of accuracy. As a cloud service, it will run its algorithms in parallel and thus have the ability to perform more computationally intense procedures, such as optimizing codons to increase gene expression, and designing the most efficient oligonucleotide sequences for PCR assembly.
Furthermore, AutoGene has the potential for integrating fabrication or design software in the future. 3D visualization techniques could provide users with views of protein structures created from the sequences or that of a related ortholog. The process for fabrication of synthetic DNA sequences could be automated through building block design, helping the the user to assemble their oligonucleotides. A number of additional features, such as evaluating overlapping segments of oligonucleotides, and evaluating melting temperatures for PCR assembly, may also be built on top of this project.
Using a clear user interface, a large, highly curated feature database, and through harnessing the power of cloud computing, we hope to simplify the process of plasmid design. AutoGene aims to make it faster and easier for a user to both identify and modify the contents of any plasmid, enhancing sequences for greater viability and expression.
Software Goals
We recognized a need for a comprehensive piece of software that provided the means to achieve an easier Plasmid Design process. Manual annotation is simply too time consuming, tedious, and error-prone. We wanted to create something to fix this noticeable setback of existing synthetic biology software. AutoGene sets a new technological standard by bringing automation to the plasmid design process. That being said, we developed a list of specific goals we wished to achieve. We split these goals into two modules, and decided the first set of goals would be achieved by AutoPlasmid, the second set of goals would be achieved by AutoDesign, and together, we would create AutoGene.
Each of these goals encompasses an extensive list of sub-goals. Here, we discuss the more detailed goals for our primary milestones listed above.
Future Goals
Autogene is still evolving as we both finish up the design module, and fine tune the plasmid mapper. There are several features we’d like to develope in the future, including 3d protein visualization using MAYA, codon optimization, and primer design.
Codon optimization is a common synthetic biology technique used to improve the efficiency of protein translation. DNA is translated into amino acids in groups of three bases called codons, but only the first two bases determine into which amino acid the codon in translated. This allows researchers to change this third letter without affecting the resulting proteins. When a codon is ‘optimized’, this means that the codon is replaced with a version producing the same protein as the original, but which is more common in the organism’s DNA. If the codon is more common, then the thought is that it can be produced more efficiently.
Primer design is one of the most common computational tasks in synthetic biology. In order to synthesize a DNA sequence, it is necessary to break the total sequence into small, overlapping parts, which are then combined using PCR. Once we finish this feature, Autogene will truly encompass all portions of plasmid design, using the simple philosophy of Scan/Edit/Print. From the annotation of the starting sequence, to the gathering and arranging of desired features, to the development of building blocks to actually synthesize the given sequence, everything will be done through Autogene.
Autogene is the first tool of its kind in many ways, and we hope that its development will demonstrate the potential of many new techniques and technologies when applied to synthetic biology. By exporting our alignments to the cloud, we’ve shown that complex computation, the kind that before could only be performed on very high-powered computers, can be accessed by anyone with an internet connection. An increase in computational power for every researcher across the board could revolutionize synthetic biology, and scientific research in general.
Team
Members
Attributions
Sponsors
Software
Overview
AutoPlasmid
AutoDesign
Cloud
Compatibility
Screencast
Download
Requirements
Medals
Safety
Collaboration
Human Practices
Retrieved from " http://2012.igem.org/Team:Johns_Hopkins-Software/theSoftware_overview "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
