"
Page
Discussion
View source
History
teams
Log in
 
Team:Paris/Modeling/Protocol Of Characterization
From 2008.igem.org
Project
Analysis
Construction
Characterization
Perspectives
Protocol of Characterization
Contents
1 I-Principles of the Experiments
2 II-Plasmid for promoter characterization
2.1 II-A-For study with one Transcription Factor
2.2 II-B-For study with two Transcription Factor
3 III-Molecular design for Promoter Characterization Plasmid
4 IV-Promoter and Transcription Factors insertion
5 V-Resulting plasmids
5.1 V-A-For one transcription factor effect
5.2 V-B-For two transcription factors effect
6 VI-Protocole for plasmid construction
6.1 VI-A-Principal Plasmid construction
6.1.1 VI-A-1-Blocks
6.2 VI-B-Accessory plasmid construction
7 Specific Assumptions for the Experiments
I-Principles of the Experiments
To evaluate quantitativly the activity of a promoter in function of its transcription factors, we need data in which the different values of the activities are correlated with various known and controlled values of the transcription factors concentrations.
Therefore, we designed a generic plasmid in which the transcription factors are put under the control of previously characterized inducible promoter, and the studied promoter is put before a fluorescent reporter gene. In order to allow the study of the influence of two transcription factors over the tested pormoter, we chose to put the tested transcription factor (also called the "tested gene" in the following) under two different inducible systems . One is the pBAD-AraC system. The second one is an indirect system were the gene is after the Tet inducible promoter pTet.
The TetR gene would be expressed constitutively and at high rate thanks to a strong promoter (J23101) and its influence over the pTet promoter would be regulated by the concentration of the aTc molecule. That way, the production of the tested transcription factor can also be regulated, because the J23101, and the pTet have been previously characterized. In a quite similar way, we characterize the pBad promoter.
Here we show the design of two plasmids : one to test the influence of one gene and the other to test the influence of two genes over the tested promoter.
II-Plasmid for promoter characterization
II-A-For study with one Transcription Factor
II-B-For study with two Transcription Factor
III-Molecular design for Promoter Characterization Plasmid
Our aim is to make this plasmid useful not only for our project but for the whole iGEM comunity. This is why we decided to keep the Biobrick spirit as much as we could, making the plasmid compatible with the parts, so the teams using it needs only the four traditional enzymes: EcoRI, XbaI, SpeI and PstI.We wanted also to make optional the introduction of a second tested transcription factor.  The strategy is then based in two plasmids.
The principal plasmid contains everything needed to test the effect of one gene over the tested promoter activity. The second plasmid called « Accessory plasmid » can be introduced easily in the Principal Plasmid and contains the necessary elements to add the expression of a second gene to the system. The resulting plasmids are presented below.
↓ III-A-Principal plasmid ↑
↓ III-B-Accessory plasmid ↑
IV-Promoter and Transcription Factors insertion
The strategy to introduce the tested promoter and the tested transcription factor(s) is very simple. The only difficulty is that the order of insertion that we describe has to be respected to avoid unwanted restriction enzyme cuts:
↓ The Steps ↑
1st step: Introduce the tested promoter:
-Cut Promoter with EcoRI and SpeI -Cut Principal plasmid with EcoRI and XbaI
2nd step: Introduce the 1st tested transcription factor:
-Cut Gene with XbaI and SpeI -Cut Principal plasmid+Promoter with SpeI -Check the right orientation of the 1st gene by PCR with appropiated primers
If wanted…
3rd step: Introduce the 2nd gene into the Accessory plasmid
-Cut Gene with XbaI and SpeI -Cut Accessory Plasmid with XbaI and SpeI -Check the right orientation of the 2nd gene by PCR with appropiated primers4
4th step: Introduce the 2nd gene expression system into the Principal Plasmid:
-Cut Accessory Plasmid+2nd gene with PstI -Cut Principal plasmid+Promoter+1st gene with SpeI -Check the right orientation of the 2nd gene expression system by PCR with appropiated primers
↓ Illustration ↑
V-Resulting plasmids
V-A-For one transcription factor effect
V-B-For two transcription factors effect
VI-Protocole for plasmid construction
We show here our plan to make these plasmids at the experimental level.
VI-A-Principal Plasmid construction
We devided the Principal Plasmid in two main blocks to go faster:
VI-A-1-Blocks
↓ VI-A-2-Block 1 construction ↑
the result...
↓ VI-A-3-Block 2 construction ↑
Biobrick assembly steps:
Restriction sites addition by PCR:
↓ VI-A-4-Two-block assembly ↑
VI-B-Accessory plasmid construction
Addition of restriction sites to B0015 by PCR:
↓ Scheme ↑
Addition of restriction sites to pBAD-AraC by PCR:
↓ Scheme ↑
Assembly:
↓ Scheme ↑
Specific Assumptions for the Experiments
The previous experiment is used to find parameters regarding to our modelization. However, the experiments themselves must be described in the same way to interprete these parameters consistently.
In this order, the two inductible promoters will be described exactly as the others (those in the system), with similar rules of complexation and transcription.
Otherwise, as we control the quantity of transcription factor by the introduction of small molecules, we must model the diffusion/complexation/induction phenomena, which lead from the latter to the former.  By putting the cell culture in a chemostat, we will be able to control precisely the concentration of the input small molecules. At quasi steady-state, and assuming a simple, passive and quick diffusion, we consider the inner concentration, the outer medium concentration, and the input concentration beeing all equal.
↓ Quick Mathematical Development ↑
Thus, if we consider these "rules"
it gives at quasi steady-state
resulting in the following analytical expression of [Compl] and [Prot] :
...that we can introduce in the expression of the activity of pTet and pBad in function of their transcription factor.
Thanks to these considerations, we can have access to these characterizations :
↓ Prior characterizations ↑
0 = Act_J23101
1 ( [aTc]i, [TetR] ) = Act_pTet
Inv_1 ( [TF]required ) = [aTc]needed
2 ( [arab]i, [AraC] ) = Act_pBad
Inv_2 ( [TF]required ) = [arab]needed
and then
↓ System characterizations ↑
3 ( [FliA], [OmpR*] ) = Act_pFlhDC
3bis ( [FliA], [EnvZ] ) = Act_pFlhDC
4 ( [FlhDC], [FliA] ) = Act_pFliA
5 ( [FlhDC], [FliA] ) = Act_pFliL
6 ( [FlhDC], [FliA] ) = Act_pFlgA
7 ( [FlhDC], [FliA] ) = Act_pFlgB
8 ( [FlhDC], [FliA] ) = Act_pFlhB
It is important to follow the previous chronological order for the successive characterizations, because some results are reused in the next !
<Back to "Workflow on an Example" |
Retrieved from " http://2008.igem.org/Team:Paris/Modeling/Protocol_Of_Characterization "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
