<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Notebook_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Paris/Notebook/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris/Notebook/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris/Notebook/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris/Notebook/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris/Notebook/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris/Notebook/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="menu"><UL><LI><A href="https://2008.igem.org/Team:Paris/Project">Project</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Analysis">Analysis</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Construction">Construction</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Characterization">Characterization</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Perspectives">Perspectives</A></LI></UL></DIV><TABLE cellspacing="5" cellpadding="10" style="margin-top: 3px; background:#649CD7; width: 965px;"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Culture_of_Stable_strain_with_biobricks_2008"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Culture of Stable strain with biobricks 2008</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Glycerol_Stocks"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Glycerol Stocks</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Minipreps_.28QIAGEN_kit.29"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Minipreps (QIAGEN kit)</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Electrophoresis"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Electrophoresis</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Concentration_of_the_Miniprep_or_the_Midiprep"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Concentration of the Miniprep or the Midiprep</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Digestion"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Migration_after_digestion"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Migration after digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Extraction"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Extraction</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Antarctic_phosphatase"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Antarctic phosphatase</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Amplification_of_promoters"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">Amplification of promoters</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Purification_.28PROMEGA_or_QIAcube_or_QIAquick_kit.29"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Purification (PROMEGA or QIAcube or QIAquick kit)</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Gel_Slice_and_PCR_Product_Preparation"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Gel Slice and PCR Product Preparation</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Dissolving_the_Gel_Slice"><SPAN class="tocnumber">11.1.1</SPAN><SPAN class="toctext">Dissolving the Gel Slice</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Processing_PCR_reactions"><SPAN class="tocnumber">11.1.2</SPAN><SPAN class="toctext">Processing PCR reactions</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-15"><A href="#Binding_of_DNA"><SPAN class="tocnumber">11.2</SPAN><SPAN class="toctext">Binding of DNA</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Washing"><SPAN class="tocnumber">11.3</SPAN><SPAN class="toctext">Washing</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Elution"><SPAN class="tocnumber">11.4</SPAN><SPAN class="toctext">Elution</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Quantification_by_Electrophoresis"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Quantification by Electrophoresis</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Ligation"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Transformation"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#LB_and_LBA"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">LB and LBA</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Minimum_Medium"><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">Minimum Medium</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Soft_Agar_Medium"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">Soft Agar Medium</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#1000x_stock_antibiotic"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">1000x stock antibiotic</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#PCR_Screening"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">PCR Screening</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Electrophoresis_Purification_of_PCR"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">Electrophoresis Purification of PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Sequencing"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">Sequencing</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#Promoter_Characterization_Plan"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">Promoter Characterization Plan</SPAN></A></LI><LI class="toclevel-1 tocsection-29"><A href="#Protocol_to_make_competent_bacteria"><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">Protocol to make competent bacteria</SPAN></A></LI><LI class="toclevel-1 tocsection-30"><A href="#Study_of_the_doubling_time_of_the_bacteria_population"><SPAN class="tocnumber">24</SPAN><SPAN class="toctext">Study of the doubling time of the bacteria population</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Culture_of_Stable_strain_with_biobricks_2008"> Culture of Stable strain with biobricks 2008 </SPAN></H1><UL><LI> Streaks on plates with LB and the adapted antibiotics to isolate colonies
</LI><LI> Incubate O/N at 37°C
</LI><LI> Take clone with a sterile tip and put in 7.5ml LB with adaptated antibiotics (LB have to be prepared before !!!)
</LI><LI> Will be use for Miniprep and Stock in glycerol
</LI><LI> 2-3 clones isolated by Biobricks
</LI><LI> Incubate O/N at 37°C (open just a little the cap for bacteria's breathing)
</LI><LI>Don't store LB in the fridge even with antibiotics !
</LI></UL><H1><SPAN class="mw-headline" id="Glycerol_Stocks"> Glycerol Stocks </SPAN></H1><UL><LI> Remove 2.5mL of each culture and centrifuge.
</LI><LI> Discard the supernatant and resuspend pelleted bacteria in 1mL of LB.
</LI><LI> Add 500µL of 60% glycerol.
</LI><LI> Store at -20°C.
</LI></UL><H1><SPAN class="mw-headline" id="Minipreps_.28QIAGEN_kit.29"> Minipreps (QIAGEN kit) </SPAN></H1><UL><LI> centrifuge  5mL of culture 8 min at 3,500 to 4,000 g.
</LI><LI> Resuspend pelleted bacterial cells in 250 µl Buffer P1 and transfer to a microcentrifuge tube. 
</LI><LI> Add 250 µl Buffer P2 and mix thoroughly by inverting the tube 4–6 times. If using LyseBlue reagent, solution turns blue. 
</LI><LI> Add 350 µl Buffer N3 and mix immediately and thoroughly by inverting the tube 4–6 times. If using LyseBlue reagent, solution turns colorless. 
</LI><LI> Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge. 
</LI><LI> Apply the supernatant (from step 4) to the QIAprep spin column by decanting or pipetting. 
</LI><LI> Centrifuge for 30–60 s. Discard the flow-through. 
</LI><LI> Recommended: Wash the QIAprep spin column by adding 0.5 ml Buffer PB and centrifuging for 30–60 s. Discard the flow-through. This step is only required when using endA+ or other bacteria strains with high nuclease activity or carbohydrate content (see QIAprep Miniprep Handbookfor more details) 
</LI><LI> Wash QIAprep spin column by adding 0.75 ml Buffer PE and centrifuging for 30–60 s. 
</LI><LI> Discard the flow-through, and centrifuge for an additional 1 min to remove residual wash buffer. 
</LI><LI> To elute DNA, place the QIAprep column in a clean 1.5 ml microcentrifuge tube. Add 30 µl Buffer EB or pure water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.
</LI></UL><H1><SPAN class="mw-headline" id="Electrophoresis"> Electrophoresis </SPAN></H1><P>An electrophoresis can be done to check if there is Product of Miniprep
</P><UL><LI> Gel : 1% agarose with BET added (5 µL BET for 100 mL TBE)
</LI><LI> 10 µL Quick-Load 1 kb DNA Ladder
</LI><LI> 2 µL Loading Dye + 3 µL DNA
</LI></UL><H1><SPAN class="mw-headline" id="Concentration_of_the_Miniprep_or_the_Midiprep"> Concentration of the Miniprep or the Midiprep </SPAN></H1><P>By biophotometry
</P><UL><LI> Blank : 60 µL of pure water  or 55 µL of pure water + 5 µL EB
</LI><LI> Sample : 55 µL of pure water + 5 µL of sample
</LI></UL><P>Check if the ratio 260/280 is over 1,6
</P><P><B>Think about the dilution !</B></P><P>If the concentration is to low, concentrate the sample in a vacuum centrifuge. Don't forget to open the tubes!
</P><H1><SPAN class="mw-headline" id="Digestion"> Digestion </SPAN></H1><UL><LI> 1 µg of plasmid / 250 ng of PCR product
</LI><LI> Buffer (n°2) 10X : 3µL
</LI><LI> BSA 100X : 0.3µL
</LI><LI> 1 µL of each enzyme
</LI><LI> Pure water qsp 30 µL
</LI><LI> Incubate during about 2h30-3h at 37°C, then 20 minutes at 65°C (to inactivate the enzymes).
</LI></UL><H1><SPAN class="mw-headline" id="Migration_after_digestion"> Migration after digestion </SPAN></H1><UL><LI> Run the whole samples (30 µL) in a <B>1-2% agarose gel with TAE buffer</B> (a new one) 
</LI><LI> Run at 50 V until halfway  
</LI><LI> 10 µL of ladder 1kb and 100 pb on every side
</LI><LI> 30 µL of DNA + 6 µL of Loading Dye
</LI></UL><P><B>Separate each sample by an empty well !</B></P><H1><SPAN class="mw-headline" id="Extraction"> Extraction </SPAN></H1><UL><LI> For each new extraction it's important to have a new bath of TAE Buffer for the run and of ETB for the revelation
</LI><LI> Use UV at lower intensity and for shorter time as possible
</LI><LI> Use a new blade for each extraction
</LI><LI> The band weight must be less than 400 mg
</LI></UL><H1><SPAN class="mw-headline" id="Antarctic_phosphatase"> Antarctic phosphatase </SPAN></H1><P>To avoid autoligation, the vector in a construction has to be treated with phosphatase to dephosphorylate the cut end of the plasmid. The Antarctic phosphatase has the advantage to be easily and totally inactivated by heating.
</P><UL><LI> After the digestion reaction add Antarctic phosphatase buffer to a final concentration of 1x
</LI><LI> Add 1µL of Antarctic phosphatase for 1 to 5 µg of DNA
</LI><LI> Incubate at 37°C for 15 minutes and then inactivate the phosphatase at 65°C for 5 min
</LI></UL><P><B>Do not use this reaction for the insert or the ligation will not work</B></P><H1><SPAN class="mw-headline" id="Amplification_of_promoters"> Amplification of promoters </SPAN></H1><P>(to amplify the sequence in order to have enough amount of DNA to carry out the following of our experiments)
</P><UL><LI><B>Preparation of the templates</B> : Resuspend of 1 colony  in 100µl of water.
</LI><LI><B>Preparation of PCR mix</B> :
</LI></UL><P>For each samples
1 µl dNTP
10 µl Buffer Phusion 5x
2,5 µl Oligo_F
2,5 µl Oligo_R
1µl template
1 µl Phusion
50 µl qsp H2O (33µl)
</P><UL><LI> make a mix with buffer, oligos and water for n+1 samples
</LI><LI>negative control : without template
</LI><LI>positive control : known template
</LI></UL><UL><LI>Program PCR : <B>PHUSION2</B></LI></UL><P>LID : 105°C
1. 98°C   30 sec   initial denaturation
2. 98°C   10 sec   denaturation
3. 60°C (depending of the size of oligos)   30 sec  annealing
4. 72°C   (1 min for 1 kb) 
5. go to : 2   rep : 24-29
6. 72°C   5 min
7. sound : 1
8. hold : 10°C
</P><H1><SPAN class="mw-headline" id="Purification_.28PROMEGA_or_QIAcube_or_QIAquick_kit.29"> Purification (PROMEGA or QIAcube or QIAquick kit) </SPAN></H1><H2><SPAN class="mw-headline" id="Gel_Slice_and_PCR_Product_Preparation"> Gel Slice and PCR Product Preparation </SPAN></H2><H3><SPAN class="mw-headline" id="Dissolving_the_Gel_Slice"> Dissolving the Gel Slice </SPAN></H3><UL><LI> Following electrophoresis, excise DNA band from gel slice in a pre-weighed 2 mL microcentrifuge tube.
</LI><LI> Add 10µL membrane Binding Solution per 10 mg of gel slice. We prefer not to vortex and we incubate at 50-65°C until gel slice is completely dissolved (∼10 min). Quick centrifuge.
</LI></UL><H3><SPAN class="mw-headline" id="Processing_PCR_reactions"> Processing PCR reactions </SPAN></H3><P>For products above 40 pb
</P><UL><LI>Add an equal volume of Membrane Binding Solution to the PCR reaction.
</LI></UL><H2><SPAN class="mw-headline" id="Binding_of_DNA"> Binding of DNA </SPAN></H2><UL><LI> Insert the SV Minicolumn into Collection Tube.
</LI><LI> Transfer dissolved gel mixture or prepared PCR product to the Minicolumn assembly. Incubate at room temperature for 1 minute.
</LI><LI> Centrifuge at 16,000 x <I>g</I> for 1 minute. Discard the flowthrough and reinsert Minicolumn into Collection Tube.
</LI></UL><H2><SPAN class="mw-headline" id="Washing"> Washing </SPAN></H2><UL><LI> Add 700 µL Membrane Wash Solution (ethanol added). Centrifuge at 16,000 x <I>g</I> for 1 minute. Discard flowthrough and reinsert the Minicolumn into Collection Tube.
</LI><LI> Repeat Step 4 with 500 µL Membrane Wash Solution. Centrifuge at 16,000 x <I>g</I> for 5 minutes.
</LI><LI> Empty the Collection Tube and recentrifuge the column assembly for 1 min with the microcentrifuge lid open (or off) to allow evaporation of any residual ethanol.
</LI></UL><H2><SPAN class="mw-headline" id="Elution"> Elution </SPAN></H2><UL><LI> Carefully transfer Minicolumn to a clean 1.5 mL microcentrifuge  tube.
</LI><LI> Add 30 µL of pure water or buffer EB (Qiagen). Incubate at room temparature for 1 minute. Centifuge at 16,000 x <I>g</I> for 1 minute.
</LI><LI> Discard Minicolumn and store at 4 or -20°C.
</LI></UL><H1><SPAN class="mw-headline" id="Quantification_by_Electrophoresis"> Quantification by Electrophoresis </SPAN></H1><UL><LI> Gel : 1.5-2% agarose with BET added (5 µL BET for 100 mL TBE)
</LI><LI> 10 µL Quick-Load 100pb or 1 kb DNA Ladder
</LI><LI> 2 µL Loadind Dye + 3 µL DNA
</LI></UL><H1><SPAN class="mw-headline" id="Ligation"> Ligation </SPAN></H1><UL><LI> 2 µL Ligase Buffer 10X
</LI><LI> V<SUB>v</SUB> µL of measured [DNA]<SUB>Vector</SUB> in ng/µL (30 to 50 ng)
</LI><LI> V<SUB>i</SUB> µL of measured [DNA]<SUB>Insert</SUB> in ng/µL
</LI><LI> Pure water QSP 20 µL
</LI><LI> 1 µL T4 DNA ligase at 400 000 U/mL concentration
</LI><LI> O/N at 16°C
</LI></UL><P><I>V<SUB>i</SUB> = (3 x [DNA]<SUB>Vector</SUB> x V<SUB>v</SUB> x Lenght<SUB>Insert</SUB>) <B>/</B> ([DNA]<SUB>Insert</SUB> x Lenght<SUB>Vector</SUB>)</I></P><H1><SPAN class="mw-headline" id="Transformation"> Transformation </SPAN></H1><P>Use chemically lab-made competent cells or from a kit (DH5α, TOP10, Mach1)
</P><UL><LI> Defroze competent cells on ice during 5'
</LI><LI> Add 5 µl of DNA Ligation in 50 µL of competent bacterias (or 1 µL for the positive control puc19)
</LI><LI> Incubate 30' on ice 
</LI><LI> Heat-shock the cells during 30&quot; at 42°C without shaking
</LI><LI> Put 2' on ice
</LI><LI> Add 250 µL of pre-warmed SOC medium (42°C)
</LI><LI> Incubate 1h at 37°C under shaking (225rpm)
</LI><LI> Spin at 5.000rpm during 30&quot;
</LI><LI> Remove 150 µL of supernatant
</LI><LI> Resuspend the pellet in the 150 µL left
</LI><LI> Spread on appropriate plates
</LI><LI> Incubate O/N at 37°C
</LI></UL><H1><SPAN class="mw-headline" id="LB_and_LBA"> LB and LBA </SPAN></H1><UL><LI> LB medium: Dissolve 25mg/L of Luria Broth in purified water. Autoclave the solution. Store at room temperature and take care of any contamination
</LI></UL><UL><LI> LBA medium: Dissolve and gently mix with a magnetic barrel 40mg/L of Luria Broth Agar in purified water. Autoclave the solution. Store at room temperature and take care of any contamination. To make some plates heat the solidified medium with microwave. Wait untill you can hold the bottle in your hands and then add the appropriate antibiotic to a final concentration of 1x. Provide approximatively 20mL per plate in a sterile hood. Returned the plates and store them at +4°C
</LI></UL><H1><SPAN class="mw-headline" id="Minimum_Medium"> Minimum Medium </SPAN></H1><UL><LI>Minimum Medium:
</LI></UL><P>- B1 Vitamin 0.1%
- Uracile 0.2%
- MgSO<SUB>4</SUB> 2 mM
- CaCl<SUB>2</SUB> 0.1 mM
- Glycerol 0.4%
- Casamino acids 0.1%
- M9 minimal salts medium 1x
- H<SUB>2</SUB>O QSP 200 mL
</P><P>Filter the medium with a cell-culture unit of filtration.  If needed add the appropriate antibiotic to a final concentration of 1x
</P><H1><SPAN class="mw-headline" id="Soft_Agar_Medium"> Soft Agar Medium </SPAN></H1><UL><LI> Add 3.5 to 4g of agar per liter of LB or Minimum Medium. Autoclave the solution. Store at room temperature and take care of any contamination. To make some plates heat the solidified medium with microwave. Wait untill you can hold the bottle in your hands and then add the appropriate antibiotic to a final concentration of 1x. Provide approximatively 20mL per plate in a sterile hood. Returned the plates and store them at +4°C
</LI></UL><H1><SPAN class="mw-headline" id="1000x_stock_antibiotic"> 1000x stock antibiotic </SPAN></H1><UL><LI> Chloramphenicol: dissolve 30mg/mL of chloramphenicol in ethanol 100%. Store at room temperature
</LI><LI> Tetracyclin: dissolve 12,5mg/mL of tetracyclin in a 50/50 ethanol/water mix. Store at -20°C
</LI><LI> Ampicillin: dissolve 100mg/mL of ampicillin in water. Filter the solution. Store at +4°C
</LI><LI> Kanamycin: dissolve 100mg/mL of kanamycin in water. Filter the solution. Store at room temperature
</LI></UL><H1><SPAN class="mw-headline" id="PCR_Screening"> PCR Screening </SPAN></H1><P>Use of 8 clones of Ligation transformants for screening PCR
</P><UL><LI> One sterile tip of each clone's colony per PCR tube
</LI><LI> Use sterile tip to start 7.5mL O/N culture
</LI></UL><P><B>After</B>, add
</P><UL><LI> 12.5 µL Mix
</LI><LI> 0.5 µL Oligo VF (10µM)
</LI><LI> 0.5 µL Oligo VR (10µM)
</LI><LI> 11,5 µL pure water
</LI><LI>negative control : without clone's colony
</LI><LI>positive control 
</LI></UL><P>Store the tubes on ice waiting for PCR attains 95°C then put the tubes in the machine
</P><UL><LI> Program : <B>SCREENIN</B></LI></UL><P>LID 105°C
1. 95°C   5min
2. 95°C   30 sec
3. 55°C   30 sec
4. 72°C   (1 min for 1kb) 
5. go to : 2   rep : 29
6. sound : 1
7. hold : 4°C
</P><H1><SPAN class="mw-headline" id="Electrophoresis_Purification_of_PCR"> Electrophoresis Purification of PCR </SPAN></H1><UL><LI> 10µl of ladder 100 pb or 1 kb 
</LI><LI> 4 µl of screening PCR 
</LI><LI> Migration at 100V on <B>1,5%</B> gel until 3/4 way
</LI></UL><H1><SPAN class="mw-headline" id="Sequencing"> Sequencing </SPAN></H1><P><A href="http://institut.cochin.inserm.fr/rubric_recherche/Plates-Formes/sequencage_genomique/I18NFolder.2005-02-10.4781618697/page2/fr" class="external text" rel="nofollow">Sequencing COCHIN</A></P><H1><SPAN class="mw-headline" id="Promoter_Characterization_Plan"> Promoter Characterization Plan </SPAN></H1><P>For theoretical consideration, see <A href="/Team:Paris/Modeling/estimation" title="Team:Paris/Modeling/estimation">estimation of parameters</A></P><UL><LI><B>Standart construct</B></LI></UL><P>‎
</P><UL><LI><B>Order of work</B></LI></UL><P>The same colour coded steps can be perfomed at the same time if elements needed are available
The order for treating the colours should of course be:
</P><UL><LI><B>Material needed</B></LI></UL><TABLE border="1"><TBODY><TR style="text-align: center;"><TD><B>Availability</B></TD></TR><TR style="text-align: center;"><TD colspan="4"><CENTER><B>Promoters</B></CENTER></TD></TR><TR style="text-align: center;"><TD>J23101
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pTet
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlhDC
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFliA
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFliL
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlgA
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlgB
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlhB
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD colspan="4"><CENTER><B>Genes</B></CENTER></TD></TR><TR style="text-align: center;"><TD>tetR
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>GFP E0240
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>OmpR*
</TD><TD>no
</TD></TR><TR style="text-align: center;"><TD>EnvZ*
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>FliA
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>FlhDC
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD colspan="4"><CENTER><B>Plasmids</B></CENTER></TD></TR><TR style="text-align: center;"><TD>pSB3K3 (ORI p15A)
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pSB4T5 (ORI pSC101)
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD colspan="4"><CENTER><B>RBS</B></CENTER></TD></TR><TR style="text-align: center;"><TD>B0032
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD colspan="4"><CENTER><B>Terminators</B></CENTER></TD></TR><TR style="text-align: center;"><TD>B0010
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>B0012
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD colspan="4"><CENTER><B>Bacterial Strains</B></CENTER></TD></TR><TR style="text-align: center;"><TD>Top10
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>FliA -/ FlhDC -
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>FlgM -
</TD><TD>yes
</TD></TR></TBODY></TABLE><UL><LI><B>Summary table</B></LI></UL><P>This table contains the promoters we need to characterize, the transcription factors whose effect on the promoter we want to test, and the plasmid we want to obtain in order to carry out each characterization
</P><TABLE border="1"><TBODY><TR style="text-align: center;"><TD><B>Promoter</B></TD><TD><B> Promoter</B><P><B>Availability</B></P></TD><TD><B>TF gene</B></TD><TD><B>TF gene</B><P><B>Availability</B></P></TD><TD><B>Done/Not done</B></TD></TR><TR style="text-align: center;"><TD>J23101
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pTet
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>tetR
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>GFP E0240
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlhDC
</TD><TD>yes
</TD><TD>OmpR*
</TD><TD>no
</TD></TR><TR style="text-align: center;"><TD>pFlhDC
</TD><TD>yes
</TD><TD>EnvZ*
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlhDC
</TD><TD>yes
</TD><TD>FliA
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFliA
</TD><TD>yes
</TD><TD>FlhDC
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFliA
</TD><TD>yes
</TD><TD>FliA
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFliL
</TD><TD>yes
</TD><TD>FlhDC
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFliL
</TD><TD>yes
</TD><TD>FliA
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlgA
</TD><TD>yes
</TD><TD>FlhDC
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlgA
</TD><TD>yes
</TD><TD>FliA
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pflgB
</TD><TD>yes
</TD><TD>FlhDC
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlgB
</TD><TD>yes
</TD><TD>FliA
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlhB
</TD><TD>yes
</TD><TD>FlhDC
</TD><TD>yes
</TD></TR><TR style="text-align: center;"><TD>pFlhB
</TD><TD>yes
</TD><TD>FliA
</TD><TD>yes
</TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Protocol_to_make_competent_bacteria"> Protocol to make competent bacteria </SPAN></H1><P>1. Use non competent bacteria (ex: MG1655) stocked in 1.5 mL LB (20% Glycerol): put a sterile tip in the 1.5 mL stock tube and then place it in a 50 mL Falcon with 5 ml LB medium. Over Night culture at 37°C / 300 rpm
</P><P>2. 1/100 dilution in LB medium QSP 50 mL in an erlenmeyer of 250 mL
</P><P>3. Culture at 37°C / 300 rpm untill OD<SUB>600</SUB> reach 0.6
</P><P>4. Fast cooling at +4°C by gently shaking the erlen in ice
</P><P>Before: prepare CaCl<SUB>2</SUB> 0.1M.
</P><UL><LI> Add 5,56 g in 500 mL H<SUB>2</SUB>O (Gibco)
</LI><LI> dissolve the CaCl<SUB>2</SUB> by mixing the suspension with the help of a magnetic stirrer
</LI><LI> Filter the solution with a cell-culture unit of filtration
</LI><LI> Aliquotes the filtered solution: 25mL in a 50 mL Falcon. Storage at +4°C
</LI></UL><P>5. Use pre-cooled centrifuge at +4°C. Centrifuge 50 mL of the culture in 50 mL falcon: +4°C / 5 min / 5000 rpm
</P><P>6. Discard supernatant by inverting the tube, and resuspend the pellet with 1 mL of cold CaCl<SUB>2</SUB> and mix gently the suspension by up and down
</P><P>7. Add cold CaCl<SUB>2</SUB> QSP 20 mL and incubate 30 min / +4°C
</P><P>8. Centrifuge the suspension : +4°C / 5 min / 5000 rpm
</P><P>9. Discard supernatant by inverting the tube, and resuspend the pellet with 1 mL of cold CaCl<SUB>2</SUB> and mix gently the suspension by up and down
</P><P>10. <A href="https://2008.igem.org/Team:Paris/Notebook/Protocols#Transformation" class="external text" rel="nofollow">Transform</A> or freeze the competent cells. Freeze the competent cells in 50 µL aliquots in the 0.1M CaCl<SUB>2</SUB> medium with 15% glycerol.
</P><P>11. After transformation, prepare a <A href="https://2008.igem.org/Team:Paris/Notebook/Protocols#Glycerol_Stocks" class="external text" rel="nofollow">Glycerol Stock</A> or/and use the transformed bacteria to study <A href="https://2008.igem.org/Team:Paris/Notebook/Protocols#Study_of_the_doubling_time_of_the_bacteria_population" class="external text" rel="nofollow">the doubling time of the bacteria population</A></P><H1><SPAN class="mw-headline" id="Study_of_the_doubling_time_of_the_bacteria_population"> Study of the doubling time of the bacteria population </SPAN></H1><UL><LI> Unfreeze strains from the glycerol stock containing or not the plasmid with a contitutive promoter and a fluorescent reporter gene
</LI><LI> Put a sterile tip in the 1.5 mL stock tube and then place it in a 50 mL Falcon with 5 ml of <A href="https://2008.igem.org/Team:Paris/Notebook/Protocols#Minimum_Medium" class="external text" rel="nofollow">minimum medium</A>. Over Night (16h) culture at 37°C / 300 rpm
</LI><LI> Measure the OD<SUB>600</SUB> (Linear Zone Measurement &lt;1.5)
</LI><LI> Dilute the culture medium in the same new medium to obtain an approximative OD<SUB>600</SUB> of 0.01 (V<SUB>final</SUB>= 50 mL in a 250 ml erlenmeyer)
</LI><LI> Incubate à 37°C / 300 rpm and measure the OD<SUB>600</SUB> every 20 min
</LI><LI> Determine the doubling time population and compare the strains containing or not the plasmid with a contitutive promoter and a fluorescent reporter gene
</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Paris/Notebook/Protocols">http://2008.igem.org/Team:Paris/Notebook/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris/Notebook/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris/Notebook/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris/Notebook/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris/Notebook/Protocols&amp;oldid=59509" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>