<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Paris_Analysis_Design1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Paris/Analysis/Design1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Paris/Analysis/Design1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris/Analysis/Design1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Paris/Analysis/Design1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Paris/Analysis/Design1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Paris/Analysis/Design1</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Paris/Network_Design1&amp;redirect=no" title="Team:Paris/Network Design1">Team:Paris/Network Design1</A>)</DIV><DIV id="menu"><UL><LI><A href="https://2008.igem.org/Team:Paris/Project">Project</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Analysis">Analysis</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Construction">Construction</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Characterization">Characterization</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Perspectives">Perspectives</A></LI></UL></DIV><TABLE cellspacing="5" cellpadding="10" style="margin-top: 3px; background:#649CD7; width: 965px;"><TBODY><TR valign="top"><TD style="background:#fff"><CENTER><DIV style="color:#275D96; font-size:2em;">Network Design - Part 1</DIV></CENTER><DIV id="links"><SPAN>Other pages:</SPAN><UL><LI><A href="https://2008.igem.org/Team:Paris/Analysis/Design1">The FIFO Behaviour</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Analysis/Design2">Oscillations</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Analysis/Design3">The synchronization Module</A></LI><LI><A href="https://2008.igem.org/Team:Paris/Analysis/Design3#Global_genetic_network">Global Genetic Network</A></LI></UL></DIV></TD></TR><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Generating_a_FIFO_sequential_order_of_expression"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Generating a FIFO sequential order of expression</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_easiest_way_to_control_a_sequential_expression_of_genes_:_the_LIFO_order"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The easiest way to control a sequential expression of genes : the LIFO order</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#The_rules_to_generate_a_FIFO_temporal_order"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">The rules to generate a FIFO temporal order</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#The_simplest_way_to_generate_a_FIFO_:_a_simple_cascade_of_genes"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">The simplest way to generate a FIFO : a simple cascade of genes</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#The_more_sophisticated_way_to_generate_a_FIFO_behavior_:_a_particular_Multiple_Output_Feed-Forward_Loop"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">The more sophisticated way to generate a FIFO behavior : a particular Multiple Output Feed-Forward Loop</SPAN></A></LI><LI class="toclevel-3"><A href="#Definition_of_a_FFL"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Definition of a FFL</SPAN></A></LI><LI class="toclevel-3"><A href="#The_different_types_of_FFL"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">The different types of FFL</SPAN></A></LI><LI class="toclevel-3"><A href="#The_type_1_coherent_Feed_Forward_Loop_with_an_OR_gate_introduces_a_delay_after_the_extinction_of_the_signal"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">The type 1 coherent Feed Forward Loop  with an OR gate introduces a delay after the extinction of the signal</SPAN></A></LI><LI class="toclevel-3"><A href="#Bibliography_:"><SPAN class="tocnumber">1.2.6</SPAN><SPAN class="toctext">Bibliography :</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-6"><A href="#Implementation_of_our_FIFO"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Implementation of our FIFO</SPAN></A></LI><LI class="toclevel-3"><A href="#Description_of_the_genetic_pathway_that_lead_to_flagella_assembly"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Description of the genetic pathway that lead to flagella assembly</SPAN></A></LI><LI class="toclevel-3"><A href="#The_expression_of_the_flagellar_genes_is_precisely_ordered"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">The expression of the flagellar genes is precisely ordered</SPAN></A></LI><LI class="toclevel-3"><A href="#A_quantitative_blueprint_of_the_dynamics_of_the_flagella_gene_network"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">A quantitative blueprint of the dynamics of the flagella gene network</SPAN></A></LI><LI class="toclevel-3"><A href="#The_dynamics_is_easily_tunable.2C_which_is_a_great_tool_for_engineering"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">The dynamics is easily tunable, which is a great tool for engineering</SPAN></A></LI><LI class="toclevel-3"><A href="#Bibliography"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">Bibliography</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Generating_a_FIFO_sequential_order_of_expression">Generating a FIFO sequential order of expression</SPAN></H1><P>A clock that would only indicate one hour, would not be very useful. The same goes for our BacteriO'clock. It is necessary that we add several distinct events in each cycle. We want to use three genes activated successively. 
</P><H2><SPAN class="mw-headline" id="The_easiest_way_to_control_a_sequential_expression_of_genes_:_the_LIFO_order">The easiest way to control a sequential expression of genes : the LIFO order</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">The washing of dirty plates traditionally follows a LIFO order : the last plate put on the stack is the first one to be washed</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">Network 1 : The Single Input Module can generate a LIFO temporal order.</DIV></DIV></DIV><P>
The temporal order of expression that is the easiest to set up is the <I>Last In First Out</I> temporal order. The genetic network that generates this behavior is the <I>Single Input Module (SIM)</I> (<I>Network 1</I>). A transcription factor X activates the expression of different genes Z<SUB>i</SUB>. When X concentration increases, the threshold of activation K<SUB>i</SUB> of the different genes Z are reached in a precise order. The expression will stop in the opposite order when X is not expressed anymore. To cut a long story short, the first gene to be switched on will be the last one to be OFF. 
</P><P>This order poses a real problem to us : <B>several states of the system are redundant</B> (<I>Table 1</I>). We can not accept a clock that would display twice the same hour !
</P><H2><SPAN class="mw-headline" id="The_rules_to_generate_a_FIFO_temporal_order">The rules to generate a FIFO temporal order</SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:692px;"><DIV class="thumbcaption">Table 1 : Comparison of the LIFO and the FIFO orders. (● : gene is OFF, O : gene is ON)</DIV></DIV></DIV></DIV><P>In the contrary, <B>in a FIFO order, there is no confusion possible between different states</B> (<I>Table 1</I>) : they are all distincts. To generate this complex behavior, there are two major possibilities. 
</P><H3><SPAN class="mw-headline" id="The_simplest_way_to_generate_a_FIFO_:_a_simple_cascade_of_genes">The simplest way to generate a FIFO : a simple cascade of genes</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:200px;"><DIV class="thumbcaption">Network 2 : A cascade of gene : the simplest way to generate a FIFO behavior.</DIV></DIV></DIV><P>The first one, the simplest, which is also the most unsophisticated, is a simple cascade of genes (<I>Network 2</I>). When the first gene Z<SUB>1</SUB> is switched on, it activates the second gene Z<SUB>2</SUB>, then the third gene Z<SUB>3</SUB> is turned on. When gene Z<SUB>1</SUB> is turned off, Z<SUB>2</SUB> is switched off first, then Z<SUB>3</SUB>... 
</P><P>This network is really simple to implement but presents several problems that makes it <B>incompatible with our project</B>. First of all, the cascade is not resistant to mutations : when one gene is mutated, the cascade is definitely broken. Secondly, the time scale if far too long. Each activation step would take approximately one hour. One full cycle would last almost six hours. Last but not least, the cascade is probably not very resistant to intracellular noise, it is not robust enough.
</P><H3><SPAN class="mw-headline" id="The_more_sophisticated_way_to_generate_a_FIFO_behavior_:_a_particular_Multiple_Output_Feed-Forward_Loop">The more sophisticated way to generate a FIFO behavior : a particular Multiple Output Feed-Forward Loop</SPAN></H3><P>The other way to implement a FIFO is based on a logic structure called Feed Forward Loop.
</P><DIV class="toggleswitch">↓ Click here to know more about Feed Forward Loops ↑</DIV><DIV class="toggled" style="margin: auto; width: 935px;"><TABLE cellspacing="2" cellpadding="10" style="background: black;" width="100%"><TBODY><TR valign="top"><TD style="background: white;"><H3><SPAN class="mw-headline" id="Definition_of_a_FFL">Definition of a FFL</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption">Fig. 1 : Structure of a type 1 coherent Feed Forward Loop</DIV></DIV></DIV><P>
A Feed-Forward Loop is a genetic network composed of three nodes. This strong network motif is composed of a transcription factor X that regulates a second transcription factor, Y, and both X and Y regulate Z <I>(Figure 1)</I>.
</P><H3><SPAN class="mw-headline" id="The_different_types_of_FFL">The different types of FFL</SPAN></H3><P>
Depending on the type of regulations between the different nodes, we can define eight types of FFL that can be classified into two groups : <I>coherent</I> and <I>incoherent</I> FFLs. In <I>coherent</I> FFLs, the indirect path has the same overall sign as the direct path. The most abundant FFL is the type-1 coherent FFL (C1-FFL) <I>(Figure 1)</I>.
</P><H3><SPAN class="mw-headline" id="The_type_1_coherent_Feed_Forward_Loop_with_an_OR_gate_introduces_a_delay_after_the_extinction_of_the_signal">The type 1 coherent Feed Forward Loop  with an OR gate introduces a delay after the extinction of the signal</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Fig. 2 : The C1-FFL with OR logic in the flagella system of <I>E. coli</I>.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Fig. 3 : Promoter dynamics after an OFF step of X, in the presence of Y. The results are shown for the wild-type bacterium, and for a bacterium in which the gene for Fli1 was deleted from the genome. The FFL generates a delay after an OFF step of X</DIV></DIV></DIV><P>In addition to the signs of the edges, to understand the dynamics of the FFL, we must also know how the inputs from the two regulators X and Y are integrated at the promoter of the gene Z. Uri ALON considers that there are two biologically reasonable logic functions : &quot;AND&quot; logic, in which <I>both</I> X and Y activities are need to be high in order to turn on Z expression and  &quot;OR&quot; logic in which <I>either</I> X or Y is sufficient <I>(Figure 2)</I>.
</P><P>If the input function of the promoter of the gene Z is &quot;OR&quot;, Z is expressed when X activity is high. There is no delay following the expression of X. But when X is not expressed anymore, its concentration decreases and reach the activation threshold of Y and Z. Y is not expressed anymore but as the concentration decreases, Z is still expressed. <B>The OR-gate C1-FFL allow the gene Z to be expressed about one more hour after the gene X is OFF.</B><I>(Figure 3)</I></P><H3><SPAN class="mw-headline" id="Bibliography_:">Bibliography :</SPAN></H3><UL><LI><A href="http://www.nature.com/ng/journal/v31/n1/abs/ng881.html" class="external text" rel="nofollow">Shen-Orr <I>et al</I>. (2002)</A></LI></UL><UL><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/14607112?ordinalpos=1&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum" class="external text" rel="nofollow">Mangan <I>et al</I>. (2003)</A></LI></UL><UL><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/14530388?ordinalpos=2&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum" class="external text" rel="nofollow">Mangan <I>et al</I>. (2003)</A></LI></UL><UL><LI><A href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16729041" class="external text" rel="nofollow">Kalir <I>et al</I>. (2005)</A></LI></UL></TD></TR></TBODY></TABLE></DIV><P>More precisely, the genetic network is called Multiple Output FFL (<I>Network 3</I>). Each gene Z<SUB>i</SUB> is regulated by both X and Y, as in a typical FFL. X activates Z<SUB>i</SUB> when its concentration reaches the threshold K<SUB>X/Zi</SUB>. It is the same for Y and its respective thresholds.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Network 3 : Multiple Output FFL is the logic structure that permit FIFO behavior. The thresholds must be ordered is a precise way : K<SUB>X/Z1</SUB>&lt; K<SUB>X/Z2</SUB>&lt; K<SUB>X/Z3</SUB> and K<SUB>X/Z1</SUB>&gt; K<SUB>X/Z2</SUB>&gt; K<SUB>X/Z3</SUB></DIV></DIV></DIV><P>
All the MO FFL do not behave according to the FIFO order, three conditions are necessary and sufficient.
</P><UL><LI> The input function for each gene Z<SUB>i</SUB> must be a <I>OR</I> or a <I>SUM</I> gate.
</LI><LI> To have the order of expression Z<SUB>1</SUB>, Z<SUB>2</SUB> and then Z<SUB>3</SUB> at the switching ON, the thresholds must be ordered like this : 
</LI></UL><CENTER> K<SUB>X/Z1</SUB>&lt; K<SUB>X/Z2</SUB>&lt; K<SUB>X/Z3</SUB></CENTER><UL><LI> To have the FIFO order, Z<SUB>1</SUB> must be the first gene to be switched OFF when X is no more expressed. When X is not expresses anymore, its concentration decreases. The threshold K<SUB>X/Z1</SUB> will be reached after the thresholds K<SUB>X/Z2</SUB> and K<SUB>X/Z3</SUB>. As Y is still expressed when X decreases, its own concentration will decrease with a delay. To have a FIFO, Y concentration must reach K<SUB>Y/Z1</SUB> before K<SUB>X/Z2</SUB> and K<SUB>X/Z3</SUB>. The necessary order must be : 
</LI></UL><CENTER> K<SUB>Y/Z1</SUB>&gt; K<SUB>Y/Z2</SUB>&gt; K<SUB>Y/Z3</SUB></CENTER><P>
This network has several advantages compared to the cascade. It is more resistant to mutations : if gene Z<SUB>1</SUB> is mutated, genes Z<SUB>2</SUB> and Z<SUB>3</SUB> still behave as they should do. This is particularly interesting if we need more than three genes. The MOFFL is more robust concerning intracellular noisy fluctuations of concentration. The temporal expression of genes is more tunable : the expression of gene Z<SUB>2</SUB> can be fully activated even if Z<SUB>1</SUB> is not concentrated enough.
</P><H2><SPAN class="mw-headline" id="Implementation_of_our_FIFO">Implementation of our FIFO</SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:472px;"><DIV class="thumbcaption">Network 4 : Final design of our FIFO order generator system</DIV></DIV></DIV></DIV><P>Now that the necessaries and sufficient conditions have been depicted, we can start to describe our FIFO. Rather than designing it from scratch we decided to look in nature where we found and adopted the flagella system of construction in <I>Escherichia coli</I> which then inspired us further.
</P><DIV class="toggleswitch">↓ Click here to know more about <I>E. coli</I> flagella. ↑</DIV><DIV class="toggled" style="margin: auto; width: 935px;"><TABLE cellspacing="2" cellpadding="10" style="background: black;" width="100%"><TBODY><TR valign="top"><TD style="background: white;"><H3><SPAN class="mw-headline" id="Description_of_the_genetic_pathway_that_lead_to_flagella_assembly">Description of the genetic pathway that lead to flagella assembly</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 1 : The genetically defined hierarchy of the flagellar operons in <I>Escherichia coli</I>.</DIV></DIV></DIV><P>The construction of <I>E. coli</I> flagellar system is organized into a complex hierarchy. The different parts of the flagella are successively assembled parts by parts. This architecture is also present in the network of genes that contributes to its set-up. 
</P><P>A global transcription factor FlhDC activates the synthesis of the flagella. This class 1 gene encodes a protein that activates the expression of class 2 genes. Those genes lead to the construction of the basal body of the flagella and its hook. Then FliA, a protein coded by a class 2 gene activates, with FlhDC to the expression of class 3 genes (<I>Figure 1</I>).
</P><H3><SPAN class="mw-headline" id="The_expression_of_the_flagellar_genes_is_precisely_ordered">The expression of the flagellar genes is precisely ordered</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:223px;"><DIV class="thumbcaption">Figure 2 : The sequential expression of flagellar genes</DIV></DIV></DIV>When the flagella system is turned on, the genes are expressed sequentially. flhD is expressed first, then the class 2 genes, and lastly the class 3 genes. At the heart of each class, the promoters are turned on sequentially, with significant delays, in the order <I>fliL, fliE, fliF, flgA, flgB, flhB,</I> and <I>fliA</I> (<I>Figure 2</I>). 
<P>The observed order correponds to the spatial position of the gene products during flafellar motor assemblu, going from the cytoplasmic to the extracellular sides.
</P><H3><SPAN class="mw-headline" id="A_quantitative_blueprint_of_the_dynamics_of_the_flagella_gene_network">A quantitative blueprint of the dynamics of the flagella gene network</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:406px;"><DIV class="thumbcaption">Figure 3 : Quantitative blueprint of the dynamics of class 2 genes of the flagella gene network. β is the activation coefficient corresponding to a transcription factor and a promoter (unit : GFP/OD).</DIV></DIV></DIV><A href="http://www.ncbi.nlm.nih.gov/pubmed/15186773?ordinalpos=2&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum|" class="external text" rel="nofollow">Kalir <I>et al.</I> (2004)</A> evaluated the strength of the interactions between the different transcription factors fliA and flhDC and the class 2 promoters. They established a quantitative blueprint, considering the strength of the interactions and the functions that integrate multiple input (<I>Figure 3</I>). The structure of this network is typically a Multiple Output FFL, each class 2 gene is regulated by both flhDC and fliA, which is itself regulated by flhDC. This network is the logic circuit that can generate a FIFO behavior.
<P>They determined that the input function for <I>fliL</I> promoter is a SUM gate in relation to FliA and FlhDC. 
</P><CENTER>It means that: <B>Promoter_Activity([FlhDC],[FliA]) = Promoter_Activity([FlhDC],0) + Promoter_Activity(0,[FliA])</B></CENTER><P>The existence of a SUM input function for the other promoters has not been demonstrated but Uri ALON and his team assume that all the class 2 promoters have the same behavior. This is the first condition to create a FIFO.
</P><P>The coefficients of activation for <I>fliL</I>, <I>flgA</I> and <I>flhB</I> are ordered in the right way to behave as a FIFO : 
</P><CENTER><B>β<SUB>1</SUB>&lt;β<SUB>5</SUB>&lt;β<SUB>6</SUB></B> and <B>β'<SUB>1</SUB>&gt;β'<SUB>5</SUB>&gt;β'<SUB>6</SUB></B></CENTER><H3><SPAN class="mw-headline" id="The_dynamics_is_easily_tunable.2C_which_is_a_great_tool_for_engineering">The dynamics is easily tunable, which is a great tool for engineering</SPAN></H3><P>The model suggests that, in order to change the response time, one can change the numbers on the arrows. For the earliest promoter, <I>fliL</I>, decreasing β<SUB>1</SUB> should make the rise time longer and the maximal expression lower compared to the wild-type promoter. To experimentally change β<SUB>1</SUB>, Shiraz KALIR and Uri ALON inserted point mutations in the FlhDC binding site of the <I>fliL</I> promoter. Those mutations made the promoter activity later and weaker.
</P><H3><SPAN class="mw-headline" id="Bibliography">Bibliography</SPAN></H3><UL><LI><A href="http://www.sciencemag.org/cgi/content/abstract/292/5524/2080|" class="external text" rel="nofollow">Kalir <I>et al.</I> (2001)</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/pubmed/15186773?ordinalpos=2&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_DefaultReportPanel.Pubmed_RVDocSum|" class="external text" rel="nofollow">Kalir <I>et al.</I> (2004)</A></LI></UL></TD></TR></TBODY></TABLE></DIV><UL><LI>For X we chose <I>flhDC</I> and its natural promoter.
</LI><LI>For Y, we chose <I>fliA</I> and its natural promoter.
</LI><LI>For Z<SUB>1</SUB>, we chose <I>fliL</I> promoter and ECFP-LVA to report the expression. We chose ECFP because this fluorophore is not very fluorescent. As <I>fliL</I> promoter is the most strongly activated promoter, it compensate its weakness.  
</LI><LI>For Z<SUB>2</SUB>, we chose <I>flgA</I> promoter and YFP-LVA to report the expression.
</LI><LI>For Z<SUB>3</SUB>, we chose <I>flhB</I> promoter and mRFP-LVA to report the expression. mRFP is the fluorophore that has the longest time of maturation. When put last in the FIFO, it helps to separate the different states.
</LI></UL><P>The LVA tails are very important to reduce the fluorophores half lives from 20 hours minimum to 30 minutes. Those three fluorophores are compatible together, regarding to their absorption and emission wavelength.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Paris/Analysis/Design1">http://2008.igem.org/Team:Paris/Analysis/Design1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Paris/Analysis/Design1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Paris/Analysis/Design1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Paris/Analysis/Design1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Paris/Analysis/Design1&amp;oldid=102584" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>