<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_RHIT_Parts_html"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:RHIT/Parts.html">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:RHIT/Parts.html&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:RHIT/Parts.html&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:RHIT/Parts.html&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:RHIT/Parts.html" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:RHIT/Parts.html</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Parts</TITLE></P><DIV id="top"><DIV class="navig-bar"><UL class="nav nav-pills"><LI class="dropdown"><A href="https://2013.igem.org/Team:RHIT">Home</A></LI><LI class="dropdown"><A href="Team.html" data-toggle="dropdown" class="dropdown-toggle">Team </A></LI><LI><A href="https://2013.igem.org/Team:RHIT/Members.html">Members</A></LI><LI><A href="https://igem.org/Team.cgi?id=1154">Official Profile</A></LI><LI><A href="https://2013.igem.org/Team:RHIT/University.html">University</A></LI></UL><LI class="dropdown"><A href="https://2013.igem.org/Team:RHIT/Project.html">Project</A></LI><LI class="dropdown"><A href="https://2013.igem.org/Team:RHIT/Notebook.html" data-toggle="dropdown" class="dropdown-toggle">Lab Work </A></LI><LI><A href="https://2013.igem.org/Team:RHIT/Protocol.html">Protocol</A></LI><LI><A href="https://2013.igem.org/Team:RHIT/Notebook.html">Notebook</A></LI><LI class="active"><A href="https://2013.igem.org/Team:RHIT/Parts.html">Parts</A></LI><LI class="dropdown"><A href="https://2013.igem.org/Team:RHIT/Modeling.html" data-toggle="dropdown" class="dropdown-toggle">Modeling </A></LI><LI><A href="https://2013.igem.org/Team:RHIT/Modeling.html">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:RHIT/HandsOn.html">Hands On</A></LI><LI class="dropdown"><A href="https://2013.igem.org/Team:RHIT/Safety.html">Safety</A></LI><LI class="dropdown"><A href="https://2013.igem.org/Team:RHIT/Outreach.html">Outreach</A></LI><LI class="dropdown"><A href="https://2013.igem.org/Team:RHIT/Attributions.html">Attributions</A></LI></DIV></DIV><DIV class="mainWrapper"><DIV class="sideNav"><DIV id="cctitle">How Technical</DIV><DIV id="contentChanger"><LABEL id="slider" class="checkbox toggle candy"><P><SPAN>Less</SPAN><SPAN>More</SPAN></P></LABEL></DIV><SPAN class="tech"><UL class="sideBar"><LI><STRONG><I>Saccharomyces cerevisiae</I></STRONG></LI><LI> - <A href="#main1">FUS1 Promoter</A></LI><LI> - <A href="#main2">His3 Protein</A></LI><LI> - <A href="#main3">Constitutive Yeast Promoter</A></LI><LI> - <A href="#main4">Lactate Dehydrogenase</A></LI><LI> - <A href="#main5">Yeast Terminator</A></LI><LI><STRONG><A href="#main7" style="color:black;"><I>Escherichia coli</I></A></STRONG></LI><LI> - <A href="#main6">Constitutive <I><I><I>E. coli</I></I></I> Promoter and Ribosome Binding Site</A></LI><LI> - <A href="#main7">Ice Nucleation Protein</A></LI><LI> - <A href="#main8">Mating Factor Alpha</A></LI><LI> - <A href="#main9">Lactate-Sensitive Promoter</A></LI><LI> - <A href="#main10">His3 Protein</A></LI><LI> - <A href="#main11"><I><I><I>E. coli</I></I></I> Terminator </A></LI></UL></SPAN><SPAN class="notTech"><UL class="sideBar"><LI><STRONG><I>Saccharomyces cerevisiae</I></STRONG></LI><LI> - <A href="#main12">FUS1 Promoter</A></LI><LI> - <A href="#main13">His3 Protein</A></LI><LI> - <A href="#main14">Constitutive Yeast Promoter</A></LI><LI> - <A href="#main15">Lactate Dehydrogenase</A></LI><LI> - <A href="#main16">Yeast Terminator</A></LI><LI><STRONG><A href="#main17" style="color:black;"><I>Escherichia coli</I></A></STRONG></LI><LI> - <A href="#main17">Constitutive <I><I><I>E. coli</I></I></I> Promoter and Ribosome Binding Site</A></LI><LI> - <A href="#main18">Ice Nucleation Protein</A></LI><LI> - <A href="#main19">Mating Factor Alpha</A></LI><LI> - <A href="#main20">Lactate-Sensitive Promoter</A></LI><LI> - <A href="#main21">His3 Protein</A></LI><LI> - <A href="#main22"><I><I><I>E. coli</I></I></I> Terminator </A></LI></UL></SPAN></DIV><DIV class="mainContent" style="margin-bottom:113px;"><DIV class="main"><DIV class="bottomPart" style="font-size:13pt;"><P>The systems (one for <I><I><I>E. coli</I></I></I> and one for Yeast) described during the project explanation are broken up into their parts below. Each of the parts have a specific function and the two constructs together form a positive feedback loop that allows the two organisms to stay in close proximity.  </P></DIV></DIV><SPAN class="tech"><DIV style="font-weight:bold; font-size:20pt; margin-left:25%;margin-bottom:30px"><I>Saccharomyces cerevisiae</I></DIV><DIV class="main" id="main1"><DIV class="details"><DIV class="title">FUS1 Promoter</DIV></DIV><DIV class="bottomPart">
            The product of the FUS1 gene, Fus1, plays a critical role in the mating pheromone response pathway of yeast, upregulating the production of genes necessary for cell fusion and mating.  This gene is activated by the phosphorylated form of Ste12, which is activated through previous interactions in the pathway.  Using the Ste12 regulated promoter of FUS1 allows for mating-sensitive gene expression.  If it functions as expected, the genes to be expressed should be produced in response to the haploid yeast cells binding their complementary mating factor.

McCaffrey G, Clay FJ, Kelsay K, and Sprague GF Jr (1987) Identification and regulation of a gene required for cell fusion during mating of the yeast <I>Saccharomyces cerevisiae</I>. Mol. Cell. Biol. 7 (8), 2680-2690

        </DIV></DIV><DIV class="main" id="main2"><DIV class="details"><DIV class="title">His3 Protein</DIV><DIV class="shortDescription">This essential gene has been knocked out in the strain, meaning that the only way the yeast can survive is if the mating factor response pathway has been activated.</DIV></DIV><DIV class="bottomPart">
            A common auxotrophic marker in yeast is a deletion of the HIS3 gene, which is responsible for the production of the His3 protein (Imidazoleglycerol-phosphate dehydratase).  Due to the deletion, strains are unable to grow on media lacking a histadine supplement unless they have a way to express the gene or an analog.  By placing the gene in a construct with the appropriate promoter, a group can select for a particular transformed strain by placing it on selective media lacking histadine.  In CUPID, this protein has been linked to the FUS1 promoter, allowing for the selection of yeast being stimulated by mating pheromone.  

Saccharomyces Genome Database. (1999). Sequence for a region of HIS3/YOR202W. Retrieved from: http://www.yeastgenome.org/cgi-bin/getSeq?query=YOR202W&amp;seqtype=Coding%20sequence%20(CDS)&amp;format=fasta

        </DIV></DIV><DIV class="main" id="main3"><DIV class="details"><DIV class="title">Constitutive Yeast Promoter</DIV></DIV><DIV class="bottomPart">
            Promoters can respond to a number of different stimuli, and a number are constantly activated in a cell.  Transcriptional elongation factor is expressed in yeast constitutively and is responsible for the elongation of the RNA transcript.  Since the gene is constitutively expressed, its promoter can be used to constitutively express a gene of interest in yeast.  

Mumberg D, Müller R, Funk M (1995) Yeast vectors for the controlled expression of heterologous proteins in different genetic backgrounds. Gene 156 (1), 119-122



        </DIV></DIV><DIV class="main" id="main4"><DIV class="details"><DIV class="title">Lactate Dehydrogenase</DIV></DIV><DIV class="bottomPart">
           It has been observed that yeast expressing lactate dehydrogenase (LDH) can be used to produce lactate in the media they are growing in.  A number of other organisms have promoters that are sensitive to the presence of lactate in growth media.  By having the yeast produce lactate, it could activate a promoter in an organism growing in the same media, thereby having genes only expressed when in the presence of yeast with this gene.

Branduardi P, Sauer M, De Gioia L, Zampella G, Valli M, et. al (2006) Lactate production yield from engineered yeasts is dependent on the host background, the lactate dehydrogenase source, and the lactate export. Microbial Cell Factories. 5 (4)




        </DIV></DIV><DIV class="main" id="main5"><DIV class="details"><DIV class="title">Yeast Terminator</DIV></DIV><DIV class="bottomPart">
          To stop the transcription of the genes we are expressing in each of these two pathways, we use a transcriptional terminator specific to yeast.  Since we can only express one gene on each transcript, each ends in the same way with a nonselective terminator.  

Goffeau A, Barrell BG, Bussey H, Davis RW, Dujon B et al (1996) Life with 6000 genes. Science 274 (5287), 546, 563-567



        </DIV></DIV><DIV style="font-weight:bold; font-size:20pt; margin-left:30%;margin-bottom:30px"><I>Escherichia coli</I></DIV><DIV class="main" id="main6"><DIV class="details"><DIV class="title">Constitutive <I><I><I>E. coli</I></I></I> Promoter and Ribosome Binding Site</DIV></DIV><DIV class="bottomPart">
        As with the yeast constitutive promoter, this promoter allows for a gene to continuously be expressed in <I><I>E. coli</I></I>. This promoter is drawn from the registry of standard biological parts. Although this sequence promotes the continuous transcription of the gene we would like to express, it does not have a ribosome binding site to allow for translation. By adding such a site to the end of the sequence, it could not now be used for constitutive expression of a protein.

Registry of Standard Biological Parts. August 8, 2006. Retrieved June 7, 2013 from Registry of Standard Biological Parts: http://partsregistry.org/Part:BBa_J23119

Registry of Standard Biological Parts. January 31, 2003. Retrieved June 7, 2013 from Registry of Standard Biological Parts: http://partsregistry.org/Part:BBa_B0034 

        </DIV></DIV><DIV class="main" id="main7"><DIV class="details"><DIV class="title">Ice Nucleation Protein</DIV></DIV><DIV class="bottomPart">
       In 2012, the Pennsylvania iGEM team produced a protein that would allow for the expression of other proteins on the surface of <I><I>E. coli</I></I>. This protein was based off of an ice nucleation protein found in Xanthomonas campestris that allowed for the nucleation and formation of ice crystals. The modified form of this protein results in a shorter sequence, and with the addition of a chain of glycine and serine residues, can be used to have the cell surface display a particular protein.

Registry of Standard Biological Parts. October 10, 2012. Retrieved June 7, 2013 from Registry of Standard Biological Parts: http://partsregistry.org/Part:BBa_K811003

        </DIV></DIV><DIV class="main" id="main8"><DIV class="details"><DIV class="title">Mating Factor α</DIV></DIV><DIV class="bottomPart">
     Mating factor α (MFα) is one of the mating pheromones that yeast produce. When a cell with the a-mating type detects MFα in its environment, it will halt its cell cycle and attempt to mate by budding a schmoo in the direction of the mating factor. Simply having the mating factor in the media is enough to cause cellular response, and this same response should be possible if another organism were displaying the factor on its surface, as with the ice nucleation protein on <I><I>E. coli</I></I>.

Bardwell L (2004) A walk-through of yeast mating pheromone response pathway. Peptides 25: 1465-1476

        </DIV></DIV><DIV class="main" id="main9"><DIV class="details"><DIV class="title">Lactate-Sensitive Promoter</DIV></DIV><DIV class="bottomPart">
A number of organisms have promoters that are sensitive to lactate in their environment.  By using such a promoter, an organism can turn on specific genes when in the presence of lactate, usually those involved in its breakdown and metabolism.  However, by placing a different gene after this promoter, it is possible to have it expressed only when the cells are exposed to lactate in their environment.  It is also important to note that many pathways that depend on the presence of lactate are inhibited in the presence of glucose, since it has significantly more stored energy than lactate or other partially broken down sugars.  This made it important to find a promoter that would still function even in the presence of glucose.

Lodi T, Goffrini P, Bolondi I, and Ferrero I (1998) Transcriptional regulation of the KlDLD gene, encoding the mitochondrial enzyme D-lactate ferricytochrome c oxidoreductase in Kluyveromyces lactis: effect of Klhap2 and fog mutations. Curr Genet 34, 12-20
        </DIV></DIV><DIV class="main" id="main10"><DIV class="details"><DIV class="title">His3 Protein</DIV></DIV><DIV class="bottomPart">
As with the His3 protein used in yeast, it is possible to use a similar approach when selecting for <I><I>E. coli</I></I>. <I><I>E. coli</I></I> has a gene, hisB, that codes for an analog of His3, performing the same function as the yeast protein. By using a strain that is a mutant for hisB, one can select for transformed colonies by placing them on media lacking histadine. If this gene is produced via an inducible pathway, it would require the organism to be receiving that stimulus in order to live on media lacking histadine.

Saccharomyces Genome Database. (1999). Sequence for a region of HIS3/YOR202W. Retrieved from: http://www.yeastgenome.org/cgi-bin/getSeq?query=YOR202W&amp;seqtype=Coding%20sequence%20(CDS)&amp;format=fasta

        </DIV></DIV><DIV class="main" id="main11"><DIV class="details"><DIV class="title"><I><I><I>E. coli</I></I></I> Terminator</DIV></DIV><DIV class="bottomPart">
To end the transcription of both the constructs to be expressed in <I><I><I>E. coli</I></I></I>, a bidirectional terminator is to be used.  This terminator is capable of terminating transcription in either a forward or reverse direction when placed at the end of a gene.  Although the designed system should only be expressed in the forward direction, the terminator can be used to stop the transcription of genes if open reading frames in the reverse direction are detected.

Registry of Standard Biological Parts. July 17, 2003. Retrieved June 7, 2013 from Registry of Standard Biological Parts: http://partsregistry.org/Part:BBa_B0015 


        </DIV></DIV></SPAN><SPAN class="notTech"><DIV class="main" id="main12"><DIV class="details"><DIV class="title">FUS1 Promoter</DIV></DIV><DIV class="bottomPart">
           The FUS1 gene is turned on when yeast encounters the opposite mating type as part of its preparation for mating. This means any gene hooked up to this promoter is turned on any time the yeast comes into contact with the opposite mating factor. 

        </DIV></DIV><DIV class="main" id="main13"><DIV class="details"><DIV class="title">His3 Protein</DIV></DIV><DIV class="bottomPart">
           Histidine is an amino acid necessary for nearly all forms of life, and in order to be synthesized by cells, many genes must work together in order to produce it. This gene is one of those genes necessary for histidine production. Without a functional copy of histidine, the cell will die unless it can obtain histidine from its environment.  By hooking up this gene to the FUS1 promoter, we can make it so that the only way yeast will survive is when the opposite mating factor is provided by the <I><I><I>E. coli</I></I></I>, or when histidine is provided in the media.
        </DIV></DIV><DIV class="main" id="main14"><DIV class="details"><DIV class="title">Constitutive Yeast Promoter</DIV></DIV><DIV class="bottomPart">
           Some genes in cells need to always be on at relatively constant levels in order to keep the cell alive. These genes have what are called “constitutive” promoters, which means that they are always on and don’t change their expression regardless of what is happening to the cell.



        </DIV></DIV><DIV class="main" id="main15"><DIV class="details"><DIV class="title">Lactate Dehydrogenase</DIV></DIV><DIV class="bottomPart">
           Lactate Dehydrogenase is a protein that can be used to synthesize a compound known as lactate. This is done by converting a chemical normally used for energy production into lactate. Lactate can be used to induce many lactate-sensitive promoters found in nature. In our system, we’re giving the ability to produce lactate, and making the <I><I><I>E. coli</I></I></I> dependent upon lactate in order to survive, allowing us to provide a large amount of selective pressure to keep the two organisms bound.




        </DIV></DIV><DIV class="main" id="main16"><DIV class="details"><DIV class="title">Yeast Terminator</DIV></DIV><DIV class="bottomPart">
          Much like promoters act as a green light telling transcription to begin and a gene to be made, terminators are also required to act as red lights telling transcription to stop once the gene was been made into RNA. 



        </DIV></DIV><DIV style="font-weight:bold; font-size:20pt; margin-left:30%;margin-bottom:30px"><I>Escherichia coli</I></DIV><DIV class="main" id="main17"><DIV class="details"><DIV class="title">Constitutive <I><I><I>E. coli</I></I></I> Promoter and Ribosome Binding Site</DIV></DIV><DIV class="bottomPart">
        Just like what was discussed with the constitutive yeast promoter, <I><I><I>E. coli</I></I></I> uses a constitutive promoter to keep gene expression constant at a stable level. The promoter we chose however, did not come with a Ribosomal Binding Site, a sequence which is required for translation to occur allowing the gene to be fully expressed. So, we merely added the Ribosomal Binding Site immediately after our promoter.

        </DIV></DIV><DIV class="main" id="main18"><DIV class="details"><DIV class="title">Ice Nucleation Protein</DIV></DIV><DIV class="bottomPart">
       This protein was initially taken from bacteria that lived on a plant, and its purpose is to aid in the formation of ice crystals on the plant. This protein sits on the surface of the bacteria, and displays a smaller protein. Last year, the Queen’s iGEM team used this part to express a protein of their own choosing on the surface of <I><I><I>E. coli</I></I></I>, showing that others could use this part for surface of nearly any protein on <I><I><I>E. coli</I></I></I>.

        </DIV></DIV><DIV class="main" id="main19"><DIV class="details"><DIV class="title">Mating Factor Alpha</DIV></DIV><DIV class="bottomPart">
     Much like how animals have two genders, yeast has two mating types: mating type a and mating type alpha. Yeast cells secrete small proteins that correspond to their mating type, and when yeast of the opposite mating type encounters these proteins (also called mating factors), they grow in the direction that they sensed the opposite yeast. In order for reproduction to occur, an a cell and an alpha must come into contact with one another, fuse, and then divide into 4 cells. In order for this to happen, the expression of many genes has to change in the cell, so there are many promoters that are sensitive to when a mating factor of the opposite type is sensed. In our project we are using this to our advantage by expressing mating factor on the surface of <I><I><I>E. coli</I></I></I> using the Ice Nucleation Protein described above and using a mating-sensitive promoter on an essential gene to construct a system where the only way yeast can survive is if <I><I><I>E. coli</I></I></I> binds to its surface.
        </DIV></DIV><DIV class="main" id="main20"><DIV class="details"><DIV class="title">Lactate-Sensitive Promoter</DIV></DIV><DIV class="bottomPart">
Up until now, we’ve thoroughly discussed how the yeast is dependent upon <I><I><I>E. coli</I></I></I> for its survival, but in order for our system to work, we need the <I><I><I>E. coli</I></I></I> to be dependent upon the yeast as well. We did this by hooking up an essential gene for <I><I><I>E. coli</I></I></I> to a promoter that is only turned on by exposure to lactate, a chemical we’ve made the yeast secrete. This means that the only way the <I><I><I>E. coli</I></I></I> can survive, is if the <I><I><I>E. coli</I></I></I> is close to the yeast to absorb the lactate the yeast is secreting.
        </DIV></DIV><DIV class="main" id="main21"><DIV class="details"><DIV class="title">His3 Protein</DIV></DIV><DIV class="bottomPart">
Just like with the yeast, the essential gene we are using as a selective pressure is a gene in the histidine pathway. This gene is very similar to the gene we used in the yeast, but has been optimized to work in <I><I><I>E. coli</I></I></I>.
        </DIV></DIV><DIV class="main" id="main22"><DIV class="details"><DIV class="title"><I><I><I>E. coli</I></I></I> Terminator</DIV></DIV><DIV class="bottomPart">
As explained above with the yeast terminator, just like the promoters act as green lights, terminators act as red lights. However, different species have different terminator preferences, so we selected one that is optimized for <I><I><I>E. coli</I></I></I>.

        </DIV></DIV></SPAN></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:RHIT/Parts.html">http://2013.igem.org/Team:RHIT/Parts.html</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:RHIT/Parts.html" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:RHIT/Parts.html" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:RHIT/Parts.html&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:RHIT/Parts.html&amp;oldid=233315" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>