Team:ShanghaitechChina/Description
igem2016:ShanghaiTech
Solar Hunter
Home
Team
Members
Attributions
Project
Integrative Biohydrogen System
Semiconductor Nanomaterials
Engineered Biofilms
Hydrogenase Gene Clusters
Parts
Notebook
Human Practice
InterLab
Safety
Special Prize
Integrated Human Practices
Education and Public Engagement
Measurement
Applied Design
Improve the characterization
Optimize the codon
Overview
This year, we have done the two improvement work:
1. We improved the characterization of biobrick: BBa_K1583003 which was originally characterized by iGEM15_TU_Delft.
2. We optimized the codons for a major functional part of Hydrogenase, HydA. This biobrick: BBa_K535002 was originally designed by: iGEM11_UNAM-Genomics_ Mexico.
In the following content, we introduce our work in detail to illustrate why we think we met the criteria.
Improve the characterization
>Contribution:
Biobrick: BBa_K1583003
Group: ShanghaitechChina
Author: Lechen Qian, Shijie Gu
Summary: We created new way to characterize this biobrick which was originally designed and characterized by iGEM15_TU_Delft. We utilize NTA-Metal-Histag coordination chemistry and fluorescence emission traits of Quantum Dots (QDs) in our project to improve the characterization. We demonstrated the validity of the approach for measurement of biofilm composed by CsgA-His density of E. coli curli system and think highly of this characterization for its general application in other biofilm systems. Also, we utilized TEM to help us scrutinize the binding effect in microscopic world.
>Improvement:
Quantum dots binding test
In order to test the effect of binding between CsgA-Histag mutant and inorganic nanoparticles, we apply same amount of suspended QDs solution into M63 medium which has cultured biofilm for 72h. After 1h incubation, we used PBS to mildly wash the well, and the result was consistent with our anticipation: On the left, CsgA-Histag mutant were induced and thus secreted biofilm, and firmly attached with QDS and thus show bright fluorescence. Therefore, we ensure the stable coordinate bonds between CsgA-Histag mutant and QDs can manage to prevent QDs from being taken away by liquid flow. The picture was snapped by ChemiDoc MP,BioRad, false colored.
Fig. 1：Binding test between CsgA-his and Quantum dots. The image was snaped by ChemiDoc MP,BioRad, false colored.
Comparison test of Quantum dots Binding between CsgA-his and CsgA
In order to prove the effect of binding between CsgA-Histag mutant and inorganic nanoparticles is distinct, we apply same amount of suspended CdSeS/ZnS QDs solution followed by the same procedure mentioned above. After 1h incubation, we used PBS washing 2 times. The picture verify our postulation: On the left, CsgA-Histag mutant were induced and its biofilm bind with QDS. CsgA biofilm without Histag cannot bind with QDs thus its red fluorescence is much weaker.
Fig. 2：Comparison test of Quantum dots Binding between CsgA-his and CsgA.
CdS nanorods Templating
As for biofilm characterization, transmission electron microscopy is frequently to be used to visualize the nanofiber network. However, TEM is not very efficient to visualize soft matter due to the less dense of elections produced on soft matter even after negative staining. Amazingly, after incubation with CdS nanorods , the biofilm areas are densely templated by better conductive materials such as CdS nanorods  and we can easily confirm the expression of biofilm.
Fig. 3：Representative TEM images of biotemplated CdS nanorods on CsgA-His. After applied inducer, CsgA-His mutant constructed and expressed to form biofilm composed by CsgA-His subunits. Incubation with nanorods for 1h, nanomaterials are densely attached to biofilm.
Details see ShanghaitechChina team's protocol
Optimize the codon
>Contribution:
Biobrick: BBa_K2132005
Group: ShanghaitechChina
Author: Yifan Chen
Summary: We optimized [FeFe] Hydrogenases originally from the bacterium Clostridium acetobutylicum (Original coding sequence: hydA, BBa_K535002 , designed by: iGEM11_UNAM-Genomics_ Mexico. Optimized coding sequence: hydA with SpyTag and Histag BBa_K2132005 ) to accept electrons and therefor enable catalytic production of hydrogen in our project. The optimized coding sequence would produce more protein, theoretically. And optimization also improved the activity of [FeFe] Hydrogenases according to the experiment that we did.
>Improvement:
Codon usage bias adjustment
We analysed the Codon Adaptation Index (CAI) of the optimized coding sequence and the original one. And the distribution of codon usage frequency along the length of the gene sequence is increased from 0.33 to 0.97. A CAI of 1.0 is considered to be perfect in the desired expression organism, and a CAI of > 0.8 is regarded as good, in terms of high gene expression level.
Fig. 4：The distribution of codon usage frequency along the length of the gene sequence.
We also compared the Frequency of Optimal Codons (FOP). The value of 100 is set for the codon with the highest usage frequency for a given amino acid in the desired expression organism. As we can see, the percentage of 91-100 increased largely, from 36 to 86, after the optimization.
Fig. 5：The percentage distribution of codons in computed codon quality groups.
What's more, we removed repeat sequences to break the Stem-Loop structures, which impact ribosomal binding and stability of mRNA.
Max Direct Repeat
Max Inverted Repeat
Max Dyad Repeat
After Optimization
Size:15 Distance:3 Frequency:2
None
None
Before Optimization
Size:16 Distance:231 Frequency:2
None
Size: 13 Tm: 34.6 Start Positions: 680, 1357
Table 1: Removed repeat sequences information
>Conclusion:
A wide variety of factors regulate and influence gene expression levels, and after taking into consideration as many of them as possible, OptimumGene™ produced the single gene that can reach the highest possible level of expression.
In this case, the native gene employs tandem rare codons that can reduce the efficiency of translation or even disengage the translational machinery. We changed the codon usage bias in E. coli by upgrading the CAI from 0.33 to 0.97 . GC content and unfavorable peaks have been optimized to prolong the half-life of the mRNA. The Stem-Loop structures, which impact ribosomal binding and stability of mRNA, were broken.
Fig. 6：The protein alignment of new and old protein.
