Team:ShanghaitechChina/Proof
igem2016:ShanghaiTech
Solar Hunter
Home
Team
Members
Attributions
Project
Integrative Biohydrogen System
Semiconductor Nanomaterials
Engineered Biofilms
Hydrogenase Gene Clusters
Parts
Notebook
Human Practice
InterLab
Safety
Special Prize
Integrated Human Practices
Education and Public Engagement
Measurement
Applied Design
Abstract
Main Achievements
Engineered Biofilm Device 1
Engineered Biofilm Device 2
Hydrogen production and enzyme Function
Solar Hunter in a Nutshell
Solar Hunter is an artificial hydrogen-producing system comprising biofilm-anchored semiconductor nanorods (NRs) which efficiently convert photons to electrons, and engineered strain expressing [FeFe] hydrogenase that can efficiently catalyze Hydrogen production upon receiving the electrons donated by NRs.  The success of this integrative hydrogen-producing system relies on robust construction and functional characterization of each part separately. We have proved that we successfully constructed and characterized our components, as revealed below. In total, we have constructed 4 devices and will introduce them one by one in this session.
For the full demonstration of the system with all the components, please refer to Demonstration of our Work
Major Achievements in Construction of Our Devices
1. Successful production and characterization of engineered biofilms;  Successful demonstration that engineered biofilms composed of CsgA-histagged fusion protein allowed firm binding of semiconductor nanomaterials.
We constructed two major devices in this sub-project and tested their functions:
Device 1:   CsgA-Histag
Device 2:   His-CsgA-SpyCatcher-Histag: BioBrick BBa_K2132001
We will briefly demonstrate the successfulness of two devices below.  For more detailed information about these devices, Please refer to Engineered Biofilms
2. Successful construction of hydrogenase gene clusters  in one plasmid based on Acembl system, as confirmed by gene sequence and successful protein expression revealed by SDS and Western Blot.  This device is named Device 3.
Please refer to Hydrogenase Session for more details.
3. Successful demonstration of normal catalytic properties of hydrogenates using freely-flowing CdS Nanorods. This make device 4.
Engineered Biofilm Device 1
A. To allow easy recycling of precious semiconductor nanomaterials, we utilized engineered biofilms to anchor nanomaterials via metal coordination chemistry. Please refer to Engineered Biofilms for details of the successful construction and characterization of engineered biofilms that allow firm binding of nanomaterials. Key data are reproduced below.
Device 1
First, we constructed and test CsgA-Histag expression and binding validity with nanomaterials under inducer aTc.
1. Congo Red：successful secretion and expression
Fig 1. Fluorescence test of CsgA-His binding with nanomaterials
This assay indicated the success in expression of the self-assembly curli fibers.
2. Crystal Violet Assay： quantification test of biofilm
Further, we used crystal violet assay to obtain quantitative data about the relative density of cells and biofilm adhesion to multi-wells cluster dishes.
Fig 2.Crystal violet assay of CsgA-Histag.
This assay indicates the success in expression of the self-assembly curli fibers. The difference between induced strains secreted CsgA-Histag and ΔCsgA manifest a distinct extracellular biofilm production in the modified strain.
3.Quantum dots fluorescence test: successful binding test of Histag  with nanomaterials (CdSeS/CdSe/ZnS core/shell quantum dots) in macroscopic world
After confirming that our parts success in biofilm expression, we are going to test the effect of binding between CsgA-Histag   mutant and inorganic nanoparticles. The result was consistent with our anticipation: On the left, CsgA-Histag   mutant were induced and QDs are attached with biofilms, thus show bright fluorescence. Therefore, we ensure the stable coordinate bonds between CsgA-Histag   mutant and QDs.The picture was snapped by ChemiDoc MP,BioRad, false colored.
Fig 3. Fluorescence test of CsgA-His binding with nanomaterials
4. TEM: visualization of binding test
Since biofilm nanofibers are thin and inconspicuous against the background, we harness CdSe QDs binding to highlight the biofilm area.
Fig 4.Representative TEM images of biotemplated  CdS quantum dots on CsgA-His.
Finally, transmission electron microscopy(TEM) visualize the microscopic binding effect of CsgA-Histag fused biofilm with CdS nanorods in comparison with image of pure nanofiber composed by CsgA-Histag and one without inducer. Thus we ultimately confirm the viability of bio-abiotic hybrid system.
Fig 5.Representative TEM images of biotemplated  CdS nanorods on CsgA-His.
Engineered Biofilm Device 2
Device 2
Second, we constructed His-CsgA-SpyCatcher-Histag device and successfully characterized its expression and binding effect through the following five steps. we want to achieve the complex design with extra function of binding SpyTag-linked enzymes in addition to its nanomaterial-binding through Histag. This is realized with our Part BBa_K2132001 under the promoter of aTc.
1. Congo Red：successful secretion and expression
After CR dye, the figure indicates that the His-CsgA-SpyCatcher-Histag mutant induced by 0.25 μg ml-1 of aTc and cultured for 72h at 30℃ successfully secreted a thin-layer biofilm on the plate which are stained to brown-red color by CR. This assay also proved that the new and challenging construction of appending a large protein onto CsgA subunits will work accurately and effectively.
Fig 5. Congo Red Assay of His-CsgA-SpyCatcher-Histag.
2. Quantum dots fluorescence test: successful binding test of Histag  with nanomaterials
Then comes to the next part: we want to check if SpyCatcher protein will be too large to cause steric hindrance effect on Histag  peptide. The best approach to verify is the fluorescence assay of binding with nanomaterials. We use His-CsgA-SpyCatcher-Histag as demo.
Fig 6. Fluoresence Nanomaterials Binding Assay of His-CsgA-SpyCatcher-Histag
From this assay, we assure that the SpyCatcher will not impose negative effect on the binding between inorganic material and biofilm.
3. TEM: visualization of binding test
TEM further characterize the biofilm expressed by strains secreted His-CsgA-SpyCatcher-Histag  (HSCH). The distinct nanofiber network manifests the large biofilm expression.
Fig 7. aTc induced secretion of His-CsgA-SpyCatcher-Histag  visualized by TEM. Without the presence of inducer, there’s no nanofiber formation around scattered bacteria.
CsgA-His can interface with different inorganic materials since they form the coordinate bonds with the same ligand, Co-NTA, on nanomaterials. Here we use to AuNPs in place of quantum dots and nanomaterials to characterize the validity of Histags on CsgA fused amyloid protein and meanwhile prove the versatility of our biofilm-based platform.  As the figures shown, we confirm the feasibility of our newly constructed biobricks to template inorganic material and thus form bio-abiotic hybrid system.
Fig 8. After aTc induced, biofilm secreted by His-CsgA-SpyCatcher-Histag   organizes AuNP around the cells.
5.Spy System binding test: successful functional test of SpyCatcher on CsgA fused protein
In the following part, we try to test the viability of SpyCatcher protein fused on CsgA amyloid subunits to see if it’s ideal to bind with SpyTag on Hydrogenase.
As figure illustrated, His-CsgA-SpyCatcher-Histag mutant incubated with mCherry-SpyTag show a clear biofilm-associated mcherry fluorescence signal, which indicating the accurate conformation and function of the SpyTag and SpyCatcher linkage system. The distinct localization highlight of red fluorescence on E.coli, which to a large extent prove the specificity of our desired linkage between SpyTag and SpyCatcher system.
Fig 9.mcherry-SpyTag fluorescence protein binding test of His-CsgA-SpyCatcher-(Histag).
So far above, we proved that two engineered biofilm devices function properly. Later, we tested different inducer concentration gradient to find out the best induction condition.
Inducer concentration optimization
We cultured all E. coli mutants in multi-wells with increasing inducer gradient. The result demonstrated in accordance that 0.25 μg ml-1 of aTc will induce the best expression performance of biofilm, which is exactly the inducer concentration we applied in the project.
Fig 10.Inducer concentration gradient test.
Hydrogenese gene clusters
Hydrogenese gene clusters
High-activity hydrogenase is necessary for our system. To achieve efficient enzymatic activities, we codon-optimized and constructed the whole hydrogenase gene clusters (from Clostridium acetobutylicum) by leveraging the multi-expression Acembl System.  Please refer to Hydrogenase Session for more details.
In this section,we start with a single device of one gene from hydrogenase(From BBa_K2132004 ; BBa_K2132005 ; BBa_K2132006 ; BBa_K2132007 ; BBa_K2132008 ). A single device including following feagment
Different resistance gene for better selection.
Same RBS site to recruit ribosome with equal chance .
Same T7/Lac promoter to help make the moderate expression level of hydrogenase gene clusters.
Same cre-loxp recombination site to utilizes Cre recombinase to integrate four basic device into a compositive one.
BR322 origin in acceptor vs R6k gamma origin in donors to ensure the selection of compositive  device and achieve a stable inheritance
Figure 11 Integration of four basic plasmid backbones into one.
We basically relied on the Acembl system for hydrogenases gene cluster construction and finished the cloning of single device with sequencing confirmation. (Click to see the detail sequenced information: HydA-SpyCatcher , HydA-SpyTag , HydE , HydF , HydG )
In using the system, however, we can either fuse 4 single plasmids with one step of Cre recombination or do it step by step, integrating each plasmid one at a time. In order to gain higher success rate, we choose the second way.  We fused pACE-Histag-TEV-HydA-Spytag/pACE-Histag-TEV-HydA-Spycatcher with pDK-HydF together as the first step. To test if we successfully fused the two, we use single restricted endonuclease digestion of XhoI. The restriction gives two bands on a 1% TAE Gel, in accordance with the band predicted by SnapGene®.
Figure 12A Fusion of plasmid 1 and plasmid 4.
Single restricted-endonuclease digestion of Xhol in pACE-Histag-TEV-HydA-Spytag x  pDK-HydF gives two bands. The left pic refers to expected results based on SnapGene® software prediction, with two bands at 5427bp and 2146bp, respectively.  The right figure refers to the experimental results, which is in good agreement with the software prediction.
Figure 12B Fusion of plasmid 2 and plasmid 4.
Single restricted-endonuclease digestion of Xhol in pACE-Histag-TEV-HydA-Spycatcher x  pDK-HydF gives two bands. The left pic refers to expected results based on SnapGene® software prediction, with two bands at 5427bp and 2455bp, respectively.  The 2455bp is larger than 2146bp due to the larger SpyCatcher. The right figure refers to the experimental results, which is in good agreement with the software prediction.
Figure 12A/B shows that plasmid1/2 and 4 are successfully fused.
Second step:Fusion of plasmid in step one and plasmid 3.
We test through the selection of LB solid plate with three resistance, Ampicillin, Chloramphenicol, and kanamycin. Then we use single restricted endonuclease digestion of XhoI. There should be two kinds of ways in fusing. Comparing our electrophoresis band with the prediction by SnapGene®, we confirmed the kind we obtained.
Figure 12C Fusion of the plasmid in step one(12A) and plasmid 3.
After the fusion of the plasmid in step one and plasmid 3, there will be one more enzyme restriction site of XhoI. Single restricted-endonuclease digestion of Xhol in pACE-Histag-TEV-HydA-SpyTag x  pDK-HydF x  pDC-HydE gives two bands. The left pic refers to expected results based on SnapGene® software prediction, with three bands at 5427bp, 2897bp and 2249bp, respectively. The right figure refers to the experimental results, which is in good agreement with the software prediction.
Figure 12D Fusion of the plasmid in step one(12B) and plasmid 3.
After the fusion of the plasmid in step one and plasmid 3, there will be one more enzyme restriction site of XhoI. Single restricted-endonuclease digestion of Xhol in pACE-Histag-TEV-HydA-SpyCatcher x  pDK-HydF x  pDC-HydE gives two bands. The left pic refers to expected results based on SnapGene® software prediction, with three bands at 5427bp, 2897bp and 2558bp, respectively. The right figure refers to the experimental results, which is in good agreement with the software prediction.
Figure 1C/D shows that plasmids obtained in step 1 and plasmid 3 are successfully fused.
Final step:Fusion of plasmid in step 2 and 5.
This fusion was conferred many possibilities due to the multiple loxP sites that are potentially recognized by Cre, and the fact that some fused loxP sites are reversely separated. However, since the plasmid in step 2 and plasmid 5 are put into the reaction in equal molar, the fully fused plasmid has a better chance. In parallel, we mixed four (plasmid 1/2, 3, 4, 5) plasmids together. After characterization by endonuclease restriction, we obtained the final plasmid. In addition, we find that the mixing of four in one reaction is not efficient.
Figure 12E Fusion of the plasmid in step (12C) and plasmid 3.
For a whole fused plasmid, It becomes hard to analyze it with just Xho I single enzyme. The bar at 3k actually accounts for two bars, with a separation of 20bp. In the picture, although the four bands predicted by SnapGene® can be found on our real gel, it is less clear.
Given the inconvenience with testing by restriction, we turned to resistance screening. The result is that it is resistant to four antibodies (Ampicillin, Chloramphenicol, kanamycin and Spectinomycin).
Expression of the hydrogenase.
As we had successfully get the device,the next step is to induce the expression of the hydrogenase.
To see, we use the antibody of Histag to show the specific of HydA-spycatcher and HydA-spytag and got the result. While we can not avoid the other protein with a similar affinity.
Hydrogen production and enzyme Function
Hydrogen production system with free-flowing CdS nanorod.
The first hydrogen production data using our system is the pink curve (curve 1) in Figure 13. It shows that lighting can induce hydrogen production in a closed system with nano rods (NR), mediator Methyl Viologen, and IPTG-induced bacteria transformed with fused plasmid. To prove that every element of the system is necessary and that it is our hydrogenase that produced the hydrogen rather than NR, we conducted a series of experiments.
To see whether NR is necessary and whether the hydrogen is produced by the reaction between NR and water under lighting rather than our hydrogenase, we conducted the experiment where the system does not contain nano rods or contain only nano rods. The data is summarized in Figure 13A. The red curve (curve 2) represents the system with the transformed bacterial suspension but without nano rods (NR). The flat curve shows that the system without NR could not produce hydrogen with light; NR is necessary for the system. The black curve (curve 3) represents a system in which only NR and mediators are present, with no bacteria. The flat curve shows that it could not produce hydrogen, which proves that the elements of the bacteria is necessary in the synthesis of hydrogen.
Figure 14
click to enlarge the figure
Hydrogen production evolution curve (Sensor Data/ Hydrogen amount vs Time) with different components. The pink curve (curve 1) in all pictures is the hydrogen production with all the components, nano rods (NR), IPTG induction, and the bacteria transformed with our hydrogenase plasmid. The rest are data with one or two components missing. In particular, data in the integrated picture are categorized into Figure 14A and 14B. Figure 14A shows the system with or without nano rods or with nano rods alone, and Figure 14B represents the system with or without induction. The curve 3 in each of the specific figure is the blank control with not transformed E. coli BL21. This series of experiments show that only when both nano rods (NR) and IPTG-induced transformed bacteria are present can the system produce hydrogen in a stable way.
Another step in proving that it is that the hydrogenase is indeed responsible for hydrogen production is to contrast the production level between the induced and un-induced bacteria suspension. The experiment we conducted are summarized in Figure 14B In this set of experiment, the blue line (curve 4) acts as our blank control. In this group, we use the wild type BL21 cells without plasmid. Although we can see a positive oscillation during a short time in the curve, the production was not at high rate and is likely due to the native hydrogenase in E. coli. The green curve (curve 5) represents the transformed bacterial with no induction of IPTG after 12h cultivation. The flat curve shows that it could not produce hydrogen, which proves that the induction of the hydrogenase expression is necessary. To further confirm, we did another experiment using bacteria that have grown 36 hours with no induction. The purple curve (curve 6) clearly contrasts the induced BL21 and the non-induced one. With curve 4 to 6, we have demonstrated that, with the help of NR, it was our hydrogenase in the system that produced the hydrogen we detected.
b) 9.Bidirectional catalytic property of [FeFe] hydrogenase
As mentioned earlier, hydrogenase catalyzes the reversible oxidation of molecular hydrogen (H2). Thus, when we “turn off” the production mode, we should be able to see the consumption of hydrogen by hydrogenase. In testing this bidirectional catalytic property, conducted an experiment where we turned on and turned off the light alternately. The data is shown below in Figure 15. During lighting period, the hydrogen production increases, until we shut off the light at points that correspond to the tips. The curve then goes downward, showing that the hydrogen concentration is lowered, an evidence of the consumption of hydrogen. It is noteworthy that the hydrogenase shows the greatest production rate at the beginning of lighting: a transient sharp rise can be observed at the valleys. It is also obvious that each period of “light-on light-off” gives similar curves, which implies that our hydrogenase is stable.
Figure 15 Verifying the bidirectional catalytic property of [FeFe] hydrogenase.
During the period under lighting, the hydrogen production increases, until we shut off the light at points that correspond to the tips. The curve then goes downward, showing that the hydrogen concentration is lowered, an evidence of the consumption of hydrogen.
igem2016:ShanghaiTech
Solar Hunter
Home
Team
Members
Attributions
Project
Integrative Biohydrogen System
Semiconductor Nanomaterials
Engineered Biofilms
Hydrogenase Gene Clusters
Parts
Notebook
Human Practice
InterLab
Safety
Special Prize
Integrated Human Practices
Education and Public Engagement
Measurement
Applied Design
