<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ShanghaitechChina_Biofilm skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ShanghaitechChina/Biofilm</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>igem2016:ShanghaiTech</TITLE></P><DIV class="container-fluid"><DIV class="navbar-header"><A href="https://2016.igem.org/Team:ShanghaitechChina" class="navbar-brand" style="font-size:28px;margin-top:2%">Solar Hunter</A></DIV><DIV class="collapse navbar-collapse" id="mainNavBar"><UL class="nav navbar-nav navbar-right"><LI><A href="https://2016.igem.org/Team:ShanghaitechChina" style="font-size:20px;">Home</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-size:20px;">Team</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Team" style="font-size:20px;">Members</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Attributions" style="font-size:20px;">Attributions</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-size:20px;  ">Project</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/IBS" style="font-size:20px;">Integrative Biohydrogen System</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Nanomaterials" style="font-size:18px;">Semiconductor Nanomaterials</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Biofilm" style="font-size:18px;">Engineered Biofilms</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Hydrogen" style="font-size:18px;">Hydrogenase Gene Clusters</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Parts" style="font-size:20px;  ">Parts</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Notebook" style="font-size:20px;  ">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Human_Practice" style="font-size:20px;  ">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/InterLab" style="font-size:20px;  ">InterLab</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Safety" style="font-size:20px;  ">Safety</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown" style="font-size:20px;  ">Special Prize</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Integrated_Practices" style="font-size:20px;  ">Integrated Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Engagement" style="font-size:20px;  ">	Education and Public Engagement</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Measurement" style="font-size:20px;  ">Measurement</A></LI><LI><A href="https://2016.igem.org/Team:ShanghaitechChina/Design" style="font-size:20px;  ">Applied Design</A></LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><H1 align="center">Connection to Project</H1><DIV class="col-lg-9"><STRONG>Biofilms function as a platform to sustain the whole system in vitro.</STRONG> Biofilm-anchored nanorods can efficiently convert photons to electrons, which transfer to engineered strain producing FeFe hydrogenase gene cluster, thereby achieving high-efficiency in biohydrogen production. In addition, a brilliant traits, the intrinsic adherence of biofilms towards various interfaces, allows us to grow biofilms on easy-separation micro-beads.<STRONG> Based on those merits, biofilm stand out by facilitating recyclable usage of the biofilm-anchored NRs and endowing this whole system with recyclability.</STRONG></DIV></DIV></DIV></DIV><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><H1 align="center">Introduction</H1></DIV><DIV class="col-lg-12">
             Biofilms are ubiquitous as they can be found both in human and some extreme environments. They can be formed on inert surfaces of devices and equipment, which will be hard to clean and cause dysfunction of the device.
       <STRONG>However, we view it through different lenses to transform this ill impact into merits.</STRONG>  We envision to establish the Solar Hunter system on <EM>E.coli</EM>’s biofilm. Biofilm can substantially increase the resistance of bacteria to adverse conditions like acid or oxidative stress and form a stable and balanced system. These traits make them<STRONG> easily grow with low cost and elevate bacteria’s adaptability.</STRONG> So it will be a good practice to applied to industry environment. 
 </DIV><DIV class="col-lg-8">
What’s more, biofilm can automatically assemble extracellularly grow by static adherence, which facilitates regeneration and recycling in mass production in industry. Startlingly, biofilms can also serve as a<STRONG> synthetic nonconductive biological platform for self-assembling function materials.</STRONG> The amyloid protein CsgA, which is the dominant component in <EM>E.coli</EM>, <STRONG>can be programmed to append small peptide</STRONG> without losing biological activity of peptide and self-assemble function of CsgA. Also, it has been tested that CsgA subunits fused with not too large peptide can be tolerated by curli export machinery and maintain the self-assembly function[1].<STRONG> In all, we highlight the recyclability, stability, scalability, and versatility of biofilm in our system as a design that is truly applicable.</STRONG></DIV></DIV></DIV><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><H1 align="center">Motivation</H1></DIV><DIV class="col-lg-12">
              For the reasons above, biofilms become our best candidate to engineer and would be equipped with some additional functions we want. Here, we conceive the semiconductor-enzyme system linked to the <EM>E.coli</EM>’s biofilm, whose subunits are engineered respectively with PolyHistidine tags and SpyTag and SpyCatcher system from FbaB protein to provide binding sites for inorganic nanomaterials and enzymes. 

Based on these ideas, we constructed several E.coli strains which secreted:
<UL><LI> CsgA-Histag </LI><LI> His-CsgA-SpyCatcher-Histag </LI><LI> His-CsgA-SpyCatcher </LI></UL>
We envision two application system to utilize the biofilm display to establish the whole biohydrogen platform:<H4><B>Plan 1. Strains secrected CsgA-his or His-CsgA-SpyCatcher-(Histag) biofilms for binding nanorods + Strain producing hydrogenase HydA</B></H4><H4><B>Device: CsgA-Histag/His-CsgA-SpyCatcher-(Histag)</B></H4>
We learned that inorganic nanomaterials can template on biofilm by utilizing Co/Ni-NTA- -Histag coordination bond. Therefore, we want leverage synthetic biological engineering to program CsgA biofilm from <EM>E.coli</EM> with Histag to endow its capability to bind with nanomaterials (i.e. quantum dots, nanorods), form a bio-abiotic interface platform and produce electrons. Through this approach, we want to realize producing hydrogen by attach nanorods onto biofilms. The electrons from nanorods excited by sunlight can transfer into engineered hydrogenase-producing strain through mediator solution and accepted by hydrogenases which are not secreted. Since anaerobic hydrogenase will not be exposed to oxygen directly in this way, we view it as a practical and promising way to conduct in lab and consequently realize biohydrogen. <H4><B>Plan 2. Strain secreted His-CsgA-Spycatcher-(Histag) biofilms for binding nanorods + purified  hydrogenase HydA-SpyTag</B></H4><H4><B>Device: His-CsgA-SpyCatcher-(Histag)</B></H4>
Based on this concept, we want to construct a catalytic system outside cells. After extracted and purified from strain which produce hydrogenase, the HydA-Spytag engineered enzyme could covalently bind with SpyCatcher protein on the Strain secrected His-CsgA-Spycatcher-(Histag) biofilms. At the meanwhile, nanorods are firmly attach to biofilm as well for there are histags on biofilm subunits. Electrons from nanorods excited by light thus transfer directly to purified HydA due to short spacial distance and achieve hydrogen production in vitro.
Our ultimate goal is to harness this bio-abiotic hybrid system to efficiently convert solar energy into alternative energy or other high value-added industrial products.</DIV></DIV></DIV><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><H1 align="center">Key Achievements</H1></DIV><DIV class="col-lg-12">
        We tested and proved that all the device we constructed work well:
1. Strains with engineered CsgA subunits :
1) CsgA-Histag   2) His-CsgA-SpyCatcher-Histag   3) His-CsgA-SpyCatcher
can successfully expressed, secreted and realized self-assembly extracellularly.
2. Small peptide histag on CsgA subunits can function well and attach to the ligands on nanorods and quantum dots.
3. Large protein SpyCatcher on CsgA subunits are also able to be secreted by transporter machinery and successfully from nanofibers. We also prove the biological function of SpyCatcher after appending on CsgA subunits, thus provide potential for our second plan mentioned above.
       </DIV></DIV></DIV><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><H1 align="center">Mechanism</H1></DIV><DIV class="col-lg-12">
              We focused on the bacterial amyloid curli structure. The curli consists of two kinds of amyloid proteins bound together and extending on the cell membrane. CsgA, the main subunit, can self-assemble in the extracellular space creating an amyloid nanowire while CsgB is the part which anchors to the membrane, nucleating CsgA and facilitates extension of nanowire.  CsgA is about 13-kDa, whose transcription needs to be regulated by CsgD and expression are processed by CsgE, F and secreted with the assistance of CsgC, G (these all belong to curli genes cluster. After secretion, CsgA assembles automatically to form amyloid nanofibers, whose diameter is around 4-7 nm and length varies [1]. CsgA subunits secreted by different bacteria individuals will not have trouble in assembling and bridging each other, therefore finally achieving the goal as extensive as an organized community network. 
We constructed a library of CsgA biobricks (see<A href="https://2016.igem.org/Team:ShanghaitechChina/Parts">Parts</A>) which are respectively modified with different small peptide domain, endowing the biofilm with designed functions. The expression of CsgA is strictly controlled by inducer anhydrotetracycline (aTc) and its biomass can be tuned by the concentration of inducer (<A href="#p6">Results and Optimization</A>) so that the biofilm is only formed when we need it and is conductive to be well operated when our system is industrialized. Next, we demonstrate the experiments we conducted to test the expression, quantify the biomass, and analyze the viability of different CsgA biobricks.</DIV></DIV></DIV><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><H1 align="center">Construction and Characterization</H1></DIV><DIV class="col-lg-12"><H3 class="bg">Principles of methods of characterization</H3><P><BUTTON type="button" id="1"><B>Congo Red Assay:</B></BUTTON></P><DIV id="x1" style="display:none;width:96%">
Congo Red dye is a classic method to detect amyloid protein [2]. Amyloid can be visualized and quantified through the staining of Congo Red because Congo Red molecules obtain an oriented arrangement on amyloid fibrils. This property can be ascribed to the hydroxyl groups on the amyloid and hydrogen bonding on the Congo Red [3]. It only takes approximately 20 minutes to dye so it is indeed a good practice in lab to crudely test the expression of biofilms.</DIV><P><BUTTON type="button" id="2"><B>Crystal Violet Assay:</B></BUTTON></P><DIV id="x2" style="display:none;width:96%">
Crystal violet is a triarylmethane dye used as a histological stain to classify biomass. This is a simple assay practical and useful for obtaining quantitative data about the relative quantity of cells which adhere to multi-wells cluster dishes. After solubilization, the amount of dye taken up by the monolayer can be quantitated in a plate reader. [4]<P style="text-align:center"><B>Fig 1. </B> Crystal violet and Congo Red reagent.</P></DIV><P><BUTTON type="button" id="3"><B>Transmission Electron Microscope(TEM):</B></BUTTON></P><DIV id="x3" style="display:none">
In order to visualize the formation and different appearance of biofilm nanowire network, we utilize transmission electron microscope to directly look into the microscopic world. TEM can visualize nano-structure with the maximal resolution of 0.2nm which is beyond the range of optical microscope. 

In using TEM, samples must be prepared accordingly. The first step is to apply UAc on objects. After object is covered by UAc, the certain area would absorb or cause scattering of electrons and therefore the detector cannot receive transmissive electrons through copper grid, thus leaving a dark shadowy appearance of sample in the image.<P style="text-align:center"><B>Fig 2. </B> TEM device at the National Center for Protein Science Shanghai.</P></DIV><P><BUTTON type="button" id="4"><B>Quantum Dots Binding Assay:</B></BUTTON></P><DIV id="x4" style="display:none">

Mechanisms of Quantum dots binding assay have been introduced in detail in Quantum Dots part. We utilizing Co/Ni-NTA-Metal-Histag coordination chemistry and fluorescence emission traits of Quantum Dots (QDs) to bind with the histidine in Histags on our biofilm and thus characterize its formation. The whole linkage is performed by forming firm coordinate bonds. They could be applied to quick detection of biofilm expression of His-tagged proteins with naked eye under UV light owing to the photoluminescence of QDs, and accurate concentration measurement under fluorescence spectrum (A detailed protocol for repeatable measurements is included in our Wikipage). </DIV><DIV class="col-lg-12" style="margin-top:30px"><H2 class="bg"><B>PLAN 1</B></H2><H3 class="bg">Construction of CsgA-Histag</H3>
CsgA-HisTag is a part from the previous year IGEM competition. It is documented by team TU_Delft with the Part ID <A href="http://parts.igem.org/Part:BBa_K1583003">BBa_K1583003</A>. However, its status not released. Luckily, we obtained the sequence from Allen Chen at Harvard. The two shared the same amino acid sequence, with some difference in the DNA sequence, possibly modified due to the PARTS Standards. We used the Histags on the CsgA-Histag mutant as the binding site of CdS nanorods, meanwhile, we applied methods described previously to characterize CsgA.<H3 id="Characterization" class="bg">Characterization</H3><H4><B>1. Congo Red:Successful biofilm secretion and expression</B></H4>
The series of Congo Red assay are aim to visualize the expression of biofilms. To produce curli, we spread the CsgA-Histag mutant <EM>E.coli</EM> onto a low-nutrition culture medium, YESCA- CR plates[1] (Details in <A href="https://2016.igem.org/Team:ShanghaitechChina/Notebook#biofilm">protocol:Biofilm Part</A>) Red staining indicates amyloid production.<P style="text-align:center"><B>Fig 3.</B> Congo red assay of CsgA-Histag on YESCA plates</P>
The figures shown above point out that the CsgA-Histag mutant induced by 0.25 µg/ml of aTc produced amyloid structures which are dyed red by CR in comparison to the negative control after 72h culture at 30?. This assay indicates the success in expression of the self-assembly curli fibers. <H4><B>2. Crystal Violet Assay:quantification test of biofilms </B></H4>
Further, we use crystal violet assay to simply obtain quantitative information about the relative density of cells and biofilms adhering to multi-wells cluster dishes. As illustrated in pictures, CsgA-Histag mutant distinguishes itself in absorbance after applying standard crystal violet staining procedures (See<A href="https://2016.igem.org/Team:ShanghaitechChina/Notebook#biofilm">protocol:Biofilm Part</A>) in comparison to strain ?CsgA and 30% acetic acid negative control. There’s certain amount of background absorption of strain ?CsgA because the dye can stain the remaining <EM>E.coli</EM> adhering to the well. This difference between induced strains secreted CsgA-Histag and ?CsgA manifest a distinct extracellular biofilm production in the modified strain. <P style="text-align:center"><B>Fig 4.</B> Crystal violet assay of CsgA-Histag.</P><H4><B>3. Quantum dots fluorescence test: successful binding test of Histag with nanomaterials (CdSeS/CdSe/ZnS core/shell quantum dots)</B></H4></DIV><DIV class="col-lg-7"><B>New characterization of the <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1583003">PART BBa_K1583003</A></B>
After confirming that our parts success in biofilms expression, we are going to test the effect of binding between CsgA-Histag mutant and inorganic nanoparticles. We apply suspended QDs solution into M63 medium which has cultured biofilm for 72h. After 1h incubation, we used PBS to mildly wash the well, and the result was consistent with our anticipation: On the left, CsgA-Histag mutant were induced and QDs are attached with biofilms, thus show bright fluorescence. Therefore, we ensure the stable coordinate bonds between CsgA-Histag mutant and QDs can manage to prevent QDs from being taken away by liquid flow. The picture was snapped by ChemiDoc MP,BioRad, false colored.</DIV><DIV class="col-lg-5"><P style="text-align:center"><B>Fig 5.</B> Fluorescence test of CsgA-His binding with nanomaterials</P></DIV><DIV class="col-lg-12"><H4><B>4. TEM: visualization of binding test</B></H4>
Since biofilm nanofibers are thin and inconspicuous against the background under TEM, we harness CdSe QDs binding to highlight the biofilm area. The first image illustrates biofilm areas which are densely covered by QDs after induced for 72h and incubated, compared to the second image which is not incubated with nanoparticles CdSe. The third one is a negative control without inducer, bacteria scattered without forming biofilm<CENTER><P style="text-align:center"><B>Fig 6.</B> Representative TEM images of biotemplated  CdSe quantum dots on CsgA-His. After applied inducer, CsgA-His mutant constructed and expressed to form biofilm composed by CsgA-His subunits. Incubation with QDs for 1h, nanomaterials are densely attached to biofilm.
</P></CENTER>
Finally, transmission electron microscopy(TEM) visualize the microscopic binding effect of CsgA-Histag fused biofilm with CdS nanorods in comparison with image of pure nanofiber composed by CsgA-Histag and one without inducer. From the first picture, it shows biofilm areas are densely covered by CdS nanorods. As can be clearly seen from the second figure, with inducer, there’s distinct nanofibers outside the bacteria contrast to the third picture in which <EM>E.coli</EM> are not induced. Thus we ultimately confirm the viability of bio-abiotic hybrid system.<P style="text-align:center"><B>Fig 7.</B> Representative TEM images of biotemplated  CdS nanorods on CsgA-Histag. </P><H3 class="bg">Construction of His-CsgA-SpyCatcher-Histag/ His-CsgA-SpyCatcher</H3><B>PARTS:<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2132001">BBa_K2132001</A></B>
In light of the immunization platform of biofilm for enzymes, we need some tags acting like glues or stickers that could be connected to the tags on the enzyme. The SpyCatcher and SpyTag system seem like a good choice for us. The SpyCatcher on the biofilm will mildly bind the SpyTag on the enzyme. Note that there is no the other way around, given that the huge size (138 amino acids) may impair the normal function of some delicate enzyme, hydrogenase in our case. For more details for the principles of SpyCatcher and SpyTag and our motivation on this system, see <A href="#p5">Extracellular Linkage System</A>.  On top of the linkage to the enzyme, we would like to equip the biofilm the ability to bind nanorods and quantum dots. This goal makes the construction of His-CsgA-SpyCatcher-Histag or His-CsgA-SpyCatcher necessary. The two sequences are submitted as our first two original parts. See webpage of the parts here: <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2132001">BBa_K2132001</A>
In constructing the sequence, we simply used Gibson Assembly to assemble the clips of CsgA, SpyCatcher, Histag and the plasmid backbone together at one single reaction. For more details and the sequencing data, please click the <A href="https://static.igem.org/mediawiki/2016/2/21/Histag-CsgA-SpyCatcher-Histag_PROTOCOL.pdf">pdf </A>here.
In constructing the parts, we had been worried about whether the huge SpyCatcher will interfere with the CsgA secretion and whether they will secret together. Careful characterization of each subunit proves that the two parts work excellently, in consistence with previous findings[4].  <H3 class="bg">Characterization</H3>
Since the sequence is actually a fusion protein, we identified each unit individually in characterization.<H4><B>1. Congo Red:successful biofilm secretion and expression</B></H4><B class="tc">His-CsgA-SpyCatcher-Histag</B></DIV><DIV class="col-lg-5"><CENTER><P style="text-align:center"><B>Fig 8.</B> Congo Red Assay of His-CsgA-SpyCatcher-Histag</P></CENTER></DIV><DIV class="col-lg-7">After CR dye, the figure indicates that the His-CsgA-SpyCatcher-Histag mutant induced by 0.25 µg/ml of aTc and cultured for 72h at 30? successfully secreted a thin-layer biofilm on the plate which are stained to brown-red color by CR, compared to the negative control with no inducer. (Because the ratio between Congo Red dye and Brilliant Blue dye is not in the best state which leads to the unapparent phenomenon through the lens, the brown red biofilm is easy to be identified visually.) This assay also proved that the new and challenging construction of appending a large protein onto CsgA subunits will work accurately and effectively.</DIV><DIV class="col-lg-8"><B class="tc">His-CsgA-SpyCatcher</B>
After 72h culture, we scratched the biofilm down from the well and apply 25 µg/ml of Congo Red into solution. Then centrifuged and washed by PBS for several times, we get the result: newly His-CsgA-SpyCatcher mutant induced by 0.25 µg/ml of aTc was stained to bright-red color by CR, compared to the negative control with no inducer and the color can’t be washed away. This assay also manifested the success in construction of His-CsgA-SpyCatcher mutant and add versatility to our biofilm platform.<H4><B>2. Quantum Dots Fluorescence Test: successful binding test of Histag with nanomaterials</B></H4>
Then comes to the next part: we want to check if SpyCatcher protein will be too large to cause steric hindrance effect on Histag peptide. The best approach to verify is the fluorescence assay of binding with nanomaterials. </DIV><DIV class="col-lg-4"><P style="text-align:center"><B>Fig 9.</B> Congo Red Assay of His-CsgA-SpyCatcher</P></DIV><DIV class="col-lg-12"><P><B class="tc">His-CsgA-SpyCatcher-Histag</B></P></DIV><DIV class="col-lg-5"><CENTER><P style="text-align:center"><B>Fig 10.</B> Quantum Dots Templating Assay on His-CsgA-SpyCatcher-Histag Biofilm.</P></CENTER></DIV><DIV class="col-lg-7">
After applying the same steps as introduced above, the bottom of left well show a large area of bright fluorescence, manifesting His-CsgA-SpyCatcher-Histag mutant secreted biofilms under the control of inducer and Histags on it is not blocked by SpyCatcher protein. What is more, it is firmly attached with inorganic materials (i.e.quantum dots) through the ligand. From this assay, we assure that the SpyCatcher will not impose negative effect on the binding between nanomaterial and biofilm. The picture was snapped by ChemiDoc MP, BioRad, false colored.</DIV><DIV class="col-lg-12"><B class="tc">His-CsgA-SpyCatcher</B></DIV><DIV class="col-lg-7">
Using the same approach, we also conducted binding assay of His-CsgA-SpyCatcher with QDs to characterize the expression of biofilm and  the visual result shows vividly that His-CsgA-SpyCatcher can bind successfully with the QDs with the existence of inducer aTc, which shows the functional similarity in CsgA-Histag. The picture was snapped by BioRad ChemiDoc MP, false colored.</DIV><DIV class="col-lg-5"><P style="text-align:center"><B>Fig 11.</B> Quantum dots templating assay on His-CsgA-SpyCatcher biofilm.</P></DIV><DIV class="col-lg-12"><H4><B>3. TEM: visualization of binding test</B></H4>

TEM further characterize the biofilm expressed by strains secreted His-CsgA-SpyCatcher-Histag (HSCH) and His-CsgA-SpyCatcher (HSC) respectively. The distinct nanofiber network manifested the large biofilm expression.<P style="text-align:center"><B>Fig 12.</B> aTc induced secretion of His-CsgA-SpyCatcher-Histag and His-CsgA-SpyCatcher visualized by TEM. Without the presence of inducer, there’s no nanofiber formation around scattered bacteria.</P>
CsgA-His can interface with different inorganic materials since they form the coordinate bonds with the same ligand, Co-NTA, on nanomaterials. Here we use to AuNPs in place of quantum dots and nanomaterials to characterize the validity of Histags on CsgA fused amyloid protein and meanwhile prove the versatility of our biofilm-based platform.  As the figures shown, we confirm the feasibility of our newly constructed biobricks to template inorganic material and thus form bio-abiotic hybrid system.<P style="text-align:center"><B>Fig 13.</B> After aTc induced, biofilm secreted by His-CsgA-SpyCatcher-Histag and His-CsgA-SpyCatcher mutants organize AuNP around the cells. In contrast with the third one without inducer, there’s nothing templating on the seemingly smooth outermembrane of bacteria.</P></DIV></DIV></DIV><DIV class="col-lg-12" style="margin-top:30px"><H2 class="bg"><B>PLAN 2</B></H2><H1 align="center">Extracellular Linkage System</H1></DIV><DIV class="col-lg-12"><H3 class="bg">SpyTag and SpyCatcher [5]</H3><H4><B>Introduction and Motivation: SpySystem</B></H4>
We want to attach enzyme to biofilm, so we turn to a widely applied linkage system, SpyTag and SpyCatcher, originally discovered from Streptococcus pyogenes. By splitting its fibronectin-binding protein FbaB domain, we obtain a relatively small peptide SpyTag with 13 amino acids and a bigger protein partner, SpyCatcher, with 138 amino acids [6]. The advantage of this system lies in the following three aspects. Firstly, they can spontaneously form a covalently stable bond with each other which guarantee the viability of the permanent linkage. The second point is quick reaction within 10 min, which will stand out by its efficiency in industrial application. Besides, the whole process proceeds in mild condition (room temperature), thus set lower requirement for reaction both in lab and future practice. Therefore, we design to leverage this advantageous system to achieve the binding of biofilm with specific enzyme. <H4><B> Design</B></H4>
Appending SpyTag to CsgA subunit is a traditional and hackneyed approach to modify biofilm posttranscriptionally. Here, we challenge to attach larger part, SpyCatcher, to CsgA to enrich the versatility of biofilm platform. For one thing, we intend to pioneer new approach. For another aspect is that we concern SpyCatcher is too large that might jeopardize the biological activity and function of the enzyme. After comprehensive consideration, we decide to append SpyTag and SpyCatcher respectively to CsgA subunits and enzyme, and successfully prove their feasibility and stability.</DIV><DIV><H4><B>Construction of SpyTag-mCherry</B></H4>
The construction of SpyTag-mCherry, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2132003">PART BBa_K2132003</A>, (with SpyTag lying at the N-terminal of mCherry) basically involved two PCR rounds for adding the SpyTag to mCherry. Then the sequence of SpyTag-mCherry was linked to the pET22b(+) backbone between the restriction sites of NdeI, XhoI. This led to the easy induction with IPTG. For more details and the sequencing data, please click the <A href="https://static.igem.org/mediawiki/2016/f/f6/MCherrySpyTagProtocol.pdf">pdf </A>here. The characterization of  SpyTag-mCherry is below.
<H4><B>Characterization</B></H4>
As figure illustrated, His-CsgA-SpyCatcher-Histag mutant incubated with mCherry-SpyTag show a clear biofilm-associated mcherry fluorescence signal, which indicating the accurate conformation and function of the SpyTag and SpyCatcher linkage system. The third figure is merged by the first and second figures of each sample are snapped respectively under green laser field with 558 nm wavelength and bright field of fluorescence microscopy, Zeiss Axio Imager Z2. As for controls, strains secreted CsgA–Histag and ?CsgA both are unable to specifically attach to SpyTag thus no distinct localization highlight of red fluorescence on <EM>E.coli</EM>. That to a large extent prove the specificity of our desired linkage between SpyTag and SpyCatcher system. <P style="text-align:center"><B>Fig 14. </B> The first figures of each sample are snapped under green laser of 558 nm wavelength and mCherry-SpyTags emit red fluorescence. The second figures of each sample are snapped under bright field of fluorescence microscopy and we can clearly see a group of bacteria.. The third figures are merged by the first and second ones. All photos are taken by Zeiss Axio Imager Z2.</P></DIV></DIV><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><H1 align="center">Results and Optimization</H1></DIV><DIV class="col-lg-12">
 
The new CsgA mutants we obtained or newly constructed, and applied in our Solar Hunter project are as follows:
<UL><LI> CsgA-Histag </LI><LI> His-CsgA-SpyCatcher-Histag </LI><LI> His-CsgA-SpyCatcher </LI></UL>
We cultured all E.coli mutants in multi-wells with increasing inducer gradient. The result demonstrated in accordance that 0.25 µg/ml of aTc will induce the best expression performance of biofilm. The possible reason for higher concentration of inducer strangely leading into low production of biofilm might lie in that aTc, a kind of antibiotic, can be harmful to protein synthesis in bacteria. We speculate there’s an antagonism between the effect of promoting expression and impeding growth brought by aTc and 0.25 µg/ml of aTc just reach the optimal point.            
Crystal Violet Assay of the peptide fusion mutants library transformed into ?CsgA strains. Firstly, we apply 0.1% crystal violet solution to all mutants to gain quantitative data about their relative expression and secretion performance. The result was read and exported by BioTek CYTATION5. CsgA-His mutant stands out as the highest peak in absorbance after washing. (See protocal ) The reason why His-CsgA-SpyCatcher-Histag and His-CsgA-SpyCatcher mutant doesn’t perform as well as CsgA-His might be ascribed to size hindrance of SpyCatcher, which impedes the transport process of CsgA mutant subunits from inner area to extracellular environment by CsgG outermenbrane exporter, whose pore size is around 2 nm. Yet, the differences between induced CsgA-His, His-CsgA-SpyCatcher-Histag, His-CsgA-SpyCatcher mutant and negative control exhibit a success extracellular biofilm production in all our constructed and modified strains. 
Nanoparticle binding assay of all constructs library. CdSeS/ZnS core/shell QDs which emit red fluorescence under 365nm UV light are templated by CsgA-His mutant incubated in M63 solution. We added equivalent amount of QDs solution into M63 medium with mutant <EM>E.coli</EM> which have been cultured for approximately 72h and all mutants were induced by aTc. After 1h incubation, we apply PBS washing 3 times to wash away the unbinding quantum dots. Pictures demonstrate CsgA-His were produced by three mutants we constructed and QDs are templated on biofilms .<P style="text-align:center"><B>Fig 15. </B>Nanomaterial binding test. Images were shot by iPhone 5s under 365nm UV light, Tanon UV-100
</P></DIV></DIV></DIV><DIV class="content"><DIV class="row"><DIV class="col-lg-12"><H1 align="center">Reference</H1></DIV><DIV class="col-lg-12">
[1] P. Q. Nguyen, Z. Botyanszki, P. K. R. Tay, N. S. Joshi, Programmable biofilm-based materials from engineered curli nanofibres. Nature communications 5,  (2014). 
[2] A. Marcus, E. Sadimin, M. Richardson, L. Goodell, B. Fyfe, Fluorescence Microscopy Is Superior to Polarized Microscopy for Detecting Amyloid Deposits in Congo Red–Stained Trephine Bone Marrow Biopsy Specimens. American journal of clinical pathology 138, 590-593 (2012).
[3] H. Puchtler, F. Sweat, M. Levine, On the binding of Congo red by amyloid. Journal of Histochemistry &amp; Cytochemistry 10, 355-364 (1962). 
[4] N.-M. Dorval Courchesne, A. Duraj-Thatte, P. K. R. Tay, P. Q. Nguyen, N. S. Joshi, Scalable Production of Genetically Engineered Nanofibrous Macroscopic Materials via Filtration. ACS Biomaterials Science &amp; Engineering,  (2016).
[5] Z. Botyanszki, P. K. R. Tay, P. Q. Nguyen, M. G. Nussbaumer, N. S. Joshi, Engineered catalytic biofilms: Site-specific enzyme immobilization onto E. coli curli nanofibers. Biotechnology and bioengineering 112, 2016-2024 (2015).
[6] B. Zakeri et al., Peptide tag forming a rapid covalent bond to a protein, through engineering a bacterial adhesin. Proceedings of the National Academy of Sciences 109, E690-E697 (2012).
</DIV><DIV class="col-lg-12"><H1 align="center">Acknowledgement<DIV class="col-lg-12"><UL><LI>Our transmission electron microscope work was performed at the National Center for Protein Science Shanghai  </LI></UL></DIV><DIV class="bs-docs-sidebar hidden-print hidden-xs hidden-sm affix"><UL id="sidebar" class="nav bs-docs-sidenav "><LI><A href="#p0">Connection to Project</A></LI><LI><A href="#p1">Introduction</A></LI><LI><A href="#p2">Motivation</A></LI><LI><A href="#p3">Mechanism</A></LI><LI><A href="#p4">Construction and Characterization</A></LI><LI><A href="#p10" sytle="font-size=14px;margin-left:15px">PLAN 1</A></LI><LI><A href="#p5" sytle="font-size=14px;margin-left:15px">PLAN 2</A></LI></UL><LI><A href="#p6">Results and Optimization</A></LI><LI><A href="#p9">Reference and Acknowlegment</A></LI></DIV></H1></DIV></DIV></DIV></BODY></HTML>