Team:ShanghaitechChina/Hydrogen
igem2016:ShanghaiTech
Solar Hunter
Home
Team
Members
Attributions
Project
Integrative Biohydrogen System
Semiconductor Nanomaterials
Engineered Biofilms
Hydrogenase Gene Clusters
Parts
Notebook
Human Practice
InterLab
Safety
Special Prize
Integrated Human Practices
Education and Public Engagement
Measurement
Applied Design
Connection
Why [FeFe] Hydrogenase?
Hydrogenases
Construction
Principle
Result
Expression
Reference
Connection to the Project
In our sun-powered biofilm-interfaced hydrogen-producing system, hydrogenase  harnessed in engineered E.coli are conceived to efficiently catalyze proton reduction upon receiving electrons originally donated by semiconductor nanomaterials. Electron transportation from semiconductors to hydrogenase could be bridged and facilitated by the use of mediators, methyl viologen. To achieve efficient enzymatic activities, we codon-optimized and constructed the whole hydrogenase gene clusters (from Clostridium acetobutylicum) by leveraging the multi-expression Acembl System.
Why [FeFe] Hydrogenase?
Figure 1A Hydrogenase is an enzyme that catalyses the reversible oxidation of molecular hydrogen (H2)
Hydrogenase can be sub-classified into three different types based on the active site metal content: iron-iron hydrogenase ([FeFe] hydrogenase), nickel-iron hydrogenase ([NiFe] hydrogenases), and iron hydrogenase. In contrast to [NiFe] hydrogenases, [FeFe] hydrogenases are generally more active in production of molecular hydrogen. Turnover frequency (TOF) in the order of 10,000 s−1 have been reported in literature for [FeFe] hydrogenases from Clostridium pasteurianum.[1] This has led to intense research focusing on the use of [FeFe] hydrogenase for sustainable production of H2.[2]
Normal E. coli bacteria contain [NiFe] hydrogenase, but the activity and expressive rate is non-obvious. For the above reasons, we decided to construct  [FeFe] hydrogenases gene cluster for sustainable production of H2.
Figure 1B The inner structure of [FeFe]-hydrogensase.
The main functional catalytic group in [FeFe]-hhydrogenase is considered to be an iron-sulfur cluster domain with a di-iron center covalently linked to a dithiolate group.
Hydrogenases
At molecular level, the gene sequences involved in producing hydrogenase in different species vary wildly. In our study, we focus on hydrogenase gene cluster from Clostridium acetobutylicum. The important genes include hydA, hydEF, hydG, which are expressed as HydA, HydE and HydF, HydG respectively.  We will briefly introduce these enzymes below. （Tip：click enzymes to have fun:）
Figure 2A HydA is the main catalytic unit, whereas the rest of the hyd genes are co-expressed to achieve a stable maturation of the final functional HydA.
Figure 2B HydE, as well as HydG have a radical-SAM motif. In most of the cases, these two enzymes might form a complex to fulfill their functions in helping the HydA mature.
Figure 2C HydF, whose N-termiatal domain is homoligous to the GTPase family and C-terminatal domain putatively contains a iron-sulfur center binding motif CxHx45HCxxC,is considered to provide energy during the process.
Figure 2D HydG,besides a radical-SAM motif Cx3Cx2C,it also has motif Cx2Cx22C to bind the redox center- [4Fe4S] cluster. The [4Fe4S] cubic cluster might transer to HydA.
Our goal is to transplant the gene clusters of [FeFe]-hydrogenase from Clostridium acetobutylicum into E. coli, and engineer a strain that could effectively produce hydrogen. Previous work for transferring [FeFe]-hydrogenase into E.coli using a two-plasmid system been demonstrated by Yuki Honda, et al. [4]  Specifically, they used the pETDuet-1 and pCDFDuet-1 system to carry the hydEA and hydFG sequence separately.  However, their method for gene manipulation was laborious and the results were not efficient, as expression of HydA, HydE, HydF, HydG is not controlled in a synchronized way. In addition, the two-plasmid system runs certain risk in the stability of the strain[4]. We made significant improvements on the system using a high-efficiency and multi-expression Acembl system by leveraging the power of synthetic biology, .
Construction of [FeFe]-hydrogenases gene cluster
Principle of Molecular Cloning
To ensure normal enzyme activity, we need to make sure that these four enzymes are simultaneously expressed in E. coli with a moderate amount. The well-established high-efficiency Acembl system [5] came into our sight.
We adopted this Acembl system as a multi-expression system with special DNA replication origin and Cre-loxP site, which utilizes Cre recombinase to integrate four basic plasmid backbones into  one. (Figure 3) Descriptions are as follows.
The Acembl system in our project involves four plasmids, pACE, pDC, pDS, and pDk, and each contains one of the four gene sequences we would like to fuse (Figure 3A-D).
Figure 3A Integration of four basic plasmid backbones into one.
Figure 3B 1.Histag-TEV-HydA-Spytag in pACE(pACE-HydA-Tag in abbreviaFon/pladmid 1) Figure 3C 3.HydE in pDC(pDC-HydE in abbreviaFon/plasmid3) Figure 3D 4. HydF in pDK (pDK-HydF in abbreviaFon/plasmid4) Figure 3E 5. HydG in pDS(pDS-HydG in abbreviaFon/plasmid5)
Figure 3B-E The single plasmids to fuse by Acembl system. We obtained five sequence-confirmed single plasmids including the RBS, promoter region and loxP site. All those functional sequence have been sequenced.
(Click to see the detail sequenced information: HydA-SpyCatcher , HydA-SpyTag , HydE , HydF , HydG )
In particular, pACE is the “acceptor” plasmid with hydA sequence, while others are the “donor” plasmids with the auxiliary protein sequences. With one-step Cre recombination and subsequent transformation into BL21 or DH5a, we would obtain strictly fused plasmid with either all gene circuits integrated in one big plasmid or non-fused single plasmids. The screening of successful assembly involves different resistance (Ampicillin / Chloramphenicol / spectinomycin) and different kinds of origin. In pACE1, it has a replication origin that can be recognized by common DH5a or BL21. In pDC,pDS,pDk, it has a special origin (R6K gamma ori) can be recognized only by a mutation strain of E. coli. (PirHC or PirLC, which can express pir gene product for its replication.) Only a successful fusion into the acceptor plasmid can it propagate, using the accepters ori. Therefore, we efficiently put all four hyd sequences on one single plasmid, avoiding the potential problems imposed by the two-plasmid system.
The basis of our constructs, the four sequences, are not directly obtained from bacteria. But they are all codon-optimized to ensure high-level expression.  (The original sequences of hydrogenase are found on www.genome.jp. )
Results of cloning
As mentioned before, we basically relied on the Acembl system for hydrogenases gene cluster construction. In using the system, however, we can either fuse 4 single plasmids with one step of Cre recombination or do it step by step, integrating each plasmid one at a time. In order to gain higher success rate, we choose the second way.
First step:Fusion of plasmid 1/2 and plasmid 4
We fused pACE-Histag-TEV-HydA-Spytag/pACE-Histag-TEV-HydA-Spycatcher with pDK-HydF together as the first step. To test if we successfully fused the two, we use single restricted endonuclease digestion of XhoI. The restriction gives two bands on a 1% TAE Gel, in accordance with the band predicted by SnapGene®.
Figure 4A Fusion of plasmid 1 and plasmid 4.
Single restricted-endonuclease digestion of Xhol in pACE-Histag-TEV-HydA-Spytag x  pDK-HydF gives two bands. The left pic refers to expected results based on SnapGene® software prediction, with two bands at 5427bp and 2146bp, respectively.  The right figure refers to the experimental results, which is in good agreement with the software prediction.
Figure 4B Fusion of plasmid 2 and plasmid 4.
Single restricted-endonuclease digestion of Xhol in pACE-Histag-TEV-HydA-Spycatcher x  pDK-HydF gives two bands. The left pic refers to expected results based on SnapGene® software prediction, with two bands at 5427bp and 2455bp, respectively.  The 2455bp is larger than 2146bp due to the larger SpyCatcher. The right figure refers to the experimental results, which is in good agreement with the software prediction.
Figure 4A/B shows that plasmid1/2 and 4 are successfully fused.
Second step:Fusion of plasmid in step one and plasmid 3.
We test through the selection of LB solid plate with three resistance, Ampicillin, Chloramphenicol, and kanamycin. Then we use single restricted endonuclease digestion of XhoI. There should be two kinds of ways in fusing. Comparing our electrophoresis band with the prediction by SnapGene®, we confirmed the kind we obtained.
Figure 4C Fusion of the plasmid in step one(4A) and plasmid 3.
After the fusion of the plasmid in step one and plasmid 3, there will be one more enzyme restriction site of XhoI. Single restricted-endonuclease digestion of Xhol in pACE-Histag-TEV-HydA-SpyTag x  pDK-HydF x  pDC-HydE gives two bands. The left pic refers to expected results based on SnapGene® software prediction, with three bands at 5427bp, 2897bp and 2249bp, respectively. The right figure refers to the experimental results, which is in good agreement with the software prediction.
Figure 4D Fusion of the plasmid in step one(4B) and plasmid 3.
After the fusion of the plasmid in step one and plasmid 3, there will be one more enzyme restriction site of XhoI. Single restricted-endonuclease digestion of Xhol in pACE-Histag-TEV-HydA-SpyCatcher x  pDK-HydF x  pDC-HydE gives two bands. The left pic refers to expected results based on SnapGene® software prediction, with three bands at 5427bp, 2897bp and 2558bp, respectively. The right figure refers to the experimental results, which is in good agreement with the software prediction.
Figure 1C/D shows that plasmids obtained in step 1 and plasmid 3 are successfully fused.
Final step:Fusion of plasmid in step 2 and 5.
This fusion was conferred many possibilities due to the multiple loxP sites that are potentially recognized by Cre, and the fact that some fused loxP sites are reversely separated. However, since the plasmid in step 2 and plasmid 5 are put into the reaction in equal molar, the fully fused plasmid has a better chance. In parallel, we mixed four (plasmid 1/2, 3, 4, 5) plasmids together. After characterization by endonuclease restriction, we obtained the final plasmid. In addition, we find that the mixing of four in one reaction is not efficient.
Figure 4E Fusion of the plasmid in step (4C) and plasmid 3.
For a whole fused plasmid, It becomes hard to analyze it with just Xho I single enzyme. The bar at 3k actually accounts for two bars, with a separation of 20bp. In the picture, although the four bands predicted by SnapGene® can be found on our real gel, it is less clear.
Given the inconvenience with testing by restriction, we turned to resistance screening. The result is that it is resistant to four antibodies (Ampicillin, Chloramphenicol, kanamycin and Spectinomycin). Figure 4E shows that plasmids obtained in step 2 and plasmid 4 are successfully fused. Thus, we obtained a plasmid with all four subunits, HydA, HydE, HydF, HydG, fused together. The next step is to induce the expression of the hydrogenase.
Expression of the hydrogenase.
As we had successfully get the device,the next step is to induce the expression of the hydrogenase.
To see, we use the antibody of Histag to show the specific of HydA-spycatcher and HydA-spytag and got the result. While we can not avoid the other protein with a similar affinity.
Reference
1. C. Bieniossek et al., Automated unrestricted multigene recombineering for multiprotein complex production. Nature methods 6, 447-450 (2009).
2. Y. Honda, H. Hagiwara, S. Ida, T. Ishihara, Application to Photocatalytic H2 Production of a Whole‐Cell Reaction by Recombinant Escherichia coli Cells Expressing [FeFe]‐Hydrogenase and Maturases Genes. Angewandte Chemie,  (2016).
3. P. W. King, M. C. Posewitz, M. L. Ghirardi, M. Seibert, Functional studies of [FeFe] hydrogenase maturation in an Escherichia coli biosynthetic system. Journal of bacteriology 188, 2163-2172 (2006).
4. C. Madden et al., Catalytic turnover of [FeFe]-hydrogenase based on single-molecule imaging. Journal of the American Chemical Society 134, 1577-1582 (2011).
5. P. R. Smith, A. S. Bingham, J. R. Swartz, Generation of hydrogen from NADPH using an [FeFe] hydrogenase. international journal of hydrogen energy 37, 2977-2983 (2012).
