"
Page
Discussion
View source
History
teams
Log in
 
Team:NTNU Trondheim/template
From 2014.igem.org
Team:NTNU_Trondheim/notebook - 2014.igem.org
"
Log in
 
Team:NTNU_Trondheim/Home
From 2014.igem.org
NTNU Genetically Engineered Machines
Previous
Tour Page
Next
Tour Page
Start
Tour!
Home
Project
Overview
Background
Wet Lab
Dry Lab
Human Practices
Future Applications
Safety
Team
Notebook
Overview
Protocols
Plasmids
Sponsors
Project description
Synechocystis conjugation BioBricks
Carbon fixation BioBrick
Project Description
Introduction
CO2 emissions have recieved a lot of attention in modern times, due to concerns that high emission levels are facilitating global warming. Consequently, a lot of research is focused on ways of reducing CO2 emissions from industry, and ways of fixating atmospheric CO2 at a greater than normal rate.
Our project is attempting to produce a BioBrick, which when placed inside photosynthetic bacteria, increases their rate of CO2 fixation. In order to achieve this, we first need to construct a BioBrick that allows inducible expression of non native genes in our chassis; Synechocystis sp. PCC 6803.
Making a Synechocystis compatible vector
In order to start working with Synechocystis, we have to construct a plasmid that will allow insertion of foreign genes into the Synechocystis genome. As Synechocystis does not retain inserted plasmids, transformations must make use of conjugation in order to make the insert part its genome.
To achieve this we have constructed a plasmid with eight parts:
"Right flank" sequence
Constitutionally active promoter + RBS
LacI repressor gene
Kanamycin resistance
LacI inducible promoter + RBS
Red fluorescent protein
"Left flank" sequence
Plasmid backbone
The flank sequences have been chosen such that they are homologous to a pair of sequences in a neutral site of the Synechocystis genome. When this plasmid is transformed into Synechocystis, the DNA between the two flank sequences is conjugated into its genome. By growing cultures in a selection medium containing Kanamycin, the insert is kept inside the genome, allowing us to express foreign genes inside the organism.
Increasing the CO2 fixation rate
After verifying that the expression system is working as intended in Synechocystis, we will replace the fluorescent protein gene with one that is predicted to increase the rate of carbon fixation in the organism. The way we will idenfity this gene is by use of metabolic modelling. Once a candidate gene has been detected, it will be cloned and transformed into Synechocystis.
One candiate gene is glucose oxidase, which is not originally present in Synechocystis. This gene encodes the enzyme Glucose Oxidase, which essentially reduces the oxygen concentration inside the cell. RuBisCO, the CO2 fixating enzyme in photosynthetic organisms, has a high affinity for binding O2, which can interfere with CO2 binding. Reducing O2 concentrations could therefore lead to an increased rate of CO2 fixation in Synechocystis.
Retrieved from " https://2013.igem.org/Team:Cornell/team/bios "
Recent changesa>
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
Retrieved from " http://2014.igem.org/Team:NTNU_Trondheim/template "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
