"
Page
Discussion
View source
History
teams
Log in
 
Team:NTNU Trondheim/Notebook
From 2014.igem.org
Team:NTNU_Trondheim/notebook - 2014.igem.org
"
 
Team:Cornell/notebook
From 2013.igem.org
NTNU Genetically Engineered Machines
Previous
Tour Page
Next
Tour Page
Start
Tour!
Home
Project
Overview
Modelling
Considerations
Safety
Judging criteria
Team
Notebook
Protocols
Acknowledgements
Sponsors
Attributions
Jump to:
April
Week 18
June
Week 23
Week 24
Week 25
Week 26
July
Week 27
Week 28
Week 29
Week 30
Week 31
August
Week 32
Week 33
Week 34
Week 35
September
Week 36
Week 37
Week 38
Week 39
October
Week 40
Week 41
Week 42
Week 43
Notebook
Filter by subteam:
show all subteams
show technical details
"_"
Wet Lab
Dry Lab
Human Practices
Wiki
Presenters
only
only
only
only
only
Week 18
(28/04 - 04/05)
April 29th
show technical details
{{{techdetail}}}
Open Day at NTNU. The NTNU iGEM team explained about synthetic biology and iGEM, as well as showed and explained about some simple experimental methods (e.g. gel electrophoresis).
Week 23
(02/06 - 08/06)
June 3rd
show technical details
Prepared SOC and yB solutions for future lab work.
Sterilized material and solutions needed for future lab work.
June 4th
show technical details
{{{tech}}}
Made LB plates with ampicillin and ampicillin + kanamycin.
June 5th
show technical details
{{{tech}}}
Inoculated E. coli DH5α in SOC medium overnight.
June 6th
show technical details
OD of culture was 0.3160 after 100 minutes.
Made competent E. coli DH5α cells.
Week 24
(09/06 - 15/06)
June 10th
show technical details
A plasmid containing ampicillin resistance was used for the transformation, and the cells were incubated overnight on LB plates with ampicillin. Plates showed a bacterial blanket the next day; the cells were apparently super competent.
Test of heat-shock transformation efficiency of competent E. coli DH5α from June 6th.
June 11th
show technical details
Only 1 µl was used for the transformation, the rest of the BioBrick stock solution was stored -20 °C. BBa_J23101 was located at plate 4, 17F. BBa_B0034 was located at plate 4, 1N and BBa_C0012 was located at plate 4, 1P. The J BioBrick is a promotor region, B BioBrick is an RBS region and C BioBrick is LacI repressor gene. Plates showed decent growth the next day; transformation was a success.
Rehydrated BioBrick BBa_J23101 , BBa_B0034 and BBa_C0012 , and heat-shock transformed them into competent E. coli DH5α cells, and incubated the cultures on LB plates with ampicillin overnight.
June 12th
show technical details
The medium was clear the next day, indicating no growth.
Inoculated BioBrick colonies from June 11th in SOC medium containing ampicillin.
June 13th
show technical details
The failed inoculation attempt on June 12th could indicate something wrong with the LB plates with ampicillin. Growth of non-transformed cells supported this. Most likely the ampicillin was aliquotted to the medium at a temperature causing the antibiotic to denature.
Negative control of non-transformed E. coli DH5α on LB plates with ampicillin.
Week 25
(16/06 - 22/06)
June 16th
show technical details
The negative control showed no growth, meaning the LB plates with ampicillin + kanamycin were made correctly! Plates with J, B and C BioBricks showed reasonable growth the next day.
Negative control of non-transformed E. coli DH5α on LB plates with ampicillin + kanamycin.
Made new LB plates with ampicillin.
Heat-shock transformed E. coli DH5α with BioBricks BBa_J23101 , BBa_B0034 and BBa_C0012 and plated onto LB with ampicillin.
June 17th
show technical details
Left_flank and Right_flank sequences were needed to achieve homologous recombination of insert into Synechocystis sp. PCC 6803 genome. Kanamycin_resistance optimized for Synechocystis was needed as screening mechanism in order to select for transformed colonies. Successful PCR amplification of Left_flank (556 bp) and Kanamycin_resistance (944 bp) sequence. Right_flank (544 bp) sequence amplification failed.
PCR amplification of Left_flank , Righ_flank and Kanamycin_resistance with Synechocystis PCC. δslr0906 as template using touchdown PCR .
Gel electrophoresis verification of PCR products on agarose gel
June 18th
show technical details
J (35 bp), B (12 bp) and C (1153 bp) BioBricks all showed proper band lengths after verification on agarose gel. Note that it is not possible to physically see B and J on the gel, but the backbone attached to B and J could be seen. The mCherry gene did not show as a band on the gel, possibly due to incompatibilities between the mCherry gene and the primers. The mCherry gene was to be used as a marker gene, giving visual confirmation of successful transformation; however, because of the possible compatibility issues we decided to explore other options. In the end we decided to use RFP as marker.
Mini-prep of PCR products from June 17th.
PCR amplification of mCherry gene, BioBricks BBa_J23101 , BBa_B0034 and BBa_C0012 using touchdown PCR .
Gel electrophoresis verification of PCR products on agarose gel
June 19th
show technical details
J and B BioBricks were digested with XbaI, while C BioBrick was digested with NotI-HF. Digest of J, B and C should have gievn a band of 2989 bp, 2097 bp and 2063 bp respectively on the gel; low concentration of DNA after digest, but all samples had correct band lengths.
Digest of BioBricks BBa_J23101 , BBa_B0034 and BBa_C0012 .
Gel electrophoresis verification of digest on agarose gel.
June 20th
/a> solutions for future lab work.a href="http://www.snapgene.com/resources/plasmid_files/fluorescent_protein_genes_and_plasmids/mCherry/">mCherry/div> ∓ctype=text/javascript">/a> solutions for future lab work.a href="http://www.snapgene.com/resources/plasmid_files/fluorescent_protein_genes_and_plasmids/mCherry/">mCherry/div> ="nb-onetech-i nohilite">show technical details
The upstream part (J) was cut with EcoRI-HF and S∓ctype=text/javascript">/a> solutions for future lab work.peI restriction enzymes, the downstream part (B) was cut with XbaI and PstI-HF restriction enzymes, while the psB1C3 backbone was cut with EcoRI-HF, PstI-HF and DpnI restriction enzymes. The restriction mixtures were left at 37 °C, heat killed at 80 °C for 30 minutes, then stored at -20 °C.
Made LB plates with chloramphenicol.
Digest of BBa_J23101 , BBa_B0034 and psB1C3 according to the 3A assembly method.
Week 26
(23/06 - 29/06)
June 23rd
show technical details
Ligation was performed at 42 °C for 20 minutes using Taq ligase with ratio 3 between amount of insert and backbone. The ligation mixture was split into two aliquots (1 µl and 9 µl) and transformed into competent E. coli DH5α at 45 °C and plated out on LB plates with chloramphenicol. The plates showed no growth the next day. Negative control of the chloramphenicol plates did not show growth either, meaning that the plates were not the issue. Other reasons might be: (1) using Taq ligase instead of T4 ligase might alter the ligation efficiency, and hence the transformation efficiency; however, somw growth would still be expected; (2) issues with the enzymatic digestion. According to the 3A assembly method, the downstream part and the backbone is supposed to be cut with PstI, not PstI-HF. PstI-HF is heat resistant and cannot be heat killed. The remaining enzymatic activity of PstI-HF could therefore have disrupted the Taq ligase activity. However, freezing should have inactivated PstI-HF activity, meaning it is not likely to cause problems; and (3) low concentrations of B and J. NanoDrop measured B at 5.3 ng/µl concentration and J at 10.1 ng/µl concentration.
Ligation and heat-shock transformation of digestion mixtures from June 20th.
June 24th
show technical details
Variations from June 23rd included: (1) the ligation was performed at 45 °C for 20 minutes and (2) the entire ligation mixture (10 µl) was transformed into one aliquot of competent E. coli DH5α cells; however, ligation still failed.
The gel verification of digest showed an unexpected band of the J BioBrick, and based on failed ligation attempts, it was decided to inoculate new BioBrick colonies.
Attempted ligation and heat-shock transformation of the digestion mixtures from June 20th again.
Gel electrophoresis verification of digest from June 20th on agarose gel.
Inoculated new colonies of BBa_J23101 , BBa_B0034 and BBa_C0012 .
June 25th
show technical details
Enzymatic digestion performed with similar procedure as June 20th: upstream part (J) cut with EcoRI-HF and SpeI, downstream part (B) cut with XbaI and PstI-HF and psB1C3 backbone cut with EcoRI-HF and PstI-HF. Digestion mixtures were left at 37 °C for one hour, then heat-killed at 80 °C for 30 minutes, before being stored at -20 °C.
Verification of Left_flank, Right_flank and Kanamycin_resistance on gel showed expected bands at 566 bp, 552 bp and 961 bp respectively.
Mini-prep of BioBrick colonies from June 24th.
Enzymatic digestion of newly mini-prepped BBa_J23101 , BBa_B0034 and psB1C3 backbone .
New PCR amplification of Left_flank , Righ_flank and Kanamycin_resistance using touchdown PCR .
Gel electrophoresis verification of PCR products on agarose gel.
June 26th
show technical details
The problem with gel verification of B and J is the length of B and J which is 12 bp and 35 bp respectively. They are too short to appear on the gel, and it is therefore impossible to whether or not the samples contain the correct sequence of DNA. However, because these are BioBricks, it is assumed the transformed E. coli DH5α cells contain correct plasmids.
The ligation procedure was performed as June 23rd with a few exceptions: (1) increased the volume of ligation mixture, and hence the amount of ligase. This was because it was assumed that the residual activity of PstI-HF could be counteracted by increasing Taq ligase concentration; and (2) created a short time series for ligation, one sample at 42 °C for 20 minutes and another one for one hour. Both ligation times resulted in growth on LB plates with chloramphenicol.
Gel electrophoresis verification of BBa_J23101 , BBa_B0034 and psB1C3 backbone backbone digest.
Another attempted ligation of digested B, J and psB1C3 backbone from June 25th, and heat-shock transformation into competent E. coli DH5α cells.
June 27th
show technical details
The ligation procedure was performed as June 26th, except that the ligation time was 20 minutes. Transformation into competent E. coli DH5α cells followed standard protocol. The next day two colonies were observed on the plate.
QIAquick PCR purification of Left_flank , Righ_flank and Kanamycin_resistance PCR product from June 25th.
Digested and ligated Right_Flank, Kanamycin_resistance and psB1A3 backbone , then heat-shock transformed it into competent E. coli DH5α cells.
Inoculated colonies from ligation of BBa_J23101 , BBa_B0034 and psB1C3 backbone from June 26th.
June 28th
show technical details
{{{tech}}}
Mini-prep of inoculated colonies from BBa_J23101 , BBa_B0034 and psB1C3 backbone ligation on June 26th.
Week 27
(30/06 - 06/07)
June 30th
show technical details
. The gel verification of the ligation on June 26th showed only one band corresponding to the length of a backbone (~2000 bp). Normally, the gel would run for 50 minutes at 80 V, but because of the length of JB (47 bp), it was set to run only 30 minutes. However, despite the shorter run-time, no band corresponding to JB was seen. GelGreen binds DNA and visualises it. We assume that shorter DNA segments binds less GelGreen, which would make the absence of a JB band on the gel understandable. We therefore assumed that the JB ligation was a success despite the failed verification. JB ligation to LF and psB1A3 backbone showed growth the next day.
Gel electrophoresis verification of BBa_J23101 , BBa_B0034 and psB1C3 backbone ligation on agarose gel.
Digestion , ligation and heat-shock transformation of the ligated JB with Left_flank and psB1A3 backbone .
July 1st
show technical details
A different enzyme was used for the gel verification process because gel verification thus far had been prone to errors. We speculated that the difference in enzyme activity of EcoRI-HF, PstI-HF, SpeI and XbaI could have caused uneven cutting of the plasmids, which could have produced erroneous bands.
The agarose gel of NotI-HF digested {RF, Kan, psB1A3} and {J,B, psB1A3} showed bands at around 1000 bp for all samples, which is wrong in both cases.
New gel electrophoresis verification of BBa_J23101 , BBa_B0034 and psB1C3 backbone ligation on agarose gel using NotI-HF restriction enzyme.
Also verified Righ_flank , Kanamycin_resistance and psB1A3 backbone ligation from June 27th on agarose gel using NotI-HF.
Inoculation of Left_flank , JB and psB1A3 backbone colonies from June 30th.
July 2nd
show technical details
All samples showed erroneous bands on the agarose gel, and it was therefore decided to restart the whole ligation process.
Mini-prep of {LF, JB, psB1A3} from June 30th.
Gel electrophoresis verification of {RF, Kan, psB1A3}, {J,B, psB1A3} and {Left_flank, JB, psB1A3} on agarose gel using NotI-HF restriction enzyme.
Inoculated new colonies from plates containing colonies from the original BBa_J23101 and BBa_B0034 BioBricks.
July 3rd
show technical details
The gel showed correct band lengths for all samples. J, LF and Kan were digested using EcoRI-HF and SpeI. B and RF were digested using XbaI and PstI, while the backbones were digested using EcoRI-HF and PstI. Since it was not possible to heat-kill the high fidelity PstI enzyme, it was decided to attempt digestion with the non-high fidelity one. This should remove residual enzymatic activity of all restriction enzymes after heat-kill.
Mini-prep of inoculated BBa_J23101 and BBa_B0034 colonies from July 2nd.
Gel electrophoresis verification of J and B on an agarose gel using NotI-HF restriction enzyme.
Digested J, B, Left_flank , Righ_flank , Kanamycin_resistance , psB1A3 backbone and psB1C3 backbone .
July 4th
show technical details
Ratio between J/B and backbone in the ligation mixture was 2.5, and T4 ligase was used. Previously Taq ligase had been used, but because of poor transformation efficiency we decided to try T4 ligase. Growth was seen the next day. NanoDrop of Kan and RF products showed 84.3 and 95.7 ng/mL respectively.
Ligated BBa_J23101 , BBa_B0034 and psB1C3 backbone , then heat-shock transformed plasmid into competent E. coli DH5α .
Amplified Kanamycin_resistance and Righ_flank using touchdown PCR .
July 5th
show technical details
{{{tech}}}
Picked and inoculated one colony from BBa_J23101 , BBa_B0034 and psB1C3 backbone ligation from July 4th.
July 6th
show technical details
No colonies were seen on the LB plates containing presumed ligated Kan and RF. New colonies of J,B and ps1C3 backbone were picked because the inoculated mixture from July 5th was blank, indicating that there had been no growth.
Ligated Kanamycin_resistance , Righ_flank and psB1A3 backbone , then heat-shock transformed plasmid into competent E. coli DH5α .
Picked and inoculated three colonies from BBa_J23101 , BBa_B0034 and psB1C3 backbone ligation from July 4th.
Week 28
(07/07 - 13/07)
July 7th
show technical details
NanoDrop of amplified backbone showed an average concentration of 50 ng/mL for all samples. The ligation ratio of insert and backbone was 3. T4 ligase was used. The next day colonies were observed for both ligations.
Amplification of psB1A3 backbone , psB1C3 backbone and psB1K3 backbone using touchdown PCR .
Digestion , ligation and heat-shock transformed of Left_flank , JB with psB1A3 backbone and Kanamycin_resistance , Righ_flank with psB1A3 backbone.
July 8th
show technical details
Construct was optimised to work in Synechocystis sp. PCC 6803.
Made LB plates with ampicillin and with chloramphenicol.
Heat-shock transformed synthesised promoter + RBS region ( Construct ) into competent E. coli DH5α .
One colony from each ligation on July 7th were picked and inoculated.
July 9th
show technical details
NanoDrop of LF+JB and Kan+RF showed a concentration of 44.7 and 47.6 ng/mL respectively. BBa_E1010 was picked from plate 3, 11N.
Mini-prep of inoculated colonies of Left_flank +JB and Kanamycin_resistance + Righ_flank .
Digestion , ligation and heat-shock transformation of LFJB, BBa_C0012 and psB1C3 backbone .
Rehydrated BioBrick BBa_E1010 and heat-shock transformed into competent E. coli DH5α .
July 10th
show technical details
The sequencing results arrived approximately a week later with news that the sequence that was submitted did not correspond to the desired sequence. The next day BBa_E1010 and KRF did not show growth in the liquid medium.
Gel electrophoresis verification of digest of Left_flank +JB, BBa_C0012 , BBa_E1010 and psB1C3 backbone from July 9th on agarose gel using NotI-HF restriction enzyme.
LFJB on psB1A3 backbone and KRF on psB1A3 backbone sent for sequencing.
Inoculated new colonies of LFJB and KRF and BBa_E1010 because the gel verification did not give expected bands.
July 11th
show technical details
NanoDrop of Construct and LFJB showed an average concentration of 103 and 70 ng/mL respectively. The gel only showed bands corresponding to the length of a backbone (~2000 bp), meaning that all the samples were incorrect.
Mini-prep of Construct and Left_flank +JB samples.
Gel electrophoresis verification of all samples of Construct and LFJB on agarose gel using XbaI and PstI restriction enzymes.
Picked and inoculated new colonies of BBa_E1010 and KRF.
July 12th
show technical details
NanoDrop of BBa_E1010 and KRF showed an average concentration of 96.6 and 118.9 ng/mL respectively. Since PstI was being used instead of PstI-HF due to heat resistance issues with the high fidelity enzyme, the buffer used during digest had to be changed. Instead of using CutSmart buffer, Neb buffer was used for PstI and SpeI. PstI should still work in CutSmart buffer even though it would not have maximum activity; however, due to problems with digests and ligations it was decided to try and change the buffer. The gel showed no bands, and we could only conclude that the samples does not contain the correct DNA sequence.
Mini-prep of BBa_E1010 and KRF solutions.
Digest of E1010, Construct and psB1K3 backbone .
Gel electrophoresis verification of KRF samples on agarose gel.
July 13th
show technical details
The gel still did not show expected bands for BBa_E1010 and Construct .
Made new SOC medium and new LB plates with kanamycin.
Another gel electrophoresis verification attempt with the digests from July 12th.
Week 29
(14/07 - 20/07)
July 14th
show technical details
Tested XbaI, PstI, PstI-HF, EcoRI-HF, SpeI and NotI-HF. All enzymes cut the plasmid properly, and gave expected bands on the gel. This means that there is nothing wrong with the enzymatic activity.
Used a ratio of 1 in the ligation of insert and backbone.
Gel electrophoresis verification of restriction enzymes.
Attempted a new digestion , ligation and heat-shock transformed of Kanamycin_resistance , Righ_flank and psB1A3 backbone .
July 15th
show technical details
NanoDrop after PCR amplification of LF, RF and Kan showed concentration of 54.4, 113 and 69.6 ng/mL respectively. NanoDrop measurements after the digest showed concentrations reaching zero for all samples. It was later realised that wrong combination of enzymes had been used, which neatly explains the puzzling results. The amounts of enzyme and DNA was determined according to enzymatic activity in relation to weight of DNA present.
BBa_K576005 was found in plate 1, well 10A. BBa_K576005 equals J and B BioBrick ligated together. We decided to use this BioBrick instead of J and B separately because of all the trouble we were having.
Gel electrophoresis verification of Left_flank , Righ_flank and Kanamycin_resistance using touchdown PCR .
Digested RF, Kan and psB1A3 backbone .
Rehydrated BioBrick BBa_K576005 and heat-shock transformed into competent E. coli DH5α .
July 16th
show technical details
. NanoDrop of LF, BBa_C0012, BBa_E1010, Kan, RF, psB1C3 backbone, BBa_K576005 and Construct showed 133.4, 14.4, 232.2, 215.7, 228.6, 170.6, 206.2 and 378 ng/mL respectively. We decided to give up on the 3A assembly method, since it obviously was not working for us. Instead we designed primers for Gibson assembly.
PCR amplification of Left_flank , BBa_C0012 , BBa_E1010 , Kanamycin_resistance , Righ_flank , psB1C3 backbone , BBa_K576005 and Construct for Gibson assembly using designed primers.
Made a new batch of competent E. coli DH5α cells.
July 17th
show technical details
The previous attempt resulted in very low concentration of BBa_C0012. This time around three different templates were used; (1) from the initial Mini-Prep from June 25th; (2) from the initial LB plate from June 16th; and (3) DNA from the initial PCR from June 18th. NanoDrop of BBa_C0012 samples were as following; (1) 34.5 ng/mL; (2) 6.3 ng/mL; and (3) 88.4 ng/mL. The gel showed correct bands for all samples apart from the psB1K3 backbone .
PCR amplification of BBa_C0012 for Gibson assembly using designed primers.
Made two batches of LB plates with kanamycin + chloramphenicol + IPTG.
Gel electrophoresis verification of the DNA fragments from July 16th and July 17th. The gel still did not show expected bands.
July 18th
show technical details
{{{techdetail}}}
Gel verification repeated for psB1C3 backbone. DNA fragment did not appear on gel.
PCR amplification of (1) BBa_C0012 , psB1K3 backbone , Kanamycin_resistance previously used for Gibson assembly  and (2) psB1K3 backbone from  the iGEM kit.
PCR products purified with PCR purification kit. Concentration measured with nanodrop.
PCR products verified with gel electrophoresis.
Verification of newly made competent cells were done by transforming with BBa_J23101 .
July 20th
show technical details
{{{techdetail}}}
Colonies with BBa_C0012 was inocluated in 6 ml SOC with ampicillin.
Colonies with BBa_J23115 was inoculated in 6 ml SOC ol>    with chloramphenicol.
Week 30
(21/07 - 27/07)
July 21st
show technical details
{{{techdetail}}}
Mini-prep of BBa_C0012 and BBa_J23115 . Concentration measured at 52.7 ng/µl for BBa_C0012 and 104 ng/µl for BBa_J23115 with nanodrop .
PCR amplification was done with Q5-polymerase of samples consisting of PI with backbone primers, chloramphenicol backbone from mini-prep of BBa_J23115 , earlier Mini-prep BBa_C0012 and original linearized plasmid with backbone primers.
PCR products purified with PCR purification kit. Products run on gel and concentration measured with nanodrop.
July 22nd
show technical details
{{{techdetail}}}
heat-shock transformation of biobricks BBa_C0012 well 2N plate 2 and BBa_ I14044 well 16D plate 2 taken from kit as alternative to LacI repressor BBa_C0012 .
July 23rd
show technical details
{{{techdetail}}}
Colony PCR of biobricks BBa_C0012 and BBa_ I14044 with Q5-polymerase.
PCR products purified with PCR purification kit. Attempted verification with gel electrophoresis, but results were inconclusive.
July 24th
show technical details
{{{techdetail}}}
Colony PCR of biobricks BBa_C0012 and BBa_ I14044 with Q5-polymerase done once more, but with four different colonies.
PCR products purified with PCR purification kit. Samples run on gel.
July 25th
show technical details
{{{techdetail}}}
Colony PCR of biobricks BBa_C0012 , BBa_ I14044 , Right Flank, Left flank, construct and chloramphenicol backbone with phusion enzyme and new primers. PCR amplification was also done with BBa_K576005 and BBa_E1010 . Two duplicates were made of each sample.
PCR products verified with gel electrophoresis. Gel showed that all parts needed for Gibson assembly were successfully made. The backbone made from BBa_C0012 was not seen on the gel.
Duplicate products were mixed together and purified with PCR purification kit. BBa_ I14044 was discarded as BBa_C0012 showed correct bands.
Gibson assembly was performed with the PCR products. Reaction was inoculated at 50 degrees Celsius for 60 minutes. Product stored at -20 degrees Celsius.
July 26th
show technical details
{{{techdetail}}}
heat-shock transformation of Gibson assembly product was done with E. coli DH5α cells.
July 27th
show technical details
{{{techdetail}}}
Inspection of plates from yesterday showed no growth.
Week 31
(28/07 - 03/08)
July 28th
show technical details
{{{techdetail}}}
Gibson assembly done once more with same products as in previous entry. Performed with 0.2 pmol of chloramphenicol backbone, and with 0.6 pmol of remaining DNA fragemnts. Gibson assembly done with a time series of 15 minutes, 30 minutes and 60 minutes.
Partial ligation of Left Flank + BBa_ K576005 , BBa_C0012 + construct + BBa_E1010 and Kanamycin_resistance + Right Flank. Verified with gel electrophoresis.
1  µl of Gibson product stored at -20 degrees Celsius as backup. The rest was transformed with heat-shock transformation into competent E. coli DH5α cells. Plated out on plates with kanamycin and IPTG, and on plates with chloramphenicol, kanamycin and IPTG.
July 29th
show technical details
{{{techdetail}}}
Plates showed one colony. Plates further incubated at 37 degrees Celsius.
Separated the partial ligation products from yesterday with gel electrophoresis. Cut out fragments from gel and purified with gel purification kit. Concentration of fragments measured with nanodrop. Gibson assembly performed with partial ligation fragments at 50 degrees Celsius for 15 minutes.
PCR amplification of (a) Gibson assembly done with partial ligation, (b) Gibson time series (15, 30 and 60 minutes), (c) Gibson assembly done July 25th and for backbones containing either kanamycin or chloramphenicol.
New plates with kanamycin and IPTG and plates with kanamycin, chloramphenicol and IPTG was made.
Transformed a biobrick from well 6B plate 4 containing kanamycin backbone with heat-shock transformation into competent E. coli DH5α cells.
July 30th
show technical details
{{{techdetail}}}
Gel electrophoresis done on PCR products from July 29th. Gel showed that Gibson assembly done July 25th did not result in correct assembly. The Gibson time series showed double bands on gel. The products were left in freezer. The Gibson assembly done with the partial ligation fragments were purified with PCR purification kit and concentration was measured with nanodrop (70.8 ng/µl). Chloramphenicol and kanamycin backbone verified, gel showed weaker bands for the chloramphenicol backbones.
Gibson assembly performed with the Gibson partial ligation and chloramphenicol backbone with a reaction time of 15 minutes at 50 degrees Celsius.
Transformed Gibson assembly product with heat-shock transformation into competent E. coli DH5α cells, except for 1 µl stored as backup.
The single colony from the 15 minutes Gibson assembly done July 28th plated out on plate with kanamycin, chloramphenicol and IPTG and also inoculated in liquid medium overnight.
The PCR product of the biobrick from well 6B plate 4 containing kanamycin backbone was run with gel electrophoresis. Gel showed no DNA fragments of the correct length. Inoculated in liquid medium with kanamycin overnight.
July 31st
show technical details
{{{techdetail}}}
Mini-prep of the inoculated colony of the Gibson 15 minutes assembly. Digested, run on gel and PCR amplified. PCR product run on gel.
Week 32
(04/08 - 10/08)
August 4th
show technical details
{{{techdetail}}}
Inoculation of Gibson product assembled with partial ligation in 5 ml SOC with kanamycin, chloramphenicol and IPTG. Incubated at 37 degrees Celsius overnight.
The Gibson partial ligation was also plated out on plates containing kanamycin, chloramphenicol and IPTG.
August 5th
show technical details
{{{techdetail}}}
Mini-prep of inoculation from yesterday with the Gibson partial ligation. Concentration measured at 67.6 ng/µl with nanodrop.
OD730 of Synechocystis culture  was determined to be 0.346. Culture was concentrated to OD730: 2.5 before transforming with the 15 minutes Gibson PCR product.
August 6th
show technical details
{{{techdetail}}}
The Gibson partial ligation product was amplified with PCR with phusion enzyme.
The amplified Gibson partial ligation product was purified with PCR purification kit.
The purified Gibson partial ligation product was analyzed with gel electrophoresis.
Week 35
(25/08 - 31/08)
August 26th
show technical details
{{{techdetail}}}
Faculty blog – Wrote a blog about iGEM for the Natural Sciences faculty at NTNU.
August 27th
show technical details
{{{techdetail}}}
Interview on local radio (NRK P2 Echo) – discussing importance of molecular biology and iGEM to the global scientific community.
August 27th
show technical details
{{{techdetail}}}
Assoc. Prof. Drew Endy presentation – Presented our project to students and staff of the Biotechnology Institute as a support presentation for Assoc. Prof. Drew Endy.
Week 37
(08/09 - 14/09)
September 12th
show technical details
{{{techdetail}}}
PCR amplification of LF (left flank), RF (right flank) and kan (kanamycin)  from Synechocystis with kanamycin resistance and pSB1C3 from biobrick E1010.
Week 38
(15/09 - 21/09)
September 16th
show technical details
{{{techdetail}}}
Verification of LF, RF, Kan and pSB1C3 was done with gel electrophoresis. Samples were purified with PCR purification kit. LF, RF, Kan, pSB1C3, GOx (glucose oxidase), Lac (lac operator) was digested with EcoRI-HF and PstI-HF for 90 minutes at 37 degrees Celsius. Digests purified with PCR purification kit and stored in freezer.
September 17th
show technical details
{{{techdetail}}}
Concentration of LF, RF, Kan, pSB1C3, GOx and Lac was measured with nanodrop, and amplified with PCR. Two parallels of LF, RF and Kan (no. 1 and no.2) were run. PCR products were verified with gel electrophoresis and purified with PCR purification kit. Purified products stored in freezer.
September 18th
show technical details
{{{techdetail}}}
Digestion of LF, RF, Kan, pSB1C3, GOx, Syn: LF1, RF1, Kan, pSB1C3, GOx, Syn with EcoRI and PstI, LF2 with EcoRI and SpeI, RF2 and Kan with XbaI and PstI. Samples inoculated at 37 degrees Celsius for 90 minutes, and thereafter purified with PCR purification kit. Concentrations were measured with nanodrop. The following digests were ligated with T4-ligase: pSB1C3 + LF1 (1:3), pSB1C3 + RF1 (1:3), pSB1C3 + Kan1 (1:3), pSB1C3 + GOx (1:3), pSB1C3 + Syn (1:5), pSB1C3 + LF2 + Kan2 (1:3:3). Ligation samples left in freezer overnight.
September 19th
show technical details
{{{techdetail}}}
Transformed yesterday’s ligations into E. coli DH5α cells. Inoculated overnight at 37 degrees Celsius.
Week 39
(22/09 - 28/09)
September 22nd
show technical details
{{{techdetail}}}
Two colonies were picked from LF1, RF1, Kan1, GOx, Syn, LF2+Kan2 and used for colony PCR. Colonies of Kan1 or LF2+Kan2 were inoculated in 4 ml SOC containing 1µg/ml chloramphenicol and kanamycin, while the remaining samples were inoculated in 4 ml SOC containing 1µg/ml chloramphenicol. PCR-products were verified with gel electrophoresis.
September 23rd
show technical details
{{{techdetail}}}
Mini-prep of LF1, RF2, Kan1, GOx1, Syn1, LF+Kan1.  Digest of 2x LF2+Kan2 with EcoRI and SpeI. Digests was cleaned with PCR cleanup, and ligated overnight with RF2 and pSB1C3 using T4-ligase.
September 24th
show technical details
{{{techdetail}}}
Transformed 2x of pSB1C3, LF, Kan and RF into E. coli DH5α. Incubated at 37⁰C overnight.
Week 40
(29/09 - 05/10)
September 25th
show technical details
{{{techdetail}}}
Four colonies from two of yesterdays plates were selected, and inoculated in 4 ml SOC with kanamycin. The same colonies were also used for colony PCR. PCR-products verified with gel electrophoresis.
September 26th
show technical details
{{{techdetail}}}
Researcher's Night at NTNU. The NTNU iGEM team explained about synthetic biology and iGEM, as well as showed and explained about some simple experimental methods (e.g. gel electrophoresis).
September 26nd
show technical details
{{{techdetail}}}
Mini-prep of 2 parallels of LF+Kan+RF+pSB2C3 (ligation). Digested with EcoRI and NotI. Samples run on gel, one sample verified and selected for Biobrick shipping.
October 1st
show technical details
{{{techdetail}}}
Concentration of miniprepped biobricks LF (24.8 ng/µl), RF (33.3 ng/µl), Kan (40.9 ng/µl), GOx (30.1 ng/µl), Lac (34.6 ng/µl) and Kan + RF (39.8 ng/µl) were measured with nanodrop.
Retrieved from " https://2013.igem.org/Team:Cornell/notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
Retrieved from " http://2014.igem.org/Team:NTNU_Trondheim/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
