<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NTNU_Trondheim_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NTNU_Trondheim/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NTNU_Trondheim/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NTNU_Trondheim/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NTNU Trondheim/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Team:NTNU_Trondheim/notebook - 2014.igem.org</TITLE></P><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cornell/notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>NTNU Genetically Engineered Machines</TITLE></P><DIV class="tour" style="left:0;display:none"><A id="tourPrev" class="button radius"> PreviousTour Page </A></DIV><DIV class="tour" style="right:0;display:none"><A id="tourNext" class="button radius"> NextTour Page </A></DIV><DIV class="tour" style="right:0;display:none"><A id="tourStart" class="button radius"> StartTour! </A></DIV><DIV class="row header off-canvas"><DIV class="twelve columns centered header no-pad" id="header-wrapper"><UL class="nav-bar"><LI class="pg-home"><A href="https://2014.igem.org/Team:NTNU_Trondheim">Home</A></LI><LI class="has-flyout pg-project pg-project_background pg-project_wetlab pg-project_drylab pg-project_hprac pg-project_safety pg-project_future"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Project">Project</A></LI><LI class="pg-project"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Project">Overview</A></LI><LI class="pg-project_background"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Project/Modelling">Modelling</A></LI><LI class="pg-project_background"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Project/Considerations">Considerations</A></LI><LI class="pg-project_safety"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Project/Safety">Safety</A></LI><LI class="pg-project_safety"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Judging">Judging criteria</A></LI></UL><LI class="pg-outreach pg-outreach_presentations pg-outreach_panels pg-outreach_awareness"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Team">Team</A></LI><LI class="pg-notebook pg-notebook_protocols pg-notebook_plasmids"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Notebook">Notebook</A></LI><LI class="pg-outreach pg-outreach_presentations pg-outreach_panels pg-outreach_awareness"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols">Protocols</A></LI><LI class="has-flyout pg-outreach pg-outreach_presentations pg-outreach_panels pg-outreach_awareness"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Acknowledgements/Sponsors">Acknowledgements</A></LI><LI class="pg-project"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Acknowledgements/Sponsors">Sponsors</A></LI><LI class="pg-project_background"><A href="https://2014.igem.org/Team:NTNU_Trondheim/Acknowledgements/Attributions">Attributions</A></LI></DIV></DIV><DIV class="row"><DIV class="two columns"><UL class="side-nav" style="margin-left:15px"><LI><H6> Jump to: </H6></LI><LI class="nb-month"><DIV style="top:7px">April</DIV></LI><LI id="week18" onclick="weekFilter(this)"><A class="nb-week" href="#18">Week 18</A></LI><LI class="nb-month"><DIV style="top:45px">June</DIV></LI><LI id="week23" onclick="weekFilter(this)"><A class="nb-week" href="#23">Week 23</A></LI><LI id="week24" onclick="weekFilter(this)"><A class="nb-week" href="#24">Week 24</A></LI><LI id="week25" onclick="weekFilter(this)"><A class="nb-week" href="#25">Week 25</A></LI><LI id="week26" onclick="weekFilter(this)"><A class="nb-week" href="#26">Week 26</A></LI><LI class="nb-month"><DIV style="top:65px;margin-left:2px">July</DIV></LI><LI id="week27" onclick="weekFilter(this)"><A class="nb-week" href="#27">Week 27</A></LI><LI id="week28" onclick="weekFilter(this)"><A class="nb-week" href="#28">Week 28</A></LI><LI id="week29" onclick="weekFilter(this)"><A class="nb-week" href="#29">Week 29</A></LI><LI id="week30" onclick="weekFilter(this)"><A class="nb-week" href="#30">Week 30</A></LI><LI id="week31" onclick="weekFilter(this)"><A class="nb-week" href="#31">Week 31</A></LI><LI class="nb-month"><DIV style="top:52px;margin-left:-9px">August</DIV></LI><LI id="week32" onclick="weekFilter(this)"><A class="nb-week" href="#32">Week 32</A></LI><LI id="week33" onclick="weekFilter(this)"><A class="nb-week" href="#33">Week 33</A></LI><LI id="week34" onclick="weekFilter(this)"><A class="nb-week" href="#34">Week 34</A></LI><LI id="week35" onclick="weekFilter(this)"><A class="nb-week" href="#35">Week 35</A></LI><LI class="nb-month"><DIV style="top:52px;margin-left:-22px">September</DIV></LI><LI id="week36" onclick="weekFilter(this)"><A class="nb-week" href="#36">Week 36</A></LI><LI id="week37" onclick="weekFilter(this)"><A class="nb-week" href="#37">Week 37</A></LI><LI id="week38" onclick="weekFilter(this)"><A class="nb-week" href="#38">Week 38</A></LI><LI id="week39" onclick="weekFilter(this)"><A class="nb-week" href="#39">Week 39</A></LI><LI class="nb-month"><DIV style="top:55px;margin-left:-12px">October</DIV></LI><LI id="week40" onclick="weekFilter(this)"><A class="nb-week" href="#40">Week 40</A></LI><LI id="week41" onclick="weekFilter(this)"><A class="nb-week" href="#41">Week 41</A></LI><LI id="week42" onclick="weekFilter(this)"><A class="nb-week" href="#42">Week 42</A></LI><LI id="week43" onclick="weekFilter(this)"><A class="nb-week" href="#43">Week 43</A></LI></UL></DIV><DIV class="ten columns team-bios-container"><DIV class="row" id="ugd-members"><DIV class="twelve columns"><H2 class="centered">Notebook</H2></DIV></DIV><DIV class="row last-ele" style="padding-bottom:0px"><H4 style="margin-left:15px;text-align:left">Filter by subteam: 
					<DIV id="nb-all" class="nb-all" onclick="showAll()">show all subteams</DIV><DIV id="nb-alltech" class="nb-all-i" onclick="showAllTech()" style="margin-right:15px">show technical details</DIV></H4></DIV><DIV class="row last-ele nb-labels" style="padding-bottom:0px"><DIV style="opacity:0.00;width:65px">&quot;_&quot;</DIV><DIV>Wet Lab</DIV><DIV>Dry Lab</DIV><DIV>Human Practices</DIV><DIV>Wiki</DIV><DIV>Presenters</DIV></DIV><DIV class="row"><DIV class="two columns nb-filter"><DIV id="wet-only" class="nb-only" onclick="onlyFilter(this)">only</DIV></DIV><DIV class="two columns nb-filter"><DIV id="dry-only" class="nb-only" onclick="onlyFilter(this)">only</DIV></DIV><DIV class="two columns nb-filter"><DIV id="hprac-only" class="nb-only" onclick="onlyFilter(this)">only</DIV></DIV><DIV class="two columns nb-filter"><DIV id="wiki-only" class="nb-only" onclick="onlyFilter(this)">only</DIV></DIV><DIV class="two columns nb-filter"><DIV id="anim-only" class="nb-only" onclick="onlyFilter(this)">only</DIV></DIV></DIV><DIV id="week18entry" class="nb-week" style:=""><DIV class="twelve columns"><H3 class="centered">Week 18</H3><H5 class="centered">(28/04 - 04/05)</H5><DIV class="entry nb-hprac"><DIV><H6>April 29th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Open Day at NTNU. The NTNU iGEM team explained about synthetic biology and iGEM, as well as showed and explained about some simple experimental methods (e.g. gel electrophoresis).
                </OL></P></DIV></DIV></DIV></DIV><DIV id="week23entry" class="nb-week" style="display: block"><DIV class="twelve columns"><H3 class="centered">Week 23</H3><H5 class="centered">(02/06 - 08/06)</H5><DIV class="entry nb-wet"><DIV><H6>June 3rd
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><P><OL><LI>Prepared <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#2">SOC</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#3">yB</A> solutions for future lab work.</LI><LI>Sterilized material and solutions needed for future lab work.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 4th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Made <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with ampicillin and ampicillin + kanamycin.
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 5th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Inoculated <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A> in <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#2">SOC</A> medium overnight.
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 6th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">OD of culture was 0.3160 after 100 minutes.</DIV><P>		Made <A href="http://unitedscientists.org/labs/norway/NTNU/PhotoSynLab/wiki/table-of-contents-2/protocol-molecular/heat-shock-transformation-2">competent <I>E. coli</I></A> DH5α cells.
</P></DIV></DIV></DIV></DIV><DIV id="week24entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 24</H3><H5 class="centered">(09/06 - 15/06)</H5><DIV class="entry nb-wet"><DIV><H6>June 10th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">A plasmid containing ampicillin resistance was used for the transformation, and the cells were incubated overnight on <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with ampicillin. Plates showed a bacterial blanket the next day; the cells were apparently super competent.</DIV><P>		Test of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#23">efficiency</A> of competent <I>E. coli</I> DH5α from June 6th. 

</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 11th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Only 1 µl was used for the transformation, the rest of the BioBrick stock solution was stored -20 °C. <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A> was located at plate 4, 17F. <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> was located at plate 4, 1N and <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> was located at plate 4, 1P. The J BioBrick is a promotor region, B BioBrick is an RBS region and C BioBrick is LacI repressor gene. Plates showed decent growth the next day; transformation was a success.</DIV><P>		Rehydrated BioBrick <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A>, and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformed</A> them into competent <I>E. coli</I> DH5α cells, and incubated the cultures on <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with ampicillin overnight.

</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 12th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The medium was clear the next day, indicating no growth. </DIV><P>		Inoculated BioBrick colonies from June 11th in <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#2">SOC</A> medium containing ampicillin. 

</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 13th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The failed inoculation attempt on June 12th could indicate something wrong with the LB plates with ampicillin. Growth of non-transformed cells supported this. Most likely the ampicillin was aliquotted to the medium at a temperature causing the antibiotic to denature. </DIV><P>		Negative control of non-transformed <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A> on <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with ampicillin. 

</P></DIV></DIV></DIV></DIV><DIV id="week25entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 25</H3><H5 class="centered">(16/06 - 22/06)</H5><DIV class="entry nb-wet"><DIV><H6>June 16th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The negative control showed no growth, meaning the LB plates with ampicillin + kanamycin were made correctly! Plates with J, B and C BioBricks showed reasonable growth the next day.</DIV><P><OL><LI>Negative control of non-transformed <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A> on <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with ampicillin + kanamycin.</LI><LI>Made new LB plates with ampicillin.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">Heat-shock transformed</A><I>E. coli</I> DH5α with BioBricks <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> and plated onto LB with ampicillin.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 17th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Left_flank and Right_flank sequences were needed to achieve homologous recombination of insert into <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#22"><I>Synechocystis</I> sp. PCC 6803</A> genome. Kanamycin_resistance optimized for <I>Synechocystis</I> was needed as screening mechanism in order to select for transformed colonies. Successful PCR amplification of Left_flank (556 bp) and Kanamycin_resistance (944 bp) sequence. Right_flank (544 bp) sequence amplification failed.</DIV><P><OL><LI>PCR amplification of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A> with <I>Synechocystis</I> PCC. δslr0906 as template using touchdown <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#9">PCR</A>. </LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of PCR products on agarose gel</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 18th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">J (35 bp), B (12 bp) and C (1153 bp) BioBricks all showed proper band lengths after verification on agarose gel. Note that it is not possible to physically see B and J on the gel, but the backbone attached to B and J could be seen. The mCherry gene did not show as a band on the gel, possibly due to incompatibilities between the mCherry gene and the primers. The mCherry gene was to be used as a marker gene, giving visual confirmation of successful transformation; however, because of the possible compatibility issues we decided to explore other options. In the end we decided to use RFP as marker.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#7">Mini-prep</A> of PCR products from June 17th.</LI><LI>PCR amplification of <A href="http://www.snapgene.com/resources/plasmid_files/fluorescent_protein_genes_and_plasmids/mCherry/">mCherry</A> gene, BioBricks <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> using touchdown <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#9">PCR</A>.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of PCR products on agarose gel</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 19th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">J and B BioBricks were digested with <I>Xba</I>I, while C BioBrick was digested with <I>Not</I>I-HF. Digest of J, B and C should have gievn a band of 2989 bp, 2097 bp and 2063 bp respectively on the gel; low concentration of DNA after digest, but all samples had correct band lengths.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#8">Digest</A> of BioBricks <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A>.</LI><LI>Gel electrophoresis verification of digest on agarose gel.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 20th
			<DIV class="">/a&gt; solutions for future lab work.a href=&quot;http://www.snapgene.com/resources/plasmid_files/fluorescent_protein_genes_and_plasmids/mCherry/&quot;&gt;mCherry/div&gt;
∓ctype=text/javascript&quot;&gt;/a&gt; solutions for future lab work.a href=&quot;http://www.snapgene.com/resources/plasmid_files/fluorescent_protein_genes_and_plasmids/mCherry/&quot;&gt;mCherry/div&gt;
=&quot;nb-onetech-i nohilite&quot;&gt;show technical details</DIV></H6><DIV class="nb-tech">The upstream part (J) was cut with <I>Eco</I>RI-HF and <I>S∓ctype=text/javascript&quot;&gt;/a&gt; solutions for future lab work.pe</I>I restriction enzymes, the downstream part (B) was cut with <I>Xba</I>I and <I>Pst</I>I-HF restriction enzymes, while the psB1C3 backbone was cut with <I>Eco</I>RI-HF, <I>Pst</I>I-HF and <I>Dpn</I>I restriction enzymes. The restriction mixtures were left at 37 °C, heat killed at 80 °C for 30 minutes, then stored at -20 °C.</DIV><P><OL><LI>Made <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with chloramphenicol.</LI><LI>Digest of <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3</A> according to the <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#11">3A assembly</A> method.</LI></OL></P></DIV></DIV></DIV></DIV><DIV id="week26entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 26</H3><H5 class="centered">(23/06 - 29/06)</H5><DIV class="entry nb-wet"><DIV><H6>June 23rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Ligation was performed at 42 °C for 20 minutes using <I>Taq</I> ligase with ratio 3 between amount of insert and backbone. The ligation mixture was split into two aliquots (1 µl and 9 µl) and transformed into competent <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A> at 45 °C and plated out on <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with chloramphenicol. The plates showed no growth the next day. Negative control of the chloramphenicol plates did not show growth either, meaning that the plates were not the issue. Other reasons might be: (1) using <I>Taq</I> ligase instead of <I>T4</I> ligase might alter the ligation efficiency, and hence the transformation efficiency; however, somw growth would still be expected; (2) issues with the enzymatic digestion. According to the <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#11">3A assembly</A> method, the downstream part and the backbone is supposed to be cut with PstI, not PstI-HF. PstI-HF is heat resistant and cannot be heat killed. The remaining enzymatic activity of PstI-HF could therefore have disrupted the <I>Taq</I> ligase activity. However, freezing should have inactivated PstI-HF activity, meaning it is not likely to cause problems; and (3) low concentrations of B and J. NanoDrop measured B at 5.3 ng/µl concentration and J at 10.1 ng/µl concentration.</DIV><P><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#12">Ligation</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A> of digestion mixtures from June 20th. 
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 24th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Variations from June 23rd included: (1) the ligation was performed at 45 °C for 20 minutes and (2) the entire ligation mixture (10 µl) was transformed into one aliquot of competent <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A> cells; however, ligation still failed. 
		The gel verification of digest showed an unexpected band of the J BioBrick, and based on failed ligation attempts, it was decided to inoculate new BioBrick colonies.</DIV><P><OL><LI>Attempted <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#12">ligation</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A> of the digestion mixtures from June 20th again.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of digest from June 20th on agarose gel.</LI><LI>Inoculated new colonies of <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A>.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 25th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Enzymatic digestion performed with similar procedure as June 20th: upstream part (J) cut with <I>Eco</I>RI-HF and <I>Spe</I>I, downstream part (B) cut with <I>Xba</I>I and <I>Pst</I>I-HF and psB1C3 backbone cut with <I>Eco</I>RI-HF and <I>Pst</I>I-HF. Digestion mixtures were left at 37 °C for one hour, then heat-killed at 80 °C for 30 minutes, before being stored at -20 °C.
		Verification of Left_flank, Right_flank and Kanamycin_resistance on gel showed expected bands at 566 bp, 552 bp and 961 bp respectively. </DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#7">Mini-prep</A> of BioBrick colonies from June 24th.</LI><LI>Enzymatic digestion of newly mini-prepped <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A>.</LI><LI>New PCR amplification of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A> using touchdown <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#9">PCR</A>.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of PCR products on agarose gel.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 26th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The problem with gel verification of B and J is the length of B and J which is 12 bp and 35 bp respectively. They are too short to appear on the gel, and it is therefore impossible to whether or not the samples contain the correct sequence of DNA. However, because these are BioBricks, it is assumed the transformed <I>E. coli</I> DH5α cells contain correct plasmids. 
		The ligation procedure was performed as June 23rd with a few exceptions: (1) increased the volume of ligation mixture, and hence the amount of ligase. This was because it was assumed that the residual activity of <I>Pst</I>I-HF could be counteracted by increasing <I>Taq</I> ligase concentration; and (2) created a short time series for ligation, one sample at 42 °C for 20 minutes and another one for one hour. Both ligation times resulted in growth on <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with chloramphenicol.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A> backbone digest.</LI><LI>Another attempted <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#12">ligation</A> of digested B, J and psB1C3 backbone from June 25th, and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A> into competent <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A> cells.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 27th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The ligation procedure was performed as June 26th, except that the ligation time was 20 minutes. Transformation into competent <I>E. coli</I> DH5α cells followed standard protocol. The next day two colonies were observed on the plate.</DIV><P><OL><LI>QIAquick PCR purification of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A> PCR product from June 25th.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#8">Digested</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#12">ligated</A> Right_Flank, Kanamycin_resistance and <A href="http://parts.igem.org/Part:pSB1A3">psB1A3 backbone</A>, then <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformed</A> it into competent <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A> cells.</LI><LI>Inoculated colonies from ligation of <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A> from June 26th.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>June 28th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#7">Mini-prep</A> of inoculated colonies from <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A> ligation on June 26th.
</P></DIV></DIV></DIV></DIV><DIV id="week27entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 27</H3><H5 class="centered">(30/06 - 06/07)</H5><DIV class="entry nb-wet"><DIV><H6>June 30th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">. The gel verification of the ligation on June 26th showed only one band corresponding to the length of a backbone (~2000 bp). Normally, the gel would run for 50 minutes at 80 V, but because of the length of JB (47 bp), it was set to run only 30 minutes. However, despite the shorter run-time, no band corresponding to JB was seen. GelGreen binds DNA and visualises it. We assume that shorter DNA segments binds less GelGreen, which would make the absence of a JB band on the gel understandable. We therefore assumed that the JB ligation was a success despite the failed verification. JB ligation to LF and psB1A3 backbone showed growth the next day.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A> ligation on agarose gel.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#8">Digestion</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#12">ligation</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A> of the ligated JB with <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A> and <A href="http://parts.igem.org/Part:pSB1A3">psB1A3 backbone</A>.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 1st
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">A different enzyme was used for the gel verification process because gel verification thus far had been prone to errors. We speculated that the difference in enzyme activity of <I>Eco</I>RI-HF, <I>Pst</I>I-HF, <I>Spe</I>I and <I>Xba</I>I could have caused uneven cutting of the plasmids, which could have produced erroneous bands. 
		The agarose gel of <I>Not</I>I-HF digested {RF, Kan, psB1A3} and {J,B, psB1A3} showed bands at around 1000 bp for all samples, which is wrong in both cases.</DIV><P><OL><LI>New <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">gel electrophoresis verification</A> of <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A> ligation on agarose gel using <I>Not</I>I-HF restriction enzyme.</LI><LI>Also verified <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A> and <A href="http://parts.igem.org/Part:pSB1A3">psB1A3 backbone</A> ligation from June 27th on agarose gel using <I>Not</I>I-HF.</LI><LI>Inoculation of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>, JB and psB1A3 backbone colonies from June 30th.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 2nd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">All samples showed erroneous bands on the agarose gel, and it was therefore decided to restart the whole ligation process.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#7">Mini-prep</A> of {LF, JB, psB1A3} from June 30th.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of {RF, Kan, psB1A3}, {J,B, psB1A3} and {Left_flank, JB, psB1A3} on agarose gel using <I>Not</I>I-HF restriction enzyme.</LI><LI>Inoculated new colonies from plates containing colonies from the original <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A> and <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> BioBricks.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 3rd
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The gel showed correct band lengths for all samples. J, LF and Kan were digested using <I>Eco</I>RI-HF and <I>Spe</I>I. B and RF were digested using <I>Xba</I>I and <I>Pst</I>I, while the backbones were digested using <I>Eco</I>RI-HF and <I>Pst</I>I. Since it was not possible to heat-kill the high fidelity <I>Pst</I>I enzyme, it was decided to attempt digestion with the non-high fidelity one. This should remove residual enzymatic activity of all restriction enzymes after heat-kill.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#7">Mini-prep</A> of inoculated <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A> and <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> colonies from July 2nd.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of J and B on an agarose gel using <I>Not</I>I-HF restriction enzyme.</LI><LI>Digested J, B, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A>, <A href="http://parts.igem.org/Part:pSB1A3">psB1A3 backbone</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A>.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 4th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Ratio between J/B and backbone in the ligation mixture was 2.5, and <I>T4</I> ligase was used. Previously <I>Taq</I> ligase had been used, but because of poor transformation efficiency we decided to try <I>T4</I> ligase. Growth was seen the next day. NanoDrop of Kan and RF products showed 84.3 and 95.7 ng/mL respectively.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#12">Ligated</A><A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A>, then <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformed</A> plasmid into competent <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A>.</LI><LI>Amplified  <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A> using touchdown <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#9">PCR</A>.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 5th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{tech}}}</DIV><P>		Picked and inoculated one colony from <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A> ligation from July 4th.
</P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 6th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">No colonies were seen on the <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> containing presumed ligated Kan and RF. New colonies of J,B and ps1C3 backbone were picked because the inoculated mixture from July 5th was blank, indicating that there had been no growth.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#12">Ligated</A><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A> and <A href="http://parts.igem.org/Part:pSB1A3">psB1A3 backbone</A>, then <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformed</A> plasmid into competent <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A>.</LI><LI>Picked and inoculated three colonies from <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>, <A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A> ligation from July 4th.</LI></OL></P></DIV></DIV></DIV></DIV><DIV id="week28entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 28</H3><H5 class="centered">(07/07 - 13/07)</H5><DIV class="entry nb-wet"><DIV><H6>July 7th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">NanoDrop of amplified backbone showed an average concentration of 50 ng/mL for all samples. The ligation ratio of insert and backbone was 3. <I>T4</I> ligase was used. The next day colonies were observed for both ligations.</DIV><P><OL><LI>Amplification of <A href="http://parts.igem.org/Part:pSB1A3">psB1A3 backbone</A>, <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A> and <A href="http://parts.igem.org/Part:pSB1K3">psB1K3 backbone</A> using touchdown <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#9">PCR</A>.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#8">Digestion</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#12">ligation</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformed</A> of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>, JB with psB1A3 backbone and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A> with psB1A3 backbone.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 8th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Construct was optimised to work in <I>Synechocystis</I> sp. PCC 6803.</DIV><P><OL><LI>Made <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with ampicillin and with chloramphenicol.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">Heat-shock transformed</A> synthesised promoter + RBS region (<A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#19">Construct</A>) into competent <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A>.</LI><LI>One colony from each ligation on July 7th were picked and inoculated.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 9th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">NanoDrop of LF+JB and Kan+RF showed a concentration of 44.7 and 47.6 ng/mL respectively. BBa_E1010 was picked from plate 3, 11N.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#7">Mini-prep</A> of inoculated colonies of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>+JB and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A>+<A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A>.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#8">Digestion</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#12">ligation</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A> of LFJB, <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A>.</LI><LI>Rehydrated BioBrick <A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A> and heat-shock transformed into competent <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A>.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 10th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The sequencing results arrived approximately a week later with news that the sequence that was submitted did not correspond to the desired sequence. The next day BBa_E1010 and KRF did not show growth in the liquid medium.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of digest of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>+JB, <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A>, <A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A> and <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A> from July 9th on agarose gel using <I>Not</I>I-HF restriction enzyme.</LI><LI>LFJB on <A href="http://parts.igem.org/Part:pSB1A3">psB1A3 backbone</A> and KRF on psB1A3 backbone sent for sequencing.</LI><LI>Inoculated new colonies of LFJB and KRF and BBa_E1010 because the gel verification did not give expected bands.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 11th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">NanoDrop of Construct and LFJB showed an average concentration of 103 and 70 ng/mL respectively. The gel only showed bands corresponding to the length of a backbone (~2000 bp), meaning that all the samples were incorrect. </DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#7">Mini-prep</A> of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#19">Construct</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>+JB samples.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of all samples of Construct and LFJB on agarose gel using <I>Xba</I>I and <I>Pst</I>I restriction enzymes.</LI><LI>Picked and inoculated new colonies of <A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A> and KRF.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 12th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">NanoDrop of BBa_E1010 and KRF showed an average concentration of 96.6 and 118.9 ng/mL respectively. Since <I>Pst</I>I was being used instead of <I>Pst</I>I-HF due to heat resistance issues with the high fidelity enzyme, the buffer used during digest had to be changed. Instead of using CutSmart buffer, Neb buffer was used for <I>Pst</I>I and <I>Spe</I>I. <I>Pst</I>I should still work in CutSmart buffer even though it would not have maximum activity; however, due to problems with digests and ligations it was decided to try and change the buffer. The gel showed no bands, and we could only conclude that the samples does not contain the correct DNA sequence.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#7">Mini-prep</A> of <A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A> and KRF solutions.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#8">Digest</A> of E1010, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#19">Construct</A> and <A href="http://parts.igem.org/Part:pSB1K3">psB1K3 backbone</A>.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of KRF samples on agarose gel.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 13th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The gel still did not show expected bands for <A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#19">Construct</A>.</DIV><P><OL><LI>Made new <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#2">SOC</A> medium and new <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with kanamycin.</LI><LI>Another <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">gel electrophoresis verification</A> attempt with the digests from July 12th.</LI></OL></P></DIV></DIV></DIV></DIV><DIV id="week29entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 29</H3><H5 class="centered">(14/07 - 20/07)</H5><DIV class="entry nb-wet"><DIV><H6>July 14th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">Tested <I>Xba</I>I, <I>Pst</I>I, <I>Pst</I>I-HF, <I>Eco</I>RI-HF, <I>Spe</I>I and <I>Not</I>I-HF. All enzymes cut the plasmid properly, and gave expected bands on the gel. This means that there is nothing wrong with the enzymatic activity. 
		Used a ratio of 1 in the ligation of insert and backbone.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of restriction enzymes.</LI><LI>Attempted a new <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#8">digestion</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#12">ligation</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformed</A> of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A> and <A href="http://parts.igem.org/Part:pSB1A3">psB1A3 backbone</A>.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 15th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">NanoDrop after PCR amplification of LF, RF and Kan showed concentration of 54.4, 113 and 69.6 ng/mL respectively. NanoDrop measurements after the digest showed concentrations reaching zero for all samples. It was later realised that wrong combination of enzymes had been used, which neatly explains the puzzling results. The amounts of enzyme and DNA was determined according to enzymatic activity in relation to weight of DNA present.
		BBa_K576005 was found in plate 1, well 10A. BBa_K576005 equals J and B BioBrick ligated together. We decided to use this BioBrick instead of J and B separately because of all the trouble we were having.</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A> using touchdown <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#9">PCR</A>.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#8">Digested</A> RF, Kan and <A href="http://parts.igem.org/Part:pSB1A3">psB1A3 backbone</A>.</LI><LI>Rehydrated BioBrick <A href="http://parts.igem.org/Part:BBa_K576005">BBa_K576005</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformed</A> into competent <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A>.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 16th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">. NanoDrop of LF, BBa_C0012, BBa_E1010, Kan, RF, psB1C3 backbone, BBa_K576005 and Construct showed 133.4, 14.4, 232.2, 215.7, 228.6, 170.6, 206.2 and 378 ng/mL respectively. We decided to give up on the 3A assembly method, since it obviously was not working for us. Instead we designed primers for Gibson assembly.</DIV><P><OL><LI>PCR amplification of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#17">Left_flank</A>, <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A>, <A href="http://parts.igem.org/Part:BBa_E1010">BBa_E1010</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A>, <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#16">Righ_flank</A>, <A href="http://parts.igem.org/Part:pSB1C3">psB1C3 backbone</A>, <A href="http://parts.igem.org/Part:BBa_K576005">BBa_K576005</A> and <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#19">Construct</A> for Gibson assembly using designed primers.</LI><LI>Made a new batch of competent <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#21"><I>E. coli</I> DH5α</A> cells.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 17th
			<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">The previous attempt resulted in very low concentration of BBa_C0012. This time around three different templates were used; (1) from the initial Mini-Prep from June 25th; (2) from the initial LB plate from June 16th; and (3) DNA from the initial PCR from June 18th. NanoDrop of BBa_C0012 samples were as following; (1) 34.5 ng/mL; (2) 6.3 ng/mL; and (3) 88.4 ng/mL. The gel showed correct bands for all samples apart from the <A href="http://parts.igem.org/Part:pSB1K3">psB1K3 backbone</A>.</DIV><P><OL><LI>PCR amplification of <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> for Gibson assembly using designed primers.</LI><LI>Made two batches of <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#1">LB plates</A> with kanamycin + chloramphenicol + IPTG.</LI><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#15">Gel electrophoresis verification</A> of the DNA fragments from July 16th and July 17th. The gel still did not show expected bands.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 18th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI>Gel verification repeated for psB1C3 backbone. DNA fragment did not appear on gel.</LI><LI>PCR amplification of (1) <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A>, <A href="http://parts.igem.org/Part:pSB1K3">psB1K3 backbone</A>,  <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#18">Kanamycin_resistance</A> previously used for Gibson assembly  and (2)  <A href="http://parts.igem.org/Part:pSB1K3">psB1K3 backbone</A> from  the iGEM kit.</LI><LI>PCR products purified with PCR purification kit. Concentration measured with nanodrop.</LI><LI>PCR products verified with gel electrophoresis.</LI><LI>Verification of newly made competent cells were done by transforming with <A href="http://parts.igem.org/Part:BBa_J23101">BBa_J23101</A>.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 20th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI>Colonies with <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> was inocluated in 6 ml SOC with ampicillin.</LI><LI>Colonies with <A href="http://parts.igem.org/Part:BBa_J23115">BBa_J23115</A> was inoculated in 6 ml SOC ol&gt;
			with chloramphenicol.</LI></OL></P></DIV></DIV></DIV></DIV><DIV id="week30entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 30</H3><H5 class="centered">(21/07 - 27/07)</H5><DIV class="entry nb-wet"><DIV><H6>July 21st
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#7">Mini-prep</A> of <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> and <A href="http://parts.igem.org/Part:BBa_J23115">BBa_J23115</A>. Concentration measured at 52.7 ng/µl for <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> and 104 ng/µl for <A href="http://parts.igem.org/Part:BBa_J23115">BBa_J23115</A> with nanodrop . </LI><LI>PCR amplification was done with Q5-polymerase of samples consisting of PI with backbone primers, chloramphenicol backbone from mini-prep of <A href="http://parts.igem.org/Part:BBa_J23115">BBa_J23115</A>, earlier <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#7">Mini-prep</A>BBa_C0012 and original linearized plasmid with backbone primers. </LI><LI>PCR products purified with PCR purification kit. Products run on gel and concentration measured with nanodrop.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 22nd
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A> of biobricks <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> well 2N plate 2 and <A href="http://parts.igem.org/Part:BBa_I14044">BBa_ I14044</A> well 16D plate 2 taken from kit as alternative to LacI repressor <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A>. </LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 23rd
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI> Colony PCR of biobricks <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> and <A href="http://parts.igem.org/Part:BBa_I14044">BBa_ I14044</A> with Q5-polymerase. </LI><LI> PCR products purified with PCR purification kit. Attempted verification with gel electrophoresis, but results were inconclusive.  </LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 24th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI> Colony PCR of biobricks <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> and <A href="http://parts.igem.org/Part:BBa_I14044">BBa_ I14044</A> with Q5-polymerase done once more, but with four different colonies. </LI><LI> PCR products purified with PCR purification kit. Samples run on gel. </LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 25th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI> Colony PCR of biobricks <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A>,  <A href="http://parts.igem.org/Part:BBa_I14044">BBa_ I14044</A>, Right Flank, Left flank, construct and chloramphenicol backbone with phusion enzyme and new primers. PCR amplification was also done with <A href="http://parts.igem.org/Part: BBa_K576005 "> BBa_K576005 </A> and <A href="http://parts.igem.org/Part: &gt; BBa_E1010">  BBa_E1010</A>. Two duplicates were made of each sample.</LI><LI> PCR products verified with gel electrophoresis. Gel showed that all parts needed for Gibson assembly were successfully made. The backbone made from <A href="http://parts.igem.org/Part: &gt; BBa_C0012 ">  BBa_C0012 </A> was not seen on the gel.</LI><LI> Duplicate products were mixed together and purified with PCR purification kit.  <A href="http://parts.igem.org/Part:BBa_I14044">BBa_ I14044</A> was discarded as  <A href="http://parts.igem.org/Part:BBa_C0012">BBa_C0012</A> showed correct bands.</LI><LI> Gibson assembly was performed with the PCR products. Reaction was inoculated at 50 degrees Celsius for 60 minutes. Product stored at -20 degrees Celsius.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 26th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI><A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A> of Gibson assembly product was done with <I>E. coli</I> DH5α cells.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 27th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI> Inspection of plates from yesterday showed no growth.</LI></OL></P></DIV></DIV></DIV></DIV><DIV id="week31entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 31</H3><H5 class="centered">(28/07 - 03/08)</H5><DIV class="entry nb-wet"><DIV><H6>July 28th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI>Gibson assembly done once more with same products as in previous entry. Performed with 0.2 pmol of chloramphenicol backbone, and with 0.6 pmol of remaining DNA fragemnts. Gibson assembly done with a time series of 15 minutes, 30 minutes and 60 minutes.</LI><LI>Partial ligation of Left Flank +<A href="http://parts.igem.org/Part:BBa_K576005">BBa_ K576005</A>, <A href="http://parts.igem.org/Part: BBa_C0012"> BBa_C0012</A> + construct + <A href="http://parts.igem.org/Part: BBa_E1010"> BBa_E1010</A> and Kanamycin_resistance + Right Flank. Verified with gel electrophoresis.</LI><LI>1  µl of Gibson product stored at -20 degrees Celsius as backup. The rest was transformed with <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A> into competent <I>E. coli</I> DH5α cells. Plated out on plates with kanamycin and IPTG, and on plates with chloramphenicol, kanamycin and IPTG.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 29th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI>Plates showed one colony. Plates further incubated at 37 degrees Celsius.</LI><LI>Separated the partial ligation products from yesterday with gel electrophoresis. Cut out fragments from gel and purified with gel purification kit. Concentration of fragments measured with nanodrop. Gibson assembly performed with partial ligation fragments at 50 degrees Celsius for 15 minutes. </LI><LI>PCR amplification of (a) Gibson assembly done with partial ligation, (b) Gibson time series (15, 30 and 60 minutes), (c) Gibson assembly done July 25th and for backbones containing either kanamycin or chloramphenicol.</LI><LI>New plates with kanamycin and IPTG and plates with kanamycin, chloramphenicol and IPTG was made.</LI><LI> Transformed a biobrick from well 6B plate 4 containing kanamycin backbone with <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A> into competent <I>E. coli</I> DH5α cells.</LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 30th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI>Gel electrophoresis done on PCR products from July 29th. Gel showed that Gibson assembly done July 25th did not result in correct assembly. The Gibson time series showed double bands on gel. The products were left in freezer. The Gibson assembly done with the partial ligation fragments were purified with PCR purification kit and concentration was measured with nanodrop (70.8 ng/µl). Chloramphenicol and kanamycin backbone verified, gel showed weaker bands for the chloramphenicol backbones.</LI><LI>Gibson assembly performed with the Gibson partial ligation and chloramphenicol backbone with a reaction time of 15 minutes at 50 degrees Celsius. </LI><LI>Transformed Gibson assembly product with  <A href="https://2014.igem.org/Team:NTNU_Trondheim/Protocols#13">heat-shock transformation</A> into competent <I>E. coli</I> DH5α cells, except for 1 µl stored as backup.</LI><LI>The single colony from the 15 minutes Gibson assembly done July 28th plated out on plate with kanamycin, chloramphenicol and IPTG and also inoculated in liquid medium overnight.</LI><LI>The PCR product of the biobrick from well 6B plate 4 containing kanamycin backbone was run with gel electrophoresis. Gel showed no DNA fragments of the correct length. Inoculated in liquid medium with kanamycin overnight. </LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>July 31st
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI>Mini-prep of the inoculated colony of the Gibson 15 minutes assembly. Digested, run on gel and PCR amplified. PCR product run on gel.</LI></OL></P></DIV></DIV></DIV></DIV><DIV id="week32entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 32</H3><H5 class="centered">(04/08 - 10/08)</H5><DIV class="entry nb-wet"><DIV><H6>August 4th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI>Inoculation of Gibson product assembled with partial ligation in 5 ml SOC with kanamycin, chloramphenicol and IPTG. Incubated at 37 degrees Celsius overnight.</LI><LI>The Gibson partial ligation was also plated out on plates containing kanamycin, chloramphenicol and IPTG. </LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 5th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI>Mini-prep of inoculation from yesterday with the Gibson partial ligation. Concentration measured at 67.6 ng/µl with nanodrop.</LI><LI>OD730 of <I> Synechocystis </I> culture  was determined to be 0.346. Culture was concentrated to OD730: 2.5 before transforming with the 15 minutes Gibson PCR product. </LI></OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>August 6th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL><LI>The Gibson partial ligation product was amplified with PCR with phusion enzyme.</LI><LI>The amplified Gibson partial ligation product was purified with PCR purification kit.</LI><LI>The purified Gibson partial ligation product was analyzed with gel electrophoresis.</LI></OL></P></DIV></DIV></DIV></DIV><DIV id="week35entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 35</H3><H5 class="centered">(25/08 - 31/08)</H5><DIV class="entry nb-hprac"><DIV><H6>August 26th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
 Faculty blog – Wrote a blog about iGEM for the Natural Sciences faculty at NTNU.
               </OL></P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>August 27th<DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Interview on local radio (NRK P2 Echo) – discussing importance of molecular biology and iGEM to the global scientific community.
                </OL></P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>August 27th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Assoc. Prof. Drew Endy presentation – Presented our project to students and staff of the Biotechnology Institute as a support presentation for Assoc. Prof. Drew Endy.                
</OL></P></DIV></DIV></DIV></DIV><DIV id="week37entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 37</H3><H5 class="centered">(08/09 - 14/09)</H5><DIV class="entry nb-wet"><DIV><H6>September 12th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
PCR amplification of LF (left flank), RF (right flank) and kan (kanamycin)  from <I> Synechocystis </I>  with kanamycin resistance and pSB1C3 from biobrick E1010.
</OL></P></DIV></DIV></DIV></DIV><DIV id="week38entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 38</H3><H5 class="centered">(15/09 - 21/09)</H5><DIV class="entry nb-wet"><DIV><H6>September 16th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Verification of LF, RF, Kan and pSB1C3 was done with gel electrophoresis. Samples were purified with PCR purification kit. LF, RF, Kan, pSB1C3, GOx (glucose oxidase), Lac (lac operator) was digested with <I>EcoRI-HF</I> and <I>PstI-HF</I> for 90 minutes at 37 degrees Celsius. Digests purified with PCR purification kit and stored in freezer.                 
</OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 17th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Concentration of LF, RF, Kan, pSB1C3, GOx and Lac was measured with nanodrop, and amplified with PCR. Two parallels of LF, RF and Kan (no. 1 and no.2) were run. PCR products were verified with gel electrophoresis and purified with PCR purification kit. Purified products stored in freezer.
                </OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 18th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Digestion of LF, RF, Kan, pSB1C3, GOx, Syn: LF1, RF1, Kan, pSB1C3, GOx, Syn with <I>EcoRI</I> and <I>PstI</I>, LF2 with <I>EcoRI</I> and <I>SpeI</I>, RF2 and Kan with  <I>XbaI</I> and <I>PstI</I>. Samples inoculated at 37 degrees Celsius for 90 minutes, and thereafter purified with PCR purification kit. Concentrations were measured with nanodrop. The following digests were ligated with T4-ligase: pSB1C3 + LF1 (1:3), pSB1C3 + RF1 (1:3), pSB1C3 + Kan1 (1:3), pSB1C3 + GOx (1:3), pSB1C3 + Syn (1:5), pSB1C3 + LF2 + Kan2 (1:3:3). Ligation samples left in freezer overnight. 
                </OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 19th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Transformed yesterday’s ligations into <I>E. coli</I> DH5α cells. Inoculated overnight at 37 degrees Celsius.
                </OL></P></DIV></DIV></DIV></DIV><DIV id="week39entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 39</H3><H5 class="centered">(22/09 - 28/09)</H5><DIV class="entry nb-wet"><DIV><H6>September 22nd
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Two colonies were picked from LF1, RF1, Kan1, GOx, Syn, LF2+Kan2 and used for colony PCR. Colonies of Kan1 or LF2+Kan2 were inoculated in 4 ml SOC containing 1µg/ml chloramphenicol and kanamycin, while the remaining samples were inoculated in 4 ml SOC containing 1µg/ml chloramphenicol. PCR-products were verified with gel electrophoresis.                 
                </OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 23rd
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Mini-prep of LF1, RF2, Kan1, GOx1, Syn1, LF+Kan1.  Digest of 2x LF2+Kan2 with <I>EcoRI</I> and <I>SpeI</I>. Digests was cleaned with PCR cleanup, and ligated overnight with RF2 and pSB1C3 using T4-ligase.               
                </OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 24th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Transformed 2x of pSB1C3, LF, Kan and RF into <I>E. coli</I> DH5α. Incubated at 37⁰C overnight.                
                </OL></P></DIV></DIV></DIV></DIV><DIV id="week40entry" class="nb-week"><DIV class="twelve columns"><H3 class="centered">Week 40</H3><H5 class="centered">(29/09 - 05/10)</H5><DIV class="entry nb-wet"><DIV><H6>September 25th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Four colonies from two of yesterdays plates were selected, and inoculated in 4 ml SOC with kanamycin. The same colonies were also used for colony PCR. PCR-products verified with gel electrophoresis.                 
                </OL></P></DIV></DIV><DIV class="entry nb-hprac"><DIV><H6>September 26th
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Researcher's Night at NTNU. The NTNU iGEM team explained about synthetic biology and iGEM, as well as showed and explained about some simple experimental methods (e.g. gel electrophoresis).
                </OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>September 26nd
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Mini-prep of 2 parallels of LF+Kan+RF+pSB2C3 (ligation). Digested with <I>EcoRI</I> and <I>NotI</I>. Samples run on gel, one sample verified and selected for Biobrick shipping.                  
                </OL></P></DIV></DIV><DIV class="entry nb-wet"><DIV><H6>October 1st
			 <DIV class="nb-onetech-i nohilite">show technical details</DIV></H6><DIV class="nb-tech">{{{techdetail}}}</DIV><P><OL> 
Concentration of miniprepped biobricks LF (24.8 ng/µl), RF (33.3 ng/µl), Kan (40.9 ng/µl), GOx (30.1 ng/µl), Lac (34.6 ng/µl) and Kan + RF (39.8 ng/µl) were measured with nanodrop.                 
                </OL></P></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="https://2013.igem.org/Team:Cornell/notebook">https://2013.igem.org/Team:Cornell/notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cornell/notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cornell/notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cornell/notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cornell/notebook&amp;oldid=360799" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:NTNU_Trondheim/Notebook">http://2014.igem.org/Team:NTNU_Trondheim/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NTNU_Trondheim/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NTNU_Trondheim/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NTNU_Trondheim/Notebook&amp;oldid=358177" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>