"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Diary/Week 6
From 2012.igem.org
Parts for a reliable and field ready biosensing platform
Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field. We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems.
One minute tour! :)
>> Return to page
>> Return to page
Contents
1 Judging Form
1.1 iGEM Medals for non-software teams
1.1.1 Requirements for a Bronze Medal
1.1.2 Additional Requirements for a Silver Medal
1.1.3 Additional Requirements for a Gold Medal: (one OR more)
1.2 iGEM Prizes
1.2.1 Please explain briefly why you should receive any of these special prizes:
1.3 Team_Parts
1.4 iGEM Safety
1.5 Attribution and Contributions
1.6 Comments
1.6.1 Monday
1.6.2 Tuesday
1.6.3 Wednesday
1.6.4 Thursday
1.6.5 Friday
1.6.6 Saturday
1.6.7 Sunday
Judging Form
Please help the judges by filling out this form. Tell them what medal you think you deserve and why. Tell them which special prizes you should win. Help them find your best parts. Show them how you thought about the safety of your project. Helping the judges will help you too.
Team: Cambridge
Region: Europe
iGEM Year:2012
Track:Foundational Advance
Project Name:Parts for a reliable and field ready biosensing platform
Project Abstract: Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field.
We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems.
Back to wiki
iGEM Medals for non-software teams
We believe our team deserves the following medal:
Bronze
Silver
√Gold
Because we met the following criteria (check all that apply and provide details where needed)
Requirements for a Bronze Medal
√Register the team, have a great summer, and plan to have fun at the Regional Jamboree.
√Successfully complete and submit this iGEM 2012 Judging form.
√Create and share a Description of the team's project using the iGEM wiki and the team's parts using the Registry of Standard Biological Parts.
√Plan to present a Poster and Talk at the iGEM Jamboree.
√Enter information detailing at least one new standard BioBrick Part or Device in the Registry of Standard Biological Parts. Including:
√Primary nucleaic acid sequence
√Description of function
√Authorship
Safety notes, if relevant.
√Acknowedgment of sources and references
√Submit DNA for at least one new BioBrick Part or Device to the Registry.
Additional Requirements for a Silver Medal
√Demonstrate that at least one new BioBrick Part or Device of your own design and construction works as expected; characterize the operation of your new part/device.
√Enter this information and other documentation on the part's 'Main Page' section of the Registry
Part Number(s): BBa_K911004
Additional Requirements for a Gold Medal: (one OR more)
Improve an existing BioBrick Part or Device and enter this information back on the Experience Page of the Registry.
Part Number(s): None
√Help another iGEM team by, for example, characterizing a part, debugging a construct, or modeling or simulating their system.
Link to this information on your wiki. Page name: Team:Cambridge/Outreach/Collaboration
√Outline and detail a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation.
Link to this information on your wiki.
Page name: Team:Cambridge/HumanPractices/Overview , Team:Cambridge/HumanPractices/MarketResearch , Team:Cambridge/HumanPractices/FutureDirections
Back to wiki
iGEM Prizes
All teams are eligible for special prizes at the Jamborees. more... To help the judges, please indicate if you feel you should be evaluated for any of the following special prizes:
√Best Human Practice Advance
√Best Experimental Measurement
Best Model
Please explain briefly why you should receive any of these special prizes:
Best Human Practice Advance:
We feel that we deserve this prize for three reasons:
We explored the impacts, *both positive and negative*, of synthetic biology as a solution to real world problems, through interviewing professionals working in a relevant field, namely the impact of arsenic water contamination in Bangladesh.
We recognized existing problems with the way the current direction of synthetic. On going through the registry we found that most of the characterization data for biosensing parts is often neither comparable nor replicable. We have worked to solve this issue, for example with our ratiometric dual channel output.
*Our project doesn’t stop here*, in Chanel number 6 ( Team:Cambridge/HumanPractices/FutureDirections ) we considered the future implications and technological applications of our project, as well as the means by which it could be improved by subsequent users. We feel that the end to an iGEM project should not be the conclusion of an idea, but the start of it.
Best BioBrick Measurement Approach:
It is absolutely vital that a quantitative, numerical, robust, and flexible measurement approach exists to relay information to a user that is an accurate representation of the input processed by a biological device. Working from these principles, the following was done:
We designed and built Biologger, a *cheap, arduino-based, fully functional automatic rotary device* that has an incorporated ratiolumnometer
Our project is entirely open-sourced and open-platform. We have published source code for the two applications which serve to operate the device, one for PCs and the other for Android devices, as well as the open source circuit design that provides this ratiometric reading. Furthermore, the Android app is able to receive its data wirelessly, which we feel is a great advance in BioBrick measurement.
Our dual-channel luciferase reporter was successfully tested with a dilution series of E.coli transformed with the Lux Operon (under pBAD) biobrick (Part  BBa_K325909) of the Cambridge iGEM 2010 team. It can detect, with good accuracy, both different light intensities, as well as the percentages of blue or orange frequencies in a sample.
Our device was successfully tested using artificial light to detect different frequencies (colours) as well.
Having done all the above, we believe that this fully open-sourced instrumentation kit (mechanical) chassis, electronics, software code), estimated at *$35.00* (or $85.00 if a Bluetooth modem is required), is a complete BioBrick measurement solution for any and all BioBricks with a light output.
Back to wiki
Team_Parts
To help the judges evaluate your parts, please identify 3 of your parts that you feel are best documented and are of the highest quality.
Best new BioBrick part (natural)
BBa_K911003
Best new BioBrick part (engineered)
BBa_K911004
Best improved part(s): None
List any other parts you would like the judges to examine: BBa_K911001 , BBa_K911009 , BBa_K911008
Please explain briefly why the judges should examine these other parts:
Magnesium Sensitive Riboswitch BBa_K911001
As a riboswitch sensing construct, this part is an entirely new type of biosensor (along with the fluoride construct) that could potentially change the way we think about designing input genetic circuits. Unlike the fluoride riboswitch, it is a derepression system and therefore serves to demonstrate the principle that riboswitches can be used regardless of whether they turn on or off their reporter.
Fluorescent ratiometric construct for standardizing promoter output BBa_K911009
Fluorescence is a major cornerstone for biosensors in the registry, however, most parts do not involve the use of a ratiometric output, which has been shown in the literature to provide much more reliable and meaningful data. This part not only furthers the development of ratiometric measurements in molecular biology but due to the choice of promoters and terminators it can be used to characterize the difference in activity between E. coli and B. Subtilis
Fast Germination (B. subtilis) BBa_K911008
This part is entirely novel for the registry and fully utilizes the recombination machinery inherent in the Bacillus chassis. Have spores that can germinate at a faster rate is certainly a worthy achievement and could help with experiments with B. Subtilis that any future iGEM teams may wish to perform.
Back to wiki
iGEM Safety
For iGEM 2012 teams are asked to detail how they approached any issues of biological safety associated with their projects.
The iGEM judges expect that you have answered the four safety questions Safety page on your iGEM 2012 wiki.
Please provide the link to that page: Page name: Team:Cambridge/Safety
Attribution and Contributions
For iGEM 2012 the description of each project must clearly attribute work done by the team and distinguish it from work done by others, including the host labs, advisors, and instructors.
Please provide the link to that page, or comments in the box below: Page name: Team:Cambridge/Attributions
Comments
If there is any other information about your project you would like to highlight for the judges, please provide a link to your wiki page here: Team:Cambridge/Overview/DesignProcess
Week:
1
2
3
4
5
6
7
8
9
10
11
12
13
14
The Final Month
Monday
We are suffering from some setback on the PCR front. Tom and Emmy's PCR re-runs for the large vectors did not work despite lengthening the elongation time to 5 minutes per cycle. After sequence alignments we discovered that the primers may anneal to various sites on the primer, producing a range of undesired products. We will carry out touch down PCR tomorrow to make the annealing conditions more stringent.
PCR of the mOrange gene last Friday also yielded no visible products. Looking at the primers' designs again, we think it might be due to problems with the annealing temperature. Therefore we are setting PCR reactions across a temperature gradient today to find the optimum annealing temperature.
For the first time, we actually did some Gibson assembly in a realistic setting! We (hopefully) fused together the Mg2+ fragment purified last week, along with the riboswitch DNA purified two weeks ago. If all goes according to plan, the transformation on bacillus that we later performed should provide us with some responsive bacteria by tomorrow morning. If not, we'll try again in e.coli, and see if we can make a new biobrick from the riboswitch in this chassis.
Tuesday
A day of abject failure. Bacillus from yesterday has failed to grow at all. We're leaving them in the incubator for another day, but don't hold out much hope of getting any useful results.
The problem may have been our transformation, in which one of the steps may not have been at the correct 37 °C temperature. Consequently, we're re-running the experiment, as well as trying to get the plasmid working in some TOP10 e.coli.
Gel electrophoresis of the mOrange PCR products give no visible bands. Tom discovered that a base was missing from the reverse primer- so we have ordered a revised version of it. Curiously, we also seem to have problems with our positive and negative controls.
Colony PCR of the luciferase vector followed by gel electrophoresis gave a very faint band that seems to be of the right size (the expected product was over 9kb), so the DNA is extracted. We will gibson it with the mOrange once the new primer arrives and the mOrange successfully PCR-ed up.
Touch-down PCR of pSJ150 for the fluorescent constructed yielded a single product- but not one that we expect. We will have to find a new solution.
The biobricks from the registry also appeared. Charlie streaked out the bacteria for later use.
Wednesday
Apparrently, writing off yesterday as a complete failure was somewhat pre-emptive, as it now appears that our bacillus was actually growing, albeit slowly. Given our bacteria seem to have been transformed therefore, we streaked out potentially functional colonies for testing later.
PJ suggested that pSJ150 is too large (~8.2kb) to be PCR-ed up as one-piece. He therefore gave Tom and Emmy split primers, which will amplify the vector in two chunks (3.5kb and 4.5kb). We will run the PCR products tomorrow to see if they work.
Thursday
Two deliveries today, one from our Yale contact and one from Starlabs . On the one hand, enough fluoride related bacillus to make us a couple of biobricks, on the other, enough tips to last us a lifetime. Thanks, you two!
Not all the day was so joy filled though. After growing up our streaked out bacillus, they have turned out to be e.coli. This makes our assessment that our original assessment was to hasty itself too hasty, which, while gratifyingly meta, is none the less frustrating. We're now going to try and debug our Gibson and PCR, and will probably try to rerun the vector fragments that only seemed to partially work.
Furthermore, the plates that we made for our (hopefully) transformed e.coli have nothing growing on them, meaning we probably will have to start from the beginning.
The PCR from yesterday has, however, worked! Having everything in place to perform a Gibson transformation to produce our fluorescent construct, we performed said reaction and transformed it into some TOP10. I will reserve judgement on this until next week.
Friday
It would appear that our isothermal reaction buffer was not correctly mixed, meaning all our Gibson assembly reactions had no chance of success. Having remixed the buffer, we now hope that the Gibson should work properly.
Unfortunately, we have now wasted all our TOP10 e.coli on non-functional Gibson assemblies, so now we have to create many more competent e.coli. Consequently, Jolyon got onto ordering all the materials we will need for electroporation, TOP10 being prohibitively expensive for a puny undergraduate research project.
We have also discovered that the primers for mOrange and luciferase vector (pSB1C3) have been mixed up due to incorrect labelling- that has probably caused all our problems with amplifying up the two over the past week. Nonetheless our new mOrange reversed primer has arrived, so with the corrected labelling and revised primer the mOrange has been amplified up successfully.
So the faint bands we had on Tuesday for the luciferase vector were probably not our desired product afterall, but learning our lesson from attempting to PCR pSJ150 backbone, Tom has designed split primers for the luciferase backbone and they should be delivered on Monday. With those and the correctly labelled primers hopefully we will obtain the right product that we can gibson to the mOrange.
Oli has decided to use the split primers for pSJ150 to PCR the pSJ150 backbone for the Mg riboswitch construct as well- the "product" from last Friday's PCR was rather ambiguous and it doesn't seem to have worked in the Gibson. Primers for Charlie's biosensor constructs have also arrived, so parts of the construct are PCR-ed. Unfortunately, none of these PCRs yielded any products- possibly due to a problem with the PCR master mix. We should try these again later.
Saturday
Unfortunately there are no visible colonies on our ampicillin plates, which means there might be some problems with the Gibson-ing of the fluorescent construct, or the transformation of E. coli. We will observe for another day.
Jolyon also started culturing some cells for electroporation so we could start the protocol on Monday.
Sunday
Still no colonies for the fluorescent construct plates. Meanwhile we redo Oli and Charlie's PCR: half of Oli's vector has came out, as well as Charlie's RFP. We suspect the reasoning for our failure to PCR the Mg promoter might be due to sequence inconsistency of the biobrick from the registry. Oli will attempt the failed PCRs again tomorrow.
Retrieved from " http://2012.igem.org/Team:Cambridge/Diary/Week_6 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
