"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Diary/Week 13
From 2012.igem.org
Parts for a reliable and field ready biosensing platform
Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field. We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems.
One minute tour! :)
>> Return to page
>> Return to page
Contents
1 Judging Form
1.1 iGEM Medals for non-software teams
1.1.1 Requirements for a Bronze Medal
1.1.2 Additional Requirements for a Silver Medal
1.1.3 Additional Requirements for a Gold Medal: (one OR more)
1.2 iGEM Prizes
1.2.1 Please explain briefly why you should receive any of these special prizes:
1.3 Team_Parts
1.4 iGEM Safety
1.5 Attribution and Contributions
1.6 Comments
1.6.1 Monday
1.6.2 Tuesday
1.6.3 Wednesday
1.6.4 Thursday
1.6.5 Friday
1.6.6 Saturday
1.6.7 Sunday
Judging Form
Please help the judges by filling out this form. Tell them what medal you think you deserve and why. Tell them which special prizes you should win. Help them find your best parts. Show them how you thought about the safety of your project. Helping the judges will help you too.
Team: Cambridge
Region: Europe
iGEM Year:2012
Track:Foundational Advance
Project Name:Parts for a reliable and field ready biosensing platform
Project Abstract: Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field.
We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems.
Back to wiki
iGEM Medals for non-software teams
We believe our team deserves the following medal:
Bronze
Silver
√Gold
Because we met the following criteria (check all that apply and provide details where needed)
Requirements for a Bronze Medal
√Register the team, have a great summer, and plan to have fun at the Regional Jamboree.
√Successfully complete and submit this iGEM 2012 Judging form.
√Create and share a Description of the team's project using the iGEM wiki and the team's parts using the Registry of Standard Biological Parts.
√Plan to present a Poster and Talk at the iGEM Jamboree.
√Enter information detailing at least one new standard BioBrick Part or Device in the Registry of Standard Biological Parts. Including:
√Primary nucleaic acid sequence
√Description of function
√Authorship
Safety notes, if relevant.
√Acknowedgment of sources and references
√Submit DNA for at least one new BioBrick Part or Device to the Registry.
Additional Requirements for a Silver Medal
√Demonstrate that at least one new BioBrick Part or Device of your own design and construction works as expected; characterize the operation of your new part/device.
√Enter this information and other documentation on the part's 'Main Page' section of the Registry
Part Number(s): BBa_K911004
Additional Requirements for a Gold Medal: (one OR more)
Improve an existing BioBrick Part or Device and enter this information back on the Experience Page of the Registry.
Part Number(s): None
√Help another iGEM team by, for example, characterizing a part, debugging a construct, or modeling or simulating their system.
Link to this information on your wiki. Page name: Team:Cambridge/Outreach/Collaboration
√Outline and detail a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation.
Link to this information on your wiki.
Page name: Team:Cambridge/HumanPractices/Overview , Team:Cambridge/HumanPractices/MarketResearch , Team:Cambridge/HumanPractices/FutureDirections
Back to wiki
iGEM Prizes
All teams are eligible for special prizes at the Jamborees. more... To help the judges, please indicate if you feel you should be evaluated for any of the following special prizes:
√Best Human Practice Advance
√Best Experimental Measurement
Best Model
Please explain briefly why you should receive any of these special prizes:
Best Human Practice Advance:
We feel that we deserve this prize for three reasons:
We explored the impacts, *both positive and negative*, of synthetic biology as a solution to real world problems, through interviewing professionals working in a relevant field, namely the impact of arsenic water contamination in Bangladesh.
We recognized existing problems with the way the current direction of synthetic. On going through the registry we found that most of the characterization data for biosensing parts is often neither comparable nor replicable. We have worked to solve this issue, for example with our ratiometric dual channel output.
*Our project doesn’t stop here*, in Chanel number 6 ( Team:Cambridge/HumanPractices/FutureDirections ) we considered the future implications and technological applications of our project, as well as the means by which it could be improved by subsequent users. We feel that the end to an iGEM project should not be the conclusion of an idea, but the start of it.
Best BioBrick Measurement Approach:
It is absolutely vital that a quantitative, numerical, robust, and flexible measurement approach exists to relay information to a user that is an accurate representation of the input processed by a biological device. Working from these principles, the following was done:
We designed and built Biologger, a *cheap, arduino-based, fully functional automatic rotary device* that has an incorporated ratiolumnometer
Our project is entirely open-sourced and open-platform. We have published source code for the two applications which serve to operate the device, one for PCs and the other for Android devices, as well as the open source circuit design that provides this ratiometric reading. Furthermore, the Android app is able to receive its data wirelessly, which we feel is a great advance in BioBrick measurement.
Our dual-channel luciferase reporter was successfully tested with a dilution series of E.coli transformed with the Lux Operon (under pBAD) biobrick (Part  BBa_K325909) of the Cambridge iGEM 2010 team. It can detect, with good accuracy, both different light intensities, as well as the percentages of blue or orange frequencies in a sample.
Our device was successfully tested using artificial light to detect different frequencies (colours) as well.
Having done all the above, we believe that this fully open-sourced instrumentation kit (mechanical) chassis, electronics, software code), estimated at *$35.00* (or $85.00 if a Bluetooth modem is required), is a complete BioBrick measurement solution for any and all BioBricks with a light output.
Back to wiki
Team_Parts
To help the judges evaluate your parts, please identify 3 of your parts that you feel are best documented and are of the highest quality.
Best new BioBrick part (natural)
BBa_K911003
Best new BioBrick part (engineered)
BBa_K911004
Best improved part(s): None
List any other parts you would like the judges to examine: BBa_K911001 , BBa_K911009 , BBa_K911008
Please explain briefly why the judges should examine these other parts:
Magnesium Sensitive Riboswitch BBa_K911001
As a riboswitch sensing construct, this part is an entirely new type of biosensor (along with the fluoride construct) that could potentially change the way we think about designing input genetic circuits. Unlike the fluoride riboswitch, it is a derepression system and therefore serves to demonstrate the principle that riboswitches can be used regardless of whether they turn on or off their reporter.
Fluorescent ratiometric construct for standardizing promoter output BBa_K911009
Fluorescence is a major cornerstone for biosensors in the registry, however, most parts do not involve the use of a ratiometric output, which has been shown in the literature to provide much more reliable and meaningful data. This part not only furthers the development of ratiometric measurements in molecular biology but due to the choice of promoters and terminators it can be used to characterize the difference in activity between E. coli and B. Subtilis
Fast Germination (B. subtilis) BBa_K911008
This part is entirely novel for the registry and fully utilizes the recombination machinery inherent in the Bacillus chassis. Have spores that can germinate at a faster rate is certainly a worthy achievement and could help with experiments with B. Subtilis that any future iGEM teams may wish to perform.
Back to wiki
iGEM Safety
For iGEM 2012 teams are asked to detail how they approached any issues of biological safety associated with their projects.
The iGEM judges expect that you have answered the four safety questions Safety page on your iGEM 2012 wiki.
Please provide the link to that page: Page name: Team:Cambridge/Safety
Attribution and Contributions
For iGEM 2012 the description of each project must clearly attribute work done by the team and distinguish it from work done by others, including the host labs, advisors, and instructors.
Please provide the link to that page, or comments in the box below: Page name: Team:Cambridge/Attributions
Comments
If there is any other information about your project you would like to highlight for the judges, please provide a link to your wiki page here: Team:Cambridge/Overview/DesignProcess
Week:
1
2
3
4
5
6
7
8
9
10
11
12
13
14
The Final Month
Monday
Ratiometrica: grow O/N cultures for rough IPTG induction tomorrow, ordered primers to convert into biobrick
Tuesday
Ran PCR for making SPOBB biobrick, gel yielded insufficient results. Redesigned and reordered primers
Ratiometrica: rough IPTG induction with 8 different concentrations of IPTG, after 3 hours of incubation there was unfortunately no obvious results (i.e. no visible difference between different concentrations/induced and uninduced smaples), will leave them growing overnight and observe again tomorrow; sequencing primers arrived so the plasmid is sent off to sequencing, will have results tomorrow morning; more O/N cultures grown for miniprep to put into Bacillus tomorrow.
Wednesday
Ratiometrica: Sequencing results came back- good news: CFP is inserted correctly after pSPAC, bad news: more uncertainties as to whether the K143053 and YFP are inserted correctly. Since the cells shows clear YFP under the fluorescent microscope, we cannot really tell what is wrong at the moment, more sequencing primers are ordered to verify the sequence further along the construct. Tom ran a restriction digest with the new miniprep products and they showed exactly the expected results- so we are now more certain that we have got the right construct! IPTG induction in liquid cultures yesterday really didn't give reliable results, so we have now plated cells on IPTG plates for induction and hopefully we will see results tomorrow. Bacillus has also been transformed- it's been a while since we've done Bacillus transformation, and we will know if it works tomorrow. Primers arrived today morning and PCR has been done to convert ratiometrica into a biobrick! We are having some slight problems with the linearised backbone but are slowly working our way around it... hopefully it will all come into place.
Thursday
Ratiometrica: Yesterday's IPTG induction on plates seemed to have worked! Cells on IPTG plate showed much more CFP than those on the uninduced plates. We are growing O/N cultures in minimal medium so we can test them on a plate reader to get more accurate results and to characterise the part... We are still having trouble with the linearised backbone, plan A is that this currently running PCR cycle will work out and we will have the suitable backbone, plan B is to wait for the correct primers to hopefully come in asap! Bacillus transformation from yesterday produced 1 bacillus colony- it is hard to tell if it is the thing we are looking for as ECFP and EYFP signals are rather poor in Bacillus without the ComGA codons, but there is  definitely a little bit of CFP and YFP. We have grown O/N culture of that in LB and will probably extract the genomic DNA then PCR as a verification test. Time is running out...
Friday
Ratiometrica: we just found illegal restriction sites in our construct ): It is behind the inserted promoter- that is why we didn't notice it in the first place, anyway, it means that we will have to wait for the correct primers to arrive to convert the construct into a biobrick. Meanwhile, M9 minimal cell cultures seem to take double the time to grow so we have made more so they could grow over the weekend and we will have them ready for a plate reader assay by Monday. M9 plates are made as a plan B, so if the plate reader assay doesn't work out we could still photograph plates under the fluorescent microscope as a proof for a working construct. We pelleted the ratiometrica B. sub and it seems to be fluorescing, though as usual it is hard to tell if it is only internal fluorescence... we will attempt a colony PCR tomorrow to verify.
The biobrick fragment was successfully pcr'd out of a Ps3393 miniprep using half prefix/suffix biobrick primers in order to ligate this with the standardised backbone.
Saturday
Ratiometrica: Cloudy M9 culture! Which means cells take around 48 hours to grow in M9 minimal media. Some of that culture was plated onto M9 plates and we should hopefully be able to check for fluorescence tomorrow. Colony PCR is performed on the Bacillus culture and guess what- no bands. It is likely to be a colony PCR fail though (as they always do) as positive control is not working either. We might have to use some alternative ways to test it out...
Sporage and distribution: Succesfully extracted the standardised backbone from a lux-biobrick miniprep. This was done in such that the restriction enzymes had enough base pairs to grab onto to cut at the speI and XbaI sites (rather than using the linearised backbone). This was then used with the spoVAA fragment for a ligation reaction and transformation.
Sunday
Ratiometrica: No colonies on M9 minimal medium plates after 24 hours- these cells really don't like the lack of nutrients. We will give them another day. Same goes to the cultures which have been in the 37°C incubator, they only reached an OD600 of 0.004 after 48 hours. Luckily we have some cultures from Thursday afternoon which are nice and cloudy, so we will be using that (diluted 5 times) for tomorrow morning's plate reader assay. Checked the suspected B. subtilis under the fluorescent microscope again... they don't look very fluorescent unfortunately. Maybe we haven't got it afterall. We will try transforming some again...
Sporage and distribution: Realised that the spoVAA fragment had not been restricted before ligation so this experiment was doomed to fail. However plenty of colonies were found that (after checking with a restriction digest) were proven to just be the PSB1C3 backbone. This took place due to the natural elimination when XbaI and speI sites are run through a restriction/ligation protocol.
Retrieved from " http://2012.igem.org/Team:Cambridge/Diary/Week_13 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
