<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Project_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge/Project/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Project/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Project/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Project/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Project/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Project/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="cam_page"><DIV id="cam_left"><DIV id="cam_grid_project" class="cam_grid"><DIV id="cam_grid_project_headover" class="cam_grid_headover"><UL class="subtitles"><LI><A href="https://2012.igem.org/Team:Cambridge/Project" class="cam-external" style="color: #FFFFFF">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Cambridge/Parts" class="cam-external" style="color: #FFFFFF">Parts </A></LI><LI><A href="https://2012.igem.org/Team:Cambridge/Project/DesignProcess" class="cam-external" style="color: #FFFFFF">Design Process</A></LI><LI><A href="https://2012.igem.org/Team:Cambridge/Project/Results" class="cam-external" style="color: #FFFFFF">Results</A></LI></UL><H7>Previous iGEM teams have charaterised an impressive array of inducible promoters, along with other elements of biosensing circuitry... Read More </H7></DIV></DIV><DIV id="cam_nav"><DIV id="cam_nav_row1" class="cam_nav_row"><A href="https://2012.igem.org/Team:Cambridge/Project"><DIV id="cam_nav_project" class="cam_nav_grid"><DIV id="cam_nav_project_headover" class="cam_nav_headover"><H6> Project </H6></DIV></DIV></A><A href="https://2012.igem.org/Team:Cambridge/Outreach/Overview"><DIV id="cam_nav_outreach" class="cam_nav_grid"><DIV id="cam_nav_outreach_headover" class="cam_nav_headover"><H6> Outreach </H6></DIV></DIV></A></DIV><A href="https://2012.igem.org/Team:Cambridge/Gallery/DayAtLab"><DIV id="cam_nav_row2" class="cam_nav_row"><DIV id="cam_nav_gallery" class="cam_nav_grid"><DIV id="cam_nav_gallery_headover" class="cam_nav_headover"><H6> Gallery </H6></DIV></DIV></DIV></A><A href="https://2012.igem.org/Team:Cambridge/Diary/Week_1"><DIV id="cam_nav_diary" class="cam_nav_grid"><DIV id="cam_nav_diary_headover" class="cam_nav_headover"><H6> Diary </H6></DIV></DIV></A></DIV><DIV id="cam_nav_row3" class="cam_nav_row"><A href="https://2012.igem.org/Team:Cambridge/Team"><DIV id="cam_nav_team" class="cam_nav_grid"><DIV id="cam_nav_team_headover" class="cam_nav_headover"><H6> Team </H6></DIV></DIV></A><A href="https://2012.igem.org/Team:Cambridge"><DIV id="cam_nav_home" class="cam_nav_grid"><DIV id="cam_nav_home_headover" class="cam_nav_headover"><H6> Home </H6></DIV></DIV></A><A href="https://2012.igem.org/Team:Cambridge/Safety"><DIV id="cam_nav_safety" class="cam_nav_grid"><DIV id="cam_nav_safety_headover" class="cam_nav_headover"><H6> Safety </H6></DIV></DIV></A></DIV><DIV id="cam_nav_row4" class="cam_nav_row"><A href="https://2012.igem.org/Team:Cambridge/Sponsors"><DIV id="cam_nav_sponsors" class="cam_nav_grid"><DIV id="cam_nav_sponsors_headover" class="cam_nav_headover"><H6> Sponsors </H6></DIV></DIV></A></DIV></DIV></DIV><DIV id="cam_right" class="cam-box1"><DIV id="cam_pagecontent" class="cam-box-content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Results"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#RiboSense"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">RiboSense</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Fluoride_Riboswitch"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Fluoride Riboswitch</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Magnesium_Riboswitch"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Magnesium Riboswitch</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-5"><A href="#Ratiometrica"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Ratiometrica</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Instrumentation_.28Biologger.29"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Instrumentation (Biologger)</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Sporage_and_Distribution"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Sporage and Distribution</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>Set out below are the developments the team has made over the summer, in tackling our aim and objectives.
</P><H3><SPAN class="mw-headline" id="RiboSense">RiboSense</SPAN></H3><H3><SPAN class="mw-headline" id="Fluoride_Riboswitch">Fluoride Riboswitch</SPAN></H3><P>Having obtained our DNA, the first test was a rather crude β-Galactosidase assay using cells cultured from the plates Yale sent us, the results of which are shown below, the protocol for which is found <A href="/Team:Cambridge/Protocols/beta-galactosidaseassay" title="Team:Cambridge/Protocols/beta-galactosidaseassay"><SPAN style="color:#000066"><U>here</U></SPAN></A>.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">A β-galactosidase assay of the fluoride riboswitch, the numbers represent fluoride concentration in mM</DIV></DIV></DIV></DIV><P>These results were very positive as it hinted at a good correlation between fluoride concentration and reporter output.
</P><P>
The next step was a more quantitave assay using a 96 well plate reader. This was in line with the assays done in the original paper by Breaker et al. In the paper they carried out a Miller assay, and whilst we have included the full and correct protocol on our <A href="/Team:Cambridge/Protocols/MillerAssay" title="Team:Cambridge/Protocols/MillerAssay"><SPAN style="color:#000066"><U>protocols page</U></SPAN></A>, including the appropriate calculations, we lacked the filters and so were only able to record the A450nm. We feel that this was close enough in wavelength to be a suitable proxy for the A420nm reading required to assay the o-nitrophenol produced. We ran an initial assay and indicative results are below. All the A450 bar charts are based on two repeats, error bars are not included as they would not be statistically significant.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">The timecourse of the first run of the β-galactosidase assay using crcB knockout <I>B. subtilis</I> transformed with the fluoride riboswitch</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">A450nm readings after 420 minutes with crcB knockout <I>B. subtilis</I> transformed with the fluoride riboswitch</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">A450nm readings after 420 minutes with <I>B. subtilis</I> strain 168 transformed with the fluoride riboswitch</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">A450nm readings after 420 minutes with <I>E. coli</I> carrying a plasmid containing the fluoride riboswitch</DIV></DIV></DIV></DIV><P>This data seems to show a trend, that at low fluoride concentrations there is a positive relationship between concentration and A450, the A450 then plateaus, and then at high fluoride concentration the A450 drops again, this gives the overall data a bell shape. A possible explanation for this is that there are two conflicting effects, an increase in fluoride leads to an increase in β-galactosidase expression which, in turn, leads to an increased A450. However, the increased fluoride concentrations begin to become toxic and so limit the cell’s ability to produce β-galactosidase, leading to a reduction in A450. It would also seem that the linear region of the initial positive relationship seems to lie between 0μM and 100μM. We therefore repeated the assay, with a longer time course, and with greater resolution in this range, in order to see if the original trend could be reproduced, and also to better quantify this initial correlation. Included below are the results for the repeated assay. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">The timecourse of the second run of the β-galactosidase assay using crcB knockout <I>B. subtilis</I> transformed with the fluoride riboswitch</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">A450nm readings after 790 minutes with crcB knockout <I>B. subtilis</I> transformed with the fluoride riboswitch</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">A450nm readings after 790 minutes with <I>B. subtilis</I> strain 168 transformed with the fluoride riboswitch</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">A450nm readings after 790 minutes with <I>E. coli</I> carrying a plasmid containing the fluoride riboswitch</DIV></DIV></DIV></DIV><P>For clarity, and to act as an internal control, the bar chart results are expressed as a percentage change from the same culture, after the same length of time, but with no fluoride added. This assumes that the cultures behaved the same across all fluoride concentrations over the assay, i.e. cell density and productivity were unaffected by the fluoride. This sort of assumption would no longer be needed if a ratiometric system were used, part of our motivation for developing ratiometrica. This data seems to confirm the initial findings, as well as giving greater resolution in the 0μM to 100μM range. The results for higher fluoride concentrations do not seem to quite fit the trend, but as this is consistent across all the cell types it is possible that this was due to errors in making up the solutions, rather than actually being representative of the riboswitch. This data is very encouraging as the WHO safe limit for fluoride is 1.5mg/L (according to <A href="http://dx.doi.org/10.1016/j.envpol.2006.05.007" class="external text" rel="nofollow"><SPAN style="color:#000066"><U>Farooqi et al</U></SPAN></A>) which works out at 79μM. This means our riboswitch is at its most accurate around the safe level for humans. For more information on the implications of this, have a look at our <A href="/Team:Cambridge/Outreach/HumanPractices" title="Team:Cambridge/Outreach/HumanPractices"><SPAN style="color:#000066"><U>human practices page</U></SPAN></A>.
</P><P>It should be noted that we expected a greater difference in sensitivity between the knockout <I>Bacillus</I> strain provided by Yale, the 168 strain we were using, and the <I>E. coli</I> we used. In the original paper there was a significant, nearly 100-fold, difference in sensitivity between the knock and wild type bacillus strains, but our data did not reproduce this. However, we feel it is actually very positive as it means our part is consistent across multiple chasses, and that it has the desired sensitivity to fluoride, without the need for a special mutant to be ordered. This fits with our goal for our kit to be a standard which is widely compatible.
</P><P>The next step was to place this construct in a ratiometric fluorescent construct, before eventually testing our luciferase construct with it. But, due to delays in developing the ratiometric construct, we only had one attempt at inserting and testing it and it proved unsuccessful. We shall carry on testing it and, whilst we cannot put our results on here due to the wiki freeze, we hope to present them to you at the jamborees.  However, we would like to include here what we plan to do, in order that future teams might benefit. We plan to insert the fluoride riboswitch into the ratiometrica construct and check that they work together as expected, i.e. get consistent readings for various concentrations, in spite of varying cell densities.
</P><H4><SPAN class="mw-headline" id="Magnesium_Riboswitch">Magnesium Riboswitch</SPAN></H4><P>For the magnesium riboswitch, we successfully made the following construct:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">The construct made in the pJS130 vector for the detection of changes in Mg2+ ion concentrations</DIV></DIV></DIV></DIV><P>The sequence of this construct was verified via Sanger sequencing.
</P><P>To characterize this construct, we used a 96-well plate reader to assay the effects of the different concentrations of Mg2+ and IPTG on the levels of GFP. We expected the presence of either to allow the expression of sfGFP, however because transcriptional attenuation by the riboswitch occurs before expression of the repressor protein, it was expected that Mg2+ would somehow demonstrate a dominant effect.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">Normalized final fluorescence readings from our construct. Readings were adjusted against final OD620. The construct did not work as expected.</DIV></DIV></DIV></DIV><P>As can be seen, our initial assay did not give the expected results. At differing IPTG concentrations, the response to magnesium seems to have inverted.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Typical growth curves of our construct. Magnesium concentrations varied from 1 μM (most red) to 5mM (most green).</DIV></DIV></DIV><P>We repeated this experiment, using a different range of magnesium concentrations (1μM - 5mM) and doing duplicates in alternate rows. The cells grew well, although they did not appear to do so in an exponential fashion. It is believed that this may be a feature of the minimal medium in which we were performing the assay, as failed tests using a rich defined medium (which, unfortunately, autofluoresced at GFP wavelengths due to the aromatic amino acids it contained) showed normal exponential growth with the same construct and at identical magnesium concentrations.
</P><P>Having collected OD620 data as well as fluorescence data, we hoped to generate graphs containing fluorescence readings normalized to cell density. We used the following formula to normalize our data:
</P><P>Initial analysis produced fairly convincing data, as shown in the graphs below (top images). Certainly, the part appeared to produce a convincing response to magnesium between 1 μM and ~10μM, within the sensitivity range of the original paper. A more modest increase can be seen past this point. More careful analysis of the raw data indicated that this apparent trend may have been an artifact of the normalization formula, as no particularly convincing trend can be seen in the raw final fluorescence data (bottom images).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Surface plots of our plate reader assay data. Top images have been normalized to cell population, while bottom images have not. Images on the right leave out the outlier at 1μM.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Line graphs of our plate reader assay data. Top images have been normalized to cell population, while bottom images have not. Images on the right leave out the outlier at 1μM.</DIV></DIV></DIV></DIV><P>However, visual inspection under the fluorescence microscope demonstrated that the low magnesium cultures had very different fluorescent properties, both in quantity and quality of the light produced (higher magnesium cultures had a more yellow hue). We took this as a sign that our plate reader may not be producing reliable data, perhaps not surprising given that we were having to use non-optimal emission and excitation filters. Additionally, the final fluorescence was lower than the initial fluorescence despite the fact that visual inspection demonstrated that it increased considerably from start to finish. This just goes to show the dangers of normalizing data without looking at it first! It also shows the importance of an <A href="/Team:Cambridge/Project/Standardised_Outputs" title="Team:Cambridge/Project/Standardised Outputs">internal ratiometric control channel</A> which is measured in the same way - this should eliminate the normalization artifacts that we have found from using OD620 as our normalization channel. If the plate reader was unable to measure one fluorescence channel, it is unlikely that it would be able to measure the second, eliminating the artifacts.
</P>
We then attempted to transform the construct into Bacillus subtilis, with a view to repeat the same assay in this new chassis. Because the part originally comes from bacillus, we hoped that this might give more useful data. Unfortunately, we did not have time to characterise the part in this chassis, as our transformation attempts failed. This may be an excellent starting point for any future teams seeking to explore this part.
<H3><SPAN class="mw-headline" id="Ratiometrica">Ratiometrica</SPAN></H3><P>After a lot of technical difficulties, we were able to assemble our fluorescent construct using Gibson assembly. The photo below shows the predicted and obtained digest pattern using HINDIII.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">The predicted digest, and the actual digest next to hyperladder I. The ladder is the same at the relevant molecular weights.</DIV></DIV></DIV></DIV><P>We ran a plate reader assay with ratiometrica with no induction of the reporter CFP. The results of this assay can be seen. In summary:
</P><UL><LI>No change in CFP is observed with changing OD600.
</LI><LI>YFP change is approximately proportional to OD600.
</LI><LI>The ratio of YFP to OD600 approaches a constant value at higher cell densities.
</LI></UL><P>These properties are exactly what we expected them to be. This assay also tells us that we should aim to take our ratiometric measurements only during late exponential phase, as this is where the data is most reliable.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Growth curves of 12 replicates of our ratiometrica containing e.coli. This is normal growth in e.coli, indicating that our part is not toxic.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Comparison of OD600 against YFP expression. We are aiming to use YFP as our representation of cell activity, so there should be a strong relationship between these two values. This has been observed, although only during the exponential phase of growth.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Comparison of OD600 againt CFP expression. We did not induce CFP with IPTG, so we do not expect there to be a strong relationship between OD and CFP expression. This appears to be the case, with almost no change in CFP levels with OD600.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">YFP levels divided by OD600. Because YFP should be a measure of cell density along with OD600, we expected that this ratio should approach a constant value. This appears to be the case, although only at higher cell densities.</DIV></DIV></DIV></DIV><P>A difference in CFP has also been observed between uninduced and induced (1mM IPTG) plates under the fluorescence microscope. Unfortunately, we were unable to test this induction (and the consequences of ratiometric measurement on the reliability of resulting data) with the plate reader before the deadline (possibly due to difficulties with culturing the cells in M9 minimal medium). We will reattempt this assay before the jamboree and are confident that results should be as expected.
</P><P>Transformation of B. subtilis with the plasmid has also been attempted- unfortunately we are still finetuning our B. subtilis transformation protocol. Again, we hope to show that this works before the Jamboree.
</P><P>The construct is also sequenced from the beginning and middle (as it is a long part) by Source Bioscience, below is the alignment data from the sequencing (using ClustalW2):
</P><PRE>
CLUSTAL 2.1 multiple sequence alignment
                                                                                 

seq                            ---------------------------NNNNNNNNNNNAANNNNNNNA-- 21
pJS150_seq_-_Ratiometrica      AGCCCAGTCCAGACTATTCGGCACTGAAATTATGGGTGAAGTGGTCAAGA 1050
                                                            .. ......**.....  *  

seq                            ---------------------------------------NNNNNNNTNTN 32
pJS150_seq_-_Ratiometrica      CCTCACTAGGCACCTTAAAAATAGCGCACCCTGAAGAAGATTTATTTGAG 1100
                                                                       ... ..*.:.

seq                            GTN--CNNT----------------------------------------- 39
pJS150_seq_-_Ratiometrica      GTAGCCCTTGCCTACCTAGCTTCCAAGAAAGATATCCTAACAGCACAAGA 1150
                               **   * .*                                         

seq                            -----------TTNNNNTCTN-------------CTAANTNTNAGNGCTC 65
pJS150_seq_-_Ratiometrica      GCGGAAAGATGTTTTGTTCTACATCCAGAACAACCTAATTGTGAGCGCTC 1200
                                          **....***              ****.*.*.** ****

seq                            NNAATTTTTTGNNNAATTNNTNANNNTTTATCTACN-GGTGTGTCAT-AT 113
pJS150_seq_-_Ratiometrica      ACAATTTTTTGCAAAAAGTTGTTGACTTTATCTACAAGGTGTGGCATAAT 1250
                                 *********   **: .. .:.  *********  ****** *** **

seq                            GTNTGGAANNNCNANNAGCTCACAATTAANGGATGAATTCN-AATGGTGA 162
pJS150_seq_-_Ratiometrica      GTGTGGAATTGTGAGCGGCTCACAATTAAAGGAGGAATTCAAAATGGTGA 1300
                               **.*****... .*. .************ *** ******  ********

seq                            GCAAGGGCGANGAGCTGTTCACCGGGGTGGNGCCCATCCTGGTCGAGCTG 212
pJS150_seq_-_Ratiometrica      GCAAGGGCGAGGAGCTGTTCACCGGGGTGGTGCCCATCCTGGTCGAGCTG 1350
                               **********.*******************.*******************

seq                            GACGGCGACGNGANCGGCNACAAGTTCATCGTGTCCTNCGAGGGCGAGGG 262
pJS150_seq_-_Ratiometrica      GACGGCGACGTGAACGGCCACAAGTTCAGCGTGTCCGGCGAGGGCGAGGG 1400
                               **********.** **** ********* ******* .************

seq                            CGATGCCACCTACGGCAAGCTGACCTTGAAGTTCATCTG-ACCACCGGCA 311
pJS150_seq_-_Ratiometrica      CGATGCCACCTACGGCAAGCTGACCCTGAAGTTCATCTGCACCACCGGCA 1450
                               ************************* ************* **********

seq                            AGCTGCCCGTGCCCTGGCCCACCNTCGTGACCACCCTGACTTGGGGCGTG 361
pJS150_seq_-_Ratiometrica      AGCTGCCCGTGCCCTGGCCCACCCTCGTGACCACCCTGACCTGGGGCGTG 1500
                               *********************** **************** *********

seq                            CANTGCTTCTNCCGCTACCCCGACCACATGAAGCANCANGACTTCTTCAN 411
pJS150_seq_-_Ratiometrica      CAGTGCTTCAGCCGCTACCCCGACCACATGAAGCAGCACGACTTCTTCAA 1550
                               **.******:.************************.** ********** 

seq                            TTCCGCCATGCCCGAAGGCTACTTCNAGNAGCTCACCNTCTTCTTNAAGG 461
pJS150_seq_-_Ratiometrica      GTCCGCCATGCCCGAAGGCTACGTCCAGGAGCGCACCATCTTCTTCAAGG 1600
                                ********************* ** **.*** **** ******* ****

seq                            ACNACGGCAACTACAT--------------TNAN---------------- 481
pJS150_seq_-_Ratiometrica      ACGACGGCAACTACAAGACCCGCGCCGAGGTGAAGTTCGAGGGCGACACC 1650
                               **.************:              *.*                 

seq                            --------------------------------------------------
pJS150_seq_-_Ratiometrica      CTGGTGAACCGCATCGAGCTGAAGGGCATCGACTTCAAGGAGGACGGCAA 1700
                                                                                 

</PRE><PRE>CLUSTAL 2.1 multiple sequence alignment
                                                                                 

Rat3                           -----------------------------NNNNNNNNNNNNNNNNNNTTN 21
pJS150_seq_-_Ratiometrica      AGAGTCACACTGGCTCACCTTCGGGTGGGCCTTTCTGCGTTTATAAATTT 2150
                                                              ... .. .... .   **.

Rat3                           TGNCNNN-TAATTTTATTGACAACGTCTTATTAACGTTGATATAATTTAA 70
pJS150_seq_-_Ratiometrica      TGTCAAAATAATTTTATTGACAACGTCTTATTAACGTTGATATAATTTAA 2200
                               **.*    ******************************************

Rat3                           ATTTTATTTGN-NAAAATGGGCTCGTGTTGTACAATAAATGTTACTAGAG 119
pJS150_seq_-_Ratiometrica      ATTTTATTTGACAAAAATGGGCTCGTGTTGTACAATAAATGTTACTAGAG 2250
                               **********   *************************************

Rat3                           AAAGGTGGTGNATACTAGATGGTGAGCAAGGGCGAGGAGCTGTTCACCGG 169
pJS150_seq_-_Ratiometrica      AAAGGTGGTGAATACTAGATGGTGAGCAAGGGCGAGGAGCTGTTCACCGG 2300
                               ********** ***************************************

Rat3                           GGTGGTGCCCATCCTGGTCGAGCTGGACGGCGACGTAAACGGCCACAAGT 219
pJS150_seq_-_Ratiometrica      GGTGGTGCCCATCCTGGTCGAGCTGGACGGCGACGTAAACGGCCACAAGT 2350
                               **************************************************

Rat3                           TCAGCGTGTCCGGCGAGGGCGAGGGCGATGCCACCTACGGCAAGCTGACC 269
pJS150_seq_-_Ratiometrica      TCAGCGTGTCCGGCGAGGGCGAGGGCGATGCCACCTACGGCAAGCTGACC 2400
                               **************************************************

Rat3                           CTGAAGTTCATCTGCACCACCGGCAAGCTGCCCGTGCCCTGGCCCACCCT 319
pJS150_seq_-_Ratiometrica      CTGAAGTTCATCTGCACCACCGGCAAGCTGCCCGTGCCCTGGCCCACCCT 2450
                               **************************************************

Rat3                           CGTGACCACCTTCGGCTACGGCCTGCAATGCTTCGCCCGCTACCCCGACC 369
pJS150_seq_-_Ratiometrica      CGTGACCACCTTCGGCTACGGCCTGCAATGCTTCGCCCGCTACCCCGACC 2500
                               **************************************************

Rat3                           ACATGAAGCTGCACGACTTCTTCAAGTCCGCCATGCCCGAAGGCTACGTC 419
pJS150_seq_-_Ratiometrica      ACATGAAGCTGCACGACTTCTTCAAGTCCGCCATGCCCGAAGGCTACGTC 2550
                               **************************************************

Rat3                           CAGGAGCGCACCATCTTCTTCAAGGACGACGGCAACTACAAGACCCGCGC 469
pJS150_seq_-_Ratiometrica      CAGGAGCGCACCATCTTCTTCAAGGACGACGGCAACTACAAGACCCGCGC 2600
                               **************************************************

Rat3                           CGAGGTGAAGTTCGAGGGCGACACCCTGGTGAACCGCATCGAGCTGAAGG 519
pJS150_seq_-_Ratiometrica      CGAGGTGAAGTTCGAGGGCGACACCCTGGTGAACCGCATCGAGCTGAAGG 2650
                               **************************************************

Rat3                           GCATCGACTTCAAGGAGGACGGCAACATCCTGGGGCACAAGCTGGAGTAC 569
pJS150_seq_-_Ratiometrica      GCATCGACTTCAAGGAGGACGGCAACATCCTGGGGCACAAGCTGGAGTAC 2700
                               **************************************************

Rat3                           AACTACAACAGCCACAACGTCTATATCATGGCCGACAAGCAGAAGAACGG 619
pJS150_seq_-_Ratiometrica      AACTACAACAGCCACAACGTCTATATCATGGCCGACAAGCAGAAGAACGG 2750
                               **************************************************

Rat3                           CATCAAGGTGAACTTCAAGATCCGCCACAACATCGAGGACGGCAGCGTGC 669
pJS150_seq_-_Ratiometrica      CATCAAGGTGAACTTCAAGATCCGCCACAACATCGAGGACGGCAGCGTGC 2800
                               **************************************************

Rat3                           AGCTCGCCGACCACTACCAGCAGAACACCCCCATCGGCGACGGCCCCGTG 719
pJS150_seq_-_Ratiometrica      AGCTCGCCGACCACTACCAGCAGAACACCCCCATCGGCGACGGCCCCGTG 2850
                               **************************************************

Rat3                           CTGCTGCCCGACAACCACTACCTGAGCTACCAGTCCGCCCTGAGCAAAGA 769
pJS150_seq_-_Ratiometrica      CTGCTGCCCGACAACCACTACCTGAGCTACCAGTCCGCCCTGAGCAAAGA 2900
                               **************************************************

Rat3                           CCCCNAACGAGAAGCGCGATCACATGGTCCTGCTGGAGTTCGTGACCGCC 819
pJS150_seq_-_Ratiometrica      CCCC-AACGAGAAGCGCGATCACATGGTCCTGCTGGAGTTCGTGACCGCC 2949
                               **** *********************************************

Rat3                           GCCGGGATCACTCTCGGCATGGACGAGCTGTACAAGTAATAATGATACTA 869
pJS150_seq_-_Ratiometrica      GCCGGGATCACTCTCGGCATGGACGAGCTGTACAAGTAATAATGATACTA 2999
                               **************************************************

Rat3                           GAGCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTT 919
pJS150_seq_-_Ratiometrica      GAGCCAGGCATCAAATAAAACGAAAGGCTCAGTCGAAAGACTGGGCCTTT 3049
                               **************************************************

Rat3                           CGTTTTATCTGTTGTTTGTCGGTGAACGCTCTCTACTAGTCATCATTTCC 969
pJS150_seq_-_Ratiometrica      CGTTTTATCTGTTGTTTGTCGGTGAACGCTCTCTACTAGTCATCATTTCC 3099
                               **************************************************

Rat3                           TTCCGAAAAAACGGTTGCATTTAAATCTTACATATGTAATACTTTTCAAA 1019
pJS150_seq_-_Ratiometrica      TTCCGAAAAAACGGTTGCATTTAAATCTTACATATGTAATACTTT-CAAA 3148
                               ********************************************* ****

Rat3                           GACTACATTTTGTAAGATTTGATGTTTGAANNCGGGCTGAAANATCGGTA 1069
pJS150_seq_-_Ratiometrica      GACTACATTT-GTAAGATTTGATGTTTGAG-TCGG-CTGAAAGATCG-TA 3194
                               ********** ******************. .*** ******.**** **

Rat3                           CGTACCCNNNNTTGTTTCNNNATNNNNCAG-CCNATNNNCTNNNNGNATA 1118
pJS150_seq_-_Ratiometrica      CGTACCAATTATTGTTTCGTGATTGTTCAAGCCATAACACTGTAGGGATA 3244
                               ******. .. *******...**....**. ** ::   **.. .*.***

Rat3                           NNNNNNNAAGAGNNCTTCNNCNGGNNACNANTCANNNAANNANTNAANCN 1168
pJS150_seq_-_Ratiometrica      GTGG--AAAGAGTGCTTCATCTGGTTACGA-TCAATCAAATATTCAAACG 3291
                               ....   *****..**** .*.**..**.* *** . ** .*.* ** *.

Rat3                           GNNGGNNNACNNNNTNNNN---ANNNNNNANNTTNNNCGAANNN------ 1209
pJS150_seq_-_Ratiometrica      GAGGGAG-ACGATTTTGATGAAACCAGTAACGTTATACGATGTCGCAGAG 3340
                               * .** . **. ..*.. .   *   .. * .** . ***:..       

Rat3                           ----CCNNNNNNNNTNNNNNNNNGNNNNNNNNNANNN------------- 1242
pJS150_seq_-_Ratiometrica      TATGCCGGTGTCTCTTATCAGACCGTTTCCCGCGTGGTGAACCAGGCCAG 3390
                                   **..... . *. .  .   ....   . ....             
                                      
</PRE><P>We are drafting an RFC with the aim to use this construct as a new standard to characterise promoters, both constitutive and inducible, in E. coli and B. subtilis.
</P><P>Our ratiometric luciferase construct arrived with full sequence coverage. Unfortunately, during construction it was found to be unexpectedly toxic. This hampered characterisation, as it the construct tended to be lost, and necessitated its submission to the registry in a low copy number backbone (with permission from HQ). 
</P><P>Starting from the ground up, we first showed that the part was constitutively luminescing. 
We also observed that the colonies were orange. This was not initially expected, however this is probably because we designed the construct with consensus RBSes, so any leaky transcription through the inducible promoter would result in enough protein to be visible. We showed that orange colour and luminescence cosegregated, implying that the entire construct is being lost, not just part of it.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Plate with colonies picked from an initial transformation of the construct into XL1-blue cells. Some colonies are white, some are orange.</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">The same plate photographed in a dark room with a 30s exposure, showing cosegregation of luminescence with orange colour.</DIV></DIV></DIV></DIV><P>
Given that IPTG induction seemed unnecessary for production of mOrange, we reasoned that any spectral shift in the emission of the luciferase should be visible without induction. Our next move was to see if we could detect a difference with the <A href="https://2012.igem.org/Team:Cambridge/Project/DesignProcess#Instrument" style="color:#000066">filters</A> we used for the instrumentation.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">30 second exposure, K325909 on the left, our construct on the right.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:452px;"><DIV class="thumbcaption">As above, but with the filters shifted so that the previously filtered areas are visible for comparison.</DIV></DIV></DIV></DIV><P>This was done at quite a late stage and in a slightly impromptu fashion. The photographs above directly compare the normal lux biobrick (BBa_K325909) on the left, and our construct on the right, with the same filter. There does seem to be a difference in the quantity of light coming through the filter. However it could be due to colony density (although they seem to be of similar intensities without the filter). Interesting though these results are, they are not quantitative or well-controlled enough to constitute confirmation that the luxA-mOr fusion is behaving as expected, rather they are an indication that it may be.
</P>
To categorically confirm that the spectrum is as we expected we would want to fully characterise the emission spectrum with a scanning luminometer.  We do not currently have access to a scanning luminometer in time to submit the data here before the wiki freeze, however we hope to obtain this data in the next week, and to be able to present it at the jamboree.
<H3><SPAN class="mw-headline" id="Instrumentation_.28Biologger.29">Instrumentation (Biologger)</SPAN></H3><P>Having our instrumentation completed, as can be seen in our <A href="/Team:Cambridge/Project/Instrumentation" title="Team:Cambridge/Project/Instrumentation"><SPAN style="color:#000066"><U>Instrumentation (Biologger) page,</U></SPAN></A> the sensitivity of the sensor placed in the right position was tested using a dilution series of luciferase-producing E.coli. 20ml Cultures were grown overnight from single colonies. The cultures were induced with 40ul of 1.5M arabinose (for a final concentration of 3mM).
Cultures were left for 2 1/2 hours for full induction. Subsequently, a culture was pelleted and resuspended in 4ml LB. Doubling dilutions, of volume 2ml, were made from this concentrate, down to 1/8th concentration.
1ml of each 2ml dilution was analysed in each cuvette, which was placed in the cuvette holder we made ourselves. The result was very good. An almost linear relationship was obtained when data were normalised with the sensor value taken in the dark room (the latter set at zero) without using the cuvette holder (1-(sensor value/sensor value in absolute dark)), presenting the sensitivity of the sensor to different intensities of light. This behaviour was expected due to the changing offset affecting the luciferase spectrum curve at different light intensities. The offset, using our data, was calculated to be about 0.2V for each dilution. A second graph is shown which takes into account this offset (and removes it), thus showing the presence of blue frequencies. The result was as expected, as the presence of blue frequencies throughout the dilution series is not only detected, but also found to be approximately constant. The raw data of this investigation can be found in the <A href="/Team:Cambridge/Lab_book/Week_14" title="Team:Cambridge/Lab book/Week 14"><SPAN style="color:#000066"><U>Lab book.</U></SPAN></A></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Normalised sensor data using a dilution series of bioluminescent E.coli- Concentrations on the x-axis are relative therefore an OD 600 value was also taken</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Normalised percentage of blue frequencies using the same dilution series of E.coli</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Dilution series of E.coli starting from the most concentrated on the left (number 1) to the least concentrated on the right (number 4). Cuvette number 5 is our control, as it contains only LB</DIV></DIV></DIV></DIV><P>Once the sensor was tested for sensitivity, we tested that our circuitry correctly identified different frequencies (colours) of light. As can be seen below, measurements taken from orange and blue light yield values respectively above and below those from white light (our reference point). The data was taken using constant intensity of light for each case (V.High and V.Low brightness, as specified in the application). This was done with the aid of an Android phone and a specialised software application, called Color Flashlight, downloaded from the official Market. 
</P><P>As expected from the potential-divider design of our circuitry, orange and red frequencies caused the resistance of the LDR with the orange filter to decrease, leading to a higher voltage across the LDR with the blue filter. The opposite effect was observed with blue light. The reason that the white reference point is a bit lower than 2.5V (the expected value for a non-biased circuitry with a 5V source), is because we use resistors of total net resistance 1.67 kΩ before the blue LDR. This was done to bias the circuitry towards blue (i.e. decreasing the starting value, thus the sensor identifies always a bit more blue - this can be shown in previous graphs as well) and thus cause orange light to have a larger impact when present. This was used in an attempt to compensate for the fact that the peak at 560nm (Orange) in MOrange/luciferase fusion spectrum is lower than the one at 490 nm (Blue). Even though we did not manage to test the latter with transformed bacteria, the data collected in all the previous experiments makes us confident that the instrumentation is at least adequately functional.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Sensor data for different colours at different intensities</DIV></DIV></DIV></DIV><P>As the major part of the instrumentation, the bio-electronic interface, had been made and tested, now we turned to testing the other parts of our deveoped kit. This included the mechanical chassis of the prototype, the electronic/mechatronic (sensory and motory) components, and of course the software. The overview of the finished hardware/software can be seen in our <A href="/Team:Cambridge/Project/Instrumentation" title="Team:Cambridge/Project/Instrumentation"><SPAN style="color:#000066"><U>Intstrumentation (Biologger) page</U></SPAN></A>. Below, the videos showing our instrumentation in action can be seen.
</P><H3><SPAN class="mw-headline" id="Sporage_and_Distribution">Sporage and Distribution</SPAN></H3><P>Please find detailed explanations of this part of the project on our <A href="/Team:Cambridge/Project/Sporulation_and_Germination" title="Team:Cambridge/Project/Sporulation and Germination"><SPAN style="color:#000066"><U>Sporulation and Germination main page</U></SPAN></A> and on our <A href="/Team:Cambridge/Project/DesignProcess#Sporag" title="Team:Cambridge/Project/DesignProcess"><SPAN style="color:#000066"><U>Design Process page</U></SPAN></A> 
The graph below shows the percentage of spores (strain PS3411) counted in a sample of culture that were pipetted onto germination medium containing agarose pads. The graph shows the spores germinating over time.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Percentage of spores in samples counted at different times after addition to germination medium agarose pads</DIV></DIV></DIV></DIV><P>The images below show a time lapse taken using DIC microscopy from which the time point values were calculated. Using excel's exponential line fitting, the half life of these spores is calculated to be 5000 seconds (83 minutes). This is much slower than was hoped. However, we were unable to replicate the conditions used by Prof. Setlow in the papers linked on the <A href="/wiki/index.php?title=Team:Cambridge/Team/Attributions&amp;action=edit&amp;redlink=1" class="new" title="Team:Cambridge/Team/Attributions (page does not exist)"><SPAN style="color:#000066"><U>attributions page</U></SPAN></A> due to limitations of equipment in our lab (e.g. no access to a mounted stage or difco sporulation medium).
</P><P>Furthermore, we were unsuccessful (three times!) in making spores from the wild type Bacillus 168 strain and are unable to explain this. Therefore we haven't provided comparative data to prove that there is an increase in germination rate when the fast promoter is swapped in. 
</P><P>In the following microscopy images and animation, spores show up as phase-bright spots whereas vegetative cells are dark.
</P><DIV id="blurb" style="float: left;"> Over time there is seen a reduction in the number of light spots indicating the spores are germinating. The two images below show the first and last image of the time lapse, side by side for comparison.</DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">First image taken 13 minutes after initial addition of culture to germination pad</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Last image taken 3 hours 2 minutes and 40 seconds after initial addition of culture to germination pad</DIV></DIV></DIV><P>There is a clear reduction in number of phase-bright spots, both the number decreases over time (see graph, above) and the intensity of remaining spots decreases. This is indicative of spores germinating.
</P><P><I>Published data</I></P><P>The following shows figures from [<A href="https://2012.igem.org/Team:Cambridge/Project/Sporulation_and_Germination#References" class="external text" rel="nofollow"><U>Wang et al. 2011</U></A>], supporting the spoVA overexpression.
</P><P>
This experiment shows how overexpressed spoVA (strain PS3411) increases the rate of germination. The group isolated individual spores and followed their progress as they germinated using DIC microscopy (high Normalized intensity represent fully sporulated, low Normalized intensity represent fully germinated). 
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Wild Type B.subtilis, germinated on L-alanine at 25°C</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">strain PS3411 (overexpressed spoVA) B.subtilis, germinated on L-alanine at 25°C</DIV></DIV></DIV><P>On average spores germinated faster at 45°C for both wild type (WT) and PS3411. Even at 25°C PS4311 still appears to germinate quicker than WT. 
</P><P>Spores were heat shocked at 70°C for 30 minutes prior to measurement.
</P><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Wild Type B.subtilis, germinated on L-alanine at 45°C</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">strain PS3411 (overexpressed spoVA) B.subtilis, germinated on L-alanine at 45°C</DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Cambridge/Project/Results">http://2012.igem.org/Team:Cambridge/Project/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Project/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Project/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Project/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Project/Results&amp;oldid=238466" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>