"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Attributions
From 2012.igem.org
Parts for a reliable and field ready biosensing platform
Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field. We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems.
One minute tour! :)
>> Return to page
>> Return to page
Contents
1 Judging Form
1.1 iGEM Medals for non-software teams
1.1.1 Requirements for a Bronze Medal
1.1.2 Additional Requirements for a Silver Medal
1.1.3 Additional Requirements for a Gold Medal: (one OR more)
1.2 iGEM Prizes
1.2.1 Please explain briefly why you should receive any of these special prizes:
1.3 Team_Parts
1.4 iGEM Safety
1.5 Attribution and Contributions
1.6 Comments
1.7 Attributions
1.7.1 Sporage and Distribution
1.7.2 Instrumentation
1.7.3 Ribosense
1.7.4 Ratiometrica
1.7.5 Wiki
1.7.6 Human Practices
Judging Form
Please help the judges by filling out this form. Tell them what medal you think you deserve and why. Tell them which special prizes you should win. Help them find your best parts. Show them how you thought about the safety of your project. Helping the judges will help you too.
Team: Cambridge
Region: Europe
iGEM Year:2012
Track:Foundational Advance
Project Name:Parts for a reliable and field ready biosensing platform
Project Abstract: Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field.
We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems.
Back to wiki
iGEM Medals for non-software teams
We believe our team deserves the following medal:
Bronze
Silver
√Gold
Because we met the following criteria (check all that apply and provide details where needed)
Requirements for a Bronze Medal
√Register the team, have a great summer, and plan to have fun at the Regional Jamboree.
√Successfully complete and submit this iGEM 2012 Judging form.
√Create and share a Description of the team's project using the iGEM wiki and the team's parts using the Registry of Standard Biological Parts.
√Plan to present a Poster and Talk at the iGEM Jamboree.
√Enter information detailing at least one new standard BioBrick Part or Device in the Registry of Standard Biological Parts. Including:
√Primary nucleaic acid sequence
√Description of function
√Authorship
Safety notes, if relevant.
√Acknowedgment of sources and references
√Submit DNA for at least one new BioBrick Part or Device to the Registry.
Additional Requirements for a Silver Medal
√Demonstrate that at least one new BioBrick Part or Device of your own design and construction works as expected; characterize the operation of your new part/device.
√Enter this information and other documentation on the part's 'Main Page' section of the Registry
Part Number(s): BBa_K911004
Additional Requirements for a Gold Medal: (one OR more)
Improve an existing BioBrick Part or Device and enter this information back on the Experience Page of the Registry.
Part Number(s): None
√Help another iGEM team by, for example, characterizing a part, debugging a construct, or modeling or simulating their system.
Link to this information on your wiki. Page name: Team:Cambridge/Outreach/Collaboration
√Outline and detail a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation.
Link to this information on your wiki.
Page name: Team:Cambridge/HumanPractices/Overview , Team:Cambridge/HumanPractices/MarketResearch , Team:Cambridge/HumanPractices/FutureDirections
Back to wiki
iGEM Prizes
All teams are eligible for special prizes at the Jamborees. more... To help the judges, please indicate if you feel you should be evaluated for any of the following special prizes:
√Best Human Practice Advance
√Best Experimental Measurement
Best Model
Please explain briefly why you should receive any of these special prizes:
Best Human Practice Advance:
We feel that we deserve this prize for three reasons:
We explored the impacts, *both positive and negative*, of synthetic biology as a solution to real world problems, through interviewing professionals working in a relevant field, namely the impact of arsenic water contamination in Bangladesh.
We recognized existing problems with the way the current direction of synthetic. On going through the registry we found that most of the characterization data for biosensing parts is often neither comparable nor replicable. We have worked to solve this issue, for example with our ratiometric dual channel output.
*Our project doesn’t stop here*, in Chanel number 6 ( Team:Cambridge/HumanPractices/FutureDirections ) we considered the future implications and technological applications of our project, as well as the means by which it could be improved by subsequent users. We feel that the end to an iGEM project should not be the conclusion of an idea, but the start of it.
Best BioBrick Measurement Approach:
It is absolutely vital that a quantitative, numerical, robust, and flexible measurement approach exists to relay information to a user that is an accurate representation of the input processed by a biological device. Working from these principles, the following was done:
We designed and built Biologger, a *cheap, arduino-based, fully functional automatic rotary device* that has an incorporated ratiolumnometer
Our project is entirely open-sourced and open-platform. We have published source code for the two applications which serve to operate the device, one for PCs and the other for Android devices, as well as the open source circuit design that provides this ratiometric reading. Furthermore, the Android app is able to receive its data wirelessly, which we feel is a great advance in BioBrick measurement.
Our dual-channel luciferase reporter was successfully tested with a dilution series of E.coli transformed with the Lux Operon (under pBAD) biobrick (Part  BBa_K325909) of the Cambridge iGEM 2010 team. It can detect, with good accuracy, both different light intensities, as well as the percentages of blue or orange frequencies in a sample.
Our device was successfully tested using artificial light to detect different frequencies (colours) as well.
Having done all the above, we believe that this fully open-sourced instrumentation kit (mechanical) chassis, electronics, software code), estimated at *$35.00* (or $85.00 if a Bluetooth modem is required), is a complete BioBrick measurement solution for any and all BioBricks with a light output.
Back to wiki
Team_Parts
To help the judges evaluate your parts, please identify 3 of your parts that you feel are best documented and are of the highest quality.
Best new BioBrick part (natural)
BBa_K911003
Best new BioBrick part (engineered)
BBa_K911004
Best improved part(s): None
List any other parts you would like the judges to examine: BBa_K911001 , BBa_K911009 , BBa_K911008
Please explain briefly why the judges should examine these other parts:
Magnesium Sensitive Riboswitch BBa_K911001
As a riboswitch sensing construct, this part is an entirely new type of biosensor (along with the fluoride construct) that could potentially change the way we think about designing input genetic circuits. Unlike the fluoride riboswitch, it is a derepression system and therefore serves to demonstrate the principle that riboswitches can be used regardless of whether they turn on or off their reporter.
Fluorescent ratiometric construct for standardizing promoter output BBa_K911009
Fluorescence is a major cornerstone for biosensors in the registry, however, most parts do not involve the use of a ratiometric output, which has been shown in the literature to provide much more reliable and meaningful data. This part not only furthers the development of ratiometric measurements in molecular biology but due to the choice of promoters and terminators it can be used to characterize the difference in activity between E. coli and B. Subtilis
Fast Germination (B. subtilis) BBa_K911008
This part is entirely novel for the registry and fully utilizes the recombination machinery inherent in the Bacillus chassis. Have spores that can germinate at a faster rate is certainly a worthy achievement and could help with experiments with B. Subtilis that any future iGEM teams may wish to perform.
Back to wiki
iGEM Safety
For iGEM 2012 teams are asked to detail how they approached any issues of biological safety associated with their projects.
The iGEM judges expect that you have answered the four safety questions Safety page on your iGEM 2012 wiki.
Please provide the link to that page: Page name: Team:Cambridge/Safety
Attribution and Contributions
For iGEM 2012 the description of each project must clearly attribute work done by the team and distinguish it from work done by others, including the host labs, advisors, and instructors.
Please provide the link to that page, or comments in the box below: Page name: Team:Cambridge/Attributions
Comments
If there is any other information about your project you would like to highlight for the judges, please provide a link to your wiki page here: Team:Cambridge/Overview/DesignProcess
Attributions
Please see below for the attributions of work carried out as part of our project.
Sporage and Distribution
The fast promoter swap over strains we used in sporage and distribution were developed based on work by Peter Setlow from the Setlow lab at the University of Connecticut (see references in the project section). Furthermore Barbara and Peter Setlow sent us the two E. coli plasmids used for transformation of B. Subtilis as well as the finalised spores we used in testing the construct. They also provided us with information on sporulation and germination protocols and help with designing primers for biobricking the part.
Paul Mallaband and Stuart Bell from the team made up sporulation and germination medium and carried out imaging of the spores. They also carried out all relevant pcr, gibson and restriction ligation reactions to make a biobrick of the part.
Paul Grant from the Haseloff lab was instrumental in helping with imaging spores and gave advice on staining, slide preparation and microscopy.
Instrumentation
Andreas Petrides from the team led the development of the instrumentation kit along with Paul Mallaband. Electronics and Arduino code was tackled by Andreas, python code by Paul whilst both took part in the mechanical design and sourcing of materials. The testing of the instrumentation was done by Andreas with the aid of Thomas Whittaker who prepared the biological samples.
The python code was based on that submitted at http://www.blendedtechnologies.com/realtime-plot-of-arduino-serial-data-using-python/231 . All arduino code was developed by Andreas and Paul. The android application was implemented by Andreas, based on Amarino projects' open-source code (General Public License).
Mr. Tim Love from the engineering department gave advice on software design and the Engineering department helped with supplying some of the materials required. The actual assembly of the kit was done entirely by Andreas and Paul.
Ribosense
The fluoride riboswitch, with β-galactosidase reporter, was generously supplied, both in plasmid form, and as transformed cells, by the Breaker laboratory (See references in the project page as well as the special thanks page) at the University of Yale.
The plasmid supplied was then transformed into 168 strain B. subtilis, and Top10 E. coli, by Jolyon Martin from the team. He also carried out the β-galactosidase and Miller Assays.
The magnesium riboswitch was amplified up, using primers of our own design, from a prepared B. subtilis genome sample. This sample was provided by PJ Steiner of the Haseloff lab. Oliver Meacock, from the team, then pieced this amplified DNA into pJS130, a shuttle vector also provided by PJ Steiner. Oliver Meacock also carried out the characterisation assay for the magnesium riboswitch.
All relevant primers for each riboswitch were designed by Jolyon Martin and Oliver Meacock in parallel. All PCR reactions, restriction digests, and other experiments using these parts were carried our by the pair.
Ratiometrica
James Brown from the Haseloff Lab, Cambridge has offered us invaluable advice in collecting and analysing data from our ratiometric fluorescent construct.
PJ Steiner from the Haseloff Lab has provided the original E. coli and B. subtilis shuttle vector pJS130 on which we worked.
Paul Mallaband, Emmy Tsang and Thomas Whittaker from the team designed and assembled the final ratiometric fluorescent construct using PCR and Gibson assembly. The component biobricks (other than the backbone) came from the Registry's Spring Distribution Kit. We designed the original primers for Gibson and ligation, optimised the PCR, gel and PCR extraction, and Gibson assembly protocols, and tested the construct.
Fernan Federici from the Haseloff Lab provided the original mOrange DNA template, while the fischeri LuxBrick (K325909) is from the Parts Registry. Tom and Emmy from the team have assembled the construct using Gibson assembly and PCR.
Tom from the team has designed the harveii Lux-mOrange2 construct, which was then synthesised and codon optimised by DNA 2.0.
Wiki
The javascript and css style sheets used in this wiki are based on those made by Haydn King from the Cambridge 2011 team.
Emmy from the team designed the graphics while Paul and Andreas tweaked the javascripts and css.
Background image for Human Practices on Homepage from here: http://www.flickr.com/photos/foshydog/3590414135/ (under the CCC-BY 2.0 license)
Social media icons from "Infocus Simple White Sidebar Social Media Icons": http://webtreats.mysitemyway.com/1540-infocus-simple-white-sidebar-social-media-icons/
Interface and graphics are designed using Adobe CS5 Photoshop, Illustrator and Dreamweaver. Some of the PS brushes used in both V1.0 and V2.0 of the wiki:
http://www.brusheezy.com/brushes/27476-dbd---swooshpack-lite
http://differentxdreamz.deviantart.com/art/Delicate-Praise-Brushes-134841016
http://www.brusheezy.com/brushes/18307-pretty-designs
Human Practices
Charlotte Bransfield-Garth from the team corresponded with international NGOs, conducting market research and reporting on the real world situation.
Oli from the team interviewed Dr. Konrad Siegfried, a member of the ARSOlux team testing for water contamination in Bangladesh, and also explored the future directions of the our project.
Retrieved from " http://2012.igem.org/Team:Cambridge/Attributions "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
