<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Diary_Week_3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge/Diary/Week_3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Diary/Week_3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Diary/Week_3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Diary/Week_3&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Diary/Week_3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Diary/Week 3</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="cam_page"><DIV id="wrapper_template"><DIV id="main"><DIV id="abstract" style="padding:10px;"><H8 style="color:black; background-color:white;">Parts for a reliable and field ready biosensing platform</H8><P><H9 style="color:black; background-color:white;">Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field. We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems.
</H9></P><P><H9 style="color:black; background-color:white;"><A href="http://prezi.com/tganfyuhurpi/igem-splash/"> One minute tour! :) </A></H9></P><DIV id="abstractclose" style="position: absolute; top: 300px; left: 10px;"><H9 style="color:black; background-color:white;"> &gt;&gt; Return to page </H9></DIV></DIV><DIV id="judgingform" style="padding:10px;"><DIV id="judgingclose"><H9 style="color:black; background-color:white;"> &gt;&gt; Return to page </H9></DIV><DIV id="judgingtext"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Judging_Form"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Judging Form</SPAN></A></LI><LI class="toclevel-2"><A href="#iGEM_Medals_for_non-software_teams"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">iGEM Medals for non-software teams</SPAN></A></LI><LI class="toclevel-3"><A href="#Requirements_for_a_Bronze_Medal"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Requirements for a Bronze Medal</SPAN></A></LI><LI class="toclevel-3"><A href="#Additional_Requirements_for_a_Silver_Medal"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Additional Requirements for a Silver Medal</SPAN></A></LI><LI class="toclevel-3"><A href="#Additional_Requirements_for_a_Gold_Medal:_.28one_OR_more.29"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Additional Requirements for a Gold Medal: (one OR more)</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#iGEM_Prizes"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">iGEM Prizes</SPAN></A></LI><LI class="toclevel-3"><A href="#Please_explain_briefly_why_you_should_receive_any_of_these_special_prizes:"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Please explain briefly why you should receive any of these special prizes:</SPAN></A></LI><LI class="toclevel-2"><A href="#Team_Parts"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Team_Parts</SPAN></A></LI><LI class="toclevel-2"><A href="#iGEM_Safety"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">iGEM Safety</SPAN></A></LI><LI class="toclevel-2"><A href="#Attribution_and_Contributions"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Attribution and Contributions</SPAN></A></LI><LI class="toclevel-2"><A href="#Comments"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Comments</SPAN></A></LI><LI class="toclevel-3 tocsection-1"><A href="#Monday"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">Monday</SPAN></A></LI><LI class="toclevel-3 tocsection-2"><A href="#Tuesday"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">Tuesday</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Wednesday"><SPAN class="tocnumber">1.6.3</SPAN><SPAN class="toctext">Wednesday</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Thursday"><SPAN class="tocnumber">1.6.4</SPAN><SPAN class="toctext">Thursday</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Friday"><SPAN class="tocnumber">1.6.5</SPAN><SPAN class="toctext">Friday</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Saturday"><SPAN class="tocnumber">1.6.6</SPAN><SPAN class="toctext">Saturday</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Judging_Form">Judging Form</SPAN></H1><UL><LI>Please help the judges by filling out this form. Tell them what medal you think you deserve and why. Tell them which special prizes you should win. Help them find your best parts. Show them how you thought about the safety of your project. Helping the judges will help you too.
</LI></UL><UL><LI>Team: Cambridge
</LI><LI>Region: Europe
</LI><LI>iGEM Year:2012
</LI><LI>Track:Foundational Advance
</LI><LI>Project Name:Parts for a reliable and field ready biosensing platform
</LI><LI>Project Abstract: Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field. We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems. 
</LI></UL><P><A href="https://2012.igem.org/Team:Cambridge" class="external text" rel="nofollow">Back to wiki</A></P><H2><SPAN class="mw-headline" id="iGEM_Medals_for_non-software_teams"> iGEM Medals for non-software teams</SPAN></H2><UL><LI>We believe our team deserves the following medal:
</LI><LI> Bronze
</LI><LI> Silver
</LI><LI>√Gold
</LI></UL><P>Because we met the following criteria (check all that apply and provide details where needed)
</P><H3><SPAN class="mw-headline" id="Requirements_for_a_Bronze_Medal">Requirements for a Bronze Medal</SPAN></H3><UL><LI>√Register the team, have a great summer, and plan to have fun at the Regional Jamboree.
</LI><LI>√Successfully complete and submit this iGEM 2012 Judging form.
</LI><LI>√Create and share a Description of the team's project using the iGEM wiki and the team's parts using the Registry of Standard Biological Parts.
</LI><LI>√Plan to present a Poster and Talk at the iGEM Jamboree.
</LI><LI>√Enter information detailing at least one new standard BioBrick Part or Device in the Registry of Standard Biological Parts. Including:
</LI><LI>√Primary nucleaic acid sequence
</LI><LI>√Description of function
</LI><LI>√Authorship
</LI><LI>Safety notes, if relevant.
</LI><LI>√Acknowedgment of sources and references
</LI></UL><LI>√Submit DNA for at least one new BioBrick Part or Device to the Registry.
</LI><H3><SPAN class="mw-headline" id="Additional_Requirements_for_a_Silver_Medal">Additional Requirements for a Silver Medal</SPAN></H3><UL><LI>√Demonstrate that at least one new BioBrick Part or Device of your own design and construction works as expected; characterize the operation of your new part/device.
</LI><LI>√Enter this information and other documentation on the part's 'Main Page' section of the Registry Part Number(s): <A href="http://partsregistry.org/Part:BBa_K911004" class="external text" rel="nofollow">BBa_K911004</A></LI></UL><H3><SPAN class="mw-headline" id="Additional_Requirements_for_a_Gold_Medal:_.28one_OR_more.29">Additional Requirements for a Gold Medal: (one OR more)</SPAN></H3><UL><LI>Improve an existing BioBrick Part or Device and enter this information back on the Experience Page of the Registry.Part Number(s): None
</LI><LI>√Help another iGEM team by, for example, characterizing a part, debugging a construct, or modeling or simulating their system.Link to this information on your wiki. Page name: <A href="https://2012.igem.org/Team:Cambridge/Outreach/Collaboration" class="external text" rel="nofollow">Team:Cambridge/Outreach/Collaboration</A></LI><LI>√Outline and detail a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation.Link to this information on your wiki. Page name: <A href="https://2012.igem.org/Team:Cambridge/HumanPractices/Overview" class="external text" rel="nofollow">Team:Cambridge/HumanPractices/Overview</A>,<A href="https://2012.igem.org/Team:Cambridge/HumanPractices/MarketResearch" class="external text" rel="nofollow">Team:Cambridge/HumanPractices/MarketResearch</A>,<A href="https://2012.igem.org/Team:Cambridge/HumanPractices/FutureDirections" class="external text" rel="nofollow">Team:Cambridge/HumanPractices/FutureDirections</A></LI></UL><P><A href="https://2012.igem.org/Team:Cambridge" class="external text" rel="nofollow">Back to wiki</A></P><H2><SPAN class="mw-headline" id="iGEM_Prizes">iGEM Prizes</SPAN></H2><P>All teams are eligible for special prizes at the Jamborees. <A href="https://2012.igem.org/Judging'" class="external text" rel="nofollow">more...</A> To help the judges, please indicate if you feel you should be evaluated for any of the following special prizes:
</P><UL><LI>√Best Human Practice Advance
</LI><LI>√Best Experimental Measurement
</LI><LI>Best Model
</LI></UL><H3><SPAN class="mw-headline" id="Please_explain_briefly_why_you_should_receive_any_of_these_special_prizes:">Please explain briefly why you should receive any of these special prizes:</SPAN></H3><P><B>Best Human Practice Advance:</B></P><P>We feel that we deserve this prize for three reasons: 
</P><OL><LI> We explored the impacts, *both positive and negative*, of synthetic biology as a solution to real world problems, through interviewing professionals working in a relevant field, namely the impact of arsenic water contamination in Bangladesh. 
</LI><LI> We recognized existing problems with the way the current direction of synthetic. On going through the registry we found that most of the characterization data for biosensing parts is often neither comparable nor replicable. We have worked to solve this issue, for example with our ratiometric dual channel output. 
</LI><LI> *Our project doesn’t stop here*, in Chanel number 6 (<A href="https://2012.igem.org/Team:Cambridge/HumanPractices/FutureDirections" class="external text" rel="nofollow">Team:Cambridge/HumanPractices/FutureDirections</A>) we considered the future implications and technological applications of our project, as well as the means by which it could be improved by subsequent users. We feel that the end to an iGEM project should not be the conclusion of an idea, but the start of it.
</LI></OL><P><B>Best BioBrick Measurement Approach:</B></P><P>It is absolutely vital that a quantitative, numerical, robust, and flexible measurement approach exists to relay information to a user that is an accurate representation of the input processed by a biological device. Working from these principles, the following was done:
</P><OL><LI> We designed and built Biologger, a *cheap, arduino-based, fully functional automatic rotary device* that has an incorporated ratiolumnometer
</LI><LI> Our project is entirely open-sourced and open-platform. We have published source code for the two applications which serve to operate the device, one for PCs and the other for Android devices, as well as the open source circuit design that provides this ratiometric reading. Furthermore, the Android app is able to receive its data wirelessly, which we feel is a great advance in BioBrick measurement. 
</LI><LI> Our dual-channel luciferase reporter was successfully tested with a dilution series of E.coli transformed with the Lux Operon (under pBAD) biobrick (Part  BBa_K325909) of the Cambridge iGEM 2010 team. It can detect, with good accuracy, both different light intensities, as well as the percentages of blue or orange frequencies in a sample. 
</LI><LI> Our device was successfully tested using artificial light to detect different frequencies (colours) as well.
</LI></OL><P>Having done all the above, we believe that this fully open-sourced instrumentation kit (mechanical) chassis, electronics, software code), estimated at *$35.00* (or $85.00 if a Bluetooth modem is required), is a complete BioBrick measurement solution for any and all BioBricks with a light output.
</P><P><A href="https://2012.igem.org/Team:Cambridge" class="external text" rel="nofollow">Back to wiki</A></P><H2><SPAN class="mw-headline" id="Team_Parts">Team_Parts</SPAN></H2><P>To help the judges evaluate your parts, please identify 3 of your parts that you feel are best documented and are of the highest quality. 
</P><UL><LI><DL><DT>Best new BioBrick part (natural)</DT><DD><A href="http://partsregistry.org/Part:BBa_K911003" class="external text" rel="nofollow">BBa_K911003</A></DD><DT>Best new BioBrick part (engineered)</DT><DD><A href="http://partsregistry.org/Part:BBa_K911004" class="external text" rel="nofollow">BBa_K911004</A></DD></DL></LI><LI>Best improved part(s): None
</LI></UL><P>List any other parts you would like the judges to examine:<A href="http://partsregistry.org/Part:BBa_K911001" class="external text" rel="nofollow">BBa_K911001</A>, <A href="http://partsregistry.org/Part:BBa_K911008" class="external text" rel="nofollow">BBa_K911009</A>, <A href="http://partsregistry.org/Part:BBa_K911008" class="external text" rel="nofollow">BBa_K911008</A></P><P>Please explain briefly why the judges should examine these other parts:
</P><UL><LI><B>Magnesium Sensitive Riboswitch <A href="http://partsregistry.org/Part:BBa_K911001" class="external text" rel="nofollow">BBa_K911001</A></B>As a riboswitch sensing construct, this part is an entirely new type of biosensor (along with the fluoride construct) that could potentially change the way we think about designing input genetic circuits. Unlike the fluoride riboswitch, it is a derepression system and therefore serves to demonstrate the principle that riboswitches can be used regardless of whether they turn on or off their reporter.
</LI><LI><B>Fluorescent ratiometric construct for standardizing promoter output <A href="http://partsregistry.org/Part:BBa_K911009" class="external text" rel="nofollow">BBa_K911009</A></B>Fluorescence is a major cornerstone for biosensors in the registry, however, most parts do not involve the use of a ratiometric output, which has been shown in the literature to provide much more reliable and meaningful data. This part not only furthers the development of ratiometric measurements in molecular biology but due to the choice of promoters and terminators it can be used to characterize the difference in activity between E. coli and B. Subtilis
</LI><LI><B>Fast Germination (B. subtilis) <A href="http://partsregistry.org/Part:BBa_K911008" class="external text" rel="nofollow">BBa_K911008</A></B>This part is entirely novel for the registry and fully utilizes the recombination machinery inherent in the Bacillus chassis. Have spores that can germinate at a faster rate is certainly a worthy achievement and could help with experiments with B. Subtilis that any future iGEM teams may wish to perform.
</LI></UL><P><A href="https://2012.igem.org/Team:Cambridge" class="external text" rel="nofollow">Back to wiki</A></P><H2><SPAN class="mw-headline" id="iGEM_Safety">iGEM Safety</SPAN></H2><P>For iGEM 2012 teams are asked to detail how they approached any issues of biological safety associated with their projects.
</P><P>The iGEM judges expect that you have answered the four safety questions <A href="https://2012.igem.org/Safety" class="external text" rel="nofollow">Safety page</A> on your iGEM 2012 wiki.
</P><P>Please provide the link to that page:
Page name: <A href="https://2012.igem.org/Team:Cambridge/Safety" class="external text" rel="nofollow">Team:Cambridge/Safety</A></P><H2><SPAN class="mw-headline" id="Attribution_and_Contributions">Attribution and Contributions</SPAN></H2><P>For iGEM 2012 the description of each project must clearly attribute work done by the team and distinguish it from work done by others, including the host labs, advisors, and instructors.
</P><P>Please provide the link to that page, or comments in the box below:
Page name: <A href="https://2012.igem.org/Team:Cambridge/Attributions" class="external text" rel="nofollow">Team:Cambridge/Attributions</A></P><H2><SPAN class="mw-headline" id="Comments">Comments</SPAN></H2><P>If there is any other information about your project you would like to highlight for the judges, please provide a link to your wiki page here: <A href="https://2012.igem.org/Team:Cambridge/Overview/DesignProcess" class="external text" rel="nofollow">Team:Cambridge/Overview/DesignProcess</A></P></DIV></DIV><DIV id="template_content_wide" style="position: absolute; top: 200px; left: 40px; height: 620px; width: 1110px; overflow: auto; overflow-y: scroll;"><TABLE align="center"><TBODY><TR><TH align="center">Week:
</TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_1" title="Team:Cambridge/Diary/Week 1">1</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_2" title="Team:Cambridge/Diary/Week 2">2</A></TH><TH align="center"><STRONG class="selflink">3</STRONG></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_4" title="Team:Cambridge/Diary/Week 4">4</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_5" title="Team:Cambridge/Diary/Week 5">5</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_6" title="Team:Cambridge/Diary/Week 6">6</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_7" title="Team:Cambridge/Diary/Week 7">7</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_8" title="Team:Cambridge/Diary/Week 8">8</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_9" title="Team:Cambridge/Diary/Week 9">9</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_10" title="Team:Cambridge/Diary/Week 10">10</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_11" title="Team:Cambridge/Diary/Week 11">11</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_12" title="Team:Cambridge/Diary/Week 12">12</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_13" title="Team:Cambridge/Diary/Week 13">13</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/Week_14" title="Team:Cambridge/Diary/Week 14">14</A></TH><TH align="center"><A href="/Team:Cambridge/Diary/TheFinalMonth" title="Team:Cambridge/Diary/TheFinalMonth">The Final Month</A></TH></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Monday">Monday</SPAN></H3><P>We aimed to start doing something towards our project today. To this end, Tom and Oli began making up a batch of competent <I>bacillus</I> (strain 168) for freezing and future use. The protocol for this can be found <A href="/Team:Cambridge/Protocols/TransformationofB.subtilis" title="Team:Cambridge/Protocols/TransformationofB.subtilis">here</A>.
</P><P>Paul and Andreas went over to the photonics department to begin the process of making our circuit for the analysis of the bacterial luciferase signal. They also began formalizing the design of the actual instrumentation so it could be made over the next few days.
</P><P>Jolyon managed to find an endogenous fluoride riboswitch that had been reported in bacillus subtilis. As a novel biobrick, this would have obvious applications in the analysis of water purity. Such a device could also be pitched at persons a la General Ripper, concerned about fluride contaminating their &quot;precious bodily fluids&quot;, though such a marketing strategy could produce somewhat sporadic orders. 
</P><P>Some of us also started using the faintly egomaniacal geneious program for the analysis of DNA. 
</P><P>All in all, a fairly productive first day!
</P><H3><SPAN class="mw-headline" id="Tuesday">Tuesday</SPAN></H3><P>Oli and Tom continued with the rather elongated protocol for creating competent <I>bacillus</I>. 
</P><P>Jolyon contacted the team in Yale who had discovered the <I>bacillus</I> based riboswitch for the detection of fluoride. Hopefullly we should be able to integrate it into some sort of reasonably fast acting novel bacterial biosensor which we can then hook up to our standardized bioluminescence. In order to bring about this standardization, Andreas introduced the biologists to some of the principles of control theory, so we had some of the conceptual tools to change the shapes of the input/output curves. One important possibility is using these control theory principles to reduce the noise of the response.
</P><P>Andreas and Paul also started on the software for our device today. Our current plan is to use an Arduino board to detect our inputs, communicate this information to a python script and then present this on a user interface. Very snazzy.
</P><P>In addition to this, Emmy got the completed home page online! Considerable! A graphic designer as well as a biologist in our midst...
</P><H3><SPAN class="mw-headline" id="Wednesday">Wednesday</SPAN></H3><P>The first results of our light sensor made by Andreas and the user interface made by Paul came out today. See the lab book for the results.
</P><P>We also built up a schedule for when each of our individual modules should be finished by. Our final schedule can be seen on the calendar. Hopefully we shouldn't look too unprepared when UEA visit tomorrow. By the way, UEA are visiting tomorrow.
</P><P>Continued wrangling for the already constructed plasmids also took place, with Tom trying to get his hands on an optimized lux operon for use in <I>bacillus</I>. Whether or not it is actually codon optimized is, at this time, unknown. If the worst comes to the worst however, we can just get DNA 2.0 to synthesize the whole thing and optimize it for <I>bacillus</I>. We may have finally found a use for our synthesis budget.
</P><P>Lastly, we finally manage to make up and freeze our stocks of <I>bacillus</I>. At eight fifteen, Tom and Oli finally exited the building after a ten hour <A href="/Team:Cambridge/Protocols/TransformationofB.subtilis" title="Team:Cambridge/Protocols/TransformationofB.subtilis">protocol</A>. At least we won't have to do it again for a while.
</P><P>Just how competent our attempts at competency were will be tested tomorrow.
</P><H3><SPAN class="mw-headline" id="Thursday">Thursday</SPAN></H3><P>The UEA team visited today. See pictures of their visit in the gallery. Hopefully we should be able to collaborate as soon as they manage to create their biosensor for nitrate.
</P><P>Andreas and Oli tried transforming the <I>bacillus</I> stock we made up yesterday. The plasmid used was Tag RFP-T which gives choramphenicol resistance and should turn the colonies pink. We will have to wait overnight to see if the week has been wasted.
</P><P>We've also started designing the primers we need for the various steps in the procedures. Jolyon made some potentially effective ones for extracting the fluoride riboswitch region from the <I>bacillus</I> genome. Using Gibson assembly, it should then be possible to insert this upstream of our reporter, i.e the mOrange/luciferase fusion. See our <A href="/Team:Cambridge/Ribosense/Overview" title="Team:Cambridge/Ribosense/Overview">project page</A> for how we imagine this should turn out in the end.
</P><H3><SPAN class="mw-headline" id="Friday">Friday</SPAN></H3><P>Our transformants actually seem to have worked! See the lab book for a lovely image of our pink colonies.
</P><P>Stuart took an inventory of the large quantity of freezer stocks left over from last year. Lots of molecular biology stuff - enzymes, buffers, DNA ladders, loading dyes etc. etc. Plus many helpfully unlabeled tubes from years before 2011. We probably won't be risking those.
</P><P>Emmy and Paul worked with the old lux operons from 2010, inserting the plasmids into the <I>bacillus</I> made up two days ago, as well as into some e.coli. Hopefully we should have some bioluminescence to test by the end of the weekend - useful for the Arduino kit which is becoming ever more developed as the hours go by. 
</P><P>Cake days were instituted today, with the inaugural sample being Emmy's banana bread. They may also act as excellent bargaining chips for enlisting the aid of reticent PhD students.
</P><P>Andreas has managed to find some optical filters, which he later tested in the spectrophotometer. The data was plotted in MATLAB by Oli. Andreas has also discovered that he hates the odour of e.coli. How fortunate that we are working mostly with <I>bacillus</I> instead, which has the delightful aroma of unwashed feet.
</P><H3><SPAN class="mw-headline" id="Saturday">Saturday</SPAN></H3><P>Emmy came in today to check on the e.coli and <I>bacillus</I> plates she and Paul made up yesterday. They don't seem to have grown...
</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Cambridge/Diary/Week_3">http://2012.igem.org/Team:Cambridge/Diary/Week_3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Diary/Week_3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Diary/Week_3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Diary/Week_3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Diary/Week_3&amp;oldid=298260" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>