"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Diary/Week 4
From 2012.igem.org
Parts for a reliable and field ready biosensing platform
Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field. We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems.
One minute tour! :)
>> Return to page
>> Return to page
Contents
1 Judging Form
1.1 iGEM Medals for non-software teams
1.1.1 Requirements for a Bronze Medal
1.1.2 Additional Requirements for a Silver Medal
1.1.3 Additional Requirements for a Gold Medal: (one OR more)
1.2 iGEM Prizes
1.2.1 Please explain briefly why you should receive any of these special prizes:
1.3 Team_Parts
1.4 iGEM Safety
1.5 Attribution and Contributions
1.6 Comments
1.6.1 Monday
1.6.2 Tuesday
1.6.3 Wednesday
1.6.4 Thursday
1.6.5 Friday
Judging Form
Please help the judges by filling out this form. Tell them what medal you think you deserve and why. Tell them which special prizes you should win. Help them find your best parts. Show them how you thought about the safety of your project. Helping the judges will help you too.
Team: Cambridge
Region: Europe
iGEM Year:2012
Track:Foundational Advance
Project Name:Parts for a reliable and field ready biosensing platform
Project Abstract: Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field.
We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems.
Back to wiki
iGEM Medals for non-software teams
We believe our team deserves the following medal:
Bronze
Silver
√Gold
Because we met the following criteria (check all that apply and provide details where needed)
Requirements for a Bronze Medal
√Register the team, have a great summer, and plan to have fun at the Regional Jamboree.
√Successfully complete and submit this iGEM 2012 Judging form.
√Create and share a Description of the team's project using the iGEM wiki and the team's parts using the Registry of Standard Biological Parts.
√Plan to present a Poster and Talk at the iGEM Jamboree.
√Enter information detailing at least one new standard BioBrick Part or Device in the Registry of Standard Biological Parts. Including:
√Primary nucleaic acid sequence
√Description of function
√Authorship
Safety notes, if relevant.
√Acknowedgment of sources and references
√Submit DNA for at least one new BioBrick Part or Device to the Registry.
Additional Requirements for a Silver Medal
√Demonstrate that at least one new BioBrick Part or Device of your own design and construction works as expected; characterize the operation of your new part/device.
√Enter this information and other documentation on the part's 'Main Page' section of the Registry
Part Number(s): BBa_K911004
Additional Requirements for a Gold Medal: (one OR more)
Improve an existing BioBrick Part or Device and enter this information back on the Experience Page of the Registry.
Part Number(s): None
√Help another iGEM team by, for example, characterizing a part, debugging a construct, or modeling or simulating their system.
Link to this information on your wiki. Page name: Team:Cambridge/Outreach/Collaboration
√Outline and detail a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation.
Link to this information on your wiki.
Page name: Team:Cambridge/HumanPractices/Overview , Team:Cambridge/HumanPractices/MarketResearch , Team:Cambridge/HumanPractices/FutureDirections
Back to wiki
iGEM Prizes
All teams are eligible for special prizes at the Jamborees. more... To help the judges, please indicate if you feel you should be evaluated for any of the following special prizes:
√Best Human Practice Advance
√Best Experimental Measurement
Best Model
Please explain briefly why you should receive any of these special prizes:
Best Human Practice Advance:
We feel that we deserve this prize for three reasons:
We explored the impacts, *both positive and negative*, of synthetic biology as a solution to real world problems, through interviewing professionals working in a relevant field, namely the impact of arsenic water contamination in Bangladesh.
We recognized existing problems with the way the current direction of synthetic. On going through the registry we found that most of the characterization data for biosensing parts is often neither comparable nor replicable. We have worked to solve this issue, for example with our ratiometric dual channel output.
*Our project doesn’t stop here*, in Chanel number 6 ( Team:Cambridge/HumanPractices/FutureDirections ) we considered the future implications and technological applications of our project, as well as the means by which it could be improved by subsequent users. We feel that the end to an iGEM project should not be the conclusion of an idea, but the start of it.
Best BioBrick Measurement Approach:
It is absolutely vital that a quantitative, numerical, robust, and flexible measurement approach exists to relay information to a user that is an accurate representation of the input processed by a biological device. Working from these principles, the following was done:
We designed and built Biologger, a *cheap, arduino-based, fully functional automatic rotary device* that has an incorporated ratiolumnometer
Our project is entirely open-sourced and open-platform. We have published source code for the two applications which serve to operate the device, one for PCs and the other for Android devices, as well as the open source circuit design that provides this ratiometric reading. Furthermore, the Android app is able to receive its data wirelessly, which we feel is a great advance in BioBrick measurement.
Our dual-channel luciferase reporter was successfully tested with a dilution series of E.coli transformed with the Lux Operon (under pBAD) biobrick (Part  BBa_K325909) of the Cambridge iGEM 2010 team. It can detect, with good accuracy, both different light intensities, as well as the percentages of blue or orange frequencies in a sample.
Our device was successfully tested using artificial light to detect different frequencies (colours) as well.
Having done all the above, we believe that this fully open-sourced instrumentation kit (mechanical) chassis, electronics, software code), estimated at *$35.00* (or $85.00 if a Bluetooth modem is required), is a complete BioBrick measurement solution for any and all BioBricks with a light output.
Back to wiki
Team_Parts
To help the judges evaluate your parts, please identify 3 of your parts that you feel are best documented and are of the highest quality.
Best new BioBrick part (natural)
BBa_K911003
Best new BioBrick part (engineered)
BBa_K911004
Best improved part(s): None
List any other parts you would like the judges to examine: BBa_K911001 , BBa_K911009 , BBa_K911008
Please explain briefly why the judges should examine these other parts:
Magnesium Sensitive Riboswitch BBa_K911001
As a riboswitch sensing construct, this part is an entirely new type of biosensor (along with the fluoride construct) that could potentially change the way we think about designing input genetic circuits. Unlike the fluoride riboswitch, it is a derepression system and therefore serves to demonstrate the principle that riboswitches can be used regardless of whether they turn on or off their reporter.
Fluorescent ratiometric construct for standardizing promoter output BBa_K911009
Fluorescence is a major cornerstone for biosensors in the registry, however, most parts do not involve the use of a ratiometric output, which has been shown in the literature to provide much more reliable and meaningful data. This part not only furthers the development of ratiometric measurements in molecular biology but due to the choice of promoters and terminators it can be used to characterize the difference in activity between E. coli and B. Subtilis
Fast Germination (B. subtilis) BBa_K911008
This part is entirely novel for the registry and fully utilizes the recombination machinery inherent in the Bacillus chassis. Have spores that can germinate at a faster rate is certainly a worthy achievement and could help with experiments with B. Subtilis that any future iGEM teams may wish to perform.
Back to wiki
iGEM Safety
For iGEM 2012 teams are asked to detail how they approached any issues of biological safety associated with their projects.
The iGEM judges expect that you have answered the four safety questions Safety page on your iGEM 2012 wiki.
Please provide the link to that page: Page name: Team:Cambridge/Safety
Attribution and Contributions
For iGEM 2012 the description of each project must clearly attribute work done by the team and distinguish it from work done by others, including the host labs, advisors, and instructors.
Please provide the link to that page, or comments in the box below: Page name: Team:Cambridge/Attributions
Comments
If there is any other information about your project you would like to highlight for the judges, please provide a link to your wiki page here: Team:Cambridge/Overview/DesignProcess
Week:
1
2
3
4
5
6
7
8
9
10
11
12
13
14
The Final Month
Monday
Scandal in the lab! It appears that someone at Caltech has edited our wiki, initially inserting the word 'chicken' into our project page. Then they broke our Java script - potentially accidentally. Easily rectified, but still irritating, especially since the project description deadline was yesterday. Have we been the victim of a not-so-droll ivy league-ish prank? Or is some oddly minded person co-opting this student's account? We've emailed their project supervisor, so hopefully it shouldn't happen again.
One night seems to have been too little time to accurately determine if the transformation worked, as we now have lots of e.coli colonies on the plates. The bacillus plates have not been so successful, but the plasmid was not optimized to bacillus, potentially explaining their failure. We will need to make sure that we include more controls in future to work out where our experiments fail, if they should fail.
In any case, we were able to induce the pBAD promoter with arabinose in the e.coli cells. Results can be seen in the lab book page. We also streaked some further plates to grow up some more transformed e.coli, and inoculated some cells in liquid medium which will hopefully be ready to be tested tomorrow.
The construct we are hoping to submit to DNA 2.0 has also been partially designed. The fusion of the lux α subunit and the mOrange protein has been completed, as described in the project page . The v.haveyi operon that we are hoping to have codon optimized has also been tided up, with more appropriate spacings between the genes.
Additionally, Andreas managed to finish his Arduino device. It now gives a response that is proportional to the ratio of the light intensities on the two light dependent resistors. By using a couple of coloured filters over the top of these resistors, we can measure the necessary spectral properties of our engineered luciferases. All the engineers need to do now is come up with a user interface.
Tuesday
Apologies from the editing parties at Caltech, apparently the remaining changes were not meant to be there. Unfortunately they also edited Brown's wiki. A storm may be brewing, depending how seriously they take the changes...
Anyway, apart from that little drama, lots happened in the lab today. Jolyon recieved his magnesium ion riboswitch primers - see what we did with them in the lab book.
Tom worked on making the construct we need to get ordered from DNA 2.0 using their program Gene Designer. He ran into a few issues, but Jim A. was able to give a few hints, and put him into contact with some friendly people in Newcastle who specialize in working with bacillus. If they don't know the answers, no one will.
Paul and Emmy are running into slight trouble with getting the right arabinose concentration for induction of luciferase production for the E. coli. It seems that with the optimum concentration suggested by previous years' teams there was very little light- so we added more arabinose. Cells are still not giving out any visible amount of light after 5 hours, and we now suspect that we might have killed the cells with too much sugar.
We're also having some problems deciding what approach to take with the input side of things. Our system should be very flexible, so it would be good to use many different biobricks from the registry. However, very few have been characterized in bacillus. Ideally we would want to find some way of transferring existing biobricks from e.coli so that they were reliable in bacillus, but we may find getting the right sensory proteins into bacillus complex. The idea was floated that we might want to approach the problem as a business problem, and do some market research into what the hypothetical end consumers of our product might want to sense. This would mean pitching to a specific group, potentially obscuring the main point of the project (the breadth and flexibility of the lux based system and its quantitative measurement of concentrations). For example, if we decided to create an agricultural kit as our prototype kit, we would want to create nitrate sensors, iron sensors, osmolarity sensors, fluride sensors etc. However, this would mean leaving out some very interesting additional sensory components. See what we eventually went with on our human practices page .
Wednesday
The day started with gel electrophoresis of the PCR fragments that were made yesterday. As can be seen in the lab book, our genomic extraction appears to have been successful. However, we haven't managed to make our vector properly. Another day of PCR and gel electrophoresis, I fear...
Paul and Emmy inoculated cells from the streaked plates in liquid medium- hopefully these will work.
Prototyping of the hardware that we are hoping to use progressed to foam. Andreas is hoping to make the structure look more professional-like by spraying it with various faintly toxic (yet shiny) aerosols.
In the afternoon James Brown gave an inspiring talk, leading to us redefining our modules and goals. We also have a clearer idea on what our individual roles are.
Thursday
PCR with more controls and repeats, with slightly tweaked PCR parameters, resulted in yet more failure. The primers probably need to be redesigned.
Tom gets even deeper into construct design, with some light primer design for some of the simpler constructs to break up the day. With help from Orr from Jim A's lab, we were able to identify some constitutive promoters that we could use in the ratiometric construct. We further discovered that potentially all of the components that we need for the construct are in the iGEM 2012 Spring Distribution Kit! So we could start straight away once we have got the primers.
Our final designation or the team also occurred today, with us formalising our plans for the Summer in terms of the modules we are hoping to undertake. If all goes well, you should be able to see our modules under the project page .
Friday
Charlie had a brief chat with Dr. Vijayraghavan about potential applications of our project. She offered some useful advice given her proximity to the Bangalore and Caltech teams. See what she had to say in the human practices page .
Our newly inoculated bacteria shine ever brighter, with their induction by arabinose creating more successful results than before. In particular, Andreas and Emmy took them down to the dark room and tested if Andrea's electronics were sensitive to the light levels given off by the cultures. And surprisingly enough, it worked! A consistent 5-6% increase of the original base reading. The change was reliable and repeatable. The only worry is the time period for which light is produced by the bacteria after induction; full luminesence only takes place about 5 hours after the inducing arabinose is added.
Construct design by Tom continued, with the construct for the ratiometric emission by YFP and CFP based upon inducer concentration being made in a couple of hours. Difficulties persist with the construction of the lux operon for bacillus however. There really isn't enough data in the literature to make the RBS's properly - something which we may aim to rectify.
Lastly, the primer redesign was successful. Unfortunately, primer ordering is hampered by a lack of any add to basket button on the Sigma-Aldritch. Even more unfortunately, this week seems to be holiday week for the whole of the department, and everyone who could help is away. Something may have to be done over the weekend.
Retrieved from " http://2012.igem.org/Team:Cambridge/Diary/Week_4 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
