"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Lab book/Week 10
From 2012.igem.org
Diary
Lab book
Protocols
Tom's Quotes
This is a detailed record of our work during the Summer...Read More
Project
Outreach
Gallery
Diary
Team
Home
Safety
Sponsors
Week:
3
4
5
6
7
8
9
10
11
12
13
14
Contents
1 Monday (27/08/12)
2 Tuesday (28/08/12)
3 Wednesday (29/08/12)
4 Thursday (30/08/12)
5 Friday (31/08/12)
6 Saturday (01/09/12)
7 Sunday (02/09/12)
Monday (27/08/12)
RiboSense: PCR of Mg RS from bacillus genomic DNA
3 part primers used, to amplify the 486bp from the genome, and the vector in two parts
PCR failed, only positive control came out. Will re run when new primers arrive
Tuesday (28/08/12)
Wednesday (29/08/12)
Thursday (30/08/12)
PCR of Mg2+ riboswitch from genomic DNA
Gel from amplification of the riboswitch DNA. Lanes 2 + 3: with 8 codon substitution. Lanes 4 + 5: without 8 codon substitution.
Cycle settings:
Melting - 98 °C - 10 seconds
Annealing - 58 °C - 30 seconds
Elongation - 72 °C - 30 seconds
Fragments of correct size produced for all except lane 5 produced. In this lane, DNA appears to have accumulated in the well, indicating it may be genomic. In future, will use lower numbers of template cells to avoid getting so much genomic DNA.
Products extracted and purified.
PCR of PJS130 vector for Mg riboswitch construct
Gel from amplification of PJS130 vector for MGRS construct. Lanes 2 + 3: Fragment A. Lanes 4 + 5: Fragment B with 8 codon substitution. Lanes 6 + 7: Fragment B without 8 codon substitution.
Cycle settings:
Melting - 98 °C - 10 seconds
Annealing - 60 °C - 30 seconds
Elongation - 72 °C - 110 seconds
Fragment of correct size produced in lane 5, but too faint to be extracted successfully. None of the other lanes were successful. We will try this again tomorrow, at 58 °C and with 35 cycles (as is usual) instead of 30.
Note that phusion enzyme was left with primers and template DNA for about half an hour before reaction began, possibly causing degradation of the primer DNA.
Friday (31/08/12)
PCR of PJS130 vector for MgRS construct
Gel from amplification of PJS130 fragments. Lanes 2 + 3: Fragment A. Lanes 4 + 5: Fragment B with eight codon substitution. Lanes 6 + 7: Fragment B without eight codon substitution. Lane 8: Negative control.
Cycle settings:
Melting - 98 °C - 10 seconds
Annealing - 58 °C - 30 seconds
Elongation - 72 °C - 120 seconds
Products of correct sizes (5.5kbp and 3.5 kbp) produced for all reactions, although lanes 2 and 4 failed to produce any product, despite primer smear. Most likely, template was not added, or one of the primers was not added.
Products excised and purified.
Gibson assembly of Mg2+ riboswitch construct
Reaction 1: Without 8 codon substitution: Vec A, Vec B -8 (replicate 1), Genomic -8.
Reaction 2: Without 8 codon substitution: Vec A, Vec B -8 (replicate 2), Genomic -8.
Reaction 3: With 8 codon substitution: Vec A, Vec B +8, Genomic +8 (replicate 1).
Reaction 4: With 8 codon substitution: Vec A, Vec B +8, Genomic +8 (replicate 2).
Transformation of e.coli with Gibson products
20 μl of Gibson reaction mix transformed into e.coli cells. Transformants plated out onto 100 μg/ml ampicillin plates.
Saturday (01/09/12)
Verification of Mg2+ gel extractions
Verification gel of extracted DNA. Lanes 1,7 + 13: Ladder. Lanes 2 - 4: Old vector DNA (06/08/12). Lane 5: PJS130 fragment A. Lanes 6,8 + 9: PJS130 fragment B. Lanes 10 - 12: Genomic riboswitch DNA.
DNA used in Gibson from yesterday and several weeks ago. Run on gel to verify the presence of DNA fragments of the correct size after gel extraction.
Old DNA appears to have degraded, or else products were produced in too small a quantity to show up on the gel. Fortunately, these are not/ have not been used.
Bands of the correct size for fragment A (5.5kbp), fragment B (3.5kbp) and the genomic DNA (550bp) present. Though riboswitch DNA has not come out well in this image, it was present under visual examination.
Approximate DNA concentrations in the range of 2 - 6 ng/μl.
General
Our Gibson assembly is at a lower efficiency than it should be and this is causing assembly problems. Initiated a series of controls with PJ Steiner, who gets very high efficiency.
Initial experiments - using our positive control fragments, PJ and I assembled in a 4 ul reaction (seperately). I used my master mix and his, and he used his master mix with our competent cells and his. This should reveal any difference between our reagents or cells.
Sunday (02/09/12)
Results from colony PCR (right of central ladder) and of restriction digest (left of central ladder). Restriction digest produced inconclusive results, however colony PCR indicates that the riboswitch only inserted into the second of the two colonies picked from the +8 version plate (last two lanes before ladder).
Verification of Mg2+ riboswitch construct by colony PCR
Gibson primers used to amplify out riboswitch section of plasmid produced and transformed into e.coli to verify if riboswitch had inserted into PJS130.
Band at ~550bp in lanes 8 and 9 (replicates) indicate that the riboswich has only been successfuly transformed into the second of the two colonies picked on the +8 plate, and not at all for the -8 construct.
Miniprep of plasmid DNA
E.coli from strains made yesterday miniprepped to get plasmid DNA at high concentration. After colony PCR results, all DNA other than that of the successful colony discarded.
Verification of Mg2+ riboswitch construct by restriction digest
BamH1 and Sal1 used to digest plasmid DNA produced from miniprep.
Resultant fragment run on gel.
Bands produced inconclusive, possibly because of too short an incubation time with restriction enzymes. Expected bands at ~4900bp and ~4500bp. Will re-run experiment in future to check results of colony PCR.
Gibson efficiency diagnostics
We decided to try to get to the bottom of why our Gibson assembly is so inefficient. Until this is fixed, we won't be able to do anything higher than 2-part reactions.
PJ, a member of the Haseloff lab whose gibson assemblies are working efficiently agreed to help us. Using our control fragments, PJ assembled them with our master mix and his master mix, and transformed them into his competent E.coli. Similarly, we assembled our fragments (from the same tube) with our master mix, but using his competent cells and our competent cells. This should show us if there is a problem with our mix or cells.
Retrieved from " http://2012.igem.org/Team:Cambridge/Lab_book/Week_10 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
