<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Ratiometrica_Labbook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Cambridge/Ratiometrica/Labbook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Ratiometrica/Labbook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Ratiometrica/Labbook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Ratiometrica/Labbook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Ratiometrica/Labbook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Ratiometrica/Labbook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="cam_page"><DIV id="wrapper_template"><DIV id="main"><DIV id="abstract" style="padding:10px;"><H8 style="color:black; background-color:white;">Parts for a reliable and field ready biosensing platform</H8><P><H9 style="color:black; background-color:white;">Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field. We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems.
</H9></P><P><H9 style="color:black; background-color:white;"><A href="http://prezi.com/tganfyuhurpi/igem-splash/"> One minute tour! :) </A></H9></P><DIV id="abstractclose" style="position: absolute; top: 300px; left: 10px;"><H9 style="color:black; background-color:white;"> &gt;&gt; Return to page </H9></DIV></DIV><DIV id="judgingform" style="padding:10px;"><DIV id="judgingclose"><H9 style="color:black; background-color:white;"> &gt;&gt; Return to page </H9></DIV><DIV id="judgingtext"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Judging_Form"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Judging Form</SPAN></A></LI><LI class="toclevel-2"><A href="#iGEM_Medals_for_non-software_teams"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">iGEM Medals for non-software teams</SPAN></A></LI><LI class="toclevel-3"><A href="#Requirements_for_a_Bronze_Medal"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Requirements for a Bronze Medal</SPAN></A></LI><LI class="toclevel-3"><A href="#Additional_Requirements_for_a_Silver_Medal"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Additional Requirements for a Silver Medal</SPAN></A></LI><LI class="toclevel-3"><A href="#Additional_Requirements_for_a_Gold_Medal:_.28one_OR_more.29"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Additional Requirements for a Gold Medal: (one OR more)</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#iGEM_Prizes"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">iGEM Prizes</SPAN></A></LI><LI class="toclevel-3"><A href="#Please_explain_briefly_why_you_should_receive_any_of_these_special_prizes:"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Please explain briefly why you should receive any of these special prizes:</SPAN></A></LI><LI class="toclevel-2"><A href="#Team_Parts"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Team_Parts</SPAN></A></LI><LI class="toclevel-2"><A href="#iGEM_Safety"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">iGEM Safety</SPAN></A></LI><LI class="toclevel-2"><A href="#Attribution_and_Contributions"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Attribution and Contributions</SPAN></A></LI><LI class="toclevel-2"><A href="#Comments"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Comments</SPAN></A></LI><LI class="toclevel-1 tocsection-1"><A href="#Ratiometrica_Labbook"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Ratiometrica Labbook</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Week_4"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Week 4</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Monday_.2816.2F07.2F12.29"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Monday (16/07/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Wednesday_.2818.2F07.2F12.29"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Wednesday (18/07/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Friday_.2820.2F07.2F12.29"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Friday (20/07/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Week_5"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Week 5</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Thursday_.2826.2F07.2F12.29"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Thursday (26/07/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Friday_.2827.2F07.2F12.29"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Friday (27/07/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Week_6"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Week 6</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#Monday_.2830.2F07.2F12.29"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">Monday (30/07/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Tuesday_.2831.2F07.2F12.29"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Tuesday (31/07/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Friday_.2803.2F08.2F12.29"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">Friday (03/08/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Week_7"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Week 7</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Monday_.2806.2F08.2F12.29"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">Monday (06/08/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Tuesday_.2807.2F08.2F12.29"><SPAN class="tocnumber">2.4.2</SPAN><SPAN class="toctext">Tuesday (07/08/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Wednesday_.2808.2F08.2F12.29"><SPAN class="tocnumber">2.4.3</SPAN><SPAN class="toctext">Wednesday (08/08/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Week_8"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Week 8</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Monday_.2813.2F08.2F12.29"><SPAN class="tocnumber">2.5.1</SPAN><SPAN class="toctext">Monday (13/08/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Tuesday_.2814.2F08.2F12.29"><SPAN class="tocnumber">2.5.2</SPAN><SPAN class="toctext">Tuesday (14/08/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Thursday_.2816.2F08.2F12.29"><SPAN class="tocnumber">2.5.3</SPAN><SPAN class="toctext">Thursday (16/08/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Saturday_.2818.2F08.2F12.29"><SPAN class="tocnumber">2.5.4</SPAN><SPAN class="toctext">Saturday (18/08/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Sunday_.2819.2F08.2F12.29"><SPAN class="tocnumber">2.5.5</SPAN><SPAN class="toctext">Sunday (19/08/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Week_9"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Week 9</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#Monday_.2820.2F08.2F12.29"><SPAN class="tocnumber">2.6.1</SPAN><SPAN class="toctext">Monday (20/08/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Wednesday_.2822.2F08.2F12.29"><SPAN class="tocnumber">2.6.2</SPAN><SPAN class="toctext">Wednesday (22/08/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Week_12"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Week 12</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Wednesday_.2812.2F09.2F12.29"><SPAN class="tocnumber">2.7.1</SPAN><SPAN class="toctext">Wednesday (12/09/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Friday_.2814.2F09.2F12.29"><SPAN class="tocnumber">2.7.2</SPAN><SPAN class="toctext">Friday (14/09/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-29"><A href="#Saturday_.2815.2F09.2F12.29"><SPAN class="tocnumber">2.7.3</SPAN><SPAN class="toctext">Saturday (15/09/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Sunday_.2816.2F09.2F12.29"><SPAN class="tocnumber">2.7.4</SPAN><SPAN class="toctext">Sunday (16/09/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Week_13.3D"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Week 13=</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Monday_.2817.2F09.2F12.29"><SPAN class="tocnumber">2.8.1</SPAN><SPAN class="toctext">Monday (17/09/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-33"><A href="#Tuesday_.2818.2F09.2F12.29"><SPAN class="tocnumber">2.8.2</SPAN><SPAN class="toctext">Tuesday (18/09/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#Wednesday_.2819.2F09.2F12.29"><SPAN class="tocnumber">2.8.3</SPAN><SPAN class="toctext">Wednesday (19/09/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#Thursday_.2820.2F09.2F12.29"><SPAN class="tocnumber">2.8.4</SPAN><SPAN class="toctext">Thursday (20/09/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-36"><A href="#Friday_.2821.2F09.2F12.29"><SPAN class="tocnumber">2.8.5</SPAN><SPAN class="toctext">Friday (21/09/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-37"><A href="#Saturday_.2822.2F09.2F12.29"><SPAN class="tocnumber">2.8.6</SPAN><SPAN class="toctext">Saturday (22/09/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-38"><A href="#Sunday_.2823.2F09.2F12.29"><SPAN class="tocnumber">2.8.7</SPAN><SPAN class="toctext">Sunday (23/09/12)</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Week_14"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">Week 14</SPAN></A></LI><LI class="toclevel-3 tocsection-40"><A href="#Monday_.2824.2F09.2F12.29"><SPAN class="tocnumber">2.9.1</SPAN><SPAN class="toctext">Monday (24/09/12)</SPAN></A></LI><LI class="toclevel-3 tocsection-41"><A href="#Tuesday_.2825.2F09.2F12.29"><SPAN class="tocnumber">2.9.2</SPAN><SPAN class="toctext">Tuesday (25/09/12)</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Judging_Form">Judging Form</SPAN></H1><UL><LI>Please help the judges by filling out this form. Tell them what medal you think you deserve and why. Tell them which special prizes you should win. Help them find your best parts. Show them how you thought about the safety of your project. Helping the judges will help you too.
</LI></UL><UL><LI>Team: Cambridge
</LI><LI>Region: Europe
</LI><LI>iGEM Year:2012
</LI><LI>Track:Foundational Advance
</LI><LI>Project Name:Parts for a reliable and field ready biosensing platform
</LI><LI>Project Abstract: Implementation of biosensors in real world situations has been made difficult by the unpredictable and non-quantified outputs of existing solutions, as well as a lack of appropriate storage, distribution and utilization systems. This leaves a large gap between a simple, functional sensing mechanism and a fully realised product that can be used in the field. We aim to bridge this gap at all points by developing a standardised ratiometric luciferase output in a Bacillus chassis. This output can be linked up with prototyped instrumentation and software for obtaining reliable quantified results. Additionally, we have reduced the specialized requirements for the storage and distribution of our bacteria by using Bacillus' sporulation system. To improve the performance of our biosensing platform we have genetically modified Bacillus’ germination speed. Lastly, we demonstrated the robustness of our system by testing it with a new fluoride riboswitch, providing the opportunity to tackle real life problems. 
</LI></UL><P><A href="https://2012.igem.org/Team:Cambridge" class="external text" rel="nofollow">Back to wiki</A></P><H2><SPAN class="mw-headline" id="iGEM_Medals_for_non-software_teams"> iGEM Medals for non-software teams</SPAN></H2><UL><LI>We believe our team deserves the following medal:
</LI><LI> Bronze
</LI><LI> Silver
</LI><LI>√Gold
</LI></UL><P>Because we met the following criteria (check all that apply and provide details where needed)
</P><H3><SPAN class="mw-headline" id="Requirements_for_a_Bronze_Medal">Requirements for a Bronze Medal</SPAN></H3><UL><LI>√Register the team, have a great summer, and plan to have fun at the Regional Jamboree.
</LI><LI>√Successfully complete and submit this iGEM 2012 Judging form.
</LI><LI>√Create and share a Description of the team's project using the iGEM wiki and the team's parts using the Registry of Standard Biological Parts.
</LI><LI>√Plan to present a Poster and Talk at the iGEM Jamboree.
</LI><LI>√Enter information detailing at least one new standard BioBrick Part or Device in the Registry of Standard Biological Parts. Including:
</LI><LI>√Primary nucleaic acid sequence
</LI><LI>√Description of function
</LI><LI>√Authorship
</LI><LI>Safety notes, if relevant.
</LI><LI>√Acknowedgment of sources and references
</LI></UL><LI>√Submit DNA for at least one new BioBrick Part or Device to the Registry.
</LI><H3><SPAN class="mw-headline" id="Additional_Requirements_for_a_Silver_Medal">Additional Requirements for a Silver Medal</SPAN></H3><UL><LI>√Demonstrate that at least one new BioBrick Part or Device of your own design and construction works as expected; characterize the operation of your new part/device.
</LI><LI>√Enter this information and other documentation on the part's 'Main Page' section of the Registry Part Number(s): <A href="http://partsregistry.org/Part:BBa_K911004" class="external text" rel="nofollow">BBa_K911004</A></LI></UL><H3><SPAN class="mw-headline" id="Additional_Requirements_for_a_Gold_Medal:_.28one_OR_more.29">Additional Requirements for a Gold Medal: (one OR more)</SPAN></H3><UL><LI>Improve an existing BioBrick Part or Device and enter this information back on the Experience Page of the Registry.Part Number(s): None
</LI><LI>√Help another iGEM team by, for example, characterizing a part, debugging a construct, or modeling or simulating their system.Link to this information on your wiki. Page name: <A href="https://2012.igem.org/Team:Cambridge/Outreach/Collaboration" class="external text" rel="nofollow">Team:Cambridge/Outreach/Collaboration</A></LI><LI>√Outline and detail a new approach to an issue of Human Practice in synthetic biology as it relates to your project, such as safety, security, ethics, or ownership, sharing, and innovation.Link to this information on your wiki. Page name: <A href="https://2012.igem.org/Team:Cambridge/HumanPractices/Overview" class="external text" rel="nofollow">Team:Cambridge/HumanPractices/Overview</A>,<A href="https://2012.igem.org/Team:Cambridge/HumanPractices/MarketResearch" class="external text" rel="nofollow">Team:Cambridge/HumanPractices/MarketResearch</A>,<A href="https://2012.igem.org/Team:Cambridge/HumanPractices/FutureDirections" class="external text" rel="nofollow">Team:Cambridge/HumanPractices/FutureDirections</A></LI></UL><P><A href="https://2012.igem.org/Team:Cambridge" class="external text" rel="nofollow">Back to wiki</A></P><H2><SPAN class="mw-headline" id="iGEM_Prizes">iGEM Prizes</SPAN></H2><P>All teams are eligible for special prizes at the Jamborees. <A href="https://2012.igem.org/Judging'" class="external text" rel="nofollow">more...</A> To help the judges, please indicate if you feel you should be evaluated for any of the following special prizes:
</P><UL><LI>√Best Human Practice Advance
</LI><LI>√Best Experimental Measurement
</LI><LI>Best Model
</LI></UL><H3><SPAN class="mw-headline" id="Please_explain_briefly_why_you_should_receive_any_of_these_special_prizes:">Please explain briefly why you should receive any of these special prizes:</SPAN></H3><P><B>Best Human Practice Advance:</B></P><P>We feel that we deserve this prize for three reasons: 
</P><OL><LI> We explored the impacts, *both positive and negative*, of synthetic biology as a solution to real world problems, through interviewing professionals working in a relevant field, namely the impact of arsenic water contamination in Bangladesh. 
</LI><LI> We recognized existing problems with the way the current direction of synthetic. On going through the registry we found that most of the characterization data for biosensing parts is often neither comparable nor replicable. We have worked to solve this issue, for example with our ratiometric dual channel output. 
</LI><LI> *Our project doesn’t stop here*, in Chanel number 6 (<A href="https://2012.igem.org/Team:Cambridge/HumanPractices/FutureDirections" class="external text" rel="nofollow">Team:Cambridge/HumanPractices/FutureDirections</A>) we considered the future implications and technological applications of our project, as well as the means by which it could be improved by subsequent users. We feel that the end to an iGEM project should not be the conclusion of an idea, but the start of it.
</LI></OL><P><B>Best BioBrick Measurement Approach:</B></P><P>It is absolutely vital that a quantitative, numerical, robust, and flexible measurement approach exists to relay information to a user that is an accurate representation of the input processed by a biological device. Working from these principles, the following was done:
</P><OL><LI> We designed and built Biologger, a *cheap, arduino-based, fully functional automatic rotary device* that has an incorporated ratiolumnometer
</LI><LI> Our project is entirely open-sourced and open-platform. We have published source code for the two applications which serve to operate the device, one for PCs and the other for Android devices, as well as the open source circuit design that provides this ratiometric reading. Furthermore, the Android app is able to receive its data wirelessly, which we feel is a great advance in BioBrick measurement. 
</LI><LI> Our dual-channel luciferase reporter was successfully tested with a dilution series of E.coli transformed with the Lux Operon (under pBAD) biobrick (Part  BBa_K325909) of the Cambridge iGEM 2010 team. It can detect, with good accuracy, both different light intensities, as well as the percentages of blue or orange frequencies in a sample. 
</LI><LI> Our device was successfully tested using artificial light to detect different frequencies (colours) as well.
</LI></OL><P>Having done all the above, we believe that this fully open-sourced instrumentation kit (mechanical) chassis, electronics, software code), estimated at *$35.00* (or $85.00 if a Bluetooth modem is required), is a complete BioBrick measurement solution for any and all BioBricks with a light output.
</P><P><A href="https://2012.igem.org/Team:Cambridge" class="external text" rel="nofollow">Back to wiki</A></P><H2><SPAN class="mw-headline" id="Team_Parts">Team_Parts</SPAN></H2><P>To help the judges evaluate your parts, please identify 3 of your parts that you feel are best documented and are of the highest quality. 
</P><UL><LI><DL><DT>Best new BioBrick part (natural)</DT><DD><A href="http://partsregistry.org/Part:BBa_K911003" class="external text" rel="nofollow">BBa_K911003</A></DD><DT>Best new BioBrick part (engineered)</DT><DD><A href="http://partsregistry.org/Part:BBa_K911004" class="external text" rel="nofollow">BBa_K911004</A></DD></DL></LI><LI>Best improved part(s): None
</LI></UL><P>List any other parts you would like the judges to examine:<A href="http://partsregistry.org/Part:BBa_K911001" class="external text" rel="nofollow">BBa_K911001</A>, <A href="http://partsregistry.org/Part:BBa_K911008" class="external text" rel="nofollow">BBa_K911009</A>, <A href="http://partsregistry.org/Part:BBa_K911008" class="external text" rel="nofollow">BBa_K911008</A></P><P>Please explain briefly why the judges should examine these other parts:
</P><UL><LI><B>Magnesium Sensitive Riboswitch <A href="http://partsregistry.org/Part:BBa_K911001" class="external text" rel="nofollow">BBa_K911001</A></B>As a riboswitch sensing construct, this part is an entirely new type of biosensor (along with the fluoride construct) that could potentially change the way we think about designing input genetic circuits. Unlike the fluoride riboswitch, it is a derepression system and therefore serves to demonstrate the principle that riboswitches can be used regardless of whether they turn on or off their reporter.
</LI><LI><B>Fluorescent ratiometric construct for standardizing promoter output <A href="http://partsregistry.org/Part:BBa_K911009" class="external text" rel="nofollow">BBa_K911009</A></B>Fluorescence is a major cornerstone for biosensors in the registry, however, most parts do not involve the use of a ratiometric output, which has been shown in the literature to provide much more reliable and meaningful data. This part not only furthers the development of ratiometric measurements in molecular biology but due to the choice of promoters and terminators it can be used to characterize the difference in activity between E. coli and B. Subtilis
</LI><LI><B>Fast Germination (B. subtilis) <A href="http://partsregistry.org/Part:BBa_K911008" class="external text" rel="nofollow">BBa_K911008</A></B>This part is entirely novel for the registry and fully utilizes the recombination machinery inherent in the Bacillus chassis. Have spores that can germinate at a faster rate is certainly a worthy achievement and could help with experiments with B. Subtilis that any future iGEM teams may wish to perform.
</LI></UL><P><A href="https://2012.igem.org/Team:Cambridge" class="external text" rel="nofollow">Back to wiki</A></P><H2><SPAN class="mw-headline" id="iGEM_Safety">iGEM Safety</SPAN></H2><P>For iGEM 2012 teams are asked to detail how they approached any issues of biological safety associated with their projects.
</P><P>The iGEM judges expect that you have answered the four safety questions <A href="https://2012.igem.org/Safety" class="external text" rel="nofollow">Safety page</A> on your iGEM 2012 wiki.
</P><P>Please provide the link to that page:
Page name: <A href="https://2012.igem.org/Team:Cambridge/Safety" class="external text" rel="nofollow">Team:Cambridge/Safety</A></P><H2><SPAN class="mw-headline" id="Attribution_and_Contributions">Attribution and Contributions</SPAN></H2><P>For iGEM 2012 the description of each project must clearly attribute work done by the team and distinguish it from work done by others, including the host labs, advisors, and instructors.
</P><P>Please provide the link to that page, or comments in the box below:
Page name: <A href="https://2012.igem.org/Team:Cambridge/Attributions" class="external text" rel="nofollow">Team:Cambridge/Attributions</A></P><H2><SPAN class="mw-headline" id="Comments">Comments</SPAN></H2><P>If there is any other information about your project you would like to highlight for the judges, please provide a link to your wiki page here: <A href="https://2012.igem.org/Team:Cambridge/Overview/DesignProcess" class="external text" rel="nofollow">Team:Cambridge/Overview/DesignProcess</A></P></DIV></DIV><DIV id="template_content"><H1><SPAN class="mw-headline" id="Ratiometrica_Labbook"> Ratiometrica Labbook </SPAN></H1><H2><SPAN class="mw-headline" id="Week_4">Week 4 </SPAN></H2><H3><SPAN class="mw-headline" id="Monday_.2816.2F07.2F12.29">Monday (16/07/12)</SPAN></H3><P><B>Ratiometrica- Lux: Transformation of bacillus and e.coli with lux genes from 2010</B></P><UL><LI>Arabinose induction: from <A href="https://2010.igem.org/Team:Cambridge" class="external text" rel="nofollow">Cambridge 2010 iGEM Team's</A> data, the arabinose concentration which will give the optimum amount of light is around 3mM. We hence added 0.45mg arabinose to each mL of LB.
</LI><LI>Light is detected after around 5 hours in the transformed E. coli
</LI><LI>B. subtilis plates have no growth, very possibly due to plasmid backbone pSB1C3 not being compatible in B. subtilis
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_.2818.2F07.2F12.29">Wednesday (18/07/12)</SPAN></H3><P><B>Ratiometrica-Lux: Transformation of e.coli with lux genes from 2010</B></P><UL><LI>Progress: (1) Streaked 8 25ug/ml Chloramphenicol plates with transformed e.coli and incubate overnight; (2) inoculate individual colonies from the plates in liquid medium; (3) Incubate overnight at 37 degrees
</LI></UL><H3><SPAN class="mw-headline" id="Friday_.2820.2F07.2F12.29">Friday (20/07/12)</SPAN></H3><P><B>Ratiometrica-Lux: Transformation of e.coli with lux genes from 2010</B></P><UL><LI>Arabinose induction: Using the same 3mM arabinose prepared earlier in the week, the E. coli were spun down and the arabinose was added. After 5 hours, light is observed.
</LI><LI>Testing the Arduino kit: the testing took place in the dark room, and the cell culture (in 1.5mL Eppendorf tubes) was held close to the photosensor and then taken away repeatedly. The data recorded was plotted with Matlab, as shown:
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption">Light level against time</DIV></DIV></DIV><UL><LI>It should be noted that the background light levels were falling linearly, possibly due to negative feedback on the photosensor, but there is a consistent 5-6% increase of the original base reading upon placing the cells close the the photosensor.
</LI></UL><H2><SPAN class="mw-headline" id="Week_5">Week 5</SPAN></H2><H3><SPAN class="mw-headline" id="Thursday_.2826.2F07.2F12.29">Thursday (26/07/12)</SPAN></H3><P><B>Ratiometrica: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">PCR of vectors, CFP, YFP, Promotor, Terminator and mOrange</A></B></P><UL><LI>Plasmid PJS 130 used as our backbone for isolating the magnesium riboswitch and constructing the fluorescent ratiometric plasmid. 
</LI></UL><UL><LI>Parts from registry amplified: E0020 (CFP), B0015 (Terminator), K143083 (Pveg) and E0030 (YFP).
</LI></UL><UL><LI>mOrange from lab also used
</LI></UL><UL><LI>Primers:
</LI></UL><DL><DD><UL><LI>E0020
</LI></UL></DD></DL><DL><DD><UL><LI>Forward: cacaattaaaggaggaattcaaa|ATGGTGAGCAAGGGC
</LI></UL></DD></DL><DL><DD><UL><LI>Reverse: tcgttttatttgatgcctgg|TTATTACTTGTACAGCTCGTCCA
</LI></UL></DD></DL><DL><DD><UL><LI>B0015
</LI></UL></DD></DL><DL><DD><UL><LI>Forward: acgagctgtacaagtaataa|CCAGGCATCAAATAAAACGA
</LI></UL></DD></DL><DL><DD><UL><LI>Reverse: aaaattattttgacaaaatt|TATAAACGCAGAAAGGCCC
</LI></UL></DD></DL><DL><DD><UL><LI>K143083
</LI></UL></DD></DL><DL><DD><UL><LI>Forward: tgggcctttctgcgtttata|AATTTTGTCAAAATAATTTTATTG
</LI></UL></DD></DL><DL><DD><UL><LI>Reverse: tcctcgcccttgctcaccat|ctagta|TTCACCACCTTTCTCTAGTAACA
</LI></UL></DD></DL><DL><DD><UL><LI>E0030
</LI></UL></DD></DL><DL><DD><UL><LI>Forward: tgttactagagaaaggtggtgaa|tactag|ATGGTGAGCAAGGGCG
</LI></UL></DD></DL><DL><DD><UL><LI>Reverse: gatgcctggctctagtatca|TTATTACTTGTACAGCTCGTCCA
</LI></UL></DD></DL><DL><DD><UL><LI>mOrange
</LI></UL></DD></DL><DL><DD><UL><LI>Forward: accaaaaaggaatagagt|ATGGTGAGCAAGGGCG
</LI></UL></DD></DL><DL><DD><UL><LI>Reverse: caaacttcat|ACTTCCTCCTCCTCCACTTCCTCCTCCTCC|cttgtaagctcgtccatgc
</LI></UL></DD></DL><DL><DD><UL><LI>pJS130 Mg2+
</LI></UL></DD></DL><DL><DD><UL><LI>Forward: tagaggaggtacgagtcccg|ATGAAACCAGTAACGTTATACGA
</LI></UL></DD></DL><DL><DD><UL><LI>Reverse: tacatcacaattacggaaca|CAAAATCGTCTCCCTCC
</LI></UL></DD></DL><DL><DD><UL><LI>pJS130 Fluorescent
</LI></UL></DD></DL><DL><DD><UL><LI>Forward: acgagctgtacaagtaataa|TGATACTAGAGCCAGGCATC
</LI></UL></DD></DL><DL><DD><UL><LI>Reverse: tcctcgcccttgctcaccat|TTTGAATTCCTCCTTTAATT
</LI></UL></DD></DL><UL><LI>PCR settings:
</LI></UL><DL><DD><UL><LI> 95 °C - 6mins
</LI></UL></DD></DL><DL><DD><UL><LI> 98 °C - 10secs
</LI></UL></DD></DL><DL><DD><UL><LI> 58 °C - 45secs
</LI></UL></DD></DL><DL><DD><UL><LI> 72 °C - 180secs
</LI></UL></DD></DL><DL><DD><UL><LI> Repeat above 35x
</LI></UL></DD></DL><DL><DD><UL><LI> 72 °C - 5mins
</LI></UL></DD></DL><DL><DD><UL><LI> 25 °C - 1min
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Friday_.2827.2F07.2F12.29">Friday (27/07/12)</SPAN></H3><P><B>Ribosense &amp; Ratiometrica: <A href="/Team:Cambridge/Protocols/GelElectrophoresis" title="Team:Cambridge/Protocols/GelElectrophoresis">Gel electrophoresis of PCR products</A></B></P><TABLE cellspacing="20"><TBODY><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Vector amplification gel. Lanes 2-4 Lux containing vector. Lanes 5-7 Fluorescent construct vector. No products from this gel were amplified.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Various. Lanes 3-5 mOrange amplification. Lanes 6-8 eCFP (E0020) amplification. mOrange amplification did not work.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Small products for fluorescent construct. Lanes 1-3 Terminator (B0015). Lanes 4-6 Promotor pVEG + SpoVG (K143053).</DIV></DIV></DIV></TD></TR><TR><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Various. Lanes 2-4 eYFP (E0030) amplification. Lanes 5-7 Mg2+ vector +8 codons. Lane 8 Mg2+ vector -8 codons. Lane 8 amplification did not work.</DIV></DIV></DIV></TD><TD><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Mg2+ riboswitch gel. Lanes 6+7 positive and negative controls respectively for entire run. Lanes 3-5 genomic riboswitch amplification -8 codons. Lanes 2+3 vector -8 codons.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><UL><LI>PCR fragments for Mg2+ riboswitch, luxA/mOrange fusion and fluorescent construct from yesterday separated on gels. 90mins at 100V.
</LI></UL><UL><LI>Mostly successful, but will need to repeat four runs: Three vectors (fluorescent, fusion (lux) and Mg2+ riboswitch (-8 codons) and mOrange gene.
</LI></UL><P><B>Ribosense &amp; Ratiometrica: <A href="/Team:Cambridge/Protocols/GelExtractionofDNA" title="Team:Cambridge/Protocols/GelExtractionofDNA">Purification of successfully amplified DNA from gel</A></B></P><UL><LI>Successful CFP, YFP, B0015, Pveg + SpoVG (K143053), riboswitch and vector DNA extracted from gel and purified using a minelute column. DNA frozen for later Gibson assembly.
</LI></UL><H2><SPAN class="mw-headline" id="Week_6">Week 6</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_.2830.2F07.2F12.29">Monday (30/07/12)</SPAN></H3><P><B>Ratiometrica &amp; Ribosense: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">PCR of vectors and mOrange</A></B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">mOrange PCR run at multiple temperatures, every two lanes increacing the temperature of the annealing step by 2 °C. No lanes worked.</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Top: Fusion, lux containing vector gel. Lanes 2-4, Vector DNA. Lane 5, +ve control. Lane 6, -ve control. Bottom: Vector gel. Lanes 1-3, Fluorescent contruct vector DNA. Lanes 4-6, Riboswitch construct vector DNA.</DIV></DIV></DIV><UL><LI>Not all products were obtained during Friday's PCR. Most of these missing products were large vector backbones. They are being run again, with a much longer extension time of 300s. If that fails, primers will be ordered to split the vectors into manageable chunks, and the PCR reattempted when they arrive.
</LI></UL><UL><LI>PCR cycle x35: 
</LI></UL><DL><DD><UL><LI> 15s Denaturing at 95 C
</LI><LI> 45s Annealing at 60 C
</LI><LI> 300s Extension at 72 C
</LI></UL></DD></DL><UL><LI>Remaining stray product had a slightly tricky secondary structure at the 3' end. It will be run at a series of annealing temperatures in a PCR machine capable of a temperature gradient.
</LI></UL><UL><LI>mOrange PCR run at many different temperatures, from 62 °C to 76 °C. However this doesn't seem to solve our problem (refer to Gel photos). It seems likely that it is a primer design problem.
</LI></UL><P><B>Ratiometrica: <A href="/Team:Cambridge/Protocols/GelElectrophoresis" title="Team:Cambridge/Protocols/GelElectrophoresis">Separation of vector and mOrange DNA</A></B></P><UL><LI>Positive control produced no band. No primer smear - primers may not be in mix for some reason.
</LI></UL><UL><LI>Realized correct lux vector template was not added during PCR preparation, consequently no amplification occured. Still appears to be a primer smear.
</LI></UL><UL><LI>Fluorescent vector backbone produced several bands. Appears to be due to mis-priming during PCR. Either changing the primers or raising the annealing temperature should solve this problem, but may mean that we have to do this PCR separately.
</LI></UL><UL><LI>Riboswitch vector also failed. However, the extraction from the previous PCR run may have worked. We will try producing a functional plasmid with this extraction before running this PCR again. 
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_.2831.2F07.2F12.29">Tuesday (31/07/12)</SPAN></H3><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">PCR of split fluorescent construct</A></B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Fluorescent construct gels. Top: Lanes 2-4: Fragment A - expected size 4500bp. Bottom: Lanes 2-4: Fragment B - expected size 3000bp. Lane 5: negative control. All lanes worked well.</DIV></DIV></DIV><UL><LI>Our present theory is that the vectors that we are trying to amplify is too large for the PCR to effectively take place. Given this, we are using split primers to try to reduce the size of the fragments (kindly provided by <A href="/Team:Cambridge/Special_thanks" title="Team:Cambridge/Special thanks">P.J.Steiner</A>).
</LI></UL><UL><LI>New primer sequences:
</LI></UL><DL><DD><UL><LI>Forward: tgaagtgttcgacaatataaatgtg|CGAAACGATCCTCATCCTGT
</LI></UL></DD></DL><DL><DD><UL><LI>Reverse: ACAGGATGAGGATCGTTTCG|cacatttatattgtcgaacacttca
</LI></UL></DD></DL><UL><LI>PCR Programme
</LI></UL><UL><LI>After gel electrophorsis, it was found that the PCR had worked perfectly. Given we now have all the fragments for the fluorescent construct, we will try to make it with Gibson ASAP.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_.2803.2F08.2F12.29">Friday (03/08/12)</SPAN></H3><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/Gibsonassembly" title="Team:Cambridge/Protocols/Gibsonassembly">Construction of fluorescent plasmid with Gibson Assembly</A></B></P><UL><LI><B>Isothermal reaction buffer</B> *5 remade with correct composition. Needed to order NAD+, so used some master mix from Haseloff lab.
</LI></UL><UL><LI>Various Gibson tail tagged DNA fragments produced over last few weeks assembled together using Gibson protocol:
</LI></UL><DL><DD><UL><LI>pJS130 plasmid fragment A
</LI></UL></DD></DL><DL><DD><UL><LI>pJS130 plasmid fragment B
</LI></UL></DD></DL><DL><DD><UL><LI>CFP (E0020)
</LI></UL></DD></DL><DL><DD><UL><LI>Terminator (B0015)
</LI></UL></DD></DL><DL><DD><UL><LI>pVEG + RBS (K143053)
</LI></UL></DD></DL><DL><DD><UL><LI>YFP (E0030)
</LI></UL></DD></DL><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/TransformationofE.coli" title="Team:Cambridge/Protocols/TransformationofE.coli">Transformation of TOP10 e.coli with Gibson products</A></B></P><UL><LI>3 tubes of TOP10 transformed with Gibson products. Cells plated out on 100μg/ml ampicillin plates.
</LI></UL><UL><LI>No positive control added. If no cells grow, will add positive control to all future experiments.
</LI></UL><UL><LI>Results: no growth (04/08/12)
</LI></UL><P><B>Ribosense: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">PCR of split Mg2+ vector</A></B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">results of split magnesium vector PCR. None of the products worked</DIV></DIV></DIV><UL><LI>PCR of magnesium riboswitch vector repeated with primers to split plasmid.
</LI></UL><UL><LI>Lanes 2 + 3: Fragment A (center - cut site (promotor side))
</LI></UL><UL><LI>Lanes 4 + 5: Fragment B (without 8 codon substitution) (cut site (lac I side) - center)
</LI></UL><UL><LI>Lanes 6 + 7: Fragment B (with 8 codon substitution) (cut stie (lac I side) - center)
</LI></UL><UL><LI>Gels run, found PCR was unsuccessful.
</LI></UL><P><B>Ratiometrica-Lux: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">PCR of mOrange</A></B></P><UL><LI>with the forward primer correctly labelled and the single-base deletion in the reversed primer fixed, we were able to obtain the mOrange fragment
</LI></UL><UL><LI>PCR Programme: 
</LI></UL><UL><LI>Result: bands of the correct size (Lanes 7-8)
</LI></UL><H2><SPAN class="mw-headline" id="Week_7">Week 7</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_.2806.2F08.2F12.29">Monday (06/08/12)</SPAN></H3><P><B>Ratiometrica-Lux: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">Split PCR of Lux vector</A></B></P><UL><LI>Primers for spliting pSB1C3 arrived and were used to PCR the Ratiometrica-Lux vector
</LI></UL><DL><DD><UL><LI>Fragment A: Vec FWD and Vec split RVS (expected size: 5kb)
</LI></UL></DD></DL><DL><DD><UL><LI>Fragment b: Vec split FWD and Vec RVS (expected size: 4.6kb)
</LI></UL></DD></DL><UL><LI>PCR Programme:
</LI></UL><UL><LI>Results: other than one very ambiguous band (lane 6) which might be of the right size, the others did not work.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_.2807.2F08.2F12.29">Tuesday (07/08/12)</SPAN></H3><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/Gibsonassembly" title="Team:Cambridge/Protocols/Gibsonassembly">Gibson assembly of Fluorescent construct</A></B></P><UL><LI>6-piece Gibson assembly done in triplicate
</LI></UL><DL><DD><UL><LI> Reaction 1: Tube 1 (Fragment A), Tube 4 (Fragment B), Tube 10 (CFP), Tube 13 (B0015), Tube 16 (K143053), Tube 19 (YFP)
</LI></UL></DD></DL><DL><DD><UL><LI> Reaction 2: Tube 2 (Fragment A), Tube 5 (Fragment B), Tube 11 (CFP), Tube 14 (B0015), Tube 17 (K143053), Tube 20 (YFP)
</LI></UL></DD></DL><DL><DD><UL><LI> Reaction 3: Tube 3 (Fragment A), Tube 6 (Fragment B), Tube 12 (CFP), Tube 15 (B0015), Tube 18 (K143053), Tube 21 (YFP)
</LI></UL></DD></DL><P><B>Ratiometrica-Lux: <A href="/Team:Cambridge/Protocols/PCRcolony" title="Team:Cambridge/Protocols/PCRcolony">colony PCR of Lux vector</A></B></P><UL><LI>2nd attempt at amplifying the Lux vector in two fragments, similar to one attempted yesterday
</LI></UL><UL><LI>Results: no bands of the correct size. There seems to be DNA in the well which indicates presence of very large pieces of DNA.
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_.2808.2F08.2F12.29">Wednesday (08/08/12)</SPAN></H3><P><B>Ratiometrica &amp; Ribosense: <A href="/Team:Cambridge/Protocols/ElectricalTransformation" title="Team:Cambridge/Protocols/ElectricalTransformation">Electrical transformation of competent e.coli with Gibson products</A></B></P><UL><LI>Gibson constructs from yesterday (fluorescent and magnesium riboswitch) transformed into e.coli produced yesterday.
</LI></UL><UL><LI>Cells plated out onto 50 μg/ml ampicillin plates. Put in incubator overnight.
</LI></UL><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/RestrictionDigest" title="Team:Cambridge/Protocols/RestrictionDigest">Restriction Digest of Gibson products</A></B></P><UL><LI>A restriction digest was performed to investigate if the plasmid was actually successfully constructed in the 6-piece Gibson reaction
</LI></UL><DL><DD><UL><LI>Restrictrion Enzymes: SalI (site at 5619), XhoI (site at 8040)
</LI></UL></DD></DL><DL><DD><UL><LI>Restriction Enzyme buffer: NEBuffer 3
</LI></UL></DD></DL><UL><LI>Results: No bands at all; probably because there was too little DNA.
</LI></UL><H2><SPAN class="mw-headline" id="Week_8">Week 8</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_.2813.2F08.2F12.29">Monday (13/08/12)</SPAN></H3><P><B>Ratiometrica-Lux: <A href="/Team:Cambridge/Protocols/MiniPrep" title="Team:Cambridge/Protocols/MiniPrep">Miniprep of LuxBrick plasmid</A></B></P><UL><LI>Miniprepped is done using miniprep kit supplied by Cambio
</LI><LI>Plasmids are miniprepped from 3 cultures
</LI></UL><P><B>Ratiometrica-Lux: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol"> PCR of lux vector</A></B></P><UL><LI>With split primers, similar conditions to last week (except no longer using E. coli colonies), each half in triplicates
</LI><LI>Results: Half of the vector (Fragment B) came out (gel 1), positive control worked (gel 2 lane 5)
</LI></UL><P><B>Ratiometrica-Lux: <A href="/wiki/index.php?title=Team:Cambridge/Protocols/ArabinoseInduction&amp;action=edit&amp;redlink=1" class="new" title="Team:Cambridge/Protocols/ArabinoseInduction (page does not exist)"> Arabinose induction of Lux E. coli</A></B></P><UL><LI>E. coli with the lux plasmid (K325909) O/N liquid cultures are induced with 3mM arabinose
</LI><LI>They did not produce visible bioluminescence after 7 hours. We suspect it to be a problem associated either with the concentration of cells in the culture or the amount of arabinose we added.
</LI></UL><H3><SPAN class="mw-headline" id="Tuesday_.2814.2F08.2F12.29">Tuesday (14/08/12)</SPAN></H3><P><B>Ratiometrica-Lux: <A href="/wiki/index.php?title=Team:Cambridge/Protocols/ArabinoseInduction&amp;action=edit&amp;redlink=1" class="new" title="Team:Cambridge/Protocols/ArabinoseInduction (page does not exist)">Arabinose Induction of Lux E. coli</A></B></P><UL><LI>After our failed liquid culture induction, we made LB agar plates with 3mM arabinose and the appropriate antibiotics.
</LI><LI>Cells are plated out on the arabinose plates and incubated at 37°C overnight.
</LI><LI>Colonies on plates show bioluminescence after overnight incubation
</LI></UL><H3><SPAN class="mw-headline" id="Thursday_.2816.2F08.2F12.29">Thursday (16/08/12)</SPAN></H3><P><B>Ratiometrica-Lux: <A href="/Team:Cambridge/Protocols/RestrictionDigest" title="Team:Cambridge/Protocols/RestrictionDigest"> Restriction Digest of Lux vector</A></B></P><UL><LI>Results: Digest came out exactly as expected.
</LI></UL><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol"> PCR of Flourescent Construct DNA fragments</A></B></P><UL><LI>Redo of <A href="/Team:Cambridge/Lab_book/Week_5#Friday" title="Team:Cambridge/Lab book/Week 5">PCR done at the end of July</A>, as we are running out of fragments to Gibson
</LI></UL><UL><LI>Used existing DNA fragments as template rather than biobricks from the registry
</LI></UL><UL><LI>Results: Only the fluorescent protein fragments (eCFP and eYFP) came out clearly; the smaller fragments (B0015, K143053) might have came out but was mistaken for primer dimers, the vector fragments did not come out
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_.2818.2F08.2F12.29">Saturday (18/08/12)</SPAN></H3><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">2nd attempt at PCR of fragments</A></B></P><UL><LI>Reattempt of pJS130 and smaller fragments
</LI></UL><UL><LI>Used Velocity from Bioline, new mastermix recipe is used
</LI></UL><UL><LI>Results: failed- no wanted bands, since the positive control didn't come up, it is suggested that we need to optimise the PCR protocol for Velocity if we are to use it.
</LI></UL><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/Gibsonassembly" title="Team:Cambridge/Protocols/Gibsonassembly">Gibson assembly of fluorescent construct</A></B></P><UL><LI>Gibson performed using remaining DNA fragments which has been concentrated by dehydration
</LI></UL><UL><LI>We realised that though the decreased T5 exonuclease in the gibson mastermix might have lowered the Gibson efficiency in the positive control, the fragments we are trying to assemble are much smaller (~200bp); papers said this is likely to cause trouble
</LI></UL><UL><LI>We therefore attempt 2 solutions in parallel: (1) adding the T5 only when the reaction is already heated to 50°C (using a water bath); (2) using 1/5 of the amount of T5 exo (in both in parallel)
</LI></UL><UL><LI>These are all done in duplicates, resulting in 8 Gibson reactions and subsequent transformations
</LI></UL><UL><LI>Results: (Sunday) There are 2 colonies on one of the plates without either treatment
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_.2819.2F08.2F12.29">Sunday (19/08/12)</SPAN></H3><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">3rd attempt at PCR of fragments</A></B></P><UL><LI>Similar PCR to the one from yesterday, except Velocity mastermix is further changed, and extension time is changed
</LI></UL><UL><LI>Results: failed- no bands, probably Velocity mastermix problem, will revert back to Phusion on next attempt
</LI></UL><H2><SPAN class="mw-headline" id="Week_9">Week 9</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_.2820.2F08.2F12.29">Monday (20/08/12)</SPAN></H3><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">4th attempt at PCR of fragments</A></B></P><UL><LI>Only vectors this time as we are still not sure if we have got the Velocity mastermix recipe right, as we are waiting for our orders of Phusion to arrive
</LI></UL><UL><LI>Results: no bands for any lanes including positive control
</LI></UL><P><B>Ratiometrica-Flu: Restreak of potential Ratiometrica colonies</B></P><UL><LI>One of the 2 colonies from the Gibson reaction and subsequent transformation are restreaked on a Amp 100ug/ml plate
</LI></UL><UL><LI>this is incubated 37°C overnight
</LI></UL><H3><SPAN class="mw-headline" id="Wednesday_.2822.2F08.2F12.29">Wednesday (22/08/12)</SPAN></H3><P><B>Ratiometrica-Lux: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">PCR of lux-vector</A></B></P><UL><LI>Subsplit primers for the other half of the fragment that did not come out last week arrived, also new, extended lux FWD primers as we were afraid that the original primers did not anneal properly
</LI></UL><UL><LI>We tried combinations of the new and old primers together with the subsplit and split primers
</LI></UL><UL><LI>Results: only Fragment A2 (Between subsplit FWD and split REV) worked
</LI></UL><P><B>Ratiometrica-Flu: <A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">5th attempt of PCR of fragments</A></B></P><UL><LI>Since we got our PCR protocol working, we reattempted the PCR
</LI></UL><UL><LI>Results: the smallest fragments (B0015, K143053) came out, gel extraction is done with the new concentrated protocol
</LI></UL><H2><SPAN class="mw-headline" id="Week_12"> Week 12</SPAN></H2><H3><SPAN class="mw-headline" id="Wednesday_.2812.2F09.2F12.29">Wednesday (12/09/12)</SPAN></H3><P><B>Ratiometrica - lux</B></P><UL><LI>DNA 2.0 construct arrived
</LI></UL><UL><LI>Colonies restreaked and put into 30 °C incubator.
</LI></UL><UL><LI>Only one set of colonies appears to be fluorescing, which may be of concern.
</LI></UL><H3><SPAN class="mw-headline" id="Friday_.2814.2F09.2F12.29">Friday (14/09/12)</SPAN></H3><P><B><A href="/Team:Cambridge/Protocols/PCRProtocol" title="Team:Cambridge/Protocols/PCRProtocol">PCR of α and β fragments for ratiometrica</A></B></P><UL><LI>Although ratiometrica appears to have produced successful colonies, we would still like some extra fragments in case we need to repeat our construction.
</LI></UL><UL><LI>Previous successful gel extractions of α and β used as template for PCR.
</LI></UL><UL><LI>α has successfuly been amplified, but &amp;beta produced band of the wrong size.
</LI></UL><P><B><A href="/Team:Cambridge/Protocols/GelExtractionofDNA" title="Team:Cambridge/Protocols/GelExtractionofDNA">Gel extraction of α</A></B></P><UL><LI>α successfully extracted from gel.
</LI></UL><H3><SPAN class="mw-headline" id="Saturday_.2815.2F09.2F12.29">Saturday (15/09/12)</SPAN></H3><P><B>Ratiometrica - lux</B></P><UL><LI>Attempted digestion of DNA 2.0 pure plasmid with EcoRI and SpeI, expected fragments 2648bp and 8738bp.
</LI><LI>Parallel digestion of pSB1C3 backbone ready for ligation
</LI><LI>8738 band extracted, was a little weak
</LI><LI>8738 band and digested backbone ligated together
</LI><LI>Chemically competent <I>E. coli</I> used for transformation
</LI><LI>Plated on LB Agar with 25μg/ml Chlor
</LI></UL><H3><SPAN class="mw-headline" id="Sunday_.2816.2F09.2F12.29">Sunday (16/09/12)</SPAN></H3><P><B>Ratiometrica - lux</B></P><UL><LI>Picked colonies from ligation attempt from yesterday, some red colonies on plates, these were avoided with one exception as a control.
</LI><LI>Picked colonies grown in LB with 25μg/ml Chlor
</LI></UL><H2><SPAN class="mw-headline" id="Week_13.3D">Week 13=</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_.2817.2F09.2F12.29">Monday (17/09/12)</SPAN></H3><P><B>Ratiometrica - Lux </B></P><P>Testing of the 2.0 construct:
</P><P>Numbers 33,35,36 have correct sequence as of being sent to us. However they were not visibly luminescent. *34 was.
</P><P>Overnight cultures of each of the 6 culture lines sent were set up. Additionally, inducing plates were streaked, including the 2010 lux biobrick as a luminescence control and pjs130 as an iptg induction control.
</P><P>Primers ordered to ligate 2.0 construct into pjs130 for insertion into bacillus.
</P><H3><SPAN class="mw-headline" id="Tuesday_.2818.2F09.2F12.29">Tuesday (18/09/12)</SPAN></H3><P><B>Ratiometrica - Lux </B></P><P>None of restreaked plates visibly luminescent, aside from *34 (faint), even under photon counting camera. Pjs130 not green, so iptg induction potentially didn't work. LuxBB induced and luminescent.
</P><P>Cultures induced with 1mM iptg or 3mM arabinose for luxBB, imaged after 3 hrs. Some extremely faint luminescence for 33,34,38 on photon counting camera.
</P><P>Cultures miniprepped and restriction digests carried out w/spa1. Results are shown.  Results indicate problem with the miniprep and/or damage to the construct - expected bands at 6625, 2838, 1876 and 53 bp.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Digests are of 33-38, each miniprep, cut and uncut, between 2 ladders.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Wednesday_.2819.2F09.2F12.29">Wednesday (19/09/12)</SPAN></H3><P><B> Ratiometrica - Lux </B></P>
Digest of 2.0 miniprep with different enzyme, hindIII, carried out. 15ul template DNA used in a 20ul reaction. Again, results were odd. Cut and uncut minipreps ran.<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">HindIII digest</DIV></DIV></DIV></DIV><P>Cultures of ratiometrica (fluorescent) were miniprepped to be transformed into bacillus. A restriction digest was performed on these minipreps, and produced bands of the correct size.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Ratiometrica Digest</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Ratiometrica Digest Prediction</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Thursday_.2820.2F09.2F12.29">Thursday (20/09/12)</SPAN></H3><P><B>Ratiometrica - Lux</B></P><P>Transformation of fluorescent ratiometric construct into bacillus was unsuccessful
</P><P>Transformations of XL1-blue cells with resuspended construct plasmid from DNA 2.0 yielded orange colonies, with some white, and some showing sectoring, indicating insert loss. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">An image showing the variation between fresh transformants. Note sectoring of individual colonies, indicating that the insert is being lost as the colonies grow.</DIV></DIV></DIV></DIV><P>Plates all appear to be luminescent. A random selection of colonies was picked and patched in order to investigate whether luminescence cosegregated with orange colour.
</P><H3><SPAN class="mw-headline" id="Friday_.2821.2F09.2F12.29">Friday (21/09/12)</SPAN></H3><P>Initial pick and patch experiments showed cosegregation, but we did not have immediate access to a photon-counting camera or a DSLR capable of long exposures, so we did not obtain images. Representative colonies were streaked out onto a fresh plate to be imaged later.
</P><H3><SPAN class="mw-headline" id="Saturday_.2822.2F09.2F12.29">Saturday (22/09/12)</SPAN></H3><P>A PCR was run to amplify several fragments, including:
</P><UL><LI> The ratiometric luciferase construct, to insert into;
</LI><LI> pSB3C5, a low copy number vector
</LI><LI> Oli's Mg riboswitch construct
</LI></UL><P>All fragments were obtained.
</P><H3><SPAN class="mw-headline" id="Sunday_.2823.2F09.2F12.29">Sunday (23/09/12)</SPAN></H3><P>Fragments for the fluoride riboswitch and the insertion of the 2.0 construct into pSB3C5 were all digested for two hours in 50 ul reactions, as was the pSB1C3 backbone.
</P><UL><LI>5ul NEB2
</LI><LI>1 ul EcoRI
</LI><LI>1 ul PstI
</LI><LI>0.5 ul BSA
</LI><LI>10 ul DNA
</LI><LI>32.5 ul HPLC H20
</LI></UL><P>Enzyme was heat-inactivated for 20 minutes at 65 C
</P><P>for a 6:1 insert:vector ratio (recommended by Knight on openwetware) 2.5 ul of vector digest was ligated to 15 ul insert in all cases, except the luciferase construct. In this case the insert is bigger than the vector, so the ratio was reversed. 
</P><UL><LI>3ul vector
</LI><LI>18ul insert
</LI><LI>2.5ul 10X T4 ligase buffer
</LI><LI>1.25 ul T4 ligase
</LI></UL><P>Ligation left overnight at 16C, as recommended by Knight.
</P><H2><SPAN class="mw-headline" id="Week_14">Week 14</SPAN></H2><H3><SPAN class="mw-headline" id="Monday_.2824.2F09.2F12.29">Monday (24/09/12)</SPAN></H3><P><B>Ratiometrica - Lux</B></P>
Colonies grew. However colony PCR using the sequencing primers revealed them to be transformed with recircularised backbone. <DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">An amplicon of ~300 bp indicates no insert.</DIV></DIV></DIV></DIV><P>The digestions and ligations were reattempted, this time treating the backbone with alkaline phosphatase in order to reduce or eliminate self-ligation.
</P><H3><SPAN class="mw-headline" id="Tuesday_.2825.2F09.2F12.29">Tuesday (25/09/12)</SPAN></H3><P><B>Ratiometrica - Lux </B></P><P>Images were taken to show cosegregation of orange colour and luminescence. Additionally, images were taken of the luciferase construct with a gel filter in order to indicate if there was a difference in the emission spectra.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Representative colours of different luciferase construct transformants</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">And a long exposure showing luminescence</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Potentially a difference in transmission here, perhaps reflecting a different emission spectrum?</DIV></DIV></DIV></DIV><P>See results page for more details and interpretation.
</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Cambridge/Ratiometrica/Labbook">http://2012.igem.org/Team:Cambridge/Ratiometrica/Labbook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Ratiometrica/Labbook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Ratiometrica/Labbook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Ratiometrica/Labbook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Ratiometrica/Labbook&amp;oldid=296794" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>