<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_Israel_Project_Bacillus skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion Israel/Project/Bacillus</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team: Technion 2015</TITLE></P><DIV class="nav_menu"><DIV class="main_buttons"><DIV id="project" class="main_button">Project</DIV><DIV id="Lab_Notebook" class="main_button">Lab Notebook</DIV><DIV id="Human_Practices" class="main_button">Human Practices</DIV><DIV id="Judging_Criteria" class="main_button">Judging Criteria</DIV><DIV id="Team" class="main_button">Team</DIV></DIV><NAV><DIV id="Team_Links" class="main_links"><A href="https://2015.igem.org/Team:Technion_Israel/Team/About_Us">About Us</A><A href="https://2015.igem.org/Team:Technion_Israel/Attributions">Attributions</A><A href="https://2015.igem.org/Team:Technion_Israel/Team/Sponsors">Sponsors</A><A href="https://2015.igem.org/Team:Technion_Israel/Team/gallery">Gallery</A></DIV><DIV id="Judging_Criteria_Links" class="main_links"><A href="https://2015.igem.org/Team:Technion_Israel/Parts">Parts</A><A href="https://2015.igem.org/Team:Technion_Israel/Judging_Criteria/Achievements">Achievements</A><A href="https://2015.igem.org/Team:Technion_Israel/Collaborations">Collaborations</A><A href="https://2015.igem.org/Team:Technion_Israel/Design">Design</A></DIV><DIV id="Human_Practices_Links" class="main_links"><A href="https://2015.igem.org/Team:Technion_Israel/Practices">Practices</A><A href="https://2015.igem.org/Team:Technion_Israel/Practices/safety">Safety</A></DIV><DIV id="Lab_Notebook_Links" class="main_links"><A href="https://2015.igem.org/Team:Technion_Israel/Lab_Notebook/Overview">Overview</A><A href="https://2015.igem.org/Team:Technion_Israel/Lab_Notebook/Protocols">Protocols</A></DIV><DIV id="project_Links" class="main_links"><A href="https://2015.igem.org/Team:Technion_Israel/Project/Overview">Overview</A><A href="https://2015.igem.org/Team:Technion_Israel/Project/Bacillus">Bacillus</A><A href="https://2015.igem.org/Team:Technion_Israel/Project/E_Coli">E.Coli</A><A href="https://2015.igem.org/Team:Technion_Israel/Project/NADPH">NADPH</A><A href="https://2015.igem.org/Team:Technion_Israel/Project/Comb">Comb</A><A href="https://2015.igem.org/Team:Technion_Israel/Modeling">Modeling</A></DIV></NAV></DIV><DIV class="content"><H2>Introduction</H2><H3><B>Aim</B></H3><P>To express and secrete our target gene, 3α-hydroxysteroid dehydrogenase (3α-HSD), in and from bacteria.  To prove activity of the secreted gene in breaking down dihydrotestosterone (DHT).</P><H3>Why <I>Bacillus subtilis</I>?</H3><P>When considering which type of bacteria to use for our cloning, we researched the natural microbiota of the scalp.  We found that <I>Bacillus subtilis</I> is prevalent on our scalp. <SUP><A href="#fn1" id="ref1">1</A></SUP></P><P><I>B.subtilis</I> is a gram-positive, rod-shaped bacterium found on skin, in the digestive tract, in epithelial wounds, on extremities of the human body, in livestock and in soil. <SUP><A href="#fn2" id="ref2">2</A></SUP></P><P>Today, the bacterium is used commercially in skin care products, foods for human consumption, antibiotic substitutes, etc. (2)</P><P><I>B.subtilis</I> has also the ability to secrete various proteins. </P><P>For those reasons, we chose <I>B.subtilis</I> as our chassis organism for excreting the 3α-HSD enzyme of interest.</P><P>In our lab, we used strain PY79 wild type which we received from Dr. Avigdor Eldar’s lab at Tel Aviv University.</P><H2>Expression</H2><P> As stated above, our goal was to express  the 3α-HSD protein, originally from rat liver, and check whether it is able to fold properly and break down DHT.</P><P>For this purpose, we cloned 3α-HSD  into <I>B.subtilis</I> under the inducible P<SUB>hyper-spank</SUB> promoter.</P><P>pDR111 is a shuttle vector for <I>E.coli</I> and <I>B.subtilis</I>. When in <I>E.coli</I>, an ampicillin resistance gene is expressed and when in <I>B.subtilis</I>, a spectinomycin resistance is expressed. The plasmid contains homologous regions with the <I>B. subtilis</I> genome, enabling gene introduction by recombination into the nonessential amyE locus of the chromosomal DNA as a single copy.<SUP><A href="#fn3" id="ref3">3</A></SUP></P><P>The plasmid also contains lacI gene for the inducible promoter P<SUB>hyper-spank</SUB>, making our inserted genes inducible with the addition of IPTG.</P><P>In the lab, we inserted our gene in the multiple cloning site between the NheI and SalI restriction sites, which can be seen in the basic pDR111 plasmid below.</P><P> In order to predict the target gene expression, we cloned mCherry into <I>B.subtilis</I>, since it is easier to track reporter genes. The simple, planned circuit of the gene in the shuttle vector is featured in the image below.</P><H2>Secretion</H2><P>Secretion of the  3α-HSD enzyme was an extremely important component of our project.  The secretion would enable the bacteria to live on the consumer’s scalp, as opposed to being lysed in order to recover the enzyme.  This could reduce the production costs and improve product efficiency over time.</P><P>As mentioned previously, <I>B. subtilis</I>, as well as some other gram-positive bacteria, has protein secretion mechanisms and regularly secretes proteins such as various proteases.</P><P>In order to engineer the bacteria to meet our purposes, we fused our target gene coding for 3α-HSD to the signal peptide (SP) for the gene aprE, which encodes to extracellular alkaline-serine protease (subtilisin E), the most abundant protease secreted to the medium in wildtype <I>B. subtilis</I>. <SUP><A href="#fn4" id="ref4">4</A></SUP></P><P>By doing so, the  3α-HSD  enzyme could be recognized by the secretion system of the protease and could be excreted into the extracellular medium.</P><P> The circuit can be seen below.</P><P>Click here to see our results.</P><SUP id="fn1">1. Roia, F. C.; Vanderwyk, R. W.:Resident Microbial Flora of the Human Scalp and its Relationship to Dandruff. Journal of The Society of Cosmetic Chemists. 1969, 20, 113-134.</SUP><SUP id="fn2">2. Gonzales, D. J.; Haste, N. M.; Hollands, A.; Fleming, T. C.; Hamby, M.; Pogliano, K.; Nizet, V.; Dorrestein, P.C.: Microbial competition between Bacillus subtilis and Staphylococcus aureus monitored by imaging mass spectrometry. Microbiology 2011, 157, 24985-2492.</SUP><SUP id="fn3">3. Härtle, B.; Wehrl, W.; Wiegert, T.; Homuth, G.; Schumann, W.: Development of a New Integration Site within the Bacillus subtilis Chromosome and Construction of Compatible Expression Cassettes. Journal of Bacteriology. 2001, 183, 2696-2699. 
</SUP><SUP id="fn4">4. Antelmann, H.; Tjalsma, H.; Voigt, B.; Ohlmeier, S.; Bron, S.; Van Djil, Jan Maarten.; Hecker, M.: A Proteomic View on Genome-Based Signal Peptide Predictions. Genome Research. 2001, 11, 1484-1502.
</SUP></DIV><FOOTER><DIV class="team_name"><H3>Contact Us</H3></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>