<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Technion_Israel_Lab_Notebook_Overview skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Technion Israel/Lab Notebook/Overview</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team: Technion 2015</TITLE></P><DIV class="nav_menu"><A href="https://2015.igem.org/Team:Technion_Israel" style="position:fixed; width: 170px; display:inline-block; text-decoration: none;" class="nav_home_link"><DIV class="home_text">Home</DIV></A><DIV class="main_buttons"><DIV id="project" class="main_button">Project</DIV><DIV id="Lab_Notebook" class="main_button">Lab Notebook</DIV><DIV id="Human_Practices" class="main_button">Human Practices</DIV><DIV id="Judging_Criteria" class="main_button">Judging Criteria</DIV><DIV id="Team" class="main_button">Team</DIV></DIV><NAV><DIV id="Team_Links" class="main_links"><A href="https://2015.igem.org/Team:Technion_Israel/Team/About_Us" id="Team_Links_About_Us">About Us</A><A href="https://2015.igem.org/Team:Technion_Israel/Attributions" id="Team_Links_Attributions">Attributions</A><A href="https://2015.igem.org/Team:Technion_Israel/Team/Sponsors" id="Team_Links_Sponsors">Sponsors</A><A href="https://2015.igem.org/Team:Technion_Israel/Team/gallery" id="Team_Links_gallery">Gallery</A></DIV><DIV id="Judging_Criteria_Links" class="main_links"><A href="https://2015.igem.org/Team:Technion_Israel/Parts" id="Judging_Criteria_Links_Parts">Parts</A><A href="https://2015.igem.org/Team:Technion_Israel/Judging_Criteria/Achievements" id="Judging_Criteria_Links_Achievements">Achievements</A><A href="https://2015.igem.org/Team:Technion_Israel/Collaborations" id="Judging_Criteria_Links_Collaborations">Collaborations</A></DIV><DIV id="Human_Practices_Links" class="main_links"><A href="https://2015.igem.org/Team:Technion_Israel/Practices">Practices</A><A href="https://2015.igem.org/Team:Technion_Israel/Practices/safety">Safety</A></DIV><DIV id="Lab_Notebook_Links" class="main_links"><A href="https://2015.igem.org/Team:Technion_Israel/Lab_Notebook/Overview">Overview</A><A href="https://2015.igem.org/Team:Technion_Israel/Lab_Notebook/Protocols">Protocols</A></DIV><DIV id="project_Links" class="main_links"><A href="https://2015.igem.org/Team:Technion_Israel/Project/Overview">Overview</A><A href="https://2015.igem.org/Team:Technion_Israel/Project/Secretion">Secretion</A><A href="https://2015.igem.org/Team:Technion_Israel/Project/Expression">Expression</A><A href="https://2015.igem.org/Team:Technion_Israel/Project/Cofactor">Cofactor</A><A href="https://2015.igem.org/Team:Technion_Israel/Design">Comb</A><A href="https://2015.igem.org/Team:Technion_Israel/Modeling">Modeling</A><A href="https://2015.igem.org/Team:Technion_Israel/Project/Results">Results</A></DIV></NAV></DIV><DIV class="up_button" style="text-decoration: none;"><A href="#" class="up_link" style="text-decoration: none;"><DIV class="up_button_text" style="text-decoration: none;">UP</DIV></A></DIV><DIV class="title"><H1><B>Lab Notebook</B></H1></DIV><DIV class="content"><DIV class="notebook_wrapper"><DIV class="week"><H1>Week 1: May 2-8</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We designed gBlocks for our gene (coding for the 3α-hydroxysteroid dehydrogenase (3α-HSD) enzyme): one for the original sequence and one <I>Bacillus</I>-optimized.</P><P><I>Active team members this week: Liron, Tal, and Ruth </I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>Planned the Gibson reaction for our gBlock of the gene for 3α-HSD and the plasmid N155 (from our mentors) containing pT7, RBS and the gene for mCherry.</P><P>Ordered the gBlock of 3α-HSD, from IDT, which was optimized for <I>E. coli</I> codon usage.  We also ordered primers for the Gibson reaction for the gBlock.</P><P>Performed reverse PCR to the N155 plasmid and carried out a DPN1 reaction on the product.</P><P><I>Active team members this week: Adi and Alexey</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>This week we did a lot of research.  We found that the 3α-HSD enzyme can use both NADH and NADPH as cofactors for the reduction reaction in order which inactivates dihydrotestosterone (DHT) to 3α-diol.</P><P>NADPH has been found to not only enable the reaction in the direction we want, but also to inhibit the reverse reaction of DHT (Rizner, et al) synthesis.  Therefore, we have decided to try to overproduce NADPH in a host organism, in order to support the activity of the enzyme.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>A mechanical engineering student, Michael Yannai, helped us design the first prototype of the comb.  We designed it to look like a lice comb with internal channels and thick teeth.  The main entrance to the channels is located at the side of the comb.</P><P><I>Active team members this week: Maayan</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 2: May 9-15</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We received the gBlocks and performed PCR. We had some difficulties amplifying the <I>Bacillus</I>-optimized gBlock, and since we found out that <I>B.subtilis</I> has no preferred codon usage, we decided to continue only with the gBlock of the original sequence.</P><P><I>Active team members this week: Liron, Tal, and Ruth</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>Our gBlock arrived!!!</P><P>We performed PCR on the 3α-HSD gBlock with only 10 cycles (as recommended), but received no band in the gel.</P><P>We performed PCR on the 3α-HSD gBlock in higher temperatures, but still received no bands in the gel.</P><P><I>Active team members this week: Adi</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We conducted further research about the production of NADPH in cells.  We sent e-mails to authors of journal articles to enquire about genes and knockouts associated with NADPH overproduction.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>Met with Prof. Hovav Gazit and set a date for printing the comb.</P><P><I>Active team members this week: Maayan</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 3: May 16-22</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We performed a PCR reaction for the mCherry reporter gene from pUG34 plasmid we got from Inbal Vaknin from Prof. Roee Amit’s lab at Technion and added a RBS sequence with the primers.</P><P>We cleaned the product and performed a restriction enzyme reaction with KpnI and HindIII.</P><P><I>Active team members this week: Liron, Tal, and Ruth</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>Performed PCR on the 3α-HSD gBlock in lower temperatures, no bands in the gel.</P><P><I>Active team members this week: Alexey</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>Received replies from various authors.  Prof. Toby Fuhrer, from the Institute of Molecular Systems Biology will be sending us an <I>E. Coli</I> MG1655 strain with a knockout of the <I>pgi</I> and <I>UdhA</I> genes associated with the consumption of NADPH.  Prof. Hanna Engleberg-Kulka’s lab at the Hebrew University of Jerusalem will be sending us a plasmid containing a <I>zwf</I> gene, which codes for the glucose-6-phosphate dehydrogenase enzyme related to the Pentose Phosphate Pathway, in which two moles of NADPH are produced.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>The first prototype of the comb was been printed in Prof. Hovav Gazit's lab with a 3-D printer.  Unfortunately, the channels were blocked with ABS material from the printer.   </P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 4: May 23-29</H1><DIV class="week_content"><DIV class="ecoli"><H2>Expression</H2><P>Performed PCR to the 3α-HSD gBlock with more cycles and 15°C temperature gradient- no bands in the gel.</P><P><I>Active team members this week: Alexey and Adi</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We received the knockout strain from Prof. Fuhrer! We prepared glycerol stock and competent cells of the strain for further use.</P><P>Planned the PCR and ordered primers for the <I>zwf</I> gene containing the sequence for restriction sites SpeI and XbaI in order to simplify future ligation with BioBricks.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We tried to open the channels inside the comb with a water pressure machine in Prof. Hovav Gazit's lab and mechanically with a needle.  Our attempts were unsuccessful.   </P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 5: May 30-June 5</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We met with Shira Omer from Avigdor Eldar’s lab from Tel Aviv University and Prof. Ilana Kolodkin from Weizmann Institute. They both recommended working with pDR111 plasmid (a shuttle vector which integrates into the <I>Bacillus</I> genome).</P><P><I>Active team members this week: Liron, Adi, Yael, and Ruth</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>We ordered new primers for the 3α-HSD gBlock.</P><P>PCR to the 3α-HSD gBlock with new primers and 15°C temperature gradient.  We saw weak bands in the gel, but found no DNA after cleaning.</P><P><I>Active team members this week: Alexey and Adi</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We are still waiting to receive the plasmid containing the <I>zwf</I> gene.  Apparently there are delays in the postal service.  Since we suspect that the DNA will break down, we have hired a messenger service to bring it to us early next week.</P><P>Planned activity verification experiments for when we have our clones.  We will do preliminary experiments using fluorescence, which will indicate total NADPH and NADH levels.  Since the overexpressed <I>zwf</I> gene should produce NADPH alone, the difference between the strains with the 
				insert and without the insert should indicate the increased production of NADPH in the cells.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We designed a second prototype.  We changed the location of the main entrance hole to the center-top of the comb in order to help us to open the channels, mechanically, more easily .</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 6: June 6-12</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We checked the plasmids we got from Tel Aviv University and Weizmann Institute by sending for sequencing.</P><P><I>Active team members this week: Liron, Tal, and Ruth</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>We performed a PCR reaction again with the new primers.  We saw no bands in the gel.</P><P>We planned a new strategy for cloning using restriction enzymes, with <A href="http://parts.igem.org/Part:BBa_K784023" target="_blank">BBa_K784023</A> in pSB1C3 (from iGEM12_Technion) as the backbone.</P><P><I>Active team members this week: Alexey and Adi</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We received the plasmid carrying the <I>zwf</I> gene and did PCR to enhance the gene.  We cut BioBrick <A href="http://parts.igem.org/Part:BBa_K525998" target="_blank">BBa_K525998</A>- pSB1C3 with a pT7 promoter and RBS, with SpeI and ligated with our gene. </P><P>We ordered primers for sequencing the <I>zwf</I> gene in order to verify it.</P><P>Transformation of the BioBrick into Top10. </P><P>Colony PCR to the colonies showed three positive colonies.  We sent one for sequencing.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We printed the second prototype of the comb.  The flow that came out of the teeth of the comb was not simultaneous.</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 7: June 13-19</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We designed primers for the 3α-HSD gBlock and mCherry, containing the sequences of the new restriction enzymes (SalI and NheI).</P><P><I>Active team members this week: Liron, Tal, and Ruth</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>Made starters from Top10 with <A href="http://parts.igem.org/Part:BBa_K784023" target="_blank">BBa_K784023</A> from glycerol stock, and then performed a miniprep for the <A href="http://parts.igem.org/Part:BBa_K784023" target="_blank">BBa_K784023</A>.  The DNA concentrations were low, so we repeated the miniprep.  After receiving a satisfactory 
					concentration, we cut the plasmid with SpeI and XbaI and performed a CIP reaction.</P><P>Ordered new primers for both Gibson and restriction enzymes cloning and performed PCR on the 3α-HSD gBlock with all the new primers.  We saw a band in the negative control.</P><P><I>Active team members this week: Adi</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>The sequencing results for the plasmid carrying our gene came out negative ☹.  But we didn't give up! We performed colony PCR again, and this time saw one positive colony, which we sent for sequencing.</P><P>We transformed the gene as it was received on the PQE-32 plasmid with a pT5 promoter and the <I>LacIq</I> operon into the <I>E. coli</I> MG1655 knockout and <I>E. Coli</I> BL21.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We searched for a microfluidics flow specialist and set up a meeting with Prof. Moran Bercovici from mechanical engineering faculty at Technion.</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 8: June 20-26</H1><DIV class="week_content"><DIV class="ecoli"><H2>Expression</H2><P>Performed PCR to the 3α-HSD gBlock with all the new primers and saw weak bands in the gel, but got no DNA after cleaning</P><P><I>Active team members this week: Adi</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>The sequencing results for the gene on the pSB1C3 came out positive!! We transformed the plasmid into <I>E. coli</I> BL21 and the <I>E. coli</I> MG1655 knockout.  Unfortunately no colonies were formed on the LB and agar plates, so we repeated the transformation, but to no avail.</P><P>We received a plasmid carrying the rhlR inducible operon from Prof. Roee Amit’s lab.  We will be using it to express our gene in <I>E. coli</I> MG1655, which would not be able to express genes under the pT7 promoter due to its lack of T7-RNA polymerase.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>Meeting with Prof. Moran Bercovici, a specialist in microfluidic flow.  He gave us a lot of information about the flow inside the channels.</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 9: June 27-July 3</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We performed a PCR reaction to amplify the mCherry with the new restriction sites. We also performed a PCR reaction to amplify the 3α-HSD gBlock. We cleaned these reactions and cut them both with SalI and NheI restriction enzymes and cleaned again.</P><P><I>Active team members this week: Ruth</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>Performed PCR to the 3α-HSD gBlock with all the new primers but saw no bands in the gel.  Therefore, we decided to try a new direction to stop using our gBlock and try using the gBlock of 3α-HSD with the original (non-optimized) sequence.</P><P>We ordered new primers for the gBlock of 3α-HSD with the original sequence.  We planned them based on cloning with restriction enzymes.</P><P><I>Active team members this week: Alexey and Adi</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>Repeated transformation of the pSB1C3 plasmid containing the <I>zwf</I> gene, this time using newly prepared plates and heat shock instead of electroporation.  The colonies grew! We prepared glycerol stocks of each strain.</P><P>Ordered primers containing KpnI and XbaI restriction sites for cloning the zwf gene into the plasmid containing the rhlR operon.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We designed a new version of the comb, changing the channel structure based on the advice we received during the meeting with Prof. Moran Bercovici.</P><P>We also added a handle to the comb for more comfortable use, a larger place for the syringe, and thinner teeth for better brushing.</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 10: July 4-10</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We cut the pDR111 plasmid we received from Tel Aviv University with restriction enzymes SalI and NheI.  We sent it to sequencing.</P><P>Meanwhile, after cleaning from the restriction enzyme reaction, we performed a CIP reaction and cleaned.  We performed a ligation reaction with the cleaned plasmid and pre-cut mCherry and 3α-HSD gBlock.</P><P>We did a heat shock transformation into competent <I>E. coli</I> DH5α we got from Prof. Ilana Kolodkin Gal’s laboratory at Weizmann Institute.</P><P>We performed a colony PCR for both transformations: we got 3 positive colonies out of 9 for the pDR111 and mCherry reaction, but no positive colonies for the pDR111 and 3α-HSD gBlock reaction (out of 9).  
					Therefore, we repeated the ligation step, but this time incubated overnight with a self-ligation control. We also made a starter for one of these colonies for glycerol stock and miniprep.</P><P>We transformed the pDR111 and mCherry into <I>B.subtilis</I> PY79 according to a protocol we got from Tel Aviv University.</P><P>We also ordered a new gBlock for the 3α-HSD gene with a signal peptide and with adaptations to iGEM demands for BioBricks.</P><P>We performed PCR for the 3α-HSD gBlock containing the sequence for signal peptide for aprE. After cleaning, there was a very low concentration.</P><P><I>Active team members this week: Ruth and Tal</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We performed PCR to the <I>zwf</I> gene using the primers planned specifically for ligation into the rhlR-containing plasmid.</P><P>Restriction and ligation of the <I>zwf</I> PCR product and the rhlR-containing plasmid.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We printed the new design in Prof. Moran Bercovici's lab.  Some of the channels were still blocked, but the flow from the comb teeth was simultaneous.  However, the handle was too small to be used comfortably by those with large hands.</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 11: July 11-17</H1><DIV class="week_content"><DIV class="ecoli"><H2>Expression</H2><P>Performed PCR on the 3α-HSD gBlock with the original sequence (RFC10 compatible), with new primers and a 15°C temperature gradient.  We saw good bands on gel.</P><P><I>Active team members this week: Alexey and Adi</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>Transformation of the <I>zwf</I> PCR product and the rhlR-containing plasmid into of the Top10.</P><P>Two positive colonies identified by colony PCR were sent for sequencing.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We designed two containers to fit in the comb to hold our solutions containing <I>B. subtilis</I> (which secretes the 3α-HSD enzyme) and <I>E. coli</I> (designed to overproduce NADPH).</P><P>We designed the handle and the body of the comb to be printed separately because of the enlargement of the handle was too big to be printed by the 3-D printer in one piece.</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 12: July 18-24</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We did a colony PCR to <I>B.subtilis</I> containing mCherry and <I>E. coli</I> DH5a with the 3α-HSD gBlock.</P><P>We made glycerol-stock and performed a miniprep for the positive colony of <I>E. Coli DH5α</I> with the 3α-HSD gBlock on the PDR111 plasmid.</P><P>We transformed the pDR111 plasmid containing the 3α-HSD gBlock into <I>B.subtilis</I> PY79 and performed a colony PCR.</P><P>We repeated the PCR for the signal peptide (SP) and 3α-HSD gBlock with less primer (1µl each). This time the PCR worked so we cleaned it and cut it with SalI and NheI restriction enzymes.
					We then performed an overnight ligation of the cut SP and 3α-HSD gBlock with pre-cut pDR111 plasmid.  We transformed the product into <I>E. coli</I> TOP10 and performed a colony PCR.
					Starters for positive colonies (4 and 9) were made, from which we prepared glycerol-stock and performed a miniprep.</P><P><I>Active team members this week: Ruth, Liron and Tal</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>We performed a restriction reaction of the 3α-HSD gBlock with the original sequence (RFC10 compatible)using SpeI and XbaI enzymes.  
					We then ligated the cut 3α-HSD with pSB1C3 and transformed the ligation product into <I>E. coli</I> Top10. Colonies grew in the control (ligation of pSB1C3 without 3α-HSD).
					Therefore, we performed a colony PCR with 8 colonies, but received no positive colonies.  We repeated the colony PCR, this time with 20 colonies, but received no positive colonies.</P><P>We planned the expression plasmid with the part <A href="http://parts.igem.org/Part:BBa_K1321338" target="_blank">BBa_K1321338</A>, which contains pT7 and an RBS (iGEM14_Imperial), and reporter plasmids with the parts <A href="http://parts.igem.org/Part:BBa_J06702" target="_blank">BBa_J06702</A>, containing an RBS, the mCherry-coding gene, and a terminator (iGEM2005), and
					<A href="http://parts.igem.org/Part:BBa_K1357008" target="_blank">BBa_K1357008</A>, containing an RBS and tsPurple and a terminator (iGEM14_Virginia).  We then extracted the BioBricks from the 2015 DNA Distribution Kit and transformed them into <I> E. coli</I> Top10.
					We made starters from the transformed strains and performed a miniprep.</P><P> We performed a restriction reaction of <A href="http://parts.igem.org/Part:BBa_K1321338" target="_blank">BBa_K1321338</A> with PstI and SpeI enzymes and of <A href="http://parts.igem.org/Part:BBa_J06702" target="_blank">BBa_J06702</A>, <A href="http://parts.igem.org/Part:BBa_K1357008" target="_blank">BBa_K1357008</A> with PstI and XbaI enzymes.</P><P>We performed PCR again to the 3α-HSD gBlock with the original sequence (RFC10 compatible) and cut the 3α-HSD with with SpeI and XbaI restriction enzymes.  
					We then repeated the ligation reaction of the 3α-HSD and the pSB1C3 plasmid and transformed the ligation product to <I>E. coli</I> Top10.</P><P>After colony PCR of 3α-HSD on the pSB1C3 plasmid with 13 colonies, we received 10 positive colonies!!  We sent them for sequencing, but received negative results.</P><P><I>Active team members this week: Alexey and Adi</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>The sequencing results from last week came out negative, so we re-inserted the <I>zwf</I> gene into the plasmid, performed transformation, colony PCR, and sent for sequencing.  The results came out positive!  We prepared a glycerol stock from the positive colonies.  We also performed a mini-prep and transformation into <I>E. coli</I> MG1655 with and without the gene knockouts.  We prepared glycerol stocks from these strains as well.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We printed the new version of the comb which was planned last week.</P><P>We planned a focus group survey which will enable us to receive feedback about the structure and feel of the comb.</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 13: July 25-31</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We transformed  pDR111+signal peptide+3α-HSD and an empty pDR111 into<I>B.subtilis PY79</I>.</P><P>We performed an mCherry expression experiment on the following strains: <I>B.subtilis</I> with an empty pDR111 plasmid, <I>B.subtilis</I> with mCherry, <I>E.coli</I> with empty pDR111, <I>E.coli</I> with mCherry. The <I>B. subtilis</I> strains didn’t grow so we didn’t proceed with the experiment.</P><P>We repeated the mentioned experiment, but we diluted the starters with a ratio of 1:100, rather than 1:20. The experiment was successful but we ran out of sample volume before reaching a plateau.</P><P><I>Active team members this week: Ruth, Liron and Talk</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>Due to last week's negative results, we decided to plan a new restriction strategy for <A href="http://parts.igem.org/Part:BBa_K1321338" target="_blank">BBa_K1321338</A> with 3α-HSD.  We performed a restriction reaction
					of <A href="http://parts.igem.org/Part:BBa_K1321338" target="_blank">BBa_K1321338</A> with SpeI only, and followed it by a CIP reaction.  We then ligated the <A href="http://parts.igem.org/Part:BBa_K1321338" target="_blank">BBa_K1321338</A> with 3α-HSD and transformed the ligation product into
					<I>E. Coli Top10</I>.  The colony PCR with 26 colonies resulted in 1 positive colony which we sent for sequencing.  We finally got a positive clone!!</P><P><I>Active team members this week: Alexey and Adi</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We performed an experiment over 43 hours in which we check the extracellular fluorescence (indicating the NADPH) in <I>E. coli</I> BL21 with a pUC19 plasmid (control) and BL21 with a plasmid containing the <I>zwf</I> gene under a pT7 promoter.  We performed the experiment again over 25 hours.  Both experiments showed a significant increased level of extracellular fluorescence in the strain containing the <I>zwf</I> gene when compared to the control, after 12 hours (see <A href="https://2015.igem.org/Team:Technion_Israel/Project/Results" target="_blank">Project Results</A>)!!!!.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We performed an experiment to check the flow through the comb at different liquid densities.  We check the flow and uniformity in spreading of LB containing various concentrations of glycerol.  The key is to get a uniform spread of a liquid which is thick enough not to drip from the comb and the user's skin.</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 14: August 1-7</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We performed 3 PCR reactions in order to add a signal peptide to mCherry. We got nonspecific bands in the gel, so we cut our band out of the gel and proceeded. We cut the cleaned PCR reaction with SalI and NheI restriction enzymes and ligated with pDR111.</P><P>We transformed pDR111+signal peptide+mCherry  plasmid into TOP10 using the heat-shock transformation technique.</P><P>We performed colony PCR to colonies from the transformation of pDR111+signal peptide+mCherry plasmid to TOP10.</P><P>We transformed the pDR111+signal peptide+mCherry plasmid into <I>B.subtilis</I> PY79. it didn’t work.</P><P><I>Active team members this week: Ruth, Liron and Tal</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>We planned an experiment to check for expression of the 3α-HSD enzyme</P><P>Transformation of <A href="http://parts.igem.org/Part:BBa_K1321338" target="_blank">BBa_K1321338</A> with added 3α-HSD into <I>E. coli</I> BL21.</P><P>Transformation of <A href="http://parts.igem.org/Part:BBa_K1321338" target="_blank">BBa_K1321338</A> only to <I>E. coli</I> BL21 (to use as an experimental control).</P><P>Restriction reaction of <A href="http://parts.igem.org/Part:BBa_K1321338" target="_blank">BBa_K1321338</A> with XbaI and SpeI enzymes in order to extract the pT7 and RBS from the BioBrick.  CIP reaction for the pSB1C3.</P><P>Ligation of the 3α-HSD and pSB1C3.  Transformation of ligation product to Top10.</P><P>Gel cleaning of <A href="http://parts.igem.org/Part:BBa_J06702" target="_blank">BBa_J06702</A>, <A href="http://parts.igem.org/Part:BBa_K1357008" target="_blank">BBa_K1357008</A> (cut with PstI and XbaI restriction enzymes) to get only the BioBrick without the plasmid.</P><P>Colony PCR of the gene for 3α-HSD in pSB1C3 with 13 colonies.  We received 9 positive colonies which we sent for sequencing.  The results came back as not positive.</P><P>Colony PCR of 3α-HSD between <A href="http://parts.igem.org/Part:BBa_K1321338" target="_blank">BBa_K1321338</A> and <A href="http://parts.igem.org/Part:BBa_J06702" target="_blank">BBa_J06702</A>/<A href="http://parts.igem.org/Part:BBa_K1357008" target="_blank">BBa_K1357008</A> with 13 colonies each - 13 positive colonies with <A href="http://parts.igem.org/Part:BBa_J06702" target="_blank">BBa_J06702</A>, 11 positive colonies with <A href="http://parts.igem.org/Part:BBa_K1357008" target="_blank">BBa_K1357008</A>.</P><P><I>Active team members this week: Adi and Alexey</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We attempted to check the intracellular levels of NADPH by performing lysis by sonication.  However, the fluorescence levels were negligent, and some were even lower than the blank! We concluded that the sonication method may have caused the NADPH to breakdown.</P><P>We performed a <A href="https://static.igem.org/mediawiki/2015/4/4b/Technion_Israel_2015_LysisMethodComparison.pdf" target="_blank">lysis experiment</A> in which we check various lysis procedures in order to determine intracellular NADPH in <I>E. coli</I> BL21 with pUC19 versus  <I>E. coli</I> BL21 with the <I>zwf</I> gene.  The results for each lysis method showed higher intracellular fluorescence for the strain with the gene than the control, hinting that our gene does, in fact, increase intracellular NADPH.  We decided to use either a freeze-thaw method or a short sonication method from now on, since those showed the best results.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>After getting the focus survey results and feedback about the comb, we changed the shape of the comb to get a more aesthetic look, and made a modular handle that can be easily adjusted to fit any consumer hand size.</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 15: August 8-14</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We repeated the transformation of the pDR111+signal peptide+mCherry plasmid into <I>B.subtilis</I> PY79.</P><P>We discovered that there was a mistake in one of our primers, so the signal peptide had an additional AAA (lysine). Therefore, we ordered a new primer.</P><P>With this problematic signal peptide, we tried our secretion protocol (see protocol <A href="https://static.igem.org/mediawiki/2015/e/e2/Technion_Israel_2015_protocols_B.subtilis-mCherry-secretion.pdf" target="_blank">here</A>). Apparently, there was a confusion with the samples, so we had no results.</P><P><I>Active team members this week: Ruth, Liron and Talk</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>Transformation of 3α-HSD between <A href="http://parts.igem.org/Part:BBa_K1321338" target="_blank">BBa_K1321338</A> and <A href="http://parts.igem.org/Part:BBa_J06702" target="_blank">BBa_J06702</A> (containing mCherry)/ <A href="http://parts.igem.org/Part:BBa_K1357008" target="_blank">BBa_K1357008</A> (containing tsPurple) to <I>E. coli</I> BL21 - only those containing the gene for tsPurple showed purple colonies, indicating that in those, we have over-expression of our enzyme.</P><P>First activity experiment with <I>E. coli</I> BL21 containing 3α-HSD and one with no 3α-HSD as a negative control.  We concluded that the experiment was too short.</P><P>Another activity experiment with <I>E. coli</I> BL21 as before and DH5α with and without 3α-HSD.  The results had too much noise.</P><P><I>Active team members this week: Adi and Ruth</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We performed an experiment over the course of 25 hours, checking the extracellular fluorescence (indicating NADPH levels) in <I>E. coli</I> BL21 with and without the <I>zwf</I> gene, as well as <I>E. coli</I> MG1655 with and without the gene knockouts, and with and without the <I>zwf</I> gene.
					Our results showed expected extracellular NADPH levels between 12 and 20 hours, with the <I>E. coli</I> MG1655 with a knockout of the <I>pgi</I> and <I>UdhA</I> genes, but without the <I>zwf</I> gene, showing the highest extracellular NADPH levels.  We assume that this is either because of inadequate expression of the gene under the rhlR promoter and operon, or because the metabolic imbalance was too great and cause alternative pathways to help the cell balance its NADPH levels.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We printed the final prototype of the comb!</P><P><I>Active team members this week: Maayan and Sagi</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 16: August 15-21</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We got our new primer for a signal peptide+mCherry. We performed a PCR reaction and cleaned from gel.</P><P>We amplified the signal peptide of <I>aprE</I> gene from the gBlock of our gene with the signal peptide. We cut it with EcoRI and PstI restriction enzymes, and ligated with the linearized pSB1C3 plasmid from the 2015 DNA Distribution Kit. We performed colony PCR and sent the positive colonies for sequencing.</P><P>We transformed the ligation product into TOP10. We performed colony PCR and got 3 positive colonies.</P><P>We transformed pDR111+signal peptide+mCherry plasmid into <I>B.subtilis</I> PY79 and performed colony PCR.</P><P><I>Active team members this week: Ruth, Liron and Tal</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>We performed an activity experiment with different lysate concentrations.  We got different slopes and decided to measure the results according to the slopes (the derivative of the graph).</P><P>PCR to 3α-HSD from the gBlock with the original sequence (RFC10 compatible), with pSB1C3 prefix and suffix primers.  We concluded that the optimal temperature is 60°C.</P><P>Restriction reaction of linearized pSB1C3 and 3α-HSD with the original sequence (RFC10 compatible) with EcoRI and PstI enzymes.</P><P>We produced a calibration curve for NADPH using various gains on the fluorescence plate reader in order to find the optimal gain.  We realized that until now we had been using the wrong one.  Based on our results, there is a secondary reaction, so in future activity experiments we will add DHT only after 30 minutes.</P><P>Ligation of the 3α-HSD and pSB1C3.  Transformation of the ligation product to <I>E. coli</I> Top10.</P><P><I>Active team members this week: Adi, Ruth, and Nitzan</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We tried to use the Biovision NADP+/NADPH Assay kit to check the levels of NADPH inside our genetically modified cells.  The kit protocol had to be modified since it is geared towards use for tissue cells and other eukaryotic cells.
					However, despite our changes, we received no results.  We then tried increasing the time in the extraction buffer, but still received no results.  We sent e-mails to various academics whom had published articles indicating use of the kit, asking which modifications they made to the protocol.  The suggested performing lysis with the freeze-thaw method.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 17: August 22-28</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We performed transformation of the empty plasmid, mCherry, signal peptide+mCherry, 3α-HSD, and signal peptide+3α-HSD to a new <I>B. subtilis</I> sample.</P><P><I>Active team members this week: Ruth, Liron and Tal</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>Activity experiment in 37°C to mimic ideal conditions as well as the conditions which will be available when used in a final product (yes we can!).  The enzyme is definitely active!!</P><P>Colony PCR of 3α-HSD in pSB1C3 with 13 colonies .  We receive 12 positive colonies which we sent for sequencing.  We received positive results.</P><P>Activity experiment with different concentrations of DHT.  We saw changes in activity depending on the DHT concentration.</P><P>Second activity experiment with different concentration of DHT - we got great results of the reaction rate versus DHT concentration (see <A href="https://2015.igem.org/Team:Technion_Israel/Project/Results" target="_blank">Project Results</A>)!!!!</P><P><I>Active team members this week: Adi and Ruth</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We tried using the Biovision NADP+/NADPH Assay kit along with a freeze-thaw lysis process.  However, the results came out negative.  We decided to skip using the kit in the future and, instead focus on the combined experiment testing our NADPH levels along with the cells excreting and producing the 3α-HSD enzyme.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 18: August 29-September 4</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We repeated the mCherry secretion experiment and received results as expected!!!</P><P><I>Active team members this week: Ruth, Liron and Tal</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>Activity experiment with different concentration of NADPH (25-175µM).  We got interesting results.</P><P><I>Active team members this week: Adi and Ruth</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>We checked the intracellular and extracellular NADPH over time in mediums containing different concentrations of glucose.  Glucose is a carbon source which is modified to produce glucose-6-phosphate, the substrate of the glucose-6-phosphate dehydrogenase enzyme which is coded by the <I>zwf</I> gene.</P><P><I>Active team members this week: Yael and Roni</I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 19: September 5-11</H1><DIV class="week_content"><DIV class="bacillus"><H2>Secretion</H2><P>We repeated the mCherry secretion experiment.</P><P><I>Active team members this week: Ruth, Liron and Tal</I></P></DIV><DIV class="ecoli"><H2>Expression</H2><P>Combined activity experiment with lysate from <I>E. coli</I> BL21, <I>B. subtilis</I> supernatant and NADPH from different sources.</P><P><I>Active team members this week: Adi and Ruth</I></P></DIV><DIV class="nadph"><H2>Cofactor</H2><P>Combined activity experiment with lysate from <I>E. coli</I> BL21, <I>B. subtilis</I> supernatant.  We used the extracellular medium of the <I>E. coli</I> BL21 with the <I>zwf</I> gene and the <I>E. coli</I> MG1655 Δ<I>pgi</I>Δ<I>UdhA</I>.
					</P><P><I>Active team members this week: Roni</I></P></DIV><DIV class="comb"><H2>Comb</H2><P>We performed an experiment to determine the expected shelf-life of our product in various conditions.  The samples were kept at varying temperatures (-20°C-28°C) for five days.  We concluded that the solutions would ideally be kept in a freezer.</P><P><I>Active team members this week: Alexey and Maayan </I></P></DIV></DIV></DIV><DIV class="week"><H1>Week 20: September 12-18</H1><DIV class="week_content"><DIV class="comb"><H2>Comb</H2><P>Conducted the shelf life experiments</P><P><I>Active team members this week: Alexey and Maayan </I></P></DIV></DIV></DIV></DIV></DIV><FOOTER><DIV class="team_name"><H3>Contact Us</H3></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>