<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU_SINGAPORE_Experiment skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU SINGAPORE/Experiment</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="w3-bar" id="myNavbar"><DIV class="w3-top w3-padding-large w3-transparent"><A href="https://2017.igem.org/Team:NTU_SINGAPORE" class="w3-bar-item w3-button"><FONT color="#800020"> HOME </FONT></A><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">TEAM</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Team" class="w3-bar-item w3-button"><FONT color="#800020">Our Team</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Collaborations" class="w3-bar-item w3-button"><FONT color="#800020">Collaboration</FONT></A></DIV></CLASS></DIV><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">PROJECT</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Description" class="w3-bar-item w3-button"><FONT color="#800020">Description</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Experiment" class="w3-bar-item w3-button"><FONT color="#800020">Experiment</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Results" class="w3-bar-item w3-button"><FONT color="#800020">Proof of Concept</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Demonstrate" class="w3-bar-item w3-button"><FONT color="#800020">Demonstrate</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Improve" class="w3-bar-item w3-button"><FONT color="#800020">Improve</FONT></A></DIV></CLASS></DIV><A href="https://2017.igem.org/Team:NTU_SINGAPORE/InterLab" class="w3-bar-item w3-button"><FONT color="#800020">INTERLAB STUDY</FONT></A><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">PARTS</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Basic_Part" class="w3-bar-item w3-button"><FONT color="#800020">Basic Parts</FONT></A></DIV></CLASS></DIV><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">HUMAN PRACTICE</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/OurStory" class="w3-bar-item w3-button"><FONT color="#800020">Our Story</FONT></A><FONT><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/Silver" class="w3-bar-item w3-button"><FONT color="#800020">Silver</FONT></A><FONT><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/Gold_Integrated" class="w3-bar-item w3-button"><FONT color="#800020">Gold</FONT></A></FONT></FONT></DIV></CLASS></DIV><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Attributions" class="w3-bar-item w3-button"><FONT color="#800020">ATTRIBUTIONS</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Contact Us" class="w3-bar-item w3-button"><FONT color="#800020"> CONTACT US</FONT></A><DIV id="navDemo" class="w3-bar-block w3-white w3-hide w3-hide-large w3-hide-medium"><A href="#home" class="w3-bar-item w3-button" onclick="toggleFunction()">HOME</A><A href="#research" class="w3-bar-item w3-button" onclick="toggleFunction()">RESEARCH</A><A href="#interlab study" class="w3-bar-item w3-button" onclick="toggleFunction()">INTERLAB STUDY</A><A href="#parts" class="w3-bar-item w3-button" onclick="toggleFunction()">PARTS</A><A href="#collaboration" class="w3-bar-item w3-button" onclick="toggleFunction()">COLLABORATION</A><A href="#contact" class="w3-bar-item w3-button" onclick="toggleFunction()">CONTACT</A></DIV></DIV><DIV class="column full_size"><DIV class="ex w3-container w3-padding-32" id="Introducing our Projects"><H3 class="w3-border-bottom w3-border-light-grey w3-padding-16">Introducing our Projects</H3></DIV></DIV><DIV class="content-overview" style="padding: 0px 66px; width: 872px;"><DIV class="circle-overview circle1"><A href="#TP" style="top: -9px; width:200px; height:200px;" class="overview-content"><SPAN class="overview-text">Truncation Project</SPAN></A></DIV><DIV class="circle-overview circle2"><A href="#HDR" style="top: -9px; width:200px; height:200px;" class="overview-content"><SPAN class="overview-text">Homology Directed Repair</SPAN></A></DIV><DIV class="circle-overview circle3"><A href="#EGFR" style="top: -9px; width:200px; height:200px;" class="overview-content"><SPAN class="overview-text">EGFR Project</SPAN></A></DIV></DIV><DIV class="ex column full_size"><H2> Truncation Project </H2><P>In vivo application of Cas9 and its variants faces issues with delivery. One of the best methods of Cas9 delivery is through recombinant adeno associated virus (rAAV). However, rAAV has a packing limit of approximately 4.7 kB. The Cas9 gene alone is more than 4.1 kB. This leaves very little room for optimal promoter and polyA signal, as well as for sgRNA expression. Recent applications of Cas9 variants fused to effector proteins would also not fit into rAAV vector. Thus, there is strong interest in searching for smaller Cas9 proteins that can perform on par with the well-characterized SpCas9. </P><P>A continuation of last year’s dCas9 truncation project, this year, we seek to further truncate the Streptococcus pyogenes dCas9 protein to achieve an even smaller protein still capable of sgRNA and target DNA binding. We also mutated key residues to improve dCas9 affinity for DNA, demonstrating partial recovery of our truncated dCas9 (tCas9) towards wild-type activity. </P><P>To assess tCas9 ability to bind to sgRNA and target DNA, dCas9 is fused to VPR, a tripartite transcriptional activator. Gene activation activity is then assessed against WT-dCas9-VPR. 2 methods of gene activation are performed. In our proof of concept, exogenous reporter gene activation is performed to rapidly give information of activity of tCas9s. For demonstration, endogenous gene activation is performed, giving a closer approximation of real-world application. </P><U>Truncation methodology</U><P>Prior attempts at truncating tCas9 (Freiburg 2013) were unsuccessful largely due lack of structural information. The publication of the crystal structure of Cas9 (Nishimasu) allowed for identification of domain and subdomains. Based on crystal structure and homology information, Cas9 contains 2 nuclease domains. In dCas9, these domains were mutated to be non-functional. </P><P>In this project, we rationally removed domains and subdomains that do not directly contribute to sgRNA binding or target DNA binding. Truncation is largely guided by crystal structure, by selecting domains and subdomains that, when removed, should minimally impact dCas9 folding. A short linker replaces the truncated domain. Truncation is performed at gene level, using Gibson assembly of the dCas9 plasmid. Gibson primers are designed to exclude the truncated region while introducing the short linker sequence. </P><U>Truncations achieved thus far</U><P>Of the various truncations tested, truncations in the HNH domain was found to be very well tolerated, likely due to the highly modular nature of the domain. REC2 domain can be truncated, at the cost of around 50% activity – agreeing with literature (Nishimasu). Lastly, a small RuvCIII-2 deletion is also well tolerated. </P><P>Although other truncations were also tested, it was found that these 3 truncations combine well together i.e. negative effect on gene activation is minimal when truncations were combined. The triple truncation (∆REC2 ∆RuvCIII-2 ∆HNH, henceforth referred to as ‘∆3ple’) is especially of interest since it reduces the size of the dCas9 gene to around 3.2kB – on par with the size of Sa-Cas9. </P><H2> Homology Directed Repair </H2><P> CRISPR/Cas9 has been widely used as a targeted genome editing tool in human cells. This is due to the simplicity of designing the guide RNA (gRNA), which can be easily designed to target any sequence. This simplicity is especially apparent when compared to previous gene editing tools which rely on protein-DNA interaction, such as meganucleases, zinc finger nucleases (ZFNs), or transcription activator-like effector nucleases (TALENs), which are more difficult to design.</P><P>In order to achieve targeted engineering in human cells, Cas9 protein first cuts both strands of the target DNA (guided by the gRNA via complementary base pairing), making a double-strand break (DSB). The DSB is perceived as a negative signal by the cell and hence is targeted for repair. The cell repairs the DSB by either NHEJ (non-homologous end joining) or HDR (homology-directed repair). In the context of targeted engineering, HDR is preferred since NHEJ can introduce indel mutations, whilst HDR can introduce any donor sequence that we want as long as the donor DNA is flanked by homology regions.
</P><P>Unfortunately, in living cells, NHEJ occurs more frequently than HDR. Moreover, HDR proteins are expressed only at the S and G2 phases of the cell cycle, since these are the phases when sister chromatids have already formed and can serve as a donor should a DSB occur. This poses a problem if we wish to edit non-dividing cells, such as neurons and myocytes. Therefore, improving the efficiency of the HDR pathway is important to enhance the robustness of CRISPR/Cas9.</P><P>Therefore, in order to improve HDR efficiency, we tested a fusion protein between SpCas9 and Rad52. Rad52 is one of the HDR proteins involved in mediating ssDNA annealing between homologous sequences and in assisting homologous pairing. The construct was tested in HEK293FT cells and analyzed for the HDR occurrence.</P><H2> EGFR Project </H2><P>CRISPR-associated (Cas) nucleases from a variety of different bacterial species have been studied for their genome editing capabilities, with each nuclease possessing different characteristics. SpCas9 protein, which is a CRISPR/Cas system found in Streptococcus pyogenes, is one of the most common and widely studied CRISPR-Cas types. Besides SpCas9, other Cas9 proteins, as well as the Cpf1 endonuclease, are also of interest to us because of their ability to create blunt or staggered double-stranded breaks in target DNA. This makes them potential candidates for gene inactivation, correction or repurpose. In last year's iGEM project, Team NTU performed the efficiency evaluation for 5 Cas9/Cpf1 endonucleases: Streptococcus pyogenes Cas9 (SpCas9), Staphylococcus aureus Cas9 (SaCas9), Neiserria meningitides Cas9 (NmCas9), Acidaminococcus species (AsCpf1) and Lachnospiraceaebacterium (LbCpf1). In step with the further development and refinement of CRISPR/Cas tools, attempts at introducing the CRISPR/Cas system in vivo through clinical trials have been made. As the application of CRISPR continues to gain traction, we think that there is value in evaluating and determining more suitable CRISPR/Cas systems for therapeutics to facilitate clinical trials for specific diseases. </P><P>Therefore, in iGEM 2017, we have decided to investigate the suitability of different CRISPR/Cas systems: SpCas9, SaCas9, AsCpf1 and LbCpf1 endonucleases as therapeutics to target the deletion mutation in the Epidermal Growth Factor Receptor (EGFR) gene related to non-small cell lung cancer. This deletion mutation, namely Del(E746-A750), refers to the in-frame deletion of exon 19 of the EGFR gene. The investigation was performed in the PC9 cell line, which is derived from human adenocarcinoma from lung tissue and harbors the Del(E746-A750) mutation in the EGFR gene, instead of using the Human Embryonic Kidney 293 (HEK293) cell line as in the previous year.

</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>