<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU_SINGAPORE_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU SINGAPORE/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="w3-bar" id="myNavbar"><DIV class="w3-top w3-padding-large w3-transparent"><A href="https://2017.igem.org/Team:NTU_SINGAPORE" class="w3-bar-item w3-button"><FONT color="#800020"> HOME </FONT></A><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">TEAM</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Team" class="w3-bar-item w3-button"><FONT color="#800020">Our Team</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Collaborations" class="w3-bar-item w3-button"><FONT color="#800020">Collaboration</FONT></A></DIV></CLASS></DIV><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">PROJECT</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Description" class="w3-bar-item w3-button"><FONT color="#800020">Description</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Experiment" class="w3-bar-item w3-button"><FONT color="#800020">Experiment</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Results" class="w3-bar-item w3-button"><FONT color="#800020">Proof of Concept</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Demonstrate" class="w3-bar-item w3-button"><FONT color="#800020">Demonstrate</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Improve" class="w3-bar-item w3-button"><FONT color="#800020">Improve</FONT></A></DIV></CLASS></DIV><A href="https://2017.igem.org/Team:NTU_SINGAPORE/InterLab" class="w3-bar-item w3-button"><FONT color="#800020">INTERLAB STUDY</FONT></A><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">PARTS</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Basic_Part" class="w3-bar-item w3-button"><FONT color="#800020">Basic Parts</FONT></A></DIV></CLASS></DIV><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">HUMAN PRACTICE</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/OurStory" class="w3-bar-item w3-button"><FONT color="#800020">Our Story</FONT></A><FONT><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/Silver" class="w3-bar-item w3-button"><FONT color="#800020">Silver</FONT></A><FONT><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/Gold_Integrated" class="w3-bar-item w3-button"><FONT color="#800020">Gold</FONT></A></FONT></FONT></DIV></CLASS></DIV><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Attributions" class="w3-bar-item w3-button"><FONT color="#800020">ATTRIBUTIONS</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Contact Us" class="w3-bar-item w3-button"><FONT color="#800020"> CONTACT US</FONT></A><DIV id="navDemo" class="w3-bar-block w3-white w3-hide w3-hide-large w3-hide-medium"><A href="#home" class="w3-bar-item w3-button" onclick="toggleFunction()">HOME</A><A href="#research" class="w3-bar-item w3-button" onclick="toggleFunction()">RESEARCH</A><A href="#interlab study" class="w3-bar-item w3-button" onclick="toggleFunction()">INTERLAB STUDY</A><A href="#parts" class="w3-bar-item w3-button" onclick="toggleFunction()">PARTS</A><A href="#collaboration" class="w3-bar-item w3-button" onclick="toggleFunction()">COLLABORATION</A><A href="#contact" class="w3-bar-item w3-button" onclick="toggleFunction()">CONTACT</A></DIV></DIV><DIV class="ex column full_size"><DIV class="ex w3-container w3-padding-32" id="Demonstrate"><H3 class="w3-border-bottom w3-border-light-grey w3-padding-16">Demonstrate</H3><PSTYLE> This section describes the feasibility and usefulness of our project.</PSTYLE></DIV><P>In answering the question on how Cas9 mediated gene regulation could be used to treat diseases, we realized that there is a need to bridge our work on truncated dCas9-VPR towards therapeutic applications. Other than demonstrating the potential real-world application of our truncated dCas9-VPR in a clinical setting, research towards ‘treatment of diseases’ would be more directly relatable to the public.  </P><P>Literature review of deficiency diseases revealed a few prior works concerning gene upregulation for potential treatment. Proteins that can be engineered to target specific sequences, such as transcription activator-like effector (TALE) and zinc finger (ZF) were often utilized to target endogenous promoters or enhancers. An example by Chapdelaine et al utilized TALE fused to VP64 to upregulate frataxin, demonstrating a potential treatment of Friedreich Ataxia. Much more recently, Matharu et al demonstrated potential treatment of obesity caused by Sim1 haploinsufficiency in mice. More importantly, this was achieved with CRISPRa, using dCas9-VPP64. Two diseases with well-established gene targets for upregulation were selected. 
</P><H2>Factor VII deficiency</H2><P>Factor VII (F7) deficiency is caused by the lack of Factor VII, a serine protease that triggers blood coagulation. Patients suffer from excessive, potentially life-threatening bleeding. Congenital FVII deficiency can result from mutations in the F7 promoter, leading to poor expression. Specifically, mutations in promoter transcription factor binding sites (TFBS) for Sp1 and HNF4 results in impaired binding. Barbon et al. designed TALE-VP64 targeting specific loci in the F7 promoter to upregulate F7 expression. In testing the 4 loci targeted, the authors concluded that TALE-VP64 that competes for or sterically hinders endogenous transcription factor binding are ineffective.</P><P>Thus, in our gRNA design, considerations were made to avoid overlapping either of the two TFBS. The PAM requirement of Cas9 also limited available sites, meaning sequences targeted by the author’s TALE-VP64 were not directly adaptable.</P><P>Based on the best TALE-VP64 in the paper (TF4), Cas_TF4 was adapted. However, due to PAM requirement, the target locus was shifted back into the Sp1 TFBS (CCCCTCCCC). Thus, Cas_oTF4 was also designed to target the opposite strand, using GGG (complement of the underlined CCC) as PAM. TF1 was found to not work well in the paper despite the target locus being behind the Sp1 TFBS. It is likely that the proximity was sufficient to sterically hinder Sp1 binding. Thus, Cas_TF1 was adapted to bind a few base pairs further from Sp1 TFBS.</P><P>Lastly, to emulate the protospacer design in our reporter plasmid, where the target loci is located before the tre-tight promoter, Cas_F7-338 was designed, binding to the -338 position of the promoter. 
</P><H2>Sickle cell disease and β-thalassemia</H2><P> Sickle cell disease (SCD) and beta-thalassemia are caused by mutation to the HBB locus, which codes for beta-globin.  The adult hemoglobin HbA is a tetramer made up of two beta globin and 2 alpha globin ((α2β2). In SCD, point mutations in HBB creates hemoglobin variants that can clump in low oxygen conditions. The clumping of cells can lead to vessel occlusion and hemolysis. In beta thalassemia, the mutation leads to loss of function. Severity can range from partial to total depending on number of alleles affected. Treatment, especially for thalassemia major, requires lifelong blood transfusions and transplantation of healthy bone marrow. Interestingly, beta-globin is only required in adult hemoglobin. During fetal development, the gamma-globin homologue forms the tetramer with alpha-globin instead. This fetal hemoglobin HbF has higher affinity for oxygen, enabling efficient extraction of oxygen from maternal blood through the placenta. A benign condition known as hereditary persistence of fetal hemoglobin (HPFH) has been observed to alleviate SCD and beta-thalassemia. HPFH is itself often a result of mutation in gamma-globin promoters, preventing shutdown in adulthood. The gamma globin can thus compensate for loss-of-function in beta-globin, resulting in a much milder disease phenotype. </P><P>Therefore, there is strong interest in upregulation of HbF. Chemical drugs such as hydroxyurea has been show to activate gamma globin synthesis. Unfortunately, these drugs come with negative side effects such as bone marrow toxicity. To directly upregulate HbF levels, Graslund et al. designed ZF-VP64 to target the promoter of gamma-globin gene HBG1. In their selection of the target sequence, the authors purposefully targeted a region proximal to multiple TFBS motifs, reasoning that such a region would be especially amenable to gene regulation by transcriptional activators. Of the three ZF-VP64 built, only 1 worked (gg-1) – most likely due to considerably higher affinity compared to the other ZFs (gg-2, gg-3). Indeed, this is a major weakness with zinc finger targeting – extensive and costly protein optimization would be necessary to achieve useful binding. </P><P>In adapting our gRNA design, we sought to target a sequence as close to gg-1 as possible, and failing that, to at least avoid any potential TFBS – a similar consideration to the design of F7 gRNA. Unfortunately, there are no PAM site available within the TFBS cluster such that all TFBS can be avoided. We settled with Cas_gg1/2, which would occlude the distal CCAAT motif. To avoid all the TFBS in the cluster, Cas_pFKLF was designed to bind before the FKLF motif. Lastly, we also designed Cas_HBG1-276 to replicate reporter protospacer design.</P><H2>Test with WT-dCas9-VPR</H2><P>A preliminary confirmation with WT-dCas9-VPR was performed to determine which gRNA works best for activation of the target genes. Using qPCR, mRNA level of both genes were analyzed for all cells. The HEK293FT cells used are the same as those used in our truncation endogenous tests. </P><P>Results indicated that Cas_TF1 seems ideal for F7 gene activation while Cas-gg1/2 seems ideal for HBG1 gene activation. Upregulation of both genes by dCas-VPR results in much higher fold expression, due to the use of tripartite transcriptional activator VP64-p65-Rta (VPR), as compared to only VP64 used by the original authors. Cas_pFKLF appears to upregulate F7 – possibly due to off-target effects. Based on the results here, several speculations could be made. In F7, Cas_TF1 surprisingly performed better than Cas_TF4, possibly due to the shift in target sequence further from the Sp1 TFBS when adapting for Cas9 PAM requirement. In HBG1, the considerably higher fold increase in gene expression suggests that the basal level of HBG1 expression is likely very low in HEK293FT cells. This is expected considering HBG1 expression is limited to fetal blood cells while HEK293FT is derived from embryonic kidney cells. </P><P>The high fold increase in Cas_gg1/2 targeted gene activation is especially intriguing. In the initial design stages, Cas_gg1/2 was presumed to perform poorly as dCas9-VPR binding would occlude the distal CCAAT motif, potentially preventing endogenous transcription factors from binding. Further literature review, however, revealed that a GA mutation in -117 (yellow highlight in the image) can result in HPFH. Graslund et al. noted that an unknown protein binding the CCAAT motif can be affected by this mutation, suggesting that the increased HBG1 expression in HPFH could be due to loss of binding by this unknown protein. As our Cas9_gg1/2 would occlude this -117 G, it could have emulated a GA mutation, causing loss of binding by the unknown, possibly repressive protein. This suggests that the extensive gene activation achieved by Cas9_gg1/2 may be partially contributed by this derepressive effect, compounding the strong gene activation by VPR fusion. For mRNA quantification, qPCR primers used are the same as in both papers.</P><H2>Tests with truncated dCas9-VPR</H2><P>For tests with truncated dCas9-VPR, the best performing target loci were chosen. For F7, Cas_TF1 was utilized. For HBG1, Cas_gg1/2 was utilized. </P><P>The results correlate to exogenous and especially housekeeping endogenous gene activation. While ∆3ple performed poorly in activating both genes, enhanced ∆3ple (5_6_15) with 3 mutations showed improvement in therapeutic gene activation. The design of better dCas9-VPR variants would surely increase the therapeutic utility of truncated dCas9-VPR for gene regulation.</P><P>It is important to note that these experiments were performed on HEK293FT human embryonic kidney cell line, where basal transcription of these genes are expected to be low. While one may argue that this models deficiency diseases, a better model would be performing these experiments on model cell lines that better represent the target cell the treatment is targeting. Examples would be using K562 leukemia cell lines for HBG1 or Hep10 primary hepatocytes for F7. An even better model would be to generate the disease genotype in these cells or even acquiring primary cells from patients with these diseases, which could provide much more conclusive information on whether the disease phenotype can be corrected.  
</P><H2>Conclusion</H2><P>Cas9 mediated gene activation for the therapeutic treatment of deficiency diseases holds a lot of promise. While the very same diseases studied in our project can similarly be treated by Cas9 mediated genome editing, genome editing in general still faces many major roadblocks towards clinical adoption. Off-target potential remains an unresolved issue and can have unforeseen consequences since the effects of genome editing are permanent. On the other hand, the therapeutic application of dCas9-VPR as described in our project is non-integrative and transient. Treatment can be stopped should negative side effects be discovered.</P><P>We feel that adoption of dCas9 mediated gene regulation for therapy can progress more rapidly that other applications such as genome editing. While routine dosages may be necessary due to its transient nature, this makes dCas9 safer since therapy can be stopped anytime. In addition, the requirement for routine dosages may be attractive to pharmaceutical companies and patients alike, since the possibility of a recurring revenue model means that treatment can be priced lower, improving accessibility. For some tissues, such as those implicated in the blood diseases studied here, delivery issues may even be bypassed, as the patient’s own hematopoietic stem cells may be transfected ex vivo, before being re-transplanted back. </P><P>Our work with truncated dCas9-VPR paves the way towards efficient in vivo delivery of dCas9-VPR for therapy. Presently, our ∆3ple fused to VPR weighs in at 4.6kB. Future work into more extensive deletion, or exploring truncations in other variants of Cas9 could potentially result in even shorter dCas9 variants. A smaller transcriptional activator may also be considered, since the tripartite VPR utilized here is essentially a fusion of 3 transcriptional activators. Other effectors may also be fused to our truncated dCas9, achieving other novel functionalities, such as KRAB for CRISPRi and epigenetic regulators, for therapy. </P><P>In this expansion of our project, we have demonstrated a potential real-world application on how our truncated dCas-VPR may be utilized to treat Factor VII deficiency and HBB aberrant diseases such as SCD and beta thalassemia. We feel that continued research in this direction will increase the profile of novel Cas9 applications to the public, in addition to introducing new and better treatment options to sufferers of these serious, lifelong afflictions.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>