<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTU_SINGAPORE_HP_Gold_Integrated skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTU SINGAPORE/HP/Gold Integrated</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="w3-bar" id="myNavbar"><DIV class="w3-top w3-padding-large w3-transparent"><A href="https://2017.igem.org/Team:NTU_SINGAPORE" class="w3-bar-item w3-button"><FONT color="#800020"> HOME </FONT></A><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">TEAM</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Team" class="w3-bar-item w3-button"><FONT color="#800020">Our Team</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Collaborations" class="w3-bar-item w3-button"><FONT color="#800020">Collaboration</FONT></A></DIV></CLASS></DIV><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">PROJECT</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Description" class="w3-bar-item w3-button"><FONT color="#800020">Description</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Experiment" class="w3-bar-item w3-button"><FONT color="#800020">Experiment</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Results" class="w3-bar-item w3-button"><FONT color="#800020">Proof of Concept</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Demonstrate" class="w3-bar-item w3-button"><FONT color="#800020">Demonstrate</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Improve" class="w3-bar-item w3-button"><FONT color="#800020">Improve</FONT></A></DIV></CLASS></DIV><A href="https://2017.igem.org/Team:NTU_SINGAPORE/InterLab" class="w3-bar-item w3-button"><FONT color="#800020">INTERLAB STUDY</FONT></A><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">PARTS</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Basic_Part" class="w3-bar-item w3-button"><FONT color="#800020">Basic Parts</FONT></A></DIV></CLASS></DIV><DIV class="w3-dropdown-hover"><CLASS><BUTTON class="w3-button"><FONT color="#800020">HUMAN PRACTICE</FONT></BUTTON><DIV class="w3-dropdown-content w3-bar-block w3-card-4"><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/OurStory" class="w3-bar-item w3-button"><FONT color="#800020">Our Story</FONT></A><FONT><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/Silver" class="w3-bar-item w3-button"><FONT color="#800020">Silver</FONT></A><FONT><A href="https://2017.igem.org/Team:NTU_SINGAPORE/HP/Gold_Integrated" class="w3-bar-item w3-button"><FONT color="#800020">Gold</FONT></A></FONT></FONT></DIV></CLASS></DIV><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Attributions" class="w3-bar-item w3-button"><FONT color="#800020">ATTRIBUTIONS</FONT></A><A href="https://2017.igem.org/Team:NTU_SINGAPORE/Contact Us" class="w3-bar-item w3-button"><FONT color="#800020"> CONTACT US</FONT></A><DIV id="navDemo" class="w3-bar-block w3-white w3-hide w3-hide-large w3-hide-medium"><A href="#home" class="w3-bar-item w3-button" onclick="toggleFunction()">HOME</A><A href="#research" class="w3-bar-item w3-button" onclick="toggleFunction()">RESEARCH</A><A href="#interlab study" class="w3-bar-item w3-button" onclick="toggleFunction()">INTERLAB STUDY</A><A href="#parts" class="w3-bar-item w3-button" onclick="toggleFunction()">PARTS</A><A href="#collaboration" class="w3-bar-item w3-button" onclick="toggleFunction()">COLLABORATION</A><A href="#contact" class="w3-bar-item w3-button" onclick="toggleFunction()">CONTACT</A></DIV></DIV><DIV class="ex column full_size"><DIV class="ex w3-container w3-padding-32" id="Gold Medal and Integrated Human Practices"><H3 class="w3-border-bottom w3-border-light-grey w3-padding-16">Gold Medal and Integrated Human Practices</H3><PSTYLE> Steps we took to address the concerns of the community.</PSTYLE></DIV><P>After the workshop conducted at one-north festival, we requested the public to participate in a survey. This survey was designed to gauge public interest and reservations about the use of Cas9 for therapeutic applications.</P><P>An overwhelmingly high number (85%) of participants is receptive to using a fully mature CRISPR/Cas9 technology – that is without any side effects. </P><P>When asked to consider whether they would be receptive to using CRISPR/Cas9 technology in the event of personal suffering to lung cancer – even when the issues of the treatment have yet to be fully worked out, more than half stated that they would, while 30% would consider the treatment. </P><P>However, with the current state of CRISPR/Cas9 technology, half of the public would not utilize this technology. 40% cited worries about side effects of this technology – such as potential off-target effects. </P><P>Being aware and motivated by the concerns of the majority, we decided to extend our EGFR project as an integration of public opinions into our research so that we can derive a more thoughtful conclusion.</P><P> After the workshop, we realized that we may have overlooked the possibility of off-targets. We then brainstormed and came to the idea of targeting our single-guide RNAs (sgRNAs) to wild-type EGFR gene to investigate the possibility of the off-target effect that CRISPR-Cas system could introduce since the desirable CRISPR-Cas systems should only target and perform their cutting work on the mutated allele, not on the wild-type EGFR allele. This is of concern since cancer-causing mutations in the EGFR allele includes deletions, and our design of sgRNAs may not be specific enough for disease allele. </P><P>We then designed our experiments to test the same 10 sgRNAs in the A549 cell line (sgRNAs that we previously tested in PC9 cells, more details are on our Project tab), while the remaining of the workflow was kept similar. A549 cells are derived from lung carcinomatous tissue and harbour the wild-type EGFR gene.</P><P>The T7E1 results for A549 cells with 24h transfection time were all negative as we expected, suggesting that CRISPR-Cas could not target the wild-type EGFR gene with all the sgRNAs designed. Surprisingly, positive T7E1 results were observed for some sgRNAs when A549 cells were transfected with 48h time, which may indicate that CRISPR-Cas could possibly tolerate the mutation and perform cleavage. Another parameter which could matter is the transfection time, since the cuts were only observed after 48h transfection time but not in 24h transfection time. This could mean that longer transfection time may affect the accuracy of CRISPR-Cas as well.</P><P>However, we cannot be confident enough to conclude that this indicates the off-target effect of CRISPR-Cas, since there were no duplicates generated due to time constraint. Besides carrying out duplicates, the confirmation of off-targets can be carried out using various techniques such as using in-vitro cutting assay or using different cell lines such as primary cell lines and HEK293 cells.</P><P>Over-generalized conclusion is a concern in scientific research as it may lead to the generation of insignificant results. An example of such a publication will be the paper titled, “Unexpected mutations after CRISPR-Cas9 editing in-vivo”.</P><P>In the paper, the authors conclude that CRISPR-Cas9 editing in blind mice that is supposed to only correct a mutation in the Pde6b gene also causes a large number of mutations in untargeted regions. It is because they found an unexpectedly high number of indels and single nucleotide polymorphisms (SNPs) in the CRISPR-treated mice. This conclusion is based on the assumption that the 3 mice (1 control mouse and 2 CRISPR-Cas9 treated mice) used as the subjects of the experiment were genetically identical prior to the treatment. </P><H2>Data Analysis Methodology</H2><P>To validate if the claims are really true we attempted to reidentify the SNPs and indels by using Genome Analysis ToolKit (GATK) best practices. Three mice are used as the subjects of the experiment (1 control mouse: FVB and 2 CRISPR-Cas9 treated mice: F03 and F05). Firstly, we realigned the raw sequencing data obtained from SRR5450996 (FVB control mouse), SRR5450997 (F03), SRR5450998 (F05) with mouse reference genome, mm10, by using Burrows-Wheeler Aligner (bwa mem), resulting in a BAM file. The duplicates are then marked and removed using MarkDuplicates tool from Picard. The following steps, AddOrReplaceReadGroups. Information like read group ID, read group library, read group platform, and sample name are added. This step is added because it is required by GATK pipeline for SNP and indel calling in later steps. SNP calling is locating SNPs in comparison of reference genome while indel detects the insertions and deletions with respect to the reference genome. After this step, the sequencing is now ready to be fed into GATK pipeline.</P><P>Firstly, the BAM files are realigned to produce better local alignments by using RealignerTargetCreator, IndelRealigner, Base recalibrator and PrintReads to create the final BAM file. SNPs and indels are called by using HaplotypeCaller, with mouse dbSNP142 as the reference database. The resulting output is a vcf file containing the location of indels and SNPs. SNPs are extracted from the callsets by setting SNP as the selection type for SelectVariants. The subsequent step is to apply a filter to the SNPs, the filtering is done by using hard filtering with the parameters recommended by GATK.</P><P>There are 61034 numbers of shared SNPs between F03 and F05 mice, that are not found in FVB mouse, and therefore are unique SNPs between F03 and F05 mice. In addition, there are 10837 SNPs between FVB and F05 mouse, and only 9236 number of SNPs shared between FVB and F03 mouse. The bar graphs below show the distribution of number of SNPs across different chromosomes.</P><P>This finding shows that CRISPR-Cas9 treated F03 and F05 mice share more common SNPs/variants than the control FVB mouse. As the author’s conclusion relies on the assumption that all three mice were genetically identical prior to treatment and that the mice were from a highly inbred strain, we would expect that most common variants existing prior to the treatment should be found in all three mice, not only at the two treated mice but at not the control mouse. Therefore, it suggests that F03 and F05 mouse had been more genetically related to each other than the FVB mouse prior to the treatment. The unexpected mutations identified in the original article simple represent pre-existing SNPs shared in common by the mice, and not due to of off-target mutations caused by CRISPR-Cas9. </P><P>To reiterate, our analysis of the same set of data revealed that the FVB control mouse is very different from the sample mice F03 and F05 – suggesting that they are from different parents/lineages. FVB would have sufficient natural mutations to make them sorely inadequate as negative genetic controls. Mutations observed in CRISPR-Cas9 treated mice F03 and F05 when contrasted against the FVB control mouse is most likely not due to CRISPR-Cas9 treatment. </P><P>Since then, a number of articles have been published to rebut this claim and the paper had received a second editorial. Although this paper has since been revised from publication, there are still some news article citing this paper while circulating claims that CRISPR-Cas9 causes unexpected mutations (as per 16 October 2017).</P><P>Such inaccurate news might cause misinformation to the general public and make them more wary of new technologies. Even when provided access to the source papers, the public – even the news writers generally lack the specialized knowledge to critically analyze claims. In conclusion, these controversies caused by this article served as a reminder for the science community to perform safe and responsible work. </P><P>Careful, safe and responsible experimental design and cross-checking of preliminary results before even considering publication is important in academia, even with the peer-review system in place. This is especially so for topics that have high potential to be sensationalized – such as Cas9 technologies. This would aid in fostering trust and confidence in emerging technologies such as CRISPR/Cas systems.</P><H2>Conclusion</H2><P>In the end, further experiments are required to confirm or deny the off-target effects observed in 48H A549 CRISPR/Cas treatment. Besides carrying out duplicates, the confirmation of off-target effects in the observed locus can be carried out with alternative methods, such as retesting in other EGFR wild-type cell lines, or even primary cell lines. The in-vitro cutting assay may also be tested. Bioinformatics tool was also proven to be valuable to evaluate any off-target events. Similar pipeline may be employed to accompany off-target analysis so that the evaluation is more accurate. It is possible to be implemented in our HDR project in the future. This would ensure that the HDR event will only detect the target sequence without considerable off-target editing</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>